<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-10-28 06:32:37 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental and Molecular Medicine, Experimental & Molecular Medicine</td>
          <td>4</td>
          <td>48</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum.
                </a>
              </td>
          <td>
            Gustavo S França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>8</td>
          <td>41</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41df9adbcb86e982509871382b3eba588480f4fb" target='_blank'>
              Base editing screens define the genetic landscape of cancer drug resistance mechanisms.
              </a>
            </td>
          <td>
            Matthew A Coelho, Magdalena E Strauss, Alex Watterson, Sarah Cooper, S. Bhosle, G. Illuzzi, Emre Karakoc, Cansu Dincer, Sara F Vieira, Mamta Sharma, Marie Moullet, Daniela Conticelli, Jonas Koeppel, Katrina McCarten, C. M. Cattaneo, Vivien Veninga, Gabriele Picco, Leopold Parts, J. Forment, E. Voest, J. Marioni, Andrew Bassett, Mathew J. Garnett
          </td>
          <td>2024-10-18</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6866382555382de9b8b8e3c83922b3e82d715085" target='_blank'>
              Defining heritability, plasticity, and transition dynamics of cellular phenotypes in somatic evolution.
              </a>
            </td>
          <td>
            Joshua S. Schiffman, Andrew R D'Avino, Tamara Prieto, Yakun Pang, Yilin Fan, Srinivas Rajagopalan, Catherine Potenski, Toshiro Hara, M. Suvà, Charles Gawad, Dan A Landau
          </td>
          <td>2024-09-24</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33f7b85ef899c9135146a0af70ef75b42923dd2f" target='_blank'>
              Optical genome mapping of structural variants in Parkinson’s disease-related induced pluripotent stem cells
              </a>
            </td>
          <td>
            Joanne Trinh, Susen Schaake, C. Gabbert, Theresa Lüth, Sally A Cowley, André Fienemann, Kristian K Ullrich, Christine Klein, Philip Seibler
          </td>
          <td>2024-10-19</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c3db9b2bc9f499477ff494631d94e30c054c0bf" target='_blank'>
              A Boolean model explains phenotypic plasticity changes underlying hepatic cancer stem cells emergence
              </a>
            </td>
          <td>
            Alexis Hernández-Magaña, Antonio Bensussen, Juan Carlos Martínez-García, E. Álvarez-Buylla
          </td>
          <td>2024-09-02</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf5407072a36736a7b97ab6c58b0a5238e234714" target='_blank'>
              Characterizing the allele-specific gene expression landscape in high hyperdiploid acute lymphoblastic leukemia with BASE
              </a>
            </td>
          <td>
            Jonas Andersson, Efe Aydın, Rebeqa Gunnarsson, H. Lilljebjörn, T. Fioretos, B. Johansson, K. Paulsson, Minjun Yang
          </td>
          <td>2024-10-05</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Tumors consist of cancer cells with different genetic, epigenetic, and phenotypic properties. Cancer stem cells are an important subpopulation of heterogeneous cancer cells and are capable of initiating and propagating tumors. The term cancer stem cells has become broader in efforts to understand their phenotypic plasticity to switch fates between self-renewal and differentiation. Cancer stem cell plasticity is significantly associated with the initiation of metastasis, resistance to therapy, and tumor recurrence. With our broadened knowledge of epigenetic regulation and metabolic reprogramming as key elements enabling such capabilities, an expansive body of literature has demonstrated the functional importance of each element in contributing to cancer stem cell characteristics. Recently, the direct interplay between epigenetic regulation and metabolic reprogramming has begun to be appreciated in the context of cancer stem cells with growing interest. In this review, we discuss the mechanisms by which cancer stem cells orchestrate the reciprocal regulation of cellular metabolism and epigenetic alterations. In the discussion, compelling, unanswered questions on this topic have been elaborated for the interest of the research community and how recent technological developments help tackle such research ideas. A comprehensive understanding of cancer stem cell attributes that are largely governed by epigenetic and metabolic reprogramming would enable the advancement of precise therapeutic options and the prediction of better responses to drugs, holding great promise in cancer treatment and cure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a352f514fc793046331539c8fc8b91b79542075" target='_blank'>
              Interplay between epigenetics and metabolism controls cancer stem cell plasticity
              </a>
            </td>
          <td>
            Jee-Eun Choi, Inwha Baek
          </td>
          <td>2024-09-11</td>
          <td>Frontiers in Epigenetics and Epigenomics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Understanding the mechanisms by which cancer cells switch between different adaptive states and evade therapeutic interventions is essential for clinical management. In this study, the in vivo cellular dynamics of a new chronic myeloid leukaemia cell line displaying altered phenotype and resistance to tyrosine kinase inhibitors were investigated in correlation with their parental cells for invasiveness/metastasis, angiogenic potential and population kinetics. We showed that the cells exhibiting drug resistance and plastic phenotype possess an increased capacity for invasion compared to their parental cells, that exposure to imatinib mesylate has the potential to enhance cellular motility and that in a leukaemic cell population, even a minority of plastic cells exhibit improved migratory ability. Furthermore, we show that these plastic cells have angiogenic and extravasation potential. The present study provides significant insights into the cellular dynamics displayed by a TKI‐resistant, phenotypically plastic CML cell line, using a zebrafish (Danio rerio) xenograft model.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8debdaa7aac5be72ea83fbcd5d951e3b97abea7b" target='_blank'>
              Phenotypically plastic drug‐resistant chronic myeloid leukaemia cell line displays enhanced cellular dynamics in a zebrafish xenograft model
              </a>
            </td>
          <td>
            Seda Baykal, Zeynep Yuce, G. Ozhan
          </td>
          <td>2024-10-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c063409361f89f55a497cfd8fcbb23f7e584a467" target='_blank'>
              Deciphering genetic and nongenetic factors underlying tumour dormancy: insights from multiomics analysis of two syngeneic MRD models of melanoma and leukemia
              </a>
            </td>
          <td>
            Marie-Océane Laguillaumie, Sofia Titah, Aurel Guillemette, Bernadette Neve, Frédéric Leprêtre, Pascaline Ségard, F. Shaik, Dominique Collard, Jean-Claude Gerbedoen, L. Fléchon, L. Hasan Bou Issa, Audrey Vincent, M. Figeac, Shéhérazade Sebda, C. Villenet, Jérôme Kluza, William Laine, Isabelle Fournier, Jean-Pascal Gimeno, M. Wisztorski, Salomon Manier, M. Tarhan, Bruno Quesnel, Thierry Idziorek, Yasmine Touil
          </td>
          <td>2024-09-03</td>
          <td>Biological Research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="
 Neuroblastoma (NB) is the most common pediatric solid tumor, responsible for about 15% of childhood cancer-related mortality in the United States. Multiple therapies have been established, which include chemotherapy, radiotherapy, immunotherapy and molecularly-targeted therapy, however resistance to these therapies is the major hinderance in curing NB. RAS pathway mutations are prevalent in relapsed NB and are associated with a poorer overall survival. Therefore, comprehending the mechanisms leading to therapy resistance has become an urgent need. In this study, we attempted to identify the resistance mechanism of NB cell lines towards RAS inhibitors. RAS-altered NB cell lines are treated with trametinib, a MAP kinase pathway inhibitor to perform cell viability assays, immunoblotting and RT PCR. Copy number alteration and translocation studies are performed using spectral karyotyping (SKY) and multiplex interphase fluorescent in-situ hybridization (miFISH). We observed that unlike other RAS-altered NB cell lines (SK-N-AS, CHP-212, LA-N-6, SK-N-BE(2)-C), NB-Eb-C1, which harbors a KRAS G12D mutation shows resistance to both MRTX1133 (KRAS G12D direct inhibitor) and trametinib. Immunoblot against crucial genes regulating NB showed high expression of MYCN, the most relevant biomarker of high-risk neuroblastoma. MYCN is unfortunately undruggable hence, standard treatment for MYCN-amplified neuroblastoma patients does not include targeting MYCN itself. We therefore pursued a hypothesis-driven approach towards identifying possible mechanisms responsible for MYCN driven MEKi-resistance in RAS pathway-altered NB cells. MYCN is known to cross talk with the RAS pathway, however this mechanism is not well understood in NB. Therefore, we performed the clonal assessment of NB cell lines using miFISH and to the best of our knowledge are the first to report that, NB-Eb-C1 has a hypertriploid to hypotetraploid baseline with a low copy-number gain of MYCN. To have a profound understanding of the functionality of MYCN in NB, we have taken 3 different NB cell lines with varied MYCN status. CHP-212 (MYCN amplified-extrachromosomal), SK-N-BE(2)-C (MYCN amplified-homogenously staining region), SK-N-AS (MYCN neutral). SKY and FISH revealed that the MYCN copy-number gain in NBEB cells is the result of a translocation between chromosome 2 and 5 involving MYCN. Of note, another frequently abnormally expressed gene in NB, ALK, is co-translocated with MYCN suggesting potential involvement of both genes in imparting chemoresistance. To understand the mechanism of MYCN overexpression in NBEB we will be performing HiC to check for promoter/enhancer hijacking. Also, to better define this translocation and studying other translocations involved we are performing optical genome mapping using BioNano. Other functional experiments will be performed to look at the impact of the RAS/MAPK alteration on MYCN protein stability. We are confident that this study will contribute towards developing new combinations of targeted therapies to improve outcomes in RAS-altered NB.
 Citation Format: Subhra Dash, Kerstin Heselmeyer-Haddad, Stacy Stauffer, Lucas Stauffer, Danny Wangsa, Marielli E. Yohe. Investigating mechanisms responsible for MAP kinase pathway resistance in RAS-altered neuroblastoma cells [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A053.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b3e4035267dd00bede1ab23d0c552e939dc4a96" target='_blank'>
              Abstract A053 Investigating mechanisms responsible for MAP kinase pathway resistance in RAS-altered neuroblastoma cells
              </a>
            </td>
          <td>
            Subhra Dash, Kerstin Heselmeyer-Haddad, Stacy Stauffer, Lucas Stauffer, Danny Wangsa, Marielli E. Yohe
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) remains a deadly disease with few effective treatments. Even with the advent of mutationally-directed therapies targeting the KRAS oncogene, PDAC transcriptional plasticity remains a major mechanism of therapeutic resistance. As a result, there is a desperate need to develop additional strategies to treat this deadly disease. In addition to the canonical genomic alterations in KRAS, TP53, CDKN2A, and SMAD4, PDAC tumors exhibit diverse transcriptional programs, or "cell states," with prognostic implications: PDAC tumors in the “classical state” are more responsive to upfront chemotherapy and have better outcomes, while those in the “basal state” are more aggressive with poorer prognoses. Currently, PDAC cell states are not used to direct therapy. Cell state is an integrative property shaped by both cell-intrinsic (e.g., mutations, epigenetics) and cell-extrinsic (e.g., tumor microenvironment) factors. In our prior work, single-cell analyses of clinical PDAC samples demonstrated notable cell state heterogeneity and plasticity. In addition, longitudinal monitoring of patient-derived organoids showed that cells can shift from classical to basal or coexpressor states as the environment changes, and that these cell states can drastically influence drug response. However, it remains unknown which signaling and transcriptional pathways induce basal or classical states, and whether these gene regulatory networks can be targeted therapeutically. Thus, there is a pressing need to devise scalable methods to determine essential drivers for prognostically relevant cancer cell states.Here, we aim to systematically and unbiasedly identify transcription factors (TFs) that drive basal and classical cell states in PDAC. We have successfully established a high-throughput screening pipeline to 1) over-express all human TF isoforms in a pooled format within a cohort of PDAC models; 2) sort cells using antibodies against state-specific cell surface markers; and 3) perform single-cell RNA sequencing on sorted cells to nominate individual TFs or combinations that induce specific cell state transitions. Preliminary results identify both previously reported (e.g., GATA4) and new TFs that can induce the classical state. By overexpressing TFs individually or in combination, we are able to engineer PDAC cell line models that more faithfully reflect the transcriptional phenotypes observed in patient tumors. Ultimately, we aim to use this approach to construct high-fidelity isogenic but state-variant PDAC models for use both in therapeutic screening and as a substrate for investigating mechanisms governing cancer cell state plasticity.
 Citation Format: Yuzhou Evelyn Tong, Aswanth Mahalingam, Walaa E Katan, Julia Joung, Alex K Shalek, Peter S Winter, Srivatsan Raghavan. Identifying Transcriptional Drivers of Plasticity in Pancreatic Cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C066.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c88d4d8016aad9bbc47795f53500dd4b1940096b" target='_blank'>
              Abstract C066: Identifying Transcriptional Drivers of Plasticity in Pancreatic Cancer
              </a>
            </td>
          <td>
            Yuzhou Evelyn Tong, Aswanth Mahalingam, Walaa E. Katan, Julia Joung, Alex K. Shalek, Peter S. Winter, Srivatsan Raghavan
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Glioblastoma stands as the most prevalent and aggressive primary brain tumor. The current treatment regimen involves a combination of radiotherapy and chemotherapy post-surgical tumor resection. However, for most patients, tumor recurrence transpires with a more aggressive phenotype. Tumor necrosis factor-related apoptosis-inducing ligand (TRAIL) holds promise as it selectively triggers apoptosis in tumor cells, rendering it an attractive therapeutic option. Nonetheless, numerous tumor cells, glioblastoma included, can develop acquired resistance against TRAIL therapy, yet the underlying mechanisms remain poorly elucidated. Apart from resistance to apoptosis, these mechanisms may also encompass non-genetic factors stemming from intracellular dynamics. In our previous research on generating models resistant to TRAIL, we observed differences in the expression of metabolic pathway genes in TRAIL-resistant glioblastoma cells. Our transcriptomic analysis revealed a positive correlation among pathways related to cellular metabolism in these resistant cells. Further examination of this gene family revealed common alterations across various cell lines. Additionally, to delve deeper into the role of cellular metabolism, we conducted in vivo CRISPR/Cas9 screening using a focused library targeting metabolism. We also compared mitochondria densities using MitoTracker dye between sensitive and resistant GBM cells, noting a significant increase in resistant cells. Using the Seahorse Extracellular Flux Analyzer, we measured cellular metabolism activity and observed higher oxygen consumption in resistant cells, indicative of elevated energy levels. Based on these preliminary findings, our study will undergo thorough genetic and metabolomic screening, promising significant insights into cancer metabolism. The identified target genes and metabolites could potentially serve as therapeutic targets for developing novel cancer treatments, particularly glioblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/518a0308c77c26c6504aaa96145e6989538e7b80" target='_blank'>
              P04.02.A DISCOVERY OF NEW METABOLIC PATHWAYS IMPLICATED IN THERAPY RESISTANCE IN GLIOBLASTOMA (GBM)
              </a>
            </td>
          <td>
            A. Cingöz
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfbcb18aaf8922f16ad19ad9eeb7b0ec8bc2789d" target='_blank'>
              Serial single-cell RNA sequencing unveils drug resistance and metastatic traits in stage IV breast cancer
              </a>
            </td>
          <td>
            K. Otsuji, Yoko Takahashi, Tomo Osako, Takayuki Kobayashi, T. Takano, S. Saeki, Liying Yang, Satoko Baba, Kohei Kumegawa, Hiromu Suzuki, Tetsuo Noda, Kengo Takeuchi, Shinji Ohno, T. Ueno, Reo Maruyama
          </td>
          <td>2024-10-03</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14f62e543f9ec01d9f73f5eef49615f95671e727" target='_blank'>
              Cancer drug-tolerant persister cells: from biological questions to clinical opportunities.
              </a>
            </td>
          <td>
            Mariangela Russo, Mengnuo Chen, Elisa Mariella, H. Peng, Sumaiyah K Rehman, E. Sancho, Alberto Sogari, Tzen S Toh, Nathalie Q Balaban, Eduard Batlle, R. Bernards, Mathew J. Garnett, Matthew J Hangauer, E. Leucci, J. Marine, Catherine A O'Brien, Yaara Oren, E. Patton, C. Robert, Susan M Rosenberg, Shensi Shen, A. Bardelli
          </td>
          <td>2024-09-02</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e459f679b760cbc6c5f4d15f5d578e006d0e8b9" target='_blank'>
              Establishment and characterization of 18 Sarcoma Cell Lines: Unraveling the Molecular Mechanisms of Doxorubicin Resistance in Sarcoma Cell Lines
              </a>
            </td>
          <td>
            Young-Eun Cho, Soon-Chan Kim, Ha Jeong Kim, Ilkyu Han, Ja-Lok Ku
          </td>
          <td>2024-10-02</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Neuroblastoma is the most prevalent solid tumor in early childhood, with a 5-year overall survival rate of 40–60% in high-risk cases. Therefore, the identification of novel biomarkers for the diagnosis, prognosis, and therapy of neuroblastoma is crucial for improving the clinical outcomes of these patients. In this study, we conducted the whole-exome sequencing of 48 freshly frozen tumor samples obtained from the Biobank. Somatic variants were identified and selected using a bioinformatics analysis pipeline. The mutational signatures were determined using the Mutalisk online tool. Cancer driver genes and druggable mutations were predicted using the Cancer Genome Interpreter. The most common mutational signature was single base substitution 5. MUC4, MUC16, and FLG were identified as the most frequently mutated genes. Using the Cancer Genome Interpreter, we identified five recurrent cancer driver mutations spanning MUC16, MUC4, ALK, and CTNND1, with the latter being novel and containing a missense mutation, R439C. We also identified 11 putative actionable mutations including NF1 Q1798*, Q2616*, and S636X, ALK F1174L and R1275Q, SETD2 P10L and Q1829E, BRCA1 R612S, NOTCH1 D1670V, ATR S1372L, and FGFR1 N577K. Our findings provide a comprehensive overview of the novel information relevant to the underlying molecular pathogenesis and therapeutic targets of neuroblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61b01557419f7eaa825838a38b11cbcc7b2c62d9" target='_blank'>
              Whole-Exome Sequencing Reveals Novel Candidate Driver Mutations and Potential Druggable Mutations in Patients with High-Risk Neuroblastoma
              </a>
            </td>
          <td>
            Giorgio D’Alessandris, C. Ronchi, Natakorn Nokchan, Praewa Suthapot, Pongsakorn Choochuen, Natthapon Khongcharoen, S. Hongeng, U. Anurathapan, K. Surachat, S. Sangkhathat, Thai Pediatric Cancer Atlas Tpca, Consortium
          </td>
          <td>2024-09-01</td>
          <td>Journal of Personalized Medicine</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/401dfb6b1e61a2b7931608ee7b632746b618204c" target='_blank'>
              Potential promising of synthetic lethality in cancer research and treatment.
              </a>
            </td>
          <td>
            Mohsen Karami Fath, Behnam Najafiyan, Reza Morovatshoar, Mahdieh Khorsandi, Adib Dashtizadeh, Arash Kiani, Farnoosh Farzam, Kimia Sadat Kazemi, Mohsen Nabi Afjadi
          </td>
          <td>2024-09-21</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Telomeres are critical nucleoprotein structures that safeguard chromosome ends from degradation and fusion, thereby ensuring genomic stability. Without proper maintenance, telomeres progressively shorten with each cell division, leading to dysfunction and triggering chromosomal instability. This dysfunction is a significant driver of tumorigenesis, primarily by facilitating genetic alterations such as aneuploidy, gene amplifications, and chromosomal rearrangements. Cancer cells bypass cellular aging by activating mechanisms like telomerase reactivation or the Alternative Lengthening of Telomeres (ALT) pathway to maintain telomere length. This review explores the mechanisms by which telomere dysfunction contributes to genomic instability and cancer progression, including telomere shortening, breakage-fusion-bridge (B/F/B) cycles, and the ALT pathway. Additionally, it addresses the therapeutic potential of targeting telomere maintenance, highlighting current strategies like telomerase and ALT inhibitors. However, developing telomere-based therapies presents challenges, including resistance mechanisms, off-target effects, and potential impacts on normal stem cells. Emerging research areas such as the development of biomarkers and combination therapies offer promising directions for overcoming these challenges. Understanding telomere dynamics provides novel opportunities to exploit cancer cell vulnerabilities and advance treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04b7c5196ace0bbc6bf2d90a9a3ba826ad745b1e" target='_blank'>
              Mechanisms of telomere dysfunction in cancer from genomic instability to therapy: A review
              </a>
            </td>
          <td>
            Amir Mohammad Karimi Forood
          </td>
          <td>2024-09-30</td>
          <td>International Journal of Science and Research Archive</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is an aggressive hematologic neoplasia with a complex polyclonal architecture. Among driver lesions, those involving the FLT3 gene represent the most frequent mutations identified at diagnosis. The development of tyrosine kinase inhibitors (TKIs) has improved the clinical outcomes of FLT3-mutated patients (Pt). However, overcoming resistance to these drugs remains a challenge. To unravel the molecular mechanisms underlying therapy resistance and clonal selection, we conducted a longitudinal analysis using a single-cell DNA sequencing approach (MissionBioTapestri® platform, San Francisco, CA, USA) in two patients with FLT3-mutated AML. To this end, samples were collected at the time of diagnosis, during TKI therapy, and at relapse or complete remission. For Pt #1, disease resistance was associated with clonal expansion of minor clones, and 2nd line TKI therapy with gilteritinib provided a proliferative advantage to the clones carrying NRAS and KIT mutations, thereby responsible for relapse. In Pt #2, clonal architecture was less complex, and 1st line TKI therapy with midostaurin was able to eradicate the leukemic clones. Our results corroborate previous findings about clonal selection driven by TKIs, highlighting the importance of a deeper characterization of individual clonal architectures for choosing the best treatment plan for personalized approaches aimed at optimizing outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4202a0b52113f492de1c5d891aefedd2f36b44c" target='_blank'>
              Tracking Response and Resistance in Acute Myeloid Leukemia through Single-Cell DNA Sequencing Helps Uncover New Therapeutic Targets
              </a>
            </td>
          <td>
            S. Bruno, E. Borsi, A. Patuelli, L. Bandini, M. Mancini, D. Forte, J. Nanni, Martina Barone, Alessandra Grassi, G. Cristiano, C. Venturi, V. Robustelli, Giulia Atzeni, Cristina Mosca, S. De Santis, C. Monaldi, A. Poletti, C. Terragna, A. Curti, M. Cavo, Simona Soverini, E. Ottaviani
          </td>
          <td>2024-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>1</td>
          <td>42</td>
        </tr>

        <tr id="
 We aim to offer new insights on the regulation of metabolic plasticity in renal cell carcinoma (RCC) and identify novel metabolic regulatory targets that can be used for potential treatment therapies of the disease. RCC affects an estimated 400,000 people worldwide each year with over 100,00 deaths annually that disproportionately affects racial/ethnic minorities in the United States. RCC is a term that covers an array of kidney cancer subtypes differing in incidence, histopathology, genetic and molecular alterations, as well as in clinical outcomes and prognoses. However, a common characteristic of many RCCs is that they are driven by metabolic rewiring, due to a high frequency of mutations in genes that regulate major metabolic processes in the cell. It was recently discovered that patient overall survival and prognosis has been linked to the expression of certain metabolic signatures in RCC tumor cells. Metabolic plasticity allows cancer cells to survive and adapt to different environments and metabolic stress conditions, such as nutrient deprivation. Therefore, exploring and exploiting metabolic rewiring in these cells may reveal genetic vulnerabilities and dependencies. This could lead to discovery of novel biomarkers and molecular targets that can be implicated as future therapeutic interventions for this disease. To this end, we applied functional genomic screening to expose genetic dependencies and vulnerabilities under different physiologic and metabolic conditions in patient derived medullary (PDM) RCC cells. The medullary RCC is a rare and aggressive RCC subtype that disproportionately affects the African American/Black population. We applied a CRISPR- based combinatorial screening platform that is based on co-expression of Cas9 and Cas12a nucleases and libraries of hybrid guide RNAs (hgRNA) to achieve ultra-efficient gene knockout (Aregger et al., 2021; Gonatopoulos-Pournatzis et al., 2020) in PDM RCC cells. Additionally, cells were screened under different media conditions that mimic human physiological nutrient conditions and a media deprived of lipids, one of the major metabolic substrates of the cell. By utilizing a genome-wide pooled CRISPR gene knockout screen we were able to systematically uncover genes required for cell survival and proliferation in a panel of patient-derived RCCs. We have begun to uncover genetic dependencies and major metabolic pathways that can be elucidated and exploited for future studies, such as those involving ferroptosis which is a targetable pathway for the development of cancer treatment therapies. Our findings demonstrate some of the key genetic drivers underlying cellular adaptations to different physiological environments and nutrient deprivation. In future studies we will further explore the relationship between different key metabolic regulatory genes in RCC and map for genetic interactions in major metabolic pathways.
 Citation Format: Chelsee Holloway, Josef Horak, Youngkyu Jeon, Michael Aregger. Uncovering genetic regulators of metabolic plasticity in renal cell carcinoma [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr C091.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cd9b94600307109e3720c07a3b1a13c91abd8ae" target='_blank'>
              Abstract C091: Uncovering genetic regulators of metabolic plasticity in renal cell carcinoma
              </a>
            </td>
          <td>
            Chelsee Holloway, Josef Horak, Youngkyu Jeon, Michael Aregger
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The ability of cancer cells to change and adapt poses a critical challenge to identifying curative solutions. Tumor evolution has been extensively studied from a genetic perspective, to guide clinicians in selecting the most appropriate therapeutic option based on a patient's mutational profile. However, several studies reported that tumors can evolve toward more aggressive stages or become resistant to therapies without changing their genetic makeup. Indeed, several cell-intrinsic and cell-extrinsic mechanisms contribute to tumor evolution. In this viewpoint, I focus on how chromatin, epigenetic, and transcriptional changes contribute to tumor evolution, allowing cancer cells to transition to different cell states and bypass response to therapies. Although tumor nongenetic evolution is harder to trace and predict, understanding its principles might open new therapeutic opportunities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0869302e828b2c81eac62f89984d51de011a3b74" target='_blank'>
              Nongenetic evolution of the tumor: from challenges to new therapeutic opportunities.
              </a>
            </td>
          <td>
            E. Oricchio
          </td>
          <td>2024-10-18</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Translocations have largely been implicated in tumor development. However, beyond the consequences of aberrant gene expression near the breakpoint, their effects remain unexplored. In this work, we characterize the interplay between translocations, chromatin organization and gene expression using mantle cell lymphoma (MCL) as a model. We show that in vitro induced MCL-associated translocations can drive transcriptional changes at entire chromosome arms affecting multiple genes in a regulon-like fashion. Additionally, overexpressed genes in MCL patients are enriched in the exact same genomic regions, further underlining its potential relevance for lymphomagenesis. Moreover, we demonstrate a clear link between the translocation-induced transcriptional alterations and genome organization, with genes most susceptible to change expression residing in pre-existing long-range interacting loops spanning 50 megabases. The translocation places the strong immunoglobulin enhancer into this loop, allowing the spread of its regulatory potential over the entire affected chromosome arm. Finally, we show that translocation-induced effects mainly represent expression enhancement of genes already active prior to translocation formation, highlighting the importance of the epigenetic state of the cell in which this initial hit occurs. In summary, we show that translocations can induce ample, simultaneous gene expression changes affecting entire chromosome arms, representing an important new mechanism for tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bdbb3468215c566e38e0c9ba35afc86f4ee62c1" target='_blank'>
              Translocations can drive expression changes of multiple genes in regulons covering entire chromosome arms
              </a>
            </td>
          <td>
            Anna Oncins, Roser Zaurin, Houyem Toukabri, Kimberly Quililan, José R Hernández Mora, Magdalena A. Karpinska, Erik Wernersson, Alastair Smith, Agostina Bianchi, Leone Albinati, Andrea Rivero, François Serra, Raúl Gómez, Cristina López, S. Beà, Nadia Halidi, A. Valencia, Magda Bienko, A. M. Oudelaar, Renée Beekman
          </td>
          <td>2024-10-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Therapy resistance in breast cancer is increasingly attributed to polyploid giant cancer cells (PGCCs), which arise through whole-genome doubling and exhibit heightened resilience to standard treatments. Characterized by enlarged nuclei and increased DNA content, these cells tend to be dormant under therapeutic stress, driving disease relapse. Despite their critical role in resistance, strategies to effectively target PGCCs are limited, largely due to the lack of high-throughput methods for assessing their viability. Traditional assays lack the sensitivity needed to detect PGCC-specific elimination, prompting the development of novel approaches. To address this challenge, we developed a high-throughput single-cell morphological analysis workflow designed to differentiate compounds that selectively inhibit non-PGCCs, PGCCs, or both. Using this method, we screened a library of 2,726 FDA Phase 1-approved drugs, identifying promising anti-PGCC candidates, including proteasome inhibitors, FOXM1, CHK, and macrocyclic lactones. Notably, RNA-Seq analysis of cells treated with the macrocyclic lactone Pyronaridine revealed AXL inhibition as a potential strategy for targeting PGCCs. Although our single-cell morphological analysis pipeline is powerful, empirically testing all existing compounds is impractical and inefficient. To overcome this limitation, we trained a machine learning model to predict anti-PGCC efficacy in silico, integrating chemical fingerprints and compound descriptions from prior publications and databases. The model demonstrated a high correlation with experimental outcomes and predicted efficacious compounds in an expanded library of over 6,000 drugs. Among the top-ranked predictions, we experimentally validated two compounds as potent PGCC inhibitors. These findings underscore the synergistic potential of integrating high-throughput empirical screening with machine learning-based virtual screening to accelerate the discovery of novel therapies, particularly for targeting therapy-resistant PGCCs in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f39f65e370bc9aafe05599fe5e71c2036a3f471" target='_blank'>
              High-Throughput Empirical and Virtual Screening to Discover Novel Inhibitors of Polyploid Giant Cancer Cells in Breast Cancer
              </a>
            </td>
          <td>
            Yushu Ma, Chien-Hung Shih, Jinxiong Cheng, Hsiao-Chun Chen, Li-Ju Wang, Yanhao Tan, Yu-Chiao Chiu, Yu-Chih Chen
          </td>
          <td>2024-09-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) technology enables the precise analysis of individual cell transcripts with high sensitivity and throughput. When integrated with multiomics technologies, scRNA-seq significantly enhances the understanding of cellular diversity, particularly within the tumor microenvironment. Similarly, single-cell DNA sequencing has emerged as a powerful tool in cancer research, offering unparalleled insights into the genetic heterogeneity and evolution of tumors. In the context of breast cancer, this technology holds substantial promise for decoding the intricate genomic landscape that drives disease progression, treatment resistance, and metastasis. By unraveling the complexities of tumor biology at a granular level, single-cell DNA sequencing provides a pathway to advancing our comprehension of breast cancer and improving patient outcomes through personalized therapeutic interventions. As single-cell sequencing technology continues to evolve and integrate into clinical practice, its application is poised to revolutionize the diagnosis, prognosis, and treatment strategies for breast cancer. This review explores the potential of single-cell sequencing technology to deepen our understanding of breast cancer, highlighting key approaches, recent advancements, and the role of the tumor microenvironment in disease plasticity. Additionally, the review discusses the impact of single-cell sequencing in paving the way for the development of personalized therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad5d30fa3be7681abcc5841abc609350bfd86737" target='_blank'>
              Recent progress and applications of single-cell sequencing technology in breast cancer
              </a>
            </td>
          <td>
            Yousef M. Hawsawi, Basmah Khoja, Abdullah Omar Aljaylani, Raniah Jaha, Rasha Mohammed Alderbi, Huda Alnuman, Mohammed I. Khan
          </td>
          <td>2024-09-18</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Multiple Myeloma (MM) is a hematological malignancy associated with poor prognosis and relapses. MM has a strong inherited component, and while the mutational and transcriptomic landscapes of MM have been characterized extensively, the molecular consequences of germline genetic variation in MM have not been previously explored. Here, we have leveraged matched genotype and expression data from 607 patients to map the germline and somatic regulatory variation in MM. We find that non-coding variation shapes the transcriptional landscape of MM, contributing to overall survival. These regulatory variants alter the binding of transcription factors with known importance in MM, such as IRF4, providing insights into the underlying mechanisms. These data contribute to the increasing understanding of the genetic, molecular, and cellular forces shaping MM risk and outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5c5bba32932ad19d6d4a4784baebd131dde155a" target='_blank'>
              Germline and somatic genetic effects on gene expression and outcome in patients with Multiple Myeloma
              </a>
            </td>
          <td>
            H. Natri, Linh Bui, L. Peter, Bianca Argente, A. Federico, A. Gutierrez, Mei-i Chung, Jonathan Keats, Nicholas E Banovich
          </td>
          <td>2024-10-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="While a subset of patients with metastatic melanoma achieves durable responses to immune checkpoint blockade (ICB) therapies, the majority ultimately exhibit either innate or acquired resistance to these treatments. However, the molecular mechanisms underlying resistance to ICB therapies remain elusive and are warranted to elucidate. Here, we comprehensively investigated the tumor and tumor immune microenvironment (TIME) of paired pre- and post-treatment tumor specimens from metastatic melanoma patients who were primary or secondary resistance to anti-CTLA-4 and/or anti-PD-1/PD-L1 therapies. Differentially expressed gene (DEG) analysis and single-sample gene set enrichment analysis (ssGSEA) with transcriptomic data identified cell cycle and c-MYC signaling as pathway-based resistance signatures. And weighted gene co-expression network analysis (WGCNA) revealed the activation of a cross-resistance meta-program involving key signaling pathways related to tumor progression in ICB resistant melanoma. Moreover, spatially-resolved, image-based immune monitoring analysis by using NanoString’s digital spatial profiling (DSP) and Cyclic Immunofluorescence (CyCIF) showed infiltration of suppressive immune cells in the tumor microenvironment of melanoma with resistance to ICB therapies. Our study reveals the molecular mechanisms underlying resistance to ICB therapies in patients with metastatic melanoma by conducting such integrated analyses of multi-dimensional data, and provides rationale for salvage therapies that will potentially overcome resistance to ICB therapies. Statement of translational relevance This study paves the way for the creation of innovative therapeutic strategies, aimed at subverting resistance to immune checkpoint blockade (ICB) therapies in metastatic melanoma patients. By unraveling the specific molecular mechanisms underlying resistance, scientists can design effective alternative treatments that target pathways such as pathways associated with cell cycle dysregulation and c-MYC signaling. Furthermore, through the application of advanced immune monitoring techniques such as NanoString Digital Spatial Profiling (DSP) and Cyclic Immunofluorescence (CyCIF), this study has significantly enriched our understanding of the tumor microenvironment. This enhanced characterization facilitates the discovery of potential biomarkers that may forecast a patient’s response to ICB treatment. Ultimately, these advancements could potentially refine patient outcomes and foster the development of more personalized cancer treatments in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59c359686d6b4f4f12c03d81c082e61089dfa6d8" target='_blank'>
              A Comprehensive Proteogenomic and Spatial Analysis of Innate and Acquired Resistance of Metastatic Melanoma to Immune Checkpoint Blockade Therapies
              </a>
            </td>
          <td>
            Shiyou Wei, Kuang Du, Hongbin Lan, Zhenyu Yang, Yulan Deng, Zhi Wei, Dennie T. Frederick, Jinho Lee, Marilyne Labrie, Tian Tian, Tabea Moll, Yeqing Chen, Ryan J. Sullivan, Gordon B. Mills, Genevieve M. Boland, Keith T. Flaherty, Lunxu Liu, M. Herlyn, Gao Zhang
          </td>
          <td>2024-09-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>114</td>
        </tr>

        <tr id="Lung cancer remains one of the leading causes of cancer-related deaths globally, with its complexity driven by intricate and intertwined genetic, epigenetic, and environmental factors. Despite advances in genomics, transcriptomics, and proteomics, understanding the phenotypic diversity of lung cancer has lagged behind. Next-generation phenomics, which integrates high-throughput phenotypic data with multiomics approaches and digital technologies such as artificial intelligence (AI), offers a transformative strategy for unraveling the complexity of lung cancer. This approach leverages advanced imaging, single-cell technologies, and AI to capture dynamic phenotypic variations at cellular, tissue, and whole organism levels and in ways resolved in temporal and spatial contexts. By mapping the high-throughput and spatially and temporally resolved phenotypic profiles onto molecular alterations, next-generation phenomics provides deeper insights into the tumor microenvironment, cancer heterogeneity, and drug efficacy, safety, and resistance mechanisms. Furthermore, integrating phenotypic data with genomic and proteomic networks allows for the identification of novel biomarkers and therapeutic targets in ways informed by biological structure and function, fostering precision medicine in lung cancer treatment. This expert review examines and places into context the current advances in next-generation phenomics and its potential to redefine lung cancer diagnosis, prognosis, and therapy. It highlights the emerging role of AI and machine learning in analyzing complex phenotypic datasets, enabling personalized therapeutic interventions. Ultimately, next-generation phenomics holds the promise of bridging the gap between molecular alterations and clinical and population health outcomes, providing a holistic understanding of lung cancer biology that could revolutionize its management and improve patient survival rates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9eff2978937e2ef89767871d93305b0ec9138e48" target='_blank'>
              Next-Generation Cancer Phenomics: A Transformative Approach to Unraveling Lung Cancer Complexity and Advancing Precision Medicine.
              </a>
            </td>
          <td>
            Sanjukta Dasgupta
          </td>
          <td>2024-10-22</td>
          <td>Omics : a journal of integrative biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Childhood neuroblastoma with MYCN-amplification is classified as high-risk and often relapses after intensive treatments. Immune checkpoint blockade therapy against the PD-1/L1 axis shows limited efficacy in neuroblastoma patients and the cancer intrinsic immune regulatory network is poorly understood. Here, we leverage genome-wide CRISPR/Cas9 screens and identify H2AFY as a resistance gene to the clinically approved PD-1 blocking antibody, nivolumab. Analysis of single-cell RNA sequencing datasets reveals that H2AFY mRNA is enriched in adrenergic cancer cells and is associated with worse patient survival. Genetic deletion of H2afy in MYCN-driven neuroblastoma cells reverts in vivo resistance to PD-1 blockade by eliciting activation of the adaptive and innate immunity. Mapping of the epigenetic and translational landscape demonstrates that H2afy deletion promotes cell transition to a mesenchymal-like state. With a multi-omics approach, we uncover H2AFY-associated genes that are functionally relevant and prognostic in patients. Altogether, our study elucidates the role of H2AFY as an epigenetic gatekeeper for cell states and immunogenicity in high-risk neuroblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dae04d78ec0c131d4f8ff3da9b40c7edb7f93956" target='_blank'>
              Epigenetic regulation of cell state by H2AFY governs immunogenicity in high-risk neuroblastoma.
              </a>
            </td>
          <td>
            Divya Nagarajan, Rebeca T Parracho, David Corujo, Minglu Xie, G. Kutkaite, T. Olsen, Marta Rúbies Bedós, Maede Salehi, Ninib Baryawno, M. Menden, Xingqi Chen, M. Buschbeck, Y. Mao
          </td>
          <td>2024-09-10</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Effective treatment of pediatric acute leukemia is dependent on accurate genomic classification, typically derived from a combination of multiple time-consuming and costly techniques such as flow cytometry, fluorescence in situ hybridization (FISH), karyotype analysis, targeted PCR, and microarrays (Arber et al., 2016; Iacobucci & Mullighan, 2017; Narayanan & Weinberg, 2020). We investigated the feasibility of a comprehensive single-assay classification approach using long-read sequencing, with real-time genome target enrichment, to classify chromosomal abnormalities and structural variants characteristic of acute leukemia. We performed whole genome sequencing on DNA from diagnostic peripheral blood or bone marrow for 54 pediatric acute leukemia cases with diverse genomic subtypes. We demonstrated the characterization of known, clinically relevant karyotype abnormalities and structural variants concordant with standard-of-care clinical testing. Subtype-defining genomic alterations were identified in all cases following a maximum of forty-eight hours of sequencing. In 18 cases, we performed real-time analysis – concurrent with sequencing – and identified the driving alteration in as little as fifteen minutes (for karyotype) or up to six hours (for complex structural variants). Whole genome nanopore sequencing with adaptive sampling has the potential to provide detailed genomic classification of acute leukemia specimens with reduced cost and turnaround time compared to the current standard of care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa083d181be0ae2fc8596d1c7ad6534ea54258c0" target='_blank'>
              Real-time genomic characterization of pediatric acute leukemia using adaptive sampling
              </a>
            </td>
          <td>
            Julie Geyer, K. Opoku, John Lin, Lori Ramkissoon, C. Mullighan, N. Bhakta, Thomas B. Alexander, Jeremy R. Wang
          </td>
          <td>2024-10-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>103</td>
        </tr>

        <tr id="Precision gene editing enables massively parallel identification of cancer-promoting genes Since its discovery in 2012 as a gene-editing tool, CRISPR-Cas9 has been touted as a turning point for drug discovery. Indeed, CRISPR-based technologies have an unprecedented power to interrogate the mechanistic underpinnings of disease by systematically testing the cellular effects of targeted genetic alterations (1). Disease hallmarks specific to cancer, such as unchecked cell proliferation and drug resistance, readily lend themselves to massively parallel CRISPR screens. In these screens, individual cells harboring a guide RNA that also acts as a barcode for a DNA edit of interest may acquire a competitive or diminished proliferative advantage relative to other cells in the same test pool. This allows the perturbation of thousands of genes in a single experiment. The resulting genetic maps generated in cancer cell lines (CCLs) and in cancer-derived tissues now support exceptional speed and economy in the rationalization of new drug development and may shorten bench-to-bedside timelines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be2013426359d6875b2659e5a9c78a646561186a" target='_blank'>
              Impact of CRISPR in cancer drug discovery
              </a>
            </td>
          <td>
            Jason Moffat, Alexis C. Komor, Lawrence Lum
          </td>
          <td>2024-10-25</td>
          <td>Science</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="
 Prostate cancer (PCa) is the second-leading cause of cancer mortality in U.S. men, with African American (AA) men showing higher incidence and mortality compared to European American men. In addition, AA men are more likely to be diagnosed with more aggressive PCa resulting in reduction of treatment options, especially as tumors develop therapy resistance. Thus, understanding the mechanisms underlying PCa therapy resistance is crucial for developing new therapeutic options. The upregulation of the Lens Epithelium Derived Growth Factor p75 (LEDGF/p75) by glucocorticoid receptor (GR) is a novel mechanism of PCa therapy resistance recently identified by our group. GR upregulation in PCa bypasses the blockade of androgen receptor by inhibitors such as enzalutamide and abiraterone. LEDGF/p75 is an oncogenic transcription co-activator that promotes DNA damage repair and cancer cell survival against environmental stressors including chemotherapeutic drugs such as docetaxel (DTX). An emerging concern in PCa treatment is therapy cross-resistance, in which the molecular mechanisms of resistance to a particular drug confer resistance to another drug. Our recent studies showed that GR upregulates and interacts with LEDGF/p75 forming a transcriptional complex that includes other oncoproteins such as Menin, JPO2, c-Myc and HRP2, and promotes DTX resistance (DTX-R) in PCa cells. In addition, we reported that both GR and LEDGF/p75 are upregulated in enzalutamide resistant (ENZ-R) PCa cells. We hypothesized that the GR- LEDGF/p75 transcriptional network plays a role in PCa therapy cross-resistance to both ENZ and DTX and is a promising therapeutic target. Cell viability assays revealed that ENZ-R in PCa cells confers cross-resistance to DTX. Western blot analysis revealed upregulation of members of the GR-LEDGF/p75 network such as GR, LEDGF/p75, Menin and c-Myc in ENZ-R PCa cells, consistent with similar observations in DTX-R PCa cells. Upon knockdown of GR and LEDGF/p75 in DTX-R PCa cells, we identified by RNA-seq analysis overlapping DEGs associated with both ENZ-R and DTX-R, suggesting that the GR-LEDGF/p75 transcriptional network may contribute to therapy cross-resistance. Consistent with this, mining of publicly available RNA-seq databases identified 46 overlapping differentially expressed genes (DEGs) between ENZ-R and DTX-R PCa cells. Pathway analysis of DEGs regulated by both GR and LEDGF/p75 revealed significant enrichment of Apoptosis, p53, and DNA repair pathways, which are associated with therapy resistance. Ongoing studies are evaluating the anti-cancer properties and synergistic/additive potential of various inhibitors of the GR-LEDGF/p75 transcriptional network in cellular models of ENZ-R and DTX-R. These studies provide novel mechanistic insights into PCa therapy cross-resistance and may lead to the development of novel therapeutic strategies to reduce PCa mortality disparities.
 Citation Format: Pedro T. Ochoa, Evelyn S. Sanchez-Hernandez, Kai Wen Cheng, Isaac Kremsky, Charles Wang, Carlos A. Casiano. The GR-LEDGF/p75 transcriptional network as a promising target to reduce therapy cross-resistance in prostate cancer [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr C096.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d0c6225006bf01cdfdfbdc7e8efd4e751393fc1" target='_blank'>
              Abstract C096: The GR-LEDGF/p75 transcriptional network as a promising target to reduce therapy cross-resistance in prostate cancer
              </a>
            </td>
          <td>
            P. Ochoa, Evelyn S. Sanchez-Hernandez, Kai Wen Cheng, Isaac Kremsky, Charles Wang, C. Casiano
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Long INterspersed Element-1 (LINE-1; L1) and Alu are two families of transposable elements (TEs) occupying ∼17% and ∼11% of the human genome, respectively. Though only a small fraction of L1 copies is able to produce the machinery to mobilize autonomously, Alu elements and degenerate L1 copies can hijack their functional machinery and mobilize in trans. The expression and subsequent copy number expansion of L1 and Alu can exert pathological effects on their hosts, promoting genome instability, inflammation, and cell cycle alterations. These features have made L1 and Alu promising focus subjects in studies of aging and aging diseases where they can become active. However, the mechanisms regulating variation in their expression and copy number remain incompletely characterized. Moreover, the relevance of known mechanisms to diverse human populations remains unclear, as mechanisms are often characterized in isogenic cell culture models. To address these gaps, we leveraged genomic data from the 1000 Genomes Project to carry out a trans-ethnic GWAS of L1 and Alu insertion global singletons. These singletons are rare insertions observed only once in a population, potentially reflecting recently acquired L1 and Alu integrants or structural variants, and which we used as proxies for L1/Alu-associated copy number variation. Our computational approach identified single nucleotide variants in genomic regions containing genes with potential and known TE regulatory properties, and it enriched for single nucleotide variants in regions containing known regulators of L1 expression. Moreover, we identified many reference TE copies and polymorphic structural variants that were associated with L1/Alu singletons, suggesting their potential contribution to TE copy number variation through transposition-dependent or transposition-independent mechanisms. Finally, a transcriptional analysis of lymphoblastoid cells highlighted potential cell cycle alterations in a subset of samples harboring L1/Alu singletons. Collectively, our results (i) suggest that known TE regulatory mechanisms may also play regulatory roles in diverse human populations, (ii) expand the list of genic and repetitive genomic loci implicated in TE copy number variation, and (iii) reinforce the links between TEs and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c547e837e626947d2c017f057b521d189f2c4fd3" target='_blank'>
              Multi-ancestry GWAS reveals loci linked to human variation in LINE-1- and Alu-copy numbers
              </a>
            </td>
          <td>
            Juan I. Bravo, Lucia Zhang, BA Benayoun
          </td>
          <td>2024-09-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Epimutations modify gene expression and lead to phenotypic variation while the encoding DNA sequence remains unchanged. Epimutations mediated by RNA interference (RNAi) and/or chromatin modifications can confer antifungal drug resistance and may impact virulence traits in fungi (1–5). However, whether these epigenetic modifications can be transmitted across generations following sexual reproduction was unclear. This study demonstrates that RNAi epimutations conferring antifungal drug resistance are transgenerationally inherited in the human fungal pathogen Mucor circinelloides. Our research revealed that RNAi-based antifungal resistance follows a DNA sequence-independent, non-Mendelian inheritance pattern. Small RNAs (sRNAs) are the exclusive determinants of inheritance, transmitting drug resistance independently of other known repressive epigenetic modifications. Unique sRNA signature patterns can be traced through inheritance from parent to progeny, further supporting RNA as an alternative molecule for transmitting information across generations. This study marks a significant advance in understanding epigenetic inheritance, highlighting RNAi-exclusive epimutations as a widespread phenomenon in the pathogenic M. circinelloides species complex. Understanding how epimutations occur, propagate, and confer resistance may enable their detection in other eukaryotic pathogens, provide solutions for challenges posed by rising antimicrobial drug resistance (AMR), and also advance research on phenotypic adaptability and its evolutionary implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df7aab0fd6f057abc3a0de9ddf4f9920c418d202" target='_blank'>
              RNAi epimutations conferring antifungal drug resistance are inheritable
              </a>
            </td>
          <td>
            Carlos Pérez-Arques, M. Navarro-Mendoza, Ziyan Xu, Grit Walther, Joseph Heitman
          </td>
          <td>2024-10-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Next generation sequencing of neuroblastoma (NB) tumors have revealed frequent somatic and germline genetic alterations in genes encoding proteins involved in DNA damage response (DDR) pathways. Despite being well-studied in many adult cancers, roles for DDR disruption in pediatric solid tumors remains poorly understood. To address this, patient-relevant loss-of-function mutations in DDR pathway components including Brca2, Atm, and Palb2 were incorporated into an established zebrafish MYCN transgenic model (Tg(dbh:EGFP-MYCN)). These mutations were found to enhance NB formation and metastasis in vivo, and result in upregulation of proliferation, cell cycle checkpoint and DNA damage repair transcriptional signatures, revealing potential molecular vulnerabilities in DDR-deficient NB. Zebrafish DDR-deficient NB and human NB cells with DDR protein knock-down were sensitive to the poly(ADP-ribose)-polymerase (PARP) inhibitor olaparib, and this effect was further enhanced by inhibition of the ataxia telangiectasia and rad3 related (ATR) kinase. Altogether, our data supports a functional role for DDR-deficiency in NB in vivo and therapeutic potential for combination PARP + ATR inhibition in NB patients with alterations in DDR genes. Significance This work provides the first in vivo evidence supporting a functional role for DDR-deficiency in NB by demonstrating that alterations in certain DDR pathway genes promote NB formation and metastasis. NGS and pre-clinical drug testing also provides rationale for PARP + ATR inhibitor therapy combinations for patients with NB and pathogenic DDR pathway alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bdec5d0ffc7f47bcd4a92b02bbe2b4a531d6b2a" target='_blank'>
              DNA Damage Response Deficiency Enhances Neuroblastoma Progression and Sensitivity to Combination PARP and ATR Inhibition
              </a>
            </td>
          <td>
            Madeline N Hayes, S. Cohen-Gogo, L. Kee, Alex Weiss, M. Layeghifard, Yagnesh Ladumor, Ivette Valencia-Sama, Anisha Rajaselvam, David Kaplan, A. Villani, A. Shlien, Daniel A. Morgenstern, Meredith S Irwin
          </td>
          <td>2024-09-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC), a leading cause of cancer mortality in the United States, presents significant treatment challenges due to its late diagnosis and poor prognosis. Despite advances, the five-year survival rates remain dismally low, with only a fraction of patients eligible for potentially curative surgical interventions. This review aims to comprehensively examine the current landscape of targeted therapies in PDAC, focusing on recent developments in precision medicine approaches. We explore various molecular targets, including KRAS mutations, DNA damage repair deficiencies, mismatch repair pathway alterations, and rare genetic fusions. The review discusses emerging therapies, such as PARP inhibitors, immune checkpoint inhibitors, and novel targeted agents, like RET and NTRK inhibitors. We analyze the results of key clinical trials and highlight the potential of these targeted approaches in specific patient subgroups. Recent developments in PDAC research have emphasized precision oncology, facilitated by next-generation sequencing and the identification of genetic and epigenetic alterations. This approach tailors treatments to individual genetic profiles, improving outcomes and reducing side effects. Significant strides have been made in classifying PDAC into various subtypes, enhancing therapeutic precision. The identification of specific mutations in genes like KRAS, along with advancements in targeted therapies, including small molecule inhibitors, offers new hope. Furthermore, emerging therapies targeting DNA repair pathways and immunotherapeutic strategies also show promising results. As research evolves, integrating these targeted therapies with conventional treatments might improve survival rates and quality of life for PDAC patients, underscoring the shift towards a more personalized treatment paradigm.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0623bca1b0876fdbb0324db3ab4221bfe60d5eb8" target='_blank'>
              Targeted Therapies in Pancreatic Cancer: A New Era of Precision Medicine
              </a>
            </td>
          <td>
            Bingyu Li, Qiong Zhang, Claire Castaneda, Shelly Cook
          </td>
          <td>2024-09-25</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff04d632e2b6ac550f2911954ea295c050c33ca2" target='_blank'>
              Development of a polygenic score predicting drug resistance and patient outcome in breast cancer
              </a>
            </td>
          <td>
            Divya Sahu, Jeffrey Shi, Isaac Andres Segura Rueda, Ajay Chatrath, Anindya Dutta
          </td>
          <td>2024-10-02</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Myelodysplastic syndromes (MDS) are myeloid malignancies with heterogeneous genotypes and phenotypes, characterized by ineffective haematopoiesis and a high risk of progression towards acute myeloid leukaemia (AML). Prognosis for patients treated with hypomethylating agents (HMAs), as is azacytidine, the main drug used as frontline therapy for MDS is mostly based on cytogenetics and next generation sequencing (NGS) of the initial myeloid clone. Although the critical influence of the epigenetic landscape upon cancer cells survival and development as well on tumour environment establishment is currently recognized and approached within current clinical practice in MDS, the heterogenous response of the patients to epigenetic therapy is suggesting a more complex mechanism of action, as is the case of RNA methylation. In this sense, the newly emerging field of epitranscriptomics could provide a more comprehensive perspective upon the modulation of gene expression in malignancies, as is the proof‐of‐concept of MDS. We initially did RNA methylation sequencing on MDS patients (n = 6) treated with azacytidine and compared responders with non‐responders. Afterwards, the genes identified were assessed in vitro and afterwards validated on a larger cohort of MDS patients treated with azacytidine (n = 58). Our data show that a more accurate prognosis could be based on analysing the methylome and thus we used methylation sequencing to differentially split high‐grade MDS patients with identical demographical and cytogenetic features, between azacytidine responders and non‐responders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/302c378a460869cbc4ab47244a1bece112c425c0" target='_blank'>
              RNA methylation sequencing shows different gene expression signatures for response to azacytidine therapy in high‐grade myelodysplastic syndromes
              </a>
            </td>
          <td>
            Diana Gulei, Vlad Moisoiu, David Kegyes, Rareș Drula, Sabina Iluta, A. Țigu, Mădălina Nistor, Ciprian Jitaru, Anamaria Bancos, Petra Rotariu, Corina Popovici, D. Dima, R. Tomai, Ioana Rus, Cătălin Constantinescu, R. Munteanu, Diana Cenariu, U. Sezerman, M. Zdrenghea, Jaroslav Čermák, Hermann Einsele, Gabriel Ghiaur, C. Tomuleasa
          </td>
          <td>2024-09-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Understanding the impact of genetic alterations on epigenomic phenotypes during breast cancer progression is challenging with unimodal measurements. Here, we report wellDA-seq, the first high-genomic resolution, high-throughput method that can simultaneously measure the whole genome and chromatin accessibility profiles of thousands of single cells. Using wellDA-seq, we profiled 22,123 single cells from 2 normal and 9 tumors breast tissues. By directly mapping the epigenomic phenotypes to genetic lineages across cancer subclones, we found evidence of both genetic hardwiring and epigenetic plasticity. In 6 estrogen-receptor positive breast cancers, we directly identified the ancestral cancer cells, and found that their epithelial cell-of-origin was Luminal Hormone Responsive cells. We also identified cell types with copy number aberrations (CNA) in normal breast tissues and discovered non-epithelial cell types in the microenvironment with CNAs in breast cancers. These data provide insights into the complex relationship between genetic alterations and epigenomic phenotypes during breast tumor evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d380546a2c5346cf2ff8db012770db28bb0473a0" target='_blank'>
              Single cell genome and epigenome co-profiling reveals hardwiring and plasticity in breast cancer
              </a>
            </td>
          <td>
            Kaile Wang, Yun Yan, Heba Elgamal, Jianzhuo Li, Chenling Tang, S. Bai, Zhenna Xiao, Emi Sei, Yiyun Lin, Junke Wang, Jessica Montalvan, Changandeep Nagi, Alastair M. Thompson, Nicholas E. Navin
          </td>
          <td>2024-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24869ef91b9d9f698a0aa1c1c25527b7385dd280" target='_blank'>
              Deciphering resistance mechanisms in cancer: final report of MATCH-R study with a focus on molecular drivers and PDX development
              </a>
            </td>
          <td>
            Damien Vasseur, L. Bigot, Kristi Beshiri, Juan-David Florez-Arango, F. Facchinetti, A. Hollebecque, L. Tselikas, M. Aldea, F. Blanc-Durand, A. Gazzah, D. Planchard, Ludovic Lacroix, N. Pata-Merci, Catline Nobre, Alice Da Silva, C. Nicotra, M. Ngo-Camus, F. Braye, Sergey Nikolaev, S. Michiels, Gerome Jules-Clement, K. Olaussen, Fabrice André, J. Scoazec, F. Barlesi, Santiago Ponce, Jean-Charles Soria, Benjamin Besse, Yohann Loriot, L. Friboulet
          </td>
          <td>2024-10-04</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8719f1e85495bf521f29d9969611ef5b8435eeb3" target='_blank'>
              The evolutionary theory of cancer: challenges and potential solutions.
              </a>
            </td>
          <td>
            L. Laplane, Carlo C. Maley
          </td>
          <td>2024-09-10</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Approximately 50% of patients diagnosed with ovarian cancer harbor tumors with mutations in BRCA1, BRCA2, or other genes involved in homologous recombination repair (HR). The presence of homologous recombination deficiency (HRD) is an approved biomarker for poly-ADP-ribose polymerase inhibitors (PARPis) as a maintenance treatment following a positive response to initial platinum-based chemotherapy. Despite this treatment option, the development of resistance to PARPis is common among recurrent disease patients, leading to a poor prognosis. In this study, we conducted a comprehensive analysis using publicly available datasets to elucidate the molecular mechanisms driving PARPi resistance in BRCA1-deficient ovarian cancer. Our findings reveal a central role for the interferon (IFN) pathway in mediating resistance in the context of BRCA1 deficiency. Through integrative bioinformatics approaches, we identified LY6E, an interferon-stimulated gene, as a key mediator of PARPi resistance, with its expression linked to an immunosuppressive tumor microenvironment (TME) encouraging tumor progression and invasion. LY6E amplification correlates with poor prognosis and increased expression of immune-related gene signatures, which is predictive of immunotherapy response. Interestingly, LY6E expression upon PARPi treatment resistance was found to be dependent on BRCA1 status. Gene expression analysis in the Orien/cBioPortal database revealed an association between LY6E and genes involved in DNA repair, such as Rad21 and PUF60, emphasizing the interplay between DNA repair pathways and immune modulation. Moreover, PUF60, Rad21, and LY6E are located on chromosome 8q24, a locus often amplified and associated with the progression of ovarian cancer. Overall, our study provides novel insights into the molecular determinants of PARPi resistance and highlights LY6E as a promising prognostic biomarker in the management of HRD ovarian cancer. Future studies are needed to fully elucidate the molecular mechanisms underlying the role of LY6E in PARPi resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4052305f8854485a3b85613abf68315aff09bfc" target='_blank'>
              Exploring Molecular Drivers of PARPi Resistance in BRCA1-Deficient Ovarian Cancer: The Role of LY6E and Immunomodulation
              </a>
            </td>
          <td>
            T. Petta, Joseph Carlson
          </td>
          <td>2024-09-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Background: In neuroblastoma (NB), intratumor heterogeneity (ITH) is frequently observed, but the role of cell-to-cell allele-specific copy number alterations in phenotypic variation, clonal evolution and treatment response remains to be determined. Here we investigate ITH, timing of specific genomic aberrations, single-cell replication timing and the co-evolution of the genome and transcriptome in NB tumors at single-cell resolution, with an aim to analyse subclonal dynamics and clone-specific response or resistance under targeted therapeutic pressure. Methods: In addition to germline/tumor bulk whole exome sequencing (WES), ultra-low depth (0.25x) single-cell whole-genome DNA sequencing (scDNAseq) was performed using 10x genomics Chromium single-cell CNV (scCNV) kit and 9410 tumor cells were characterized from 14 patient-derived xenografts (PDX) NB-models and 4 tumor biopsies from NB-patients, either at diagnosis (n=7), progression (n=3) or relapse (n=8). Single-cell RNA sequencing (scRNAseq) data was obtained from the same PDX and patient tumor samples (Thirant et al, 2023). 6/14 PDX models were subjected to different treatment combinations (targeted treatment with/without chemotherapy) and bulk WES was performed at two time-points, pre- and post-treatment. Results: Both monoclonal (n=7) and polyclonal (n=11) genomes were determined by allele and haplotype specific copy number (CN) alteration using both scDNAseq and scRNAseq data analysis, with 2 to 11 clones observed per polyclonal NB tumor. Whole genome duplication events (n=7) were observed in both polyclonal and monoclonal genomes. Known driver CN (segmental loss in chr1p and chr11q and gain at chr17q, or MYCN/ALK amplification) or somatic mutations (ALK/ATRX/TP53/NF1) were early clonal events.Study of replication timing (RT) based on scDNAseq revealed significant differences in RT between the MYCN amplified (n=7) and non-amplified groups (n=5), with NB tumors without MYCN amplification (no MNA) characterized by a predominance of late replicating domains, in contrast to MYCN amplified (MNA) tumors, which are enriched in early replicating domains In a PDX model of interest, scDNAseq analysis showed parallel copy number evolution of two distinct clones, subclone s1/s2. Data integration of clonal mutational profiles with pre- and post-targeted therapy (Lorlatinib) revealed clone-specific treatment response. Subclone s2 was partially responding with extinction of a sub-set of somatic alterations, whereas no change was observed in subclone s1. The replication timing (RT) profile of these two clones, subclone s1 (early-RT) and s2 (late-RT) were mutually exclusive. Genotype to phenotype analysis revealed subclone s1 genotype was preferentially expressed at transcriptomic level. Conclusion: Together, these results determine the evolutionary trajectories of NB tumors, linked to distinct replication timing and highlight opportunities for targetable early clonal alteration detection.
 Citation Format: Jaydutt Bhalshankar, Angela Bellini, Irene Jimenez, Cécile Thirant, Elnaz Saberi-Ansari, Yasmine Iddir, Alexandra Saint Charles, Charlotte Butterworth, Amira Kramdi, Virginie Raynal, Sylvain Baulande, Didier Surdez, Sakina Zaidi, Gaelle Pierron, Angel Montero Carcaboso, Birgit Geoerger, Andrei Zinovyev, Olivier Delattre, Isabelle Janoueix-lerosey, Gudrun Schleiermacher. Clonal decomposition and DNA replication states defined by scaled single-cell DNA and RNA sequencing suggest clone-specific therapeutic vulnerabilities in neuroblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B074.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02f5489383eed60c9973409b18fbcca1b1288062" target='_blank'>
              Abstract B074: Clonal decomposition and DNA replication states defined by scaled single-cell DNA and RNA sequencing suggest clone-specific therapeutic vulnerabilities in neuroblastoma
              </a>
            </td>
          <td>
            Jaydutt Bhalshankar, Angela Bellini, Irene Jimenez, Cécile Thirant, Elnaz Saberi-Ansari, Yasmine Iddir, Alexandra Saint Charles, Charlotte Butterworth, Amira Kramdi, Virginie Raynal, Sylvain Baulande, D. Surdez, Sakina Zaidi, G. Pierron, A. Carcaboso, Birgit Geoerger, Andrei Zinovyev, Olivier Delattre, I. Janoueix-Lerosey, G. Schleiermacher
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="BACKGROUND
The fate and functions of RNAs are coordinately regulated by RNA-binding proteins (RBPs), which are often dysregulated in various cancers. Known as a splicing regulator, RNA-binding motif protein 6 (RBM6) harbors tumor-suppressor activity in many cancers; however, there is a lack of research on the molecular targets and regulatory mechanisms of RBM6.


METHODS
In this study, we constructed an RBM6 knock-down (shRBM6) model in the HeLa cell line to investigate its functions and molecular targets. Then we applied improved RNA immunoprecipitation coupled with sequencing (iRIP-seq) and whole transcriptome sequencing approaches to investigate the potential role and RNA targets of RBM6.


RESULTS
Using The Cancer Genome Atlas dataset, we found that higher expression of RBM6 is associated with a better prognosis in many cancer types. In addition, we found that RBM6 knockdown promoted cell proliferation and inhibited apoptosis, demonstrating that RBM6 may act as an anti-oncogenic protein in cancer cells. RBM6 can regulate the alternative splicing (AS) of genes involved in DNA damage response, proliferation, and apoptosis-associated pathways. Meanwhile, RBM6 knockdown activated type I interferon signaling pathways and inhibited the expression of genes involved in the cell cycle, cellular responses to DNA damage, and DNA repair pathways. The differentially expressed genes (DEGs) by shRBM6 and their involved pathways were likely regulated by the transcription factors undergoing aberrant AS by RBM6 knockdown. For iRIP-seq analysis, we found that RBM6 could interact with a large number of mRNAs, with a tendency for binding motifs GGCGAUG and CUCU. RBM6 bound to the mRNA of cell proliferation- and apoptosis-associated genes with dysregulated AS after RBM6 knockdown.


CONCLUSIONS
In summary, our study highlights the important role of RBM6, as well as the downstream targets and regulated pathways, suggesting the potential regulatory mechanisms of RBM6 in the development of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f429e0c09165b4883518646756d39032c7ca6d4" target='_blank'>
              Global RNA Interaction and Transcriptome Profiles Demonstrate the Potential Anti-Oncogenic Targets and Pathways of RBM6 in HeLa Cells.
              </a>
            </td>
          <td>
            Ping Peng, Qingqing Yin, Wei Sun, Jing Han, Hao Guo, Chao Cheng, Dongbo Liu
          </td>
          <td>2024-09-23</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Sequential changes in epigenomically specified transcriptional cell states are essential for normal developmental progression. These normal processes are co-opted in cancers, allowing genetically identical cells in different states to cooperatively form heterogeneous tumors. Specific cell states may display distinct malignant properties, including variability in sensitivity to conventional therapies, resulting in heterogeneous responses to therapy and a potential mechanism of patient relapse. As a result, deriving mechanistic approaches to control cell state transitions for therapeutic benefit has been limited by the lack of tractable live-cell tools able to reflect complex and multifaceted cell states. To address this problem, here we demonstrate the design, conception, synthesis and interrogation of a new method termed “Transcriptional Reporter Elements of Cell State” (TRECS). TRECS integrates epigenomics and transcriptomics to identify endogenous genomic elements that can label and separate cell states within heterogenous populations. Using integrative three-dimentional chromatin conformation data and CRISPR deletion studies, we link the mechanism of TRECS reporter activity to control by master transcriptional regulators of distinct cell states. Implementing this system in the high-risk pediatric solid tumor neuroblastoma, we quantitate intratumoral transcriptional and epigenetic heterogeneity and observe real-time cell state plasticity. We demonstrate that neuroblastoma cell line models generally contain plastic cell populations, including one cell state with intrinsic broad chemoresistance. Capitalizing on this intrinsic cell state plasticity, we perform a high-throughput imaging-based small molecule screen to identify chemical controllers of cell state. We identify the coactivator proteins EP300/CBP as primary regulators of the neuroblastoma chemoresistant cell state. Transient disruption of EP300/CBP activity induces sustained epigenetic and transcriptional reprogramming, which fundamentally alters the chemoresistant neuroblastoma cell state, resulting in enhanced chemosensitivity. These findings demonstrate a new, scalable approach to visualize and dissect high-risk cell states with distinct malignant properties in heterogenous tumors. This platform also guides chemical-genetic strategies to control especially challenging cell states, for example, by targeting EP300/CBP maintenance of chemoresistance.
 Citation Format: Noha AM Shendy, Yang Zhang, Ha-Won Lee, Stephanie Nance, Yousef Khashana, Mohammad AM Nezhady, Elaine Ritter, Shivendra Singh, Qi Liu, Yiping Fan, Jun Yang, Anand G. Patel, Jun Qi, Taosheng Chen, Brian J. Abraham, Adam D. Durbin. A genome-derived cell state reporter permits dissection and control of intratumoral heterogeneity and chemoresistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A054.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a3089b52a90788a8d540ab999521babb47d6723" target='_blank'>
              Abstract A054 A genome-derived cell state reporter permits dissection and control of intratumoral heterogeneity and chemoresistance
              </a>
            </td>
          <td>
            N. Shendy, Yang Zhang, Ha-Won Lee, Stephanie Nance, Yousef Khashana, Mohammad A.M. Nezhady, Elaine Ritter, Shivendra Singh, Qi Liu, Yiping Fan, Jun Yang, Anand G. Patel, Jun Qi, Taosheng Chen, Brian J. Abraham, A. D. Durbin
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background/Objectives: The rat osteosarcoma cell line UMR-106 is widely used for the study of bone cancer biology but it has not been well characterized with modern genomic methods. Methods: To better understand the biology of UMR-106 cells we used a combination of optical genome mapping (OGM), long-read sequencing nanopore sequencing and RNA sequencing.The UMR-106 genome was compared to a strain-matched Sprague-Dawley rat for variants associated with human osteosarcoma while expression data were contrasted with a public osteoblast dataset. Results: Using the COSMIC database to identify the most affected genes in human osteosarcomas we found somatic mutations in Tp53 and H3f3a. OGM identified a relatively small number of differences between the cell line and a strain-matched control animal but did detect a ~45 Mb block of amplification that included Myc on chromosome 7 which was confirmed by long-read sequencing. The amplified region showed several blocks of non-contiguous rearranged sequence implying complex rearrangements during their formation and included 14 genes reported as biomarkers in human osteosarcoma, many of which also showed increased transcription. A comparison of 5mC methylation from the nanopore reads of tumor and control samples identified genes with distinct differences including the OS marker Cdkn2a. Conclusions: This dataset illustrates the value of long DNA methods for the characterization of cell lines and how inter-species analysis can inform us about the genetic nature underlying mutations that underpin specific tumor types. The data should be a valuable resource for investigators studying osteosarcoma, in general, and specifically the UMR-106 model.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc0957ae8779454bb6e3009d9a818af23c55e01e" target='_blank'>
              Characterization of the Rat Osteosarcoma Cell Line UMR-106 by Long-Read Technologies Identifies a Large Block of Amplified Genes Associated with Human Disease
              </a>
            </td>
          <td>
            Alan F. Scott, David W. Mohr, William A. Littrell, Reshma Babu, Michelle Kokosinski, V. Stinnett, Janvi Madhiwala, John Anderson, Ying S. Zou, Kathleen L. Gabrielson
          </td>
          <td>2024-09-26</td>
          <td>Genes</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Pancreatic cancer is an aggressive disease with limited therapeutic options. Gemcitabine is a clinically used chemotherapy for pancreatic cancer treatment but has limited efficacy. The molecular mechanisms responsible for gemcitabine resistance in pancreatic cancer are still unclear. Here, we analyzed the genomics of drug sensitivity in cancer dataset and identified various pancreatic cancer cells with distinct sensitivity to gemcitabine. Validation with proliferation and clonogenic assays confirmed that while CFPAC1 and PANC0327 are highly gemcitabine-sensitive, ASPC1 and PANC1 display intrinsic gemcitabine resistance. Analysis of the gene expression profile of these cells identified several genes that are differentially upregulated in resistant versus sensitive cells. Pathway enrichment analysis using the differential genes revealed the upregulation of Hippo, Wnt, MAPK signaling pathways as well as autophagy and glutathione metabolism in the resistant cells. Using quantitative PCR, immunoblotting and biochemical assays, and extracellular flux analysis methods, we have characterized the molecular distinctions between gemcitabine sensitive and resistant cells. Our results provide insight on the molecular mechanisms driving gemcitabine resistance in pancreatic cancer and could lead to novel therapeutic strategies to overcome resistance and improve patients’ treatment.
 Citation Format: Amina D Bilalbegovic, Bowen Fu, Zeribe Nwosu. Identification of novel drivers of gemcitabine resistance in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B089.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b52052963f11a91114395dd33df22ba52dd0b89f" target='_blank'>
              Abstract B089: Identification of novel drivers of gemcitabine resistance in pancreatic cancer
              </a>
            </td>
          <td>
            Amina D Bilalbegovic, Bowen Fu, Z. Nwosu
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d317e0704a11a531980e863a514ed7e3ab276870" target='_blank'>
              Advances in pathogenesis research and challenges in treatment development for acute myeloid leukemia.
              </a>
            </td>
          <td>
            Hiroki Yamaguchi
          </td>
          <td>2024-09-03</td>
          <td>International journal of hematology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Understanding the role of repetitive elements (REs) in cancer development is crucial for identifying novel biomarkers and therapeutic targets. This study investigated the locus-specific dysregulation of REs, including differential expression and methylation of REs, across 12 TCGA cancer types stratified by their genomic context (i.e., genic and intergenic REs). We found uniquely dysregulated genic REs co-regulated with their corresponding transcripts and associated with distinct biological functions in different cancer types. Uniquely dysregulated intergenic REs were identified in each cancer type and used to cluster different sample types. Recurrently dysregulated REs were identified in several cancer types, with genes associated with up-regulated genic REs involved in cell cycle processes and those associated with down-regulated REs involved in the extracellular matrix. Interestingly, 4 out of 5 REs consistently down-regulated in all 12 cancer types were located in the intronic region of the TMEM252, a recently discovered tumor suppressor gene. TMEM252 expression was also down-regulated in 10 of 12 cancer types, suggesting its potential importance across a wide range of cancer types. With the corresponding DNA methylation array data, we found a higher prevalence of hypo-methylated REs in most cancer types (10 out of 12). Despite the slight overlaps between differentially expressed REs and differentially methylated REs, we showed that methylation of locus-specific REs negatively correlates with their expression in some cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58fdef1beb592eec1a6ab24f91cfe621cf7209ac" target='_blank'>
              Dysregulation of locus-specific repetitive elements in TCGA pan-cancers
              </a>
            </td>
          <td>
            Chao Wang, Chun Liang
          </td>
          <td>2024-10-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56281f9e2c04f45376f07cc22699136fff5305aa" target='_blank'>
              Exploring the interaction between immune cells in the prostate cancer microenvironment combining weighted correlation gene network analysis and single-cell sequencing: An integrated bioinformatics analysis
              </a>
            </td>
          <td>
            Danial Hashemi Karoii, Sobhan Bavandi, Melika Djamali, Ali Shakeri Abroudi
          </td>
          <td>2024-09-30</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) is a treatment-refractory malignancy with a dismal prognosis. Targeting cancer cell plasticity—the capacity to adapt to cell-extrinsic pressure via non-genetic mechanisms—is a promising therapeutic strategy, yet cell states within PDAC harboring high plasticity remain poorly understood. Using genetically engineered mouse models driven by oncogenic KrasG12D and loss of p53 (“KPC” model), we profiled primary PDAC tumors using single-cell RNA sequencing and single cell ATAC sequencing. We identified a cancer cell differentiation state that co-expresses both epithelial and mesenchymal programs and has a distinct epigenetic landscape with accessible chromatin across gene sets that mark cancer cell states with divergent phenotypes, suggesting that this cell state is primed for cell state switching. Notably, this high-plasticity cell state (HPCS) signature aligns with a "basal" signature in human PDAC, which associates with poor prognosis and chemoresistance. To functionally interrogate the HPCS in PDAC progression, we developed a novel lineage-tracing and ablation system. Lineage-tracing experiments revealed that the HPCS has a robust capacity for differentiation into epithelial and mesenchymal states, indicating that the HPCS is functionally plastic and acts as a source of fixed malignant states. Strikingly, targeted elimination of the HPCS over short time periods resulted in tumor collapse, underscoring a critical role in tumor maintenance. Further, we found that the HPCS is highly dependent on KRAS activity, with acute KRAS inhibitor treatment leading to a selective depletion of the HPCS. Taken together, our data indicates that PDAC harbors a striking dependency on the HPCS and encourages therapeutic approaches aimed at eradicating this critical subset of cancer cells.
 Citation Format: Anupriya Singhal, Hannah C. Styers, Jonathan Rub, Zhuxuan Li, Zeynep Tarcan, Jill Hallin, Olca Basturk, Rona Yaeger, James G. Christensen, Doron Betel, Yan Yan, Elisa de Stanchina, Tuomas Tammela. Functional Dissection of the Highly Plastic Basal Cell State in Pancreatic Cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C077.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4987f81530acda2fa882016a6788c57664c546ba" target='_blank'>
              Abstract C077: Functional Dissection of the Highly Plastic Basal Cell State in Pancreatic Cancer
              </a>
            </td>
          <td>
            Anupriya Singhal, Hannah C Styers, Jonathan Rub, Zhuxuan Li, Z. Tarcan, Jill Hallin, O. Basturk, R. Yaeger, James G Christensen, D. Betel, Yan Yan, E. de Stanchina, Tuomas Tammela
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="


 Glioblastoma is an aggressive primary malignant brain tumour with poor survival. Unsuccessful improvement of treatment could be due to transcriptional plasticity of the cancer cells including stem-like cancer cells. Short-term cultured patient-derived neurospheres consisting of cancer cells and stem-like cancer cells are used in this project to find individual drug sensitivities via drug screening. Drug resistance mechanisms on transcriptome level is further explored at high resolution using single nucleus RNA sequencing (snRNA-seq) on drug- and mock-treated neurospheres to find drug vulnerabilities.



 Neurospheres were established from patients included in Protarget, a Phase II, prospective, non-randomized clinical trial. A library including BET-inhibitors was used for high-throughput drug screening to evaluate ex vivo patient drug sensitivities. DMSO-treated and drug-treated neurospheres were prepared for snRNA-seq using 10X genomics. We analyzed snRNA-seq using state-of-the art bioinformatic methods to derive e.g. differentially expressed genes, transcriptional states, and large-scale somatic copy number alterations.



 Through integrative analysis of drug- and DMSO-treated neurosphere cultures, we identified drug-specific differences in both transcriptional state, cell cycle, and stem-like signatures. We found that major copy number changes were preserved between the primary tumour surgical sample and neurosphere culture, exhibiting low average divergence. We found tumour-cell specific transcriptomic differences with clusters exhibiting more extreme drug-responses, implying that neurosphere cultures either sustain or develop sub-clonal, intrinsic cell-type specific drug-responses. One glioblastoma patient underwent treatment with a BET-inhibitor as part of a clinical trial and had initial good clinical response. The transcriptional response following treatment with the same BET-inhibitor on neurospheres from the same patient revealed an increased block in cell cycle, induced transcriptional state changes towards MES2- and OPC-like states, and revealed small clusters of tumour cells showing reduced response to the BET-inhibitor, indicating a pattern of drug resistance.



 We find overall high concordance between the initial tumour surgical sample, assessed by whole-genome sequencing, and patient-derived neurospheres at the level of large-scale copy number changes. Transcriptional subtypes change during treatment including response to a BET-inhibitor on sensitive patient-derived neurospheres. We conclude that combining high-throughput drug screening with snRNA-seq on DMSO-treated and drug-treated neurospheres is a feasible way to access response and resistance mechanisms in glioblastoma, and targeting transcriptional states could be a promising way to treat glioblastoma patients.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35ffeb471ac7dd263a23a8284dfbe8557f014c84" target='_blank'>
              P01.05.A DRUG-RESPONSE ANALYSIS AND SINGLE NUCLEUS RNA SEQUENCING ON PATIENT-DERIVED NEUROSPHERES TO REVEAL DRUG RESPONSE AND RESISTANCE MECHANISMS IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            M. J. S. Lü, J. Hendriksen, J. Ariey-Bonnet, A. Anand, S. R. Michaelsen, F. G. R. González, A. Locallo, A. Azam, J. Skjøth-Rasmussen, P. Hamerlik, D. Nørøxe, B. Hasselbalch, E. Vik-Mo, B. Kristensen, T. Urup, U. Lassen, H. Poulsen, K. Wennerberg, J. Weischenfeldt
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Genomic instability is a core characteristic of cancer, often stemming from defects in DNA damage response (DDR) or increased replication stress. DDR defects can lead to significant genetic alterations, including changes in gene copy numbers, gene rearrangements, and mutations, which accumulate over time and drive the clonal evolution of cancer cells. However, these vulnerabilities also present opportunities for targeted therapies that exploit DDR deficiencies, potentially improving treatment efficacy and patient outcomes. The development of PARP inhibitors like Olaparib has significantly improved the treatment of cancers with DDR defects (e.g., BRCA1 or BRCA2 mutations) based on synthetic lethality. This achievement has spurred further research into identifying additional therapeutic targets within the DDR pathway. Recent progress includes the development of inhibitors targeting other key DDR components such as DNA-PK, ATM, ATR, Chk1, Chk2, and Wee1 kinases. Current research is focused on optimizing these therapies by developing predictive biomarkers for treatment response, analyzing mechanisms of resistance (both intrinsic and acquired), and exploring the potential for combining DDR-targeted therapies with chemotherapy, radiotherapy, and immunotherapy. This article provides an overview of the latest advancements in targeted anti-tumor therapies based on DDR and their implications for future cancer treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cdf72d2a9fa59533e1f8a82e5fc32e8e54f1de8" target='_blank'>
              Advancing cancer therapy: new frontiers in targeting DNA damage response
              </a>
            </td>
          <td>
            Jiekun Qian, Guoliang Liao, Maohui Chen, Ren-Wang Peng, Xin Yan, Jianting Du, Renjie Huang, Maojie Pan, Yuxing Lin, Xian Gong, Guo-bing Xu, Bin Zheng, Chun Chen, Zhang Yang
          </td>
          <td>2024-09-20</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 Medulloblastoma (MB) is the most prevalent malignant brain tumor in children that demonstrates clinical and genomic heterogeneity. Of the four major subgroups, Group 3 tumors (MYC-MB), and particularly the 3gsubtype, exhibit high levels of MYC, high rates of metastasis, and have the worst prognosis. Despite treatment with surgery, radiation and chemotherapy, Group 3 medulloblastoma patients are more likely to develop recurrent tumors which are far more aggressive resulting in a 5-year survival rate after recurrence of 12%. Thus, there is an imperative need to examine the underlying mechanisms mediating treatment resistance. Recent evidence has identified cellular plasticity, as a major driver of therapy resistance. To investigate whether epigenetic mediated cell plasticity contributes to Myc-MB therapy response we performed Single Nuclei Multiome ATAC + Gene Expression analysis of paired primary and relapsed tumors. We identified the emergence of novel progenitor populations in resistant tumors with stem-like gene expression networks and altered chromatin accessibility. These emergent popuoations demostrated new enhancer-promoer interactions and exhibited increased metabolic reprogramming with potential for theraputic targeting. Because radiation is one of the key treatment modalities for Myc-MB, we chose to model radiation resistance in further studies. Single-cell RNA sequencing of outgrown in vivo irradiated orthotopic xenograft MB tumors showed an overall upregulation of metabolic pathways with the enhancement of genes that mediate oxidative phosphorylation and glutamate metabolism centering on wild-type IDH1 (wtIDH1), consistent with our patient data. IR-resistant cells showed altered chromatin occupancy of H3K3me3 and H3K27Ac. Disruption of IDH1 activity in IR resistant cells in vitro altered the epigenomic landscape, facilitating reversion to the pre-irradiated state. Interestingly, we observed that pluripotent genes with bivalent regions (H3K4me3 and H3K27me3) were enhanced in radiation-resistant cells and suppressed by IDH1 inhibition. Metabolic mass spectrometry reveled that metabolites from the citric acid cycle and synthesis of unsaturated fatty acids were among the main metabolically elevated molecules in radiation-resistant cells consistnat with the observed epigenomic reprograming. IDH1 inhibition reversed the metabolic reprograming and resensitized MB cells to radiation. In total our data identify a novel mechanism of radiation resistance in medulloblastoma with potential theraputic implications.
 Citation Format: Bethany Veo, Dong Wang, John Desisto, Rajeev Vibhakar. Mechanisms of radiation resistance in medulloblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A071.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aaed4654cb7c14fb325db4bb243c211a30ba0383" target='_blank'>
              Abstract A071 Mechanisms of radiation resistance in medulloblastoma
              </a>
            </td>
          <td>
            Bethany L. Veo, Dong Wang, J. DeSisto, Rajeev Vibhakar
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Investigating mutations, including single nucleotide variations (SNVs), gene fusions, alternative splicing and copy number variations (CNVs), is fundamental to cancer study. Recent computational methods and biological research have demonstrated the reliability and biological significance of detecting mutations from single-cell transcriptomic data. However, there is a lack of a single-cell-level database containing comprehensive mutation information in all types of cancer. Establishing a single-cell mutation landscape from the huge emerging single-cell transcriptomic data can provide a critical resource for elucidating the mechanisms of tumorigenesis and evolution. Here, we developed scTML (http://sctml.xglab.tech/), the first database offering a pan-cancer single-cell landscape of multiple mutation types. It includes SNVs, insertions/deletions, gene fusions, alternative splicing and CNVs, along with gene expression, cell states and other phenotype information. The data are from 74 datasets with 2 582 633 cells, including 35 full-length (Smart-seq2) transcriptomic single-cell datasets (all publicly available data with raw sequencing files), 23 datasets from 10X technology and 16 spatial transcriptomic datasets. scTML enables users to interactively explore multiple mutation landscapes across tumors or cell types, analyze single-cell-level mutation-phenotype associations and detect cell subclusters of interest. scTML is an important resource that will significantly advance deciphering intra-tumor and inter-tumor heterogeneity, and how mutations shape cell phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b44540801ce810af79670418ea8247b9a3e3dfa5" target='_blank'>
              scTML: a pan-cancer single-cell landscape of multiple mutation types.
              </a>
            </td>
          <td>
            Haochen Li, Tianxing Ma, Zetong Zhao, Yixin Chen, Xi Xi, Xiaofei Zhao, Xiaoxiang Zhou, Yibo Gao, Lei Wei, Xuegong Zhang
          </td>
          <td>2024-10-18</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Despite the general detriment of aneuploidy to cellular fitness, >90% of solid tumors carry an imbalanced karyotype. Regardless of this existing paradox, our understanding of the molecular responses to aneuploidy remains limited. Here, we explore these cellular stresses and unique vulnerabilities in aneuploid human mammary epithelial cells (HMECs) enriched for breast cancer-associated copy number alterations (CNAs). To uncover the genetic dependencies specific to aneuploid cells, we conducted a comprehensive, genome-wide CRISPR knockout screen targeting isogenic diploid and aneuploid HMEC lines. Our study reveals that aneuploid HMECs exhibit an increased reliance on pyrimidine biosynthesis and mitochondrial oxidative phosphorylation genes, and demonstrate heightened fitness advantages upon loss of tumor suppressor genes. Using an integrative multi-omic analysis, we confirm nucleotide pool insufficiency as a key contributor to widespread cellular dysfunction in aneuploid HMECs with net copy number gain. While diploid cells can switch seamlessly between pyrimidine synthesis and salvage, cells with increased chromosomal content exhibit p53 activation and S-phase arrest when relying on salvage alone, and exhibit increased sensitivity to DNA-damaging chemotherapeutics. This work advances our understanding of the consequences of aneuploidy and uncovers potential avenues for patient stratification and therapeutic intervention based on tumor ploidy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f75b91959ba71d0bf04e734b4ee5bf5730bc793" target='_blank'>
              An aneuploidy epistasis map reveals metabolic vulnerabilities associated with supernumerary chromosomes in cancer
              </a>
            </td>
          <td>
            R. Magesh, A. N. Kaur, F. Keller, A. Frederick, T. Tseyang, J. A. Haley, A. M. Rivera-Nieves, A. Liang, D. Guertin, J. Spinelli, S. Elledge, E. Watson
          </td>
          <td>2024-10-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="The integration of genomics into pharmacology has revolutionized the prediction of drug responses, enabling the personalization of medicine. Advances in next-generation sequencing and pharmacogenomics allow for the identification of genetic variants that influence drug efficacy and toxicity. By analyzing genetic data, researchers can develop predictive models that optimize drug selection and dosing based on individual patient genomes. This approach is particularly relevant in cancer treatment, where targeting specific genetic mutations enhances therapeutic outcomes. However, challenges remain in the annotation of non-coding regions and ethical concerns related to personalized medicine. This paper examines the current advancements in genomic technologies, key genetic variants affecting drug response, and their clinical applications, along with the potential future directions in this field. Keywords: Genomics, Pharmacogenomics, Personalized Medicine, Next-Generation Sequencing, Drug Response Prediction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd02d803d7b684652b4aec052531a8ab2115eefa" target='_blank'>
              The Role of Genomics in Predicting Drug Response
              </a>
            </td>
          <td>
            Mukamurera P. Nyiramana
          </td>
          <td>2024-09-01</td>
          <td>Research Output Journal of Public Health and Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Leukemias are a group of heterogeneous hematological malignancies driven by diverse genetic variations, and the advent of genomic sequencing technologies facilitates the investigation of genetic abnormalities in leukemia. However, these sequencing-based studies mainly focus on nuclear DNAs. Increasing evidence indicates that mitochondrial dysfunction is an important mechanism of leukemia pathogenesis, which is closely related to the mitochondrial genome variations. Here, we provide an overview of current research progress concerning mitochondrial genetic variations in leukemia, encompassing gene mutations and copy number variations. We also summarize currently accessible mitochondrial DNA (mtDNA) sequencing methods. Notably, somatic mtDNA mutations may serve as natural genetic barcodes for lineage tracing and longitudinal assessment of clonal dynamics. Collectively, these findings enhance our understanding of leukemia pathogenesis and foster the identification of novel therapeutic targets and interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a51e65e882f1837d1adb0460175f694592b51dd7" target='_blank'>
              Mitochondrial genetic variations in leukemia: a comprehensive overview
              </a>
            </td>
          <td>
            Ao Zhang, Wenbing Liu, S. Qiu
          </td>
          <td>2024-09-05</td>
          <td>Blood Science</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 Isocitrate dehydrogenase (IDH) mutant gliomas are progressive, invariably fatal brain tumors affecting young and middle-age adult patients. Previous single-cell transcriptomic characterization of IDH-mutant gliomas has uncovered a heterogeneity of cell states, with a developmental hierarchy that starts with neural progenitor-like stem cells that transition to more differentiated malignant cell states. Despite this understanding, the dynamics of these cell state transitions as tumors undergo grade progression, and the factors that facilitate or restrict cell state transitions remain poorly understood.



 Here, we characterize the cell states of IDH-mutant gliomas via single-cell gene expression (scRNA-seq) and single-cell analyses of accessible chromatin (scATAC-seq) in tumors spanning all grades using fresh patient samples and compare them with transcriptional programs and chromatin accessibility profiles of normal brain cells. We perform mechanism validation experiments using IDH-mutant glioma organoids.



 We find that tumor transcriptional cell states experience a marked shift as tumors undergo progression. In low-grade tumors, predominant cell states mimic slow-cycling oligodendrocyte progenitor-like cells (OPCs). With grade progression, predominant cell states transition to resemble highly proliferative neural progenitor-like cells (NPCs). Additionally, scATAC-seq analyses revealed novel permissive chromatin states, which could facilitate this transition. Genetic analyses identified several molecular alterations in high-grade tumors, including loss of DNA methylation, oncogene amplification, and the deletion of genes associated with the interferon (IFN) response. Those IFN genes, as well as tumor suppressors, are initially repressed by DNA hypermethylation in low-grade tumors. Lastly, we validate the role of DNA hypomethylation in favoring immune evasion with grade progression using an IDH-mutant glioma organoid model.



 Overall, our findings provide evidence that tumor progression in IDH-mutant gliomas is driven by an interplay of epigenetic and genetic mechanisms, with the former being determinant in low-grade tumors and the latter in high-grade tumors. The emergence of genetically-driven mechanisms promotes more mitotically active cell states as well as decreased anti-tumoral immune activation.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13fd62dc7724c0acd329992464a530f1dbcd46aa" target='_blank'>
              KS02.6.A SINGLE-CELL MULTIMODAL ANALYSIS OF IDH-MUTANT GLIOMAS UNCOVERS SHIFT FROM EPIGENETIC TO GENETIC DRIVERS WITH TUMOR PROGRESSION
              </a>
            </td>
          <td>
            L. G. Gonzalez Castro, J. Wu, S. Battaglia, C. El Farran, J. D’Antonio, T. Miller, M. Suvà, B. Bernstein
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Cancer is a complex disease characterized by genetic and molecular diversity, often involving dysregulation of critical cellular pathways. Recent advances in pan-cancer research have highlighted the importance of shared oncogenic mechanisms across different cancer types, providing new avenues for therapeutic exploration. Protein kinases, particularly those involved in phagocytosis, play pivotal roles in cellular homeostasis and immune response. This study systematically examines the genetic alterations and expression profiles of protein kinases associated with phagocytosis across various cancer types, using data from The Cancer Genome Atlas (TCGA) and other publicly available resources. We analyzed single nucleotide variations (SNVs), copy number variations (CNVs), methylation patterns, and mRNA expression to identify recurring alterations and their associations with survival outcomes. Our findings reveal that MET and MERTK are the most frequently mutated genes, with missense mutations dominating across cancers. CNV analysis shows significant correlations with survival in cancers like UCEC, KIRP, and KIRC, while methylation analysis indicates cancer-specific regulatory patterns affecting gene expression. Differential expression analysis highlights distinct cancer-type-specific expression profiles, with genes like MET and BTK displaying significant variation. Crosstalk pathway analysis further reveals the involvement of these kinases in key cancer-related pathways, such as epithelial-mesenchymal transition (EMT) and apoptosis. Drug sensitivity analysis identifies potential therapeutic targets, with gene expression correlating significantly with cancer cell line responsiveness to specific compounds. These findings underscore the importance of the phagocytotic kinome in cancer biology and suggest potential therapeutic strategies targeting protein kinases to enhance immune response and improve treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bec7e3ff62485c6898d190779703562a070a2f0e" target='_blank'>
              Protein Kinases in Phagocytosis: Promising Genetic Biomarkers for Cancer
              </a>
            </td>
          <td>
            Sadhika Arumilli, Hengrui Liu
          </td>
          <td>2024-10-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The study of chromosomal shape, characteristics, and behavior in somatic cell division (mitosis) during growth and development and in germ cell division (meiosis) during reproduction is known as cytogenetics. Many techniques can be used for cytogenetics, including fluorescent in situ hybridization (FISH), spectral karyotyping (SKY), multicolor FISH (M-FISH), microarray, and optical genome mapping (OGM). OGM is a novel genome-wide method that can identify structural variants (SVs) and copy number variants (CNVs) with only one test. Genomic structural information that is difficult to obtain with DNA sequencing can be promptly obtained with OGM, in which large molecule lengths can be mapped at a reasonable cost. OGM is increasingly being used to investigate chromosome abnormalities in genetic disorders and human cancer, but it was first utilized in genome assembly and research. According to recent research, OGM is capable of identifying every clinically significant variation seen in trials using conventional care. OGM is being utilized to identify genomic abnormalities in patients with malignancies and constitutional illnesses. It is regarded as a revolution in the field of cytogenetics. Rather than sequencing DNA, OGM relies on DNA labeling. Currently, the OGM technique with the Saphyr system from Bionano Genomics is a widely utilized platform for cytogenetic analysis. In conclusion, OGM can now be considered a highly reliable method for the identification of chromosomal abnormalities in the diagnosis of tumors and hematological diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26e6e0a2264e6747c604c9c870d801b4fcbe0131" target='_blank'>
              Cytogenetics and the Revolution of Optical Genome Mapping in the Diagnosis of Diseases.
              </a>
            </td>
          <td>
            Osama M Al-Amer, Yahya Khubrani
          </td>
          <td>2024-09-01</td>
          <td>Discovery medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Despite significant advancements in the treatment of cancer, therapeutic resistance remains a major hurdle for the achievement of full cures. Besides being typically driven by intratumoral heterogeneity, such acquired resistance also relies heavily on cell‐cell communication whereby extracellular vesicles (EVs) carrying resistance‐related components can be transferred from drug‐resistant cells or nontumorigenic members within tumor microenvironment (TME) to their sensitive neighbors. The cargo and membrane surface proteins of EVs derived from drug‐resistant cells and TME cells carry abundant biological information, transferring therapy‐resistant capacity to sensitive cancer cells. Hence, a deep understanding of the roles of EVs in cancer therapy resistance will facilitate identifying biomarkers and developing new approaches to restore therapy sensitivity. In this review, we summarize our current understanding regarding the causes and effects of EV‐mediated cell–cell communication in drug resistance, with a particular notice on how various kinds of cargoes derived from different cells within TME are linked to the resistant phenotype. We also discuss how this knowledge can contribute to improvements in clinical practice, that is, the opportunities and challenges of EVs in functioning as potential biomarkers in predicting therapeutic resistance and, by extension, EV‐based therapy in achieving deeper and longer‐lasting clinical responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9876d0c3e3a880c7817288b442b5c437be47fd4e" target='_blank'>
              Extracellular vesicles in cancer drug resistance: Mechanistic insights and therapeutic implications
              </a>
            </td>
          <td>
            Jing Zuo, Han Yan, Siyuan Qin, Yonghao Yan, Shiqi Wang, Li Yang, Jinlin Yang, Jun Yang
          </td>
          <td>2024-10-24</td>
          <td>MedComm – Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="TP53 functions as a central regulator in response to DNA damage and other cell stressors by inducing the expression of many protective pathways such as cell cycle arrest and apoptosis. Consequently, this gene is often found disrupted in human cancers. Elephants are a particularly interesting species for the study of cancer, by virtue of their large number of cell divisions and long lives yet low incidence of cancer. Elephants also possess multiple retrogene copies of TP53, which have previously been shown to induce strong cellular responses to DNA damage. However, most previous studies have largely focused only on African elephant TP53 retrogenes and often in non-native backgrounds. Here we generated CRISPR-Cas9 knockouts of TP53, all 29 TP53 retrogenes, or both in combination in Asian elephant fibroblasts. We find that while there is considerable overlap in the DNA damage responses of the TP53 and retrogene knockouts, there are also many unique pathways enriched in both. In particular, the retrogene knockouts exhibit strong enrichment of many extracellular pathways suggesting they may play a large role in the tumor microenvironment and mitigating metastatic growth. We also find that only a small fraction of these 29 retrogenes appear to be expressed across a variety of tissues and identify three loci that are likely driving this response. This work shows for the first time the transcriptomic effect of these retrogenes within their native background and establishes a foundation for future research into the relative contributions of these genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01cd22e9caf6290532e5ca297543c409aee6bcfc" target='_blank'>
              Engineering Asian elephant TP53: TP53 retrogene knockouts activate common and unique cancer-relevant pathways
              </a>
            </td>
          <td>
            Emil Karpinski, Nikil Badey, Esther Mintzer, Asaf Ashkenazy-Titelman, George M. Church
          </td>
          <td>2024-09-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Despite the crucial role of T cell clones in the anti-tumor activity, their characterization and association with clinical outcome following immune checkpoint inhibitors (ICI) is lacking. Here we analyzed paired single-cell RNA-sequencing/T-cell receptor sequencing of 767,606 T cells from 460 samples spanning 6 cancer types. We found a robust signature of response based on expanded CD8+ clones that differentiates between responders and non-responders. Analysis of persistent clones showed transcriptional changes that are differentially induced by therapy in the different response groups, suggesting an improved reinvigoration capacity in responding patients. Moreover, a gene trajectory analysis revealed changes in the pseudo-temporal state of de-novo clones that are associated with response to therapy. Lastly, we found that clones shared between tumor and blood are more abundant in non-responders and execute distinct transcriptional programs. Overall, our results highlight differences in clonal transcriptional states that are linked to patient response, offering valuable insights into the mechanisms driving effective anti-tumor immunity. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3504c21c4fc5c6f9ee1a5a18b7688458b8f59956" target='_blank'>
              Single-cell meta-analysis of T cells reveals clonal dynamics of response to checkpoint immunotherapy
              </a>
            </td>
          <td>
            Ofir Shorer, Asaf Pinhasi, Keren Yizhak
          </td>
          <td>2024-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f28b08db9038d5c3400d445c5aadfc149da32c65" target='_blank'>
              Centromeres in cancer: Unraveling the link between chromosomal instability and tumorigenesis.
              </a>
            </td>
          <td>
            Mohsen Karami Fath, Ahmad Nazari, Noushin Parsania, Paria Behboodi, Seyedeh Sara Ketabi, Pegah Razmjouei, Farnoosh Farzam, Seyyed-Ghavam Shafagh, Mohsen Nabi Afjadi
          </td>
          <td>2024-10-01</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d61b6a296f0f4f8539e1a19d49c6d27ddf36337d" target='_blank'>
              Deciphering the role of sphingolipid metabolism in the immune microenvironment and prognosis of esophageal cancer via single-cell sequencing and bulk data analysis
              </a>
            </td>
          <td>
            Rongzhang He, Jing Tang, Haotian Lai, Tianchi Zhang, Linjuan Du, Siqi Wei, Ping Zhao, Guobin Tang, Jie Liu, Xiufang Luo
          </td>
          <td>2024-09-27</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/416e85b3682e5ce745f93530334b6600c5e45ed0" target='_blank'>
              Joint single-cell genetic and transcriptomic analysis reveal pre-malignant SCP-like subclones in human neuroblastoma
              </a>
            </td>
          <td>
            T. Olsen, Jörg Otte, S. Mei, Bethel Tesfai Embaie, P. Kameneva, Huaitao Cheng, Teng Gao, V. Zachariadis, Ioanna Tsea, Åsa Björklund, Emil Kryukov, Ziyi Hou, Anna Johansson, Erik Sundström, Tommy Martinsson, S. Fransson, Jakob Stenman, S. S. Fard, J. Johnsen, P. Kogner, I. Adameyko, Martin Enge, P. V. Kharchenko, Ninib Baryawno
          </td>
          <td>2024-08-31</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Even though epigenetic factors contribute to oncogenesis, most human cancer models still assume that disease originates from driver DNA mutations. Thus, it is still unclear if non-genetic mechanisms are sufficient to trigger malignant transformation. Special AT-rich binding protein 2 (SATB2) is a chromatin organizer that brings distal DNA elements into close proximity, thus remodeling chromatin structures to reprogram cell-specific and/or developmentally-sensitive gene networks. Here, we discover that SATB2 generates a co-expressed circ3915 RNA that is translated into a peptide and co-locates with SATB2 in the cell nucleus. Ectopic SATB2 or circ3915 over- expression rearranges global chromatin accessibility, generates KRAS- and NFE2L2-like oncogenic gene expression patterns, and transforms lung epithelial cells independent of driver mutations. Thus, oncogenic pathways can be activated in mammalian cells without pre-disposing mutations in oncogenes or epigenetic regulators. Graphical abstract:">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2809e3fd65e4f1849899e9487a4c550596b35a99" target='_blank'>
              SATB2 and circ3915 RNA chromatin dysregulation drive KRAS-like oncogenic transformation
              </a>
            </td>
          <td>
            Rebekah Eleazer, Smitha George, Luke Shoemaker, Wesley N. Saintilnord, Kin H. Lau, Darrell Chandler, Yvonne Fondufe-Mittendorf
          </td>
          <td>2024-10-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="MicroRNAs are a conserved class of small, tissue-specific, non-coding RNAs that regulate gene expression to preserve cellular homeostasis. Proper miRNA expression is crucial for physiological balance because it affects numerous genetic pathways, including cell cycle control, proliferation, and apoptosis, through gene expression targeting. Deregulated miRNA expression has been implicated in several cancer types, including prostate cancer (PC), acting as tumor suppressors or oncogenes. Despite the availability of promising therapies to control tumor growth and progression, effective diagnostic and therapeutic strategies for different types of cancer are still lacking. PC continues to be a significant health challenge, particularly its castration-resistant (CRPC) form, which presents major therapeutic obstacles because of its resistance to conventional androgen deprivation treatments. This review explores miRNAs’ critical roles in gene regulation and cancer biology, as well as various miRNA delivery systems, highlighting their potential and the challenges in effectively targeting cancer cells. It aims to provide a comprehensive overview of the status of miRNA research in the fight against CRPC, summarizing miRNA-based therapies’ successes and limitations. It also highlights the promise of miRNAs as therapeutic agents for CRPC, underlining the need for further research to overcome existing challenges and move these therapies toward clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90859ee13ab8e70d2bc0534d1943825261dccf29" target='_blank'>
              MicroRNAs as Promising Therapeutic Agents Against Prostate Cancer Resistant to Castration—Where Are We Now?
              </a>
            </td>
          <td>
            Mariana Ferreira, Mariana Morais, Rui Medeiros, A. Teixeira
          </td>
          <td>2024-10-22</td>
          <td>Pharmaceutics</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40a14bfe5f9c18048663459b9f007d9027d3e70b" target='_blank'>
              Gene editing of NCF1 loci is associated with homologous recombination and chromosomal rearrangements
              </a>
            </td>
          <td>
            Federica Raimondi, Kah Mun Siow, Dominik Wrona, Carla Fuster-García, Oleksandr Pastukhov, Michael Schmitz, Katja Bargsten, Lucas Kissling, D. C. Swarts, G. Andrieux, Toni Cathomen, Ute Modlich, M. Jinek, Ulrich Siler, Janine Reichenbach
          </td>
          <td>2024-10-09</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Abstract Increasing incidence of Fragile X disorders (FXD) and of immune-mediated disorders in FXD suggests that additional factors besides FMR1 mutations contribute to the pathogenesis. Here, we discovered that the expression levels or splicing of specific transposon element (TE)-derived genes, regulating purine metabolism and immune responses against viral infections are altered in FXD. These genes include HLA genes clustered in chr6p21.3 and viral responsive genes in chr5q15. Remarkably, these TE-derived genes contain a low A T/C G suggesting base substitutions of A T to C G. The TE-derived genes with changed expression levels contained a higher content of 5′-CG-3′ dinucleotides in FXD compared to healthy donors. This resembles the genomes of some RNA viruses, which maintain high contents of CG dinucleotides to sustain their latent infection exploiting antiviral responses. Thus, past viral infections may have persisted as TEs, provoking immune-mediated disorders in FXD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/235377b99fb12efe55ace96fd3dd68ea05d3f5dd" target='_blank'>
              C G composition in transposon-derived genes is increased in FXD with perturbed immune system
              </a>
            </td>
          <td>
            Tamaki Suganuma, Huzaifa Hassan, Madelaine M. Gogol, Jerry L Workman
          </td>
          <td>2024-10-01</td>
          <td>Nar Molecular Medicine</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Cancer is a multifaceted disease influenced by both intrinsic cellular traits and extrinsic factors, with the tumor microenvironment (TME) being crucial in its progression. To satisfy their high proliferation and aggressiveness, cancer cell always plunders large amounts of nutrition and releases various signals to the surrounding, forming a dynamic TME with special metabolic, immune, microbial and physical characteristics. Due to the neglect of interactions between tumor cell and TME, traditional cancer therapies often struggle with challenges such as drug resistance, low efficacy, and recurrence. Importantly, with the development of gene editing technologies, particularly the CRISPR-Cas system, offers promising new strategies for cancer treatment. Combined with nanomaterials strategies, CRISPR-Cas technology exhibits precision, affordability, and user-friendliness with reduced side effects, which holds great promise for profoundly altering the TME at a genetic level, potentially leading to lasting anticancer outcomes. This review will delve into how CRISPR-Cas can be leveraged to manipulate the TME, examining its potential as a transformative anticancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b4994cb649ea8f94220d3676ac8b2eeb79b1144" target='_blank'>
              Harnessing Gene Editing Technology for Tumor Microenvironment Modulation: An Emerging Anticancer Strategy.
              </a>
            </td>
          <td>
            Shujun Feng, Yu Zhang, Yanyi Wang, Yanfeng Gao, Yujun Song
          </td>
          <td>2024-09-03</td>
          <td>Chemistry</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17ed0ba58e5446ac108522198d795c2809c0f59f" target='_blank'>
              Current Drug Resistance Mechanisms and Treatment Options in Gastrointestinal Stromal Tumors: Summary and Update.
              </a>
            </td>
          <td>
            Chunxiao He, Zilong Wang, Jiaying Yu, Shuang Mao, Xi Xiang
          </td>
          <td>2024-10-23</td>
          <td>Current treatment options in oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="T cells recognize an immense spectrum of pathogens to initiate immune responses by means of a large repertoire of T cell receptors (TCRs) that arise from somatic rearrangements of variable, diversity and joining gene segments at the TCR loci. These gene segments have emerged from a limited number of ancestral genes through a series of gene duplication events, resulting in a greatly variable number of such genes across different species. Apart from the complete V(D)J gene annotations in the human and mouse reference assemblies, little is known about the structure of TCR loci in other species. Here, we performed a comprehensive comparison of the TCRα and TCRβ gene segment clusters in mice and three of its closely related sister species. We show that the TCRα variable gene cluster is frequently rearranged, leading to deletions and sequence inversions in this region. The resulting complexity of TCR loci severely complicates the assembly of these loci and the annotation of gene segments. By jointly utilizing genomic and transcriptomic data, we show that in Mus musculus castaneus the variable gene cluster at the α locus has undergone a recent major locus contraction, leading to the loss of 74 variable gene segments. Additionally, we validated the expression of functional variable genes, including atypical ones with inverted orientation relative to other such segments. Disentangling the fine-scale structure of TCR loci in different species can provide valuable insights in the evolution and diversity of TCR repertoires.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/050dd2ec4e870352e83af91f6ed80d0e6588c1e2" target='_blank'>
              Distinct evolution at TCRα and TCRβ loci in the genus Mus
              </a>
            </td>
          <td>
            Moritz Peters, Volker Soltys, Dingwen Su, Y. F. Chan
          </td>
          <td>2024-09-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Retrotransposons are mobile DNA elements that are more active with increasing age and exacerbate aging phenotypes in multiple species. We previously reported an unexpected extension of chronological lifespan in the yeast, Saccharomyces paradoxus, due to the presence of Ty1 retrotransposons when cells were aged under conditions of mild stress. In this study, we tested a subset of genes identified by RNA-seq to be differentially expressed in S. paradoxus strains with a high-copy number of Ty1 retrotransposons compared with a strain with no retrotransposons and additional candidate genes for their contribution to lifespan extension when cells were exposed to a moderate dose of hydroxyurea (HU). Deletion of ADE8, NCS2, or TRM9 prevented lifespan extension, while deletion of CDD1, HAC1, or IRE1 partially prevented lifespan extension. Genes overexpressed in high-copy Ty1 strains did not typically have Ty1 insertions in their promoter regions. We found that silencing genomic copies of Ty1 prevented lifespan extension, while expression of Ty1 from a high-copy plasmid extended lifespan in medium with HU or synthetic medium. These results indicate that cells adapt to expression of retrotransposons by changing gene expression in a manner that can better prepare them to remain healthy under mild stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4a8b3e1192fabc90ee698772f4f9d415aa4b1e7" target='_blank'>
              Lifespan Extension by Retrotransposons under Conditions of Mild Stress Requires Genes Involved in tRNA Modifications and Nucleotide Metabolism
              </a>
            </td>
          <td>
            Patrick H. Maxwell, Mustafa Mahmood, Maya Villanueva, Kaitlyn Devine, Nina Avery
          </td>
          <td>2024-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Aberrant expression of the embryonal transcription factor TBXT (also known as brachyury) drives chordoma, a rare spinal neoplasm with no effective drug therapies. The gene network regulated by TBXT is poorly understood, and strategies to disrupt its abnormal activity for therapeutic purposes are lacking. Here, we developed TBXT-targeted designed ankyrin repeat proteins (T-DARPins) that selectively bind TBXT, inhibiting its binding to DNA and expression. In chordoma cells, T-DARPins reduced cell cycle progression, spheroid formation, and tumor growth in mice and induced morphologic changes indicative of senescence and differentiation. Combining T-DARPin-mediated TBXT inhibition with transcriptomic and proteomic analyses, we determined the TBXT regulome in chordoma cells, which comprises in particular networks involved in cell cycle regulation, DNA replication and repair, embryonal cell identity, metabolic processes, and interferon response. The analysis of selected TBXT regulome components provided new insights into chordoma biology, such as the strong upregulation of IGFBP3 upon TBXT inhibition to fine-tune part of TBXT’s downstream effectors. Finally, we assigned each TBXT regulome member a druggability status to create a resource for future translational studies and found high interferon response signaling in chordoma cell lines and patient tumors, which was promoted by TBXT and associated with strong sensitivity to clinically approved JAK2 inhibitors. These findings demonstrate the potential of DARPins to investigate the function of nuclear proteins to understand the regulatory networks of cancers driven by aberrant transcription factor activity, including novel entry points for targeted therapies that warrant testing in patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c807069740fb95738711a55a7d4d14824e2b642f" target='_blank'>
              Selective targeting of TBXT with DARPins identifies regulatory networks and therapeutic vulnerabilities in chordoma
              </a>
            </td>
          <td>
            Charles S. Umbaugh, Marie Groth, Cihan Erkut, Kwang-Seok Lee, Joana Marinho, Florian Iser, Jonas N. Kapp, Petra Schroeter, Simay Dolaner, Asli Kayserili, Julia Hartmann, Philipp Walch, Thomas F. E. Barth, K. Mellert, B. Dreier, J. Schaefer, Andreas Plückthun, Stefan Fröhling, Claudia Scholl
          </td>
          <td>2024-09-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Malignant melanoma is an aggressive cancer, with a high risk of metastasis and mortality rates, characterized by cancer cell heterogeneity and complex tumor microenvironment (TME). Single cell biology is an ideal and powerful tool to address these features at a molecular level. However, this approach requires enzymatic cell dissociation that can influence cellular coverage. By contrast, single nucleus RNA sequencing (snRNA-seq) has substantial advantages including compatibility with frozen samples and the elimination of a dissociation-induced, transcriptional stress response. To better profile and understand the functional diversity of different cellular components in melanoma progression, we performed snRNA-seq of 16,839 nuclei obtained from tumor samples along the growth of murine syngeneic melanoma model carrying a BRAFV600E mutation and collected 9 days or 23 days after subcutaneous cell injection. We defined 11 different subtypes of functional cell clusters among malignant cells and 5 different subsets of myeloid cells that display distinct global transcriptional program and different enrichment in early or advanced stage of tumor growth, confirming that this approach was useful to accurately identify intratumor heterogeneity and dynamics during tumor evolution. The current study offers a deep insight into the biology of melanoma highlighting TME reprogramming through tumor initiation and progression, underlying further discovery of new TME biomarkers which may be potentially druggable.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d37af9fbd7e323bb1c7c0e6480590eefcaf22d16" target='_blank'>
              Single-Nuclei Transcriptome Profiling Reveals Intra-Tumoral Heterogeneity and Characterizes Tumor Microenvironment Architecture in a Murine Melanoma Model
              </a>
            </td>
          <td>
            S. Parab, Valery Sarlo, S. Capellero, Luca Palmiotto, A. Bartolini, Daniela Cantarella, M. Turi, Annamaria Gullà, Elena Grassi, Chiara Lazzari, Marco Rubatto, Vanesa Gregorc, Fabrizio Carnevale-Schianca, Martina Olivero, Federico Bussolino, V. Comunanza
          </td>
          <td>2024-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="
 Background: Immune checkpoint blockades (ICBs) lack clinical efficacy in all-coming pediatric solid tumors. Gene expression-based classification of the tumor immune environment (TiME) can identify a subset of pediatric tumors with a rich immune environment, potentially actionable by ICBs. Methylation reprogramming is a major player in TiME modeling and ICB resistance, its role in pediatric tumors has been little explored. Hypothesis: dual-omic classification, combining DNA-methylation and RNA-sequencing, can identify tumors with epigenetically altered TiME, potentially responsible for ICB resistance. Objective: identify and characterize pediatric solid tumors with epigenetically altered immune phenotypes. Methods: We studied RNA-sequencing and DNA-methylation quantitative data from pediatric extra-cranial solid tumors. We used similarity network fusion (SNF) to individualize immune phenotypes. A preselected customized panel of 1226 immune genes, and their corresponding probes, were used for dual-omics clustering. Differential expression (DE) and methylation (DM) were performed between clusters using volcano3D. Scatter plots visualized the cis-regulation of probe-gene pairs with significant DE-DM. Gene set enrichment analyses (GSEA) were performed with clusterProfiler and gometh packages for DE and DM, respectively. Adjusted p values <0.01 were retained. Results: 184 samples were included. SNF clustering identified 3 phenotypes regrouping 52 (28%), 83 (45%), 49 (27%) samples in clusters (cl) 1, 2 and 3, respectively. Cl1 was characterized by a low expression of immune genes (cold phenotype), cl2 by overexpression in immune genes (hot phenotype), and cl3 by global hypermethylation (epigenetically altered phenotype). Both cl2 and cl3 overexpressed genes of immune checkpoints (CD274, PDCD1) and T-cell activator chemokines (CXCL9, CXCL10), central for ICB sensitivity. However, only cl2 overexpressed major histocompatibility complex (MHC) class I-II genes and their regulators (CIITA, TAP2), essential for antigen (Ag)-presenting machinery and immune recognition. Cl2 and 3 were also enriched for T-, B-cell and pro-inflammatory interferon gamma signaling pathways, known as ICB sensitivity biomarkers. Only cl2 was enriched in Ag processing and presentation pathway, confirming prior observation. DE-DM correlation showed that DNA-methylation programming induced overexpression of most immune genes (78%) in cl2, including CIITA, CXCR3 and MHC genes, when most immune genes (63%) were repressed by methylation in cl3. Methylation-based GSEA confirmed the epigenetic regulation of T- and B-cell signaling in cl2 and cl3, and for Ag presentation in cl2 only. Conclusion We demonstrated that DNA-methylation can reshape the TiME of pediatric solid tumors. We identified a subset of tumors with epigenetically altered immune phenotype characterized by inflamed immune environment but altered for Ag presentation by methylation reprogramming. Future studies should investigate methylation modulators to reverse these mechanisms and enable ICB sensitivity.
 Citation Format: Stéphanie Bianco, Anas Belaktib, Virgile Raufaste-Cazavieille, Charles Joly-Beauparlant, Lara Herrmann, Emeric Texeraud, Sylvie Langlois, Thomas Sontag, Alex Richard-St-Hilaire, Vincent-Philippe Lavallée, Thai Hoa Tran, Sonia Cellot, Daniel Sinnett, Arnaud Droit, Raoul Santiago. Dual-omic characterization of pediatric solid tumors identified a subset of tumors with epigenetically altered immune phenotype [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B057.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83a9f318a4bf2de77816f3008a669473ec30397b" target='_blank'>
              Abstract B057: Dual-omic characterization of pediatric solid tumors identified a subset of tumors with epigenetically altered immune phenotype
              </a>
            </td>
          <td>
            Stéphanie Bianco, Anas Belaktib, Virgile Raufaste-cazavieille, Charles Joly-Beauparlant, Lara Herrmann, Emeric Texeraud, Sylvie Langlois, Thomas Sontag, Alex Richard-St-Hilaire, Vincent-Philippe Lavallée, T. Tran, Sonia Cellot, D. Sinnett, Arnaud Droit, Raoul Santiago
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c751f02ecfd3cada2a90e62224689eeb8eaf33b4" target='_blank'>
              Forward–reverse mutation cycles in cancer cell lines under chemical treatments
              </a>
            </td>
          <td>
            Si Chen, I. S. Tyagi, Wai-Kin Mat, Muhammad A. Khan, Weijian Fan, Zhenggang Wu, Taobo Hu, Can Yang, Hong Xue
          </td>
          <td>2024-09-27</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Identification of somatic driver mutations in the noncoding genome remains challenging. To comprehensively characterize noncoding driver mutations for pancreatic ductal adenocarcinoma (PDAC), we first created genome-scale maps of accessible chromatin regions (ACRs) and histone modification marks (HMMs) in pancreatic cell lines and purified pancreatic acinar and duct cells. Integration with whole-genome mutation calls from 506 PDACs revealed 314 ACRs/HMMs significantly enriched with 3,614 noncoding somatic mutations (NCSMs). Functional assessment using massively parallel reporter assays (MPRA) identified 178 NCSMs impacting reporter activity (19.45% of those tested). Focused luciferase validation confirmed negative effects on gene regulatory activity for NCSMs near CDKN2A and ZFP36L2. For the latter, CRISPR interference (CRISPRi) further identified ZFP36L2 as a target gene (16.0 - 24.0% reduced expression, P = 0.023 - 0.0047) with disrupted KLF9 binding likely mediating the effect. Our integrative approach provides a catalog of potentially functional noncoding driver mutations and nominates ZFP36L2 as a PDAC driver gene.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db992992dda460b3aa1dd358fc40078a768ec148" target='_blank'>
              Large-scale multi-omic analysis identifies noncoding somatic driver mutations and nominates ZFP36L2 as a driver gene for pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Jun Zhong, Aidan O'Brien, Minal Patel, Daina Eiser, Michael Mobaraki, I. Collins, Li Wang, Konnie Guo, Thucnhi Truongvo, Ashley A. Jermusyk, Maura J. O’Neill, Courtney D. Dill, A. Wells, M. Leonard, J. Pippin, Struan F. A. Grant, Tongwu Zhang, T. Andresson, Katelyn E. Connelly, Jianxin Shi, H. E. Arda, Jason W. Hoskins, L. Amundadottir
          </td>
          <td>2024-09-24</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Renal cell carcinoma (RCC) has previously been considered as a single disease. However, it is currently characterized as a heterogeneous group of tumors that differ in histological features, genetic abnormalities, and variable clinical course. In normal cells, energy is produced by the cleavage of chemical bonds in nutrients through the oxidation of fats, proteins, or carbohydrates. Mutational alterations in genes associated with RCC, including VHL, FLCN, PTEN and SDH, lead to abnormal cellular adaptation to changes in oxygen status, iron metabolism and nutrients. The present paper reviews the known genetic abnormalities observed in RCC and their impact on metabolic alterations. Understanding the genetic and metabolic mechanisms underlying RCC is crucial for the development of effective therapies. Targeting specific genetic abnormalities or metabolic pathways represents a promising approach to the RCC treatment. In addition, studies into the metabolic basis of RCC contribute to the development of new biomarkers for early diagnosis and monitoring of the disease. Moreover, investigating the role of VHL, FLCN, PTEN, and SDH genes in the development of RCC provides valuable information on the molecular mechanisms behind the disease. As a result, it may lead to the development of new treatment strategies aimed at restoring the normal function of these genes or compensating for their abnormalities. Overall, an integrated approach to the study of RCC that considers genetic, metabolic, and clinical aspects will ensure that more effective treatments are developed and prognosis for patients with this disease are improved.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b1601eb66db07531257237b1f84d2ed09ae8e4e" target='_blank'>
              Genetic Variation and Metabolic Basis of Kidney Cancer: New Opportunities for Targeted Therapy
              </a>
            </td>
          <td>
            M. Urmantsev, R. I. Tavabilov, M. R. Bakeev
          </td>
          <td>2024-10-14</td>
          <td>Creative surgery and oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/827407d1ee135266c2bdec8a7b4b96ce85a8df0f" target='_blank'>
              A genome-wide CRISPR/Cas9 knockout screen identifies SEMA3F gene for resistance to cyclin-dependent kinase 4 and 6 inhibitors in breast cancer.
              </a>
            </td>
          <td>
            Yuko Kawai, Aiko Nagayama, Kazuhiro Miyao, Makoto Takeuchi, Takamichi Yokoe, Tomoe Kameyama, Xinyue Wang, T. Seki, Maiko Takahashi, T. Hayashida, Yuko Kitagawa
          </td>
          <td>2024-10-01</td>
          <td>Breast cancer</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) is the deadliest and most common form of brain tumour. The average length of survival for GBM patients remains low, in part, due to a lack of targeted therapeutics. Identiﬁcation and validation of novel therapeutic targets is therefore crucial. Chomobox 2 (CBX2; a component of polycomb repressive complex 1 (PRC1)), represses gene expression and has a pro-oncogenic role in several aggressive cancer subtypes. CBX2 expression is elevated in GBM, however, very little is known about its role in this deadly disease. We aim to assess the phenotypic and gene expression effects of preventing CBX2-chromatin interaction in patient-derived GBM cell models, and to determine the functional mechanisms by which CBX2 promotes GBM progression; thereby determining its potential as a novel therapeutic target.



 Crucial to impactful target validation is the use of translationally relevant models, this study utilises a 3D spheroid model, using patient-derived cells, which more closely recapitulates the in vivo tumour microenvironment. GBM cell monolayers and spheroids were treated with SW2_152F, a selective CBX2 inhibitor. The phenotypic effects of CBX2 inhibition were assessed by cell count, CellTiter-Glo viability assays, ﬂuorescence microscopy, and ﬂow cytometry. Genes and biological pathways differentially regulated following CBX2 inhibition were assessed by RNA-sequencing and Gene Set Enrichment Analysis (GSEA).



 Inhibition of CBX2 induced cell death and reduced cell viability in both 2D and 3D GBM cultures. RNA-seq and GSEA of SW2_152F-treated cells identiﬁed dysregulation of gene signatures involved in pro-oncogenic MYC and E2F signalling pathways, and dysregulation of genes associated with G2/M cell cycle progression.



 Preventing CBX2-chromatin interaction induces GBM cell death. Further research will identify CBX2- chromatin binding sites to characterise the activity and direct regulatory role of CBX2 in GBM. Furthermore, we will assess the phenotypic and gene expression effects of CBX2 inhibition and depletion on ex vivo maintained GBM tissue biopsies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41607adec8f62477f0d798d3212dbec12fcc049c" target='_blank'>
              THE EPIGENETIC REGULATORY PROTEIN CBX2 AS A NOVEL THERAPEUTIC TARGET FOR GLIOBLASTOMA
              </a>
            </td>
          <td>
            Ms Danielle Marsh, Prof John Greenman, Dr Mark Wade
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Chemical screens across hundreds of cell lines have shown that the drug sensitivities of human cancers can vary by genotype or lineage. However, most drug discovery studies have relied on culture media that poorly reflect metabolite levels in human blood. Here, we perform drug screens in traditional and Human Plasma–Like Medium (HPLM). Sets of compounds that show conditional anticancer activity span different phases of global development and include non-oncology drugs. Comparisons of the synthetic and serum-derived components that comprise typical media trace sets of conditional phenotypes to nucleotide synthesis substrates. We also characterize a unique dual mechanism for brivudine, a compound approved for antiviral use. Brivudine selectively impairs cell growth in low folate conditions by targeting two enzymes involved in one-carbon metabolism. Cataloged gene essentiality data further suggest that conditional phenotypes for other compounds are linked to off-target effects. Our findings establish general strategies for identifying drug-nutrient interactions and mechanisms of action by exploiting conditional lethality in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52075b84c4effc26f41a203f90514fe9f3c95c9f" target='_blank'>
              Conditional lethality profiling reveals anticancer mechanisms of action and drug-nutrient interactions
              </a>
            </td>
          <td>
            K. Flickinger, Kelli M. Wilson, Nicholas J. Rossiter, Andrea L. Hunger, Paresh V Vishwasrao, Tobie D. Lee, Carlos A. Mellado Fritz, Rebecca M Richards, Matthew D. Hall, Jason R. Cantor
          </td>
          <td>2024-10-04</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46a593c69280acd8e1ddd89d1994ea815f977cea" target='_blank'>
              Chromosomal instability: a key driver in glioma pathogenesis and progression
              </a>
            </td>
          <td>
            Adele Mazzoleni, W. Awuah, Vivek Sanker, H. Bharadwaj, N. Aderinto, J. K. Tan, H. Huang, Jeisun Poornaselvan, Muhammad Hamza Shah, O. Atallah, Aya Tawfik, Mohamed Elmanzalawi, S. H. Ghozlan, T. Abdul-Rahman, Jeremiah Adepoju Moyondafoluwa, Athanasios Alexiou, Marios Papadakis
          </td>
          <td>2024-09-04</td>
          <td>European Journal of Medical Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 Hotspot mutations in the IDH genes serve as tumor-initiating events that define a subset of adult diffuse glioma that can further be divided into either oligodendroglioma or astrocytoma subtypes. Despite standard treatment, IDH-mutant gliomas inevitably recur. Previous studies suggest that therapeutic resistance may result from a combination of intratumoral cellular heterogeneity, epigenetic evolution, and acquired genetic alterations. However, how these multilayered molecular features interact to influence the evolutionary paths of IDH-mutant gliomas remains incompletely understood.



 To chart this evolution, 75 glioma samples were longitudinally collected from 35 patients (n = 13 oligodendroglioma, n = 22 astrocytoma) and profiled using single nucleus RNA sequencing, ATAC sequencing, and bulk DNA sequencing.



 Analysis of 331,016 nuclei (snRNA) revealed eight tumor microenvironment cell types and five pan-IDH-mutant malignant cellular states. The malignant states were distributed along a cellular hierarchy of stem-like populations (neural progenitor cell-like, oligodendrocyte progenitor cell-like, and undifferentiated) and more differentiated populations (mesenchymal-like and astrocyte-like) with cycling cells enriched in the stem-like populations. Joint open chromatin accessibility data (snATAC) was available for a subset of malignant cells (71,088 nuclei) and state-specific differentially accessible peaks supported that these states are epigenetically encoded. Across both subtypes, higher tumor grade was associated with reduced malignant cell differentiation and increased cycling populations. Between the two time points, we identified that there was a longitudinal increase in the cycling, undifferentiated, and mesenchymal-like populations with a corresponding decrease in the astrocyte-like population. Longitudinal genetic analysis revealed that 19 of 35 tumors acquired at least one of the following genetic alterations: treatment-associated hypermutation, cell cycle alteration, or large changes in copy number alterations. Importantly, tumors that acquired these key genetic alterations demonstrated significant shifts towards reduced differentiation and increased cycling populations while those tumors that did not had more stable malignant profiles.



 Collectively, our results suggest a common cellular hierarchy across IDH-mutant gliomas with a shift towards reduced differentiation, increased cycling populations during disease progression that is driven by acquired genetic alterations.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d952023b3d2c6da79f53a908951847332e2e66b4" target='_blank'>
              KS02.7.A LONGITUDINAL SINGLE-CELL ANALYSES IDENTIFY DRIVERS OF GENETIC, EPIGENOMIC, AND CELLULAR EVOLUTION IN IDH-MUTANT GLIOMA
              </a>
            </td>
          <td>
            K. Johnson, A. Spitzer, F. Varn, M. Nomura, L. Garofano, T. Chowdhury, K. J. Anderson, F. D’Angelo, L. Bussema, S. Gritsch, Y. Oh, H. Moon, S. Paek, F. Bielle, A. Laurenge, A. D. Di Stefano, B. Mathon, A. Picca, M. Sanson, A. Lipsa, F. Hertel, Z. Zhao, Q. Wang, T Jiang, B. Hermes, N. Sanai, A. Gołȩbiewska, S. Niclou, J. Huse, W. Yung, A. Lasorella, M. Suvà, A. Iavarone, I. Tirosh, R. Verhaak
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Mutations in genes involved in DNA damage repair (DDR) often lead to premature aging syndromes. While recent evidence suggests that inflammation, alongside mutation accumulation and cell death, may drive disease phenotypes, its precise contribution to in vivo pathophysiology remains unclear. Here, by modeling Ataxia Telangiectasia (A-T) and Bloom Syndrome in the African turquoise killifish (N. furzeri), we replicate key phenotypes of DDR syndromes, including infertility, cytoplasmic DNA fragments, and reduced lifespan. The link between DDR defects and inflammation is attributed to the activation of the cGAS-STING pathway and interferon signaling by cytoplasmic DNA. Accordingly, mutating cGAS partially rescues germline defects and senescence in A-T fish. Double mutants also display reversal of telomere abnormalities and suppression of transposable elements, underscoring cGAS’s non-canonical role as a DDR inhibitor. Our findings emphasize the role of interferon signaling in A-T pathology and identify the cGAS-STING pathway as a potential therapeutic target for genomic instability syndromes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0762a0a7a477e0941f31815c1aa8f1b5b79a238" target='_blank'>
              The cGAS-STING pathway is an in vivo modifier of genomic instability syndromes
              </a>
            </td>
          <td>
            Marva Bergman, Uri Goshtchevsky, Tehila Atlan, Gwendoline Astre, Ryan Halabi, Hosniyah El, Eitan Moses, A. J. Lemus, BA Benayoun, Y. Tzfati, Ido Ben-Ami, I. Harel
          </td>
          <td>2024-10-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b681524478cf528a97b828064b23970db22097a" target='_blank'>
              Epigenomic heterogeneity as a source of tumour evolution.
              </a>
            </td>
          <td>
            Marthe Laisné, Mathieu Lupien, C. Vallot
          </td>
          <td>2024-10-16</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Growing evidence has demonstrated the role of mutations of tumor biomarkers in diagnosing and treating epithelial ovarian cancer. This review aims to analyze recent literature on the correlation between tumor biomarkers and chemotherapy in nonmucinous ovarian cancer, providing suggestions for personalized treatment approaches. An extensive literature search was conducted to identify relevant studies and trials. BRCA1/2 mutations are central in homologous recombination repair deficiency (HRD) in ovarian cancer, but several other genetic mutations also contribute to varying cancer risks. While the role of MMR testing in ovarian cancer is debated, it is more commonly linked to non-serous ovarian cancer, often associated with Lynch syndrome. A significant proportion of ovarian cancer patients have HRD, affecting treatment decisions in both first-line (especially in advanced stages) and second-line therapy due to HRD’s connection with platinum-based therapy and PARP inhibitors’ response. However, validated genetic tests to identify HRD have not yet been universally implemented. There is no definitive therapeutic algorithm for advanced ovarian cancer, despite ongoing efforts and multiple proposed tools. Future research should focus on expanding the utility of biomarkers, reducing resistance, and increasing the actionable biomarker pool.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0392f9f602c007142705c5124b90775a50775606" target='_blank'>
              The Role of Tumor Biomarkers in Tailoring the Approach to Advanced Ovarian Cancer
              </a>
            </td>
          <td>
            Noemi Tonti, T. Golia D’augè, Ilaria Cuccu, Emanuele De Angelis, O. D’Oria, G. Perniola, A. S. Laganà, Andrea Etrusco, Federico Ferrari, Stefania Saponara, V. Di Donato, G. Bogani, Andrea Giannini
          </td>
          <td>2024-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Patients with triple-negative breast cancer (TNBC) have limited targeted therapeutic options. PARP inhibitors (PARPi) have demonstrated an important role for BRCA-mutant patients with early TNBC. Combination approaches with PARPi can broaden the use of PARPi to a larger cohort of TNBC patients. We selected six genes from our previously identified 63-gene signature that was associated with PARPi response. siFEN1 increased cells in G2/M arrest, DNA damage and particularly apoptosis. Targeting FEN1 with a chemical inhibitor enhanced the efficacy of PARPi in 7/10 cell lines, and synergy was demonstrated mainly in PARPi-resistant TNBC cell lines. A BRCA2-mutant cell line with acquired resistance to olaparib (HCC1395-OlaR) was strongly synergistic, with a combination index value of 0.20. The combination of PARPi and FEN1 inhibition also showed synergy in a PARPi-resistant xenograft-derived organoid model. Two mechanisms which explain the underlying efficacy are rapid progression in DNA replication fork speed and enhancement of DNA damage. The combination induced the highest fork speed (47% difference in comparison to control, P<0.0001) when FEN1 inhibition and PARPi equally increased fork speed individually in a cell line with a pre-existing increase in replication stress. The combination also increased DNA damage at lower drug concentrations, driving response in most of the synergistic cell lines. Gene expression analysis suggested that the sensitizing role of FEN1 inhibition in PARPi-resistant cell lines may be due to downregulation of pathways including mismatch repair. Therefore, targeting FEN1 shows great therapeutic potential as a targeted combination approach, particularly in the context of PARPi-resistant TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51da609111db5e5375ca86bde9a72eef75e23b2b" target='_blank'>
              Targeting FEN1 to enhance efficacy of PARP inhibition in triple-negative breast cancer
              </a>
            </td>
          <td>
            Mallory I. Frederick, Elicia Fyle, Anna Clouvel, Djihane Abdesselam, Saima Hassan
          </td>
          <td>2024-10-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background: Comprehensive genomic profiling (CGP) identifies genetic alterations and patterns that are crucial for therapy selection and precise treatment development. In Colombia, limited access to CGP tests underscores the necessity of documenting the prevalence of treatable genetic alterations. This study aimed to describe the somatic genetic profile of specific cancer types in Colombian patients and assess its impact on treatment selection. Methods: A retrospective cohort study was conducted at Clínica Colsanitas S.A. from March 2023 to June 2024. Sequencing was performed on the NextSeq2000 platform with the TruSight Oncology 500 (TSO500) assay, which simultaneously evaluates 523 genes for DNA analysis and 55 for RNA; additionally, analyses were performed with the SOPHiA DDM software. The tumor mutational burden (TMB), microsatellite instability (MSI), and programmed cell death ligand 1 (PDL1) were assessed. Results: Among 111 patients, 103 were evaluated, with gastrointestinal (27.93%), respiratory (13.51%), and central nervous system cancers (10.81%) being the most prevalent. TP53 (37%), KMT2C (28%), and KRAS (21%) were frequent mutations. Actionable findings were detected in 76.7% of cases, notably in digestive (20 patients) and lung cancers (8 patients). MSI was stable at 82.52% and high at 2.91%, whilst TMB was predominantly low (91.26%). Conclusions: The test has facilitated access to targeted therapies, improving clinical outcomes in Colombian patients. This profiling test is expected to increase opportunities for personalized medicine in Colombia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93aa6bc1a8a2f18dd281151deeb80b4be0a23d78" target='_blank'>
              Advancing Cancer Care in Colombia: Results of the First In Situ Implementation of Comprehensive Genomic Profiling
              </a>
            </td>
          <td>
            Juan Javier López Rivera, Paula Rueda-Gaitán, Laura Camila Rios Pinto, Diego Alejandro Rodríguez Gutiérrez, Natalia Gomez-Lopera, J. Lamilla, Fabio Andrés Rojas Aguirre, Laura Bernal Vaca, M. Isaza-Ruget
          </td>
          <td>2024-09-01</td>
          <td>Journal of Personalized Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Pancreatic adenocarcinoma (PDAC) is the 3rd leading cause of cancer-related deaths in Canada with a 5-year survival rate at ∼12%. A major cause of this poor survival rate is the inefficiency of current treatments. Only 20% of patients are eligible for surgical resection while the other 80% are treated with chemotherapies (nab-paclitaxel, Gemcitabine, FOLFIRINOX) having limited efficacy. This resistance is due to the PDAC tumour microenvironment (TME), including cancer-associated fibroblasts (CAFs). Numerous studies show CAFs and the TME affect tumour growth, chemoresistance and metastasis, but the mechanisms underlying the effects are unknown. Epigenetic mechanisms, including DNA methylation and histone modifications, link environmental factors to altered gene expression. Genes encoding epigenetic mediators are often mutated or over-expressed in PDAC, but a role in resistance has yet to be defined. We hypothesize that epigenetic reprogramming occurs in pancreatic cancer cells in response to secreted factors from CAFs leading to increased therapy resistance. Methods: Most work to date on PDAC resistance has been performed with immortalized cell lines and in two dimensions. Therefore, to test our hypothesis, we established patient-derived organoids (PDO) and CAFs obtained from endoscopic ultrasound (EUS). Clinical data was obtained through the DERIVE (Determination of Response to Therapy in Individual Patients) database. To examine the effects of CAFs on PDOs, we obtained conditioned media (CM) from four, early passage CAFs, grown 48 hours in culture. Organoid media was supplemented 1:1 with CAF-CM and six PDOs grown in this media for up to 14 days. PDO growth and the response to gemcitabine, FOLFIRINOX or radiation was monitored using an Incucyte system. To identify changes in the epigenome and how this reflects changes in gene expression, DNA methylation arrays and ATAC-Seq was performed before and after treatment with CAF-CM and aligned to RNA-seq analysis on RNA isolated at the same time point. Results: Exposure of PDOs to CAF-CM resulted in altered phenotypes and molecular profiles of PDOs in a patient-specific fashion. Increased resistance to chemotherapy (Gemcitabine) and radiotherapy was observed in multiple PDOs. Changes in DNA methylation and chromatin accessibility was aligned with transcriptomic data and identified known and novel pathways linked to chemoresistance. Metabolomic and cytokine analysis on CAFs-CM identified differential enrichment of metabolites linked to epigenetic mediators. We are currently determining if these factors secreted by the CAFs promote chemo and radio-resistance. Conclusions: This study shows the importance in integrating TME components when examining novel therapeutic options. In addition, we show (1) CAFs secrete factors that enhance therapeutic resistance in PDOs and (2) these changes likely involve altered epigenetic reprogramming. Future experiments will identify the epigenetic factors that can be targeted to counteract chemoresistance.
 Citation Format: Emilie Jaune-Pons, Zachary Klassen, Joana Ribeiro Pinto, Maria Nica, Nadeem Hussain, Michael Sey, Ken Leslie, Matthew Cecchini, Crystal Engelage, Danielle Porplycia, Stephen Welch, Brian Yan, Christopher Pin. Primary cancer-associated fibroblasts alter tumor organoid chemosensitivity through epigenetic regulation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A058.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41115ec236606c659caa4488fce0af64b664490b" target='_blank'>
              Abstract A058: Primary cancer-associated fibroblasts alter tumor organoid chemosensitivity through epigenetic regulation
              </a>
            </td>
          <td>
            Emilie Jaune-Pons, Zachary Klassen, Joana Ribeiro Pinto, Maria Nica, Nadeem Hussain, Michael Sey, Ken Leslie, Matthew Cecchini, Crystal Engelage, Danielle Porplycia, Stephen Welch, Brian Yan, Christopher Pin
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 METTL1 is an RNA methyltransferase which catalyzes N7-methylguanosine (m7G) methylation on various RNAs, particularly on tRNAs. METTL1 has been reported to be amplified and/or overexpressed and high expression correlates with poor patient prognosis in different types of adult cancers. Our ZERO Childhood Cancer Program focuses on identifying and validating potential new drug targets and developing novel therapies for clinical use. ZERO has sequenced the whole genome and transcriptome in over 500 high-risk pediatric tumors and found that METTL1 gene copy number and gene expression are significantly higher in a subset of high-grade glioma, neuroblastoma, diffuse midline glioma and fusion positive rhabdomyosarcoma. Analysis of the publicly available SEQC microarray dataset showed that high METTL1 expression correlates with poor outcomes in MYCN amplified neuroblastoma patients. We validated the functional roles of METTL1 gene amplification and overexpression in a few pediatric cancer cell line models using the Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR) Cas13d knockdown system, in dual lentiviral constructs. We found that knocking down METTL1 significantly decreased METTL1 amplified neuroblastoma and rhabdomyosarcoma cell viability and colony formation but not in METTL1 non-amplified neuroblastoma and rhabdomyosarcoma cells. Gene Set Enrichment Analysis from RNA sequencing data revealed that knocking down METTL1 resulted in a decrease in Myc target gene expression, and an increased in apoptosis, interferon alpha response and p53 pathway hallmarks. Western blot analysis confirmed that knocking down METTL1 led to a decrease in the MCL-1 antiapoptotic protein expression and an increase in the proapoptotic protein expression such as cleaved PARP and caspase 3. Taken together, our data demonstrate that METTL1 is a potential therapeutic target in METTL1 amplified pediatric cancers.
 Citation Format: Pei Y. Liu, Steven He, Chelsea Mayoh, Alice Salib, Antoine De Weck, Paul G. Ekert. Unravelling the role of METTL1 in Myc driven pediatric cancers using CRISPR Cas13d [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A012.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64085373809830d692bd2bd083b18380125ea0bf" target='_blank'>
              Abstract A012 Unravelling the role of METTL1 in Myc driven pediatric cancers using CRISPR Cas13d
              </a>
            </td>
          <td>
            Pei Y. Liu, Steven He, C. Mayoh, A. Salib, Antoine de Weck, Paul G Ekert
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="TP53, the most frequently mutated gene in human cancer, encodes a transcriptional activator that induces myriad downstream target genes. Despite the importance of p53 in tumor suppression, the specific p53 target genes important for tumor suppression remain unclear. Recent studies have identified the p53-inducible gene Zmat3 as a critical effector of tumor suppression, but many questions remain regarding its p53-dependence, activity across contexts, and mechanism of tumor suppression alone and in cooperation with other p53-inducible genes. To address these questions, we used Tuba-seqUltra somatic genome editing and tumor barcoding in a mouse lung adenocarcinoma model, combinatorial in vivo CRISPR/Cas9 screens, meta-analyses of gene expression and Cancer Dependency Map data, and integrative RNA-sequencing and shotgun proteomic analyses. We established Zmat3 as a core component of p53-mediated tumor suppression and identified Cdkn1a as the most potent cooperating p53-induced gene in tumor suppression. We discovered that ZMAT3/CDKN1A serve as near-universal effectors of p53-mediated tumor suppression that regulate cell division, migration, and extracellular matrix organization. Accordingly, combined Zmat3-Cdkn1a inactivation dramatically enhanced cell proliferation and migration compared to controls, akin to p53 inactivation. Together, our findings place ZMAT3 and CDKN1A as hubs of a p53-induced gene program that opposes tumorigenesis across various cellular and genetic contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e5afaeddf5354538c432a3c29c86495b8e7485c" target='_blank'>
              Integrative multiomic approaches reveal ZMAT3 and p21 as conserved hubs in the p53 tumor suppression network
              </a>
            </td>
          <td>
            A. Boutelle, Aicha R. Mabene, David Yao, Haiqing Xu, Mengxiong Wang, Yuning J. Tang, Steven S. Lopez, Sauradeep Sinha, J. Demeter, Ran Cheng, Brooks A. Benard, Liz J. Valente, Alexandros P. Drainas, Martin Fischer, Ravi Majeti, D. Petrov, Peter K. Jackson, Fan Yang, M. Winslow, Mike C Bassik, L. Attardi
          </td>
          <td>2024-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ac67a3ed7d66d72ed2d1b0afc516f030fad809a" target='_blank'>
              Immunological hide-and-seek: epigenetically reprogrammed cancer cells and the dynamics of CD8+ T cells.
              </a>
            </td>
          <td>
            Jie-Ting Low, M. W. Chan, Cheng-Huang Shen, Kuo-Liang Wei
          </td>
          <td>2024-09-04</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1bbe642fe1a5320aff996d83dd54d13a05e4037" target='_blank'>
              The intersection of homologous recombination (HR) and mismatch repair (MMR) pathways in DNA repair-defective tumors
              </a>
            </td>
          <td>
            L. Incorvaia, T. B. Bazan Russo, V. Gristina, A. Perez, C. Brando, Clarissa Mujacic, Emilia Di Giovanni, M. Bono, Silvia Contino, Carla Ferrante Bannera, Maria Concetta Vitale, Andrea Gottardo, M. Peri, A. Galvano, D. Fanale, G. Badalamenti, Antonio Russo, V. Bazan
          </td>
          <td>2024-09-05</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="The chromatins are folded into three-dimensional (3D) structures inside cells, which coordinates the regulation of gene transcription by the non-coding regulatory elements. Aberrant chromatin 3D folding has been shown in many diseases, such as acute myeloid leukemia (AML), and may contribute to tumorigenesis. The anthracycline topoisomerase II inhibitors can induce histone eviction and DNA damage. We performed genome-wide high-resolution mapping of the chemotherapeutic effects of various clinically used anthracycline drugs. ATAC-seq was used to profile the histone eviction effects of different anthracyclines. TOP2A ChIP-seq was used to profile the potential DNA damage regions. Integrated analyses show that different anthracyclines have distinct target selectivity on epigenomic regions, based on their respective ATAC-seq and ChIP-seq profiles. We identified the underlying molecular mechanism that unique anthracycline variants selectively target chromatin looping anchors via disrupting CTCF binding, suggesting an additional potential therapeutic effect on the 3D genome. We further performed Hi-C experiments, and data from K562 cells treated with the selective anthracycline drugs indicate that the 3D chromatin organization is disrupted. Furthermore, AML patients receiving anthracycline drugs showed altered chromatin structures around potential looping anchors, which linked to distinct clinical outcomes. Our data indicate that anthracyclines are potent and selective epigenomic targeting drugs and can target the 3D genome for anticancer therapy, which could be used for personalized medicine to treat tumors with aberrant 3D chromatin structures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/760450c07be781a1d46120b324cb67687f8de734" target='_blank'>
              Targeting the 3D genome by anthracyclines for chemotherapeutic effects
              </a>
            </td>
          <td>
            Minkang Tan, Shengnan Sun, Yuchen Liu, Andrea A. Perreault, D. Phanstiel, Liping Dou, Baoxu Pang
          </td>
          <td>2024-10-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Adrenocortical carcinoma (ACC) is a rare endocrine malignancy with a poor prognosis and limited treatment options. Bulk genomic characterization of ACC has not yielded obvious therapeutic or immunotherapeutic targets, yet novel therapies are needed. We hypothesized that elucidating the intratumoral cellular heterogeneity by single nuclei RNA sequencing analyses would yield insights into potential therapeutic vulnerabilities of this disease. In addition to characterizing the immune cell and fibroblast landscape, our analyses of single nuclei gene expression profiles identified an adrenal cortex cell cluster exhibiting a program of replication stress and DNA damage response in primary and metastatic ACC. In vitro assessment of replication stress and DNA damage response using an ACC cell line and a series of newly-derived hormonally active patient-derived tumor organoids revealed ATR sensitivity. These findings provide novel mechanistic insight into ACC biology and suggest that an underlying dependency on ATR may be leveraged therapeutically in advanced ACC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1469f57c44a466002613ce12061fcafe073cc550" target='_blank'>
              Single Nuclei Sequencing Reveals Intratumoral Cellular Heterogeneity and Replication Stress in Adrenocortical Carcinoma
              </a>
            </td>
          <td>
            Liudmila V. Popova, Elizabeth A. R. Garfinkle, Daniel M. Chopyk, Jaye B. Navarro, Adithe Rivaldi, Yaoling Shu, Elena Lomonosova, J. Phay, Barbra S. Miller, Swati Sattuwar, Mary Mullen, Elaine R. Mardis, Katherine E. Miller, Priya H. Dedhia
          </td>
          <td>2024-10-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f3792b41778f947365ed0dac38ff748d4c49600" target='_blank'>
              L3MBTL1, a polycomb protein, promotes Osimertinib acquired resistance through epigenetic regulation of DNA damage response in lung adenocarcinoma
              </a>
            </td>
          <td>
            Zihe Zhang, Yongwen Li, Ruifeng Shi, Chaoyi Jia, Songlin Xu, Guangsheng Zhu, P. Cao, Hua Huang, Xuanguang Li, Hongbing Zhang, Minghui Liu, Chen Chen, Hongyu Liu, Chunsheng Kang, Jun Chen
          </td>
          <td>2024-09-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Mutations commonly found in AML such as DNMT3A, TET2 and ASXL1 can be found in the peripheral blood of otherwise healthy adults a phenomenon referred to as clonal hematopoiesis (CH). These mutations are thought to represent the earliest genetic events in the evolution of AML. Genomic studies on samples acquired at diagnosis, remission, and at relapse have demonstrated significant stability of CH mutations following induction chemotherapy. Meanwhile, later mutations in genes such as NPM1 and FLT3, have been shown to contract at remission and in the case of FLT3 often are absent at relapse. We sought to understand how early CH mutations influence subsequent evolutionary trajectories throughout remission and relapse in response to induction chemotherapy. Here, we assembled a retrospective cohort of patients diagnosed with de novo AML at our institution that underwent genomic sequencing at diagnosis as well as at the time of remission and/or relapse (total n = 182 patients). Corroborating prior studies, FLT3 and NPM1 mutations were generally eliminated at the time of cytologic complete remission but subsequently reemerged upon relapse, whereas DNMT3A, TET2 and ASXL1 mutations often persisted through remission. Early CH-related mutations exhibited distinct constellations of co-occurring genetic alterations, with NPM1 and FLT3 mutations enriched in DNMT3Amut AML, while CBL and SRSF2 mutations were enriched in TET2mut and ASXL1mut AML, respectively. In the case of NPM1 and FLT3 mutations, these differences vanished at the time of complete remission yet readily reemerged upon relapse, indicating the reproducible nature of these genetic interactions. Thus, early CH-associated mutations that precede malignant transformation subsequently shape the evolutionary trajectories of AML through diagnosis, therapy, and relapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45f35657f001405231a2b4d996ed0ac3f0415703" target='_blank'>
              Early drivers of clonal hematopoiesis shape the evolutionary trajectories of de novo acute myeloid leukemia
              </a>
            </td>
          <td>
            Ryan D. Chow, Priya Velu, Safoora Deihimi, Jonathan Belman, Angela Youn, Nisargbhai Shah, Selina M. Luger, Martin P. Carroll, Jennifer J D Morrissette, Robert L Bowman
          </td>
          <td>2024-09-01</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Biological processes are fundamentally dynamic, yet existing methods for capturing these temporal changes are limited. We present scDynaBar, a novel approach that integrates CRISPR-Cas9 dynamic barcoding with single-cell sequencing to enable the recording of temporal cellular events. In this system, genetic barcodes accumulate mutations over a 4-week timeframe and then are sequenced together with the transcriptome of each single cell. We propose that this gradual accumulation of genetic diversity can be exploited to create a time-ordered record of cellular events. To demonstrate this, we apply the system to track the transition from a pluripotent state to a two-cell (2C)–like state in mouse embryonic stem cells (mESCs). The results provide compelling evidence for the transient nature of the 2C-like state. Additionally, our system shows consistent mutation rates across diverse cell types in a mouse gastruloid model, underscoring its robustness and versatility across various biological contexts. This technique not only improves our ability to study cellular dynamics but also creates exciting new opportunities for future applications based on recording temporal signals at the single-cell level—in other words, using dynamic barcoding as a molecular clock.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d0a4bc0e689a5dd87c70b2f28e78dbbf5756625" target='_blank'>
              Using CRISPR barcoding as a molecular clock to capture dynamic processes at single-cell resolution
              </a>
            </td>
          <td>
            Yolanda Andres-Lopez, Alice Santambrogio, Ioannis Kafetzopoulos, Christopher D Todd, Celia Alda-Catalinas, Stephen J. Clark, Wolf Reik, I. Hernando-Herraez
          </td>
          <td>2024-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="
 Ewing sarcoma is a highly aggressive pediatric cancer driven by the EWS::FLI1 (EF) fusion oncogene. Evidence suggests that EF expression levels dictate tumor cell plasticity contributing to treatment resistance and relapse. Our study aims to elucidate the transcriptional programs and phenotypes associated with distinct EF thresholds. To dynamically fine-tune EF levels, we engineered Ewing sarcoma cell line models by C-terminally tagging endogenous EF with a fluorescent (mNG) and a degron tag (dTAG). Using mNG as a quantitative proxy for EF levels, we confirmed that increasing concentrations of the dTAG ligand led to a gradual loss of the fusion protein. Already subtle modulation of EF thresholds significantly enhanced tumor cell migration. Furthermore, a 25% EF reduction was sufficient to significantly lower anchorage-independent growth in soft agar, which was completely abolished with further EF depletion. Supported by zebrafish xenograft experiments, these findings suggest that minor EF fluctuations enhance migratory properties while still retaining significant in vitro tumor-forming potential. To characterize acute transcriptional responses to variations in EF thresholds, we performed SLAM-sequencing after 3 hours of treatment with increasing dTAG concentrations. While minor reduction of the fusion protein to 76% led to immediate downregulation of EF-activated genes enriched in GGAA microsatellites, most EF-repressed targets were upregulated only when EF levels were reduced by 58% to 96%. These de-repressed targets were significantly enriched for epithelial-to-mesenchymal transition (EMT)-related pathways. Testing EF chromatin occupancy by CUT&RUN followed by motif enrichment analysis, identified several transcription factor binding motifs linked to EF-repressed genes. Promising candidates include SP1, EGR1, and TFAP2A, all of which have previously been implicated in the regulation of metastasis-related pathways. Long-term dTAG treatment (1 to 3 weeks), followed by EF rescue through dTAG ligand washout for 7 days revealed a set of metastasis-related genes that notably remained upregulated despite a full recovery of EF levels already within 24 hours of ligand washout. This suggests that transient oncogene fluctuations may cause a sustained increase in metastatic potential of Ewing sarcoma cells. Importantly, about thirty of these genes negatively correlated with overall survival in a cohort of 196 Ewing sarcoma patients and were anti-correlated with the GGAA microsatellite-driven transcriptional repressor BCL11B, which was highly sensitive to minor changes in EF thresholds. Collectively, our results highlight the critical role of EF oncogene fluctuations in Ewing sarcoma cell plasticity and patient prognosis providing new insights into the gene regulatory mechanisms involved. Further, our findings have important implications for therapeutic approaches aiming at directly targeting EF. Understanding EF dynamics in Ewing sarcoma serves a paradigmatic approach to the identification of therapeutic vulnerabilities in fusion oncogene-driven cancers.
 Citation Format: Veveeyan Suresh, Christoph Hafemeister, Sarah Grissenberger, Caterina Sturtzel, Florencia Cidre Aranaz, Thomas Grünewald, Ana Sastre, Javier Alonso, Martin Distel, Florian Halbritter, Heinrich Kovar, Valerie Fock. Dynamic modelling of EWS::FLI1 fluctuations reveals molecular determinants of phenotypic tumor plasticity and prognosis in Ewing sarcoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A017.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/533254be9587cdde4641f2edde320cdf2280f903" target='_blank'>
              Abstract A017 Dynamic modelling of EWS::FLI1 fluctuations reveals molecular determinants of phenotypic tumor plasticity and prognosis in Ewing sarcoma
              </a>
            </td>
          <td>
            Veveeyan Suresh, Christoph Hafemeister, S. Grissenberger, C. Sturtzel, F. C. Aranaz, Thomas Grünewald, A. Sastre, Javier Alonso, Martin Distel, F. Halbritter, Heinrich Kovar, Valerie Fock
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Radiotherapy remains a cornerstone in cancer treatment, leveraging ionizing radiation to eradicate malignant cells. Its efficacy, however, is frequently challenged by the heterogeneous sensitivity of tumors and surrounding tissues to radiation. Therefore, understanding the molecular mechanisms underlying radiosensitivity is crucial for improving treatment outcomes. Among the myriad of molecular players involved, the tumor suppressor protein p53 stands out as a central regulator with significant implications for radiosensitivity. Known as the “guardian of the genome”, p53 plays a pivotal role in maintaining genomic stability and orchestrating cellular responses such as cell cycle arrest, DNA repair, apoptosis, and senescence in response to various stress signals, including radiation-induced DNA damage. Activation of p53 triggers the transcription of target genes involved in DNA repair pathways, such as p21, MDM2, and GADD45, facilitating the repair of radiation-induced DNA damage or the elimination of irreparably damaged cells. This, in turn, influences the overall radiosensitivity of tissues. Mutations in the TP53 gene, which encodes p53, are among the most frequent genetic alterations in human cancers. Loss or dysfunction of p53 can compromise the cellular response to radiation, leading to increased resistance to therapy and poorer clinical outcomes. Conversely, intact p53 function is associated with enhanced radiosensitivity due to its ability to promote cell cycle arrest and apoptosis in response to radiation-induced DNA damage. In conclusion, elucidating the molecular mechanisms by which p53 influences radiosensitivity is essential for advancing our understanding of the radiation response in cancer cells and developing more effective therapeutic approaches to cancer treatment. This review provides a comprehensive overview of the multifaceted role of p53 in modulating cellular responses to radiation, emphasizing its influence on radiosensitivity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edeb6b0914ac30942be5b714c5e52dd1ba564b4d" target='_blank'>
              Exploring the Role of p53 in Radiosensitivity: A Key Player in Cancer Therapy
              </a>
            </td>
          <td>
            Tusher- Al-Arafat, Aihong Mao, T. Katsube, Bing Wang
          </td>
          <td>2024-10-24</td>
          <td>Radiation</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Richter's transformation (RT) is defined as the evolution of chronic lymphocytic leukemia (CLL) or small lymphocytic lymphoma (SLL) into an aggressive lymphoma, most commonly diffuse large B-cell lymphoma. This complication is rare and aggressive, with poor prognosis and dismal survival. Clonal relationship with the underlying CLL/SLL, observed in ∼80% of cases, represents one of the main factors affecting prognosis. Treatment has been historically based on chemoimmunotherapy, but frequent mutations in genes involved in cell survival and proliferation-such as TP53, NOTCH1, MYC, CDKN2A-confer resistance to standard treatments. During the last years, advances in the knowledge of the biological mechanisms underlying RT allowed to identify genetic and molecular lesions that can potentially be targeted by novel selective agents. Pathway and checkpoint inhibitors, bispecific antibodies and CAR T-cell therapy are currently under investigation and represent promising treatment options. This review summarizes current biological evidence and available data on novel therapeutic agents.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a1bac15f3b796b2ef346e32ef1ee6b7161f4b12" target='_blank'>
              Advances in the understanding of molecular genetics and therapy of Richter transformation in chronic lymphocytic leukemia.
              </a>
            </td>
          <td>
            M. Deodato, A. Frustaci, Arianna Zappaterra, Alberto Rapella, C. Gambacorti-Passerini, R. Cairoli, Marco Montillo, Alessandra Tedeschi
          </td>
          <td>2024-09-02</td>
          <td>Leukemia & lymphoma</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30d35e0bab7dec79077bc3332fba4617308d5113" target='_blank'>
              Comprehensive evaluation of immunological attributes and immunotherapy responses of positive T cell function regulators in colorectal cancer
              </a>
            </td>
          <td>
            Ke Pu, Jingyuan Gao, Yang Feng, Jian Hu, Shunli Tang, Guodong Yang, Chuan Xu
          </td>
          <td>2024-10-01</td>
          <td>BMC Gastroenterology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="High copy number endometrial cancers (HCNEC) are dominated by excessive duplications scattered across the genome, termed here as the HyperDuplication GenomoPhenotype (HDGP). Although correlated with cancer progression, its biological significance and implications for therapy have not yet been established. We identified locations and sizes of duplications in 171 endometrial cancer cases and designated 71 HCNEC cases as HDGP. We also investigated the response to the pan-ERBB inhibitor afatinib in a subset of HDGP-EC cases with ERBB2/ERBB3 duplications using a patient-derived three-dimensional culture model. Our analysis demonstrates that beyond tandem duplications there is a more general pattern involving coordinated duplication of multiple distant regions of the genome, demonstrating preferential selectivity to over-expressed potential oncogenes within a broad network. This suggests that HDGP increases tumor fitness and resistance to therapy by perturbing important gene networks in concert rather than only driver genes, suggesting a mechanistic basis for the ineffectiveness of targeted drugs in these patients and highlighting the need for combination therapies in these highly aggressive cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4e4f25ff2ac4650bfe407694d21fc4d8e5d3f46" target='_blank'>
              Functional impact of the hyperduplication genomophenotype in high copy number endometrial cancer
              </a>
            </td>
          <td>
            Angela Florio, J. Smadbeck, Sarah H. Johnson, W. Lin, Dorsay Sadeghian, Sotiris Sotiriou, Rebeca Salvatori, R. Feathers, Taylor Berry, Lindsey Kinsella, F. Harris, A. Mccune, Stephen J. Murphy, Mohamed F. Ali, Abdulmohammad Pezeshki, Michael T. Barrett, Leah O. Grcevich, Ilaria Capasso, L. D. De Vitis, G. Schivardi, Tommaso Occhiali, A. Larish, John J. Weroha, M. Borad, John C. Cheville, P. Anastasiadis, Andrea Mariani, G. Vasmatzis
          </td>
          <td>2024-09-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Disruption of mechanisms that maintain genome stability is an essential factor of tumor progression. Accordingly, predisposition to the development of neoplasms is often associated with germline mutations in genes involved in DNA damage detection and repair. At the same time, impairment of DNA repair systems may be a predictor of antitumor treatment efficacy while overexpression of genes involved in DNA repair is a frequent event in various types of malignancies that can lead to development of tumor cells’ resistance to chemo- and radiotherapy. NBN (nibrin) gene encodes the subunit of the MRN complex which acts as a sensor of double-strand DNA breaks and participates in their repair by homologous recombination. Germline variants in NBN which are associated with increased risk of tumor development are generally represented by frameshift mutations that lead to the synthesis of truncated protein as well as by nonsense and some missense mutations which occur in functionally significant domains. These germline mutations result in partial loss of nibrin function and in increased frequency of spontaneous and induced chromosomal aberrations in the cells of the carriers. On the contrary, amplification of NBN locus is a predominant type of somatic mutations affecting this gene, which indicates a dual role of NBN protein in tumor progression. The results of several studies demonstrate the influence of NBN expression level and its mutational status on anti-tumor drug resistance in particular types of tumor cells and on the survival rate of patients. These data indicate that an in-depth study of different variants and their functional significance is necessary since NBN status may be essential for the choice of treatment tactics for some types of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c608441ac7c7406f3e5d539484e420368a9c4c9" target='_blank'>
              Germline and somatic alterations in NBN and their putative impact on the pathogenesis of malignant neoplasms
              </a>
            </td>
          <td>
            O. Krivtsova, D. D. Ozerova, N. L. Lazarevich
          </td>
          <td>2024-10-10</td>
          <td>Advances in Molecular Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="BACKGROUND AND AIMS
Lynch Syndrome, a hereditary disorder characterized by germline mutations in mismatch repair (MMR) genes, is a major contributor to colorectal cancers. It has also been identified in endometrial cancer. Despite the established role of MMR deficiency in tumorigenesis, the specific genomic alterations driving Lynch syndrome-associated endometrial cancer, and their overlap with colorectal cancer, remain incompletely understood. This study aims to fill this gap by performing a detailed comparative analysis of germline and somatic mutations in endometrial cancer within the context of Lynch syndrome.


METHODS
We conducted whole exome sequencing on matched germline and somatic DNA from 13 patients diagnosed with Lynch syndrome-associated endometrial cancer. High-depth sequencing was performed, followed by rigorous bioinformatics analysis to identify and annotate variants, focusing on their potential pathogenicity and relevance to both endometrial and colorectal cancer.


RESULTS
Our analysis revealed 1,118 germline and 14,051 somatic variants, with 493 variants common to both. Recurrent pathogenic mutations in MLH1, MSH2, and MSH6 were confirmed, highlighting their critical role in Lynch syndrome. Notably, frequent somatic mutations in the PIK3CA and PTEN genes were identified, implicating the PI3K/AKT/mTOR pathway as a key oncogenic driver in these cancers. Additionally, novel somatic mutations in genes related to the extracellular matrix such as FBN1 and SPARC were uncovered, suggesting a possible unique role in endometrial tumor progression.


CONCLUSIONS
This study provides new insights into the molecular basis of Lynch syndrome-associated endometrial cancer, emphasizing the overlap in oncogenic pathways with colorectal cancer. The discovery of shared and unique genetic mutations highlights the importance of developing combined treatment strategies and suggests that targeting these specific mutations could improve therapy for patients with Lynch syndrome-associated cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04b204016ea6a27e3ec168c78e8d560614b4b705" target='_blank'>
              Lynch Syndrome-associated Genomic Variants.
              </a>
            </td>
          <td>
            Robert Botea, Madalina Piron-Dumitrascu, Tiberiu-Augustin Georgescu, C. Bohiltea, S. Voinea, Valentin Nicolae Varlas, Simona Raluca Iacoban, Nicolae Suciu
          </td>
          <td>2024-09-09</td>
          <td>Journal of gastrointestinal and liver diseases : JGLD</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Recent breakthroughs in the genetic manipulation of mitochondrial DNA (mtDNA) have enabled the precise introduction of base substitutions and the effective removal of genomes carrying harmful mutations. However, the reconstitution of mtDNA deletions responsible for severe mitochondrial myopathies and age-related diseases has not yet been achieved in human cells. Here, we developed a method to engineer specific mtDNA deletions in human cells by co-expressing end-joining (EJ) machinery and targeted endonucleases. As a proof-of-concept, we used mito-EJ and mito-ScaI to generate a panel of clonal cell lines harboring a ∼3.5 kb mtDNA deletion with the full spectrum of heteroplasmy. Investigating these isogenic cells revealed a critical threshold of ∼75% deleted genomes, beyond which cells exhibited depletion of OXPHOS proteins, severe metabolic disruption, and impaired growth in galactose-containing media. Single-cell multiomic analysis revealed two distinct patterns of nuclear gene deregulation in response to mtDNA deletion accumulation; one triggered at the deletion threshold and another progressively responding to increasing heteroplasmy. In summary, the co-expression of mito-EJ and programable nucleases provides a powerful tool to model disease-associated mtDNA deletions in different cell types. Establishing a panel of cell lines with a large-scale deletion at varying levels of heteroplasmy is a valuable resource for understanding the impact of mtDNA deletions on diseases and guiding the development of potential therapeutic strategies. Graphical Abstract Highlights Combining prokaryotic end-joining with targeted endonucleases generates specific mtDNA deletions in human cells Engineering a panel of cell lines with a large-scale deletion that spans the full spectrum of heteroplasmy 75% heteroplasmy is the threshold that triggers mitochondrial and cellular dysfunction Two distinct nuclear transcriptional programs in response to mtDNA deletions: threshold-triggered and heteroplasmy-sensing">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c60438e8f46fecefc3ea7cd6e713fab4e42d857" target='_blank'>
              Engineering mtDNA Deletions by Reconstituting End-Joining in Human Mitochondria
              </a>
            </td>
          <td>
            Yinxu Fu, Max Land, Ruobing Cui, Tamar Kavlashvili, Minsoo Kim, Toby Lieber, Keunwoo Ryu, Emily DeBitetto, I. Masilionis, Rahul Saha, Meril Takizawa, Daphne Baker, Marco Tigano, Ed Reznik, Roshan Sharma, R. Chaligné, Craig B. Thompson, D. Pe’er, Agnel Sfeir
          </td>
          <td>2024-10-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Cervical cancer (CC), one of the most prevalent and detrimental gynaecologic cancers, evolves through genetic and epigenetic alterations resulting in the promotion of oncogenic activity and dysfunction of tumour-suppressing mechanisms. Despite medical advancement, the prognosis for advanced-stage patients remains extremely low due to high recurrence rates and resistance to existing treatments. Thereby, the search for potential prognostic biomarkers is heightened to unravel new modalities of CC pathogenesis and to develop novel anti-cancer therapies. Epitranscriptomic modifications, reversible epigenetic RNA modifications, regulate various biological processes by deciding RNA fate to mediating RNA interactions. This narrative review provides insight into the cellular and molecular roles of endogenous RNA-editing proteins and their associated epitranscriptomic modifications, especially N6-methyladenosine (m6A), 5-methylcytosine (m5C) and N1-methyladenosine (m1A), in governing the development, progression and metastasis of CC. We discussed the in-depth epitranscriptomic mechanisms underlying the regulation of over 50 RNAs responsible for tumorigenesis, proliferation, migration, invasion, survival, autophagy, stemness, epithelial-mesenchymal transition, metabolism (glucose, lipid, glutamate and glutamine), resistance (drug and radiation), angiogenesis and recurrence of CC. Additionally, we provided a concise overview of the therapeutic potential of targeting the altered expression of endogenous RNA-editing proteins and aberrant deposition of RNA modifications on both coding and non-coding RNAs in CC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d247af5a7e732d73b316957acc73d51f868974f4" target='_blank'>
              Epitranscriptomics and cervical cancer: the emerging role of m6A, m5C and m1A RNA modifications
              </a>
            </td>
          <td>
            Akshat D. Modi, Hira Zahid, Ashlyn Chase Southerland, Dharmeshkumar M Modi
          </td>
          <td>2024-10-08</td>
          <td>Expert Reviews in Molecular Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Serrated lesions are common precancerous pathways in colorectal cancer (CRC), but the process by which they progress to malignancy remains unclear. We aimed to elucidate this progression through a single-cell RNA landscape. We conducted single-cell RNA sequencing on three normal colonic tissues and fifteen SLs (including HPs, SSLs, SSLD, and TSAs) and integrated these data with datasets containing tumor samples. We identified three invasive malignant epithelial cell subtypes related to CRC progression: SLC1, SLC2, and tumor cell. SLC1, specific to SSLs, is involved in cell proliferation and shows a continuum of malignancy in gene expression. TSA-specific SLC2 exhibited FOXQ1 upregulation and active EMT, indicating invasiveness. The trajectory analysis showed that HPs do not progress to cancer, and different SL types are linked to the MSI status of advanced CRCs. We validated molecular drivers in premalignant lesions and later carcinogenesis. In the tumor microenvironment, CAF and pre-CAF fibroblast subtypes associated with progression were identified. During the premalignant stage, SLC1 triggered CD8+ T cell responses, while at the advanced stage, CAFs promoted tumor invasion and metastasis via FN1-CD44, influencing tumor progression and the treatment response. Our findings highlight transcriptional changes across serrated pathway stages, aiding in early CRC diagnosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2f52080cc10a19d5a436d2837e1d1e6040efb30" target='_blank'>
              Single-Cell Transcriptomics Reveals Cellular Heterogeneity and Drivers in Serrated Pathway-Driven Colorectal Cancer Progression
              </a>
            </td>
          <td>
            Jiahui Wang, Yu Zhang, Xinyi Chen, Qi Sheng, Junluo Yang, Yuyao Zhu, Yuhang Wang, Fangrong Yan, Jingya Fang
          </td>
          <td>2024-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b39abde50129bd51799285e63a0661a0178d6716" target='_blank'>
              DNA damage response in breast cancer and its significant role in guiding novel precise therapies
              </a>
            </td>
          <td>
            Jiayi Li, Ziqi Jia, Lin Dong, Heng Cao, Yansong Huang, Hengyi Xu, Zhixuan Xie, Yiwen Jiang, Xiang Wang, Jiaqi Liu
          </td>
          <td>2024-09-27</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Transgenerational epigenetic inheritance (TEI) mediated by transposable elements (TEs) is well established in plants. However, the molecular determinants and functional impact of this additional system of inheritance are poorly characterized. Through comprehensive analyses of severe DNA methylation loss over TEs in an experimental population and in >700 natural strains of Arabidopsis thaliana, we uncovered hundreds of TE epivariants in nature, most of which can be inherited independently of DNA sequence changes and thus are true epialleles. Furthermore, natural epivariants are enriched near genes and we uncovered a novel role for the Paf1 complex as a major facilitator of their prevalence. Conversely, related TE copies limit epiallelic variation, by producing the small RNAs that guide DNA methylation in trans. Crucially, natural epivariants mainly affect stress-responsive genes, making them unique targets for selection. Our findings therefore demonstrate that TEs are important mediators of transgenerational epigenetic variation in nature, with singular properties for plant adaptation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac6bd4e49538ff89fe2173d3326edef5a9f7c98a" target='_blank'>
              Transposable elements are prevalent vectors of transgenerational epigenetic inheritance in nature
              </a>
            </td>
          <td>
            P. Baduel, Louna De Oliveira, E. Caillieux, Grégoire Bohl-Viallefond, Mounia El Messaoudi, Ciana Xu, Matteo Barois, Vipin Singh, Alexis Sarazin, Martine Boccara, Elodie Gilbault, Antoine de France, L. Quadrana, O. Loudet, V. Colot
          </td>
          <td>2024-09-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd7b87d57c150bf5b9368e6def7dc5609b418052" target='_blank'>
              siRNA-based strategies to combat drug resistance in gastric cancer.
              </a>
            </td>
          <td>
            Abdulrahman Qais Khaleel, Mohammad Y Alshahrani, J. Rizaev, H. Malathi, Seema Devi, Atreyi Pramanik, Yasser Fakri Mustafa, Ahmed Hjazi, Ismoilova Muazzamxon, Beneen Husseen
          </td>
          <td>2024-10-21</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 SMARCB1 is a tumor suppressor gene and encodes a core subunit of the SWI-SNF chromatin remodeling complex. Loss of SMARCB1 leads to the development of highly aggressive pediatric malignancies, termed “SMARCB1-deficient cancers.” This includes malignant rhabdoid tumor, atypical teratoid rhabdoid tumor (ATRT), and renal medullary carcinoma (RMC). Survival rates in these patients remain low, and resistance to chemotherapeutic agents used to treat these cancers are common. One such chemotherapy is doxorubicin. Therefore, this project aims to reveal and understand the mechanisms of doxorubicin resistance in SMARCB1-deficient cancers. To address this question, we generated doxorubicin resistant (DR) cells from one ATRT and two RMC primary cell lines (BT16, PEDS005T, and RCRF1009T respectively) utilizing a dose escalation approach. This resulted in DR cell lines that are 10-fold to 30-fold resistant compared to the parental cell line. Upon generation of these stably resistant cell lines, we profiled their transcriptomes by RNA-sequencing and found an upregulation of the multidrug transporter ABCB1, also known as MDR1. To further probe these cell lines, we performed whole-genome sequencing (WGS) on two of our DR cell lines (BT16 and PEDS005T DR). Here, we found a chromosome 7 aneuploidy in BT16 DR cells, but no aneuploidy or mutations were seen in PEDS005T DR cells. ABCB1 is located on chromosome 7. We confirmed this aneuploidy via karyotyping. The mechanism of ABCB1 upregulation in the RMC DR cell lines remains unknown, suggesting multiple modalities leading to ABCB1 upregulation. We then overexpressed ABCB1 in two ATRT parental cell lines (BT16 and CHLA-06-ATRT) and one RMC cell line (PEDS005T) which led to enhanced short-term resistance. This contrasted to JMSU, a SMARCB1 wild-type bladder cancer cell line, where there was stable resistance. Together, this suggests that ABCB1 may be necessary but not sufficient to confer doxorubicin resistance in these SMARCB1-deficient cancer cells. These preliminary findings suggest that despite a common genetic driver (loss of SMARCB1), there may be different pathways to developing resistance to the DNA-damaging agent, doxorubicin. Furthermore, despite the well documented role of ABCB1 in drug resistance, there may be other factors involved in conferring stable resistance in the setting of SMARCB1-deficient cancers.
 Citation Format: Katie T. Skinner, Benjamin P. Lee, Jessica S Yoon, Andrew L. Hong. The role of ABCB1 in doxorubicin resistant SMARCB1-deficient cancers [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A069.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebdc2732137e2a0d6d27507359d94e24f1e66d2f" target='_blank'>
              Abstract A069 The role of ABCB1 in doxorubicin resistant SMARCB1-deficient cancers
              </a>
            </td>
          <td>
            Katie T. Skinner, Benjamin P Lee, Jessica S Yoon, Andrew L. Hong
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="TP53 is a tumor suppressor gene that is mutated in most cancer types and has been extensively studied in cancer research. p53 plays a critical role in regulating the expression of target genes and is involved in key processes such as apoptosis, cell cycle regulation, and genomic stability, earning it the title “guardian of the genome.” Numerous studies have demonstrated p53’s influence on and regulation of autophagy, ferroptosis, the tumor microenvironment, and cell metabolism, all of which contribute to tumor suppression. Alterations in p53, specifically mutant p53 (mutp53), not only impair its tumor-suppressing functions but also enhance oncogenic characteristics. Recent data indicate that mutp53 is strongly associated with poor prognosis and advanced cancers, making it an ideal target for the development of novel cancer therapies. This review summarizes the post-translational modifications of p53, the mechanisms of mutp53 accumulation, and its gain-of-function, based on previous findings. Additionally, this review discusses its impact on metabolic homeostasis, ferroptosis, genomic instability, the tumor microenvironment, and cancer stem cells, and highlights recent advancements in mutp53 research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ea389a27a85074a44af76e077628c890d00a2c8" target='_blank'>
              Roles of Mutant TP53 Gene in Cancer Development and Progression
              </a>
            </td>
          <td>
            Muhammad Abubakar, Baqaur Rehman
          </td>
          <td>2024-09-25</td>
          <td>Proceedings of Anticancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d9632a110490aa181f31d215bebe837e3899f41" target='_blank'>
              Inferring replication timing and proliferation dynamics from single-cell DNA sequencing data
              </a>
            </td>
          <td>
            A. Weiner, Marc J. Williams, Hongyu Shi, I. Vázquez-García, Sohrab Salehi, Nicole Rusk, Sam Aparicio, Sohrab P. Shah, Andrew McPherson
          </td>
          <td>2024-10-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e27208f944e21d4b5aca3b4a730bfecbb2ccacaf" target='_blank'>
              Single-cell RNA sequencing to map tumor heterogeneity in gastric carcinogenesis paving roads to individualized therapy
              </a>
            </td>
          <td>
            Jiao Xu, Bixin Yu, Fan Wang, Jin Yang
          </td>
          <td>2024-09-13</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Cancer cells are heterogeneous, each harboring distinct molecular aberrations and are dependent on different genes for their survival and proliferation. While successful targeted therapies have been developed based on driver DNA mutations, many patient tumors lack druggable mutations and have limited treatment options. Here, we hypothesize that new precision oncology targets may be identified through “expression-driven dependency”, whereby cancer cells with high expression of a targeted gene are more vulnerable to the knockout of that gene. We introduce a Bayesian approach, BEACON, to identify such targets by jointly analyzing global transcriptomic and proteomic profiles with genetic dependency data of cancer cell lines across 17 tissue lineages. BEACON identifies known druggable genes, e.g., BCL2, ERBB2, EGFR, ESR1, MYC, while revealing new targets confirmed by both mRNA- and protein-expression driven dependency. Notably, the identified genes show an overall 3.8-fold enrichment for approved drug targets and enrich for druggable oncology targets by 7 to 10-fold. We experimentally validate that the depletion of GRHL2, TP63, and PAX5 effectively reduce tumor cell growth and survival in their dependent cells. Overall, we present the catalog of express-driven dependency targets as a resource for identifying novel therapeutic targets in precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/328c4c5a157ff3ba7e44391b07b799fdfa4b7395" target='_blank'>
              Expression-Driven Genetic Dependency Reveals Targets for Precision Medicine
              </a>
            </td>
          <td>
            Abdulkadir Elmas, Hillary M. Layden, Jacob D. Ellis, Luke N. Bartlett, Xian Zhao, Reika Kawabata-Iwakawa, Hideru Obinata, Scott W. Hiebert, Kuan-lin Huang
          </td>
          <td>2024-10-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Introduction and Purpose: Melanoma, an aggressive malignancy from melanocytes, has a poor prognosis. Despite advances in targeted therapies and immunotherapies, drug resistance remains a challenge. This review examines the molecular pathways involved in melanoma and therapeutic strategies targeting them, aiming to improve patient outcomes and overcome treatment resistance by understanding genetic alterations and signaling cascades driving melanoma progression. 
State of Knowledge: Melanoma arises from genetic predisposition and UV radiation exposure, involving mutations in pathways like RAS/RAF/MAPK and PI3K/AKT. Targeted therapies, such as BRAF and MEK inhibitors, are effective, especially in BRAF-mutant patients. However, resistance mechanisms, including alternative pathway activation and immune evasion, limit their long-term success. Combination therapies with immunotherapies show promise in overcoming resistance. Emerging targets like NRAS, c-KIT, and c-MET offer new treatment possibilities. Ongoing challenges include identifying biomarkers for patient stratification and managing treatment-related toxicity. 
Conclusions: Understanding the molecular mechanisms of melanoma is crucial for advancing therapies. Targeted therapies and combination approaches have improved outcomes, especially in advanced cases. However, challenges like treatment resistance and toxicity require ongoing research. By exploring signaling pathways and new targets, treatment efficacy and durability can be enhanced. Continued research and clinical trials are essential to translate these advancements into practice, improving survival rates and quality of life for melanoma patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8820446b375d0816aa0b1e3bab05e5e88814d21" target='_blank'>
              Advances in Targeted Therapies and Combination Approaches for Melanoma: A Comprehensive Review
              </a>
            </td>
          <td>
            Magda Przestrzelska, Iga Ślesicka, Natalia Zozula, Zuzanna Tomczewska, Aleksandra Rykucka, Marcin Wąs, Aleksandra Latała, Justyna Kiełbasa, Agata Kowalczyk, Katarzyna Bil
          </td>
          <td>2024-10-03</td>
          <td>Quality in Sport</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e0914eac9368db3a26273ef88f18ab51249d37f" target='_blank'>
              Advances in targeting cancer epigenetics using CRISPR-dCas9 technology: A comprehensive review and future prospects.
              </a>
            </td>
          <td>
            Jeevitha Rajanathadurai, Elumalai Perumal, Jospin Sindya
          </td>
          <td>2024-09-18</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Nearly a quarter century ago, Hanahan and Weinberg conceived six unifying principles explaining how normal cells transform into malignant tumors. Their provisional set of biological capabilities acquired during tumor development  cancer hallmarks  would evolve to fourteen tenets as knowledge of cancer genomes, molecular mechanisms, and the tumor microenvironment expanded, most recently adding four emerging enabling characteristics: phenotypic plasticity, epigenetic reprogramming, polymorphic microbiomes, and senescent cells. AKT kinases are critical signaling molecules that regulate cellular physiology upon receptor tyrosine kinases and phosphatidylinositol 3-kinase activation. The complex branching of the AKT signaling network involves several critical downstream nodes that significantly magnify its functional impact, such that nearly every organ system and cell in the body may be affected by AKT activity. Conversely, tumor intrinsic dysregulation of AKT can have numerous adverse cellular and pathological ramifications, particularly in oncogenesis, as multiple tumor suppressors and oncogenic proteins regulate AKT signaling. Herein, we review the mounting evidence implicating the AKT pathway in the aggregate of currently recognized hallmarks of cancer underlying the complexities of human malignant diseases. The challenges, recent successes, and likely areas for exciting future advances in targeting this complex pathway are also discussed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfc996efb642ec81ce6e4555cd7fa9142e0c17d3" target='_blank'>
              AKT and the Hallmarks of Cancer.
              </a>
            </td>
          <td>
            Eleonora Sementino, Dalal Hassan, Alfonso Bellacosa, Joseph R Testa
          </td>
          <td>2024-10-22</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="CRISPR/Cas9 gene editing technology is a promising tool for correcting pathogenic variants for autologous cell therapies for Inborn Errors of Immunity (IEI). The present IEI correction strategies mainly focus on the knock-in of therapeutic cDNAs, or knockout of the disease-causing gene when feasible. These strategies address many single-gene defects but may disrupt gene expression and require significant optimization for each newly discovered IEI-causing gene, highlighting the need for complementary platforms that can precisely correct diverse pathogenic variants. Here, we present a safe and efficient T cell single nucleotide variant (SNV) correction pipeline based on homology-directed repair (HDR), suitable for diverse monogenic mutations. By using founder mutations of Deficiency of ADA2 (DADA2), Autoimmune polyendocrinopathy-candidiasis-ectodermal dystrophy (APECED) and Cartilage Hair Hypoplasia (CHH) as IEI models, we show that our pipeline can achieve up to 80% bi-allelic editing, with resultant functional correction of the disease phenotype in patient T cells. We do not find detectable pre-malignant off-target effects or karyotypic, transcriptomic or proteomic aberrations upon profiling patient T cells with GUIDE-seq, single cell RNA sequencing, PacBio based long-read whole genome sequencing, and high-throughput proteomics. This study demonstrates that HDR-based SNV editing is a safe and effective option for IEI T cell correction and that it could be developed to an autologous T cell therapy, as the presented protocol is scalable for a GMP-compatible workflow. This study is a step towards the development of gene correction platform that targets a broad number of monogenic mutations. GRAPHICAL ABSTRACT">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3514ea764ac815dfdaa1d9a93597841e9c3e8944" target='_blank'>
              T cell correction pipeline for Inborn Errors of Immunity
              </a>
            </td>
          <td>
            Katariina Mamia, Sólrún Kolbeinsdóttir, Zhuokun Li, Kornel Labun, Anna Komisarczuk, S. Keskitalo, Ganna Reint, Frida Høsøien Haugen, Britt Olaug Lindestad, T. Gjerdingen, Antti Tuhkala, Carolina Wieczorek Ervik, Pavel Kopcil, Nail Fatkhutdinov, Monika Szymańska, Eero Tölö, V. Glumoff, J. Saarela, T. Michelsen, Camilla Schalin-Jäntti, Johanna Olweus, Eira Leinonen, M. Varjosalo, Eivind Valen, Timo Hautala, Martin Enge, T. Martelius, Shiva Dahal-Koirala, E. Haapaniemi
          </td>
          <td>2024-09-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d0fe672a884b6c86c84e083c9b2ab93ac24f65f" target='_blank'>
              Multiplex, single-cell CRISPRa screening for cell type specific regulatory elements
              </a>
            </td>
          <td>
            Florence M. Chardon, Troy A. McDiarmid, Nicholas F. Page, R. Daza, Beth K. Martin, Silvia Domcke, Samuel Regalado, Jean-Benoît Lalanne, Diego Calderon, Xiaoyi Li, L. Starita, Stephan J. Sanders, N. Ahituv, J. Shendure
          </td>
          <td>2024-09-18</td>
          <td>Nature Communications</td>
          <td>2</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8944ad87f93a33edd3ce6c09cbae53782b87ab0f" target='_blank'>
              Single-cell transcriptomics of pediatric Burkitt lymphoma reveals intra-tumor heterogeneity and markers of therapy resistance.
              </a>
            </td>
          <td>
            Clarissa Corinaldesi, A. Holmes, Gaia Martire, A. Tosato, Domenico Rizzato, F. Lovisa, I. Gallingani, Qiong Shen, Lavinia Ferrone, Marian Harris, Kimberly Davies, Luca Molinaro, Umberto Mortara, Angelo Paolo dei Tos, Kenneth Ofori, E. d'Amore, Roberto Chiarle, Bo Ngan, E. Carraro, Marta Pillon, Shafinaz Hussein, Govind Bhagat, Marco Pizzi, L. Mussolin, Katia Basso
          </td>
          <td>2024-10-18</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Introduction: The worldwide challenge presented by human immunodeficiency virus type 1, affecting 39 million individuals, endures despite the highly active antiretroviral therapy. Limitations of it, including insufficiently addressing viral reservoirs leading to prolonged drug toxicity, resistance concerns, and economic burdens, have fueled exploration into alternative approaches. Gene editing and gene repression techniques have emerged as potential solutions. Objective: Conduct a comprehensive analysis of recent research following Preferred Reporting Items for Systematic Reviews and Meta-Analyses guidelines, focusing on genetic modification and gene repression in HIV-1-infected samples. Methods: A meticulous search identified 413 articles, of which 76 met strict criteria. Inclusion criteria involved studies on gene editing and HIV-1 expression (2019–2023), emphasizing CCR5 repression. Exclusions targeted weak evidence, lacking practical efficacy, and non-relevance to HIV-1 or genetic tools. Duplicates and non-open-access articles were also excluded. Results: The identified strategies show promise in their ability to mitigate the chronicity of the infection. However, challenges such as off-target effects, ethical considerations surrounding in vivo, ex vivo, and germline editing, and variable efficacy across studies are evident. The imperative for continued evaluation, refinement of techniques, and the development of effective administration methods is underscored. Conclusions: This topic review provides a nuanced exploration of recent advancements in gene editing and gene repression techniques for managing HIV-1. While the findings suggest promising avenues, the identified challenges necessitate ongoing research and innovation in order to harness the full potential of these strategies in the quest for an effective HIV-1 solution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53fae1f8a23cc0b10ee128f34eb4680a5249d869" target='_blank'>
              Novel Approaches in HIV-1 Cure: Genetic Editing and Gene Repression Techniques. Topic Review
              </a>
            </td>
          <td>
            Nicolas Alejandro Castilla Salcedo
          </td>
          <td>2024-10-23</td>
          <td>Salud &amp; Sociedad Uptc</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Precise transcriptional regulation is critical for cellular function and development, yet the mechanism of this process remains poorly understood for many genes. To gain a deeper understanding of the regulation of neuropsychiatric disease risk genes, we identified a total of 39 functional enhancers for four dosage-sensitive genes, APP, FMR1, MECP2, and SIN3A, using CRISPR tiling deletion screening in human induced pluripotent stem cell (iPSC)-induced excitatory neurons. We found that enhancer annotation provides potential pathological insights into disease-associated copy number variants. More importantly, we discovered that allelic enhancer deletions at SIN3A could be compensated by increased transcriptional activities from the other intact allele. Such allelic compensation effects (ACE) on transcription is stably maintained during differentiation and, once established, cannot be reversed by ectopic SIN3A expression. Further, ACE at SIN3A occurs through dosage sensing by the promoter. Together, our findings unravel a regulatory compensation mechanism that ensures stable and precise transcriptional output for SIN3A, and potentially other dosage-sensitive genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68c9cf4cae8ed81690d40ccc0776865b484f3141" target='_blank'>
              CRISPR tiling deletion screens reveal functional enhancers of neuropsychiatric risk genes and allelic compensation effects (ACE) on transcription
              </a>
            </td>
          <td>
            Xingjie Ren, Lina Zheng, Lenka Maliskova, Tsz Wai Tam, Yifan Sun, Hongjiang Liu, Jerry Lee, M. Takagi, Bin Li, Bing Ren, Wei Wang, Yin Shen
          </td>
          <td>2024-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Aim: Rs16851030, a single-nucleotide variant located in the 3'-untranslated region of the ADORA1 gene, has been proposed as a potential marker of caffeine sensitivity in apnea of prematurity. Besides, it is associated with aspirin-induced asthma and the development of acute chest syndrome. However, its functional significance is still unconfirmed. This study aimed to elucidate the functional impact of rs16851030 by using CRISPR/Cas9 approach to induce the DNA variant and attendant physiological changes.Methods: Rs16851030 was introduced into HEK293 cells via homology-directed repair (HDR). Edited cells were fluorescence-enriched, sorted, isolated, and expanded into single-cell-derived clones. The edit was confirmed by Sanger sequencing. RNA sequencing was used to analyze affected pathways.Results: Rs16851030-mutant cells showed increased susceptibility to hypoxia, a condition related to apnea of prematurity. After 24 h of hypoxia, the viability of mutant clones 1 and 2 was low compared with wild-type cells (75.45% and 74.47% vs. 96.34%). RNA sequencing revealed transcriptomic changes linked to this increased vulnerability.Conclusion: Rs16851030 impairs cellular resistance to hypoxia, suggesting its role in conditions like apnea of prematurity. Further research should investigate the molecular mechanisms and transcriptomic alterations caused by rs16851030 under hypoxic conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc99421821658dad11aad291be8443d4a7d31fb7" target='_blank'>
              Introduction of a single-nucleotide variant, rs16851030, into the ADORA1 gene increased cellular susceptibility to hypoxia.
              </a>
            </td>
          <td>
            Poh Kuan Wong, S. Syafruddin, F. Cheah, Norazrina Azmi, Pei Yuen Ng, E. Chua
          </td>
          <td>2024-10-23</td>
          <td>Personalized medicine</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Female cells randomly inactivate one X-chromosome, resulting in cellular mosaicism. Determining the bias in X-chromosomal inactivation (XCI) at single cell level may be relevant for understanding diseases prevalent in females. Here, we introduce a computational method that determines XCI profiles at single-cell level using solely sc/snRNA-Seq data. XCI analysis of skin cells from hybrid mice validates our approach and reveals biased inactivation of X-chromosomes among cell types. In human lung and brain cells, XCI status can be determined in 33.8% and 23.6% of cells. Among the patients, cells with opposite inactivation patterns differently express members of specific gene families and pathways. Alzheimer’s disease patients show reversal of XCI in cortical microglia and regional increase in biallelic expression denoting epigenetic erosion. We provide a robust utility to explore the degree and impact of XCI in single cell expression data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8505b493ed60aed63b31c5d60d74abdddba75957" target='_blank'>
              Single-cell X-chromosome inactivation analysis links biased chimerism to differential gene expression and epigenetic erosion
              </a>
            </td>
          <td>
            Rob H Henning, Thomas M. Rust, Kasper Dijksterhuis, B. Eggen, Victor Guryev
          </td>
          <td>2024-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Patients with microsatellite instability-high (MSI-H) colorectal cancer (CRC) have high tumor mutation burden and tumor immunogenicity, exhibiting a higher response rate to immunotherapy and better survival. However, a portion of MSI-H CRC patients still experience adverse disease outcomes. We aimed to identify the tumor-autonomous regulators determining these heterogeneous clinical outcomes. The Cancer Genome Atlas (TCGA) dataset was used to identify regulators in MSI-H CRC patients with unfavorable outcomes. Stable CRC tumor clones expressing targeted regulators were established to evaluate migratory and stemness properties, immune cell vulnerability, and cell-in-cell (CIC) structure formation. RNA-sequencing (RNA-seq) was used to identify enriched biological pathways in stable CRC tumor clones. Clinicopathological characterization of formalin-fixed paraffin-embedded (FFPE) MSI-H CRC specimens was performed to explore the underlying mechanisms involved. We showed that cancer/testis antigen family 45 member A1 (CT45A1) expression was upregulated in MSI-H CRC patients with poor survival outcomes. CT45A1-expressing microsatellite stable (MSS) CRC cells showed enhanced migratory ability. However, CT45A1-expressing MSI-H CRC cells, but not MSS CRC cells, showed higher resistance to natural killer (NK) cell cytotoxicity and served as outer cells in homotypic CIC structures, preventing exogenous or therapeutic antibody access to inner CRC cells. Inactivating RHO-ROCK/MLCK-MLC2 signaling with small-molecule inhibitors or short-hairpin RNAs (shRNAs) targeting myosin light chain kinase (MYLK) abolished NK cell resistance and reduced the outer cell fate of CT45A1-expressing MSI-H CRC cells. In MSI-H CRC patients, CT45A1-positive tumors exhibited increased MLC2 phosphorylation, increased outer cell fate, and decreased survival. We demonstrated that CT45A1 potentiates the advanced progression of MSI-H CRC, and targeting MLC2 phosphorylation may enhance immunotherapy efficacy in CT45A1-positive MSI-H CRC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a08f29a07f3e7ccbababc06da74550a73f5f37c3" target='_blank'>
              CT45A1-mediated MLC2 (MYL9) phosphorylation promotes natural killer cell resistance and outer cell fate in a cell-in-cell structure, potentiating the progression of microsatellite instability-high colorectal cancer.
              </a>
            </td>
          <td>
            H. Teng, Hsiang-Yueh Huang, Chun-Chi Lin, Y. Twu, Wen-Hao Yang, Wen-Chun Lin, Hsin-Yi Lan, Yen-Yu Lin, Wei-Lun Hwang
          </td>
          <td>2024-09-25</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="In recent years, large-scale sequencing efforts have identified targetable driver mutations in haematopoietic stem cells. These efforts have led to the development and approval of nine novel agents for relapsed or refractory acute myelogenous leukaemia (R/R AML). However, despite an expansion in targeted therapies, achieving a durable remission in AML and high-risk myelodysplastic syndrome (HR-MDS) remains a significant challenge, and there is an urgent need for new effective treatments. Modulation of aberrant RNA splicing has emerged as a novel therapeutic approach in myeloid diseases. Aberrant splicing drives dysregulated gene expression that promotes tumourigenesis through increased proliferation and metastatic potential, immune evasion, decreased apoptosis, and chemotherapy resistance. Mutations in spliceosomal components have been identified in numerous cancer subtypes, with mutations in RNA binding proteins SF3B1, SRSF2, U2AF1, and ZRSR2 occurring frequently in AML and in up to 60% of patients with MDS, as well as in chronic myelomonocytic leukaemia and in 10% of patients with chronic lymphocytic leukaemia. In this review, we explore therapeutic strategies targeting aberrant splicing and the potential of these approaches to drive clinical responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17b179523c7f6d0d667d6bb250cabd47b56f2a64" target='_blank'>
              Therapeutic strategies targeting aberrant RNA splicing in myeloid malignancies.
              </a>
            </td>
          <td>
            Leora S Boussi, Jeetayu Biswas, Omar Abdel-Wahab, E. Stein
          </td>
          <td>2024-10-15</td>
          <td>British journal of haematology</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="The elevated level of replication stress is an intrinsic characteristic of cancer cells. Targeting the mechanisms that maintain genome stability to further increase replication stress and thus induce severe genome instability has become a promising approach for cancer treatment. Here, we identify histone deacetylase 8 (HDAC8) as a drug target whose inactivation synergizes with the inhibition of checkpoint kinases to elicit substantial replication stress and compromise genome integrity selectively in cancer cells. We showed that simultaneous inhibition of HDAC8 and checkpoint kinases led to extensive replication fork collapse, irreversible cell-cycle arrest, and synergistic vulnerability in various cancer cells. The efficacy of the combination treatment was further validated in patient tumor-derived organoid (PDO) and xenograft mouse (PDX) models, providing important insights into patient-specific drug responses. Our data revealed that HDAC8 activity was essential for reducing the acetylation level of structural maintenance of chromosomes protein 3 (SMC3) ahead of replication forks and preventing R loop formation. HDAC8 inactivation resulted in slowed fork progression and checkpoint kinase activation. Our findings indicate that HDAC8 guards the integrity of the replicating genome, and the cancer-specific synthetic lethality between HDAC8 and checkpoint kinases provides a promising replication stress-targeting strategy for treating a broad range of cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ad8e7b369dc59096bea669bbbbe3db9dbfc35f1" target='_blank'>
              Combined HDAC8 and checkpoint kinase inhibition induces tumor-selective synthetic lethality in preclinical models.
              </a>
            </td>
          <td>
            , Yan Yan, Zih-Yao Yu, Moeez Rathore, Nian-Zhe Lee, Hui-Ju Tseng, Li-Hsin Cheng, Wei-Jan Huang, Wei Zhang, Ernest R Chan, Yulan Qing, Ming-Lun Kang, Rui Wang, Kelvin K Tsai, J. Pink, William E Harte, Stanton L. Gerson, Sung-Bau Lee
          </td>
          <td>2024-10-22</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="
 Next generation sequencing (NGS) based molecular profiling has been proven to be robust and reliable in detecting most biological attributes in acute myeloid leukemia (AML). Targeted NGS testing on genomic DNA for SNVs/Indels and on RNA for oncogenic gene fusions are commonly performed in molecular diagnostics laboratories. With the initial clinical molecular diagnostic framework established in our institution, we used whole genome and whole transcriptome sequencing (WGS and WTS) to evaluate SNVs/Indels, structural variations (SVs) and copy number variations (CNVs) in pediatric and adolescent AMLs. In addition, gene expression was assessed using WTS data to evaluate the possible pathogenicity of variants identified through DNA sequencing. In the 154 AML cases analyzed, WGS revealed recurrent AML oncogenic fusions in 88, all of which were confirmed to be in-frame fusion transcripts by WTS. AML-defining SNVs/Indels and internal tandem duplications were detected by WGS in 45 cases with supporting evidence from WTS. WGS revealed potential enhancer hijacking fusions in 10 cases (MECOM-r in 6, HOXA-r in 2 and BCL11B::TLX3 in 2). Among them, increased expression of the oncogene of interest was verified by WTS in 8 cases (6 MECOM-r and 2 HOXA-r). In the remaining two cases (FAB classification of AML M0 and M1, respectively) for which WGS suggested an SV of BCL11B::TLX3, expression of TLX3 was barely detectable and expression of BCL11B was not increased. Furthermore, global gene expression profiling did not cluster these cases within any known AML molecular categories. Together, the gene expression data of the two cases was not consistent with an SV leading to aberrant TLX3 or BCL11B activation, resulting in the final classification of AML, NOS. Among the 7 cases that initially could not be molecularly classified based on sequence variants only, gene expression assessment helped elucidate the AML class-defining genetic driver in one case. While no apparently known genetic driver was detected by WGS and WTS analyses, an acquired heterozygous frameshift variant in the N-terminal transcription activation domain of CEBPA (35% variant allele frequency, VAF) was found. Notably, the same variant in CEBPA was observed in WTS at 97% VAF, indicating the exclusive expression of the mutant allele in the tumor. Furthermore, global gene expression profiling demonstrated the same characteristic expression profile as seen in CEBPA double mutants (CEBPA-dm). Together, the gene expression assessment obtained from WTS provided an essential tool to classify this case to the molecular subgroup of CEBPA-dm. In this study, gene expression obtained from WTS was used to complement WGS, providing gene/allele-specific expression and global gene expression profiling, to verify the possible pathogenicity of sequence variants identified through DNA sequencing. We demonstrated that molecular classification of AML can be further improved using a diagnostic framework of WGS and WTS as well as integrating gene expression evaluation to complement sequence variants analysis.
 Citation Format: Lu Wang, Rebecca Voss, Victor Pastor Loyola, Maria F. Cardenas, Jing Ma, Priya Kumar, Mark R. Wilkinson, David A. Wheeler, Jeffery M. Klco. Integrating gene expression evaluation in molecular diagnostics for pediatric AML molecular classification [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B036.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfe5a12ccf6a7fe096b01673af9fcab8d9760710" target='_blank'>
              Abstract B036: Integrating gene expression evaluation in molecular diagnostics for pediatric AML molecular classification
              </a>
            </td>
          <td>
            Lu Wang, Rebecca K Voss, V. P. Loyola, Maria F. Cardenas, Jing Ma, Priya Kumar, M. Wilkinson, David A. Wheeler, J. M. Klco
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD) poses significant challenges due to its complex biological characteristics and high recurrence rate. The high recurrence rate of LUAD is closely associated with cellular dormancy, which enhances resistance to chemotherapy and evasion of immune cell destruction. Using single-cell RNA sequencing (scRNA-seq) data from LUAD patients, we categorized the cells into two subclusters: dormant and active cells. Utilizing high-density Weighted Gene Co-expression Network Analysis (hdWGCNA) and pseudo-time cell trajectory, aberrant expression of genes involved in protein O-glycosylation was detected in dormant cells, suggesting a crucial role for O-glycosylation in maintaining the dormant state. Intercellular communication analysis highlighted the interaction between fibroblasts and dormant cells, where the Insulin-like Growth Factor (IGF) signaling pathway regulated by O-glycosylation was crucial. By employing Gene Set Variation Analysis (GSVA) and machine learning, a risk score model was developed using hub genes, which showed high accuracy in determining LUAD prognosis. The model also demonstrated robust performance on the training dataset and excellent predictive capability, providing a reliable basis for predicting patient clinical outcomes. The group with a higher risk score exhibited a propensity for adverse outcomes in the tumor microenvironment (TME) and tumor mutational burden (TMB). Additionally, the 50% inhibitory concentration (IC50) values for chemotherapy exhibited significant variations among the different risk groups. In vitro experiments demonstrated that EFNB2, PTTG1IP, and TNFRSF11A were upregulated in dormant tumor cells, which also contributed greatly to the diagnosis of LUAD. In conclusion, this study highlighted the crucial role of O-glycosylation in the dormancy state of LUAD tumors and developed a predictive model for the prognosis of LUAD patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bdea9ceef7f2c70dc3e70ad8f0a54632f63dceb" target='_blank'>
              Deciphering Dormant Cells of Lung Adenocarcinoma: Prognostic Insights from O-glycosylation-Related Tumor Dormancy Genes Using Machine Learning
              </a>
            </td>
          <td>
            Chenfei Dong, Yang Liu, Suli Chong, Jiayue Zeng, Ziming Bian, Xiaoming Chen, Sairong Fan
          </td>
          <td>2024-08-31</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Breast cancer (BC) is marked by significant genetic, morphological and clinical heterogeneity. To capture this heterogeneity and unravel the molecular mechanisms driving tumor progression and drug resistance, we established a comprehensive patient-derived xenograft (PDX) biobank, focusing particularly on luminal (estrogen receptor, ER+) and young premenopausal patients, for whom PDX models are currently scarce. Across all BC subtypes, our efforts resulted in an overall success rate of 17% (26 established PDX lines out of 151 total attempts), specifically 15% in luminal, 12% in human epidermal growth factor receptor 2 positive (HER2+) and 35% in triple negative BC. These PDX mirrored morphologic and genetic features of BC from which they originated, serving as a reliable tool to investigate drug resistance and test therapeutic strategies. We focused on understanding resistance to CDK4/6 inhibitors (CDK4/6i), which are crucial in the treatment of patients with advanced luminal BC. Treating a sensitive luminal BC PDX with the CDK4/6i palbociclib revealed that, despite initial tumor shrinkage, some tumors might eventually regrow under drug treatment. RNA sequencing, followed by gene set enrichment analyses, unveiled that these PDXs have become refractory to CDK4/6i, both at biological and molecular levels, displaying significant enrichment in proliferation pathways, such as MTORC1, E2F and MYC. Using organoids derived from these PDX (PDxO), we observed that acquisition of CDK4/6i resistance conferred cross-resistance to endocrine therapy and that targeting MTORC1 was a successful strategy to overcome CDK4/6i resistance. Considered together, these results indicate that our PDX models may serve as robust tools to elucidate the molecular basis of BC disease progression and, by providing the possibility to simultaneously test different therapies on the same tumor, to surmount treatment resistance. While this approach is of course not feasible in the clinic, its exploitation in PDX may expedite the identification and development of more successful therapies for patients with advanced luminal BC. © 2024 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e518aa62c8ccd88fccbc075513337a7ece3d380" target='_blank'>
              A comprehensive luminal breast cancer patient-derived xenografts (PDX) library to capture tumor heterogeneity and explore the mechanisms of resistance to CDK4/6 inhibitors.
              </a>
            </td>
          <td>
            I. Segatto, M. Mattevi, Gian Luca Rampioni Vinciguerra, Nicole Crestan, L. Musco, Andrea Favero, Alessandra Dall'Acqua, Gabriele Di Giustino, Giorgia Mungo, Sara D'Andrea, Chiara Gava, Federica Ruggiero, Matteo Dugo, Lorenzo Gerratana, Fabio Puglisi, S. Massarut, Riccardo Bomben, Maurizio Callari, Tiziana Perin, Gustavo Baldassarre, B. Belletti
          </td>
          <td>2024-10-25</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Abstract Background Building on our prior work that RNA alternative splicing modulates the druggability of kinase fusions, this study probes the clinical significance of sole reciprocal fusions. These rare genomic arrangements, despite lacking kinase domains at the DNA level, demonstrated potential RNA‐level druggability in sporadic cases from our prior research. Methods Utilizing the large‐scale multicenter approach, we performed RNA sequencing and clinical follow‐up to evaluate a broad spectrum of kinase fusions, including ALK, ROS1, RET, BRAF, NTRK, MET, NRG1, and EGFR, in 1943 patients. Results Our findings revealed 51 instances (2.57%) of sole reciprocal fusions, predominantly in lung (57%), colorectal (14%), and glioma (10%) cancers. Comparative analysis with an MSKCC cohort confirmed the prevalence in diverse cancer types and identified unique fusion partners and chromosomal locales. Cross‐validation through RNA‐NGS and FISH authenticated the existence of functional kinase domains in subsets including ALK, ROS1, RET, and BRAF, which correlated with positive clinical responses to targeted kinase inhibitors (KIs). Conversely, fusions involving EGFR, NRG1, and NTRK1/2/3 generated nonfunctional transcripts, suggesting the need for alternative therapeutic interventions. Conclusion This inaugural multicenter study introduces a novel algorithm for detecting and treating sole reciprocal fusions in advanced cancers, expanding the patient population potentially amenable to KIs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6cc03ed5b4cf3438881405d76d4a3c612f2c0230" target='_blank'>
              Clinical relevance and druggability of sole reciprocal kinase fusions: A large‐scale study
              </a>
            </td>
          <td>
            Jiao Feng, Tonghui Ma, Chunyang Wang, Baoming Wang, Qian Liu, Zhengchuang Liu, H. Tao, Zaiyuan Ye
          </td>
          <td>2024-09-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54604fdc43654c5fa63c4063c1f8c16948e89144" target='_blank'>
              DNA repair-dependent immunogenic liabilities in colorectal cancer: opportunities from errors.
              </a>
            </td>
          <td>
            V. Amodio, P. P. Vitiello, A. Bardelli, G. Germano
          </td>
          <td>2024-09-13</td>
          <td>British journal of cancer</td>
          <td>1</td>
          <td>8</td>
        </tr>

        <tr id="While immune checkpoint inhibitors have revolutionized cancer therapy, many patients exhibit poor outcomes. Here, we show immunotherapy responses in bladder and non–small cell lung cancers are effectively predicted by factoring tumor mutation burden (TMB) into burdens on specific protein assemblies. This approach identifies 13 protein assemblies for which the assembly-level mutation burden (AMB) predicts treatment outcomes, which can be combined to powerfully separate responders from nonresponders in multiple cohorts (e.g., 76% versus 37% bladder cancer 1-year survival). These results are corroborated by (i) engineered disruptions in the predictive assemblies, which modulate immunotherapy response in mice, and (ii) histochemistry showing that predicted responders have elevated inflammation. The 13 assemblies have diverse roles in DNA damage checkpoints, oxidative stress, or Janus kinase/signal transducers and activators of transcription signaling and include unexpected genes (e.g., PIK3CG and FOXP1) for which mutation affects treatment response. This study provides a roadmap for using tumor cell biology to factor mutational effects on immune response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc5f818fb592baa5f66559ad69838daf202c25a6" target='_blank'>
              Prediction of immunotherapy response using mutations to cancer protein assemblies
              </a>
            </td>
          <td>
            JungHo Kong, Xiaoyu Zhao, Akshat Singhal, Sungjoon Park, Robin Bachelder, Jeanne Shen, Haiyu Zhang, Jimin Moon, Changho Ahn, C. Ock, Hannah Carter, T. Ideker
          </td>
          <td>2024-09-20</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>80</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b108c069a149d86683e9a9aa9e66ae175bdebcb9" target='_blank'>
              MECOM Locus classical transcript isoforms affect tumor immune microenvironment and different targets in ovarian cancer
              </a>
            </td>
          <td>
            Ning Lan, Shuheng Bai, Min Chen, Xuan Wang, Zhaode Feng, Ying Gao, B. Hui, Wen Ma, Xiangxiang Zhang, Fengyuan Hu, Wanyi Liu, Wenyang Li, Fang Wu, Juan Ren
          </td>
          <td>2024-10-19</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="BACKGROUND The mitotic intra-S DNA damage checkpoint signaling gene set is potentially involved in cancers where the genes play an important role. 17 total genes are involved in this gene set: ATF2, CHEK2, EME1, EME2, FANCD2, HUS1, HUS1B, MDC1, MRE11, MSH2, MUS81, NEK11, RAD17, RAD9A, RAD9B, TIPIN, XPC. The aim of this study is to complete a pan-cancer profile of each gene in the mitotic intra-S DNA damage checkpoint signaling gene set in order to determine potential diagnostic and prognostic purposes, while also determining how they could be used in a clinical setting as therapeutic targets to help patients. METHODS Multiomic data was acquired for the 17 genes; over 9000 samples of 33 types of cancer were analyzed to create pan-cancer profiles of CNV, mRNA expression, and pathway analysis. RESULTS The CNVs of some of these genes are associated with the survival of MESO, PCPG, BLCA, SKCM, LUAD, HNSC, LUSC, OV, and BRCA could be affected by the mRNA expression of the genes which can involve regulation of copy number. CONCLUSION With sufficient investigation, the genes involved in mitotic intra-S DNA damage checkpoint signaling may contribute to the development of cancer and may be used as biomarkers for cancer prognosis and diagnosis. To prove their clinical use for diagnosis and prognosis, however, and to create workable applications in clinical settings, further work is required. However, these pan-cancer profiles provide a more comprehensive knowledge of the mitotic intra-S DNA damage checkpoint signaling gene set in cancer as well as valuable information for future reference.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f222ed09772900dec985568ac443adca2980e33" target='_blank'>
              Potential Cancer Biomarkers: Mitotic Intra-S DNA Damage Checkpoint Genes
              </a>
            </td>
          <td>
            Kashvi Agarwal, Hengrui Liu
          </td>
          <td>2024-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65abef66cfc3b531cbb5f7b34f625f1ee91844ae" target='_blank'>
              Unraveling the key role of chromatin structure in cancer development through epigenetic landscape characterization of oral cancer
              </a>
            </td>
          <td>
            Yue Xue, Lu Liu, Ye Zhang, Yueying He, Jingyao Wang, Zicheng Ma, Tie-jun Li, Jianyun Zhang, Yanyi Huang, Yi Qin Gao
          </td>
          <td>2024-09-06</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The fundamental transcription factor p53 regulates cellular processes and integrates signals of cellular stress, triggering a coordinated response to ensure survival of cells restored to healthy function and programmed death of those that couldn’t be repaired. Unsurprisingly, this is one of the most mutated genes in human cancers, with most changes occurring in the DNA-binding domain of the protein. In this work, we take a genome-wide approach and use available resources to identify high confidence p53-target genes, that we examine in three breast cancer cell lines with different p53 status, wild type (MCF-7) and different mutations in the DNA-binding domain (MDA-MB231, T47D). Comparison of p53-targets expression in response to DNA damage by RNAseq and cellular assays reveals that MDA-MB231 have a severely impaired p53-dependent pathway functionality while T47D are much less affected. MDA-MB231 are more resistant to DNA damage yet unable to repair and able to override cell cycle arrest leading to survival while T47D are sensitive only to high dose and exposure to genotoxic agents. This data shows the variability of effects of different p53 mutations and highlight the importance of understanding the mechanisms of p53 in the context of genotoxicity-based treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d6785633c891564b2ca32ca3538063162d5c2bd" target='_blank'>
              Differential gene expression in cells with different p53 mutations identifies genome-wide p53 targets and shows distinct modulation of cellular pathways in response to DNA damage
              </a>
            </td>
          <td>
            Patricia Eror Barnes, Maria Jose de la Concha, Kioko Mwikali, B. L. Ng, Hannes Ponstingl, Alena Pance
          </td>
          <td>2024-09-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Children with Down syndrome (DS) and B-cell acute lymphoblastic leukemia (B-ALL) are at increased risk for treatment-related mortality and relapse, highlighting the need for new therapies. Leukemia cell lines (CLs) have been fundamental to understanding therapeutic responses to pharmacological agents. We generated three DS B-ALL CLs characterized with diverse genomic alterations, including IGH::CRLF2 rearrangement (BCR::ABL1 like), mutations in FLT3 and TP53, and a novel ERG::CEBPD rearrangement. DS CLs had diminished proliferation, metabolism, and mitochondrial function when compared to non-DS (NDS) CLs and interestingly, these findings were similar to NDS Philadelphia chromosome-like (Ph-like) B-ALL CLs. Based on similar mitochondrial defects and prior preclinical data using Venetoclax for Ph-like B-ALL, we hypothesized that Venetoclax would be effective in DS. Intriguingly, Venetoclax was more effective in DS when compared to both NDS and Ph-like CLs. Efficacy was observed in DS patient derived xenografts (PDXs) and diagnostic/relapsed patient samples treated with Venetoclax, which synergized with Trametinib and Vincristine. Mass spectrometry-based multiomics analyses in DS and NDS B-ALL patient samples revealed an enriched metabolite profile in DS, particularly in the hubs of glucose metabolism and polyunsaturated phosphatidylcholines and phosphatidylinositols. Transcriptome analyses in DS B-ALL patients (n=249) supported enhanced glucose and fatty acid metabolism; along with a reliance on BCL2 family members for apoptosis resistance. Venetoclax synergized with inhibition of glucose metabolism in DS B-ALL CLs. In summary, we have generated novel tools for studying DS B-ALL and identify altered metabolism in DS that responds to Venetoclax.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04f404242cc6274ce2ef0f772a4cb26039fa167f" target='_blank'>
              Metabolic vulnerabilities in Down syndrome B-cell acute lymphoblastic leukemia can be targeted using Venetoclax
              </a>
            </td>
          <td>
            Dallas Jones, Elena Woods, Zhenhua Li, Kristin L. Schaller, Eric Hoffmeyer, Ben Kooiman, Joseph Fernandez, Tegan Wharton, Dejene M. Tufa, Spencer C. Hall, George Trahan, Kelly W. Maloney, Holly L. Pacenta, Kelly Sullivan, Joaquin Espinosa, James R. Roede, John O. Marentette, Brett M Stevens, Anagha Inguva, D. Stephenson, J. Reisz, Angelo D’Alessandro, Craig T. Jordan, Jun J Yang, Michael R Verneris
          </td>
          <td>2024-10-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Cancer is a complex disease in which several molecular and cellular pathways converge to foster the tumoral phenotype. Notably, in the latest iteration of the cancer hallmarks, "nonmutational epigenetic reprogramming" was newly added. However, epigenetics, much like genetics, is a broad scientific area that deserves further attention due to its multiple roles in cancer initiation, progression, and adaptive nature. Herein, we present a detailed examination of the epigenetic hallmarks affected in human cancer, elucidating the pathways and genes involved, and dissecting the disrupted landscapes for DNA methylation, histone modifications, and chromatin architecture that define the disease. Significance: Cancer is a disease characterized by constant evolution, spanning from its initial premalignant stages to the advanced invasive and disseminated stages. It is a pathology that is able to adapt and survive amidst hostile cellular microenvironments and diverse treatments implemented by medical professionals. The more fixed setup of the genetic structure cannot fully provide transformed cells with the tools to survive but the rapid and plastic nature of epigenetic changes is ready for the task. This review summarizes the epigenetic hallmarks that define the ecological success of cancer cells in our bodies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bed379ec7953480a8811a5c6ea46e05bc377efc5" target='_blank'>
              The Epigenetic Hallmarks of Cancer.
              </a>
            </td>
          <td>
            M. Esteller, M. Dawson, Cigall Kadoch, F. Rassool, P. Jones, Stephen B Baylin
          </td>
          <td>2024-10-04</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Background: Melanoma remains one of the most challenging cancers to treat effectively with drug resistant remaining a constant concern, primarily with activating BRAF mutations. Mutations in the BRAF gene appear in approximately 50% of patients, 90% of which are V600E. Two frontline BRAF inhibitors (BRAFi), vemurafenib and dabrafenib, are frequently used to treat unresectable or metastatic BRAF V600E melanoma. Initial response rates are high, but soon thereafter, 70-80% of patients develop resistance to treatment within a year. A major mechanism of resistance is the generation of a secondary Q61K mutation in the NRAS gene. Methods: We have developed an approach in which a CRISPR-Cas complex can be designed to distinguish between mutant genes enabling resistance to standard care in tumor cells and normal genomes of healthy cells. For the first time, we demonstrated the utility of two CRISPR-directed mutation-specific editing approaches to restore BRAFi sensitivity in BRAFV600E/NRASQ61K resistant A375 cells. Results: We utilize an AsCas12a protospacer adjacent motif site created by the NRAS Q61K mutation and the Q61K mutation in the critical seed region of an SaCas9 sgRNA for Q61K-selective targeting. We show here that both approaches allow for effective NRAS targeting of only mutated-Q61K and after CRISPR-directed Q61K-targeting, previously resistant A375 cells are re-sensitized to BRAFi treatment. Conclusion: Our data support the feasibility of the development of CRISPR-Cas therapeutic approaches to the treatment of melanoma. Successful therapeutic CRISPR-directed gene editing would enable both specific and efficient editing of a mutation-specific targeting approach eliminate concern for on- and off-target damage to the genomes of healthy cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0aea0b595d066144e478150c4bdebe3df9d61c5" target='_blank'>
              Mutation-Specific CRISPR Targeting with SaCas9 and AsCas12a Restores Therapeutic Sensitivity in Treatment-Resistant Melanoma.
              </a>
            </td>
          <td>
            Brett M. Sansbury, Sophia B. Masciarelli, Salma Kaouser, Olivia M. Tharp, Kelly Banas, Eric B Kmiec
          </td>
          <td>2024-10-10</td>
          <td>The CRISPR journal</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54ad6a53d090caa0cff0b86e7a380a1302888db4" target='_blank'>
              Circulating tumor cells: from new biological insights to clinical practice
              </a>
            </td>
          <td>
            Xuyu Gu, Shiyou Wei, Xin Lv
          </td>
          <td>2024-09-02</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/efa2963e25bac9e3ab08f5a5b7df7ab88d90b9cc" target='_blank'>
              Multiomic single cell sequencing identifies stemlike nature of mixed phenotype acute leukemia
              </a>
            </td>
          <td>
            C. Peretz, Vanessa E. Kennedy, Anushka Walia, C. Delley, Andrew Koh, Elaine Tran, Iain C Clark, Corey E. Hayford, Chris D’Amato, Yi Xue, Kristina Fontanez, Aaron A. May-Zhang, Trinity Smithers, Y. Agam, Qian Wang, Haixin Dai, Ritu Roy, Aaron C Logan, Alexander E Perl, A. Abate, Adam Olshen, Catherine C Smith
          </td>
          <td>2024-09-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6dd5db0ced4bc401a8414bfb61590751edc2a295" target='_blank'>
              DDX3X dynamics, glioblastoma's genetic landscape, therapeutic advances, and autophagic interplay.
              </a>
            </td>
          <td>
            Arpit Sharma, Shruti Raut, Alok Shukla, Shivani Gupta, Amit Singh, Abha Mishra
          </td>
          <td>2024-10-05</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Aneuploidy produces myriad consequences in health and disease, yet models of the deleterious effects of chromosome amplification are still widely debated. To distinguish the molecular determinants of aneuploidy stress, we measured the effects of duplicating individual genes in cells with different chromosome duplications, in wild-type cells (SSD1+) and cells sensitized to aneuploidy by deletion of RNA-binding protein Ssd1 (ssd1Δ). We identified gene duplications that are nearly neutral in wild-type euploid cells but significantly deleterious in euploids lacking SSD1 or in SSD1+ aneuploid cells with different chromosome duplications. Several of the most deleterious genes are linked to translation. In contrast, duplication of other genes benefits multiple ssd1Δ aneuploids over controls, and this group is enriched for translational effectors. Furthermore, both wild-type and especially ssd1Δ aneuploids with different chromosome amplifications show increased sensitivity to translational inhibitor nourseothricin. We used comparative modeling of aneuploid growth defects, based on the cumulative fitness costs measured for single-gene duplication. Our results present a model in which the deleterious effects of aneuploidy emerge from an interaction between the cumulative burden of many amplified genes on a chromosome and a subset of duplicated genes that become toxic in that context. These findings provide a perspective on the dual impact of individual genes and overall genomic burden, offering new avenues for understanding aneuploidy and its cellular consequences.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80b80bbc16cc0aeafb66a3a37123bf7c793ab7f3" target='_blank'>
              The response to single-gene duplication implicates translation as a key vulnerability in aneuploid yeast.
              </a>
            </td>
          <td>
            H. A. Dutcher, James Hose, Hollis Howe, Julie Rojas, A. Gasch
          </td>
          <td>2024-10-25</td>
          <td>PLoS genetics</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) remains an intractable cancer characterized by significant inter- and intra-patient heterogeneity. Recent research has established connections between key genetic drivers and distinct transcriptional profiles in GBM, yet the interactions between genetic changes and the transcriptional and spatial heterogeneity remain unclear. This study used genetically engineered mouse models, coupled with spatially resolved and single-cell RNA sequencing, to explore how specific genetic alterations influence the tumor microenvironment (TME) and cellular responses spatially.



 We used nestin-Tva C57BL/6 mice and generated de novo GBM using overexpression of oncogenes in combinations with short hairpin RNAs (shRNAs) or deletion to various tumor suppressors, including Tp53, Cdkn2A, and Pten. We modeled key driver mutations for GBM: PDGFB, NF1, and EGFR. We conducted single-cell RNA sequencing, combined with spatially resolved transcriptomics and proteomics. Data integration techniques, including horizontal (mutual nearest neighbors) and vertical (weighted nearest neighbors) approaches, were used and analyzed with the SPATA2 software. Advanced computational methods, such as graph neural networks and single-cell deconvolution, were applied to examine GBM genotypes and the spatial variability within tumors and their TMEs.



 Our findings revealed persistent cellular heterogeneity among tumor cells under various genetic conditions. However, specific genetic alterations influenced the cells toward dominant transcriptional states. NF1 and EGFR mutations resulted in shifts towards MES and AC-like states, respectively, unlike PDGFB, which promoted a NPC/OPC-like state. These transcriptional shifts were strongly associated with distinct microenvironmental changes, such as enhanced neuronal signaling in PDGFB-driven GBMs versus pronounced immune infiltration in NF1 and EGFR models. Utilizing weighted co-expression network analysis (WGCNA) and gene set enrichment analysis (GSEA) alongside spatial-niche deconvolution, we identified a mesenchymal transition leading to an immunosuppressive TME in NF1-driven GBMs and increased lymphocyte presence in EGFR-amplified GBMs, especially around blood vessels. Similar spatial patterns were observed in human spatially resolved transcriptomic data, highlighting the complexity and variability of genetic drivers in human GBMs.



 Our genetically engineered mouse models provide a valuable platform for investigating GBM with defined driver mutations. By leveraging sophisticated analytical techniques and computational modeling, this study underscores the potential to elucidate the interactions between genetic drivers and the tumor microenvironment. Our approach paves the way for developing targeted therapies and precision medicine strategies in GBM.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d0056b4191c0de2f537409a208d3f06f5659352" target='_blank'>
              OS07.7.A DISSECTING SPATIAL DIVERSITY OF GLIOBLASTOMA GENOTYPES
              </a>
            </td>
          <td>
            J. Zhang, T. Joshi, D. Hambardzumyan, D. Heiland
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Genetic intra-tumour heterogeneity (gITH) is a universal property of all cancers. It emerges from the interplay of cell division, mutation accumulation and selection with important implications for the evolution of treatment resistance. Theoretical and data-driven approaches extensively studied gITH in ageing somatic tissues or cancers at detection. Yet, the expected patterns of gITH during and after treatment are less well understood. Here, we use stochastic birth-death processes to investigate the expected patterns of gITH across different treatment scenarios. We consider homogeneous treatment response with shrinking, growing and stable disease, and follow up investigating heterogeneous treatment response with sensitive and resistant cell types. We derive analytic expressions for the site frequency spectrum, the total mutational burden and the single-cell mutational burden distribution that we validate with computer simulations. We find that the SFS after homogeneous treatment response retains its characteristic power-law tail, while emergent resistant clones cause peaks corresponding to their sizes. The frequency of the largest resistant clone is subdominant and independent of the population size at detection, whereas the relative total number of resistant cells increases with detection size. Furthermore, the growth dynamics under treatment determine whether the total mutational burden is dominated by preexisting or newly acquired mutations, suggesting different possible treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b068763b28793a776bfaf3fe0647aae0b724cad" target='_blank'>
              On the patterns of genetic intra-tumour heterogeneity before and after treatment
              </a>
            </td>
          <td>
            Alexander Stein, Benjamin Werner
          </td>
          <td>2024-10-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 At the Gene Editing Institute, we are advancing a novel strategy for the treatment of pancreatic ductile adenocarcinoma by employing CRISPR-directed gene editing to disable the functions of key genes that block effective drug application including chemotherapy, radiation and immunotherapy. No matter how careful one is at designing a specific drug against a specific protein, ultimately, the tumor cell develops a workaround. A paradigm shift is required to truly overcome this formidable barrier. We’ve accomplished this by leveraging the effectiveness of standard care and augmenting it, thereby reducing patient suffering, increasing treatment tolerance and, most importantly, improving outcomes. With an enthusiastic response for this concept from the FDA- INTERFACT meeting, we have demonstrated that a carefully selected CRISPR/Cas complex disables the NRF2 gene in the tumor cell and significantly reduces the dose of chemotherapy required to halt pancreatic tumor cell growth, and consequently promoting cell death. The efficiency of gene editing approaches 70 % routinely, which destroys the function of NRF2 and enables additional killing of residual tumor cells by chemotherapeutic agents. As a consequence of CRISPR/Cas activity, we effectively reduce the amount of chemotherapy needed to kill tumor cells by over 20-fold. This level of reduction should enable patients to complete their treatment regimens more successfully, at lower dosages and with far fewer adverse side effects. In addition to reduced suffering, this will invariably prolong survival as we are augmenting already proven treatments. This approach is agnostic to the haplotype of the tumor cell and does not rely on specific biomarker composition; radically expanding the patient population that would be suitable for this type of treatment. We will discuss the impact of CRISPR/Cas-gene editing to disable specific genes, including NRF2 and mutant KRAS, to enable current therapies to work at greater efficacy and at lower dosages.
 Citation Format: London P McGill, Kelly H Banas, Gregory Tiesi, Eric B Kmiec. A patient first approach using CRISPR-directed gene editing as an augmentative therapy for the treatment of pancreatic ductile adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A010.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/159a6092b196d29302afbdfaca37a6f4ece01dff" target='_blank'>
              Abstract A010: A patient first approach using CRISPR-directed gene editing as an augmentative therapy for the treatment of pancreatic ductile adenocarcinoma
              </a>
            </td>
          <td>
            L. McGill, Kelly Banas, Gregory Tiesi, Eric B Kmiec
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf716f2dfde6790f1926e7c27bbe62493919a95a" target='_blank'>
              Gene and transcript expression patterns, coupled with isoform switching and long non-coding RNA dynamics in adipose tissue, underlie the longevity of Ames dwarf mice.
              </a>
            </td>
          <td>
            Sebastian Cano-Besquet, Maiyon Park, Nadia Berkley, Michelle Wong, Sarah A Ashiqueali, Sarah Noureddine, A. Gesing, Augusto Schneider, Jeffrey Mason, Michal M Masternak, Joseph M Dhahbi
          </td>
          <td>2024-10-15</td>
          <td>GeroScience</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="
 Cell morphologies represent a phenotypic integration of intrinsic cellular processes, changes in cell state, and interactions with neighboring cells. While analyses of large cell line collections such as the Cancer Dependency Map (DepMap) have greatly enhanced discovery of novel molecular biomarkers and therapeutic targets, morphological analysis of these models has not yet been performed. Hence, there is a rich opportunity to discover novel regulators and functional properties of cell morphologies by linking image-based profiling to -omics based assays. Here, we studied the transcriptional and functional properties of 16 human pancreatic cancer cell lines, associations with distinct cell morphologies, and treatment-induced remodeling. At baseline, we observed extensive morphological heterogeneity across and within cell lines, as well as differences in multicellular organization. We categorized cell line morphologies into epithelial, mesenchymal, and spheroid, and organizational patterns into tightly aggregated, multilayered, and dispersed. We performed differential expression analysis across morphological subtypes utilizing DepMap transcriptomics data and queried functional correlates including CRISPR dependency, drug sensitivity, and proclivity for metastasis. For example, we discovered that the mesenchymal morphology and multilayered organizational pattern [WH3] were associated with metastatic potential, while expressing reduced levels of tight junction and cell adhesion genes. We then explored treatment associated morphological changes and their potential as cost-effective biomarkers for cell-intrinsic resistance to chemotherapy (e.g., 5-FU and gemcitabine) and KRAS inhibitors (e.g., MRTX1133 and RMC- 6236). Cells were treated continuously and monitored with live cell imaging (Incucyte SX5) followed by Cell Painting at the three-day endpoint. We observed conserved morphological responses to therapy across cell lines including a shift towards spindle-like cell shapes with neurite-like projections in chemotherapy-treated conditions, and a cell bloating phenomena characterized by an increase in cell area in KRAS inhibitor treated conditions. To directly link transcriptional state to morphology, we performed 1000-plex spatial molecular imaging (Nanostring CosMx) at subcellular resolution to pools of cell lines, demonstrating that morphological diversity within cell lines corresponded to distinct transcriptional states. In conclusion, this study highlights the potential of harnessing cell morphological information in a rapid, cost-effective phenotyping assay to aid precision oncology efforts leveraging patient- derived in vitro models.
 Citation Format: Dennis Gong, William L Hwang. Morphological diversity predicts functional traits in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B076.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ee8c48f6e344128f81cda79e9fefc7cbd605a4f" target='_blank'>
              Abstract B076: Morphological diversity predicts functional traits in pancreatic cancer
              </a>
            </td>
          <td>
            Dennis Gong, W. L. Hwang
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="In the fields of medicine and bioscience, gene editing is increasingly recognized as a promising therapeutic approach for treating pathogenic variants in humans and other living organisms. With advancements in technology and knowledge, it is now understood that most genetic defects are caused by single-base pair variants. The ability to substitute genes using genome editing tools enables scientists and doctors to cure genetic diseases and disorders. Starting with CRISPR (clustered regularly interspaced short palindromic repeats)/Cas, the technology has evolved to become more efficient and safer, leading to the development of base and prime editors. Furthermore, various approaches are used to treat genetic disorders such as hemophilia, cystic fibrosis, and Duchenne muscular dystrophy. As previously mentioned, most genetic defects leading to specific diseases are caused by single-base pair variants, which can occur at many locations in corresponding gene, potentially causing the same disease. This means that, even when using the same genome editing tool, results in terms of editing efficiency or treatment effectiveness may differ. Therefore, different approaches may need to be applied to different types of diseases. Prevalently, due to the safety of adeno-associated virus (AAV) vectors in gene therapy, most clinical trials of gene therapy are based on AAV delivery methods. However, despite their safety and nonintegration into the host genome, their limitations, such as confined capacity, dosage-dependent viral toxicity, and immunogenicity, necessitate the development of new approaches to enhance treatment effects. This review provides the structure and function of each CRISPR-based gene editing tool and focuses on introducing new approaches in gene therapy associated with improving treatment efficiency.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/175c7ca99490d5250dd5cadc163e90b91db26af6" target='_blank'>
              Mechanism of Genome Editing Tools and Their Application on Genetic Inheritance Disorders
              </a>
            </td>
          <td>
            Dae Hwan Oh
          </td>
          <td>2024-09-16</td>
          <td>Global Medical Genetics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Extrachromosomal, circular DNA (ecDNA) is a prevalent oncogenic alteration in cancer genomes, often associated with aggressive tumor behavior and poor patient outcome. While previous studies proposed a chromatin-based mobile enhancer model for ecDNA-driven oncogenesis, its precise mechanism and impact remains unclear across diverse cancer types. Our study, utilizing advanced multi-omics profiling, epigenetic editing, and imaging approaches in three cancer models, reveals that ecDNA hubs are an integrated part of nuclear condensates and exhibit cancer-type specific chromatin connectivity. Epigenetic silencing of the ecDNA-specific regulatory modules or chemically disrupting liquid-liquid phase separation breaks down ecDNA hubs, displaces MED1 co-activator binding, inhibits oncogenic transcription, and promotes cell death. These findings substantiate the trans-activator function of ecDNA and underscore a structural mechanism driving oncogenesis. This refined understanding expands our views of oncogene regulation and opens potential avenues for novel therapeutic strategies in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5554b124d7186535bee061a70cb36ab7c432ab43" target='_blank'>
              Extrachromosomal DNA Associates with Nuclear Condensates and Reorganizes Chromatin Structures to Enhance Oncogenic Transcription
              </a>
            </td>
          <td>
            Aziz Taghbalout, Chia-Hao Tung, P. A. Clow, Ping Wang, Harianto Tjong, Chee Hong Wong, Diane D. Mao, Rahul Maurya, Meng-Fan Huang, C. Ngan, Albert H. Kim, Chia-Lin Wei
          </td>
          <td>2024-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Signaling pathways induce stereotyped transcriptional changes as stem cells progress into mature cell types during embryogenesis. Signaling perturbations are necessary to discover which genes are responsive or insensitive to pathway activity. However, gene regulation is additionally dependent on cell state-specific factors like chromatin modifications or transcription factor binding. Thus, transcriptional profiles need to be assayed in single cells to identify potentially multiple, distinct perturbation responses among heterogeneous cell states in an embryo. In perturbation studies, comparing heterogeneous transcriptional states among experimental conditions often requires samples to be collected over multiple independent experiments. Datasets produced in such complex experimental designs can be confounded by batch effects. We present Design-Aware Integration of Single Cell ExpEriments (DAISEE), a new algorithm that models perturbation responses in single-cell datasets with a complex experimental design. We demonstrate that DAISEE improves upon a previously available integrative non-negative matrix factorization framework, more efficiently separating perturbation responses from confounding variation. We use DAISEE to integrate newly collected single-cell RNA-sequencing datasets from 5-hour old zebrafish embryos expressing optimized photoswitchable MEK (psMEK), which globally activates the extracellular signal-regulated kinase (ERK), a signaling molecule involved in many cell specification events. psMEK drives some cells that are normally not exposed to ERK signals towards other wild type states and induces novel states expressing a mixture of transcriptional programs, including precociously activated endothelial genes. ERK signaling is therefore capable of introducing profoundly new gene expression states in developing embryos. Significance Statement Signaling perturbations produce heterogeneous transcriptional responses that must be measured at the single-cell level. Data integration techniques allow us to model these responses which, however, can be confounded by batch effects. We present a computational tool (DAISEE) for extracting the common and perturbation-specific features of single-cell datasets representing multiple experimental conditions while achieving efficient batch effect correction. DAISEE outperforms its predecessor and will enable accurate analysis of a broad range of single-cell datasets. DAISEE applied to new single-cell RNA sequencing data from zebrafish embryos shows that gain-of-function signaling perturbations can induce novel states. Our analysis suggests that a wild type endothelial cell-specification program can be activated in abnormal developmental contexts when the extracellular signal-regulated kinase (ERK) pathway is deregulated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9843baa0dda4952c851210ee811a954b61353c56" target='_blank'>
              Disrupted developmental signaling induces novel transcriptional states
              </a>
            </td>
          <td>
            Aleena Patel, Vanessa Gonzalez, Triveni Menon, Stanislav Y. Shvartsman, R. Burdine, M. Avdeeva
          </td>
          <td>2024-09-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="ABSTRACT Tumorigenesis is commonly attributed to Darwinian processes involving natural selection among cells and groups of cells. However, progressing tumors are those that also achieve an appropriate group phenotypic composition (GPC). Yet, the selective processes acting on tumor GPCs are distinct from that associated with classical Darwinian evolution (i.e. natural selection based on differential reproductive success) as tumors are not genuine evolutionary individuals and do not exhibit heritable variation in fitness. This complex evolutionary scenario is analogous to the recently proposed concept of ‘selection for function’ invoked for the evolution of both living and non-living systems. Therefore, we argue that it is inaccurate to assert that Darwinian processes alone account for all the aspects characterizing tumorigenesis and cancer progression; rather, by producing the genetic and phenotypic diversity required for creating novel GPCs, these processes fuel the evolutionary success of tumors that is dependent on selection for function at the tumor level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fed9ff2b803aab7c7484cdca4ce05128b6e07445" target='_blank'>
              A new perspective on tumor progression
              </a>
            </td>
          <td>
            Frédéric Thomas, James DeGregori, A. Marusyk, Antoine M. Dujon, Beata Ujvari, Jean-Pascal Capp, Robert Gatenby, A. Nedelcu
          </td>
          <td>2024-09-19</td>
          <td>Evolution, Medicine, and Public Health</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9918c8bdd0094035be3fb2b57ac7bbf09b9c90ee" target='_blank'>
              Comprehensive analysis of bulk, single-cell RNA sequencing, and spatial transcriptomics revealed IER3 for predicting malignant progression and immunotherapy efficacy in glioma
              </a>
            </td>
          <td>
            Qi Wang, Chunyu Zhang, Ying Pang, Meng Cheng, Rui Wang, Xu Chen, Tongjie Ji, Yuntong Yang, Jing Zhang, Chunlong Zhong
          </td>
          <td>2024-10-01</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Global healthcare systems have a great challenge in the form of inherited hematological diseases, which necessitates the development of new remedial strategies. By precisely targeting inherited abnormalities, Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR)-CRISPR-associated protein 9 (Cas9)-mediated ex vivo gene editing has surfaced as a promising approach to treat these diseases. This review offers a comprehensive examination of the advancements, challenges, and clinical eventuality of CRISPR-Cas9-intermediated ex vivo gene editing for inherited hematological diseases. With advancements in CRISPR-Cas9 technology, the eventuality to correct inheritable mutations responsible for inherited hematological diseases is within reach. However, challenges such as off-target effects, immune responses, and ethical considerations need to be addressed for the safe and effective perpetration of this technology. A promising understanding of how CRISPR-Cas9-intermediated gene editing functions in practice is handed by ongoing clinical studies, giving rise to the possibility of advanced remedial approaches and bettered patient issues. By addressing these complications in a human-readable format, this review attempts to provide greater understanding and appreciation for the eventuality of CRISPR-Cas9 technology in revolutionizing the treatment landscape for these challenging disorders and contribute to the ongoing discussion in the field and facilitate further exploration towards effective treatments for these challenging disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce4d496cce2c3ddc8c2b97b6be801d229f168c15" target='_blank'>
              CRISPR-CAS9-MEDIATED EX VIVO GENE EDITING FOR INHERITED HEMATOLOGICAL DISORDERS: ADVANCEMENTS, CHALLENGES, AND CLINICAL POTENTIAL
              </a>
            </td>
          <td>
            Atasi Ranjan Panda, Shreeya Das
          </td>
          <td>2024-09-01</td>
          <td>International Journal of Pharmacy and Pharmaceutical Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be1059f8b575168e0a0f7eef1949f1406d63cbad" target='_blank'>
              Alternative splicing in ovarian cancer
              </a>
            </td>
          <td>
            Liwei Wei, Yisheng Li, Jiawang Chen, Yuanmei Wang, Jianmin Wu, Huanming Yang, Yi Zhang
          </td>
          <td>2024-10-18</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa8f6ed0c99a8d797b23d1e3852690ab89f48967" target='_blank'>
              Nanotechnology and CRISPR/Cas-Mediated Gene Therapy Strategies: Potential Role for Treating Genetic Disorders.
              </a>
            </td>
          <td>
            Guneet Kaur, Jasnoor Arora, Abhinashi Singh Sodhi, Sonu Bhatia, Navneet Batra
          </td>
          <td>2024-10-24</td>
          <td>Molecular biotechnology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 Background: Rhabdomyosarcoma (RMS) is a childhood cancer that originates from soft tissue and shares features with skeletal muscle. RMS patients are treated with a combination of radiation, surgical resection, and chemotherapy (VAC; vincristine, actinomycin D and cyclophosphamide). Unfortunately, 30% of patients eventually develop relapsed tumors due to resistance to VAC or radiation, of whom have an abysmal 17% five-year survival rate. To date, the mechanisms underlying resistance to VAC is poorly understood in RMS tumors, which are desperately needed for devising new therapeutic avenues to target VAC chemoresistance and improve the outcomes for these patients. Methods: We have recently generated long-term VAC-resistant RMS cell lines (from RD and Rh41 parental lines) to model chemoresistance. These cell lines were acclimated to high dose VAC over a 3-month period. Afterwards, single clones were isolated from these chemoresistant cell lines and assessed for upregulation of the PIK3CA/AKT/ABC transporter pathway by Western blot analysis. Results: We find that within the pool of resistant clones, there are two distinct pathways that are upregulated to promote VAC-resistance, one that elevates the PIK3CA/AKT/ABC transport pathway to rapidly efflux drugs from RMS cells. Astonishingly, this same drug resistance pathway is activated in both fusion-positive and fusion-negative RMS. The upregulated ABC transporters are well known drug efflux pumps and include ABCB1/MDR1, ABCC1/MRP1 and ABCG2/BCRP. We next show that the increased transcription of these ABC transporters occurs through an AKT-dependent mechanism – akin to that described by our group for an investigational chemotherapy that combines DNA damaging agent temozolomide with Olaparib PARP-inhibitor (OT). Using an inhibitor of PIK3CA signaling, Alpelisib, we show that we can resensitize these chemoresistant cell lines to VAC and OT. Conclusions: VAC-resistance occurs in part through PIK3CA/AKT signaling to promote drug efflux. The PIK3CA/AKT pathway can be therapeutically targeted to resensitize tumors to VAC. Future studies will focus on determining the mechanisms of drug resistance in tumors that fail to upregulate the PIK3CA/AKT/ABC transport pathway.
 Citation Format: Sabateeshan Mathavarajah, Yueyang Wang, Yun Wei, Diego Antelo, David M. Langenau. Mechanisms of chemotherapy resistance in rhabdomyosarcoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A079.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/478aed93fe2d8faacd5d58608f91149a8df572c7" target='_blank'>
              Abstract A079 Mechanisms of chemotherapy resistance in rhabdomyosarcoma
              </a>
            </td>
          <td>
            Sabateeshan Mathavarajah, Yueyang Wang, Yun Wei, Diego Antelo, D. Langenau
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd905b03493f5b109589f0899bdd4480c79c0580" target='_blank'>
              Targeting mutant p53: a key player in breast cancer pathogenesis and beyond
              </a>
            </td>
          <td>
            Hina Qayoom, Burhan Ul Haq, Shazia Sofi, Nusrat Jan, Asma Jan, M. A. Mir
          </td>
          <td>2024-10-10</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 Understanding the molecular mechanisms underlying tumour progression and resistance to treatment is crucial to the development of new treatment approaches. In high risk pediatric cancers, somatic DNA alterations such as genomic amplifications, copy number alterations, translocations or mutations play an important role as molecular diagnostic, prognostic and predictive biomarkers. However it is now known that in many high risk pediatric cancers, clonal evolution most likely plays an important role in tumor progression and treatment resistance. Circulating tumour DNA, a fraction of cell free DNA can be readily isolated from plasma and now provides an important tool and surrogate for tumor molecular analyses at diagnosis, during treatment and follow-up. At diagnosis, the prospective clinical trials MICCHADO (NCT03496402) enrolled 599 patients, including high-risk pediatric cancer (Neuroblastoma, Rhabdomyosarcoma, Ewing sarcoma, other high-risk cancers). Whole -Exome-Sequencing (WES) was performed on tumor, germline DNA and cfDNA extracted from plasma at diagnosis, during treatment and follow up. Whereas all cfDNA samples obtained at follow-up in patients without evidence of disease revealed no or few tumor cell-specific SNVs, cfDNA samples obtained at relapse harbored additional, new relapse-specific SNVs in all cases, targeting genes of interest. Deep sequencing capture techniques enable to develop models of clonal evolution. In pediatric embryonal brain tumours, ctDNA isolated from CSF enables detection of tumor cell specific genetic alterations with a high sensitivity. At relapse, ctDNA studies can provide complementary information to molecular analyses of tumour samples performed within programs such as MAPPYACTS (NCT02613962), with 76% of actionable alterations detected in tumor also identified in ctDNA, while also highlighting the importance of spatial heterogeneity. MONALISA, a SIOPEN pragmatic clinical trial to MOnitor NeuroblastomA relapse with LIquid biopsy Sensitive Analysis aims to establish liquid biopsies as standard-of-care to monitor relapsed/refractory neuroblastoma. Reliable, early assessment of molecular progression or relapse determined by mRNA and/or ctDNA analysis is the main aim of this randomized clinical trial. ctDNA also enables to infer expression profiles. Gene expression levels are reflected by nucleosome positioning, and differences in nucleosome organization at transcription start sites (TSS) lead to differential clipping of fragments upon ctDNA release and distinct nucleosome footprints depending on the expression of a given gene in the originating cells. Altogether the presence of tumor genetic and epigenetic abnormalities in ctDNA can be documented in most patients with high risk pediatric cancer and frequently suggest spatial and temporal heterogeneity. Sequential studies will further elucidate mechanisms of clonal evolution, tumor progression and therapy resistance. Thus, sequential studies based on liquid biopsies are now integrated into the development and optimization of targeted treatment strategies.
 Citation Format: Gudrun Schleiermacher. Opportunities and challenges for liquid biopsies in pediatric oncology [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr IA010.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d5d9b1f8524120f05332a3107841ee2c1bb8ff8" target='_blank'>
              Abstract IA010: Opportunities and challenges for liquid biopsies in pediatric oncology
              </a>
            </td>
          <td>
            G. Schleiermacher
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d7559338158829d62b6b4147b9bfe3e99b724d9" target='_blank'>
              HLA-G neo-expression modifies genetic programs governing tumor cell lines
              </a>
            </td>
          <td>
            D. Tronik-Le Roux, M. Daouya, I. Poras, François Desgrandchamps, E. Carosella
          </td>
          <td>2024-10-03</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="To comprehensively unravel the temporal relationship between initiating and driver events and its impact on clinical outcomes, we analyzed 421 whole-genome sequencing profiles from 382 patients. Using clock-like mutational signatures, we estimated a time lag of 2-4 decades between initiating events and diagnosis. In patients with hyperdiploidy, we demonstrate that trisomies of odd-numbered chromosomes can be acquired simultaneously with other chromosomal gains, such as 1q gain. We provide evidence that hyperdiploidy is acquired after canonical IGH translocation when both events are present. Finally, patients with early 1q gain had adverse outcomes similar to those with 1q amplification (>1 extra-copies), but faring worse than those with late 1q gain. This underscores that the prognostic impact of 1q gain/amp depends more on the timing of acquisition than on the number of extra copies gained. Overall, this study contributes to a better understanding of the life history of MM and may have prognostic implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f98e16f03f4610ad7609f6b41fac703153b2f8eb" target='_blank'>
              Temporal Genomic Dynamics Shape Clinical Trajectory in Multiple Myeloma
              </a>
            </td>
          <td>
            Francesco Maura, Marcella Kaddoura, A. Poos, L. Baughn, B. Ziccheddu, Marc-Andrea Bärtsch, Anthony Cirrincione, K. Maclachlan, M. Chojnacka, B. Diamond, M. Papadimitriou, P. Blaney, L. John, Philipp Reichert, S. Huhn, Dylan Gagler, Yanming Zhang, Ahmet Dogan, A. Lesokhin, Faith Davies, Hartmut Goldschmidt, Roland Fenk, Katja C. Weisel, E. Mai, N. Korde, Gareth Morgan, S. Rajkumar, Shaji Kumar, S. Usmani, O. Landgren, Marc-Steffen Raab, Niels Weinhold
          </td>
          <td>2024-09-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>112</td>
        </tr>

        <tr id="Carcinogenesis is an evolutionary process, and mutations can fix the selected phenotypes in selective microenvironments. Both normal and neoplastic cells are robust to the mutational stressors in the microenvironment to the extent that secure their fitness. To test the robustness of genes under a range of mutagens, we developed a sequential mutation simulator, Sinabro, to simulate single base substitution under a given mutational process. Then, we developed a pipeline to measure the robustness of genes and cells under those mutagenesis processes. We discovered significant human genome robustness to the APOBEC mutational signature SBS2, which is associated with viral defense mechanisms and is implicated in cancer. Robustness evaluations across over 70,000 sequences against 41 signatures showed higher resilience under signatures predominantly causing C-to-T (G-to-A) mutations. Principal component analysis indicates the GC content at the codon’s wobble position significantly influences robustness, with increased resilience noted under transition mutations compared to transversions. Then, we tested our results in bats at extremes of the lifespan-to-mass relationship and found the long-lived bat is more robust to APOBEC than the short-lived one. By revealing robustness to APOBEC ranked highest in human (and bats with much more than number of APOBEC) genome, this work bolsters the key potential role of APOBECs in aging and cancer, as well as evolved countermeasures to this innate mutagenic process. It also provides the baseline of the human and bat genome robustness under mutational processes associated with aging and cancer. Highlights Sinabro, the sequential mutation simulator, facilitates measuring the robustness of human protein-coding sequences under all COSMIC mutational signatures. Robustness under APOBEC mutational signatures showed the largest mean and standard deviation in the human genome. Robustness to mutational signatures analysis reveals the role of APOBECs is complementary to cancer in the evolvability of cancer cells in later stages. Principal component analysis indicates that the GC content at the codon’s wobble position significantly influences robustness. A long-lived bat (Myotis myotis) has higher robustness to APOBECs than a short-lived one (Molossus molossus) than humans.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/011df1029d192af53d074e293c75b5f6a3b83940" target='_blank'>
              Human and bats genome robustness under COSMIC mutational signatures
              </a>
            </td>
          <td>
            Joon-Hyun Song, Ying Zeng, Liliana M. Dávalos, Thomas MacCarthy, Mani Larijani, Mehdi Damaghi
          </td>
          <td>2024-09-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Recent studies demonstrate growing roles for genetic mosaicism in neurodevelopmental and neuropsychiatric disorders, with the paradigm being drug-resistant pediatric focal epilepsy related to activating somatic variants in the PI3K-mTOR pathway. While identifying the genotype-associated changes at the single-cell level is fundamental to understanding disease pathophysiology, this remains technically challenging in human tissue samples with existing methods. Here, we performed single-nucleus RNA-sequencing (snRNA-seq) of 20 focal cortical dysplasia (FCD) samples removed surgically for treatment of drug-resistant epilepsy, and 10 non-FCD controls, and we developed a new approach, Genotyping Of Transcriptomes Enhanced with Nanopore sequencing (GO-TEN), that combines targeted complementary (c)DNA sequencing with snRNA-seq to perform concurrent single-nucleus genotyping and transcriptional analysis. We find that mosaic pathogenic variants in FCD do not produce a detectable novel cell identity, but instead we observe conserved cell types present both in FCD cases and non-FCD control specimens. Similarly, GO-TEN analysis shows that most pathogenic variant-carrying cells have well-differentiated neuronal or glial identities and are enriched for layer II-III excitatory neurons. We identify cell-intrinsic disruption of glutamate and GABA-A signaling pathways in variant-carrying neurons and altered intercellular signaling, making potential mechanisms for epileptogenesis in FCD. In summary, by addressing genotype-specific changes in mosaic epilepsy-associated lesions, our study highlights new potential disease mechanisms and therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc181f7065f3cb1a379b3d4368b2d029b87b1b3b" target='_blank'>
              Cell-type-informed genotyping of mosaic focal epilepsies reveals cell-autonomous and non-cell-autonomous disease-associated transcriptional programs
              </a>
            </td>
          <td>
            S. Bizzotto, E. Stronge, Maya Talukdar, Qiwen Hu, Zinan Zhou, A. Huang, B. Chhouk, Alissa M. D’Gama, Edward Yang, T. E. Green, David C. Reutens, Saul A. Mullen, Michael S. Hildebrand, Russell J Buono, A. Poduri, S. Khoshkhoo, Christopher A. Walsh
          </td>
          <td>2024-10-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Most phenotype-associated genetic variants map to non-coding regulatory regions of the human genome. Moreover, variants associated with blood cell phenotypes are enriched in regulatory regions active during hematopoiesis. To systematically explore the nature of these regions, we developed a highly efficient strategy, Perturb-multiome, that makes it possible to simultaneously profile both chromatin accessibility and gene expression in single cells with CRISPR-mediated perturbation of a range of master transcription factors (TFs). This approach allowed us to examine the connection between TFs, accessible regions, and gene expression across the genome throughout hematopoietic differentiation. We discovered that variants within the TF-sensitive accessible chromatin regions, while representing less than 0.3% of the genome, show a ∼100-fold enrichment in heritability across certain blood cell phenotypes; this enrichment is strikingly higher than for other accessible chromatin regions. Our approach facilitates large-scale mechanistic understanding of phenotype-associated genetic variants by connecting key cis-regulatory elements and their target genes within gene regulatory networks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd75703a3e0196006c63fecef421b6d1dc25cee5" target='_blank'>
              Transcription factor networks disproportionately enrich for heritability of blood cell phenotypes
              </a>
            </td>
          <td>
            Jorge D. Martin-Rufino, Alexis Caulier, Seayoung Lee, Nicole Castano, Emily King, Samantha Joubran, Marcus Jones, Seth R Goldman, U. Arora, Lara Wahlster, Eric S. Lander, V. Sankaran
          </td>
          <td>2024-09-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Fragile X syndrome (FXS), the leading genetic cause of intellectual disability, arises from FMR1 gene silencing and loss of the FMRP protein. N6-methyladenosine (m6A) is a prevalent mRNA modification essential for post-transcriptional regulation. FMRP is known to bind to and regulate the stability of m6A-containing transcripts. However, how loss of FMRP impacts on transcriptome-wide m6A modifications in FXS patients remains unknown. To answer this question, we generated cortical neurons differentiated from induced pluripotent stem cells (iPSC) derived from healthy subjects and FXS patients. In electrophysiology recordings, we validated that synaptic and neuronal network defects in iPSC-derived FXS neurons corresponded to the clinical EEG data of the patients from which the corresponding iPSC line was derived. In analysis of transcriptome-wide methylation, we show that FMRP deficiency led to increased translation of m6A writers, resulting in hypermethylation that primarily affecting synapse-associated transcripts and increased mRNA decay. Conversely, in the presence of an m6A writer inhibitor, synaptic defects in FXS neurons were rescued. Taken together, our findings uncover that an FMRP-dependent epi-transcriptomic mechanism contributes to FXS pathogenesis by disrupting m6A modifications in FXS, suggesting a promising avenue for m6A-targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df3d7886af5b605cb94d6ce27298f14a423e6ea5" target='_blank'>
              An iPSC model of fragile X syndrome reflects clinical phenotypes and reveals m6A-mediated epi-transcriptomic dysregulation underlying synaptic dysfunction
              </a>
            </td>
          <td>
            Lu Lu, Avijite Kumer Sarkar, Lan Dao, Yanchen Liu, Chunlong Ma, Phyo Han Thwin, Xuyao Chang, George Yoshida, Annie Li, Cenjing Wang, Crace Westerkamp, Lauren Schmitt, Maag Chelsey, Monzon Stephanie, Yu Zhao, Yaping Liu, Xiong Wang, Ling-Qiang Zhu, Dan Liu, Jason Tchieu, Makoto Miyakoshi, Haining Zhu, Christina Gross, Ernest Pedapati, Nathan Salomonis, Craig Erickson, Ziyuan Guo
          </td>
          <td>2024-10-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Introduction. The TP53 gene encodes the p53 protein, crucial for DNA damage response, apoptosis, and cell cycle regulation. In chronic lymphocytic leukemia (CLL), TP53 loss due to 17p deletion or mutation leads to poor chemoimmunotherapy response and shorter survival. 
Patients with unmutated immunoglobulin heavy chain variable (IGHV) status exhibit a more aggressive disease course, poorer outcomes, and reduced response to standard chemoimmunotherapy. These genetic variations significantly affect disease progression, treatment choice, and response to therapy. 
Case Report. In November 2023, a 61-year-old man presented to our clinic with complaints of generalized weakness and rapid fatigue since October 2023. CLL was diagnosed in 2018 through pathohistological and immunohistochemical analysis of the bone marrow. The patient received no therapy until 2023 due to the absence of active disease signs. From March 28, 2023, to July 25, 2023, the patient received chemoimmunotherapy with the bendamustine and rituximab (BR) regimen. Five courses of therapy were administered; however, further treatment was discontinued due to side effects that occurred during the fifth course of bendamustine administration. A subsequent diagnostic evaluation aimed at assessing treatment efficacy, prognosis, and genetic profiling included a cytomorphological analysis, which revealed that lymphocytes comprised 80% of the cellular composition. Immunocytological analysis confirmed the presence of monoclonal atypical lymphocytes, accounting for 70% of the cell population. Next-generation sequencing identified a pathogenic TP53 mutation, and fragment analysis confirmed an unmutated IGHV status. The presence of the TP53 mutation and unmutated IGHV status categorizes the patient as being in the very high-risk group, for which the use of Bruton’s tyrosine kinase inhibitors (BTKIs) and/or B-cell lymphoma-2 (BCL-2) inhibitors is recommended. 
Conclusions. This case highlights the significance of comprehensive genetic profiling in CLL patients using techniques such as fluorescence in situ hybridization, polymerase chain reaction, and next-generation sequencing. Such profiling detects molecular genetic alterations, facilitating personalized and effective treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a9a770a1a98466f2dfc1c687a0f49ae58b9afa5" target='_blank'>
              Genetic Profiling in Chronic Lymphocytic Leukemia: The Role of TP53 Mutations and IGHV Status in Treatment Decision-Making – A Case Report
              </a>
            </td>
          <td>
            Kateryna Khurdepa, Oksana Karnabeda
          </td>
          <td>2024-10-16</td>
          <td>Galician Medical Journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract The p53 tumor suppressor gene governs a multitude of complex cellular processes that are essential for anti-cancer function and whose dysregulation leads to aberrant gene transcription, activation of oncogenic signaling and cancer development. Although mutations can occur at any point in the genetic sequence, missense mutations comprise the majority of observed p53 mutations in cancers regardless of whether the mutation is germline or somatic. One biological process involved in both mutant and wild-type p53 signaling is the N6-methyladenosine (m6A) epitranscriptomic network, a type of post-transcriptional modification involved in over half of all eukaryotic mRNAs. Recently, a significant number of findings have demonstrated unique interactions between p53 and the m6A epitranscriptomic network in a variety of cancer types, shedding light on a previously uncharacterized connection that causes significant dysregulation. Cross-talk between wild-type or mutant p53 and the m6A readers, writers and erasers has been shown to impact cellular function and induce cancer formation by influencing various cancer hallmarks. Here, this review aims to summarize the complex interplay between the m6A epitranscriptome and p53 signaling pathway, highlighting its effects on tumorigenesis and other hallmarks of cancer, as well as identifying its therapeutic implications for the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e91fbc5b77b6bf7efe4bda3817610c3bb0dedde" target='_blank'>
              Decoding the molecular symphony: interactions between the m6A and p53 signaling pathways in cancer
              </a>
            </td>
          <td>
            R. Shoemaker, Mo-Fan Huang, Ying-Si Wu, Cheng-Shuo Huang, Dung-Fang Lee
          </td>
          <td>2024-07-09</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Preclinical models such as cell lines and mice are the backbone of drug development and experimental-mechanistic oncology. However, we currently lack a detailed understanding of the direct clinical relevance of data collected in most preclinical models, hampering the development of new treatments. Despite this, few formal approaches have been proposed to determine how the various preclinical models represent/resemble primary patient tumors. Here, we present the first comprehensive single-cell RNA-seq analysis of neuroblastoma across an extensive cohort of patient tumors and a variety of preclinical model systems (n = 126 total samples assembled – the largest cohort of its kind). By developing an unsupervised machine learning method, which we term “automatic consensus nonnegative matrix factorization” (acNMF), we have integrated and contrasted the transcriptional landscapes of patient tumors with those of cell lines, patient-derived xenografts (PDX), and genetic mouse models (GEMM). We discovered that the dominant adrenergic gene expression programs commonly found in neuroblastoma patient tumors were generally preserved across all preclinical models. However, the presumptive chemo-resistant mesenchymal-like programs, while identifiable in cell lines, were primarily restricted to subpopulations of cancer-associated fibroblasts and Schwann-like cells in vivo. Surprisingly however, a mesenchymal-like program could be acutely chemotherapy-induced in GEMM and was evident in pre-treated patient and PDX samples, suggesting a previously uncharacterized mechanism of therapy escape resulting from an acute shift in cell state. In addition, our approach could further delineate the classical neuroblastoma adrenergic and mesenchymal gene expression programs, discovering for example, novel subpopulations of cancer associated fibroblasts and reproducible subtypes of adrenergic programs. These behaviors were conserved across tumors and preclinical models, which we validated by RNA in situ hybridization, an ultra-sensitive, high resolution, spatial transcriptomics technology. Overall, we offer a nuanced, high-resolution view of neuroblastoma pre-clinical systems for advancing therapeutic development, as well as a generalizable set of computational tools, which can be applied in other diseases. We have created an open-source web resource, featuring this integrated map to aid the scientific community in further exploration of these integrated data (available at http://pscb.stjude.org).
 Citation Format: Richard H. Chapple, Xueying Liu, Sivaraman Natarajan, Margaret I.M. Alexander, Yuna Kim, Anand G. Patel, Christy W. LaFlamme, Min Pan, William C. Wright, Hyeong-Min Lee, Yinwen Zhang, Meifen Lu, Selene C. Koo, Courtney Long, John Harper, Chandra Savage, Melissa D. Johnson, Thomas Confer, Walter J. Akers, Michael A. Dyer, Heather Sheppard, John Easton, Paul Geeleher. An integrated single-cell RNA-seq map of human neuroblastoma tumors and preclinical models uncovers divergent mesenchymal-like gene expression programs [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B006.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b38392d6531b277e9f9ee236ca213de0c124f13" target='_blank'>
              Abstract B006: An integrated single-cell RNA-seq map of human neuroblastoma tumors and preclinical models uncovers divergent mesenchymal-like gene expression programs
              </a>
            </td>
          <td>
            Richard H. Chapple, Xueying Liu, Sivaraman Natarajan, M. I. Alexander, Yuna Kim, Anand G. Patel, Christy W. LaFlamme, Min Pan, William C. Wright, Hyeong-Min Lee, Yinwen Zhang, Meifen Lu, Selene C. Koo, Courtney Long, John Harper, Chandra Savage, Melissa D. Johnson, Thomas Confer, Walter J. Akers, Michael A. Dyer, Heather Sheppard, John Easton, P. Geeleher
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Abstract Pleiotropy, the phenomenon in which a single gene influences multiple traits, is a fundamental concept in genetics. However, the evolutionary mechanisms underlying pleiotropy require further investigation. In this study, we conducted parallel gene knockouts targeting 100 transcription factors in 2 strains of Saccharomyces cerevisiae. We systematically examined and quantified the pleiotropic effects of these knockouts on gene expression levels for each transcription factor. Our results showed that the knockout of a single gene generally affected the expression levels of multiple genes in both strains, indicating various degrees of pleiotropic effects. Strikingly, the pleiotropic effects of the knockouts change rapidly between strains in different genetic backgrounds, and ∼85% of them were nonconserved. Further analysis revealed that the conserved effects tended to be functionally associated with the deleted transcription factors, while the nonconserved effects appeared to be more ad hoc responses. In addition, we measured 184 yeast cell morphological traits in these knockouts and found consistent patterns. In order to investigate the evolutionary processes underlying pleiotropy, we examined the pleiotropic effects of standing genetic variations in a population consisting of ∼1,000 hybrid progenies of the 2 strains. We observed that newly evolved expression quantitative trait loci impacted the expression of a greater number of genes than did old expression quantitative trait loci, suggesting that natural selection is gradually eliminating maladaptive or slightly deleterious pleiotropic responses. Overall, our results show that, although being prevalent for new mutations, the majority of pleiotropic effects observed are evolutionarily transient, which explains how evolution proceeds despite complicated pleiotropic effects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74979a3b8c99e59fe3aefc2b4e6cac48414c9093" target='_blank'>
              Most Pleiotropic Effects of Gene Knockouts Are Evolutionarily Transient in Yeasts
              </a>
            </td>
          <td>
            Li Liu, Yao Liu, Lulu Min, Zhenzhen Zhou, Xingxing He, Yunhan Xie, Waifang Cao, Shuyun Deng, Xiaoju Lin, Xionglei He, Xiaoshu Chen
          </td>
          <td>2024-09-01</td>
          <td>Molecular Biology and Evolution</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eddab72a32f5e67003082a5b17959fbc71f348a1" target='_blank'>
              Gene therapy for polygenic or complex diseases
              </a>
            </td>
          <td>
            Tingting Wu, Yu Hu, Liang V Tang
          </td>
          <td>2024-09-04</td>
          <td>Biomarker Research</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Base editing shows promise for the correction of human mutations at a higher efficiency than other repair methods and is especially attractive for mutations in large genes that are not amenable to gene augmentation therapy. Here, we demonstrate a comprehensive workflow for in vitro screening of potential therapeutic base editing targets for the USH2A gene and empirically validate the efficiency of adenine and cytosine base editor/guide combinations for correcting 35 USH2A mutations. Editing efficiency and bystander edits are compared between different target templates (plasmids versus transgenes) and assays (Next generation sequencing versus Sanger), as well as comparisons between unbiased empirical results and computational predictions. Based on these observations, practical assay recommendations are discussed. Finally, a humanized knock-in mouse model was created with the best-performing target, the nonsense mutation c.11864G>A p.(Trp3955*). Split-intein AAV9 delivery of editing reagents resulted in the restoration of USH2A protein and a correction rate of 65 ± 3% at the mutant base pair and of 52 ± 3% excluding bystander amino acid changes. This efficiency compares favorably to a prior genome editing strategy tested in the retina that completed a clinical trial and demonstrates the effectiveness of this overall strategy to identify and test base editing reagents with the potential for human therapeutic applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48b11e55d59e66e72cd37b7653ddc5c10089e197" target='_blank'>
              Systematic empirical evaluation of individual base editing targets: validating therapeutic targets in USH2A and comparison of methods
              </a>
            </td>
          <td>
            Yuki Tachida, K. Manian, Rossano Butcher, Jonathan M. Levy, Nachiket C. Pendse, E. Hennessey, David R. Liu, Eric A. Pierce, Qin Liu, J. Comander
          </td>
          <td>2024-10-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd6bcaf46c997cceee1e5ce02e4d68cb36cec179" target='_blank'>
              Targeting mRNA-coding genes in prostate cancer using CRISPR/Cas9 technology with a special focus on androgen receptor signaling
              </a>
            </td>
          <td>
            Mobina Tabibian, Fahimeh Salasar Moghaddam, E. Motevaseli, S. Ghafouri-Fard
          </td>
          <td>2024-10-17</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="


 High-grade gliomas (HGGs) are the most aggressive type of gliomas and have the poorest outcomes. Chromatin remodeling (CR) genes have been implicated in multiple oncogenic pathways in numerous cancer types. In gliomagenesis, CR genes have been implicated in regulating stemness of glioma cells, the tumor microenvironment (TME), and resistance to therapies.



 We performed molecular profiling of 4244 HGGs and evaluated associations of CR mutations with other cancer related biomarkers, infiltration by immune cells, and immune gene expression. We also evaluated the association between CR mutations and survival in IDH WT HGG patients.



 Nearly 10% of HGGs carry mutations in CR genes, with higher prevalence (15%) in HGGs with IDH mutations. Analysis of co-occurrence with other biomarkers revealed that CR-mutated HGGs possess favorable genetic alterations which may have prognostic value. CR-mutated HGGs with wild type IDH demonstrated colder TME and worse OS overall compared to the CR-wild type HGGs.



 Our study reveals the prognostic effects of CR mutations in HGG and points to several biomarker candidates that could suggest sensitivity to emerging therapeutic strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18a6ec214f04da421e41c3579984b7e7f5d8d94d" target='_blank'>
              Biological and prognostic relevance of epigenetic regulatory genes in high-grade gliomas (HGGs)
              </a>
            </td>
          <td>
            Sonikpreet Aulakh, Joanne Xiu, Andrew Hinton, S. Darabi, M. Demeure, S. Sengupta, Santosh Kesari, David M Ashley, Ashley Love Sumrall, M. Glantz, D. Spetzler
          </td>
          <td>2024-10-04</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Synovial Sarcoma (SySa) is an aggressive soft tissue sarcoma that accounts for 5 – 10% of all soft tissue sarcomas. Current treatment involves radiation and radical surgery including limb amputation, highlighting the urgent need to develop targeted therapies. We reasoned that transcriptional rewiring by the fusion protein SS18-SSX, the sole oncogenic driver in SySa, creates specific vulnerabilities that can be exploited for treatment. To uncover genes that are selectively essential for SySa, we mined The Cancer Dependency Map (DepMap) data to identify genes that specifically impact the fitness of SySa compared to other tumor cell lines. Targeted CRISPR library screening of SySa-selective candidates revealed that the small ubiquitin-like modifier 2 (SUMO2) was one of the strongest dependencies both in vitro as well as in vivo. TAK-981, a clinical-stage small molecule SUMO2 inhibitor potently inhibited growth and colony-forming ability. Strikingly, transcriptomic studies showed that pharmacological SUMO2 inhibition with TAK-981 treatment elicited a profound reversal of a gene expression program orchestrated by SS18-SSX fusions. Of note, genetic or pharmacological SUMO2 inhibition reduced global and chromatin levels of the SS18-SSX fusion protein with a concomitant reduction in histone 2A lysine 119 ubiquitination (H2AK119ub), an epigenetic mark that plays an important role in SySa pathogenesis. Taken together, our studies identify SUMO2 as a novel, selective vulnerability in SySa. Since SUMO2 inhibitors are currently in Phase 1/2 clinical trials for other cancers, our findings present a novel avenue for targeted treatment of synovial sarcoma. SIGNIFICANCE Our study identifies SUMO2 as a selective dependency in synovial sarcoma. We demonstrate that the SUMO2/3 inhibitor TAK-981 impairs sarcomagenesis and reverses the SS18-SSX fusion-driven oncotranscriptome. Our study indicates that SUMO2 inhibition may be an attractive therapeutic option in synovial sarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b1863d8b2a5acaf7a4dc2a730005a178b4452ec" target='_blank'>
              SUMO2 Inhibition Reverses Aberrant Epigenetic Rewiring Driven by Synovial Sarcoma Fusion Oncoproteins and Impairs Sarcomagenesis
              </a>
            </td>
          <td>
            Rema Iyer, Anagha Deshpande, Aditi Pedgaonkar, Pramod Akula Bala, Taehee Kim, Gerard L. Brien, D. Finlay, K. Vuori, A. Soragni, Rabi Murad, Aniruddha J. Deshpande
          </td>
          <td>2024-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Background: Genetic studies of ovarian cancer (OC) have historically focused on BRCA1/2 mutations, lacking other studies of homologous recombination repair (HRR). Poly (ADP-ribose) polymerase inhibitors (PARPi) exploit synthetic lethality to significantly improve OC treatment outcomes, especially in BRCA1/2 deficiency patients. Objectives: Our study aims to construct a mutation map of HRR genes in OC and identify factors influencing the efficacy of PARPi. Design: A retrospective observational analysis of HRR gene variation data from 695 OC patients from March 2019 to February 2022 was performed. Methods: The HRR gene variation data of 695 OC patients who underwent next-generation sequencing (NGS) in the First Affiliated Hospital of Zhengzhou University were retrospectively collected. Clinical data on the use of PARPi in these patients were also gathered to identify factors that may interfere with the efficacy of PARPi. Results: Out of 127 pathogenic variants in the BRCA1/2 genes, 104 (81.9%) were BRCA1 mutations, and 23 (18.1%) were BRCA2 mutations. Among the 59 variants of uncertain significance (VUS), 20 (33.9%) were BRCA1, while 39 (66.1%) were BRCA2 mutations. In addition to BRCA1/2, HRR gene results showed that 9 (69%) of 13 were HRR pathway pathogenic variants; and 16 (1.7%) of 116 VUS were Food and Drug Administration (FDA)-approved mutated HRR genes. Notably, the treatment regimen significantly influenced the effectiveness of PARPi, especially when using first-line maintenance therapy, leading to enhanced progression-free survival (PFS) compared to alternative protocols. Conclusion: Focusing on HRR gene mutations and supporting clinical research about PARPi in OC patients is crucial for developing precision treatment strategies and enhancing prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d41e2645926ac18cf48bd055b15a6838f0eb8b2d" target='_blank'>
              Genetic landscape of homologous recombination repair and practical outcomes of PARPi therapy in ovarian cancer management
              </a>
            </td>
          <td>
            Mengyu Chen, Ningjing Lei, Ruixia Guo, Liping Han, Qinghe Zhao, Yang Zhao, Luojie Qiu, Fengling Wu, Shan Jiang, Ningyao Tong, Kunmei Wang, Siyu Li, Lei Chang
          </td>
          <td>2024-01-01</td>
          <td>Therapeutic Advances in Medical Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Hepatocellular adenomas (HAs) are tumors that can develop under different conditions, including in patients harboring a germline mutation in HNF1A. However, little is known about the pathogenesis of such disease. This work aims to better define what mechanisms lie under the development of this condition. Six HAs were sampled from the liver of a 17-year-old male affected by diabetes and multiple hepatic adenomatosis harboring the heterozygous pathogenic germline variant c.815G>A, p.(Arg272His) in HNF1A, which has a dominant negative effect. All HAs were molecularly characterized. Four of them were shown to harbor a second somatic HNF1A variant and one had a mutation in the ARID1A gene, while no additional somatic changes were found in the remaining HA and normal parenchyma. A transcriptomic profile of the same HA samples was also performed. HNF1A biallelic mutations were associated with the up-regulation of several pathways including the tricarboxylic acid cycle, the metabolism of fatty acids, and mTOR signaling while angiogenesis, endothelial and vascular proliferation, cell migration/adhesion, and immune response were down-regulated. Contrariwise, in the tumor harboring the ARID1A variant, angiogenesis was up-modulated while fatty acid metabolism was down-modulated. Histological analyses confirmed the molecular data. Independently of the second mutation, energetic processes and cholesterol metabolism were up-modulated, while the immune response was down-modulated. This work provides a complete molecular signature of HNF1A-associated HAs, analyzing the association between specific HNF1A variants and the development of HA while identifying potential new therapeutic targets for non-surgical treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26474672d542f89d0fa38cb2c1556ad93882621f" target='_blank'>
              Genomic and Transcriptomic Profile of HNF1A-Mutated Liver Adenomas Highlights Molecular Signature and Potential Therapeutic Implications
              </a>
            </td>
          <td>
            A. Faini, F. Arruga, M. Pinon, Valeria Bracciamà, Francesco Edoardo Vallone, Fiorenza Mioli, M. Sorbini, Martina Migliorero, A. Gambella, Damiano Carota, Isaac Giraudo, P. Cassoni, Silvia Catalano, Renato Romagnoli, Antonio Amoroso, P. L. Calvo, T. Vaisitti, Silvia Deaglio
          </td>
          <td>2024-09-29</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d65d1445af3114dab08deaa9bc460f66343fa73" target='_blank'>
              Alternative splicing of PBRM1 mediates resistance to PD-1 blockade therapy in renal cancer.
              </a>
            </td>
          <td>
            Namjoon Cho, Seung-Yeon Kim, Sung-Gwon Lee, Chungoo Park, Sunkyung Choi, Eun-Mi Kim, Kee K. Kim
          </td>
          <td>2024-10-07</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29a40ea797c9beb02b883d2874be040346de7222" target='_blank'>
              Longitudinal multiomics analysis of aggressive pituitary neuroendocrine tumors: comparing primary and recurrent tumors from the same patient, reveals genomic stability and heterogeneous transcriptomic profiles with alterations in metabolic pathways
              </a>
            </td>
          <td>
            Keiko Taniguchi-Ponciano, S. Hinojosa-Alvarez, Jesús Hernández-Pérez, R. Chavez-Santoscoy, Ilan Remba-Shapiro, G. Guinto, Erika Magallon-Gayon, Benjamin Telles-Ramirez, Rodrigo Ponce de Leon-Conconi, S. Vela-Patiño, Sergio Andonegui-Elguera, Amayrani Cano-Zaragoza, F. Martinez-Mendoza, Jacobo Kerbel, M. Loza-Mejía, Juan Rodrigo-Salazar, A. Mendez-Perez, C. Aguilar-Flores, A. Chávez-González, E. Ortiz-Reyes, Erick Gomez-Apo, L. Bonifaz, D. Marrero-Rodríguez, Moises Mercado
          </td>
          <td>2024-08-31</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Maintaining engineered cell populations’ genetic stability is a key challenge in synthetic biology. Synthetic genetic constructs compete with a host cell’s native genes for expression resources, burdening the cell and impairing its growth. This creates a selective pressure favouring mutations which alleviate this growth defect by removing synthetic gene expression. Non-functional mutants thus spread in cell populations, eventually making them lose engineered functions. Past work has attempted to limit mutation spread by coupling synthetic gene expression to survival. However, these approaches are highly context-dependent and must be tailor-made for each particular synthetic gene circuit to be retained. In contrast, we develop and analyse a biomolecular controller which depresses mutant cell growth independently of the mutated synthetic gene’s identity. Modelling shows how our design can be deployed alongside various synthetic circuits without any re-engineering of its genetic components, outperforming extant gene-specific mutation spread mitigation strategies. Our controller’s performance is evaluated using a novel simulation approach which leverages resource-aware cell modelling to directly link a circuit’s design parameters to its population-level behaviour. Our design’s adaptability promises to mitigate mutation spread in an expanded range of applications, whilst our analyses provide a blueprint for using resource-aware cell models in circuit design.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2420835e594d4a65b9dfcd0a4c88ad37ec47058" target='_blank'>
              Model-guided gene circuit design for engineering genetically stable cell populations in diverse applications
              </a>
            </td>
          <td>
            Kirill Sechkar, Harrison Steel
          </td>
          <td>2024-09-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The transcriptional state of a cell provides a multi-parameter representation of gene expression programs that reflect its identity and phenotype. However, current transcriptomic profiling technologies result in destruction of the biological sample, preventing direct analysis of transcriptional dynamics in the same living cells over time. Here, we developed a synthetic RNA export system called cellular ‘self-reporting’ to address this fundamental technological limitation. We repurposed the murine leukemia virus retroviral protein Gag to enable diverse types of immortalized and primary mammalian cells to package cellular RNA molecules in virus-like particles (VLPs) for export into the extracellular environment. We applied self-reporting to interrogate the transcriptome-wide dynamics that occur during neuronal differentiation from induced pluripotent stem cells and detected gene expression profiles from individual live cells. Leveraging this genetically encodable approach, we expanded the capabilities of self-reporting through molecular engineering of VLP components. Pseudotyping VLPs with epitope-tagged envelope proteins enabled multiplexed selective live-cell readout of transcriptional states from heterogeneous co-cultures. Furthermore, structure-guided protein engineering of Gag fusions with human RNA binding domains improved the mRNA representation in self-reporting readouts and enabled the directed export of libraries of synthetic barcode transcripts. Taken together, this work establishes self-reporting as a facile and broadly enabling technology for live-cell, transcriptome-scale profiling of dynamic processes across diverse cell types and biological applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6bd9363ab1dcbb456eb0b3e0944d84bfdb375d0c" target='_blank'>
              Live-cell transcriptomics with engineered virus-like particles
              </a>
            </td>
          <td>
            M. Najia, Jacob Borrajo, Anna Le, FuNien Tsai, Jeremy Y. Huang, Linda G. Griffith, George Q Daley, P. Blainey
          </td>
          <td>2024-10-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Resistance to therapy remains a significant challenge in cancer treatment, often due to the presence of a stem-like cell population that drives tumor recurrence post-treatment. Moreover, many anticancer drugs inadvertently induce plasticity, reverting differentiated cancer cells to drug-resistant stem-like states. Addressing this phenomenon is crucial for advancing cancer therapeutics. In this study, we introduce a robust statistical framework based on multi-type branching processes that can dissect tumor dynamics and drug effects from high throughput drug screening data. Through comprehensive in silico experiments, we show the efficacy of our framework in estimating parameters governing population dynamics and drug responses in a heterogeneous tumor population where cell state transitions are influenced by the drug. Finally, using recent in vitro data involving AGS-SORE6+/- cells treated with ciclopirox olamine, we show how our framework can be used to confirm the presence of drug-induced cell plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85013788a6ef1ec3c633fabb86b8c0544b0624bd" target='_blank'>
              Inferring Drug-induced Plasticity via Drug Screen Data
              </a>
            </td>
          <td>
            Chenyu Wu, E. B. Gunnarsson, J. Foo, K. Leder
          </td>
          <td>2024-09-27</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93f34d93ff1b305ad73729aa0a43e0b0a160605f" target='_blank'>
              Characterization of MYC Rearrangements in Multiple Myeloma: an Optical Genome Mapping Approach
              </a>
            </td>
          <td>
            Jung Yoon, Taesung Jeon, J. Kwon, Soo-Young Yoon
          </td>
          <td>2024-09-20</td>
          <td>Blood Cancer Journal</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Diffuse large B cell lymphomas (DLBCL) are highly aggressive tumors. Their genetic complexity and heterogeneity have hampered the development of novel approaches for precision medicine. Our study aimed to develop a personalized therapy for DLBCL by utilizing the CRISPR/Cas system to induce knockouts (KO) of driver genes, thereby causing cancer cell death while minimizing side effects. We focused on OCI-LY3 cells, modeling DLBCL, and compared them with BJAB cells as controls. Analysis of whole exome sequencing revealed significant mutations in genes like PAX5, CD79B, and MYC in OCI-LY3 cells. CRISPR/Cas9-mediated KO of these genes resulted in reduced cancer cell viability. Subsequent single and dual gRNA targeting of PAX5 mutations inhibited proliferation specifically in OCI-LY3 cells. Moreover, dual gRNA targeting of PAX5 and MYC induced chromosomal rearrangements, reducing cell proliferation substantially. However, targeting single intronic mutations did not affect cell viability, highlighting the importance of disrupting protein function. Targeting multiple mutations simultaneously addresses intra-tumoral heterogeneity, and the transient delivery of CRISPR/Cas9 allows for permanent gene disruption. While challenges such as incomplete editing efficiency and delivery limitations exist, further optimization may enhance therapeutic efficacy. Overall, our findings demonstrate the efficacy of CRISPR/Cas9 in targeting oncogenic mutations, opening avenues for precision medicine in DLBCL treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6638da8f9e731128d18c1e54c6964f784156f835" target='_blank'>
              Targeting DLBCL by mutation-specific disruption of cancer-driving oncogenes
              </a>
            </td>
          <td>
            Najmeh Heshmatpour, S. M. Kazemi, Niklas D. Schmidt, Sarita R. Patnaik, Patrick Korus, Bodo G. C. Wilkens, Arturo Macarrón Palacios
          </td>
          <td>2024-10-14</td>
          <td>Frontiers in Genome Editing</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) poses significant challenges in veterinary medicine, with limited treatment options and poor survival rates. While substantial progress has been made in characterizing human AML, translating these advancements to veterinary practice has been hindered by limited molecular understanding and diagnostic tools. The case study presented illustrates the application of whole genome sequencing in diagnosing AML in a dog, showcasing its potential in veterinary oncology. Our approach facilitated comprehensive genomic analysis, identifying mutations in genes that may be associated with AML pathogenesis in dogs, such as KRAS, IKZF1, and RUNX1. However, without supportive evidence of its clinical utility (eg, association with response to treatment or prognosis), the information is limited to exploration. This article reviews the comparative features of canine AML with human AML and discusses strategies to shrink the knowledge gap between human and veterinary medicine with cost-effective next-generation sequencing (NGS) techniques. By utilizing these approaches, the unique and shared molecular features with human AML can be identified, aiding in molecular classification and therapeutic development for both species. Despite the promise of NGS, challenges exist in implementing it into routine veterinary diagnostics. Cost considerations, turnaround times, and the need for robust bioinformatics pipelines and quality control measures must be addressed. Most importantly, analytical and clinical validation processes are essential to ensure the reliability and clinical utility of NGS-based assays. Overall, integrating NGS technologies into veterinary oncology holds great potential for advancing our understanding of AML and improving disease stratification, in hopes of improving clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8bd6fdd9b3631dd5e67d1617f989917515a9291" target='_blank'>
              Advancements in genetic analysis: Insights from a case study and review of next-generation sequencing techniques for veterinary oncology applications.
              </a>
            </td>
          <td>
            R. Harris, Jillian Nolan, Dylan Ammons, Samantha Beeson, Douglas Thamm, Anne Avery
          </td>
          <td>2024-10-04</td>
          <td>Veterinary clinical pathology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="OBJECTIVES
The TP53 mutation, a prevalent tumor suppressor gene alteration, is linked to chemotherapy resistance, increased relapse rates and diminished overall survival (OS) in acute myeloid leukemia (AML) patients.


METHODS
In this study, we characterize the TP53 mutation phenotypes across various AML cohorts utilizing The Cancer Genome Atlas (TCGA) data. We devised a TP53-related prognostic signature derived from differentially expressed genes between mutated and wild-type TP53 AML specimens. In-depth analyses were conducted, encompassing genetic variation, immune cell infiltration and prognostic stratification.


RESULTS
A six-gene TP53-related signature was established using least absolute shrinkage and selection operator (LASSO)-Cox regression, demonstrating robust prognostic predictability. This signature exhibited strong performance in both the OHSU validation cohorts, an independent Gene Expression Omnibus (GEO) validation cohort (GSE71014) and proved by results of the in vivo experiment. Finally, we used single cell database (GSE198681) to observe the characteristics of these six genes.


DISCUSSION
Our study may facilitate the development of efficacious therapeutic approaches and provide a novel idea for future research. Conclusion: The TP53-related signature and pattern hold the potential to refine prognostic stratification and underscore emerging targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b57809ff1c4369159931bf563a0e583afd0ffd0c" target='_blank'>
              Elucidating the immune landscape and potential prognostic model in acute myeloid leukemia with TP53 mutation.
              </a>
            </td>
          <td>
            Gelan Zhu, J. Cai, Wanbin Fu, Yue Sun, Ting Wang, Hua Zhong
          </td>
          <td>2024-09-27</td>
          <td>Hematology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Neuroendocrine transdifferentiation (NEtD), also commonly referred to as lineage plasticity, emerges as an acquired resistance mechanism to molecular targeted therapies in multiple cancer types, predominately occurs in metastatic epidermal growth factor receptor (EGFR)‐mutant non‐small cell lung cancer treated with EGFR tyrosine kinase inhibitors and metastatic castration‐resistant prostate cancer treated with androgen receptor targeting therapies. NEtD tumors are the lethal cancer histologic subtype with unfavorable prognosis and limited treatment. A comprehensive understanding of molecular mechanism underlying targeted‐induced plasticity could greatly facilitate the development of novel therapies. In the past few years, increasingly elegant studies indicated that NEtD tumors share key the convergent genomic and phenotypic characteristics irrespective of their site of origin, but also embrace distinct change and function of molecular mechanisms. In this review, we provide a comprehensive overview of the current understanding of molecular mechanism in regulating the NEtD, including genetic alterations, DNA methylation, histone modifications, dysregulated noncoding RNA, lineage‐specific transcription factors regulation, and other proteomic alterations. We also provide the current management of targeted therapies in clinical and preclinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7693a38c0d6b0d0a5a68e9cf1e1598b353f12ed8" target='_blank'>
              Neuroendocrine transdifferentiation in human cancer: molecular mechanisms and therapeutic targets
              </a>
            </td>
          <td>
            Jun Jiang, Donghui Han, Jiawei Wang, Weihong Wen, Rui Zhang, Weijun Qin
          </td>
          <td>2024-10-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 Multiple myeloma is a plasma cell malignancy that accounts for 1-2% of all cancer diagnoses and 10-15% of hematopoietic neoplasms. Clonal rearrangement of the IG locus suggests a monoclonal model for pathogenesis. The presence of cytogenetic abnormalities in >90% of cases suggests transformation by chromosomal instability and rearrangement. Classification and consensus guidelines by The International Myeloma Working Group (IMWG) and others reflect this by categorizing myeloma based on ploidy, translocations, and chromosomal abnormalities.
 Interestingly, a portion of Emory’s test volume for cancer gene mutation profiling by second generation short-read sequencing is for multiple myeloma cases. This suggests our 75-gene myeloid neoplasm panel (MMP75) is being utilized for medical purposes that are not met by cytogenetic and microarray characterizations performed during the diagnostic workup of multiple myeloma. Inspired by this observation, this study investigates providers motivations for ordering MMP75 and assesses the adequacy of our current implementation for their stated purposes.
 Anecdotes from ordering physicians indicate interest in MMP75 results when informing prognosis, selecting treatments, and evaluating for treatment associated secondary neoplasms. A more formal survey is underway that will statistically capture how the results of MMP75 testing were ultimately used: to inform diagnosis, determine prognosis, guide treatment, explain progression, identify/rule-out secondary neoplasia, or no effect on management.
 We also explore the variants called by MMP75 when performed using bone marrow from patients who received a cytogenetic workup for multiple myeloma at some point. A retrospective analysis of the 5-year interval between Jan 2019 and Jan 2024 identified 156 cases meeting this criterion. The number of pathogenic variants per case ranged between 1 and 8, with mutations detected across 42 genes. The genes most frequently identified with pathogenic mutations were TP53 (22.4%), DNMT3A (19.2%), TET2 (14.7%), and ASXL1 (12.2%). All four have been associated with adverse outcomes in patients with multiple myeloma. Activating events in therapeutic targets included mutations in the RAS/RAF pathway (16.7%), JAK/STAT signaling (2.6%), and the kinase KIT (0.6%).
 While these findings support the use of MMP75 for detection of relevant mutations, the genetic variants identified are not unique to multiple myeloma. Mutations detected in DNMT3A, TET2, and ASXL1 are common in clonal hematopoiesis of indeterminant potential (CHIP) and myeloid neoplasms. Hotspot mutations in TP53 and activating events in the RAS/RAF pathway also occur in various hematologic malignancies. This test therefore detects events that occur in both the plasma cell compartment and other cells of origin. This encumbers distinguishing variants that inform on a patient’s multiple myeloma from events that suggest a secondary neoplasm. Future directions include determining whether the effect of plasma cell enrichment on a mutations variant allele frequency (VAF) in repeat testing can help distinguish true multiple myeloma variants from events in other cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/987b88a74ed80e2065d8c8a3c7edc9ab6f63f8af" target='_blank'>
              Improving the Utility of a Cancer Gene Mutation Panel for Multiple Myeloma Patients
              </a>
            </td>
          <td>
            Lalit Patel, Linsheng Zhang, Geoffrey Smith, David L. Jaye, Thomas Schneider
          </td>
          <td>2024-10-01</td>
          <td>American Journal of Clinical Pathology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The tumour suppressor factor p53 plays an essential role in regulating numerous cellular processes, including the cell cycle, DNA repair, apoptosis, autophagy, cell metabolism and immune response. TP53 is the most commonly mutated gene in human cancers. These mutations are primarily non‑synonymous changes that produce mutant p53 proteins characterized by loss of function, a dominant negative effect on p53 tetramerisation and gain of function (GOF). GOF mutations not only disrupt the tumour‑suppressive activities of p53 but also endow the mutant proteins with new oncogenic properties. Recent studies analysing different pathogenic features of mutant p53 in cancer‑derived cell lines have demonstrated that restoring wild‑type p53, rather than removing GOF mutations, reduces cancer cell growth. These findings suggest that therapeutic strategies for reactivating wild‑type p53 function in cancer cells may bring a greater benefit than approaches halting mutant p53. This approach could involve the use of small molecules, gene therapy and other methods to re‑establish wild‑type p53 activity. This review describes the complexity of the biological activities of different p53 mutants and summarizes the current therapeutic approaches to restore p53 function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e48833d89250389a497918ef135eacb5f7e0504d" target='_blank'>
              TP53 mutations in cancer: Molecular features and therapeutic opportunities (Review).
              </a>
            </td>
          <td>
            M. Tornesello
          </td>
          <td>2024-10-24</td>
          <td>International journal of molecular medicine</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9a93850c21d9d18806483d79ec8a05dda4a648d" target='_blank'>
              Treatment resistance to melanoma therapeutics on a single cell level
              </a>
            </td>
          <td>
            Lijun Yao, B. Krasnick, Ye Bi, Sunantha Sethuraman, S. Goedegebuure, A. Weerasinghe, Christopher Wetzel, Qingsong Gao, A. Oyedeji, Jacqueline L Mudd, Matthew A. Wyczalkowski, M. Wendl, Li Ding, Ryan C. Fields
          </td>
          <td>2024-09-19</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Dysfunction of epigenetic modulators, such as the SWI/SNF complex, is a wide-spread but relatively ill-defined feature of a broad spectrum of cancer entities. Among SWI/SNF-mutant entities, SMARCB1-deficient cancers, such as the highly aggressive Epithelioid Sarcoma (EpS), are characterized by this genetic event in an otherwise rather silent mutational landscape. This renders EpS an ideal model to study how epigenetic reprogramming by a single mutation can contribute to tumorigenesis. Hence, to characterize and compare the function of the SMARCB1-deficient, residual and the physiological SWI/SNF complex in cancer, we generated a panel of SMARCB1 re-expressing EpS cell lines and employed a functional multi-omics approach. Here, we show that SWI/SNF holds canonical characteristics of both tumor-suppressors and proto-oncogenes due to its multi-faceted role in the regulation of the epigenome. Our data indicates that the loss of SMARCB1 causes an overall loss of SWI/SNF chromatin affinity at cis-regulatory enhancer elements, inducing a preference for uncontrolled proliferation and cell cycle progression as opposed to development and differentiation. We further demonstrate that EpS cell lines depend on residual SWI/SNF action to maintain clonogenicity and proliferation. Consequently, EpS cell lines exhibit markedly increased sensitivity to pharmacological inhibition of the residual SWI/SNF when compared with SWI/SNF-proficient cancer entities. Collectively, our results from the EpS model shed new light on how a single mutation can rewire the pleiotropic effects of an epigenetic master regulator and provide inroads for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ba81f1fa7d5881ab59ab9ddb07d9a1d9290af17" target='_blank'>
              Loss of SMARCB1 evokes targetable epigenetic vulnerabilities in Epithelioid Sarcoma
              </a>
            </td>
          <td>
            Jia Xiang Jin, Fabia Fuchslocher, Martha J. Carreño-Gonzalez, Felina Zahnow, A. K. Ceranski, Rainer Will, Dominic Helm, Felix Bestvater, A. Banito, R. Imle, S. Ohmura, F. Cidre-Aranaz, T. G. Grünewald
          </td>
          <td>2024-09-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="
 Advances in preclinical models that recapitulate chemorefractory and relapsed disease are needed to better predict the efficacy of an expanding and promising armamentarium of drug candidates being tested in early-phase pediatric clinical trials. Here, we used longitudinal magnetic resonance imaging to design an individualized, dose-escalating treatment regimen that induces evolution of neuroblastoma in the Th-MYCN genetically-engineered mouse model, concomitant with the acquisition of resistance to temozolomide, a standard chemotherapy used in treatment of refractory, relapsed neuroblastoma patients within European early-phase clinical trials. MRI longitudinally identified the development of intra-tumoral heterogeneity. Molecular profiling of expanding, treatment-refractory regions identified prominent up-regulation of the nor-adrenergic core regulatory signature and deregulation of the CDK2 pathway. Treatment with a CDK2/9 inhibitor led to significant response and an overall survival benefit in temozolomide-resistant Th-MYCN tumors and allografts generated from these resistant tumours. These findings demonstrate the utility of genetically-engineered mouse models as platforms to dissect the evolution of chemoresistance in neuroblastoma and they provide a mechanistic rationale to support the evaluation of CDK2/9 in ongoing paediatric phase I studies of chemotherapy combined with temozolomide in relapsed, treatment refractory neuroblastoma patients.
 Citation Format: Evon Poon, Tong Liang, Giuseppe Barone, Barbara M. Costa, Sara Heuss, Simon P. Robinson, Charles Lin, Paul Workman, Yann Jamin, Louis Chesler. Acquired resistance to temozolomide in the Th-MYCN mouse as a clinically-relevant platform to evaluate novel therapeutic strategies against high-risk neuroblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A066.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6cbf60b6dfa50cf3144b983799fd4b6d993e0d1e" target='_blank'>
              Abstract A066 Acquired resistance to temozolomide in the Th-MYCN mouse as a clinically-relevant platform to evaluate novel therapeutic strategies against high-risk neuroblastoma
              </a>
            </td>
          <td>
            E. Poon, Tong Liang, Giuseppe Barone, Barbara M. Costa, Sara Heuss, Simon P. Robinson, Charles Lin, Paul Workman, Y. Jamin, Louis Chesler
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="The effectiveness of drug treatments is profoundly influenced by individual responses, which are shaped by gene expression variability, particularly within pharmacogenes. Leveraging single-cell RNA sequencing (scRNA-seq) data, our study explores the extent of expression variability among pharmacogenes in a wide array of cell types across eight different human tissues, shedding light on their impact on drug responses. Our findings broaden the established link between variability in pharmacogene expression and drug efficacy to encompass variability at the cellular level. Moreover, we unveil a promising approach to enhance drug efficacy prediction. This is achieved by leveraging a combination of cross-cell and cross-individual pharmacogene expression variation measurements. Our study opens avenues for more precise forecasting of drug performance, facilitating tailored and more effective treatments in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11b12a7bcbec6921ec291037e369d1c317c0110c" target='_blank'>
              Deciphering single-cell gene expression variability and its role in drug response.
              </a>
            </td>
          <td>
            Sizhe Liu, Liang Chen
          </td>
          <td>2024-09-15</td>
          <td>Human molecular genetics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Epidermal growth factor receptor (EGFR) exon 20 insertion (ex20ins) mutations are the third most frequent EGFR mutation type, following only exon 19 deletions and exon 21 L858R point mutations. EGFR ex20ins mutations are found in approximately 4%-12% of all EGFR-positive non-small cell lung cancers (NSCLCs). Unlike classical EGFR mutations, EGFR ex20ins mutations display remarkable subtype diversity and heterogeneity. Patients harboring these mutations generally have an inferior prognosis because of insensitivity to conventional treatment approaches such as immunotherapy, chemotherapy, and targeted therapy. Consequently, there remains a significant unmet medical need for efficacious treatments. Recently, amivantamab and sunvozertinib have demonstrated notable efficacy as first-line treatments, and several other promising novel targeted drugs are also challenging the status quo of traditional first-line platinum-based chemotherapy regimens. These developments are anticipated to further improve survival outcomes for NSCLC patients with EGFR ex20ins mutations. Hence, this review summarizes the epidemiology, molecular attributes, detection methodologies, and therapeutic advancements for EGFR ex20ins mutations in NSCLC, and briefly discusses the mechanisms of drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8ec6ed4ff84d07da29d14bd4fe1520fd9fe16a2" target='_blank'>
              Targeting EGFR Exon 20 Insertion Mutations in Non-small-Cell Lung Cancer: Changes in Treatment Strategies are Coming
              </a>
            </td>
          <td>
            Haoli Liu, Jing Qin, Xinyu Qian
          </td>
          <td>2024-01-01</td>
          <td>Cancer Control : Journal of the Moffitt Cancer Center</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="High expression of the long non-coding RNA (lncRNA) FAM30A has been previously associated with leukemic stem cell (LSC) activity and poor prognosis in both adult and paediatric acute myeloid leukaemia (AML) patients, yet it has not been functionally studied. This study provides the first cellular characterization of FAM30A focussing on an internal tandemly organised region, referred to as FAM30A repeats. FAM30A levels correlated with canonical AML LSC signatures and FAM30A depletion decreased cell viability as well as increased sensitivity to chemotherapeutics. It also inhibited colony formation, promoted granulocytic differentiation and abrogated leukemic engraftment in murine bone marrow in vivo. Overexpression of FAM30A repeats in this setting enhanced stemness, proliferation, chemoresistance, and engraftment thus highlighting the biological relevance of this region for LSC biology. On the molecular level, FAM30A repeats interact with the pro-LSC regulator Musashi-2 (MSI2), positively influencing expression of its targets including RUNX1 isoforms. We herein uncover that this FAM30A-MSI2-RUNX1 regulatory loop is of potential relevance for LSC maintenance in AML. These findings provide valuable insights into FAM30A’s cellular role and highlight its targeting potential for eliminating LSCs and improving treatment outcomes in AML patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e944155596f6ba606bda7e6372bd8e29a902812" target='_blank'>
              The long non-coding RNA FAM30A regulates the Musashi2-RUNX1 axis and is required for LSC function in AML cells
              </a>
            </td>
          <td>
            Jaime Calvo Sánchez, Mary T Scott, Athina Varnava, Tala Alakhras, Alice Wedler, Danny Misiak, N. Bley, C. Ihling, A. Sinz, Karen Keeshan, S. Hüttelmaier, David Vetrie, Marcel Köhn
          </td>
          <td>2024-10-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Radiation is a mainstay of lung cancer treatment; however, resistance frequently develops. Identifying novel therapeutic targets to increase radiation sensitivity is crucial. S6K1 is a serine/threonine kinase known to regulate protein translation which is associated with radioresistance, but the mechanisms involved are unknown. We proposed to determine whether S6K1 promotes radioresistance by regulating DNA repair in lung cancer. Colony formation, protein expression and proliferation were assessed. S6K1 was modulated pharmacologically by either PF-4708671 or genetically by Crispr-Cas9. Higher radioresistance levels in lung cancer cells were associated with lower phosphoactivation of MRN complex members, a key activator of radiation-induced DNA repair signaling. We also found lower levels of p-ATM, a target of the MRN complex, in more radioresistant cells, which was associated with a lower expression of γ-H2AX cafter radiation. Further, genetic and pharmacological S6K1 targeting sensitized lung cancer cells to low doses of radiation (p ≤ 0.01). Additionally, S6K1−/− deletion increased the phosphoactivation of MRN complex members, indicating that S6K1 itself can shut down DNA damage regulated by MRN signaling. This is the first report showing that S6K1 inhibition radiosensitizes lung cancer cells by decreasing MRN complex-regulated DNA repair signaling. Future studies should evaluate the role of S6K1 as a target to overcome radioresistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/623356ed9108e3db105e7395f2b3d04852458dc3" target='_blank'>
              S6K1 Controls DNA Damage Signaling Modulated by the MRN Complex to Induce Radioresistance in Lung Cancer
              </a>
            </td>
          <td>
            Ali Calderon-Aparicio, Jun He, N. Simone
          </td>
          <td>2024-09-28</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Diffuse large B cell lymphoma, not otherwise specified (DLBCL, NOS) is the most common type of non-Hodgkin lymphoma (NHL). Significant efforts have been focused on utilizing advanced genomic technologies to further subclassify DLBCL, NOS into clinically relevant subtypes. These efforts have led to the implementation of novel algorithms to support optimal risk-oriented therapy and improvement in the overall survival of DLBCL patients. The pathogenesis of DLBCL at the molecular level indicates copy number variation (CNV) as one of the major forms of genetic alterations in the somatic mutational landscape. Random deregulation that results in complex breaks of chromosomes and restructuring of shattered chromosomal segments is called chromothripsis. Gene expression changes influenced by chromothripsis have been reported in cancer and congenital diseases. This chaotic phenomenon results in complex CNV, gene fusions, and amplification and loss of tumor suppressor genes. We present herein a summary of the most clinically relevant genomic aberrations, with particular focus on copy number aberrations in a case that highlights DLBCL, NOS arising from relapsed Hodgkin lymphoma. The focus of our study was to understand the relationship between the clinical, morphological, and genomic abnormalities in DLBCL, NOS through multiple techniques for therapeutic considerations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e29495d2020d233538184aa5ca4d900590a2a9ff" target='_blank'>
              Deregulation and Shattering of Chromosomal Segments Containing Multiple Oncogenic Targets in the Pathogenesis of Diffuse Large B Cell Lymphoma, Not Otherwise Specified (DLBCL, NOS)
              </a>
            </td>
          <td>
            A. Yenamandra, Rebecca B. Smith, Adam C. Seegmiller, Brianna N. Smith, Debra L. Friedman, C. M. Smith
          </td>
          <td>2024-09-18</td>
          <td>DNA</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b06a2eaa2b57ddf94f1c14246c5af32dbe827fd8" target='_blank'>
              Decoding the epitranscriptome: a new frontier for cancer therapy and drug resistance
              </a>
            </td>
          <td>
            Lu Tang, Hua Tian, Qi Min, Huili You, Mengshuang Yin, Liqiong Yang, Yueshui Zhao, Xu Wu, Mingxing Li, F. Du, Yu Chen, Shuai Deng, Xiaobing Li, Meijuan Chen, Li Gu, Yuhong Sun, Zhangang Xiao, Wanping Li, Jing Shen
          </td>
          <td>2024-10-21</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00f59acd0b5bfd043cfd54293ab6200061c06ac4" target='_blank'>
              Integrated analysis of single-cell RNA-seq and bulk RNA-seq revealed key genes for bone metastasis and chemoresistance in prostate cancer.
              </a>
            </td>
          <td>
            Hongai Bai, Zhenyue Li, Yueyue Weng, Facai Cui, Wenpu Chen
          </td>
          <td>2024-10-12</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Amino-terminal (Nt-) acetylation (NTA) is a common protein modification, affecting 80% of cytosolic proteins in humans. The human essential gene, NAA10, encodes the enzyme NAA10, as the catalytic subunit for the N-terminal acetyltransferase A (NatA) complex, including the accessory protein, NAA15. The first human disease directly involving NAA10 was discovered in 2011, and it was named Ogden syndrome (OS), after the location of the first affected family residing in Ogden, Utah, USA. Since that time, other variants have been found in NAA10 and NAA15. Here we describe the generation of 31 iPSC lines, with 16 from females and 15 from males. This cohort includes CRISPR-mediated correction to the wild-type genotype in 4 male lines, along with editing one female line to generate homozygous wild-type or mutant clones. Following the monoclonalizaiton and screening for X-chromosome activation status in female lines, 3 additional pairs of female lines, in which either the wild type allele is on the active X chromosome (Xa) or the pathogenic variant allele is on Xa, have been generated. Subsets of this cohort have been successfully used to make cardiomyocytes and neural progenitor cells (NPCs). These cell lines are made available to the community via the NYSCF Repository.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eab1adf8656f9aef233d0cf25ef9cb89566ccecd" target='_blank'>
              A repository of Ogden syndrome patient derived iPSC lines and isogenic pairs by X-chromosome screening and genome-editing.
              </a>
            </td>
          <td>
            Josephine Wesely, Tom Rusielewicz, Yu-Ren Chen, Brigham Hartley, Dayna McKenzie, Matthew K. Yim, Colin Maguire, Ryan Bia, Sarah Franklin, Rikhil Makwana, E. Marchi, Manali Nikte, Soha Patil, Maria Sapar, Dorota Moroziewicz, Lauren Bauer, Jeannie T. Lee, Frederick J. Monsma, Dan Paull, G. Lyon
          </td>
          <td>2024-09-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="
 Background: Dynamic regulation of gene expression is fundamental for cellular adaptation to exogenous stressors. PTEFb-mediated promoter proximal pause-release of Pol II is a conserved regulatory mechanism for synchronous transcriptional induction best described in response to heat shock, but this pro-survival role has not been examined in the applied context of cancer therapy. Design/Method: In order to examine the dynamics of chromatin reorganization following radiotherapy, we performed a combination of ChIP-, ATAC-, and RNA-seq in model systems of diffuse intrinsic pontine glioma (DIPG) and other pediatric high-grade gliomas (pHGG) following IR exposure. We interrogated IR-induced gene expression in the presence or absence of PTEFb blockade, including both mechanistic and functional consequences of concurrent inhibition or genetic depletion. We utilized culture models with live cell imaging to assess the therapeutic synergy of PTEFb inhibition with IR, as well as the therapeutic index of this intervention relative to normal controls. Finally, we employed orthotopic models of pHGG treated with conformal radiotherapy and CNS-penetrant PTEFb inhibitors in order to assess tolerability and anti-tumor effect in vivo. Results: Rapid genome-wide redistribution of active chromatin features and PTEFb facilitates Pol II pause-release to drive nascent transcriptional induction within hours of exposure to therapeutic ionizing radiation. Concurrent inhibition of PTEFb imparts a transcription elongation defect, abrogating canonical adaptive programs such as DNA damage repair and cell cycle regulation. This combination demonstrates a potent, synergistic therapeutic potential agnostic of glioma subtype, leading to a marked induction of tumor cell apoptosis and prolongation of xenograft survival.Conclusion: These studies reveal a central role for PTEFb underpinning the early adaptive response to radiotherapy, opening new avenues for combinatorial treatment in these lethal malignancies.
 Citation Format: Faye M. Walker, Lays Martin Sobral, Etienne Danis, Bridget Sanford, Ilango Balakrishnan, Dong Wang, Angela Pierce, Sana Karam, Natalie J. Serkova, Nicholas K. Foreman, Sujatha Venkataraman, Robin Dowell, Rajeev Vibhakar, Nathan A. Dahl. Rapid PTEFb-dependent transcriptional reorganization underpins the glioma adaptive response to radiotherapy [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B078.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29cbc389c63c005639d0ccef6f75bb33d60a2a03" target='_blank'>
              Abstract B078: Rapid PTEFb-dependent transcriptional reorganization underpins the glioma adaptive response to radiotherapy
              </a>
            </td>
          <td>
            Faye Walker, L. M. Sobral, Etienne P. Danis, Bridget Sanford, Ilango Balakrishnan, Dong Wang, Angela M Pierce, Sana D Karam, Natalie J Serkova, Nicholas K. Foreman, S. Venkataraman, Robin Dowell, Rajeev Vibhakar, N. Dahl
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Background Epigenetics denotes heritable alterations in gene expression patterns independent of changes in DNA sequence. Epigenetic therapy seeks to reprogram malignant cells to a normal phenotype and has been extensively investigated in oncology. This study conducts a bibliometric analysis of epigenetic therapy in cancer, providing a comprehensive overview of current research, identifying trends, and highlighting key areas of investigation. Methods Publications concerning epigenetic inhibitors in cancer spanning 2004 to 2023 were retrieved from the Web of Science Core Collection (WoSCC). Co-occurrence analysis using VOSviewer assessed current status and focal points. Evolutionary trends and bursts in the knowledge domain were analyzed using CiteSpace. Bibliometrix facilitated topic evolution and revealed trends in keywords. National, institutional, and author affiliations and collaborations were also examined. Results A total of 2,153 articles and reviews on epigenetic therapy in oncology were identified, demonstrating a consistent upward trend over time. The United States (745 papers), University of Texas MD Anderson Cancer Center (57 papers), and Stephen B. Baylin (27 papers) emerged as the most productive country, institution, and author, respectively. Keyword co-occurrence analysis identified five primary clusters: tumor, DNA methylation, epigenetic therapy, expression, and immunotherapy. In the past 5 years, newly emerging themes with increased centrality and density include “drug resistance,” “immunotherapy,” and “combination therapy.” The most cited publication reviewed current understanding of potential causes of epigenetic diseases and proposed future therapeutic strategies. Conclusion In the past two decades, the importance of epigenetic therapy in cancer research has become increasingly prominent. The United States occupies a key position in this field, while China, despite having published a large number of related papers, still has relatively limited influence. Current research focuses on the “combination therapy” of epigenetic drugs. Future studies should further explore the sequencing and scheduling of combination therapies, optimize trial designs and dosing regimens to improve clinical efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df9baed6fff272ce0f1cbc502a348d307f715a03" target='_blank'>
              Research hotspots and trends of epigenetic therapy in oncology: a bibliometric analysis from 2004 to 2023
              </a>
            </td>
          <td>
            Sisi Li, Xinrui Liang, Qing Shao, Guanwen Wang, Yuxin Huang, Ping Wen, Dongping Jiang, Xiaohua Zeng
          </td>
          <td>2024-09-12</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Saccharomyces cerevisiae is an excellent model to study the effect of external cues on cell division and stress response. 5-Fluorocuracil (5-FU) has been used to treat solid tumors since several decades. The drug was initially designed to interfere with DNA replication but was later found to exert its antiproliferative effect also via RNA-dependent processes. Since 5-FU inhibits the activity of the 3'-5'-exoribonuclease Rrp6 in yeast and mammals, earlier work has compared the effect of 5-FU treatment and RRP6 deletion at the transcriptome level in diploid synchronized yeast cells. To facilitate interpreting the expression data we have developed an improved 5-Fluorouracil RNA (5-FUR) expression viewer. Users can access information via genome coordinates and systematic or standard names for mRNAs and Xrn1-dependent-, stable-, cryptic-, and meiotic unannotated transcripts (XUTs, SUTs, CUTs, and MUTs). Normalized log2-transformed or linear data can be displayed as filled diagrams, line graphs or color-coded heatmaps. The expression data are useful for researchers interested in processes such as cell cycle regulation, mitotic repression of meiotic genes, the effect of 5-FU treatment and Rrp6 deficiency on the transcriptome and expression profiles of sense/antisense loci that encode overlapping transcripts. The viewer is accessible at http://5fur.genouest.org.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ad442f4d43c5032d0d97fedb33e587d2cd06478" target='_blank'>
              The 5-Fluorouracil RNA Expression Viewer (5-FUR) Facilitates Interpreting the Effects of Drug Treatment and RRP6 Deletion on the Transcriptional Landscape in Yeast.
              </a>
            </td>
          <td>
            Ugo Szachnowski, Oliver Sallou, Matéo Boudet, Anthony Bretaudeau, M. Wéry, Antonin Morillon, Michael Primig
          </td>
          <td>2024-09-30</td>
          <td>Yeast</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="
 Mutations in DNA are a known cause of cancer. However, they become oncogenic only under specific cellular conditions and within certain tissue contexts. Tissue injury often leads to the aberrant activation of developmental pathways, eroding markers of cellular differentiation and giving rise to transient cellular states. These states resemble developmental progenitors, characterized by higher transcriptional plasticity. Oncogenes, particularly KRAS, co-opt cellular plasticity by acting on tissue-specific enhancers, preventing the resolution of acinar plasticity and maintaining the pancreas in a pro-inflammatory phase that progresses to cancer1,2. However, the specific role of tissue-specific enhancers in pancreas regeneration and cancer initiation remains unexplored, and strategies to target these regenerative processes are yet to be developed. Our laboratory investigates non-mutational mechanisms of cancer, with a special interest in the intersection of developmental biology and cancer, aiming to elucidate the processes of cellular plasticity and cancer initiation. Recent studies have shown that acinar cells possess remarkable cellular plasticity, giving rise to a diverse population of cellular states in pancreas regeneration and cancer contexts3-5. We have discovered that the transcription factor Sox4 is necessary for the specification of tuft cells and gastric metaplasia found in chronic pancreatitis and in pancreatic intraepithelial lesions (PanINs). Moreover, we have shown that cellular plasticity has a non-cell autonomous protective effect; losing Sox4 and cellular heterogeneity in PanINs results in a fibrotic and immunosuppressive tumor microenvironment and disease progression. Our results demonstrated that targeting cellular plasticity accelerates or prevents cancer initiation through autonomous and non-autonomous cellular processes6. In this study, we leverage a comprehensive resource of ncRNAs transcribed from enhancer elements of pancreas development, together with their inferred target genes and correlations with clinical outcomes. We focused on an ncRNA associated with the risk of pancreatic cancer. To study its function, we developed two novel mouse strains for tissue-specific deletion, enabling precise manipulation and study of its role in the pancreas and other tissues. Our phenotypic analysis demonstrates that the ncRNA is activated following tissue injury and is essential for developing PanINs. Moreover, we show that it regulates cell-cell communication after tissue injury. This mechanism operates independently of cellular plasticity but significantly creates a pro-inflammatory environment that favors cancer development. Our results demonstrate that enhancer elements are actionable targets to regulate cell fate decisions in regeneration and cancer. Additionally, KRAS-induced cancer initiation depends on developmental regulatory elements, opening the window to develop prophylactic interventions to delay tumor onset.
 Citation Format: Luis Arnes. Regulatory Elements of Pancreas Development License the Initiation of Pancreatic Ductal Adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C059.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/698dc456f639e681131533a5b64aac4181ad1d22" target='_blank'>
              Abstract C059: Regulatory Elements of Pancreas Development License the Initiation of Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Luis Arnes
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a2dde0164af0025d5f19bf691294f177add4f7b" target='_blank'>
              Low-pass whole genome sequencing of circulating tumor cells to evaluate chromosomal instability in triple-negative breast cancer
              </a>
            </td>
          <td>
            S. Di Cosimo, M. Silvestri, C. De Marco, Alessia Calzoni, M. C. De Santis, Maria Grazia Carnevale, C. Reduzzi, Massimo Cristofanilli, Vera Cappelletti
          </td>
          <td>2024-09-03</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="The human fungal pathogen Candida albicans poses a significant burden on global health, causing high rates of mortality and antifungal drug resistance. C. albicans is a heterozygous diploid organism that reproduces asexually. Structural variants (SVs) are an important source of genomic rearrangement, particularly in species that lack sexual recombination. To comprehensively investigate SVs across clinical isolates of C. albicans, we conducted long read sequencing and genome-wide SV analysis in three distantly related clinical isolates. Our work included a new, comprehensive analysis of transposable element (TE) composition, location and diversity. SVs and TEs are frequently close to coding sequences and many SVs are heterozygous, suggesting that SVs might impact gene and allele-specific expression. Most SVs are uniquely present in only one clinical isolate, indicating that SVs represent a significant source of intra-species genetic variation. We identified multiple, distinct SVs at the centromeres of Chromosome 4 and Chromosome 5, including inversions and transposon polymorphisms. These two chromosomes are often aneuploid in drug resistant clinical isolates, and can form isochromosome structures with breakpoints near the centromere. Further screening of 100 clinical isolates confirmed the widespread presence of centromeric SVs in C. albicans, often appearing in a heterozygous state, indicating that SVs are contributing to centromere evolution in C. albicans Together, these findings highlight that SVs and TEs are common across diverse clinical isolates of C. albicans and that the centromeres of this organism are important sites of genome rearrangement.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d91c6400a0a9a35ce915f65f0f15c27e6b526ae9" target='_blank'>
              Candida albicans isolates contain frequent heterozygous structural variants and transposable elements within genes and centromeres.
              </a>
            </td>
          <td>
            Ursula Oggenfuss, R. Todd, Natthapon Soisangwan, Bailey Kemp, Alison Guyer, Annette Beach, Anna Selmecki
          </td>
          <td>2024-10-22</td>
          <td>Genome research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Direct lineage reprogramming into dopaminergic (DA) neurons holds great promise for the more effective production of DA neurons, offering potential therapeutic benefits for conditions such as Parkinson's disease. However, the reprogramming pathway for fully reprogrammed DA neurons remains largely unclear, resulting in immature and dead-end states with low efficiency. In this study, using single-cell RNA sequencing, the trajectory of reprogramming DA neurons at multiple time points, identifying a continuous pathway for their reprogramming is analyzed. It is identified that intermediate cell populations are crucial for resetting host cell fate during early DA neuronal reprogramming. Further, longitudinal dissection uncovered two distinct trajectories: one leading to successful reprogramming and the other to a dead end. Notably, Arid4b, a histone modifier, as a crucial regulator at this branch point, essential for the successful trajectory and acquisition of mature dopaminergic neuronal identity is identified. Consistently, overexpressing Arid4b in the DA neuronal reprogramming process increases the yield of iDA neurons and effectively reverses the disease phenotypes observed in the PD mouse brain. Thus, gaining insights into the cellular trajectory holds significant importance for devising regenerative medicine strategies, particularly in the context of addressing neurodegenerative disorders like Parkinson's disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42bfa61a76443945c179bc5c94c72de94ea5c5df" target='_blank'>
              Epigenetic Dynamics in Reprogramming to Dopaminergic Neurons for Parkinson's Disease.
              </a>
            </td>
          <td>
            Byounggook Cho, Junyeop Kim, Sumin Kim, Saemin An, Yerim Hwang, Yunkyung Kim, Daeyeol Kwon, Jongpil Kim
          </td>
          <td>2024-09-16</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background: Patients with hormone receptor-positive (HR+)/HER2- metastatic breast cancer (mBC) generally receive hormonal therapy (HT) combined with CDK4/6 inhibitors (CDK4/6i). Despite this treatment, resistance mechanisms to CDK4/6i emerge and the majority of these patients experience disease progression (PD). This highlight the necessity to uncover the resistance mechanism to CDK4/6i through the identification of specific biomarkers. The primary objective is to assess the accuracy and feasibility of a novel multi-gene target panel NGS assay on circulating tumor DNA (ctDNA) to detect molecular alterations of AKT1, ERBB2, ESR1, KRAS, PIK3CA, and TP53 genes in women with BC undergoing HT plus CDK4/6i treatment. Secondarily, the study aims to explore the relationship between genomic profiling and clinical outcomes. Materials and Methods: Plasma samples were collected from 16 patients diagnosed with advanced/locally advanced HR+/HER2- BC at 2 time points: T0 (baseline) and at T1 (3 months after CDK4/6i treatment). Starting from 2 mL of plasma, ctDNA was isolated and libraries were set up using the Plasma-SeqSensei (PQS)® Breast Cancer IVD Kit, sequenced on Nextseq 550 and analyzed using the Plasma-SeqSensei™ IVD Software®. Results: Among the five patients who presented PD, three had PIK3CA mutations and, of these, two showed a higher mutant allele frequency (MAF) at T1. In three patients with stable disease and in eight patients with partial response, the MAF of the detected alterations decreased dramatically or disappeared during CDK4/6i treatment. Conclusions: Based on our findings, the liquid biopsy analysis using the PQS panel seems to be both feasible and accurate, demonstrating a strong sensitivity in detecting mutations. This exploratory analysis of the clinical outcome associated to the mutational status of patients highlights the potential of molecular analysis on liquid biopsy for disease monitoring, although further validation with a larger patient cohort is necessary to confirm these preliminary observations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56cafee5ca185d21631ed12255567f16d464d5e4" target='_blank'>
              A Novel AKT1, ERBB2, ESR1, KRAS, PIK3CA, and TP53 NGS Assay: A Non-Invasive Tool to Monitor Resistance Mechanisms to Hormonal Therapy and CDK4/6 Inhibitors
              </a>
            </td>
          <td>
            A. Virga, C. Gianni, M. Palleschi, D. Angeli, F. Merloni, R. Maltoni, Paola Ulivi, Giovanni Martinelli, U. De Giorgi, S. Bravaccini
          </td>
          <td>2024-09-26</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Abstract Cellular crosstalk mediated by ligand–receptor interactions largely complicates the tumour ecosystem, resulting in heterogeneous tumour microenvironments that affect immune response and clinical benefits from immunotherapy. Epigenetic mechanisms are pivotal to expression changes of immune‐related genes and can modulate the anti‐tumour immune response. However, the functional consequences of disrupted epigenetic regulators (ERs) on ligand–receptor interactions in the tumour microenvironment remain largely unexplored. Here, we proposed mutations of ERs in perturbed interactions (MERIN), a molecular network‐based approach that incorporates multi‐omics data, to infer the potential consequences of ER mutations on ligand–receptor interaction perturbations. Leveraging cancer genomic profiles and molecular interaction data, we comprehensively decoded the functional consequences of ER mutations on dysregulated ligand–receptor interactions across 33 cancers. The dysregulated ligand–receptor genes were indeed enriched in cancer and immune‐related function. We demonstrated the potential significance of PD1–PDL1 interaction‐related ER mutations in stratifying cancer patients from multiple independent data cohorts. The ER mutation group showed distinct immunological characterizations and prognoses. Furthermore, we highlighted that the ER mutations could potentially predict clinical outcomes of immunotherapy. Our computational and clinical assessment underscore the utility of MERIN for elucidating the functional relevance of ER mutations in cancer immune response, potentially aiding patients' stratification for immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f1c94edf4edb1035d9676cb0af0e88e572eaf00" target='_blank'>
              Decoding functional impact of epigenetic regulator mutations on ligand–receptor interaction perturbations for evaluation of cancer immunotherapy
              </a>
            </td>
          <td>
            Aiai Shi, Chaohuan Lin, Jie Lyu
          </td>
          <td>2024-09-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Transcriptional enhancers are non-coding DNA elements that regulate gene transcription in a temporal and tissue-specific manner. Despite advances in computational and experimental methods, identifying enhancers and their target genes essential for specific biological processes remains challenging. Determining target genes for enhancers is also complex and often relies on indirect, low-resolution, and/or assumptive methodologies. To identify and functionally perturb enhancers at their endogenous sites without altering their sequence, we performed a pooled tiling CRISPR activation (CRISPRa) screen surrounding PHOX2B, a master regulator of neuronal cell fate and a key player in neuroblastoma development. This screen allowed the de novo identification of CRISPRa responsive elements (CaREs) that alter cellular growth within the 2 Mb genomic region. To determine CaRE target genes, we developed TESLA-seq (TargEted SingLe cell Activation), which combines CRISPRa screening with targeted single-cell RNA-sequencing and enables the parallel readout of the effect of hundreds of enhancers on all genes in the locus. While most TESLA-revealed CaRE-gene relationships involved neuroblastoma-related regulatory elements already active in the system, we found many CaREs and target connections normally active only in other tissue types or with no previous evidence and induced out of context by CRISPRa. This highlights the power of TESLA-seq to reveal gene regulatory networks, including edges active outside of a given experimental system. Highlights Systematically perturbed regulatory landscape in a 2 Mb genomic region surrounding PHOX2B to identify hundreds of CRISPRa-responsive elements that affect cellular growth Developed TESLA-seq as a principled molecular approach to find gene targets of dozens of candidate regulatory elements Validated interactions between identified regulatory elements and target genes and characterized their genomic features Integrated a compendium of epigenomic datasets to identify regulatory relationships induced out of context or with no previous evidence">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6f4af902573832fb2aa9994e5101eab3eac7003" target='_blank'>
              Sensitive dissection of a genomic regulatory landscape using bulk and targeted single-cell activation
              </a>
            </td>
          <td>
            Dubravka Vučićević, Che-Wei Hsu, Lorena Sofia Lopez Zepeda, M. Burkert, Antje Hirsekorn, Ilija Bilić, Nicolai Kastelić, M. Landthaler, S. Lacadie, U. Ohler
          </td>
          <td>2024-09-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="
 Cancers evolve not only through the acquisition and clonal transmission of somatic mutations but also by non-genetic mechanisms that modify cell phenotype. Such epigenetic causes of intra-tumoral heterogeneity may be particularly important in pediatric cancers with low rates of genomic mutations. Hepatoblastoma, the most common primary liver malignancy in children, has one of the lowest mutational burdens of all cancers, providing an ideal system to elucidate non-genetic mechanisms of intra-tumoral heterogeneity. Up to 80-90% of hepatoblastomas have activating mutations in CTNNB1 and the Wnt pathway. Yet, despite their simple genomics, these tumors exhibit significant histologic heterogeneity, including epithelial (embryonal and fetal histology) and mesenchymal components. Using transcriptomics and patient-derived tumoroids, we find that hepatobiliary lineage programs modulate the transcriptional outcome of mutated CTNNB1. A subset of cholangiocytic-like tumor cells express the growth factor FGF19, which promotes proliferation of neighboring cells in conjunction with activated Wnt signaling. To characterize the transcriptional profiles of the distinct histologic components of hepatoblastoma, we used laser capture microdissection to isolate tumor regions identified as embryonal, fetal, or mesenchymal histology, as well as adjacent normal liver. While all tumor regions showed increased expression of Wnt target genes as compared to normal liver, the embryonal components showed even higher expression of Wnt target genes as compared to the fetal components, as well as increased proliferation markers and higher expression of the embryonic biliary transcription factor SOX4. RNA in situ hybridization in primary hepatoblastomas revealed distinctive foci of embryonal hepatoblastoma cells that express the growth factor FGF19. These FGF19-expressing cells were surrounded by FGF19-negative cells that expressed KLB, the co-receptor for FGF19 signaling, and showed an increased proliferation. We therefore hypothesized that FGF19 acts in a paracrine fashion to drive cell cycle progression of hepatoblastoma cells in cooperation with constitutively active Wnt signaling. We further generated patient-derived tumoroids to systematically test the requirements for exogenous growth factors. We found that despite activated Wnt signaling, FGF19-negative tumor cells required additional exogenous growth factors to proliferate. In some tumoroids, subsets of cells expressed FGF19 endogenously, downstream of Wnt/beta-catenin and SOX4, acting as a paracrine signal. We conclude that the embryonic biliary transcription factor SOX4 cooperates with mutated beta-catenin, inducing FGF19 as a paracrine growth signal that can promote proliferation of FGF19-negative tumor cells. Thus, in this pediatric cancer presumed to originate from a multipotent hepatobiliary progenitor, lineage-driven heterogeneity results in a functional growth advantage, a non-genetic mechanism whereby developmental lineage programs influence tumor evolution, with implications for treatment.
 Citation Format: Peng V. Wu, Matt Fish, Florette K. Hazard, Chunfang Zhu, Sujay Vennam, Hannah Walton, Joanna Przybyl, Maurizio Morri, Norma Neff, Robert B. West, Roel Nusse. Single cell transcriptomics and patient-derived tumoroids reveal that hepatobiliary lineage programs cooperate with Wnt pathway mutations to drive cell proliferation in hepatoblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B024.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da0809899c0e27593ad23e793e639268c8de6597" target='_blank'>
              Abstract B024: Single cell transcriptomics and patient-derived tumoroids reveal that hepatobiliary lineage programs cooperate with Wnt pathway mutations to drive cell proliferation in hepatoblastoma
              </a>
            </td>
          <td>
            Peng V. Wu, Matt Fish, F. Hazard, Chunfang Zhu, S. Vennam, Hannah Walton, Joanna Przybyl, Maurizio Morri, Norma Neff, Robert B. West, Roel Nusse
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Molecular profiles of thymomas and recurrent thymomas are far from being defined. Herein, we report an analysis of a comprehensive genetic profile (CGP) in a highly selected cohort of recurrent thymomas. Among a cohort of 426 thymomas, the tissue was available in 23 recurrent tumors for matching the biomolecular results obtained from primary and relapse samples. A control group composed of non-recurrent thymoma patients was selected through a propensity score match analysis. CGP was performed using the NGS Tru-SightOncology assay to evaluate TMB, MSI, and molecular alterations in 523 genes. CGP does not differ when comparing initial tumor with tumor relapse. A significantly higher frequency of cell cycle control genes alterations (100.0% vs. 57.1%, p = 0.022) is detected in patients with early recurrence (<32 months) compared to late recurrent cases. The CGPs were similar in recurrent thymomas and non-recurrent thymomas. Finally, based on NGS results, an off-label treatment or clinical trial could be potentially proposed in >50% of cases (oncogenic Tier-IIC variants). In conclusion, CGPs do not substantially differ between initial tumor vs. tumor recurrence and recurrent thymomas vs. non-recurrent thymomas. Cell cycle control gene alterations are associated with an early recurrence after thymectomy. Multiple target therapies are potentially available by performing a comprehensive CGP, suggesting that a precision medicine approach on these patients could be further explored.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1342cb4ef183c5952433e74e938198b53fb109e" target='_blank'>
              Comparative Analysis of Comprehensive Genomic Profile in Thymomas and Recurrent Thymomas Reveals Potentially Actionable Mutations for Target Therapies
              </a>
            </td>
          <td>
            F. Lococo, E. De Paolis, Jessica Evangelista, A. Dell’Amore, Diana Giannarelli, M. Chiappetta, Annalisa Campanella, C. Sassorossi, Alessandra Cancellieri, Fiorella Calabrese, Alessandra Conca, E. Vita, Angelo Minucci, Emilio Bria, Angelo Castello, Andrea Urbani, Federico Rea, S. Margaritora, G. Scambia
          </td>
          <td>2024-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Aging is an important risk factor for tumorigenesis. Metabolic reprogramming is a hallmark of both aging and tumor initiation. However, the manner in which the crosstalk between aging and metabolic reprogramming affects the tumor microenvironment (TME) to promote tumorigenesis was poorly explored. We utilized a computational approach proposed by our previous work, MMP3C (Modeling Metabolic Plasticity by Pathway Pairwise Comparison), to characterize aging-related metabolic plasticity events using pan-cancer bulk RNA-seq data. Our analysis revealed a high degree of metabolically organized heterogeneity across 17 aging-related cancer types. In particular, a higher degree of several energy generation pathways, i.e., glycolysis and impaired oxidative phosphorylation, was observed in older patients. Similar phenomena were also found via single-cell RNA-seq analysis. Furthermore, those energy generation pathways were found to be weakened in activated T cells and macrophages, whereas they increased in exhausted T cells, immunosuppressive macrophages, and Tregs in older patients. It was suggested that aging-induced metabolic switches alter glucose utilization, thereby influencing immune function and resulting in the remodeling of the TME. This work offers new insights into the associations between tumor metabolism and the TME mediated by aging, linking with novel strategies for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c92f52b4b6e5f7172ec007a7f5c79b9ac1800330" target='_blank'>
              Metabolic Reprogramming Induced by Aging Modifies the Tumor Microenvironment
              </a>
            </td>
          <td>
            Xingyu Chen, Zihan Wang, Bo Zhu, Min Deng, Jiayue Qiu, Yunwen Feng, Ning Ding, Chen Huang
          </td>
          <td>2024-10-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Non-coding regulatory sequences play essential roles in adjusting gene output to cellular needs and are thus critical to animal development and health. Numerous such sequences have been identified in mammalian genomes ranging from transcription factors binding motifs to recognition sites for RNA-binding proteins and non-coding RNAs. The advent of CRISPR has raised the possibility of assigning functionality to individual endogenous regulatory sites by facilitating the generation of isogenic cell lines that differ by a defined set of genetic modifications. Here we investigate the usefulness of this approach to assign function to individual miRNA binding sites. We find that the process of generating isogenic pairs of mammalian cell lines with CRISPR-mediated mutations introduces extensive molecular and phenotypic variability between biological replicates making any attempt of assigning function to the binding site essentially impossible. Our work highlights an important consideration when employing CRISPR editing to characterize non-coding regulatory sequences in cell lines and calls for the development and adoption of alternative strategies to address this question in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f1d3a055f759b974342766678fe87bc5f92b153" target='_blank'>
              CRISPR-based dissection of miRNA binding sites using isogenic cell lines is hampered by pervasive noise
              </a>
            </td>
          <td>
            Mahendra Prajapat, Joana A. Vidigal
          </td>
          <td>2024-09-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Simple Summary The therapeutic landscape of Chronic Lymphocytic Leukemia (CLL) has dramatically changed in recent years, with a shift from chemoimmunotherapy towards many new targeted agents, such as Bruton Tyrosine Kinase inhibitors (BTKi) and anti-BCL-2. We now need an accurate re-definition of prognostic/predictive parameters to support clinicians in choosing the more appropriate option. This review explores recent advances in the molecular biology of chronic lymphocytic leukemia, with the aim of identifying the most important biomarkers to define prognosis and to guide first-line and second-line treatment. Abstract Chronic Lymphocytic Leukemia (CLL) is the most frequent type of leukemia in Western countries. In recent years, there have been important advances in the knowledge of molecular alterations that underlie the disease’s pathogenesis. Very heterogeneous prognostic subgroups have been identified by the mutational status of immunoglobulin heavy variable genes (IGVH), FISH analysis and molecular evaluation of TP53 mutations. Next-generation sequencing (NGS) technologies have provided a deeper characterization of the genomic and epigenomic landscape of CLL. New therapeutic targets have led to a progressive reduction of traditional chemoimmunotherapy in favor of specific biological agents. Furthermore, in the latest clinical trials, the minimal residual disease (MRD) has emerged as a potent marker of outcome and a guide to treatment duration. This review focuses on recent insights into the understanding of CLL biology. We also consider the translation of these findings into the development of risk-adapted and targeted therapeutic approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3c13c68ecb506ccc3f3b3f0d7d44f962adc11fb" target='_blank'>
              Recent Advances in the Molecular Biology of Chronic Lymphocytic Leukemia: How to Define Prognosis and Guide Treatment
              </a>
            </td>
          <td>
            A. Arcari, Lucia Morello, Elena Borotti, Elena Ronda, Angela Rossi, Daniele Vallisa
          </td>
          <td>2024-10-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) remains the most lethal cancer and will soon be the second most common cause of cancer related death. While regimens containing DNA damaging agents such as FOLFIRINOX and PARP inhibitors have derived clinical benefits for some patients, their efficacy invariably fails over time. This presents a significant clinical challenge, and thus there is an urgent need for novel therapeutic strategies which are able to overcome the acquisition of resistance in PDAC. Clinically relevant models of treatment resistance were generated from patient-derived cell lines by extended exposure to chemotherapy agents. Synergy scoring, clonogenicity assays, flow cytometry, immunofluorescence and transcriptomic analysis were used to investigate the efficacy of combined ATR and PARP inhibition in re-sensitising resistant PDAC to treatment. Acquisition of resistance was associated with transcriptomic shifts in cell cycle checkpoint regulation, metabolic control, DNA damage response (DDR), programmed cell death and the replication stress response. Additionally, combined treatment with the ATR inhibitor (ceralasertib), and the PARP inhibitor (olaparib) was synergistic in all models of acquired resistance. Sequential treatment using ceralasertib prior to olaparib was highly effective at low dose for DDR proficient cell lines, whereas DDR deficient models responded better when treated with olaparib first. We provide in vitro evidence of a novel therapeutic strategy to overcome acquired PARP inhibitor and platinum resistance in PDAC by using sequential exposure to ceralasertib and Olaparib. A sequential regimen may be more tolerable and should be investigated clinically to circumvent dose limiting toxicity in concurrent combinations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3162301dd3095937530db76522192901b74cb218" target='_blank'>
              Sequential ATR and PARP Inhibition Overcomes Acquired DNA Damaging Agent Resistance in Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Katharine J. Herbert, Rosie Upstill-Goddard, Stephan B. Dreyer, Selma Rebus, Christian Pilarsky, Debabrata Mukhopadhyay, Christopher J. Lord, Andrew Biankin, F. Froeling, David K. Chang
          </td>
          <td>2024-09-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e206396092ebd32f9a052e89e6643d141dcd6ce" target='_blank'>
              High-resolution and programmable RNA-IN and RNA-OUT genetic circuit in living mammalian cells
              </a>
            </td>
          <td>
            Min Zhang, Xue Zhang, Yongyue Xu, Yanhui Xiang, Bo Zhang, Zhen Xie, Qiong Wu, Chunbo Lou
          </td>
          <td>2024-10-10</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Overexpression of the MYC oncogene, encoding c-MYC protein, contributes to the pathogenesis and drug resistance of acute myeloid leukemia (AML) and many other hematopoietic malignancies. Although standard chemotherapy has predominated in AML therapy over the past five decades, the clinical outcomes and patient response to treatment remain suboptimal. Deeper insight into the molecular basis of this disease should facilitate the development of novel therapeutics targeting specific molecules and pathways that are dysregulated in AML, including fms-like tyrosine kinase 3 (FLT3) gene mutation and cluster of differentiation 33 (CD33) protein expression. Elevated expression of c-MYC is one of the molecular features of AML that determines the clinical prognosis in patients. Increased expression of c-MYC is also one of the cytogenetic characteristics of drug resistance in AML. However, direct targeting of c-MYC has been challenging due to its lack of binding sites for small molecules. In this review, we focused on the mechanisms involving the bromodomain and extra-terminal (BET) and cyclin-dependent kinase 9 (CDK9) proteins, phosphoinositide-Akt-mammalian target of rapamycin (PI3K/AKT/mTOR) and Janus kinase-signal transduction and activation of transcription (JAK/STAT) pathways, as well as various inflammatory cytokines, as an indirect means of regulating MYC overexpression in AML. Furthermore, we highlight Food and Drug Administration (FDA)-approved drugs for AML, and the results of preclinical and clinical studies on novel agents that have been or are currently being tested for efficacy and tolerability in AML therapy. Overall, this review summarizes our current knowledge of the molecular processes that promote leukemogenesis, as well as the various agents that intervene in specific pathways and directly or indirectly modulate c-MYC to disrupt AML pathogenesis and drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1135bf881c820d3eca90352e7790a726703df16e" target='_blank'>
              Targeting Molecular Signaling Pathways and Cytokine Responses to Modulate c-MYC in Acute Myeloid Leukemia.
              </a>
            </td>
          <td>
            Kyle Gu, Harry A. May, Min H. Kang
          </td>
          <td>2024-09-14</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2352f41e08e0411c3c2eeaf443ff948521637f4" target='_blank'>
              Deletion of exons 45 to 55 in the DMD gene: from the therapeutic perspective to the in vitro model
              </a>
            </td>
          <td>
            Javier Poyatos-García, Patricia Soblechero-Martín, A. Liquori, A. López-Martínez, Pilar Maestre, Elisa González-Romero, Rafael P. Vázquez-Manrique, Nuria Muelas, Gema García-García, Jessica Ohana, Virginia Arechavala-Gomeza, J. J. Vilchez
          </td>
          <td>2024-10-01</td>
          <td>Skeletal Muscle</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 While neoantigen-directed therapies have primarily focused on tumor-specific somatic mutations, emerging evidence suggests that some malignancies aberrantly translate regions of the genome outside of annotated open reading frames (ORFs), leading to HLA-I presentation of cryptic peptides. Non-canonical HLA-I bound peptides (ncHLAp) can arise from aberrant translation of genomic elements like 5’ and 3’ untranslated regions (UTRs), long noncoding RNAs (lncRNAs), retained introns, and translation in alternative reading frames. Pancreatic cancer has a low-to-intermediate mutational burden; hence, efforts to broaden the landscape targetable antigens in this disease are greatly needed. Here, we leveraged twelve pancreatic cancer (PDAC) patient-derived organoids (PDOs) to purify and enrich the malignant compartment from low tumor cellularity tumor specimens. PDOs were subjected to extensive genomic (whole genome sequencing) and transcriptomic (RNA-sequencing) profiling to enable mutation calling and HLA typing. We then employed a personalized proteogenomic platform coupled with high-depth immunopeptidomics and empirically identified >90,000 unique PDAC HLA-I-bound peptides (HLAp). We detected HLAp arising from somatic mutations (both missense and frameshift) in a subset of PDAC patients; however, we did not detect any shared mutation-derived neoepitopes in our patient cohort. Moreover, we empirically identified >1,700 ncHLAp, primarily arising from translation of novel unannotated open reading frames (nuORFs), and a substantial proportion of ncHLAp were shared amongst PDAC patients with the appropriate HLA haplotype. We developed a highly stringent pipeline using immunopeptidomics and ribosome-sequencing to investigate translation of nuORFs across a range of healthy tissues, including healthy thymus, and found that >500 ncHLAp exhibit highly-specific cancer-restricted translation. We next investigated the immunogenicity of PDAC-restricted ncHLAp and mutation-derived HLAp using an ex vivo T cell priming and expansion platform. Here we demonstrate that a subset of both mutation-derived and PDAC-restricted ncHLAp harbor robust immunogenicity. ncHLAp-specific T cell receptors (TCRs) were identified from reactive cytotoxic T lymphocytes (CTLs) via single-cell TCR-sequencing, and antigen specificity was confirmed after TCR reconstruction and cloning. Finally, using a newly developed organoid: T cell co-culture platform, we demonstrate that ncHLAp-reactive TCR-redirected T cells exhibit robust cytotoxicity and tumoricidal activity in PDAC, underscoring the translational potential for this novel class of antigens. Collectively, we have shown that cryptic epitopes, arising from aberrant translation in PDAC, represent a particularly promising class of antigens for next generation immune-based therapies.
 Citation Format: Zackery A Ely, Zachary J Kulstad, Gurcan Gunaydin, Sudarsana Addepalli, Eva K Verzani, Jennifer G Abelin, Marta Casarrubios, Karl R Clauser, Xilin Wang, Isabelle Lippincott, Cedric Louvet, Tom Schmitt, Miles Agus, Kevin S Kapner, Connor J Hennessey, James Cleary, Sine R Hadrup, Susan Klaeger, Jennifer Su, Alex M Jaeger, Brian M Wolpin, Srivatsan Raghavan, Eric Smith, Philip D Greenberg, Andrew J Aguirre, Steven A Carr, Tyler Jacks, William A Freed-Pastor. Discovery and therapeutic potential of novel cryptic peptides in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr PR-07.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d04f2440d3a56000134da94c90e982d27b5f5ca1" target='_blank'>
              Abstract PR-07: Discovery and therapeutic potential of novel cryptic peptides in pancreatic cancer
              </a>
            </td>
          <td>
            Zackery A. Ely, Zachary J. Kulstad, Gurcan Gunaydin, Sudarsana Addepalli, Eva K. Verzani, Jennifer G. Abelin, Marta Casarrubios, Karl R Clauser, Xilin Wang, Isabelle Lippincott, Cedric Louvet, Tom Schmitt, Miles Agus, Kevin S. Kapner, Connor J Hennessey, James M. Cleary, S. Hadrup, Susan Klaeger, Jennifer Su, Alex M. Jaeger, B. Wolpin, Srivatsan Raghavan, Eric Smith, Philip D. Greenberg, A. Aguirre, Steven A. Carr, T. Jacks, William A. Freed-Pastor
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Abstract Disruptions in normal development and the emergence of health conditions often result from the malfunction of vital genes in the human body. Decades of scientific research have focused on techniques to modify or substitute defective genes with healthy alternatives, marking a new era in disease treatment, prevention and cure. Recent strides in science and technology have reshaped our understanding of disorders, medication development and treatment recommendations, with human gene and cell therapy at the forefront of this transformative shift. Its primary objective is the modification of genes or adjustment of cell behaviour for therapeutic purposes. In this review, we focus on the latest advances in gene and cell therapy for treating human genetic diseases, with a particular emphasis on FDA and EMA‐approved therapies and the evolving landscape of genome editing. We examine the current state of innovative gene editing technologies, particularly the CRISPR‐Cas systems. As we explore the progress, ethical considerations and prospects of these innovations, we gain insight into their potential to revolutionize the treatment of genetic diseases, along with a discussion of the challenges associated with their regulatory pathways. This review traces the origins and evolution of these therapies, from conceptual ideas to practical clinical applications, marking a significant milestone in the field of medical science.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/605617113766e025c90c18b7b7452f93c13a0842" target='_blank'>
              Gene and cell therapy of human genetic diseases: Recent advances and future directions
              </a>
            </td>
          <td>
            Busra Cetin, Fulya Erendor, Y. E. Eksi, A. Şanlioğlu, S. Sanlioglu
          </td>
          <td>2024-09-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="BACKGROUND
The clinical management of chronic myeloid leukemia (CML) patients requires the identification of the type of BCR::ABL1 transcript at diagnosis and the monitoring of its expression and potential tyrosine kinase inhibitor (TKI) resistance mutations during treatment. Detection of resistant mutation requires transcript type-specific amplification of BCR::ABL1 from RNA.


METHODS
In this study, a custom RNA-based next-generation sequencing (NGS) assay (Dup-Seq BCR::ABL1) that enables (a) the identification of BCR::ABL1 transcript type and (b) the detection of resistance mutations from common and atypical BCR::ABL1 transcript types was developed and validated. The assay design covers BCR exon 1 to ABL1 exon 10 and employs duplicate PCR amplification for error correction. The custom data analysis pipeline enables breakpoint determination and overlapped mutation calling from duplicates, which minimizes the low-level mutation artifacts.


RESULTS
This study demonstrates that this novel assay achieves high accuracy (positive percent agreement (PPA) for fusion: 98.5%; PPA and negative percent agreement (NPA) for mutation at 97.8% and 100.0%, respectively) and sensitivity (limit of detection (LOD) for mutation detection at 3% from 10 000 copies of BCR::ABL1 input).


CONCLUSIONS
The Dup-Seq BCR::ABL1 assay not only allows for the identification of BCR::ABL1 typical and atypical transcript types and accurate and sensitive detection of TKI-resistant mutations but also simplifies molecular testing work flow for the clinical management of CML patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c413e19fd7d542721e914a796a2298f73b3d650" target='_blank'>
              A Novel Next-Generation Sequencing Assay for the Identification of BCR::ABL1 Transcript Type and Accurate and Sensitive Detection of TKI-Resistant Mutations.
              </a>
            </td>
          <td>
            Zhenyu Yan, Lin Shi, Wei Li, Weihua Liu, C. Galderisi, Cynthia Spittle, Jin Li
          </td>
          <td>2024-09-03</td>
          <td>The journal of applied laboratory medicine</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0943aff1f710e095be6a9439cd65322a1fc6aa0e" target='_blank'>
              Genomic Balancing Act: deciphering DNA rearrangements in the complex chromosomal aberration involving 5p15.2, 2q31.1, and 18q21.32.
              </a>
            </td>
          <td>
            Zain Dardas, Dana Marafi, Ruizhi Duan, Jawid M Fatih, O. El-Rashidy, Christopher M. Grochowski, Claudia M B Carvalho, S. Jhangiani, W. Bi, Haowei Du, Richard A Gibbs, J. Posey, D. Calame, Maha S. Zaki, James R. Lupski
          </td>
          <td>2024-09-10</td>
          <td>European journal of human genetics : EJHG</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="All subtypes of rhabdomyosarcoma (RMS) show skeletal muscle differentiation but each has different oncogenic mechanisms including recurrent PAX3 and PAX7 gene fusions (alveolar), a hotspot mutation in MYOD1 (spindle/sclerosing) and mutations in other oncogenes and tumor suppressors (embryonal, pleomorphic). This range of genomic findings typically requires several different DNA and RNA-based diagnostic assays. Here, we utilize a clinically validated, targeted RNA next-generation sequencing (RNAseq) panel to simultaneously detect MYOD1 transcript levels for lineage assignment, PAX3 and PAX7 translocations and mutations in MYOD1 and other RMS-related genes for definitive subtyping in a single assay. RNA-based detection of MYOD1 p.L122R and other mutations were orthogonally validated with digital droplet (dd)PCR or DNA-based NGS. Expanding the utility of clinical-grade RNAseq data beyond the detection of gene fusions is a cost-effective and time-efficient approach for more comprehensive screening of RMS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7e47bc521700facb67966c59b363c93b774e0f8" target='_blank'>
              Integrated Molecular Profiling of Rhabdomyosarcoma Subtypes by Targeted RNAseq
              </a>
            </td>
          <td>
            Matthew R Avenarius, Ashley Patton, Nehad Mohamed, Pan, Xiaokang, Dan Jones
          </td>
          <td>2024-10-14</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88cace8d561b58e28d3ee8aa2930fd2c3ef573b1" target='_blank'>
              Loss of EMI1 compromises chromosome stability and is associated with cellular transformation in colonic epithelial cell contexts.
              </a>
            </td>
          <td>
            Rubi Campos Gudiño, Nicole M. Neudorf, Demi Andromidas, Zelda Lichtensztejn, K. McManus
          </td>
          <td>2024-10-02</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4b80c87e540615a9cfc5879e1a24f75c5dd8d1b" target='_blank'>
              The interplay of mutagenesis and ecDNA shapes urothelial cancer evolution.
              </a>
            </td>
          <td>
            Duy D. Nguyen, William Hooper, Weisi Liu, Timothy R Chu, Heather Geiger, Jennifer Shelton, Minita J. Shah, Zoe R. Goldstein, L. Winterkorn, Adrienne Helland, M. Sigouros, J. Manohar, Jenna Moyer, Majd Al Assaad, Alissa Semaan, Sandra Cohen, Florencia P. Madorsky Rowdo, David Wilkes, Mohamed Osman, Rahul R. Singh, A. Sboner, Henkel Valentine, Phillip Abbosh, Scott T. Tagawa, D. Nanus, J. Nauseef, Cora N. Sternberg, Ana M. Molina, Douglas Scherr, G. Inghirami, Juan Miguel Mosquera, O. Elemento, N. Robine, B. Faltas
          </td>
          <td>2024-10-09</td>
          <td>Nature</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89a29762d25b9a77c8fd5e2a5d7d19911938e8b1" target='_blank'>
              Integrate and conquer: pan-cancer proteogenomics uncovers cancer vulnerabilities and therapeutic opportunities
              </a>
            </td>
          <td>
            Debomita Chakraborty, Rossana Romero, K. Rajalingam
          </td>
          <td>2024-10-15</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="
 Personalized medicine programs, including the Zero Childhood Cancer Program (ZERO), perform molecular analysis of individual patient tumours to identify molecular targets for therapy. The comprehensive and unbiased sequencing approaches used by ZERO, including whole genome sequencing (WGS) and RNA sequencing (RNAseq), enables detection of novel structural variants (SVs) that may be cryptic to standard cytogenetic techniques or panel sequencing approaches. Identification of Receptor Tyrosine Kinase (RTK) activating variants in paediatric cancers, for example NTRK2 gene fusions, is a high priority given the availability and remarkable clinical success of RTK inhibitors. Here we describe a novel internal tandem duplication (ITD) in NTRK2, encoding TRKB, in a ZERO patient with CNS neuroblastoma. This SV was detected by WGS and was notably missed by RNAseq-based gene fusion and SV detection. The ITD spans exons 10-13 of NTRK2, is in-frame, and includes the juxtamembrane and transmembrane protein domains. We hypothesized that this ITD would result in constitutive activation of TRKB and would be sensitive to TRK-targeted therapies. We overexpressed NTRK2 ITD, wild-type NTRK2, and SPECC1L-NTRK2 (an established TRK-activating fusion), in the interleukin-3 (IL-3)-dependent cell line, Ba/F3, and the neuroblastoma cell line, SH-SY5Y. In these models, we showed that like wild-type NTRK2, NTRK2 ITD was expressed on the cell surface. NTRK2 ITD was sufficient to transform Ba/F3 cells to IL-3 independence through constitutive activation of TRKB and downstream signaling pathways PI3K/AKT and MEK/ERK. NTRK2 ITD-expressing cells were sensitive to TRK inhibitors, including larotrectinib, at similar doses to SPECC1L-NTRK2-expressing cells. Interestingly, we observed that cells expressing NTRK2 ITD were specifically sensitive to MEK inhibition, which was not observed in SPECC1L-NTRK2-expressing cells, suggesting the mechanisms by which these variants drive transformation is different. Indeed, in silico structural analysis showed that the duplicated region of NTRK2 ITD contains two internalized tyrosine residues whose phosphorylation could both drive autophosphorylation of the receptor and act as docking sites for adaptor proteins. Mutation of these tyrosine residues delayed transformation of Ba/F3 cells. This study functionally characterizes a novel NTRK2 ITD and shows that this variant is transforming and sensitive to TRK inhibition. While ITDs have been described in other RTK genes, notably FGFR1 and FLT3, structurally these have involved the kinase domain or the juxtamembrane domain. This is the first report of an ITD that spans the transmembrane domain of an RTK, characterizing an additional mechanism by which RTKs can be activated in cancer. This study highlights the value of unbiased WGS approaches to identify novel SVs and identifies another class of patients that may benefit from RTK inhibitor therapy.
 Citation Format: Lauren M. Brown, Gabor Tax, Pablo Acera Mateos, Antoine de Weck, Steve Foresto, Fatimah Jalud, Teresa Sadras, Pamela Ajuyah, Paulette Barahona, M. Emmy M. Dolman, Marie Wong, Chelsea Mayoh, Mark J. Cowley, Paul G. Ekert. A novel NTRK2-activating internal tandem duplication characterizes a new mechanism of receptor tyrosine kinase activation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B002.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ae609e3759161918ab8221b777b148ec5605447" target='_blank'>
              Abstract B002: A novel NTRK2-activating internal tandem duplication characterizes a new mechanism of receptor tyrosine kinase activation
              </a>
            </td>
          <td>
            Lauren M. Brown, G. Tax, Pablo Acera Mateos, Antoine de Weck, Steve Foresto, Fatimah B Jalud, Teresa Sadras, P. Ajuyah, P. Barahona, M. E. M. Dolman, Marie Wong, C. Mayoh, Mark J. Cowley, Paul G Ekert
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="
 Oncogenic protein dosage is tightly regulated to enable cancer cells to adapt and survive. In healthy cells, translation initiation of onco-mRNA transcripts encoding oncogenes and growth factors is tightly regulated to maintain accurate protein expression levels. However, this process is broken in cancer cells, which rely on the maintenance of high dosage of oncogenic proteins for their survival and to drive cancer development. Whether this is regulated at the level of translational control and the key factors in cis and trans remain unknown. The Myc oncogene is a central paradigm of an exquisitely regulated oncogene and a major driver of pancreatic ductal adenocarcinoma (PDAC). While there has been a tremendous effort to target Myc in cancer, clinically relevant Myc inhibition remains a challenge. Therefore, identifying selective promoters of MYC translation may present a novel therapeutic opportunity. Employing a genome-wide CRISPRi screen for modulators of MYC mRNA translation initiation in PDAC cells, we identified regulators of selective MYC translation and validated four RNA binding proteins (RBPs), including epitranscriptome modifiers. We focused on a top hit, a little studied RBP, RBM42, which is highly expressed in PDAC and predicts poor patient survival. We found that RBM42 binds and selectively regulates the translation of MYC and a precise, yet vital suite of pro-oncogenic transcripts, including JUN and EGFR. Mechanistically, employing immunoprecipitation-mass spectrometry analysis, we found that RMB42 is a novel ribosome-associated protein (RAP). Through RNA structure determination and mutagenesis analysis, we showed that RBM42 directly binds and remodels the MYC 5’UTR RNA structure, facilitating the formation of the translation pre-initiation complex and increasing MYC translation. Importantly, we demonstrated that RBM42 is necessary for human PDAC cell growth and fitness and PDAC tumorigenesis in xenograft mouse models in a Myc-dependent manner in vivo. In PDAC patient samples, RBM42 expression is correlated with Myc protein levels and transcriptional activity. This work transforms our understanding of the translational code in cancer and offers a new therapeutic opening to target the expression of oncogenes.
 Citation Format: Joanna R Kovalski, Goksu Sarioglu, Vishvak Subramanyam, Grace Hernandez, Gilles Rademaker, Juan A Oses-Prieto, Macey Slota, Kwun Wah Wen, Grace E Kim, Alma L Burlingame, Hani Goodarzi, Rushika M Perera, Davide Ruggero. Functional screen for mediators of onco-mRNA translation specificity in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A078.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6940dcc1be9dcaf6b8a1bc0d89818408eab3c22" target='_blank'>
              Abstract A078: Functional screen for mediators of onco-mRNA translation specificity in pancreatic cancer
              </a>
            </td>
          <td>
            Joanna R. Kovalski, Goksu Sarioglu, Vishvak Subramanyam, Grace A. Hernandez, Gilles Rademaker, J. Oses-Prieto, Macey Slota, Kwun Wah Wen, Grace E. Kim, Alma L. Burlingame, Hani Goodarzi, Rushika M. Perera, Davide Ruggero
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Eukaryotic cells respond to stress by altering coding and non-coding gene expression programs. Alongside many approaches and regulatory mechanisms, long non-coding RNAs (lncRNA) are finding a significant place in gene regulation, suggesting an involvement in various cellular processes and pathophysiology. LncRNAs are regulated by many transcription factors, including SMAR1 and p53, which are tumor suppressor genes. SMAR1 inhibits cancer cell metastasis and invasion and is also known to inhibit apoptosis during low-dose stress in coordination with p53. Data mining analysis suggested that these tumor suppressor genes might coregulate the lncRNA RP11-431M3.1 in colon cancer cells. Importantly, RP11-431M3.1 expression was found to be negatively correlated with patient survival rates in a number of cancers. Oxidative stress occurs when an imbalance in the body is caused by reactive oxygen species (ROS). This imbalance is known to be important in the development/pathogenesis of colon cancer. We are researching the role and control of this lncRNA in HCT116 cells under conditions of oxidative stress. We observed a dose-dependent differential expression of lncRNA upon H2O2 treatment and found that p53 and SMAR1 bind differentially to the promoter in response to the dose of stress inducer used. RP11-431M3.1 was observed to sponge miR-138 which has an important target gene, hypoxia-inducible factor (HIF1A). miR-138 was observed to bind differentially to RP11-431M3.1 and HIF1A RNA depending on the dose of oxidative stress. Furthermore, the knockdown of RP11-431M3.1 decreased the migration and proliferation of colon cancer cells. Our results suggest a previously undescribed regulatory mechanism through which RP11-431M3.1 is transcriptionally regulated by SMAR1 and p53, target HIF1A through miR-138, and highlight its potential as a therapeutic and diagnostic marker for cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f27ec17ce3dd08accccf76324d285e276a690d99" target='_blank'>
              SMAR1 and p53-regulated lncRNA RP11-431M3.1 enhances HIF1A translation via miR-138 in colorectal cancer cells under oxidative stress.
              </a>
            </td>
          <td>
            G. S. Bose, Shruti Jindal, Kiran Gautam Landage, Aarzoo Jindal, Monali Prakash Mahale, Abhijeet Kulkarni, S. Mittal
          </td>
          <td>2024-09-06</td>
          <td>The FEBS journal</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 Children born with Down syndrome (DS), a genetic condition caused by the presence of an additional chromosome 21 (trisomy 21 or T21), have a substantially higher risk of developing childhood leukaemia. Approximately 30% of DS infants develop a preleukemic condition called transient abnormal myelopoiesis (TAM). TAM is strictly associated with GATA1 truncating mutations on a T21 background. Most TAM cases spontaneously resolve within the first few months of life. However, by the age of five, approximately 10% of cases progress to megakaryoblastic/erythroid leukaemia, known as myeloid leukaemia of DS (ML-DS). This final step of transformation is often driven by specific third-hit somatic mutations, most of which affects genes encoding JAK kinases, cohesin complexes, or epigenetic regulators. These well-defined stepwise genetic aberrations associated with ML-DS pathogenesis provide a unique and highly tractable system to elucidate the molecular changes underlying leukaemogenesis. Using single-cell mRNA sequencing of primary human samples, we directly interrogate the transcriptional changes underpinning the multi-step pathogenesis of ML-DS. We observe an expansion of megakaryocyte-erythrocyte progenitors (MEPs) specifically in T21 foetal liver compared to diploid or other trisomies. However, it is the GATA1 mutation that accounts for the majority of the transcriptomic changes towards that of the leukaemic blasts. Furthermore, investigation of an aggressive TAM case reveals an enrichment in expression of the leukaemogenic gene module associated with the full-blown leukaemia state ML-DS. We also demonstrate that TAM and ML-DS blasts still retain transcriptional signatures of normal haematopoiesis, showing heterogeneous features of differentiation towards erythrocytes, megakaryocytes, and mast cells. Finally, by integrating genomic and transcriptomic data, we directly defined the genomic evolution, and its corresponding transcriptional consequences, that underpins a treatment-refractory case of ML-DS. Overall, we have generated the first comprehensive single-cell mRNA atlas of TAM / ML-DS, enabling unbiased quantitative characterisation of the transcriptional transformation along ML-DS pathogenesis. Our study explores the molecular signature associated with progression of TAM to ML-DS, as well as the mechanism underlying differences in treatment response. These insights offer the possibility of more meaningful clinical interpretations and potential to explore novel therapeutic targets for these conditions.
 Citation Format: Mi K. Trinh, Matthew D. Young, Conor Parks, Agnes Oszlanczi, Toochi Ogbonnah, Di Zhou, Angus Hodder, Konstantin Schuschel, Hasan Issa, Laura Jardine, Jan-Henning Klusmann, Jack Bartram, Sam Behjati. Transcriptional evolution from normal foetal haematopoiesis to myeloid leukaemia in Down syndrome [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A018.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60f4dde5c53348eeefdb9808ec633733f445cffc" target='_blank'>
              Abstract A018 Transcriptional evolution from normal foetal haematopoiesis to myeloid leukaemia in Down syndrome
              </a>
            </td>
          <td>
            Mi K. Trinh, Matthew D. Young, Conor Parks, Agnes Oszlanczi, Toochi Ogbonnah, Di Zhou, A. Hodder, Konstantin Schuschel, Hasan Issa, Laura Jardine, J. Klusmann, Jack Bartram, Sam Behjati
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88f0f651c9fac9c8c40b22fc5b21b4578646df27" target='_blank'>
              Multi-omics features of immunogenic cell death in gastric cancer identified by combining single-cell sequencing analysis and machine learning
              </a>
            </td>
          <td>
            Shu-Long Dai, Jian-Qiang Pan, Zhen-Rong Su
          </td>
          <td>2024-09-18</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/194aecc5e41432721e4f927f28919d72b01b2b19" target='_blank'>
              Customizable gene sensing and response without altering endogenous coding sequences.
              </a>
            </td>
          <td>
            Fabio Caliendo, Elvira Vitu, Junmin Wang, Shuo-Hsiu Kuo, Hayden Sandt, C. Enghuus, Jesse Tordoff, Neslly Estrada, James J. Collins, Ron Weiss
          </td>
          <td>2024-09-12</td>
          <td>Nature chemical biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The tumor microenvironment (TME) is an immensely complex ecosystem1,2. This complexity underlies difficulties in elucidating principles of spatial organization and using molecular profiling of the TME for clinical use3. Through statistical analysis of 96 spatial transcriptomic (ST-seq) datasets spanning twelve diverse tumor types, we found a conserved distribution of multicellular, transcriptionally covarying units termed ‘Spatial Groups’ (SGs). SGs were either dependent on a hierarchical local spatial context – enriched for cell-extrinsic processes such as immune regulation and signal transduction – or independent from local spatial context – enriched for cell-intrinsic processes such as protein and RNA metabolism, DNA repair, and cell cycle regulation. We used SGs to define a measure of gene spatial heterogeneity – ‘spatial lability’ – and categorized all 96 tumors by their TME spatial lability profiles. The resulting classification captured spatial variation in cell-extrinsic versus cell-intrinsic biology and motivated class-specific strategies for therapeutic intervention. Using this classification to characterize pre-treatment biopsy samples of 16 non-small cell lung cancer (NSCLC) patients outside our database distinguished responders and non-responders to immune checkpoint blockade while programmed death-ligand 1 (PD-L1) status and spatially unaware bulk transcriptional markers did not. Our findings show conserved principles of TME spatial biology that are both biologically and clinically significant.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a400ff77ff3e3d5b4307dd66b458357a7b860383" target='_blank'>
              Conserved principles of spatial biology define tumor heterogeneity and response to immunotherapy
              </a>
            </td>
          <td>
            Vivek Behera, Hannah Giba, Ue-yu Pen, Anna Di Lello, Benjamin A. Doran, Alessandra Esposito, Apameh Pezeshk, C. Bestvina, Justin Kline, M. Garassino, Arjun S. Raman
          </td>
          <td>2024-10-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="BACKGROUND/AIM
The emergence of novel DNA damage repair (DDR) pathways in molecular-target therapy drugs (MTTD) has shown promising outcomes in treating patients with metastatic castration-resistant prostate cancer (mCRPC). About 25% of mCRPC patients have actionable deleterious aberrations in DDR genes, primarily in the homologous recombination (HR) pathway. However, the response rate in patients with BRCA1/2 or mutations in HRR-related genes is only 45%-55%, when exposed to poly ADP ribose polymerase (PARP) inhibitor-based therapy (PARPi). A frequent characteristic feature of prostate cancer (PC) is the occurrence of genomic rearrangement that affects the transmembrane protease serine 2 (TMPRSS2) and E26 transformation-specific (ETS)- transcription factor-related gene (ERG).


MATERIALS AND METHODS
In this study, a total of 114 patients with mCRPC had their RNA and DNA sequenced using next-generation sequencing.


RESULTS
Based on their genetic profile of deleterious gene alterations of BRCA1/2 or ATM, six patients were selected for PARPi. Patients with TMPRSS2:ERG gene fusion and homozygous alteration in ATM or BRCA2 (n=2) or heterozygous alterations (BRCA1 or BRCA2) and lack of TMPRSS2:ERG gene fusion (n=2) did not show clinical benefit from PARPi (treatment duration <16 weeks). In contrast, patients (n=2) without TMPRSS2:ERG gene fusion and homozygous deleterious alterations in ATM or BRCA2 all had clinical benefit from PARPi (treatment duration ≥16 weeks).


CONCLUSION
The TMPRSS2:ERG transcript product might be used as a PARPi resistance biomarker.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccb09c69e7cd8aed30d5976f567f7199cd33c56e" target='_blank'>
              TMPRSS2:ERG Gene Fusion Might Predict Resistance to PARP Inhibitors in Metastatic Castration-resistant Prostate Cancer.
              </a>
            </td>
          <td>
            Tim S Poulsen, Anders Nygaard Lørup, Per Kongsted, R. Eefsen, M. Højgaard, Estrid Høgdall
          </td>
          <td>2024-09-30</td>
          <td>Anticancer research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="DNA repair activity diminishes with age and genetic mutations, leading to a significantly increased risk of cancer and other diseases. Upregulating the DNA repair system has emerged as a potential strategy to mitigate disease susceptibility while minimizing cytotoxic side effects. However, enhancing DNA repair activity presents significant challenges due to the inherent inefficiency in activator screening processes. Additionally, pinpointing a critical target that can effectively upregulate overall repair processes is complicated as the available information is somewhat sporadic. In this review, we discuss potential therapeutic targets for upregulating DNA repair pathways, along with the chemical structures and properties of reported small-molecule activators. We also elaborate on the diverse mechanisms by which these targets modulate repair activity, highlighting the critical need for a comprehensive understanding to guide the development of more effective therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf64add08ecc3d0be655106d39c828b55e519b54" target='_blank'>
              Therapeutic upregulation of DNA repair pathways: strategies and small molecule activators.
              </a>
            </td>
          <td>
            Juhyung Song, Cheoljun Park, Francis Cabanting, Y. Jun
          </td>
          <td>2024-10-11</td>
          <td>RSC medicinal chemistry</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Synovial Sarcomas (SS) are characterized by the presence of the SS18::SSX fusion gene, which protein product induce chromatin changes through remodeling of the BAF complex. To elucidate the genomic events that drive phenotypic diversity in SS, we performed RNA and targeted DNA sequencing on 91 tumors from 55 patients. Our results were verified by proteomic analysis, public gene expression cohorts and single-cell RNA sequencing. Transcriptome profiling identified three distinct SS subtypes resembling the known histological subtypes: SS subtype I and was characterized by hyperproliferation, evasion of immune detection and a poor prognosis. SS subtype II and was dominated by a vascular-stromal component and had a significantly better outcome. SS Subtype III was characterized by biphasic differentiation, increased genomic complexity and immune suppression mediated by checkpoint inhibition, and poor prognosis despite good responses to neoadjuvant therapy. Chromosomal abnormalities were an independent significant risk factor for metastasis. KRT8 was identified as a key component for epithelial differentiation in biphasic tumors, potentially controlled by OVOL1 regulation. Our findings explain the histological grounds for SS classification and indicate that a significantly larger proportion of patients have high risk tumors (corresponding to SS subtype I) than previously believed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea5e414a7a3b033cc20bf7f0e410a6830f2ad049" target='_blank'>
              Molecular Profiling Defines Three Subtypes of Synovial Sarcoma.
              </a>
            </td>
          <td>
            Yi Chen, Yanhong Su, Xiaofang Cao, I. Siavelis, I. Leo, Jianming Zeng, P. Tsagkozis, A. Hesla, A. Papakonstantinou, Xiao Liu, Wen-Kuan Huang, Binbin Zhao, Cecilia Haglund, M. Ehnman, Henrik Johansson, Yingbo Lin, Janne Lehtiö, Yifan Zhang, O. Larsson, Xuexin Li, Felix Haglund de Flon
          </td>
          <td>2024-09-10</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Sickle cell disease (SCD) is a hereditary blood disorder characterized by the presence of abnormal hemoglobin molecules and thus distortion (sickling) of the red blood cells. SCD causes chronic pain and organ damage and shortens life expectancy. Gene therapy emerges as a potentially curative approach for people with SCD who lack a matched sibling donor for hematopoietic stem cell transplantation. Here, we review recent progress in gene therapy for SCD and focus on innovative technologies that target the genetic roots of the disease. We also review the challenges associated with gene therapy, including oncogenic risks, and the need for refined delivery methods. Despite these hurdles, the rapidly evolving landscape of gene therapy for SCD raises hope for a paradigm shift in the treatment of this debilitating disease. As research progresses, a deeper understanding of the molecular mechanisms involved and continuous improvements in gene-editing technologies promise to bring gene therapy for SCD closer to mainstream clinical application, offering a transformative, curative option for patients with this genetic disorder.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c6e785d7cbf41248336e372c5a5fb5eb1bf4e0c" target='_blank'>
              Treating Sickle Cell Disease: Gene Therapy Approaches.
              </a>
            </td>
          <td>
            M. Cavazzana, Alice Corsia, Mégane Brusson, A. Miccio, Michaela Semeraro
          </td>
          <td>2024-09-11</td>
          <td>Annual review of pharmacology and toxicology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Non-small cell lung cancer (NSCLC) remains a disease with a poor prognosis despite the advances in therapies. NSCLC with actionable oncogenic alterations represent a subgroup of diseases for which tyrosine kinase inhibitors (TKIs) have shown relevant and robust impact on prognosis, both in early and advanced stages. While the introduction of powerful TKIs increases the ratio of potentially curable patients, the disease does develop resistance over time through either secondary mutations or bypass activating tracks. Therefore, new treatment strategies are being developed to either overcome this inevitable resistance or to prevent it, and proteolysis targeting chimera agents (PROTACs) are among them. They consist of two linked molecules that bind to a target protein and an E3 ubiquitin ligase that causes ubiquitination and degradation of proteins of interest. In this paper, we review the rationale for PROTAC therapy and the current development of PROTACs for oncogene-addicted lung cancer. Moreover, we critically analyze the strengths and limitations of this promising technique that may help pave the way for future perspectives.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/866e44b9a44363bae9549987792f66d0f87f7e0e" target='_blank'>
              Proteolysis Targeting Chimera Agents (PROTACs): New Hope for Overcoming the Resistance Mechanisms in Oncogene-Addicted Non-Small Cell Lung Cancer
              </a>
            </td>
          <td>
            N. Cordani, Daniele Nova, Luca Sala, M. Abbate, F. Colonese, D. Cortinovis, S. Canova
          </td>
          <td>2024-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background: Gliomas are neoplasms of the central nervous system that originate in glial cells. The genetic characteristics of this type of neoplasm are the loss of function of tumor suppressor genes such as TP53 and somatic mutations in genes such as IDH1/2. Additionally, in clinical cases, de novo single nucleotide polymorphisms (SNP) are reported, of which their pathogenicity and their effects on the function and stability of the protein are known. Methodology: Non-synonymous SNPs were analyzed for their structural and functional effect on proteins using a set of bioinformatics tools such as SIFT, PolyPhen-2, PhD-SNP, I-Mutant 3.0, MUpro, and mutation3D. A structural comparison between normal and mutated residues for disease-associated coding SNPs was performed using TM-aling and the SWISS MODEL. Results: A total of 13 SNPs were obtained for the TP53 gene, 1 SNP for IDH1, and 1 for IDH2, which would be functionally detrimental and associated with disease. Additionally, these changes compromise the structure and function of the protein; the A161S SNP for TP53 that has not been reported in any databases was classified as detrimental. Conclusions: All non-synonymous SNPs reported for TP53 were in the region of the deoxyribonucleic acid (DNA) binding domain and had a great impact on the function and stability of the protein. In addition, the two polymorphisms detected in IDH1 and IDH2 genes compromise the structure and activity of the protein. Both genes are related to the development of high-grade gliomas. All the data obtained in this study must be validated through experimental approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6597e0eb6fecbfff6b6800003b6d0c3a0127247" target='_blank'>
              Bioinformatic Characterization of the Functional and Structural Effect of Single Nucleotide Mutations in Patients with High-Grade Glioma
              </a>
            </td>
          <td>
            Sara Vélez Gómez, Juliana María Martínez Garro, León Darío Ortiz Gómez, Jorge Emilio Salazar Flórez, Fernando P Monroy, R. G. Peláez Sánchez
          </td>
          <td>2024-10-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The family of LINE1 transposable elements underwent a massive expansion in mammalian genomes. While traditionally viewed as a mutagenic selfish element, recent studies point to roles for LINE1 in early mouse development, T cell quiescence and neurogenesis. Here we show that human LINE1 RNA is essential for self-renewal and identity of human embryonic stem cells (hESCs). Silencing of LINE1 using either antisense oligonucleotides or CRISPR interference in naïve hESCs leads to a strong induction of 8C-like cells (8CLCs). We found that genes derepressed upon LINE1 KD are not uniformly distributed across the genome, with an enrichment for chromosome 19, which includes key markers of the 8C state such as TPRX1. Silencing of TPRX1, but not other putative 8C regulators p53 or H3.XY, suppresses the induction of the 8C program in LINE1 KD hESCs. We found that LINE1 RNA is preferentially localized to the lamina and periphery of the nucleolus in hESCs. Sequencing of Lamina-Associated Domains (LADs) and Nucleolus-Associated Domains (NADs) reveals a preferential association of chromosome 19 with NADs in hESCs. However, 8CLCs have a distinct nucleolar morphology and a lower association of chromosome 19 and TPRX1 loci with the nucleolus relative to naïve and primed hESCs, suggesting a role for nucleolar dynamics in the 8CLC-hESC transition. In agreement, LINE1 KD leads to disruption of nucleolar architecture with signs of nucleolar stress. Independent perturbations of the nucleolus induce the 8C program in hESCs. Genes induced by LINE1 KD are enriched for targets of Polycomb Repressive Complex (PRC2), and inhibition of PRC2 leads to a strong induction of 8C genes. Our results indicate that LINE1 coordinates nuclear compartmentalization and chromatin-mediated gene repression to prevent developmental reversion of hESCs. Highlights Knockdown of LINE1 induces TPRX1-dependent emergence of 8C-like cells in hESCs. Genes de-repressed upon LINE1 KD are enriched for Chr 19 and PRC2 targets. 8CLCs display dissociation of Chromosome 19 from the nucleolus. Disruption of the nucleolus or inhibition of PRC2 strongly induce the 8C program.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34781a9454759e1cbbdf47b8d98d4f390cccbdd4" target='_blank'>
              LINE1 promotes nuclear compartmentalization to repress the 8-cell state in embryonic stem cells
              </a>
            </td>
          <td>
            Juan Zhang, Lamisa Ataei, Liang Wu, Kirti Mittal, Linh Huynh, Shahil Sarajideen, Abdul Mazid, David P. Cook, D. Trcka, Kevin Tse, Jeffrey L. Wrana, Michael M. Hoffman, Miguel A Esteban, M. Ramalho-Santos
          </td>
          <td>2024-09-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Retrotransposons are invasive genetic elements, which replicate by copying and pasting themselves throughout the genome in a process called retrotransposition. The most abundant retrotransposons by number in the human genome are Alu and LINE-1 elements, which comprise approximately 40% of the human genome. The ability of retrotransposons to expand and colonize eukaryotic genomes has rendered them evolutionarily successful and is responsible for creating genetic alterations leading to significant impacts on their hosts. Previous research suggested that hypomethylation of Alu and LINE-1 elements is associated with global hypomethylation and genomic instability in several types of cancer and diseases, such as neurodegenerative diseases, obesity, osteoporosis, and diabetes mellitus (DM). With the advancement of sequencing technologies and computational tools, the study of the retrotransposon’s association with physiology and diseases is becoming a hot topic among researchers. Quantifying Alu and LINE-1 methylation is thought to serve as a surrogate measurement of global DNA methylation level. Although Alu and LINE-1 hypomethylation appears to serve as a cellular senescence biomarker promoting genomic instability, there is sparse information available regarding their potential functional and biological significance in DM. This review article summarizes the current knowledge on the involvement of the main epigenetic alterations in the methylation status of Alu and LINE-1 retrotransposons and their potential role as epigenetic markers of global DNA methylation in the pathogenesis of DM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a07f55c13ed59ce933c2fb2995ef34b1c1fa76b5" target='_blank'>
              Retrotransposons and Diabetes Mellitus
              </a>
            </td>
          <td>
            Andromachi Katsanou, Charilaos Kostoulas, E. Liberopoulos, A. Tsatsoulis, I. Georgiou, S. Tigas
          </td>
          <td>2024-09-01</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Genome-wide CRISPR screens have emerged as powerful tools for uncovering the genetic underpinnings of diverse biological processes. Incisive screens often depend on directly measuring molecular phenotypes, such as regulated gene expression changes, provoked by CRISPR-mediated genetic perturbations. Here, we provide quantitative measurements of transcriptional responses in human cells across genome-scale perturbation libraries by coupling CRISPR interference (CRISPRi) with barcoded expression reporter sequencing (CiBER-seq). To enable CiBER-seq in mammalian cells, we optimize the integration of highly complex, barcoded sgRNA libraries into a defined genomic context. CiBER-seq profiling of a nuclear factor kappa B (NF-κB) reporter delineates the canonical signaling cascade linking the transmembrane TNF-alpha receptor to inflammatory gene activation and highlights cell-type-specific factors in this response. Importantly, CiBER-seq relies solely on bulk RNA sequencing to capture the regulatory circuit driving this rapid transcriptional response. Our work demonstrates the accuracy of CiBER-seq and its potential for dissecting genetic networks in mammalian cells with superior time resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f33a5d37dba6ea92909caf57b17709293f40346" target='_blank'>
              CRISPRi with barcoded expression reporters dissects regulatory networks in human cells
              </a>
            </td>
          <td>
            Jinyoung Kim, Ryan Y Muller, Eliana R. Bondra, Nicholas T. Ingolia
          </td>
          <td>2024-09-06</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>50</td>
        </tr>

        <tr id="An important challenge for mammalian cell engineering is the unexpected response of transgenes to native transcriptional regulation pathways. One transgene can show different levels of expression at different genomic sites, in different cell types, and under different growth conditions. Collisions between transcription and DNA replication, heterochromatin encroachment, and viral defense have been linked to transgene silencing. In this study we identify fatty acid metabolism as another mediator of transgene behavior. Adipocyte secretome-induced lipogenesis in epithelial breast cancer cells was accompanied by the loss of expression from a Tet-TA regulated pCMV-AmCyan fluorescent protein (CFP) transgene. Transcription profiling verified the activation of lipid droplet biosynthesis genes, and revealed repression of loci at every chromosome, consistent with the idea that lipogenesis affects the availability of substrates and cofactors for global chromatin remodeling. Pre-induction of pCMV prevented full transgene silencing during lipogenesis. Our results provide new insights into the influence of the lipogenic epigenetic state on transgene behavior.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/523cfb6147ed3575e854d91fb51a5db9d67d279c" target='_blank'>
              Tet Transgene Activation is Disrupted in Lipogenic Triple Negative Breast Cancer Cells
              </a>
            </td>
          <td>
            Ashley Townsel, Yifei Wu, Maya Jaffe, Cara E. Shields, Karmella A. Haynes
          </td>
          <td>2024-10-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Alterations in DNA damage repair genes in advanced prostate cancer (PC) may impact responses to therapy and clinical outcomes. This study described homologous recombination repair (HRR) testing patterns and clinical outcomes among patients with metastatic castration-resistant prostate cancer (mCRPC) by HRR alteration status and race in the United States (US). Methods Clinical data in the nationwide (US-based) Flatiron Health-Foundation Medicine, Inc. (FMI) Metastatic PC Clinico-Genomic Database were evaluated (01/01/2011–12/31/2022). Patients initiating first-line (1L) mCRPC therapy on or after mCRPC diagnosis were included. Testing patterns, time-to-next treatment, overall survival (OS), and time-to-prostate specific antigen response were described. Results Of the 1367 patients with mCRPC and at least one HRR panel test prior to or on the date of 1L mCRPC therapy initiation, 332 (24.3%) were HRR positive (White patients: n = 219 [66.0%]; Black patients: n = 37 [11.1%]) and 1035 (75.7%) were HRR negative (White patients: n = 702 [67.8%]; Black patients: n = 84 [8.1%]). The mean time between first positive test and 1L mCRPC therapy initiation date was 588 days (White patients: 589 days; Black patients: 639 days). Among HRR positive relative to negative patients, trends for faster progression (respective 12-month rate overall: 71.1% and 63.7%; White patients: 72.5% and 64.0%; Black patients: 65.4% and 56.4%), shorter OS (respective 24-month rate overall: 46.8% and 51.9%; White patients: 48.6% and 46.2%; Black patients: 52.8% and 54.1%), and decreased treatment response (respective 12-month rate overall: 24.3% and 37.9%; White patients: 24.5% and 35.2%; Black patients: 17.0% and 43.9%) were observed. Conclusion Patients with mCRPC positive for HRR alterations tended to exhibit poorer treatment responses and clinical outcomes than those with a negative status. These findings highlight the importance of timely genetic testing in mCRPC, particularly among Black patients, and the need for improved 1L targeted therapies to address the unmet need in HRR positive mCRPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42b5b000d1e3f13e0b5a22c9691a361f7dc5edd2" target='_blank'>
              Homologous Recombination Repair Testing Patterns and Outcomes in mCRPC by Alteration Status and Race
              </a>
            </td>
          <td>
            M. Bilen, I. Khilfeh, C. Rossi, Erik Muser, L. Morrison, A. Hilts, Lilian Diaz, Patrick Lefebvre, D. Pilon, Daniel George
          </td>
          <td>2024-09-01</td>
          <td>ClinicoEconomics and Outcomes Research: CEOR</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="The relationship between BRAF-V600E mutations, mitochondrial DNA amplification and microsatellite instability-high (MSI-H) in colorectal cancer (CRC) has yet to be fully elucidated. The aim of the present study was to assess the association between the MSI status and BRAF-V600E gene mutations/clinicopathological features/mitochondrial DNA amplification in CRC. A non-interventional study analysis was performed using the clinicopathological features of 455 patients with CRC. Immunohistochemistry was used to evaluate four mismatch repair proteins (MutS homolog 2, MutS homolog 6, MutL homolog 1 and postmeiotic segregation increased 2), Ki-67 index, and programmed cell death protein 1 (PD-1) and programmed cell death-ligand 1 (PD-L1) expression. Additionally, PCR coupled with capillary electrophoresis were used to ascertain the MSI status. Moreover, amplification refractory mutation system-PCR was used to detect BRAF-V600E gene mutation and fluorescence in situ hybridization analysis was used to assess mitochondrial DNA. A total of 455 patients were divided into the MSI high (MSI-H) group (n=52) and microsatellite stability (MSS) group (n=403) based on their MSI status. Compared with the results of immunohistochemistry of four mismatch repair proteins, the consistency rate between mismatch repair protein deficiency and MSI was 94.23%. There were significant differences in PD-L1, primary tumor site, clinical stage, degree of differentiation, tumor size, lymph node metastasis and the occurrence of multiple primary tumors between the MSI-H group and MSS group (P<0.05 or P<0.001). However, there were no significant differences for sex, age, PD-1, Ki-67 expression and BRAF-V600E. The 24–60-month survival rate of the patients in the MSI-H group was significantly higher than that of those in the MSS group (P<0.05). Furthermore, the number of mitochondrial DNA was significantly amplified in the MSI-H group. In conclusion, the present study demonstrated that the combined detection of PD-L1 and MSI in patients with CRC can provide more accurate and effective guidance for personalized treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5feaa7383704c34d700372d523136046568f74f8" target='_blank'>
              Association between microsatellite instability status, clinicopathological features and mitochondrial DNA amplification in patients with colorectal cancer
              </a>
            </td>
          <td>
            Junmi Lu, Hong Tan, Tao Guo, Xi Chen, Zhongyi Tong
          </td>
          <td>2024-09-26</td>
          <td>Oncology Letters</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="In recent years, it has been demonstrated that many of the pervasive genetic defects throughout cancerogenesis occur in genes encoding chromatin regulators (CRs). We analyzed the distribution and characteristics of well-studied CRs across tens of thousands of tumor samples. Our analysis revealed that tumors with mutations in CRs are associated with high tumor mutational burden (TMB). The co-occurrence of mutations in multiple CRs was linked with a further increase in TMB. Given that TMB may predict the clinical response to immune checkpoint inhibitor (ICI) treatment, we investigated the relationship between mutations in CRs and ICI response. We found that patients harboring mutations in CRs exhibited improved responses to ICI treatment, comparable to those with deficiencies in canonical DNA repair pathways. Overall, this study uncovered significant relationships between mutations in chromatin regulators and critical features of cancer, underscoring the need for further functional and clinical studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a483b13f9b49765baf5b4ffa7cd0de2126755c55" target='_blank'>
              Tumors with mutations in chromatin regulators are associated with higher mutational burden and improved response to checkpoint immunotherapy
              </a>
            </td>
          <td>
            G. Kungulovski, M. Gjorgjievska, S. Mehandziska, D. Bukovec, M. Risteski, I. Kungulovski, Z. Mitrev
          </td>
          <td>2024-10-16</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Natural killer (NK) cells exhibit potent cytotoxic activity against various cancer cell types. Over the past five decades, numerous methodologies have been employed to elucidate the intricate molecular mechanisms underlying NK cell-mediated tumor control. While significant progress has been made in elucidating the interactions between NK cells and tumor cells, the regulatory factors governing NK cell-mediated tumor cell destruction are not yet fully understood. This includes the diverse array of tumor ligands recognized by NK cells and the mechanisms that NK cells employ to eliminate tumor cells. Methods In this study, we employed a genome-wide CRISPR/Cas9 screening approach in conjunction with functional cytotoxicity assays to delineate the proteins modulating the susceptibility of colon adenocarcinoma HCT-116 cells to NK cell-mediated cytotoxicity. Results Analysis of guide RNA (gRNA) distribution in HCT-116 cells that survived co-incubation with NK cells identified ICAM-1 as a pivotal player in the NKp44-mediated immune synapse, with NKp44 serving as an activating receptor crucial for the elimination of HCT-116 tumor cells by NK cells. Furthermore, disruption of genes involved in the apoptosis or IFN-γ signaling pathways conferred resistance to NK cell attack. We further dissected that NK cell-derived IFN-γ promotes mitochondrial apoptosis in vitro and exerts control over B16-F10 lung metastases in vivo. Conclusion Monitoring ICAM-1 levels on the surface of tumor cells or modulating its expression should be considered in the context of NK cell-based therapy. Additionally, considering the diffusion properties of IFN-γ, our findings highlight the potential of leveraging NK cell-derived IFN-γ to enhance direct tumor cell killing and facilitate bystander effects via cytokine diffusion, warranting further investigation. WHAT IS ALREADY KNOWN ON THIS TOPIC NK cells play a crucial role in identifying and eliminating various cancer cell types. However, the mechanisms that regulate NK cell-mediated destruction of tumor cells are not yet fully understood. This involves the array of tumor ligands that NK cells recognize and the processes they utilize to carry out tumor cell elimination. WHAT THIS STUDY ADDS Our research emphasizes the critical role of ICAM-1 in NKp44-mediated destruction of HCT-116 tumor cells. Additionally, we found that interfering with genes related to apoptosis or IFN-γ signaling pathways increased resistance to NK cell attack. We showed that IFN-γ produced by NK cells induces mitochondrial apoptosis in vitro and helps regulate B16-F10 lung metastases in vivo. HOW THIS STUDY MIGHT AFFECT RESEARCH, PRACTICE OR POLICY Given the ability of IFN-γ to diffuse, our findings suggest that NK cell-derived IFN-γ can be harnessed to directly kill tumor cells and trigger bystander effects through cytokine spread. This approach holds promise for further exploration. Additionally, assessing or manipulating ICAM-1 levels on tumor cell surfaces could enhance the effectiveness of NK cell-based therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7312fc3a41477554893f985736277783c1d1ccb1" target='_blank'>
              Genome-wide CRISPR/Cas9 screen reveals factors that influence the susceptibility of tumor cells to NK cell-mediated killing
              </a>
            </td>
          <td>
            S. Guia, A. Fenis, Justine Galluso, Hakim Medjouel, Bertrand Escalière, Angelica Modelska, M. Vienne, Noella Lopes, Amélie Pouchin, Benjamin Rossi, Laurent Gauthier, Sandrine Roulland, Eric Vivier, Émilie Narni-Mancinelli
          </td>
          <td>2024-10-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) is a hallmark of aggressive cancer, contributing to both oncogene amplification and tumor heterogeneity. Here, we used Hi-C, super-resolution imaging, and long-read sequencing to explore the nuclear architecture of MYC-amplified ecDNA in colorectal cancer cells. Intriguingly, we observed frequent spatial proximity between ecDNA and 68 repetitive elements which we called ecDNA-interacting elements or EIEs. To characterize a potential regulatory role of EIEs, we focused on a fragment of the L1M4a1#LINE/L1 which we found to be co-amplified with MYC on ecDNA, gaining enhancer-associated chromatin marks in contrast to its normally silenced state. This EIE, in particular, existed as a naturally occurring structural variant upstream of MYC, gaining oncogenic potential in the transcriptionally permissive ecDNA environment. This EIE sequence is sufficient to enhance MYC expression and is required for cancer cell fitness. These findings suggest that silent repetitive genomic elements can be reactivated on ecDNA, leading to functional cooption and amplification. Repeat element activation on ecDNA represents a mechanism of accelerated evolution and tumor heterogeneity and may have diagnostic and therapeutic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4ade3bf8fe0fe3ba6189e2762e5b189878e923e" target='_blank'>
              Enhancer activation from transposable elements in extrachromosomal DNA
              </a>
            </td>
          <td>
            Katerina Kraft, Sedona E. Murphy, Matthew G Jones, Quanming Shi, Aarohi Bhargava-Shah, Christy Luong, King L. Hung, B. J. He, Rui Li, Seung K. Park, Natasha E Weiser, J. Luebeck, V. Bafna, J. Boeke, Paul S. Mischel, A. Boettiger, Howard Y. Chang
          </td>
          <td>2024-09-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>112</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/744ca5586c9c614b8098bc8c2e1803f3bc1c7ab2" target='_blank'>
              An atlas of genetic effects on cellular composition of the tumor microenvironment.
              </a>
            </td>
          <td>
            Yimin Cai, Zequn Lu, Can Chen, Ying Zhu, Zhirui Chen, Zuyou Wu, Jingyi Peng, Xu Zhu, Ziying Liu, Bin Li, Ming Zhang, Jinyu Huang, Yanmin Li, Yizhuo Liu, Qianying Ma, Chunyi He, Shuo Chen, Wen Tian, Linyun Fan, Caibo Ning, Hui Geng, Bin Xu, Haijie Li, Xu Zhu, Jun Fang, Xiaoyang Wang, Shao-Kai Zhang, Meng Jin, Chaoqun Huang, Xiaojun Yang, Jianbo Tian, Xiaoping Miao
          </td>
          <td>2024-09-02</td>
          <td>Nature immunology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Chromatin is not randomly distributed within the nucleus, but organized in a three-dimensional structure that plays a critical role in genome functions. Сohesin and condensins are conserved multi-subunit protein complexes that participate in mammalian genome organization by extruding chromatin loops. The fine temporal regulation of these complexes is facilitated by a number of other proteins, one of which is microcephalin (Mcph1). Mcph1 prevents condensin II from associating with chromatin through interphase. Loss of Mcph1 induces chromosome hypercondensation; it is not clear to what extent this reorganization affects gene expression. In this study, we generated several mouse embryonic stem cell (mESC) lines with knockout of the Mcph1 gene and analyzed their gene expression profile. Gene Ontology analyses of differentially expressed genes (DEGs) after Mcph1 knockout revealed gene categories related to general metabolism and olfactory receptor function but not to cell cycle control previously described for Mcph1. We did not find a correlation between the DEGs and their frequency of lamina association. Thus, this evidence questions the hypothesis that Mcph1 knockout-mediated chromatin reorganization governs gene expression in mESCs. Among the negative effects of Mcph1 knockout, we observed numerous chromosomal aberrations, including micronucleus formation and chromosome fusion. This confirms the role of Mcph1 in maintaining genome integrity described previously. In our opinion, dysfunction of Mcph1 may be a kind of “Rosetta stone” for deciphering the function of condensin II in the interphase nucleus. Thus, the cell lines with knocked-out Mcph1 can be used to further study the influence of chromatin structural proteins on gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34d65d4d4365d60abd6b73f946f30c7616d79df9" target='_blank'>
              Generation and analysis of mouse embryonic stem cells with knockout of the Mcph1 (microcephalin) gene
              </a>
            </td>
          <td>
            A. Yunusova, A. V. Smirnov, T. Shnaider, I. Pristyazhnuk, S.Y. Korableva, N. R. Battulin
          </td>
          <td>2024-09-01</td>
          <td>Vavilov Journal of Genetics and Breeding</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Simple Summary When oncogenes, which are genes that can cause cancer, become active, they disrupt normal cell processes, especially DNA replication. This disruption is known as replication stress (RS), in which the DNA copying process is stalled or damaged. To prevent cancer, cells usually have a defense system called the DNA damage response (DDR), which can prevent damaged cells from turning into cancer. However, some cells manage to survive this stress by developing replication stress tolerance (RST). These cells can continue to grow, leading to genomic instability (GIN), which is a key feature of cancer. GIN causes various genetic changes that make cells more likely to become cancerous and more difficult to treat. This review explains how oncogenes cause RS and how cells cope with it, leading to the development of cancer. Understanding these processes can help in developing new cancer treatments. Abstract Activation of oncogenes disturbs a wide variety of cellular processes and induces physiological dysregulation of DNA replication, widely referred to as replication stress (RS). Oncogene-induced RS can cause replication forks to stall or collapse, thereby leading to DNA damage. While the DNA damage response (DDR) can provoke an anti-tumor barrier to prevent the development of cancer, a small subset of cells triggers replication stress tolerance (RST), allowing precancerous cells to survive, thereby promoting clonal expansion and genomic instability (GIN). Genomic instability (GIN) is a hallmark of cancer, driving genetic alterations ranging from nucleotide changes to aneuploidy. These alterations increase the probability of oncogenic events and create a heterogeneous cell population with an enhanced ability to evolve. This review explores how major oncogenes such as RAS, cyclin E, and MYC induce RS through diverse mechanisms. Additionally, we delve into the strategies employed by normal and cancer cells to tolerate RS and promote GIN. Understanding the intricate relationship between oncogene activation, RS, and GIN is crucial to better understand how cancer cells emerge and to develop potential cancer therapies that target these vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28c929d9fb4e293ab97bee74d87f261f4dc36600" target='_blank'>
              Tolerance of Oncogene-Induced Replication Stress: A Fuel for Genomic Instability
              </a>
            </td>
          <td>
            Taichi Igarashi, K. Yano, Syoju Endo, B. Shiotani
          </td>
          <td>2024-10-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Continuous chromosome missegregation over successive mitotic divisions, known as chromosomal instability (CIN), is common in cancer. Increasing CIN above a maximally tolerated threshold leads to cell death due to loss of essential chromosomes. Here, we show in two tissue contexts that otherwise isogenic cancer cells with higher levels of CIN are more sensitive to ionizing radiation, which itself induces CIN. CIN also sensitizes HPV-positive and HPV-negative head and neck cancer patient derived xenograft (PDX) tumors to radiation. Moreover, laryngeal cancers with higher CIN prior to treatment show improved response to radiation therapy. In addition, we reveal a novel mechanism of radiosensitization by docetaxel, a microtubule stabilizing drug commonly used in combination with radiation. Docetaxel causes cell death by inducing CIN due to abnormal multipolar spindles rather than causing mitotic arrest, as previously assumed. Docetaxel-induced CIN, rather than mitotic arrest, is responsible for the enhanced radiation sensitivity observed in vitro and in vivo, challenging the mechanistic dogma of the last 40 years. These results implicate CIN as a potential biomarker and inducer of radiation response, which could provide valuable cancer therapeutic opportunities. Statement of Significance Cancer cells and laryngeal tumors with higher chromosome missegregation rates are more sensitive to radiation therapy, supporting chromosomal instability as a promising biomarker of radiation response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/324e0283c2a65fbf5296691fea31358474092654" target='_blank'>
              Chromosomal instability increases radiation sensitivity
              </a>
            </td>
          <td>
            P. Cosper, M. Paracha, Kathryn M. Jones, Laura C. F. Hrycyniak, Les Henderson, Ava Bryan, D. Eyzaguirre, Emily McCunn, Elizabeth Boulanger, Jun Wan, K. Nickel, Vanessa L Horner, Rong Hu, Paul M Harari, R. Kimple, B. Weaver
          </td>
          <td>2024-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Abstract Radiation‐induced tissue injury (RITI) is the most common complication in clinical tumor radiotherapy. Due to the heterogeneity in the response of different tissues to radiation (IR), radiotherapy will cause different types and degrees of RITI, which greatly limits the clinical application of radiotherapy. Efforts are continuously ongoing to elucidate the molecular mechanism of RITI and develop corresponding prevention and treatment drugs for RITI. Single‐cell sequencing (Sc‐seq) has emerged as a powerful tool in uncovering the molecular mechanisms of RITI and for identifying potential prevention targets by enhancing our understanding of the complex intercellular relationships, facilitating the identification of novel cell phenotypes, and allowing for the assessment of cell heterogeneity and spatiotemporal developmental trajectories. Based on a comprehensive review of the molecular mechanisms of RITI, we analyzed the molecular mechanisms and regulatory networks of different types of RITI in combination with Sc‐seq and summarized the targeted intervention pathways and therapeutic drugs for RITI. Deciphering the diverse mechanisms underlying RITI can shed light on its pathogenesis and unveil new therapeutic avenues to potentially facilitate the repair or regeneration of currently irreversible RITI. Furthermore, we discuss how personalized therapeutic strategies based on Sc‐seq offer clinical promise in mitigating RITI.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43cb0e2375143a24b5e75c3b8bc90e1f65f24386" target='_blank'>
              Mechanisms of radiation‐induced tissue damage and response
              </a>
            </td>
          <td>
            Lin Zhou, Jiaojiao Zhu, Yuhao Liu, Ping Zhou, Yongqing Gu
          </td>
          <td>2024-09-20</td>
          <td>MedComm</td>
          <td>1</td>
          <td>12</td>
        </tr>

        <tr id="The tumor suppressor gene TP53 encodes the DNA binding transcription factor p53 and is one of the most commonly mutated genes in human cancer. Tumor suppressor activity requires binding of p53 to its DNA response elements and subsequent transcriptional activation of a diverse set of target genes. Despite decades of close study, the logic underlying p53 interactions with its numerous potential genomic binding sites and target genes is not yet fully understood. Here, we present a database of DNA and chromatin-based information focused on putative p53 binding sites in the human genome to allow users to generate and test new hypotheses related to p53 activity in the genome. Users can query genomic locations based on experimentally observed p53 binding, regulatory element activity, genetic variation, evolutionary conservation, chromatin modification state, and chromatin structure. We present multiple use cases demonstrating the utility of this database for generating novel biological hypotheses, such as chromatin-based determinants of p53 binding and potential cell type-specific p53 activity. All database information is also available as a precompiled sqlite database for use in local analysis or as a Shiny web application.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91f24a956f031769841584e7d92965e89a42014c" target='_blank'>
              p53motifDB: integration of genomic information and tumor suppressor p53 binding motifs
              </a>
            </td>
          <td>
            Gabriele Baniulyte, S. M. Hicks, Morgan A. Sammons
          </td>
          <td>2024-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4aa3f8da17c5a1e64257f76fd2c37200840abc1" target='_blank'>
              Identification of SLC22A17 DNA methylation hotspot as a potential biomarker in cutaneous melanoma
              </a>
            </td>
          <td>
            Alessandro Lavoro, L. Falzone, Giuseppe Gattuso, Giuseppe N Conti, Rosario Caltabiano, G. Madonna, M. Capone, J. McCubrey, P. Ascierto, Massimo Libra, S. Candido
          </td>
          <td>2024-10-02</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>110</td>
        </tr>

        <tr id="
 Rhabdoid tumors (RTs) are rare, highly aggressive pediatric malignancies which occur in kidney, soft-parts and brain. They are characterized by a complete inactivation of the SMARCB1 tumor suppressor gene encoding a core subunit of the chromatin remodeling SWI/SNF complex. Prognosis for children with RTs is poor as, in many instances, these tumors are resistant to conventional type chemotherapy. Pharmacological inhibition of EZH2 is a promising strategy to treat some tumors with loss of function in SMARCB1. However, the clinical response rate remains low, and both primary and secondary resistance have been reported. Understanding the mechanisms of resistance to EZH2 inhibition may allow new therapeutic hypothesis. For this purpose, we realized a genome-wide CRISPR-Cas9 knockout screening on two rhabdoid cell lines treated with EZH2 inhibitor. We identified TP53 loss as the sole consistent gene knock-out able to confer some resistance to EZH2 inhibition in both cell lines. Conversely, we demonstrated that MDM2/MDM4 inhibition, in a TP53-dependant manner, strongly synergized with EZH2 inhibition to control cell lines viability in vitro. We further treated 4 rhabdoid patient-derived xenografts with UNC1999, Idasanutlin and the combination of both and observed a potent tumor growth control upon combined EZH2 and MDM2/MDM4 inhibitions. To conclude, our results strongly encourage to assess the actual efficacy of combined MDM2/MDM4 and EZH2 inhibitors in the treatment of patients with SMARCB1-deficient rhabdoid tumors.
 Citation Format: Céline Chauvin, Tiphaine Hery, Rachida Bouarich, Fariba Nemati, Diego Teyssonneau, Camille Fouassier, Chiara Giudiceandrea, Zhi-Yan Han, Sakina Zaidi, Didier Surdez, Sergio Roman-Roman, Didier Decaudin, Raphaël Margueron, Franck Bourdeaut. Genome-wide CRISPR/Cas9 library screening identified TP53 as a critical driver for resistance to EZH2 inhibitor in rhabdoid tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A051.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85bf34da504cd580597c4be8039c1663b824b215" target='_blank'>
              Abstract A051 Genome-wide CRISPR/Cas9 library screening identified TP53 as a critical driver for resistance to EZH2 inhibitor in rhabdoid tumors
              </a>
            </td>
          <td>
            C. Chauvin, Tiphaine Héry, Rachida Bouarich, F. Némati, Diego Teyssonneau, Camille Fouassier, Chiara Giudiceandrea, Zhi-Yan Han, Sakina Zaidi, Didier Surdez, S. Roman-Roman, Didier Decaudin, Raphaël Margueron, F. Bourdeaut
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Background Hepatocellular carcinoma (HCC) is a prevalent and heterogeneous tumor with limited treatment options and unfavorable prognosis. The crucial role of a disintegrin and metalloprotease (ADAM) gene family in the tumor microenvironment of HCC remains unclear. Methods This study employed a novel multi-omics integration strategy to investigate the potential roles of ADAM family signals in HCC. A series of single-cell and spatial omics algorithms were utilized to uncover the molecular characteristics of ADAM family genes within HCC. The GSVA package was utilized to compute the scores for ADAM family signals, subsequently stratified into three categories: high, medium, and low ADAM signal levels through unsupervised clustering. Furthermore, we developed and rigorously validated an innovative and robust clinical prognosis assessment model by employing 99 mainstream machine learning algorithms in conjunction with co-expression feature spectra of ADAM family genes. To validate our findings, we conducted PCR and IHC experiments to confirm differential expression patterns within the ADAM family genes. Results Gene signals from the ADAM family were notably abundant in endothelial cells, liver cells, and monocyte macrophages. Single-cell sequencing and spatial transcriptomics analyses have both revealed the molecular heterogeneity of the ADAM gene family, further emphasizing its significant impact on the development and progression of HCC. In HCC tissues, the expression levels of ADAM9, ADAM10, ADAM15, and ADAM17 were markedly elevated. Elevated ADAM family signal scores were linked to adverse clinical outcomes and disruptions in the immune microenvironment and metabolic reprogramming. An ADAM prognosis signal, developed through the utilization of 99 machine learning algorithms, could accurately forecast the survival duration of HCC, achieving an AUC value of approximately 0.9. Conclusions This study represented the inaugural report on the deleterious impact and prognostic significance of ADAM family signals within the tumor microenvironment of HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/000a5d98243250c138efd38742555d600655a9e5" target='_blank'>
              Unraveling the role of ADAMs in clinical heterogeneity and the immune microenvironment of hepatocellular carcinoma: insights from single-cell, spatial transcriptomics, and bulk RNA sequencing
              </a>
            </td>
          <td>
            Junhong Chen, Qihang Yuan, Hewen Guan, Yuying Cui, Chang Fu, Tianfu Wei, Kai Liu
          </td>
          <td>2024-09-13</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1dca550adace3299de2700d468a5c183e07ebc1" target='_blank'>
              Widespread mutagenesis and chromosomal instability shape somatic genomes in systemic sclerosis
              </a>
            </td>
          <td>
            Sriram Vijayraghavan, Thomas Blouin, James McCollum, Latarsha Porcher, François Virard, Jiri Zavadil, Carol Feghali-Bostwick, Natalie Saini
          </td>
          <td>2024-10-15</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="CRISPR/Cas9 based genome editing has advanced our understanding of a myriad of important biological phenomena. Important challenges to multiplex genome editing in maize include assembly of large complex DNA constructs, few genotypes with efficient transformation systems, and costly/labor-intensive genotyping methods. Here we present a ‘fast-edit’ approach for multiplex CRISPR/Cas9 genome editing system that delivers a single compact DNA construct via biolistics to type I embryogenic calli, followed by a novel efficient genotyping assay to identify desirable editing outcomes. We first demonstrate the creation of heritable mutations at multiple target sites within the same gene. Next, we successfully created individual and stacked mutations for multiple members of a gene family. Genome sequencing found off-target mutations are rare. Multiplex genome editing was achieved for both the highly transformable inbred line H99 and Illinois Low Protein1 (ILP1), a genotype where transformation has not previously been reported. In addition to screening transformation events for deletion alleles by PCR, we also designed PCR assays that selectively amplify deletion or insertion of a single nucleotide, the most common outcome from DNA repair of CRISPR/Cas9 breaks by non-homologous end-joining. The Indel- Selective PCR (IS-PCR) method enabled rapid tracking of multiple edited alleles in progeny populations. The ‘end to end’ pipeline presented here for multiplexed CRISPR/Cas9 mutagenesis can be applied to accelerate maize functional genomics in a broader diversity of genetic backgrounds.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cc5a9d008a68fa3882779d3e691b6cae9cf9fc7" target='_blank'>
              Efficient mutagenesis of maize inbreds using biolistics, multiplex CRISPR/Cas9 editing, and Indel-Selective PCR
              </a>
            </td>
          <td>
            Maruti Nandan Rai, Brian Rhodes, Stephen Jinga, Praveena Kanchupati, Edward Ross, Shawn R. Carlson, Stephen P. Moose
          </td>
          <td>2024-10-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="In recent decades, the field of genetics has revolutionized our understanding of biology. In particular, we know about how genetic information encodes the blueprint for life and how variations and mutations in these sequences contribute to disease or health. What is unique is that the genetic trait changes do not always caused by any changes in genetic sequences. 
Epigenetics refers to the study of how environmental factors and lifestyle choices can affect the gene activity. Changes in the heritable traits of organisms can be caused by modification of gene expression rather than alteration to the underlying DNA sequence. Therefore, this change does not involve any DNA mutation. 
DNA methylation is a biological process of adding a methyl group to the DNA molecule. The activity of DNA segments can change without changing the sequence by the process of methylation. This process plays an important role in normal development and aberrant DNA methylation patterns are associated with various diseases. Histones are highly basic proteins around which DNA is wound. The modifications in histone protein plays a fundamental role in most biological processes and can alter the accessibility of the DNA for transcription, as well as influencing gene expression. There are non-coding RNA molecules that do not encode proteins but play vital role in regulating the gene expression by interacting with other molecules and influencing chromatin structure. 
Over the past few decades, mechanisms of epigenetics have emerged as a crucial factor in governing every biological process in the human body. Throughout human life, these mechanisms have been impacted various physiological and pathological conditions. For example, cancer is often characterized by epigenetics abnormalities, such as hypermethylation of tumor suppressor genes or hypomethylation of oncogenes. Epigenetic modifications are also implicated in complex diseases like cardiovascular disease, autoimmune conditions and neurodegenerative disorder. Beyond these diseases, epigenetics also offers insights into health and development. Furthermore, understanding the epigenetic mechanisms influence human health, leading to new preventive strategies and treatment of various diseases. Early life experiences, including nutrition and stress, can lead to epigenetic changes that affect long-term health outcomes of the individual.   
Advances in this research can pinpoint any epigenetic changes that affect long-term health outcomes. The advances of epigenetic research can lead to early diagnosis and preventive measures. Identifying the epigenetic markers associated with the diseases enable early detection of the diseases and more effective preventative strategies. Some epigenetic therapies like; drugs that modify histone acetylation or DNA methylation, are already in development and provide future treatments for disease. To understand how life style factors, influence epigenetic regulation can lead to personalized lifestyle recommendations that influence the public health strategies">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6963fc644883e01d78b4f2a627be037841fc025" target='_blank'>
              The Role of Epigenetics in Disease and Health: Beyond DNA Sequences
              </a>
            </td>
          <td>
            .. Diki
          </td>
          <td>2024-08-31</td>
          <td>Pakistan BioMedical Journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Simple Summary Irinotecan is a widely used chemotherapy drug for colorectal cancer, but overcoming resistance to irinotecan remains a significant challenge. FL118, a drug similar to irinotecan, has shown promise as a new treatment option. FL118 functions by reducing the levels of a protein called survivin, which, in turn, decreases the expression of RAD51, a key protein involved in DNA repair. FL118 has been demonstrated to reduce cancer cell viability and overcome irinotecan resistance in colorectal cancer cells, suggesting it could serve as an effective treatment following irinotecan. Abstract Background/Objectives: Irinotecan, a camptothecin (CPT) derivative, is commonly used as a first-line therapy for colorectal cancer (CRC), but resistance remains a significant challenge. This study aims to explore the therapeutic potential of FL118, another CPT derivative, with a focus on overcoming resistance to irinotecan. Methods: The effects of FL118 on CRC cells were evaluated, and bioinformatics analysis was performed on RNA-seq data. Transfection was conducted to observe the knockdown effect of survivin, and the in vivo efficacy of FL118 was assessed using a xenograft model. Results: FL118 induces apoptosis, G2/M arrest, and DNA damage. A notable mechanism of action of FL118 is a reduction in survivin levels, which downregulates the expression of RAD51, a key marker of homologous recombination, and attenuates DNA repair processes. Given that SN38 is the active metabolite of irinotecan, FL118 reduces cell viability and RAD51 in SN38-resistant LOVO cells. Conclusions: Our findings provide effective insights into the antitumor activity of FL118 and its potential as a therapeutic agent for overcoming irinotecan resistance in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d19093f191d9d1383ff5d9089dfc1860caa3a82" target='_blank'>
              FL118 Enhances Therapeutic Efficacy in Colorectal Cancer by Inhibiting the Homologous Recombination Repair Pathway through Survivin–RAD51 Downregulation
              </a>
            </td>
          <td>
            Jungyoun Kim, Yeyeong Jeong, Youkeun Shin, Sung Eun Kim, Sang Joon Shin
          </td>
          <td>2024-10-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Myeloid neoplasms originate from the clonal proliferation of hematopoietic stem cells, which is driven by the acquisition of somatic genetic mutations. Within these disorders, myelodysplastic syndromes (MDS) are specifically characterized by morphologic abnormalities (dysplasia) and impaired maturation of myeloid precursors (ineffective hematopoiesis), resulting in peripheral blood cytopenia. Several studies have advanced the field of MDS, with a few landmark papers leading to a paradigm shift, opening new avenues of research and enabling a molecular revolution. These seminal papers include the first description of the 5q- syndrome, the identification of somatic mutations of TET2 in myeloid neoplasms, the detection of common pathway mutations in the splicing machinery, and the discovery of clonal hematopoiesis. The somatic genomic landscape of MDS is now well-defined. Genes that are recurrently mutated include epigenetic regulators, as well as genes of RNA splicing machinery, transcription regulation, DNA repair control, cohesin complex, and signal transduction. Furthermore, several disorders with a germline genetic predisposition to MDS have been identified, collectively accounting for up to 15% of all MDS cases. Genomic profiling can significantly improve the diagnostic approach to MDS, allowing the identification of distinct nosologic entities such as SF3B1-mutant or TP53-mutant MDS. The Molecular International Prognostic Scoring System for MDS (IPSS-M) has already proven to be a valuable tool for individualized risk assessment and treatment decisions. In addition, the recently developed molecular taxonomy of MDS will likely facilitate the implementation of precision medicine approaches for these disorders. This will necessitate the establishment of specialized infrastructures within public health systems, involving close collaboration between healthcare institutions, academia, and the life sciences industry.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6811c5986693d3b027b3f548b29079c86dbe146d" target='_blank'>
              Genome sequencing in the management of myelodysplastic syndromes and related disorders.
              </a>
            </td>
          <td>
            Mario Cazzola, L. Malcovati
          </td>
          <td>2024-10-24</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24d8d8f2f89f8ee70047ce83a13996f46168eafe" target='_blank'>
              Star wars against leukemia: attacking the clones.
              </a>
            </td>
          <td>
            Monika M. Toma, Tomasz Skorski
          </td>
          <td>2024-09-02</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 The alteration of the glioblastoma genome by epigenetic mechanisms that share functions with normal developmental processes, such as self-renewal and fate specification of neural stem cells (NSC), is a key piece of evidence that links brain cancer pathogenesis with dysregulated stem cell functions. A patient-specific comparison of glioblastoma stem cells (GSC) with NSC, a putative cell of origin of at least a proportion of these tumours, is not feasible as patient-matched endogenous NSC are not surgically accessible and all epigenetic studies in glioblastoma have so far compared epigenetic changes of different tumours with each other or to comparators obtained from foetal brains or an unrelated donor. We reasoned that availability of syngeneic GSC and NSC pairs would allow to identify crucial epigenetic differences on a patient-specific basis and would provide essential therapeutic contrast to define disease-and patient-intrinsic biomarkers of drug response that are less confounded by germline variation.



 We have derived GSC from IDH-wildtype glioblastoma and harnessed state-of-the-art stem cell technologies to generate patient-matched fibroblast-derived EPSC, which were induced to NSC (iNSC). We generated transcriptome, DNA methylome, ChIP-Seq (H3K4me3, H3K27ac, H3K3me3, H3K27me3) and ATACSeq datasets for 10 GIC/iNSC pairs.



 Comparative integrated analysis has shown that redistribution of selected histone marks as well as shifts in chromatin states across the genome identifies differentially expressed genes, including novel druggable regulatory mechanisms in GSC, which are specific of the neoplastic context. In particular, we identified METTL7B as an essential regulator of lineage specification and plasticity in human glioblastoma, with impact on both tumour size and invasiveness in an in vivo xenograft model. Single cell transcriptomic analysis of these tumours and of cerebral organoids derived from expanded potential stem cells (EPSCs) overexpressing METTL7B reveal a regulatory role for the gene in the neural stem cells-to-astrocyte differentiation trajectory. Mechanistically, METTL7B downregulates the expression of key neuronal differentiation players, including SALL2, via post-translational modifications of histone marks.



 SYNGN is an epigenetic platform which can identify novel disease mechanisms and druggable targets in glioblastoma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be96bd21ac249c6fcb9083c430d82766e4fcbf5a" target='_blank'>
              P01.14.B SYNGN: A NOVEL EPIGENETIC PLATFORM TO DISCOVER PATIENT-SPECIFIC DISEASE MECHANISMS AND DRUGGABLE TARGETS IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            S. Marino, C. Vinel, M. Constantinou
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8352ce06dc7c139e4b3281df64d69ea0476999e3" target='_blank'>
              Molecular genetics, therapeutics and RET inhibitor resistance for medullary thyroid carcinoma and future perspectives
              </a>
            </td>
          <td>
            Ying Zhang, Wei-Hui Zheng, Shi-Hong Zhou, Jia-Lei Gu, Qing Yu, Yi-Zhou Zhu, Yu-Jie Yan, Zhi Zhu, Jin-Biao Shang
          </td>
          <td>2024-09-28</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Background: Transposable elements (TEs) are dynamic repetitive regions which generate mutations and structural variants. TP53 plays a crucial role in suppressing TE movement to maintain genomic stability. The relationship between TP53 and TEs has been extensively studied in tumours, but not the germline. Individuals with germline TP53 pathogenic variants have Li-Fraumeni Syndrome (LFS), a cancer predisposition syndrome with a high lifetime risk of cancer in various tissues. This study aims to characterize the TE landscape in individuals with Li-Fraumeni Syndrome and determine how this contributes to their increased cancer risk. Methods: MELT and xTea were used to identify TEs in children with (n=48) or without (n=198) a germline TP53 variant. TE calls were merged with SURVIVOR and variants were annotated with AnnotSV. To assess the influence of TP53 on TE location, we quantified TEs across genomic windows and identified the top 100 significantly different regions with the Mann-Whitney U test, adjusting for multiple comparisons. We used these regions to develop a gradient-boosted tree model with 5-fold cross-validation to predict TP53 status. Results: Individuals with germline TP53 variants harboured significantly fewer ALU and LINE1 elements in their germline genome compared to the control dataset (p<0.001). The phenomenon was consistent across chromosomes and significant in chromosomes 5 and 11 (FDR<0.05). A gradient-boosted tree model was able to differentiate patients with and without a germline TP53 variant with an AUPRC of 0.77 on an unseen test set. Conclusion: Germline TP53 variants may effect the frequency and location of germline TE insertions, which may influence nearby genomic variations and lead to cancer development. Analyzing TEs in individuals with LFS will enhance our understanding of accelerated cancer development in these patients, informing future research for diagnostic and therapeutic approaches.
 Citation Format: Brianne Laverty, Shilpa Yadahalli, Vallijah Subasri, David Malkin. The role of TP53 on transposable elements in pediatric cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A010.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4995c29bd2050c705f9a5b8da916aacb0aa2601c" target='_blank'>
              Abstract A010 The role of TP53 on transposable elements in pediatric cancer
              </a>
            </td>
          <td>
            B. Laverty, Shilpa Yadahalli, Vallijah Subasri, D. Malkin
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Desmoplastic small round cell tumor (DSRCT) is an aggressive sarcoma subtype that is driven by the EWS-WT1 chimeric transcription factor. The prognosis for DSRCT is poor, and major advances in treating DSCRT have not occurred for over two decades. To identify effective therapeutic approaches to target DSRCT, we conducted a high-throughput drug sensitivity screen in a DSRCT cell line assessing chemosensitivity profiles for 79 small-molecule inhibitors. DSRCT cells were sensitive to PARP and ATR inhibitors (PARPi, ATRi), as monotherapies and in combination. These effects were recapitulated using multiple clinical PARPi and ATRi in three biologically distinct, clinically-relevant models of DSRCT, including cell lines, a patient-derived xenograft (PDX)-derived organoid model, and a cell line-derived xenograft mouse model. Mechanistically, exposure to a combination of PARPi and ATRi caused increased DNA damage, G2/M checkpoint activation, micronuclei accumulation, replication stress, and R-loop formation. EWS-WT1 silencing abrogated these phenotypes and was epistatic with exogenous expression of the R-loop resolution enzyme RNase H1 in reversing the sensitivity to PARPi and ATRi monotherapies. The combination of PARPi and ATRi also induced EWS-WT1-dependent cell-autonomous activation of the cGAS/STING innate immune pathway and cell surface expression of PD-L1. Taken together, these findings point towards a role for EWS-WT1 in generating R-loop-dependent replication stress that leads to a targetable vulnerability, providing a rationale for the clinical assessment of PARPi and ATRi in DSRCT.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33f00435911403846b95b540256a684bbb112555" target='_blank'>
              Replication Stress is an Actionable Genetic Vulnerability in Desmoplastic Small Round Cell Tumors.
              </a>
            </td>
          <td>
            Asuka Kawai-Kawachi, Madison M Lenormand, Clémence Astier, Noé Herbel, Meritxell B Cutrona, Carine Ngo, M. Garrido, Thomas Eychenne, N. Dorvault, Laetitia Bordelet, Fei Fei Song, Ryme Bouyakoub, Anastasia Loktev, Antonio Romo-Morales, C. Henon, Léo Colmet-Daage, Julien Vibert, Marjorie Drac, R. Brough, Etienne Schwob, Oliviano Martella, G. Pinna, Janet M Shipley, Sibylle Mittnacht, Astrid Zimmermann, Aditi Gulati, Olivier Mir, Axel Le Cesne, M. Faron, Charles Honoré, Christopher J. Lord, Roman Chabanon, S. Postel-Vinay
          </td>
          <td>2024-10-16</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/505f6d9a24bb0cc1b413b063823fa87cd784ad4b" target='_blank'>
              Extracting regulatory active chromatin footprint from cell-free DNA
              </a>
            </td>
          <td>
            Kevin Lai, Katharine Dilger, Rachael Cunningham, Kathy T Lam, Rhea Boquiren, Khiet Truong, Maggie C. Louie, Richard Rava, Diana Abdueva
          </td>
          <td>2024-09-04</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the major causes of morbidity and mortality worldwide, resulting from the accumulation of genetic and epigenetic alterations in several oncogenes and tumor suppressor genes. Recent studies have identified germline and somatic mutations in the exonuclease domain regions of both epsilon polymerase (POLE) and delta polymerase (POLD1) genes in CRCs. We sought to examine the mutation of these genes in a series of sporadic CRCs. To do this, we extracted DNA from 100 primary CRC samples and 40 corresponding normal tissue samples, which had been previously characterized for clinicopathological and molecular features. We employed a combination of quick-multiplex consensus (QMC)-polymerase chain reaction (PCR) and co-amplification at lower denaturation temperature (COLD)-PCR, followed by high-resolution melting (HRM) analysis and Sanger sequencing, to investigate the exonuclease domain regions of POLE and POLD1 genes for somatic mutations that may potentially alter the proofreading activities of these genes. In silico predictions of the functional significance of the identified genetic alterations were performed using the protein variation effect analyzer, sorting intolerant from tolerant, and PON-P2 algorithms. We identified a total of eight new and non-recurrent somatic variants in the endonuclease domains of POLE and two in POLD1. Nine out of ten variants caused amino acid substitutions, whereas one resulted in a stop codon. Although no significant associations or correlations were found between the POLE/POLD1 mutations and the clinicopathological or molecular features of the CRC cases, most of the POLE/POLD1-mutated cases were microsatellite stable (90%) and aneuploid (80%). Furthermore, in silico analyses showed that nine of the ten variants would likely cause some adverse effect on protein function. Ten somatic variants with predicted proofreading activity-altering effects have been identified in the endonuclease domains of POLE and POLD1 in CRC using a combination of QMC-PCR, COLD-PCR, HRM analyses, and Sanger sequencing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22443d76e404b9ff0c03738771a51c34f113cb72" target='_blank'>
              Exploring somatic mutations in POLE and POLD1: Their role in colorectal cancer pathogenesis and potential therapeutic strategies
              </a>
            </td>
          <td>
            Hersh A. Ham-Karim, Narmeen Ahmad, Mohammad Ilyas
          </td>
          <td>2024-09-06</td>
          <td>Tumor Discovery</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Hematopoietic stem cells (HSCs) maintain homeostasis in the hematopoietic ecosystem, which is tightly regulated at multiple layers. Acute myeloid leukemia (AML) is a severe hematologic malignancy driven by genetic and epigenetic changes that lead to the transformation of leukemia stem cells (LSCs). Since somatic mutations in DNA methylation-related genes frequently occur in AML, DNA methylation is widely altered and functions as a starting engine for initiating AML. Additionally, RNA modifications, especially N6-methyladenosine (m6A), also play an important role in the generation and maintenance of the hematopoietic ecosystem, and AML development requires reprogramming of m6A modifications to facilitate cells with hallmarks of cancer. Given the complex pathogenesis and poor prognosis of AML, it is important to fully understand its pathogenesis. Here, we mainly focus on DNA methylation and RNA m6A modification in hematopoiesis and AML and summarize recent advances in this field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92f8b80ecba345e57d84b39e3572cdbd8a3dd106" target='_blank'>
              Epigenetic modifications in hematopoietic ecosystem: a key tuner from homeostasis to acute myeloid leukemia
              </a>
            </td>
          <td>
            Shu-Qi Yao, Rongxia Guo, Wen Tian, Yanbing Zheng, Jin Hu, G-X Han, Rong Yin, Fuling Zhou, Haojian Zhang
          </td>
          <td>2024-09-12</td>
          <td>Blood Science</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c904b2f681f70c0b549880d31ccc37b010f98cbc" target='_blank'>
              Characterisation of APOBEC3B-Mediated RNA editing in breast cancer cells reveals regulatory roles of NEAT1 and MALAT1 lncRNAs.
              </a>
            </td>
          <td>
            Chi Zhang, Yu-Jing Lu, Mei Wang, Bingjie Chen, Feifei Xiong, C. Mitsopoulos, Olivia Rossanese, Xiuling Li, Paul A Clarke
          </td>
          <td>2024-09-25</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="
 Olaparib and temozolomide (OT) combination therapy is in clinical trial evaluation for adolescent rhabdomyosarcoma (RMS) muscle cancers. Unfortunately, resistance to OT has been reported in other cancers, with no counterstrategies available. Using preclinical mouse xenograft experiments, we show that OT is effective at curbing RMS growth, yet a subset of tumors develop resistance that is associated with transcriptomic changes that occur in the absence of recurrent genomic mutation. Importantly, a vast majority of resistant RMS models upregulate the PIK3CA/AKT pathway, which in turn activates NRF2 transcription factor phosphorylation and subsequent transcriptional expression of multidrug resistance ABC transport proteins that rapidly efflux drugs from cells. Using dynamic real-time imaging of tumor growth and assessing drug responses at single-cell resolution in human cell line models, we found that the PIK3CA inhibitor alpelisib re-sensitized resistant RMS cells to OT therapy by suppressing the expression of ABC transport proteins. Excitingly, RMS uses the same PIK3CA/AKT pathway activation to drive resistance to standard-of-care combination therapy vincristine, actinomycin D, and cyclophosphamide (VAC) and the combination of OT + alpelisib effectively killed VAC-resistant RMS. Alpelisib also re-sensitized resistant RMS to chemotherapy in preclinical xenograft mouse models resulting in reduced tumor burden and extended disease-free survival. Our work defines a common resistance pathway in RMS and has credentialled a new preclinical strategy to kill therapy-resistant RMS.
 Citation Format: Yueyang Wang, Qiqi Yang, Chuan Yan, David M. Langenau. The PIK3CA/AKT pathway drives therapy resistance in rhabdomyosarcoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A072.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de779864c638c41dc5737016f4c37a97b2fb0f5c" target='_blank'>
              Abstract A072 The PIK3CA/AKT pathway drives therapy resistance in rhabdomyosarcoma
              </a>
            </td>
          <td>
            Yueyang Wang, Qiqi Yang, Chuan Yan, D. Langenau
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="
 Since the approval of the first checkpoint inhibitor, ipilimumab (α-CTLA4), there continues to be enthusiasm around leveraging the immune system to eradicate solid tumors. However, unknowns about the cellular mechanism(s) that cause some patients to durably respond while others relapse, continue to stymie the field. To support ongoing efforts elucidating cell subsets responsible for differences in patient responses, we aim to evaluate the functional heterogeneity of T-cell subsets responding to checkpoint inhibitors in mouse models of solid tumors. Single-cell RNA-seq has enabled discoveries of transcriptionally defined cell subsets, but understanding their function require inference from transcription, flow-based validation biased by pre-selection of subsets with specific markers, or reliance on surrogate markers for cell-extrinsic functions (e.g., cytotoxicity). Thus, functional heterogeneity could be obscured. To address this, we leveraged a novel technology that converges live, cellular functions with transcriptomics at the resolution of single cell-cell interactions. This technology captures tens of thousands of individual cells or defined combinations of groups of cells in bio-compatible hydrogel compartments, enabling longitudinal imaging over days to weeks. Throughout this period, functions are characterized via bright-field or immunofluorescence readouts, including antibodies, cytokine capture beads, and dyes. Cells are then lysed within compartments to generate barcoded libraries for mRNA-sequencing and gene signatures from individual cells computationally deconvoluted. With this technology, we developed a novel assay that enables longitudinal, unbiased functional characterization of individual T cells mapped to single-cell transcriptomics. Single mouse primary T cells were encapsulated with one or more MC38 target cells and imaged over time. We validated combinations of different readouts can be measured from the same T-cell, including cell-counts for proliferation, CD8/3 expression to differentiate T cells from target cells, activation markers, and / or target cell death via Sytox. Thus, single T cells can be multi-functionally evaluated over time, enabling the study of attributes inaccessible at high scale with prior technologies, including kinetics of killing or the number of target cells killed by a single T-cell. For assays combining cytotoxicity and surface receptors, we generated barcoded libraries from functionally characterized T cells for sequencing and developed an analytical pipeline that deconvolutes mRNA from single T cells mapped to their multi-functional profiles, to identify novel pathways that regulate anti-tumor functions. We will leverage this workflow to functionally characterize single T cells in tumor-bearing mice treated with checkpoint inhibitors and map the data to underlying transcriptomics. In doing so, we aim to discover functional subsets, potentially with identical transcriptional profiles, to support hypothesis generation around mechanisms governing anti-tumor efficacy of checkpoint inhibitors.
 Citation Format: Jacqueline Yee, Praveer Sharma, Yunmin Li, Isaac Thomas, Pier F Gherardini, Teresa Ai, Mostafa Ronaghi, Gary Schroth, Matthew H Spitzer. Novel workflow for characterizing T-cell functional heterogeneity in response to checkpoint inhibitors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor Immunology and Immunotherapy; 2024 Oct 18-21; Boston, MA. Philadelphia (PA): AACR; Cancer Immunol Res 2024;12(10 Suppl):Abstract nr A025.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b89db5bb53ca20d5ac128fa18040b839f5041ca9" target='_blank'>
              Abstract A025: Novel workflow for characterizing T-cell functional heterogeneity in response to checkpoint inhibitors
              </a>
            </td>
          <td>
            Jacqueline Yee, Praveer Sharma, Yunmin Li, Isaac Thomas, Pier F Gherardini, Teresa Ai, Mostafa Ronaghi, Gary Schroth, Matthew H Spitzer
          </td>
          <td>2024-10-18</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc2b01750179731a3467c6191264b3a3625ebdeb" target='_blank'>
              Altered methylation of imprinted genes in neuroblastoma: implications for prognostic refinement
              </a>
            </td>
          <td>
            Medha Suman, Maja Löfgren, S. Fransson, Jewahri Idris Yousuf, Johanna Svensson, A. Djos, Tommy Martinsson, P. Kogner, T. Kling, Helena Carén
          </td>
          <td>2024-08-31</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Multiple myeloma (MM) is a disease characterized by spatiotemporal heterogeneity of tumor clones. Different genetic aberrations can be observed simultaneously in tumor cells from different loci, and as the disease progresses, new subclones may appear. The role of liquid biopsy, which is based on the analysis of tumor DNA circulating in the blood plasma, continues to be explored in MM. Here, we present an analysis of the STR profiles and mutation status of the KRAS, NRAS, and BRAF genes, evaluated in plasma free circulating tumor DNA (ctDNA), CD138+ bone marrow cells, and plasmacytomas. The prospective single-center study included 97 patients, with a median age of 55 years. Of these, 94 had newly diagnosed symptomatic MM, and three had primary plasma cell leukemia. It should be noted that if mutations were detected only in ctDNA, “non-classical” codons were more often affected. A variety of adverse laboratory and clinical factors have been associated with the detection of rare KRAS or NRAS gene mutations in bone marrow or ctDNA, suggesting that these mutations may be factors of an unfavorable prognosis for MM. Liquid biopsy studies provide undeniable fundamental information about tumor heterogeneity and clonal evolution in MM. Moreover, we focus on using liquid biopsy to identify new high-risk factors for MM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ef6c42652b61bb294400045b1d3bc2249d75f9b" target='_blank'>
              Loss of Heterozygosity and Mutations in the RAS-ERK Pathway Genes in Tumor Cells of Various Loci in Multiple Myeloma
              </a>
            </td>
          <td>
            M. Soloveva, M. Solovev, N. Risinskaya, E. Nikulina, I. Yakutik, B. Biderman, Tatiana Obukhova, Y. Chabaeva, Sergej Kulikov, Andrey Sudarikov, L. Mendeleeva
          </td>
          <td>2024-08-30</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Drug resistance in tumors constitutes a significant obstacle to tumor therapy. Head and neck squamous cell carcinoma (HNSCC) presents a major challenge due to its deep anatomical location, limited space, and complex structure. These factors complicate surgical procedures and hinder the effectiveness of chemoradiotherapy, leading to poor prognosis and reduced quality of life. However, there is hope in the form of circular RNAs (circRNAs), non-coding RNA molecules with a closed-loop structure that exhibits superior stability and resistance to degradation compared to linear RNAs. Recent advances in high-throughput sequencing and bioinformatics technology revealed that circRNAs participate in tumor proliferation, invasion, migration, and drug resistance. This review aims to summarize current research progress on the involvement of circRNAs in drug resistance of HNSCC and provide valuable insights for the prevention and mitigation of drug resistance in HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f79955e57ba8f160fd4d0536e1d8ad58c88c872c" target='_blank'>
              Research progress on the role and mechanism of circular RNA in drug resistance of head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Hongli Zeng, Junshang Ge, Yi Meng, Qian Wang, Mei Yang, Zhaoyang Zeng, Wei Xiong, Xuyu Zu
          </td>
          <td>2024-09-02</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract The acquisition of subsequent genetic lesions (clonal evolution, CE) and/or the expansion of existing clones (CEXP) contributes to clonal dynamics (CD) in myelodysplastic syndromes (MDS). Although CD plays an important role in high‐risk patients in disease progression and transformation into acute myeloid leukemia (AML), knowledge about CD in lower‐risk MDS (LR‐MDS) patients is limited due to lack of robust longitudinal data considering the long clinically stable courses of the disease. In this retrospective analysis, we delineate the frequency and the prognostic impact of CD in an unselected real‐world cohort of LR‐MDS patients. We screened 68 patients with a median follow‐up of 40.5 months and a median of 7.5 (range: 2–22) timepoints for CE and CEXP detected by chromosomal banding analysis, fluorescence in situ hybridization, sequencing, and molecular karyotyping. In 30/68 patients, 47 CE events and a CD rate of 1 event per 4 years were documented. Of note, patients with at least 1 CE event had an increased probability for subsequent treatment. Unexpectedly, CE did not correlate with inferior outcomes, which could be reasonably explained by CD detection triggering the subsequent start of a disease‐modifying therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ea97769d53cd530a3310f70504efa64dd62b64b" target='_blank'>
              Comprehensive sequential genetic analysis delineating frequency, patterns, and prognostic impact of genomic dynamics in a real‐world cohort of patients with lower‐risk MDS
              </a>
            </td>
          <td>
            P. Mazzeo, C. Ganster, John Wiedenhöft, K. Shirneshan, K. Rittscher, Elzbieta B. Brzuszkiewicz, D. Steinemann, Maximilian Schieck, C. Müller-Thomas, Hannes Treiber, F. Braulke, U. Germing, K. Sockel, E. Balaian, J. Schanz, U. Platzbecker, Katharina S. Götze, D. Haase
          </td>
          <td>2024-09-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Combined tracking of clonal evolution and chimeric cell phenotypes could enable detection of the key cellular populations associated with response following therapy, including after allogeneic hematopoietic stem cell transplantation (HSCT). We demonstrate that mitochondrial DNA (mtDNA) mutations co-evolve with somatic nuclear DNA mutations at relapse post-HSCT and provide a sensitive means to monitor these cellular populations. Further, detection of mtDNA mutations via single-cell ATAC with select antigen profiling by sequencing (ASAP-seq) simultaneously determines not only donor and recipient cells, but also their phenotype, at frequencies of 0.1-1%. Finally, integration of mtDNA mutations, surface markers, and chromatin accessibility profiles enables the phenotypic resolution of leukemic populations from normal immune cells, thereby providing fresh insights into residual donor-derived engraftment and short-term clonal evolution following therapy for post-transplant leukemia relapse. As throughput evolves, we envision future development of single-cell sequencing-based post-transplant monitoring as a powerful approach for guiding clinical decision making.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/454ae3e4512380b5fd1a0c7892e2f418e6fca333" target='_blank'>
              Tracking rare single donor and recipient immune and leukemia cells after allogeneic hematopoietic cell transplantation using mitochondrial DNA mutations.
              </a>
            </td>
          <td>
            Livius Penter, Nicoletta Cieri, Katie Maurer, M. Kwok, Haoxiang Lyu, Wesley S Lu, Giacomo Oliveira, Satyen H Gohil, I. Leshchiner, Caleb A. Lareau, Leif S. Ludwig, Donna S. Neuberg, Haesook T Kim, Shuqiang Li, L. Bullinger, Jerome Ritz, Gaddy Getz, Jacqueline S Garcia, Robert J. Soiffer, K. Livak, Catherine J. Wu
          </td>
          <td>2024-09-05</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>42</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024'],
    y: [9],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>