<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-08-25 09:30:19 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="DHFR gene amplification is commonly present in methotrexate (MTX)-resistant colon cancer cells and acute lymphoblastic leukemia. In this study, we proposed an integrative framework to characterize the amplified region by using a combination of single-molecule real-time sequencing, next-generation optical mapping, and chromosome conformation capture (Hi-C). We identified an amplification unit spanning 11 genes, from the DHFR gene to the ATP6AP1L gene position, with high adjusted interaction frequencies on chromosome 5 (~2.2 Mbp) and a twenty-fold tandemly amplified region, and novel inversions at the start and end positions of the amplified region as well as frameshift insertions in most of the MSH and MLH genes were detected. These mutations might stimulate chromosomal breakage and cause the dysregulation of mismatch repair. Characterizing the tandem gene-amplified unit may be critical for identifying the mechanisms that trigger genomic rearrangements. These findings may provide new insight into the mechanisms underlying the amplification process and the evolution of drug resistance. Sequencing a large region of DNA containing many surplus copies of genes linked to drug resistance in colon cancer cells may illuminate how these genomic rearrangements arise. Such regions of gene amplification are highly repetitive, making them impossible to sequence using ordinary methods, and little is known about how they are generated. Using advanced methods, Jeong-Sun Seo at Seoul National University Bundang Hospital in South Korea and co-workers sequenced a region of gene amplification in colon cancer cells. The amplified region was approximately 20 times the length of that in healthy cells and contained many copies of an eleven-gene segment, including a gene implicated in drug resistance. The region also contained mutations in chromosomal repair genes which would disrupt repair pathways. These results illuminate the genetic changes that lead to gene amplification and drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental and Molecular Medicine, Experimental & Molecular Medicine</td>
          <td>5</td>
          <td>49</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum.
                </a>
              </td>
          <td>
            Gustavo S. França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>51</td>
          <td>44</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a67b3f2125e256fc001c7ec119739d1e135a5d03" target='_blank'>
              A Mathematical Model of Persister Cell Plasticity and Its Impact on Adaptive Cancer Therapy
              </a>
            </td>
          <td>
            Adithya Mathews, D. Basanta
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="ABSTRACT KRAS is mutationally activated in 45-50% of colorectal cancer (CRC) cases, and while KRAS-targeted therapies have shown some clinical promise, upfront and acquired resistance limit their efficacy. To explore the acute response and mechanisms underlying KRAS inhibitor resistance, we used targeted exome sequencing and single-cell spatial transcriptomics to analyze patient-matched pre-treatment, on-treatment, and progression biopsies from patients treated with combined KRAS G12C and EGFR inhibition. Acquired genetic events were identified in most patients at progression but were often subclonal and coexisted with transcriptional adaptive states. Mesenchymal, YAP, and fetal-like transcriptional signatures predominated in resistant tumors, while tumor cell-intrinsic inflammatory programs were induced in the early treatment phase. Single-cell spatial analysis revealed significant intratumoral heterogeneity, with diverse adaptive states predominating in different zones of individual tumors. Using human and murine organoid models, we show that these drug-induced inflammatory programs are cancer-cell autonomous and precede the emergence of regenerative fetal-like programs associated with drug resistance. We uncover TBK1 as a promising target to abrogate the early inflammatory adaptive phase and enhance responses to KRAS inhibition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75a7967138a21f862a2f0477898f16663439bb11" target='_blank'>
              Concurrent genetic and non-genetic resistance mechanisms to KRAS inhibition in CRC
              </a>
            </td>
          <td>
            Salvador Alonso, Kevan Chu, Marie J Parsons, Elizabeth Granowsky, H. Gunasinghe, Jinru Shia, R. Yaeger, Lukas E. Dow
          </td>
          <td>2025-08-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Multidrug resistance (MDR) significantly contributes to colon cancer recurrence, making it essential to understand its molecular basis for improved therapies. This study aimed to identify genes and pathways involved in resistance to standard chemotherapeutics by comparing transcriptome profiles of sensitive and paclitaxel-induced MDR colonospheres. Cell viability and growth were assessed following treatment with 5-fluorouracil, oxaliplatin, irinotecan, bevacizumab, and cetuximab. Drug concentrations in culture media posttreatment were measured using high-performance liquid chromatography (HPLC). RNA sequencing (RNA-seq) of untreated sensitive and resistant colonospheres identified differentially expressed genes linked to baseline resistance. Our results confirmed cross-resistance in the resistant model, showing highest oxaliplatin tolerance may involve mechanisms beyond efflux. Transcriptome analysis highlighted upregulation of PIGR and activation of the ribosomal signaling pathway as potential resistance mediators. Notably, AKR1B10, a gene linked to chemotherapeutic detoxification, was overexpressed, whereas genes related to adhesion and membrane transport were downregulated. The overexpression of ribosomal protein genes suggests ribosome biogenesis plays a key role in acquired resistance. These findings suggest that targeting ribosome biogenesis and specific deregulated genes such as PIGR and AKR1B10 may offer promising strategies to overcome MDR in colon cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6aa3128769906dcf129a6a3196212e61e40551f" target='_blank'>
              Transcriptomic Landscape of Paclitaxel-Induced Multidrug Resistance in 3D Cultures of Colon Cancer Cell Line DLD1
              </a>
            </td>
          <td>
            S. Dragičević, J. Dinić, Milena Ugrin, Marija Vidovic, T. Babic, A. Nikolić
          </td>
          <td>2025-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fa29e8cd8f6fdd768b529f92b9275ea04ce76dc" target='_blank'>
              Genetic landscape of cancer: mechanisms, key genes, and therapeutic implications.
              </a>
            </td>
          <td>
            Arun Karnwal, Joydeep Dutta, Aqueel-Ur-Rehman, Abdel Rahman Mohammad Said Al-Tawaha, Natalia Nesterova
          </td>
          <td>2025-08-17</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Cyclin dependent kinases 4 and 6 inhibitors have brought great improvements in the treatment of luminal breast cancer, but resistance is a major clinical hurdle. Multiple biomarkers of resistance have been proposed, but none is currently utilized in clinical practice. By performing single-cell RNA sequencing of seven palbociclib-naïve luminal breast cancer cell lines and palbociclib-resistant derivatives, we show that established biomarkers and pathways related to CDK4/6i resistance present marked intra- and inter- cell-line heterogeneity. Transcriptional features of resistance could be already observed in naïve cells correlating with levels of sensitivity (IC50) to palbociclib. Resistant derivatives showed transcriptional clusters that significantly varied for proliferative, estrogen response signatures or MYC targets. This marked heterogeneity was validated in the FELINE trial where, compared to the sensitive ones, ribociclib-resistant tumors developed higher clonal diversity at genetic level and showed greater trascriptional variability for genes associated with resistance. A potential signature of resistance inferred from the cell-line models, positively enriched for MYC targets and negatively enriched for estrogen response markers, was probed on the FELINE trial, separating sensitive from resistant tumors and revealing higher heterogeneity in resistant versus sensitive cells. These data suggest that heterogeneity for CDK4/6 inhibitors resistant markers might facilitate the development of resistance and challenge the validation of clinical biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab65930181b447a2afea51540e3bba84721ca231" target='_blank'>
              Single-cell transcriptomics reveals biomarker heterogeneity linked to CDK4/6 Inhibitor resistance in breast cancer cell lines
              </a>
            </td>
          <td>
            I. Migliaccio, M. Bonechi, Dario Romagnoli, Giulia Boccalini, F. Galardi, C. Guarducci, A. Nardone, R. Schiff, L. Biganzoli, L. Malorni, M. Benelli
          </td>
          <td>2025-07-31</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff04e163a7aa41f773b08a8fee437665a57ee718" target='_blank'>
              CB2R-induced differentiation epigenetically restrains cancer plasticity enabling adaptive therapy.
              </a>
            </td>
          <td>
            Nuria G. Martínez-Illescas, Gemma Pérez-García, María Rubert-Hernández, Camila Quezada-Gutiérrez, Estrella Martín-Zapater, Luis M Alonso-Colmenar, Carmen Hernández, Ángela Zarco-Cuadrillero, Eduardo Caleiras, Miguel Muñoz-Ruiz, Ana Payo-Payo, Laura Frías-Aldeguer, Cristina Sánchez, Mercedes Balcells-Camps, E R Edelman, María Salazar-Roa
          </td>
          <td>2025-08-01</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Anaplastic thyroid cancer (ATC) is the most aggressive form of thyroid cancer. Despite recent advances in treating BRAFV600E-driven ATC, therapy resistance remains a significant challenge, often resulting in disease progression and death. Leveraging a focused CRISPR/KO screen in parallel with a CRISPR/activation screen, both tailored on response to BRAFV600E inhibitor treatment, we identified TAZ (encoded by the WWTR1 gene) deficiency as synthetically lethal with BRAF inhibitor in ATC. TAZ is overexpressed in ATC compared to well-differentiated thyroid tumors. We demonstrate that TAZ-deficient ATC cells display heightened sensitivity to BRAF inhibitors both in vitro and in vivo. Using gene essentiality score across a large panel of cancer cell lines, we found that BRAFV600E-driven cancers are highly sensitive to TAZ loss, unlike their counterparts with wild-type BRAF and non-BRAFV600E. Mechanistically, we demonstrate that dabrafenib triggers the Unfolded Protein Response (UPR) under ER stress and suppresses protein synthesis. TAZ loss represses the UPR, reverses the inhibition of protein synthesis, and triggers increased cell death by ferroptosis in dabrafenib-treated ATC. Collectively, our findings unveil TAZ as a new target to overcome resistance to BRAF inhibitors in undifferentiated thyroid cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2537688c884004a06a6382cdb89bfdc63b85ffff" target='_blank'>
              CRISPR-Based Gene Dependency Screens reveal Mechanism of BRAF Inhibitor Resistance in Anaplastic Thyroid Cancer
              </a>
            </td>
          <td>
            Shawn Noronha, Yue Liu, Gaga Geneti, Haojian Li, Xiaolin Wu, David Sun, Vaibhavi Gujar, Takashi Furusawa, Alexei Lobanov, Maggie Cam, Lipika R. Pal, N. Nair, Chi-Ping Day, E. Ruppin, Chandrayee Gosh, Jiangnan Hu, Suresh Kumar, T. Andresson, King Chan, Maura O’Neill, Raj Chari, Y. Pommier, Jaydira Del Rivero, Urbain Weyemi, E. Kebebew, Myriem Boufraqech
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>84</td>
        </tr>

        <tr id="Abstract Cancer genomic profiling has revolutionized cancer research and clinical practice by facilitating personalized medicine approaches. Next-generation sequencing technologies have played a pivotal role in expanding the possibilities of cancer genomics, providing a comprehensive view of the genomic landscape of tumors and identifying potential therapeutic targets. In Japan, the approval of several cancer-related gene panel tests has accelerated the implementation of cancer genomic medicine. However, there remains a need for evidence-based selection criteria for the different panel tests available. Among these tests, ACTOnco+® stands out as a comprehensive cancer genomic profiling tool. In this study, we compared 110 samples from 29 different tumor types using FoundationOne® CDx (324 genes for DNA analysis) and ACTOnco+® (440 genes for DNA and 31 genes for RNA analysis). Overall, the mutation profiles between the 2 assays exhibited 82.8% positive agreement in reported sequence alterations in clinically actionable genes, including single nucleotide variants and insertions–deletions. Copy number gains showed concordance of 76.9%, and copy number losses in 66.7%. In the case of KIAA1549-BRAF fusion-positive astrocytoma, the fusion event was not detected by DNA only test, but RNA sequencing identified the rearrangement. The patient exhibited clinical benefit from MEK inhibitor treatment. Tumor mutational burden and microsatellite instability (MSI) demonstrated high concordance across various cancer types. ACTOnco+® identified a total of 329 heterozygous deletions, which were subsequently validated for reliability using FISH analysis. Gene profiling with ACTOnco+® exhibited comparable results to FoundationOne® CDx, thereby contributing to future personalized medicine endeavors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/584447fccf78026ffcd29a7adc016edfe5f11d6e" target='_blank'>
              Characterization of a novel comprehensive genomic profiling test with better detection of heterozygous deletions and RNA-based gene fusion analysis
              </a>
            </td>
          <td>
            Ryouta Kakuta, Akira Takemoto, Kousuke Tanimoto, Rika Noji, Ryo Kudo, Yoshihito Kano, Yayoi Ando, Tatsunori Simoi, Yusuke Kinugasa, Sadakatsu Ikeda
          </td>
          <td>2025-08-01</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Abstract Phenotypic plasticity is a recognized mechanism of therapeutic resistance in prostate cancer (PCa), however current knowledge of driver mechanisms and therapeutic interventions are limited. Using genetically engineered mouse models (GEMMs) devoid of Pten and Rb1, we previously demonstrated the chromatin reprogramming factor enhancer of zeste homolog 2 (EZH2) as an important regulator of alternative transcription programs promoting phenotypic plasticity. Here, using a multi-omics approach we demonstrate that EZH2 regulates multilineage cell states dependent on the RNA binding protein Tristetraprolin (TTP) that mediates RNA stability and activation of translation. Combined chemical inhibition of EZH2 and PI3K/mTORC1 resulted in superior anti-tumor activity in murine and human phenotypic plastic models and was most significant when this combination was used with castration or enzalutamide. Together, these data indicate phenotypic plasticity dependence on coordination between EZH2, TTP and mTORC1 signaling that represent novel therapeutic dependencies for this lethal PCa phenotype. Graphical abstract Significance Statement EZH2 plays a key role in driving cell state transitions in neuroendocrine prostate cancer (NEPC), guiding cancer cells towards a more aggressive, therapy-resistant cell type. This transformation creates a specific vulnerability, as NEPC cells become highly reliant on both EZH2 and PI3K/mTORC1-translation signaling. Targeting this dependency, we demonstrate that combining EZH2 with PI3K/mTORC1 inhibition provides effective suppression of NEPC cell growth, offering a promising therapeutic strategy for treating this challenging and aggressive prostate cancer subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12b27ab9c67e47d28ae41dcca4d51f6a0aabec7f" target='_blank'>
              EZH2-TTP-mTORC1 Axis Drives Phenotypic Plasticity and Therapeutic Vulnerability in Lethal Prostate Cancer
              </a>
            </td>
          <td>
            Beatriz German, Katherine L. Morel, Teia Noel, N. Boufaied, Deborah L. Burkhart, Sujun Chen, Felipe Dezem, Xintao Qui, Henry W. Long, Stefan DiFazio, S. Baca, Ayesha A. Shafi, Matthew L. Freedman, Himisha Beltran, Christopher J. Sweeney, H. He, Myles Brown, Jasmine T. Plummer, Simon R V Knott, David P. Labbé, Leigh Ellis
          </td>
          <td>2025-08-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4f390eea56caf49eb99fd6eb79ecae7370b3e6e" target='_blank'>
              Leveraging synthetic genetic array screening to identify therapeutic targets and inhibitors for combatting azole resistance in Candida glabrata.
              </a>
            </td>
          <td>
            Catrin C Williams, Jane Usher
          </td>
          <td>2025-08-11</td>
          <td>Microbiology spectrum</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe4a2e884a937abc2345c5e03e509d49bf56bc09" target='_blank'>
              Transcriptome-wide decoding the roles of aberrant splicing in melanoma MAPK-targeted resistance evolution.
              </a>
            </td>
          <td>
            Jing Yu, Xiujing He, Xueyan Wang, Chune Yu, Xian Jiang, Yanna Li, Xinyu Liu, Ya Luo, Xuemei Chen, Sisi Wu, Lu Si, Jing Jing, Xuelei Ma, Hubing Shi
          </td>
          <td>2025-07-18</td>
          <td>EMBO reports</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background: Despite remarkable progress in clinical management of patients and intensified screening, colorectal cancer remains the second most common cause of cancer-related death in the United States. The recent surge of next generation sequencing has enabled genomic analysis of colorectal cancer genomes. However, to date, there is little information about leveraging gene expression data and integrating it with somatic mutation information to discover potential biomarkers and therapeutic targets. Here, we integrated gene expression data with somatic mutation information to discover potential diagnostic and prognostic biomarkers and molecular drivers of colorectal cancer. Methods: We used publicly available gene expression and somatic mutation data generated on the same patient populations from The Cancer Genome Atlas. We compared gene expression data between tumors and controls and between dead and alive. Significantly differentially expressed genes were evaluated for the presence of somatic mutations and subjected to functional enrichment analysis to discover molecular networks and signaling pathways enriched for somatic mutations. Results: The investigation revealed a signature of somatic mutated genes transcriptionally associated with colorectal cancer and a signature of significantly differentially expressed somatic mutated genes distinguishing dead from alive. Enrichment analysis revealed molecular networks and signaling pathways enriched for somatic mutations. Conclusions: Integrative bioinformatics analysis combining gene expression with somatic mutation data is a powerful approach for the discovery of potential diagnostic and prognostic biomarkers and potential drivers of colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b07360f9e176b06a8f4430f9efcd7e8e8c345f47" target='_blank'>
              An Integrative Genomics Approach for the Discovery of Potential Clinically Actionable Diagnostic and Prognostic Biomarkers in Colorectal Cancer
              </a>
            </td>
          <td>
            Mark Fertel, D. Alawad, Chindo Hicks
          </td>
          <td>2025-07-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Circulating tumor cells (CTCs) are linked to cancer progression and poor prognosis, offering valuable genetic insights into tumors. Accurate detection of genomic alterations in CTCs is essential for improving cancer diagnosis and treatment. To address this, we develop Uniform Chromosome Conformation Capture (Uni-C), a method for profiling 3D chromatin architecture and genomic alterations at the single-cell level. Using Uni-C, we analyze CTCs from pancreatic cancer patient-derived xenograft (PDX) and spontaneous tumor mouse models. In the PDX model, integrating data from seven CTCs captures 88.7% of SNPs and INDELs, and 75.0% of structural variants present in tumor tissue. These findings indicate that variants detected in CTCs reflect tumor genomic features. Notably, we observe chromatin conformation differences between mitotic and interphase CTCs, suggesting potential markers of cell vitality. In the spontaneous tumor model, we identify driver gene mutations in CTCs and predict neoantigens, advancing early cancer detection and treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcdb93a8b0e33fd2f466feefe41fc91b323a6623" target='_blank'>
              Identification of multiple genomic alterations and prediction of neoantigens from circulating tumor cells at the single-cell level
              </a>
            </td>
          <td>
            Xiaochen Gao, Xinyu Li, Weize Xu, Ming Jiao, Yu Guo, Jiajia Wang, Weihao Wang, Jiling Feng, Qianqian Guo, Chengchao Wu, Taiyu Zhang, Yuqin Yang, Da Lin
          </td>
          <td>2025-07-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ffde0bbfbd90bb0bf5e018161ce107f39f7740f" target='_blank'>
              Assessing the current molecular understanding of therapeutic targets in osteosarcoma.
              </a>
            </td>
          <td>
            Chao Zhang, Ji-Lan Yang
          </td>
          <td>2025-08-07</td>
          <td>Expert opinion on therapeutic targets</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Metastatic uveal melanoma is an aggressive disease with limited effective therapeutic options. To comprehensively map monogenic and digenic dependencies, we performed CRISPR–Cas9 screening in ten extensively profiled human uveal melanoma cell line models. Analysis involved genome-wide single-gene and combinatorial paired-gene CRISPR libraries. Among our 76 uveal melanoma-specific essential genes and 105 synthetic lethal gene pairs, we identified and validated the CDP-diacylglycerol synthase 2 gene (CDS2) as a genetic dependency in the context of low CDP-diacylglycerol synthase 1 gene (CDS1) expression. We further demonstrate that CDS1/CDS2 forms a synthetic lethal interaction in vivo and reveal that CDS2 knockout results in the disruption of phosphoinositide synthesis and increased cellular apoptosis and that re-expression of CDS1 rescues this cell fitness defect. We extend our analysis using pan-cancer data, confirming increased CDS2 essentiality in diverse tumor types with low CDS1 expression. Thus, the CDS1/CDS2 axis is a therapeutic target across a range of cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cea3171a10bd393fb0f6c1167b72877c7aa37fbb" target='_blank'>
              The synthetic lethal interaction between CDS1 and CDS2 is a vulnerability in uveal melanoma and across multiple tumor types
              </a>
            </td>
          <td>
            Pui Ying Chan, Diana Alexander, Ishan Mehta, Larissa Satiko Alcantara Sekimoto Matsuyama, V. Harle, R. Olvera-León, Jun Sung Park, Fernanda G Arriaga-González, L. van der Weyden, S. Cheema, Vivek Iyer, Victoria Offord, David Barneda, Phillip Hawkins, Len Stephens, Z. Kozik, Michael Woods, K. Wong, G. Balmus, A. Vinceti, N. A. Thompson, M. Del Castillo Velasco-Herrera, L. Wessels, J. van de Haar, Emanuel Gonçalves, Sanju Sinha, M. E. Vázquez‐Cruz, L. Bisceglia, F. Raimondi, J. Choudhary, Sumeet Patiyal, Anjan Venkatesh, F. Iorio, Colm J. Ryan, D. J. Adams
          </td>
          <td>2025-07-04</td>
          <td>Nature Genetics</td>
          <td>1</td>
          <td>40</td>
        </tr>

        <tr id="Drug resistance and distant metastases are leading causes of mortality in colorectal cancer (CRC), yet the molecular mechanisms linking these processes remain elusive. In this study, we demonstrate that acquired resistance to oxaliplatin, a first-line chemotherapeutic in CRC, enhances metastatic potential through transcriptional reprogramming. Using a clinically relevant dosing regimen, we generated oxaliplatin-resistant CRC cells that displayed increased metastatic potential. Integrated transcriptomic and phenotypic analyses revealed that dysregulated cholesterol biogenesis amplifies TGF-β signaling, which in turn drives expression of SERPINE1, which serves as a key effector of both oxaliplatin resistance and metastasis. Furthermore, we uncovered a SERPINE1-associated nine-gene expression signature, RESIST-M, that robustly predicts overall and relapse-free survival across distinct patient cohorts. Notably, RESIST-M stratifies a high-risk subtype of CMS4/iCMS3-fibrotic patients that display the poorest prognosis, underscoring its clinical relevance. Targeting of SERPINE1 or cholesterol biosynthesis re-sensitized resistant, pro-metastatic cells to oxaliplatin in mouse xenograft models. Altogether, this study uncovers a mechanistic link between metabolic rewiring and transcriptional plasticity underlying therapy-induced metastasis in primary CRC. Additionally, it also reveals actionable vulnerabilities that offer both prognostic value and therapeutic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/461233a4140dd31d355e46865c3174f542e2818f" target='_blank'>
              Modeling oxaliplatin resistance in colorectal cancer reveals a SERPINE1-based gene signature (RESIST-M) and therapeutic strategies for pro-metastatic CMS4 subtype
              </a>
            </td>
          <td>
            Stephen Qi Rong Wong, Mohua Das, Kenzom Tenzin, Niranjan Shirgaonkar, Huiwen Chua, Lin Xuan Chee, Ahpa Sae Yeoh, Astley Aruna Murugiah, Wei Yong Chua, Madelaine Skolastika Theardy, Ethan Subel, M. Thangavelu, Jane Vin Chan, Choon-Kong Yap, Iain Bee Huat Tan, Petros K Tsantoulis, Sabine Tejpar, J. Loo, Ramanuj Dasgupta
          </td>
          <td>2025-07-16</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Targeted next generation sequencing (NGS) using multigene panels has become an effective tool for comprehensive genomic analysis in cancer, overcoming limitations of single gene assays. Nonetheless, outsourcing these assays to external laboratories and the extended turnaround time (~ 3 weeks) required for obtaining results may impede timely clinical management of cancer patients. We developed an oncopanel targeting 61 cancer-associated genes and validated its efficacy by performing NGS on 43 unique samples including clinical tissues, external quality assessment samples, and reference controls. The assay detected 794 mutations including all 92 known variants from orthogonal methods. Overall performance measures of the assay showed 99.99% repeatability and 99.98% reproducibility. Likewise, sensitivity to detect unique variants was 98.23%, with specificity at 99.99%, precision at 97.14% and accuracy at 99.99% all at 95% CI. Notably, clinically actionable mutations were observed in key genes such as KRAS, EGFR, ERBB2, PIK3CA, TP53 and BRCA1. The average turnaround time from sample processing to results was reduced to 4 days. These findings demonstrate a sensitive, high throughput oncopanel that is suitable for use in routine clinical testing. The shorter turnaround time of the assay has the potential to significantly improve patient care by facilitating more timely and personalized clinical interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c54e17f4c7bc3e73bffed6c54c9ae3d62ae5d76" target='_blank'>
              A targeted next-generation sequencing panel for identification of clinically relevant mutation profiles in solid tumours
              </a>
            </td>
          <td>
            Kakoli Das, Mandy Li Ian Tay, Elena Yaqing Yong, K. Chuah
          </td>
          <td>2025-07-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Introduction Balanced chromosomal abnormalities (BCAs) are structural variations that can underlie a wide spectrum of neurodevelopmental disorders, often remaining undetected by conventional diagnostic approaches. Whole-genome sequencing (WGS) allows for base-pair resolution of structural variants across the entire genome, making it a powerful tool to detect cryptic chromosomal rearrangements and refine breakpoint mapping. RNA sequencing (RNA-Seq), by enabling the detection of gene expression changes and fusion transcripts, provides complementary functional insights into the consequences of genomic alterations. This study integrated WGS and RNA-Seq to precisely characterize the breakpoints and assess the functional impact of de novo BCAs in two unsolved cases of Neurodevelopmental Disorders. Materials and methods Short read WGS was used to identify the chromosomal breakpoints and gene disruptions caused by BCAs. RNA-Seq on blood RNA was employed to detect differential gene expression and potential fusion transcripts of disrupted genes. Results In the first case, the inversion inv(8) (p11.2q13) disrupted two genes at the breakpoints, namely, CHD7 and SLC20A2. These genes are in opposite orientations, and the inversion realigned them in the same direction, generating two novel fusion genes. Disruption of CHD7 confirmed the suspected diagnosis of CHARGE syndrome. The interruption of SLC20A2, commonly associated with neurological symptoms, prompted further clinical evaluation. RNA-Seq identified in-frame fusion transcripts from the chimeric genes in the blood, suggesting a potential deleterious phenotypic effect. In the second case, WGS revealed a balanced translocation t(17; 22) (q25; q13) that disrupted EP300 at 22q25, confirming Rubinstein-Taybi syndrome. The concurrent disruption of RBFOX3 at 17q13 suggested additional neurological implications, particularly related to epilepsy. Transcriptomic analysis demonstrated the monoallelic and significantly reduced expression of EP300. Conclusion These findings highlight the crucial role of WGS in identifying disease-associated BCAs and underscore the complementary value of RNA-Seq in assessing their functional consequences. This integrated approach enhanced diagnostic accuracy and clinical management, paving the way for more comprehensive and personalized care in these two patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d8f54f85ebf927d3f8b7d5f4a6236e7f717fa51" target='_blank'>
              Integrating genome and transcriptome analysis to decipher balanced structural variants in unsolved cases of neurodevelopmental disorders
              </a>
            </td>
          <td>
            S. Mellone, Alice Spano, Denise Vurchio, Giulia Borgonovi, Alessandro Ugonotti, Giulia Paglino, Alba Bianco, Sara Ronzani, Maurizio Sciancalepore, Flavia Prodam, Amanda Papa, Maurizio Viri, Umberto Dianzani, Mara Giordano
          </td>
          <td>2025-07-07</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="In ~70% of patients with pancreatic ductal adenocarcinoma, the TP53 gene acquires gain-of-function (GOF) mutations leading to rapid disease progression. Specifically, missense p53 (misp53) GOF mutations associate with therapy resistance and worse clinical outcomes. However, the molecular functions of distinct misp53 mutants in plasticity and therapy response remain unclear. Integrating multicenter patient data and multi-omics, we report that the misp53R273H/C mutant is associated with cell cycle progression and a basal-like state compared to the misp53R248W/Q mutant. Loss of misp53R273H/C decreased tumor growth and liver metastasis while prolonging survival in preclinical models. We found that misp53R273H/C specifically regulated the Rb/DREAM axis involved in cell cycle regulation. Notably, a clinical CDK4/6 inhibitor reduced misp53R273H/C mutant expression. However, it triggered MAPK/ERK-mediated resistance mechanisms, enhancing cell survival and resistance to CDK4/6 inhibitors. Combining MAPK/ERK and CDK4/6 inhibitors reduced misp53R273H/C-associated oncogenic functions. Thus, distinct misp53 mutants show unique cell-intrinsic plasticity, therapeutic vulnerabilities, and resistance mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8366ad2cf60feed0070f18be69dfac6ab275f732" target='_blank'>
              TP53 missense–specific transcriptional plasticity drives resistance against cell cycle inhibitors in pancreatic cancer
              </a>
            </td>
          <td>
            Laura Urbach, Lena Wieland, Frederike Penz, Rebecca Diya Samuel, Stefan Küffer, Lukas Klein, Christof Lenz, Ulrich Sax, Michael Ghadimi, Ramona Schulz-Heddergott, E. Hessmann, V. Ellenrieder, Nelson Dusetti, Shiv K. Singh
          </td>
          <td>2025-07-04</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Breast cancer continues to be the primary cause of cancer-related deaths among women globally, with increased rates of incidence and mortality, highlighting the critical need for effective treatment strategies. Recent developments have introduced a variety of treatment options that address the molecular diversity of breast cancer; nonetheless, drug resistance remains a significant barrier to achieving favorable results. This review explains the crucial role of genetic and epigenetic changes in contributing to therapeutic resistance, in addition to other factors such as increased drug efflux, enhanced DNA repair, evasion of senescence, tumor heterogeneity, the tumor microenvironment (TME), and epithelial-to-mesenchymal transition (EMT). Genetic modifications, including mutations in oncogenes and tumor suppressor genes, disrupt essential signaling pathways, facilitating resistance to chemotherapy and targeted therapies. At the same time, epigenetic modifications - like DNA methylation, alterations to histones, and dysregulation of non-coding RNAs - reprogram gene expression, supporting adaptive resistance mechanisms. These molecular abnormalities contribute to the plasticity of tumors, allowing cancer cells to evade therapeutic approaches. This review consolidates recent discoveries regarding how these genetic and epigenetic modifications affect treatment responses and resistance in breast cancer, highlighting their interaction with disease advancement. By pinpointing new drug targets, including immunotherapeutic strategies, this article seeks to shed light on the molecular underpinnings of chemoresistance, aiding in the refinement of existing treatment protocols. A more profound understanding of these mechanisms offers the potential for developing precision therapies to overcome resistance, reduce relapse rates, and improve clinical outcomes for breast cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f843678da290bd94b0c5bd14d4941f6b50d5f9a" target='_blank'>
              Decoding breast cancer treatment resistance through genetic, epigenetic, and immune-regulatory mechanisms: from molecular insights to translational perspectives
              </a>
            </td>
          <td>
            Suryendu Saha, Samikshya Mahapatra, Sinjan Khanra, Barnalee Mishra, Biswajit Swain, Diksha Malhotra, Swarnali Saha, Venketesh K Panda, Kavita Kumari, Sarmistha Jena, Sandeep Thakur, Pawan K. Singh, Gopal C Kundu
          </td>
          <td>2025-07-21</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="ABSTRACT KRAS mutations are among the most prevalent oncogenic drivers in non-small cell lung cancer (NSCLC), yet the mechanisms of therapeutic resistance to KRAS inhibitors in these cancers remains poorly understood. Here, we deploy high-throughput CRISPR base editing screens to systematically map resistance mutations to three mechanistically distinct KRAS-targeted therapies, including KRAS-G12C(OFF) inhibitor (adagrasib), RAS(ON) G12C-selective tri-complex inhibitor (RMC-4998), and RAS(ON) multi-selective tri-complex inhibitor (RMC-7977). Using both a saturation Kras tiling approach and cancer-associated mutation library, we identify common and compound-selective second-site resistance mutations in Kras , as well as gain-of-function and loss-of-function variants across cancer-associated genes that rewire signaling networks in a context-dependent manner. Notably, we identify a recurrent missense mutation in capicua ( Cic ), that promotes resistance to RMC-7977 in vitro and in vivo. Moreover, we show that targeting NFκB signaling in CIC-mutant cells can resensitize them to RAS pathway inhibition and overcome resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/597a0301e9cbbccb635fb6b4893ce93089ff3713" target='_blank'>
              Genetic mechanisms of resistance to targeted KRAS inhibition
              </a>
            </td>
          <td>
            Bianca J. Diaz, Max Kops, Sara Bernardo, Henri Schmidt, Elizabeth Grankowsky, Adrian Vega, Chen Zhang, Matthew Bott, Maria Skamagki, Aidan C A Tomlinson, Nicole A Vita, Alyna Katti, Mark Labrecque, Ida Aronchik, Mallika Singh, Lukas E. Dow
          </td>
          <td>2025-08-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Therapeutic resistance in cancer is increasingly understood as a dynamic outcome of the interplay among tumour metabolism, epigenetic regulation, and immune modulation, rather than a consequence of genetic mutations alone. At this intersection, lactylation-a recently discovered lactate-derived post-translational modification (PTM)-acts as a molecular bridge linking metabolic rewiring to transcriptional and post-transcriptional control. Meanwhile, microRNAs (miRNAs) have emerged as essential regulators of metabolic adaptation and epigenetic plasticity. However, the interaction between lactylation and miRNA networks remains largely underexplored. Herein, we present a comprehensive and structured assessment of this emerging interdisciplinary field. We begin with a focused overview of tumour metabolic reprogramming and the enzymology of lactylation, establishing the biochemical context for its regulatory functions. We then examine how miRNAs interpret and reinforce metabolic cues, particularly through three interrelated regulatory frameworks. Lactylation of histones and DNA repair proteins has been shown to activate oncogenic miRNA clusters; these same miRNAs, in turn, enhance glycolytic flux and fine-tune the activity of lactylation enzymes, forming bidirectional feedback loops. Under stress conditions such as hypoxia or chemotherapy, lactate accumulation selectively suppresses DNA-repair-targeting and Pro-apoptotic miRNAs, stabilising BRCA1/RAD51-mediated repair programs. Furthermore, lactylation-dependent miRNA signals disseminate via extracellular vesicles, contributing to T cell exhaustion and macrophage M2 polarisation, thus shaping an immunosuppressive tumour microenvironment. Based on these converging mechanisms, we highlight potential therapeutic strategies, including the co-targeting of LDHA and oncogenic miRNAs (e.g., nanoparticle-mediated delivery of anti-miR-21), as well as liquid biopsy-based monitoring of circulating H3K18la and miRNA signatures to predict emerging resistance. By integrating metabolic, epigenetic, and immunologic perspectives, we position lactylation-miRNA crosstalk as a central regulatory axis in cancer therapy resistance and offer a conceptual framework to inform the development of future mechanism-driven interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3528e4d33d5572d7ac2200199d0f17c55ff6aa48" target='_blank'>
              Lactylation-mediated miRNA regulation in cancer therapy resistance
              </a>
            </td>
          <td>
            Yiyi Shou, Ruiqi Liu, Hao Xiong, Xiaoyan Chen, Luanluan Huang, Ran Huang, Hailong Sheng, Haibo Zhang, Yanwei Lu, Haiwei Guo
          </td>
          <td>2025-08-19</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="We present a targeted sequencing-based pipeline that profiles microsatellite instability (MSI) at single-nucleotide resolution. Targeted amplicons from the five widely studied Bethesda panel microsatellite loci were sequenced using Oxford Nanopore Technology in two microsatellite unstable colorectal cancer cell lines (HCT15, HCT116), two microsatellite stable cancer cell lines (TK6, U2OS), and two peripheral blood mononuclear cell samples from healthy donors. An anchor-extension algorithm was developed to capture repeat motifs while allowing interruptions, using a threshold informed by platform-specific error. Cluster-aware Dirichlet-multinomial and beta-binomial tests were applied for between-sample comparisons while accounting for read-level clustering within samples. The algorithm revealed distinct repeat profiles in HCT15 and HCT116 compared to other cell types and uncovered allelic diversity across samples at different MSI loci. Our approach complements existing short tandem repeat callers by preserving read-level diversity and delivering targeted, quantitative MSI calls with potential applications in mechanistic research and clinical assay development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39f6c719d9a7134526351d0fb156b95a3a524126" target='_blank'>
              MSIanalyzer: Targeted Nanopore Sequencing Enables Single Nucleotide Resolution Analysis of Microsatellite Instability Diversity
              </a>
            </td>
          <td>
            Ting Zhai, D. Laverty, Zachary D. Nagel
          </td>
          <td>2025-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background Comprehensive analysis of single-cell transcriptome and chromatin accessibility will contribute to interpret the heterogeneity of acute myeloid leukemia (AML). We hypothesize that integrating single-cell transcriptomic and chromatin accessibility landscapes underlying t(8;21) AML will provide valuable insights into its heterogeneous cellular properties and gene regulatory programs. Methods Here, we conducted paired single-cell RNA-sequencing (scRNA-seq) and single-cell ATAC sequencing on bone marrow samples from newly diagnosed t(8;21) AML patients and healthy controls. Genetic signatures extracted from scRNA-seq were built and validated across three independent cohorts (German AMLCG1999, GSE106291 and TCGA LAML). Results We identified TCF12, a core component of AML1-ETO-containing transcription factor complex (AETFC), as the most active transcription factor in blast cells, driving a universally repressed chromatin state. Furthermore, we delineated two functionally distinct T cell subsets, revealing that EOMES-mediated transcriptional regulation promotes the expansion of a cytotoxic T cell population (T cells_2; high GNLY, NKG7 and GZMB expression), with an increased clonality and a tendency for drug resistance. In addition, we discovered a novel leukemic CMP-like cluster characterized by high TPSAB1, HPGD and FCER1A expression. Leveraging machine learning-based integration of multi-omic profiles, we identified a robust 9-gene prognostic signature, which demonstrated significant predictive value for AML outcomes across three independent cohorts. Conclusions This multi-omics study provides unprecedented insights into the transcriptional and epigenetic heterogeneity of t(8;21) AML, providing a potential actionable tool for clinical risk stratification. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06659-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb6d672d2579a9875bb647a603a272b4cc733a93" target='_blank'>
              A multi-omic single-cell landscape reveals transcription and epigenetic regulatory features of t(8;21) AML
              </a>
            </td>
          <td>
            Xue-Ping Li, Yan Gao, Bai-Tian Zhao, Yujun Dai, J. Mao, Yang Liang, Lu Jiang
          </td>
          <td>2025-07-24</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Despite more than two decades since the completion of the first draft of the Human Genome Project, a substantial proportion of human genes remain poorly characterized in terms of their functions. Functional genomics aims to elucidate the roles and interactions of genes and genetic elements, providing insights into their involvement in various biological processes. In this context, the perturbomics approach—a systematic analysis of phenotypic changes resulting from gene function modulation—offers valuable insights into the function of unannotated genes. With the advent of CRISPR–Cas-based genome and epigenome editing, CRISPR screens have become the method of choice for perturbomics studies, enabling the identification of target genes whose modulation may hold therapeutic potential for diseases such as cancer, cardiovascular disorders and neurodegeneration. These findings contribute to the development of targeted drug therapies and the design of gene and cell therapies for regenerative medicine. Here we highlight recent technical advances in CRISPR-based perturbomics, focusing on more physiologically relevant, single-cell-level analyses and their successful applications in discovering novel therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e6e66e4709148af526446d85f7955b03efede5a" target='_blank'>
              Perturbomics: CRISPR–Cas screening-based functional genomics approach for drug target discovery
              </a>
            </td>
          <td>
            Byung-Sun Park, Mieun Lee, Jaeyeol Kim, Tackhoon Kim
          </td>
          <td>2025-07-01</td>
          <td>Experimental & Molecular Medicine</td>
          <td>2</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1eb5b9806f99885b27e221a4649bd483b17f7a1" target='_blank'>
              Optimizing treatment for metastatic castration-resistant prostate cancer: Food and Drug Administration-approved therapies, emerging strategies, and biomarker-driven approaches.
              </a>
            </td>
          <td>
            Faisal Saeed, Jacob E. Berchuck, M. Bilen, Jatin S. Gandhi, B. Nazha, Jacqueline T. Brown, David M Schuster, Ashesh B. Jani, Jindan Yu, Lara Harik
          </td>
          <td>2025-08-15</td>
          <td>Cancer</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="The dynamic nature of longitudinal tumor evolution across patients presents challenges in designing effective drugs. Here we aimed to determine tumor evolution resistance mechanisms and explore candidate drugs for specific tumor evolution types. We conducted longitudinal pharmacogenomic analysis of datasets of 73 samples in 34 patients among a National Cancer Center refractory lung cancer cohort (n = 199). Genomic profiles were determined to identify evolutionary trees in each patient, which were classified into tumor evolution groups according to the predominant truncal mutations, TP53 and epidermal growth factor receptor. These groups were categorized into persistence, extinction and expansion groups according to the status of these two clones. Pharmacogenomic profile analysis identified that XAV-939 was effective for the epidermal growth factor receptor-extinction group exhibiting epithelial-to-mesenchymal transition-activated resistance. In addition, MYC+ subclones were maintained similarly to drug-tolerant residual cells throughout the evolution period. Moreover, MYC+ lung adenocarcinoma showed a poor outcome and had higher risk of transformation to small-cell lung cancer. Furthermore, the epithelial-to-mesenchymal transition-activated and MYC+ subclones were implicated in concurrent epidermal growth factor receptor–tyrosine kinase inhibitor resistance. Finally, our drug screening identified barasertib, an aurora kinase inhibitor, as a triple-combination candidate with epidermal growth factor receptor–tyrosine kinase inhibitors and XAV-939 for MYC+ cells. This study demonstrates the utility of longitudinal pharmacogenomic analysis to develop treatment strategies according to individual tumor evolution type. The study underscores the importance of integrating genomic and pharmacogenomic profiling in clinical practice to tailor treatments according to tumor evolution type.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95eff8844cf7d953e515c95691ab37e19818828b" target='_blank'>
              Longitudinal pharmacogenomic analysis of refractory lung cancer to identify therapeutic candidates for epidermal growth factor receptor–tyrosine kinase inhibitor resistance subclones
              </a>
            </td>
          <td>
            Namhee Yu, Mihwa Hwang, B. Ahn, Youngjoo Lee, Sehwa Hong, Hanna Sim, Bo Ram Song, Sunshin Kim, Charny Park, Ji-Youn Han
          </td>
          <td>2025-07-01</td>
          <td>Experimental & Molecular Medicine</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Lung cancer remains a leading cause of cancer-related mortality due to its capacity for silent metastasis and the significant challenges in achieving effective treatment. Currently, targeted therapies and chemotherapies are the primary options for advanced or inoperable lung cancer; however, their efficacy is often undermined by the cancer's ability to develop resistance through both genetic and non-genetic mechanisms. This review explores recent advances in understanding metabolic reprogramming in non-small cell lung cancer (NSCLC), focusing on its critical role in cancer progression. NSCLC cells exhibit heterogeneous activation of metabolic pathways influenced by their oncogenic mutations. Notably, their metabolic phenotypes evolve in response to environmental stressors and therapeutic pressures. Moreover, NSCLC cells engage in metabolic crosstalk with their microenvironment to enhance survival, leveraging distinct metabolic adaptations at both primary and metastatic sites. Despite extensive preclinical studies evaluating novel therapeutic strategies targeting these metabolic pathways, many have failed in clinical trials due to severe adverse effects. This is because the targeted pathways are crucial not only for cancer cells but also for normal cellular functions. Future research must prioritize approaches that selectively disrupt cancer-specific metabolic regulation to improve therapeutic outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a854e0477bbe5fbd82c8f4b77d0195460cff4ca2" target='_blank'>
              The Emerging Roles of Metabolic Reprogramming in Non-Small Cell Lung Cancer Progression.
              </a>
            </td>
          <td>
            Beatriz P Peixoto, Rebecca A Clague, Joshua P Reddy, Hiromi I Wettersten
          </td>
          <td>2025-07-08</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bde1cb673888229ad9d05dcf1add5eeba27a1400" target='_blank'>
              Alternative splicing generates HER2 isoform diversity underlying antibody-drug conjugate resistance in breast cancer.
              </a>
            </td>
          <td>
            G. Guardia, C. D. Dos Anjos, Aline Rangel-Pozzo, F. F. dos Santos, A. Birbrair, P. Asprino, A. A. Camargo, P. Galante
          </td>
          <td>2025-07-15</td>
          <td>Genome research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6cd705c1549c5bd8681f0a403abe74109eee0036" target='_blank'>
              Advancements in Targeted Therapies and Pharmacogenomics for Personalized Breast Cancer Treatment: The Role of Gene SNPs in Treatment Resistance.
              </a>
            </td>
          <td>
            D. Tripathi, Neal M Davies, P. S. Rajinikanth, Prashant Pandey
          </td>
          <td>2025-06-27</td>
          <td>Current gene therapy</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The spatial organization of the genome within the nucleus is a fundamental regulator of gene expression, genome stability, and cell identity. This review addresses the central question of how nuclear genome architecture contributes to disease mechanisms and diagnostics, and how technological advances enable its clinical exploration. We first outline the principles of nuclear genome architecture, including chromosome territories, replication timing, and 3D domains, and their role in gene regulation and disease. We then explore the mechanisms and consequences of chromosomal rearrangements, and how replication dynamics intersect with epigenetic regulation and genome stability. Diagnostic tools are presented in chronological progression, from conventional cytogenetics to high-resolution genomic and single-cell techniques. A dedicated section focuses on cancer cytogenomics and its clinical implications. We further highlight emerging technologies for 3D genome and epigenome profiling and their integration into diagnostic workflows. Finally, we discuss current challenges, such as standardization and cost, and the transformative potential of multi-omics and artificial intelligence for future precision diagnostics. Overall, we provide a comprehensive overview of how cytogenetics and cytogenomics contribute to the understanding and clinical diagnosis of genetic and neoplastic diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0253c2480a5ecc52bcd5c1eda31a7ec8e922ab1" target='_blank'>
              Cytogenetics and Cytogenomics in Clinical Diagnostics: Genome Architecture, Structural Variants, and Translational Applications
              </a>
            </td>
          <td>
            Concetta Federico, Desiree Brancato, Francesca Bruno, Elvira Coniglio, V. Sturiale, Salvatore Saccone
          </td>
          <td>2025-06-30</td>
          <td>Genes</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4eaa0126e7f9afd60d57ab46895cddf6ea4cbe3" target='_blank'>
              Pangenome-based identification of cryptic pathogenic variants in undiagnosed rare disease patients
              </a>
            </td>
          <td>
            S. S. Jang, S. Kim, S. Lee, S. Kim, J. Moon, J. Kim, J. Chae
          </td>
          <td>2025-07-11</td>
          <td>None</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Tumor drug resistance represents a major challenge in contemporary cancer therapeutics, significantly compromising the clinical efficacy of chemotherapy, targeted therapy, and immunotherapy. While existing research has elucidated the critical role of tumor cell-intrinsic mechanisms in drug resistance—including genomic instability, persistent activation of signaling pathways and aberrant epigenetic modifications—emerging evidence highlights the crucial involvement of dynamic remodeling within the tumor microenvironment (TME) in driving therapeutic resistance. The TME fosters drug resistance through dynamic remodeling, creating hypoxic conditions, immunosuppressive networks, and metabolic stress, which collectively impair treatment response and promote therapeutic escape. Advances in multi-omics technologies now enable a comprehensive, multi-dimensional analysis of these interactions, integrating genomic, epigenomic, transcriptomic, proteomic, and metabolomic data to uncover critical molecular networks and vulnerabilities. In this review, we explore the key mechanisms by which the TME influences drug resistance, discuss how multi-omics approaches enhance our understanding of these processes and evaluate emerging therapeutic strategies aimed at reprogramming the TME to overcome resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0679c6b5fd0da9cf7502220f6fe761b9cdab9b5" target='_blank'>
              Multi-omics dissection of tumor microenvironment-mediated drug resistance: mechanisms and therapeutic reprogramming
              </a>
            </td>
          <td>
            Fanghua Chen, Yuandong Fu, Gaigai Bai, J. Qiu, Keqin Hua
          </td>
          <td>2025-07-07</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="With advancements in technology, single-cell RNA sequencing has emerged as a power-ful tool in cancer drug discovery. This technique enables the construction of gene expression profiles at the individual cell level, offering detailed insights into cellular heterogeneity and molecular path-ways involved in tumor development. It enables researchers to gain a deeper understanding of tumor heterogeneity. Researchers can study cell subpopulations and gene expression patterns. This under-standing helps in identifying potential drug targets. Additionally, it aids in predicting therapeutic responses. This high-resolution gene expression analysis provides a new perspective and oppor-tunity for cancer drug development, which is expected to accelerate the discovery and development process of new anti-cancer drugs. This article provides a comprehensive overview of the basic pro-cesses and developmental trajectory of single-cell RNA sequencing technology, with a particular emphasis on its applications in various aspects of cancer drug discovery. It also addresses the chal-lenges faced by single-cell RNA sequencing and potential future directions. This review aims to enhance readers' understanding of single-cell sequencing, inspire new ideas for oncology drug de-velopment, and support the translation of clinical research into practice, ultimately enabling physi-cians to design more precise and personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f74b7c7821b0c57a9649d29962bfb630eb54595" target='_blank'>
              Single-Cell Transcriptomics: Technical Advances, Applications and Challenges in Cancer Drug Discovery.
              </a>
            </td>
          <td>
            Yueying Yang, Lingyu Meng, Teng Zhang, Jianjun Tan
          </td>
          <td>2025-07-30</td>
          <td>Current cancer drug targets</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Dysregulated RNA modifications contribute to cancer progression and therapy resistance, yet the underlying mechanism often remains unknown. Here, we perform CRISPR-based synthetic lethality screens to systematically explore the role of RNA modifications in mediating resistance to anti-leukaemic drugs. We identify the TRMT5-mediated formation of N1-methylguanosine (m1G) in the tRNA anticodon loop as essential for mediating drug tolerance to cytarabine and venetoclax in acute myeloid leukemia (AML). TRMT5 methylates nearly all mitochondrial and nuclear tRNAs with a guanosine at position 37, but its role in promoting drug tolerance specifically depends on its mitochondrial function. TRMT5 is essential for the dynamic upregulation of mitochondrial mRNA translation and oxidative phosphorylation (OXPHOS), which are critical for sustaining drug tolerance in leukemia cells. This mitochondrial dependency correlates with therapy outcomes in leukemia patients: lower expression of electron transport chain genes is linked to poorer outcomes in a cohort of nearly 100 AML patients undergoing first induction therapy. Finally, we demonstrate that targeted depletion of TRMT5 protein using a conditional degron, in conjunction with cytarabine and venetoclax treatment, synergistically induces cell death in drug-tolerant AML cells. Thus, our study reveals TRMT5 as promising drug target for therapy-resistant leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5f65b910416a4ff6ee4895803cf1d0c098e3d11" target='_blank'>
              Disrupting tRNA modifications to target mitochondrial vulnerabilities in drug-resistant leukemia cells.
              </a>
            </td>
          <td>
            C. Pauli, M. Kienhöfer, M. Blank, Oguzhan Begik, Christian Rohde, Sarah Mn Zimmermann, Laura Werner, D. Heid, Fu Xu, Katharina Weidenauer, Sylvain Delaunay, Nadja Krall, Katrin Trunk, Duoduo Zhao, Fengbiao Zhou, Laia Llovera, Ollivier Alexane, Anke Heit-Mondrzyk, U. Platzbecker, C. Baldus, H. Serve, M. Bornhäuser, C. Vågbø, S. Benitah, Jeroen Krijgsveld, Eva Maria Novoa, Carsten Müller-Tidow, Michaela Frye
          </td>
          <td>2025-08-01</td>
          <td>Blood</td>
          <td>0</td>
          <td>80</td>
        </tr>

        <tr id="GBM is an aggressive primary malignant brain tumor that has a poor prognosis. Molecular characterization of GBM has shown that EGFR mutations are present in over 50% of tumors. However, EGFR inhibitors have not shown clinical efficacy in contrast to other EGFR-driven neoplasms due to the unique EGFR biology found in GBM. Upfront combinatorial therapy featuring EGFR tyrosine kinase inhibitors (TKI) may overcome these challenges. To identify combinatorial drug targets within the kinome, we temporally characterized drug-induced kinome rewiring in isogenic, genetically engineered Cdkn2a-deleted mouse astrocytes expressing human EGFRvIII. We utilize RNA sequencing and multiplex inhibitor beads, coupled with mass spectrometry, to demonstrate that kinome rewiring exhibits both shared and unique kinases after acquired resistance develops to EGFR TKI, despite using models with a common genetic background. Additionally, we noted that kinases altered in the acute setting are distinct from those in acquired resistance. By identifying kinome vulnerabilities throughout the acute, dynamic drug response process, we generated a kinase signature associated with EGFR inhibition. Further molecular interrogation of signature genes revealed that drug treatment induces an unexpected increase in Cdk6 protein, but not mRNA, despite live cell imaging and transcriptomic evidence indicating decreased proliferation. Survival experiments with orthotopic allografts show that upfront combination inhibition of Cdk6, using abemaciclib, and EGFR, using neratinib, significantly prolonged median survival compared to neratinib alone. Our findings suggest that identifying and inhibiting targets with synthetic lethality in the upfront combinatorial setting is a viable approach for precision oncology and may help provide an avenue to overcome the resistance mechanisms that contributed to the failures of EGFR as a molecular target in GBM. Supplementary Information The online version contains supplementary material available at 10.1186/s40478-025-02068-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a56593dc844cb2719b95171b4134828b89a3d2b" target='_blank'>
              Identifying and exploiting combinatorial synthetic lethality by characterizing adaptive kinome rewiring of EGFRvIII-driven glioblastoma
              </a>
            </td>
          <td>
            Acta Neuropathologica, Benjamin Lin, Abigail K. Shelton, Ryan Miller, E. Smithberger, Julia K Ziebro, Kasey R. Skinner, R. Bash, Richard Kirkman, A. Stamper, Madison K. Butler, Alex R. Flores, Steven P. Angus, Michael P. East, T. Cloughesy, David A. Nathanson, Michael E. Berens, J. Sarkaria, Zev A. Binder, Donald M. O’Rourke, Timothy C. Howton, Brittany N. Lasseigne, Christopher D. Willey, Gary L. Johnson, Anita Hjelmeland, Frank B Furnari, C. Miller
          </td>
          <td>2025-06-28</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c95f957c61adab36c7dec0eefa4bae427dbcc6b3" target='_blank'>
              CRISPR-Edited Cell Lines: A New Era in Functional Oncology Research.
              </a>
            </td>
          <td>
            Amita Joshi Rana, M.D.S.A.D.I.Q.U.E. Hussain, Ali Hanbashi, Faroq Kamli, Gyas Khan, Marwa Qadri, Saeed A Al-Qahtani, Mohammad Gayoor Khan, Sushil S. Burle, Vikas Jakhmola, Gaurav Gupta
          </td>
          <td>2025-08-13</td>
          <td>Current pharmaceutical design</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Understanding the role of cancer hotspot mutations is essential for unraveling mechanisms of tumorigenesis and identifying therapeutic vulnerabilities. Correcting cancer mutations with base editing is a novel, yet promising approach for investigating the biology of driver mutations. Results Here, we present a versatile platform to investigate the functional impact of cancer hotspot mutations through adenine base editing in combination with transcriptomic profiling. Using this approach, we correct TP53 hotspot mutations in cancer cell lines derived from diverse tissues, followed by mRNA sequencing to evaluate transcriptional changes. Remarkably, correcting these mutations not only reveals the dependency on mutant allele expression but also restores highly conserved tumor-suppressive transcriptional programs, irrespective of tissue origin or co-occurring mutations, highlighting a shared p53-dependent regulatory network. Our findings demonstrate the utility of this base editing platform to systematically interrogate the functional consequences of cancer-associated mutations and their downstream effects on gene expression. Conclusions This work establishes a robust framework for studying the transcriptional dynamics of cancer hotspot mutations and sheds light on the conserved biological processes reinstated by p53 correction, offering potential avenues for future targeted therapies. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03667-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0d441d58b01f4910f8c55a860db9d7cbefe24b5" target='_blank'>
              A base editing platform for the correction of cancer driver mutations unmasks conserved p53 transcription programs
              </a>
            </td>
          <td>
            Pascal Wang, Rituparno Sen, Frank Buchholz, Shady Sayed
          </td>
          <td>2025-07-22</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Cancer therapy resistance remains a major challenge, with limited resources available for systematically studying its underlying mechanisms at the patient level. The existing databases are either restricted to bulk RNA-seq data, lack single-cell resolution, or provide limited clinical annotations, making them insufficient for in-depth exploration of the tumor microenvironment (TME) dynamics in therapy resistance. To bridge this gap, we present CellResDB, a patient-derived platform comprising nearly 4.7 million cells from 1391 patient samples across 24 cancer types. CellResDB provides comprehensive annotations of TME features linked to therapy resistance. To enhance accessibility, we include an intelligent robot, CellResDB-Robot, which facilitates intuitive data retrieval and analysis. In summary, CellResDB represents a valuable resource for cancer therapy and provides an experimental protocol for applying large language models (LLMs) within the biomedical database. CellResDB is freely available at https://cellknowledge.com.cn/cellresponse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e259dc3f38a852bd1ee92860ed4d64506f3ee24b" target='_blank'>
              Deciphering cancer therapy resistance via patient-level single-cell transcriptomics with CellResDB
              </a>
            </td>
          <td>
            Tianyuan Liu, Huiyuan Qiao, Liping Ren, Xiucai Ye, Quan Zou, Yang Zhang
          </td>
          <td>2025-07-15</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Prostate cancer is one of the most common malignant tumors occurring in the male reproductive system. Androgen deprivation therapy (ADT) is the primary treatment for prostate cancer. However, approximately 10–20% of advanced prostate cancer patients progress to castration-resistant prostate cancer (CRPC) within five years, posing a significant challenge to treatment. There is an urgent need to explore the causes and mechanisms of prostate cancer progression, and to identify new therapeutic targets and predictive markers. Utilize RNA-seq to screen for differentially expressed genes and alternative splicing events between androgen-sensitive prostate cancer cells and castration-resistant prostate cancer cells. Perform bioinformatics analysis on the differentially expressed genes. The expression levels of these genes were confirmed by real-time quantitative PCR (RT-qPCR); alternative splicing events related to castration resistance were verified through RT-PCR. The impact of the S100A6 gene on the proliferation and growth capabilities of PC3 and DU145 cells was analyzed using CCK8, colony formation, and EdU assays. Cell migration was assessed using a wound healing assay. The RNA-seq analysis results showed that there were 4,830 differentially expressed genes in non-hormone-sensitive prostate cancer. GO enrichment analysis of these differentially expressed genes revealed significant enrichment in tumor-related signaling pathways such as regulation of cell cycle process, cell division, and regulation of cell projection organization. Further RT-qPCR detection results indicated that the mRNA expression of four genes, including S100A6, was upregulated, consistent with the transcriptome sequencing results. Downregulation of S100A6 expression significantly inhibited the proliferation and migration of non-hormone-dependent prostate cancer. In this study, we also identified many splicing alterations in cancer-related genes, with castration resistance-induced alternative splicing events closely related to the interaction networks of regulation of cell cycle process, cell division, and regulation of cell projection organization. Our research findings suggest that S100A6 may serve as a molecular marker or therapeutic target associated with castration resistance in prostate cancer, providing new insights into the molecular basis of this disease. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-11397-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8350b9dbff378bc43ca0f122bb3ffa5b66be6348" target='_blank'>
              Screening and validation of key genes involved in castration-resistant prostate cancer based on transcriptomics sequencing
              </a>
            </td>
          <td>
            Guangquan Tong, Xiaomeng Jing, Yuning Yang, Xiaowei Wang, Junlin Lu, Jingyi Hu, Yang Wang, F. A. Jumuddin, Wenjing Zhang, Yuesheng Lv
          </td>
          <td>2025-07-15</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cancer remains one of the leading health threats globally, with therapeutic resistance being a long-standing challenge across chemotherapy, radiotherapy, targeted therapy, and immunotherapy. In recent years, the association between epigenetic modification abnormalities and therapeutic resistance in tumors has garnered widespread attention, spurring interest in the development of approaches to target epigenetic factors. In this review, we explore the widespread dysregulation and crosstalk of various types of epigenetic modifications, including DNA methylation, histone modifications, and non-coding RNA changes, which interact through complex regulatory networks in tumors. Clinically, single-targeted therapy based on epigenetic modification usually has its limited effect against cancer. However, the combination of epigenetic drugs with other treatment modalities, such as chemotherapy, targeted therapy, or immunotherapy, shows potential for synergistically enhancing efficacy and reducing drug resistance. Therefore, we evaluate the possibility and potential mechanisms of targeting epigenetic modifications to overcome resistance in cancer therapy, and discuss the challenges and opportunities in moving epigenetic therapy into clinical practice. Moreover, the application of multi-omics technologies will aid in identifying core epigenetic factors from complex epigenetic networks, enabling precision treatment and overcoming therapeutic resistance in tumors. Furthermore, the development of spatial multi-omics technologies, by providing spatial coordinates of cellular and molecular heterogeneity, revolutionizes our understanding of the tumor microenvironment, offering new perspectives for precision therapy. In summary, the combined application of epigenetic therapies and the integration of multi-omics technologies herald a new direction for cancer treatment, holding the potential to achieve more effective personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8def3b2fb87b962d999f696cd613916b46b863b4" target='_blank'>
              Targeting epigenetic regulators as a promising avenue to overcome cancer therapy resistance
              </a>
            </td>
          <td>
            Jiawei Song, Ping Yang, Canting Chen, Weiqun Ding, O. Tillement, Hao Bai, Shuyu Zhang
          </td>
          <td>2025-07-18</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Simple Summary Cancer treatments that use drugs resembling DNA building blocks, known as nucleotide analogs, that insert themselves into DNA and prevent replication often stop working over time because cancer cells become resistant. This review explains how reversible epigenetic changes help cancer cells avoid being killed by these drugs. These changes do not alter the DNA sequence but can turn important genes on or off. We describe how cancer cells use these mechanisms to repair damaged DNA, pump drugs out of the cell, or break them down. We highlight the role of DNA packaging proteins (histones), small RNA molecules, and chemical tags on DNA that control how genes behave. Understanding these changes is important because it may allow clinicians to resensitize tumors and make them sensitive to treatment again and prevent relapse. We also explore new therapies, including drugs and gene editing tools, that target these epigenetic changes to improve the success of chemotherapy. These findings could lead to better outcomes for patients with difficult-to-treat cancers, especially those with pancreatic, colorectal, or breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbeef0502ae8a946f9634749621e6cdd28f4f464" target='_blank'>
              Epigenetic Drivers of Chemoresistance in Nucleobase and Nucleoside Analog Therapies
              </a>
            </td>
          <td>
            John Kaszycki, Minji Kim
          </td>
          <td>2025-07-01</td>
          <td>Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Targeted therapy has markedly improved outcomes for patients with non-small cell lung cancer (NSCLC). However, the emergence of drug resistance remains a major clinical challenge, limiting long-term treatment efficacy. Clustered regularly interspaced short palindromic repeats (CRISPR)/Cas9, a revolutionary gene-editing technology, offers precise and efficient genetic modifications, providing new insights into the mechanisms of drug resistance in NSCLC. The present review explored the application of CRISPR/Cas9 in overcoming resistance associated with key oncogenic drivers, including EGFR, KRAS, ALK, ROS1, MET and BRAF. It summarized recent advances in CRISPR-based strategies to reverse resistance, enhance targeted therapy effectiveness and develop potential therapeutic interventions. Additionally, it discussed current limitations, including off-target effects, delivery challenges and ethical concerns, while highlighting future directions for clinical translation. Using CRISPR/Cas9 technology may pave the way for novel, personalized treatment approaches in NSCLC, ultimately improving patient outcome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ddd19a50ff46be19254c0572146995de6cd1223" target='_blank'>
              Harnessing CRISPR/Cas9 to overcome targeted therapy resistance in non-small cell lung cancer: Advances and challenges (Review)
              </a>
            </td>
          <td>
            Jianting Du, Xian Gong, Renjie Huang, Bin Zheng, Chun Chen, Zhang Yang
          </td>
          <td>2025-07-09</td>
          <td>Oncology Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Drug resistance continues to be the principal limiting factor in achieving a cure for patients with cancer, significantly hindering the long-term efficacy of novel cancer drugs. Accumulating evidence has shown that metabolites derived from tumor cells regulate immune cell metabolism via tumor microenvironment crosstalk. However, as immunometabolic research has deepened, the leading role played by the intrinsic metabolic regulation of immune cells in the drug resistance of tumor cells has been discovered. Immune metabolites have been shown to cause immune resistance, target therapy resistance, and chemotherapy resistance, and drugs that target immune metabolism have great potential. To date, researchers have not fully explored the impact of immune-derived metabolites on tumor cells and their influence on the responsiveness to cancer drugs. In this review, we focus on the lactate, fatty acid, glucose, and nucleotide metabolic alterations that take place in T cells and macrophages and how these changes can impair anti-tumor immunity, ultimately promoting tumor cell survival and decreasing responsiveness to the corresponding therapeutic approaches. We present the current developments in drugs targeting immunometabolic pathways and propose constructive suggestions, such as precise delivery to immune cell targets to enhance efficacy and safety, offering novel perspectives for cancer drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/577d110e8e2913a10ea918be3d7d4c6669ac4c37" target='_blank'>
              Immune cell metabolism in cancer drug resistance: Advances in target discovery and clinical translation.
              </a>
            </td>
          <td>
            Liangjie Sun, GuoWei Zhao, Shuhang Wang, Ning Li
          </td>
          <td>2025-06-30</td>
          <td>Chinese journal of cancer research = Chung-kuo yen cheng yen chiu</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cancer progression and development are influenced by a complex interplay between inherited (germline) and acquired (somatic) mutations. Current cancer genomic research in Middle Eastern populations predominantly utilizes targeted panels to examine predefined genes, potentially overlooking a broader spectrum of genomic contributions to cancer predisposition. Addressing this gap, this study adopts an unbiased approach using whole exome sequencing (WES) data to identify both high- and low-penetrance genetic variants within the United Arab Emirates (UAE) population. This investigation features a case-control matching analysis comprising 62 patients diagnosed with various cancer types and 142 unrelated healthy controls. The results showed a potential association between cancer predisposition and variants within. The results demonstrate an association between cancer predisposition and variants within C-terminal Binding Protein 2 (CTBP2), DNA Polymerase Theta (POLQ) and Tektin 4 (TEKT4). Gene set enrichment analysis showed enriched pathways associated with cancer-related biological processes such as DNA repair and depleted pathways related to translation, cellular metabolic process, and mitochondrial functions. This study highlights that the distinctive genetic composition of underrepresented groups influences the penetrance of pathogenic variants that could contribute to hereditary cancer risk in ways that diverge from patterns observed in more extensively researched cohorts. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-05296-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/972fb28f89280ac4ce146e45e2b00c9a74cb002a" target='_blank'>
              Pan-Cancer Exome-wide analysis of germline mutational patterns and pathways
              </a>
            </td>
          <td>
            Halima Alnaqbi, Michael Olbrich, Noora Zayed, Mira Mousa, S. Azzam, S. Tirmazy, Habiba S. Alsafar
          </td>
          <td>2025-07-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65c33305648009e1896ce514a1bce98da6447d96" target='_blank'>
              Epigenetic dysregulation in cancer: mechanisms, diagnostic biomarkers and therapeutic strategies.
              </a>
            </td>
          <td>
            Komal Imran, Muhammad Javed Iqbal, Muhammad Mushtaq Ahmed, Ammara Khalid, H. Cortés, O. Reyes-Hernández, Gabriela Figueroa-González, G. Leyva-Gómez, L. Falzone, Massimo Libra, Federica Longo, Javad Sharifi‐Rad, D. Calina
          </td>
          <td>2025-07-21</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Retroviral transduction of cancer cell lines has been used to find genes related to therapy resistance. Characterization of virus integration sites (VIS) pinpointing these genes has been cumbersome. This study defines a sequencing-based protocol to rapidly characterize genomic loci near VIS. The protocol selectively amplifies VIS-genome junctions using the retroviral vector neomycin (NEO) gene, Genomic Walker Adapter approach, linker-mediated NEO-PCR (LM-NEO-PCR) or biotinylated NEO-capture, followed by long terminal repeats PCR (LTR-PCR). LTR-genome junctions were sequenced (NGS), reads mapped, quantified, and linked to genes. The protocol was tested on DNA from single clones holding twenty reported VIS loci and on multiplex and diluted clone DNA samples. Our VIS-NGS protocol enriched significantly (p < 0.02) more loci at high reads coverage in samples with VIS compared to negative controls. The protocol found seventeen reported VIS loci (85%) in single clone DNAs, of which fifteen loci (88%) were also detected in multiplex samples. Six loci were evaluated for all dilutions, with three loci detected at lowest 1% clone proportion. The protocol can be conducted in two weeks and successfully found almost all VIS loci in single VIS clones and detected half of the evaluated loci at low clone proportion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc457805a586e034f6d53bc20d3aa71d5bdfc7ad" target='_blank'>
              Novel protocol for mapping virus integration sites in genes involved in therapy resistance
              </a>
            </td>
          <td>
            M. Massimino, Zhangzan Huang, J. Helmijr, Camilla Muritti, Anna Uijtdewillegen, P. Vigneri, John W M Martens, Maurice P H M Jansen
          </td>
          <td>2025-07-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eec7c3cc61289a3954a3ddd8b996ed6f9c80850e" target='_blank'>
              The adaptive molecular landscape of reprogrammed telomeric sequences
              </a>
            </td>
          <td>
            Melania D’Angiolo, Benjamin P. Barré, Sakshi Khaiwal, J. Muenzner, Johan Hallin, Matteo De Chiara, Nicolò Tellini, J. Warringer, M. Ralser, Eric Gilson, Gianni Liti
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Background Histone modifications are key epigenetic regulators of cell differentiation and have been intensively studied in many cell types and tissues. Nevertheless, we still lack a thorough understanding of how combinations of histone marks at the same genomic location, so-called chromatin states, are linked to gene expression, and how these states change in the process of differentiation. To receive insight into the epigenetic changes accompanying the differentiation along the chondrogenic lineage we analyzed two publicly available datasets representing (1) the early differentiation stages from embryonic stem cells into chondrogenic cells and (2) the direct differentiation of mature chondrocyte subtypes. Results We used ChromHMM to define chromatin states of 6 activating and repressive histone marks for each dataset and tracked the transitions between states that are associated with the progression of differentiation. As differentiation-associated state transitions are likely limited to a reduced set of genes, one challenge of such global analyses is the identification of these rare transitions within the large-scale data. To overcome this problem, we have developed a relativistic approach that quantitatively relates transitions of chromatin states on defined groups of tissue-specific genes to the background. In the early lineage, we found an increased transition rate into activating chromatin states on mesenchymal and chondrogenic genes while mature chondrocytes are mainly enriched in transition between activating states. Interestingly, we also detected a complex extension of the classical bivalent state (H3K4me3/H3K27me3) consisting of several activating promoter marks besides the repressive mark H3K27me3. Within the early lineage, mesenchymal and chondrogenic genes undergo transitions from this state into active promoter states, indicating that the initiation of gene expression utilizes this complex combination of activating and repressive marks. In contrast, at mature differentiation stages the inverse transition, the gain of H3K27me3 on active promoters, seems to be a critical parameter linked to the initiation of gene repression in the course of differentiation. Conclusions Our results emphasize the importance of a relative analysis of complex epigenetic data to identify chromatin state transitions associated with cell lineage progression. They further underline the importance of serial analysis of such transitions to uncover the diverse regulatory potential of distinct histone modifications like H3K27me3. Supplementary Information The online version contains supplementary material available at 10.1186/s13072-025-00594-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86740c087751af9f15d6ec845b2ae74c06df1ebd" target='_blank'>
              Bath: a Bayesian approach to analyze epigenetic transitions reveals a dual role of H3K27me3 in chondrogenesis
              </a>
            </td>
          <td>
            Christoph Neu, M. Wuelling, Christoph Waterkamp, Daniel Hoffmann, Andrea Vortkamp
          </td>
          <td>2025-06-27</td>
          <td>Epigenetics & Chromatin</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Castration-resistant prostate cancer is a heterogeneous disease with variable phenotypes commonly observed in later stages of the disease. These include cases that retain expression of luminal markers and those that lose hormone dependence and acquire neuroendocrine features. While there are distinct transcriptomic and epigenomic differences between castration-resistant adenocarcinoma and neuroendocrine prostate cancer, the extent of overlap and degree of diversity across tumor metastases in individual patients has not been fully characterized. Here we perform combined DNA methylation, RNA-sequencing, H3K27ac, and H3K27me3 profiling across metastatic lesions from patients with CRPC/NEPC. Integrative analyses identify DNA methylation-driven gene links based on location (H3K27ac, H3K27me3, promoters, gene bodies) pointing to mechanisms underlying dysregulation of genes involved in tumor lineage (ASCL1, AR) and therapeutic targets (PSMA, DLL3, STEAP1, B7-H3). Overall, these data highlight how integration of DNA methylation with RNA-sequencing and histone marks can inform intraindividual epigenetic heterogeneity and identify putative mechanisms driving transcriptional reprogramming in castration-resistant prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdf4a71accfefcd4f60da4f3b6e81d8837cae193" target='_blank'>
              Intraindividual epigenetic heterogeneity underlying phenotypic subtypes of advanced prostate cancer
              </a>
            </td>
          <td>
            Kei Mizuno, Sheng-Yu Ku, V. B. Venkadakrishnan, Martin K. Bakht, M. Sigouros, Joanna Chan, A. Trigos, Jordan H. Driskill, J. Manohar, Abigail King, Adam G. Presser, Min Jin Kim, Alok K. Tewari, H. Long, David Quigley, T. Choueiri, Steven P. Balk, Sarah J. Hill, J. Mosquera, David J Einstein, Shahneen Sandhu, M. Taplin, H. Beltran
          </td>
          <td>2025-07-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="


 Tumor tissues exhibit cell-specific genetic variations and gene expression profiles critical to disease diagnosis, progression, and treatment outcomes. Single cell sequencing offers detailed insights into the heterogeneity of cancer cells which is pivotal for developing personalized treatment strategies. However, current high-throughput methods primarily detect gene expression levels, lacking information on genetic variants such as mutations or gene fusions. Understanding tumor responses to therapy requires linking cell-type specific mutations with gene expression profiles to understand cancer development and progression.



 We developed FocuSCOPE, a high-throughput single cell sequencing solution that simultaneously measures gene expression and detects genetic variantion from individual cells. Single cell partitioning is performed using SCOPE-chip®, a portable microwell chip the size of a microscopy slide. The technology utilizes barcoding microbeads conjugated with oligos to efficiently capture mRNA and target sequences. The optimized chemistry facilitates amplification and library construction of both mRNA and target regions, including point mutations, fusion genes, and non-polyadenylated RNAs such as viral RNAs.



 NB4, CCRF, and K562 cell lines were mixed in equal proportions. Libraries were prepared using the FocuSCOPE Blood Cancer Panel. Experiments demonstrated the specific detection of KRAS and TP53 mutations in mixed cell lines (NB4, CCRF, and K562) using the FocuSCOPE Blood Cancer Panel. The NB4 cell line contained KRAS (A18D) and TP53 (R248Q) mutations, CCRF contained KRAS (G12D) and TP53 (R248Q, R175H), and K562 contained only the BCR-ABL1 fusion gene. Additionally, FocuSCOPE effectively captured PML-RARA and BCR-ABL1 fusion genes with high sensitivity.



 FocuSCOPE enables comprehensive analysis of gene expression profiles and targeted genetic sequences, capturing driver mutations, fusion genes, and non-polyA sequences such as viral sequences with high sensitivity, thus advancing our understanding of tumor biology and therapeutic responses. This makes it an invaluable tool for personalized medicine, guiding therapeutic decisions and improving patient outcomes.



 Ankhita Nair, Rachael Lee, Sujatha Narayanankutty, Yingting Wang, Jonathan Scolnick. FocuSCOPE: A Multi-omics Solution to Simultaneously Analyse Genetic Variants and Transcriptome in Single Cells in Leukemia [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P17.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fafbd9d94e3c2b4f9a27806a0df8f1e19c248cc6" target='_blank'>
              Abstract P17: FocuSCOPE: A Multi-omics Solution to Simultaneously Analyse Genetic Variants and Transcriptome in Single Cells in Leukemia
              </a>
            </td>
          <td>
            Ankhita Nair, Rachael Lee, Sujatha Narayanankutty, Yingting Wang, Jonathan Scolnick
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bcaffd86557cddf1faae3fdc7b2f95bd830f2b9" target='_blank'>
              Pan-cancer analysis in the real-world setting uncovers immunogenomic drivers of acquired resistance post-immunotherapy
              </a>
            </td>
          <td>
            Mohamed Reda Keddar, Sebastian Carrasco Pro, Roy Rabbie, Z. Atak, A. Stewart, S. Hammond, Douglas C. Palmer, Ross Stewart, Kathleen Burke, Benjamin Sidders, Jessica Davies, Jonathan R Dry, I. Martincorena, Sajan Khosla, A. Schoenfeld, Martin L. Miller
          </td>
          <td>2025-07-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="Background Osteosarcoma (OS) shows a multitude of genetic and chromosomal abnormalities together with large biological heterogeneity. These features limited the identification of novel drugs to treat patients with metastases and/or chemo-resistant tumors. The purpose of this study was to create additional resources for drug screening by generating patient-derived xenograft (PDXs) and PDX-derived cell lines that reflect the spectrum of OS heterogeneity. Methods PDXs were derived from OS collected at diagnosis, surgical resections, or metastases. PDX-derived cell lines were also established. Targeted DNA sequencing and digital PCR were applied to identify major genetic alterations. High-throughput drug screening by using a library of 2880-FDA approved compounds and conventional MTT assays were performed to identify the most effective drugs against in vitro and in vivo growth of chemo-resistant OS. Results Targeted DNA sequencing demonstrated alterations in the most commonly amplified oncogenes, such as MYC, CCNE1, DDR2, CDK4, MDM2, and AURKA. Recurrent deletions and SNVs were found in TP53, CDKN2A, RB1, PTEN, and VHL. Copy number variant (CNV) alterations in PDXs, PDX-derived cell lines and xenografts developed from cell lines (CDX) correlated very well with those observed in the matched original human tumors. Drug screenings identified and repurposed five compounds with efficacy against chemoresistant OS. In this context, we prioritized ixabepilone as a drug capable of inducing tumor regression in mice. Conclusions We enriched the scientific community with additional, molecularly characterized OS models to be used for testing novel therapies and supported the inclusion of ixabepilone into treatment plans for chemoresistant OS. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03440-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23016fdd463bf1bcdb3a8bfe081d0f07e246c765" target='_blank'>
              Genomic profiling of a collection of patient-derived xenografts and cell lines identified ixabepilone as an active drug against chemo-resistant osteosarcoma
              </a>
            </td>
          <td>
            M. Manara, F. Bruzzese, Laura Formentini, L. Landuzzi, L. Pazzaglia, M. Laginestra, Marianna Carrabotta, M. Roca, Federica Iannelli, Laura Grumetti, L. Addi, A. Parra, C. Cristalli, M. Pasello, Alberto Bavelloni, Francesca Carreras, F. Ruzzi, G. Bianchi, M. Gambarotti, A. Righi, Alfredo Budillon, Pier-Luigi Lollini, Katia Scotlandi
          </td>
          <td>2025-07-08</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) encompasses tumors arising in the colon (CC) and rectum (RC), often treated as a single disease despite emerging evidence of biological divergence. Understanding the molecular differences between CC and RC is critical for improving diagnosis, prognosis, and therapeutic strategies. Methods We performed an integrated genomic and transcriptomic analysis of CC and RC data from The Cancer Genome Atlas (TCGA) to investigate their degree of similarity and observed that these tumors present distinct molecular profiles, which suggest an evolution through divergent pathways. Comparative analyses included copy number alterations (CNAs), somatic mutations, driver gene prediction, differential gene expression, pathway enrichment, and survival analysis. Results Chromosomal analyses revealed that 43% of focal and 77% of large-scale CNAs were specific of CC, while 10.5% and 57% were specific of RC with 8% of mutant genes unique to CC and 0.18% to RC. CC and RC presented distinct profiles of gene mutations, with CC showing significantly higher tumor mutational burden (0.51 muts/Mb vs 0.28 muts/Mb in RC). Distinct mutational signatures were identified, with CC characterized by a higher frequency of PIK3CA, BRAF, and DNAH1 mutations, while RC showed enrichment for TP53 and NRAS mutations. Importantly, analysis of predicted non-canonical driver genes identified ACVR1B, LTBP4, SETD1A as CC-specific drivers and C4BPA, EHD1 as RC-specific drivers, underscoring divergent oncogenic mechanisms. However, the most substantial divergence was observed in transcriptomic profiling, with 56% and 33% of DEGs (in CC and RC, respectively) that were tumor-type specific. Notably, RC tumors segregated into two distinct transcriptional subtypes (Cluster 1 and Cluster 2), with Cluster 1 showed a more heterogeneous Consensus Molecular Subtypes (CMS) distribution, while Cluster 2 enriched in CMS4 (mesenchymal) and CMS3 (metabolic) consensus molecular subtypes. Accordingly, Gene Set Enrichment Analysis revealed CC-specific upregulation of Wnt, MYC, and mTOR signaling pathways, and RC-specific enrichment of GPCR and neuronal development pathways. On the other hand, pseudogene expression was significantly higher in CC, suggesting differential mechanisms of transcriptional dysregulation. Finally, we identified an RC-specific multigene survival signature as a prognostic model involving upregulation of C2CD4B, HSPD1P1, LINC01356, CBX3P9, GATA2-AS1 and downregulation of ATP5F1EP2, HSP90AB3P and SNRPFP1. Conclusions Collectively, our findings provide robust molecular evidence that CC and RC follow divergent oncogenic pathways, emphasizing the need for site-specific biomarker development and therapeutic targeting in colorectal cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06908-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eab7228806c5ae698ac668178d1fa7e354b1b537" target='_blank'>
              Distinctive chromosomal, mutational and transcriptional profiling in colon versus rectal cancers
              </a>
            </td>
          <td>
            Maria Teresa De Angelis, Antonia Rizzuto, Angela Amaddeo, Carlo Sagnelli, Niccolò Vono, Michela Reda, Valentina Lise, Luca Parrillo, Carmela de Marco, Donatella Malanga, Gianluca Santamaria, Giuseppe Viglietto
          </td>
          <td>2025-08-06</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b2aba01b3716bf8fef6d8f3f3f6e3783dfc01cf" target='_blank'>
              Recent advances in novel targeting mechanisms for colorectal cancer.
              </a>
            </td>
          <td>
            Zhiping Li, Hailing Wang, Yan Liu, Ahequeli Gemingnuer, Yinan Wang, Xin Meng
          </td>
          <td>2025-07-11</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 The intricate relationship between inflammation and cancer progression has been well-documented, with chronic inflammation playing a pivotal role in tumor development and drug resistance. This project investigates how cancer-intrinsic inflammation influences drug response, aiming to identify key regulators that could serve as therapeutic targets to overcome drug resistance in colorectal cancer (CRC). We hypothesize that distinct inflammation types within cancer cells modulate their response to chemotherapy, potentially driving resistance. Our study is structured around two specific aims: First, we investigate the inflammation-type-dependent drug response using single-cell barcoded colon cancer cells. Our preliminary studies demonstrated that CRC cells in an inflamed microenvironment, particularly those with STING pathway activation, exhibit distinct responses to 5-FU treatment compared to non-inflamed cells. This finding underscores the need to consider inflammation as a critical factor in cancer treatment strategies. By characterizing the response of inflamed and non-inflamed cells to various drug treatments at single-cell resolution, we aim to identify the specific inflammation-driven mechanisms that contribute to drug resistance. Second, we focus on the characterization and validation of key regulators identified in inflamed CRC cells. We will validate candidate genes through functional assays, including gene knockdown experiments, to uncover the molecular pathways through which inflammation modulates drug response. By linking the distinct inflammatory states within cancer cells to drug resistance, we aim to identify novel biomarkers and therapeutic targets that could enhance the effectiveness of existing treatments. Our research has the potential to significantly impact the treatment of CRC by providing a deeper understanding of how inflammation drives drug resistance, ultimately guiding the development of more effective, personalized therapies for patients with inflamed tumors. This approach addresses the urgent need for targeted strategies to overcome the challenges posed by inflammation-associated drug resistance in cancer therapy.



 Kerem Fidan, Semih Can Akincilar, Tim Stuart, Vinay Tergaonkar. Deciphering Inflammation-Driven Drug Resistance Mechanisms in Colorectal Cancer through Single-Cell Barcoding [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P71.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb480f230d054f0fd849886c3b02be9cc7e768af" target='_blank'>
              Abstract P71: Deciphering Inflammation-Driven Drug Resistance Mechanisms in Colorectal Cancer through Single-Cell Barcoding
              </a>
            </td>
          <td>
            Kerem Fidan, S. C. Akıncılar, Tim Stuart, Vinay Tergaonkar
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6adfd64895599aa6bae6a1f8896e93b0d56bc5ac" target='_blank'>
              ADAR1-mediated RNA editing in breast cancer: molecular mechanisms and therapeutic implications.
              </a>
            </td>
          <td>
            Si-Yu Chen, Si-Ying Chen, Shu Yang, Yi Li, Shengyong Yang
          </td>
          <td>2025-08-09</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Breast cancer (BC) is one of the most common types of malignancy diagnosed globally. Metastasis plays a major role in most of the cancer-related mortality among affected patients. Despite the advances in the areas of early detection and localized treatment modalities, there prevail several challenges which the therapeutic strategies encounter, like drug resistance, tumor heterogeneity, and drug delivery. This review presents a comprehensive and detailed overview of organ-specific metastasis that occur in BC, specifically emphasizing key sites such as the bone, liver, lung, and brain. It also outlines the significance of various therapies like chemotherapies, endocrine therapies, targeted therapies and immunotherapies that have been clinically approved to date. The review specifically emphasizes the molecular mechanisms by which non-coding RNAs (ncRNAs) act to exert their effects in regulating drug resistance. It also addresses the new advances in nanotechnology-based drug delivery systems (DDS) that function to enhance the specificity of treatments while simultaneously reducing systemic toxicity. Beyond ncRNAs, this review also explores other critical mechanisms of drug resistance in metastatic BC, including efflux transporter activity, target gene mutations, and micro-environmental factors, to mention a few. Moreover, the review also discusses the clinical significance of combination therapies and new therapeutic strategies, including the use of repurposed drugs and the concepts of personalized medicine. A greater understanding of the ncRNA-mediated signaling pathways, in combination with the latest advances in drug delivery systems, has the potential to greatly improve therapeutic efficacy and could result in more favorable clinical outcomes in the treatment of metastatic BC (MBC).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a2fe23da14c71b938c6019fd6995e4bfcec570b" target='_blank'>
              Bridging molecular insights and clinical application: non-coding RNAs, targeted drug delivery, and metastatic breast cancer therapy
              </a>
            </td>
          <td>
            Sohini Chakraborty, Satarupa Banerjee
          </td>
          <td>2025-07-26</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Docetaxel is a chemotherapeutic agent widely used for breast cancer treatment; however, its efficacy is often limited by drug resistance and associated toxicity. This review examines the molecular mechanisms of docetaxel resistance in breast cancer and discusses research advances and future directions for overcoming this challenge. Key resistance mechanisms include alterations in drug targets (microtubules), increased drug efflux, suppression of apoptosis, activation of survival signalling pathways, epithelial-to-mesenchymal transition (EMT), and cancer stem cell enrichment. An evolutionary perspective distinguishes between intrinsic and acquired resistance, emphasising the need for adaptive therapeutic strategies. Recent advances in genomic profiling, non-coding RNA research, novel drug combinations, and biomarker-guided therapies have also been reviewed. Emerging approaches, such as targeting the tumour microenvironment, harnessing immunotherapy, and implementing adaptive dosing schedules, have been discussed. This review emphasises the understanding of resistance as a multifactorial phenomenon that requires multipronged interventions. Research has aimed to identify predictive biomarkers, develop targeted agents to reverse resistance, and design rational combination strategies to improve patient outcomes. Progress in deciphering and targeting docetaxel resistance mechanisms holds promise for enhancing treatment responses and extending survival in patients with breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb368d7af0b16140c6442111947c4c43ba088b35" target='_blank'>
              Docetaxel Resistance in Breast Cancer: Current Insights and Future Directions
              </a>
            </td>
          <td>
            F. Postigo-Corrales, Asunción Beltrán-Videla, A. D. Lázaro-Sánchez, Ana María Hurtado, P. Conesa-Zamora, Ana Belén Arroyo, G. Luengo-Gil
          </td>
          <td>2025-07-23</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/481b8053e41e58296473fbe96b5f5b8ab91cf715" target='_blank'>
              Reversible therapeutic resistance in EGFR mutant lung cancer caused by RB1 loss induced lineage plasticity.
              </a>
            </td>
          <td>
            Letian Zhang, M. Chinnam, Bradley Balk, Xiaojing Zhang, Yanqing Wang, Tatiana Shaurova, Shamshad Alam, Saraswati Pokharel, Prashant Singh, Justine Jacobi, Jianming Wang, Eduardo Cortes Gomez, Michael Habitzruther, Mukund Seshadri, B. Atanassov, Wei Wu, E. Benevolenskaya, C. Blakely, Pamela Hershberger, Mark Long, T. Bivona, David W. Goodrich
          </td>
          <td>2025-08-06</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97e0ce2d3aba4ba6a29f1b308846500c4a3d7ec7" target='_blank'>
              Beyond inversions and deletions: the evolutionary and functional insights from translocations, fissions, and fusions in animal genomes.
              </a>
            </td>
          <td>
            Célian Diblasi, M. Saitou
          </td>
          <td>2025-08-01</td>
          <td>Heredity</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="ABSTRACT Background Cisplatin and its derivatives remain a cornerstone in the treatment of solid malignancies. Resistance is a major factor limiting their clinical utility. Objectives In the present study, we set out to interrogate therapeutic approaches to target cisplatin‐resistant cancer cells. We focused on therapies exploiting metabolic pathways that are altered in drug‐resistant cells. We sought to find an existing therapy that has monotherapy efficacy against cisplatin‐resistant cancer cells that can also re‐sensitize to cisplatin. Methods We used lung and ovarian cancer cell lines with acquired resistance to cisplatin together with drug sensitivity assays, conducted both with monotherapies and cisplatin combinations. Results We show that cancer cell lines with acquired resistance to cisplatin have altered levels of enzymes involved in glutamine metabolism, which can result in differential sensitivity to targeted agents. We show that expression of one of these enzymes—the glutamate‐cystine antiporter SLC7A11, up‐regulated 6‐fold in a cisplatin‐resistant lung cancer cell line—has potential prognostic significance in lung cancer but not ovarian cancer. After identifying a common dependency of cisplatin‐resistant cancer cells upon extracellular glutamine, we then evaluate the utility of the long‐standing anti‐leukemic therapy asparaginase (ASNase)—which possesses both asparaginase and glutaminase activity—as a potential approach. We show ASNase preferentially inhibits the proliferation of cisplatin‐resistant cancer cells and can potentially re‐sensitize these cells to cisplatin. Conclusions Our results underpin the prevalence of altered metabolism in cisplatin‐resistant cells and highlight the potential utility of re‐purposing ASNase to target these cells, warranting further investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f061244858dd7ffcf91e4f9a475213a0f0dca65" target='_blank'>
              Repurposing Asparaginase Therapy to Target Cisplatin‐Resistant Cancer Cells
              </a>
            </td>
          <td>
            Jiantao Wang, Nasim Pouryaghoub, Robert Strauss, Jiri Bartek, S. Zhang, Sean G. Rudd
          </td>
          <td>2025-08-10</td>
          <td>Fundamental & Clinical Pharmacology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Genetic factors play a critical role in the development of lipoprotein disorders, which significantly contribute to atherosclerotic cardiovascular disease (ASCVD). Traditional management of these conditions has relied on lipid-lowering therapies, which require lifelong adherence. Recent advancements in gene addition and editing technologies offer novel and potentially transformative approaches for treating lipoprotein disorders by targeting the relevant genetic pathways for each disease. This review revisits major monogenic and polygenic disorders of lipoprotein metabolism, including familial hypercholesterolemia, elevated lipoprotein(a), and familial chylomicronemia syndrome, and discusses the genetic-based therapies for management. RNA-based, gene addition and gene editing therapies, including Clustered Regularly Interspaced Short Palindromic Repeats, base editing and interventions whereby, are highlighted for their potential to provide durable treatments which overcome the adherence challenge. Integration of machine learning for risk prediction and the use of polygenic risk scores to enhance risk stratification further demonstrate the promise of personalized approaches, and overall potential for gene-based treatments to revolutionize ASCVD prevention and management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e00880b93e608120c1a6f91fe00e3d00aa4b66d6" target='_blank'>
              Gene therapy and genome editing for lipoprotein disorders.
              </a>
            </td>
          <td>
            Chen Gurevitz, Archna Bajaj, Amit V Khera, Ron Do, H. Schunkert, K. Musunuru, Robert S. Rosenson
          </td>
          <td>2025-07-02</td>
          <td>European heart journal</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Jumping translocations (JTs) are rare chromosomal abnormalities that play a crucial role in the pathogenesis of various cancer types. These rearrangements, especially those involving chromosome 1q, are frequently associated with tumor progression, therapeutic resistance, and poor prognosis. One gene of particular interest, human Jumping Translocation Breakpoint (JTB), has been identified at the site of translocation breakpoints and exhibits complex, context-dependent roles in cancer biology. JTB protein functions as a pivotal regulator in mitosis, chromosomal segregation, apoptosis, and cellular metabolism. It is functionally linked with the chromosomal passenger complex (CPC) and is implicated in processes such as epithelial–mesenchymal transition (EMT), immune evasion, and therapy resistance, especially in breast and prostate cancers. Advances in genomic, transcriptomic, and proteomic research have highlighted the significant potential of JTB as a diagnostic biomarker and a target for therapeutic interventions. This review underscores the dual role of JTB as both a tumor suppressor and oncogene, depending on the cellular context, and advocates for its continued investigation at the genomic, transcriptomic, and proteomic levels. Understanding JTB’s multifaceted contributions to tumor biology may pave the way for novel biomarkers and targeted treatments in cancer management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48960a3d042990da8ae30d21a39837a5c48dded4" target='_blank'>
              From Jumping Gene to Cancer: Revisiting the Role of JTB Protein
              </a>
            </td>
          <td>
            Taniya M Jayaweera, Madhuri Jayathirtha, Krishan Weraduwage, Petra Kraus, C. Darie, A. Neagu
          </td>
          <td>2025-07-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80509a0d0dd4086ca6871a4474226159831e914e" target='_blank'>
              From Discovery to Diagnosis: A Perspective for Circulating Tumor Cells in Personalized Oncology.
              </a>
            </td>
          <td>
            C. Alix-Panabières, Klaus Pantel
          </td>
          <td>2025-08-18</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Cancer cells utilize codon-biased translation to fuel tumorigenesis and drug resistance, but underlying mechanisms remain poorly understood. Here, we show ALKBH1 is overexpressed in acute myeloid leukemia (AML) and essential for leukemia stem/initiating cell (LSC/LIC) self-renewal and AML development/maintenance, whereas dispensable for normal hematopoiesis. ALKBH1 enhances mitochondrial assembly/function and oxidative phosphorylation (OXPHOS), crucial for AML survival/proliferation and resistance to venetoclax, a potent BCL2 inhibitor and widely-used first-line targeted therapy for AML in clinic. Mechanistically, ALKBH1 catalyzes 5-formylcytosine (f5C) at tRNA wobble positions, reprograming decoding and facilitating codon-biased translation, a mechanism we term "Epitranslatomic Midas touch", which in turn drives leukemogenesis and drug resistance by promoting synthesis of key oncogenic proteins like WDR43. Targeting ALKBH1, particularly together with venetoclax, exhibited potent anti-leukemia efficacy in preclinical models with favorable safety profiles. Collectively, our findings elucidate ALKBH1's pivotal role in codon-biased translation and tumorigenesis, and propose a novel therapeutic strategy for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a3e11cf9a76aba0c7266c1034a7a9bfa25e9f8a" target='_blank'>
              ALKBH1 drives tumorigenesis and drug resistance via tRNA decoding reprogramming and codon-biased translation.
              </a>
            </td>
          <td>
            Chao Shen, Yuan Che, Keren Zhou, Kitty Wang, Wei Li, Diyuan Xue, Tong Wu, Lu Yang, Meiling Chen, Yue Sheng, Chengwan Zhang, S. Robinson, Huiying Chen, Lillian Sau, Zhenhua Chen, M. Wunderlich, Li Han, Tingting Tang, Y. Qing, Dong Wu, Miao Sun, Keith Leung, Katarzyna Dąbrowska, P. Pirrotte, Yueh-Chiang Hu, Lucy Y Ghoda, Bin Zhang, J. Mulloy, Minjie Wei, Guido Marcucci, Chuan He, Xiaolan Deng, Jianjun Chen
          </td>
          <td>2025-08-04</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Abstract Large chromosomal copy number gains are ubiquitous throughout cancer types. However, which genes drive their selective advantage is not well established, and therefore they are hardly utilized in clinical practice. Our analysis of copy number patterns in pan-cancer datasets suggests that the selective advantage of copy number gains is largely driven by known oncogenes. Analysis of CRISPR screening data identifies a list of 101 genes that are likely to mediate the effect of these gains, which is highly enriched in annotated oncogenes but also contains genes that have not been implicated in cancer so far. Moreover, we show that specific gains are associated with drug sensitivity or resistance, with a strong enrichment of gains of oncogenes with increased sensitivity to inhibitors targeting these specific genes. Finally, we provide examples where gains can function as relevant clinical biomarkers for diagnosis and treatment. Thus, large copy number gains exert their selective advantage through known and novel oncogenes, and their systematic analysis could advance precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecd5fd07460798ed6e6a37e20560052b55679a6c" target='_blank'>
              Low-amplitude copy number gains shape cancer through known and novel oncogenes with associated therapeutic vulnerabilities
              </a>
            </td>
          <td>
            T. F. Eleveld, B. Ylstra, Leendert H J Looijenga
          </td>
          <td>2025-07-19</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The role of G-quadruplexes (G4s) in gene regulation has been widely documented, especially in gene promoters. However, the transcriptional mechanisms involving G4s in other regulatory regions remain largely unexplored. In this study, we integrated the G4-DNA data derived from 22 breast cancer patient-derived tumor xenograft (PDTX) models and MCF7 cell line as potential breast cancer-associated G4s (BC-G4s). Genome-wide analysis showed that BC-G4s are more prevalent in gene promoters and the first introns. The genes accommodating promoter or intronic BC-G4s show significantly higher transcriptional output than their non-G4 counterparts. The biased distribution of BC-G4s in close proximity to the transcription start site (TSS) is associated with an enrichment of transcription factor (TF) interactions. A significant negative correlation was detected between the G4–TF interactions within the first introns and their cognate promoters. These different interactions are complementary rather than redundant. Furthermore, the differentially expressed genes (DEGs) harboring promoter and first intron BC-G4s are significantly enriched in the cell cycle pathway. Notably, promoter BC-G4s of DEGs could be a central hub for TF–TF co-occurrence. Our analysis also revealed that G4-related single nucleotide variants (SNVs) affect the stability of G4 structures and the transcription of disease-related genes. Collectively, our results shed light on how BC-G4s within promoters and first introns regulate gene expression and reinforce the critical role of G4s and G4-related genes in breast cancer-associated processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6951122fcae4ae20d0e429a03632112d1f8ad46" target='_blank'>
              The Relevance of G-Quadruplexes in Gene Promoters and the First Introns Associated with Transcriptional Regulation in Breast Cancer
              </a>
            </td>
          <td>
            Huiling Shu, Ke Xiao, Wenyong Zhu, Rong Zhang, Tiantong Tao, Xiao Sun
          </td>
          <td>2025-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is a genetically heterogeneous malignancy in which single-gene mutations serve as critical drivers of tumor initiation, progression, and therapeutic resistance. Advances in high-throughput genomics and liquid biopsy technologies have highlighted the clinical utility of mutations in genes such as TP53, CTNNB1, and TERT as diagnostic, prognostic, and predictive biomarkers. These mutations disrupt key oncogenic pathways, modulate the tumor immune microenvironment, and contribute to intratumoral heterogeneity, complicating disease management. Mutation-guided precision medicine, including telomerase inhibitors, Wnt/β-catenin pathway modulators, and immune checkpoint blockade, offers promising avenues for individualized treatment in HCC. However, challenges persist in translating these findings into clinical practice due to mutation complexity, resistance mechanisms, and limitations in biomarker standardization. Emerging strategies such as multi-omics integration, artificial intelligence, and gene editing technologies hold potential to overcome these barriers and facilitate the development of personalized therapeutic regimens. This review summarizes the molecular mechanisms, clinical applications, and translational challenges of single-gene mutations in HCC, with the aim of guiding future research and precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55adee4a03b42037fd3b05bf7d89cf7022deda59" target='_blank'>
              Single-Gene Mutations in Hepatocellular Carcinoma: Applications and Challenges in Precision Medicine
              </a>
            </td>
          <td>
            Haiyang Yu, Xiangxiang Wu, Yiting Liu, Congling Xin, Yu Zhou, Xiaoyi Ding
          </td>
          <td>2025-07-10</td>
          <td>International Journal of Medical Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Sporadic colorectal cancer (CRC), the third leading cause of cancer-related death globally, arises through a continuum from normal tissue to adenomas, progressing from low-grade (LGD) to high-grade dysplasia (HGD); yet, the early epigenetic drivers of this transition remain unclear. To investigate these events, we profiled LGD and HGD adenomas using EM-seq, and identified a consensus differential methylation signature (DMS) of 626 regions through two independent bioinformatics pipelines. This signature effectively distinguished LGD from HGD in both tissue and plasma-derived cell-free DNA (cfDNA), highlighting specific methylation patterns. Functional annotation indicated enrichment for regulatory elements associated with transcription factor activity and cell signaling. Applying the DMS to the TCGA CRC dataset revealed three tumor subtypes with increasing hypermethylation and one normal cluster. The most hypermethylated subtype exhibited poor survival, high mutation burden, and disrupted transcriptional networks. While overlapping with classical CpG Island Methylator Phenotype (CIMP) categories, the DMS captured a broader spectrum of methylation alterations. These findings suggest that the DMS captures functionally relevant, antecedent epigenetic alterations in CRC progression, enabling the robust stratification of dysplasia severity and tumor subtypes. This signature holds promise for enhancing preclinical detection and molecular classification, and warrants further evaluation in larger prospective cohorts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c27b0ef917c9876876a5f66fb35ed8c368fc721c" target='_blank'>
              Multiomics Signature Reveals Network Regulatory Mechanisms in a CRC Continuum
              </a>
            </td>
          <td>
            Juan Carlos Higareda-Almaraz, Francesco Mattia Mancuso, P. Canal-Noguer, Kristi Kruusmaa, Arianna Bertossi
          </td>
          <td>2025-07-23</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Simple Summary Although targeted therapies and immune checkpoint inhibitors have improved patient outcomes, most metastatic renal cell carcinoma tumors eventually become resistant to treatment and progress. In this study, we used a syngeneic mouse model of metastatic renal cell carcinoma to investigate how gene expression changes during disease progression and in response to treatment with cabozantinib, an anti-PD-1 antibody, or both. We found that different treatments activated distinct molecular pathways in metastatic tumors. These treatment-specific changes influenced how tumors responded to subsequent therapies. Our findings suggest that the biology of metastatic renal cell carcinoma evolves depending on prior treatment, and that selecting second-line therapies based on these molecular changes may improve outcomes. This research provides insights that could help guide future treatment strategies and support the development of more effective, personalized therapy sequences for patients with metastatic renal cell carcinoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89a2406df3fc5a264b372ca5f6f1b3b41d0f1310" target='_blank'>
              Treatment-Induced Gene Expression Changes in Metastatic Renal Cell Carcinoma: Insights from a Syngeneic Mouse Model
              </a>
            </td>
          <td>
            Kohshi Okabe, Toshiaki Tanaka, Tetsuya Shindo, Yuki Kyoda, Sachiyo Nishida, Kohei Hashimoto, K. Kobayashi, N. Masumori
          </td>
          <td>2025-07-01</td>
          <td>Current Oncology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Multiple myeloma (MM) is a type of hematological cancer that arises from uncontrolled proliferation of plasma cells. In addition to frequent genetic mutations, malignant plasma cells are characterized by alterations to the epigenome. Myeloma cells display a genome-wide loss of DNA methylation and a corresponding increase in ‘active’ chromatin modifications. The dramatic epigenetic remodeling that occurs in cancer genomes is associated with loss of silencing at transposable elements, which can impact genome regulation. Through paired epigenome and transcriptome profiling of patient derived MM samples, we have found that loss of DNA methylation in MM genomes results in the formation of partially methylated domains that are variable across patients. This loss of DNA methylation coincides with the expression of hundreds of transcripts driven by LINE-1 (L1) retrotransposons that are epigenetically silenced in normal cells. MM samples can be stratified based on L1 activity with distinct gene expression signatures. The high L1 samples are characterized by a more proliferative, less differentiated state as well as inhibition of interferon and genome defense pathways. Several L1 promoters generate chimeric transcripts with adjacent oncogenes. We further find that KRAB-zinc finger proteins (KZFPs) that are responsible for the epigenetic silencing of L1s have abnormally low abundance in MM samples with high L1 activity. These results indicate that cell proliferation in MM is associated with a loss of KZFP expression and activation of L1 elements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e8c94d95f4ddee7048f28a27ac43f9a3d704355" target='_blank'>
              Methylation variability and LINE-1 activation in multiple myeloma
              </a>
            </td>
          <td>
            Qianhui Wan, Amy Leung, Mahek Vinod Bhandari, Hiroyuki Kato, Joo Y. Song, Dustin Schones
          </td>
          <td>2025-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a09fe0116b674341564f20d065726804460dc81" target='_blank'>
              Identification of hub genes and potential molecular mechanisms of tumor immune microenvironment-related radiotherapy sensitivity in locally advanced cervical cancer.
              </a>
            </td>
          <td>
            Yichen Zan, Qing Liu, Yue Yin, Chongxin Guo, Jing Xu, Junhan Gao, Guanglu Dong
          </td>
          <td>2025-06-30</td>
          <td>Reproductive sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Aurora kinase A (AURKA) is aberrantly expressed in a large number of tumors and promotes tumor progression by regulating the cell cycle, chromosomal instability, and drug resistance. However, its heterogeneous expression and combination therapy benefit in prostate adenocarcinoma (PRAD) is unclear. Methods In this study, we integrated TCGA pan-cancer multi-omics data and GEO data to analyze the RNA, methylation, protein expression, and genomic alteration characteristics of AURKA. We then used single-cell RNA sequencing to resolve the functional heterogeneity of AURKA in the PRAD epithelial cell subpopulation and verified its impact on the malignant phenotype of desmoplasia-resistant prostate cancer cells in in vitro experiments. This research also analyzed the prognostic risk stratification of AURKA subpopulations in combination with various indicators and the potential benefit of AURKA inhibitors in combination with various treatments. Results The pan-cancer analysis demonstrated that AURKA expression heterogeneity was present among urological tumors at different molecular levels, and the positive correlation of AURKA alteration with MYC and E2F pathways was conserved in pan-cancer. Epithelial cell subpopulations with high expression of AURKA (epi3/4/6) promoted proliferation by regulating cell cycle and DNA repair, while low expression subsets (epi1/2/7) activated TNF-α and androgen receptor (AR) pathways to mediate drug resistance. In particular, AURKA may serve as a compensatory pathway to support tumor activity after AR inhibition in prostate cancer, a complex mechanism not seen in other tumors. AURKA-overexpressing patients with low Gleason scores or high PSA have a poor prognosis in clinical analysis. Furthermore, a comprehensive drug sensitivity co-analysis found that AURKA inhibitors may benefit from targeted therapy, ADC therapy, and immunotherapy. TMB and CD274 expression were the biomarkers of AURKA high-expression patients with PRAD for clinical outcome. Conclusion AURKA expression heterogeneity has been identified as a critical factor in the progression of PRAD and the development of drug resistance. The molecular subtyping of AURKA can serve as a precise strategy for combination therapy and provide a theory for the combination of AURKA inhibitors and targeted/immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/396eba19ede00209d3509820e66ea1756b82e606" target='_blank'>
              Aurora kinase-a expression heterogeneity and potential benefit of combination therapy in prostate adenocarcinoma
              </a>
            </td>
          <td>
            Ru Chen, Qianyi Qiu, Weiting Xie, Jun Lin, Rong Liu, Jianhui Chen, Shaoxing Zhu, Yiming Su
          </td>
          <td>2025-07-11</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Copy number gene amplification and associated overexpression of driver oncogenes are genetic events that contribute to cancer progression and drug resistance. MDS1 and EVI1 Complex locus (MECOM) gene is copy number amplified and overexpressed in aggressive epithelial ovarian cancers. The biological function and precise molecular mechanism of MECOM in the progression and drug resistance of ovarian cancer remain unclear. Here, we unravel MECOM as a regulator of KRAS and its downstream MAP Kinase signalling pathway, and also identify epigenetic inhibitor JIB-04 as a pharmacological agent targeting MECOM/KRAS axis. RNAi-mediated attenuation of MECOM in ovarian cancer cells harboring MECOM amplification reduced their proliferation, impaired colony formation, and impeded cellular migration. ChIP-qPCR analysis confirmed binding of MECOM to the KRAS promoter, suggesting direct regulation of the KRAS gene at the transcriptional level. Further, MECOM promoted cellular proliferation by regulating KRAS-mediated ERK/ZEB1 signalling cascade. The anti-tumorigenic effects due to MECOM loss were phenocopied by the treatment of ovarian cancer cells harboring MECOM amplification with JIB-04 epigenetic inhibitor targeting Jumonji domain histone demethylase enzymes. By ChIP-qPCR, we show that JIB-04 induced transcriptional changes of MECOM by altering H3K27me3 demethylation at its promoter region. We further report that ovarian cancer cells expressing high-MECOM levels exhibit cisplatin resistance, which could be effectively reversed upon pre-treatment with JIB-04. The therapeutic efficacy of JIB-04 was further demonstrated in mice bearing ovarian cancer cell xenografts, where JIB-04 slowed down the tumor growth in corroboration with diminishing MECOM expression. RNA-sequencing analysis identified potential cisplatin resistance gene, SUB1, being regulated by JIB-04-mediated modulation of MECOM expression. Altogether, these data suggest that epigenetic silencing of MECOM by JIB-04 mediated H3K27me3 modulation is an important mechanism in ovarian cancer and provide a new therapeutic target for the treatment of ovarian cancers harboring MECOM amplification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a776f5b72c3bc81fd8c696f0bdd3032a4ca281b9" target='_blank'>
              Epigenetic targeting of MECOM/KRAS axis by JIB-04 impairs tumorigenesis and cisplatin resistance in MECOM-amplified ovarian cancer
              </a>
            </td>
          <td>
            Ibha Singh, Amarnath Karna, Anita Prajapati, Ujjawal Solanki, Archana Mukherjee, Sheetal Uppal, Pawan Malhotra, Manoj Kumar, P. Agarwal
          </td>
          <td>2025-07-15</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Disease-associated variants can lead to variable phenotypic outcomes, but the biological mechanisms underlying this variability remain poorly understood. We developed a framework to investigate this phenomenon using the 16p12.1 deletion as a paradigm of variable expressivity. Using induced pluripotent stem cell models from affected families and CRISPR-edited lines with the 16p12.1 deletion, we found that the deletion and secondary variants in the genetic background jointly influenced chromatin accessibility and expression of neurodevelopmental genes. Cellular analyses identified family-specific phenotypes, including altered inhibitory neuron production and neural progenitor cell proliferation, which correlated with head-size variation. CRISPR activation of individual 16p12.1 genes variably rescued these defects by modulating key signaling pathways such as TGF-{beta} and PI3K-AKT. Integrative analyses further identified regulatory hubs, including transcription factors FOXG1 and JUN, as mediators of these effects. Our study provides a functional framework for investigating how individual genetic architectures contribute to phenotypic variability in genetic disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8eba5322d12d293981db603db996dc82168694a5" target='_blank'>
              An integrated framework for functional dissection of variable expressivity in genetic disorders
              </a>
            </td>
          <td>
            Jiawan Sun, Serena Noss, Deepro Banerjee, V. H. Bhavana, Corrine Smolen, Maitreya Das, Belinda Giardine, Anisha Prabhu, David J. Amor, K. Pope, Paul J Lockhart, S. Girirajan
          </td>
          <td>2025-07-23</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10519a9f5e1766f9e6003dc61586e873fd6de1a7" target='_blank'>
              An optimised computational approach for the identification of somatic structural variants in cancer
              </a>
            </td>
          <td>
            Sara Waise, Nana Mensah, Tom Lesluyes, J. Demeulemeester, Adrienne M. Flanagan, N. Pillay, P. Van Loo
          </td>
          <td>2025-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Prostate cancer continues to be the most common cause of cancer-related disease and mortality among men worldwide, especially in the advanced stages, notably metastatic castration-resistant prostate cancer (mCRPC), which poses significant treatment challenges. Docetaxel, a widely used chemotherapeutic agent, has long served as the standard treatment, offering survival benefits and mitigation. However, its clinical impact is frequently undermined by the development of chemoresistance, which is a formidable challenge that leads to treatment failure and disease progression. The mechanisms driving docetaxel resistance are diverse and complex, encompassing modifications in androgen receptor signaling, drug efflux transporters, epithelial-mesenchymal transition (EMT), microtubule alterations, apoptotic pathway deregulation, and tumor microenvironmental influences. Recent evidence suggests that extracellular RNAs influence drug responses, further complicating the resistance landscape. This review offers a broad discussion on the mechanisms of resistance and explores novel therapeutic approaches to address them. These include next-generation taxanes, targeted molecular inhibitors, immunotherapies, and combination regimens that can be designed to counteract specific resistance pathways. By broadening our understanding of docetaxel resistance, this review highlights potential strategies to improve therapeutic efficacy and the potential to enhance outcomes in patients with advanced treatment-resistant prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/538206b7fcdd2ed8debda33969b113a8ee54b834" target='_blank'>
              Beyond Docetaxel: Targeting Resistance Pathways in Prostate Cancer Treatment
              </a>
            </td>
          <td>
            Tayo Alex Adekiya
          </td>
          <td>2025-08-01</td>
          <td>BioChem</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Therapy-induced acquired resistance limits the clinical effectiveness of mutation-specific KRAS inhibitors in colorectal cancer (CRC). Here, we investigated whether broad-spectrum, active-state RAS inhibitors meet similar limitations. We found that KRAS-mutant CRC cell lines were sensitive to the RAS(ON) multiselective RAS inhibitor RMC-7977, given that treatment resulted in RAS-RAF-MEK-ERK pathway inhibition; halted proliferation; and, in some cases, induced apoptosis. RMC-7977 initially reduced the activity of a compartment-specific, dual-color reporter of ERK activity, with reporter reactivation emerging after long-term dose escalation. These drug-resistant cell populations exhibited distinct patterns of phospho-protein abundance, transcriptional activities, and genomic mutations, including a Y71H mutation in KRAS and an S257L mutation in RAF1. Transgenic expression of KRASG13D, Y71H or RAF1S257L in drug-sensitive CRC cells induced resistance to RMC-7977. CRC cells that were resistant to RMC-7977 and harboring RAF1S257L exhibited synergistic sensitivity to concurrent inhibition of RAS and RAF. Our findings demonstrate the power of reporter-assisted screening together with single-cell analyses for dissecting the complex landscape of therapy resistance. The strategy offers opportunities to develop clinically relevant combinatorial treatments to counteract the emergence of resistant cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/062fb5066a8ceecc8901f69c1d9a2835ca85a048" target='_blank'>
              Reporter-based screening identifies RAS-RAF mutations as drivers of resistance to active-state RAS inhibitors in colorectal cancer.
              </a>
            </td>
          <td>
            Oleksandra Aust, Moritz R T Thiel, Eric Blanc, Mareen Lüthen, Viola Hollek, Rosario Astaburuaga-García, B. Klinger, Francisca Böhning, Alexandra Trinks, D. Beule, Björn Papke, David Horst, Nils Blüthgen, C. Sers, C. Der, M. Morkel
          </td>
          <td>2025-07-22</td>
          <td>Science signaling</td>
          <td>0</td>
          <td>122</td>
        </tr>

        <tr id="Effective identification of oncogenic mutations is essential for diagnosis, forecasting resistance, and metastasis in remission. It is required for an optimal drug regimen. We develop a framework to discover mutations that co-exist in different oncoproteins, and those that are excluded, likely encoding oncogene-induced senescence. First, mapping the proteins onto pathways assists combinatorial drug selections and helps to detect metastases. Second, it provides the molecular basis for synthetic lethality, to date investigated at the genome level. Our pan-cancer profiles of ~60,000 tumor sequences, detect 3424 co-existing tumor-specific mutations. Mapping them onto pathways indicates that they preferentially promote specific primary tumors. We uncover metastatic mutations and provide metastatic breast-cancer markers. This work not only clarifies the mechanistic basis of intratumor mutational diversity but usefully reveals markers for metastasis in patients’ genomes and introduces a novel computational framework for detecting metastasis based on tumor mutational profiles. Mapping the mutations onto pathways provides an invaluable metastasis-targeting resource, guiding drug combinations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd8dbee9272fd01bd4b589963df2977d50caa0e8" target='_blank'>
              Mutations in tumor signaling, metastases, and synthetic lethality establish distinct patterns
              </a>
            </td>
          <td>
            B. R. Yavuz, Uğur Şahin, Hyunbum Jang, Ruth Nussinov, Nurcan Tunçbağ
          </td>
          <td>2025-08-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="ABSTRACT Introduction Next‐generation sequencing‐based comprehensive cancer genomic profiling (CGP) tests are beneficial for refining diagnosis and personalized treatment of various cancers. However, the clinical impact of CGP, as covered by public health insurance in the management of sarcomas, remains unknown. Especially, the data on the utility of the newly emerging dual DNA–RNA panel compared to the conventional DNA‐only panel in clinical settings is lacking. Therefore, we evaluated the utility of CGP in routine clinical practice for sarcoma treatment. Patients and Methods In this study, three types of DNA panel and one DNA–RNA panel, reimbursed by Japanese public health insurance, were utilized. We detected oncogenic and druggable gene mutations and genotype‐matched therapies. Results One hundred and thirty‐six patients were included in this study. Based on the detection of highly histology‐specific translocations in the sequencing results, 2.2% of patients were re‐classified. In patients with translocation‐related sarcomas, a DNA–RNA panel identified more histology‐specific fusion genes than DNA panels (p = 0.0035). Specifically, 86.8% and 39.0% of patients had oncogenic and druggable genomic alterations, respectively. Of these, 9.6% underwent genotype‐matched therapy, with a 36.3% response rate and an 81.8% disease control rate. Patients who were administered genomically matched therapy had better overall survival (OS) than those who did not in patients with metastatic or advanced sarcoma with no prior chemotherapy (3‐year OS: 83.3% vs. 48.0%, p = 0.42). Patients with TP53 and RB1 mutations had worse OS than those without. Germline findings were detected in 11.0% of the patients, one of whom had a truly germline origin. Conclusions This study suggests that publicly reimbursed CGP tests, particularly the dual DNA–RNA panel, could be beneficial for refining diagnostic precision in selected sarcoma subtypes, treatment decisions, detecting the germline findings, and prognosis prediction in routine clinical settings for sarcoma. The implementation of genotype‐matched therapies showed favorable clinical outcomes and improved the prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35f63a84e1907839e26eef4d1f8d8db48869e5a3" target='_blank'>
              Real‐World Data of Comprehensive Cancer Genomic Profiling Tests Performed in the Routine Clinical Setting in Sarcoma
              </a>
            </td>
          <td>
            E. Nakata, D. Ennishi, Tatsunori Osone, K. Ninomiya, Shuta Tomida, Takuto Itano, T. Fujiwara, T. Kunisada, Naoyuki Ida, Hideki Yamamoto, Mashu Futagawa, T. Shimoi, Hiroyuki Yanai, Akira Hirasawa, S. Toyooka, Masahiro Tabata, Toshifumi Ozaki
          </td>
          <td>2025-08-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Tumor Treating Fields (TTFields) therapy is an approved cancer treatment modality, based on non-invasive application of electric fields to the tumor region. Proteomic and cell biology methods revealed a versatile mechanism of action to be involved in the response to TTFields. In the current research we performed whole transcriptome analysis across tumor types to identify pan-cancer responses to TTFields. For this we collected samples from control and TTFields-treated human cancer cell lines of gastric cancer, pancreatic cancer, ovarian cancer, non-small cell lung carcinoma, pleural mesothelioma, and glioblastoma. The transcriptomic analysis supported previous reported effects: downregulation of pathways associated with cell cycle, cell growth, and proliferation; downregulation of DNA replication and the FA-BRCA DNA repair pathway; and upregulation of cellular responses to stress—senescence, autophagy, and apoptosis. Notably, previously unrecognized downstream effects of TTFields were revealed on cellular metabolism, with downregulation of protein and RNA metabolism, and upregulation of steroid biosynthesis. Additional DNA repair pathways were also found to be downregulated, including nucleotide excision repair, base excision repair, and mismatch repair. In conclusion, this study revealed similar response patterns to TTFields across different tumor types, re-enforcing some already pinpointed mechanisms, while revealing new mechanisms. Unlocking these new mechanisms may allow identification of potential new cancer treatments for application together with TTFields based on mechanistical compatibility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1627b07ec04103438850967d9453d926d5ec1e3d" target='_blank'>
              The transcriptomic fingerprint of cancer response to Tumor Treating Fields (TTFields)
              </a>
            </td>
          <td>
            K. Wainer-Katsir, A. Haber, H. Fishman, L. Ding, Michael D. Story, Renfei Du, U. Kahlert, L. Mannarino, F. Mirimao, M. Lupi, Maurizio D’Incalci, G. Lavy-Shahaf, Hila Ene, R. Frechtel-Gerzi, Zeina Drawshy, A. Martinez-Conde, E. Dor-On, Y. Porat, M. Giladi, U. Weinberg, Y. Palti
          </td>
          <td>2025-07-10</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21690ca2e89ea2f16a09c71f18f6490b7208397c" target='_blank'>
              High Throughput Screening Identifies Small Molecules that Synergize with MRTX1133 Against Acquired Resistant KRAS G12D Mutated CRC
              </a>
            </td>
          <td>
            Natalie Thielen, Ning Wei, Emiko Nagai, Edward Chu, S. Kitamura, Chaoyuan Kuang
          </td>
          <td>2025-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background Osteosarcoma is the most common primary malignant bone tumor, with high invasiveness and metastatic potential and a poor prognosis in patients with metastatic cancer. Despite the rapid advancements in genomics in recent years that provided new perspectives for studying the molecular mechanisms of osteosarcoma, the understanding of its tumor heterogeneity and evolutionary mutation process remains limited. Methods In this study, whole-exome evolutionary profiling was performed on data from the TARGET database representing 61 osteosarcoma cases. Subclonal architectures were reconstructed to characterize mutational trajectories. Differential mutation analysis was used to identify candidate metastasis-associated mutations. These features were used to build a metastasis-prediction classifier, which was cross-validated and tested on an independent external cohort. Finally, Suppes’ probabilistic theory of causality was integrated with cohort data to infer high-frequency evolutionary paths linked to metastasis. Results A linear evolutionary trajectory was observed in 62% of patients, indicating sequential clonal expansion. Eight key mutations were closely associated with metastatic progression. The classifier achieved 83% accuracy in cross-validation and maintained robust performance on the external validation set. Through causal inference, distinct evolutionary routes underpinning metastasis were uncovered, with ATRX mutations frequently occurring as early events that reshaped clonal dynamics and facilitated tumor spread. Conclusions In this study, the dynamic evolutionary landscape of osteosarcoma metastasis was delineated, an early metastasis classification model was constructed, and the impact of early clonal ATRX mutations on metastasis initiation were highlighted. These findings offer potential avenues for the early diagnosis and risk assessment of osteosarcoma. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06796-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68812275cd636690cb798f47d80f2494eb2d39fb" target='_blank'>
              Whole‑exome evolutionary profiling of osteosarcoma uncovers metastasis‑related driver mutations and generates an independently validated predictive classifier
              </a>
            </td>
          <td>
            Zhen Wang, Zhe Wang, Ruoyu Wang, Zumin Wang, Xiangyang Cao, Rui Chen, Zebing Ma, Shanshan Liang, Shuai Tao
          </td>
          <td>2025-07-07</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="MYC is a key driver in many aggressive and therapy-resistant cancers. We have developed and characterized a small-molecule MYC inhibitor named MYCi975. To uncover combination strategies for MYC inhibitors, we conducted a genome-wide CRISPR screen using MYCi975. This screen revealed a notable synthetic lethality when MYC inhibition was paired with disruption of mitochondrial complex I components, but not other complexes. Mechanistically, MYC inhibition reduced oxidative phosphorylation and glycolysis, triggering a compensatory up-regulation of complex I genes. Consequently, genetic or pharmacological targeting of complex I sensitized tumors to MYCi975 treatment, leading to increased purine catabolism and infiltration of CD8+ T cells and macrophages into tumors. Additionally, a wide range of tumor cells with lower complex I expression showed increased MYC dependency. These results indicate that metabolic adaptation to MYC inhibition exposes a targetable weakness at complex I and provide a rational strategy for combination therapy with emerging MYC inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70cd4c2898105cbb0d41fa7290f74973a1ee0f8e" target='_blank'>
              Impaired mitochondrial metabolism is a critical cancer vulnerability for MYC inhibitors
              </a>
            </td>
          <td>
            William Yang, Qianyu Guo, Songhua Quan, Zachary R. Chalmers, J. B. Parker, M. Truica, Mary F Dufficy, Megan M Kerber, Karthik Vasan, D. G. Gupta, A. W. Steffeck, Hao Pan, Mohammed Siddiqui, H. T. Pham, G. Schiltz, Debabrata Chakravarti, Navdeep S. Chandel, S. Abdulkadir
          </td>
          <td>2025-07-16</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8160c99e1605cc6ad6c9ab6ffeb17ddfb879e35e" target='_blank'>
              Bioinformatics-Based Exploration of LMO4-Regulated Hippo Pathway to Reverse Adriamycin Resistance in Breast Cancer.
              </a>
            </td>
          <td>
            Yue Wu, Jiahui Gao, Zejie Su, Anran Gao, Jingxiu Zhao, Guanying Qiao, Minling Ye, Dongdong Liu, Jianhua Xu
          </td>
          <td>2025-08-01</td>
          <td>IUBMB life</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Paclitaxel remains a cornerstone of cancer therapy, yet resistance significantly limits its clinical efficacy. Emerging evidence implicates circular RNAs (circRNAs) as key regulators of paclitaxel resistance, offering potential biomarkers and therapeutic targets. This review synthesizes current knowledge on circRNA-mediated resistance mechanisms, categorizing their roles as promoters or suppressors of chemoresistance. We highlight two major pathways—miRNA sponging and protein interactions—through which circRNAs alter drug sensitivity. Moreover, we focus on enhancement of epithelial–mesenchymal transition and modulation of autophagy as two significant downstream pathways influenced by the established primary mechanisms. Notably, our analysis reveals context-dependent dualities, where certain circRNAs exert opposing effects in different cancers, underscoring the need for tissue-specific therapeutic strategies. Furthermore, we identify critical gaps in understanding circRNA regulatory networks and their clinical translatability. By evaluating circRNA signatures associated with paclitaxel response, this review proposes a framework for biomarker development and combination therapies to circumvent resistance. Our findings emphasize the urgency of functional studies and standardized profiling methods to harness circRNAs for personalized oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b6cc3f7c3626a0076f8fb95907d4419e7ad6fdb" target='_blank'>
              Circular RNAs: key regulators of paclitaxel sensitivity and resistance in cancer
              </a>
            </td>
          <td>
            Zahra Tajik, S. Ghafouri-Fard
          </td>
          <td>2025-08-18</td>
          <td>European Journal of Medical Research</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Background Despite substantial progress in discovering novel therapeutic targets and compounds, cancer hyperproliferative cells (CHCs) play a central role in driving tumor progression in gastric cancer (GC). High-resolution single-cell RNA sequencing (scRNA-seq) integrated with spatial transcriptomics sequencing (ST-seq) elucidating the role of the CHC microenvironment in driving GC progression remains lacking. Methods Through integrated analysis of 40 scRNA-seq samples and 6 ST samples from human gastric tissues spanning tumor progression, we delineate the phenotypic plasticity of tumor epithelium and characterize the transcriptional trajectory from normal gastric cells to CHCs, a process associated with RRM2 overexpression. Finally, we validated the analysis results through cellular and animal experiments. Results Our findings demonstrate that RRM2 knockdown triggers ferroptosis in GC cells by promoting lipid peroxidation and intracellular iron accumulation. We further screen a compound and RRM2 inhibitor, osalmid, which effectively suppresses tumor growth in GC cell xenografts by triggering ferroptosis, thus offering a new therapeutic option for GC. Conclusions This study elucidates the critical role of RRM2 + CHCs in gastric cancer development and progression, providing novel insights for targeted therapies. The discovery of osalmid as a potential therapeutic agent offers a promising new treatment strategy for GC. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06847-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ad62270a5ce806049ffe5312712ee483297a8c1" target='_blank'>
              Integration of scRNA-seq and ST-seq identifies hyperproliferative RRM2+ cells features and therapeutic targets in gastric cancer
              </a>
            </td>
          <td>
            Shuai Ping, Xiong Jia, Yanan Tian
          </td>
          <td>2025-07-15</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Ovarian clear cell cancer (OCCC) is a pathological type of ovarian cancer. OCCC responds poorly to platinum-based chemotherapy drugs and immunotherapy. Currently, there is a lack of targeted drugs for OCCC. It is very urgent to find new therapeutic targets for OCCC. Methods We used data from the eQTLGen consortium and deCODE, combined with large Genome-Wide Association Study (GWAS) cohort data, to perform drug target Mendelian randomization (MR) analysis. Colocalization analysis was used to identify target genes. Then, we analyzed single-cell sequencing data from the GEO database, combing with gene enrichment analysis to explore possible molecular mechanisms. Drug screening and molecular docking verified the medicinal value of the targets. Results Three genes were selected in the MR analysis. Colocalization supported two of them, PPP1R14A and PTGS2. Single-cell sequencing analysis showed that these genes were related to tumor immunity. Drug prediction and molecular docking showed that PTGS2 had druggable potential. Conclusions This study identified two potential drug targets for OCCC. These drug targets may be relevant to tumor immunotherapy. However, further studies are necessary to confirm these findings and clarify the underlying mechanisms. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03043-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e696d343753dbcc5795652fe97b66bbf106f34f5" target='_blank'>
              Integrated eQTL-pQTL Mendelian randomization and single-cell sequencing reveal therapeutic targets in ovarian clear cell cancer
              </a>
            </td>
          <td>
            Yuwen Qi, Yang Gao, Hongbing Cai
          </td>
          <td>2025-07-01</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is an aggressive malignancy with a poor prognosis that is driven primarily by oncogenic KRAS mutations present in > 90% of cases. KRAS mutations, particularly the G12D mutation which dominates in PDAC, fuel tumor initiation, progression, and immune evasion, thereby contributing to therapy resistance. Nevertheless, KRAS has long been considered “undruggable” due to its structure. Recent advances have spurred transformative progress in direct KRAS inhibition. While FDA-approved mutation-specific and pan-KRAS inhibitors show limited efficacy in PDAC, emerging agents (MRTX1133 and RMC-9805) have demonstrated preclinical promise. However, resistance remains a critical hurdle and is driven by pathway reactivation, secondary mutations, and metabolic adaptations. Alternative strategies targeting upstream regulators (SHP2 and SOS1) aim to block KRAS activation and associated resistance mechanisms. Preclinical studies have also highlighted synergistic benefits of combining KRAS inhibitors with MEK, PI3K, or CDK4/6 inhibitors, which are now undergoing clinical evaluation. Immunotherapies, including KRAS-targeted vaccines and adoptive T-cell therapies, have further expanded the therapeutic landscape of enhancing KRAS-targeted therapies in PDAC. The molecular basis of KRAS-driven PDAC, current inhibitors, resistance mechanisms, and innovative strategies are discussed herein to address treatment barriers. Opportunities to improve clinical outcomes are underscored in this challenging malignancy by integrating insights from preclinical and clinical research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d8cfbe6080878693ce57a6e261fa2c4f9ddf499" target='_blank'>
              Drugging the ‘undruggable’ KRAS: breakthroughs, challenges, and opportunities in pancreatic cancer
              </a>
            </td>
          <td>
            Nawaz Khan, Umar Raza, Syed Aqib Ali Zaidi, Muhadaisi Nuer, Kayisaier Abudurousuli, Yipaerguli Paerhati, Alifeiye Aikebaier, Wenting Zhou
          </td>
          <td>2025-07-07</td>
          <td>Cancer Biology & Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Introduction CDK4/6 inhibitors are cornerstone therapies for advanced HR+/HER2- breast cancer, yet treatment response heterogeneity remains a major clinical challenge. This study integrates single-cell regulatory landscapes with multi-omics data to decode resistance mechanisms and develop predictive biomarkers for CDK4/6 inhibitor response stratification. Methods Single-cell RNA-seq data (GSE158724, n=14 samples) and bulk multi-omics profiles (TCGA-BRCA, n=1,059; GSE186901, n=90) were analyzed. Gene regulatory networks were reconstructed using SCENIC to identify resistance-specific regulons. The Tumor Prognostic Regulon Index (TPRI) was derived from five prognostic transcription factors and validated in independent cohorts. Experimental validation including qPCR of core TFs was performed in patient-derived samples. Multimodal predictive models integrating TPRI, differentially expressed genes, and miRNAs were developed using logistic regression, with performance assessed via ROC/AUC analysis. Results We identified 86 resistance-associated regulons and established TPRI based on five prognostic TFs (ATF1, TEAD4, NFIL3, FOXO1, ETV3). TPRI significantly stratified patients into high/low-risk groups with differential overall survival and treatment response (Fisher’s exact test P=0.0237). qPCR confirmed elevated expression of these TFs in resistant tumors (P<0.01). High-risk patients exhibited increased stemness indices (mRNAsi, P<2.2e-16) and mTOR pathway activation. The multimodal model (TPRI + top 30 DEGs + top 30 miRNAs) achieved superior prognostic accuracy (95%CI:0.6575-0.75). Discussion This study establishes TPRI as a novel biomarker for CDK4/6 inhibitor response prediction, validated through multi-omics integration and qPCR confirmation. The model provides actionable risk stratification, where high-risk patients may benefit from combinatorial mTOR-targeted therapies. Limitations include sample size constraints for methylation integration. Future studies should validate these findings in prospective clinical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2af1d91144e6176cc73b2a5ca97b747f7333a1d7" target='_blank'>
              Integrating single-cell regulatory atlas and multi-omics data for differential treatment response and multimodal predictive modeling in CDK 4/6 inhibitor-treated breast cancer
              </a>
            </td>
          <td>
            Li Yan, Sijie Chen, Ran Ran, Shidi Zhao, Jing Huang, Jin Yang
          </td>
          <td>2025-07-17</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Epidermal growth factor receptor (EGFR) mutations occur in approximately 10–20% of Caucasian and up to 50% of Asian patients with oncogene-addicted non-small cell lung cancer (NSCLC). Most frequently, alterations include exon 19 deletions and exon 21 L858R mutations, which confer sensitivity to EGFR tyrosine kinase inhibitors (TKIs). In the last decade, the third-generation EGFR-TKI osimertinib has represented the first-line standard of care for EGFR-mutant NSCLC. However, the development of acquired mechanisms of resistance significantly impacts long-term outcomes and represents a major therapeutic challenge. The mesenchymal–epithelial transition (MET) gene amplification and MET protein overexpression have emerged as prominent EGFR-independent (off-target) resistance mechanisms, detected in approximately 25% of osimertinib-resistant NSCLC. Noteworthy, variability in diagnostic thresholds, which differ between fluorescence in situ hybridization (FISH) and next-generation sequencing (NGS) platforms, complicates its interpretation and clinical applicability. To address MET-driven resistance, several therapeutic strategies have been explored, including MET-TKIs, antibody–drug conjugates (ADCs), and bispecific monoclonal antibodies, and dual EGFR/MET inhibition has emerged as the most promising strategy. In this context, the bispecific EGFR/MET antibody amivantamab has demonstrated encouraging efficacy, regardless of MET alterations. Furthermore, the combination of the ADC telisotuzumab vedotin and osimertinib has been associated with activity in EGFR-mutant, c-MET protein-overexpressing, osimertinib-resistant NSCLC. Of note, several novel agents and combinations are currently under clinical development. The success of these targeted approaches relies on tissue re-biopsy at progression and accurate molecular profiling. Yet, tumor heterogeneity and procedural limitations may challenge the feasibility of re-biopsy, making biomarker-agnostic strategies viable alternatives.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d5b36dc58c59a2e86bc451390872a0a87ad2555" target='_blank'>
              Management of MET-Driven Resistance to Osimertinib in EGFR-Mutant Non-Small Cell Lung Cancer
              </a>
            </td>
          <td>
            Panagiotis Agisilaos Angelopoulos, A. Passaro, I. Attili, P. Trillo Aliaga, C. Corvaja, G. Spitaleri, E. Battaiotto, E. Del Signore, G. Curigliano, F. de Marinis
          </td>
          <td>2025-06-30</td>
          <td>Genes</td>
          <td>1</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c32a97dbee7d52daeda75142b34c4c8f28f256d2" target='_blank'>
              ZBTB11 depletion targets metabolic vulnerabilities in KRAS inhibitor-resistant PDAC.
              </a>
            </td>
          <td>
            Nathan L Tran, Jiewei Jiang, Min Ma, Gillian E. Gadbois, K. Gulay, Alyssa L Verano, Cara R. Schiavon, Elena Rebollo, Haowen Zhou, Chun-Teng Huang, Rabi Murad, David A Scott, Uri Manor, Anne G. Bang, Hervé Tiriac, A. Lowy, Eric S Wang, F. Ferguson
          </td>
          <td>2025-08-11</td>
          <td>Nature chemical biology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Prostate cancer (PCa) is a leading cause of cancer-related mortality among men. Radiotherapy is the cornerstone of PCa treatment. However, a major limitation of radiotherapy is the development of resistance, which compromises treatment efficacy. Reactive oxygen species (ROS), which are generated by radiation, have a dual role in PCa by inducing DNA damage and apoptosis, while also promoting tumor progression and radioresistance. Elevated ROS levels enhance metabolic reprogramming, activate oncogenic pathways, and influence the tumor microenvironment by modulating immune responses and promoting the epithelial-mesenchymal transition (EMT). Key molecular mechanisms, including the Nrf2/Keap1 signaling axis, Bcl-2 mutations, and Speckle-type POZ protein alterations, contribute to radioresistance by enhancing antioxidant defenses and DNA repair capacity. Additionally, the interplay between hypoxia, androgen receptor variants (AR-Vs), and ferroptosis regulators further influence radiotherapy outcomes. Understanding these resistance mechanisms is crucial for developing targeted strategies to enhance radiosensitivity and improve therapeutic outcomes in PCa patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f9f60cf373e93dea28bab21b0e3ffe7de720751" target='_blank'>
              Mechanisms underlying prostate cancer sensitivity to reactive oxygen species: overcoming radiotherapy resistance and recent clinical advances
              </a>
            </td>
          <td>
            Meidan Wang, Rui Xing, Liqun Wang, Mingyue Pan, Ruoyun Zhang, Ting Li, Weiqiang Sun, Jing Zhou
          </td>
          <td>2025-07-10</td>
          <td>Cancer Biology & Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="BACKGROUND
Glioblastoma (GBM) displays remarkable cell state plasticity, a major contributor to therapeutic resistance and tumor progression. While epigenetic mechanisms play a central role in driving this plasticity, the key regulators remain poorly understood, and developing effective therapeutic strategies targeting them has been challenging.


METHODS
We investigated the role of BRD2, a key regulator of NF-κB mediated mesenchymal (MES) transition, using GBM patient-derived xenograft (PDX) cell lines, CRISPR-mediated knock-in/knockout approaches, RNA-seq, and in vitro and in vivo modeling. BET inhibitors were employed to target MES gene expression and sensitize GBM to radiation therapy.


RESULTS
We found that PTEN loss induces RelA chromatin localization and acetylation-mediated recruitment of BRD2 to the MES gene promoters. BRD2 binding is essential for maintaining MES gene expression and phenotype. Genetic ablation or loss-of-function mutation of BRD2 bromodomains reverses MES transition, enhances radiation sensitivity, and improves survival in orthotopic xenograft models. Additionally, treatment with a brain-penetrant BD2-selective inhibitor suppresses the MES phenotype and increases radiation sensitivity of GBM stem cells in vitro.


CONCLUSION
Our study identifies BRD2 as a key mediator of MES transition in GBM, with its bromodomains playing a crucial role in driving cell state plasticity. Targeting BRD2 with BD2-selective inhibitors offers a promising therapeutic strategy to overcome radiation resistance and improve outcomes for GBM patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aa778fbdf5d6872b2c63bfc97f4589c5f7dfb10" target='_blank'>
              BRD2 Bromodomain-Mediated Regulation of Cell State Plasticity Modulates Therapy Response in Glioblastoma.
              </a>
            </td>
          <td>
            Raghavendra Vadla, Brett Taylor, Yohei Miyake, Benjamin Lin, Daisuke Kawauchi, Shunichiro Miki, Nidhi Nathwani, Brandon M Jones, Yashpreet Kaur, Abhinaba Banerjee, Philip Pham, J. Tsang, Albert Baldwin, David A Nathanson, Donald P. Pizzo, C. Miller, Frank B Furnari
          </td>
          <td>2025-07-19</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Prostate cancer (PCa) is a highly heterogeneous disease, with castration-resistant prostate cancer (CRPC) and neuroendocrine prostate cancer (NEPC) representing its most aggressive and therapy-resistant forms. Emerging evidence indicates that lineage plasticity—driven by key transcription factors such as Octamer Binding Factor 4 (OCT4)—plays a crucial role in therapeutic resistance and disease progression. OCT4, in coordination with SOX2 and NANOG, acts as a master regulator of stemness and is frequently upregulated in prostate cancer stem cells (PCSCs). This upregulation contributes to tumor initiation, metastasis, and resistance to both androgen deprivation therapy (ADT) and chemotherapy. In this review, we explore the role of OCT4 in mediating lineage plasticity in prostate cancer, with particular emphasis on its involvement in treatment resistance and neuroendocrine differentiation. We also examine therapeutic strategies aimed at targeting OCT4 directly, such as microRNA-mediated suppression, small-molecule inhibitors, and suicide gene therapy, as well as indirect approaches that modulate OCT4 expression via FGFR and NF-κB signaling pathways. While these strategies offer promising avenues, challenges such as adaptive resistance and the intricate signaling networks within PCSCs remain significant hurdles. A deeper understanding of the molecular mechanisms underlying OCT4-driven plasticity may pave the way for novel therapeutic approaches and improved outcomes in advanced prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a8ce283789b3c96575337b7647df8d11de9b2ba" target='_blank'>
              Unlocking the Role of OCT4 in Cancer Lineage Plasticity: A Cross-Cancer Perspective with an Emphasis on Prostate Cancer
              </a>
            </td>
          <td>
            Mohammad Esfini Farahani, Yanquan Zhang, A. O. Akinyemi, Fatemeh Seilani, Md Rakibul Alam, Xiaoqi Liu
          </td>
          <td>2025-07-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Tumor relapse remains a significant obstacle to successful therapy. Preclinical animal models that accurately reflect tumor relapse in patients are urgently needed. Here, we employed a dual recombinase-mediated genetic system to genetically trace and ablate proliferating cells in a polyomavirus middle T antigen (PyMT)-induced spontaneous murine breast cancer model. This system enabled the acute ablation of cells that had undergone proliferation within a defined time window, resulting in a drastic tumor shrinkage, followed by a gradual tumor relapse due to the presence of residual low-cycling cells. We then applied single-cell RNA sequencing (scRNA-seq) to unbiasedly compare the tumor ecosystems of the primary and relapsed PyMT tumors. Compared with the primary tumors, the relapsed tumors exhibited a higher proportion of cancer stem cells and pro-tumor γδ T cells, as well as co-expression of Spp1 and Vegfa in multiple myeloid cell populations – features that predict poor therapeutic response and unfavorable outcomes in human breast cancer patients. Collectively, this proliferation tracing and ablation model emulates chemotherapies that preferentially eliminate proliferating cancer cells, serving as a robust tool and a valuable resource for testing novel therapeutic strategies in relapsed tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95fdfebff6c6c3226be581651d876e8c2d584753" target='_blank'>
              Modeling tumor relapse using proliferation tracing and ablation transgenic mouse
              </a>
            </td>
          <td>
            Chuang Zhao, Xin-nan Zheng, Han-Ying Huang, Lin Tian
          </td>
          <td>2025-07-17</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Human induced pluripotent stem cells (iPSCs) have transformed biomedical research by enabling the generation of diverse cell types from accessible somatic tissues. However, certain fundamental biological properties, such as the genetic and epigenetic determinants of iPSC proliferation, remain poorly characterized. We measured the growth of iPSC lines derived from 602 unique donors using high-throughput time-lapse imaging, quantified proliferation through a growth Area-Under-the-Curve (gAUC) phenotype, and correlated gAUC with the gene expression and genotype of the cell lines. We identified 3,091 genes associated with gAUC, many of which are well established regulators of cell proliferation. We also found that rare deleterious variants in WDR54 were associated with reduced iPSC growth and that WDR54 was differentially expressed with respect to gAUC. Although no common variants showed a genome-wide association with gAUC, iPSC lines from monozygotic twins were highly correlated, and common genetic variation explained approximately 71-75% of the variance in iPSC growth rates. These results indicate a complex genetic architecture of iPSC growth rates, where rare, large-effect variants in important growth regulators, including WDR54, are layered onto a highly polygenic background. These findings have important implications for the design of pooled iPSC-based studies and disease models, which may be confounded by intrinsic growth differences.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d46302769c03d6ab5c0d658d81d85d4ec438bae8" target='_blank'>
              Genetics of growth rate in induced pluripotent stem cells
              </a>
            </td>
          <td>
            Brian N. Lee, Henry J. Taylor, Filippo Cipriani, Narisu Narisu, Catherine C. Robertson, A. Swift, Neelam Sinha, Tingfen Yan, L. Bonnycastle, Nathan Dale, Annie Butt, Hemant Parsaud, Stefan Semrau, Joshua W Knowles, Ivan Carcamo-Orive, Agnieszka D’Antonio-Chronowska, K. Frazer, Leslie G. Biesecker, S. Noggle, Michael R. Erdos, Dan Paull, Francis S. Collins, D. L. Taylor
          </td>
          <td>2025-07-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="Non-small cell lung cancer (NSCLC) accounts for over 80% of lung cancer cases and is often diagnosed at advanced stages. Resistance to immunotherapy in NSCLC involves genetic mutations, tumor microenvironment (TME) characteristics, treatment history, and age-related factors. Despite increasing use of immune checkpoint inhibitors (ICIs), resistance mechanisms remain poorly understood. Key genetic alterations associated with resistance include STK11, KEAP1, and EGFR mutations, particularly with low tumor mutational burden (TMB). The immunosuppressive tumor microenvironment, characterized by regulatory T cells and myeloid-derived suppressor cells, can hinder ICI efficacy. Metabolic alterations and deficient antigen presentation contribute to resistance. Prior treatments can alter the tumor microenvironment, affecting subsequent immunotherapy responses. Age-related factors, including immunosenescence, influence resistance, with older patients having higher TMB and more immunogenic microenvironments. Strategies to overcome resistance include combination therapies, biomarker-driven approaches, and targeting novel pathways. Combining ICIs with chemotherapy or radiation can enhance antitumor responses. Biomarker approaches, such as TMB and PD-L1 expression assessment, help tailor therapies. Exploring novel targets like RIG-I and STING pathways may provide additional solutions. Understanding these factors is crucial for developing personalized strategies to overcome immunotherapy resistance in NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/125abb896755dee7a90cc7e2811f84b34866e955" target='_blank'>
              Overcoming immunotherapy resistance in non-small cell lung cancer: a narrative review of related factors.
              </a>
            </td>
          <td>
            L. Gutiérrez-Babativa, Nicolle Wagner-Gutíerrez, Leonardo Rojas, Jairo Zuluaga, Oscar Arrieta, Andrés F. Cardona
          </td>
          <td>2025-08-01</td>
          <td>Immunotherapy</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Tumors represent dynamically evolving populations of mutant cells, and many advances have been made in understanding the biology of their progression. However, there are key unresolved questions about the conditions that support a cell’s initial transformation, which cannot be easily captured in patient populations and are instead modeled using transgenic cellular or animal systems. Results Here, we use extensive patient atlas data to define common features of the tumor DNA methylation landscape as they compare to healthy human cells and apply this benchmark to evaluate 21 engineered human and mouse models for their ability to reproduce these patterns. Notably, we find that genetically induced cellular transformation rarely recapitulates the widespread de novo methylation of Polycomb regulated promoter sequences as found in clinical samples, but can trigger global changes in DNA methylation levels that are consistent with extensive proliferation in vitro. Conclusions Our results raise pertinent questions about the relationship between genetic and epigenetic aspects of tumorigenesis as well as provide an important molecular reference for evaluating existing and emerging tumor models. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03650-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b654b38672fde0e6d98d2fd32826d3930c1b68a1" target='_blank'>
              Direct genetic transformation bypasses tumor-associated DNA methylation alterations
              </a>
            </td>
          <td>
            Sara Hetzel, Eran Hodis, Elena Torlai Triglia, Alexander Kovacsovics, Kathleen Steinmann, A. Gnirke, Meiying Cui, Daniel McQuaid, Raha Weigert, Georg Pohl, M. Muzumdar, Serge Leyvraz, Ulrich Keilholz, M. Yaspo, Aviv Regev, H. Kretzmer, Zachary D. Smith, Alexander Meissner
          </td>
          <td>2025-07-17</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Background: Cancer is a complex and adaptive disease characterised by uncontrolled cell proliferation, tissue invasion, and metastasis. Previously thought to be a genetic disease caused by DNA abnormalities, recent research demonstrates the critical role of metabolic reprogramming in cancer development. Cancer cells experience severe metabolic changes in the tumour microenvironment (TME), which include nutritional deprivation, hypoxia, and low pH.
Main text: The Warburg effect is a dramatic shift in which cells rely on aerobic glycolysis instead of oxidative phosphorylation (OXPHOS) for rapid energy production and growth. MYC, an oncogene, and TP53, a tumour suppressor, regulate metabolic alterations by increasing glycolysis, glutaminolysis, and other pathways necessary for tumour survival. Mitochondrial failure causes genomic instability, cancer, and resistance to apoptosis, increasing reliance on glycolysis and glutaminolysis. Cancer stem cells (CSCs) and resistant tumour forms are metabolically flexible, allowing them to adapt to environmental changes. The TME regulates metabolic pathways that promote tumour growth, including HIFs and AMPK.
Conclusion: Therapeutic drugs that target altered metabolic pathways, such as glycolytic enzyme inhibitors, glutaminolysis, and mitochondrial function, have shown promise in both preclinical and clinical studies. However, the metabolic flexibility of cancer cells and tumour heterogeneity make treatment problematic. Recent therapeutic strategies combining metabolic inhibitors, chemotherapy, and immunotherapy have yielded promising results.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b876dcea35fe7ae25ff90a9b6f3bd1444d55f527" target='_blank'>
              Unravelling the pathways driving tumour growth and resistance
              </a>
            </td>
          <td>
            A. Ubhenin, J. Ikebuiro, O. Erharuyi, Y. Abdulmalik, T. Danjuma
          </td>
          <td>2025-06-30</td>
          <td>Babcock University Medical Journal</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cancer is the origin of a novel tissue that attracts resources, spreads beyond boundaries, avoids normal controls, and escapes immunity. How does a novel tissue arise? The puzzle is that two seemingly different processes appear to be the primary driving force. On the one hand, overwhelming evidence links (epi)genetic driver mutations to the origin and progression of tumors. Common oncogenic mutations such as KRAS accelerate cell division, and common knockouts of tumor suppressors such as TP53 abrogate cell death or checks on cell division. On the other hand, cancerous tissues create complex traits that require intricate changes in cells and multiple interactions between different cell types. Such novelty often arises by hijacking the developmental plasticity that normally creates the diverse cells and tissues of our bodies from a single original zygotic cell. How can we reconcile the simple genetic changes in carcinogenesis with the complex developmental plasticity that creates novel tissues? This perspective advocates a new model. (Epi)genetic mutations release developmental plasticity. That developmental plasticity creates novel cellular interactions and complex tissues. Initially, novel traits created by developmental plasticity may not be stably heritable, thus subsequent (epi)genetic changes must stabilize the phenotypic novelty. Recent studies show how classic oncogenic and tumor suppressor driver mutations, such as KRAS and TP53, may primarily act in early carcinogenesis as broad releasers of developmental plasticity rather than as stimulators of cell division or knockout of limitations on cellular clonal expansion. In the new model, genetics releases, plasticity creates, and genetics stabilizes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0abe05e544bd1630685c7bbe6b925595c19894a8" target='_blank'>
              How cancer arises: Genetics releases, plasticity creates, genetics stabilizes
              </a>
            </td>
          <td>
            Steven A Frank
          </td>
          <td>2025-07-31</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5484870331a7d0dee6989fa1ddf5ea8e0b01c6cb" target='_blank'>
              Functional genomics and tumor microenvironment analysis reveal prognostic biological subtypes in Mantle cell lymphoma
              </a>
            </td>
          <td>
            Sunandini Sharma, Roshia Ali, Alyssa Bouska, Dylan Jochum, Meghana Kesireddy, Simeon Mahov, Joseph Lownik, Weiwei Zhang, Waseem G Lone, Mahfuza Afroz Soma, Alicia Gamboa, Vaishnavi Devarkonda, Dalia Elgamal, Atqiya Fariha, Adnan Mansoor, Douglas Stewart, Peter Martin, Brian K. Link, R. Advani, Paul M. Barr, A. Goy, A. Mehta, M. Kamdar, D. Stephens, Veronika Bachanova, L. Smith, Ryan D Morin, P. Pararajalingam, M. Lunning, Kai Fu, Dennis Wiesenberger, Wing C. Chan, J. Khoury, T. Greiner, J. Vose, Akil Merchant, Chengfeng Bi, J. Iqbal
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>111</td>
        </tr>

        <tr id="Triple negative breast cancer (TNBC) remains one of the most aggressive and therapeutically challenging subtypes of BC due to its lack of targeted treatment options and high plasticity. Increasing evidence suggests that epigenetic mechanisms, including DNA methylation, histone modifications, and non-coding RNA regulation, play a pivotal role in facilitating TNBC phenotype by influencing the expression of genes involved in subtype reprogramming. Modulating these epigenetic marks offers a promising strategy to drive TNBC subtype conversion toward less aggressive forms that are more responsive to treatment, thereby enhancing therapeutic efficacy. By targeting enzymes such as DNA methyltransferases and histone modifiers using epigenetic drugs (epidrugs), TNBC cells can be re-sensitized to hormone therapy, chemotherapy, and targeted treatments. This review delves into the role of epigenetic modulation in harnessing the plasticity of TNBC to drive its conversion into subtypes with better prognoses and discusses problems and limitations associated with the use of epidrugs in this context. By promoting subtype conversion, epidrugs offer a promising strategy for providing more personalized and effective therapies for TNBC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ada35a90892ec0be34375adfcf309b46149ff70" target='_blank'>
              Epigenetic Therapeutics: Reprogramming Triple-Negative Breast Cancer into Responsive Subtypes.
              </a>
            </td>
          <td>
            Wafaa S. Ramadan, A. Al-Azawi, L. Lozon, Rawan R. Kawaf, Ragheb Alsheikh Zein, Yahia El-Gharib, Raafat El-Awady
          </td>
          <td>2025-07-01</td>
          <td>Endocrine-related cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="SUMMARY During aging and cellular senescence, repetitive elements are frequently transcriptionally derepressed across species and cell types. Among these, the most abundant repeats by copy number in the human genome are Alu retrotransposons. Though Alu elements are often studied for their mutagenic potential, there is increasing appreciation for their contributions to other biological functions, including pro-inflammatory signaling and mitochondrial dysfunction. However, a comprehensive analysis of Alu-driven molecular changes remains to be conducted, and Alu’s potential contributions to aging features remain incompletely characterized. Here, we show that overexpression of an AluJb transposon in human primary IMR-90 fibroblasts leads to large-scale alterations across the transcriptome, cellular proteome, and secretome. Functional genomics analyses reveal alterations in aging/senescence pathways, broadly, and mitochondrial metabolism, proteostasis, cell cycle, and extracellular matrix pathways, more specifically. Our results demonstrate that Alu transcriptional upregulation is sufficient to drive widespread disruptions to cellular homeostasis that mirror aging-associated alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17f6625bec424a778a5bfdd6ab04b11196fc8086" target='_blank'>
              A multi-omics analysis of human fibroblasts overexpressing an Alu transposon reveals widespread disruptions in aging-associated pathways
              </a>
            </td>
          <td>
            Juan I. Bravo, Eyael Tewelde, Christina D. King, J. Bons, Samah Shah, Jacob P. Rose, Judith Campisi, Birgit Schilling, B. Benayoun
          </td>
          <td>2025-07-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Growth hormone (GH) signaling has been implicated in tumor progression and therapy resistance across multiple cancer types, yet its role in bladder cancer remains largely unexplored. In this study, we investigated the impact of GH and its receptor (GHR) on therapy resistance and disease progression in urothelial carcinoma (UC) through integrated transcriptomic and in vitro analyses. Transcriptomic profiling of The Cancer Genome Atlas bladder cancer cohort revealed that high tumoral GHR expression was associated with differential upregulation of genes involved in drug efflux, epithelial-to-mesenchymal transition (EMT), and extracellular matrix (ECM) remodeling. Notably, elevated GHR levels correlated with significantly reduced overall survival in patients with UC. In parallel, in vitro experiments demonstrated that GH promotes chemoresistance in UC cell lines via upregulation of ATP-binding cassette-containing (ABC) transporters and activation of EMT. GH also modulated ECM-remodeling-associated genes in a chemotherapy-dependent manner, including matrix metalloproteinases and tissue inhibitors of metalloproteinases. Importantly, these effects were abrogated by Pegvisomant, a GHR antagonist, indicating the functional relevance of GH/GHR signaling in the mediation of these phenotypes. Collectively, our findings support a mechanistic role for GH signaling in driving therapy resistance and tumor aggressiveness in bladder cancer and suggest GHR antagonism as a potential therapeutic strategy to improve treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26f8b97219a1f5f0a9e16e7142057f197a9d88cb" target='_blank'>
              Growth Hormone Signaling in Bladder Cancer: Transcriptomic Profiling of Patient Samples and In Vitro Evidence of Therapy Resistance via ABC Transporters and EMT Activation
              </a>
            </td>
          <td>
            Emily Davis, Lydia J. Caggiano, Hannah Munholland, R. Basu, D. Berryman, J. Kopchick
          </td>
          <td>2025-07-23</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="ABSTRACT Germline pathogenic variants (GPVs, ‘mutations’) causing inherited susceptibility to certain cancers (cancer susceptibility genes, CSGs) broadly belong to one of two main classes—loss of function variants in tumour suppressor genes (TSGs) or gain of function variants in proto‐oncogenes (an over‐simplification). Genomic analyses of tumours identify ‘driver mutations’ promoting tumour growth and somatic variants which contribute to ‘mutation signatures’ which, with histopathology, can be used to subclassify cancers with implications for causality and treatment. The identification of susceptible individuals is important, as they and their relatives may be at elevated risk of tumours, and this can influence optimal cancer treatment. Classically, cancer risk assessment utilises family history, lifestyle/environment factors, and any non‐neoplastic clinical findings, followed by genetic testing of high/moderate penetrance CSGs. In cancer cases not caused by highly penetrant CSGs, multiple variants conferring relatively small risks play a major role. These were discovered by genome‐wide association (GWAS) studies. The utility of polygenic risk scores (PRS) derived from multiple such variants for clinical risk profiling is being assessed. Access to genetic tests is improved by widening eligibility criteria for testing and empowering non‐genetic clinicians to identify CSG GPVs and manage carriers. This will contribute to expanding programmes of screening, prevention and early detection (SPED), with personalised surveillance and prophylactic interventions, and exploit knowledge of the molecular mechanisms of cancer susceptibility to develop novel cancer therapies. In some jurisdictions, population testing is being considered, but GPV penetrance in this setting can be unclear, and the public health implications are complex.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee21b45930d11565d54a3022b9abc97a84ec0804" target='_blank'>
              Inherited Susceptibility to Cancer: Past, Present and Future
              </a>
            </td>
          <td>
            Shirley V Hodgson, W. Foulkes, E. Maher, C. Turnbull
          </td>
          <td>2025-07-21</td>
          <td>Annals of Human Genetics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Oncogenic retroviruses are known for their pathogenesis via insertional mutagenesis, in which the presence of a provirus and its transcriptional control elements alter the expression of a nearby or surrounding host gene. There are reports of proviral integration driving oncogenesis in people with HIV and the use of HIV-derived vectors for gene therapy has raised concern about oncogenic side effects. To study this issue, we used an in vitro primary human CD4 + T cell infection model developed in our laboratory to identify HIV-1 integration sites that might influence cell proliferation or survival. Combining integration site analysis and bulk RNA sequencing, we found that an upregulated STAT3 signature due to proviral insertional mutagenesis was associated with persistent HIV-infected CD4 + T cells. HIV+ persistent cells also expressed a STAT3-related anti-apoptotic and cytotoxic phenotype that resembles that of HIV-associated T cell lymphomas. HIV insertional mutagenesis of STAT3 and expression of its downstream targets provides a model of HIV-associated T cell lymphomas that can be used to further determine the oncogenic drivers of HIV-associated lymphomas, both AIDS- and gene therapy-associated, and, potentially, to evaluate therapeutics against these HIV-associated cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/309400b908a0604bb16da7990788bd142c72eac7" target='_blank'>
              In vitro HIV DNA integration in STAT3 drives T cell persistence—A model of HIV-associated T cell lymphoma
              </a>
            </td>
          <td>
            M. Rist, M. Kaku, John M. Coffin
          </td>
          <td>2025-07-01</td>
          <td>PLOS Pathogens</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Throughout the last century, aneuploidy has been cemented as a hallmark of cancer. Although the association of aneuploidy with tumorigenesis has been well established, the role of these genetic imbalances in tumor formation has only recently begun to be elucidated. Advancements in genomics have revealed the complexity and context dependence of the effect of aneuploidy on cancer growth, while developments in genetic editing have allowed for proper modeling of specific aneuploidies. In this review, we discuss the key factors to consider when studying the role of aneuploidy in cancer and the tools that are available to do so. We then highlight recent studies that establish phenotypic contributions of aneuploidy to tumorigenicity. In particular, we highlight how general aneuploidy and chromosomal instability affect the tumor microenvironment and how specific chromosomal alterations, including the loss of chromosome 9p and the gain of chromosomes 8q and 1q, influence tumor behavior and therapeutic responses. Finally, we emphasize the potential of targeting aneuploidy-induced vulnerabilities to improve cancer treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37b1804dd6a9ac26725f995589c82459ba79d112" target='_blank'>
              Modeling and targeting general and chromosome-specific aneuploidy in cancer.
              </a>
            </td>
          <td>
            Aleah Goldberg, Maria Trifas, Teresa Davoli
          </td>
          <td>2025-07-28</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Epigenetic changes, such as DNA methylation, chromatin remodeling, and histone modifications, regulate gene expression without altering the DNA sequence. This review systematically analyzed over 500 studies including human cell line experiments (n>200), animal models (n>50), clinical cohort studies (n>100), and bioinformatics analyses retrieved from PubMed, Web of Science, and TCGA (The Cancer Genome Atlas). Studies increasingly show that genes involved in glucose and lipid metabolism, energy production, and modulation of metabolic hormones are regulated through epigenetic mechanisms. On the other hand, various metabolites participate in epigenetic modifications as coenzymes or substrates. Therefore, a greater understanding of the crosstalk between metabolism and epigenetics in cancer-related pathways could lead to the identification of key signaling molecules for targeted therapies, and raise the possibility of using dietary interventions to modulate epigenetic markers for individualized treatment. In this review, we have summarized the metabolic and epigenetic regulatory networks in cancer development, including glycolipid metabolic reprograming, the role of metabolites produced by the glut flora and tumor microenvironment, and key epigenetic drivers such as non-coding RNAs (ncRNAs). Data were curated from peer-reviewed articles, grounded in mechanistic studies using cell lines (SW480, MCF7 (Michigan cancer foundation-7)) and animal models (APC-mutant mice), with a focus on mechanistic studies, omics analyses, and translational research. Furthermore, we have discussed the potential of therapeutically targeting these pathways, along with the current challenges and future research directions, and a new strategy for reversing therapeutic drug resistance based on metabolism and epigenetic interaction was systematically explored.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0447a75110b1468dd9ba2d78ee2627a7a828fee6" target='_blank'>
              Metabolism and epigenetics in cancer: toward personalized treatment
              </a>
            </td>
          <td>
            Xiaoman Zhang, Dequan Liu, Sulan Yin, Yaru Gao, Xiaorui Li, Guangzhen Wu
          </td>
          <td>2025-07-25</td>
          <td>Frontiers in Endocrinology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="CRISPR-Cas9 technology has rapidly advanced as a transformative genome-editing platform, facilitating precise genetic modifications and expanding therapeutic opportunities across various diseases. This review explores recent developments and clinical translations of CRISPR applications in oncology, genetic and neurological disorders, infectious diseases, immunotherapy, diagnostics, and epigenome editing. CRISPR has notably progressed in oncology, where it enables the identification of novel cancer drivers, elucidation of resistance mechanisms, and improvement of immunotherapies through engineered T cells, including PD-1 knockout CAR-T cells. Clinical trials employing CRISPR-edited cells are demonstrating promising results in hematologic malignancies and solid tumours. In genetic disorders, such as hemoglobinopathies and muscular dystrophies, CRISPR-Cas9 alongside advanced editors like base and prime editors show significant potential for correcting pathogenic mutations. This potential was affirmed with the FDA's first approval of a CRISPR-based therapy, Casgevy, for sickle cell disease in 2023. Neurological disorders, including Alzheimer's, ALS, and Huntington's disease, are increasingly targeted by CRISPR approaches for disease modelling and potential therapeutic intervention. In infectious diseases, CRISPR-based diagnostics such as SHERLOCK and DETECTR provide rapid, sensitive nucleic acid detection, particularly valuable in pathogen outbreaks like SARS-CoV-2. Therapeutically, CRISPR systems target viral and bacterial genomes, offering novel treatment modalities. Additionally, CRISPR-mediated epigenome editing enables precise regulation of gene expression, expanding therapeutic possibilities. Despite these advances, significant challenges remain, including off-target effects, delivery methodologies, immune responses, and long-term genomic safety concerns. Future improvements in editor precision, innovative delivery platforms, and enhanced safety assessments will be essential to fully integrate CRISPR-based interventions into standard clinical practice, significantly advancing personalised medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef2c68c64e6d5ba82e4b21a0830f080c5ac867e1" target='_blank'>
              CRISPR Technology in Disease Management: An Updated Review of Clinical Translation and Therapeutic Potential.
              </a>
            </td>
          <td>
            Bahareh Farasati Far, Marziyeh Akbari, Mohammad Amin Habibi, Morteza Katavand, Sherko Nasseri
          </td>
          <td>2025-07-20</td>
          <td>Cell proliferation</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Purpose: Although the combination of androgen deprivation therapy with docetaxel or abiraterone acetate and prednisone has become a standard treatment for metastatic hormone-sensitive prostate cancer (mHSPC) and has shown improved overall survival, a subset of patients still progress to castration-resistant disease. However, the underlying molecular features in these patients remain poorly understood.Materials and Methods: We performed single-cell RNA sequencing (scRNA-seq) on 12 tissue samples, including 2 mHSPC samples, 7 primary prostate cancer (PCa) samples, and 3 matched normal samples from 7 patients. Raw sequencing data was processed by Cell Ranger software (10X Genomics) and aligned to the human reference genome (GRCh38). A comprehensive analysis of samples from patients with mHSPC was also conducted and validated using a cohort of 52 patients with mHSPC.Results: Our results identified distinct subpopulations within luminal and mononuclear phagocyte (MNP) clusters characterized by proliferative activation associated with unfavorable clinical outcomes. Furthermore, we observed that the MNP cluster exhibited significant proliferation activity. To understand the underlying mechanisms associated with aggressiveness, we conducted cell-cell interaction, copy number variation and pseudotime analysis. Utilizing 87 network genes from scRNA-seq, we classified 52 mHSPC patients into 2 molecular subtypes and demonstrated their correlation with distinct transcriptomic profiles and survival outcomes. Importantly, a 14-gene signature derived from 3 distinct gene sets exhibited a strong association with patient survival and drug response. The prognostic value of our findings was further validated in large-scale cohorts comprising localized PCa, metastatic PCa, mHSPC, and metastatic castration-resistant PCa patients.Conclusion: This study provides valuable insights into the identification of high-risk patients, novel biomarkers, and potential therapeutic targets for individuals with mHSPC. Furthermore, the results in this study can serve as a basis for future investigations aimed at refining prognostic strategies and developing targeted therapies for patients with mHSPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/450db38b74f006113af6f667e3cedb5cf553cec7" target='_blank'>
              Determinant of Aggressive Phenotype in Metastatic Hormone-Sensitive Prostate Cancer Depends on an Intrinsic, Highly Aggressive Cell Cluster: Integrated Single-Cell RNA and Whole Transcriptomic Sequencing Analyses
              </a>
            </td>
          <td>
            Minyong Kang, B. Jee, Jiwoong Yu, Soohyun Hwang, Kyunghee Park, Kyung Yeon Han, Wan Song, H. Sung, H. Jeon, Byong Change Jeong, S. Seo, Se Hoon Park, Woong-Yang Park, S. Jeon
          </td>
          <td>2025-07-31</td>
          <td>Journal of Urologic Oncology</td>
          <td>1</td>
          <td>34</td>
        </tr>

        <tr id="Approximately 2.6% of live births in the United States are conceived using assisted reproductive technologies (ARTs). While some ART procedures, including in vitro fertilization (IVF) and intracytoplasmic sperm injection, are known to alter the epigenetic landscape of early embryonic development, their impact on DNA sequence stability is unclear. Here, we leverage the strengths of the laboratory mouse model system to investigate whether a standard ART regimen—ovarian hyperstimulation, gamete isolation, IVF, embryo culture, and embryo transfer—affects genome stability. Age-matched cohorts of ART-derived and naturally conceived C57BL/6J inbred mice were reared in a controlled setting and whole genome sequenced to ∼50x coverage. Using a rigorous pipeline for de novo single nucleotide variant (dnSNV) discovery, we observe a ∼30% increase in the dnSNV rate in ART-compared to naturally-conceived mice. Analysis of the dnSNV mutation spectrum identified signature contributions related to germline DNA repair activity, affirming expectations and evidencing the quality of our dnSNV calls. We observed no enrichment of dnSNVs in specific genomic contexts, suggesting that the observed rate increase in ART-derived mice is a general genome-wide phenomenon. Similarly, we show that the developmental timing of dnSNVs is similar in ART- and natural-born cohorts. Together, our findings show that ART is moderately mutagenic in house mice and motivate future work to define the precise procedure(s) associated with this increased mutational vulnerability. While we caution that our findings cannot be immediately translated to humans, they nonetheless emphasize a pressing need for investigations on the potential mutagenicity of ART in our species. SIGNIFICANCE STATEMENT This study investigates whether assisted reproductive technologies (ARTs) increase the risk of inherited genetic mutations in offspring. Using a well-controlled mouse model system, we compared the de novo mutation burden in genomes of mice conceived through ART to a naturally conceived cohort. We find a ∼30% increase in new DNA mutations in ART-conceived mice, suggesting that ART procedures have a genome destabilizing effect. This increase in mutation rate appears to be uniform across the genome, rather than attributable to specific genomic contexts. While we caution against the direct translation of our findings to humans, our work nonetheless highlights the need for further research into the genetic safety of ART in people.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bdfbaa7908e3b1759956bbd1d3334471005f5d2" target='_blank'>
              Increased rate of de novo single nucleotide mutation in house mice born through assisted reproduction
              </a>
            </td>
          <td>
            Laura Blanco-Berdugo, Alexis C Garretson, Beth L. Dumont
          </td>
          <td>2025-06-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38d19ed9e3446e483414686738058488478cf912" target='_blank'>
              Transcription Factor MYB Upregulates IQGAP3 to Mediate DNA Repair and Promote 5-FU Resistance in Gastric Cancer Cells.
              </a>
            </td>
          <td>
            Lizhe Huang, Piyao Gao, Pengcheng Xiao, Zhongyang Chen, Sen Zhang
          </td>
          <td>2025-08-01</td>
          <td>Drug development research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Cancer development and progression are driven by the accumulation of somatic genetic alterations, which occur in a specific temporal order. However, how the order of mutations impacts cancer phenotypes of solid tumors remains poorly understood. To address this, we developed a novel computational framework, IMOP-Cancer (Identifying Mutation Order Pairs in Cancer), to identify mutation gene pairs whose order influences cancer phenotypes. We applied IMOP-Cancer to The Cancer Genome Atlas-Lung Adenocarcinoma (TCGA-LUAD) cohort and identified 446 key mutation order pairs, with 34 pairs significantly associated with prognosis. Mutation order impacts cancer phenotypes, as demonstrated by CSMD3 and PTPRD (tumor proliferation) and TP53 and NAV3 (immune modulation), with effects validated in four independent datasets. We further presented the impact of mutation pairs on cancer phenotypes through case studies in the TCGA cohorts of bladder urothelial carcinoma (BLCA), and colon adenocarcinoma (COAD). We extended this analysis to 33 cancer cohorts from TCGA portal, identifying 106 034 critical mutation pairs across 17 cancers, with 3036 pairs co-occurring in multiple cancers. Shared mutation pairs across cancers also showed distinct effects on cancer phenotype. Our study highlights the importance of mutation order in cancer progression and diversity, offering new insights into the temporal dynamics of co-occurring mutations and paving the way for personalized treatment strategies and improved diagnosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f348ae5004fc1076373867ea177acfe998124027" target='_blank'>
              IMOP-Cancer: identifying mutation order pairs impacting cancer phenotypes
              </a>
            </td>
          <td>
            Yijing Zhang, Shaobo Kang, Renjie Dou, Wanmei Zhang, Yuanyuan Liu, Yang Wu, Dongxue Li, Fangfang Fan, Yanyan Ping
          </td>
          <td>2025-07-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background Precursor lesions like polyps and adenomas in the colon commonly precede colorectal cancer (CRC), advancing through the “normal-polyp-adenoma-carcinoma” sequence towards malignancy. Yet, the cellular heterogeneity and molecular mechanisms involved in CRC development remain inadequately characterized. Methods To understand the molecular mechanisms driving the onset and progression of CRC, we conducted a comprehensive analysis of ten clinical colorectal samples representing sequential pathological stages using single-cell RNA sequencing (scRNA-seq). Validation was performed through immunofluorescence and immunohistochemistry analyses in a separate human colorectal tissue cohort. Additional verification was carried out using bioinformatics analyses of public TCGA and GEO datasets. Results Our comprehensive analyses not only reveal the cellular diversity and transcriptomic differences throughout disease progression but also highlight the importance of leveraging ligand–receptor gene expression to distinguish various cell subtypes. Subsequent examination and validation with a larger sample cohort uncover the specific involvement of ligand–receptor genes, transcription factors, immunoglobulin genes, and heat shock genes in regulating immune responses and microenvironment changes during colorectal tumorigenesis. Conclusions Our extensive transcriptome dataset provides valuable insights and acts as a fundamental resource to deepen our understanding of the complex molecular landscape in CRC. This dataset facilitates improved diagnostic accuracy and the creation of more personalized therapeutic approaches. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06785-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70e15cc889bc4d3cf181a1a8c15dbc72bc14a7b3" target='_blank'>
              A dynamic molecular landscape in colorectal cancer progression at single-cell resolution
              </a>
            </td>
          <td>
            Jianwen Sheng, Nianshuang Li, Shuai Li, Yuman Ye, Yuanhang Liao, Yupeng Zhang, Huan Chen, Mengxian Tu, Guping Zhong, Chaoliang Xiong, Pan Zheng, Yuting Lei, Zekun He, Xingxing He, Yao Zhang, Lei Wang, Xiaolin Gao, Yin Zhu, Jianping Liu, Huizhen Fan
          </td>
          <td>2025-07-01</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The epigenetic deregulation of CpG islands (CGIs) plays a crucial role in cancer initiation and progression. CGIs comprise 1-2% of the human genome and are rich in differentially methylated regions (DMRs) that can serve as cancer biomarkers in clinical samples and liquid biopsies. Focusing epigenetic sequencing on CpG-rich sequences, including CGIs and avoiding non-informative regions, offers an efficient and sensitive approach for cancer identification and tracking, especially within samples containing excess of unaltered, normal DNA. To this end, we have developed Adaptor-anchored Methylation amplification via Proximity Primers (aMAPP), a versatile PCR-based enrichment method. aMAPP employs specially designed primers to selectively enrich either methylated or unmethylated CpGs, depending on the upstream methylation conversion method employed. aMAPP achieves high coverage of genome-wide CGIs and detects hundreds of DMRs in tumor samples compared to adjacent normal tissue using ultra-low depth sequencing (∼300,000 reads). It enables tracing of aberrant methylation down to allelic frequency 0.01% in dilutions of tumor DNA and in cell-free DNA samples, can be applied using picogram amounts of DNA, and can be adapted to enrich either small panels of cancer-specific DMRs, or the majority (>90%) of genomic CGIs and CpGs. aMAPP offers a simple, cost-effective, and highly sensitive approach for capturing the epigenetic footprint of genome-wide CpGs and identifying aberrantly methylated or un-methylated genomic regions. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/030972eec1be1d377d8b7bfd84130f3e834f7bbf" target='_blank'>
              Genome-wide extraction of differentially methylated DNA regions using adapter-anchored proximity primers
              </a>
            </td>
          <td>
            Farzaneh Darbeheshti, Hayet Radia Zeggar, Hamzeh Salmani, Yibin Liu, Ruolin Liu, V. Adalsteinsson, G. M. Makrigiorgos
          </td>
          <td>2025-06-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Natural biological agents that can transform normal somatic cells into cancer stem cells, have not been identified. We earlier reported that cell free chromatin particles (cfChPs) that circulate in blood of cancer patients can horizontally transfer themselves to healthy cells to induce dsDNA breaks and inflammation. Here we show that a single cell clone D5 developed from NIH3T3 mouse fibroblast cells treated with cfChPs isolated from sera of cancer patients exhibited upregulation of stemness related transcription factors OCT4, SOX2 and KLF4 and surface markers CD34, CD44 and CD133, and the ability to form spheroids in appropriate culture medium. Transcriptome analysis revealed upregulation of the stemness-associated gene Kruppel-like factor 4 (Klf4), a key Yamanaka factor for the induction of pluripotent stem cells. Additionally, several cancer-related pathways involved in invasion, metastasis, and stemness were found to be upregulated. These pathways included those related to ERBB signaling, P53, cellular senescence, ATM, CTCF and apoptosis. Subcutaneous inoculation into SCID mice resulted in development of malignant tumours which expressed the endoderm marker alpha fetoprotein confirming their stem cell property. Our results suggest the cfChPs that circulate in blood of cancer patients can transform susceptible somatic cells into cancer stem cells with the potential to promote metastasis. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-09686-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5905889dedfeab60767171035c4266975f0de7d2" target='_blank'>
              Induction of cancer stem cells from mouse fibroblasts
              </a>
            </td>
          <td>
            S. Shabrish, Leimarembi Devi Naorem, Shubham Mohanty, Rohit Kumar, Relestina Lopes, Nabila Akhter, Naveen Kumar Khare, Ruchi Joshi, Rajashree Sawant, G. V. Raghuram, Vishal Jadhav, Sushma Shinde, Pratik Chandrani, Indraneel Mittra
          </td>
          <td>2025-07-15</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Introduction Treatment resistance is a major hurdle in diffuse large B-cell lymphoma (DLBCL) therapy. Here, we assessed the relevance of the Musashi (MSI) RNA-binding protein family for DLBCL treatment efficacy. As important gene expression regulators, these proteins have previously been associated with tumorigenesis, treatment failure, and reduced survival in other malignancies, making them promising candidates for assessment in the context of DLBCL outcome and therapy resistance. Methods We first leveraged publicly available gene expression studies to determine expression and prognostic relevance of MSI1 and MSI2 in DLBCL. We then characterized MSI2 co-expressed therapy-relevant signaling. After performing MSI2 knockdown experiments we investigated subsequent effects on DLBCL gene expression in vitro using qPCR, Western blot, protein arrays, and flow cytometry. Finally, cell viability assays and clonogenic assessments were used to assess resistance to radiation, vincristine, and doxorubicin chemotherapy. Results MSI2 was overexpressed and prognostically unfavorable in univariable and multivariable analyses in DLBCL while MSI1 showed very low expression. High MSI2 expression was associated with increased stemness and DNA repair signaling. MSI2 knockdown led to a loss of stemness-associated markers and compromised DNA repair protein activation while increasing radiation-induced DNA double-strand break levels. Cell survival after either radiotherapy, vincristine or doxorubicin chemotherapy was impaired after MSI2 knockdown in follow-up analyses, suggesting a radio- and chemosensitizing effect. Discussion We propose that MSI2, a prognostic marker, may modulate the susceptibility of DLBCL towards genotoxic therapy. Suppressing MSI2 may hold promise to sensitize DLBCL to DNA-targeted treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a873df352503c3224c8f6ec89d594d3543c9ab8" target='_blank'>
              Prognostic marker Musashi-2 modulates DNA damage response and radioresistance in diffuse large B-cell lymphoma
              </a>
            </td>
          <td>
            Timo Habig, Lasse Reichstein, Kathrin A. Brücksken, M. Sicking, Jan Labisch, M. Oertel, Eberhard Korsching, Georg Lenz, Stephan Hailfinger, Burkhard Greve, Fabian M Troschel, H. Eich
          </td>
          <td>2025-08-06</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The field of hematology has experienced a substantial evolution with the acknowledgment of epigenetic processes as essential factors in the development of hematological malignancies. This review article examines the influence of epigenetic alterations, namely DNA methylation and histone modifications, on the onset and advancement of conditions such as acute myeloid leukemia and myelodysplastic syndromes. We discuss how these epigenetic modifications lead to the deregulation of gene expression, eventually promoting leukemogenesis. The emergence of epigenetic therapies, such as DNA methyltransferase inhibitors (e.g., azacitidine and decitabine), histone deacetylase inhibitors (e.g., vorinostat and romidepsin), and enhancer of zeste homologue 2 inhibitors (e.g., tazmetostat), demonstrates the potential to reverse aberrant epigenetic modifications and restoring normal cellular functions. Moreover, we highlight innovative therapeutic approaches, including combination therapies and CRISPR-based epigenetic editing tools, which are influencing the future of treatment for hematological malignancies. Despite promising results, challenges such as off-target effects, drug resistance, and the need for personalized approaches remain significant barriers to effective treatment. We emphasize that further study is required to improve delivery systems, comprehend resistance mechanisms and develop precision medicine strategies that can tailor therapies to individual patient profiles. By integrating benchside discoveries with clinical applications, this review aims to illuminate the transformative potential of epigenetic therapies in improving patient outcomes in hematology. Graphical abstract Epigenetic dysregulation including aberrant DNA methylation, histone modifications, and chromatin remodeling is a key driver in the initiation and progression of hematologic malignancies. Therapeutic strategies targeting these epigenetic mechanisms aim to restore normal gene expression, induce differentiation, and overcome drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cfd8b658a221de404276462dcd055a301f17f27" target='_blank'>
              The epigenetic revolution in hematology: from benchside breakthroughs to clinical transformations
              </a>
            </td>
          <td>
            Mahdis Abdar Esfahani, Nazli Servatian, Ali Jihad Hemid Al-Athari, Elaf Salah Mehdi Khafaja, Hamideh Rahmani Seraji, Hamed Soleimani Samarkhazan
          </td>
          <td>2025-07-01</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Therapeutic resistance remains a primary obstacle to curing cancer. Healthy cells exposed to genotoxic insult rapidly activate both p53-dependent and -independent non-genetic programs that pause the cell cycle and direct either DNA repair or apoptosis. Cancer cells exploit these same pathways as they respond to stresses induced by cancer therapies. In this study, we investigated a potential role for upstream stimulatory factor 1 (USF1) and USF2 in the p53-independent response of lymphoma cells to genotoxic therapy. We previously found that lymphocytes utilize the responsiveness of USF1 to double-stranded DNA breaks to coordinate T cell receptor beta (Tcrb) gene expression during V(D)J recombination. Here, microarray gene expression analysis of derivatives of the p53-deficient mouse B lymphoma cell line, M12, revealed that simultaneously depleting cells of both USF1 and USF2 altered the expression of 940 gene transcripts (>1.50-fold change, < 0.05 FDR), relative to cells expressing a scrambled control shRNA. Seven days after exposure to a single sublethal (5 Gy) dose of ionizing radiation, USF-depleted (USFKD) cells exhibited widespread and distinct transcriptional responses from those of irradiated controls (5035 and 5054 differentially expressed gene transcripts, respectively, with roughly half shared between both cell types). Gene ontology analyses revealed that USF knockdown induced numerous changes in the expression of genes critical for immune development and function while diminishing the responsiveness of genes linked to DNA damage pathways. Microarray findings were confirmed by RT-qPCR for a panel of genes responsive to USF knockdown and/or irradiation. These findings shed further light on transcriptional responses to ionizing radiation that manifest over time in transformed cells, identifying a novel p53-independent role in lymphocytic DNA damage stress responses for USF.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6bb6187205d63162ff7fe5e73d3f94fd90940ba" target='_blank'>
              Knockdown of USF1 and USF2 drives prolonged changes in the gene expression response of M12-5B3 cells to DNA damage
              </a>
            </td>
          <td>
            Kimberly S Bellingham-Johnstun, Lisa A Metzger, Jennifer L Stone, Kevin Gerrish, Pierre R Bushel, Michael L Sikes
          </td>
          <td>2025-07-14</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Breast cancer remains one of the most prevalent malignancies worldwide, and radiation therapy is a central component of its management. However, intrinsic or acquired resistance to radiation significantly compromises therapeutic efficacy. This systematic review aimed to identify and evaluate molecular mechanisms and interventions that influence radiation sensitivity in breast cancer models. A comprehensive PubMed search was conducted using the terms “breast cancer” and “radiation resistance” for studies published between 2002 and 2024. Seventy-nine eligible studies were included. The most frequently investigated mechanisms included the dysregulation of the PI3K/AKT/mTOR and MAPK signaling pathways, enhanced DNA damage repair via non-homologous end joining (NHEJ), and the overexpression of cancer stem cell markers such as CD44+/CD24−/low and ALDH1. Several studies highlighted the role of non-coding RNAs, particularly the lncRNA DUXAP8 and microRNAs such as miR-21, miR-144, miR-33a, and miR-634, in modulating radiation response. Components of the tumor microenvironment, including cancer-associated fibroblasts and immune regulators, also contributed to radiation resistance. By synthesizing current evidence, this review provides a consolidated resource to guide future mechanistic studies and therapeutic development. This review highlights promising molecular targets and emerging strategies to enhance radiosensitivity and offers a foundation for translational research aimed at improving outcomes in radiation-refractory breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a4cbc0cd4ca4c1e106a823d69b14920681deda3" target='_blank'>
              Molecular Mechanisms of Radiation Resistance in Breast Cancer: A Systematic Review of Radiosensitization Strategies
              </a>
            </td>
          <td>
            Emma Mageau, Ronan Derbowka, Noah Dickinson, Natalie Lefort, A. T. Kovala, D. Boreham, T. C. Tai, Christopher Thome, S. Tharmalingam
          </td>
          <td>2025-07-24</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="X chromosome inactivation (XCI) is crucial for balancing X-linked gene dosage in female cells by randomly silencing one X chromosome during early embryogenesis. However, accurately classifying cells based on the parental origin of the inactivated X chromosome in single-cell samples remains challenging. Here we present FemXpress, a computational tool leveraging X-linked single nucleotide polymorphisms (SNPs) to group cells based on the origin of the inactivated X chromosome in female single-cell RNA sequencing (scRNA-Seq) data. FemXpress performs robustly on both simulated and real datasets, without requiring parental genomic information, and can also identify genes that escape XCI. Applying FemXpress to single-cell RNA-Seq data from multiple tissues of a cynomolgus monkey, we reveal heterogeneity in XCI origin across organs and cell types. In each organ, we identify candidate XCI-escaping genes, and within each cell type, we observe gene expression differences associated with XCI origin, potentially contributing to phenotypic variability. Furthermore, FemXpress demonstrated strong performance in phasing XCI in scRNA-Seq datasets from embryos and colon tumors. In summary, FemXpress provides a powerful approach for XCI status analysis, offering new insights into XCI dynamics at single-cell resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/478a669d81b3cf05029932ee293479ee1f9bcebc" target='_blank'>
              FemXpress: Systematic Analysis of X Chromosome Inactivation Heterogeneity in Female Single-Cell RNA-Seq Samples.
              </a>
            </td>
          <td>
            Xin Wang, Yingke Ma, Fan Li, Wentao Cui, Tianshi Pan, Siqi Wang, Sinan Ma, Qingtong Shan, Chao Liu, Yukai Wang, Ying Zhang, Yuanchun Zhou, Wei Li, Pengfei Wang, Qi Zhou, Guihai Feng
          </td>
          <td>2025-06-29</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b37ee620d5697800e8d511cd10bb157702e6caa" target='_blank'>
              Clinical and analytical validation of MI Cancer Seek®, a companion diagnostic whole exome and whole transcriptome sequencing-based comprehensive molecular profiling assay.
              </a>
            </td>
          <td>
            V. Domenyuk, Kasey Benson, Peggy Carter, Dan Magee, Jian Zhang, Nitin Bhardwaj, Hongseok Tae, James Wacker, Foram Rathi, Siobhan Miick, Aastha Kohli, Joshua Carroll, Lori Cuyugan, Evelyn Perez, Wayeesha Zhang, John Collins, Patrick Kennedy, Jeremy Ellis, A. Stark, Andrey Loskutov, Brittany Cuttone, Blake Taylor, Rebecca A Feldman, Jeff Swenson, David Bryant, R. Hahn-Lowry, Raunaq Kaushal, Jennifer R. Ribeiro, J. Abraham, M. Radovich, George W Sledge, M. Oberley, D. Spetzler
          </td>
          <td>2025-08-13</td>
          <td>Oncotarget</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Invasive Lobular Carcinoma (ILC), a distinct subtype of breast cancer, is hallmarked by E-Cadherin loss, slow proliferation, and strong hormone receptor positivity. ILC faces significant challenges in clinical management due to advanced stage at diagnosis, late recurrence, and development of resistance to endocrine therapy - a cornerstone of ILC treatment. To elucidate the mechanisms underlying endocrine resistance in ILC, ILC cell lines (MDA-MB-134-VI, SUM44PE) were generated to be resistant to tamoxifen, a selective estrogen receptor modulator. The tamoxifen-resistant (TAMR) cells exhibit a 2-fold increase in tamoxifen IC50 relative to parental cells. Metabolomics and RNA-sequencing revealed deregulation of alanine, aspartate, and glutamate metabolism, purine metabolism, and arginine and proline metabolism in TAMR cells. Among the fifteen commonly dysregulated genes in these pathways, low argininosuccinate synthase (ASS1) expression was identified in the TAMR cells and was significantly correlated with poor outcome in ILC patients, specifically in the context of endocrine therapy. Our study reveals methylation-mediated silencing of ASS1 in TAMR cells as a likely mechanism of downregulation. Demethylation restored ASS1 expression and correspondingly reduced tamoxifen IC50 toward parental levels. Nucleic acid biosynthesis is augmented in TAMR cells, evidenced by an increase in nucleotide intermediates. Both TAMR cell lines demonstrated increased expression of several nucleic acid biosynthesis enzymes, including PAICS, PRPS1, ADSS2, CAD, and DHODH. Furthermore, CAD, the key multifunctional protein of the de novo pyrimidine biosynthesis pathway, is differentially activated in TAMR cells. Treating TAMR cells with Decitabine, a demethylating agent, or Farudodstat, a pyrimidine biosynthesis inhibitor, markedly augmented the efficacy of tamoxifen. Collectively, our study unveils ASS1 downregulation as a novel mechanism underlying acquired tamoxifen resistance in ILC and establishes a metabolic link between ASS1 and nucleic acid biosynthesis. Restoring ASS1 expression or inhibiting pyrimidine biosynthesis reinstated tamoxifen sensitivity. ASS1 could be a potential biomarker and therapeutic target in tamoxifen-resistant ILC patients, warranting further investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cabd502be0bff86bc6cbc40b147417307dfd4285" target='_blank'>
              Invasive lobular carcinoma: integrated multi-omics analysis reveals silencing of Argininosuccinate synthase and upregulation of nucleotide biosynthesis in tamoxifen resistance
              </a>
            </td>
          <td>
            Annapurna Gupta, Fouad Choueiry, Jesse Reardon, Nikhil Pramod, Anagh Kulkarni, Eswar Shankar, S. Sizemore, Daniel G. Stover, J. Zhu, B. Ramaswamy, S. Majumder
          </td>
          <td>2025-07-11</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the most common cancers, with an incidence that has increased significantly over the last 20 years. The World Health Organization (WHO), under its cancer program, identifies CRC as the third most prevalent cancer worldwide with a high mortality rate, especially in patients under the age of 50. Despite advances in surgery, chemotherapy, radiotherapy, and molecular targeted therapy, CRC shares a low survival rate and poor prognosis due to late diagnosis. To address these challenges, research into alternative diagnostic and therapeutic strategies has increasingly focused on molecular mechanisms, including noncoding RNAs (ncRNAs). Circular RNAs (circRNAs), a subclass of endogenous ncRNAs characterized by their covalently enclosed loop structure, demonstrate greater stability than linear RNAs, making them potential candidates for clinical applications. The circRNAs possess differential expressions in cancers and function as tumor suppressive or oncogenic activities. This review discusses the recent findings on circRNAs and their potential for biomarkers and therapeutic targets in CRC. As circRNAs can serve as miRNA sponges, increase or decrease mRNA expression, and function to regulate an oncogenic or tumor suppressor pathway, there may be the possibility for an early diagnosis, prognosis, or therapeutic role of circRNAs in CRC. Highlighting the clinical implications of circRNAs, this review discusses their potential to transform current CRC management strategies and address critical gaps in timely diagnosis and effective treatment. It also emphasized the need for further clinical research to validate their utility and translate these findings into practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db80f20f575f8c248f315abb0f46ec8b7f78d57b" target='_blank'>
              A review of circular RNAs in colorectal cancer: insights into biomarker discovery and therapeutic targeting
              </a>
            </td>
          <td>
            Sibin Nambidi, Karunakaran Bhuvaneswari Shruthi, Sujatha Padmanaban Kanimozhe, Antara Banerjee, S. Kondaveeti, A. Duttaroy, S. Pathak
          </td>
          <td>2025-07-04</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69596b9ce15efb640dc5ee8ae2fb7a482a3b87c4" target='_blank'>
              ALKBH5 suppresses m6A mRNA modification of FOXM1 to drive Cetuximab resistance in KRAS-mutant colorectal cancer.
              </a>
            </td>
          <td>
            Hu Han, Yan Li, Zhiyi Lin, Xiaoping Ma, Wukui Huang, Cengceng Lu, Rongyan Ma, Rui Han
          </td>
          <td>2025-07-07</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Lung cancer remains one of the leading causes of death associated with cancer globally, with non-small cell lung cancer (NSCLC) accounting for 80–85% of all lung cancer cases. Despite its high prevalence, the underlying mechanisms of NSCLC have not been completely clarified, and current therapeutic strategies face significant limitations. Recent research has revealed the important role of long non-coding RNAs (lncRNAs) in NSCLC, particularly in regulating processes such as autophagy and drug resistance. LncRNAs are a class of non-coding RNA molecules, typically with transcript lengths exceeding 200 nucleotides, and have been the subject of extensive investigation in recent years. Their involvement in critical cellular processes has opened up new research avenues for precision medicine in NSCLC. This review aims to offer a comprehensive analysis of the mechanisms by which lncRNAs regulate autophagy and drug resistance in NSCLC, explore their potential clinical applications as diagnostic biomarkers and therapeutic targets, and provide both theoretical foundations and practical guidance to advance precision medicine in this area. By deepening our understanding of the role of lncRNAs in NSCLC, this article also highlights the promising potential of lncRNA-based approaches for the diagnosis and treatment of this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e40e815b59b56c1f5022ef84ddee5fd9bccc122" target='_blank'>
              The Regulatory Role of LncRNAs in Modulating Autophagy and Drug Resistance in Non-Small-Cell Lung Cancer: Focus on Targeted Therapeutic Approaches
              </a>
            </td>
          <td>
            Shuncai Dai, Yuxin Zhong, Jianfu Lu, Linjiang Song
          </td>
          <td>2025-07-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Neuroblastoma (NB) is the most common extracranial solid tumor in early childhood and frequently presents with bone marrow (BM) metastasis, particularly in high-risk cases. Metastatic NB cells residing in the BM exhibit distinct biological behaviors and are closely associated with treatment resistance and poor prognosis. Emerging evidence suggests that metabolic reprogramming is a hallmark of NB progression; however, its regulatory landscape within the bone marrow microenvironment remains poorly understood. This study aimed to systematically elucidate the molecular mechanisms underlying metabolic reprogramming in bone marrow–infiltrating metastatic NB cells by integrating single-cell transcriptomic data, bulk RNA-sequencing profiles, and in vitro functional validation. Single-cell RNA-sequencing data from 17 bone marrow aspirates of NB patients—with and without marrow infiltration—were retrieved from the GEO database, while bulk RNA-seq data from 155 NB tumor samples were obtained from the TARGET database. A suite of bioinformatics tools, including Seurat, Harmony, CellChat, and SCENIC, was employed to perform multi-dimensional analyses such as cell subtype annotation, intercellular communication mapping, and transcription factor regulon construction. AUCell scoring, differential gene expression analysis, random survival forest modeling, and Kaplan–Meier survival analysis were conducted to identify key genes involved in metabolic reprogramming and to characterize their immune infiltration features. Gene Set Enrichment Analysis (GSEA), Gene Set Variation Analysis (GSVA), and single-cell pathway scoring were applied to predict the metabolic and immune pathways associated with these candidate genes. Furthermore, co-expression network analysis was used to evaluate the relationships between candidate genes and known NB regulatory factors. Finally, in vitro experiments were carried out to validate the roles of selected genes in regulating NB cell proliferation, migration, invasion, and mitochondrial function. Single-cell transcriptomic analysis of bone marrow–derived NB samples identified nine distinct cell subpopulations with diverse intercellular communication networks. Among these, the ligand–receptor pair MDK–NCL emerged as a key mediator of cell–cell signaling. Regulatory network analysis revealed five critical regulons—JUND, JUNB, FOS, E2F1, and KLF16—closely associated with metabolic reprogramming in NB. The “Neuroblastoma cell” cluster displayed markedly elevated metabolic activity. Through integrated analyses, five core metabolic reprogramming genes (MRPL21, NHP2, RPL13, RPL18A, and RPL27A) were identified and shown to be significantly associated with poor prognosis. High expression of these genes correlated with an immunosuppressive tumor microenvironment, characterized by reduced infiltration of monocytes, M1 macrophages, and T cells. Functional enrichment analysis revealed that these genes were primarily enriched in oxidative phosphorylation, MYC targets, PI3K-Akt, and p53 signaling pathways. Co-expression network analysis further demonstrated that MRPL21 and NHP2 positively correlated with known NB regulatory genes TP53, NRAS, and NKIF1B. In vitro assays confirmed that MRPL21 knockdown significantly impaired NB cell proliferation, migration, invasion, and mitochondrial oxidative phosphorylation. This study identified five key genes involved in metabolic reprogramming of bone marrow–infiltrating NB cells, which are closely associated with immunosuppressive microenvironment formation and enrichment in tumor-associated metabolic pathways. Among these, MRPL21 plays a pivotal role in regulating NB cell proliferation and mitochondrial function, underscoring its potential as a promising therapeutic target. These findings suggest that integrated strategies targeting both tumor metabolism and the immune microenvironment may provide new avenues for the treatment of high-risk, metastatic NB. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-13626-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b395447d21c8867ac393546695c70f8056f2bca" target='_blank'>
              Integrated single-cell and transcriptomic analysis of bone marrow-derived metastatic neuroblastoma reveals molecular mechanisms of metabolic reprogramming
              </a>
            </td>
          <td>
            Jing Chu, Rong Qin, Shu-Jing Wang, Qiang Wang, Qiang Wu
          </td>
          <td>2025-08-05</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f50cadaedcfebf721bc65589160da6901136b146" target='_blank'>
              TRIM24 as a therapeutic target in endocrine treatment-resistant breast cancer.
              </a>
            </td>
          <td>
            Nuno Padrão, Sebastian Gregoricchio, Nils Eickhoff, Jing Dong, L. Luzietti, D. Bossi, T. Severson, Joseph Siefert, Arianna Calcinotto, L. Buluwela, Maria Donaldson Collier, Simak Ali, Leonie S Young, J-P. Theurillat, D. Varešlija, W. Zwart
          </td>
          <td>2025-08-19</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="R-loop is a special DNA–RNA hybrid genomic structure. Since its identification, the molecular mechanisms of physiological processes such as class switch recombination have been revealed, uncontrolled regulation of which has become the underlying cause of many diseases. With the development of molecular biology techniques, R-loops found at different sites and formed via different mechanisms have been discovered. These non-classical R-loops participate in various cellular activities via different mechanisms. A set of sophisticated regulatory mechanisms are required to control the number of various R-loops in vivo. The levels of non-classical R-loops in tumor cells differ significantly from those in normal cells; in addition, the regulatory mechanisms for establishing R-loop homeostasis differ, which may be utilized to develop breakthrough therapies for various tumors. In this review, we summarize the current state of knowledge regarding non-classical R-loops in tumor cells, the mechanisms that promote or inhibit this structure, the effects of this structure on tumor cells, and the possible therapeutic targets. By systematically elucidating the pathogenic mechanisms of atypical R-loops, we can achieve precision targeting of tumors and revolutionize clinical precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/676d0195748900abca38448fb2ef5985171fe4bf" target='_blank'>
              Atypical R-loops in cancer: decoding molecular chaos for therapeutic gain
              </a>
            </td>
          <td>
            Yuan Sun, Sheng Wang, N. Ge, Jintao Guo, Guoxin Wang, Fan Yang, Siyu Sun
          </td>
          <td>2025-08-14</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/290d92451b26f978216d79d0c80c8e4f6746115f" target='_blank'>
              LaGrACE: estimating gene program dysregulation with latent regulatory network.
              </a>
            </td>
          <td>
            Minxue Jia, Haiyi Mao, Mengli Zhou, Yu-Chih Chen, P.V. Benos
          </td>
          <td>2025-06-30</td>
          <td>Molecular systems biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="ABSTRACT Prenatal genetic diagnostics have undergone a remarkable transformation, progressing from early cytogenetic techniques such as karyotyping and fluorescence in situ hybridization (FISH) to chromosomal microarray analysis (CMA) and, most recently, whole exome sequencing (WES). WES has emerged as a groundbreaking tool, allowing for identifying single-gene mutations, small insertions and deletions, and other pathogenic variants responsible for rare and complex diseases. Unlike conventional approaches, which primarily detect large chromosomal abnormalities, WES provides a high-resolution analysis of the fetal genome, significantly improving diagnostic accuracy and enabling early intervention. This review explores the historical evolution of prenatal genetic testing, highlighting key milestones from the introduction of cytogenetics in the 1960s to the integration of WES in clinical practice over the last decade. WES has proven instrumental in diagnosing monogenic disorders, uncovering the genetic basis of fetal anomalies, and investigating cases of stillbirth and recurrent pregnancy loss (RPL). However, despite its immense clinical utility, challenges such as the interpretation of variants of uncertain significance (VUS), ethical concerns surrounding incidental findings, and the financial burden associated with sequencing continue to impact its widespread adoption. Future directions in WES include its potential integration with non-invasive prenatal testing (NIPT), advancements in artificial intelligence (AI)-driven bioinformatics, and its role in precision medicine, offering more personalized and data-driven approaches to prenatal care. As technological innovations continue to enhance the speed, accuracy, and affordability of WES, its role as a cornerstone of modern prenatal diagnostics is expected to expand, shaping the future of fetal genetic screening and clinical decision-making.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bea8aa269d1f1297db08909adc12f64eda53cf6" target='_blank'>
              The evolution of prenatal Whole Exome Sequencing: from cytogenetics to precision medicine
              </a>
            </td>
          <td>
            I. Săbău, L. Bohiltea, V. Varlas, Adelina Silvana Gheorghe, Maria Riza, N. Suciu, I. Ceaușu
          </td>
          <td>2025-05-20</td>
          <td>Archive of Clinical Cases</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Forward genetic screening is a powerful approach to assign functions to genes and can be used to elucidate the many genes whose functions remain unknown. A key step in forward genetic screening is mapping: identification of the gene causing the phenotype. Existing mapping methods use a bioinformatic mapping-by-sequencing approach based on allelic frequency calculations that often identify large genomic regions which contain an intractable number of candidate genes for testing. Here, we describe WheresWalker, a modern mapping-by-sequencing algorithm that identifies a mutation-containing interval and then supports positional cloning to shrink the interval, which drastically reduces the number of potential candidates, allowing for extremely rapid mutation identification. We validated this method using mutants from a forward genetic mutagenesis screen in zebrafish for modifiers of ApoB-lipoprotein metabolism. WheresWalker correctly mapped and identified novel zebrafish mutations in mttp, apobb.1, and mia2 genes, as well as a previously published mutation in maize. Further, we used WheresWalker to identify a previously unappreciated ApoB-lipoprotein metabolism-modifying locus, slc3a2a.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da4d641d98d989cb22290cc4efc3f054e07645d6" target='_blank'>
              Phenotype to genotype: A new and rapid approach using whole-genome sequencing
              </a>
            </td>
          <td>
            McKenna Feltes, A. Zimin, Sofia Angel, Nainika Pansari, Monica R. Hensley, Jennifer L. Anderson, Meng-Chieh Shen, M. Klemek, Yi Shen, Vighnesh S. Ginde, Hannah Kozan, Nhan V. Le, Vivian P. Truong, Meredith H. Wilson, Steven L Salzberg, Steven A. Farber
          </td>
          <td>2025-07-01</td>
          <td>PLOS Genetics</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Background
Copy number variations (CNVs) play an important role in human genetic disorders. Detection of exon-level CNVs is crucial for accurate clinical diagnosis. The CytoScan XON Array, a high-resolution microarray, was recently developed to detect exonic CNVs of various genes.


Methods
We evaluated the clinical performance of the CytoScan XON Array using 59 patient samples with previously identified CNVs, confirmed via methods including multiple ligation-dependent probe amplification (MLPA), gene-dose PCR, and mRNA assay. Concordance between CytoScan XON and orthogonal methods was evaluated in target regions, and diagnostic utility was compared with that of genome sequencing (GS)-based CNV calling tools through analysis of false-positive CNVs in non-target genomic regions.


Results
For target regions, the CytoScan XON Array achieved concordance rates of 89.8% and 92.5% at the exon and gene levels, respectively, for all CNV calls. Concordance was higher for multi-exon CNVs (100%) than that for single-exon CNVs (82.6%, P =0.03). For non-target regions, false-positive CNV calls were reduced to fewer than 0.01 per gene per person through filtering strategies. The array exhibited false-positive detection rates within dosage-sensitive genes comparable with those of GS-based tools.


Conclusions
The CytoScan XON Array, a reliable tool for detecting exon-level CNVs in target regions, can serve as a complementary approach to GS-based CNV calling tools for genome-wide CNV screening with high resolution. However, its performance for single-exon CNVs requires further optimization. Cross-validation with GS-based CNV calling tools is recommended to improve diagnostic accuracy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64a08a61a6f8683a7068bd9bbb76ec77849b8894" target='_blank'>
              High-resolution Chromosomal Microarray with Diagnostic Potential for Detecting Exon-level Copy Number Variations Using Targeted and Non-targeted Approaches.
              </a>
            </td>
          <td>
            Yeseul Kim, Jee-Soo Lee, Boram Kim, M. Kim, Sung Im Cho, Seung-Won Chae, H. Shin, Hoyeon Lee, Ji Yeon Kim, Moon-Woo Seong
          </td>
          <td>2025-07-30</td>
          <td>Annals of laboratory medicine</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Glioblastoma (GB) is the most common and aggressive brain derived tumor. It often shows genetic alterations in kinase signaling pathways, such as the Pi3K/mTOR and RAS/MAPK pathways, that frequently converge onto oncogenic processes. However, it is unknown to what extend co-vulnerabilities exist within this network and which kinase drug targets are promising for GB treatment. We investigated the drug sensitivity of GB cell line models to monotherapy and synergy effects in dual combination therapy to targeting components of Pi3K/mTOR and RAS/MAPK pathways. In addition, we examined cell line drug sensitivities in relation to their individual genetic tumor driving lesions (i.e. NF1 alterations as well as transcriptomic defined GB subtypes). Synergy levels were correlated to in-lab generated phospoproteomic data. Lastly, serial or simultaneous addition of MEK and mTOR inhibitors were investigated in longitudinal experiments. Dual inhibition of MEK and mTOR resulted in synergistic effects, which associated with NF1-deficiency. Strong synergy effects was also associated with the mesenchymal subtype. Dual inhibition of MEK and mTOR led to prolonged growth inhibition in GB spheroids. In addition, sequential drug treatment resulted in similar growth inhibitory effects compared to simultaneous combination therapies. Our findings highlight the potential of dual inhibition strategies targeting multiple kinases for the treatment of GB, particularly in NF1-deficient and mesenchymal tumors, the most lethal subtype of GB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/049087437ba8fac6737126641a3cf7ff0b99a6f7" target='_blank'>
              Combined inactivation of MEK and mTOR can lead to synergistic cell death in glioblastoma models and associates with NF1-deficiency and a mesenchymal subtype.
              </a>
            </td>
          <td>
            F. Cornelissen, Yoran Broersma, R. Narayan, Rogier Dik, S. Piersma, Richard de Goeij-de Haas, T. Pham, D. Noske, W. Vandertop, Connie R. Jimenez, Bart A Westerman
          </td>
          <td>2025-08-01</td>
          <td>Molecular cancer therapeutics</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Mutations in tumor protein p53 (TP53) and ataxia telangiectasia mutated (ATM) genes are frequently observed across various solid and hematologic malignancies and are associated with genomic instability, treatment resistance, and poor clinical outcomes. These alterations compromise the G1/S cell cycle checkpoint and increase cellular dependence on compensatory DNA damage response (DDR) pathways, including the ataxia telangiectasia and Rad3-related protein kinase (ATR)–checkpoint kinase 1 (CHK1)–WEE1 G2 checkpoint kinase (WEE1) axis. This has led to the development of DDR-targeted therapies that exploit synthetic lethality in tumors with TP53 or ATM dysfunction. Inhibitors targeting ATM, ATR, CHK1, and WEE1 have all shown encouraging activity in early-phase clinical trials, particularly in biomarker-enriched subgroups. Poly(ADP-ribose) polymerase (PARP) inhibitors—originally approved for BRCA1/2-mutated breast cancers—are now being evaluated in TP53- or ATM-deficient tumors, often in combination with other DDR-targeting agents to enhance efficacy. Clinical trials increasingly support the efficacy of DDR inhibitors in biomarker-defined DDR-deficient tumors, specifically beyond BRCA mutations. This review summarizes current understanding of DDR-targeted strategies in TP53- and ATM-mutant cancers, with an emphasis on relevant clinical data and ongoing trials. Expanding the clinical use of DDR inhibitors based on molecular profiles may provide new therapeutic options for genomically unstable tumors across adult and pediatric populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51c9cb81a9e0dfbd38939e93feb0489096524fb3" target='_blank'>
              Therapeutic Targeting of DNA Damage Response Pathways in TP53- and ATM-Mutated Tumors
              </a>
            </td>
          <td>
            Y. Shim
          </td>
          <td>2025-07-01</td>
          <td>Brain Tumor Research and Treatment</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Although immune checkpoint inhibitors (ICIs) hold promise for those diagnosed with advanced human lung adenocarcinoma (LUAD), notable heterogeneity in patient responses and the complex tumor microenvironment (TME) limits their clinical utility while providing clinical benefit. To identify new therapeutic targets to pursue in chimeric antigen receptor (CAR) T-cell therapy, we leveraged an integrative multi-omic approach. This study identified three oncogenic drivers, karyopherin α2 (KPNA2), golgi membrane protein 1 (GOLM1) and thymidine kinase 1 (TK1), that are overexpressed in LUAD, spatially enriched in malignant niches and associated with significantly reduced overall survival (log-rank P < 0.01). Furthermore, functional interrogation using RNAi-mediated gene silencing in LUAD cell lines (NCI-H1650, A549), demonstrated their essential roles in mediating protumorigenic pathways. Specifically, CRISPR interference of KPNA2 or TK1 inhibited cellular proliferation and invasion while also phenocopying a pro-apoptotic effect with cisplatin. GOLM1 depletion inhibited PD-L1 upregulation and restored CD8 + T-cell cytotoxicity in co-culture. Mechanistically, dual knockdown targets, KPNA2 and TK1, were shown to restore deregulated STAT3 signaling that promotes cell survival and also enhance MHC class I antigen presentation implicating functional roles for KPNA2 and TK1 in linked processes of intrinsic oncogenesis and immunoediting by immune evasion and suppression. In summary, KPNA2, GOLM1, and TK1 are functionally relevant molecular coordinators for tumor cell autonomous proliferation and TME mediated immunosuppression. Their membrane expression (notable in the case of PD-L1) and functional roles in immune modulation, make them reasonable targets of CAR T-cell immunotherapy. We posit that a combined strategy targeting KPNA2, GOLM1 and TK1 with or without PD-1/PD-L1 axis inhibitors could provide a better strategy to treat patients with ICI resistant LUAD and generate prolonged and stable immune remodeling. Supplementary Information The online version contains supplementary material available at 10.1186/s40001-025-02955-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbb248ed2c42e77c5eb8fc8c37967d8c58decf41" target='_blank'>
              Coordinating oncogenesis and immune evasion: KPNA2, GOLM1, and TK1 as novel CAR T-cell targets in lung adenocarcinoma
              </a>
            </td>
          <td>
            Wangrui Liu, Hui Zou, Lei Guo, Zhonghua Zhou, Yahui Xie, Huaqi Guo, Gang Wei, Kai Zhang, Hui Yin, Shiyin Wei, Jiachang Chi
          </td>
          <td>2025-08-19</td>
          <td>European Journal of Medical Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The DNA repair protein RAD18 activates ‘Y-family’ Trans-Lesion Synthesis (TLS) DNA polymerases that are DNA damage-tolerant and potentially error-prone. RAD18 is also frequently overexpressed and pathologically activated in cancer cells. However, the extent to which RAD18 shapes cancer genomes and impacts tumorigenesis is unclear. Therefore, we tested the effect of Rad18 status on chemically-induced and oncogene-driven tumorigenesis. In a chemically-induced oral carcinogenesis model, acute (2-16 days) 4NQO-treatment induces expression of Rad18 and TLS polymerase mRNAs in mouse oral epithelial cells prior to emergence of oral squamous cell carcinomas (OSCCs). Chronic (8 week) 4NQO-treatment leads to onset of oral tumors that is accelerated in Rad18-/- mice when compared with Rad18+/+ animals. Analysis of OSCC genomes reveals increased levels of G(C)>T(A) transversions in Rad18-/- tumors when compared with Rad18+/+. Therefore, Rad18 promotes error-free bypass of 4NQO-induced DNA lesions and suppresses 4NQO-induced oral carcinogenesis. In a KrasG12D-induced lung carcinogenesis model, Rad18-deficiency did not affect rates or incidence of oncogene-induced lung tumors or mutations. Taken together, we demonstrate that Rad18 has context-specific tumor-suppressive activity. Given the prevalence of 4NQO-like environmental exposures, RAD18 is highly likely to shape human cancer genomes and perhaps influence other aspects of the tumorigenic process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/318ddf85ddf5e7011f464ee8a8d2e2b51144fcdd" target='_blank'>
              DIFFERENTIAL ROLES OF RAD18 IN REPRESSING CARCINOGEN- AND ONCOGENE-DRIVEN MUTAGENESIS IN VIVO
              </a>
            </td>
          <td>
            J. R. Anand, Bethany Wagner, Jitong Lou, Qisheng Gu, Yang Yang, Gaith N Droby, Di Wu, Akankshya Jena, Jialiu Xie, Yuri Fedoriw, Bernard E Weissman, Scott E. Williams, Cyrus Vaziri
          </td>
          <td>2025-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background The KRAS gene is a member of the Ras protein family. KRAS mutations are prevalent oncogenic drivers in non-small cell lung cancer (NSCLC), even in the Chinese populations, where they account for 10–15% of cases, and are correlated with aggressive disease and poor clinical outcomes. Despite recent breakthroughs in direct KRAS G12C inhibitors (e.g., sotorasib and adagrasib), the therapeutic efficacy of these inhibitors remains limited; the median progression-free survival of KRAS-mutated NSCLC patients rarely exceeds 6 months, and actionable therapies for non-G12C variants are lacking. Based on this foundation, the study aimed to identify novel potential targeted therapeutic strategies for patients with KRAS-mutant NSCLC through systematic target screening. Methods The following three-step process and selection criteria were employed to identify the KRAS function-related genes: (I) the genes were differentially expressed in the NSCLC tissues compared to the normal tissues; (II) the differentially expressed genes were highly expressed in the KRAS-mutated tissues compared to the KRAS-wild-type NSCLC tissues in The Cancer Genome Atlas (TCGA) cohort; and (III) the genes had a dependency score of >0.9 in the KRAS-mutated NSCLC cell lines from the Cancer Cell Line Encyclopedia based on the dataset. We then examined the prognostic value of the KRAS function-related genes in TCGA cohort (504 lung adenocarcinoma samples) and validated their prognostic value in the GSE72094 dataset (398 lung adenocarcinoma samples). Additionally, we conducted a gene set enrichment analysis (GSEA) to investigate the potential mechanisms by which the candidate genes affected prognosis, and performed a drug sensitivity analysis to identify compounds exhibiting sensitivity to candidate gene expression levels in the KRAS-mutated NSCLC cell lines. Results We identified four KRAS function-related genes and showed that ATR expression was significantly associated with overall survival (P=0.008). After adjusting for age, gender and TNM stage (Tumor-Node-Metastasis stage), high ATR expression remained an independent predictor of a worse prognosis in KRAS-mutated NSCLC [hazard ratio (HR) =2.192; P=0.01] in the TCGA cohort, and this prognostic significance was validated in the GSE72094 dataset (HR =2.06; P=0.02). The GSEA results showed that the enriched genes in the ATR high-expression group were significantly associated with ubiquitin mediated proteolysis, pathways in cancer, and the mitogen-activated protein kinase signaling pathway compared to the ATR low-expression group. Additionally, the drug sensitivity analysis identified two compounds (i.e., AZ20 and AZD6738) that were sensitive to ATR expression. Conclusions ATR holds promise as both a prognostic marker and therapeutic target for KRAS-mutated NSCLC. Our findings may assist in the prediction of prognosis and the development of novel targeted therapies for this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b2025a8674c703e824e201cb7b82693e1795503" target='_blank'>
              ATR expression as a prognostic biomarker in KRAS-mutated non-small cell lung cancer
              </a>
            </td>
          <td>
            Li Jiang, Chao Luo, Peng Zhang, Renyu Wang, Lili Tang, Xufeng Deng, Anna T. Allemann, A. Abhyankar, Hong Zheng, Jigang Dai
          </td>
          <td>2025-07-01</td>
          <td>Journal of Thoracic Disease</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Esophageal squamous cell carcinoma (ESCC) is the sixth leading cause of cancer-related deaths globally, with more than half of cases occurring in China. Although chemoradiotherapy (CRT) is a major treatment modality for ESCC, its efficacy is often limited by tumor resistance. The 5-year survival rate of esophageal cancer patients is less than 30%, and the prognosis is poor. The mechanisms of induced resistance in cancer mainly include multi-dimensional factors such as the regulation of the tumor microenvironment (TME), the metabolic reprogramming of cancer-associated fibroblasts (CAFs), and the imbalanced proliferation of cancer stem cells. New strategies are urgently needed to break through this bottleneck. The study primarily focuses on reviewing the regulation of the tumor microenvironment, abnormal drug metabolism and transport, epigenetic modifications, and signaling pathways, aiming to reveal the mechanisms of CRT resistance, summarize strategies to overcome CRT resistance, and explore new treatment strategies such as targeted therapy and immunotherapy combination. This review emphasize the importance of multidisciplinary collaboration and individualized treatment to improve the cure rate and survival of ESCC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1521451f119c1dab5be6cfe7415787cbde42aa9" target='_blank'>
              Molecular Mechanisms of Chemoradiotherapy Resistance in Esophageal Cancer and Clinical Response Strategies: From the Tumor Microenvironment to Precision Medicine
              </a>
            </td>
          <td>
            Yujiao Sun
          </td>
          <td>2025-06-27</td>
          <td>Highlights in Science, Engineering and Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The rarity of recurrent somatic mutations poses a challenge for the targeted treatment of neuroblastoma (NB). Differentiation therapy is an encouraging prospect, with cyclin-dependent kinase inhibitors (CDKis) representing a promising avenue for promoting NB differentiation. This study investigated three CDKis (abemaciclib, fadraciclib, and dinaciclib) alone or combined with retinoic acid (RA) to assess the effects on morphology, growth, gene expression, and the induction of immunogenic cell death in NB cell lines with (LAN-1 and CHLA-90) and without (CHLA-172) MYCN amplification. All cell lines demonstrated sensitivity to CDK inhibition. Notably, low-dose abemaciclib promoted cellular differentiation, as evidenced by the emergence of stromal-like morphological features and upregulation of the differentiation markers STMN4 and ROBO2. Treatment with abemaciclib or fadraciclib led to the upregulation of calnexin and holocytochrome C, which are part of the global stress response, along with the protein p27, which arrests the cell cycle. Molecularly, CDKis sensitivity correlated with an increased CDK4-specific copy number, along with a partial deletion of CDKN2a in two cases (LAN-1, CHLA-172). The addition of RA augmented the effects of the monotherapy, particularly in LAN-1 cells, in both 2D and 3D culture, and both treatments triggered immunogenic cell death, evidenced by calreticulin translocation. Transcriptomic analysis of LAN-1 and CHLA-90 cells revealed that genes deregulated by monotherapy (fadraciclib or RA) were re-regulated in the presence of the second drug. Combination therapy significantly downregulated CRABP2 and CYP26B1, both of which are involved in RA metabolism and its degradation. Furthermore, CCNE2, MYBL2, and MCM4 were strongly suppressed in the fadraciclib/RA combination, confirming the induction of cell cycle arrest. CDKi treatments promote NB differentiation via ER stress, with cytotoxicity enhanced by RA co-treatment. This may increase NB immunogenicity and support immunotherapy eligibility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85ca193304bf6f6433a4c52b2f5a75f164abeec5" target='_blank'>
              CDK inhibitors promote neuroblastoma cell differentiation and increase sensitivity to retinoic acid—a promising combination strategy for therapeutic intervention
              </a>
            </td>
          <td>
            Fatemeh Shokraie, Larissa Lechermeier, Pia Bordihn, Philipp Kaps, Steffen Möller, Anna Sophie Schulz, Björn Schneider, Dirk Koczan, Samira Khanipour Roshan, Holger N Lode, C. Classen, Olga Hahn, S. Troschke-Meurer, C. Maletzki
          </td>
          <td>2025-08-02</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Resistance to CDK4/6 inhibitors (CDK4/6i) is a major challenge in treating hormone receptor-positive, HER2-negative metastatic breast cancer. This study aimed to identify a biomarker predictive of resistance that could also serve as a therapeutic target. Circulating IL-6 levels in 166 patients significantly increased at progression, making IL-6 a non-invasive biomarker to predict CDK4/6i resistance. Knockdown of IL-6 re-sensitized resistant cells to palbociclib and endocrine therapy, underscoring IL-6’s critical role in maintaining resistance. Patient-derived xenograft models from patients who progressed within 3 months versus ≥6 months of palbociclib therapy revealed distinct transcriptomic profiles, with later progressors exhibiting IL-6/STAT3 activation, epithelial-to-mesenchymal transition, and immune evasion signatures. Treatment with TTI-101, a STAT3 inhibitor, significantly reduced tumor growth and improved survival in these models, providing preclinical validation for targeting the IL-6/STAT3 axis. These findings support using IL-6 to guide personalized treatment and combining STAT3 inhibitors with CDK4/6i as a transformative strategy to overcome resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08735c14db476a16d97e533ff51d162f3e014327" target='_blank'>
              IL-6 predicts CDK4/6 inhibitor resistance, identifying STAT3 as a target in HR + /HER2-negative metastatic breast cancer
              </a>
            </td>
          <td>
            N. Kettner, Tuyen N Bui, Juliana Navarro-Yepes, Sanaz Ghotbaldini, Bethanie Quintela, Catherine K. Luo, Nghi Lam, X. Rao, A. Raghavendra, Yan Wang, Nancy Azizian, T. Kris Eckols, Moses M. Kasembeli, K. Evans, Min Yi, Hannah F Wingate, Jing Wang, Aysegul A. Sahin, Funda Meric-Bernstam, Kelly K Hunt, S. Damodaran, David J Tweardy, Debu Tripathy, K. Keyomarsi
          </td>
          <td>2025-07-25</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8030ad41247de8846ce913550c0870c5e34232e" target='_blank'>
              Colon cancer cells evade drug action by enhancing drug metabolism.
              </a>
            </td>
          <td>
            Bojie Cong, Teena Thakur, Alejandro Huerta Uribe, Evangelia Stamou, Sindhura Gopinath, Owen Sansom, Oliver D. K. Maddocks, Ross L Cagan
          </td>
          <td>2025-07-10</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Background: Tumor-associated macrophages (TAMs) are critical regulators of the hepatocellular carcinoma (HCC) microenvironment, yet their epigenetic heterogeneity and regulatory programs remain poorly defined. Methods: We performed integrative analysis on single-cell RNA-seq and ATAC-seq profiling of HCC patients to dissect TAM subtypes at high resolution. By correlating chromatin accessibility with gene expression, we identified cell-type-specific candidate cis-regulatory elements (CREs). TAM subsets with prognostic significance were determined through integration with HCC clinical cohorts. Pseudotime and multi-regional analyses were used to uncover regulatory trajectories underlying macrophage phenotypic transitions. The identification framework of a super-enhancer (SE) was constructed, and potential therapeutic targets were prioritized using drug–gene interaction data. Results: We delineated the regulatory landscape of TAMs in HCC, revealing cell-type-specific chromatin accessibility patterns underlying TAM heterogeneity. The 65,342 CREs linked to gene expression were identified, with distal CREs contributing most to cell-type-specific regulation. Notably, SPP1+ TAMs were found to be enriched in tumor cores and associated with poor prognosis in HCC. Liver-resident Kupffer cells showed progressive loss of the core transcription factors SPIC and MAFB, suggesting a potential transition into SPP1+ TAMs under tumor pressure. We identified 133 SPP1+ TAM-specific SEs and constructed a TF–SE–target gene regulatory network. Notably, 13 target genes showed higher drug–gene interaction effects, highlighting their therapeutic potential. Conclusions: This study provides the chromatin accessibility map of TAMs in HCC and reveals how distal CRE-driven transcriptional programs shape TAM states. Our findings lay the foundation for understanding the epigenetic regulation of TAM heterogeneity and nominate potential targets for TAM-directed immunotherapy in HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e02327a10ea72222cca675d378cb3f0e78ed3b3" target='_blank'>
              Integrated Single-Cell Analysis Dissects Regulatory Mechanisms Underlying Tumor-Associated Macrophage Plasticity in Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Yu Gu, Wenyong Zhu, Zhihui Zhang, Huiling Shu, Hao Huang, Xiao Sun
          </td>
          <td>2025-07-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b7bd3ebe84ec1189f788307a3b3b752befef2ae" target='_blank'>
              Integrated Single-Cell and Spatial Transcriptomic Analysis Reveals an Aging-Associated Fibroblast Subtype Linked to Tumor Progression in Human Skin
              </a>
            </td>
          <td>
            Haibin Wu, Danping Pan
          </td>
          <td>2025-07-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer represents a significant global health hazard marked by elevated morbidity and mortality rates. Furthermore, the majority of tumor therapies encounter challenges, including metastasis, recurrence, and drug resistance. Consequently, it is essential to identify a specific and efficient tumor treatment approach. In recent years, the ongoing investigation and comprehension of tumors have led to significant attention towards cancer stem cells (CSCs). CSCs can facilitate tumor progression via self-renewal, differentiation capabilities, and multidrug resistance. Their function as a fundamental contributor to tumor heterogeneity, drug resistance, recurrence, and metastasis has emerged as a significant focus in cancer therapy research. In recent years, microRNAs (miRNAs) have been identified as crucial post-transcriptional regulators in biological processes, including chemosensitivity, self-renewal, apoptosis, invasion, and metastasis of cancer stem cells (CSCs). This paper systematically reviews the molecular mechanisms through which miRNAs influence the characteristics of cancer stem cells by targeting essential genes (e.g., SOX2, EGFR, c-Met) and modulating signaling pathways, including Wnt/β-catenin, Notch, Hedgehog, and PI3K/Akt. Furthermore, we investigated the viability of miRNAs as non-invasive biomarkers for cancer diagnosis and prognosis evaluation, examined the similarities and attributes of pivotal miRNAs in modulating cancer stem cell functionality, and deliberated on therapeutic approaches stemming from miRNA regulation of cancer stem cell activity. We anticipate that this research will yield novel insights into targeted cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c085b3dd2e8cad808f86e5932aff3c84c64bec4" target='_blank'>
              Regulatory Functions of microRNAs in Cancer Stem Cells: Mechanism, Facts, and Perspectives
              </a>
            </td>
          <td>
            Xingmei Mao, Sixue Peng, Yan Lu, Linjiang Song
          </td>
          <td>2025-07-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="BRAF and MEK inhibitors (BRAF/MEKi) have radically changed the treatment landscape of advanced BRAF mutation-positive tumours. However, limited efficacy and emergence of drug resistance are major barriers for successful treatments. Here, by using relevant preclinical models, we find that Connexin43 (Cx43), a protein that plays a role in cell-to-cell communication, enhances the effectiveness of BRAF/MEKi by recruiting DNA repair complexes to lamin-associated domains and promoting persistent DNA damage and cellular senescence. The nuclear compartmentalization promoted by Cx43 contributes to genome instability and synthetic lethality caused by excessive DNA damage, which could provide a therapeutic approach for these tumours to overcome drug resistance. Based on these findings, we designed a drug combination using small extracellular vesicles (sEVs) to deliver the full-Cx43 in combination with the BRAF/MEKi. This study reveals Cx43 as a regulator of DNA repair and BRAF/MEKi response, highlighting the therapeutic potential that this approach could eventually have in the clinic to overcome the limitations of current therapies and improve treatment outcomes for patients with advanced BRAF mutant tumours.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1e4426f618e52ca064c1a54a223e0ac95626bea" target='_blank'>
              Cx43 enhances response to BRAF/MEK inhibitors by reducing DNA repair capacity
              </a>
            </td>
          <td>
            A. Varela-Vázquez, A. Guitián-Caamaño, P. Carpintero-Fernández, Alexander Carneiro-Figueira, Vanesa Álvarez, M. Varela-Eirín, Teresa Calleja-Chucla, S. B. Bravo-López, Anxo Vidal, Juan Sendón-Lago, Marina Rodríguez-Candela Mateos, José R Caeiro, Victoria Sanz-Moreno, Trond Aasen, Miguel G. Blanco, Guadalupe Sabio, María Quindós, Carmen Rivas, David Santamaría, Carlos Fernandez-Lozano, E. Fonseca, Pablo Huertas, Berta Sánchez-Laorden, Constance Alabert, M. Mayán
          </td>
          <td>2025-07-04</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/827b6d7c12ecbffd0d9766944bd9dcb122ff76bc" target='_blank'>
              Epithelial state-transitions permit inflammation-induced tumorigenesis
              </a>
            </td>
          <td>
            E. Jarman, Anabel Martinez Lyons, Yuelin Yao, Aleksandra Rozyczko, S. Waddell, Andreea Gradinaru, Paula Olaizola, Kyle Davies, Rachel V. Guest, Stephanie Röessler, Timothy J. Kendall, Owen J. Sansom, A. Khamseh, Luke Boulter
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="ABSTRACT DNA methylation is a common epigenetic modification that maintains the integrity of the DNA sequence while profoundly influencing gene expression and phenotypic variation. Aberrant DNA methylation has been associated with the onset and progression of diseases, including cancer, metabolic disorders, and neurodevelopmental disorders. Recent advancements in detection technology led to a gradual increase in the exploration of DNA methylation as a valuable biomarker for cancer diagnosis and therapy. Single-base resolution has been achieved for whole-genome methylation analyses through second-generation sequencing technology, significantly enhancing detection efficiency. Additionally, PCR-based methods offer simple and feasible solutions for methylation analysis. In this review, we discuss various methods for detecting DNA methylation, focusing on bisulfite conversion-based techniques, methylation-sensitive restriction enzyme methods, enzyme conversion-based methods, third-generation sequencing approaches, and artificial intelligence. Furthermore, we briefly summarize the methylation biomarkers used for tumor diagnosis and the corresponding sample types employed. We believe that this information provides valuable insights for selecting and optimizing DNA methylation analysis tools.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecc2bff17d09231bf27c4fb389c540837b1a009a" target='_blank'>
              Advancements in DNA methylation technologies and their application in cancer diagnosis
              </a>
            </td>
          <td>
            Yang Yang, Xiaosha Wen, Li Wang
          </td>
          <td>2025-07-28</td>
          <td>Epigenetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6eee44887a88c522cc74478c5f67d3f1813be9b0" target='_blank'>
              DPYSL3B is a regulator of chemoresistance via DNA repair and metabolic reprogramming in prostate cancer
              </a>
            </td>
          <td>
            R. Kaarijärvi, H. Kaljunen, Nella Itkonen, Kseniia Bureiko, K. Ketola
          </td>
          <td>2025-07-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Space ionizing radiation is a major risk factor for astronauts, yet its molecular mechanisms remain poorly understood. This study employs an integrative approach to investigate the effects of space ionizing radiation on molecules, modules, biological functions, associated diseases, and potential therapeutic drugs. Using paired samples from five donors subjected to acute ex vivo 2Gy gamma-ray irradiation, we analyzed miRNA and gene expression profiles in human peripheral blood lymphocytes collected 24 h post-exposure, combined with heterogeneous network analysis, identifying 179 key molecules (23 transcription factors, 10 miRNAs, and 146 genes) and 5 key modules. Functional enrichment analysis revealed associations with processes such as cell cycle regulation, cytidine deamination, cell differentiation, viral carcinogenesis, and apoptosis. Radiation was also significantly linked to neoplasms and digestive system diseases. Furthermore, we predicted 20 potential therapeutic compounds, including small molecules (e.g., Navitoclax) and Traditional Chinese Medicine ingredients (e.g., Genistin, Saikosaponin D), which may alleviate radiation-induced damage such as pulmonary fibrosis and oxidative stress. These findings provide novel insights into the molecular mechanisms of space ionizing radiation and may contribute to developing effective strategies to protect astronaut health during space missions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45bab4ebbff08f4904eff8b3fb60d25f1f1fd6d3" target='_blank'>
              Exploring molecular and modular insights into space ionizing radiation effects through heterogeneous gene regulatory networks
              </a>
            </td>
          <td>
            Mengqin Yuan, Tao Zhang, Haizhou Liu, Min Long, Quan Wang, Wei Jiang
          </td>
          <td>2025-07-18</td>
          <td>NPJ Microgravity</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 The androgen receptor (AR) plays a critical role in the development and progression of prostate cancer by regulating key cellular processes such as cell proliferation and apoptosis. Although traditional AR-targeted therapies have shown initial success, acquired resistance remains a significant clinical challenge, often driven by AR alterations and somatic gene mutations associated with homologous recombination deficiency (HRD). Approximately 20% of advanced prostate cancer cases exhibit HRD, resulting in substantial genomic instability and complicating treatment. Fortunately, Food and Drug Administration–approved poly(ADP-ribose) polymerase inhibitors, including olaparib and rucaparib, exploit synthetic lethality to target prostate cancer with HRD, and additional drugs targeting DNA damage response (DDR) proteins are under development. Emerging evidence suggests that AR activity enhances DDR gene expression, with multiple DDR proteins localized near androgen-regulated regions, highlighting a close interaction between AR and DDR pathways. Consequently, recent preclinical and clinical studies have investigated combining AR-targeted therapies with treatments that induce DNA damage, such as radiation therapy, or inhibit DNA repair mechanisms. This review discusses AR's role in cellular processes, the interplay between AR and DDR, and recent advances in prostate cancer treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4bcf547c4cdb4775dcc9944086243c67a7115b7" target='_blank'>
              The interconnection between AR and DNA damage response pathways in prostate cancer
              </a>
            </td>
          <td>
            Mallory Sands, Sam Adams, Jihaeng Lee, Michael Li, Mae Wang, Thomas Walsh, Leo Leon, Adriana Zablah, Matthew Haerens, Zhichao Liu, Jianneng Li
          </td>
          <td>2025-08-19</td>
          <td>Current Urology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="As we enter the age of personalized medicine, healthcare is increasingly focused on tailoring diagnoses and treatments based on patients' genetic and environmental circumstances. A critical component of a person's physiological makeup is their immune system, but individual genetic variation in many immune system genes has remained resistant to analysis using classical whole-genome or targeted sequencing approaches. In particular, germline adaptive immune system genes, like immunoglobulin (IG) and T cell receptor (TR) genes, are particularly hard to genotype using classic reference-based methods owing to their highly repetitive and homologous nature. In this paper, we present ImmunoTyper2, a new computational toolkit for genotyping the variable genes of the IG lambda and kappa, and the TR loci with short-read whole genome sequence data, using an integer linear programming formulation, as an update to the ImmunoTyper-SR suite, which focused on IGHV region only. We evaluate its genotyping performance using Mendelian concordance analysis in 590 trios from the 1000 Genomes Project, benchmarking 40 samples against HPRC assembly-derived genotypes, and assessing robustness through sequencing depth analysis and parameter sensitivity tests. We introduce allele call confidence metrics to help quantify reliability. We also perform a prospective disease association study, applying ImmunoTyper2 to a WGS data set from a cohort of 461 COVID-19 patients from the COVNET Consortium to demonstrate how it can be applied to investigate genetic associations with disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a34d06538904247946e24c5dc11a99e3b25b59b" target='_blank'>
              Genotyping of selected germline adaptive immune system loci using short-read sequencing data.
              </a>
            </td>
          <td>
            M. K. Ford, Ananth Hari, Meredith Yeager, L. Mirabello, Stephen J. Chanock, Ibrahim Numanagić, S. C. Sahinalp
          </td>
          <td>2025-08-05</td>
          <td>Genome research</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Atypical teratoid rhabdoid tumors (ATRTs) are aggressive central nervous system tumors mainly affecting young children. Extensive molecular characterization based on gene expression and DNA methylation patterns has solidly established three major ATRT subtypes (MYC, SHH and TYR), which show distinct clinical features, setting the basis for more effective, targeted treatment regimens. Transcriptional activity of transposable elements (TEs), like LINE1s and LTRs, is tightly linked with human cancers as a direct consequence of lifting epigenetic repression over TEs. The sole recurrent biallelic loss-of-function mutation in SMARCB1 in ATRTs, a core component of the SWI/SNF chromatin remodeling complex, raises the question of how TE transcription contributes to ATRT development. Here, we comprehensively investigate the transcriptional profiles of 1.9M LINE1 and LTR elements across ATRT subtypes in primary human samples. We find TE transcription profiles allow sample stratification into ATRT subtypes. The TE activity signature in the ATRT-MYC subtype is unique, setting these tumors apart from SHH and TYR ATRTs. More specifically, ATRT-MYC show broadly reduced transcript levels of LINE1 and ERVL-MaLR subfamilies. ATRT-MYC also displayed significantly less LTR and LINE1 loci with bidirectional promoter activity. Furthermore, we identify 849 differentially transcribed TEs in primary samples, which are predictive towards established ATRT-SHH and -MYC cell line models. In summary, including TE transcription profiles into the molecular characterization of ATRTs might reveal new tumor vulnerabilities leading to novel therapeutic interventions, such as immunotherapy. Supplementary Information The online version contains supplementary material available at 10.1186/s40478-025-02078-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c60e5327059223ad1194aac3ab94faeb7d65828" target='_blank'>
              Expression of LTR and LINE1 transposable elements defines atypical teratoid/rhabdoid tumor subtypes
              </a>
            </td>
          <td>
            M.V. Hamann, Shweta Godbole, Maisha Adiba, Sabrina M. Leddy, Jelena Navolić, Ghazaleh Tabatabai, Daniel J Merk, Ulrike C. Lange, J. Neumann
          </td>
          <td>2025-07-22</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Abstract Extrachromosomal DNAs (ecDNAs), which replicate and segregate in a non-Mendelian manner, serve as vectors for accelerated tumor evolution. By integrating chromatin accessibility, whole-genome sequencing, and Hi-C-based genome topology data from a cohort of metastatic Castration-Resistant Prostate Cancer (mCRPC) cases, we show that epigenetically activated repeat DNA, amplified in ecDNAs, drive oncogene overexpression. Specifically, we identify a subgroup of mCRPCs (20%) characterized by clusters of accessible LINE1 repeat DNA elements flanking the androgen receptor (AR) gene. These LINE1 elements are co-amplified with AR and provide binding sites for prostate-lineage transcription factors, including AR, FOXA1 and HOXB13. Accessible LINE1 elements establish novel 3D chromatin interactions with the AR gene, forging a new regulatory plexus driving AR overexpression and confers resistance to androgen signaling inhibitors. Our findings indicate how tumor evolution is driven by the convergence of genetic and epigenetic alterations on repeat DNA, activating and amplifying them to allow oncogene overexpression. Statement of significance We show how tumor evolution is driven by the convergence of genetic and epigenetic alterations on repeat DNA elements, resulting in their activation as regulatory elements and co-amplification in ecDNAs with oncogenes in mCRPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e6c5dce2ec927f5cff069554fa04e5223fd82a5" target='_blank'>
              Genetic and Epigenetic Reprogramming of Transposable Elements Drives ecDNA-Mediated Metastatic Prostate Cancer
              </a>
            </td>
          <td>
            Lisanne Mout, Thaidy Moreno-Rodriguez, Giacomo Grillo, Ankita Nand, Tina Keshavarzian, S. Bahl, Komaldeep Kang, M. Bootsma, Emma Minnee, Stanley Zhou, Kathleen H. Burns, Eva Corey, Peter Nelson, S. Dehm, , W. Zwart, Felix Feng, David Quigley, M. Lupien
          </td>
          <td>2025-08-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Colorectal cancer (CRC) treatment remains challenging due to genetic heterogeneity and resistance mechanisms. To address this, we developed a drug discovery pipeline using patient-derived primary CRC cultures with diverse genomic profiles. These cultures closely resemble certain molecular characteristics of primary and metastatic CRC, highlighting their promise as a translational platform for therapeutic evaluation. Importantly, our engineered model and patient-derived cells reflect the complexity and heterogeneity of primary tumors, not observed with standard immortalized cell lines, offering a more clinically relevant system, although further validation is needed. High-throughput screening (HTS) of 4255 compounds identified 33 with selective efficacy against CRC cells, sparing normal, healthy epithelial cells. Among the tested combinations, everolimus (mTOR inhibitor) and uprosertib (AKT inhibitor) demonstrated promising synergy at clinically relevant concentrations, with favorable therapeutic windows confirmed across tested patient-derived cultures. Notably, this synergy, revealed through advanced models, might have been overlooked in traditional immortalized cell lines, highlighting the translational advantage of patient-derived systems. Furthermore, the integration of machine learning into the HTS pipeline significantly improved scalability, cost-efficiency, and predictive accuracy. Our findings underscore the potential of patient-derived materials combined with machine learning-enhanced drug discovery to advance personalized therapies. Specifically, mTOR-AKT inhibition emerges as a promising strategy for CRC treatment, paving the way for more effective and targeted therapeutic approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b44bc31581c51d6eaf637f51510217abbfb946c" target='_blank'>
              Integrated transcriptomic and functional modeling reveals AKT and mTOR synergy in colorectal cancer
              </a>
            </td>
          <td>
            M. Dulęba, E. Zimoląg, J. Szuszkiewicz, M. Serocki, Joanna Szklarczyk, Olga Dracz, A. Kurzejamski, Izabella Więckowska, M. Chmiel, B. Lipert, Katarzyna Sarad, Joanna Krawczyk, O. Bryzghalov, Agata Stachowicz-Wałaszek, Karolina Pyziak, Joanna Drozdowska, Krzysztof Baczyński, Konrad Wojtowicz, Maurycy Chronowski, Krzysztof Rataj, Magdalena Otrocka, M. Mikula, Paula Feliksiak, Rafal Dziadziuszko, T. Rzymski, Andrew Thomason, K. Brzózka, Andrzej Mazan
          </td>
          <td>2025-07-31</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Background: SARS-CoV-2, which drove the 2019 COVID-19 pandemic, continues to engender inquiries into the role of host genetic factors in disease susceptibility. Despite the identification of over 1,000 genes potentially associated with SARS-CoV-2 and COVID-19, the mechanisms connecting genetic variants to phenotype remain elusive. To shed light on these mechanisms, we undertook an integrated analysis, merging data from whole-genome association analyses of COVID-19 with methylome and transcriptomic. Methods: Study includes African American adults from the GENE-FORECAST study, encompassing 371 individuals with whole genome sequencing (WGS), 203 with DNA methylation, and 321 with RNA sequencing (RNA-Seq) of blood. 53.3% of participants reported COVID-19. Significant loci associated with COVID-19 were examined within the framework of methylation quantitative trait loci (mQTL) which located near the gene-of-original (cis-mQTL) and expression quantitative trait loci (eQTL) which located near the gene-of-origin (cis-eQTL), enabling analysis to assess mediators between genetic variants and COVID-19 status. Results: Our analysis identified four intronic variants and confirmed a missense variant, rs1052067, in PMF1 associated with COVID-19. Causal mediation analysis revealed that the combination of genetic variants within PMF1, epigenomics, and transcriptomics mapped four pathways influencing COVID-19 status. These pathways include: rs9659072 -> DNAm at chr1:156285845 (annotated to TMEM79) -> ENSG00000198715:13 (annotated to GLMP); rs12083543 -> DNAm at chr1:155951748 (ARHGEF2) -> ENSG00000198715:13 (GLMP); rs1052067 -> DNAm at chr1:155951748 (ARHGEF2)-> ENSG00000198715:13 (GLMP); rs1543294 -> ENSG00000198715:13 (GLMP) -> DNAm at chr1:156077518 (MEX3A). Conclusions: Through integrated multi-omics analyses, we identified genetic variants whose effects on COVID-19 susceptibility are mediated by changes in DNA methylation and mRNA expression. These findings offer insights into potential mechanistic pathways that merit further exploration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00c7bc8e6839adbc276042c4324afad16196de83" target='_blank'>
              Multi-omics analyses of the complex interplay between genetic variants, DNA methylation, and gene expression in COVID-19.
              </a>
            </td>
          <td>
            Guanjie Chen, Lisa DeRoo, Gabriel Goodney, A. Doumatey, Jie Zhou, A. Adeyemo, C. Rotimi, Amadou Gaye
          </td>
          <td>2025-07-01</td>
          <td>American journal of physiology. Heart and circulatory physiology</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5f661d74703dc0b2d864c7015c79812dfe307ea" target='_blank'>
              Decoding the Transcriptional Complexity of the Human BRCA2 DNA Repair Gene Using Hybrid-seq.
              </a>
            </td>
          <td>
            P. Adamopoulos, Michaela A. Boti, Konstantina Athanasopoulou, P. Tsiakanikas, Glykeria N. Daneva, Andreas Scorilas
          </td>
          <td>2025-07-10</td>
          <td>Biochemical genetics</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background The APOBEC (apolipoprotein B mRNA editing enzyme, catalytic polypeptide-like) family plays a vital mutagenic role in diverse human malignancies. Nevertheless, the biological characteristics of APOBEC family members and their clinical significance in cancer have not been comprehensively explored. Our primary objective was to characterize the distribution and clinical relevance of APOBEC family members and their mutations across multiple cancer types, with a particular focus on thyroid carcinoma (THCA). Methods In this study, we employed an integrated, multi-faceted approach combining diverse statistical and bioinformatics methods. Data sources included whole-exome sequencing (WES), targeted next-generation sequencing (NGS), bulk RNA sequencing (RNA-seq), single-cell RNA sequencing (scRNA-seq), Western blot assays, drug sensitivity analyses, and in vivo animal models. Results APOBEC mutations occupy a significant position in the mutation landscape of THCA. High APOBEC mutation enrichment scores were strongly associated with poor prognosis, immune evasion, and increased risk of malignant progression in THCA patients. These mutations contribute to tumor advancement and influence cellular differentiation within the tumor microenvironment. Additionally, we developed a prognostic APOBEC mutagenesis model using machine learning, which was validated across multiple THCA cohorts. In vitro and in vivo experiments demonstrated that APOBEC2 inhibition effectively suppressed tumor proliferation, metastasis, and glycolytic activity, while simultaneously enhancing immune activation and boosting the efficacy of immune checkpoint inhibitors. Conclusion Our study systematically characterizes APOBEC mutations across various cancer types and underscores their promise as biomarkers for aggressive tumor phenotypes, prognostic assessment, and immunotherapy response to THCA. These findings were accomplished through multi-omics sequencing and validated through comprehensive in vitro and in vivo experiments. Supplementary Information The online version contains supplementary material available at 10.1186/s12575-025-00288-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95761133ff7f07fa29822ab7f50151c4ca2027f9" target='_blank'>
              Comprehensive Insights into APOBEC Mutations in Thyroid Cancer: Prognostic and Therapeutic Discoveries
              </a>
            </td>
          <td>
            Wei Luo, Feng Liu, Mengyu Li, Jialong Yu, Ziyun Liu, Xuan Cheng, Yue Huang, Yu Liu, Mei Tao, Yuqi Wang, Yiping Zou, Xiaobin Shang, Chao Yang, X. Ruan, Yanchao Qin, X. Zheng
          </td>
          <td>2025-07-26</td>
          <td>Biological Procedures Online</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8da4d10721badd9a65f0ec25fedb84c3d3183aaf" target='_blank'>
              Direct functional HOXA9/DNA-binding competitors versus epigenetic inhibitors of HOXA9 expression on cell proliferation, death and differentiation processes in the model of MLL-rearranged acute myeloid leukemia.
              </a>
            </td>
          <td>
            Julie Vrevin, Mélanie Lambert, Marine Andrique, Nathalie Jouy, M. David-Cordonnier
          </td>
          <td>2025-08-04</td>
          <td>Investigational new drugs</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="INTRODUCTION
Chronic myeloid leukemia (CML) is a disease characterized by a single diagnostic molecular abnormality and epitomizes a genetically based diagnosis and management. Targeted therapy with tyrosine kinase inhibitors has revolutionized treatment and prognosis of this disease; however, a proportion of patients will experience resistance to therapy and progress to blast phase, the terminal phase of this disease. Emerging genomic profiling in CML reveals a genetically heterogenous landscape which may permit further risk stratification, personalization of treatment, and the potential for drug development.This review encompasses original literature exploring the genomic profile of CML and the potential impact of additional genomic abnormalities on prognosis and treatment response.


EXPERT OPINION
Somatic variants in cancer-associated genes, particularly ASXL1, are of prognostic and potential therapeutic significance in CML. Up-front next-generation sequencing is therefore useful when available in all patients with newly diagnosed CML and repeat testing should be considered at timepoints of suboptimal treatment response, TKI resistance, and progression. Gene rearrangements and fusions are an emerging class of mutation that may confer adverse prognostic risk and hence use of a robust testing method that enables detection of such abnormalities is desirable.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16f846d792b4b438a93d642bb6eb890bbaae7db4" target='_blank'>
              How will genomic testing impact a clinician's choice for managing chronic myeloid leukemia?
              </a>
            </td>
          <td>
            Jessica Sia, N. Shanmuganathan
          </td>
          <td>2025-07-10</td>
          <td>Expert review of hematology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id=" Osteosarcoma (OS), the most common primary bone cancer of mesenchymal origin in children and young adolescents, remains a challenge due to metastasis and resistance to chemotherapy. It displays severe aneuploidy and a high mutation frequency which drive tumor initiation and progression; however, recent studies have highlighted the role of epigenetic modifications as a key driver of OS pathogenesis, independent of genetic mutations. DNA and RNA methylation, histone modifications and non-coding RNAs are among the major epigenetic modifications which can modulate the expression of oncogenes. Abnormal activity of these mechanisms contributes to gene dysregulation, metastasis and immune evasion. Therapeutic targeting against these epigenetic mechanisms, including inhibitors of DNA and RNA methylation as well as regulators of RNA modifications, can enhance tumor suppressor gene activity. In this review, we examine recent studies elucidating the role of epigenetic regulation in OS pathogenesis and discuss emerging drugs or interventions with potential clinical utility. Understanding of tumor- specific epigenetic alterations, coupled with innovative therapeutic strategies and AI-driven biomarker discovery, could pave the way for personalized therapies based on the molecular profile of each tumor and improve the management of patients with OS. ">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69115f288109b08c5dc3c9431da8115c0747b13f" target='_blank'>
              Epigenetic Modifications in Osteosarcoma: Mechanisms and Therapeutic Strategies
              </a>
            </td>
          <td>
            Maria A. Katsianou, Dimitrios Andreou, Penelope Korkolopoulou, Eleni-Kyriaki Vetsika, C. Piperi
          </td>
          <td>2025-07-28</td>
          <td>Life</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2528b4a53c81ca371bb9e25c23c6ee2756be3b41" target='_blank'>
              Underlying biology, challenges and emergent concepts in the treatment of relapsed and refractory pediatric T-cell acute lymphoblastic leukemia.
              </a>
            </td>
          <td>
            Patrícia Amaral, Rhona Christie, D. O. Gresham, Emma J M Lucas, Luyao Kevin Xu, Lena Behrmann, Jonathan Bond, S. Degerman, Frederik W van Delft, S. Goossens, Melanie M. Hagleitner, Chris Halsey, Nicholas Jones, T. Lammens, F. V. van Leeuwen, Marc R. Mansour, Panagiotis Ntziachristos, David O'Connor, João T Barata
          </td>
          <td>2025-08-14</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Increasing evidence underscores the driving role of coding and non-coding variants in cancer development. Analyzing gene sets in biological processes offers deeper insights into the molecular mechanisms of carcinogenesis. Here, we developed geMER to identify candidate driver genes genome-wide by detecting mutation enrichment regions within coding and non-coding elements. We subsequently designed a pipeline to identify a core driver gene set (CDGS) that broadly promotes carcinogenesis across multiple cancers. CDGS comprising 25 genes for 25 cancers displayed instability in DNA aberrations. Variants within the TTN enrichment region may influence the folding of the I-set domain by altering local polarity or side-chain chemistry properties of amino acids, potentially disrupting its antigen-binding capacity in LUAD. Multi-omics analysis revealed that APOB emerged as a candidate oncogene in LIHC, whose genetic alterations within the enrichment region may activate key TFs, upregulate DNA methylation levels, modulate critical histone modifications, and enhance transcriptional activity in the HepG2 and A549 cell lines compared to Panc1. Additionally, CDGS mutation status was an independent prognostic factor for the pan-cancer cohort. High-risk patients tended to develop an immunosuppressive microenvironment and demonstrated a higher likelihood of responding to ICI therapy. Finally, we provided a user-friendly web interface to explore candidate driver genes using geMER (http://bio-bigdata.hrbmu.edu.cn/geMER/).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fe1ede7baa4480b19bb1878aa07140d59ffd789" target='_blank'>
              A core driver gene set identified based on geMER reveals its potential driver mechanism in pan-cancer
              </a>
            </td>
          <td>
            Jing Gan, Yuncong Wang, Zhuoran Shi, Haoyu Hu, Manyi Xu, Xinrong Li, Wenbo Dong, Jiaheng He, Yusen Zhao, Yakun Zhang, Yue Sun, Caiyu Zhang, Qianyi Lu, Shangwei Ning, Yan Jin, Hui Zhi
          </td>
          <td>2025-08-09</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="The growth of a tumor is tightly linked to the distribution of its cells along a continuum of activation states. Here, we systematically decode the activation state architecture (ASA) in a glioblastoma (GBM) patient cohort through comparison to adult murine neural stem cells. Modelling of these data forecasts how tumor cells organize to sustain growth and identifies the rate of activation as the main predictor of growth. Accordingly, patients with a higher quiescence fraction exhibit improved outcomes. Further, DNA methylation arrays enable ASA-related patient stratification. Comparison of healthy and malignant gene expression dynamics reveals dysregulation of the Wnt-antagonist SFRP1 at the quiescence to activation transition. SFRP1 overexpression renders GBM quiescent and increases the overall survival of tumor-bearing mice. Surprisingly, it does so through reprogramming the tumor’s stem-like methylome into an astrocyte-like one. Our findings offer a framework for patient stratification with prognostic value, biomarker identification, and therapeutic avenues to halt GBM progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9dee763118a03533523bd5d7836ea98324ad3594" target='_blank'>
              Cross-species comparison reveals therapeutic vulnerabilities halting glioblastoma progression
              </a>
            </td>
          <td>
            Leo Carl Foerster, Oguzhan Kaya, Valentin Wüst, Diana-Patricia Danciu, Vuslat Akcay, Milica Bekavac, Kevin Chris Ziegler, Nina Stinchcombe, Anna Tang, S. Kleber, Joceyln Tang, Jan Brunken, Irene Lois-Bermejo, Noelia Gesteira-Perez, Xiujian Ma, Ahmed Sadik, P. Le, Kevin Petrecca, Christiane A Opitz, Haikun Liu, C. R. Wirtz, Angela Goncalves, A. Marciniak-Czochra, Simon Anders, A. Martin-Villalba
          </td>
          <td>2025-08-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Background Infants diagnosed with B cell acute lymphoblastic leukemia (B-ALL) and t(4;11) chromosomal rearrangement display poor therapeutic response, associated to the low expression of B lymphocyte factor HDAC7. This study was conceived to identify a therapeutic strategy for t(4;11) B-ALL that restores optimal HDAC7 expression. Methods A multiomics approach in a large infant pro-B-ALL cohort was employed to identify HDAC7’s repression mechanism. These data, combined with cell culture assays in a variety of pro-B-ALL cell lines with differential HDAC7 levels, led us to define a novel combination therapy. Murine leukemia models and ex vivo assays using patient-derived xenografts (PDX) were employed to assess the benefits of this therapy when incorporated to glucocorticoid-based chemotherapy. Results Our data demonstrates that HDAC7 is epigenetically silenced by EZH2 and KMT2A::AFF1 fusion protein. Remarkably, the Menin-1 inhibitor MI-538 restores HDAC7 expression, and the effect is enhanced by class I HDAC inhibitor chidamide. This treatment drives leukemic pro-B cells towards a more differentiated state and impairs aberrant proliferation in an HDAC7-dependent manner. This newly identified therapy increases glucocorticoid sensitivity of PDX cells ex vivo, by repressing RUNX2 transcription factor. Finally, combining MI-538 and chidamide with standard chemotherapy reduces PDX cells engraftment in vivo and delays relapse. Conclusions The combined therapy proposed, based on Menin-1 inhibition, improves t(4;11) B-ALL cells’ response to standard therapy, an effect partially mediated by HDAC7 induction. Therefore, this novel therapy opens a new field for personalized treatments in high-risk leukemia, especially for infants presenting low expression of HDAC7 B cell factor. Supplementary Information The online version contains supplementary material available at 10.1186/s40364-025-00810-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2cbcee9d6f996ab0d44da486acb7d87805ed9357" target='_blank'>
              HDAC7 induction combined with standard-of-care chemotherapy provides a therapeutic advantage in t(4;11) infant B-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            O. de Barrios, Ingrid Ocón-Gabarró, M. Gusi-Vives, O. Collazo, A. Meler, P. A. Romecín, A. Martínez-Moreno, J. R. Tejedor, Mario F Fraga, P. Schneider, M. Bardini, G. Cazzaniga, Rolf Marschalek, Ronald W W Stam, Clara Bueno, Pablo Menéndez, Maribel Parra
          </td>
          <td>2025-07-28</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Chronic myeloid leukemia (CML) is a leading example of a malignancy where a molecular targeted therapy revolutionized treatment but has rarely led to cures. Overcoming tyrosine kinase inhibitor (TKI) drug resistance remains a challenge in the treatment of CML. We have recently identified miR-185 as a predictive biomarker where reduced expression in CD34+ treatment-naïve CML cells was associated with TKI resistance. We have also identified PAK6 as a target gene of miR-185 that was upregulated in CD34+ TKI-nonresponder cells. However, its role in regulating TKI resistance remains largely unknown. In this study, we specifically targeted PAK6 in imatinib (IM)-resistant cells and CD34+ stem/progenitor cells from IM-nonresponders using a lentiviral-mediated PAK6 knockdown strategy. Interestingly, the genetic and pharmacological suppression of PAK6 significantly reduced proliferation and increased apoptosis in TKI-resistant cells. Cell survivability was further diminished when IM was combined with PAK6 knockdown. Importantly, PAK6 inhibition in TKI-resistant cells induced cell cycle arrest in the G2-M phase and cellular senescence, accompanied by increased levels of DNA damage-associated senescence markers. Mechanically, we identified a PAK6-mediated MDM2-p21 axis that regulates cell cycle arrest and senescence. Thus, PAK6 plays a critical role in determining alternative cell fates in leukemic cells, and targeting PAK6 may offer a therapeutic strategy to selectively eradicate TKI-resistant cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d13c94ff00c650649c31950307bc9a23bc8c504" target='_blank'>
              Identification of a PAK6-Mediated MDM2/p21 Axis That Modulates Survival and Cell Cycle Control of Drug-Resistant Stem/Progenitor Cells in Chronic Myeloid Leukemia
              </a>
            </td>
          <td>
            Andrew Wu, Min Chen, Athena Phoa, Zesong Yang, Donna L. Forrest, Xiaoyan Jiang
          </td>
          <td>2025-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2944900d47d857ea99c57a79dff86ef949dfd7c" target='_blank'>
              Genomic alterations in the YAP/TAZ pathway are associated with stem cell-like castration-resistant prostate cancer
              </a>
            </td>
          <td>
            M. Roskes, A. Martínez-Fundichely, S. Cohen, M. Balaban, C. K. Wong, W. Li, T. A. Gonzalez, A. Tehim, H. Xu, S. ElNaggar, M. Myers, R. Bareja, P. Dorsaint, K. Gorski, M. Asad, M. A. Assaad, B. D. Robinson, M. Sigouros, E. Barnett, J. Manohar, S. Tagawa, D. Nanus, A. Molina, J. Nauseef, C. N. Sternberg, J. Mosquera, H. Scher, A. Sboner, B. Raphael, Y. Chen, E. Khurana
          </td>
          <td>2025-07-02</td>
          <td>medRxiv : the preprint server for health sciences</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Background Hepatocellular carcinoma accounts for 90% of primary liver malignancies and ranks as the fourth leading cause of global cancer mortality. DNA methylation drives HCC pathogenesis by orchestrating tumor suppressor silencing and oncogenic pathway activation, fundamentally governing tumor heterogeneity and therapeutic resistance. While alterations in white blood cell counts have emerged as independent prognostic indicators in HCC progression, their causal relationship with tumor epigenetics remains unclear. Given that HCC tumors actively secrete cytokines and remodel systemic immunity through epigenetic mechanisms, peripheral white blood cell dynamics serve as a compelling surrogate for studying tumor-driven immune dysregulation and identifying methylation-based therapeutic targets. Methods We integrated DNA methylation profiles from The Cancer Genome Atlas-Liver Hepatocellular Carcinoma (TCGA-LIHC) with summary statistics from six genome-wide association studies of white blood cell counts. Using these data, we applied a two-sample Mendelian Randomization (MR) framework to systematically investigate the causal effects of HCC-related CpG sites on white blood cell counts. The robustness of our findings was established through stringent criteria, comprehensive sensitivity analyses, and Bayesian colocalization, followed by functional validation with TCGA data and a transcriptome-wide association study (TWAS). Results Our analysis identified 26 HCC-specific CpG sites with causal effects on white blood cell counts. A multi-step validation pipeline integrating Bayesian colocalization and tumor-specific expression analysis robustly confirmed three core mediator genes: BTN3A2, TRIM27, and S100A12. TWAS provided independent, gene-level validation, revealing BTN3A2 as a powerful regulator of lymphocyte and neutrophil counts, TRIM27 as a key immunometabolic checkpoint, and S100A12 as a modulator of innate immunity. Conclusion This study establishes epigenome-immune crosstalk in HCC by identifying BTN3A2, S100A12 and TRIM27 as central methylation-immunoregulatory hubs. The discovered methylation-immune axes provide mechanistic insights for developing combination therapies targeting epigenetic checkpoints to reverse immunosuppressive microenvironments in advanced HCC. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-14693-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/282ecf09d34731818d82e7690045faf501ada4b4" target='_blank'>
              Decoding the epigenetic-immune nexus in hepatocellular carcinoma: a Mendelian randomization study reveals BTN3A2, S100A12 and TRIM27 as white blood cell regulators
              </a>
            </td>
          <td>
            Yang Qiu, Haiyan Zhang, Xiao Yu, Jielu Pan, Dinghong Xiao, Hongyu Miao, Ruiqing Wang, Nan Shen, Peimin Pu, Yinling Zhou, Zhidong Liu, Lin He, Lianjun Xing
          </td>
          <td>2025-08-08</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Cell type-specific transcriptional heterogeneity in embryonic mouse skin is well-documented, but few studies have investigated the regulatory mechanisms. Here, we present high-throughput single-cell chromatin accessibility and transcriptome sequencing (HT-scCAT-seq), a method that simultaneously profiles transcriptome and chromatin accessibility. We utilized HT-scCAT-seq to dissect the gene regulatory mechanism governing epidermal stratification, periderm terminal differentiation, and fibroblast specification. Results By linking chromatin accessibility to gene expression, we identify candidate cis-regulatory elements (cCREs) and their target genes which are crucial for dermal and epidermal development. We describe cells with similar gene expression profiles that exhibit distinct chromatin accessibility statuses during periderm terminal differentiation. Finally, we characterize the underlying lineage-determining transcription factors and demonstrate that ALX4 and RUNX2 are candidate transcription factors regulators of the dermal papilla lineage development through in silico perturbation analysis and CUT&Tag experiment. Conclusions Overall, HT-scCAT-seq represents a powerful tool for unraveling the spatiotemporal dynamics of gene regulation in single cells. Our results advance the understanding of embryonic skin development while providing a scalable framework for investigating regulatory mechanisms across diverse biological systems and disease contexts. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03652-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/747c145f394c82a8a3bbf273f6afa4e007611092" target='_blank'>
              Deciphering gene regulatory programs in mouse embryonic skin through single-cell multiomics analysis
              </a>
            </td>
          <td>
            Q. Deng, Pengfei Cai, Yingjie Luo, Zhongjin Zhang, Wen Ma, Zijie Huang, Xiaoya Chen, Shijie Hao, Weiguang Ma, Jiangshan Xu, Chunqing Wang, Mengnan Cheng, Xiumei Lin, Ru Zhou, Shanshan Duan, Junjie Chen, Ronghai Li, Xuyang Shi, Chang Liu, Liang Wu, Peng Gao, Jianting Li, Xiao Yang, Xiangdong Wang, Jun Xie, Longqi Liu, Yue Yuan, Chuanyu Liu
          </td>
          <td>2025-07-01</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Female cancers such as breast and gynaecological cancers contribute to a significant global health burden and are a leading cause of fatality among women. With current treatment options often limited by resistance to cytotoxic drugs, side effects and lack of specificity to the cancer, there is a pressing need for alternative treatments. Recent research has highlighted the promising role of non-coding RNAs (ncRNA) in regulating these issues and providing more targeted approaches to suppressing key cancer pathways. This review explores the involvement of the various types of non-coding RNAs in regulating key oncogenic pathways, namely, the MAPK, PI3K/Akt/mTOR, Wnt/β-catenin and p53 pathways, in a range of female cancers such as breast, cervical, ovarian and endometrial cancers. Evidence from a multitude of studies suggests that non-coding RNAs function as double-edged swords, serving as both oncogenes and tumour suppressors, depending on their expression and cellular interactions. By mapping and investigating these regulatory interactions, this review demonstrates the complexity and dual functionality of ncRNAs in cancer. Understanding these complex mechanisms is essential for the development of new and effective ncRNA-based diagnostic methods and targeted therapies in female cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4edcace8dce2041819a34f6be3b0322cc08e2907" target='_blank'>
              The Role of Non-Coding RNAs in the Regulation of Oncogenic Pathways in Breast and Gynaecological Cancers
              </a>
            </td>
          <td>
            Ammar Ansari, Aleksandra Szczesnowska, Natalia Haddad, A. Elbediwy, Nadine Wehida
          </td>
          <td>2025-08-06</td>
          <td>Non-Coding RNA</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Background RNA sequencing (RNA-seq) is emerging as a valuable tool for identifying disease-causing RNA transcript aberrations that cannot be identified by DNA-based testing alone. Previous studies demonstrated some success in utilizing RNA-seq as a first-line test for rare inborn genetic conditions. However, DNA-based testing (increasingly, whole genome sequencing) remains the standard initial testing approach in clinical practice. The indications for RNA-seq after a patient has undergone DNA-based sequencing remain poorly defined, which hinders broad implementation and funding/reimbursement. Methods In this study, we identified four specific and familiar clinical scenarios, and investigated in each the diagnostic utility of RNA-seq on clinically accessible tissues: (i) clarifying the impact of putative intronic or exonic splice variants (outside of the canonical splice sites), (ii) evaluating canonical splice site variants in patients with atypical phenotypes, (iii) defining the impact of an intragenic copy number variation on gene expression, and (iv) assessing variants within regulatory elements and genic untranslated regions. Results These hypothesis-driven RNA-seq analyses confirmed a molecular diagnosis and pathomechanism for 45% of participants with a candidate variant, provided supportive evidence for a DNA finding for another 21%, and allowed us to exclude a candidate DNA variant for an additional 24%. We generated evidence that supports two novel Mendelian gene-disease associations (caused by variants in PPP1R2 and MED14) and several new disease mechanisms, including the following: (1) a splice isoform switch due to a non-coding variant in NFU1, (2) complete allele skew from a transcriptional start site variant in IDUA, and (3) evidence of a germline gene fusion of MAMLD1-BEND2. In contrast, RNA-seq in individuals with suspected rare inborn genetic conditions and negative whole genome sequencing yielded only a single new potential diagnostic finding. Conclusions In summary, RNA-seq had high diagnostic utility as an ancillary test across specific real-world clinical scenarios. The findings also underscore the ability of RNA-seq to reveal novel disease mechanisms relevant to diagnostics and treatment. Supplementary Information The online version contains supplementary material available at 10.1186/s13073-025-01494-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b1a4792ea15366ba9539ebb29120d269348c3bb" target='_blank'>
              Clinical applications of and molecular insights from RNA sequencing in a rare disease cohort
              </a>
            </td>
          <td>
            Jamie C. Stark, Neta Pipko, Yijing Liang, Anna Szuto, Chung Ting Tsoi, Megan A. Dickson, Kyoko E. Yuki, Huayun Hou, Sydney Scholten, Kenzie Pulsifer, Meryl Acker, Meredith Laver, Harsha Murthy, Olivia M. Moran, Emily Bonnell, Nicole S. Y. Liang, Jashanpreet Sidhu, Lucie Dupuis, Mohammad M. Ghahramani Seno, M. Chard, R. Jobling, Jessie Cameron, Rose Chami, M. Inbar‐Feigenberg, Michael D. Wilson, David Chitayat, K. Boycott, L. Kyriakopoulou, Roberto Mendoza-Londono, Christian R. Marshall, J. J. Dowling, Gregory Costain, A. Deshwar
          </td>
          <td>2025-07-01</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="A major goal in evolutionary biology and biomedicine is to understand the complex interactions between genetic variants, the epigenome, and gene expression. However, the causal relationships between these factors remain poorly understood. mSTARR-seq, a methylation-sensitive massively parallel reporter assay, is capable of identifying methylation-dependent regulatory activity at many thousands of genomic regions simultaneously and allows for the testing of causal relationships between DNA methylation and gene expression on a region-by-region basis. Here, we develop a multiplexed mSTARR-seq protocol to assay naturally occurring human genetic variation from 25 individuals from 10 localities in Europe and Africa. We identify 6957 regulatory elements in either the unmethylated or methylated state, and this set was enriched for enhancer and promoter chromatin annotations, as expected. The expression of 58% of these regulatory elements is modulated by methylation, which is generally associated with decreased transcription. Within our set of regulatory elements, we use allele-specific expression analyses to identify 8020 sites with genetic effects on gene regulation; further, we find that 42.3% of these genetic effects vary in direction or magnitude between methylated and unmethylated states. Sites exhibiting methylation-dependent genetic effects are enriched for GWAS and EWAS annotations, implicating them in human disease. Compared with data sets that assay DNA from a single European ancestry individual, our multiplexed assay is able to uncover more genetic effects and methylation-dependent genetic effects, highlighting the importance of including diverse genomes in assays that aim to understand gene regulatory processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d8c0aba589430fda6268dbe62323f10c40bdeec" target='_blank'>
              Uncovering methylation-dependent genetic effects on regulatory element function in diverse genomes
              </a>
            </td>
          <td>
            Rachel M. Petersen, Christopher M. Vockley, Amanda J. Lea
          </td>
          <td>2025-07-14</td>
          <td>Genome Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background Kirsten Rat Sarcoma (KRAS) copy number amplification has emerged as an oncogenic driver in colorectal cancer (CRC), in addition to KRAS mutation. However, its clinical significance remains poorly understood. Notably, CRC patients with wild-type KRAS but harboring KRAS amplification have shown resistance to anti-EGFR therapy, representing an unmet clinical need. Methods In this study, we comprehensively investigated the impact of KRAS amplification—alone and in conjunction with mutation—on clinicopathological characteristics, immune infiltration, and therapeutic response. KRAS copy number variation (CNV) was classified into amplification and non-amplification groups. KRAS mutational status was determined as wild-type (WT) or mutant (MUT) using qPCR and Sanger sequencing. CD8⁺ T lymphocyte infiltration was evaluated by immunohistochemistry in resected CRC specimens. Clinical, immune, and survival data were analyzed in association with KRAS status. RNA-seq was performed to identify differentially expressed genes (DEGs) and enriched pathways. Patient-derived xenograft (PDX) models were used to assess responses to targeted therapies. Results We found that KRAS amplification was more frequent in WT KRAS CRC (21.4%) than in MUT KRAS CRC (6.5%). In the WT subgroup, KRAS amplification was associated with poor prognosis and increased KRAS protein expression and downstream pathway activation. In contrast, amplification had little effect in mutant KRAS tumors. KRAS copy number was inversely correlated with CD8⁺ T cell infiltration, suggesting a role in immune evasion. Importantly, low CD8⁺ T cell density combined with KRAS amplification predicted adverse outcomes. Therapeutically, KRAS-amplified PDX models were resistant to anti-EGFR treatment. However, combined MEK and CDK4/6 inhibition overcame this resistance. Conclusions KRAS amplification constitutes a distinct oncogenic driver in CRC and a potential biomarker for therapeutic stratification. Our findings support the clinical rationale for dual MEK and CDK4/6 inhibition in KRAS-amplified CRC and highlight the need for biomarker-guided clinical trials to optimize treatment strategies in this subset of patients. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06864-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd01174f41ba37a6094968fc4a8b39ec9e5e025b" target='_blank'>
              KRAS amplification in colorectal cancer: correlations with clinicopathological features and prognosis in patients and prediction of response to targeted therapy
              </a>
            </td>
          <td>
            Xuanhua Yang, Shiwei Feng, Qi Zhang, Wenxin Li, Shujuan Li, Min Zhi, Ping Lan, Sijing Cheng
          </td>
          <td>2025-07-24</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Metastases are the leading cause of cancer-related deaths. The spread of neoplasms involves multiple mechanisms, with metastatic tumors exhibiting molecular behaviors distinct from their primary counterparts. The key hallmarks of metastatic lesions include chromosomal instability, copy number alterations (CNAs), and a reduced degree of subclonality. Furthermore, metabolic adaptations such as enhanced glycogen synthesis and storage, as well as increased fatty acid oxidation (FAO), play a critical role in sustaining energy supply in metastases and contributing to chemoresistance. FAO promotes the infiltration of macrophages into the tumor, where they polarize to the M2 phenotype, which is associated with immune suppression and tissue remodeling. Additionally, the tumor microbiome and the action of cytotoxic drugs trigger neutrophil extravasation through inflammatory pathways. Chemoresistant neutrophils in the tumor microenvironment can suppress effector lymphocyte activation and facilitate the formation of neutrophil extracellular traps (NETs), which are linked to drug resistance. This article examines the genomic features of metastatic tumors, along with the metabolic and immunological dynamics within the metastatic tumor microenvironment, and their contribution to drug resistance. It also discusses the molecular mechanisms underlying resistance to chemotherapeutic agents commonly used in the treatment of metastatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7adea99fe5f3be60db7314e38e07d8d5a4381e12" target='_blank'>
              Analyzing the Blueprint: Exploring the Molecular Profile of Metastasis and Therapeutic Resistance
              </a>
            </td>
          <td>
            G. Avalos-Navarro, M. Gallegos-Arreola, Emmanuel Reyes-Uribe, Luis Felipe Jave Suárez, Gildardo Rivera-Sánchez, Héctor Rangel-Villalobos, Ana Luisa Madriz-Elisondo, Itzae Adonai Gutierrez Hurtado, J. J. Varela-Hernández, R. Ramírez-Patiño
          </td>
          <td>2025-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Background Chronic myeloid leukemia (CML) is a myeloproliferative neoplasm characterized by the presence of the Philadelphia chromosome (chromosome 22). This cytogenetic abnormality gives rise to the BCR::ABL1 fusion gene, which encodes the constitutively active BCR-ABL1 protein tyrosine kinase, driving uncontrolled proliferation and impaired apoptosis of hematopoietic stem and progenitor cells, leading to leukemogenesis. Imatinib mesylate (IM), a first-generation tyrosine kinase inhibitor (TKI) specifically targeting the BCR-ABL1 oncoprotein, represents the standard first-line therapy for patients with CML. However, imatinib resistance remains a major therapeutic challenge. Objective This study aims to elucidate the role of the ZFAS1/STAT3 signaling axis in mediating imatinib resistance in CML by promoting metabolic reprogramming, with a particular focus on alterations in glucose metabolism. Methods Imatinib-resistant (IM-R) K562 cells were used to investigate the functional role of ZFAS1gene. Following ZFAS1 knockdown, assessments of cell viability, apoptosis, and glucose metabolism were performed. The interaction between ZFAS1 and IGF2BP2, as well as its regulatory effect on STAT3 expression and glycolysis-related genes (including HIF1α, LDHA, and PDK1) were examined using qRT-PCR and western blotting. Additionally, the impact of STAT3 overexpression and glycolysis inhibition (2-DG) on IM sensitivity were examined. Results Our findings revealed that ZFAS1 expression was significantly upregulated in IM-R CML patient samples and IM-R K562 cells. Silencing of ZFAS1 enhanced cellular sensitivity to IM, inhibited glucose metabolism reprogramming, and promoted apoptosis. Mechanistically, ZFAS1 was found to interact with IGF2BP2, facilitating the stabilization of STAT3 mRNA and leading to increased STAT3 expression. This, in turn, resulted in the upregulation of key glycolytic genes. Overexpression of STAT3 reversed the effects of ZFAS1 knockdown by restoring glycolytic activity and re-establishing IM resistance. Additionally, 2-DG treatment effectively reversed STAT3-induced IM resistance by inhibiting glycolysis. Conclusion These findings demonstrate that the ZFAS1/STAT3 signaling axis contributes to imatinib resistance in CML through the modulation of glucose metabolism. Targeting this regulatory pathway may represent a novel therapeutic strategy to overcome TKI resistance in CML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/773f57bc005e9ca8395d33d3e07201c0882f8437" target='_blank'>
              ZFAS1/STAT3 axis modulates imatinib resistance of chronic myeloid leukemia cells through glucose metabolism reprogramming
              </a>
            </td>
          <td>
            Lan Yang, Yanqiu Han
          </td>
          <td>2025-07-08</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Breast cancer remains one of the leading causes of cancer-related morbidity and mortality among women worldwide. While mutations in BRCA1 and BRCA2 have long been recognized as key drivers of hereditary breast cancer, recent advances in genomic research have uncovered a growing set of additional genes such as PALB2, CHEK2, ATM, FANCM, and RAD51C—that contribute to breast cancer susceptibility. These genes are involved in critical molecular pathways, including homologous recombination DNA repair, cell cycle checkpoint control, apoptosis, and tumor suppression. Dysfunction in these pathways, often triggered by germline mutations, can lead to genomic instability, uncontrolled proliferation, and tumorigenesis. This review provides a comprehensive analysis of these emerging genetic factors, detailing their mechanisms of action, pathway interactions (e.g., BRCA–PALB2–RAD51 axis, ATM–CHEK2–p53 checkpoint signaling), and their impact on both familial and sporadic forms of breast cancer. We also discuss the clinical relevance of integrating multi-gene panels and polygenic risk scores (PRS) into personalized risk assessment and prevention strategies. By highlighting the molecular pathways disrupted by these gene mutations, we emphasize the importance of expanding genetic testing beyond BRCA1/2. A deeper understanding of these pathways not only refines risk stratification but also opens new avenues for targeted therapy, early detection, and tailored clinical management of breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d949c73ab2411bd172c49fda9d693c1d37e0094" target='_blank'>
              Unraveling emerging genes and their functional pathways in breast cancer susceptibility and progression
              </a>
            </td>
          <td>

          </td>
          <td>2025-07-07</td>
          <td>International Journal of Biosciences (IJB)</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Epithelial ovarian cancer (EOC), accounting for 90–95% of all ovarian cancer (OC) cases, is the most lethal gynaecological malignancy, primarily due to late-stage diagnosis and the development of chemoresistance. While initial responses to Platinum- and Taxane-based chemotherapy are favorable, nearly 70% of patients relapse within five years. Although signaling pathways such as PI3K/AKT, MAPK, NF-κB, Notch, and Wnt/β-catenin have been individually studied in the context of chemoresistance, recent evidence highlights the importance of dynamic feedback loops and crosstalk among these networks in sustaining the resistant phenotype. Moreover, dysregulated microRNAs (miRNAs), as post-transcriptional regulators, fine-tune these pathways, creating self-sustaining circuits that promote drug efflux, inhibit apoptosis, and maintain cancer stemness. Reciprocal regulation between miRNAs and signaling components establishes robust networks that amplify chemoresistant phenotypes. The review provides a comprehensive overview of the molecular mechanisms driving chemoresistance, emphasising critical elements of signalling pathways and associated miRNAs that contribute to resistance and may function as biomarkers or therapeutic targets to mitigate chemoresistance. To improve clinical outcomes, future research should focus on identifying resistance-associated miRNA signatures and targeting nodal points within miRNA-signaling networks, thereby enabling the development of personalized therapies to overcome drug resistance in EOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ed5645bca47c8ea9b29c49088a162133eb18020" target='_blank'>
              Signaling networks and MiRNA crosstalk in ovarian cancer chemoresistance
              </a>
            </td>
          <td>
            K. Sreedhara, Ranganath Pai, Raksha Nayak, Samyak Pandey, Dileep Kumar, Sachindra Kumar
          </td>
          <td>2025-08-14</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cancerous transcriptome alterations in carcinoma cells could be originated from either genetic copy number changes or epigenetic reprogramming. Ovarian cancer (OV) is the most malignant gynecologic tumor, known for high aneuploidy with robust copy number alterations. However, low aneuploidy ovarian tumors are also frequently found, indicating an essential contribution of epigenetic factors during tumorigenesis and cancer development. Chromatin remodeling modulates the transcriptome epigenetically in a variety of cancer types, but its role in OV is still unclear. Hence, we investigated a cohort of 102 OV patients, analyzed transcriptomic and clinical data from public databases, and performed cellular experiments. We found that RUVBL2, a subunit of the INO80 complex, functions as the key oncogenic chromatin remodeler in OV. RUVBL2 is upregulated in tumors, particularly in low-aneuploidy cases, and is associated with poor prognosis. RUVBL2 drives nucleosome dynamics and elevates chromatin accessibility selectively at promoter regions. The landscape of RUVBL2-dependent modulation of chromatin accessibility and the transcriptome exhibits activation of various transcription factors, especially the AP-1 family, and upregulation of a series of key genes, including CDKN3, MYBL2, and ZNF144, resulting in mediation of cell cycle and Hippo signaling pathway to promote DNA synthesis and cell proliferation. Hence, RUVBL2-dependent chromatin remodeling plays a key role in oncogenic reprogramming of the transcriptome in OV. These findings provide novel insights into the molecular etiology of OV and disclose potential biomarkers and drug targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b2407b3becd0c884fdbbc4e31bad36bc2c5c367" target='_blank'>
              Profiling of RUVBL2-Induced Transcriptome Alterations Highlights a Critical Role for Chromatin Remodeling in Ovarian Cancer.
              </a>
            </td>
          <td>
            Renhao Xue, Yingjie Wang, Xiaomei Luo, Hao Zhang, Dongcheng Guan, Shuo Shi, Yu Wang
          </td>
          <td>2025-07-01</td>
          <td>BioFactors</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Next-generation sequencing (NGS) technologies have revolutionized research and precision medicine in patients with cancer. Progress in this area has been accompanied by the development of efficient and robust bioinformatics methods along with computational resources able to handle the growing amount and complexity of sequencing data. Importantly, the implementation of such approaches has not been uniform around the globe and several regions, including Latin American countries, remain lagging behind in cancer genomics and precision oncology. Likewise, numerous studies have highlighted the complexity and particularities of such populations in terms of genetic background, healthcare systems and human and technological resources. In this review, we aim to describe current clinical applications of NGS-based tests, focusing on their bioinformatics analyses and implementation in Latin America. Furthermore, we describe several opportunities for development, perspectives, and challenges that face genomic data analysis in this geographical area. We expect this review to provide an up-to-date overview of cancer genomics and bioinformatics in Latin America, serving as a valuable resource for both local and international cancer researchers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/262cb50e2c99a11aa10cf456b1bf1a0a0086c5f7" target='_blank'>
              Cancer genomics and bioinformatics in Latin American countries: applications, challenges, and perspectives
              </a>
            </td>
          <td>
            E. S. Torres-Narváez, D. Mendivelso-González, Juan Andrés Artunduaga-Alvarado, Oscar Ortega-Recalde
          </td>
          <td>2025-07-09</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="ABSTRACT Background and Aim Historical genetic sequencing of specific cancer variants has been superseded by comprehensive genomic profiling (CGP). This narrative review aimed to capture current international evidence on the clinical utility of CGP for cancer prevention, detection and treatment. Materials and Methods A literature search of three databases was performed to identify key studies on the frequency of germline and somatic variants in adult cancers and the extent to which they inform diagnosis, management and outcome. Findings were inductively mapped and narratively synthesised. Results Consolidated results from 95 original research papers showed that pathogenic germline (familial) variants are found in ~10% of adults with cancer, of whom 53%–61% are offered germline genotype‐directed treatment. Importantly, 50% of germline carriers would not have satisfied the eligibility criteria for genetic testing and/or reported a negative family history. Actionable somatic variants occur in 27%–88% of cases, which markedly impact the diagnosis for cancers of unknown primary. Matched treatments were identified for 31%–48% of cancer patients, of whom 33%–45% received it. Response and survival rates were better in individuals receiving matched therapies compared to those receiving standard of care or unmatched therapies. Trials show that circulating tumour DNA (ctDNA) assays are feasible and sensitive. The relatively non‐invasive ctDNA sample collection is appealing for cancers with inaccessible or unknown primary sites, and serial monitoring of residual disease and/or treatment response. Conclusions As matched therapies are underutilised due to declining patient condition and fewer prior therapies predicting better response rates, research is needed on the suitability of cancer genomic profiling as a frontline test.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fedb0d01f281e00aec9cacae6c6620746559f77" target='_blank'>
              Utility of Germline, Somatic and ctDNA Testing in Adults With Cancer
              </a>
            </td>
          <td>
            E. DeBortoli, Ella McGahan, Tatiane Yanes, Jennifer Berkman, L. Aoude, A. K. Smit, Akira Gokoolparsadh, Azure Hermes, Lyndsay Newett, Mackenzie Bourke, Susan Hanson, Helen Hughes, Oliver Hofmann, I. Goranitis, Rebekah McWhirter, Vivienne Milch, Julia Steinberg, A. McInerney-Leo
          </td>
          <td>2025-08-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Background Lactate-driven metabolic reprogramming and histone lactylation play pivotal roles in bladder cancer (BLCA) progression, yet their underlying mechanisms and regulatory genes remain poorly understood. Methods Using transcriptomic data from The Cancer Genome Atlas (TCGA), we identified lactylation-associated genes and constructed a prognostic signature. Comprehensive bioinformatics analyses were conducted to assess immune infiltration, tumor microenvironment characteristics, and the lactylation landscape at the single-cell level. Furthermore, we performed in vitro experiments to evaluate the biological functions of key lactylation-related genes in BLCA cells. Results Six lactylation-related hub genes were identified, among which FASN and RUNX2 were significantly upregulated in BLCA and associated with poor prognosis. Single-cell analyses revealed elevated lactylation signatures in tumor epithelial and immune cells. Knockdown of FASN or RUNX2 in BLCA cell lines significantly suppressed cell proliferation, induced apoptosis, and reduced intracellular lactate levels. Correspondingly, global protein lactylation was diminished, with dominant modification signals observed around 40 kDa, indicating a potential set of non-histone proteins as key functional targets. Conclusions Our study highlights a metabolic-enzymatic axis wherein FASN and RUNX2 regulate lactate-driven protein lactylation in BLCA. These findings provide new insights into the non-histone functions of lactylation and suggest potential therapeutic targets at the intersection of metabolism and tumor immunity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/289e32e91ef0307999ca340970cf0b389dea75fa" target='_blank'>
              Identification of lactylation-associated immune and metabolic regulators in bladder cancer via integrated bulk and single-cell transcriptomics
              </a>
            </td>
          <td>
            Yanjun Chen, Yu Sun, Xiaoyan Liu
          </td>
          <td>2025-07-09</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="With age, clonal expansions occur pervasively across normal tissues yet only in rare instances lead to cancer, despite being driven by well-established cancer drivers. Characterization of the factors that influence clonal progression is needed to inform interventional approaches. Germline genetic variation influences cancer risk and shapes tumor mutational profile, but its influence on the mutational landscape of normal tissues is not well known. Here we studied the impact of germline genetic variation on clonal hematopoiesis (CH) in 731,835 individuals. We identified 22 new CH-predisposition genes, most of which predispose to CH driven by specific mutational events. CH-predisposition genes contribute to unique somatic landscapes, reflecting the influence of germline genetic backdrop on gene-specific CH fitness. Correspondingly, somatic–germline interactions influence the risk of CH progression to hematologic malignancies. These results demonstrate that germline genetic variation influences somatic evolution in the blood, findings that likely extend to other tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bcdcfe442326883ec1ff1d1b7880290f079cb82" target='_blank'>
              Germline genetic variation impacts clonal hematopoiesis landscape and progression to malignancy
              </a>
            </td>
          <td>
            Jie Liu, Duc Tran, Liying Xue, B. Wiley, C. Vlasschaert, Caroline J. Watson, Hamish A J MacGregor, X. Zong, I. Chan, Indraniel Das, M. Uddin, A. Niroula, Gabriel Griffin, Benjamin L Ebert, T. Mack, Y. Pershad, B. Sharber, Michael F. Berger, A. Zehir, R. Ptashkin, Ross L. Levine, E. Papaemmanuil, V. Joseph, Teng Gao, Y. Kemel, D. Mandelker, Konrad H Stopsack, Paul D. P. Pharoah, Semanti Mukherjee, Li Ding, Yinyin Cao, MJ Walter, Jamie R Blundell, Nilanjan Chatterjee, K. Offit, Lucy A. Godley, Daniel C. Link, Z. Stadler, Alexander G. Bick, P. Natarajan, Kelly L Bolton
          </td>
          <td>2025-07-15</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>132</td>
        </tr>

        <tr id="Precision medicine is a likely future for all cancer treatment, but may have its greatest impact for less common, high-mortality and molecularly heterogeneous cancers. TFCP2-rearranged Rhabdomyosarcoma is a rare, aggressive cancer with poor survival due to a lack of effective therapies, as well as relevant models to facilitate research. Here, we establish the first matched patient-derived xenograft and cell line model for TFCP2-rearranged intraosseous Rhabdomyosarcoma (IORMS), coupled with comprehensive multi-omic and functional analysis, to discover and preclinically validate novel actionable molecular targets for this malignancy. Sequencing analyses of matched patient tumor and xenograft material revealed alterations in gene networks associated with the oncogenic, potentially targetable PI3K/AKT pathway. Preclinical assessments revealed that targeting the pathway with a small molecule PI3K/mTOR inhibitor dactolisib presents a promising treatment approach for this rare cancer, decreasing cancer cell viability in vitro and significantly reducing tumor growth in vivo. Parallel identification of the codeletion of adjacent genes CDKN2A and MTAP in these tumors, led us to further explore PRMT5 inhibition as a potential therapeutic approach. Strikingly, combined inhibition of PRMT5 and PI3K/mTOR signaling synergistically enhanced anti-tumor response and significantly improved survival in vivo. This study highlights the importance of new patient-derived models for the elucidation of the biology of rare cancers, and identification of new therapeutic entry points, with clear implications for the future treatment of TFCP2-rearranged IORMS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7665541fdcead6fb0f6ce32aaae8798477a5fc9a" target='_blank'>
              Integration of whole genome sequencing analysis with unique patient-derived models reveals clinically relevant drug targets in TFCP2 fusion-defined rhabdomyosarcoma.
              </a>
            </td>
          <td>
            Patrick Bergsma, Sean Porazinski, Aji Istadi, Diego Chacon-Fajardo, Yasir Mahmood, Silvia Lombardi, Diana Schuhmacher, Henry Barraclough-Franks, D. Strbenac, Claude Dennis, Payam Faizi-Sobbi, Emer Cahill, V. Bhadri, Jeneffer De Almeida Silva, T. Lum, J. Wykes, T. Manzie, C. Palme, J. Yang, Jonathan R. Clark, Ruta Gupta, M. Pajic
          </td>
          <td>2025-07-07</td>
          <td>Molecular cancer therapeutics</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Tumor protein p53 target gene 1 (TP53TG1) is a significant long non-coding RNA (lncRNA) located on human chromosome 7q21.12, exhibiting dysregulated expression in several cancer tissues, and operates both as an oncogene and a tumor suppressor. TP53TG1 modulates gene expression by several mechanisms, including “competitive RNA” activity, competitive miRNA binding, control of critical signaling pathways, and facilitation of epigenetic changes. TP53TG1 affects tumor proliferation, migration, invasion, and apoptosis through intricate mechanisms, modifies tumor cells resistance to therapies, influences the tumor immune microenvironment and metabolic reprogramming, and may present various avenues for clinical therapeutic intervention. Notwithstanding considerable advancements in comprehending the molecular function of TP53TG1, some challenges persist, especially regarding the efficient and targeted administration of TP53TG1-based therapeutics. Advanced RNA delivery technologies, including lipid nanoparticles (LNPs), sphingomyelin-based nanosystems, and polymer-based nanocarriers, have demonstrated the capacity to surmount these obstacles. This review aims to clarify the expression pattern of TP53TG1 in 10 malignant tumors, its biological relevance, its potential as a therapeutic target, and its importance in clinical applications, thereby offering a novel perspective on molecular targeting in cancer. In the future, integrating TP53TG1 with conventional medications may augment efficacy, offer personalized treatments, and eventually increase the prognosis for patients with aggressive or drug-resistant cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f226208fa68047280df52857f7f84b24287ce57b" target='_blank'>
              Targeting TP53TG1: a promising prognostic biomarker and therapeutic target for personalized cancer therapy
              </a>
            </td>
          <td>
            Haodong He
          </td>
          <td>2025-07-07</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background: Colorectal cancer (CRC) is a prevalent gastrointestinal malignancy, ranking third in incidence and second in cancer-related mortality. Despite therapeutic advances, challenges such as chemotherapy toxicity and drug resistance persist. Thus, there is an urgent need for novel CRC treatments. However, developing new drugs is time-consuming and resource-intensive. As a more efficient approach, drug repurposing offers a promising alternative for discovering new therapies. Methods: In this study, we screened 1068 small molecular compounds from an FDA-approved drug library in CRC cells. Menadione was selected for further study based on its activity profile. Mechanistic analysis included a cell death pathway PCR array, differential gene expression, enrichment, and network analysis. Gene expressions were validated by RT-qPCR. Results: We identified menadione as a potent anti-tumor drug. Menadione induced three programmed cell death (PCD) signaling pathways: necroptosis, apoptosis, and autophagy. Furthermore, we found that the anti-tumor effect induced by menadione in CRC cells was mediated through a key gene: MAPK8. Conclusions: By employing methods of cell biology, molecular biology, and bioinformatics, we conclude that menadione can induce multiple forms of PCD in CRC cells by activating MAPK8, providing a foundation for repurposing the “new use” of the “old drug” menadione in CRC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8523c733477ecc5f9256ddf23b8ee799e99619d" target='_blank'>
              Screening of an FDA-Approved Drug Library: Menadione Induces Multiple Forms of Programmed Cell Death in Colorectal Cancer Cells via MAPK8 Cascades
              </a>
            </td>
          <td>
            Liyuan Cao, Weiwei Song, Jinli Sun, Yang Ge, Wei Mu, Lei Li
          </td>
          <td>2025-07-31</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Lesch–Nyhan disease (LND) is associated with a complete deficiency of hypoxanthine-guanine phosphoribosyltransferase (HPRT) activity due to mutations in the HPRT1 gene. Although the physiopathology of LND-related neurological manifestations remains unknown, a defective neuronal developmental process is the most widely accepted hypothesis. We generated an HPRT-deficient line from the pluripotent human embryonic cell line NT2/D1 by CRISPR-Cas9 and induced its differentiation along neuroectodermal lineages by retinoic acid treatment. As levels of folic acid in the culture media may affect results in LND models, we employed physiological levels of folate. The effect of HPRT deficiency on neural development-related gene expression was evaluated using two methodological approaches: a directed qPCR array of genes related to neuronal differentiation, and global gene expression by RNAseq. HPRT-deficient pluripotent cells presented altered expression of genes related to pluripotency in human embryonic stem cells, such as DPPA3 and CFAP95, along with genes of the homeobox gene family. HPRT-deficient pluripotent cells were able to differentiate along neuro-ectodermal lineages but presented consistent dysregulation of several genes from the homeobox gene family, including EN1 and LMX1A. GO enrichment analysis of up- and downregulated genes in HPRT-deficient cells showed that the most significant biological processes affected are related to development and nervous system development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c9428cd1b573671431fe9d600913d73a1c68291" target='_blank'>
              Gene Expression Analysis of HPRT-Deficient Cells Maintained with Physiological Levels of Folic Acid
              </a>
            </td>
          <td>
            Rosa J Torres, Gerard Valentines-Casas, Claudia Cano-Estrada, Neus Ontiveros, José M López
          </td>
          <td>2025-07-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Esophageal adenocarcinoma (EAC) is a highly lethal cancer of the upper gastrointestinal tract with rising incidence in western populations. To decipher EAC disease progression and therapeutic response, we performed multiomic analyses of a cohort of primary and metastatic EAC tumors, incorporating single-nuclei transcriptomic and chromatin accessibility sequencing, along with spatial profiling. We recovered tumor microenvironmental features previously described to associate with therapy response. We subsequently identified five malignant cell programs, including undifferentiated, intermediate, differentiated, epithelial-to-mesenchymal transition, and cycling programs, which were associated with differential epigenetic plasticity and clinical outcomes, and for which we inferred candidate transcription factor regulons. Furthermore, we revealed diverse spatial localizations of malignant cells expressing their associated transcriptional programs and predicted their significant interactions with microenvironmental cell types. We validated our findings in three external single-cell RNA-seq and three bulk RNA-seq studies. Altogether, our findings advance the understanding of EAC heterogeneity, disease progression, and therapeutic response.



 Josephine Yates, Camille Mathey-Andrews, Jihye Park, Amanda Garza, Andreanne Gagne, Samantha Hoffman, Kevin Bi, Breanna Titchen, Connor Hennessey, Joshua Remland, Matthew Carnes, Erin Shannon, Sabrina Camp, Siddhi Balamurali, Shweta Kiran Cavale, Zhixin Li, Akhouri Kishore Raghawan, Agnieszka Kraft, Genevieve Boland, Andrew J. Aguirre, Nilay S. Sethi, Valentina Boeva, Eliezer M. Van Allen. Cell states and neighborhoods in distinct clinical stages of primary and metastatic esophageal adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Artificial Intelligence and Machine Learning; 2025 Jul 10-12; Montreal, QC, Canada. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(13_Suppl):Abstract nr A046.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b559d25313fad22976c28c00d01803263f1319a3" target='_blank'>
              Abstract A046: Cell states and neighborhoods in distinct clinical stages of primary and metastatic esophageal adenocarcinoma
              </a>
            </td>
          <td>
            Josephine Yates, Camille Mathey-Andrews, Jihye Park, Amanda E. Garza, Andréanne Gagné, Samantha Hoffman, Kevin Bi, Breanna M. Titchen, Connor Hennessey, Joshua Remland, Matthew Carnes, Erin Shannon, Sabrina Y. Camp, Siddhi Balamurali, Shweta Kiran Cavale, Zhixin Li, Akhouri Kishore Raghawan, Agnieszka Kraft, Genevieve M Boland, A. Aguirre, Nilay S. Sethi, Valentina Boeva, Eliezer M Van Allen
          </td>
          <td>2025-07-10</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Cancer cells can be induced to dormancy initially by specific cancer therapies, but can be reactivated for subsequent relapse as therapy-resistant cancer cells. Although the treatment-induced dormancy-to-reactivation switch is an important process in tumour spread and recurrence, little is known about the underlying molecular mechanisms, particularly the metabolic underpinnings. In this study, we demonstrated that the tryptophan catabolism-related tryptophan 2,3-dioxygenase (TDO2) -kynurenine (Kyn) -aryl hydrocarbon receptor (AhR) signalling axis was responsible for both sustaining the survival of dormant prostate cancer cells induced by androgen deprivation therapy (ADT) and promoting the reactivation of dormant cells and their recurrent outgrowth, which facilitated the development of therapeutic resistance by allowing the dormancy-to-reactivation switch. Mechanistically, we found that ADT upregulated the expression of TDO2 to produce Kyn, which activated AhR and maintained the survival of ADT-induced dormant cells. Interestingly, the switch of transcription factors from the androgen receptor (AR) to the glucocorticoid receptor (GR) modulated the persistent expression of TDO2 and promoted the reactivation of dormant cells through the same TDO2-Kyn-AhR signalling axis. Additionally, tumour recurrence following ADT was delayed by pharmacological suppression of TDO2-Kyn-AhR signalling with a TDO2 inhibitor or an AhR inhibitor. In summary, we describe a signalling circuit mediated by tryptophan metabolism for regulating tumour cell dormancy and recurrence and propose TDO2 as a new target for the treatment of androgen-sensitive prostate cancer patients in combination with ADT.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5138204cfb5fe840351972fdea392629fa575735" target='_blank'>
              AR to GR switch modulates differential TDO2-Kyn-AhR signalling to promote the survival and recurrence of treatment-induced dormant cells in prostate cancer
              </a>
            </td>
          <td>
            Sangsang Li, Yifan Zhang, Maoxing Luo, Weiwei Zhou, Yitong Chen, Dinglan Wu, Qiang Wei, Yan Chang, Hailiang Hu
          </td>
          <td>2025-08-05</td>
          <td>Cell Discovery</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="As DNA variants accumulate in somatic stem cells, become selected or evolve neutrally, they may ultimately alter tissue function. When, and how, selection occurs in homeostatic tissues is incompletely understood. Here, we introduce SCIFER, a scalable method that identifies selection in an individual tissue, without requiring knowledge of the driver event. SCIFER also infers self-renewal and mutation dynamics of the tissue’s stem cells, and the size and age of selected clones. Probing bulk whole-genome sequencing data of nonmalignant human bone marrow and brain, we detected pervasive selection in both tissues. Selected clones in hematopoiesis, with or without known drivers, were initiated uniformly across life. In the brain, we found pre-malignant clones with glioma-initiating mutations and clones without known drivers. In contrast to hematopoiesis, selected clones in the brain originated preferentially from childhood to young adulthood. SCIFER is broadly applicable to renewing somatic tissues to detect and quantify selection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6381edb3d3b2ef9cddaaae818d10d5852bbd41f" target='_blank'>
              Detecting and quantifying clonal selection in somatic stem cells
              </a>
            </td>
          <td>
            Verena Körber, N. A. Jakobsen, N. Ansari-Pour, Rachel Moore, Nina Claudino, M. Metzner, Eva Thielecke, Franziska Esau, B. Usukhbayar, Mirian Angulo Salazar, Simon Newman, B. Kendrick, Adrian H Taylor, Rasheed Afinowi-Luitz, R. Gundle, B. Watkins, Kim Wheway, D. Beazley, S. Dakin, Antony Palmer, Andrew J. Carr, Paresh Vyas, Thomas Höfer
          </td>
          <td>2025-07-03</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Background: The existence of genetically, epigenetically, and phenotypically distinct cell populations inside and between tumors is known as tumor heterogeneity, and it is one of the main barriers to effective cancer treatment. This intricacy affects the likelihood of metastasis, therapeutic resistance, and disease recurrence, rendering single-omics methods and conventional diagnostics inadequate for whole-tumor profiling. As a result, multi-omics methods, which incorporate data from multiple biological layers, such as transcriptomics, proteomics, metabolomics, genomes, and epigenomics, have emerged as powerful tools for thoroughly examining intra- and inter-tumoral complexity. 
Methods: A comprehensive literature synthesis was conducted, emphasizing high-impact studies that illustrate technological innovation and translational impact in multi-omics applications. Key case studies in glioblastoma, non-small cell lung cancer, and breast cancer are highlighted to demonstrate real-world clinical relevance. 
Aims and Objectives: This paper explores the ways in which integrated multi-omics has transformed our understanding of clonal dynamics, tumor growth, and resistance mechanisms while charting a path toward precision oncology. 
Key Insights: The latest methods, such as single-cell multi-omics, spatial transcriptomics, and proteogenomics, were examined, along with computational frameworks including network-based models, probabilistic inference algorithms, and AI-driven tools that make it easier to integrate high-dimensional data. Along with discussing new technologies like in vivo biosensors, organoid-based modeling, and point-of-care omics, the function of the tumor microenvironment, lineage tracing, and liquid biopsies in monitoring the real-time progression of tumors was also discussed. Translational hurdles, including cost, complexity, and ethical issues were addressed while highlighting the importance of equity and worldwide access. 
Conclusion: Multi-omics has great promise for truly personalized oncology by providing the integrated insights required for dynamic monitoring, predictive diagnosis, and tailored therapy design in future cancer care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/093106f85c13011bf6b7a0e38996483840456f08" target='_blank'>
              Decoding Tumor Heterogeneity through Multi Omics: Insights into Cancer Evolution, Microenvironment and Therapy Resistance
              </a>
            </td>
          <td>
            Paul-Miki Raluchukwu Ibekwe, Elizabeth Anuoluwa Akintayo, Cecilia Ndiuwem Okuku, Ismaila Muhammed, Fuhad M. Jeje, Okun, Oseghale, Kemiki Olalekan Ademola, M. D. Badru, Lydia Amarachi Onwuemelem
          </td>
          <td>2025-07-07</td>
          <td>Journal of Cancer and Tumor International</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Lung cancer remains the leading cause of cancer‐related death in both women and men in the United States and globally. Even among early‐stage patients who undergo surgical resection, a significant portion of patients develop metastases. Notably, approximately 30% of patients with stage I lung cancer experience relapse. For decades, protein‐coding genes dominated cancer research, driven by the belief that these genes were the primary contributors to tumorigenesis. Despite advances in treatment our understanding of fundamental mechanismsdriving lung cancer progression remains limited. The overall 5‐year survival rate for all stages of lung cancer combined is approximately 20%. Surgical resection remains the best option for early‐stage diseases, whereas chemotherapy, immunotherapy or combination therapies are primarily employed in advanced stages. Frequent treatment failure due to therapy resistance, highlight the urgent need to identify novel biomarkers for early diagnosis, prognosis, and the development of more effective therapies. This underscores the necessity and urgency of researching lncRNAs. The advent of next‐generation sequencing and other high‐throughput technologies guided the discovery of a new class of molecules, non‐coding RNAs (ncRNAs) that play a role in many aspects of cellular physiology. Among the various types of ncRNAs, long non‐coding RNAs (lncRNAs)—which have transcripts longer than 200 nucleotides—have emerged as key regulatory molecules in a myriad of cell functions by interacting with DNA, with other RNAs, including mRNA, miRNA and with proteins. Importantly, lncRNAs play crucial roles in cancer progression, including metastasis by activating oncogenic pathways, promoting epithelial‐mesenchymal transition, remodelling the extracellular matrix, and inducing angiogenesis. Notably, they can function as both oncogenes and tumour suppressors. MALAT1 is one of the lncRNAs that contribute to metastasis and resistance to tyrosine kinase inhibitors in lung cancer. This review summarizes the role of lncRNAs in cancer, with a specific emphasis on their contributions to lung cancer metastasis. Key points LncRNAs are significant players in lung cancer metastasis. LncRNAs can be used as therapeutic target in lung cancer. LncRNAs can be used as prognostic factors in lung cancer. Exosomal lncRNAs can be used to predict prognosis in lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25e5c37705b8c6ecc88dd0b50619485fcb12a8dc" target='_blank'>
              The role of long non‐coding RNAs in lung cancer metastasis: Molecular mechanisms, pathogenesis and clinical implications
              </a>
            </td>
          <td>
            Musaffe Tuna, Gordon B Mills, Christopher I Amos
          </td>
          <td>2025-08-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="ABSTRACT RNA epigenetics, also referred to as epitranscriptomics, has emerged as a critical regulatory layer in cancer biology, extending beyond the scope of traditional DNA and histone modifications. It encompasses a series of dynamic posttranscriptional processes—including RNA biosynthesis, splicing, transport, stability, degradation, translation, and chemical modifications—orchestrated by RNA‐binding proteins (RBPs) and noncoding RNAs (ncRNAs). Collectively, these mechanisms influence mRNA fate, shape transcriptional output, and reprogram the tumor microenvironment. Importantly, both coding RNA and ncRNA are themselves subjected to epigenetic regulation, forming intricate feedback loops that contribute to oncogenesis, immune evasion, metastasis, and therapeutic resistance. In this review, we systematically synthesize the current understanding of RNA metabolism and RNA epigenetic modifications during tumor progression, with a particular focus on the roles of RBPs and RNA modifications. Furthermore, we highlight recent advances in RNA‐based therapeutic strategies, including mRNA vaccines, antisense oligonucleotides, siRNAs, and circRNA scaffolds. These innovative approaches offer promising avenues for targeting transcriptionally active yet genomically “undruggable” cancer drivers. Together, our synthesis provides a comprehensive framework for understanding RNA epigenetics in tumor biology and lays the groundwork for precision oncology guided by transcriptome plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de7fa009ea26201e23b24a2c7b7bd45eb23b5831" target='_blank'>
              RNA Epigenetics in Cancer: Current Knowledge and Therapeutic Implications
              </a>
            </td>
          <td>
            Shanhe Huang, Zonglin Li, Weilong Lin, Ruihui Xie, Hai Huang
          </td>
          <td>2025-08-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The Warburg effect, a hallmark of cancer, entails a metabolic shift from oxidative phosphorylation to aerobic glycolysis and is tightly regulated by complex signaling pathways. Decoding the dynamic signaling pathways underlying oncogenic metabolic reprogramming remains a major analytical challenge. Here, we report a cascaded DNA logic gate strategy capable of profiling the miR-210/CYGB/p53 axis that regulates the Warburg effect in triple-negative breast cancer (TNBC) cells. By logically integrating an H2O2-responsive DNAzyme (Dz) module with an entropy-driven reaction (EDR), the INHIBIT-AND logic device simultaneously interrogates upstream (miR-210) and downstream (CYGB mRNA, H2O2) effectors, producing dual amplified fluorescent outputs in response to specific pathway states. This programmable system enables live-cell, logic-based decoding of the Warburg effect, distinguishing TNBC from other breast cancer subtypes and dynamically reflecting pathway modulation. Our approach establishes a generalizable framework for using molecular computation to analyze complex biological circuits, providing a powerful tool for precision pathway profiling and therapeutic strategy development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1e118a5499a6460056978aa8415b46448de10aa" target='_blank'>
              Cascade DNA Logic Strategy for Profiling the MiRNA-210-Targeted Signaling Pathway in the Warburg Effect of Triple-Negative Breast Cancer.
              </a>
            </td>
          <td>
            Jingyuan Yu, Xiaojun Zhang, Hao Sun, Yan Du
          </td>
          <td>2025-07-01</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Abstract DNA-damaging agents (DDAs) have long been used in cancer therapy. However, the mechanisms by which DDAs induce cell death are not fully understood, and drug resistance remains a major clinical challenge. Schlafen 11 (SLFN11) was identified as the gene most strongly correlated with sensitivity to DDAs based on mRNA expression levels. SLFN11 sensitizes cancer cells to DDAs by cleaving and decreasing tRNALeu(TAA) levels. Elucidating the detailed mechanism by which SLFN11 induces cell death is expected to provide insights into overcoming drug resistance. Here, we show that, upon administration of DDAs, SLFN11 cleaves tRNALeu(TAA), triggering ER stress and protein aggregate formation, leading to cell death regulated by inositol-requiring enzyme 1 alpha (IRE1α). These responses were significantly alleviated by SLFN11-knockout or transfection of tRNALeu(TAA). Proteomic analysis suggests tRNALeu(TAA) influences proteins essential for maintaining proteostasis, especially those involved in ubiquitin-dependent proteolysis. Additionally, we identified the cleavage sites of tRNALeu(TAA) generated by SLFN11 in cells and revealed that tRNA fragments contribute to ER stress and cell death. These findings suggest that SLFN11 plays a crucial role in proteostasis by regulating tRNAs and thus determines cell fate under DDA treatment. Consequently, targeting SLFN11-mediated tRNA regulation could offer a novel approach to improve cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f04bd7e8511d20689e4f33be87517895928d1815" target='_blank'>
              SLFN11-mediated tRNA regulation induces cell death by disrupting proteostasis in response to DNA-damaging agents
              </a>
            </td>
          <td>
            Yuki Iimori, Teppei Morita, Takeshi Masuda, Shojiro Kitajima, N. Kono, Shun Kageyama, Josephine Galipon, Atsuo T Sasaki, Akio Kanai
          </td>
          <td>2025-08-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Heritable gene activity variations that do not alter the underlying DNA sequence are known as epigenetic modifications. Mutations that disrupt genome structure and function are key drivers of oncogenesis. In addition to genetic mutations that cause direct disruptions in the DNA sequence, epigenetic changes can affect gene expression, which helps the development of cancerous traits. Changes in DNA methylation and histone mark patterns are the main drivers of these epigenetic modifications, as they encourage the growth and spread of tumors. In this review, we explore the substantial implications of epigenetic control on tumor genesis, metastatic behaviour, metabolic pathways, and the tumor microenvironment, delving into the intricacies of this intricate regulation. We pay special attention to the dysregulation at every stage of epigenetic modulation, which includes, but is not limited to, abnormalities in the enzymes that modify histones and methylate DNA, subunit loss or fusions in chromatin remodeling complexes, and disruptions in higher-order chromatin structure. We also highlight the development of inhibitors of medications targeted at epigenetic enzymes and summarize the abnormal roles of enzymes in DNA methylation, histone acetylation, and histone methylation during tumour progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97d006bb544ee4859c5808a78f45579e1cfd4a19" target='_blank'>
              Epigenetic regulators in cancer therapy and progression
              </a>
            </td>
          <td>
            Hany E. Marei
          </td>
          <td>2025-06-28</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Epigenetic modifications are heritable, reversible alterations that causally reshape chromatin architecture and thereby influence DNA repair without changing nucleotide sequence. DNA methylation, histone modifications and non-coding RNAs profoundly influence DNA repair mechanisms and genomic stability. Aberrant epigenetic patterns in cancer compromise DNA damage recognition and repair, therefore impairing homologous recombination (HR), non-homologous end joining (NHEJ), and base excision repair (BER) by suppressing key repair genes and lowering access to repair sites. Then it is dissected how loss-of-function mutations in Switch/Sucrose non-fermentable, imitation switch and CHD (Chromodomain helicase DNA-binding) chromatin-remodeling complexes impair nucleosome repositioning, preventing effective damage sensing and assembly of repair machinery. Non-coding RNAs contribute to epigenetic silencing at DNA break sites, exacerbating repair deficiencies. This review evaluates recent advances concerning epigenetic dysfunction and DNA repair impairment. It is also highlighted that nanoparticle-mediated delivery strategies are designed to overcome pharmacologic resistance. It is presented how epigenetic dysregulation of DNA repair can guide more effective and drug-resistant cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d922eca60c4fcba055a3683ed299a04149ee715f" target='_blank'>
              Epigenetic Dysregulation in Cancer: Implications for Gene Expression and DNA Repair-Associated Pathways
              </a>
            </td>
          <td>
            Nina Rembiałkowska, Katarzyna Rekiel, Piotr Urbanowicz, Mateusz Mamala, Karolina Marczuk, Maria Wojtaszek, Marta Żywica, Eivina Radzevičiūtė-Valčiukė, V. Novickij, J. Kulbacka
          </td>
          <td>2025-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f72e0c1deac040357dc9e7ae4b906709cffda79a" target='_blank'>
              Methylation dynamics in the decades preceding acute myeloid leukaemia
              </a>
            </td>
          <td>
            Adriana V.A. Fonseca, Christopher Boniface, Caroline J. Watson, Samuel Hackett, C. Gabbutt, Akemi D. Ramos-Yamasaki, Amanda B. Tan, Jose Montoya, R. Strogantsev, Lars L. P. Hanssen, Yuexuan Zhang, S. Apostolidou, A. Gentry-Maharaj, Sadik C Esener, Paul T Spellman, Trevor Graham, U. Menon, Hisham Mohammed, J. Blundell
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains a leading cause of cancer-related mortality globally, posing significant treatment challenges, particularly in its metastatic form (mCRC). This review comprehensively examines the pivotal role of RAS mutations, specifically KRAS and NRAS, which are detected in approximately 40–45% of mCRC cases, and their impact on treatment decisions and patient outcomes. We assess the effectiveness of standard treatments within the RAS mutant population, highlighting the challenges and limitations these therapies face. Recent advancements in targeted therapies, particularly the focus on novel agents such as KRAS G12C inhibitors, including sotorasib and adagrasib, have shown promising efficacy in overcoming resistance to conventional treatments. Furthermore, this review discusses future directions, emphasizing the need for research into non-RAS targets to address the complexities of resistance mechanisms and improve therapeutic outcomes. This review aims to provide a detailed overview of the current treatments and innovative approaches, supporting the development of personalized management strategies for patients with mCRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/462e15cd6a754c3946344410b05543b29153cfa7" target='_blank'>
              RAS Mutations in Advanced Colorectal Cancer: Mechanisms, Clinical Implications, and Novel Therapeutic Approaches
              </a>
            </td>
          <td>
            O. Sütçüoğlu, H. Yıldırım, Elvina Almuradova, Damla Günenç, Suayib Yalcin
          </td>
          <td>2025-06-30</td>
          <td>Medicina</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92a0745094dde86c826e8220fdb2990989d2b040" target='_blank'>
              Artificial Intelligence-Guided Molecular Determinants of PI3K Pathway Alterations in Early-Onset Colorectal Cancer Among High-Risk Groups Receiving FOLFOX
              </a>
            </td>
          <td>
            M. D. Fernando C. Diaz, B. Waldrup, Ph.D Francisco G. Carranza, Sophia Manjarrez, M. Enrique Velazquez-Villarreal
          </td>
          <td>2025-08-21</td>
          <td>None</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Although long interspersed element-1 (LINE-1) retrotransposons are poor prognostic indicators of non-small cell lung cancer, the genetic consequences of aberrant LINE-1 expression remain poorly understood. In this study, human bronchial epithelial cells were exposed chronically to 25 µM NiCl₂ to induce malignant transformation and compared to PBS-treated control cells or lung cancer cell lines. A 4 kb intronic LINE-1 insertion into the NACC2 (nucleus accumbens-associated protein 2) locus in nickel-transformed cells was associated with significant reductions in NACC2 mRNA and protein, along with constitutive increases in HDM2 and TP53α mRNAs, and aberrant expression of TP53β and TP53γ mRNAs. Steady-state levels of p53 and RB decreased, while EGFR and LINE-1 ORF1p increased. Silencing of NACC2 in TP53 wildtype NCI-H460 or TP53-null NCI-1299 cell lines replicated many of these changes, with profiles varying as a function of p53 status. Stable overexpression of HDM2 increased LINE-1 ORF1p in cancer cell lines. Human lung adenocarcinomas with wild-type TP53 showed sexually dimorphic profiles, with higher HDM2 levels in females than males, and shifts in LINE-1 ORF1p, p53, and RB that mirrored cancer cell lines. This study identifies the NACC2 locus as a target of LINE-1 retrotransposition and highlights critical interactions with the HDM2/TP53/RB regulatory axis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2adfffe6caad02c3b19f108090ea32e2ac145796" target='_blank'>
              LINE1 insertion into the NACC2 locus disrupts the HDM2 axis and activates lung oncogenic signaling
              </a>
            </td>
          <td>
            P. Bojang, Yingshan Wang, Kenneth S Ramos
          </td>
          <td>2025-07-24</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cervical cancer (CC) poses a substantial global health challenge and it ranks as the fourth most prevalent malignancy among women worldwide. Management strategies include surgical intervention, radiotherapy, chemotherapy and emerging systemic treatments. Although advancements in immunotherapy and targeted therapies have been achieved, the aggressive metastatic nature of the disease, coupled with immune evasion and drug resistance, continues to limit overall survival rates. Therefore, there remains an urgent need to identify novel treatment modalities and more effective therapeutic agents. As fundamental regulators of epigenetic modifications, histone alterations serve a critical role in controlling gene expression, DNA repair mechanisms and cellular differentiation. These modifications include acetylation, methylation, phosphorylation, ubiquitination, ADP-ribosylation and glycosylation, as well as the more recently identified lactylation and palmitoylation. By restructuring chromatin and facilitating interactions among histones, DNA and regulatory proteins, these modifications exert a substantial influence on cellular functions. Aberrant histone modifications contribute to tumorigenesis, tumor heterogeneity and resistance to conventional anticancer therapies, making them a key focus of oncological research. In recent years, therapeutic strategies targeting histone modifications have gained increasing attention in the treatment of CC. Among these epigenetic alterations, histone acetylation and deacetylation have been extensively studied, with numerous histone deacetylase inhibitors showing promise in preclinical studies. The present review explores the patterns of histone modifications in CC, emphasizing their molecular roles in tumor progression, metastasis and therapeutic resistance. Additionally, histone modification-driven therapeutic targets are examined, laying the groundwork for future precision medicine approaches in CC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9dc39c2ad958de58c571126dd5fb3dbfb68daa22" target='_blank'>
              Histone modifications in cervical cancer: Epigenetic mechanisms, functions and clinical implications (Review)
              </a>
            </td>
          <td>
            Xuewei Li, Min Zhou, Jing Yu, Shaohui Yu, Zheng Ruan
          </td>
          <td>2025-08-01</td>
          <td>Oncology Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Neuroendocrine prostate cancer (NEPC), an aggressive and highly malignant subtype of castration-resistant prostate cancer (CRPC), arises through drug resistance mechanisms involving genomic alterations, epigenetic remodeling, tumor microenvironment (TME) reprogramming, and lineage plasticity. A hallmark of NEPC is its independence from the androgen receptor (AR) pathway, evidenced by diminished or absent AR expression—a key barrier to effective clinical management. Therefore, a comprehensive understanding of NEPC pathogenesis and progression is essential, as it facilitates the identification of potential therapeutic targets and the development of more effective therapies. In this review, we summarize the typical characteristics of NEPC and describe its clinical diagnosis, relevant imaging modalities, current treatment strategies, and associated therapeutic difficulties. We also highlight the core events driving NEPC formation. Furthermore, we discuss potential therapeutic targets for the disease and review pharmacological agents that have demonstrated efficacy against NEPC, aiming to offer innovative perspectives and potential research directions for future NEPC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58ac5d561d4b3dff4622ddbdfd3ef0e9f2868489" target='_blank'>
              New Insights into Potential Therapeutic Targets for Neuroendocrine Prostate Cancer: From Bench to Clinic
              </a>
            </td>
          <td>
            Maoping Cai, Fengtao Zheng, Yang-Zi Ren, Chuqian Zhen, Dujiang Fu, Xian-lu Song, Qing Li, Yuanyuan Qu, Zhe-Sheng Chen, Shan-Chao Zhao
          </td>
          <td>2025-06-29</td>
          <td>Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) remains a challenging hematological malignancy due to its genetic heterogeneity, high relapse rates, and limited therapeutic options for refractory cases. FMS-like tyrosine kinase 3 (FLT3)-internal tandem duplication (FLT3-ITD) mutations are among the most frequent genetic alterations in AML, associated with poor prognosis and treatment resistance. In this study, we investigated the antileukemic potential of compound HI042, identified from a library of 78 molecules, focusing on its effects on FLT3-ITD-mutated AML models. HI042 selectively reduced the viability of FLT3-ITD-positive cell lines, induced apoptosis, disrupted cell cycle progression, and diminished the clonogenic potential. Chemoinformatics analysis revealed structural similarities between HI042 and retinoic acid analogues, known for their differentiation-inducing effects. Consistently, HI042 treatments increased the level of differentiation markers, including CD11b and transcription factors such as PU.1 and C/EBPs, particularly in MOLM-13 cells. Furthermore, combining HI042 with the FLT3 inhibitor quizartinib synergistically enhanced apoptosis and reduced cell proliferation. These findings highlight HI042's dual activity in inducing differentiation and apoptosis while synergizing with established therapies. Overall, HI042 emerges as a promising candidate for targeted therapies against FLT3-ITD-mutated AML, addressing a critical need for novel treatment strategies for this high-risk AML subgroup.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cf422509ae57d269aedf31cafa00342cf19e574" target='_blank'>
              Screening for Antileukemia Agents in FMS-like Tyrosine Kinase 3 (FLT3)-Mutated Acute Myeloid Leukemia Cells.
              </a>
            </td>
          <td>
            Lívia Bassani Lins de Miranda, W. Ferraz, K. Lima, J. Carlos, Fernando Moura Gatti, Rodrigo Heleno Alves, G. Trossini, J. Machado-Neto
          </td>
          <td>2025-07-22</td>
          <td>ACS pharmacology & translational science</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Genome instability is a hallmark of cancer, often driven by mutations and altered expression of genome maintenance factors involved in DNA replication and repair. Rap1 Interacting Factor 1 (RIF1) plays a crucial role in genome stability and is implicated in cancer pathogenesis. Cells express two RIF1 splice variants, RIF1-Long and RIF1-Short, which differ in their ability to protect cells from DNA replication stress. Here, we investigate differential expression and splicing of RIF1 in cancer cell lines following replication stress and in patients using matched normal and tumour data from The Cancer Genome Atlas (TCGA). Overall RIF1 expression is altered in several cancer types, with increased transcript levels in colon and lung cancers. RIF1 also exhibits distinct splicing patterns, particularly in specific breast cancer subtypes. In Luminal A (LumA), Luminal B (LumB), and HER2-enriched breast cancers (HER2E), RIF1 Exon 31 tends to be excluded, favouring RIF1-Short expression and correlating with poorer clinical outcomes. These breast cancer subtypes also tend to exclude other short exons, suggesting length-dependent splicing dysregulation. Basal breast cancer also shows exon exclusion, but unlike other subtypes, it shows no short-exon bias. Surprisingly, however, in basal breast cancer, RIF1 Exon 31 is not consistently excluded, which may impact prognosis since RIF1-Long protects against replication stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ca94768efdf45848f937fd47caaaca2fe19dbc3" target='_blank'>
              Dysregulated Alternative Splicing in Breast Cancer Subtypes of RIF1 and Other Transcripts
              </a>
            </td>
          <td>
            Emma Parker, Laura Akintche, Alexandra Pyatnitskaya, S. Hiraga, A. Donaldson
          </td>
          <td>2025-07-29</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea68d175c763147c781a0448b1e766b436554f3d" target='_blank'>
              Therapeutic strategies for fragile X syndrome and implications for other gene-silencing disorders.
              </a>
            </td>
          <td>
            Aseel Gadban, Keith M Gunapala, Verdon Taylor, N. Benvenisty
          </td>
          <td>2025-07-17</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/805d87d60453c7f2f9127c01c1cd531181467286" target='_blank'>
              Potential therapeutic targets in chronic myeloid leukemia.
              </a>
            </td>
          <td>
            Manvi Bansal, Malkhey Verma
          </td>
          <td>2025-07-17</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Aggressive and therapy-resistant glioblastoma is among the most lethal malignant tumors in humans. Complete surgical resection is often unachievable; therefore, combination chemoradiotherapy is used to target tumor cells residual beyond the resection margin. This approach induces DNA damage in tumor cells and activates the apoptosis pathway. Unfortunately, recurrence remains a major clinical challenge, frequently manifesting as more aggressive and treatmentresistant glioblastoma phenotypes. The DNA repair and damage response (DDR) pathways are critical for maintaining genome stability. While defects in these mechanisms contribute to oncogenesis, they also make tumor cells vulnerable to DNA-damaging therapy, as the cells become dependent on residual repair capacity. It is of paramount importance to understand the molecular components of these mechanisms and to identify potential therapeutic/pharmacological targets for improving outcomes in glioblastoma patients. A subpopulation of stem-like cells, designated as glioblastoma cancer stem cells (CSCs), has been identified as a critical factor in the initiation, maintenance, and recurrence of tumors. These cells exhibit therapy resistance due to enhanced DNA repair capacity. In addition, emerging evidence suggests a link between carbohydrate metabolism and DNA repair pathways, thereby revealing novel therapeutic vulnerabilities in glioblastoma. This review examines current strategies targeting DNA repair mechanisms in glioblastoma. We present a synopsis of recent advancements in research concerning the mechanisms and factors involved in the elimination of DNA damage induced by ionizing radiation and temozolomide (TMZ). Furthermore, we explore the potential of DNA repair pathway inhibitors under investigation in preclinical and clinical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8edb7bc241371338e935ca9559370edd806d8316" target='_blank'>
              DNA Damage and Repair in Glioblastoma: Emerging Therapeutic Perspectives
              </a>
            </td>
          <td>
            I. Gareev, O. Beylerli, S. Roumiantsev
          </td>
          <td>2025-07-01</td>
          <td>Creative surgery and oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="In persons living with HIV (PWH), non-AIDS-related tumors, including colorectal cancer (CRC), have become major health concerns worldwide since the introduction of highly active antiretroviral therapy. To date, no study has addressed the underlying molecular mechanisms in PWH with CRC. To explore the impact of PWH with CRC, we sequenced total RNA and DNA from individuals with HIV-negative and PWH formalin-fixed paraffin-embedded (FFPE) CRC for transcriptome and genome analyses. We performed RNA and DNA extraction from FFPE samples, library preparation, total RNA sequencing, and whole-genome sequencing. A total of 1,705 genes were found to be differentially expressed genes (DEGs), including 1,121 upregulated DEGs and 584 downregulated DEGs, in PWH compared with HIV-negative CRC. Functional pathway analysis revealed that the DEGs were enriched mainly in infectious and immune diseases and various metabolic processes. The immune infiltration results revealed that the numbers of activated dendritic cells (aDCs), natural killer T cells (NKT cells), and T follicular helper cells (Tfh cells) were greater and that the number of memory B cells was lower in patients with CRC than in PWH. Twelve hub genes involved in interferon-stimulated genes (ISGs)-IFI44, MX1, OAS1, OAS3, BST2, IFIT1, FGF2, EGF, CCL3, CCL4, SHH, and PPARG-are positively related to aDC, NKT, Tfh, and memory B cells. We found highly analogous insertions, deletions, and functional annotations of the detected single nucleotide polymorphisms and indel mutations in PWH and patients with HIV-negative CRC. This study provides new insights into crucial ISGs, immune infiltration, immune variants, and pathways involved in CRC with HIV infection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5cccc5f8e56af8b89c04e12b2c557c686201dd5f" target='_blank'>
              Whole-Genome and RNA Sequencing Reveal Novel Insights into the Pathogenesis of Colorectal Cancer in Persons Living with HIV.
              </a>
            </td>
          <td>
            Yuxue Gao, Pengxiang Yang, Yuanyue Guan, Qiqi Ning, Jing Chang, Dexi Chen, F. Wei, Yulin Zhang, Yuening Zhang
          </td>
          <td>2025-06-30</td>
          <td>AIDS research and human retroviruses</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="In recent years, the development of targeted therapies for tumors with KRAS mutations has progressed rapidly, rendering the notion of KRAS as “undruggable” outdated. However, targeted therapies for KRAS mutations still face numerous challenges, including resistance, efficacy concerns, toxicity issues, and hurdles in drug development. Exploring alternative treatment modalities is thus essential. Extensive research has demonstrated that KRAS mutations significantly influence the immune microenvironment, presenting both challenges and opportunities for immunotherapy. Interestingly, it has been observed that different KRAS mutations and co-mutation subtypes exhibit significant variations in their immunological microenvironments, which undoubtedly impact immunotherapy choices. Here, we review the history of KRAS-targeted therapy, highlighting existing challenges, and summarize changes in the immune microenvironment of KRAS-mutated cancers and their potential therapeutic targets. We compare differences in the immune microenvironment across various mutation types and co-mutation subtypes, and offer perspectives on future research directions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/993d360fa5818a3754af4b74cbddfac722d927a4" target='_blank'>
              Precision immune regulation in KRAS-mutated cancers: the final piece of the puzzle?
              </a>
            </td>
          <td>
            Shenao Fu, Jiayao Ma, Changjing Cai, Jun Tan, Xiangying Deng, Hong Shen, Shan Zeng, Yihong Chen, Ying Han
          </td>
          <td>2025-07-03</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="


 Mutation-induced drug resistance is a major obstacle in effective cancer treatment. We present a framework that integrates computational chemistry and AI for structure-based drug design targeting drug resistant mutations. To demonstrate the application of our framework, we use Kirsten Rat Sarcoma (KRAS) oncogene as a proof of concept. KRAS is one of the most mutated oncogenes in pancreatic, colorectal, and lung cancers. Mutations in KRAS cause its prolonged activation and excessive cell growth. While the primary mutant KRAS G12C responds to approved covalent inhibitors, several secondary mutations in the binding site (e.g., G12C/Y96C, G12C/Y96S, G12C/Y96D) lead to drug resistance. To understand conformational differences between treatment-sensitive and treatment-resistant populations and to enable structure-based drug design, we conducted molecular dynamics simulations on several drug-free KRAS mutants. Each simulation was performed in triplicate, and trajectory clustering was applied to extract the most populated conformations. Molecular features were calculated for the representative structures. The resulting data were analyzed using three supervised machine learning (ML) models: logistic regression, random forest, and support vector machine. Distinct structural differences in protein dynamics were observed between the two groups, particularly in the switch II binding site region, where covalent inhibitors bind. Variations were detected in residue conformations and the spatial arrangement of molecular features such as hydrogen bond donors and acceptors, as well as aromatic and aliphatic groups. Using ML, we identified that the molecular features of the most populated protein conformations differed significantly between treatment-sensitive and treatment-resistant systems. Notably, solvent exposure and conformational flexibility of residues G10, E62, and H95 within the switch II binding site emerged as the most predictive features of treatment sensitivity, alongside other features such as Lennard-Jones 1-4 energy and backbone mean square displacement. Given these differences, pharmacophores describing the physicochemical and spatial properties of switch II binding site conformations have been extracted for resistant and sensitive systems and will serve as input conditions for generative molecular design. To achieve that, we will utilize existing string- and graph-based generative ML models to design ligands within the binding site of the target. In this approach, the protein binding pocket will be represented by the coordinates of the pharmacophore, guiding the construction of molecular graphs for ligands. Generative ML applied to structure-based drug design will enable the discovery of potential bioactive compounds at an increased rate by accessing vast chemical space. Incorporating protein dynamics into this process provides deeper insights into mutation-induced drug resistance by revealing critical molecular determinants. These insights are essential for guiding the design of selective small molecules against difficult-to-target proteins.



 Katarzyna Mizgalska, Denis J. Imbody, Eric B. Haura, Wayne C. Guida, Aleksandra Karolak. A computational chemistry and AI-driven framework for structure-based drug design informed by underlying factors of mutation-induced drug resistance: A study of KRAS [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Artificial Intelligence and Machine Learning; 2025 Jul 10-12; Montreal, QC, Canada. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(13_Suppl):Abstract nr A016.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cab6eba90479670591b2362f44dfeea90c745cc" target='_blank'>
              Abstract A016: A computational chemistry and AI-driven framework for structure-based drug design informed by underlying factors of mutation-induced drug resistance: A study of KRAS
              </a>
            </td>
          <td>
            Katarzyna Mizgalska, Denis Imbody, E. Haura, Wayne C. Guida, Aleksandra Karolak
          </td>
          <td>2025-07-10</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The discovery of microRNAs (miRNAs) and their pivotal role in gene regulation has opened up new avenues for innovative cancer treatments. Recent years have witnessed extensive research into the intricate mechanisms of miRNAs and their impact on solid tumors. These small non-coding RNA molecules are central to gene regulation and are frequently dysregulated in various cancers, particularly solid tumors. Dysregulation of specific miRNAs can initiate, progress, and metastasize tumors, making them appealing targets in cancer therapy. This article explores recent studies on identifying specific miRNAs associated with solid tumors and their influence on crucial signaling pathways. These findings enable precise targeting of cancer cells, reducing damage to healthy tissues and minimizing side effects commonly associated with conventional cancer treatments. Understanding the complex regulatory networks governed by miRNAs allows researchers and clinicians to develop highly effective, personalized treatment strategies, heralding a new era of tailored cancer medicine. Ongoing research in this field holds immense promise for pioneering targeted therapies that can significantly improve outcomes and the quality of life for individuals battling solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebfcdaabb6bf533a742e4406c00d3d245acbc737" target='_blank'>
              Exploring microRNA targeting as a promising approach for solid tumor treatment
              </a>
            </td>
          <td>
            B. Shademan, Vahidreza Karamad, Alireza Nourazarian, Ç. BİRAY AVCI
          </td>
          <td>2025-07-21</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Pathogenic variants of the RB1 gene have commonly been found in many cancer types, including retinoblastoma. Nonsense mutations are the most common mutation type in retinoblastoma; however, few cell lines mimic nonsense mutations in the RB1 gene that are commonly observed in patients. Here, we established retinoblastoma-like cell lines carrying mono- and bi-allelic nonsense mutations in the RB1 gene. We introduced the R552X mutation using target activation-induced cytidine deaminase base editing and successfully constructed cell lines carrying mono- and bi-allelic mutations. The model cell lines showed decreased RB1 expression at both the mRNA and protein levels, and increased cell proliferation. Furthermore, we rescued the nonsense mutation in the RB1 gene in model cell lines by converting stop codon 552 to tryptophan using an adenine base editor. This approach may be applicable for establishing cell lines with pathogenic variants found in patients and suggests a strong potential for the application of gene editing as a therapeutic strategy. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-10600-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ea3a4888b8a55d32192a68005fc9979ffdbc161" target='_blank'>
              Establishment and rescue of fibroblast cell lines carrying a nonsense mutation of RB1 by CRISPR-based base editing
              </a>
            </td>
          <td>
            Youngri Jung, Eunhee Seo, Soyeon Yang, Sangsu Bae, Jeong Hun Kim, Hyeon-Ki Jang, Dong Hyun Jo
          </td>
          <td>2025-07-11</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background MED12 is a key regulator of transcription and chromatin architecture, essential for normal hematopoiesis. While its dysregulation has been implicated in hematological malignancies, the mechanisms driving its upregulation in acute myeloid leukemia (AML) remain poorly understood. We investigated MED12 expression across AML subgroups by integrating chromatin accessibility profiling, histone modification landscapes, and DNA methylation (DNAm) patterns. Functional assays using DNMT inhibition were performed to dissect the underlying regulatory mechanisms. Results MED12 shows subtype-specific upregulation in AML compared to hematopoietic stem and progenitor cells, independent of somatic mutations. Chromatin accessibility profiling reveals that the MED12 locus is epigenetically primed in AML blasts, with increased DNase hypersensitivity at regulatory elements. Histone modification analysis demonstrates strong H3K4me3 and H3K27ac enrichment around the transcription start site (TSS), consistent with promoter activation, while upstream and intragenic regions exhibit enhancer-associated marks (H3K4me1, H3K27ac). Notably, hypermethylation within TSS-proximal regulatory regions (TPRRs)—including promoter-overlapping and adjacent CpG islands—correlates with ectopic MED12 overexpression, challenging the canonical view of DNAm as strictly repressive. Functional studies show that DNMT inhibition via 5-azacytidine reduces MED12 expression despite promoter demethylation in cells with hypermethylated TPRRs, suggesting a noncanonical role for DNA methylation in maintaining active transcription. Furthermore, MED12 expression positively correlates with DNMT3A and DNMT3B expression, implicating these methyltransferases in sustaining its epigenetic activation. Conclusion This study identifies a novel regulatory axis in which aberrant DNA methylation, rather than genetic mutation, drives MED12 upregulation in AML. Our findings suggest that TPRR hypermethylation may function noncanonically to support transcriptional activation, likely in cooperation with enhancer elements. These results underscore the importance of epigenetic mechanisms in AML and highlight enhancer-linked methylation as a potential contributor to oncogene dysregulation. Future studies should further explore the role of noncanonical methylation-mediated gene activation in AML pathogenesis and therapeutic targeting. Supplementary Information The online version contains supplementary material available at 10.1186/s13072-025-00610-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecd4516155d5e48bceb4d470aa0e3620ac17928a" target='_blank'>
              Epigenetic regulation of MED12: a key contributor to the leukemic chromatin landscape and transcriptional dysregulation
              </a>
            </td>
          <td>
            Arundhati Chavan, Cassidy Jones, Whit Lawrence, S. R. Choudhury
          </td>
          <td>2025-07-14</td>
          <td>Epigenetics & Chromatin</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="ABSTRACT Background Epigenetic regulatory genes (epiRG) are pivotal in the epigenetic regulation of the human genome, primarily through DNA and histone modifications. These genes are frequently mutated in human cancers, particularly bladder cancer (BC). However, the functional impact of epiRG mutations on patient outcomes remains poorly understood. Methods In this study, we developed gene signatures for the most frequent genomic aberrations of epiRG using The Cancer Genome Atlas Bladder Carcinoma (TCGA‐BLCA) dataset and validated these signatures with independent tumor expression profiles for prognostic relevance. Furthermore, we evaluated the role of these signature scores in the immune system within the tumor microenvironment (TME). Finally, we assessed the correlation between epiRG and global DNA methylation. Results Our results indicated that the inferred aberration‐specific signature scores were more predictive of patient stratification than the genomic aberrations. Notably, certain signature scores were significantly associated with patient progression, whereas others correlated with the tumor immune microenvironment via interactions with the immune system. Patients with mutations had high signature scores in CREBBP‐mut and EP300‐mut, which revealed poor overall survival. Conversely, KDM6A‐mut signatures showed an opposite trend, with low scores linking to favorable prognosis through enhanced immune activity. Also, other epiRG signature scores were strongly correlated with the immune system in TME and successfully predicted patients who responded to immunotherapy. Global methylation analysis revealed that high signature scores of KDM6A‐mut are associated with hypomethylation. Conclusions These findings collectively establish epiRG signature scores as powerful biomarkers that integrate genomic, epigenetic, and immune microenvironment features for improved prognostic prediction in bladder cancer. This integrative approach not only advances our understanding of epigenetic mechanisms in BC but also offers potential for developing innovative prognostic tools and therapeutic strategies tailored to personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21c006331fc69988a65728ddc6ae59549e19f926" target='_blank'>
              Integrative Analysis Reveals the Prognostic Effects of Epigenetic Regulators in Bladder Cancer
              </a>
            </td>
          <td>
            Venugopalareddy Mekala, Yupei Lin, Xiang Wang, Naail Chowdhury, Jianrong Li, Chao Cheng
          </td>
          <td>2025-07-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The purpose of the study was a comprehensive analysis of molecular genetic changes in tumor and normal tissues of patients with colorectal cancer using whole-genome sequencing to identify de novo mutations, microsatellite instability patterns, and mutational signatures associated with tumor development.Material and Methods. A single-time cross-sectional study was conducted in the Kaliningrad region. Colorectal cancer samples were analyzed using whole-genome next-generation sequencing (WGS). The human reference genome GRCh38.p14 was used for read mapping. Bioinformatic analysis of signatures was conducted separately for each patient. 71 pairs of samples were sequenced: complete genomes of tumor biopsies and DNA isolated from blood samples for 71 patients. The average age among the analyzed patients was 69 years, with a median of 71 years.Results. Genetic alterations associated with tumor development and progression, as well as potential therapeutic targets were identified. Using the Oncovar database, 151 driver mutations were detected in 62 patients. Using the CIVIC database, 70 marker mutations were found in 45 patients. It was found that the most common driver mutations in the study cohort were mutations in the KRAS gene (KRAS:G35T, KRAS:G35A, KRAS:G38A), associated with tumor resistance to Panitumumab and Cetuximab. The frequency of the most common driver mutations in the study cohort was low, indicating high heterogeneity of colorectal cancer. Analysis of the tumor mutational burden (TMB) revealed a high average value of 5.5, suggesting a significant potential of immunotherapy in colorectal cancer treatment. In the studied population, the mutational signatures SBS1, SBS5, and SBS39 were the most common.Conclusion. The identified diversity of genomic changes may be associated with DNA replication errors, exposure to external factors, and genetic predisposition. The pattern of mutational signatures was very similar among all patients in the studied samples, suggesting a single mutagenic factor is involved in the development of colorectal cancer in all patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/410fb5988f2bc832f5b1b887492d03eded83e97b" target='_blank'>
              Genetic landscape of colorectal cancer. A pilot study in the Russian Federation
              </a>
            </td>
          <td>
            E. V. Semina, K. A. Kubasova, V. Kakotkin, S. V. Rodimov, M. Agapov
          </td>
          <td>2025-08-04</td>
          <td>Siberian journal of oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract This review explores innovative therapeutic strategies for treating central nervous system (CNS) tumors by targeting their unique metabolic dependencies. This approach marks a significant departure from traditional cytotoxic treatments, focusing instead on the metabolic vulnerabilities created by the tumor’s microenvironment and genetic profile. A key area of interest is the de novo pyrimidine synthesis pathway, which is crucial for DNA and RNA synthesis, DNA repair, and protein glycosylation. We highlight the potential of dihydroorotate dehydrogenase (DHODH) inhibitors, which have shown promising anti-tumor activity in preclinical models. The blood–brain barrier, while a challenge for drug delivery, may enhance the efficacy of these inhibitors by maintaining a unique metabolic environment in the brain. Specific brain tumors, such as glioblastoma multiforme, MYC-amplified medulloblastoma, and IDH mutant gliomas, exhibit heightened sensitivity to DHODH inhibition. We suggest that the unique metabolic environment of the brain could make DHODH a more effective therapeutic target for brain tumors compared to other cancer types. Despite the speculative nature of these findings, the compelling preclinical data warrant further investigation into brain-penetrant DHODH inhibitors for CNS malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75baff8754ebbbe7d45ccbf3a518f1778444b13c" target='_blank'>
              Exploiting genetic and environmental vulnerabilities to target DHODH in CNS tumors
              </a>
            </td>
          <td>
            Eli E Bar, J. Losman
          </td>
          <td>2025-01-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Tumor evolution is shaped by selective pressures imposed by physiological factors as the tumor naturally progresses to colonize local and distant tissues, as well as by therapy. However, the distinction between these two types of pressures and their impact on tumor evolution remain elusive, mainly, due to extensive intra-tumor heterogeneity. To disentangle the effects of these selective pressures, we analyze data from diverse cohorts of patients, of both treated and untreated cancers. We find that, despite the wide variation across patients, the selection strength on tumor genomes in individual patients is stable and largely unaffected by tumor progression in the primary settings, with some cancer-specific signatures detectable in the progression to metastases. However, we identify a nearly universal shift toward neutral evolution in tumors that resist treatment and demonstrate that this regime is associated with worse prognosis. We validate these findings on both published and original datasets. We suggest that monitoring the selection regime during cancer treatment can assist clinical decision-making in many cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a95ee20056cad563b0d086b8e70065c4daacf4f" target='_blank'>
              Genome-level selection in tumors as a universal marker of resistance to therapy
              </a>
            </td>
          <td>
            Erez Persi, P. Sudalagunta, Yuri I. Wolf, Rafael R. Canevarolo, Mehdi Damaghi, K. Shain, A. Silva, Eugene V. Koonin
          </td>
          <td>2025-07-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Proto-oncogenes in the RAS superfamily play dual roles in maintaining cellular homeostasis, such as regulating growth signals and contributing to cancer development through proliferation and deregulation. Activating proto-oncogenes in vitro transforms cells, underscoring their centrality in gene regulation and cellular networks. Despite decades of research, poor outcomes in advanced cancers reveal gaps in understanding Ras-driven mechanisms or therapeutic strategies. This narrative review examines RAS genes and Ras proteins in both housekeeping functions, such as cell growth, apoptosis, and protein trafficking, as well as in tumorigenesis, integrating insights from human (HRAS, KRAS, NRAS), mouse (Hras, Kras, Nras), and Drosophila melanogaster (ras) models. While RAS mutations are tightly linked to human tumors, the interplay between their standard and oncogenic functions remains complex. Even within the same tissue, distinct cancer pathways—such as the mitogen-activated protein kinase (MAPK) and phosphoinositide 3-kinase (PI3K) pathways—can drive varied disease courses, complicating treatment. Advanced-stage cancers add further challenges, including heterogeneity, protective microenvironments, drug resistance, and adaptive progression. This synthesis organizes current knowledge of RAS gene regulation and Ras protein function from genomic alterations and intracellular signaling to membrane dynamics and extracellular interactions, offering a layered perspective on the Ras pathway’s role in both housekeeping and tumorigenic contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/796354d7dffc674d220d0fc486fafe630423dfaf" target='_blank'>
              RESEARCH CHALLENGES IN STAGE III AND IV RAS-ASSOCIATED CANCERS: A Narrative Review of the Complexities and Functions of the Family of RAS Genes and Ras Proteins in Housekeeping and Tumorigenesis
              </a>
            </td>
          <td>
            Richard A. McDonald, A. Varela-Ramírez, Amanda K. Ashley
          </td>
          <td>2025-07-25</td>
          <td>Biology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="BACKGROUND/AIM
While normal cells are highly regulated, cancer cells take a dysregulated path which bolsters their survival. Currently, a limited number of uniform treatments are available for cancer cure. Our goal was to deprive cancer cells of the key nutrient methionine and determine what effect it would have on cell death and alterations in DNA methylation of prostate cancer cells.


MATERIALS AND METHODS
PC3 and other cell lines were transfected with plasmid gene constructs for methionine gamma lyase deaminase (MEGL), a methionine-degrading enzyme, targeted for expression in the cytoplasm (cMEGL) or the nucleus (nMEGL). For assessing cell death due to MEGL-mediated methionine deprivation, a standard 3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyltetrazolium bromide assay was used. PC3, and DU145 prostate cancer cells were selected for whole-methylome sequencing to determine the effects of MEGL expression. Key gene products comprising the Prolaris Molecular Score, specifically 31 cell-cycle progression genes, were chosen for assessing putative differences in methylome.


RESULTS
Treatment with MEGL gene targeted for expression in either the cytoplasm or nucleus caused significant cell death, similar to that due to the anticancer drug methotrexate. Azacytidine showed no effect on PC3 cell death. Propargylglycine, an inhibitor of MEGL, prevented cell death. Methylome analysis showed increased methylation of two genes: Spindle and kinetochore-associated complex subunit 1 (SKA1), origin recognition complex subunit 6 (ORC6L), and reduced methylation of six promoters: BUB1 mitotic checkpoint serine/threonine kinase B (BUB1B), PDZ binding kinase (PBK), baculoviral IAP repeat-containing 5 (BIRC5), centromere protein M (CENPM), DNA topoisomerase II alpha (TOP2A), minichromosome maintenance 10 replication initiation factor (MCM10), upon forced expression of MEGL.


CONCLUSION
Methionine deprivation through MEGL-targeted gene therapy may be a viable option for inducing cancer cell death compared to unrestricted levels of methionine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4960faa090cc3a8364fc997a8a0c4944dac73e5" target='_blank'>
              Methionine Deprivation-induced Cancer Cell Death and Methylation Changes in Key Genes and Gene Promoters of Prostate Cancer Cell Lines.
              </a>
            </td>
          <td>
            Yatin Srinivash Ramesh Babu, K. Venkatachalam
          </td>
          <td>2025-08-01</td>
          <td>Anticancer research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14e9ad63d60d3424441f26d5aee0efbd26af0b9f" target='_blank'>
              A single-cell atlas of ribosomal protein heterogeneity across human tissues reveals phenotypes of biological and clinical significance
              </a>
            </td>
          <td>
            Aishwarya Murali, Chartsiam Tipgomut, Anand D. Jeyasekharan, Himanshu Sinha
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Acquired non-genetic resistance mechanisms to existing therapies contribute to poor outcomes for acute myeloid leukemia (AML) patients, and inability to target leukemic stem cells (LSCs) can lead to relapse. To overcome these challenges, we tested whether LSCs have dependencies on PI3 kinase (PI3K). We found that LSCs are susceptible to isoform-selective targeting of PI3K and are particularly dependent on the P110 alpha isoform of PI3K. We discovered that PI3K inactivation leads to dynamic changes in EZH2/PRC2 function in leukemic cells, and we uncovered downregulation of EZH2 protein levels as a resistance mechanism in response to PI3K inhibition. We found that PI3K inhibition in AML cells can lead to compensatory upregulation of EZH1, and that EZH1 knockdown can sensitize AML cells to PI3K inhibition. We leveraged this resistance mechanism by combining a PI3K inhibitor with an EZH1/2 dual inhibitor, which successfully overcomes the acquired resistance and leads to sustained targeting of AML cells ex vivo and in murine AML and PDX models in vivo. This study identifies a promising novel therapeutic regimen for targeting LSCs in AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/151e9472e03c9c24a242cfaacfbad5bb660b4476" target='_blank'>
              Exploiting an Epigenetic Resistance Mechanism to PI3 Kinase Inhibition in Leukemic Stem Cells
              </a>
            </td>
          <td>
            Shira G Glushakow-Smith, Imit Kaur, Simone Sidoli, Shayda Hemmati, Ellen Angeles, Taneisha Sinclair, Samarpana Chakraborty, Aaliyah Battle, Kristina Ames, Swathi-Rao Narayanagari, Rotila Hyka, Mark Soto, Melissa Tracy, Jayaram Vankudoth, Seiya Kitamura, Linde Miles, U. Steidl, A. Shastri, Amit Verma, K. Gritsman
          </td>
          <td>2025-07-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Oral squamous cell cancer (OSCC) is a major global health issue, ranking sixth in prevalence, particularly in Asia. The diagnosis often occurs late due to inadequate early screening, resulting in a dismal five-year survival rate of around 50%. This document provides a comprehensive analysis of drug-based treatments for oral cavity carcinoma, focusing on chemotherapy, immune modulation, and novel approaches like nanoparticle therapies. Despite advancements in these methods, drug resistance remains a significant obstacle that adversely affects patient outcomes. The research highlights the critical role of long ncRNAs in the progression and treatment of OSCC. These long ncRNAs, which are over 200 nucleotides long, play essential roles in gene regulation and tumor growth, including mechanisms of drug resistance. Some long ncRNAs may promote or inhibit tumor development and influence the effectiveness of anti-cancer drugs like cisplatin. Additionally, the review explores how the tumor microenvironment and immune responses interact, suggesting that inflammation may accelerate the progression of oral cancer. By synthesizing insights from extensive literature, this review clarifies the complex relationship between long ncRNAs and OSCC treatment. The study aimed to improve treatment efficacy and increase survival rates for patients with oral squamous cell carcinoma by identifying potential therapeutic targets. The findings underscore the importance of integrating molecular insights into treatment strategies to combat drug resistance and enhance patient outcomes in oral cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/680f8ab3d2bbad8990d7480a9760079550d9fbe0" target='_blank'>
              Oral squamous cell carcinoma pharmacological treatment; A long non-coding RNAs (long ncRNAs) story
              </a>
            </td>
          <td>
            Azita Sadeghzade, Ali Jafarian, Amir Hossein Davodpour, Mohsen Pouresmaeliyan Roumani, Meysam Mohammadikhah, Sarah Qutaiba Badraldeen, Sajad Ataei Azimi
          </td>
          <td>2025-07-01</td>
          <td>International Journal of Molecular and Cellular Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Mutations that activate the small GTPase KRAS are a frequent genetic alteration in cancer, and drug discovery efforts have led to inhibitors that block KRAS activity. We sought to better understand oncogenic KRAS signaling and the cytostatic effects of drugs that target this system. We performed proteomic analyses to investigate changes in protein abundance and posttranslational modifications in inhibitor-treated human KRAS-mutant pancreatic (KRAS G12C and G12D) and lung cancer (KRAS G12C) cells. The inhibitors used target these mutant forms of KRAS, the downstream effectors MEK and ERK, and the upstream regulators SHP2 and SOS1. Comparisons of phosphoproteomes between cell lines revealed a core KRAS signaling signature and cell line-specific signaling networks. In all cell lines, phosphoproteomes were dominated by different degrees of autonomous, oncogenic KRAS activity. Comparison of phosphoproteomes after short and long drug exposures revealed the temporal dynamics of KRAS-MEK-ERK axis inhibition that resulted in cell cycle exit. This transition to a quiescent state occurred in the absence of substantial proteome remodeling but included broad changes in protein phosphorylation and ubiquitylation. The collective data reveal insights into oncogenic KRAS signaling, place many additional proteins into this functional context, and implicate cell cycle exit as a mechanism by which cells evade death upon KRAS signaling inhibition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2163ec601f10210697958516b4e39127c69ccee" target='_blank'>
              Proteomic analyses identify targets, pathways, and cellular consequences of oncogenic KRAS signaling.
              </a>
            </td>
          <td>
            Nicole Kabella, Florian P. Bayer, Konstantinos Stamatiou, Miriam Abele, Amirhossein Sakhteman, Yun-Chien Chang, Vinona Wagner, Antje Gabriel, Johannes Krumm, Maria Reinecke, Melanie Holzner, Michael Aigner, Matthew The, Hannes Hahne, Florian Bassermann, Christina Ludwig, Paola Vagnarelli, Bernhard Kuster
          </td>
          <td>2025-07-29</td>
          <td>Science signaling</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Simple Summary Gynecological cancers, such as endometrial, cervical, and ovarian cancer, are among the leading causes of death in women worldwide. Scientists are searching for new ways to detect these diseases earlier and to tailor treatment options better. To achieve this, changes in genes responsible for DNA repair are being analyzed. When these repair processes are weakened, DNA damage can accumulate, increasing the risk of cancer development. In our study, we reviewed the latest scientific publications to identify genetic variations that may be linked to the development of gynecological cancers. These findings may contribute in the future to the early detection of these diseases and to the development of therapies tailored to the individual genetic profiles of patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e542d9860b43270a69e0d944b9a7034ae1638d99" target='_blank'>
              Genetic Polymorphisms in Base Excision Repair (BER) and Nucleotide Excision Repair (NER) Pathways as Potential Biomarkers for Gynecological Cancers: A Comprehensive Literature Review
              </a>
            </td>
          <td>
            Magdalena Szatkowska, Julita Zdrada-Nowak
          </td>
          <td>2025-06-27</td>
          <td>Cancers</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/888d9f82216c90d7528814f40ba0327b18c81f36" target='_blank'>
              Genetic Control of tRNA-Derived Fragments Contributes to Cancer Risk.
              </a>
            </td>
          <td>
            Bin Li, Hanting Li, Yan Li, Zhirui Chen, Danyi Zou, Chunyi He, Songqiang Li, Kexin Hu, Caibo Ning, Yanmin Li, Hui Geng, Shuhui Yang, Chaoqun Huang, Xiaojun Yang, Qiuhong Wang, Xu Zhu, Xiangpan Li, Ying Zhu, Meng Jin, Xiaoping Miao, Jianbo Tian
          </td>
          <td>2025-08-07</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Genetic alterations alone cannot account for the diverse phenotypes of cancer cells. Even cancers with the same driver mutation show significant transcriptional heterogeneity and varied responses to therapy. However, the mechanisms underpinning this heterogeneity remain under-explored. Here, we find that novel enhancer usage is a common feature in acute lymphoblastic leukemia (ALL). In particular, KMT2A::AFF1 ALL, an aggressive leukemia with a poor prognosis and a low mutational burden, exhibits substantial transcriptional heterogeneity between individuals. Using single cell multiome analysis and extensive chromatin profiling, we reveal that much transcriptional heterogeneity in KMT2A::AFF1 ALL is driven by novel enhancer usage. By generating high resolution Micro Capture-C data in primary patient samples, we identify patient-specific enhancer activity at key oncogenes such as MEIS1 and RUNX2, driving high levels of expression of both oncogenes in a patient-specific manner. Overall, our data show that enhancer heterogeneity is highly prevalent in KMT2A::AFF1 ALL and may be a mechanism that drives transcriptional heterogeneity in cancer more generally.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d523b069d65585980e3b6635ddde12c81e15186" target='_blank'>
              Enhancer heterogeneity in acute lymphoblastic leukemia drives differential gene expression in patients.
              </a>
            </td>
          <td>
            Alastair L. Smith, N. Denny, Catherine Chahrour, Kim Sharp, Marta Arachi, A. M. Dopico-Fernandez, N. Elliott, Joe R Harman, Thomas Jackson, H. Geng, Owen Smith, Jonathan Bond, Irene Roberts, Ronald W W Stam, Nicholas T Crump, James O. J. Davies, Anindita Roy, Thomas A. Milne
          </td>
          <td>2025-07-29</td>
          <td>Blood</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b2a0c8aae8bd565763175795d01ffdf375af883" target='_blank'>
              Ongoing genome doubling shapes evolvability and immunity in ovarian cancer.
              </a>
            </td>
          <td>
            Andrew McPherson, I. Vázquez-García, Matthew A. Myers, Duaa H Al-Rawi, Matthew Zatzman, A. Weiner, Samuel Freeman, N. Mohibullah, Gryte Satas, Marc J. Williams, Nicholas Ceglia, Danguolė Norkūnaitė, Allen W. Zhang, Jun Li, J. Lim, Michelle Wu, Seongmin Choi, Eliyahu Havasov, Diljot S. Grewal, Hongyu Shi, Minsoo Kim, RF Schwarz, Tom L. Kaufmann, K. Dinh, F. Uhlitz, Julie Tran, Yushi Wu, Ruchi Patel, Satish Ramakrishnan, DooA Kim, Justin Clarke, Hunter Green, Emily Ali, Melody DiBona, Nancy Varice, Ritika Kundra, V. Broach, Ginger J Gardner, K. Roche, Y. Sonoda, O. Zivanovic, Sarah H. Kim, R. Grisham, Ying L Liu, A. Viale, Nicole Rusk, Yulia Lakhman, Lora H. Ellenson, S. Tavaré, Sam Aparicio, Dennis S Chi, Carol Aghajanian, N. Abu-Rustum, C. Friedman, Dmitriy Zamarin, Britta Weigelt, S. Bakhoum, Sohrab P. Shah
          </td>
          <td>2025-07-16</td>
          <td>Nature</td>
          <td>0</td>
          <td>93</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC), an aggressive neuroendocrine carcinoma, has an extremely poor prognosis. ASCL1 and NEUROD1 are key regulators of neuroendocrine features, and previous studies have suggested that SCLC plasticity occurs during the transition from ASCL1-positive (SCLC-A) to NEUROD1-positive (SCLC-N) subtypes. In this study, we attempted to understand the transcriptional programs governed by ASCL1 and NEUROD1 to identify markers of SCLC plasticity. Immunohistochemistry and epigenome and transcriptome analyses in ASCL1/NEUROD1 double-positive SCLC cells (SCLC-A/N) revealed co-expression of ASCL1 and NEUROD1 in almost half of SCLC cases. Genome-wide profiling of histone modifications, ASCL1 and NEUROD1 binding sites, and gene co-expression patterns revealed that both ASCL1 and NEUROD1 are active in SCLC-A/N and regulate partially distinct target genes. Furthermore, SCLC-A/N exhibited characteristics that were intermediate between SCLC-A and SCLC-N subtypes. NEUROD1 knockout, followed by RNA-seq, suggested an association between NEUROD and NHLH transcription factors that might shape the NEUROD1-mediated regulatory network. Small RNA-seq further indicated that miR-139-5p is specifically expressed in NEUROD1-positive SCLC, and transcriptomic studies suggested that miR-139-5p might regulate an array of pathologically relevant genes in collaboration with other NEUROD1-associated miRNAs. Our integrative analyses provide deeper insights into SCLC heterogeneity and multi-layered transcriptional programs differentially governed by ASCL1 and NEUROD1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7bf9c97f8b5f0436d9541af1bdffece02f4242ee" target='_blank'>
              Integrative epigenome and transcriptome analyses reveal transcriptional programs differentially regulated by ASCL1 and NEUROD1 in small cell lung cancer
              </a>
            </td>
          <td>
            Hiroshi Takumida, Akira Saito, Yugo Okabe, Yasuhiro Terasaki, Yu Mikami, Hidenori Tanaka, Masami Suzuki, Yu Hamaguchi, Chao Zeng, Michiaki Hamada, Hiroshi I Suzuki, H. Kage, Masafumi Horie
          </td>
          <td>2025-07-01</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab733a8835d2537eb7a1dc0f915fffde2f98ed35" target='_blank'>
              An updated patent review on rational combinations of HDAC inhibitors for cancer chemotherapy (2020 - present): part 1-patent granted.
              </a>
            </td>
          <td>
            Yugal Kishor Shukla, Vandana, Vivekananda Mandal, V. Asati, Raj K. Keservani, Sanjay Kumar Bharti
          </td>
          <td>2025-08-12</td>
          <td>Expert opinion on therapeutic patents</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Abstract Transposable elements (TE), also called transposons, are repetitive DNA sequences making up half of the human genome. Initially, TEs were described as “junk DNA” because they lack specific function. However, they have been recognized for their ability to replicate and integrate into different genomic locations; this “jumping gene” activity results in genomic instability, variation of the chromosomal architecture, and transcriptional dysregulation, all of which represent major hallmarks of cancer. In this respect, the involvement of TE in tumorigenesis was extensively studied and their role as diagnostic and therapeutic tools in cancer is now well-established. Transposons’ products, including TE-derived cancer antigens, transcripts, and associated epigenetic modifications, mainly hypomethylation, were found to be promising biomarkers in several types of cancer ensuring early disease detection and prognosis. In addition, TE are currently used to design innovative approaches, with transposon-based systems, namely, Sleeping Beauty and PiggyBac, enabling precise genomic modifications and novel strategies for cancer therapy. Therefore, the aim of this review is to provide an overview on the dual application of TE as diagnostic and therapeutic tools in cancer, paving the way to improved clinical outcomes. Clinical trial number Not applicable.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4636ade80b58aaf2a75f12fb6871d3d8ab2244c5" target='_blank'>
              Exploring transposable elements: new horizons in cancer diagnostics and therapeutics
              </a>
            </td>
          <td>
            Ahlam Chaaban, Reem Sleem, Jana Santina, Mohamad Rima, José-Noel Ibrahim
          </td>
          <td>2025-07-12</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background A key to understanding cancer is to determine the impact on the cellular pathways caused by the repertoire of DNA changes accrued in a cancer cell. Exploring the interactions between genomic aberrations and the expressed transcriptome can not only improve our understanding of the disease but also identify potential therapeutic approaches. Results Using random forest models, we successfully identified transcriptional patterns associated with the loss of wild-type activity in cancer-related genes across various tumour types. While genes like TP53 and CDKN2A exhibited unique pan-cancer transcriptional patterns, others like ATRX, BRAF, and NRAS showed tumour-type-specific expression patterns. We also observed that genes like AR and ERBB4 did not lead to strong detectable patterns in the transcriptome when disrupted. Our investigation has also led to the identification of genes highly associated with transcriptional patterns. For instance, DRG2 emerged as the top contributor in classification of ATRX alterations in lower-grade gliomas and was significantly downregulated in ATRX mutant tumours. Additionally, transcriptional features important in classification of PTEN aberrations, such as CDCA8, AURKA, and CDC20, were found to be closely related to PTEN function. Conclusions Our findings demonstrate the utility of machine learning in interpretation of cancer genomic data and provide new avenues for development of targeted therapies tailored to individual patients with cancer. Our analysis on the transcriptome revealed genes with expression levels strongly correlated with alterations in cancer-related genes. Additionally, we identified AURKA inhibitors as potential therapeutic option for tumours with alterations in tumour suppressors like FBXW7 or NSD1. Supplementary Information The online version contains supplementary material available at 10.1186/s12915-025-02339-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e1f512af72f3e98e7eac17d6531fcc9f244b2e0" target='_blank'>
              Transcriptional patterns of cancer-related genes in primary and metastatic tumours revealed by machine learning
              </a>
            </td>
          <td>
            Faeze Keshavarz-Rahaghi, E. Pleasance, Steven J M Jones
          </td>
          <td>2025-08-07</td>
          <td>BMC Biology</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Immune checkpoint blockade and cancer immunotherapy have revised the cancer treatment therapeutic environment, but patient reactions are excessively difference. The targeted population in this systematic review is animal trials in order to understand any genetic markers effecting the efficacy of the immunotherapy. The research tries using murine models like syngeneic, genetic engineered mice and humanized mice to define key genes that are determinants of therapeutic response (PD-L1, IFN-gamma, JAK1/2, CTNNB1 and EZH2). High expression of PD-L1 and IFN-γ reported positive correlation with response and survival whereas mutation in JAK1/2 and CTNNB1 activation reported resistance because of ineffective immune infiltration and stimulation events. Inhibition of EZH2 increased immune activity partly which indicates the role of epigenetic control. The predictive value of these markers was confirmed by the analysis of data via ANOVA, Kaplan-Meier survival curves, correlation, and bioinformatics tools. Animal models will continue to be essential to mechanisms, despite species-related and methodological limitation. These results confirm the use of genetic profiling in personalized immunotherapy strategies and point to the importance of additional translational data in models more humanized as well as multi-omics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/845932ec61e6b82c7d0c7ccc9ea937f5a7aad54f" target='_blank'>
              Genetic Markers for Predicting Cancer Immunotherapy Response
              </a>
            </td>
          <td>
            Mamidisetti Sai Ganesh, Nallani Venkata Rama Rao, Nadendla. Rama Rao, Banavathu Tulasya Naik, Doddigaralla Salman Raju
          </td>
          <td>2025-08-04</td>
          <td>Journal of Pharmacology, Genetics and Molecular Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Breast cancer patients who express estrogen receptor α (ER) typically receive endocrine therapy as a first-line treatment. Most ER-positive breast cancer patients initially respond to endocrine therapy, but up to 40% of patients develop resistance over time, and the main mechanism is aberrant activation of ER signaling pathways. Recent research has shown that cancer's uncontrolled proliferative capacity and a hostile microenvironment can induce endoplasmic reticulum (EnR) stress and trigger unfolded protein response (UPR), which plays a crucial role in determining cell fate. Furthermore, increasing evidence suggests that noncoding ribonucleic acids (RNAs), which are involved during the crosstalk between the UPR and ER signaling pathways, play an essential role in endocrine resistance. In this review, we provide an overview of long noncoding RNA (lncRNA) that is involved both in endocrine resistance and UPR. We provide new insights into potential treatment strategies to overcome endocrine resistance in ER-positive breast cancer patients by targeting the lncRNA that plays key roles in endocrine resistance and UPR signaling. Finally, we discuss the current state and future directions for targeting lncRNAs to improve clinical outcomes in endocrine-resistant breast cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4692ed07176028a188031985ca7bc44cfc350d2e" target='_blank'>
              The role of UPR-related long noncoding RNAs in development of endocrine resistance in breast cancer.
              </a>
            </td>
          <td>
            Wen Liu, Sahil Gupta, Sanjeev Gupta
          </td>
          <td>2025-08-01</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8fda3215ef96d1797670c5063bd5db4df44e918" target='_blank'>
              Genetic variants in oncogenic miRNA and 3' untranslated region of tumor suppressor genes: emerging insight into cancer genetics.
              </a>
            </td>
          <td>
            Anita Choudhary, Anil Kumar, A. Munshi
          </td>
          <td>2025-06-30</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="The Jurkat cell line, derived from a case of T-cell lymphoblastic leukaemia, is widely employed as a model T-cell for biomedical research, including for the preclinical characterisation of cellular therapies such as chimeric antigen receptor T-cells. Here, we characterised genomic, transcriptomic, and functional features of Jurkat clone E6-1 cells from three different laboratories and compared these with Jurkat E6-1 cells from the American Type Culture Collection (ATCC). We identified marked karyotypic heterogeneity both between and within Jurkat E6-1 populations, confirmed through chromosomal microarray. Whole exome sequencing of each cell population revealed both shared and unique mutational profiles, and transcriptomic profiles varied markedly between Jurkat E6-1 cell populations. Finally, the Jurkat E6-1 cell populations exhibited substantial variations in immunophenotype and cytokine production, which were consistent with the genotypic and transcriptomic changes observed. In summary, we identify substantial genomic heterogeneity both between and within Jurkat E6-1 cell populations, highlighting the genomic instability of this line. These genomic changes were associated with differences in protein expression and cytokine production that may affect functional assays. Our findings highlight the importance of monitoring cell passage number, characterising cell lines, and replacing cell line stocks to assure the accuracy, reproducibility, and translatability of assays employing Jurkat E6-1 cells. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-95903-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b26a4327fd37f7b7bc2ba21d6b52773a27421f0b" target='_blank'>
              Jurkat T-cell lines exhibit marked genomic instability affecting karyotype, mutational profile, gene expression, immunophenotype and function
              </a>
            </td>
          <td>
            Andrew Wilson, N. Dasyam, Felix O'Hagan, Dianna Farrell, Claire Turner, Anne Jay, Alfonso Schmidt, Rachel Beddow, Tina Lillis, Tom Chin, Rachel Perret, Michelle McCulley, R. Weinkove
          </td>
          <td>2025-07-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="PARP inhibitors (PARPi), recently introduced for treating metastatic castration-resistant prostate cancer (mCRPC), have heightened interest in molecular profiling for pathogenic aberrations in homologous recombination DNA repair (HRR) genes in all mCRPC patients. Liquid biopsy offers a viable alternative to archival tumor tissue for genetic analysis. In this study, we assessed the feasibility and utility of combining mutational panel sequencing with shallow whole genome sequencing (sWGS) to refine HRR status determination from plasma in prostate cancer (PCa) patients. The mutational profile of 16 HRR genes was assessed in 63 PCa patients: 28.6% of patients harbored putative pathogenic variants in HRR-related genes. A HRR-mutant status was defined for 10 patients (15.8%). Through the integration of sWGS data, plasma samples non-informative about somatic alterations were identified, and germline/somatic origin of HRR mutations was defined. Matched tumor tissue was available for 41 patients, with an 85.7% concordance rate between plasma and tissue mutational analyses. Additionally, we explored the copy number variation (CNV) profile using sWGS and it was found concordant with the literature PCa profiles. Our findings demonstrated that ctDNA analysis through liquid biopsy is a reliable alternative to tissue-based methods for identifying SNVs and CNVs. However, concordance was affected by ctDNA levels in plasma and clonal hematopoiesis. The data highlight the utility of integrating sWGS with targeted mutation analysis for comprehensive molecular profiling of PCa patients. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-05384-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad00381f62d746252d19dc78da101336888d0016" target='_blank'>
              Mutational and low-coverage whole genome sequencing identifies actionable DNA repair alterations in prostate cancer plasma DNA
              </a>
            </td>
          <td>
            A. Virga, Milena Urbini, Maurizio Polano, E. Petracci, G. Tedaldi, G. Gurioli, G. Marisi, D. Angeli, Andrea Ambrosini-Spaltro, Giovanni De Luca, Ilaria Cangini, Valentina Zampiga, Giovanna Cenacchi, Giuseppe Toffoli, C. Casadei, M. C. Cursano, V. Conteduca, U. de Giorgi, Paola Ulivi
          </td>
          <td>2025-07-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Given the prevalence of RAS mutations in various cancers, personalized therapeutic approaches, guided by molecular markers, are essential. Farnesyltransferase inhibitors (FTIs) have emerged as potential therapeutic options; however, they also face obstacles such as toxicity and limited efficacy. Alternative strategies, such as direct inhibitors combined with pathway modulators, RNA interference, and gene-editing technologies, are under clinical investigation. The targeting of RAS, complicated by its structural nuances, particularly in the G domain, has advanced with the identification of druggable pockets such as the SW-II pocket. This breakthrough has led to the development of targeted therapeutics, such as sotorasib and adagrasib, for KRAS G12C-mutated non-small cell lung cancer (NSCLC). However, these advancements face challenges, including adaptive resistance and the necessity for isoform selectivity. New inhibitors, such as LY3537982 or GDC-6036, are promising, but achieving effective and selective RAS inhibition remains a significant challenge. Additionally, clinical trials have highlighted variability in patient responses, attributing limited treatment efficacy to resistance mechanisms, including on-target mutations and off-target pathway activations. Finally, the RAS oncogene, traditionally viewed as predominantly pro-cancerous, plays a complex role in oncogenesis, with recent evidence suggesting context-dependent effects, such as inducing senescence in certain cells. This shift in understanding underscores the therapeutic potential of manipulating the interplay between RAS and TP53 mutations in cancer. In conclusion, the complexity of effectively targeting the RAS-RAF-ERK pathway is exacerbated by the diverse resistance mechanisms. Challenges such as off-target effects and delivery issues remain significant barriers in the introduction of effective therapies based on RAS inhibitors. This overview highlights the evolving nature of targeting RAS in cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8dda4fec514efa73d1390e9273305a191b377020" target='_blank'>
              The complex journey of targeting RAS in oncology
              </a>
            </td>
          <td>
            Katarzyna Wasiak, Damian Ciunowicz, Amelia Kierasińska-Kałka, Marta Węgierska, Marcin Pacholczyk, Piotr Rieske, Ewelina Stoczyńska-Fidelus
          </td>
          <td>2025-07-01</td>
          <td>BMC Cancer</td>
          <td>1</td>
          <td>14</td>
        </tr>

        <tr id="There is a lack of genetic biomarkers for predicting response to immune checkpoint blockade (ICB) therapy in cancer. The recent discovery that loss-of-function mutations in the gene encoding the protein phosphatase 2A (PP2A) scaffold protein PPP2R1A confer sensitivity to immune checkpoint blockade in ovarian clear cell carcinoma, therefore represents a breakthrough. Mechanistically, mutations in the PPP2R1A gene induce a strong interferon gamma response in tumor cells, which enhances infiltration of activated CD8+ T cells into the tumor. The activity of these T cells is then fortified by ICB. Furthermore, preclinical studies have shown that PP2A inhibition leads to the generation of neoantigens by disrupting RNA splicing, and PP2A inhibition can remodel the immune microenvironment of tumors to enhance responses to ICB. The finding that loss-of-function PPP2R1A mutations predict benefit from immunotherapy also suggests that pharmacological inhibition of PP2A may act synergistically with ICB therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8541b81fbcfabd62dc3e05a2a27e83fb1d85025" target='_blank'>
              A single gene mutation predicts response to immune checkpoint blockade in ovarian clear cell carcinoma.
              </a>
            </td>
          <td>
            M. H. Dias, R. Bernards
          </td>
          <td>2025-07-30</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Mutant KRAS has been implicated in driving a quarter of all cancer types. Although inhibition of the KRASG12C mutant protein has shown clinical promise, there is still a need for therapies that overcome resistance and target non-KRASG12C mutations. KRAS activates downstream MYC, which is also a challenging-to-drug oncoprotein. We have developed an "inverted" RNAi molecule with the passenger strand of a MYC-targeting siRNA fused to the guide strand of a KRAS-targeting siRNA. The chimeric molecule simultaneously inhibits KRAS and MYC, showing marked improvements in efficacy beyond the individual siRNA components. This effect is mediated by 5'-dT overhangs following endosomal metabolism. The synergistic RNAi activity led to a >10-40-fold improvement in inhibiting cancer viability in vitro. When conjugated to an epidermal growth factor receptor (EGFR)-targeting ligand, the chimeric siRNA was delivered to and internalized by tumor cells. As compared with individual targeting siRNAs, the chimeric design resulted in considerably improved metabolic stability in tumors, enhanced silencing of both oncogenes, and reduced tumor progression in multiple cancer models. This inverted chimeric design establishes proof-of-concept for ligand-directed, dual-silencing of KRAS and MYC in cancer and constitutes an innovative molecular strategy for co-targeting any two genes of interest, which has broad implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1afbec09737211f6f2ff3789e350810d140c4a0" target='_blank'>
              Inverted chimeric RNAi molecules synergistically co-target MYC and KRAS in KRAS-driven cancers.
              </a>
            </td>
          <td>
            Y. Chareddy, H. Huggins, S. S. Sahoo, Lyla Stanland, C. Gutierrez-Ford, Kristina M. Whately, Lincy Edatt, S. Azam, Matthew C Fleming, Jonah Im, A. Porrello, Imani Simmons, Jillian Perry, A. Bowers, M. Egli, C. Pecot
          </td>
          <td>2025-07-22</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="This study examines targeted degradation of the transcription factor MYB as a novel therapeutic avenue in acute myeloid leukemias. Its relevance stems from the limited efficacy of existing treatment regimens and the emergence of drug‐resistant disease forms. The work’s novelty lies in the integrated comparison of genetic models for MYB destabilization, chemical dTAG approaches, and the repurposing of low-molecular-weight compounds mebendazole and vitaferin A. Data are synthesized on MYB’s transformational properties critical for leukemic-clone maintenance, and experimental findings are reviewed on the suppression of transcriptional programs and induction of blast-cell death following complete factor elimination. Special attention is devoted to prospects for developing PROTAC degraders and molecular glues capable of catalyzing ubiquitin-dependent proteolysis of MYB at low compound doses. The research aims to construct a comprehensive assessment of the efficacy and safety of MYB-degradation strategies and to identify directions for further preclinical investigation. Methods include comparative analysis of peer-reviewed literature, critical appraisal of in vitro and in vivo data, and synthesis of pharmacodynamic profiles of the repurposed agents. The overall evaluation highlights the approach’s potential to overcome therapeutic resistance and improve patient survival. The findings will interest pharmacologists, oncologists, clinical researchers, and specialists in chemical biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cd3eb1da1160da3c3448b57b37bb25924e4e35b" target='_blank'>
              Targeted Degradation of MYB as a Novel Therapeutic Strategy for Acute Myeloid Leukemias
              </a>
            </td>
          <td>
            Oleksandra Bondarenko
          </td>
          <td>2025-07-05</td>
          <td>The American Journal of Interdisciplinary Innovations and Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7e75d3ecf904d9e7aa51e15582bd637748462a3" target='_blank'>
              Novel mutations of SRPX facilitate the stemness and malignant progression of glioma.
              </a>
            </td>
          <td>
            Siyuan Tang, Chunhui Qu, Mingyu Zhang, Peijun Zhou, Xingzhi Peng, Zhuan Zhou, Liangfang Shen, Lifang Yang
          </td>
          <td>2025-07-15</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background/Objectives: The HOX genes encode a family of homeodomain-containing transcription factors that have important roles in defining cell and tissue identity in embryonic development, but which also show deregulated expression in many cancers and have been shown to have pro-oncogenic roles. Due to their functionally redundant nature, strategies to target HOX protein function in cancer have focused on their interaction with their PBX cofactor using competitive peptides such as HXR9. HOX/PBX inhibition triggers apoptosis through a sudden increase in target gene expression, including Fos, DUSP1, and ATF3, which are otherwise repressed by HOX/PBX binding. Methods: We analyzed publicly available transcriptomic data in the R2 platform. Results: We show that a specific subgroup of HOX genes is negatively correlated with Fos, DUSP1, and ATF3 expression in prostate cancer, and that this subgroup also shows a strong positive corelation with pathways that support tumour growth, most notably DNA repair and aminoacyl tRNA biosynthesis, and a negative correlation with genes that promote cell adhesion and prevent motility. In addition, this set of HOX genes strongly correlates with patient age, reflecting a previously identified progressive loss of regulation of HOX expression in normal peripheral blood cells. Conclusions: Our findings indicate these HOX genes may have pro-oncogenic functions in prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac048edd372281c4cd71437128a80c986a12e4bb" target='_blank'>
              A Subset of HOX Genes Negatively Correlates with HOX/PBX Inhibitor Target Gene Expression and Is Associated with Apoptosis, DNA Repair, and Metabolism in Prostate Cancer
              </a>
            </td>
          <td>
            R. Morgan, Christopher Smith, H. Pandha
          </td>
          <td>2025-07-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Simple Summary Cancer is a leading cause of death worldwide, and researchers are continuously working to uncover its drivers and new therapeutic targets. In this review, we explain the role of the oncogenic protein CREPT that is overexpressed in the vast majority of cancers, leading to poor overall survival. CREPT ablation has shown sustained tumor regression in vitro and in mouse models. CREPT activation mediates cancer progression by enhancing the cell cycle by promoting chromatin loop formation. Thus, selectively targeting CREPT expression may provide alternative strategies for inhibiting abnormal cell division, metastasis, and tumor growth, and may improve the efficacy of the currently available therapies in combination, and will be key to reversing cancerous growth in patients with CREPT-driven cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c17bae866b8a91dd27eb2fc5ec68639a8ba51279" target='_blank'>
              Disrupting Cell Cycle Machinery: CREPT Is an Emerging Target in Cancer Therapy
              </a>
            </td>
          <td>
            Umar Farooq, Jun Li, Zhijie Chang
          </td>
          <td>2025-07-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Viral oncolysis is considered a promising cancer treatment method because of its good tolerability and durable anti-tumor effects. Compared with other oncolytic viruses, Newcastle disease virus (NDV) has some distinct advantages. As an RNA virus, NDV does not recombine with the host genome, making it safer compared with DNA viruses and retroviruses; NDV can induce syncytium formation, allowing the virus to spread among cells without exposure to host neutralizing antibodies; and its genome adheres to the hexamer genetic code rule (genome length as a multiple of six nucleotides), ensuring accurate replication, low recombination rates, and high genetic stability. Although wild-type NDV has a killing effect on various tumor cells, its oncolytic effect and working mechanism are diverse, increasing the complexity of generating engineered oncolytic viruses with NDV. This study aims to employ whole-genome CRISPR-Cas9 knockout screening and RNA sequencing to identify putative key regulatory factors involved in the interaction between NDV and human colon cancer HCT116 cells and map their global interaction networks. The results suggests that NDV infection disrupts cellular homeostasis, thereby exerting oncolytic effects by inhibiting cell metabolism and proliferation. Meanwhile, the antiviral immune response triggered by NDV infection, along with the activation of anti-apoptotic signaling pathways, may be responsible for the limited oncolytic efficacy of NDV against HCT116 cells. These findings not only enhance our understanding of the oncolytic mechanism of NDV against colonic carcinoma but also provide potential strategies and targets for the development of NDV-based engineered oncolytic viruses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e05426a39d80cc8e94e4737d115ccac397145186" target='_blank'>
              Genome-Wide Screening Reveals the Oncolytic Mechanism of Newcastle Disease Virus in a Human Colonic Carcinoma Cell Line
              </a>
            </td>
          <td>
            Yu Zhang, Shufeng Feng, Gaohang Yi, Shujun Jin, Yongxin Zhu, Xiaoxiao Liu, Jinsong Zhou, Hai Li
          </td>
          <td>2025-07-25</td>
          <td>Viruses</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f9a4403306942197c6791291eb7e939f11dbad7" target='_blank'>
              Transposable elements in health and disease: Molecular basis and clinical implications.
              </a>
            </td>
          <td>
            Yaqiang Hong, Nian Liu
          </td>
          <td>2025-08-15</td>
          <td>Chinese medical journal</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="T-cell acute lymphoblastic leukaemia (T-ALL) exhibits exceptionally high levels of DNA methylation, with silencing of the DNA demethylating enzyme TET2 implicated in T-ALL’s hypermethylation phenotype. We propose that DNA hypomethylating agents (HMAs) could be particularly potent in T-ALL cells with this phenotype. Here, we used a reversible DNMT1-specific inhibitor and the conventional HMAs, 5-azacytidine and decitabine, to assess the effects of global DNA methylation loss in T-ALL cell lines and the potential of using HMAs as targeted therapeutic agents in T-ALL. We demonstrate that removal of DNA methylation, even in the absence of DNA damage, results in cell death and that toxicity is negatively correlated with methylation levels. Notably, whereas DNA demethylation caused limited transcriptional changes, key tumour suppressor genes, including TET2, were upregulated in a methylation-dependent manner. Few endogenous retroviruses or immune-related genes were reactivated after demethylation, challenging the contribution of ‘viral mimicry’ to HMA toxicity. Together, these findings provide fundamental insights into the role of DNA methylation in T-ALL, demonstrating that the removal of DNA methylation alone is sufficient to (i) induce cell death in T-ALL cell lines and (ii) reactivate silenced tumour suppressor genes. Our findings support the development of therapies targeting the unique methylation phenotype of T-ALL. Supplementary Information The online version contains supplementary material available at 10.1186/s13148-025-01915-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/815502bed5a016e503e5f469d0f05d1f341e3aa5" target='_blank'>
              Exposing the DNA methylation-responsive compartment of the leukaemic genome in T-ALL cell lines support its potential as a novel therapeutic target in T-ALL
              </a>
            </td>
          <td>
            Maike Bensberg, Aida Selimović-Pašić, Lisa Haglund, Júlia Goldmann, Sandra Hellberg, Colm E. Nestor
          </td>
          <td>2025-07-03</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Transposons are mobile genetic elements capable of moving within the genome. Leveraging this property—particularly the cut-and-paste mechanism of DNA transposons—has enabled the development of technologies for inserting exogenous DNA fragments into host genomes. While targeted integration is a key goal for therapeutic applications, this review highlights the value of their intrinsic randomness. By combining the ability to freely design the DNA cargo with the stochastic nature of transposon integration, it becomes possible to generate highly sensitive reporter cells. These can be used to efficiently identify functional markers, uncover novel signaling pathways, and establish innovative platforms for drug screening. As more subfamilies of transposons become available for research use, their complementary biases may enhance the coverage and diversity of genome-wide screening approaches. Although inherently unpredictable, this strategy embraces randomness as a strength, and we propose that it holds great promise for driving new advances in biology, cellular engineering, and medical research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4d806cc8a8f4473c66fef98ec788d4bd5b59391" target='_blank'>
              Random Insertion Reporter Gimmicks Powered by Cut-and-Paste DNA Transposons
              </a>
            </td>
          <td>
            Yamato Kasahara, Kentaro Semba, Shinya Watanabe, K. Ishikawa
          </td>
          <td>2025-07-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Multiple myeloma (MM) is a hematologic malignancy characterized by abnormal plasma cell proliferation, with lenalidomide emerging as a primary treatment. However, prolonged use often leads to drug resistance, underscoring the need to understand the resistance mechanisms. Protein post-translational modifications (PTMs) play crucial roles in disease development, including chemoresistance. Here, we investigate the involvement of new types of PTMs, focusing on lysine lactylation (Kla), in lenalidomide-resistance. Glycolysis-driven elevation of Kla levels was observed in lenalidomide-resistant MM cells, and the subsequent inhibition of glycolytic activity significantly reversed the lenalidomide-resistance phenotype. Through quantitative proteome, lactylome, and acetylome analyses, we identified 7493 proteins, 1241 Kla sites, and 9313 lysine acetylation (Kac) sites, thereby revealing differential protein expression and PTM profiles in lenalidomide-resistant cells. Proteomic analysis revealed that a series of chemoresistance-related proteins were upregulated, and a number of Cullin-RING Ligase 4-Cereblon (CRL4CRBN) regulatory factors were downregulated. Lactylome analysis revealed that numerous chemoresistance-related proteins exhibited increased Kla levels in lenalidomide-resistant MM cells, suggesting that Kla played an important role in the development of lenalidomide-resistance in LenR MM cells. Notably, histone H4K8la was associated with upregulation of chemoresistance-related genes cyclin-dependent kinase 6 (CDK6) and enoyl-CoA hydratase (ECHS1). Our findings shed light on the epigenetic mechanisms underlying lenalidomide-resistance in MM, offering insights for overcoming chemoresistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07d724e5318db67692c1b88de82a846d5c939a37" target='_blank'>
              Quantitative Proteomics Reveals the Role of Lysine Lactylation in Lenalidomide-Resistance in Multiple Myeloma Cells.
              </a>
            </td>
          <td>
            Xinlong Guo, Xuelian Ren, Cong Yan, He Huang
          </td>
          <td>2025-07-01</td>
          <td>ACS chemical biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="CDKN2A/MTAP co-deletion occurs frequently in non-small cell lung cancer and other solid tumors, including glioblastoma and pancreatic ductal adenocarcinoma. Lung cancer remains the leading cause of cancer-related mortality, and fewer than 15% of glioblastoma or pancreatic cancer patients survive 5 years, underscoring the need for more effective therapies. PRMT5 is a synthetic-lethal dependency in MTAP-null tumors and an attractive therapeutic target for CDKN2A/MTAP-deleted cancers. A new revolutionary class of inhibitors, referred to as MTA-cooperative PRMT5 inhibitors, has shown promising results in ongoing early phase clinical trials. Nonetheless, effective cancer treatment typically requires therapeutic combinations to improve response rates and defeat emergent resistant clones. Thus, we sought to determine whether perturbation of other pathways could improve the efficacy of MTA-cooperative PRMT5 inhibitors. A paralog and single gene targeting CRISPR library was used to screen MTAP-deleted cancers in the presence or absence of MTA-cooperative PRMT5 inhibitors. Loss of several genes sensitized to PRMT5 inhibition, including members of the MAPK pathway. Chemical inhibition of MAPK pathway members using KRAS, MEK, ERK, and RAF inhibitors synergized with PRMT5 inhibition to kill CDKN2A/MTAP-null, RAS-active tumors. Further, MTA-cooperative PRMT5 inhibitors combined with either KRAS or RAF inhibitors led to complete responses in vivo, emphasizing the potential benefit for patients. Lastly, cell lines resistant to KRAS inhibition were not resistant to MTA-cooperative PRMT5 inhibitors and vice versa, suggesting non-cross-reactive mechanisms of resistance. Overall, this study identifies therapeutic combinations with MTA-cooperative PRMT5 inhibitors that may offer significant benefit to patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa2f9d69996913d906ee879ebdccd11283112c02" target='_blank'>
              CRISPR-Drug Combinatorial Screening Identifies Effective Combination Treatments for MTAP-deleted Cancer.
              </a>
            </td>
          <td>
            Nikola Knoll, Sarah Masser, Blanka Bordas, R. Ebright, Guangyan Li, Devishi Kesar, E. Destefanis, Nicholas Kania, Diego J Rodriguez, Jayu Jen, Sydney E Zagar, Caleb Mensah, Zixin Chen, Samuel J Moffitt, Erhumuoghene M Enakireru, Yao He, Baomou Feng, Mira K Chokshi, Cyrus Y Jin, Srivatsan Raghavan, William R Sellers, K. Mulvaney
          </td>
          <td>2025-07-22</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Immunotherapy has emerged as a key modality in cancer treatment, yet its effectiveness varies significantly among patients, often due to the metabolic stress imposed by the tumor microenvironment. Hypoxia, a major factor in the tumor microenvironment, results from the high metabolic rate of tumor cells and inadequate vascularization, impairing immune cells’ function and potentially influencing gene expression profiles. Despite the widespread use of quantitative real-time PCR in immunological studies, to the best of our knowledge, data on reference gene stability in human peripheral blood mononuclear cells under hypoxic conditions is limited. In our study, we assessed the expression stability of commonly used reference genes (S18, HPRT, IPO8, RPL13A, SDHA, PPIA, and UBE2D2) in both non-stimulated and CD3/CD28-activated peripheral blood mononuclear cells cultured under normoxic, hypoxic (1% O2), and chemically induced hypoxic conditions for 24 h. Analysis using four different algorithms—delta Ct, geNorm, NormFinder, and BestKeeper—identified RPL13A, S18, and SDHA as the most suitable reference genes for human peripheral blood mononuclear cells under hypoxic conditions. In contrast, IPO8 and PPIA were found to be the least suitable housekeeping genes. The study provides essential insights into the stability of reference genes in peripheral blood mononuclear cells under hypoxic conditions, a critical but understudied aspect of immunological research. Given the significant impact of hypoxia on T cell metabolism and function in the tumor microenvironment, selecting reliable reference genes is crucial for accurate gene expression analysis. Our findings will be valuable for future studies investigating hypoxia-driven metabolic reprogramming in immune cells, ultimately contributing to a better understanding of T cell responses in cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b494539091eb61f0af4b8a426f9120314cfa058f" target='_blank'>
              Selection of Stable Reference Genes for Gene Expression Studies in Activated and Non-Activated PBMCs Under Normoxic and Hypoxic Conditions
              </a>
            </td>
          <td>
            Artur Wardaszka, A. Smolarska, Piotr Bednarczyk, Joanna Katarzyna Bujak
          </td>
          <td>2025-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25e95b02ea1db5bc99e0be363a686e23ca650900" target='_blank'>
              Single-cell eQTL mapping identifies disease-specific genetic control of COVID-19 infection
              </a>
            </td>
          <td>
            J. Astley, A. J. Kwok, B. Hollis, COMBAT Consortium, C. Dendrou, S. Sansom, J. C. Knight, A. J. Mentzer, Y. Luo, L. Jostins-Dean
          </td>
          <td>2025-06-30</td>
          <td>None</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Abstract Loss or downregulation of major histocompatibility complex (MHC) molecules represents a key mechanism by which tumours escape immune recognition and acquire resistance to immunotherapeutic interventions. This review focuses on the central regulatory pathways. These includes transcriptional repression, lysosomal degradation, and post‐translational modifications that disrupt MHC stability, trafficking, and surface expression. We highlight how these mechanisms impair antigen presentation and contribute to tumour immune evasion. In addition, we explore emerging therapeutic strategies focused on reactivating MHC expression to enhance tumour immunogenicity and improve the efficacy of immunotherapy. Finally, we discuss the translational potential of these approaches and the remaining challenges, including tumour heterogeneity, immunotoxicity and dynamic regulation within the tumour microenvironment, that must be addressed to optimize MHC‐targeted interventions in cancer immunotherapy. Highlights Tumour cells evade immune surveillance by downregulating MHC expression through transcriptional repression, lysosomal degradation and post‐translational modifications. Pharmacological agents interventing epigenetic and metabolic can upregulate MHC expression and improve T cell activation. Combination strategies potentiate immunotherapy efficacy by reinvigorating tumour immunogenicity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b02642acf42e64a6f9032d3b5a78c0a188ca134f" target='_blank'>
              Decoding MHC loss: Molecular mechanisms and implications for immune resistance in cancer
              </a>
            </td>
          <td>
            Pei Lin, Yunfan Lin, Xu Chen, Xinyuan Zhao, Li Cui
          </td>
          <td>2025-07-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Sickle cell disease (SCD) is the most common hemoglobinopathy, caused by a mutation in the β-globin gene of hemoglobin. It predisposes patients to painful Vaso-occlusive crises (VOC) and multi-organ dysfunctions. The disease exhibits significant phenotypic variability, making it challenging to predict severity and outcomes. This study aimed to characterize the whole blood gene expression profile of Bahraini SCD patients, identifying differentially expressed genes during steady-state (n = 10) and VOC (n = 10) compared to healthy controls (n = 8). Analysis revealed 2073 and 3363 dysregulated genes during steady-state and VOC, respectively, compared to controls, with 1078 genes differentially expressed during VOC versus steady-state. Gene Ontology (GO) enrichment analysis highlighted significant deregulation in immune and hematopoietic pathways, including down-regulation of critical genes for immune modulation and hematopoietic balance. Notably, the transcription factor RUNX3, involved in immune cell differentiation and inflammation, was among the 668 down-regulated genes. RUNX3 was four-fold down-regulated in microarray analysis, three-fold in PCR, and showed a mean protein concentration of 11.13 pg/mL during VOC compared to 457.93 pg/mL during steady-state (p < 0.01). These findings suggest that RUNX3 may serve as a potential biomarker for VOC. Future large-scale validation, additional proteomic studies, and functional investigations are recommended to confirm its clinical utility and significance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04a062fb29ce464eb3bdeb6e6597e44042378d51" target='_blank'>
              The Whole Blood Transcriptomic Analysis in Sickle Cell Disease Reveals RUNX3 as a Potential Marker for Vaso-Occlusive Crises
              </a>
            </td>
          <td>
            S. Taha, Hawra Abdulwahab, Muna Aljishi, Ameera Sultan, Moiz Bakhiet, Salvatore Spicuglia, Mohamed Belhocine
          </td>
          <td>2025-06-30</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="


 Control of cell division is tightly regulated in eukaryotic cells, and dysfunction in cell cycle checkpoints is a key hallmark of malignant transformation that promotes a fitness advantage over non-cancer cells. One of the most critical mechanisms of cell cycle regulation is via the cyclin dependent kinases (CDKs), which connect resource availability sensing and growth signaling with cell division and transcription elongation processes. Novel combination therapy approaches to co-target cell cycle and transcriptional CDKs may improve cancer-specific targeting of CDK dysfunction. In the current study, we assessed the effectiveness of fadraciclib, a new CDK2/9 inhibitor, for the treatment of advanced colorectal cancer (CRC).



 A panel of eighteen CRC patient-derived organoids (PDOs) were used to assess the efficacy of fadraciclib. Efficacy was further validated in patient-derived xenografts (PDXs). CDK2/9 target inhibition, cell cycle arrest, and cell killing mechanisms were investigated using western blotting, flow cytometry, and immunofluorescence staining, respectively.



 CRC PDOs exhibited greater sensitivity to fadraciclib compared to chemotherapy and palbociclib. This efficacy was validated in vivo using three matched patient-derived xenografts (PDXs), showing significant tumor growth inhibition with fadraciclib compared to vehicle (p<0.05) and no serious adverse effects. Fadraciclib induced G2/M cell cycle arrest, leading to multipolar mitosis and anaphase catastrophe.



 Our results using patient-derived models suggest that fadraciclib is a promising therapy for advanced CRC by inhibiting CDK2/9, which affects critical pathways in cell cycle regulation and transcription.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f24e0b1ceb93e513f1201f67a1ccdb5200644cb" target='_blank'>
              Dual targeting of conserved cell cycle and transcription programs in advanced colorectal cancer by fadraciclib
              </a>
            </td>
          <td>
            Mohammad Zokaasadi, Wylie K Watlington, Divya L Dayanidhi, John B Mantyh, Gabrielle Rupprecht, Shannon J McCall, David G Blake, Jason A Somarelli, David S Hsu
          </td>
          <td>2025-08-08</td>
          <td>Evolution, Medicine, and Public Health</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Over the years, the relevance of gene therapy has increased as an alternative to conventional drugs. Pharmaceutical industries have been directing their efforts toward the creation of novel gene‐based therapies for a broad range of diseases. RNA‐based therapeutics can efficiently trigger gene silencing and, despite the initial research setbacks, this technology laid the groundwork for advancements in RNA‐based drug design that ultimately contributed to the remarkably rapid advance of messenger RNA (mRNA) vaccines to combat the COVID‐19 pandemic. In light of this information, RNA medicines are currently well‐positioned to have a noteworthy impact in clinical practice. Various RNA medicines have received clinical approval, while others are still in the research phase or undergoing preclinical trials. In this review, an overview of RNA‐based therapeutics is aimed to provide, including antisense oligonucleotides and RNA interference‐based mechanisms, mRNA‐encoded approaches, aptamers, and clustered regularly interspaced short palindromic repeats (CRISPR‐Cas)‐mediated genome editing. Furthermore, the commonly used chemistries in drug delivery systems and strategies for targeting specific cells, as well as the key nanocarriers used in RNA loading, are explored. The main advances and current challenges will be thoroughly summarized.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a834aec20ba942c5a570445a154b6438e49ad84a" target='_blank'>
              RNA‐Based Therapeutics: Cutting‐Edge Advances in Clinical Research
              </a>
            </td>
          <td>
            D. Ferreira, L. R. Rodrigues
          </td>
          <td>2025-07-01</td>
          <td>Advanced Therapeutics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The impact of MYC point mutations on cancer development and progression is poorly explored, particularly compared with other MYC genetic alterations such as translocations and amplifications. MYC point mutations were first observed more than 40 years ago in lymphoid malignancies, and some of these were functionally characterized. In the following decades, only a few studies on MYC point mutations were reported until recently, when analyses of the myriad cancer-related high-throughput sequencing studies brought new life to this research topic and expanded the range of malignancies involved. However, to date, all this information can only be retrieved consulting the specific literature or navigating publicly available databases, and a proper collection and systematic description of these genetic changes is urgently needed. In this review, we run through the steps of the MYC point mutations line of research with a comprehensive illustration of the recurrent variants occurring at the MYC coding, non-coding and regulatory regions in cancer. With this work, we also aim to highlight the current gaps of knowledge to stimulate the research in this field which could ultimately result in the release of its translational potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d123406ed96af6e46fc4b8cbc92394a082d3e153" target='_blank'>
              MYC Point Mutations in Cancer: A Reboot and a Sequel.
              </a>
            </td>
          <td>
            Davide De Luca, Cristina Munafò, L. Lorenzi, Francesco Cucco
          </td>
          <td>2025-07-24</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The induced “recombinogenic situation” in hematopoietic stem cells and the activation of the cell’s reparative systems create the basis for recombination events between fragments of extracellular double-stranded DNA delivered into the cell and chromosomal DNA or other forms of the reparative-recombination process. In mouse and rat model organisms as well as in human bone marrow cells, changes in the amount of telomeric DNA in hematopoietic stem cells were assessed as an indicator of repair and recombination events that have occurred. In all experiments performed, recombinant human angiogenin was used as a comparison factor. Dot blot hybridization showed that in the colony cells obtained from the bone marrow cells of the model organisms as well as from human bone marrow cells treated with a double-stranded DNA preparation, there was a significant increase in the amount of telomeric DNA. Amplification of telomeric DNA in colony cells is not associated with contamination of the original DNA preparation with which the bone marrow cells were treated. Treatment of bone marrow cells with DNA that does not carry telomeric sequences (AluI PCR fragment) does not lead to an increase in the amount of telomeric DNA in the cells of grown colonies. This suggests the participation in the amplification of telomeric DNA of an extrachromosomal DNA template carrying telomeric DNA. It has been established that treatment of bone marrow cells with angiogenin also leads to an increase in telomeric DNA in colony cells. A comparison of the type of colonies with the intensity of hybridization (i. e. the amount of telomeric DNA in the sample) suggests that the increase in the amount of detectable telomeric DNA following treatment with angiogenin and hDNAgr has a fundamentally different origin. Western blot analysis and real-time PCR revealed that the increase in the amount of telomeric DNA following treatment of bone marrow cells with a double-stranded DNA preparation does not correlate with the activity of endogenous/exogenous telomerase. For angiogenin, it has been shown that an increase in the amount of telomeric DNA may be the result of activation of endogenous telomerase activity. A principle has been developed for the amplification of a new genetic trait that came into hematopoietic stem cells with extracellular double-stranded DNA material and was fixed in the recipient genome or was transitively present in the cell as new genetic information.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11f0c77ec133a09bb58f64518224081f769056e8" target='_blank'>
              Concept of natural genome reconstruction. Part 3. Analysis of changes in the amount of telomeric DNA in colony cells as a new amplified feature that arose during the processing of hematopoietic bone marrow stem cells
              </a>
            </td>
          <td>
            V. Ruzanova, S. Oshikhmina, G. S. Ritter, E. Dolgova, S. Kirikovich, E. Levites, Y. R. Efremov, T. V. Karamysheva, A. Bogomolov, M.I. Meschaninova, A. L. Mamaev, O. Taranov, S. V. Sidorov, S. D. Nikonov, O. Leplina, A. Ostanin, E. Chernykh, N. A. Kolchanov, A. Proskurina, S. S. Bogachev
          </td>
          <td>2025-07-01</td>
          <td>Vavilov Journal of Genetics and Breeding</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Conventional wisdom holds that nuclear oncogenes and tumor suppressors initiate malignant transformation. However, mounting research suggests that mitochondrial dysfunction—rooted in the unique evolutionary history and genetic autonomy of mitochondria—may serve as a more fundamental driver of oncogenesis. This paper proposes a “mitochondria-first” hypothesis of cancer, emphasizing the pivotal role of mitochondrial DNA (mtDNA) mutations, metabolic reprogramming, and immune evasion. By examining the evolutionary conflict between host and mitochondria, evaluating high mtDNA mutation rates, and highlighting the disruptive potential of mitochondrial transfer to immune cells, we outline robust mechanisms through which mitochondria could ignite cancer development. We also discuss emerging diagnostic and therapeutic approaches that target mitochondrial integrity, offering a potential paradigm shift in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a09759d1fd0b99bd9f5921041ad32e37eb247e60" target='_blank'>
              Are Mitochondria the True Origin of Cancer? A Hypothesis-Driven Perspective
              </a>
            </td>
          <td>
            Sergio Da Silva
          </td>
          <td>2025-07-01</td>
          <td>Onco</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Simple Summary Non-small cell lung cancer (NSCLC) often harbors oncogenic driver alterations such as epidermal growth factor receptor (EGFR), anaplastic lymphoma kinase (ALK), or Kirsten rat sarcoma viral oncogene homolog (KRAS), which can be effectively targeted with tyrosine kinase inhibitors (TKIs). However, tumors frequently contain additional co-occurring mutations in genes like tumor protein p53 (TP53), serine/threonine kinase 11 (STK11), and Kelch-like ECH-associated protein 1 (KEAP1), which may affect tumor growth, immune evasion, and treatment response. These co-mutations contribute to molecular heterogeneity and are associated with worse prognosis or resistance to therapy. This review summarizes the spectrum and frequency of co-mutations in NSCLC, their prognostic and predictive implications, and their relevance to targeted therapy and immune checkpoint inhibitors (ICIs). Understanding co-mutational profiles is critical to improving precision oncology strategies and tailoring treatment in advanced NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e66415b6ab52cee6417f7da434d635431ca9b4e" target='_blank'>
              Co-Occurring Genomic Alterations in NSCLC: Making Order into a Crowded List
              </a>
            </td>
          <td>
            I. Attili, Federico Pio Fabrizio, F. Marinis
          </td>
          <td>2025-07-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Oncogenic alterations in fibroblast growth factor receptor (FGFR)-family proteins occur across cancers, including pediatric gliomas. Our genomic analysis of 11,635 gliomas across ages finds that 5.3% of all gliomas harbor FGFR alterations, with an incidence of almost 9% in pediatric gliomas. Alterations in FGFR proteins are differentially enriched by age, tumor grade, and histology, with FGFR1 alterations associated with glioneuronal histologies. Leveraging isogenic systems, we confirm FGFR1 alterations to induce downstream Mitogen Activated Protein Kinase (MAPK) and mTOR signaling pathways, drive gliomagenesis, activate neuronal transcriptional programs and exhibit sensitivity to MAPK pathway and pan-FGFR inhibitors. Finally, we perform a retrospective analysis of clinical responses in children diagnosed with FGFR-altered gliomas and find that treatment with currently available inhibitors is largely associated with stability of disease. This study provides key insights into the biology of FGFR1-altered gliomas, therapeutic strategies to target them and associated challenges that still need to be overcome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/328f68005932db97028a1454f9fd6fef0c86bb93" target='_blank'>
              A diverse landscape of FGFR alterations and co-mutations suggests potential therapeutic strategies in pediatric low-grade gliomas
              </a>
            </td>
          <td>
            April A Apfelbaum, Eric Morin, D. Sturm, Georges Ayoub, Jeromy J. Digiacomo, Sher Bahadur, Bhavyaa Chandarana, Phoebe Power, Margaret M Cusick, Dana Novikov, Prem Prabhakar, Robert E Jones, J. Vogelzang, Connor C Bossi, Seth Malinowski, Lewis Woodward, Tania A Jones, John Jeang, Sarah W Lamson, Jared Collins, Kelly Y Cai, Jacquelyn S Jones, Sehee Oh, Hyesung Jeon, Jinhua Wang, A. Cameron, P. Rechter, Angela De Leon, K. Murugesan, M. Montesion, L. Albacker, S. Ramkissoon, Cornelis M. van Tilburg, Emily C Hardin, P. Sievers, Felix Sahm, K. Yeo, Tom Rosenberg, Susan N Chi, Karen D Wright, S. Hébert, Sydney Peck, A. Picca, Valérie Larouche, Samuele Renzi, S. Buhrlage, T. Bale, Amy Smith, M. Touat, Nada Jabado, Eric S Fischer, Michael J Eck, Lissa C. Baird, Olaf Witt, Claudia L Kleinman, Quang-De Nguyen, Denise Sheer, S. Alexandrescu, David T W Jones, Keith L. Ligon, P. Bandopadhayay
          </td>
          <td>2025-07-31</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad16f7c674380966cee80a45864fc4919cc810eb" target='_blank'>
              Multiplexed assays of variant effect for clinical variant interpretation.
              </a>
            </td>
          <td>
            Abbye E. McEwen, Malvika Tejura, Shawn Fayer, Lea M. Starita, Douglas M. Fowler
          </td>
          <td>2025-07-21</td>
          <td>Nature reviews. Genetics</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72ce4746633f2d4d42f9a7aafe673ea0e10428f4" target='_blank'>
              Genome-wide profiling the integration patterns with T7-PCR.
              </a>
            </td>
          <td>
            Ang Li, Chichen Xiao, Jinwen Wang, Xiao Huang, Xionglei He, Kehui Liu, Fan-Nv Xia
          </td>
          <td>2025-08-01</td>
          <td>The Plant journal : for cell and molecular biology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="The androgen receptor inhibitor enzalutamide is one of the principal treatments for metastatic prostate cancer. Most patients respond. However, a subset is primary refractory. Seeking to understand enzalutamide extreme non-response (ENR), we analyzed RNA-sequencing in biopsies from men treated prospectively on an enzalutamide clinical trial. We focused on those with ENR (progression within 3 months) vs. long-term response (progression after 24 months). We identified an ENR program linked to proliferation, epithelial-to-mesenchymal transition, and stemness. High expression of this program in additional datasets was independently linked to poor tumor control with AR targeting but favorable tumor control with docetaxel, another standard treatment. CDK2 was implicated in the ENR program. CDK2 suppression reduced the ENR program and viability of ENR program-high prostate cancer models. The ENR gene program is predictive of non-response to AR targeting. Patients whose tumors harbor this program may be good candidates for docetaxel or CDK2 inhibitor clinical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac30a65263ffc420cc7d1e37eb83811a328b459e" target='_blank'>
              Transcriptional profiling clarifies a program of enzalutamide extreme non-response in lethal prostate cancer
              </a>
            </td>
          <td>
            A. Kumaraswamy, Ya-Mei Hu, Joel A. Yates, Chao Zhang, Eva Rodansky, Dhruv Khokhani, Diana Flores, Zhi Duan, Yi Zhang, Shaadi Tabatabaei, Rachel Slottke, Shangyuan Ye, Primo Lara, Adam Foye, Charles J. Ryan, David A Quigley, Jiaoti Huang, Rahul Aggarwal, R. E. Reiter, Max S. Wicha, Tomasz M. Beer, M. Rettig, Martin Gleave, Christopher P Evans, Owen N. Witte, Joshua M. Stuart, George V. Thomas, Felix Y Feng, Eric J. Small, Zheng Xia, J. Alumkal
          </td>
          <td>2025-07-07</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Background Tumor mutation burden (TMB), the total number of nonsynonymous mutations in the tumor genome, is a well-established biomarker for predicting responses to immune checkpoint blockade (ICB) therapy across various cancers. Patients with high TMB tend to exhibit better responses to ICB. Recently, targeted gene panels have been developed to estimate TMB before treatment. These panels are enriched for calcium ion-binding genes. However, a direct link between TMB and calcium ion-binding genes has not been reported in the literature to date. Methods The association between TMB and calcium ion-binding genes was analyzed using mutation data from The Cancer Genome Atlas (TCGA) database. In addition, a pan-cancer model was constructed to estimate TMB based solely on calcium ion-binding genes. The model’s predictive power for ICB response was validated using independent datasets. Finally, enrichment analysis was performed to investigate the biological connections between calcium ion-binding genes and TMB. Results Calcium ion-binding genes were enriched among the TMB-predictive model genes in 27 out of 33 cancer types. Among these, 19 cancer types exhibited strong predictive performance, with R² values greater than 0.5 in our pan-cancer model based on calcium ion-binding genes. The model effectively estimated TMB and identified ICB responders in independent datasets, including lung adenocarcinoma and melanoma. Enrichment analysis further suggested that calcium ion-binding genes may influence TMB through signal transduction pathways. Conclusions These findings establish a novel association between calcium ion-binding genes and TMB, demonstrating the feasibility of a pan-cancer TMB estimation model based on calcium ion-binding genes. This approach may enhance TMB estimation and improve ICB response prediction across multiple cancers. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03184-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9a4ff34e1fd9730096e49cd5425ebee584e0fa0" target='_blank'>
              Calcium ion-binding genes can predict tumor mutation burden and immune checkpoint blockade response in a pan-cancer model
              </a>
            </td>
          <td>
            Wan-Yu Lin, Chien-jung Huang, Yu-Chao Wang
          </td>
          <td>2025-07-16</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b369f79b329c117da716bed78cecbcb615823b8f" target='_blank'>
              JAK2 signalling to HNRNPA1 represses retrotransposon activity in haematopoietic stem cells
              </a>
            </td>
          <td>
            Jeremy W. Deuel, Andrejs Makarovs, Sheidasadat Goosheh, George Winder, Hisham Mohammed, Jason S. Carroll, Michael Spencer Chapman, C. Kapadia, Peter J. Campbell, J. Nangalia, E. Laurenti, C. Schneider, T. Oellerich, Anthony R Green, S. Loughran
          </td>
          <td>2025-07-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Cancer remains one of the most formidable global health challenges due to its complex pathogenesis, late-stage diagnosis, and limited therapeutic options for many subtypes. Despite advancements in reducing cancer-related mortality through lifestyle modifications and early interventions, it continues to rank among the top ten causes of death worldwide. A growing body of evidence highlights the critical role of non-coding RNAs, particularly microRNAs (miRNAs), in cancer biology. MiRNAs are short, non-coding RNA molecules approximately 19–25 nucleotides in length that regulate gene expression at the post-transcriptional level. Among these, miRNA-1303 has garnered increasing attention due to its aberrant expression across a variety of malignancies and its involvement in diverse cellular processes, including cell proliferation, apoptosis, migration, invasion, and chemoresistance. In this review, we provide a comprehensive overview of miRNA-1303, emphasizing its dysregulation in different cancer types such as colorectal, breast, prostate, and lung cancers. We explore the molecular mechanisms and signaling pathways modulated by miRNA-1303, including its interaction with oncogenes, tumor suppressors, and key cellular networks like PI3K/AKT, Wnt/β-catenin, and MAPK pathways. We also discuss its functional role in tumor development and progression, supported by both in vitro and in vivo studies. Furthermore, we evaluate the potential of miRNA-1303 as a clinically relevant biomarker for early cancer detection, prognosis prediction, and treatment monitoring. Its expression patterns have been correlated with tumor stage, metastasis, and patient survival, suggesting promising utility in clinical decision-making. In addition, we explore the emerging therapeutic strategies targeting miRNA-1303, including miRNA mimics, inhibitors, and delivery systems designed to modulate its expression in cancer cells. By synthesizing current findings, this review underscores the clinical significance of miRNA-1303 in oncology and proposes future directions for its application in personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cc4113503577ac3e30314feefc6d7bf89e1993d" target='_blank'>
              MicroRNA-1303 in cancer pathogenesis and therapy: clinical implications for biomarker development and targeted treatment strategies
              </a>
            </td>
          <td>
            Lingzi Zheng, Ling Hu, Bita Badehnoosh
          </td>
          <td>2025-07-11</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Simple Summary Clinical outcomes can be linked to the fundamental role played by the regulatory 3D architecture of the genome in a living cell and the unique properties offered as a molecular biomarker modality. Here, we describe how this molecular phenomenon acts as an imprint of the cellular genetic, epigenetic and metabolic status which form discriminatory stable conditional structures that inform on the state of the cell and its phenotype. Consequently, 3D architecture, as a biomarker, offers a more comprehensive translation from molecular readouts to clinical outcomes, with examples and evidence provided by the highly accurate diagnostic and prognostic clinical tests already reduced to practice in the US and UK. Moreover, with support from Large Language Models, we can elucidate the complexities and inter-relationships of molecular pathways and regulation using a 3D genomic architecture knowledge base, to gain insights into various molecular modalities and associated clinical conditions. Abstract Background/Objectives: The 3-dimensional (3D) architecture of the genome in the nucleus of a living cell plays an unexpected yet fundamental regulatory role in cell biology. As an imprint of the cellular genetic, epigenetic and metabolic status, it discriminates pathological conditions through conditional changes to long-range 3D interactions (up to 300 kb) and thus could act as a powerful molecular biomarker linked closely to clinical outcomes. Methods: Here an assessment is made of the latest paradigm shift in molecular biology from a supply chain where information flows from DNA to RNA to protein, to the concept of heritable 3D folding of the genome reflecting the epigenetic and metabolic state of the cell, and which serves as a molecular biomarker for complex clinical outcome. Results: While biomarkers based on individual components of the supply chain fail to accurately reflect clinical outcomes, 3D genomics offers highly informative insights, exemplified for immuno-oncology and prostate cancer diagnosis by clinical tests of superior performance, already in practice in the US and UK. Conclusions: A more complete understanding of human biology will require models that account for the flow of information to and from the 3D genomic architecture in living cells, together with its regulation and logic. Integrating these principles into biomarker discovery and therapeutic design, along with other frontline approaches in precision medicine, including multi-omics and other system-level tools, will be essential for advancing precision medicine beyond its current limitations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afe42115687f08e85dd377e033fb44be5d4f1cc1" target='_blank'>
              Paradigm Lost
              </a>
            </td>
          <td>
            Jane Mellor, E. Hunter, A. Akoulitchev
          </td>
          <td>2025-06-28</td>
          <td>Cancers</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Over 85% of the population experience acne at some point in their lives, with its severity spanning a quantitative spectrum, from mild, transient outbreaks to more persistent, severe forms of the condition. Moderate to severe disease poses a substantial global burden arising from both the physical and psychological impacts of this highly visible condition. The analytical approach taken in this study aimed to address the impact of variation in the dichotomisation of acne case control status, driven by ascertainment and study design, on effect size estimates across independent genetic association studies of acne. Through a fixed intercept meta-regression framework, we combined evidence genome-wide for association with acne across studies in which case-control status had been ascertained in different settings, allowing for different severity threshold definitions. Across a combined sample of 73,997 cases and 1,103,940 controls of European, South Asian and African American ancestry we identify genetic variation at 165 genomic loci that influence acne risk. There is evidence for both shared and ancestry specific components to the genetic susceptibility to acne and for sex differences in the magnitude of effect of risk alleles at three loci. We observe that common genetic variation explains 13.4% of acne heritability on the liability scale. Consistent with the hypothesis that genetic risk primarily operates at the level of individual pilosebaceous units, a polygenic score derived from this case-control study of acne susceptibility is associated with both self-reported and clinically assessed acne severity in adolescence, further strengthening the link between genetic risk and disease severity. Prioritisation of causal genes at the identified acne risk loci, provides genetic validation of the targets of established and emerging acne therapies, including retinoid treatments. The identified acne risk loci are enriched for genes encoding downstream effectors of RXRA signalling, including SOX9 and components of the WNT and p53 pathways. Illustrating that the control of stem cell lineage plasticity and cellular fate are important mechanisms through which genetic variation influences acne susceptibility within the pilosebaceous unit.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ea8b3a59293aae2a9d4bb67e79e5cb03a1c6579" target='_blank'>
              Genome-wide association meta-regression identifies stem cell lineage orchestration as a key driver of acne risk
              </a>
            </td>
          <td>
            Jessye Maxwell, B. L. Mitchell, Xinyi DuHarpur, Luba M Pardo, W. Witkam, N. Dand, M. Bartels, Michael J Betti, D. Boomsma, Xianjun Dong, Zachary Gerring, S. Finer, Fiona A. Hagenbeek, J. Hottenga, George Hripcsak, L. Huilaja, Kristian Hveem, Benjamin M. Jacobs, M. Kals, James Kaufman-Cook, J. Kettunen, Atlas Khan, K. Kingo, K. Kiryluk, M. Løset, G. Lunter, M. Lupton, Josine L Min, Nicholas G Martin, S. Medland, D. Neijzen, T. Nijsten, T. Nikopensius, C. Olsen, Lynn Petukhova, A. Reigo, M. E. Rentería, Rossella Rispoli, J. Saklatvala, E. Sliz, Kaisa Tasanen-Määttä, Laurent F Thomas, R. Trembath, M. Vaht, D. V. van Heel, C. Weng, D. C. Whiteman, Jonathan Barker, Catherine Smith, Michael A. Simpson
          </td>
          <td>2025-06-28</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>129</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/743f697012228792a8f0e4e8d5e5913ecf0ed66d" target='_blank'>
              Targeting epigenetic modifications as an emerging immunotherapeutic strategy for cancers.
              </a>
            </td>
          <td>
            R. I. Kumar, Kavya Jain, Karan Raj Rai, Harshnna Gururajan, Koustav Sarkar
          </td>
          <td>2025-08-19</td>
          <td>Immunologic research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="PURPOSE
Tumor next-generation sequencing (NGS) enables molecular profiling to identify actionable mutations and guide targeted therapy. Adolescent and young adults (AYAs) with advanced sarcoma often face poor prognosis and limited systemic options. This study evaluated the clinical utility of tumor NGS in this population.


MATERIALS AND METHODS
A retrospective cohort study was conducted at Chris O'Brien Lifehouse, Sydney, including consecutive AYA patients with sarcoma who underwent tumor NGS between 2015 and 2025. Sequencing was performed through four molecular screening programs: Molecular Screening and Therapeutics, Cancer Screening Program, Personalized Rare and Infrequent Solid Malignancies, Zero Childhood Cancer Program, and commercial platform Foundation Medicine (FoundationOne CDx). Platforms used whole-genome sequencing, whole-exome sequencing, or targeted panels, assessing between 324 and 523 genes. Analyzed biomarkers included somatic mutations, gene fusions, tumor mutational burden (TMB), and microsatellite instability. Key end points included frequency of actionable mutations, treatment changes, and outcomes from NGS-guided therapies.


RESULTS
NGS was successfully performed in 108 of 115 patients (94%). Actionable mutations were identified in 28 patients (24.4%), with molecular confirmation of diagnosis in four (3.5%). TMB was reported in 93 cases: 88 (94.6%) had low TMB (<10 mut/MB) and five (5.4%) had high TMB (≥10 mut/MB). Sixteen patients (14.8%) received NGS-directed therapy, mostly through clinical trials. Of them, 12 (75%) experienced disease progression, and only five (4.4%) derived clinical benefit (stable disease, partial response, or complete response). Among high TMB patients accessing immune checkpoint inhibitors, all but one progressed.


CONCLUSION
Although few AYA patients with sarcoma derived clinical benefit from NGS-directed therapy (<5%), NGS proved useful for diagnosis and trial access. The absence of routine RNA sequencing may have limited the detection of fusion-driven sarcomas. Future integration of broader molecular techniques and collaborative infrastructure is essential to fully realize the potential of NGS in rare and high-risk sarcomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6e23b0b4b64aa82dabcbaf00cdf104ed2a2fccc" target='_blank'>
              Real-World Clinical Utility of Tumor Next-Generation Sequencing in Adolescent and Young Adult Patients With Sarcoma.
              </a>
            </td>
          <td>
            Karan Gupta, P. Grimison, Karl Fernandez, Victoria Choi, V. Bhadri
          </td>
          <td>2025-07-01</td>
          <td>JCO precision oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Anthracyclines have been clinically well established in cancer chemotherapy for decades. The main limitations of this drug class are the development of resistance and severe side effects. In the present investigation, we analyzed 30 anthracyclines in a panel of 59 cell lines of the National Cancer Institute, USA. The log10IC50 values varied from −10.49 M (3′-deamino-3′-(4″-(3″-cyano)morpholinyl)-doxorubicin, 1) to −4.93 M (N,N-dibenzyldaunorubicin hydrochloride, 30). Multidrug-resistant NCI-ADR-Res ovarian cancer cells revealed a high degree of resistance to established anthracyclines (between 18-fold to idarubicin (4) and 166-fold to doxorubicin (13) compared to parental, drug-sensitive OVCAR8 cells). The resistant cells displayed only low degrees of resistance (1- to 5-fold) to four other anthracyclines (7, 18, 28, 30) and were even hypersensitive (collaterally sensitive) to two compounds (1, 26). Live cell time-lapse microscopy proved the cross-resistance of the three chosen anthracyclines (4, 7, 9) on sensitive CCRF/CEM and multidrug-resistant CEM/ADR5000 cells. Structure–activity relationships showed that the presence of tertiary amino functions is helpful in avoiding resistance, while primary amines rather increased resistance development. An α-aminonitrile function as in compound 1 was favorable. Investigating the mRNA expression of 49 ATP-binding cassette (ABC) transporter genes showed that ABCB1/MDR1 encoding P-glycoprotein was the most important one for acquired and inherent resistance to anthracyclines. Molecular docking demonstrated that all anthracyclines bound to the same binding domain at the inner efflux channel side of P-glycoprotein with high binding affinities. Kaplan–Meier statistics of RNA sequencing data of more than 8000 tumor biopsies of TCGA database revealed that out of 23 tumor entities high ABCB1 expression was significantly correlated with worse survival times for acute myeloid leukemia, multiple myeloma, and hepatocellular carcinoma patients. This indicates that ABCB1 may serve as a prognostic marker in anthracycline-based chemotherapy regimens in these tumor types and a target for the development of novel anthracycline derivatives.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0509a300730a6303969f2ff8af79884f58309d0" target='_blank'>
              Predictive and Prognostic Relevance of ABC Transporters for Resistance to Anthracycline Derivatives
              </a>
            </td>
          <td>
            Rümeysa Yücer, Rossana Piccinno, Ednah Ooko, M. Dawood, Gerhard Bringmann, T. Efferth
          </td>
          <td>2025-07-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/033e837d89a8b29610b2b0ee6157dde8b14222f2" target='_blank'>
              A circular RNA overcomes acquired resistance to BET inhibitors by antagonizing IGF2BP2-mediated c-MYC translation in TNBC.
              </a>
            </td>
          <td>
            Jiawei Guo, Ke Li, Yue Ming, Yitong Pan, Shuangyan Tan, Hulin Ma, Shuang Chen, Yingying Duan, Yong Peng
          </td>
          <td>2025-07-08</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>1</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87e2a56734bd4735ea966fc9adf5ca89a77f1711" target='_blank'>
              Increased antiandrogen enzalutamide sensitivity via altering prostate cancer stem cell traits through modulating the androgen receptor-mediated CDR1/circCDR1-AS/miR-1290/BMP4 signaling.
              </a>
            </td>
          <td>
            Diwei Huo, Minggui Si, Kexin Yu, Xiaoxue Fang, Hongbo Liu, Donglong Li, Zhengxing Chen, Jinguo Li, Ruicong Xu, Xinwang Su, Yongfeng Du, Xuebin Ma, Xunwei Wang, Pengbo Li, Huike Yang, Xiujie Chen, Keliang Wang
          </td>
          <td>2025-07-04</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="PURPOSE
The development of DNA damage response (DDR)-directed therapies is a major area of clinical investigation, yet to date Poly (ADP-ribose) polymerase inhibitors (PARPi) remain the only approved therapy in this space. Major challenges to DDR-targeted therapies in the post-PARPi era are the context dependency of DDR alterations and the presence of pre-existing resistance in this heavily pre-treated population. Blood samples from patients with tumors harboring defects in DDR genes were evaluated the feasibility of liquid biopsy platform for detecting complex genomic events such as BRCA1/2 reversions, HRD signatures, PV allele status, and differentially methylated regions for accurate quantitation of TF.


PATIENTS AND METHODS
Overall, 173 patients enrolled in two Phase 1/2 clinical trials (TRESR; NCT04497116, ATTACC; NCT04972110) were selected. The pre-treatment circulating tumor DNA (ctDNA) samples were analyzed from these patients, harboring pathogenic variants (PVs) in DDR genes.


RESULTS
In a phase I heavily pretreated patient population with DDR defects, ctDNA can detect complex genomic alterations (HRD, biallelic loss, complex reversions) that historically require tumor tissue biopsies. Within the cohort of BRCA-associated tumor types previously treated with PARPi or platinum, HRD reversions were detected in 44% of evaluable patients and included large genomic rearrangements leading to deletion of whole or partial exons which have been underrepresented in the literature due to technological limitations.


CONCLUSIONS
This study showcases the genomic complexity of DDR-altered tumors as revealed through baseline ctDNA profiling, an understanding of which is crucial for the future clinical development of novel DDR-directed therapies and combinations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/043203747a8ddc3d9b7f71fc5c57dd4716d7a3cc" target='_blank'>
              Genomic and epigenomic ctDNA profiling in liquid biopsies from heavily pre-treated patients with DNA damage response-deficient tumors.
              </a>
            </td>
          <td>
            Ian M. Silverman, Joseph D Schonoft, B. Herzberg, A. Yablonovitch, E. Lagow, Patrick C. Fiaux, Pegah Safabakhsh, Sunantha Sethuraman, Danielle Ulanet, Julia Yang, Insil Kim, P. Basciano, Michael Cecchini, Elizabeth K. Lee, Stephanie Lheureux, E. Fontana, B. A. Carneiro, J. Reis-Filho, T.A. Yap, M. Zinda, Ezra Y Rosen, V. Rimkunas
          </td>
          <td>2025-07-24</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Oncogenic KRAS induces metabolic rewiring in pancreatic ductal adenocarcinoma (PDAC) characterized, in part, by dependency on de novo pyrimidine biosynthesis. Pharmacologic inhibition of dihydroorotate dehydrogenase (DHODH), an enzyme in the de novo pyrimidine synthesis pathway, delays pancreatic tumor growth; however, limited monotherapy efficacy suggests that compensatory pathways may drive resistance. Here, we use an integrated metabolomic, proteomic and in vitro and in vivo DHODH inhibitor-anchored genetic screening approach to identify compensatory pathways to DHODH inhibition (DHODHi) and targets for combination therapy strategies. We demonstrate that DHODHi alters the apoptotic regulatory proteome thereby enhancing sensitivity to inhibitors of the anti-apoptotic BCL2L1 (BCL-XL) protein. Co-targeting DHODH and BCL-XL synergistically induces apoptosis in PDAC cells and patient-derived organoids. The combination of DHODH inhibition with Brequinar and BCL-XL degradation by DT2216, a proteolysis targeting chimera (PROTAC), significantly inhibits PDAC tumor growth. These data define mechanisms of adaptation to DHODHi and support combination therapy targeting BCL-XL in PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be9183ca2d7210b8ec4357f3959c0684f349b63b" target='_blank'>
              De novo pyrimidine biosynthesis inhibition synergizes with BCL-XL targeting in pancreatic cancer
              </a>
            </td>
          <td>
            Huan Zhang, Naiara Santana-Codina, Qijia Yu, Clara Poupault, Claudia Campos, Xing-Lei Qin, Nicole Sindoni, Marina Ciscar, A. Padhye, Miljan Kuljanin, Junning Wang, Matthew J. Dorman, Peter Bross, Andrew J. Aguirre, Stephanie K Dougan, K. Sarosiek, J. Mancias
          </td>
          <td>2025-07-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Background LIN28B, an oncofetal RNA-binding protein regulating stem cell self-renewal and oncogenic signaling via let-7 miRNA suppression, is implicated in diverse malignancies but remains poorly characterized in juvenile myelomonocytic leukemia (JMML). Despite its reported overexpression in approximately 50% of JMML cases, the epigenetic mechanisms driving LIN28B dysregulation and its clinical relevance for risk stratification are undefined. Therefore, this study aimed to elucidate the epigenetic regulation of LIN28B in JMML, and determine its potential as a biomarker for risk stratification. Methods We analyzed LIN28B expression and methylation in 24 JMML patients and validated the methylation classification in an independent validation cohort (n=62). Gene Set Enrichment Analysis (GSEA) of Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) was conducted to find out the potential mechanisms underlying the differentially expressed LIN28B. Results Among 24 JMML patients, more than 58% showed overexpression of LIN28B, with the majority exhibiting hypomethylation in a newly identified upstream promoter region. Patients with LIN28B-positive showed concurrent upregulation of WT1, POU1F1 and GATA5, downregulation of HEY1 and enriched for cell cycle pathways. Hierarchical clustering identified a high-methylation (HM) subgroup enriched for adverse features: age >2 years, PTPN11 mutations, and hypermethylation phenotypes. In an independent cohort, the HM patients demonstrated inferior 5-year overall survival (OS, 49.2% vs. 87.0%) and event-free survival (EFS, 40.1% vs. 87.0%) versus low-methylation counterparts. Conclusions LIN28B activation via promoter hypomethylation defines a high-risk JMML subgroup with cell cycle. Methylation-based stratification predicts survival, positioning LIN28B as a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be70c41a467e73a4830c835951fac94a4a1ecf8d" target='_blank'>
              LIN28B hypomethylation drives oncogenic signaling and stratifies poor prognosis in juvenile myelomonocytic leukemia
              </a>
            </td>
          <td>
            Junchen Lai, Yan Miao, Xia Qin, Yu-Chen Lin, Chengjuan Luo, Changying Luo, Xinan Wang, Chen Zhou, Jing Chen
          </td>
          <td>2025-07-01</td>
          <td>Translational Pediatrics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Prostate cancer is the most prevalent cancer among men worldwide and exhibits significant genetic heritability. In this study, we performed an integrative analysis combining chromatin accessibility profiling, transcriptomics, and two-stage case-control studies, alongside an unbiased phenome-wide exploration in the FinnGen cohort. This comprehensive approach identified the enhancer-associated single-nucleotide polymorphism rs7077830 at chromosome 10q11 as a critical modulator of prostate cancer susceptibility. Mechanistic investigations revealed that rs7077830 exerted allele-specific enhancer activity, driving NCOA4 expression via a ZNF384-mediated enhancer-promoter interaction. CRISPR/Cas9-mediated single-nucleotide editing confirmed the direct regulatory role of rs7077830 on NCOA4 and its contribution to prostate cancer progression. Functional studies demonstrated that NCOA4 acted as a tumor suppressor by promoting ferroptosis. Furthermore, NCOA4 expression modulated sensitivity to ferroptosis-inducing agents. Strikingly, rs7077830 genotypes were directly linked to ferroptosis status in prostate cancer. These findings highlight the role of noncoding genetic variants in prostate cancer pathogenesis, provide insights into enhancer-driven ferroptosis regulation, and suggest a promising therapeutic avenue for prostate cancer patients carrying specific rs7077830 genotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78490ad8e9f9afb3d3853a6b0cad7e6e00f061a7" target='_blank'>
              Integrated Omics and Multi-Cohort Analyses Identify an Enhancer Variant Linking Ferroptosis to Precision Therapy in Prostate Cancer.
              </a>
            </td>
          <td>
            Sheng Ma, Yue Ge, Zequn Lu, Junbiao Zhang, Qin Zhang, Yuzheng Peng, Zirui Xi, Hao Peng, Meng-Yao Xu, Zezhong Xiong, Yuan Gao, Yanan Wang, Le Li, Chunyu Zhang, Zheng Chao, Tengfei Li, Ruiyun Zhang, Qiang Zhang, David A Horne, Baojun Wang, Xu Gao, Xu Zhang, Jianbo Tian, Jing Li, Meng Jin, Gonghong Wei, Zhihua Wang
          </td>
          <td>2025-08-04</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Breast cancer remains the most prevalent malignant tumor and a leading cause of cancer-related mortality among women worldwide. The disease comprises distinct molecular subtypes that influence tumor behavior, metastatic potential, and therapeutic response. This review presents recent advances in breast cancer research, with a particular emphasis on molecular and epigenetic mechanisms that contribute to tumor development, progression, and treatment resistance. Key signaling pathways—including estrogen receptor (ER), human epidermal growth factor receptor 2 (HER2), Notch, fibroblast growth factor receptor (FGFR), Wnt, and Hedgehog—play essential roles in the regulation of mammary stem cells and oncogenesis. Increasing evidence highlights the significance of epigenetic alterations such as DNA methylation, histone modifications, and microRNA expression in modulating gene activity relevant to tumor initiation and therapy resistance. Epigenetic molecular targets, including DNA methyltransferases (DNMTs), histone deacetylases (HDACs), EZH2, and non-coding RNAs, are gaining attention for their potential use in diagnosis, prognosis, and targeted therapy. Integration of multigene panel testing with epigenetic biomarkers has facilitated improved risk assessment and the development of individualized treatment strategies. Moreover, novel therapeutic approaches—such as CAR-T cell therapy, nanoparticle-mediated drug delivery systems, and the involvement of circular RNAs (circRNAs) in immune modulation—offer promising directions for precision medicine. This review consolidates current insights into the molecular and epigenetic landscape of breast cancer to provide a comprehensive understanding of disease complexity and to inform the development of more effective, personalized treatment options.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8400e20214b339b7e7810249446b370e6720099d" target='_blank'>
              Recent Insights Into Breast Cancer: Molecular Pathways, Epigenetic Regulation, and Emerging Targeted Therapies
              </a>
            </td>
          <td>
            Quratulain Maqsood, Muhammad Umer Khan, Tehreem Fatima, Sania Khalid, Zaryab Ikram Malik
          </td>
          <td>2025-07-01</td>
          <td>Breast Cancer : Basic and Clinical Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="CRISPR/Cas9 gene-editing technology has revolutionized biomedical research by enabling precise, efficient, and cost-effective genome manipulation. This paper focuses on the application of CRISPR/Cas9 in cancer research, with an emphasis on triple-negative breast cancer (TNBC), a particularly aggressive and treatment-resistant subtype of breast cancer that lacks expression of estrogen, progesterone, and HER2 receptors. Breast cancer is the most commonly diagnosed cancer among women worldwide, and TNBC represents one of its most challenging forms due to the absence of effective targeted therapies. While hormone receptor-positive and HER2-positive subtypes benefit from endocrine and targeted treatments, TNBC patients are often limited to chemotherapy, which is associated with high toxicity and limited efficacy. This review synthesizes recent CRISPR-based studies that identify oncogenes and tumor suppressors relevant to TNBC, explores advanced delivery systems such as non-viral nanoparticles, and evaluates the therapeutic promise of this technology. Despite current limitations—including off-target effects, delivery challenges, and ethical concerns—CRISPR/Cas9 offers a promising path toward developing personalized, gene-targeted treatments for TNBC and other hard-to-treat cancers.

Key words: CRISPR/Cas9, gene editing, breast cancer, triple negative breast cancer">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c17f19abfb2d0fe3b181a39613fc90c46bf2f21" target='_blank'>
              Review of Current Applications of CRISPR Gene Editing Techniques in Breast Cancer Therapeutics Development
              </a>
            </td>
          <td>
            Yichen Zhang
          </td>
          <td>2025-08-01</td>
          <td>International Journal of Health Sciences and Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Comprehensive genomic profiling (CGP) is a molecular diagnostic tool with increasing use in cancer research and treatment. There are several commercialized CGP assays with variable targeted genes, while the differences between medium- and large-sized panels in refining genomic profiling strategies, optimizing resource allocation and enhancing clinical applications require evaluation. In the present study, patients with triple-negative breast cancer (TNBC) from the Veterans General Hospital TAipei-Yung-Ling foundation sinO-canceR study were initially assayed using a medium-sized CGP panel (Oncomine comprehensive panel), and the remaining nucleic acid specimens were re-sequenced with a large-sized panel (TruSight Oncology 500). The molecular profiling between the two sequencing panels was compared. A total of 108 breast cancer samples were successfully assayed using both platforms and 272 variants were reported at least once by either type of CGP. The reported variants were among actionable genes (AKT1, BRCA1/2, PALB2, ERBB2, PIK3CA and PTEN), which can be acted upon or have clinical relevance for therapeutic intervention and TP53. The concordance rate between the medium- and large-sized panels was 34.6%, which was enhanced to 58.9% after excluding polymorphisms, out-of-targeted region variants and those with low variant allele frequency (#x003C;10%), with variants of TP53, ERBB2 and PTEN being mostly enhanced. A majority of discordance came from TruSight Oncology 500-detected only variants, especially BRCA1, BRCA2 and PALB2. In conclusion, the results indicated that only one-third of actionable mutations could be detected consistently between the medium- and large-sized CGP panels using the default analytical pipelines, while extensive bioinformatics analyses improved concordance substantially. The large-sized panel detected more variants, thereby enhancing clinical actionability. With more therapeutic targets revealed in the future, CGP may be particularly impactful in refining strategies for TNBC management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec8f8c51dffb87429763b41f009f6f95cce17938" target='_blank'>
              Comprehensive genomic profiling of Taiwanese triple-negative breast cancer samples with medium- and large-sized sequencing panels: A comparative study implicating treatment allocations
              </a>
            </td>
          <td>
            Chi-Cheng Huang, Yi-Chen Yeh, Y. Tsai, Yen-Shu Lin, Ta-Chung Chao, Chun-Yu Liu, Hsiang-Ling Ho, Ling-Ming Tseng
          </td>
          <td>2025-08-05</td>
          <td>Biomedical Reports</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="BACKGROUND/AIM
Comprehensive genomic profiling (CGP) with tissue- and blood-based next-generation sequencing (NGS) is integral to the delivery of personalized medicine for targeted cancer therapy. This study aimed to evaluate the variant concordance for somatic variants using two clinical NGS systems for conducting both tissue- and blood-based analyses: Genexus-OCA v3 (OCA) vs. FoundationOne CDx (F1) for tissues and Genexus OPA (OPA) vs. FoundationOne CDx Liquid (F1L) for blood.


PATIENTS AND METHODS
The concordance of genomic alterations between the two NGS analyses was compared in six patients with breast, head, and neck cancers using tissue and circulating tumor DNA biopsies.


RESULTS
A total of 130 genes were common between F1 and OCA, and 41 between F1L and OPA. When comparing FoundationOne to Genexus for common genes, the sensitivity and specificity of OCA and OPA were 55% and 99%, respectively. Nine single-nucleotide variants (SNVs), one copy number alteration (CNA), and one fusion were detected by both Genexus and FoundationOne. However, one SNV (MAP2K1 F53V), two CNAs (AKT3 and MYC), and one fusion (ESR-CCDC170) were detected only in Genexus, whereas two SNVs (TP53 Q331* and KRAS G12V) were detected only in FoundationOne.


CONCLUSION
The two cancer genome panels were equivalent but not perfect in terms of the detection of variants using tissue and blood, indicating that different assays and analytical methods may have influenced the results. When performing CGPs, it is important to consider the characteristics of each NGS-based CGP test and the genetic variants associated with each disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbe1f8fa1288486304d422d7baf60accd4be21e6" target='_blank'>
              Comparison of Comprehensive Genomic Profiling Testing "Ion Torrent Genexus Sequencer" With FoundationOne.
              </a>
            </td>
          <td>
            Kenji Fujiyoshi, Rie Sugihara, Naoki Miyamoto, Yoriko Watanabe, Tomoya Sudo, Sanae Numata, Jun Akiba, H. Abe, Yuka Ichinose, Kenji Inoue, Shuich Ozono, T. Ono, Kentaro Orioka, Masaki Kashihara, Ryousuke Kajiwara, Hiroyuki Kawano, Akihiko Kawahara, Ryuta Takase, Uhi Toh, Kazuaki Hashimoto, T. Hisaka, Shingo Hirai, M. Mitsuoka, Daiki Miyazaki, Fumi Yoshitomi, Ken Yamamoto, H. Umeno, Masahisa Nomura, Yoshiki Naito
          </td>
          <td>2025-08-01</td>
          <td>Anticancer research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb38d7a583d55b4bb6ef65c3e20e36feb7ce5d7e" target='_blank'>
              Tumor-initiating genetics and therapy drive divergent molecular evolution in IDH-mutant gliomas.
              </a>
            </td>
          <td>
            T. Chowdhury, C. M. S. Tesileanu, E. Kocakavuk, K. Johnson, Jooho Lee, Z. Erson-Omay, Chaewon Heo, Ken Aldape, Samirkumar B Amin, K. Anderson, D.M. Ashley, Jill S. Barnholtz-Sloan, D. Brat, Andrew R Brodbelt, Ana Valeria Castro, Elizabeth B Claus, Jennifer M Connelly, Joseph F Costello, Indrani Datta, Carol Elliott, Gaetano Finocchiaro, P. French, Hui K Gan, L. Garofano, Padmaja L. Ghospurkar, A. Gołȩbiewska, P. S. Gundla, B. Hermes, Chibo Hong, Y. Hoogstrate, Craig Horbinski, Jason T. Huse, A. Iavarone, Cihat Karadag, M. Khasraw, M. Kouwenhoven, Peter S LaViolette, Kay Li, A. Lowman, Katy McCortney, H. Moon, Sanai Nader, M. Nasrallah, H. K. Ng, D. Ormond, Marta Padovan, S. Paek, Laila M. Poisson, Sushant Puri, Erica Shen, Mehta Shwetal, Andrew E. Sloan, W. Vallentgoed, Erwin G. Van Meir, Rachael Vaubel, Taylor E. Wade, A. Walenkamp, C. Watts, Tobias Weiss, Micheal Weller, P. Wesseling, Kerryn Westcott, Bart A Westerman, Gordon Y Ye, W. Yung, F. Varn, R. Verhaak
          </td>
          <td>2025-07-15</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>93</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/862fec5f5a5db3e84ecc51f2f10f8d5bd51650ab" target='_blank'>
              LCK-targeting molecular glues overcome resistance to inhibitor-based therapy in T-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            S. Yoshimura, Marisa Actis, Justin T. Seffernick, Jamie Jarusiewicz, Anup Aggarwal, Angelina Li, Yong Li, DongGeun Lee, Lei Yang, Anand Mayasundari, Zoran Rankovic, Marcus Fischer, Gisele Nishiguchi, Jun J. Yang
          </td>
          <td>2025-07-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9115f0403b6f299c8cd2bfd1aafebb32f08419d1" target='_blank'>
              Telomere Biology Disorder Predisposition in Bone Marrow Failure and Clonal Hematopoiesis
              </a>
            </td>
          <td>
            ∗. DaijingNie, Xvxin Li
          </td>
          <td>2025-07-08</td>
          <td>None</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Response gene to complement 32 (RGC32), a complement activation-inducible factor broadly expressed in normal human tissues, has been implicated in tumorigenesis through its dysregulated expression in various malignancies and its involvement in critical oncogenic processes. Despite its established roles in cancer biology, RGC32 remains uncharacterized in diffuse large B-cell lymphoma (DLBCL). This study provides the first comprehensive investigation of RGC32 expression patterns and functional contributions to DLBCL pathogenesis, elucidating its potential as a novel therapeutic target or prognostic biomarker in this disease. Methods Immunohistochemical (IHC) staining of RGC32 was performed on specimens from 32 Reactive hyperplasia lymphoid (RHL) patients and 80 DLBCL patients. To evaluate the role of RGC32 in DLBCL, lentivirus vectors either encoding shRGC32 or shControl were transfected into DLBCL cell lines. RNA-sequencing (RNA-seq) analysis was performed between shRGC32 and shControl stably transfected OCI-LY1 cells and functional enrichment analyses used gene ontology (GO) and kyoto encyclopedia of genes and genomes (KEGG). In order to explored its functions in vivo, xenograft models were established by subcutaneously injecting shRGC32 and shControl transfected DLBCL cells into SCID beige mice. Results Immunohistochemical analysis revealed RGC32 overexpression in DLBCL tissues contrast with RHL, and was associated advanced Ann Arbor stage (p = 0.043), B symptoms (p = 0.020), and poor progression-free survival (p = 0.015) and overall survival (p = 0.035). Functional studies demonstrated that RGC32 knockdown via shRNA significantly suppressed DLBCL cell proliferation in vitro and in vivo, with xenograft models showing reduced tumor growth and Ki-67 expression. RNA-seq analysis linked RGC32 depletion to downregulation of cell proliferation and impaired DNA damage repair (DDR) mechanisms. Western blot showed RGC32 knockdown could suppress ATM/ATR/CHK1 pathway and increase the tumor mutational burden (TMB). Furthermore, after inhibition of RGC32, infiltration of CD8+ T cells was increased in DLBCL tumor microenvironment (TME). Conclusions This study highlights that RGC32 is a novel molecule in DLBCL progression and might be a potential therapeutic target for DLBCL therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78dc8ef431c9b4ad9b48222a8eef16b0fca96480" target='_blank'>
              Response gene to complement 32 promotes tumorigenesis by mediating DNA damage repair and inhibits CD8+ T cells infiltration in diffuse large B-cell lymphoma
              </a>
            </td>
          <td>
            Xiyuan Zhang, T. Lu, Chunlei Shi, M. Ding, Ling Wang, Xinting Hu, Xin Wang
          </td>
          <td>2025-07-16</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The HOXA gene locus coordinates body patterning, hematopoiesis, and differentiation. While studying blood phenotype-associated variation within the HOXA locus, we identified a genetic variant, rs17437411, associated with globally reduced blood counts, protection from blood cancers, and variation in anthropometric phenotypes. We find that this variant disrupts the activity of a previously unstudied antisense long non-coding RNA (lncRNA) located between HOXA7 and HOXA9 , which we have named HOTSCRAMBL . The HOTSCRAMBL variant disrupts lncRNA function and reduces human hematopoietic stem cell (HSC) self-renewal. Mechanistically, HOTSCRAMBL enables appropriate expression and splicing of HOXA genes in HSCs, most notably HOXA9 , in an SRSF2-dependent manner. Given the critical role of HOXA gene expression in some blood cancers, we also demonstrate that HOTSCRAMBL variation or deletion compromises HOXA -dependent acute myeloid leukemias. Collectively, we show how insights from human genetic variation can uncover critical regulatory processes required for effective developmental gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/854c6d558bc4abf3b6a29b21c9a18aa5b2bee350" target='_blank'>
              Genetic variation reveals a homeotic long noncoding RNA that modulates human hematopoietic stem cells
              </a>
            </td>
          <td>
            Peng Lyu, Gaurav Agarwal, Chun-Jie Guo, Tianyi Ye, Chen Weng, Mateusz Antoszewski, Samantha Joubran, Alexis Caulier, Michael Poeschla, V. G. Sankaran
          </td>
          <td>2025-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Introduction: The PTEN gene is one of the main tumour suppressors, which is essential in regulating cell growth and preventing carcinogenesis. In endometrial cancer, mutations or loss of function of PTEN are frequently associated with disease progression, making this gene a relevant target for studies on molecular mechanisms and potential therapeutic approaches. Uncontrolled activation of the PI3K/AKT pathway, resulting from PTEN inactivation, favours cell proliferation and resistance to apoptosis, contributing to tumour aggressiveness. Methology: This study is based on a systematic review of the scientific literature, analysing articles published between 2010 and 2025 in databases such as PubMed, Scopus and Web of Science. Studies that investigate the relationship between PTEN mutations and the development of endometrial cancer, considering molecular, clinical and therapeutic aspects, were included. The selection of articles followed strict inclusion and exclusion criteria, prioritising research with significant samples and well-established methodologies. Results: The data analysed indicate that PTEN mutations are present in approximately 50% of endometrial cancer cases, being more frequent in high-grade tumours. These mutations compromise the function of the PTEN protein, allowing cells with DNA damage to avoid apoptosis and continue to proliferate. In addition, tumours with PTEN mutations are more resistant to conventional treatments, such as chemotherapy and radiotherapy. Recent studies suggest that inhibition of the PI3K/AKT pathway may be a promising strategy to contain tumour progression in patients with PTEN mutations. Discussion: The presence of PTEN mutations in endometrial cancer reinforces its importance as a prognostic and therapeutic biomarker. Uncontrolled activation of the PI3K/AKT pathway contributes to exacerbated cell proliferation, resistance to apoptosis, and tumour angiogenesis, favouring cancer dissemination. Therapeutic strategies aimed at modulating this pathway are being investigated, including specific PI3K inhibitors and targeted therapies. However, challenges such as heterogeneity of mutations and the interaction of PTEN with other molecular pathways require more personalised approaches for the treatment of the disease. Conclusion: The PTEN gene plays an essential role in regulating cell growth and preventing endometrial cancer. Its mutation or loss of function is directly related to tumour progression, therapeutic resistance, and poor prognosis of the disease. Understanding these mechanisms paves the way for the development of new therapeutic strategies, including targeted therapies and PI3K/AKT pathway inhibitors, which may improve clinical outcomes for patients. Future studies should focus on identifying biomarkers associated with PTEN and improving targeted therapies to optimise the management of endometrial cancer">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/830421e6c05eb941747f7017645e77bb052d1fb7" target='_blank'>
              The Role of PTEN Tumor Suppressor Genes in Endometrial Cancer
              </a>
            </td>
          <td>
            Caio Siqueira De La Paz, Laíza Virnia Cortes Viana, Maísa Marques Bueno
          </td>
          <td>2025-08-06</td>
          <td>International Healthcare Review (online)</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Targeted demethylation by DNA glycosylases (DNGs) results in differential methylation between parental alleles in the endosperm, which drives imprinted expression. Here, we performed RNA sequencing on endosperm derived from DNG mutant mdr1 and wild-type endosperm. Consistent with the role of DNA methylation in gene silencing, we find 96 gene and 86 TE differentially expressed (DE) transcripts that lost expression in the hypermethylated mdr1 mutant. Compared with other endosperm transcripts, the mdr1 targets are enriched for TEs (particularly Helitrons), and DE genes are depleted for both core genes and GO term assignments, suggesting that the majority of DE transcripts are TEs and pseudo-genes. By comparing DE genes to imprinting calls from prior studies, we find that the majority of DE genes have maternally biased expression, and approximately half of all maternally expressed genes (MEGs) are DE in this study. In contrast, no paternally expressed genes (PEGs) are DE. DNG-dependent imprinted genes are distinguished by maternal demethylation and expression primarily in the endosperm, so we also performed EM-seq on hybrids to identify maternal demethylation and utilized a W22 gene expression atlas to identify genes expressed primarily in the endosperm. Overall, approximately ⅔ of all MEGs show evidence of regulation by DNA glycosylases. Taken together, this study solidifies the role of MDR1 in the regulation of maternally expressed, imprinted genes and TEs and identifies subsets of genes with DNG-independent imprinting regulation. Significance Statement This work investigates the transcriptome changes resulting from the loss of function of DNA glycosylase MDR1, revealing that, in wild-type endosperm, targets of MDR1 are expressed predominantly from the maternal allele and this expression is suppressed in mutants. Furthermore, by combining expression data, DNA methylation data, and developmental expression data, we are able to categorize all maternally expressed, imprinted genes based on DNA glycosylase dependent or independent regulatory methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e38df6b4bc62b7ce1643805d5dd45feb9a54b999" target='_blank'>
              MDR1 DNA glycosylase regulates the expression of genomically imprinted genes and Helitrons
              </a>
            </td>
          <td>
            Kaitlin M. Higgins, J. Gent, Sarah N. Anderson
          </td>
          <td>2024-08-01</td>
          <td>The Plant Journal</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Progress in molecular medicine has resulted in novel cancer therapies, with microRNAs (miRNAs) recognized as promising instruments for cancer detection and treatment. MicroRNAs are tiny non-coding RNA sequences that manipulate gene expression at the post-transcriptional stage and are involved in cellular differentiation and death. miR-32-5p demonstrates oncogenic activity in several cancers, while c-MYC oncogene is a well-known driver of cancer, promoting tumor growth by stimulating cell proliferation, blocking apoptosis, and suppressing immune responses. This study aimed to examine how inhibiting miR-32-5p affected the behavior of breast tumor cells (MCF-7), particularly focusing on changes in cellular apoptosis and proliferation, and to investigate its relationship with c-MYC expression. A locked nucleic acid (LNA)-based inhibitor was used to knock down miR-32-5p in MCF-7 breast cancer cells. Cell viability was assessed using MTT assays at 24, 48, and 72 h post-transfection. Apoptotic and necrotic cell populations were differentiated using Annexin-V labeling with Propidium iodide. Expression levels of miR-32-5p and c-MYC were evaluated using quantitative Real-Time PCR before and after transfection with the miR-32-5p inhibitor. Inhibition of miR-32-5p significantly reduced MCF-7 cell viability at 48 h post-transfection (P < 0.002) and increased apoptotic cells to approximately 17% (vs. 0.3% in controls, P < 0.05), concomitant with significant downregulation of c-MYC mRNA (P < 0.006). The lowest level of miR-32-5p expression was observed at 48 h following transfection, with levels gradually increasing by 72 h. This study is the first to demonstrate a plausible regulatory relationship between miR-32-5p and c-MYC in breast tumor cells. The significant reduction in cell viability and increase in apoptosis following miR-32-5p inhibition, likely mediated through c-MYC downregulation, suggests a plausible pathway that may be targeted for therapeutic intervention in breast cancer. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/532ae00d954ab26c4423bce9c0c68dd5069e2e84" target='_blank'>
              Targeting miR-32-5p suppresses c-MYC-driven proliferation and induces apoptosis in MCF-7 breast cancer cells
              </a>
            </td>
          <td>
            Ahmed I Khoder, I. H. El-Sayed, Yasser B M Ali
          </td>
          <td>2025-07-25</td>
          <td>Medical Oncology (Northwood, London, England)</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Tumor gene alterations can serve as predictive biomarkers for therapy response. The nucleotide excision repair (NER) helicase ERCC2 carries heterozygous missense mutations in approximately 10% of bladder tumors, and these may predict sensitivity to cisplatin treatment. To explore the clinical actionability of ERCC2 mutations, we assembled a multinational cohort of 2,012 individuals with bladder cancer and applied the highly quantitative CRISPR-Select assay to functionally profile recurrent ERCC2 mutations. We also developed a single-allele editing version of CRISPR-Select to assess heterozygous missense variants in their native context. From the cohort, 506 ERCC2 mutations were identified, with 93% being heterozygous missense variants. CRISPR-Select pinpointed deleterious, cisplatin-sensitizing mutations, particularly within the conserved helicase domains. Importantly, single-allele editing revealed that heterozygous helicase-domain mutations markedly increased cisplatin sensitivity. Integration with clinical data confirmed that these mutations were associated with improved response to platinum-based neoadjuvant chemotherapy. Comparison with computational algorithms showed substantial discrepancies, highlighting the importance of precision functional assays for interpreting mutation effects in clinically relevant contexts. Our results demonstrate that CRISPR-Select provides a robust platform to advance biomarker-driven therapy in bladder cancer and supports its potential integration into precision oncology workflows.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/298c9b125eade81e30854dcc4c068fdf43533872" target='_blank'>
              Quantitative functional profiling of ERCC2 mutations deciphers cisplatin sensitivity in bladder cancer
              </a>
            </td>
          <td>
            J. Börcsök, Diyavarshini Gopaul, Daphne Devesa-Serrano, Clémence Mooser, Nicolas Jonsson, M. Cagiada, D. Stormoen, M. N. Ataya, B. Guercio, H. Kaimakliotis, G. Iyer, Kresten Lindorff-Larsen, L. Dyrskjøt, Kent W. Mouw, Z. Szallasi, C. S. Sørensen
          </td>
          <td>2025-08-15</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Abstract Metastable epialleles (MEs) are genomic loci at which epigenetic marks are established stochastically during early embryonic development and maintained during subsequent differentiation and throughout life, leading to stable epigenetic and phenotypic variation among genetically identical individuals. Although MEs were first described in mice over 20 years ago, the extent of epigenetic metastability in the mouse genome remains unknown. We present the first unbiased genome-wide screen for MEs in mice. Using deep whole-genome bisulfite sequencing across tissues derived from the three embryonic germ layers in isogenic C57BL/6J mice, we identified only 29 MEs, precisely localizing them and documenting their rarity. Consistent with recent findings, we found no effects of maternal dietary methyl donor supplementation on ME methylation in the offspring, challenging previous assertions that MEs generally exhibit developmental plasticity. Most but not all MEs are associated with intracisternal A-particle (IAP) elements, tending to localize to the 5′ end of the IAP. Additionally, we discovered autosomal regions at which systemic interindividual variation in DNA methylation is associated with sex, providing insights into sex-associated epigenetic development that apparently precedes sexual differentiation. Our findings indicate that expression of transcription factors, including CCCTC-binding factor (CTCF) and specific KRAB zinc finger proteins during early embryonic development, plays a key role in orchestrating stochastic establishment and/or maintenance of DNA methylation at metastable transposable elements. Overall, these findings advance our understanding of the genomic determinants of epigenetic metastability and suggest that interindividual epigenetic variation at MEs is unlikely to be a major determinant of phenotypic variation among isogenic mice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/708a43f6a2e4e91d381b16e3a28eb5d04dfb8092" target='_blank'>
              Mouse metastable epialleles are extremely rare
              </a>
            </td>
          <td>
            Chathura J Gunasekara, U. Maduranga, Taylor Zhang, Jonathan N Wells, Maria S. Baker, Eleonora Laritsky, Yumei Li, C. Coarfa, Yi Zhu, R. Waterland
          </td>
          <td>2025-07-19</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Foot-and-mouth disease (FMD) is an infectious disease in animals caused by the foot-and-mouth disease virus (FMDV). However, the mechanism of FMDV infection in host cells remains unclear. In this study, we utilized 12C6 heavy ion irradiation technology to process BHK-21 cells and systematically screened and evaluated cell lines with distinct virus replication characteristics. We subsequently employed proteomics to detect the differences between these cell lines and the control BHK-21 cell line following 12C6 heavy ion irradiation. Both cell lines exhibited common downregulation of cell adhesion molecules but also exhibited distinct upregulation pathways. In terms of immune and metabolic responses, BHK-5 infection triggered an immune response, including the upregulation of cytokine–cytokine receptor signaling pathways and lysosome-related pathways, while the upregulation of drug metabolism pathways enhanced the ability to metabolize exogenous substances. Conversely, BHK-7 infection tended to promote metabolic pathway changes that favor virus replication, such as the upregulation of folate biosynthesis, polysaccharide degradation, and linolenic acid metabolism pathways. Additionally, we observed significant downregulation of Cbr3 in cell lines that promoted virus replication and significant upregulation in those that inhibited virus replication. Upon validating the results in Cbr3 knockout cells, we found that knocking out Cbr3 could increase FMDV replication by increasing the cellular content of prostaglandin E2 (PGE2), suggesting a close relationship between FMDV replication and PGE2 levels. This method can increase the production efficiency of FMDV vaccines while reducing manufacturing costs. This study innovatively employed 12C6 heavy ion irradiation technology to induce cell transformation and explored its impact on FMDV, offering a new perspective for understanding virus replication mechanisms and potentially providing a target and idea for developing novel antiviral strategies. Supplementary Information The online version contains supplementary material available at 10.1007/s00018-025-05628-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ceb7df4add47002d29186ee424c9a7ff044a17f" target='_blank'>
              The impact of 12C6 heavy ion irradiation-induced cellular mutations on the replication of the foot-and-mouth disease virus and the role of Cbr3
              </a>
            </td>
          <td>
            Xiangdong Song, Shiyu Tao, Fanglan An, Xiaoming Li, Yanyan Chang, Xuerong Liu, Yan Cui
          </td>
          <td>2025-06-28</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>4</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [0, 8],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>