<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-04-07 09:28:51 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="DHFR gene amplification is commonly present in methotrexate (MTX)-resistant colon cancer cells and acute lymphoblastic leukemia. In this study, we proposed an integrative framework to characterize the amplified region by using a combination of single-molecule real-time sequencing, next-generation optical mapping, and chromosome conformation capture (Hi-C). We identified an amplification unit spanning 11 genes, from the DHFR gene to the ATP6AP1L gene position, with high adjusted interaction frequencies on chromosome 5 (~2.2 Mbp) and a twenty-fold tandemly amplified region, and novel inversions at the start and end positions of the amplified region as well as frameshift insertions in most of the MSH and MLH genes were detected. These mutations might stimulate chromosomal breakage and cause the dysregulation of mismatch repair. Characterizing the tandem gene-amplified unit may be critical for identifying the mechanisms that trigger genomic rearrangements. These findings may provide new insight into the mechanisms underlying the amplification process and the evolution of drug resistance. Sequencing a large region of DNA containing many surplus copies of genes linked to drug resistance in colon cancer cells may illuminate how these genomic rearrangements arise. Such regions of gene amplification are highly repetitive, making them impossible to sequence using ordinary methods, and little is known about how they are generated. Using advanced methods, Jeong-Sun Seo at Seoul National University Bundang Hospital in South Korea and co-workers sequenced a region of gene amplification in colon cancer cells. The amplified region was approximately 20 times the length of that in healthy cells and contained many copies of an eleven-gene segment, including a gene implicated in drug resistance. The region also contained mutations in chromosomal repair genes which would disrupt repair pathways. These results illuminate the genetic changes that lead to gene amplification and drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental & Molecular Medicine, Experimental and Molecular Medicine</td>
          <td>4</td>
          <td>49</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum.
                </a>
              </td>
          <td>
            Gustavo S França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>31</td>
          <td>42</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Identifying genetic dependencies in human colon cancer could help identify effective treatment strategies. Genome-wide CRISPR-Cas9 dropout screens have the potential to reveal genetic dependencies, some of which could be exploited as therapeutic targets using existing drugs. In this study, we comprehensively characterized genetic dependencies present in a colon cancer organoid avatar, and validated tumor-specific selectivity of select pharmacologic agents. We conducted a genome-wide CRISPR dropout screen to elucidate the genetic dependencies that interacted with select driver somatic mutations. We found distinct genetic dependencies that interacted with WNT, MAPK, PI3K, TP53, and mismatch repair pathways and validated targets that could be exploited as treatments for this specific subtype of colon cancer. These findings demonstrate the utility of functional genomic screening in the context of personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6f08c27268b9a856764ce30d936babdccabacfc" target='_blank'>
              Comprehensive genomic dependency landscape of a human colon cancer organoid
              </a>
            </td>
          <td>
            Sanam Khalili, Atefeh Mohseninia, Changlong Liu, Carolyn E. Banister, Paige Heine, M. Khazan, Sidney E. Morrison, P. Gokare, Glenn S Cowley, Barbara A. Weir, David Pocalyko, Kurtis E. Bachman, Phillip J. Buckhaults
          </td>
          <td>2025-03-14</td>
          <td>Communications Biology</td>
          <td>1</td>
          <td>14</td>
        </tr>

        <tr id="This article reviews the impact of single-cell sequencing (SCS) on cancer biology research. SCS has revolutionized our understanding of cancer and tumor heterogeneity, clonal evolution, and the complex interplay between cancer cells and tumor microenvironment. SCS provides high-resolution profiling of individual cells in genomic, transcriptomic, and epigenomic landscapes, facilitating the detection of rare mutations, the characterization of cellular diversity, and the integration of molecular data with phenotypic traits. The integration of SCS with multi-omics has provided a multidimensional view of cellular states and regulatory mechanisms in cancer, uncovering novel regulatory mechanisms and therapeutic targets. Advances in computational tools, artificial intelligence (AI), and machine learning have been crucial in interpreting the vast amounts of data generated, leading to the identification of new biomarkers and the development of predictive models for patient stratification. Furthermore, there have been emerging technologies such as spatial transcriptomics and in situ sequencing, which promise to further enhance our understanding of tumor microenvironment organization and cellular interactions. As SCS and its related technologies continue to advance, they are expected to drive significant advances in personalized cancer diagnostics, prognosis, and therapy, ultimately improving patient outcomes in the era of precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b38b69885e5e53fa9260297634ff329303ec057" target='_blank'>
              Single-Cell Sequencing: Genomic and Transcriptomic Approaches in Cancer Cell Biology
              </a>
            </td>
          <td>
            Ana Ortega-Batista, Yanelys Jaén-Alvarado, Dilan Moreno-Labrador, Natasha Gómez, Gabriela García, Erika N. Guerrero
          </td>
          <td>2025-02-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="Epidermal growth factor receptor tyrosine kinase inhibitors (EGFR-TKIs) have demonstrated remarkable efficacy in treating non-small cell lung cancer (NSCLC), but acquired resistance greatly reduces efficacy and poses a significant challenge to patients. While numerous studies have investigated the mechanisms underlying EGFR-TKI resistance, its complexity and diversity make the existing understanding still incomplete. Traditional approaches frequently struggle to adequately reveal the process of drug resistance development through mean value analysis at the overall cellular level. In recent years, the rapid development of single-cell RNA sequencing technology has introduced a transformative method for analyzing gene expression changes within tumor cells at a single-cell resolution. It not only deepens our understanding of the tumor microenvironment and cellular heterogeneity associated with EGFR-TKI resistance but also identifies potential biomarkers of resistance. In this review, we highlight the critical role of single-cell RNA sequencing in lung cancer research, with a particular focus on its application to exploring the mechanisms of EGFR-TKI-acquired resistance in NSCLC. We emphasize its potential for elucidating the complexity of drug resistance mechanism and its promise in informing more precise and personalized treatment strategies. Ultimately, this approach aims to advance NSCLC treatment toward a new era of precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b57a8423fcdeff9dbe3c23c8eb5bf687e8a733ce" target='_blank'>
              Single-Cell RNA Sequencing in Unraveling Acquired Resistance to EGFR-TKIs in Non-Small Cell Lung Cancer: New Perspectives
              </a>
            </td>
          <td>
            Lin Peng, Siyou Deng, Jinjie Li, Yujie Zhang, Li Zhang
          </td>
          <td>2025-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Objective Ovarian cancer (OC) ranks among the foremost causes of mortality in gynecological malignancies, with chemoresistance being the primary factor contributing to unfavorable prognosis. This work seeks to clarify the mechanisms of resistance-related lactylation in OC, intending to offer novel theoretical foundations and therapy strategies for addressing chemoresistance. Methods Through the combined analysis of bulk RNA-seq and single-cell RNA-seq data, we initially found lactylation genes linked to chemoresistance. Subsequently, we employed differential expression analysis, survival analysis, enrichment analysis, and other methodologies to further investigate the roles and molecular mechanisms of these genes in tumor resistance. Ultimately, we investigated the differential expression of these genes in resistant and non-resistant tissues and cells via experimentation. Results We found two candidate genes associated with lactylation chemoresistance, ALDH1A1 and S100A4. Analysis of single-cell data indicated that tumor cells represent the primary cell subpopulation relevant to resistance studies. Subpopulation analysis indicated that several tumor cell subtypes were markedly linked to resistance, with elevated expression levels of ALDH1A1 and S100A4 in the resistant subpopulation, notably correlating with various immunological and metabolic pathways. Analysis of metabolic pathways indicated that oxidative phosphorylation and glycolysis activity was elevated in the resistant subpopulation, and lactic acid buildup was associated with chemoresistance. The investigation of the marker gene protein-protein interaction network in the resistant subgroup elucidated the intricate interactions among these genes. The expression levels of ALDH1A1 and S100A4 in the OC tissues of the platinum-resistant cohort were markedly elevated compared to the sensitive cohort, with a considerable rise in S100A4 expression observed in resistant OC cells, demonstrating co-localization with lactylation. Conclusion This work elucidates the significant function of lactylation in OC chemoresistance and identifies ALDH1A1 and S100A4 as possible genes associated with drug resistance. These findings enhance our comprehension of the mechanisms behind chemoresistance in OC and offer critical insights for the formulation of novel therapeutic options.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e8903cd68465d50c11566b876c6f8d732f1ba01" target='_blank'>
              Integration of scRNA-seq and bulk RNA-seq to reveal the association and potential molecular mechanisms of metabolic reprogramming regulated by lactylation and chemotherapy resistance in ovarian cancer
              </a>
            </td>
          <td>
            Fang Ren, Xiaoao Pang, Feng Jin, Nannan Luan, Houhua Guo, Liancheng Zhu
          </td>
          <td>2025-02-28</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91c35578020ea8ee2f21fa56db96b33c568e8752" target='_blank'>
              Structural patterns and transcriptional effects of integrated Epstein-Barr virus revealed by long-read sequencing and RNA-sequencing in Nasopharyngeal Carcinoma
              </a>
            </td>
          <td>
            Zahra Kardan, Adway Kadam, A. Chai, S. Cheong, Sampath K Loganathan
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a4cbce0b7fb1d47370357ef7f4f5d368e5b4543" target='_blank'>
              Genetic Profiling of Acute and Chronic Leukemia via Next-Generation Sequencing: Current Insights and Future Perspectives
              </a>
            </td>
          <td>
            Laras Pratiwi, Fawzia Hanum Mashudi, Mukti Citra Ningtyas, H. Sutanto, P. Z. Romadhon
          </td>
          <td>2025-03-28</td>
          <td>Hematology Reports</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background Transposable elements (TEs), which constitute nearly half of the human genome, have long been regarded as genomic "dark matter". However, their reactivation in tumor cells, resulting in the production of TE-chimeric transcripts (TCTs), has emerged as a potential driver of cancer progression. The complexity and full extent of these transcripts remain elusive, largely due to the limitations of short-read next-generation sequencing technologies. These methods have struggled to comprehensively capture the diversity and structure of TCTs, particularly those involving short interspersed nuclear elements (SINEs) or closely co-transcribed TEs. Methods Leveraging full-length cDNA sequencing technology based on nanopore sequencing platform, we developed a customized pipeline for identifying and quantifying TCTs in 19 lung adenocarcinoma (LUAD) cell lines. The short-read RNA-seq dataset from a LUAD corhort (~ 200 tumor samples) was employed to validate the identified TCTs and explore their association with tumor progression. To assess the functional roles of a specific TCTs, cell migration and cell proliferation assays were performed. Results We uncovered 208 unique TCT candidates in the LUAD cell lines. Our approach allowed for the identification of cryptic promoters and terminators within non-transposing TEs. Notably, we identified a chimeric transcript involving MIR_HKDC1, which appears to play a significant role in the progression of LUAD. Furthermore, the expression of these TCTs were associated with poor clinical outcomes in a cohort of LUAD patients, suggesting their potential as novel biomarkers for both LUAD progression and prognosis. Conclusions Our study underscores the application of long-read sequencing to unravel the complex landscape of TCTs in LUAD. We provide a comprehensive characterization of TCTs in LUAD, exploring their potential regulatory roles in cancer progression. These findings contribute to a deeper understanding of the genomic intricacies underlying cancer, and offer new directions for the development of targeted therapies and personalized treatment strategies for LUAD. This research highlights the potential of TCTs as both biomarkers and therapeutic targets in the oncogenesis, offering new insights into the interplay between transposable elements and gene regulation in cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-13888-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f1fcc8488427a1fecd1258b2070d8c49656eecc" target='_blank'>
              Long-read RNA sequencing enables full-length chimeric transcript annotation of transposable elements in lung adenocarcinoma
              </a>
            </td>
          <td>
            Yang Li, Yahui Liu, Yingxin Xie, Yaxuan Wang, Jing Wang, Huan Wang, Lin Xia, Dan Xie
          </td>
          <td>2025-03-15</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Mismatch repair (MMR) is a crucial DNA repair pathway that maintains genomic integrity by correcting replication errors and various forms of DNA damage. MMR deficiency (MMRd) leads to increased mutation rates, microsatellite instability, and contributes to tumorigenesis in multiple cancer types. Using a CRISPR-Cas9-mediated knockout assay in human isogenic cell lines, we characterised mutational profiles in MMR-deficient cells. Our findings revealed expected increases in mutation burden and the emergence of known MMR-associated mutational signatures. Notably, we identified a previously unconnected process, SBS57, and linked it to germline single-nucleotide polymorphisms and MMR-driven indels in MMRd cells, establishing its association with tensor signature TS27. Comparative analyses of in vitro MMRd profiles and in vivo tumour data uncovered key differences in mutational signatures, highlighting the biological context dependence of MMR-associated mutations. Furthermore, we provide direct experimental evidence that MMR plays a role in repairing 5-methylcytosine deamination, a repair process previously inferred from tumour sequencing data. These findings offer novel insights into MMR deficiency, shedding light on previously uncharacterised mutational mechanisms and their implications in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85a293399722ef8f7c62c0c36dd67ff9d94d28e7" target='_blank'>
              Dissecting DNA-mismatch-repair-driven mutational processes in human cells
              </a>
            </td>
          <td>
            Michel Owusu, Jörg Menche, Joanna Loizou, Donate Weghorn
          </td>
          <td>2025-02-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fbd7856e741845eb17452c67c39e622695a90ba" target='_blank'>
              Advances in DNA replication research for precision medicine
              </a>
            </td>
          <td>
            Md. Mosharraf Hossen
          </td>
          <td>2025-03-08</td>
          <td>Journal of Bioscience and Environment Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Traditionally, the evolutionary perspective of cancer has been understood as gradual alterations in passenger/driver genes that lead to branching phylogeny. However, in cases of prostate adenocarcinoma and kidney renal cell carcinoma, macroevolutionary landmarks like chromoplexy and chromothripsis are frequently observed. Unfortunately, short-read sequencing techniques often miss these significant macroevolutionary changes, which involve multiple translocations and deletions at the chromosomal level. To resolve such genomic dark matters, we provided high-fidelity long-read sequencing data (78–92 Gb of ~Q30 reads) of six genitourinary tumour cell lines (one benign kidney tumour and two kidney and three prostate cancers). Based on these data, we obtained 12 high-quality, partially phased genome assemblies (Contig N50 1.85–29.01 Mb; longest contig 2.02–171.62 Mb), graph-based pan-genome variant sets (11.57 M variants including 60 K structural variants), and 5-methylcytosine sites (14.68%–27.05% of the CpG sites). We also identified several severe chromosome aberration events, which would result from chromosome break and fusion events. Our cancer genome assemblies will provide unprecedented resolution to understand cancer genome instability and chromosomal aberration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39aabbe27003a6111c9e819cce1949d58b416548" target='_blank'>
              Genome assembly resources of genitourinary cancers for chromosomal aberration at the single nucleotide level
              </a>
            </td>
          <td>
            Hyunho Han, H. Lee, Min Gyu Kim, Yoo Sub Shin, Jin Soo Chung, Jun Kim
          </td>
          <td>2025-04-01</td>
          <td>Scientific Data</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/398c45e50f2a25bf4d4f179535f4ed528664ab8c" target='_blank'>
              Genomic profiling reveals molecular heterogeneity in patients with Richter transformation (RT) and chronic lymphocytic leukemia (CLL)
              </a>
            </td>
          <td>
            Shulan Tian, Hanyin Wang, S. Parikh, Yuanhang Liu, Helen Jin-4 Lee, Erik Jessen, Eric W. Klee, Yucai Wang, Fan Leng, Min Shi, Cinthya J Zepeda-Mendoza, Rong He, Saad J. Kenderian, Linda B. Baughn, D. L. Dyke, P. Hampel, Neil E. Kay, Esteban Braggio, S. Slager, Huihuang Yan, Wei Ding
          </td>
          <td>2025-03-18</td>
          <td>None</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="Introduction A hallmark of small cell lung cancer (SCLC) is its recalcitrance to therapy. While most SCLCs respond to frontline therapy, resistance inevitably develops. Identifying phenotypes potentiating chemoresistance and immune evasion is a crucial unmet need. Previous reports have linked upregulation of the DNA damage response (DDR) machinery to chemoresistance and immune evasion across cancers. However, it is unknown if SCLCs exhibit distinct DDR phenotypes. Methods To study SCLC DDR phenotypes, we developed a new DDR gene analysis method and applied it to SCLC clinical samples, in vitro, and in vivo model systems. We then investigated how DDR regulation is associated with SCLC biology, chemotherapy response, and tumor evolution following therapy. Results Using multi-omic profiling, we demonstrate that SCLC tumors cluster into three DDR phenotypes with unique molecular features. Hallmarks of these DDR clusters include differential expression of DNA repair genes, increased replication stress, and heightened G2/M cell cycle arrest. SCLCs with elevated DDR phenotypes exhibit increased neuroendocrine features and decreased “inflamed” biomarkers, both within and across SCLC subtypes. Clinical analyses demonstrated treatment naive DDR status was associated with different responses to frontline chemotherapy. Using longitudinal liquid biopsies, we found that DDR Intermediate and High tumors exhibited subtype switching and coincident emergence of heterogenous phenotypes following frontline treatment. Conclusions We establish that SCLC can be classified into one of three distinct, clinically relevant DDR clusters. Our data demonstrates that DDR status plays a key role in shaping SCLC phenotypes and may be associated with different chemotherapy responses and patterns of tumor evolution. Future work targeting DDR specific phenotypes will be instrumental in improving patient outcomes. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02291-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/671674f412412e8378b84a598f7d2a87a83c70ce" target='_blank'>
              DNA damage response signatures are associated with frontline chemotherapy response and routes of tumor evolution in extensive stage small cell lung cancer
              </a>
            </td>
          <td>
            Benjamin B. Morris, Simon Heeke, Yuanxin Xi, L. Diao, Qi Wang, Pedro Rocha, Edurne Arriola, Myung Chang Lee, Darren R. Tyson, K. Concannon, K. Ramkumar, C. Stewart, R. Cardnell, Runsheng Wang, Vito Quaranta, Jing Wang, J. Heymach, B. Nabet, D. Shames, C. Gay, L. Byers
          </td>
          <td>2025-03-20</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>107</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4df1cf0d85a605bfa799ff58abdd0710b0b2a957" target='_blank'>
              Optical Pooled Screening for the Discovery of Regulators of the Alternative Lengthening of Telomeres Pathway
              </a>
            </td>
          <td>
            Isabel Quintanilla, Benura Azeroglu, M. A. Sagar, Travis H. Stracker, Eros Lazzerini Denchi, Gianluca Pegoraro
          </td>
          <td>2025-02-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4dd3ee6516c7e617bd89a2ae4e379bb55d71b50" target='_blank'>
              Population-scale cellular GUIDE-seq-2 and biochemical CHANGE-seq-R profiles reveal human genetic variation frequently affects Cas9 off-target activity
              </a>
            </td>
          <td>
            C. Lazzarotto, Yichao Li, Ashley R. Flory, Jacqueline Chyr, Muyu Yang, Varun Katta, Elizabeth Urbina, G. Lee, Rachael K Wood, Azusa Matsubara, S. Rashkin, Jian Ma, Yong Cheng, Shengdar Q. Tsai
          </td>
          <td>2025-02-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/019e491b785ffb0bdd2ceea7ba005ed8f5742762" target='_blank'>
              Relapse-specific genetic patterns in the exomic mutational landscape in acute myeloid leukemia
              </a>
            </td>
          <td>
            Wardah Qureshi, Aisha Jamal, Atia Gohar, Marium Imran Mansuri, Muhammad Irfan, Ishtiaq Ahmad Khan, Muhammad Shakeel
          </td>
          <td>2025-03-25</td>
          <td>Egyptian Journal of Medical Human Genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Aggressive tumors pose ultra-challenges to drug resistance. Anti-cancer treatments are often unsuccessful, and single-cell technologies to rein drug resistance mechanisms are still fruitless. The National Cancer Institute defines aggressive cancers at the tissue level, describing them as those that spread rapidly, despite severe treatment. At the molecular, foundational level, the quantitative biophysics discipline defines aggressive cancers as harboring a large number of (overexpressed, or mutated) crucial signaling proteins in major proliferation pathways populating their active conformations, primed for their signal transduction roles. This comprehensive review explores highly aggressive cancers on the foundational and cell signaling levels, focusing on the differences between highly aggressive cancers and the more treatable ones. It showcases aggressive tumors as harboring massive, cancer-promoting, catalysis-primed oncogenic proteins, especially through certain overexpression scenarios, as predisposed aggressive tumor candidates. Our examples narrate strong activation of ERK1/2, and other oncogenic proteins, through malfunctioning chromatin and crosslinked signaling, and how they activate multiple proliferation pathways. They show the increased cancer heterogeneity, plasticity, and drug resistance. Our review formulates the principles underlying cancer aggressiveness on the molecular level, discusses scenarios, and describes drug regimen (single drugs and drug combinations) for PDAC, NSCLC, CRC, HCC, breast and prostate cancers, glioblastoma, neuroblastoma, and leukemia as examples. All show overexpression scenarios of master transcription factors, transcription factors with gene fusions, copy number alterations, dysregulation of the epigenetic codes and epithelial-to-mesenchymal transitions in aggressive tumors, as well as high mutation loads of vital upstream signaling regulators, such as EGFR, c-MET, and K-Ras, befitting these principles.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbd64f2509541e7332b94c3a04278ee5d7e930fb" target='_blank'>
              Molecular principles underlying aggressive cancers
              </a>
            </td>
          <td>
            Ruth Nussinov, B. R. Yavuz, Hyunbum Jang
          </td>
          <td>2025-02-17</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>3</td>
          <td>47</td>
        </tr>

        <tr id="Over 80% of children with acute lymphoblastic leukemia (pALL) can be cured by treating them with multiple chemotherapeutic agents administered over several years, whereas pALL is incurable with 1-3 medications, suggesting significant variation in drug susceptibility across clonal populations. While bulk sequencing studies indicate that pALL cells contain relatively few genetic variants compared to other cancers, the true extent of genetic diversity at the single-cell level remains unknown. Here, we used three complementary approaches to investigate pALL genetic heterogeneity: error-corrected bulk sequencing, single-cell exome sequencing, and primary template-directed amplification (PTA)-enabled single-cell genome sequencing. We discovered that some ETV6-RUNX1 samples harbor multiple independent ras clones and that individual pALL cells harbor substantially more mutations (mean 3,553 per cell) than detected in bulk samples (mean 965 mutations), with variant signatures suggesting both early and late APOBEC-driven mutagenesis in ETV6-RUNX1 patients. Using PTA-based phylogenetic analysis of over 150 single-cell genomes from four pALL patients, we identified heritable phenotypes associated with specific genetic alterations, including some low-frequency clones that are preferentially selected for during chemotherapy treatment. Our findings reveal previously undetected genetic diversity in pALL and suggest that pre-existing mutations influence treatment response, with implications for future therapeutic strategies. This study provides a high-resolution framework for understanding cancer clonal evolution during treatment, yielding important new insights for developing more effective therapeutic approaches for pALL. Key Points Increasing the accuracy and resolution of sequencing from bulk to error-corrected bulk to single clone to single-cell sequencing reveals increasing levels of genetic diversity in pALL, including the presence of clone-specific driver and treatment resistance-associated mutations. Using PTA-enabled scWGS, single pALL cells contain several fold more mutations per cell than detected in bulk sequencing, which, when multiplied across billions of cells, reveal hidden population-scale genetic complexity. Some phenotypes in pALL are heritable, including treatment resistance, where we identify pre-existing rare clones with relapse-associated mutations that preferentially survive four weeks of standard treatment in patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cfebdd69571f854dae1eb84acf1f1884a093ad9" target='_blank'>
              Measuring Longitudinal Genome-wide Clonal Evolution of Pediatric Acute Lymphoblastic Leukemia at Single-Cell Resolution
              </a>
            </td>
          <td>
            Yakun Pang, Tamara Prieto, V. Gonzalez-Pena, Athena Aragon, Yuntao Xia, Sheng Kao, Sri Rajagopalan, John Zinno, Jean Quentin, Julien Laval, Dennis Yuan, Nathaniel D. Omans, David Klein, Matthew MacKay, I. De Vlaminck, John Easton, William Evans, D. Landau, Charles Gawad
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Genome-wide human genetic studies have identified inherited cis-regulatory loci variants that predispose to cancers. However, the mechanisms by which these germline variants influence cancer progression, particularly through gene expression and proteostasis control, remain unclear. By analyzing genomic data from a gastric cancer (GC) case-control study (2,117 individuals), focusing on the ubiquitin-specific protease (USP) family, we identify the single nucleotide polymorphism (SNP) rs72856331 (G>A) in the promoter region of the proto-oncogene USP47 as a putative susceptibility allele for GC (OR = 0.78, P = 0.015). Mechanistically, the risk allele G is associated with enhanced USP47 expression, mediated by altered recruitment of the transcription factor GLI3 and changes in the epigenetic status at promoter. CRISPR/Cas9-mediated single-nucleotide conversion into risk allele G results in increased GLI3 binding and subsequent USP47 upregulation. The depletion of GLI3 results in a reduction of cancer-related phenotypes, similar to those observed following USP47 knockdown. Furthermore, we identify Snai1 as a deubiquitination target of USP47, explaining USP47-dependent activation of epithelial-mesenchymal transition pathway and tumor progression. Our findings identify an important genetic predisposition that implicates the perturbation of transcription and proteostasis programs in GC, offering insights into prevention and therapeutic strategies for genetically stratified patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc1084b39e6f7bd79c603ae5a0dd8fa214a3f635" target='_blank'>
              An inherited predisposition allele promotes gastric cancer via enhancing deubiquitination-mediated activation of epithelial-to-mesenchymal transition signaling.
              </a>
            </td>
          <td>
            Bolin Tao, Zhenning Wang, Xuanyi Wang, Aixia Song, Jiaxian Liu, Jianan Wang, Qin Zhang, Zhaolin Chen, Zixian Wang, Wenjie Xu, Menghong Sun, Yanong Wang, Ping Zhang, Tao Xu, Gong-Hong Wei, F. X. Chen, Mengyun Wang
          </td>
          <td>2025-02-25</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is a highly heterogeneous tumor, with significant variation in malignant cells, posing challenges for treatment and prognosis. However, this heterogeneity offers opportunities for personalized therapy. Methods The consensus non-negative matrix factorization algorithm was employed to analyze single-cell transcriptomic data from CRC, which helped identify malignant cell expression programs (MCEPs). Subsequently, a crosstalk network linking MCEPs with immune/stromal cell trajectory development was constructed using Monocle3 and NicheNet. Additionally, bulk RNA-seq data were utilized to systematically explore the relationships between MCEPs, clinical features, and genetic mutations. A prognostic model was then established through Lasso and Cox regression analyses, integrating clinical data into a nomogram for personalized risk prediction. Furthermore, key genes associated with MCEPs and their potential therapeutic targets were identified using protein-protein interaction networks, followed by molecular docking to predict drug-binding affinity. Results We classified CRC malignant cell transcriptional states into eight distinct MCEPs and successfully constructed crosstalk networks between these MCEPs and immune or stromal cells. A prognostic model containing 15 genes was developed, demonstrating an AUC greater than 0.8 for prognostic evaluation over 1 to 10 years when combined with clinical features. A key drug target gene TIMP1 was identified, and several potential targeted drugs were discovered. Conclusion This study demonstrated that characterization of the malignant cell transcriptional programs could effectively reveal the biological features of highly heterogeneous tumors like CRC and exhibit significant potential in tumor prognosis assessment. Our research provides new theoretical and practical directions for CRC prognosis and targeted therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2db62177c3b56c216ca489cc0239cfb1c0df367f" target='_blank'>
              Single-cell and spatial transcriptomic analysis reveals tumor cell heterogeneity and underlying molecular program in colorectal cancer
              </a>
            </td>
          <td>
            Teng Wang, Zhaoming Chen, Wang Wang, Heng Wang, Shenglong Li
          </td>
          <td>2025-03-12</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db41c8c1cf9654af3ad975bf6a786aaf0b60bc69" target='_blank'>
              Somatic repetitive element insertions Define New Biomarkers in Pan-cancer genome
              </a>
            </td>
          <td>
            Lin Xia, Kailing Tu, Huan Wang, Xuyan Liu, Yahui Liu, Qilin Zhang, Ziting Feng, Chunyan Yu, Maolin Li, Yan Huang, Jing Wang, Ranlei Wei, Tianfu Zeng, Xuenan Pi, Chen Ling, Qianwen Zhang, Yixin Qiao, Shengzhong Hou, Zhuoyuan Zhang, Ailin Wei, Haotian Liao, Wenhao Guo, Jifeng Liu, Zhong Wu, Zhoufeng Wang, Ang Li, Weimin Li, Dan Xie
          </td>
          <td>2025-03-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background and Objective: The discovery of novel molecular biomarkers via next-generation sequencing technologies has revolutionized how glioblastomas (GBMs) are classified nowadays. This has resulted in more precise diagnostic, prognostic, and therapeutic approaches to address this malignancy. The present work examines the applications of single-cell RNA sequencing (scRNA-seq) in GBM, focusing on its potential to address tumor complexity and therapeutic resistance and improve patient outcomes. Methods: A scoping review of original studies published between 2009 and 2024 was conducted using the PUBMED and EMBASE databases. Studies in English or Spanish related to single-cell analysis and GBM were included. Key Findings: The database search yielded 453 publications. Themes related to scRNA-seq applied for the diagnosis, prognosis, treatment, and understanding of the cancer biology of GBM were used as criteria for article selection. Of the 24 studies that were included in the review, 11 focused on the tumor microenvironment and cell subpopulations in GBM samples, 5 investigated the use of sequencing to elucidate the GBM cancer biology, 3 examined disease prognosis using sequencing models, 3 applied translational research through scRNA-seq, and 2 addressed treatment-related problems in GBM elucidated by scRNA-seq. Conclusions: This scoping review explored the various clinical applications of scRNA-seq technologies in approaching GBM. The findings highlight the utility of this technology in unraveling the complex cellular and immune landscapes of GBM, paving the way for improved diagnosis and personalized treatments. This cutting-edge approach might strengthen treatment strategies against tumor progression and recurrence, setting the stage for multi-targeted interventions that could significantly improve outcomes for patients with aggressive, treatment-resistant GBMs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50f255f1e2df6321b7a03cd03e70b813bcc99665" target='_blank'>
              Current Applications of Single-Cell RNA Sequencing in Glioblastoma: A Scoping Review
              </a>
            </td>
          <td>
            Edgar G. Ordóñez-Rubiano, Nicolás Rincón-Arias, William J. Shelton, A. F. Salazar, María Alejandra Sierra, Raphael Bertani, D. Gómez-Amarillo, Fernando Hakim, Matías Baldoncini, César Payán-Gómez, A. Cómbita, Sandra C. Ordonez-Rubiano, Rafael Parra-Medina
          </td>
          <td>2025-03-01</td>
          <td>Brain Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1e688482cb691231e9728515c865331942ee770" target='_blank'>
              Integrating gene expression, mutation and copy number data to identify driver genes of recurrent chromosome-arm losses
              </a>
            </td>
          <td>
            Ron Saad, Ron Shamir, Uri Ben-David
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Breast cancer is the most prevalent neoplasm in women. ER+ (Luminal subtype), representing over 70% of breast tumors, is a genetically diverse group. Structural and Numerical-Chromosomal instability initiates tumor development and is recognized as the primary driver of genetic alteration in luminal breast tumors. Genomic instability refers to the increased tendency of cancer cells to accumulate genomic alterations during cell proliferation. The cell cycle check-point response to constant and stable genomic alterations in tumor cells drives this process. The impact of CNV patterns and aneuploidies in cell cycle and proliferation perturbation has recently been highlighted by scientists in Luminal breast tumors. The impact of chromosomal instability on cancer therapy and prognosis is not a new concept. Still, the degree of emerging genomic instability leads to prognosis alteration following cell cycle deregulation by chromosomal instability could be predicted by CNVs-based reclassification of breast tumors. In this review, we try to explain the effect of CIN in the cell cycle that ended with genomic instability and altered prognosis and the impact of CIN in decision-making for a therapy strategy for patients with luminal breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06bef9c6cdff104c948a88f734e3a20c0747386c" target='_blank'>
              A new insight into the impact of copy number variations on cell cycle deregulation of luminal-type breast cancer
              </a>
            </td>
          <td>
            Amir Mahdi Khamaneh, Davoud Jafari-Gharabaghlou, Khalil Ansarin, Pouya Pazooki, Zahra Akbarpour, B. Naghili, Nosratollah Zarghami
          </td>
          <td>2025-02-12</td>
          <td>Oncology Reviews</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Colorectal cancer (CRC) represents the third-leading cause of cancer-related deaths. Knowledge covering diverse cellular and molecular data from individual patients has become valuable for diagnosis, prognosis, and treatment selection. Here, we present an in-depth comparative mRNA-seq and microRNA-seq analysis of tissue samples from 32 CRC, pairing tumors with adjacent healthy tissues. The differential expression gene (DEG) analysis revealed an interconnection between nutrients, metabolic programs, and cell cycle pathways. We focused on the impact of overexpressed SLC7A11 (xCT) and SLC3A2 genes which compose the cystine/glutamate transporter (Xc-) system. To assess the oncogenic potency of the Xc-system in a cellular setting, we applied a knowledge-based approach for analyzing gene perturbations from CRISPR screens across various cell types as well as using a variety of functional assays in five primary patient-derived organoid cell models to functionally verify our hypothesis. We identified a previously undescribed cell surface protein signature predicting chemotherapy resistance and further highlighted the causality and potential of pharmacological blockage of ferroptosis as promising avenue for cancer therapy. Biological processes such as redox homeostasis, ion/amino acid transporters and de novo nucleotide synthesis were associated with these co-dependent genes in patient specimens. This study highlighted a number of overlooked genes as potential clinical targets for CRC and promotes stem cell-based, patient-individual in vitro model systems as a versatile partner platform to functionally validate in silico predictions, with focus on SLC7A11 and its associated genes in tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06923ea47f54b55109b8b50bd2b097751afa1aed" target='_blank'>
              Patient-Specific Pharmacogenomics unveils xCT as a Key Regulator for Druggable Metabolic and Proliferation Pathways in Colon Cancer
              </a>
            </td>
          <td>
            Marco Strecker, Keren Zohar, Martin Böttcher, Thomas Wartmann, Henry Freudenstein, Maximilian Doelling, Mihailo Andric, W. Shi, O. Kakhlon, Katrin Hippe, Beatrix Jahnke, Dimitrios Mougiakakos, Franziska Baenke, Daniel Stange, R. Croner, M. Linial, U. D. Kahlert
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Chimeric Antigen Receptor T (CAR-T) cells offer a promising strategy for cancer treatment. These CAR-T cells are either autologous or allogeneic T cells that are genetically modified to express a chimeric antigen receptor targeting a specific tumor antigen. Ongoing research aims to optimize the CAR-T cell efficacy, including strategies to modulate their metabolism. One such approach involves inducing transgene expression by activating the GCN2 kinase signaling pathway through dietary deprivation of an essential amino acid. In this study, we investigated the general impact of a 6-hour leucine deprivation on primary activated human T cells using RNA-seq technology. Our analysis identified 3,431 differentially expressed genes between T cells cultured in regular medium and those cultured in leucine-deprived medium. Gene Set Enrichment Analysis revealed that “TNFα signaling via NFκB”, “interferon-γ response”, and “unfolded protein response” gene sets were positively enriched, while “mTORC1 signaling”, “Myc targets”, and “oxidative phosphorylation” gene sets were negatively enriched. To further evaluate the involvement of GCN2 kinase in regulating the differential gene expression during the 6-hour leucine deprivation, T cells were cultured with or without a GCN2 inhibitor. We found that 59% of the differentially expressed genes in our dataset were dependent on the kinase GCN2 (n = 2028), with 1,140 up-regulated and 888 down-regulated genes. These findings suggest a promising strategy to enhance CAR-T cell efficacy by combining short amino acid starvation with transient overexpression of a target gene.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7518ff0dd3a5481cf6365679f81c8157f814beb" target='_blank'>
              Transcriptomic analysis of human primary T cells after short-term leucine-deprivation and evaluation of kinase GCN2’s role in regulating differential gene expression
              </a>
            </td>
          <td>
            Aurore Dougé, Gwendal Cueff, Céline Keime, V. Carraro, C. Jousse, Paul Rouzaire, A. Bruhat
          </td>
          <td>2025-02-18</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Multiple myeloma (MM), the second most common hematologic malignancy, is characterized by the clonal expansion of myeloma cells and accumulation of genetic lesions. MM progression is accompanied by increased aggressiveness and drug resistance. Even the goal of “cure” remains hard to reach for most patients, advances in diagnosis and treatment have allowed some to achieve durable remissions and transition to plateau phase. Single-cell sequencing, with its powerful ability to analyze cellular heterogeneity and molecular patterns at ground-breaking resolution, is informative for deciphering tumors and their microenvironment. In this review, we summarize the new insights of studies facilitated by emerging single-cell sequencing into clonal evolution, myeloma-supported microenvironment transformation, epigenetic changes, and novel prognostic and therapeutic strategies for MM, revealing the key mechanisms underlying MM progression and the direction of future efforts. With the continuous expansion of the research scope and optimization of related technologies, single-cell sequencing is expected to revolutionize our understanding of the biology and evolutionary dynamics of MM and contribute to the radical and precise improvement of treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b99a1f2cb92c3a6d34aa29596c928af0010bece5" target='_blank'>
              Single-cell sequencing reveals the mechanisms of multiple myeloma progression: clarity or confusion?
              </a>
            </td>
          <td>
            Yunhui Xiang, G. Sun, Lvbo Tian, Pinpin Xiang, Chunbao Xie
          </td>
          <td>2025-02-07</td>
          <td>Annals of Hematology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The classical paradigm of drug screening often faces significant limitations due to the challenges associated with identifying molecular or cellular read-outs that are relevant to specific diseases. This issue is particularly pronounced for the thousands of diseases of genetic origin, given the abundance of databases listing disease-associated changes in gene expression and various transcripts, indicating that potential read-outs may be concealed within these resources. To remedy this, an alternative approach was tested: compounds were evaluated for their effects on gene expression and alternative splicing in a healthy cell model, and the resulting data were matched to molecular signatures of diseases. A subset of 50 FDA-approved drugs was tested on mesenchymal stem cells derived from a human pluripotent stem cell line. Over half of the compounds altered gene expression, many affecting pathways linked to monogenic diseases. One hit, increased SQSTM1 expression induced by prazosin, was further validated in ALS models caused by SQSTM1 haploinsufficiency, including patient-derived fibroblasts, SQSTM1-depleted hiPSC-derived motor neurons, and a zebrafish model. Extending this paradigm could involve testing diverse cell types and larger drug libraries. One Sentence Summary A novel drug screening approach for monogenic diseases integrating human pluripotent stem cell derivatives and RNA sequencing to profile gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2756ee22739ee10642a0c818c71c07402f6f6f2e" target='_blank'>
              Integrative Analysis of Drug-Gene Expression Signatures in Human Pluripotent Stem Cells Identifies Novel Drug Candidates for ALS and Monogenic Diseases
              </a>
            </td>
          <td>
            Florine Roussange, Jacqueline Gide, Johana Tournois, Michel Cailleret, A. Boland, Christophe Battail, J. Deleuze, Hélène Polvèche, Didier Auboeuf, Knut Brockmann, E. Kabashi, A. Marian, Lina El Kassar, Sophie Blondel, François Salachas, G. Bruneteau, Marc Peschanski, C. Martinat, Sandrine Baghdoyan
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/996e0da402cc341f0ff28812c9362a9660b3a867" target='_blank'>
              Abstract A026: Deciphering natural and synthetic genetic programs governing human T cell function with combinatorial CRISPR activation and interference screens
              </a>
            </td>
          <td>
            Zachary Steinhart, Jeffrey Perera, Dmytro Dorovskyi, Luke Workley, Carinna Tran, Rosmely Hernandez, Zhongmei Li, Max Foisey, Daniel Goodman, Kole T. Roybal, B. Shy, Alexander Marson
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Molecular profiles of mesenchymal stem cells (MSCs) are needed to standardize the composition and effectiveness of MSC therapeutics. This study employs RNA sequencing to identify genes to be used in concert with CD264 as a molecular profile of aging MSCs at a clinically relevant culture passage. CD264− and CD264+ populations were isolated by fluorescence-activated cell sorting from passage 4 MSC cultures. CD264+ MSCs exhibited an aging phenotype relative to their CD264− counterpart. Donor-matched CD264−/+ mRNA samples from 5 donors were subjected to pair-ended, next-generation sequencing. An independent set of 5 donor MSCs was used to validate differential expression of select genes with quantitative reverse transcription PCR. Pairwise differential expression analysis identified 2,322 downregulated genes and 2,695 upregulated genes in CD264+ MSCs relative to donor-matched CD264− MSCs with a Benjamini–Hochberg adjusted p-value (BH padj) < 0.1. Nearly 25% of these genes were unique to CD264−/+ MSCs and not differentially expressed at a significance level of BH padj < 0.1 in previous RNA sequencing studies of early- vs. late-passage MSCs. Least Absolute Shrinkage and Selection Operator regression identified microtubule-associated protein 1A (MAP1A) and pituitary tumor-transforming gene 1 (PTTG1) as predictive genes of CD264+ MSCs. Combined MAP1A and PTTG1 expression correctly classified CD264 status of MSC samples with an accuracy of 100%. Differential expression and predictive ability of MAP1A and PTTG1 compared favorably with that of existing senescence markers expressed in early passage CD264−/+ MSCs. This study provides the first linkage of MAP1A to CD264, aging and senescence. Our findings have application as quality metrics to standardize the composition of MSC therapies and as molecular targets to slow/reverse cellular aging. Supplementary Information The online version contains supplementary material available at 10.1007/s10616-025-00724-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df97a5ece885215c844356881d36483043d3e173" target='_blank'>
              RNA sequencing identifies MAP1A and PTTG1 as predictive genes of aging CD264+ human mesenchymal stem cells at an early passage
              </a>
            </td>
          <td>
            Margaret K. Giler, H. A. Tucker, Amanda K. Foote, Avery G. Francis, Sean D. Madsen, Yao-Zhong Liu, Kim C. O’Connor
          </td>
          <td>2025-02-19</td>
          <td>Cytotechnology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="InfoScan is a novel bioinformatics tool designed for the comprehensive analysis of full-length single-cell RNA sequencing (scRNA-seq) data. It enables the identification of unannotated transcripts and rare cell populations, providing a powerful platform for transcriptome characterization. In this study, InfoScan was applied to glioblastoma multiforme (GBM), identifying a rare “neoplastic-stemness” subpopulation exhibiting cancer stem cell-like features. Functional analyses suggested that tumor-associated macrophages (TAMs) secrete SPP1, which binds to CD44 on neoplastic-stemness cells, activating the PI3K/AKT pathway and driving lncRNA transcription to promote metastasis. Integration of TCGA and CGGA datasets further supported these findings, highlighting key mutations associated with the neoplastic-stemness subpopulation. Drug sensitivity assays indicated that neoplastic-stemness cells might be sensitive to omipalisib, a PI3K inhibitor, pointing to a potential therapeutic target. InfoScan offers a robust framework for exploring complex transcriptomic landscapes and characterizing rare cell populations, providing valuable insights into GBM biology and advancing precision cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d19fc728a932329a1c6b81702c24f21513cb2f5" target='_blank'>
              InfoScan: A New Transcript Identification Tool Based on scRNA-Seq and Its Application in Glioblastoma
              </a>
            </td>
          <td>
            Shiqiang Mei, Jinjin Huang, Zhen Zhang, Haotian Lei, Qiaojuan Huang, Lianghu Qu, Lingling Zheng
          </td>
          <td>2025-02-28</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancer stem cells (CSCs), are a critical subpopulation within tumours, and are defined by their capacity for self-renewal, differentiation, and tumour initiation. These unique traits contribute to tumour progression, metastasis, and resistance to conventional treatments like chemotherapy and radiotherapy, often resulting in cancer recurrence and poor patient outcomes. As such, CSCs have become focal points in developing advanced cancer therapies. This review highlights progress in CSC-targeted treatments, including chimeric antigen receptor T-cell (CAR-T) therapy, immunotherapy, molecular targeting, and nanoparticle-based drug delivery systems. Plant-derived compounds and gene-editing technologies, such as clustered regularly interspaced short palindromic repeats (CRISPR), are explored for their potential to enhance precision and minimize side effects. Metabolic pathways integral to CSC survival, such as mitochondrial dynamics, mitophagy (regulated by dynamin-related protein 1 [DRP1] and the PINK1/Parkin pathway), one-carbon metabolism, amino acid metabolism (involving enzymes like glutaminase (GLS) and glutamate dehydrogenase (GDH]), lipid metabolism, and hypoxia-induced metabolic reprogramming mediated by hypoxia-inducible factors (HIF-1α and HIF-2α), are examined as therapeutic targets. The adaptability of CSCs through autophagy, metabolic flexibility, and epigenetic regulation by metabolites like α-ketoglutarate, succinate, and fumarate is discussed. Additionally, extracellular vesicles and nicotinamide adenine dinucleotide (NAD⁺) metabolism are identified as pivotal in redox balance, DNA repair, and epigenetic modifications. Addressing challenges such as tumour heterogeneity, immune evasion, and treatment durability requires interdisciplinary collaboration. Advancing CSC-targeted therapies is essential for overcoming drug resistance and preventing cancer relapse, paving the way for transformative cancer treatments. This review underscores the importance of leveraging innovative technologies and fostering collaboration to revolutionize cancer treatment. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdfd29a8c49b166f1e1f2b3241c057ea76b860a3" target='_blank'>
              Unveiling the future of cancer stem cell therapy: a narrative exploration of emerging innovations
              </a>
            </td>
          <td>
            Joseph Nhyira Obisi, Abike Ndidiamaka Josephine Abimbola, O. Babaleye, Peter Kwame Atidoglo, Saviour Godswealth Usin, Eudora Nwanaforo, Faith Sutu Patrick-Inezi, I. V. Fasogbon, Joseph Chimezie, Christianah Adebimpe Dare, Oluwadoyinsayemi Oluwadamilare Kuti, Daniel Ejim Uti, Humphrey Chukwudi Omeoga
          </td>
          <td>2025-03-22</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21b08a5b6171c80002da0a1517c54be1999c70c6" target='_blank'>
              MED12 dysregulation: insights into cancer and therapeutic resistance.
              </a>
            </td>
          <td>
            R. Bhole, Jagruti Shinkar, Sonali Labhade, Pawan N Karwa, Harshad S. Kapare
          </td>
          <td>2025-03-19</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Lung cancer exhibits substantial inter- and intra-tumor heterogeneity, with features that present significant challenges in advancing biomarker discovery and the development of targeted therapeutics. To fill this gap, we employed single-cell RNA sequencing (scRNA-seq) and advanced bioinformatics tools to evaluate the transcriptomic heterogeneity of immortalized, non-transformed (BEAS2B) and transformed (H460) lung epithelial cell lines and their responses to carcinogen challenge. Gene expression profiles resolved four primary clusters further discretized into unique subclusters based on genetic signatures and phenotypic profiles. Profiles of long non-coding RNAs (lncRNAs) identified microRNA host genes, antisense RNA genes, divergent transcript, and long intergenic non-coding RNAs as contributors to cellular heterogeneity. These findings indicate that distinct patterns of gene expression, remarkably in lncRNAs, define cellular heterogeneity in non-transformed versus transformed cells. These features can be exploited for the development of therapies directed at specific cell subpopulations in precancerous lesions and within lung tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/233c01bff81dc65b1be1e7c0ebbee44d4a90ef36" target='_blank'>
              Single-Cell RNA Sequencing Reveals Extensive Heterogeneity and Unique Gene Trajectories in Non-Transformed and Transformed Human Lung Epithelial Cells: Insights into the Role of LncRNAs in Tumor Heterogeneity
              </a>
            </td>
          <td>
            Sokviseth Moeng, Andres D. Chamorro-Parejo, Minsun S. Jeon, James J. Cai, Kenneth S Ramos
          </td>
          <td>2025-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>1</td>
          <td>6</td>
        </tr>

        <tr id="Aneuploidy is one of the most common mechanisms of adaptation to environmental selection in vegetatively growing cells. However, it is still unclear what advantages aneuploidy provides over other types of genomic alterations. In this study, we aimed to determine if beneficial aneuploidy phenotypes are driven primarily by combinatorial expression changes of multiple genes using budding yeast as a model system. To determine the impact of complex aneuploidy on cellular fitness and resistance to a variety of drug treatments, we generated yeast collections with nearly every possible combination of two chromosome gains or losses. In addition, we genetically dissected aneuploid chromosomes using partial deletions to identify chromosomal regions that contribute to aneuploidy-driven drug resistance. Strong resistance phenotypes consistently came from combinations of chromosomes or chromosomal regions. We identified up to five regions on a single chromosome that contribute to resistance to a single drug treatment. Finally, we found that the strongest aneuploidy phenotypes came from synergistic effects between copy number changes of different chromosomes or chromosomal regions. These results provide insights into how combining subtle gene expression changes across the genome impacts aneuploidy phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/288386141655e3d2d8bd580753d7c57e3ca32011" target='_blank'>
              Combinatorial effects of multiple genes contribute to beneficial aneuploidy phenotypes
              </a>
            </td>
          <td>
            M. S. Koller, Claudia Himmelbauer, Sarah Fink, Madhwesh C. Ravichandran, Christopher S. Campbell
          </td>
          <td>2025-03-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/984621d182e32177c7e7be128d21ab0e1df4a135" target='_blank'>
              Lineage plasticity of the integrated stress response is a hallmark of cancer evolution
              </a>
            </td>
          <td>
            Shiqi Diao, Jia Yi Zou, Shuo Wang, N. Ghaddar, Jason E. Chan, Hyungdong Kim, Nicolas Poulain, C. Koumenis, M. Hatzoglou, Peter Walter, Nahum Sonenberg, John Le Quesne, Tuomas Tammela, A. Koromilas
          </td>
          <td>2025-02-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Non-coding RNAs (ncRNAs) play crucial roles in colorectal cancer (CRC) development and progression. Recent developments in single-cell transcriptome profiling methods have revealed surprising levels of expression variability among seemingly homogeneous cells, suggesting the existence of many more cell types than previously estimated. This review synthesizes recent advances in ncRNA research in CRC, emphasizing single-cell bioinformatics approaches for their analysis. We explore computational methods and tools used for ncRNA identification, characterization, and functional prediction in CRC, with a focus on single-cell RNA sequencing (scRNA-seq) data. The review highlights key bioinformatics strategies, including sequence-based and structure-based approaches, machine learning applications, and multi-omics data integration. We discuss how these computational techniques can be applied to analyze differential expression, perform functional enrichment, and construct regulatory networks involving ncRNAs in CRC. Additionally, we examine the role of bioinformatics in leveraging ncRNAs as diagnostic and prognostic biomarkers for CRC. We also discuss recent scRNA-seq studies revealing ncRNA heterogeneity in CRC. This review aims to provide a comprehensive overview of the current state of single-cell bioinformatics in ncRNA CRC research and outline future directions in this rapidly evolving field, emphasizing the integration of computational approaches with experimental validation to advance our understanding of ncRNA biology in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df0d488fffc7d97d7489f557a3f2dfce15740d80" target='_blank'>
              Single-Cell Transcriptomic Approaches for Decoding Non-Coding RNA Mechanisms in Colorectal Cancer
              </a>
            </td>
          <td>
            M. Gondal, H. Farooqi
          </td>
          <td>2025-03-10</td>
          <td>Non-Coding RNA</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Aneuploidy is prevalent in cancer and associates with fitness advantage and poor patient prognosis. Yet, experimentally induced aneuploidy initially leads to adverse effects and impaired proliferation, suggesting that cancer cells must adapt to aneuploidy. We performed in vitro evolution of cells with extra chromosomes and obtained cell lines with improved proliferation and gene expression changes congruent with changes in aneuploid cancers. Integrated analysis of cancer multi-omics data and model cells revealed increased expression of DNA replicative and repair factors, reduced genomic instability, and reduced lysosomal degradation. We identified E2F4 and FOXM1 as transcription factors required for adaptation to aneuploidy in vitro and in cancers and validated this finding. The adaptation to aneuploidy also coincided with specific copy number aberrations that correlate with poor patient prognosis. Chromosomal engineering mimicking these aberrations improved aneuploid cell proliferation, while loss of previously present extra chromosome impaired it. The identified common adaptation strategies suggest replication stress, genomic instability, and lysosomal stress as common liabilities of aneuploid cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a0977541bd5b827628ce81b0938c0df52c0a295" target='_blank'>
              Proteogenomic analysis reveals adaptive strategies for alleviating the consequences of aneuploidy in cancer
              </a>
            </td>
          <td>
            Jan-Eric Bökenkamp, Kristina Keuper, Stefan Redel, Karen Barthel, Leah Johnson, A. Wieland, M. Räschle, Zuzana Storchová
          </td>
          <td>2024-03-07</td>
          <td>The EMBO Journal</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52ce7ec0649fbc5d3e5e12bff74054318a2f6bce" target='_blank'>
              MEDUSA for Identifying Death Regulatory Genes in Chemo-genetic Profiling Data.
              </a>
            </td>
          <td>
            Megan E. Honeywell, Michael J. Lee
          </td>
          <td>2025-02-07</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Chromosomal instability (CIN) is involved in about 70% of colorectal cancers (CRCs) and is associated with poor prognosis and drug resistance. From a clinical perspective, a better knowledge of these tumour’s biology will help to guide therapeutic strategies more effectively. Methods We used high-density chromosomal microarray analysis to evaluate CIN level of patient-derived organoids (PDOs) and their original mCRC tissues. We integrated the RNA-seq and mass spectrometry-based proteomics data from PDOs in a functional interaction network to identify the significantly dysregulated processes in CIN. This was followed by a proteome-wGII Pearson correlation analysis and an in silico validation of main findings using functional genomic databases and patient-tissues datasets to prioritize the high-confidence CIN features. Results By applying the weighted Genome Instability Index (wGII) to identify CIN, we classified PDOs and demonstrated a good correlation with tissues. Multi-omics analysis showed that our organoids recapitulated genomic, transcriptomic and proteomic CIN features of independent tissues cohorts. Thanks to proteotranscriptomics, we uncovered significant associations between mitochondrial metabolism and epithelial-mesenchymal transition in CIN CRC PDOs. Correlating PDOs wGII with protein abundance, we identified a subset of proteins significantly correlated with CIN. Co-localisation analysis in PDOs strengthened the putative role of IPO7 and YAP, and, through in silico analysis, we found that some of the targets give significant dependencies in cell lines with CIN compatible status. Conclusions We first demonstrated that PDO models are a faithful reflection of CIN tissues at the genetic and phenotypic level. Our new findings prioritize a subset of genes and molecular processes putatively required to cope with the burden on cellular fitness imposed by CIN and associated with disease aggressiveness. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03308-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39fada8ad025d98e9dfc80fcb5a196964bddeb3e" target='_blank'>
              Decoding chromosomal instability insights in CRC by integrating omics and patient-derived organoids
              </a>
            </td>
          <td>
            Federica Papaccio, M. Cabeza-Segura, B. Garcia-Micò, F. Gimeno-Valiente, S. Zúñiga-Trejos, V. Gambardella, M. F. Gutierrez-Bravo, C. Martínez-Ciarpaglini, P. Rentero-Garrido, Tania Fleitas, Susana Roselló, J. Carbonell-Asins, M. Huerta, D. Moro-Valdezate, D. Roda, N. Tarazona, Manuel M. Sánchez del Pino, Andrés Cervantes, Josefa Castillo
          </td>
          <td>2025-02-28</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da95ac3ec1b6b3cbe7baf6551a5457106bc739f1" target='_blank'>
              DNA damage response in cancer: New treatment strategies
              </a>
            </td>
          <td>
            Hang-Hao Zhang
          </td>
          <td>2025-03-31</td>
          <td>New Cell</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/584bbd4f29ec6dc8d7d0e916a78b0dd2a1379b9e" target='_blank'>
              Long and Accurate: How HiFi Sequencing is Transforming Genomics.
              </a>
            </td>
          <td>
            Bo Wang, Peng Jia, Shenghan Gao, Huanhuan Zhao, Gaoyang Zheng, Linfeng Xu, Kai Ye
          </td>
          <td>2025-02-07</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="A longstanding barrier in genome engineering with CRISPR-Cas9 has been the inability to measure Cas9 edit outcomes and their functional effects at single-cell resolution. Here we present Superb-seq, a new technology that leverages T7 in situ transcription and single-cell RNA sequencing to jointly measure on- and off-target Cas9 edits and their effects on gene expression. We performed Superb-seq on 10,000 K562 cells, targeting four chromatin remodeler genes with seven guide RNAs. Superb-seq identified 11,891 edit events in 6,230 edited cells at all seven on-target sites and at an additional 36 off-target sites. Although the seven guides were selected for high specificity, six of them caused off-target edits, ranging in frequency from 0.03% to 18.6% of cells. A notable off-target edit within the first intron of USP9X disrupted the expression of this gene and over 150 downstream genes. In summary, Cas9 off-targeting is pervasive due to a combination of rare and common edit events, occurs primarily within introns of off-target genes, and can exert widespread effects on gene expression. Superb-seq uses off-the-shelf kits, standard equipment, and requires no virus, which will enable genome-wide CRISPR screens in diverse cell types as well as functional characterization of clinically-relevant guides.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e413aab82e3b99df4d044d401e4ffc522bbaad3" target='_blank'>
              Joint single-cell profiling of CRISPR-Cas9 edits and transcriptomes reveals widespread off-target events and their effects on gene expression
              </a>
            </td>
          <td>
            Michael H. Lorenzini, Brad Balderson, Karthyayani Sajeev, Aaron Ho, G. McVicker
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/143b072951804a4c0de0ec5707365e255796efaa" target='_blank'>
              Using single-cell perturbation screens to decode the regulatory architecture of splicing factor programs
              </a>
            </td>
          <td>
            Miquel Anglada-Girotto, Samuel Miravet-Verde, Luis Serrano
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/330a5593d326f97de03e4da3203d107ef7927e48" target='_blank'>
              Extending differential gene expression testing to handle genome aneuploidy in cancer
              </a>
            </td>
          <td>
            Katsiaryna Davydzenka, G. Caravagna, Guido Sanguinetti
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancer genomes have a complicated landscape of mutations, including large-scale rearrangements known as structural variants (SVs). These SVs can disrupt genes or regulatory elements, playing a critical role in cancer development and progression. Despite their importance, accurate identification of somatic structural variants (SVs) remains a significant bottleneck in cancer genomics. Long-read sequencing technologies hold great promise in SV discovery, and there is an increasing number of efforts to develop new tools to detect them. In this study, we employ eight widely used SV callers on paired tumor and matched normal samples from both the NCI-H2009 lung cancer cell line and the COLO829 melanoma cell line, the latter of which has a well-established somatic SV truth set. Following separate variation detection in both tumor and normal DNA, the VCF merging procedure and a subtraction method were used to identify candidate somatic SVs. Additionally, we explored different combinations of the tools to enhance the accuracy of true somatic SV detection. Our analysis adopts a comprehensive approach, evaluating the performance of each SV caller across a spectrum of variant types and numbers in finding cancer-related somatic SVs. This study, by comparing eight different tools and their combinations, not only reveals the benefits and limitations of various techniques but also establishes a framework for developing more robust SV calling pipelines. Our findings highlight the strengths and weaknesses of current SV calling tools and suggest that combining multiple tools and testing different combinations can significantly enhance the validation of somatic alterations. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-92750-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6b50dd05db294032c45ba6780a2c802ad31817a" target='_blank'>
              Benchmarking long-read structural variant calling tools and combinations for detecting somatic variants in cancer genomes
              </a>
            </td>
          <td>
            Safa Kerem Aydin, Kübra Yılmaz, Ahmet Acar
          </td>
          <td>2025-03-13</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background The genomic architecture of eukaryotes exhibits dynamic spatial and temporal changes, enabling cellular processes critical for maintaining viability and functional diversity. Recent advances in sequencing technologies have facilitated the dissection of genomic architecture and functional activity at single-cell resolution, moving beyond the averaged signals typically derived from bulk cell analyses. Main body The advent of single-cell genomics and epigenomics has yielded transformative insights into cellular heterogeneity, behavior, and biological complexity with unparalleled genomic resolution and reproducibility. This review summarizes recent progress in the characterization of genomic architecture at the single-cell level, emphasizing the impact of structural variation and chromatin organization on gene regulatory networks and cellular identity. Conclusion Future directions in single-cell genomics and high-resolution epigenomic methodologies are explored, focusing on emerging challenges and potential impacts on the understanding of cellular states, regulatory dynamics, and the intricate mechanisms driving cellular function and diversity. Future perspectives on the challenges and potential implications of single-cell genomics, along with high-resolution genomic and epigenomic technologies for understanding cellular states and regulatory dynamics, are also discussed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e21cbb2f5aa4a29eac0f638906ce12441220483a" target='_blank'>
              Deciphering single-cell genomic architecture: insights into cellular heterogeneity and regulatory dynamics
              </a>
            </td>
          <td>
            Byunghee Kang, Hyeonji Lee, Tae-Young Roh
          </td>
          <td>2025-02-11</td>
          <td>Genomics & Informatics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Profiling circulating cell-free DNA (cfDNA) has become a fundamental practice in cancer medicine, but the effectiveness of cfDNA at elucidating tumor-derived molecular features has not been systematically compared to standard single-lesion tumor biopsies in prospective cohorts of patients. The use of plasma instead of tissue to guide therapy is particularly attractive for patients with small cell lung cancer (SCLC), due to the aggressive clinical course of this cancer, which makes obtaining tumor biopsies exceedingly challenging. Methods In this study, we analyzed a prospective cohort of 49 plasma samples obtained before, during, and after treatment from 20 patients with recurrent SCLC. We conducted cfDNA low-pass whole genome sequencing (0.1X coverage), comparing it with time-point matched tumor characterized using whole-exome (130X) and transcriptome sequencing. Results A direct comparison of cfDNA and tumor biopsy revealed that cfDNA not only mirrors the mutation and copy number landscape of the corresponding tumor but also identifies clinically relevant resistance mechanisms and cancer driver alterations not detected in matched tumor biopsies. Longitudinal cfDNA analysis reliably tracks tumor response, progression, and clonal evolution. Sequencing coverage of plasma DNA fragments around transcription start sites showed distinct treatment-related changes and captured the expression of key transcription factors such as NEUROD1 and REST in the corresponding SCLC tumors. This allowed for the prediction of SCLC neuroendocrine phenotypes and treatment responses. Conclusions cfDNA captures a comprehensive view of tumor heterogeneity and evolution. These findings have significant implications for the non-invasive stratification of SCLC, a disease currently treated as a single entity. Supplementary Information The online version contains supplementary material available at 10.1186/s13073-025-01438-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a75bdacf15f78eabb80ad03155a6f7ee66350fd9" target='_blank'>
              Genomic alterations and transcriptional phenotypes in circulating free DNA and matched metastatic tumor
              </a>
            </td>
          <td>
            N. Takahashi, L. Pongor, Shivam P. Agrawal, Mariya Shtumpf, Ankita Gurjar, V. Rajapakse, Ahmad Shafiei, Christopher W. Schultz, Sehyun Kim, Diana Roame, Paula Carter, R. Vilimas, Samantha Nichols, Parth Desai, W. Figg, Mohammad Bagheri, V. B. Teif, Anish Thomas
          </td>
          <td>2025-02-25</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Abstract Background Basic research identified oncogenic driver mutations in lung cancer (LC). However, <10% of patients carry driver mutations. Thus, most patients are not recommended for first‐line kinase inhibitor (KI)–based therapies. Through enabling technologies and bioinformatics, we gained deep insight into patient‐specific signalling networks which permitted novel KI‐based treatment options in LC. Methods We performed molecular pathology, transcriptomics and miRNA profiling across 95 well‐characterized LC patients. We confirmed results based on cross‐linked immunoprecipitation‐sequencing data, and used N = 524 adeno‐ and 497 squamous cell carcinomas as validation sets. We employed the PamGene platform to identify aberrant kinases, validated the results by evaluating independent siRNA and CRISPR‐mediated mRNA knockdown studies in human LC cell lines. Results Transcriptomics revealed 439, 1240, 383 and 320 significantly upregulated genes, respectively, for adeno‐, squamous, neuroendocrine and metastatic cases, and there are 1092, 1477, 609 and 1267 downregulated DEGs. Based on gene enrichment analysis and experimentally validated miRNA–gene interactions, we constructed regulatory networks specific for adeno‐, squamous, neuroendocrine and metastatic LC. Molecular profiling discovered 137 significantly upregulated kinases (range 2–26‐fold) of which 65 and 72, respectively, are tyrosine and serine‐threonine kinases while 6 kinases carry driver mutations. Meanwhile, there are 21 kinases commonly upregulated irrespective of the histological type of LC. Bioinformatics decoded networks in which kinases function as master regulators. Typically, the networks consisted of 14, 9, 16 and 19 highly regulated kinases in adeno‐, squamous, neuroendocrine and metastatic LC. Inhibition of kinases which function as master regulators disrupted the signalling networks, and their gene knock‐down studies confirmed inhibition of cell proliferation in a panel of human LC cell lines. Additionally, the proposed molecular profiling enables KI‐based therapies in patients with acquired drug resistance. Conclusions Our study broadens the perspective of KI‐based therapies in LC, and we propose a framework to overcome acquired drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e411b62bb637c0a1f20c567137cbfb1f5a70ea0b" target='_blank'>
              Comprehensive transcriptome, miRNA and kinome profiling identifies new treatment options for personalized lung cancer therapy
              </a>
            </td>
          <td>
            Shen Zhong, Yvonne Börgeling, Patrick Zardo, Danny Jonigk, J. Borlak
          </td>
          <td>2025-02-24</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdd747492529d370113dee3945f74bf5ec74e07f" target='_blank'>
              High levels of DNA replication initiation factors indicate ATRi sensitivity via excessive origin firing
              </a>
            </td>
          <td>
            A. Lumeau, PL Pfuderer, S. De Angelis, JA Scarth, MA Guscott, N. Shaikh, FB Copley, H. Gerdes, Emilie L Alard, PR Cutillas, F. Mardakheh, MA Boemo, JV Forment, SE McClelland
          </td>
          <td>2025-02-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Uveal melanoma is a lethal intraocular tumour, in which the presence of various genetic alterations correlates with the risk of metastatic dissemination and survival. Here, we tested the detectability of all key mutations and chromosomal changes from RNA sequencing data in 80 primary uveal melanomas studied by The Cancer Genome Atlas (TCGA) initiative, and in five prospective cases. Whereas unsupervised gene expression profiling strongly indicated the presence of chromosome 3 alterations, it was not reliable in identifying other alterations. Though, the presence of both chromosome 3 and 8q copy number alterations could be successfully inferred from expressed allelic imbalances of heterozygous common single nucleotide polymorphisms. Most mutations were adequately recognised in the RNA by their nucleotide changes (all genes), alternative splicing around the mutation (BAP1) and transcriptome-wide aberrant splicing (SF3B1). Notably, in the TCGA cohort we detected previously unreported mutations in BAP1 (n = 3) and EIF1AX (n = 5), that were missed by the original DNA sequencing. In our prospective cohort, all genetic alterations were successfully identified by combining the described approaches. In conclusion, a transcriptional analysis presents insights into the expressed tumour genotype and its phenotypic consequences and may augment or even substitute DNA-based approaches, with potential applicability in research and clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52de86a247ab47490fc1cc4dbe933d1594ca77e9" target='_blank'>
              Identification of diagnostic and prognostic genetic alterations in uveal melanoma using RNA sequencing
              </a>
            </td>
          <td>
            R. Nell, M. Versluis, D. Cats, Hailian Mei, R. Verdijk, W. Kroes, G.P.M. Luyten, Martine J Jager, P. A. van der Velden
          </td>
          <td>2025-03-10</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Objective RNA-seq delivers valuable insights both to transcriptional patterns and mutational landscapes for transcribed genes. However, as tumour cell lines frequently lack their matched-normal counterpart, variant calling without the paired normal sample is still challenging. In order to exclude variants of common genetic variation without a matched-normal control, filtering strategies need to be developed to identify tumour relevant variants in cell lines. Results Here, variants of 29 breast cancer cell lines were called on RNA-seq data via HaplotypeCaller. Low read depth sites, RNA-edit sites, and low complexity regions in coding regions were excluded. Common variants were filtered using 1000 genomes, gnomAD, and dbSNP data. Starting from hundred thousands of single nucleotide variants and small insertions and deletions, about thousand variants remained after filtering for each sample. Extracted variants were validated against the Catalogue of Somatic Mutations in Cancer (COSMIC) for 10 cell lines included in both data sets. Approximately half of the COSMIC variants were successfully called. Importantly, missing variants could mainly be attributed to sites with low read depth. Moreover, filtered variants also included all 10 cancer gene census COSMIC variants, a condensed hallmark variant set. Supplementary Information The online version contains supplementary material available at 10.1186/s13104-025-07140-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5417aa6262174df782473083114e78a8cbd8f677" target='_blank'>
              Refined variant calling pipeline on RNA-seq data of breast cancer cell lines without matched-normal samples
              </a>
            </td>
          <td>
            Sonja Eberth, Julia Koblitz, Laura Steenpass, C. Pommerenke
          </td>
          <td>2025-02-15</td>
          <td>BMC Research Notes</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="ABSTRACT The non‐coding genome, constituting 98% of human DNA, remains largely unexplored, yet holds potential for identifying new biomarkers and therapeutic targets in acute lymphoblastic leukemia (ALL). In this study, we conducted a systematic analysis of recurrent somatic non‐coding single nucleotide variants (SNVs) in pediatric B‐cell precursor (BCP) ALL. We leveraged whole genome sequencing (WGS) data from 345 pediatric BCP ALL cases, representing all major genetic subtypes and identified 346 mutational hotspots that harbored somatic SNVs in at least three cases. Through the integration of paired RNA sequencing along with published ChIP‐seq and ATAC‐seq data, we found 128 non‐coding hotspots associated with differentially expressed genes nearby, which were enriched for cis‐regulatory elements, demonstrating the effectiveness of multi‐omics integration in distinguishing pathogenic mutations from passengers. We identified one mutational hotspot that was associated with increased expression of the leukemia‐associated gene NRAS in three primary ALLs. Micro‐C analysis in the leukemia cell line demonstrated interactions between the hotspot region and NRAS regulatory elements. Dual luciferase assays indicated that the mutations disrupted regulatory interactions and CRISPR‐mediated deletion of the region significantly upregulated NRAS, confirming the hypothesized regulatory link. Altogether, we provide new insights into the functional roles of non‐coding mutations in leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6089d599cb2b181049865efd98f322d1bb52759" target='_blank'>
              Discovery of Cis‐Regulatory Mechanisms via Non‐Coding Mutations in Acute Lymphoblastic Leukemia
              </a>
            </td>
          <td>
            Efe Aydın, Eleanor L Woodward, Gladys Telliam Dushime, Rebeqa Gunnarsson, H. Lilljebjörn, L. Moura-Castro, T. Fioretos, B. Johansson, K. Paulsson, Minjun Yang
          </td>
          <td>2025-03-01</td>
          <td>Genes, Chromosomes & Cancer</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (ecDNA) are commonly produced within the nucleus to drive genome dynamics and heterogeneity, enabling cancer cell evolution and adaptation. However, the mechanisms underlying ecDNA biogenesis remain poorly understood. Here using genome-wide CRISPR screening in human cells, we identified the BRCA1-A and the LIG4 complexes mediate ecDNA production. Following DNA fragmentation, the upstream BRCA1-A complex protects DNA ends from excessive resection, promoting end-joining for circularization. Conversely, the MRN complex, which mediates end resection and thus antagonizes the BRCA1-A complex, suppresses ecDNA formation. Downstream, LIG4 conservatively catalyzes ecDNA production in Drosophila and mammals, with patient tumor ecDNA harboring junctions marked by LIG4 activity. Notably, disrupting LIG4 or BRCA1-A in cancer cells impairs ecDNA-mediated adaptation, hindering resistance to both chemotherapy and targeted therapies. Together, our study reveals the roles of the LIG4 and BRCA1-A complexes in ecDNA biogenesis, and uncovers new therapeutic targets to block ecDNA-mediated adaptation for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65472598115e4d60921b909d76b9f2af808705a0" target='_blank'>
              BRCA1-A and LIG4 complexes mediate ecDNA biogenesis and cancer drug resistance
              </a>
            </td>
          <td>
            Oliver W. Chung, Shun Yao, Fu Yang, Ling Wang, Christian G Cerda-Smith, Haley M Hutchinson, Kris C. Wood, Weijia Su, Mustafa Khasraw, Lee Zou, Dale A. Ramsden, Z. Zhang
          </td>
          <td>2025-02-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) has the highest mortality rate of all breast cancer subtypes and currently lacks effective targeted therapies. LARP6 is an RNA-binding protein associated with cancer promotion, but its mechanism of action in TNBC remains unclear. We conducted RNA sequencing (RNA-seq) and improved RNA immunoprecipitation and sequencing (iRIP-seq) to identify the differentially expressed genes (DEGs) and alternative splice sites bound and regulated by LARP6 in MDA-MB-231 cells. Finally, both RT-qPCR and RIP-qPCR were employed for verification. Our study revealed that LARP6 overexpression altered the expression levels of 171 genes and that the number of regulated alternative splicing events (RASEs) exceeded 1000. The regulated alternative splicing genes (RASGs) corresponding to RASEs were enriched in biological processes such as DNA repair, the cell cycle, and the cellular response to DNA damage stimulus. In addition, we found that LARP6 tends to bind the CGACGAG motif. The intersection of peak-related genes with RASGs suggested that LARP6 can bind to 16 genes and regulate their alternative splicing (AS), thus playing an important role in TNBC progression. Our research indicated that LARP6 may promote the proliferation and invasion of TNBC cells by directly regulating the AS of related genes, providing new clues for targeted therapy for TNBC. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-92351-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7eb087ff58a5d637f585f09e70365a21011c3f17" target='_blank'>
              The RNA-binding protein LARP6 regulates the alternative splicing of related genes in MDA-MB-231 cells
              </a>
            </td>
          <td>
            Li Guo, Yaobang Liu, Shuxun Yan, Hong Li, Kai Zhang, Jinping Li
          </td>
          <td>2025-03-06</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Mutations in the TP53 gene and chromosomal instability (CIN) are two of the most common alterations in cancer. CIN, marked by changes in chromosome numbers and structure, drives tumor development, but is poorly tolerated in healthy cells, where developmental and tissue homeostasis mechanisms typically eliminate cells with chromosomal abnormalities. Mechanisms that allow cancer cells to acquire and adapt to CIN remain largely unknown. Tumor suppressor protein p53, often referred to as the “guardian of the genome”, plays a critical role in maintaining genomic stability. In cancer, CIN strongly correlates with TP53 mutations, and recent studies suggest that p53 prevents the propagation of cells with abnormal karyotypes arising from mitotic errors. Furthermore, p53 dysfunction is frequent in cells that underwent whole-genome doubling (WGD), a process that facilitates CIN onset, promotes aneuploidy tolerance, and is associated with poor patient prognosis across multiple cancer types. This review summarizes current insights into p53’s role in protecting cells from chromosome copy number alterations and discusses the implications of its dysfunction for the adaption and propagation of cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ee772438cd2bfb17d973a17539b1ffdae477941" target='_blank'>
              The Role of p53 Mutations in Early and Late Response to Mitotic Aberrations
              </a>
            </td>
          <td>
            Anna Hertel, Zuzana Storchová
          </td>
          <td>2025-02-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The chromosome 5p15.33 region, which encodes telomerase reverse transcriptase (TERT), harbors multiple germline variants identified by genome-wide association studies (GWAS) as risk for some cancers but protective for others. Here, we characterize a variable number tandem repeat within TERT intron 6, VNTR6-1 (38-bp repeat unit), and detect a strong link between VNTR6-1 alleles (Short: 24-27 repeats, Long: 40.5-66.5 repeats) and GWAS signals rs2242652 and rs10069690 within TERT intron 4. Bioinformatics analyses reveal that rs10069690-T allele increases intron 4 retention while VNTR6-1-Long allele expands a polymorphic G-quadruplex (G4, 35-113 copies) within intron 6, with both variants contributing to variable TERT expression through alternative splicing and nonsense-mediated decay. In two cell lines, CRISPR/Cas9 deletion of VNTR6-1 increases the ratio of TERT-full-length (FL) to the alternative TERT-β isoform, promoting apoptosis and reducing cell proliferation. In contrast, treatment with G4-stabilizing ligands shifts splicing from TERT-FL to TERT-β isoform, implicating VNTR6-1 as a splicing switch. We associate the functional variants VNTR6-1, rs10069690, and their haplotypes with multi-cancer risk and age-related telomere shortening. By regulating TERT splicing, these variants may contribute to fine-tuning cellular longevity and replicative potential in the context of stress due to tissue-specific endogenous and exogenous exposures, thereby influencing the cancer risk conferred by this locus.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3d3384fa00e95094541c65edba3285e24efeab9" target='_blank'>
              Genetic regulation of TERT splicing affects cancer risk by altering cellular longevity and replicative potential
              </a>
            </td>
          <td>
            Oscar Florez-Vargas, Michelle Ho, Maxwell H Hogshead, B. Papenberg, Chia-Han Lee, Kaitlin Forsythe, Kristine Jones, Wen Luo, Kedest Teshome, Cornelis Blauwendraat, Kimberly J Billingsley, M. Kolmogorov, Melissa Meredith, B. Paten, Raj Chari, Chi Zhang, John S Schneekloth, M. Machiela, S. Chanock, S. Gadalla, S. Savage, Sam M Mbulaiteye, Ludmila Prokunina-Olsson
          </td>
          <td>2025-02-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a271cd04acf8979b5f4cf725c9c1e047db0d2529" target='_blank'>
              CINner: Modeling and simulation of chromosomal instability in cancer at single-cell resolution.
              </a>
            </td>
          <td>
            K. Dinh, I. Vázquez-García, Andrew Chan, Rhea Malhotra, A. Weiner, Andrew McPherson, Simon Tavaré
          </td>
          <td>2025-04-03</td>
          <td>PLoS computational biology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea52e3e92b1a5c4c3caddeca60f128feaaaf952b" target='_blank'>
              Single-cell and spatial transcriptomic insights into glioma cellular heterogeneity and metabolic adaptations
              </a>
            </td>
          <td>
            Yixin Fu, Yong Yi, Yongxiang Shao, Jingcheng Jiang, Qingshan Deng
          </td>
          <td>2025-04-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52d993b12a12216338adf29eefc996f4f14d40a3" target='_blank'>
              Systematic silencing of oncogenic fusion transcripts with ultra-precise CRISPR-Cas13b
              </a>
            </td>
          <td>
            Wenxin Hu, Honglin Chen, J. Casan, Carolyn Shembrey, Steven He, Lauren M Brown, Timothy P. Hughes, Deborah L. White, Antoine de Weck, I. Voskoboinik, Joseph A Trapani, Paul G Ekert, Teresa Sadras, Mohamed Fareh
          </td>
          <td>2025-02-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Hepatocellular carcinoma is the most prevalent primary liver cancer and arises from hepatocytes, which are very metabolically active and contain abundant mitochondria. Despite extensive characterisation of nuclear driver genes and pathways, the role of crosstalk between the mitochondrial and nuclear genomes in mitochondrial mutagenesis remains poorly understood. To address this, we leveraged a chemical carcinogenesis model of liver cancer in four mouse strains. We analysed hundreds of tumours with paired whole-genome sequencing and total RNA sequencing to comprehensively characterise mitochondrial mutations and expression. Following de novo assembly of three new mitochondrial genomes, we devised an accurate and efficient heteroplasmy detection approach and analysed mitochondrial DNA (mtDNA) content and transcription. Across strains, there was high concordance of the number of heteroplasmies, mutation signatures, and variant allele frequencies. In contrast, the heteroplasmy rate and genic locations differed between strains, suggesting strain-specificity in DNA repair mechanisms in early tumour development. Independent of age or environmental factors, tumours had lower mtDNA content than adjacent normal liver, suggesting mitochondrial loss during tumourigenesis. Additionally, within individual strains, mtDNA content differences were associated with nuclear driver gene choice. Our comprehensive characterisation of mitochondrial genomes and expression patterns demonstrates that both germline and somatic genetic variation influence tumour mtDNA content, mutation patterns, and gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7507e40e7369ffa2dd7aab3b39dffb6074ecd240" target='_blank'>
              Germline and somatic variation influence nuclear-mitochondrial crosstalk in tumourigenesis
              </a>
            </td>
          <td>
            Maëlle Daunesse, Vasavi Sundaram, Elissavet Kentepozidou, John Connelly, Craig J. Anderson, Núria López-Bigas, Colin A. Semple, D. T. Odom, Martin S. Taylor, Maša Roller, Sarah J. Aitken, Paul Flicek
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>94</td>
        </tr>

        <tr id="Cancer is a complex and highly lethal disease marked by unchecked cell proliferation, aggressive behavior, and a strong tendency to metastasize. Despite significant advancements in cancer diagnosis and treatment, challenges such as early detection difficulties, drug resistance, and adverse effects of radiotherapy or chemotherapy continue to threaten patient survival. MicroRNAs (miRNAs) have emerged as critical regulators in cancer biology, with miR-506 being extensively studied and recognized for its tumor-suppressive effects across multiple cancer types. This review examines the regulatory mechanisms of miR-506 in common cancers, focusing on its role in the competing endogenous RNA (ceRNA) network and its effects on cancer cell proliferation, apoptosis, and migration. We also discuss the potential of miR-506 as a therapeutic target and its role in overcoming drug resistance in cancer treatment. Overall, these insights underscore the therapeutic potential of miR-506 and its promise in developing novel cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06c1c0de824cb3b6994467264a7d64c1f9bef206" target='_blank'>
              Therapeutic potential of microRNA-506 in cancer treatment: mechanisms and therapeutic implications
              </a>
            </td>
          <td>
            Shuzhen Mao, Junyan Li, Jiahui Huang, Lili Lv, Qilian Zhang, Qing Cheng, Xiaojing Liu, Zhiwei Bi, Jing Yao
          </td>
          <td>2025-04-03</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9f68c71be9a5f4d5ac85f15c79176d7904112ca" target='_blank'>
              Long-read single-cell RNA sequencing enables the study of cancer subclone-specific genotypes and phenotypes in chronic lymphocytic leukemia.
              </a>
            </td>
          <td>
            Gage S. Black, Xiaomeng Huang, Yi Qiao, Philip Moos, Deepa Sampath, Deborah M. Stephens, J. Woyach, Gabor T. Marth
          </td>
          <td>2025-02-18</td>
          <td>Genome research</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Simple Summary Proto-oncogenes and tumor-suppressor genes play important roles in preventing the development of tumors. The former supports healthy cell growth and division, while the latter regulates cell division. Mutations in proto-oncogenes, converting them into oncogenes, and mutations in tumor-suppressor genes can lead to uncontrolled cell proliferation and cancer development. This review explores key molecular pathways associated with these genes, highlighting their influence on cancer progression and treatment resistance, which are crucial for creating effective, personalized, and targeted treatments for patients. This comprehensive review also discusses the recent advancements and emerging technologies that enhance the analysis, diagnosis, prevention, and treatment of cancer at the genomic level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4acb0abcdb89e4becb0be9ae1ca6990d3d6cdf49" target='_blank'>
              Translational Advances in Oncogene and Tumor-Suppressor Gene Research
              </a>
            </td>
          <td>
            R. Stojchevski, Edward Agus Sutanto, Rinni Sutanto, N. Hadzi-Petrushev, M. Mladenov, Sajal Raj Singh, J. K. Sinha, Shampa Ghosh, Bhuvaneshwar Yarlagadda, Krishna Kumar Singh, Prashant Verma, Sonali Sengupta, Rakesh Bhaskar, Dimiter B Avtanski
          </td>
          <td>2025-03-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Background Acute myeloid leukemia (AML) is the most common type of acute leukemia, accounting for 20% of cases in children and adolescents. Genome-wide studies have identified genes that are commonly mutated in AML, including many epigenetic regulators involved in either DNA methylation (DNMT3A, TET2, IDH1/2) or histone post-translational modifications (ASXL1, EZH2, MLL1). Several cell lines derived from AML patients are widely used in cancer research. Whether important differences in these cell lines exist remains poorly characterised. Results Here, we used RNA sequencing (RNA-Seq) to contrast the transcriptome of four commonly used AML-derived cell lines: THP-1, NOMO-1, MOLM-13 bearing the common initiating t(9;11) translocation, and MV4.11 bearing the t(4;11) translocation. Gene set enrichment analyses and comparison of key transcription and epigenetic regulator genes revealed important differences in the transcriptome, distinguishing these AML models. Among these, we found striking differences in the expression of clusters of genes located on chromosome 19 encoding Zinc Finger (ZNF) transcriptional repressors. Low expression of many ZNF genes within these clusters is associated with poor survival in AML patients. Conclusion The present study offers a valuable resource by providing a detailed comparative characterisation of the transcriptome of cell lines within the same AML subtype used as models for leukemia research. Supplementary Information The online version contains supplementary material available at 10.1186/s12864-025-11415-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a934cae686db7528797f8dea84b54367fe76eeb" target='_blank'>
              Transcriptomic characterisation of acute myeloid leukemia cell lines bearing the same t(9;11) driver mutation reveals different molecular signatures
              </a>
            </td>
          <td>
            Elise Georges, William Ho, Miren Urrutia Iturritza, L. Eory, Kamila Malysz, U. Sobhiafshar, Alan L. Archibald, Daniel J. Macqueen, Barbara Shih, David Garrick, Douglas Vernimmen
          </td>
          <td>2025-03-25</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56c8bae667d5d467ee9b1edf8b1b3bbcf92dc895" target='_blank'>
              Single-cell sequencing for lung cancer research: Progress and prospects
              </a>
            </td>
          <td>
            Zhongsha Li, Wei Su, Bingbing Bai, Yuesong Wu, Baoyin Zhang, Dejuan Xiang, Gilwa Borjigin, Jinhai Deng, Jian Sun, Mingzhu Yin
          </td>
          <td>2025-03-12</td>
          <td>Eurasian Journal of Medicine and Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aac8334d0db490037d00a83f0b8a5cb44474c730" target='_blank'>
              Single-cell analysis reveals cellular heterogeneity, gene expression profiles, and pathway dynamics in acne vulgaris.
              </a>
            </td>
          <td>
            Xiaoyi Yang, Xiaoyan Wang, Jiankang Yang
          </td>
          <td>2025-02-07</td>
          <td>Archives of dermatological research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Aim: As intrinsic resistance - often driven by concurrent genomic alterations in tumor suppressor genes or oncogenes - remains a major challenge in oncology, this work aimed to comprehensively analyze BRAF somatic alterations across cancer types and identify new potential therapeutic strategies to overcome drug resistance.
 Methods: We conducted an extensive analysis of genomics, transcriptomics, and clinical data retrieved from public repositories, including cBioPortal. Our comprehensive analysis examined BRAF alterations [point mutations, structural variants (SVs) and copy number alteration] in more than 217,000 tumor samples across 120 distinct tumor types from primary and metastatic sites in both adult and pediatric cohorts, focusing on mutual exclusivity and co-occurrence of mutations in other oncogenes or tumor suppressors. The work also explores the association of BRAF somatic alterations with survival, clinical and molecular features.
 Results: Analysis of mutation frequencies across cancer types revealed that BRAFV600E represents approximately 90% of all BRAF alterations. While melanoma and thyroid carcinoma show the highest prevalence of BRAF mutations, followed by colorectal and non-small cell lung cancer in terms of absolute number of patients harboring BRAF mutations worldwide, notably high mutation frequencies were identified in rare malignancies, including hairy-cell leukemia, ganglioglioma, and serous borderline ovarian tumors. The comprehensive analysis of genomic profiling data across these tumors uncovered distinct patterns of co-occurring and mutually exclusive alterations in oncogenes and tumor suppressor genes, illuminating resistance mechanisms and suggesting novel therapeutic combinations.
 Conclusion: Comprehensive genomic profiling is critical for optimizing targeted therapy and overcoming drug resistance in BRAF -mutated cancers. The identification of co-occurring alterations provides opportunities for rational combination therapies, emphasizing the importance of detailed mutation profiling in developing effective treatment strategies across diverse cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7fa7f528d2835416489339d23ab2a839922e3b0" target='_blank'>
              Unraveling BRAF alterations: molecular insights to circumvent therapeutic resistance across cancer types
              </a>
            </td>
          <td>
            Caterina Perfetto, Marianna Aprile, Simona Cataldi, Elisa Giovannetti, Valerio Costa
          </td>
          <td>2025-03-24</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Mesothelioma cell heterogeneity encompasses diverse morphological and molecular characteristics observed within tumors, significantly impacting disease progression, treatment outcomes, and the development of targeted therapies. This heterogeneity has long posed challenges for accurate diagnosis and effective treatment, but understanding its complexities offers the potential for novel diagnostic modalities and therapeutic interventions. This study employed single-cell RNA sequencing (scRNA-seq) to investigate mesothelioma cell heterogeneity from various sources, including cell culture (CC), peritoneal lavage (Lav) from the tumor microenvironment, and circulating tumor cells (CTC) in murine models. Gene set enrichment analysis was used to identify distinct gene signatures for each subpopulation. The results revealed unique characteristics for mesothelioma cells depending on their origin. In the CC group, up-regulated genes were primarily involved in tumor cell cycle control, proliferation, and apoptosis. In the CTC group, up-regulated genes were associated with cancer cell stemness. The Lav group showed up-regulated genes facilitating interactions between tumor cells and the microenvironment, such as epithelial–mesenchymal transition and immune responses mediated by IFN-α and IFN-γ. Some pathways were shared among all tumor cells, suggesting the potential for transitioning between functional states under specific conditions. This may be the first study to explore circulating mesothelioma cell heterogeneity using scRNA-seq. The distinct gene signatures identified in each mesothelioma cell subpopulation likely play critical roles in tumor initiation and progression, offering potential novel targets for therapeutic intervention. These findings could help inform the development of more effective, personalized treatments for mesothelioma, ultimately improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38354bc1c5ef98aa13d3c5253963e1d30d5518bf" target='_blank'>
              Mesothelioma cell heterogeneity identified by single cell RNA sequencing
              </a>
            </td>
          <td>
            Licun Wu, Zongjie Wang, Amin Zia, S. Kelley, Marc de Perrot
          </td>
          <td>2025-03-13</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a48c7f88008ee3f6f5e730cd7155fc1a412d588b" target='_blank'>
              Genomic medicine and personalized treatment: a narrative review
              </a>
            </td>
          <td>
            Adil Khan, Anchal Ramesh Barapatre, Nadir Babar, Joy Doshi, Mohamd Ghaly, Kirtan Ghanshyam Patel, Shayan Nawaz, Uswa Hasana, Swara Punit Khatri, Shilpa Pathange, Abhinya Reddy Pesaru, C. Puvvada, Marium Billoo, Usama Jamil
          </td>
          <td>2025-02-13</td>
          <td>Annals of Medicine &amp; Surgery</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/572fd6e1dc6d845d6923c7249ae200e2136415b1" target='_blank'>
              Genome-wide transcriptomic response of whole blood to radiation
              </a>
            </td>
          <td>
            Ahmed Salah, D. Wollschläger, Maurizio Callari, H. Schmidberger, Federico Marini, S. Zahnreich
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The ability to identify gene functions and interactions in specific cellular contexts has been greatly enabled by functional genomics technologies. CRISPR-based genetic screens have proven invaluable in elucidating gene function in mammalian cells. Single-cell functional genomics methods, such as Perturb-seq and Spear-ATAC, have made it possible to achieve high-throughput mapping of the functional effects of gene perturbations by profiling transcriptomes and DNA accessibility, respectively. Combining single-cell chromatin accessibility and transcriptomic data via multiomic approaches has facilitated the discovery of novel cis and gene regulatory interactions. However, pseudobulk readouts from cell populations can often cloud the interpretation of results due to a heterogeneous response from cells receiving the same genetic perturbation, which could be mitigated by using transcriptional profiles of single cells to subset the ATAC-seq data. Existing methods to capture CRISPR guide RNAs to simultaneously assess the impact of genetic perturbations on RNA and ATAC profiles require either cloning of gRNA libraries in specialized vectors or implementing complex protocols with multiple rounds of barcoding. Here, we introduce CAT-ATAC, a technique that adds CRISPR gRNA capture to the existing 10X Genomics Multiome assay, generating paired transcriptome, chromatin accessibility and perturbation identity data from the same individual cells. We demonstrate up to 77% guide capture efficiency for both arrayed and pooled delivery of lentiviral gRNAs in induced pluripotent stem cells (iPSCs) and cancer cell lines. This capability allows us to construct gene regulatory networks (GRNs) in cells under drug and genetic perturbations. By applying CAT-ATAC, we were able to identify a GRN associated with dasatinib resistance, indirectly activated by the HIC2 gene. Using loss of function experiments, we further validated that the gene, ZFPM2, a component of the predicted GRN, also contributes to dasatinib resistance. CAT-ATAC can thus be used to generate high-content multidimensional genotype-phenotype maps to reveal novel gene and cellular interactions and functions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5994854c014d82e61919b765b3c023100d1d1160" target='_blank'>
              Simultaneous single-cell CRISPR, RNA, and ATAC-seq enables multiomic CRISPR screens to identify gene regulatory relationships
              </a>
            </td>
          <td>
            Kaivalya Shevade, Yeqing Angela Yang, Kevin Feng, Karl Mader, V. Sevim, Jacob Parsons, Gunisha Arora, Hasnaa Elfawy, Rachel Mace, S. Federman, Rustam Esanov, Shawn Shafer, Eric D Chow, Laralynne Przybyla
          </td>
          <td>2025-02-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Advances in single-cell multi-omics technologies have deepened our understanding of cancer biology by integrating genomic, transcriptomic, epigenomic, and proteomic data at single-cell resolution. These single-cell multi-omics technologies provide unprecedented insights into tumour heterogeneity, tumour microenvironment, and mechanisms of therapeutic resistance, enabling the development of precision medicine strategies. The emerging field of single-cell multi-omics in genomic medicine has improved patient outcomes. However, most clinical applications still depend on bulk genomic approaches, which fail to directly capture the genomic variations driving cellular heterogeneity. In this review, we explore the common single-cell multi-omics platforms and discuss key analytical steps for data integration. Furthermore, we highlight emerging knowledge in therapeutic resistance and immune evasion, and the potential of new therapeutic innovations informed by single-cell multi-omics. Finally, we discuss the future directions of the application of single-cell multi-omics technologies. By bridging the gap between technological advancements and clinical implementation, this review provides a roadmap for leveraging single-cell multi-omics to improve cancer treatment and patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7137167161ba1e0d324611dc617b5e8bcaedde60" target='_blank'>
              Single-Cell Multi-Omics: Insights into Therapeutic Innovations to Advance Treatment in Cancer
              </a>
            </td>
          <td>
            Angel Guan, Camelia Quek
          </td>
          <td>2025-03-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD) remains a leading cause of cancer-related mortalities, characterized by substantial genetic heterogeneity that challenges a comprehensive understanding of its progression. This study employs next-generation sequencing data analysis to transform our comprehension of LUAD pathogenesis. Integrating epigenetic and transcriptomic data of LUAD patients, this approach assessed the critical regulatory occurrences, identified therapeutic targets, and offered profound insights into cancer molecular foundations. We employed the DNA methylation data to identify differentially methylated CpG sites and explored the transcriptome profiles of their adjacent genes. An intersectional analysis of gene expression profiles uncovered 419 differentially expressed genes (DEGs) influenced by smoke-induced differential DNA methylation, among which hub genes, including mitochondrial ribosomal proteins (MRPs), and ribosomal proteins (RPs) such as MRPS15, MRPS5, MRPL33, RPL24, RPL7L1, MRPL15, TUFM, MRPL22, and RSL1D1, were identified using a network-based approach. These hub genes were overexpressed and enriched to RNA processing, ribosome biogenesis, and mitochondrial translation, which is critical in LUAD progression. Enhancer Linking Methylation/Expression Relationship (ELMER) analysis revealed transcription factor (TF) binding motifs, such as JUN, NKX23, FOSB, RUNX3, and FOSL1, which regulated these hub genes through methylation-dependent enhancer dynamics. Predominant hypomethylation of MRPs and RPs disrupted mitochondrial function, contributed to oxidative phosphorylation (OXPHOS) and metabolic reprogramming, favoring cancer cell survival. The survival analysis validated the clinical relevance of these hub genes, with high-expression cohorts exhibiting poor overall survival (OS) outcomes enlightened their relevance in LUAD pathogenesis and presented the potential for developing novel targeted therapeutic strategies. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-93769-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f19b44fa19b901c527db0c43e3ead70766aaf9b" target='_blank'>
              Unraveling the intricate molecular landscape and potential biomarkers in lung adenocarcinoma through integrative epigenomic and transcriptomic profiling
              </a>
            </td>
          <td>
            Arnab Mukherjee, Manon Boonbangyang, Mukunthan K.S.
          </td>
          <td>2025-03-17</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background & Aims Patients with colorectal cancer have heterogeneous clinical responses to chemotherapy, although clinical guidelines advise little variability in treatment selection based on molecular tumor features. Precision oncology research typically utilizes patient-derived tumor organoids (PDTO) to predict clinical outcomes, but such efforts are often not directed towards identification of molecular factors underlying differential responses to therapy. Methods Bulk RNA-sequencing was performed on treatment-naive PDTOs, and gene expression data was combined to drug sensitivity data to identify transcriptomic features associated with low in vitro sensitivity to chemotherapy. Whole-exome sequencing was performed on primary tumors to infer the somatic mutations of PDTOs and used to identify somatic mutations associated with differential in vitro drug responses. Publicly available gene expression and drug sensitivity data sets were used to validate the results. RNA interference was used for functional validation. Results PDTOs with low chemosensitivity had high JAK-STAT pathway activity resulting from high expression of interferon-stimulated genes. Evidence from single-cell RNA-sequencing confirmed chemotherapy-induced expression of interferon-stimulated genes in epithelial cells of cancers with partial response. EPSTI1 knockdown decreased cancer cell viability and sensitized cells to chemotherapy. Conclusions Sustained interferon signaling in epithelial cancer cells contributes to incomplete pathologic response in colorectal cancer. The findings highlight the potential of JAK-STAT inhibition or TRAIL pathway activation to enhance chemotherapy efficacy. Future studies investigating pharmacologic modulation of these pathways in preclinical CRC models are needed to determine their viability as therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5ab87f4ae646ab5e983158fb38a2489ac5894d6" target='_blank'>
              Sustained epithelial interferon signaling modulates incomplete pathologic response in colorectal cancer
              </a>
            </td>
          <td>
            E. Erkan, Emmi Hämäläinen, Julia Kolikova, Kornelia Kuc, K. Ojala, Meiju Kukkonen, Ismaïl Hermelo, S. Koskensalo, T. Tarvainen, Alli Leppä, I. Keränen, Carola Haapamäki, E. Karjalainen, Monica Carpelan-Holmström, L. Renkonen-Sinisalo, Laura E Koskenvuo, P. Puolakkainen, J. Mecklin, Elina Virolainen, M. Nykter, L. Aaltonen, Anna H. Lepistö, Ari Ristimäki, Toni T Seppälä
          </td>
          <td>2025-03-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="Abstract CRISPR/Cas9-mediated homology-directed repair (HDR) allows precise gene editing, but its efficiency remains low for certain cell types, such as human induced pluripotent stem cells (hiPSCs). In this study, we aimed to introduce the CTNNA1: c.2023C>T (p.Q675*) genetic alteration, which is associated with Hereditary Diffuse Gastric Cancer, into hiPSCs using CRISPR/Cas9. We designed a single-guide RNA targeting the alteration site and a single-stranded oligonucleotide donor DNA template for HDR-based repair. Herein, we report the successful introduction of the CTNNA1: c.2023C>T homozygous alteration in one hiPSC line, which resulted in severe phenotypic changes, including impaired colony formation and cell proliferation. Additionally, we established a straightforward protocol to assess hiPSCs karyotype integrity, ensuring the chromosomal stability required for the gene-editing process. This protocol involves routine G-banding analysis that is required for regular quality controls during handling of hiPSCs. This study demonstrates an efficient approach to precisely edit hiPSCs by CRISPR/Cas9 and highlights the essential role of CTNNA1 expression in maintaining hiPSC viability. Our methodology provides a valuable framework for modeling disease-associated alterations in human-derived cellular models that can be reproduced for other genes and other types of cell lines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77b6de2d94791247728d12901bea9b29daa88c32" target='_blank'>
              A protocol for karyotyping and genetic editing of induced pluripotent stem cells with homology-directed repair mediated CRISPR/Cas9
              </a>
            </td>
          <td>
            Silvana Lobo, Rita Barbosa-Matos, Sofia Dória, A. Pedro, Ana Brito, D. Ferreira, Carla Oliveira
          </td>
          <td>2025-02-28</td>
          <td>Biology Methods & Protocols</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Mapping genetic interactions (GIs) is crucial for understanding genetic network complexity. In this study, we investigate the utility of Cas13d, a CRISPR system targeting RNA, for GI mapping and compare it to Cas9 and Cas12a, two DNA nucleases commonly used for GI mapping. We find that Cas13d induces faster target gene perturbation and generates more uniform cell populations with double perturbations than Cas9 or Cas12a. We then encounter Cas13d gRNA-gRNA interference when concatenating gRNAs targeting different genes into one gRNA array, which we overcome by a dual promoter gRNA expression strategy. Moreover, by concatenating three gRNAs targeting the same gene into one array, we are able to maximize the Cas13d-mediated knockdown effects. Combining these strategies enhances proliferation phenotypes while reducing library size and facilitates reproducible quantification of GIs in oncogenic signaling pathways. Our study highlights the potential of Cas13d for GI mapping, promising advancements in understanding therapeutically relevant drug response pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ef2ae6c7368d6a4d9d50c4f417829a563bfed3b" target='_blank'>
              Evaluation of Cas13d as a tool for genetic interaction mapping
              </a>
            </td>
          <td>
            Ghanem El Kassem, Jasmine Hillmer, Michael Boettcher
          </td>
          <td>2025-02-14</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b66fc40e62ecd1d0cde61c3ba23696dc636af88" target='_blank'>
              Longitudinal Profiling of Circulating Tumor DNA Reveals the Evolutionary Dynamics of Metastatic Prostate Cancer During Serial Therapy.
              </a>
            </td>
          <td>
            Yuehui Zhao, Naveen Ramesh, Ping Xu, Emi Sei, Min Hu, S. Bai, Patricia Troncoso, Ana M Aparicio, Christopher J. Logothetis, Paul G Corn, Nicholas E Navin, A. Zurita
          </td>
          <td>2025-02-24</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Background Lung cancer is one of the most commonly diagnosed cancers. N6-methyladenosine (m6A) modification has a profound impact on RNA translation, splicing, transportation, and stability. Aims This research aimed to identify and verify m6A-modified signatures for Lung adenocarcinoma (LUAD) tumorigenesis. Objective Our previous mRNA-seq and m6A-seq data from 26 pairs of LUAD samples and tumor-adjacent normal tissues are used. Methods Univariate Cox regression analysis and the least absolute shrinkage and selection operator (LASSO) analysis were used to estimate the significance of 37 collected m6A regulators. WGCNA was constructed to identify the genes correlated with LUAD tumorigenesis. Pearson correlation analysis between mRNA-seq and m6A-seq data was used to identify the m6A-correlation genes. Results LASSO-Cox analysis identified 18 m6A significant regulators. The top 3 regulators, including METTL16, FTO, and SRSF10, and their downstream genes which were reported in the literature were analysed to confirm their role in LUAD tumorigenesis. Blue and brown coexpression modules were chosen as key modules for LUAD tumorigenesis. At last, we intersected Lasso-downstream genes, m6A-correlation genes, with blue or brown module genes. As a result, 56 m6A-modified gene signatures were obtained. Among them, AKAP9, PLXNB2, BRPF3, HPS4, EXOC7, and KLF6 have an inconsistent expression in protein and mRNA levels, probably due to m6A modification. In addition, these genes may be involved in regulating drug resistance. Conclusions 56 m6A-modified gene signatures for LUAD tumorigenesis were obtained from Pearson correlation analysis between mRNA-seq and m6A-seq data, along with LASSO and WGCNA analysis. Among them, AKAP9, PLXNB2, BRPF3, HPS4, EXOC7 and KLF6 play a crucial role in LUAD tumorigenesis in an m6A modification-dependent manner. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-02106-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8a1734ebf3b7cc16ea04d4fbcf6ec4bc799a0c4" target='_blank'>
              Identification of m6A-modified gene signatures in lung adenocarcinoma tumorigenesis and their potential role in drug resistance
              </a>
            </td>
          <td>
            Xiaomin Han, Qiang Ma, Ruyi Chang, Siyuan Xin, Guojun Zhang, Ruilong Wang, Yukun Wang
          </td>
          <td>2025-03-25</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/460492d7d53dee1267ea7e9d4f8344913e2a4226" target='_blank'>
              Dead-End protein expression, function, and mutation in cancer: a systematic review.
              </a>
            </td>
          <td>
            Homa Faraji, F. Banakar, Arash Sadri, A. Ebrahim-Habibi
          </td>
          <td>2025-03-07</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/654c576507b39c0969a5f79ea0daae0b983731a4" target='_blank'>
              Riding the Wave of Progress: Examining the Current Landscape and Future Potential of MicroRNAs in Cancer Gene Therapy.
              </a>
            </td>
          <td>
            Saleem Ahmad, Hanif Khan, Njoka Irene Muthoni, Mohammed Alissa, Awaji Y. Safhi, F. Y. Sabei, Osama Abdulaziz, Khadijah Hassan Khan, Essam H Ibrahim, S. Khan
          </td>
          <td>2025-03-24</td>
          <td>Current gene therapy</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c926ae9ba30483b00d605482594a078fa10068fc" target='_blank'>
              The landscape of cell lineage tracing.
              </a>
            </td>
          <td>
            Ye Feng, Guang Liu, Haiqing Li, Lin Cheng
          </td>
          <td>2025-02-28</td>
          <td>Science China. Life sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Brain metastasis represents a significant challenge in oncology, driven by complex molecular and epigenetic mechanisms that distinguish it from primary tumors. While recent research has focused on identifying genomic mutation drivers with potential clinical utility, these strategies have not pinpointed specific genetic mutations responsible for site-specific metastasis to the brain. It is now clear that successful brain colonization by metastatic cancer cells requires intricate interactions with the brain tumor ecosystem and the acquisition of specialized molecular traits that facilitate their adaptation to this highly selective environment. This is best exemplified by widespread transcriptional adaptation during brain metastasis, resulting in aberrant gene programs that promote extravasation, seeding, and colonization of the brain. Increasing evidence suggests that epigenetic mechanisms play a significant role in shaping these pro-brain metastasis traits. This review explores dysregulated chromatin patterns driven by chromatin remodeling, histone modifications, DNA/RNA methylation, and other epigenetic regulators that underpin brain metastatic seeding, initiation, and outgrowth. We provide novel insights into how these epigenetic modifications arise within both the brain metastatic tumor and the surrounding brain metastatic tumor ecosystem. Finally, we discuss how the inherent plasticity and reversibility of the epigenomic landscape in brain metastases may offer new therapeutic opportunities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fe358dc3b7575e837baed5ca2b106b75b4f36a3" target='_blank'>
              The epigenetic landscape of brain metastasis.
              </a>
            </td>
          <td>
            Aoibhín M Powell, Louise Watson, L. Luzietti, S. Prekovic, L. Young, D. Varešlija
          </td>
          <td>2025-02-27</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Genome sequencing of cancer and normal tissues, alongside single-cell transcriptomics, continues to produce findings that challenge the idea that cancer is a 'genetic disease', as posited by the somatic mutation theory (SMT). In this prevailing paradigm, tumorigenesis is caused by cancer-driving somatic mutations and clonal expansion. However, results from tumor sequencing, motivated by the genetic paradigm itself, create apparent 'paradoxes' that are not conducive to a pure SMT. But beyond genetic causation, the new results lend credence to old ideas from organismal biology. To resolve inconsistencies between the genetic paradigm of cancer and biological reality, we must complement deep sequencing with deep thinking: embrace formal theory and historicity of biological entities, and (re)consider non-genetic plasticity of cells and tissues. In this Essay, we discuss the concepts of cell state dynamics and tissue fields that emerge from the collective action of genes and of cells in their morphogenetic context, respectively, and how they help explain inconsistencies in the data in the context of SMT.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f9b3a10a05d5290d45b66f194c7384d1f7517bb" target='_blank'>
              The end of the genetic paradigm of cancer.
              </a>
            </td>
          <td>
            Sui Huang, Ana M Soto, C. Sonnenschein
          </td>
          <td>2025-03-01</td>
          <td>PLoS biology</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Head and neck squamous cell carcinoma (HNSCC) remains a prevalent and lethal malignancy, with a five-year survival rate of just 50% for cases of locally advanced disease. Chromosomal aberrations, particularly the deletion of the short arm of chromosome 3 (3p), have been strongly associated with poor prognosis and more aggressive tumor phenotypes. The tumor microenvironment (TME) plays a pivotal role in tumor progression and resistance to therapy. This study aims to elucidate the impact of 3p deletion on the TME, immune cell infiltration, and treatment response in HNSCC, to identify novel therapeutic targets to improve patient outcomes. We analyzed single-cell RNA sequencing (scRNA-seq) data from the Gene Expression Omnibus (GEO) and bulk transcriptome data from The Cancer Genome Atlas (TCGA). Pseudo-time trajectory and cell-cell communication analyses were performed with the Monocle and CellChat packages. The Wilcoxon test was used to evaluate the differential gene expression between wild-type (wt) and mutant (mut) groups. Prognostic models were developed using the Least Absolute Shrinkage and Selection Operator (LASSO) algorithm and Cox regression analyses to find the genes related to survival, with survival analysis conducted via Kaplan-Meier curves. Gene set enrichment analysis (GSEA) was employed to investigate pathway dysregulation, and immune cell infiltration was assessed using various immune scoring methodologies to explore the differences immune environment. The Tumor Immune Dysfunction and Exclusion (TIDE) database was utilized to predict the potential efficacy of immune checkpoint inhibitors. mRNA and protein expression levels of SPP1 were examined by RT-qPCR and Western blotting, while cell proliferation was assessed using the CCK8 assay. The mut group demonstrated significant alterations in cellular composition, characterized by increased endothelial cells and macrophages and decreased fibroblasts and CD8 + T cells, indicative of an immunosuppressive TME. Differential expression analysis revealed downregulation of immune pathways, including antigen processing and presentation, T cell receptor signaling, and B cell receptor signaling pathways in the mut group, along with enhanced metabolic activity in glycolysis and lipid metabolism. The prognostic model identified nine key genes associated with poor survival in HNSCC. The mut group exhibited poorer overall survival and a more immunosuppressive microenvironment compared to the wt group, which correlated with the outcomes observed in high-risk versus low-risk groups. High-risk patients also showed a diminished response to immunotherapy compared to low-risk patients. Additionally, SPP1 emerged as a critical gene associated with chemotherapy resistance and macrophage M2 polarization. This study demonstrates that 3p deletion significantly reshapes the TME, contributing to poor prognosis in HNSCC by fostering an immunosuppressive environment and enhancing chemoresistance. These findings highlight the potential for developing targeted therapies that address the genetic and immunological landscape of HNSCC. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-92078-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55bd28583d34995b3cfd1d2547de4bdf67a2acf7" target='_blank'>
              Integrative single cell transcriptomic analysis reveals 3p deletion associated tumor microenvironment and chemoresistance in head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Xiaochuan Chen, Siqi Xu, Junping Pan, Wenqian Xu, Hanxuan Yang, Xin Chen, Ronghui Chen, Yuan Wang, Sufang Qiu
          </td>
          <td>2025-03-10</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Abstract Functional characterization of genetic variants has the potential to advance the field of precision medicine by enhancing the efficacy of current therapies and accelerating the development of new approaches to combat genetic diseases. MUTYH is a DNA repair enzyme that recognizes and repairs oxidatively damaged guanines [8-oxoguanine (8-oxoG)] mispaired with adenines (8-oxoG·A). While some mutations in the MUTYH gene are associated with colorectal cancer, most MUTYH variants identified in sequencing databases are classified as variants of uncertain significance. Convoluting clinical classification is the absence of data directly comparing homozygous versus heterozygous MUTYH mutations. In this study, we present the first effort to functionally characterize MUTYH variants using precision genome editing to generate heterozygous and homozygous isogenic cell lines. Using a MUTYH-specific lesion reporter in which we site-specifically incorporate an 8-oxoG·A lesion in a fluorescent protein gene, we measure endogenous MUTYH enzymatic activity and classify them as pathogenic or benign. Further, we modify this reporter to incorporate the MUTYH repair intermediate (8-oxoG across from an abasic site) and validate it with co-immunoprecipitation experiments to demonstrate its ability to characterize the mechanism by which MUTYH mutants are defective at DNA repair.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2e7c0e491d2ef8489374e03b9d4bf701b388bba" target='_blank'>
              Precision genome editing and in-cell measurements of oxidative DNA damage repair enable functional and mechanistic characterization of cancer-associated MUTYH variants
              </a>
            </td>
          <td>
            Carlos A Vasquez, Nicola R B Osgood, Marcanthony U Zepeda, Dominika K Sandel, Quinn T. Cowan, Malalage N. Peiris, Daniel J Donoghue, Alexis C. Komor
          </td>
          <td>2025-03-20</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dd1c02bf99dc94856478bd198e131107c59bd8c" target='_blank'>
              KDM5 demethylases suppress R-loop-mediated “viral mimicry” and DNA damage in breast cancer cells
              </a>
            </td>
          <td>
            Lena Lau, Kurt W Henderson, Ahu Turkoz, Sara Linker, Dörte Schlesinger, Brad Townsley, Brian Egan, Shoba Ragunathan, Robert Rollins, Xianju Bi, Zhijian J. Chen, Oleg Brodsky, Clifford Restaino, Murali Gururajan, Kristen Jensen-Pergakes, Anders Malarstig, Chames Kermi, Paul C Moore, Marie Classon
          </td>
          <td>2025-03-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Glioblastoma (GBM) is an aggressive malignant brain tumor with almost inevitable recurrence despite multimodal management with surgical resection and radio-chemotherapy. While several genetic, proteomic, cellular, and anatomic factors interplay to drive recurrence and promote treatment resistance, the epigenetic component remains among the most versatile and heterogeneous of these factors. Herein, the epigenetic landscape of GBM refers to a myriad of modifications and processes that can alter gene expression without altering the genetic code of cancer cells. These processes encompass DNA methylation, histone modification, chromatin remodeling, and non-coding RNA molecules, all of which have been found to be implicated in augmenting the tumor’s aggressive behavior and driving its resistance to therapeutics. This review aims to delve into the underlying interactions that mediate this role for each of these epigenetic components. Further, it discusses the two-way relationship between epigenetic modifications and tumor heterogeneity and plasticity, which are crucial to effectively treat GBM. Finally, we build on the previous characterization of epigenetic modifications and interactions to explore specific targets that have been investigated for the development of promising therapeutic agents.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3dc0fa03a9f64a66bc6beac8c2a3a49bfa95cc71" target='_blank'>
              The epigenetic mechanisms involved in the treatment resistance of glioblastoma
              </a>
            </td>
          <td>
            Aanya Shahani, Hasan Slika, Ahmad Elbeltagy, Alexandra Lee, Christopher Peters, Toriyn Dotson, Divyaansh Raj, Betty Tyler
          </td>
          <td>2025-02-25</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e945cd17bb6878accfa941b57523d14ed0d229c" target='_blank'>
              Genomic and transcriptomic sequencing in prostate cancer.
              </a>
            </td>
          <td>
            Safiullah Rifai, Azimullah Rifai, Xiaolei Shi, M. A. Khan, Wei Guang, Linbo Wang, Luke Tallon, Arif Hussain
          </td>
          <td>2025-02-28</td>
          <td>Current opinion in oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="High-grade gliomas (HGG) are incurable brain malignancies in children and adults. Breakthrough advances in transcriptomic technologies unveiled the intricate diversity of cellular states and their spatial organization within HGGs. We qualitatively integrated 55 neoplastic transcriptomic signatures described in 17 single-cell and spatial RNA sequencing-based studies. Our review delineates a spectrum of cellular states, represented by the expression of specific genes, which can be conceptualized along a “reactive-developmental programs” axis. Additionally, we discussed the potential cues influencing these cellular states, including how spatial organization may impact transcriptomic dynamics. Leveraging these insightful discoveries, we discussed a novel, evolutive way to integrate the different transcriptomic signatures in two or three dimensions, incorporating developmental states, their proliferative capacity, and their possible transition towards reactive states. This integrated analysis illuminates the diverse cellular landscape of HGGs and provides a valuable resource for further elucidation of malignant mechanisms, and for the design of therapeutic endeavors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eda3dd1ef6762a74a82f982637ae107d4fb74853" target='_blank'>
              An integrated perspective on single-cell and spatial transcriptomic signatures in high-grade gliomas
              </a>
            </td>
          <td>
            Célia Lemoine, Marc-Antoine Da Veiga, B. Rogister, Caroline Piette, Virginie Neirinckx
          </td>
          <td>2025-02-11</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/006d2b04fc1884e4bfd784cf684ee4d760f680ad" target='_blank'>
              Exploring the tumor suppressor role of RIN1 in familial thyroid carcinoma.
              </a>
            </td>
          <td>
            Luna Picello, Mattia Dalle Nogare, Daniele Puggina, Cecilia Salvoro, G. Pennelli, G. Gaudenzi, S. Carra, Monica Oldani, Davide Gentilini, Laura Fugazzola, A. Faggiano, Giovanni Vitale, Gianluca Occhi, Giovanni Vazza
          </td>
          <td>2025-03-01</td>
          <td>Endocrine-related cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="ABSTRACT The transcription factor p53 is the most frequently impaired tumor suppressor in human cancers. In response to various stress stimuli, p53 activates transcription of genes that mediate its tumor-suppressive functions. Distinctive characteristics of p53 outlined here enable a well-defined program of genes involved in cell cycle arrest, apoptosis, senescence, differentiation, metabolism, autophagy, DNA repair, anti-viral response, and anti-metastatic functions, as well as facilitating autoregulation within the p53 network. This versatile, anti-cancer network governed chiefly by a single protein represents an immense opportunity for targeted cancer treatment, since about half of human tumors retain unmutated p53. During the last two decades, numerous compounds have been developed to block the interaction of p53 with the main negative regulator MDM2. However, small molecule inhibitors of MDM2 only induce a therapeutically desirable apoptotic response in a limited number of cancer types. Moreover, clinical trials of the MDM2 inhibitors as monotherapies have not met expectations and have revealed hematological toxicity as a characteristic adverse effect across this drug class. Currently, combination treatments are the leading strategy for enhancing efficacy and reducing adverse effects of MDM2 inhibitors. This review summarizes efforts to identify and test therapeutics that work synergistically with MDM2 inhibitors. Two main types of drugs have emerged among compounds used in the following combination treatments: first, modulators of the p53-regulated transcriptome (including chromatin modifiers), translatome, and proteome, and second, drugs targeting the downstream pathways such as apoptosis, cell cycle arrest, DNA repair, metabolic stress response, immune response, ferroptosis, and growth factor signaling. Here, we review the current literature in this field, while also highlighting overarching principles that could guide target selection in future combination treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3895fe8ca5319763195e6c375cf525ba1915bfa" target='_blank'>
              Harnessing p53 for targeted cancer therapy: new advances and future directions
              </a>
            </td>
          <td>
            Z. Andrysik, Joaquin M Espinosa
          </td>
          <td>2025-03-03</td>
          <td>Transcription</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="The hierarchical organization of the eukaryotic genome is crucial for nuclear activities and cellular development. Genetic aberrations can disrupt this 3D genomic architecture, potentially driving oncogenesis. However, current research often lacks a comprehensive perspective, focusing on specific mutation types and singular 3D structural levels. Here, pathological changes from chromosomes to nucleotides are systematically cataloged, including 10 789 interchromosomal translocations (ICTs), 18 863 structural variants (SVs), and 162 769 single nucleotide polymorphisms (SNPs). The multilayered analysis reveals that fewer than 10% of ICTs disrupt territories via potent 3D interactions, and only a minimal fraction of SVs disrupt compartments or intersect topologically associated domain structures, yet these events significantly influence gene expression. Pathogenic SNPs typically show reduced interactions within the 3D genomic space. To investigate the effects of variants in the context of 3D organization, a two-phase scoring algorithm, 3DFunc, is developed to evaluate the pathogenicity of variant-gene pairs in cancer. Using 3DFunc, IGHV3-23's critical role in chronic lymphocytic leukemia is identified and it is found that three pathological SNPs (rs6605578, rs7814783, rs2738144) interact with DEFA3. Additionally, 3DGAtlas is introduced, which provides a highly accessible 3D genome atlas and a valuable resource for exploring the pathological effects of genetic mutations in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/daa65f99b6475aca863361e2ed2a67fe83692271" target='_blank'>
              A 3D Genome Atlas of Genetic Variants and Their Pathological Effects in Cancer.
              </a>
            </td>
          <td>
            Li Tang, M. C. Hill, Mingxing He, Junhao Chen, Zirui Wang, Patrick T. Ellinor, Min Li
          </td>
          <td>2025-03-25</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0314619e822b8ab25ecabe0ad0985b3e4d4d9a40" target='_blank'>
              SCRaMbLE: a versatile tool for genome manipulation.
              </a>
            </td>
          <td>
            Li Cheng, Jielin Li, Yaojun Chen, Junbiao Dai
          </td>
          <td>2025-02-12</td>
          <td>Physiology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background: The cell cycle, a critical and intricate biological process, comprises various phases, and its dysregulation plays a pivotal role in tumorigenesis and metastasis. The exploration of cell cycle-based molecular subtypes across pan-cancers, along with the application of synthetic lethality concepts, holds promise for advancing cancer therapies. Methods: A pan-cancer analysis was conducted to assess the cell cycle serves as a reliable signature for classifying molecular subtypes and to understand the potential clinical application of genes as potential drug targets based on synthetic lethality. Results: Molecular subtypes derived from cell cycle features in certain cancers, particularly kidney-related malignancies, exhibited distinct immune characteristics. Synthetic lethal interactions within the cell cycle pathway were common, with significant genetic interactions further identifying potential drug targets through the exploitation of genetic relationships with key driver genes. Additionally, miRNAs and lncRNAs may influence the cell cycle through miRNA:mRNA interactions and ceRNA networks, thereby enriching the genetic interaction landscape. Conclusions: These findings suggest that the cell cycle pathway could serve as a promising molecular subtype signature to enhance cancer prognostication and offer potential targets for anticancer drug development through synthetic lethality.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74f0509708b74787ef228103fd17aa6cf4b5bae5" target='_blank'>
              Cell Cycle-Based Molecular Features via Synthetic Lethality and Non-Coding RNA Interactions in Cancer
              </a>
            </td>
          <td>
            Shizheng Xiong, Jiaming Jin, Xinmiao Zhao, Yang Zhao, Zhiheng He, Haochuan Guo, Chengjun Gong, Jiafeng Yu, Li Guo, Tingming Liang
          </td>
          <td>2025-03-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40c8b8c8967239c4ebdf5921b795eefe1341f9d3" target='_blank'>
              Single-cell eQTL mapping reveals cell subtype-specific genetic control and mechanism in malignant transformation of colorectal cancer.
              </a>
            </td>
          <td>
            Can Chen, Yimin Cai, Wenlong Hu, Kai Tan, Zequn Lu, Xu Zhu, Ziying Liu, Chunyi He, Guangping Xu, Ruizhe Zhang, Caibo Ning, Shuheng Ruan, Jiayan Gao, Xiaojun Yang, Yongchang Wei, Xu Zhu, Xiangpan Li, Faxi Wang, Fubing Wang, Jiaoyuan Li, Meng Jin, Bin Li, Ying Zhu, Jianbo Tian, Xiaoping Miao
          </td>
          <td>2025-03-04</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29817e74ca4bd94e1017f077991d98ef4dbbdc5b" target='_blank'>
              De novo detection of somatic variants in high-quality long-read single-cell RNA sequencing data.
              </a>
            </td>
          <td>
            Arthur Dondi, Nico Borgsmüller, Pedro F. Ferreira, Brian J. Haas, Francis Jacob, V. Heinzelmann-Schwarz, N. Beerenwinkel
          </td>
          <td>2025-03-19</td>
          <td>Genome research</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Background Acute lymphoblastic leukemia (ALL) is the most common childhood cancer, comprising approximately 25% of pediatric malignancies. Notably, chromosomal aberrations and genetic alterations play a central role in the pathogenesis of ALL, serving as critical diagnostic and prognostic markers. In this study, we use array-based comparative genomic hybridization (array-CGH) to explore the landscape of copy number variations (CNVs) and variants of uncertain significance (VUS) in 67 Malaysian childhood ALL patients with normal karyotype. Results A total of 36 pathogenic CNVs (26 gains, 10 losses) were identified in 19 (28.4%) patients which harbor genes related to the development of ALL. The genes include the MLLT3 (9p21.3), ETV6 (12p13.2), RUNX1 (21q22.12), ERG (21q22.2) and DMD (Xp21.1). On the other hand, a total of 46 variants of uncertain significance (VUS) was observed in 34 (50.7%) patients. Conclusions Our study indicates that array-CGH is able to identify and characterize the CNVs responsible for the pathogenesis of childhood ALL. However, further studies are required to determine the pathogenic implications of VUS in the development of childhood ALL. Supplementary Information The online version contains supplementary material available at 10.1186/s13039-025-00709-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa4d7cd5e5ce30655ab47d932b22f9c51cbf343c" target='_blank'>
              Genomic landscape of childhood acute lymphoblastic leukemia in Malaysia: insights from array-CGH
              </a>
            </td>
          <td>
            Azli Ismail, Fadly Ahid, Wong Nyuk Moi, N. Kamaluddin, E. Esa, Y. M. Yusoff, Z. A. Seman, Muhammad Asyraff Mohammed, Elizabeth George, Asmida Isa, Z. Zakaria
          </td>
          <td>2025-03-28</td>
          <td>Molecular Cytogenetics</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Gene network models provide a foundation for graph theory approaches, aiding in the novel discovery of drug targets, disease genes, and genetic mechanisms for various biological functions. Disease genetics must be interpreted within the cellular context of disease-associated cell types, which cannot be achieved with datasets consisting solely of organism-level samples. Single-cell RNA sequencing (scRNA-seq) technology allows computational distinction of cell states which provides a unique opportunity to understand cellular biology that drives disease processes. Importantly, the abundance of cell samples with their transcriptome-wide profile allows the modeling of systemic cell-type-specific gene networks (CGNs), offering insights into gene-cell-disease relationships. In this review, we present reference-based and de novo inference of gene functional interaction networks that we have recently developed using scRNA-seq datasets. We also introduce a compendium of CGNs as a useful resource for cell-type-resolved disease genetics. By leveraging these advances, we envision single-cell network biology as the key approach for mapping the gene-cell-disease axis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9aa7b9ecda0e5aabd9c3870f99cf01184de548a3" target='_blank'>
              Single-cell network biology enabling cell-type-resolved disease genetics
              </a>
            </td>
          <td>
            Junha Cha, Insuk Lee
          </td>
          <td>2025-03-27</td>
          <td>Genomics & Informatics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The cellular programs that mediate therapy resistance are often important drivers of metastasis, a phenomenon that needs to be understood better to improve screening and treatment options for cancer patients. Although this issue has been studied extensively for chemotherapy, less is known about a causal link between resistance to radiation therapy and metastasis. We investigated this problem in triple-negative breast cancer (TNBC) and established that radiation resistant tumor cells have enhanced metastatic capacity, especially to bone. Resistance to radiation increases the expression of integrin β3 (ITGβ3), which promotes enhanced migration and invasion. Bioinformatic analysis and subsequent experimentation revealed an enrichment of RNA metabolism pathways that stabilize ITGβ3 transcripts. Specifically, the RNA binding protein heterogenous nuclear ribonucleoprotein L (HNRNPL), whose expression is regulated by Nrf2, mediates the formation of circular RNAs (circRNAs) that function as competing endogenous RNAs (ceRNAs) for the family of let-7 microRNAs that target ITGβ3. Collectively, our findings identify a novel mechanism of radiation-induced metastasis that is driven by alterations in RNA metabolism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14903a27d305650e2051ca5d1898fdc092718345" target='_blank'>
              Resistance to Radiation Enhances Metastasis by Altering RNA Metabolism
              </a>
            </td>
          <td>
            Ayush Kumar, Kensei Kishimoto, H. Goel, C. Wisniewski, Rui Li, Brendan Pacheco, L. J. Zhu, William A. Flavahan, A. Mercurio
          </td>
          <td>2025-02-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="


 Recent progress in parasite genomics has is enabled for greater understanding of genetic structure of parasites, including the species of Plasmodium, Leishmania, and Trypanosoma. Cutting-edge next-generation sequencing (NGS) techniques, such as high-throughput sequencing, has enabled the discovery of essential genes linked to the metabolic pathways, drug resistance, and life cycle adaptations. Advancements in the field of genomics have enabled the deep understanding of biological processes and evolutionary adaptations of parasites.



 Investigating the genomes in various parasite strains has enhanced our understanding of their evolutionary adaptations, enhancing our capacity to predict epidemics and develop therapies that effectively act against various parasitic strains. Synthetic biology has also proposed novel treatment approaches, including the gene therapies and bioengineered microbes, that shows potential in fighting or inhibiting parasite illnesses. Revolutionary genome-editing methods, such as CRISPR-Cas9, have enabled the accurate genetic alterations, expediting the progress of sophisticated medicinal therapies, that are specifically designed for parasite management and eradication.



 Microbiome engineering, an emerging area, provides a novel opportunities for disease prevention by integrating the techniques such as transplanting faecal microbiota and genetically tailored bacteria to restore microbial equilibrium and decrease parasite populations. Nevertheless, the enduring stability of modified microbiomes, possible environmental hazards, and ethical concerns related to gene editing emphasise the necessity for stringent safety measures and monitoring by regulatory authorities. Using these technologies responsibly and ethically is crucial to guarantee the continuous therapeutic advancement.



">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a096b660a945d20f83a8eed57efbfc62f1dfd3e3" target='_blank'>
              Synthetic biology and parasite genomics: engineering parasite-resistant human microbiomes for sustainable disease prevention
              </a>
            </td>
          <td>
            E. S. Al-Malki
          </td>
          <td>2025-02-18</td>
          <td>Beni-Suef University Journal of Basic and Applied Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Single nucleotide polymorphisms (SNPs) in DNA repair genes can impair protein structure and function, contributing to disease development, including cancer. Non-synonymous SNPs (nsSNPs) in the LIG3 gene are linked to genomic instability and increased cancer risk, particularly acute myeloid leukemia (AML). This study aims to identify the most deleterious nsSNPs in the LIG3 gene and potential therapeutic targets for DNA repair restoration in AML. We employed in-silico computational methods to analyze LIG3 nsSNPs, using PredictSNP and Mutation3D to assess pathogenicity. Subsequently, molecular docking and dynamics simulations were conducted to evaluate ligand-binding affinities and protein stability. Out of the 12,191 mapped SNPs, 132 were nsSNPs located in the coding region. Among these, 18 nsSNPs were identified as detrimental including 12 destabilizing and 6 stabilizing nsSNPs. Nine cancer-associated nsSNPs, including L381R and R528C, were predicted due to their structural and functional impacts. Further analysis revealed key phosphorylation and methylation sites, such as 529S and 224R. Molecular dynamics simulations highlighted stable interactions of compounds AHP-MPC and DM-BFC with wild-type and R528C mutant LIG3 proteins, while R671G and V781M mutants showed instability. Protein-protein interaction networks and functional enrichment linked LIG3 to DNA repair pathways. Kaplan-Meier analysis associated high LIG3 expression with improved survival in breast cancer and AML, suggesting its role as a prognostic biomarker. This study emphasizes the mutation-specific effects of LIG3 nsSNPs on protein stability and ligand interactions. We recommend identifying DM-BFC to advance personalized medicine approaches for targeting deleterious variants, following in vitro and in vivo validation for AML treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87ba18559370ff8c79b7d669e449bdf299eda4ad" target='_blank'>
              Computational investigation unveils pathogenic LIG3 non-synonymous mutations and therapeutic targets in acute myeloid leukemia
              </a>
            </td>
          <td>
            Md. Arif Hossen, Umme Mim Sad Jahan, Md. Arju Hossain, Khalid Hossain Asif, Ahona Rahman, Sabbir Ahmed, Moin Uddin, Md Faisal Amin, Muhammad Abdul Bari, M. Islam, Mohammad Kamruzzaman, Soharth Hasnat, Mohammad Nasir Uddin, Tofazzal Islam, M. Hoque
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56b7ab64f4c6c0ffff56fec877ba1f8433465976" target='_blank'>
              Single-cell RNA sequencing revealed changes in the tumor microenvironment induced by radiotherapy for cervical cancer and the molecular mechanism of mast cells in immunosuppression.
              </a>
            </td>
          <td>
            L. Xue, Linzhi Gao, Shimin Zhou, Chaofan Yan, Xian Zhang, Wei Lin, Hu Li, Yuan Shen, Xiaoyu Wang
          </td>
          <td>2025-03-14</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="ABSTRACT Colibactin is a genotoxic secondary metabolite produced by certain Enterobacteriaceae strains that populate the intestine and produces a specific mutational signature in human colonocytes. However, the host pathways involved in colibactin response remain unclear. To address this gap, we performed genome-wide CRISPR/Cas9 knockout screens and RNA sequencing utilizing live pks+ bacteria and a synthetic colibactin analog. We identified 20 enriched genes with a MAGeCK score of >2.0 in both screens, including proteasomal subunits (e.g., PSMG4 and PSMD4), RNA processing factors (e.g., SF1 and PRPF8), and RNA polymerase III (e.g., CRCP), and validated the role of PSMD4 in colibactin sensitization. PSMD4 knockout in HEK293T and HT-29 cells promoted cell viability and ameliorated G2-M cell cycle arrest but did not affect the amount of phosphorylated H2AX foci after exposure to synthetic colibactin 742. Consistent with these observations, PSMD4−/− cells had a significantly higher colony formation rate and bigger colony size than control cells after 742 exposure. These findings suggest that PSMD4 regulates cell cycle arrest following colibactin-induced DNA damage and that cells with PSMD4 deficiency may continue to replicate despite DNA damage, potentially increasing the risk of malignant transformation. IMPORTANCE Colibactin has been implicated as a causative agent of colorectal cancer. However, colibactin-producing bacteria are also present in many healthy individuals, leading to the hypothesis that some aspects of colibactin regulation or host response dictate the molecule’s carcinogenic potential. Elucidating the host-response pathways involved in dictating cell fate after colibactin intoxication has been difficult, partially due to an inability to isolate the molecule. This study provides the first high-throughput CRISPR/Cas9 screening to identify genes conferring colibactin sensitivity. Here, we utilize both bacterial infection and a synthetic colibactin analog to identify genes directly involved in colibactin response. These findings provide insight into how differences in gene expression may render certain individuals more vulnerable to colibactin-initiated tumor formation after DNA damage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36c10ad4bc58eb16a2a76999a8e88cc374bfe8d0" target='_blank'>
              Genome-scale CRISPR/Cas9 screening reveals the role of PSMD4 in colibactin-mediated cell cycle arrest
              </a>
            </td>
          <td>
            Michael W. Dougherty, Ryan M. Hoffmann, Maria C. Hernandez, Yougant Airan, R. Gharaibeh, Seth B. Herzon, Ye Yang, Christian Jobin
          </td>
          <td>2025-02-07</td>
          <td>mSphere</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Background The ataxia-telangiectasia mutated (ATM) kinase phosphorylates and activates several downstream targets that are essential for DNA damage repair, cell cycle inhibition and apoptosis. Germline biallelic inactivation of the ATM gene causes ataxia-telangiectasia (A-T), and heterozygous pathogenic variant (PV) carriers are at increased risk of cancer, notably breast cancer. This study aimed to investigate whether DNA methylation profiling can be useful as a biomarker to identify tumors arising in ATM PV carriers, which may help for the management and optimal tailoring of therapies of these patients. Methods Breast tumor enriched DNA was prepared from 2 A-T patients, 27 patients carrying an ATM PV, 6 patients carrying a variant of uncertain clinical significance and 484 noncarriers enrolled in epidemiological studies conducted in France and Australia to investigate genetic and nongenetic factors involved in breast cancer susceptibility. Genome-wide DNA methylation analysis was performed using the Illumina Infinium HumanMethylation EPIC and 450K BeadChips. Correlation between promoter methylation and gene expression was assessed for 10 tumors for which transcriptomic data were available. Results We found that the ATM promoter was hypermethylated in 62% of tumors of heterozygous PV carriers compared to the mean methylation level of ATM promoter in tumors of noncarriers. Gene set enrichment analyses identified 47 biological pathways enriched in hypermethylated genes involved in neoplastic, neurodegenerative and metabolic-related pathways in tumor of PV carriers. Among the 327 differentially methylated promoters, promoters of ARHGAP40, SCGB3A1 (HIN-1), and CYBRD1 (DCYTB) were hypermethylated and associated with a lower gene expression in these tumors. Moreover, using three different deep learning algorithms (logistic regression, random forest and XGBoost), we identified a set of 27 additional biomarkers predictive of ATM status, which could be used in the future to provide evidence for or against pathogenicity in ATM variant classification strategies. Conclusions We showed that breast tumors that arise in women who carry an ATM PV display a specific genome-wide DNA methylation profile. Specifically, the methylation pattern of 27 key gene promoters was predictive of ATM PV status of the women. These genes may also represent new medical prevention and therapeutic targets for these women. Supplementary Information The online version contains supplementary material available at 10.1186/s13058-025-01988-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bebfd1b3f4c2e2d684ea0f216e88d7bab9a7fec5" target='_blank'>
              Breast tumors from ATM pathogenic variant carriers display a specific genome-wide DNA methylation profile
              </a>
            </td>
          <td>
            Nicolas M Viart, Anne-Laure Renault, Séverine Eon-Marchais, Yue Jiao, L. Fuhrmann, Sophia Murat El Houdigui, Dorothée Le Gal, Eve Cavaciuti, Marie-Gabrielle Dondon, Juana Beauvallet, Virginie Raynal, D. Stoppa-Lyonnet, A. Vincent-Salomon, Nadine Andrieu, M. Southey, F. Lesueur
          </td>
          <td>2025-03-11</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>118</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e83d84b8f1cfdb09485b80edd7f374acbe1eeac9" target='_blank'>
              Molecular mechanisms of Hippo pathway in tumorigenesis: therapeutic implications.
              </a>
            </td>
          <td>
            M. Saadh, Hanan Hassan Ahmed, Radhwan Abdul Kareem, A. K. Bishoyi, R. Roopashree, Debasish Shit, Renu Arya, Abhishek Sharma, Kakhramon Khaitov, Hayder Naji Sameer, Ahmed Yaseen, Zainab H. Athab, Mohaned Adil
          </td>
          <td>2025-02-27</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background Acute myelocytic leukemia (AML) is a common hematological malignancy in adults. Although several risk stratifications based on cytogenetic and molecular abnormalities are available to guide the indications for allogeneic hematopoietic cell transplantation (allo-HCT), determining optimal treatment strategies for AML remains challenging. In this study, using transcriptome datasets, we investigated the association between event-free survival (EFS) in intensively treated AML patients and the aberrant expression of endogenous viral element (EVE)-derived open reading frames (ORFs), which have been reported to be associated with the pathophysiology of various malignancies and have the potential to serve as neoantigens in specific cancers. Results The expression levels of human endogenous retrovirus family K9 (HERVK9) ORFs were associated with EFS, independent of conventional risk stratification. Furthermore, AML cells with higher levels of HERVK9 expression exhibited enhanced antigen processing and presentation, along with increased expression of genes associated with adaptive immune responses and apoptosis, indicating that aberrant HERVK9 expression may initiate an anti-neoplastic immune response via increased antigen presentation. Conclusions HERVK9 expression may have serve as a crucial prognostic indicator that could aid in determining the indications for upfront allo-HCT in AML patients. Supplementary Information The online version contains supplementary material available at 10.1186/s12977-025-00661-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5f29628a7105d4d628b72c6ef996a0f86eb8dca" target='_blank'>
              Aberrant expression of human endogenous retrovirus K9-derived elements is associated with better clinical outcome of acute myelocytic leukemia
              </a>
            </td>
          <td>
            Ryo Yanagiya, So Nakagawa, M. Onizuka, Ai Kotani
          </td>
          <td>2025-04-01</td>
          <td>Retrovirology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3df1af238c5c0bc411539014f0c361b61ef1f54" target='_blank'>
              Single Cell Resolution Tracking of Cutaneous T-Cell Lymphoma Reveals Clonal Evolution in Disease Progression
              </a>
            </td>
          <td>
            Hannah K. Dorando, Jared M. Andrews, Nicholas Borcherding, Chaz C. Quinn, J. Schmidt, Oam Khatavkar, Jahnavi Aluri, Michael T. Harmon, Marcus P. Watkins, Anastasia Frank, Megan A. Cooper, Amy C. Musiek, Neha Mehta-Shah, Jacqueline E. Payton
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background The tumor suppressor protein, p53, which is mutated in half of human tumors, plays a critical role in cellular responses to DNA damage and maintenance of genome stability. Therefore, increasing our understanding of the p53 pathway is essential for improving cancer treatment and diagnosis. Methods This study, which aimed to identify genes and pathways that mediate resistance to p53 upregulation, used genome-wide CRISPR-Cas9 loss-of-function screening done with Nutlin-3a, which inhibits p53-MDM2 interaction, resulting in p53 accumulation and apoptotic cell death. We used bioinformatics analysis for the identification of genes and pathways that are involved in the p53 pathway and cell survival assays to validate specific genes. In addition, we used RNA-seq to identify differentially expressed p53 target genes in gene knockout (KO) cell lines. Results Our screen revealed three significantly enriched pathways: The heparan sulfate glycosaminoglycan biosynthesis, diphthamide biosynthesis and Hippo pathway. Notably, TRIP12 was significantly enriched in our screen. We found that TRIP12 is required for the p53-dependent transcription of several pro-apoptotic genes. Conclusion Our study has identified two novel pathways that play a role in p53-mediated growth restriction. Moreover, we have highlighted the interaction between the Hippo and the p53 pathways. Interestingly, we have shown that TRIP12 plays an important function in the p53 pathway by selectively affecting its role as a transcription factor. Supplementary Information The online version contains supplementary material available at 10.1186/s10020-025-01141-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61f2f128a395ad66b24bd497a265456014e21a60" target='_blank'>
              Genome-wide screening in human embryonic stem cells identifies genes and pathways involved in the p53 pathway
              </a>
            </td>
          <td>
            Amir Haddad, T. Golan‐lev, N. Benvenisty, Michal Goldberg
          </td>
          <td>2025-03-13</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Simple Summary Nilotinib and other tyrosine kinase inhibitors (TKIs) target the BCR-ABL1 oncoprotein in Chronic Myeloid Leukemia (CML), reducing the growth of leukemic stem cells (LSCs) and promoting their death. However, some LSCs persist in the bone marrow (BM), which may lead to relapse and disease progression. We analyzed the gene expression profiling of BM CD34+/lin− cells from 79 Chronic-Phase CML patients at diagnosis and after 12 months of nilotinib treatment, comparing them to the healthy cells from 10 donors (CTRLs). Our study identified key genes involved in eight crucial biological pathways: CML, cell cycle, JAK-STAT, PI3K-Akt, MAPK, Ras, NF-kB, and ABC transporters. We observed a progressive down-regulation of the expression levels of several genes across these pathways from diagnosis to 12 months of nilotinib treatment and to CTRLs. This could suggest that nilotinib effects create selective pressure, potentially supporting the survival and self-renewal of LSCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01d3a05e9350f9c4633e6a38d029b612b50597a4" target='_blank'>
              Bone Marrow CD34+/lin− Cells of Patients with Chronic-Phase Chronic Myeloid Leukemia (CP-CML) After 12 Months of Nilotinib Treatment Exhibit a Different Gene Expression Signature Compared to the Diagnosis and the Corresponding Cells from Healthy Subjects
              </a>
            </td>
          <td>
            Alessandra Trojani, E. Pungolino, Barbara Di Camillo, L. E. Bossi, Cassandra Palumbo, M. D’adda, A. Perego, Mauro Turrini, C. Elena, L. Borin, A. Iurlo, Simona Malato, Francesco Spina, M. L. Latargia, P. Spedini, Salvatore Artale, M. Anghilieri, M. Carraro, C. Bucelli, A. Beghini, R. Cairoli
          </td>
          <td>2025-03-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a prevalent malignant tumor characterized by high global incidence and mortality rates. Furthermore, it is imperative to comprehend the molecular mechanisms underlying its development and to identify effective prognostic markers. These efforts are crucial for pinpointing potential therapeutic targets and enhancing patient survival rates. Therefore, we develop a novel prognostic model aimed at providing new theoretical support for clinical prognosis evaluation and treatment. We downloaded data from the Gene Expression Omnibus (GEO) and The Cancer Genome Atlas (TCGA) databases. Subsequently, we performed single-cell analysis and developed a prognostic model associated with colorectal cancer. We divided the scRNA-seq dataset (GSE221575) into 19 cell clusters and classified these clusters into 11 distinct cell types using marker genes. Using univariate Cox regression and LASSO (Least Absolute Shrinkage and Selection Operator) analyses, we developed a prognostic model consisting of 9 genes. Based on our 9-gene model, we divided patients into high-risk and low-risk groups using the median risk score. The high-risk group demonstrated significant positive correlations with M0 macrophages, CD8+ T cells, and M2 macrophages. The enrichment analyses indicate significant enrichment of immune-related pathways in the high-risk group, including HEDGEHOG_SIGNALING, Wnt signaling pathway, and cell adhesion molecules. Drug sensitivity analysis revealed that the low-risk group was sensitive to 5 chemotherapeutic drugs, while the high-risk group was sensitive to only 1. Additionally, we developed a highly reliable nomogram for clinical application. This suggests that the risk score derived from our modeling analysis is highly effective for stratifying colorectal cancer samples. This study comprehensively applied bioinformatics methods to construct a risk score model. The model showed good predictive performance, offering potential guidance for individualized treatment of colorectal cancer patients. Furthermore, it may provide valuable insights into the disease’s pathogenesis and identify potential therapeutic targets for further research. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-91761-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a4ee3ba4be7a7e3b5e63b304d989c55900905b7" target='_blank'>
              Integrated analysis of single-cell and bulk RNA-sequencing to predict prognosis and therapeutic response for colorectal cancer
              </a>
            </td>
          <td>
            Liyang Cai, Xin Guo, Yucheng Zhang, Huajie Xie, Yongfeng Liu, Jianlong Zhou, Huolun Feng, Jiabin Zheng, Yong Li
          </td>
          <td>2025-03-07</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="To maintain cell fitness, deleterious genetic alterations are buffered by compensatory changes in additional genes. In cancer, buffering processes could be targeted by synthetic lethality. However, despite the large-scale identification of synthetic lethal effects in preclinical models, evidence that these operate clinically is limited. This impedes the application of synthetic lethal approaches. By integrating molecular profiling data from >9,000 cancers with synthetic lethal screens, we show that transcriptomic buffering of tumor suppressor gene (TSG) loss by hyperexpression of synthetic lethal partners is a common phenomenon, extending to multiple TSGs and histotypes. Transcriptomic buffering is also notable in cancers that phenocopy TSG loss, such as BRCAness cancers, where expression of BRCA1/2 synthetic lethal genes correlates with clinical outcome. Synthetic lethal genes that exhibit transcriptomic buffering also represent more robust synthetic lethal effects. These observations have implications for understanding how tumor cells tolerate TSG loss, in part explain transcriptomic architectures in cancer and provide insight into target selection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29a52e19b56a0e26f2474ec866322bdccd6f6854" target='_blank'>
              The transcriptomic architecture of common cancers reflects synthetic lethal interactions
              </a>
            </td>
          <td>
            S. Haider, R. Brough, Santiago Madera, Jacopo Iacovacci, Aditi Gulati, Andrew J. Wicks, J. Alexander, S. Pettitt, Andrew N J Tutt, C. J. Lord
          </td>
          <td>2025-03-03</td>
          <td>Nature Genetics</td>
          <td>1</td>
          <td>33</td>
        </tr>

        <tr id="Coronary artery disease (CAD) is the leading cause of death worldwide. Recently, hundreds of genomic loci have been shown to increase CAD risk, however, the molecular mechanisms underlying signals from CAD risk loci remain largely unclear. We sought to pinpoint the candidate causal coding and non-coding genes of CAD risk loci in a cell type-specific fashion. We integrated the latest statistics of CAD genetics from over one million individuals with epigenetic data from 45 relevant cell types to identify genes whose regulation is affected by CAD-associated single nucleotide variants (SNVs) via epigenetic mechanisms. Applying two statistical approaches, we identified 1,580 genes likely involved in CAD, about half of which have not been associated with the disease so far. Enrichment analysis and phenome-wide association studies linked the novel candidate genes to disease-specific pathways and CAD risk factors, corroborating their disease relevance. We showed that CAD-SNVs are enriched to regulate gene expression by affecting the binding of transcription factors (TFs) with cellular specificity. Of all the candidate genes, 23.5% represented non-coding RNAs (ncRNA), which likewise showed strong cell type specificity. We conducted a proof-of-concept biological validation for the novel CAD ncRNA gene IQCH-AS1. CRISPR/Cas9-based gene knockout of IQCH-AS1, in a human preadipocyte strain, resulted in reduced preadipocyte proliferation, less adipocyte lipid accumulation, and atherogenic cytokine profile. The cellular data are in line with the reduction of IQCH-AS1 in adipose tissues of CAD patients and the negative impact of risk alleles on its expression, suggesting IQCH-AS1 to be protective for CAD. Our study not only pinpoints CAD candidate genes in a cell type-specific fashion but also spotlights the roles of the understudied ncRNA genes in CAD genetics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/894c0b3eed40de06b0fd932a3214f3acdff103b5" target='_blank'>
              Cell type-specific epigenetic regulatory circuitry of coronary artery disease loci
              </a>
            </td>
          <td>
            Dennis Hecker, Xiaoning Song, Nina Baumgarten, A. Diagel, N. Katsaouni, Ling Li, Shuangyue Li, Ranjan Kumar Maji, Fatemeh Behjati Ardakani, Lijiang Ma, Daniel Tews, M. Wabitsch, J. Björkegren, H. Schunkert, Zhifen Chen, Marcel H. Schulz
          </td>
          <td>2025-02-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The mutational landscape of phylogenetically ultraconserved elements (UCEs), especially those in noncoding DNAs (ncUCEs), and their functional relevance in cancers remain poorly characterized. Here, we perform a systematic analysis of whole-genome and in-house targeted UCE sequencing datasets from more than 3000 patients with cancer of 13,736 UCEs and demonstrate that ncUCE somatic alterations are common. Using a multiplexed CRISPR knockout screen in colorectal cancer cells, we show that the loss of several altered ncUCEs significantly affects cell proliferation. In-depth functional studies in vitro and in vivo further reveal that specific ncUCEs can be enhancers of tumor suppressors (such as ARID1B) and silencers of oncogenic proteins (such as RPS13). Moreover, several miRNAs located in ncUCEs are recurrently mutated. Mutations in miR-142 locus can affect the Drosha-mediated processing of precursor miRNAs, resulting in the down-regulation of the mature transcript. These results provide systematic evidence that specific ncUCEs play diverse regulatory roles in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d50d62f517b45fea38f094fa47dd4b743a633f64" target='_blank'>
              The mutational landscape and functional effects of noncoding ultraconserved elements in human cancers
              </a>
            </td>
          <td>
            R. Bayraktar, Yitao Tang, M. Dragomir, Cristina Ivan, Xinxin Peng, L. Fabris, Jianhua Zhang, A. Carugo, S. Aneli, Jintan Liu, M. Chen, Sanjana Srinivasan, Iman Sahnoune, E. Bayraktar, K. Akdemir, Meng Chen, P. Narayanan, Wilson Huang, L. Ott, A. K. Eterovic, O. Villarreal, M. M. Mohammad, M. Peoples, Danielle M Walsh, J. A. Hernandez, Margaret B Morgan, K. Shaw, Jennifer S Davis, David Menter, Constantine S Tam, Paul Yeh, S. Dawson, L. Rassenti, T. Kipps, Tanja Kunej, Z. Estrov, S. Joosse, Luca Pagani, C. Alix-Panabières, Klaus Pantel, Alessandra Ferajoli, Andrew Futreal, I. Wistuba, Milan Radovich, S. Kopetz, Michael J Keating, Giulio F. Draetta, John S. Mattick, Han Liang, George A Calin
          </td>
          <td>2025-02-19</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>99</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a87c4098a87feebf01a217fad3de111af3e86862" target='_blank'>
              Abstract IA002: Functional omics for novel therapeutic combinations in myeloid malignancies
              </a>
            </td>
          <td>
            J. Tyner
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Somatic mobilization of LINE-1 (L1) has been implicated in cancer etiology. We analyzed a recent TCGA data release comprised of nearly 5000 pan-cancer paired tumor-normal whole-genome sequencing (WGS) samples and ~9000 tumor RNA samples. We developed TotalReCall an improved algorithm and pipeline for detection of L1 retrotransposition (RT), finding high correlation between L1 expression and “RT burden” per sample. Furthermore, we mathematically model the dual regulatory roles of p53, where mutations in TP53 disrupt regulation of both L1 expression and retrotransposition. We found those with Li-Fraumeni Syndrome (LFS) heritable TP53 pathogenic and likely pathogenic variants bear similarly high L1 activity compared to matched cancers from patients without LFS, suggesting this population be considered in attempts to target L1 therapeutically. Due to improved sensitivity, we detect over 10 genes beyond TP53 whose mutations correlate with L1, including ATRX, suggesting other, potentially targetable, mechanisms underlying L1 regulation in cancer remain to be discovered.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09c16a8e22a8e26a1832c9dc60f1f0a97f3c5163" target='_blank'>
              Pan-cancer multi-omic model of LINE-1 activity reveals locus heterogeneity of retrotransposition efficiency
              </a>
            </td>
          <td>
            Alexander Solovyov, Julie M. Behr, D. Hoyos, Eric Banks, Alexander W Drong, Bryan Thornlow, Jimmy Z Zhong, Enrique Garcia-Rivera, Wilson McKerrow, Chong Chu, Cédric Arisdakessian, Dennis M Zaller, J. Kamihara, Liyang Diao, M. Fromer, Benjamin D. Greenbaum
          </td>
          <td>2025-02-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Background Migrasomes, recently discovered cellular substructures, may play a crucial role in cancer progression, treatment response, and prognosis. However, the prognostic value of migrasome-associated long non-coding RNAs (lncRNAs) in colon adenocarcinoma (COAD) remains unexplored. Methods RNA-seq data from 459 COAD patients, including clinical characteristics and outcome information, were obtained from The Cancer Genome Atlas. A risk model was constructed through co-expression analysis of migrasome genes and lncRNAs, followed by Cox regression and least absolute shrinkage and selection operator analysis to identify prognostic lncRNAs. Functional enrichment analyses were performed to elucidate underlying biological mechanisms. Immune landscape characterization utilized ESTIMATE, CIBERSORT, Tumor Immune Estimation Resource (TIME), and single-sample Gene Set Enrichment Analysis (ssGSEA). Drug sensitivity analysis was conducted for select therapeutic agents. Results Nine prognostic lncRNAs (AC010463.3, AL590483.4, AP005264.1, ZEB1-AS1, AC104088.1, PRKAR1B-AS2, AC009315.1, SUCLG2-AS1, and AC006111.2) were identified and incorporated into a risk model. Low-risk patients demonstrated significantly improved survival outcomes. The model exhibited independent prognostic capability, with AUCs of 0.783, 0.749, and 0.713 for one-, three-, and five-year survival, respectively, in the training cohort. High-risk patients displayed reduced overall survival and elevated tumor mutation burden. Additionally, these patients showed decreased sensitivity to therapeutic agents, including Oxaliplatin, Irinotecan, and 5-Fluorouracil. Conclusion Our novel migrasome-associated lncRNA signature demonstrates robust predictive capacity for both prognosis and chemotherapeutic sensitivity in COAD, potentially facilitating personalized treatment strategies and improved patient management. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-02197-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/081cc01a6847cf137157802eb2a50ffa335c8848" target='_blank'>
              Comprehensive identification of a migrasomes-associated long non-coding RNA signature to predict the prognosis and treatment options in colon adenocarcinoma
              </a>
            </td>
          <td>
            Zhen Zheng, Hui Liu, Quan Xu, Wei Cui, Kaitai Liu
          </td>
          <td>2025-03-27</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Retrospective data support overall survival (OS) advantage to high clonal tumor mutation burden (cTMB), high clonal neoantigen load (cNEO) and low intratumor heterogeneity (ITH) in cancer patients who receive immunotherapy. In order to explore this relationship prospectively with Vigil, a triple function targeted immunotherapy involving ovarian cancer patients in long term follow up of the Phase 2b VITAL trial, we developed an exome sequencing procedure and associated bioinformatics pipeline to determine clonal signal patterns. DNA libraries containing exome sequences tagged with unique molecular identifiers (UMI) were prepared from paired samples and sequenced on Illumina sequencers to high coverage depths of ~ 930X (tumor) and ~ 130X (normal). Raw sequence reads were processed into optimized binary alignment map (BAM) files, using the UMI information. The BAM files were inputted into modules for calling MHC-I alleles, annotating single nucleotide variants (SNVs) and small insertions/deletions (InDels), and for determination of allelic copy number. The outputs were used to predict the sequence of peptide neoantigens and to perform clonality analysis in order to assign each SNV and InDel in a patient tumor sample to a primary clone or subclone. The Clonal Neoantigen pipeline was further assessed using whole exome Illumina sequencing data from three previously published studies. Evaluation of the pipeline using synthetic sequencing data from a sub-clonal deconvolution tool benchmarking study, showed positive predictive value (PPV) and positive percent agreement (PPA) of > 97.5% and > 96.5%, respectively, for SNV and InDel detection with minimum requirements for variant density and allele fraction. Haplotype calls from the Clonal Neoantigen pipeline MHC-I/ MHC-II typing module matched a published benchmark for 91.5% of the calls in a sample of 99 patients. Analysis of exome sequencing data from 14 patients with advanced melanoma revealed a strong correlation between cTMB values determined by the Clonal Neoantigen pipeline as compared to those calculated from the published data (R2 = 0.99). Following validation, the wet lab process and Clonal Neoantigen pipeline was applied to a set of matched normal, tumor, and Vigil product samples from 9 (n = 27 samples) ovarian cancer subjects entered into the VITAL (CL-PTL-119) trial. Results demonstrated marked correlation (R2 = 0.98) of cTMB between tumor used to construct Vigil and Vigil product. Correlation between tumor and Vigil for the cNEO and ITH metrics, showed R2 values of 0.95 and 0.87, respectively. The consistency of the Clonal Neoantigen pipeline results with previously published data as well as the agreement between results for tumor and Vigil for the entire system provide a strong basis of support for utilization of this pipeline for prospective determination of cTMB, cNEO, and ITH values in clinical tumor tissue in order to explore possible correlative relationships with clinical response parameters.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16718d2277420c09078544ae1939c2c13bd37fd2" target='_blank'>
              Exome sequencing shows same pattern of clonal tumor mutational burden, intratumor heterogenicity and clonal neoantigen between autologous tumor and Vigil product
              </a>
            </td>
          <td>
            David Willoughby, E. Bognar, L. Stanbery, Casey Nagel, G. Wallraven, Aman Pruthi, Nicholas Bild, Ericca Stamper, Donald Rao, Adam Walter, John Nemunaitis
          </td>
          <td>2025-03-13</td>
          <td>Scientific Reports</td>
          <td>1</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/553bd748dfc1bbe7f16c0bd9b9403ce389d41277" target='_blank'>
              AI-directed gene fusing prolongs the evolutionary half-life of synthetic gene circuits
              </a>
            </td>
          <td>
            Itamar Menuhin-Gruman, Matan Arbel, Doron Naki, Shaked Bergman, T. Tuller
          </td>
          <td>2025-02-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcd871168cb54fad2bfcadb7ada6ba23bd54105e" target='_blank'>
              The Farm Animal Genotype-Tissue Expression (FarmGTEx) Project.
              </a>
            </td>
          <td>
            L. Fang, J. Teng, Qing Lin, Zhonghao Bai, Shuli Liu, Dailu Guan, Bingjie Li, Yahui Gao, Yali Hou, Mian Gong, Zhangyuan Pan, Ying Yu, Emily L. Clark, Jacqueline Smith, K. Rawlik, Ruidong Xiang, A. Chamberlain, Michael E. Goddard, Mathew Littlejohn, G. Larson, D. MacHugh, John F. O’Grady, Peter Sørensen, G. Sahana, M. Lund, Zhihua Jiang, Xiangchun Pan, Wen Gong, Haihan Zhang, Xi He, Yuebo Zhang, Ning Gao, Jun He, G. Yi, Yuwen Liu, Zhonglin Tang, P. Zhao, Yang Zhou, Liangliang Fu, Xiao Wang, Dan Hao, Lei Liu, Siqian Chen, Robert S. Young, Xia Shen, Charley Xia, Hao Cheng, Li Ma, John B. Cole, R. Baldwin, Cong-Jun Li, C. P. Van Tassell, Benjamin D Rosen, Nayan Bhowmik, Joan Lunney, Wansheng Liu, Leluo Guan, Xin Zhao, E. Ibeagha-Awemu, Yonglun Luo, Lin Lin, O. Canela‐Xandri, M. Derks, R. Crooijmans, M. Gòdia, Ole Madsen, MAM. Groenen, J. Koltes, Christopher K Tuggle, Fiona M. McCarthy, Dominique Rocha, Elisabetta Giuffra, M. Amills, A. Clop, Maria Ballester, G. Tosser-Klopp, Jing Li, Chao Fang, Ming Fang, Qishan Wang, Zhuocheng Hou, Qin Wang, Fuping Zhao, Lin Jiang, Guiping Zhao, Zhengkui Zhou, Rong Zhou, He-he Liu, Juan Deng, Long Jin, Mingzhou Li, Delin Mo, Xiaohong Liu, Yaosheng Chen, Xiaolong Yuan, Jiaqi Li, Shuhong Zhao, Yi Zhang, Xiangdong Ding, D. Sun, Hui-Zeng Sun, Cong-Jun Li, Yu Wang, Yu Jiang, Dongdong Wu, Wenwen Wang, Xinzhong Fan, Qin Zhang, Kui Li, Hao Zhang, N. Yang, Xiaoxiang Hu, Wen Huang, Jiuzhou Song, Yang Wu, Jian Yang, Weiwei Wu, C. Kasper, Xinfeng Liu, Xiaofei Yu, Leilei Cui, Xiang Zhou, Seyoung Kim, Wei Li, Hae Kyung Im, E. Buckler, Bing Ren, Michael C. Schatz, Jingyi Jessica Li, A. Palmer, L. Frantz, Huaijun Zhou, Zhe Zhang, George E. Liu
          </td>
          <td>2025-03-17</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="The phenotype of a mutation often differs across genetically distinct individuals. In the most extreme case, a gene can be essential for viability in one genetic background, but dispensable in another. Although genetic context-dependency of mutant phenotypes is frequently observed, the underlying causes often remain elusive. Here, we investigated the genetic changes responsible for differences in gene essentiality across 18 genetically diverse natural yeast strains. First, we identified 39 genes that were essential in the laboratory reference strain but not required for viability in at least one other genetic background, suggesting that the natural strain contained suppressor variants that could bypass the need for the essential gene. We then mapped and validated the causal bypass suppressor variants using bulk segregant analysis and allele replacements. Bypass suppression was generally driven by a single modifier gene that tended to differ between genetic backgrounds. The suppressors often indirectly counteracted the effect of deleting the essential gene, for instance by changing the transcriptome of a cell. Context-dependent essential genes and their bypass suppressors were frequently co-mutated across 1,011 yeast isolates and identified naturally occurring evolutionary trajectories. Overall, our results highlight the relatively high frequency of bypass suppression in natural populations, as well as the underlying variants and mechanisms. A thorough understanding of the causes of genetic background effects is crucial for the interpretation of genotype-to-phenotype relationships, including those associated with human disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3807dbb2ff664332cc4f31ac739a72b593b98e5b" target='_blank'>
              The modifiers that cause changes in gene essentiality
              </a>
            </td>
          <td>
            Amandine Batté, Núria Bosch-Guiteras, Carles Pons, Marina Ota, Maykel Lopes, Sushma Sharma, Nicolò Tellini, Claire Paltenghi, Michelle Conti, Kwan Ting Kan, Uyen Linh Ho, Michaël Wiederkehr, Jonas Barraud, Mark Ashe, Patrick Aloy, Gianni Liti, A. Chabes, Leopold Parts, Jolanda van Leeuwen
          </td>
          <td>2025-03-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Human genome sequencing efforts in healthy and diseased individuals continue to identify a broad spectrum of genetic variants associated with predisposition, progression, and therapeutic outcomes for diseases like cancer1–6. Insights derived from these studies have significant potential to guide clinical diagnoses and treatment decisions; however, the relative importance and functional impact of most genetic variants remain poorly understood. Precision genome editing technologies like base and prime editing can be used to systematically engineer and interrogate diverse types of endogenous genetic variants in their native context7–9. We and others have recently developed and applied scalable sensor-based screening approaches to engineer and measure the phenotypes produced by thousands of endogenous mutations in vitro10–12. However, the impact of most genetic variants in the physiological in vivo setting, including contextual differences depending on the tissue or microenvironment, remains unexplored. Here, we integrate new cross-species base editing sensor libraries with syngeneic cancer mouse models to develop a multiplexed in vivo platform for systematic functional analysis of endogenous genetic variants in primary and disseminated malignancies. We used this platform to screen 13,840 guide RNAs designed to engineer 7,783 human cancer-associated mutations mapping to 489 endogenous protein-coding genes, allowing us to construct a rich compendium of putative functional interactions between genes, mutations, and physiological contexts. Our findings suggest that the physiological in vivo environment and cellular organotropism are important contextual determinants of specific gene-variant phenotypes. We also show that many mutations and their in vivo effects fail to be detected with standard CRISPR-Cas9 nuclease approaches and often produce discordant phenotypes, potentially due to site-specific amino acid selection- or separation-of-function mechanisms. This versatile platform could be deployed to investigate how genetic variation impacts diverse in vivo phenotypes associated with cancer and other genetic diseases, as well as identify new potential therapeutic avenues to treat human disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2f71f3527a2a35f380d7482a2555f7e6eeef4ab" target='_blank'>
              Multiplexed in vivo base editing identifies functional gene-variant-context interactions
              </a>
            </td>
          <td>
            Jonuelle Acosta, Grace A. Johnson, Samuel I. Gould, Kexin Dong, Yovel Lendner, Diego Detrés, Ondine Atwa, Jari Bulkens, Samuel Gruber, Manuel E. Contreras, Alexandra N. Wuest, Varun K. Narendra, Michael T. Hemann, Francisco J. Sánchez-Rivera
          </td>
          <td>2025-02-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Background Understanding the intricate tumor microenvironment (TME) is crucial for elucidating the mechanisms underlying the progression of cervical squamous cell carcinoma (CSCC) and its response to anti-PD-1 therapy. Methods In this study, we characterized 50,649 cells obtained from the CSCC for single-cell RNA sequencing and integrated bulk sequencing data from The Cancer Genome Atlas (TCGA) and clinical samples to explore their cell composition, metabolic processes, signaling pathways, specific transcription factors, lineage tracking and response to immunotherapy. In vivo experiments were performed to validate the function of key cell subsets. Results We identified ten major cell type and 35 subsets of stromal and immune cells in TME and observed distinct patterns in the metabolic processes and signaling pathways of these cells between tumor and normal tissues. Furthermore, PCNA clamp-associated factor (PCLAF)+ tumor-associated epithelial cell (TAEpis) was negatively correlated with the number of C-X-C motif chemokine ligand 13 (CXCL13)+ CD8+ T cells, overall survival, and response to anti-programmed cell death-1(PD-1) therapy in patients with CSCC. Both in vivo and in vitro experiments demonstrated that PCLAF+ TAEpis promotes the apoptosis of CD8+ T and tumor growth, while also inhibiting T cell infiltration and function. Conclusion Our findings illuminate the heterogeneity of the complex TME in CSCC and offer evidence supporting PCLAF+ TAEpis as a promising therapeutic target. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-03725-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c01c23a0f55159d7cf4e2578f437c46b0c10e1b" target='_blank'>
              Single-cell transcriptomic analyses reveal heterogeneity and key subsets associated with survival and response to PD-1 blockade in cervical squamous cell carcinoma
              </a>
            </td>
          <td>
            Xia Li, Zhao Zhao, Yanmei Cheng, Jiaqin Yan, Fang Ren, Yanyan Jia, Juanhua Li, Binhui Wang, Junqi Liu, Chenyin Wang, Meimei Gao, Hao Gu, Mingliang Fan, Huirong Shi, M. Ji, Qitai Zhao
          </td>
          <td>2025-03-13</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38d11642e20fef657158a832ae45b3c22aa7b26f" target='_blank'>
              Abstract B025: Integrative clinical and molecular analysis of 665 next-generation in vitro cancer models generated by the the Human Cancer Models Initiative (HCMI) for advancing precision medicine and functional drug discovery
              </a>
            </td>
          <td>
            Dina Elharouni, Mushriq Al-Jazrawe, Seongmin Choi, Merve Dede, T. Hinoue, Sean A Misek, Heeju Noh, Luca Zanella, Moony Tseng, Hayley E. Francies, Priya Sridevi, Rachana Agarwal, Cindy W Kyi, Julyann Pérez-Mayoral, Megan J Stine, Eva Y. Tonsing-Carter, James M Clinton, The Hcmi Network, Peter W. Laird, Calvin J Kuo, O. Elemento, David L. Spector, A. Cherniack, Kyle Ellrott, M. Ferguson, R. Beroukhim, Katie Hoadley, N. Robine, Mathew Garnett, Andrea Califano, P. Spellman, D. Tuveson, Keith L. Ligon, Daniela S Gerhard, Louis M Staudt, Jesse S Boehm
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>148</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a6ca066f0de020212e2771b80c63997af778abe" target='_blank'>
              Abstract A044: Immune profiling tools for adult and pediatric cancer: From whole genome and transcriptome sequencing to immunotherapy clinical trial design
              </a>
            </td>
          <td>
            Yaoqing Shen, Chelsea Mayoh, Kathleen Wee, E. Pleasance, E. Titmuss, R. Corbett, L. Williamson, Zakhar Krekhno, A. Nabbi, Raoul Santiago, Stéphanie Bianco, Scott Davidson, A. Shlien, K. Yuki, Denise Connolly, Daniel A. Morgenstern, M. Bonakdar, Greg Taylor, V. Csizmok, C. Grisdale, Melissa McConechy, Jing Xu, John H. Dupuis, K. Mungall, E. Chuah, A. Mungall, J. Nelson, Stephen Yip, Sophie Sun, Howard Lim, D. Renouf, J. Laskin, S. Cohen-Gogo, S. Rassekh, R. Deyell, M. Marra, P. Ekert, S. Jones
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>99</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f6c4d310ebc720bef67a3e653d0a6beb661d459" target='_blank'>
              Historical Evolution and CRISPR-Mediated Gene Editing Applications in NK Cell Therapy
              </a>
            </td>
          <td>
            Ziqing Yan
          </td>
          <td>2025-02-21</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="ABSTRACT Background In B‐cell precursor acute lymphoblastic leukemia (BCP‐ALL), chromosomal translocations are strongly associated with prognoses. RNA sequencing (RNA‐seq) is a powerful technology that reveals a close correlation between types of translocation and patterns of gene expression in clinical samples of BCP‐ALL. Cancer cell lines are powerful research tools, and thus, we built a larger series of BCP‐ALL cell lines and performed RNA‐seq analysis to confirm their utility as a model system. Methods We performed RNA‐seq in a total of 94 BCP‐ALL cell lines, including 80 cell lines with 8 representative types of translocations. Results In the UMAP visualization, a close association was confirmed between the types of fusion genes and patterns of gene expression. In the cluster analysis of the gene expression profile, each type of fusion gene showed a clear association with the expression profile in the top 51 variable genes. Of clinical importance, the majority of the top variable genes in the BCP‐ALL cell lines also showed a significant association with the types of fusion genes in the clinical samples. When an association of 125 cell cycle‐related genes with the percentage of S and G2/M phases in 67 cell lines was evaluated, a significant positive correlation with cell cycle progression was confirmed in 10 cell cycle‐related genes (HDAC2, CDC23, YWHAG, MAD2L1, CCNH, ANAPC7, CDC6, ANAPC5, ORC3, andRBX1). Moreover, significant upregulation and downregulation of 40 and 10 genes, respectively, were observed in the cell lines established at relapse compared with those established at diagnosis. Four (SP6, CCNE1, HIST1H2BH, and DECR2) and two (EVI2B and SYN1) of these genes were also significantly higher and lower, respectively, in the clinical samples at relapse than in those at diagnosis. Conclusion Large series of BCP‐ALL cell lines is a powerful research tool for studying the mechanisms of leukemogenesis and the disease progression of BCP‐ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c14bf4771e63afa160dc9961de41a9da67d0a47d" target='_blank'>
              Utility of a Large Series of B‐Cell Precursor Acute Lymphoblastic Leukemia Cell Lines as a Model System
              </a>
            </td>
          <td>
            Minori Tamai, Chiaki Komatsu, K. Kagami, S. Kasai, K. Akahane, K. Goi, Kanji Sugita, C. Tomoyasu, Toshihiko Imamura, Hiroaki Goto, Takeshi Inukai
          </td>
          <td>2025-03-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Whole genome doubling (WGD) is a frequent event in cancer evolution associated with chromosomal instability, metastasis, and poor prognosis. While the genomic consequences of WGD are well documented, the effects of non-genetic alterations that accompany WGD, such as changes to cell and nuclear size, on tetraploid (4N) cancer cell physiology are less understood. Here, we show that cell and nuclear volume do not always scale with DNA content after WGD in cancer cells, resulting in 4N cells that differ in size. We find that small size is associated with enhanced cell fitness, mitotic fidelity, and tumorigenicity in 4N cancer cells and with poor patient survival in WGD-positive human cancers. Overall, these results suggest that cell and nuclear size contribute to the tumorigenic potential of 4N cancer cells and could be an important prognostic marker in human tumors that undergo WGD. Statement of Significance We report that WGD generates tetraploid cancer cells that vary in size, with larger cells displaying high chromosomal instability and smaller cells exhibiting high fitness and tumorigenicity. Furthermore, WGD status and cancer cell nuclear size in human tumors correlated with patient survival, demonstrating the clinical relevance of this association.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49f68f3dcdb4dd414fdeb5e81fa097ba1603de66" target='_blank'>
              Cell and nuclear size are associated with chromosomal instability and tumorigenicity in cancer cells that undergo whole genome doubling
              </a>
            </td>
          <td>
            Mathew Bloomfield, Sydney Huth, Daniella McCausland, Ron Saad, Nazia Bano, Tran Chau, Megan Sweet, N. Baudoin, Andrew McCaffrey, Kai Fluet, Eva M. Schmelz, Uri Ben-David, Daniela Cimini
          </td>
          <td>2025-04-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/243ddfb9d9bb3f9a966b17fe378947695a450d86" target='_blank'>
              Chromosomal instability in tumor suppressor genes–altered prostate cancer.
              </a>
            </td>
          <td>
            M. Garcia de Herreros, Natalia Jiménez, J. Padrosa, Òscar Reig, L. Fernández-Mañas, L. Ferrer-Mileo, S. García-Esteve, A. Font Pous, V. Ruiz de Porras, Begoña Mellado
          </td>
          <td>2025-02-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The landscapes of somatic mutation in normal cells inform on the processes of mutation and selection operative throughout life, permitting insight into normal ageing and the earliest stages of cancer development. Here, by whole-genome sequencing of 238 microdissections from 30 individuals, including 18 with gastric cancer, we elucidate the developmental trajectories of normal and malignant gastric epithelium. We find that gastric glands are units of monoclonal cell populations which accrue ∼28 somatic single nucleotide variants per year, predominantly attributable to endogenous mutational processes. In individuals with gastric cancer, glands often show elevated mutation burdens due to acceleration of mutational processes linked to proliferation and oxidative damage. These hypermutant glands were primarily detected in the gastric antrum and were mostly associated with chronic inflammation and intestinal metaplasia, known cancer risk factors. Unusually for normal cells, gastric epithelial cells often carry recurrent trisomies of specific chromosomes, which are highly enriched in a subset of individuals. Surveying approximately 8,000 gastric glands by targeted sequencing, we found somatic driver mutations in a distinctive repertoire of known cancer genes, including ARID1A, CTNNB1, KDM6A and ARID1B. Their prevalence increases with age to occupy approximately 5% of the gastric epithelial lining by age 60 years. Our findings provide insights into the intrinsic and extrinsic influences on somatic evolution in the gastric epithelium, in healthy, precancerous and malignant states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a80de4630550f795baa857c6ee76f35822aa4bc" target='_blank'>
              The somatic mutation landscape of normal gastric epithelium
              </a>
            </td>
          <td>
            Tim H. H. Coorens, G. Collord, Hyungchul Jung, Yichen Wang, L. Moore, Y. Hooks, Krishnaa Mahbubani, K. Saeb-Parsy, P. J. Campbell, I. Martincorena, Suet Yi Leung, Michael R Stratton
          </td>
          <td>2024-03-19</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>88</td>
        </tr>

        <tr id="Cancer cell lines are valuable models for studying tumor biology, yet their genomic evolution during culture can compromise experimental reproducibility. We conducted a detailed genomic analysis of the triple-negative breast cancer cell line MDA-MB-231, examining sublines obtained from different sources, at various time points, and across distinct passages. We introduce the concept of intraline heterogeneity (ILH) to highlight the genomic variability observed among these sublines. Our analyses revealed extensive genomic diversity, including differences in single nucleotide variants (SNVs) and copy number alterations (CNAs). In particular, CNAs exhibited remarkable heterogeneity, with pronounced chromosomal gains and losses between sublines, underscoring the impact of genomic instability on ILH. These findings suggest that ILH may influence experimental outcomes, emphasizing the importance of considering passage-specific genomic characterization to ensure consistency and reliability in cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c7a90a295650f938dfb30675bf97324372367f9" target='_blank'>
              Intraline genomic heterogeneity of the triple-negative breast cancer MDA-MB-231 cell line
              </a>
            </td>
          <td>
            Nair Varela-Rouco, N. Estévez-Gómez, C. Fernández-Santiago, Laura Tomás, Miriam Pérez, Daniel García-Souto, J. Pasantes, R. Piñeiro, J. M. Alves, D. Posada
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Epigenetics is currently considered the investigation of inheritable changes in gene expression that do not rely on DNA sequence alteration. Significant epigenetic procedures are involved, such as DNA methylations, histone modifications, and non-coding RNA actions. It is confirmed through several investigations that epigenetic changes are associated with the formation, development, and metastasis of various cancers, such as colorectal cancer (CRC). The difference between epigenetic changes and genetic mutations is that the former could be reversed or prevented; therefore, cancer treatment and prevention could be achieved by restoring abnormal epigenetic events within the neoplastic cells. These treatments, consequently, cause the anti-tumour effects augmentation, drug resistance reduction, and host immune response stimulation. In this article, we begin our survey by exploring basic epigenetic mechanisms to understand epigenetic tools and strategies for treating colorectal cancer in monotherapy and combination with chemotherapy or immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/719dc00daf58fbc383533083eb81aecec4828ea4" target='_blank'>
              Epigenetic marvels: exploring the landscape of colorectal cancer treatment through cutting-edge epigenetic-based drug strategies
              </a>
            </td>
          <td>
            A. Tahghighi, Effat Seyedhashemi, Javad Mohammadi, Arash Moradi, Aria Esmaeili, Majid Pornour, Kimia Jafarifar, S. M. Ganji
          </td>
          <td>2025-02-22</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f39bd0ddc99a69058c299e72e0d1b4296584ccd0" target='_blank'>
              Therapeutic targeting of chromatin alterations in leukemia and solid tumors.
              </a>
            </td>
          <td>
            Florian Perner, Tobias Berg, D. Sasca, Sophie-Luise Mersiowsky, Jayant Y. Gadrey, Johanna Thomas, Michael W M Kühn, Michael Lübbert
          </td>
          <td>2025-04-03</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b338d775526840d18d525abb3d293c825f24385" target='_blank'>
              Abstract B011: Harnessing epigenetic and metabolic biomarkers for targeted immunotherapy in colorectal cancer: Advances in precision medicine for solid tumors
              </a>
            </td>
          <td>
            Ji Min Lee
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Diabetes mellitus (DM) is a global health crisis affecting millions, with islet transplantation emerging as a promising treatment strategy to restore insulin production. This review synthesizes the current research on single-cell and spatial transcriptomics in the context of islet transplantation, highlighting their potential to revolutionize DM management. Single-cell RNA sequencing, offers a detailed look into the diversity and functionality within islet grafts, identifying specific cell types and states that influence graft acceptance and function. Spatial transcriptomics complements this by mapping gene expression within the tissue’s spatial context, crucial for understanding the microenvironment surrounding transplanted islets and their interactions with host tissues. The integration of these technologies offers a comprehensive view of cellular interactions and microenvironments, elucidating mechanisms underlying islet function, survival, and rejection. This understanding is instrumental in developing targeted therapies to enhance graft performance and patient outcomes. The review emphasizes the significance of these research avenues in informing clinical practices and improving outcomes for patients with DM through more effective islet transplantation strategies. Future research directions include the application of these technologies in personalized medicine, developmental biology, and regenerative medicine, with the potential to predict disease progression and treatment responses. Addressing ethical and technical challenges will be crucial for the successful implementation of these integrated approaches in research and clinical practice, ultimately enhancing our ability to manage DM and improve patient quality of life.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4d2a871c9156c1f2749ccafc8536877666a5a4e" target='_blank'>
              Single-cell genomics and spatial transcriptomics in islet transplantation for diabetes treatment: advancing towards personalized therapies
              </a>
            </td>
          <td>
            Lisha Mou, Tony Bowei Wang, Yuxian Chen, Ziqi Luo, Xinyu Wang, Zuhui Pu
          </td>
          <td>2025-02-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Malignant pleural mesothelioma (MPM) is a rare yet highly aggressive malignancy with a severe prognosis. Compounded by the lack of effective treatment modalities, MPM remains a formidable health challenge. Therefore, the identification of actionable liabilities is critical for advancing precision medicine to combat this lethal disease. Here, we exploit an unbiased genome-wide CRISPR screen, integrating and cross-comparing three MPM cell lines with nonmalignant mesothelial cells, to selectively map the gene targets whose depletion indicates a common dependency in MPM cells. This systematic approach unveils a cohort of verifiable genes, among which BUB1, a mitotic checkpoint serine/threonine kinase, emerges as a high-confidence hit in cancer cells. Cellular and molecular studies demonstrate that genetic depletion or pharmacological inhibition of BUB1 profoundly impairs MPM cell survival and growth while inducing G2/M cell cycle arrest, cellular senescence, and apoptosis, and attenuating functional hallmarks of aggressive cancer cells. Transcriptomic profiling of BUB1-depleted cells discloses differential gene expression signatures congruent with cell fate phenotypes, including the reprogramming of mitotic network genes. Mechanistically, BUB1 is indispensable for the proper localization of essential mitotic regulators MAD1, MAD2, and Shugoshin (SGO1), thereby ensuring the functionality of the spindle assembly checkpoint (SAC). Furthermore, BUB1 ablation leads to cytokinesis failure and multinucleation, a phenotype characterized by the downregulation of CDC20, Cyclin A, and Cyclin B, and a reciprocal upregulation of the cyclin-dependent kinase inhibitor p21. Clinically, MPM tumors exhibit elevated levels of BUB1, and high BUB1 expression is associated with shorter patient survival. Our novel findings accentuate comparative CRISPR screens as a powerful platform to explore tumor cell-selective gene essentiality and propose BUB1 kinase as a potential marker and druggable vulnerability with therapeutic implications for MPM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/147c280cc9a37e6a39ce87d2c82cecf0b12eda26" target='_blank'>
              Genome-wide CRISPR screen identifies BUB1 kinase as a druggable vulnerability in malignant pleural mesothelioma
              </a>
            </td>
          <td>
            E. Cakiroglu, Sude Eris, Ozden Oz, G. Karakülah, Serif Senturk
          </td>
          <td>2025-04-03</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Abstract Single-cell combinatorial indexing (sci) methods have addressed major limitations of throughput and cost for many single-cell modalities. With the incorporation of linear amplification and three-level barcoding in our suite of methods called sci-L3, we further addressed the limitations of uniformity in single-cell genome amplification. Here, we build on the generalizability of sci-L3 by extending it to template strand sequencing (sci-L3-Strand-seq), genome conformation capture (sci-L3-Hi-C), and the joint profiling of RNA and chromatin accessibility (sci-L3-RNA/ATAC). We demonstrate the ease of adapting sci-L3 to these new modalities by only requiring a single-step modification of the original protocol. As a proof of principle, we show our ability to detect sister chromatid exchanges, genome compartmentalization, and cell state-specific features in thousands of single cells. We anticipate sci-L3 to be compatible with additional modalities, including DNA methylation (sci-MET) and chromatin-associated factors (CUT&Tag), and ultimately enable a multi-omics readout of them.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/504a5342be4c37a94d1bc91a8c3fcdf178c63541" target='_blank'>
              Generalization of the sci-L3 method to achieve high-throughput linear amplification for replication template strand sequencing, genome conformation capture, and the joint profiling of RNA and chromatin accessibility
              </a>
            </td>
          <td>
            Peter Chovanec, Yi Yin
          </td>
          <td>2025-02-08</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5883dfe71ffdf910e9c20145e8db93ca834ac43" target='_blank'>
              In vivo expansion of gene-targeted hepatocytes through transient inhibition of an essential gene.
              </a>
            </td>
          <td>
            M. De Giorgi, So Hyun Park, Adam B. Castoreno, Mingming Cao, Ayrea Hurley, Lavanya Saxena, Marcel A Chuecos, Christopher J Walkey, Alexandria M. Doerfler, Mia N Furgurson, M. C. Ljungberg, Kalyani R Patel, Sarah Hyde, Tyler Chickering, Stephanie Lefebvre, Kelly Wassarman, Patrick Miller, June Qin, M. Schlegel, Ivan Zlatev, Jun Han, C. Beeton, R. Li, J. Kim, James F. Martin, K. Bissig, Vasant Jadhav, Gang Bao, William R. Lagor
          </td>
          <td>2025-02-12</td>
          <td>Science translational medicine</td>
          <td>1</td>
          <td>36</td>
        </tr>

        <tr id="Colon cancer (CC) is a common malignancy with rising incidence worldwide. Despite advances in treatment strategies, many patients still face a poor prognosis due to the development of drug resistance. Long non-coding RNAs (lncRNAs) have emerged as important regulators of various biological processes and have been implicated in cancer progression. Among them, colorectal neoplasia differentially expressed (CRNDE) has drawn attention for its potential roles in different cancers. However, its specific functions in CC remain unclear. In this study, we identified CRNDE as highly expressed in CC, contributing to tumor progression and drug resistance. Mechanically, CRNDE is regulated by the transcription factor TFAP2A. Additionally, CRNDE inhibits pyroptosis, a form of programmed cell death, by promoting the ubiquitin-mediated degradation of RIPK3, thereby reducing the sensitivity of CC cells to 5-fluorouracil (5-FU). Our findings suggest that the TFAP2A/CRNDE/RIPK3 axis plays critical roles in colon cancer progression and chemoresistance, highlighting potential therapeutic targets for improving treatment outcomes. Supplementary Information The online version contains supplementary material available at 10.1007/s10142-025-01545-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f20bcfbeb6577568a7ff02bdc23116269869d367" target='_blank'>
              TFAP2A-regulated CRNDE enhances colon cancer progression and chemoresistance via RIPK3 interaction
              </a>
            </td>
          <td>
            Xin Gao, Yanming Huang, Tonghui Wei, Jingmin Xue, Filippov Iurii, Laishou Yang, Liying Wang, Hao Li, Genshen Mo, Yuze Huang, Haonan Xie, Hang Wang, Shenghan Lou, Peng Han
          </td>
          <td>2025-02-15</td>
          <td>Functional & Integrative Genomics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d82c852aa82f168fc01790167512bfdf3a4c0023" target='_blank'>
              RNA-sequencing: A reliable tool to unveil transcriptional landscape of paediatric B-other acute lymphoblastic leukaemia.
              </a>
            </td>
          <td>
            Clara Vicente-Garcés, Guerau Fernández, Elena Esperanza-Cebollada, Mercè Richarte-Franqués, Alba Crespo-Carrasco, Sara Montesdeoca, Ignacio Isola, Edurne Sarrate, Esther Cuatrecasas, Susana Rives, J. Dapena, Mireia Camós, Nerea Vega-García
          </td>
          <td>2025-03-30</td>
          <td>British journal of haematology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8599e8314fc9026e2db49b39f76e5203a6d7080" target='_blank'>
              Allele-specific genomics decodes gene targets and mechanisms of the non-coding genome
              </a>
            </td>
          <td>
            Tim P. Hasenbein, Sarah Hoelzl, S. Engelhardt, Daniel Andergassen
          </td>
          <td>2025-03-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The use of CRISPR to knockdown or knockout genes is a powerful tool for understanding the specific role of a gene in disease development. However, it can cause many unanticipated changes to the transcriptome that are not detected by DNA amplification and Sanger sequencing of the target site. Various RNA-sequencing techniques can be used to identify these changes and effectively gauge the full impact of the CRISPR knockout, thereby providing a means of selecting appropriate clones for further experimentation. Background/Objectives: RNA-seq data from 4 CRISPR knockout experiments were analyzed and techniques developed to both confirm the success of the CRISPR modifications and identify potential issues. Methods: A broad-based analysis of RNA-sequencing data identified many CRISPR-based changes not identified by PCR amplification of DNA around the CRISPR target site. These changes included an inter-chromosomal fusion event, exon skipping, chromosomal truncation, and the unintentional transcriptional modification and amplification of a neighboring gene. Conclusions: The inadvertent modifications identified by the evaluation of 4 CRISPR experiments highlight the value of using RNA-seq to identify transcriptional changes to cells altered by CRISPR, many of which cannot be recognized by evaluating DNA alone. Specific guidelines are presented for designing and analyzing CRISPR experiments using RNA-seq data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/220a9a7ebf2665e1a3e4bd65d2ca79919ea760ce" target='_blank'>
              Techniques for Validating CRISPR Changes Using RNA-Sequencing Data
              </a>
            </td>
          <td>
            S. Rathe, Tracy A. Marko, Elizabeth N. Edwards, Paige Hazelton Ridder, Jyotika Varshney, Kyle B. Williams, James E. Johnson, B. Moriarity, D. A. Largaespada
          </td>
          <td>2025-03-24</td>
          <td>Genes</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e707dd4d2590e8deca1a1e653d9b70d94049240c" target='_blank'>
              Suppress the proliferation process of glioblastoma by breaking off the tumor - microenvironment interaction by CRIPSR -mediated technology
              </a>
            </td>
          <td>
            Wei Wang
          </td>
          <td>2025-02-27</td>
          <td>MedScien</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Small-cell lung cancer (SCLC) is a highly aggressive malignancy characterized by rapid proliferation, early metastasis, and frequent recurrence, which contribute to a poor prognosis. SCLC is defined by the near-universal inactivation of key tumor suppressor genes, notably TP53 and RB1, which play central roles in its pathogenesis and resistance to therapy. The p53 family of proteins, including p53, p63, and p73, is essential to maintaining cellular homeostasis and tumor suppression. TP53 mutations are almost ubiquitous in SCLC, leading to dysregulated apoptosis and cell cycle control. Moreover, p73 shows potential as a compensatory mechanism for p53 loss, while p63 has a minimal role in this cancer type. In this review, we explore the molecular and functional interplay of the p53 family in SCLC, emphasizing its members’ distinct yet interconnected roles in tumor suppression, immune modulation, and therapy resistance. We highlight emerging therapeutic strategies targeting these pathways, including reactivating mutant p53, exploiting synthetic lethality, and addressing immune evasion mechanisms. Furthermore, this review underscores the urgent need for novel, isoform-specific interventions to enhance treatment efficacy and improve patient outcomes in this challenging disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc096d40431e9f2c1e7043ab1a608082051ec709" target='_blank'>
              Recent Research on Role of p53 Family in Small-Cell Lung Cancer
              </a>
            </td>
          <td>
            Minho Jeong, Kee-Beom Kim
          </td>
          <td>2025-03-26</td>
          <td>Cancers</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Genes and genetic variants associated with human disease are continually being discovered, but validating their causative roles and mechanisms remains a significant challenge. CRISPR/Cas9 genome editing in model organisms like zebrafish can enable phenotypic characterization of founder generation (F0) knockouts (Crispants), but existing approaches are not amenable to high-throughput genetic screening due to high variability, cost, and low phenotype penetrance. To overcome these challenges, here we provide guide RNA (gRNA) selection rules that enable high phenotypic penetrance of up to three simultaneous knockouts in F0 animals following injection of 1–2 gRNAs per gene. We demonstrate a strong transcriptomic overlap in our F0 knockouts and stable knockout lines that take several months to generate. We systematically evaluated this approach across 324 gRNAs targeting 125 genes and demonstrated its utility in studying epistasis, characterizing paralogous genes, and validating human disease gene phenotypes across multiple tissues. Applying our approach in a high-throughput manner, we screened and identified 10 novel neurodevelopmental disorders and 50 hearing genes not previously studied in zebrafish. Altogether, our approach achieves high phenotypic penetrance using low numbers of gRNAs per gene in F0 zebrafish, offering a robust pipeline for rapidly characterizing candidate human disease genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d442e32931385bdcd94b5a33eaea8e6b64c86c8" target='_blank'>
              Optimizing gRNA selection for high-penetrance F0 CRISPR screening for interrogating disease gene function
              </a>
            </td>
          <td>
            Sheng-Jia Lin, Kevin Huang, C. Petree, Wei Qin, Pratishtha Varshney, Gaurav K Varshney
          </td>
          <td>2025-02-27</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Recently revised brain tumor classification suggested a glioma treatment strategy that takes into consideration molecular variants in IDH1 and TP53 marker genes. While pathogenic variants of IDH1 and TP53 can be accompanied by chromosomal instability (CIN), the impact of IDH1 and TP53 mutations on genome stability remains unstudied. Elevated CIN might provide therapeutic targets, based on synergistic effects of chemotherapy with CIN-inducing drugs. Using an assay based on human artificial chromosomes, we investigated the impact of common glioma missense mutations in IDH1 and TP53 on chromosome transmission and demonstrated that IDH1R132H and TP53R248Q variants elevate CIN. We next found enhanced CIN levels and the sensitivity of IDH1R132H/WT and TP53R248Q/R248Q genotypes, introduced into U87 MG glioma cells by CRISPR/Cas9, to different drugs, including conventional temozolomide. It was found that U87 MG cells carrying IDH1R132H/WT exhibit dramatic sensitivity to paclitaxel, which was independently confirmed on cell cultures derived from patients with naturally occurring IDH1R132H/WT. Overall, our results suggest that the development of CIN-enhancing therapy for glioma tumors with the IDH1R132H/WT genotype could be advantageous for adjuvant treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/641df7ee46092e60a05501256dd4211c20f30f9f" target='_blank'>
              Therapy enhancing chromosome instability may be advantageous for IDH1R132H/WT gliomas
              </a>
            </td>
          <td>
            Nikolay V Goncharov, Ivan Baklanov, Valeriia S Gulaia, Anastasiia P Shuliak, Daria V Lanskikh, Valeriia Zhmenia, M.E. Shmelev, N. Shved, Jing Wu, Mikhail Liskovykh, Vladimir Larionov, N. Kouprina, Vadim Kumeiko
          </td>
          <td>2025-02-13</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Chronic lymphocytic leukemia (CLL) is characterized by the accumulation of monoclonal mature B lymphocytes in peripheral blood, bone marrow, lymphoid tissues, and extranodal sites. Genes involved in RNA splicing such as SF3B1 and U1 are frequently mutated in CLL, leading to altered splicing and generation of tumor neoepitopes. To study the impact of these mutations on the tumor microenvironment (TME), we have developed a comprehensive single-cell atlas of unmutated CLL encompassing 26 bone marrow and lymph node tumor samples from 23 U-CLL patients with mutations in U1 (n=7), SF3B1 (n=8), or without mutations in splicing genes (n=10). We observed high intra-tumor heterogeneity, discerning 12 transcriptional programs, one linked to the U1 g.A3>C mutation and characterized by NFKB hyperactivation. T cell and NK compartments exhibited site- and mutation-specific enrichment, with increased CD4+ regulatory cells (Treg) and CD8+ exhausted cells in lymph nodes, while U1-mutant tumors showed increased CD8+ cytotoxic activity, with a predominance of effector-like CD8+ cells. Single-cell T cell receptor sequencing revealed clonotype expansion in U1-mutated tumors, particularly in CD8+ effector and exhausted cells, suggesting a neoantigen-driven immune response. Cell-to-cell interaction analysis identified CD44 as a key mediator in U1-mutated tumors, showing pro-B survival interactions as those involving MIF-CD44-CD74. Furthermore, interactions between CD80 on CLL cells and CTLA4 on Tregs and CD8+ exhausted were upregulated, reflecting an immunosuppressive phenotype associated with U1 mutated CLL. These findings highlight the complex interplay between mutations in CLL and the TME, offering novel avenues for alternative therapeutic strategies for U-CLL with mutations in U1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/338007fb60dcc4b1652c081c06c04390110d3e38" target='_blank'>
              U1 snRNA mutation reshapes tumor microenvironment in chronic lymphocytic leukemia: a role for CD44-mediated signaling
              </a>
            </td>
          <td>
            Sara López-Tamargo, Javier Fernández-Mateos, Pablo Bousquets-Muñoz, Laura Llaó-Cid, F. Nadeu, Ares M. Farran, Cristina Olivar-Fernández, Ana de la Fuente-González, Andrea Aran, Roberto Martínez-Soler, , Manel Juan, J. Martín-Subero, D. Colomer, E. Campo, A. Gutiérrez-Fernández, Xose S Puente
          </td>
          <td>2025-04-01</td>
          <td>None</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="Tongue squamous cell carcinoma (TSCC), a subtype of head and neck squamous cell carcinoma, is characterized by frequent chemoresistance. Genetic mutations commonly observed in TSCC play a critical role in malignant progression; thus, elucidating their functional significance is essential for developing effective treatment strategies. To more accurately investigate the relationship between mutations and chemoresistance, we established low-passage TSCC cells, CTSC-1, obtained from a chemoresistant patient, and CTSC-2, from a treatment-naïve patient. Sanger sequencing revealed a specific TP53 mutation (Q331*) in CTSC-1, leading to the loss of the tetramerization and C-terminal regulatory domains. Notably, CTSC-1 cells harboring TP53-Q331* and CTSC-2 cells with TP53 knockout that have been engineered to ectopically express TP53-Q331* exhibit enhanced chemoresistance and increased cancer stem cell-like properties. Mechanistically, TP53-Q331* upregulates the expression of inhibitor of DNA binding 2 (ID2), which is crucial for maintaining the stemness of TSCC cells. Subsequently, ID2 activates the expression of nucleotide excision repair (NER) pathway-related genes ERCC4 and ERCC8, thereby enhancing the chemoresistance in TSCC. In conclusion, our study demonstrates that the TP53-Q331* mutation enhances TSCC chemoresistance through an ID2-mediated NER pathway, providing a potential prognostic marker and therapeutic target for TSCC chemotherapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9563aecd404844781094bd5fefafcbc9075af8dc" target='_blank'>
              A Truncated Mutation of TP53 Promotes Chemoresistance in Tongue Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            Xiaoyun Du, Junheng Zheng, Xiangwan Lu, Yan Zhang
          </td>
          <td>2025-03-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="PURPOSE It is a clinical challenge to select patients for BRAF-targeted therapy because of the lack of predictive biomarkers besides the BRAF V600E mutation. By analyzing the genome, transcriptome, and proteome, this study investigated the association between baseline molecular alterations and outcomes of BRAF-targeted therapy. PATIENTS AND METHODS Fresh tumor tissue from patients enrolled in the Copenhagen Prospective Personalized Oncology study was collected and underwent comprehensive molecular profiling. RESULTS TP53 comutations were most frequently detected. Patients with a TP53 wild-type tumor had a significantly longer median progression-free survival than those with TP53 comutations (hazard ratio, 2.8 [95% CI, 1.13 to 7.08]; P = .02). RNAseq revealed a distinct gene expression signature for patients with long-term disease control (LDC), including hallmarks of cell cycle arrest and proliferation in the p53 pathway. The protein analysis demonstrated that ubiquitin-conjugating enzyme EK2 was significantly downregulated in patients with LDC. CONCLUSION Using a multiomic approach, we identified molecular alterations associated with treatment outcomes. The potential of analyzing multiomic data is promising and should be prioritized in translational cancer research to uncover the full potential within precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d33023bdab6e4dfcabecc3f50f8af28550770e9" target='_blank'>
              Multiomics Identifies Potential Molecular Profiles Associated With Outcomes After BRAF-Targeted Therapy in Patients With BRAF V600E-Mutated Advanced Solid Tumors
              </a>
            </td>
          <td>
            M. Eriksen, Anne M Hansen, Annelaura B Nielsen, F. Mundt, Matthias Mann, U. Lassen, L. B. Ahlborn, M. Højgaard, I. Spanggaard, Camilla Qvortrup, C. Yde, K. Rohrberg
          </td>
          <td>2025-03-01</td>
          <td>JCO Precision Oncology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Anti-PD-1/PD-L1 immune checkpoint blockade (ICB) therapy has revolutionized clinical cancer treatment, while abnormal PD-L1 or HLA-I expression in patients can significantly impact the therapeutic efficacy. Somatic mutations in cancer cells that modulate these critical regulators are closely associated with tumor progression and ICB response. However, a systematic interpretation of cancer immune-related mutations is still lacking. Here, we harnessed the ABEmax system to establish a large-scale sgRNA library encompassing approximately 820,000 sgRNAs that target all feasible serine/threonine/tyrosine residues across the human genome, which systematically unveiled thousands of novel mutations that decrease or augment PD-L1 or HLA-I expression. Beyond residues associated with phosphorylation events, our screens also identified functional mutations that affect mRNA or protein stability, DNA binding capacity, protein-protein interactions, and enzymatic catalytic activity, leading to either gene inactivation or activation. Notably, we uncovered certain mutations that concurrently modulate PD-L1 and HLA-I expression, represented by the clinically relevant mutation SETD2_Y1666. We demonstrated that this mutation induces consistent phenotypic effects across multiple cancer cell lines and enhances the efficacy of immunotherapy in different tumor models. Our findings provide an unprecedented resource of functional residues that regulate cancer immunosurveillance, offering valuable guidance for clinical diagnosis, ICB therapy, and the development of innovative drugs for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0151a36f604dedfea97d97f1dfa3c6ae2a57eb97" target='_blank'>
              Massively parallel interrogation of human functional variants modulating cancer immunosurveillance
              </a>
            </td>
          <td>
            Ying Liu, Yongshuo Liu, Xuran Niu, Ang Chen, Yizhou Li, Ying Yu, Binrui Mo, Zhiheng Liu, Tao Xu, Jie Cheng, Zeguang Wu, Wensheng Wei
          </td>
          <td>2025-03-19</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 Cancer remains a significant global health challenge, claiming numerous lives annually and posing complex hurdles in management. The disease's multifaceted nature, combined with disparities in healthcare infrastructure and socioeconomic factors worldwide, complicates effective treatment. With alarming mortality rates, there is an urgent need to enhance prevention and early detection strategies. Early detection not only enables less invasive treatments but also improves the chances of successful recovery, highlighting the importance of proactive approaches. Lung cancer ranks highest in mortality rates, followed by breast, prostate, and colon cancers. Cancer is often seen as both a "genetic disease," involving gene mutations, and an "environmental disease," influenced by various environmental factors affecting gene expression. Metabolic alterations are a hallmark of cancer, and disruptions in cellular stability and function significantly contribute to its progression. Non-coding RNAs (ncRNAs) play crucial regulatory roles across cancers, with diverse structures and functions. They act as oncogenes or tumor suppressors depending on the cancer type, intricately modulating genetic and epigenetic processes. Recent studies challenge the perception of ncRNAs' insignificance, revealing their substantial impact on regulatory mechanisms. Indeed, ncRNAs intricately connect genetic networks, influencing vital protein effectors that dictate cellular responses and fate. Consequently, dysregulated ncRNAs play a pivotal role in disease pathogenesis, acting as either oncogenic drivers or tumor suppressors through aberrant expression in various cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e52de2b0fb9cae18b101db41cf959bce17a1986" target='_blank'>
              The role of non-coding RNAs (ncRNAs) and their potential connection with cancer
              </a>
            </td>
          <td>
            Safura Absalan, Hamidreza Vaziri
          </td>
          <td>2025-03-22</td>
          <td>Egyptian Journal of Medical Human Genetics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a genetically and cellularly heterogeneous disease. We characterized 120 AMLs using genomic and transcriptomic analyses, including single-cell RNA sequencing. Our results reveal an extensive cellular heterogeneity that distorts the bulk transcriptomic profiles. Selective examination of the transcriptional signatures of >90,000 immature AML cells identified four main clusters, thereby extending current genomic classification of AML. Notably, NPM1 mutated AML could be stratified into two novel, clinically relevant classes, with NPM1class I associated with downregulation of MHC class II and excellent survival following hematopoietic stem cell transplantation (HSCT). NPM1class II was instead associated with resistance to allogeneic T cells in an ex vivo co culture assay, and importantly, dismal survival following HSCT. These findings provide new insights into the cellular state space of AML, define new diagnostic entities, and highlight potential therapeutic intervention points. Key Points The bulk transcriptional profiles of AML are mainly driven by a diverse set of cellular signatures. Single cell RNA-sequencing of the most common AML subtypes reveals marked heterogeneity extending beyond current genomic classification schemes. NPM1-mutated AML can be divided into two new classes, with distinct immune evasion mechanisms and survival after transplantation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15ce425737814eabb8f9dbfc7144c4c613998394" target='_blank'>
              The cellular state space of AML unveils novel NPM1 subtypes with distinct clinical outcomes and immune evasion properties
              </a>
            </td>
          <td>
            H. Lilljebjörn, Pablo Peña-Martínez, Hanna Thorsson, Rasmus Henningsson, M. Rissler, N. Landberg, Noelia Puente-Moncada, Sofia von Palffy, Vendela Rissler, Petr Stanek, Jonathan Desponds, Xiangfu Zhong, Gunnar Juliusson, Vladimir Lazarevic, Sören Lehmann, Magnus Fontes, H. Ågerstam, Carl Sandén, Christina Orsmark-Pietras, T. Fioretos
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/344c45deee3ee4d6c328b84a6e73872831228859" target='_blank'>
              Abstract B019: Cyclin E1 amplification promotes chromosomal instability and distinct vulnerabilities in gastroesophageal adenocarcinoma
              </a>
            </td>
          <td>
            Zhong Wu, Ankur Nagaraja, Chunyang Bao, Jin Zhou, A. Bass
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Oral cancer is one of the most prevalent malignant neoplasms globally, with its microenvironment being extremely complicated and heterogeneous with regard to distribution of different types of cells, which creates a barrier against treatment. We integrated single-cell RNA sequencing, pseudotime analysis, and intercellular communication to identify subtypes, state transitions, and interactions in the development of oral cancer and their roles in tumor progression and immune response. Methods Principal component analysis (PCA) followed by dimensionality reduction techniques (UMAP and t-SNE) revealed multiple populations in the single-cell data from oral cancer. Analysis of gene expression patterns associated individual genes with distinct principal components that described features of cell subtypes or states in the oral cancer microenvironment. We also conducted gene set enrichment analysis (GSEA) and cell–cell communication network analysis to investigate the important signaling pathways as well as potential cell–cell communications. Results Gene expression patterns linked to oral cancer cell subtypes and states in this study included cell membrane, signal transduction and immune response in different biological processes. Pseudotime analysis showed differentiation trajectories and expression of essential differentiation genes were also altered. Cell–cell communication network analysis revealed that myeloid cells interacted heavily with myeloid cells and they also had strong interactions with endothelial cells. Analysis of the MIF signaling pathway network showed participation of various cell types in MIF signaling, suggesting its potential significance in the oral cancer microenvironment. Conclusion The intercellular communication complexity of TME in oral cancer is reported here with the first-ever single cell analysis and data providing new perspective regarding the heterogeneity of oral cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d8d5845a6bd00754561294af9f1ae51a6774014" target='_blank'>
              Single-cell pseudotime and intercellular communication analysis reveals heterogeneity and immune microenvironment in oral cancer
              </a>
            </td>
          <td>
            Hanjun Liu, Lian Xie, Xuemin Xing, Lili Hou, Jinfeng Zhang, Ling Yang
          </td>
          <td>2025-02-10</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Objective Active surveillance (AS) offers a viable alternative to surgical intervention for the management of indolent papillary thyroid carcinoma (PTC), helping to minimize the incidence of unnecessary treatment. However, the broader adoption of AS is hindered by the need for more reliable diagnostic markers. This study aimed to identify the differences between indolent and progressive PTC and find new targets for biomarker development and therapeutic strategies. Methods We used single-cell RNA sequencing (scRNA-seq) to analyze cellular differences in 10 early-stage PTC tumors. Findings were validated in an additional 25 tumors using cell co-culture, migration assays, immunofluorescence staining, flow cytometry, and analysis of data from The Cancer Genome Atlas (TCGA). Results Tumor-infiltrating B cells (TIL-B), particularly germinal center B cells (GC-B), were more abundant in indolent PTC. These cells suppressed thyroid cell proliferation in both indolent and progressive cases, though indolent PTC had a higher capacity to recruit peripheral B cells. In indolent cases, TIL-B cells showed increased proliferation and formed clusters within tertiary lymphoid structures (TLS). PTPRC-CD22 interactions were identified as potential drivers of TIL-B cell proliferation. Markers linked to GC-B cells, such as LMO2, were highlighted as potential diagnostic and prognostic indicators for indolent PTC. Conclusion This study provides insights into the cellular landscape of early-stage PTC, revealing distinct tumor and immune microenvironment features in indolent and progressive cases. These findings advance the understanding of indolent PTC biology and support the development of reliable diagnostic and prognostic biomarkers. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03341-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c059a59d61bc9cea405825a6d5d7499c3788b8e" target='_blank'>
              Single-cell transcriptomics analysis reveals that the tumor-infiltrating B cells determine the indolent fate of papillary thyroid carcinoma
              </a>
            </td>
          <td>
            Chunmei Li, Pei Wang, Zhizhong Dong, Weihan Cao, Yan-Jun Su, Jian-ming Zhang, Shuyan Zhao, Zhiyuan Wang, Zi Lei, Li Shi, Ruochuan Cheng, Wen Liu
          </td>
          <td>2025-03-11</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Genetic instability is a hallmark of cancer, and mutation hotspots in human cancer genomes co-localize with alternative DNA structure-forming sequences (e.g. H-DNA), implicating them in cancer etiology. H-DNA has been shown to stimulate genetic instability in mammals. Here, we demonstrate a new paradigm of genetic instability, where a cancer-associated H-DNA-forming sequence accumulates more oxidative lesions than B-DNA under conditions of oxidative stress (OS), often found in tumor microenvironments. We show that OS results in destabilization of the H-DNA structure and attenuates the fold increase in H-DNA-induced mutations over control B-DNA in mammalian cells. Furthermore, the mutation spectra revealed that the damaged H-DNA-containing region was processed differently compared to H-DNA in the absence of oxidative damage in mammalian cells. The oxidatively modified H-DNA elicits differential recruitment of DNA repair proteins from both the base excision repair and nucleotide excision repair mechanisms. Altogether, these results suggest a new model of genetic instability whereby H-DNA-forming regions are hotspots for DNA damage in oxidative microenvironments, resulting in its altered mutagenic processing. Our findings provide valuable insights into the role of OS in DNA structure-induced genetic instability and may establish H-DNA-forming sequences as promising genomic biomarkers and potential therapeutic targets for genetic diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/560e3fa66aab00812738293c545080a843274a94" target='_blank'>
              Oxidative damage within alternative DNA structures results in aberrant mutagenic processing
              </a>
            </td>
          <td>
            Maha Zewail-Foote, Imee M A Del Mundo, Alex W. Klattenhoff, Karen M. Vasquez
          </td>
          <td>2025-02-08</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The metabolic enzyme aldehyde dehydrogenase 1A1 (ALDH1A1), a cancer stem cell marker associated with poor outcomes in breast cancer, has emerged as a promising therapeutic target in TNBC. The aim of this study was to investigate the role of ALDH1A1 in radiation resistance and redox stress in triple negative breast cancer (TNBC). Functional knockouts of ALDH1A1 were generated by the CRISPR/Cas9-mediated deletion of ALDH1A1 in the SUM159 cell line, and three distinct clonal populations were isolated. Genetic targeting was confirmed by Sanger sequencing, and the loss of ALDH1A1 protein expression was validated by Western blotting. Functional assays assessed ALDEFLUOR activity, cell viability, self-renewal capacity, and reactive oxygen species (ROS) levels with or without radiation in both the bulk population and clonal lines. Interestingly, ALDEFLUOR activity was uniformly lost across all clonal lines; however, functional effects of ALDH1A1 loss on redox stress, survival, and radiation sensitivity were observed in only one clonal population. These findings highlight significant variability in the role of ALDH1A1 among clonal populations, reflecting the complexity of tumor heterogeneity. This underscores the importance of accounting for tumor heterogeneity when targeting ALDH1A1, as certain TNBC subpopulations may rely more heavily on ALDH1A1 function. These insights are critical for developing effective ALDH1A1-targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce39f8fe1789f12e83b81bf503275e5a974f1d83" target='_blank'>
              CRISPR/Cas9 Targeting of Aldehyde Dehydrogenase 1A1 Reveals Heterogeneous Roles in Radiation Response and Redox Stress Across Clonal Lines in Triple-Negative Breast Cancer
              </a>
            </td>
          <td>
            Grace O. Ajayi, Aihui Ma, Shirin R. Modarai, Lynn M. Opdenaker, Jennifer Sims-Mourtada
          </td>
          <td>2025-03-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64abcfa81e97e4ccd6aa92a436d6301cbd6a137c" target='_blank'>
              CRISPR-Cas9: Transforming Functional Genomics, Precision Medicine, and Drug Development - Opportunities, Challenges, and Future Directions.
              </a>
            </td>
          <td>
            Zuber Khan, Mumtaz, Sumedha Gupta, Sidharth Mehan, Tarun Sharma, Manjeet Kumar, Pankaj Kumar Maurya, Arun Kumar Sharma, Ghanshyam Das Gupta, A. Narula
          </td>
          <td>2025-03-21</td>
          <td>Current gene therapy</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4625a1647e0225fc85616b697ca8ae26c76a3c30" target='_blank'>
              Transformative CRISPR-Cas9 Technologies: A Review of Molecular Mechanisms, Precision Editing Techniques, and Clinical Progress in Sickle Cell Disease.
              </a>
            </td>
          <td>
            Komal, Prabhjot Kaur, Nidhi Arora, Jyotiram A. Sawale, Amandeep Singh
          </td>
          <td>2025-03-05</td>
          <td>Current drug metabolism</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Epigenetic modifications, which reversibly regulate gene expression without altering the DNA sequence, are increasingly described in the literature as essential elements in the processes leading to cancer development. SETDB1 regulates histone 3 (H3) K9 di- and trimethylation, promoting heterochromatin formation, and plays a key role in gene silencing. Epigenetic deregulation of SETDB1 expression appears to be involved in different cancers types, particularly in aggressive, relapsing or treatment-resistant subtypes. Despite advances in research, the full range of mechanisms through which this protein acts remains unclear; however, it is evident that SETDB1 has a pivotal role, particularly in the mesenchymal stem cells differentiation, tumor evasion and treatment resistance. Its role in genetically complex sarcomas, such as osteosarcoma, has not been fully explored, although recent Omics analyses suggest its presence and amplification in osteosarcoma. Given its involvement in osteoblastogenesis and adipogenesis, we discuss the potential of SETDB1 as a key target for new therapeutic strategies in osteosarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b378d2be6324f27ab95439dedf132da8d385c12" target='_blank'>
              SETDB1 amplification in osteosarcomas: Insights from its role in healthy tissues and other cancer types
              </a>
            </td>
          <td>
            Elodie Verdier, Nathalie Gaspar, Maria Eugénia Marques da Costa, A. Marchais
          </td>
          <td>2025-02-12</td>
          <td>Oncotarget</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Mutations in the DNA can affect cancer development and progression not only by changing the amino acid chain but also by affecting different regulatory elements such as transcription factors. This study introduces a novel pipeline to identify “mutation blocks” - small genomic areas with high mutation rates that potentially influence transcription factors binding. By analyzing GCB lymphoma patient data, mutations blocks were identified that correlated with gene expression changes and were linked to transcription factor activity. These mutation blocks suggest a selection for mutations that alter gene regulation, contributing to lymphoma development. The analysis identified 56 mutation blocks in germinal center B-cell like diffuse large B-cell lymphoma (GCB DLBCL) patients’ genomes, affecting genes such as BCL2, MYC, SGK1, and PIM1, and linked to transcription factors including MSC, TCFL5, HOXB7, FOXP3, and ZBTB6. A machine learning model that used gene ontology data suggested further potential transcription factors-gene pairs influencing cancer. These findings highlight the role of synonymous and silent mutations in altering transcription factors binding and gene expression, offering insights into the mechanisms of GCB lymphoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76d3b64e91469debfcea770e479a146974e05ae2" target='_blank'>
              Novel Driver Mutations in GCB Lymphoma Patients That Affect Transcription Factors Binding
              </a>
            </td>
          <td>
            Ofek Shami-Schnitzer, T. Tuller
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddac82763361480546b876aad89a46e0a8e233fa" target='_blank'>
              Single Cell RNA Sequencing in Colorectal Cancer Immunology: Recent Updates, Application, and Emerging Challenges.
              </a>
            </td>
          <td>
            Sabrina George, Nor Adzimah Johdi
          </td>
          <td>2025-02-12</td>
          <td>Current molecular medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Increased stemness of cancer cells exacerbates radioresistance, thereby greatly limiting the efficacy of radiotherapy. In order to study the changes in cancer cell stemness during radiotherapy, we established a radioresistance model of human non-small cell lung cancer A549 cells and obtained A549 radioresistant cells (A549-RR). We sampled the cells at different time points during the modeling process and investigated the heterogeneity of each group of cells using single-cell sequencing. Cells in the early stages of fractionated irradiation were found to be significantly up-regulated in stemness, and a subpopulation of cells producing this response was screened and referred to as “radiation-induced stemness-responsive cancer cells”. They were undergoing stemness response, energy metabolism reprogramming, and progressively differentiating into cells with more diverse and malignant phenotypes in order to attenuate the killing effect of radiation. Furthermore, we demonstrated that such responses might be driven by the activation of the EGFR-Hippo signaling pathway axis, which also plays a crucial role in the development of radioresistance. Our study reveals the dynamic evolution of cell subpopulation in cancer cells during fractionated radiotherapy; the early stage of irradiation can determine the destiny of the radiation-induced stemness-responsive cancer cells. The activation of stemness-like phenotypes during the development of radioresistance is not the result of dose accumulation but occurs during the early stage of radiotherapy with relatively low-dose irradiation. The degree of the radiation-induced stemness response of cancer cells mediated by the EGFR-Hippo signaling pathway might be a potential predictor of the efficacy of radiotherapy and the development of radioresistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c6498ad18eccc13924e712513e028e4bc431ff4" target='_blank'>
              Single-Cell Sequencing Reveals the Role of Radiation-Induced Stemness-Responsive Cancer Cells in the Development of Radioresistance
              </a>
            </td>
          <td>
            Zheng Shi, Cuilan Hu, Jiadi Liu, Wei Cheng, Xiaohua Chen, Xiongxiong Liu, Yanyu Bao, Haidong Tian, Boyi Yu, Feifei Gao, Fei Ye, Xiaodong Jin, Chao Sun, Qiang Li
          </td>
          <td>2025-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09d171557e9628c9a32e1f797d2ae45ffca94dcd" target='_blank'>
              Cytoplasmic WEE1 promotes resistance to PD-1 blockade through hyperactivation of the HSP90A/TCL1/AKT signaling axis in NANOGhigh tumors.
              </a>
            </td>
          <td>
            Suyeon Kim, Hyo-Jung Lee, Seungho Lee, Jo Eun Chung, Se Jin Oh, K. Song, Eunho Cho, Min Kyu Son, Heeju Kwon, Seung-Jong Kim, Chaeleen Lee, Suhwan Chang, Tae Woo Kim
          </td>
          <td>2025-03-11</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdf3896d81e701493b4d97e132a04a0cb6710489" target='_blank'>
              Insights into Extrachromosomal DNA in Cancer: Biogenesis, Methodologies, Functions, and Therapeutic Potential.
              </a>
            </td>
          <td>
            Xudong Mao, Guocheng Rao, Gonghui Li, Shihan Chen
          </td>
          <td>2025-02-13</td>
          <td>Advanced biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18a99a90f864b231f78785ccac2d5080c6a00e33" target='_blank'>
              Explainable AI-driven prediction of APE1 inhibitors: enhancing cancer therapy with machine learning models and feature importance analysis.
              </a>
            </td>
          <td>
            Aga Basit Iqbal, Tariq Ahmad Masoodi, A. Bhat, M. Macha, Assif Assad, Syed Zubair Ahmad Shah
          </td>
          <td>2025-02-21</td>
          <td>Molecular diversity</td>
          <td>1</td>
          <td>35</td>
        </tr>

        <tr id="Pancreatic adenocarcinoma (PAAD) is a fatal malignant tumor of the digestive system, and immunotherapy has currently emerged as a key therapeutic approach for treating PAAD, with its efficacy closely linked to T-cell subsets and the tumor immune microenvironment. However, reliable predictive markers to guide clinical immunotherapy for PAAD are not available. We analyzed the single-cell RNA sequencing (scRNA-seq) data focused on PAAD from the GeneExpressionOmnibus (GEO) database. Then, the information from the Cancer Genome Atlas (TCGA) database was integrated to develop and validate a prognostic risk model derived from T-cell marker genes. Subsequently, the correlation between these risk models and the effectiveness of immunotherapy was explored. Analysis of scRNA-seq data uncovered six T-cell subtypes and 1837 T-cell differentially expressed genes (DEGs). Combining these data with the TCGA dataset, we constructed a T-cell prognostic risk model containing 16 DEGs, which can effectively predict patient survival and immunotherapy outcomes. We have found that patients in the low-risk group had better prognostic outcomes, increased immune cell infiltration, and signs of immune activation compared to those in the high-risk group. Additionally, analysis of tumor mutation burden showed higher mutation rates in patients with PAAD in the high-risk group. Risk scores with immune checkpoint gene expression and drug sensitivity analysis provide patients with multiple therapeutic targets and drug options. Our study constructed a prognostic risk model for PAAD patients based on T-cell marker genes, providing valuable insights into predicting patient prognosis and the effectiveness of immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2eccd9b99a66967879131e3edf663927ebf7fa1" target='_blank'>
              Construction of T-Cell-Related Prognostic Risk Models and Prediction of Tumor Immune Microenvironment Regulation in Pancreatic Adenocarcinoma via Integrated Analysis of Single-Cell RNA-Seq and Bulk RNA-Seq
              </a>
            </td>
          <td>
            Ding-ya Sun, Yijie Hu, Jun Peng, Shan Wang
          </td>
          <td>2025-03-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb2e9255d35debfd9a15445b40a466e6a5d29ad6" target='_blank'>
              Chromosomal Instability and Clonal Heterogeneity in Breast Cancer: From Mechanisms to Clinical Applications
              </a>
            </td>
          <td>
            María Paula Meléndez-Flórez, O. Ortega-Recalde, N. Rangel, Milena Rondón-Lagos
          </td>
          <td>2025-04-04</td>
          <td>Cancers</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background The mechanisms of epigenetic regulation emerge as a fundamental determinant in the complex landscape of cancer initiation and advancement. However, the specific impact of epigenetic regulation on cancer progression remains unclear. To explore the relationship between epigenetic regulation and cancer progression, we utilized transcriptomic data from The Cancer Genome Atlas (TCGA) datasets to investigate the association. Methods We obtained transcriptomic data of epigenetic gene dataset from the TCGA database and calculated an epigenetic score using the Least Absolute Shrinkage and Selection Operator (LASSO) Cox model. Additionally, we created a nomogram that integrates the epigenetic score and clinical features, providing a more comprehensive tool for tumor patients prognosis assessment. Results We calculated the epigenetic score based on the expression levels of epigenetic-related genes. The nomogram we developed incorporates the epigenetic score and clinical characteristics. The epigenetic score was positively correlated with the expression of genes related to hallmarkers of cancer, including glycolysis, epithelial-mesenchymal transition (EMT), cell cycle, DNA repair, angiogenesis, and inflammatory response. Furthermore, we performed gene ontology (GO) and kyoto encyclopedia of genes and genomes (KEGG) analysis to explore the signaling pathways and biological processes in high epigenetic score group. Conclusion The epigenetic scoring system developed in this investigation represents an innovative approach that demonstrates remarkable potential in forecasting survival trajectories across diverse cancer types. These groundbreaking insights not only illuminate the intricate interactions between epigenetic mechanisms and gene expression regulation in oncological contexts, but also indicate that the derived epigenetic metric could potentially emerge as a significant prognostic biomarker for cancer outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b405c467d91505b22b6bd6b604ad22bfbaf352f" target='_blank'>
              Exploring the role of epigenetic regulation in cancer prognosis with epigenetic score
              </a>
            </td>
          <td>
            Ruiguang Zhang, Shimin Jin, Qi Xu, Rongxiao Dai
          </td>
          <td>2025-02-18</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The appearance of cancer progresses through a multistep process that includes genetic, epigenetic, mutational, inflammatory and metabolic disturbances to signaling pathways within an organ. The combined influence of these changes will dictate the growth properties of the cells; the direction of further malignancy depends on the severity of these “disturbances”. The molecular mechanisms driving abnormal inflammation and metabolism are beginning to be identified and, in some cases, are quite prominent in pre-condition states of cancer and are significant drivers of the malignant phenotype. As such, utilizing signaling pathways linked to inflammation and metabolism as biomarkers of cancer is an emerging method and includes pathways beyond those well characterized to drive metabolism or inflammation. In this review, we will discuss several emerging elements influencing proliferation, inflammation and metabolism that may play a part as drivers of the cancer phenotype. These include AMPK and leptin (linked to metabolism), NOD2/RIPK2, TAK1 (linked to inflammation), lactate and pyruvate transporters (monocarboxylate transporter [MCT], linked to mitochondrial biogenesis and metabolism) and RASSF1A (linked to proliferation, cell death, cell cycle control, inflammation and epigenetics). We speculate that the aforementioned elements are important drivers of carcinogenesis that should be collectively referenced as being involved in “ImmunoMET Oncogenesis”, a new tripartite description of the role of elements in driving cancer. This term would suggest that for a better understanding of cancer, we need to understand how proliferation, inflammation and metabolic pathways are impacted and how they influence classical drivers of malignant transformation in order to drive ImmunoMET oncogenesis and the malignant state.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f600df6865b40197b8b8a33d72a983d145a378e4" target='_blank'>
              ImmunoMet Oncogenesis: A New Concept to Understand the Molecular Drivers of Cancer
              </a>
            </td>
          <td>
            Reshma Sirajee, Sami El Khatib, L. Dieleman, Mohamed Salla, S. Baksh
          </td>
          <td>2025-02-27</td>
          <td>Journal of Clinical Medicine</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Cancer is complex because of the critical imbalance in genetic regulation as characterized by both the overexpression of oncogenes (OGs), mainly through mutations, amplifications, and translocations, and the inactivation of tumor-suppressor genes (TSGs), which entail the preservation of genomic integrity by inducing apoptosis to counter the malignant growth. Reviewing the intricate molecular interplay between OGs and TSGs draws attention to their cell cycle, apoptosis, and cancer metabolism regulation. In the present review, we discuss seminal discoveries, such as Knudson’s two-hit hypothesis, which framed the field’s understanding of cancer genetics, leading to the next breakthroughs with next-generation sequencing and epigenetic profiling, revealing novel insights into OG and TSG dysregulation with opportunities for targeted therapy. The key pathways, such as MAPK/ERK, PI3K/AKT/mTOR, and Wnt/β-catenin, are presented in the context of tumor progression. Importantly, we further highlighted the advances in therapeutic strategies, including inhibitors of KRAS and MYC and restoration of TSG function, despite which mechanisms of resistance and tumor heterogeneity pose daunting challenges. A high-level understanding of interactions between OG-TSGs forms the basis for effective, personalized cancer treatment—something to strive for in better clinical outcomes. This synthesis should integrate foundational biology with translation and, in this case, contribute to the ongoing effort against cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b78e398fe4233b0d93ca9c44f0387aa4d32d4ab" target='_blank'>
              Exploring the Genetic Orchestra of Cancer: The Interplay Between Oncogenes and Tumor-Suppressor Genes
              </a>
            </td>
          <td>
            Sajal Raj Singh, Rakesh Bhaskar, Shampa Ghosh, Bhuvaneshwar Yarlagadda, Krishna Kumar Singh, Prashant Verma, Sonali Sengupta, M. Mladenov, N. Hadzi-Petrushev, R. Stojchevski, J. K. Sinha, Dimiter B Avtanski
          </td>
          <td>2025-03-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eed4652faa332d57f89c15c04ea896c21df38766" target='_blank'>
              Individualized pseudogenes networks for survival prognosis in B-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Arturo Kenzuke Nakamura-García, M. Kuijjer, J. Espinal-Enríquez
          </td>
          <td>2025-03-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0045e60f5f09ff48941f2c2a8009799ec50aa5d7" target='_blank'>
              Stem Cell-Parenchymal Fusion: Communication and Gene Regulation
              </a>
            </td>
          <td>
            JoAnne Huang, Fateme Nazaryabrbekoh, Joohyun Kim, Jangwook P. Jung
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Introduction: Leukocyte adhesion deficiency type 1 (LAD1) is a severe inborn error of immunity caused by mutations in the ITGB2 gene, which encodes the beta-2 integrin subunit (CD18). These mutations lead to the absence or deficiency of CD18/CD11a, b, and c heterodimers, crucial for leukocyte adhesion and immune function. CRISPR-Cas9 Gene editing technology represents a promising approach for correcting these genomic defects restore the stable expression of CD18 and reverse the disease. Methods: We developed a CRISPR-Cas9-based gene correction strategy using Jurkat cells and patient-derived lymphoblastoid cell lines as surrogates for hematopoietic progenitor cells. Three candidate gRNAs were first predicted in silico using CRISPOR and experimentally tested in wild-type ITGB2-expressing Jurkat cells to identify the gRNA with the highest genomic DNA cleavage efficiency. The most efficient gRNA was then paired with espCas9 and used alongside five homology-directed repair templates (HDRs) (single-stranded donor oligonucleotides, ssODNs) to repair ITGB2 defects in patient-derived lymphoblastoid cell lines. CD18 expression levels in edited cells were quantified via flow cytometry, and whole-genome sequencing (WGS) was conducted to assess off-target effects and insertion accuracy. Results: Among the three candidate gRNAs, 2-rev gRNA exhibited the highest genomic cleavage rate in Jurkat cells. Using this gRNA with espCas9 and HDR-2, we achieved a 23% restoration of CD18 expression in LAD1 patient-derived cells, a level sufficient to change the disease course from severe to moderate. Whole-genome sequencing confirmed the absence of off-target mutations or undesired DNA insertions, demonstrating high specificity and precision in gene correction. Discussion: This CRISPR-Cas9-based method provides a precise and effective approach for correcting ITGB2 mutations in LAD1 patients. The high-fidelity gene editing process, validated through WGS, supports its potential for future applications in CD34+ hematopoietic stem cell therapies. The approach can be further optimized for clinical translation, offering a path toward a stable and long-term cure for LAD1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af64b4b32aef04c80bbc3b2f45c925d4b04fb25b" target='_blank'>
              Lymphoblastoid and Jurkat cell lines are useful surrogate in developing a CRISPR-Cas9 method to correct leukocyte adhesion deficiency genomic defect
              </a>
            </td>
          <td>
            Ahmad R. Ramadan, Noureddine Ben Khalaf, Khaled Trabelsi, H. Bakheit, I. Ben‐Mustapha, M. Barbouche, M. Fathallah
          </td>
          <td>2025-03-21</td>
          <td>Frontiers in Bioengineering and Biotechnology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background: Clear cell renal cell carcinoma (ccRCC) is a prevalent and aggressive subtype of kidney cancer, often associated with metastasis and recurrence. Identifying key genes involved in ccRCC progression is critical for improving treatment strategies and patient outcomes. Methods: We performed a large-scale genome-wide CRISPR screening to identify genes crucial to ccRCC progression using the DepMap database. For discovery and validation, we integrated multi-omics data from The Cancer Genome Atlas (TCGA), GEO, and the NJMU-ccRCC clinical cohort. Bioinformatics analyses, including differential expression, pathway enrichment, and protein-protein interaction network analysis, were conducted to elucidate the biological functions. To validate our findings, we employed immunohistochemistry, qRT-PCR, and various cellular assays to investigate the role of PRC1 in ccRCC. Results: CRISPR screening identified PRC1 as a key gene significantly overexpressed in ccRCC tissues from the DepMap database. Elevated PRC1 expression was associated with poor overall survival, disease-specific survival, and progression-free interval. Silencing PRC1 in ccRCC cell lines inhibited cell proliferation, migration, and colony formation. Functional enrichment analyses revealed that PRC1 is involved in essential processes such as cell cycle regulation, mitosis, and cytokinesis. Additionally, PRC1 expression was correlated with the activation of the Wnt/β-catenin pathway, suggesting that PRC1 plays a pivotal role in tumor progression. Conclusion: PRC1 emerges as a promising biomarker and therapeutic target for ccRCC. Elevated PRC1 expression is associated with poor prognosis, and its inhibition suppresses ccRCC cell proliferation and migration. Our findings underscore the crucial role of PRC1 in ccRCC progression and highlight the need for further investigation into its molecular mechanisms and therapeutic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f7e34d29eb95f0f25970817abd06150852dbc62" target='_blank'>
              Large-scale genomic-wide CRISPR screening revealed PRC1 as a tumor essential candidate in clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            Baochao Li, Yongsheng Pan, Jiajin Wu, Chenkui Miao, Zengjun Wang
          </td>
          <td>2025-03-03</td>
          <td>International Journal of Medical Sciences</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Human T-cell lymphotropic virus type 1 (HTLV-1) infection causes the uncommon and deadly cancer known as adult T-cell leukemia/lymphoma (ATLL), which affects mature T cells. Its clinical appearance is varied, and its prognosis is often miserable. Drug resistance to conventional therapies confers significant therapeutic challenges in the management of ATLL. This review discusses the emerging role of epigenetic medical advances in the treatment of ATLL, focusing on DNA methyltransferase inhibitors, histone deacetylase inhibitors, histone methyltransferase inhibitors, and BET inhibitors. Indeed, several classes of epigenetic therapies currently exhibit trailed efficacy in preclinical and clinical studies: DNA methyltransferase inhibitors like azacitidine and decitabine reexpression of silenced tumor suppressors; histone deacetylase inhibitors like vorinostat and romidepsin induce cell cycle arrest and apoptosis; bromodomain and extra-terminal inhibitors like JQ1 disrupt oncogenic signaling pathways. Whereas preclinical and early clinical data indicate modest to good efficacy for such treatments, significant challenges remain. Here, we discuss the current state of understanding of epigenetic dysregulation in ATLL and appraise the evidence supporting the use of these epi-drugs. However, despite the opened doors of epigenetic treatment, much more research is required with regard to showing the best combinations of drugs and their resistance mechanisms, the minimization of adverse effects, and how this hope will eventually be translated into benefit for the patient with ATLL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a1ce69e97a9adaccd0f62087c29dd1f173f2a27" target='_blank'>
              Advances in epigenetic treatment of adult T-cell leukemia/lymphoma: a comprehensive review
              </a>
            </td>
          <td>
            Arash Letafati, Rabeeh Mehdigholian Chaijani, Fahimeh Edalat, Nazila Eslami, Hanieh Askari, Farideh Askari, Sara Shirvani, Hamed Talebzadeh, Mahdiyeh Tarahomi, Nila MirKhani, Faeze Karimi, Mehdi Norouzi, Sayed-Hamidreza Mozhgani
          </td>
          <td>2025-03-01</td>
          <td>Clinical Epigenetics</td>
          <td>1</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ee52870e604baf41aae12c6d88c6bacc4d5a388" target='_blank'>
              Updates on DNA Repair Gene Deficiency in Colorectal Cancer (dMMR)
              </a>
            </td>
          <td>
            Gustavo Sevá-Pereira, C. S. R. Coy, Carlos Augusto Real Martinez
          </td>
          <td>2025-03-01</td>
          <td>Journal of Coloproctology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Despite remarkable advancements in therapeutic strategies, a considerable proportion of patients with bladder cancer (BC) still experience disease progression and unfavorable prognosis. The heterogeneity and biological functions of tumor endothelial cells (ECs) during BC progression remain poorly understood. We collected scRNA-seq data from BC samples and identified two EC subpopulations through hierarchical clustering analysis. The activity of signaling pathways in distinct EC subpopulations was assessed utilizing AUCell analysis. Gene regulatory networks (GRN) were constructed and analyzed for different EC subpopulations using the pySCENIC algorithm. Additionally, we investigated the association between the abundance of EC subpopulations and both clinical prognosis and immune cell infiltration. The biological effects of ESM1 protein on BC cells were further validated through EdU and Transwell assays. We analyzed 7,519 CD45-negative single cells from BC tissues and discerned two distinct EC subpopulations. The two subpopulations were characterized by high expression of ESM1 (S1 ECs) and CXCL2 (S2 ECs), respectively. In S1 ECs, we observed significant activation of signaling pathways involved in tumor promotion, including angiogenesis and cell proliferation. Additionally, our GRN analysis uncovered notable differences in transcription factor activity between S1 and S2 ECs. Moreover, ESM1 protein promoted proliferation and migration of BC cells. Patients with higher abundance of the S1 EC subpopulation exhibited more unfavorable clinical outcomes and increased infiltration of inhibitory immune cells. Our findings elucidate the transcriptional profiles and biological roles of the high ESM1-expression endothelial cell subpopulation in BC. This subpopulation is associated with poor prognosis and immunosuppressive tumor microenvironment. Accordingly, targeting endothelial cells with high ESM1 expression may offer a novel therapeutic strategy for patients with BC. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-95731-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4283029ca72adeb8c04b30a9f72593fd7af08134" target='_blank'>
              Single-cell sequencing uncovers a high ESM1-expression endothelial cell subpopulation associated with bladder cancer progression and the immunosuppressive microenvironment
              </a>
            </td>
          <td>
            Yifeng Qiu, Yuhan Wang, Jiahe Liu, Baohua Liu, Kai Sun, Qi Hou
          </td>
          <td>2025-03-30</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Investigating the pivotal role of CTHRC1 in the tumor microenvironment of triple-negative breast cancer (TNBC). Method The RNA transcriptomic data obtained from the Cancer Genome Atlas and single-cell sequencing data from TNBC in Gene Expression Omnibus (GEO) were acquired and subjected to analysis. A comprehensive investigation was conducted with a specific focus on characterizing CTHRC1 in TNBC and its correlation with invasive genes. Furthermore, additional analyses were performed to explore the relationship between CTHRC1, tumor immune cell infiltration, and immunotherapy in TNBC. The expression of CTHRC1 in the tumor microenvironment, cellular differentiation, and cellular communication was systematically analyzed using single-cell data from TNBC. Result The expression of CTHRC1 in patients with TNBC gradually increases concomitantly with the progression of tumor T-stage and N-stage. Simultaneously, there is a concurrent increase in the expression of most invasive gene sets. Furthermore, there is a significant augmentation in both infiltration abundance and activity of M2-type macrophages associated with elevated levels of CTHRC1 expression. Single-cell data reveal an upregulated expression of the invasive gene set in CTHRC1-positive cancer associated fibroblasts (CAFs), thereby modulating their interaction with M2-type macrophages. Multiple immunofluorescence analyses confirmed that CTHRC1 modulates immune cell infiltration and tumor cell invasion through the mediation of CAFs. Conclusion CTHRC1 was a molecule that exhibits characteristic expression in TNBC. CTHRC1 positive CAFs exert regulatory effects within the immunosuppressive microenvironment of TNBC by modulating M2-type macrophages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4be4e81b525de2c62cbaf620445907b577884546" target='_blank'>
              Single-cell and transcriptome analyses revealed CTHRC1 a potential therapeutic target mediating invasion and tumor microenvironment in TNBC: experimental validation
              </a>
            </td>
          <td>
            Hong Wan, Zichen Ling, Yuwei Xie, Han Jiang, Zhifan Ruan, Dashuai Yang, Xiaowei Yang, Jing Pei
          </td>
          <td>2025-03-11</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="MYC plays a pivotal role in the biology of various sarcoma subtypes, acting as a key regulator of tumor growth, proliferation, and metabolic reprogramming. This oncogene is frequently dysregulated across different sarcomas, where its expression is closely intertwined with the molecular features unique to each subtype. MYC interacts with critical pathways such as cell cycle regulation, apoptosis, and angiogenesis, amplifying tumor aggressiveness and resistance to standard therapies. Furthermore, MYC influences the tumor microenvironment by modulating cell–extracellular matrix interactions and immune evasion mechanisms, further complicating therapeutic management. Despite its well-established centrality in sarcoma pathogenesis, targeting MYC directly remains challenging due to its “undruggable” protein structure. However, emerging therapeutic strategies, including indirect MYC inhibition via epigenetic modulators, transcriptional machinery disruptors, and metabolic pathway inhibitors, offer new hope for sarcoma treatment. This review underscores the importance of understanding the intricate roles of MYC across sarcoma subtypes to guide the development of effective targeted therapies. Given MYC’s central role in tumorigenesis and progression, innovative approaches aiming at MYC inhibition could transform the therapeutic landscape for sarcoma patients, providing a much-needed avenue to overcome therapeutic resistance and improve clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b3e9fb9dbedb9674622b99a03bb647022b7fb21" target='_blank'>
              Untangling the Role of MYC in Sarcomas and Its Potential as a Promising Therapeutic Target
              </a>
            </td>
          <td>
            Fabio Sias, Stefano Zoroddu, Rossana Migheli, L. Bagella
          </td>
          <td>2025-02-25</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="CRISPR-Cas technology has revolutionized biology by enabling precise DNA and RNA edits with ease. However, significant challenges remain for translating this technology into clinical applications. Traditional protein engineering methods, such as rational design, mutagenesis screens, and directed evolution, have been used to address issues like low efficacy, specificity, and high immunogenicity. These methods are labor-intensive, time-consuming, and resource-intensive and often require detailed structural knowledge. Recently, computational strategies have emerged as powerful solutions to these limitations. Using artificial intelligence (AI) and machine learning (ML), the discovery and design of novel gene-editing enzymes can be streamlined. AI/ML models predict activity, specificity, and immunogenicity while also enhancing mutagenesis screens and directed evolution. These approaches not only accelerate rational design but also create new opportunities for developing safer and more efficient genome-editing tools, which could eventually be translated into the clinic.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6cb55c796a93b1a1e6ebc7882f1d2a726253cbc6" target='_blank'>
              Engineering a New Generation of Gene Editors: Integrating Synthetic Biology and AI Innovations
              </a>
            </td>
          <td>
            B. Chia, Yu Fen Samantha Seah, Bolun Wang, Kimberle Shen, Diya Srivastava, W. Chew
          </td>
          <td>2025-02-25</td>
          <td>ACS Synthetic Biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Tissues and organs develop from single founder cells, which give rise to distinct cell lineages that contribute to regeneration and maintenance of homeostasis in the adult. Tracing the genealogical relationships between individual cells and their gene expression signatures is an important step towards understanding how these processes are regulated in human health and disease. Here, we present mt-SCITE, a computational method for inferring the evolutionary history of dividing cells based on mitochondrial mutations detected in single cells. We show that mt-SCITE outperforms existing methods in accurately estimating the correct mitochondrial mutation tree from simulated data. To validate our method, we applied mt-SCITE on in vitro expanded T-cells profiled with bulk ATAC sequencing, where the clonal relationships were determined independently from T cell receptor sequences. We used our method to reconstruct the division histories and transcriptional heterogeneity of clonally related CD8+ T cells from single-cell RNA sequencing data obtained from a healthy human donor. This allowed us to track the in vivo development of CD8+ T cell clones post vaccination and resolve subclonal lineage relationships without the need for genetic barcoding. Our analysis revealed that T cell clones can adopt predominantly memory-like or effector-like states, while some exhibit mixed identities. Taken together, our method represents a novel approach to study cell lineage development in humans.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b46463e933e01c11da7ed98772b7d2ff4e8ab033" target='_blank'>
              Cell lineage inference from mitochondrial mutations
              </a>
            </td>
          <td>
            Joanna Hård, S. Seidel, Pedro F. Ferreira, Pietro Berkes, Katharina Jahn, Carl-Johan Eriksson, Jeff E. Mold, Jakob Michaelsson, Jack Kuipers, N. Beerenwinkel
          </td>
          <td>2025-03-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/471fa22c73f7c1a8083ff473dd937339ee492d0c" target='_blank'>
              Abstract A022: Decoding aneuploidy: Identifying drivers and therapeutic targets in recurrent breast cancer copy number alterations
              </a>
            </td>
          <td>
            Khalid N. Al-Zahrani, Ellen Langille, A. Obersterescu, Christopher Lowden, Katie Teng, Lauren Caldwell, David Cook, Miguel Pérez-Castro, Cynthia H Chiu, Alec Bahcheli, R. Tsai, Jacob Berman, Kin Chan, Linkang Zhang, K.W. Annie Bang, Michael Parsons, Adele Lopes, Jocelyn Nurtanto, E. I. Temel, Iosifina Fotiadou, Julien Dessapt, Hartland Jackson, Sean E Egan, J. Reimand, Jeffrey Wrana, Daniel Schramek
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="ABSTRACT Human endogenous retroviruses (HERVs) are proviral relics of infections that affected primates' germ line. Many HERV elements retain a residual capacity to encode transcripts and proteins that have been occasionally domesticated for the host physiology. In addition, HERV transcriptional modulation is of great interest to clarify the etiology of complex disorders such as cancer, even if a few studies assessed the specific HERV loci modulated in tumor tissues. In the present work, we used a transcriptomic approach to investigate the specific expression of ~3300 HERV loci in paired tumor and normal tissues of 7 colorectal cancer (CRC) patients. A total of 102 HERVs were significantly modulated in CRC, with a general tendency towards downregulation. Of note, among the 42 upregulated HERVs 23 belonged to the HERV‐H group, that is the most investigated in CRC. De novo transcriptome reconstruction and qPCR validation allowed to identify a transcript from a HERV‐H locus on chromosome Xp22.3 with high specific expression in CRC samples, potentially encoding for a partial Pol protein. These results provide a detailed description of HERV transcriptional variations in CRC and its interindividual variability, identifying a HERV‐H transcript that deserves further investigation for its possible impact on tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8d57225f528a23471e9e5e65e4532aff3132bd2" target='_blank'>
              HERV Modulation in Colorectal Carcinoma Patients: A Snapshot of Endogenous Retroviral Transcriptome
              </a>
            </td>
          <td>
            Nicole Grandi, Ching-Hsuan Liu, Saili Chabukswar, Daniele Carta, Yun Yen, Liang-Tzung Lin, E. Tramontano
          </td>
          <td>2025-02-24</td>
          <td>Journal of Medical Virology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Background Tuberculosis, a persistent global health threat, necessitates a comprehensive understanding of the genes and pathways crucial for the survival and virulence of the causative pathogen, Mycobacterium tuberculosis. Working with M. tuberculosis (M.tb) presents significant challenges; therefore, the use of M. smegmatis as a surrogate system for conducting genetic studies of M.tb has proven to be highly valuable. Development of novel genetic tools to probe cellular processes accelerates the progress in the field of drug development and also helps in understanding the basic physiology of the bacterium. Results This study reports the successful implementation and evaluation of the CRISPR-Cas12a system for gene repression in Mycobacterium smegmatis, a surrogate for M. tuberculosis. We engineered a Cas12a-based CRISPR interference (CRISPRi) system and assessed its functionality. Targeting 45 genes with a single sgRNA per gene, we achieved efficient gene repression, leading to marked phenotypic changes. Each knockdown strain was evaluated individually for growth phenotypes, and a comparison of the results with the reported essential gene library probed with dCas9 demonstrated congruous results across diverse gene categories. The study shows that CRISPR/Cas12a system can be effectively utilised with a single gene specific target for efficient silencing of the gene and highlights the importance of subsequent growth assays required to evaluate the vulnerability of targeted gene silencing. Conclusion Our findings reveal the robustness and versatility of the dCas12a-CRISPRi system in M. smegmatis, providing a valuable tool for functional genomics research. This work showcases the potential of the dCas12a-CRISPRi system in investigating essential genes, enabling a deeper understanding of the biology and potential therapeutic targets in mycobacterium species. Supplementary Information The online version contains supplementary material available at 10.1186/s13036-025-00490-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fd956491f5700a578d8ac40b6d89d08cd7f2d58" target='_blank'>
              CRISPR/Cas12a-mediated gene silencing across diverse functional genes demonstrates single gene-specific spacer efficacy in Mycobacterium smegmatis
              </a>
            </td>
          <td>
            Megha Sodani, C. S. Misra, Savita Kulkarni, D. Rath
          </td>
          <td>2025-02-28</td>
          <td>Journal of Biological Engineering</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Extrachromosomal circular DNA (eccDNA) has potential in tumor diagnosis, particularly for improving diagnostic accuracy and early cancer detection; however, many challenges remain in its application to clinical practice. Methods We conducted a Circle‐Seq analysis on clinical samples at different stages of colorectal cancer progression to examine the dynamic changes of eccDNA during the progression of colorectal cancer. We used breakpoint-specific PCR to verify candidate eccDNAs identified by Circle‐Seq. The results were further validated using the AOM/DSS-induced colorectal cancer model. Results There was an increase in the abundance of eccDNA with the progression of colorectal cancer. The genes associated with these eccDNA molecules were primarily related to signaling pathways involved in tumor development and metastasis. Our analysis also revealed that eccDNA abundance positively correlates with gene expression, and eccDNA derived from specific genes has potential value for the early diagnosis of tumors. Conclusions This study revealed a connection between eccDNA and colorectal cancer progression and highlights the clinical potential of eccDNA for the early diagnosis of colorectal cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s10020-025-01164-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d3999071e24587252e2469121996c435197d05b" target='_blank'>
              Extrachromosomal circular DNA as a novel biomarker for the progression of colorectal cancer
              </a>
            </td>
          <td>
            Quanpeng Qiu, Yi Ding, Xiaolong Guo, Jing Han, Jiaqi Zhang, Yaping Liu, Junjun She, Yinnan Chen
          </td>
          <td>2025-04-01</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background/Objectives: Aberrant hypermethylation in the promoter regions of tumor suppressor genes facilitates the pathogenesis and progression of cancer. Therefore, inhibitors targeting DNA methyltransferase (DNMT) have been tested in clinical studies. However, the current monotherapy of DNMT inhibitors shows limited efficacy. Furthermore, the mechanism of action of DNMT inhibitors is DNA replication-dependent. To address these limitations, we developed a novel core–shell-type “epigenetics control (EpC) nanocarrier” that encapsulated decitabine (5-aza-dC) in the PLGA core nanoparticle and hybridized TET1 gene-encoding pDNA on the lipid shell surface. This study aimed to evaluate whether the dual delivery of DNMT inhibitors and pDNA of TET1 could synergistically enhance tumor suppressor gene expression and induce cell cycle arrest and/or apoptosis in cancer cells. Herein, we demonstrate the potential of the EpC carrier in HCT116 human colon cancer cells to upregulate tumor suppressor gene expression and rapidly achieve cell cycle arrest. Methods: PLGA core nanoparticles were prepared by the W/O/W double emulsion method. The formation of core–shell nanoparticles and complexation with pDNA were investigated and optimized by dynamic light scattering, zeta potential measurement, and agarose gel electrophoresis. The cellular uptake and transfection efficiency were measured by confocal laser scanning microscopy and a luciferase assay, respectively. The expression of p53 protein was detected by Western blotting. The anti-tumor effects of the EpC nanocarrier were evaluated by cell cycle analysis and an apoptosis assay. Results: The EpC nanocarrier delivered the DNMT inhibitor and TET gene-encoding pDNA into HCT116 cells. It promoted the expression of the tumor suppressor protein p53 and induced rapid cell cycle arrest in the G2/M phase in HCT116 cells. Conclusions: Our findings suggest that the dual-targeting of DNMT and TET enzymes effectively repairs aberrant DNA methylation and induces growth arrest in cancer cells, and the dual-targeting strategy may contribute to the advancement of epigenetic cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/310794284715bd67edbc662582ab25897ba7a464" target='_blank'>
              Novel Epigenetics Control (EpC) Nanocarrier for Cancer Therapy Through Dual-Targeting Approach to DNA Methyltransferase and Ten-Eleven Translocation Enzymes
              </a>
            </td>
          <td>
            Risa Mitsuhashi, Kiyoshi Sato, Hiroyoshi Kawakami
          </td>
          <td>2025-02-11</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="ABSTRACT Background Cancer development is closely associated with complex alterations in the tumor microenvironment (TME). Among these, immune cells within the TME play a huge role in personalized tumor diagnosis and treatment. Objectives This review aims to summarize the diversity of immune cells in the TME, their impact on patient prognosis and treatment response, and the contributions of single‐cell RNA sequencing (scRNA‐seq) in understanding their functional heterogeneity. Methods We analyzed recent studies utilizing scRNA‐seq to investigate immune cell populations in the TME, focusing on their interactions and regulatory mechanisms. Results ScRNA‐seq reveals the functional heterogeneity of immune cells, enhances our understanding of their role in tumor antibody responses, and facilitates the construction of immune cell interaction networks. These insights provide guidance for the development of cancer immunotherapies and personalized treatment approaches. Conclusion Applying scRNA‐seq to immune cell analysis in the TME offers a novel pathway for personalized cancer treatment. Despite its promise, several challenges remain, highlighting the need for further advancements to fully integrate scRNA‐seq into clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36e80d2a964412d958098f4ad446c2f9779f2b7e" target='_blank'>
              Analyze the Diversity and Function of Immune Cells in the Tumor Microenvironment From the Perspective of Single‐Cell RNA Sequencing
              </a>
            </td>
          <td>
            Lujuan Ma, Yu Luan, Lin Lu
          </td>
          <td>2025-03-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Monoallelic deletion of the short arm of chromosome 17 (del17p) is a recurrent abnormality in cancers with poor outcomes. Best studied in relation to haematological malignancies, associated functional outcomes are attributed mainly to loss and/or dysfunction of TP53, which is located at 17p13.1, but the wider impact of deletion of other genes located on 17p is poorly understood. 17p is one of the most gene-dense regions of the genome and includes tumour suppressor genes additional to TP53, genes essential for cell survival and proliferation, as well as small and long non-coding RNAs. In this review we utilise a data-driven approach to demarcate the extent of 17p deletion in multiple cancers and identify a common loss-of-function gene signature. We discuss how the resultant loss of heterozygosity (LOH) and haploinsufficiency may influence cell behaviour but also identify vulnerabilities that can potentially be exploited therapeutically. Finally, we highlight how emerging animal and isogenic cell line models of del17p can provide critical biological insights for cancer cell behaviour.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c6b9704e0bc8ddb9fedc0099d11922be33aebac" target='_blank'>
              Deletion of 17p in cancers: Guilt by (p53) association
              </a>
            </td>
          <td>
            Francisca van Kampen, Abigail Clark, Jamie Soul, Aditi Kanhere, Mark Glenn, Andrew R Pettitt, Nagesh Kalakonda, J. Slupsky
          </td>
          <td>2025-02-18</td>
          <td>Oncogene</td>
          <td>1</td>
          <td>32</td>
        </tr>

        <tr id="Background Prostate cancer (PCa) is the second most common malignant neoplasm in males and is the fifth leading cause of cancer-related mortality. Due to the use of prostate-specific antigen (PSA) screening and improved biopsy techniques, persons identified with early-stage prostate cancer often have a positive prognosis after comprehensive treatment. Nonetheless, prostate cancer is a latent illness that may present as an asymptomatic tumor in individuals aged 20–30. The overall survival (OS) of men with advanced PCa is significantly diminished. Consequently, there is an immediate want for innovative, accurate biomarkers to detect early prostate cancer. Methods This research analyzed the interaction network of differentially expressed genes (DEGs) related to metabolite interconversion enzymes in PCa by gene expression microarray data, single-cell RNA sequencing, oncogenes, and tumor suppressor genes (TSGs) utilizing bioinformatics techniques. This kind of analysis has not been documented in prior studies. Results We then used a dataset acquired by the Cancer Genome Atlas (TCGA) to confirm our findings. Genes including CYP3A5, PDE8B, AOX1, BNIPL, FADS2, RRM2, ALDH3B2, and GSTM2 may be significant in the diagnosis and treatment of PCa. Conclusion Our objective was to provide new perspectives on the molecular properties and pathways of DEGs in PCa and to uncover potential biomarkers that play a crucial role in the genesis and progression of PCa. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-01926-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfa0f09862c96ff9cec51a61499fdcc67366281e" target='_blank'>
              Analysis of microarray and single-cell RNA-seq identifies gene co-expression, cell–cell communication, and tumor environment associated with metabolite interconversion enzyme in prostate cancer
              </a>
            </td>
          <td>
            Danial Hashemi Karoii, Ali Shakeri Abroudi, Nadia Forghani, Sobhan Bavandi, Melika Djamali, Hamoon Baghaei, Sana Shafaeitilaki, Ehsan HasanZadeh
          </td>
          <td>2025-02-13</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background Cell-free DNA (cfDNA) in plasma carries epigenetic signatures specific to tissue or cell of origin. Aberrant methylation patterns in circulating cfDNA have emerged as valuable tools for noninvasive cancer detection, prenatal diagnostics, and organ transplant assessment. Such epigenetic changes also hold significant promise for the diagnosis of neurodegenerative diseases, which often progresses slowly and has a lengthy asymptomatic period. However, genome-wide cfDNA methylation changes in neurodegenerative diseases remain poorly understood. Results We used whole-genome bisulfite sequencing (WGBS) to profile age-dependent and ALS-associated methylation signatures in cfDNA from 30 individuals, including young and middle-aged controls, as well as ALS patients with matched controls. We identified 5,223 age-related differentially methylated loci (DMLs) (FDR < 0.05), with 51.6% showing hypomethylation in older individuals. Our results significantly overlapped with age-associated CpGs identified in a large blood-based epigenome-wide association study (EWAS). Comparing ALS patients to controls, we detected 1,045 differentially methylated regions (DMRs) in gene bodies, promoters, and intergenic regions. Notably, these DMRs were linked to key ALS-associated pathways, including endocytosis and cell adhesion. Integration with spinal cord transcriptomics revealed that 31% of DMR-associated genes exhibited differential expression in ALS patients compared to controls, with over 20 genes significantly correlating with disease duration. Furthermore, comparison with published single-nucleus RNA sequencing (snRNA-Seq) data of ALS demonstrated that cfDNA methylation changes reflects cell-type-specific gene dysregulation in the brain of ALS patients, particularly in excitatory neurons and astrocytes. Deconvolution of cfDNA methylation profiles suggested altered proportions of immune and liver-derived cfDNA in ALS patients. Conclusions cfDNA methylation is a powerful tool for assessing age-related changes and ALS-specific molecular dysregulation by revealing perturbed locus, genes, and the proportional contributions of different tissues/cells to the plasma. This technique holds promise for clinical application in biomarker discovery across a broad spectrum of neurodegenerative disorders. Supplementary Information The online version contains supplementary material available at 10.1186/s13578-025-01366-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f50dea3e9f7300d7800974584eb194f23dc33f9" target='_blank'>
              Whole-genome bisulfite sequencing of cell-free DNA unveils age-dependent and ALS-associated methylation alterations
              </a>
            </td>
          <td>
            Yulin Jin, Karen N Conneely, Wenjing Ma, Robert K Naviaux, Teepu Siddique, Emily G. Allen, Sandra Guingrich, Robert M Pascuzzi, Peng Jin
          </td>
          <td>2025-02-20</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/687d3d9b838e13f050d770abff405a6b760ad3ff" target='_blank'>
              Diverse transcriptomic and mutational patterns but limited functional pathway alterations in patient-derived SS cells
              </a>
            </td>
          <td>
            F. Gallardo, Evelyn Andrades, Arnau Iglesias, Mariano R Maqueda, T. Lobo-Jarne, Jessica González, Joan Bertran, David Conde, Eva Rodriguez, Beatriz Bellosillo, Ramon M. Pujol, A. Bigas, Lluís Espinosa
          </td>
          <td>2025-03-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/adb21c2801e782dbb110eb435c1cbbe7b47f6623" target='_blank'>
              Identifying gene expression and cellular pathways involved in glomerular AL-amyloidosis and correlation with experimental data: seeking novel therapeutic interventions.
              </a>
            </td>
          <td>
            Guillermo A. Herrera, J. Teng, Chun Zeng, L. Del Pozo-Yauner, Bing Liu, E. Turbat-Herrera
          </td>
          <td>2025-02-21</td>
          <td>Ultrastructural pathology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Cellular senescence, a stable state of cell cycle arrest induced by various stressors or genomic damage, is recognized as a hallmark of cancer. It exerts a context-dependent dual role in cancer initiation and progression, functioning as a tumor suppressor and promoter. The complexity of senescence in cancer arises from its mechanistic diversity, potential reversibility, and heterogeneity. A key mediator of these effects is the senescence-associated secretory phenotype (SASP), a repertoire of bioactive molecules that influence tumor microenvironment (TME) remodeling, modulate cancer cell behavior, and contribute to therapeutic resistance. Given its intricate role in cancer biology, senescence presents both challenges and opportunities for therapeutic intervention. Strategies targeting senescence pathways, including senescence-inducing therapies and senolytic approaches, offer promising avenues for cancer treatment. This review provides a comprehensive analysis of the regulatory mechanisms governing cellular senescence in tumors. We also discuss emerging strategies to modulate senescence, highlighting novel therapeutic opportunities. A deeper understanding of these processes is essential for developing precision therapies and improving clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/286d205296f9a0a8d06235700589eef3dccbbc34" target='_blank'>
              Regulation of cellular senescence in tumor progression and therapeutic targeting: mechanisms and pathways
              </a>
            </td>
          <td>
            Bowei Liu, Zhigang Peng, Hao Zhang, Nan Zhang, Zao-bin Liu, Zhiwei Xia, Shaorong Huang, Peng Luo, Quan Cheng
          </td>
          <td>2025-04-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/147bfd57152033ce1c79fd9d00f386b5bffcdc1b" target='_blank'>
              Exploring ID4 as a Driver of Aggression and a Therapeutic Target in Triple-Negative Breast Cancer
              </a>
            </td>
          <td>
            C. Toro, S. Real, S. Laurito, MT Branham
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Despite advances, Glioblastoma (GBM) treatment remains challenging due to its rapid progression and resistance to therapies. Objectives This study aimed to investigate the role of anoikis—a mechanism by which cells evade programmed cell death upon detachment from the extracellular matrix—in GBM progression and prognosis. Methods Utilizing single-cell sequencing and bulk-transcriptome sequencing data from TCGA, GEO, and CGGA databases, we performed comprehensive bioinformatics analyses. We identified anoikis-related genes, constructed a prognostic model using 101 machine learning algorithms, and validated its clinical utility across multiple cohorts.Finally, we also verified the expression of model genes and the function of key gene in clinical samples and cell lines. Results Single-cell sequencing revealed heterogeneous expression of anoikis-related genes across distinct cell populations within GBM. MES-like Malignant cells and Myeloids exhibited higher enrichment of these genes, implicating their role in anoikis resistance and tumor aggressiveness. The prognostic model, based on identified genes, effectively stratified patients into high-risk and low-risk groups, demonstrating significant differences in survival outcomes. Mutation and tumor microenvironment analyses highlighted distinct genetic landscapes and immune cell infiltration patterns associated with different risk groups. SLC43A3 emerged as a key gene, showing significant upregulation in tumor tissues and correlating with poor prognosis in GBM. Conclusion This study provides insights into the molecular mechanisms of anoikis resistance in GBM, underscoring its critical role in tumor progression and patient prognosis. The developed prognostic model offers a promising tool for personalized treatment strategies and warrants further exploration of targeted therapies to improve outcomes for GBM patients. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-03752-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90776aa9e4e44d828b89f11f0b82b3f7faa8093b" target='_blank'>
              Unraveling anoikis in glioblastoma: insights from single-cell sequencing and prognostic modeling
              </a>
            </td>
          <td>
            Q. Tang, Chenfeng Ma, Jiaheng Xie, Qixiang Zhang, Bingtao Zhang, Weiqi Bian, Qingyu Lu, Zeyu Wan, Wei Wu
          </td>
          <td>2025-03-26</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Over the past decade, circular RNAs (circRNAs) have gained recognition as a novel class of genetic molecules, many of which are implicated in cancer pathogenesis via different mechanisms, including drug resistance, immune escape, and radio-resistance. ExosomalcircRNAs, in particular, facilitatecommunication between tumour cells and micro-environmental cells, including immune cells, fibroblasts, and other components. Notably, micro-environmental cells can reportedly influence tumour progression and treatment resistance by releasing exosomalcircRNAs. circRNAs often exhibit tissue- and cancer-specific expression patterns, and growing evidence highlights their potential clinical relevance and utility. These molecules show strong promise as potential biomarkers and therapeutic targets for cancer diagnosis and treatment. Therefore, this review aimed to briefly discuss the latest findings on the roles and resistance mechanisms of key circRNAs in the treatment of various malignancies, including lung, breast, liver, colorectal, and gastric cancers, as well as haematological malignancies and neuroblastoma.This review will contribute to the identification of new circRNA biomarkers for the early diagnosis as well as therapeutic targets for the treatment of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddb80961cd695209a157b69a05f62bd12ae66e96" target='_blank'>
              Role of circular RNAs in cancer therapy resistance
              </a>
            </td>
          <td>
            Wenjuan Liu, Jiling Niu, Yanfei Huo, Long Zhang, Linyu Han, Nasha Zhang, Ming Yang
          </td>
          <td>2025-02-25</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="T cell-based immunotherapies hold promise in treating cancer by leveraging the immune system’s recognition of cancer-specific antigens1. However, their efficacy is limited in tumours with few somatic mutations and substantial intratumoural heterogeneity2–4. Here we introduce a previously uncharacterized class of tumour-wide public neoantigens originating from RNA splicing aberrations in diverse cancer types. We identified T cell receptor clones capable of recognizing and targeting neoantigens derived from aberrant splicing in GNAS and RPL22. In cases with multi-site biopsies, we detected the tumour-wide expression of the GNAS neojunction in glioma, mesothelioma, prostate cancer and liver cancer. These neoantigens are endogenously generated and presented by tumour cells under physiologic conditions and are sufficient to trigger cancer cell eradication by neoantigen-specific CD8+ T cells. Moreover, our study highlights a role for dysregulated splicing factor expression in specific cancer types, leading to recurrent patterns of neojunction upregulation. These findings establish a molecular basis for T cell-based immunotherapies addressing the challenges of intratumoural heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34beca577def84b7592da411aab4f563850a4d1a" target='_blank'>
              Tumour-wide RNA splicing aberrations generate actionable public neoantigens
              </a>
            </td>
          <td>
            Darwin W. Kwok, Nicholas O Stevers, Iñaki Etxeberria, Takahide Nejo, Maggie Colton Cove, Lee H. Chen, Jangham Jung, K. Okada, S. Lakshmanachetty, M. Gallus, Abhilash Barpanda, Chibo Hong, Gary Chan, Jerry Liu, Samuel H. Wu, Emilio Ramos, A. Yamamichi, P. Watchmaker, Hirokazu Ogino, A. Saijo, Aidan Du, Nadia Grishanina, James Woo, Aaron Diaz, Shawn L. Hervey-Jumper, Susan Marina Chang, Joanna J Phillips, Arun P Wiita, Christopher A. Klebanoff, Joseph F Costello, Hideo Okada
          </td>
          <td>2025-02-19</td>
          <td>Nature</td>
          <td>1</td>
          <td>16</td>
        </tr>

        <tr id="A one-two punch sequential regimen of senescence-inducing agents followed by senolytic drugs has emerged as a novel therapeutic strategy in cancer. Unfortunately, cancer cells undergoing therapy-induced senescence (TIS) vary widely in their sensitivity to senotherapeutics, and companion diagnostics to predict the response of TIS cancer cells to a specific senolytic drug are lacking. Here, we hypothesized that the ability of the BH3 profiling assay to functionally measure the mitochondrial priming state—the proximity to the apoptotic threshold—and the dependencies on pro-survival BCL-2 family proteins can be exploited to inform the sensitivity of TIS cancer cells to BH3-mimetics. Replicative, mitotic, oxidative, and genotoxic forms of TIS were induced in p16-null/p53-proficient, BAX-deficient, and BRCA1-mutant cancer cells using mechanistically distinct TIS-inducing cancer therapeutics, including palbociclib, alisertib, doxorubicin, bleomycin, and olaparib. When the overall state of mitochondrial priming and competence was determined using activator peptides, the expected increase in overall mitochondrial priming was an exception rather than a generalizable feature across TIS phenotypes. A higher level of overall priming paralleled a higher sensitivity of competent TIS cancer cells to BCL-2/BCL-xL- and BCL-xL-targeted inhibitors when comparing TIS phenotypes among themselves. Unexpectedly, however, TIS cancer cells remained equally or even less overally primed than their proliferative counterparts. When sensitizing peptides were used to map dependencies on anti-apoptotic BCL-2 family proteins, competent TIS cancer cells appeared to share a dependency on BCL-xL. Furthermore, regardless of senescence-inducing therapeutic, stable/transient senescence acquisition, or genetic context, all TIS phenotypes shared a variable but significant senolytic response to the BCL-xL-selective BH3 mimetic A1331852. These findings may help to rethink the traditional assumption of the primed apoptotic landscape of TIS cancer cells. BCL-xL is a conserved anti-apoptotic effector of the TIS BCL2/BH3 interactome that can be exploited to maximize the efficacy of “one-two punch” senogenic-senolytic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fde2da4d883f4df39e92d6849e05d7a212f9e15d" target='_blank'>
              Mitochondrial priming and response to BH3 mimetics in “one-two punch” senogenic-senolytic strategies
              </a>
            </td>
          <td>
            Júlia López, Àngela Llop-Hernández, S. Verdura, Eila Serrano-Hervás, Eva Martinez-Balibrea, J. Bosch-Barrera, E. Teixidor, E. López-Bonet, B. Martín-Castillo, Josep Sardanyés, Tomás Alarcón, Ruth Lupu, E. Cuyás, Javier A. Menendez
          </td>
          <td>2025-03-07</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Background Gastric cancer (GC) remains a leading cause of cancer-related mortality, with over one million new cases and 769,000 deaths reported in 2020. Despite advancements in chemotherapy, surgery, and targeted therapies, delayed diagnosis due to overlooked early symptoms leads to poor prognosis. Methods We integrated bulk RNA sequencing and single-cell RNA sequencing datasets from TCGA, GEO, and OMIX001073, employing normalization, batch effect correction, and dimensionality reduction methods to identify key cell populations associated with GC invasion and epithelial-mesenchymal transition (EMT), as well as analyze the tumor immune microenvironment. Results Our analysis identified the MUC5AC+ malignant epithelial cell cluster as a significant player in GC invasion and EMT. Cluster 1, representing this cell population, exhibited higher invasion and EMT scores compared to other clusters. Survival analysis showed that high abundance in cluster 0 correlated with improved survival rates (P=0.012), whereas cluster 1 was associated with poorer outcomes (P=0.045). A prognostic model highlighted ANXA5 and GABARAPL2 as two critical genes upregulated in GC tumors. High-risk patients demonstrated increased immune cell infiltration and worse prognosic. Analysis of tumor mutation burden (TMB) indicated that patients with low TMB in the high-risk group had the worst prognosis. Wet-lab validation experiments confirmed the oncogenic role of ANXA5, showing its facilitation of cell proliferation, invasion, and migration while suppressing apoptosis. Conclusion This study offers novel insights into the subpopulations of malignant epithelial cells in GC and their roles in tumor progression. It provides a prognostic model and potential therapeutic targets to combat GC, contributing crucial understanding to the fundamental mechanisms of drug resistance in gastrointestinal cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f9fbc569ffcfa0009c2b9e523878c6293cec12b" target='_blank'>
              Integrative analysis of single-cell and bulk RNA sequencing reveals the oncogenic role of ANXA5 in gastric cancer and its association with drug resistance
              </a>
            </td>
          <td>
            Deng-Gang Chen, Peng Zhang, Li Gong, Hailang Wei, Guanghui Yu, Tingting Zhang, Chen Bai
          </td>
          <td>2025-03-07</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc1c1d4b6b33d505126dfa7a012fa59e98c2b420" target='_blank'>
              Abstract A094: Single-cell Transcriptomics Unveils Novel Regulators of MHC Expression: Implications for Cancer Immunotherapy
              </a>
            </td>
          <td>
            M. Gondal, Yi Bao, R. Mannan, Jing Hu, A. Chinnaiyan, Marcin P. Cieslik
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6192f93a04789d7b2aabf4cf16ca2d7d42091ce4" target='_blank'>
              Innovative CRISPR-Cas9 enhances cell rejuvenation, addressing disease
              </a>
            </td>
          <td>
            Linfeng Zhang
          </td>
          <td>2025-03-20</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Prostate cancer (PCa) is a significant global health concern, with its incidence and mortality rates projected to rise due to population aging. In this study, we utilized PCa transcriptome data from public databases and applied bioinformatics methods to identify three prognostic genes (CDC20, RAD51, and TTK) related to centrosome duplication in PCa. CDC20 is involved in cell cycle regulation, RAD51 in deoxyribonucleic acid double-strand break repair, and TTK in spindle assembly checkpoint function and cell proliferation. We constructed a risk model and a nomogram model, both demonstrating moderate to good predictive performance with area under the curve values ranging from 0.611 to 0.765 at different time points. Gene set enrichment analysis revealed that these genes were enriched in 64 pathways, including the cell cycle pathway, which is dysregulated in cancer. Furthermore, we analyzed the immune microenvironment and identified 13 differential immune cells and 13 differential immune checkpoints between high- and low-risk groups, providing insights into potential immunotherapy targets for PCa. In conclusion, this study contributes to a deeper understanding of PCa pathogenesis and lays important theoretical and experimental foundations for developing new diagnostic markers and treatment strategies. Future research requires more clinical samples and continued monitoring of the mechanism of these genes in PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8581bf4b2f1cfe894fadfd64d88a1382b8cc282" target='_blank'>
              Prognostic Value of Centrosome Replication-Related Genes in Prostate Cancer Based on Transcriptomic and Mendelian Randomization
              </a>
            </td>
          <td>
            Qizhong Lu, Yufan Wu, Qiwei Yu, Jun Ouyang
          </td>
          <td>2025-03-01</td>
          <td>American Journal of Men's Health</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Tumor-associated macrophages (TAMs) play pivotal roles in innate immunity and contribute to the advancement of lung cancer. We aimed to identify novel TAM-related biomarkers and significance of macrophage infiltration in lung adenocarcinoma (LUAD) through an integrative analysis of single-cell RNA-sequencing (scRNA-seq) data. To describe the cell atlas and construct a novel prognostic signature in LUAD. Methods The gene signature linked to TAMs was identified utilizing Scanpy from the scRNA-seq dataset GSE131907. Subsequent analysis involved evaluating the expression levels of these genes, their potential molecular mechanisms, and prognostic significance in LUAD using data from The Cancer Genome Atlas (TCGA) and Gene Expression Omnibus (GEO) databases. We also constructed a risk score models through LASSO Cox regression for these genes. The underlying mechanism was further elucidated through the application of GSEA, ESTIMATE, TIDE, and other bioinformatic algorithms. Results Single-cell atlas was described by analyze 29 scRNA-seq samples from 19 LUAD patients. The TAMs-related gene signature (TGS) was identified as an independent prognostic factor by LASSO Cox regression analysis using differential expression genes (DEGs) derived from pro- and anti-inflammatory macrophage cells. Risk score model including nine TAMs-related genes (FOSL1, ZNF697, ADM, UBE2S, TICAM1, S100P, BIRC3, TLE1, and DEFB1) were obtained for prognosis construction. Moreover, the risk model underwent additional validation in four external GEO cohorts: GSE31210, GSE72094, GSE26939, and GSE30219. Interestingly, TGS-high tumors revealed enrichments in TGF-β signaling and hypoxia pathways, which shown low immune infiltration and immunosuppression by ESTIMATE and TIDE algorithm. The TGS-high risk group exhibited lower richness and diversity in the T-cell receptor (TCR) repertoire. Conclusion This study introduces a novel TGS score developed through LASSO Cox regression analysis, utilizing DEGs in pro- and anti-inflammatory macrophage cells. High TGS tumors exhibited enrichment in TGF-β signaling and hypoxia pathways, suggesting their potential utility in predicting prognosis and immune responses in patients with LUAD. These results offer promising implications for the development of therapeutic strategies for LUAD. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-01834-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab542107a71d04ec4e797429aa65c1f0ffcb1f72" target='_blank'>
              Single cell transcriptomic analysis reveals tumor immune infiltration by macrophage cells gene signature in lung adenocarcinoma
              </a>
            </td>
          <td>
            Xiaotong Guo, Youjun Deng, Wenjun Jiang, Heng Li, Yisheng Luo, Huachuan Zhang, Hao Wu
          </td>
          <td>2025-03-03</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f250f1f9ca61e747d3f45521c173e479dc5a6f63" target='_blank'>
              Chemical enhancement of DNA repair in aging
              </a>
            </td>
          <td>
            J. C. Macedo, Maria M da Silva, Joana M. Magalhães, Carlos Sousa-Soares, M. Inês Ala, Mafalda Galhardo, Rui Ribeiro, Monika Barroso-Vilares, Mafalda Sousa, Paula Sampaio, E. Logarinho
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="The expression of cytosolic aldehyde dehydrogenases (ALDHs), which mediate the last step in the pathway of the synthesis of all-trans retinoic acid, is dysregulated in various types of human cancer, and has been associated with the development of cancer stem cells (CSCs) in solid tumors and hematological malignancies. CSCs are considered a minor fraction of cancer cells with the capacity to initiate neoplastic tumors. ALDH1A1 serves a crucial role in the emergence of the CSC phenotype, induces the malignant behavior of cancer cells and promotes treatment resistance. Notably, ALDH1A1-induced therapy resistance is not exclusive to just one group of drugs, but affects diverse types of drugs that use different mechanisms to kill cells. This diversity of drug resistance-inducing effects is associated with the stemness-supporting functions of ALDH1A1. The inhibition of ALDH1A1 activity using chemicals or the depletion of ALDH1A1 via genetic approaches, such as the use of small interfering RNA, can overcome diverse pathways of therapy resistance. In the context of breast cancer, it is critical that only a fraction of malignant cells are expected to manifest stem-like features, which include increased expression of ALDH1A1. From the angle of disease prognosis, the extent of the association of ALDH1A1 with increased malignant behavior and drug resistance remains to be determined through the application of cutting-edge methods that detect the expression of tracked biomarkers within tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/378f86e62d74f35111b7a740e827c0ac03143fb8" target='_blank'>
              ALDH1A1 in breast cancer: A prospective target to overcome therapy resistance (Review)
              </a>
            </td>
          <td>
            L. Varışlı, P. Zoumpourlis, Demetrios A. Spandidos, V. Zoumpourlis, S. Vlahopoulos
          </td>
          <td>2025-03-04</td>
          <td>Oncology Letters</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Individual aging is a complex biological process involving multiple levels, with molecular changes existing in heterogeneity across different cell types and tissues, being regulated by both internal and external factors. Traditional senescence markers, including p16, cell morphological changes, and cell cycle arrest, can only partially reflect the complexity of senescence. Single-cell omics technology facilitates the integration of multi-faceted data, including gene expression profiles, spatial dynamics, chromatin accessibility and metabolic pathways. This comprehensive approach enhances the development of biomarkers, granting us a more profound insight into the heterogeneity inherent within senescent cell populations. In this review, we summarize the application of single cell multi-omics approaches in analyzing senescence mechanisms and potential intervention targets from the perspectives of transcriptomics, epigenetics, metabolomics, and proteomics, explore the potential of developing new senescence markers at the cellular level using machine learning algorithms and artificial intelligence in bioinformatics analysis. Finally, we further discuss the challenges and prospective trajectories within this research domain to provide a more comprehensive perspective on dissecting the regulatory networks of senescence cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb05feca31b360e2a4184cb1c6c1adbe6ecbd8d5" target='_blank'>
              Towards Precision Aging Biology: Single-Cell Multi-Omics and Advanced AI-Driven Strategies.
              </a>
            </td>
          <td>
            Sijia Xie, Xinwei Luo, Feitong Hong, Yijie Wei, Yuduo Hao, Xueqin Xie, Xiaolong Li, Guangbo Xie, Fuying Dao, Hao Lyu
          </td>
          <td>2025-03-17</td>
          <td>Aging and disease</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Gastric cancer (GC) remains a global health challenge due to its high mortality rate and the lack of specific diagnostic methods. Tumor heterogeneity significantly hinders effective treatment, necessitating advanced techniques to dissect its complexity. Artificial intelligence (AI) offers transformative potential in uncovering intricate cellular dynamics and gene regulatory networks. This study leverages single-cell RNA sequencing (scRNA-seq) combined with transcriptome analysis, enhanced by AI-driven analytics, to explore the tumor microenvironment and identify novel prognostic markers and therapeutic targets in GC. Methods scRNA-seq and transcriptome datasets of GC patients were analyzed using AI-enhanced methodologies to unravel tumor heterogeneity and microenvironmental dynamics. Macrophage subsets were identified as critical components within the GC microenvironment. High-variance gene screening in these subsets pinpointed apolipoprotein E (APOE) as a hub gene. Experimental validation of APOE expression in GC samples and functional studies in GC cell lines were conducted. Results Bioinformatics and AI-enabled analyses confirmed the immunosuppressive role of APOE in GC. An immune-related survival model was developed to predict immunotherapy responses and patient prognoses. Mechanistically, APOE was found to induce immunosuppression through M2 macrophage polarization, promoting tumor progression and leading to poorer outcomes in GC patients. Conclusion This study highlights the potential of AI-driven approaches in elucidating the role of APOE in GC progression. APOE’s regulatory effects on M2 macrophages underscore its value as a prognostic marker and therapeutic target, paving the way for precision medicine in GC management. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-02100-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/987e5761ad8244faf71073710e21f1e10a5477f1" target='_blank'>
              Artificial intelligence-driven integration of single-cell RNA sequencing and transcriptome analysis to decipher APOE’s role in gastric cancer prognosis and therapy
              </a>
            </td>
          <td>
            Ni Li, Jinquan Lin, Chunyu Wang, Xiao Ran, Zhi Zhao
          </td>
          <td>2025-03-13</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Oncogene amplification is a key driver of cancer pathogenesis. Both breakage fusion bridge (BFB) cycles and extrachromosomal DNA (ecDNA) can lead to high oncogene copy numbers, but the impact of BFB amplifications on intratumoral heterogeneity, treatment response, and patient survival remains poorly understood due to detection challenges with DNA sequencing. We introduce an algorithm, OM2BFB, designed to detect and reconstruct BFB amplifications using optical genome mapping (OGM). OM2BFB demonstrates high precision (>93%) and recall (92%) in identifying BFB amplifications across cancer cell lines, patient-derived xenograft models, and primary tumors. Comparisons using OGM reveal that BFB detection with our AmpliconSuite toolkit for short-read sequencing also achieves high precision, though with reduced sensitivity. We identify 371 BFB events through whole genome sequencing of 2557 primary tumors and cancer cell lines. BFB amplifications are prevalent in cervical, head and neck, lung, and esophageal cancers, but rare in brain cancers. Genes amplified through BFB exhibit lower expression variance, with limited potential for regulatory adaptation compared to ecDNA-amplified genes. Tumors with BFB amplifications (BFB(+)) show reduced structural heterogeneity in amplicons and delayed resistance onset relative to ecDNA(+) tumors. These findings highlight ecDNA and BFB amplifications as distinct oncogene amplification mechanisms with differing biological characteristics, suggesting distinct avenues for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87febc9c07e2846c41020472ac578cf946a0ca8e" target='_blank'>
              Breakage fusion bridge cycles drive high oncogene number with moderate intratumoural heterogeneity
              </a>
            </td>
          <td>
            Siavash Raeisi Dehkordi, I. Wong, Jing Ni, J. Luebeck, Kaiyuan Zhu, Gino Prasad, Lena Krockenberger, Guanghui Xu, Biswanath Chowdhury, Utkrisht Rajkumar, Ann Caplin, Daniel Muliaditan, Aditi Gnanasekar, C. Coruh, Q. Jin, Kristen Turner, Shu Xian Teo, A. W. Pang, Ludmil B. Alexandrov, Christelle En Lin Chua, F. Furnari, J. Maciejowski, Thomas G Paulson, Julie A. Law, Howard Y. Chang, Feng Yue, Ramanuj DasGupta, Jean J. Zhao, P. Mischel, V. Bafna
          </td>
          <td>2025-02-10</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Cancer chemoresistance presents a challenge in oncology, often leading to treatment failure and disease progression. CD44, a multifunctional cell surface glycoprotein, has garnered attention for its involvement in various aspects of cancer biology. Through alternative splicing, CD44 can form isoforms with the inclusion of only standard exons, typical for normal tissue, or with the addition of variant exons, frequently expressed in cancer tissue and associated with chemoresistance. The functions of CD44 involved in regulation of cancer signaling pathways are being actively studied, and the significance of specific variant exons in modulating cell death pathways, central to the response of cancer cells to chemotherapy, begins to become apparent. This review provides a comprehensive analysis of the association of CD44 variant exons/total CD44 with clinical outcomes of patients undergoing chemotherapy. The role of CD44 variant exons v6, v9 and others with a significant effect on patient chemotherapy outcomes by means of key cellular death pathways such as apoptosis, ferroptosis and autophagy modulation is further identified, and their impact on drug resistance is highlighted. An overview of clinical trials aimed at targeting variant exon-containing isoforms is provided, and possible directions for further development of CD44-targeted therapeutic strategies are discussed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c896fa2073872384fe9590cc27feab2b38f88e9" target='_blank'>
              CD44 variant exons induce chemoresistance by modulating cell death pathways
              </a>
            </td>
          <td>
            Maria Yanova, Evgeniya Stepanova, Diana Maltseva, Alexander Tonevitsky
          </td>
          <td>2025-03-06</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c906c5a3862aab5cd84c1716303e1ffc00c0344c" target='_blank'>
              NSD3: A Promising Target for Cancer Therapy.
              </a>
            </td>
          <td>
            Ting Huang, Bowen Zhang, Yifan Yang, Qiong Lin, Genbao Shao
          </td>
          <td>2025-03-26</td>
          <td>Cell biochemistry and function</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="One-carbon metabolism (1CM) has been reported to promote cancer progression across various malignancies. While 1CM is critical for cell proliferation by enhancing nucleotide synthesis, its physiological roles within different cell types in the tumor immune microenvironment (TIME) still remain unclear. In this study, we analyzed bulk-RNA sequencing and single-cell RNA sequencing (scRNA-seq) data from lung adenocarcinoma (LUAD) patients to elucidate the functional roles of 1CM within the TIME. Moreover, we examined scRNA-seq data from patients treated with immunotherapy across various cancers, including LUAD, glioblastoma, renal cell carcinoma, colorectal cancer, and triple-negative breast cancer. Compared to other cell types, 1CM gene profiles are significantly enriched in a specific subset of T cells. Intriguingly, these high-1CM T cells are identified as proliferative intermediate exhausted T cells (Texint). Furthermore, these proliferative Texint received the most robust CD137 signaling. Consistently, analysis of scRNA-seq data from LUAD patients undergoing anti-PD1 immunotherapy demonstrated that proliferative Texint exhibited higher 1CM scores and increased CD137 signaling. This observation was particularly pronounced in non-responders to immunotherapy, where the Texint population was significantly expanded. We further established that 1CM is a prominent signaling pathway in proliferative Texint in patients resistant to immunotherapy across multiple cancer types. Collectively, we identify CD137 signaling as a distinctive pathway in proliferative Texint of LUAD patients who do not respond to immunotherapy. These findings propose that targeting 1CM may represent a novel therapeutic strategy to enhance the efficacy of immunotherapy by mitigating Texint proliferation in diverse cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a799bb85faaaac3fa04136a525f7e0a65fbc4560" target='_blank'>
              One-carbon metabolism is distinct metabolic signature for proliferative intermediate exhausted T cells of ICB-resistant cancer patients
              </a>
            </td>
          <td>
            Ye-Chan Park, Yesueng Hwang, Jae Woong Jeong, Chae Min Lee, Minki Kim, Sugyeong Jo, Seyeon Joo, Nahee Hwang, Sungsoon Fang
          </td>
          <td>2025-02-14</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background The tumor suppressor p53 (Trp53), also known as p53, is the most commonly mutated gene in cancer. Canonical p53 DNA damage response pathways are well characterized and classically thought to underlie the tumor suppressive effect of p53. Challenging this dogma, mouse models have revealed that p53-driven apoptosis and cell cycle arrest are dispensable for tumor suppression. Here, we investigated the inverse context of a p53 mutation predicted to drive the expression of canonical targets but is detected in human cancer. Methods We established a novel mouse model with a single base pair mutation (GAG>GAT, p53E221D) in the DNA-Binding domain that has wild-type function in screening assays, but is paradoxically found in human cancer in Li-Fraumeni syndrome. Using mouse p53E221D and the analogous human p53E224D mutants, we evaluated expression, transcriptional activation, and tumor suppression in vitro and in vivo. Results Expression of human p53E224D from cDNA translated to a fully functional p53 protein. However, p53E221D/E221D RNA transcribed from the endogenous locus is mis-spliced resulting in nonsense-mediated decay. Moreover, fibroblasts derived from p53E221D/E221D mice do not express a detectable protein product. Mice homozygous for p53E221D exhibited increased tumor penetrance and decreased life expectancy compared to p53WT/WT animals. Conclusions Mouse p53E221D and human p53E224D mutations lead to splice variation and a biologically relevant p53 loss of function in vitro and in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97a5ab888592bac1b40fc9b15ac14955958bd526" target='_blank'>
              Mis-splicing drives loss of function of p53E224D point mutation
              </a>
            </td>
          <td>
            Ian C. Lock, Nathan H Leisenring, Warren Floyd, Eric S. Xu, Lixia Luo, Yan Ma, Erin C. Mansell, Diana M. Cardona, Chang-Lung Lee, D. Kirsch
          </td>
          <td>2025-03-05</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Abstract Many cancer risk variants are located within enhancer regions and lack sufficient molecular interpretation. Here, we constructed the first comprehensive atlas of enhancer RNA (eRNA)‐mediated genetic effects from 28 033 RNA sequencing samples across 11 606 individuals, identifying 21 073 eRNA quantitative trait loci (eRNA‐QTLs) significantly associated with eRNA expression. Mechanistically, eRNA‐QTLs frequently altered binding motifs of transcription factors. In addition, 28.48% of cancer risk variants are strongly colocalized with eRNA‐QTLs. A pan‐cancer eRNA‐based transcriptome‐wide association study is conducted across 23 major cancer types, identifying 626 significant cancer susceptibility eRNAs predicted to modulate cancer risk via eRNA, from which 54.90% of the eRNA target genes are overlooked by traditional gene expression studies, and most are essential for cancer cell proliferation. As proof of principle validation, the enhancer functionality of two newly identified susceptibility eRNAs, CCND1e and SNAPC1e, is confirmed through CRISPR inhibition and shRNA‐mediated knockdown, resulting in a marked decrease in the expression of their respective target genes, consequently suppressing the proliferation of prostate cancer cells. The study underscores the essential role of eRNA in unveiling new cancer susceptibility genes and establishes a strong framework for enhancing our understanding of human cancer etiology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d3dbe877352b4ba15bd8f60bc6801dc3a4efef8" target='_blank'>
              Enhancer RNA Transcriptome‐Wide Association Study Reveals a Distinctive Class of Pan‐Cancer Susceptibility eRNAs
              </a>
            </td>
          <td>
            Wenyan Chen, Zeyang Wang, Yinuo Wang, Jianxiang Lin, Shuxin Chen, Hui Chen, Xuelian Ma, X. Zou, Xing Li, Y. Qin, Kewei Xiong, Xixian Ma, Qi Liao, Y. Qiao, Lei Li
          </td>
          <td>2025-02-14</td>
          <td>Advanced Science</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Epigenomic modifications—such as DNA methylation, histone acetylation, and histone methylation—and their implications in tumorigenesis, progression, and treatment have emerged as a pivotal field in cancer research. Tumors undergo metabolic reprogramming to sustain proliferation and metastasis in nutrient-deficient conditions, while suppressing anti-tumor immunity in the tumor microenvironment (TME). Concurrently, immune cells within the immunosuppressive TME undergo metabolic adaptations, leading to alterations in their immune function. The complicated interplay between metabolites and epigenomic modulation has spotlighted the significance of epigenomic regulation in tumor immunometabolism. In this review, characteristics of the epigenomic modification associated with tumors are systematically summarized alongside with their regulatory roles in tumor metabolic reprogramming and immunometabolism. Classical and emerging approaches are delineated to broaden the boundaries of research on the crosstalk research on the crosstalk between tumor immunometabolism and epigenomics. Furthermore, we discuss potential therapeutic strategies that target tumor immunometabolism to modulate epigenomic modifications, highlighting the burgeoning synergy between metabolic therapies and immunotherapy as a promising avenue for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b7b18f960568f0266743f9b084b520cf67c3989" target='_blank'>
              Bridging epigenomics and tumor immunometabolism: molecular mechanisms and therapeutic implications
              </a>
            </td>
          <td>
            Xiaowen Xie, Weici Liu, Zhiyuan Yuan, Hanqing Chen, Wenjun Mao
          </td>
          <td>2025-03-08</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c04a318a2957547a076d2f3d531ff4b2b92de22f" target='_blank'>
              PARP inhibitors elicit distinct transcriptional programs in homologous recombination competent castration-resistant prostate cancer
              </a>
            </td>
          <td>
            Moriah Cunningham, Jasibel Vasquez-Gonzalez, Samantha M. Barnada, Salome Tchotorlishvili, Latese Jones, Hailey Shankle, Tessa Mulders, Namra Ajmal, C. Solomides, Ayesha A. Shafi, Leonard G. Gomella, Wm Kevin Kelly, Steven B. McMahon, M. Schiewer
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH), characterized by the expansion of hematopoietic stem and progenitor cells harboring somatic mutations, has emerged as a significant age-related phenomenon with profound implications for human health. While initially recognized in the 1960s, recent technological advances have revealed its complex nature and widespread prevalence, affecting up to 84% of individuals aged ≥ 70 years. The clinical significance of CH extends beyond its well-established role as a precursor to hematological malignancies, encompassing its association with cardiovascular diseases, chronic kidney disease, and other non-malignant disorders. This comprehensive review synthesizes the current understanding of CH, focusing on recent advances in genetic and molecular mechanisms, particularly the roles of commonly mutated genes such as DNMT3A, TET2, and ASXL1. We address the emerging distinction between myeloid and lymphoid CH, their differential impacts on disease progression, and the complex interplay between CH and inflammation. Special attention is given to newly identified genetic determinants of clonal expansion rates and their implications for disease progression. The review also examines the revolutionary concept of passenger-approximated clonal expansion rate and its utility in understanding CH dynamics. Furthermore, we discuss therapeutic strategies targeting inflammatory pathways and their potential in mitigating CH-associated complications. By integrating recent findings from genetic, molecular, and clinical studies, this review provides a framework for understanding CH as a systemic condition and highlights promising directions for therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7375f0846811e9250a784d2b67a30ac0f6b0da37" target='_blank'>
              Clonal hematopoiesis: elements associated with clonal expansion and diseases
              </a>
            </td>
          <td>
            Gangpyo Ryu, Youngil Koh, S. Jaiswal, S. Yoon
          </td>
          <td>2025-03-13</td>
          <td>Blood Research</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Recombinant adeno-associated viral vectors (rAAV) are a powerful tool for gene delivery but have a limited DNA carrying capacity. Efforts to expand this genetic payload have focused on engineering the vector components, such as dual trans-splicing vectors which double the delivery size by exploiting the natural concatenation of rAAV genomes in host nuclei. We hypothesized that inefficient dual vector transduction could be improved by modulating host factors which affect concatenation. Since factors mediating concatenation are not well defined, we performed a genome-wide screen to identify host cell regulators. We discover that Homologous Recombination (HR) is inhibitory to dual vector transduction. We demonstrate that depletion or inhibition of HR factors BRCA1 and Rad51 significantly increase reconstitution of a large split transgene by increasing both concatenation and expression from rAAVs. Our results define roles for DNA damage repair in rAAV transduction and highlight the potential for pharmacological intervention to increase genetic payload of rAAV vectors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de6693e1cc9fcaaa6e44ac852bdcdb47fc66775d" target='_blank'>
              Double-strand break repair pathways differentially affect processing and transduction by dual AAV vectors
              </a>
            </td>
          <td>
            Anna C Maurer, Brian Benyamini, O. Whitney, Vinson B. Fan, Claudia Cattoglio, Djem U. Kissiov, G. Dailey, X. Darzacq, Matthew D Weitzman, Robert Tjian
          </td>
          <td>2025-02-11</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eed8e001dc646667003aedc35613a33f0517e9a4" target='_blank'>
              Common variation in meiosis genes shapes human recombination phenotypes and aneuploidy risk
              </a>
            </td>
          <td>
            S. A. Carioscia, A. Biddanda, M. R. Starostik, X. Tang, E. R. Hoffmann, Z. P. Demko, R. C. McCoy
          </td>
          <td>2025-04-04</td>
          <td>None</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract For genome editing, the use of CRISPR ribonucleoprotein (RNP) complexes is well established and often the superior choice over plasmid-based or viral strategies. RNPs containing dCas9 fusion proteins, which enable the targeted manipulation of transcriptomes and epigenomes, remain significantly less accessible. Here, we describe the production, delivery, and optimization of second generation CRISPRa RNPs (dRNPs). We characterize the transcriptional and cellular consequences of dRNP treatments in a variety of human target cells and show that the uptake is very efficient. The targeted activation of genes demonstrates remarkable potency, even for genes that are strongly silenced, such as developmental master transcription factors. In contrast to DNA-based CRISPRa strategies, gene activation is immediate and characterized by a sharp temporal precision. We also show that dRNPs allow very high-target multiplexing, enabling undiminished gene activation of multiple genes simultaneously. Applying these insights, we find that intensive target multiplexing at single promoters synergistically elevates gene transcription. Finally, we demonstrate in human stem and differentiated cells that the preferable features of dRNPs allow to instruct and convert cell fates efficiently without the need for DNA delivery or viral vectors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ae64466d50f495675fc82663a1b7dac6851bcc6" target='_blank'>
              Efficient DNA- and virus-free engineering of cellular transcriptomic states using dCas9 ribonucleoprotein (dRNP) complexes
              </a>
            </td>
          <td>
            Tobias Schmidt, Maximilian F. Wiesbeck, Luisa Eg er t, Thi-T ram T ruong, Anna Danese, Lukas Voshagen, Simon Imhof, Matilde Iraci Borgia, Deeksha, Andrea M Neuner, Anna Köf er le, Ar ie Geer lof, André Santos, Dias Mourão, Stefan H Stricker
          </td>
          <td>2025-03-20</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="To gain insight into biological mechanisms that cause resistance to DNA damage, we performed parallel pooled genetic CRISPR-Cas9 screening for survival in high risk HNSCC subtypes. Surprisingly, and in addition to ATM, DNAPK, and NFKB signaling, JAK1 was identified as a driver of tumor cell radiosensitivity. Knockout of JAK1 in HNSCC increases cell survival by enhancing the DNA damage-induced G2 arrest, and both knockout and JAK1 inhibition with abrocitinib prevent subsequent formation of radiation-induced micronuclei. Loss of JAK1 function does not affect canonical CDK1 signaling but does reduce activation of PLK1 and AURKA, kinases that regulate both G2 and M phase progression. Correspondingly, JAK1 KO was found to cause mitotic defects using both EdU labeling and live cell imaging techniques. Given this insight, we evaluated Kif18a inhibition as an approach to exacerbate mitotic stress and enhance the efficacy of radiation. These studies establish Kif18a inhibition as a novel strategy to counteract therapeutic resistance to DNA damage mediated by G2 cell cycle arrest.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eddfa8407309bcd878a23af220d2c4449530b55c" target='_blank'>
              Loss of JAK1 Function Causes G2/M Cell Cycle Defects Vulnerable to Kif18a Inhibition
              </a>
            </td>
          <td>
            Vanessa Kelley, Marta Baro, William Gasperi, Nicholas Ader, Hannah Lea, Hojin Lee, C. Phoomak, Lilian Kabeche, Megan C. King, Joseph N. Contessa
          </td>
          <td>2025-02-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ea4c6f0326f442ed8f79641ba270dbbfd88e4e5" target='_blank'>
              Locking the gates of immortality: targeting alternative lengthening of telomeres (ALT) pathways.
              </a>
            </td>
          <td>
            Apurwa Mishra, Trupti N Patel
          </td>
          <td>2025-02-18</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="MYC-driven (MYC+) cancers are aggressive and often fatal. MYC dysregulation is a key event in these cancers, but overexpression of MYC alone is not always enough to cause cancer. Plasmocytoma Variant Translocation 1 (PVT1), a long non-coding RNA (lncRNA) adjacent to MYC on chromosome 8 is a rearrangement hotspot in many MYC+ cancers. In addition to being co-amplified with MYC, the genomic rearrangement at PVT1 involves translocation, which has had obscure functional consequences. We report that translocation at the PVT1 locus cause asymmetric enrichment of 5’-PVT1 and loss of 3’-PVT1. Despite being classified as a non-coding RNA, the retained 5’ region of PVT1 generates a circular RNA (CircPVT1) that codes for the novel peptide we call Firefox (FFX). FFX augments AKT signaling and synergistically activates MYC and mTORC1 in these cells. Further, the 3’ end of PVT1, which is lost during the translocation, codes for a tumor-suppressing micropeptide we named as Honeybadger (HNB). We demonstrate that HNB interacts with KRAS and disrupts the activation of KRAS effectors. Loss of HNB leads to activation of RAS/MAPK signaling pathway, and enhances MYC stability by promoting phosphorylation of MYC at Ser62. These findings identify PVT1 as a critical node that synchronizes MYC, AKT, and RAS-MAPK activities in cancer. Our study thus identifies a key mechanism by which rearrangements at the PVT1 locus activate additional oncogenic pathways that synergize with MYC to exacerbate the aggressiveness of MYC+ cancers. This newfound understanding explains the poor prognosis associated with MYC+ cancers and offers potential therapeutic targets that could be leveraged in treatment strategies for these cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e076955c4c71d3d95422db7f1fcfc733c5d61e3" target='_blank'>
              Synergistic RAS-MAPK and AKT Activation in MYC-Driven Tumors via Adjacent PVT1 Rearrangements
              </a>
            </td>
          <td>
            Ashutosh Tiwari, Utkarsha Paithane, Jordan Friedlein, Kojiro Tashiro, Olivier Saulnier, Karina Barbosa, Quang Trinh, Bryan Hall, Shrawantee Saha, Aditi Soni, Takuma Nakashima, Andrey Bobkov, Lynn Fujimoto, Rabi Murad, Svetlana Maurya, Mayank Saraswat, Shahab Sarmashghi, Joshua T. Lange, Sihan Wu, M. Masihi, Srija Ghosh, Gazal Hemmati, Owen S. Chapman, Liam D Hendrikse, Brian James, J. Luebeck, Tanja Eisemann, Theophilos Tzaridis, Deepak Rohila, Robyn Leary, Jyotika Varshney, Badrinath R. Konety, S. Dehm, Yasuhiko Kawakami, R. Beroukhim, D. Largaespada, Lincoln Stein, Lukas Chavez, Hiromichi Suzuki, William A. Weiss, Jianhua Zhao, Aniruddha J. Deshpande, R. Wechsler-Reya, Michael D Taylor, Anindya Bagchi
          </td>
          <td>2025-02-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>148</td>
        </tr>

        <tr id="Glucose metabolism reprogramming provides significant insights into the development and progression of malignant tumors. This study aims to explore the temporal-spatial evolution of the glucose metabolism in HCC using single-cell sequencing and spatial transcriptomics (ST), and validates G6PD as a potential therapeutic target for HCC.We collected single-cell sequencing data from 7 HCC and adjacent non-cancerous tissues from the GSE149614 database, and ST data from 4 HCC tissues from the HRA000437 database. Pseudotime analysis was performed on the single-cell data, while ST data was used to analyze spatial metabolic activity. High-throughput sequencing and experiments, including wound healing, CCK-8, and transwell assays, were conducted to validate the role and regulatory mechanisms of G6PD in HCC.Our study identified a progressive upregulation of PPP-related genes during tumorigenesis. ST analysis revealed elevated PPP metabolic scores in the central and intermediate tumor regions compared to the peripheral zones. High-throughput sequencing and experimental validation further suggested that G6PD-mediated regulation of HCC cell proliferation, migration, and invasion is likely associated with glutathione metabolism and ROS production. Finally, Cox regression analysis cofirmed G6PD as an independent prognostic factor for overall survival in HCC patients.Our study provides novel insights into the changes in glucose metabolism in HCC from both temporal and spatial perspectives. We experimentally demonstrated that G6PD regulates proliferation, migration, and invasion in HCC and propose G6PD as a prognostic marker and therapeutic metabolic target for the HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92d4060fcfc333045f5d44b00ea2e74dde3d1885" target='_blank'>
              Single-cell sequencing and spatial transcriptomics reveal the evolution of glucose metabolism in hepatocellular carcinoma and identify G6PD as a potential therapeutic target
              </a>
            </td>
          <td>
            Deyang Xi, Yinshuang Yang, Jiayi Guo, Mengjiao Wang, Xuebing Yan, Chunyang Li
          </td>
          <td>2025-03-25</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/faa65e1fd6a4ddcb57e0eaf994498d400b2eb5d2" target='_blank'>
              Tissue-specific mutagenesis from endogenous guanine damage is suppressed by Polκ and DNA repair
              </a>
            </td>
          <td>
            Yang Jiang, M. Przybilla, Linda Bakker, Foster C. Jacobs, Dylan Mckeon, Roxanne van der Sluijs, Juliette Wezenbeek, Joeri van Strien, Jeroen Willems, Alex E E Verkennis, Jamie Barnett, Adrian Baez Ortega, Federico Abascal, Peter W. Villalta, P. Knipscheer, I. Martincorena, S. Balbo, Juan Garaycoechea
          </td>
          <td>2025-03-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/055214dbdb29cbe5585cc333e89737a58f2ba504" target='_blank'>
              The Epigenetic Landscape: From Molecular Mechanisms to Biological Aging.
              </a>
            </td>
          <td>
            Rachel Evangelina, Subhashree Ganesan, Melvin George
          </td>
          <td>2025-03-17</td>
          <td>Rejuvenation research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Prognostic transcriptomic signatures for prostate cancer (PCa) often overlook the cellular origin of expression changes, an important consideration given the heterogeneity of the disorder. Current clinicopathological factors inadequately predict biochemical recurrence, a critical indicator guiding post-treatment strategies following radical prostatectomy. To address this, we conducted a meta-analysis of four large-scale PCa datasets and found 33 previously reported PCa-associated genes to be consistently up-regulated in prostate tumours. By analysing single-cell RNA-sequencing data, we found these genes predominantly as markers in epithelial cells. Subsequently, we applied 97 advanced machine-learning algorithms across five PCa cohorts and developed an 11-gene epithelial expression signature. This signature robustly predicted biochemical recurrence-free survival (BCRFS) and stratified patients into distinct risk categories, with high-risk patients showing worse survival and altered immune cell populations. The signature outperformed traditional clinical parameters in larger cohorts and was overall superior to published PCa signatures for BCRFS. By analysing peripheral blood data, four of our signature genes showed potential as biomarkers for radiation response in patients with localised cancer and effectively stratified castration-resistant patients for overall survival. In conclusion, this study developed a novel epithelial gene-expression signature that enhanced BCRFS prediction and enabled effective risk stratification compared to existing clinical- and gene-expression-derived prognostic tools. Furthermore, a set of genes from the signature demonstrated potential utility in peripheral blood, a tissue amenable to minimally invasive sampling in a primary care setting, offering significant prognostic value for PCa patients without requiring a tumour biopsy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0bf6f0e9c87d76cfdec8216eb4d7e07bdd3f760" target='_blank'>
              Integration of single-cell and bulk RNA-sequencing data reveals the prognostic potential of epithelial gene markers for prostate cancer.
              </a>
            </td>
          <td>
            Z. Mou, Lorna W Harries
          </td>
          <td>2025-02-19</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Overexpression of the H3K36 histone methyltransferase NSD2 in t(4;14) multiple myeloma (MM) is an early, oncogenic event, and understanding its impact on genomic organisation and expression is relevant to understanding MM biology. We performed epigenetic, transcriptional and phenotypic profiling of the t(4;14) KMS11 myeloma cell line and its isogenic translocation knock out (TKO) to characterise the sequelae of NSD2 overexpression. We found a marked global impact of NSD2 on gene expression and DNA organisation implicating cell identity genes; notably the early lymphocyte regulator, LAIR1 and MM cell surface markers, including CD38, a classical marker of plasma cells which was reduced in TKO cells. Plasma cell transcription factors such as PRDM1, IRF4 and XBP1 were unaffected, suggesting a downstream direct gene effect of NSD2 on cell identity. Changes in cell surface markers suggest an altered surface immunophenotype. Our findings suggest a role for NSD2 in maintaining MM cell identity, with potential implications for future therapeutic strategies based on targeting of NSD2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3900ea300ad687a680fc22372f0a01df92719adc" target='_blank'>
              NSD2-epigenomic reprogramming and maintenance of plasma cell phenotype in t(4;14) myeloma
              </a>
            </td>
          <td>
            Andrea Gunnell, Scott T. Kimber, R. Houlston, Martin Kaiser
          </td>
          <td>2025-03-21</td>
          <td>Oncotarget</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Genomic structural variants (SVs) are a major source of genetic diversity in humans. Here, through long-read sequencing of 945 Han Chinese genomes, we identify 111,288 SVs, including 24.56% unreported variants, many with predicted functional importance. By integrating human population-level phenotypic and multi-omics data as well as two humanized mouse models, we demonstrate the causal roles of two SVs: one SV that emerges at the common ancestor of modern humans, Neanderthals, and Denisovans in GSDMD for bone mineral density and one modern-human-specific SV in WWP2 impacting height, weight, fat, craniofacial phenotypes and immunity. Our results suggest that the GSDMD SV could serve as a rapid and cost-effective biomarker for assessing the risk of cisplatin-induced acute kidney injury. The functional conservation from human to mouse and widespread signals of positive natural selection suggest that both SVs likely influence local adaptation, phenotypic diversity, and disease susceptibility across diverse human populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/383a5425249b871df88f4291d3bc3dbd683e73f8" target='_blank'>
              Long-read sequencing of 945 Han individuals identifies structural variants associated with phenotypic diversity and disease susceptibility
              </a>
            </td>
          <td>
            J. Gong, Huiru Sun, Kaiyuan Wang, Yanhui Zhao, Yechao Huang, Qinsheng Chen, H. Qiao, Yang Gao, Jialin Zhao, Y. Ling, R. Cao, J. Tan, Qi Wang, Yanyun Ma, Jing Li, Jing Luo, Sijia Wang, Jiucun Wang, Guoqing Zhang, Shuhua Xu, F. Qian, F. Zhou, Huiru Tang, Dali Li, F. Sedlazeck, Li Jin, Yuting Guan, Shaohua Fan
          </td>
          <td>2025-02-10</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91ba15cf2f6390a1ec99834e397f27bbbddac7ca" target='_blank'>
              PFDN1 silencing disrupts critical cancer pathways in triple-negative breast cancer: investigating migration, cell cycle, and apoptosis as a new target therapy.
              </a>
            </td>
          <td>
            Abdulrahman M Alokda, A. Soffar, A. I. Yousef, Fawziya A R Ibrahim, Tarek S El-Sewedy, Alaa Elmetwalli
          </td>
          <td>2025-03-10</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9113b4cede2cf7486243e4f5408b0e991ee33d2d" target='_blank'>
              High-throughput single-cell DNA methylation and chromatin accessibility co-profiling with SpliCOOL-seq
              </a>
            </td>
          <td>
            Qingmei Shen, Enze Deng, Jingna Zhang, Qifeng Yang, Dan Su, Xiaoying Fan
          </td>
          <td>2025-03-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) are a heterogeneous group of tumor cells that play a significant role in tumorigenesis, therapeutic resistance, and recurrence in liver hepatocellular carcinoma (LIHC). This study combines clinical data sets from The Cancer Genome Atlas (TCGA) and the International Cancer Genome Consortium (ICGC) with bulk RNA sequencing data. This study also features the GSE156625 single-cell RNA sequencing (scRNA) data set from the GEO to explore the prognostic significance of CSC biomarkers (BCSCs) in LIHC. In this research, we introduce a developed prognostic risk model that relies on nine specific BCSCs, including ADM, CCL5, CD274, DLGAP5, HOXD9, IGF1, S100A9, SOCS2, and TNFRSF11B. It was found that high-risk patients experience shorter overall survival rates when compared to low-risk patients. Additionally, the study characterized the composition of immune cells within the tumor microenvironment (TME) and revealed significant variations in gene-expression levels and mutation rates between different risk groups. The model suggests that liver cancer progression might be driven by immune evasion independent of PD-L1 and highlights the potential of the low-risk BCSC group being sensitive to various treatments. Our findings offer a promising foundation for personalized LIHC therapy and highlight the need for further experimental validation of the roles of these CSCs in disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ccc91ecd266e7bcb8a4e400c1045bb16c558c70" target='_blank'>
              Identification of a Cancer Stem Cell-Related Gene Signature in Hepatocellular Carcinoma Based on Single-Cell RNA-Seq and Bulk RNA-Seq Analysis
              </a>
            </td>
          <td>
            Jing Wu, Xu Liu, Sheng Huang, Wei Liu
          </td>
          <td>2025-03-24</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Aberrant alternative splicing, prevalent in cancer, impacts various cancer hallmarks involving proliferation, angiogenesis, and invasion. Splicing disruption often results from somatic point mutations rewiring functional pathways to support cancer cell survival. We introduce iSoMAs (iSoform expression and somatic Mutation Association), an efficient computational pipeline leveraging principal component analysis technique, to explore how somatic mutations influence transcriptome-wide gene expression at the isoform level. Applying iSoMAs to 33 cancer types comprising 9,738 tumor samples in The Cancer Genome Atlas, we identified 908 somatically mutated genes significantly associated with altered isoform expression across three or more cancer types. Mutations linked to differential isoform expression occurred through both cis- and trans-acting mechanisms, involving well-known oncogenes/suppressor genes, RNA binding protein and splicing factor genes. With wet-lab experiments, we verified direct association between TP53 mutations and differential isoform expression in cell cycle genes. Additional iSoMAs genes have been validated in the literature with independent cohorts and/or methods. Despite the complexity of cancer, iSoMAs attains computational efficiency via dimension reduction strategy and reveals critical associations between regulatory factors and transcriptional landscapes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d1e96c7e45926632d80fe6cd693fccd8a255dd7" target='_blank'>
              iSoMAs: Finding isoform expression and somatic mutation associations in human cancers.
              </a>
            </td>
          <td>
            Hua Tan, Valer Gotea, S. Jaiswal, Nancy E Seidel, David O. Holland, Kevin Fedkenheuer, Abdel Elkahloun, Sara R. Bang-Christensen, L. Elnitski
          </td>
          <td>2025-03-07</td>
          <td>PLoS computational biology</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Multiple myeloma (MM) remains an incurable disease primarily due to the emergence of drug resistance, and the underlying mechanisms remain unclear. Extrachromosomal circular DNAs (eccDNAs) are prevalent in cancer genomes of both coding and non-coding regions. However, the role of non-coding eccDNA regions that serve as enhancers has been largely overlooked. Here, genome-wide profiling of serum eccDNAs from donors and MM patients who responded well or poorly to bortezomib-lenalidomide-dexamethasone (VRd) therapy is characterized. A high copy number of eccDNA ANKRD28 (eccANKRD28) predicts poor therapy response and prognosis but enhanced transcriptional activity. Established VRd-resistant MM cell lines exhibit a higher abundance of eccANKRD28, and CRISPR/Cas9-mediated elevation of eccANKRD28 desensitizes bortezomib and lenalidomide treatment both in vitro and in vivo. Integrated multi-omics analysis (H3K27ac ChIP-seq, scRNA-seq, scATAC-seq, CUT&Tag, et al.) identifies eccANKRD28 as an active enhancer involved in drug resistance driven by the key transcription factor, POU class 2 homeobox 2 (POU2F2). POU2F2 interacts with sequence-specific eccANKRD28 as well as RUNX1 and RUNX2 motifs to form the protein complex, which activates the promoter of oncogenes, including IRF4, JUNB, IKZF3, RUNX3, and BCL2. This study elucidates the potential transcriptional network of enhancer eccANKRD28 in MM drug resistance from a previously unrecognized epigenetic perspective.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea6e610cf350e732841fb77da78765020fd07704" target='_blank'>
              Enhancer Extrachromosomal Circular DNA ANKRD28 Elicits Drug Resistance via POU2F2-Mediated Transcriptional Network in Multiple Myeloma.
              </a>
            </td>
          <td>
            Binzhen Chen, Jia Liu, Yaoxin Zhang, Changming Shi, Di Zhu, Guoqiang Zhang, Fei Xiao, L. Zhong, Minyue Zhang, Lai Guan Ng, Honghui Huang, Tingting Lu, Jian Hou
          </td>
          <td>2025-04-01</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Abstract Hypoxia-inducible factor (HIF)-1 is a transcription factor that regulates the expression of target genes associated with oxygen homeostasis under hypoxic conditions, thereby contributing to tumor development and progression. Accumulating evidence has demonstrated that HIF-1α mediates different biological processes, including tumor angiogenesis, metastasis, metabolism, and immune evasion. Thus, overexpression of HIF-1α is strongly associated with poor prognosis in cancer patients. Natural compounds are important sources of anticancer drugs and studies have emphasized the decisive role of these mediators in modulating HIF-1α. Therefore, the pharmacological targeting of HIF-1α has emerged as a novel cancer therapeutic approach in recent years. The novelty of this review is that it summarizes natural products targeting HIF-1α in colorectal cancer that have not been presented earlier. We studied research publications related to the HIF-1α domain in cancer from 2010 to 2024. However, our main focus was to identify a better targeted approach for colorectal carcinoma management. Our review described HIF-1α role in tumor progression, summarizes the natural compounds employed as HIF-1α inhibitors, and discusses their potential in the development of natural compounds as HIF-1α inhibitors for colorectal cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76d0f41ebd3bbba4a4cc4f5dd3d089e4c659f102" target='_blank'>
              Recent Update of Natural Compounds as HIF-1α Inhibitors in Colorectal Carcinoma
              </a>
            </td>
          <td>
            P. Pandey, Sorabh Lakhanpal, Danish Mahmood, Lalji Baldaniya, Han Na Kang, Sungho Hwang, Sojin Kang, M. Choi, Seungjoon Moon, Shivam Pandey, Kamlesh Chaudhary, Fahad Khan, Bonglee Kim
          </td>
          <td>2025-03-01</td>
          <td>Drug Design, Development and Therapy</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Melanoma is the deadliest form of skin cancer, and its treatment poses significant challenges due to its aggressive nature and resistance to conventional therapies. Long non-coding RNAs (lncRNAs) represent a new frontier in the search for suitable targets to control melanoma progression and invasiveness. Indeed, lncRNAs exploit a wide range of regulatory functions along chromatin remodeling, gene transcription, post-transcription, transduction, and post-transduction to ultimately tune multiple cellular processes. The understanding of this intricate and flexible regulatory network orchestrated by lncRNAs in pathological conditions can strategically support the rational identification of promising targets, ultimately speeding up the setup of new therapeutics to integrate the currently available approaches. Here, the most recent findings on lncRNAs involved in melanoma will be analyzed. In particular, the functional links between their mechanisms of action and some frequently underestimated features, like their different subcellular localizations, will be highlighted.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f08119f9fccb5c4f5a79dd1ca1781581cce224f1" target='_blank'>
              Unravelling the Regulatory Roles of lncRNAs in Melanoma: From Mechanistic Insights to Target Selection
              </a>
            </td>
          <td>
            Beatrice Moras, C. Sissi
          </td>
          <td>2025-02-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/255726480b1fba8b96c7a797db393035b828db59" target='_blank'>
              Abstract B022: Decipher actionable tumor suppressor defects for personalized cancer medicine
              </a>
            </td>
          <td>
            Feiyu Chen, Di Zhao
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Lung squamous cell carcinoma (LUSC), accounting for 30% of lung cancer cases, lacks adequate research due to limited understanding of its molecular abnormalities. Our study analyzed public LUSC datasets to explore the tumor microenvironment (TME) composition using scRNA-seq from two cohorts. Applying non-negative matrix factorization, we identified unique malignant cell phenotypes, or meta-programs (MPs), based on gene expression patterns. Survival analysis revealed the clinical relevance of these MPs. Findings illuminated a TME landscape enriched with immune cells—CD8 + T, exhausted T, CD4 + T, and naive T cells—and suggested roles for myeloid cells, like cDC1 and pDCs, in LUSC progression. Different MPs highlighted the heterogeneity of malignant cells and their clinical implications. Targeting MP-specific genes may enable personalized therapy, especially for early-stage LUSC. This study offers insights into immune cell function in tumor dynamics, identifies MPs, and paves the way for novel LUSC strategies, enhancing early intervention, personalized treatment, and prognosis, ultimately improving patient outcomes. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-93916-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a302b866a51ae52b8b8d7352c3feab56f45e3d0e" target='_blank'>
              Single-cell RNA sequencing reveals potential therapeutic targets in the tumor microenvironment of lung squamous cell carcinoma
              </a>
            </td>
          <td>
            Junda Fan, Yu Chen, Yue Gong, Hongmei Sun, Rui Hou, Xiaoya Dou, Yanping Zhang, Cheng Huo
          </td>
          <td>2025-03-26</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c5574ba347be751b5260edd71a2fcb0e3909c08" target='_blank'>
              Integrative single-cell and bulk RNA-seq analysis identifies lactylation-related signature in osteosarcoma.
              </a>
            </td>
          <td>
            Zhou Xie, Xiao Qu, Jun Zhang, Yanran Huang, Runhan Zhao, Dagang Tang, Ningdao Li, Zhule Wang, Xiaoji Luo
          </td>
          <td>2025-03-12</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Telomeres are dynamic complexes at the ends of chromosomes that are made up of protective proteins and tandem repeating DNA sequences. In the large majority of cancer cells, telomere length is maintained by telomerase, an enzyme that elongates telomeres. Telomerase activation is seen in the majority of cancer, which permits uncontrol cell proliferation. About 90% of human malignancies show telomere dysfunction and telomerase reactivation; as a result, telomerase activation plays a special role as a practically universal stage on the way to malignancy. This review understands the structural and functional of telomere and telomerase, mechanisms of telomerase activation in oncogenesis, biomarkers and therapeutic targets. Therapeutic strategies targeting telomerase, including antisense oligonucleotides, G-quadruplex stabilizers, immunotherapy, small-molecule inhibitors, gene therapy, Telomerase-Responsive Drug Release System, have shown promise in preclinical and clinical settings. Advances in telomere biology not only illuminate the complex interplay between telomeres, telomerase, and cancer progression but also open avenues for innovative, targeted cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0a88e3f76f3fb38bb6a5ed80e16846195631c78" target='_blank'>
              The role of telomere and telomerase in cancer and novel therapeutic target: narrative review
              </a>
            </td>
          <td>
            Temesgen Baylie, Mohammed Jemal, Gelagay Baye, Mamaru Getinet, G. Amare, Adane Adugna, D. Abebaw, Zigale Hibstu, B. Tegegne, Endalkachew Gugsa, Tadegew Adane, Gedefaw Getie, Baye Ashenef, Deresse Sinamaw
          </td>
          <td>2025-02-14</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background High transcription levels are essential for cancer cells to maintain their malignant phenotype. While RNA polymerases (POLRs) have been implicated in various transcriptional mechanisms, their impact on the tumor microenvironment (TME) remains poorly understood. Methods We analyzed publicly available pan-cancer cohorts to evaluate the expression and genomic alterations of POLRs. Focusing on head and neck squamous cell carcinoma (HNSC), we integrated bulk RNA sequencing, single-cell, and spatial transcriptome data to identify POLR2C expression patterns and its potential regulation by Yin Yang 1 (YY1). In vitro and in vivo experiments were conducted to validate the functional role of the YY1-POLR2C axis in cancer proliferation and immune modulation. Results POLRs were found to be aberrantly expressed in cancers and associated with genomic alterations. In HNSC, POLR up-regulation was linked to poor prognostic features. POLR2C was significantly up-regulated in malignant cells, and its expression appeared to be transcriptionally regulated by YY1. Functional studies demonstrated that the YY1-POLR2C axis drives cell-cycle dysregulation and malignant proliferation in HNSC. Additionally, high POLR expression negatively correlated with immune cell infiltration and facilitated immune evasion. Mechanistically, POLRs mediated frequent interactions between malignant and immune cells, potentially contributing to resistance to immunotherapy. Conclusion This study highlights the dual role of POLRs in promoting malignant proliferation and shaping an immunosuppressive TME. POLR2C, regulated by YY1, emerges as a critical mediator in HNSC and a promising target for precision therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5caf5c8af345e8c1d8ae660db4ba00bbab580435" target='_blank'>
              Comprehensive Analysis of Multi-Omics Data on RNA Polymerase as an Adverse Factor in Head and Neck Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            Yue Gu, Jie Zhang, Yuan Liu, Qian Zhang, Qi-Feng Geng
          </td>
          <td>2025-03-01</td>
          <td>Journal of Inflammation Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Abstract Background Pseudogene‐derived lncRNAs are widely dysregulated in cancer. Technological advancements have facilitated the functional characterization of increasing pseudogenes in cancer progression. However, the association between pseudogenes and RNA N6‐methyladenosine (m6A) modification in cancer, as well as the underlying mechanisms, remains largely unexplored. Methods We analyzed the expression of 12 146 pseudogenes and comprehensively examined the m6A modification of RNAs derived from them and their paralogs. Through integrative analysis of multi‐omics data, we explored the associations between pseudogene dysregulation and m6A, identifying critical pseudogenes involved in HGSOC progression. Tumour promotion role of RPS15AP12 and its cognate parent gene was characterized by cell proliferation, transwell assays, and scratch assays in ovarian cells and xenograft nude mice. RNA decay assays were used to reveal the participation of m6A in decreasement of RPS15AP12 lncRNA stability. Luciferase reporter assays were performed to verify that RPS15AP12 enhances RPS15A expression by competitively binding to miR‐96‐3p. Western blot and phosphorylation assays were performed to investigate the impairment of RPS15AP12 towards the sensors of MAVS (RIG‐I and MDA5), and downstream p‐TBK1 and p‐IRF3. Finally, ELISA assays were performed to explore the regulatory role of RPS15AP12 in IFN‐β expression. Results M6A is distributed across over a thousand pseudogenes, and hypomethylation leads to their upregulation in HGSOC. We identified a processed pseudogene, RPS15AP12, upregulated by FTO‐mediated m6A demethylation. RPS15AP12 enhances the growth ability and metastatic capabilities of ovarian cancer (OC) cells via functioning as a competitive endogenous RNA (ceRNA) for its host gene, RPS15A, through the sequestration of miR‐96‐3p. Importantly, the deletion of RPS15AP12 diminishes the expression of RPS15A, leading to the upregulation of anti‐tumour immune responses by activating RIG‐I and MDA5 and downstream p‐TBK1 and p‐IRF3 as well as IFN‐β levels. Conclusion Our findings expand the understanding of m6A‐modulated pseudogenes in tumour growth and anti‐tumour innate immunity in OC. Key Points Genome‐wide profiling reveals the redistribution of m6A modification on pseudogene‐derived lncRNAs and m6A redistribution‐relevant dysregulation of pseudogenes in HGSOC. RPS15AP12, as a representative processed pseudogene, is up‐regulated by FTO‐mediated demethylation and acts as a miRNA sponge to promote RPS15A expression via competitively binding to miR‐96‐3p. RPS15AP12/RPS15A axis inhibits MAVS sensors (RIG‐I and MDA5) and downstream IFN‐β levels in ovarian cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7f32a6bc1ac1ae8503c9d4c32bcdde0ae2bb380" target='_blank'>
              Genome‐wide profiling of N6‐methyladenosine‐modified pseudogene‐derived long noncoding RNAs reveals the tumour‐promoting and innate immune‐restraining function of RPS15AP12 in ovarian cancer
              </a>
            </td>
          <td>
            Jie Xu, Yifei Ren, Jiayi Lu, Fengjiang Qin, Dan Yang, Chunyan Tang, Yu Yang, Jing Xu, Tao Liu, Ping Yi
          </td>
          <td>2025-02-25</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f1f61ef381af091b6db128907436bc9e8a1a514" target='_blank'>
              Directed evolution expands CRISPR-Cas12a genome editing capacity
              </a>
            </td>
          <td>
            E. Ma, Kai Chen, Honglue Shi, Kevin Wasko, Isabel Esain-Garcia, M. Trinidad, Kaihong Zhou, Jinjuan Ye, Jennifer A. Doudna
          </td>
          <td>2025-03-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35da01c8609a7028b2e2ec26d0b8011e3cf58489" target='_blank'>
              A single-cell transcriptomic comparison between small intestinal neuroendocrine tumors and their progenitor
              </a>
            </td>
          <td>
            Fredrik Axling, Elham Barazeghi, Per Hellman, O. Norlén, S. Backman, P. Stålberg
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Rationale: Non-small cell lung cancer (NSCLC) is a predominant cause of cancer-related mortality, with its progression and treatment resistance significantly influenced by cancer stem cells (CSCs) and their complex intercellular communication mechanisms. Small extracellular vesicles (sEVs) have emerged as pivotal mediators of intercellular signaling, affecting tumor microenvironment modulation and therapeutic resistance. This study investigates the role of CSC-derived sEVs in transmitting stemness traits through the selective sorting of pyruvate kinase M2 phosphorylated at the Y105 site (pY105-PKM2), mediated by the adaptor protein IQGAP1, which supports CSC maintenance and drug resistance in NSCLC. Methods: In vitro and in vivo experiments, including proteomic and transcriptomic analyses, were conducted to identify key regulators of sEV-mediated signaling. Immunoprecipitation, proximity ligation assays, and immunofluorescence were used to examine the role of IQGAP1 in the sorting of pY105-PKM2 into sEVs. Functional assays, including sphere formation, chemoresistance tests, metabolic assessments, and cell cycle analysis, were conducted to evaluate the effects of sEV-mediated delivery of pY105-PKM2 on recipient cells. Additionally, immunohistochemistry and survival analysis were performed on tumor samples from NSCLC patients to establish clinical correlations. Results: We unveiled a novel mechanism by which CSC-derived sEVs transmit stemness traits to replenish the CSC pool in NSCLC. CSC-derived sEVs were enriched with pY105-PKM2, correlating with enhanced stemness, chemoresistance, and poor clinical outcomes. Mechanistically, IQGAP1 was identified as an adaptor facilitating the selective sorting of pY105-PKM2 into sEVs through interactions with the ESCRT component TSG101. Recipient cells treated with CSC-derived sEVs exhibited metabolic reprogramming, slower cell cycle progression, and enhanced chemoresistance. The synergistic role of IQGAP1 and pY105-PKM2 was confirmed, highlighting their critical contributions to CSC maintenance and malignant progression. Conclusion: This study highlights the critical role of CSC-derived sEVs in NSCLC progression and therapy resistance through the IQGAP1-mediated selective sorting of pY105-PKM2. By uncovering this novel pathway, our findings provide valuable insights into CSC pool replenishment and therapeutic resistance mechanisms in NSCLC, identifying IQGAP1 and pY105-PKM2 as promising therapeutic targets for mitigating CSC-driven malignancy and enhancing treatment efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa8844821c21b603caec632dd4e93148e05d6265" target='_blank'>
              A novel pathway for stemness propagation and chemoresistance in non-small cell lung cancer via phosphorylated PKM2-loaded small extracellular vesicles
              </a>
            </td>
          <td>
            Jingyi Wang, Liu Liu, Xinyu Gao, Xiyu Liu, Yitian Dai, Zijun Mao, Shengzhe Huang, Junjian Li, Dongliang Wang, Yu Qi, Yingwen Han, Yunjing Xu, C. Y. Chua, A. Grattoni, Wenhui Xie, Hao Yang, Gang Huang
          </td>
          <td>2025-02-24</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="The primary objective of this study is to conduct a pan-cancer analysis of CHRDL1 expression, to determine its correlation with patient survival rates, immune cell infiltration, and drug sensitivity. Additionally, the study aimed to further validate the mechanistic role of CHRDL1 in lung adenocarcinoma (LUAD), clarifying its contribution to tumorigenesis and evaluating its potential as a therapeutic target for LUAD.We employed bioinformatics strategies to analyze CHRDL1 expression using data from The Cancer Genome Atlas (TCGA) and the Genotype-Tissue Expression Project (GTEx). Survival analysis was executed with GEPIA2, while drug sensitivity to chemotherapeutic agents was evaluated via the CellMiner database. Mutational profiles were examined using cBioPortal, and the immune microenvironment was assessed through the TIMER database. To substantiate our findings, we conducted in vitro cellular assays and in vivo animal models to validate the mechanistic actions of CHRDL1 in LUAD.CHRDL1 expression levels showed significant variation across different cancer types, with tumor tissues typically demonstrating lower expression compared to their normal counterparts. In certain cancers, elevated CHRDL1 expression was linked to poorer survival outcomes, whereas in LUAD, it was associated with improved survival. Furthermore, CHRDL1 expression correlated with the IC50 values of multiple chemotherapeutic drugs and played a role in modulating the immune microenvironment. We discovered that CHRDL1 inhibits the epithelial-mesenchymal transition (EMT) in LUAD through the TGF-β pathway.CHRDL1 exerts a complex influence on cancer development and progression, particularly in LUAD, by impacting tumor progression, immune regulation, chemosensitivity, and EMT regulation. This research offers valuable insights into the overarching mechanisms of cancer progression and aids in the discovery of innovative therapeutic strategies for LUAD treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd79eedadd08697bb7daedaf44fd4b0c2b964ff9" target='_blank'>
              Pan-cancer analysis of CHRDL1 expression and its mechanistic role in inhibiting EMT via the TGF-β pathway in lung adenocarcinoma
              </a>
            </td>
          <td>
            Honghong Dong, Yahui Tian, Shaowei Xin, Yujie Guo, Suxin Jiang, Zitong Wan, Huai-Yu Wang, Yong Han
          </td>
          <td>2025-03-31</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Bacteriophages constitute one of the largest reservoirs of genes of unknown function in the biosphere. Even in well-characterized phages, the functions of most genes remain unknown. Experimental approaches to study phage gene fitness and function at genome scale are lacking, partly because phages subvert many modern functional genomics tools. Here we leverage RNA-targeting dCas13d to selectively interfere with protein translation and to measure phage gene fitness at a transcriptome-wide scale. We find CRISPR Interference through Antisense RNA-Targeting (CRISPRi-ART) to be effective across phage phylogeny, from model ssRNA, ssDNA and dsDNA phages to nucleus-forming jumbo phages. Using CRISPRi-ART, we determine a conserved role of diverse rII homologues in subverting phage Lambda RexAB-mediated immunity to superinfection and identify genes critical for phage fitness. CRISPRi-ART establishes a broad-spectrum phage functional genomics platform, revealing more than 90 previously unknown genes important for phage fitness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d283ad1ac40041b683846c272f3ed7e85e462ab" target='_blank'>
              CRISPRi-ART enables functional genomics of diverse bacteriophages using RNA-binding dCas13d
              </a>
            </td>
          <td>
            Benjamin A. Adler, Muntathar J. Al-Shimary, Jaymin R. Patel, Emily G. Armbruster, David Colognori, Emeric Charles, Kate V Miller, Arushi Lahiri, Michael Cui, Agnès Oromí-Bosch, Angela Voelker, M. Trinidad, Jina Lee, Sebastien Beurnier, Ron S. Boger, Jason Nomburg, R. Barrangou, Vivek K. Mutalik, J. Schoeniger, Joseph A. Pogliano, David F. Savage, Jennifer A. Doudna, Brady F. Cress
          </td>
          <td>2025-02-26</td>
          <td>Nature Microbiology</td>
          <td>2</td>
          <td>71</td>
        </tr>

        <tr id="CRISPR-Cas9 systems have revolutionized genome editing, but the off-target effects of Cas9 limit its use in clinical applications. Here, we systematically evaluate FrCas9, a variant from Faecalibaculum rodentium, for cell and gene therapy (CGT) applications and compare its performance to SpCas9 and OpenCRISPR-1. OpenCRISPR-1 is a CRISPR system synthesized de novo using large language models (LLMs) but has not yet undergone systematic characterization. Using AID-seq, Amplicon sequencing, and GUIDE-seq, we assessed the on-target activity and off-target profiles of these systems across multiple genomic loci. FrCas9 demonstrated higher on-target efficiency and substantially fewer off-target effects than SpCas9 and OpenCRISPR-1. Furthermore, TREX2 fusion with FrCas9 reduced large deletions and translocations, enhancing genomic stability. Through screening of 1903 sgRNAs targeting 21 CGT-relevant genes using sequential AID-seq, Amplicon sequencing, and GUIDE-seq analysis, we identified optimal sgRNAs for each gene. Our high-throughput screening platform highlights FrCas9, particularly in its TREX2-fused form, as a highly specific and efficient tool for precise therapeutic genome editing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6808f0ec0ae3c51721dc4169f97a812e9c172d0f" target='_blank'>
              Systematic high-throughput evaluation reveals FrCas9’s superior specificity and efficiency for therapeutic genome editing
              </a>
            </td>
          <td>
            Rui Tian, Xun Tian, Meiying Yang, Yuping Song, Tingting Zhao, Chaoyue Zhong, Wei Zhu, Ping Zhou, Zhiqiang Han, Zheng Hu
          </td>
          <td>2025-03-26</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a73dc13e33e938dc26c7fd4908040282b9950aea" target='_blank'>
              Advances in CRISPR-enabled genome-wide screens in yeast.
              </a>
            </td>
          <td>
            Nicholas R. Robertson, Sangcheon Lee, Aida Tafrishi, I. Wheeldon
          </td>
          <td>2025-03-20</td>
          <td>FEMS yeast research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93950b2f46ff84064096cec296af5119b5270d3d" target='_blank'>
              Aberrant Expression of Non-Coding RNAs in Pediatric T Acute Lymphoblastic Leukemia and Their Potential Application as Biomarkers
              </a>
            </td>
          <td>
            Neila Luciano, Luigi Coppola, Giuliana Salvatore, P. Primo, Rosanna Parasole, P. Mirabelli, F. M. Orlandella
          </td>
          <td>2025-03-31</td>
          <td>Genes</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Objectives/Goals: Large-scale tumor sequencing efforts have led to annotations of novel cancer hotspot mutations that may underlie driver or cooperative function. We have sought to define the molecular consequences of such hotspots associated with pediatric DICER1 syndrome cancers, with the ultimate goal of revealing novel targets that may inform new standards of care. Methods/Study Population: We have performed genomic analysis to identify tumor types (in TCGA and MSK-IMPACT patient data) for which mutations in the Dicer1 gene (encoding Dicer protein) emerge as the dominant signature of driver function. As Dicer is a critical RNA processing factor responsible for the generation of microRNAs, which are posttranscriptional gene regulatory molecules, we have modeled these mutations in human embryonic stem cells in order to study the direct effects on miRNAs and their target genes in an isogenic background. In addition to providing the required setting for unambiguous attribution of function to specific mutations, clonal human ES cells offer an opportunity for modeling of both developmental and cancer requirements associated with altered Dicer function. Results/Anticipated Results: Through generation of genomics and functional datasets from matched genotypes in Dicer mutated human ES cells, we have identified specific alterations in miRNAs and their effects on target genes. Unexpectedly, we found direct evidence for both loss of function and gain of function attributable to Dicer mutations. In addition, through integrated analysis of genomic data from tumor sequencing datasets and our human ES cell models, we have identified potential miRNA and target gene alterations that underlie tumorigenic potential, nominating gene candidates for targeted therapy in DICER1 syndrome. Direct mouse modeling of such candidate gene targets has revealed evidence for driver function of identified miRNA and their targets. Discussion/Significance of Impact: DICER1 syndrome cancers comprise a wide variety of rare pediatric tumor types. Presently, we still lack an effective standard of care. Furthermore, the previous lack of molecular profiling precluded targeted therapy opportunities. Our precise knock-in modeling of Dicer hotspots and deep profiling of relevant tumors now provide candidate targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/597fce8f39319f3d39a09a21bf59ed3021208522" target='_blank'>
              518 Modeling of cancer mutations found in pediatric DICER1 syndrome informs novel therapeutic targeting strategies
              </a>
            </td>
          <td>
            David Jee, Seungjae Lee, Dapeng Yang, Robert W. Rickert, Renfu Shang, Danwei Huangfu, Eric C. Lai
          </td>
          <td>2025-03-26</td>
          <td>Journal of Clinical and Translational Science</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fb803b3e058115ac9f37e42f7b7a8e8748c0c02" target='_blank'>
              Next-Generation Sequencing-Based In Silico Transgene Integration Profiling Tool Accelerates Cell Line Genetic Characterizations.
              </a>
            </td>
          <td>
            Zhenqiu Huang, Hyo-Young Jeong, Luke Nelson, Caitlyn Devine, Guanghua Benson Li, Lin Zhang
          </td>
          <td>2025-03-01</td>
          <td>Biotechnology journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="ERBB2 (HER2) is a well-studied oncogene with several driver mutations apart from the well-known amplification defect in some breast cancers. We used the GigaAssay to test the functional effect of HER2 missense mutations on its receptor tyrosine kinase function. The GigaAssay is a modular high-throughput one-pot assay system for simultaneously measuring molecular function of thousands of genetic variants at very high accuracy. The activities of 5,886 mutations were classified, significantly more than mutants previously reported. These variants include 112 new in vitro, 10 known, and 9 new in vivo gain-of-function (GOF) mutations. Many of the GOFs spatially cluster in sequence and structure, supporting the activation mechanisms of heterodimerization with EGFR and release of kinase inhibition by the juxtamembrane domain. Retrospective analysis of patient outcomes from the Genomic Data Commons predicts increased survival with the newly identified HER2 GOF variants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b136d3cab282e49a197992ab3575cc9958742822" target='_blank'>
              New Gain-of-Function Mutations Prioritize Mechanisms of HER2 Activation
              </a>
            </td>
          <td>
            C. Giacoletto, Liz J. Valente, Lancer Brown, Sara Patterson, Rewatee H Gokhale, S. Mockus, Wayne W. Grody, Hong-Wen Deng, Jerome I. Rotter, Martin R. Schiller
          </td>
          <td>2025-03-04</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background Cancer stem cells (CSCs) are crucial for lung adenocarcinoma (LUAD). This study investigates tumor stem cell gene signatures in LUAD using single-cell RNA sequencing (scRNA-seq) and bulk RNA sequencing (RNA-seq), aiming to develop a prognostic tumor stem cell marker signature (TSCMS) model. Methods LUAD scRNA-seq and RNA-seq data were analyzed. CytoTRACE software quantified the stemness score of tumor-derived epithelial cell clusters. Gene Set Variation Analysis (GSVA) identified potential biological functions in different clusters. The TSCMS model was constructed using Lasso-Cox regression, and its prognostic value was assessed through Kaplan–Meier, Cox regression, and receiver-operating characteristic (ROC) curve analyses. Immune infiltration was evaluated using the Cibersortx algorithm, and drug response prediction was performed using the pRRophetic package. TAF10 functional investigations in LUAD cells involved bioinformatics analysis, qRT-PCR, Western blot, immunohistochemistry, and assays for cell proliferation. Results Seven distinct cell clusters were identified by CytoTRACE, with epithelial cell cluster 1 (Epi_C1) showing the highest stemness potential. The TSCMS model included 49 tumor stemness-related genes; high-risk patients exhibited lower immune and ESTIMATE scores and increased tumor purity. Significant differences in immune landscapes and chemotherapy sensitivity were observed between risk groups. TAF10 positively correlated with RNA expression-based stemness scores in various tumors, including LUAD. It was over-expressed in LUAD cell lines and clinical tumor tissues, with high expression linked to poor prognosis. Silencing TAF10 inhibited LUAD cell proliferation and tumor sphere formation. Conclusions This study demonstrates the TSCMS model's prognostic value in LUAD, reveals insights into immune infiltration and therapeutic response, and identifies TAF10 as a potential therapeutic target. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06243-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5653c962ff1a70fb0bb0f0fb2592a34ab549e9a8" target='_blank'>
              Integration of single-cell and bulk RNA sequencing to identify a distinct tumor stem cells and construct a novel prognostic signature for evaluating prognosis and immunotherapy in LUAD
              </a>
            </td>
          <td>
            Fengyun Zhao, Mengting Chen, Tianjiao Wu, M. Ji, Fugui Li
          </td>
          <td>2025-02-22</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/498b068e38445ba010a97c92197f812fab2d458c" target='_blank'>
              Abstract PR010: Genome-wide CRISPR screens in CD8+ T cells identify cullin-RING E3 ubiquitin ligase complex as a negative regulator of long-term effector function
              </a>
            </td>
          <td>
            Justin D. Saco, Daniel Karin, Nataly Naser Al Deen, Katie M. Campbell, Justin Patten, Yichen Fan, Egmidio Medina, Antoni Ribas
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The introduction of poly-ADP ribose polymerase (PARP) inhibitors (PARPi) has completely changed the treatment landscape of breast cancer susceptibility 1-2 (BRCA1-BRCA2)-mutant cancers and generated a new avenue of research in the fields of DNA damage response and cancer therapy. Despite this, primary and secondary resistances to PARPi have become a challenge in the clinic, and novel therapies are urgently needed to address this problem. After two decades of research, a unifying model explaining sensitivity of cancer cells to PARPi is still missing. Here, we review the current knowledge in the field and the increasing evidence pointing to a crucial role for replicative gaps in mediating sensitization to PARPi in BRCA-mutant and 'wild-type' cancer cells. Finally, we discuss the challenges to be addressed to further improve the utilization of PARPi and tackle the emergence of resistance in the clinical context.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59502fb812b3b4643f159aa0e72d8a002c1dd744" target='_blank'>
              'Where is my gap': mechanisms underpinning PARP inhibitor sensitivity in cancer.
              </a>
            </td>
          <td>
            Lauryn Buckley-Benbow, Alessandro Agnarelli, Roberto Bellelli
          </td>
          <td>2025-02-10</td>
          <td>Biochemical Society transactions</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Rationale: Despite substantial advancement in the treatment of B-cell acute lymphoblastic leukemia (B-ALL), it remains a leading cause of cancer mortality in children due to the high relapse rate. Moreover, the long-term survival rates for adult B-ALL patients are still less than 40%. The B-ALL patients carrying MLL rearrangements or BCR-ABL fusion represent high-risk B-ALL subtypes that face particularly dismal prognoses. This study aims to identify innovative therapeutic vulnerability for high-risk B-ALL. Methods: The CRISPR-Cas9 screen was conducted to pinpoint genes essential for high-risk B-ALL cell survival/growth. Both in vitro and in vivo models were then employed to investigate the pathological role of ZNF217 in high-risk B-ALL. To characterize the downstream functionally essential targets of ZNF217, we performed RNA-seq and CUT&RUN-seq, followed by integrative bioinformatics analysis and experimental validation. Results: Through the focused CRISPR-Cas9 screening, ZNF217 emerged as the most essential gene for the cell survival/growth of B-ALL driven by MLL rearrangement or BCR-ABL. Through in vitro gain- and loss-of-function assays, we demonstrated that ZNF217 is indeed required for B-ALL cell survival/growth. Moreover, we established the B-ALL xenograft model and patient-derived xenograft (PDX) model and demonstrated that ZNF217 depletion significantly suppressed B-ALL progression and substantially extended the survival of recipient mice. Through integrative multiple-omics analysis, we elucidated that ZNF217 exerts its oncogenic role in B-ALL through both CoREST-dependent and CoREST-independent mechanisms. Furthermore, we characterized FOS as a functionally essential downstream target of ZNF217, and ZNF217 inhibited FOS expression in a CoREST-independent manner. Conclusions: Our findings highlight ZNF217 as a promising therapeutic target for the treatment of high-risk B-ALL, such as those carrying MLL-rearrangements or BCR-ABL fusion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a506db79238d620427652c02a2a5ec31786dbecb" target='_blank'>
              CRISPR screening reveals ZNF217 as a vulnerability in high-risk B-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Xi Qin, Keren Zhou, Lei Dong, Lu Yang, Wei Li, Zhenhua Chen, Chao Shen, Li Han, Yangchan Li, A. Chan, S. Pokharel, Y. Qing, Meiling Chen, K. Wang, Keith Leung, Lillian Sau, C. Chen, Xiaolan Deng, R. Su, Jianjun Chen
          </td>
          <td>2025-02-18</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f37dc921f83b90daf61b10b8456e0c63c032e710" target='_blank'>
              [Pathogenic large duplication in TP53 as a hereditary predisposing factor in breast cancer].
              </a>
            </td>
          <td>
            Viktória Kovács, H. Butz, János Papp, T. Pócza, Kornél Vince Grolmusz, Petra Nagy, Attila Patócs, A. Bozsik
          </td>
          <td>2025-03-21</td>
          <td>Magyar onkologia</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a92d215150e22b00d817410c07ac0e4856717a42" target='_blank'>
              Multidimensional OMICs reveal ARID1A orchestrated control of DNA damage, splicing, and cell cycle in normal-like and malignant urothelial cells.
              </a>
            </td>
          <td>
            Rebecca M Schlösser, Florian Krumbach, Eyleen Corrales, G. Andrieux, Christian Preisinger, Franziska Liss, A. Golzmann, M. Boerries, Kerstin Becker, Ruth Knüchel, Stefan Garczyk, Bernhard Lüscher
          </td>
          <td>2025-04-01</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c32ba0e230de094bf228cc062386115a9ca1917" target='_blank'>
              Integrated multi-omics analysis reveals the functional and prognostic significance of lactylation-related gene PRDX1 in breast cancer
              </a>
            </td>
          <td>
            Qinqing Wu, Heng Cao, Jiangdong Jin, Dongxu Ma, Yixiao Niu, Yanping Yu, Xiang Wang, Yiqin Xia
          </td>
          <td>2025-04-04</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97ce46a7540ecdc592f6a5090f63d5146e79c065" target='_blank'>
              Inflammation-driven Oncogenesis: Precision Medicine and Immunotherapeutic Strategies
              </a>
            </td>
          <td>
            Almuthanna K. Alkaraki, Alaa A. A. Aljabali, Mohammad A. Obeid
          </td>
          <td>2025-02-24</td>
          <td>Mini-Reviews in Medicinal Chemistry</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6a706f94dfd8cb5a4704edb83d5ed63a78c5cff" target='_blank'>
              A novel stroma-dependent leukemia cell line from a patient with mixed-phenotype acute leukemia with Ph chromosome and PAX5 mutation.
              </a>
            </td>
          <td>
            Shoko Ishii, Yasuhiro Arakawa, Hiroto Ishii, K. Yokoyama, Hiroki Yokoyama, Takeshi Saito, Shingo Yano
          </td>
          <td>2025-02-20</td>
          <td>International journal of hematology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Expression levels of ATP-binding cassette (ABC) transporters are known to be increased in various tumor cells, including in breast cancer, and they are responsible for mediating drug resistance, leading to treatment failure. In the present study, gene expression array analysis revealed that among ABC transporter subtypes, ABC subfamily G member 8 (ABCG8) was one of the most increased in radiotherapy-resistant triple-negative breast cancer (RT-R-TNBC) cells compared with in TNBC cells. ABCG8 is involved in sterol efflux; however, its role in cancer is not well known. Therefore, the present study investigated the effect of ABCG8 on tumor progression in RT-R-TNBC cells. Gene expression profiling was conducted using the QuantiSeq 3′ mRNA-Seq Service, followed by western blotting to confirm protein levels. Loss-of-function assays using small interfering RNA (si) transfection were performed to assess the roles of ABCG8 and its regulatory signaling pathways. RT-R-MDA-MB-231 cells exhibited increased cholesterol levels in both cells and the surrounding media via induction of sterol regulatory element binding protein 1 (mature form) and fatty acid synthase. siABCG8 transfection increased intracellular cholesterol levels but decreased cholesterol levels in the media, indicating an accumulation of cholesterol inside cells. Additionally, RT-R-MDA-MB-231 cells exhibited increased levels of β-catenin compared with MDA-MB-231 cells, which was significantly reduced by ABCG8 knockdown. Furthermore, ABCG8 knockdown led to cell cycle arrest in the G2/M phase in RT-R-MDA-MB-231 cells by reducing Polo-like kinase 1 (PLK1) and Cyclin B1 expression. RT-R-MDA-MB-231 cells also exhibited increased phosphorylated-low-density lipoprotein (LDL) receptor-related protein 6 (LRP6) levels compared with MDA-MB-231 cells, and these were decreased by siABCG8 transfection. LRP6 siRNA transfection decreased β-catenin, PLK1 and Cyclin B1 expression. In addition, feedback mechanisms such as liver X receptor and inducible degrader of LDL were decreased in RT-R-MDA-MB-231 cells under normal conditions compared with in MDA-MB-231 cells. To the best of our knowledge, the present study was the first to suggest that the cholesterol exported by ABCG8, not inside the cells, may affect cancer progression via the LRP6/Wnt/β-catenin signaling pathway in RT-R-TNBC. The regulation of this pathway may offer a potential therapeutic strategy for the treatment of RT-R-TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a057625eb93a3173c7a3b1671e3d5105799112eb" target='_blank'>
              ABCG8-mediated sterol efflux increases cancer cell progression via the LRP6/Wnt/β-catenin signaling pathway in radiotherapy-resistant MDA-MB-231 triple-negative breast cancer cells
              </a>
            </td>
          <td>
            Y. Ko, Ju Yeong Won, Hana Jin, Nam Binh Nguyen, Yaeram Won, Vedaste Nsanzimana, S. Yun, Sang Won Park, Hye Jung Kim
          </td>
          <td>2025-03-20</td>
          <td>International Journal of Molecular Medicine</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da0026f814610b80b221cc4995959120d34e1c0a" target='_blank'>
              Deciphering WNT signalling vulnerabilities in soft tissue sarcoma.
              </a>
            </td>
          <td>
            Marina Pérez-Capó, Esther Martinez-Font, Elena Prados, Rafael Ramos, Raúl Sánchez Morillas, Maria Rosa Martorell, Oliver Vögler, R. Alemany, A. Obrador-Hevia
          </td>
          <td>2025-03-20</td>
          <td>Pathobiology : journal of immunopathology, molecular and cellular biology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03e8bbeec84df302b9adf6fafec7b5584f0a9749" target='_blank'>
              Advancement of Gene Therapy in Cancer
              </a>
            </td>
          <td>
            Kahkashan Sultana, Ayesha Fatima
          </td>
          <td>2025-03-15</td>
          <td>Journal of Advanced Scientific Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d3959b933b4c238cc534f4bd3bbf4f10f6618c5" target='_blank'>
              Recent Advances in Anti-Aging Therapeutic Strategies Targeting DNA Damage Response and Senescence-Associated Secretory Phenotype-Linked Signaling Cascade.
              </a>
            </td>
          <td>
            Jawad Nadeem, Razia Sultana, Amna Parveen, Sun Yeou Kim
          </td>
          <td>2025-02-26</td>
          <td>Cell biochemistry and function</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The ataxia-telangiectasia mutated (ATM) protein kinase plays a critical role in activating the cellular response to DNA double-strand breaks and promoting homology-directed repair. ATM is frequently mutated in cancer, contributing to an accumulation of DNA damage that drives genomic instability. To exploit cancer cells’ inherent vulnerability to DNA damage, various small molecule inhibitors have been developed that target ATM. ATM inhibitors have shown great versatility in preclinical studies and increasing use in the clinic. Here, we review the development of ATM inhibitors and their role in cancer therapy. We describe their limitations and the advances that have led to increases in both the number and diversity of active clinical trials targeting ATM. We also discuss ATM’s role in personalized medicine and the current challenges to more widespread use of ATM inhibitors in the clinic.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c1ae4df6ed1675fe0ee6d9bf7493ed3c92eb628" target='_blank'>
              The Development of ATM Inhibitors in Cancer Therapy
              </a>
            </td>
          <td>
            Elizabeth A. Ampolini, Judit Jimenez-Sainz, David T Long
          </td>
          <td>2025-03-01</td>
          <td>Targeted Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25ebeca51216d7303220f613018ac007aff3e6b2" target='_blank'>
              Integrative genome-wide aberrant DNA methylation and transcriptome analysis identifies diagnostic markers for colorectal cancer.
              </a>
            </td>
          <td>
            Hengyang Shen, Zhenling Wang, Yang Chen, Changzhi Huang, Lei Xu, Ying Tong, Hongqiang Zhang, Yunfei Lu, Shuwei Li, Zan Fu
          </td>
          <td>2025-03-10</td>
          <td>Archives of toxicology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The canonical DNA damage response (DDR) maintains genome stability, involving DNA synthesis/cell cycle arrest. However, unchallenged cells proliferate when they are continually exposed to low-level/endogenous replication stress. We previously discovered and characterized a noncanonical cellular response that is specific to nonblocking replication stress, i.e., low-level stress (LoL-DDR), in primary cells. Although this response generates replication stress-induced reactive oxygen species (RIR), it triggers a program that prevents the accumulation of premutagenic 8-oxo-guanine (8-oxoG). Primary cells control RIR production via NADPH oxidases. Increasing the severity of replication stress above a precise threshold triggers the canonical DDR, leading to cell cycle arrest and RIR suppression, resulting in a peak-shaped dose response for RIR production. Here, we show that the LoL-DDR is dysregulated in cancer cell lines, which exhibit the following differences compared with primary cells: 1-RIR are not detoxified under high-level stress conditions, resulting in a continuous increase in the dose-response curve of RIR production; 2-RIR are not produced by NADPH oxidases; and 3-replication stress favors the accumulation of the premutagenic 8-oxoG. Moreover, using an in vitro breast cancer progression model, we show that LoL-DDR dysregulation occurs at an early stage of cancer progression. Since, conversely, ROS trigger replication stress this establishes a “vicious circle” involving replication-stress and ROS that continuously jeopardizes genome integrity that should fuel and amplify tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edfa2fab8a78a43f5b506788fc0cb6d9a375f31e" target='_blank'>
              Dysregulation of replication stress-induced ROS in transformed cell lines: a vicious circle at cancer initiation
              </a>
            </td>
          <td>
            S. Ragu, E. Dardillac, Sylvain Caillat, J. Ravanat, Bernard S. Lopez
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Background Outcomes under anti-PD-(L)1 therapy have been variable in advanced non-small cell lung cancer (NSCLC) without reliable predictive biomarkers so far. Targeted next-generation sequencing (NGS) of circulating tumor DNA (ctDNA) has demonstrated potential clinical utility to support clinical decisions, but requires prior tumor genetic profiling for proper interpretation, and wide adoption remains limited due to high costs. Methods Tumor-agnostic low-coverage ctDNA whole genome sequencing (lcWGS) was used to longitudinally track genome-wide copy number variations (CNVs) and fragmentation features in advanced NSCLC patients (n = 118 samples from 49 patients) and healthy controls (n = 57). Tumor PD-L1 expression was available for comparison. Findings Fragmentation features and CNVs were complementary indicators, whose combination significantly increased ctDNA detection compared to single-marker assessments (+ 20.3% compared to CNV analysis alone). Baseline fragment length alterations, but not CNVs, were significantly associated with subsequent progression-free survival (PFS; hazard ratio [HR] = 4.10, p = 6.58e-05) and could improve PFS predictions based on tumor PD-L1 expression alone (HR = 2.70, p = 0.019). Residual CNVs or aberrant fragmentation of ctDNA under ongoing therapy could stratify patients according to the subsequent response duration (median 5.8 vs. 47.0 months, p = 1.13e-06). The integrative analysis of ctDNA fragment characteristics at baseline, tumor PD-L1 expression, and residual ctDNA under ongoing treatment constituted the strongest independent predictor of PFS (p = 6.25e-05) and overall survival (p = 1.3e-03) in multivariable analyses along with other clinicopathologic variables. Interpretation This study demonstrates the feasibility and potential clinical utility of lcWGS for the tumor-agnostic stratification and monitoring of advanced NSCLC under PD-(L)1 blockade based on CNV and fragmentomic profiling. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03348-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/263d81db230c34f72ebf729f4d1766f8b109dae5" target='_blank'>
              Low-coverage whole genome sequencing of cell-free DNA to predict and track immunotherapy response in advanced non-small cell lung cancer
              </a>
            </td>
          <td>
            F. Janke, Mateo Gasser, A. Angeles, A. Riediger, Magdalena Görtz, Louise Appenheimer, Astrid K. Laut, Simon J. Ogrodnik, Sabrina Gerhardt, A. Stenzinger, M. Schneider, M. Thomas, P. Christopoulos, Holger Sültmann
          </td>
          <td>2025-03-08</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Most genetic variants associated with complex traits and diseases occur in non-coding genomic regions and are hypothesized to regulate gene expression. To understand the genetics underlying gene expression variability, we characterize 14,324 ancestrally diverse RNA-sequencing samples from the NHLBI Trans-Omics for Precision Medicine (TOPMed) program and integrate whole genome sequencing data to perform cis and trans expression and splicing quantitative trait locus (cis-/trans-e/sQTL) analyses in six tissues and cell types, most notably whole blood (N=6,454) and lung (N=1,291). We show this dataset enables greater detection of secondary cis-e/sQTL signals than was achieved in previous studies, and that secondary cis-eQTL and primary trans-eQTL signal discovery is not saturated even though eGene discovery is. Most TOPMed trans-eQTL signals colocalize with cis-e/sQTL signals, suggesting many trans signals are mediated by cis signals. We fine-map European UK BioBank GWAS signals from 164 traits and colocalize the resulting 34,107 fine-mapped GWAS signals with TOPMed e/sQTL signals, finding that of 10,611 GWAS signals with a colocalization, 7,096 GWAS signals colocalize with at least one secondary e/sQTL signal. These results demonstrate that larger e/sQTL analyses will continue to uncover secondary e/sQTL signals, and that these new signals will benefit GWAS interpretation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93d4d05c38b07b5509f977fc72067af450ca02ed" target='_blank'>
              Cross-cohort analysis of expression and splicing quantitative trait loci in TOPMed
              </a>
            </td>
          <td>
            Peter Orchard, T. Blackwell, L. Kachuri, Peter J. Castaldi, M. Cho, S. Christenson, P. Durda, Stacey B. Gabriel, C. Hersh, S. Huntsman, Seungyong Hwang, R. Joehanes, Mari Johnson, Xingnan Li, Honghuang Lin, Ching-Ti Liu, Yongmei Liu, A. Mak, A. Manichaikul, David T. Paik, A. Saferali, Joshua D. Smith, K. Taylor, Russell P. Tracy, Jiongming Wang, Mingqiang Wang, J. Weinstock, Jeffrey Weiss, Heather E. Wheeler, Ying Zhou, S. Zoellner, Joseph Wu, Luisa Mestroni, S. Graw, Matthew Taylor, Victor E. Ortega, Craig W Johnson, Weiniu Gan, Gonçalo R. Abecasis, Deborah A. Nickerson, Namrata Gupta, K. Ardlie, Prescott G. Woodruff, Yinan Zheng, Russell P. Bowler, D. Meyers, Alexander P Reiner, C. Kooperberg, E. Ziv, Vasan S Ramachandran, Martin G Larson, L. Cupples, E. G. Burchard, Edwin K. Silverman, Steve Rich, N. Heard-Costa, Hua Tang, Jerome I. Rotter, Albert V Smith, Daniel Levy, F. Aguet, Laura Scott, L. Raffield, Stephen C. J. Parker
          </td>
          <td>2025-02-21</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Abstract Background The efficacy of immune checkpoint blockade (ICB) depends on restoring immune recognition of cancer cells that have evaded immune surveillance. Transforming growth factor-beta (TGFβ) is associated with immune-poor, so-called cold tumors whereas loss of its signaling promotes DNA misrepair that could stimulate immune response. Methods We analyzed transcriptomic data from IMvigor210, The Cancer Genome Atlas, and Tumor Immune Syngeneic MOuse data sets to evaluate the predictive value of high βAlt, a score representing low expression of a signature consisting of TGFβ targets and high expression of genes involved in error-prone DNA repair. The immune context of βAlt was assessed by evaluating tumor-educated immune signatures. An ICB-resistant, high βAlt preclinical tumor model was treated with a TGFβ inhibitor, radiation, and/or ICB and assessed for immune composition and tumor control. Results We found that a high βAlt score predicts ICB response yet is paradoxically associated with an immune-poor tumor microenvironmentcancer in both human and mouse tumors. We postulated that high βAlt cancers consist of cancer cells in which loss of TGFβ signaling generates a TGFβ rich, immunosuppressive tumor microenvironment. Accordingly, preclinical modeling showed that TGFβ inhibition followed by radiotherapy could convert an immune-poor, high βAlt tumor to an immune-rich, ICB-responsive tumor. Mechanistically, TGFβ inhibition increased activated natural killer (NK) cells, which were required to recruit lymphocytes to respond to ICB in irradiated tumors. NK cell activation signatures were also increased in high βAlt, cold mouse and human tumors that responded to ICB. Conclusions These studies indicate that loss of TGFβ signaling competency and gain of error-prone DNA repair identifies a subset of cold tumors that are responsive to ICB. Our mechanistic studies show that inhibiting TGFβ activity can convert a high βAlt, cold tumor into ICB-responsive tumors via NK cells. A biomarker consisting of combined TGFβ, DNA repair, and immune context signatures is a means to prospectively identify patients whose cancers may be converted from cold to hot with appropriate therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16f08083204e934f5372293d784704ac2a30ac3a" target='_blank'>
              Identification of a conserved subset of cold tumors responsive to immune checkpoint blockade
              </a>
            </td>
          <td>
            Jade Moore, Jim Gkantalis, Inés Guix, William Chou, Kobe Yuen, Ann A. Lazar, Matthew Spitzer, Alexis J. Combes, M. Barcellos-Hoff
          </td>
          <td>2025-03-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>1</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/564696ec86cb40931955901184003710aa166dc0" target='_blank'>
              Abstract B017: An integrated workflow for multiomic characterization and functional precision medicine testing of solid tumor samples
              </a>
            </td>
          <td>
            Colin Paul, Amber Bullock, Anthony Chatman, Km Shams Ud Doha, Harrison Leong, Sylvia Beam, Chris Yankaskas, Pradip S. Thakuri, Matt Dallas, David Kuninger
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Objectives/Goals: Investigating the B-cell acute lymphoblastic leukemia (B-ALL)-associated germline SNP rs7090445, located in intron 3 of ARID5B, which is more frequently observed in individuals of Hispanic/Latino descent. Investigating the mechanisms behind this inherited single nucleotide polymorphisms (SNP) that may contribute to the higher incidence of B-ALL in this population. Methods/Study Population: Specific Aim 1: We hypothesize ARID5B SNP rs7090445 disrupts intrinsic enhancer function. Identification of critical DNA looping events impacted by ARID5B variants using Capture C. Affinity purification-mass spectrometry to identify potential ARID5B transcription mediators. Specific Aim 2: We hypothesize the B-ALL-associated SNP leads to a partial human B-cell differentiation block. Utilize Cas9-mediated homology-directed repair to create ARID5B SNP in primary human hematopoietic stem cells. Gene-edited HSCs will be differentiated into B cells using an ex vivo system. Fluorescence-activated cell sorting to sort our pool of cells into stages of B-cell development spectrum. Amplicon sequencing and variant allele frequency of rs7090445 SNP to evaluate its impact on B-cell development. Results/Anticipated Results: This proposal is conceptually innovative as it seeks to understand the mechanism by which the B-ALL-associated SNP rs7090445 in intron 3 of ARID5B disrupts enhancer function and investigates its impact on human B-cell development. Future research will investigate a tumor-suppressive role of ARID5B and whether it constitutes a “first-hit” of leukemogenesis. Discussion/Significance of Impact: Successful completion of this research will elucidate the critical role of the B-ALL-associated ARID5B SNP rs7090445 in human B-cell development and leukemogenesis. As this SNP is more prevalent in Hispanic/Latino populations, it will also provide crucial insights into the genetic factors behind the elevated incidence of B-ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4345d4a2e4bf5156f2152b887906b55727b5a5a1" target='_blank'>
              71 Investigating gene regulatory mechanisms associated with B-cell acute lymphoblastic leukemia incidence in Hispanic/Latino populations
              </a>
            </td>
          <td>
            Clarissa Garcia, Julian Grandvallet, Illaria Iacobucci, Jun Yang, C. Mullighan, Tzu Phang, Matthew T. Witkowski
          </td>
          <td>2025-03-26</td>
          <td>Journal of Clinical and Translational Science</td>
          <td>0</td>
          <td>103</td>
        </tr>

        <tr id="Chemoresistance represents a major challenge for osteosarcoma treatment. Despite the improved knowledge of cancer biology, the core determinants of cisplatin (DDP) resistance in osteosarcoma remain unclear and deserve further exploration. Here, RFWD3 is identified as a key regulator of DDP sensitivity in osteosarcoma using a genome-wide CRISPR screen. It is demonstrated that RFWD3 is overexpressed in post-chemotherapy osteosarcoma tissues compared to pre-chemotherapy tissues. Knocking out RFWD3 increased the sensitivity of osteosarcoma cells to DDP treatment. Mechanistically, RFWD3 bound to and ubiquitinated PHGDH at the Lys137 residue, promoting its degradation and conserving cellular oxidized nicotinamide adenine dinucleotide (NAD+). The resulting surplus of NAD+ enhanced the TCA cycle, leading to increased production of aspartic acid and glutamic acid for de novo nucleotide biosynthesis. In addition, virtual screening techniques are employed to identify Lomitapide as a specific inhibitor of the RFWD3-PHGDH interaction, capable of disrupting the binding between RFWD3 and PHGDH. It is found that Lomitapide exhibits a significant synergistic anti-osteosarcoma effect when combined with DDP. In conclusion, a specific role of RFWD3 in regulating nucleotide metabolism is revealed and comprised of targetable candidates for overcoming chemoresistance in osteosarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffe4a6c0c662b6150e0f25a7a03fb8e70f8ebcc2" target='_blank'>
              RFWD3 Reprograms Nucleotide Metabolism Through PHGDH to Induce Chemoresistance In Osteosarcoma.
              </a>
            </td>
          <td>
            Wenchao Zhang, Chi Yin, Lin Qi, Zhongyue Liu, Ruiling Xu, Chao Tu, Zhihong Li
          </td>
          <td>2025-02-28</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Tumorigenesis in mammals is driven by inherited genetic variants, environmental factors and random errors during normal DNA replication that lead to cancer-causing mutations. These factors initiate uncontrolled cellular proliferation and disrupt the regulation of critical checkpoints. A few mammalian species possess unique protective mechanisms that enable them to resist widespread cancer development and achieve longevity. Tissue-specific tumor protection adds another layer of complexity to this diversity. Breast cancer is a leading cause of human mortality, particularly among females. Driven by the need for new strategies in treatment and prevention, this opinion article explores and supports the idea that herbivores are more resistant to mammary cancer than carnivores and omnivores. This diversity has occurred despite the remarkably similar basic mammary biology. Herbivores’ meatless diet cannot explain the differences in cancer resistance, which have accompanied species segregation since the Jurassic era. To investigate the causes of this diversity, the characteristics of tumorigenesis in the human breast—and to a lesser extent in other carnivores—have been compared with data from retrospective analyses of bovine mammary tumor development across various locations over the past century. Well-established genomic, cellular, and systemic triggers of breast cancer exhibit different, or less pronounced tissue-specific activity in the bovine mammary gland, accompanied by novel bovine-specific protective mechanisms. Together, these factors contribute to the near absence of breast cancer in bovines and offer a basis for developing future anticancer strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0cb9fafc9e4ae8646c09de8af8d088ea1242ab7" target='_blank'>
              Mammalian Species-Specific Resistance to Mammary Cancer
              </a>
            </td>
          <td>
            I. Barash
          </td>
          <td>2025-03-06</td>
          <td>Journal of Mammary Gland Biology and Neoplasia</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Cancer predisposition syndromes mediated by recessive cancer genes generate tumors via somatic variants (second hits) in the unaffected allele. Second hits may or may not be sufficient for neoplastic transformation. Here we performed whole-genome and whole-exome sequencing on 479 tissue biopsies from a child with neurofibromatosis type 1, a multisystem cancer-predisposing syndrome mediated by constitutive monoallelic NF1 inactivation. We identified multiple independent NF1 driver variants in histologically normal tissues, but not in 610 biopsies from two nonpredisposed children. We corroborated this finding using targeted duplex sequencing, including a further nine adults with the same syndrome. Overall, truncating NF1 mutations were under positive selection in normal tissues from individuals with neurofibromatosis type 1. We demonstrate that normal tissues in neurofibromatosis type 1 commonly harbor second hits in NF1, the extent and pattern of which may underpin the syndrome’s cancer phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09992e3740e0c66b2f55eb33131f8350c3e4f847" target='_blank'>
              Cancer-independent somatic mutation of the wild-type NF1 allele in normal tissues in neurofibromatosis type 1
              </a>
            </td>
          <td>
            Thomas R. W. Oliver, A. Lawson, H. Lee-Six, Anna Tollit, Hyunchul Jung, Y. Hooks, R. Sanghvi, Matthew D. Young, T. Butler, Pantelis Nicola, T. Treger, S. Lensing, G. A. Burke, Kristian Aquilina, Ulrike Löbel, I. Cortés-Ciriano, Darren Hargrave, Mette Jorgensen, Flora A. Jessop, Tim H. H. Coorens, Adrienne M. Flanagan, K. Allinson, I. Martincorena, Thomas S Jacques, S. Behjati
          </td>
          <td>2025-02-25</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="Background TP53 mutations are linked to aggressive progression and chemoresistance in gastric cancer (GC). Frameshift mutation is the second most common mutation type of TP53. However, the consequences of this mutation type in GC were not well understood, and targeted therapies for cancer patients harboring frameshift mutations were also not established. Histone methylation significantly influences tumorigenesis in TP53-mutated cancers, and related inhibitors are emerging as specific therapeutic strategies. Methods and results By treating GC cell lines harboring various TP53 mutation types with a library of histone demethylase inhibitors, we identified that GSK690, a reversible inhibitor of lysine-specific demethylase 1 (LSD1), selectively inhibits GC cells harboring TP53 frameshift mutations without nuclear localization sequence (NLS) (termed TP53 Frameshift NLS), which accounts for 89% TP53 frameshift mutations in GC patients. GSK690 showed significant specific inhibition in vitro and in vivo against this subtype by inducing G1/S cell cycle arrest via the LSD1-CCNA2 axis. Importantly, dual-luciferase assays and ChIP-qPCR confirmed that the loss of transcriptional repression activities of p53 in drives LSD1 upregulation in TP53 Frameshift NLS cancer cells. Conclusions In summary, our results indicate that the nuclear localization deficiency of p53 accounts for increased expression of LSD1 in TP53 Frameshift NLS GCs. GSK690 inhibits cell cycle progression and tumor growth by suppressing aberrantly activated LSD1-CCNA2 signaling in this GC subtype, counteracting malignant proliferation and thereby providing a precise therapeutic strategy for GC patients with TP53 Frameshift NLS. Supplementary Information The online version contains supplementary material available at 10.1186/s13148-025-01829-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1998881bbe977bcf7672e918b2a1b2bf09e4b12e" target='_blank'>
              LSD1 is a targetable vulnerability in gastric cancer harboring TP53 frameshift mutations
              </a>
            </td>
          <td>
            Suzeng Wang, Chunyu Yang, Junhui Tang, Kaiqing Wang, Hao Cheng, Surui Yao, Zhaohui Huang, Bo-jian Fei
          </td>
          <td>2025-02-18</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Abstract Oncogene activation in normal untransformed cells induces DNA replication stress and creates a dependency on DNA damage response (DDR) mechanisms for cell survival. Different oncogenic stimuli signal via distinct mechanisms in every cancer setting. The DDR is also pathologically reprogrammed and deployed in diverse ways in different cancers. Because mutant KRAS is the driver oncogene in 90% of pancreatic ductal adenocarcinomas (PDACs), here we have investigated DDR mechanisms by which KRAS-induced DNA replication stress is tolerated in normal human pancreatic epithelial cells [human pancreatic nestin-expressing (HPNE) cells]. Using a candidate screening approach, we identify TRIP13 as a KRASG12V-induced messenger RNA that is also expressed at high levels in PDAC relative to normal tissues. Using genetic and pharmacological tools, we show that TRIP13 is necessary to sustain ongoing DNA synthesis and viability specifically in KRASG12V-expressing cells. TRIP13 promotes survival of KRASG12V-expressing HPNE cells in a homologous recombination (HR)-dependent manner. KRASG12V-expressing HPNE cells lacking TRIP13 acquire hallmark HR deficiency phenotypes, including sensitivity to inhibitors of translesion synthesis and poly-ADP ribose polymerase. Established PDAC cell lines are also sensitized to intrinsic DNA damage and therapy-induced genotoxicity following TRIP13 depletion. Taken together, our results expose TRIP13 as an attractive new and therapeutically tractable vulnerability of KRAS-mutant PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c52be837d7841e9eb90ef56e65d2eb6df322017a" target='_blank'>
              TRIP13 protects pancreatic cancer cells against intrinsic and therapy-induced DNA replication stress
              </a>
            </td>
          <td>
            J. R. Anand, G. Droby, Sayali Joseph, Urvi Patel, Xingyuan Zhang, Jeffrey A. Klomp, C. Der, Jeremy E. Purvis, Samuel C. Wolff, Jessica Bowser, Cyrus Vaziri
          </td>
          <td>2025-03-01</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>122</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb7e063d7f27e7ff5ba777247b0efc2ec0bcfdb4" target='_blank'>
              Functional analysis of cancer-associated germline risk variants.
              </a>
            </td>
          <td>
            Laura N. Kellman, Poornima H. Neela, Suhas Srinivasan, Z. Siprashvili, Ronald L. Shanderson, Audrey W. Hong, Deepti S Rao, Douglas F Porter, D. Reynolds, Robin M. Meyers, M. Guo, Xue Yang, Yang Zhao, Glenn G Wozniak, L. Donohue, R. Shenoy, Lisa A Ko, Duy T. Nguyen, Smarajit Mondal, Omar S. Garcia, L. Elcavage, Ibtihal Elfaki, Nathan S. Abell, Shiying Tao, Christopher M Lopez, Stephen B. Montgomery, Paul A. Khavari
          </td>
          <td>2025-02-17</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Rationale: Ovarian cancer is a highly lethal gynecological malignancy with common platinum resistance. Lactylation is involved in multiple biological processes. Thus, we explored the role of histone and non-histone lactylation in platinum resistance, providing a potential therapeutic target to overcome platinum resistance in ovarian cancer. Methods: We utilized gene set enrichment analysis to investigate lactylation-related pathway alterations between platinum-resistant and platinum-sensitive patients from the TCGA cohort. Differential expression of H3K9la was demonstrated using Western blotting and immunohistochemistry. Progression-free and overall survival were determined using a log-rank test. Drug response to cisplatin was evaluated by CCK8, apoptosis flow cytometry, and clonogenic assays in vitro. ChIP-seq and ChIP-qPCR assays were performed to identify downstream targets of H3K9la, which was further confirmed by qRT-PCR. LC-MS/MS was conducted to identify specific lactylation sites for RAD51. Co-IP was used to reveal the interaction between GCN5 and H3K9la or RAD51la. Cell line-derived and patient-derived xenograft (PDX) models of ovarian cancer were constructed for the in vivo experiments. Results: Our study showed elevated histone lactylation, especially of H3K9la, in platinum-resistant ovarian cancer. Moreover, high H3K9la indicated platinum resistance and poor prognosis of ovarian cancer. Impairing H3K9la enhanced response to cisplatin. Mechanistically, H3K9la directly activated RAD51 and BRCA2 expression to facilitate homologous recombination (HR) repair. Furthermore, RAD51K73la enhanced HR repair and subsequently conferred cisplatin resistance. H3K9la and RAD51K73la shared the same upstream regulator, GCN5. Notably, a GCN5 inhibitor remarkably improved the tumor-killing ability of cisplatin in PDX models of ovarian cancer. Conclusions: Our study demonstrated the essential role of histone and RAD51 lactylation in HR repair and platinum resistance. It also identified a potential therapeutic strategy to overcome platinum resistance and improve prognosis in ovarian cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5ab75585545ad6fb4a52167fc59fff4dafb2d9e" target='_blank'>
              Targeting platinum-resistant ovarian cancer by disrupting histone and RAD51 lactylation
              </a>
            </td>
          <td>
            Chenggong Sun, Xiao Li, Qiuli Teng, Xihan Liu, Li Song, H. Schiöth, Huan Wu, Xinyu Ma, Zhaoyang Zhang, Changjian Qi, Haocheng Zhang, Kun Song, Qing Zhang, Beihua Kong
          </td>
          <td>2025-02-10</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c201d9c82319326a06dbf117cce45d84a25d29e" target='_blank'>
              Identification of a germline deep intronic PTEN-deletion leading to exonization through whole genome and targeted RNA sequencing.
              </a>
            </td>
          <td>
            Morgane Boedec, Camille Aucouturier, Mathias Cavaillé, R. Leman, L. Castéra, Hélène Delhomelle, N. Uhrhammer, Virginie Bernard, Sophie Giraud, Eulalie Lasseaux, Natalie Jones, Marie Bidart, N. Boutry‐Kryza, C. Noguès, C. Colas, C. Maugard, Sophie Krieger, Ahmed Bouras
          </td>
          <td>2025-02-07</td>
          <td>Familial cancer</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb7435e798afc5c276c5bcf5f266a5cdfa975999" target='_blank'>
              Abstract B008: Targeting DNA damage responsive pathways in cancer therapy
              </a>
            </td>
          <td>
            Junjie Chen
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Ovarian cancer remains one of the most lethal gynecological malignancies owing to its high recurrence rate and chemotherapeutic resistance. MYC is a well-known proto-oncogene that is frequently amplified in ovarian cancer and has been implicated in drug resistance. Previously, we established a new promoter–reporter system combined with a CRISPR activation library to identify unknown MYC regulators, and M1AP was identified as a novel MYC regulator. However, considering the insufficient explanation for the absence of guide RNA (gRNA) of MYC, this present study explored methods to prevent the gRNA of MYC itself from binding. This study first modified the promoter–reporter vector to improve its quality, then conducted CRISPR screening and analyzed candidate genes as MYC promoter regulators using next-generation sequencing in OVSAHO ovarian cancer cells. Eighty-six genes had ≥ 1000 reads, and Pearson’s correlation coefficient analysis was performed on the cBioPortal of the Cancer Genomics database. Fourteen genes were identified as candidate MYC regulators with positive and significant correlations with MYC. Seven genes, including CYP4v2, ASPH, ANP32D, PCED1A, ABI1, FUZ, and HOOK2, demonstrated significantly higher luciferase activity than the control genes. Four genes, including ABI1, PCED1A, HOOK2, and CYP4v2, activated the MYC promoter, which showed over twofold higher activity than the control when overexpressed using a vector. In conclusion, four genes that activate MYC promoters were identified in an ovarian cancer cell line using the CRISPR library system with a modified promoter–reporter tool. These results will prove helpful in the development of novel treatment strategies for ovarian cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s13048-025-01644-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9220d2085ff0b29849fe6d91dbca5cdaecc789b" target='_blank'>
              Modified screening of MYC promotor region elements using the CRISPR library in ovarian cancer
              </a>
            </td>
          <td>
            Akiko Yamamoto, Yosuke Tanaka, S. Ishibashi, Masumi Ikeda, Keisuke Sugita, Masanori Ono, Hirotaka Nishi, Morito Kurata
          </td>
          <td>2025-04-02</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Over the past few years, the tumor microbiome is increasingly recognized for its multifaceted involvement in cancer initiation, progression, and metastasis. With the application of 16S ribosomal ribonucleic acid (16S rRNA) sequencing, the intratumoral microbiome, also referred to as tumor-intrinsic or tumor-resident microbiome, has also been found to play a significant role in the tumor microenvironment (TME). Understanding their complex functions is critical for identifying new therapeutic avenues and improving treatment outcomes. This review first summarizes the origins and composition of these microbial communities, emphasizing their adapted diversity across a diverse range of tumor types and stages. Moreover, we outline the general mechanisms by which specific microbes induce tumor initiation, including the activation of carcinogenic pathways, deoxyribonucleic acid (DNA) damage, epigenetic modifications, and chronic inflammation. We further propose the tumor microbiome may evade immunity and promote angiogenesis to support tumor progression, while uncovering specific microbial influences on each step of the metastatic cascade, such as invasion, circulation, and seeding in secondary sites. Additionally, tumor microbiome is closely associated with drug resistance and influences therapeutic efficacy by modulating immune responses, drug metabolism, and apoptotic pathways. Furthermore, we explore innovative microbe-based therapeutic strategies, such as engineered bacteria, oncolytic virotherapy, and other modalities aimed at enhancing immunotherapeutic efficacy, paving the way for microbiome-centered cancer treatment frameworks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb1690fc16e391c65c0901d80f1a2ddc6ab3025c" target='_blank'>
              Tumor microbiome: roles in tumor initiation, progression, and therapy
              </a>
            </td>
          <td>
            Shengxin Zhang, Jing Huang, Zedong Jiang, Huan Tong, Xuelei Ma, Yang Liu
          </td>
          <td>2025-02-08</td>
          <td>Molecular Biomedicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is one of the leading causes of cancer-related death worldwide, and its onset and progression are closely associated with epigenetic modifications, particularly post-translational modifications of histones (HPTMs). In recent years, advances in mass spectrometry (MS) have revealed a series of novel HPTMs, including succinylation (Ksuc), citrullination (Kcit), butyrylation (Kbhb), lactylation (Kla), crotonylation (Kcr), and 2-hydroxyisobutyrylation (Khib). These modifications not only expand the histone code but also play significant roles in key carcinogenic processes such as tumor proliferation, metastasis, and metabolic reprogramming in HCC. This review provides the first comprehensive analysis of the impact of novel HPTMs on gene expression, cellular metabolism, immune evasion, and the tumor microenvironment. It specifically focuses on their roles in promoting tumor stem cell characteristics, epithelial–mesenchymal transition (EMT), and therapeutic resistance. Additionally, the review highlights the dynamic regulation of these modifications by specific enzymes, including “writers,” “readers,” and “erasers.”">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa3200dafb5f7a30deaf8fb5764586a7d405d1e8" target='_blank'>
              Novel histone modifications and liver cancer: emerging frontiers in epigenetic regulation
              </a>
            </td>
          <td>
            Zhonghua Wang, Ziwen Liu, Mengxin Lv, Zhou Luan, Tao Li, Jinhua Hu
          </td>
          <td>2025-02-20</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Patient sex influences a wide range of cancer phenotypes, including prevalence, response to therapy and survival endpoints. Molecular sex differences have been identified at all levels of the central dogma. It is hypothesized that these molecular differences may drive the observed clinical sex differences. Yet despite a growing catalog of molecular sex differences in a range of cancer types, their specific functional consequences remain unclear. To directly assess how patient sex impacts cancer cell function, we evaluated 1,209 cell lines subjected to CRISPR knockout, RNAi knockdown or drug exposures. Despite limited statistical power, we identified pan- and per-cancer sex differences in gene essentiality in six sex-linked and fourteen autosomal genes, and in drug sensitivity for two compounds. These data fill a gap in our understanding of the link between sex-differential molecular effects and patient phenotypes. They call for much more careful and systematic consideration of sex-specific effects in mechanistic and functional studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5d6a69917ecc07994b736e3c65c24ffb01a16f6" target='_blank'>
              Sex Differences in Cancer Functional Genomics: Gene Dependency and Drug Sensitivity
              </a>
            </td>
          <td>
            Nicole Zeltser, Chenghao Zhu, Jieun Oh, Constance H. Li, P. Boutros
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee23eae46a9ddd2433ad4acafc75ff3551fa2d98" target='_blank'>
              ANGEL2 modulates wildtype TP53 translation and doxorubicin chemosensitivity in colon cancer.
              </a>
            </td>
          <td>
            C. A. Lucchesi, S. Mantrala, Darren Tran, N. Batra, Avani Durve, Conner Suen, Jin Zhang, Paramita M Ghosh, Xinbin Chen
          </td>
          <td>2025-03-07</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Cancers that retain wild type TP53 presumably harbor other clonal alterations that permitted their precursors to bypass p53-mediated growth suppression. Consequently, studies that employ TP53-wild type cancer cells and their isogenic derivatives may systematically fail to appreciate the full scope of p53 functionality. Several TP53 phenotypes are known to be absent in the widely used isogenic HCT116 colorectal cancer (CRC) model, which originated from a tumor that had retained wild type TP53. In contrast, we show that restoration of p53 in the TP53-mutant CRC cell line DLD-1 impeded cell proliferation, increased levels of senescence and sensitized cells to ionizing radiation (IR). To study p53 in a non-cancer context, we disrupted TP53 in hTERT-RPE1 cells. Derived from primary cells that were immortalized in vitro, hTERT-RPE1 expressed striking p53-dependent phenotypes and appeared to select for p53 loss during routine culture. hTERT-RPE1 expressed a p53-responsive transcriptome that was highly representative of diverse experimental systems. We discovered several novel downstream p53 targets of potential clinical relevance including ALDH3A1, which is involved in the detoxification of aldehydes and the metabolism of reactive oxygen species, and nectin cell adhesion molecule 4 (NECTIN4) which encodes a secreted surface protein that is overexpressed in many tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56bfbd25a9a6de3dee6da6ae82f799aa171d1483" target='_blank'>
              Robust p53 phenotypes and prospective downstream targets in telomerase-immortalized human cells
              </a>
            </td>
          <td>
            Jessica J. Miciak, Lucy Petrova, Rhythm Sajwan, Aditya Pandya, Mikayla Deckard, Andrew J Munoz, Fred Bunz
          </td>
          <td>2025-02-18</td>
          <td>Oncotarget</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc949647e3b9b0b845365c842f739d457eb9e866" target='_blank'>
              Functional profiling of p53 and RB cell cycle regulatory proficiency suggests mechanism-driven molecular stratification in endometrial carcinoma.
              </a>
            </td>
          <td>
            Zelei Yang, S. Mogre, Hyeji Jun, Ruiyang He, Sneha Ghosh Chaudhary, U. R. Bhattarai, Shannan J. Ho Sui, Ursula A Matulonis, Suzan Lazo, Aniket Shetty, A. Cameron, Quang-De Nguyen, Sarah J Hill
          </td>
          <td>2025-02-11</td>
          <td>Cancer research communications</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89e38976a47d12226a011902bfacb13e2fb36e59" target='_blank'>
              Genome-wide CRISPR screen in human T cells reveals regulators of FOXP3.
              </a>
            </td>
          <td>
            Kelvin Y. Chen, Tatsuya Kibayashi, Ambre Giguelay, Mayu Hata, Shunsuke Nakajima, N. Mikami, Yusuke Takeshima, Kenji Ichiyama, Ryusuke Omiya, Leif S. Ludwig, Kunihiro Hattori, Shimon Sakaguchi
          </td>
          <td>2025-03-26</td>
          <td>Nature</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29f17ec0f574b24441e84e586e6063343737a2a9" target='_blank'>
              Massively parallel in vivo Perturb-seq screening.
              </a>
            </td>
          <td>
            Xinhe Zheng, Patrick C Thompson, Cassandra M. White, Xin Jin
          </td>
          <td>2025-02-12</td>
          <td>Nature protocols</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="Schlafen11 (SLFN11) is a key gene in p53-independent apoptosis through ribosome stalling; however, systematic research has been conducted on its role in the tumor immune microenvironment, clinical application, and immunotherapy response across pan-cancer.Public data were downloaded and multi-omics approaches were used to investigate the relationship between the expression level of SLFN11 and spatial position, biological function, immune landscape, and clinical application values. Cell Counting Kit-8 assay and quantitative real-time PCR were used to validate the expression level of SLFN11 and drug sensitivity in colorectal cancer samples.Our study revealed that SLFN11 was downregulated in most cancers and correlated with DNA repair, the P53 pathway and immune response in tumor development progress by multi-omics analysis. Dysregulated SLFN11 is accompanied by several immune cell infiltrations and immune-related regulators, which can be a promising screening and prognostic biomarker and chemotherapy predictive target for clinical application. In vitro experiments proved that downregulated SLFN11 is a useful diagnostic biomarker and is linked to imatinib resistance in colorectal cancer.The expression level of SLFN11 has a substantial promise as a valuable biomarker for diagnosis and a predictive indicator for assessing the effectiveness of chemotherapy and immunotherapy in human cancers, which deserves further additional basic experiments and clinical trials to prove.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c098d42efd569f7e477b717e0186198035c93eb9" target='_blank'>
              Comprehensive pan-cancer analysis indicates key gene of p53-independent apoptosis is a novel biomarker for clinical application and chemotherapy in colorectal cancer
              </a>
            </td>
          <td>
            Jianing Yan, Jingzhi Wang, Min Miao, Yongfu Shao
          </td>
          <td>2025-03-27</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [1, 20],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>