<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-01-27 09:25:48 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental and Molecular Medicine, Experimental & Molecular Medicine</td>
          <td>4</td>
          <td>49</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum.
                </a>
              </td>
          <td>
            Gustavo S França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>14</td>
          <td>41</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Phenotypic plasticity, the capacity of cells to transition between distinct phenotypic and lineage states over time, is a genetically and epigenetically encoded trait essential for normal development and adult tissue homeostasis. In cancer, phenotypic plasticity programs can be deployed aberrantly to enable disease progression and acquired therapeutic resistance. Cancer phenotypic plasticity is a current barrier to achieving cures for advanced cancers using available molecularly targeted therapies. This review summarizes the complex and interconnected molecular pathways implicated in phenotypic plasticity, both in the context of normal tissue homeostasis and cancer. Molecular pathways convergent between these contexts are highlighted while pathways enabling plasticity are distinguished from those that specify the phenotype of already plastic cells. Key unresolved questions in the field are discussed along with emerging technologies that may be used to help answer them.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91d0b45e5336c08fda7f8b88d683482dae5405ff" target='_blank'>
              The molecular determinants of phenotypic plasticity in homeostasis and neoplasia
              </a>
            </td>
          <td>
            Bradley Balk, David W. Goodrich
          </td>
          <td>2024-12-13</td>
          <td>Cancer Heterogeneity and Plasticity</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Prostate cancer is a common malignancy that in 5%-30% leads to treatment-resistant and highly aggressive disease. Metastasis-potential and treatment-resistance is thought to rely on increased plasticity of the cancer cells-a mechanism whereby cancer cells alter their identity to adapt to changing environments or therapeutic pressures to create cellular heterogeneity. To understand the molecular basis of this plasticity, genomic studies have uncovered genetic variants to capture clonal heterogeneity of primary tumors and metastases. As cellular plasticity is largely driven by non-genetic events, complementary studies in cancer epigenomics are now being conducted to identify chromatin variants. These variants, defined as genomic loci in cancer cells that show changes in chromatin state due to the loss or gain of epigenomic marks, inclusive of histone post-translational modifications, DNA methylation and histone variants, are considered the fundamental units of epigenomic heterogeneity. In prostate cancer chromatin variants hold the promise of guiding the new era of precision oncology. In this review, we explore the role of epigenomic heterogeneity in prostate cancer, focusing on how chromatin variants contribute to tumor evolution and therapy resistance. We therefore discuss their impact on cellular plasticity and stochastic events, highlighting the value of single-cell sequencing and liquid biopsy epigenomic assays to uncover new therapeutic targets and biomarkers. Ultimately, this review aims to support a new era of precision oncology, utilizing insights from epigenomics to improve prostate cancer patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8f247f6de2138b02c15cb85166ee95210b75593" target='_blank'>
              Epigenomics-guided precision oncology: Chromatin variants in prostate tumor evolution.
              </a>
            </td>
          <td>
            K. Furlano, Tina Keshavarzian, Nadine Biernath, Annika Fendler, M. De Santis, Joachim Weischenfeldt, Mathieu Lupien
          </td>
          <td>2025-01-23</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Phenotypic plasticity, the ability of cells to adapt their behavior in response to genetic or environmental changes, is a fundamental biological process that drives cellular diversity in both normal and pathological contexts, including in tumor evolution. While chromosomal instability and somatic copy number alterations (CNAs) are known to influence cellular states, it remains difficult to separate genetic from cell non-autonomous mechanisms that govern transcriptional variability. Here, we present Echidna, a Bayesian hierarchical model that integrates single-cell RNA sequencing (scRNA-seq) and bulk whole-genome sequencing (WGS) data to quantify the impact of CNAs on gene expression dynamics. By jointly inferring clone-specific CNA profiles and uncovering clonal dependencies, Echidna bridges genomic and transcriptomic landscapes within and across multiple time points, enabling the decoupling of gene dosage effects from cell-extrinsic factors on phenotypic plasticity. Applying Echidna to patient tumor specimens, we demonstrate its superior performance in clonal reconstruction and derive insights into resistance mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b10523ab303976a5ed9fe5bc9d69a9a688fccfd9" target='_blank'>
              Echidna: A Bayesian framework for quantifying gene dosage effect impacting phenotypic plasticity
              </a>
            </td>
          <td>
            Joy Linyue Fan, Mingxuan Zhang, William O’Brien, Joshua D. Myers, Johannes C. Melms, Jana Biermann, Edridge K. D’Souza, Somnath Tagore, Nicolas Beltran-Velez, Kevin Hoffer-Hawlik, Alexander Preau, Isha Arora, Sharanya Chatterjee, Benjamin Izar, Elham Azizi
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Nongenetic plasticity has emerged as a key driver of cancer drug resistance, yet its precise origins, nature, and consequences are not fully clarified. This review examines technological, computational, and conceptual developments in the nongenetic determinants of drug resistance. We begin by proposing refined definitions of cellular state, fate, and plasticity. We subsequently contextualize the findings from multimodal approaches to investigate plasticity, highlighting how new single-cell lineage-tracing methods provide opportunities for quantitatively capturing state transitions, their timescales and heritability, and how they contribute to resistance mechanisms. We also draw parallels with concepts from developmental biology and microbial persistence research. Next, we cover the role that computational approaches have played in revealing the otherwise latent patterns of heterogeneity that underlie plasticity from complex datasets. We conclude by emphasizing the need for standardized terminology in this rapidly evolving field and the path from bench to bedside.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e38fadd15ca85570a489d3058bda72d96573ce1d" target='_blank'>
              Navigating Nongenetic Plasticity in Cancer Drug Resistance
              </a>
            </td>
          <td>
            Hanxiao Sun, Nitu Kumari, Madeline E. Melzer, Vadim Backman, Yogesh Goyal
          </td>
          <td>2024-12-11</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Mutations and genome instability are hallmarks of cancer. The advanced Next Generation Sequencing(NGS) technology now enables rapid and cost-effective analysis for tumoral and normal genomic translational research. In this research, mRNA sequencing was applied to get the whole transcriptome data of mouse B16F10 melanoma and CT26 colon carcinoma cells. Up regulation genes and down regulation genes were identified in tumor cells with differentially expressed genes(DEGs) analysis. The DEGs were analyzed with GO, KEGG and Reactome pathway enrichment study. Results demonstrated the DEGs and enriched pathways were mainly associated with cell proliferation and biosynthetic processing. Further, gene fusion and alternative splicing analysis reveled fusion genes and skipped exons were extensive exist within the chromosomes of tumor cells. Together, the integrated DEGs and bioinformatics analysis in this article may shed light on further research of cancer treating targets and molecular mechanisms of carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6e94aa8f9282678d155caf1a32035cd06536087" target='_blank'>
              Identification and analysis of DEGs of mouse tumor cells by integrated bioinformatics methods
              </a>
            </td>
          <td>
            Xizhong Jing, Baiyong Li, Yifei Wang
          </td>
          <td>2024-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer cells within tumors exhibit a wide range of phenotypic states driven by non-genetic mechanisms in addition to extensively studied genetic alterations. Conversions among cancer cell states can result in intratumoral heterogeneity which contributes to metastasis and development of drug resistance. However, mechanisms underlying the initiation and/or maintenance of such phenotypic plasticity are poorly understood. In particular, the role of intercellular communications in phenotypic plasticity remains elusive. In this study, we employ a multiscale inference-based approach using single-cell RNA sequencing (scRNA-seq) data to explore how intercellular interactions influence phenotypic dynamics of cancer cells, particularly cancers undergoing epithelial-mesenchymal transition. Our inference approach reveals that signaling interactions between cancerous cells in small cell lung cancer (SCLC) result in seemingly contradictory behaviors—reinforcing the cellular phenotypes and maintaining population-level intratumoral heterogeneity. Additionally, we find a recurring signaling pattern across multiple types of cancer in which the mesenchymal-like subtypes utilize signals from other subtypes to reinforce its phenotype, further promoting the intratumoral heterogeneity. We use a mathematical model based on ordinary differential equations to show that inter-subtype communication accelerates the development of heterogeneous tumor populations. Our work highlights the critical role of intercellular signaling in sustaining intratumoral heterogeneity, and our approach of computational analysis of scRNA-seq data can infer inter- and intra-cellular signaling networks in a holistic manner. Significance Single cell-based inference approach reveals a key role of intercellular signaling in maintaining intratumoral heterogeneity. Cell-cell communications stabilize newly acquired cell states and diverse phenotypes of cell populations in multiple cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fa5cf6c7e9bc3e2c2f27d636cb203f597857db1" target='_blank'>
              Intercellular signaling stabilizes single-cell level phenotypic transitions and accelerates the reestablishment of equilibrium of heterogeneous cancer cell populations
              </a>
            </td>
          <td>
            Daniel Lopez, Darren R Tyson, Tian Hong
          </td>
          <td>2025-01-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Understanding the molecular mechanisms underlying rapid adaptation to stress is a fundamental question in evolutionary biology. A common mechanism is copy number changes, which can affect single genes, larger genomic regions, or even entire chromosomes in the case of eukaryotes (aneuploidy). Bacterial species with genomes comprising more than one chromosome are common, yet the role of adaptive copy number changes affecting entire chromosomes is unknown. Here, by using a bacterium with two chromosomes (chrI and chrII) as model system and combining evolution experiments in the presence of severe nutrient stress with whole-genome sequencing, we show that chromosomal copy number changes can drive rapid adaptation. We detected increased copy number of the entire chrII in almost half of the sequenced clones, isolated from twelve independent populations. In most cases, copy number increases were driven by mutations in the replication protein of chrII. A higher chrII copy number is likely to be beneficial because chrII codes for the operon responsible for metabolizing D-gluconic acid, the only carbon source present in the medium. Our results revealed a novel mechanism for rapid adaptation in bacteria. Given the prevalence of bacteria with secondary chromosomes, and the enrichment of secondary chromosomes in genes associated with interaction with the environment, this may be an important mechanism for adapting to novel environments that has been overlooked.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68e1750352edfd93ae418d5a02e63e98887bb1a2" target='_blank'>
              Chromosomal plasticity can drive rapid adaptation in bacteria
              </a>
            </td>
          <td>
            Luis Vega-Cabrera, O. Skovgaard, Macarena Toll-Riera
          </td>
          <td>2024-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae4bbd7ae49e10933acda8e20cbd7e3bb473c50d" target='_blank'>
              Endosomal pH is an evolutionarily conserved driver of phenotypic plasticity in colorectal cancer
              </a>
            </td>
          <td>
            Hari Prasad, H. Bv, A. R. Subbalakshmi, S. Mandal, M. Jolly, Sandhya S. Visweswariah
          </td>
          <td>2024-12-19</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Intra tumor heterogeneity complicates cancer therapy by providing tumors with the ability to alter their phenotypes and become more therapy resistant. Here, we tested the hypothesis that identifying and modulating expression of key state-specific transcription factors could be used as a strategy for driving cells to a more therapy-sensitive state. Recent single-cell studies have explored the inter and intra tumoral heterogeneity of osteosarcoma and identified gene pathways enriched in specific cell states. For example, metastatic tumors are characterized by an expression of genes in the TNF-α, PI3K, TGFß and mTOR pathways. We identified similar profiles in osteosarcoma patient-derived xenograft-derived cell lines and potential transcription factor drivers of these states. We then used perturb-seq to downregulate expression of key transcription factors and evaluated the effect of these modulations on single cell RNA profiles and drug responses. Knockdown of NFE2L3 or NR0B1 increased the proportion of cells sensitive to targeted therapy. This approach, which could potentially be applied to other cancers, could be used as a strategy to increase the response to targeted therapies by increasing the proportion of cells in a drug-sensitive state. Highlights Distinct transcriptomic states were identified in osteosarcoma cell lines using single-cell RNA sequencing. Lineage tracing identified states with differential sensitivity to therapy. Using perturb-seq, we identified transcription factors that drive cells towards a more sensitive state. The transcription factors NR0B1 and NFE2L3 were identified as targets capable of reprogramming cells to a sensitive state.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6bc16f1a1ca4653fbc0122d06703e74482e021a" target='_blank'>
              Targeting Osteosarcoma heterogeneity to improve therapeutic response
              </a>
            </td>
          <td>
            Brenda Melano, Truc Dinh, Bahar Zirak, Mehran Karimzadeh, Brian J Woo, Benedict Choi, Hani Goodarzi, E. A. Sweet-Cordero, San Francisco
          </td>
          <td>2025-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c3eb15125d0d2e4a0c6a634c10b2bffbe96d5f8" target='_blank'>
              Epigenetic modifications and metabolic gene mutations drive resistance evolution in response to stimulatory antibiotics.
              </a>
            </td>
          <td>
            Hui Lin, Donglin Wang, Qiaojuan Wang, Jie Mao, Lutong Yang, Yaohui Bai, Jiuhui Qu
          </td>
          <td>2025-01-16</td>
          <td>Molecular systems biology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d50475b8002866903aae4806efbfc187f7cb121" target='_blank'>
              Canonical and noncanonical NOTCH signaling in the nongenetic resistance of cancer: distinct and concerted control.
              </a>
            </td>
          <td>
            Xianzhe Huang, Wenwei Chen, Yanyan Wang, Dmytro Shytikov, Yanwen Wang, Wangyi Zhu, Ruyi Chen, Yuwei He, Yanjia Yang, Wei Guo
          </td>
          <td>2025-01-02</td>
          <td>Frontiers of medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Ex vivo drug response profiling is emerging as a valuable tool for identifying drug resistance mechanisms and advancing precision medicine in hematological cancers. However, the functional impact of dysregulation of the epigenome and transcriptome in this context remains poorly understood. In this study, we combined ex vivo drug sensitivity profiling with transcriptomic and epigenomic analyses in diagnostic samples from 597 pediatric B-cell precursor acute lymphoblastic leukemia (BCP-ALL) patients. Ex vivo resistance to antimetabolites (e.g., cytarabine, thioguanine), glucocorticoids (e.g., dexamethasone, prednisolone), and doxorubicin was independently associated with reduced relapse-free survival (RFS, p < 0.05). Molecular profiling identified pre-treatment DNA methylation and gene expression patterns distinguishing resistant from sensitive cases, revealing key drug resistance signatures. These included aberrant expression of genes related to heme metabolism (e.g. ATPV06A) and KRAS signaling (e.g. ABCB1). Notably, we also observed atypical expression of genes usually restricted to T-cells and other immune cells (e.g., ITK) in resistant BCP-ALL cells. Our findings demonstrate that ex vivo drug response patterns are predictive of clinical outcomes and reflect intrinsic molecular states associated with drug tolerance. This integrative multi-omics approach highlights potential therapeutic targets and underscores the value of functional precision medicine in identifying treatment vulnerabilities in pediatric ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1b0bb7a98b1f284795f68f2172436eb0a2df655" target='_blank'>
              Ex Vivo Drug Responses and Molecular Profiles of 597 Pediatric Acute Lymphoblastic Leukemia Patients
              </a>
            </td>
          <td>
            Anna Pia Enblad, O. Krali, Henrik Gezelius, Anders Lundmark, Kristin Blom, Claes Andersson, J. Palle, B. Frost, S. Ryhänen, Trond Flægstad, Ó. Jónsson, Kjeld Schmiegelow, Arja Harila, Peter Nygren, Rolf Larsson, Gudmar Lönnerholm, J. Nordlund, Anna Pia
          </td>
          <td>2024-12-20</td>
          <td>None</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Small cell carcinoma of the esophagus (SCCE) is a rare and aggressively progressing malignancy that presents considerable clinical challenges.Although chemotherapy can effectively manage symptoms during the earlystages of SCCE, its long-term effectiveness is notably limited, with theunderlying mechanisms remaining largely undefined. In this study, weemployed single-cell RNA sequencing (scRNA-seq) to analyze SCCE samplesfrom a single patient both before and after chemotherapy treatment. Our analysisrevealed significant cellular plasticity and alterations in the tumormicroenvironment’s cellular composition. Notably, we observed an increase intumor cell diversity coupled with reductions in T cells, B cells, and myeloid-likecells. The pre-treatment samples predominantly featured carcinoma cells in amiddle transitional state, while post-treatment samples exhibited an expandedpresence of cells in terminal, initial-to-terminal (IniTerm), and universally alteredstates. Further analysis highlighted dynamic interactions between tumor cells andimmune cells, with significant changes detected in key signaling pathways, suchas TIGIT-PVR and MDK-SDC4. This study elucidates the complex dynamics of cellplasticity in SCCE following chemotherapy, providing new insights and identifyingpotential therapeutic targets to enhance treatment efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ae84357d27c8ef8315335bde0ededde5ffcf7d8" target='_blank'>
              Single-cell RNA sequencing elucidates cellular plasticity in esophageal small cell carcinoma following chemotherapy treatment
              </a>
            </td>
          <td>
            Qinkai Zhang, Ziyu Gao, Ru Qiu, Jizhao Cao, Chunxiao Zhang, Wei Qin, Meiling Yang, Xinyue Wang, Ciqiu Yang, Jie Li, Dongyang Yang
          </td>
          <td>2025-01-09</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="KRAS was long deemed undruggable until the discovery of the switch-II pocket facilitated the development of specific KRAS inhibitors. Despite their introduction into clinical practice, resistance mechanisms can limit their effectiveness. Initially, tumors rely on mutant KRAS, but as they progress, they may shift to alternative pathways, resulting in intrinsic resistance. This resistance can stem from mechanisms like epithelial-to-mesenchymal transition (EMT), YAP activation, or KEAP1 mutations. KRAS inhibition often triggers cellular rewiring to counteract therapeutic pressure. For instance, feedback reactivation of signaling pathways such as MAPK, mediated by receptor tyrosine kinases, supports tumor cell survival. Inhibiting KRAS disrupts protein homeostasis, but reactivation of MAPK or AKT can restore it, aiding tumor cell survival. KRAS inhibition also causes metabolic reprogramming and protein re-localization. The re-localization of E-cadherin and Scribble from the membrane to the cytosol causes YAP to translocate to the nucleus, where it drives MRAS transcription, leading to MAPK reactivation. Emerging evidence indicates that changes in cell identity, such as mucinous differentiation, shifts from alveolar type 2 to type 1 cells, or lineage switching from adenocarcinoma to squamous cell carcinoma, also contribute to resistance. In addition to these nongenetic mechanisms, secondary mutations in KRAS or alterations in upstream/downstream signaling proteins can cause acquired resistance. Secondary mutations in the switch-II pocket disrupt drug binding, and known oncogenic mutations affect drug efficacy. Overcoming these resistance mechanisms involves enhancing the efficacy of drugs targeting mutant KRAS, developing broad-spectrum inhibitors, combining therapies targeting multiple pathways, and integrating immune checkpoint inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83c1ae3fc98dfd6ef9151275fdf77ddcf612521a" target='_blank'>
              Mechanisms of Resistance to KRAS Inhibitors: Cancer Cells' Strategic Use of Normal Cellular Mechanisms to Adapt.
              </a>
            </td>
          <td>
            Noritaka Tanaka, Hiromichi Ebi
          </td>
          <td>2024-12-27</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/241144354fceff41346aae0d566101a01ccaa823" target='_blank'>
              Advances in cancer genomics and precision oncology.
              </a>
            </td>
          <td>
            Yonjong Heo, Woo-Jin Kim, Yong-Joon Cho, Jae-Won Jung, Nam-Soo Kim, Ik-Young Choi
          </td>
          <td>2025-01-23</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Carcinogenesis often involves significant alterations in the cancer genome, marked by large structural variants (SVs) and copy number variations (CNVs) that are difficult to capture with short-read sequencing. Traditionally, cytogenetic techniques are applied to detect such aberrations, but they are limited in resolution and do not cover features smaller than several hundred kilobases. Optical genome mapping (OGM) and nanopore sequencing [Oxford Nanopore Technologies (ONT)] bridge this resolution gap and offer enhanced performance for cytogenetic applications. Additionally, both methods can capture epigenetic information as they profile native, individual DNA molecules. We compared the effectiveness of the two methods in characterizing the structural, copy number and epigenetic landscape of a clear cell renal cell carcinoma tumor. Both methods provided comparable results for basic karyotyping and CNVs, but differed in their ability to detect SVs of different sizes and types. ONT outperformed OGM in detecting small SVs, while OGM excelled in detecting larger SVs, including translocations. Differences were also observed among various ONT SV callers. Additionally, both methods provided insights into the tumor’s methylome and hydroxymethylome. While ONT was superior in methylation calling, hydroxymethylation reports can be further optimized. Our findings underscore the importance of carefully selecting the most appropriate platform based on specific research questions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2a68c85c3d1dfc77fbfc1f8f772053fa84be16b" target='_blank'>
              Long-read structural and epigenetic profiling of a kidney tumor-matched sample with nanopore sequencing and optical genome mapping
              </a>
            </td>
          <td>
            Sapir Margalit, Zuzana Tulpová, Tahir Detinis Zur, Yael Michaeli, J. Deek, Gil Nifker, Rita Haldar, Y. Gnatek, Dorit Omer, Benjamin Dekel, H. Baris Feldman, Assaf Grunwald, Yuval Ebenstein
          </td>
          <td>2025-01-07</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Drug resistance is the chief cause of treatment failure for therapies targeting chronic and infectious diseases. Whether the emergence of resistance is accelerated by environmental exposure to low levels of therapeutics remains controversial. Here, we report a non-genetic mechanism of stress adaptation that promotes heritable resistance to the widely used antifungal drug fluconazole. In the human fungal pathogen Candida albicans, transient exposure to subtherapeutic fluconazole doses induces a protective response that we term para-resistance. Like conventional resistance mechanisms, para-resistance is heritable. However, it does not arise from genetic mutations and can revert spontaneously. Systematic analyses of para-resistant isolates suggest that its key regulators include the stress-activated MAP kinase Hog1, the histone deacetylase subunit Snt1, the chromatin regulator Rap1, and the Sko1 transcriptional factor. Notably, molecules that disrupt biomolecular condensation and prion propagation – crucial for the inheritance of protein assemblies – block the induction of para-resistance, whereas inhibiting histone deacetylases facilitates its induction. We find that para-resistance is common in clinical isolates and, remarkably, passage through the mammalian gut triggers its acquisition, compromising fluconazole’s therapeutic efficacy. Our work defines a pervasive, prion-like epigenetic mechanism of stress adaptation and highlights potential strategies to mitigate the rapid emergence of drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6a3a4969dd38cce29e6bd3dee1a41bff2be4085" target='_blank'>
              Stress-driven emergence of heritable non-genetic drug resistance
              </a>
            </td>
          <td>
            Jinglin L. Xie, Sifei Yin, Theodore S. Yang, Kiran Chandrasekher, Luke Hanson, Sang Hu Kim, Lucas Esqueda, Catherine A. Hogan, Niaz Banaei, June L. Round, Kyla S. Ost, Judith Berman, Dan Jarosz
          </td>
          <td>2024-12-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Epigenetic regulator genes play critical roles in controlling cell identity and are frequently disrupted in breast cancers, suggesting a key driver role in this disease and its associated phenotypes. However, specific epigenetic drivers (epidrivers) of mammary cell plasticity and their mechanistic contributions to this phenotype are poorly characterized. Methods To identify potential epidrivers of the emergence of mesenchymal breast cancer stem cell-like phenotypes in non-tumorigenic mammary cells, we employed a CRISPR/Cas9 loss-of-function screening strategy targeting epigenetic regulator genes. This approach was followed by an in-depth validation and characterization of epigenomic, transcriptomic, proteomic and phenotypic changes resulting from the disruption of the putative epidriver gene BAP1. Results Our investigation revealed that loss of the histone deubiquitinase BAP1 impacts cellular processes associated with breast cancer cell plasticity such as epithelial-to-mesenchymal transition (EMT) and actin cytoskeleton organization. In addition, we unveiled that BAP1 loss resulted in an overall less permissive chromatin and downregulated gene expression, impacting programs that control cellular glycosylation and leading to decreased glycan abundance and complexity. BAP1 rescue restored the expression of several deregulated genes in a catalytic activity-dependent manner, suggesting that BAP1-mediated cell identity and glycosylation regulation are largely dependent on its histone deubiquitinase activity. Conclusions Overall, our results point to BAP1 disruption as a driver of mammary cell plasticity and reveal a novel role of BAP1 as an epigenetic regulator of cellular glycosylation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/744135b0816e44d41619b9824fc988445e6f4363" target='_blank'>
              Disruption of the epigenetic regulator BAP1 drives chromatin remodeling leading to the emergence of cells with breast cancer stem cell properties and aberrant glycosylation
              </a>
            </td>
          <td>
            Mariana Gomes da Silva Araujo, Aurélie Sallé, Vincent Cahais, Claire Renard, C. Cuenin, Caroline Pires Poubel, Stéphane Keita, Thorsten Mosler, Christine Carreira, Gabrielle Goldman Levy, Lya Parres, Ekaterina Bourova-Flin, Sophie Rousseaux, Saadi Khochbin, A. Ghantous, Siniša Habazin, M. Pučić-Baković, Erika Cosset, G. Lauc, Z. Herceg, Rita Khoueiry
          </td>
          <td>2024-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="This study aimed to generate Car- and Pac-resistant cell lines from the human lung adenocarcinoma H1792 cell line, designated as H1792/Car and H1792/Pac, and perform transcriptome sequencing to identify potential targets. Common differentially expressed genes (Co-DEGs) in both resistant cell lines were identified, followed by hub gene identification. Online validation was conducted through GEPIA and Kaplan–Meier Plotter platforms, with experimental validation performed using real-time quantitative PCR (RT-qPCR). After six months, the H1792/Car and H1792/Pac cell lines exhibited a 10.7-fold and 5.6-fold increase in resistance to Car and Pac, respectively. Flow cytometry analysis demonstrated that both resistant cell lines were resistant to cell cycle arrest and apoptosis induced by Car or Pac. Transcriptomic sequencing identified 123 Co-DEGs, including 72 upregulated and 51 downregulated genes, consistently expressed in both H1792/Car and H1792/Pac cell lines. Among these, 13 hub genes were identified, with colony-stimulating factor 3 (CSF3) uniquely associated with post-progression survival (PPS) in adenocarcinoma patients undergoing chemotherapy. Notably, CSF3 expression was significantly elevated in both H1792/Car and H1792/Pac compared to parental cells. These findings underscore the value of drug-resistant models in uncovering critical biomarkers. CSF3 emerges as a promising guiding marker or potential molecular target for optimizing Car- and Pac-based doublet regimens.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/319b42d1b63fc345fe5d5188434196b326a0a009" target='_blank'>
              Transcriptomic Profiling of Carboplatin- and Paclitaxel-Resistant Lung Adenocarcinoma Cells Reveals CSF3 as a Potential Biomarker for the Carboplatin Plus Paclitaxel Doublet Regimens
              </a>
            </td>
          <td>
            Pritsana Raungrut, Suchanan Tanyapattrapong, Thipphanet Masjon, Saowanee Maungchanburi, P. Thongsuksai
          </td>
          <td>2024-12-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e385b264839697ab3b154e181f427818a7fd2d8b" target='_blank'>
              Biological and genetic characterization of a newly established human primary multidrug-resistant distal cholangiocarcinoma cell line, CBC3T-6
              </a>
            </td>
          <td>
            Mingzhen Bai, Ruoshui Wang, Chongfei Huang, Ruyang Zhong, Ningzu Jiang, Wenkang Fu, Ningning Mi, Long Gao, Yuyao Jin, Haidong Ma, Jie Cao, Haiying Yu, Qiang Jing, Chao Zhang, Ping Yue, Yong Zhang, Yanyan Lin, Hengwei Zhang, Wenbo Meng
          </td>
          <td>2024-11-29</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Recent studies indicate that the development of drug resistance and increased invasiveness in melanoma is largely driven by transcriptional plasticity rather than canonical coding mutations. Understanding the mechanisms behind cell identity shifts in oncogenic transformation and cancer progression is crucial for advancing our understanding of melanoma and other aggressive cancers. While distinct melanoma phenotypic states have been well characterized, the processes and transcriptional controls that enable cells to shift between these states remain largely unknown. In this study, we initially leverage the well-established zebrafish melanoma model as a high-throughput system to dissect and analyze transcriptional control elements that are hijacked by melanoma. We identify key characteristics of these elements, making them translatable to human enhancer identification despite the lack of direct sequence conservation. Building on our identification of a zebrafish sox10 enhancer necessary for melanoma initiation, we extend these findings to human melanoma, identifying two human upstream enhancer elements that are critical for full SOX10 expression. Stable biallelic deletion of these enhancers using CRISPR-Cas9 induces a distinct phenotype shift across multiple human melanoma cell lines from a melanocytic phenotype towards an undifferentiated phenotype and is also characterized by an increase in drug resistance that mirrors clinical data including an upregulation of NTRK1, a tyrosine kinase, and potential therapeutic target. These results provide new insights into the transcriptional regulation of SOX10 in human melanoma and underscore the role of individual enhancer elements and potentially NTRK1 in driving melanoma phenotype plasticity and drug resistance. Our work lays the groundwork for future gene-based and combination kinase-inhibitor therapies targeting SOX10 regulation and NTRK1 as a potential avenue for enhancing the efficacy of current melanoma treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99954db143d577a902929c4be9a53feb2b61c4ba" target='_blank'>
              Specific SOX10 enhancer elements modulate phenotype plasticity and drug resistance in melanoma
              </a>
            </td>
          <td>
            Sophia “Noah” DeGeorgia, Charles K Kaufman
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Antimicrobial combination therapy is widely used to combat Mycobacterium tuberculosis (Mtb), yet resistance rates continue to rise. Mutator strains, with defects in DNA repair genes, are a known source of resistance in other bacterial infections, but their global role in Mtb remains unclear. Here, we provide a comprehensive overview of the contribution of single nucleotide polymorphisms (SNPs) in DNA Repair, Replication, and Recombination (3R) genes to resistance in Mtb. Through large-scale bioinformatics analysis of 53,589 whole-genomes, we identified 18 novel SNPs linked to genotypic drug resistance in 3R genes, covering 12.5% of clinical isolates/strains with available genome sequences. Notably, a number of the detected SNPs were positively selected during Mtb evolution. Experimental mutation frequency tests further indicated functional defects of these sequence variants in key DNA repair pathways. Our findings indicate that the role of mutability in 3R genes plays a significant role in Mtb resistance at the global level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8834cea19a0a1579b6f2abeb80d3493166b930f" target='_blank'>
              Genome wide analyses reveal the role of mutator phenotypes in the emergence of drug resistance in Mycobacterium tuberculosis
              </a>
            </td>
          <td>
            R. Zein-Eddine, A. Le Meur, S. Skouloubris, L. Jelsbak, G. Refrégier, H. Myllykallio
          </td>
          <td>2025-01-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c2f707eacba8ac9d3c84de9541b146c76849cd9" target='_blank'>
              Exosome-delivered NR2F1-AS1 and NR2F1 drive phenotypic transition from dormancy to proliferation in treatment-resistant prostate cancer via stabilizing hormonal receptors
              </a>
            </td>
          <td>
            Wenbin Chen, Yiyou Mao, YiYuan Zhan, Wenfeng Li, Jun Wu, Xiangming Mao, Bin Xu, Fangpeng Shu
          </td>
          <td>2024-12-18</td>
          <td>Journal of Nanobiotechnology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fae4e9e70a4e6d9c16865710a48fa98c28a99bb4" target='_blank'>
              The role of gene copy number variation in antimicrobial resistance in human fungal pathogens
              </a>
            </td>
          <td>
            Adarsh Jay, David F. Jordan, Aleeza Gerstein, Christian R. Landry
          </td>
          <td>2025-01-06</td>
          <td>npj Antimicrobials and Resistance</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4ae77252749b1c986ef7ff79cc410ca86b264c4" target='_blank'>
              Comparison of baseline global gene expression profiles of prostate cancer cell lines LNCaP and DU145
              </a>
            </td>
          <td>
            Khalid Ahmed, Zhannur Omarova, Alisalman Sheikh, G. Abuova, K. Ghias, Syed Hani Abidi
          </td>
          <td>2024-12-31</td>
          <td>BMC Research Notes</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81cf24082e18b8a822679c8c6c602531a443fe70" target='_blank'>
              Nucleotide metabolism-associated drug resistance gene NDUFA4L2 promotes colon cancer progression and 5-FU resistance
              </a>
            </td>
          <td>
            Hongxin He, Shiyao Zheng, Shangkun Jin, Weijie Huang, Enhao Wei, Shen Guan, Chunkang Yang
          </td>
          <td>2025-01-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Abstract In a comprehensive study to decipher the multi-layered response to the chemotherapeutic agent temozolomide (TMZ), we analyzed 427 genomes and determined mutational patterns in a collection of ∼40 isogenic DNA repair-deficient human TK6 lymphoblast cell lines. We first demonstrate that the spontaneous mutational background is very similar to the aging-associated mutational signature SBS40 and mainly caused by polymerase zeta-mediated translesion synthesis (TLS). MSH2-/- mismatch repair (MMR) knockout in conjunction with additional repair deficiencies uncovers cryptic mutational patterns. We next report how distinct mutational signatures are induced by TMZ upon sequential inactivation of DNA repair pathways, mirroring the acquisition of chemotherapy resistance by glioblastomas. The most toxic adduct induced by TMZ, O6-meG, is directly repaired by the O6-methylguanine-DNA methyltransferase (MGMT). In MGMT-/- cells, MMR leads to cell death and limits mutagenesis. MMR deficiency results in TMZ resistance, allowing the accumulation of ∼105 C > T substitutions corresponding to signature SBS11. Under these conditions, N3-methyladenine (3-meA), processed by base excision repair (BER), limits cell survival. Without BER, 3-meA is read through via error-prone TLS, causing T > A substitutions but not affecting survival. Blocking BER after abasic site formation results in large deletions and TMZ hypersensitization. Our findings reveal potential vulnerabilities of TMZ-resistant tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdcf8efd6ed1b3006f0325ffb555aaabb03b08a5" target='_blank'>
              Comprehensive whole-genome sequencing reveals origins of mutational signatures associated with aging, mismatch repair deficiency and temozolomide chemotherapy
              </a>
            </td>
          <td>
            Taejoo Hwang, Lukasz Karol Sitko, Ratih Khoirunnisa, Fernanda Navarro-Aguad, David M Samuel, Hajoong Park, Banyoon Cheon, Luthfiyyah Mutsnaini, Jaewoong Lee, Burçak Otlu, Shunichi Takeda, Semin Lee, Dmitri Ivanov, Anton Gartner
          </td>
          <td>2024-12-05</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Endometrial cancer (EC), a prevalent gynecological malignancy, presents significant challenges due to its genetic complexity and heterogeneity. The genomic landscape of EC is underpinned by genetic alterations, such as mutations in PTEN, PIK3CA, and ARID1A, and chromosomal abnormalities. The identification of molecular subtypes-POLE ultramutated, microsatellite instability (MSI), copy number low, and copy number high-illustrates the diverse genetic profiles within EC and underscores the need for subtype-specific therapeutic strategies. The integration of multi-omics technologies such as single-cell genomics and spatial transcriptomics has revolutionized our understanding and approach to studying EC and offers a holistic perspective that enhances the ability to identify novel biomarkers and therapeutic targets. The translation of these multi-omics findings into personalized medicine and precision oncology is increasingly feasible in clinical practice. Targeted therapies such as PI3K/AKT/mTOR inhibitors have demonstrated the potential for improved treatment efficacy tailored to specific genetic alterations. Despite these advancements, challenges persist in terms of variability in patient responses, the integration of genomic data into clinical workflows, and ethical considerations. This review explores the genomic underpinnings of EC, from genes to clinical practice. It highlights the ongoing need for multidisciplinary research and collaboration to address the complexities of EC and improve diagnosis, treatment, and patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/963ff5c76acf5a14126e1c277fe0385e9dec52b9" target='_blank'>
              From Genes to Clinical Practice: Exploring the Genomic Underpinnings of Endometrial Cancer.
              </a>
            </td>
          <td>
            T. Molefi, Lloyd Mabonga, Rodney Hull, Motshedisi Sebitloane, Zodwa Dlamini
          </td>
          <td>2025-01-20</td>
          <td>Cancers</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Background Osteosarcoma is a malignant tumor originating from mesenchymal bone tissue, characterized by high malignancy and poor prognosis. Despite progress in comprehensive treatment approaches, the five-year survival rate remains largely unchanged, highlighting the need to clarify its underlying mechanisms and discover new therapeutic targets. Methods This study utilized RNA sequencing data from multiple public databases, encompassing osteosarcoma samples and healthy controls, along with single-cell RNA sequencing data. Various methods were utilized, such as differential expression analysis of genes, analysis of metabolic pathways, and weighted gene co-expression network analysis (WGCNA), to pinpoint crucial genes. Using this list of genes, we developed and validated a prognostic model that incorporated risk signatures, and we evaluated the effectiveness of the model through survival analysis, immune cell infiltration examination, and drug sensitivity evaluation. Results We analyzed gene expression and metabolic pathways in nine samples using single-cell sequencing data. Initially, we performed quality control and clustering, identifying 21 statistically significant cell subpopulations. Metabolic analyses of these subpopulations revealed heterogeneous activation of metabolic pathways. Focusing on the osteoblastic cell subpopulation, we further subdivided it into six groups and examined their gene expression and differentiation capabilities. Differential expression and enrichment analyses indicated that tumor tissues were enriched in cytoskeletal and structural pathways. Through WGCNA, we identified core genes negatively correlated with four highly activated metabolic pathways. Using osteosarcoma patient data, we developed a risk signature model that demonstrated robust prognostic predictions across three independent cohorts. Ultimately, we performed a thorough examination of the model, which encompassed clinical and pathological characteristics, enrichment analysis, pathways associated with cancer markers, and scores of immune infiltration, highlighting notable and complex disparities between high-risk and low-risk populations. Conclusion This research clarifies the molecular mechanisms and metabolic features associated with osteosarcoma and how they relate to patient outcomes, offering novel perspectives and approaches for targeted therapy and prognostic assessment in osteosarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3c754f2161a5b2138f06ad7cb757b73f892978d" target='_blank'>
              Exploring the heterogeneity of osteosarcoma cell characteristics and metabolic states and their association with clinical prognosis
              </a>
            </td>
          <td>
            Sen Qin, YaoFeng Hu, RuCui Deng, Zhe Wang
          </td>
          <td>2024-12-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="With the rapid advances in cancer research, the list of variants and genes that drive human diseases is constantly expanding. Moreover, the FDA has approved more cancer therapies that incorporate a broader set of genomic features than simple gene variants such as Tumor Mutation Burden (TMB), microsatellite instability status (MSI), and fusion events in gene families such as the NTRK receptors. These features currently require multiple testing methods (IHC/FISH/etc.). With the cost of NGS testing dropping, it is now possible to envision an NGS assay able to reliably detect these features without the need for additional testing. The EIPM multidisciplinary team has developed EXaCT-2: a whole exome sequencing (WES) assay that gives the coverage of a targeted assay on cancer genes and the breadth to detect copy number events, cancer-related fusions, and viruses, which can facilitate diagnostic and therapeutic decisions for cancer patients. We evaluated EXaCT-2 on 250 matched tumor/normal pairs and compared its performance with orthogonally validated results. We show the assay achieves the expected coverage of critical cancer genes, provides a better characterization of somatic copy number alterations, detects common cancer rearrangements and viruses, and enables the accurate estimation of global molecular metrics, such as tumor mutational burden and microsatellite instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd2c47d3b761e9f5ea55d7bc3672fcf26321970f" target='_blank'>
              EXaCT-2: An augmented and customizable oncology-focused whole exome sequencing platform
              </a>
            </td>
          <td>
            P. H. Waltman, P. Chandra, K. W. Eng, D. C. Wilkes, H. Park, C. Pabon, T. Kane, P. Delpe, E. Fernandez, K. Gorski, B. Bhinder, M. Simi, A. King, N. Greco, J. Manohar, J. M. Tang, P. Zisimopoulos, M. A. Assaad, T. Teneyck, D. Roberts, J. Monge, F. Demichelis, W. Tam, M. M. Ouseph, Alexandros Sigaras, H. Beltran, H. Rennert, N. Lindeman, W. Song, J. Solomon, J. M. Mosquera, R. Kim, J. Catalano, D. Hassane, M. Sigouros, O. Elemento, A. Alonso, A. Sboner
          </td>
          <td>2024-12-08</td>
          <td>None</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Breast cancer is the most prevalent cancer and the leading cause of cancer-related mortality among women. Recent breakthroughs in breast cancer therapeutics have significantly enhanced outcomes for hormone receptor-positive and HER2-negative subtypes. However, the emergence of drug resistance, particularly in triple-negative breast cancer, presents a formidable challenge. The intricate interplay of genetic and metabolic diversity within breast cancer cells is pivotal to its development. By reprogramming metabolic pathways, cancer cells can adapt and thrive, meeting the demands of survival, growth, and invasion. These metabolic shifts also play a key role in the development of resistance to conventional therapies. This review explores the genetic and metabolic complexities of breast cancer, emphasizing the diverse subtypes and their unique profiles. We examine how genetic variations and metabolic alterations contribute to breast cancer development and progression, influencing both treatment efficacy and resistance. By integrating insights into the genetic background and metabolic reprogramming of breast cancer subtypes, this review aims to highlight the genetic variations and metabolic alterations that contribute to the pathogenesis of breast cancer, with a vision of advancing more precise and effective targeted therapies as well as discovering of novel diagnostic and prognostic markers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd8d2c71ef9a50520221d0171aab11b20229faa4" target='_blank'>
              Genetic heterogeneity and metabolic reprogramming in breast cancer
              </a>
            </td>
          <td>
            Yarkın Dolaş, Ayşe Buruş, B. Çeltikçi
          </td>
          <td>2024-12-29</td>
          <td>Acta Medica</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62b1219475c2c72e7d5e477e93702ece4b16c3cc" target='_blank'>
              A Genome-Wide CRISPR/Cas9 Screen Identifies Regulatory Genes for Stem Cell Aging.
              </a>
            </td>
          <td>
            Peng Su, Yi-liang Miao
          </td>
          <td>2024-12-21</td>
          <td>Methods in molecular biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cell-cell fusion has been implicated in various physiological and pathological processes, including cancer progression. This study investigated the role of cell-cell fusion in non-small cell lung cancer (NSCLC), focusing on its contribution to chemoresistance and tumor evolution. By co-culturing drug-sensitive and drug-resistant NSCLC cell lines, we observed spontaneous cell-cell fusion events, particularly under gefitinib selection. These fused cells exhibited enhanced fitness and a higher degree of chemoresistance compared to parental lines across a panel of 12 chemotherapeutic agents. Further analysis, including fluorescence imaging and cell cycle analysis, confirmed nuclear fusion and increased DNA content in the fused cells. Bulk RNA sequencing revealed genomic heterogeneity in fused cells, including enrichment of gene sets associated with cell cycle progression and epithelial-mesenchymal transition, hallmarks of cancer malignancy. These findings demonstrate that cell-cell fusion contributes significantly to therapeutic resistance and the promotion of aggressive phenotypes in NSCLC, highlighting its potential as a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f78ae5e6d6b40de2fc2ee9f6021e6a3e7d5997a6" target='_blank'>
              Cell-Cell Fusion in NSCLC Confers a Fitness Benefit Under Drug Selection
              </a>
            </td>
          <td>
            Paulameena V. Shultes, D. Tadele, A. Durmaz, D. Weaver, R. Barker-Clarke, Mina N. Dinh, Siqing Liu, Endalkachew A. Alemu, Simon Rayner, Jacob G. Scott
          </td>
          <td>2024-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="While the key aspects of genetic evolution and their clinical implications in clear cell renal-cell carcinoma (ccRCC) are well-documented, how genetic features co-evolve with the phenotype and tumor microenvironment (TME) remains elusive. Here, through joint genomic-transcriptomic analysis of 243 samples from 79 patients recruited to the TRACERx Renal study, we identify pervasive non-genetic intratumor heterogeneity, with over 40% not attributable to genetic alterations. By integrating tumor transcriptomes and phylogenetic structures, we observe convergent evolution to specific phenotypic traits, including cell proliferation, metabolic reprogramming and overexpression of putative cGAS-STING repressors amid high aneuploidy. We also uncover a co-evolution between the tumor and the T cell repertoire, as well as a longitudinal shift in the TME from an anti-tumor to an immunosuppressive state, linked to the acquisition of recurrently late ccRCC drivers 9p loss and SETD2 mutations. Our study reveals clinically-relevant and hitherto underappreciated non-genetic evolution patterns in ccRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c84f2fcb6ee92e2f103bced79a769e625785b29b" target='_blank'>
              Tracking non-genetic evolution from primary to metastatic ccRCC: TRACERx Renal.
              </a>
            </td>
          <td>
            Ángel Fernández-Sanromán, A. Fendler, B. J. Tan, A. Cattin, C. Spencer, Rachael Thompson, L. Au, Irene Lobon, H. Pallikonda, Alice Martin, F. Byrne, Antonia Franz, Anna Mikolajczak, Haseeb Rahman, Z. Tippu, S. T. Shepherd, Hugang Feng, Daqi Deng, Andrew Rowan, Lisa M. Pickering, A. J. Furness, Kate Young, David Nicol, S. Rudman, Tim O’Brien, K. Edmonds, Ashish Chandra, S. Hazell, Kevin Litchfield, G. Kassiotis, J. Larkin, S. Turajlic
          </td>
          <td>2025-01-08</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d53231039f1a5af21021a2a1249dd46836990843" target='_blank'>
              High-throughput method characterizes hundreds of previously unknown antibiotic resistance mutations
              </a>
            </td>
          <td>
            Matthew J Jago, Jake K. Soley, Stepan Denisov, Calum J. Walsh, Danna R. Gifford, B. Howden, Mato Lagator
          </td>
          <td>2025-01-17</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Developing resistance to androgen receptor (AR) signaling inhibitors is a significant challenge in the treatment of castration-resistant prostate cancer. Prolonged use of inhibitors like enzalutamide can cause prostate cancer cells to undergo lineage reprogramming, transitioning to neuroendocrine subtypes that no longer rely on AR signaling. These neuroendocrine subtypes are among the most aggressive forms of prostate cancer. During this process of lineage plasticity, cancer cells experience extensive transcriptional rewiring and acquire stem-like properties characterized by increased stemness. Research has shown that prostate cancer cells gain these stem-like traits through the expression of stem cell-associated proteins such as NANOG, particularly under stable and accumulating conditions. The post-translational modification of NANOG at specific sites is critical for maintaining its stability, which in turn enhances the tumorigenic potential of the cells. This review discusses the mechanisms by which NANOG phosphorylation promotes stemness and lineage plasticity in prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d23842d78f2e67e6ef1863e8dd082e1ac56f105a" target='_blank'>
              Molecular mechanisms driving lineage plasticity in prostate cancer: NANOG and beyond
              </a>
            </td>
          <td>
            Hamed Maasoumyhaghighi, Mansoureh Nouri, Jinghui Liu, Xiaoqi Liu
          </td>
          <td>2024-12-27</td>
          <td>Cancer Heterogeneity and Plasticity</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Medulloblastoma (MB) is the most prevalent malignant brain tumor in children, exhibiting clinical and genomic heterogeneity. Of the four major subgroups, Group 3 tumors (MYC-MB), display high levels of MYC and metastasis rates. Despite treatment with surgery, radiation, and chemotherapy, patients with Group 3 MB are more likely to develop aggressive recurrent tumors with poor survival. To examine resistance mechanisms, single nuclei multiome analysis of matched primary and recurrent tumors was performed in this study. A persistent progenitor population supporting resistance to therapy was identified. Additionally, distinct chromatin landscapes are linked to altered transcription and correspond to metabolic reprogramming. In vivo modeling of radiation resistance resolves similar chromatin-based metabolic reprogramming focused on wild-type isocitrate dehydrogenase (IDH1) activity. IDH1 inhibition reverses resistance-mediated chromatin changes and enables radiation re-sensitization. Ultimately, these findings demonstrate the efficacy of single-cell multiome analysis in elucidating resistance mechanisms and identifying novel target pathways for MYC-driven medulloblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/beeb800863a129b3f026b22eed6f7669390988ee" target='_blank'>
              Single-cell multi-omics analysis identifies metabolism-linked epigenetic reprogramming as a driver of therapy-resistant medulloblastoma
              </a>
            </td>
          <td>
            Rajeev Vibhakar, Bethany L. Veo, Dong Wang, J. DeSisto, Angela M Pierce
          </td>
          <td>2024-12-13</td>
          <td>Research Square</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8a65325c104ad2755155bebd8b8e82939a9a6fc" target='_blank'>
              Transposon mediated functional genomic screening for BRAF inhibitor resistance reveals convergent Hippo and MAPK pathway activation events.
              </a>
            </td>
          <td>
            Li Chen, Iulian Pruteanu-Malinici, A. Dastur, Xunqin Yin, Dennie Frederick, R. I. Sadreyev, Cyril H. Benes
          </td>
          <td>2025-01-24</td>
          <td>Scientific reports</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31f970e4e2fc5fa77ad8483f91874195b589fe0b" target='_blank'>
              Single-cell RNA-seq reveals diverse molecular signatures associated with Methotrexate resistance in primary central nervous system lymphoma cells.
              </a>
            </td>
          <td>
            Rosuke Osako, A. Hayano, Atsushi Kawaguchi, R. Yamanaka
          </td>
          <td>2024-12-05</td>
          <td>Journal of neuro-oncology</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="The main objective of this study was to explore and identify new genetic targets in small-cell lung cancer (SCLC) through transcriptomics analysis and Mendelian randomization (MR) analysis, which will help in the subsequent development of new therapeutic interventions.In this study, we extracted the SCLC dataset from the Gene Expression Omnibus (GEO) database, processed the data, and screened out differentially expressed genes (DEGs) using R software. Based on expression quantitative trait loci data and the genome-wide association study data of SCLC, MR analysis was used to screen the genes closely related to SCLC disease, which intersect with DEGs to obtain co-expressed genes (CEGs), and the biological functions and pathways of CEGs were further explored by enrichment analysis. In addition, the CIBERSORT algorithm was applied to assess the level of immune cell infiltration in SCLC and to analyze the correlation between CEGs and immune cells. Meanwhile, we performed a survival analysis on these five CEGs using an independent cohort of SCLC patients. Finally, the results for the target genes were validated.In this study, 857 DEGs were identified, including 443 up-regulated and 414 down-regulated genes, and 5 CEGs (PSAT1, PSRC1, COLEC12, PLLP, HP) that were significantly associated with SCLC were identified through further intersecting. The results of enrichment analyses indicated that CEGs play important roles in several key functions and pathways. Immune-cell-related analysis revealed the unique distribution of immune cell infiltration in SCLC and the mechanism of immune cell regulation by CEGs. Survival analysis results indicated that PSRC1 was significantly correlated with the overall survival of SCLC, and the survival rate of the high-expression group was markedly lower than that of the low-expression group. Finally, the consistency of the results between the validation group analyses and MR analysis confirmed that the results of this study is reliable.The CEGs and their associated functions and pathways screened in this study may be potential targets of therapeutic intervention in SCLC by targeting specific molecular pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77842bf1a95d4aa6340eee7f38794ec66937f841" target='_blank'>
              Exploring potential therapeutic targets for small cell lung cancer based on transcriptomics combined with Mendelian randomization analysis
              </a>
            </td>
          <td>
            Zhicheng Liao, Pengcheng Jia, Yifan Li, Zhihui Zheng, Jizhou Zhang
          </td>
          <td>2025-01-13</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Immune escape is a critical hallmark of cancer progression and underlies resistance to multiple immunotherapies. However, it remains unclear when the genetic events associated with immune escape occur during cancer development. Here, we integrate functional genomics studies of immunomodulatory genes with a tumor evolution reconstruction approach to infer the evolution of immune escape across 38 cancer types from the Pan-Cancer Analysis of Whole Genomes dataset. Different cancers favor mutations in different immunomodulatory pathways. For example, the antigen presentation machinery is highly mutated in colorectal adenocarcinoma, lung squamous cell carcinoma, and chromophobe renal cell carcinoma, and the protein methylation pathway is highly mutated in bladder transitional cell carcinoma and lung adenocarcinoma. We also observe different timing patterns in multiple immunomodulatory pathways. For instance, mutations impacting genes involved in cellular amino acid metabolism were more likely to happen late in pancreatic adenocarcinoma. Mutations in the glucocorticoid receptor regulatory network pathway tended to occur early, while mutations in the TNF pathways were more likely to occur late in B-cell non-Hodgkin lymphoma. Mutations in the NOD1/2 signaling pathway and DNA binding transcription factor activity tended to happen late in breast adenocarcinoma and ovarian adenocarcinoma. Together, these results delineate the evolutionary trajectories of immune escape in different cancer types and highlight opportunities for improved immunotherapy of cancer. Significance Despite its critical role in cancer progression, the evolution of immune escape is poorly understood. We integrate functional genomics and tumor evolution reconstruction and infer immune escape trajectories across cancer types. Our results have important implications for developing biomarkers for immunoprevention and treatment strategies for immune escape of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a491b543872787c05b0f657359563c86a9093f6" target='_blank'>
              Evolutionary trajectories of immune escape across cancers
              </a>
            </td>
          <td>
            Wenjie Chen, Toby M. Baker, Zhihui Zhang, Huw A. Ogilvie, P. Van Loo, S. Gu
          </td>
          <td>2025-01-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Resistance to drugs is one of the major issues affecting the response to pharmacological treatments for tumors. Different mechanisms have been proposed to explain the development of cancer drug resistance (CDR), and several approaches to overcome it have been suggested. However, the biological basis of CDR remains unclear. Here, we investigated whether mitochondrial damage and consequent mitochondrial dysfunction are major causes of drug resistance in different tumors. To this end, we used cell lines from three tumors: hepatocellular carcinoma, breast cancer, and colon cancer. We then applied a protocol that recapitulates chemotherapy regimens in patients, rendering each cell line resistant to the drug commonly used in their respective treatments. The combination of cellular respiration analysis, gene expression analysis of cytochrome c oxidase isoforms, and mass spectrometry assessment of cardiolipin reveals that mitochondrial dysfunction is the underlying cause of the resistant phenotype. Importantly, we disclosed for the first time the rapid inhibition of oxidative phosphorylation (OXPHOS) by L-lactate, the major product of fermentation. Finally, we demonstrated that inhibition of lactic acid fermentation and activation of OXPHOS can increase drug sensitivity in all tested drug-resistant cancer cells. Taken together, our results suggest that inhibiting fermentation and enhancing mitochondrial function in cancer cells may be a concrete option to control the worrisome phenomenon of CDR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcdb4a2eced91fb616948e3a8a82b2b795dd7c9f" target='_blank'>
              Dysfunctional mitochondrial bioenergetics sustains drug resistance in cancer cells.
              </a>
            </td>
          <td>
            D. Gnocchi, D. Nikolic, S. Russo, M. L. Matrella, R. R. Paparella, Sujeet Kumar, S. Karki, Carlo Sabbà, Tiziana Cocco, Simona Lobasso, A. Mazzocca
          </td>
          <td>2025-01-24</td>
          <td>American journal of physiology. Cell physiology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Synthetic lethality is defined as a type of genetic interaction where the combination of two genetic events results in cell death, whereas each of them separately does not. Synthetic lethality can be a useful tool in personalised oncology. MLH1 is a cancer-related gene that has a central role in DNA mismatch-repair and TP53 is the most frequently mutated gene in cancer. To identify genetic events that can lead to tumour death once either MLH1 or TP53 is mutated, a genome-wide genetic screening was performed. Thus, mutations in all protein-coding genes were introduced into haploid human embryonic stem cells (hESCs) with and without loss-of-function mutations in the MLH1 or TP53 genes. These experiments uncovered a list of putative hits with EXO1, NR5A2, and PLK2 genes for MLH1, and MYH10 gene for TP53 emerging as the most promising candidates. Synthetic lethal interactions of these genes were validated genetically or chemically using small molecules that inhibit these genes. The specific effects of SR1848, which inhibits NR5A2, ON1231320 or BI2536, which inhibits PLK2, and blebbistatin, which inhibits MYH10, were further validated in cancer cell lines. Finally, animal studies with CCL xenografts showed the selective effect of the small molecule BI2536 on MLH1-null tumours and of blebbistatin on TP53-mutated tumours. Thus, demonstrating their potential for personalised medicine, and the robustness of genetic screening in haploid hESCs in the context of cancer therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf8258266a9c323d57d9c42f16e5c72059537055" target='_blank'>
              Genome-Wide Screening in Haploid Stem Cells Reveals Synthetic Lethality Targeting MLH1 and TP53 Deficient Tumours.
              </a>
            </td>
          <td>
            R. Cashman, Guy Haim-Abadi, Elyad Lezmi, Hagit Philip, J. Nissenbaum, Ruth Viner-Breuer, Chen Kozulin, T. Golan‐lev, Aseel Gadban, Shiri Spinner-Potesky, O. Yanuka, O. Kopper, N. Benvenisty
          </td>
          <td>2025-01-15</td>
          <td>Cell proliferation</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Cell-cell fusion is a tightly controlled process in the human body known to be involved in fertilization, placental development, muscle growth, bone remodeling, and viral response. Fusion between cancer cells results first in a whole-genome doubled state, which may be followed by the generation of aneuploidies; these genomic alterations are known drivers of tumor evolution. The role of cell-cell fusion in cancer progression and treatment response has been understudied due to limited experimental systems for tracking and analyzing individual fusion events. To meet this need, we developed a molecular toolkit to map the origins and outcomes of individual cell fusion events within a tumor cell population. This platform, ClonMapper Duo (‘CMDuo’), identifies cells that have undergone cell-cell fusion through a combination of reporter expression and engineered fluorescence-associated index sequences paired to random barcode sets. scRNA-seq of the indexed barcodes enables the mapping of each set of parental cells and fusion progeny throughout the cell population. In triple negative breast cancer cells CMDuo uncovered subclonal transcriptomic hybridization and unveiled distinct cell-states which arise in direct consequence of homotypic cell-cell fusion. CMDuo is a platform that enables mapping of cell-cell fusion events in high-throughput single cell data and enables the study of cell fusion in disease progression and therapeutic response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05fe68cd8b32acb61197d1f1f0a4cc878efa91c6" target='_blank'>
              Mapping cell-cell fusion at single-cell resolution
              </a>
            </td>
          <td>
            Andrea L. Gardner, Lan Zheng, Kennedy Howland, Andrew Saunders, Andrea Ramirez, Patrik Parker, Chisom Iloegbunam, Daylin Morgan, T. Jost, Amy Brock
          </td>
          <td>2024-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Breast cancer (BC) is considered as the most frequent cancer among women and a significant contributor to mortality. The CRISPR/Cas9 gene-editing tool has promising applications for BC drug resistance. It is a unique and creative approach that has lately attracted attention and can be used to fix gene alterations related to multidrug resistance. Recent research has effectively investigated and targeted particular genes linked to BC treatment resistance using CRISPR/Cas9 gene editing, including those linked to hormone receptor signaling, drug efflux transporters, and DNA repair pathways. The CRISPR/Cas9 technology's selective disruption or mutation of these genes provides valuable information about their role in resistance and paves the path for cutting-edge treatment options. CRISPR/Cas9 gene editing can overcome BC treatment resistance by identifying crucial genetic variables and revealing new therapeutic targets Despite the advantages, there are limitations in the study on CRISPR/Cas9-based gene editing for BC treatment resistance, for example, off-target effects and the improvement of delivery techniques are still major issues. Successful clinical translation depends on methods to improve the specificity and effectiveness of CRISPR/Cas9 editing and to solve these constraints. This review aims to explore the possibility of CRISPR/Cas9 gene editing as an innovative method of combating BC medication resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4162c264e1c96e6bc11bcded7869da58b56f7ba" target='_blank'>
              Molecular Mechanisms of Breast Cancer Drug Resistance and CRISPR/Cas9 Strategies to Overcome
              </a>
            </td>
          <td>
            B. Hussen, Bnar Saleh Ismael, Saman S. Abdulla, Noor Haval Jamal, S. A. Mustafa, Zana Baqi Najmalddin, Mohammed Fatih Rasul
          </td>
          <td>2024-12-30</td>
          <td>BioMed Target Journal</td>
          <td>1</td>
          <td>27</td>
        </tr>

        <tr id="Metabolic reprogramming, particularly lipid metabolism, is a hallmark of cancer progression and a critical vulnerability in prostate cancer (PCa). Two pivotal studies, one by Lu et al. and the other by Dairo et al., have illuminated the roles of HOXB13, a transcriptional regulator, and fatty acid synthase (FASN), a key enzyme in lipid biosynthesis, in the metabolic dysregulation of PCa. The Lu et al. study highlights HOXB13’s role in androgen receptor (AR)-independent PCa, where it regulates lipid metabolism via epigenetic mechanisms involving histone deacetylase HDAC3. A G84E mutation in HOXB13 disrupts this regulation, leading to increased lipid synthesis and a pro-metastatic phenotype. The Dairo et al. study demonstrates that FASN hypomethylation, coupled with increased expression, drives lipid biosynthesis critical for tumor growth. Both studies establish a link between HOXB13 mutations and FASN dysregulation, underscoring their interplay in PCa biology. Therapeutically, pharmacological inhibition of FASN mitigates the aggressive features of HOXB13-deficient or mutant PCa, highlighting lipid metabolism as a promising target. Despite their strengths, including robust methodologies, limitations include reliance on preclinical models and the need for broader patient diversity. These studies collectively emphasize the potential of metabolic and epigenetic interventions for precision medicine in PCa, paving the way for novel therapies targeting lipid metabolism in patients with specific genetic and epigenetic profiles.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc58c2230658f37cc79a630362aeb84c369fdaa5" target='_blank'>
              Metabolic Reprogramming in Prostate Cancer: The Roles of HOXB13 and FASN in Tumor Progression and Therapy
              </a>
            </td>
          <td>
            Faisal Saeed, Lara Harik
          </td>
          <td>2025-01-14</td>
          <td>Serican Journal of Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Extensive intratumor heterogeneity in glioblastoma (GBM) impedes successful treatment and complicates drug discovery as it is not obvious which cells a tumor is most dependent on. Here, we posit that single-cell-resolution transcriptomic data can be integrated with loss-of-function screens to identify the most critical cells to target within a tumor. We parsed CRISPR screen data from the Dependency Map (DepMap) Consortium and identified a GBM Dependency Signature (GDS) − 168 genes that are essential for GBM cell viability in vitro. Through similarity scoring of GDS transcriptomic profiles in single-cell RNA-sequencing (scRNA-seq) data and iterative hierarchical clustering, we identify and report here 3 single-cell vulnerability states (VS) characterized in 49 GBM tumors using both scRNA-seq and spatial transcriptomic data. These VS reflect single-cell gene dependencies and differ significantly in enrichment profiles and spatial distributions. Importantly, the proportion of VS in each GBM tumor is variable, suggesting a means of stratifying patients in clinical trials. Collectively, we have developed a novel computational pipeline to identify unique vulnerability states in GBM and other cancers, which can be used to identify existing or novel drugs for incurable diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4aef7e027ff56302092f68c8a779be84bebb597e" target='_blank'>
              Identification of disease-specific vulnerability states at the single-cell level
              </a>
            </td>
          <td>
            Matthew D’Antuono, Madison Sharp, Rishika Chowdary, Michael E. Ivan, Ricardo J Komotar, Robert K. Suter, N. Ayad
          </td>
          <td>2024-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Despite applying conventional predictive methodologies to obtain genomic insights, predicting drug sensitivity for healthcare organizations in the USA remains a daunting challenge. Cancer is a highly dynamic, adaptive disease tumor cells have repeatedly shown the capability to evolve mechanisms whereby therapeutic interventions can be evaded. Besides, one genomic alteration seldom predicts drug sensitivity. This research project aimed to address the challenges of predicting drug sensitivity by leveraging the GDSC dataset, an extensive resource connecting genomic profiles of cancer cell lines with their sensitivity to a wide range of anti-cancer drugs. This research's key focus was identifying robust genomic markers, including any specific mutations, gene expression patterns, or epigenetic modifications associated with drug sensitivity or resistance. Advanced machine learning and statistical methods were utilized by the predictive models to analyze complex relations that may exist between different genomic alterations and their drug sensitivity. The dataset used for this research project was derived from the Kaggle website. This dataset was compiled by the research project Genomics of Drug Sensitivity in Cancer collaboration between the Sanger Institute in the United Kingdom and the Massachusetts General Hospital Cancer Center in the United States. In their investigation, there was a massive screening of human cancer cell lines with a wide range of anti-cancer drugs. Data collection was performed by large-scale screening of diverse anti-cancer drugs against human cancer cell lines of various types. Cell viability was measured using the Cell-Titer-Glo assay following 72 hours of drug treatment. Several machine learning models were deployed, namely, Random Forest, Linear Regression, and XG-Boost, which exhibited specific strengths. Specific performance metrics used included MSE, RMSE, MAE, and R². As the statistics indicate, among the three models, Random Forest stands out and performs the best on this dataset across all metrics. A smaller value of MAE, MSE, and RMSE signifies that it provided the best forecast for the target variable. It also gave the highest R-squared value. Application of drug sensitivity prediction analysis in cancer can provide an overview of the mechanisms that underlie both tumor response and resistance by investigating the model predictions. The proposed predictive models have the potential to make significant impacts on clinical decision-making in cancer therapy. Predictive models derive informed decisions regarding a patient's risk of recurrence of disease, their response to certain therapies, and their prognosis based on complex clinical and genomic data. 
Keywords: Genomic Predictors; Drug Sensitivity; Genomic Markers; Personalized medicine; Machine Learning; Random Forest Algorithm.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a3597d32cedd5f2f5690112a3136db0246ef249" target='_blank'>
              Genomic predictors of drug sensitivity in cancer: Integrating genomic data for personalized medicine in the USA
              </a>
            </td>
          <td>
            Laxmi Pant, Abdullah Al Mukaddim, Md khalilor Rahman, Abdullah Al Sayeed, Md Sazzad Hossain, MD Tushar Khan, Adib Ahmed
          </td>
          <td>2024-12-09</td>
          <td>Computer Science &amp; IT Research Journal</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/feaea0d43cfc2d7247e33264b3b0a5a3472dbf9e" target='_blank'>
              Mutational signature analysis predicts bacterial hypermutation and multidrug resistance
              </a>
            </td>
          <td>
            Kalen M. Hall, Leonard G. Williams, Richard D Smith, Erin A Kuang, Robert K. Ernst, Christine M Bojanowski, William C Wimley, L. Morici, Z. Pursell
          </td>
          <td>2025-01-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Background Cellular plasticity, involving loss of lineage determination and emergence of hybrid cell states, plays a pivotal role in non-small cell lung cancer (NSCLC) disease progression and therapy resistance. However, the full spectrum of atypical states generated in human NSCLC and the pathways that regulate them are yet to be fully elucidated. Here we examine the role of developmental programmes, alveogenesis and branching morphogenesis (BM), in regulating phenotypic diversity in NSCLC. Methods Transcriptomic analysis of epithelial cells isolated from murine lungs at different stages of organogenesis were used to derive gene signatures for developmental programmes. Bulk tissue transcriptomic datasets from human NSCLC and non-neoplastic control samples were used to identify whether developmental programmes were associated with molecular, morphological, and clinical parameters. Single-cell RNA-sequencing was used to identify malignant cell states in human NSCLC (n = 16,621 epithelial cells from 72 samples) and protein level validation of these states was carried out using multiplexed immunohistochemistry (n = 40). Results Mutually antagonistic regulation of alveogenesis and BM was found to account for a significant proportion of transcriptomic variance in human NSCLC bulk tissue datasets. BM activation was associated with poor overall survival rates in five independent lung adenocarcinoma (LUAD) cohorts (p=2.04e-13); and was significantly prognostic for resistance to tyrosine kinase inhibitors (TKIs; p=0.003) and immune checkpoint blockade (ICBs; p=0.014), in pre-treatment biopsies. Single-cell RNA-sequencing analysis revealed that malignant LUAD cells with loss of alveolar lineage fidelity predominantly acquired inflamed or basal-like cellular states, which were variably persistent in samples from TKI and ICB recurrence. Conclusions Our results show LUAD tumours undergo reversion from an alveogenic to branching morphogenic phenotype during disease progression, generating inflamed or basal-like cell states that are variably persistent following TKI or ICB treatments. These findings identify prognostic biomarkers for therapy response and underscore the role of different cell states in resistance to multiple treatment modalities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/390cd095a97603dc458c49ab8aaa4b76e0155529" target='_blank'>
              Developmental programmes drive cellular plasticity, disease progression and therapy resistance in lung adenocarcinoma
              </a>
            </td>
          <td>
            Kamila J Bienkowska, Stephany Gallardo Y, Nur S Zainal, Matthew Ellis, Maria-Antoinette Lopez, Judith Austine, Sai Pittla, S. Chee, A. Alzetani, Emily C Shaw, Christian H Ottensmeier, Gareth J Thomas, Christopher Hanley
          </td>
          <td>2024-12-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Cancer’s epigenetic landscape, a labyrinthine tapestry of molecular modifications, has long captivated researchers with its profound influence on gene expression and cellular fate. This review discusses the intricate mechanisms underlying cancer epigenetics, unraveling the complex interplay between DNA methylation, histone modifications, chromatin remodeling, and non-coding RNAs. We navigate through the tumultuous seas of epigenetic dysregulation, exploring how these processes conspire to silence tumor suppressors and unleash oncogenic potential. The narrative pivots to cutting-edge technologies, revolutionizing our ability to decode the epigenome. From the granular insights of single-cell epigenomics to the holistic view offered by multi-omics approaches, we examine how these tools are reshaping our understanding of tumor heterogeneity and evolution. The review also highlights emerging techniques, such as spatial epigenomics and long-read sequencing, which promise to unveil the hidden dimensions of epigenetic regulation. Finally, we probed the transformative potential of CRISPR-based epigenome editing and computational analysis to transmute raw data into biological insights. This study seeks to synthesize a comprehensive yet nuanced understanding of the contemporary landscape and future directions of cancer epigenetic research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf231d1889782a99759e5b17e3a47e9d23349015" target='_blank'>
              Mechanisms and technologies in cancer epigenetics
              </a>
            </td>
          <td>
            Z. Sherif, O. Ogunwobi, H. Ressom
          </td>
          <td>2025-01-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="All forms of genetic variation originate from new mutations, making it crucial to understand their rates and mechanisms. Here, we use long-read PacBio sequencing to investigate de novo mutations that accumulated in 12 inbred mouse lines derived from three commonly used inbred strains (C3H, C57BL/6, and FVB) maintained for 8-15 generations in a mutation accumulation (MA) experiment. We built chromosome-level genome assemblies based on the MA line founders' genomes, and then employed a combination of read and assembly-based methods to call the complete spectrum of new mutations. On average, there are ~45 mutations per haploid genome per generation, about half of which (54%) are insertions and deletions shorter than 50 bp (indels). The remainder are single nucleotide mutations (SNMs, 44%) and large structural mutations (SMs, 2%). We found that the degree of DNA repetitiveness is positively correlated with SNM and indel rates, and that a substantial fraction of SMs can be explained by homology-dependent mechanisms associated with repeat sequences. Most (90%) indels can be attributed to microsatellite contractions and expansions, and there is a marked bias towards 4 bp indels. Among the different types of SMs, tandem repeat mutations have the highest mutation rate, followed by insertions of transposable elements (TEs). We uncover a rich landscape of active TEs, and notable differences in their spectrum among MA lines and strains, and a high rate of gene retroposition. Our study offers novel insights into mammalian genome evolution, and highlights the importance of repetitive elements in shaping genomic diversity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4966070a9e9e2ca28b9c82696887c588bee4c86" target='_blank'>
              The rate and spectrum of new mutations in mice inferred by long-read sequencing.
              </a>
            </td>
          <td>
            Eugenio López-Cortegano, Jobran Chebib, Anika Jonas, Anastasia Vock, Sven Künzel, P. Keightley, D. Tautz
          </td>
          <td>2024-12-02</td>
          <td>Genome research</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="
 The concept of synthetic lethality has long held promise for the next generation of genetically targeted cancer therapies to follow the success of PARP inhibitors in the treatment of BRCA-mutant cancers. The classical definition of synthetic lethality arose in Drosophila with the observation that loss of either of two genes independently had little effect on cell viability but loss of both led to cell death. This definition has now been expanded to capture pairs of genes in which genetic alteration of one gene and pharmacological inhibition of the other leads to death of cancer cells while sparing the normal cells which lack the genetic alteration, leading to a broad therapeutic index. Functional genomics screening (using unbiased RNAi or CRISPR-based technology) has enabled the systematic discovery of novel synthetic lethal targets for drug discovery. The selective dependence of MTAP-deleted cells on PRMT5 is one of the strongest genetic interactions observed in early RNAi screens. Approximately 10-15% of all human cancer is MTAP-deleted, providing a large and diverse patient population. MTA-cooperative PRMT5 inhibitors, which bind preferentially in the presence of MTA, have been developed by scientists at Tango Therapeutics and elsewhere to leverage the synthetic lethal relationship between PRMT5 and MTAP loss. MTA-cooperative PRMT5 inhibitors inhibit PRMT5 in MTAP-deleted cancer cells and spare MTAP-expressing normal cells leading to clinically well-tolerated and efficacious therapies. MTA-cooperative PRMT5 inhibitors have broad therapeutic potential both as single agent and in combination with other therapies.
 Citation Format: Kimberly Briggs.Leveraging synthetic lethality for the development of novel cancer therapies. [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Optimizing Therapeutic Efficacy and Tolerability through Cancer Chemistry; 2024 Dec 9-11; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(12_Suppl):Abstract nr IA002.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/539ee15239f2d3141b9b8d5ed5ea99f9aadfdf6f" target='_blank'>
              Abstract IA002: Leveraging synthetic lethality for the development of novel cancer therapies
              </a>
            </td>
          <td>
            Kimberly J. Briggs
          </td>
          <td>2024-12-09</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Many agents that show promise in preclinical cancer models lack efficacy in patients due to patient heterogeneity that is not captured in traditional assays. To address this problem, we have developed GENEVA, a platform that measures the molecular and phenotypic consequences of drug perturbations within diverse populations of cancer cells at single-cell resolution, both in vitro and in vivo. Here, we apply GENEVA to study the KRAS G12C inhibitors, recapitulating known properties of these drugs and uncovering a previously unknown role for mitochondrial activation in cell death induced by KRAS inhibition. We demonstrate that this finding can be leveraged for the development of combination therapies with greater efficacy. Finally, we show that the application of GENEVA with in vivo mouse models revealed epithelial to mesenchymal transition (EMT) as a key mechanism for resistance to KRAS G12C inhibition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/198b25a5af544d70c25d15829b150e55c9e084e8" target='_blank'>
              Multiplexed mosaic tumor models reveal natural phenotypic variations in drug response within and between populations
              </a>
            </td>
          <td>
            Johnny X. Yu, Jung Min Suh, Katerina D. Popova, Kristle Garcia, Tanvi Joshi, Bruce Culbertson, Jessica B. Spinelli, Vishvak Subramanyam, Kevin Lou, K. Shokat, Jonathan Weissman, Hani Goodarzi
          </td>
          <td>2024-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>131</td>
        </tr>

        <tr id="Organisms constantly face environmental stressors that threaten their cellular and genomic integrity. In their response, pathogen-associated molecular patterns (PAMPs) and/or damage-associated molecular patterns (DAMPs) are detected by pattern recognition receptors (PRRs) and trigger the innate immune response. In this study we tested the hypothesis that DAMPs contribute to radiation-induced cellular plasticity in Glioblastoma (GBM). GBM is known to be organized hierarchically with a small number of glioma-initiating cells (GICs) driving treatment resistance and recurrences. Using patient-derived GBM specimens, we employed sphere forming capacity assays and in vitro extreme limiting dilution assays to examine how innate immune receptor signaling impacts the maintenance and self-renewal of GICs. By leveraging an imaging system for putative GICs we determined de novo induction of GICs from non-stem glioma cells. We find that GIC maintenance after irradiation is mediated by cGAS-independent STING signaling, possibly involving signaling through TLR4 and TLR9. Induction of radiation-induced plasticity involves TLR3 signaling, with potential roles for other receptors and processes modulated by MyD88. These findings suggest that targeting innate immune signaling could prevent radiation-induced cellular plasticity for potential therapeutic benefit.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5ddc1e74601131515f0f5cca20f0a178a3981ff" target='_blank'>
              Targeting Innate Immune Signaling in Glioma-Initiating Cells Impairs Self-Renewal and Radiation-Induced Cellular Plasticity
              </a>
            </td>
          <td>
            Anjelica M. Cardenas, Linda Azizi, A. Ioannidis, Kruttika Bhat, Ling He, Mahya Mohammadi, Frank Pajonk
          </td>
          <td>2024-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Cancer cell lines are the most common in-vitro models for the evaluation of anti-cancer drug sensitivities. Past studies have been conducted to decipher and characterize the pharmacogenomic feature of cell lines based on other omics data, such as genomic mutation data and whole-genome RNA sequencing (RNA-seq) profiles. In particular, proteomic data is also an essential component for the characterization of tumours. However, different from RNA-seq datasets rich in numerous transcriptome profiles of cancer cell lines and cell viability assay of drug responses, the pharmacogenomic protein quantifications are relatively scarce. With the availability of the recently enriched proteomic dataset ProCan-DepMapSanger, we systematically evaluated the interplays among genomic mutations, transcription, and protein expressions across cancer cell lines. In general, blood cancers have higher RNA-protein correlations than those in solid cancers. The differential expression analysis on protein data helped identify more expressional and functional impact of genomic mutations of cancer genes. We also integrated the proteomic map with drug molecular chemical features to construct a bi-modal machine learning model to infer the drug sensitivities of cancer cell lines. Our results demonstrated that protein quantifications can lead to better drug response prediction performance than the model trained on transcriptome profiles. In addition, integrating protein data with drug chemical features, represented as molecular graphs and learned by Graph Neural Network, outperformed the state-of-the-art model DeepOmicNet for drug response prediction in proteomics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c89e8ade30b207677f590882fddb023a8cd320d3" target='_blank'>
              Drug Response Modeling across Cancers: Proteomics vs. Transcriptomics
              </a>
            </td>
          <td>
            Zetian Zheng, Lei Huang, Fuzhou Wang, Linjing Liu, Jixiang Yu, Weidun Xie, Xingjian Chen, Xiangtao Li, Ka-Chun Wong
          </td>
          <td>2024-12-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28cbf9783c62fb0f01914152c3e6d88182c98411" target='_blank'>
              LKB1 inactivation promotes epigenetic remodeling-induced lineage plasticity and antiandrogen resistance in prostate cancer
              </a>
            </td>
          <td>
            Fei Li, Pengfei Dai, Huili Shi, Yajuan Zhang, Juan He, Anuradha Gopalan, Dan Li, Yu Chen, Yarui Du, Guoliang Xu, Weiwei Yang, Chao Liang, Dong Gao
          </td>
          <td>2025-01-01</td>
          <td>Cell Research</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="Background Genomic data is essential for clinical decision-making in precision oncology. Bioinformatic algorithms are widely used to analyze next-generation sequencing (NGS) data, but they face two major challenges. First, these pipelines are highly complex, involving multiple steps and the integration of various tools. Second, they generate features that are human-interpretable but often result in information loss by focusing only on predefined genetic properties. This limitation restricts the full potential of NGS data in biomarker extraction and slows the discovery of new biomarkers in precision oncology. Methods We propose an end-to-end deep learning (DL) approach for analyzing NGS data. Specifically, we developed a multiple instance learning DL framework that integrates somatic mutation sequences to predict two compound biomarkers: microsatellite instability (MSI) and homologous recombination deficiency (HRD). To achieve this, we utilized data from 3,184 cancer patients obtained from two public databases: The Cancer Genome Atlas (TCGA) and the Clinical Proteome Tumor Analysis Consortium (CPTAC). Results Our proposed deep learning method demonstrated high accuracy in identifying clinically relevant biomarkers. For predicting MSI status, the model achieved an accuracy of 0.98, a sensitivity of 0.95, and a specificity of 1.00 on an external validation cohort. For predicting HRD status, the model achieved an accuracy of 0.80, a sensitivity of 0.75, and a specificity of 0.86. Furthermore, the deep learning approach significantly outperformed traditional machine learning methods in both tasks (MSI accuracy, p-value = 5.11×10−18; HRD accuracy, p-value = 1.07×10−10). Using explainability techniques, we demonstrated that the model’s predictions are based on biologically meaningful features, aligning with key DNA damage repair mutation signatures. Conclusion We demonstrate that deep learning can identify patterns in unfiltered somatic mutations without the need for manual feature extraction. This approach enhances the detection of actionable targets and paves the way for developing NGS-based biomarkers using minimally processed data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad073d80b11f20c88ed32369a6393b6a5510e50a" target='_blank'>
              Deep Learning for Biomarker Discovery in Cancer Genomes
              </a>
            </td>
          <td>
            Michaela Unger, C. Loeffler, L. Žigutytė, Srividhya Sainath, Tim Lenz, Julien Vibert, Andreas Mock, S. Fröhling, Trevor A. Graham, Zunamys I. Carrero, J. N. Kather
          </td>
          <td>2025-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Loss of function screens using shRNA (short hairpin RNA) and CRISPR (clustered regularly interspaced short palindromic repeats) are routinely used to identify genes that modulate responses of tumor cells to anti-cancer drugs. Here, by integrating GSEA (Gene Set Enrichment Analysis) and CMAP (Connectivity Map) analyses of multiple published shRNA screens, we identified a core set of pathways that affect responses to multiple drugs with diverse mechanisms of action. This suggests that these pathways represent “weak points” or “Achilles heels”, whose mild disturbance should make cancer cells vulnerable to a variety of treatments. These “weak points” include proteasome, protein synthesis, RNA splicing, RNA synthesis, cell cycle, Akt-mTOR, and tight junction-related pathways. Therefore, inhibitors of these pathways are expected to sensitize cancer cells to a variety of drugs. This hypothesis was tested by analyzing the diversity of drugs that synergize with FDA-approved inhibitors of the proteasome, RNA synthesis, and Akt-mTOR pathways. Indeed, the quantitative evaluation indicates that inhibitors of any of these signaling pathways can synergize with a more diverse set of pharmaceuticals, compared to compounds inhibiting targets distinct from the “weak points” pathways. Our findings described here imply that inhibitors of the “weak points” pathways should be considered as primary candidates in a search for synergistic drug combinations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d68c29dcb0705107711d0d05473f868fd7dbb129" target='_blank'>
              Cancer Cell’s Achilles Heels: Considerations for Design of Anti-Cancer Drug Combinations
              </a>
            </td>
          <td>
            Valid Gahramanov, Frederick S. Vizeacoumar, Alain Morejon Morales, Keith Bonham, M. Sakharkar, Santosh Kumar, Franco J Vizeacoumar, A. Freywald, Michael Y. Sherman
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Despite its relatively recent emergence, single-cell sequencing has cemented its place in scientific research. It has grown exponentially in less than two decades since its start, with broad impact in the biological sciences. The blood represents an attractive system for early development and application of single-cell technologies. As a result, single-cell analyses in blood and leukemia have led the way in describing how cellular heterogeneity affects cancer progression. In this review, we discuss the technological and conceptual advances brought by single-cell genomics, ranging from genetic evolution and differentiation states that mediate drug resistance to the complex interactions required for immunotherapy responses. These high-resolution insights are starting to enter clinical assessment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38fa592467bc471ca2cb2ea46b2951c1d4aa2628" target='_blank'>
              Insights into Leukemia from Single-Cell Genomics
              </a>
            </td>
          <td>
            Nurefsan Sariipek, Peter van Galen
          </td>
          <td>2024-12-11</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Sinonasal squamous cell carcinoma (SNSCC) is a rare and aggressive malignancy with limited treatment options, necessitating comprehensive molecular characterization to uncover actionable therapeutic targets. Here, we integrate bulk and single-cell transcriptomics, epigenomics, and DNA methylation profiling to construct a detailed molecular and cellular landscape of SNSCC. Our analysis identifies five transcriptionally and epigenetically distinct malignant subpopulations, including hypoxic (TC1) and proliferative (TC2) clusters, with TC1 showing significant association with poor clinical outcomes. We uncover a hypoxia-driven angiogenic axis, wherein TC1 cells secrete adrenomedullin (ADM), MIF, and VEGFA to activate endothelial tip cells, fostering tumor angiogenesis. Epigenetic reprogramming, characterized by alterations in DNA methylation and chromatin accessibility, underpins these transcriptional programs, with AP-1 transcription factors emerging as key regulators. Notably, ADM expression is epigenetically controlled, correlates with advanced clinical stage, and predicts reduced survival in head and neck cancer patients. This multi-omic study highlights the tumor heterogeneity and microenvironmental interactions driving SNSCC progression, revealing epigenetically regulated pathways as promising therapeutic targets for future intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0f1bcee9ecccbc4aae5987d7312ce58acb94e80" target='_blank'>
              Multi-omic profiling identifies malignant subpopulations and a hypoxia-driven angiogenic axis as therapeutic vulnerabilities in sinonasal squamous cell carcinoma
              </a>
            </td>
          <td>
            Chaelin You, Jaewoo Park, Jung Yeon Jang, Myeong S. Yu, Yoo-Sam Chung, Keunsoo Kang, Jihwan Park, Ji Heui Kim, Kyuho Kang
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Abstract Alterations in the interplay between the nucleus and the cell cycle during cancer development lead to a state of genomic instability, often accompanied by observable morphologic aberrations. Tumor cells can regulate these aberrations to evade cell death, either by preventing or eliminating genomic instability. In epithelial ovarian cancer, overexpression of claudin-4 significantly contributes to therapy resistance through mechanisms associated with genomic instability regulation. However, the molecular mechanisms underlying claudin-4 overexpression in epithelial ovarian cancer remain poorly understood. In this study, we modified claudin-4 expression and employed a unique claudin mimic peptide to investigate claudin-4’s function. Our findings show that claudin-4 supports ovarian cancer cell survival by stabilizing the genome through nuclear and cell-cycle remodeling. Specifically, claudin-4 induced nuclear constriction by excluding lamin B1 and promoting perinuclear F-actin accumulation, thereby altering nuclear structure and dynamics. Similarly, cell-cycle modifications due to claudin-4 overexpression resulted in fewer cells entering the S-phase and reduced genomic instability in tumors. Importantly, disrupting claudin-4’s biological effects using claudin mimic peptide and forskolin increased the efficacy of PARP inhibitor treatment, correlating with alterations in the oxidative stress response. Our data indicate that claudin-4 protects tumor genome integrity by modulating the crosstalk between the nucleus and the cell cycle, leading to resistance to genomic instability formation and the effects of genomic instability–inducing agents. Significance: High-grade serous ovarian carcinoma is marked by chromosomal instability, which can serve to promote disease progression and allow cancer to evade therapeutic insults. The report highlights the role of claudin-4 in regulating genomic instability and proposes a novel therapeutic approach to exploit claudin-4–mediated regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/def54cc761c279cc713a50f2bd7fb9ee4de37118" target='_blank'>
              Claudin-4 Stabilizes the Genome via Nuclear and Cell-Cycle Remodeling to Support Ovarian Cancer Cell Survival
              </a>
            </td>
          <td>
            Fabian R. Villagomez, Julie Lang, Daniel Nunez-Avellaneda, K. Behbakht, Hannah L Dimmick, Patricia Webb, K. Nephew, Margaret Neville, Elizabeth R. Woodruff, Benjamin G. Bitler
          </td>
          <td>2024-12-03</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Chromosomal inversions are structural mutations resulting in the reversal of the gene order along the corresponding genomic region. Due to their influence on recombination patterns, they can have a major influence on genetic variation and the evolutionary process. Accordingly, inversions can act as supergenes that keep together co-adapted gene complexes that form the genetic basis of many complex phenotypes in diverse organisms. In this book chapter, I will present an analysis pipeline to investigate the influence of two common cosmopolitan inversion, In(2L)t and In(3R)Payne, on genome-wide genetic variation and differentiation in world-wide populations of the vinegar fly Drosophila melanogaster. We will use single-individual and pooled resequencing data in combination with population genomics analysis tools to explore the impact of these two inversions on genetic variation, population structure, and clinal variation in natural populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/905a067e82a9aaa60453684c2c953608a81fdbab" target='_blank'>
              The influence of chromosomal inversions on genetic variation and clinal patterns in genomic data of Drosophila melanogaster
              </a>
            </td>
          <td>
            M. Kapun
          </td>
          <td>2024-12-02</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Osteosarcoma (OS) is a prevalent primary malignant bone tumor, typically managed through a combination of neoadjuvant chemotherapy and surgical interventions. Recent advancements in early detection and the use of novel chemotherapeutic agents have significantly improved the 5-year survival rate of OS patients. However, some patients fail to achieve the desired treatment outcomes despite undergoing intensive chemotherapy and surgicals procedures, with chemotherapy resistance emerging as a critical factor contributing to therapeutic failure in OS. Noncoding RNAs (ncRNAs) are a group of RNAs that lack protein-coding capacity but play a crucial role in tumor progression by modulating various biological characteristics of cancer cells, such as proliferation, apoptosis, migration, invasion, and drug resistance. Emerging evidence indicates that the dysregulated expression of numerous ncRNAs in OS cells can influence the response to chemotherapeutic agents by modulating processes such as cell apoptosis, signaling pathways, intracellular drug concentrations, and cell autophagy. This review aims to elucidate the roles and mechanisms of ncRNAs in mediating drug resistance in OS, offering new insights for investigating novel pathways underlying drug resistance, overcoming tumor resistance to therapeutics, and developing innovative chemotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbb08d17a2f1faca4973ae29ad1b2a3dacb83f65" target='_blank'>
              Unraveling the impact of noncoding RNAs in osteosarcoma drug resistance: a review of mechanisms and therapeutic implications.
              </a>
            </td>
          <td>
            Y. Pei, Shenglong Li
          </td>
          <td>2024-12-31</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Replication timing (RT) allows us to analyze temporal patterns of genome-wide replication, i.e., if genes replicate early or late during the S-phase of the cell cycle. RT has been linked to gene expression in normal and diseased acute and chronic states such as cancer. However, studies done to date focused on bulk cell populations that required tens of thousands of cells for RT analysis. Here, we developed an affordable novel single cell (sc)-multiomics approach to simultaneously analyze RT and gene expression from cells or nuclei. We used this approach to generate sc-RT profiles and sc-gene expression data from the well-established human liver cancer cell line, HepG2. We demonstrated that as few as 17 mid S-phase cells were sufficient to produce cell-type specific pseudo bulk RT profiles that had a high correlation to previously published HepG2 bulk RT profiles. The sc-RT profiles allowed us to visualize how individual cells progressed through genome replication. We were also able to demonstrate high-resolution correlations between RT and gene expression within each individual cell, which to our knowledge, has not been reported. We observed trends that were conserved between individual cells, as well as cell-to-cell variations, which were not possible to detect with the bulk RT studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36c8d64daee99beea62c15fd3842a66ac8960670" target='_blank'>
              Development of a Novel Single Cell Multiomics Approach for Simultaneous Analysis of Replication Timing and Gene Expression
              </a>
            </td>
          <td>
            Anala V. Shetty, Clifford J. Steer, Walter C. Low
          </td>
          <td>2025-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9043774f4d71a3f413b248306070648bf9c170e4" target='_blank'>
              Multi-omics approaches reveal that diffuse midline gliomas present altered DNA replication and are susceptible to replication stress therapy
              </a>
            </td>
          <td>
            Anastasia E. Hains, Kashish Chetal, Tsunetoshi Nakatani, Joana G. Marques, Andreas Ettinger, C. B. Junior, Adriana Gonzalez-Sandoval, Renjitha Pillai, Mariella G. Filbin, M. Torres-Padilla, Ruslan I. Sadreyev, Capucine Van Rechem
          </td>
          <td>2024-12-20</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Osteosarcoma (OS) is the most prevalent malignant bone tumor in adolescents and young adults. OS cells grow in a permissive local microenvironment which modulates their behavior and facilitates all steps in tumor development (e.g., proliferation/quiescence, invasion/migration, and drug resistance) and contributes to their intrinsic heterogeneity. The lung parenchyma is the most common metastatic site in OS, and metastatic foci are frequently associated with a poor clinical outcome. Although multiple factors may be responsible for the disease, including genetic mutations (e.g., Rb and p53), the molecular mechanism of development of OS remains unclear, and the conventional treatment for OS is still based on a sequential approach that combines chemotherapy and surgery. Also, despite the increase in clinical trials, the survival rates for OS have not improved. Non-specific targeting therapies thus show poor therapeutic effects, along with side effects at high doses. For these reasons, many efforts have been made to characterize the complex genome of OS thanks to the whole-exome analysis, with the aim of identifying predictive biomarkers to give these patients a better therapeutic option. This review aims to summarize and discuss the main recent advances in OS molecular research for precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62807be22de865f1f377913f1a5b07abd346d218" target='_blank'>
              Whole-Exome Analysis and Osteosarcoma: A Game Still Open
              </a>
            </td>
          <td>
            C. Chiappetta, Carlo Della Rocca, Claudio Di Cristofano
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Despite extensive research on antibiotic resistance, the potential effects of antibiotic treatments on bacterial tolerance and resistance remain a significant concern. Although bacterial cells adopt a variety of mutational strategies to resist unfavorable circumstances, it is still unclear how antibiotic tolerance and resistance mechanisms affect bacterial fitness characteristics and whether evolved mutants exhibit similar properties across different cell populations subjected to the same conditions. Here, we used Escherichia coli, a fluoroquinolone antibiotic (ofloxacin), and adaptive laboratory evolutionary experiments to demonstrate that ofloxacin tolerance and resistance can evolve independently across different cell populations exposed to identical conditions. Fitness attributes, such as lag score, doubling time, competition score, and other metabolic features, were variably affected by antibiotic tolerance and resistance mechanisms. However, we did not observe strong and apparent correlations between fitness trade-offs and antibiotic tolerance and resistance. While our whole-genome sequencing identified some shared mutations, such as single nucleotide polymorphisms in the icd gene (a crucial citric acid cycle gene), evolved cell populations exhibited diverse genetic mutations without a clear pattern of a conserved evolutionary pathway. Our study also identifies unique phenotypes, such as those displaying significantly lower minimum inhibitory concentration levels compared to the parental strain yet showing remarkably high tolerance to the same antibiotic. Altogether, our study, examining the phenotypic and mutational landscapes of fluoroquinolone-induced strains, contributes to our understanding of complex bacterial adaptation mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d5760f3238ce410965ca0f13f5c55e28d885a91" target='_blank'>
              The Diverse Phenotypic and Mutational Landscape Induced by Fluoroquinolone Treatment
              </a>
            </td>
          <td>
            S. G. Mohiuddin, Pouria Kavousi, Diego Figueroa, Sreyashi Ghosh, Mehmet A. Orman
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Background: Drug resistance is a major obstacle to the successful treatment of non-small cell lung cancer (NSCLC), the most common subtype of lung cancer. The purpose of this research was to look at possible treatment approaches and the processes behind drug resistance in non-small cell lung cancer.
Methods: From March 2022 to February 2024, a prospective observational research was carried out at Lady Reading Hospital in Peshawar. 128 NSCLC patients in all were enrolled. Next-generation sequencing, DNA methylation tests, and immunological profiling were used to examine the genetic, epigenetic, and environmental components linked to medication resistance. Multivariate logistic regression, t-tests, chi-square tests, and Kaplan-Meier survival analysis were among the statistical techniques used.
Results: T790M mutations contributed to resistance, with EGFR mutations found in 35% of patients. Ten percent of patients had MET amplifications, while seven percent of patients had ALK rearrangements. In resistant cancers, epigenetic changes such as hypoacetylation of histones and hypermethylation of CDKN2A were common. Immunosuppressive cells and cancer-associated fibroblasts were abundant in the tumor microenvironment. The median overall survival (OS) of patients with drug-resistant NSCLC was 14 months, a substantial decrease from the 22 months seen in patients without resistance (p < 0.001).
Conclusions: The research emphasizes how genetic, epigenetic, and microenvironmental variables all play a role in the complex process of medication resistance in non-small cell lung cancer. Molecular profiling and targeted medicines along with personalized treatment techniques are critical to overcome resistance and enhancing patient outcomes. Larger, more varied populations and the creation of cutting-edge treatment approaches should be the main topics of future study.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96863261655834ce5a911e129e2f5a3a8ee64f44" target='_blank'>
              Understanding the Mechanisms of Drug Resistance in Non-Small Cell Lung Cancer Potential Therapeutic Approaches
              </a>
            </td>
          <td>
            Afnan Ali Shah, Waqar Ahmad Tahir, Owais Nawaz Khan, A. Abid, Yasar Umar Khan, Talha Mazhar
          </td>
          <td>2024-12-31</td>
          <td>Innovative Research in Applied, Biological and Chemical Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Cellular senescence plays a significant role in tissue aging. Senescent cells, which resist apoptosis while remaining metabolically active, generate endogenous DNA-damaging agents, primarily reactive oxygen species. Efficient DNA repair is therefore crucial in these cells, especially when they undergo senescence escape, resuming DNA replication and cellular proliferation. To investigate whether senescent cell transcriptomes reflect adequate DNA repair capacity, we conducted a comprehensive meta-analysis of 60 transcriptomic datasets comparing senescent to proliferating cells. Our analysis revealed a striking downregulation of genes encoding essential components across DNA repair pathways in senescent cells. This includes pathways active in different cell cycle phases such as nucleotide excision repair, base excision repair, nonhomologous end joining and homologous recombination repair of double-strand breaks, mismatch repair and interstrand crosslink repair. The downregulation observed suggests a significant accumulation of DNA lesions. Experimental monitoring of DNA repair readouts in cells that underwent radiation-induced senescence supported this conclusion. This phenomenon was consistent across various senescence triggers and was also observed in primary cell lines from aging individuals. These findings highlight the potential of senescent cells as ‘ticking bombs’ in aging-related diseases and tumors recurring following therapy-induced senescence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22e8d9a9bd09842f0e1e6fb75af8a57d50e982d1" target='_blank'>
              Broad repression of DNA repair genes in senescent cells identified by integration of transcriptomic data
              </a>
            </td>
          <td>
            Yann Frey, M. Haj, Y. Ziv, Ran Elkon, Yosef Shiloh
          </td>
          <td>2024-12-31</td>
          <td>Nucleic Acids Research</td>
          <td>1</td>
          <td>40</td>
        </tr>

        <tr id="Surveillance of drug resistance and the discovery of novel targets-key objectives in the fight against malaria-rely on identifying resistance-conferring mutations in Plasmodium parasites. Current approaches, while successful, require laborious experimentation or large sample sizes. To elucidate shared determinants of antimalarial resistance that can empower in silico inference, we examined the genomes of 724 Plasmodium falciparum clones, each selected in vitro for resistance to one of 118 compounds. We identified 1448 variants in 128 recurrently mutated genes, including drivers of antimalarial multidrug resistance. In contrast to naturally occurring variants, those selected in vitro are more likely to be missense or frameshift, involve bulky substitutions, and occur in conserved, ordered protein domains. Collectively, our dataset reveals mutation features that predict drug resistance in eukaryotic pathogens.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1f30299235adf20cff0919e33302a042257592b" target='_blank'>
              Systematic in vitro evolution in Plasmodium falciparum reveals key determinants of drug resistance.
              </a>
            </td>
          <td>
            M. Luth, Karla P. Godinez-Macias, Daisy Chen, John Okombo, Vandana Thathy, Xiu Cheng, Sindhu Daggupati, Heledd Davies, Satish K. Dhingra, Jan M Economy, R. C. Edgar, Maria G. Gomez-Lorenzo, E. Istvan, J. Jado, Gregory M. LaMonte, Bruno Melillo, Sachel Mok, Sunil K Narwal, T. Ndiaye, S. Ottilie, Sara Palomo Diaz, Heekuk Park, S. Peña, Frances Rocamora, Tomoyo Sakata-Kato, Jennifer L. Small-Saunders, Robert L Summers, Patrick K. Tumwebaze, M. Vanaerschot, Guoqin Xia, Tomas Yeo, Ashley You, F. Gamo, Daniel E. Goldberg, M. Lee, C. McNamara, Daouda Ndiaye, Philip J. Rosenthal, Stuart L. Schreiber, G. Serra, J. L. De Siqueira-Neto, T. Skinner-Adams, A. Uhlemann, Nobutaka Kato, A. Lukens, D. Wirth, D. Fidock, E. Winzeler
          </td>
          <td>2024-11-29</td>
          <td>Science</td>
          <td>1</td>
          <td>89</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e77d5a82f46b1391391ade8c249b0d27a2b059b" target='_blank'>
              Comprehensive genomic characterization of early-stage bladder cancer
              </a>
            </td>
          <td>
            F. Prip, P. Lamy, S. Lindskrog, T. Strandgaard, I. Nordentoft, K. Birkenkamp-Demtröder, N. Birkbak, Nanna Kristjánsdóttir, Asbjørn Kjær, T. Andreasen, J. Ahrenfeldt, J. S. Pedersen, A. Rasmussen, Gregers G Hermann, K. Mogensen, Astrid C Petersen, Arndt Hartmann, M. Grimm, Marcus Horstmann, R. Nawroth, U. Segersten, D. Sikic, K. V. van Kessel, E. Zwarthoff, Tobias Maurer, Tatjana Simic, Per-Uno Malmström, N. Malats, J. B. Jensen, Francisco X. Real, L. Dyrskjøt
          </td>
          <td>2025-01-03</td>
          <td>Nature Genetics</td>
          <td>1</td>
          <td>68</td>
        </tr>

        <tr id="Short tandem repeats (STRs) are a rich source of genetic variation, but are difficult to genotype. While specialized repeat variant callers exist, they typically assume a euploid human genome. This means recent findings regarding phenotypic effects of STR variants in human health and disease cannot be readily extended to polyploid organisms or cancer, which is characterised by copy number alterations (CNAs). Here we present ConSTRain, a novel STR variant caller that explicitly accounts for the copy number of loci in its genotyping approach. We benchmark ConSTRain using a euploid human 100X whole genome sequencing sample where it calls STR allele lengths for over 1.7 × 106 loci in under 20 minutes with an accuracy of 98.28%. Subsequently, we show that ConSTRain resolves complex STR genotypes in an artificial trisomy 21 sample and a polyploid Dwarf Cavendish banana harbouring a large duplication. Finally, we analyse a microsatellite instable colorectal cancer tumoroid, where ConSTRain tackles CNAs and whole-genome duplications. ConSTRain is the first STR variant caller that allows for the investigation of repeats affected by CNAs, aneuploidies, and polyploid genomes. This unlocks the investigation of STRs across a wide range of contexts and organisms where they previously could not be easily studied.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42abd302650eb39af080a99a7d76a61bf128e039" target='_blank'>
              Genotyping Short Tandem Repeats Across Copy Number Alterations, Aneuploidies, and Polyploid Organisms
              </a>
            </td>
          <td>
            M. Verbiest, Elena Grassi, Andrea Bertotti, Maria Anisimova
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The P-glycoprotein efflux pump, encoded by the MDR1 gene, is an ATP-driven transporter capable of expelling a diverse array of compounds from cells. Overexpression of this protein is implicated in the multi-drug resistant phenotype observed in various cancers. Numerous studies have attempted to decipher the impact of genetic variants within MDR1 on P-glycoprotein expression, functional activity, and clinical outcomes in cancer patients. Among these, three specific single nucleotide polymorphisms—T1236C, T2677G, and T3435C - have been the focus of extensive research efforts, primarily through in vitro cell line models and clinical cohort analyses. However, the findings from these studies have been remarkably contradictory. In this study, we employ a computational, data-driven approach to systematically evaluate the effects of these three variants on principal stages of the gene expression process. Leveraging current knowledge of gene regulatory mechanisms, we elucidate potential mechanisms by which these variants could modulate P-glycoprotein levels and function. Our findings suggest that all three variants significantly change the mRNA folding in their vicinity. This change in mRNA structure is predicted to increase local translation elongation rates, but not to change the protein expression. Nonetheless, the increased translation rate near T3435C is predicted to affect the protein’s co-translational folding trajectory in the region of the second ATP binding domain. This potentially impacts P-glycoprotein conformation and function. Our study demonstrates the value of computational approaches in elucidating the functional consequences of genetic variants. This framework provides new insights into the molecular mechanisms of MDR1 variants and their potential impact on cancer prognosis and treatment resistance. Furthermore, we introduce an approach which can be systematically applied to identify mutations potentially affecting mRNA folding in pathology. We demonstrate the utility of this approach on both ClinVar and TCGA and identify hundreds of disease related variants that modify mRNA folding at essential positions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3513112218d83dd11f2ce2e7d98d7d9261c6c5e8" target='_blank'>
              Computational Analysis of MDR1 Variants Predicts Effect on Cancer Cells via their Effect on mRNA Folding
              </a>
            </td>
          <td>
            T. Gutman, T. Tuller
          </td>
          <td>2024-12-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb6bea0ac205c045a3acd7f52f1841e20780dc8e" target='_blank'>
              Utilizing sc-linker to integrate single-cell RNA sequencing and human genetics to identify cell types and driver genes associated with non-small cell lung cancer
              </a>
            </td>
          <td>
            Yangfan Zhou, Liang Zhao, Meimei Cai, Doudou Luo, Y. Pang, Jianhao Chen, Qicong Luo, Qin Lin
          </td>
          <td>2025-01-23</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Dysregulated epigenetic programs that restrict differentiation, reactivate fetal genes, and confer phenotypic plasticity are critical to colorectal cancer (CRC) development. By screening a small molecule library targeting epigenetic regulators using our dual reporter system, we found that inhibiting histone deacetylase (HDAC) 1/2 promotes CRC differentiation and anti-tumor activity. Comprehensive biochemical, chemical, and genetic experiments revealed that on-target blockade of the HDAC1/2 catalytic domain mediated the differentiated phenotype. Unbiased profiling of histone posttranslational modifications induced by HDAC1/2 inhibition nominated acetylation of specific histone lysine residues as potential regulators of differentiation. Genome-wide assessment of implicated marks indicated that H3K27ac gains at HDAC1/2-bound regions associated with open chromatin and upregulation of differentiation genes upon HDAC1/2 inhibition. Disrupting H3K27ac by degrading acetyltransferase EP300 rescued HDAC1/2 inhibitor-mediated differentiation of a patient-derived CRC model using single cell RNA-sequencing. Genetic screens revealed that DAPK3 contributes to CRC differentiation induced by HDAC1/2 inhibition. These results highlight the importance of specific chemically targetable histone modifications in governing cancer cell states and epigenetic reprogramming as a therapeutic strategy in CRC. BRIEF SUMMARY HDAC1/2 inhibition promotes colorectal cancer differentiation via gains in H3K27ac, which can be reversed by blocking its acetyltransferase EP300.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/207eb16a60f11154e81c963033e896c8385daaa7" target='_blank'>
              Chemical perturbations impacting histone acetylation govern colorectal cancer differentiation
              </a>
            </td>
          <td>
            Pornlada Likasitwatanakul, Zhixin Li, Paul Doan, S. Spisák, Akhouri Kishore Raghawan, Qi Liu, Priscilla Liow, Sunwoo Lee, David Chen, Pratyusha Bala, P. Sahgal, Daulet Aitymbayev, Jennifer S. Thalappillil, Malvina Papanastasiou, William Hawkins, Steven A. Carr, Haeseong Park, James M. Cleary, Jun Qi, Nilay S. Sethi
          </td>
          <td>2024-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4883d3048c2471ea2cd7612bfae069249c9382eb" target='_blank'>
              Integrated analysis of cell cycle and p53 signaling pathways related genes in breast, colorectal, lung, and pancreatic cancers: implications for prognosis and drug sensitivity for therapeutic potential
              </a>
            </td>
          <td>
            Jiyauddin Khan, Priyanjana Ghosh, U. Bajpai, Kountay Dwivedi, D. Saluja
          </td>
          <td>2024-12-01</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Understanding the mechanisms of drug resistance in Mycobacterium tuberculosis (MTB) is essential for the rapid detection of resistance and for guiding effective treatment, ultimately contributing to reducing the global burden of tuberculosis (TB). Under anti-TB drugs pressure, MTB continues to accumulate resistance loci. The current repertoire of known resistance-associated mutations requires further refinement, necessitating efficient methods for the timely identification of potential resistance sites. Here, we introduce xAI-MTBDR, an explainable artificial intelligence framework designed to identify new resistance-associated mutations and predict drug resistance in MTB. It outperforms state-of-the-art methods in predicting drug resistance for all first-line drugs, and scoring each mutation’s contribution to resistance. By leveraging public whole-genome sequencing data from nearly 40,000 MTB isolates, the framework identified 788 candidate resistance-related mutations and revealed 30 potential resistance markers, several of which are structurally located closer to their respective drugs compared to known resistance mutations. Furthermore, these scores enabled the framework to efficiently subgroup isolates with different resistance mechanisms and reflect varying levels of resistance. The framework serves as a valuable tool for accurate detection of drug-resistant MTB and offers new insights into its underlying mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82a491d3ac5efa19842f5a8f507588bda7f5b18d" target='_blank'>
              An explainable artificial intelligence framework reveals mutations associated with drug resistance in Mycobacterium tuberculosis
              </a>
            </td>
          <td>
            Hui Cen, Peng Zhang, Yunchao Ling, Guoping Zhao, Guoqing Zhang
          </td>
          <td>2024-12-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Success of phage therapies is limited by bacterial defenses against phages. While a large variety of anti- phage defense mechanisms has been characterized, how expression of these systems is distributed across individual cells and how their combined activities translate into protection from phages has not been studied. Using bacterial single-cell RNA sequencing, we profiled the transcriptomes of ∼50,000 cells from cultures of a human pathobiont, Bacteroides fragilis, infected with a lytic bacteriophage. We quantified the asynchronous progression of phage infection in single bacterial cells and reconstructed the infection timeline, characterizing both host and phage transcriptomic changes as infection unfolded. We discovered a subpopulation of bacteria that remained uninfected and determined the heterogeneously expressed host factors associated with protection. Each cell’s vulnerability to phage infection was defined by combinatorial phase-variable expression of multiple genetic loci, including capsular polysaccharide (CPS) biosynthesis pathways, restriction-modification systems (RM), and a previously uncharacterized operon likely encoding fimbrial genes. By acting together, these heterogeneously expressed phase-variable systems and anti-phage defense mechanisms create a phenotypic landscape where distinct protective combinations enable the survival and re-growth of bacteria expressing these phenotypes without acquiring additional mutations. The emerging model of complementary action of multiple protective mechanisms heterogeneously expressed across an isogenic bacterial population showcases the potent role of phase variation and stochasticity in bacterial anti-phage defenses. One Sentence Summary Combinatorial phenotypic states with differential vulnerability to phage infection across a Bacteroides fragilis population enable a small number of super-resistant bacterial cells to evade the phage without the need for acquiring mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10ce7b8bbb533eea20bff9d1748f05179e8fd243" target='_blank'>
              Combinatorial phenotypic landscape enables bacterial resistance to phage infection
              </a>
            </td>
          <td>
            Anika Gupta, Norma Morella, Dmitry Sutormin, Naisi Li, Karl D Gaisser, Alexander Robertson, Yaroslav Ispolatov, Georg Seelig, Neelendu Dey, A. Kuchina
          </td>
          <td>2025-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc4ce21cb54a65498b2c72289ec6bd7eb2e7dc62" target='_blank'>
              Characterization of shared neoantigens landscape in Mismatch Repair Deficient Endometrial Cancer
              </a>
            </td>
          <td>
            E. De Paolis, C. Nero, E. Micarelli, Guido Leoni, A. Piermattei, R. Trozzi, Elisa Scarselli, A. D’Alise, Luciano Giacò, M. De Bonis, A. Preziosi, Gennaro Daniele, Diletta Piana, T. Pasciuto, Gianfranco Zannoni, Angelo Minucci, G. Scambia, Andrea Urbani, Francesco Fanfani
          </td>
          <td>2024-12-20</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Glioblastoma (GBM), the most prevalent primary malignant brain tumor, remains challenging to treat due to extensive inter- and intra-tumor heterogeneity. This variability demands combination treatments to improve therapeutic outcomes. A significant obstacle in treating GBM is the expression of O6-methylguanine-DNA methyltransferase, a DNA repair enzyme that reduces the efficacy of the standard alkylating agent, temozolomide, in about 50% of patients. This underscores the need for novel, more targeted therapies. Our study investigates the metabolic-epigenetic impact of combining SN-38, a novel topoisomerase inhibitor inducing DNA double-strand breaks, with rabusertib, a checkpoint kinase 1 inhibitor. We identified this synergistic combination through high-throughput drug screening across a panel of GBM cell lines using a cancer drug library combined with SN-38. A secondary metabolic screening with the PEDS algorithm demonstrated a synergistic modulation of purine, one-carbon, and redox metabolism. Furthermore, the combined treatment led to the significant depletion of epigenetically relevant metabolites such as 5-methyl-cytosine, acetyl-lysine, and trimethyl-lysine. Reduced intermediates of the glutathione cycle indicated increased cellular stress following combinatorial treatment. Overall, the combination of SN-38 and rabusertib synergistically disrupts metabolites associated with epigenetic adaptations, leading to cytotoxicity independent of O6-methylguanine-DNA methyltransferase status, thereby underpinning this combination as a promising candidate for combinatorial therapy in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ed32abd0bdaa8ba2724fd42a8df45084943d6e1" target='_blank'>
              Targeting Metabolic and Epigenetic Vulnerabilities in Glioblastoma with SN-38 and Rabusertib Combination Therapy.
              </a>
            </td>
          <td>
            J. Chiou, Valeria Impedovo, Yen Bao Huynh, Ruggiero Gorgoglione, Luiz O. F. Penalva, A. Lodi, Andrew J. Brenner, Stefano Tiziani
          </td>
          <td>2025-01-08</td>
          <td>International journal of molecular sciences</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Evolution by natural selection is expected to be a slow and gradual process. In particular, the mutations that drive evolution are predicted to be small and modular, incrementally improving a small number of traits. However, adaptive mutations identified early in microbial evolution experiments, cancer, and other systems often provide substantial fitness gains and pleiotropically improve multiple traits at once. We asked whether such pleiotropically adaptive mutations are common throughout adaptation or are instead a rare feature of early steps in evolution that tend to target key signaling pathways. To do so, we conducted barcoded second-step evolution experiments initiated from 5 first-step mutations identified from a prior yeast evolution experiment. We then isolated hundreds of second-step mutations from these evolution experiments, measured their fitness and performance in several growth phases, and conducted whole genome sequencing of the second-step clones. Here, we found that while the vast majority of mutants isolated from the first-step of evolution in this condition show patterns of pleiotropic adaptation—improving both performance in fermentation and respiration growth phases—second-step mutations show a shift towards modular adaptation, mostly improving respiration performance and only rarely improving fermentation performance. We also identified a shift in the molecular basis of adaptation from genes in cellular signaling pathways towards genes involved in respiration and mitochondrial function. Our results suggest that the genes in cellular signaling pathways may be more likely to provide large, adaptively pleiotropic benefits to the organism due to their ability to coherently affect many phenotypes at once. As such, these genes may serve as the source of pleiotropic adaptation in the early stages of evolution, and once these become exhausted, organisms then adapt more gradually, acquiring smaller, more modular mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c50e1354d98f18f12597024756458bd2f24fea92" target='_blank'>
              A high-resolution two-step evolution experiment in yeast reveals a shift from pleiotropic to modular adaptation
              </a>
            </td>
          <td>
            Grant Kinsler, Yuping Li, Gavin Sherlock, Dmitri Petrov
          </td>
          <td>2024-12-01</td>
          <td>PLOS Biology</td>
          <td>2</td>
          <td>6</td>
        </tr>

        <tr id="High-grade serous ovarian cancer (HGSOC), the predominant subtype of epithelial ovarian cancer, is frequently diagnosed at an advanced stage due to its nonspecific early symptoms. Despite standard treatments, including cytoreductive surgery and platinum-based chemotherapy, significant improvements in survival have been limited. Understanding the molecular mechanisms, immune landscape, and drug sensitivity of HGSOC is crucial for developing more effective and personalized therapies. This study integrates insights from cancer immunology, molecular profiling, and drug sensitivity analysis to identify novel therapeutic targets and improve treatment outcomes. Utilizing single-cell RNA sequencing (scRNA-seq), the study systematically examines tumor heterogeneity and immune microenvironment, focusing on biomarkers influencing drug response and immune activity, aiming to enhance patient outcomes and quality of life.scRNA-seq data was obtained from the GEO database in this study. Differential gene expression was analyzed using gene ontology and gene set enrichment methods. InferCNV identified malignant epithelial cells, while Monocle, Cytotrace, and Slingshot software inferred subtype differentiation trajectories. The CellChat software package predicted cellular communication between malignant cell subtypes and other cells, while pySCENIC analysis was utilized to identify transcription factor regulatory networks within malignant cell subtypes. Finally, the analysis results were validated through functional experiments, and a prognostic model was developed to assess prognosis, immune infiltration, and drug sensitivity across various risk groups.This study investigated the cellular heterogeneity of HGSOC using scRNA-seq, focusing on tumor cell subtypes and their interactions within the tumor microenvironment. We confirmed the key role of the C2 IGF2+ tumor cell subtype in HGSOC, which was significantly associated with poor prognosis and high levels of chromosomal copy number variations. This subtype was located at the terminal differentiation of the tumor, displaying a higher degree of malignancy and close association with stage IIIC tissue types. The C2 subtype was also associated with various metabolic pathways, such as glycolysis and riboflavin metabolism, as well as programmed cell death processes. The study highlighted the complex interactions between the C2 subtype and fibroblasts through the MK signaling pathway, which may be closely related to tumor-associated fibroblasts and tumor progression. Elevated expression of PRRX1 was significantly connected to the C2 subtype and may impact disease progression by modulating gene transcription. A prognostic model based on the C2 subtype demonstrated its association with adverse prognosis outcomes, emphasizing the importance of immune infiltration and drug sensitivity analysis in clinical intervention strategies.This study integrates molecular oncology, immunotherapy, and drug sensitivity analysis to reveal the mechanisms driving HGSOC progression and treatment resistance. The C2 IGF2+ tumor subtype, linked to poor prognosis, offers a promising target for future therapies. Emphasizing immune infiltration and drug sensitivity, the research highlights personalized strategies to improve survival and quality of life for HGSOC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f67893a76513f786963588a0c8fdf842d93da2fd" target='_blank'>
              Characterizing tumor biology and immune microenvironment in high-grade serous ovarian cancer via single-cell RNA sequencing: insights for targeted and personalized immunotherapy strategies
              </a>
            </td>
          <td>
            Fu Zhao, Xiaojing Jiang, Yumeng Li, Tianjiao Huang, Zhikai Xiahou, Wenyang Nie, Qian Li
          </td>
          <td>2025-01-17</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Abstract Large genetic variants can be generated via homologous recombination (HR), such as polymerase theta-mediated end joining (TMEJ) or single-strand annealing (SSA). Given that these HR-based mechanisms leave specific genomic signatures, we developed GDBr, a genomic signature interpretation tool for DNA double-strand break repair mechanisms using high-quality genome assemblies. We applied GDBr to a draft human pangenome reference. We found that 78.1% of non-repetitive insertions and deletions and 11.0% of non-repetitive complex substitutions contained specific signatures. Of these, we interpreted that 98.7% and 1.3% of the insertions and deletions were generated via TMEJ and SSA, respectively, and all complex substitutions via TMEJ. Since population-level pangenome datasets are being dramatically accumulated, GDBr can provide mechanistic insights into how variants are formed. GDBr is available on GitHub at https://github.com/Chemical118/GDBr.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a24886490d8b6e2e3d68ad1604f388da5f8ce50c" target='_blank'>
              GDBr: genomic signature interpretation tool for DNA double-strand break repair mechanisms
              </a>
            </td>
          <td>
            Hyunwoo Ryu, H. Han, Chuna Kim, Jun Kim
          </td>
          <td>2025-01-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Klebsiella pneumoniae, an opportunistic pathogen, is pervasively distributed across the world. Its escalating antibiotic resistance poses a serious threat to global public health. The mechanisms behind this resistance remain largely elusive. In this study, we performed antibiotic susceptibility testing on several clinical isolates of Klebsiella pneumoniae, and a reference strain ATCC13883, and then analysed their transcriptomic profiles to identify genes and pathways associated with antibiotic resistance. Our results showed that a clinical isolate DY16KPN may counteract antibiotics by enhancing the biosynthesis of building blocks of bacterial cell, such as fatty acids, proteins, and DNA, and reducing transmembrane transport. Increased butanoate metabolism and lipopolysaccharide biosynthesis may also contribute to the drug-resistance of Klebsiella pneumoniae. Additionally, we identified resistance-promoting mutations in gene promoter regions, which regulate the activity of downstream drug-resistant genes and pathways. Our results also demonstrated that DY16KPN counterbalances the trimethoprim/sulfamethoxazole-mediated inhibitory effect on the synthesis of tetrahydrofolates and DNA by up-regulating the expression of targeted enzymes of trimethoprim/sulfamethoxazole, dihydrofolate reductase and dihydropteroate synthase.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cd7409d1c446b41d87b6fccc9e04701e699807c" target='_blank'>
              Transcriptomic analysis reveals pathways underlying the multi-antibiotic resistance of Klebsiella pneumoniae.
              </a>
            </td>
          <td>
            Ying Liu, Zhihui Niu, Rile Wu, Dezhi Yang, Jun Chen, Guoqing Liu, Jun Zhao
          </td>
          <td>2024-12-17</td>
          <td>IET systems biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background: Most tumors initially respond to treatment, yet refractory clones subsequently develop owing to resistance mechanisms associated with cancer cell plasticity and heterogeneity. Methods: We used a chemical biology approach to identify protein targets in cancer cells exhibiting diverse driver mutations and representing models of tumor lineage plasticity and therapy resistance. An unbiased screen of a drug library was performed against cancer cells followed by synthesis of chemical analogs of the most effective drug. The cancer subtype target range of the leading drug was determined by PRISM analysis of over 900 cancer cell lines at the Broad Institute, MA. RNA-sequencing and enrichment analysis of differentially expressed genes, as well as computational molecular modeling and pull-down with biotinylated small molecules were used to identify and validate RPS6KB1 (p70S6K or S6K1) as an essential target. Genetic restoration was used to test the functional role of S6K1 in cell culture and xenograft models. Results: We identified a novel derivative of the antihistamine drug ebastine, designated Super-ebastine (Super-EBS), that inhibited the viability of cancer cells representing diverse KRAS and EGFR driver mutations and models of plasticity and treatment resistance. Interestingly, PRISM analysis indicated that over 95% of the diverse cancer cell lines tested were sensitive to Super-EBS and the predicted target was the serine/threonine kinase S6K1. S6K1 is upregulated in various cancers relative to counterpart normal/benign tissues and phosphorylated-S6K1 predicts poor prognosis for cancer patients. We noted that inhibition of S6K1 phosphorylation was necessary for tumor cell growth inhibition, and restoration of phospho-S6K1 rendered tumor cells resistant to Super-EBS. Inhibition of S6K1 phosphorylation by Super-EBS induced caspase-2 dependent apoptosis via inhibition of the Cdc42/Rac-1/p-PAK1 pathway that led to actin depolymerization and caspase-2 activation. The essential role of S6K1 in the action of Super-EBS was recapitulated in xenografts, and knockout of S6K1 abrogated tumor growth in mice. Conclusion: S6K1 is a therapeutic vulnerability in tumors exhibiting intrinsic and/or acquired resistance to treatment">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e4047ca455e4b4317a09e2a2f99cd529981915d" target='_blank'>
              S6K1 is a Targetable Vulnerability in Tumors Exhibiting Plasticity and Therapy Resistance
              </a>
            </td>
          <td>
            Saptadwipa Ganguly, R. Burikhanov, Vitaliy M. Sviripa, Sally Ellingson, Jieyun Jiang, Christian M. Gosser, David Orren, Eva M. Goellner, G.G. Shenoy, Mahadev Rao, John A. D’Orazio, C. F. Brainson, Chang-Guo Zhan, Peter H. Spielmann, David S. Watt, Vivek M. Rangnekar
          </td>
          <td>2025-01-01</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Breast Cancer (BrCa) exhibits a high phenotypic heterogeneity, leading to the emergence of aggressive clones and the development of drug resistance. Considering the BrCa heterogeneity and that metabolic reprogramming is a cancer hallmark, we selected seven BrCa cell lines with diverse subtypes to provide their comprehensive metabolome characterization: five lines commonly used (SK-Br-3, T-47D, MCF-7, MDA-MB-436, and MDA-MB-231), and two patient-derived xenografts (Hbcx39 and Hbcx9). We characterized their endometabolomes using 1H-NMR spectroscopy. We found distinct metabolite profiles, with certain metabolites being common but differentially accumulated across the selected BrCa cell lines. High levels of glycine, lactate, glutamate, and formate, metabolites known to promote invasion and metastasis, were detected in all BrCa cells. In our experiment setting were identified unique metabolites to specific cell lines: xanthine and 2-oxoglutarate in SK-Br-3, 2-oxobutyrate in T-47D, cystathionine and glucose-1-phosphate in MCF-7, NAD+ in MDA-MB-436, isocitrate in MDA-MB-231, and NADP+ in Hbcx9. The unique and enriched metabolites enabled us to identify the metabolic pathways modulated in a cell-line-specific manner, which may represent potential candidate targets for therapeutic intervention. We believe this study may contribute to the functional characterization of BrCa cells and assist in selecting appropriate cell lines for drug-response studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa4c21e05117b322ca21c171873276565e0f755d" target='_blank'>
              Metabolic Profiling of Breast Cancer Cell Lines: Unique and Shared Metabolites
              </a>
            </td>
          <td>
            Mariana Gallo, Elena Ferrari, F. Brugnoli, Anna Terrazzan, Pietro Ancona, Stefano Volinia, V. Bertagnolo, Carlo M. Bergamini, A. Spisni, T. Pertinhez, Nicoletta Bianchi
          </td>
          <td>2025-01-24</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="The MCF-7R breast cancer cell line, developed by treating the parental MCF-7 cells with increasing doses of doxorubicin, serves as a model for studying acquired multidrug resistance (MDR). MDR is a major challenge in cancer therapy, often driven by overexpression of the efflux pump P-glycoprotein (P-gp) and epigenetic modifications. While many P-gp inhibitors show promise in vitro, their nonspecific effects on the efflux pump limit in vivo application. Curcumin, a natural compound with pleiotropic action, is a nontoxic P-gp inhibitor capable of modulating multiple pathways. To explore curcumin’s molecular effects on MCF-7R cells, we analyzed the expression of genes involved in DNA methylation and transcription regulation, including ABCB1/MDR1. Reduced representation bisulfite sequencing further unveiled key epigenetic changes induced by curcumin. Our findings indicate that curcumin treatment not only modulates critical cellular processes, such as ribosome biogenesis and cytoskeletal dynamics, but also reverses the resistant phenotype, toward that of sensitive cells. This study highlights curcumin’s potential as an adjuvant therapy to overcome chemoresistance, offering new avenues for pharmacological strategies targeting epigenetic regulation to re-sensitize resistant cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a25fe3bb8a31c48e3c18c599af20c640c2363916" target='_blank'>
              Epigenetic and Cellular Reprogramming of Doxorubicin-Resistant MCF-7 Cells Treated with Curcumin
              </a>
            </td>
          <td>
            P. Poma, S. Rigogliuso, M. Labbozzetta, Aldo Nicosia, Salvatore Costa, M. Ragusa, M. Notarbartolo
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Therapy resistance still constitutes a common hurdle in the treatment of many human cancers and is a major reason for treatment failure and patient relapse, concomitantly with a dismal prognosis. In addition to "intrinsic resistance", e.g., acquired by random mutations, cancer cells typically escape from certain treatments ("acquired resistance") by a large variety of means, including suppression of apoptosis and other cell death pathways via upregulation of anti-apoptotic factors or through inhibition of tumor-suppressive proteins. Therefore, ideally, the tumor-cell-restricted induction of apoptosis is still considered a promising avenue for the development of novel, tumor (re)sensitizing therapies. A growing body of evidence has highlighted the multifaceted role of tripartite motif 25 (TRIM25) in controlling different aspects of tumorigenesis, including chemotherapeutic drug resistance. Accordingly, overexpression of TRIM25 is observed in many tumors and frequently correlates with a poor patient survival. In addition to its originally described function in antiviral innate immune response, TRIM25 can play critical yet context-dependent roles in apoptotic- and non-apoptotic-regulated cell death pathways, including pyroposis, necroptosis, ferroptosis, and autophagy. The review summarizes current knowledge of molecular mechanisms by which TRIM25 can interfere with different cell death modalities and thereby affect the success of currently used chemotherapeutics. A better understanding of the complex repertoire of cell death modulatory effects by TRIM25 is an essential prerequisite for validating TRIM25 as a potential target for future anticancer therapy to surmount the high failure rate of currently used chemotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9eb1e40b262aff1ec5083769b832df042944b012" target='_blank'>
              TRIM25: A Global Player of Cell Death Pathways and Promising Target of Tumor-Sensitizing Therapies.
              </a>
            </td>
          <td>
            W. Eberhardt, Usman Nasrullah, Josef Pfeilschifter
          </td>
          <td>2025-01-07</td>
          <td>Cells</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Resistance to BRAF and MAPK inhibitors is a significant challenge in melanoma treatment, driven by adaptive and acquired mechanisms that allow tumour cells to evade therapy. Here, we examined early signalling responses to single and combined BRAF and MAPK inhibition in a BRAFV600E, drug-sensitive melanoma cell line and a drug-resistant ARID1A-knockout (KO) derivative. ARID1A, frequently mutated in melanoma, is associated with resistance and immune evasion. Using an innovative systems biology approach that integrates transcriptomics, proteomics, phosphoproteomics, and functional kinomics through matrix factorization and network analysis, we identified key signalling alterations and resistance mechanisms. We found that ARID1A-KO cells exhibited transcriptional rewiring, sustaining MAPK1/3 and JNK activity post-treatment, bypassing feedback sensitivity observed in parental cells. This rewiring suppressed PRKD1 activation, increased JUN activity—a central resistance network node—and disrupted PKC dynamics through elevated basal RTKs (e.g., EGFR, ROS1) and Ephrin receptor activity post-treatment. ARID1A mutations also reduced HLA-related protein expression and enriched extracellular matrix components, potentially limiting immune infiltration and reducing immunotherapy efficacy. Our graph-theoretical multi-omics approach uncovered novel resistance-associated signalling pathways, identifying PRKD1, JUN, and NCK1 as critical nodes. While receptor activation redundancies complicate single-target therapies, they also present opportunities for combination strategies. This study highlights ARID1A’s role in reshaping signalling and immune interactions, offering new insights into melanoma resistance mechanisms. By identifying actionable targets, including JUN and immune pathways, we provide a foundation for developing integrated therapeutic strategies to overcome resistance in BRAF/MAPK inhibitor-treated melanoma. One sentence summary This study reveals how ARID1A-mediated transcriptional rewiring drives resistance to MAPK inhibitors in melanoma by altering signalling pathways, immune interactions, and receptor dynamics, highlighting potential targets for combinatorial therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6bdb990bb1257653b7e587028f560dd7ef66b129" target='_blank'>
              ARID1A-induced transcriptional reprogramming rewires signalling responses to drug treatment in melanoma
              </a>
            </td>
          <td>
            Charlie George Barker, Sumana Sharma, Ana Mafalda Santos, Konstantinos-Stylianos Nikolakopoulos, Athanassios D. Velentzas, Franziska Völlmy, A. Minia, V. Pliaka, M. Altelaar, Gavin J Wright, Leonidas G. Alexopoulos, D. Stravopodis, E. Petsalaki
          </td>
          <td>2024-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="The orchestration of DNA repair is of fundamental importance to the maintenance of genomic integrity and tumor suppression. DNA damage must be detected in the context of the varied chromatin landscape, its presence must be communicated throughout the cell to alter many ongoing processes, and the machinery that will mend the lesion must be recruited to the damage site. In my presentation, I will discuss our recent efforts in mapping genome maintenance pathways using genome-scale CRISPR/Cas9 screens in human cells. I will highlight how these screens can be used to identify new genome stability factors, characterize drug responses and provide new insights into the genetic architecture of the genome stability network. I will finally present how these screens can also identify potentially actionable synthetic lethal genetic interactions that could form the basis of new oncology drug discovery efforts.
 Citation Format: Daniel Durocher. Synthetic lethality: Pathways to therapeutic discovery. [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Optimizing Therapeutic Efficacy and Tolerability through Cancer Chemistry; 2024 Dec 9-11; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(12_Suppl):Abstract nr IA003">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c5b7589059a6134690d03df4a91c9f05b4284ea" target='_blank'>
              Abstract IA003: Synthetic lethality: Pathways to therapeutic discovery
              </a>
            </td>
          <td>
            Daniel Durocher
          </td>
          <td>2024-12-09</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/033d9d641f5cc9ef741b9f061b7709ec5c750d88" target='_blank'>
              Characterizing the evolutionary dynamics of cancer proliferation in single-cell clones with SPRINTER
              </a>
            </td>
          <td>
            Olivia Lucas, S. Ward, Rija Zaidi, Abigail Bunkum, A. Frankell, David A Moore, M. Hill, Wing Kin Liu, Daniele Marinelli, E. Lim, S. Hessey, C. Naceur-Lombardelli, A. Rowan, Sukhveer Kaur Purewal-Mann, Haoran Zhai, M. Dietzen, Boyue Ding, G. Royle, Sam Aparicio, N. Mcgranahan, M. Jamal-Hanjani, N. Kanu, C. Swanton, S. Zaccaria
          </td>
          <td>2024-11-29</td>
          <td>Nature Genetics</td>
          <td>1</td>
          <td>69</td>
        </tr>

        <tr id="Background Multiple myeloma (MM) is a hematological malignancy characterized by the clonal proliferation of malignant plasma cells within the bone marrow. The disease’s complexity is underpinned by a variety of genetic and molecular abnormalities that drive its progression. Methods This review was conducted through a state-of-The-art literature search, primarily utilizing PubMed to gather peer-reviewed articles. We focused on the most comprehensive and cited studies to ensure a thorough understanding of the genetic and molecular landscapes of MM. Results We detail primary and secondary alterations such as translocations, hyperdiploidy, single nucleotide variants (SNVs), copy number alterations (CNAs), gene fusions, epigenetic modifications, non-coding RNAs, germline predisposing variants, and the influence of the tumor microenvironment (TME). Our analysis highlights the heterogeneity of MM and the challenges it poses in treatment and prognosis, emphasizing the distinction between driver mutations, which actively contribute to oncogenesis, and passenger mutations, which arise due to genomic instability and do not contribute to disease progression. Conclusion & Future Perspectives We report key controversies and challenges in defining the genetic drivers of MM, and examine their implications for future therapeutic strategies. We discuss the importance of systems biology approaches in understanding the dependencies and interactions among these alterations, particularly highlighting the impact of double and triple-hit scenarios on disease outcomes. By advancing our understanding of the molecular drivers and their interactions, this review sets the stage for novel therapeutic targets and strategies, ultimately aiming to improve clinical outcomes in MM patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec433a1270b41378c5571d49631b7d920ddc502b" target='_blank'>
              The Genetic and Molecular Drivers of Multiple Myeloma: Current Insights, Clinical Implications, and the Path Forward
              </a>
            </td>
          <td>
            Meghana Ram, Molly Fraser, Júnia Vieira dos Santos, R. Tasakis, Ariana Islam, Jannah Abo-Donia, Samir Parekh, A. Laganà
          </td>
          <td>2024-12-01</td>
          <td>Pharmacogenomics and Personalized Medicine</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Targeted cancer therapies often induce a reversible drug-tolerant state in subpopulations of cells, akin to bacterial persistence. Precise characterization of these “cancer persisters” may inform the design of more effective treatment strategies. A previous investigation into the transition to persistence of colorectal cancer cell lines has revealed a distinct dependence on drug presence and concentration, not typical of bacterial systems. Leveraging these findings, this study uses mathematical modeling to explore intermittent treatment protocols aimed at diminishing the long-term fitness of the treated population, thereby enhancing therapeutic efficacy. We adapt a mathematical model originally designed for bacteria to describe colorectal cancer population dynamics in response to a series of treatment and release cycles. The model predicts the long-term increase or reduction of a treated population, as well as its asymptotic composition, leading us to identify success and failure regions within a clinically accessible parameter space, also in combination with hypothetical drugs that act on persisters. Strikingly, our analysis suggests, perhaps counter-intuitively, that optimal treatment outcomes may be achieved in correspondence of non-zero recovery periods and lower than currently administered in the clinics drug concentrations. Furthermore, by incorporating patient drug pharmacokinetics in the model, we demonstrate that intermittent dosing strategies currently explored in clinical trials can be optimized to potentially rival the efficacy of continuous dosing regimens. These findings underscore the potential of mathematical models in guiding the design of optimal treatment protocols by fine-tuning non-trivial decisional trade-offs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c41c142151d9a1f87899953c865fcf3fb7c41366" target='_blank'>
              Optimal treatment for drug-induced cancer persisters involves release periods and intermediate drug doses
              </a>
            </td>
          <td>
            Mattia Corigliano, Arianna Di Bernardo, M. Cosentino Lagomarsino, Simone Pompei
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f43e9b27ccadb97b1a4a5f9973cf63bf20b5294" target='_blank'>
              Single-cell RNA-sequencing and genome-wide Mendelian randomisation along with abundant machine learning methods identify a novel B cells signature in gastric cancer
              </a>
            </td>
          <td>
            Qi Ma, Jie Gao, Yuan Hui, Zhi-Ming Zhang, Yu-Jie Qiao, Bin-Feng Yang, Ting Gong, Duo-Ming Zhao, Bang-Rong Huang
          </td>
          <td>2025-01-06</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="In this review, we explore the application of next-generation sequencing in liver cancer research, highlighting its potential in modern oncology. Liver cancer, particularly hepatocellular carcinoma, is driven by a complex interplay of genetic, epigenetic, and environmental factors. Key genetic alterations, such as mutations in TERT , TP53 , and CTNNB1 , alongside epigenetic modifications such as DNA methylation and histone remodeling, disrupt regulatory pathways and promote tumorigenesis. Environmental factors, including viral infections, alcohol consumption, and metabolic disorders such as nonalcoholic fatty liver disease, enhance hepatocarcinogenesis. The tumor microenvironment plays a pivotal role in liver cancer progression and therapy resistance, with immune cell infiltration, fibrosis, and angiogenesis supporting cancer cell survival. Advances in immune checkpoint inhibitors and chimeric antigen receptor T-cell therapies have shown potential, but the unique immunosuppressive milieu in liver cancer presents challenges. Dysregulation in pathways such as Wnt/β-catenin underscores the need for targeted therapeutic strategies. Next-generation sequencing is accelerating the identification of genetic and epigenetic alterations, enabling more precise diagnosis and personalized treatment plans. A deeper understanding of these molecular mechanisms is essential for advancing early detection and developing effective therapies against liver cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf941fd49f418f7109e7e4bbca5152a8ddaa54e7" target='_blank'>
              Advances and challenges in molecular understanding, early detection, and targeted treatment of liver cancer
              </a>
            </td>
          <td>
            Ji Shi, Xu Zhu, Jun-Bo Yang
          </td>
          <td>2025-01-27</td>
          <td>World Journal of Hepatology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58506d08da7a5b25a950b60c249aaba7b6397bbd" target='_blank'>
              Neutral drift upon threshold-like selection promotes variation in antibiotic resistance phenotype
              </a>
            </td>
          <td>
            A. N. Erdoğan, P. Dasmeh, Raymond D. Socha, Johnson Chen, Ben Life, Rachel Jun, Linda Kiritchkov, Dan Kehila, Adrian W. R. Serohijos, N. Tokuriki
          </td>
          <td>2024-12-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edfaace030e74345b79142b5ae90516ba4841735" target='_blank'>
              High-throughput sequencing: a breakthrough in molecular diagnosis for precision medicine.
              </a>
            </td>
          <td>
            Dipali Dongare, Shaik Shireen Nishad, Sakshi Y Mastoli, S. Saraf, Nidhi Srivastava, Abhishek Dey
          </td>
          <td>2025-01-22</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7f6cf05d6f649a55a948cbad9eac9ac4b6467ec" target='_blank'>
              Modern biology of extrachromosomal DNA: A decade-long voyage of discovery
              </a>
            </td>
          <td>
            Qing-Lin Yang, Yipeng Xie, Kailiang Qiao, Jun Yi Stanley Lim, Sihan Wu
          </td>
          <td>2025-01-01</td>
          <td>Cell Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c5a9f2f7d6e562bbd4a865f9dd5dc95d52e44f1" target='_blank'>
              Transformer-based modeling of Clonal Selection and Expression Dynamics reveals resistance mechanisms in breast cancer
              </a>
            </td>
          <td>
            Nathan D Maulding, Jun Zou, Wei Zhou, C. Metcalfe, Josh Stuart, Xin Ye, M. Hafner
          </td>
          <td>2025-01-10</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0297441e9d4a24ced84d1909df04d7019094986" target='_blank'>
              Functional screening reveals genetic dependencies and diverging cell cycle control in atypical teratoid rhabdoid tumors
              </a>
            </td>
          <td>
            Daniel J. Merk, F. Tsiami, S. Hirsch, B. Walter, L. Haeusser, Jens D. Maile, Aaron Stahl, M. Jarboui, Anna Lechado-Terradas, Franziska Klose, S. Babaei, Jakob Admard, N. Casadei, C. Roggia, Michael Spohn, J. Schittenhelm, Stephan Singer, U. Schüller, F. Piccioni, N. Persky, Manfred Claassen, M. Tatagiba, Philipp J. Kahle, D. Root, M. Templin, Ghazaleh Tabatabai
          </td>
          <td>2024-12-02</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02c600bc41b490f1030b42efdd32168810e5ead4" target='_blank'>
              The dynamics of loss of heterozygosity events in genomes.
              </a>
            </td>
          <td>
            Abhishek Dutta, J. Schacherer
          </td>
          <td>2025-01-02</td>
          <td>EMBO reports</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Oncogenes are typically overexpressed in tumor tissues and often linked to poor prognosis. However, recent advancements in bioinformatics have revealed that many highly expressed genes in tumors are associated with better patient outcomes. These genes, which act as tumor suppressors, are referred to as “paradoxical genes.” Analyzing The Cancer Genome Atlas (TCGA) confirmed the widespread presence of paradoxical genes, and KEGG analysis revealed their role in regulating tumor metabolism. Mechanistically, discrepancies between gene and protein expression-affected by pre- and post-transcriptional modifications-may drive this phenomenon. Mechanisms like upstream open reading frames and alternative splicing contribute to these inconsistencies. Many paradoxical genes modulate the tumor immune microenvironment, exerting tumor-suppressive effects. Further analysis shows that the stage- and tumor-specific expression of these genes, along with their environmental sensitivity, influence their dual roles in various signaling pathways. These findings highlight the importance of paradoxical genes in resisting tumor progression and maintaining cellular homeostasis, offering new avenues for targeted cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3eab8741264f274ad63584e4f394278cc1295e34" target='_blank'>
              Discovery of paradoxical genes: reevaluating the prognostic impact of overexpressed genes in cancer
              </a>
            </td>
          <td>
            Dequan Liu, Lei Liu, Xiangyu Che, Guangzhen Wu
          </td>
          <td>2025-01-22</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Bacterial genomic mutations in Staphylococcus aureus (S. aureus) have been detected in isolated resistant clinical strains, yet their mechanistic effect on the development of antimicrobial resistance remains unclear. The resistance-associated regulatory systems acquire adaptive mutations under stress conditions that may lead to a gain of function effect and contribute to the resistance phenotype. Here, we investigate the effect of a single-point mutation (T331I) in VraS histidine kinase, part of the VraSR two-component system in S. aureus. VraSR senses and responds to environmental stress signals by upregulating gene expression for cell wall synthesis. A combination of enzyme kinetics, microbiological, and transcriptomic analysis revealed the mechanistic effect of the mutation on VraS and S. aureus. Michaelis Menten’s kinetics show that the VraS mutation caused an increase in the autophosphorylation rate of VraS and enhanced its catalytic efficiency. The introduction of the mutation through recombineering coupled with CRISPR-Cas9 counterselection to the Newman strain wild-type (WT) genome doubled the minimum inhibitory concentration of three cell wall-targeting antibiotics. The mutation caused an enhanced S. aureus growth rate at sub-lethal doses of the antibiotics, confirming the causative effect of mutation on bacterial persistence. Transcriptomic analysis showed a genome-wide alteration in gene expression levels and protein-protein interaction network of the mutant compared to the WT strain after exposure to vancomycin. The results suggest that vraS mutation causes several mechanistic changes at the protein and cellular levels that favor bacterial survival under antibiotic stress and cause the mutation-harboring strains to become the dominant population during infection. Importance Rising antimicrobial resistance (AMR) is a global health problem. Mutations in the two- component system have been linked to drug- resistance in Staphylococcus aureus, yet the exact mechanism through which these mutations work is understudied. We investigated the T331I mutation in the vraS gene linked to sensing and responding to cell wall stress. The mutation caused changes at the protein level by increasing the catalytic efficiency of VraS kinase activity. Introducing the mutation to the genome of an S. aureus strain resulted in changes in the phenotypic antibiotic susceptibility, growth kinetics, and genome-wide transcriptomic alterations. By a combination of enzyme kinetics, microbiological, and transcriptomic approaches, we highlight how small genetic changes can significantly impact bacterial physiology and survival under antibiotic stress. Understanding the mechanistic basis of antibiotic resistance is crucial to guide the development of novel therapeutic agents to combat AMR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f0ce015d74667f80681dd7299da3e08caeb3120" target='_blank'>
              A mechanistic understanding of the effect of Staphylococcus aureus VraS histidine kinase single point mutation on antibiotic resistance
              </a>
            </td>
          <td>
            Liaqat Ali, Salima Karki, G. Boorgula, Amir Mekakda, Brittnee Cagle-White, Shrijan Bhattarai, Robert Beaudoin, Aryanna Blakeney, Sanjay Singh, S. Srivastava, May H. Abdelaziz
          </td>
          <td>2025-01-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Plants host a range of DNA elements capable of self-replication. These molecules, usually associated to the activity of transposable elements or viruses, are found integrated in the genome or in the form of extrachromosomal DNA. The activity of these elements can impact genome plasticity by a variety of mechanisms, including the generation of structural variants, the shuffling of regulatory or coding DNA sequences across the genome, and DNA endoduplication. This plasticity can dynamically alter gene expression and genome stability, ultimately affecting plant development or the response to environmental changes. While the activation of these elements is often considered deleterious to the genome, their role in creating variation is important in adaptation and evolution. Moreover, the mechanisms by which mobile DNA proliferate have been exploited for plant engineering, or contributed to understand how desirable traits can be generated in crops. In this review, we discuss the origins and the roles of mobile DNA element activity on genome plasticity and plant biology, as well as their potential function and current application in plant biotechnology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b762e6df52387f3493832fc69614173777627231" target='_blank'>
              The role of mobile DNA elements in the dynamic of plants genome plasticity.
              </a>
            </td>
          <td>
            Robyn Emmerson, M. Catoni
          </td>
          <td>2025-01-10</td>
          <td>Journal of experimental botany</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The study by Grossmann and colleagues uses single-nucleus RNA sequencing in a cohort of matched high-risk neuroblastoma primary tumor samples, obtained from the same patient at diagnosis and definitive surgery, to identify persister cells that survive induction chemotherapy. These persister cells utilize mechanisms of chemoresistance that are both tumor-intrinsic and tumor-extrinsic, are highly dependent on the original genetic profile of the tumor, and represent novel, patient-specific targets to precisely inhibit chemoresistance and disease recurrence. See related article by Grossmann et.al., p. 2387.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17960d164122a139c932c5adc28d8d07f55f0399" target='_blank'>
              Characterization of Persister Cells Provides Insights into Mechanisms of Therapy Resistance in Neuroblastoma.
              </a>
            </td>
          <td>
            Amber B. Wolf, C. P. Reynolds, Eveline Barbieri
          </td>
          <td>2024-12-02</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Human cancers are heterogeneous. Their genomes evolve from genetically diverse germlines in complex and dynamic environments, including exposure to potential carcinogens. This heterogeneity of humans, our environmental exposures, and subsequent tumours makes it challenging to understand the extent to which cancer evolution is predictable. Addressing this limitation, we re-ran early tumour evolution hundreds of times in diverse, inbred mouse strains, capturing genetic variation comparable to and beyond that found in human populations. The sex, environment, and carcinogenic exposures were all controlled and tumours comprehensively profiled with whole genome and transcriptome sequencing. Within a strain, there was a high degree of consistency in the mutational landscape, a limited range of driver mutations, and all strains converged on the acquisition of a MAPK activating mutation with similar transcriptional disruption of that pathway. Despite these similarities in the phenotypic state of tumours, different strains took markedly divergent paths to reach that state. This included pronounced biases in the precise driver mutations, the strain specific occurrence of whole genome duplication, and differences in subclonal selection that reflected both cancer susceptibility and tumour growth rate. These results show that interactions between the germline genome and the environment are highly deterministic for the trajectory of tumour genome evolution, and even modest genetic divergence can substantially alter selection pressures during cancer development, influencing both cancer risk and the biology of the tumour that develops.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/caebf07ac0a60cd920fb19088ba70b1c3f57d1ba" target='_blank'>
              Genetic background sets the trajectory of cancer evolution
              </a>
            </td>
          <td>
            Sarah J. Aitken, Frances Connor, Christine Feig, Tim F. Rayner, M. Lukk, Juliet Luft, Stuart Aitken, Claudia Arnedo-Pac, James F. Hayes, Michael D. Nicholson, Vasavi Sundaram, Jan C. Verburg, John Connelly, Craig J. Anderson, Mikaela Behm, Susan Campbell, Maëlle Daunesse, Ailith Ewing, Vera B. Kaiser, Elissavet Kentepozidou, O. Pich, A. Redmond, Javier Santoyo-Lopez, Inés Sentís, Lana Talmane, Paul Flicek, Núria López-Bigas, Colin A. Semple, Martin S. Taylor, D. T. Odom
          </td>
          <td>2025-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="Abstract Cancer research has historically focused on the somatic mutation theory, viewing cancer as a consequence of genetic mutations. However, this perspective has limitations in explaining phenomena like tumor reversion and cancer heterogeneity. This paper introduces an alternative approach: viewing cancer as a complex information-processing system shaped by its microenvironment. By integrating historical data on tumor reversion and insights into evolutionary dynamics, I propose a reframing of cancer biology. This process-oriented perspective highlights the role of cellular plasticity and adaptive behaviors, offering new pathways for therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d33c3cc92fad268ef72893a64fd8d179664753cb" target='_blank'>
              Rethinking cancer evolution: from genetic mutations to complex information systems in tumor reversion
              </a>
            </td>
          <td>
            M. Tez
          </td>
          <td>2024-12-13</td>
          <td>Evolution, Medicine, and Public Health</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Mitosis is a critical phase of the cell cycle and a vulnerable point where cancer cells can be effectively disrupted, leading to cell death and inhibition of tumor growth. However, challenges such as drug resistance remain significant in clinical applications. During mitosis, mRNA translation is generally downregulated, while non-canonical translation of specific transcripts proceeds. Here, we demonstrate that mitotic cancer cells redistribute ribosomes toward the 5’ untranslated region (5’ UTR) and the start of the coding sequence (CDS), enhancing the translation of thousands of upstream open reading frames (uORFs) and upstream overlapping open reading frames (uoORFs). This mitotic induction of uORF/uoORF enriches the presentation of immunopeptides at the surface of cancer cells following treatment with mitotic inhibitors. Functional assays indicate the potential of such neoepitopes to provoke cancer-cell killing by T cells. Altogether, our findings highlight the therapeutic potential of targeting uORF/uoORF-derived neoepitopes in combination with mitotic inhibitors to enhance immune recognition and tumor cell elimination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e58b9a8310537bb25bde77518c59ca5cd45e675f" target='_blank'>
              Upstream open reading frame translation enhances immunogenic peptide presentation in mitotically arrested cancer cells
              </a>
            </td>
          <td>
            Alexander Kowar, Jonas P. Becker, Rossella Del Pizzo, Zhiwei Tang, Julien Champagne, Pierré-René Körner, Jasmine Montenegro Navarro, Fiona Megan Tilghman, Hanan Sakeer, Angelika B. Riemer, R. Agami, Fabricio Loayza-Puch
          </td>
          <td>2025-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Mutations with a synergistic effect on tumorigenesis were positively selected during tumor evolution, showing a tendency to co-occur in cancer genomes. To systematically gain insight of tumorigenic crosstalk at the mutation level, we utilized the FP-Growth algorithm in frequent pattern mining to create an unbiased map of co-occurring mutations from clinical genome sequencing data. We identified 100,933 frequent co-occurring mutation pairs across 22 cancer types. The co-mutation pairs involving cancer driver genes highlighted potential mechanisms for collaborative tumor formation, suggesting promising targets to disrupt tumorigenesis. Additionally, the large number of intra-gene co-mutations with strong dependencies indicated possible combinational effects on the oncogenic properties of specific proteins, which might be overlooked when individual mutations are not hotspots. Furthermore, our whole-genome mining strategy revealed a significant number of co-mutations in non-coding regions, with an enrichment of regulatory elements suggesting their potential role in modulating the co-expression of related genes. Finally, we demonstrated that our pipeline could systematically identify high-order co-mutations. In summary, our study provides the most extensive co-occurring mutations in tumor development, offering valuable insights into mechanisms and potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10963d53c10fd0cd38d8c059ea7d0809367afa03" target='_blank'>
              Unbiased co-occurring mutational landscapes in tumorigenesis
              </a>
            </td>
          <td>
            Yizhou Li, Yangfang Xiong, Wensheng Wei
          </td>
          <td>2025-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Introduction Esophageal squamous cell carcinoma (ESCC) accounts for 80% of esophageal cancer (EC) worldwide. The molecular characteristics of locally advanced ESCC have been extensively studied. Methods In this study, we investigate the genomic and transcriptomic characteristics and try to provide the basic T-cell receptors (TCRs) dynamics and its genomic and transcriptome association during the radiochemotherapy of ESCC using multi-omics analysis. Results A total of 23 patients with pathologic diagnoses of locally advanced ESCC were enrolled. The median tumor mutational burden (TMB) of the 23 ESCC patients were 3.47 mutations/ Mb (mega-base). The TP53, RTK/RAS, and NOTCH pathways were concurrently prevalent in ESCC. Besides, some less prevalent pathways, including WNT and HIPPO pathways also exhibited superior frequencies in ESCC. Meantime, we found the immune-hot tumor had higher immune infiltration scores. The median TMB in the progression-free survival (PFS) low group was significantly higher than that in the PFS-high group. The chromosomal copy number variation (CNV) burden of the neutrophil-to-lymphocyte ratio (NLR)-high group appeared to be higher than that of the NLR-low group, and the StromalScore in the NLR-low group was significantly higher. Clonality score was significantly increased from pre-treat to post-treat and from on-treat to post-treat. Shannon index was significantly decreased from pre-treat to post-treat and from on-treat to posttreat. Richness was significantly decreased from pre-treat to post-treat. Discussion Multiomics analysis provided the basic TCRs dynamics and their genomic and transcriptome association during the radio-chemotherapy of 23 locally advanced ESCC in China, and provided a valuable insights into the heterogeneity and the tumor microenvironment and treatment responses. Meantimes, the identification of biomarkers and the exploration of their association with treatment outcomes could have important implications for clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2295b8708f9d5a68cac1e6b06cb1f22ce1ffd3b8" target='_blank'>
              Genomic, transcriptomic, and T cell receptor profiling in stratifying response to first-line chemoradiotherapy or radiotherapy for esophageal squamous cell carcinoma
              </a>
            </td>
          <td>
            Xiaqin Zhang, Jianhong Lian, Fukun Chen, Kai Wang, Haoyuan Xue, S. Jia, Weili Wang, Zhongkang Li, Hua Liang, Hongwei Li
          </td>
          <td>2025-01-06</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="BACKGROUND
The majority of patients with prostate cancer (PCa) exhibit intrinsic resistance to immune checkpoint blockade (ICB) following radiotherapy (RT). This resistance is generally attributed to the limited antigen presentation of heterogeneous cells within tumors. Here, we aimed to isolate and characterize these diverse subgroups of tumor post-RT to understand the molecular mechanisms of their resistance to ICB.


METHODS
Single-cell RNA-sequencing (scRNA-seq) was used to profile senescent cancer cell clusters induced by RT in LNCaP cells. The expression and phosphorylation levels of ataxia telangiectasia and Rad3-related protein (ATR) were assessed by immunohistochemistry in clinical samples from patients with or without RT. Co-immunoprecipitation, mutagenesis, and Western blotting were used to measure the interactions between proteins. Xenograft experiments were performed to assess the tumor immune response in the mice.


RESULTS
We identified a subset of PCa cells that exhibited resistance to RT, characterized by a reduced antigen presentation capability, which enhanced their ability to evade immune detection and resist cytotoxic T-lymphocyte-associated protein 4 (CTLA-4) blockade. scRNA-seq revealed that the senescent state was a transient phase of PCa cells post-RT, particularly in CTLA-4 blockade treatment-resistant cells. This state was marked by increased cytosolic ATR level. Cytosolic ATR phosphorylated CD86 in its cytosolic domain and enhanced the interaction between CD86 and its E3 ligase MARCH1 through electrostatic attraction. Depletion or inhibition of Atr increased the sensitivity to immune attack and improved responses to anti-Ctla-4 antibody treatment in a mouse model.


CONCLUSIONS
Our findings indicate that the activation of cytosolic ATR, which is associated with cellular senescence, impedes the effectiveness of combined RT and ICB treatments. This discovery may provide valuable insights for improving the efficacy of combined RT and ICB therapies in PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54111804377a9c0bbc1679bc34e9106d5b8ef9e5" target='_blank'>
              Radiotherapy-resistant prostate cancer cells escape immune checkpoint blockade through the senescence-related ataxia telangiectasia and Rad3-related protein.
              </a>
            </td>
          <td>
            Chenyi Shao, Yingyi Zhang, Hang Li, Jiajia Chen, Ting Huang, Jiaze Li, Simeng Wen, Sen Wang, Saijun Fan, Yu Zhao
          </td>
          <td>2024-12-19</td>
          <td>Cancer communications</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Fungal genome sizes exhibit more than a 100-fold variation, largely driven by the expansion of repetitive sequences such as transposable elements (TEs). Genomic defenses have evolved to counteract TE proliferation through silencing mechanisms operating at the epigenetic or transcript level. In fungi, repeat-induced point mutation (RIP) targets TEs by recognizing repetitive sequences and inducing mutagenesis. However, the prevalence of RIP across the fungal kingdom and the fidelity of the canonical C-to-T mutation signatures remain unclear. In this study, we address these gaps by tracking shifts in genome architecture across the fungal kingdom. We show that the loss of RIP components is associated with an 80% (∼30 Mb) increase in genome size in ascomycetes (class of Leotiomycetes). To track the impact of genome defenses, we designed a quantitative screen for RIP-like mutation signatures. The phylum of ascomycetes was unique in showing enrichment in mutation signatures in non-coding and repetitive sequences, consistent with a phylogenetically restricted occurrence of RIP-like genome defense systems. Then, we gene functions associated in a phylogenetic context with RIP-like mutation signatures. We identified a zinc-finger protein as the strongest candidate underpinning a novel mechanism of genome defenses. Our findings reveal the multifaceted drivers of genome defense systems and their close ties to genome size evolution in fungi, highlighting how proximate molecular mechanisms can shape genome evolution on deep phylogenetic scales.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6148de5b7f0cf13898d5f28034ac9ab633d52a0" target='_blank'>
              A systematic screen for genetic factors underpinning transposon defense systems across the fungal kingdom
              </a>
            </td>
          <td>
            Thomas Badet, D. Croll
          </td>
          <td>2025-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is an aggressive malignancy that poses significant challenges due to high rates of relapse and resistance to treatment, particularly in older populations. While therapeutic advances have been made, survival outcomes remain suboptimal. The evolution of DNA and RNA sequencing technologies, including whole-genome sequencing (WGS), whole-exome sequencing (WES), and RNA sequencing (RNA-Seq), has significantly enhanced our understanding of AML at the molecular level. These technologies have led to the discovery of driver mutations and transcriptomic alterations critical for improving diagnosis, prognosis, and personalized therapy development. Furthermore, single-cell RNA sequencing (scRNA-Seq) has uncovered rare subpopulations of leukemia stem cells (LSCs) contributing to disease progression and relapse. However, widespread clinical integration of these tools remains limited by costs, data complexity, and ethical challenges. This review explores recent advancements in DNA/RNA sequencing in AML and highlights both the potential and limitations of these techniques in clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51e5d43f554c8fe98964b8130a7d1804ae561d5a" target='_blank'>
              Advances in DNA/RNA Sequencing and Their Applications in Acute Myeloid Leukemia (AML)
              </a>
            </td>
          <td>
            Fatimah Ahmed, Jiang Zhong
          </td>
          <td>2024-12-25</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Proteins that emerge de novo from noncoding DNA could negatively or positively influence cellular physiology in the sense of providing a possible adaptive advantage. Here, we employ two approaches to study such effects in a human cell line by expressing random sequences and mouse de novo genes that lack homologs in the human genome. We show that both approaches lead to differential growth effects of the cell clones dependent on the sequences they express. For the random sequences, 53% of the clones decreased in frequency, and about 8% increased in frequency in a joint growth experiment. Of the 14 mouse de novo genes tested in a similar joint growth experiment, 10 decreased, and 3 increased in frequency. When individually analysed, each mouse de novo gene triggers a unique transcriptomic response in the human cells, indicating mostly specific rather than generalized effects. Structural analysis of the de novo gene open reading frames (ORFs) reveals a range of intrinsic disorder scores and/or foldability into alpha-helices or beta sheets, but these do not correlate with their effects on the growth of the cells. Our results indicate that de novo evolved ORFs could easily become integrated into cellular regulatory pathways, since most interact with components of these pathways and could therefore become directly subject to positive selection if the general conditions allow this.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43878c4781a378a056438c19941008a1a26fd3df" target='_blank'>
              Expression of Random Sequences and de novo Evolved Genes From the Mouse in Human Cells Reveals Functional Diversity and Specificity
              </a>
            </td>
          <td>
            Silvia Aldrovandi, Johana Fajardo Castro, K. Ullrich, Amir Karger, Victor Luria, D. Tautz
          </td>
          <td>2024-12-01</td>
          <td>Genome Biology and Evolution</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4cb54307571359f46baeb0ac619041e8a5efa2f" target='_blank'>
              Integrating single-cell RNA-seq and bulk RNA-seq to construct a neutrophil prognostic model for predicting prognosis and immune response in oral squamous cell carcinoma
              </a>
            </td>
          <td>
            Jinhang Wang, Zifeng Cui, Qiwen Song, Kaicheng Yang, Yanping Chen, Shixiong Peng
          </td>
          <td>2024-12-26</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background In the present day, hepatocellular carcinoma (HCC) remains a formidable threat to human health. Actin-related protein 10 (ACTR10) is related to tyrosine kinase inhibitor (TKI) resistance. A comprehensive analysis of ACTR10 in HCC will further our understanding of the molecular mechanisms underlying this resistance phenomenon, shedding light on potential therapeutic strategies for combating TKI resistance in HCC. Methods We conducted an integration of high-throughput datasets across various centers, analyzing ACTR10 expression using the Cancer Cell Line Encyclopedia (CCLE) and assessing its implications through clustered regularly interspaced short palindromic repeats (CRISPR) knockout screen. Pathogenic mechanisms were elucidated through enrichment analysis. Prognostic assessment utilized Kaplan-Meier survival and univariate Cox analyses. An integrated analysis of gene expression profiles related to TKI in HCC was conducted, and TKI resistance mechanisms were explored through enrichment analysis. Potential therapeutic drugs were identified using the Drug Gene Budger database and molecular docking techniques. Results The standardized mean difference (SMD) of 0.34 (95% confidence interval (CI): 0.22 - 0.45, P < 0.05) and ACTR10-dependent growth in HCC cells confirm its upregulation in HCC. The area under the summary receiver operating characteristic (sROC) curve was 0.69, indicating moderate discriminative ability of ACTR10 in HCC patients. ACTR10 exerts its pro-cancer effect by influencing RNA splicing, mRNA processing and nucleocytoplasmic transport. A hazard ratio of 2.19 (95% CI: 1.56 - 3.08, P < 0.05) identifies ACTR10 as an independent prognostic risk factor. Additionally, the SMD of 0.88 (95% CI: 0.01 - 0.76, P < 0.05) validates ACTR10 as a TKI-resistance gene, mediating resistance via enhanced exocytosis, autophagy, and apoptosis in HCC patients. Trichostatin A emerges as a prospective targeted agent for HCC. Conclusion The upregulation of ACTR10 accelerates HCC progression, promotes TKI resistance, and emerges as a prospective target for the treatment of HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7267df6c8eaa54e7c4164cc28fdee3a3c18889b4" target='_blank'>
              ACTR10 Overexpression Facilitates the Progression and Tyrosine Kinase Inhibitor Resistance in Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Jie Luo, Kai Qin, Rong-Quan He, Jian Di Li, Zhi-guang Huang, Bin-Tong Yin, Tong Wu, Y. Chen, Di-Yuan Qin, Jia-Yuan Luo, Mei Wu, Bang-Teng Chi, Gang Chen, Jian Jun Li, Y. Huang
          </td>
          <td>2024-12-01</td>
          <td>World Journal of Oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a321392f82837178158946451c3a5d2c2f02daa" target='_blank'>
              Genomic and transcriptomic profiling of radioresistant prostate and head and neck cancers implicate a BAHD1-dependent modification of DNA damage at the heterochromatin
              </a>
            </td>
          <td>
            Chaw Yee Beh, Celestia P X Yeo, Boon Hao Hong, E. Tan, Kah M Tan, Dennis J J Poon, Pek L Chu, Dewi Susanti, Pei Ling Tai, Monica Ryu, James Proudfoot, E. Yeo, Khee Chee Soo, Melvin L. K. Chua
          </td>
          <td>2024-12-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3869199f12165ae05c494a5d6b43dbd887168390" target='_blank'>
              Germline structural variant as the cause of Lynch Syndrome in a family from Ecuador
              </a>
            </td>
          <td>
            Gemma Llargués-Sistac, L. Bonjoch, J. Muñoz, Xavier Domínguez-Rovira, T. Ocaña, M. I. Álvarez-Mora, Celia Badenas, Anna Esteve-Codina, Carlos Reyes-Silva, Gabriela Jaramillo-Koupermann, María Teresa Rodrigo, Sandra López-Prades, M. Cuatrecasas, Antoni Castells, F. Balaguer, L. Moreira, Guerau Fernández, S. Castellví-Bel
          </td>
          <td>2025-01-17</td>
          <td>NPJ Genomic Medicine</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5889adf5def7b284cf9f7d22530eab68401de07f" target='_blank'>
              PlexinD1 is a driver and a therapeutic target in advanced prostate cancer.
              </a>
            </td>
          <td>
            Jing Wei, Jing Wang, Wen Guan, Jingjing Li, Tianjie Pu, E. Corey, Tzu-Ping Lin, Allen C Gao, Boyang Jason Wu
          </td>
          <td>2025-01-02</td>
          <td>EMBO molecular medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Oncogenesis and tumor progression are shaped by somatic alterations in the cancer genome and features of the tumor immune microenvironment (TME). How interactions of these two systems influence tumor development and clinical outcomes remains incompletely understood. To address this challenge, we developed the multi-omics analysis framework PACIFIC to systematically integrate genetic cancer drivers and infiltration profiles of immune cells with clinical information. In an analysis of 8500 cancer samples, we report 34 immunogenomic interactions (IGXs) in 13 cancer types in which context-specific combinations of genomic alterations and immune cell activities associate with disease outcomes. Risk associations of IGXs are potentially explained by tumor-intrinsic and microenvironmental metrics of immunogenicity and differential expression of therapeutic targets. In luminal-A breast cancer, MEN1 deletion combined with reduced neutrophils is associated with poor prognosis and deregulation of immune signalling pathways. These findings help elucidate how cancer drivers interact with TME to contribute to tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7541ff011e197e5ec1cc98d2bd31291176ffd633" target='_blank'>
              Combinations of genomic alterations and immune microenvironmental features associate with patient survival in multiple cancer types
              </a>
            </td>
          <td>
            Masroor Bayati, Zoe P. Klein, Alec Bahcheli, Mykhaylo Slobodyanyuk, Jeffrey To, Kevin C. L. Cheng, Jigyansa Mishra, Diogo Pellegrina, Kissy Guevara-Hoyer, Chris McIntosh, Mamatha Bhat, J. Reimand
          </td>
          <td>2024-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5cae0c3f9678085fee30003f0aae12e5e2f38618" target='_blank'>
              Multiple myeloma long-term survivors exhibit sustained immune alterations decades after first-line therapy
              </a>
            </td>
          <td>
            R. Lutz, F. Grünschläger, Malte Simon, M. Awwad, M. Bauer, S. Yousefian, N. Beumer, L. Jopp-Saile, Anastasia Sedlmeier, Llorenç Solé-Boldo, Bogdan Avanesyan, Dominik Vonficht, Patrick Stelmach, Georg Steinbuss, T. Boch, Simon Steiger, M. Baertsch, N. Prokoph, K. Rippe, B. Durie, Claudia Wickenhauser, A. Trumpp, Carsten Müller-Tidow, D. Huebschmann, Niels Weinhold, M. Raab, B. Brors, Hartmut Goldschmidt, Charles D. Imbusch, M. Hundemer, Simon Haas
          </td>
          <td>2024-11-29</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="While cancer has traditionally been considered a genetic disease, mounting evidence indicates an important role for non-genetic (epigenetic) mechanisms. Common anti-cancer drugs have recently been observed to induce the adoption of reversible drug-tolerant cell states, thereby accelerating the evolution of drug resistance. Determining how to optimally balance the competing goals of killing the tumor bulk and delaying resistance evolution in this scenario is a nontrivial question of high clinical importance. In this work, we use a combined mathematical and computational approach to study optimal dosing of anti-cancer drug treatment under drug-induced cell plasticity. Our results show that the optimal treatment steers the tumor into a fixed equilibrium composition while balancing the trade-off between cell kill and tolerance induction in a precisely quantifiable way. Under linear induction of tolerance, a low-dose constant strategy is optimal in equilibrium, while under uniform induction of tolerance, alternating between a large dose and no dose is best. The directionality of drug induction, whether the drug elevates transitions from sensitivity to tolerance or inhibits transitions back, significantly affects optimal dosing. To demonstrate the applicability of our approach, we use it to identify an optimal low-dose strategy for colorectal cancer using publicly available in vitro data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/586103632d689f330d569f057f914a090bf47bb0" target='_blank'>
              Optimal dosing of anti-cancer treatment under drug-induced plasticity
              </a>
            </td>
          <td>
            E. B. Gunnarsson, Benedikt Vilji Magn'usson, J. Foo
          </td>
          <td>2024-12-20</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The high-throughput sequencing (HTS) is now a highly favoured technology in the field of genome research. A distinctive feature of this sequenc-ing method is its data-yielding capability, which is capable of generating more than 100 times than the first-generation Sanger sequencing platform. HTS technology has been widely adopted for its advantages, including high through-put, sensitivity, automaticity, information density and cost-effectiveness. Not only does it help in the treatment and diagnosis of multiple diseases, but it also provides new insights into the research in molecular biology of tumors. Moreover, circulating tumor DNA (ctDNA) tests based on HTS technology are increasingly extensively implemented for clinical purposes. In this review, we will focus on the significant achievements and performances of the HTS, and first-hand data from extensive experience will be summarized and analyzed to discuss the advantages and specifics associated with each sequencing system and further summarize the characteristics of their clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fde5c1e1a7571ad4e771c075ea269c28c6ce755b" target='_blank'>
              High throughput sequencing technology and its clinical application in circulating tumor DNA detection in patients with tumors.
              </a>
            </td>
          <td>
            Chonghe Xu, D. Zhou, Mei Zhu
          </td>
          <td>2024-12-03</td>
          <td>Investigación Clínica</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8158e3a828dce6018501fd1d5df759ffab0ffaa" target='_blank'>
              Integration of bulk and single-cell RNA-seq reveals prognostic gene signatures in patients with bladder cancer treated with immune checkpoint inhibitors
              </a>
            </td>
          <td>
            Mina Cho, Hyun Chang, Ju Han Kim
          </td>
          <td>2024-12-21</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The discovery of oncogenes and tumor suppressor genes led to a better understanding of tumorigenesis, and prompted the development of molecularly targeted therapy. Over the past 30 years, many new drugs, which are primarily aimed at activated oncogenic proteins in signal transduction pathways involved in cell proliferation and survival, have been introduced in the clinic. Despite its rational design, the overall efficacy of targeted therapy has been modest. Recently, the noncoding RNAs (ncRNAs) have emerged as key regulators of important cellular processes in addition to the known regulatory proteins. It now appears that dual epigenetic regulatory systems exist in higher eukaryotic cells: a ncRNA network that governs essential cell functions, like cell fate decision and maintenance of homeostasis, and a protein-based system that presides over core physiological processes, like cell division and genomic maintenance. Modifications of the ncRNA network due to altered ncRNAs can cause the cell to shift towards to neoplastic phenotype; this is cancer initiation. Mutations in the well-known cancer driver genes provide the incipient cancer cell with a selective growth advantage and fuel its consequent clonal expansion. Because of the crucial role of the altered ncRNAs in tumorigenesis, targeting them may be a reasonable therapeutic strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d4bbdeb17c896767d39cab33f2cc2532e16bb98" target='_blank'>
              Implications of noncoding RNAs for cancer therapy: Are we aiming at the right targets?
              </a>
            </td>
          <td>
            Amil Shah
          </td>
          <td>2025-01-16</td>
          <td>Exploration of Targeted Anti-tumor Therapy</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Pseudomonas aeruginosa is an opportunistic human pathogen causing various severe infections. Understanding genetic mechanisms of its metabolic versatility aids in developing novel antibacterial drugs and therapeutic strategies to address multidrug-resistant P. aeruginosa infections. The metabolism of nucleotides and amino acids contributes to the cycle of two key biological macromolecules in the genetic central dogma. Guanine deaminase (GuaD) catalyzes the deamination of guanine to produce xanthine to maintain the homeostasis of the nucleotide pool, and transporters specific to BCAAs (termed as BraT) import BCAAs to keep its intracellular availability level. However, little is known about the evolution of GuaD and BraT in P. aeruginosa population. Here, two copies turned out to be widespread in P. aeruginosa population for each of GuaD and BraT. The phylogenic analysis demonstrated that GuaD1 and BraB were inherited from the ancestor of Pseudomonas, while GuaD2 and BraZ were additionally acquired via evolutionary events in the ancestors of P. aeruginosa. The functional divergence of two copies was supported by different distribution patterns of dN/dS ratios, divergent expression levels, differentially co-expressed genes, and their functional enrichment modules with few intersections. Besides, some co-expressed genes with known functions are involved in infecting hosts, forming biofilm and resisting antibiotic treatment. Taken together, functional divergence following copy number increase and differentiation of co-expression networks might confer greater metabolic potential to P. aeruginosa, especially in response to host immune responses and antibiotic treatments in clinical settings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7927b5b28ef799e2d95504a663925f20ba1760b6" target='_blank'>
              Evolution of two metabolic genes involved in nucleotide and amino acid metabolism in Pseudomonas aeruginosa
              </a>
            </td>
          <td>
            Yutong Wu, Yuqi Shi, Xiaohui Liang
          </td>
          <td>2024-12-17</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f8a77c179e3e4f5ecf72af9fc5a5da53d8facac" target='_blank'>
              Overview of a comparative analysis of microsatellite instability and standard mismatch repair protein-deficiency tests in a large cancer cohort.
              </a>
            </td>
          <td>
            András Kiss, M. L. Nádorvári, J. Kulka, T. Barbai, E. Raso, István Kenessey, Gábor Lotz, József Tímár
          </td>
          <td>2024-12-04</td>
          <td>Pathologie</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Metabolic rewiring is required for cancer cells to survive in harsh microenvironments and is considered to be a hallmark of cancer. Specific metabolic adaptations are required for a tumor to become invasive and metastatic. Cell division and metabolism are inherently interconnected, and several cell cycle modulators directly regulate metabolism. Here, we report that TBK1, which is a noncanonical IKK kinase with known roles in cell cycle regulation and TLR signaling, affects cellular metabolism in cancer cells. While TBK1 is reported to be overexpressed in several cancers and its enhanced protein level correlates with poor prognosis, the underlying molecular mechanism involved in the tumor-promoting role of TBK1 is not fully understood. In this study, we show a novel role of TBK1 in regulating cancer cell metabolism using combined metabolomics, transcriptomics, and pharmacological approaches. We find that TBK1 mediates the regulation of nucleotide and energy metabolism through aldo-keto reductase B10 (AKRB10) and thymidine phosphorylase (TYMP) genes, suggesting that this TBK1-mediated metabolic rewiring contributes to its oncogenic function. In addition, we find that TBK1 inhibitors can act synergistically with AKRB10 and TYMP inhibitors to reduce cell viability. These findings raise the possibility that combining these inhibitors might be beneficial in combating cancers that show elevated levels of TBK1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a02a736e9562fec8efc444bbd368f2506195967" target='_blank'>
              TBK1 Reprograms Metabolism in Breast Cancer: An Integrated Omics Approach.
              </a>
            </td>
          <td>
            M. Maan, Neha Jaiswal, Min Liu, Harold I Saavedra, S. Chellappan, Mainak Dutta
          </td>
          <td>2024-12-13</td>
          <td>Journal of proteome research</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a278e50ce1c75eb976a4bb58887b3881b23ccb24" target='_blank'>
              Deep CRISPR mutagenesis characterizes the functional diversity of TP53 mutations
              </a>
            </td>
          <td>
            Julianne S Funk, Maria Klimovich, Daniel Drangenstein, Ole Pielhoop, Pascal Hunold, Anna Borowek, Maxim Noeparast, Evangelos Pavlakis, M. Neumann, Dimitrios-Ilias Balourdas, Katharina Kochhan, N. Merle, Imke Bullwinkel, M. Wanzel, Sabrina Elmshäuser, J. Teply-Szymanski, Andrea Nist, Tara Procida, Marek Bartkuhn, Katharina Humpert, M. Mernberger, R. Savai, Thierry Soussi, A. Joerger, Thorsten Stiewe
          </td>
          <td>2025-01-07</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c49e06286441ee42660f276752afffa0be81c30" target='_blank'>
              Dysregulation of GAS5-miRNA-Mediated Signaling Pathways in Cancer Pathobiology: A Comprehensive Exploration of Pathways Influenced by this Axis.
              </a>
            </td>
          <td>
            Enwa Felix Oghenemaro, Abdulrahman Qais Khaleel, J. Rizaev, R. Roopashree, Muath Suliman, Syeda Wajida Kazmi, Ahmed Hjazi, Pranchal Rajput, Yasser Fakri Mustafa, M. Abosaoda
          </td>
          <td>2024-12-24</td>
          <td>Biochemical genetics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6451661e2ad7d9a2a91b73895cceedbfd09e48b4" target='_blank'>
              Mechanisms of Staphylococcus aureus Antibiotics Resistance Revealed by Adaptive Laboratory Evolution.
              </a>
            </td>
          <td>
            Shiwei Ma, Yufan Xu, Juanjuan Ma, Dan Luo, Zixin Huang, Longlong Wang, Weile Xie, Zhen Luo, Huanhuan Zhang, Jijie Jiang, Yaozhong Jin, Jianming Zhang, Jianguo Zhu, Zhe Wang
          </td>
          <td>2025-01-01</td>
          <td>Current microbiology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Simple Summary Adult B-cell acute lymphoblastic leukemia (B-ALL) is characterized by genetic heterogeneity and high relapse rate. Forty-four adult B-ALL patients were studied at both diagnosis and relapse by next-generation sequencing (NGS) in order to identify genetic alteration driving relapse. Four main genetic pathways leading to relapse in adults were identified: IKZF1plus genetic profile, RAS mutations and TP53 alterations in Philadelphia chromosome (Ph)-negative B-ALL and acquisition of ABL1 mutations in Ph-positive patients. In addition, three clonal evolution patterns were identified: the persistent clone trajectory (25%), the expanding clone trajectory (11%) and the therapy-boosted trajectory (48%). Our results reveal the presence of preferential biological pathways leading to relapse in adult B-ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54416c39b5f84bc7f01293f1564700b87fc9be40" target='_blank'>
              Preferential Genetic Pathways Lead to Relapses in Adult B-Cell Acute Lymphoblastic Leukemia
              </a>
            </td>
          <td>
            Josgrey Navas-Acosta, Alberto Hernández-Sánchez, Teresa González, Ángela Villaverde Ramiro, Sandra Santos, Cristina Miguel, J. Ribera, Isabe L Granada, M. Morgades, Ricardo Sánchez, E. Such, S. Barrena, Juana Ciudad, Julio Dávila, Natalia de Las Heras, Alfonso García-de Coca, J. Labrador, J. Queizán, Sandra Martín, Alberto Orfao, J. Ribera, Rocío Benito, J. Hernández-Rivas
          </td>
          <td>2024-12-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="The development of sequencing technologies and their increased accessibility in clinical services and genetic laboratories have considerably accelerated the identification of genetic variants associated with rare diseases (RDs). Among these, Mendelian disorders of the epigenetic machinery (MDEM) are rare monogenic diseases characterized by the presence of mutations in genes encoding epigenetic regulators that play a key role in organismal development and cellular functions. Loss of function of these regulators is expected to lead to epigenetic modifications that profoundly affect genome expression and cellular identity. Disruptions in DNA methylation profiles have been documented in MDEMs, emerging as a useful diagnostic tool. The current challenge is to determine whether and how these epigenomic alterations drive the mechanisms underlying the clinical manifestations in patients suffering from this class of diseases. Studying MDEMs may therefore shed light on the important role of epigenetic information in health and disease, particularly the mechanisms involved in the development and understanding of complex pathologies, such as neurodevelopmental disorders and cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b00b4062f8130453f8f77acda434d081dca0a7cd" target='_blank'>
              [Genetic diseases of epigenetic machinery].
              </a>
            </td>
          <td>
            Maud de Dieuleveult, Guillaume Velasco
          </td>
          <td>2024-12-01</td>
          <td>Medecine sciences : M/S</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d68732e730bf76384988cc8a9370de7f622451ad" target='_blank'>
              Targeting KRAS: from metabolic regulation to cancer treatment
              </a>
            </td>
          <td>
            Yanyan Shi, Huiling Zheng, Tianzhen Wang, Shengpu Zhou, Shiqing Zhao, Mo Li, Baoshan Cao
          </td>
          <td>2025-01-11</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11552fa246e107609d4eac99ffa639f2882fddc5" target='_blank'>
              How to combine multiple tools for the genetic diagnosis work-up of pediatric B-cell acute lymphoblastic leukemia.
              </a>
            </td>
          <td>
            Gloria Hidalgo-Gómez, Bárbara Tazón-Vega, Carlos Palacio, Silvia Saumell, Noemí Martínez-Morgado, Víctor Navarro, Laura Murillo, Pablo Velasco, Thais Murciano, C. Díaz de Heredia, Francesc Bosch, Gemma Armengol, Margarita Ortega
          </td>
          <td>2025-01-23</td>
          <td>Annals of hematology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The present study investigated the involvement of human papillomavirus (HPV)16 and HPV18 in oropharyngeal malignancies in order to understand the oncogenic mechanisms, and to identify biomarkers for early detection and treatment targets. Given the rising incidence of HPV-associated cancer, particularly in India, this holds significance in elucidating the molecular basis of these diseases. Structural validation of HPV16 and 18 oncoproteins E6 and E7 was conducted using computational tools, while gene expression profiles related to oral squamous cell carcinoma (OSCC) were analyzed to assess differential expression. The presence of HPV in patient tissue sections was examined using reverse transcription-PCR. The present study revealed the interactions of HPV16 and 18 E6/E7 oncoproteins, highlighting their role in cancer progression by targeting key tumor suppressors, such as p53 and retinoblastoma protein. Further analysis demonstrated the involvement of HPV16 and 18 E6/E7 oncoproteins in cancer pathways, signaling and telomere regulation, which supports the development of future targeted therapies. HPV16 and 18 E6/E7 represent promising therapeutic targets in OSCC, and provide further insights into potential diagnostic and treatment avenues. The present study contributes to the current understanding of HPV-associated cancer and innovative strategies in disease management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68b946bfa96ac703dbc51d89d33f4740867a55be" target='_blank'>
              Molecular analysis of HPV16 and HPV18 oncogenes in oral squamous cell carcinoma: Structural, transcriptomic and in vitro insights
              </a>
            </td>
          <td>
            Shazia Fathima Hussain, S. Abullais, Kavitha Bottu, Logalakshmi Thirumani, Iffath Misbah, I. Madar, Nuha S. Alghamdi, M. Karobari
          </td>
          <td>2025-01-03</td>
          <td>Oncology Letters</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f79d34836731e3bc98f26657fe4be3cd3ea9eb09" target='_blank'>
              The systemic evolutionary theory of the origin of cancer (SETOC): an update
              </a>
            </td>
          <td>
            Antonio Mazzocca, Giovanni Ferraro, Giovanni Misciagna
          </td>
          <td>2025-01-14</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Bioinformatics is a rapidly evolving field charged with cataloging, disseminating, and analyzing biological data. Bioinformatics started with genomics, but while genomics focuses more narrowly on the genes comprising a genome, bioinformatics now encompasses a much broader range of omics technologies. Overcoming barriers of scale and effort that plagued earlier sequencing methods, bioinformatics adopted an ambitious strategy involving high-throughput and highly automated assays. However, as the list of omics technologies continues to grow, the field of bioinformatics has changed in two fundamental ways. Despite enormous success in expanding our understanding of the biological world, the failure of bulk methods to account for biologically important variability among cells of the same or different type has led to a major shift toward single-cell and spatially resolved omics methods, which attempt to disentangle the conflicting signals contained in heterogeneous samples by examining individual cells or cell clusters. The second major shift has been the attempt to integrate two or more different classes of omics data in a single multimodal analysis to identify patterns that bridge biological layers. For example, unraveling the cause of disease may reveal a metabolite deficiency caused by the failure of an enzyme to be phosphorylated because a gene is not expressed due to aberrant methylation as a result of a rare germline variant. Conclusions: There is a fine line between superficial understanding and analysis paralysis, but like a detective novel, multi-omics increasingly provides the clues we need, if only we are able to see them.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71c8992acca13b0990e41eb50ae3f2ac144fb351" target='_blank'>
              From Omics to Multi-Omics: A Review of Advantages and Tradeoffs
              </a>
            </td>
          <td>
            C. N. Hayes, Hikaru Nakahara, A. Ono, M. Tsuge, Shiro Oka
          </td>
          <td>2024-11-29</td>
          <td>Genes</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Background Cancer remains one of the most significant public health challenges worldwide. A widely recognized hallmark of cancer is the ability to sustain proliferative signaling, which is closely tied to various cell cycle processes. Centromere Protein A (CENPA), a variant of the standard histone H3, is crucial for selective chromosome segregation during the cell cycle. Despite its importance, a comprehensive pan-cancer bioinformatic analysis of CENPA has not yet been conducted. Methods Data on genomes, transcriptomes, and clinical information were retrieved from publicly accessible databases. We analyzed CENPA’s genetic alterations, mRNA expression, functional enrichment, association with stemness, mutations, expression across cell populations and cellular locations, link to the cell cycle, impact on survival, and its relationship with the immune microenvironment. Additionally, a prognostic model for glioma patients was developed to demonstrate CENPA’s potential as a biomarker. Furthermore, drugs targeting CENPA in cancer cells were identified and predicted using drug sensitivity correlations and protein-ligand docking. Results CENPA exhibited low levels of gene mutation across various cancers. It was found to be overexpressed in nearly all cancer types analyzed in TCGA, relative to normal controls, and was predominantly located in the nucleus of malignant cells. CENPA showed a strong association with the cancer cell cycle, particularly as a biomarker for the G2 phase. It also emerged as a valuable diagnostic and prognostic biomarker across multiple cancer types. In glioma, CENPA demonstrated reliable prognostic potential when used alongside other prognostic factors. Additionally, CENPA was linked to the immune microenvironment. Drugs such as CD-437, 3-Cl-AHPC, Trametinib, BI-2536, and GSK461364 were predicted to target CENPA in cancer cells. Conclusion CENPA serves as a crucial biomarker for the cell cycle in cancers, offering both diagnostic and prognostic value.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd288f4dd8225d97f5117baac02d9eac80a26ed1" target='_blank'>
              Single-cell and bulk RNA sequencing analysis reveals CENPA as a potential biomarker and therapeutic target in cancers
              </a>
            </td>
          <td>
            Hengrui Liu, Miray Karsidag, Kunwer S. Chhatwal, Panpan Wang, Tao Tang
          </td>
          <td>2025-01-16</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Drug resistance mechanisms in human pathogenic Candida species are constantly evolving. Over time, these species have developed diverse strategies to counter the effects of various drug classes, making them a significant threat to human health. In addition to well-known mechanisms such as drug target modification, overexpression, and chromosome duplication, Candida species have also developed permeability barriers to antifungal drugs through reduced drug import or increased efflux. The genomes of Candida species contain a multitude of drug resistance genes, many of which encode membrane efflux transporters that actively expel drugs, preventing their toxic accumulation inside the cells and contributing to multidrug resistance. This brief personal retrospective piece for the ‘Thematic Issue on Celebrating 30 Years of Cdr1 Research: new trends in antifungal therapy and drug resistance’ looks back as to how antifungal research has shifted focus since the identification of the first multidrug transporter gene, CDR1 (Candida Drug Resistance 1), leading to new insights into how reduced azole permeability across Candida cell membranes influences antifungal susceptibility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf541baa287767c3eb6de9b16b226444cf868712" target='_blank'>
              Cdr1 in Focus: A Personal Reflection on Multidrug Transporter Research
              </a>
            </td>
          <td>
            Rajendra Prasad
          </td>
          <td>2025-01-21</td>
          <td>FEMS Yeast Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e5ba64df098cc1a4c221cfe3104b836d138b7b9" target='_blank'>
              Genome-wide association studies are enriched for interacting genes
              </a>
            </td>
          <td>
            Peter T. Nguyen, Simon G. Coetzee, Irina Silacheva, Dennis J. Hazelett
          </td>
          <td>2025-01-15</td>
          <td>BioData Mining</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="The association between RET and multiple endocrine neoplasia type 2 was established in 1993 and remains one of the very few oncogenes for which distinct phenotypes (medullary thyroid cancer or phaeochromocytoma) are associated with the same hot-spot variants occurring either in germline or somatic DNA. Somatic RET fusion events have also been described in several cancers, including papillary thyroid cancer, non-small cell lung cancer, breast cancer, salivary gland cancer and pancreatic cancer. Highly selective RET inhibitors have improved outcomes in RET-altered cancers and have been well tolerated. Nevertheless, primary and acquired drug resistance has been observed, arising from distinct genomic alterations either in RET (on target resistance) or via alternate oncogenic pathways (bypass resistance). The same mechanisms of resistance have been observed across multiple cancer types, which implies RET-altered cancers evolve away from RET addiction via stochastic subclonal events. Understanding these mechanisms is crucial for identifying therapeutic opportunities to overcome resistance. Successful treatment targeting bypass oncogenes has been reported in several instances, at least for short-term outcomes; in contrast, although several compounds have been reported to overcome on-target RET alterations, none have yet been translated into routine clinical practice and this remains an area of urgent clinical need.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce4b80f3267e487a6f53e42db32a243722c94787" target='_blank'>
              Mechanisms of resistance to RET-directed therapies.
              </a>
            </td>
          <td>
            R. J. Clifton-Bligh
          </td>
          <td>2024-12-01</td>
          <td>Endocrine-related cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Gene expression is coordinated by a multitude of transcription factors (TFs), whose binding to the genome is directed through multiple interconnected epigenetic signals, including chromatin accessibility and histone modifications. These complex networks have been shown to be disrupted during aging, disease, and cancer. However, profiling these networks across diverse cell types and states has been limited due to the technical constraints of existing methods for mapping DNA:Protein interactions in single cells. As a result, a critical gap remains in understanding where TFs or other chromatin remodelers bind to DNA and how these interactions are perturbed in pathological contexts. To address this challenge, we developed a transformative single-cell immuno-tethering DNA:Protein mapping technology. By coupling a species-specific antibody-binding nanobody to a cytosine base editing enzyme, this approach enables profiling of even weak or transient factor binding to DNA, a task that was previously unachievable in single cells. Thus, our Docking & Deamination followed by sequencing (D&D-seq) technique induces cytosine-to-uracil edits in genomic regions bound by the target protein, offering a novel means to capture DNA:Protein interactions with unprecedented resolution. Importantly, this technique can be seamlessly incorporated into common single-cell multiomics workflows, enabling multimodal analysis of gene regulation in single cells. We tested the ability of D&D-seq to record TF binding both in bulk and at the single-cell level by profiling CTCF and GATA family members, obtaining high specificity and efficiency, with clear identification of TF footprint and signal retention in the targeted cell subpopulations. Furthermore, the deamination reaction showed minimal off-target activity, with high concordance to bulk ChIP-seq reference data. Applied to primary human peripheral blood mononuclear cells (PBMCs), D&D-seq successfully identified CTCF binding sites and enabled integration with advanced machine-learning algorithms for predicting 3D chromatin structure. Furthermore, we integrated D&D-seq with single-cell genotyping to assess the impact of IDH2 mutations on CTCF binding in a human clonal hematopoiesis sample, uncovering altered binding and chromatin co-accessibility patterns in mutant cells. Altogether, D&D-seq represents an important technological advance enabling the direct mapping of TF or chromatin remodeler binding to the DNA in primary human samples, opening new avenues for understanding chromatin and transcriptional regulation in health and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c35b9d0d36121406dcb7489958fdbc0f064e833" target='_blank'>
              Single-cell mapping of regulatory DNA:Protein interactions
              </a>
            </td>
          <td>
            Wei-Yu Chi, Sang-Ho Yoon, Levan Mekerishvili, S. Ganesan, Catherine Potenski, Franco Izzo, Dan A Landau, Ivan Raimondi
          </td>
          <td>2025-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b1f5a7ee68cb6b5638219d48a2b0e5c5d417966" target='_blank'>
              A complex systems approach to mosaic loss of the Y chromosome.
              </a>
            </td>
          <td>
            Xihan Guo, Xueqin Dai
          </td>
          <td>2024-12-16</td>
          <td>GeroScience</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="It is thought that cells surviving ionizing radiation exposure repair DNA double-strand breaks (DSBs) and restore their genomes. However, the recent biochemical and genetic characterization of DSB repair pathways reveals that only homologous recombination (HR) can function in an error-free manner and that the non-homologous end joining (NHEJ) pathways canonical NHEJ (c-NHEJ), alternative end joining (alt-EJ), and single-strand annealing (SSA) are error-prone, and potentially leave behind genomic scars and altered genomes. The strong cell cycle restriction of HR to S/G2 phases and the unparalleled efficiency of c-NHEJ throughout the cell cycle, raise the intriguing question as to how far a surviving cell "reaches" after repairing the genome back to its pre-irradiation state. Indeed, there is evidence that the genomes of cells surviving radiation treatment harbor extensive genomic alterations. To directly investigate this possibility, we adopted next-generation sequencing (NGS) technologies and tested a normal human fibroblast cell line, 82-6 hTert, after exposure up to 6 Gy. Cells were irradiated and surviving colonies expanded and the cells froze. Sequencing analysis using the Illumina sequencing platform and comparison with the unirradiated genome detected frequent genomic alterations in the six investigated radiation survivor clones, including translocations and large deletions. Translocations detected by this analysis and predicted to generate visible cytogenetic alterations were frequently (three out of five) confirmed using mFISH cytogenetic analysis. PCR analysis of selected deletions also confirmed seven of the ten examined. We conclude that cells surviving radiation exposure tolerate and pass to their progeny a wide spectrum of genomic alterations. This recognition needs to be integrated into the interpretation of biological results at all endpoints, as well as in the formulation of mathematical models of radiation action. NGS analysis of irradiated genomes promises to enhance molecular cytogenetics by increasing the spectrum of detectable genomic alterations and advance our understanding of key molecular radiobiological effects and the logic underpinning DSB repair. However, further developments in the technology will be required to harness its full potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b3247759666806f5cf74d149fcc038397e2c814" target='_blank'>
              NGS Detects Extensive Genomic Alterations in Survivors of Irradiated Normal Human Fibroblast Cells.
              </a>
            </td>
          <td>
            A. Soni, Daniela Beisser, E. Mladenov, Matthias Höller, Inken Wohlers, Vladimir Nikolov, S. Magin, Tamara Mussfeldt, Ludger Klein-Hitpass, Michael N. Cornforth, B. Loucas, Sven Rahmann, G. Iliakis
          </td>
          <td>2024-12-27</td>
          <td>Radiation research</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Metastasis remains a major cause of cancer mortality. AbstractThis study, expanding upon previous findings in the MMTV-PyMT model, investigated four independent mouse models, representing luminal (MMTV-PyMT, MMTV-Myc), HER2-amplified (MMTV-Her2) and triple negative (C3(1)TAg) breast cancer subtypes. Consistent with previous results, limited evidence for metastasis-associated somatic point mutations was found for all models. We also found that oncogenic drivers significantly influenced the number and size of metastasis-specific copy number variations (MSCNVs), but common driver-independent MSCNVs were rare. Furthermore, analyzing a cohort with varying genetic backgrounds while maintaining a constant oncogenic driver (PyMT) revealed that genetic background profoundly impacts MSCNVs. Transcriptome analysis demonstrated that oncogenic drivers strongly shaped metastasis-specific gene expression (MSGE), with each driver exhibiting distinct expression profiles. In contrast, MSGE in the PyMT-F1 cohort was more variable across strains. Despite the diversity of MSCNV and MSGE, functional analysis revealed that both mechanisms converge on the modulation of key cellular processes, including immune responses, metabolism, and extracellular matrix interactions. These findings emphasize the complex interplay between oncogenic drivers and genetic background in shaping the genomic and transcriptional landscapes of metastatic lesions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0340ce98aabd282b36902293b1006accf1c9547b" target='_blank'>
              Investigating the genomic landscape of mouse models of breast cancer metastasis
              </a>
            </td>
          <td>
            Christina Ross, Karol Szczepanek, Jack Sanford, Tinghu Qiu, Larry Pearce, Kent W. Hunter
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Motivation Genomic biotechnology has rapidly advanced, allowing for the inference and modification of genetic and epigenetic information at the single-cell level. While these tools hold enormous potential for basic and clinical research, they also raise difficult issues of how to design studies to deploy them most effectively. In designing a genomic study, a modern researcher might combine many sequencing modalities and sampling protocols, each with different utility, costs, and other tradeoffs. This is especially relevant for studies of somatic variation, which may involve highly heterogeneous cell populations whose differences can be probed via an extensive set of biotechnological tools. Efficiently deploying genomic technologies in this space will require principled ways to create study designs that recover desired genomic information while minimizing various measures of cost. Results The central problem this paper attempts to address is how one might create an optimal study design for a genomic analysis, with particular focus on studies involving somatic variation that occur most often with application to cancer genomics. We pose the study design problem as a stochastic constrained nonlinear optimization problem. We introduce a Bayesian optimization framework that iteratively optimizes for an objective function using surrogate modeling combined with pattern and gradient search. We demonstrate our procedure on several test cases to derive resource and study design allocations optimized for various goals and criteria, demonstrating its ability to optimize study designs efficiently across diverse scenarios. Availability and implementation https://github.com/CMUSchwartzLab/StudyDesignOptimization">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/143147eb452ae222669c29fe9d6382ddb00144ad" target='_blank'>
              Optimizing design of genomics studies for clonal evolution analysis
              </a>
            </td>
          <td>
            Arjun Srivatsa, Russell Schwartz
          </td>
          <td>2024-12-02</td>
          <td>Bioinformatics Advances</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2fc69ff58eaa4bfa3b4fa917eb7dfeb9e6159b0" target='_blank'>
              Genomic and phenotypic stability of fusion-driven pediatric sarcoma cell lines
              </a>
            </td>
          <td>
            Merve Kasan, Florian H. Geyer, Jana Siebenlist, Martin Sill, R. Öllinger, Tobias Faehling, Enrique de Álava, D. Surdez, U. Dirksen, Ina Oehme, Katia Scotlandi, O. Delattre, Martina Müller-Nurasyid, R. Rad, Konstantin Strauch, T. G. Grünewald, F. Cidre-Aranaz
          </td>
          <td>2025-01-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3899cef9922a83c1043ba54cdf83bc829243942" target='_blank'>
              The role of genetic diversity, epigenetic regulation, and sex-based differences in HIV cure research: a comprehensive review
              </a>
            </td>
          <td>
            Punitha Letchumanan, Kumitaa Theva Das
          </td>
          <td>2025-01-03</td>
          <td>Epigenetics & Chromatin</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d173ca4b0414b7ce03e7c7ceb9b88bb092a887d" target='_blank'>
              Chemical inducers of proximity: precision tools for apoptosis in transcriptional regulation
              </a>
            </td>
          <td>
            Kaituo Huang
          </td>
          <td>2024-12-20</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="ABSTRACT Chemotherapy is a potent tool against cancer, but drug resistance remains a major obstacle. To combat this, understanding the molecular mechanisms behind resistance in cancer cells and the protein expression changes driving these mechanisms is crucial. Targeting the Ubiquitin‐Proteasome System (UPS) has proven effective in treating multiple myeloma and shows promise for solid tumours. Despite initial success with the proteasome inhibitor bortezomib, acquired resistance soon after treatment poses a significant challenge to its efficacy. In this study, we explored proteins potentially involved in acquired resistance to bortezomib using label‐free nLC–MS/MS proteomic analysis. The investigation revealed 299 proteins with notable differences in expression levels in the bortezomib‐resistant PC3 prostate cancer cell line. Using bioinformatics tools, we illustrated the top 10 gene ontology (GO) processes [e.g., translational initiation (p = 5.964E‐10), CRD‐mediated mRNA stabilisation (p = 1.636E‐5), and hydrogen ion transmembrane transport (p = 6.46E‐5)] and the top 20 KEGG [e.g., metabolic pathways (p = 7.601E‐13), biosynthesis of amino acids (p = 3.834E‐12), and chemical carcinogenesis—reactive oxygen species (p = 1.891E‐4)] and REACTOME [e.g., metabolism (p = 4.182E‐21), translation (p = 9.484E‐18), and Nonsense‐Mediated Decay (NMD) (p = 1.829E‐8)] pathways in the PC3‐resistant cells. We further refined our results by comparing them with globally validated TCGA datasets. We correlated the 299 proteins identified through proteomic analysis with tumour aggressiveness and resistance by comparing them with the TCGA nodal metastasis N0 vs. N1 datasets using the UALCAN portal and identified 37 proteins consistent with our results. We believe that a combination of bortezomib with chemotherapeutics targeting these proteins could be effective in overcoming the resistance developed against bortezomib.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fb7e9b77b4e308aef46ff7c96b68009d3d9b402" target='_blank'>
              Comparative Analysis of Acquired Resistance to Bortezomib in Prostate Cancer Cells Using Proteomic and Bioinformatic Tools
              </a>
            </td>
          <td>
            Semih Seker, B. Sahin, A. Yerlikaya
          </td>
          <td>2025-01-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Abstract Every protein progresses through a natural lifecycle from birth to maturation to death; this process is coordinated by the protein homeostasis system. Environmental or physiological conditions trigger pathways that maintain the homeostasis of the proteome. An open question is how these pathways are modulated to respond to the many stresses that an organism encounters during its lifetime. To address this question, we tested how the fitness landscape changes in response to environmental and genetic perturbations using directed and massively parallel transposon mutagenesis in Caulobacter crescentus. We developed a general computational pipeline for the analysis of gene-by-environment interactions in transposon mutagenesis experiments. This pipeline uses a combination of general linear models, statistical knockoffs, and a nonparametric Bayesian statistical model to identify essential genetic network components that are shared across environmental perturbations. This analysis allows us to quantify the similarity of proteotoxic environmental perturbations from the perspective of the fitness landscape. We find that essential genes vary more by genetic background than by environmental conditions, with limited overlap among mutant strains targeting different facets of the protein homeostasis system. We also identified 146 unique fitness determinants across different strains, with 19 genes common to at least two strains, showing varying resilience to proteotoxic stresses. Experiments exposing cells to a combination of genetic perturbations and dual environmental stressors show that perturbations that are quantitatively dissimilar from the perspective of the fitness landscape are likely to have a synergistic effect on the growth defect.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09d35fab11c10334ca080d2ea7ac3de9caeecc10" target='_blank'>
              Discovering genetic modulators of the protein homeostasis system through multilevel analysis
              </a>
            </td>
          <td>
            Vishal Sarsani, Berent Aldikacti, Tingting Zhao, Shai He, Peter Chien, Patrick Flaherty
          </td>
          <td>2024-12-23</td>
          <td>PNAS Nexus</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a4ebfde7a932000ba8510eefbb0bb716c35e7fa" target='_blank'>
              The landscape of fusion transcripts in plants: a new insight into genome complexity
              </a>
            </td>
          <td>
            P. Chitkara, Ajeet Singh, Rashmi Gangwar, Rohan Bhardwaj, Shafaque Zahra, Simran Arora, Fiza Hamid, Ajay Arya, Namrata Sahu, Srija Chakraborty, Madhulika Ramesh, Shailesh Kumar
          </td>
          <td>2024-12-04</td>
          <td>BMC Plant Biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="High fidelity of replication is important to preserve genomic integrity and ensure healthy progeny. Perturbations of replication, also known as replication stress, is frequently observed in cancer cells and is considered a cancer cell-specific trait. Although replication stress drives genomic instability and tumor progression, it also generates a targetable cancer-specific vulnerability. In order to identify potential therapeutic targets in cancer cells that experience replication stress, we performed a genome wide genetic screen in human HAP1 cells challenged with low doses of replication stress-inducing drugs. We identified a large set of genes that specifically hamper cell survival in the context of replication stress. In addition to well-known players in the replication stress response and DNA repair, such as RNASEH1, BRIP1, and MDC1, we identified several genes with no prior described role in DNA replication, damage tolerance or repair. We validated that the loss of GIGYF2, HNRNPA2B1, and SUMO2 renders cells more vulnerable to replication stress. For GIGYF2 and SUMO2, we could implicate a role in homologous recombination. Taken together, our replication stress screen identified several known as well as some novel factors that protect against the toxic implications of replication stress. These factors could entail potential therapeutic targets for cancer cells experiencing replication stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddea308b1694526a4e6fc1d9af3b1fa983ba1e3c" target='_blank'>
              Identification of critical factors of the replication stress response in human cells
              </a>
            </td>
          <td>
            L. M. Janssen, Empar Baltasar Perez, C. Vaarting, Abdelghani Mazouzi, M. Raaben, T. Brummelkamp, René H. Medema, J. Raaijmakers
          </td>
          <td>2025-01-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="

Dysregulation of cellular signaling pathways leads to changes in proliferation, differentiation,
and apoptosis, leading to cancer. This review aims to provide insight into the major three signaling
pathways implicated in cancer development and progression: the Hedgehog (Hh), Phosphoinositide
3-kinase (PI3K), and Notch pathways. Abnormal activation of the Hedgehog pathway,
which has been linked to several malignancies, including medulloblastoma and basal cell carcinoma,
is primarily responsible for controlling the phases of embryonic development and tissue homeostasis.
The intricate involvement of Hh signaling in cancer stem cell maintenance, epithelial-mesenchymal
transition, and tumor microenvironment modulation underscores its significance as a therapeutic
target. Similarly, dysregulation of the PI3K pathway, a crucial mediator of cell growth, survival, and
metabolism, is prevalent across multiple cancer types. Mutations in PI3K pathway components lead
to uncontrolled cell proliferation and evasion of apoptosis, highlighting its potential as a therapeutic
avenue. Various inhibitors targeting PI3K and its downstream effectors have shown promise in preclinical
and clinical settings. Additionally, the Notch signaling pathway, crucial for cell fate determination
and tissue patterning during development, exhibits dysregulated activity in numerous cancers.
Notch pathway alterations contribute to tumor initiation, progression, and metastasis, presenting
opportunities for targeted therapies. The review discusses current therapeutic strategies targeting
these pathways, including small-molecule inhibitors, monoclonal antibodies, and combination therapies.
Challenges, such as drug resistance and toxicity are addressed, along with emerging therapeutic
approaches to enhance treatment efficacy. In conclusion, understanding the intricate crosstalk and
dysregulation of signaling pathways in cancer provides valuable insights into disease mechanisms
and therapeutic avenues, paving the way for more effective and personalized cancer treatments.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d02399e0fdd7e272270f3a2861dfe98ea9d9217" target='_blank'>
              An Insight into Signalling Pathways in Cancer: Hedgehog, PI3K, and Notch Pathways and Therapeutic Perspectives
              </a>
            </td>
          <td>
            Janvi Rohilla, Rakhi Mishra, Shruti Varshney, Rupa Mazumder, Avijit Mazumder, Anurag Chauhan, M. Chaitanya, P. Tyagi
          </td>
          <td>2024-11-29</td>
          <td>Letters in Drug Design &amp; Discovery</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The phenomenon of adaptive mutation in bacteria presents several challenges to classical models of evolution, particularly regarding the observed coordination of mutation patterns across populations. Here, we examine statistical evidence from evolved Escherichia coli populations showing mutation enrichment up to 256-fold above background rates in key metabolic genes, with remarkable temporal stability that appears to transcend known bacterial communication mechanisms. We propose a theoretical framework suggesting that bacteria may utilize quantum coherent oscillations, potentially mediated through synchronized membrane potential fluctuations, to achieve this degree of coordination in adaptive mutation. Building on Sprouffske et al.’s findings that high mutation rates can limit adaptive evolution in E. coli, we examined mutation patterns in four key metabolic genes (pykF, topA, cspC, and rpoC) previously identified as targets of selection. While Sprouffske found that extremely high mutation rates impaired adaptation, our analysis reveals that these genes show non-random enrichment patterns (p < 1.76×10-34 for pykF) that maintain remarkable temporal stability across multiple timepoints. Recent advances in quantum biology have demonstrated sustained quantum coherence in biological systems, we present a model for how quantum-mediated bacterial signaling could potentially contribute to adaptive mutation. The framework makes several experimentally testable predictions about mutation patterns, population dynamics, and coherence times in bacterial populations under stress conditions. While substantial experimental validation remains necessary, this work provides specific approaches for investigating potential quantum contributions to bacterial adaptation, with implications for understanding evolutionary mechanisms and bacterial stress responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c15202aa7837996110092bdd84ad8961fcfb080a" target='_blank'>
              Could Quantum-Mediated Bacterial Signaling Explain Adaptive Mutation?
              </a>
            </td>
          <td>
            Patrick Ross
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="MicroRNAs (miRNAs) are post-transcriptional regulators involved in key biological processes. Lineage-specific losses of multiple miRNA families during evolution are rare. If they occur, they often coincide with significant changes in gene regulation with consequences extending to body plan modifications. Recently, 15 mammalian miRNA families were found missing in the Eumuroidea, including mouse and rat. It remains unknown to what extent this has affected the gene regulatory network in this lineage. Here, we shed light on the implications of these losses. We integrate in silico characterizations of these miRNAs with differential gene expression analyses in human and mouse inducible pluripotent stem cells (iPSCs) overexpressing two representatives, Mir-197 and Mir-769. This revealed that partly the same genes are downregulated in response to the miRNA overexpression in mouse and human. We dated the emergence of the miRNA target sites in the corresponding transcripts to before the mammalian divergence. Therefore, the response to miRNA overexpression in mouse iPSCs likely represents remnants of an ancient gene regulatory network that existed already in the last common ancestor of human and mouse. This provides strong evidence that the miRNA loss disrupted established regulatory connections in Eumuroidea. Notably, we detected an accompanying loss of at least 37 transcription factors in in this lineage. Thus, Eumuroidea have modified their gene regulatory networks on two levels, transcriptionally and post-transcriptionally. It will now be interesting to precisely chart the differences in gene regulation and how this affects the use of mouse and rat as model systems for human disease. Author summary The phenotypic specifics of a species are jointly determined by its sets of structural and regulatory genes, where the latter control gene expression in a spatial and temporal manner. Micro-RNAs (miRNAs) are short regulatory genes and alter gene expression post-transcriptionally. Over evolutionary timescales, they are rarely lost in larger numbers on individual lineages, highlighting their functional relevance. The model species mouse and rat, together with their relatives in the Eumuroidea lack 15 miRNA families that are otherwise largely conserved throughout the mammals. If, and to what extent, the corresponding gene loss has affected gene regulation in the last common ancestor of Eumuroidea is unknown. Here, we provide evidence that the corresponding loss of miRNA genes is not compensated by other genes. We trace remnants of the ancestral regulatory network under control of two miRNAs in mouse, providing first time evidence that the miRNA loss in the Eumuroidea must have had consequences for gene expression regulation. We further show that protein-coding genes that have been co-lost with the 15 miRNAs are enriched for transcription factors. Jointly, this reveals that the diversification of Eumuroidea during mammalian evolution was accompanied by a change in gene expression regulation on the transcriptional and post-transcriptional level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f814f43dd7def60b542ffaccd394458b9eb13d6e" target='_blank'>
              Concerted loss of microRNAs uncovers multi-level restructuring of gene regulation in rodents
              </a>
            </td>
          <td>
            Felix Langschied, M. Leisegang, Stefan Günther, F. Hahner, Ralf Peter Brandes, Ingo Ebersberger
          </td>
          <td>2024-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The global population's health is significantly and severely threatened by antimicrobial resistance. Consequently, the scientific community allocates substantial of resources and efforts to confront this challenge. By contrast, the majority of these endeavours are focused on antibiotics, and research on antifungal resistance (AFR) is significantly underrepresented. Fungal pathogens acquire drug resistance through a variety of mechanisms. Innovative antifungal treatments and the enhancement of the efficacy of existing antifungals can be facilitated by a comprehensive understanding of the mechanisms by which fungal infections acquire drug resistance. Chromatin structure and gene expression regulation are critical components of fungal species' adaptation to antifungal stress, which suggests a potential therapeutic approach to AFR. This suggests that developing strategies that concentrate on these mechanisms may be a viable approach for controlling antifungal resistance. For the regulation of a diverse array of fungal biology components, epigenetic pathways are indispensable in medical mycology. The development process and the capacity to modify and adapt physical characteristics and resistance to antifungals that are used to treat fungal infections are critically dependent on these methods. The development process and the ability to modify and adapt physical characteristics and resistance to antifungals that are used to treat fungal infections are critically dependent on these methods. A significant concern is increasing resistance to the limited therapeutic options that are available to manage invasive fungal infections, such as histone acetylation and methylation, chromatin remodelling, and gene silencing through heterochromatin, which inhibit prevailing drug-resistance mechanisms. This review discusses the significance of epigenetic pathways in mediating drug resistance in fungi as well as mechanisms of antifungal drug resistance">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b714470b16b023b24367f2fc71a3ef61a0e65b4" target='_blank'>
              The Role of Epigenetic Mechanisms in Modulating Antifungal Resistance in Fungi
              </a>
            </td>
          <td>
            Aya A Tarek, Mohamed N Hassan, Yasmine Tartor
          </td>
          <td>2024-12-01</td>
          <td>Zagazig Veterinary Journal</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The DNA damage response (DDR) and cellular metabolism exhibit a complex, bidirectional relationship crucial for maintaining genomic integrity. Studies across multiple organisms, from yeast to humans, have revealed how cells rewire their metabolism in response to DNA damage, supporting repair processes and cellular homeostasis. We discuss immediate metabolic shifts upon damage detection and long-term reprogramming for sustained genomic stability, highlighting key signaling pathways and participating molecules. Importantly, we examine how DNA repair processes can conversely induce metabolic changes and oxidative stress through specific mechanisms, including the histone H2A variant X (H2AX)/ataxia telangiectasia mutated (ATM)/NADPH oxidase 1 (Nox1) pathway and repair-specific ROS signatures. The review covers organelle-specific responses and metabolic adaptations associated with different DNA repair mechanisms, with a primary focus on human cells. We explore the implications of this DDR–metabolism crosstalk in cancer, aging, and neurodegenerative diseases, and discuss emerging therapeutic opportunities. By integrating recent findings, this review provides a comprehensive overview of the intricate interplay between DDR and cellular metabolism, offering new perspectives on cellular resilience and potential avenues for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1db2e6dc985bea558e8c521d973eb4eb92104a14" target='_blank'>
              Metabolic Rewiring in the Face of Genomic Assault: Integrating DNA Damage Response and Cellular Metabolism
              </a>
            </td>
          <td>
            Wenjian Ma, Sa Zhou
          </td>
          <td>2025-01-23</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The ability of cancer cells to alter their identity, known as lineage plasticity, is crucial for tumor progression and therapy resistance. In lung adenocarcinoma (LUAD), tumor progression is characterized by a gradual loss of lineage fidelity and the emergence of non-pulmonary identity programs. This can lead to hybrid-identity (hybrid-ID) states in which developmentally incompatible identity programs are co-activated within individual cells. However, the molecular mechanisms underlying these identity shifts remain incompletely understood. Here, we identify the gastrointestinal (GI) transcriptional regulator HNF4α as a critical driver of tumor growth and proliferation in KRAS-driven LUAD. In LUAD cells that express the lung lineage specifier NKX2-1, HNF4α can induce a GI/liver-like state by directly binding and activating its canonical targets. HNF4α also forms an aberrant protein complex with NKX2-1, which disrupts NKX2-1 localization and dampens pulmonary identity within hybrid-ID LUAD. Sustained signaling through the RAS/MEK pathway is critical for maintaining the hybrid-ID state. Moreover, RAS/MEK inhibition augments NKX2-1 chromatin binding at pulmonary-specific genes and induces resistance-associated pulmonary signatures. Finally, we demonstrate that HNF4α depletion enhances sensitivity to pharmacologic KRASG12D inhibition. Collectively, our data show that co-expression of opposing lineage specifiers leads to a hybrid identity state that can drive tumor progression and dictate response to targeted therapy in LUAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/207f2a257f041c211d79bb19f39ca01a49097c42" target='_blank'>
              Opposing lineage specifiers induce a pro-tumor hybrid-identity state in lung adenocarcinoma
              </a>
            </td>
          <td>
            Gabriela Fort, Henry Arnold, S. Camolotto, Rushmeen Tariq, Anna Waters, Kayla T. O'Toole, Eric L. Snyder
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="BRCA1, a critical tumor suppressor gene, plays an essential role in maintaining genomic stability through its involvement in DNA double-strand break repair, particularly via homologous recombination. Loss or impairment of BRCA1 function disrupts this repair pathway, resulting in genomic instability and significantly increased susceptibility to breast and ovarian cancers. To elucidate the molecular mechanisms by which BRCA1 mutations contribute to tumorigenesis, we employed quantitative mass spectrometry-based proximity labeling and affinity purification to identify cancer-specific protein-protein interactions (PPIs). Our integrated omics and visualization analysis of interactors revealed that the BRCA1-Y1853ter mutant, through its interaction with BARD1, perturbs the interactome and impacts cellular processes within the cytoplasm and nucleoplasm. Structural data further corroborated these findings, showing enhanced binding between the mutant BRCA1 and specific interactors, suggesting an altered functional profile. Together, these observations raise the hypothesis that the BRCA1-Y1853ter mutant may exhibit gain-of-function characteristics, providing new insights into the molecular and cellular effects of mutations in the BRCA1 C-Terminal (BRCT) domain and their implications for the pathogenesis of breast and ovarian cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7081d3dc4a7fe94ebd83524e06e557bb05ca412" target='_blank'>
              Differential Remodeling of the BRCA1 Interactome by a Unique Minimal Clinical Truncation Mutation
              </a>
            </td>
          <td>
            Jingjing Chen, Mihaela E. Sardiu, M. Rekowski, Jules Nde, Zachary Clark, S. Anant, Michael P. Washburn, Roy A Jensen
          </td>
          <td>2024-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Questions in cancer have engaged systems biologists for decades. During that time, the quantity of molecular data has exploded, but the need for abstractions, formal models, and simplifying insights has remained the same. This review brings together classic breakthroughs and recent findings in the field of cancer systems biology, focusing on cancer-cell pathways for tumorigenesis and therapeutic response. Cancer cells mutate and transduce information from their environment to alter gene expression, metabolism, and phenotypic states. Understanding the molecular architectures that make each of these steps possible is a long-term goal of cancer systems biology pursued by iterating between quantitative models and experiments. We argue that such iteration is the best path to deploying targeted therapies intelligently so that each patient receives the maximum benefit for their cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8544584d0cff558bd248c81e02267385e425b55" target='_blank'>
              Systems Biology of the Cancer Cell.
              </a>
            </td>
          <td>
            Kevin A. Janes, M. Lazzara
          </td>
          <td>2024-12-17</td>
          <td>Annual review of biomedical engineering</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="SUMMARY DNA, is assaulted by endogenous and exogenous agents that lead to the formation of damage. In order to maintain genome integrity DNA repair pathways must be efficiently activated to prevent mutations and deleterious chromosomal rearrangements. Conversely, genome rearrangement is also necessary to allow genetic diversity and evolution. The antagonist interaction between maintenance of genome integrity and rearrangements determines genome shape and organization. Therefore, it is of great interest to understand how the whole linear genome structure behaves upon formation and repair of DNA damage. For this, we used long reads sequencing technology to identify and to characterize genomic structural variations (SV) of wild‐type Arabidopsis thaliana somatic cells exposed either to UV‐B, to UV‐C or to protons irradiations. We found that genomic regions located in heterochromatin are more prone to form SVs than those located in euchromatin, highlighting that genome stability differs along the chromosome. This holds true in Arabidopsis plants deficient for the expression of master regulators of the DNA damage response (DDR), ATM (Ataxia‐telangiectasia‐mutated) and ATR (Ataxia‐telangiectasia‐mutated and Rad3‐related), suggesting that independent and alternative surveillance processes exist to maintain integrity in genic regions. Finally, the analysis of the radiations‐induced deleted regions allowed determining that exposure to UV‐B, UV‐C and protons induced the microhomology‐mediated end joining mechanism (MMEJ) and that both ATM and ATR repress this repair pathway.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52b13004cb07afbb75c796371a93fdfc391d332b" target='_blank'>
              Characterization of radiations‐induced genomic structural variations in Arabidopsis thaliana
              </a>
            </td>
          <td>
            Salimata Ousmane Sall, Abdelmalek Alioua, Sébastien Staerck, Stéfanie Graindorge, M. Pellicioli, Jacky Schuler, Catherine Galindo, Quentin Raffy, Marc Rousseau, J. Molinier
          </td>
          <td>2024-12-01</td>
          <td>The Plant Journal</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="ABSTRACT The resistance of commonly used clinical antibiotics, such as daptomycin (DAP), has become increasingly serious in the fight against Staphylococcus aureus (S. aureus) infection. It is essential to explore key pathogenicity-driven genes/proteins in bacterial infection and antibiotics resistance, which contributes to develop novel therapeutic strategies against S. aureus infections. The nt5 gene of S. aureus, encoding 5’-nucleotidase (NT5), is nearly unknown for its function in drug resistance and bacterial infection. Herein, to reveal nt5 gene role in drug resistance and infection ability of S. aureus, we performed nt5C166T gene mutation using a clustered regulatory interspaced short palindromic repeat ribonucleic acid (RNA)-guided base editing system to investigate the lose-of-function of NT5 protein. Subsequent transcriptome sequencing of the mutant strain revealed that nt5 inactivation caused changes in cell membrane integrity and inhibited nucleotide metabolism, suggesting the nt5 gene may be involved in bacterial drug resistance and virulence. The mutant strain exhibited enhanced tolerance to DAP treatment by attenuating cell membrane potential dissipation and slowing deoxyribonucleic acid release. Moreover, the nt5 mutation alleviated abscess degree of mouse kidneys caused by S. aureus infection byreducing the expression of IL-1β, IL-6, and IL-18. The nt5 mutant strain was easily swallowed by host immune cells, resulting in weak bacterial toxicity of the S. aureus mutant in the bacterial infection process. In summary, nt5 gene mutation confers tolerance to DAP and a lower bacterial capacity to form kidney abscesses through phagocytosis of host immune cells, which indicates the targeted inhibition of NT5 protein would offer a potential new therapeutic strategy against S. aureus infection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6a0cc9682d5939f3381491b119adb48bc61ac9" target='_blank'>
              Staphylococcus aureus nt5 gene mutation through CRISPR RNA-guided base editing weakens bacterial virulence and immune evasion
              </a>
            </td>
          <td>
            Xinpeng Liu, Lan Huang, Yang Ye, Haiyi Wang, Min Tang, Fuqiang He, Zijing Xia, Shi Deng, Peng Zhang, Ruiwu Dai, Shufang Liang
          </td>
          <td>2025-01-22</td>
          <td>Virulence</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is a highly aggressive and clinically challenging subtype of breast cancer, lacking the expression of estrogen receptor (ER), progesterone receptor (PR), and HER2/neu. The absence of these receptors limits therapeutic options necessitating the exploration of novel treatment strategies. Epigenetic modifications, which include DNA methylation, histone modifications, and microRNA (miRNA) regulation, play a pivotal role in TNBC pathogenesis and represent promising therapeutic targets. This review delves into the therapeutic potential of epigenetic interventions in TNBC, with a focus on DNA methylation, histone modifications, and miRNA therapeutics. We examine the role of DNA methylation in gene silencing within TNBC and the development of DNA methylation inhibitors designed to reactivate silenced tumor suppressor genes. Histone modifications, through histone deacetylation and acetylation in particular, are critical in regulating gene expression. We explore the efficacy of histone deacetylase inhibitors (HDACi), which have shown promise in reversing aberrant histone deacetylation patterns, thereby restoring normal gene function, and suppressing tumor growth. Furthermore, the review highlights the dual role of miRNAs in TNBC as both oncogenes and tumor suppressors and discusses the therapeutic potential of miRNA mimics and inhibitors in modulating these regulatory molecules to inhibit cancer progression. By integrating these epigenetic therapies, we propose a multifaceted approach to target the underlying epigenetic mechanisms that drive TNBC progression. The synergistic use of DNA methylation inhibitors, HDACi, and the miRNA-based therapies offers a promising avenue for personalized treatment strategies, aiming to enhance the clinical outcome for patients with TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f782a5505b5b2b26e898e78e4e01384ddeb62d3" target='_blank'>
              Unlocking the epigenetic code: new insights into triple-negative breast cancer
              </a>
            </td>
          <td>
            G. Mahendran, Ann Dharshika, Ricardo Romero-Moreno, Nadeeshika Wickramarachchige Dona, S. H. G. S. Sarasija, Sumeth Perera, Gayathri N. Silva, Fang Yang, Guochun Zhang, Wickramarachchige Dona, Perera Sarasija, Silva. This
          </td>
          <td>2024-12-18</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5361264ba7fa5b03a05e9caae598d056bebbf71c" target='_blank'>
              Multi-omic profiling highlights factors associated with resistance to immuno-chemotherapy in non-small-cell lung cancer.
              </a>
            </td>
          <td>
            Yilv Yan, Dongqing Sun, Junjie Hu, Yue Chen, Liangdong Sun, Huansha Yu, Yicheng Xiong, Zhida Huang, Haoran Xia, Xinsheng Zhu, D. Bian, Fenghuan Sun, Likun Hou, Chunyan Wu, Orion R Fan, Haiyang Hu, An Zeng, Lele Zhang, Yi Eve Sun, Chenfei Wang, Peng Zhang
          </td>
          <td>2024-12-10</td>
          <td>Nature genetics</td>
          <td>1</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24be616e06c75b57b6ca8151f14f1e70c2c9348f" target='_blank'>
              PIK3CA mutation fortifies molecular determinants for immune signaling in vascular cancers.
              </a>
            </td>
          <td>
            Donghee Lee, Emma C Kozurek, Md Abdullah, Ethan J Wong, Rong Li, Z. Liu, Hai Dang Nguyen, Erin Dickerson, Jong Hyuk Kim
          </td>
          <td>2024-12-21</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Here, we have discussed the molecular mechanisms of p53-responsive microRNAs dysregulation in response to genotoxic stress in diffuse large B-cell lymphoma (DLBCL) patients. The role of micro ribonucleic acids (microRNAs) in p53-signaling cellular stress has been studied. MicroRNAs are the small non-coding RNAs, which regulate genes expression at post-transcriptional level. Many of them play a crucial role in carcinogenesis and may act as oncogenes or suppressor of tumor growth. The understanding of the effect of p53-responsive microRNA dysregulation on oncogenesis achieved in recent decades opens wide opportunities for the diagnosis, prediction and of microRNA-based cancer therapy. Development of new bioinformatics tools and databases for microRNA supports DLBCL research. We overview the studies on the role of miRNAs in regulating gene expression associated with tumorigenesis processes, with particular emphasis on their role as tumor growth-suppressing factors. The starting point is a brief description of the classical microRNA biogenesis pathway and the role of p53 in regulating the expression of these molecules. We analyze various molecular mechanisms leading to this dysregulation, including mutations in the TP53 gene, DNA methylation, changes in host-genes expression or microRNA gene copy number, mutations in microRNA and microRNA biogenesis genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/740d5b189850125f69ad77979cf0d447a7b59eef" target='_blank'>
              Deregulation mechanisms and therapeutic opportunities of p53-responsive microRNAs in diffuse large B-cell lymphoma
              </a>
            </td>
          <td>
            E. Voropaeva, Yuriy L. Orlov, Anastasia B. Loginova, O. Seregina, Vladimir N. Maksimov, T. I. Pospelova
          </td>
          <td>2025-01-07</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="As failure rates for traditional antimicrobial therapies escalate, recent focus has shifted to evolution-based therapies to slow resistance. Collateral sensitivity—the increased susceptibility to one drug associated with evolved resistance to a different drug—offers a potentially exploitable evolutionary constraint, but the manner in which collateral effects emerge over time is not well understood. Here, we use laboratory evolution in the opportunistic pathogen Enterococcus faecalis to phenotypically characterize collateral profiles through evolutionary time. Specifically, we measure collateral profiles for 400 strain-antibiotic combinations over the course of 4 evolutionary time points as strains are selected in increasing concentrations of antibiotic. We find that at a global level—when results from all drugs are combined—collateral resistance dominates during early phases of adaptation, when resistance to the selecting drug is lower, while collateral sensitivity becomes increasingly likely with further selection. At the level of individual populations; however, the trends are idiosyncratic; for example, the frequency of collateral sensitivity to ceftriaxone increases over time in isolates selected by linezolid but decreases in isolates selected by ciprofloxacin. We then show experimentally how dynamic collateral sensitivity relationships can lead to time-dependent dosing windows that depend on finely timed switching between drugs. Finally, we develop a stochastic mathematical model based on a Markov decision process consistent with observed dynamic collateral profiles to show measurements across time are required to optimally constrain antibiotic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a35c7b082fdfc513f8363c2aedd186ae1aecdfe3" target='_blank'>
              Dynamic collateral sensitivity profiles highlight opportunities and challenges for optimizing antibiotic treatments
              </a>
            </td>
          <td>
            Jeff Maltas, Anh Huynh, Kevin B. Wood
          </td>
          <td>2025-01-01</td>
          <td>PLOS Biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Abstract The molecular basis of adaptive evolution and cancer progression are both complex processes that share many striking similarities. The potential adaptive significance of environmentally-induced epigenetic changes is currently an area of great interest in both evolutionary and cancer biology. In the field of cancer biology intense effort has been focused on the contribution of stress-induced non-coding RNAs (ncRNAs) in the activation of epigenetic changes associated with elevated mutation rates and the acquisition of environmentally adaptive traits. Examples of this process are presented and combined with more recent findings demonstrating that stress-induced ncRNAs are transferable from somatic to germline cells leading to cross-generational inheritance of acquired adaptive traits. The fact that ncRNAs have been implicated in the transient adaptive response of various plants and animals to environmental stress is consistent with findings in cancer biology. Based on these collective observations, a general model as well as specific and testable hypotheses are proposed on how transient ncRNA-mediated adaptive responses may facilitate the transition to long-term biological adaptation in both cancer and evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbfc2bb67d7c58db953ed76c627eb64e92c4eb6a" target='_blank'>
              Adaptive Significance of Non-coding RNAs: Insights from Cancer Biology
              </a>
            </td>
          <td>
            John F. McDonald
          </td>
          <td>2025-01-01</td>
          <td>Molecular Biology and Evolution</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Immune checkpoint inhibitors (ICIs) are extremely effective in a subgroup of mismatch repair-deficient (MMRd) cancers, but ∼50% remain resistant to treatment. We have shown for the first time that this may be due to the differential regulation of factors linked to response to ICIs upon loss of the different MMR genes. Here, we show that increased PD-L1 expression is observed upon loss of the MMR genes MLH1, MSH2 and PMS2. However, this is not true upon loss of MSH6. Here, we show that this is due to a novel role for MSH6 as a direct regulator of PD-L1 transcription, dependent on recruitment by the histone trimethyltransferase SETD2. Next-generation sequencing of MLH1 and MSH6 knockout (KO) cells revealed that MSH6 KO cells have significantly lower microsatellite instability in comparison to MLH1 KO cells, despite MSH6 KO cells having a higher mutational burden. These findings emphasise the need for gene-specific stratification in the MMRd cohort.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d6ede7d9d243c2456000d9151a205cf93b9af95" target='_blank'>
              The DNA Mismatch repair protein, MSH6 is a novel regulator of PD-L1 expression
              </a>
            </td>
          <td>
            Kirsten Brooksbank, Charlotte Smith, E. Maniati, Amy Gibson, Wai Yiu Tse, Amy Kate Hall, Jun Wang, T. Sharp, Sarah A Martin
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Metastatic melanoma remains a major clinical challenge. Large-scale genomic sequencing of melanoma has identified bona fide activating mutations in RAC1, which are associated with resistance to BRAF-targeting therapies. Targeting the RAC1-GTPase pathway, including the upstream activator PREX2 and the downstream effector PI3Kβ, could be a potential strategy for overcoming therapeutic resistance, limiting melanoma recurrence, and suppressing metastatic progression. Here, we used genetically engineered mouse models and patient-derived BRAFV600E-driven melanoma cell lines to dissect the role of PREX2 in melanomagenesis and response to therapy. While PREX2 was dispensable for the initiation and progression of melanoma, its loss conferred sensitivity to clinically relevant therapeutics targeting the MAPK pathway. Importantly, genetic and pharmacological targeting of PI3Kβ phenocopied PREX2 deficiency, sensitizing model systems to therapy. These data reveal a druggable PREX2/RAC1/PI3Kβ signaling axis in BRAF-mutant melanoma that could be exploited clinically.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d015227301e9822a2f9c55706223dee9a9babf44" target='_blank'>
              Targeting the PREX2/RAC1/PI3Kβ Signaling Axis Confers Sensitivity to Clinically Relevant Therapeutic Approaches in Melanoma.
              </a>
            </td>
          <td>
            Catriona A. Ford, Dana Koludrovic, Patricia P. Centeno, M. Foth, Elpida Tsonou, Nikola Vlahov, Nathalie Sphyris, K. Gilroy, Courtney Bull, Colin Nixon, Bryan Serrels, Alison F. Munro, John C Dawson, Neil O. Carragher, Valeria Pavet, David C Hornigold, Philip D. Dunne, Julian Downward, Heidi C. E. Welch, Simon T Barry, Owen J. Sansom, A. Campbell
          </td>
          <td>2024-12-05</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f7b4b6b0cd06f7bc05e01f01febfa31643b31b8" target='_blank'>
              Prevalent integration of genomic repetitive and regulatory elements and donor sequences at CRISPR-Cas9-induced breaks
              </a>
            </td>
          <td>
            C. Bi, Baolei Yuan, Yingzi Zhang, Mengge Wang, Yeteng Tian, Mo Li
          </td>
          <td>2025-01-20</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="5-Methylcytosine (5mC) is a common source of somatic mutations. Deamination of 5mC to thymine generates a G/T mismatch, which occurs spontaneously and must be repaired prior to DNA replication to avoid mutation. We generated genetically engineered mice and cell lines to define DNA repair pathways that protect against methylation damage. We observed a low background mutation rate in mouse bone marrow or colon, typically 0.2-0.5 CG>TG mutations/genome/day. This increased 3-7 fold in cells lacking the glycosylase Methyl-binding domain 4 (Mbd4), one of the few glycosylases capable of excising thymine from G/T mismatches. We found no role for Thymine DNA glycosylase (Tdg) in methylation damage repair. Instead, our results support cooperation between Mbd4 and the mismatch repair (MMR) complex MutSα (Msh6:Msh2), evident through elevated rates of methylation damage in Msh6-deficient cells; increasing to 2.6-4.8 CG>TG mutations/genome/day in primary cells and up to 13.9 CG>TG mutations/genome/day in cell lines. Our findings support the view that MutSα has DNA repair activity outside of replication. While loss of Mbd4 elevates methylation damage selectively, the broader functionality of MutSα explains why mutational signatures linked to Msh6-deficiency are variable and reflect the replicative history of the cell.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d89af6527eda652ec94285879400d97b8fdfca77" target='_blank'>
              Mbd4 and MutSα protect cells from spontaneous deamination of 5-methylcytosine
              </a>
            </td>
          <td>
            R. Bilardi, C. Flensburg, Zhen Xu, Emily B. Derrick, Andrew Kueh, I. Majewski
          </td>
          <td>2024-12-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Genomic reorganisation between species and horizontal gene transfer have been considered the most important mechanism of biological adaptation under selective pressure. Still, the impact of mobile genes in microbial ecology is far from being completely understood. Here we present the collection and characterisation of microbial consortia enriched from environments contaminated with emerging pollutants, such as non-steroidal anti-inflammatory drugs. We have obtained and further enriched two ibuprofen-degrading microbial consortia from two unrelated wastewater treatment plants. We have also studied their ability to degrade the drug and the dynamics of the reorganisations of the genetic information responsible for its biodegradation among the species within the consortium. Our results show that genomic reorganisation within microorganisms and species rearrangements occur rapidly and efficiently during the selection process, which may be facilitated by plasmids and/or transposable elements located within the sequences. We show the evolution of at least two different plasmid backbones on samples from different locations, showing rearrangements of genomic information, including genes encoding activities for IBU degradation. As a result, we found variations in the expression pattern of the consortia after evolution under selective pressure, as an adaptation process to the new conditions. This work provides evidence for changes in the metagenomes of microbial communities that allow adaptation under a selective constraint -ibuprofen as a sole carbon source- and represents a step forward in knowledge that can inspire future biotechnological developments for drug bioremediation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e84c96d5ab7e7b0f460499b83019aa7bc04828d" target='_blank'>
              Genetic and species rearrangements in microbial consortia impact biodegradation potential.
              </a>
            </td>
          <td>
            Z. Saati-Santamaría, Pilar Navarro-Gómez, Juan A Martínez-Mancebo, Maitane Juárez-Mugarza, Amando Flores, Inés Canosa
          </td>
          <td>2025-01-25</td>
          <td>The ISME journal</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Intron retention is a type of alternative splicing in which introns remain unspliced in mature RNA transcripts. In order to explore the landscape and consequences of genetically regulated intron retention, we perform an intron retention quantitative trait loci (irQTL) analysis in 49 human tissues across 838 individuals. We identify 8,624 unique intron retention events associated with genetic polymorphisms. 1,369 irQTLs (16%) are also associated with genome-wide association study (GWAS) traits, suggesting that genetically regulated intron retention may play a causal role in the regulation of these GWAS traits. 1,999 irQTLs (23%) colocalize with eQTLs to their respective gene, suggesting that intron retention and steady-state gene expression levels are regulated by shared sets of genetic variants. Through a colocalization analysis of irQTL:eQTL events, we demonstrate that irQTLs are sufficient to generate eQTLs when one of the alternatively spliced transcripts is preferentially targeted by the nonsense mediated decay (NMD) pathway. Surprisingly, for intron retention events whose potential NMD effects can be confidently predicted based on their positions within known gene annotations, we find that 58.8% (923/1570) of the colocalized irQTL and eQTL pairs show effect size directions that are discordant with the NMD model. Moreover, we find that irQTLs are significantly more likely to occur in the same gene with the same effect size direction as compared to exon skipping QTLs. Through mathematical modeling and analysis of experimental perturbation data, we provide evidence that eQTLs are able to generate irQTLs by altering the steady state ratios of spliced and unspliced transcripts, and we postulate that this mechanism may partially underlie the widespread intron retention observed previously in various biological conditions. Taken together, these results show that intron retention and steady state gene expression levels are closely intertwined to regulate phenotypic traits.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7207fff2c80c573e0b32d3cba5d95f5f23832382" target='_blank'>
              Origins and Implications of Intron Retention Quantitative Trait Loci in Human Tissues
              </a>
            </td>
          <td>
            Eddie Park, Yi Xing
          </td>
          <td>2024-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The growing prevalence of antibiotic-resistant bacteria within the human microbiome has become a pressing global health crisis. While antibiotics have revolutionized medicine by significantly reducing mortality and enabling advanced medical interventions, their misuse and overuse have led to the emergence of resistant bacterial strains. Key resistance mechanisms include genetic mutations, horizontal gene transfer, and biofilm formation, with the human microbiota acting as a reservoir for antibiotic resistance genes (ARGs). Industrialization and environmental factors have exacerbated this issue, contributing to a rise in infections with multidrug-resistant (MDR) bacteria, such as methicillin-resistant Staphylococcus aureus (MRSA) and carbapenem-resistant Enterobacteriaceae. These resistant pathogens compromise the effectiveness of essential treatments like surgical prophylaxis and chemotherapy, increase healthcare costs, and prolong hospital stays. This crisis highlights the need for a global One-Health approach, particularly in regions with weak regulatory frameworks. Innovative strategies, including next-generation sequencing (NGS) technologies, offer promising avenues for mitigating resistance. Addressing this challenge requires coordinated efforts, encompassing research, policymaking, public education, and antibiotic stewardship, to safeguard current antibiotics and foster the development of new therapeutic solutions. An integrated, multidimensional strategy is essential to tackle this escalating problem and ensure the sustainability of effective antimicrobial treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a3dd99c233a325d1af9d14880833e0f1ff7573e" target='_blank'>
              From Cure to Crisis: Understanding the Evolution of Antibiotic-Resistant Bacteria in Human Microbiota.
              </a>
            </td>
          <td>
            Hamed Tahmasebi, Neda Arjmand, Marzieh Monemi, Ali Babaeizad, Farnaz Alibabaei, Negar Alibabaei, A. Bahar, Valentyn Oksenych, Majid Eslami
          </td>
          <td>2025-01-09</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="ABSTRACT Background SET domain‐containing protein 4 (SETD4) is a histone methyltransferase that has been shown to modulate cell proliferation, differentiation, and inflammatory responses by regulating histone H4 trimethylation (H4K20me3). Previous reports have demonstrated its function in the quiescence of cancer stem cells as well as drug resistance in several cancers. A limited number of systematic studies have examined SETD4's role in the tumor microenvironment, pathogenesis, prognosis, and therapeutic response. Methods Utilizing The Cancer Genome Atlas database, and other publicly accessible platforms, we comprehensively analyzed SETD4 gene expression, methylation patterns, and prognostic significance. Furthermore, we investigated its association with cancer‐related pathways, the immune microenvironment, immunotherapy markers, and drug resistance signatures of chemotherapy. Additionally, qRT‐PCR was performed to validate SETD4 expression in clinical specimens. Results The expression of SETD4 was abnormal across a variety of cancer types and the expression of SETD4 in colorectal cancer tissues was verified in clinical specimens. The upregulation of SETD4 may be a prognostic risk factor predicting poor overall survival and progression‐free survival. The analysis revealed that the mRNA level of SETD4 was modulated by promoter methylation, and patients with lower methylation levels showed shorter survival times. Pathway analysis showed that SETD4 influenced several key cell cycle pathways, including the G2M checkpoint, and mitotic spindle pathways. In addition, SETD4 negatively affects immune cell infiltration in most cancers, including B cells, CD8 T cells, and macrophages. The correlation between SETD4 and cancer stemness as well as homologous recombination deficiency varied across tumor types, suggesting that SETD4 may play a multifaceted role in tumor resistance. Notably, we identified several potential agents targeting SETD4. Conclusions This study demonstrates that SETD4 is an immune‐oncogenic molecule in multiple cancers, with the potential to be a diagnosis, prognosis, and targeted therapy marker.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f76f7155098d30f047c5ee80fc1fcf8c03b367a" target='_blank'>
              Identification of SETD4 as an Onco‐Immunological Biomarker Encompassing the Tumor Microenvironment, Prognoses, and Therapeutic Responses in Various Human Cancers
              </a>
            </td>
          <td>
            Yuyun Zhong, Ruiqi Wang, Zijie Huang, Zhaoting Hu, Bin Peng, Bin Chen, Liyue Sun
          </td>
          <td>2025-01-01</td>
          <td>Immunity, Inflammation and Disease</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42786039d5d90ecc6efc329111435b0ff4acfda0" target='_blank'>
              Homologous recombination deficiency (HRD) diagnostics: underlying mechanisms and new perspectives.
              </a>
            </td>
          <td>
            Andrey Kechin, M. Koryukov, Regina Mikheeva, M. Filipenko
          </td>
          <td>2024-12-26</td>
          <td>Cancer metastasis reviews</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Purpose Endocrine therapy in combination with CDK4/6 inhibition doubles the progression-free survival of patients with advanced ER+ breast cancer, but resistance is inevitable, leaving patients with limited treatment options. Experimental Design We performed unbiased genome-wide CRISPR/Cas9 knockout screens using ER+ breast cancer cells to identify novel drivers of resistance to combination endocrine therapy (tamoxifen) and CDK4/6 inhibitor (palbociclib) treatment. Screen hits were validated by CRISPR/Cas9 knockout models, mechanistic analyses and evaluation of patient samples. Results Our screens identified the inactivation of JNK signalling, including loss of the kinase MAP2K7, as a key driver of combination resistance. We developed multiple CRISPR/Cas9 knockout ER+ breast cancer cell lines (MCF-7 and T-47D) to investigate the effects of MAP2K7, MAPK8 and MAPK9 loss. MAP2K7 knockout increased metastatic burden in vivo and led to impaired JNK-mediated stress responses, as well as promoting cell survival and reducing senescence entry following endocrine therapy and CDK4/6 inhibitor treatment. Mechanistically, this occurred via loss of the AP-1 transcription factor c-JUN, leading to an attenuated response to combination endocrine therapy plus CDK4/6 inhibition. Furthermore, we analysed ER+ advanced breast cancer patient cohorts and found that inactivation of the JNK pathway was associated with increased metastatic burden, and low pJNKT183/Y185 activity correlated with a poorer response to systemic endocrine and CDK4/6 inhibitor therapies. Conclusions Overall, we demonstrate that suppression of JNK signalling enables persistent growth during combined endocrine therapy and CDK4/6 inhibition. Furthermore, our data provide a pre-clinical rationale to screen patients’ tumours for JNK signalling deficiency prior to receiving combined endocrine therapy and CDK4/6 inhibition. STATEMENT OF TRANSLATIONAL RELEVANCE Resistance to CDK4/6 inhibitors in the context of endocrine therapy resistance presents an urgent clinical challenge for the management of estrogen receptor positive (ER+) breast cancer. However, the mechanisms driving resistance to this therapeutic combination remain poorly understood. Here, we have identified inactivation of JNK signalling, specifically loss of the JNK kinase MAP2K7, as a major determinant of resistance to combined endocrine therapy and CDK4/6 inhibition in ER+ breast cancer. We uncovered that MAP2K7 loss augments tumour survival in vitro and in vivo. This occurs via disruption of activator protein-1 (AP-1) transcription factors, and thus prevents the induction of therapy-induced senescence and JNK-induced stress response. These findings reveal a critical tumour suppressor role for the JNK pathway in ER+ breast cancer, highlighting the importance of identifying patients with deficient JNK signalling and cautioning against the development of JNK inhibitors for this setting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ea27aa25966a8a8e422d458bc9b0ffc44d3b78e" target='_blank'>
              JNK pathway suppression drives resistance to combination endocrine therapy and CDK4/6 inhibition in ER+ breast cancer
              </a>
            </td>
          <td>
            Sarah Alexandrou, Christine S. Lee, Kristine J. Fernandez, Celine E. Wiharja, Leila Eshraghi, John Reeves, Daniel A. Reed, N. Portman, Zoe Phan, H. Milioli, Iva Nikolic, Antonia L. Cadell, David R. Croucher, K. Simpson, Elgene Lim, Theresa E Hickey, Ewan K. A. Millar, Carla L. Alves, Henrik J Ditzel, C. E. Caldon
          </td>
          <td>2025-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Pediatric acute myeloid leukemia (pAML) is a clonal disease with recurrent genetic alterations that affect epigenetic states. However, the implications of epigenetic dysregulation in disease progression remain unclear. Here, we interrogated single-cell and clonal level chromatin accessibility of bone marrow samples from 28 pAML patients representing multiple subtypes using mtscATAC-seq, which revealed distinct differentiation hierarchies and abnormal chromatin accessibility in a subtype-specific manner. Innate immune signaling was commonly enhanced across subtypes and related to improved advantage of clonal competition and unfavorable prognosis, with further reinforcement in a relapse-associated leukemia stem cell-like population. We identified a panel of 31 innate immunity related genes to improve the risk classification of pAML patients. By comparing paired diagnosis and post-chemotherapy relapse samples, we showed that primitive cells significantly reduced MHC class II signaling, suggesting an immune evasion mechanism to facilitate their expansion at relapse. Key regulators orchestrating cell cycle dysregulation were identified to contribute to pAML relapse in drug-resistant clones. Our work establishes the single-cell chromatin accessibility landscape at clonal resolution and reveals the critical involvement of epigenetic disruption, offering insights into classification and targeted therapies of pAML patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e993d1844c9aff473d0effdd0e0a49f56c123ecc" target='_blank'>
              Single-cell epigenetic and clonal analysis decodes disease progression in pediatric acute myeloid leukemia.
              </a>
            </td>
          <td>
            Boyu Cui, Lanlan Ai, Minghui Lei, Yongjuan Duan, Chao Tang, Jingliao Zhang, Yan Gao, Xuan Li, Caiying Zhu, Yingchi Zhang, Xiaofan Zhu, Tomoya Isobe, Wenyu Yang, Berthold Gottgens, Ping Zhu
          </td>
          <td>2024-12-11</td>
          <td>Blood</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Cancer is a multifactorial process involving genetic, epigenetic, physiological, and metabolic changes. The ability of tumours to regulate new reactive pathways is essential for their survival. A key aspect of this involves the decision-making process of cancer cells as they balance the exploitation of surrounding and distant tissues for their own benefit while avoiding the rapid destruction of the host. Nutrition plays a central role in these processes but is inherently limited. Understanding how tumour cells interact with non-tumoural tissues to acquire nutrients is crucial. In this review, we emphasise the utility of Drosophila melanogaster as a model organism for dissecting the complex oncogenic networks underlying these interactions. By studying various levels—from individual tumour cells to systemic markers—we can gain new insights into how cancer adapts and thrives. Moreover, developing innovative technologies, such as high-throughput methods and metabolic interventions, enhances our ability to explore how tumours adapt to different conditions. These technological advances allow us to explore tumour adaptations and open new opportunities for potential therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edc422f043ced142c0695662e09ec939b17ed8e3" target='_blank'>
              Metabolic Adaptations in Cancer and the Host Using Drosophila Models and Advanced Tools
              </a>
            </td>
          <td>
            Ernesto Saez-Carrion, Mario Aguilar-Aragon, Lucía García-López, Maria Dominguez, M. L. Uribe
          </td>
          <td>2024-11-29</td>
          <td>Cells</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="ABSTRACT Diffuse large B‐cell lymphoma (DLBCL) patients that fail to achieve a complete metabolic response with frontline immunochemotherapy have a poor prognosis. Genomic profiling has led to a broader understanding of the molecular drivers in DLBCL, but it is unknown how well current classifiers identify patients that will experience primary treatment resistance (PTR). Using whole exome and RNA sequencing data from newly diagnosed DLBCL patients, we evaluated the genomic landscape of PTR and compared it to that of non‐PTR DLBCL. We found a significant increase in the frequency of TP53 (34% vs. 15%, p = 0.005) and ARID1A mutations (21% vs. 7%, p = 0.007) in PTR cases, with pathway analysis further demonstrating a downregulation of TP53 and an increase in chromatin modifying pathways. These results suggest that TP53 and ARID1A may be key mediators of PTR and important pathways contributing to the poor outcomes. We found that the current molecular classifiers were unable to identify PTR cases at diagnosis. However, our newly identified high‐risk signature identified 46% of PTR cases at diagnosis. Overall, these results contribute to our understanding of the genomic landscape of patients with primary treatment resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8dad9b8db809f309807b25afc6dc6fdb4c12b2eb" target='_blank'>
              Molecular Features of Diffuse Large B‐Cell Lymphoma Associated With Primary Treatment Resistance
              </a>
            </td>
          <td>
            A. M. Bock, K. Wenzl, J. Novak, M. Stokes, M. Hopper, J. Krull, A. Dropik, V. Sarangi, María Ortiz, Nicholas Stong, C. Huang, M. Maurer, Rebecca L. King, U. Farooq, Yucai Wang, T. Witzig, S. Ansell, T. Habermann, J. Cerhan, A. K. Gandhi, Grzegorz Nowakowski, A. Novak
          </td>
          <td>2024-11-29</td>
          <td>Hematological Oncology</td>
          <td>0</td>
          <td>128</td>
        </tr>

        <tr id="We conducted an investigation into the characteristics of single-cell differentiation data in gliomas, with a focus on developing DAPK1-based prognostic markers to predict patient outcomes. Dysregulated expression of DAPK1 has been associated with the invasive behavior of various malignancies, including gliomas. However, the precise role and underlying mechanisms of DAPK1 in gliomas remain inadequately understood.We performed analyses on RNA-seq and microarray datasets from The Cancer Genome Atlas (TCGA) and Gene Expression Omnibus (GEO), in addition to single-cell RNA sequencing (scRNA-seq) data from glioma patients available in GEO. Utilizing the Seurat R package, we identified gene clusters associated with survival from the scRNA-seq data. Prognostic models were developed using LASSO and stepwise regression algorithms. Furthermore, we assessed the predictive potential of these genes within the immune microenvironment and their relevance in immunotherapy contexts.Our scRNA-seq data analysis revealed 32 distinct cell clusters corresponding to 10 cell types. Through dimensionality reduction and clustering, we identified three glial cell subpopulations based on their differentiation trajectories. DAPK1, serving as a marker gene for the terminal subpopulation, exhibited an association with poor prognosis.DAPK1-based prognostic models show promise for accurately predicting outcomes in glioblastoma and glioma. An in-depth examination of DAPK1’s specific mechanisms in glioblastoma could elucidate its role in immunotherapy response. Targeting the DAPK1 gene may offer therapeutic benefits for glioma patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af0fd67f17577e9d1b4649b5fe0ecd703f94585d" target='_blank'>
              Single-cell sequencing uncovers the mechanistic role of DAPK1 in glioma and its diagnostic and prognostic implications
              </a>
            </td>
          <td>
            , Yan-Yu Ding, Si-Guo Zhao, Jie-Hui Zhao, Yu Gu, , Fang Zhang, 
          </td>
          <td>2025-01-24</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) play a central role in tumor progression, recurrence, and resistance to conventional therapies, making them a critical focus in oncology research. This review provides a comprehensive analysis of CSC biology, emphasizing their self-renewal, differentiation, and dynamic interactions with the tumor microenvironment (TME). Key signaling pathways, including Wnt, Notch, and Hedgehog, are discussed in detail to highlight their potential as therapeutic targets. Current methodologies for isolating CSCs are critically examined, addressing their advantages and limitations in advancing precision medicine. Emerging technologies, such as CRISPR/Cas9 and single-cell sequencing, are explored for their transformative potential in unraveling CSC heterogeneity and informing therapeutic strategies. The review also underscores the pivotal role of the TME in supporting CSC survival, promoting metastasis, and contributing to therapeutic resistance. Challenges arising from CSC-driven tumor heterogeneity and dormancy are analyzed, along with strategies to mitigate these barriers, including novel therapeutics and targeted approaches. Ethical considerations and the integration of artificial intelligence in designing CSC-specific therapies are discussed as essential elements of future research. The manuscript advocates for a multi-disciplinary approach that combines innovative technologies, advanced therapeutics, and collaborative research to address the complexities of CSCs. By bridging existing gaps in knowledge and fostering advancements in personalized medicine, this review aims to guide the development of more effective cancer treatment strategies, ultimately improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99151138c6fc7abf05753313b772a7f6497b1128" target='_blank'>
              Deciphering the Role of Cancer Stem Cells: Drivers of Tumor Evolution, Therapeutic Resistance, and Precision Medicine Strategies
              </a>
            </td>
          <td>
            Mohamed El-Tanani, S. Rabbani, S. Satyam, I. Rangraze, A. Wali, Yahia El-Tanani, Alaa A. A. Aljabali
          </td>
          <td>2025-01-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Background Tumor is a complex and dynamic ecosystem formed by the interaction of numerous diverse cells types and the microenvironments they inhabit. Determining how cellular states change and develop distinct cellular communities in response to the tumor microenvironment is critical to understanding cancer progression. Tumour-associated macrophages (TAMs) are an important component of the tumour microenvironment and play a crucial role in cancer progression. This study was designed to identify cell-state-specific M2 macrophage markers associated with gastric cancer (GC) prognosis through integrative analysis of single-cell RNA sequencing (scRNA-seq) and bulk RNA-seq data using a machine learning framework named EcoTyper. Results The results showed that TAMs were classified into M1 macrophages, M2 macrophages, monocytes, undefined macrophages and dendritic cells, with M2 macrophages predominating. EcoTyper assigned macrophages to different cell states and ecotypes. A total of 168 cell-state-specific M2 macrophage markers were obtained by integrative analysis of scRNA-seq and bulk RNA-seq data. These markers could categorize GC patients into two clusters (clusters A and B) with different survival and M2 macrophages infiltration abundance. Cell adhesion molecules, cytokine-cytokine receptor interaction, JAK/STAT pathway, MAPK pathway were significantly enriched in cluster A, which had worse survival and higher M2 macrophages infiltration. Conclusion In conclusion, this study profiles a single-cell atlas of intratumor heterogeneity and defines the cell states and ecotypes of TAMs in GC. Furthermore, we have identified prognostically relevant cell-state-specific M2 macrophage markers. These findings provide novel insights into the tumor ecosystem and cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0d8a2fb2bd7548dde928bcb962ded6e0380b6c3" target='_blank'>
              Integrated Analysis of Single-Cell and Bulk RNA-Sequencing Based on EcoTyper Machine Learning Framework Identifies Cell-State-Specific M2 Macrophage Markers Associated with Gastric Cancer Prognosis
              </a>
            </td>
          <td>
            A-Kao Zhu, Guang-Yao Li, Fang-Ci Chen, Jia-Qi Shan, Yu-Qiang Shan, Chen-Xi Lv, Zhi-Qiang Zhu, Yi-Ren He, Lu-Lu Zhai
          </td>
          <td>2024-12-01</td>
          <td>ImmunoTargets and Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="ABSTRACT Background Acute lymphocytic leukemia (ALL), characterized by uncontrolled growth of abnormal lymphocytes, predominantly affects children. Genetic analysis focusing on genes and microRNAs reveals important information about the biology of ALL, enabling accurate diagnosis and treatment. This study examines gene and microRNA expression in B cell ALL to improve early diagnosis and personalized treatment. Methods Bone marrow samples were collected from patients both before and after the induction phase of chemotherapy. Comprehensive diagnostic techniques including flow cytometry, molecular assays, real‐time PCR for common translocations, karyotyping, and complete blood count (CBC) analysis were employed. These methods were utilized to determine the type and risk assessment of ALL, identify specific gene and microRNA expressions, and measure blood cell counts. Results The study comprised 12 patients, all under the age of 18. Post‐treatment RT‐PCR analysis revealed significant reductions in the expression of the ABCB1 gene, miR‐129‐5p, and miR‐9‐5p following the induction phase of chemotherapy. Karyotype analysis indicated that two patients were hypodiploid; unfortunately, both of these patients did not survive. Conclusion MicroRNAs and ABC genes serve as predictive and prognostic biomarkers in Acute Lymphoblastic Leukemia (ALL) and should be carefully reconsidered. It is more accurate to state that while microRNAs and ABC genes may potentially influence treatment response in ALL, further research is crucial to fully understand their roles in determining treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36d6c6bf058c1b51d999c0b567643ee6cccfac4f" target='_blank'>
              ATP‐Binding Cassette Transporter Genes and microRNAs in Pediatric B‐Cell ALL: Expression Insights
              </a>
            </td>
          <td>
            Reza Nekouian, Pooya Faranoush, Fatemeh Khesali, Parisa Shams, M. Foroughi‐Gilvaee, Negin Sadighnia, Dorsa Fallah Azad, MohammadAli Ehsani, Mohammad Faranoush
          </td>
          <td>2024-12-23</td>
          <td>Journal of Clinical Laboratory Analysis</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Immunomodulatory agents (IMiDs) are a major class of drugs for treating multiple myeloma (MM); however, acquired resistance to IMiDs remains a significant clinical challenge. While alterations in cereblon (CRBN) and its pathway are known to contribute to IMiD resistance, they account for only 20-30% of cases, and the underlying mechanisms in the majority of the resistance cases remain unclear. Here, we identified ADAR1 as a novel driver of lenalidomide resistance in MM. We showed that lenalidomide activates the MDA5-mediated dsRNA-sensing pathway in MM cells, leading to interferon (IFN)-mediated apoptosis, with ADAR1 as the key regulator. Mechanistically, ADAR1 loss increased lenalidomide sensitivity through endogenous dsRNA accumulation, which in turn triggered dsRNA-sensing pathways and enhanced IFN responses. Conversely, ADAR1 overexpression reduced lenalidomide sensitivity, attributed to increased RNA editing frequency, reduced dsRNA accumulation and suppression of the dsRNA-sensing pathways. In summary, we report the involvement of ADAR1-regulated dsRNA-sensing in modulating lenalidomide sensitivity in MM. These findings highlight a novel RNA-related mechanism underlying lenalidomide resistance and underscore the potential of targeting ADAR1 as a novel therapeutic strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0d2cf7eab8ed9db1ca8ad24d91b4f52aff132ba" target='_blank'>
              ADAR1-regulated cytoplasmic dsRNA-sensing pathway is a novel mechanism of lenalidomide resistance in multiple myeloma.
              </a>
            </td>
          <td>
            Mun Yee Koh, Tae-Hoon Chung, Nicole Xin Ning Tang, S. Toh, Jianbiao Zhou, T. Tan, Leilei Chen, W. Chng, Phaik Ju Teoh
          </td>
          <td>2024-12-09</td>
          <td>Blood</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="Cellular differentiation and homeostasis rely on complex mechanisms to control gene expression, enabling the different cell lineages of an organism to establish and then "memorize" different epigenetic states. The processes that control gene expression are centered on chromatin, a complex of DNA, histone proteins and RNA, whose structure is finely regulated. Targeted epigenomic engineering tools make it possible to interfere with and study these processes, revealing the logic of epigenetic memory mechanisms. This article reviews the main classes of targeted epigenome modification tools and illustrates how they can be used to better understand and modify the epigenome of cells, paving the way for potentially revolutionary therapeutic prospects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54bf48ee21647af051d470ca1e8d40b7d4279ad6" target='_blank'>
              [Targeted epigenome engineering].
              </a>
            </td>
          <td>
            Hedvika Martin, Michel Wassef
          </td>
          <td>2024-12-01</td>
          <td>Medecine sciences : M/S</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Immune checkpoint inhibitors (ICIs) have shown promise in treating triple-negative breast cancer (TNBC), but resistance to these therapies remains a significant challenge. AT-rich interactive domain 1A (ARID1A), a component of the SWItch/Sucrose Non-Fermentable chromatin remodeling complex, is frequently mutated in TNBC and is associated with increased programmed cell death ligand 1 expression, which contributes to immune evasion. Paradoxically, this mutation may make TNBC potentially more responsive to ICIs. Chromatin-mediated gene expression requires a balance between ARID1A and enhancer of zeste homolog 2 (EZH2), a histone methyltransferase, and ARID1A deficiency results in enhanced EZH2 activity, contributing to various oncologic processes. Epigenetic modulation through EZH2 inhibition could exploit the synthetic lethality between ARID1A deficiency and EZH2 activity, which may reduce the immunosuppressive tumor microenvironment and enhance infiltration and activity of cytotoxic T-cells within the tumor, thereby synergizing with immune checkpoint inhibition. This review explores the potential of EZH2 inhibition as a therapeutic strategy to overcome immune checkpoint resistance in ARID1A-deficient TNBC. In addition, the role of ARID1A deficiency as a radiosensitizer is also discussed in the context of combination therapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3e104e567f4ebbcd4a524a3c9a3d48c826aa2e5" target='_blank'>
              Targeting EZH2 to mitigate immune checkpoint resistance in ARID1A-deficient triple-negative breast cancer
              </a>
            </td>
          <td>
            Lauren Lukas, Hye Ri Han, Evanthia T. Roussos Torres, Aaron G. Baugh, Oliver Bell, Jason C. Ye, Kenneth Wong, Vyshnavi Pachipulusu, Hualin Zhang, Alan L. Epstein
          </td>
          <td>2025-01-16</td>
          <td>Advances in  Radiotherapy &amp; Nuclear Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d60a3bedb4bf1a029a736f9136afc2a07aef47f" target='_blank'>
              Decoding the functional impact of the cancer genome through protein-protein interactions.
              </a>
            </td>
          <td>
            Haian Fu, Xiulei Mo, Andrey A. Ivanov
          </td>
          <td>2025-01-14</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="The complex signaling network within the breast tumor microenvironment is crucial for its growth, metastasis, angiogenesis, therapy escape, stem cell maintenance, and immunomodulation. An array of secretory factors and their receptors activate downstream signaling cascades regulating breast cancer progression and metastasis. Among various signaling pathways, the EGFR, ER, Notch, and Hedgehog signaling pathways have recently been identified as crucial in terms of breast cancer proliferation, survival, differentiation, maintenance of CSCs, and therapy failure. These receptors mediate various downstream signaling pathways such as MAPK, including MEK/ERK signaling pathways that promote common pro-oncogenic signaling, whereas dysregulation of PI3K/Akt, Wnt/β-catenin, and JAK/STAT activates key oncogenic events such as drug resistance, CSC enrichment, and metabolic reprogramming. Additionally, these cascades orchestrate an intricate interplay between stromal cells, immune cells, and tumor cells. Metabolic reprogramming and adaptations contribute to aggressive breast cancer and are unresponsive to therapy. Herein, recent insights into the novel signaling pathways operating within the breast TME that aid in their advancement are emphasized and current developments in practices targeting the breast TME to enhance treatment efficacy are reviewed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/989fb511b18a1724eebed53dc79d3ad283387011" target='_blank'>
              Molecular Insights on Signaling Cascades in Breast Cancer: A Comprehensive Review.
              </a>
            </td>
          <td>
            Venketesh K. Panda, Barnalee Mishra, Samikshya Mahapatra, Biswajit Swain, Diksha Malhotra, Suryendu Saha, Sinjan Khanra, Priyanka Mishra, Sambhunath Majhi, Kavita Kumari, Angitha N. Nath, Swarnali Saha, Sarmistha Jena, Gopal C. Kundu
          </td>
          <td>2025-01-13</td>
          <td>Cancers</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Currently, numerous associations between genetic polymorphisms and various diseases have been characterized through Genome-Wide Association Studies. The majority of clinically significant polymorphisms are localized in non-coding regions of the genome. While modern bioinformatic resources make it possible to predict molecular mechanisms that explain the influence of non-coding polymorphisms on gene expression, such hypotheses require experimental verification. This review discusses the methods for elucidating the molecular mechanisms underlying the dependence of disease pathogenesis on specific genetic variants within non-coding sequences. A particular focus is on the methods to identify the transcription factors with binding efficiency contingent upon polymorphic variations. Despite remarkable progress in the bioinformatic resources enabling the prediction of the impact of polymorphisms on disease pathogenesis, the imperative for experimental approaches to this inquiry still persists.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0e300f57318f3e11948f7c4bd85a221591ff36c" target='_blank'>
              Methods for Functional Characterization of Genetic Polymorphisms of Non-Coding Regulatory Regions of the Human Genome (Review)
              </a>
            </td>
          <td>
            A. Uvarova, E. A. Tkachenko, E. M. Stasevich, E. A. Zheremyan, K. Korneev, D. V. Kuprash
          </td>
          <td>2024-12-25</td>
          <td>Biohimiâ</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Introduction Despite remarkable progress in Pediatric Acute Myeloid Leukemia (pAML) treatments, the relapsed disease remains difficult to treat, making it pertinent to identify novel biomarkers of prognostic/therapeutic significance. Material and methods Bone marrow samples from 21 pAML patients were analyzed using single cell RNA sequencing, functional assays with ARMH1 knockdown and overexpression were performed in leukemia cell lines to evaluate impact on proliferation and migration, and chemotherapy sensitivity. Mitochondrial function was assessed via Seahorse assay, ARMH1 interacting proteins were studied using co-immunoprecipitation. Bulk RNA-seq was performed on ARMH1knockdown and over expressing cell lines to evaluate the pathways and networks impacted by ARMH1. Results Our data shows that ARMH1, a novel cancer-associated gene, is highly expressed in the malignant blast cells of multiple pediatric hematologic malignancies, including AML, T/B-ALL, and T/B-MPAL. Notably, ARMH1 expression is significantly elevated in blast cells of patients who relapsed or have a high-risk cytogenetic profile (MLL) compared to standard-risk (RUNX1, inv (16)). ARMH1 expression is also significantly correlated with the pediatric leukemia stem cell score of 6 genes (LSC6) associated with poor outcomes. Perturbation of ARMH1 (knockdown and overexpression) in leukemia cell lines significantly impacted cell proliferation and migration. The RNA-sequencing analysis on multiple ARMH1 knockdown and overexpressing cell lines established an association with mitochondrial fatty acid synthesis and cell cycle pathways.The investigation of the mitochondrial matrix shows that pharmacological inhibition of a key enzyme in fatty acid synthesis regulation, CPT1A, resulted in ARMH1 downregulation. ARMH1 knockdown also led to a significant reduction in CPT1A and ATP production as well as Oxygen Consumption Rate. Our data indicates that downregulating ARMH1 impacts cell proliferation by reducing key cell cycle regulators such as CDCA7 and EZH2. Further, we also established that ARMH1 is a key physical interactant of EZH2, associated with multiple cancers. Conclusion Our findings underscore further evaluation of ARMH1 as a potential candidate for targeted therapies and stratification of aggressive pAML to improve outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f9c3fb31bae52e61839b92c250ccd940a2b13b6" target='_blank'>
              ARMH1 is a novel marker associated with poor pediatric AML outcomes that affect the fatty acid synthesis and cell cycle pathways
              </a>
            </td>
          <td>
            Mojtaba Bakhtiari, Sean C Jordan, Hope L. Mumme, Richa Sharma, Mala Shanmugam, Swati S. Bhasin, Manoj K. Bhasin
          </td>
          <td>2024-12-05</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1708204ce5d02d098867433430233edb7d10ec3" target='_blank'>
              Profiling the epigenome using long-read sequencing.
              </a>
            </td>
          <td>
            Tianyuan Liu, Ana Conesa
          </td>
          <td>2025-01-08</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Cancer is a complex and multifaceted group of diseases characterized by highly intricate mechanisms of tumorigenesis and tumor progression, which complicates diagnosis, prognosis, and treatment. In recent years, targeted therapies have gained prominence by focusing on specific mutations and molecular features unique to each tumor type, offering more effective and personalized treatment options. However, it is equally critical to explore the genetic commonalities across different types of cancer, which has led to the rise of pan-cancer studies. These approaches help identify shared therapeutic targets across various tumor types, enabling the development of broader and potentially more widely applicable treatment strategies. This review aims to provide a comprehensive overview of key concepts related to tumors, including tumorigenesis processes, the tumor microenvironment, and the role of extracellular vesicles in tumor biology. Additionally, we explore the molecular interactions and mechanisms driving tumor progression, with a particular focus on the pan-cancer perspective. To achieve this, we conducted an in silico analysis using publicly available datasets, which facilitated the identification of both common and divergent genetic and molecular patterns across different tumor types. By integrating these diverse areas, this review offers a clearer and deeper understanding of the factors influencing tumorigenesis and highlights potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a5b3ea0ad38a337f0df54ee493320c36455e356" target='_blank'>
              Exploring the Complexity of Pan-Cancer: Gene Convergences and in silico Analyses
              </a>
            </td>
          <td>
            Leandro Teodoro, Ana Claudia Carreira, M.C. Sogayar
          </td>
          <td>2024-12-01</td>
          <td>Breast Cancer : Targets and Therapy</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92776a13bf2bfc07c4e1f9b75c01119340528b43" target='_blank'>
              Elevated mutation rates in multi-azole resistant Aspergillus fumigatus drive rapid evolution of antifungal resistance
              </a>
            </td>
          <td>
            Michael J. Bottery, N. van Rhijn, H. Chown, J. Rhodes, Brandi N. Celia-Sanchez, Marin T. Brewer, M. Momany, Matthew C. Fisher, Christopher G. Knight, M. Bromley
          </td>
          <td>2024-12-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/103b75e315d591fa4510717dd10f578400ede160" target='_blank'>
              Central control of dynamic gene circuits governs T cell rest and activation
              </a>
            </td>
          <td>
            Maya M. Arce, Jennifer M. Umhoefer, Nadia Arang, Sivakanthan Kasinathan, Jacob W. Freimer, Zachary Steinhart, Haolin Shen, Minh T N Pham, Mineto Ota, Anika Wadhera, Rama Dajani, Dmytro Dorovskyi, Y. Chen, Qi Liu, Yuan Zhou, D. Swaney, K. Obernier, B. Shy, J. Carnevale, Ansuman T. Satpathy, N. Krogan, Jonathan K. Pritchard, Alex Marson
          </td>
          <td>2024-12-11</td>
          <td>Nature</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Aggressive variant prostate cancer (AVPC) is characterized by a molecular signature involving combined defects in TP53, RB1, and/or PTEN (AVPC-TSGs), identifiable through immunohistochemistry or genomic analysis. The reported prevalence of AVPC-TSG alterations varies widely, reflecting differences in assay sensitivity, treatment pressure, and disease stage evolution. Although robust clinical evidence is still emerging, the study of AVPC-TSG alterations in prostate cancer (PCa) is promising. Alterations in TP53, RB1, and PTEN, as well as the combined loss of AVPC-TSGs, may have significant implications for prognosis and treatment. These biomarkers might help predict responses to various therapies, including hormonal treatments, cytotoxic agents, radiotherapy, and targeted therapies. Understanding the impact of these molecular alterations in patients with PCa is crucial for personalized management. In this review, we provide a comprehensive overview of the emerging prognostic and predictive roles of AVPC-TSG alterations across PCa stages. Moreover, we discuss the implications of different methods used for detecting AVPC-TSG alterations and summarize factors influencing their prevalence. As our comprehension of the genomic landscape of PCa disease deepens, incorporating genomic profiling into clinical decision making will become increasingly important for improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc76168bc6cc3f716d4ad9c939472b94406c6f4f" target='_blank'>
              The Emerging Predictive and Prognostic Role of Aggressive-Variant-Associated Tumor Suppressor Genes Across Prostate Cancer Stages
              </a>
            </td>
          <td>
            Martino Pedrani, Jessica Barizzi, Giuseppe Salfi, Alessandro Nepote, Irene Testi, Sara Merler, Luis Castelo-Branco, R. P. Mestre, Fabio Turco, Luigi Tortola, Jean-Philippe P. Theurillat, S. Gillessen, U. Vogl
          </td>
          <td>2025-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ff4ae0eaa9128342a0dedd747e19622d92c7f71" target='_blank'>
              FDFT1 maintains glioblastoma stem cells through activation of the Akt pathway
              </a>
            </td>
          <td>
            Hui Mo, Jiajia Shao, Zhun Li, Peiting Zeng, Xinke Yin, Yongsheng Huang, Peng Wang, Jianwei Liao
          </td>
          <td>2024-12-20</td>
          <td>Stem Cell Research & Therapy</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Epigenetic dysregulation is a hallmark of Acute Myeloid Leukemia (AML), with mutations in DNA Methyltransferases (e.g., DNMT3A) being frequent and promising therapeutic targets. DNMTs form complexes with Histone Methyltransferases (HMTs), driving gene silencing loop via chromatin methylation crosstalk. However, potential connections between this DNMTs/HMTs cooperative activity and oncogenic requirements across the AML mutational spectrum remain poorly understood. Here, we demonstrate that AMLs carrying DNMT3A and Nucleophosmin (NPM1) mutations exhibit a specific epigenetic vulnerability toward a complex formed by DNMTs and G9a, a specific histone H3 Lysine 9 Methyltransferase (H3K9-HMT). Dual inhibition of DNMT/G9a restores differentiation, reduces tumor growth, and spares healthy progenitors compared to standard hypomethylating agents. Mechanistically, DNMT/G9a regulates NPM1 stability, inhibits HOXA9/MEIS1 activity, and triggers interferons (IFN) response via viral mimicry pathways by modulating hypermethylated retrotransposons. Collectively, our data unravel specific epigenetic vulnerabilities within the complex AML mutational landscape and provide a compelling rationale for the design of personalized epigenetic therapies with enhanced efficacy and safer clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33522a862f647024962f40881508708e0269adea" target='_blank'>
              DNMT/G9a Complex Inhibition Uncovers Epigenetic Vulnerabilities and Induces IFN-Response in Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Angelique Schönefeld, Costanza Zanetti, Eric C. Schmitt, Kevin Woods, Ashley Westerback, Sidney M. Mitsch, Frank Rühle, Simge Kelekçi, Lisa Fol, Viral Shah, Miriam Sanchez-Saez, Sophia Thevissen, Alexander Schäffer, Ulrike Regina Schmits, D. Sasca, Thomas Kindler, Catherine Wölfel, Clarissa Feuerstein-Akgoz, Clarissa Gerhäuser, P. Lutsik, J. Rivière, J. Hecker, Katharina S Götze, U. Platzbecker, H. Echchannaoui, Matthias Theobald, C. Plass, Jan Padeken, H. Medyouf, B. Guezguez
          </td>
          <td>2024-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Background Cancer stem cells are characterized by self-renewal, clonal tumor initiation capacity, and treatment resistance, which play essential roles in the tumor progression of prostate cancer (PCa). In this study, we aim to explore the features of cancer stemness and characterize the expression of stem cell-related genes for PCa. Methods We downloaded RNA-seq data and related clinical information from The Cancer Genome Atlas (TCGA) database. The mRNA stemness index (mRNAsi) was analyzed for various clinical features, overall survival (OS), and disease-free survival (DFS), and a weighted gene co-expression network analysis (WGCNA) was performed to identify crucial gene modules and key genes, which may play a role in CSCs. The key gene functions were verified using multiple databases, including the TCGA and Gene Expression Omnibus database (GEO). Next, we explored the potential function of the modules and genes obtained using WGCNA using an enrichment analysis. Finally, we performed in vitro experiments for further verification. Results We found that mRNAsi were higher in PCa tissues than in normal tissues, and the mRNAsi were closely related to the clinical characteristics of PCa. A total of 16 key genes associated with the mRNAsi scores were identified by WGCNA analysis, including NCAPG, NEK2, DLGAP5, CENPA, CENPF, TPX2, GTSE1, KIF4A, NEIL3, CDC25C, UBE2C, CDCA5, MELK, SKA3, NUF2, and BIRC5. These genes were explicitly highly expressed in PCa across TCGA cancers and were validated in 3 independent GEO PCa datasets. The functional annotations of the key genes were linked with the cell proliferation processes. NUF2 may be a potential biomarker for PCa. In vitro experiments showed that knockdown NUF2 reduced the proliferation and migration of PCa cells. Conclusion The 16 key genes identified in this study significantly correlate with PCa stem cell characteristics and showed prognosis-oriented effects in PCa patients. Further, the NUF2 gene may be used as a drug target for treating PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72bc9466fa18820a99adb15c098ec91082dee0a3" target='_blank'>
              NUF2 is associated with cancer stem cell characteristics and a potential drug target for prostate cancer
              </a>
            </td>
          <td>
            Dongxu Zhang, Pu Liang, Qi Wang, Bowen Xia, Liqian Yu, Xiaopeng Hu
          </td>
          <td>2024-12-05</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Human pathogen Streptococcus pneumoniae forms multiple epigenetically and phenotypically distinct intra-populations by invertase PsrA-driven inversions of DNA methyltransferase hsdS genes in the colony opacity-determinant (cod) locus. As manifested by phase switch between opaque and transparent colonies, different genome methylation patterns or epigenomes confer pathogenesis-associated traits, but it is unknown how the pathogen controls the hsdS inversion orientations. Here, we report our finding of the SpxA1-TenA toxin-antitoxin (TA) system that regulates the orientations of hsdS inversions, and thereby bacterial epigenome and associated traits (e.g., colony opacity) by targeting pneumococcal protein synthesis. SpxA1 and TenA were found to constitute a highly conserved type II TA system in S. pneumoniae, primarily based on the observation that overexpressing toxin TenA led to growth arrest in E. coli and enhanced autolysis in S. pneumoniae, and the antitoxin SpxA1 repressed the transcription of the spxA1-tenA operon. When the transcription of tenA was de-repressed by a spontaneous AT di-nucleotide insertion/deletion in the promoter region of the spxA1-tenA operon, TenA bound to the ribosome maturation factor RimM, and thereby reduced the cellular level of alternative sigma factor ComX (known for the activation of natural transformation-associated genes). Attenuation of ComX expression in turn enhanced the transcription of the invertase gene psrA, which favored the formation of the transparent colony phase-associated hsdS allelic configurations in the cod locus. Phenotypically, moderate expression of TenA dramatically reshaped pneumococcal epigenome and colony opacity. Because spontaneous variations frequently occur during bacterial growth in the number of the AT di-nucleotides in the promoter region of the spxA1-tenA operon, this locus acts as a programmed genetic switch that generates pneumococcal subpopulations with epigenetic and phenotypic diversity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd33d7ab497299e98fba78410719174886a039f7" target='_blank'>
              The SpxA1-TenA toxin-antitoxin system regulates epigenetic variations of Streptococcus pneumoniae by targeting protein synthesis
              </a>
            </td>
          <td>
            Shaomeng Wang, Xiu-Yuan Li, Mengran Zhu, Haiteng Deng, Juanjuan Wang, Jing-Ren Zhang
          </td>
          <td>2024-12-01</td>
          <td>PLOS Pathogens</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Colorectal cancer (CRC) has increased at an alarming rate amongst younger (< 50 years) individuals. Such early-onset colorectal cancer (EOCRC) has been particularly notable within the Hispanic/Latino population. Yet, this population has not been sufficiently profiled in terms of two critical elements of CRC -- the MYC proto-oncogene and WNT signaling pathway. Here, we performed a comprehensive multi-omics analysis on 30 early-onset and 37 late-onset CRC ([≥] 50 years) samples from Hispanic/Latino patients. Our analysis included DNA exome sequencing for somatic mutations, somatic copy number alterations, and global and local genetic similarity. Using RNA sequencing, we also assessed differential gene expression, cellular pathways, and gene fusions. We then compared our findings from early-onset Hispanic/Latino patient samples with publicly available data from Non-Hispanic White cohorts. Across all early-onset patients, which had a median 1000 Genomes Project Peruvian-in-Lima-like (1KG-PEL-like) genetic similarity proportion of 60%, we identified 41 WNT pathway genes with significant mutations. Six important examples were APC, TCF7L2, DKK1, DKK2, FZD10, and LRP5. Notably, patients with mutations in DKK1 and DKK2 had the highest 1KG-PEL-like proportion (79%). When we compared the Hispanic/Latino cohort to the Non-Hispanic White cohorts, four of these key genes -- DKK1, DKK2, FZD10, and LRP5 -- were significant in both risk association analyses and differential gene expression. Interestingly, early-onset tumors (vs. late-onset) exhibited distinct somatic copy number alterations and gene expression profiles; the differences included MYC and drug-targetable WNT pathway genes. We also identified a novel WNT gene fusion, RSPO3, in early-onset tumors; it was associated with enhanced WNT signaling. This integrative analysis underscores the distinct molecular features of EOCRC cancer in the Hispanic/Latino population; reveals potential avenues for tailored precision medicine therapies; and emphasizes the importance of multi-omics approaches in studying colorectal carcinogenesis. We expect this data to help contribute towards reducing cancer health disparities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62d0668a8b4dd0379f87ebaa20ad3d4dabb63bc8" target='_blank'>
              Assessment of MYC Gene and WNT Pathway Alterations in Early-Onset Colorectal Cancer Among Hispanic/Latino Patients Using Integrated Multi-Omics Approaches
              </a>
            </td>
          <td>
            F. Carranza, B. Waldrup, Y. Yin, Y. Amzaleg, M. Postel, D. Craig, J. Carpten, B. Salhia, D. Hernandez, N. Gutierrez, C. Ricker, J. Culver, C. Chavez, M. Stern, L. Baezconde-Garbanati, H.-J. Lenz, E. Velazquez-Villarreal
          </td>
          <td>2024-12-08</td>
          <td>None</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="A significant number of castration-resistant prostate cancer (CRPC) evolve into a neuroendocrine (NE) subtype termed NEPC, leading to resistance to androgen receptor (AR) pathway inhibitors and poor clinical outcomes. Through Hi-C analyses of a panel of patient-derived xenograft tumors, here we report drastically different 3D chromatin architectures between NEPC and CRPC samples. Such chromatin re-organization was faithfully recapitulated in vitro on isogenic cells undergoing NE transformation (NET). Mechanistically, neural transcription factor (TF) NKX2-1 is selectively and highly expressed in NEPC tumors and is indispensable for NET across various models. NKX2-1 preferentially binds to gene promoters, but it interacts with chromatin-pioneering factors such as FOXA2 at enhancer elements through chromatin looping, further strengthening FOXA2 binding at NE enhancers. Conversely, FOXA2 mediates regional DNA demethylation, attributing to NE enhancer priming and inducing NKX2-1 expression, forming a feed-forward loop. Single-cell multiome analyses of isogenic cells over time-course NET cells identify individual cells amid luminal-to-NE transformation, exhibiting intermediate epigenetic and transcriptome states. Lastly, NKX2-1/FOXA2 interacts with, and recruits CBP/p300 proteins to activate NE enhancers, and pharmacological inhibitors of CBP/p300 effectively blunted NE gene expression and abolished NEPC tumor growth. Thus, our study reports a hierarchical network of TFs governed by NKX2-1 in regulating the 2D and 3D chromatin re-organization during NET and uncovers a promising therapeutic approach to eradicate NEPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0f60ea05e69028e7c615b1e7620f9deafcd0cdd" target='_blank'>
              Epigenetic remodeling and 3D chromatin reorganization governed by NKX2-1 drive neuroendocrine prostate cancer
              </a>
            </td>
          <td>
            Xiaodong Lu, Viriya Keo, Irina Cheng, Wanqing Xie, Galina A Gritsina, Juan Wang, Q. Jin, Peng Jin, Feng Yue, Martin G. Sanda, Victor Corces, Nicolas Altemose, Jonathan C. Zhao, Jindan Yu
          </td>
          <td>2024-12-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="The accumulation of germline mutations underpins population diversity and drives genetic evolution. Despite the availability of extensive phylogenetic data, the lack of suitable methodologies has hindered the comprehensive characterization of germline mutational processes across evolutionary trees. To address this, we develop a robust three-step methodology that extracts germline mutational processes from alignments of closely related species. First, we estimate regional, branch-specific trinucleotide mutational spectra from a multispecies alignment. Second, we extract mutational processes jointly across an evolutionary clade by analyzing mutation rate variation along the genome using Reciprocal Principal Components Analysis (RPCA). Finally we filter artifactual mutational signatures using DNA symmetry. Applying this method to five primate clades and a rodent outgroup clade revealed nine distinct mutational processes. Notably, five of these processes were consistently observed across all six groups. We identified underling biological mechanism for at least seven of the processes, highlighting phenomena such as biased gene conversion, bulky lesion resolution, and maternal mutagenesis. We validated identified processes using human and non-human polymorphism data. This study offers new insights into the biology and evolution of mutagenesis in primates and introduces a methodological toolkit to investigate mutational processes across phylogenies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08874cb963d779e2ed8115054d6ecf603a380a6a" target='_blank'>
              A Phylogeny-Based Approach to Discover Mutational Processes in Primates
              </a>
            </td>
          <td>
            James C. Kitch, Vladimir Seplyarskiy
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="An important direction in cancer prevention is the development of novel strategies that can be used in the treatment by affecting the mechanisms of the regulatory pathways and genomic elements involved in malignant transformation. Strategy proposed in this study is to initiate cell death mechanisms in response to the depletion of the energy resources within a cancer cell due to the uncontrolled spread of mobile genetic elements throughout its genome. The calculation results obtained using a preliminary mathematical model that simulates cellular bioenergetic balance, taking into account the energy consumption for retrotransposition of the mobile elements such as LINE-1 and SINE, show that this scenario is plausible. Intracellular resources undergo a critical redistribution when affecting the genomic defense mechanisms and the LINE-1 transcription rate. This leads to a sharp increase in energy consumption for retrotransposon transcription that causes a significant decrease in the pool of free ATPs in the cell.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05095e01db5773f04351486017740af52565a16e" target='_blank'>
              Control of the Activity of Mobile Elements in Cancer Cells as a Strategy for Anticancer Therapy
              </a>
            </td>
          <td>
            S. R. Pavlov, V. V. Gursky, M. Samsonova, A. Kanapin, A. A. Samsonova
          </td>
          <td>2024-12-15</td>
          <td>Biofizika</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Formation of templated insertions at DNA double-strand breaks (DSBs) is very common in cancer cells. The mechanisms and enzymes regulating these events are largely unknown. Here, we investigated templated insertions in yeast at DSBs using amplicon sequencing across a repaired locus. We document very short (most ∼5–34 bp), templated inverted duplications at DSBs. They are generated through a foldback mechanism that utilizes microhomologies adjacent to the DSB. Enzymatic requirements suggest a hybrid mechanism wherein one end requires Polδ-mediated synthesis while the other end is captured by nonhomologous end joining (NHEJ) or by alternative end joining (Alt-EJ). This process is exacerbated in mutants with low levels or mutated RPA (rtt105Δ; rfa1-t33) or extensive resection deficiency (sgs1Δ exo1Δ). Templated insertions from various distant genomic locations also increase in RPA mutants as well as in rad27Δ and originate from fragile regions of the genome. Among complex insertions, common events are insertions of two sequences, originating from the same locus and with inverted orientation. We propose that these inversions are also formed by microhomology-mediated template switching. Together, we propose that a shortage of RPA, typical in cancer cells, may be a factor that stimulates the formation of templated insertions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63e3eaa71a9c5239af5efc218e8d2317e017347b" target='_blank'>
              RPA and Rad27 limit templated and inverted insertions at DNA breaks
              </a>
            </td>
          <td>
            Yang Yu, Xin Wang, Jordan Fox, Qian Li, Yang Yu, P. Hastings, Kaifu Chen, Grzegorz Ira
          </td>
          <td>2024-12-03</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e6b74de7021622cacaa229773949fb0310d8a9a" target='_blank'>
              A practice-oriented genome-profiling study for acute myeloid leukemia using the novel HANDLE system: HM-screen-JAPAN02.
              </a>
            </td>
          <td>
            H. Arai, N. Hosono, Sung-Gi Chi, Kentaro Fukushima, Daisuke Ikeda, Satoshi Iyama, Akihiko Gotoh, Takayuki Ikezoe, Chikashi Yoshida, Goichi Yoshimoto, Junya Kanda, Naoto Takahashi, E. Sakaida, Kensuke Usuki, Takahiro Yamauchi, Yosuke Minami
          </td>
          <td>2024-12-16</td>
          <td>International journal of hematology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9342acbba0e99dc63b9eedf38a334853e0c4e4e7" target='_blank'>
              The role of mitophagy-related genes in prognosis and immunotherapy of cutaneous melanoma: a comprehensive analysis based on single-cell RNA sequencing and machine learning.
              </a>
            </td>
          <td>
            Jun Tian, Lei Zhang, Kexin Shi, Li Yang
          </td>
          <td>2025-01-11</td>
          <td>Immunologic research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The resistance of tumors to immune checkpoint inhibitors (ICI) may be intricately linked to cellular senescence, although definitive clinical validation remains elusive. In this study, comprehensive pan-cancer scRNA-seq analyses identify fibroblasts as exhibiting the most pronounced levels of cellular senescence among tumor-associated cell populations. To elucidate this phenomenon, a fibroblast senescence-associated transcriptomic signature (FSS), which correlated strongly with protumorigenic signaling pathways and immune dysregulation that fosters tumor progression, is developed. Leveraging the FSS, the machine learning (ML) framework demonstrates exceptional accuracy in predicting ICI response and survival outcomes, achieving superior area under curve (AUC) values across validation, testing, and in-house cohorts. Strikingly, FSS consistently outperforms established signatures in predictive robustness across diverse cancer subtypes. From an integrative analysis of 17 CRISPR/Cas9 libraries, CDC6 emerges as a pivotal biomarker for pan-cancer ICI response and prognostic stratification. Mechanistically, experimental evidence reveals that CDC6 in tumor cells orchestrates fibroblast senescence via TGF-β1 secretion and oxidative stress, subsequently reprogramming the tumor microenvironment and modulating ICI response. These findings underscore the translational potential of targeting fibroblast senescence as a novel therapeutic strategy to mitigate immune resistance and enhance antitumor efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83eb29659bd661ae208e4642398f8b3b7069e404" target='_blank'>
              A Distinguished Roadmap of Fibroblast Senescence in Predicting Immunotherapy Response and Prognosis Across Human Cancers.
              </a>
            </td>
          <td>
            Dongjie Chen, Pengyi Liu, Jiayu Lin, Longjun Zang, Yihao Liu, Shuyu Zhai, Xiongxiong Lu, Yuanchi Weng, Hongzhe Li
          </td>
          <td>2024-12-30</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Simple Summary Cancer cells arise from multiple complementary genomic and epigenomic abnormalities that deregulate the pathways controlling cell proliferation, cell survival, and tissue homeostasis. Epigenetic modifications of histones, DNA, chromatin structure, and non-coding RNA expression are emerging as critical determinants of gene expression and essential drivers of tumor cell heterogeneity and neoplastic cell phenotypes. Multipotent stem-like cells (also referred to as cancer stem cells, CSCs) are critical determinants of tumor propagation, therapeutic resistance, and recurrence following treatment. However, our understanding of how these multidimensional epigenetic modifications cooperatively drive cancer cell stemness and how to efficaciously target them is currently inadequate to positively impact patient outcomes. Abstract Background/Objectives: CSCs are critical drivers of the tumor and stem cell phenotypes of glioblastoma (GBM) cells. Chromatin modifications play a fundamental role in driving a GBM CSC phenotype. The goal of this study is to further our understanding of how stem cell-driving events control changes in chromatin architecture that contribute to the tumor-propagating phenotype of GBM. Methods: We utilized computational analyses to identify a subset of clinically relevant genes that were predicted to be repressed in a Polycomb repressive complex 2 (PRC2)-dependent manner in GBM upon induction of stem cell-driving events. These associations were validated in patient-derived GBM neurosphere models using state-of-the-art molecular techniques to express, silence, and measure microRNA (miRNA) and gene expression changes. Advanced Poly(β-amino ester) nanoparticle formulations (PBAEs) were used to deliver miRNAs in vivo to orthotopic human GBM tumor models. Results: We show that glioma stem cell (GSC) formation and tumor propagation involve the crosstalk between multiple epigenetic mechanisms, resulting in the repression of the miRNAs that regulate PRC2 function and histone H3 lysine 27 tri-methylation (H3K27me3). We also identified miR-217-5p as an EZH2 regulator repressed in GSCs and showed that miR-217-5p reconstitution using advanced nanoparticle formulations re-activates the PRC2-repressed genes, inhibits GSC formation, impairs tumor growth, and enhances the effects of ionizing radiation in an orthotopic model of GBM. Conclusions: These findings suggest that inhibiting PRC2 function by targeting EZH2 with miR-217-5p advanced nanoparticle formulations could have a therapeutic benefit in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a650a5c81b8e5180cf08ed0736baa5ed09de3eb0" target='_blank'>
              miR-217-5p NanomiRs Inhibit Glioblastoma Growth and Enhance Effects of Ionizing Radiation via EZH2 Inhibition and Epigenetic Reprogramming
              </a>
            </td>
          <td>
            Jack Korleski, Sweta Sudhir, Yuan Rui, Christopher Caputo, Sophie Sall, Amanda L. Johnson, Harmon Khela, Tanmaya Madhvacharyula, Anisha Rasamsetty, Yunqing Li, B. Lal, Weiqiang Zhou, Karen Smith-Connor, Stephany Tzeng, Jordan Green, J. Laterra, H. Lopez-Bertoni
          </td>
          <td>2024-12-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fc6bfc5e3065b396e263e4793dde2e7908cde4e" target='_blank'>
              CRISPR-StAR enables high-resolution genetic screening in complex in vivo models.
              </a>
            </td>
          <td>
            E. Uijttewaal, Joonsun Lee, Annika Charlotte Sell, Naomi Botay, Gintautas Vainorius, Maria Novatchkova, J. Baar, Jiaye Yang, Tobias Potzler, Sophie van der Leij, Christopher Lowden, Julia Sinner, Anais Elewaut, Milanka Gavrilovic, Anna Obenauf, Daniel Schramek, Ulrich Elling
          </td>
          <td>2024-12-16</td>
          <td>Nature biotechnology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d182842a9c5b9ec9f888e5c17ac019a50f732fb3" target='_blank'>
              Beginning at the ends: telomere and telomere-based cancer therapeutics.
              </a>
            </td>
          <td>
            Zahra Sadr, Masoumeh Ghasemi, Soheyla Jafarpour, Reyhaneh Seyfi, Aida Ghasemi, Elham Boustanipour, H. Khorshid, Naeim Ehtesham
          </td>
          <td>2024-12-06</td>
          <td>Molecular genetics and genomics : MGG</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Histones are key epigenetic factors for regulating the accessibility and compaction of eukaryotic genomes, affecting the replication, repair, and expression of DNA. Recent studies have demonstrated that histone missense mutations can perturb normal histone function, promoting the development of phenotypically distinguishable cancers. However, most histone mutations observed in cancer patients remain enigmatic in their potential to promote cancer development. To assess the oncogenic potential of histone missense mutations, we have gathered whole-exome sequencing data for the tumors of over 12,000 patients. Overall, histone mutations occurred in about 16% of cancer patients, although specific cancer types showed substantially higher rates. Using a combination of genomic, structural, and biophysical analyses, we found several predominant modes of action, where cancer missense mutations in histones affected acidic patches and protein binding interfaces in a cancer-specific manner and targeted interaction sites with specific DNA repair proteins. Consistent with this finding, we observed a high tumour mutational burden in patients with histone mutations affecting interactions of DNA repair proteins. We also identified potential cancer driver mutations in several histone genes, including histone H4-a highly conserved histone without previously documented driver mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00566c578efc8bbf332002783a58fe37b3053b70" target='_blank'>
              Cancer Histone Mutations Impact Binding and DNA Repair Processes, Leading to Increased Mutagenesis
              </a>
            </td>
          <td>
            Daniel Espiritu, Yiru Sheng, Yunhui Peng, Daria Ostroverkhova, Shuxiang Li, David Landsman, Maria J. Aristizabal, Anna R Panchenko
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Discovered more than four decades ago, nonsense-mediated mRNA decay (NMD) plays a fundamental role in the regulation of gene expression and is a major contributor to numerous diseases. With advanced technologies, several novel approaches aim to directly circumvent the effects of disease-causing frameshift and nonsense mutations. Additional therapeutics aim to globally dampen the NMD pathway in diseases associated with pathway hyperactivation, one example being Fragile X Syndrome. In other cases, therapeutics have been designed to hijack or inhibit the cellular NMD machinery to either activate or obviate transcript-specific NMD by modulating pre-mRNA splicing. Here, we discuss promising approaches employed to regulate NMD for therapeutic purposes and highlight potential challenges in future clinical development. We are optimistic that the future of developing target-specific and global modulators of NMD (inhibitors as well as activators) is bright and will revolutionize the treatment of many genetic disorders, especially those with high unmet medical need.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f8d79dce5cfe95746b7b429088778df04c8485b" target='_blank'>
              Exploring the therapeutic potential of modulating nonsense-mediated mRNA decay.
              </a>
            </td>
          <td>
            Mary McMahon, L. Maquat
          </td>
          <td>2024-12-12</td>
          <td>RNA</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="Philadelphia chromosome-like B-cell acute lymphoblastic leukemia (Ph-like ALL) is driven by genetic alterations that induce constitutive kinase signaling and is associated with chemoresistance and high relapse risk in children and adults. Preclinical studies in the most common CRLF2-rearranged/JAK pathway-activated Ph-like ALL subtype have shown variable responses to JAK inhibitor-based therapies, suggesting incomplete oncogene addiction and highlighting a need to elucidate alternative biologic dependencies and therapeutic vulnerabilities, while the ABL-class Ph-like ALL subtype appears preferentially sensitive to SRC/ABL- or PDGFRB-targeting inhibitors. Which patients may be responsive versus resistant to tyrosine kinase inhibitor (TKI)-based precision medicine approaches remains a critical knowledge gap. Using bulk and single-cell multiomics analyses, we profiled residual cells from CRLF2-rearranged or ABL1-rearranged Ph-like ALL patient-derived xenograft models treated in vivo with targeted inhibitors to identify TKI-resistant subpopulations and potential mechanisms of therapeutic escape. We detected a specific MYC dependency in Ph-like ALL cells and defined a new leukemia cell subpopulation with senescence-associated stem cell-like features regulated by AP-1 transcription factors. This dormant ALL subpopulation was effectively eradicated by dual pharmacologic inhibition of BCL-2 and JAK/STAT or SRC/ABL pathways, a clinically-relevant therapeutic strategy. Single cell-derived molecular signatures of this senescence and stem/progenitor-like subpopulation further predicted poor clinical outcomes associated with other high-risk genetic subtypes of childhood B-ALL and thus may have broader prognostic applicability beyond Ph-like ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b87bb41065d73fc1bc25a49641c54717955deab" target='_blank'>
              Targeting Senescent Stemlike Subpopulations in Philadelphia Chromosome-Like Acute Lymphoblastic Leukemia.
              </a>
            </td>
          <td>
            Yang-yang Ding, Jonathan H Sussman, Kellyn Madden, J. Loftus, Robert K Chen, C. D. Falkenstein, D. A. Barcenas Lopez, David A Hottman, Benjamin Mathier, Wenbao Yu, Jason Xu, Changya Chen, Chia-Hui Chen, Bing He, S. Bandyopadhyay, Zhan Zhang, DongGeun Lee, Hong Wang, Junmin Peng, Chi V. Dang, Kai Tan, S. Tasian
          </td>
          <td>2025-01-07</td>
          <td>Blood</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Cancer, a complex and heterogeneous disease, continues to be a major global health concern. Despite advancements in diagnostics and therapeutics, the aggressive nature of certain cancers remain a significant challenge, necessitating a deeper understanding of the underlying molecular mechanisms driving their severity and progression. Cancer severity and progression depend on cellular properties such as cell migration, cell division, cell shape changes, and intracellular transport, all of which are driven by dynamic cellular microtubules. Dynamic properties of microtubules, in turn, are regulated by an array of proteins that influence their stability and growth. Among these regulators, End Binding (EB) proteins stand out as critical orchestrators of microtubule dynamics at their growing plus ends. Beyond their fundamental role in normal cellular functions, recent research has uncovered compelling evidence linking EB proteins to the pathogenesis of various diseases, including cancer progression. As the field of cancer research advances, the clinical implication of EB proteins role in cancer severity and aggressiveness become increasingly evident. This review aims to comprehensively explore the role of microtubule-associated EB proteins in influencing the severity and aggressiveness of cancer. We also discuss the potential significance of EB as a clinical biomarker for cancer diagnosis and prognosis and as a target for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e8a843dc90163f1664e05266a959467a1810e6f" target='_blank'>
              End Binding Proteins: Drivers of Cancer Progression.
              </a>
            </td>
          <td>
            Dhakshmi Sasankan, Renu Mohan
          </td>
          <td>2024-12-19</td>
          <td>Cytoskeleton</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17bb4f23b9e1892130e632d422e0888edddfeef1" target='_blank'>
              Molecular heterogeneity and MYC dysregulation in triple-negative breast cancer: genomic advances and therapeutic implications.
              </a>
            </td>
          <td>
            Priya, Arun Kumar, Dhruv Kumar
          </td>
          <td>2025-01-01</td>
          <td>3 Biotech</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Chronic myeloid leukemia (CML) is a malignant hyperplastic tumor that originats from pluripotent hematopoietic stem cells in the bone marrow. The introduction of tyrosine kinase inhibitors (TKIs) has significantly improved the survival rates of CML patients. This study aimed to identify immune-related genes (IRGs) associated with the response to imatinib therapy in CML. Gene expression profiles from imatinib-treated CML patients were obtained from the Gene Expression Omnibus (GEO) database and categorized into high- and low-score groups based on immune scores calculated using the ESTIMATE algorithm. Subsequent bioinformatics analysis identified 428 differentially expressed IRGs in the CML context. Functional enrichment analysis revealed that these genes were involved in immune-related pathways, including T cell receptor signaling and cytokine-cytokine receptor interaction. Finally, based on five modules in weighted gene co-expression network analysis (WGCNA) and the top-ranked degree, 10 hub genes were identified. Receiver operating characteristic (ROC) analysis in two GEO datasets identified IL10RA, SCN9A, and SLC26A11 as potential biomarkers for predicting imatinib response. We further validated these biomarkers in an independent clinical cohort of 60 CML patients treated with imatinib. Results from quantitative real-time (qRT) PCR revealed high expression of IL10RA and SLC26A11 in responders, while SCN9A showed low expression. All three genes had an AUC greater than 0.75, confirming their potential as predictive biomarkers. These findings deepen our understanding of functional characteristics and immune-related molecular mechanisms underlying imatinib response and offer promising predictive biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c0c7ebb98e6af8ee93c928a53af2e158b31811e" target='_blank'>
              Immune-Related Genes for the Prediction of Response to Imatinib therapy in Chronic Myeloid Leukemia.
              </a>
            </td>
          <td>
            Pu Yang, Qian Yu
          </td>
          <td>2024-12-23</td>
          <td>Carcinogenesis</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The explosion of next-generation sequencing technologies has allowed researchers to move from studying single genes, to thousands of genes, and thereby to also consider the relationships within gene networks. Like others, we are interested in understanding how developmental and evolutionary forces shape the expression of individual genes, as well as the interactions among genes. To this end, we characterized the effects of genetic background and developmental environment on brain gene coexpression in two parallel, independent evolutionary lineages of Trinidadian guppies (Poecilia reticulata). We asked whether connectivity patterns among genes differed based on genetic background and rearing environment, and whether a gene’s connectivity predicted its propensity for expression divergence. In pursuing these questions, we confronted the central challenge that standard approaches fail to control the Type I error and/or have low power in the presence of high dimensionality (i.e., large number of genes) and small sample size, as in many gene expression studies. Using our data as a case study, we detail central challenges, discuss sample size guidelines, and provide rigorous statistical approaches for exploring coexpression differences with small sample sizes. Using these approaches, we find evidence that coexpression relationships differ based on both genetic background and rearing environment. We report greater expression divergence in less connected genes and suggest this pattern may arise and be reinforced by selection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffb7f6929569fe684cb5a3f56de734a873153a7a" target='_blank'>
              FLEXIBILITY IN GENE COEXPRESSION AT DEVELOPMENTAL AND EVOLUTIONARY TIMESCALES
              </a>
            </td>
          <td>
            Eva K. Fischer, Youngseok Song, Wen Zhou, K. Hoke
          </td>
          <td>2024-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="T-cell acute lymphoblastic leukaemia (T-ALL) is a heterogeneous malignant disease with high relapse and mortality rates. To characterise the multiomics features of T-ALL, we conducted integrative analyses using single-cell RNA, TCR and chromatin accessibility sequencing on pre- and post-treatment peripheral blood and bone marrow samples of the same patients. We found that there is transcriptional rewiring of gene regulatory networks in T-ALL cells. Some transcription factors, such as TCF3 and KLF3, showed differences in activity and expression levels between T-ALL and normal T cells and were associated with the prognosis of T-ALL patients. Furthermore, we identified multiple malignant TCR clonotypes among the T-ALL cells, where the clonotypes consisted of distinct combinations of the same TCR α and β chain per patient. The T-ALL cells displayed clonotype-specific immature thymocyte cellular characteristics and response to chemotherapy. Remarkably, T-ALL cells with an orphan TCRβ chain displayed the strongest stemness and resistance to chemotherapy. Our study provided transcriptome and epigenome characterisation of T-ALL cells categorised by TCR clonotypes, which may be helpful for the development of novel predictive markers to evaluate treatment effectiveness for T-ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b35386d641d4da0518c18007554d1299175192fd" target='_blank'>
              Single-Cell Multiomics Reveals TCR Clonotype-Specific Phenotype and Stemness Heterogeneity of T-ALL Cells.
              </a>
            </td>
          <td>
            Songnan Sui, Xiaolei Wei, Yue Zhu, Qiuyue Feng, Xianfeng Zha, Lipeng Mao, Boya Huang, Wen Lei, Guobing Chen, Huien Zhan, Huan Chen, Ru Feng, Chengwu Zeng, Yangqiu Li, Oscar Junhong Luo
          </td>
          <td>2024-12-15</td>
          <td>Cell proliferation</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The exposome was proposed following the realization that most human diseases have an environmental rather than a genetic (hereditary) origin. Non-communicable diseases are, in fact, the consequence of multiple exposures that activate a sequence of stages in a multistage process that already starts in early life. This attracted attention to both the multiplicity (in fact, potentially the totality) of exposures humans are exposed to since conception and to the life-long perspective of disease causation. In this paper, we examine an extension of the exposome concept that incorporates a Darwinian approach based on the concept of phenotypic plasticity. One of the theses is that interpreting exposome science as “precision environmental research” is only a partial interpretation, largely focused on chemical exposures, while a broadening of the perspective is needed, also in light of the planetary crisis. Such broadening involves the incorporation of basic concepts from evolutionary biology and medicine, including the ability of organisms to adapt to rapidly changing environments. We refer in particular to cancer and “Darwinian carcinogenesis.”">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4836d1a329da15d078de5795c5b386880fd84f3b" target='_blank'>
              An evolutionary perspective for the exposome
              </a>
            </td>
          <td>
            P. Vineis, Sonia Dagnino
          </td>
          <td>2024-02-09</td>
          <td>Exposome</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Youthful gene expression and function can be restored in old cells by brief activation of transcription factors (TFs), a process known as partial reprogramming. Partial reprogramming holds promise as a therapeutic approach, but current methods have only employed a small set of pluripotency TFs that also reprogram cell identity in undesirable ways. It remains unknown if other TFs might be able to reprogram cell age, how long these effects last, and how often TFs act in concert to promote youthful phenotypes. To address these questions, we developed a screening technology called Partial-seq to measure the effect of partial reprogramming on gene expression across thousands of TF sets at once. We used Partial-seq to screen >4000 TF sets in aged human T cells, a keystone cell type of the immune system that declines in function with age. We discovered that many TFs beyond pluripotency factors restored youthful gene expression and cell states, without abolishing T cell identity. TF combinations were more likely than individual factors to reverse cell age, and many of these effects persisted after TF activation ceased. Our results suggest that the phenotypic age of human T cells can be durably reversed by partial reprogramming with diverse TFs. The diversity of TF sets discovered in our screens suggests that reprogramming cell age might be accomplished without perturbing cell identity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33fce7d6f2ff1adcedf00e09609177688f7d0c53" target='_blank'>
              PARTIAL REPROGRAMMING REVERSES FEATURES OF AGING IN HUMAN T LYMPHOCYTES
              </a>
            </td>
          <td>
            Jacob Kimmel
          </td>
          <td>2024-12-01</td>
          <td>Innovation in Aging</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background/Objectives: AML is an aggressive malignant disease characterized by aberrant proliferation and accumulation of immature blast cells in the patient’s bone marrow. Chemotherapeutic treatment can effectively induce remission and re-establish functional hematopoiesis. However, many patients experience chemoresistance-associated relapse and disease progression with a poor prognosis. The identification of molecular determinants of chemoresistance that could serve as potential targets for the therapeutic restoration of chemosensitivity has proven to be challenging. Methods: To address this, we have analyzed longitudinal changes in the expression of microRNAs during disease progression in a small set of four AML patients, combined with gene ontology (GO) pathway analysis and evaluation of gene expression data in patient databases. Results: MicroRNA profiling of bone marrow samples at diagnosis and after relapse revealed significant differential expression of a large number of microRNAs between the two time points. Subsequent GO pathway analysis identified 11 signal transduction pathways likely to be affected by the differential miRNA signatures. Exemplary validation of the FoxO signaling pathway by gene expression analysis confirmed significant upregulation of FOXO1 and the target genes GADD45 and SOD2. Conclusions: Here, we show how a microRNA-based pathway prediction strategy can be used to identify differentially regulated signaling pathways that represent potential targets for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3ad3edfd6af9770fc155f2d3924cf9e30fae927" target='_blank'>
              MicroRNA Screening Reveals Upregulation of FoxO-Signaling in Relapsed Acute Myeloid Leukemia Patients
              </a>
            </td>
          <td>
            Paula Reichelt, S. Bernhart, U. Platzbecker, Michael Cross
          </td>
          <td>2024-12-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e31d8028a34805d9bd64c4990406de1eae29e1a1" target='_blank'>
              Age-related genomic alterations and chemotherapy sensitivity in osteosarcoma: insights from cancer genome profiling analyses.
              </a>
            </td>
          <td>
            Hidetatsu Outani, Masachika Ikegami, Yoshinori Imura, Shou Nakai, Haruna Takami, Yuki Kotani, A. Inoue, S. Okada
          </td>
          <td>2024-12-17</td>
          <td>International journal of clinical oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background Acute myeloid leukemia (AML) is a hematologic tumor with poor prognosis and significant clinical heterogeneity. By integrating transcriptomic data, single-cell RNA sequencing data and independently collected RNA sequencing data this study aims to identify key genes in AML and establish a prognostic assessment model to improve the accuracy of prognostic prediction. Materials and methods We analyzed RNA-seq data from AML patients and combined it with single-cell RNA sequencing data to identify genes associated with AML prognosis. Key genes were screened by bioinformatics methods, and a prognostic assessment model was established based on these genes to validate their accuracy. Results The study identified eight key genes significantly associated with AML prognosis: SPATS2L, SPINK2, AREG, CLEC11A, HGF, IRF8, ARHGAP5, and CD34. The prognostic model constructed on the basis of these genes effectively differentiated between high-risk and low-risk patients and revealed differences in immune function and metabolic pathways of AML cells. Conclusion This study provides a new approach to AML prognostic assessment and reveals the role of key genes in AML. These genes may become new biomarkers and therapeutic targets that can help improve prognostic prediction and personalized treatment of AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fba51cf3073c5201e7804e4ac1e1ae1bb4b49f8" target='_blank'>
              Single-cell transcriptomics reveals heterogeneity and prognostic markers of myeloid precursor cells in acute myeloid leukemia
              </a>
            </td>
          <td>
            Guangfeng He, Lai Jiang, Xuancheng Zhou, Yuheng Gu, Jingyi Tang, Qiang Zhang, Qingwen Hu, Gang Huang, Ziye Zhuang, Xinrui Gao, Ke Xu, Yewei Xiao
          </td>
          <td>2024-12-16</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The integrated stress response (ISR) is a conserved eukaryotic signaling pathway that responds to diverse stress stimuli to restore proteostasis. The strength and speed of ISR activation must be tuned properly to allow protein synthesis while maintaining proteostasis. Here, we describe how genetic perturbations change the dynamics of the ISR in budding yeast. We treated ISR dynamics, comprising timecourses of ISR activity across different levels of stress, as a holistic phenotype. We profiled changes in ISR dynamics across thousands of genetic perturbations in parallel using CRISPR interference with barcoded expression reporter sequencing (CiBER-seq). We treated cells with sulfometuron methyl, a titratable inhibitor of branched-amino acid synthesis, and measured expression of an ISR reporter. Perturbations to translation such as depletion of aminoacyl-tRNA synthetases or tRNA biogenesis factors reduced cell growth and caused a strikingly proportionate activation of the ISR activation. In contrast, impaired ribosome biogenesis reduced basal ISR activity and weakened ISR dynamics. Reduced ribosome capacity may lower the demand for amino acids and thereby explain these changes. Our work illustrates how CiBER-seq enables high-throughput measurements of complex and dynamic phenotypes that shed light on adaptive and homeostatic mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5725328bae6e97ada63ca77f5c500e9dd0a3db19" target='_blank'>
              Mapping the Genetic Architecture of the Adaptive Integrated Stress Response in S. cerevisiae
              </a>
            </td>
          <td>
            Rachel Baum, Jinyoung Kim, Ryan Y. Muller, Nicholas T. Ingolia
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="The DNA double-strand break (DSB) always occurs within genome. The pathways that repair the DSB are essential in keeping cell viability. This review will focus on the recent understanding of TMEJ, especially two key protein factors involved in the TMEJ repair pathway2, DNA polymerase theta (Polθ) and DNA polymerase delta (Polδ), and discuss the potential mechanism of choosing repair pathways when there is a DNA Double Strand Break. We dissect TMEJ's unique mechanism of action, including recognition of DNA ends, microhomology search, end pairing, and DNA synthesis, by focusing on the enzyme Polθ as the central player to understand the repair pathway choice. We also discuss how BRCA-mutated cancer uses the TMEJ repair pathway to facilitate breast cancer cells' growth3. This review aims to provide a comprehensive overview of TMEJ's cellular functions, regulatory mechanisms, and pivotal role in BRCA-mutated breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45ba6a1c7fb204a897897b1b79890ca6faef600e" target='_blank'>
              Polymerase Theta-mediated DNA End-Joining Repair in the Maintenance of Genome Stability
              </a>
            </td>
          <td>
            Xiangyu Yan
          </td>
          <td>2024-12-24</td>
          <td>Transactions on Materials, Biotechnology and Life Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Autism Spectrum Disorder (ASD) is a complex and heterogeneous disorder, which has been reported affected by many different causative genes. In recent years, single-nucleus RNA sequencing (snRNA-seq) have been widely used to reveal variation in gene expression across cell types during the pathology of ASD. However, the vast majority of current studies only consider information on the single modality of gene expression profile obtained from snRNA-seq. Recent bioinformatics studies have successfully captured key transcriptomic information on alternative polyadenylation (APA) from single-cell RNA-seq (scRNA-seq) data to reveal APA dynamics between cell types. Here we integrated multimodal information of gene expression and APA to gain a more comprehensive understanding of the molecular mechanisms of ASD. APA sites were first identified at the single cell level in patients with ASD and healthy populations under different cell types. Gene features were selected by two unsupervised methods using profiles of APA usages and gene expression, respectively. Then these gene features were integrated to build a cell-type-specific classification model based on multiview privileged support vector machines to predict whether a cell is from a ASD patient or a healthy control. With this model, the contribution of an individual modality was calculated, and genes from different modalities were ranked by importance score to identify high-risk genes that are highly associated with ASD. The functions of these high-risk genes were found to be associated with ASD-related biological functions and significantly different across cell types, highlighting the cell-type-specificity of ASD. These high confidence risk genes can serve as potential biomarkers for the diagnosis and treatment of ASD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12cf9da755d59daf8c7da747343466f69076be65" target='_blank'>
              Identification of cell-type-specific genes associated with autism by integrating gene expression and alternative polyadenylation profiles
              </a>
            </td>
          <td>
            Xingyu Bi, Liping Kang, Xiaohui Wu
          </td>
          <td>2024-12-03</td>
          <td>2024 IEEE International Conference on Bioinformatics and Biomedicine (BIBM)</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="BACKGROUND
Acquired resistance to temozolomide (TMZ) chemotherapy due to DNA mismatch repair (MMR) enzyme deficiency is a barrier to improving outcomes for IDH wild-type glioblastoma (GBM) patients. KL-50 is a new imidazotetrazine-based therapeutic designed to induce DNA interstrand cross links, and subsequent double-stranded breaks, in an MMR-independent manner in cells with O-6-Methylguanine-DNA Methyltransferase (MGMT) deficiency. Previous research showed its efficacy against LN229 glioma cells with MMR and MGMT knockdown. Its activity against patient-derived GBM that model post-TMZ recurrent tumors is unclear.


METHODS
We created MMR-deficient GBM patient-derived xenografts through exposure to TMZ, followed by treatment with additional TMZ or KL-50. We also generated isogenic, MSH6 knockout patient-derived GBM and tested them for sensitivity to TMZ and KL-50.


RESULTS
KL-50 extended the median survival of mice intracranially engrafted with either patient-derived TMZ-naïve GBM6 or TMZ-naïve GBM12 by 1.75-fold and 2.15-fold, respectively (p<0.0001). A low dose (4 Gy) of fractionated RT further extended the survival of KL-50 treated GBM12 mice (median survival=80 days for RT+ KL-50 vs. 71 days KL-50 alone, P=0.018). KL-50 also extended the median survival of mice engrafted with post-TMZ, MMR-deficient GBM6R-m185 (140 days for KL-50 vs. 37 days for vehicle, p<0.0001). MSH6-KO increased TMZ IC50 for GBM6 and GBM12 cultures by >5-fold and >12-fold for cell death and live cell count outputs, respectively. In contrast, MSH6-KO actually decreased KL-50 IC50 by 10-80%.


CONCLUSION
KL-50-based compounds are a promising new strategy for the treatment of MGMT-deficient, MMR-deficient GBM that recurs after frontline TMZ.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03f3a0f339bf4219c2fc489415b10b402bba9dd1" target='_blank'>
              The novel DNA cross-linking agent KL-50 is active against patient-derived models of new and recurrent post-temozolomide mismatch repair-deficient glioblastoma.
              </a>
            </td>
          <td>
            Matthew McCord, Thomas Sears, Wenxia Wang, Rahul Chaliparambil, Shejuan An, Jann Sarkaria, C. James, Bruce Ruggeri, Susan E. Gueble, R. Bindra, C. Horbinski
          </td>
          <td>2024-12-10</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Mutations in the KRAS oncogene can mediate resistance to radiation. KRAS mutation (mut) driven tumors have been reported to express cancer stem cell (CSC)-like features and may harbor metabolic liabilities through which CSC-associated radioresistance can be overcome. We established a radiation/drug screening approach that relies on the growth of 3D spheres under anchorage-independent and lipid-limiting culture conditions, which promote stemness and lipogenesis. In this format, we screened 32 KRASmut-enriched lung cancer models. As predicted from published data, CB-839, a glutaminase inhibitor, displayed the highest degree of radiosensitization in KRASmut models with LKB1 co-mutations. Radiosensitization by inhibition of stearoyl-CoA desaturase-1, SCD1, displayed a similar genotype preference though the data also implicated KEAP1 co-mutation and SCD1 expression as potential predictors of radiosensitization. In an isogenic model, KRASmut cells were characterized by increased SCD1 expression and a higher ratio of monounsaturated fatty acids (MUFA) to saturated fatty acids. Accordingly, pharmacological inhibition or depletion of SCD1 radiosensitized isogenic KRASmut but not wild-type cells. The radiosensitizing effect was notably small, especially compared to several DNA repair inhibitors. As an alternative strategy to targeting MUFA metabolism, adding polyunsaturated FAs (PUFA) phenocopied some aspects of SCD1 inhibition, suppressed tumor growth in vivo, and opposed the CSC-like phenotype of KRASmut cells. In conclusion, we report a 3D screening approach that recapitulates clinically relevant features of KRASmut tumors and can be leveraged for therapeutic targeting of metabolic vulnerabilities. Our data highlight pronounced inter-tumoral heterogeneity in radiation/drug responses and the complexity of underlying genomic dependencies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7bc5ae81e5c37c5c26b932eed3934aecd7b153b2" target='_blank'>
              Targeting Monounsaturated Fatty Acid Metabolism for Radiosensitization of KRAS Mutant 3D Lung Cancer Models.
              </a>
            </td>
          <td>
            Shan Lu, Xiao Pan, Eva Volckova, Anjali Shinde, Schuyler Fuller, Regina K. Egan, Jianli Ma, Jong Kung, Christopher J Ott, Aaron N Hata, Cyril H. Benes, Jing X Kang, Henning Willers
          </td>
          <td>2025-01-21</td>
          <td>Molecular cancer therapeutics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e9558944d48747d281f714408c8c756ade188a3" target='_blank'>
              Genetic and epigenetic characterization of sarcoma stem cells across subtypes identifies EZH2 as a therapeutic target
              </a>
            </td>
          <td>
            Edmond O'Donnell, Maria Muñoz, Ryan Davis, Jessica Bergonio, R. L. Randall, Clifford Tepper, Janai R. Carr-Ascher
          </td>
          <td>2025-01-09</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract RNA modification has emerged as a crucial area of research in epigenetics, significantly influencing tumor biology by regulating RNA metabolism. N‐acetyltransferase 10 (NAT10)‐mediated N4‐acetylcytidine (ac4C) modification, the sole known acetylation in eukaryotic RNA, influences cancer pathogenesis and progression. NAT10 is the only writer of ac4C and catalyzes acetyl transfer on targeted RNA, and ac4C helps to improve the stability and translational efficiency of ac4C‐modified RNA. NAT10 is highly expressed and associated with poor prognosis in pan‐cancers. Based on its molecular mechanism and biological functions, ac4C is a central factor in tumorigenesis, tumor progression, drug resistance, and tumor immune escape. Despite the increasing focus on ac4C, the specific regulatory mechanisms of ac4C in cancer remain elusive. The present review thoroughly analyzes the current knowledge on NAT10‐mediated ac4C modification in cancer, highlighting its broad regulatory influence on targeted gene expression and tumor biology. This review also summarizes the limitations and perspectives of current research on NAT10 and ac4C in cancer, to identify new therapeutic targets and advance cancer treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc21647bfb08f2bead5c258ee67888a30cbd41b0" target='_blank'>
              The role and mechanism of NAT10‐mediated ac4C modification in tumor development and progression
              </a>
            </td>
          <td>
            Zhuoran Gu, Libin Zou, Xinjian Pan, Yang Yu, Yongqiang Liu, Zhijin Zhang, Ji Liu, Shiyu Mao, Junfeng Zhang, Changcheng Guo, Wei Li, Jiang Geng, Wentao Zhang, Xudong Yao, Bing Shen
          </td>
          <td>2024-12-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Neoadjuvant, endocrine, and targeted therapies have significantly improved the prognosis of breast cancer (BC). However, due to the high heterogeneity of cancer, some patients cannot benefit from existing treatments. Increasing evidence suggests that amino acids and their metabolites can alter the tumor malignant behavior through reshaping tumor microenvironment and regulation of immune cell function. Breast cancer cell lines have been identified as methionine-dependent, and methionine restriction has been proposed as a potential cancer treatment strategy.We integrated transcriptomic and single-cell RNA sequencing (ScRNA-seq) analyses based on The Cancer Genome Atlas (TCGA) database and Gene Expression Omnibus (GEO) datasets. Then we applied weighted gene co-expression network analysis (WGCNA) and Cox regression to evaluate methionine metabolism-related genes (MRGs) in BC, constructing and validating a prognostic model for BC patients. Immune landscapes and immunotherapy were further explored. Finally, in vitro experiments were conducted to assess the expression and function of key genes APOC1.In this study, we established and validated a prognostic signature based on eight methionine-related genes to predict overall survival (OS) in BC patients. Patients were further stratified into high-risk and low-risk groups according to prognostic risk score. Further analysis revealed significant differences between two groups in terms of pathway alterations, immune microenvironment characteristics, and immune checkpoint expression. Our study shed light on the relationship between methionine metabolism and immune infiltration in BC. APOC1, a key gene in the prognostic signature, was found to be upregulated in BC and closely associated with immune cell infiltration. Notably, APOC1 was primarily expressed in macrophages. Subsequent in vitro experiments demonstrated that silencing APOC1 reduced the generation of tumor-associated macrophages (TAMs) with an M2 phenotype while significantly decreasing the proliferation, invasion, and migration of MDA-MB-231 and MDA-MB-468 breast cancer cell lines.We established a prognostic risk score consisting of genes associated with methionine metabolism, which helps predict prognosis and response to treatment in BC. The function of APOC1 in regulating macrophage polarization was explored.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31d865c02ec071f3b620933e096d65cab7e82d29" target='_blank'>
              Integrating single-cell RNA-seq and bulk RNA-seq to explore prognostic value and immune landscapes of methionine metabolism-related signature in breast cancer
              </a>
            </td>
          <td>
            Yanxian Gao, Ziyu Feng, Hailong Zhao, Xinghai Liu, Muyu Zhu, Xiafei Yu, Xiaoan Liu, Xian Wu, Jing Tao
          </td>
          <td>2025-01-14</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Chemotherapy often kills a large fraction of cancer cells but leaves behind a small population of drug- tolerant persister cells. These persister cells survive drug treatments through reversible, non-genetic mechanisms and cause tumour recurrence upon cessation of therapy. Here, we report a drug tolerance mechanism regulated by the germ-cell-specific H3K4 methyltransferase PRDM9. Through histone proteomic, transcriptomic, lipidomic, and ChIP-sequencing studies combined with CRISPR knockout and phenotypic drug screen, we identified that chemotherapy-induced PRDM9 upregulation promotes metabolic rewiring in glioblastoma stem cells, leading to chemotherapy tolerance. Mechanistically, PRDM9-dependent H3K4me3 at cholesterol biosynthesis genes enhances cholesterol biosynthesis, which persister cells rely on to maintain homeostasis under chemotherapy- induced oxidative stress and lipid peroxidation. PRDM9 inhibition, combined with chemotherapy, resulted in strong anti-cancer efficacy in preclinical glioblastoma models, significantly enhancing the magnitude and duration of the antitumor response by eliminating persisters. These findings demonstrate a previously unknown role of PRDM9 in promoting metabolic reprogramming that enables the survival of drug-tolerant persister cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44ca423a805e163b74f1186c734661fa71cbd4de" target='_blank'>
              Histone methyltransferase PRDM9 promotes survival of drug-tolerant persister cells in glioblastoma
              </a>
            </td>
          <td>
            George L. Joun, Emma G. Kempe, Brianna Chen, Jayden Sterling, Ramzi H. Abbassi, W. Daniel du Preez, A. Recasens, Teleri Clark, Tian Y. Du, Jason K.K. Low, Hani Kim, Pengyi Yang, Jasmine Khor, Monira Hoque, Dinesh C. Indurthi, M. Kuchibhotla, Ranjith Palanisamy, William T Jorgensen, Andrew P. Montgomery, Jennifer R. Baker, S. Higginbottom, E. Tomaskovic-Crook, J. M. Crook, Lipin Loo, Bryan W. Day, G. G. Neely, Ernesto Guccione, Terrance G. Johns, Michael Kassiou, A. S. Don, L. Munoz
          </td>
          <td>2025-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Background: RAD51C, a critical member of the RAD51 paralog family, is essential for homologous recombination (HR)-mediated DNA repair, a pathway crucial for maintaining genomic stability. Mutations in RAD51C have been linked to cancer susceptibility, particularly in breast and ovarian cancers, where impaired DNA repair mechanisms contribute to genomic instability and tumor progression. Despite its clinical significance, the functional impact of specific RAD51C variants remains poorly understood, necessitating a comprehensive investigation into their biological implications. 
Methods: This study classified RAD51C gene variants into damaging and tolerant categories using computational prediction tools, including SIFT, PolyPhen, CADD, MetaLR, and Mutation Assessor. Variants were prioritized based on consensus scores and classified as high-confidence damaging variants. Correlation and agreement among tools were analyzed to refine predictions. Principal Component Analysis (PCA) and clustering methods were employed to group variants based on prediction patterns. Protein-protein interaction (PPI) networks and pathway enrichment analyses were conducted to contextualize damaging variants within broader biological systems, with a focus on their roles in HR, DNA repair, and cellular processes. 
Results: A total of 2526 variants were analyzed, with damaging variants showing consistent patterns across tools. Consensus scores highlighted 302 high-confidence damaging variants, which were associated with disrupted biological processes, including double-strand break repair via homologous recombination, telomere maintenance, and regulation of cell cycle checkpoints. PPI analysis revealed an interconnected network with 11 nodes and 54 edges, with a clustering coefficient of 0.982, indicating tightly coordinated interactions among DNA repair proteins. Pathway enrichment analyses identified significant associations with homologous recombination (FDR = 2.55E-17) and the Fanconi anemia pathway (FDR = 2.96E-06). 
Conclusion: This study provides a comprehensive framework for assessing the functional impacts of RAD51C variants by integrating computational predictions with biological analyses. The findings underscore the importance of RAD51C in HR and DNA repair pathways, offering insights into its role in genomic stability and cancer progression. These results can inform the prioritization of variants for experimental validation and guide therapeutic strategies targeting DNA repair deficiencies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16240c1097223735006d241b04503aff291d133b" target='_blank'>
              Statistical Analysis of Gene Variants for Homologous Recombination Pathways of DNA Repair leading to Cancer Susceptibility
              </a>
            </td>
          <td>
            Usha Adiga, B. Jyoti, P. Reddemma, Alfred J. Augustine, S. Vasishta
          </td>
          <td>2024-12-27</td>
          <td>International Journal of Statistics in Medical Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="IDH-mutant low-grade gliomas (LGGs) are slow-growing brain tumors that frequently progress to aggressive high-grade gliomas that have dismal outcomes. In a recent study, Wu and colleagues provide critical insights into the mechanisms underlying malignant progression by analyzing single-cell gene expression and chromatin accessibility across different tumor grades. Their findings support a two-phase model: in early stages, tumors are primarily driven by oligodendrocyte precursor-like cells and epigenetic alterations that silence tumor suppressors like CDKN2A and activate oncogenes such as PDGFRA. As the disease advances, the tumors become sustained by more proliferative neural precursor-like cells, where genetic alterations, including PDGFRA, MYCN, and CDK4 amplifications and CDKN2A/B deletion, drive tumor progression. The study further highlights a dynamic regulation of interferon (IFN) signaling during progression. In low-grade IDH-mutant gliomas, IFN responses are suppressed through epigenetic hypermethylation, which can be reversed with DNMT1 inhibitors or IDH inhibitors, leading to reactivation of the IFN pathway. In contrast, higher-grade gliomas evade IFN signaling through genetic deletions of IFN gene clusters. These findings emphasize a broader epigenetic-to-genetic shift in oncogenic regulation that drives glioma progression, provides a valuable framework for understanding the transition from indolent tumors to lethal malignancies, and has implications for therapy and clinical management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/598e5f4259516cb6e86cc401bbe6fe663aa489ab" target='_blank'>
              Switching Drivers: Epigenetic Rewiring to Genetic Progression in Glioma.
              </a>
            </td>
          <td>
            K. Drucker, Robert B. Jenkins, Daniel Schramek
          </td>
          <td>2024-12-30</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Comprehensive genetic analysis of tumors with exome or whole genome sequencing has enabled the identification of the genes that are recurrently mutated in cancer. This has stimulated a series of exciting advances over the past 15 years, guiding us to new molecular biomarkers and therapeutic targets among the common mature B-cell neoplasms. In particular, diffuse large B-cell lymphoma (DLBCL), follicular lymphoma (FL) and Burkitt lymphoma (BL) have each been the subject of considerable attention in this field. Currently, more than 850 genes have been reported as targets of protein-coding mutations in at least one of these entities. To reduce this to a manageable size, we describe a systematic approach to prioritize and categorize these genes, based on the quality and type of supporting data. For each entity, we provide a list of candidate driver genes categorized into Tier 1 (high-confidence genes), Tier 2 (candidate driver genes), or Tier 3 (lowest-confidence genes). Collectively, this reduces the number of high-confidence genes for these three lymphomas to a mere 144. This further affirms the substantial overlap between the genes relevant in DLBCL and each of FL and BL. These highly curated and annotated gene lists will continue to be maintained as a resource to the community. These results emphasize the extent of the knowledge gap regarding the role of each of these genes in lymphomagenesis. We offer our perspective on how to accelerate the experimental confirmation of drivers using a variety of model systems, using these lists as a guide for prioritizing genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dadf53884a55922c51a797521fda638d01af815c" target='_blank'>
              Audit of B-cell Cancer Genes.
              </a>
            </td>
          <td>
            K. Coyle, K. Dreval, Daniel J Hodson, Ryan D Morin
          </td>
          <td>2025-01-24</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is the most aggressive breast cancer subtype with substantial disease heterogeneity, limited treatment options, and dismal clinical outcomes. Some TNBCs display homologous recombination deficiency (HRD), a phenotype with elevated genomic burden and worse prognosis if left untreated but chemotherapeutic sensitivity. While the molecular landscape of TNBC is distinct from other breast cancer subtypes, the TNBC-specific link between HRD and epigenome-wide methylation has not been established. This study reports two independent cohorts of TNBC tumors (n = 32 and n = 58) with HRD and epigenomic landscapes measured by the Multiplex Ligation-dependent Probe Amplification assay and the Illumina MethylationEPIC arrays, respectively. Genome-wide copy number and methylation alterations were significantly higher in HRD (all p <.05). Methylation of genome-wide repeat element Alu and transcriptional regulatory regions were significantly lower in HRD (all p <.05). An age-adjusted epigenome-wide association study of the continuous HRD probability scores revealed significant loci (all FDR <0.05) that were depleted from the CpG-rich "island" regions often seen in gene promoters but enriched in the CpG-poor "open sea" regions localized to gene enhancers. The significant loci implicated well-known candidate genes involved in the epithelial-to-mesenchymal transition, Wnt signaling, and DNA damage response. Supervised machine learning of HRD with nucleotide-specific methylation as the input enabled clinically relevant tumor stratification. Taken together, this study provides novel biological and translational insights into HRD in TNBCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5080511deac3bf2a96cf31c6d0433b409f61f60d" target='_blank'>
              Extensive epigenomic dysregulation is a hallmark of homologous recombination deficiency in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Youdinghuan Chen, Lucas A Salas, Jonathan D. Marotti, Nicole P. Jenkins, Chao Cheng, Todd W. Miller, A. Kettenbach, Brock C. Christensen
          </td>
          <td>2024-12-05</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Regulatory mutations, coding sequence alterations, and gene deletions and duplications are generally expected to have qualitatively different effects on fitness. We aim to ground this expectation within a theoretical framework using evolutionary simulations of gene regulatory networks (GRNs) controlling the expression of fitness-related genes. We examined the distribution of fitness effects as a function of the type of mutation and the topology of the gene network. Contrary to our expectation, the GRN topology had more influence on the effect of mutations than the type of mutation itself. In particular, the topology conditioned (i) the speed of adaptation, (ii) the distribution of fitness effects, and (iii) the degree of pleiotropy, acting as explanatory factor for all mutation types. All mutations had the potential to participate in adaptation, although their propensity to generate beneficial variants differed according to the network topology. In scalefree networks, the most common topology for biological networks, coding mutations were more pleiotropic and overrepresented in both beneficial and deleterious mutations, while regulatory mutations were more often neutral. However, this observation was not general, as this pattern was reversed in alternative networks. These results highlight the critical role of network topology in defining mutations’ contributions to adaptation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/416ce7026f77cde0b3183fa7cbeac65ad92bf503" target='_blank'>
              Gene network topology drives the mutational landscape of gene expression
              </a>
            </td>
          <td>
            Sylvain Pouzet, Arnaud Le Rouzic
          </td>
          <td>2024-11-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="BACKGROUND
The management of pediatric acute myeloid leukemia (AML) is based on the prognostic risk classification of initial leukemia. Targeted next-generation sequencing (NGS) is a reliable method used to identify recurrently mutated genes of pediatric AML and associated prognosis.


METHODS
In this study, we retrospectively evaluated the prognostic, and therapeutic utility of a targeted NGS panel covering twenty-five genes, in 21 children with de novo and 8 with relapsed or secondary AML.   Results. Variants were detected in 44.8% of patients, and 63.2% of them were in the signaling pathway genes. The number of variants per patient and diversity increased with age. The panel results affected hematopoietic stem cell transplantation decisions, especially in core binding factor AML, and allowed the categorization of diseases according to current classifications. Panel results also pointed out predisposition to germline leukemia to the extent of the panel coverage. No targeted therapy was used based on the variants, and none of the variants were used to monitor minimal residual disease.


CONCLUSIONS
Targeted NGS results, along with well-known genetic aberrations and treatment responses, can guide treatment modalities. The coverage of the routine panels should include proven mutations of childhood AML and germline leukemia predisposition genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b4601dddbafa15e4c68929e64e4ba8e0770802f" target='_blank'>
              The importance of targeted next-generation sequencing based genomic profiling in the diagnosis of childhood acute myeloid leukemia: a single center experience.
              </a>
            </td>
          <td>
            D. Kaçar, B. Çavdarlı, Ayça Koca Yozgat, Melek Işık, F. Kurtipek, Fatma Tuba Yıldırım, T. Bayhan, Dilek Gürlek Gökçebay, Namık Yaşar Özbek, N. Yaralı
          </td>
          <td>2024-12-19</td>
          <td>The Turkish journal of pediatrics</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2962c853fd82d5be105d8daa9d66daafd430b25e" target='_blank'>
              Elongation factor 2 in cancer: a promising therapeutic target in protein translation
              </a>
            </td>
          <td>
            X. Jia, Chuntian Huang, Fangfang Liu, Zigang Dong, Kangdong Liu
          </td>
          <td>2024-12-20</td>
          <td>Cellular & Molecular Biology Letters</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca8eb632e8dcc4d371aa3266d6271acffef83ed2" target='_blank'>
              The integrated stress response drives MET oncogene overexpression in cancers.
              </a>
            </td>
          <td>
            M. Cerqua, Marco Foiani, C. Boccaccio, Paolo M Comoglio, D. Altintas
          </td>
          <td>2025-01-07</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Structural variations (SVs) play important roles in genetic diversity, evolution, and carcinogenesis and are, as such, important for human health. However, it remains unclear how spatial proximity of double-strand breaks (DSBs) affects the formation of SVs. To investigate if spatial proximity between two DSBs affects DNA repair, we used data from 3C experiments (Hi-C, ChIA-PET, and ChIP-seq) to identify highly interacting loci on six different chromosomes. The target regions correlate with the borders of mega-base sized Topologically Associated Domains (TADs), and we used CRISPR-Cas9 nuclease and pairs of single guide RNAs (sgRNAs) against these targets to generate DSBs in both K562 cells and H9 human embryonic stem cells (hESC). Droplet Digital PCR (ddPCR) was used to quantify the resulting recombination events, and high-throughput sequencing was used to analyze the chimeric junctions created between the two DSBs. We observe a significantly higher formation frequency of deletions and inversions with DSBs in proximity as compared to deletions and inversions with DSBs not in proximity in K562 cells. Additionally, our results suggest that DSB proximity may affect the ligation of chimeric deletion junctions. Taken together, spatial proximity between DSBs is a significant predictor of large-scale deletion and inversion frequency induced by CRISPR-Cas9 in K562 cells. This finding has implications for understanding SVs in the human genome and for the future application of CRISPR-Cas9 in gene editing and the modelling of rare SVs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9534f559b29005a3cc7a49a03b73e3b8f75c23b4" target='_blank'>
              Characterization of the role of spatial proximity of DNA double-strand breaks in the formation of CRISPR-Cas9-induced large structural variations.
              </a>
            </td>
          <td>
            Mikkel Dahl-Jessen, T. Terkelsen, R. Bak, Uffe Birk Jensen
          </td>
          <td>2025-01-13</td>
          <td>Genome research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Introduction Hypoxia responses are critical for myriad physiological and pathological processes, such as development, tissue repair, would healing, and tumorigenesis. microRNAs (miRNAs) are a class of small non-coding RNAs that exert their functions by inhibiting the expression of their target genes, and miR-210 is the miRNA universally and most conspicuously upregulated by hypoxia in mammalian systems. For its relationship to hypoxia, miR-210 has been studied extensively, yet no consensus exists on the roles and mechanisms of miR-210 in human physiological processes or diseases, and we know little about genuine miR-210 target genes in humans. Methods To better investigate the functions and mechanisms of human miR-210, therefore, we derived the human miR-210 gene knockout (KO) 293T cell lines using the CRISPR/Cas9 technology. We then examined the cellular phenotypes and gene expression profiles of 293T cells under normoxia and hypoxia conditions. Results and Discussion We found that the loss of miR-210 altered a variety of cellular phenotypes including proliferation and apoptosis. Subsequent global gene expression analyses identified plausible mechanisms underlying these phenotypic changes in 293T cells. In particular, we showed that miR-210 might target the expression of BNIP3L as a potential mechanism to suppress apoptosis. Surprisingly, the mRNA levels of most previously reported miR-210 target genes were not induced upon miR-210 KO, suggesting a need to reexamining and studying human miR-210 functions directly and comprehensively. Thus, our work established a human cellular system and opportunity to unravel the complexity of the regulatory networks by miR-210.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2de9fca0fc73bea5e4d2b5e6bb3d58b2f4b8c51b" target='_blank'>
              miR-210 loss leads to widespread phenotypic and gene expression changes in human 293T cells
              </a>
            </td>
          <td>
            Xiaoxiao Zhang, Zhen Meng, Chengyong Yang, Chenghao Wang, Kexin Zhang, Anxin Shi, Jingjing Guo, Yong Feng, Yan Zeng
          </td>
          <td>2024-12-16</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aa136c947da96c060fe9e8b5c25af5b195959fd" target='_blank'>
              Developing novel Lin28 inhibitors by computer aided drug design
              </a>
            </td>
          <td>
            Victor M Matias-Barrios, Mariia Radaeva, Graciella Rosellinny, Qiongqiong Jia, Ning Xie, Monica T L Villanueva, Hanadi Ibrahim, Jason Smith, Martin Gleave, N. Lallous, Suzana K Straus, Artem Cherkasov, Xuesen Dong
          </td>
          <td>2025-01-12</td>
          <td>Cell Death Discovery</td>
          <td>1</td>
          <td>18</td>
        </tr>

        <tr id="Inadequate response to androgen deprivation therapy (ADT) frequently arises in prostate cancer, driven by cellular mechanisms that remain poorly understood. Here, we integrated single-cell RNA sequencing, single-cell multiomics, and spatial transcriptomics to define the transcriptional, epigenetic, and spatial basis of cell identity and castration response in the mouse prostate. Leveraging these data along with a meta-analysis of human prostates and prostate cancer, we identified cellular orthologs and key determinants of ADT response and resistance. Our findings reveal that mouse prostates harbor lobe-specific luminal epithelial cell types distinguished by unique gene regulatory modules and anatomically defined androgen-responsive transcriptional programs, indicative of divergent developmental origins. Androgen-insensitive, stem-like epithelial populations - resembling human club and hillock cells - are notably enriched in the urethra and ventral prostate but are rare in other lobes. Within the ventral prostate, we also uncovered two additional androgen-responsive luminal epithelial cell types, marked by Pbsn or Spink1 expression, which align with human luminal subsets and may define the origin of distinct prostate cancer subtypes. Castration profoundly reshaped luminal epithelial transcriptomes, with castration-resistant luminal epithelial cells activating stress-responsive and stemness programs. These transcriptional signatures are enriched in tumor cells from ADT-treated and castration-resistant prostate cancer patients, underscoring their likely role in driving treatment resistance. Collectively, our comprehensive cellular atlas of the mouse prostate illuminates the importance of lobe-specific contexts for prostate cancer modeling and reveals potential therapeutic targets to counter castration resistance. Significance Statement Androgen deprivation therapy is a mainstay in prostate cancer treatment, yet many patients eventually develop castration-resistant disease—a lethal progression driven by poorly understood cellular mechanisms. Our study provides a comprehensive cellular map of the prostate, identifying key determinants of normal organization and castration-induced remodeling. By pinpointing the cell types and molecular programs that confer ADT responsiveness or resistance, our findings offer new directions for prostate cancer modeling and pave the way toward novel therapeutic strategies aimed at enhancing ADT efficacy and preventing the emergence of castration-resistant prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47e36c2b5c9a5688d815f8fbf4d281fd5d7c15cd" target='_blank'>
              Cellular cartography reveals mouse prostate organization and determinants of castration resistance
              </a>
            </td>
          <td>
            Han-Byoul Cho, Yuping Zhang, J. Tien, R. Mannan, Jie Luo, S. Narayanan, Somnath Mahapatra, Jing Hu, G. Shelley, Gabriel Cruz, Miriam Shahine, Lisha Wang, Fengyun Su, Rui Wang, Xuhong Cao, S. Dhanasekaran, Evan T. Keller, S. Pitchiaya, A. Chinnaiyan
          </td>
          <td>2024-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>167</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is the most common and highly heterogeneous acute leukemia in adults. Despite the continuous optimization of prognostic risk stratification and treatment regimens, the overall long-term survival rate of AML patients remains low. The emergence of single cell sequencing (SCS) technology has overcome the defects of traditional sequencing to a certain extent, and realized the transformation of the research of malignant tumors from cell population level to single cell level. Consequently, it has been widely applied in many biological fields. This review mainly focuses on the application of SCS technology in the analysis of tumor heterogeneity, disease evolution, changes in tumor microenvironment, elucidation of drug resistance mechanisms and search for potential therapeutic targets in AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b68c408e59a37e382b2006f70f24e926e3badf47" target='_blank'>
              [Research Advance of Single Cell Sequencing Technology in Acute Myeloid Leukemia--Review].
              </a>
            </td>
          <td>
            Gen-Wang Chen, Lei Liu, Chun-Mei Fan
          </td>
          <td>2024-12-01</td>
          <td>Zhongguo shi yan xue ye xue za zhi</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98e96a8cc9646f040d8ba0ebf38bb7db617ee46a" target='_blank'>
              Low dose DNA methyltransferase inhibitors potentiate PARP inhibitors in homologous recombination repair deficient tumors
              </a>
            </td>
          <td>
            R. Pacaud, Scott Thomas, Sibapriya Chaudhuri, Ann Lazar, Luika Timmerman, P. Munster
          </td>
          <td>2025-01-16</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="DNA repair involves various intricate pathways that work together to maintain genome integrity. XPF (ERCC4) is a structural endonuclease that forms a heterodimer with ERCC1 that is critical in both single-strand break repair (SSBR) and double-strand break repair (DSBR). Although the mechanistic function of ERCC1/XPF has been established in nucleotide excision repair (NER), its role in long-patch base excision repair (BER) has recently been discovered through the 5′-Gap pathway. This study briefly explores the roles of XPF in different pathways to emphasize the importance of XPF in DNA repair. XPF deficiency manifests in various diseases, including cancer, neurodegeneration, and aging-related disorders; it is also associated with conditions such as Xeroderma pigmentosum and fertility issues. By examining the molecular mechanisms and pathological consequences linked to XPF dysfunction, this study aims to elucidate the crucial role of XPF in genomic stability as a repair protein in BER and provide perspectives regarding its potential as a therapeutic target in related diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ec8dd83ccab56542b88608ecabf7dc8acc0d8d5" target='_blank'>
              Beyond Nucleotide Excision Repair: The Importance of XPF in Base Excision Repair and Its Impact on Cancer, Inflammation, and Aging
              </a>
            </td>
          <td>
            Dhara Gohil, Rabindra Roy
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06c93468a39c35f57560b703a968a53bd367dddf" target='_blank'>
              Single-cell RNA-seq analysis reveals microenvironmental infiltration of myeloid cells and pancreatic prognostic markers in PDAC
              </a>
            </td>
          <td>
            Yanying Fan, Lili Wu, Xinyu Qiu, Han Shi, Longhang Wu, Juan Lin, Jie Lin, Tianhong Teng
          </td>
          <td>2025-01-23</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The mitochondrial genome (mtDNA) is an important source of inherited extranuclear variation. Clonal increases in mtDNA mutation heteroplasmy have been implicated in aging and disease, although the impact of this shift on cell function is challenging to assess. Reprogramming to pluripotency affects mtDNA mutation heteroplasmy. We reprogrammed three human fibroblast lines with known heteroplasmy for deleterious mtDNA point or deletion mutations. Quantification of mutation heteroplasmy in the resulting 76 induced pluripotent stem cell (iPSC) clones yielded a bimodal distribution, creating three sets of clones with high levels or absent mutation heteroplasmy with matched nuclear genomes. iPSC clones with elevated deletion mutation heteroplasmy show altered growth dynamics, which persist in iPSC-derived progenitor cells. We identify transcriptomic and metabolic shifts consistent with increased investment in neutral lipid synthesis as well as increased epigenetic age in high mtDNA deletion mutation iPSC, consistent with changes occurring in cellular aging. Together, these data demonstrate that high mtDNA mutation heteroplasmy induces changes occurring in cellular aging.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07003adbe30c731047a58b8ae874dca402e30f77" target='_blank'>
              Increased mitochondrial mutation heteroplasmy induces aging phenotypes in pluripotent stem cells and their differentiated progeny.
              </a>
            </td>
          <td>
            A. Vandiver, Alejandro Torres, Amberly Sanden, Thang L Nguyen, Jasmine Gasilla, Mary T Doan, Vahan Martirosian, Austin N. Hoang, J. Wanagat, Michael A Teitell
          </td>
          <td>2024-12-16</td>
          <td>Aging cell</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Clinical risk scores in chronic myeloid leukemia (CML) are inadequate for identifying chronic phase (CP) patients at high-risk of blast crisis (BC) progression. The lack of accurate predictive tests hamper timely interventions, including stem cell transplants, that are more effective in early disease. By interrogating a single cell atlas of primary imatinib resistance for a BC-like gene expression signature, we identify aberrant CD42A+ megakaryocytic and CD10+CD19+ lymphoid progenitor expansion, as well as STAT1- and IFNγ-related inflammatory programs, as consistent features in the bone marrow and peripheral blood of CP patients at high-risk of BC transformation. We develop multi-color flow cytometry-based tests (MFC) to detect these features in CP patients at the time of diagnosis. Validating our MFC panels on a combined Australia-Singapore cohort comprising 28 CP patients, including 14 who underwent transformation, we demonstrate their ability to detect 100% of CP patients who transform with no false positives. Our findings highlight small populations of inflamed hematopoietic stem and progenitor cells, present at CP diagnosis, as powerful harbingers of future BC transformation. The ability of MFC panels to detect these cells at diagnosis support their inclusion as accurate risk-assessment tools to improve management of high-risk patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/695b8c7eb368f693a97922122a5ffdc4d6cd3154" target='_blank'>
              Single-cell transcriptomics-guided development of flow cytometric tests predicting chronic myeloid leukemia blast crisis transformation at chronic phase diagnosis
              </a>
            </td>
          <td>
            Mengge Yu, Vaidehi Krishnan, Pavithra Shyamsundar, K. Lee, Florian Schmidt, Xi Ren, Charles T.H. Chuah, Shyam Prabhakar, V. Saunders, D. Yeung, N. Shanmuganathan, Agnes S. M. Yong, Timothy P. Hughes, S. T. Ong
          </td>
          <td>2025-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Epigenetic inheritance can be influenced by environmental conditions and somatic inputs originating from different tissues. Such transgenerational inheritance is reversable as it does not lead to mutations and thus alterations in the genetic information. Epigenetic mechanisms such as small RNAs (sRNAs) in the germline silence transposable elements (TEs), which comprise a significant portion of the genome. TE mobility can be mutagenic and is thought to contribute to genome evolution by altering genome structure and diversification. In C. elegans elevated temperature leads to desilencing of TEs in the germline, suggesting that environmental conditions could impact the integrity of inheritable genomes. Here, we show that dysfunction of the AFD thermosensory neurons leads to desilencing of TEs in the germline. The TE activity results in de novo insertions and mutagenesis. Our data indicate that neuronal sensing of environmental conditions can alter genetic inheritance through stress-induced TE mutagenesis thus shaping genome evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2690098c3f1459a2382bf854a60e7cd005d96da4" target='_blank'>
              Thermosensory neurons control genetic inheritance through regulation of germline transposons
              </a>
            </td>
          <td>
            Péter R. Szántó, David H. Meyer, Guy Teichman, Francy J. Pérez-Llanos, Daniel Rickert, Karl Köhrer, O. Rechavi, B. Schumacher
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="RAD18 is a conserved eukaryotic E3 ubiquitin ligase that promotes genome stability through multiple pathways. One of these is gap-filling DNA synthesis at active replication forks and in post-replicative DNA. RAD18 also regulates homologous recombination (HR) repair of DNA breaks; however, the current literature describing the contribution of RAD18 to HR in mammalian systems has not reached a consensus. To investigate this, we examined three independent RAD18-null human cell lines. Our analyses found that loss of RAD18 in HCT116, but neither hTERT RPE-1 nor DLD1 cell lines, resulted in elevated sister chromatid exchange, gene conversion, and gene targeting, i.e., HCT116 mutants were hyper-recombinogenic (hyper-rec). Interestingly, these phenotypes were linked to RAD18's role in PCNA K164 ubiquitination, as HCT116 PCNAK164R/+ mutants were also hyper-rec, consistent with previous studies in rad18-/- and pcnaK164R avian DT40 cells. Importantly, the knockdown of UBC9 to prevent PCNA K164 SUMOylation did not affect hyper-recombination, strengthening the link between increased recombination and RAD18-catalyzed PCNA K164 ubiquitination, but not K164 SUMOylation. We propose that the hierarchy of post-replicative repair and HR, intrinsic to each cell type, dictates whether RAD18 is required for suppression of hyper-recombination and that this function is linked to PCNA K164 ubiquitination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ade4d531e6f454e305c916fc9c53a0c286d620e" target='_blank'>
              Cell Type Specific Suppression of Hyper-Recombination by Human RAD18 Is Linked to Proliferating Cell Nuclear Antigen K164 Ubiquitination.
              </a>
            </td>
          <td>
            Colette B. Rogers, W. Leung, Ryan M. Baxley, Rachel E. Kram, Liangjun Wang, Joseph P. Buytendorp, Khoi Le, D. Largaespada, Eric A. Hendrickson, A. Bielinsky
          </td>
          <td>2025-01-20</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Engineering cell fate is fundamental for optimizing stem cell-based therapies aimed at replacing cells in patients suffering from trauma or disease. By timely administering molecular regulators—such as transcription factors, RNAs, or small molecules—in a process that mimics in vivo embryonic development, stem cell differentiation can be guided toward a specific cell fate. A significant challenge in scaling up these therapies is that such differentiation strategies often result in mixed cellular populations. While synthetic biology approaches have been proposed to increase the yield of desired cell types, designing gene circuits that effectively redirect cell fate decisions requires mechanistic insight into the dynamics of endogenous regulatory networks that govern decision-making. In this work, we present a biomolecular adaptive controller based on an Incoherent Feedforward Loop (IFFL)-like topology designed to favor a specific cell fate. This controller requires minimal knowledge of the endogenous network as it exhibits adaptive, non-reference-based behavior. The synthetic circuit operates through a sequestration mechanism and a delay introduced by an intermediate species, producing an output that asymptotically approximates a discrete temporal derivative of its input, provided there is a sufficiently fast sequestration rate. By allowing the controller to actuate over a target species involved in the decision-making process, a tunable, synthetic bias is created that favors the production of the desired species with minimal alteration to the overall equilibrium landscape of the endogenous network. Through theoretical and computational analysis, we provide design guidelines for the controller’s optimal operation, evaluate its performance under parametric perturbations, and extend its applicability to various examples of common multistable systems in biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b540a64fb3aa27b889f01e2b17a8cadd0ea0d35" target='_blank'>
              Engineering cell fate with adaptive feedback control
              </a>
            </td>
          <td>
            Frank Britto Bisso, Giulia Giordano, Christian Cuba Samaniego
          </td>
          <td>2024-11-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad33c5772a7c6f391f659ca36bbfaca426706c80" target='_blank'>
              Improving the odds of survival: transgenerational effects of infections.
              </a>
            </td>
          <td>
            V. Spanou, Theano Andriopoulou, E. Giamarellos‐Bourboulis, Mihai G Netea
          </td>
          <td>2025-01-22</td>
          <td>EMBO molecular medicine</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e4d6b17f62c34eb4147f40c895b9a78313eb007" target='_blank'>
              C3+ cancer-associated fibroblasts promote tumor growth and therapeutic resistance in gastric cancer via activation of the NF-κB signaling pathway
              </a>
            </td>
          <td>
            Zhenxiong Zhao, S. Xiong, Jianpeng Gao, Yingjing Zhang, Ergang Guo, Yakai Huang
          </td>
          <td>2024-12-20</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f23f58ff8f0c69ee80ac960c4ff630df39788c5" target='_blank'>
              LGR5: An emerging therapeutic target for cancer metastasis and chemotherapy resistance
              </a>
            </td>
          <td>
            W. Wang, N. Lokman, Simon C Barry, M. Oehler, C. Ricciardelli
          </td>
          <td>2025-01-17</td>
          <td>Cancer Metastasis Reviews</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1f1a3134c1ae8bfb1d2f982a790258260dcc9c9" target='_blank'>
              Advances in epigenetic therapies for B-cell non-hodgkin lymphoma.
              </a>
            </td>
          <td>
            Weiwen Hu, Lanlan Zang, Xiaoxi Feng, Shuhui Zhuang, Liudi Chang, Yongjing Liu, Jinyan Huang, Yuanyuan Zhang
          </td>
          <td>2024-12-09</td>
          <td>Annals of hematology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Over the last three decades, human genetics has gone from dissecting high-penetrance Mendelian diseases to discovering the vast and complex genetic etiology of common human diseases. In tackling this complexity, scientists have discovered the importance of numerous genetic processes – most notably functional regulatory elements – in the development and progression of these diseases. Simultaneously, scientists have increasingly used multiplex assays of variant effect to systematically phenotype the cellular consequences of millions of genetic variants. In this article, we argue that the context of genetic variants – at all scales, from other genetic variants and gene regulation to cell biology to organismal environment – are critical components of how we can employ genomics to interpret these variants, and ultimately treat these diseases. We describe approaches to extend existing experimental assays and computational approaches to examine and quantify the importance of this context, including through causal analytic approaches. Having a unified understanding of the molecular, physiological, and environmental processes governing the interpretation of genetic variants is sorely needed for the field, and this perspective argues for feasible approaches by which the combined interpretation of cellular, animal, and epidemiological data can yield that knowledge.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc2f3e163ee6e921d0ab8446acca00c0495f358b" target='_blank'>
              Understanding genetic variants in context
              </a>
            </td>
          <td>
            Nasa Sinnott-Armstrong, Stanley Fields, Fritz Roth, Lea M. Starita, Cole Trapnell, Judit Villen, Douglas M. Fowler, C. Queitsch
          </td>
          <td>2024-12-03</td>
          <td>eLife</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Abstract Historically considered downstream effects of tumorigenesis—arising from changes in DNA content or chromatin organization—nuclear alterations have long been seen as mere prognostic markers within a genome‐centric model of cancer. However, recent findings have placed the nuclear envelope (NE) at the forefront of tumor progression, highlighting its active role in mediating cellular responses to mechanical forces. Despite significant progress, the precise interplay between NE components and cancer progression remains under debate. In this review, we provide a comprehensive and up‐to‐date overview of how changes in NE composition affect nuclear mechanics and facilitate malignant transformation, grounded in the latest molecular and functional studies. We also review recent research that uses advanced technologies, including artificial intelligence, to predict malignancy risk and treatment outcomes by analyzing nuclear morphology. Finally, we discuss how progress in understanding nuclear mechanics has paved the way for mechanotherapy—a promising cancer treatment approach that exploits the mechanical differences between cancerous and healthy cells. Shifting the perspective on NE alterations from mere diagnostic markers to potential therapeutic targets, this review calls for further investigation into the evolving role of the NE in cancer, highlighting the potential for innovative strategies to transform conventional cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53ec969f050943713ac6f56187d24a844cc59674" target='_blank'>
              At the nucleus of cancer: how the nuclear envelope controls tumor progression
              </a>
            </td>
          <td>
            F. Paganelli, Alessandro Poli, Serena Truocchio, Alberto M. Martelli, Carla Palumbo, Giovanna Lattanzi, Francesca Chiarini
          </td>
          <td>2025-01-24</td>
          <td>MedComm</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/575e0cd67bcf68fb44b85f6259c1ef6837884bb7" target='_blank'>
              Tousled-like kinase loss confers PARP inhibitor resistance in BRCA1-mutated cancers by impeding non-homologous end joining repair
              </a>
            </td>
          <td>
            Min-ah Kim, Banseok Kim, Jihyeon Jeon, Jonghyun Lee, Hyeji Jang, Minjae Baek, Sang-Uk Seo, Dongkwan Shin, Anindya Dutta, Kyung Yong Lee
          </td>
          <td>2025-01-22</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3950003e8f9c888797ac3d3b9f85eeb622acd4f3" target='_blank'>
              A landscape of X-inactivation during human T cell development
              </a>
            </td>
          <td>
            Björn Gylemo, Maike Bensberg, Viktoria Hennings, C. Lundqvist, A. Camponeschi, Dóra Goldmann, Huan Zhang, Aida Selimović-Pašić, Antonio Lentini, O. Ekwall, Colm E. Nestor
          </td>
          <td>2024-12-04</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Interrogating regulatory epigenetic alterations during tumor progression at the resolution of single cells has remained an understudied area of research. Here we developed a highly sensitive single-nucleus CUT&RUN (snCUT&RUN) assay to profile histone modifications in isogenic primary, metastatic, and cisplatin-resistant head and neck squamous cell carcinoma (HNSCC) patient-derived tumor cell lines. We find that the epigenome can be involved in diverse modes to contribute towards HNSCC progression. First, we demonstrate that gene expression changes during HNSCC progression can be comodulated by alterations in both copy number and chromatin activity, driving epigenetic rewiring of cell states. Furthermore, intratumour epigenetic heterogeneity (ITeH) may predispose subclonal populations within the primary tumour to adapt to selective pressures and foster the acquisition of malignant characteristics. In conclusion, snCUT&RUN serves as a valuable addition to the existing toolkit of single-cell epigenomic assays and can be used to dissect the functionality of the epigenome during cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ccec03801dafc8df15febd551c3a9189ef4f702" target='_blank'>
              Single-nucleus CUT&RUN elucidates the function of intrinsic and genomics-driven epigenetic heterogeneity in head and neck cancer progression.
              </a>
            </td>
          <td>
            H. Womersley, Daniel Muliaditan, Ramanuj DasGupta, L. F. Cheow
          </td>
          <td>2024-12-02</td>
          <td>Genome research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Background Most molecular classifications of cancer are based on bulk-tissue profiles that measure an average over many distinct cell-types. As such, cancer subtypes inferred from transcriptomic or epigenetic data are strongly influenced by cell-type composition and do not necessarily reflect subtypes defined by cell-type specific cancer-associated alterations, which could lead to suboptimal cancer classifications. Methods To address this problem, we here propose the novel concept of cell-type specific combinatorial clustering (CELTYC), which aims to group cancer samples by the molecular alterations they display in specific cell-types. We illustrate this concept in the context of DNA methylation data of liver and kidney cancer, deriving in each case novel cancer subtypes and assessing their prognostic relevance against current state-of-the-art prognostic models. Results In both liver and kidney cancer, we reveal improved cell-type specific prognostic models, not discoverable using standard methods. In the case of kidney cancer, we show how combinatorial indexing of epithelial and immune-cell clusters define improved prognostic models driven by synergy of high mitotic age and altered cytokine signaling. We validate the improved prognostic models in independent datasets, and identify underlying cytokine-immune-cell signatures driving poor outcome. Conclusions In summary, cell-type specific combinatorial clustering is a valuable strategy to help dissect and improve current prognostic classifications of cancer in terms of the underlying cell-type specific epigenetic and transcriptomic alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d052ec02be986508482ae1f68379c54cb0ed96e8" target='_blank'>
              Cell-type specific subtyping of epigenomes improves prognostic stratification of cancer
              </a>
            </td>
          <td>
            Qi Luo, Andrew E. Teschendorff
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The p53 protein has attracted huge research interest over several decades due to its role as one of the most important tumor suppressors in mammals, which orchestrates a synchronous response from normal cells in the body to various forms of stress. The diverse cellular activities of the p53 protein are regulated mainly via its post-translational modifications (PTMs). PTMs affect p53 on several levels: at the level of the assembly of tetrameric complexes on DNA to transactivate its target genes, at the level of the assembly of tetrameric complexes on DNA to transactivate its target genes; at the level of proteolysis in the absence of stress; and on the contrary, at the level of augmented protein stability in response to stress signals. Disruptions in these regulatory mechanisms can lead to deviations from normal cellular function, boosting tumor initiation and progression. Conversely, targeted interventions in these pathways could prove beneficial for the development of antitumor therapies. Advancing our understanding of p53 modifiers and the proteins involved in its regulation equips researchers with an expanded toolkit for studying cellular processes and for developing biologically active molecules that influence p53-mediated responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd3fe1959718caec99cd867c8f3aa0f229e81022" target='_blank'>
              p53: The Multifaceted Roles of Covalent Modifications in Cancer
              </a>
            </td>
          <td>
            Tatiana A. Grigoreva, Angelina A. Romanova, V. Tribulovich, Nikolay B. Pestov, Ruslan A Oganov, Diana K. Kovaleva, T. Korneenko, N. A. Barlev
          </td>
          <td>2024-12-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Abstract Background Chromosomal instability (CIN), a hallmark of cancer, is commonly linked to poor prognosis in high‐grade prostate cancer (PCa). Paradoxically, excessively high levels of CIN may impair cancer cell viability. Consequently, understanding how tumours adapt to CIN is critical for identifying novel therapeutic targets. Methods Bioinformatic analyses were conducted to identify genes overexpressed in PCa tissues using The Cancer Genome Atlas (TCGA) and GEO datasets. Western blotting and immunohistochemistry assays were applied to determine the expression levels of euchromatic histone lysine methyltransferase 2 (EHMT2), pT232‐Aurora B and Cullin 3 (CUL3). The proliferation of cells was measured through CCK‐8 tests, clonogenesis and subcutaneous xenografts of human PCa cells in BALB/c nude mice. Live cell imaging, immunofluorescence (IF) and flow cytometry were used to confirm the role of EHMT2 in PCa cell mitosis. Co‐immunoprecipitation, Western blotting and IF assays further elucidated the underlying molecular mechanisms. Results EHMT2 was highly expressed in metastatic PCa tissues exhibiting elevated CIN and was strongly associated with adverse clinical outcomes in patients with PCa. Silencing EHMT2 impaired cell division, inducing G2/M‐phase arrest and mitotic catastrophe in PCa cells. Mechanistically, EHMT2 is indispensable to ensure the full activation of Aurora B through centromeric R‐loop‐driven ATR–CHK1 pathway, with EHMT2 protein expression peaking during the G2/M‐phase. Moreover, CUL3 was identified as a binding partner of EHMT2, mediating its polyubiquitination and destabilising its protein levels. Conclusions This study reveals a CUL3–EHMT2–Aurora B regulatory axis that safeguards accurate chromosome segregation in PCa cells, supporting the potential therapeutic application of EHMT2 inhibitors. Key points Euchromatic histone lysine methyltransferase 2 (EHMT2) is overexpressed in advanced prostate cancer, restraining catastrophic chromosomal instability (CIN) and enhancing cell fitness. EHMT2 functions via the centromeric R‐loop‐driven ATR–CHK1–Aurora B pathway to promote chromosomal stability. EHMT2 confers enzalutamide resistance via activating Aurora B. Cullin 3 (CUL3) promotes EHMT2 destabilisation via deubiquitination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6cc5ece350ce05d9ec1131335e290684b93468c8" target='_blank'>
              EHMT2‐mediated R‐loop formation promotes the malignant progression of prostate cancer via activating Aurora B
              </a>
            </td>
          <td>
            Yuyang Zhang, Mingqin Su, Yiming Chen, Li Cui, Wei Xia, Renfang Xu, Dong Xue, Xiansheng Zhang, Xingliang Feng
          </td>
          <td>2025-01-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is a highly invasive and highly lethal malignant tumor. Due to the difficulty in early diagnosis and limited treatment options, the 5-year survival rate is less than 10%. Traditional treatments such as surgery, chemotherapy, and radiotherapy have limited effects on PDAC, which is mainly attributed to its unique tumor microenvironment (TME), common gene mutations (such as KRAS, and TP53), and immunosuppressive properties. In recent years, with the advancement of molecular biology and genomics technologies, the molecular mechanisms of PDAC have been studied more deeply, revealing the gene mutations and regulatory networks associated with its pathogenesis. In particular, KRAS mutations have become an important research direction for targeted therapy. However, the effect of immunotherapy in PDAC is limited by the immune escape characteristics of TME. This article systematically summarizes the key gene mutations in PDAC and their regulatory mechanisms, including epigenetic regulation, microRNA, histone modification and other influencing factors. CRISPR-Cas9 shows great potential in correcting gene mutations and reshaping TME. In addition, combining immunotherapy with targeted drug delivery (such as nanotechnology) can improve the precision of treatment and reduce side effects. These advances provide a comprehensive reference for future personalized and effective treatment options for PDAC">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1538e4df893f0e562277858358833ccdc8a4f65" target='_blank'>
              Genetic Mutation and Regulation in Pancreatic Ductal Adenocarcinoma Cancer: Developing Relevant Molecular Biomarkers
              </a>
            </td>
          <td>
            Xiaomi Ai
          </td>
          <td>2025-01-15</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Renal clear cell carcinoma (ccRCC) presents a unique landscape of genetic and epigenetic modifications, the understanding of which is crucial for the development of targeted therapies and improved prognostication. This study explores the differential expression of histone‐related genes (HRGs) in ccRCC and correlates these findings with patient clinical outcomes. By leveraging the Cancer Genome Atlas (TCGA) data, we performed a robust multidimensional analysis of HRG expression profiles in ccRCC versus adjacent normal tissues. Our results demonstrate a significant upregulation of several key epigenetic regulators, including UHRF1, KDM5A, EZH2, PRDM6, and TWIST1, in tumor samples, with statistical significance suggesting their involvement in tumorigenesis and progression. Paired expression analysis within patient‐matched samples confirmed the consistency of overexpression in tumors. The prognostic relevance of these genes was underscored through survival analyses, which revealed a clear stratification of patients into distinct risk categories based on their expression profiles. The integration of these genetic markers with clinical parameters facilitated the development of a predictive nomogram, yielding a quantifiable tool for survival prediction. Our comprehensive analysis elucidates the profound impact of epigenetic dysregulation in ccRCC and proposes a novel set of biomarkers for disease diagnosis, prognostic stratification, and potential therapeutic targeting, marking a significant stride toward precision oncology in renal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/057b9174164151a3b1cfe3112a48500f0dca6ee3" target='_blank'>
              Epigenetic Alterations in Renal Clear Cell Carcinoma: A Comprehensive Expression and Clinical Outcome Analysis
              </a>
            </td>
          <td>
            Dafei Weng, Zhenqian Qin, Xuefeng Yuan, Xuping Jiang, Hua Rui
          </td>
          <td>2025-01-01</td>
          <td>Journal of Clinical Pharmacy and Therapeutics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="DNMT3A and TET2 are epigenetic regulator genes commonly mutated in age-related clonal hematopoiesis (CH). Despite having opposed epigenetic functions, these mutations are associated with increased all-cause mortality and a low risk for progression to hematological neoplasms. While individual impacts on the epigenome have been described using different model systems, the phenotypic complexity in humans remains to be elucidated. Here we make use of a natural inflammatory response occurring during coronavirus disease 2019 (COVID-19), to understand the association of these mutations with inflammatory morbidity (acute respiratory distress syndrome-ARDS) and mortality. We demonstrate the age-independent, negative impact of DNMT3A mutant CH on COVID-19-related ARDS and mortality. Using single cell (sc-) proteogenomics we show that DNMT3A mutations involve myeloid and lymphoid lineage cells. Using single cell multiomics sequencing, we identify cell-specific gene expression changes associated with DNMT3A mutations, along with significant epigenomic deregulation affecting enhancer accessibility, resulting in overexpression of IL32, a proinflammatory cytokine that can result in inflammasome activation in monocytes and macrophages. Finally, we show with single cell resolution that the loss of function of DNMT3A is directly associated with increased chromatin accessibility in mutant cells. Hence, we demonstrate the negative prognostic impact of DNMT3Amt CH on COVID-19 related ARDS and mortality. DNMT3Amt CH in the context of COVID-19, was associated with inflammatory transcriptional priming, resulting in overexpression of IL32. This overexpression was secondary to increased chromatic accessibility, specific to DNMT3Amt CH cells. DNMT3Amt CH can thus serve as a potential biomarker for adverse outcomes in COVID-19.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be6f734acb174ae5d3944a4074bf5564fa474087" target='_blank'>
              Single cell multiomics reveal divergent effects of DNMT3A and TET2 mutant clonal hematopoiesis in inflammatory response.
              </a>
            </td>
          <td>
            Wazim Mohammed Ismail, Jenna A. Fernandez, M. Binder, T. Lasho, Minsuk Kim, Susan M. Geyer, Amelia Mazzone, C. Finke, A. Mangaonkar, Jeong-Heon Lee, Liguo Wang, Kwan Hyun Kim, Vernadette A. Simon, Fariborz Rakhshan Rohakthar, Amik Munankarmy, Seul Kee Byeon, S. Schwager, Jonathan J Harington, Melissa R. Snyder, Keith D Robertson, Akhilesh Pandey, Eric D. Wieben, Nicholas Chia, Alexandre Gaspar-Maia, Mirinal S Patnaik
          </td>
          <td>2024-12-04</td>
          <td>Blood advances</td>
          <td>1</td>
          <td>63</td>
        </tr>

        <tr id="ABSTRACT Background CREB binding protein (CREBBP) is a key epigenetic regulator, altered in a fifth of relapsed cases of acute lymphoblastic leukemia (ALL). Selectively targeting epigenetic signaling may be an effective novel therapeutic approach to overcome drug resistance. Anti‐tumor effects have previously been demonstrated for GSK‐J4, a selective H3K27 histone demethylase inhibitor, in several animal models of cancers. Methods To characterize the effect of GSK‐J4, drug response profiling, CRISPR‐Dropout Screening, BH3 profiling and immunoblotting were carried out in ALL cell lines or patient derived samples. Results Here we provide evidence that GSK‐J4 downregulates cyclic AMP‐responsive element‐binding protein (CREB) and CREBBP in B‐cell precursor‐ALL cell lines and patient samples. High CREBBP expression in BCP‐ALL cell lines correlated with high GSK‐J4 sensitivity and low dexamethasone sensitivity. GSK‐J4 treatment also induced Bcl‐2 and Bcl‐XL dependency and apoptosis. Conclusions This study proposes H3K27 demethylase inhibition as a potential treatment strategy for patients with treatment‐resistant ALL, using CREBBP as a biomarker for drug response and combining GSK‐J4 with venetoclax and navitoclax as synergistic partners.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6268afa1f23d7dedcd6b2c8390d3c0a47ae3fe38" target='_blank'>
              Inhibiting H3K27 Demethylases Downregulates CREB‐CREBBP, Overcoming Resistance in Relapsed Acute Lymphoblastic Leukemia
              </a>
            </td>
          <td>
            Juan Lazaro-Navarro, C. Alcon, M. Dorel, Lina Alasfar, L. Bastian, Claudia Baldus, K. Astrahantseff, M. Yaspo, J. Montero, Cornelia Eckert
          </td>
          <td>2025-01-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09816baa7e13a8a81394d1d42faa90a68058ed21" target='_blank'>
              Stochastic demethylation and redundant epigenetic suppressive mechanisms generate highly heterogeneous responses to pharmacological DNA methyltransferase inhibition
              </a>
            </td>
          <td>
            Mie K. Jakobsen, S. Traynor, A. Y. Nielsen, Christina Dahl, Mette Staehr, S. T. Jakobsen, M. S. Madsen, Rasmus Siersbaek, Mikkel G Terp, Josefine B. Jensen, C. B. Pedersen, Anup Shrestha, J. R. Brewer, P. H. Duijf, Odd L. Gammelgaard, Henrik J Ditzel, Alexei F Kirkin, Per Guldberg, Morten F. Gjerstorff
          </td>
          <td>2025-01-23</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/501a2344c548bcfc786460170b1feb2e0d711776" target='_blank'>
              Sperm-borne small non-coding RNAs: potential functions and mechanisms as epigenetic carriers
              </a>
            </td>
          <td>
            Muhammad Naveed, Zhaokang Shen, Jianqiang Bao
          </td>
          <td>2025-01-17</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4211b81d0b19e4c7f2e0cfa7c734b541dc3fa943" target='_blank'>
              Epigenetics in the modern era of crop improvements.
              </a>
            </td>
          <td>
            Yan Xue, Xiaofeng Cao, Xiangsong Chen, Xian Deng, Xing Wang Deng, Yong Ding, Aiwu Dong, Cheng-Guo Duan, Xiaofeng Fang, L. Gong, Zhizhong Gong, Xiaofeng Gu, Chongsheng He, Hang He, Shengbo He, Xin-Jian He, Yan He, Yuehui He, Guifang Jia, Danhua Jiang, Jianjun Jiang, Jinsheng Lai, Zhaobo Lang, Chenlong Li, Qing Li, Xingwang Li, Bao Liu, Bing Liu, Xiao Luo, Yijun Qi, Weiqiang Qian, Guodong Ren, Qingxin Song, Xianwei Song, Zhixi Tian, Jia-Wei Wang, Yuan Wang, Liang Wu, Zhe Wu, Rui Xia, Jun Xiao, Lin Xu, Zhengyi Xu, Wenhao Yan, Hongchun Yang, J. Zhai, Yijing Zhang, Yusheng Zhao, Xuehua Zhong, Dao-Xiu Zhou, Ming Zhou, Yue Zhou, Bo Zhu, Jian-Kang Zhu, Qikun Liu
          </td>
          <td>2025-01-08</td>
          <td>Science China. Life sciences</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Contemporary studies of Bruton tyrosine kinase inhibitor (BTKi) resistance focus on mutations in the B-cell receptor (BCR) pathway, but alternative mechanisms of resistance remain undefined. Here, we sought to identify novel predictive markers of acquired resistance to acalabrutinib, a second-generation BTKi, in patients with chronic lymphocytic leukemia (CLL). Clinical samples from 41 patients with relapsed/refractory or treatment-naive CLL receiving acalabrutinib as part of a clinical trial (NCT02029443) were divided into two groups: those who continued to respond to treatment (NP, n = 23) and those who developed progressive disease on acalabrutinib therapy (PD, n = 18). Peripheral blood mononuclear cells (PBMCs) from the two groups of patients were profiled at baseline (BL) and at a second timepoint (T2) by RNA-seq and flow cytometry. Our findings show a correlation between acquired resistance to acalabrutinib and upregulation of integrin alpha-4 (ITGA4; CD49d), the BCR surface receptor CD79B, and oncogenes such as MYC, LAG3, and MCL1 in CLL cells. High surface expression of CD49d and CD79b prior to acalabrutinib therapy was associated with increased risk of disease progression on acalabrutinib in patients with CLL. When stratified by pretreatment CD49d surface expression, the CD49dhi group (defined as ≥ 30% CD49d+ cells at baseline) showed reduced acalabrutinib-induced lymphocytosis and higher levels of tumor proliferation markers such as CD38 and Ki-67 compared with the CD49dlo group (defined as < 30% CD49d+ cells at baseline). In summary, CD49d and CD79b are useful predictive markers for CLL progression on acalabrutinib. Trial Registration: ClinicalTrials.gov identifier: NCT02029443.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b053dbd4c2ad2b6255e8a35cc7abc17c7293f46b" target='_blank'>
              Molecular Profiling Identifies CD49d and CD79b as Predictive Markers for Acquired Acalabrutinib Resistance in Patients With Chronic Lymphocytic Leukemia.
              </a>
            </td>
          <td>
            Elena Bibikova, Sara Parsa, Muskan Floren, Brian Law, Tracy Clevenger, Jean P. Cheung, G. de Jesus, K. Burke, Michael Gulrajani, Kyoko Yamaguchi, Phuong Do, Brian Dougherty, David Whitston, Graham Brock, V. Munugalavadla, M. Frigault, T. N. Hartmann, John C. Byrd, R. Furman, Jennifer R. Brown, T. Covey, Andrew Mortlock
          </td>
          <td>2024-12-23</td>
          <td>Hematological oncology</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="For long, genome-wide coverage has been used as a measure of sequencing quality and quantity, but the biology hidden beneath has not been fully exploited. Here we performed comparative analyses on genome-wide coverage profiles between nucleus genome DNA (gDNA) samples from the 1000 Genomes Project (n=3,202) and cell-free DNA (cfDNA) samples from healthy controls (n=113) or cancer patients (n=362). Regardless of sample type, we observed an overall conserved landscape with coverage segmentation, where similar levels of coverage were shared among adjacent windows of genome positions. Besides GC-content, we identified protein-coding gene density and nucleosome density as major factors affecting the coverage of gDNA and cfDNA, respectively. Differential coverage of cfDNA vs gDNA was found in immune-receptor loci, intergenic regions and non-coding genes, reflecting distinct genome activities in different cell types. A further rise in coverage at non-coding genes/intergenic regions and a further drop of coverage at protein-coding genes/genic regions within cancer cfDNA samples suggested a relative loss of contribution by normal cells. Importantly, we observed the distinctive convergence of coverage in cancer-derived cfDNA, with the extent of convergence positively correlated to stages. Based on the findings we developed and validated an outlier-detection approach for cfDNA-based cancer screening without the need of cancer samples for training. The method achieved 97% sensitivity on pediatric sarcomas (n=241) and 44% sensitivity on early-stage lung cancers (n=36) with >90% specificity for condition-matched tasks, 100% sensitivity on late-stage cancers (n=85) for condition-unmatched tasks, outperforming current benchmarks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58763bc618f0e854c5267409a63aba70151b9e8e" target='_blank'>
              Coverage landscape of the human genome in nucleus DNA and cell-free DNA
              </a>
            </td>
          <td>
            Jiaqi Luo, Shuaicheng Li
          </td>
          <td>2024-12-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Single-cell sequencing can provide novel insights into the understanding and treatment of diseases. In cancer, for example, intratumor heterogeneity is a major cause of treatment resistance and relapse. Although technological progress has substantially increased the throughput of sequenced cells, single-cell sequencing remains cost and labor-intensive. Multiplexing, i.e., the pooling and subsequent joint preparation and sequencing of samples, followed by a demultiplexing step, is a common practice to reduce expenses and confounding batch effects, especially in single-cell RNA sequencing. Results Here, we introduce demoTape, a computational demultiplexing method for targeted single-cell DNA sequencing (scDNA-seq) data based on a distance metric between individual cells at single-nucleotide polymorphisms loci. To validate demoTape, we sequence three B-cell lymphoma patients separately and multiplexed on the Tapestri platform. We find similar genotypes, clones, and evolutionary histories in all three samples when comparing the individual with the demultiplexed samples. Using the three individually sequenced samples, we simulate multiplexed ground truth data and show that demoTape outperforms state-of-the-art demultiplexing methods designed for RNA sequencing data. Additionally, we demonstrate through downsampling that the inferred clonal composition remained largely stable for samples with fewer cells despite the inevitable loss in resolution of low-frequency clones. Conclusions Multiplexing and subsequent genotype-based demultiplexing of scDNA-seq will reduce costs and workload, eventually allowing the sequencing of more samples. This will open new possibilities and accelerate the investigation of biological questions where cellular heterogeneity on the genomic level plays a crucial role.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/854d13588a800839a9fb4c222c2fbfc27f0a2f60" target='_blank'>
              DemoTape: Computational demultiplexing of targeted single-cell sequencing data
              </a>
            </td>
          <td>
            Nico Borgsmüller, Jack Kuipers, Johannes Gawron, Marco Roncador, Marcel Pohly, Erkin Acar, Thi Huong Lan Do, Stefanie Reisenauer, Mirjam Judith Feldkamp, C. Beisel, Thorsten Zenz, A. Moor, N. Beerenwinkel
          </td>
          <td>2024-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/937fb72f5c07d87edcc2493e093c8ba719c5c0a0" target='_blank'>
              TET2 cascade: a novel regulator of chromatin structure and leukaemogenesis
              </a>
            </td>
          <td>
            Wolfram C. M. Dempke, K. Fenchel
          </td>
          <td>2025-01-06</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The diverse radiation types in medical treatments and the natural environment elicit complex biological effects on both cancerous and non-cancerous tissues. Radiation therapy (RT) induces oncological responses, from molecular to phenotypic alterations, while simultaneously exerting toxic effects on healthy tissue. N6-methyladenosine (m6A), a prevalent modification on coding and non-coding RNAs, is a key epigenetic mark established by a set of evolutionarily conserved enzymes. The interplay between m6A modification and radiobiology of cancerous and non-cancerous tissues merits in-depth investigation. This review summarizes the roles of m6A in the biological effects induced by ionizing radiation and ultraviolet (UV) radiation. It begins with an overview of m6A modification and its detection methods, followed by a detailed examination of how m6A dynamically regulates the sensitivity of cancerous tissues to RT, the injury response in non-cancerous tissues, and the toxicological effects of UV exposure. Notably, this review underscores the importance of novel regulatory mechanisms of m6A and their potential clinical applications in identifying epigenetically modulated radiation-associated biomarkers for cancer therapy and estimation of radiation dosages. In conclusion, enzyme-mediated m6A-modification triggers alterations in target gene expression by affecting the metabolism of the modified RNAs, thus modulating progression and radiosensitivity in cancerous tissues, as well as radiation effects on normal tissues. Several promising avenues for future research are further discussed. This review highlights the importance of m6A modification in the context of radiation biology. Targeting epi-transcriptomic molecules might potentially provide a novel strategy for enhancing the radiosensitivity of cancerous tissues and mitigating radiation-induced injury to normal tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b51d3e0ec15a1766878bdbaa0b5458c9eed89844" target='_blank'>
              The interplay between RNA m6A modification and radiation biology of cancerous and non-cancerous tissues: a narrative review
              </a>
            </td>
          <td>
            Yajia Cheng, Yue Shang, Shuqin Zhang, Saijun Fan
          </td>
          <td>2024-12-15</td>
          <td>Cancer Biology & Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background: In eukaryotes with a double-stranded linear DNA genome, the loss of terminal DNA during replication is inevitable due to an end-replication problem; here, telomeres serve as a buffer against DNA loss. Thus, the activation of the telomere maintenance mechanism (TMM) is a prerequisite for malignant transformation. Methods: We compared neurofibroma (NF, benign) and malignant peripheral nerve sheath tumors (MPNSTs) occurring in the same patient with type 1 neurofibromatosis, where each NF–MPNST pair shared the same genetic background and differentiation lineage; this minimizes the genetic bias and contrasts only those changes that are related to malignant transformation. A total of 20 NF–MPNST pairs from 20 NF1 patients were analyzed. Whole-transcriptome sequencing (WTS) was conducted to profile the transcriptional relationship, and whole-genome sequencing (WGS) was performed to measure the telomere length. Results: We identified 22 differentially expressed genes (DEGs) during the malignant transformation of MPNSTs. Among them, NELL2 activated PAX7, which sequentially activated RAD52, the recombinase of RAD52-dependent alternative lengthening of telomeres (ALT). RAD52 elongated MPNSTs–telomeres (p = 0.017). Otherwise, neither NELL2 nor PAX7 affected telomere length (p = 0.647 and p = 0.354, respectively). RAD52 increased MPNSTs–telomeres length, independently of NELL2 and PAX7 in multiple analyses (p = 0.021). The group with increased telomere length during the malignant transformation showed inferior overall survival (OS) (HR = 3.809, p = 0.038) to the group without increased telomere length. Accordingly, the group with increased PAX7 showed inferior OS (HR = 4.896, p = 0.046) and metastasis-free survival (MFS) (HR = 9.129, p = 0.007) in comparison to the group without increased PAX7; the group with increased RAD52 showed inferior MFS (HR = 8.669, p = 0.011) in comparison to the group without increased RAD52. Conclusions: We suggest that the NELL2-PAX7 transcriptional cascade activates RAD52-dependent ALT to increase telomere length during the malignant transformation of MPNSTs, resulting in a poor prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec682f73e8b37948b58b7e542b7ea83e47ad44c3" target='_blank'>
              NELL2-PAX7 Transcriptional Cascade Suggests Activation Mechanism for RAD52-Dependent Alternative Lengthening of Telomeres During Malignant Transformation of Malignant Peripheral Nerve Sheath Tumors: Elongation of Telomeres and Poor Survival
              </a>
            </td>
          <td>
            Jungwoo Lee, Eunji Choi, Hyoju Kim, Young-Joon Kim, Seung Hyun Kim
          </td>
          <td>2025-01-23</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Convergent evolution, the evolution of the same or similar phenotypes in phylogenetically independent lineages, is a widespread phenomenon in nature. If the genetic basis for convergent evolution is predictable to some extent, it may be possible to infer organismic phenotypes and the capability of organisms to utilize new ecological resources based on genome sequence data. While repeated amino acid changes have been studied in association with convergent evolution, relatively little is known about the potential contribution of repeated gene copy number changes. In this study, we explore whether gene copy number changes of particular gene families are linked to diet shifts in mammals and assess if trophic ecology can be inferred from the copy numbers of a specific set of gene families. Using 86 mammalian genome sequences, we identified 24 gene families with a trend toward higher copy numbers in herbivores, carnivores, and omnivores, even after phylogenetic corrections. We were able to confirm previous findings on genes such as amylase, olfactory receptors, and xenobiotic metabolism genes, and identify novel gene families whose copy numbers correlate with dietary patterns. For example, omnivores exhibited higher copy numbers of genes encoding regulators of translation. We also established a discriminant function based on the copy numbers of 13 gene families that can help predict trophic ecology to some extent. These findings highlight a possible association between convergent evolution and repeated copy number changes in specific gene families, suggesting the potential to develop a method for predicting animal ecology from genome sequence data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50eead68f5d706a835bb815a993c4fe8a8d340ce" target='_blank'>
              Convergent evolution and predictability of gene copy numbers associated with diets in mammals.
              </a>
            </td>
          <td>
            Kayla Wilhoit, Shun Yamanouchi, Bo-Jyun Chen, Yo Y. Yamasaki, Asano Ishikawa, Jun Inoue, Wataru Iwasaki, Jun Kitano
          </td>
          <td>2025-01-24</td>
          <td>Genome biology and evolution</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/053c90f2e024c025545d0a836542f7f28e9a61c7" target='_blank'>
              KLF5 loss sensitizes cells to ATR inhibition and is synthetic lethal with ARID1A deficiency
              </a>
            </td>
          <td>
            Samah W. Awwad, Colm Doyle, Josie Coulthard, Aldo S. Bader, Nadia Gueorguieva, Simon Lam, Vipul Gupta, R. Belotserkovskaya, Tuan-Anh Tran, Shankar Balasubramanian, S. P. Jackson
          </td>
          <td>2025-01-08</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Our understanding of the vertebrate immune system is dominated by a few model organisms such as mice. This use of a few model systems is reasonable if major features of the immune systems evolve slowly and are conserved across most vertebrates, but may be problematic if there is substantial macroevolutionary change in immune responses. Here, we present a test of the macroevolutionary stability, across 15 species of jawed fishes, of the transcriptomic response to a standardized immune challenge. Intraperitoneal injection of an immune adjuvant (alum) induces a fibrosis response in nearly all jawed fishes, which in some species contributes to anti-helminth resistance. Despite this conserved phenotypic response, the underlying transcriptomic response is highly inconsistent across species. Although many gene orthogroups exhibit differential expression between saline versus alum-injected fish in at least one species, few orthogroups exhibit consistent differential expression across species. This result suggests that although the phenotypic response to alum (fibrosis) is highly conserved, the underlying gene regulatory architecture is very flexible and cannot readily be extrapolated from any one species to fishes (or vertebrates) more broadly. The vertebrate immune response is remarkably changeable over macroevolutionary time, requiring a diversity of model organisms to describe effectively.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02997822d89638ffcc115ee4c55d0d06267f2668" target='_blank'>
              Macroevolutionary changes in gene expression response to an immune stimulus across the diversity of fishes
              </a>
            </td>
          <td>
            Ben A. Flanagan, Lauren E. Fuess, M. Vrtílek, Andrea Roth-Monzón, Daniel I. Bolnick
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The World Health Organization's system for classifying and grading primary tumors of the Central Nervous System conjectures the clinical-biological course of the oncogenic process based on morphological, genetic, histological, and immunohistochemical parameters. These principles are fundamental for a progression in the classification of these tumors, to guarantee the promotion of a more precise diagnosis. In this sense, it is important to understand the process of oncotic cell formation, which is the result of mutations in intra and extracellular control pathways. In this way, genes that act to induce the cell cycle, under normal conditions, when mutated, can result in a dysregulation of the progress of the cycle, causing alterations in the control factors and, consequently, phenotypic transformations in the cell. Thus, to understand the role of genes in modulating primary tumors in the Central Nervous System, mutations in the genes most prevalently related to Gliomas, Meningiomas, and Medulloblastomas were addressed highlighting their influences on the development of these tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76a2d441a03edf17b4beaa6aa54a6e378832f0e6" target='_blank'>
              Cellular Regulation and Oncogenesis of Primary Tumors in the Central Nervous System
              </a>
            </td>
          <td>
            Leonnardo Altoé Miranda Lemos, Caroline Stadler, Izabelly Constantinov Lemos, Junio Pereira Pardins, Victoria Beatriz Podolan Sauka
          </td>
          <td>2024-12-01</td>
          <td>Arquivos Brasileiros de Neurocirurgia: Brazilian Neurosurgery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Increased expression of branched-chain amino acid (BCAA) transaminase 1 (BCAT1) often correlates with tumor aggressiveness and drug resistance in cancer. We have recently reported that BCAT1 was overexpressed in a subgroup of T-cell acute lymphoblastic (T-ALL) samples, especially those with NOTCH1 activating mutations. Interestingly, BCAT1-depleted cells showed pronounced sensitivity to DNA-damaging agents such as etoposide; however, how BCAT1 regulates this sensitivity remains uncertain. Here, we provide further clues on its chemo-sensitizing effect. Indeed, BCAT1 protein regulates the non-homologous end joining (c-NHEJ) DNA repair pathway by physically associating with the KU70/KU80 heterodimer. BCAT1 inhibition during active repair of DNA double-strand breaks (DSBs) led to increased KU70/KU80 acetylation and impaired c-NHEJ repair, a dramatic increase in DSBs, and ultimately cell death. Our results suggest that, in T-ALL, BCAT1 possesses non-metabolic functions that confer a drug resistance mechanism and that targeting BCAT1 activity presents a novel strategy to improve chemotherapy response in T-ALL patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc99a18fefd0d2dfb9db2a0ff7404a49e575c24e" target='_blank'>
              BCAT1 Associates with DNA Repair Proteins KU70 and KU80 and Contributes to Regulate DNA Repair in T-Cell Acute Lymphoblastic Leukemia (T-ALL)
              </a>
            </td>
          <td>
            Valeria Tosello, Chiara Rompietti, A. Papathanassiu, G. Arrigoni, E. Piovan
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background/Objectives: Treatment with tyrosine kinase inhibitors (TKIs) in chronic myeloid leukemia (CML) has revolutionized disease management and has transformed CML from a life-threatening disease to a chronic condition for many patients. However, overcoming resistance, particularly related to leukemic stem cells (LSC) that can persist even when the bulk of the leukemic cells are eliminated, remains a significant challenge. Methods: K562 and KU812 cell lines were treated in vitro with the TKI Imatinib (IM). Gene expression, protein analysis, and metabolomic screening were conducted to investigate the ability of the drug to enhance stem cell (SC) features. Moreover, a gene ontology analysis was performed on different available datasets, to further consolidate our data. Results: 48 h of IM treatment can significantly increase the expression of genes related to SC self-renewal, particularly SOX2 and OCT 3/4. Interestingly, these modulations occur in cells that remain alive after drug treatment and that displayed features consistent with leukemia stem-like CML cells, suggesting that SC genes levels are crucial even in cell population survived upon TKI treatment. Moreover, after in silico analysis of available data, we observed an enrichment of SOX2/NANOG and OCT 3/4 signatures after TKI treatment, thus strengthening our results. Conclusions: Our results confirmed the relevance of LSC features after TKI treatment, highlighting the need for more effective and potentially curative strategies targeting LSCs to overcome resistance in CML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28fb6d29d24fd74f86c83c9e97c902987037fdab" target='_blank'>
              Tyrosine Kinase Inhibitor Therapy Enhances Stem Cells Profile and May Contribute to Survival of Chronic Myeloid Leukemiastem Cells.
              </a>
            </td>
          <td>
            Simone Rocco, A. Maglione, Valentina Schiavo, Alessandro Ferrando, Carmen Fava, D. Cilloni, Barbara Pergolizzi, C. Panuzzo
          </td>
          <td>2025-01-10</td>
          <td>Journal of clinical medicine</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Numerical abnormalities in chromosomal states, referred to as aneuploidy, is commonly observed in many cancer cells. Although numerous internal and external factors induce aneuploidy, the primary cause of aneuploidy in humans remains unclear. DNA damage is identified as a potential cause of aneuploidy by inducing chromosome segregation errors. However, a direct relationship between DNA damage and aneuploidy remains poorly understood. A major reason for this is the extremely low frequency of aneuploidy in cultured cells, making quantitative analyses challenging. In this study, we investigated the relationship between DNA damage and aneuploidy in cell lines containing minichromosomes. These chromosomes are more prone to loss than normal chromosomes, with the rate of loss substantially increased following exposure to various DNA-damaging agents. To determine whether damaged chromosomes were subjected to direct loss or whether chromosome loss occurred as an indirect consequence of a prolonged G2 phase or other factors, we used the CRISPR-Cas9 system to introduce a single DNA double-strand break (DSB) on a minichromosome. The rate of minichromosome loss increased by approximately seven-fold compared with that of the control. Furthermore, the loss rate was significantly elevated in the absence of KU70, a key factor in non-homologous end joining, and upon inhibition of ataxia telangiectasia mutated (ATM), a DNA damage checkpoint protein. Finally, two closely spaced nicks, believed to generate a 5’-overhang, were also shown to induce minichromosome loss. These findings indicated that a single DSB or two closely spaced nicks can cause aneuploidy if improperly repaired in vertebrates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/657f1468476733a77f4e32e0f0402d6f84ad1e95" target='_blank'>
              Quantitative analysis of the frequency of chromosome loss following DSB induction
              </a>
            </td>
          <td>
            Seiya Matsuno, Ryo Ishida, Ryotaro Kawasumi, Kouji Hirota, Takuya Abe
          </td>
          <td>2025-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Antibiotic resistance is frequently observed shortly after the clinical introduction of an antibiotic. Whether and how frequently that resistance occurred before the introduction is harder to determine, as isolates could not have been tested for resistance before an antibiotic was discovered. Historical collections, like the British National Collection of Type Cultures (NCTC), stretching back to 1885, provide a window into this history. Here we match 1,817 sequenced high-quality genomes from the NCTC collection to their respective year of isolation to study resistance genes before and concurrent with the age of antibiotics. Concordant with previous work, we find resistance genes in both pathogens and environmental samples before the age of antibiotics. While generally rare before the introduction of an antibiotic, we find an associated increase in frequency with antibiotic introduction. Finally, we observe a trend of resistance elements becoming both increasingly mobile and nested within multiple mobile elements as time goes on. More broadly, our findings suggest that likely-functional antibiotic resistance genes were circulating in clinically relevant isolates before the age of antibiotics, but human usage is associated with increasing both their overall prevalence and mobility. DATA SUMMARY Genome assemblies downloaded and analyzed are in Supplementary Table 1, and computational tools used are found in the Methods. The authors confirm all supporting data, code and protocols have been provided within the article or through supplementary data files. Impact statement Historical collections of microbial isolates enable researchers to both investigate the past and identify interesting trends over time. In this study, we queried over 1,800 isolate genomes in one such collection for genomic variation linked to antibiotic resistance. We show that numerous isolates cultured before the introduction of a given antibiotic contain genomic variation linked to antibiotic resistance; however, this phenomenon remained relatively rare. We demonstrate a strong association between the year a given antibiotic was clinically introduced and a rise in prevalence of genomic resistance to that antibiotic. Finally, we show that while mobile elements are common throughout the isolates and timeframe analyzed, genomic resistance has become increasingly mobile as time has gone on. This study shows that as expected, the clinical introduction of a given antibiotic is correlated with an increase in resistance to that antibiotic but also was linked with increased mobility of genes and alleles conferring resistance. However, we note that the effect of deposition bias in the collection cannot be excluded. Our work also indicates that numerous microbial pangenomes of pathogens naturally contained genomic resistance to a given antibiotic even before anthropogenic use of that antibiotic. Taken together, we demonstrate that although human use may affect the prevalence and mobility of genomic resistance in clinical isolates; for most antibiotics, genomic resistance existed within the pangenomes of sampled pathogens prior to clinical introduction. Quantifying and understanding the impact of antibiotic introduction in the past helps us understand how the introduction of novel antibiotics can impact bacteria; allowing better reaction to novel resistant infections as they arise.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f85d7249535977636b664b4d8c4d2bb8093b3e6" target='_blank'>
              Genomic resistance in historical clinical isolates increased in frequency and mobility after the age of antibiotics
              </a>
            </td>
          <td>
            Arya Kaul, Célia Souque, Mische Holland, M. Baym
          </td>
          <td>2025-01-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [11, 3],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>