<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-10-06 09:30:14 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="DHFR gene amplification is commonly present in methotrexate (MTX)-resistant colon cancer cells and acute lymphoblastic leukemia. In this study, we proposed an integrative framework to characterize the amplified region by using a combination of single-molecule real-time sequencing, next-generation optical mapping, and chromosome conformation capture (Hi-C). We identified an amplification unit spanning 11 genes, from the DHFR gene to the ATP6AP1L gene position, with high adjusted interaction frequencies on chromosome 5 (~2.2 Mbp) and a twenty-fold tandemly amplified region, and novel inversions at the start and end positions of the amplified region as well as frameshift insertions in most of the MSH and MLH genes were detected. These mutations might stimulate chromosomal breakage and cause the dysregulation of mismatch repair. Characterizing the tandem gene-amplified unit may be critical for identifying the mechanisms that trigger genomic rearrangements. These findings may provide new insight into the mechanisms underlying the amplification process and the evolution of drug resistance. Sequencing a large region of DNA containing many surplus copies of genes linked to drug resistance in colon cancer cells may illuminate how these genomic rearrangements arise. Such regions of gene amplification are highly repetitive, making them impossible to sequence using ordinary methods, and little is known about how they are generated. Using advanced methods, Jeong-Sun Seo at Seoul National University Bundang Hospital in South Korea and co-workers sequenced a region of gene amplification in colon cancer cells. The amplified region was approximately 20 times the length of that in healthy cells and contained many copies of an eleven-gene segment, including a gene implicated in drug resistance. The region also contained mutations in chromosomal repair genes which would disrupt repair pathways. These results illuminate the genetic changes that lead to gene amplification and drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental & Molecular Medicine, Experimental and Molecular Medicine</td>
          <td>5</td>
          <td>49</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum.
                </a>
              </td>
          <td>
            Gustavo S. França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>62</td>
          <td>45</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Phenotypic plasticity is a recognized mechanism of therapeutic resistance in prostate cancer (PCa), however current knowledge of driver mechanisms and therapeutic interventions are limited. Using genetically engineered mouse models (GEMMs) devoid of Pten and Rb1, we previously demonstrated the chromatin reprogramming factor enhancer of zeste homolog 2 (EZH2) as an important regulator of alternative transcription programs promoting phenotypic plasticity. Here, using a multi-omics approach we demonstrate that EZH2 regulates multilineage cell states dependent on the RNA binding protein Tristetraprolin (TTP) that mediates RNA stability and activation of translation. Combined chemical inhibition of EZH2 and PI3K/mTORC1 resulted in superior anti-tumor activity in murine and human phenotypic plastic models and was most significant when this combination was used with castration or enzalutamide. Together, these data indicate phenotypic plasticity dependence on coordination between EZH2, TTP and mTORC1 signaling that represent novel therapeutic dependencies for this lethal PCa phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da6b5a736953854485b4e25a30ec801bc37c5359" target='_blank'>
              EZH2-TTP-mTORC1 Axis Drives Phenotypic Plasticity and Therapeutic Vulnerability in Lethal Prostate Cancer
              </a>
            </td>
          <td>
            Beatriz German, Katherine L. Morel, Teia Noel, N. Boufaied, Deborah L. Burkhart, Sujun Chen, Felipe Dezem, Xintao Qui, Henry W. Long, Stefan DiFazio, S. Baca, Ayesha A. Shafi, Matthew L Freedman, H. Beltran, Christopher J Sweeney, H. He, Myles Brown, Jasmine T Plummer, Simon R V Knott, David P. Labbé, Leigh Ellis
          </td>
          <td>2025-08-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="Platinum-based chemotherapy resistance remains a critical barrier in colorectal cancer (CRC) treatment. In this study, cytogenetic karyotyping was combined with transcriptomic profiling (RNA-seq) to elucidate resistance mechanisms by comparing two CRC cell lines: oxaliplatin-sensitive HCT116 and its resistant derivative HCT116 oxpl-R. Karyotyping unveiled tetraploidization and extensive genomic rearrangements in resistant cells, accompanied by prominent transcriptomic changes: 1807 differentially expressed genes (1216 upregulated and 519 downregulated). Pathway enrichment highlighted altered redox homeostasis and metabolic adaptation. Specifically, HCT116 oxpl-R cells exhibited elevated reactive oxygen species (ROS) production and enhanced energy metabolism. These findings establish a direct association between structural genomic alterations, transcriptional rewiring, and functional phenotypes in platinum resistance, providing a framework for targeting metabolic vulnerabilities in refractory CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4df1c5397ea3bf691ad1c8a6e4e4cf327eb00a1d" target='_blank'>
              Transcriptomic and Cytogenetic Analysis of Oxaliplatin-Resistant Colorectal Adenocarcinoma HCT116 Cells to Identify Markers Associated with Platinum Resistance
              </a>
            </td>
          <td>
            A. Morshneva, O. Gnedina, Ksenia Fedotova, Natalija Yartseva, Nikolay Aksenov, M. Igotti
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5cd5596bbafd3812efcd1d8c0a4bfdf21b8096e0" target='_blank'>
              Adaptive cancer therapy: can non-genetic factors become its achilles heel?
              </a>
            </td>
          <td>
            Gábor Valcz, Robert A. Gatenby, B. Újvári, E. Buzás, Béla Molnár
          </td>
          <td>2025-09-22</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The development of drug-resistant cell lines is essential for understanding the mechanisms of drug resistance and identifying strategies to overcome treatment failure in cancer therapy. Resistance models enable preclinical evaluation of novel compounds, repurposed drugs, and combination therapies. To generate resistant cells, parental cancer cell lines are repeatedly exposed to incrementally increasing concentrations of the target drug over several weeks. Cells that survive and proliferate at each stage are selected, expanded, and exposed to higher drug doses. The development of resistance is confirmed by quantifying and comparing the half-maximal inhibitory concentration (IC50) values between parental and resistant cells using cell viability assays and nonlinear regression analysis. Significantly increased IC50 values indicate successful adaptation to drug pressure and the development of resistance. These drug-resistant cell lines are available for comprehensive analysis, such as microarray and single-cell sequencing, as well as various in vitro or in vivo experiments. These models provide valuable tools for investigating potential therapeutic strategies to overcome drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcb7439fafb8c53daf4f72d4753cdd3f7c5b1b9f" target='_blank'>
              Development of Drug-resistant Cell Lines for Experimental Procedures.
              </a>
            </td>
          <td>
            S. Kadomoto, G. Shelley, Atsushi Mizokami, Evan T. Keller
          </td>
          <td>2025-08-12</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Drug-tolerant persister (DTP) cells are a subpopulation of cancer cells capable of surviving therapeutic stress through reversible, non-genetic adaptations. These cells contribute to minimal residual disease and eventual tumor relapse. Understanding the mechanisms that govern the entry into and exit from the DTP state—such as epigenetic remodeling, metabolic rewiring, and transcriptional plasticity—reveals actionable vulnerabilities. This article reviews the biological basis of DTP reversibility, outlines the major challenges in targeting these cells, and proposes innovative therapeutic strategies including epigenetic inhibitors, metabolic disruptors, and adaptive dosing regimens. We also highlight the importance of biomarker development and dynamic monitoring. Targeting DTP cells at their reversible stage may prevent permanent resistance, offering a promising avenue to improve treatment durability and patient outcomes in cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1eb48d2762dc332cbef357e887ed9a593167f02" target='_blank'>
              Drug-tolerant persister cell in cancer: reversibility, microenvironmental interplay, and therapeutic strategies
              </a>
            </td>
          <td>
            Haifeng Li, Wenlong Xu, Wenqi Cheng, Guanxiao Yu, Dong Tang
          </td>
          <td>2025-08-14</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Most advanced melanomas initially respond to targeted therapy but eventually relapse. Rather than acquiring new mutations, resistance is driven by drug-tolerant persister cells that enter a reversible drug-refractory state. We developed MeRLin, a high-resolution lineage tracing platform integrating cellular barcoding, single-cell transcriptomics, RNA fluorescence in situ hybridization (FISH), and computational analyses to track clonal and transcriptional dynamics in patient-derived melanoma models during prolonged therapy. Clonal dynamics revealed that persister subpopulations first responded to treatment but persisted and expanded during minimal residual disease, ultimately leading to tumor recurrence. Pre-treatment melanoma populations diversified into four conserved persister states characterized by stress-like, lipid metabolism, PI3K signaling, and extracellular matrix remodeling programs associated with adaptive resistance. Spatial transcriptomics showed the organization of these adaptive programs and a complex signaling network of autocrine and paracrine interactions among persister subpopulations. Barcoded RNA-FISH enabled spatial mapping of clonal identity and gene expression, revealing in situ co-localization of a dominant resistant clone with SLC2A1 expression. MeRLin provides a robust framework for dissecting cancer heterogeneity and identifying vulnerabilities in persister populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c4ead317fd2f11125da9ef801fb0c7bcc0217f3" target='_blank'>
              Clonal dynamics shaped by diverse drug-tolerant persister states in melanoma resistance
              </a>
            </td>
          <td>
            Haiyin Li, Yeqing Chen, Jessica Kaster, Maggie Dunne, Min Xiao, Ling Li, Monzy Thomas, Nazifa Promi, Dylan Fingerman, G. Brown, Qiuxian Zheng, Xingyue Zhu, McKenna Reale, Andrew Patterson, Le Gao, Xuxiang Zhang, Siqi Jiang, Tianxing Hu, Hanzhang Fang, Jianlan Ren, Cong Qi, Luyang Wang, Haiwei Mou, Gatha Thacker, Eric Ramirez Salazar, J. Villanueva, Arjun Raj, Dave SB Hoon, Tian Bin, Jozef Madzo, Zhi Wei, Noam Auslander, M. Herlyn
          </td>
          <td>2025-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>117</td>
        </tr>

        <tr id="The mismatch repair system is critical for correcting base–base mismatches and insertion-deletion loops during DNA replication. Deficiencies in MMR (due to mutations in MLH1, MSH2, MSH6, or PMS2) lead to microsatellite instability and contribute to the development of various cancers, such as Lynch syndrome-related colorectal cancer and sporadic tumors. This review will delve into the molecular basis of MMR deficiency. Additionally, the review will cover diagnostic approaches for detecting MSI and MMR deficiency, including next-generation sequencing and PCR-based methods. The implications for treatment will be discussed, emphasizing immune checkpoint inhibitors (e.g., pembrolizumab) that target tumors with high mutational burdens due to MMR deficiency, as well as novel therapeutic approaches like synthetic lethality exploiting DNA repair vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15c3fe776b0060c3cc36839d638f2791e2a3b03a" target='_blank'>
              Mismatch Repair Deficiency and the Role of Non-Canonical Functions in Cancer: Diagnosis and Therapeutic Implications
              </a>
            </td>
          <td>
            Alicja Dąbrowska, Jakub Mastalerz, Zofia Łapińska, I. Deszcz, Agnieszka Chwiłkowska, Nina Rembiałkowska
          </td>
          <td>2025-09-24</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Aim: Growing evidence points to non-genetic mechanisms underlying long-term resistance to cancer therapies. These mechanisms involve pre-existing or therapy-induced transcriptional cell states that confer resistance. However, the relationship between early transcriptional responses to treatment and the eventual emergence of resistant states remains poorly understood. Furthermore, it is unclear whether such early resistance-associated transcriptional responses are evolutionarily conserved. In this study, we examine the similarity between early transcriptional responses and long-term resistant states, assess their clinical relevance, and explore their evolutionary conservation across species. Methods: We integrated datasets on early drug responses and long-term resistance from multiple cancer cell lines, bacteria, and yeast to identify early transcriptional changes predictive of long-term resistance and assess their evolutionary conservation. Using genome-wide CRISPR-Cas9 knockout screens, we evaluated the impact of genes associated with resistant transcriptional states on drug sensitivity. Clinical datasets were analyzed to explore the prognostic value of the identified resistance-associated gene signatures. Results: We found that transcriptional states observed in drug-naive cells and shortly after treatment overlapped with those seen in fully resistant populations. Some of these shared features appear to be evolutionarily conserved. Knockout of genes marking resistant states sensitized ovarian cancer cells to Prexasertib. Moreover, early resistance gene signatures effectively distinguished therapy responders from non-responders in multiple clinical cancer trials and differentiated premalignant breast lesions that progressed to malignancy from those that remained benign. Conclusion: Early cellular transcriptional responses to therapy exhibit key similarities to fully resistant states across different drugs, cancer types, and species. Gene signatures defining these early resistance states have prognostic value in clinical settings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af4c9b8cf4452549a749a1c9bb16bff08ecee299" target='_blank'>
              Resistance signatures manifested in early drug response across cancer types and species
              </a>
            </td>
          <td>
            Cole Ruoff, Allison V. Mitchell, Priya Mondal, Vishaka Gopalan, Arashdeep Singh, MM Gottesman, Sridhar Hannenhalli
          </td>
          <td>2025-08-26</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Resistance to targeted cancer therapies is a significant barrier to favorable treatment outcomes. Malignant cells can tolerate and resist drug treatments due to their biological flexibility. Specifically, slow-cycling drug-resistant cells may achieve permanent resistance to the treatment or restore sensitivity upon cessation of therapy. Enhancing cancer treatment methodologies necessitates a deeper understanding of the adaptability of tumor cells. Drug resistance and cellular heterogeneity are closely associated with cancer cell adaptability. Alterations in cellular signaling, interactions with the tumor microenvironment, and genetic and epigenetic alterations are all implicated. Analyzing these pathways will enhance our understanding of how cancer cells evolve and evade treatment. Two effective strategies to address cancer cell adaptability are to target specific biological pathways and to employ combination therapies. The progression of cancer therapy methodologies relies on comprehending and exploring the concept of cancer cell adaptability. Understanding tumor heterogeneity and drug resistance necessitates identifying the cellular, molecular, and genetic processes that govern cancer cell plasticity. This understanding enables the development of more personalized and effective cancer therapies, leading to improved treatment outcomes. Clinical trial number Not applicable.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/714e22df09170482d5d0d601474075162f529a30" target='_blank'>
              Cancer cell plasticity and therapeutic resistance: mechanisms, crosstalk, and translational perspectives
              </a>
            </td>
          <td>
            Saeid Ghorbian
          </td>
          <td>2025-09-26</td>
          <td>Hereditas</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fa29e8cd8f6fdd768b529f92b9275ea04ce76dc" target='_blank'>
              Genetic landscape of cancer: mechanisms, key genes, and therapeutic implications.
              </a>
            </td>
          <td>
            Arun Karnwal, Joydeep Dutta, Aqueel-Ur-Rehman, Abdel Rahman Mohammad Said Al-Tawaha, Natalia Nesterova
          </td>
          <td>2025-08-17</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb14185df59742e1c30589cf1835a68e0538e9b9" target='_blank'>
              Tracking clonal evolution during treatment in ovarian cancer using cell-free DNA.
              </a>
            </td>
          <td>
            Marc J. Williams, I. Vázquez-García, Grittney K. Tam, Michelle Wu, Nancy Varice, Eliyahu Havasov, Hongyu Shi, Duaa H Al-Rawi, Gryte Satas, Hannah Lees, J. Lee, Matthew A. Myers, Matthew Zatzman, Nicole Rusk, Emily Ali, Ronak H Shah, Michael F. Berger, Neeman Mohibullah, Yulia Lakhman, Dennis S Chi, N. Abu-Rustum, Carol Aghajanian, Andrew McPherson, Dmitry Zamarin, Brian Loomis, Britta Weigelt, C. Friedman, Sohrab P. Shah
          </td>
          <td>2025-10-01</td>
          <td>Nature</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="


 KRAS is a major oncogenic driver in pancreatic ductal adenocarcinoma (PDAC), mutationally activated in approximately 90% of cases. Mutations in this oncogene have been associated with more aggressive disease, poorer outcomes, and have remained hard to drug for over three decades. Recently developed small molecule inhibitors of KRAS (KRASi) have shown promising efficacy in advanced, previously treated PDAC patients but ultimately, most patients develop acquired resistance. Approximately half of these resistance cases do not present a putative genomic driver. In preclinical studies using genetically engineered mouse models of PDAC, we and others have demonstrated that cell state identity along the classical/epithelial-mesenchymal axis is a key determinant of response and resistance to KRASi. Notably, acute KRASi treatment initially induces a strong and selective bottlenecking of the malignant population in vivo, resulting in tumors that are predominantly classical with depleted mesenchymal characteristics. While these findings highlight a potent cell state-selective effect of acute KRASi treatment on PDAC cells, the underlying mechanisms driving the drug-induced cell state selection and plasticity remain poorly characterized. These mechanisms may reveal novel therapeutic targets for developing combination therapies that improve therapeutic responses. To comprehensively map and modulate epithelial-mesenchymal (E-M) cell state plasticity in response to KRASi treatment, we performed CRISPRi Perturb-seq (single-cell gene expression readout) with lineage tracing on a patient-derived PDAC cell line treated with the RAS(ON) multi-selective inhibitor RMC-7977. We captured 734,092 high-quality single-cell transcriptomic profiles and 156,119 unique clones across 60 genetic perturbations, including E- and M-specific transcription factors (TF) inferred to influence E-M plasticity from previous single-cell lineage tracing experiments. To optimally model clonal growth and transition rates, we profiled cells across a time series at day 6, day 15 and day 22, after 1 week of DMSO or KRASi treatment. We developed a hierarchical generative probabilistic model to jointly infer E↔M transition rates and cell state-specific growth rates, while capturing the effects of perturbations and KRASi treatment on state dynamics. Analysis of longitudinally tracked clones revealed that KRASi treatment specifically reduced the M-state growth rate while increasing the probability of M→E transitions compared to DMSO. Notably, we identified several TFs, displaying KRASi-dependent changes in state transition dynamics, either promoting (ELF3, MEIS2 among others) or decreasing (ZNF281, IRF9, among others) probabilities of M→E transitions when compared to non-targeting controls. These preclinical findings suggest that perturbation of these factors may be used as a paradigm to prevent cell state plasticity upon acute KRASi treatment and could be used to homogenize tumor populations towards a treatment-sensitive state.



 Julien Dilly, Mike Bogaev, Lynn Bi, Abigail Collins, Martin Jankowiak, Aziz Al'Khafaji, Kyle E. Evans, Mehrtash Babadi, Thouis R. Jones, Elisa Donnard, David T. Ting, Nir Hacohen, Dana Pe'er, Eric S. Lander, Andrew J. Aguirre, Arnav Mehta. Mapping and modulating epithelial-mesenchymal plasticity under RAS(ON) multi-selective inhibition in PDAC through lineage tracing and Perturb-seq [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B120.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/095259190df0afbd227ee5a60c0db74dfea61a15" target='_blank'>
              Abstract B120: Mapping and modulating epithelial-mesenchymal plasticity under RAS(ON) multi-selective inhibition in PDAC through lineage tracing and Perturb-seq
              </a>
            </td>
          <td>
            Julien Dilly, Mike Bogaev, Lynn Bi, Abigail Collins, Martin Jankowiak, Aziz Al’Khafaji, Kyle E Evans, M. Babadi, , Elisa Donnard, David T Ting, N. Hacohen, D. Pe’er, Eric S. Lander, A. Aguirre, Arnav Mehta
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>114</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is characterized by a pronounced inter- and intra-tumoral heterogeneity, which fuels chemoresistance and contributes to high mortality rates. Previously, we have developed a branched organoid system embedded in collagen matrices that robustly recapitulates the phenotypic heterogeneity seen in both murine and human PDAC1,2. These organoids display complex, self-organized branching morphogenesis and give rise to distinct, spatially ordered tumor cell populations that reflect their underlying molecular profiles and differentiation states. Importantly, we show that the observed heterogeneity is not random but governed by defined transcriptional programs, particularly epithelial-to-mesenchymal plasticity, that drive the emergence of discrete tumor-cell states. Using integrated phenotypic and transcriptomic profiling, we map this diversity to specific biological functions in vivo, demonstrating that each organoid phenotype corresponds to a tumor-cell state with unique metastatic potential, and therapeutic vulnerabilities. Moreover, we identify dynamic, treatment-induced phenotype reprogramming events that are targetable, paving the way for rational design of state-specific therapeutic interventions. Building on our findings, we are now focusing on elucidating intra-organoid heterogeneitysuch as tip–trunk hierarchies at the single-cell level, to uncover the mechanisms of self-organization and to determine how distinct organoid phenotypes and subpopulations contribute to liver metastasis and treatment resistance. In summary, we have established a scalable and mechanistically informative organoid platform that enables in vitro modeling of PDAC heterogeneity. This system provides a framework for dissecting the tumor cell–intrinsic drivers of phenotypic plasticity and for developing phenotype-guided treatment strategies aimed at overcoming resistance and improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47eb75a781046a1e97c826bc3797a33d46a03686" target='_blank'>
              LEVERAGING ORGANOID TECHNOLOGIES TO CHARACTERIZE DYNAMIC TUMOR CELL STATES DRIVING TREATMENT RESISTANCE IN PANCREATIC CANCER
              </a>
            </td>
          <td>

          </td>
          <td>2025-08-21</td>
          <td>European Journal of Histochemistry</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Phenotypic plasticity is a recognized mechanism of therapeutic resistance in prostate cancer (PCa), however current knowledge of driver mechanisms and therapeutic interventions are limited. Using genetically engineered mouse models (GEMMs) devoid of Pten and Rb1, we previously demonstrated the chromatin reprogramming factor enhancer of zeste homolog 2 (EZH2) as an important regulator of alternative transcription programs promoting phenotypic plasticity. Here, using a multi-omics approach we demonstrate that EZH2 regulates multilineage cell states dependent on the RNA binding protein Tristetraprolin (TTP) that mediates RNA stability and activation of translation. Combined chemical inhibition of EZH2 and PI3K/mTORC1 resulted in superior anti-tumor activity in murine and human phenotypic plastic models and was most significant when this combination was used with castration or enzalutamide. Together, these data indicate phenotypic plasticity dependence on coordination between EZH2, TTP and mTORC1 signaling that represent novel therapeutic dependencies for this lethal PCa phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb1223372b66c898bfc66493d1d8cf7781f04f51" target='_blank'>
              EZH2-TTP-mTORC1 Axis Drives Phenotypic Plasticity and Therapeutic Vulnerability in Lethal Prostate Cancer
              </a>
            </td>
          <td>
            Leigh Ellis, Beatriz German, Katherine L. Morel, Teia Noel, Nadia Boufaied, Deborah L. Burkhart, Sujun Chen, Felipe Dezem, Xintao Qiu, H. Long, Stefan DiFazio, S. Baca, Ayesha Shafi, Matthew L Freedman, H. Beltran, Christopher J Sweeney, H. He, Myles Brown, Jasmine T Plummer, Simon Knott, David Labbé
          </td>
          <td>2025-08-21</td>
          <td>Research Square</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="


 Cancers develop and evolve by overcoming a variety of selective pressures including cell-intrinsic growth-inhibitory mechanisms, microenvironmental stresses, and therapeutic interventions. In addition to mutational mechanisms, cancer cell adaptation via transcriptional plasticity remains an important but poorly understood driver of cancer cell fitness. For example, ctDNA analysis of disease progression samples after KRAS inhibitor (KRASi) treatment identified genomically-defined mechanisms of resistance in only approximately 50% of patients, suggesting that non-genomic mechanisms play an equally important role. Thus, there is a pressing need to devise systematic and scalable methods to determine essential drivers of prognostically-relevant pancreatic cancer cell states, and to engineer state-relevant models that can be applied to discover new therapeutic approaches that overcome these modes of resistance. Here, we present a strategy to systematically identify transcriptional drivers of inflammation- and therapy-resistant cell states in pancreatic cancer. We have established a robust screening pipeline to 1) over-express human transcription factor (TF) isoforms in a pooled format within cancer models; 2) perform positive-selection screens under a variety of selective pressures to define transcriptional mechanisms of inflammation tolerance and KRASi resistance; 3) perform single-cell RNA sequencing to define resistance phenotypes; and 4) conduct mechanistic studies to examine the downstream signaling networks that facilitate cancer cell growth in the setting of these selective pressures. We uncover TFs and states that drive resistance in a tissue- and perturbation-specific manner, as well as states that are robust to multiple different selective pressures. A subset of TFs and associated gene programs induce tolerance to chronic interferon exposure in our screens and are reflected in vivo in the setting of immunotherapy resistance. We also identify distinct TF families that drive resistance to pharmacologic KRAS inhibition. Notably, cancer cell states that confer resistance to KRAS inhibition also exhibit cross-resistance to other MAPK pathway inhibitors and to microenvironmental signals, suggesting that cell state adaption may contribute to increased cancer cell fitness and multi-drug resistance as cancers progress. More broadly, our strategy enables the systematic identification and mapping of cell states that drive cancer cell fitness to in vivo clinical atlas datasets, providing a framework for interpretation of clinical phenotypes and predictive modeling of cell state evolution. Furthermore, the cell state-defined models developed here provide a substrate for new therapeutic discovery using cancer cell states as biomarkers, a key next step to developing combination strategies that overcome therapeutic resistance and improve outcomes for pancreatic cancer patients.



 Yuzhou Evelyn. Tong, Aswanth Mahalingam, Zixin Chen, Jacob Smigiel, Tsukasa Shibue, Andew Navia, Alex K. Shalek, Lorin Crawford, Ava Amini, Peter S. Winter, Srivatsan Raghavan. Defining cancer cell states that drive inflammation tolerance and KRAS inhibitor resistance in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B103.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1dd25ee2338a106f19e6d004e36d07be5c2aef6" target='_blank'>
              Abstract B103: Defining cancer cell states that drive inflammation tolerance and KRAS inhibitor resistance in pancreatic cancer
              </a>
            </td>
          <td>
            , Aswanth Mahalingam, Zixin Chen, Jacob Smigiel, Tsukasa Shibue, Andew Navia, Alex K. Shalek, Lorin Crawford, Ava P. Amini, Peter S. Winter, Srivatsan Raghavan
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4da08fc2c51aefcac002abe7af73df18eb83fa23" target='_blank'>
              Integrative proteo-genomic profiling uncovers key biomarkers of lapatinib resistance in HER2-positive breast cancer.
              </a>
            </td>
          <td>
            J. Steggall, V. Rajeeve, N. Al-Subaie, A. Naeem, A. Ikram, A. Naeem, A. Hayat
          </td>
          <td>2025-09-13</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Background/Objectives: Next-Generation Sequencing (NGS) has transformed cancer diagnostics and treatment by enabling comprehensive genomic profiling of tumors. This review aims to summarize the current applications of NGS in oncology, highlighting its role in early detection, precision therapy, and disease monitoring. Methods: We conducted a comprehensive review of the recent literature, focusing on the application of NGS in cancer care. Results: NGS enables high-resolution genomic profiling, identifying actionable mutations (e.g., EGFR, KRAS, and ALK) and immunotherapy biomarkers (e.g., PD-L1, TMB, and MSI), guiding personalized treatment selection and improving outcomes in advanced malignancies. Liquid biopsy enhances diagnostic accessibility and enables real-time monitoring of minimal residual disease and treatment resistance. Despite these advances, widespread clinical adoption remains constrained by technical limitations (e.g., coverage uniformity and sample quality), economic challenges (high costs and complex reimbursement), and interpretative issues, including the management of variants of uncertain significance (VUSs). Conclusions: NGS is central to precision oncology, enabling molecularly driven cancer care. Integration with artificial intelligence, single-cell sequencing, spatial transcriptomics, multi-omics, and nanotechnology promises to overcome current limitations, advancing personalized treatment strategies. Standardization of workflows, cost reduction, and improved bioinformatics expertise are critical for its full clinical integration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29924fb52e872fdb917a78ad329fe64a89de4bc1" target='_blank'>
              Next-Generation Sequencing: A Review of Its Transformative Impact on Cancer Diagnosis, Treatment, and Resistance Management
              </a>
            </td>
          <td>
            Alexandru Isaic, Nadica Motofelea, Teodora Hoinoiu, A. Motofelea, Ioan Cristian Leancu, Emanuela Stan, Simona R. Gheorghe, A. Duțu, Andreea M. Crintea
          </td>
          <td>2025-09-23</td>
          <td>Diagnostics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Chemotherapy is a foundational element in cancer therapy; however, the development of resistance to antineoplastic agents presents a significant challenge to achieving enduring therapeutic triumphs. The acquisition of drug resistance by malignant cells is the result of a complex interplay between inherent and acquired mechanisms. This phenomenon leads to a reduction in pharmacological effectiveness and contributes to treatment failure. This review investigates the primary mechanisms contributing to chemoresistance in specific malignancies, namely lung, pancreatic, and thyroid cancers, along with non-Hodgkin lymphoma. The primary focus of this research is on genetic modifications, interactions within the tumor microenvironment, and cellular adaptations at the molecular level. The present study places particular emphasis on the contributions of drug efflux pumps, DNA repair pathways, epithelial-mesenchymal transition (EMT), microRNAs, and the suppression of apoptosis in mediating this resistance. In addition, we examine promising approaches to counteract drug resistance, encompassing multi-agent regimens, inhibitors targeting specific molecular pathways, and innovative therapeutic modalities. A comprehensive understanding of these fundamental mechanisms is imperative for the development of sophisticated therapeutic interventions that not only improve patient outcomes but also effectively address resistance. The resistance pathways and prospective translational approaches discussed are drawn from both preclinical and clinical investigations, providing a thorough perspective. Keywords: The following terms are relevant to the study: chemoresistance, tumor microenvironment, molecular pathways, targeted therapy, and resistance mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ec567f4501baf205432eef42dce8eb9f17d9a8c" target='_blank'>
              Mechanisms of Chemoresistance in Solid and Hematologic Malignancies: Challenges and Future Perspectives
              </a>
            </td>
          <td>
            Badiaa Batlamous, Boutaina Elgharbaoui, Imane Bensalim, Mohamed Khalis
          </td>
          <td>2025-09-24</td>
          <td>Middle Eastern Cancer and Oncology Journal</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Human genome resequencing typically involves mapping reads to a reference genome to call variants; however, this approach suffers from both technical and reference biases, leaving many duplicated and structurally polymorphic regions of the genome unmapped. Consequently, existing variant benchmarks, generated by the same methods, fail to assess these complex regions. To address this limitation, we present a telomere-to-telomere genome benchmark that achieves near-perfect accuracy (i.e. no detectable errors) across 99.4% of the complete, diploid HG002 genome. This benchmark adds 701.4 Mb of autosomal sequence and both sex chromosomes (216.8 Mb), totaling 15.3% of the genome that was absent from prior benchmarks. We also provide a diploid annotation of genes, transposable elements, segmental duplications, and satellite repeats, including 39,144 protein-coding genes across both haplotypes. To facilitate application of the benchmark, we developed tools for measuring the accuracy of sequencing reads, phased variant call sets, and genome assemblies against a diploid reference. Genome-wide analyses show that state-of-the-art de novo assembly methods resolve 2–7% more sequence and outperform variant calling accuracy by an order of magnitude, yielding just one error per 100 kb across 99.9% of the benchmark regions. Adoption of genome-based benchmarking is expected to accelerate the development of cost-effective methods for complete genome sequencing, expanding the reach of genomic medicine to the entire genome and enabling a new era of personalized genomics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc9eb566f5ca36d1f2d3bccbe8022cfc8c0b963a" target='_blank'>
              A complete diploid human genome benchmark for personalized genomics
              </a>
            </td>
          <td>
            Nancy F. Hansen, Nathan Dwarshuis, Hyun Joo Ji, A. Rhie, Hailey Loucks, Glennis A. Logsdon, Mitchell R. Vollger, Jessica M. Storer, Juhyun Kim, Eleni Adam, Nicolas Altemose, D. Antipov, Mobin Asri, Sofia Barreira, Stephanie C. Bohaczuk, Andrey V. Bzikadze, S. A. Carioscia, Andrew Carroll, Kuan-Hao Chao, Yanan Chu, Arun Das, Peter Ebert, A. English, M. Fleharty, Laura E. Fleming, G. Formenti, A. Guarracino, Gabrielle Hartley, Katharine M. Jenike, Jenna Kalleberg, Yu Kang, Robert King, Josipa Lipovac, Mira Mastoras, Matthew W. Mitchell, S. Negi, Nathan D. Olson, Keisuke K. Oshima, L. F. Paulin, Brandon D. Pickett, David Porubsky, Jane Ranchalis, D. Ranjan, M. Rautiainen, Harold Riethman, Robert D Schnabel, F. Sedlazeck, Kishwar Shafin, M. Šikić, Steven J. Solar, Alex Sweeten, W. Timp, Justin Wagner, DongAhn Yoo, Ying Zhou, Erik Garrison, E. Eichler, Michael C Schatz, Andrew B. Stergachis, Rachel J. O’Neill, K. Miga, Steven L Salzberg, S. Koren, J. Zook, A. Phillippy
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>81</td>
        </tr>

        <tr id="The molecular basis of therapy resistance in multiple myeloma (MM) remains poorly understood. Here, we performed single-cell RNA sequencing coupled with VDJ-targeted sequencing of highly purified primary MM cells from patient bone marrow. This approach uncovered cellular heterogeneity and phenotypic plasticity along the CD138 axis, accompanied by drastic epigenetic alterations. Notably, therapy-resistant subpopulations were identified within a minor fraction of CD138- MM cells, and were shown via CRISPR/Cas9 screening to be vulnerable to splicing pathway inhibition. Consistently, this fraction of CD138- MM cells showed increased differential splicing associated with overexpression of SR protein family splicing factors. Among these splicing factors, RNA-binding protein 39 (RBM39) was specifically overexpressed in therapy-resistant cells and involved in aberrant splicing. Both genetic and pharmacological RBM39 inhibition exhibited a significant selective lethal effect on therapy-resistant CD138- MM cells. Collectively, our findings identify distinct therapy-resistant MM subpopulations and highlight the splicing pathway as a promising therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1dfb101a52f0fa47f40e1c355892c0f32082d1b" target='_blank'>
              Identification of a CD138-negative therapy-resistant subpopulation in multiple myeloma with vulnerability to splicing factor inhibition.
              </a>
            </td>
          <td>
            Takahiro Kamiya, Masahiko Ajiro, M. Oshima, S. Koide, Yaeko Nakajima-Takagi, Kazumasa Aoyama, Akiho Tsuchiya, Satoshi Kaito, Naoki Itokawa, Ryoji Ito, Kiyoshi Yamaguchi, Yoichi Furukawa, Bahityar Rahmutulla, Atsushi Kaneda, Takayuki Shimizu, N. Doki, Taku Kikuchi, Nobuhiro Tsukada, Masayuki Yamashita, Shinichiro Okamoto, Akihide Yoshimi, Keisuke Kataoka, Atsushi Iwama
          </td>
          <td>2025-09-02</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="BACKGROUND
Colorectal cancer (CRC), a predominant cancer type globally, is one of the most common cancers worldwide. Immhibune-checkpoint initors (ICIs) have robust efficacy in the treatment of patients with metastatic DNA mismatch repair (dMMR) CRC, however, some of the patients among them demonstrate resistance to immunotherapy, the underlying molecular mechanisms remain elusive.


METHODS
Using single-cell sequencing data, we identified and annotated cell populations, systematically comparing cell proportions and gene expression changes across different groups. Gene set enrichment analysis (GSEA) was conducted to identify significantly enriched signaling pathways in T cell subpopulations. Furthermore, we assessed the impact of transcriptional dysregulation in TSTR and γδ_T cells on tumor immunotherapy responses and drug resistance. The potential role of T cell subset interactions in modulating drug resistance and sensitivity was investigated through comprehensive cell communication analysis. Lastly, clinical samples from 190 CRC patients were collected, with paired adjacent normal tissues. Immunohistochemistry (IHC) was performed using antibodies against FOS and KLRB1.


RESULTS
T cells were identified as the predominant cell population influencing immunotherapy outcomes. T cell subsets exhibited distinct functional characteristics, with notable differences in their distribution between resistant and sensitive tumor groups. Specifically, exhausted T cells (Tex), GZMK + T, TSTR, regulatory T cells (Treg), and γδ_T cells were associated with therapeutic resistance. For Tex and GZMK + T cells, resistance was correlated with activation of antigen processing and presentation pathways, whereas oxidative stress pathways were downregulated. In contrast, γδ_T cells in the sensitive group exhibited activation of protein folding pathways, which may contribute to anti-tumor immune responses. Transcriptional network dysregulation in TSTR and γδ_T cells was observed in the drug-resistant group. Cell-to-cell communication analysis showed stronger interactions among T cell subpopulations, with changes in key signaling pathways linked to treatment resistance. Additionally, downregulation of the CD69-KLRB1 signaling pathway was identified as a potential mechanism of drug resistance in CRC. Lastly, high expression of FOS was significantly associated with a worse prognosis, whereas high expression of KLRB1 predicted improved clinical outcomes, including prolonged overall survival and progression-free survival, and emerged as an independent prognostic factor for CRC patients.


CONCLUSION
This study highlights the pivotal role of T cell subsets in patients with metastatic dMMR CRC who resistance to anti-PD-1 therapy, revealing that transcriptional dysregulation and impaired cell communication networks are central mechanisms underlying drug resistance. Notably, KLRB1 has been identified as a promising biomarker for immunotherapy response in CRC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3475e66480ee040f7a2e535d7ffc47aa52446ec" target='_blank'>
              Single-cell transcriptome analysis reveals regulatory programs associated with tumor resistance during immunotherapy in colorectal cancer.
              </a>
            </td>
          <td>
            Yan Chen, Tao Liu, Guangtao Min, Cong Wang, Dali Xi, Lei Jiang
          </td>
          <td>2025-09-10</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/739921fbe29bda5d659d1ced9f84aedc8ae3f8d2" target='_blank'>
              Next-generation oncology: integrative therapeutic frontiers at the crossroads of precision genomics, immuno-engineering, and tumor microenvironment modulation.
              </a>
            </td>
          <td>
            Ahmad M Alamri, Abdullah A. Assiri, Bushra Khan, Najeeb Ullah Khan
          </td>
          <td>2025-09-20</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background: Multiple Myeloma (MM) is a malignant plasma cell dyscrasia that progresses through the consecutive asymptomatic, often undiagnosed, precancerous stages of Monoclonal Gammopathy of Undetermined Significance (MGUS) and Asymptomatic Multiple Myeloma (SMM). MM is characterized by low survival rates, severe complications and drug resistance; therefore, understanding the molecular mechanisms of progression is crucial. This study aims to detect genetic mutations, both germline and somatic, that contribute to disease progression and drive tumorigenesis at the final stage of MM, using samples from patients presenting MGUS or SMM, and newly diagnosed MM patients. Methods: Mutations were identified through a fully computational pipeline, implemented in a Linux and RStudio environment, applied to each patient sequence, obtained through single-cell RNA-sequencing (scRNA-seq), separately. Structural and functional mutation types were identified by stage, along with the affected genes. The analysis included quality control, removal of the Unique Molecular Identifiers (UMIs), trimming, genome mapping and result visualization. Results: The findings revealed frequent germline and somatic mutations, with distinct structural and functional patterns across disease stages. Mutations in key genes were identified, pointing to molecules that may play a central role in carcinogenesis and disease progression. Notable examples include the HLA-A, HLA-B and HLA-C genes, as well as the KIF, EP400 and KDM gene families, with the first four already confirmed. Comparative analysis between the stages highlighted molecular transition events from one stage to another. Emphasis was given to novel genes discovered in newly diagnosed MM patients, that might contribute to the tumorigenesis that takes place. Conclusions: This study contributes to the understanding of the genetic basis of plasma cell dyscrasias and the transition events between the stages, offering insights that could aid in early detection and diagnosis, guide the development of personalized therapeutic strategies, and improve the understanding of mechanisms responsible for resistance to existing therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db8c2ca600e5f8d213e747b7e35b473639690b95" target='_blank'>
              Genomic Insights into Tumorigenesis in Newly Diagnosed Multiple Myeloma
              </a>
            </td>
          <td>
            Marina Kyriakou, C. Papaloukas
          </td>
          <td>2025-08-23</td>
          <td>Diagnostics</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Through most of medical history, treatments for metastatic cancers were ineffective, and rapid patient death was inevitable. Over the past five decades, a worldwide drug development effort has introduced a remarkable range of new cancer treatment strategies and agents so that virtually all metastatic cancers have one or more effective therapeutic options to prolong life. Yet most metastatic cancers remain fatal, and increasingly, the proximate cause of death is evolution. Local or systemic therapies applied to large, heterogeneous cancer populations elicit complex short- and long-term adaptive responses. Cells already possessing the molecular machinery of resistance obtain a stepwise fitness benefit relative to treatment-sensitive cells, allowing increased proliferation. Cells, otherwise sensitive to the treatment, may survive when in epigenetic states resistant to the treatment-induced death pathway or microenvironmental conditions that reduce drug delivery/efficacy, followed by a transition to "hard-wired" resistance allowing proliferation. These dynamics, enabled by the vast information content of the human genome, can produce diverse adaptive strategies in response to virtually all available treatments. Thus, oncology is rapidly approaching an era in which patient death is caused not by the absence of effective therapies but rather by eco-evolutionary dynamics that defeat initially successful treatments. Emerging evidence suggests that explicit integration of evolutionary principles to control or eliminate resistant populations can improve outcomes. In this issue of Cancer Research, Hockings and colleagues present an important evolutionary strategy to delay or prevent the evolution of resistance in ovarian cancer, with broad potential application. See related article by Hockings et al., p. 3503.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa08f9c64d9497ce73be1df571e19d588bc63780" target='_blank'>
              The Approaching Darwinian Era in Oncology.
              </a>
            </td>
          <td>
            Robert A. Gatenby
          </td>
          <td>2025-09-15</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Current cancer therapies often fail due to tumor heterogeneity and rapid resistance evolution. A new evolutionary framework, ‘selection for function,’ proposes that tumor progression is driven by group phenotypic composition (GPC) and its interaction with the microenvironment, not by individual cell traits. This perspective opens new therapeutic avenues: targeting the tumor’s functional networks rather than individual cells. Real-time tracking of GPC changes could inform adaptive treatments, delaying progression and resistance. By integrating evolutionary and ecological principles with conventional therapies, this strategy aims to transform cancer from a fatal to a manageable chronic disease. Crucially, it does not necessarily require new drugs but offers a way to repurpose existing therapies to impair a tumor’s evolutionary potential. By steering tumor evolution toward less aggressive states, this approach could improve prognosis and long-term patient survival compared to current methods. We argue that leveraging GPC dynamics represents a critical, yet underexplored, opportunity in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b53fd3b887f1a92efe08b517e2a13e55225513a1" target='_blank'>
              Leveraging selection for function in tumor evolution: System-level cancer therapies
              </a>
            </td>
          <td>
            F. Thomas, Jean-Pascal Capp, A. Dujon, A. Marusyk, K. Asselin, M. Campone, Pascal Pujol, Catherine Alix-Panabières, Benjamin Roche, B. Újvári, Robert Gatenby, A. Nedelcu
          </td>
          <td>2025-08-18</td>
          <td>Evolution, Medicine, and Public Health</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="The discovery of genomics has redefined oncology into precision rather than standardized treatment. But
therapeutic resistance of either kind, intrinsic or acquired, is a serious impediment to sustained success. The present review
outlines the domains on the crossroad between genomic intelligence and resistance evolution, where multi-omics profiling,
high-throughput sequencing, and AI-based analytics are explaining the challenging complexity of the tumor and predicting
resistance pathways. We review the contribution of clonal evolution, tumor plasticity and adaptive signaling in resistance to
therapy, and we promote moving towards active molecular monitoring rather than snapshot ting genome. To preempt and
target therapeutic escape we suggest a framework of adaptive precision oncology that comprehends real-time biomarkers,
liquid biopsy follow-up and resistance-predictive algorithms. New approaches like combination therapy, recalibration of
treatment with the help of AI, and longitudinal monitoring of genomes are mentioned. We also deal with moral and logistical
obstacles of adopting these strategies, and especially in low-resource environments. The new era of precision oncology We
now find ourselves in the next era of precision oncology that aims to personalization extend to real-time responsiveness, so
that cancer treatment can increasingly be more adaptable, more resilient and more durable, achieved through personalizing
the rapidly changing tumor biology with real-time dynamically changing genomic insights.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7508321bc4abb3899cb6fc37344230d9a86147e" target='_blank'>
              Genomic Intelligence and Resistance Evolution: Redefining Oncotherapeutic Strategies in Precision Oncology
              </a>
            </td>
          <td>
            Muchukota Sushma, Bharathi Bhogenahalli Venkatappa, Gowthami V., Nigel Viju Thomas, Souman Samanta
          </td>
          <td>2025-08-28</td>
          <td>International Journal of Innovative Science and Research Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is characterized by profound intratumoral heterogeneity and cellular plasticity, which drive therapeutic resistance. While molecular subtyping into classical and basal-like states has provided a framework, the biology of cells transitioning between these phenotypes remains poorly understood. This study aimed to characterize and validate a transitional, co-expressor cell state at the single-cell, spatial, and proteomic levels to elucidate mechanisms of subtype plasticity. We employed an integrative, multi-platform approach combining single-cell RNA sequencing (scRNA-seq), multiplex immunohistochemistry (mIHC), and laser microdissection with mass spectrometry-based proteomics (LMD-MS) to investigate tumor cells co-expressing the classical marker GATA6 and the basal-like marker KRT17. scRNA-seq analysis revealed that a significant population of tumor cells co-expresses GATA6 and KRT17, existing along a transcriptional continuum between the classical and basal-like poles. Pseudotime trajectory analysis identified a path from the GATA6+ classical state towards the KRT17+/GATA6+ transitional state, governed by the upregulation of pathways including MYC signaling and epithelial-mesenchymal transition (EMT). To validate these findings spatially, mIHC confirmed the presence of KRT17+/GATA6+ double-positive cells, frequently located at the interface between well-differentiated, GATA6-rich glandular structures and poorly differentiated, KRT17-rich infiltrative fronts. Furthermore, LMD-MS was used to provide proteomic confirmation. By isolating pure populations of GATA6+ and double-positive cells, we confirmed that transitional cells are enriched in proteins associated with metabolic reprogramming and cellular motility. Our findings demonstrate that PDAC plasticity is not a simple switch but a dynamic continuum involving a tangible transitional cell state. These co-expressor cells, with their unique metabolic and signaling dependencies, represent a key driver of tumor progression and a potential reservoir for therapeutic resistance. Targeting the specific vulnerabilities of this transitional phenotype offers a promising strategy to overcome plasticity, prevent relapse, and improve therapeutic outcomes in PDAC.



 Lyanne A. Delgado Coka, Jorge Villar Samaniego, Thomas Beggs, Anastasia Terbaci, Tristan Kwai, Karen Bai, Natalia Marchenko, Scott Powers, Luisa Escobar Hoyos, Kenneth Shroyer. Multimodal Validation of a Plastic Transitional State in Pancreatic Ductal Adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B122.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5a10f45ec5eb895ff99dbba41e30b63f2ad6188" target='_blank'>
              Abstract B122: Multimodal Validation of a Plastic Transitional State in Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Lyanne A. Delgado Coka, Jorge Villar Samaniego, Thomas Beggs, Anastasia Terbaci, Tristan Kwai, Karen Bai, N.Ye Marchenko, Scott Powers, Luisa Escobar Hoyos, Kenneth R. Shroyer
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Acute Myeloid Leukemia (AML) is a genetically and clinically heterogeneous malignancy marked by poor prognosis and limited therapeutic options, especially in older patients. While conventional treatments such as the “7 + 3” chemotherapy regimen and allogeneic stem cell transplantation remain standard care options, the advent of next-generation sequencing (NGS) has transformed our understanding of AML’s molecular complexity. Among the emerging hallmarks of AML, metabolic reprogramming has gained increasing attention for its role in supporting leukemic cell proliferation, survival, and therapy resistance. Distinct AML subtypes—shaped by specific genetic alterations, including FLT3, NPM1, and IDH mutations—exhibit unique metabolic phenotypes that reflect their underlying molecular landscapes. Notably, FLT3-ITD mutations are associated with enhanced reactive oxygen species (ROS) production and altered energy metabolism, contributing to disease aggressiveness and poor clinical outcomes. This review highlights the interplay between metabolic plasticity and genetic heterogeneity in AML, with a particular focus on FLT3-driven metabolic rewiring. We discuss recent insights into how these metabolic dependencies may be exploited therapeutically, offering a rationale for the development of metabolism-targeted strategies in the treatment of FLT3-mutated AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bdd39e68513bba88417b983cb0b3114bb567964" target='_blank'>
              Metabolic Signature of FLT3-Mutated AML: Clinical and Therapeutic Implications
              </a>
            </td>
          <td>
            C. Banella, Gianfranco Catalano, Maura Calvani, Eleonora Candi, N. Noguera, S. Travaglini
          </td>
          <td>2025-09-01</td>
          <td>Journal of Personalized Medicine</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Hematologic malignancies, including leukemia, lymphoma, and multiple myeloma, represent a diverse group of blood cancers driven by complex oncogenic pathways and genetic aberrations. Despite advances in treatment, resistance and relapse remain significant clinical challenges. This review presents an integrative approach to blood cancer therapy, highlighting the convergence of precision medicine, CRISPR-based gene editing, immunotherapy (including CAR-T cells and immune checkpoint inhibitors), and targeted modulation of amino acid metabolism. We investigate the molecular pathways driving tumour development and resistance, highlighting how multi-omic profiling facilitates personalised therapy approaches. Recent breakthroughs in reprogramming immune responses and editing driver mutations have revolutionized therapeutic paradigms. Additionally, the metabolic dependencies of cancer cells, particularly in amino acid biosynthesis and catabolism, offer novel vulnerabilities. This meticulous synthesis outlines a framework for translational techniques that integrate molecular targeting with immunological and metabolic regulation to improve treatment effectiveness and surmount therapeutic resistance in haematologic malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/602acd2f85cdfeb967bf671ca25b0e041a71b90f" target='_blank'>
              Deciphering and targeting oncogenic pathways through integrated approaches and amino acid metabolism in hematologic malignancies
              </a>
            </td>
          <td>
            Farhan Ikhtiar, Adil Jamal, Syed M Safeer Mehdi Bokhari
          </td>
          <td>2025-09-16</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d2cfb28da058ae0d8b29e85290850287448ab79" target='_blank'>
              Transition between cell states of sensitivity reveals molecular vulnerability of drug-tolerant cells.
              </a>
            </td>
          <td>
            L. Peyre, Marielle Péré, Mickael Meyer, Benjamin Bian, Marina Moureau-Barbato, Walid Djema, B. Mari, Georges Vassaux, Jérémie Roux
          </td>
          <td>2025-10-01</td>
          <td>Molecular systems biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a leading cause of global cancer mortality, with therapeutic resistance constituting a major barrier to sustained clinical benefit. Mitochondrial metabolic reprogramming has emerged as a central adaptive mechanism that enables CRC cells to withstand hypoxia and therapeutic pressure, while concurrently driving resistance to chemotherapy, targeted agents, and immunotherapy. In this Review, we explore how mitochondrial metabolism contributes to therapeutic resistance, with particular emphasis on metabolic plasticity, redox balance, and organelle quality control. We also assess enabling technologies such as spatial transcriptomics, proteomics, and patient-derived organoids, and discuss their translational relevance in stratifying metabolic vulnerabilities and informing individualized therapies. Targeting mitochondrial rewiring represents a compelling strategy to overcome resistance and drive progress toward personalized CRC therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ce513809806c925f57ad8aa4b27d5aa1115dc29" target='_blank'>
              Mitochondrial metabolic reprogramming in colorectal cancer: mechanisms of resistance and future clinical interventions
              </a>
            </td>
          <td>
            Xiuxiu Qiu, Ao Wang, Jiahui Wang, Zhanxia Zhang, Li Tao
          </td>
          <td>2025-08-09</td>
          <td>Cell Death Discovery</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="The cellular plasticity of neuroblastoma is defined by a mixture of two major cell states, adrenergic and mesenchymal, which may contribute to therapy resistance. However, how neuroblastoma cells switch cellular states during therapy remains largely unknown, and how to eradicate neuroblastoma regardless of its cell state is a clinical challenge. To better understand the cellular plasticity of neuroblastoma in chemoresistance, we define the transcriptomic and epigenetic map of adrenergic and mesenchymal types of neuroblastomas using human and murine models treated with indisulam, a selective RBM39 degrader. We show that cancer cells not only undergo a bidirectional switch between adrenergic and mesenchymal states, but also acquire additional cellular states, reminiscent of the developmental pliancy of neural crest cells. These cell state alterations are coupled with epigenetic reprogramming and dependency switching of cell state–specific transcription factors, epigenetic modifiers, and targetable kinases. Through targeting RNA splicing, indisulam induces an inflammatory tumor microenvironment and enhances the anticancer activity of natural killer cells. The combination of indisulam with anti-GD2 immunotherapy results in a durable, complete response in high-risk transgenic neuroblastoma models, providing an innovative, rational therapeutic approach to eradicate tumor cells regardless of their potential to switch cell states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce8bfb0658591ed4d276e6df844046df62c0e809" target='_blank'>
              RBM39 degrader invigorates innate immunity to eradicate neuroblastoma despite cancer cell plasticity
              </a>
            </td>
          <td>
            Shivendra Singh, Jie Fang, Hongjian Jin, Lee-Ann Van de Velde, Andrew Cortes, Jiani Chen, Sivaraman Natarajan, E. Poon, Qiong Wu, Christopher L. Morton, Mary A. Woolard, Waise Quarni, Jacob A Steele, Jon P Connelly, Liusheng He, Rebecca Thorne, Gregory Turner, Thomas Confer, Melissa Johnson, W. Caufield, Burgess Freeman, Timothy Lockey, A. Murphy, P. J. Murray, Takashi Owa, S. Pruett-Miller, Ruoning Wang, L. Chesler, Julie R. Park, A. Davidoff, John Easton, Xiang Chen, Paul G. Thomas, Jun Yang
          </td>
          <td>2025-09-17</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Phenotypic plasticity is a key mechanism of metastatic progression and cancer therapy resistance. This hallmark of human malignancies is enabled by highly conserved epigenetic mechanisms that control gene expression. Functional alterations in DNA methylation and histone post-translational modifications have been extensively described as drivers of metastatic dissemination and therapy resistance. Pharmacological inhibitors of epigenetic enzymes can revert these alterations, thereby stopping cancer progression and counteracting the emergence of resistant clones. Despite promising pre-clinical evidence, the clinical implementation of epigenetic therapies in solid cancers has led to disappointing results. Several factors can explain these challenges, including the lack of rational combinations. Notably, response to epigenetic treatments can be heterogeneous and short-lived. A liquid biopsy technology that allows the measure of specific epigenetic alterations enables patient selection and therapy monitoring, leading to the development of precision epigenetic therapies. In this review, we discuss the state of the art of this emerging treatment modality, and we identify key challenges that need to be overcome to reach the full potential of this new therapeutic concept.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/876ad36ab7595e359c53da7e87448ed401afa714" target='_blank'>
              Precision epigenetic therapies in oncology
              </a>
            </td>
          <td>
            Arundhati Chaudhary, Kayleigh J A Orchard, F. Salani, Theodora Partsou, Mark Eccleston, Guido Bocci, A. Italiano, Francesco Crea
          </td>
          <td>2025-09-19</td>
          <td>Cancer Metastasis Reviews</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Introduction Chemotherapy response variability in cancer patients necessitates novel strategies targeting chemoresistant populations. While combinatorial regimens show promise through synergistic pharmacological interactions, traditional pathway enrichment methods relying on static gene sets fail to capture drug-induced dynamic transcriptional perturbations. Methods To address this challenge, we developed the Pathway-Responsive Gene Sets (PRGS) framework to systematically identify chemoresistance-associated pathways and guide therapeutic intervention. Comparative evaluation of three computational strategies (GSEA-like method, Hypergeometric test-based method, Bates test-based method) revealed that the GSEA-like methodology exhibited superior performance, enabling precise identification of drug-induced pathway dysregulation. Results Key experimental findings demonstrated PRGS’s superiority over conventional Pathway Member Gene Sets (PMGS), exhibiting statistical independence (p < 0.0001) and enhanced detection of chemotherapy-driven pathway dysregulation. Application of PRGS to the GDSC dataset identified 8 resistance-associated pathways. Screening of agents targeting these pathways yielded candidates with predicted anti-resistance activity. An in vitro cellular experiment demonstrated that the bortezomib-bleomycin combination exhibited synergistic cytotoxicity (IDAcomboScore = 0.014) in T47D cells, highlighting the potential of PRGS-guided therapeutic strategies. Discussion This study establishes a PRGS-based methodological framework that integrates genomic perturbations with precision oncology, demonstrating its capacity to decode resistance mechanisms and guide therapeutic development through dynamic pathway analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/814bf51ab320b8d56ddbcfefe1cd9a9aa4f449e3" target='_blank'>
              Drug discovery for chemotherapeutic resistance based on pathway-responsive gene sets and its application in breast cancer
              </a>
            </td>
          <td>
            Dehua Feng, Jingwen Hao, Lingxu Li, Jian Chen, Xinying Liu, Ruijie Zhang, Huirui Han, Tianyi Li, Xuefeng Wang, Xia Li, Lei Yu, Bing Li, Jin Li, Limei Wang
          </td>
          <td>2025-09-16</td>
          <td>Frontiers in Bioinformatics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Chemotherapy is the most widely used and effective treatment method in today's cancer treatment. However, the rise of cancer drug resistance has limited chemotherapy effectiveness. The molecular mechanisms and related genes by which many cancer cells acquire resistance to chemotherapy drugs remain to be identified. Recent advancements of CRISPR-Cas9 genome-editing techniques have drawn rigorous exploration of precision oncology applications. CRISPR-Cas9 technology can knock out, inhibit or activate the expression of specific genes. By observing the phenotype of gene-edited cells, drug-resistant genes in carcinoma cells can be identified. This paper will decipher the molecular basis of CRISPR-Cas9 technology in targeted gene modification and give examples to illustrate its application in the screening of drug resistance genes in three types of human cancers. Additionally, the application limitations and emerging horizons of CRISPR-Cas9 gene modification technique in strategies for overcoming anticancer drug resistance are discussed. The continuous development and optimization of this technology are expected to provide strong support for breaking through the bottlenecks in tumor treatment and pushing precision cancer medicine to new heights.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0492def30740e765a8ca58409631a4cc538a87f" target='_blank'>
              CRISPR-Cas9 Gene-Editing Technology for the Screening of Drug Resistance Genes in Tumor Cells
              </a>
            </td>
          <td>
            Zisheng Zeng
          </td>
          <td>2025-09-09</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Therapeutic resistance in cancer is increasingly understood as a dynamic outcome of the interplay among tumour metabolism, epigenetic regulation, and immune modulation, rather than a consequence of genetic mutations alone. At this intersection, lactylation-a recently discovered lactate-derived post-translational modification (PTM)-acts as a molecular bridge linking metabolic rewiring to transcriptional and post-transcriptional control. Meanwhile, microRNAs (miRNAs) have emerged as essential regulators of metabolic adaptation and epigenetic plasticity. However, the interaction between lactylation and miRNA networks remains largely underexplored. Herein, we present a comprehensive and structured assessment of this emerging interdisciplinary field. We begin with a focused overview of tumour metabolic reprogramming and the enzymology of lactylation, establishing the biochemical context for its regulatory functions. We then examine how miRNAs interpret and reinforce metabolic cues, particularly through three interrelated regulatory frameworks. Lactylation of histones and DNA repair proteins has been shown to activate oncogenic miRNA clusters; these same miRNAs, in turn, enhance glycolytic flux and fine-tune the activity of lactylation enzymes, forming bidirectional feedback loops. Under stress conditions such as hypoxia or chemotherapy, lactate accumulation selectively suppresses DNA-repair-targeting and Pro-apoptotic miRNAs, stabilising BRCA1/RAD51-mediated repair programs. Furthermore, lactylation-dependent miRNA signals disseminate via extracellular vesicles, contributing to T cell exhaustion and macrophage M2 polarisation, thus shaping an immunosuppressive tumour microenvironment. Based on these converging mechanisms, we highlight potential therapeutic strategies, including the co-targeting of LDHA and oncogenic miRNAs (e.g., nanoparticle-mediated delivery of anti-miR-21), as well as liquid biopsy-based monitoring of circulating H3K18la and miRNA signatures to predict emerging resistance. By integrating metabolic, epigenetic, and immunologic perspectives, we position lactylation-miRNA crosstalk as a central regulatory axis in cancer therapy resistance and offer a conceptual framework to inform the development of future mechanism-driven interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3528e4d33d5572d7ac2200199d0f17c55ff6aa48" target='_blank'>
              Lactylation-mediated miRNA regulation in cancer therapy resistance
              </a>
            </td>
          <td>
            Yiyi Shou, Ruiqi Liu, Hao Xiong, Xiaoyan Chen, Luanluan Huang, Ran Huang, Hailong Sheng, Haibo Zhang, Yanwei Lu, Haiwei Guo
          </td>
          <td>2025-08-19</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Although checkpoint immunotherapy has primarily focused on CD8⁺ T cells, emerging evidence highlights an important role for cytotoxic CD4⁺ T cells in mediating therapeutic responses. However, research on the functional properties of cytotoxic CD4⁺ T cells in the context of immunotherapy is still at an early stage and remains insufficiently defined. Utilizing single-cell RNA-sequencing datasets obtained from metastatic melanoma patients treated with checkpoint inhibitors targeting PD-1 and/or CTLA-4, we performed transcriptomic profiling of conventional CD4⁺ T cells, excluding proliferative and regulatory (FOXP3⁺) subsets, and compared responders and non-responders as distinct groups. Importantly, our analysis identified distinct clusters that discriminate between responders and non-responders, with cytotoxic CD4⁺ T cells occupying a central position within these clusters. In responder-specific clusters, cytotoxic CD4⁺ T cells exhibited features of early activation, whereas clusters specific to non-responders were characterized by an exhausted phenotype. Notably, non-responder-specific clusters were positioned proximally to Treg-like clusters, suggesting a potential transition from cytotoxic to regulatory CD4⁺ T cell states in non-responders. Our findings reinforce the emerging concept that cytotoxic CD4⁺ T cells play a central role in mediating immunotherapy responses. These results provide a foundation for the development of predictive biomarkers and novel therapeutic strategies aimed at modulating CD4⁺ T cell differentiation. Supplementary Information The online version contains supplementary material available at 10.1007/s00262-025-04145-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34d377360f1a0a58a9eeb849ea470a9019aa6e62" target='_blank'>
              Cytotoxic CD4⁺ T cells exhibit an immunosuppressive shift in checkpoint immunotherapy resistance in melanoma patients
              </a>
            </td>
          <td>
            H. R. Bae, Byeongchan Son, Kyoungho Hwang, Suntae Kim, Howard A. Young, Eun-Young Kwon
          </td>
          <td>2025-09-01</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1eb5b9806f99885b27e221a4649bd483b17f7a1" target='_blank'>
              Optimizing treatment for metastatic castration-resistant prostate cancer: Food and Drug Administration-approved therapies, emerging strategies, and biomarker-driven approaches.
              </a>
            </td>
          <td>
            Faisal Saeed, Jacob E. Berchuck, M. Bilen, Jatin S. Gandhi, B. Nazha, Jacqueline T. Brown, David M Schuster, Ashesh B. Jani, Jindan Yu, Lara Harik
          </td>
          <td>2025-08-15</td>
          <td>Cancer</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb5a38dd74bce74d21cd64c9a442cf5e375b46f4" target='_blank'>
              Targeting Glioblastoma Cell State Plasticity for Enhanced Therapeutic Efficacy.
              </a>
            </td>
          <td>
            S. Cirigliano, R. Singhania, James G Nicholson, Isha Monga, Yushan Wan, C. Haywood, Ashlesha Muley, Skylar A. Giacobetti, Howard A. Fine
          </td>
          <td>2025-09-15</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Tuberculosis, caused by Mycobacterium tuberculosis, is a severe and persistent global public health issue, particularly exacerbated by the emergence of multidrug-resistant and extensively drug-resistant strains. This study employed pan-genomic approaches to analyze different strains with various resistance profiles, examining the diversity of bacterial genetic evolution in relation to mutations in resistance-related genes. The findings indicate that resistance-related genes are mostly core genes (94%), with a preference for base mutations closely associated with nonsynonymous mutations at resistance sites. Interestingly, while the majority of drugs induce positive selection in target genes, the tlyA gene under the influence of amikacin (AMI) undergoes passive selection. Cluster analysis of target genes suggests consistency between SNP clusters and drug-resistant clusters, revealing a strong correlation between bacterial evolutionary branches and resistance profiles. Consequently, based on pan-genome evolutionary characteristics, we identified the drug-resistant mutation pattern (DRMP) that can serve as a molecular fingerprint and indicator for drug sensitivity, aiding in the assessment and guidance of drug selection for treating different strains and the formulation of individualized treatment plans. This research not only enhances our understanding of the mechanisms of drug resistance in M. tuberculosis but also offers new perspectives for the development of new drugs, which is crucial for global tuberculosis control.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bed772fde7201b9c382d71c1e667c94ce6730951" target='_blank'>
              Drug selection based on pan-genomics genetic features of Mycobacterium tuberculosis
              </a>
            </td>
          <td>
            Xiangcheng Sun, Panpan Xu, Yun Shi, Ning Wang, Yan Li
          </td>
          <td>2025-09-02</td>
          <td>Frontiers in Microbiology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a74a710dbabf1414500dcedb29ef09db85188c8e" target='_blank'>
              Gene editing in cancer therapy: overcoming drug resistance and enhancing precision medicine.
              </a>
            </td>
          <td>
            Hyeonjeong Park, Suyeun Yu, Taeyoung Koo
          </td>
          <td>2025-09-18</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="BACKGROUND
Multiple myeloma (MM) is a malignancy marked by uncontrolled plasma cell proliferation, immune evasion, and drug resistance. Despite advances in treatment, the disease remains incurable due to relapses and drug resistance. This study aims to investigate the molecular and cellular interactions within the myeloma microenvironment using single-cell RNA sequencing (scRNA-seq), Mendelian randomization (MR), and pathway analysis to uncover therapeutic targets.


METHODS
We used scRNA-seq to analyze tumor, immune, and stromal cell interactions in MM. Data were processed using Seurat for clustering, dimensionality reduction, and cell-type annotation. Gene Set Variation Analysis (GSVA) and cell-cell interaction analysis were performed to identify signaling pathways involved in disease progression. Two-sample MR was applied to explore causal relationships between genetic variants and gene expression. Finally, molecular docking simulations were used to identify potential small molecule modulators of key proteins involved in MM.


RESULTS
We identified significant immune-related signaling pathways, including PI3K-AKT-mTOR, WNT-β-catenin, and TGF-β, upregulated in immune cells within the MM microenvironment. Genes such as HLA-C, CTSS, and LRRFIP1 showed positive causal relationships with MM, while SHISA5 and ISG15 exhibited protective roles. Cell communication analysis revealed key ligand-receptor interactions between immune and tumor cells. Molecular docking identified promising small molecules like actein and aflatoxin B1 targeting ISG15 and TAGLN2.


CONCLUSIONS
This study reveals key genetic drivers and immune modulation mechanisms in MM. Targeting immune-related pathways, such as PI3K-AKT-mTOR and WNT-β-catenin, and small molecules targeting ISG15 and TAGLN2 could offer new therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1ddcfc1e5a8f4f32f9fd598c516ebb1c5e5f270" target='_blank'>
              Single-cell RNA sequencing reveals immune regulatory mechanisms and molecular therapeutic strategies in the microenvironment of multiple myeloma.
              </a>
            </td>
          <td>
            Qiang Su, Kehan Long, Mozili Adu, Meijun Jiang, Qiaochu Li, Xufeng Wan, Jian Cao, Yan Yue, Shuoyuan Li, Zhendong Ying, Qibin Liu, Chaoqun You, Zhuang Zhang, Duan Wang
          </td>
          <td>2025-09-03</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/318599f6810efcf5bf3014e92955379d25d16eda" target='_blank'>
              Synthetic lethality in cancer drug discovery: challenges and opportunities.
              </a>
            </td>
          <td>
            Emanuel Gonçalves, Colm J. Ryan, David J. Adams
          </td>
          <td>2025-09-11</td>
          <td>Nature reviews. Drug discovery</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Background Glioblastoma, the most common primary malignant brain tumor, has a median survival of less than two years. This is due in part to a subpopulation of cells called glioblastoma stem cells (GSCs), which drive tumor recurrence. Transposable elements (TEs) are expressed at higher levels in cancer stem cells, enhancing the oncogenic potential and plasticity of cells through changes in gene expression, fusion transcript generation, and genomic rearrangement. Results Leveraging a large previously published dataset, we investigated the expression of TEs in bulk RNA sequencing data from 42 GSCs to identify subpopulations defined by their TE expression profile. Using telescope, a locus-specific approach to quantifying TE expression, we identified 858 TE loci that were expressed and defined two groups of GSCs using a consensus clustering approach. These TE-driven clusters displayed significant differences in both transcription factor (TF) and gene expression, with one group significantly enriched for a mesenchymal signature based on Gene Set Enrichment Analysis. Next, we extracted the locations and sequences of the TE regulatory domains and elucidated TF binding motifs within the TE sequences. This showed that the SOX11 consensus motif was enriched in the 5’ untranslated region of differentially expressed long interspersed nuclear elements (LINE). SOX11, a known inducer of LINE expression, was significantly under-expressed in the mesenchymal GSC cluster, which correlated with the concurrent decreased expression of LINE transcripts. These loci also overlapped with the enhancer elements of genes that were significantly downregulated, suggesting a potential link between TF binding to TE regulatory regions and gene expression. Conclusions Although further mechanistic studies are required, the identified link between TE location, TE and TF expression, and corresponding gene expression suggests that TEs may play a regulatory role in GSC transcription regulation. The current findings highlight the need for further investigation into the role of TEs in defining the gene regulatory and expression landscapes of GSCs. Future studies in this area could have therapeutic implications, given that glioblastoma recurrence may be driven by these cells. Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00370-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ceb4559d6649970b8ef9337621d841fa0e46f7e4" target='_blank'>
              Transposable element dynamics in glioblastoma stem cells: insights from locus-specific quantification
              </a>
            </td>
          <td>
            Mattia Pizzagalli, Yusuke Suita, Owen P. Leary, Nikos Tapinos
          </td>
          <td>2025-09-02</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer drug resistance remains a major barrier to durable treatment success, often leading to relapse despite advances in precision oncology. While combination therapies are being increasingly investigated, such as chemotherapy with small molecule inhibitors, predicting drug response and identifying rational drug combinations based on resistance mechanisms remain major challenges. Therefore, a proteome-wide, single-gene overexpression screening platform is essential for guiding rational therapy selection. Here, we present BOGO (Bxb1-landing pad human ORFeome-integrated system for a proteome-wide Gene Overexpression), a robust, scalable, and reproducible screening platform that enables single-copy, site-specific integration and overexpression of ~19,000 human ORFs across cancer cell models. Using BOGO, we identified drug-specific response drivers for 16 chemotherapeutic agents and integrated clinical datasets to uncover proliferation and resistance-associated genes with prognostic potential. Drug response similarity networks revealed both shared and unique mechanisms, highlighting key pathways such as autophagy, apoptosis, and Wnt signaling, and notable resistance-associated genes including BCL2, POLD2, and TRADD. In particular, we proposed a synergistic combination of the BCL2 family inhibitor ABT-263 (Navitoclax®) and the DNA analog TAS-102 (Lonsurf®), which revealed that lysosomal modulation is a key mechanism driving DNA analog resistance. This combination therapy selectively enhanced cytotoxicity in colorectal and pancreatic cancer cells in vitro, and demonstrated therapeutic benefit in vivo in both cell line-derived xenograft (CDX) and patient-derived xenograft (PDX) models. Together, these findings establish BOGO as a powerful gene overexpression perturbation platform for systematically identifying chemoresistance and chemosensitization drivers, and for discovering rational combination therapies. Its scalability and reproducibility position BOGO as a broadly applicable tool for functional genomics and therapeutic discovery beyond cancer resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb42d8c647bdd3aef172a4ebd8dbd580fe7fe538" target='_blank'>
              BOGO: A Proteome-Wide Gene Overexpression Platform for Discovering Rational Cancer Combination Therapies
              </a>
            </td>
          <td>
            Kyeong Beom Jo, M. Alruwaili, Da-Eun Kim, Yongjun Koh, Hyeyeon Kim, Kwontae You, Ji-Sun Kim, Saba Sane, Yanqi Guo, Jacob P. Wright, H. Lim, Maricris N. Naranjo, Atina G. Coté, Frederick P. Roth, David E. Hill, Jung-Hyun Choi, Hunsang Lee, Kenneth A. Matreyek, Kyle K.-H. Farh, Jong-Eun Park, Hyunkyung Kim, Andrei V. Bakin, Dae-Kyum Kim
          </td>
          <td>2025-09-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The addition of CDK4/6 inhibitors to endocrine therapy has significantly improved outcomes in HR+/HER2-breast cancer. However, variable patient responses and acquired resistance remain a clinical challenge. We therefore defined the comprehensive molecular response to palbociclib, the most clinically used CDK4/6 inhibitor. Global analyses of gene expression, protein abundance, splicing, and chromatin accessibility revealed broad patterns and specific changes that result from CDK4/6-inhibition in breast cancer cells. We uncovered unexpected feedback between CDK4/6 and estrogen-response signaling, which has clear clinical implications. We also revealed a widespread alternative splicing program that partially overlapped with genes whose expression is regulated, and which is expected to impact protein function. These molecular changes nominated combination therapies that interfere with the activation of CDKs or ERα. Accordingly, co-targeting CDK7, which regulates CDK2, CDK4/6 and ERα, additively impacted cell fitness. Collectively, these data reveal a complex, multi-tiered response to CDK4/6 inhibition, with implications for therapeutic efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ceb1151f3964ee516b8b4bec2c0e19c51711b54" target='_blank'>
              Charting the Multi-level Molecular Response to Palbociclib in ER-Positive Breast Cancer
              </a>
            </td>
          <td>
            Archishma Kavalipati, Amy Aponte, Michael E Sullivan, Sarah L Whittington, José C Martínez, Grant A. Goda, Maria M. Aleman, Michael J Emanuele, Daniel Dominguez
          </td>
          <td>2025-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background: Melanoma, the deadliest human skin cancer, frequently harbors activating BRAF mutations, with V600E being the most prevalent. These alterations drive constitutive activation of the MAPK pathway, promoting uncontrolled cell proliferation, survival, and dissemination. The advent of BRAFi and MEKi has significantly improved outcomes in BRAF V600-mutant melanoma. However, therapeutic resistance remains a major clinical barrier. Methods: This review integrates recent findings from preclinical and clinical studies to delineate resistance mechanisms to BRAF-targeted therapy. It categorizes resistance into primary (intrinsic), adaptive, and acquired forms, and analyzes their molecular underpinnings, including genetic and epigenetic alterations, pathway reactivation, and microenvironmental interactions. Results: Primary resistance is linked to pre-existing genetic and epigenetic changes that activate alternative signaling pathways, such as PI3K-AKT. Adaptive and acquired resistance includes secondary BRAF mutations, pathway redundancy, phenotype switching, and immune and stromal interactions. High-throughput sequencing has revealed novel mutations, including NRAS, NF1, and PTEN alterations, that contribute to resistance. Discussion: Understanding the multifaceted nature of resistance is critical to improving outcomes in advanced melanoma. This review highlights emerging strategies to overcome resistance, including combinatorial therapies, metabolic targeting, and biomarker-driven approaches, aiming to inform future therapeutic development and precision oncology strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50fe52d09f6a6cd013c45f83f652a44de19330f7" target='_blank'>
              Molecular Basis of BRAF Inhibitor Resistance in Melanoma: A Systematic Review
              </a>
            </td>
          <td>
            Ilaria Cosci, V. Salizzato, P. Del Fiore, J. Pigozzo, Valentina Guarneri, Simone Mocellin, A. Ferlin, Sara Mathlouthi, L. Piccin, Mariangela Garofalo
          </td>
          <td>2025-08-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Abstract In Mycobacterium tuberculosis , a prevalent and deadly pathogen, resistance to antibiotics evolves primarily through non-synonymous mutations in proteins. Sequence-based analyses are currently used to understand the genetic basis of antibiotic resistance, either via genotype-phenotype association, or via signals of convergent evolution. These methods focus on primary sequence and usually neglect other biological signals such as protein structural information. We hypothesize that integrating the structural context of mutations improves the prediction of effects on function and phenotype. We curate high confidence structural annotations for the M. tuberculosis proteome from 1,371 crystallography and 2,316 AlphaFold predictions, and combine the structures with mutations from over 31,000 clinical M. tuberculosis isolates. We demonstrate that mutations in proteins known to cause resistance are clustered in 3D space, even in proteins where inactivating mutations at any position are thought to cause resistance. We develop a statistic to search the M. tuberculosis proteome for signal of clustered non-synonymous mutations, finding over 450 proteins that display this signal, many of which have a known relationship with antibiotic resistance. Innovatively, we show that a supervised classifier trained on structure features alone has an F1 score of 94.6% at classifying mutations as resistance-conferring. This work demonstrates that protein structure provides useful information for categorizing which variants may cause antibiotic resistance, even when the majority of structures are AI-predicted.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/736eca4109e1243533868a8b023c33d1b51f37d5" target='_blank'>
              The structural context of mutations in proteins predicts their effect on antibiotic resistance
              </a>
            </td>
          <td>
            , Mahbuba Tasmin, R. Vargas, Maha R. Farhat
          </td>
          <td>2025-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 Acquired treatment resistance in homologous recombination (HR) deficient high-grade serous ovarian cancer (HGSOC) commonly involves restoration of HR, often through secondary, somatic reversion mutations in HR genes. Our recent analysis of HR deficient, end-stage HGSOC found that most resistance mechanisms, including reversion mutations, are subclonal. The aim of this study is to investigate how the subclonal, reversion-negative cells persist during treatment. We are investigating single cell sequencing approaches that enable genotyping of individual cells, such that the phenotype of cells without reversions can be studied. In a pilot experiment we have used two HGSOC cell lines with known reversion mutations. To genotype individual cells and thus overcome the limitations of standard single-cell RNA sequencing (scRNA-seq), we combined short-read whole-transcriptome scRNA-seq with long-read, targeted single-cell cDNA sequencing on an Oxford Nanopore MinION using a 44-gene hybridization capture panel. A custom bioinformatics pipeline, incorporating Flexiplex for multiallelic codon resolution, annotated cell barcodes with their HR driver, reversion and TP53 mutations. Genotypes were then transferred to the short-read dataset following outlier and doublet filtering, and transcriptomic differences between cells with and without reversions were identified through differential gene expression analysis. We have repeated this protocol on a patient ascites sample. Prior short-read DNA sequencing from the cell lines and matched tumor samples defined the driver BRCA1 mutation, reversion events, pathogenic TP53 mutations, and whether there was loss of heterozygosity (LOH) at the BRCA1 loci. We successfully genotyped BRCA1 driver and reversion alleles in both cell lines in the long-read data. Rare cells heterozygous wildtype for the BRCA1 driver mutation were detected, suggesting either back-reversion events or subclones that escaped LOH - cells that were undetectable by bulk short-read methods. Cell genotypes were mapped to the short-read whole transcriptome data, and transcriptional programs that may support cell survival were identified in the non-reverted cells. Our approach demonstrates the utility of unbiased long-read sequencing for detecting BRCA1/2 reversions at single-cell resolution, including potential back-reversion subclones that are challenging to identify through bulk sequencing approaches. The differential gene expression results have identified potential resistance mechanisms. Ongoing work includes incorporation of fusion detection, structural-variant calling, and splice isoform analysis in our bioinformatics pipeline to further elucidate resistance pathways. We anticipate this work will result in an increased understanding of the resistance mechanisms that arise in HGSOC patients with HR deficiency, including identification of subclonal resistance mechanisms that cannot be detected through bulk sequencing approaches.



 Lauren Tjoeka, Stuart Bencraig, David Yoannidis, Madelynne Willis, Joy Hendley, AOCS Study Group, Timothy Semple, Alicia Oshlack, Elizabeth L. Christie. Characterizing resistance at single cell resolution in HR deficient HGSOC [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr PR019.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7dc7b444080721cb95cb0b3c57900c2232de880b" target='_blank'>
              Abstract PR019: Characterizing resistance at single cell resolution in HR deficient HGSOC
              </a>
            </td>
          <td>
            Lauren Tjoeka, Stuart Bencraig, David Yoannidis, Madelynne O. Willis, J. Hendley, Aocs Study Group, T. Semple, Alicia Oshlack, Elizabeth L. Christie
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Intra-tumoral heterogeneity (ITH) is common in malignant tumors and arises from dynamic variations across genetic, epigenetic, transcriptomic, proteomic, metabolic, and microenvironmental factors. This complexity drives tumor evolution and treatment resistance, undermining the accuracy of clinical diagnosis, prognosis, and treatment planning. Despite recent advances in multi-omics technologies that enable comprehensive mapping of ITH across molecular layers, major challenges remain in clinical translation. This review outlines the principles and clinical applications of eight major omics modalities in the context of ITH: genomics, single-cell genomics, transcriptomics, epigenomics, proteomics, radiomics, microbiome, and metabolomics. We highlight the unique contributions of each omics platform to tumor profiling and emphasize how their integration enhances biological interpretation, patient stratification, and biomarker discovery. Furthermore, we will focus more extensively on the limitations of applying these approaches to ITH analysis. Instead of providing an exhaustive catalog, this review highlights major controversies, technical hurdles, and conceptual gaps that impede the clinical translation of multi-omics-based ITH analysis, with the aim of addressing ITH-related clinical challenges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3a8e32658c6f97ddac24dad4088bab77b9c897d" target='_blank'>
              Integration of multi-omics approaches in exploring intra-tumoral heterogeneity
              </a>
            </td>
          <td>
            Mengmeng Dong, Liping Wang, Ning Hu, Yueli Rao, Zhen Wang, Yu Zhang
          </td>
          <td>2025-08-29</td>
          <td>Cancer Cell International</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c95f957c61adab36c7dec0eefa4bae427dbcc6b3" target='_blank'>
              CRISPR-Edited Cell Lines: A New Era in Functional Oncology Research.
              </a>
            </td>
          <td>
            Amita Joshi Rana, M.D.S.A.D.I.Q.U.E. Hussain, Ali Hanbashi, Faroq Kamli, Gyas Khan, Marwa Qadri, Saeed A Al-Qahtani, Mohammad Gayoor Khan, Sushil S. Burle, Vikas Jakhmola, Gaurav Gupta
          </td>
          <td>2025-08-13</td>
          <td>Current pharmaceutical design</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="ABSTRACT Cancer metabolic reprogramming is a fundamental hallmark that enables tumor cells to sustain their malignant behaviors. Beyond its role in supporting growth, invasion, and migration, metabolic rewiring actively contributes to anticancer drug resistance. Cancer cells not only reshape their own metabolism but also engage in aberrant metabolic crosstalk with nonmalignant components within the tumor microenvironment (TME). These metabolic alterations create multiple barriers to the efficacy of drug therapies, including chemotherapy, targeted therapy, and immunotherapy. Despite growing evidence, an integrated understanding of how metabolic reprogramming contributes to the development of drug resistance and how it may be therapeutically targeted to overcome the resistance remains incomplete. This review summarizes recent progresses in tumor‐intrinsic and TME‐associated metabolic alterations that contribute to drug resistance by sustaining metabolic needs and modulating nonmetabolic processes and explores the upstream regulatory mechanisms driving these changes, focusing particularly on glucose, lipid, and amino acid metabolism. We also discuss the current advances in the integration of small molecule inhibitors targeting cancer metabolism to address drug resistance. By consolidating mechanistic insights and therapeutic opportunities, this review highlights metabolic reprogramming as a promising intervention point to overcome anticancer drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c779e78bcb053e66126c8246e6269dc9bb736c67" target='_blank'>
              Metabolic Reprogramming: A Crucial Contributor to Anticancer Drug Resistance
              </a>
            </td>
          <td>
            Yunhan Zhu, Weijie Yan, Lingfeng Tong, Jie Yang, Shengfang Ge, Jiayan Fan, Renbing Jia, X. Wen
          </td>
          <td>2025-09-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Cancer remains a leading cause of mortality worldwide, characterized by cells that exhibit 'monster traits' such as uncontrolled proliferation, invasiveness, resistance to apoptosis, and immune evasion. Traditional therapies often target single hallmarks of cancer, leading to limited success and eventual resistance. This paper proposes a novel, integrative therapeutic framework inspired by evolutionary adaptations in species with remarkable cancer resistance, such as elephants (Loxodonta Africana). Elephants possess enhanced TP53-mediated cancer surveillance and DNA repair mechanisms, which contribute to their extremely low cancer incidence. We hypothesize that by synthetically recapitulating these 'anti-monster' factors—such as augmented p53 activity, enhanced DNA repair, and controlled inflammation—in human cells, we can reprogram malignant phenotypes towards benignity. Our proposed strategy combines synthetic biology, gene therapy (including CRISPR/Cas systems), and multi-targeted pharmacological approaches to simultaneously address multiple cancer hallmarks. By leveraging advanced *in vitro* organoid and *in vivo* models, this research aims to pioneer a new paradigm in oncology that moves beyond cytotoxic elimination towards cellular reprogramming and durable cancer prevention, potentially yielding a significant reduction in lifetime cancer risk.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c541ffbed17a7ff84c98c50ab50d257ba1c2dd4" target='_blank'>
              Engineering Cancer Resistance: A Novel Framework Inspired by Evolutionary Adaptations
              </a>
            </td>
          <td>
            Hafiz Muhammadismail Azher Syed
          </td>
          <td>2025-09-29</td>
          <td>Wah Academia Journal of Health and Nutrition</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Gene expression heterogeneity generates subpopulations of tumor cells that can evade therapeutic pressure. This heterogeneity has been observed in both primary Estrogen Receptor alpha positive (ERα+) breast tumors and cell lines. Therefore, understanding the mechanisms regulating expression heterogeneity is critical towards developing effective therapies. A key contributor to gene expression variability is the stochastic nature of transcription. Transcription occurs in a probabilistic, burst-like manner, in which gene activation occurs intermittently, producing RNA in pulses and interspersed with transcriptional off-periods. The estrogen-responsive gene TFF1 is expressed in the majority of ERα+ breast tumors and exemplifies such heterogeneity, with transcriptional inactivity ranging from minutes to several days. Here, we identify the molecular mechanism underlying the wide range in TFF1 expression by analyzing cells sorted based on their TFF1 activity levels. We observed that TFF1 inactive (TFF1low) cells exhibit a repressive chromatin state marked by H3K27me3 at the TFF1 promoter and enhancer. Despite global similarity in ERα binding, occupancy at the TFF1 regulatory elements was selectively reduced in TFF1low cells, resulting in fewer active alleles and diminished transcriptional bursting frequency. Conversely, TFF1high cells exhibited more active TFF1 alleles and hyperbursting. These cells also retained sensitivity to endocrine therapy, while TFF1low cells displayed reduced drug responsiveness. Genome-wide, differentially enriched H3K27me3 regions correlated with variable expression of estrogen-responsive genes, highlighting a broader regulatory mechanism that links chromatin state to expression variability. Together, our findings establish how repressive chromatin dynamics contribute to gene expression heterogeneity and endocrine resistance in ERα+ breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffa89e16be6e3a720a965e4891206ee026e2fe10" target='_blank'>
              H3K27me3 chromatin heterogeneity reveals variable cell responses to estrogen and endocrine treatment
              </a>
            </td>
          <td>
            Chapus Fleur, Christopher R. Day, Laura G Kammel, Pelin Yasar, Brian D. Bennett, Erica Scappini, Charles J Tucker, Maria Sifre, Celyn Bregio, Joseph Rodriguez
          </td>
          <td>2025-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Intratumoral heterogeneity in pancreatic ductal adenocarcinoma (PDAC) is characterized by the coexistence of classical epithelial, basal, and mesenchymal cancer cell states. Targeting cancer cell plasticity—the ability of tumor cells to adapt to extrinsic pressures via non-genetic mechanisms—has emerged as a promising therapeutic strategy. In parallel, the immunosuppressive tumor microenvironment remains a major barrier to effective treatment in PDAC. Yet, whether specific cell states possess distinct functional properties that drive key tumor behaviors—such as phenotypic plasticity or immune evasion—remains poorly understood. Using genetically engineered mouse models driven by oncogenic Kras G12D and loss of Trp53 (“KPC” model), we profiled primary PDAC tumors using single-cell RNA sequencing and single cell ATAC sequencing. We find that the KPC model harbors a basal state, which harbors transcriptional signature that aligns with the "basal" signature in human PDAC and is linked to poor prognosis and chemoresistance. The basal state co-expresses both epithelial and mesenchymal programs and has a distinct epigenetic landscape with accessible chromatin across gene sets that mark cancer cell states with divergent phenotypes, suggesting that this cell state is primed for cell state switching. To functionally interrogate the basal state in PDAC progression, we developed a novel lineage-tracing and ablation system. Lineage-tracing experiments revealed that the basal state has a robust capacity for differentiation into epithelial and mesenchymal states, indicating that the basal state is functionally plastic and acts as a source of fixed malignant states. Strikingly, targeted elimination of the basal state over short time periods resulted in tumor collapse, underscoring a critical role in tumor maintenance. Further, we find that the basal state is a key driver of therapeutic resistance: rather than being eliminated by treatment, it persists through phenotypic adaptation in response to both KRAS-targeted therapies and cytotoxic chemotherapy. Co-elimination of the basal state alongside these therapies leads to deeper and more durable responses, underscoring its functional role in tumor persistence under therapy. In parallel, we observe that the basal state plays a central role in enforcing immunosuppression in PDAC tumors. Basal state ablation rapidly reshapes the tumor immune microenvironment, and we observe loss of cancer-associated fibroblasts, polarization of PDAC-resident macrophages toward a pro-inflammatory phenotype, and increased infiltration of T and NK cells. Cytokine profiling of basal-ablated tumors revealed a dramatic reduction in multiple immune-modulatory and immunosuppressive cytokines. Taken together, our data indicate that PDAC harbors a striking dependency on the basal state for both therapeutic resistance and immune evasion, and encourages therapeutic approaches aimed at eradicating this critical subset of cancer cells.



 Anupriya Singhal, Kate Ryan, Samuel Rose, Hannah C. Styers, Nikhita Pasnuri, Jonathan Rub, Jung Yun Kim, Ashlyn Moore, Jill Hallin, Zeynep C. Tarcan, Olca Basturk, Rona Yaeger, James G. Christensen, Yan Yan, Elisa de Stanchina, Dana Pe'er, Tuomas Tammela. Functional Dissection of the Highly Plastic Basal Cell State in Pancreatic Cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B108.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6879ea5062e58b2315581060e5992acf02efdbb" target='_blank'>
              Abstract B108: Functional Dissection of the Highly Plastic Basal Cell State in Pancreatic Cancer
              </a>
            </td>
          <td>
            Anupriya Singhal, Kate Ryan, Samuel Rose, Hannah C Styers, Nikhita Pasnuri, Jonathan Rub, Jung Yun Kim, Ashlyn Moore, Jill Hallin, Z. Tarcan, O. Basturk, R. Yaeger, James G Christensen, Yan Yan, E. de Stanchina, D. Pe’er, Tuomas Tammela
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Microsatellite-stable metastatic colorectal cancer (MSS mCRC) is currently treated with chemotherapy and targeted agents based on RAS and BRAF mutational status. Although these therapies offer initial benefit, most patients rapidly develop resistance, with fewer than 20% remaining progression-free at two years. This review aims to synthesize emerging evidence on the metabolic mechanisms driving treatment resistance in MSS mCRC, with a particular focus on the immune-metabolic signature (IMMETCOLS) classification. We conducted a comprehensive review of preclinical models, transcriptomic datasets, and clinical trial results addressing metabolic adaptations to chemotherapy and targeted therapies in MSS mCRC. The IMMETCOLS framework defines three metabolic subtypes—IMC1, IMC2, and IMC3—each associated with distinct resistance mechanisms. IMC1 exhibits glycolysis and transforming growth factor-β (TGF-β)-dependent signaling enriched in inflammatory fibroblasts, conferring resistance to chemotherapy. IMC2 relies on oxidative phosphorylation and glutamine metabolism, supporting antioxidant defenses and resistance to both cytotoxic agents and anti-EGFR therapies. IMC3 demonstrates lactate-fueled respiration and pentose phosphate pathway activation, contributing to redox balance, DNA repair, and resistance to targeted therapies such as anti-BRAF or KRAS inhibitors. All subtypes display metabolic plasticity under therapeutic pressure. Emerging clinical data support tailoring targeted therapy combinations based on IMMETCOLS subtype, particularly in BRAF- and HER2-positive populations. Understanding subtype-specific metabolic rewiring in MSS mCRC offers novel opportunities to overcome drug resistance. Targeting the metabolic vulnerabilities defined by the IMMETCOLS signature may improve response durability and inform precision treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/662ed873ba87c81960ffe96e54fa075a4697d04f" target='_blank'>
              Metabolic plasticity drives specific mechanisms of chemotherapy and targeted therapy resistance in metastatic colorectal cancer
              </a>
            </td>
          <td>
            Mariam Rojas, Malena Manzi, S. Madurga, Fernando Enrique García Velásquez, Maira Alejandra Romero, Silvia Marín, Marta Cascante, Joan Maurel
          </td>
          <td>2025-09-23</td>
          <td>Exploration of Targeted Anti-tumor Therapy</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Neoadjuvant chemotherapy (NAC) is a cornerstone in the treatment of early-stage high-risk breast cancers (BC), particularly in triple-negative, HER2-positive, and selected hormone receptor-positive subtypes. However, its effectiveness is frequently hindered by intrinsic or acquired resistance, resulting in a significant residual cancer burden (RCB) in more than half of patients. Despite extensive genomic profiling, reliable predictive biomarkers for treatment response remain limited, impeding the development of personalized therapeutic strategies. Emerging evidence highlights tumor metabolic reprogramming as a key non-genetic mechanism contributing to NAC resistance. In this review, we critically examine current advances in metabolic imaging and metabolomics as tools to predict NAC response in BC. We also discuss the role of the tumor microenvironment (TME), including hypoxia and acidosis, in shaping metabolic plasticity and fostering treatment resistance. Furthermore, we explore novel therapeutic strategies targeting metabolic pathways, ranging from enzyme inhibition to dietary interventions, and the use of advanced preclinical models. Together, these insights offer a metabolic framework for overcoming NAC resistance and advancing precision oncology in early-stage BC. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03500-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1de78332c9b34ec581f5a78aef2b8dfd8a219776" target='_blank'>
              Resistance to neoadjuvant chemotherapy in breast cancers: a metabolic perspective
              </a>
            </td>
          <td>
            Manon Desgres, Melis Poyraz, Buse Sari, Francois P. Duhoux, Cédric van Marcke, C. Corbet
          </td>
          <td>2025-08-11</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Lineage plasticity, a critical hallmark of cancer progression, enables tumor cells to evade inhibition of primary oncogenic pathways through histologic transformation. This adaptive process, driven by stemness-associated features and epigenetic reprogramming, poses significant challenges in treatment. Using non-small cell lung cancer and prostate cancer as models, we examine the utility of tissue and liquid biopsies in detecting histologic transformations and tailoring treatments to specific subtypes, which has profound clinical implications, potentially improving outcomes in patients with advanced, therapy-resistant disease. We also discuss emerging therapeutic strategies, including novel molecular targets, and address ongoing clinical challenges in managing treatment-emergent histologic transformation.


SIGNIFICANCE
The advent of highly effective molecularly targeted therapies results in increased recognition of treatment-emergent histologic transformation. This review not only summarizes current evidence on diagnosis and management of lineage plasticity but also explores therapeutic strategies under study, outlining a framework for clinical translation and successful drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49dc4debf7b809a5797b2dd12ac58569636b53ca" target='_blank'>
              Histologic Transformation in Cancer: The Path for Clinical Translation.
              </a>
            </td>
          <td>
            Ioannis A. Vathiotis, Avisek Banerjee, Utsav Sen, Felix C Saalfeld, Debdatta Halder, A. Charpidou, Konstantin Syrigos, H. Horinouchi, Ashutosh K Tewari, Mark A Dawson, M. Wermke, Triparna Sen
          </td>
          <td>2025-09-04</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Background Retroelements are repetitive sequences that comprise 42% of the human genome and are strictly regulated through various epigenetic mechanisms. Examining retroelement expression on a locus-specific level in relation to cancer can uncover distinct disease signatures. Results Using over 5000 RNA-sequencing samples, we assessed retroelement transcription across 23 tissue systems, 159 cell types, 1019 cancer cell lines, and cells isolated from various stages of embryogenesis using the specialized software tool, Telescope. In healthy individuals, 11,388 retroelements were found to be actively transcribed and dynamically regulated in a tissue- and cell type-dependent manner. Using the adult human body as a reference, we observed that 94% of cancer cell lines displayed elevated transcription of at least one cancer-specific retroelement, providing a three-fold larger reservoir of cancer biomarkers (1182) than our comparable analysis of annotated protein-coding genes (338). The precise retroelements that were transcribed following tumorigenesis were influenced by the originating location, with cancers of the blood, lungs, and soft tissue displaying the highest origin-specific activation. Moreover, nearly half of the cancer-specific retroelement loci, mostly from the HERV-H family, were found to be expressed during early embryonic development. Conclusions Our data demonstrate that elevated transcription of certain tissue-specific and embryonic retroelements can be considered as a hallmark of tumorigenesis. Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00376-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e98e92e95f88b68527f2a00c19cdab4d4246b04c" target='_blank'>
              Strict retroelement regulation is frequently lost following cancer transformation and generates a promising reservoir of cancer biomarkers
              </a>
            </td>
          <td>
            Eric Russ, Sergey Iordanskiy
          </td>
          <td>2025-10-03</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="

Cancer remains one of the most complex diseases globally, driven by genetic and epigenetic
alterations that disrupt normal cellular processes. This complexity is further amplified by
mechanisms such as uncontrolled cell proliferation, inhibition of apoptosis, angiogenesis, immune
evasion, and metabolic reprogramming, all of which contribute to tumor growth and metastasis.
While advancements in diagnostic technologies, including next-generation sequencing and liquid
biopsies, have facilitated early detection and the development of personalized therapeutic strategies,
traditional treatments such as chemotherapy, radiotherapy, and immunotherapy continue to
face significant limitations, including drug resistance, adverse side effects, and tumor heterogeneity.
Recent studies have emphasized the potential of bacteriophages in cancer treatment. As viruses
that specifically target bacteria, phages offer unique advantages due to their genetic modifiability,
roles in immunotherapy, and capabilities in targeted drug delivery. They can reshape the tumor microenvironment,
enhance immune responses, and deliver therapeutic agents directly to cancer
cells. Moreover, phages hold promise in addressing antibiotic resistance in cancer patients and improving
the efficacy of traditional therapies through combination strategies. However, challenges
remain regarding the optimization of phage stability, bioavailability, and tumor-specific targeting.
Research and clinical trials on the therapeutic application of phages are critical to unlocking their
full potential as innovative approaches to overcome the limitations of current cancer treatments.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f176f476b0e17d22a339146ab7610aed3ab80193" target='_blank'>
              Bacteriophages in Cancer Treatment
              </a>
            </td>
          <td>
            Mustafa Ustundag, Berrin Ustundag
          </td>
          <td>2025-08-22</td>
          <td>Current Cancer Therapy Reviews</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Chemotherapy resistance remains a formidable challenge to the treatment of high-grade serous ovarian cancer (HGSOC). The drug-tolerant cells may originate from a small population of inherently resistant cancer stem cells (CSCs) in primary tumors. In contrast, sufficient evidence suggests that drug tolerance can also be transiently acquired by nonstem cancer cells. Regardless of the route, key regulators of this plastic process are poorly understood. Here, we utilized multiomics, tumor microarrays, and epigenetic modulation to demonstrate that SOX9 is a key chemo-induced driver of chemoresistance in HGSOC. Epigenetic upregulation of SOX9 was sufficient to induce chemoresistance in multiple HGSOC lines. Moreover, this upregulation induced the formation of a stem-like subpopulation and significant chemoresistance in vivo. Mechanistically, SOX9 increased transcriptional divergence, reprogramming the transcriptional state of naive cells into a stem-like state. Supporting this, we identified a rare cluster of SOX9-expressing cells in primary tumors that were highly enriched for CSCs and chemoresistance-associated stress gene modules. Notably, single-cell analysis showed that chemo treatment results in rapid population-level induction of SOX9 that enriches for a stem-like transcriptional state. Altogether, these findings implicate SOX9 as a critical regulator of early steps of transcriptional reprogramming that lead to chemoresistance through a CSC-like state in HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/016a8c0db4a1babe91a08a91f054acbb57a8720c" target='_blank'>
              SOX9 drives a stem-like transcriptional state and platinum resistance in high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            A. Duval, Fidan Seker-Polat, Magdalena Rogozinska, Meric Kinali, A. Walts, Ozlem Neyisci, Yaqi Zhang, Zhonglin Li, Edward Tanner, Allison E. Grubbs, Sandra Orsulic, Daniela Matei, M. Adli
          </td>
          <td>2025-10-01</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Aberrant DNA methylation has been described in nearly all human cancers, yet its interplay with genomic alterations during tumor evolution is poorly understood. To explore this, we performed reduced representation bisulfite sequencing on 217 tumor and matched normal regions from 59 patients with non-small cell lung cancer from the TRACERx study to deconvolve tumor methylation. We developed two metrics for integrative evolutionary analysis with DNA and RNA sequencing data. Intratumoral methylation distance quantifies intratumor DNA methylation heterogeneity. MR/MN classifies genes based on the rate of hypermethylation at regulatory (MR) versus nonregulatory (MN) CpGs to identify driver genes exhibiting recurrent functional hypermethylation. We identified DNA methylation-linked dosage compensation of essential genes co-amplified with neighboring oncogenes. We propose two complementary mechanisms that converge for copy number alteration-affected chromatin to undergo the epigenetic equivalent of an allosteric activity transition. Hypermethylated driver genes under positive selection may open avenues for therapeutic stratification of patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/127ba85daf25e695241e6cc43a8f791c28684c9d" target='_blank'>
              DNA methylation cooperates with genomic alterations during non-small cell lung cancer evolution
              </a>
            </td>
          <td>
            F. Gimeno-Valiente, C. Castignani, E. Larose Cadieux, Nana E. Mensah, Xiaohong Liu, Kezhong Chen, O. Chervova, Takahiro Karasaki, C.E. Weeden, Corentin Richard, Siqi Lai, Carlos Martínez-Ruiz, E. Lim, A. Frankell, T. Watkins, Georgia Stavrou, Ieva Usaite, Wei-Ting Lu, Daniele Marinelli, S. Saghafinia, G. Wilson, P. Dhami, H. Vaikkinen, Jonathan Steif, S. Veeriah, R. Hynds, Martin Hirst, C. Hiley, A. Feber, Özgen Deniz, M. Jamal-Hanjani, N. Mcgranahan, Stephan Beck, J. Demeulemeester, M. Tanić, C. Swanton, P. Van Loo, N. Kanu
          </td>
          <td>2025-09-01</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Background Immune checkpoint inhibitors (ICIs) targeting programmed death-1 (PD-1) have demonstrated significant clinical benefit in colorectal cancer (CRC) with microsatellite instability-high (MSI-H) status, yet a substantial subset of patients remains resistant to therapy. Understanding the mechanisms of resistance and identifying new therapeutic targets are urgently needed. Methods We established a murine MC38-based MSI-H CRC model to investigate the variability in PD-1 treatment response and performed single-cell RNA sequencing (scRNA-seq) on tumors from patients with PD-1-sensitive and PD-1-resistant CRC. Immune cell subsets and signaling pathways were analyzed using CellChat and AUCell, and gene expression enrichment was performed. Functional assays, including RNA-seq, ChIP-seq, qRT-PCR, and Co-IP, were employed to characterize the role of MNDA and its downstream targets. Results scRNA-seq analysis revealed a significant enrichment of polymorphonuclear neutrophil myeloid-derived suppressor cells (PMN-MDSCs) in PD-1-resistant MSI-H CRC, accompanied by increased lactylation activity and high MNDA expression. Further investigations revealed that MNDA recruits the histone acetyltransferase EP300 to the CXCR2 promoter, mediating H3K18 lactylation and enhancing CXCR2 transcription. This promotes PMN-MDSC infiltration and the establishment of an immunosuppressive microenvironment, thereby impairing PD-1 efficacy. Inhibition of the MNDA/EP300-CXCR2 axis reversed H3K18la modification, reduced PMN-MDSC infiltration, remodeled the immune microenvironment, and restored sensitivity to PD-1 therapy. Conclusions Our study reveals a novel mechanism by which MNDA-driven H3K18 lactylation of the CXCR2 promoter facilitates immune evasion and PD-1 resistance in MSI-H CRC. Targeting the MNDA/EP300-CXCR2 regulatory axis represents a promising strategy for overcoming ICI resistance and enhancing the therapeutic benefit in CRC. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-07097-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f63561fa4907ea76d18e3dde39a94c0132ce001" target='_blank'>
              MNDA promotes immunosuppression in microsatellite instability-high colorectal cancer by facilitating PMN-MDSC infiltration via H3K18 lactylation
              </a>
            </td>
          <td>
            Xuan Zhang, Wen-Bin Wang, , Xiao-Qiong Chen, Qing Feng, Quan Yang, Ji-Biao Li, Wei Xiong, Tao Wu
          </td>
          <td>2025-10-03</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Antimicrobial resistance (AMR) is traditionally discussed in the context of horizontally acquired resistance genes and point mutations at target loci. However, this gene-centred model fails to account for a large number of clinically important modalities of resistance. There is now substantial evidence implicating bacteria in the ability to escape the effects of antibiotics in a variety of non-canonical ways, which are not considered in traditional diagnostic and surveillance pipelines. Among these factors, we can list those arising from global regulatory networks, phase variability, epigenetic tuning, small RNAs, genome structural variability, and phenotypic states like tolerance and persistence. This review will blend the current knowledge on these alternative pathways of resistance and underscore how they intersect with canonical genetic determinants. We will highlight cases where resistance emerges in the absence of known resistance genes, analyse the role of regulatory plasticity in efflux pump expression and membrane remodelling, and examine the contributions of bacterial stress responses and post-transcriptional control. Additionally, we will address methodological gaps in the detection of these mechanisms and their implications for clinical treatment failure, resistance surveillance, and drug development. By integrating insights from molecular microbiology, systems biology, and genomics, this review aims to offer a framework for understanding AMR as a multifaceted, context-dependent phenotype, not merely a genotype. We conclude by identifying knowledge gaps and suggesting priorities for research and diagnostic innovation in this evolving field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05cfbc4cd9dd576d2ed3bfaa746258988e167067" target='_blank'>
              Beyond Genes: Non-Canonical Mechanisms Driving Antimicrobial Resistance in Bacteria
              </a>
            </td>
          <td>
            Leonard Koolman, C. Emenike, Debasis Mitra, S. Chattaraj
          </td>
          <td>2025-10-01</td>
          <td>Bacteria</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Genetic mechanisms regulating gene expression encompass complex processes such as transcription, translation, epigenetic modifications, and interactions of regulatory elements. These mechanisms play a crucial role in shaping phenotypic diversity in humans. High-throughput technologies, such as expression microarrays and next-generation sequencing (NGS), have enabled precise analysis of transcripts for thousands of genes genome-wide. These methods have enabled researchers to measure gene expression levels in various tissues and cells and to gain deeper insights into previously inaccessible biological processes. Numerous studies show that gene expression varies significantly among individuals. However, there are also notable differences between populations from different continental groups, driven by genetic, epigenetic, environmental factors, and natural selection. Furthermore, disease states represent an important factor influencing gene activity, as they can significantly alter the transcriptomic profiles of individual cells. In this context, comparative population genetic studies help uncover the molecular mechanisms underlying complex phenotypic traits and identify population-specific features of transcriptomic profiles in both health and disease. However, despite significant progress in this field, many aspects remain underexplored. Specifically, the distribution of gene expression variability among populations, the degree of research coverage for specific ethnic groups, the spectrum of biological materials used, and the contribution of population affiliation to observed differences in gene expression during pathological conditions require further investigation. This review presents an overview of contemporary research focused on analyzing variability in expression profiles across different human populations. It summarizes findings from individual studies, outlines the advantages and limitations of the methods employed, highlights key research directions in population transcriptomics, and discusses potential practical applications of the data obtained.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e020705578f481ead7a3caa26fc484de5385fb09" target='_blank'>
              Population transcriptomics: a novel tool for studying genetic diversity in human populations under normal and pathological conditions
              </a>
            </td>
          <td>
            A. A. Babovskaya, E. A. Trifonova, V. A. Stepanov
          </td>
          <td>2025-09-01</td>
          <td>Vavilov Journal of Genetics and Breeding</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cutaneous melanoma (CM), with a continuously rising incidence worldwide, represents the most aggressive type of skin cancer, and it leads to the majority of skin cancer-related deaths. Approximately 50% of CM carry the activating BRAFV600 mutation and, although BRAF inhibitors have demonstrated clinical efficacy, most patients often develop early resistance to treatment. Aberrant expression of non-coding RNAs (ncRNAs), which represent less than 2% of the entire transcriptome, has been implicated in CM development and progression. By using BRAFV600-mutant CM in vitro and in vivo models, we have recently demonstrated that the loss of Spry1 expression impairs BRAFV600-mutant CM progression. Therefore, the extensive long and small ncRNA datasets generated in this study might represent a valuable resource for the characterization of their roles in BRAFV600-mutant CM initiation and progression upon Spry1 loss, thus providing a comprehensive resource to support future studies on BRAFV600-mutant CM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/710c46752f6a68d20467ff6b95252e5ee4402a37" target='_blank'>
              Non-coding RNA profiling in BRAFV600E-mutant cutaneous melanoma before and after Spry1 depletion
              </a>
            </td>
          <td>
            J. Lamberti, Domenico Memoli, Barbara Montico, Francesco Silvestro, R. Guerrieri, F. Colizzi, Alessandro Weisz, A. Salvati, E. Fratta, G. Nassa
          </td>
          <td>2025-09-02</td>
          <td>Scientific Data</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Background Psoriasis is a chronic inflammatory skin disease involving dysregulated immune responses and complex genetic factors. This study combines single-cell RNA sequencing (scRNA-seq), gene expression profiling, and genetic analysis to explore cellular and molecular contributors to psoriasis. Methods Single-cell RNA-seq data (n = 3 psoriasis, n = 2 control; GSE228421) were used for cell-type annotation and functional characterization. T cell subsets were analyzed for differentiation trajectories and cell-cell communication. Differentially expressed genes in mucosal-associated invariant T (MAIT) cells were evaluated by enrichment analysis. Candidate gene causality was tested via eQTL-based Mendelian randomization (MR) and supported by bulk RNA-seq validation. Results MAIT cells were enriched in psoriatic lesions and exhibited strong intercellular interactions. Functional analyses revealed activation of IL6-JAK-STAT3 signaling, TNF-NFκB pathway, and glycolysis in MAIT cells. MR identified RPS20 as a protective factor (OR = 0.5994, p = 0.011) and PFN1 as a potential risk gene (OR = 1.7229, p = 0.037), with PFN1 highly expressed in MAIT cells. Colocalization analysis showed no significant genetic overlap between PFN1 expression and psoriasis risk. Metabolic profiling revealed differential pathway involvement in PFN1+ and PFN1− MAIT cells. Conclusion Our integrative analysis highlights MAIT cells and PFN1 as likely contributors to psoriasis pathogenesis. These findings offer insights into immune and metabolic alterations, suggesting potential targets for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48055881b868991adfc41d78b968062c6c8aae6e" target='_blank'>
              MAIT Cells with High PFN1 Expression Mediate Immune Activation and Metabolic Reprogramming in Psoriasis
              </a>
            </td>
          <td>
            Man-Ning Wu, Yue-Min Zou, Xiang-Nan Zhou, Sangwon Hong, Lei Wang, Yan-Ping Bai
          </td>
          <td>2025-09-09</td>
          <td>Clinical, Cosmetic and Investigational Dermatology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Osteosarcoma is a genomically complex tumor characterized by widespread structural rearrangements. This complexity has limited development of therapeutic strategies informed by molecular mechanisms of oncogenesis. We hypothesized that epigenetic mechanisms could drive distinct subtypes of osteosarcoma. Through analysis of chromatin accessibility, we identified an "early osteoblast-derived" (EOD) cell state characterized by upregulation of transcription factors associated with early bone development, and a "late osteoblast-derived" state (LOD), characterized by upregulation of genes involved in late bone development. We then defined core regulatory circuitries governing the underlying gene expression programs in these two cell states. Multiomic single-cell analysis indicates that these cell states co-exist in a single tumor. Finally, using a panel of patient-derived xenograft models, we identified differential drug responses dependent on these cellular states. These findings create opportunities for developing new combination therapy strategies for osteosarcoma treatment and underscore the value of defining epigenetic subtypes in highly genomically complex cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/061de65b88f0b6c7fc51a022f24db3f11873c460" target='_blank'>
              Epigenetic and transcriptional programs define osteosarcoma subtypes and establish targetable vulnerabilities.
              </a>
            </td>
          <td>
            E. López-Fuentes, Andrew S Clugston, Alex G Lee, L. Sayles, Natalie Sorensen, María V Pons Ventura, Stanley G Leung, Truc Dinh, Marcus R. Breese, E. Sweet-Cordero
          </td>
          <td>2025-10-03</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Cancer research has traditionally focused on identifying driver genes, those with mutations that initiate tumorigenesis. The Cancer Driver Gene (CDG) paradigm, further supported by the observation of oncogene addiction in tumors, has successfully guided the development of targeted therapies. However, the limitations of this driver-centric view, highlighted by the broad emergence of frequent therapeutic resistance, the presence of driver mutations in healthy tissues or individuals, and the lack of identifiable drivers in many tumors, call for a shift in perspective and clinical practice. The latest network controllability perspective on cancer cells introduced the concept of Cancer Keeper Genes (CKGs) and a CKG-based paradigm for cancer therapeutics. The new concept encompasses the concept of non-oncogene addiction, emphasizing reliance on non-mutated pathways crucial for maintaining oncogenic cellular states. Here, we explore the transition towards a system-level understanding of cancer based on the CKG paradigm, emphasizing the essential role of genes required for tumor maintenance, irrespective of their initiating function or mutational capacity. We discuss clinical implications of this paradigm shift, highlighting the progress made so far and potential of targeting non-driver CKGs, genes involved in processes like DNA damage response, proteostasis, and metabolism, as a promising strategy to overcome therapeutic challenges and achieve more durable cancer control. Targeting these maintenance vulnerabilities represents a critical evolution in precision oncology, moving towards therapies designed to dismantle the networks sustaining malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e80123ffc1524bf35e125c1b2645e60b9bc7703" target='_blank'>
              From Cancer Drivers to Cancer Keepers: Paradigm Shift and Clinical Implications
              </a>
            </td>
          <td>
            Xizhe Zhang, Weixiong Zhang
          </td>
          <td>2025-08-22</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background. Antimicrobial resistance (AMR) poses a significant global health threat, necessitating a deeper understanding of bacterial adaptation mechanisms. Introduction. This study investigates the genotypic and phenotypic evolutionary trajectories of Escherichia coli under meropenem and gentamicin selection, and it benchmarks these findings against florfenicol-evolved strains. Methodology. Utilizing a downsized, three-layer acrylic modified “Microbial Evolution and Growth Arena (MEGA-plate) system”—scaled to 40 × 50 cm for sterile handling and uniform 37 °C incubation—we tracked adaptation over 9–13 days, enabling real-time visualization of movement across antibiotic gradients. Results. Meropenem exposure elicited pronounced genetic heterogeneity and morphological remodeling (filamentous and circular forms), characteristic of SOS-mediated division arrest and DNA-damage response. In contrast, gentamicin exposure produced a uniform resistance gene profile and minimal shape changes, suggesting reliance on conserved defenses without major morphological adaptation. Comprehensive genomic analysis revealed a core resistome of 22 chromosomal loci shared across all three antibiotics, highlighting potential cross-resistance and the central roles of baeR, gadX, and marA in coordinating adaptive responses. Gene ontology enrichment underscored the positive regulation of gene expression and intracellular signaling as key themes in resistance evolution. Discussion. Our findings illustrate the multifaceted strategies E. coli employs—combining metabolic flexibility with sophisticated regulatory networks—to withstand diverse antibiotic pressures. This study underscores the utility of the MEGA-plate system in dissecting spatiotemporal AMR dynamics in a controlled yet ecologically relevant context. Conclusions. The divergent responses to meropenem and gentamicin highlight the complexity of resistance development and reinforce the need for integrated, One Health strategies. Targeting shared regulatory hubs may open new avenues for antimicrobial intervention and help preserve the efficacy of existing drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2ac7589d729720679400cf686393d27d70cc65d" target='_blank'>
              Deciphering Common Genetic Pathways to Antibiotic Resistance in Escherichia coli Using a MEGA-Plate Evolution System
              </a>
            </td>
          <td>
            Nami Morales-Durán, A. León-Buitimea, Roberto Álvarez Martínez, J. R. Morones-Ramírez
          </td>
          <td>2025-08-01</td>
          <td>Antibiotics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Chromosomal rearrangements that generate novel fusion genes are a hallmark of acute myeloid leukemia (AML). Depletion experiments in cell line models have suggested that their continued expression is required for maintaining their leukemic phenotype and that fusion genes therefore represent ideal cancer-specific therapeutic targets. However, to which extent this result holds true for the different stages of hematopoietic development in primary cells and whether therapeutic agents can be efficiently delivered to those cells is still unclear. In this study, we demonstrate that primary AML cells harboring the chromosomal translocation t(8;21) are critically dependent on the corresponding fusion gene, RUNX1::RUNX1T1, to suppress differentiation and maintain stemness. Silencing RUNX1::RUNX1T1 expression using siRNA-loaded lipid nanoparticles induces substantial changes in chromatin accessibility, thereby redirecting the leukemia-associated transcriptional network towards a myeloid differentiation program. Single-cell analyses reveal that this transcriptional reprogramming is associated with the depletion of immature stem and progenitor-like cell populations, accompanied by an expansion of granulocytic and eosinophilic/mast cell-like populations with impaired self-renewal capacity. These findings underscore the essential role of RUNX1::RUNX1T1 in sustaining AML and highlight the therapeutic potential of targeting fusion gene expression in primary AML cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5469845f04d542fe9db7d0ec4a2180ccb6b7c697" target='_blank'>
              Fusion Gene Depletion Eliminates Stemness and Induces Bidirectional Differentiation of Acute Myeloid Leukemia.
              </a>
            </td>
          <td>
            Polina K Derevyanko, L. Swart, L. D. Mata Casimiro, Anita van Oort, M. Du Plessis, Luca van den Brink, Minoo Ashtiani, C. Zwaan, A. Krippner-Heidenreich, Constanze Bonifer, Raymond Schiffelers, Josef Vormoor, S. Kellaway, Olaf Heidenreich
          </td>
          <td>2025-09-24</td>
          <td>Blood</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Multiple myeloma (MM) is highly heterogeneous, with relapse occurring in the majority of cases, and recent advancements in single-cell RNA sequencing (scRNA-seq), sc-metabolism profiling, and bulk RNA-seq have facilitated the identification of cell subpopulations and metabolic reprogramming at the single-cell level, uncovering novel molecular mechanisms. This study aims to establish a multi-omics atlas of MM, characterizing the cell subpopulations and signaling pathways that drive immune evasion and disease progression. Additionally, sc-metabolic profiling identifies reprogramming patterns and informs therapeutic screening. We integrated scRNA-seq and bulk RNA-seq data using R to analyze immune and non-immune cell features and pathways in MM. Metabolic reprogramming was assessed via sc-metabolic profiling, and drug candidates were screened through multi-omics integration, with efficacy evaluated in vitro using CCK-8 assays, flow cytometry, Western blotting, and CalcuSyn software. Novel MM subpopulations were identified, including myeloma-activated hematopoietic stem cells and ISG15+ B cells, which correlated with survival and were validated by multiplex immunofluorescence. IFN-γ is primarily secreted by effector memory CD8+T cells, and IFN-α is primarily secreted by non-classical monocytes, driving an IFN-γ/α-B2M feedback loop. Multi-omics identified four drug candidates, each demonstrating anti-tumor effects against myeloma cell lines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ed7155f8bb3f2fadcad9b5654646e878c15255d" target='_blank'>
              Multi-omics reveals immune features in immune and non-immune cells, an IFN-γ/IFN-α-B2M positive feedback loop, and targeted metabolic therapy in multiple myeloma
              </a>
            </td>
          <td>
            Chen Li, Yaping Liao, Ling Xu, Yan Chen
          </td>
          <td>2025-09-08</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Diffuse large B-cell lymphoma (DLBCL) is a highly heterogeneous and aggressive hematologic malignancy, with the activated B-cell-like (ABC) subtype displaying particularly poor prognosis due to inherent treatment resistance and elevated recurrence rates. Despite advances in targeted therapies and immunotherapies, a significant proportion of patients experience relapse or refractory disease, highlighting the urgent need for novel biomarkers and innovative therapeutic strategies to improve clinical outcomes. Methods A multi-dimensional analysis of SH3BP5 expression was performed across DLBCL subtypes, integrating transcriptomic, proteomic, and clinical datasets to assess its correlation with immune infiltration, tumor metabolism, and patient prognosis. Single-cell RNA sequencing data were employed to examine the tumor microenvironment (TME) with higher resolution. Further analysis of the association between SH3BP5 and immune checkpoint gene expression was conducted to explore its potential role in immunotherapy response. Functional in vitro assays were carried out to assess the impact of SH3BP5 knockdown on DLBCL cell proliferation and apoptosis. Results The analysis revealed that SH3BP5 is preferentially overexpressed in the ABC subtype of DLBCL across multiple datasets and validated cohorts, and its high expression is significantly associated with poor overall survival. Single-cell transcriptomic profiling demonstrated that SH3BP5 is mainly expressed in malignant B cells and inversely correlated with immune cell infiltration, particularly CD8 + T cells. Mechanistically, pathway enrichment and metabolic assays indicated that SH3BP5 is linked to mitochondrial metabolic reprogramming, promoting oxidative phosphorylation (OXPHOS) and potentially contributing to reduced responsiveness to immune checkpoint inhibitors (ICIs). Functional studies showed that SH3BP5 knockdown significantly suppressed DLBCL cell proliferation, induced apoptosis, and reduced tumor cell viability in vitro. Conclusion This study suggests that SH3BP5 may serve as a prognostic biomarker and a potential therapeutic target in DLBCL, particularly within the ABC subtype. By delineating its associations with immune evasion and metabolic reprogramming, these findings provide a mechanistic basis for further exploration of SH3BP5-targeted interventions to help overcome therapy resistance. Future studies in larger clinical cohorts and functional models are warranted to validate these results and assess the potential of integrating SH3BP5 expression profiling into precision medicine strategies for DLBCL. Trial registration The study was registered in the Chinese Clinical Trial Registry (ChiCTR2200060430; http://www.chictr.org.cn/) on June 1, 2022. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06951-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d27222190fd1301e27cfd9869b08b07fc58aa185" target='_blank'>
              SH3BP5-driven metabolic-immune crosstalk in DLBCL: a prognostic biomarker and therapeutic target for reshaping immunosuppressive microenvironment
              </a>
            </td>
          <td>
            Tong Wu, Yi Yang, Yuan Zong, Jiawen Zhao, Xiaoyu Zhao, Lei Li, Yiming Gao, Ning Li, Liting Jiang, Yinyin Xie
          </td>
          <td>2025-09-24</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background: The number one cause of cancer-related deaths across the globe is metastatic spread of malignant tumors. To determine therapeutic targets, it is important to understand cellular and genetic changes that facilitate invasion, dissemination, and colonization. Although this field can be improved through genomics, the interactions between tumor-intrinsic alterations and the microenvironment are not fully outlined, and thus, current metastasis-preventive approaches are limited. 
Objective: The purpose of the study was to examine cellular changes and genetic mutations related to tumor metastasis, to examine individual patient features, and to examine clinical correlations that point to possible areas of therapeutic intervention. 
Study Design: A retrospective observational study. 
Place and Duration of study: Pakistan Institute of Medical Sciences, PIMS, Islamabad, Pakistan. Jan 2024 to December 2024 
Methods: Review of 100 patients with histologically proven malignant tumors was a retrospective observational study. Institutional records were used to collect demographic and clinical data. Recurrent mutations in TP53, KRAS, and PIK3CA of tumor tissue samples and epithelial-mesenchymal transition markers were assessed by immunohistochemistry. Statistical analyses comprised descriptive statistics, calculation of mean ages and standard deviation and comparison of groups using Student t-test with a significance level set to p < 0.05. 
Results: 100 patients (52 males and 48 females) were used as the cohort. The mean age was 59.4 years +- 11.2 SD. Genomic findings identified TP53 mutations in 45 percent of patients, KRAS mutations in 28 percent and PIK3CA mutations in 18 percent. Loss of E-cadherin expression was also found in 36 percent of samples and was associated with greater invasiveness. Analysis of the metastatic burden revealed that patients who had both TP53 and KRAS mutations had a much greater metastatic burden than patients without (p = 0.018). Clinical associations suggested that genetic changes and epithelial-mesenchymal transition signatures operated as a combined effect on progression, which underscored a suggestion that molecular heterogeneity is the basis of metastatic potential. 
Conclusion: We have shown that genetic mutations, together with cellular reprogramming (epithelial-mesenchymal transition) contribute to metastasis. Combined TP53 and KRAS mutations and their relationship with metastatic progression are correlated, which is important to identify potential risk stratification biomarkers. The combination of molecular and cellular analysis provides informative tools to develop therapies that will help intercept metastatic progression, which ultimately will raise the prognosis and survival rates in patients with progressive malignant tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95a13e9e21cb70713f15ecb05360a37152fe398f" target='_blank'>
              Analyzing The Cellular and Genetic Alterations That Contribute to The Metastatic Spread of Malignant Tumors
              </a>
            </td>
          <td>
            Rakhshanda Shamim, Zaid Waqar, Adeel Haidar, Ammar Niazi, Muhammad Mustafa Sadiq, Bushra Anam Ali
          </td>
          <td>2025-09-23</td>
          <td>Journal of Neonatal Surgery</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Breast cancer (BC) is one of the most common malignancies worldwide. Its complex molecular mechanisms make traditional treatments ineffective in some patients. In recent years, advances in genetic engineering technologies have opened up new possibilities for precision medicine in BC. Researchers have attempted to intervene in cancer cell proliferation and induce apoptosis through various genetic strategies. Preclinical studies are increasing, and therapeutic targets and delivery technologies are expanding. However, current research has largely focused on specific cases or single technologies, lacking a systematic integration and analysis of various genetic strategies from a mechanistic perspective. This paper reviews three major genetic engineering strategies for BC based on their therapeutic mechanisms: RNA interference, gene compensatory expression, and gene repair and deletion. Representative studies, key technologies, challenges, and development trends are analyzed for each. Furthermore, this paper summarizes common challenges across these different mechanisms regarding delivery efficiency, stability, and targeting, and proposes potential directions for future synergistic strategies. This paper helps clarify current research approaches in BC gene therapy from a mechanistic perspective, providing a reference for the design of subsequent cross-mechanism integration and precision intervention strategies. However, issues such as mechanism overlap and diversification of delivery platforms still need further in-depth exploration. Future research can focus more on multi-mechanism combination strategies, individualized treatment pathways, and feasibility assessment of clinical transformation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38bb75ae7dd66792d49d2225441735e8c43eeb18" target='_blank'>
              Applications of Three Gene Engineering Mechanisms in Breast Cancer Therapy
              </a>
            </td>
          <td>
            Yifan Tian
          </td>
          <td>2025-09-03</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 In contrast to prevailing wisdom that cancer is a genetic disease that evolves via Darwinian selection, it is increasingly recognized that in addition to genetic mutations, intratumoral heterogeneity also results through non-genetic mechanisms. Furthermore, it is now evident that the malignant phenotype results from complex interactions between genetic and non-genetic mechanisms. These interactions contribute to functional changes across multiple spatiotemporal scales, from molecular to cellular to tissue level, creating a heterogeneous cancer cell population. Cancer Systems Biology is a comprehensive, up-to-date treatise that addresses these issues. The main intent of this compendium is to encourage cross-pollination of ideas from experts in multiple disciplines including basic and translational science researchers, systems/mathematical biologists, and biophysicists and clinicians interested in decoding a systems-level emergent view of cancer so that a deeper understanding of disease progression can be gained and more effective treatment strategies can be designed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a03788cb0b91aa174c4a7535dcca74621b417df9" target='_blank'>
              Cancer Systems Biology
              </a>
            </td>
          <td>
            Ravi Salgia, M. Jolly, Prakash Kulkarni, Govindan Rangarajanm
          </td>
          <td>2025-09-01</td>
          <td>None</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="ABSTRACT Background Cisplatin and its derivatives remain a cornerstone in the treatment of solid malignancies. Resistance is a major factor limiting their clinical utility. Objectives In the present study, we set out to interrogate therapeutic approaches to target cisplatin‐resistant cancer cells. We focused on therapies exploiting metabolic pathways that are altered in drug‐resistant cells. We sought to find an existing therapy that has monotherapy efficacy against cisplatin‐resistant cancer cells that can also re‐sensitize to cisplatin. Methods We used lung and ovarian cancer cell lines with acquired resistance to cisplatin together with drug sensitivity assays, conducted both with monotherapies and cisplatin combinations. Results We show that cancer cell lines with acquired resistance to cisplatin have altered levels of enzymes involved in glutamine metabolism, which can result in differential sensitivity to targeted agents. We show that expression of one of these enzymes—the glutamate‐cystine antiporter SLC7A11, up‐regulated 6‐fold in a cisplatin‐resistant lung cancer cell line—has potential prognostic significance in lung cancer but not ovarian cancer. After identifying a common dependency of cisplatin‐resistant cancer cells upon extracellular glutamine, we then evaluate the utility of the long‐standing anti‐leukemic therapy asparaginase (ASNase)—which possesses both asparaginase and glutaminase activity—as a potential approach. We show ASNase preferentially inhibits the proliferation of cisplatin‐resistant cancer cells and can potentially re‐sensitize these cells to cisplatin. Conclusions Our results underpin the prevalence of altered metabolism in cisplatin‐resistant cells and highlight the potential utility of re‐purposing ASNase to target these cells, warranting further investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f061244858dd7ffcf91e4f9a475213a0f0dca65" target='_blank'>
              Repurposing Asparaginase Therapy to Target Cisplatin‐Resistant Cancer Cells
              </a>
            </td>
          <td>
            Jiantao Wang, Nasim Pouryaghoub, Robert Strauss, J. Bartek, S. Zhang, Sean G. Rudd
          </td>
          <td>2025-08-10</td>
          <td>Fundamental & Clinical Pharmacology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="The field of oncology has been extensively studied to design more effective and efficient treatments. This review explores the advanced techniques that are transforming our comprehension of cancer and its constituents. Specifically, it highlights the signaling pathways that drive tumor progression, angiogenesis, and resistance to therapy, as well as the modern approaches used to identify and characterize these pathways within the tumor microenvironment (TME). Key pathways discussed in this review include vascular endothelial growth factor (VEGF), programmed cell death protein 1/programmed cell death ligand 1 (PD-1/PD-L1), cytotoxic T-lymphocyte-associated protein 4 (CTLA-4), and various extracellular matrix (ECM) pathways. Conventional methods of diagnosis have yielded sufficient knowledge but have failed to reveal the heterogeneity that exists within the TME, resulting in gaps in our understanding of the cellular interaction and spatial dynamics. Single-cell sequencing (SCS) and spatial transcriptomics (ST) are effective tools that can enable the dissection of the TME with the resolution capacity of a single cell. SCS allows the capture of the unique genetic and transcriptomic profiles of individual cells along with rare cell types and new therapeutic targets. ST complements this by providing a spatial map of gene expression, showing the gene expression profiles within the tumor tissue at specific sites with good accuracy. By mapping gene expression patterns at a single cell level and correlating them with the spatial locations, researchers can uncover the intricate networks and microenvironmental influences that contribute to tumor heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edfe2d5d0e8233b17d2256f6ca476fecb056c29a" target='_blank'>
              Decoding the Tumor Microenvironment: Insights and New Targets from Single-Cell Sequencing and Spatial Transcriptomics
              </a>
            </td>
          <td>
            Shriya Pattabiram, Prakash Gangadaran, Sanjana Dhayalan, Gargii Chatterjee, Danyal Reyaz, Kruthika Prakash, Raksa Arun, R. Rajendran, B. Ahn, K. N. Aruljothi
          </td>
          <td>2025-09-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Pancreatic cancer is marked by a poor prognosis and an exceptionally high mortality rate, with its aggressive nature contributing to its classification as a highly malignant disease. For effective therapeutic strategies, the development of sophisticated and regulated DNA manipulation methods is essential. Originally part of the prokaryotic immune system, CRISPR/Cas has emerged as a pivotal genome-editing tool with promising applications in pancreatic cancer research and therapy. This gene editing method is known for simplicity, rapid advancement, and superior precision compared to earlier techniques. Its adaptability allows precise gene editing for therapeutic purposes, including oncogene silencing and correction of pathogenic mutations. Additionally, CRISPR-driven gene editing has facilitated the development of pancreatic cancer models, which serve as valuable platforms for drug discovery and personalized treatment strategies, offering deeper insights into the genetic landscape of pancreatic tumors. This article provides an overview of the current applications of CRISPR technology in gene therapy and cancer research, particularly in the context of pancreatic cancer, and lays the foundation for future studies. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ab39b4071496f7653becd6d74d5e06cbbcdd5a1" target='_blank'>
              CRISPR/Cas technologies in pancreatic cancer research and therapeutics: recent advances and future outlook
              </a>
            </td>
          <td>
            Hanieh Noormohamadi, Hamed Soleimani Samarkhazan, Maryam Kargar, F. Maroufi, Nazli Servatian, Fatemeh Davami
          </td>
          <td>2025-08-11</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="PAX3-FOXO1, an oncogenic transcription factor, drives a particularly aggressive subtype of rhabdomyosarcoma (RMS) by enforcing gene expression programs that support malignant cell states. Here, we showed that PAX3-FOXO1+ RMS cells exhibit altered pyrimidine metabolism and increased dependence on enzymes involved in de novo pyrimidine synthesis, including dihydrofolate reductase (DHFR). Consequently, PAX3-FOXO1+ cells displayed increased sensitivity to inhibition of DHFR by the chemotherapeutic drug methotrexate, and this dependence was rescued by provision of pyrimidine nucleotides. Methotrexate treatment mimicked the metabolic and transcriptional impact of PAX3-FOXO1 silencing, reducing expression of genes related to PAX3-FOXO1-driven malignant cell states. Accordingly, methotrexate treatment slowed the growth of multiple PAX3-FOXO1+ tumor xenograft models but not the fusion-negative counterparts. Taken together, these data demonstrate that PAX3-FOXO1 induces cell states characterized by altered pyrimidine dependence and nominate methotrexate as an addition to the current therapeutic arsenal for treatment of these malignant pediatric tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ba839c9a2b458336e1dd8e4d0285ec2394738d6" target='_blank'>
              PAX3-FOXO1 Drives Targetable Cell State-Dependent Metabolic Vulnerabilities in Rhabdomyosarcoma.
              </a>
            </td>
          <td>
            Katrina I. Paras, Julia S. Brunner, Angela M Montero, Jacob A Boyer, Benjamin T. Jackson, Sangita Chakraborty, Abigail Xie, Kristina Guillan, Armaan Siddiquee, Lourdes Pajuelo Torres, Joshua D. Rabinowitz, A. Kung, Daoqi You, F. D. Dela Cruz, Lydia W. S. Finley
          </td>
          <td>2025-09-05</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Oncogene targeted cancer therapies can provide deep responses but frequently suffer from acquired resistance.1 Therapeutic approaches to treat tumours which have acquired drug resistance are complicated by continual tumour evolution and multiple co-occurring resistance mechanisms.2,3 Rather than treating resistance after it emerges, it may possible to prevent it by inhibiting the adaptive processes which initiate resistance but these are poorly understood.4 Here we report that residual cancer persister cells that survive oncogene targeted therapy are growth arrested by drug stress-induced intrinsic Type I interferon (IFN) signaling. To escape growth arrest, persister cells leverage apoptotic machinery to transcriptionally suppress interferon-stimulated genes (ISGs). Mechanistically, persister cells sublethally engage apoptotic caspases to activate DNA endonuclease DNA Fragmentation Factor B (DFFB, also known as Caspase-Activated DNase (CAD)) which induces DNA damage, mutagenesis, and stress response factor Activating Transcription Factor 3 (ATF3). ATF3 limits Activator Protein-1 (AP1)-mediated ISG expression sufficiently to allow persister cell regrowth. Persister cells deficient in DFFB or ATF3 exhibit high ISG expression and are consequently unable to regrow. Therefore, sublethal apoptotic stress paradoxically promotes regrowth of residual cancer cells that survive drug treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21f810637300567b18dd70e40b7019befcd5ee94" target='_blank'>
              DFFB suppresses interferon to enable cancer persister cell regrowth
              </a>
            </td>
          <td>
            August F Williams, David A. G. Gervasio, C. Turkal, Anna E. Stuhlfire, Michael X. Wang, B.E. Mauch, Rhea Plawat, Ariel H Nguyen, Michelle H. Paw, Mehrshad Hairani, Cooper P. Lathrop, Sophie H. Harris, Jennifer L. Page, Matthew J Hangauer
          </td>
          <td>2025-08-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Non-small cell lung cancer (NSCLC) was a major cause of cancer-related mortality globally. Despite advancements in immunotherapy and targeted therapies, clinical outcomes were still limited by tumor heterogeneity and treatment resistance. The transcription factor (TF) FOS, a key component of the AP-1 complex, was linked to tumor progression and therapy resistance in various cancers, but its precise mechanisms remained unclear, and its role in lung adenocarcinoma (LUAD) was unknown. We investigated the tumor microenvironment (TME) of LUAD using single-cell RNA sequencing (scRNA-seq) to identify potential therapeutic vulnerabilities and FOS-driven mechanisms. Methods We identified fourteen cell types by analyzing scRNA-seq data from LUAD samples (GSE164789) using Seurat (v4.4.0) and Harmony for batch correction. InferCNV was utilized to characterize the tumor cell subtypes after they were clustered using marker genes. CytoTRACE and Monocle were used to create pseudotime trajectories in order to map differentiation states. CellChat revealed intercellular communication networks, while SCENIC identified TF regulatory modules. The CCK-8, Edu, Transwell, and wound healing assays showed that FOS knockdown functionally validated A549 and NCI-H1975 cells. Furthermore, a prognostic model was developed. Results We discovered that invasive LUAD was dominated by a highly stem-like C0 MAFF+ tumor cell subtype that produced chemokines and activated lipid metabolism. These cells stimulated immunosuppression and tumor-associated macrophage (TAM) differentiation by interacting with macrophages via MIF-(CD74+CD44) signaling. Experiments using FOS knockdown demonstrated its role in maintaining invasion, migration, and proliferation. Using the MTRS model, patients were categorized into high- and low-risk cohorts, high-risk patients exhibited unique drug sensitivities. Immunoprofile analysis revealed higher M1 macrophages in high-risk patients, suggesting that FOS inhibition could repolarize TAMs and enhance immunotherapies. Conclusion Our studies show that FOS is a main regulator of C0 MAFF+ TCs in LUAD, polarizing macrophages via MIF and rewiring lipid metabolism to support cancer. The MTRS model offers clinical value for risk assessment even if FOS inhibition shows promise as a therapeutic approach to raise immunotherapy efficacy. Targeting the FOS could cause TME immunosuppression to be disrupted, thus LUAD presents a fresh precision oncology approach.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe2ef348cf0a48b9b9c178cc598309bdb56456d8" target='_blank'>
              Targeted and personalized immunotherapy in lung adenocarcinoma: single-cell RNA sequencing of MAFF+ tumor cells and the therapeutic potential of FOS
              </a>
            </td>
          <td>
            Xiangsong Cheng, Shu Chen, Yilong Fu, Runze Jiang, Yanlong Jing, Bizhu Zhao, Dong Guo, Liangyu Wang, Zi Ye, Yumeng Li, Xianliang Chen
          </td>
          <td>2025-08-27</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Tumor gene alterations can serve as predictive biomarkers for therapy response. The nucleotide excision repair (NER) helicase ERCC2 carries heterozygous missense mutations in approximately 10% of bladder tumors, and these may predict sensitivity to cisplatin treatment. To explore the clinical actionability of ERCC2 mutations, we assembled a multinational cohort of 2,012 individuals with bladder cancer and applied the highly quantitative CRISPR-Select assay to functionally profile recurrent ERCC2 mutations. We also developed a single-allele editing version of CRISPR-Select to assess heterozygous missense variants in their native context. From the cohort, 506 ERCC2 mutations were identified, with 93% being heterozygous missense variants. CRISPR-Select pinpointed deleterious, cisplatin-sensitizing mutations, particularly within the conserved helicase domains. Importantly, single-allele editing revealed that heterozygous helicase-domain mutations markedly increased cisplatin sensitivity. Integration with clinical data confirmed that these mutations were associated with improved response to platinum-based neoadjuvant chemotherapy. Comparison with computational algorithms showed substantial discrepancies, highlighting the importance of precision functional assays for interpreting mutation effects in clinically relevant contexts. Our results demonstrate that CRISPR-Select provides a robust platform to advance biomarker-driven therapy in bladder cancer and supports its potential integration into precision oncology workflows.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/298c9b125eade81e30854dcc4c068fdf43533872" target='_blank'>
              Quantitative functional profiling of ERCC2 mutations deciphers cisplatin sensitivity in bladder cancer
              </a>
            </td>
          <td>
            J. Börcsök, Diyavarshini Gopaul, Daphne Devesa-Serrano, Clémence Mooser, Nicolas Jonsson, M. Cagiada, D. Stormoen, M. N. Ataya, B. Guercio, H. Kaimakliotis, G. Iyer, Kresten Lindorff-Larsen, L. Dyrskjøt, Kent W. Mouw, Z. Szallasi, C. S. Sørensen
          </td>
          <td>2025-08-15</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="


 RAS inhibitors (RASi) have the potential to change the clinical landscape for pancreatic ductal adenocarcinoma (PDAC) and improve patient survival. Nevertheless, clinical resistance is likely to reduce the long-term benefits of RASi, and combinations that can bypass or overcome resistance will be needed. Here, we studied the dynamics of resistance to the RAS multi(ON) inhibitor RMC-6236 (daraxonrasib). Prior to frank drug resistance, PDAC cells persist in a drug tolerant state characterized by reduced proliferation; however, only a fraction of the drug-tolerant persister cells contributes to resistance. To identify and study the relevant, resistance-contributing cells, we developed a highly multiplexed, lentiviral DNA barcode system that is compatible with single-cell technologies, enabling paired lineage tracing and phenotypic profiling throughout RASi treatment. Our lineage tracing studies suggest that drug persistence and resistance are fueled by a subset of cells that are primed to survive RASi treatment. Ongoing work is directed at defining and tracking the cell states associated with each stage of resistance emergence.



 Jayne C. McDevitt, Connor J. Hennessey, Robert Vander Velde, Sydney M. Shaffer, Ben Z. Stanger. Clonal dynamics and molecular mechanisms mediating resistance to RAS inhibition [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B082.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95c1bd513ed6b1024a532086d606e43f01e09bb7" target='_blank'>
              Abstract B082: Clonal dynamics and molecular mechanisms mediating resistance to RAS inhibition
              </a>
            </td>
          <td>
            Jayne C McDevitt, Connor J Hennessey, R. V. Velde, Sydney M. Shaffer, Ben Z. Stanger
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Simple Summary Breast cancer (BC) remains a significant global health challenge, partly due to its ability to develop resistance to treatment. BC resistance arises from various biological mechanisms, including cancer cells actively expelling drugs, altering their internal metabolic and signaling pathways, evading the immune system, or persisting as drug-tolerant cancer stem cells. These changes occur in the context of a tumor microenvironment that further shields the cancer, limiting treatment access. As the disease advances, particularly when it spreads to other body parts, effective treatment becomes increasingly difficult. Recent advancements in artificial intelligence (AI) offer promising avenues for improving early detection, refining diagnoses, enhancing clinical decision-making and thereby personalizing treatment strategies. This review examines the biological mechanisms that cause BC resistance and its treatment. It further explores AI’s potential to address critical challenges, transforming BC therapy to provide improved survival rates and enhanced quality of life for patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfb5357bf49f1fab994fbcce36bcc1738760cec3" target='_blank'>
              The Underlying Mechanisms and Emerging Strategies to Overcome Resistance in Breast Cancer
              </a>
            </td>
          <td>
            Krishnaswamy Kannan, Alagarsamy Srinivasan, Aarthi Kannan, Nawab Ali
          </td>
          <td>2025-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Diffuse midline gliomas (DMGs) are universally fatal pediatric brain tumors with a median survival of just 11 months. Despite radiation therapy offering temporary benefit, there are no effective long-term treatments, and targeted therapies have shown minimal impact. The infiltrative nature of DMGs, combined with their critical brainstem location, underscores the urgent need to identify novel mechanisms driving tumor progression and resistance. Cellular senescence is a state of permanent cell-cycle arrest induced by stress and has traditionally been viewed as tumor-suppressive. However, recent evidence in adult cancers implicates senescent cells as active contributors to tumor growth, treatment resistance, and relapse through the secretion of pro-tumorigenic factors known as the senescence-associated secretory phenotype (SASP). While senescence has emerged as a hallmark of cancer in adults, its role in pediatric tumors such as DMGs remains largely unexplored. Using single-cell RNA sequencing of treatment-naïve DMG specimens, we identified rare populations of senescent glioma cells and microglia that would be undetectable using conventional approaches. Computational analyses revealed predicted paracrine interactions between senescent and non-senescent cells including pathways previously linked to tumor invasiveness and resistance. This study aims to define how senescent cells and their secretory programs shape the tumor microenvironment, promote DMG progression, and contribute to therapy resistance. By elucidating these mechanisms, we hope to uncover new therapeutic vulnerabilities in DMGs and expand our broader understanding of senescence as a driver of malignancy in pediatric cancers.



 Timothy H. Chang, Alexander Nassar, Ria Kedia, Shriya M. Rangaswamy, Claudia Kleinman, Pratiti Bandopadhayay. Uncovering senescence-driven pathways in diffuse midline glioma progression and treatment resistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr B040.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/785f69efbe0f5fccc4e2b6c0500f58472565b0dc" target='_blank'>
              Abstract B040: Uncovering senescence-driven pathways in diffuse midline glioma progression and treatment resistance
              </a>
            </td>
          <td>
            Timothy H. Chang, Alexander Nassar, Ria Kedia, Shriya Rangaswamy, Claudia L Kleinman, P. Bandopadhayay
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f4e4fc33eb2e8814d38d33c359427c463ea1f89" target='_blank'>
              Resistance to CDK7 inhibitors directed by acquired mutation of a conserved residue in cancer cells.
              </a>
            </td>
          <td>
            Chun-Fui Lai, Victoria I. Cushing, Ellen Olden, C. Bevan, R. C. Coombes, Basil J. Greber, L. Buluwela, Simak Ali
          </td>
          <td>2025-09-08</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="This article provides a comprehensive overview of targeted and epigenetic therapies for acute myeloid leukemia (AML), highlighting their role in advancing treatment strategies. Recognizing that AML arises from complex genetic and epigenetic alterations, a deeper understanding of its pathogenesis has paved the way for precision medicine. The article reviews key targeted therapies, including IDH inhibitors (e.g., ivosidenib, enasidenib) for IDH mutations, FLT3 inhibitors (e.g., midostaurin, gilteritinib) for FLT3 alterations, BCL2 inhibitors (e.g., venetoclax) impacting apoptosis, and emerging Menin inhibitors for specific subtypes like KMT2A-rearranged or NPM1-mutant AML. It also discusses epigenetic therapies that target reversible changes, such as DNMT inhibitors (e.g., azacitidine, decitabine) for DNA methylation and HDAC inhibitors for histone modifications, along with other promising agents under development targeting epigenetic regulators. Furthermore, the article explores gene therapy as a significant future direction, detailing advancements in gene editing (CRISPR/Cas9) and strategies designed to shield healthy hematopoietic cells from targeted toxicity (e.g., CD33, CD123 editing) or to enhance the anti-leukemic function of immune cells (e.g., CAR-T, TCR therapy). The future perspective emphasizes the importance of developing synergistic combination therapies and leveraging molecular insights for truly personalized treatment approaches to overcome therapeutic challenges and improve long-term outcomes for AML patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1f1a005625d793e87f3fac27421531cd23f4756" target='_blank'>
              Targeted and epigenetic therapies for acute myeloid leukemia treatment
              </a>
            </td>
          <td>
            Marzieh Shokoohi, Mobin Ghaderi, Milad Mokarami, Homaira Arian, Sara Mohammadnabi, Arastoo Farokhi
          </td>
          <td>2025-09-26</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 The dynamic evolution of the immune tumor microenvironment (TME) during targeted therapy is a critical yet poorly understood determinant of treatment response and resistance. While most studies compare immune states before and after treatment, temporal immune changes during therapy remain largely uncharacterized, limiting development of effective combination strategies. Here, we investigated immune dynamics throughout targeted therapy using mouse melanoma models that recapitulate human therapeutic responses. Single-cell RNA sequencing (scRNA-seq) identified an inflection point where the inflamed TME during tumor regression, characterized by robust NK cell infiltration, transitions to an immune-excluded state upon onset of drug-tolerant residual disease. We uncovered a unique macrophage subset that orchestrates NK cell recruitment during regression. Depletion of these macrophages using LysM-cre;iDTR mice significantly reduced NK cell infiltration. Extending these findings to human cancer, longitudinal scRNA-seq analysis of melanoma and lung cancer patient samples revealed dynamic NK cell infiltration during targeted therapy, establishing a direct link between innate immune remodeling and treatment outcome. Unlike prior prognostic studies assessing immune states at single time points, our results provide mechanistic evidence of a temporal relationship between NK cell infiltration and therapeutic efficacy. Together, these findings position immune evolution as a driver of acquired resistance and identify macrophage–NK cell crosstalk as a therapeutically actionable axis to overcome immune exclusion and improve targeted therapy.



 Chia-Hsin Hsu, Andrew C. White. Cancer acquires therapy resistance by converting immune infiltration to exclusion through NK cell-macrophage interactions [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Mechanisms of Cancer Immunity and Cancer-related Autoimmunity; 2025 Sep 24-27; Montreal, QC, Canada. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(9 Suppl):Abstract nr A012.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/064d31f044dc7aac267bdcec71d797f7f005e82d" target='_blank'>
              Abstract A012: Cancer acquires therapy resistance by converting immune infiltration to exclusion through NK cell-macrophage interactions
              </a>
            </td>
          <td>
            Chia-Hsin Hsu, 
          </td>
          <td>2025-09-24</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4df2588e9805448486d7a055c5c952eb019840ed" target='_blank'>
              Analysis of Naturally Occurring Somatic Insertions in the Human Genome.
              </a>
            </td>
          <td>
            , Zarko Manojlovic, Timothy Okitsu, C. Okitsu, Jordan Wlodarczyk, Nick Schillingford, Ramzi Bawab, Yong Hwee Eddie Loh, Michael R. Lieber
          </td>
          <td>2025-09-26</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Simple Summary Tumors grow in hostile environments where oxygen and nutrients are limited, and chemotherapy adds further stress. Under these conditions, cancer cells suppress normal protein production but activate an alternative mechanism called IRES-mediated translation. IRES elements functionally allow ribosomes to continue producing essential proteins when normal translation is impaired. This cap-independent translation mechanism permits cancer cells to produce key survival proteins, such as BCL2, XIAP, and NRF2, which are essential to block cell death, resist chemotherapy, and adapt to stressful conditions. IRES-mediate translation supports metabolic reprogramming and other cellular processes such as cancer cell plasticity, epithelial-to-mesenchymal transition, and cell survival. Understanding the mechanism of IRES-mediated translation may open new perspectives for therapies against aggressive and therapy-resistant tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/870d8cdc2a659d55d9fa2deee610d585af1de403" target='_blank'>
              Targeting Cancer Translational Plasticity: IRES-Driven Metabolism and Survival Within the Tumor Microenvironment
              </a>
            </td>
          <td>
            F. Damiano, Benedetta Di Chiara Stanca, L. Giannotti, E. Stanca, Angela Francesca Dinoi, L. Siculella
          </td>
          <td>2025-08-22</td>
          <td>Cancers</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c96cb29c7c4ee18f843780f85e8c5d424d7abe77" target='_blank'>
              Epigenetic modulation in cancer drug discovery: promising targets and clinical applications.
              </a>
            </td>
          <td>
            S. Vickram, Shofia Saghya Infant, M. Sivasubramanian, Saravanan Anbalagan, Mathan Muthu Chinnakannu Marimuthu, Hitesh Chopra
          </td>
          <td>2025-09-02</td>
          <td>Pharmacological reports : PR</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Paclitaxel remains a cornerstone of cancer therapy, yet resistance significantly limits its clinical efficacy. Emerging evidence implicates circular RNAs (circRNAs) as key regulators of paclitaxel resistance, offering potential biomarkers and therapeutic targets. This review synthesizes current knowledge on circRNA-mediated resistance mechanisms, categorizing their roles as promoters or suppressors of chemoresistance. We highlight two major pathways—miRNA sponging and protein interactions—through which circRNAs alter drug sensitivity. Moreover, we focus on enhancement of epithelial–mesenchymal transition and modulation of autophagy as two significant downstream pathways influenced by the established primary mechanisms. Notably, our analysis reveals context-dependent dualities, where certain circRNAs exert opposing effects in different cancers, underscoring the need for tissue-specific therapeutic strategies. Furthermore, we identify critical gaps in understanding circRNA regulatory networks and their clinical translatability. By evaluating circRNA signatures associated with paclitaxel response, this review proposes a framework for biomarker development and combination therapies to circumvent resistance. Our findings emphasize the urgency of functional studies and standardized profiling methods to harness circRNAs for personalized oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b6cc3f7c3626a0076f8fb95907d4419e7ad6fdb" target='_blank'>
              Circular RNAs: key regulators of paclitaxel sensitivity and resistance in cancer
              </a>
            </td>
          <td>
            Zahra Tajik, S. Ghafouri-Fard
          </td>
          <td>2025-08-18</td>
          <td>European Journal of Medical Research</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75865e4fe743320d35269c6c38da63e48d3392a1" target='_blank'>
              Lineage plasticity and histological transformation: tumor histology as a spectrum.
              </a>
            </td>
          <td>
            Xiaoling Li, Eric E. Gardner, Sonia Molina-Pinelo, Clare Wilhelm, Ping Mu, Á. Quintanal-Villalonga
          </td>
          <td>2025-09-30</td>
          <td>Cell research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ea21e9584f32082e944d86332497710d034ecd5" target='_blank'>
              Identification of antibiotic resistance genes from whole genome and metagenome sequencing datasets
              </a>
            </td>
          <td>
            Haseeb Manzoor, Li Hao, M. R. Kayani
          </td>
          <td>2025-08-13</td>
          <td>One Health Advances</td>
          <td>1</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68d27cc9b475222eaa7f4e83e26c9db176ee78f1" target='_blank'>
              Novel mutations in transducin-like enhancer of split 6 cause infertility in women with embryonic arrest.
              </a>
            </td>
          <td>
            Jifan Tan, Yue Jiang, Ching Wan, Kehan Zhu, Yan Xu, Yanwen Xu, Jia Huang
          </td>
          <td>2025-09-27</td>
          <td>Journal of assisted reproduction and genetics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Breast cancer (BC) remains one of the most prevalent cancers worldwide and a significant cause of cancer-related mortality among women. Despite significant advancements in understanding the genetic foundations of BC, numerous research initiatives have historically focused on protein-coding genes, which constitute merely about 2% of the human genome. This focus has produced significant insights into oncogenes such as HER2, TP53, and BRCA1, along with tumor suppressor genes. Nonetheless, it has led to the non-coding portions of the genome garnering relatively less focus. Recent studies illuminate the crucial significance of non-coding DNA in cancer biology, highlighting its regulatory roles and influence on tumor formation, metastasis, and treatment resistance. This review examines the importance of non-coding DNA in BC. It provides an in-depth analysis of essential non-coding regions, their functions in gene regulation and chromatin structure, and their implications for various BC subtypes. Examining these facets, we seek to reveal the potential of non-coding DNA as a viable source of novel diagnostic markers and treatment approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/966867774ecd7f0803dece977fed36246e346fe6" target='_blank'>
              The Role of Non-Coding Regions in Breast Cancer: From Gene Regulation to Therapeutic Implications
              </a>
            </td>
          <td>
            Hussein Sabit, Sarah Sobhy, Shaimaa Abdel-Ghany, Al-Hassan Soliman Wadan, Olubukola Ayodele, Yasser Albrahim, Hirendra Banerjee, Ahmed Elhashash, B. Arneth
          </td>
          <td>2025-09-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Tumor mutational burden (TMB) has emerged as a potential surrogate for neoantigen load and an indicator of immune checkpoint (IC)-blockade response; however, its precise significance in breast cancer (BC) is not fully understood. Here, we comprehensively characterized the genomic repertoire of BCs with a TMB ≥ 10 mut/Mb (TMB-high [n = 527]) to identify putative predictors of importance. The predominant mutational signature was apolipoprotein B mRNA-editing enzyme catalytic polypeptide (APOBEC) in 64.7% of tumors. TMB-high BCs were enriched in KMT2C, ARID1A, PTEN, NF1, and RB1 alterations, which are associated with APOBEC mutagenesis. Further identified were loss-of-function ARID1A and PTEN alterations, which are linked to immune cell exclusion. ESR1 p.E380Q prevailed among all ESR1 hotspot mutations, supporting APOBEC-mediated effects. Finally, mutations in DNA damage response and repair genes were seen at a higher frequency than in non-TMB-high BCs. These findings provide justification for exploring combined pharmacologic inhibition to improve IC-based efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e8980a3a4147f92cf6718bb89169af5da92bd71" target='_blank'>
              Genomic characterization of tumor mutational burden-high breast carcinomas
              </a>
            </td>
          <td>
            Theodore Vougiouklakis, Chad M. Vanderbilt, Satshil Rana, A. Mohanty, F. Pareja, E. Brogi, Christopher J. Schwartz, M. Arcila, Marc Ladanyi, Hanna Y Wen, D. Ross
          </td>
          <td>2025-08-08</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Colorectal cancer (CRC) represents the third leading cause of cancer-related deaths. Integrating cellular and molecular data from individual patients has become valuable for diagnosis, prognosis, and treatment selection. Here, we present a comparative mRNA-seq analysis of tissue samples from 32 CRC patients, pairing tumors with adjacent healthy tissues. Differential expression gene (DEG) analysis revealed dysregulated metabolic programs. We focused on the impact of overexpressed SLC7A11 (xCT) and SLC3A2, which compose the cystine/glutamate transporter (Xc-) system. To assess the oncogenic potential of the Xc- system, we analyzed gene perturbations from CRISPR screens across various cell types and used functional assays in five primary patient-derived organoid models. We identified a previously uncharacterized cell surface protein signature predicting chemotherapy resistance and highlighted the causality and potential of pharmacological blockage of ferroptosis as a promising avenue for cancer therapy. Redox homeostasis, ion/amino acid transporters, and regulators of neuronal survival and differentiation were pathways associated with these co-dependent genes in patient specimens. This study highlights several potential clinical targets for CRC therapy and promotes the use of patient-derived organoids oids to functionally validate in silico predictions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af3aa92ab85625c6d369fc2c084fc1ee49ca6118" target='_blank'>
              Patient-specific pharmacogenomics demonstrates xCT as predictive therapeutic target in colon cancer with possible implications in tumor connectivity.
              </a>
            </td>
          <td>
            M. Strecker, Keren Zohar, Martin Böttcher, T. Wartmann, H. Freudenstein, Maximilian Doelling, M. Andric, W. Shi, O. Kakhlon, Katrin Hippe, Beatrix Jahnke, D. Mougiakakos, Franziska Baenke, Daniel Stange, R. Croner, M. Linial, U. Kahlert
          </td>
          <td>2025-09-24</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63e99818fa12da57a725338e19c1d3d4c809e4ac" target='_blank'>
              Structural variants are enriched in deleterious visible phenotypes in Drosophila.
              </a>
            </td>
          <td>
            Alejandra Samano, Matthew Musat, Mihir Junaghare, Asad Ahmad, Mehlum Ali, Sebastian Alves, Sreeram Pasupuleti, Jelisha Perera, Omar Saada, Brady Sabido, Trevor Smith, Sophie Walz, Mahul Chakraborty
          </td>
          <td>2025-08-21</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c32a97dbee7d52daeda75142b34c4c8f28f256d2" target='_blank'>
              ZBTB11 depletion targets metabolic vulnerabilities in KRAS inhibitor-resistant PDAC.
              </a>
            </td>
          <td>
            Nathan L Tran, Jiewei Jiang, Min Ma, Gillian E. Gadbois, K. Gulay, Alyssa L Verano, Cara R. Schiavon, Elena Rebollo, Haowen Zhou, Chun-Teng Huang, Rabi Murad, David A Scott, Uri Manor, Anne G. Bang, Hervé Tiriac, A. Lowy, Eric S Wang, F. Ferguson
          </td>
          <td>2025-08-11</td>
          <td>Nature chemical biology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Chronic myeloid leukemia (CML) is a hematological malignancy driven by diverse genetic aberrations, with the Philadelphia chromosome and its resultant BCR‑ABL1 fusion gene constituting key pathogenic drivers. Atypical BCR‑ABL1 fusion transcripts have distinctive structural and functional properties. Structural divergence in these variants leads to functional alterations of encoded oncoproteins, potentially influencing disease progression and therapeutic responsiveness. Conventional diagnostic modalities, including reverse transcription‑PCR and fluorescence in situ hybridization, may fail to detect rare variants, necessitating complementary high‑sensitivity techniques such as next‑generation sequencing). Tyrosine kinase inhibitors (TKIs), including imatinib and dasatinib, remain cornerstone treatments; however, marked inter‑variant heterogeneity in TKI responsiveness is observed: Patients harboring e13a3/e14a3 transcripts generally show favorable prognoses, while those with e1a3/e6a2 variants demonstrate an increased risk of relapse and/or TKI resistance, often requiring multimodal strategies combining chemotherapy or allogeneic hematopoietic stem cell transplantation. Although Chimeric Antigen Receptor)‑T cell therapy has shown promise in treating (Philadelphia chromosome‑positive B‑cell Acute Lymphoblastic Leukemia, its application in CML, particularly in variants such as e1a3 or e6a2, is not currently recommended as a first‑line treatment. Despite advances in elucidating the clinical implications of fusion gene heterogeneity in leukemogenesis, the prognostic value of atypical BCR‑ABL1 isoforms requires further validation through multicenter studies with extended cohorts. This review aimed to summarize cases of atypical fusion genes in CML, with analysis of clinical characteristics, therapeutic interventions, and prognostic outcomes, to provide clinicians with enhanced reference material for improved patient management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ec3c00c917da742543a693d4c05a294ce191255" target='_blank'>
              The silent players: Atypical BCR‑ABL isoforms as biomarkers and therapeutic hurdles in CML pathogenesis (Review).
              </a>
            </td>
          <td>
            Xin Zhou, Ai Li, Dexiao Kong, Yuqi Shi, Peipei Zhang, Ning-Ning Shan
          </td>
          <td>2025-09-24</td>
          <td>Oncology reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Epigenetic dysregulation has been increasingly understood to be a key factor in the development of myeloid malignancies, often acting alongside genetic alterations to disrupt normal hematopoiesis. The inherent reversibility of epigenetic changes has provided an excellent opportunity for therapeutic intervention. Hypomethylating agents (HMA) preferentially alter gene expression in heavily methylated malignant myeloid cells by inhibiting DNA methyltransferases thereby altering gene expression. They have since become foundation therapies in myelodysplastic syndrome (MDS) and in older patients with acute myeloid leukemia (AML). However, complete responses to HMA monotherapy are limited and usually non-durable. In recent years, novel approaches have been sought to overcome resistance and expand the role of epigenetic therapies in MDS and AML as well as in less well-studied myeloid malignancies such as myeloproliferative neoplasms (MPNs) and MDS/MPN overlap syndromes. Combination regimens that synergistically pair HMAs or other epigenetically active therapeutics with agents targeting apoptosis, cellular metabolism, and immune evasion have also shown early promise in improving patient outcomes. Oral formulations of HMAs have made maintenance strategies more convenient and tolerable for patients, with demonstrated benefits in AML and ongoing investigations in MDS and post-transplant settings. This review will explore these novel therapeutic strategies in the treatment of myeloid malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14b616f7da39ac33cb77f5ad7d9b809d8c14dc8a" target='_blank'>
              Novel approaches to the use of hypomethylating agents in myeloid malignancies.
              </a>
            </td>
          <td>
            Justin Cheng, Casey O’Connell
          </td>
          <td>2025-09-25</td>
          <td>Epigenomics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The mechanism underlying the pathogenic impact of mutations within intrinsically disordered regions of proteins remains enigmatic, and the mechanisms behind compensatory responses to these perturbations lie within an even deeper veil of obscurity. This study focuses on the compensatory mechanisms of single nucleotide variants within the disordered C-terminal tail of the BTG2 protein, a crucial regulator in cell cycle control. Here, we develop a novel approach by combining molecular dynamics simulations with time-dependent linear response theory to accurately compute the long-distance coupling dynamics between the tail and the structured domain. Using this approach, we reveal how specific mutations can counteract the functional disruptions caused by a known disease-associated mutation, V141M. Our findings demonstrate that the disordered tail regulates critical binding sites allosterically, and a weakening of this modulation may contribute to disease manifestation. However, compensatory mutations restore lost interactions between the disordered region and binding sites, exerting long-distance dynamic control over both critical binding sites and the mutation site 141M. This secondhand allosteric control could be a general mechanism for compensatory mutations to rescue function. These insights not only illuminate the pathogenic mechanisms at play but also offer a framework for identifying potential therapeutic targets in diseases associated with disordered protein regions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44953c05dc0147543f5bd00f5cc928d283c8664e" target='_blank'>
              Second-order allosteric control as a mechanism for compensatory mutations in B-cell translocation gene 2.
              </a>
            </td>
          <td>
            Nicholas J. Ose, P. Campitelli, Tushar Modi, S. Ozkan
          </td>
          <td>2025-08-25</td>
          <td>Protein science : a publication of the Protein Society</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4026bb201d062046e64e9ee6c696085e559cd08a" target='_blank'>
              Treatment of NSCLC after chemoimmunotherapy - are we making headway?
              </a>
            </td>
          <td>
            M. Reck, N. Frost, Solange Peters, Bernard A. Fox, R. Ferrara, R. Savai, F. Barlesi
          </td>
          <td>2025-08-14</td>
          <td>Nature reviews. Clinical oncology</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="ABSTRACT Glioblastoma is a highly aggressive and devastating brain malignancy with dismal prognosis and extremely limited therapeutic options. Identification of prognostic biomarkers and therapeutic targets from multi‐omics data is critical for improving patient outcomes. In this study, we investigated the clinical significance of cellular heterogeneity and super‐enhancer‐driven regulatory networks, which are critically implicated in glioblastoma progression and treatment resistance. We first performed scRNA‐seq to dissect tumour microenvironment heterogeneity, identifying 16 distinct cell clusters, including astrocytes, macrophages, and CD8+ T cells. CellChat analysis revealed key intercellular signalling pathways, with astrocytes and macrophages acting as central communication hubs. To integrate bulk RNA sequencing data, we applied the Scissor algorithm to identify survival‐associated cell states. By combining single‐cell and bulk transcriptomic data, we uncovered 642 survival‐related genes, including QKI and RBM47, which robustly predicted patient survival and immunotherapy response. Furthermore, WGCNA analysis identified seven co‐expression modules and super enhancer‐regulated networks orchestrated by transcription factors (RFX2, RFX4) and hub genes (NEAT1, CFLAR). These networks stratified patients into high‐ and low‐risk groups with significant survival differences. Collectively, our findings elucidate the intricate interplay between cellular heterogeneity and super enhancer‐driven gene regulation in glioblastoma, providing a translational framework for targeting oncogenic hubs and modulating microenvironment interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7616587b7549f42937faeb12de6e3dcf343225e7" target='_blank'>
              Integration of Single‐Cell RNA and Bulk RNA Sequencing Reveals Cellular Heterogeneity and Identifies Survival‐Associated Regulatory Networks in Glioblastoma
              </a>
            </td>
          <td>
            Zijun Xu, Bohan Xi, Jiaming Huang, Liqiang Zhang, Sifu Cui, Xianwei Wang, Dong Chen, Shupeng Li
          </td>
          <td>2025-08-13</td>
          <td>IET Systems Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Despite extensive prior research on prostate cancer (PCa) transcriptomics, the molecular mechanisms underlying the disease’s progression, particularly in the castration-resistant or metastatic stages, remain incompletely understood. The majority of recent research has concentrated on bulk RNA sequencing, which could mask the variation found in tumor microenvironments. This study aims to address this gap by integrating single-cell RNA sequencing (scRNA-seq) and bulk RNA sequencing with weighted gene co-expression network analysis (WGCNA) to investigate the molecular mechanisms of PCa at a higher resolution. In order to further individualized treatment plans for PCa, we aim to discover important genes and signaling pathways that could be used as therapeutic targets. We first preprocessed expression profile data from prostate cancer tissue samples, selecting 9,809 high-quality cells from a dataset. Following batch correction with Harmony and dimensionality reduction with principal component analysis (PCA), we used the Louvain clustering algorithm to divide the cells into discrete subtypes. The clusters were then visualized using t-SNE. This resulted in 16 cellular subtypes categorized into five major cell types: epithelial cells, monocytes, endothelial cells, CD8 + T-cells, and fibroblasts. Analysis of receptor-ligand pairs uncovered significant interactions between monocytes and both tumor cells and endothelial cells. Applying the high-dimensional WGCNA (hdWGCNA) method to construct a gene co-expression network, we detected seven gene modules, four of which were highly expressed in tumor cell subtypes and contained 380 key genes. Combining pathway analysis, we ultimately screened six key genes: CNPY2, CPE, DPP4, IDH1, NIPSNAP3A, and WNK4. We used Cox univariate regression and least absolute shrinkage and selection operator (lasso) regression techniques to build a prognostic prediction model that included these six important genes based on clinical data gathered from PCa patients. The prognostic prediction model constructed in this study demonstrated excellent predictive performance in both the training set and an external validation set, with the high-risk group showing significantly lower overall survival (OS) than the low-risk group. Furthermore, there was a substantial correlation found between risk scores and several immune-related gene sets, chemotherapeutic drug sensitivity, and tumor immune infiltration. High- and low-risk groups exhibited significant differences in immune cell content, immune factor levels, and immune dysfunction. Further analysis revealed significant correlations between the expression levels of model genes and multiple disease-related genes. Through Gene Set Variation Analysis (GSVA) and Gene Set Enrichment Analysis (GSEA), we uncovered perturbations in multiple signaling pathways in high- and low-risk groups, potentially impacting the prognosis of PCa patients. This study uncovers key genes and signaling pathways in the prostate cancer tumor microenvironment, particularly genes such as CNPY2, CPE, DPP4, IDH1, NIPSNAP3A and WNK4, which have potential as therapeutic targets. Our findings provide new insights into personalized treatment strategies for PCa and warrant further clinical validation in the future. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-15682-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/779fa642ce8a3880c524211b95657271b91e6425" target='_blank'>
              Integrative analysis of molecular mechanisms in prostate cancer via single-cell RNA sequencing and weighted gene co-expression network analysis
              </a>
            </td>
          <td>
            Jing Zhai, Yizhou Wang, Yu Zhang, Wenhui Zhu, Xinyu Xu, Yu Peng, Guanxiong Ding
          </td>
          <td>2025-09-26</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Hypoxia-inducible factor (HIF) is a master regulator of cancer cell adaptation to tumor hypoxia and is involved in cancer progression. Single-cell (sc) differences in the HIF response allow for tumor evolution and cause therapy resistance. These sc-differences are usually ascribed to tumor microenvironmental differences and/or clonal (epi)genetic variability. However, the sc-heterogeneity of the HIF response in otherwise identical cells cultured under defined in vitro conditions has not yet been addressed. Therefore, we analyzed the sc-response to hypoxia in nonclonal cell lines and multiple clonal derivatives, including HIF-1α or HIF-2α knockouts. While HIF-1α and HIF-1 target mRNA sc-heterogeneity was slightly higher than global transcription or specific housekeeping messenger RNAs (mRNAs), HIF-2α and especially HIF-2 target mRNA sc-heterogeneity was extraordinary, and remained in independent clones following HIFα knockouts. Unexpectedly, neither HIF-2α mRNA nor nuclear protein levels correlated with target mRNA levels. Unsupervised but not supervised HIF target gene dimensionality reduction revealed the initial sample composition after scRNA-seq, demonstrating that, owing to sc-heterogeneity, individual HIF target genes are not sufficient to unequivocally identify hypoxic cancer cells. In conclusion, the pronounced intrinsic sc-heterogeneity of the HIF response represents a hitherto unrecognized feature of cancer cells that impairs clinical HIF pathway-dependent cancer cell identification and targeting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b54dd117e53122de89768b0fc8a01ce303d40e2" target='_blank'>
              Inherent single-cell heterogeneity of the transcriptional response to hypoxia in cancer cells
              </a>
            </td>
          <td>
            Małgorzata Wilk, Thomas Knöpfel, Stana M Burger, S. N. Khodo, Roland H. Wenger
          </td>
          <td>2025-08-28</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is an aggressive and heterogeneous liver cancer with restricted therapy selections and poor diagnosis. Although there have been great advances in genomics, the molecular mechanisms essential to HCC progression are not yet fully implicit, particularly at the single-cell stage. This research utilized single-cell RNA sequencing technology to evaluate transcriptional heterogeneity, immune cell infiltration, and potential therapeutic targets in HCC. A detailed bioinformatics pipeline used in the experiment included quality control, feature selection, dimensionality reduction using Principal Component Analysis (PCA), Uniform Manifold Approximation and Projection (UMAP), and t-distributed stochastic neighbor embedding (t-SNE), clustering, differential gene expression, pseudotime trajectory inference, and immune cell profiling with GSEA and survival analysis examining potential biomarkers of survival. Key findings include the identification of 1178 differentially expressed genes (DEGs), with macrophage infiltration contributing to immune evasion. Notably, APOE and ALB are linked to a better prognosis, while XIST and FTL are associated with poor survival. The potential drug candidates include IGMESINE in the case of SERPINA1 and PKR-A/MITZ for APOA2 in the gene-drug interaction analysis. Graph Neural Network (GNN) is used to predict drug-gene interactions and rank potential therapeutic candidates. The model shows robust predictive performance (R²: 0.9867, MSE: 0.0581) and identifies important drug candidates, such as Gadobenate Dimeglumine and Fluvastatin, and describes repurposing opportunities in network analysis, enhancing computational drug discovery for novel treatments. This research sheds new light on HCC tumor evolution, immune suppression, and the potential drug target based on the viewpoint of the importance of single-cell approaches in liver cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21e827b59658da5d3c8017bf844efafdcd1183fd" target='_blank'>
              Integrating single-cell RNA sequencing and artificial intelligence for multitargeted drug design for combating resistance in liver cancer
              </a>
            </td>
          <td>
            Houhong Wang, Youyuan Yang, Junfeng Zhang, Wenli Chen, Jingyou Dai, Changquan Li, Qing Li
          </td>
          <td>2025-09-02</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Development is characterized by dynamic changes in gene expression as cells traverse genetic pathways and make lineage-specific commitments. Transcription factors, which drive gene expression, and microRNAs, the largest class of post-transcriptional regulators, often function together within the same genetic networks. These interactions frequently include direct regulation of one another and shared target genes, forming feedback and feedforward loops that fine-tune gene expression to establish and maintain cell identity. The interplay between transcriptional and post-transcriptional regulation is particularly extensive during development, where disruptions in gene expression programs can cause cells to become trapped in immature proliferative states that result in paediatric cancers. This review focuses on the intricate cross-regulation between transcription factors and microRNAs, highlighting their contributions to developmental cancers with a particular emphasis on neuroblastoma, the most prevalent extracranial solid tumour in children, which arises from the failure of neural crest-derived cells to properly differentiate during sympathoadrenal development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/008a3cb7bc612567fe4ce248462398bebd221cff" target='_blank'>
              Regulation and dysregulation of microRNA - transcription factor axes in differentiation and neuroblastoma
              </a>
            </td>
          <td>
            Fakhira H Nazki, C. Bracken
          </td>
          <td>2025-08-08</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Background Single base substitution (SBS) mutations, particularly C > T and T > C, are increased owing to unrepaired DNA replication errors in mismatch repair-deficient (MMRd) cancers. Excess CpG > TpG mutations have been reported in MMRd cancers defective in mismatch detection (dMutSα), but not in mismatch correction (dMutLα). Somatic CpG > TpG mutations conventionally result from unrepaired spontaneous deamination of 5’-methylcytosine throughout the cell cycle, causing T:G mismatches and signature SBS1. It has been proposed that MutSα detects those mismatches, prior to error correction by base excision repair (BER). However, other evidence appears inconsistent with that hypothesis: for example, MutSα is specifically expressed in S/G2 phases of the cell cycle, and defects in replicative DNA polymerase proofreading specifically cause excess CpG > TpG mutations in signature SBS10b. Methods We analysed mutation spectra and COSMIC mutation signatures in whole-genome sequencing data from 1803 colorectal cancers (164 dMutLα, 20 dMutSα) and 596 endometrial cancers (103 dMutLα, 9 dMutSα) from the UK 100,000 Genomes Project. We mapped each C > T mutation to its genomic features, including normal DNA methylation state, replication timing, transcription strand, and replication strand, to investigate the mechanism(s) by which these mutations arise. Results We confirmed that dMutSα tumours specifically had higher CpG > TpG burdens than dMutLα tumours. We could fully reconstitute the observed dMutSα CpG > TpG mutation spectrum by adding CpG > TpG mutations in proportion to their SBS1 activity to the dMutLα spectrum. However, other evidence indicated that the SBS1 excess in dMutSα cancers did not come from 5’-methylcytosine deamination alone: non-CpG C > T mutations were also increased in dMutSα cancers; and, in contrast to tumours deficient in BER, CpG > TpG mutations were biased to the leading DNA replication strand, at similar levels in dMutSα and dMutLα cancers, suggesting an origin in DNA replication. Other substitution mutations usually corrected by BER were not increased in dMutSα tumours. Conclusions There is a CpG > TpG and SBS1 excess specific to dMutSα MMRd tumours, consistent with previous reports, and we find a general increase in somatic C > T mutations. Contrary to some other studies, the similar leading replication strand bias in both dMutSα and dMutLα tumours indicates that at least some of the excess CpG > TpG mutations arise via DNA replication errors, and not primarily via the replication-independent deamination of 5’-methylcytosine. Supplementary Information The online version contains supplementary material available at 10.1186/s13073-025-01525-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2f0be2946a1331e8205341cd8d8b7fbe7da7996" target='_blank'>
              Replication-associated mechanisms contribute to an increased CpG > TpG mutation burden in mismatch repair-deficient cancers
              </a>
            </td>
          <td>
            Joseph C Ward, Ignacio Soriano, Steve Thorn, Juan Fernández-Tajes, Kitty Sherwood, Güler Gül, Joost Scheffers, A. Frangou, Ben Kinnersley, Ioannis Kafetzopoulos, Duncan Sproul, Sara Galavotti, C. Palles, Andreas J. Gruber, David N. Church, Ian Tomlinson
          </td>
          <td>2025-08-25</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="


 Platinum-based chemotherapeutic drugs induce intra- and interstrand purine crosslinks that causes DNA damage and tumor cell death. While these drugs are front-line treatments for multiple human cancers – including high-grade serous ovarian cancer (HGSOC) – platinum resistance and tumor recurrence remains a major clinical challenge. Here, we report findings from both a genome-wide CRISPRi screen and an RNA-binding protein (RBP)-focused CRISPRi screen aimed at identifying modulators of sensitivity and resistance of HGSOC cells to two widely used platinum agents, cisplatin and oxaliplatin, as well as to an microtubule-targeting taxane. Using an in vitro treatment regimen designed to mimics clinical treatment protocols, we found that loss of genes in the Fanconi anemia (FA) and homologous recombination (HR) DNA repair pathways markedly increased the killing of p53-deficient HGSOC cells following treatment with cisplatin or oxaliplatin. In contrast, knockdown of genes in the nucleotide excision repair (NER) pathway—despite its known role in repairing platinum-induced intrastrand crosslinks, particularly in G1—had a significantly lesser effect. These results suggest that platinum-induced cytotoxicity in HGSOC is primarily driven by replication stress, which is exacerbated by disruption of error-free DNA repair pathways active during the S- and G2 phases of the cell cycle. Our screens also identified several known and novel RBPs as potent sensitizers to both platinum agents and taxanes. Depletion of these RBPs resulted in substantial activation of interferon signaling, a hallmark of innate immune activation and cellular stress responses. Among them, the RNA-editing enzyme ADAR1 emerged as a key modulator of drug resistance. ADAR1 catalyzes adenosine-to-inosine RNA editing, thereby altering mRNA coding potential, secondary structure, and subcellular localization. Additionally, ADAR1 has been implicated in regulating the integrated stress response (ISR), a signaling network that enables cellular adaptation to environmental stress. As part of this stress adaptation, we observed that platinum and taxol treatment caused a translational shift from canonical cap-dependent translation mediated by eIF4G1 to a cap-independent mechanism reliant on eIF4G2, supporting selective translation of stress-responsive mRNAs. Together, our findings suggest that ADAR1-dependent suppression of interferon signaling and activation of the ISR contributes to resistance to both platinum-based chemotherapy and microtubule-targeting agents, which together comprise the current standard of care for HGSOC.



 Yi Wen Kong, Erika D. Handly, Robin A. Lu, Molly A. Bird, Jack Schaeffer, Samkyu P. Yaffe, Michael B. Yaffe. CRISPRi screening reveals RNA editing and the integrated stress response as major determinants of platinum and taxol sensitivity in high-grade serous ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr PR009.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/912428b4f14100051b63d5819642408cb6578a72" target='_blank'>
              Abstract PR009: CRISPRi screening reveals RNA editing and the integrated stress response as major determinants of platinum and taxol sensitivity in high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            Yi Wen Kong, E. Handly, Robin A. Lu, Molly A. Bird, Jack W. Schaeffer, Samkyu P. Yaffe, Michael B Yaffe
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Malignant tumors remain one of the leading causes of mortality worldwide. Conventional therapies, though effective at killing cancer cells, are accompanied by systemic cytotoxicity. It highlights the priority for precision oncology. Tumor neoantigens represent promising targets for selective immunotherapy, arising from cancer-specific mutations. At the same time, CRISPRCas9 genome editing provides remarkable accuracy in genetic manipulation, opening new avenues for cancer research. This paper reviews the emerging convergence of CRISPRCas9 technology and neoantigen-based interventions. It showcases recent progress in CRISPR-based neoantigen validation and the engineering of immune effector cells, as well as their application in personalized therapies. Additionally, this paper focuses on several important translational challenges that need to be overcome before broad clinical application, including tumor heterogeneity, off-target effects, and clinical safety. By assembling the current evidence and future scenarios in this field, this review aims to highlight that the combination of CRISPR genome editing with neoantigen targets would be a new direction in the field of precision oncology that may shape the future of cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85a4a7b9e7b67fef6945b9291a7e77c60ecc9358" target='_blank'>
              Precision Oncology at the Intersection of CRISPRCas9 and Tumor Neoantigens
              </a>
            </td>
          <td>
            Qingya Wang
          </td>
          <td>2025-10-02</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Cancer remains a major global health challenge, with high prevalence and mortality rates emphasizing the urgent need for innovative treatment strategies. Although precision oncology offers tailored therapies based on genetic profiles, the clinical translation of genomic insights has been slow. Drug repurposing, using existing FDA‐approved drugs for new indications, presents a cost‐effective and time‐efficient alternative. This study investigates MGAM as a potential direct target of alpha‐glucosidase inhibitors in colorectal cancer (CRC), explores its biomarker potential, and evaluates gene expression patterns across diverse cancers. Using RNA‐Seq data from Recount3, Firebrowse, and gene set co‐expression analysis databases, we analyzed the differential expression of MGAM and its paralog MGAM2 across 33 cancer types. We examined mutation profiles, methylation status, survival impact, immune cell infiltration, and drug‐mRNA interactions. Validation was performed via real‐time PCR and whole‐exome sequencing (WES) in CRC patients. MGAM and MGAM2 showed differential expression across multiple cancers, with MGAM2 upregulated and MGAM downregulated in gastrointestinal tumors. Both genes were linked to key cancer‐related pathways, including metabolism, apoptosis, cell cycle regulation, and epithelial‐mesenchymal transition. MGAM exhibited frequent mutations and aberrant methylation in several cancers. Their expression correlated with immune cell infiltration and drug sensitivity, highlighting potential for therapy planning. Diagnostic modeling showed over 80% accuracy. In CRC patients, MGAM downregulation was confirmed in 64 samples, and WES revealed a novel MGAM mutation (rs2960746). These findings underscore MGAM and MGAM2 as promising biomarkers and therapeutic targets, supporting their relevance in advancing personalized oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0138c7685899dec8663add8d3359263dd4a2f1f8" target='_blank'>
              Pan‐cancer analysis and oncogenic implications of MGAM and MGAM2: Toward precision oncology and drug repurposing in colorectal cancer
              </a>
            </td>
          <td>
            Rawaa Chasib Mezher, H. Azari, Reza Khayami, Hamid Fiuji, Farzaneh Alizadeh, L. Goshayeshi, Majid Rajabian, Ibrahim Saeed Gataa, Seyed Mahdi Hassanian, Amir Avan
          </td>
          <td>2025-08-27</td>
          <td>Journal of Cell Communication and Signaling</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee02ab2976f9bde7aae32f5b73d008d1d42a390d" target='_blank'>
              Refining treatment strategies for non-small cell lung cancer lacking actionable mutations: insights from multi-omics studies.
              </a>
            </td>
          <td>
            Andrea Rocca, Lucio Crinò, Luca Braga, F. Salton, B. Ruaro, M. Confalonieri, D. Generali, P. Confalonieri
          </td>
          <td>2025-08-23</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83bb8c6ba0429e042acd5b6b051c6318cb9fb300" target='_blank'>
              Identification, functional insights and therapeutic targeting of EMT tumour states.
              </a>
            </td>
          <td>
            Anqi Dong, Cédric Blanpain
          </td>
          <td>2025-09-25</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2ab9d9615fda47177afd65cb9d942ab1bd6dc87" target='_blank'>
              Thymic epithelial cells amplify epigenetic noise to promote immune tolerance.
              </a>
            </td>
          <td>
            Noah Gamble, Jason A Caldwell, Joshua McKeever, Caroline Kaiser, Alexandra Bradu, Peyton J. Dooley, Sandy Klemm, William J. Greenleaf, Narutoshi Hibino, Aaron R. Dinner, Andrew S. Koh
          </td>
          <td>2025-08-20</td>
          <td>Nature</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Located on chromosome 10, the RET gene encodes a transmembrane receptor tyrosine kinase that, when activated by a ligand, is essential for cell differentiation, proliferation, and survival. RET specifically supports the development of neuronal tissues and neural crest-derived cell types. However, RET is a proto-oncogene and mutations can lead to aggressive cancers like thyroid and lung tumours, or endocrine syndromes. Treatments for RET-driven cancers remain limited, relying on therapies that use RET-specific inhibitors. However, they show limited success in clinical settings due to adverse side effects and acquired resistance over time. To find alternative treatments, integrating the principles of synthetic dosage lethality (a genetic interaction causing a lethal effect in cancer cells) with machine learning may identify novel gene targets to complement existing therapies. In previous studies, we sought candidate genes that induce cell death when depleted in cells overexpressing oncogenic RET, but not in normal cells. A previously identified candidate, NCAM1, which is involved in neuronal cell migration, has been found to contribute to the differentiation of neural cell types. Using the SH-SY5Y neuroblastoma cell line as a model, multiple techniques and assays were performed to validate the synthetic lethal relationship between NCAM1 and RET. NCAM1 knockdown cell lines were generated using lentiviral transduction, and a live/dead cell assay was used to assess potential cytotoxic effects of gene knockdown. Additionally, a cycloheximide assay was performed to examine RET half-life by tracking its degradation over twelve hours. Over the summer, the project enabled us to reassess gene priority rankings using synthetic dosage lethal logic, characterize phenotypes of NCAM1 knockdown SH-SY5Y cells, optimize various assays under in-vitro conditions, and consolidate our understanding of NCAM1’s relationship to RET. The project thus demonstrates the value of combining bioinformatic and in-lab techniques as a productive approach to finding precise therapeutic targets against RET-driven cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08538d4a42ece74adec0afb573a8e0619996891d" target='_blank'>
              Identifying Novel Gene Targets in RET-Driven Cancers Using Synthetic Dosage Lethality Models
              </a>
            </td>
          <td>
            Zahra Simpson-Stairs
          </td>
          <td>2025-08-29</td>
          <td>Inquiry@Queen's Undergraduate Research Conference Proceedings</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="ABSTRACT The human Killer cell Immunoglobulin‐like Receptor (KIR) genes, found on chromosome 19, encode for cell surface protein receptors that, through interaction with their ligand, modulate the action of Natural Killer (NK) cells and some subsets of T lymphocytes. KIR genes exhibit extensive variation through variable gene content, copy number, and allele polymorphism. The combination of KIR genes and their ligands is implicated in various clinical settings including haematopoietic stem cell and solid organ transplant, and infectious disease progression. KIR gene content has been used in the selection of optimal stem cell donors with haplotype variations in recipient and donor giving differential clinical outcomes. With the introduction of massively parallel clonal next generation sequencing and single molecule long read third generation sequencing, allele level determination of KIR genotypes has become feasible. We describe a method for amplicon‐based long read sequencing on the Oxford Nanopore Technologies platform that provides largely unambiguous allele level typing of KIR genes. The method was validated using DNA extracted from 48 10th International Histocompatibility Workshop (IHWS) cell lines with previously published allele level KIR genotypes and 176 Western Australian samples previously tested for the presence or absence of KIR genes. Our long‐read sequencing method was able to accurately determine KIR alleles with an overall concordance of 97%–99% with the published data. Importantly, phasing ambiguity caused by the inability to phase heterozygous base positions over long stretches of gene sequence was resolved in several samples. Thus, our long read PCR sequencing strategy can be used to determine KIR genotypes at allele resolution level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f07dc264b92eef3e3ce6dcccdb40f00d68f9fed" target='_blank'>
              Allele Level Sequencing of Killer Cell Immunoglobulin‐Like Receptor Genes Using Oxford Nanopore Long Read Sequencing
              </a>
            </td>
          <td>
            Jonathan Downing, Daniel Abromeit, L. Truong, F. Mobegi, Patricia Martinez, Lloyd J D'Orsogna, D. de Santis
          </td>
          <td>2025-09-01</td>
          <td>Hla</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Deoxyribonucleic acid (DNA) serves as the fundamental blueprint for life, governing both the structural development and functional regulation of living organisms. The technology explores the modern understanding of DNA role in heredity and molecular genetics that highlighted key discoveries that have shaped the field. The discovery of intricate molecular mechanisms such as epigenetic modifications, non-coding RNA and the gene environment interactions has evolved beyond Mendelian genetics. The mechanism of genetic inheritance, replication and expression illustrates how traits are passed from one generation to the next generation. This review article highlights how mutation and variations in DNA sequences influence health, evolution and disease susceptibility. A significant focus is placed on revolutionary CRISPR-Cas9 Jain editing technology which has transformed genetic research and holds immense promise for future therapeutic applications. After enabling precise modifications in the genome, CRISPR opens new avenues for the treatment of genetic disorders and advancement in biotechnology. This review emphasizes how the integration of classical molecular genetics with cutting edge gene editing techniques offers a comprehensive understanding of DNA function for addressing agricultural and global health challenges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd606e61309a0083af92e77dc5cd4dc92727de21" target='_blank'>
              MODERN INSIGHTS OF DNA FUNCTION AND HEREDITY: A REVIEW OF MOLECULAR GENETICS AND CRISPR GENE
              </a>
            </td>
          <td>
            Mariam Fatima, Mariam Fatima¹, Muhammad Haseeb, Anwar Tarar¹, Arooba Maryam², Muhammad Azhar, Ud Din³˒⁴, Ifra Muzaffar¹, H.M. Saif, UR Rahman⁵, Rida Sattar⁶
          </td>
          <td>2025-08-21</td>
          <td>Journal of Medical &amp; Health Sciences Review</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Cancer is still a huge problem because of the complication of genetic heterogeneity and emergence of drug resistance which leads to poor therapeutic responses. Conventional drug development is usually restricted in their capacity to discover the exact molecular target and anticipate the drug responses. This is because the integration of CRISPR-based genome editing technologies into cancer drug screening does two main services: it offers a new direction of discovering gene-drug interactions and targeting against drugs. The study has used A549, MCF-7, and HCT116 cancer cell lines to examine how the consequences of knocking out and activating certain genes (KRAS, TP53, MYC, PTEN, and EGFR) affect a cell in terms of response to chemotherapeutic agents ( d o x rubicin, paclitaxel and c is platinum ). The design of the experiment was the inclusion of drug treatments at different concentrations, cell viability assays, measurement of gene expression and quantifying the protein. The statistical tests, such as ANOVA and post-hoc testing, were used in order to determine the statistical significance of the changes. The data showed that KRAS knockout sensitized the A549 cells to doxorubicin (IC50 = 0.35 10 9 ) and TP53 knockout rendered the cells resistant (IC50 = 1.50 10 9 ). MYC CRISPR interference decreased sensitivity to paclitaxel whereas MYC activation increased it. PTEN knockout (in MCF- 7) rendered cells more susceptible to cisplatin and activation induced resistance. Pearson correlation analysis demonstrated that there were significant correlations between the change in expression of the genes and cell viability indicating the functional relevance of the CRISPR alterations. The presented study also notes that CRISPR-based functional screening can be a promising field of precision oncology and can help to understand the molecular underpinning of drug resistance and discover new targets of therapeutics against cancer">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6621891a1e50692e473d10afd771d57415d90fde" target='_blank'>
              CRISPR-BASED GENOME EDITING TECHNOLOGIES IN CANCER RESEARCH AND THEIR APPLICATIONS IN FUNCTIONAL DRUG SCREENING
              </a>
            </td>
          <td>
            Aiman Nishat, Ahmad Ashraf, Tahir Hasan, Abrar Hussain, Muhammad Afaq, Haseeb Khaliq, Muhammad Kamran Tahir, Abul Hassan Khan, Gul Sika
          </td>
          <td>2025-08-14</td>
          <td>Journal of Medical &amp; Health Sciences Review</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Chronic myeloid leukemia (CML) is a hematologic malignancy originating from hematopoietic stem cells and driven by the BCR-ABL fusion oncogene. Imatinib (IM), a tyrosine kinase inhibitor, is commonly used as a frontline therapy for CML. However, some patients exhibit primary resistance or show persistent molecular evidence of disease despite treatment. Emerging studies indicate that exosome-derived microRNAs (miRNAs) play a role in mediating drug resistance and may serve as promising biomarkers for cancer diagnosis and predicting therapeutic response. This study aimed to investigate the plasma exosomal miRNA expression profiles in CML patients to identify potential biomarkers associated with IM resistance. Exosomes were isolated from plasma samples of both IM-sensitive and IM-resistant CML patients. The exosomal miRNA content was analyzed using RNA sequencing, followed by differential expression analysis, which revealed 13 upregulated and 21 downregulated miRNAs in IM-resistant patients. Subsequent bioinformatics analysis indicated significant enrichment in pathways related to autophagy and PI3K-Akt signaling. Notably, miR-451a and miR-16–2-3p were among the most significantly upregulated miRNAs in exosomes from IM-resistant individuals. Interestingly, miR-16–2-3p expression showed a strong inverse correlation with clinical laboratory results, specifically blood urea nitrogen and creatinine levels. This pilot study identified plasma exosomal miRNAs, particularly miR-451a and miR-16–2-3p, as potential biomarkers for imatinib resistance in chronic myeloid leukemia. Target gene prediction was performed to explore their regulatory roles. Despite the limited sample size, these findings enhance our understanding of drug resistance mechanisms and warrant further validation in larger cohorts to assess their clinical relevance and therapeutic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ffa66d21071c796372f54b926f9da678ac08d63" target='_blank'>
              Exosomal miRNA expression profiling in patients with imatinib resistant Chronic myeloid leukemia: A pilot study
              </a>
            </td>
          <td>
            R. Navakanitworakul, Pirun Saelue, Tipparat Penglong, P. Molika, Natakorn Nokchan, Natta Tansila, H. Buncherd, Supinya Thanapongpichat, Kanitta Srinoun
          </td>
          <td>2025-08-29</td>
          <td>PLOS One</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="Understanding how genes influence drug responses is critical for advancing personalized cancer treatments. However, identifying these gene-drug interactions in a physiologically relevant human system remains a challenge, as it requires a model that reflects the complexity and heterogeneity among individuals. Here we show that large-scale CRISPR-based genetic screens, including knockout, interference (CRISPRi), activation (CRISPRa), and single-cell approaches, can be applied in primary human 3D gastric organoids to systematically identify genes that affect sensitivity to cisplatin. Our screens uncover genes that modulate cisplatin response. By combining CRISPR perturbations with single-cell transcriptomics, we resolve how genetic alterations interact with cisplatin at the level of individual cells and uncover an unexpected link between fucosylation and cisplatin sensitivity. We identify TAF6L as a regulator of cell recovery from cisplatin-induced cytotoxicity. These results highlight the utility of human organoid models for dissecting gene-drug interactions and offer insights into therapeutic vulnerabilities in gastric cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/306fe56d4d5b9a4de12be73a14dcdbee9c2f3137" target='_blank'>
              Large-scale CRISPR screening in primary human 3D gastric organoids enables comprehensive dissection of gene-drug interactions
              </a>
            </td>
          <td>
            Yuan-Hung Lo, Hudson T Horn, Mo-Fan Huang, Wei-Chieh Yu, Chia-Mei Young, Qing Liu, Madeline Tomaske, Martina Towers, Antonia Dominguez, M. Bassik, Dung-Fang Lee, L. S. Qi, Jonathan S. Weissman, Jin Chen, Calvin J. Kuo
          </td>
          <td>2025-08-14</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>64</td>
        </tr>

        <tr id="Background Multiple myeloma (MM) is characterized by extensive intratumoral heterogeneity and complex interactions within the bone marrow microenvironment, yet the cellular and molecular drivers of treatment resistance remain poorly defined. Protein tyrosine phosphatase receptor gamma (PTPRG) has emerged as a candidate tumor suppressor in various malignancies by antagonizing proliferative and survival signaling, but its functional and prognostic relevance in MM has not been established. Methods We analyzed 103,171 single‐cell transcriptomes from 18 MM samples (10 optimal responders [OR] and 8 suboptimal responders [SOR] to bortezomib–melphalan–prednisone) to investigate cell‐type composition, malignant plasma cell subclusters, and tumor–microenvironment crosstalk. InferCNV was used to distinguish malignant plasma cells, which were further reclustered and correlated with bulk prognostic phenotypes. Differential expression, pathway enrichment, transcription‐factor activity, pseudotime trajectory, and ligand–receptor interaction analyses were performed. Finally, bulk datasets (GSE9782, GSE2658, MMRF-CoMMpass) and in vitro knockdown assays in U266 and NCI-H929 cells were used to validate the prognostic and functional role of PTPRG. Results Eleven major cell types were annotated, with plasma cells, T/NK cells, and CD14+ monocytes predominating; SOR samples exhibited an expanded plasma‐cell fraction and reduced T/NK, CD14+ monocyte, pre-B, and HSPC populations. Among 35,944 malignant plasma cells, five subclusters were defined; one subcluster (MalPlasma3) was enriched in SOR samples and harbored 93.1% of cells associated with poor survival. MalPlasma3 and “worse‐survival” cells showed activation of stemness, E2F/MYC targets, and G2M checkpoint pathways, driven by transcription factors E2F8, E2F7, FOXM1, E2F1, and TIMELESS. Pseudotime analysis revealed a bifurcating differentiation toward a resistant phenotype, accompanied by upregulation of cell‐cycle and proliferation modules. In the OR group, enhanced cytotoxic features in NK, effector, and naïve T cells, along with IGF1–IGF1R and IFNG–IFNGR signaling, suggested a supportive microenvironment. In contrast to the known role as a tumor suppressor in solid and hematologic cancers, our integrative analyses identified PTPRG among seven stemness‐related genes upregulated in MalPlasma3 and poor‐survival cells, which was echoed in the observed reduced cell viability and increased apoptosis in MM cell lines following siRNA‐mediated PTPRG knockdown. Conclusions This single‐cell multi‐omic dissection implicates a proliferative, stem‐like MalPlasma3 subcluster and identified PTPRG as a key mediator of drug resistance and poor outcome in MM, offering novel prognostic biomarkers and therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48b1aa46cace7effec1d51940ecccc2bdb09fa3e" target='_blank'>
              Single-cell profiling uncovers PTPRG-driven stemness in malignant plasma cells and signatures of treatment failure in multiple myeloma
              </a>
            </td>
          <td>
            Jiewen Tan, Jinman Zhong, Yueping He, Yunman Xu, Chang Chen, Dan Xiong
          </td>
          <td>2025-09-18</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Essential epigenetic dependencies have become evident in many cancers. Based on functional antagonism between BAF/SWI/SNF and PRC2 in SMARCB1-deficient sarcomas, we and colleagues recently completed the clinical trial of the EZH2 inhibitor tazemetostat, leading to its FDA approval. However, the principles of tumor response to epigenetic therapy in general, and tazemetostat in particular, remain unknown. First, using functional genomics of patient tumors and diverse experimental models, we define molecular mechanisms of tazemetostat resistance in SMARCB1-deficient sarcomas and rhabdoid tumors. We found distinct classes of acquired mutations that converge on the RB1/E2F axis and decouple EZH2-dependent differentiation and cell cycle control. This allows tumor cells to escape tazemetostat-induced G1 arrest despite EZH2 inhibition, and suggests a general mechanism for effective EZH2 therapy. We provide prospective biomarkers for therapy stratification, including the cell cycle-regulated PRICKLE1 gene. To circumvent tazemetostat resistance and improve response in vitro and in vivo we develop the rational combination of tazemetostat with barasertib, to target the downstream cell cycle kinase AURKB. Second, through transcriptomic inference of SMARCB1-deficient tumor cells, we predicted ATR as a target to overcome resistance to EZH2 inhibition. We found that EZH2 inhibition induces DNA damage, at least in part via upregulation of the transposase-derived PGBD5 nuclease. Finally, we establish a second rational combination strategy for increased anti-tumor efficacy between tazemetostat and the ATR inhibitor, elimusertib, by leveraging the synthetic lethal interaction with PGBD5-dependent DNA damage repair signaling via ATR. Thus, the epigenetic therapy, EZH2 inhibition, induces a targetable collateral vulnerability through activation of the PGBD5-ATR axis. This establishes a mechanistically rational combination strategy for targeting EZH2 and ATR in SMARCB1-deficient cancers based on an EZH2-PGBD5-ATR synthetic lethal axis and is immediately translatable to clinical trials for patients with these aggressive malignancies.



 Helen S Mueller. Overcoming clinical resistance to EZH2 inhibition using rational epigenetic combination therapy [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr A048.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e20005fa9b7768828704bae3a839702d20b1fe7d" target='_blank'>
              Abstract A048: Overcoming clinical resistance to EZH2 inhibition using rational epigenetic combination therapy
              </a>
            </td>
          <td>
            Helen S. Mueller
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a heterogeneous disease characterized by a broad spectrum of molecular alterations that influence clinical outcomes. TP53 mutations define one of the most lethal subtypes of acute myeloid leukemia (AML), driving resistance to nearly all available treatment modalities, including venetoclax plus azacitidine (VenAza). Yet, the molecular basis of this resistance, beyond affecting transactivation of BCL-2 family genes, has remained elusive. Here, we demonstrate that VenAza treatment leads to reduced transcriptional upregulation of the p53 signaling pathway in TP53 mutant/deficient AML compared to wild-type AML. Functionally, TP53 mutant/deficient AML exhibits selective failure in apoptosis induction rather than impaired G1 arrest or senescence. Despite inhibition of pro-apoptotic BAX and selective enrichment for MCL-1 in TP53 mutant isogenic AML cells, compensatory upregulation of BIM preserved functional mitochondrial outer membrane permeabilization (MOMP). TP53 mutant primary AML tumors at baseline also had retained capacity for MOMP. Instead, TP53 mutant AML exhibited disruption in caspase-3/7 activation to evade apoptosis after VenAza therapy – decoupling the mitochondrial and executioner phases of apoptosis. Importantly, this “post-MOMP brake” is not a bystander effect but itself a driver of VenAza and chemotherapy resistance in TP53 mutant/deficient AML. This previously unrecognized mechanistic insight shifts the focus from mitochondrial priming to terminal caspase blockade in TP53 mutant AML and opens the door for urgently needed therapeutic strategies that reignite apoptosis at its execution point.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56075976a6297045972be1b8a77a5e2796abdc68" target='_blank'>
              TP53 mutations drive therapy resistance via post-mitochondrial caspase blockade
              </a>
            </td>
          <td>
            Ahmed M Mamdouh, E. Olesinski, Fang Qi Lim, S. Jasdanwala, Yang Mi, Nicole Si, En Lin, Daniel Tan, En Liang, Nishtha Chitkara, L. Hogdal, R. C. Lindsley, A. Letai, Koji Itahana, Shruti Bhatt
          </td>
          <td>2025-08-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="


 Despite decades of progress in ovarian cancer treatment, therapeutic resistance remains a persistent and lethal challenge. Emerging evidence suggests polyploid giant cancer cells (PGCCs) as key drivers of this resistance. PGCCs are distinguished by multinucleation or a single oversized nucleus containing multiple chromosome sets. Polyploid cell populations are present in premalignant lesions and expand with disease progression and therapeutic stress. PGCCs demonstrate heightened resistance to treatment relative to non-PGCCs and can regenerate non-PGCCs via budding once therapeutic pressure is alleviated, leading to relapse. Supported by extensive preclinical and clinical evidence, we hypothesize that elimination of PGCCs may reduce therapeutic resistance in ovarian cancer. Given their significant role in resistance and relapse, PGCCs remain largely untargeted due to a lack of scalable detection methods. Standard viability assays such as MTT, XTT, and ATP-based screens fail to capture the rare but critical PGCC subpopulation. Current gold-standard methods, fluorescence-activated cell sorting (FACS) coupled with manual microscopy, while accurate, are time-intensive and unsuitable for high-throughput drug screening. Thus, a rapid and accurate technique for studying PGCCs is essential. In this pilot study, we have developed a high-throughput single-cell morphological analysis pipeline capable of precisely quantifying PGCC numbers and proportions. The pipeline accurately quantifies PGCCs based on nuclear and cellular features, including size and DNA content. Leveraging this platform, we screened a library of 2,726 FDA-approved compounds to identify agents that effectively kill PGCCs. Hits from this screen were validated in 3D spheroid cultures, revealing compounds with potent anti-PGCC activity. Complementary transcriptomic analysis (RNA-Seq) of flow-sorted PGCCs versus non-PGCCs unveiled profound dysregulation of the FOXM1 signaling axis, implicating this pathway as a key regulator of PGCC formation and survival. Consistent with this, the high-throughput screen identified Thiostrepton, a known FOXM1 inhibitor, as a selective anti-PGCC agent. Across multiple ovarian cancer cell lines, FOXM1 inhibitors outperformed standard-of-care agents (Paclitaxel, Cisplatin, Olaparib) in eliminating PGCCs. siRNA knockdown of FOXM1 significantly reduced PGCC numbers, particularly under Paclitaxel stress. Moreover, FOXM1 inhibition triggered apoptosis in PGCCs, an effect reversible with the pan-caspase inhibitor Z-VAD-FMK. Together, these findings underscore PGCCs as a therapeutically actionable cell state driving resistance in ovarian cancer. Our integrative single-cell morphological and transcriptomic platform enables rapid discovery of anti-PGCC agents and establishes FOXM1 inhibition as a compelling strategy to eradicate this resilient subpopulation. This pipeline offers broad utility for overcoming therapy resistance across diverse malignancies.



 Hsiao-Chun Chen, Yuan Zhang, Yushu Ma, Yu-Chih Chen. Combating ovarian cancer resistance via polyploid cancer cell targeted screening [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A023.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62f27ef6b0280aa103e9f9c96579bbfa985df28c" target='_blank'>
              Abstract A023: Combating ovarian cancer resistance via polyploid cancer cell targeted screening
              </a>
            </td>
          <td>
            Hsiao-Chun Chen, Yuan Zhang, Yushu Ma, Yu-Chih Chen
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="RNA-targeting techniques have emerged as powerful tools in cancer research and therapeutics, offering precise and programmable control over gene expression at the post-transcriptional level. Once viewed as passive intermediates in the central dogma, RNA molecules are now recognized as dynamic regulators of cellular function, capable of influencing transcription, translation, and epigenetic regulation. Advances in high-throughput sequencing technologies, transcriptomics, and structural RNA biology have uncovered a diverse landscape of coding and non-coding RNAs involved in oncogenesis, drug resistance, and tumor progression. In response, several RNA-targeting strategies have been developed to modulate these transcripts, including antisense oligonucleotides (ASOs), RNA interference (RNAi), CRISPR-Cas13 systems, small molecules, and aptamers. This review provides a comparative analysis of these technologies, highlighting their molecular mechanisms, therapeutic potential, and current limitations. Emphasis is placed on the translational progress of RNA-targeting agents, including recent FDA approvals and ongoing clinical trials for cancer indications. Through a critical comparison of these strategies, this review underscores the growing significance of RNA-targeting technologies as a foundation for next-generation cancer therapeutics and precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/beb6347a27c8706b96bee6d80e4ac4c34aab454f" target='_blank'>
              RNA-Targeting Techniques: A Comparative Analysis of Modern Approaches for RNA Manipulation in Cancer Research and Therapeutics
              </a>
            </td>
          <td>
            Michaela A. Boti, Marios A. Diamantopoulos, Andreas Scorilas
          </td>
          <td>2025-10-02</td>
          <td>Genes</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cisplatin resistance presents a significant challenge in cancer therapy, emphasizing the necessity for identifying new regulatory elements that influence drug response. Recent research has revealed the importance of long noncoding RNAs (lncRNAs) in chemotherapy resistance, with LINC02381 identified as a potential regulatory factor. 
Through an in-depth bioinformatics analysis, we investigated the impact of LINC02381 on cisplatin resistance in ovarian cancer across various datasets. By conducting differential expression analysis, survival analysis, gene set enrichment analysis (GSEA), and constructing protein-protein interaction (PPI) networks, we identified key pathways associated with LINC02381 expression. 
The results indicated that the altered expression of LINC02381 in patients treated with cisplatin was associated with reduced survival. Functional studies and correlation analyses further demonstrated that this LncRNA influences critical pathways and genes related to apoptosis, efflux, DNA repair, and EMT. Lastly, through an examination of its interactions with microRNA and protein networks, we identified LINC02381 as a ceRNA implicated in cisplatin resistance. 
Our findings suggest that LINC02381 may influence cisplatin sensitivity in ovarian cancer and establish a basis for further experimental validation, including molecular assays or in vivo analyses, and suggest the potential therapeutic targeting of LINC02381 to combat chemoresistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35a8ae366b9c19996905f6b9e1ed9f603948832f" target='_blank'>
              The Role of LINC02381 in Modulating Cisplatin Resistance in Ovarian Cancer: A Bioinformatics Approach
              </a>
            </td>
          <td>
            Zeinab Karbalaei Pazoki, B. M. Soltani, Mostafa Hosseini, Shiva Irani
          </td>
          <td>2025-09-26</td>
          <td>Iranian Journal of Allergy, Asthma and Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Simple Summary DNA repair mechanisms are crucial for maintanence of DNA and cell integrity. When disrupted, they can lead to increased mutation rate and contribute to the cancerogenesis. On the other hand, cancer cells that bear mutations in genes involved in DNA repair are more prone to DNA damaging insults which opens opportunities for treatments involving synthetic lethality. microRNAs are short non-coding RNAs that regulate expression by binding to gene transcripts and inducing mRNA degradation or inhibition of translation. Here, we review the miRNA-mediated dysregulation of genes involved in DNA damage response (DDR) and DNA repair pathways in ovarian cancer (OvCa), one of the deadliest female malignancy. We also discuss miRNAs, which affect response to OvCa therapy by regulating PARP1 (Poly(ADP-Ribose) Polymerase-1), a central gene of the OvCa synthetic lethality treatments. Finally, we address the limitations of miRNAs as diagnostic biomarkers and potential targets for therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bd4db6c3d00cf07b80c8f608dd0b112d9dfce41" target='_blank'>
              DNA Damage and Repair in Ovarian Cancer: Focus on MicroRNAs
              </a>
            </td>
          <td>
            Katarzyna D. Arczewska, A. Piekiełko-Witkowska
          </td>
          <td>2025-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is characterized by rapid progression, therapeutic resistance, and a profoundly immunosuppressive tumor microenvironment. Emerging evidence suggests that endoplasmic reticulum (ER)-associated macromolecules play critical roles in tumor adaptation. In this study, we performed a multi-omics investigation of orosomucoid-like protein 2 (ORMDL2), a conserved ER membrane protein involved in sphingolipid biosynthesis and ER stress regulation, and uncovered its regulatory functions in GBM progression. Transcriptomic analyses across The Cancer Genome Atlas (TCGA), and Chinese Glioma Genome Atlas (CGGA) revealed elevated ORMDL2 expression in GBM tissues which causes poor prognosis. The MetaCore pathway and Gene Set Enrichment Analysis (GSEA) identified ORMDL2's involvement in antigen presentation via a major histocompatibility complex I (MHC class I), unfolded protein response (UPR), and mitochondrial apoptotic signaling. Single-cell RNA-sequencing data and the Human Protein Atlas showed ORMDL2 enrichment in tumor stromal cells. Pharmacogenomic correlation via the Genomics in Drug Sensitivity in Cancer (GDSC) and Cancer Therapeutics Response Portal (CTRP) database suggested that ORMDL2 expression was associated with resistance to DNA damage response inhibitors such as etoposide, doxorubicin, talazoparib, and might interact with sphingolipid-targeting compounds. Collectively, our findings establish ORMDL2 as a multi-functional macromolecular regulator of immune suppression and therapeutic resistance in GBM, providing new mechanistic insights and potential targets for translational medicines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f5a5f2b3cb520c8099ca5e5eaf8e443e9c0dbc4" target='_blank'>
              Targeting ORMDL2 in glioblastoma through integrated landscape of high-throughput sequencing and pharmacogenomic analysis
              </a>
            </td>
          <td>
            Fitria Sari Wulandari, , Sachin Kumar, Ngoc Phung Ly, Juan Lorell Ngadio, Dahlak Daniel Solomon, Do Thi Minh Xuan, Gabriela Natasha Sutandi, , , Yung-Kuo Lee, , , Ching-Chung Ko
          </td>
          <td>2025-09-27</td>
          <td>International Journal of Medical Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Transposable elements (TEs), once considered genomic “junk”, are now recognized as critical regulators of genome function and human disease. These mobile genetic elements—including retrotransposons (long interspersed nuclear elements [LINE-1], Alu, short interspersed nuclear element-variable numbers of tandem repeats-Alu [SVA], and human endogenous retrovirus [HERV]) and DNA transposons—are tightly regulated by multilayered mechanisms that operate from transcription through to genomic integration. Although typically silenced in somatic cells, TEs are transiently activated during key developmental stages—such as zygotic genome activation and cell fate determination—where they influence chromatin architecture, transcriptional networks, RNA processing, and innate immune responses. Dysregulation of TEs, however, can lead to genomic instability, chronic inflammation, and various pathologies, including cancer, neurodegeneration, and aging. Paradoxically, their reactivation also presents new opportunities for clinical applications, particularly as diagnostic biomarkers and therapeutic targets. Understanding the dual role of TEs—and balancing their contributions to normal development and disease—is essential for advancing novel therapies and precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f9a4403306942197c6791291eb7e939f11dbad7" target='_blank'>
              Transposable elements in health and disease: Molecular basis and clinical implications
              </a>
            </td>
          <td>
            Yaqiang Hong, Nian Liu
          </td>
          <td>2025-08-15</td>
          <td>Chinese Medical Journal</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="BACKGROUND
Emerging RAS inhibitors show promise in treating KRAS-mutated malignancies, but resistance mechanisms limit their clinical efficacy. Given recent clinical findings associating KRAS mutations with reduced response to neoadjuvant therapy in rectal cancer (RC), we aimed to investigate their impact on treatment outcomes and explore potential therapeutic strategies.


METHODS
We conducted a retrospective analysis of 390 rectal cancer patients to evaluate the association of KRAS mutations with disease-free survival (DFS) and response to therapy. We assessed the efficacy of KRAS inhibitors in rectal cancer cell lines, patient-derived organoids (PDOs), and patient-derived cell lines (PDCLs), and explored adaptive resistance mechanisms through transcriptomic profiling and unbiased drug screening experiments.


RESULTS
Mutant KRAS was associated with a reduced DFS and RCs harboring G12C and G12V mutations had less complete pathological responses to neo-adjuvant therapies. KRAS-mutated RC cells demonstrated adaptive resistance to KRAS inhibitors, characterized by transcriptomic restoration of oncogenic pathways, including MYC and E2F, and upregulation of ERBB2/3 expression. Consistently, drug screening identified EGFR family inhibitors as potent combinatorial partners, effectively overcoming KRAS inhibitor tolerance by inducing apoptosis. In patient-derived models, the pan-RAS inhibitor RMC-6236 combined with EGFR inhibitors demonstrated significant synergistic effects and prevented long-term tumor cell outgrowth.


CONCLUSION
Our findings point to the negative impact of KRAS mutations, particularly G12C and G12V, on RC treatment outcomes. Adaptive resistance by upregulation of ERBB genes limits the efficacy of KRAS inhibitors. Combining these with pan-ERBB inhibitors enhances anti-tumor effects in patient-derived cellular RC models, showing its potential as an alternative to the combination with anti-EGFR antibodies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9be2135f6b5cf4f9e01a3b8ac31ac51a76a3401" target='_blank'>
              Pan-ERBB Inhibitors Synergize With KRAS Inhibitors in Rectal Cancer.
              </a>
            </td>
          <td>
            Jonas Buchloh, M. Spitzner, Hauke Zimmermann, Xin Fang, Constanza Tapia Contreras, Carolin Schneider, Tiago de Oliveria, Stefan Küffer, Michael Linnebacher, Felix Rühlmann, Lena-Christin Conradi, Matthias Wirth, Michael Ghadimi, Marian Grade, Jochen Gaedcke, Günter Schneider
          </td>
          <td>2025-09-16</td>
          <td>United European gastroenterology journal</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Patients with estrogen receptor-positive (ER+) breast carcinoma often receive tamoxifen as an anti-estrogen therapy; however, a significant number still experience disease recurrence during treatment. The molecular mechanisms behind tamoxifen resistance are not yet fully understood. TAM mainly works by inhibiting estrogen receptor signaling pathways, inducing apoptosis, and causing growth arrest. Factors that influence TAM response include impaired bioactivation of tamoxifen and changes in the expression of tumor suppressor genes and proto-oncogenes, which are essential for regulating the cell cycle. Additionally, host factors, such as the processes involved in TAM bioactivation and drug transporters, play a crucial role in the therapeutic response. Understanding the interplay between tumor characteristics and host factors could provide valuable insights into TAM efficacy. Variations in gene expression associated with tamoxifen resistance may result from a combination of genetic mutations and epigenetic modifications. Clarifying the precise mechanisms of TAM response is crucial for improving treatment efficacy. Several therapeutic strategies have been explored to enhance TAM response, including drug replacement, epigenetic therapy, natural compounds, and innovative approaches like Antisense Oligonucleotides (ASOs). Combining these treatment modalities may improve patient survival more effectively than using tamoxifen alone. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b84dc4cafbad584398c4bc7b40f7ceccbd206990" target='_blank'>
              Genetic and epigenetic contribution to tamoxifen resistance in breast cancer, concepts to drive therapeutic strategies
              </a>
            </td>
          <td>
            R. Jahangiri
          </td>
          <td>2025-09-30</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Multidrug resistance is a significant barrier in cancer therapy largely due to poorly understood regulatory mechanisms. Here we reveal that certain anticancer drugs can bind to newly synthesized proteins prior to reaching their canonical targets, resulting in various forms of protein damage. This binding disrupts protein functions, particularly those of mitochondrial proteins, resulting in substantial cytotoxicity. The protein damage is further exacerbated by mitochondrial reactive oxygen species generated as a consequence of the initial damage, creating a positive feedback loop. In response, cancer cells rapidly initiate a chain of events, which we term the Protein Damage Response (PDR). This includes damage recognition primarily mediated by protein ubiquitination and subsequent damage clearance via the proteasome system. Notably, patients with advanced, drug-resistant metastatic breast or colon cancers exhibit elevated proteasome activity. In an effort to predict drug resistance, we developed a sensitive kit for detecting proteasome levels, enabling the identification and subtyping of patients with high proteasome activity to support tailored therapeutic strategies. Using a three-dimensional tumor slice culture-based drug sensitivity assay and an investigator-initiated clinical trial, we demonstrate that three clinically approved proteasome inhibitors effectively overcome multidrug resistance in colon and breast cancer patients with elevated proteasome activity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1785188ad7f7984f485f66a5779a54c95bdf57ed" target='_blank'>
              Suppressing protein damage response to overcome multidrug resistance in cancer therapy
              </a>
            </td>
          <td>
            Fangyuan Shao, Zongjie Li, Hao Xiao, Yujun Chen, Yuheng Zhang, Ling Li, Yuzhong Peng, Xinyi Li, Yuxing Hou, Bo Li, Xiangpeng Chu, Maoxin Ran, Dongyang Tang, Xi Han, Jiaxin Yao, Cuiting Zhang, Lijian Wang, Haifeng Li, N. Shao, Kai Miao, Xiaoling Xu, Yanxia Shi, Changhua Zhang, Jun Yan, Ying Lin, Chu-Xia Deng
          </td>
          <td>2025-09-30</td>
          <td>Cell Discovery</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is one of the most lethal malignancies and is projected to become the second leading cause of cancer-related death within the next decade. Population-scale genomic analyses of PDAC tumors have revealed extensive chromosomal instability (CIN), indicating high levels of genomic diversification that may influence therapy response. However, the mechanisms driving CIN during PDAC progression and its role in treatment response remain poorly understood. We hypothesize that specific patterns of genomic rearrangements emerging during PDAC evolution can inform two key aspects of tumor biology: (i) the underlying processes driving CIN, which may be therapeutically targetable, and (ii) subclonal CIN signatures that predispose tumors to develop chemoresistance.



 To investigate the dynamics of CIN and chemoresistance, we employed an in-house single-cell clonal outgrowth assay integrated with single-cell whole-genome sequencing (sc-WGS) and mathematical modelling of clonal growth. Experiments were performed in PDX-derived primary cultures and patient-derived 3D organoids. We tracked clonal outgrowth under both untreated conditions and following exposure to chemotherapeutic agents relevant to PDAC treatment. Comparative analyses of single-cell copy number profiles between parental populations and drug-exposed clones allow us to identify any de novo genomic alterations acquired during treatment, as well as pre-existing events enriched by drug selection.



 We observed ongoing CIN in PDAC cells, leading to continuous generation of new CNAs and enhanced cell-to-cell genomic diversity. Our data demonstrate pronounced heterogeneity in copy number states and growth dynamics among individual clones from the same cell line. This intrinsic heterogeneity provides a reservoir for evolutionary selection and may promote the emergence of drug-resistant phenotypes. We were able to analyse which clones carried pre-existing alterations and/or which generated new ones under the pressure of chemotherapy, allowing us to understand the diversity of responses and general principles of CNA changes in response to therapy. These findings underscore the innate adaptability of PDAC cells under therapeutic pressure.



 Our study highlights the dynamic nature of CIN in PDAC and its critical role in promoting chemoresistance. The identification of specific genomic alterations enriched in resistant subclones could offer potential biomarkers predictive of therapy response. This approach provides new insights into the interplay between CIN, cell-to-cell heterogeneity, and therapeutic adaptation, with potential future implications for guiding personalized treatment strategies in PDAC.



 Audrey Lumeau, Molly Anne. Guscott, Nathaniel Mon Pere, Isabel Nichols, Benjamin Werner, Nelson Dusetti, Sarah E. McClelland. Tracking genome evolution and chemoresistance in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B129.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b332fbacb3e055d135e8c8f4dfd0dbc1124774d" target='_blank'>
              Abstract B129: Tracking genome evolution and chemoresistance in pancreatic cancer
              </a>
            </td>
          <td>
            A. Lumeau, Molly A Guscott, N. M. Père, Isabel Nichols, Benjamin Werner, Nelson Dusetti, Sarah E. McClelland
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="R-loop is a special DNA–RNA hybrid genomic structure. Since its identification, the molecular mechanisms of physiological processes such as class switch recombination have been revealed, uncontrolled regulation of which has become the underlying cause of many diseases. With the development of molecular biology techniques, R-loops found at different sites and formed via different mechanisms have been discovered. These non-classical R-loops participate in various cellular activities via different mechanisms. A set of sophisticated regulatory mechanisms are required to control the number of various R-loops in vivo. The levels of non-classical R-loops in tumor cells differ significantly from those in normal cells; in addition, the regulatory mechanisms for establishing R-loop homeostasis differ, which may be utilized to develop breakthrough therapies for various tumors. In this review, we summarize the current state of knowledge regarding non-classical R-loops in tumor cells, the mechanisms that promote or inhibit this structure, the effects of this structure on tumor cells, and the possible therapeutic targets. By systematically elucidating the pathogenic mechanisms of atypical R-loops, we can achieve precision targeting of tumors and revolutionize clinical precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/676d0195748900abca38448fb2ef5985171fe4bf" target='_blank'>
              Atypical R-loops in cancer: decoding molecular chaos for therapeutic gain
              </a>
            </td>
          <td>
            Yuan Sun, Sheng Wang, N. Ge, Jintao Guo, Guoxin Wang, Fan Yang, Siyu Sun
          </td>
          <td>2025-08-14</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/861fd3ffe0c8c1bb6ed63c62ce2660ccaa3e238b" target='_blank'>
              miRNA-mediated resistance mechanisms in prostate cancer: implications for targeted therapy and metastatic progression.
              </a>
            </td>
          <td>
            Mostafa M. Mostafa, M. K. A. El-Aziz, D. Ellakwa
          </td>
          <td>2025-08-29</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4671f71130c02c103abea19511db04e587b6744b" target='_blank'>
              Evolutionary drivers of divergent collateral sensitivity responses during antibiotic therapy.
              </a>
            </td>
          <td>
            Sander K. Casier, Bram Lories, Hans P. Steenackers
          </td>
          <td>2025-08-22</td>
          <td>Nature ecology & evolution</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs), which replicate and segregate in a non-Mendelian manner, serve as vectors for accelerated tumor evolution. By integrating chromatin accessibility, whole-genome sequencing, and Hi-C-based genome topology data from a cohort of metastatic Castration-Resistant Prostate Cancer (mCRPC) cases, we show that epigenetically activated repeat DNA, amplified in ecDNAs, drive oncogene overexpression. Specifically, we identify a subgroup of mCRPCs (20%) characterized by clusters of accessible LINE1 repeat DNA elements flanking the androgen receptor (AR) gene. These LINE1 elements are co-amplified with AR and provide binding sites for prostate-lineage transcription factors, including AR, FOXA1 and HOXB13. Accessible LINE1 elements establish novel 3D chromatin interactions with the AR gene, forging a new regulatory plexus driving AR overexpression and confers resistance to androgen signaling inhibitors. Our findings indicate how tumor evolution is driven by the convergence of genetic and epigenetic alterations on repeat DNA, activating and amplifying them to allow oncogene overexpression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e6c5dce2ec927f5cff069554fa04e5223fd82a5" target='_blank'>
              Genetic and Epigenetic Reprogramming of Transposable Elements Drives ecDNA-Mediated Metastatic Prostate Cancer
              </a>
            </td>
          <td>
            Lisanne Mout, Thaidy Moreno-Rodriguez, Giacomo Grillo, Ankita Nand, Tina Keshavarzian, S. Bahl, Komaldeep Kang, M. Bootsma, Emma Minnee, Stanley Zhou, Kathleen H. Burns, E. Corey, Peter Nelson, S. Dehm, Shuang G Zhao, W. Zwart, Felix Y. Feng, David A Quigley, M. Lupien
          </td>
          <td>2025-08-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d108c58970b821d1c2170e04d8c4196841969e5f" target='_blank'>
              Phage therapy and its role in cancer treatment and control.
              </a>
            </td>
          <td>
            H. A. Abd El-Moaty, A. Doghish, Hebatallah Ahmed Mohamed Moustafa, Wagiha S. Elkalla, Ghadir A Sayed, Nourhan H Elshami, S. S. A. Mageed, Osama A Mohammed, Mahmoud A. Elrebehy, Kareem AlFarsi, Amr M. Abdelfatah, Aly M Waseem, Ahmed O Abdulkader, Rabab S. Hamad, Akram N Salah
          </td>
          <td>2025-09-17</td>
          <td>Folia microbiologica</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) has emerged as an advanced biological technology capable of resolving the complexity of cancer landscapes at single-cell resolution. Spatial transcriptomics(ST), as an innovative complementary approach, effectively compensates for the lack of spatial information inherent in scRNA-seq data. This review explores the rapidly evolving integration of scRNA-seq and ST and their transformative role in deciphering the tumor microenvironment (TME). We highlight how these technologies jointly uncover cellular heterogeneity, stromal-immune interactions, and spatial niches driving tumor progression and therapy resistance. Moving beyond previous reviews, we emphasize emerging computational strategies for data integration—including deconvolution and mapping approaches—and evaluate their applications in characterizing immune evasion, fibroblast diversity, and cell-cell communication networks. Ultimately, this review provides a forward-looking perspective on how spatial multi-omics are poised to advance precision oncology through spatially-informed biomarkers and diagnostic tools. We conclude that the full clinical potential of these technologies relies on closing the gap between analytical innovation and robust clinical implementation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84801b1d17cdb40a9473ba25f6bac709b5c4c887" target='_blank'>
              Single-cell and spatial transcriptomics integration: new frontiers in tumor microenvironment and cellular communication
              </a>
            </td>
          <td>
            Wenxin Shi, Zhiqiang Zhang, Xiaotong Xu, Yanpeng Tian, Li Feng, Xianghua Huang, Yanfang Du, Zhongkang Li
          </td>
          <td>2025-10-02</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The tumour suppressor protein p53 plays a central role in safeguarding genomic integrity through the regulation of DNA repair, cell cycle arrest, and apoptosis. Mutations in TP53, particularly within its DNA-binding domain, are among the most frequent genetic alterations in human cancers and are strongly associated with chemoresistance and poor prognosis. In this study, all TP53 mutations reported in the COSMIC database were systematically mapped onto all experimentally resolved TP53 three-dimensional structures available in the Protein Data Bank, supplemented with AlphaFold-predicted models to achieve full structural coverage. Mutations were classified according to their structural context—protein core, interface regions, ligand- and zinc-binding sites, and intrinsically disordered regions—and evaluated using complementary sequence- and structure-based predictive tools. The analysis revealed distinct mutational hotspots, differential distribution across structural regions, and context-dependent effects on stability and DNA-binding capacity. Notably, a subset of mutations exhibited consistent predictions of high destabilisation across all structural contexts, underscoring their potential as drivers of functional inactivation. By providing a comprehensive structural map of TP53 alterations, this work offers a valuable resource for understanding mutation-specific mechanisms of p53 dysfunction and for guiding the development of precision therapeutic strategies aimed at restoring its tumour-suppressive functions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85f3fc7b1e2b5e98df635979848fcd53d6ac30ab" target='_blank'>
              Mutational Disruption of TP53: A Structural Approach to Understanding Chemoresistance
              </a>
            </td>
          <td>
            Ali F. Alsulami
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1cae074f29f78d9d694a20a3e12fe166bc58ebc" target='_blank'>
              RAD51-mediated homologous recombination is a pro-tumour driver pathway.
              </a>
            </td>
          <td>
            Bernard S. Lopez
          </td>
          <td>2025-09-24</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Alternative splicing in HIV-1 has been a central focus of decades of research, uncovering key mechanisms of viral gene regulation, immune evasion, and therapeutic response – yet, no reference resource has existed to support transcriptome-wide analysis, limiting adoption of modern computational methods. We present HIV Atlas ( https://ccb.jhu.edu/HIV_Atlas ), the first reference-quality annotation of HIV-1 and SIV transcriptional diversity. We manually curated transcriptomes for HIV-1 HXB2 and SIV mac239 and developed Vira, an automated annotation-transfer method specifically designed to address unique challenges of viral genome biology, to generate high-quality annotations for 2,077 complete HIV-1 genomes. Using the resources presented in our work, we evaluated conservation of splice sites, revealing near-perfect preservation of major donors and acceptors. Furthermore, using several public datasets, we demonstrate how HIV Atlas enhances methodology, improves the quality and novelty of results, and opens novel avenues for research, supporting more accurate and comprehensive analyses of bulk, single-cell, and spatial RNA-seq in HIV-1 studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1270dba893a67448bd87a98a989e0077c4e63841" target='_blank'>
              Comprehensive Transcriptome Annotation of Thousands of HIV-1 Genomes
              </a>
            </td>
          <td>
            Ales Varabyou, Mykhaylo Artamonov, Sophia Cheng, Diane L. Bolton, Steven L Salzberg, M. Perțea
          </td>
          <td>2025-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Cancer treatment is an ongoing challenge, as directly targeting oncogenic drivers is often unfeasible in many patients due to the lack of druggable targets. This has led to the exploration of alternative strategies, such as exploiting synthetic lethality (SL) relationships between genes. SL facilitates the indirect targeting of oncogenic drivers, as exemplified by the clinical success of PARP inhibitors against BRCA-mutated tumors. Advances in high-throughput perturbation screens and multi-omics technologies have deepened our understanding of SL relationships, while computational models enhance SL predictions to better reflect biological complexity. However, while numerous experimental and computational methods have been developed to identify SL interactions, difficulties remain in translating these findings into clinical applications.This review combines recent progress on SL relationships in cancer with emerging insights into epigenetic regulation, highlighting how epigenetic drugs (epidrugs) can provide new opportunities for targeted interventions, offering a way to minimize off-target effects and enhance therapeutic precision. To advance SL-based therapies, efforts must focus not only on identifying new SL interactions but also on consolidating existing knowledge and integrating experimental and computational approaches to characterize the vulnerabilities of cancer cells. Strengthening this foundation will be critical for the effective development of SL-based cancer treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a9988cdf76b342d040b8e183dfc1c5b3f9d7baa" target='_blank'>
              Epigenetic synthetic lethality as a cancer therapeutic strategy: synergy of experimental and computational approaches.
              </a>
            </td>
          <td>
            Maria Farina-Morillas, Laia Ollé-Monràs, Silvana Ce Maas, Isabel de Rojas-P, Miguel F. Segura, Jose A. Seoane
          </td>
          <td>2025-08-25</td>
          <td>Epigenomics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Cell identity, tightly regulated by transcription factors (TFs), is frequently disrupted in cancer, driving tumor progression and metastasis. In pancreatic ductal adenocarcinoma (PDAC), alterations in cell identity contribute to disease heterogeneity and therapy resistance. Integrated genomic, transcriptomic, and proteomic analyses have defined two major PDAC subtypes: classical and basal. Notably, recent evidence suggests that changes in subtype identity can enable resistance to targeted therapies. These subtypes are distinguished by relative levels of endodermal lineage-specifying TFs such as FOXA1/2 and HNF4α, which are maintained in the classical subtype but downregulated in the basal subtype. Understanding how these TFs shape cellular identity is critical for identifying therapeutic vulnerabilities and mechanisms of drug resistance. We have shown that HNF4⍺ promotes the classical subtype program of PDAC. Although FOXA1/2 and HNF4α regulate gene expression in normal tissues, it is still unclear whether such cooperation is necessary to drive the classical transcriptional program in PDAC. We hypothesize that FOXA1/2 and HNF4α cooperatively regulates the gene expression program that regulates the classical subtype program in PDAC. Using human cell lines and murine PDAC organoid models, we reconstituted and modulated TF expression to evaluate downstream gene expression and TF genomic binding via ChIP-seq. Conditional alleles of FoxA1/2 in our murine organoid lines reveal that loss of FOXA1/2 lead to a decrease in the classical program. When we overexpress HNF4α in FOXA1/2-null organoids, it leads to a decrease in organoid growth; therefore, illustrating the ability of HNF4α to regulate growth independent of FOXA1/2. In a human cell line, we find that co-expression of FOXA1/2 and HNF4a activates the classical B signature while FoxA2 alone activates the Classical A signature, indicating that FOXA2 and HNF4a activate subtype-specific transcriptional programs. Additionally, we demonstrate that the overexpression of HNF4a in a subcutaneous xenograft model confers resistance to KRASG12C inhibition, leading to an increase in tumor size and weight. Unexpectedly, we find that these TFs do not cooperatively regulate gene expression; rather, in the human cell line, FOXA2 interferes with HNF4α genomic binding, suggesting a competitive or inhibitory interaction. Overall, our work highlights how lineage-defining TFs govern PDAC cell identity and therapeutic response.



 Walter A. Orellana, Sydney N. Larsen, Eric L. Snyder. FOXA1/2 and HNF4α Regulate Molecular Subtype and Therapeutic Response in Pancreatic Ductal Adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B107.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10be5668747dfa95fe8d23992fe6617771de029e" target='_blank'>
              Abstract B107: FOXA1/2 and HNF4α Regulate Molecular Subtype and Therapeutic Response in Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Walter A Orellana, Sydney N. Larsen, Eric L. Snyder
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c4704796a9cf8557ad095d79824927f1f6b68e0" target='_blank'>
              Revealing genetic drivers of ovarian cancer and chemoresistance: insights from whole-genome CRISPR-knockout library screens.
              </a>
            </td>
          <td>
            Tali S. Skipper, Kristie-Ann Dickson, Christopher E. Denes, Matthew A. Waller, Tian Y. Du, G. Neely, N. Bowden, Alen Faiz, Deborah J. Marsh
          </td>
          <td>2025-08-28</td>
          <td>Cellular oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Human endogenous retroviruses constitute a significant portion of the human genome and play complex roles in gene regulation and disease processes. However, the expression pattern and disease associations of specific retroviral loci remain pooly understood. This study examines the expression and regulatory mechanisms of these retroviral elements in immune cells. Utilizing single-cell RNA sequencing data from peripheral blood mononuclear cells, we identify 41,460 expressed retroviral loci, with 1936 showing cell type-specific expression. We further detect 3463 conditionally independent expression quantitative trait loci linked to retroviral elements, highlighting their potential role in mediating genetic variants and disease associations. Notably, these retroviral sequences associate significantly with autoimmune diseases, with specific loci demonstrating pleiotropic associations with disease-related genes. Our findings suggest that these elements are not merely genomic remnants but active participants in cellular regulation and disease progression. This work advances understanding of human-retroviral coevolution and highlights potential therapeutic targets in immune disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c11df69e317870e5c63c88d7648ee572723112ba" target='_blank'>
              Single-cell eQTL mapping of human endogenous retroviruses reveals cell type-specific genetic regulation in autoimmune diseases
              </a>
            </td>
          <td>
            Fan Zhu, Yi Liu, Jiehao Lei, Xinxing Li, Zexu Jiang, Xuan Dong, Ying Gu, Young Li
          </td>
          <td>2025-08-14</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Breast cancer remains the most frequently diagnosed malignancy among women worldwide, with significant morbidity and mortality rates despite advances in early detection and treatment. This review provides a comprehensive overview of recent developments in breast cancer research, spanning genetic and molecular insights, novel diagnostic techniques, and emerging therapeutic strategies. The advent of next-generation sequencing and multi-omics approaches has deepened our understanding of tumor heterogeneity, revealing key genetic drivers, epigenetic regulators, and the role of cancer stem cells in disease progression. Early detection strategies have also evolved with digital breast tomosynthesis and molecular breast imaging, offering improved sensitivity and specificity. On the therapeutic front, breakthroughs in targeted treatments—including cyclin-dependent kinase 4/6 and phosphoinositide 3-kinase inhibitors, antibody–drug conjugates, and immune checkpoint inhibitors—have transformed patient outcomes. The integration of chimeric antigen receptor T-cell therapy and mRNA-based therapeutics holds great promise in overcoming treatment resistance and improving long-term survival. However, challenges such as treatment accessibility, drug resistance, and disparities in healthcare persist, particularly in low- and middle-income regions. Emerging technologies, including artificial intelligence-driven diagnostics and risk-adapted screening, are paving the way for more precise and personalized interventions. This review highlights the latest innovations and ongoing challenges in breast cancer research, emphasizing the need for continued efforts to translate scientific advancements into clinical practice to improve patient outcomes globally.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32c3c761954bdae167d685c04a34eda5ce49f0c7" target='_blank'>
              Clinical advancements in breast cancer research: A comprehensive review
              </a>
            </td>
          <td>
            P. Puvvula, Ria Sahasra Puvvula
          </td>
          <td>2025-09-23</td>
          <td>Tumor Discovery</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="TP53 mutations drive oncogenesis and therapeutic resistance in myelodysplastic syndromes (MDSs) and acute myeloid leukemia (AML), impairing p53-regulated functions such as apoptosis, immune surveillance, and genomic stability, leading to immune evasion and metabolic reprogramming. The tumor microenvironment in TP53-mutated MDS and AML fosters leukemic progression through cytokine dysregulation, altered metabolism, and immune suppression. Current therapies, including chemotherapy and hypomethylating agents, offer limited efficacy, resulting in poor overall survival rates for these high-risk patients. However, novel therapeutic approaches provide promising avenues, including MDM2 inhibitors, p53-reactivating agents, pathway-targeted inhibitors (Hedgehog, Wnt, NF-κB), immune modulation (checkpoint inhibitors, CAR-T therapy), metabolic interventions (fatty acid metabolism, glycolysis), and gene-editing technologies (CRISPR/Cas9, base editing). This review explores the mechanisms of immune dysfunction in TP53-mutated MDS and AML while highlighting emerging therapeutic strategies, emphasizing the integration of targeted, metabolic, and immune-modulating therapies as a transformative approach to improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fd676eb4edce0f1928207f8f6cec1ec2c9dedb2" target='_blank'>
              TP53-mutated MDS and AML: immune dysregulation, tumor microenvironment, and emerging therapeutic strategies
              </a>
            </td>
          <td>
            Marwah Albakri
          </td>
          <td>2025-08-20</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 High-grade serous ovarian cancer (HGSOC) remains the most lethal gynecologic malignancy, with a 5-year survival rate below 50%. Despite initial responsiveness to platinum-based chemotherapy and the implementation of targeted therapies such as PARP inhibitors, the majority of patients relapse and develop treatment-resistant disease. Emerging evidence suggests that the spatial organization and dynamic interactions among tumor, immune, and stromal cells within the tumor microenvironment (TME) play a critical role in therapeutic response and disease progression. However, traditional transcriptomic approaches lack the spatial context necessary to fully understand these interactions. The Xenium spatial transcriptomics (ST) platform represents an innovative technology that enables high-resolution, in situ mapping of gene expression within intact tissue architecture. By preserving the spatial relationships between cell types and molecular states, ST provides the opportunity to explore how the TME evolves under therapeutic pressure and contributes to resistance. We performed spatial transcriptomic profiling on FFPE tumor sections from three matched longitudinal pairs of HGSOC samples obtained prior to treatment and upon PARPi progression. Utilizing the Xenium 5K Plex platform from 10X Genomics, we quantified the expression of 5000 genes in over 2.5 million cells. Cell type annotation was performed via reference mapping and marker gene expression. Differential expression and gene ontology enrichment analyses were conducted to identify key transcriptional changes in tumor clusters between paired naïve and progression samples. We furthermore describe spatial cellular neighborhoods and their changes upon development of resistance. Characterization of the tumors reveals great morphological heterogeneity between patients. Comparative analysis revealed distinct shifts in the cancer cell abundance, cancer cell area and spatial arrangement of the TME between naïve and progression samples. Progression samples exhibited increased expression of, for example, hypoxia-related genes, particularly in cancer cells. Spatial mapping identified distinct cellular neighborhoods, including niches with a high density of neutrophil- or macrophage-associated transcripts, which were notably more abundant in progression samples, suggesting a role in immune evasion and tumor progression. This study demonstrates the potential of spatial transcriptomics to elucidate the architectural and transcriptional changes of the ovarian cancer microenvironment at the single-cell level. While based on early findings, our results suggest the relevance of hypoxia, stromal remodeling, and niches with distinct cellular compositions in contributing to disease progression. These data support further investigation into spatially targeted therapeutic strategies and the use of ST as a tool for microenvironment-driven biomarker discovery.



 Sara K. Bystrup, Kathleen J. Imbach, Sergi Cervilla García, Arola Fortian Bernabeu, Daniella Grases, Eduard Porta Pardo, Jordi Barretina, Lorena Valdivieso Almeida, Pau Guillén Sentís, Adrià Bernat Peguera. Mapping the tumor microenvironment: Spatial Transcriptomic insights into ovarian cancer progression and resistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A003.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cf6f443298e3305bcbf89470d2e118a90600ac1" target='_blank'>
              Abstract A003: Mapping the tumor microenvironment: Spatial Transcriptomic insights into ovarian cancer progression and resistance
              </a>
            </td>
          <td>
            Sara K. Bystrup, Kathleen J. Imbach, Sergi Cervilla García, Arola Fortian Bernabeu, Daniella Grases, Eduard Porta Pardo, Jordi Barretina, Lorena Valdivieso Almeida, Pau Guillén Sentís, Adrià Bernat Peguera
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Neuroendocrine prostate cancer (NEPC) evolves as an aggressive phenotype during prolonged androgen deprivation therapy, lacking effective clinical management. Here, we elucidated a reciprocal metabolic-epigenetic mechanism involving a positive feedback loop between glycolysis and the transcription factor LHX2 that promotes PCa progression. Mechanistically, enzalutamide-mediated inhibition of the androgen receptor (AR) led to upregulation of key glycolytic enzymes. Elevated glycolysis resulted in lactate accumulation and subsequent histone lactylation, which in turn enhanced LHX2 expression. Reciprocally, LHX2 transcriptionally activated the lactate dehydrogenase LDHA, which further amplified lactate production. Furthermore, LHX2 augmented the expression of the lineage-determining enzyme DNMT1, potentiating neuroendocrine gene expression and tumor growth. Significantly, the antiviral agent paritaprevir could directly bind to LHX2, effectively suppressing neuroendocrine marker expression and tumor progression. These findings uncover a potential role for LHX2 in orchestrating NEPC development, offering putative therapeutic targets for treating this aggressive cancer phenotype and overcoming drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ef2fefc98e5fd3352706deab4254aa3b33bda2c" target='_blank'>
              LHX2 Rewires the Metabolic and Epigenetic Landscape to Drive Tumor Progression in Prostate Cancer.
              </a>
            </td>
          <td>
            Jun Jiang, Shaojie Liu, Chao Xu, Limin He, Hongji Li, Yike Zhou, Zhengxuan Li, Yu Li, Fa Yang, Yuchen Wei, Tong Lu, Keying Zhang, Jingliang Zhang, Pang Wang, Li Guo, Changhong Shi, Weihong Wen, Rui Zhang, Donghui Han, Weijun Qin
          </td>
          <td>2025-09-03</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background Transposable elements (TEs) represent a threat to genome integrity due to their proliferative capacity. Eukaryotic cells silence TEs through different molecular mechanisms, including the deposition of repressive histone marks. Previous studies have shown that TE repressive marks can spread to neighboring sequences. However, evidence for this spreading leading to nearby gene silencing remains limited. Similarly, whether TEs induce changes in the enrichment of active histone marks genome-wide, and the potential impact on gene expression have not been widely studied. Results In this work, we perform a comprehensive study of the epigenetic effects of 2235 TEs and their potential effects on nearby gene expression on Drosophila melanogaster head, gut, and ovary. While most TEs (816) induce the enrichment of the H3K9me3 repressive mark, with stronger epigenetic effects in the ovary, a substantial number (345 TEs) induce the enrichment of the H3K27ac active mark, particularly in the gut. We find that 70% of the H3K9me3 enriched TEs associated with expression changes downregulate the nearby gene, and 50% of the H3K27ac enriched TEs associated with expression changes lead to gene upregulation. These changes in expression affect specific regulatory networks only in the head. Furthermore, TE epigenetic effects on gene expression are genomic context dependent. Finally, we find that 221 TEs also affect gene expression by disrupting regions enriched for histone marks. Conclusions Overall, our results show that TEs contribute to the generation of regulatory novelty through epigenetic changes, with these epigenetic effects not restricted to gene silencing and being genomic context dependent. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03705-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75c9f4509d1e24d95361876bc4f99d2975a0de9c" target='_blank'>
              The epigenetics effects of transposable elements are genomic context dependent and not restricted to gene silencing in Drosophila
              </a>
            </td>
          <td>
            Marta Coronado-Zamora, Josefa González
          </td>
          <td>2025-08-18</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The persistent residual tumor cells that survive after chemotherapy are a major cause of treatment failure, but their survival mechanisms remain largely elusive. These cancer cells are typically characterized by a quiescent state with suppressed activity of MYC and MTOR. We observed that the MYC-suppressed persistent triple-negative breast cancer (TNBC) cells are metabolically flexible and can upregulate mitochondrial oxidative phosphorylation (OXPHOS) genes and respiratory function (“OXPHOS-high” cell state) in response to DNA-damaging anthracyclines such as doxorubicin, but not to taxanes. The elevated biomass and respiratory function of mitochondria in OXPHOS-high persistent cancer cells were associated with mitochondrial elongation and remodeling suggestive of increased mitochondrial fusion. A genome-wide CRISPR editing screen in doxorubicin-persistent OXPHOS-high TNBC cells revealed BCL-XL gene as the top survival dependency in these quiescent tumor cells, but not in their untreated proliferating counterparts. Quiescent OXPHOS-high TNBC cells were highly sensitive to BCL-XL inhibitors, but not to inhibitors of BCL2 and MCL1. Interestingly, inhibition of BCL-XL in doxorubicin-persistent OXPHOS-high TNBC cells rapidly abrogated mitochondrial elongation and respiratory function, followed by caspase 3/7 activation and cell death. The platelet-sparing proteolysis targeted chimera (PROTAC) BCL-XL degrader DT2216 enhanced the efficacy of doxorubicin against TNBC xenografts in vivo without induction of thrombocytopenia that is often observed with the first-generation BCL-XL inhibitors, supporting the development of this combinatorial treatment strategy for eliminating dormant tumor cells that persist after treatment with anthracycline-based chemotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfb7ad9e0cb28b546f19e574e42e36713a6ee110" target='_blank'>
              Quiescent OXPHOS-high triple-negative breast cancer cells that persist after chemotherapy depend on BCL-XL for survival
              </a>
            </td>
          <td>
            Slawomir Andrzejewski, Marie Winter, Leandro Encarnacao Garcia, O. Akinrinmade, F. D. Marques, Emmanouil Zacharioudakis, Anna Skwarska, J. Aguirre-Ghiso, Marina Konopleva, Guangrong Zheng, Susan Fineberg, Daohong Zhou, E. Gavathiotis, Tao Wang, Eugen Dhimolea
          </td>
          <td>2025-08-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="


 Randomized trials in adult patients have shown that rituximab, a monoclonal antibody targeting CD20 (MS4A1 gene), can improve treatment response in de novo Acute Lymphoblastic Leukemia (ALL). These findings have led to clinical trials incorporating rituximab into first-line chemotherapy for pediatric patients with high- or intermediate-risk CD20+ ALL. While immunotherapy in this context might improve survival, resistance to antibody-based therapies remains a major challenge, and its interaction with chemotherapy has not been studied in large cohorts of childhood ALL (cALL). Given the potential toxicities of immunotherapies, identifying predictive markers is critical for precision medicine. We hypothesized that the expression and relative abundance of MS4A1 isoforms influence the response of leukemic cells to rituximab in combination with standard chemotherapy. To test this, we studied MS4A1 transcriptional variability at cALL diagnosis using short-read (Illumina, n=31) and long-read (Oxford Nanopore Technologies ONT, n=2) RNA-seq. We obtained bone marrow aspirates of patients enrolled in the ALLIC-GATLA 2010 clinical protocol. ONT RNA-seq was performed at the Genomics Center of Facultad de Ciencias Exactas y Naturales, Universidad de Buenos Aires, while Illumina RNA-seq was conducted at Macrogen, Korea. We used Salmon, FLAIR and IsoQuant to annotate and quantify known and novel MS4A1 transcripts. Our results showed high heterogeneity among patients, both in transcriptional diversity and total transcript abundance. Notably, the MS4A1-204 isoform, which lacks the rituximab-binding domain, was expressed at diagnosis in >50% of patients. Isoform quantification differed significantly between ONT and Illumina, despite similar relative sequencing coverage of exons. Illumina predicted similar proportions of isoforms 201, 212, and 206, whereas ONT predicted dominance of the 201 isoform. Nevertheless, despite ONT sequencing depth was 6 to 11-fold lower than Illumina, molecular subtype predictions were concordant between both platforms. The discovery of novel MS4A1 transcripts using FLAIR and IsoQuant in ONT-sequenced samples showed differences in transcript identities. Structural analysis of two novel isoforms suggested the coding of putative proteins lacking extracellular or transmembrane domains, which are important for rituximab binding or may interfere with protein function. In conclusion, we successfully implemented ONT transcriptome sequencing in Argentina for ALL molecular subtyping, isoform annotation, quantification and novel transcript discovery. However, we detected high variability between sequencing platforms and bioinformatic tools, highlighting the need for further studies to improve reproducibility. These results are crucial for designing prospective studies evaluating patients treated with rituximab plus chemotherapy, currently underway in a multicentric clinical trial in Argentina. Additionally, this analysis could be extended to whole-transcriptome isoform diversity, complementing the characterization of MS4A1.



 Maria Sol Ruiz, Lucila Viappiani, Daniel Avendaño, Ignacio Gomez Mercado, Marina L Ingravidi, Geraldine Gueron, Javier Cotignola. Characterization of CD20 transcriptional diversity using short- and long-read sequencing in pediatric acute lymphoblastic leukemia: potential implications for rituximab plus chemotherapy treatment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr B003.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9570d8b058543a8f0614390dc61e70e4dcf1f779" target='_blank'>
              Abstract B003: Characterization of CD20 transcriptional diversity using short- and long-read sequencing in pediatric acute lymphoblastic leukemia: potential implications for rituximab plus chemotherapy treatment
              </a>
            </td>
          <td>
            Maria Sol Ruiz, Lucila Viappiani, Daniel Avendaño, Ignacio Gomez Mercado, Marina L. Ingravidi, G. Gueron, J. Cotignola
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/557ee2fb83678fffa15e9db9ad44ba30b15ed208" target='_blank'>
              Systematic decoding of functional enhancer connectomes and risk variants in human glioma.
              </a>
            </td>
          <td>
            Jinfang Bi, Weipeng Mo, Man Liu, Yingjie Song, Qian Xiao, Sibo Fan, Wenbin Wang, Tengfei Shi, Yaoqiang Zheng, Jie Lian, Rong Liu, Bohan Chen, Xiaofeng Huang, Peng Li, Zhongfang Zhao, Jiandang Shi, Lei Zhang, Guangsong Su, Nu Zhang, Wange Lu
          </td>
          <td>2025-08-19</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 The androgen receptor (AR) plays a critical role in the development and progression of prostate cancer by regulating key cellular processes such as cell proliferation and apoptosis. Although traditional AR-targeted therapies have shown initial success, acquired resistance remains a significant clinical challenge, often driven by AR alterations and somatic gene mutations associated with homologous recombination deficiency (HRD). Approximately 20% of advanced prostate cancer cases exhibit HRD, resulting in substantial genomic instability and complicating treatment. Fortunately, Food and Drug Administration–approved poly(ADP-ribose) polymerase inhibitors, including olaparib and rucaparib, exploit synthetic lethality to target prostate cancer with HRD, and additional drugs targeting DNA damage response (DDR) proteins are under development. Emerging evidence suggests that AR activity enhances DDR gene expression, with multiple DDR proteins localized near androgen-regulated regions, highlighting a close interaction between AR and DDR pathways. Consequently, recent preclinical and clinical studies have investigated combining AR-targeted therapies with treatments that induce DNA damage, such as radiation therapy, or inhibit DNA repair mechanisms. This review discusses AR's role in cellular processes, the interplay between AR and DDR, and recent advances in prostate cancer treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4bcf547c4cdb4775dcc9944086243c67a7115b7" target='_blank'>
              The interconnection between AR and DNA damage response pathways in prostate cancer
              </a>
            </td>
          <td>
            Mallory Sands, Sam Adams, Jihaeng Lee, Michael Li, Mae Wang, Thomas Walsh, Leo Leon, Adriana Zablah, Matthew Haerens, Zhichao Liu, Jianneng Li
          </td>
          <td>2025-08-19</td>
          <td>Current Urology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c0e77a0dc71f8f861306d03d6b2410b93243bdc" target='_blank'>
              Next-generation T cell immunotherapies engineered with CRISPR base and prime editing: challenges and opportunities.
              </a>
            </td>
          <td>
            Karl Petri, E. D’Ippolito, A. Künkele, Ulrike Köhl, Dirk H. Busch, H. Einsele, M. Hudecek
          </td>
          <td>2025-09-19</td>
          <td>Nature reviews. Clinical oncology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="


 Epigenetic dysregulation drives many pediatric tumors, including neuroblastoma, an aggressive malignancy that accounts for approximately 15% of childhood cancer deaths. Drugs targeting epigenetic regulators have emerged, with FDA approvals of tazemetostat for patients with metastatic or locally advanced epithelioid sarcoma with incomplete resection and for patients with relapsed or refractory follicular lymphoma with EZH2 mutations. More recently, the menin inhibitor revumenib was FDA approved for children and adults with relapsed/refractory KMT2A-rearranged acute myeloid leukemia. Unfortunately, these drugs are generally not curative, and resistance develops by multiple mechanisms suggesting that combination therapies are needed. Our laboratory recently identified some neuroblastomas, particularly MYCN amplified, that are dependent on the SAGA complex, a transcriptional coactivator complex that regulates gene expression by modifying histones. Knockout of TADA2B, a gene essential to the histone acetyltransferase function of SAGA, or treatment with GSK699, a selective degrader of KAT2A and KAT2B, the canonical catalytic acetyltransferases of SAGA, causes cell cycle arrest and delayed progression in xenograft models of neuroblastoma. In order to nominate combination strategies with SAGA inhibitors, we used orthogonal approaches: DepMap nearest neighbor analysis and a CRISPR sensitizer screen. Utilizing the Broad Institute’s Cancer Dependency Map (DepMap), we identified EZH2, a core member of the PRC2 complex, as a co-dependency with the SAGA complex in neuroblastoma. Furthermore, we conducted a CRISPR/Cas9 screen with a focused sgRNA library (1,350 genes) to identify additional epigenetic sensitizers and drivers of resistance in neuroblastoma cells treated with GSK699. EZH2 knockout scored as sensitizing to GSK699 while knockout of BRD2, a member of the BET family of proteins and an epigenetic reader, scored as rendering resistance to GSK699. The utilization of both in vitro and preclinical studies confirmed that pharmacological inhibition of KAT2A/B using GSK699 and EZH2 using tazemetostat induced a combinatorial inhibitory effect on neuroblastoma tumor proliferation compared to single agent treatment, with marked prolongation of survival in xenograft models with combination therapy. Transcriptional analysis of tumors treated with combination therapy showed increases in epithelial to mesenchymal transition (EMT) programs and myogenesis differentiation pathways, a surprising finding under active investigation. In conclusion, our data demonstrates that inhibition of the SAGA complex in combination with EZH2 inhibition effectively suppresses the proliferation of MYCN-amplified neuroblastoma supporting further evaluation of this combination in models of neuroblastoma, including deeper mechanistic studies.



 Melinda Soeung, Kenneth Ross, Lucy Merickel, Silvi Salhotra, Audrey Taillon, Allen Basanthakumar, Kimberly Stegmaier. Combination Therapeutic Targeting of Chromatin Complexes in Neuroblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr A043.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5216fb3aee2d0a8a7d9f8e5c04d43544aa31dd4b" target='_blank'>
              Abstract A043: Combination Therapeutic Targeting of Chromatin Complexes in Neuroblastoma
              </a>
            </td>
          <td>
            Melinda Soeung, Kenneth Ross, L. Merickel, S. Salhotra, A. Taillon, A. Basanthakumar, K. Stegmaier
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Expansions of short tandem repeats (STRs) are the cause of a class of human hereditary disorders called repeat expansion diseases (REDs). Most REDs are neurodegenerative or neurodevelopmental diseases such as Huntington’s disease, myotonic dystrophy, fragile X syndrome, and Friedreich’s ataxia. Some common neurodegenerative diseases, including Alzheimer’s and Parkinson’s disease, have also been associated with STR expansions. Many cellular processes such as meiotic recombination, DNA replication, and mismatch repair have been shown to promote STR instability. However, STR instability is likely the result of a variety of factors, and many questions regarding this phenomenon remain to be answered. In this review, we summarize recent studies that propose DNA single-strand breaks as drivers of large-scale STR instability, in both dividing and non-dividing cells, and discuss additional evidence that supports this model. We also highlight the FANCD2- and FANCI-associated nuclease 1 protein, which was shown to be the strongest genetic modifier of several REDs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e0da5948bfe8eeb0f167a2e9c2237af7d2aca7e" target='_blank'>
              Emerging drivers of DNA repeat expansions
              </a>
            </td>
          <td>
            Liangzi Li, W. S. Scott, S. Mirkin
          </td>
          <td>2025-08-13</td>
          <td>Biochemical Society Transactions</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Abstract Non-homologous end joining (NHEJ) is a DNA repair pathway that directly ligates broken DNA ends without the need for a homologous template, whereas translesion synthesis (TLS) is a DNA damage tolerance mechanism in which specialized DNA polymerases bypass lesions on the template strand. Although both pathways play critical roles in maintaining genome integrity across organisms, they inherently introduce mutations. Here, we investigate how these two pathways contribute to spontaneous and genotoxic stress–induced genomic alterations in the yeast Yarrowia lipolytica. A NHEJ-deficient mutant (ku70) and three TLS-deficient mutants (rev1, rev3, and rad30) are subjected to mutation accumulation experiments, followed by whole-genome sequencing. Our results show that the deletion of KU70 has no significant effect on the rates of spontaneous single-nucleotide variations (SNVs), small insertions and deletions, or chromosomal rearrangements, while the deletion of REV1 and REV3 leads to significant reductions in spontaneous SNV rates. These findings indicate that TLS but not the NHEJ pathway is a major contributor to spontaneous mutagenesis in Y. lipolytica. Moreover, exposure to 0.02% methyl methanesulfonate and 80 J/m2 ultraviolet (UV) radiation resulted in 48- and 107-fold increases in SNV rates, respectively. These induced SNVs were largely dependent on DNA polymerases Rev1 and ζ, further underscoring their central roles in genotoxic stress–induced mutagenesis. We observe that DNA polymerase η can suppress C to T and C to A substitutions while promoting T to C mutations, exhibiting a dual function in regulating mutagenesis under UV treatment. Phenotypic evolution experiments reveal that TLS activity enhances the adaptive potential of Y. lipolytica under oxidative stress, underlying its broader impact on environmental fitness. Together, these findings provide new insights into the distinct roles of the NHEJ and TLS pathways in preserving genome integrity in Y. lipolytica. Key points • The NHEJ pathway has a limited role in spontaneous genomic alterations in Y. lipolytica. • DNA polymerases Rev1 and ζ contribute to most UV- and MMS-induced mutations. • The dual roles of Pol η in UV-induced mutations were revealed. • NHEJ and TLS pathways are crucial to phenotypic evolution of Y. lipolytica. Supplementary Information The online version contains supplementary material available at 10.1007/s00253-025-13575-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15d93f494460655f7859b1e3dd9cb7f63f2e1036" target='_blank'>
              The roles of NHEJ and TLS pathways in genomic alterations and phenotypic evolution in the yeast Yarrowia lipolytica
              </a>
            </td>
          <td>
            Cen Yan, Ye-Ke Wang, Yuan-Ru Xiong, Xin-Qiu Zhou, Yuan-Chun Fang, Ruo-Tian Nie, Cunqi Ye, Ke Zhang, Dao-Qiong Zheng
          </td>
          <td>2025-08-15</td>
          <td>Applied Microbiology and Biotechnology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Heteroresistance is a form of antibiotic resistance in which a minor subpopulation of resistant cells coexists with a majority susceptible population. Colistin heteroresistance is common among Enterobacter cloacae clinical isolates, threatens its utility as a last-line therapeutic, and has become a model with which to understand the fundamental bases of heteroresistance. Despite numerous insights, the mechanism by which phenotypic heterogeneity is generated within the population and leads to colistin heteroresistance has been unclear. Here, using a transposon-based mutagenesis screen, we identify the sigma factor σEas the source of heterogeneity in population-wide colistin resistance levels. Single-cell tracking experiments revealed that σEis active in only one percent of the population at baseline, and only those cells with active σE survive colistin exposure. However, σE expression and population heterogeneity are insufficient for survival, as a mutant lacking the PhoPQ two-component system controlling lipid A modifications necessary for colistin resistance retains heterogeneity but loses colistin resistance. These findings lead to a new paradigm in heteroresistance, where the combinatorial action of multiple regulatory systems, encompassing a heterogeneity generator and a distinct resistance generator, are required to give rise to colistin heteroresistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f1f5c7e42785648fa5d1416ef151ee1dec8264f" target='_blank'>
              Combinatorial action of regulatory systems generates colistin heteroresistance
              </a>
            </td>
          <td>
            Jacob E. Choby, E. Crispell, Muqing Ma, L. M. Vu, Mary-Kate Key, Robert K. Ernst, Minsu Kim, David S. Weiss
          </td>
          <td>2025-08-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Epigenetic regulation, encompassing DNA methylation, histone modifications, and non-coding RNA activities, is a crucial mechanism through which gene expression is modified without corresponding changes in genomic DNA sequences. Dysregulation of these mechanisms can lead to histone and DNA modifications that either suppress or enhance the expression of disease progression-related genes. Among these regulatory processes, histone modifications are particularly significant, as they contribute to genomic stability, DNA repair, and chromatin dynamics, all of which influence breast cancer initiation and progression. This review offers a detailed analysis of the current state of research centered on epigenetic regulatory factors, with a particular focus on the role that histone modifications play in the treatment of breast cancer. It also examines the interplay between epigenetic modifications and the effectiveness of radiotherapy and chemotherapy when treating breast cancer. Lastly, this article explores the potential of epigenetic regulatory factors as viable targets for the future design of new anticancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/379da6773863c74043489d2ccffa9dd0b5e23c2c" target='_blank'>
              Epigenetic regulation espeically histone modifications in breast cancer: A viable and emerging targeted therapeutic strategy
              </a>
            </td>
          <td>
            Yibing Zhou, Haotian Liu, Weimin Hong, Haotian Su, Yuxiao Mu, Yijie Cheng, Chaoshen Wu, Xuli Meng, Da Qian
          </td>
          <td>2025-09-22</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Human Papillomavirus (HPV), particularly high-risk strains such as HPV16 and HPV18, is a leading cause of cervical cancer and a significant risk factor for several other epithelial malignancies. While the oncogenic mechanisms of viral proteins E6 and E7 are well characterized, the broader effects of HPV infection on host transcriptional regulation remain less clearly defined. This study explores the hypothesis that conserved genomic motifs within the HPV genome may act as molecular decoys, sequestering human transcription factors (TFs) and thereby disrupting normal gene regulation in host cells. Such interactions could contribute to oncogenesis by altering the transcriptional landscape and promoting malignant transformation.We conducted a computational analysis of the genomes of high-risk HPV types using MEME-ChIP for de novo motif discovery, followed by Tomtom for identifying matching human TFs. Protein–protein interactions among the predicted TFs were examined using STRING, and biological pathway enrichment was performed with Enrichr. The analysis identified conserved viral motifs with the potential to interact with host transcription factors (TFs), notably those from the FOX, HOX, and NFAT families, as well as various zinc finger proteins. Among these, SMARCA1, DUX4, and CDX1 were not previously associated with HPV-driven cell transformation. Pathway enrichment analysis revealed involvement in several key biological processes, including modulation of Wnt signaling pathways, transcriptional misregulation associated with cancer, and chromatin remodeling. These findings highlight the multifaceted strategies by which HPV may influence host cellular functions and contribute to pathogenesis. In this context, the study underscores the power of in silico approaches for elucidating viral–host interactions and reveals promising therapeutic targets in computationally predicted regulatory network changes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/019a01c1572bf807f787812d1b2ed2b11781bb4f" target='_blank'>
              HPV as a Molecular Hacker: Computational Exploration of HPV-Driven Changes in Host Regulatory Networks
              </a>
            </td>
          <td>
            M. Chetta, Alessandra Rosati, N. Bukvic
          </td>
          <td>2025-08-27</td>
          <td>Viruses</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Immunotherapy has emerged as a promising treatment option for triple-negative breast cancer (TNBC); however, the pronounced heterogeneity of the tumor immune microenvironment significantly hinders the prediction of therapeutic efficacy, with effective biomarkers also remaining limited. This study utilizes single-cell RNA sequencing (scRNA-seq) and transcriptome analysis to dissect the immune landscapes of TNBC, emphasizing CD8 effector T cells (CD8Teff) as potential predictors of immune checkpoint inhibitors (ICIs) treatment response. Immune profiling revealed that CD8Teff cells were predominantly enriched in "hot" tumors strongly correlating with improved progression-free and overall survival. Notably, the cytokine C-X-C Motif Chemokine Ligand 13 (CXCL13) emerged as a key regulator, with its high expression marking an immune-active tumor microenvironment favorable to ICI efficacy. Comprehensive analysis uncovered critical interactions between CD8Teff cells and other immune and stromal components, supporting the formation of an immunologically active tumor microenvironment (TME) conducive to tumor control. Additionally, metabolic reprogramming of CD8Teff cells in responsive tumors highlighted pathways that may influence ICIs efficacy. CD8Teff cells significantly influence the TME and predict ICI success in TNBC, with CXCL13 as a pivotal modulator. A pathology-based artificial intelligence model for CD8Teff recognition was developed, achieving an area under the curve (AUC) of 0.823 in the training cohort and 0.805 in the validation cohort. This study provides an in-depth characterization of the TNBC immune microenvironment, identifying CD8Teff functionality, CXCL13 signaling, and immune-metabolic pathways as critical determinants of immunotherapy success. These findings support the development of personalized treatment strategies based on immune profiling to enhance therapeutic efficacy in TNBC. Supplementary Information The online version contains supplementary material available at 10.1186/s43556-025-00306-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bfd680374201a021e9ef78818435e2a0634a514" target='_blank'>
              Single-cell RNA sequencing identifies CD8Teff cell activation as a predictive biomarker in triple-negative breast cancer immunotherapy
              </a>
            </td>
          <td>
            L. Mao, Zebang Zhang, Yongjian Chen, Qing Peng, Zhenjun Huang, Wenhao Ouyang, D. Zeng, W. Ren, Zifan He, Tang Li, Zehua Wang, Ruichong Lin, Jianli Zhao, Jiannan Wu, Herui Yao, Yunfang Yu
          </td>
          <td>2025-09-19</td>
          <td>Molecular Biomedicine</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Chronic lymphocytic leukemia (CLL) is the most common type of leukemia in Western populations and remains incurable despite significant therapeutic advancements. Over the past decade, the treatment landscape has evolved from traditional chemoimmunotherapy to targeted oral agents, including Bruton’s tyrosine kinase inhibitors (BTKis) and BCL2 inhibitors (BCL2is), which have demonstrated superior efficacy and tolerability, especially in elderly patients. Venetoclax, a BCL2i, induces apoptosis in CLL cells through selective inhibition of the anti-apoptotic BCL2 protein, while BTKis, such as ibrutinib and its next-generation analogs, disrupt B-cell receptor signaling critical to CLL cell survival. However, resistance to both drug classes has emerged, including mutations in BTK and BCL2, prompting the exploration of novel therapeutic strategies. This review outlines the molecular basis and clinical implications of these resistance mechanisms, as well as emerging therapeutic solutions, including non-covalent BTKis like pirtobrutinib and BTK-targeting PROTAC degraders such as BGB-16673 and NX-2127. Additionally, we discuss promising combination therapies incorporating BTKis, BCL2is, and anti-CD20 monoclonal antibodies. Finally, we highlight the growing role of measurable residual disease (MRD) as a biomarker to guide treatment duration and evaluate therapeutic success. As resistance mechanisms continue to emerge, tailoring therapy based on underlying biology will be critical to sustaining disease control and enhancing outcomes in patients with CLL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0956883756565c425ce66e36bee2fa5445ca9e94" target='_blank'>
              Resistance Mutations in CLL: Genetic Mechanisms Shaping the Future of Targeted Therapy
              </a>
            </td>
          <td>
            Samantha Sekeres, Erica N. Lamkin, Eduardo Bravo, Allison Cool, Justin Taylor
          </td>
          <td>2025-09-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The GATA3 noncoding variant rs3824662 has been implicated in the pathogenesis of Ph-like B-ALL, where it is associated with extensive chromatin reorganisation, resulting in the dysregulation of multiple genes, including CRLF2 overexpression. Given the altered chromatin landscape and increased accessibility of GATA3 binding regions associated with the rs3824662 variant, we investigated the potential role of enhancer RNAs (eRNAs) located near the GATA3 locus in regulating CRLF2 expression. We found that the expression of eRNA_G3, located at chr10:8,443,562–8,449,563, was positively correlated with CRLF2 expression. Notably, eRNA_G3 was significantly upregulated in Ph-like ALL cases carrying the GATA3 rs3824662 variant. These findings suggest that both rs3824662 and eRNAs may cooperatively contribute to the regulatory mechanisms governing GATA3 and CRLF2 expression in Ph-like ALL. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-10262-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a6bfc4308f5a1eb04611b56714ff4964a37c1d8" target='_blank'>
              Enhancer RNAs contribute to genome reprogramming driven by a GATA3 noncoding variant in leukaemia
              </a>
            </td>
          <td>
            Thayana da Conceição Barbosa, C. P. Poubel, A.L.T. Maciel, C. B. Blunck, Rafael Lima Dos Santos, M. Boroni, M. B. Mansur, M. Emerenciano
          </td>
          <td>2025-08-09</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="ABSTRACT Background Chronic myeloid leukemia (CML) is a clonal malignancy propelled by the BCR::ABL1 fusion gene originating from the Philadelphia chromosome. This gene activates ABL tyrosine kinase, which enhances the survival of leukemic cells. Although tyrosine kinase inhibitors (TKIs) have significantly advanced the treatment of CML, resistance to these inhibitors presents a substantial hurdle. Consequently, novel therapeutic strategies targeting resistance mechanisms independent of BCR::ABL1 are urgently needed. Methods This study investigated the potential impact of combining WEE1 inhibitors, particularly MK‐1775, with vitamin K2 (VK2) in treating CML. To analyze differentially expressed and spliced transcripts in CML, we examined mRNA profiles from peripheral blood mononuclear cells of five patients with CML (during chronic and blast phases) and five healthy controls. The samples were analyzed using deep sequencing. Differential expression analyses were performed using RaNA‐Seq and Heatmapper, the latter of which was designed for complex data set visualizations. Results WEE1 controls the G2/M checkpoint to prevent early mitosis, and blocking it increases the cytotoxicity of agents that damage deoxyribonucleic acid, especially in cancers lacking p53. VK2, a micronutrient, exerts anticancer effects against various malignancies. Gene expression studies have indicated that PKMYT1 expression is elevated in CML but not WEE1 cells. MK‐1775 successfully halted the growth of both standard and TKI‐resistant CML cell lines by triggering apoptosis via caspase 3/7 activation. VK2 reduced the viability of CML cells and increased cytotoxicity. A combined regimen of MK‐1775 and VK2 markedly decreased colony growth, disrupted mitochondrial membrane potential, and increased death in CML cells, including those resistant to TKIs. Conclusions The results suggest that a combination of MK‐1775 and VK2 represents a potentially effective treatment strategy for CML, especially in drug‐resistant cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/635521a554ad2c4e931009718d411cefa0b2c2c7" target='_blank'>
              Combination of WEE1 Inhibitor and Vitamin K2 Enhances Therapeutic Efficacy in Chronic Myeloid Leukemia
              </a>
            </td>
          <td>
            S. Okabe, Yuya Arai, Akihiko Gotoh, D. Akahane
          </td>
          <td>2025-08-28</td>
          <td>Cancer Innovation</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Background/Objectives: DNA-damaging agents can contribute to genetic instability, and such agents are often used in cancer chemotherapeutic regimens due to their cytotoxicity. Thus, understanding the mechanisms involved in DNA damage processing can not only enhance our knowledge of basic DNA repair mechanisms but may also be used to develop improved chemotherapeutic strategies to treat cancer. The high-mobility group box protein 1 (HMGB1) is a known nucleotide excision repair (NER) cofactor, and its family member HMGB3 has been implicated in chemoresistance in ovarian cancer. Here, we aim to understand the potential role(s) of HMGB3 in processing DNA damage. Methods: A potential role in NER was investigated using HMGB3 knockout human cell lines in response to UV damage. Subsequently, potential roles in DNA interstrand crosslink (ICL) and DNA double-strand break (DSB) repair were investigated using mutagenesis assays, metaphase spreads, foci formation, a variety of DNA repair assays, and TagSeq analyses in human cells. Results: Interestingly, unlike HMGB1, HMGB3 does not appear to play a role in NER. We found evidence to suggest that HMGB3 is involved in the processing of both DSBs and ICLs in human cells. Conclusions: These novel results elucidate a role for HMGB3 in DNA damage repair and, surprisingly, also indicate a distinct role of HMGB3 in DNA damage repair from that of HMGB1. These findings advance our understanding of the role of HMGB3 in chemotherapeutic drug resistance and as a target for new chemotherapeutic strategies in the treatment of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1261da2765ba41543d61e2f0da2291c8e870115b" target='_blank'>
              High-Mobility Group Box Protein 3 (HMGB3) Facilitates DNA Interstrand Crosslink Processing and Double-Strand Break Repair in Human Cells
              </a>
            </td>
          <td>
            Jillian Dangerfield, Anirban Mukherjee, W. Reh, Anna M. Battenhouse, Karen M. Vasquez
          </td>
          <td>2025-09-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Increasing evidence underscores the driving role of coding and non-coding variants in cancer development. Analyzing gene sets in biological processes offers deeper insights into the molecular mechanisms of carcinogenesis. Here, we developed geMER to identify candidate driver genes genome-wide by detecting mutation enrichment regions within coding and non-coding elements. We subsequently designed a pipeline to identify a core driver gene set (CDGS) that broadly promotes carcinogenesis across multiple cancers. CDGS comprising 25 genes for 25 cancers displayed instability in DNA aberrations. Variants within the TTN enrichment region may influence the folding of the I-set domain by altering local polarity or side-chain chemistry properties of amino acids, potentially disrupting its antigen-binding capacity in LUAD. Multi-omics analysis revealed that APOB emerged as a candidate oncogene in LIHC, whose genetic alterations within the enrichment region may activate key TFs, upregulate DNA methylation levels, modulate critical histone modifications, and enhance transcriptional activity in the HepG2 and A549 cell lines compared to Panc1. Additionally, CDGS mutation status was an independent prognostic factor for the pan-cancer cohort. High-risk patients tended to develop an immunosuppressive microenvironment and demonstrated a higher likelihood of responding to ICI therapy. Finally, we provided a user-friendly web interface to explore candidate driver genes using geMER (http://bio-bigdata.hrbmu.edu.cn/geMER/).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fe1ede7baa4480b19bb1878aa07140d59ffd789" target='_blank'>
              A core driver gene set identified based on geMER reveals its potential driver mechanism in pan-cancer
              </a>
            </td>
          <td>
            Jing Gan, Yuncong Wang, Zhuoran Shi, Haoyu Hu, Manyi Xu, Xinrong Li, Wenbo Dong, Jiaheng He, Yusen Zhao, Yakun Zhang, Yue Sun, Caiyu Zhang, Qianyi Lu, Shangwei Ning, Yan Jin, Hui Zhi
          </td>
          <td>2025-08-09</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50c6b58495df894089ef027a0154706304d13320" target='_blank'>
              Programmable antisense oligomers for phage functional genomics.
              </a>
            </td>
          <td>
            Milan Gerovac, Leandro Buhlmann, Yan Zhu, Svetlana Ðurica-Mitić, Valentin Rech, Samuel Carien, T. Gräfenhan, Linda Popella, Jörg Vogel
          </td>
          <td>2025-09-10</td>
          <td>Nature</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="The combination of 5-fluorouracil and irinotecan (FOLFIRI) remains a standard-of-care treatment for metastatic colorectal cancer (mCRC), yet benefits only about half of patients. Using patient-derived xenografts (PDXs), we investigated the biological underpinnings of this heterogeneous response. FOLFIRI-resistant models showed transcriptional upregulation of innate immunity and mitochondrial metabolism genes, together with reduced expression of the DNA polymerase POLD1. Sensitive counterparts exhibited a BRCAness-like phenotype with genomic scars of homologous recombination (HR) deficiency, not caused by genetic or epigenetic loss of HR genes but by low abundance of the RAD51 recombinase. In tumoroids, forced RAD51 overexpression attenuated HR deficiency-related scars and chemotherapy-induced damage, while HR inhibition through ATM blockade enhanced drug sensitivity. The predictive relevance of key response determinants was validated in clinical samples. This work illuminates functional, non-genetic facets of BRCAness in mCRC and introduces actionable biomarkers and targets, offering prospects to improve clinical decision-making and broaden therapeutic options for chemorefractory patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fb783603c147c9c964fbfcc1a8e05ff7e8e4445" target='_blank'>
              The molecular and functional landscape of resistance to FOLFIRI chemotherapy in metastatic colorectal cancer.
              </a>
            </td>
          <td>
            Marco Avolio, S. Leto, F. Sassi, B. Lupo, Elena Grassi, Irene Catalano, E. Zanella, V. Vurchio, F. Cottino, Petros K Tsantoulis, L. Lazzari, Paolo Luraghi, Martina Ferri, Francesco Galimi, E. Berrino, S. Bellomo, Marco Viviani, A. Sogari, G. Mauri, F. Tosi, F. Cruciani, A. Sartore-Bianchi, S. Siena, Felice Borghi, V. Torri, E. Élez, J. Tabernero, Maria Nieva, C. Montagut, N. Tarazona, Andrés Cervantes, Sabine Tejpar, A. Bardelli, C. Marchiò, S. Marsoni, Andrea Bertotti, L. Trusolino
          </td>
          <td>2025-09-22</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="The development of de novo resistance is a major disadvantage in molecularly targeted therapies. While much focus is on cell-intrinsic mechanisms, the microenvironment is also known to play a crucial role. This study examines interactions between cancer cells and cancer associated fibroblasts (CAFs) to understand the local crosstalk facilitating residual disease. Using a hybrid-discrete-continuum model, we explore how treatment-induced stress responses can elicit CAF activation and how breaks in treatment allow microenvironment normalisation. We investigate how fluctuating environmental conditions shape the local crosstalk and ultimately drive residual disease. Our experimentally calibrated model identifies environmental and treatment conditions that allow tumour eradication and those that enable survival. We find two distinct mechanisms that underpin residual disease: vasculature-limited drug delivery and CAF-mediated rescue. This work provides a better understanding of the mechanisms that drive the creation of localised residual disease, crucial to informing the development of more effective treatment protocols.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52e202efa4d7c859cf189d67f82a4d897793861f" target='_blank'>
              The role of environmentally mediated drug resistance in facilitating the spatial distribution of residual disease
              </a>
            </td>
          <td>
            Amy Milne, A. Marusyk, P. Maini, Alexander Anderson, Noemi Picco
          </td>
          <td>2025-08-09</td>
          <td>Communications Biology</td>
          <td>3</td>
          <td>17</td>
        </tr>

        <tr id="DNA replication, as the foundation of genetic information transmission and the continuity of life, relies on high fidelity and dynamic regulatory mechanisms that are central to maintaining genome stability. Through literature analysis, this review systematically summarizes the core molecular mechanisms of DNA replication initiation, elongation, proofreading, and repair, while comparing the commonalities and differences between prokaryotic and eukaryotic systems. Comparative analysis demonstrates that while prokaryotic and eukaryotic DNA replication share conserved high-fidelity principles, they exhibit fundamental mechanistic divergences. Advances in technologies such as cryo-electron microscopy have uncovered the structure and dynamics of the replication machinery and its coupling with epigenetic regulation. Future research should employ integrative multi-omics approaches to decipher the spatiotemporal regulation of replication, facilitating therapeutic interventions for replication-associated pathologies including cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/951febf5def0fd8aa3519a39b3eb248cfca4ae46" target='_blank'>
              DNA Replication and the Transmission of Genetic Information: A Molecular Mechanism Perspective
              </a>
            </td>
          <td>
            Ziying Wang
          </td>
          <td>2025-09-24</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The SS18::SSX oncogene is the driver of synovial sarcoma, an aggressive cancer presenting in young adults that has poor long-term outcomes. Over the past five years, significant progress has been made in understanding the molecular, genomic, and epigenetic mechanisms underlying synovial sarcoma. This review synthesizes recent advancements in synovial sarcoma, including diagnostic pathology, genomic profiling, SS18::SSX biology, epigenetic dysregulation, proteomics, targetable pathways and immunotherapy. Key findings include the identification of rare but instructive alternative gene fusions, the roles of PRC1 and of liquid-liquid phase separation in SS18::SSX-mediated oncogenesis, and the development of epigenetic and engineered T-cell therapies. These advances offer new hope for improved treatments and outcomes in synovial sarcoma patients, though challenges remain in overcoming resistance and ensuring equitable access to emerging therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9194070a3830c48d1eb33aa8623f50b92cbe3b84" target='_blank'>
              Molecular and epigenetic oncogenesis in synovial sarcoma: implications for cancer biology, diagnosis and treatment
              </a>
            </td>
          <td>
            Amy Xueqi Wang, Kevin B. Jones, Torsten O Nielsen
          </td>
          <td>2025-08-23</td>
          <td>Oncogene</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="


 TP53, often referred to as the guardian of the genome, is one of the most frequently mutated genes in human cancers. High grade serous ovarian cancer (HGSOC) has a TP53 mutation rate of up to 96%. Given its multi-faceted role in human cancers, there have been many efforts to therapeutically target mutant p53. APR-246 is a small molecule drug that is proposed to bind mutant p53 and promote its thermodynamic stabilization, allowing the tumor suppressor to regain wild-type-like conformation and functionality. However, the mechanism of this drug has remained a contentious topic, as it has been shown to act via various p53-independent mechanisms, including modulation of the redox system. To address this, we have performed a systematic functional genetic screening for the identification of APR-246 rescuable p53 mutants. Our approach leverages a pooled TP53 mutagenesis library containing ∼8,000 amino acid variants, spanning the entirety of the TP53 coding sequence. Cell based screenings against sub-lethal doses of APR-246 in two ovarian cancer cell lines, SKOV3 and OVCAR5, has revealed that sensitivity to this drug is cell-line dependent rather than mutant p53-dependent. Our results suggest that p53-independent factors such as intrinsic resistance to ferroptosis may play a more significant role in response to APR-246. We propose that resistance to ferroptosis provides cellular conditions which enable identification of p53 mutants that are sensitive to APR-246. This is evidenced by our results which show that genetic screen in OVCAR5, which is more intrinsically resistant to ferroptosis, uncovers a greater number of potentially sensitive mutants, including R175H, a mutant largely recognized as rescuable by APR-246. Further, we have identified GPX4, an anti-ferroptosis gene, as a key determinant of APR-246 sensitivity. Indeed, we demonstrate synergy between APR-246 and GPX4 inhibitor, ML-210. These studies support the hypothesis that APR-246 efficacy is largely attributable to p53-independent mechanisms and that attenuation of these off-target effect may reveal potentially sensitive p53 mutants.



 Anais Saunders, Caili Tong, Anthony N. Karnezis, Gary S. Leiserowitz, Jeremy R. Chien. Functional genetic screen of TP53 mutants reveals APR-246 sensitivity is mediated by p53 independent effects [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A046.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7beb6941bcf2a1d1e3d25e6917a37bc8794a91d" target='_blank'>
              Abstract A046: Functional genetic screen of TP53 mutants reveals APR-246 sensitivity is mediated by p53 independent effects
              </a>
            </td>
          <td>
            Anais Saunders, Caili Tong, A. Karnezis, G. Leiserowitz, Jeremy Chien
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Breast cancer is one of the most common malignancies among women globally, with an annually increasing incidence rate. Its complex pathogenesis and high heterogeneity pose significant challenges to clinical diagnosis and treatment. Traditional diagnostic methods and therapeutic approaches have limitations in improving patient survival rates and prognosis, thus urgently necessitating the identification of novel biomarkers and therapeutic targets. MicroRNA (miRNA), a class of endogenous non-coding small RNA molecules with a length of approximately 20–24 nucleotides, finely regulates gene expression by binding to the 3’ untranslated region of target mRNAs, inhibiting gene translation, or promoting mRNA degradation. In 2024, the Nobel Prize in Physiology or Medicine was awarded for research related to miRNA. Numerous studies have demonstrated that miRNAs play pivotal roles in the initiation, progression, invasion, metastasis, and drug resistance of breast cancer. Aberrant expression of specific miRNAs is closely associated with the molecular subtypes, prognosis, and treatment response of breast cancer, suggesting their potential as diagnostic and prognostic biomarkers. To explore the potential value of miRNA in the diagnosis and treatment of breast cancer, this article systematically reviews the latest research progress on the role of miRNAs in the diagnosis and treatment of breast cancer, with a focus on their application as tumor markers in early diagnosis, molecular subtyping, and therapeutic response monitoring. It elucidates the possibilities of miRNAs as therapeutic targets and tools in targeted therapy, including the current research status of miRNA mimics and inhibitors in breast cancer treatment. Furthermore, it analyzes the role of miRNAs in prognosis assessment, exploring their correlation with patient survival rates, recurrence risks, and treatment responsiveness. Additionally, this article discusses the challenges faced by miRNA research in precision medicine for breast cancer and future directions, providing new insights and strategies for early diagnosis, individualized treatment, and prognosis assessment of breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/302a8130c8ecf1046d266c188e11099501cd566d" target='_blank'>
              MicroRNAs in breast cancer—new frontiers in diagnosis, targeted therapy, and prognosis assessment
              </a>
            </td>
          <td>
            Jinsuo Xiao, Lanhui Zhang, Ruifan Su, Bo Zhao, Yuanyuan Dang, Chuanlin Zhao, Sujin Wang, Teng Qi, Fuqing Ji
          </td>
          <td>2025-09-01</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer has recently overtaken other conditions as the foremost cause of death worldwide. While numerous
conventional treatment modalities and cytotoxic immunotherapies are available, the intricate nature of tumor biology—
driven by a multitude of genetic and cellular factors underlying tumorigenesis and metastasis—necessitates the
development of advanced treatment approaches capable of acting on cellular as well as genetic fronts. Amid these
advancements, Chimeric Antigen Receptor (CAR) T-cell therapy has risen as a pioneering frontier in the domain of T-cell
engineering. Here, T cells obtained from the patient are engineered ex vivo to carry synthetic receptors designed to
recognize tumor-associated antigens. A key feature is that these modified receptors can identify tumor antigens without
requiring MHC presentation. Recently, CAR-T cell therapy has achieved significant clinical milestones, inducing
remission in up to 80% of patients with hematologic malignancies, particularly those with acute lymphoblastic leukemia
(ALL) and certain subtypes of non-Hodgkin lymphoma, such as large B-cell lymphoma. The anti-CD19 CAR design,
UCART19, has demonstrated remarkable therapeutic potency in managing relapsed or refractory hematologic
malignancies. Furthermore, other surface antigens, including CD20 and CD22, prevalent in various leukemias and
lymphomas, are being actively explored as therapeutic targets, with multiple clinical trials underway. Although its
application is presently concentrated on blood cancers, the integration of advanced modalities—such as bispecific CARs,
tandem CARs, inhibitory CARs, multi-antigen targeting, CRISPR-based gene editing, and nanoparticle-mediated
delivery—holds the promise of significantly enhancing its efficacy. These advancements may extend its utility to both
hematologic and solid tumors, offering a treatment paradigm that is not only rapid and precise but also safer compared to
traditional modalities. The purpose of this review is to offer an in-depth analysis of the benefits and emerging
developments in CAR T-cell immunotherapy, while underscoring its rising advantage over conventional treatments like
chemotherapy and radiotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba5c74588893737227ed28d56c33635129d1547e" target='_blank'>
              Targeting Cancer at its Core: Cellular and Genetic Approaches Via CAR-T Cell Therapy CAR-T Cell Therapy of Cancer
              </a>
            </td>
          <td>
            Bandaru Aashritha, Ambati Akshara, Nandini Kongara, P. V. Babu
          </td>
          <td>2025-09-04</td>
          <td>International Journal of Innovative Science and Research Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="A rare disease is generally defined as a medical condition that affects a small proportion of the population, though specific thresholds vary across countries. Despite regional differences, these definitions consistently reflect the low prevalence of such conditions, the limited availability of effective treatments, and the pressing need for targeted research and regulatory support. As a result of their rarity and low commercial potential, rare diseases have historically represented an area of market failure, where investment and research have been limited and often neglected. However, since the 1990s, each country has guaranteed continuous support to research and development projects to promote the advancements of rare disease treatments, achieving a growth rate greater than that of the entire pharmaceutical industry. In this review, we examine the status of orphan drug development using an advanced therapy medicinal product (ATMP) approach in the growing rare disease market, with a particular focus on cell therapies and gene therapies, which constitute the most actively developed and clinically applied categories within ATMPs. We also explore strategic approaches through which the orphan drug industry can utilize ATMPs, especially these two modalities, to enhance its competitiveness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fb2ccc6113bccae4bf1abe5e01112d77e7abd03" target='_blank'>
              Advances in Cell and Gene Therapy for Rare Disease Treatment.
              </a>
            </td>
          <td>
            Hyun Jin Baek, Yoojun Nam, Yeri Alice Rim, Ji Hyeon Ju
          </td>
          <td>2025-09-01</td>
          <td>International journal of stem cells</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Cancer progression is often accompanied by epigenetic silencing of tumor-suppressor microRNAs such asmiR-200c, a key regulator of epithelial-to-mesenchymal transition (EMT) and metastasis. Given the reversible nature of DNA methylation, we employed a CRISPR/dCas9-TET1 system to target the miR-200c promoter and restore its expression in MCF-7 and MDA-MB-231 breast cancer cell lines. Two gRNAs were designed to flank CpG-rich regions of the miR-200c promoter, and their individual or combined delivery enabled site-specific demethylation. Co-transfection with both gRNAs resulted in a synergistic increase in miR-200c expression, likely due to expanded coverage of dCas9-TET1 recruitment. This upregulation led to the downregulation of key EMT-related transcription factors ZEB1, ZEB2, and the oncogene KRAS, as well as increased E-cadherin expression in MDA-MB-231 cells. However, E-cadherin changes in MCF-7 cells were minimal, highlighting the complex and context-dependent nature of epigenetic regulation. Functional assays further confirmed the anti-tumorigenic effects of miR-200c restoration, with reduced cell viability and increased apoptosis, effects more pronounced in MDA-MB-231 cells, which initially exhibited higher miR-200c promoter methylation. Collectively, our findings demonstrate that CRISPR/dCas9-TET1-mediated epigenetic editing effectively reactivates miR-200c, reverses EMT-associated gene expression, and impairs tumor cell aggressiveness, supporting its potential as a targeted therapeutic strategy in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14e592b60e9ed7df393f38257baac9b31bbd24eb" target='_blank'>
              CRISPR/dCas9-TET1–mediated epigenetic editing reactivates miR-200c in breast cancer cells
              </a>
            </td>
          <td>
            Mahyar Zahraei, Yasamin Azimi, Morteza Karimipour, Fatemeh Rahimi-Jamnani, Vahideh Valizadeh, Masoumeh Azizi
          </td>
          <td>2025-08-28</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background Osteosarcoma is the most common primary malignant tumor of bone and occurs predominantly in children and adolescents during periods of accelerated skeletal growth. Within the tumor microenvironment (TME), vascular smooth muscle cells (VSMCs) may acquire osteogenic characteristics. Nuclear factor of activated T cells c1 (NFATc1) is central to skeletal biology, yet its behavior in VSMCs in osteosarcoma remains insufficiently defined. Methods We profiled fresh osteosarcoma tissues by single-cell RNA sequencing on the 10x Genomics Chromium platform. Mechanical and enzymatic dissociation yielded single-cell suspensions; viable cells (> 85%) were enriched; libraries were prepared with Single Cell 3′ v3 reagents and sequenced on NovaSeq 6000. Reads were processed with Cell Ranger (GRCh38). Following log-normalization, 2,000 highly variable genes were retained. Harmony integration reduced batch structure. We computed 50 principal components, generated UMAP/t-SNE embeddings, and applied k-nearest neighbor/Louvain clustering (k = 20; resolution = 0.5). Cell types were assigned from differentially expressed markers cross-checked with CellMarker and PanglaoDB. NFATc1 distribution and co-expression patterns were assessed by Pearson correlation. Results Nine major cell classes were identified, including VSMCs, fibroblasts, macrophages, T/NK subsets, B cells, neutrophils, mast cells, and inflammatory myeloid cells. NFATc1 expression was most prominent in VSMCs, with a bimodal distribution within this lineage. In VSMCs, NFATc1 co-varied with SERGEF, APOE, and CCS, pointing to coordinated cytoskeletal remodeling, lipid handling, and copper homeostasis during osteogenic reprogramming. Conclusions At single-cell resolution, NFATc1 marks a VSMC fraction with osteogenic features in osteosarcoma. The associated network suggests testable links to metabolism and matrix remodeling; functional perturbation will be required to establish causality. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03606-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e3638bf4253781f5bd9cf49aea68da6335438a6" target='_blank'>
              Single-cell RNA sequencing reveals NFATc1-mediated regulatory mechanisms of vascular smooth muscle cell osteogenic transformation in the osteosarcoma microenvironment
              </a>
            </td>
          <td>
            Xuejiao Yi, Shilin Fu, Chunliang Zhang, Tianle Zhao, Huan Feng, Dehui Qian
          </td>
          <td>2025-09-29</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Glioblastoma multiforme represents the most common primary malignant brain tumor in adults, with a 90% 2-year recurrence rate despite standard therapy of surgical resection followed by chemo- and radiotherapy. Yet, the cellular mechanisms underlying therapeutic resistance remain poorly understood. By integrating analysis of single-cell RNA sequencing results of 41 primary and 43 recurrent tumor specimens with experimental validation, we identify neuro-oligodendrocyte precursors as the predominant cellular population driving tumor recurrence through establishment of protective microenvironmental niches. We discover that CTNNA3 and MBP exhibit paradoxical roles in promoting survival in primary tumors while conferring chemoresistance in recurrent GBM, revealing that these factors determine treatment efficacy rather than directly driving relapse. Functional assays confirmed their role as mediators of extracellular matrix remodeling, establishing a mechanistic link between tumor microenvironmental remodeling and therapeutic resistance. By revealing how recurrent tumors engineer self-protective niches, our study provides a new framework for developing combination therapies that disrupt these resistance mechanisms and provides foundation for overcoming treatment failure in this universally fatal disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ec11a04512f1a7cb7a16592e9fd8fa7748da070" target='_blank'>
              CTNNA3 and MBP Mediate Treatment Resistance Through Tumor Microenvironment Remodeling in Recurrent Glioblastoma
              </a>
            </td>
          <td>
            Wenna Mao
          </td>
          <td>2025-09-24</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/648ee47786050552f00a0c31f0d21f6a61f09642" target='_blank'>
              LEF1 confers resistance to DNA-damaging chemotherapies through upregulation of PARP1 and NUMA1 in ovarian cancer.
              </a>
            </td>
          <td>
            Lin Huang, Mengna Zhu, Mengqing Chen, F. Ying, Qiulei Wu, Pan Liu, Wenhan Li, Guoqing Li, Yuewen Gao, Shuyan Yi, Wenwen Wang, Yiping Wen, Si Sun, Jing Cai, Man Xiao
          </td>
          <td>2025-09-10</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Metabolic reprogramming is a key feature of clear cell renal cell carcinoma (ccRCC), and metabolic abnormality can lead to significant changes in gene expression, resulting in the immunosuppressive microenvironment. In this study, we used a combination of single-cell RNA sequencing and bulk RNA sequencing to investigate the relationships between ccRCC metabolic reprogramming and immune exhaustion. Metabolic subtypes of ccRCC patients were constructed using bulk RNA sequencing. Tumor cells of different metabolic subtypes were analyzed and extracted by the Scissor algorithm, using single-cell RNA sequencing. The molecular mechanisms of abnormal metabolic regulating tumor immunity were explored using cell–cell communication analysis. In addition, the correlations between relevant molecules and immune exhaustion signals were verified in ccRCC by immunohistochemistry. The molecular mechanisms of metabolic abnormalities leading to immune exhaustion were validated via Western blotting, ELISA, cell co-culture and immunotherapy models. ccRCC patients can be divided into MT1 and MT2 metabolic subtypes. The MT2 subtype has a poorer prognosis and lower response to immunotherapy. Abnormal metabolism of arachidonic acid is a prominent feature of the MT2 subtype, and activates the MDK signaling pathway. As a secreted protein, MDK can further recruit immunosuppressive cells, such as Treg, Tex, and TAM. Blocking the arachidonic acid COX metabolic pathway significantly reduces the expression and secretion levels of MDK, thereby reprogramming the tumor microenvironment to promote anti-tumor immunity. Abnormal metabolism of arachidonic acid plays an important role in promoting immune exhaustion by activating the MDK signaling pathway. MDK may serve as an important biomarker for predicting the immune therapy response in ccRCC. By reducing MDK secretion, targeting blockade of arachidonic acid metabolism may be an effective treatment strategy to enhance the efficacy of immunotherapy in ccRCC. Supplementary Information The online version contains supplementary material available at 10.1007/s10238-025-01807-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7f6cdee7e8710af1e672e045bcfb43ec330de5b" target='_blank'>
              Metabolic reprogramming of arachidonic acid in clear cell renal carcinoma promotes an immunosuppressive microenvironment by activating MDK signaling pathway
              </a>
            </td>
          <td>
            Jiaxi Yao, Tong Xu, Chenyuan Wang, Junfeng Xie, Qing Jiang
          </td>
          <td>2025-08-13</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The mutagenic translesion synthesis (TLS) pathway, which is critically dependent on REV1's ability to recruit inserter TLS polymerases and the POLζ extender polymerase, enables cancer cells to bypass DNA lesions while introducing mutations that likely contribute to the development of chemotherapy resistance and secondary malignancies. Targeting this pathway represents a promising therapeutic strategy. Here, we demonstrate that the expression of the C-terminal domain (CTD) of human REV1, a ca. 100 amino acid scaffold essential for TLS polymerase interactions, disrupts REV1/POLζ-dependent TLS in mammalian cells. Inducible expression of REV1-CTD in multiple human and murine cancer cell lines sensitizes cells to DNA-damaging agents such as cisplatin, benzo[a]pyrene diol epoxide, and methyl methanesulfonate, without intrinsic cytotoxicity. REV1-CTD expression increases genomic instability, decreases mutagenesis, and enhances G2 arrest following genotoxic stress. Mutational disruption of the CTD's interaction interfaces abrogates these effects, confirming a dominant-negative mechanism via sequestration of TLS components. In a xenograft mouse model, REV1-CTD expression markedly enhances cisplatin efficacy, significantly reducing tumor burden. These findings establish the REV1-CTD as an effective dominant-negative inhibitor of TLS and support its development as a therapeutic agent delivered to cancer cells to enhance the efficacy of genotoxic chemotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/119ebde0313a7f12ca9995ad4532ab0aaa8b95c3" target='_blank'>
              Sensitization of cancer cells to DNA-damaging agents by expression of the REV1 C-terminal domain: Implications for chemotherapy.
              </a>
            </td>
          <td>
            Ke Bian, Charley Gruber, Emine Byers, Paul Leclerc, Michael T. Hemann, Graham C. Walker
          </td>
          <td>2025-09-10</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Chemoresistance is a major obstacle in cancer treatment, and it often results in treatment failure and disease progression. Among the plethora of factors contributing to chemoresistance, the transcription factor Krüppel-like factor 4 (KLF4) has emerged as a pivotal player. This review discusses the role of KLF4 in orchestrating various mechanisms underlying cancer chemoresistance. KLF4, originally identified as a regulator of cell differentiation and proliferation, has recently gained attention for its role in modulating cellular responses to chemotherapeutic agents. Through complex regulatory networks, KLF4 modulates the process of drug efflux, DNA repair, apoptotic signaling, tumor heterogeneity, and cancer cell stemness, leading to the development of cancer chemoresistance. Additionally, tissue or cell types specific post-translational modification (PTM) of KLF4 plays a significant role in the development of cancer chemoresistance. The review explores emerging possibilities and available information that can be utilised to understand the mechanism of chemoresistance mediated by KLF4 in cancer. In conclusion, understanding the complex mechanisms through which KLF4 orchestrates cancer chemoresistance opens promising avenues for developing more effective therapeutic interventions to combat treatment-resistant cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26e91a9cb65a0f14dcdac5b83f903eda917dba7f" target='_blank'>
              KLF4 in cancer chemoresistance: molecular mechanisms and therapeutic implications
              </a>
            </td>
          <td>
            Suresh Singh Yadav, Punita Kalia, Navneet Kaur, Sourav Sharma, Shambhavi Thakur, Soni Kumari, R. R. Nair
          </td>
          <td>2025-09-22</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Acute lymphoblastic leukemia (ALL) is the leading cause of cancer-related mortality in pediatrics. Between 15-30% of patients relapse, and current treatments are associated with acute and long-term toxicity. The immune component of the tumor microenvironment is an under-explored factor that could play a role in the establishment or progression of the disease, as well as modulate response to chemo- and immunotherapies. We characterized the molecular profile of leukemias from 32 patients with pediatric ALL treated under the ALLIC-GATLA 2010 clinical protocol in Argentina by transcriptome sequencing (Illumina). We estimated the immune component through digital cytometry (MIXTURE) and a cytolytic score based on the expression of five genes. We searched for single nucleotide variants (SNV) in the transcriptome (CTAT-Mutations) that could alter the interaction with immune cells in the tumor microenvironment. For this, we analyzed 843 genes from Reactome collections associated with the immune system. Variants were filtered and annotated using publicly available germline and somatic variant databases. The group of patients with higher abundance of cytotoxic cells (Z-score>1) was associated with an increased risk of relapse/death (HR=4.5; p=0.05), and higher levels of CX3CR1 (q-val<0.05) and TIM-3 (q-val=0.06). Analysis of an independent high-risk cohort (TARGET B-ALL phase II, N=88), confirmed that a higher cytolytic score was associated with the risk of relapse in an univariate (HR=2.66, p=0.001) and multivariate model including cytolytic score (p<0.006), protocol (p=0,07), ETV6::RUNX1 (p:>0.05) and TCF3::PBX1 status (p<0.001). Cytolytic score was not associated with molecular subtype, age, sex or early response to treatment (p>0.05). For SNV analysis, we developed a threshold value for the QUAL parameter to select for high-confidence variants, establishing a specificity of 95%. Eleven variants were found in nine patients, being 8/11 classified by Franklin as TIER3 in CEP290, UBE3B, CPNE3, DUSP4, DAPP1, RNF135, TBC1D10C and TLR1; 2/11 as TIER1 in TP53, KRAS; and 1/11 as TIER4 in BTN3A3. Patient harboring variants in these genes were significantly associated with a lower GSVA score in the Reactome dataset “MHC_Class_II_Antigen_Presentation” (q-val<0.01). We found a variant in UBE3B, located in the active site of the E3 ubiquitin ligase, and associated with a low GSVA score in the Reactome “Antigen presentation through MHC class I” pathway. A novel variant in CPNE3 gene was found, located in a highly conserved residue and predicted to be deleterious by multiple tools. Conclusion: the abundance of cytotoxic cells was independent of molecular subtype and associated with a worse prognosis in independent cohorts. Further analysis of the variants found in this study has the potential to identify novel mechanisms modulating the interaction between the immune system and leukemic cells in ALL.



 Ignacio Gomez Mercado, Daniel Avendaño, Sabrina Ledesma Bazan, Pablo Sanchis, Maria Cecilia Riccheri, Geraldine Gueron, Javier Cotignola, Maria Sol Ruiz. Characterization of cytotoxic cell abundance and genetic variants in immune related genes in pediatric acute lymphoblastic leukemia patients [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr A040-PR008.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc8f83b4eeb02fdcb7483db9e8c77e60a6f1fd7e" target='_blank'>
              Abstract A040-PR008: Characterization of cytotoxic cell abundance and genetic variants in immune related genes in pediatric acute lymphoblastic leukemia patients
              </a>
            </td>
          <td>
            Ignacio Gomez Mercado, Daniel Avendaño, Sabrina Ledesma Bazan, P. Sanchis, M. C. Riccheri, G. Gueron, J. Cotignola, Maria Sol Ruiz
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background Head and Neck Squamous Cell Carcinoma (HNSCC) is a prevalent and deadly cancer with limited treatment options, for which autophagy has a dual impact. Nitazoxanide is a broad-spectrum antiviral drug showing promise in treating cancer, but its role in regulating autophagy in HNSCC remains unexplored. Materials and methods The molecular mechanisms were explored through Differential expression analysis, functional enrichment analysis, risk prognostic model, nomogram model, GSEA, GSVA, WGCNA, mutation analysis, scRNA analysis, spatial transcriptomic analysis, and L1000FWD analysis. And the impact of Nitazoxanide was experimentally verified. Results We identified 32 differentially expressed ARGs and 4 hub genes. They are involved in autophagy, apoptosis, and human cytomegalovirus infection. And hub genes were identified as independent prognostic factors for HNSCC. GSEA, GSVA and WGCNA uncovered the role of the dysregulation of IFN-α response pathway in HNSCC. scRNA and spatial transcriptomic analyses revealed the expression models of each hub gene. Furthermore, potential drugs were explored using L1000FWD analysis. Additionally, Nitazoxanide’s function was verified in vitro. Conclusion Our findings established that Nitazoxanide has the potential to play a role in the treatment of HNSCC. We suggested that Nitazoxanide acts on NKX2-3, PINK1, BIRC5 and CDKN2A to inhibit the development of HNSCC through regulating autophagy, and it is a promising candidate drug for HNSCC. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03372-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15de8fc0cf296d0b065336ac7cc5bd1b8626e7af" target='_blank'>
              Integrating single-cell RNA sequencing and spatial transcriptomics reveals the therapeutic effect of nitazoxanide in head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Xiaohua Jiang, Liqi Chen, Zixia Xu, Hangzhe Sun, Jing Ye, Ziyi Guo, M. Xiao
          </td>
          <td>2025-08-08</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Ovarian cancer represents a typically immune “cold” tumor, where obvious immunosuppression, spatial T-cell exclusion, and cellular dysfunction collectively limit immunotherapy effectiveness. Especially in high-grade serous ovarian carcinoma (HGSOC), the immune low-response state is driven by complex interactions among tumor-associated macrophages (TAMs), suppressive stromal networks, and the T-cell compartment (regulatory T cells, Tregs, and exhausted effector T cells). Emerging multi-omics technologies—particularly single-cell RNA sequencing and spatial transcriptomics—have showed the heterogeneity and spatial immune organization underlying this suppressed state. Here, we integrate these datasets to describe TAM phenotypes and spatial niches, T-cell exhaustion, Tregs accumulation, NK-cell dysfunction, and stromal barriers that enforce exclusion. We then derive phenotype-guided combination strategies to remodel the tumor microenvironment and improve responsiveness to immune checkpoint blockade. This synthesis provides a concise, multi-dimensional framework for precision immunotherapy and for overcoming resistance in immune-low ovarian cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/880f6c405f773eb38fd078c68256074438417fa2" target='_blank'>
              Decoding immune low-response states in ovarian cancer: insights from single-cell and spatial transcriptomics for precision immunotherapy
              </a>
            </td>
          <td>
            Yalan Yan, Jiaan Lu, Huanyu Luo, Zizhang Wang, Ke Xu, Lexin Wang, Qin Wang
          </td>
          <td>2025-09-19</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Colorectal cancer is one of the most prevalent malignant tumors worldwide, and it has one of the highest rates of mortality and morbidity. Its emergence and progression are intricately linked to various genetic and epigenetic factors influencing colonic epithelial cells. Long non-coding RNAs function as crucial regulators of the epigenetic landscape, playing significant roles in the development, progression, invasion, metastasis, and drug resistance of various tumors, particularly colorectal cancer. Epigenetic modifications, such as DNA methylation, RNA methylation, and histone modification, facilitate heritable gene expression regulation without changing the underlying DNA sequence. Recent studies have increasingly highlighted the intricate interactions between lncRNAs and these epigenetic modifications in colorectal cancer, illuminating their participation in critical processes, including cell proliferation, apoptosis, invasion, and metastasis. This review concentrates on the relationship between long non-coding RNAs and several significant epigenetic modifications that drive colorectal cancer development, offering fresh insights for future research that aims at devising more effective treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31b080b73592dba79e60d81f2a7fb3048fe78b62" target='_blank'>
              Long non-coding RNAs promote colorectal cancer development through other epigenetic modifications
              </a>
            </td>
          <td>
            Guojuan Wang, Qingqing Ma, Naicheng Zhu, Yuwei Yan, Chen Zhong, Xinlin Hong, Yujuan Lai, Xiudan Chen, Nanxin Li, Wenyan Yu
          </td>
          <td>2025-09-28</td>
          <td>Gastroenterology Report</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract DNA-damaging agents (DDAs) have long been used in cancer therapy. However, the mechanisms by which DDAs induce cell death are not fully understood, and drug resistance remains a major clinical challenge. Schlafen 11 (SLFN11) was identified as the gene most strongly correlated with sensitivity to DDAs based on mRNA expression levels. SLFN11 sensitizes cancer cells to DDAs by cleaving and decreasing tRNALeu(TAA) levels. Elucidating the detailed mechanism by which SLFN11 induces cell death is expected to provide insights into overcoming drug resistance. Here, we show that, upon administration of DDAs, SLFN11 cleaves tRNALeu(TAA), triggering ER stress and protein aggregate formation, leading to cell death regulated by inositol-requiring enzyme 1 alpha (IRE1α). These responses were significantly alleviated by SLFN11-knockout or transfection of tRNALeu(TAA). Proteomic analysis suggests tRNALeu(TAA) influences proteins essential for maintaining proteostasis, especially those involved in ubiquitin-dependent proteolysis. Additionally, we identified the cleavage sites of tRNALeu(TAA) generated by SLFN11 in cells and revealed that tRNA fragments contribute to ER stress and cell death. These findings suggest that SLFN11 plays a crucial role in proteostasis by regulating tRNAs and thus determines cell fate under DDA treatment. Consequently, targeting SLFN11-mediated tRNA regulation could offer a novel approach to improve cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f04bd7e8511d20689e4f33be87517895928d1815" target='_blank'>
              SLFN11-mediated tRNA regulation induces cell death by disrupting proteostasis in response to DNA-damaging agents
              </a>
            </td>
          <td>
            Yuki Iimori, Teppei Morita, Takeshi Masuda, Shojiro Kitajima, N. Kono, Shun Kageyama, Josephine Galipon, A. Sasaki, Akio Kanai
          </td>
          <td>2025-08-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is the second leading cause of cancer death worldwide, largely due to the limited efficacy of current therapies in advanced stages of the disease. Most cases of HCC develop in the setting of chronic liver disease, particularly cirrhosis, where ongoing cycles of inflammation, hepatocyte death and regeneration foster the gradual accumulation of genetic and epigenetic alterations that promote malignant transformation. These molecular changes contribute to the high degree of tumour heterogeneity observed in HCC, a major factor underlying resistance to current treatments. As a result, sustained clinical responses to existing therapies, such as tyrosine kinase inhibitors, anti-angiogenic agents and immune checkpoint inhibitors, remain uncommon. In this context, a growing body of evidence has identified epigenetic dysregulation as a key driver of tumour progression and therapeutic resistance, highlighting a new frontier for intervention. This review provides clinicians and researchers with a comprehensive overview of the emerging field of epigenetic therapies in HCC, summarising results from both completed and ongoing clinical trials involving the so-called 'epidrugs'. Importantly, we discuss how targeting epigenetic mechanisms may not only suppress tumour growth but also enhance the effectiveness of current therapies by reversing resistance pathways. By translating complex molecular insights into tangible therapeutic strategies, epigenetics is poised to reshape the future of HCC management, offering renewed hope for more durable and personalised treatment responses in a disease where progress is urgently needed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ca914bc3e6f4b196e21231ee5479dd3a9a6ded8" target='_blank'>
              Epigenetic therapies in hepatocellular carcinoma: emerging clinical tools and applications.
              </a>
            </td>
          <td>
            Barbara Bueloni, Maite G Fernandez-Barrena, Esteban J. Fiore, Matías A. Avila, J. Bayo, Guillermo Mazzolini
          </td>
          <td>2025-09-23</td>
          <td>Gut</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99f0c6259c21cf5580851e190063781e3ca6d347" target='_blank'>
              Cancer subclone detection based on DNA copy number in single-cell and spatial omic sequencing data.
              </a>
            </td>
          <td>
            Chi-Yun Wu, Jiazhen Rong, A. Sathe, Paul R Hess, Billy T Lau, Susan M. Grimes, Sijia Huang, Hanlee P Ji, Nancy R. Zhang
          </td>
          <td>2025-09-01</td>
          <td>Nature methods</td>
          <td>1</td>
          <td>18</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains a major global challenge, with growing attention to its pathogenesis as mediated by the gut microbiome and epigenetic regulation. Despite therapeutic progress, clinical management remains difficult. CRC accounts for ~10% of cancers and is the second leading cause of cancer death worldwide. Romania bears a substantial burden, with many diagnoses at advanced stages. Etiology—Integrated Genetic, Environmental, and Microbial Determinants. Hereditary syndromes explain 10–15% of cases; most are sporadic, with hypermutated MSI/POLE (~15%), non-hypermutated chromosomal instability (~85%), and a CpG island methylator phenotype (~20%). GWAS implicate loci near SMAD7, TCF7L2, and CDH1; in Romania, SMAD7 rs4939827 associates with risk. Lifestyle exposures—high red/processed meat, low fiber, adiposity, alcohol, and smoking—shape susceptibility. Microbiome–Epigenome Interactions. Dysbiosis promotes carcinogenesis via genotoxins (e.g., colibactin), hydrogen sulfide, activation of NF-κB/STAT3, barrier disruption, and epigenetic remodeling of DNA methylation and microRNAs. Fusobacterium nucleatum, enterotoxigenic Bacteroides fragilis, and pks+ Escherichia coli exemplifies these links. Population-Specific Risk—Romania within Lifestyle–Microbiome Evidence. Incidence is rising, including early-onset disease. Romania lacks CRC-specific microbiome datasets. However, metabolic cohorts show loss of butyrate producers, enrichment of pathobionts, and SCFA imbalance—patterns that mirror European CRC cohorts—and exhibit regional heterogeneity. Beyond Fusobacterium nucleatum. Additional oncobacteria shape tumor biology. Peptostreptococcus stomatis activates integrin α6/β4→ERBB2–MAPK and can bypass targeted inhibitors, while Parvimonas micra enhances WNT/β-catenin programs and Th17-skewed immunity. Together, these data support a systems view in which microbial cues and host epigenetic control jointly drive CRC initiation, progression, metastasis, and treatment response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b63cfd4181c0a26226fd793b68b2268313fd047" target='_blank'>
              Gut Microbiome-Mediated Genetic and Epigenetic Alterations in Colorectal Cancer: Population-Specific Insights
              </a>
            </td>
          <td>
            Simona Turcu, F. Grama, M. Gazouli
          </td>
          <td>2025-09-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract DNA double-strand breaks (DSBs) represent critical events in genome integrity, arising from both endogenous cellular processes and exogenous factors. These breaks are implicated in various genomic aberrations and chromosomal rearrangements, leading to cancers and genetic disorders. Common and rare fragile sites, containing repetitive elements and non-B DNA structures, are particularly prone to breakage under replication stress, which play a pivotal role in cancer development and genetic diseases. Accurate quantification of DNA breaks in the context of repetitive sequences such as microsatellites or non-B DNA structures is technically challenging. We have been comparing four different methods to reliably quantify DSBs in repetitive DNA, namely Southern blot, DSB-PCR, real-time DSB-qPCR, and digital PCR (dPCR). We show here that dPCR offers enhanced sensitivity and specificity compared to other methods. This provides significant applications for future disease diagnosis, understanding molecular mechanisms generating chromosomal breakage and for the development of gene therapies for microsatellite expansion disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccab30f2a498f797c04595f2b9078ba3ca2f3603" target='_blank'>
              Fast and accurate quantification of double-strand breaks in microsatellites by digital PCR
              </a>
            </td>
          <td>
            Cécile Palao, Adèle Kovacs, M. Teixeira, G. Richard
          </td>
          <td>2025-08-09</td>
          <td>Biology Methods & Protocols</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92a0745094dde86c826e8220fdb2990989d2b040" target='_blank'>
              Artificial Intelligence-Guided Molecular Determinants of PI3K Pathway Alterations in Early-Onset Colorectal Cancer Among High-Risk Groups Receiving FOLFOX
              </a>
            </td>
          <td>
            M. D. Fernando C. Diaz, B. Waldrup, Ph.D Francisco G. Carranza, Sophia Manjarrez, M. Enrique Velazquez-Villarreal
          </td>
          <td>2025-08-21</td>
          <td>None</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Dysregulated amino acid metabolism creates cancer-specific vulnerabilities. Neuroblastoma tumors have dysregulated arginine metabolism that renders them sensitive to systemic arginine deprivation. Arginase therapy has been proposed as a therapeutic approach for neuroblastoma treatment and has a favorable safety profile in pediatric cancer patients, however optimal therapeutic combinations remain unexplored. Methods The anti-tumor effects of BCT-100, a pegylated human arginase, were studied in neuroblastoma cell models by metabolite profiling, proteomics, and viability, clonogenicity, and protein translation assays. BCT-100 efficacy was assessed in the Th-MYCN transgenic neuroblastoma mouse model and in neuroblastoma cell line and patient-derived xenograft models. Results In vitro, depletion of arginine by BCT-100 arrested protein translation and cellular proliferation, with effects on clonogenicity enhanced in combination with standard-of-care chemotherapeutics SN-38/temozolomide and mafosfamide/topotecan. In vivo, BCT-100 treatment spared liver arginine while significantly depleting plasma and tumor arginine in Th-MYCN mice, and extended tumor latency (> 100 vs. 45.5 days) in mice pre-emptively treated at weaning. In mice with established tumors, BCT-100 prolonged tumor progression delay when combined with standard-of-care chemo- (> 90 vs. 25 days) or chemo-immuno-therapy (49.5 vs. 35.5 days). Tumor progression delay was also observed in cell line and patient-derived xenografts with BCT-100 treatment, including relapsed/refractory disease models. No increased toxicity was observed with the addition of BCT-100 to established therapies. Conclusions The arginase BCT-100 profoundly disrupts neuroblastoma growth in vitro and in vivo, an effect enhanced in combination with standard-of-care chemo-immuno-therapy. Our data supports further assessment of arginine-depleting combination therapies as a new treatment strategy for neuroblastoma. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03502-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/987476ebc9c7b054d8efdce32b4f05b8e11dc8af" target='_blank'>
              Arginine depletion potentiates standard-of-care chemo-immunotherapy in preclinical models of high-risk neuroblastoma
              </a>
            </td>
          <td>
            Kimberley M. Hanssen, Jayne E. Murray, Ruby Pandher, S. Alfred, Laura D. Gamble, Jennifer Brand, Erin Mosmann, Frances K Kusuma, Crystal Mak, Adam Kearns, Alvin Kamili, C. Atkinson, Alexis Z. Minchaca, Jean Bertoldo, David S. Ziegler, Francis Mussai, Paul N Cheng, M. Norris, Jamie I. Fletcher, Michelle Haber
          </td>
          <td>2025-08-14</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Cancer risk increases with age, and cellular senescence may be a major contributor to cellular carcinogenesis. Enormous efforts have been made to investigate the interrelation between aging and tumors, but little is known about the comparative features of normal aging, cellular senescence, and cancer at single-cell resolution. By integrating analyses of genomics, epigenomics, and bulk and single-cell transcriptomics, we revealed a directionally opposite transcriptional profile between cellular senescence and tumorigenesis at the single-cell level, which may be affected by epigenomic regulations. A total of 648 aging-dependent senescence-associated coregulated modules (SACMs), disproportionately affecting the reproductive systems of both females and males, were initially defined across 17 tissues. Single-cell analysis revealed that aging primarily affects endothelial cells, followed by T cells, epithelial cells, macrophages, and fibroblasts. Opposite directions of change in gene expression between aging and cancer can commonly be observed in endothelial, fibroblast, and epithelial cells, which may prompt the opposing patterns of gene expression between tissue aging and epithelial carcinoma at the bulk level. A similar pattern of expression can be observed in immune cells, which are characterized by decreased self-renewal with aging, but this pattern is reversed in epithelial carcinoma. Our study highlighted the role of senescence as a natural barrier against tumor formation and supported the idea that aging-related systemic environment changes create a protumorigenic milieu.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af4073352af61f280e6d1b0c8dc5f8cd7b466fe5" target='_blank'>
              Integrative Transcriptomic Analysis Decodes the Interplay Between Aging, Senescence, and Cancer.
              </a>
            </td>
          <td>
            Jiale Cai, Deng Wu, Dahua Xu, Fujin Liu, Peihu Li, Dehua Zheng, Zhizhou Xu, Meng Cao, Yutong Shen, Nihui Zhang, Guanghui Tian, Chengyi Fan, Bo Wang, Kongning Li, Xiaoman Bi
          </td>
          <td>2025-09-03</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Breast cancer, including triple-negative breast cancer (TNBC), is the most commonly diagnosed cancer in women, with subtypes differing in treatment options and prognoses. In particular, TNBC, characterized by the absence of estrogen receptor (ER), progesterone receptor (PR), and human epidermal growth factor receptor 2 (HER2) expression, is the most aggressive subtype, with limited treatment options, high metastatic rates, and poor survival outcomes. In recent years, epigenetic studies have emerged as a promising tool for analyzing gene expression and alterations in TNBC, providing potential insights into the development of novel therapeutic strategies. Epigenetic mechanisms, such as DNA methylation, histone modifications, and non-coding RNA (ncRNA)-mediated gene silencing, play a crucial role in the development and progression of TNBC. Research into these mechanisms holds significant promise for the development of personalized therapeutic approaches, potentially improving outcomes for TNBC patients. This review provides a comprehensive overview of recent advances in research on epigenetic alterations in TNBC, with an emphasis on potential clinical applications aimed at improving survival and quality of life in TNBC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14bb8d18aee143e8dc709e4d257b0093ce8c0e92" target='_blank'>
              Epigenetic alterations in triple-negative breast cancer and their clinical implications for diagnosis and therapy
              </a>
            </td>
          <td>
            Eun-Jin Go, Ji Hoon Oh
          </td>
          <td>2025-09-22</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is a particularly aggressive and therapeutically challenging subtype of breast cancer, defined by the lack of estrogen receptor, progesterone receptor, and human epidermal growth factor receptor 2 expression. This absence of actionable molecular targets contributes to its resistance to conventional treatments. This review provides an overview of the mechanistic functions, interrelated processes, and therapeutic implications of several programmed cell death (PCD) pathways—including apoptosis, pyroptosis, necroptosis, autophagy, and ferroptosis—in the context of TNBC pathogenesis and treatment. A conceptual framework is proposed for leveraging these interconnected cell death pathways as a basis for novel targeted interventions. Given the complex interplay among various PCD forms characterized by shared features such as inflammation, mitochondrial dysfunction, and overlapping molecular mediators, this integrated network offers promising opportunities for combinatorial therapeutic strategies. Modulation of one cell death pathway may influence others, potentially amplifying therapeutic efficacy. Furthermore, these PCD pathways are highly relevant to immunotherapy outcomes, offering a foundation for synergistic treatment modalities. This review provides an in-depth analysis of the crosstalk between immune-based therapies and PCD, along with a comprehensive discussion of derived therapeutic approaches. However, tumor diversity, resistance mechanisms, and discrepancies between preclinical models and human physiology pose major challenges in applying these findings clinically. The overarching goal is to present innovative insights and strategies to enhance the clinical management of TNBC and ultimately improve patient outcomes. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cffafc754f749a880744d1044e91f332bd05583" target='_blank'>
              Programmed cell death in triple-negative breast cancer
              </a>
            </td>
          <td>
            Yaqi Liu, Jinwei He, Jialu Chen, Tianshun Chen, Wei Li, Zhihui Yang, Fancai Zeng
          </td>
          <td>2025-10-02</td>
          <td>Cellular & Molecular Biology Letters</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Over the last decade, immunotherapy has revolutionized lung cancer treatments, particularly in non-small cell lung cancer, where immune checkpoint inhibitors have achieved significant clinical success. However, high percentages of patients do not respond initially or eventually develop a resistance to these therapies. This review explores the evolution and challenges of immunotherapy in lung cancer, highlighting its clinical milestones and intrinsic and extrinsic resistance mechanisms. We investigate tumor-intrinsic resistance factors, including alterations in antigen presentation, the loss of Beta-2 microglobulin function, impaired interferon signaling, immune editing, epigenetic modifications, and tumor-extrinsic resistance, such as an immunosuppressive lung tumor microenvironment, dysregulated cytokine profiles, and the upregulation of immune checkpoints. Then, we focus on the emerging role of resistance biomarkers and the development of personalized treatment strategies to overcome these challenges. The complex interplay between tumor biology and immune modulation in lung cancer paves the way for novel approaches for improving the effectiveness of immunotherapeutic treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be521ad9dbd5fb1b3f963d8423b3bfabe7d4614d" target='_blank'>
              Unraveling Resistance in Lung Cancer Immunotherapy: Clinical Milestones, Mechanistic Insights, and Future Strategies
              </a>
            </td>
          <td>
            Maria Vitale, Raffaella Pagliaro, G. Viscardi, Lucio Pastore, Giuseppe Castaldo, F. Perrotta, Susan F M Campbell, Andrea Bianco, Filippo Scialò
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="BCL-2 is a central regulator of apoptosis and inhibits cell death by sequestering pro-apoptotic BH3 alpha-helices within a hydrophobic surface groove. While venetoclax, a BH3-mimetic drug, has transformed the treatment of BCL-2–driven malignancies, its efficacy is increasingly limited by acquired resistance mutations that disrupt small-molecule binding yet preserve anti-apoptotic function—reflecting a remarkable structural adaptation. Here, we employ hydrocarbon-stapled alpha-helices derived from the BAD BH3 motif as conformation-sensitive molecular probes to investigate this therapeutic challenge. The stapled peptides not only retain high-affinity binding to all BCL-2 variants but also show enhanced potency to select venetoclax-resistant mutants. Structural analyses, including X-ray crystallography and hydrogen-deuterium exchange mass spectrometry (HDX MS), demonstrate that these stapled helices restore native BH3 engagement by reversing the conformational consequences of resistance mutations. Notably, we identify a serendipitous interaction between the α3–α4 region of BCL-2 and hydrocarbon staple, which further compensates for altered groove conformation and contributes to mutant binding affinity. Together, these findings offer mechanistic insights into BCL-2 drug resistance and reveal a blueprint for designing next-generation inhibitors that overcome this clinically significant barrier to durable treatment responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c989e0417fad14adefa4b6808df4b60e2e5666a7" target='_blank'>
              Structural insights into chemoresistance mutants of BCL-2 and their targeting by stapled BAD BH3 helices
              </a>
            </td>
          <td>
            Thomas M. DeAngelo, Utsarga Adhikary, Kyle J. Korshavn, , Clara R. Brotzen-Smith, Christina M Camara, S. Dhe-Paganon, Gregory H. Bird, T. Wales, L. Walensky
          </td>
          <td>2025-09-29</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="ABSTRACT: CRISPR (Clustered Regularly Interspaced Short Palindromic Repeats) technology has emerged as a groundbreaking tool for genome editing, revolutionizing the management of genetic diseases. Its precision, efficiency, and adaptability enable targeted modification of DNA sequences, offering therapeutic potential for a wide range of inherited disorders. This technology utilizes the Cas9 endonuclease, guided by RNA, to introduce site-specific cuts in the genome, facilitating the correction, disruption, or replacement of genes. Applications include addressing monogenic disorders such as cystic fibrosis, sickle cell anemia, and Duchenne muscular dystrophy, as well as polygenic conditions and diseases linked to somatic mutations, including certain cancers. Furthermore, CRISPR has opened avenues for ex vivo and in vivo gene therapies, including the use of stem cells and viral vectors for effective delivery. While significant ethical and technical challenges remain, such as the risk of unintended genetic modifications and equitable access to therapy, CRISPR represents a paradigm shift in the treatment of genetic diseases. These abstract highlights the current state, challenges, and prospects of CRISPR technology as a transformative tool in precision medicine.
Keywords: CRISPR; precision medicine; monogenic disorders; polygenic conditions">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dddc887f1058108c924a6d20f7fb8e66ce8c29aa" target='_blank'>
              Application of genetic scissoring in the management of genetically related diseases
              </a>
            </td>
          <td>
            A. Ogunlakin, O. Olusegun, Samuel Olusegun, D. Anejukwo, Olubunmi Asaleye, Holiness Balogun, Oluwadamilola Adedoyin, O. Ojo, O. Ogunlakin, M. Sonibare
          </td>
          <td>2025-08-29</td>
          <td>Dietary Supplements and Nutraceuticals</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Drug tolerant persisters (DTPs) drive cancer therapy resistance by temporarily evading drug action, allowing multiple routes to eventual permanent resistance. Despite clear evidence for DTPs, the timing of their emergence, proliferative nature, and how their population dynamics arise from measured single-cell kinetics remain poorly understood. Here we use time-lapse microscopy data from two cancer cell lines, integrating single-cell and population measurements, to develop a quantitative description of drug persistence. Contrary to the expectation that increasing levels of genotoxic stress should lead to slower times to division and faster times to death, we observe minor changes in the single-cell intermitotic and death time distributions upon increasing cisplatin concentration. Yet, population decay rates increase 3-fold, suggesting a surprising independence of the overall dynamics from the measured birth and death rates. To explain this phenomenon, we argue that the observed lineage correlations and concentration-dependent decay rates imply cell-state dependent fate choices made both pre and post-cisplatin as opposed to just post-drug birth/death rate-based competitive fate choices. We demonstrate that these cell-states, present in the ancestors of DTP and sensitive cells, exhibit no difference in cycling speed and are inherited across 2-3 cellular generations. A stochastic model implementing these rules simultaneously recapitulates the observed decay rates and cell-fate correlations, also explaining how pre-drug fate decisions are consistent with barcoding experiments where barcode diversity remains unchanged after drug administration. Our results provide a powerful perspective on drug tolerance based on general arguments, without requiring knowledge of the underlying molecular architecture of the heterogeneous cell states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b499c056c673d79d24c4d2c5acdf399c0d8012aa" target='_blank'>
              Inheritable cell-states shape drug-persister correlations and population dynamics in cancer cells
              </a>
            </td>
          <td>
            Anton Iyer, Adrián E. Granada, Shaon Chakrabarti
          </td>
          <td>2024-11-03</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Mutagenesis is an effective strategy to generate genetic resources for functional genomics and crop improvement. It is widely accepted that chemical-induced mutations are randomly distributed across the genome. However, the randomness of such mutations remains a matter of scientific debate due to a lack of systematic investigation. Although the basic features of mutations induced by different mutagens are known, it remains unresolved whether these mutations correlate with the chromatin state and gene expression. In this study, genomes of the wild-type and 4,619 mutagenized M3 lines derived from treatment with two concentrations of ethyl methane sulfonate (EMS) are sequenced, identifying over 7 million single-nucleotide mutations (SNPms) that covered over 97% of genes in the rice genome. Comprehensive analyses integrated with genomic, epigenomic, and expression data revealed that SNPms are more frequently located in open chromatin regions and actively expressed genes. Within expressed genes, heterozygous SNPms are enriched both upstream and downstream of transcription start sites (TSSs), whereas homozygous SNPms are enriched significantly in the upstream regions. These findings demonstrate that EMS-induced mutations are non-randomly distributed across the rice genome, with a preference for functionally important genomic regions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/260e5cecc1b9786aa38924a6ed3852eca46cabe4" target='_blank'>
              Non-Random Distribution of EMS-Induced Mutations Reveals Preference for Open Chromatin and Expressed Genes in Rice.
              </a>
            </td>
          <td>
            Xuefeng Yao, Yanhong Liu, Zhiyong Li, Guo-Qiang Jiang, Wensheng Wang, Hong Lu, Huihui Li, Zefu Lu, Chun-Ming Liu
          </td>
          <td>2025-08-13</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Neuroendocrine prostate cancer (NEPC) is a treatment-resistant subtype that arises through lineage plasticity, allowing tumor cells to bypass androgen receptor (AR)–targeted therapies. In a recent study, Lu et al. define a transcriptional and epigenetic hierarchy that drives this transdifferentiation. The pioneer factor FOXA2 initiates enhancer remodeling and regional DNA demethylation, while the neural lineage transcription factor NKX2-1 is required to complete the NE program. Together, these factors reorganize 3D chromatin architecture and activate lineage-specific genes through enhancer–promoter looping. Crucially, the histone acetyltransferase p300/CBP is essential cofactor in this process. Pharmacologic inhibition of p300/CBP with CCS1477 suppresses NE gene expression and impairs tumor growth in NEPC models. These findings offer a mechanistic insight of lineage plasticity and highlight p300/CBP as promising therapeutic targets. The study also raises key questions about the stability and reversibility of chromatin remodeling and sets a framework for understanding enhancer-driven plasticity in other cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3331955821f5eab1873b6edce226793e7651773a" target='_blank'>
              FOXA2 and NKX2-1 Orchestrate Neuroendocrine Lineage Plasticity in Prostate Cancer
              </a>
            </td>
          <td>
            Ka-wing Fong
          </td>
          <td>2025-10-01</td>
          <td>Serican Journal of Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="PARP inhibitors exploit synthetic lethality in BRCA1/2-mutated ovarian cancers but are limited by emerging therapeutic resistance. Therefore, novel biomarkers predicting PARP inhibitor response are urgently needed. In this study, we performed integrative analysis using drug sensitivity, patient survival, gene dependency, and expression data to identify biomarkers associated with PARP inhibitor response in ovarian cancer. Mutations in BRCA1, MLL2, NF1, and SMARCA4 were associated with increased sensitivity to PARP inhibitors, suggesting potential synthetic lethality with PARP1. In contrast, SMAD4 mutations were linked to PARP inhibitor resistance, and low SMAD4 expression was associated with poor overall survival in patients with ovarian cancer. Further gene dependency score (GDS)-based screening revealed 51 candidate genes potentially involved in SMAD4-mediated resistance. Functional enrichment revealed associations with stress response, tumor-associated signaling pathways, and additional processes. Subsequent correlation and survival analyses nominated ACACA, PRPF4B, and TUBD1 as potential therapeutic targets. Notably, low ACACA expression in patients with low SMAD4 expression was associated with improved survival, indicating its relevance in overcoming PARP inhibitor resistance. This study contributes to predicting clinical outcomes in ovarian cancer and developing personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d80b49672c28e84e41d8bee90691985b721fb554" target='_blank'>
              Identification of Anticancer Targets in Ovarian Cancer Using Genomic Drug Sensitivity Data
              </a>
            </td>
          <td>
            Yebin Son, Jae Yong Ryu
          </td>
          <td>2025-09-29</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC), the most prevalent primary liver cancer, continues to pose a significant global health burden due to its high mortality rate. In addition to genetic alterations, epigenetic aberrations, including DNA methylation, histone modifications, chromatin remodeling, and noncoding RNA (ncRNA) dysregulation, play critical roles in HCC initiation and progression. Notably, miR-375 and miR-483-5p are among the most dysregulated miRNAs in HCC, with their altered expression levels closely associated with tumor stage and patient survival. These epigenetic modifications offer promising therapeutic avenues due to their reversibility and dynamic nature. Furthermore, specific epigenetic signatures such as CDH1 promoter hypermethylation and HOTAIR overexpression are being explored as potential biomarkers for early detection and treatment response. In this chapter, we review recent advances in the epigenetic landscape of HCC and discuss their diagnostic and therapeutic implications, highlighting their potential to improve patient outcomes through personalized medicine approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e32b3133cdd46cb1e5dedf967c8b74c8ee431edc" target='_blank'>
              Epigenetic Alterations in Hepatocellular Carcinoma: Mechanisms, Biomarkers, and Therapeutic Implications
              </a>
            </td>
          <td>
            A. Wali, Abid Reza Ansari, P. Mir, Mohamed El-Tanani, Rasha Babiker, M.D.S.A.D.I.Q.U.E. Hussain, Jasreen Uppal, Asma Ishrat Zargar, R. H. Mir
          </td>
          <td>2025-08-27</td>
          <td>Pharmaceuticals</td>
          <td>1</td>
          <td>16</td>
        </tr>

        <tr id="Intratumor heterogeneity is a hallmark of cancer, enabling subpopulations of cells to evade therapy, adapt to immune attack, and thrive in diverse microenvironments. While retrospective genomic and epigenomic analyses have mapped the large-scale histories of tumor evolution, they cannot capture the rapid, dynamic changes in cell state that occur as individual cells divide. Panagopoulos and colleagues recently developed a cellular platform to monitor the role of transient replication stress in real time, tracking sister cells as they divide and replicate. The authors use these techniques to show that daughter and granddaughter cells can inherit very different states, often leading to further cellular instability. This work broadens our understanding of how diverse cell states arise from oncogenic stress and how cellular heterogeneity emerges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/beadadba41d03604ec2c3cef748b8a0844e1482c" target='_blank'>
              Generations in Flux: Tracking the Birth of Cellular Heterogeneity in Cancer.
              </a>
            </td>
          <td>
            Aaron McKenna
          </td>
          <td>2025-08-18</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Genomic antigen loss is a recurring mechanism of resistance to chimeric antigen receptor T-cell (CAR-T) and T-cell engagers (TCE) in relapsed/refractory multiple myeloma (RRMM). Yet, it remains unclear whether these events are acquired under treatment or merely selected from pre-existing, undetectable clones. By leveraging chemotherapy mutational signatures as temporal barcodes within whole genome sequencing data, we could time genomic antigen escape in 4 out of 11 RRMM patients. In all cases, the biallelic loss was driven by genomic events acquired after exposure to BCMA- and GPRC5D-targeted CAR-T/TCE, and not present at baseline. Longitudinal digital PCR analysis corroborated that resistance mutations were undetectable at therapy initiation but emerged preceding relapse. Among 752 newly diagnosed patients only 2.7% and 9% had monoallelic inactivation of TNFRSF17 and GPRC5D, respectively, with no biallelic loss. Our findings suggest limited utility of mutational screening prior to CAR-T/TCE, while underscoring the importance of dynamic surveillance during therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/481837333886a5830de7147acdc462180a9ceabf" target='_blank'>
              Timing genomic antigen loss in multiple myeloma treated with T-cell redirecting immunotherapies.
              </a>
            </td>
          <td>
            M. Papadimitriou, Sungwoo Ahn, B. Diamond, Holly Lee, J. McIntyre, M. Truger, Michael A Durante, B. Ziccheddu, Katalin Osz, O. Landgren, L. Rasche, Nizar J. Bahlis, Paola Neri, F. Maura
          </td>
          <td>2025-09-04</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6acdfefa7d1e9916c129010f57a43699739821df" target='_blank'>
              Cancer gene therapy: historical perspectives, current applications, and future directions.
              </a>
            </td>
          <td>
            Jingyi Zeng, Jingwen Luo, Yingchun Zeng
          </td>
          <td>2025-09-29</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="INTRODUCTION
Metastatic colorectal cancer (mCRC) remains a leading cause of cancer mortality worldwide, with limited long-term survival despite therapeutic advances. The increasing understanding of its molecular heterogeneity has paved the way for precision medicine approaches aiming to optimize treatment efficacy and reduce unnecessary toxicity.


AREAS COVERED
This review provides an in-depth analysis of the current and emerging molecular targets in mCRC, including RAS, BRAF, HER2, and microsatellite instability. We discuss the clinical relevance of tumor sidedness, hyperselection panels, EGFR ligand expression, and rare alterations such as NTRK, RET, and ALK fusions. The review also explores the evolving role of KRAS G12C inhibitors, HER2-targeted therapies, and the application of liquid biopsy - particularly circulating tumor DNA (ctDNA) - in treatment monitoring, rechallenge strategies, and resistance detection. Literature was selected through a comprehensive review of recent clinical trials, consensus guidelines, and translational studies.


EXPERT OPINION
Personalized treatment is now an attainable goal in mCRC. While promising, broader implementation of molecular-driven strategies requires overcoming challenges such as resistance mechanisms, assay standardization, and equitable access. The integration of innovative agents with real-time molecular monitoring holds the key to a more dynamic and effective management of mCRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68d84552ed9fbf737b1d558e20dc110dcd8268ee" target='_blank'>
              Personalized therapy in metastatic colorectal cancer: biomarker-driven use of biologics.
              </a>
            </td>
          <td>
            F. Sullo, Chiara Gallio, Laura Matteucci, Alessandro Bittoni, M. Muratore, Luca Esposito, Bianca Ceredi, Graziana Gallo, Paola Ulivi, I. Rapposelli, A. Passardi
          </td>
          <td>2025-09-04</td>
          <td>Expert opinion on biological therapy</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="This review provides a comprehensive synthesis of current knowledge on immunotherapy resistance in non-small cell lung cancer (NSCLC), a disease that accounts for approximately 85% of all lung cancer cases and remains the leading cause of cancer-related death worldwide. Although immune checkpoint inhibitors (ICIs) have significantly improved survival for a subset of patients with advanced NSCLC, over 70% of cases ultimately exhibit primary or acquired resistance, underscoring the urgent need to understand the underlying mechanisms. The review categorizes resistance into tumor-intrinsic and tumor-extrinsic processes and provides an in-depth mechanistic analysis of how factors such as tumor antigen loss, impaired antigen presentation, cGAS-STING pathway dysregulation, metabolic reprogramming in tumor microenvironment (TME), immune cell exhaustion, and microbiomes collectively contribute to immune escape. In parallel, the influence of the lung and gut microbiome on shaping immunotherapy responses is discussed, with emphasis on microbial dysbiosis, immunosuppressive metabolite production, and TME remodeling. Therapeutic strategies to overcome resistance are also discussed, including combination approaches involving chemotherapy, radiotherapy, and antiangiogenic agents, as well as epigenetic modulators (HDAC and BET inhibitors). Moreover, the review explores bispecific antibodies, antibody-drug conjugates, and small-molecule agents that enhance T cell function or disrupt immunosuppressive signaling networks. By integrating insights from preclinical models and clinical trials, the review underscores the necessity of biomarker-guided patient stratification, combination immunotherapy approaches, and interventions that restore tumor immunogenicity. It concludes that a multipronged therapeutic strategy, one that addresses both immune evasion and TME-induced suppression, holds the greatest promise for improving response durability and advancing personalized immunotherapy for NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3df76f6889b4c3ea7f132198d5c6301465fae771" target='_blank'>
              Immunotherapy resistance in non-small cell lung cancer: from mechanisms to therapeutic opportunities
              </a>
            </td>
          <td>
            Huiyu Wang, Xiaomin Niu, Zhenning Jin, Shaoxing Zhang, Rong Fan, Hua Xiao, Shen S. Hu
          </td>
          <td>2025-08-23</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9095797e23ad8944a80f33c6e2aa42554f1acad9" target='_blank'>
              Uncovering Cellular Interactome Drivers of Immune Checkpoint Inhibitor Response in Advanced Melanoma Patients
              </a>
            </td>
          <td>
            Shay Ladd, Anne M. Talkington, Mary O’Sullivan, R. Barnes, Remziye E. Wessel, Gabriel F. Hanson, Sepideh Dolatshahi
          </td>
          <td>2025-09-08</td>
          <td>Cellular and Molecular Bioengineering</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f305a3d3aa4c5343ff379224a4358725aa81e74e" target='_blank'>
              ACSS2-Mediated Metabolic-Epigenetic Crosstalk Drives Fulvestrant Resistance and Represents a Novel Therapeutic Target.
              </a>
            </td>
          <td>
            A. Mogol, Jin Young Yoo, A. Eve, Mahima Goel, David J Dutton, Claire P. Schane, Audrey Lam, Debapriya Dutta, Betsy Barnick, Eda D Erdogan, Erik R. Nelson, Maria Grosse-Perdekamp, Zeynep Madak Erdogan
          </td>
          <td>2025-09-23</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Colorectal cancer (CRC), as a highly prevalent malignant tumor worldwide, has a persistently high incidence and mortality rate. In recent years, metabolic reprogramming and immunosenescence have received extensive attention as key mechanisms for tumorigenesis, development and treatment resistance. Metabolic reprogramming not only provides energy and biosynthetic precursors for tumor cells, but also regulates immune responses by reconstructing the tumor microenvironment (TME). Immunosenescence is characterized by the depletion of effector immune cell function and the increase in the proportion of immunosuppressive cells. The two jointly promote the immune escape and therapeutic resistance of CRC. This article systematically reviews the research progress of metabolic reprogramming and immunosenescence in colorectal cancer and explores the related targeted therapeutic strategies, aiming to provide a new theoretical perspective for the precise treatment of CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5118ecb0b3c3e7bd3760fc9c69d9e13a24fb67dc" target='_blank'>
              Metabolic reprogramming and immunosenescence in colorectal cancer: mechanisms and therapeutic implications
              </a>
            </td>
          <td>
            Jun Zhu, Kuan Shen, Qiang Su, Guozhong Yao, Xiaochen Chen
          </td>
          <td>2025-08-13</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is a highly aggressive subtype of breast cancer, lacking the expression of estrogen receptor, progesterone receptor, and HER2, which leads to poor prognosis and limited treatment options. Despite advances in targeted therapies, TNBC patients often fail to benefit due to its heterogeneity, drug resistance, and immune evasion. Gene editing technologies such as CRISPR/Cas9 and RNA interference offer promising strategies for precise gene modulation, yet their clinical translation is challenged by delivery efficiency, off-target effects, and safety concerns. This review systematically summarizes key regulatory genes implicated in TNBC progression, including those involved in invasion, proliferation, and chemoresistance, such as BRCA1/2, TP53, MUC1, EGFR, MYC, and others. We further discuss advances in gene editing tools and their combination with targeted delivery materialsranging from viral vectors (AAV, lentivirus) to nonviral platforms (lipid nanoparticles, polymer-based systems, and bioderived vesicles)highlighting their potential to enhance editing specificity, minimize immune response, and overcome tumor microenvironment barriers. Finally, we address biosafety, ethical concerns, and mitigation strategies such as high-fidelity Cas9 variants and AI-assisted off-target prediction. Collectively, this review provides a comprehensive framework for future research and clinical application of targeted gene editing in TNBC, aiming to develop safer, more effective, and personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01340f159e29a0963fbf18ee30b93bcf3d8bf597" target='_blank'>
              Application of Gene Editing Technology Based on Targeted Delivery Materials in TNBC
              </a>
            </td>
          <td>
            Peng Qu, Xue Li, Jun Liu, Xiaohan Su, Shiqi Han, Yali Wang, Yunbo Luo, Jinsui Li, Cui Ma, Shishan Deng, Lingmi Hou, Qi Liang, Panke Cheng
          </td>
          <td>2025-09-08</td>
          <td>ACS Omega</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Although xenotransplantation has been revolutionized by the development of genome-edited pigs, it is still unknown whether these pigs and their offspring remain genomically stable. Here, we show that GGTA1-knockout (GTKO) pigs accumulated an average of 1205 genome-wide genetic mutations, and their filial 1 (F1) offspring contained an average of 18 de novo mutations compared with wild-type controls and their parents. The majority of mutations were in regions annotated as intergenic regions, without altering protein functions, and were not located at predicted off-target mutation sites. RNA-sequencing analysis and phenotype observations indicated that the accumulated mutations may have only a limited influence on GTKO pigs and most of the mutations in the GTKO pigs could be attributed to the electrotransfection of plasmids into cells. This is the first report that genetic mutations in genome-edited pigs are inherited stably by the next generation, providing a reference for the safe application and a standard approach to breed genome-edited pigs for xenotransplantation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/218c09acaaa59ca5db66362a81d2de17f5d053aa" target='_blank'>
              Genome-wide Genetic Mutations Accumulated in Pigs Genome-edited for Xenotransplantation and Their Filial Generation.
              </a>
            </td>
          <td>
            X. Huo, Xianhui Sun, Xiangyang Xing, Jing Lu, Jingjing Zhang, Yanyan Jiang, Xiao Zhu, Changlong Li, Jianyi Lv, Meng Guo, Lixue Cao, Xin Liu, Zhenwen Chen, Dengke Pan, Shunmin He, Chen Zhang, Xiaoyan Du
          </td>
          <td>2025-08-20</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d07528eee7ff3d627e8787aab243a7521b7075a4" target='_blank'>
              Targeting p53-MDM2 pathway with novel triazole-oxazole hybrids: a fragment-based drug discovery approach for next-generation cancer therapies.
              </a>
            </td>
          <td>
            Apurva Prajapati, Hitesh Patel
          </td>
          <td>2025-09-23</td>
          <td>Molecular diversity</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="CRISPR-based technologies have revolutionized plant science by enabling precise modulation of gene function, including CRISPR activation (CRISPRa), a recently emerging strategy which shows particular promise for enhancing disease resistance through targeted gene upregulation. Unlike conventional CRISPR editing, which introduces double-stranded DNA breaks and permanent genomic changes, CRISPRa employs a deactivated Cas9 (dCas9) fused to transcriptional activators. This system allows quantitative and reversible gene activation without altering the DNA sequence, offering a gain-of-function (GOF) like enhanced blight resistance in staple crops. Despite its potential, the limited adoption of CRISPRa in plant biology to date underscores the need for future studies to fully harness its capabilities for crop improvement. This review addresses the groundbreaking and relatively underexplored potential of CRISPR activation (CRISPRa) systems for GOF studies in plant biology, and advocates for the adoption of CRISPRa to discover and harness genetic variation for enhancing disease resistance. We present recent advancements in CRISPRa technology, emphasizing its successful application in boosting plant immunity. Moreover, we discuss the synergistic potential of integrating CRISPRa with functional genomics tools such as genome-wide association studies (GWAS) and multi-omics approaches to identify and characterize key resistance genes. Additionally, we highlight ongoing progress in developing plant-specific programmable transcriptional activators (PTAs) to optimize CRISPRa efficiency. Challenges associated with achieving transgene-free overexpression and the deployment of alternative CRISPR systems are also explored. Together, these advances position CRISPRa as a transformative tool for future crop breeding strategies aimed at achieving durable, broad-spectrum disease resistance and sustainability in agriculture.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b94e737bcc6c850c85b75ac8cfe89554d5b60f5b" target='_blank'>
              CRISPR activation: identifying and using novel genes for plant disease resistance breeding
              </a>
            </td>
          <td>
            John E. McLaughlin, Idrice Carther Kue Foka, Michael A. Lawton, Rong Di
          </td>
          <td>2025-09-08</td>
          <td>Frontiers in Genome Editing</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly lethal and immunologically "cold" cancer, with a 5-year survival rate of 13%. Despite the success of immunotherapy in other malignancies, its efficacy in PDAC remains limited due to complex immune evasion mechanisms and a pro-tumorigenic immune microenvironment. Several lines of evidence suggest that PDAC progresses from acinar-to-ductal metaplasia (ADM) to pancreatic intraepithelial neoplasia (PanIN) lesions to invasive cancer, accompanied by increasing immunosuppression. We hypothesize that understanding the spatial and temporal transitions of immune cell states during tumor evolution is key to uncovering mechanisms of immune dysfunction and guiding new therapeutic strategies. Using single-cell RNA sequencing (scRNA-seq) and multispectral imaging, we aim to uncover comprehensive, large-scale, and longitudinal changes in the immune microenvironment during PDAC progression. Here, we utilized the KrasG12D/+; Trp53R172H/+; Ptf1a-Cre (KPC) genetically engineered mouse model, and through longitudinal histological analysis of pancreas, we delineated the trajectory of disease progression, with PanIN I-II lesions appearing by 8–9 weeks, PanIN III by 11–12 weeks, and locally invasive PDAC by 16 weeks of age. To study the temporal evolution of immune cell-state transitions, we performed scRNA-seq on CD45+ enriched immune cells isolated from KPC mouse pancreas collected at 4, 8, 12, and 16 weeks using the 10x Genomics Chromium platform. To examine the spatial dynamics of immune cells, we performed multispectral imaging at defined time points using the Lunaphore COMET platform with a panel of over 15 antibodies, enabling simultaneous visualization of several markers on a single tissue section. In our analysis to date, we identified subsets of pro-tumorigenic macrophages emerging during PDAC development, along with a marked increase in myeloid-derived suppressor cells (MDSCs) in late-stage tumors. We also observed dynamic shifts in other immune cell populations evolving alongside tumor progression. Notably, we uncovered distinct subsets of monocytes, macrophages, and T cells exhibiting tumor-promoting potential, progressively enriched during the advanced stages of PDAC. Using multispectral imaging, we generated a comprehensive spatial map of the immune microenvironment across PDAC progression, capturing patterns of immune exclusion, infiltration, and cell–cell neighborhood interactions. Our study highlights the multi-layered immune dysfunction that evolves from early to late stages of PDAC - features that are challenging to capture in human samples. CD45+ enrichment improved the detection of rare and transient immune subsets by mitigating the masking effects caused by abundant cancer and stromal cells. Together, these findings provide an unbiased and detailed view of the evolving immune landscape in PDAC, laying the foundation for identifying microenvironmental and cell-intrinsic factors that either constrain or promote malignant progression, as well as for developing immune-based therapeutic strategies.



 Atul Verma, Doreen Klingler, Linda Bergmayr, Tanvi V. Inamdar, Jonas Rosendahl, Michael Böttcher, Stefan Hüttelmaier, Patrick Michl, Philipp Jurmeister, Sebastian Krug. Tracing the immune landscape: Temporal and spatial remodeling during pancreatic cancer progression [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A106.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28e0f49d549158981070f5d866ac858fc605d13b" target='_blank'>
              Abstract A106: Tracing the immune landscape: Temporal and spatial remodeling during pancreatic cancer progression
              </a>
            </td>
          <td>
            Atul Verma, Doreen Klingler, Linda Bergmayr, T. V. Inamdar, Jonas Rosendahl, Michael Böttcher, Stefan Hüttelmaier, Patrick Michl, Philipp Jurmeister, Sebastian Krug
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01fb40ddbc348d6bdf921a2ddd2696429a42111c" target='_blank'>
              Energy Deficiency-Induced ATG4B Nuclear Translocation Inhibits PRMT1-Mediated DNA Repair and Promotes Leukemia Progression.
              </a>
            </td>
          <td>
            Zhenkun Wang, Xianli Zhang, Yuanyuan Zhou, Shuyi Lin, Yuting Fu, Haiping Liu, Qiongdan Gao, YanHong Xiao, Zhao Yin, Shuang Liu, Kexiu Huang, Suqi Deng, Juan Du, Hui Zeng, Jia Wang, Jinping Zheng, Qinghua Zhou, Tianfeng Chen, Xiaoling Gao, Zhenyu Ju, Bo Liu
          </td>
          <td>2025-08-11</td>
          <td>Advanced science</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="ABSTRACT Cancer remains the most lethal disease globally, despite the significant progress made in early screening, surgery, and therapeutic development in recent decades. Programmed cell death (PCD) is a genetically regulated process essential for eliminating aberrant cells, yet its dysregulation drives tumorigenesis and therapy resistance. In this review, we present a complete discovery timeline of them and comprehensively synthesize the roles and mechanisms of major PCD forms, such as apoptosis, necroptosis, autophagy, pyroptosis, ferroptosis, and cuproptosis, across diverse cancer types. We not only detail the molecular mechanisms, dual functions, and alterations of these PCD modalities in cancers, but also summarize their interconnections and intrinsic crosstalk. Furthermore, we comprehensively discuss how diverse therapies, including chemotherapy, radiotherapy, immunotherapy, targeted agents, and hormone therapy, engage and manipulate specific PCD pathways, revealing the involvement of PCD in cancer treatment mechanisms. This review integrates extensive preclinical and clinical evidence on PCD‐targeted therapies with an in‐depth focus on ferroptosis, including its regulatory networks and therapeutic relevance. Special emphasis is placed on prostate cancer, highlighting the PCD‐based translational opportunities in this common malignancy. Taken together, we provide novel insights into the complex interplay between PCD and cancer biology and offer a framework for developing precision oncology therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1c45427c1d2eb5ce79e949eeb0b16e4625cc2c1" target='_blank'>
              Programmed Cell Death in Cancer
              </a>
            </td>
          <td>
            Yuang Wei, William Hankey, Dongliang Xu, Fuwen Yuan
          </td>
          <td>2025-08-31</td>
          <td>MedComm</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id=" 
Background: Accurate and early identification of lung cancer biomarkers is crucial for effective management yet remains a clinical challenge. CRISPR-based technologies provide highly sensitive and specific tools that can be adapted for both diagnostic and therapeutic use in oncology.
Methods: A prospective study was conducted including 120 patients with confirmed lung cancer and 60 healthy individuals as controls at a tertiary care center. A CRISPR-Cas12a diagnostic platform was developed to detect EGFR exon 19 deletions, KRAS G12C mutations, and ALK rearrangements in tumor tissue and plasma samples. In parallel, CRISPR-Cas9 editing was applied to patient-derived lung cancer cell cultures to selectively disrupt EGFR mutant alleles. The diagnostic performance of the CRISPR assay was compared against conventional PCR and next-generation sequencing (NGS).
Results: The CRISPR-Cas12a assay successfully detected as few as 10 DNA copies/µL within one hour. In tissue samples, the assay achieved 96.7% sensitivity and 98.3% specificity relative to NGS. In circulating tumor DNA extracted from plasma, sensitivity reached 92.1%. Functional assays demonstrated that CRISPR-Cas9-mediated disruption of EGFR mutations reduced cell viability by approximately 65% and suppressed downstream oncogenic signaling pathways.
Conclusion: CRISPR-based strategies demonstrate considerable promise for both precise detection and targeted intervention in lung cancer. The integration of CRISPR diagnostics with gene-editing applications could significantly advance precision oncology by enabling earlier diagnosis and patient-tailored therapeutic options.
 ">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b66e666da19106418ca11e5bfbe956b66a8dd301" target='_blank'>
              CRISPR-BASED APPROACHES FOR THE DETECTION AND TARGETING OF LUNG CANCER BIOMARKERS: A NEW ERA IN PRECISION ONCOLOGY
              </a>
            </td>
          <td>
            Ibtasam Mazhar, Maimona Sadia, Muhammad Hassam Saleem, Ateeqah Siddique, Areeba Musferah, Areej Safdar, Anood Malik
          </td>
          <td>2025-08-31</td>
          <td>Insights-Journal of Health and Rehabilitation</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Genetic alterations in p53 (TP53) are found in most solid malignancies, including pancreatic cancer (PANC). Mutant p53 (p53mut) contributes to cancer progression and metastasis. Existing therapeutic options for p53mut cancers are ineffective and cause toxic side effects, stressing the need for better therapies. To address these clinical problems, we developed a two-drug therapeutic strategy that selectively targets p53mut cancers. This novel strategy combines a thymidine analogue (trifluorothymidine (TFT), a component of TAS102, acting as an inducer of DNA damage), and an inhibitor of poly (ADP) ribose polymerase (PARPi) that works as an amplifier of DNA damage. Incorporation of TFT into DNA provokes post-replicative repair generating single-strand DNA break intermediates. Repair of these intermediates is assisted by PARP, while inhibition of PARP increases more lethal double-strand DNA breaks. In normal p53 wild-type (WT) cells, TAS102 and PARPi activate p53-dependent G1/S checkpoint, limiting DNA damage and promoting repair. In p53mut cells with compromised G1/S checkpoint, TAS102 and PARPi cooperate as the inducer-amplifier pair to increase lethal DNA breaks and cell death. The TAS102-PARPi strategy was validated in preclinical studies. Our first-in-human phase I study with TAS102-talazoparib regimen (NCT04511039) showed that this regimen is safe with promising efficacy. The current work investigated the mechanisms underlying the response to TAS102-PARPi regimen in p53mut cancer cells. Our data show that TAS102-PARPi induces the G2/M checkpoint mediated by ATR kinase in a p53-independent manner. ATR acts through the G2-checkpoint kinases (CHK1 and WEE1) to inactivate CDK1 and stop transition to mitosis. The transcriptome analysis showed that TAS102-PARPi treatment increased expression of Claspin and inactivated PLK1-driven mitotic pathway. Claspin is critical for activation of DNA damage and replication checkpoints by facilitating ATR-CHK1 signaling, whereas PLK1 mediates inactivation of the G2-checkpoint kinases and degradation of Claspin for normal cell cycle progression and re-entry into mitosis. Thus, the PLK1-Claspin network may play a key regulatory role in G2-checkpoint in response to TAS102-PARPi by restraining premature reentry into cell cycle of cells with unrepaired DNA. The next major question was whether TAS102-PARPi regimen can cooperate with drugs disrupting G2-checkpoint. New data show that the efficacy of TAS102-PARPi therapy is greatly enhanced by a sequential application of G2-kinase WEE1 inhibitor. Our data show that this sequential triple-drug strategy works in a synthetic-lethality manner to cause massive cell death in p53mut cancers. The triple-drug strategy greatly improved tumor control in PANC patient-derived xenografts with no signs of major toxicities to normal tissues. In summary, this study provides the scientific foundation for our novel sequential triple-drug strategy that demonstrates greater efficacy against pancreatic cancer with lower toxicity.



 Andrei Bakin, Mohammed Alruwaili, Ashley Guo, Thomas Melendy, Barbara Foster, Christos Fountzilas. Effective targeting p53 mutant pancreatic cancer by novel drug-combination strategy leveraging their DNA damage liabilities [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B001.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a93f85c005599dc813a5323d69a7be402a3898b5" target='_blank'>
              Abstract B001: Effective targeting p53 mutant pancreatic cancer by novel drug-combination strategy leveraging their DNA damage liabilities
              </a>
            </td>
          <td>
            A. Bakin, M. Alruwaili, Ashley Guo, T. Melendy, Barbara Foster, C. Fountzilas
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The alkylating agents temozolomide (TMZ) and lomustine (CCNU) are the most effective systemic agents for malignant gliomas. However, resistance - whether intrinsic or acquired - inevitably develops in all patients, and these tumors remain incurable. Although CCNU has demonstrated clinical benefit, its clinical use has been relatively limited due to a less favorable safety profile compared to TMZ. Recently, interest in CCNU and other nitrosoureas has been renewed in light of positive clinical trials in both adult and pediatric gliomas. Despite this renewed attention, critical questions remain unaddressed regarding the use of nitrosoureas. While resistance and response to temozolomide have been associated with the status of both MGMT and the mismatch repair DNA repair pathway, our understanding of the unique mechanisms of resistance to nitrosoureas beyond MGMT remains limited. Recent advances in molecular biology, preclinical models and the use of longitudinal analyses of treated samples offer new insights, offering opportunities to refine the clinical and develop novel strategies. In this review, we explore the current role of nitrosoureas in glioma treatment, examine known and emerging mechanisms of sensitivity and resistance to these agents, and explore potential for combination approaches to enhance their efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0c17d81fca992371795e4b4faab3b073a663913" target='_blank'>
              Revisiting an old therapy for new, promising combinations: biology and perspectives of lomustine in glioma treatment.
              </a>
            </td>
          <td>
            C. Bellamy, Cynthia Hajal, Keith L. Ligon, M. Touat
          </td>
          <td>2025-08-13</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cebb7efc870646ab91862d4a77c390cd760bc112" target='_blank'>
              Dual targeting of CDK6 and LSD1 is synergistic and overcomes differentiation blockade in AML.
              </a>
            </td>
          <td>
            Lise M Brault, E. Voisset, Mathieu Desaunay, Antonia Boudet, Paraskevi Kousteridou, S. Letard, Nadine Carbuccia, A. Goubard, Rémy Castellano, Yves Collette, J. Vernerey, Isabelle Vigon, J. Pasquet, Patrice Dubreuil, Sophie Lopez, P. de Sepulveda
          </td>
          <td>2025-08-29</td>
          <td>EMBO molecular medicine</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Epigenetic regulation is critical to B cell development, guiding gene expression via DNA methylation, histone modifications, chromatin remodeling, and noncoding RNAs. In mature B cell neoplasms, particularly diffuse large B cell lymphoma (DLBCL), follicular lymphoma (FL), and chronic lymphocytic leukemia (CLL), these mechanisms are frequently disrupted. Recurrent mutations in key epigenetic regulators such as EZH2, KMT2D, CREBBP, and TET2 lead to altered chromatin states, repression of tumor suppressor genes, and enhanced oncogenic signaling. Dysregulation of specific microRNAs (e.g., miR-155, miR-21) further contributes to pathogenesis and therapeutic resistance. In DLBCL, hypermethylation of SMAD1 and CREBBP mutations are associated with immune evasion and chemoresistance. In FL, EZH2 gain-of-function and KMT2D loss-of-function mutations alter germinal center B cell programming, while in CLL, DNA hypomethylation patterns reflect the cell of origin and correlate with clinical outcome. Targeted therapies such as the EZH2 inhibitor tazemetostat have demonstrated efficacy in EZH2-mutant FL, while HDAC and BET inhibitors show variable responses across B cell malignancies. The limitations of current epigenetic therapies reflect the complexity of targeting epigenetic dysregulation rather than therapeutic futility. These challenges nonetheless highlight the relevance of epigenetic alterations as biomarkers and therapeutic targets, with potential to improve the management of mature B cell neoplasms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a068788c20e74398875dae829a5f27eeefbe604a" target='_blank'>
              Unraveling the Epigenetic Landscape of Mature B Cell Neoplasia: Mechanisms, Biomarkers, and Therapeutic Opportunities
              </a>
            </td>
          <td>
            Nawar Maher, F. Maiellaro, J. Ghanej, S. Rasi, R. Moia, G. Gaidano
          </td>
          <td>2025-08-22</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="SUMMARY Metastasis is an emergent continuum, driven by evolving reciprocal adaptations between continuously disseminating tumor cells (DTCs) and the specialized metastatic niches of distant organs. The interplay between intrinsic and niche-driven mechanisms that enables DTCs to survive and home to distant organs remains incompletely understood. Here, using MetTag, a single-cell barcoding and transcriptome profiling approach with time-stamped batch identifiers (BC.IDs), we mapped temporal, clonal dynamics of DTCs and the immune cell landscape across ovarian cancer metastatic niches. Deep sequencing of barcodes revealed preferred enrichment of early-disseminated clones across metastatic niches. Mechanistically, single-cell RNA sequencing (scRNA-seq) coupled with velocity analyses in ascites and metastasis-bearing omenta uncovered an emergent, distinct interferon-gamma (IFNγ) centric transcriptional trajectory, enriched among early seeding clones. Moreover, in vivo CRISPR/Cas9 screening of metastatic niche-specific signatures demonstrated that genes belonging to the ascites IFNγ signature, including Marco, Gbp2b, and Slfn1, are functionally important for peritoneal metastasis. Knockout of IFNγ receptor 1 (Ifngr1) in tumor cells significantly reduced metastatic burden and extended survival, underscoring the importance of tumor cell intrinsic IFNγ signaling in ovarian cancer metastasis. Furthermore, we identified that the tumor intrinsic IFNγ response and ascites-derived tumor-associated macrophages (TAMs) protect cancer cells from anoikis-mediated death within the IFNγ-rich ascites environment. Our study resolves temporal dynamics of disseminating tumor cells and highlights an ascites-driven, IFNγ program as a necessary pro-metastatic adaptation in the ovarian metastasis cascade.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a95e0d9496ced48f75294cbc20541bbba656bbf" target='_blank'>
              Temporal Clonal Tracing and Functional Perturbation Reveal Niche-Adaptive and Tumor-Intrinsic IFNγ Dependencies Driving Ovarian Cancer Metastasis
              </a>
            </td>
          <td>
            Emilija Aleksandrovic, Shaneann Fross, Samantha M. Golomb, Xiyu Liu, Zhuo Zhao, Nikitha M. Das, T. Reese, Wei Ma, Jacqueline Lopez, M. S. Stack, Min Zhao, Siyuan Zhang
          </td>
          <td>2025-08-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Replication stress (RS) poses a threat to genome stability and drives genomic rearrangements. The homologous recombination (HR) pathway repairs stalled replication forks (RFs) and prevents such instability. Through an E3 ubiquitin ligase screen aimed at identifying regulators of RAD51, we identified macrophage erythroblast attacher (MAEA), a core component of C-terminal to Lish (CTLH) E3 ubiquitin ligase complex, as a regulator of the HR pathway. Loss of MAEA impairs RAD51 recruitment at stalled RFs, leading to increased sensitivity to RS-inducing agents and excessive degradation of nascent DNA strands. Mechanistically, MAEA associates with and mediates the ubiquitylation of Ku80, enabling its removal from RF ends and facilitating the loading of RAD51. Notably, MAEA deficiency is associated with a developmental disorder involving microcephaly, craniofacial abnormalities, ocular defects, and heart malformations. Functional assays show that disease-linked MAEA variants (R34C, E349G, Y394D, and M396R) are defective in RS response. These findings establish MAEA as an essential factor in RF protection and genome integrity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c499707b53080fc3870cbe7d66d1176114a45109" target='_blank'>
              Pathogenic variants in MAEA disrupt DNA replication fork stability and are associated with developmental abnormalities in humans
              </a>
            </td>
          <td>
            Elham Zeinali, Fatemeh Mashayekhi, Rabih Abou Farraj, Talah Hasanni, Marie-christine Caron, Yan Coulombe, A. Fitieh, J. N. M. Glover, Jean-Yves Masson, Ismail Hassan Ismail
          </td>
          <td>2025-08-29</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Human papillomavirus (HPV) is a major etiological factor in cervical, anal, and oropharyngeal cancers. Although prophylactic vaccines have substantially reduced infection rates, effective therapeutic options for established HPV-associated malignancies remain limited. This review provides an up-to-date overview of emerging strategies to treat HPV-driven tumours. Key approaches include immune checkpoint inhibitors, therapeutic vaccines such as VGX-3100 and PRGN-2012, and gene-editing tools like CRISPR/Cas9. Epigenetic drugs, particularly histone deacetylase inhibitors, show promise in reactivating silenced tumour suppressor genes and enhancing antitumour immunity. In addition, natural bioactive compounds and plant-derived molecules are being explored as complementary anti-HPV agents, while drug repurposing and combination therapies offer cost-effective opportunities to broaden treatment options. We also highlight the role of patient-derived organoid models as powerful platforms for personalized drug screening and functional assessment. By integrating these therapeutic innovations with precision oncology approaches, this review outlines a multidimensional framework aimed at improving clinical outcomes and quality of life for patients with HPV-associated cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c470c11ec7ad7e2174f0a27714b34305b11d10e" target='_blank'>
              Emerging Therapeutic Strategies for HPV-Related Cancers: From Gene Editing to Precision Oncology
              </a>
            </td>
          <td>
            M. O. Cakir, Guldide Kayhan, B. Yilmaz, Mustafa Ozdogan, G. H. Ashrafi
          </td>
          <td>2025-09-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Treatments for high-grade serous ovarian cancer (HGSOC) are initially effective but most invariably fail. Although they can successfully suppress the bulk of the tumour cell population, residual cancer cells can enter alternative therapy-resistant cell fates highlighted by proliferative arrest. Understanding the nature of these fates and how cells may resume uncontrolled proliferation will lead to the development of new treatments for HGSOC. In this study, we examine the response of HGSOC cells to standard of care cisplatin chemotherapy and to the RNA Polymerase I transcription inhibitor CX-5461/Pidnarulex, two drugs that elicit a potent DNA damage response and growth arrest. Here, we identify that HGSOC cells exposed to these therapies show multiple hallmarks of therapy-induced senescence (TIS) and derive a core TIS gene expression signature irrespective of genetic background or senescence trigger. Given that TIS is a potentially escapable state, we have performed a focussed drug screen to identify drugs that eradicate senescent HGSOC cells. We identify that therapy-induced senescent HGSOC cells, including those with decreased sensitivity to senolytic drugs that inhibit the pro-survival protein BCL-XL, can be eliminated using drugs that induce ferroptosis, an iron-dependent form of cell death. Mechanistically, we demonstrate that senescent HGSOC cells have altered expression of regulators of iron metabolism leading to intracellular iron overload that underpins this targetable vulnerability. Together, we highlight elevated levels of iron as a TIS biomarker in HGSOC and the potential of inducing ferroptosis to eradicate residual HGSOC cells following initial therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3787e084ffe3e100d6ec83768df6b1bb600d9e4a" target='_blank'>
              Exploiting dysregulated iron homeostasis to eradicate persistent high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            Carmelo Cerra, Madeleine R. C. Tancock, N. Thio, Ada Koo, AnnRann Wong, Karla J Cowley, Swati Varshney, Madelynne O. Willis, Kaylene Simpson, D. Bowtell, E. Sanij, Elizabeth L Christie, Richard B. Pearson, Jian Kang, Keefe T. Chan
          </td>
          <td>2025-09-25</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Prostate cancer (PCa) remains a major global health concern, ranking among the most prevalent cancer in men worldwide. Despite the availability of various therapeutic options, the clinical efficacy of current anti-PCa agents is often compromised by drug resistance and adverse effects. DNA minor groove binders offer a potential therapeutic alternative, owing to their selective mechanism of action and favorable safety profiles. In the present study, we utilized a multiomics strategy to investigate the molecular impact of novel compound MGB4. LNCaP cells were treated with doxorubicin, MGB4, or a combination of both, followed by LC-MS/MS-based untargeted proteomics and metabolomics analyses. One-way ANOVA (p-value <0.05) revealed 55 significantly dysregulated proteins and 57 altered metabolites across treatments. Our findings indicate that both MGB4 and doxorubicin impacted key cellular pathways, including inhibition of translation and alterations in sphingolipid and amino acid metabolism, while doxorubicin and the combination therapy also reduced spermine and spermidine metabolism. Notably, the combined treatment exhibited synergistic effects, significantly impacting purine metabolism and reducing metabolite levels more than individual therapies. This study provides key molecular insights into MGB4 and doxorubicin's mechanisms, supporting MGB4 as a potential prostate cancer drug candidate.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0baef9cfc704d2fe10207a5529ebf62ac3bfbc81" target='_blank'>
              Untargeted Multiomics of LNCaP Cell Line Treated with a Novel DNA Minor Groove Binder and/or Doxorubicin Using Mass Spectrometry.
              </a>
            </td>
          <td>
            Ruba A Zenati, Nelson C. Soares, Hasan Y. Alniss, Hamza M. Al-Hroub, Raafat El-Awady, A. Abuhelwa, Wafaa S Ramadan, Shereen M. Aleidi, W. El‐Huneidi, Eman Abu-Gharbieh, K. Alzoubi, Yasser K. Bustanji, M. Semreen
          </td>
          <td>2025-09-04</td>
          <td>Journal of proteome research</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Background Pancreatic ductal adenocarcinoma (PDAC) is one of the most lethal malignancies, with limited treatment options and poor prognosis. Recent advances in cancer genomic analysis enable the identification of actionable gene alterations, opening new opportunities for personalized therapy. Among these, homologous recombination DNA repair (HRR) gene alterations are associated with distinct biological behavior, favorable prognosis, and increased sensitivity to platinum-based chemotherapy. However, the prognostic impact of coexisting mutations in key driver genes—KRAS, TP53, CDKN2A, and SMAD4—within HRR-altered PDAC remains poorly understood. Methods We retrospectively analyzed PDAC patients who underwent genomic profiling testing with FoundationOne® CDx between June 2019 and December 2021 through the Center for Cancer Genomics and Advanced Therapeutics (C-CAT) database. We compared the prevalence and prognostic significance of key gene alterations between HRR-altered and HRR–wild-type (WT) tumors. Results Of 2,381 PDAC patients, 274 (11.5%) harbored HRR alterations. These patients showed significantly longer overall survival (OS) than those with HRR-WT tumors (HR = 0.66, p = 0.002). The frequencies of KRAS, TP53, and CDKN2A mutations were less frequent in HRR-altered tumors. TP53 mutation was independently associated with poorer OS across both HRR subgroups, while CDKN2A alteration was a poor prognostic factor in HRR-WT tumors. Interestingly, SMAD4 alteration was linked to improved survival in the HRR-altered group. Conclusion HRR-altered PDAC has a distinct genomic profile and is associated with a favorable prognosis. Our findings demonstrate that coexisting alterations are significant prognostic factors in both HRR-altered and HRR–wild-type tumors. These results highlight the clinical relevance of incorporating comprehensive genomic profiling into routine care to stratify patient prognosis better and inform individualized treatment strategies in PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ada42c90fd2263fd4d9bb1466e612ccff4b0a39f" target='_blank'>
              Prognostic impact of gene alterations via homologous recombination DNA repair gene alteration status in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Yusuke Kawanaka, Chiaki Inagaki, Masaki Okura, S. Mitani, T. Takahama, K. Yonesaka, Y. Chiba, Kazuhiko Nakagawa, H. Kawakami, H. Hayashi
          </td>
          <td>2025-08-25</td>
          <td>Frontiers in Medicine</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="


 Cisplatin remains a standard first-line therapy for epithelial ovarian cancer; however, the development of chemoresistance leads to poor prognosis and high recurrence rates. Kaplan–Meier analysis of patient data (TCGA) revealed that cisplatin-sensitive tumors are associated with significantly improved overall survival, underscoring the need for strategies to overcome resistance in clinical settings. To investigate the underlying mechanisms, we conducted a comprehensive bioinformatics analysis of publicly available GEO datasets from cisplatin-treated ovarian cancer patients (n = 255). This analysis identified six key molecular drivers of resistance: ZBTB33 (Kaiso), pregnane X receptor (PXR), NF-κB, HER2 (ERBB2), P-glycoprotein (P-gp/ABCB1), and HIF1A. These targets were validated in ovarian tumor specimens via immunohistochemistry, confirming elevated expression in chemo-resistant disease. Additionally, the quantitative real-time PCR analysis confirms the transcriptional upregulation of the six resistance-associated genes in cisplatin-resistant SKOV3 and OVCAR-5 ovarian cancer cells, consistent with the IHC findings. The average fold change in mRNA transcripts ranged from 2.4 for P-glycoprotein (P-gp) to 5 for both NF-kB and Kaiso (ZBTB33). Although less well studied in ovarian cancer, Kaiso (ZBTB33) is known to regulate EMT and tumor invasion in other solid tumors. Functional studies using SKOV3 and OVCAR-5 cell lines demonstrated that knockdown of Kaiso (ZBTB33) via RNA interference significantly increased cisplatin-induced cell death, indicating a direct role in therapeutic resistance. Furthermore, we investigated the synergistic effects of combining SDA, a plant-based omega-3 fatty acid known to inhibit NF-κB, and cisplatin on the cell death of SKOV3 and OVCAR-5 cell lines, comparing the results to those of each compound used individually. Interestingly, co-treatment with stearidonic acid (SDA) synergistically enhanced the cytotoxicity of cisplatin at a lower dose in both cell models. These findings reveal a clinically relevant resistance signature and highlight the therapeutic potential of combinatorial strategies that target both transcriptional regulators (e.g., Kaiso) and inflammatory signaling (e.g., NF-κB). Dual targeting of these pathways may resensitize tumors to cisplatin and improve outcomes for patients with advanced ovarian cancer.



 Mahmoud Mansour, Sabrina Van Ginkel, Maaz Kamal Alata, Ibrahim Bani, Isra Elhussin. Molecular signature of cisplatin resistance in ovarian cancer identifies therapeutic opportunities for re-sensitization [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A025.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a654db1a318364cee4d66d47d1f28ac0d3c685d" target='_blank'>
              Abstract A025: Molecular signature of cisplatin resistance in ovarian cancer identifies therapeutic opportunities for re-sensitization
              </a>
            </td>
          <td>
            Mahmoud Mansour, Sabrina Van Ginkel, M. Alata, Ibrahim Bani, Isra Elhussin
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Abstract Through natural or synthetic lineage barcodes, single-cell technologies now enable the joint measurement of molecular states and clonal identities, providing an unprecedented opportunity to study cell fate and dynamics. Yet, most computational methods for inferring cell development and differentiation rely exclusively on transcriptional similarity, overlooking the lineage information encoded by lineage barcodes. This limitation is exemplified by T cells, where subtle transcriptional differences mark divergent fates with distinct biological activity. Single-cell RNA and matched TCR sequencing is now ubiquitous in the analysis of clinical samples, where the TCR sequence provides an endogenous clonal barcode and could reveal clonal T cell responses. We present Clonotrace, a computational framework that jointly models gene expression and clonotype information to infer cell state transitions and fate biases with higher fidelity. While motivated by challenges in analyzing T cell populations, especially in the tumor microenvironment and immunotherapy settings, Clonotrace is broadly applicable to any lineage-barcoded single-cell dataset. Across diverse systems including T cells, hematopoietic differentiation, and cancer therapy resistance models, Clonotrace reveals differentiation hierarchies, distinguishes unipotent from multipotent states, and identifies candidate fate-determining genes driving lineage commitment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0987fea6e48fd7cd3a2b34fbb805d81b9c4a798" target='_blank'>
              Deciphering Cell Fate and Clonal Dynamics via Integrative Single-Cell Lineage Modeling
              </a>
            </td>
          <td>
            Nancy R Zhang, Yuntian Fu, Divij Mathew, Mingshuang Wang, Xinyi Chen, Kevin Z Lin, Dylan Schaff, Sydney M Shaffer, Drew Pardoll, Christina Jackson
          </td>
          <td>2025-09-24</td>
          <td>Research Square</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Purpose The alternative splicing of mRNA precursors allows one gene to yield multiple proteins with distinct functions. CDC-like kinases serve as pivotal regulators of alternative splicing. Control of protein expression also occurs at the level of DNA through histone methylation and demethylation. We investigated the activity of two CLK inhibitors, cirtuvivint and CC-671, and the LSD1 inhibitor iadademstat alone and in combination with anticancer drugs or investigational agents. Methods Well-characterized patient-derived cancer cell lines from the PDMR (https://pdmr.cancer.gov/models/database.htm) were used along with standard human cancer cell lines. Multi-cell type-tumor spheroids were grown from a ratio of 6:2.5:1.5 malignant cells, endothelial cells, and mesenchymal stem cells. Following three days of growth, the spheroids were exposed to the single agents or combinations at concentrations up to the clinical Cmax value for each agent, if known. After seven days of exposure, cell viability was assessed using the CellTiter-Glo 3D assay and spheroid volume was assessed by bright field imaging. Results Several of the targeted oncology drugs exhibited additive and greater-than-additive cytotoxicity when combined with a CLK inhibitor, or the LSD1 inhibitor. These agents included the XPO1 inhibitor, eltanexor, and the KRAS G12D specific inhibitor MRTX-1133 which had activity in tumor lines harboring the KRAS G12D mutation. LSD1 inhibition was effective with ubiquitin proteasome pathway inhibitors. Conclusion: These findings may provide guidance for development of clinical trial combination regimens including cirtuvivint, CC-671 or iadademstat. Full data sets are available on PubChem.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbd17fd73c746ccc133073188d85cd1b611794f0" target='_blank'>
              RNA processing kinase inhibitors and epigenetic inhibitors in combination with oncology drugs or investigational agents in multi-cell type patient-derived tumor cell line spheroids
              </a>
            </td>
          <td>
            Beverly A. Teicher, T. Dexheimer, Thomas Silvers, N. Coussens, Eric M Jones, Steven D. Gore, Mark W. Kunkel, James H. Doroshow
          </td>
          <td>2025-09-12</td>
          <td>Cancer Chemotherapy and Pharmacology</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Whole-genome duplication (WGD) represents an evolutionarily conserved process occurring in prokaryotes, eukaryotes, and somatic mammalian tissues. While developmentally programmed WGD supports normal tissue regeneration, unscheduled WGD drives chromosomal instability and oncogenic progression in cancer. Recent studies have clarified dual roles of WGD across physiological homeostasis and disease pathogenesis. Here, we review the prevalence of WGD, the molecular mechanisms driving its major causes and its biological consequences. In addition, we highlight recent advancements in WGD detection, including both conventional cytogenetic techniques and newly developed high-throughput sequencing approaches. The integration of multi-omics and machine learning further improves ploidy analysis, particularly in cancer research. Together, these insights establish WGD as a critical regulator of development, regeneration, and disease and underscore the importance of emerging computational and sequencing tools for its precise characterization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ebbe235d853294a695ca1852f4aab18a1bed1ef" target='_blank'>
              The dual role of whole-genome duplication: biological mechanisms, functional consequences, and detection advances.
              </a>
            </td>
          <td>
            Yawei Song, Jiajie Yang, Shuheng Wu, Wei Wu
          </td>
          <td>2025-09-01</td>
          <td>Acta biochimica et biophysica Sinica</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10f3ac7d8f1f9723125587e98e6c28a4657680b4" target='_blank'>
              Integrated control of cancer stemness by σ1 receptor in advanced prostate cancer.
              </a>
            </td>
          <td>
            G. Civenni, G. Sandrini, Jessica Merulla, C. Musumeci, Elisa Federici, Arianna Vallegra, A. Kokanovic, Simone Mosole, D. Shinde, Elisa Sorrenti, Alyssa J J Paganoni, Martina Marchetti, Riccardo Valzelli, D. Albino, Matteo Pecoraro, Andrea Rinaldi, Marco Bolis, Roger Geiger, Tobias Winge, Catharina Holtschulte, E. Laurini, S. Pricl, G. M. Carbone, Bernhard Wünsch, C. Catapano
          </td>
          <td>2025-09-02</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Cancer is a global health threat, with its treatment modalities transitioning from single therapies to integrated treatments. This paper systematically explores the key technological systems in modern cancer treatment and their application value. Modern cancer treatment relies on four core technological pillars: omics, bioinformatics, network pharmacology (NP), and molecular dynamics (MD) simulation. Omics technologies integrate various biological molecular information, such as genomics, proteomics and metabolomics, providing foundational data support for drug research. But the differences in data and the challenges of integrating it often lead to biased predictions, and that’s a big limitation for this technology. Bioinformatics utilizes computer science and statistical methods to process and analyze biological data, aiding in the identification of drug targets and the elucidation of mechanisms of action. It is important to note that the prediction accuracy largely depends on the algorithm chosen. Consequently, this dependence may affect the reliability of the research results. NP, based on systems biology, studies drug-target-disease networks, revealing the potential for multitargeted therapies. That said, this method may overlook important aspects of biological complexity, such as variations in protein expression. This oversight can lead to overestimating the effectiveness of multi-targeted therapies, resulting in false positives in efficacy assessments, which somewhat limits its practical usefulness. MD simulation examines how drugs interact with target proteins by tracking atomic movements, thus enhancing the precision of drug design and optimization. Nevertheless, this technology faces practical challenges, such as high computational costs and sensitivity of model accuracy to the parameters of the force field. The synergistic application of these technologies significantly shortens the drug development cycle and promotes precision and personalization in cancer therapy, bringing new hope to patients for successful treatment. However, researchers still face challenges like the variability of data. Future efforts need to use Artificial Intelligence (AI) to establish standardized data integration platforms, develop multimodal analysis algorithms, and strengthen preclinical-clinical translational research to drive breakthrough advancements in cancer treatment. With the ongoing technological improvements, the vision of personalized medicine—tailored treatments based on individual patient characteristics—will gradually be realized, significantly enhancing treatment efficacy and improving patients’ quality of life.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22d7169f2229a12330c9accac896702442028fe4" target='_blank'>
              Strategies for the drug development of cancer therapeutics
              </a>
            </td>
          <td>
            Hongyan Liu, Yanpin Ma, Wenjuan Chen, Xinyu Gu, Jiachun Sun, Penghui Li
          </td>
          <td>2025-09-30</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Lethal prostate cancer (PCa) has passed through at least two evolutionary bottlenecks: acquisition of metastatic potential and development of castration resistance. A better understanding of how this impacts genetic heterogeneity across metastatic sites is needed to develop strategies to block metastatic spread and overcome resistance. By leveraging deep whole-exome sequencing of 93 tumors from 26 patients, we examined patterns of metastatic dissemination and clonal evolution of PCa. Phylogenetic reconstruction and mathematical modeling enabled quantification of the number of mutations and clones in the cancer ecosystem and characterization of each patient's disease as an evolutionary process. While mutations of the earliest pathogenic genetic drivers of PCa were truncal, most other likely passenger mutations, copy-number alterations, and clones arose after the cancer had spread and were confined to individual metastatic sites due to polyclonal and polyphyletic seeding. Single-tissue sequencing tended to overestimate mutation clonality and, apart from truncal drivers, underestimate mutation rates for both individual patients and cohorts. This study highlights the independent evolution of metastatic lesions, which has implications for diagnostic and targeted therapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f198ff2332798cb036a19b4692ee525e77c812c" target='_blank'>
              Divergent Clonal Evolution and Early Dissemination Promote Genetic Heterogeneity of Metastases in Castration Resistant Prostate Cancer.
              </a>
            </td>
          <td>
            Noshad Hosseini, R. Mannan, Ryan J. Rebernick, Fengyun Su, Rui Wang, Xuhong Cao, Ana Lako, Dattatreya Mellacheruvu, Jing Hu, J. Alumkal, Zachery R Reichert, Rohit Malik, Rohit Mehra, A. Chinnaiyan, M. Cieslik
          </td>
          <td>2025-08-18</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>169</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1110fcfacea74bd5588618a3f029d0c49019fbe" target='_blank'>
              Mechanisms and genomic implications of break-induced replication.
              </a>
            </td>
          <td>
            Adel Atari, Haoyang Jiang, Roger A. Greenberg
          </td>
          <td>2025-08-22</td>
          <td>Nature structural & molecular biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="ABSTRACT Purpose Metastatic castration-resistant prostate cancer (mCRPC) remains a significant therapeutic challenge and a leading cause of cancer-related mortality in men. PARP inhibitors like Olaparib are effective in homologous recombination repair (HRR)-deficient tumors, but resistance often arises through DNA repair restoration. This study explores the role of the structure-specific endonuclease subunit SLX1, a catalytic subunit of the SLX1-SLX4 endonuclease complex, in Olaparib resistance. Methods Data from The Cancer Genome Atlas (TCGA) were used for expression and survival analyses. The CRPC cell line DU145, which harbors BRCA1 and BRCA2 mutations, was used as a cell model for both in vitro and in vivo studies. Results Elevated SLX1A expression in prostate cancer tissues was associated with significantly reduced progression-free and overall survival. SLX1 protein was upregulated in androgen-resistant prostate cancer cell lines (DU145, 22RV1, PC3) and further increased in Olaparib-resistant DU145 (DU145-OR) cells. Silencing SLX1 via shRNA enhanced Olaparib sensitivity, reducing colony formation and increasing DNA damage and apoptosis in DU145 and DU145-OR cells. Mechanistically, SLX1 knockdown disrupted SLX4 interactions with critical DNA repair proteins (ERCC1-XPF, PLK1, and TOPBP1), impairing DNA repair complex stability. In vivo, SLX1-silenced DU145 xenografts treated with Olaparib showed significantly reduced tumor growth with decreased Ki-67 expression and increased apoptosis/necrosis compared to controls. Conclusion This study highlights SLX1 as both a prognostic marker and potential therapeutic target to enhance PARPi efficacy in advanced prostate cancer. Targeting SLX1 may be a promising strategy to overcome Olaparib resistance in mCRPC patients with homologous recombination deficiency.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02640e28be0a0d352e94d2990883b754339fb49e" target='_blank'>
              SLX1 silencing overcomes Olaparib resistance in metastatic castration-resistant prostate cancer by disrupting SLX4-mediated DNA repair complexes
              </a>
            </td>
          <td>
            Xin Zhao, Shiyun Feng, Xiaoping Nitie, Shibu Muluo, Yi Lei
          </td>
          <td>2025-08-11</td>
          <td>Cancer Biology & Therapy</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Through natural or synthetic lineage barcodes, single-cell technologies now enable the joint measurement of molecular states and clonal identities, providing an unprecedented opportunity to study cell fate and dynamics. Yet, most computational methods for inferring cell development and differentiation rely exclusively on transcriptional similarity, overlooking the lineage information encoded by lineage barcodes. This limitation is exemplified by T cells, where subtle transcriptional differences mark divergent fates with distinct biological activity. Single-cell RNA and matched TCR sequencing is now ubiquitous in the analysis of clinical samples, where the TCR sequence provides an endogenous clonal barcode and could reveal clonal T cell responses. We present Clonotrace, a computational framework that jointly models gene expression and clonotype information to infer cell state transitions and fate biases with higher fidelity. While motivated by challenges in analyzing T cell populations, especially in the tumor microenvironment and immunotherapy settings, Clonotrace is broadly applicable to any lineage-barcoded single-cell dataset. Across diverse systems including T cells, hematopoietic differentiation, and cancer therapy resistance models, Clonotrace reveals differentiation hierarchies, distinguishes unipotent from multipotent states, and identifies candidate fate-determining genes driving lineage commitment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74b1a84299cf4bd86e16928f5b0911b15d09c6d9" target='_blank'>
              Deciphering Cell Fate and Clonal Dynamics via Integrative Single-Cell Lineage Modeling
              </a>
            </td>
          <td>
            Yuntian Fu, D. Mathew, Mingshuang Wang, Xinyi E. Chen, Kevin Z Lin, Dylan Schaff, Sydney M Shaffer, Drew M Pardoll, Christina Jackson, Nancy R Zhang
          </td>
          <td>2025-09-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Chromosomal instability (CIN) and epigenetic reprogramming are central drivers of breast cancer progression, yet the mechanisms connecting them remain elusive. We uncover a direct role of EZH2 histone methyltransferase in promoting CIN in triple-negative breast cancer (TNBC). Across breast cancers, EZH2 expression correlates with copy number alterations, and its catalytic activity is associated with increased CIN in metastasis-initiating cells. Pharmacological EZH2 inhibition suppressed CIN, revealing an unexpected vulnerability. Integrated chromatin and transcriptome profiling identified Tankyrase (TNKS), a poly(ADP-ribose) polymerase, as a direct transcriptional target of EZH2. Mechanistically, EZH2-mediated TNKS suppression disrupts CPAP (centrosomal P4.1-associated protein), driving centrosome overduplication, multipolar mitosis and exacerbated CIN. In vivo, CIN suppression is a critical mechanism underlying the anti-metastatic effects of EZH2 inhibition. These findings delineate a previously unrecognized epigenetic mechanism governing CIN and establish EZH2 inhibitors as the first therapeutic agents capable of directly suppressing CIN, underscoring the need for trials with metastasis-focused endpoints.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5010199d8cc937113c87be8a59e3c3e7e0efc720" target='_blank'>
              Epigenetic regulation of chromosomal instability by EZH2 methyltransferase.
              </a>
            </td>
          <td>
            Yang Bai, Albert S. Agustinus, Shira Yomtoubian, Cem Meydan, Dylan R McNally, Liron Yoffe, Melissa J Hubisz, Marvel Tranquille, Sneha Pramod, Christy Hong, Magdalena L Plasilova, Aakanksha R Kapoor, Arshdeep Singh, Henry G. Withers, Lukas E. Dow, A. Laughney, B. Bhinder, O. Elemento, Ari M Melnick, S. Bakhoum, Vivek Mittal
          </td>
          <td>2025-10-02</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="ABSTRACT Liquid biopsy represents a transformative approach in oncology, enabling noninvasive disease detection and monitoring through epigenetic signals in circulating tumor DNA (ctDNA), nucleosomes, and noncoding RNAs. Tumor initiation is driven by epigenetic modifications, including DNA methylation, histone alterations, and dysregulated noncoding RNAs, which disrupt gene regulation, cell cycle control, DNA repair, and metastatic processes. This review systematically examines recent evidence on DNA methylation, histone marks (e.g., H3K27me3, H3K18ac), and noncoding RNAs (miRNAs, lncRNAs) as biomarkers for early cancer detection, prognosis, and therapeutic response. Particular focus is placed on aberrant DNA methylation (e.g., hypermethylation of CDKN2A, RASSF1A) and altered histone modifications (e.g., EZH2‐mediated silencing) as indicators of tumor heterogeneity and evolution. Stable and specific in biofluids, noncoding RNAs such as oncogenic miR‐21, tumor‐suppressive miR‐34a, and metastasis‐associated MALAT1/HOTAIR further enhance clinical applicability. Recent detection methods, including bisulfite sequencing, ChIP‐seq, and RNA‐seq, have advanced biomarker profiling, though challenges remain in standardization and low‐abundance detection. With over 12 active clinical studies validating their utility, integration of epigenetic markers with AI and multiomics holds promise for individualized, dynamically guided oncology care. Future innovations, such as chromatin accessibility analysis and cfDNA fragmentation profiling, may further refine diagnostic precision and therapeutic monitoring.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77f80572fe0354408f2985b22c2727d7355f5dac" target='_blank'>
              Transforming Cancer Diagnostics: The Emergence of Liquid Biopsy and Epigenetic Markers
              </a>
            </td>
          <td>
            Debalina Saha, Pritam Kanjilal, Mandeep Kaur, Soumya V. Menon, Ayash Ashraf, M. Kumar, T. Alqahtani, Shikha Atteri, D. Uti, Bikram Dhara
          </td>
          <td>2025-09-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Lung squamous cell carcinoma (LUSC) remains a lethal malignancy with limited therapeutic advancements, underscoring the need to identify novel oncogenic drivers. This study integrates multi-omics analyses-including ChIP-seq, bulk RNA-seq, single-cell ATAC/RNA-seq, and spatial transcriptomics-to delineate enhancer-driven transcriptional networks in 59 matched LUSC tumors and adjacent normal tissues. The analyses identify 3,447 tumor-specific oncogenic enhancers (SOEs) enriched for master transcription factors (SOX2, TP63, KLF5, GRHL2) that orchestrate malignant programs. Single-cell epigenomic profiling reveals these SOEs to be exclusively active in a cancer cell-like cluster, pinpointing PTPRZ1, a receptor tyrosine phosphatase, as a top SOE-driven candidate. Functional studies demonstrate that PTPRZ1 is essential for LUSC-tumorigenesis and tumor cells proliferation and migration in vitro and in vivo. Spatial transcriptomic analysis further implicates midkine (MDK) as the ligand activating PTPRZ1 in LUSC tumor tissue. Mechanistically, PTPRZ1 mediates MDK-induced PI3K phosphorylation that is essential for LUSC tumor growth. This work illustrates enhancer addiction as a hallmark of LUSC, identifies the PTPRZ1-MDK axis as an important targetable pathway, and establishes a paradigm for dissecting epigenetic vulnerabilities in solid tumors through multi-omics integration. These findings advance precision oncology strategies for LUSC, bridging epigenomic dysregulation to therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfdd6b0f8500832ade4c7417d38653f9c0588ff5" target='_blank'>
              Enhancer Reprogramming Reveals the Tumorigenic Role of PTPRZ1 in Lung Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Yong-Qiang Ning, Jie Wang, Yihui Zhang, Hongrui Li, Zenan Jiang, Xinyu Su, Xiao Yang, Xiaoxuan Wu, Bing Wei, Jianming Ying, Jian Yan, Wenbin Li
          </td>
          <td>2025-09-03</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a645774843921b1750028307066a9681c4de577" target='_blank'>
              The neural crest-associated gene ERRFI1 is involved in melanoma progression and resistance toward targeted therapy.
              </a>
            </td>
          <td>
            Nina Wang, Qian Sun, Daniel Novák, Lei Zhu, Juliane Poelchen, Tamara Steinfass, Yiman Wang, V. Umansky, Jochen Utikal
          </td>
          <td>2025-10-03</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="The disruption of the epigenetic mechanisms of gene expression regulation due to the emergence of pathogenic variants in genes-encoding elements of epigenetic machinery leads to the development of chromatinopathies. This group of hereditary diseases includes 179 syndromes, some of which present with overlapping phenotypes. Despite the variety of approaches to molecular diagnostics of chromatinopathies, it is not always possible to establish the molecular diagnosis by traditional methods; thus, the issue of optimizing diagnostic algorithms remains relevant. One of the most rapidly expanding areas of post-genomic molecular diagnostics is episignature detection, which relies on genome-wide DNA methylation analysis. This article aims to represent an original approach to indirect diagnostics of chromatinopathies on the example of Rubinstein–Taybi syndrome 1, which is based on the analysis of the methylation level of a limited set of loci designed to reproduce its classic episignature. In the current study, we apply two methods of targeted quantitative analysis of DNA methylation, which are relatively accessible and can be integrated into diagnostic practice. We demonstrate that Rubinstein–Taybi syndrome 1 episignature may be successfully reduced to a single locus of human genome, and that quantitative bisulfite DNA methylation analysis at this locus allows accurate identification of the Rubinstein–Taybi syndrome 1 patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3537a5166624f1652437f23c675d33be54fbf812" target='_blank'>
              DNA Methylation at a Single Locus of Human Genome Accurately Recapitulates Episignature of CREBBP-Related Rubinstein–Taybi Syndrome
              </a>
            </td>
          <td>
            Olga A. Zemlianaia, A. Kalinkin, A. Tanas, Anna V. Efremova, I. Volodin, O. Ismagilova, Anton S. Smirnov, D. Zaletaev, Marina V Nemtsova, S. Kutsev, V. Strelnikov
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="AMP-activated protein kinase (AMPK) is a highly conserved serine/threonine kinase that functions as a central regulator of cellular energy status. In cancer, where metabolic reprogramming enables rapid proliferation and survival under stress, AMPK functions as a metabolic checkpoint that restrains tumor growth by inhibiting biosynthetic pathways and promoting catabolic processes, such as autophagy and fatty acid oxidation. Given its role in opposing many hallmarks of cancer metabolism, AMPK has attracted significant interest as a therapeutic target. This review examines the molecular mechanisms by which AMPK influences tumor progression and evaluates the preclinical and clinical evidence for pharmacological AMPK activation using agents such as metformin, phenformin, and canagliflozin. While promising anti-tumor effects have been reported in specific contexts—such as HER2-positive breast cancer, colorectal cancer, and metabolically distinct lung cancer subtypes—clinical efficacy remains variable. Limitations include indirect activation mechanisms, low tissue penetrance, tumor heterogeneity, and lack of reliable biomarkers for patient selection. We discuss emerging strategies to overcome these challenges, including combination therapies, metabolic stratification, and the development of direct AMPK activators or mRNA-based delivery platforms. Together, these insights support a renewed focus on AMPK as a modifiable node in cancer metabolism and a candidate for integration into precision oncology frameworks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33f9fe32addc45e0e116ae1b3a6cfa5d6734699f" target='_blank'>
              Targeting AMPK for Cancer Therapy: Metabolic Reprogramming as a Therapeutic Strategy
              </a>
            </td>
          <td>
            Minseo Hong, Jea-Hyun Baek
          </td>
          <td>2025-09-26</td>
          <td>Oncology Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Metastasis is the main cause of failure in anticancer therapies, and is frequently related to poor prognosis for patients. The true challenge in extending cancer patient life expectancy, eventually managing cancer as a chronic disease with periodic but controllable relapses, relies on the development of effective therapeutic strategies specifically targeting key mechanisms involved in the metastatic cascade. Traditional chemotherapy with alkylating agents, microtubule inhibitors, and antimetabolites has shown limited efficacy against metastatic cells, largely due to the emergence of chemoresistant populations that undergo epithelial-to-mesenchymal transition (EMT), promoting the colonization of distant organs and sustaining metastatic progression. This scenario has spurred significant efforts to identify small molecules and biologics capable of interfering with specific steps in the metastatic process. In this review, we provide an overview of recent advances involving small interfering RNAs (siRNAs) and microRNAs (miRNAs) in cancer therapy. Although most of these agents are still under investigation and have not yet been approved for clinical use, insights into their development stage offer valuable information to identify new targets in the ongoing fight against metastasis. Particular emphasis is placed on the role of chemical modifications applied to siRNAs, such as backbone, sugar, terminal, base, and conjugation changes, and how these factors influence their stability, immunogenicity, and targeting precision. By integrating these aspects into the discussion, this review provides a focused and up-to-date resource for researchers in medicinal chemistry, drug delivery, and pharmaceutical formulation, where molecular design plays a critical role in therapeutic success.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0dbbc8021fb135eb5717ae3c5e587b5f4b68b62c" target='_blank'>
              Knocking on Cells’ Door: Strategic Approaches for miRNA and siRNA in Anticancer Therapy
              </a>
            </td>
          <td>
            Massimo Serra, Alessia Buccellini, M. Paolillo
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="INTRODUCTION
Multiple myeloma (MM) is an incurable clonal B-cell malignancy characterized by the accumulation of neoplastic plasma cells in the bone marrow (BM). Many pieces of evidence indicate that hypoxia promotes MM progression, but the underlying mechanisms are not well known.


METHODS
We analyzed gene expression profiles of 3 MM cell lines under hypoxia and the MMRF CoMMpass project. We validated the expression patterns of hypoxia-associated genes (HAGs) in CD138+ BM cells from MM patients at different stages. Single-cell RNA sequencing data were used to analyze the performance of HAGs in the BM microenvironment.


RESULTS
We identified 17 HAGs differentially expressed in three MM cell lines under hypoxia. While in the MMRF project, we identified 92 differentially expressed HAGs in newly diagnosed MM patients. MM cell lines and the MMRF project shared 9 HAGs, including ADM, BNIP3L, EGLN1, FAM162A, HMOX1, PDK1, PLOD1, STAT5B, and TFRC. Notably, 8 of them were significantly associated with the overall survival of MM patients, and 6 were significantly associated with the MM patient survival in the first year after diagnosis. Then, hypoxia pressure scores calculated using these genes displayed significant differences between MM patients and healthy individuals. Further, we validated the expression patterns of HAGs using another cohort data and performed qRT-PCR using our own samples, and the results confirmed severe hypoxia existed in plasma cells and other cell types of the BM microenvironment of MM patients compared to healthy individuals.


CONCLUSION
Taken together, our findings may contribute to the treatment and prognosis prediction of MM patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9473a008423522556ea8e825486e97240efc7ead" target='_blank'>
              Transcriptome Analysis Identifies Functional and Prognostic Hypoxia-Associated Genes in Multiple Myeloma.
              </a>
            </td>
          <td>
            Lijia Hou, Jing Zhang, Q. Ran, Zhongjun Li, Maoshan Chen
          </td>
          <td>2025-09-26</td>
          <td>International journal of laboratory hematology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Simple Summary Ionizing radiation can lead to breaks in the double-stranded structure of DNA. Although cells have mechanisms that can repair such breaks, misrejoining events can occur that lead to mutations in an individual’s genome. When mutations affect certain genes, such as oncogenes and tumor-suppressor genes, cancer can result. Although ionizing radiation is a known cause of cancer, it has been difficult to determine how much of an excess cancer risk results from low-dose radiation exposure. Radiation signatures that distinguish radiation-induced cancers from spontaneous cancers are thus useful for assessing cancer risks after exposure to low-dose radiation. Recent advances in high-throughput sequencing technologies have enabled the comprehensive profiling of mutations in individual cancer cells. Analyses of radiation-associated cancer genomes have identified an increase in small deletions and chromosome rearrangements, including large deletions, inversions, and translocations, which are likely to be formed by the misrejoining of DNA double-strand breaks. Among these, chromosome rearrangements leading to cancer-related mutational events are potentially useful as biomarkers and as therapeutic targets for treating radiation-induced cancer. This review summarizes research on radiation-induced mutational signatures found in animal and human cancers, consolidates current knowledge, and discusses the future perspectives in this research field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a33c28820d79dcb3eabee840cf5431bbdeaec073" target='_blank'>
              Mutational Signatures in Radiation-Induced Cancer: A Review of Experimental Animal and Human Studies
              </a>
            </td>
          <td>
            K. Daino, C. Tsuruoka, Atsuko Ishikawa, S. Kakinuma, T. Imaoka
          </td>
          <td>2025-08-29</td>
          <td>Biology</td>
          <td>1</td>
          <td>22</td>
        </tr>

        <tr id="Acute lymphoblastic leukemia (ALL) remains a formidable therapeutic challenge, particularly within high-risk cohorts. Advances in next-generation sequencing have elucidated critical mutations that significantly influence prognosis and therapeutic decision-making. Tyrosine kinase inhibitors (TKIs) have significantly improved treatment outcomes in Philadelphia chromosome-positive (Ph+) ALL. Meanwhile, emerging therapies such as monoclonal antibodies and chimeric antigen receptor (CAR) T-cell therapies show promise for B-cell ALL, although they are associated with considerable toxicities. These developments underscore the persistent need for alternative therapeutic strategies that can benefit a wider range of patients. In this study, human ALL cell lines—characterized by either wild-type or mutant tumor protein p53 (TP53) status—were treated with RG7388 (an MDM2 (mouse double minute 2 homolog) inhibitor) and BBI608 (a STAT3 (signal transducer and activator of transcription 3) inhibitor), both as single agents and in combination. Cell viability was quantified using XTT assays, while apoptosis was assessed via flow cytometry. Additionally, immunoblotting and qRT-PCR were employed to evaluate changes in protein and gene expression, respectively. RG7388 demonstrated potent growth inhibition in the majority of ALL cell lines, with p53-mutant cell lines exhibiting resistance. BBI608 reduced cell viability across all tested cell lines, though with variable sensitivity. Notably, the combination of RG7388 and BBI608 elicited synergistic anti-proliferative effects in p53 wild-type and partially functional p53-mutant cells, enhancing apoptosis and stabilizing p53 protein levels. In contrast, MOLT-4 cells, which harbor concurrent TP53 and STAT3 mutations, did not benefit from the combination treatment, indicating an inherent resistance phenotype within this subset. Collectively, these findings highlight the therapeutic potential of combined MDM2 and STAT3 inhibition in ALL, particularly in p53 wild-type and partially functional p53-mutant contexts. This combinatorial approach augments apoptosis and tumor growth suppression, offering a promising avenue for expanding treatment options for a broader patient population. Further investigation is warranted to validate these preclinical findings and to explore translational implications in genetically diverse ALL subsets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/175bf78e5a77155f44b8ab2779e6277c64a69ad7" target='_blank'>
              Synergistic MDM2-STAT3 Inhibition Demonstrates Strong Anti-Leukemic Efficacy in Acute Lymphoblastic Leukemia
              </a>
            </td>
          <td>
            E. Aptullahoğlu, Emrah Kaygusuz
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Persistence of JAK2-mutated clones that may undergo clonal evolution and malignant transformation remains a challenge in myeloproliferative neoplasms (MPN), Novel therapeutic approaches to attenuate clonal evolution and progression to blast-phase are therefore urgently needed. LSD1 (KDM1A) inhibitors reduce symptoms and clonal burden in MPN, but whether these compounds may be effective in advanced disease stages remained so far elusive. Using a chromatin-focused CRISPR-Cas9 screen, we identified the histone methyltransferase DOT1L as a synthetic lethal target under pharmacologic LSD1 inhibition. DOT1L knockout impaired cellular fitness, reduced proliferation, and prolonged survival in xenografts. Furthermore, genetic inactivation of DOT1L increased LSD1 inhibitor sensitivity up to 100-fold resulting in cell cycle arrest and apoptosis induction in TP53 mutant blast-phase MPN. Mechanistically, we have identified a novel, non-canonical function of DOT1L which co-occupied LSD1-bound enhancers and contributed to the repression of transcriptional programs independent of its enzymatic activity. DOT1L loss cooperated with LSD1 inhibitors to activate tumor suppressive programs, while pharmacologic inhibition of DOT1Ls catalytic activity failed to elicit comparable effects. These findings indicate that leveraging DOT1L targeting via protein degradation or RNA interference, rather than conventional enzymatic inhibition, could enhance the therapeutic efficacy of LSD1 inhibitors in blast-phase MPN.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd3997846e172a52e0436c7851168e81909eb612" target='_blank'>
              The histone-methyltransferase DOT1L cooperates with LSD1 to control cell division in blast-phase MPN
              </a>
            </td>
          <td>
            Karl Kapahnke, Thomas Plenge, Tabea Klaus, Manoj K. Gupta, Disha Anand, Tamer T. Önder, Birgit Perner, T. Schnöder, Felicitas R. Thol, Frederik Damm, Florian H. Heidel, Florian Perner
          </td>
          <td>2025-08-08</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Abstract LINE-1 retrotransposition is common in human cancers and rearrangements at insertion sites can contribute to cancer-driving oncogene amplifications and promote genome instability. However, the mechanisms underlying rearrangements of L1 retrotransposition intermediates are poorly understood. To address this gap, we developed GFP-based recombination reporter assays to study the formation of L1 retrotransposition-mediated rearrangements. Using these reporters combined with long-read sequencing approaches, we find that L1 retrotransposition intermediates can recombine with distal DNA breaks to generate chromosomal rearrangements. We also find that two distinct L1 insertion intermediates can recombine with each other to generate chromosomal rearrangements. Both types of rearrangements depend on L1-encoded ORF2p endonuclease and reverse transcriptase activities. Using these reporters, we discover that L1 retrotransposition-mediated rearrangements are robustly induced when the recombining sequences share homology and that their formation requires the homologous recombination factor BRCA1. Given the repetitive nature of our genome, these findings highlight the risk of L1 insertion intermediates becoming substrates for aberrant recombination and promoting genome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb644439b03063ebcc926aea5075b191a155a103" target='_blank'>
              L1 insertion intermediates recombine with one another or with DNA breaks to form genome rearrangements
              </a>
            </td>
          <td>
            Carlos Mendez-Dorantes, Jupiter C. Kalinowski, Cheuk-Ting Law, Phillip Schofield, Aidan Burn, Kathleen H. Burns
          </td>
          <td>2025-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The rapid evolution of CRISPR technology has revolutionized molecular biology, and among the various systems, CRISPR/Cas12a stands out for its high specificity and efficient collateral cleavage activity. This review article focuses on the recent advancements and applications of split technology within the CRISPR/Cas12a framework, highlighting its transformative role in molecular diagnostics and biosensing. Split technology innovatively divides functional nucleic acid components into modular segments that are activated by specific targets, significantly enhancing the specificity and sensitivity of biosensors. This design addresses the inherent limitations of traditional sensor systems, enabling the direct detection of ultrashort nucleic acids and improved discrimination of single-nucleotide variants, thereby facilitating the simultaneous detection of multiple biomolecules. The versatility of split-enabled biosensors extends beyond genetic testing, making them valuable tools in diagnostics, therapeutics, and environmental science. Despite challenges such as crRNA degradation and reassembly kinetics, ongoing research and engineering solutions continue to enhance the stability and performance of these systems. This review synthesizes the foundational principles, recent advancements, and potential applications of split technology while also identifying challenges and opportunities for future exploration. Ultimately, our insights provide a comprehensive resource to leverage the full potential of CRISPR/Cas12a-based split technology in advancing biosensing methodologies and clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/550c78a50b0190ecf4974deb77b760330bacb449" target='_blank'>
              Advancements and Applications of Split Technology in CRISPR/Cas12a: Transforming Molecular Diagnostics and Biosensing
              </a>
            </td>
          <td>
            Saikarthik Jayakumar, Srinivasan Vengadassalapathy, Santhosh Venkadassalapathy, Sheela Durairajan, Radha Vijayaraj, Lakshmanan Govindan
          </td>
          <td>2025-09-01</td>
          <td>Biosensors</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Tumor immunity has emerged as a focal point for cancer research. Although tumor immunotherapy represents a promising approach to cancer treatment, its effectiveness is often hindered by the heterogeneity of the tumor microenvironment (TME) and immune escape mechanisms. In recent years, small non-coding RNAs (sncRNAs) have attracted increasing attention in tumor immunology due to their essential role in gene regulation. This review systematically examines the multifaceted regulatory functions of sncRNAs in tumor immunity, with a focus on six major subtypes: microRNA, siRNA, piRNA, snoRNA, tsRNA, and snRNA. The molecular mechanisms by which these sncRNAs reshape the TME are discussed, including their roles in modulating immune cell differentiation (e.g., T cell polarization, macrophage phenotype transition), regulating immune checkpoint expression (PD-1/PD-L1, CTLA-4, Tim-3, LAG-3), and influencing tumor antigen presentation. This review also explores the dynamic network through which sncRNAs contribute to tumor immune escape. Furthermore, this study highlights the clinical potential of sncRNAs as liquid biopsy biomarkers and their application prospects in therapeutic strategies, such as targeted silencing of immunosuppressive molecules via nano-delivery systems, combination treatments with radiotherapy and chemotherapy, and Chimeric Antigen Receptor T-cell (CAR-T) therapy. Despite current challenges, including limited delivery efficiency and off-target effects, emerging technologies like AI-assisted sequence design and organ-on-a-chip models present new opportunities for clinical translation. This comprehensive review provides a theoretical foundation and translational insights for elucidating the functional network of sncRNAs in tumor immunology and advancing precise therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7dcee9c888595b1a7bd3be0ebd8ff37c9ea4926" target='_blank'>
              Small non-coding RNAs: key regulatory factors and potential therapeutic targets in tumor immunity
              </a>
            </td>
          <td>
            Zihan Liu, Haotian Dong, Chengyuan Ye, Jianing Yan, Min Miao, Y. Shao
          </td>
          <td>2025-09-11</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Antimicrobial resistance (AMR) has emerged as the most critical threat to global health, contributing to over 1.2 million deaths annually and projected to cause up to 10 million deaths per year by 2050 if left unaddressed. Traditional antibiotic development focused on structural modification of existing molecules has failed to match the rapid evolution of resistance mechanisms. This review underscores a paradigm shift, positioning chemistry not as a supplementary tool but as a transformative force in AMR mitigation. We explore a spectrum of chemistry-driven strategies, including efflux pump inhibitors that restore antibiotic efficacy by over 80% in vitro, β-lactamase inhibitors with 50–90% re-sensitization capacity, and antisense oligonucleotides demonstrating >70% gene silencing efficiency for resistance determinants like blaCTX-M and ndm-1. Novel CRISPR-based antimicrobials delivered via chemically engineered nanoparticles have shown the potential to restore colistin susceptibility in >60% of resistant strains. Furthermore, smart nanomaterials and targeted delivery systems have achieved up to 5-fold increases in localized drug concentration, significantly reducing off-target effects. These precision-based approaches enabled by synthetic chemistry, materials science, and molecular engineering offer versatile, adaptable, and resistance-mitigating solutions. We highlight regulatory challenges, scalability concerns, and the imperative for interdisciplinary collaboration to bridge lab innovations with clinical implementation. Chemistry remains central in the quest for sustainable, long-term solutions to AMR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68b7418d109214cb1e1b98ab55c64d1f879b925f" target='_blank'>
              Antimicrobial Resistance (AMR): Chemistry Solutions Beyond Traditional Antibiotics
              </a>
            </td>
          <td>
            Victor Ayoola Owosagba, Josiah Stephen, Blessing Ginika Eke, Fortune Itojie Ebiala, Christopher Ogbonna Okonkwo, Omoyele Olatunji Alli, Anurika Vivian Ajaero, Mariam Aanu Hammed, Susuti Richard Yerima
          </td>
          <td>2025-08-12</td>
          <td>Journal of Life Science and Public Health</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Ovarian cancer (OvCa) remains largely refractory to immune checkpoint blockade (ICB), owing to a profoundly immunosuppressive tumor microenvironment (TME). Single-cell RNA sequencing (scRNA-seq) of patient tumors has revealed a TME enriched in immunosuppressive macrophages and characterized by exclusion of T cells from the tumor core, features strongly associated with ICB resistance. However, the molecular mechanisms underlying the establishment of this immune landscape remain poorly understood. To identify regulators of OvCa immunity, we employed Perturb-map, a first-of-its-kind spatial functional genomics platform that enables multiplexed CRISPR/Cas9 knockout of candidate genes while preserving the spatial context of tumor–TME interactions in situ. We targeted 35 genes, prioritized through ligand–receptor interaction analysis of human OvCa scRNA-seq data, in an immunocompetent OvCa mouse model. Among these, IL-4 emerged as a key driver of ICB resistance. While IL-4 knockout did not affect tumor growth in untreated animals, it substantially remodeled the local TME and rendered tumors responsive to anti-PD-1 therapy. Unexpectedly, we found that both mouse and human ovarian cancer cells are a source of IL-4, a cytokine traditionally restricted to immune cells. Analysis of patient tumors revealed that tumor-associated macrophages exhibit a robust IL-4, induced transcriptional signature, implicating IL-4 in shaping the immunosuppressive TME in human disease, consistent with our preclinical findings (Cell, 2024). Furthermore, we observed that standard-of-care chemotherapy regimens, including carboplatin, paclitaxel, and cisplatin—induce a dose-dependent increase in IL-4 expression in both murine and human OvCa cell lines. This chemotherapy-induced IL-4 upregulation may contribute to immune evasion and recurrence following initial therapeutic response. Therapeutically, combined blockade of IL-4Rα and PD-1 significantly delayed tumor progression in vivo compared to monotherapy, suggesting that IL-4 signaling is both a necessary and targetable mechanism of immune resistance. These findings are clinically actionable, as Dupilumab, an FDA-approved anti-IL-4Rα antibody, is already in clinical use and well-tolerated. Our results uncover a previously unrecognized, chemotherapy-induced immunoregulatory axis in ovarian cancer and provide a strong rationale for therapeutic strategies that combine IL-4 pathway inhibition with ICB to overcome resistance and improve patient outcomes.



 Gurkan Mullaoglu, Hunter Potak, Paula Guerrero Gonzalez, Gardner Davlin, Joshua Leinwand, Dmitriy Zamarin, Brian Brown, Alessia Baccarini. Tumor-derived IL-4 links chemotherapy to immunotherapy resistance in ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr PR015.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/462188b4ccc770fdbf7b28c6a3fda711e0408972" target='_blank'>
              Abstract PR015: Tumor-derived IL-4 links chemotherapy to immunotherapy resistance in ovarian cancer
              </a>
            </td>
          <td>
            Gurkan Mullaoglu, Hunter T. Potak, Paula Guerrero Gonzalez, Gardner Davlin, Joshua Leinwand, Dmitriy Zamarin, Brian D. Brown, Alessia Baccarini
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="BACKGROUND
Therapies for diffuse glioma induce DNA damage response (DDR), and strategies to exploit DDR defects are active areas of investigation. While global DNA methylation profiling effectively classifies gliomas into subtypes, the epigenetic and gene expression patterns of DDR genes, and their contribution to tumor classification and outcomes, have yet to be fully elucidated. Thus, dissecting the DDR epigenetics, gene expression, and single-cell heterogeneity may reveal key molecular characteristics, refine prognosis, and identify novel treatment strategies and resistance mechanisms.


METHODS
We characterized DDR epigenetics and gene expression of TCGA glioblastomas (GBM) and low-grade gliomas (LGG). Single-cell protein analysis by imaging mass cytometry (IMC) was performed on a separate cohort of 118 diffuse gliomas.


RESULTS
Analysis of TCGA cohorts revealed two DDR methylation groups that correlated with IDH mutation status and previously reported molecular groups. DDR transcription profiling further classified tumors into four groups. Those with high DDR transcription across pathways were linked to poor survival independent of IDH or MGMT status, and potentially improved prognostication beyond established biomarkers. Single-cell characterization of a separate cohort revealed intratumoral DDR diversity and identified proliferative tumor cells with high DDR protein expression across pathways that are associated with unfavorable grade and survival.


CONCLUSIONS
Tumor-level epigenetic and transcriptional DDR signatures alone can distinguish molecular-defined diagnosis and outcomes of gliomas beyond established biomarkers. A higher abundance of glioma cells with high DDR effector expression across pathways is associated with poor survival. Thus, clinical assessment of pan-DDR expression may inform prognosis and identify potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0b0fbeaa6fa3878bda82552131f1d944fc87086" target='_blank'>
              DNA damage response profile distinguishes poor-acting gliomas with shared methylome signatures.
              </a>
            </td>
          <td>
            Nalin Leelatian, Charu Singh, Richard Bouffard, R. Sundaram, Kirsten E Diggins, William Sullivan, Sateja Paradkar, Zeynep E. Erson Omay, B. Mobley, Susan E. Gueble, Juan C Vasquez, R. Bindra
          </td>
          <td>2025-08-27</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f396f45a05f69851b7610dc8198d4d5a27e35207" target='_blank'>
              Characterization of cis-regulatory elements and functional variants in colorectal cancer using epigenomics and CRISPRi screenings.
              </a>
            </td>
          <td>
            Zequn Lu, Can Chen, Heng Zhang, Bin Li, Yizhuo Liu, Jiayi Guo, Runying Xu, Ke Shi, Qianying Ma, Ming Zhang, Yimin Cai, Jinyu Huang, Hui Geng, Linyun Fan, Caibo Ning, Yanmin Li, Shuo Chen, W. Tian, Kexin Hu, Haijie Li, Xiaojun Yang, Chaoqun Huang, Yongchang Wei, Xu Zhu, Xiangpan Li, Zhen Xiong, M. Cai, Xiaoyang Wang, Shaokai Zhang, Hongda Chen, M. Dai, Kun Chen, Mingjuan Jin, Meng Jin, Ying Zhu, Jianbo Tian, Xiaoping Miao
          </td>
          <td>2025-08-25</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Background Metastatic lung carcinoma poses considerable treatment difficulties. It exhibits cellular characteristics that differ from those of early-stage cancer. Immunotherapy demonstrates enhanced hope for patients with advanced and metastatic lung cancer. Tumor-associated macrophages (TAMs) may be a contributing factor that diminishes the effectiveness of immunotherapy. Secreted phosphoprotein 1 (SPP1)+ TAMs are considered to possess immunosuppressive characteristics, as their interaction with CD8+ T lymphocytes results in the exhaustion of these cells. Methods We analyzed single-cell RNA sequencing datasets of lung cancer metastases in order to examine the development of brain metastatic tumors. Furthermore, the cell–cell interactions between SPP1+ TAM cells and CD8+ T cells within primary and brain metastatic cancers were systematically compared. Additionally, we utilized a flow cytometer and immunofluorescence to demonstrate how SPP1 affects the function of CD8+ T cells. In vitro, we generated SPP1-overexpressing macrophages and performed qPCR, Western blot, and co-culture assays with or without anti-SPP1 or anti-A2AR treatment to evaluate immunosuppressive effects. Results The results demonstrated that the proportions of immune cells in metastatic brain tissues are lower, while the infiltration of macrophages is higher. SPP1+ TAMs contribute to immune suppression in lung cancer by limiting the activation of CD8+ T cells and cytokine production. Anti-SPP1 treatment positively impacts CD8+ T cell function, counteracting SPP1-induced dysfunction and facilitating the production of cytokines. SPP1 overexpression in macrophages enhanced their immunosuppressive phenotype by upregulating CD73 and cytokines such as IL-10 and TGF-β, leading to impaired CD8⁺ T cell function via A2AR signaling; notably, neutralization of SPP1 or A2AR successfully restored CD8⁺ T cell activity. Conclusions Our findings characterize the immunological environment of both primary and metastatic lung cancer, highlighting SPP1-mediated immune suppression as a potential therapeutic target to restore T cell responses. Preclinical data demonstrate that Anti-SPP1 antibodies can reverse T cell exhaustion and enhance immune responses in lung cancer. Supplementary Information The online version contains supplementary material available at 10.1007/s00262-025-04180-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/659292b315a12f27348d32dba8f9ec9a4a03cedd" target='_blank'>
              Single-cell transcriptomic analyses provide insights into SPP1+ TAM-mediated immune suppression and CD8+ T cell dysfunction in lung cancer
              </a>
            </td>
          <td>
            Rahma Taher Almgrami, Tengyue Zhang, Qitai Zhao, Mingming You, Jinyan Liu, Yi Zhang
          </td>
          <td>2025-09-29</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Today, science and medicine are striving to develop novel techniques for treating deadly diseases, including a wide range of cancers. Efforts are being made to better understand the molecular and biochemical mechanisms of tumor cell functioning, but a particular emphasis has recently been given to investigating immune cells residing in the tumor microenvironment, which may lead to revolutionary benefits in the design of new immunotherapies. Among these cells, tumor-associated macrophages (TAMs) are highly abundant and act as critical regulators of ovarian cancer progression, metastasis, and resistance to therapy. Their dual nature—as drivers of malignancy and as potential therapeutic mediators—has positioned them at the forefront of research into next-generation immunotherapies. As therapeutic targets, approaches include blocking macrophage recruitment (e.g., CSF-1/CSF-1R inhibitors), selectively depleting subsets of TAMs (e.g., via Folate Receptor Beta), or reprogramming immunosuppressive M2-like macrophages toward an anti-tumor M1 phenotype. On the other hand, macrophages can also serve as a therapeutic tool—they may be engineered to enhance anti-tumor immunity, as exemplified by the development of Chimeric Antigen Receptor Macrophages (CAR-Ms), or leveraged as delivery vehicles for targeted drug transport into the tumor microenvironment. A particularly innovative strategy involves Macrophage–Drug Conjugates (MDCs), which employs the transfer of iron-binding proteins (TRAIN) mechanism for precise intracellular delivery of therapeutic agents, thereby enhancing drug efficacy while minimizing systemic toxicity. This review integrates current knowledge of TAM biology, highlights emerging therapeutic approaches, and underscores the promise of macrophage-based interventions in ovarian cancer. By integrating macrophage-targeting strategies with advanced immunotherapeutic platforms, novel treatment paradigms may be determined that could substantially improve outcomes for patients with ovarian cancer and other solid tumors. Our work highlights that macrophages should be a particular area of research interest in the context of cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fed78827a6d291f21f77902208331601e4ff69f1" target='_blank'>
              Macrophages—Target and Tool in Tumor Treatment: Insights from Ovarian Cancer
              </a>
            </td>
          <td>
            M. Górczak, Łukasz Kiraga
          </td>
          <td>2025-09-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/743f697012228792a8f0e4e8d5e5913ecf0ed66d" target='_blank'>
              Targeting epigenetic modifications as an emerging immunotherapeutic strategy for cancers.
              </a>
            </td>
          <td>
            R. I. Kumar, Kavya Jain, Karan Raj Rai, Harshnna Gururajan, Koustav Sarkar
          </td>
          <td>2025-08-19</td>
          <td>Immunologic research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Mutations reflect the net effects of myriad types of damage, replication errors, and repair mechanisms, and thus are expected to differ across cell types with distinct exposures to mutagens, division rates, and cellular programs. Yet when mutations in humans are decomposed into a set of “signatures”, one single base substitution signature, SBS5, is present across cell types and tissues, and predominates in post-mitotic neurons as well as male and female germlines [1–3]. The etiology of SBS5 is unknown. By modeling the processes by which mutations arise, we infer that SBS5 is the footprint of errors in DNA synthesis triggered by distinct types of DNA damage. Supporting this hypothesis, we find that SBS5 rates increase with signatures of endogenous and exogenous DNA damage in cancerous and non-cancerous cells and co-vary with repair rates along the genome as expected from model predictions. These analyses indicate that SBS5 captures the output of a “funnel”, through which multiple sources of damage result in a similar mutation spectrum. As we further show, SBS5 mutations arise not only from translesion synthesis but also from DNA repair, suggesting that the signature reflects the occasional, shared use of a polymerase.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0289ae32bcbca4304d80ec26b288975aaa0f5e9" target='_blank'>
              Collateral mutagenesis funnels multiple sources of DNA damage into a ubiquitous mutational signature
              </a>
            </td>
          <td>
            Natanael Spisak, Marc de Manuel, Molly Przeworski
          </td>
          <td>2025-09-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a threat to public health, with a global incidence and mortality of over 1.9 million and 0.9 million people, respectively. Anti-epidermal growth factor receptor (EGFR)-based treatment is recommended for CRC with wild-type rat sarcoma (RAS). However, after continuous treatment with this regimen, acquired resistance occurs, which hampers the prognosis of patients. The median progression-free survival of patients with metastatic CRC receiving first-line anti-EGFR-based treatment is 10–13 months. The widely recognized mechanisms that induce acquired anti-EGFR resistance are related to the activation of the RAS/RAF/MEK/ERK and PI3K/AKT/mTOR signaling pathways. In addition, novel mechanisms that induce acquired resistance, such as microsatellite stability/mismatch repair status, noncoding RNAs, the tumor microenvironment, exosome-mediated intracellular communication, and post-transcriptional modification, are being discovered. To improve personalized medication, biomarkers with predictive value for acquired anti-EGFR resistance are recognized from both tumoral samples and liquid biopsies. On the basis of the identified mechanisms, clinicians have developed several treatment strategies to cope with acquired anti-EGFR resistance. This review provides an overview of acquired anti-EGFR resistance in RAS wild-type CRC by summarizing the common genes and proteins, potential novel mechanisms, and predictive biomarkers related to acquired anti-EGFR resistance, as well as treatment strategies to address this resistance. This review may serve as a potential reference for exploring possible treatment strategies for acquired anti-EGFR resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce156c01d06746945c1d8cd107bf601f31353493" target='_blank'>
              An overview of mechanisms, biomarkers, and treatment strategies for acquired anti-EGFR resistance in RAS wild-type colorectal cancer
              </a>
            </td>
          <td>
            Chenxiang Ding, Ruoyu Yao, Yong Wang, Hongjie Liu, Chunyan Li, Hua Xie
          </td>
          <td>2025-09-29</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39afb12b1cf53d01e3a00f3383b9ed1def1b93f7" target='_blank'>
              PARP Inhibitors in Gastric Cancer: Unlocking Precision Oncology.
              </a>
            </td>
          <td>
            Derek Tai, Vitor Goes, Sharanya Kumar, Pranati Shah, Farris Al-Manaseer, Daniel Park, Christiana Crook, Sofia Guzman, Xiaolin Zhu, Daneng Li, D. Castillo
          </td>
          <td>2025-09-17</td>
          <td>The oncologist</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Hematopoietic stem cells (HSCs) are pivotal in regenerating the blood and immune systems and have been widely used in treating hematologic malignancies like leukemia and lymphoma. Advances in gene-editing technologies, such as CRISPR/Cas9 and lentiviral vectors, have enabled the modification of HSCs to enhance their therapeutic potential, offering a continuous source of tumor-targeting immune cells. However, challenges related to long-term engraftment, safety, and delivery mechanisms remain. Methods This systematic review, following PRISMA guidelines, evaluated 19 preclinical and translational studies on gene-edited HSCs for leukemia and lymphoma treatment. The review focused on gene-editing methodologies, preclinical outcomes, delivery challenges, and translational barriers. Databases such as PubMed, Web of Science, and Embase were searched using keywords related to gene editing, HSCs, and hematologic malignancies. Findings Gene-edited HSCs demonstrated promising preclinical efficacy, with CAR-engineered HSCs showing durable tumor clearance and multilineage immune reconstitution. Lentiviral vectors were the most common delivery method, but concerns about insertional mutagenesis persist. CRISPR/Cas9 offered precise editing but faced challenges with low homology-directed repair (HDR) efficiency in quiescent HSCs. Non-viral delivery methods, such as electroporation, showed potential for safer gene editing but require further optimization. Base editing technologies, while not requiring HDR, present their own delivery challenges that need to be addressed. Suicide gene strategies were effective in mitigating safety risks, while preconditioning regimens improved engraftment success. Conclusions Gene-edited HSCs hold significant promise for treating leukemia and lymphoma, offering long-term immune persistence and tumor clearance. However, challenges in gene delivery efficiency, safety, and engraftment must be addressed for clinical translation. Future research should focus on improving editing precision, scalable manufacturing, and robust safety monitoring to advance these therapies toward clinical trials. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03529-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e04af03546b1d80ab71eb520e4b72fd75b50cd93" target='_blank'>
              Gene-edited hematopoietic stem cells for leukemia and lymphoma treatment: a systematic review of preclinical and translational evidence
              </a>
            </td>
          <td>
            Hassan Nourmohammadi, Mashallah Babashahi, Mohammad Panji, Safa Radmehr
          </td>
          <td>2025-10-02</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The BRAF protein regulates cell growth and division through key signaling pathways. Mutations in BRAF, particularly the V600E variant, are frequently observed in colorectal cancer (CRC) and are associated with poor prognosis and therapeutic challenges. Tumors harboring certain BRAF mutations often exhibit primary resistance to BRAF inhibitor monotherapies. Over time, these tumors can also develop acquired resistance, further complicating treatment. In this study, we employed replica exchange molecular dynamics simulations combined with machine learning techniques to investigate the structural alterations induced by BRAF mutations and their contribution to drug resistance. Our analyses revealed that conformational changes in mutant BRAF proteins associated with dabrafenib residues psi494, phi600, phi644, phi663, psi675, and phi677 were sufficient for classifying drug-resistant vs. drug-sensitive variants. Similarly, for vemurafenib, residues psi450, phi484, phi495, phi518, psi622, and phi622 were the key residues that influence drug binding and resistance mechanisms. These residues are located in the N-lobe of CR3, which is responsible for ATP binding and the regulation of BRAF kinase activity. These findings offer deeper insights into the molecular basis of BRAF-driven resistance and provide predictive models for phenotypic outcomes of various BRAF mutations. The study underscores the importance of targeting specific BRAF variants for more effective, personalized therapeutic strategies in drug-resistant CRC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bcf6858b6f2686bd3d3197f896e957a981c35dc" target='_blank'>
              Combining Molecular Dynamics and Machine Learning to Predict Drug Resistance Causing Variants of BRAF in Colorectal Cancer
              </a>
            </td>
          <td>
            Longsheng Xie, Christopher Lockhart, Dmitri K Klimov, M. S. Jafri
          </td>
          <td>2025-08-30</td>
          <td>Molecules</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="DNA repair is a multifaceted biological process that involves multiple pathways to counter the types of damage the genome encounters throughout life. In the past decade zebrafish became a popular model organism to study various aspects of vertebrate DNA repair, and the characterization of several mutant lines deficient in key players of the repair pathways has significantly contributed to our understanding of the roles the corresponding proteins play in the maintenance of genomic integrity. Interestingly, the base-excision repair (BER) pathway remained one of the less characterized DNA repair processes in fish. Here we provide a detailed characterization of zebrafish deficient in one of the key components of BER, the uracil-DNA glycosylase Unga. We show that while these fish are viable, they display an altered response to genotoxic stress and unga mutant males show an interesting form of subfertility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a43c49560d6251c2408bc41d8723ce8251daad0" target='_blank'>
              Zebrafish Unga Is Required for Genomic Maintenance upon Genotoxic Stress and Male Fertility
              </a>
            </td>
          <td>
            Latifa Kazzazy, Flóra Huba, Bálint Lóránt Hausz, Dávid Mező, Viktória Perey-Simon, Bálint Jezsó, Abdulrahman Seddik, Zoran Marinović, Judit Tóth, Angéla Békési, Beáta G. Vértessy, Máté Varga
          </td>
          <td>2025-09-01</td>
          <td>Journal of Developmental Biology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Genomic instability is a hallmark of aging and cancer. A key contributor to genomic instability includes alternative DNA structures, such as cruciform-forming inverted repeats (IRs). Short IRs (< 100 bps) are abundant in the human genome, mutagenic, and enriched at mutation hotspots in human cancer genomes. Using an innovative mutation-reporter mouse model, we showed that short IRs are mutagenic in vivo. Further, we found that aging exacerbates IR-induced genomic instability, as evidenced by increased mutation frequencies and altered spectra in the spleen and brain of mice harboring either a short IR or control B-DNA sequence at 2 and 24 months of age. These findings establish a link between aging and enhanced mutagenesis at short IRs, providing a unique in vivo platform to investigate age-related mechanisms of DNA structure-mediated genomic instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbdc4812813b7511aa9fae9d1c0a6308731d4e5f" target='_blank'>
              Aging Increases Short, Inverted Repeat-Mediated Genomic Instability In Vivo.
              </a>
            </td>
          <td>
            Pooja Mandke, Pallavi Kompella, Guliang Wang, Karen M. Vasquez
          </td>
          <td>2025-09-30</td>
          <td>Aging cell</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Abstract The murine endogenous retrovirus MERVL is dynamically activated in a small population of in vitro cultured mouse embryonic stem cells (mESCs) exhibiting totipotent-like features. Yet, the relationship between MERVL activation and cell fate decisions of mESCs is incompletely understood. Through a genome-wide knockout screen, we discovered that MERVL activity is intrinsically linked to DNA damage response pathways. Loss of Ints7, a backbone subunit of the Integrator complex, increased DNA damage and triggered MERVL expression. Mechanistically, Ints7 depletion induced phosphorylation of Kap1, increased chromatin accessibility at MERVL loci, and activated the p53–Dux axis to drive MERVL transcription. Intriguingly, DNA damage-induced MERVL resurgence followed the cleavage of caspase-3, often accompanying a process known as anastasis—cell survival after transient apoptotic signaling. Collectively, our study uncovered that MERVL activation in mESCs is integrated into the cellular circuit for decision-making in response to DNA damage, suggesting that sublethal caspase activation can influence the developmental potential of stem cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9647f46def2d9561823487f197db2f3a1327b93a" target='_blank'>
              Ints7 deficiency activates DNA damage response to elicit resurgence of endogenous retrovirus MERVL and anastasis of embryonic stem cells
              </a>
            </td>
          <td>
            Yunfan Shen, Li Li, Huiling Ni, Hui Li, Mingrui Xu, Xiaoyang Tan, Zhangjie Li, Pishun Li, Fang Chen, Song Mao, Gongping Sun, Kai Yuan
          </td>
          <td>2025-08-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The accumulated knowledge on the biology of the HIV-1 virus has led to the emergence of technologies that exploit the architecture of retroviruses and their integration or vectorization properties. This field of study constitutes retroviral vectorology, democratized in laboratories by the use of lentiviral vectors. By hijacking retroviral assembly, other systems are emerging and are increasingly mentioned in recent literature. In particular, defective retroviral particles are capable of transiently delivering effectors that act on the genome: they thus appear to be more suitable tools for delivering genetic scalpels, whose persistence in the target cell or organism is not required. Since the description of the CRISPR Cas9 system in 2012, genome engineering techniques have continued to evolve in terms of capacity and reliability. Several derivatives of the CRISPR system can now modify the human genome with nucleotide-level precision. Introducing these effectors into the cell or organism remains a major technical challenge that vector scientists are striving to overcome. This review describes the major retroviral systems used for genome manipulation. Following an overview of genetic engineering techniques, we will see how researchers have developed a wide range of genomic tools by manipulating different processes in the retroviral architecture.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62ec389903eeddaa21dd5bd87231b14ea2d7fadd" target='_blank'>
              [Harnessing retroviral engineering for genome reprogramming].
              </a>
            </td>
          <td>
            P. Mangeot, T. Ohlmann
          </td>
          <td>2025-08-01</td>
          <td>Medecine sciences : M/S</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/258c585d858690cd52a8f16499c3f52ebcd32f02" target='_blank'>
              Genetic regulation of cell type-specific chromatin accessibility shapes immune function and disease risk
              </a>
            </td>
          <td>
            A. Xue, J. Fan, O. A. Dong, H. L. Huang, P. C. Allen, E. Spenceley, E. Sagi-Zsigmond, B. Bowen, A. Cuomo, A. Henry, H. Tanudisastro, Z. Qiao, L. Chen, E. Ben-David, K. Farh, D. Neavin, A. S. Lee, A. Senabouth, C. Bartie, R. A. McCloy, V. Chin, K. M. de Lange, G. Figtree, A. W. Hewitt, D. G. MacArthur, J. E. Powell
          </td>
          <td>2025-08-29</td>
          <td>None</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Simple Summary This study found that RBMX is highly expressed in human osteosarcoma. Through animal models combined with single-cell transcriptome sequencing analysis, it was discovered that knockout of RBMX significantly enhanced the infiltration of immune cells in the tumor microenvironment, suggesting that intervening in the expression of RBMX may help make tumors more likely to respond to immunotherapy. This study reveals that CD8+T cells may be important cells regulated by RBMX. Further experiments revealed that RBMX regulates the expression of H2-K1 and THBS1, driving the depletion of CD8+T cells in osteosarcoma through cell communication. This study may provide potential targets for reshaping the immune microenvironment of osteosarcoma and improving the therapeutic effect of osteosarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/703a197b1ec112a245ad6c288dc9ed70bf8339fb" target='_blank'>
              The RNA-Binding Protein RBMX Mediates the Immunosuppressive Microenvironment of Osteosarcoma by Regulating CD8+T Cells
              </a>
            </td>
          <td>
            Yu Qiu, Chao Pu, Chengguang Wang, Zhengxue Quan
          </td>
          <td>2025-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="PURPOSE OF REVIEW
Acute myeloid leukemia (AML) characterized by NPM1 mutations or KMT2A rearrangements depends on abnormal epigenetic programs mediated by menin, a critical scaffold protein for sustaining the expression of oncogenic HOX/MEIS1 genes. Therefore, menin inhibitors have become a promising class of AML treatments.


RECENT FINDINGS
Early-phase trials have shown that agents such as revumenib, ziftomenib, bleximenib, and enzomenib are active, particularly in relapsed/refractory disease, with 23-48% of patients achieving composite complete remission. However, the durability of single-agent treatment remains limited, and resistance mechanisms, such as MEN1 mutations or transcriptional reprogramming, have been identified. Consequently, combination approaches involving venetoclax, hypomethylating agents, or chemotherapy are being investigated to improve response depth and duration. Recent SAVE, KOMET-007, and cAMeLot-2 trials have demonstrated high overall response rates (68%-100%) and encouraging MRD-negative complete remissions when combining menin inhibitors with venetoclax-based regimens. This has been observed even in patients who have previously received venetoclax. Combinations with intensive chemotherapy (e.g., 7 + 3) in the frontline setting have also yielded high CRc rates (up to 94% in NPM1-mutated AML) without exacerbating toxicity.


SUMMARY
These findings justify the integration of menin inhibitors into the AML therapeutic landscape, and support ongoing randomized trials to confirm their benefit in both frontline and relapse or refractory settings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30efc9044b0dac7eb9b858bad0224206b5fa44af" target='_blank'>
              The role of menin inhibitors in acute myeloid leukemia.
              </a>
            </td>
          <td>
            A. Isidori, Giovanni Marconi
          </td>
          <td>2025-08-08</td>
          <td>Current opinion in oncology</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="


 BRCA1, BRCA2, and PALB2 mutations frequently involve shifting of the gene reading frame, resulting in the production of premature stop codons and truncated proteins. Frequently, the latter are subject to misfolding and proteasomal degradation. We have found that in response to prolonged PARP inhibitor (PARPi) treatment, multiple common pathogenic BRCA1 mutations use post-transcriptional mechanisms to produce truncated and hypomorphic protein isoforms. BRCA1 protein isoforms could induce PARPi resistance when expressed at sufficiently high levels. Moreover, we discovered several BRCA1 genetic alterations, such as secondary splice site mutations, and Alu-mediated gene rearrangements, which result in the production of protein isoforms that avoid misfolding and promote PARPi resistance. Finally, we are currently assessing the impact of PALB2 hypomorphic proteins on cancer initiation and will discuss recent results.



 Neil Johnson. BRCA mutations and protein products in cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr IA022.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86b875969d2d788ca87a548ad50a77f9264de3b1" target='_blank'>
              Abstract IA022: BRCA mutations and protein products in cancer
              </a>
            </td>
          <td>
            Neil Johnson
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Tumor-associated macrophages (TAMs) display remarkable functional heterogeneity, yet the molecular mechanisms driving their diverse phenotypes remain elusive. Using CRISPR screens in primary macrophages, we identified tumor-derived factors, including lactic acid, PGE2, and GM-CSF, as key modulators of TAM polarization. These factors interact synergistically and antagonistically to shape distinct TAM phenotypes that are highly conserved across human cancers. Mechanistically, lactic acid and PGE2 jointly induce angiogenic gene programs while suppressing GM-CSF-driven MHC-II expression at the chromatin level, creating mutually exclusive distributions of proangiogenic and MHC-II+ TAMs, which are differentially localized to specific spatial niches in the tumor microenvironment. Furthermore, we showed that shifting TAMs to an interferon-responsive phenotype, triggered by Adar inactivation, significantly promotes the infiltration of effector CD8+ T cells through specific receptor-ligand interactions. These findings uncover a conserved mechanism of TAM polarization and offer insights into therapeutic strategies for TAM reprogramming to potentiate cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cb6bdb7d04c92080704c1d38a2b724c54141d29" target='_blank'>
              Functional genetic screens reveal key pathways instructing the molecular phenotypes of tumor-associated macrophages.
              </a>
            </td>
          <td>
            Youxue Lu, Ce Luo, Lanxiang Huang, Genyi Wu, Lihan Zhong, Jieyu Chu, Fubing Wang, Zexian Zeng, Deng Pan
          </td>
          <td>2025-09-04</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="This review article explores the role of immuno-radiotherapy in the context of genome instability and tumour evolution. Genomic changes in tumours exist in a delicate balance with the immune system, offering evolutionary pathways to adapt and grow but risking provoking an immune response. Rapid developments across both immunotherapy and radiotherapy have raised questions about the potential benefits combination therapy, and how best to identify ideal treatment populations. Here we discuss foundational studies of genomic instability and tumour evolution, how these paradigms translate into immune surveillance and evasion, and subsequently go on to explore recent preclinical and clinical studies of both treatment modalities. Understanding how cancers evolve in the context of the immune system could provide a key insight in delivering better therapies that could overcome treatment resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4d64cec361f889e2be65100e62ef8d285da2f26" target='_blank'>
              Tumour Evolution Driving Genome Instability, Immune Interactions, and Response to Radiotherapy
              </a>
            </td>
          <td>
            G. Seed, France Truong, Rahma Riahi, Ben O’Leary
          </td>
          <td>2025-06-09</td>
          <td>Cancer journal (Sudbury, Mass.)</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="


 African Americans (AAs) have higher incidence rates of advanced lung cancer when compared to European Americans (EAs). DNA double-strand breaks (DNA DSBs) are the major cause of genomic instability that initiates cancer. Our previous work found AA lung cancer patients have more germline DNA DSB repair deficiencies than EAs, suggesting unrelated AAs may share genomic risk factors for developing lung cancer. We hypothesize African ancestry drives disease-causing alterations in DNA DSB repair pathway genes.



 ClinVar criteria were used to profile germline pathogenic variants (PVs) across the four major DNA DSB repair pathways: homologous recombination repair (HRR), classical non-homologous end joining (c-NHEJ), alternative end-joining (alt-EJ), and single-strand annealing (SSA). The discovery cohort consisted of populations of African (n=37,545) and European (n=622,057) descent in the Genome Aggregation Database. The validation cohort included study participants in the NIH All of Us Research Program (African, n=50,080) and (European, n=125,860). In silico RNA (SpliceA1, Pangolin, and PhyloP) and protein (VEP and CADD) annotation tools were used to predict functional consequences.



 In the African ancestry populations, 20 PVs across eight HRR genes, nine PVs across six c-NHEJ genes, six PVs across three alt-EJ genes, and 14 PVs across five SSA genes we discovered. In comparison, 115 PVs across 11 HRR genes, 52 PVs across eight c-NHEJ genes, 43 PVs across six alt-EJ genes, and 113 PVs across seven SSA genes were observed in the European ancestry populations. Interestingly, a RAD51 PV was exclusively enriched in African ancestry and not European ancestry populations for the HRR pathway. By contrast, NHEJ1, EXO1 and DCLRE1C PVs were uniquely present in those with European ancestry and not in individuals with African ancestry for the alt-EJ and SSA pathways, respectively. Frameshift mutations were the most common functional protein change in both populations.



 A total of 32 germline PVs across 13 DNA DSB repair genes were observed in African ancestry populations only, suggesting genetic ancestry may drive DNA repair deficiencies that increase susceptibility to developing lung cancer.



 Musinu Zakari, Egbenoma A. Aigboeghian, Khadijah A. Mitchell. Germline pathogenic variant profiling of DNA double-strand break repair pathways in populations with African and European genetic ancestry [abstract]. In: Proceedings of the 18th AACR Conference on the Science of Cancer Health Disparities; 2025 Sep 18-21; Baltimore, MD. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2025;34(9 Suppl):Abstract nr C077.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c11a81bc23bab29ba6a3aa864a49c700d5fec40e" target='_blank'>
              Abstract C077: Germline pathogenic variant profiling of DNA double-strand break repair pathways in populations with African and European genetic ancestry
              </a>
            </td>
          <td>
            Musinu Zakari, Egbenoma A. Aigboeghian, Khadijah A. Mitchell
          </td>
          <td>2025-09-18</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Functional inactivation of tumor suppressor genes drives cancer initiation, progression, and treatment responses. Most tumor suppressor genes are inactivated through 1 of 2 well-characterized mechanisms: DNA-level mutations, such as point mutations or deletions, and promoter DNA hypermethylation. Here, we report a distinct third mechanism of tumor suppressor inactivation based on alterations to the histone rather than DNA code. We demonstrated that PAX2 is an endometrial tumor suppressor recurrently inactivated by a distinct epigenetic reprogramming event in more than 80% of human endometrial cancers. Integrative transcriptomic, epigenomic, 3D genomic, and machine learning analyses showed that PAX2 transcriptional downregulation is associated with replacement of open/active chromatin features (H3K27ac/H3K4me3) with inaccessible/repressive chromatin features (H3K27me3) in a framework dictated by 3D genome organization. The spread of the repressive H3K27me3 signal resembled a pearl necklace, with its length modulated by cohesin loops, thereby preventing transcriptional dysregulation of neighboring genes. This mechanism, involving the loss of a promoter-proximal superenhancer, was shown to underlie transcriptional silencing of PAX2 in human endometrial cancers. Mouse and human preclinical models established PAX2 as a potent endometrial tumor suppressor. Functionally, PAX2 loss promoted endometrial carcinogenesis by rewiring the transcriptional landscape via global enhancer reprogramming. The discovery that most endometrial cancers originate from a recurring epigenetic alteration carries profound implications for their diagnosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5897c9bf68c93e4fb62e86f78e7178d27155eec3" target='_blank'>
              A distinct mechanism of epigenetic reprogramming silences PAX2 and initiates endometrial carcinogenesis
              </a>
            </td>
          <td>
            Subhransu S. Sahoo, Susmita G Ramanand, Ileana C. Cuevas, Yunpeng Gao, Sora Lee, Ahmed Abbas, Xunzhi Zhang, Ashwani Kumar, Prasad R. Koduru, Sambit Roy, Russell R. Broaddus, Victoria L Bae-Jump, Andrew B. Gladden, Jayanthi Lea, Elena Lucas, Chao Xing, Akio Kobayashi, Ram-Shankar Mani, Diego H. Castrillon
          </td>
          <td>2025-08-15</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="A cornerstone of research to improve cancer outcomes involves studies of model systems to identify causal drivers of oncogenesis, understand mechanisms leading to metastases, and develop new therapeutics. While most cancer types are represented by large cell line panels that reflect diverse neoplastic genotypes and phenotypes found in patients, prostate cancer is notable for a very limited repertoire of models that recapitulate the pathobiology of human disease. Of these, Lymph node carcinoma of the prostate (LNCaP) has served as the major resource for basic and translational studies. Here, we delineated the molecular composition of LNCaP and multiple substrains through analyses of whole genome sequences, transcriptomes, chromatin structure, AR cistromes, and functional studies. Our results determined that LNCaP exhibits substantial subclonal diversity, ongoing genomic instability and phenotype plasticity. While several oncogenic features were consistently present across strains, others were unexpectedly variable such as ETV1 expression, Y chromosome loss, a reliance on WNT and glucocorticoid receptor activity, and distinct AR alterations maintaining AR pathway activation. These results document the inherent molecular heterogeneity and ongoing genomic instability that drive diverse prostate cancer phenotypes and provide a foundation for the accurate interpretation and reproduction of research findings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/851530a0dad65b81ca7cbda1909bc73f4b9c3e44" target='_blank'>
              Multiomic assessments of LNCaP and derived cell strains reveal determinants of prostate cancer pathobiology.
              </a>
            </td>
          <td>
            Arnab Bose, Armand Bankhead Iii, Ilsa Coleman, Thomas Persse, Wanting Han, Patricia C. Galipeau, B. Hanratty, Tony L. Chu, Jared M. Lucas, Dapei Li, Rabeya Bilkis, P. Itagi, Sajida Hassan, Mallory Beightol, Minjeong Ko, R. Dumpit, Michael C. Haffner, C. Pritchard, G. Ha, Peter S Nelson
          </td>
          <td>2025-09-16</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>22</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [0, 15],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>