<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-11-10 09:38:33 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="DHFR gene amplification is commonly present in methotrexate (MTX)-resistant colon cancer cells and acute lymphoblastic leukemia. In this study, we proposed an integrative framework to characterize the amplified region by using a combination of single-molecule real-time sequencing, next-generation optical mapping, and chromosome conformation capture (Hi-C). We identified an amplification unit spanning 11 genes, from the DHFR gene to the ATP6AP1L gene position, with high adjusted interaction frequencies on chromosome 5 (~2.2 Mbp) and a twenty-fold tandemly amplified region, and novel inversions at the start and end positions of the amplified region as well as frameshift insertions in most of the MSH and MLH genes were detected. These mutations might stimulate chromosomal breakage and cause the dysregulation of mismatch repair. Characterizing the tandem gene-amplified unit may be critical for identifying the mechanisms that trigger genomic rearrangements. These findings may provide new insight into the mechanisms underlying the amplification process and the evolution of drug resistance. Sequencing a large region of DNA containing many surplus copies of genes linked to drug resistance in colon cancer cells may illuminate how these genomic rearrangements arise. Such regions of gene amplification are highly repetitive, making them impossible to sequence using ordinary methods, and little is known about how they are generated. Using advanced methods, Jeong-Sun Seo at Seoul National University Bundang Hospital in South Korea and co-workers sequenced a region of gene amplification in colon cancer cells. The amplified region was approximately 20 times the length of that in healthy cells and contained many copies of an eleven-gene segment, including a gene implicated in drug resistance. The region also contained mutations in chromosomal repair genes which would disrupt repair pathways. These results illuminate the genetic changes that lead to gene amplification and drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental & Molecular Medicine, Experimental and Molecular Medicine</td>
          <td>5</td>
          <td>50</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum
                </a>
              </td>
          <td>
            Gustavo S. França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>63</td>
          <td>45</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Most advanced melanomas initially respond to targeted therapy but eventually relapse. Rather than acquiring new mutations, resistance is driven by drug-tolerant persister cells that enter a reversible drug-refractory state. We developed MeRLin, a high-resolution lineage tracing platform integrating cellular barcoding, single-cell transcriptomics, RNA fluorescence in situ hybridization (FISH), and computational analyses to track clonal and transcriptional dynamics in patient-derived melanoma models during prolonged therapy. Clonal dynamics revealed that persister subpopulations first responded to treatment but persisted and expanded during minimal residual disease, ultimately leading to tumor recurrence. Pre-treatment melanoma populations diversified into four conserved persister states characterized by stress-like, lipid metabolism, PI3K signaling, and extracellular matrix remodeling programs associated with adaptive resistance. Spatial transcriptomics showed the organization of these adaptive programs and a complex signaling network of autocrine and paracrine interactions among persister subpopulations. Barcoded RNA-FISH enabled spatial mapping of clonal identity and gene expression, revealing in situ co-localization of a dominant resistant clone with SLC2A1 expression. MeRLin provides a robust framework for dissecting cancer heterogeneity and identifying vulnerabilities in persister populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c4ead317fd2f11125da9ef801fb0c7bcc0217f3" target='_blank'>
              Clonal dynamics shaped by diverse drug-tolerant persister states in melanoma resistance
              </a>
            </td>
          <td>
            Haiyin Li, Yeqing Chen, Jessica Kaster, Maggie Dunne, Min Xiao, Ling Li, Monzy Thomas, Nazifa Promi, Dylan Fingerman, G. Brown, Qiuxian Zheng, Xingyue Zhu, McKenna Reale, Andrew Patterson, Le Gao, Xuxiang Zhang, Siqi Jiang, Tianxing Hu, Hanzhang Fang, Jianlan Ren, Cong Qi, Luyang Wang, Haiwei Mou, Gatha Thacker, Eric Ramirez Salazar, J. Villanueva, Arjun Raj, Dave SB Hoon, Tian Bin, Jozef Madzo, Zhi Wei, Noam Auslander, M. Herlyn
          </td>
          <td>2025-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>118</td>
        </tr>

        <tr id="Acquired therapy resistance is a dynamic process associated with early epigenetic modifications reshaping gene transcription across multiple cellular pathways, ultimately giving rise to drug-tolerant persister (DTP) cells. Unraveling the mechanisms that sustain DTP cell survival and drive their evolution into stable drug-resistant cells (DRC) is crucial for developing targeted therapies. Cisplatin-tolerant and cisplatin-resistant models were established using liver and gastric cancer cell lines for the first time to explore these mechanisms. Our investigation centered on the distinct epigenetic landscapes of DTP and DRC cells following cisplatin exposure. RNA sequencing revealed that DTP cells exhibit downregulation of pathways involved in cell cycle regulation, DNA replication, transcription, and chromatin maintenance while upregulating those associated with cell–cell communication and cytokine signaling. Interestingly, these transcriptional changes revert in the DRC state, suggesting a high degree of plasticity during the DTP phase. Furthermore, DTP cells have elevated levels of heterochromatin markers, H3K9me3, H3K27me3, and HP1α, along with their methyltransferases, G9a and Ezh2. Knockdown studies and inhibition of the enzyme activity of these modifiers showed suppression of DTP cell emergence. Valproic acid (VPA), a phase III candidate, was further assessed in vivo, where its sequential administration with cisplatin significantly reduced tumor burden versus cisplatin alone. These findings highlight the therapeutic promise of targeting epigenetic modifications to pre-sensitize cancer cells to chemotherapy, thereby restricting the survival advantage of DTP cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/649db90cb654c2e41c431aeb4e6b9016419e7a71" target='_blank'>
              The synergistic action of HDAC inhibitor with cisplatin impedes survival and proliferation of drug-tolerant persister in gastric and liver cancer cells
              </a>
            </td>
          <td>
            Anjali Singh, Abhiram Natu, Flevia Anthony, Hemalatha Muthu, B. Khade, Duane T. Smoot, Hassan Ashktorab, Sanjay Gupta
          </td>
          <td>2025-10-08</td>
          <td>Clinical Epigenetics</td>
          <td>1</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c21e29753393c78c9a7147383291954089d02fd" target='_blank'>
              KDM5-driven transcriptional noise fuels plasticity-led awakening and relapse in paediatric cancer
              </a>
            </td>
          <td>
            Alejandro Allo Anido, Cecilia Roux, Emilia Chen, Siân Hamer, Abigail Shea, Ayeh Sadat Sadr, Christie English, Charlotte Butterworth, Harvey Che, Angella Bellini, Birgit Geoerger, Gudrun Schleiermacher, L. Chesler, Michael David Morgan, Alejandra Bruna
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Chemotherapy resistance remains a formidable challenge to the treatment of high-grade serous ovarian cancer (HGSOC). The drug-tolerant cells may originate from a small population of inherently resistant cancer stem cells (CSCs) in primary tumors. In contrast, sufficient evidence suggests that drug tolerance can also be transiently acquired by nonstem cancer cells. Regardless of the route, key regulators of this plastic process are poorly understood. Here, we utilized multiomics, tumor microarrays, and epigenetic modulation to demonstrate that SOX9 is a key chemo-induced driver of chemoresistance in HGSOC. Epigenetic upregulation of SOX9 was sufficient to induce chemoresistance in multiple HGSOC lines. Moreover, this upregulation induced the formation of a stem-like subpopulation and significant chemoresistance in vivo. Mechanistically, SOX9 increased transcriptional divergence, reprogramming the transcriptional state of naive cells into a stem-like state. Supporting this, we identified a rare cluster of SOX9-expressing cells in primary tumors that were highly enriched for CSCs and chemoresistance-associated stress gene modules. Notably, single-cell analysis showed that chemo treatment results in rapid population-level induction of SOX9 that enriches for a stem-like transcriptional state. Altogether, these findings implicate SOX9 as a critical regulator of early steps of transcriptional reprogramming that lead to chemoresistance through a CSC-like state in HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/016a8c0db4a1babe91a08a91f054acbb57a8720c" target='_blank'>
              SOX9 drives a stem-like transcriptional state and platinum resistance in high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            A. Duval, Fidan Seker-Polat, Magdalena Rogozinska, Meric Kinali, A. Walts, Ozlem Neyisci, Yaqi Zhang, Zhonglin Li, Edward Tanner, Allison E. Grubbs, Sandra Orsulic, Daniela Matei, M. Adli
          </td>
          <td>2025-10-01</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3565fa0fad40d6b64564b524def98cd2cdcfa949" target='_blank'>
              Metabolic plasticity drives mechanisms of PARP inhibitor persistence in ovarian cancer
              </a>
            </td>
          <td>
            Adriana Del Pino Herrera, Joon-Hyun Song, Karla Torres-Arciga, Susoma Halder, Kadin El Bakkouri, Mehdi Damaghi, Meghan C. Ferrall-Fairbanks
          </td>
          <td>2025-10-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Understanding the complex molecular mechanisms driving drug resistance in glioblastoma (GBM) is crucial to develop effective therapeutic strategies. While prior studies have identified resistance mechanisms tied to specific drugs or pathways, a multi-modal molecular analysis of resistance across a range of drug classes is lacking. We identified highly drug-resistant (n = 5) and drug-sensitive (n = 4) cultures from a cohort of 32 patient-derived glioblastoma stem cell (GSC) cultures screened against a broad panel of ~ 500 anti-cancer drugs. To elucidate the key drivers of drug resistance, we performed integrative profiling of stemness, differentiation capacity, global gene expression, mutation profiles, and DNA methylation patterns between the two groups. Despite heterogeneous gene expression profiles, drug-resistant GSCs showed consistent upregulation of ATP-binding cassette (ABC) drug efflux transporters, stemness, and extracellular matrix (ECM)-related genes. Compared to drug-sensitive GSCs, drug-resistant GSCs exhibited more pronounced stem-like properties and reduced differentiation capacity. Notably, genes linked to axonogenesis displayed significant CpG island hypomethylation in drug-resistant GSCs. This study suggests a pivotal role for GSC plasticity, stemness maintenance, and ECM-mediated drug evasion in GBM treatment resistance. Our findings highlight the adaptive and dynamic nature of resistance mechanisms in GSCs, emphasizing the need for comprehensive molecular insights to inform targeted therapeutic strategies in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f39de902ec6cfcaa0e52251d953e6e404ad13d38" target='_blank'>
              Drug resistant glioblastoma stem cells exhibit enriched stemness signatures and share extracellular matrix overexpression
              </a>
            </td>
          <td>
            Lance L Estabillo, E. Skaga, S. Halldorsson, Einar O Vik-Mo, C. Sandberg
          </td>
          <td>2025-10-27</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The cellular plasticity of neuroblastoma is defined by a mixture of two major cell states, adrenergic and mesenchymal, which may contribute to therapy resistance. However, how neuroblastoma cells switch cellular states during therapy remains largely unknown, and how to eradicate neuroblastoma regardless of its cell state is a clinical challenge. To better understand the cellular plasticity of neuroblastoma in chemoresistance, we define the transcriptomic and epigenetic map of adrenergic and mesenchymal types of neuroblastomas using human and murine models treated with indisulam, a selective RBM39 degrader. We show that cancer cells not only undergo a bidirectional switch between adrenergic and mesenchymal states, but also acquire additional cellular states, reminiscent of the developmental pliancy of neural crest cells. These cell state alterations are coupled with epigenetic reprogramming and dependency switching of cell state–specific transcription factors, epigenetic modifiers, and targetable kinases. Through targeting RNA splicing, indisulam induces an inflammatory tumor microenvironment and enhances the anticancer activity of natural killer cells. The combination of indisulam with anti-GD2 immunotherapy results in a durable, complete response in high-risk transgenic neuroblastoma models, providing an innovative, rational therapeutic approach to eradicate tumor cells regardless of their potential to switch cell states. Cell state plasticity of neuroblastoma cells is linked to therapy resistance. Here, the authors develop a transcriptomic and epigenetic map of indisulam (RBM39 degrader) resistant neuroblastoma, demonstrating bidirectional cell state switching accompanied by increased NK cell activity, which they therapeutically enhance by the addition of an anti-GD2 antibody.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce8bfb0658591ed4d276e6df844046df62c0e809" target='_blank'>
              RBM39 degrader invigorates innate immunity to eradicate neuroblastoma despite cancer cell plasticity
              </a>
            </td>
          <td>
            Shivendra Singh, Jie Fang, Hongjian Jin, Lee-Ann Van de Velde, Andrew Cortes, Jiani Chen, Sivaraman Natarajan, E. Poon, Qiong Wu, Christopher L. Morton, Mary A. Woolard, Waise Quarni, Jacob A Steele, Jon P Connelly, Liusheng He, Rebecca Thorne, Gregory Turner, Thomas Confer, Melissa Johnson, W. Caufield, Burgess Freeman, Timothy Lockey, A. Murphy, P. J. Murray, Takashi Owa, S. Pruett-Miller, Ruoning Wang, L. Chesler, Julie R. Park, A. Davidoff, John Easton, Xiang Chen, Paul G. Thomas, Jun Yang
          </td>
          <td>2025-09-17</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b95fbb2531e397416e257c3c0be1df4a9857f2e4" target='_blank'>
              Integrated single-cell whole genome sequencing and spatial transcriptomics reveal latent intra-tumoral heterogeneity in ovarian cancer
              </a>
            </td>
          <td>
            Rania Bassiouni, Yuxin Jin, L. Gibbs, Jing Qian, Solomon O. Rotimi, Heather Miller, Michelle G. Webb, S. Rajpara, Javier Arias-Stella, David W Craig, Lynda D. Roman, John D Carpten
          </td>
          <td>2025-10-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a hematologic malignancy characterized by the accumulation of myeloid blasts in the bone marrow. Despite the availability of potential curative treatments, patients frequently experience unfavorable outcomes. One crucial aspect contributing to relapse is the plasticity of leukemic clones, which enables them to switch between active proliferation and dormancy. The adaptability of AML underscores the need for novel therapies targeting AML-specific proteins. To address this, genome-wide CRISPR screens can be utilized to identify cancer entity-specific vulnerabilities. Leveraging publicly available functional genomics datasets and comparing AML with non-AML cancer cell lines, we identified a significant dependency on the cell cycle-regulating gene MYBL2 in AML. We describe MYBL2 as a key driver of AML cell growth and proliferation, highlighting its established role as a cell cycle regulator. Also, our findings uncover its previously unrecognized function as an inhibitor of cellular senescence. A knockdown of MYBL2 induces cell cycle arrest in the G2/M phase with subsequent induction of apoptosis in vitro, and reduces leukemic burden in a patient-derived xenograft (PDX) model in vivo. Interestingly, some AML cells evade apoptosis and enter a senescent-like phenotype upon MYBL2-knockdown, which is reversible upon re-expression of MYBL2. Finally, analyses of clinical data from two publicly available patient cohorts demonstrate a lower probability of survival in patients with higher MYBL2 expression, further hinting at the potential relevance of MYBL2 in AML. In conclusion, our findings demonstrate the essential role of MYBL2 in AML, governing the balance between cell proliferation, cell survival and senescence, ultimately influencing the fate of AML cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bab0ba7401319632cc10fcda395ad12a7cf902e" target='_blank'>
              Cell cycle regulator MYBL2 is a distinct vulnerability in acute myeloid leukemia
              </a>
            </td>
          <td>
            Sandra Küchler, Silke Brilloff, Silvia Schäfer, Elahe Rahimian, Vida Kufrin, Shraddha S Peri, J. Musa, T. Grünewald, Denis M Schewe, C. Ball, M. Bornhäuser, H. Glimm, Marius Bill, A. A. Wurm
          </td>
          <td>2025-10-20</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Background: Doxorubicin (DOX) is a widely used chemotherapeutic agent, but its efficacy is often limited by cancer cell resistance. Although multiple DOX resistance mechanisms have been characterized, the global transcriptomic alterations underlying this phenomenon remain poorly understood. The aim of this work was to determine whether a common transcriptional response associated with DOX desensitization exists across tumor cells of different origins and to identify the core elements of this response. Methods: We performed an integrated bioinformatics analysis, including: analysis of independent transcriptomic datasets (comparing DOX-resistant neuroblastoma, breast, and cervical carcinoma cells to their DOX-sensitive counterparts), functional annotation of differentially expressed genes, reconstruction and topology analysis of gene networks, text mining, and survival analysis. The findings were validated through in vitro functional tests, RT-PCR, and analysis of the Cancer Therapeutics Response Portal and The Cancer Genome Atlas. Results: We showed that DOX resistance in cancer cells is associated with cytoskeletal reorganization, modulation of cell adhesion, cholesterol biosynthesis, and dysregulation of mTORC1, Wnt, and Gβγ signaling pathways. Network analysis identified a conserved regulome of 37 resistance-linked genes, with GJA1, SEH1L, TCF3, TUBA4A, and ZYX emerging as central hubs (mean degree: 8.7–19.7; mean fold change: 2.4–21.3). Experimental validation in DOX-resistant KB-8-5 cervical carcinoma cells and their sensitive counterparts (KB-3-1) confirmed enhanced cellular adhesion and reduced intracellular cholesterol levels associated with chemoresistance, supporting our in silico findings. A detailed follow-up analysis verified the upregulation of these hub genes in chemoresistant cells and their correlation with poor clinical outcomes across multiple cancer types. Conclusions: This integrative analysis identifies conserved transcriptomic signatures of DOX resistance and highlights hub genes GJA1, SEH1L, TCF3, TUBA4A, and ZYX with potential as predictive biomarkers and therapeutic targets. Targeting these pathways may help overcome chemoresistance and improve treatment outcomes in cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f2b259a82bfdbeb3c8cb64a5419e6a90ac8b1c5" target='_blank'>
              Novel Cross-Cancer Hub Genes in Doxorubicin Resistance Identified by Transcriptional Mapping
              </a>
            </td>
          <td>
            A. D. Moralev, O. Markov, M. Zenkova, A. Markov
          </td>
          <td>2025-10-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d97eb5848239c5dad0bd09d537985b5d408f38de" target='_blank'>
              Melanoma to rhabdomyosarcoma plasticity in the setting of immunotherapy
              </a>
            </td>
          <td>
            Andrew D. Knight, Emily J. Robitschek, Jia-Ren Lin, Dennie T. Frederick, Alvin Shi, Ana B. Larque, B. Miao, Rumya Raghavan, T. Sharova, John H. Shin, P. Sorger, M. Kellis, Keith T Flaherty, N. Hacohen, Genevieve M. Boland, Ivan Chebib, David Liu, Ryan J. Sullivan, Arnav Mehta
          </td>
          <td>2025-11-04</td>
          <td>None</td>
          <td>0</td>
          <td>116</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1e72f57d35ee3dac82999a8b9759c889a0508af" target='_blank'>
              Genetic and transcriptomic alterations underlying aggressiveness in 5-fluorouracil-resistant HCT116 cells
              </a>
            </td>
          <td>
            Pornchai Sooksaen, A. Thim-uam, Ratsada Praphasawat, Chutamas Thepmalee, Kanokkarn Phromnoi, Damratsamon Surangkul, Surisak Prasarnpun, Chakkraphong Khonthun
          </td>
          <td>2025-10-10</td>
          <td>Medical Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acd8a87b52ce435726d0713cb35d9987d7862c73" target='_blank'>
              The two sides of resistance: aggressiveness and mitotic instability as the Achilles’ heel of Osimertinib-resistant NSCLC
              </a>
            </td>
          <td>
            P. Armanetti, E. Cabrera San Millán, R. Mercatelli, Chiara Cavallini, Alice Chiodi, E. Mosca, Arianna Consiglio, Nicola Losito, Luca Menichetti, E. Bramanti, Azhar Ali, Elena Levantini, Giorgia Maroni
          </td>
          <td>2025-09-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="This study explores the molecular impact of cisplatin, Actemra (tocilizumab), remdesivir, and SARS-CoV-2 infection on colorectal cancer (CRC) SW-480 cells using RNA sequencing (RNA-Seq). Differential gene expression analysis revealed treatment-specific transcriptional changes. Cisplatin led to the downregulation of genes involved in lipid metabolism and focal adhesion, while remdesivir upregulated chromatin remodeling pathways. SARS-CoV-2 infection altered cytokine signaling, particularly IL-17 and TNF-α. Principal component and hierarchical clustering analyses confirmed distinct gene expression profiles across treatments. ELISA assays validated the reduction of ACE2 and CD147 protein levels following drug treatment. MTT assays demonstrated remdesivir’s cytotoxicity at high doses. These findings highlight potential drug-gene interactions and suggest that combining transcriptomic profiling with functional assays may guide personalized therapeutic strategies for CRC patients, especially those co-infected with SARS-CoV-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5eae066da47093f217085c9697d40cc942624f2a" target='_blank'>
              RNA sequencing reveals transcriptional signatures of drug response and SARS-CoV-2 interaction in colorectal cancer cells
              </a>
            </td>
          <td>
            Alaa K. Hameed, Ifad K. Abd Al-Shibly, Rana A. Ghaleb
          </td>
          <td>2025-09-18</td>
          <td>Frontiers in Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1b28a4cea8f32f994db559bdedf94de2b973cd5" target='_blank'>
              Clonal Evolution of Pediatric Acute Myeloid Leukemia and Its Contribution to Disease Relapse
              </a>
            </td>
          <td>
            M. Umeda, P. Kolekar, A. L. Huskey, S. Suryaprakash, J. Ma, H. Li, T. Westover, M. P. Walsh, G. Song, Y. Liu, Q. Tran, V. Balagopal, , S. W. Brady, T. A. Gruber, S. Pounds, P. E. Mead, S. E. Karol, H. Inaba, J. Rubnitz, R. Ribeiro, X. Ma, J. Klco
          </td>
          <td>2025-11-06</td>
          <td>None</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="Hypoxia is a well-recognized clinical feature of solid tumors, including lung cancer, and is associated with poor prognosis due to its role in promoting resistance to chemotherapy and radiotherapy. To investigate the cellular consequences of hypoxia, we cultured A549 lung adenocarcinoma cells under 1% O2 and examined their growth, cell cycle distribution, and redox status. Hypoxia significantly reduces cell proliferation and induced G1 phase cell cycle arrest, suggesting a cytostatic effect. Biochemical analysis showed a 2.64- and 2.04-fold increase in total and mitochondrial reactive oxygen species (ROS) levels, respectively, along with an elevated total thiol levels under hypoxic conditions compared to normoxia. To assess the reversibility of the hypoxic response, we performed a reciprocal oxygen exposure experiment where cells initially grown under hypoxia were re-exposed to normoxia, and vice versa. To explore the underlying molecular mechanism, we analyzed transcriptomic datasets (GEO accession: GSE48134 and GSE42416) which revealed that hypoxia downregulated key genes involved in energy metabolism (e.g., PDK4, G6PD), cell cycle progression (e.g., CCND1, CDK2), and redox regulation (e.g., GCLM, TXNRD1, NQO1, GCLC). Further, few of redox-related genes were validated by RT-PCR in A549 cells cultured under hypoxia and normoxia for 24 h. Importantly, cyclic hypoxia (intermittent hypoxia-reoxygenation) conditions showed partial restoration of some of these transcripts, supporting the transient nature of hypoxic stress, consistent with our in vitro observations. Furthermore, transcriptome profiles from adenocarcinoma patients (GEO accession: GSE30979) also match our cell line observations. Thus, our results clearly show that hypoxia causes a temporary cell cycle arrest in lung cancer cells, which is reversible when oxygen is restored.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e77bbcdb23f2ddac965c8e06176945c4fad76ded" target='_blank'>
              Hypoxia-drives reversible cell cycle arrest in lung cancer cells via modulation of cellular redox and gene expression
              </a>
            </td>
          <td>
            D. Maurya, Varshita Mehta, Babita Singh
          </td>
          <td>2025-09-29</td>
          <td>Medical Oncology (Northwood, London, England)</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51a501c277b7efc44209952f58d1318feb42e211" target='_blank'>
              YAP1 drives aggressive and therapy resistant state in melanoma through reprogramming the chromatin and regulating immune evasive programs
              </a>
            </td>
          <td>
            Nana Chen, Gracia M. Bonilla, Abdullah AI Emran, Brian Lin, Haymanti Bhanot, Yang Sun, Avinash Waghray, Mateo Useche, David Liu, Genevieve M Boland, Ivana L de la Serna, V. Sancisi, Martin Sattler, Jayaraj Rajagopal, Xu Wu, Ruslan Sadreyev, David E Fisher, S. Saladi
          </td>
          <td>2025-09-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Metastatic colorectal cancer is commonly treated with oxaliplatin. However, patients may develop resistance to treatment over time, and currently, there are no validated predictive biomarkers for this resistance. A differential analysis of the transcriptome and DNA methylome of colorectal cancer cell lines, classified by their varying IC50 values for oxaliplatin, revealed that genes associated with resistance were enriched for interferon regulatory factor pathways. In contrast, sensitive genes showed enrichment for transcription, amino acid metabolism, development, and binding motifs of c-MyC:Max, AP1 and others. In univariate Cox’s proportional hazard model analysis, it was found that UBE2H expression is linked to shorter survival time in the TCGA dataset and was further validated across five GEO datasets of colorectal cancer. The transcription of UBE2H, along with its gene body methylation, and copy number variation was found to be higher in resistant cell lines compared to sensitive ones. Additionally, UBE2H levels were higher in cancer samples than in control samples, while the promoter methylation is lower in cancer samples than in control samples. In groups with high UBE2H, there was an increased infiltration of eight cell types, including CD8+ T cells and type 2 T helper cells. Conversely, only T helper 17 cells showed reduced infiltration. Moreover, UBE2H expression was positively correlated with checkpoint inhibitors, CTLA4 and PDCD1, along with immune regulatory genes, such as FOXP3, IL10, IGFB1, CD274, and LAG3, etc. Analysis of single cell RNA-sequencing data revealed that UBE2H expression is elevated in undifferentiated and proliferative cells located at the base of intestinal crypts in normal colon tissue. Our findings suggest that UBE2H could serve as a resistance marker to oxaliplatin, as it is associated with methylation, the presence of proliferative and undifferentiated cancer cells, and immune exhaustion. The proposed analytical pipeline may also be applicable to other cancers and diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/853b5cb3a79ef305fa7eb4ff1e22fd4ca52bed15" target='_blank'>
              An integrative analysis of transcriptome, methylome and single-cell RNA sequencing data identifies UBE2H as a marker of oxaliplatin resistance in colorectal cancer
              </a>
            </td>
          <td>
            Jinman Fang, Qizhi Zhu, Bo Hong, Xueling Li, Hongzhi Wang
          </td>
          <td>2025-10-31</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Head and neck squamous cell carcinoma (HNSCC) remains a prevalent and aggressive malignancy, characterized by a lack of targeted therapies and limited clinical benefits. Here, we conducted an optimized whole-genome CRISPR screen across five HNSCC cell lines aimed at identifying actionable genetic vulnerabilities for rapid preclinical evaluation as novel targeted therapies. Given their critical role in cancer, cyclin-dependent kinases (CDKs) were prioritized for further investigation. Among these, CDK7 was identified as an essential and targetable gene across all five cell lines, prompting its selection for in-depth functional and molecular characterization. Genetic and pharmacological inhibition of CDK7 significantly and consistently reduced tumor cell proliferation due to generalized cell cycle arrest and apoptosis induction. Additionally, CDK7 knockout (KO) and selective inhibitors (YKL-5-124 and samuraciclib) demonstrated potent antitumor activity, effectively suppressing tumor growth in HNSCC patient-derived organoids (PDOs), as well as in both cell line- and patient-derived xenograft (PDX) mouse models with minimal toxicity. Mechanistically, CDK7 inhibition led to a broad downregulation of gene sets related to cell cycle progression and DNA repair, and significantly reduced the transcription of essential genes and untargetable vulnerabilities identified by our CRISPR screen. These findings highlight CDK7 as a promising therapeutic target for HNSCC. Our study provides strong evidence of the robust antitumor activity of CDK7-selective inhibition in disease-relevant preclinical models, strongly supporting its progression to clinical testing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4085230d5db63fd39c16115dd67e53fe6113079" target='_blank'>
              CDK7-targeted therapy effectively disrupts cell cycle progression and oncogenic signaling in head and neck cancer
              </a>
            </td>
          <td>
            María Otero-Rosales, M. Álvarez-González, Irene Pazos, Beatriz de Luxán-Delgado, Sonia Del Marro, Esperanza Pozo-Agundo, Mar Rodríguez-Santamaría, Ana López-Fernández, D. Corte-Torres, R. Granda-Díaz, S. Álvarez-Teijeiro, Iván Fernández-Vega, Corina Lorz, Ramón García-Escudero, J. P. Rodrigo, Konstantinos Tzelepis, G. Vassiliou, Irene Ferrer, Mónica Álvarez-Fernández, J. García-Pedrero, F. Hermida-Prado
          </td>
          <td>2025-11-06</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bd95123f481a9cef05baf3ace1238c5ad7fb915" target='_blank'>
              A telomere-to-telomere map of somatic mutation burden and functional impact in cancer
              </a>
            </td>
          <td>
            Min-Hwan Sohn, Danilo Dubocanin, Mitchell R. Vollger, Youngjun Kwon, Anna Minkina, Katherine M. Munson, Samuel FM Hart, Jane E. Ranchalis, Nancy L. Parmalee, Adriana E. Sedeño-Cortés, Jeffrey Ou, N. Y. Au, Stephanie Bohaczuk, Brianne Carroll, C. D. Frazar, William T. Harvey, Kendra Hoekzema, Meng-Fan Huang, Caitlin N Jacques, Dana M. Jensen, J. T. Kolar, R. Lee, Jiadong Lin, Kelsey Loy, Taralynn Mack, Yizi Mao, Meranda M. Pham, E. Ryke, Josh Smith, Lila M. Sutherlin, Elliott G Swanson, Jeffrey M. Weiss, SMaHT Assembly, C. Carvalho, Tim H. H. Coorens, Kelley Harris, Chia-Lin Wei, E. Eichler, Nicolas Altemose, James T. Bennett, Andrew B. Stergachis
          </td>
          <td>2025-10-13</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f47c334737b17de2a5aca09e969a1cbb905e569" target='_blank'>
              E2F3 amplification primes bladder cancer cells for premature mitosis
              </a>
            </td>
          <td>
            Kathryn A. Wierenga, Qingwu Liu, Isabel Nieland, Pepijn Rakers, Anita van den Heuvel, Mara Pateli, Richard W. Wubbolts, Frank Riemers, Saskia van Essen-Dorresteijn, E. V. van Liere, B. Westendorp
          </td>
          <td>2025-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc9320984fb7ca80afa8b2d49d724dd86390a378" target='_blank'>
              Transposable Elements and Homotypic Niches Drive Immune Dynamics and Resistance in Melanoma Epigenetic-based immunotherapy
              </a>
            </td>
          <td>
            Erika Ciervo, Francesco Ceccarelli, A. D. Di Giacomo, Piera Grisolia, A. Covre, Z. M. Besharat, Antonio De Falco, F. Caruso, Luigi Laezza, Luigi Ferraro, Gloria Mas Martin, M. F. Lofiego, Tommaso Sani, E. Ferretti, Yan Guo, Sean B. Holden, R. Mortarini, A. Anichini, M. Maio, T. Noviello, M. Ceccarelli
          </td>
          <td>2025-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e0d7b578f97dbff41a41bb90de3b031320cfe41" target='_blank'>
              Unravelling drug resistant proteotypes through phenotype-resolved proteomics of single-cell derived colonies
              </a>
            </td>
          <td>
            Di Qin, Ridhima Das, S. Schallenberg, K. Riecken, I. Tinhofer, Fabian Coscia
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Treatments for high-grade serous ovarian cancer (HGSOC) are initially effective but most invariably fail. Although they can successfully suppress the bulk of the tumour cell population, residual cancer cells can enter alternative therapy-resistant cell fates highlighted by proliferative arrest. Understanding the nature of these fates and how cells may resume uncontrolled proliferation will lead to the development of new treatments for HGSOC. In this study, we examine the response of HGSOC cells to standard of care cisplatin chemotherapy and to the RNA Polymerase I transcription inhibitor CX-5461/Pidnarulex, two drugs that elicit a potent DNA damage response and growth arrest. Here, we identify that HGSOC cells exposed to these therapies show multiple hallmarks of therapy-induced senescence (TIS) and derive a core TIS gene expression signature irrespective of genetic background or senescence trigger. Given that TIS is a potentially escapable state, we have performed a focussed drug screen to identify drugs that eradicate senescent HGSOC cells. We identify that therapy-induced senescent HGSOC cells, including those with decreased sensitivity to senolytic drugs that inhibit the pro-survival protein BCL-XL, can be eliminated using drugs that induce ferroptosis, an iron-dependent form of cell death. Mechanistically, we demonstrate that senescent HGSOC cells have altered expression of regulators of iron metabolism leading to intracellular iron overload that underpins this targetable vulnerability. Together, we highlight elevated levels of iron as a TIS biomarker in HGSOC and the potential of inducing ferroptosis to eradicate residual HGSOC cells following initial therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3787e084ffe3e100d6ec83768df6b1bb600d9e4a" target='_blank'>
              Exploiting dysregulated iron homeostasis to eradicate persistent high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            Carmelo Cerra, Madeleine R. C. Tancock, N. Thio, Ada Koo, AnnRann Wong, Karla J Cowley, Swati Varshney, Madelynne O. Willis, Kaylene Simpson, D. Bowtell, E. Sanij, Elizabeth L Christie, Richard B. Pearson, Jian Kang, Keefe T. Chan
          </td>
          <td>2025-09-25</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Colorectal cancer (CRC) presents significant therapeutic challenges, particularly due to the development of resistance to standard chemotherapeutic agents such as irinotecan. In this study, we aimed to investigate the molecular and phenotypic mechanisms underlying irinotecan resistance in CRC using the LIM1215 cell line model. Transcriptomic analysis demonstrated that drug withdrawal induced major transcriptional reprogramming, characterized by downregulation of ABC transporters (ABCB1 and ABCG2), extracellular matrix-related genes, and markers of epithelial-to-mesenchymal transition (EMT), alongside reactivation of cell cycle pathways. Drug screening further indicated that resistant cells maintained under irinotecan pressure exhibited a multidrug-resistant phenotype, while withdrawn cells regained sensitivity, particularly to tyrosine kinase inhibitors. Supplementation with the efflux inhibitor Elacridar partially restored drug sensitivity in resistant cells, emphasizing the role of transporter-mediated efflux in maintaining resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3002080effb079c5d4239d8ceb95fa35b9194a90" target='_blank'>
              Phenotypic plasticity including drug efflux drives reversible irinotecan resistance in LIM1215 colorectal cancer cells.
              </a>
            </td>
          <td>
            Tugce Di̇lber, Gizem Damla Yalcin, Kübra Yılmaz, Ahmet Acar
          </td>
          <td>2025-10-28</td>
          <td>FEBS letters</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Aim: Urinary bladder cancer (UBC) often develops chemoresistance, reducing treatment effectiveness. This study aimed to investigate diverse molecular mechanisms underlying acquired resistance by establishing and characterizing a comprehensive panel of UBC cell lines resistant to common chemotherapeutics. Methods: Fifteen UBC cell lines were examined: three parental lines (RT-112, TCC-SUP, UMUC-3) and twelve derived sublines adapted to cisplatin, vinblastine, or gemcitabine. Drug sensitivity was assessed using the SRB assay. Resistance mechanisms were explored via quantitative real-time PCR (targeting genes including ABCB1, dCK, hENT1, ECHDC1, TUBB3), Western blotting (assessing proteins such as p21, Cyclin B, and Mcl-1), and biochemical assessment of glutathione levels and redox state. Results: The adapted sublines exhibited distinct resistance profiles and cross-resistance patterns. Gene expression and protein analyses revealed drug- and lineage-specific alterations, involving factors such as p21, Cyclin B, and Mcl-1. Changes in glutathione metabolism were also associated with resistance. Notably, no single, universal mechanism accounted for resistance across the entire panel. Conclusion: UBC cells develop diverse, context-dependent adaptive strategies to resist cisplatin, vinblastine, and gemcitabine. These findings highlight the complexity of chemoresistance mechanisms. The characterized cell line panel represents a valuable resource for future studies aimed at understanding and overcoming drug resistance in bladder cancer, suggesting that personalized therapeutic approaches may be necessary.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2fbd01421411fd7eb4380d44e0e070090cf7648" target='_blank'>
              Distinct adaptive strategies to cisplatin, vinblastine and gemcitabine in a panel of chemoresistant bladder cancer cell lines
              </a>
            </td>
          <td>
            Monika Cuprych-Belter, Agnieszka Łupicka-Słowik, Artur Anisiewicz, Martin Michaelis, J. Cinatl jr., Mateusz Psurski
          </td>
          <td>2025-09-12</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e5c397ac071eb956e6c0c3d19f024d33e098a95" target='_blank'>
              The Breakome of BRCA1 and BRCA2 Pathway Mutation Carriers Reveals Early Processes in Breast Oncogenesis
              </a>
            </td>
          <td>
            Sara Oster Flayshman, Osama Hidmi, Jackelyn A. Alva-Ornelas, Jonathan Monin, Mark A. LaBarge, Victoria L Seewaldt, Y. Drier, Rami I. Aqeilan
          </td>
          <td>2025-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="KRAS mutations represent the most common driver genetic alterations in multiple malignant tumors, particularly exerting well-defined oncogenic driver roles in non-small cell lung cancer, colorectal cancer, and pancreatic ductal adenocarcinoma. The recent success of KRAS G12C mutation-specific allosteric inhibitors marks a therapeutic milestone in KRAS-targeted oncology. However, the rapid emergence of drug resistance in clinical applications has significantly limited the durable efficacy of these agents. The resistance mechanisms exhibit profound complexity, encompassing multidimensional pathways such as secondary/co-occurring mutations, compensatory reprogramming of signaling pathways, cellular lineage plasticity, and immune evasion within the tumor microenvironment. Studies have demonstrated that the identification of resistance-associated biomarkers is not only of critical clinical value for predicting treatment response and early warning of resistance but also provides a key entry point for dissecting resistance mechanisms. This review systematically summarizes the latest research advances in KRAS-mutant inhibitor resistance biomarkers, with a focus on the analysis of resistance molecular mechanisms, discovery of predictive biomarkers, current limitations and challenges, and exploration of biomarker-based combination therapy strategies. By integrating basic research and clinical data, we highlight the breakthroughs required to realize the application value of resistance biomarkers and prospects future research directions and priorities, aiming to provide a biomarker-related theoretical framework for precision therapy of KRAS-mutant tumors and accelerate the clinical translation of resistance-overcoming strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dedd636638a7ce054ecf1cb2befdf53d765c7e0b" target='_blank'>
              Advances in biomarkers of resistance to KRAS mutation-targeted inhibitors
              </a>
            </td>
          <td>
            Lihao Wang, Dandan Wei, Shanshan Li, Shiqing Jiang
          </td>
          <td>2025-10-08</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb14185df59742e1c30589cf1835a68e0538e9b9" target='_blank'>
              Tracking clonal evolution during treatment in ovarian cancer using cell-free DNA.
              </a>
            </td>
          <td>
            Marc J. Williams, I. Vázquez-García, Grittney K. Tam, Michelle Wu, Nancy Varice, Eliyahu Havasov, Hongyu Shi, Duaa H Al-Rawi, Gryte Satas, Hannah Lees, J. Lee, Matthew A. Myers, Matthew Zatzman, Nicole Rusk, Emily Ali, Ronak H Shah, Michael F. Berger, Neeman Mohibullah, Yulia Lakhman, Dennis S Chi, N. Abu-Rustum, C. Aghajanian, Andrew McPherson, Dmitry Zamarin, Brian Loomis, Britta Weigelt, C. Friedman, Sohrab P. Shah
          </td>
          <td>2025-10-01</td>
          <td>Nature</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="Neoadjuvant chemotherapy (NAC) is a critical component of breast cancer treatment, but the molecular mechanisms underlying resistance remain poorly understood. This study aimed to identify transcriptomic changes associated with NAC resistance across four breast cancer subtypes: Luminal A, Luminal B/HER2-positive, Luminal B/HER2-negative, and Triple-Negative Breast Cancer (TNBC). RNA-seq analysis was performed on paired pre- and post-NAC breast cancer samples from 32 non-responders. Differentially expressed genes (DEGs) were identified, and functional enrichment analyses were conducted. Protein-protein interaction (PPI) networks were constructed to identify hub genes. Tumor microenvironment (TME) infiltration was estimated using deconvolution algorithms. The results revealed distinct gene expression profiles between pre- and post-NAC samples, with FOS and NR4A1 being common DEGs across all subtypes. Enriched pathways varied among subtypes, including signal transduction, estrogen biosynthesis, extracellular matrix organization, dendritic cell activation, and B cell activation. TME analysis showed increased infiltration of specific immune cell populations after NAC, including CD4 memory T cells, regulatory T cells, neutrophils, macrophages, and mast cells, varying by subtype. These findings suggest that NAC modulates gene expression, cellular activity, and TME interactions, potentially contributing to treatment resistance. Understanding the molecular determinants of NAC resistance is crucial for developing targeted therapeutic strategies and improving outcomes for breast cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e52a24e6d865b78f25fc1cec6097a262050bb3b1" target='_blank'>
              Molecular determinants of neoadjuvant chemotherapy resistance in breast cancer: An analysis of gene expression and tumor microenvironment
              </a>
            </td>
          <td>
            H. M. Guevara-Nieto, Carlos Orozco-Castaño, R. Parra-Medina, Jenny Nathaly Poveda-Garavito, J. Garai, Jovanny Zabaleta, Liliana Lopez-Kleine, A. Cómbita
          </td>
          <td>2025-10-14</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Tumor suppressor NF1 is recurrently mutated in glioblastoma, leading to aberrant activation of Ras/rapidly accelerated fibrosarcoma (RAF)/MEK signaling. However, how tumor heterogeneity shapes the molecular landscape and efficacy of targeted therapies remains unclear. Here, we combined bulk and single-cell genomics of human somatic NF1-mutant, isocitrate dehydrogenase (IDH) wild-type glioblastomas with functional studies in cell lines and mouse intracranial tumor models to identify mechanisms of tumor heterogeneity underlying clinical outcome and MEK inhibitor response. Targeted DNA sequencing identified CDKN2A/B homozygous deletion as a poor prognostic marker in somatic NF1-mutant, but not NF1 wild-type, glioblastoma. Single-nucleus RNA sequencing of human patient NF1-mutant glioblastomas demonstrated that mesenchymal-like (MES-like) tumor cells were enriched for MEK activation signatures. Single-cell RNA-sequencing of mouse intracranial glioblastomas treated with the MEK inhibitor selumetinib identified distinct responses among tumor subpopulations. MEK inhibition selectively depleted MES-like cells, and selumetinib-resistant MES-like cells upregulated Ras signaling while resistant non-MES cells expressed markers of glial differentiation. Finally, genome-wide CRISPR interference screens validated Ras/RAF/MEK signaling as a key mediator of selumetinib response. Repression of the RAF regulator SHOC2 sensitized glioblastomas to selumetinib in vitro and in vivo, suggesting a synergistic treatment strategy. Taken together, these results highlighted the heterogeneity of NF1-mutant glioblastomas and informed future combination therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee9b553a3b7f2afede17ca71f5d4515ccec597b2" target='_blank'>
              Tumor heterogeneity underlies clinical outcome and MEK inhibitor response in somatic NF1-mutant glioblastoma
              </a>
            </td>
          <td>
            S. Pan, K. Mirchia, E. Payne, S. J. Liu, N. Al-Adli, Zain Peeran, Poojan D. Shukla, Jacob S. Young, Rohit Gupta, Jasper Wu, J. Pak, T. Ozawa, Brian Na, Alyssa T Reddy, Steve E Braunstein, Joanna J. Phillips, Susan Chang, D. Solomon, A. Perry, D. R. Raleigh, Mitchel S. Berger, Adam Abate, H. Vasudevan
          </td>
          <td>2025-09-23</td>
          <td>JCI Insight</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Advanced sarcomas have a poor prognosis and limited therapeutic options. Disease recurrence is caused by persistent cells that survive drug treatments. The alkylating agent trabectedin, when combined with the poly (ADP-ribose) polymerase 1 (PARP1) inhibitor olaparib, exhibits variable antitumor effects in advanced sarcomas. In this study, we demonstrate that the expression of the transcription factor OCT4 is upregulated in persistent cells that survive treatment with trabectedin and olaparib, through the cGAS-STING-IRF3-IFNβ pathway. This route also leads to the upregulation of natural killer (NK) and cytokine-induced killer (CIK) lymphocyte activating ligands. These molecular events enhance the antitumor efficacy of immunotherapy with NK and CIK cells, targeting both the bulk population and residual drug-tolerant cells. In conclusion, the activation of the cGAS-STING pathway has a double-edged effect, enriching the OCT4+ persistent cell population while increasing the expression of NK/CIK ligands. The addition of olaparib to trabectedin potentiates the cGAS-STING pathway activation and the upregulation of NKG2DLs, while simultaneously counteracting the OCT4 overexpression. Therefore, sequential treatment with trabectedin and olaparib followed by NK/CIK immunotherapy represents a promising strategy against advanced sarcomas and warrants further investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/506de56eb87cad4e65432310520f88b9e3f761bd" target='_blank'>
              The cGAS-STING pathway is a master regulator of OCT4 expression in persistent sarcoma cells and enhances cellular immunotherapy with NK and CIK lymphocytes
              </a>
            </td>
          <td>
            Giorgia Giordano, A. Merlini, F. Capozzi, Giulio Ferrero, Cristina Tucciarello, Shahab Majidi, Simona Comparato, G. Mesiano, Elisabetta Liuzzi, F. Galvagno, Annamaria Massa, Silvia Brusco, V. Leuci, Chiara Riganti, S. Cutrupi, Michele De Bortoli, Lorenzo D’Ambrosio, Dario Sangiolo, G. Grignani, Y. Pignochino
          </td>
          <td>2025-09-23</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4273a0ebbda27839741b0bafaebff9ecc71a2adf" target='_blank'>
              Defining heterogeneity in core regulatory circuitry reveals HOXB3 condensation as a potential target in glioblastoma
              </a>
            </td>
          <td>
            Chuanxia Zhang, Yijing He, Xiudan Zhan, Jiaming Yu, Shao Xu, Qinkai Zhang, Ru Qiu, Xinyue Wang, Zhuoxing Gao, Meng Huang, Wenyong Long, Qing Liu, Wei Zhao
          </td>
          <td>2025-09-30</td>
          <td>Nature Cell Biology</td>
          <td>1</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c09a200a976abe18d2ada20b9002895dd27ac2f5" target='_blank'>
              Prognostic and Therapeutic Implications of BRAF Mutations in Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Darren Lee, Yazan Abu-Shihab, Kate Plas, D. Nicolet, K. Mrózek, M. Routbort, K. Patel, C. Walker, Jill Buss, Andrew Stiff, Andrea Laganson, C. Dinardo, N. Daver, T. Kadia, F. Ravandi, Andrew J Carroll, J. Kolitz, B. Powell, William Blum, Maria R. Baer, G. Marcucci, Geoffrey L. Uy, Wendy Stock, Richard M. Stone, L. J. Medeiros, John C. Byrd, J. Blachly, Robert L. Bowman, J. Tyner, S. Loghavi, Ann-Kathrin Eisfeld, Linde A. Miles
          </td>
          <td>2025-10-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>119</td>
        </tr>

        <tr id="Introduction Chemotherapy response variability in cancer patients necessitates novel strategies targeting chemoresistant populations. While combinatorial regimens show promise through synergistic pharmacological interactions, traditional pathway enrichment methods relying on static gene sets fail to capture drug-induced dynamic transcriptional perturbations. Methods To address this challenge, we developed the Pathway-Responsive Gene Sets (PRGS) framework to systematically identify chemoresistance-associated pathways and guide therapeutic intervention. Comparative evaluation of three computational strategies (GSEA-like method, Hypergeometric test-based method, Bates test-based method) revealed that the GSEA-like methodology exhibited superior performance, enabling precise identification of drug-induced pathway dysregulation. Results Key experimental findings demonstrated PRGS’s superiority over conventional Pathway Member Gene Sets (PMGS), exhibiting statistical independence (p < 0.0001) and enhanced detection of chemotherapy-driven pathway dysregulation. Application of PRGS to the GDSC dataset identified 8 resistance-associated pathways. Screening of agents targeting these pathways yielded candidates with predicted anti-resistance activity. An in vitro cellular experiment demonstrated that the bortezomib-bleomycin combination exhibited synergistic cytotoxicity (IDAcomboScore = 0.014) in T47D cells, highlighting the potential of PRGS-guided therapeutic strategies. Discussion This study establishes a PRGS-based methodological framework that integrates genomic perturbations with precision oncology, demonstrating its capacity to decode resistance mechanisms and guide therapeutic development through dynamic pathway analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/814bf51ab320b8d56ddbcfefe1cd9a9aa4f449e3" target='_blank'>
              Drug discovery for chemotherapeutic resistance based on pathway-responsive gene sets and its application in breast cancer
              </a>
            </td>
          <td>
            Dehua Feng, Jingwen Hao, Lingxu Li, Jian Chen, Xinying Liu, Ruijie Zhang, Huirui Han, Tianyi Li, Xuefeng Wang, Xia Li, Lei Yu, Bing Li, Jin Li, Limei Wang
          </td>
          <td>2025-09-16</td>
          <td>Frontiers in Bioinformatics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9628d3e5281a106e55cc014d5bb9cff1f576bc47" target='_blank'>
              Dosage responses of aneuploid autosomal chromosomes
              </a>
            </td>
          <td>
            Natali Papanicolaou, Sebastian Wettersten, Guilherme Maia, Antonio Lentini, B. Reinius
          </td>
          <td>2025-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d62c6090571c67ea1645ed1df6849f539cf798b" target='_blank'>
              Molecular and cellular dynamics of measurable residual disease progression in myelodysplastic syndromes
              </a>
            </td>
          <td>
            Yu-Hsiang Chen, Timothy Bi, Minhang Xu, Jaehyun Lee, Wesley S. Lu, Mengchuan Zhuang, A. Velarde, D. McCurry, Hui Yang, G. Al-Atrash, Gabriela Rondon, E. Shpall, F. Jelloul, F. N. Cervoni-Curet, Sassine Ghanem, Celine Kerros, Priya Koppikar, J. Molldrem, G. Garcia-Manero, Ankit Patel, Pavan Bachireddy
          </td>
          <td>2025-10-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Gene expression heterogeneity generates subpopulations of tumor cells that can evade therapeutic pressure. This heterogeneity has been observed in both primary Estrogen Receptor alpha positive (ERα+) breast tumors and cell lines. Therefore, understanding the mechanisms regulating expression heterogeneity is critical towards developing effective therapies. A key contributor to gene expression variability is the stochastic nature of transcription. Transcription occurs in a probabilistic, burst-like manner, in which gene activation occurs intermittently, producing RNA in pulses and interspersed with transcriptional off-periods. The estrogen-responsive gene TFF1 is expressed in the majority of ERα⁺ breast tumors and exemplifies such heterogeneity, with transcriptional inactivity ranging from minutes to several days. Here, we identify the molecular mechanism underlying the wide range in TFF1 expression by analyzing cells sorted based on their TFF1 activity levels. We observed that TFF1 inactive (TFF1low) cells exhibit a repressive chromatin state marked by H3K27me3 at the TFF1 promoter and enhancer. Despite global similarity in ERα binding, occupancy at the TFF1 regulatory elements was selectively reduced in TFF1low cells, resulting in fewer active alleles and diminished transcriptional bursting frequency. Conversely, TFF1high cells exhibited more active TFF1 alleles and hyperbursting. These cells also retained sensitivity to endocrine therapy, while TFF1low cells displayed reduced drug responsiveness. Genome-wide, differentially enriched H3K27me3 regions correlated with variable expression of estrogen-responsive genes, highlighting a broader regulatory mechanism that links chromatin state to expression variability. Together, our findings establish how repressive chromatin dynamics contribute to gene expression heterogeneity and endocrine resistance in ERα⁺ breast cancer. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffa89e16be6e3a720a965e4891206ee026e2fe10" target='_blank'>
              H3K27me3 chromatin heterogeneity reveals variable cell responses to estrogen and endocrine treatment
              </a>
            </td>
          <td>
            Fleur Chapus, Christopher R. Day, Laura G Kammel, Pelin Yasar, Brian D. Bennett, Erica Scappini, Charles J Tucker, Maria Sifre, Celyn Bregio, Joseph Rodriguez
          </td>
          <td>2025-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Oxaliplatin resistance remains a critical barrier to effective colorectal cancer treatment. The molecular mechanisms underlying this resistance are not fully understood, highlighting the need to define the transcriptional alterations that contribute to therapeutic failure. Accordingly, a comparative transcriptome analysis was performed on oxaliplatin-resistant colorectal cancer cells (HCT-116-ROx) and their parental counterparts (HCT-116) using RNA sequencing in this study. Differentially expressed gene (DEG) analysis was conducted using a quasi-likelihood negative binomial model. Gene Ontology (GO) enrichment and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway enrichment were carried out using the topGO and clusterProfiler packages, respectively. To confirm the robustness of the transcriptomic data, the genes with the most significant expression changes, based on false discovery rate-adjusted P value less than 0.05 and a |logFC| > 2 thresholds, were selected for validation by quantitative real-time PCR (qRT-PCR). A total of 313 DEGs were identified, including ALDH3A1 and TACSTD2 (upregulated) and IFITM1 (downregulated); these three genes were chosen for validation by qRT-PCR. Gene Ontology enrichment revealed significant changes in cell motility, redox regulation, and extracellular matrix remodeling. KEGG analysis indicated upregulation of ferroptosis, glutathione metabolism, and lysosome-related pathways, and downregulation of p53 signaling, oxidative phosphorylation, and cancer-specific pathways. Oxaliplatin-resistant colorectal cancer cells undergo multifaceted transcriptional reprogramming that promotes redox homeostasis, metabolic adaptation, and structural plasticity while suppressing apoptotic and mitochondrial functions. These changes support chemoresistance and may represent potential therapeutic targets to restore drug sensitivity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e5ab501b314c92e8ff167ffcb8251950e164cb1" target='_blank'>
              Multifaceted transcriptional reprogramming supports oxaliplatin chemoresistance in colorectal cancer cells.
              </a>
            </td>
          <td>
            Gizem Calibasi-Kocal, Hasan Kurter, Zerrin Isik
          </td>
          <td>2025-10-09</td>
          <td>Anti-cancer drugs</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d2cfb28da058ae0d8b29e85290850287448ab79" target='_blank'>
              Transition between cell states of sensitivity reveals molecular vulnerability of drug-tolerant cells.
              </a>
            </td>
          <td>
            L. Peyre, Marielle Péré, Mickael Meyer, Benjamin Bian, Marina Moureau-Barbato, Walid Djema, B. Mari, Georges Vassaux, Jérémie Roux
          </td>
          <td>2025-10-01</td>
          <td>Molecular systems biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee9ca9b266d6e3d48dee0fe7d9f0b768bc883a83" target='_blank'>
              Functional interrogation uncovers a critical role for a high-plasticity cell state in lung adenocarcinoma
              </a>
            </td>
          <td>
            J. Chan, Chun-Hao Pan, Jonathan Rub, Klavdija Krause, Emma Brown, Gary Guzman, Zeda Zhang, Hannah C Styers, Griffin Hartmann, Zhuxuan Li, Xueqian Zhuang, Scott W. Lowe, D. Betel, Yan Yan, Tuomas Tammela
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="ABSTRACT Dysregulation of RNA binding proteins (RBPs) is a hallmark in cancerous cells. In acute myeloid leukaemia (AML) RBPs are key regulators of tumour proliferation. While classical RBPs have defined RNA binding domains, RNA recognition and function in AML by non-canonical RBPs (ncRBPs) remain unclear. Given the inherent complexity of targeting AML broadly, our goal was to uncover potential ncRBP candidates critical for AML survival using a CRISPR/Cas-based screening. We identified the glycolytic enzyme glyceraldehyde-3-phosphate dehydrogenase (GAPDH) as a pro-proliferative factor in AML cells. Based on cross-linking and immunoprecipitation (CLIP), we are defining the global targetome, detecting novel RNA targets mainly located within 5’UTRs, including GAPDH, RPL13a, and PKM. The knockdown of GAPDH unveiled genetic pathways related to ribosome biogenesis, translation initiation, and regulation. Moreover, we demonstrated a stabilizing effect through GAPDH binding to target transcripts including its own mRNA. The present findings provide new insights on the RNA functions and characteristics of GAPDH in AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a1eb12cd38da4497c6babe78ae3d7117c11f510" target='_blank'>
              RNA binding of GAPDH controls transcript stability and protein translation in acute myeloid leukemia
              </a>
            </td>
          <td>
            Sama Shamloo, Jeffrey L. Schloßhauer, Shashank Tiwari, Kim Denise Fischer, Omar Almolla, Yohana Ghebrechristos, Lisa Kratzenberg, Aathma Merin Bejoy, Ioannis Aifantis, Francesco Boccalatte, Eric Wang, Jochen Imig
          </td>
          <td>2025-11-01</td>
          <td>RNA Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Triple Negative Breast Cancer (TNBC) accounts for ~20% of all breast cancers and results in thousands of deaths every year. The median survival of TNBC patients sharply declines with the development of chemoresistance and metastatic disease. Although high expression of ETS1 in TNBC has been associated with aggressiveness, the mechanisms of ETS1 in TNBC therapy relapse are poorly understood. Here, we show that ETS1 is responsible for driving acquired drug resistance in the TNBC cell line models resistant to 5’-Fluorouracil and doxorubicin. Protein kinase, DNAPKcs (aka PRKDC) mediated phosphorylation of ETS1 at Serine 251 residue enhances protein stability by preventing ETS1’s degradation, thus enhancing ETS1-driven resistance mechanisms. Further, transcriptomic profiling of resistant cells and TNBC patients showed that phosphorylated-ETS1 could activate genes of the E2F, MYC and G2/M pathways, resulting in enhanced DNA synthesis and proliferation, leading to resistance. DNAPKcs inhibitors resulted in ETS1 degradation, inhibition of proliferation gene circuits and subsequent apoptosis in resistant TNBC cells. Phospho-S251 ETS1 and associated ETS1-driven proliferative gene signatures were observed in drug-resistant TNBC patients. Our findings suggest that DNAPKcs-mediated phosphorylation of ETS1 promotes chemoresistance in TNBC patients and can be targeted using DNAPKcs kinase inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/369249b88967a207d2b77ad6f3a1c0e6c9f8db87" target='_blank'>
              Enhanced ETS1 stability by DNAPKcs orchestrates transcriptional changes during chemoresistance in triple negative breast cancer
              </a>
            </td>
          <td>
            Aiindrila Dhara, Imlimaong Aier, Souhadri Das, Manash Sarkar, Ramandeep Kaur, P. Varadwaj, Samrat Daripa, S. K. Hira, Anindya Halder, Nirmalya Sen
          </td>
          <td>2025-10-27</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Abstract Spontaneous mutation rates and spectra are influenced by an intricate interplay of processes including DNA replication, proofreading, and diverse DNA damage repair pathways. Although significant progress has been made in characterizing the functions of individual genes involved in these processes, their direct effects on mutation rates and spectra remain unclear. In this study, we employed a systematic gene knockout approach coupled with mutation accumulation (MA) experiments and whole-genome sequencing (WGS) to investigate the mutational landscape of Saccharomyces cerevisiae. We targeted 136 gene-deletion strains encompassing nearly all known genes associated with DNA replication and repair. Analysis of 1,081 MA lines revealed that 114 of the 136 genes significantly influenced at least one type of mutation rate. Furthermore, deletions of specific genes led to marked shifts in mutational biases and spectra, with some deletions amplifying existing biases and others reversing them entirely. In contrast, mitochondrial mutation rates were notably less affected, with no significant impact detected. Importantly, comparative analysis revealed striking similarities between yeast mutational spectrum and those observed in human cancers with the same pathway deficiencies, suggesting conserved functional roles across species. In conclusion, our findings provided critical insights into the genetic underpinnings of these signatures and underscoring the utility of yeast as a model for studying the molecular basis of cancer-associated mutational processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5323012223c8e2fdfc841a6c54bcc4ec219997da" target='_blank'>
              Comprehensive Mutational Landscape of Yeast Mutator Strains Reveals the Genetic Basis of Mutational Signatures in Cancer
              </a>
            </td>
          <td>
            Lei Liu, Danyang Sun, Haoxuan Liu
          </td>
          <td>2025-10-01</td>
          <td>Molecular Biology and Evolution</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Aberrant expression of long noncoding RNAs (lncRNAs) serves as a critical driver in carcinogenesis and tumor evolution across various human cancers. Among these lncRNAs, small nucleolar RNA host gene 15 (SNHG15) has emerged as a key regulator in multiple cellular processes. Its dysregulation is frequently implicated in disease pathogenesis, particularly cancer. In this investigation, we delineated the functional landscape and mechanistic basis of SNHG15 in lung carcinoma. Cell cycle synchronization experiments demonstrated phase-dependent SNHG15 expression, showing maximal transcript abundance during G2/M transition. SNHG15 knockdown substantially curbed tumor growth in vitro (A549 cell line) and in vivo (subcutaneous PDX models). Notably, SNHG15 orchestrated genomic surveillance mechanisms through tripartite regulation: (1) cell cycle checkpoint coordination, (2) transcriptional machinery reactivation, and (3) nucleotide excision repair. Mechanistically, proteomic profiling coupled with RNA immunoprecipitation identified CREB5 as a direct binding partner mediating SNHG15-dependent survival signaling under genotoxic stress. These results position the SNHG15-CREB5 axis as an important player governing genomic fidelity in lung adenocarcinoma, suggesting druggable targets for precision oncology approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca1f937a74141f7d8c6f5eb7bb745d4f086d4cf3" target='_blank'>
              The role of lncRNA SNHG15 in UV-induced DNA damage repair
              </a>
            </td>
          <td>
            Xin Zhang, Jiahui Lin, Zixuan Fan, Sai Zhang, Meiqi Yang, Tingting Zhu, Qing Lu, Yuming Wang
          </td>
          <td>2025-10-09</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/573448152ca34657743158301dc6d5b323113cd1" target='_blank'>
              TP53-mutant AML with ribosomal gene loss exhibits impaired protein translation and sensitivity to HSP90 inhibition
              </a>
            </td>
          <td>
            Jean-François Spinella, J. Chagraoui, I. Boivin, Guillaume Richard Carpentier, Céline Moison, N. Mayotte, François Béliveau, Vincent-Philippe Lavallée, J. Hébert, Guy Sauvageau
          </td>
          <td>2025-10-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Background Acetyl-CoA carboxylase alpha (ACACA), a crucial rate-limiting enzyme governing de novo biosynthesis of fatty acids, drives oncogenic metabolic reprogramming in diverse malignancies. However, the multiomics investigation and immunological implications of ACACA across cancers remain unclear. Methods We performed a comprehensive pan-cancer analysis of ACACA via transcriptomic, proteomic, and clinical data from The Cancer Genome Atlas (TCGA), Clinical Proteomic Tumor Analysis Consortium (CPTAC), and the Human Protein Atlas (HPA) databases. Then, single-cell RNA sequencing acquired from the Gene Expression Omnibus (GEO) database was employed to map the expression pattern of ACACA in the tumor microenvironment (TME). Subsequently, functional validation experiments were conducted in lung cancer and sarcoma cells. Results High ACACA expression was associated with poor survival in various cancers, particularly those exhibiting dysregulated lipid metabolism. Immune profiling revealed that elevated ACACA expression was associated with low infiltration of CD8+ T cells and activated natural killer (NK) cells. Single-cell analysis of lung adenocarcinoma revealed that ACACA was expressed predominantly within malignant cells and contributed to an immunosuppressive microenvironment through migration inhibitory factor (MIF) signaling and the extracellular matrix (ECM) remodeling pathway. Furthermore, in vitro studies demonstrated that ACACA inhibition suppresses fatty acid synthesis and tumor growth in lung cancer and sarcoma cells. Conclusions Our study establishes ACACA as a key metabolic regulator that links lipid metabolism to immune evasion and drug resistance, highlighting its potential as a promising therapeutic target across cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ef9b3c72fa2e6ad7dbbe3729933ef811be46587" target='_blank'>
              Integrated single-cell and bulk RNA sequencing analysis reveals ACACA as a potential prognostic and immunotherapeutic biomarker across cancers
              </a>
            </td>
          <td>
            Haihua He, Zhen Zhang, Leifeng Chen, Fushan Gao, Yuze Wu, Lina Yi, Fei Shao, Yibo Gao, Jie He
          </td>
          <td>2025-10-15</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f0d2028082631983321af009576821353c38710" target='_blank'>
              Pharmacological Induction of Irreversible Senescence in Drug-Tolerant Persister Cells Prevents Tumor Relapse
              </a>
            </td>
          <td>
            Bowen Wang, Yun Zhi, Pengqi Wang, Wenbo Guo, Jin Gu, Hanqing He, Kezhang He, Wei Zhou, Ting Wang, Nana Huang, Haixia Yang, Dan Wang, Tianhua Ma, Sheng Ding
          </td>
          <td>2025-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="ABSTRACT The DNA damage response (DDR), an evolutionarily conserved surveillance mechanism enabling cells to detect genomic lesions and cell cycle arrest for repair, coordinates genomic stability through key effector kinases. In Candida albicans, our prior work characterized Rad53-mediated transcriptional reprogramming based on the RAD53 deletion strain, establishing its canonical DDR functions. However, Rad53 activation dynamically orchestrates cell cycle synchronization and repair coordination during genotoxic challenge. Investigating cellular modifications resulting from increased Rad53 functionality is essential for a deeper understanding of its regulatory response. To this end, we overexpressed Rad53 revealing its dual regulatory capacity in stress adaptation and developmental plasticity. Global transcriptomic profiling demonstrated that RAD53 overexpression enhances methyl methanesulfonate (MMS) resistance while paradoxically stimulating filamentous growth, accompanied by upregulation of DNA repair/replication genes (RFA3, POL30) and non-canonical DDR targets including cell wall remodeling factors. Comparative analysis of RAD53 deletion and overexpression transcriptomes identified enriched transcription factors, notably Sfl1, Yox1, and Rfx1. In particular, Rfx1 mediated the resistance to genotoxic stress resulting from RAD53 overexpression or deletion. Surprisingly, Rad53 overexpression substantially hindered hyphal formation and virulence, whereas its kinase-dead mutation restored these deficiencies. Notably, Rad53’s role in mediating morphogenesis was independent of Rfx1. This multi-modal functionality positions Rad53 as a critical node interfacing genomic stability, developmental plasticity, and pathogenicity in C. albicans. Our study enhances the understanding of the DDR pathway in C. albicans, providing a potential target for treating this fungal pathogen.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/618ff870ecbc422dc2701275aa24d5074e975660" target='_blank'>
              Rad53 orchestrates divergent pathways governing genotoxic resistance, morphogenesis, and virulence in Candida albicans: Mechanistic decoupling through overexpression analysis
              </a>
            </td>
          <td>
            Jia Wang, Shaling Yang, Yi Zhou, Huaxin Cai, Jinrong Feng
          </td>
          <td>2025-10-06</td>
          <td>Virulence</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df773895d5d3bb014e2dd4dc6276d432f2994491" target='_blank'>
              YAP1 defines an emergent, plastic population of relapsed small cell lung cancer
              </a>
            </td>
          <td>
            C. Stewart, K. Ramkumar, Runsheng Wang, Yuanxin Xi, A. Halliday, L. Diao, Qi Wang, Alejandra G Serrano, Sarah M. Groves, Simon Heeke, Azusa Tanimoto, L. Kaiser, Whitney E Lewis, M. Bose, Pedro Da Rocha, L. Karacosta, Vito Quaranta, Jing Wang, Julie George, Luisa Maren, Solis Soto, Bingnan Zhang, J. Heymach, L. Byers, C. Gay
          </td>
          <td>2025-10-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4da08fc2c51aefcac002abe7af73df18eb83fa23" target='_blank'>
              Integrative proteo-genomic profiling uncovers key biomarkers of lapatinib resistance in HER2-positive breast cancer.
              </a>
            </td>
          <td>
            J. Steggall, V. Rajeeve, N. Al-Subaie, A. Naeem, A. Ikram, A. Naeem, A. Hayat
          </td>
          <td>2025-09-13</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a highly aggressive brain tumor with limited treatment options and poor survival. Mitochondrial dysfunction and metabolic reprogramming, particularly the Warburg effect, are increasingly recognized as critical drivers of GBM progression. Here, we integrated single-cell (scRNA-seq) and bulk RNA-sequencing (bulk RNA-seq) data to comprehensively examine mitochondria-associated genes in GBM. We identified differentially expressed mitochondrial genes with prognostic significance and constructed a 9-gene risk signature—ACOT7, THEM5, MTHFD2, ABCB7, PICK1, PDK3, ARMCX6, GSTK1, and SSBP1—using LASSO and Cox regression. This signature robustly stratified patients into high- and low-risk groups in both The Cancer Genome Atlas (TCGA) and Chinese Glioma Genome Atlas (CGGA) cohorts, remaining an independent prognostic factor in multivariate analyses. Time-dependent ROC AUCs were 0.729, 0.813, and 0.828 at 1, 2, and 3 years in TCGA, and 0.597, 0.650, and 0.546 in CGGA. A nomogram integrating the signature with clinical variables achieved AUCs of 0.649, 0.820, and 0.854 at 1, 3, and 5 years, with good 1/2/3-year calibration. Functional enrichment and clustering analyses revealed distinct metabolic phenotypes and survival differences between subtypes. Single-cell pseudotime analysis showed a transition from oxidative phosphorylation to glycolysis in malignant cells, aligning with the Warburg effect and implicating metabolic reprogramming in immune modulation. Our findings underscore the prognostic value of mitochondria-associated genes and suggest potential therapeutic targets for disrupting GBM metabolism. Overall, these results establish a mitochondria-centric prognostic model supported by single-cell context; findings are hypothesis-generating and warrant prospective and experimental validation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1def4820c25938da49857764d61ab2d3606bd5c5" target='_blank'>
              Comprehensive analysis of mitochondria-associated genes in glioblastoma via single-cell and bulk RNA sequencing
              </a>
            </td>
          <td>
            Fangyuan Wang, Ao Huo, Minghe Li, Chenglin Zhao, Yupei Guo, Zhou Jing, Qingyi Wang, Bingqi Lu, Hao Li, Wulong Liang, Weihua Hu, Xudong Fu
          </td>
          <td>2025-10-27</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b81b381c14e5b1230a77d230dfeaa5698f06f2cc" target='_blank'>
              Characterizing resistance in prostate cancer at a single cell level with hormonal treatment and epigenetic inhibitors.
              </a>
            </td>
          <td>
            Morgan Howells, Priyadarsini Gangadharannambiar, Calum McMullen, Francesco Crea, W. Bacon, Colette Christiansen
          </td>
          <td>2025-11-03</td>
          <td>The Journal of pharmacy and pharmacology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Alterations in 3D chromatin conformation may disrupt the interplay between promoters and distal enhancers. How gene regulatory circuits are reshaped during ionizing radiation-induced tumorigenesis remains unclear, and little is known about the mediators that drive these processes. To decipher the chromatin alterations in radiation-induced lung cancer, we performed ATAC-seq, RNA-seq and Hi-C analyses of human bronchial epithelial cells and corresponding radiation-induced malignantly transformed cell lines. We found that this malignant transformation is accompanied by chromatin switching from the inactive B compartment to the active A compartment, an increased number of TADs and gained ATAC-seq peaks that mediate new distal chromatin contacts. We identified tumour protein 63 (TP63) as a mediator of new chromatin-accessible sites that anchor tumour-specific chromatin contacts in radiation-induced tumour cells. A TP63-mediated accessible chromatin site anchors a tumour-specific TAD boundary and multiple tumour-specific chromatin loops, which might underlie MYC oncogene activation during malignant transformation. Radiation-induced malignantly transformed cells undergo multi-scale 3D genome reorganization. TP63 acts as a key mediator, creating accessible chromatin sites that form tumour-specific loops activating the MYC oncogene during malignant transformation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79b8217e99ba9977673505cae49c7863f218434b" target='_blank'>
              TP63 mediates the generation of tumour-specific chromatin loops that underlie MYC activation in radiation-induced tumorigenesis
              </a>
            </td>
          <td>
            Qiaowei Liu, Shuai Jiang, Xiang Xu, Kang Xu, Yawen Luo, Zongyuan Yu, Meida Xiang, Zhouyang Xu, Lijie Wang, Sujie Zhang, H. Tao, Xuhui Yang, Chongbo Jiang, Xiaoguang Qi, Hao Li, Yiming Gao, Yao Li, Shihui Fu, Pingkun Zhou, Xiaochen Bo, Hebing Chen, Xiaohua Chen, Yi Hu
          </td>
          <td>2025-09-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Dysregulation of early B cell lymphopoiesis—the process guiding cellular immunity development—can lead to malignancy, making it crucial to understand its regulatory mechanisms. We generated a multiomics resource comprising paired chromatin accessibility and gene expression profiles across eight human B cell precursor populations, providing a detailed characterization of early human B cell development. Integrative analysis revealed highly cell type–specific regulatory elements and enabled the reconstruction of the gene regulatory network governing differentiation. We identified putative candidate regulons, such as ELK3, enriched in pro–B cells and potentially involved in cell cycle progression. Regulons from bulk data were projected onto single-cell data, validating their activity and refining the regulatory landscape. This resource enabled identification of active regulatory programs and transformation-associated states in B cell acute lymphoblastic leukemia. The publicly available atlas provides a valuable resource for understanding B cell development and disease, supporting future efforts to decode regulatory programs in immunity and hematologic malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72948d6bed4b92194fb9e2b0dbf1593531ef8b66" target='_blank'>
              Uncovering the regulatory landscape of early human B cell lymphopoiesis and its implications in the pathogenesis of B-ALL
              </a>
            </td>
          <td>
            Núria Planell, Xabier Martínez-de-Morentin, Daniel Mouzo, D. Lara-Astiaso, A. Vilas-Zornoza, P. S. Martín-Uriz, D. Alignani, B. Paiva, A. Maillo, Aleksandra Kurowska, Nerea Berastegui, P. García-Olloqui, Arantxa Urdangarin, Peri Noori, A. Ortega-Legarreta, Mikel Hernaez, V. Lagani, N. Kiani, Matthias Merkenschlager, T. Ezponda, J. Martín-Subero, Ricardo N. Ramirez, Jesper N. Tegnér, F. Prósper, D. Gómez-Cabrero
          </td>
          <td>2025-10-10</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="The regulatory mechanisms governing transcriptional programs in the cancer genome remain elusive, particularly those concerning cell-type specificity. We carefully curated single-cell assay for transposase-accessible chromatin sequencing (scATAC-seq) and single-cell RNA sequencing (scRNA-seq) data from eight distinct carcinoma tissues, including breast, skin, colon, endometrium, lung, ovary, liver, and kidney. Using single-cell multi-omics analysis, we identified extensive open chromatin regions and constructed peak-gene link networks, which can reveal distinct cancer gene regulation and genetic risks. We further explored conserved epigenetic regulation across cell types within cancer and elucidated their functional implications. Moreover, we identified cell-type-associated transcription factors (TFs) that regulate key cellular functions, such as the TEAD family of TFs, which widely control cancer-related signaling pathways in tumor cells. In colon cancer, we further identified tumor-specific TFs that are more highly activated in tumor cells than in normal epithelial cells, including CEBPG, LEF1, SOX4, TCF7, and TEAD4, which are pivotal in driving malignant transcriptional programs and represent potential therapeutic targets, as corroborated by single-cell sequencing data from multiple sources and in vitro experiments. Our findings provide a comprehensive understanding of the regulatory dynamics underlying carcinomas and offer valuable insights into potential therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acdeeefa6fa9c95e73d3a569b4614cf1e9a4ee17" target='_blank'>
              Single-cell multi-omics analysis reveals cancer regulatory elements of transcriptional programs and clinical implications
              </a>
            </td>
          <td>
            Xiaowei Tang, Qiaoling Zhang, Zichu Shen, Jian Xiao, Minghao Li, Xiangyan Meng, Chenyi Wang, Guangze Zhang, Anhang Liu, Yuxin Yin
          </td>
          <td>2025-10-21</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1126d43689098cc7096980496ac4ba1303c669a" target='_blank'>
              APOBEC3A-Induced DNA Damage Drives Polymerase θ Dependency and Synthetic Lethality in Cancer
              </a>
            </td>
          <td>
            Abhishek Bose, Weisi Liu, Paul Yoo, Alina Sami, Michal T. Boniecki, M. Deshpande, Mohamed Osman, Duy D. Nguyen, U. D. Castillo, P. Giannakakou, B. Bhinder, William F Hooper, O. Elemento, N. Robine, Milaid Granadillo Rodríguez, J. Gerhardt, K. Mouw, L. Chelico, B. M. Faltas
          </td>
          <td>2025-10-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a39d99b88fdac73c609342d555fba7b0f41448d" target='_blank'>
              Regulatory network and spatial modeling reveal cooperative mechanisms of resistance and immune escape in ER+ breast cancer
              </a>
            </td>
          <td>
            Yijia Fan, Sarthak Sahoo, M. Jolly, Jason T. George
          </td>
          <td>2025-10-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65a53aeb0c15e601e15d7f66e640ef0193a0c8f9" target='_blank'>
              Lineage origin and microenvironment shape neuroblastoma transcriptional state and plasticity
              </a>
            </td>
          <td>
            Nora Fresmann, Julia Köppke, A. Gauert, Anis Senoussi, Pedro Olivares-Chauvet, Marie Schott, Lennart Höfer, A. Henssen, N. Rajewsky, B. Spanjaard, Anja I. H. Hagemann, J. Junker
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c57f7b414b240c72f02d9113a840fdad776a055" target='_blank'>
              BRCA2-deficiency Causes Global Transcriptomic Alterations in Endothelial Cells
              </a>
            </td>
          <td>
            Sepideh Nikfarjam, Aman Singh, Shuhan Bu, Hien C. Nguyen, Krishna K. Singh
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a185da472d5e884ad916a2af2cce445855a3e882" target='_blank'>
              Single-cell lineage trajectory defines CDK inhibitor-sensitive cells-of-origin in esophageal squamous cell cancer
              </a>
            </td>
          <td>
            Kyung-Pil Ko, Jie Zhang, Sohee Jun, Jae-Ll Park
          </td>
          <td>2025-10-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The recurrence of clinically advanced cancers is an evolutionary consequence of standard-of-care chemotherapies generally administered at maximum tolerated doses to kill as many cancer cells as possible. The inevitable appearance of resistance raises the possibility of shifting treatment goals from complete tumor eradication to long-term disease control. The latter approach is employed by adaptive therapy, which aims to inhibit the evolutionary dynamics governing the spread of resistant tumor phenotypes. Adaptive therapy changes focus from the cancer cells that are responsive to therapy to those that are resistant and ultimately govern outcome. This therapeutic approach retains a pool of sensitive cancer cells to compete with the therapy-resistant ones through dynamic dose modulation and/or timing. Thus, fluctuations of treatment-sensitive cells are used to control the resistant population and prolong tumor control with existing therapy agents. Here, we explore non-genetic mechanisms of resistance, including the protective role of the tumor stroma, the epithelial-to-mesenchymal transition, the overexpression of drug efflux pumps, and the extracellular vesicle-mediated transfer of them. These mechanisms can increase the size of the resistant population at the expense of the sensitive one, reducing the ability of adaptive therapy to force tumor evolution into controllable cycles.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5cd5596bbafd3812efcd1d8c0a4bfdf21b8096e0" target='_blank'>
              Adaptive cancer therapy: can non-genetic factors become its achilles heel?
              </a>
            </td>
          <td>
            Gábor Valcz, Robert A. Gatenby, B. Újvári, E. Buzás, Béla Molnár
          </td>
          <td>2025-09-22</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="In Mycobacterium tuberculosis, a prevalent and deadly pathogen, resistance to antibiotics evolves primarily through non-synonymous mutations in proteins. Sequence-based analyses are currently used to understand the genetic basis of antibiotic resistance, either via genotype-phenotype association, or via signals of convergent evolution. These methods focus on primary sequence and usually neglect other biological signals such as protein structural information. We hypothesize that integrating the structural context of mutations improves the prediction of effects on function and phenotype. We curate high confidence structural annotations for the M. tuberculosis proteome from 1,371 crystallography and 2,316 AlphaFold predictions, and combine the structures with mutations from over 31,000 clinical M. tuberculosis isolates. We demonstrate that mutations in proteins known to cause resistance are clustered in 3D space, even in proteins where inactivating mutations at any position are thought to cause resistance. We develop a statistic to search the M. tuberculosis proteome for signal of clustered non-synonymous mutations, finding over 450 proteins that display this signal, many of which have a known relationship with antibiotic resistance. Innovatively, we show that a supervised classifier trained on structure features alone has an F1 score of 94.6% at classifying mutations as resistance-conferring. This work demonstrates that protein structure provides useful information for categorizing which variants may cause antibiotic resistance, even when the majority of structures are AI-predicted.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/736eca4109e1243533868a8b023c33d1b51f37d5" target='_blank'>
              The structural context of mutations in proteins predicts their effect on antibiotic resistance
              </a>
            </td>
          <td>
            Anna G Green, Mahbuba Tasmin, R. Vargas, M. Farhat
          </td>
          <td>2025-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f8838d113f21c285c0455db7c93dfda5adc298f" target='_blank'>
              A switch from histone methyltransferase-EZH2 to demethylase KDM6A activity marks reinitiation of proliferation of drug treated colorectal cancer cells
              </a>
            </td>
          <td>
            Subhashree Chatterjee, Ritika Jaiswal, Aniruddha Roy, Shibasish Chowdhury, Sudeshna Mukherjee, Rajdeep Chowdhury
          </td>
          <td>2025-11-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/098344733a7758eefb37e7a0275fbd5613565b00" target='_blank'>
              Genomic and Epigenetic Interplay Drives Adaptive Fusion via Reproduction Trade-Off
              </a>
            </td>
          <td>
            Yang Chen, Lin Bi, Xiaoli Fan, Shuang Liang, Chengyuan Li, Yihong Wang, Sophie G. Martin, Gaowen Liu
          </td>
          <td>2025-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Metastatic breast cancer (MBC) remains the primary cause of mortality in breast cancer patients, driven by tumor heterogeneity, cellular evolution, and therapy-resistant clones. Traditional bulk transcriptomics, although informative, fail to capture rare subpopulations and context-specific gene expression, which are crucial for understanding disease progression. Single-cell transcriptomics (SCT) has emerged as a transformative approach, enabling high-resolution analysis of individual cells to reveal tumor composition, lineage dynamics, and transcriptional plasticity. This review highlights how SCT reshapes our understanding of MBC by mapping tumor evolution, identifying cancer stem-like cells, and characterizing states of epithelial-mesenchymal transition. We explore how SCT reveals clonal and spatial heterogeneity, and how tumor microenvironment components, including immune, stromal, and endothelial cells, interact with cancer cells to support immune evasion and the formation of a metastatic niche. SCT also uncovers mechanisms of therapeutic resistance, including transcriptional reprogramming and the survival of drug-tolerant subpopulations. Integrating SCT with spatial transcriptomics and multi-omics platforms offers a comprehensive view of the MBC ecosystem and may uncover novel therapeutic targets. We further discuss the translational potential of SCT for biomarker discovery, liquid biopsy development, and precision oncology. We address current technical challenges and future directions for clinical application. SCT is poised to transform MBC research and guide next-generation therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf97dd9ea39bc843d16cb57d14bc4af2cfce4592" target='_blank'>
              Single-cell transcriptomics in metastatic breast cancer: mapping tumor evolution and therapeutic resistance
              </a>
            </td>
          <td>
            Xu Han, Xin Li, Ling Bai, Gangling Zhang
          </td>
          <td>2025-10-23</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Resistance to targeted cancer therapies is a significant barrier to favorable treatment outcomes. Malignant cells can tolerate and resist drug treatments due to their biological flexibility. Specifically, slow-cycling drug-resistant cells may achieve permanent resistance to the treatment or restore sensitivity upon cessation of therapy. Enhancing cancer treatment methodologies necessitates a deeper understanding of the adaptability of tumor cells. Drug resistance and cellular heterogeneity are closely associated with cancer cell adaptability. Alterations in cellular signaling, interactions with the tumor microenvironment, and genetic and epigenetic alterations are all implicated. Analyzing these pathways will enhance our understanding of how cancer cells evolve and evade treatment. Two effective strategies to address cancer cell adaptability are to target specific biological pathways and to employ combination therapies. The progression of cancer therapy methodologies relies on comprehending and exploring the concept of cancer cell adaptability. Understanding tumor heterogeneity and drug resistance necessitates identifying the cellular, molecular, and genetic processes that govern cancer cell plasticity. This understanding enables the development of more personalized and effective cancer therapies, leading to improved treatment outcomes. Not applicable.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/714e22df09170482d5d0d601474075162f529a30" target='_blank'>
              Cancer cell plasticity and therapeutic resistance: mechanisms, crosstalk, and translational perspectives
              </a>
            </td>
          <td>
            S. Ghorbian
          </td>
          <td>2025-09-26</td>
          <td>Hereditas</td>
          <td>1</td>
          <td>15</td>
        </tr>

        <tr id="Phenotypic plasticity is a key mechanism of metastatic progression and cancer therapy resistance. This hallmark of human malignancies is enabled by highly conserved epigenetic mechanisms that control gene expression. Functional alterations in DNA methylation and histone post-translational modifications have been extensively described as drivers of metastatic dissemination and therapy resistance. Pharmacological inhibitors of epigenetic enzymes can revert these alterations, thereby stopping cancer progression and counteracting the emergence of resistant clones. Despite promising pre-clinical evidence, the clinical implementation of epigenetic therapies in solid cancers has led to disappointing results. Several factors can explain these challenges, including the lack of rational combinations. Notably, response to epigenetic treatments can be heterogeneous and short-lived. A liquid biopsy technology that allows the measure of specific epigenetic alterations enables patient selection and therapy monitoring, leading to the development of precision epigenetic therapies. In this review, we discuss the state of the art of this emerging treatment modality, and we identify key challenges that need to be overcome to reach the full potential of this new therapeutic concept.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/876ad36ab7595e359c53da7e87448ed401afa714" target='_blank'>
              Precision epigenetic therapies in oncology
              </a>
            </td>
          <td>
            Arundhati Chaudhary, Kayleigh J A Orchard, F. Salani, Theodora Partsou, Mark Eccleston, Guido Bocci, A. Italiano, Francesco Crea
          </td>
          <td>2025-09-19</td>
          <td>Cancer Metastasis Reviews</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df66d9d10b79290100e939f48a6a4eda6aef615a" target='_blank'>
              NR0B1 alters the 9-cis-retinoic acid response in Ewing Sarcoma cells
              </a>
            </td>
          <td>
            Benjamin D. Sunkel, Meng Wang, Rachel D Dreher, M. Sammons, Lindsay Ryan, A. Gross, Scott Friedland, Ryan D. Roberts, Emily R. Theisen, Elaine R. Mardis, Richard K. Wilson
          </td>
          <td>2025-10-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cf3f4a8479510c49ff95777ff501eb3f2838138" target='_blank'>
              Epigenetic priming promotes tyrosine kinase inhibitor resistance and oncogene amplification.
              </a>
            </td>
          <td>
            Rebecca M. Starble, Eric G Sun, Rana Gbyli, Jonathan S. D. Radda, Jiuwei Lu, Tyler B Jensen, Ning Sun, Nelli Khudaverdyan, Ting Zhao, Bomiao Hu, M. Melnick, Shuai Zhao, Nitin Roper, G. Wang, A. Tackett, Yinsheng Wang, Jikui Song, Katerina Politi, Siyuan Wang, Andrew Z Xiao
          </td>
          <td>2025-10-23</td>
          <td>Nature structural & molecular biology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e127ecdb0d709635e11801397bb592a991279b49" target='_blank'>
              Locus-specific transcriptional regulation of transposable elements by p53
              </a>
            </td>
          <td>
            Julia M. Freewoman, Andrew J. Rosato, Thomas M. Russell, Feng Cui
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Retroelements are repetitive sequences that comprise 42% of the human genome and are strictly regulated through various epigenetic mechanisms. Examining retroelement expression on a locus-specific level in relation to cancer can uncover distinct disease signatures. Using over 5000 RNA-sequencing samples, we assessed retroelement transcription across 23 tissue systems, 159 cell types, 1019 cancer cell lines, and cells isolated from various stages of embryogenesis using the specialized software tool, Telescope. In healthy individuals, 11,388 retroelements were found to be actively transcribed and dynamically regulated in a tissue- and cell type-dependent manner. Using the adult human body as a reference, we observed that 94% of cancer cell lines displayed elevated transcription of at least one cancer-specific retroelement, providing a three-fold larger reservoir of cancer biomarkers (1182) than our comparable analysis of annotated protein-coding genes (338). The precise retroelements that were transcribed following tumorigenesis were influenced by the originating location, with cancers of the blood, lungs, and soft tissue displaying the highest origin-specific activation. Moreover, nearly half of the cancer-specific retroelement loci, mostly from the HERV-H family, were found to be expressed during early embryonic development. Our data demonstrate that elevated transcription of certain tissue-specific and embryonic retroelements can be considered as a hallmark of tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e98e92e95f88b68527f2a00c19cdab4d4246b04c" target='_blank'>
              Strict retroelement regulation is frequently lost following cancer transformation and generates a promising reservoir of cancer biomarkers
              </a>
            </td>
          <td>
            Eric Russ, Sergey Iordanskiy
          </td>
          <td>2025-10-03</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="G-quadruplexes have been identified as a promising anti-cancer target because of their ability to modulate the stability of mRNAs encoding oncogenes, tumor suppressor genes, and other potential therapeutic targets. Deregulation of DNA damage and Unfolded Protein Response pathways in cancer cells may create vulnerabilities that can be exploited therapeutically. Previous studies have shown variations in the relative expression of DDR and UPR components in canine lymphoma and leukemia cell lines CLBL-1, CLB70, and GL-1. In the present study, we report the presence of G-quadruplex structures in these canine cell lines. Downregulation of the expression of DDR and UPR components at the mRNA level was observed in the CLBL-1 and CLB70 cell lines after stabilization of G4 structures using the ligand PhenDC3. In contrast, in GL-1 cells, important components of the DDR pathway, such as PARP1, GADD45A, and PIK3CB were upregulated in response to PhenDC3 treatment. Downregulation of DDIT4 mRNA expression, which encodes an important UPR component, was detected in the CLBL-1 and GL-1 cell lines after PhenDC3 exposure. These results suggest that G4 structures can be used to manipulate the expression of potential targets to treat lymphoma in dogs. A substantial enrichment of DNA replication and pyrimidine metabolism pathways was found in the GL-1 cell line after G4 stabilization. This finding suggests that PhenDC3 may induce DNA replication stress in this cell line. Collectively, these results support the feasibility of employing canine cancer cells as a model system to investigate the role of G-quadruplex structures in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/266b5cfaedc49c459cf184dd7b4aeb1ea558a388" target='_blank'>
              Stabilization of G-Quadruplexes Modulates the Expression of DNA Damage and Unfolded Protein Response Genes in Canine Lymphoma/Leukemia Cells
              </a>
            </td>
          <td>
            Beatriz Hernández-Suárez, David A. Gillespie, Ewa Dejnaka, B. Obmińska-Mrukowicz, A. Pawlak
          </td>
          <td>2025-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aa96a0707b7c6c78fe5bd4b2a45727093019bee" target='_blank'>
              Targeting the crosstalk of metabolism reprogramming and replication stress: novel strategy to combat cancer
              </a>
            </td>
          <td>
            Wenbo Liu, Xiangyan Jiang, Yong Ma, Huiguo Qing, Yingcun Bao, Zuoyi Jiao
          </td>
          <td>2025-09-26</td>
          <td>Medical Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 Hypoxia is a hallmark of the glioblastoma (GB) and a critical driver of tumor progression, cellular plasticity, and resistance to standard therapy. While its role in metabolic reprogramming is established, the mechanisms by which hypoxia shapes tumor evolution at the genomic and epigenetic levels remain poorly defined.



 To investigate the impact of hypoxia on glioblastoma evolution, we employed a multimodal approach that integrated multi-omic spatial transcriptomics and whole genome DNA methylation profiling, to explore spatial-hypoxia depended genomic alterations. We developed a hypoxia ex-vivo culturing system using human GB tissue slices analyzed by RNA-seq whole genome sequencing and whole genome methylation profiling. Further, an immunofluorescence imaging based machine learning method was established to evaluate cell cycle arrest and lineage transitions within an intact tissue architecture.



 Spatial analysis of patient-derived samples revealed that hypoxia-enriched niches are closely associated with chromosomal instability, particularly marked by the published CX2 signature—defined by short, oscillatory single-copy alterations indicative of homologous recombination defects. In ex vivo tumor slices, hypoxia induced a distinct transcriptional program characterized by activation of the unfolded protein response, metabolic reprogramming, and G0-G1 cell cycle arrest, as confirmed by immunofluorescence image analysis and consistent with prior spatial transcriptomics data. Simultaneously, hypoxia promoted epigenetic remodeling at the promoters of key neurodevelopmental genes, including WNT8B and HMGB4, pointing toward a priming of cells into an early neuronal progenitor-like state and increased chromatin accessibility of genes regulating the G0-G1 to G1-S transition. Notably, whole-genome profiling of hypoxia-treated slices identified copy number gains converging in MAPK signaling genes of BRAF/RAF fusions—alterations absent in conventional glioblastoma cell lines. Collectively, these findings highlight a tissue-specific mechanism by which hypoxia drives oncogenic signaling and neuronal lineage plasticity.



 Our results define a hypoxia-driven evolutionary program in GB, characterized by concomitant BRAF/MAPK pathway activation and neuronal epigenetic reprogramming. These adaptations promote the emergence of slow-cycling, therapy-resistant subclones poised for recurrence. Targeting the hypoxia-MAPK-lineage axis may present a promising therapeutic strategy to overcome resistance and delay GB relapse.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af2ad53cd3cc41596d1b40ea865e837da4df51f1" target='_blank'>
              P02.19.B BRAF/MAPK ACTIVATION AND NEURONAL LINEAGE PRIMING DEFINE A HYPOXIA-DRIVEN EVOLUTIONARY PROGRAM IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            G. Villa, N. Conrad, G. Kastner, J. Kückelhaus, J. Benotmane, Y. Yabo, J. Zhang, R. Sankowski, O. Schnell, D. Delev, N. Neidert, D. Heiland
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Ultra-accurate detection of rare somatic mutations is critical for understanding mutational processes in human disease, aging, and environmental exposures, yet current methods are limited by error rates, restricted genome coverage, and high DNA input. We present UDSeq, a duplex sequencing protocol combining random fragmentation, efficient UMI ligation, and quantitative input control to achieve near-complete genome/exome representation from as little as 100 pg DNA. Benchmarking in human sperm estimates a UDSeq error rate of ∼2.5×10-9 per base pair. UDSeq captures mutational signatures from heterogeneous populations without clonal expansion, reproduces exposure-specific patterns in cell lines and rodent models, and enables cross-species profiling. Compared with prior duplex methods, UDSeq yields up to fourfold more usable duplex molecules, improves library conversion, and remains cost-effective. We include a step-by-step protocol with quality-control checkpoints for fragment size, ligation yield, library conversion, and duplication rate. UDSeq provides a scalable, low-input platform for accurate profiling of somatic mutagenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a25aa13b255931a4c667fbbd3b80aa9966be011" target='_blank'>
              A Universal Duplex Sequencing Approach for Accurate Detection of Somatic Mutations
              </a>
            </td>
          <td>
            Shuvro P. Nandi, Yuhe Cheng, Shams Al-azzam, Safa Saeed, Audrey Kristin, Nadia Sunico, Isabella Stuewe, Zichen Jiang, Luka Culibrk, Maria Zhivagui, Xiaoxu Yang, Rachel Wise, Foster C. Jacobs, Bérénice Chavanel, M. Korenjak, Mia Petljak, S. Balbo, Laurie G Hudson, Ke Jian Liu, Jiri Zavadil, Joseph G. Gleeson, Ludmil B. Alexandrov
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf3ca6346229918f35167d16749694c1e3b771aa" target='_blank'>
              STAG2 cohesin cooperates with DREAM to maintain quiescence and suppress tumourigenesis in the urothelium
              </a>
            </td>
          <td>
            Maria Ramal, E. Lapi, M. Kalisz, Jaime Martínez de Villarreal, Meritxell Novillo-Font, M. Marqués, E. Zarzuela, Marta Isasa, Osvaldo Graña-Castro, Jianming Xu, Ana Cayuela, François Radvanyi, Mireia Novell, Jorge Martínez-Torrecuadrada, M. Barallobre, I. Bernard-Pierrot, María Marti-Marimon, Maria L Arbonés, M. Marti-Renom, Francisco X. Real
          </td>
          <td>2025-10-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0f43327ba689a502fd2aba63a16648713112594" target='_blank'>
              Epigenomic Heterogeneity of Non-Functional Pancreatic Neuroendocrine Tumors Uncovered by Single nucleus and Spatial ATAC Profiling
              </a>
            </td>
          <td>
            Dejiang Wang, Xiangjun Di, Fu Gao, Gaocai Li, Liyun Lin, Shujie He, Di Zhang, Junyi Yu Jin, Yuanxin Liang, Michael Cecchini, Jill Lacy, John Kunstman, Pamela Kunz, Jing Du, Yang Liu
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3948f28751f8205d2389f7d1df87013dbea193d3" target='_blank'>
              RUNX1T1-HDAC Reprogramming of the HOX Code Signaling Drives a Targetable Pan-Cancer Lineage Plasticity
              </a>
            </td>
          <td>
            Yuyin Jiang, Siyuan Cheng, Catherine Yijia Zhang, Xiao Jin, Yaru Xu, I. Y. Kim, Ping Mu
          </td>
          <td>2025-10-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Isocitrate dehydrogenase 1 (IDH1) mutations are key drivers of glioma biology, influencing tumor aggressiveness and treatment response. To elucidate their molecular impact, we performed proteome analysis on patient-derived (PD) and U87MG glioma cell models with either mutant or wild-type IDH1. We quantified over 6000 protein groups per model, identifying 1594 differentially expressed proteins in PD-AS (IDH1MUT) vs. PD-GB (IDH1WT) and 904 in U87MUT vs. U87WT. Both IDH1MUT models exhibited enhanced MHC antigen presentation and interferon signaling, indicative of an altered immune microenvironment. However, metabolic alterations were model-dependent: PD-AS cells shifted toward glycolysis and purine salvage, while U87MUT cells retained oxidative phosphorylation, potentially due to D2-hydroxyglutarate (2OHG)-mediated HIF1A stabilization. We also observed a predominance of downregulated DNA repair proteins in IDH1MUT models, particularly those involved in homologous recombination. In contrast, RB1 and ASMTL were strongly upregulated in both IDH1MUT models, implicating them in DNA repair and cellular stress responses. We also found distinct expression patterns of proteins regulating histone methylation in IDH1MUT cells, favoring increased methylation of H3K4, H3K9, and H3K36. A key driver of this may be the upregulation of SETD2 in PD-AS, an H3K4 and H3K36 trimethyltransferase linked to the recruitment of HIF1A as well as DNA mismatch repair proteins. This study uncovers candidate biomarkers and pathways relevant to glioma progression and therapeutic targeting, but also underscores the complexity of predicting glioma pathogenesis and treatment responses based on IDH1 mutation status. While proteome profiling provides valuable insights, a comprehensive understanding of IDH1MUT gliomas will likely require integrative multi-omics approaches, including DNA/RNA methylation profiling, histone and protein post-translational modification analyses, and targeted DNA damage and repair assays.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2740eea5fa4be364de22267706ccdb117ba04ffc" target='_blank'>
              Proteomic Analysis Uncovers Enhanced Inflammatory Phenotype and Distinct Metabolic Changes in IDH1 Mutant Glioma Cells
              </a>
            </td>
          <td>
            Sigrid Ravn Berg, Alessandro Brambilla, Lars Hagen, Animesh Sharma, C. Vågbø, N. Liabakk, Miroslava Kissova, Miquel Arano Barenys, Magnar Bjørås, S. Torp, Geir Slupphaug
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD), a subtype of non-small cell lung cancer (NSCLC), has high incidence and poor prognosis. Although anti-programmed cell death protein 1 (PD1) therapy and epidermal growth factor receptor (EGFR) inhibitors benefit some patients, many remain unresponsive. Genome-wide association studies (GWAS) can identify risk loci, while single-cell transcriptomics enables exploration of genetic variations and mechanisms in LUAD. We integrated GWAS data from FinnGen, IEU OpenGWAS, and GWAS Catalog with single-cell and spatial transcriptomics from GEO to investigate LUAD genetics and pathology. GWAS analyses identified SNP loci, risk factors, LD Score, and SMR results, revealing a novel apoptosis-related locus, Serine/Threonine Kinase 24 (STK24) (SMR P < 0.05). Single-cell analysis identified STK24-positive epithelial cells (STK24posEpi) and their transcription factors (SCENIC), developmental trajectory (Monocle), and tumor microenvironment roles (CellCall). Spatial mapping (RCTD) and trajectory analysis (stlearn) were combined with cell communication (CellChat), spatial signal flow (Commot), and co-localization (Misty). Bayesian deconvolution linked STK24posEpi to TCGA-LUAD bulk transcriptomes, assessing overall survival (OS), immune infiltration, and clinical traits. A prognostic model (STK24SuperPC, mean c-index 0.61) was built using multiple machine learning algorithms to predict survival and immunotherapy response. Laboratory experiments tested STK24’s effects on LUAD cell proliferation and apoptosis. STK24 was identified as a LUAD risk factor (pSMR < 0.05). STK24posEpi communicated with fibroblasts, endothelial cells, and others via MIF and PDGF pathways, and was associated with apoptosis (Importance > 0.5), metastasis (Importance > 2), DNA repair (Importance > 1), and DNA damage (Importance > 2). Located at the origin of the trajectory, STK24posEpi correlated with poor prognosis (HR > 0, P < 0.05) and reduced lymphocyte infiltration (Pearson’s r < 0, P < 0.05). STK24 overexpression enhanced LUAD cell proliferation, migration, and colony formation, while reducing apoptosis, increasing BCL-2 and Caspase-3, and decreasing Cleaved-Caspase-3 expression. STK24, an apoptosis-related gene, promotes LUAD progression by enhancing proliferation and inhibiting apoptosis, with STK24posEpi interacting in the tumor microenvironment through PDGF and MIF pathways. These findings offer potential therapeutic targets and support personalized LUAD treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f038883aea606036c62cbeb3f18e89605908569d" target='_blank'>
              Genome-wide association, single-cell, and spatial transcriptomics analyses reveal the role of the STK24-expressing positive cells in LUAD progression and the tumor microenvironment, identifying STK24 as a potential therapeutic target
              </a>
            </td>
          <td>
            Junzhi Liu, Huimin Li, Yuheng Jiao, Meng Hou
          </td>
          <td>2025-10-30</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec7c296f01ad5b58abb7169e8898ff697ed61fde" target='_blank'>
              Pan-cancer analysis of patient-specific gene regulatory landscapes identifies recurrent PD-1 pathway dysregulation linked to outcome
              </a>
            </td>
          <td>
            T. Belova, Ping-Han Hsieh, Ladislav Hovan, Daniel Osorio, M. Kuijjer
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Chemoresistance remains a major challenge in addressing T-cell lymphoblastic lymphoma/leukemia (T-LBL/ALL), underscoring the necessity for novel strategies to unravel the molecular factors driving resistance. Through transcriptomic profiling, circBPTF was found to be markedly overexpressed in chemoresistant samples. Further functional experiments demonstrated that BPTF-665aa, the protein product of circBPTF, plays a pivotal role in mediating resistance. Notably, BPTF-665aa prevents the ubiquitination degradation of full-length BPTF, and promotes chromatin accessibility at key promoter sites, such as that of c-Myc promter 2 (P2), facilitating transcriptional activation crucial for cellular survival and proliferation under therapeutic stress. Structural studies confirmed the motifs of BPTF-665aa, including the Plant Homeodomain (PHD) finger and Bromodomain, essential for its chromatin remodeling function. HY-B0509 was identified as a small-molecule inhibitor of BPTF-665aa, with molecular docking and dynamics simulations showing stable binding to critical residues within the protein’s active site. Overall, this study introduces a new mechanism where circBPTF affects chromatin accessibility, causing chemoresistance, making BPTF-665aa as a potential therapeutic target for treating T-LBL/ALLs. This study identifies BPTF-665aa, a protein translated from circBPTF, as a key regulator of chromatin accessibility in T-cell lymphoblastic lymphoma/leukemia (T-LBL/ALL). BPTF-665aa promotes chromatin remodeling at oncogenic loci, including the c-Myc promoter, facilitating transcriptional activation of resistance-associated genes. Furthermore, we identify a small molecule inhibitor targeting BPTF-665aa, highlighting its potential as a therapeutic strategy. These findings establish a new paradigm linking circRNA-derived proteins and chromatin architecture, positioning BPTF-665aa as a promising therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17596bccd73211f83b30b6a17b37b615a3eeedbc" target='_blank'>
              BPTF-665aa mediate chromatin remodeling drives chemoresistance in T-LBL/ALL
              </a>
            </td>
          <td>
            , Mei Li, Zhen-Zhong Zhou, , Yong Zhu, Yuan Zhang, Xu Liu, Hai-Long Li, Jing Feng, Li-Yan Song, , , , Wei-Juan Huang
          </td>
          <td>2025-11-07</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5994676ec22291e594d0de0ef39f5ccc905b6ab" target='_blank'>
              Integrative analysis of spatiotemporal transcriptomics delineates dynamic cell states in squamous tumorigenesis
              </a>
            </td>
          <td>
            Varsha Sreekanth, Timur Rusanov, Farah Yousef, Mark A Sanborn, Su Yeon Yeon, Vijayalakshmi Ananthanarayanan, Olga Karginova, Jalees Rehman, A. Salahudeen
          </td>
          <td>2025-10-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf5c0b9b2392adef20fc428fa46257f3b2a9438f" target='_blank'>
              Persistent interferon signaling and clonal expansion mark early events in DNA methylation damage-induced liver cancer
              </a>
            </td>
          <td>
            Lee J. Pribyl, Jennifer E. Kay, J. Corrigan, Lindsay B. Volk, Monét Norales, Norah A. Owiti, Evan A. Kowal, Ishwar N. Kohale, Ilana S Nazari, Matilda R. Swanson, Aimee C. Moise, Duanduan Ma, Stuart S. Levine, Emily Michelsen, R. G. Croy, Timothy Ragan, D. Torous, Svetlana L. Avlasevich, S. Dertinger, Sebastian E. Carrasco, Leona D. Samson, J. Essigmann, Forest M. White, B. Engelward
          </td>
          <td>2025-10-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5143128d7c214ae6fd617f30d5f2500e3f9594d8" target='_blank'>
              A Unifying Mechanism for Shared Splicing Aberrations in Splicing Factor Mutant Cancers
              </a>
            </td>
          <td>
            Prajwal C. Boddu, Rahul Roy, Stephen Hutter, Francis Baumgartner, Wenxue Li, Gabriele Todisco, F. Ficara, M. G. Della Porta, Yansheng Liu, T. Haferlach, Manoj M. Pillai
          </td>
          <td>2025-10-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Treatment-induced neuroendocrine prostate cancer (NEPC) represents a lethal evolution of prostate adenocarcinoma under androgen receptor pathway inhibition, posing a significant clinical challenge. In a recent landmark study, Wang et al. introduced an innovative internal Z-score based approach to comprehensively characterize the transcription factor (TF) landscape in prostate cancer progression, uncovering distinct TF profiles associated with adenocarcinoma and NEPC lineages. Notably, the study proposes a three-phase model of NEPC transdifferentiation—comprising de-differentiation, dormancy, and re-differentiation—revealing dynamic shifts in TF expression that underpin lineage plasticity and therapeutic resistance. This commentary critically evaluates the methodological advancements, the functional significance of the identified TF signatures, and the broader implications of these findings for developing novel therapeutic strategies. By delineating the molecular events driving the transition from androgen receptor (AR)-dependent adenocarcinoma to treatment-resistant NEPC, this work underscores the potential of targeting early and dormant phases of transdifferentiation to improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f432a498dcbb709202897ddd04a1b59a52e28fa2" target='_blank'>
              Transcription Factor Dynamics in Neuroendocrine Prostate Cancer Development
              </a>
            </td>
          <td>
            Yu Wang, Yuzhuo Wang
          </td>
          <td>2025-10-24</td>
          <td>Holistic Integrative Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/408e4356836091a48672426b49d0e709f36de3ec" target='_blank'>
              The transcriptional regulatory circuit as a driver and therapeutic target in CML blast crisis
              </a>
            </td>
          <td>
            Liling Jiang, Enzhe Lou, Jinxin Fang, Lizhen Jiang, Guanjie Peng, Jinghong Chen, Bingyuan Liu, Bo Lu, Yi Meng, Haichuan Zhang, Aochu Liu, Qiong Mao, Peilong Lai, Yueyuan Zheng, Jinbao Liu, Xianping Shi
          </td>
          <td>2025-11-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78f9f6afd97add9714e69419ecf796522c175c3f" target='_blank'>
              CRISPRi Screen Identifies a Novel Growth Suppressor lncRNA, INSTAR, in Human Monocytes
              </a>
            </td>
          <td>
            Christy Montano, Eric Malekos, S. Covarrubias, Sol Katzman, Lisa Sudek, Jillian Ward, S. Carpenter
          </td>
          <td>2025-10-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Primary prostate cancer presents with multifocal lesions and unpredictable clinical behavior, posing significant challenges for effective prognosis. To address this, we investigate the epigenomic landscape of prostate tumor biopsies from 25 treatment-naïve male patients by analyzing chromatin compartmentalization patterns. Our analysis reveals two distinct molecular subtypes: one with a Low Degree of Decompartmentalization (LDD) and another with a High Degree of Decompartmentalization (HDD). Here we show that the HDD subgroup exhibits extensive chromatin reorganization associated with diminished oncogenic potential. This subtype shows repression of molecular pathways involved in extracellular matrix remodeling and cellular plasticity. From this distinction, we derive an 18-gene transcriptional signature capable of differentiating HDD from LDD cases. Importantly, we validate the prognostic relevance of this signature in multiple independent cohorts totaling over 900 patients. Our findings suggest that epigenetic-derived signature at the time of diagnostic biopsy can offer a powerful tool for risk stratification in prostate cancer. Prostate cancer (PCa) evaluation remains challenging due to its heterogenous and multifocal nature. Here, the authors reconstruct the nuclear chromatin compartmentalization of PCa patients-derived biopsies distinguishing two epigenetic subtypes and deriving an 18-gene signature with prognostic value on TCGA samples that is further validated with multiple independent PCa cohorts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/696cb38bd331720bed277dcfa9f4701c78f389c7" target='_blank'>
              Chromatin remodeling restrains oncogenic functions in prostate cancer
              </a>
            </td>
          <td>
            V. Rosti, Giovanni Lembo, C. Petrini, Francesca Gorini, Roberto Quadri, C. Cordiglieri, Margherita Mutarelli, Elisa Salviato, Elisabetta Casari, Emanuele Di Patrizio Soldateschi, Emanuele Montanari, G. Albo, Francesco Ripa, Alessandra Fasciani, Mariacristina Crosti, E. De Lorenzis, M. Maggioni, V. Vaira, Maria Vivo, Francesco Ferrari, Chiara Lanzuolo
          </td>
          <td>2025-10-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a leading cause of cancer-related mortality, necessitating the development of novel therapeutic strategies. In the present study, we identified TP53-regulating kinase (TP53RK) as a critical regulator of CRC cell survival and proliferation using a custom clustered regularly interspaced short palindromic repeats (CRISPR)/CRISPR-associated protein 9 library screen targeting serine/threonine kinase-related genes. TP53RK was significantly overexpressed in CRC tissues and was correlated with copy number amplification. Functional validation revealed that TP53RK depletion induced DNA replication stress, apoptosis, and cell cycle arrest, independent from p53 status. Mechanistically, TP53RK stabilized cell division cycle 7 (CDC7), a key kinase regulating DNA replication origin activation, ensuring robust minichromosome maintenance complex protein (MCM) phosphorylation and replication fork progression. Disruption of the TP53RK-CDC7 axis led to reduced MCM2 enrichment at replication origins and impaired DNA replication dynamics. Moreover, TP53RK overexpression sensitized cells to DNA replication stress (aphidicolin) and CDC7 inhibition (XL413), highlighting its potential as a therapeutic strategy. These findings establish TP53RK as a pivotal regulator of DNA replication fidelity and genomic stability, thereby providing a promising therapeutic target for CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a87356ada79d745ccc5da3c1105d2fc2922b6dd0" target='_blank'>
              Identifying TP53RK as a key regulator of colorectal cancer survival and a potential therapeutic target
              </a>
            </td>
          <td>
            Younghee Choi, Seowoo Park, Seojin Yang, Eun-ju Kim, Youngwon Cho, Hye-Ju Yang, Eugene C Yi, Sang-Hyun Song, Tae-You Kim
          </td>
          <td>2025-10-16</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Transcriptional regulatory elements (TREs) orchestrate gene expression programs fundamental to cellular identity and transitions between physiological and pathological states. Decoding the regulatory logic of human biology requires resolving where, when, and how these elements are transcriptionally engaged. Here, we profiled the active transcriptional regulatory landscape across all major organ systems and a broad spectrum of developmental and disease states using PRO-cap, a high-resolution method that captures nascent transcription start sites with unprecedented sensitivity and specificity. This atlas of active TREs highlights elements shaped by their cellular contexts and evolutionary constraints, sheds light on the genetic architecture of human traits and diseases, and reveals how patterns of transcription initiation and pausing encode regulatory logic. In cancer, nascent transcription enables the delineation of lineage-specific regulatory states, metastatic adaptations, and the co-option of pre-existing programs. Together, these findings establish nascent transcription as a core dimension of gene regulation, illuminating principles that govern development, physiology, and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfa390b89a37848a8f83c759acd4c4c93e92f26e" target='_blank'>
              The regulatory landscape of nascent transcription in human health and disease
              </a>
            </td>
          <td>
            Sagar R. Shah, You Chen, A. K. Leung, Priscila V. Castilho Navarro, Mauricio I. Paramo, Juhi Gupta, Aishwarya Gurumurthy, Rebuma Firdessa Fite, Annika K. Weimer, Qian Du, Ahmed M. Mohyeldin, Dieter Egli, Remi J. Creusot, Russell J H Ryan, Michael P. Snyder, Andrew G. Clark, John T. Lis, Haiyuan Yu
          </td>
          <td>2025-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Nucleoside analogues are used as drugs and as labels in lab-based research. However, the effect of different nucleoside analogue mechanism(s) on cell sensitivity or mutagenesis is unclear. This is particularly important in cancer treatments where checkpoint proteins and DNA damage factors are often mutated. We tested 6 nucleoside analogues in the fission yeast, Schizosaccharomyces pombe. We found that the mutations in the DNA replication checkpoint cause unique sensitivity profiles towards chemotherapeutic nucleoside analogues (gemcitabine, 5-fluorouracil, cytarabine) and the non-clinical analogue bromodeoxyuridine. Antiretroviral compounds, zidovudine and lamivudine, did not alter cell growth. We compared half-maximal inhibitory concentration (IC50) doses between checkpoint deficient yeast strains, examining culture growth and DNA mis-segregation. Intriguingly, gemcitabine and bromodeoxyuridine doses above the IC50 promoted better growth. Above each compound’s IC50 dose we saw that cells were insensitive to nucleoside analogue re-exposure, particularly in DNA replication checkpoint mutants (cds1Δ, rad3Δ). Thus, pairing nucleoside analogue use with personal genomics may inform drug choice, dose, and schedule. Finally, these data indicate that resistance may be predictable, informing clinical strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea9673674f99d856da45d7531d1ed654d61e6d4f" target='_blank'>
              Checkpoint-Dependent Sensitivities to Nucleoside Analogues Uncover Specific Patterns of Genomic Instability
              </a>
            </td>
          <td>
            Zainab Burhanuddin Kagalwala, Mohammed Ayan Chhipa, Zohreh Kianfard, Essam Karam, Sirasie P. Magalage, Sarah A Sabatinos
          </td>
          <td>2025-07-14</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 CDK4/6 inhibitors exhibit therapeutic potential in several tumor entities. Their clinical efficacy in glioblastoma might be limited by intrinsic and acquired resistance mechanisms. Understanding underlying genetic and molecular factors of this resistance might inform the design of effective therapeutic strategies. Genome-wide CRISPR-Cas9 screens offer a powerful approach to systematically identify resistance drivers and synthetic lethal interactions, paving the way for rational combination therapies.



 We conducted genome-wide CRISPR-Cas9 knockout and activation screens in human glioma cell lines LN229 and GS9 under CDK4/6 inhibition using Brunello and Calabrese libraries. Top hits were validated using functional assays. Furthermore, we conducted bulk RNA sequencing of treated and untreated cells for transcriptomic insights into drug response modifiers and molecular pathways involved.



 Functional genomics identified Ambra1 and CCNE1 as key drivers of resistance. Both Ambra1 knockout and CCNE1 overexpression, conferred resistance across both long-term and stem-like glioma models. Additionally, knockout of CHEK1 and FAM122A uncovered synthetic lethal interactions with CDK4/6 inhibition. Functional validation confirmed that glioma cells rely on CHEK1 or FAM122A for cell cycle progression, with cell line specific sensitivity. Transcriptomic analyses consistently highlighted alterations in pathways associated with cell cycle regulation, DNA replication, and the Fanconi anemia pathway. Of note, several differentially expressed genes overlapped with screen hits, supporting the robustness of our findings. Notably, pharmacological inhibition of CHEK1, both in vitro and ex vivo, enhanced sensitivity to CDK4/6 inhibition in a sequence-dependent manner, indicating a critical role of the sequence of drug administration.



 Our functional genomics approach uncovers mediators of acquired resistance and synthetic lethality in the context to CDK4/6 inhibitors in glioma. Co-targeting of CHEK1 or FAM122A with CDK4/6 inhibitors represents a promising therapeutic strategy to overcome resistance and improve the therapeutic efficacy. Ongoing in vivo studies using xenograft and syngeneic glioma models aim to validate the survival benefit of these combinations and to elucidate their impact on the tumor microenvironment.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfaeb6b136b57493744dd9b571df497dca29c55b" target='_blank'>
              OS05.8.A DISSECTING RESISTANCE TO CDK4/6 INHIBITION IN EXPERIMENTAL GLIOMA VIA GENOME-WIDE CRISPR-CAS9 SCREENING
              </a>
            </td>
          <td>
            S. Surender, L. Haeusser, L. Kuhlburger, F. Tsiami, L. Maise, S. Nahnsen, S. Beck, D. Merk, G. Tabatabai
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3066257378e89ecfacf6bd297cae20bfa3bfa3db" target='_blank'>
              p53 activity licenses transcriptional regulation by YAP/TAZ-TEAD to shape expression landscapes during tumorigenesis
              </a>
            </td>
          <td>
            Cole Martin, William B. Sullivan, Scott Bang, Joseph R. Gomes, A. C. Edwards, C. Der, Cheng Fan, Yusha Liu, John P. Morris
          </td>
          <td>2025-10-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>124</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is characterized by rapid progression, therapeutic resistance, and a profoundly immunosuppressive tumor microenvironment. Emerging evidence suggests that endoplasmic reticulum (ER)-associated macromolecules play critical roles in tumor adaptation. In this study, we performed a multi-omics investigation of orosomucoid-like protein 2 (ORMDL2), a conserved ER membrane protein involved in sphingolipid biosynthesis and ER stress regulation, and uncovered its regulatory functions in GBM progression. Transcriptomic analyses across The Cancer Genome Atlas (TCGA), and Chinese Glioma Genome Atlas (CGGA) revealed elevated ORMDL2 expression in GBM tissues which causes poor prognosis. The MetaCore pathway and Gene Set Enrichment Analysis (GSEA) identified ORMDL2's involvement in antigen presentation via a major histocompatibility complex I (MHC class I), unfolded protein response (UPR), and mitochondrial apoptotic signaling. Single-cell RNA-sequencing data and the Human Protein Atlas showed ORMDL2 enrichment in tumor stromal cells. Pharmacogenomic correlation via the Genomics in Drug Sensitivity in Cancer (GDSC) and Cancer Therapeutics Response Portal (CTRP) database suggested that ORMDL2 expression was associated with resistance to DNA damage response inhibitors such as etoposide, doxorubicin, talazoparib, and might interact with sphingolipid-targeting compounds. Collectively, our findings establish ORMDL2 as a multi-functional macromolecular regulator of immune suppression and therapeutic resistance in GBM, providing new mechanistic insights and potential targets for translational medicines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f5a5f2b3cb520c8099ca5e5eaf8e443e9c0dbc4" target='_blank'>
              Targeting ORMDL2 in glioblastoma through integrated landscape of high-throughput sequencing and pharmacogenomic analysis
              </a>
            </td>
          <td>
            Fitria Sari Wulandari, Chih-Yang Wang, Sachin Kumar, Ngoc Phung Ly, Juan Lorell Ngadio, Dahlak Daniel Solomon, Do Thi Minh Xuan, Gabriela Natasha Sutandi, Hung-Yun Lin, Hui-Ru Lin, Yung-Kuo Lee, Hao-Chien Yang, Ming-Cheng Tsai, Ching-Chung Ko
          </td>
          <td>2025-09-27</td>
          <td>International Journal of Medical Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e33fb7e42f73e65912640c7f54217de318d7753d" target='_blank'>
              Base-editing a single missense mutation in A20 enhances CAR-T cell efficacy
              </a>
            </td>
          <td>
            Adam Blaisdell, Stefanie Bachl, Luis R. Sandoval, Carter Ching, C. Bowman, Nupura Kale, Manu Prabandham, Morgan Diolaiti, Claire Havig, Rommel Advincula, Nika Lenci, Zhongmei Li, Emily Yamashita, Charlotte H. Wang, Shimin Zhang, Qi Liu, Philip Achacoso, Dorothea Stibor, Inger Øynebråten, Jin Seo, Alan Ashworth, Alex Marson, C. Ye, B. Malynn, Justin Eyquem, J. Carnevale, Averil Ma
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Reconstructing and understanding intra-tumor heterogeneity, the coexistence of multiple genetically distinct subclones within the tumor of a patient, and tumor development is essential for resolving carcinogenesis and for identifying mechanisms of therapy resistance. While bulk sequencing can provide a broad view on tumoral complexity/heterogeneity of a patient, single-cell analysis remains essential to identify rare subclones that might drive chemotherapy resistance. In this study, we performed an integrated analysis of bulk and single-cell DNA sequencing data of core-binding factor acute myeloid leukemia patients, defined by the presence of a RUNX1::RUNX1T1 or CBFB::MYH11 fusion gene. By single-cell sequencing, we inferred tumor phylogenies for 8 patients at diagnosis including patient-specific somatic variants, somatic copy-number alterations and fusion genes, and studied clonal evolution under the pressure of chemotherapy for 3 patients. As a result, we developed an approach to reliably integrate subclonal somatic copy number alterations into phylogenetic trees and clonal evolution analysis, obtaining unprecedented resolution of intra-tumor heterogeneity in CBF AML. We were able to show that the fusion gene is among the earliest events of leukemogenesis at single-cell level. We identified remaining tumor clones in 6 patients with complete remission samples indicating incomplete eradication of the tumor clones. Here, we show that identifying the order of mutation acquisition can provide valuable insights into evolutionary history, offering a framework to improve drug selection in the era of targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f79dc8d9b0526060e06d2b2413c27f6af3f0f4c7" target='_blank'>
              Resolving intra-tumor heterogeneity and clonal evolution of core-binding factor acute myeloid leukemia patients with single-cell resolution
              </a>
            </td>
          <td>
            Raphael Hablesreiter, Paulina M. Strzelecka, Klara Kopp, Natalia Estrada, A. Dolnik, Marlon Tilgner, Coral Fustero-Torre, Felicitas R. Thol, F. Heidel, M. Heuser, L. Haghverdi, Lars Bullinger, F. Christen, Frederik Damm
          </td>
          <td>2025-10-28</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Anticancer drug resistance remains a significant challenge to the efficacy of cancer treatment, with DNA repair enzymes contributing to this resistance. We hypothesized that thymine DNA glycosylases (TDGs) may be involved in anticancer drug resistance given their dual function of DNA repair and demethylation as well as investigated their possible involvement in the induction of β-catenin in SNUC5 cells resistant to 5-fluorouracil (SNUC5/5-FUR) and oxaliplatin (SNUC5/OXTR). The expression of TDG and phospho-β-catenin increased in both resistant cell types when compared to that in SNUC5 cells. Moreover, knockdown of TDG significantly suppressed phospho-β-catenin expression in both resistant cell types, resulting in enhanced sensitivity to anticancer drugs. TDG binding to the β-catenin promoter was stronger in both resistant cell types than in SNUC5 cells, showing a decreased methylation pattern in the CpG islands of the β-catenin promoter. Furthermore, another DNA demethylase, ten-eleven translocation 1 (TET1), showed the same pattern as TDG in both resistant cell types. Additionally, TDG significantly interacted more with TET1 in both resistant cell types than in SNUC5 cells, enhancing binding to the same locus in the β-catenin promoter. These findings suggest that TDG may be a promising target molecule for overcoming drug resistance in colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/def655ba0cc554974966a66e65dd66380f445b21" target='_blank'>
              Involvement of the DNA Demethylase Thymine DNA Glycosylase in Anticancer Drug Resistance
              </a>
            </td>
          <td>
            K. Kang, M. Piao, Herath Mudiyanselage Maheshika Madhuwanthi Senavirathna, Hye-Jin Boo, Sang-Pil Yoon, Joo Mi Yi, Jinny Park, J. Hyun
          </td>
          <td>2025-10-15</td>
          <td>Biomolecules & Therapeutics</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Therapeutic resistance remains a critical barrier in oncology, frequently leading to cancer relapse after initial treatment response. Growing evidence suggests the presence of drug-tolerant persisters (DTPs), a rare subpopulation of cancer cells that survives chemotherapy by entering a reversible specific adaptation. Unlike classical cell resistance, the DTP phenotype is independent of genetic changes and maintained through dynamic regulatory mechanisms. DTPs are phenotypically heterogeneous and can exhibit stem-like and quiescent cell phenotypes, non- or slow proliferation, and remarkable plasticity due to a di-pause-like state and executing epithelial–mesenchymal transition (EMT) or transdifferentiation processes. Despite advances in research, the molecular mechanisms underlying DTPs’ biology and their role in cancer relapse remain only partially understood. The review summarizes the current progress in processes that lead to the acquisition of cellular persistence status, which, in turn, constitute areas of vulnerability that can be exploited in cancer therapy. We highlight anti-DTP therapeutic strategies, including epigenetic modification, cell signaling and transcriptional regulation, metabolic reprogramming, and modification of cell interactions within the tumor microenvironment. Furthermore, we focus on the potential role of nanomaterials in the combat against DTPs. Nanoparticles not only act as part of the drug delivery process, enabling precise DTP targeting and enhancing intracellular drug accumulation, but their intrinsic properties can also be used to eradicate DTPs directly or by enhancing the effectiveness of other therapeutic strategies. The integrated approach offers strong potential to eliminate tumor persistence, prevent recurrence, and improve long-term patient outcomes beyond conventional therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/495b98c4c7bc3661dae9aa58d4006e54132b7ae3" target='_blank'>
              Targeting Drug-Tolerant Persister Cancer Cells: Can Nanomaterial-Based Strategies Be Helpful for Anti-DTP Therapies?
              </a>
            </td>
          <td>
            Prachi Ghoderao, Eliza Kwiatkowska-Borowczyk, Sanjay Sahare, H. Dams-Kozlowska
          </td>
          <td>2025-11-04</td>
          <td>Pharmaceutics</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="


 Glioblastoma relapse remains inevitable, placing these tumors among the most difficult to treat. Treatment challenges include inherent drug resistance, tumour’s plastic nature, and suppressive tumour microenvironment (TME). In other cancers, treatment-induced plasticity has been associated with the emergence of drug-tolerant persister cells, which survive therapy by undergoing reversible phenotypic changes towards resistant states. The role of persisters in glioblastoma remains to be unravelled. It is unclear to what extent the responses are patient- and treatment-specific, and how much they depend on the tumor microenvironment.



 We applied a cohort of molecularly characterized patient-derived models to investigate patient-specific responses to chemotherapeutics (Temozolomide, VAL-083). Treatment efficacy and its correlation with molecular features were assessed ex vivo using organoids and stem-like cultures as well as in vivo through orthotopic xenografts. To mimic clinical treatment schedules, we assessed functional responses in time, incorporating prolonged exposure to treatment and drug holiday period over several weeks. Analysis included tumor cell viability, cell cycle, DNA damage and repair, senescence and quiescence features. Phenotypic adaptation of glioblastoma and microenvironmental cells in vivo were analysed with MRI, single-cell RNA-seq, spatial transcriptomics and IHC.



 Assessment of drug efficacy across a cohort of patient-derived models revealed resistant versus adaptive responder groups, which were treatment-specific. While resistant cells did not show functional changes in cell viability and cell cycle upon prolonged drug exposure, sensitive cells show a biphasic growth curve characteristic of persister cells. The adaptation towards the least proliferative phenotype was reversible during drug holiday period, further confirming a role of a transient drug-tolerant state. Chemotherapeutic treatment in orthotopic xenografts partially reduced tumor growth in vivo and activated apoptosis and DNA repair pathways in glioblastoma cells. These changes coincided with an increased infiltration of tumor-associated macrophages.



 Our findings highlight the critical roles of both tolerance and intrinsic resistance mechanisms. The responder groups appear to be specific to the type of treatment and the unique characteristics of each patient’s tumor, emphasizing the need to evaluate treatment-induced plasticity in a patient-specific context. Glioblastoma cells are further supported by the adaptation of the microenvironment towards a tumor-supportive crosstalk.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17ea2a43463b7a7c67d446cbd650a87fb74e2953" target='_blank'>
              P01.18.B MODELING GLIOBLASTOMA TREATMENT RESPONSES: INSIGHTS INTO TUMOR PLASTICITY AND THE ROLE OF THE TUMOR MICROENVIRONMENT
              </a>
            </td>
          <td>
            M. de Lucas Sanz, H. Allahverdi, B. Nosirov, P. Moreno Sánchez, M. Baroni Milan, A. Lipsa, M. Rezaeipour, C. Dording, D. Brown, J. M. Langlands, K. Grzyb, Y. Yabo, J. Zhang, D. Heiland, A. Michelucci, P. Nazarov, A. Gołȩbiewska
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="The dysregulation of the DNA damage response (DDR) plays a vital role in cancer development and inadequate response to anti-cancer therapy. We investigated the involvement of DNA polymerase theta (Polθ), the principal component of the microhomology-mediated end joining (MMEJ) pathway, in the pathogenesis of multiple myeloma (MM). We explored the expression levels of Polθ and assessed its prognostic relevance in a cohort of 124 MM patients at our institution. We demonstrated the impacts of Polθ inhibition on genomic stability and cell viability in MM using shRNA and a pharmacological inhibitor. To elucidate the role of Polθ-mediated MMEJ in repairing melphalan-induced DNA damage, we employed Western blot and EJ2-GFP reporter assay to evaluate the activation of the MMEJ pathway following melphalan exposure. We analyzed the synergistic cytotoxicity between the Polθ inhibitor and melphalan in MM, both in vitro and in vivo. The impacts of combination therapy on genomic damage and mitotic catastrophe were examined by comet assay, Western blot, and immunofluorescence. We revealed that Polθ was markedly upregulated in MM. The enhanced expression of Polθ was found to be significantly associated with advanced disease stages and adverse clinical outcomes. Polθ inhibition promoted genomic instability and impaired MM growth, underscoring the polymerase's function in sustaining myeloma survival under ongoing genomic stress. Furthermore, we observed a further increase in Polθ expression and MMEJ repair efficiency following melphalan treatment. Polθ inhibitors sensitized MM cells to melphalan by increasing unrepaired DNA damage and inducing mitotic catastrophe, both in vitro and in vivo. Collectively, our results confirm Polθ as a promising prognostic biomarker and highlight its potential to serve as a synthetic lethal target that can be used in combination with melphalan for the treatment of MM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a916ab34fef94b64abac58ae251e6ce253205a64" target='_blank'>
              Polymerase theta inhibition impairs tumor growth and amplifies melphalan-induced DNA damage in multiple myeloma
              </a>
            </td>
          <td>
            Qun Li, Chi Ma, Liping Zuo, Aoshuang Xu, Bo Zhang, F. Zhao, Lifei Liu, Xin Zhao, Yu Hu, Chunyan Sun
          </td>
          <td>2025-10-10</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The rapid evolution of sequencing technologies has profoundly advanced precision oncology. Whole-exome sequencing (WES), whole-genome sequencing (WGS), and whole-transcriptome sequencing (RNA-Seq) enable comprehensive characterization of tumor biology by detecting actionable mutations, gene fusions, splice variants, copy number alterations, and pathway dysregulation. These approaches also provide critical insights into biomarkers such as homologous recombination deficiency (HRD), tumor mutational burden (TMB), and microsatellite instability (MSI), which are increasingly essential for guiding therapeutic decisions. Importantly, comprehensive genomic profiling not only refines patient stratification for targeted therapies but also sheds light on tumor–immune interactions and the tumor microenvironment, paving the way for more effective immunotherapeutic combinations. WGS is considered the gold standard for detecting germline mutations and complex structural variants, while WES remains central for detecting somatic driver mutations that guide targeted therapies. RNA-Seq complements these methods by capturing gene expression dynamics, identifying clinically relevant fusions, and revealing mechanisms of resistance. Together with advances in bioinformatics and artificial intelligence, these tools translate molecular data into actionable strategies for patient care. This review integrates insights from WGS, WES, and RNA-Seq with an overview of FDA- and EMA-approved targeted therapies, organized by tumor type, and highlights the molecular signaling pathways that drive cancer development and treatment. By bridging genomic profiling with regulatory-approved therapies, we outline current advances and future perspectives in delivering personalized cancer care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28dfbd87c701593d0e68f1b2d6aa4002ef2eff85" target='_blank'>
              Advances in Precision Oncology: From Molecular Profiling to Regulatory-Approved Targeted Therapies
              </a>
            </td>
          <td>
            P. Brlek, V. Škaro, Nenad Hrvatin, Luka Bulić, Ana Petrović, Petar Projić, Martina Smolić, Parth Shah, D. Primorac
          </td>
          <td>2025-10-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Purpose FOS-like antigen 1 (FRA1), encoded by FOSL1, is an inducible subunit of the AP-1 transcription factor complex and regulates gene expression in response to proliferative and environmental cues. Although FRA1 has been linked to cancer progression, its role in early transformation and radiation responses remains unclear. Methods CRISPR-engineered human CGL1 cells—a hybrid of HeLa and normal fibroblasts—were used to evaluate the impact of FRA1 overexpression and knockout on neoplastic transformation. Transformation frequency, clonogenic survival, DNA damage recognition and repair, and cell cycle distribution were assessed following irradiation. Transcriptomic profiling was performed under baseline and serum-stimulated conditions. Results FRA1 loss markedly increased both spontaneous and radiation-induced transformation frequency, while overexpression suppressed transformation under both conditions. FRA1-deficient cells were sensitized to radiation-induced cell killing, despite intact DNA damage recognition and repair. In contrast, FRA1 overexpression promoted G2/M accumulation post-irradiation, suggesting enhanced checkpoint activation. Transcriptomic profiling revealed that FRA1 remodels AP-1 complex composition and functions as a transcriptional repressor of mitogen- and stress-responsive genes. FRA1-mediated repression was observed across gene networks involved in extracellular matrix remodeling, hypoxia signaling, inflammation, and proliferation, under both baseline and serum-stimulated conditions. Conclusion These findings establish FRA1 as a key modulator of neoplastic transformation and radiation response, acting primarily through transcriptional repression of pro-tumorigenic signaling pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1eb780bfa4e21042a750aefe2ef72a09f6e8a78f" target='_blank'>
              FRA1 (FOSL1) suppresses neoplastic transformation and modulates radiation responses via transcriptional control of mitogenic and stress-responsive networks
              </a>
            </td>
          <td>
            W. Al-Khayyat, T. Laframboise, Jessica Dougherty, Marc S. Mendonca, D. Boreham, T. C. Tai, Christopher Thome, S. Tharmalingam
          </td>
          <td>2025-09-22</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Emerging evidence suggests that post-translational modifications (PTMs) contribute to osteosarcoma pathogenesis, yet their exact molecular roles require further elucidation. Using the AddModuleScore method, we classified tumor cells on the basis of PTMs scores via single-cell RNA sequencing (scRNA-seq). The robust cell type decomposition (RCTD) approach was then applied to map these single-cell groupings onto spatial transcriptomics (ST), enabling the analysis of cell dependencies and the identification of distinct tumor cell subtypes. A prognostic model was constructed using bulk transcriptomic data to predict therapeutic outcomes in immunotherapy. Laboratory experiments were carried out to confirm the biological function of vimentin (VIM). PTMs scores were significantly elevated in tumor cells, stratifying osteoblastic cells (os) into two clusters: PTMs highos and PTMs lowos. The PTMs highos phenotype exhibited pronounced malignant characteristics and closely interacted with fibroblasts in both scRNA-seq and ST analyses. A set of ten hub genes was identified, forming a consensus machine learning-derived post-translational modification gene signature (CMDPTMS) with strong prognostic predictive capability. Although the high-CMDPTMS group (over the median risk score) was linked to poor outcomes and diminished benefit from immunotherapy, seven drugs were identified that may offer therapeutic promise for these patients. Finally, we confirmed that VIM can inhibit the growth and migration of OS cells. In summary, by integrating bulk RNA-seq, scRNA-seq, and ST, we introduced that CMDPTMS may serve as a powerful tool for enhancing OS prognosis prediction and optimizing immunotherapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/615e86f961c9b41f18269413e8e3b6315010a820" target='_blank'>
              Single-cell and spatial transcriptomics reveal post-translational modifications in osteosarcoma progression and tumor microenvironment
              </a>
            </td>
          <td>
            Yue Cui, Yue Wu, Dongyu Jiang, Tao Ding
          </td>
          <td>2025-10-09</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The alternative splicing of mRNA precursors allows one gene to yield multiple proteins with distinct functions. CDC-like kinases serve as pivotal regulators of alternative splicing. Control of protein expression also occurs at the level of DNA through histone methylation and demethylation. We investigated the activity of two CLK inhibitors, cirtuvivint and CC-671, and the LSD1 inhibitor iadademstat alone and in combination with anticancer drugs or investigational agents. Well-characterized patient-derived cancer cell lines from the PDMR (https://pdmr.cancer.gov/models/database.htm) were used along with standard human cancer cell lines. Multi-cell type-tumor spheroids were grown from a ratio of 6:2.5:1.5 malignant cells, endothelial cells, and mesenchymal stem cells. Following three days of growth, the spheroids were exposed to the single agents or combinations at concentrations up to the clinical Cmax value for each agent, if known. After seven days of exposure, cell viability was assessed using the CellTiter-Glo 3D assay and spheroid volume was assessed by bright field imaging. Several of the targeted oncology drugs exhibited additive and greater-than-additive cytotoxicity when combined with a CLK inhibitor, or the LSD1 inhibitor. These agents included the XPO1 inhibitor, eltanexor, and the KRAS G12D specific inhibitor MRTX-1133 which had activity in tumor lines harboring the KRAS G12D mutation. LSD1 inhibition was effective with ubiquitin proteasome pathway inhibitors. Conclusion: These findings may provide guidance for development of clinical trial combination regimens including cirtuvivint, CC-671 or iadademstat. Full data sets are available on PubChem.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbd17fd73c746ccc133073188d85cd1b611794f0" target='_blank'>
              RNA processing kinase inhibitors and epigenetic inhibitors in combination with oncology drugs or investigational agents in multi-cell type patient-derived tumor cell line spheroids
              </a>
            </td>
          <td>
            Beverly A. Teicher, T. Dexheimer, Thomas Silvers, N. Coussens, Eric M Jones, Steven D. Gore, Mark W. Kunkel, James H. Doroshow
          </td>
          <td>2025-09-12</td>
          <td>Cancer Chemotherapy and Pharmacology</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Genome instability is a central hallmark of cancer, driven by aberrant DNA damage responses that facilitate tumor evolution and resistance to therapy. Although canonical non-homologous end joining and homologous recombination are well-characterized pathways for repairing DNA double-strand breaks (DSBs), recent advances have revealed that cancer cells increasingly depend on alternative end-joining (alt-EJ) to survive persistent DNA damage that arises from intrinsic stresses or external therapies. Alt-EJ, characterized by its reliance on microhomologous sequences at DSB sites, promotes mutation accumulation and chromosomal rearrangements, thereby driving genomic instability and tumor progression. Despite its pivotal role in cancer biology, the molecular regulation, contextual determinants and dualistic role of alt-EJ in maintaining genome integrity compared with promoting instability remain incompletely understood. The present review integrated the latest mechanistic insights into alt-EJ, elucidated its regulatory networks and interactions with canonical DSB repair pathways and discussed its consequences for cancer genome integrity and evolution. Furthermore, it highlighted the emerging potential of alt-EJ as a therapeutic vulnerability for cancer, underscoring the urgent need to translate these discoveries into innovative treatment strategies aimed at overcoming therapy resistance and improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50b23b0e6f34e53664100d4135bd748432a49add" target='_blank'>
              Emerging insights into alternative end-joining: Mechanisms, genome instability and therapeutic opportunities in cancer (Review)
              </a>
            </td>
          <td>
            Nasir Azam, Xiangyu Liu, Xinghan Li, Lin Ma, Qi Liu
          </td>
          <td>2025-10-09</td>
          <td>International Journal of Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Small cell carcinoma of the esophagus (SCCE) is a rare and highly aggressive malignancy with limited therapeutic options and poor prognosis. The paucity of clinical specimens and lack of established experimental models have hindered a comprehensive understanding of its cellular heterogeneity and tumor microenvironment. Methods We performed single-cell RNA sequencing on SCCE samples, and integrated them with publicly available scRNA-seq datasets from esophageal squamous cell carcinoma (ESCC), esophageal adenocarcinoma (EAC), and adjacent normal tissues (NT) from ESCC and EAC cases. An integrative transcriptomic analysis was conducted to identify cell types, infer malignant states, reconstruct differentiation trajectories, evaluate immune landscapes, and investigate fibroblast subtypes and cell–cell communication networks. Results SCCE tumors were characterized by a predominance of malignant epithelial cells and exhibited a profoundly immunosuppressed phenotype, with reduced immune infiltration and widespread downregulation of immune checkpoint genes. Malignant epithelial cells showed pronounced chromosomal instability and were classified into three transcriptionally distinct subtypes with divergent differentiation trajectories. The tumor microenvironment featured a complex stromal compartment, with enrichment of extracellular matrix fibroblasts (eCAFs) characterized by elevated ELF3 regulatory activity, and collagen-driven signaling predominantly mediated by inflammatory CAFs (iCAFs). SCCE also showed the most intricate cell–cell communication network among esophageal cancer subtypes. Conclusion Our single-cell atlas offers a detailed view of the cellular heterogeneity and microenvironmental complexity of SCCE, highlighting its distinct tumor architecture, immune exclusion, and stromal reprogramming. These findings provide a valuable resource for understanding SCCE biology and form a basis for future mechanistic and exploratory biological investigations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6db8bbf656956c2fda0265e682ad202bf970eea8" target='_blank'>
              Single-cell transcriptomic analysis reveals epithelial and microenvironmental heterogeneity in small cell carcinoma of the esophagus
              </a>
            </td>
          <td>
            X. Yin, Xiaopeng Li, Lili Mi, Jiaojiao Hou, Fei Yin
          </td>
          <td>2025-10-08</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Alterations in chromatin-associated proteins contribute to the rewiring of transcriptional networks, promoting lymphomagenesis, disease progression, and resistance to therapies. Thus, epigenetic modulators offer a promising therapeutic avenue. Marginal zone lymphoma (MZL) is characterized by methylation changes, recurrent mutations in genes encoding chromatin remodelers and transcription regulators. Epigenetic reprogramming can sustain resistance to targeted agents. To identify novel therapeutic targets in MZL, we took advantage of the Structural Genomics Consortium (SGC) collection of chemical probes that inhibit or antagonize proteins involved in epigenetic signaling (www.thesgc.org). We exposed MZL cell lines, including models of secondary resistance to BTK, BCL2 and PI3K inhibitors, to 46 epigenetic compounds, spanning molecules targeting bromodomain (BRD), methyltransferases (MT), lysine demethylases (KDM) and histone acetyltransferases (HAT).



 Cells were exposed to the SGC library compounds. Cell viability was assessed after 72h. Compounds giving >40% reduction in cell proliferation versus control were considered active. Selected hits were validated in full dose-response assays.



 Two MZL cell lines (VL51, Karpas1718) and four derivatives with secondary resistance to BTK, PI3K, and BCL2 inhibitors (VL51-Ide, VL51-Copa, VL51-Ibru, Karpas1718-Ibru) were exposed to the SGC library compounds at 100 nM and 5 μM. Resistant derivatives were generally more sensitive to various epigenetic compounds than their parental cells. Only the BET inhibitor (i) JQ1 and the DOT1L-i SGC 0946 showed activity in the parental cells (both 100 nM and 5 μM, both models). Conversely, six compounds were active in resistant models at 100 nM: JQ1 and SGC 0946 (4/4), the PRMT5-i LLY 283 (2/4), and the BRPF2/TAF1-i BAY 299 (1/4). At 5 μM, 13 compounds were active: JQ1 and SGC 0946 in 4/4, 8 compounds in at least 2 models (targeting PRMT5, BRPF2/TAF1, G9a/GLP, JMJD3/UTX, BRD9, p300/CBP, EZH2) and 3 compounds in at least 1 model (PRMT7, p300/CBP, PRMT1/3/4/6/8). We then performed validation MTT experiments for 8 compounds (BAY 299, BI 9564, GSK 2801, I-BRD9, LLY 238, SGC 0946, SGC 3027, UNC 0642) using 6 concentrations of each molecule in all models. Higher activity in resistant derivatives was largely confirmed for the Karpas1718 model, and not in VL51, in which similar activity was seen in parental and resistant cells. The most active compounds were BAY 299 (BRPF2/TAF1; median IC50 6 nM), SGC 3027 (PRMT7; 229 nM) and SGC 0946 (DOT1L, 380 nM), exhibiting dose-dependent cytotoxicity across parental and resistant MZL models, followed by the others, all with median IC50s lower than or equal to 2 μM.



 Our findings reveal novel potential epigenetic vulnerabilities (PRMT7, DOT1L, and BRPF2/TAF1) in lymphoma models, also with acquired resistance to targeted therapies, besides confirming PRMT5 and BET bromodomains as valid targets. Compounds targeting these proteins should be further explored as single agents and in combinations.



 Alberto Jesus. Arribas, Luca Licenziato, Eleonora Cannas, Luca Aresu, Afua Adjeiwaa. Mensah, Francesco Bertoni. Epigenetic vulnerabilities in marginal zone lymphoma [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C027.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62cbf4328ab2c7b5c9c31fe8bd744d96696423fe" target='_blank'>
              Abstract C027: Epigenetic vulnerabilities in marginal zone lymphoma
              </a>
            </td>
          <td>
            A. Arribas, L. Licenziato, Eleonora Cannas, Luca Aresu, A. Mensah, Francesco Bertoni
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c454f3d7e351cc169def09b1bb6a18830063a225" target='_blank'>
              PAN-CANCER ANALYSES IDENTIFY ONCOGENIC DRIVERS, EXPRESSION SIGNATURES, AND THERAPEUTIC VULNERABILITIES IN RHO GTPase PATHWAY GENES
              </a>
            </td>
          <td>
            Rubén Fernández, L. F. Lorenzo-Martín, Víctor Quesada, Xosé R. Bustelo
          </td>
          <td>2025-09-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d14f9c11a97b22eaeff5c49c954b6592650bfe82" target='_blank'>
              Mapping the Telomeric 3D Interactome with Telomere-C Reveals Repetitive Element Hubs Associated with Telomere Maintenance
              </a>
            </td>
          <td>
            Yi-An Chen, Ogechukwu Mbegbu, Noelle H. Fukushima, T. R. Ranallo-Benavidez, Tianpeng Zhang, Floris P Barthel
          </td>
          <td>2025-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c81035972b65c627d1b19e7302e769d3ddd0b086" target='_blank'>
              A Single-Cell Atlas of the Breast Cancer Microenvironment Identifies Subtype-Specific Tumor-Immune Landscapes and Vulnerabilities
              </a>
            </td>
          <td>
            Liron Zisman Schachter, Asaf Pinhasi, Keren Yizhak
          </td>
          <td>2025-09-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e980a76e192e63af260762d40719bac841d722f3" target='_blank'>
              REV1 inhibition enhances trinucleotide repeat mutagenesis
              </a>
            </td>
          <td>
            Ava Siegel, Daniel Almstead, Naveen Kothandaraman, Jessica Reich, Erica N. Lamkin, Josh A Victor, Aarzoo Grover, Kanayo Ikeh, Hannah Koval, Andrew Crompton, Hongjun Jang, Hyejin Lee, Roxana Del Rio Guerra, Dmitry M Korzhnev, M. Kyle Hadden, Jiyong Hong, Pei Zhou, Nimrat Chatterjee
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background/Objectives: Short inverted repeats (SIRs) are abundant DNA motifs capable of forming secondary structures, such as hairpins and cruciforms, that can induce genome instability. However, their mutational consequences in cancer, particularly in osteosarcoma (OS), remain largely unexplored. Methods: In this study, we systematically identified over 5.2 million SIRs in the human genome and analyzed their mutational patterns across six common cancer types. Results: We found that increased small insertion and deletion (INDEL) density within SIR spacer regions represents a consistent feature across cancers, whereas elevated single nucleotide variant (SNV) and structural breakpoint density is cancer-type specific. Integrating whole-genome sequencing data from 13 OS patients, we found that both SNVs and INDELs are significantly enriched within SIR spacer regions in OS. Notably, genomic regions with higher SIR density tend to accumulate more somatic mutations, suggesting a link between SIR abundance and local genome instability. SIR-associated mutations frequently occur in oncogenes and tumor suppressor genes, including TP53, NFATC2, MECOM, LRP1B, RB1, CNTNAP2, and PTPRD, as well as in long non-coding RNAs. Mutational signature analysis further suggests that defective DNA mismatch repair and homologous recombination may act in concert with SIR-induced DNA structural instability to drive OS development. Conclusions: Our findings highlight SIRs as mutational hotspots and potential drivers of osteosarcoma pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3071c0367ebe9199d637aacf953a87bc0377296" target='_blank'>
              Short Inverted Repeats as Mutational Hotspots and Putative Drivers of Genome Instability in Osteosarcoma
              </a>
            </td>
          <td>
            Minghua Li, Chun Liang
          </td>
          <td>2025-10-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d545356a888be0cc54d6000ee5eedd795258580" target='_blank'>
              SpliceUp: Predicting SF3B1 mutations in individual cells via aberrant splice site activation from scRNA-seq data
              </a>
            </td>
          <td>
            Maksim Kholmatov, K. D. Prummel, B. Guezguez, Eva Hellström-Lindberg, Pedro L. Moura, Judith B. Zaugg
          </td>
          <td>2025-09-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Circular single-stranded DNA (cssDNA), an emerging nucleic acid vector, exhibits significant clinical potential for treating genetic disorders, enabling gene editing, and advancing oncotherapy. Its unique attributes, including high stability, structural simplicity, conformational flexibility, and low molecular mass, establish it as a promising gene therapy tool. Our study reveals that cssDNA demonstrates superior expression efficiency over conventional plasmids across diverse tumor cell lines. Notably, cssDNA expression is enhanced under certain tumor microenvironment (TME) conditions (glucose deficiency, glutamine deficiency and hypoxia) compared to normal condition. Mechanistically, these TME conditions induce significant cell cycle perturbations, particularly pronounced G1 arrest. Intriguingly, transfected cssDNA expression peaks during the late G2/M phase, immediately preceding entry into the G1 phase. We further identify S-phase kinase-associated protein 2 (SKP2) as a critical regulator of cssDNA expression under TME conditions. SKP2 inhibition directly or indirectly notably enhances the expression levels of cssDNA. These findings confirm cssDNA’s advantages as a gene expression vector and how specific TME conditions modulate its expression via cell cycle and SKP2-dependent mechanisms in vitro. This work provides a scientific foundation for cssDNA-based cancer therapy and opens new avenues for future clinical translation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a904197f8868f962bae1c8795cb37bde58b3f61" target='_blank'>
              The tumor microenvironment enhances the expression of cssDNA by modulating cell cycle signaling pathways via SKP2
              </a>
            </td>
          <td>
            Dandan Shao, Jinghao Wang, Kexuan Zou, Yatao Chen, Pengfei Zhang, Jie Song
          </td>
          <td>2025-11-05</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Glioblastoma (GBM) remains the most aggressive primary brain tumour in adults, marked by pronounced cellular heterogeneity, diffuse infiltration, and resistance to conventional treatment. In recent years, transcriptomic profiling has provided valuable insights into the molecular mechanisms that govern the progression of glioblastoma. This systematic review aims to synthesise the current literature on dysregulated gene expression in GBM, focusing on gene signatures associated with stemness, immune modulation, extracellular matrix remodelling, metabolic adaptation, and therapeutic resistance. Methods We conducted a systematic search of PubMed, The Cancer Genome Atlas (TCGA), Chinese Glioma Genome Atlas (CGGA), and the GlioVis portal for studies published between January 2005 and April 2025, limited to English-language reports. Studies were eligible if they included adult glioblastoma tissue or patient-derived datasets and reported gene-level expression or clinical associations. Reviews, commentaries, and studies on non-GBM gliomas were excluded. Screening followed the PRISMA 2020 checklist, with 410 records initially identified, 90 duplicates removed, and 125 studies retained after full-text review. Data were synthesised descriptively, and findings were validated against TCGA/CGGA expression datasets to ensure consistency across cohorts. Results We categorised recurrently dysregulated genes by their biological function, including transcription factors (SOX2, ZEB2), growth factor receptors (EGFR, PDGFRA), immune-related markers (PD-L1, TAP1, B2M), extracellular matrix regulators (MMP2, LAMC1, HAS2), and metabolic genes (SLC7A11, PRMT5, NRF2). For each group, we examine the functional consequences of transcriptional alterations and their role in driving key glioblastoma phenotypes, including angiogenesis, immunosuppression, invasiveness, and recurrence. Conclusion We further discuss the prognostic implications of these gene signatures and evaluate their potential utility in precision medicine, including current clinical trials that target molecular pathways identified through transcriptomic data. This review highlights the power of gene expression profiling to stratify glioblastoma subtypes and improve personalised therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f178b970f37a5487e44c1e368727e8a6a513f43" target='_blank'>
              RNA Expression Signatures in Glioblastoma: A Systematic Review of Tumour Biology and Therapeutic Targets
              </a>
            </td>
          <td>
            Amber Hassan, Badr Hafiz, Taghreed A. Alsinani, Rakan Bokhari, D. Mirdad, Awab T. Tayyib, Alaa Alkhotani, Ahmad Fallata, Iman Mirza, Eyad Faizo, Saleh Baeesa, Huda Alghefari, Maher Kurdi
          </td>
          <td>2025-10-22</td>
          <td>Oncology Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/106cf167da77fcf718e98566d67aa1a48f009f0c" target='_blank'>
              Telomere-to-telomere gap-free genome assembly of a male donkey and the identification of novel SVs associated with functional genes
              </a>
            </td>
          <td>
            Tao Yang, Ran Yang, Yu Liu, Mo Feng, Yuanyuan Li, Hetong Zhang, Xinyu Wang, Ruyu Yao, Jiahui Wu, Wenhui Xing, Shiyu Qian, Chunjiang Zhao
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="

 Colon cancer is a common disease, treated with few chemotherapeutic agents with similar treatment sequencing despite its heterogeneity. A significant proportion of patients are diagnosed with metastasis, and resistance to antineoplastic drugs is associated with disease progression and therapeutic failure. It is known that the tumor microenvironment plays an essential role in cancer progression, contributing to processes that may be associated with therapeutic resistance mechanisms in colon cancer. In this study, we aim to identify a gene expression signature and its relationship with immune cell infiltration in colon cancer, contributing to the identification of potential resistance biomarkers.




 An
 in silico
 study was conducted using RNA-seq data from The Cancer Genome Atlas Program (TCGA) samples, subdivided into two groups (treatment-resistant and non-resistant), taking into account the molecular subgroups (CMS1, CMS2, CMS3, and CMS4). The following algorithms were used: i.
 Limma
 was applied to identify differentially expressed genes; ii. WGCNA was applied to construct co-expression networks; iii. CIBERSORT was applied to estimate the proportion of infiltrating immune cells; and iv. TIMER was applied to explore the relationship between core genes and immune cell content.




 Twenty differentially expressed genes (DEGs) were found, with 18 related to the group considered resistant to oncologic treatment and presenting poorer overall survival. T CD4 memory resting cells and M0 and M2 macrophages were found in more significant proportions in the analyzed samples and more infiltrated in the tumor microenvironment, the higher the expression of some of these resistance DEGs. Additionally, these genes correlate with biological aspects of neuronal differentiation, axogenesis, and synaptic transmission.



 The gene expression signature suggests the presence of differentially expressed synaptic membrane genes, which may be involved in neuronal pathways that influence the tumor microenvironment, potentially serving as future biomarkers. Furthermore, the presence of M0 and M2 macrophages and T CD4 memory resting cells suggests a potential interaction that may play a role in therapeutic resistance.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd3d04147fc5526a1e2f859a73c8cdc7c99f7a9f" target='_blank'>
              Gene expression profile in colon cancer therapeutic resistance and its relationship with the tumor microenvironment
              </a>
            </td>
          <td>
            P. Doria, G. Rocha, Vanessa Dybal Bertoni, Roberto de Souza Batista dos Santos, Mariana Araújo-Pereira, Clarissa A Gurgel
          </td>
          <td>2025-10-29</td>
          <td>Frontiers in Bioinformatics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) remains a formidable therapeutic challenge, with high relapse rates rooted not only in genetic heterogeneity, but also in dynamic and reversible epigenetic dysregulation that drive resistance to conventional and targeted small-molecule therapies. Although small-molecule inhibitors targeting epigenetic regulators have demonstrated preclinical efficacy and initial clinical activity, therapeutic gains are often limited by transient responses, intra- and inter-patient heterogeneity, and an incomplete understanding of predictive biomarkers. Besides, clinical outcomes have often been less robust than preclinical data suggested, partly due to the complexity of AMLs genetic and epigenetic heterogeneity, the limitations of model systems, and intrinsic resistance mechanisms. Combination approaches, such as pairing epigenetic agents with chemotherapy, immunotherapy, or other targeted drugs are under investigation. However, clinical efficacy remains inconsistent and requires a mechanistic rationale to avoid antagonism. To move forward, it is essential to delineate specific epigenetic dependencies in resistance-prone AML subtypes, design rational, biomarker-guided combination regimens, and enhance specificity and pharmacodynamic profiles of epigenetic inhibitors. Such an integrated strategy holds promise for elevating the effectiveness of epigenetic therapies in AML. Continued research is essential to refine epigenetic-based precision medicine and ultimately improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8e1118798f14d99fa1bf83fc376d07fc1df6c5c" target='_blank'>
              Epigenetic mechanisms of drug resistance in acute myeloid leukemia: advances in small-molecule targeted therapy
              </a>
            </td>
          <td>
            Jialing Tang, Xibo Ye, Zhangyang Xu, Tianjiao Min, Zherun Tian, Li Ma, Ying He, Beier Jiang
          </td>
          <td>2025-10-20</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Cancer can recur when a subset of tumor cells, termed persister cells, survive therapy and re-enter the cell cycle. The cellular lineages that give rise to persister cells and the mechanisms that confer the persister state remain poorly understood. Through single-cell multiomic profiling (snRNA-seq and snATAC-seq) on a cohort of (1) non-malignant fallopian tube, (2) treatment-naïve, and (3) neoadjuvant chemotherapy (NACT)-treated high-grade serous ovarian cancer (HGSOC) patient samples, we identified an epigenetic signature that defines the chemotherapy-tolerant persister state. The changes in chromatin accessibility characterizing the signature were identified in residual NACT tumors but are also present in treatment-naive samples from patients who later developed resistance. Furthermore, this epigenetic signature independently predicted chemotherapy response in patient-derived xenograft models of HGSOC and in a separate patient cohort. Cells enriched in the persister state arose from multiple lineages and displayed activation of oncogenic pathways, including altered stress responses, epithelial to mesenchymal transition, and changes to the cell cycle promoting quiescence. Finally, we identified a subset of genes that are epigenetically primed for expression before treatment and are upregulated after treatment. These findings suggest that an intrinsic epigenetic program primes tumor cells towards chemotherapy tolerance and reveal new vulnerabilities that could be exploited to delay or prevent cancer recurrence.



 Mihai G. Dumbrava, Wazim M. Ismail, Leticia Sandoval, Amelia Mazzone, Syed Mohammed Musheer Aalam, Megan L. Ritting, Xiaonan Hou, Yiwen Xie, Shariska Harrington, Scott H. Kaufmann, Nagarajan Kannan, S. John Weroha, Alexandre Gaspar-Maia. Single cell multiomic analysis of high-grade serous ovarian carcinoma reveals an intrinsic epigenetic program that primes chemotherapy tolerance in persister cells [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr PR008.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a450af46a8e3f0b0c1a2127ce5282e5d4428e5a8" target='_blank'>
              Abstract PR008: Single cell multiomic analysis of high-grade serous ovarian carcinoma reveals an intrinsic epigenetic program that primes chemotherapy tolerance in persister cells
              </a>
            </td>
          <td>
            Mihai G. Dumbrava, W. Ismail, Leticia Sandoval, Amelia Mazzone, Syed Mohammed Musheer Aalam, Megan L. Ritting, Xiaonan Hou, Yiwen Xie, Shariska P Harrington, Scott H. Kaufmann, Nagarajan Kannan, S. Weroha, Alexandre Gaspar-Maia
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Acquisition of resistance to anti-cancer therapies is a multistep process initiated by the survival of drug-tolerant persister cells. Accessibility of drug-tolerant persister cells in patients is limited, which has hindered understanding the mechanisms driving their emergence. Here, using multiple patient-derived models to isolate persister cells, we showed that these cells are transcriptionally plastic in vivo and return to a common treatment naïve-like state upon relapse, regardless of treatment. Hallmarks of the persister state in TNBC across treatment modalities included high expression of basal keratins together with activation of stress response and inflammation pathways. These hallmarks were also activated in HER2+ breast and lung cancer cells in response to targeted therapies. Analysis of gene regulatory networks identified AP-1, NF-κB and IRF/STAT as the key drivers of this hallmark persister state. Functionally, FOSL1, an AP-1 member, drove cells to the persister state by binding enhancers and reprogramming the transcriptome of cancer cells. On the contrary, cancer cells without FOSL1 had a decreased ability to reach the persister state. By defining hallmarks of TNBC persistence on multiple therapies, this study provides a resource to design effective combination therapeutic strategies that limit resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88dba0b22030a811fd28fd3f8f3324b7f9286705" target='_blank'>
              Characterization of Drug-Tolerant Persister Cells in Triple-Negative Breast Cancer Identifies a Shared Persistence Program across Treatments and Patients.
              </a>
            </td>
          <td>
            Léa Baudre, Gregoire Jouault, Pacôme Prompsy, Melissa Saichi, Sarah Gastineau, Christophe Huret, Laura Sourd, A. Dahmani, Elodie Montaudon, F. Dingli, D. Loew, Elisabetta Marangoni, J. Marsolier, Céline Vallot
          </td>
          <td>2025-11-06</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Genetic instability is a hallmark of cancer cells. Homologous recombination (HR) plays a pivotal role in maintaining genome stability through its DNA repair and replication fork escort functions. Therefore, HR is classified as a tumour suppressor pathway. Consistently, many HR genes are mutated in cancer, especially in hereditary breast and ovarian cancer. However, although RAD51 controls the central steps of HR, no RAD51 mutations are associated with cancer predisposition, constituting the “RAD51 paradox”. One of the potential explanations for the “RAD51 paradox” is that mutations affecting mediator/accessory genes (such as BRCA1 or BRCA2) in cancer result in the absence of RAD51 on damaged DNA, leaving access to alternative exclusively mutagenic repair processes, such as single-strand annealing (SSA) or alternative end-joining (A-EJ), which can rescue some cell viability but also increase genetic instability. This raises the question of whether cancer predisposition actually results from HR deficiency itself or from alternative, nonconservative repair pathways. One study assessing this question in a mouse model revealed that decreasing RAD51 HR activity without stimulating SSA or A-EJ in vivo not only does not favour tumorigenesis but rather protects against it. These data suggest that RAD51-controlled HR is not a tumour suppressor but rather favours tumour progression. Cancer cells are highly proliferative, actively replicating their genomes, and are therefore subjected to high replication stress; pathways enabling them to cope with this massive replication stress, such as HR, should help them survive and proliferate, contrary to the belief dogma that HR acts as a tumour suppressor pathway. We propose that HR/RAD51, through its essential role in overcoming replication stress, should facilitate cancer progression as soon as early pretumorigenic hyperplasia states that trigger an active replication program, challenging commonly accepted views.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1cae074f29f78d9d694a20a3e12fe166bc58ebc" target='_blank'>
              RAD51-mediated homologous recombination is a pro-tumour driver pathway
              </a>
            </td>
          <td>
            Bernard S. Lopez
          </td>
          <td>2025-09-24</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="BACKGROUND
Structural Variants (SVs) are defined as genomic variants affecting more than 500 base pairs. They include deletions, insertions, inversions, translocations, tandem repeats, and copy number variations (CNVs). These SVs contribute significantly to genetic complexity and are involved in human evolution, genetic disorders, and cancer. Over 50% of the SVs cannot be detected due to limitations in methods and technologies. The short-read sequencing technologies are limited in detecting single-nucleotide variants and fail to resolve complex, repetitive and intronic regions.


SUMMARY
The advent of long-read sequencing (LRS) technologies, such as Oxford Nanopore and PacBio platforms, has revolutionised SV detection. These platforms allow accurate characterisation of diverse variant types, from simple deletions to complex chromothripsis events, and support de novo assembly, haplotype phasing, and resolution of repetitive or structurally complex genomic regions. One major outcome is the completion of the telomere-to-telomere (T2T) human reference genome. This review summarises recent advances in LRS for SV detection, including sequencing platforms, bioinformatic tools, analysis, and validation strategies. The clinical applications, especially in rare disease diagnostics, are illustrated with two novel cases which were successfully resolved using both LRS approaches.


KEY MESSAGE
Long-read sequencing can solve the limitations of short-read sequencing in SV detection, providing better insights into genome disorders. It enables the detection of repeat and difficult-to-resolve regions of the genome and facilitates clinical diagnoses to base-level breakpoint detection. Despite challenges such as high cost, data interpretation, and clinical linking, continued advancements are elevating LRS as an invaluable tool in precision genomic medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c39bcf4958d938e1f4df92d98461ff79cbc6183" target='_blank'>
              Deciphering the Structural Variants by Long-Read Genome Sequencing: Technology, Applications, and Case Illustrations.
              </a>
            </td>
          <td>
            U. Dutta, Ashwin Dalal
          </td>
          <td>2025-10-30</td>
          <td>Cytogenetic and genome research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b69f1ebb5999d2aa0b3b1b9147382f2b93bbbf41" target='_blank'>
              Influence of Cultivation Conditions on Expression of Transcription Factor Genes in Glioblastoma Cells
              </a>
            </td>
          <td>
            A. I. Rezekina, D. V. Mazur, M. I. Shakhparonov, N. V. Antipova
          </td>
          <td>2025-09-28</td>
          <td>Russian Journal of Bioorganic Chemistry</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 The formation of cancer and responsiveness to therapy is governed by a constantly evolving balance between somatic variation and dynamic epigenetic regulation. This evolution has been recognized as a significant obstacle challenging the development of durable cancer therapies. Thus, targeting epigenetic reprogramming in human cancers is a key focus for the design of new drugs to potentially deliver robust and durable anti-tumor activity. Lysine acetyltransferases (KATs) are a promising class of epigenetic targets that operate within interchangeable multi-subunit complexes to regulate cell cycle, pluripotency, and lineage determination. Within this class, KAT6A and KAT7 have been recognized as mechanistically intertwined modulators of stem cell identity and lineage commitment through partially overlapping control of chromatin organization specified by histone H3 acetylation of lysine 23 (H3K23) and lysine 14 (H3K14). Multiple lines of orthogonal evidence point to a “paralog” genetic relationship between these enzymes to support cancer cell autonomous survival in lineage-enriched CCLE subsets. Moreover, recent preclinical findings in pediatric and adult leukemia indicate the need to inhibit both KAT6 and KAT7 to deliver durable anti-tumor responses. These observations prompted evaluation of concomitant inhibition of KAT6 and KAT7 as a therapeutic approach in biomarker-defined solid cancers. Here, we report an equipotent KAT6/KAT7 dual inhibitor with high selectivity versus the structurally similar KAT family members KAT5/KAT8. We observed synergistic impact on cell viability and pharmacodynamic (PD) modulation of H3K23/H3K14 acetylation in biomarker positive breast, lung, and ovarian cancers with dual KAT6/7 inhibition versus selective inhibition of KAT6 or KAT7. Quantitative transcriptomic and epigenomic profiling of cancer cell responses to dual KAT6/7i revealed global inhibition of H3K23 and H3K14 acetylation as well as local suppression of H3K27ac-dependent enhancer elements at gene promoters controlling lineage specification, stemness, and WNT signaling. In vitro single cell RNA sequencing and protein expression analysis of ER+ PDX derived breast cancer cells indicated that dual KAT6/7 inhibition can overcome KAT6i associated adaptive drug resistance, downregulating drug tolerant persister cell states and metabolic adaptation. In vivo efficacy studies in biomarker positive breast and lung CDX models confirmed enhanced tumor growth inhibition and PD modulation for dual KAT6/7 versus selective KAT6 inhibition. Notably, dual KAT6/7 inhibition promoted durable and stable anti-tumor responses in multiple standard-of-care resistant and ESR1-mutant breast cancer PDXs that could not be achieved with KAT6 inhibition alone. In vivo bulk transcriptomics analysis of dual KAT6/7 treated PDX tumor samples highlighted disruption of lineage identity and metabolic rewiring. These observations indicate equipotent inhibition of KAT7 and KAT6A/B has the potential to deliver deep and durable anti-tumor activity in biomarker-selected patient populations.



 Manav Gupta, Scott R. Tyler, Nour Ghaddar, Pranav Gupta, Steven L. Spivak, Katelyn N. Lukas, Carl Schultz, Anjali Bisaria, Zabrisky Roland, Kelly S. Trego, Mason Appel, Parker Y. Jameson, Jacob Burch-Konda, Brittany Cruzan, Angelica Gonzalez-Sanchez, Monah Khalaj, Jay Prakash. Jain, Michael E . Dalziel, Julie Deichert, Sunjay Sethi, Ivan G. Shabalin, Rebeca Choy, Diana M. Munoz, Christian R. Frey, Peter Teriete, Josh Taygerly, Yuchen Bai, Richard Zang, Xin Linghu, Claire L. Neilan, Paul A. Barsanti, Michael A . White, Brian T. Jones. Dual KAT6/7 inhibition delivers robust monotherapy anti-tumor activity in biomarker selected indications and disrupts emergence of drug-tolerant persister cell populations [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C022.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7cc963d6c0afb39c048f1457ab9b594f9c49864" target='_blank'>
              Abstract C022: Dual KAT6/7 inhibition delivers robust monotherapy anti-tumor activity in biomarker selected indications and disrupts emergence of drug-tolerant persister cell populations
              </a>
            </td>
          <td>
            Manav Gupta, Scott R. Tyler, N. Ghaddar, Pranav Gupta, Steven L. Spivak, Katelyn N. Lukas, Carl Schultz, Anjali Bisaria, Zabrisky Roland, K. Trego, Mason J. Appel, Parker Y. Jameson, Jacob Burch-Konda, Brittany Cruzan, Angelica Gonzalez-Sanchez, Monah Khalaj, J. Jain, Michael E. Dalziel, Julie Deichert, Sunjay Sethi, I. Shabalin, Rebeca M. Choy, Diana M. Muñoz, Christian R. Frey, P. Teriete, J. Taygerly, Yuchen Bai, Richard Zang, Linghu Xin, Claire L. Neilan, P. Barsanti, Michael A. White, 
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The search for novel tumor-specific markers and therapeutic targets is driving the development of more effective and personalized treatment strategies for cancer patients. This article focuses on investigating a promising new source of biomarkers—readthrough transcripts, or downstream-of-gene (DoG) transcripts. These transcripts are extended products of gene transcription that continue into intergenic regions and can overlap neighboring genes, sometimes giving rise to cis-splicing of adjacent gene (cis-SAGe) transcripts. Recent studies suggest that besides frequently being a “transcriptional noise”, DoG transcripts can perform regulatory functions, serve as a source of novel protein products, and act as prognostic markers of patient survival across various cancers. This article aims to investigate the regulatory mechanisms and functional significance of readthrough transcripts in tumors, to identify currently known tumor-specific variants with potential utility as cancer biomarkers or therapeutic targets, and to evaluate the most suitable approaches for their detection. The knowledge gained through this research may provide a foundation for the development of diagnostic test systems and the design of new anticancer drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/005f6133a4fcac56d4aa30b3f8401ba10e194f84" target='_blank'>
              Regulatory Mechanisms and Functional Roles of Readthrough Transcripts in Tumorigenesis
              </a>
            </td>
          <td>
            Aleksander Modestov, G. Zakharova, E. Poddubskaya, Anton Buzdin
          </td>
          <td>2025-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Osteosarcoma is a genomically complex tumor characterized by widespread structural rearrangements. This complexity has limited development of therapeutic strategies informed by molecular mechanisms of oncogenesis. We hypothesized that epigenetic mechanisms could drive distinct subtypes of osteosarcoma. Through analysis of chromatin accessibility, we identified an "early osteoblast-derived" (EOD) cell state characterized by upregulation of transcription factors associated with early bone development, and a "late osteoblast-derived" state (LOD), characterized by upregulation of genes involved in late bone development. We then defined core regulatory circuitries governing the underlying gene expression programs in these two cell states. Multiomic single-cell analysis indicates that these cell states co-exist in a single tumor. Finally, using a panel of patient-derived xenograft models, we identified differential drug responses dependent on these cellular states. These findings create opportunities for developing new combination therapy strategies for osteosarcoma treatment and underscore the value of defining epigenetic subtypes in highly genomically complex cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/061de65b88f0b6c7fc51a022f24db3f11873c460" target='_blank'>
              Epigenetic and transcriptional programs define osteosarcoma subtypes and establish targetable vulnerabilities.
              </a>
            </td>
          <td>
            E. López-Fuentes, Andrew S Clugston, Alex G Lee, L. Sayles, Natalie Sorensen, María V Pons Ventura, Stanley G Leung, Truc Dinh, Marcus R. Breese, E. Sweet-Cordero
          </td>
          <td>2025-10-03</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Genetic variation at thousands of short tandem repeats (STRs), which consist of consecutive repeated sequences of 1-6bp, has been statistically associated with gene expression and other molecular phenotypes in humans. However, the causality and regulatory mechanisms for most of these STRs remains unknown. Massively parallel reporter assays (MPRA) enable testing the regulatory activity of a large number of synthesized variants, but have not been applied to STRs due to experimental and computational challenges. Here, we optimized an MPRA framework based on random barcoding to study the impact of variation in repeat copy number on expression. We first performed an MPRA on sequences derived from 30,516 promoter-proximal STR loci along with up to 152bp of genomic context, testing 3-4 variants with differing repeat copy numbers for each locus in HEK293T cells. We identified 1,366 loci with significant associations between repeat copy number and expression, which were enriched for positive effect sizes (P=2.08e-110). We then designed a second MPRA in which we performed deeper perturbations, including systematic manipulation of the repeat unit sequence, orientation, and copy number, with 200-300 perturbations for each of the 300 loci with the strongest signals. Our results revealed that the repeat unit sequence is the primary driver of differences in the relationship between copy number and expression across loci, whereas orientation and flanking sequence have weaker effects, primarily for AT-rich repeat units. The high resolution of these perturbations enabled us to detect non-linear effects, most notably for AAAC/GTTT repeats, which emerge only beyond a certain copy number threshold. Finally, we observed that a subset of STRs in our library show expression levels that are tightly linked with predicted DNA secondary structure formation. We repeated our perturbation MPRA in HeLa S3 cells under wildtype and RNase H1 knockdown conditions, which, via reduction in RNase H1 activity, are expected to hinder resolution of R-loops. This demonstrated that associations between copy number and expression at G-quadruplex-forming CCCCG/CGGGG repeats are particularly sensitive to loss of RNase H1, providing support for an R-loop mediated mechanism for these repeats. Altogether, we establish STRs as a critical component of the non-coding regulatory grammar and provide a framework for understanding how this dynamic form of genetic variation shapes gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ad23e384b94840a5506d6ec02839f2c2721fcd3" target='_blank'>
              Systematic evaluation of the impact of promoter proximal short tandem repeats on expression
              </a>
            </td>
          <td>
            Xuan Zhang, Lingzhi Zhang, Ellice Wang, Susan Benton, Eduardo Modolo, Mikhail Maksimov, Sharona Shleizer-Burko, Qingqing Gong, Catherine Wang, M. Lamkin, E. Mendenhall, Melissa Gymrek, A. Goren
          </td>
          <td>2025-09-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/679fc499895ef9321451ab5cf3bc0bc1060162ad" target='_blank'>
              DNA shape and epigenomics distinguish the mechanistic origin of human genomic structural variations
              </a>
            </td>
          <td>
            Nadejda B. Boev, Mark B. Gerstein, Sushant Kumar
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88aadf318bf49329614cbd65f8b3f0ff419cae30" target='_blank'>
              Pan-cancer analysis reveals context-dependent roles of LINE-1 ORF1p in immune regulation and copy number alterations
              </a>
            </td>
          <td>
            E. Kaparos, Wenjing Zhang, Ryusei Miyanaga, Wilson H. McKerrow, J. Boeke, Teresa Davoli, P. Mita, Kelly Ruggles, David Fenyö
          </td>
          <td>2025-10-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>113</td>
        </tr>

        <tr id="Radiotherapy resistance remains a major clinical challenge, largely driven by tumors’ ability to dynamically adapt through complex molecular networks. Critically, the ubiquitin system has emerged as a critical regulator of this resistance. This review examines how the ubiquitin system orchestrates radiotherapy resistance through spatiotemporal control of DNA repair fidelity, metabolic reprogramming, and immune evasion. We explore how the ubiquitin code, defined by its chain topology diversity (such as K48-linked proteolysis versus K63-mediated signaling) and crosstalk with phosphorylation, SUMOylation, and acetylation, generates diverse resistance mechanisms. These mechanisms, however, also present vulnerabilities exploitable for radio-sensitization. Notably, monoubiquitylation of both histone and non-histone protein collaboratively modulates chromatin dynamics and DNA damage responses to maintain genome integrity during radiation. Furthermore, ubiquitination critically regulates caner metabolism, reprogramming processes such as ferroptosis susceptibility, hypoxia adaptation, and nutrient flux, thereby creating targetable vulnerabilities for radio-sensitization. While targeting key E3 ligases and deubiquitinases (DUBs) shows preclinical promise, clinical translation faces obstacles including functional redundancy, unintended on-target toxicity, and adaptive tumor responses. Distinct from other post-translational modifications (PTMs), the ubiquitin system offers unique clinical advantages: its dynamic reversibility, chain topology diversity, and recent breakthroughs in targeted degradation (e.g., PROTACs) enable precise disruption of radioresistance networks. By integrating these mechanistic insights with biomarker-guided therapeutic strategies, ubiquitin-targeting agents are emerging as fundamental components of next-generation radiotherapy protocols.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f21d3a6716308e53e33306aeb9e0bc2d5a80f91" target='_blank'>
              Decoding the ubiquitin network: molecular mechanisms and therapeutic vulnerabilities for precision radio-sensitization in cancer
              </a>
            </td>
          <td>
            Fu-Ying Zhao, Li-Ye Shi, Lu-Jiao Yu, Si-Qi Wang, Chu-Yao Sun, Na Zhang, Hua-Qin Wang
          </td>
          <td>2025-10-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Platinum-based chemotherapeutic drugs induce intra- and interstrand purine crosslinks that causes DNA damage and tumor cell death. While these drugs are front-line treatments for multiple human cancers – including high-grade serous ovarian cancer (HGSOC) – platinum resistance and tumor recurrence remains a major clinical challenge. Here, we report findings from both a genome-wide CRISPRi screen and an RNA-binding protein (RBP)-focused CRISPRi screen aimed at identifying modulators of sensitivity and resistance of HGSOC cells to two widely used platinum agents, cisplatin and oxaliplatin, as well as to an microtubule-targeting taxane. Using an in vitro treatment regimen designed to mimics clinical treatment protocols, we found that loss of genes in the Fanconi anemia (FA) and homologous recombination (HR) DNA repair pathways markedly increased the killing of p53-deficient HGSOC cells following treatment with cisplatin or oxaliplatin. In contrast, knockdown of genes in the nucleotide excision repair (NER) pathway—despite its known role in repairing platinum-induced intrastrand crosslinks, particularly in G1—had a significantly lesser effect. These results suggest that platinum-induced cytotoxicity in HGSOC is primarily driven by replication stress, which is exacerbated by disruption of error-free DNA repair pathways active during the S- and G2 phases of the cell cycle. Our screens also identified several known and novel RBPs as potent sensitizers to both platinum agents and taxanes. Depletion of these RBPs resulted in substantial activation of interferon signaling, a hallmark of innate immune activation and cellular stress responses. Among them, the RNA-editing enzyme ADAR1 emerged as a key modulator of drug resistance. ADAR1 catalyzes adenosine-to-inosine RNA editing, thereby altering mRNA coding potential, secondary structure, and subcellular localization. Additionally, ADAR1 has been implicated in regulating the integrated stress response (ISR), a signaling network that enables cellular adaptation to environmental stress. As part of this stress adaptation, we observed that platinum and taxol treatment caused a translational shift from canonical cap-dependent translation mediated by eIF4G1 to a cap-independent mechanism reliant on eIF4G2, supporting selective translation of stress-responsive mRNAs. Together, our findings suggest that ADAR1-dependent suppression of interferon signaling and activation of the ISR contributes to resistance to both platinum-based chemotherapy and microtubule-targeting agents, which together comprise the current standard of care for HGSOC.



 Yi Wen Kong, Erika D. Handly, Robin A. Lu, Molly A. Bird, Jack Schaeffer, Samkyu P. Yaffe, Michael B. Yaffe. CRISPRi screening reveals RNA editing and the integrated stress response as major determinants of platinum and taxol sensitivity in high-grade serous ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr PR009.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/912428b4f14100051b63d5819642408cb6578a72" target='_blank'>
              Abstract PR009: CRISPRi screening reveals RNA editing and the integrated stress response as major determinants of platinum and taxol sensitivity in high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            Yi Wen Kong, E. Handly, Robin A. Lu, Molly A. Bird, Jack W. Schaeffer, Samkyu P. Yaffe, Michael B Yaffe
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Thyroid carcinomas are phenotypically heterogeneous malignancies. Advances in molecular and cellular technologies have revealed genetic, epigenetic, and nongenetic factors underlying this heterogeneity. Our study aimed to assess the impact of single and combined treatments with anticancer agents (Carboplatin, Doxorubicin, Paclitaxel, Avastin), natural compounds (Quercetin), and epigenetic modulators (suberoylanilide hydroxamic acid and 5-Azacytidine) on the expression of long noncoding RNAs, methylation regulators, and functional features in the human thyroid cancer cell line K1. Methods: Treated and untreated K1 cells were used throughout experiments to evaluate the drug-induced cytotoxicity, apoptosis, cell cycle distribution, cytokine release, gene expression, and global DNA methylation levels. Results: Some single- and combined-drug treatments modulated both cell cycle progression and apoptotic events, demonstrating anti-tumor activity of the tested compounds. Gene expression analysis showed treatment-specific regulation of target genes and lncRNAs, including both upregulation and downregulation across different drug combinations. All treatments resulted in increased global DNA methylation levels compared to the untreated controls. Several combinations significantly upregulated DNMT1 and DNMT3B, while concomitantly decreased EZH2 levels. Conclusions: These coordinated epigenetic changes highlight the therapeutic potential of combining epigenetic modulators with chemotherapeutic agents, suggesting a strategy to prevent or reverse treatment resistance and improve outcomes in thyroid cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85aa1eba8f910f8930ce4647663ca6adba36af18" target='_blank'>
              Epigenetic Remodeling in Thyroid Cancer: New Dimensions of Targeted Therapy Through lncRNA Modulation
              </a>
            </td>
          <td>
            A. Albulescu, A. Fudulu, M. Mihăilă, I. Iancu, Adriana Pleșa, M. Bostan, A. Botezatu, L. Brasoveanu, C. Hotnog
          </td>
          <td>2025-10-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background/Objectives: Targeting Hsp90β with isoform-selective inhibitors offers a promising therapeutic strategy with reduced toxicity compared to pan-Hsp90 inhibition. However, mechanisms of resistance to Hsp90β-selective inhibition remain poorly defined. This study aimed to identify molecular determinants of Hsp90β dependency and pharmacologic resistance across cancer types. Methods: We integrated gene dependency, transcriptomic, proteomic, metabolomic, and drug sensitivity data from the Cancer Cell Line Encyclopedia with in vitro validation using the Hsp90β-selective inhibitor, NDNB-25. Comparative and correlation analyses were performed to identify resistance-associated pathways, followed by network and combination drug testing to validate functional interactions. Results: Resistant cell lines exhibited extensive rewiring of Rho GTPase signaling, cytoskeletal remodeling, and metabolic adaptation, including mitochondrial dysfunction and redox imbalance. Integrated analyses linked these phenotypes to aryl hydrocarbon receptor (AHR) activation and compensatory Hsp90α expression. Experimental validation confirmed increased kynurenine levels, a known endogenous AHR ligand, in NDNB-25–acquired resistant cells. Gene–drug network integration revealed collateral sensitivity to carboplatin, which synergized with Hsp90β inhibition in resistant models. Conclusions: This study defines the molecular features and adaptive programs underlying resistance to Hsp90β-selective inhibition and identifies therapeutic vulnerabilities that can be exploited to overcome it. The findings establish a systems-level framework for predicting Hsp90β inhibitor response and support rational combination strategies, including carboplatin co-treatment, for future preclinical development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c66c8f9799671f41a6a62a8af2b30f03bc5ed697" target='_blank'>
              Integrative Multi-Omics Analyses Reveal Mechanisms of Resistance to Hsp90β-Selective Inhibition
              </a>
            </td>
          <td>
            Ian Mersich, Eahsanul Anik, Aktar Ali, Brian S. J. Blagg
          </td>
          <td>2025-10-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Regorafenib, a multikinase inhibitor, is widely used to treat hepatocellular carcinoma. However, chemoresistance poses a significant challenge to its long-term efficacy. This study conducted a genome-wide CRISPR/Cas9 knockout screen in liver cancer cell lines to identify key regulators of regorafenib resistance and elucidate the underlying molecular mechanisms. The screen identified ARHGAP35 as a critical negative regulator of regorafenib resistance. ARHGAP35 depletion conferred resistance in HepG2 and Huh7 cells, while regorafenib-resistant variants (HepG2-R and Huh7-R) exhibited decreased ARHGAP35 expression. Reintroducing ARHGAP35 restored drug sensitivity. Further analysis revealed that reduced ARHGAP35 expression facilitated epithelial-mesenchymal transition (EMT) by activating the RhoA signaling pathway. Notably, RhoA inhibition reversed EMT and restored regorafenib sensitivity. These findings highlight ARHGAP35 as a key modulator of regorafenib resistance through RhoA suppression, offering potential therapeutic targets to combat chemoresistance in liver cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1b3a4892777e73e2cf92f82b25659fdd83d51c4" target='_blank'>
              A genome-wide screen identified ARHGAP35 as a regulator of regorafenib resistance in liver cancer.
              </a>
            </td>
          <td>
            Kun Chen, Miaomiao Zhang, Yuexin Liu, Zhengnan Dong, Yaoji Liang, Jinzhang Zeng, Jie Liu
          </td>
          <td>2025-10-16</td>
          <td>Anti-cancer drugs</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf7e3130dd32248dd76079d5a6fd192ad3103a10" target='_blank'>
              Germline-limited transposon remnants foster somatic genome diversification
              </a>
            </td>
          <td>
            Bozhidar-Adrian Stefanov, M. Nowacki
          </td>
          <td>2025-11-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Cancer is a multifaceted disease characterized by uncontrolled cell division resulting from substantial disruptions of normal biological processes. Central to its development is cellular transformation, which involves a dynamic sequence of events including chromosomal translocations, genetic mutations, abnormal DNA methylation, post-translational protein modifications, and other genetic and epigenetic alterations. These changes compromise physiological regulatory mechanisms and contribute to accelerated tumor growth. A critical factor in this process is the dysregulation of transcription factors (TFs) which regulate gene expression and DNA transcription. Dysregulation of TFs initiates a cascade of biochemical events, such as abnormal DNA replication, that further enhance cell proliferation and increase genomic instability. This microenvironment not only sustains tumor growth but also promotes the accumulation of somatic mutations, thereby fueling tumor evolution and heterogeneity. In this study, we employed an in silico approach to identify TFs regulating 622 key genes whose mutations are implicated in carcinogenesis. Transcriptional regulatory networks were analyzed through bioinformatics methods to elucidate molecular pathways involved in cancer development. A thorough understanding of these processes may help to clarify the function of dysregulated TFs and facilitate the development of novel therapeutic approaches designed to make cancer treatments personalized and efficacious.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/020eeb95aab1edda2cdedffd56c0eb8b8019a670" target='_blank'>
              From Transcription Factors Dysregulation to Malignancy: In Silico Reconstruction of Cancer’s Foundational Drivers—The Eternity Triangle
              </a>
            </td>
          <td>
            A. Cammarota, A. Carrizzo, M. De Marco, N. Bukvic, Francesco Jacopo Romano, Alessandra Rosati, M. Chetta
          </td>
          <td>2025-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Abstract Genomic instability and the accumulation of DNA damage are hallmarks of cancer, often resulting from defects in DNA repair pathways. While normal cells rely on highly coordinated DNA damage response (DDR) mechanisms to maintain genomic integrity, cancer cells exploit aberrant DDR regulation to sustain uncontrolled proliferation and survival. Despite significant advancements in chemotherapy, targeted therapy, and immunotherapy, the emergence of resistance remains a major challenge in cancer treatment. Small molecule inhibitors targeting key DDR proteins have emerged as promising therapeutic agents, not only as direct anticancer drugs but also as indispensable tools for dissecting the molecular intricacies of DNA repair. Recent therapeutic approaches leverage synthetic lethality and DDR pathway vulnerabilities to selectively eradicate tumor cells while minimizing damage to normal tissues. These inhibitors provide insights into mechanisms of tumor resistance, facilitating the rational design of combination therapies to enhance treatment efficacy. This review examines the latest advancements in DNA repair-targeted therapeutics, with a focus on small molecule inhibitors currently under clinical investigation. Additionally, we discuss emerging strategies for optimizing DDR-targeted interventions, including biomarker-driven patient selection and rational drug combinations. Understanding these molecular interactions will contribute to the development of novel, more effective treatment paradigms for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62086e2b8dc746fe8a2545d0353e820f59f91c95" target='_blank'>
              Targeting DNA repair mechanisms in cancer therapy: the role of small molecule DNA repair inhibitors
              </a>
            </td>
          <td>
            Seula Jeong, Yuheon Chung, Soomin Heo, Kyungjae Myung
          </td>
          <td>2025-11-03</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Endometrial cancer represents one of the most common gynecologic malignancies worldwide, with molecular profiling emerging as a transformative approach to its classification and treatment. The traditional histopathological model, based on tumor grade and morphology has evolved with the advent of genomic technologies, leading to the development of The Cancer Genome Atlas (TCGA) molecular subtypes: POLE ultra mutated, microsatellite instability-high (MSI-H) or mismatch repair-deficient (DMMR), copy-number low, and copy-number high. POLE-mutated tumors, characterized by ultrahigh mutational burden and microsatellite stability, generally have excellent prognosis, whereas MSI-H/ DMMR tumors respond favorably to immune checkpoint inhibitors. This review article was intended to underlie the importance of personal treatment of every patient because they exhibit unique molecular characteristics, leading to variations in prognosis and therapeutic response. Therefore, tailoring management to these molecular profiles is essential to achieve optimal outcomes. Integrating immunohistochemistry, PCR, and next-generation sequencing facilitates accurate molecular classification and individualized therapy. The convergence of molecular pathology and genomics represents a paradigm shift in endometrial cancer care, positioning POLE and MMR testing as cornerstone biomarkers for truly personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/916625046da400792441b823ecd6e0f73c749886" target='_blank'>
              Integrating POLE Mutations and Mismatch Repair Status in Personalized Medicine for Endometrial Cancer
              </a>
            </td>
          <td>
            Erin Nabila Putri, Willy Sandhika
          </td>
          <td>2025-10-31</td>
          <td>Magna Scientia Advanced Research and Reviews</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 DHX9, a multi-functional RNA helicase, plays important roles in maintaining genome stability by unwinding various nucleic acid secondary structures that may lead to replication stress and DNA damage if left unresolved. DHX9 is overexpressed in multiple cancers, with selective vulnerability to DHX9 inhibition noted in tumors characterized by genomic instability and high replicative stress. ATX-559 is an orally bioavailable, small-molecule inhibitor of DHX9 helicase activity currently in Phase 1 clinical development. ATX-559 induces robust anti-tumor activity in a variety of preclinical models with genomic instability, including those with microsatellite instability-high (MSI-H) and/or deficient mismatch repair (dMMR), BRCA alterations, and homologous recombination repair deficiency (HRD).



 ATX-559 anti-proliferative activity was tested in a multi-indication cell panel screen. Triple negative breast cancer (TNBC) and colorectal cancer (CRC) cells were used to evaluate ATX-559 mechanistic activity using immunofluorescence imaging, dot blot, and western blot assays. ATX-559 was dosed orally BID to evaluate its anti-tumor activity in CRC and TNBC cell line-derived xenograft (CDX) models, as well as a panel of 64 patient-derived xenograft (PDX) models including TNBC, Ovarian, CRC, endometrial, and gastric cancers. qPCR was used to evaluate the target engagement PD biomarker, circBRIP1.



 ATX-559 treatment leads to increased accumulation of R-loops and G-quadruplexes resulting in increased replicative stress, subsequent DNA damage, and cell death selectively in cancer models with genomic instability and replicative stress. ATX-559 sensitivity was enriched in dMMR/MSI-H cell lines including CRC. Sensitivity was also enriched in BRCA altered TNBC and HRD ovarian cancer cell lines. ATX-559 is well tolerated in vivo, resulting in robust dose-dependent tumor growth inhibition and regression selectively observed in CRC dMMR/MSI-H and BRCA altered TNBC CDX models, along with a well correlated PK/PD/efficacy relationship. Additionally, ATX-559 demonstrated enriched sensitivity in a large panel of BRCA altered TNBC (75%), dMMR/MSI-H (68%), and HRD ovarian (55%) PDX models.



 Together, these findings represent preclinical validation of DHX9 as a selective therapeutic in cancers with genomic instability and replication stress. ATX-559 is currently in a first-in-human, Phase 1/2, open-label, dose-escalation and expansion study to evaluate the safety profile of ATX-559 and to determine the recommended phase 2 dose (RP2D) and preliminary antitumor activity in patients with locally advanced or metastatic solid tumors, and molecularly defined cancers (NCT06625515).



 Sunaina Nayak, Jennifer Castro, Monique Laidlaw, Priya Rajaratnam, Maureen Lynes, Stuart Ince, Jason A. Sager, Serena J. Silver. ATX-559, a first-in-class, clinical-stage DHX9 inhibitor, as a targeted therapeutic for molecularly defined tumors with genomic instability and replicative stress [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr B045.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb2a8e9dfeed1b9e094182590e0697b62580f888" target='_blank'>
              Abstract B045: ATX-559, a first-in-class, clinical-stage DHX9 inhibitor, as a targeted therapeutic for molecularly defined tumors with genomic instability and replicative stress
              </a>
            </td>
          <td>
            S. Nayak, Jennifer Castro, Monique Laidlaw, Priya Rajaratnam, Maureen M. Lynes, Stuart Ince, Jason A. Sager, Serena J. Silver
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background/Objectives: During cancer development, tumor cells accumulate somatic mutations, which could generate tumor-specific neoantigens. The aberrant protein can be recognized by the immune system as no-self, triggering an immune response against cells expressing this aberrant protein which could mediate tumor control or rejection. Since the expression of this mutated protein is exclusive to tumor cells, great efforts are being made to identify neoantigens of relevance in the development of new cancer treatment strategies. In comparison to adulthood tumors, pediatric malignancies present fewer mutations and thus fewer potential neoantigens. Acute lymphoblastic leukemia (ALL) is the most common pediatric malignancy worldwide that can be benefited by the identification of neoantigens for immunotherapy approaches, the landscape of neoantigens in ALL is not well known, therefore the aim of our study was to identify potential neoantigens in ALL pediatric patients. Methods: To identify neoantigens in ALL, whole-exome sequencing of matched tumor-normal cells from pediatric cases was performed, with these data HLA-I alleles predicted and somatic mutations identified to propose potential neoantigens based on binding affinity of mutated peptide-HLA-I. Results: We found a strong correlation between tumor mutational burden (TMB) and neoantigen load (p < 0.001) but no correlation with prognosis. Furthermore, TMB and neoantigens were greater in ALL patients with at least one mutated DNA mismatch repair gene (p < 0.001). Also, differences between B- and T-cell ALL were found but statistical significance did not remain after permutation. Conclusions: The presence of neoantigens in pediatric cases with ALL makes the neoantigen-based immunotherapy a promising new strategy for the treatment of this malignancy, especially for patients with relapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/924894c8892c6596749a9249cd4cf79014fb16ed" target='_blank'>
              Somatic Mutations in DNA Mismatch Repair Genes, Mutation Rate and Neoantigen Load in Acute Lymphoblastic Leukemia
              </a>
            </td>
          <td>
            Diana Karen Mendiola-Soto, L. Gómez-Romero, J. C. Núñez-Enríquez, J. Flores-Lujano, E. Jiménez‐Hernández, A. Medina‐Sanson, V. Bekker-Méndez, M. Mata-Rocha, M. Pérez-Saldívar, D. A. Duarte-Rodríguez, J. R. Torres-Nava, J. G. Peñaloza‐González, L. V. Flores-Villegas, R. Amador-Sánchez, M. Velázquez-Aviña, J. A. Martín-Trejo, L. E. Merino-Pasaye, K. Solís-Labastida, R. M. Espinosa-Elizondo, C. Pérez-Amado, D. May-Hau, O. Sepúlveda-Robles, H. Rosas-Vargas, J. Mejía‐Aranguré, S. Jiménez-Morales
          </td>
          <td>2025-09-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d68b17c92ffe6cab39b80d1a0d838e3cbc4b57d" target='_blank'>
              EP300 genomic redistribution following E-cadherin inhibition in MCF7 cancer cells: evidence for early epigenetic reprogramming
              </a>
            </td>
          <td>
            H. Kaabi, Aseel Sharaireh, Hanan Aljohani
          </td>
          <td>2025-10-13</td>
          <td>Histochemistry and Cell Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Despite the initial responses to the tyrosine kinase inhibitor (TKI) for cancer therapy, many patients often relapse with no curative regimens available. Further, the ability to target therapeutic agents to cancer cells with appropriate doses remains challenging in the clinic, especially for leukemia. Here, we show that naïve CML cells are dynamically heterogeneous in colony formation. Larger clones expand while smaller ones diminish and eventually disappear. Compared to resistant cells, parental populations, including CD44+ stem cells, form a greater number of larger, solid spheroids. Upregulation of fat mass and obesity associated protein (FTO), an RNA N6-methyladenosine demethylase, and stem cell markers (e.g., CD44, CD133, CD25) is more obvious in resistant cells compared to parental cells. FTO inhibitors (e.g., CS1, FB23-2) appreciably impair the growth of resistant cells either alone or in combination with nilotinib. FTO protein expression is unexpectedly upregulated by CS1 or FB23-2 treatment in multiple leukemia cell lines. We then constructed RNA nanoparticles encapsulating FTO siRNAs and conjugated with anti-CD133 RNA aptamers. We showed that, compared to negative control, these nanoparticles were taken up much more efficiently by resistant cells that highly express CD133. Treatment with the CD133-guided FTO siRNA nanoparticles efficiently silenced FTO expression in resistant cells, which leads to a significant reduction in their colony and spheroid formation. These findings offer new insights into cancer drug resistance and advance the application of RNA nanotechnology for treating leukemia. The research provides a foundation for developing novel, targeted therapies for resistant leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/610e20fd12fcf52179a62018bf638a0c7e862c87" target='_blank'>
              CD133-Guided RNA Nanoparticle Delivery of FTO siRNA Impairs Leukemia Resistance to Tyrosine Kinase Inhibitor Therapy
              </a>
            </td>
          <td>
            Huiqin Bian, Changli Zhou, Hiroaki Koyama, Wen Gao, Sicheng Bian, Tao Cheng, Xiaonan Han, William Tse, Shujun Liu, Alexander Miron
          </td>
          <td>2025-10-15</td>
          <td>RNA NanoMed</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Histone acetylation is widely assumed to directly instruct gene activation. Among acetylated residues, H4K16ac is one of the most abundant modifications, conserved across all eukaryotes. Despite its established role in X-chromosome hyperactivation in Drosophila, its function in mammalian cells has remained elusive. Here, we show that in human somatic cells, H4K16ac does not substantially affect gene expression, but instead controls the spatiotemporal program of genome replication. By combining a meta-analysis of public datasets and perturbation experiments designed to minimize confounding effects, we found that H4K16ac is neither associated with nor required for transcriptional activity. Rather, H4K16ac depletion resulted in premature replication of heterochromatic regions and widespread alterations in replication timing across the genome. These defects were driven by the aberrant activation of cryptic replication origins at long terminal repeats—repetitive elements typically marked by H4K16ac and whose sequence context resembles that of canonical origins in euchromatic regions. Our findings reveal an unexpected role for one of the most prevalent chromatin modifications and uncover a new regulatory mechanism that safeguards genome replication fidelity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58d31c25c7ef175d7c8a34dda15d005ef97496d8" target='_blank'>
              Mammalian H4K16ac regulates the spatiotemporal order of genome replication rather than gene expression
              </a>
            </td>
          <td>
            Marta Milan, Valeria Runfola, Manthan Patel, Lucia Falbo, R. Noberini, C. Soriani, S. Rodighiero, Tiziana Bonaldi, Vincenzo Costanzo, Madapura M. Pradeepa, Paola Scaffidi
          </td>
          <td>2025-09-23</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Resistance to chemotherapy, targeted agents, and particularly immunotherapy remains the principal challenge in the management of gastrointestinal malignancies. This review aims to comprehensively delineate the molecular and microenvironmental drivers of resistance, with emphasis on mechanisms impacting immunotherapy response, and evaluate emerging, mechanism−guided interventions (including immunotherapeutic combinations) for precision therapy. We first examine intrinsic mechanisms—including drug−target alterations, dysregulated drug metabolism and efflux, hyperactivation of DNA damage repair pathways, and epigenetic remodeling—and extrinsic influences stemming from the tumor microenvironment and extracellular matrix remodeling. We then highlight epithelial–mesenchymal transition (EMT) as a critical nexus that integrates stromal cues with cell−intrinsic survival programs, thereby promoting drug efflux and immune evasion. Next, we discuss how single−cell and spatial omics, liquid biopsy, patient−derived organoids, and AI−enabled analytics facilitate subclone−level mapping of resistance networks and real−time tracking of clonal evolution. Finally, we review mechanism−based strategies—including KRAS G12C inhibitors, efflux−pump antagonists, apoptosis reactivators, and epigenetic/autophagy modulators—and propose an integrated, multimodal regimen leveraging immunotherapy where appropriate, informed by real-time drug sensitivity data (e.g., from liquid biopsy), dynamic biomarkers and AI−driven optimization to overcome resistance and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aebd9ae366f20576b8816fc8c01cf3154ec38755" target='_blank'>
              Integrated molecular and microenvironmental drivers of drug resistance in gastrointestinal cancers: mechanisms, immunotherapy challenges, and precision strategies
              </a>
            </td>
          <td>
            Heng Xu, Jiaan Lu, Jiangying Wu, Kangling Zhang, Xuancheng Zhou, Ziqi Gao, Jingqi Feng, Ziye Zhuang, Xiaolin Zhong
          </td>
          <td>2025-10-28</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 KRAS is a major oncogenic driver in pancreatic ductal adenocarcinoma (PDAC), mutationally activated in approximately 90% of cases. Mutations in this oncogene have been associated with more aggressive disease, poorer outcomes, and have remained hard to drug for over three decades. Recently developed small molecule inhibitors of KRAS (KRASi) have shown promising efficacy in advanced, previously treated PDAC patients but ultimately, most patients develop acquired resistance. Approximately half of these resistance cases do not present a putative genomic driver. In preclinical studies using genetically engineered mouse models of PDAC, we and others have demonstrated that cell state identity along the classical/epithelial-mesenchymal axis is a key determinant of response and resistance to KRASi. Notably, acute KRASi treatment initially induces a strong and selective bottlenecking of the malignant population in vivo, resulting in tumors that are predominantly classical with depleted mesenchymal characteristics. While these findings highlight a potent cell state-selective effect of acute KRASi treatment on PDAC cells, the underlying mechanisms driving the drug-induced cell state selection and plasticity remain poorly characterized. These mechanisms may reveal novel therapeutic targets for developing combination therapies that improve therapeutic responses. To comprehensively map and modulate epithelial-mesenchymal (E-M) cell state plasticity in response to KRASi treatment, we performed CRISPRi Perturb-seq (single-cell gene expression readout) with lineage tracing on a patient-derived PDAC cell line treated with the RAS(ON) multi-selective inhibitor RMC-7977. We captured 734,092 high-quality single-cell transcriptomic profiles and 156,119 unique clones across 60 genetic perturbations, including E- and M-specific transcription factors (TF) inferred to influence E-M plasticity from previous single-cell lineage tracing experiments. To optimally model clonal growth and transition rates, we profiled cells across a time series at day 6, day 15 and day 22, after 1 week of DMSO or KRASi treatment. We developed a hierarchical generative probabilistic model to jointly infer E↔M transition rates and cell state-specific growth rates, while capturing the effects of perturbations and KRASi treatment on state dynamics. Analysis of longitudinally tracked clones revealed that KRASi treatment specifically reduced the M-state growth rate while increasing the probability of M→E transitions compared to DMSO. Notably, we identified several TFs, displaying KRASi-dependent changes in state transition dynamics, either promoting (ELF3, MEIS2 among others) or decreasing (ZNF281, IRF9, among others) probabilities of M→E transitions when compared to non-targeting controls. These preclinical findings suggest that perturbation of these factors may be used as a paradigm to prevent cell state plasticity upon acute KRASi treatment and could be used to homogenize tumor populations towards a treatment-sensitive state.



 Julien Dilly, Mike Bogaev, Lynn Bi, Abigail Collins, Martin Jankowiak, Aziz Al'Khafaji, Kyle E. Evans, Mehrtash Babadi, Thouis R. Jones, Elisa Donnard, David T. Ting, Nir Hacohen, Dana Pe'er, Eric S. Lander, Andrew J. Aguirre, Arnav Mehta. Mapping and modulating epithelial-mesenchymal plasticity under RAS(ON) multi-selective inhibition in PDAC through lineage tracing and Perturb-seq [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B120.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/095259190df0afbd227ee5a60c0db74dfea61a15" target='_blank'>
              Abstract B120: Mapping and modulating epithelial-mesenchymal plasticity under RAS(ON) multi-selective inhibition in PDAC through lineage tracing and Perturb-seq
              </a>
            </td>
          <td>
            Julien Dilly, Mike Bogaev, Lynn Bi, Abigail Collins, Martin Jankowiak, Aziz Al’Khafaji, Kyle E Evans, M. Babadi, Thouis R. Jones, Elisa Donnard, David T Ting, N. Hacohen, D. Pe’er, Eric S. Lander, A. Aguirre, Arnav Mehta
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>116</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28967441eb1f46f1a9e983b8df32cfb979b71b23" target='_blank'>
              Chromatin and gene-regulatory dynamics of human pulmogenesis by single cell multiomic sequencing
              </a>
            </td>
          <td>
            Samuel H. Kim, Soon Il Higashino, Lea C. Steffes, Li Li, Anca M. Pașca, Maya Kumar, Christin S. Kuo, Mark A. Krasnow, William J. Greenleaf
          </td>
          <td>2025-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="The era of targeted therapies has significantly advanced our understanding of cancer growth and metastasis. Intrinsic or acquired drug resistance remains a major challenge, rendering clinically overt metastatic disease incurable in most patients. This review first examines key clinical trials and their primary outcomes involving targeted therapies in the most common advanced solid tumors, along with the main mechanisms underlying drug resistance. Recently, micrometastatic disease has emerged as a novel focus of investigation aimed at definitively curing advanced solid tumors. Accordingly, this review explores the biology of micrometastases, current challenges in their detection and monitoring, and the main strategies proposed to prevent their progression. The potential roles of nanotechnology and artificial intelligence‐driven predictive models are also discussed. Furthermore, we highlight specific characteristics of micrometastatic disease that favor immune modulation, and we evaluate the effectiveness of an immunotherapy regimen that inhibits immune suppression. The lead time provided by serum tumor markers, used experimentally to better track the progression of otherwise undetectable micrometastatic disease, also forms the mechanistic basis for a novel protocol we propose to prevent relapse in high‐risk cancer patients. This innovative protocol holds scientific relevance being supported by an appropriate mathematical model and ready for immediate application in clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d0a2d7b2866fae7f5ad7e2d51fefa760b9024b2" target='_blank'>
              Targeted Therapies in the Most Common Advanced Solid Tumors, Drug Resistance, and Counteracting Progressive Micrometastatic Disease: The Next Frontier of Research
              </a>
            </td>
          <td>
            Andrea Nicolini, Paola Ferrari, Roberto Silvestri, Dario A. Bini
          </td>
          <td>2025-09-27</td>
          <td>MedComm</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Cancers develop and evolve by overcoming a variety of selective pressures including cell-intrinsic growth-inhibitory mechanisms, microenvironmental stresses, and therapeutic interventions. In addition to mutational mechanisms, cancer cell adaptation via transcriptional plasticity remains an important but poorly understood driver of cancer cell fitness. For example, ctDNA analysis of disease progression samples after KRAS inhibitor (KRASi) treatment identified genomically-defined mechanisms of resistance in only approximately 50% of patients, suggesting that non-genomic mechanisms play an equally important role. Thus, there is a pressing need to devise systematic and scalable methods to determine essential drivers of prognostically-relevant pancreatic cancer cell states, and to engineer state-relevant models that can be applied to discover new therapeutic approaches that overcome these modes of resistance. Here, we present a strategy to systematically identify transcriptional drivers of inflammation- and therapy-resistant cell states in pancreatic cancer. We have established a robust screening pipeline to 1) over-express human transcription factor (TF) isoforms in a pooled format within cancer models; 2) perform positive-selection screens under a variety of selective pressures to define transcriptional mechanisms of inflammation tolerance and KRASi resistance; 3) perform single-cell RNA sequencing to define resistance phenotypes; and 4) conduct mechanistic studies to examine the downstream signaling networks that facilitate cancer cell growth in the setting of these selective pressures. We uncover TFs and states that drive resistance in a tissue- and perturbation-specific manner, as well as states that are robust to multiple different selective pressures. A subset of TFs and associated gene programs induce tolerance to chronic interferon exposure in our screens and are reflected in vivo in the setting of immunotherapy resistance. We also identify distinct TF families that drive resistance to pharmacologic KRAS inhibition. Notably, cancer cell states that confer resistance to KRAS inhibition also exhibit cross-resistance to other MAPK pathway inhibitors and to microenvironmental signals, suggesting that cell state adaption may contribute to increased cancer cell fitness and multi-drug resistance as cancers progress. More broadly, our strategy enables the systematic identification and mapping of cell states that drive cancer cell fitness to in vivo clinical atlas datasets, providing a framework for interpretation of clinical phenotypes and predictive modeling of cell state evolution. Furthermore, the cell state-defined models developed here provide a substrate for new therapeutic discovery using cancer cell states as biomarkers, a key next step to developing combination strategies that overcome therapeutic resistance and improve outcomes for pancreatic cancer patients.



 Yuzhou Evelyn. Tong, Aswanth Mahalingam, Zixin Chen, Jacob Smigiel, Tsukasa Shibue, Andew Navia, Alex K. Shalek, Lorin Crawford, Ava Amini, Peter S. Winter, Srivatsan Raghavan. Defining cancer cell states that drive inflammation tolerance and KRAS inhibitor resistance in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B103.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1dd25ee2338a106f19e6d004e36d07be5c2aef6" target='_blank'>
              Abstract B103: Defining cancer cell states that drive inflammation tolerance and KRAS inhibitor resistance in pancreatic cancer
              </a>
            </td>
          <td>
            Yuzhou Evelyn Tong, Aswanth Mahalingam, Zixin Chen, Jacob Smigiel, Tsukasa Shibue, Andew Navia, A. Shalek, Lorin Crawford, Ava P. Amini, Peter S. Winter, Srivatsan Raghavan
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Immune checkpoint inhibitors have transformed treatment for several cancers, yet clinical trials of programmed cell death protein 1 (PD-1) blockade in glioblastoma (GBM) have consistently failed to show therapeutic benefit. While some studies have reported treatment-related transcriptional changes, particularly in T cells, findings remain limited and inconsistent. The aim of this study was to investigate changes in tumor cells and tumor-associated macrophages (TAMs) after PD-1 blockade in recurrent GBM using spatial transcriptomics. We performed Digital Spatial Profiling (GeoMx, NanoString) on FFPE tumor samples from 26 patients with matched primary and recurrent IDH-wildtype GBM, including 16 patients who received neoadjuvant nivolumab at recurrence. Tumor (SOX2⁺) and TAM (IBA1⁺) segments were selected for targeted spatial analysis. Following quality control and filtering, transcriptomic profiles were compared between nivolumab-treated and untreated recurrent tumors. PD-1 blockade did not induce detectable gene expression changes in either tumor cells or TAMs. There were no significant differences in global expression profiles or in more targeted analyses of malignant cell states, cell cycle activity, interferon signaling, or myeloid transcriptional programs. These results consistently indicate that neoadjuvant PD-1 blockade does not elicit measurable responses at the spatial transcriptomic level in tumor cells or TAMs in recurrent GBM. These findings align with the lack of clinical benefit observed in trials and highlight the need for alternative strategies to improve immunotherapy outcomes in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d6af3945d99a9f12a27763251dbd874f6e32349" target='_blank'>
              Spatial transcriptomic analysis reveals lack of response to PD-1 blockade in recurrent glioblastoma
              </a>
            </td>
          <td>
            S. Artzi, M. N. Klausen, D. Harwood, S. Michaelsen, S. Maarup, A. Locallo, V. Fougner, N. Bager, N. Hammouda, D. Nørøxe, B. Hasselbalch, U. Lassen, J. Weischenfeldt, B. Kristensen
          </td>
          <td>2025-09-17</td>
          <td>Acta Neuropathologica</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Ovarian high-grade serous carcinoma (HGSC) is an aggressive ovarian cancer with a heterogeneous tumor microenvironment (TME). Advances in single-cell RNA sequencing (scRNA-seq) and spatially-resolved transcriptomics have enabled the study of complex TME. This study explores connections between molecular subtypes described from bulk transcriptomes and spatial domains characterized by distinct gene expression in HGSC and their variability between patients. We explored both intraand inter-tumor heterogeneity across 2D space and identified differing spatial patterns of gene expression pertaining to immune pathways and vasculature development. Functional characterization of tumor spaces revealed potentially shared cell states across molecular subtypes, while correlation analysis underscored subtype-specific spatial anti-colocalization between spots exhibiting antigen-presenting functions and B cell-mediated immunity. Lastly, we performed spatially-aware cell-cell communication analysis on the spatial samples and identified a molecular subtype specific difference in total signaling activity and heterogeneity in Midikine signaling between the differentiated subtype. Our results suggest that generating multiple tissue slices per patient might be necessary to enable comprehensive characterization of HGSC spatial transcriptomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12564a0c02e3362085c018e2a380154da3a01959" target='_blank'>
              Characterizing intra- and inter-tumor heterogeneity in Ovarian high-grade serous carcinoma subtypes using single-cell and spatial transcriptomics
              </a>
            </td>
          <td>
            Weishan Li, Laurie Grieshober, Jason Gertz, Adriana Ivich, J. Doherty, Casey S. Greene, SC Hicks
          </td>
          <td>2025-09-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 While PARP inhibitors (PARPi) have demonstrated clinical benefit in patients with BRCA1/2-mutated ovarian cancer by exploiting synthetic lethality, only a subset of patients respond, and many eventually acquire resistance. PARPi exert cytotoxic effects by blocking single-strand break repair and trapping PARP on DNA, resulting in replication fork stalling and subsequent double-strand break formation. However, cells may circumvent these effects through translesion DNA synthesis (TLS), a DNA damage tolerance pathway that allows replication to continue past DNA lesions using specialized polymerases, potentially leading to replication fork recovery and reduced therapeutic efficacy. We hypothesized that TLS contributes to PARPi resistance in BRCA-deficient ovarian cancer by enabling cells to bypass PARP-DNA complexes and avoid lethal fork collapse.



 We used a panel of homologous recombination (HR)-deficient epithelial ovarian cancer (EOC) cell lines and patient-derived organoids (PDOs) to investigate the role of TLS in mediating resistance to PARPi. DNA fiber assays were performed to assess replication fork dynamics following treatment with olaparib or talazoparib. TLS activity was disrupted genetically by depleting REV1 and POLH or pharmacologically using the TLS inhibitor JH-RE-06. Synergistic effects with olaparib were evaluated in vitro. In vivo efficacy and resistance prevention were assessed using both orthotopic xenograft and syngeneic ovarian cancer mouse models.



 PARPi treatment significantly increased replication fork stalling and the expression of monoubiquitinated PCNA, a key marker of TLS activation. Depletion of REV1, a key factor in TLS, sensitized BRCA2-deficient PEO1 cells to olaparib and intensified replication fork stalling, indicating that TLS is functionally activated to counteract PARPi-induced replication stress. Among TLS polymerases, POLη (encoded by POLH) played a predominant role in facilitating fork progression and promoting cell survival following PARPi treatment. The TLS inhibitor JH-RE-06 synergized with olaparib to induce greater cytotoxicity in HR-deficient EOC cells and PDOs. In vivo, JH-RE-06 enhanced the anti-tumor efficacy of olaparib and delayed or prevented the emergence of PARPi resistance in both BRCA2-deficient xenograft and syngeneic ovarian cancer models. Notably, long-term olaparib exposure led to secondary BRCA2 mutations and acquired resistance in PEO1 cells, while no such mutations or resistant clones emerged in POLH-depleted PEO1 cells, suggesting that TLS may promote the development of acquired PARPi resistance by facilitating secondary BRCA1/2 mutations and partial restoration of HR.



 Our findings reveal that TLS, particularly POLη-mediated bypass, plays a key role in promoting replication fork rescue and survival in BRCA-deficient ovarian cancer treated with PARPi. Inhibiting TLS with JH-RE-06 enhances PARPi sensitivity and prevents resistance, providing a strong rationale for co-targeting TLS in patients with HR-deficient ovarian cancer.



 Na Li, Qianyun Ge, Yajing Yang, Linzhou Wang, Aidan Li, Jessica Miao, Junran Zhang, Qi-En Wang. Targeting translesion synthesis to enhance PARP inhibitor efficacy and prevent resistance in BRCA-deficient ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A054.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06785f3b639dc809e66ebcf6eb7d4426d019354e" target='_blank'>
              Abstract A054: Targeting translesion synthesis to enhance PARP inhibitor efficacy and prevent resistance in BRCA-deficient ovarian cancer
              </a>
            </td>
          <td>
            Na Li, Qianyun Ge, Yajing Yang, Linzhou Wang, Aidan Li, Jessica Miao, Junran Zhang, Qi-En Wang
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b270922941e97bc91f75cdb1ab9aa350f3916bd" target='_blank'>
              Mitotic bypass and endocycling promote cancer cell survival after genotoxic chemotherapy
              </a>
            </td>
          <td>
            Kevin Truskowski, Louis T A Rolle, George Butler, Margaret E Yang, Kenneth J. Pienta, S. Amend
          </td>
          <td>2025-11-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Ovarian cancer is a highly lethal malignancy, with treatment efficacy frequently compromised by the development of cisplatin resistance. This study investigates the molecular mechanisms driving cisplatin resistance, focusing on interferon-induced protein with tetratricopeptide repeats 1 (IFIT1), a key interferon-stimulated gene. Through comprehensive transcriptome profiling, we systematically compared gene expression patterns between cisplatin-sensitive and cisplatin-resistant ovarian cancer cells, identifying a consistent upregulation of IFIT1 in resistant cells. Functional assays provided evidence that IFIT1 plays a central role in mediating cisplatin resistance. Mechanistically, IFIT1 was found to be associated with activation of inflammatory signaling pathways, suggesting its involvement in resistance-related immune regulation. Clinically, elevated IFIT1 expression significantly correlated with an unfavorable prognosis in ovarian cancer patients. Moreover, we demonstrated that hypoxia-inducible factor-1 alpha transcriptionally regulates IFIT1, linking its expression to hypoxia response. Overall, these findings identify IFIT1 as a key driver of cisplatin resistance in ovarian cancer, shedding light on its clinical significance and the regulatory networks underpinning therapy resistance. IFIT1 holds promise both as a prognostic indicator and as a target for therapeutic intervention. However, its clinical translation will depend on rigorous validation and in-depth exploration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e3e993b5a5faff584d9a192dda2fe082dc88120" target='_blank'>
              Overcoming cisplatin resistance in ovarian cancer by targeting IFIT1-mediated inflammation responses
              </a>
            </td>
          <td>
            Min Hu, Xuejiao Zhao, Xiuming Li, Bin Liu, Chunluan Yuan
          </td>
          <td>2025-11-06</td>
          <td>Tumor Discovery</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Reprogramming gene expression at the translational level drives intestinal tumorigenesis. Codon decoding during translation elongation relies on tRNA modifications, while their pathological relevance in colorectal cancer remains to be elucidated. Here, we show that AlkB homolog 8 (ALKBH8), a uridine 34 (U34) tRNA methyltransferase, is a direct target of Wnt/β-catenin and is upregulated in colorectal cancer. Genetic ablation of ALKBH8 inhibits the development of intestinal tumors in Apcmin/+, azoxymethane/dextran sulfate sodium (AOM/DSS), and xenograft models. Loss of ALKBH8 induces ribosome pausing at adenine-ending codons, impairing the translation elongation of mRNAs enriched with these codons. Specifically, ALKBH8 regulates the translation of KRAS proto-oncogene in a codon-dependent manner. Rescue experiments demonstrate that the methyltransferase activity of ALKBH8 is required for its translation-promoting function. Together, our findings reveal ALKBH8-dependent mRNA translation as a critical mediator of intestinal tumorigenesis, underscoring its potential as a promising target for colorectal cancer therapy. ALKBH8 is a tRNA methyltransferase, specifically modifying the wobble uridine (U34) in tRNAs. Here, the authors discover that ALKBH8 is a direct transcriptional target of β-catenin, and that mediates intestinal tumorigenesis through regulation of the translation of KRAS in a codon-dependent manner.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/741e7e4c4eea499624351c733df3953216fc634e" target='_blank'>
              ALKBH8-mediated codon-specific translation promotes colorectal tumorigenesis
              </a>
            </td>
          <td>
            Yu Qian, Canlan Wu, Saisai Wei, Sujun Yan, Junxuan Peng, Lei Yu, Yunyi Gao, Jingyu Hou, Wentao Yu, Zhanghui Chen, Jun Zhang, Xiangwei Gao
          </td>
          <td>2025-10-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is one of the most lethal malignancies and is projected to become the second leading cause of cancer-related death within the next decade. Population-scale genomic analyses of PDAC tumors have revealed extensive chromosomal instability (CIN), indicating high levels of genomic diversification that may influence therapy response. However, the mechanisms driving CIN during PDAC progression and its role in treatment response remain poorly understood. We hypothesize that specific patterns of genomic rearrangements emerging during PDAC evolution can inform two key aspects of tumor biology: (i) the underlying processes driving CIN, which may be therapeutically targetable, and (ii) subclonal CIN signatures that predispose tumors to develop chemoresistance.



 To investigate the dynamics of CIN and chemoresistance, we employed an in-house single-cell clonal outgrowth assay integrated with single-cell whole-genome sequencing (sc-WGS) and mathematical modelling of clonal growth. Experiments were performed in PDX-derived primary cultures and patient-derived 3D organoids. We tracked clonal outgrowth under both untreated conditions and following exposure to chemotherapeutic agents relevant to PDAC treatment. Comparative analyses of single-cell copy number profiles between parental populations and drug-exposed clones allow us to identify any de novo genomic alterations acquired during treatment, as well as pre-existing events enriched by drug selection.



 We observed ongoing CIN in PDAC cells, leading to continuous generation of new CNAs and enhanced cell-to-cell genomic diversity. Our data demonstrate pronounced heterogeneity in copy number states and growth dynamics among individual clones from the same cell line. This intrinsic heterogeneity provides a reservoir for evolutionary selection and may promote the emergence of drug-resistant phenotypes. We were able to analyse which clones carried pre-existing alterations and/or which generated new ones under the pressure of chemotherapy, allowing us to understand the diversity of responses and general principles of CNA changes in response to therapy. These findings underscore the innate adaptability of PDAC cells under therapeutic pressure.



 Our study highlights the dynamic nature of CIN in PDAC and its critical role in promoting chemoresistance. The identification of specific genomic alterations enriched in resistant subclones could offer potential biomarkers predictive of therapy response. This approach provides new insights into the interplay between CIN, cell-to-cell heterogeneity, and therapeutic adaptation, with potential future implications for guiding personalized treatment strategies in PDAC.



 Audrey Lumeau, Molly Anne. Guscott, Nathaniel Mon Pere, Isabel Nichols, Benjamin Werner, Nelson Dusetti, Sarah E. McClelland. Tracking genome evolution and chemoresistance in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B129.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b332fbacb3e055d135e8c8f4dfd0dbc1124774d" target='_blank'>
              Abstract B129: Tracking genome evolution and chemoresistance in pancreatic cancer
              </a>
            </td>
          <td>
            A. Lumeau, Molly A Guscott, N. M. Père, Isabel Nichols, Benjamin Werner, Nelson Dusetti, Sarah E. McClelland
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34ae3edf56d18af7c1ee555489c875e8d629f4c8" target='_blank'>
              RRAS and RRAS2 mutations are oncogenic drivers in lung cancer and are sensitive to the pan-RAS inhibitor RMC-6236
              </a>
            </td>
          <td>
            A. Pfeil, Tom Zhang, Ryan Cheng, M. Mattar, Leo Gili, Rachel Lai, I. Khodos, Rohan P. Master, Jenna-Marie Dix, Andrea M Gazzo, K. Arbour, E. de Stanchina, Christopher A Febres-Aldana, Marc Ladanyi, S. Yang, R. Somwar
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a2c0bd20d360ee798769e48db594be4df9b6168" target='_blank'>
              Comparative analysis of single-stranded and non-canonical DNA formation in human and other ape cells with telomere-to-telomere genomes
              </a>
            </td>
          <td>
            Jacob P. Sieg, Huiqing Zeng, Hana Pálová, Linnéa Smeds, S. Mohanty, Angelika Lahnsteiner, Francesca Chiaromonte, Kateryna D. Makova
          </td>
          <td>2025-11-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="The transcription of satellite DNA is highly sensitive to environmental factors and represents a source of genomic instability. Therefore, tight regulation of (peri)centromeric transcription is essential for genome maintenance. Antibiotics are routinely used for in vitro studies and for medical treatment, however, their effect on pericentromeric satellite DNA transcription was not investigated. Here we show that antibiotics geneticin and hygromycin B, conveniently used in cell culture, as well as rifampicin (along with five other antibiotics), used to treat bacterial infections, increase transcription of a major human pericentromeric alpha satellite DNA in cell lines at standard concentrations. However, response differs among cell lines - maximal increase in A-1235 cells is obtained by rifampicin while in HeLa cells and fibroblasts by geneticin. There is also a positive correlation between antibiotic concentration and the level of alpha satellite transcription. The increase of transcription is accompanied with either H3K9me3 decrease or H3K18ac increase at tandemly arranged alpha satellite arrays while H3K4me2 remains unchanged. Our results suggest that induced alpha satellite DNA transcription upon antibiotic stress could be linked to epigenetic changes - histone modifications H3K9me3 and H3K18ac, which are associated with transcription of heterochromatin.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03ef88bdc2c47f36c0fdbaf887b60122c711579a" target='_blank'>
              Antibiotics induce overexpression of alpha satellite DNA accompanied with epigenetic changes at alpha satellite arrays as well as genome-wide
              </a>
            </td>
          <td>
            Sven Ljubić, Maja Matulić, Damir Đermić, Maria Chiara Feliciello, Alfredo Procino, Đurđica Ugarković, I. Feliciello
          </td>
          <td>2025-09-26</td>
          <td>Epigenetics & Chromatin</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="BRD9 is involved in multiple physiological and pathological pathways, yet its functional role and molecular mechanisms in gastric cancer (GC) remain largely unexplored. Addressing this knowledge gap is critical given the persistent global mortality burden of GC and the limited efficacy of current therapeutic strategies. BRD9 expression in GC patients was systematically analyzed using immunohistochemical (IHC) assays and transcriptomic datasets. Comprehensive functional validation, employing cellular and murine tumor models, elucidated BRD9’s role in GC progression. Molecular pathways underlying BRD9-mediated gastric carcinogenesis were delineated through integrated approaches, including RNA sequencing, co-immunoprecipitation (co-IP), subcellular fractionation, and luciferase reporter assays. BRD9 was significantly overexpressed in GC and associated with poor patient prognosis. Functionally, BRD9 promoted GC cell proliferation and enhanced DNA damage repair capacity. Mechanistically, elevated BRD9 expression inhibited p53 nuclear translocation via direct binding, subsequently activating the E2F transcription factor family. Notably, we identified that E2F1 directly bound to and transactivated the BRD9 promoter, establishing a positive feedback loop that sustains BRD9 expression. Additionally, BRD9 knockdown sensitized GC cells to cisplatin and oxaliplatin treatment. These findings highlight the critical role of BRD9 in GC progression and its therapeutic potential. The BRD9-p53-E2F1 axis acts as a crucial regulator of GC cell proliferation and DNA damage response. Targeting BRD9 pharmacologically could be a novel therapeutic approach to enhance chemotherapy efficacy and improve treatment outcomes in GC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1dadff4be7ebdf814a21760a8db20acee07e5dfa" target='_blank'>
              BRD9-p53-E2F1 circuit orchestrates cell growth and DNA damage repair in gastric cancer
              </a>
            </td>
          <td>
            Qingqing Zhou, Qi Wang, Yantao Duan, Chi Zhang, Tengfei Liu, Hengrui Liu, Mindi Zhao, Zhihuang Hu
          </td>
          <td>2025-10-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="OBJECTIVE
Large-scale CRISPR screens have identified essential genes across cancer cell lines, but links between tumor functional properties and specific dependencies require investigation to reveal the mechanisms underlying dependencies and broaden understanding of targeted therapy.


METHODS
We selected 47 breast cancer cell lines from the Cancer Cell Line Encyclopedia (CCLE) with multi-omics data including gene dependency; somatic mutations; copy number alterations; and transcriptomic, proteomic, metabolomic, and methylation data. We established a dependency marker association (DMA) analytic pipeline by using linear regression modeling to assess associations between 3,874 representative gene dependencies and multi-omics markers. Additionally, we conducted non-negative matrix factorization clustering, to stratify breast cancer cell lines according to gene dependency features, and investigated cluster-specific DMAs.


RESULTS
We interpreted valuable DMAs according to two primary aspects. First, dependencies associated with gain-of-function alterations revealed addiction to lactate transporter SLC16A3, thus suggesting a promising therapeutic target. Second, dependencies associated with loss-of-function alterations included synthetic lethality (SL), collateral SL, and prioritized metabolic SL, encompassing paralog SL (e.g., IMPDH1 and IMPDH2), single pathway SL (e.g., GFPT1 and UAP1), and alternative pathway SL (e.g., GPI and PGD). DMA analysis of the two clusters with divergent dependency signatures demonstrated that cluster1 cell lines exhibited extensive metabolism with mitochondrial protein dependencies, whereas cluster2 displays enhanced cell signaling, and reliance on DNA replication and membrane organelle regulators.


CONCLUSIONS
We established a DMA analysis pipeline linking the gene dependencies of breast cancer cell lines to multi-omics characteristics, thus elucidating the underpinnings of tumor dependencies and offering a valuable resource for developing novel precision treatment strategies incorporating relevant markers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/753908ccf816b6d23f787d4f37906f4b14d28c45" target='_blank'>
              Comprehensive investigation of the molecular basis of cancer dependencies suggests therapeutic options for breast cancer.
              </a>
            </td>
          <td>
            Rui Ding, Zhimin Shao, Tianjian Yu
          </td>
          <td>2025-10-23</td>
          <td>Cancer biology & medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Uncontrolled proliferation, resistance to apoptosis, inability to maintain genome integrity, and, recently, epigenetic reprogramming are all hallmarks of cancer. A number of gene expression and cell signaling networks control these-often-interconnected processes, while the study of their deregulation is in the forefront of cancer research for decades. Here we present data from cells and patients indicating that KMT2C, one of the most frequently mutated proteins in solid malignancies, is involved in all these processes. Its loss, a bad prognosis marker in bladder cancer, is associated with activation of the PI3K/PDK/AKT oncogenic/antiapoptotic axis, and tolerance to DNA damage during cell cycle progression. On the other hand, these cells suffer from mitotic stress that can be therapeutically exploited. Treatment with a PLK1 inhibitor showed high efficacy in vivo, and was associated with mitotic catastrophe and cellular senescence, providing evidence that targeting genes that promote mitotic progression could be a promising therapeutic approach in the subset of tumors with KMT2C loss.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a45d418115a05c2f3fb12ff53ed7255c53656e4" target='_blank'>
              KMT2C inactivation leads to PTEN downregulation and tolerance to DNA damage during cell cycle progression
              </a>
            </td>
          <td>
            T. Rampias, Andreas Goutas, Dimitris Karagiannis, Z. Kanaki, Antigoni Makri, L. Hoxhallari, Fotini Ε. Koukouzeli, Varvara Paraskevopoulou, Dimitra Tsouraki, Nikolaos Paschalidis, M. Avgeris, Andreas Scorilas, A. Klinakis
          </td>
          <td>2025-10-28</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Esophageal squamous cell carcinoma (ESCC) is a highly invasive malignant tumor, and its tumor microenvironment (TME) plays a pivotal role in disease progression and treatment resistance. As key components of the TME, fibroblasts and keratinocytes are hypothesized to synergistically regulate tumor biological behavior. This study integrated single-cell transcriptomic data with bulk RNA-seq data from ESCC to investigate the interaction mechanisms between fibroblasts and keratinocytes within the tumor microenvironment. Rigorous quality control, standardization, and clustering analyses were performed on the single-cell data, identifying ten distinct cell subpopulations, including fibroblasts and keratinocytes. The proportions and expression profile differences of these two cell types were compared between experimental and control groups. Cell–cell communication analysis deciphered ligand-receptor pairs and associated signaling pathways governing intercellular interactions. Molecular subtyping of The Cancer Genome Atlas (TCGA) cohort samples was subsequently performed using non-negative matrix factorization (NMF) based on interaction-specific genes. A multi-gene prognostic model was constructed via least absolute shrinkage and selection operator (LASSO) regression, and its predictive efficacy was evaluated. Finally, functional enrichment, immune microenvironment assessment, and differential expression analyses revealed differences in biological functions and immune infiltration patterns across distinct risk subtypes. By integrating the gene expression profiles of these two cell types, a novel molecular classification system was successfully established, enabling the stratification of ESCC patients into C1(high-risk) and C2(low-risk) subtypes with marked differences in survival outcomes. Patients classified as subtype C2 had a median survival of 3.8 months, significantly longer than the 1.8 months observed in the high-risk subtype C1, highlighting the substantial clinical relevance of this classification. Further analysis revealed that the high-risk cohort exhibited significant enrichment in metabolism-related pathways, such as endocytosis and glyceride metabolism, whereas the low-risk cohort was predominantly associated with pathways involved in cell junctions and structural maintenance. Mechanistically, the interaction between these two cell types jointly promotes the reprogramming of the tumor immune microenvironment by regulating immune cell infiltration, HLA molecule expression, and immune checkpoint activity. Furthermore, the study identified key genes, including SLIT2 and SFRP1, which display cell-type-specific expression patterns in ESCC and hold potential as prognostic biomarkers or therapeutic targets. This research provides novel insights into the regulatory mechanisms of the ESCC tumor microenvironment, laying a theoretical foundation for the development of precision therapeutic strategies targeting the tumor microenvironment. This study elucidates the synergistic regulatory mechanisms between fibroblasts and keratinocytes within the tumor microenvironment of ESCC. These findings offer a novel theoretical foundation for molecular classification, prognosis prediction, and precision treatment of ESCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cec3cca186de01a25c15c9e25673d6418ec7eece" target='_blank'>
              Mechanisms of synergistic regulation of the tumor microenvironment by fibroblasts and keratinocytes in esophageal squamous cell carcinoma
              </a>
            </td>
          <td>
            Yujie Weng, Rong Jia, Pengfei Ning
          </td>
          <td>2025-11-07</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Glioblastoma (GBM), as one of the most common and aggressive primary brain tumors, pose significant challenges in diagnosis and treatment, highlighting the urgent need for the identification of novel biomarkers, NFKBIE, which may play a critical role in tumor progression, immune modulation, and therapeutic response. We investigated the role of NFKBIE in GBM through both bioinformatics analysis and experimental validation. Initially, bioinformatics analysis based on public databases revealed significant differential expression of NFKBIE across various cancer types, particularly in GBM, where elevated expression was correlated with poor patient prognosis, including overall survival, disease-specific survival, and progression-free survival. Furthermore, genetic alterations in NFKBIE, such as copy number variations (CNVs) and tumor mutational burden (TMB), were significantly associated with tumor progression. In the experimental validation phase, we utilized small interfering RNA (siRNA) technology to silence NFKBIE expression in GBM cell lines, including U87 and T98G. Functional assays were then performed to assess the impact of NFKBIE knockdown. Cell proliferation assays, including CCK-8 and clonogenic assays, demonstrated that silencing NFKBIE significantly inhibited the proliferation of GBM cells. Migration and invasion assays, including wound healing and Transwell assays, further confirmed that NFKBIE knockdown markedly suppressed the migratory and invasive abilities of GBM cells. Additionally, apoptosis assays, such as TUNEL staining, revealed that NFKBIE silencing significantly promoted apoptosis in GBM cells. Western blot analysis confirmed the changes in the expression of key markers associated with proliferation, stemness, migration, invasion, and apoptosis. Furthermore, in a mouse xenograft model, NFKBIE knockdown significantly slowed tumor growth. Histological analysis of the excised tumors confirmed that reduced NFKBIE expression was closely associated with the inhibition of tumor growth. Notably, NFKBIE silencing also led to the downregulation of components of the Hedgehog signaling pathway, suggesting that NFKBIE may regulate tumor progression through modulation of this critical pathway. In conclusion, high NFKBIE expression is strongly associated with GBM progression and poor prognosis, and targeting its expression may offer a promising therapeutic strategy for GBM treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/567a01090acb470e17dfe52d5931751c0e81fc63" target='_blank'>
              NFKBIE as a prognostic biomarker and therapeutic target for GBM: role in Hedgehog signaling activation
              </a>
            </td>
          <td>
            Feng Mo, Yang Li, Ce Dong, Lijun Yang, Xiaoli Ma, Baogen Pan, Z. Fan
          </td>
          <td>2025-10-06</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="We have investigated changes in chromosome conformation, nuclear organization, and transcription during differentiation and maturation of control and mutant motor neurons harboring hexanucleotide expansions in the C9orf72 gene that cause amyotrophic lateral sclerosis (ALS). Using an in vitro reprogramming, differentiation and neural maturation protocol, we obtained highly purified populations of post-mitotic motor neurons for both normal and diseased cells. As expected, as fibroblasts are reprogrammed into iPSCs, and as iPSCs differentiate into motor neurons, chromatin accessibility, chromosome conformation, and nuclear organization change along with large-scale alterations in transcriptional profiles. We find that the transcriptome changes extensively during the first three weeks of post-mitotic neuronal maturation, with thousands of genes changing expression, but then is relatively stable for the next three weeks. In contrast, chromosome conformation and nuclear organization continue to change over the entire 6-week maturation period: chromosome territoriality increases, long-range interactions along chromosomes decrease, compartmentalization strength increases, and centromeres and telomeres increasingly cluster. In motor neurons derived from ALS patients such changes in chromosome conformation were much reduced. Chromatin accessibility changes also showed delayed maturation. The transcriptome in these cells matured relatively normally but with notable changes in expression of genes involved in lipid, sterol and mitochondrial function. We conclude that neural maturation is associated with large scale post-mitotic changes in gene expression, chromosome conformation and nuclear organization, and that these processes are defective in motor neurons derived from ALS patients carrying C9orf72 hexanucleotide repeat expansions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8ebb664b6f1b21244e0cfd38a3721d45d012b3f" target='_blank'>
              Dynamic changes in chromosome and nuclear architecture during maturation of normal and ALS C9orf72 motor neurons
              </a>
            </td>
          <td>
            Özgün Uyan, Snehal Sambare, Marlies E. Oomen, Nicholas Wightman, A. Schooley, Joseph R. Klim, Houda Belaghzal, Özkan Aydemir, Betul Akgol-Oksuz, Zeynep Sena Agim Uslu, Kevin Eggan, Robert H. Brown, Job Dekker
          </td>
          <td>2025-09-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/832132e602fe2a8fad05d9648f87894306e0e923" target='_blank'>
              Lineage-determining transcription factors constrain cohesin to drive multi-enhancer oncogene regulation
              </a>
            </td>
          <td>
            Yeqiao Zhou, Atishay Jay, Noah G Burget, Tobias Friedrich, Sora Yoon, Jessica Alsing, Guy Nir, Rudolf Grosschedl, Golnaz Vahedi, R. Faryabi
          </td>
          <td>2025-10-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70a793be64c8f637d9173f596004a270d178f84a" target='_blank'>
              The transcriptome of highly aneuploid senescent cells reveals interaction with the tumour microenvironment and a pro-survival role of Yorkie
              </a>
            </td>
          <td>
            Kaustuv Ghosh, Aishwarya Kunchur, Marco Milán
          </td>
          <td>2025-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Prior research shows that Akt-dependent phosphorylation of TopBP1 in S phase results in the switch of TopBP1/Treslin binding to TopBP1/E2F1 binding, which is important to prevent replication re-initiation in late S and G2 phases. Here, we demonstrate that contact, but not conformational, mutant p53 can override this switch by binding to both TopBP1 and Treslin, thereby facilitating persistent TopBP1/Treslin interaction in late S and G2 phases, which ultimately leads to over-firing of replication initiation. This increases micronuclei formation, which is further enhanced by genotoxic stressors such as doxorubicin, PARP inhibitors, or ATR inhibitors. Consequently, contact mutant p53 increases the sensitivity of cancer cells to a TopBP1-BRCT7/8 inhibitor combined with PARP or ATR inhibitors. Importantly, contact mutant p53 induces micronuclei formation and MRE11 expression, thereby activating cGAS-STING pathway and enhancing response to immune checkpoint inhibition. This finding is validated in murine mammary tumor allografts and further corroborated by clinical data. This study uncovers how contact mutant p53 drives replication over-firing in late S/G2, activates cGAS-STING, and enhances immune checkpoint inhibitor response, offering new insights into therapeutic outcomes of different mutant p53 variants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47fd482e67931dfe6c2c10b90186756c89216708" target='_blank'>
              Mutant p53 variants differentially impact replication initiation and activate cGAS-STING to affect immune checkpoint inhibition
              </a>
            </td>
          <td>
            Kang Liu, L. A. W. Garan, Fang-Tsyr Lin, Weei-Chin Lin
          </td>
          <td>2025-11-05</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7aadf2778bea4e8d563036eda473ff737cd085e3" target='_blank'>
              Gene-sized DNA insertion at genomic safe harbors in human cells using a site-directed transposase
              </a>
            </td>
          <td>
            James E Short, Lisa Sharek, Joshua F. Meckler, I. Stoytchev, Christopher T Tran, Clara Errard, Brian E Hew, Brandon E Johnson, David F Waller, Shanfu Xie, Joseph J Higgins, Raymond Tabibiazar, Jesse B Owens
          </td>
          <td>2025-09-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8768ee203fcd258706f2498c5fcf4ab4120a2eda" target='_blank'>
              Somatic evolution of prostate cancer: mutation, selection, and epistasis across disease stages
              </a>
            </td>
          <td>
            Moein Rajaei, Alexander Yang, Kira A. Glasmacher, Christopher N. Cross, Nic Fisk, Elizabeth B. Perry, Jeff Mandell, S. Gaffney, Takafumi N. Yamaguchi, J. Livingstone, Jose Costa, Peter Humphrey, Vincent L. Cannataro, P. Boutros, Jeffrey P. Townsend
          </td>
          <td>2025-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Understanding tumor heterogeneity is essential for advancing cancer treatment. Clonal reconstruction methods play a pivotal role in deciphering this heterogeneity. Our goal is to develop a clonal reconstruction approach that is clinically applicable, easy to implement, and capable of delivering both high-speed performance and excellent reconstruction accuracy. We present MyClone, a probabilistic method designed to reconstruct the clonal composition of tumors using deep sequencing genomic data. MyClone processes read counts and copy number information of single nucleotide variants derived from deep sequencing data, enabling it to determine the mutational composition of clones and the cancer cell fractions of these mutations. Compared to existing clonal reconstruction methods, MyClone enhances clustering accuracy and cancer cell fraction prediction when applied to deep-targeted sequencing data and bulk tumor sequencing data with deep sequencing coverage. Additionally, MyClone achieves a substantial improvement in computational speed. We rigorously validated MyClone’s performance using both simulated and real clinical datasets and applied it to analyze a circulating tumor DNA sequencing dataset from 139 metastatic breast cancer patients. In this analysis, we explored mechanisms of drug resistance in metastatic breast cancer and identified 10 mutated genes potentially associated with drug resistance or sensitivity. For deeply sequenced data, MyClone outperforms existing methods on both targeted sequencing and bulk tumor data. Its high computational efficiency and reconstruction accuracy position MyClone as a promising tool for broad clinical application in cancer treatment. The source code is publicly available at https://github.com/Hansen0413/myclone-code.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/008b5a4c97ae8500318c69c4b98363d3ad251446" target='_blank'>
              MyClone: rapid and precise reconstruction of clonal population structures for tumors
              </a>
            </td>
          <td>
            Qihan Guo, Dan Lv, Bo Lan, Haili Qian, Fei Ma, Ting Chen
          </td>
          <td>2025-10-08</td>
          <td>BMC Bioinformatics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 Tunneling nanotubes (TNTs) are dynamic membranous protrusions that are critical for intercellular communication in cancer, yet their transient nature and lack of definitive molecular markers pose challenges for systematic study. To address this issue gap in knowledge, we used bulk and single-cell transcriptomic sequencing approaches to identify differentially expressed genes in ovarian cancer cells enriched for TNTs to identify molecular pathways and genes potentially implicated in TNT formation, maintenance and function. We used edgeR and Seurat to perform differential expression analysis on ovarian cancer cells grown in conditions favoring upregulation of TNT formation and further performed single-cell sequencing of the same cell type isolated via spatiotemporal genomic analysis (SaGA) and fluorescence-activated cell sorting (FACS). We found 771, 32 and 279 genes upregulated in bulk sequenced TNT forming SKOV3, COV362 and ES-2 ovarian carcinoma cell lines, respectively. Gene families that repeatedly came up across all three of the cell lines included carbonic anhydrases, matrix metalloproteinases, SERPIN (Serine Protease Inhibitor-30), and SNORA (Small Nucleolar RNAs). We narrowed down candidate genes of interest to 23 differentially expressed genes (DEGs), many of which are involved in cellular stress response, organization of the extracellular matrix or intracellular cytoskeleton, cell adhesion, and other pathways. Further analyzing the single cell sequenced ES-2 cells, we found the differential expression of KISS1 gene, a metastasis suppressor implicated in cytoskeletal organization, significantly upregulated in TNT-enriched cells, highlighting its potential role in TNT formation. Other genes, such as EPHB1, MMP3, and WNT7B, were also upregulated in TNT-enriched cells, suggesting their involvement in regulating cell adhesion, migration, and differentiation. Conversely, we observed downregulation of BGN, a gene associated with promoting metastatic potential and NFkB signaling in cancer cells, as well as SYNGR1, a gene traditionally linked to synaptic transmission but also involved in vesicle exocytosis, which may influence TNT formation.



 Rojina Sapkota. RNA-seq analysis to uncover drivers of cancer cells forming tunneling nanotubes for intercellular communication [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr B015.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb5389ea72947e09856895a8de975c87a724ed10" target='_blank'>
              Abstract B015: RNA-seq analysis to uncover drivers of cancer cells forming tunneling nanotubes for intercellular communication
              </a>
            </td>
          <td>
            Rojina Sapkota
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cervical cancer (CC) remains among the top causes of death for women worldwide, especially in low-income countries, where screening strategies are less widespread. Treatment strategies are mainly based on DNA-damaging agents, though resistance mechanisms still pose a substantial challenge. Among the cellular components that mediate treatment resistance, long non-coding RNAs (lncRNAs) stand out because of their broad regulatory effects. They are involved in virtually all drug resistance mechanisms, such as drug efflux, DNA repair, evasion of cell death, and aberrant epigenetic modifications. Although resistance mechanisms are fundamentally similar in most cancers, the underlying regulatory networks vary substantially. Here, we review the literature for lncRNAs involved in treatment resistance mechanisms in general, and then focus on lncRNAs that mediate resistance in CC. We found a broad area of opportunity in lncRNA research in resistant CC, as the lncRNAs involved are still to be described. These master regulators are promising candidates for response markers and therapeutic targets. May this compilation serve as the basis for further descriptions of the regulatory roles of lncRNA in CC treatment resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38374134a894f6b5db043d7b741d396bd485cc0d" target='_blank'>
              Long noncoding RNAs involved in therapeutic response: implications for cervical cancer drug resistance
              </a>
            </td>
          <td>
            Samuel Trujano-Camacho, Carlos Contreras-Romero, Verónica García-Castillo, David Sánchez-Marín, Mercedes Olvera-Valencia, M. Rodríguez-Dorantes, O. Peralta-Zaragoza, David Cantú- De León, E. López-Urrutia, C. Pérez-Plasencia
          </td>
          <td>2025-10-21</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a39534227ca18263af64802892a94b5211231b6b" target='_blank'>
              Failure of DNA repair leads to the tumorigenic transformation of a regenerating blastema in Drosophila imaginal discs
              </a>
            </td>
          <td>
            Tanroop Kaur, Ada Repiso, Ying Liu, Paula Doria, Joan Urgell, Luis Rodriguez-Escudero, Katerina Karkali, Yanhui Hu, Norbert Perrimon, E. Martín-Blanco
          </td>
          <td>2025-10-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="


 Ovarian cancer remains the most lethal gynecologic malignancy, with five-year survival rates below 50% largely due to late-stage diagnosis and the development of drug resistance. The advent of precision medicine offers new opportunities to improve patient outcomes. To date, molecular profiling has identified key therapeutic targets in ovarian cancer, including homologous recombination deficiency (HRD), BRCA mutations, and mismatch repair defects. PARP inhibitors have demonstrated significant clinical benefit in patients with HRD-positive tumors. Also, folate receptor alpha (FRa) targeting antibody drug conjugate, mirvetuximab soravtansine has demonstrated improved overall survival in platinum resistant ovarian cancer expressing high levels of FRa. To exploit DNA replication vulnerabilities in ovarian cancer cells, inhibitors of cell cycle regulators ATR and CHK1, have been studied in ovarian cancer preclinical and clinical settings. DHX9 helicase inhibition represents another novel approach, targeting RNA/DNA helicase activity critical for transcriptional regulation and DNA repair processes in cancer cells. Additional emerging targets include KIF18A that regulates mitotic progression and microtubule dynamics, which have demonstrated selective activity in chromosomally instable high-grade serous ovarian cancer cells. Novel therapeutic targets such as DHX9 helicase and KIF18A, offer new opportunities for patients with limited treatment options. Future directions include expanding biomarker testing, developing combination therapeutic strategies that exploit multiple vulnerabilities simultaneously, and addressing resistance mechanisms.



 Jung-Min Lee. Advancing precision therapy through molecular profiling and targeting DNA replication vulnerabilities in ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr IA018.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4c03eea4f891eb2bca142fb8a480f45ccc5bf2a" target='_blank'>
              Abstract IA018: Advancing precision therapy through molecular profiling and targeting DNA replication vulnerabilities in ovarian cancer
              </a>
            </td>
          <td>
            Jung-Min Lee
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract The maintenance of genomic integrity is paramount for normal cell physiology and survival as well as avoidance of carcinogenesis. Cellular DNA is periodically subjected to a myriad of exogenous and endogenous threats and requires constant monitoring to limit genomic instability. To this end, cells possess an intricate DNA damage response and repair (DDR) module comprised of different classes of protein players. The DNA helicases, ATP-dependent enzymes that unwind the DNA double helix, are one such important class of proteins, which act as a linchpin between the recognition and resolution of DNA damage via facilitating various DNA repair processes. Dysfunction or absence of DDR helicase function is implicated in several human disorders including Bloom syndrome, Werner syndrome, Rothmund–Thomson syndrome, and Fanconi anemia. Somatic helicase mutations or dysregulation of helicase function can also contribute to cancer development, progression, and chemotherapy sensitivity, making helicases a promising target class for chemotherapeutic drug intervention. In addition, recent discoveries have identified some DDR helicases in novel synthetic lethal interactions. In this critical review, we will focus on human DNA helicases that are directly or indirectly involved in DDR with special emphasis on their mechanistic actions and clinical implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40deeb85611db34dede498af9b0ad1f2d2a9a398" target='_blank'>
              DNA repair helicases: from mechanistic understanding to therapeutic implications
              </a>
            </td>
          <td>
            Vijay Menon, Susan E. Gueble
          </td>
          <td>2025-10-07</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The TP53 gene is one of the most frequently mutated genes in human cancers. Mutations often result in loss of tumor-suppressive functions and acquisition of oncogenic properties by p53, contributing to tumor progression and resistance to therapy. Among structural p53 mutations, Y220C is particularly notable for creating a surface-exposed hydrophobic pocket that destabilizes the protein while preserving partial function, making it a promising target for pharmacological reactivation. In this study, we performed a structure-guided phenotypic screen of an in-house heterocyclic compound library to identify novel small-molecule modulators of p53-Y220C. This led to the identification of a series of (1H-pyrrol-1-yl)indazole derivatives (JC16, JC36, JC65), structurally inspired by known Y220C binders. JC16 and JC36 exhibited selective cytotoxicity and pro-apoptotic activity in p53-Y220C mutant cancer cell lines, with minimal effects in wild-type or p53-null cells. These compounds induced a mutant-to-wild-type conformational shift in cellular p53-Y220C, accompanied by transcriptional activation of canonical p53 target genes, including BBC3 (PUMA) and MDM2. Western blot analysis revealed that in HUH7 cells, these effects occurred without a corresponding increase in total p53 protein levels, suggesting a mechanism based on conformational reactivation. Our findings position JC16 and JC36 as early-stage chemical leads with potential to restore mutant p53 function in a context-dependent manner. While their exact mechanism of action remains to be fully elucidated, these results provide a foundation for further development of indazole-based scaffolds as reactivators of the p53-Y220C mutant in cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02b0e7288fbc76b6467af63075f3c2aaa9b2814f" target='_blank'>
              Cellular activity upregulation of the thermolabile p53 cancer mutant Y220C by small molecule indazole derivatives
              </a>
            </td>
          <td>
            Raniya Khadiullina, V. Chasov, Elvina Gilyazova, Damir Davletshin, R. Mirgayazova, Rimma N. Mingaleeva, Joseph R Stephenson Clarke, Matthias G J Baud, E. Bulatov
          </td>
          <td>2025-11-07</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Glioblastoma (GBM) is marked by profound spatial heterogeneity. This complexity fuels tumor progression and therapy resistance. Although recent spatial transcriptomics (ST) studies have defined transcriptional niches, or metaprograms, the systems level regulatory logic linking upstream signaling, transcriptional programs, and therapeutic vulnerabilities remains unresolved. Here, we introduce an integrated, multi layered framework that unifies transcription factor and pathway activity inference (STAN and SPAN) with ligand/receptor and drug target mapping, thereby constructing a systems-level regulatory atlas of GBM across 26 tumors. This atlas uncovers spatial niches organized around transcriptional hubs and signaling programs, including hypoxic mesenchymal regions coordinated by HIF1A and SOX2 with mTORC1 signaling, macrophage rich areas driven by STAT3 mediated immune modulation, and astrocyte-like states shaped by lipid metabolic regulators such as SREBF2. By coupling this regulatory atlas to drug target predictions, we identify niche specific therapeutic vulnerabilities, such as VEGF blockade in HIF1A high regions and MAO-B inhibition in mesenchymal states. Finally, we provide an open, interactive web resource to make these data broadly accessible. This work provides a generalizable blueprint for linking spatial regulation to therapeutic hypotheses across diverse human diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a40d513760c094d00d7ea814fe1f34729bf0c35" target='_blank'>
              A Multi-Layered Atlas of Spatial Regulatory Programs and Therapeutic Vulnerabilities in Glioblastoma
              </a>
            </td>
          <td>
            Linan Zhang, Matthew Lu, Xiaojun Ma, H. Osmanbeyoglu
          </td>
          <td>2025-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Acute leukemias are highly aggressive hematologic malignancies that demand intensive chemotherapy regimens. However, drug toxicity remains a major barrier to treatment success and patient survival. In this context, pharmacogenomics offers a promising strategy by identifying single-nucleotide variants (SNVs) that influence drug metabolism, efficacy, and toxicity, ultimately impacting treatment outcomes. This study analyzed data from the ClinPGx/PharmGKB database to identify clinically annotated variants related to chemotherapy response in Acute Myeloid Leukemia (AML) and Acute Lymphoblastic Leukemia (ALL). A total of 24 variants were curated for AML and 57 for ALL. Among these, nonsynonymous variants were most frequent in ALL (31.6%), while synonymous variants predominated in AML (33.3%). Although traditionally considered neutral, synonymous and intronic variants may influence gene expression through regulatory or splicing mechanisms. The analysis revealed clinically significant variants associated with chemotherapy response, particularly in the ABCB1 gene, observed in 12.5% of AML and 10.5% of ALL cases. Several variants, particularly TPMT, NUDT15, ABCC1, SLC28A3, and RARG, were associated with severe adverse effects such as myelotoxicity, mucositis, cardiotoxicity, and hepatotoxicity. This study reinforces the importance of genetic variants in modulating the therapeutic response and toxicity to chemotherapy drugs in acute leukemias. Analysis of ClinPGx/PharmGKB data emphasizes ABCB1 as a potential resistance marker and supports pre-treatment genotyping of genes like TPMT and NUDT15 to prevent severe toxicities. Future advances should include the expansion of pharmacogenetic studies in underrepresented populations and the clinical validation of new markers in prospective trials, aiming to consolidate precision medicine as a routine part of the therapeutic management of acute leukemias.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41d398f2f78483d0f180028688c26b519950cb78" target='_blank'>
              Pharmacogenomics Applied to Acute Leukemias: Identifying Clinically Relevant Genetic Variants
              </a>
            </td>
          <td>
            Flávia Melo Cunha de Pinho Pessoa, Isabelle Magalhães Farias, B. Nogueira, C. B. Machado, Igor Valentim Barreto, Anna Karolyna da Costa Machado, Guilherme Passos de Morais, Leidivan Sousa da Cunha, Deivide de Sousa Oliveira, André Pontes Thé, Rodrigo Monteiro Ribeiro, Patrícia Maria Pontes Thé, Manoel Odorico de Moraes Filho, Maria Elisabete Amaral de Moraes, Caroline Aquino Moreira-Nunes
          </td>
          <td>2025-10-22</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The DREAM complex has emerged as a central repressor of DNA repair, raising questions as to whether such repression exerts long-term effects on human health. Here we establish that DREAM activity significantly impacts lifetime somatic mutation burden, and that such effects are linked to altered lifespan and age-related disease pathology. First, joint profiling of DREAM activity and somatic mutations across a single-cell atlas of 21 mouse tissues shows that cellular niches with lower DREAM activity have decreased mutation rates. Second, DREAM activity predicts the varied lifespans observed across 92 mammals, with low activity marking longer-lived species. Third, reduced DREAM activity in Alzheimer’s patients predicts late disease onset and decreased risk for severe neuropathology. Finally, we show DREAM knockout protects against mutation accumulation in vivo, reducing single-base substitutions by 4.2% and insertion/deletions by 19.6% in brains of mice. These findings position DREAM as a key regulator of aging.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ead31f56ffdb72707b88fc850bc30a7d800daa3" target='_blank'>
              The DREAM complex links somatic mutation, lifespan, and disease
              </a>
            </td>
          <td>
            Zane Koch, Shuvro P. Nandi, Kate Licon, Arturo Bujarrabal-Dueso, David H. Meyer, Safa Saeed, Pirunthan Perampalam, Frederick A. Dick, B. Schumacher, Ludmil B. Alexandrov, T. Ideker
          </td>
          <td>2025-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="Abstract Background/Aim: Cancer remains a leading cause of mortality globally, driven by complex molecular mechanisms and characterized by significant biological heterogeneity across cancer types. We aimed to discover mitogen-activated protein kinases (MAPKs) family members as both biomarkers and therapeutic targets in different cancer types. MAPKs are key signaling molecules regulating cell proliferation, differentiation, stress response, and apoptosis. Dysregulation of MAPK pathways has been implicated in the onset and progression of multiple cancers, contributing to tumor growth, metastasis, and therapeutic resistance. Given their diverse roles across cancer types, systematic analysis of MAPK gene expression, mutations, and interactions with tumor microenvironment is essential. Materials and Methods: The present study undertakes a comprehensive transcriptomic analysis of MAPKs in nine major cancer types using RNA-Seq datasets from The Cancer Genome Atlas (TCGA). RNA-Seq data were analyzed to identify differentially expressed MAPKs across nine cancer types using DESeq2, NOIseq, and limma. Significant genes (adjusted p<0.05) were subjected to GO and KEGG enrichment (EnrichR), mutational profiling (cBioPortal), and Kaplan-Meier survival analysis. Tumor immune infiltration was further assessed using TIMER2.0 to explore immune–gene interactions. Results: Through rigorous differential expression analysis, we identified key MAPKs that are significantly altered in liver hepatocellular carcinoma (LIHC) and lung adenocarcinoma (LUAD). Specifically, six MAPKs (MAPK3, MAPK7, MAPK9, MAPK10, MAPK12, and MAPK13) were found to be differentially expressed in LIHC, while MAPK6 emerged as the sole significant candidate in LUAD. Functional enrichment and pathway analysis revealed that these MAPKs are involved in critical oncogenic pathways, including MAPK-mediated transcriptional regulation and stress-activated signaling cascades. Mutational profiling and survival analysis further validated the prognostic significance of these genes, with several MAPKs showing strong associations with reduced patient survival. Tumor immune infiltration analysis indicated potential roles of these MAPKs in modulating immune responses within the tumor microenvironment. Also, MAPKs identified in this study are structurally related, suggesting that targeting them collectively may enhance therapeutic efficacy and overcome resistance mechanisms. Conclusion: Our integrated approach underscores the value of MAPK family members as both biomarkers and therapeutic targets in LIHC and LUAD. This study contributes important insights into MAPK-related oncogenic processes and supports the development of targeted therapies under the framework of precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19b14065bbb6005ef268d03b15d61fb336481cae" target='_blank'>
              Investigating the Roles of MAPKs as Precision Oncology Targets: A Multi-cancer Expression and Survival Study
              </a>
            </td>
          <td>
            Md Nayab Sulaimani, Prithvi Singh, Mohammad Umar Saeed, Sobia Zaidi, Afzal Hussain, M. Alajmi, Anas Shamsi, Ravins Dohare, M. Hassan
          </td>
          <td>2025-10-28</td>
          <td>Cancer Genomics & Proteomics</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Topoisomerase 1 (Top1) removes transcription-related helical torsions and thus plays an important role in preventing genome instability instigated by the formation of non-canonical DNA secondary structures. The genetically tractable Saccharomyces cerevisiae model proved effective in defining the critical function of Top1 to prevent recombination and chromosomal rearrangement at G4-forming genomic loci and studying the human cancer-associated Top1 mutants through the expression of analogous yeast mutants. We previously showed that cleavage-defective Top1 mutants strongly elevate the rate of recombination at G4 DNA, which involves binding to G4 DNA and interaction with the protein nucleolin (Nsr1 in yeast). Here, we further explored the mechanism of genome instability induced by the yeast Top1Y740* mutant, analogous to the human Top1W765Stop mutant conferring resistance to CPT. We show that yTop1Y740* elevates duplications as well as recombination specifically at G4-forming sequences. Interestingly, SUMOylation of yTop1Y740*, which does not affect the G4 DNA-binding or Nsr1-interaction by this mutant, is necessary for such elevated G4-specific genome instability. Many tumors with mutations at the C-terminal residues of Top1, particularly W765, have significantly high G4-associated mutations, underscoring the importance of further investigation into how SUMOylation affects the function of these Top1 mutants at G4 DNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/414ccc1f67726dc580ae95a50a7d8e620a487775" target='_blank'>
              Mutations and Recombination at G4 DNA-Forming Sequences Exacerbated by CPT-Resistant Mutant Topoisomerase 1 Is Dependent on SUMOylation
              </a>
            </td>
          <td>
            Shivani Singh, Xinji Zhu, Nayun Kim
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="BACKGROUND
Diffuse midline gliomas (DMGs), particularly diffuse intrinsic pontine gliomas (DIPGs), are aggressive pediatric brain tumors with a median survival of less than 12 months. Notably, approximately 70% of these tumors harbor P53 pathway alterations, including TP53 or PPM1D mutations. Identifying precision therapies for this subset is crucial. This study aims to employ transcriptomics-guided high-throughput drug screening to identify effective treatments for DIPG/DMG with P53 pathway alterations.


METHODS
Transcriptomic profiling of 98 patient samples containing 31 DIPGs revealed key activated pathways. Patient-derived cell lines were subjected to high-throughput screening using a cell cycle-targeting drug library. Lead candidates were validated both in vitro and in orthotopic xenograft models, while combination therapies were assessed for their ability to overcome TP53-mutant resistance.


RESULTS
Transcriptomic analysis revealed activation of P53 and cell cycle pathways in DIPGs. High-throughput drug screening identified SN-38, a topoisomerase I inhibitor, as selectively targeting TP53 wild-type tumor cells by inducing G2 arrest and apoptosis. TP53 knockdown abolished SN-38's efficacy, while PPM1D knockdown enhanced sensitivity, confirming a TP53-dependent mechanism. Conversely, TP53-mutated cells exhibited SN-38 resistance via ATR pathway activation. Combining SN-38 with the ATR inhibitor AZ20 restored apoptosis and suppressed TP53-mutant tumor growth in vitro and in vivo.


CONCLUSIONS
Guided by transcriptomic profiling, this study utilized high-throughput drug screening to identify SN-38 as a potential therapy for TP53 wild-type DIPG/DMG, while the SN-38 and AZ20 combination was effective against TP53-mutant tumors. This approach provides a promising treatment strategy for this malignancy and establishes a novel paradigm for drug screening in DIPG/DMG.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a7ee270c7115fa9b3b5a7437784e7892bea4467" target='_blank'>
              Transcriptomics-Guided High-Throughput Drug Screening Identifies Potent Therapies for P53 Pathway Alterated DIPG/DMG.
              </a>
            </td>
          <td>
            Zhuan-Xin Jiang, Luyang Xie, Hang Zhou, Yibo Geng, Xiong Xiao, Tian Li, Yuxuan Deng, Mingxin Zhang, Shaobo Shan, Cheng Xu, Liwei Zhang
          </td>
          <td>2025-09-17</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Oral squamous cell carcinoma (OSCC) is a major health burden in most parts of the world, and pathogenesis of the disease is strongly associated with a complex combination between genetic mutations and epigenetic alterations. The current study will explore the DNA methylation phenomenon and how it affects the silencing of tumour suppressor genes and activate oncogenes in OSCC in a bid to explain the driving molecular process behind tumour development. We evaluated 50 samples of the OSCC tissues with 50 of the adjacent normal tissues, evaluated DNA methylation patterns with methylation-specific PCR and quantitative reverse-transcriptase PCR measured the expression of the DNA repair genes like BRCA1 and MLH1, among others. This finding showed different levels of DNA methylation of cancerous and normal tissues with hypermethylation causing the inactivation of important tumor suppressor genes and hypomethylation causing the activation of oncogenes. In addition, the downregulation of DNA repair genes was noted to be highly significant in OSCC samples indicating that genomic-instability may be related to epigenetic changes. These results demonstrate that aberrant DNA methylation is central to OSCC growth and progression, thus helping in the future use of methylation patterns that can be used as early detection, diagnosis, and prognostic biomarkers. Our findings support the idea that genetic, along with epigenetic, profiling is an issue of key importance toward comprehending OSCC biology and personalized therapeutic interventions. It is recommended that further confirmation be carried out to verify the clinical significance of such epigenetic markers in bigger cohorts, as well as testing such markers in guided therapies, which might eventually lead to better patient outcomes in the management of oral cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50fd5e50c6888572f5cfc48d30ee386841f2ea57" target='_blank'>
              Epigenetic and genetic events of oral squamous cell carcinoma: perspective on DNA methylation, silencing of tumor suppressor gene, and activating oncogenes.
              </a>
            </td>
          <td>
            Zainab Nizar Jawad
          </td>
          <td>2025-10-07</td>
          <td>Cellular and molecular biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1532e1a41f5f79814eb3fb790e35c92b5a006139" target='_blank'>
              The Androgen Receptor and MYC synergise to modulate the synthesis of Siglec-7 ligands in prostate cancer
              </a>
            </td>
          <td>
            Adam Duxfield, Rebecca Garnham, Esme Hutton, Adam A. Dowle, Jamie Wills, Sara Luzzi, Ryan Nelson, Emma Lishman-Walker, Rianna Magee, A. Buskin, A. Hepburn, Emirhan Tekoglu, Kaidan Maloney-Friar, Fiona Frame, Norman Maitland, Ann Hedley, Holly Henderson, Benjamin McCullough, Bharat Gowardhan, K. Sahadevan, S. McCracken, L. Gaughan, Rakesh Heer, Craig N. Robson, Kelly Coffey, Nathan Lack, Nathalie Signoret, M. Fascione, Emma Scott
          </td>
          <td>2025-10-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="


 RAS inhibitors (RASi) have the potential to change the clinical landscape for pancreatic ductal adenocarcinoma (PDAC) and improve patient survival. Nevertheless, clinical resistance is likely to reduce the long-term benefits of RASi, and combinations that can bypass or overcome resistance will be needed. Here, we studied the dynamics of resistance to the RAS multi(ON) inhibitor RMC-6236 (daraxonrasib). Prior to frank drug resistance, PDAC cells persist in a drug tolerant state characterized by reduced proliferation; however, only a fraction of the drug-tolerant persister cells contributes to resistance. To identify and study the relevant, resistance-contributing cells, we developed a highly multiplexed, lentiviral DNA barcode system that is compatible with single-cell technologies, enabling paired lineage tracing and phenotypic profiling throughout RASi treatment. Our lineage tracing studies suggest that drug persistence and resistance are fueled by a subset of cells that are primed to survive RASi treatment. Ongoing work is directed at defining and tracking the cell states associated with each stage of resistance emergence.



 Jayne C. McDevitt, Connor J. Hennessey, Robert Vander Velde, Sydney M. Shaffer, Ben Z. Stanger. Clonal dynamics and molecular mechanisms mediating resistance to RAS inhibition [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B082.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95c1bd513ed6b1024a532086d606e43f01e09bb7" target='_blank'>
              Abstract B082: Clonal dynamics and molecular mechanisms mediating resistance to RAS inhibition
              </a>
            </td>
          <td>
            Jayne C McDevitt, Connor J Hennessey, R. V. Velde, Sydney M. Shaffer, Ben Z. Stanger
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Chromosomal rearrangements that generate novel fusion genes are a hallmark of acute myeloid leukemia (AML). Depletion experiments in cell line models have suggested that their continued expression is required for maintaining their leukemic phenotype and that fusion genes therefore represent ideal cancer-specific therapeutic targets. However, to which extent this result holds true for the different stages of hematopoietic development in primary cells and whether therapeutic agents can be efficiently delivered to those cells is still unclear. In this study, we demonstrate that primary AML cells harboring the chromosomal translocation t(8;21) are critically dependent on the corresponding fusion gene, RUNX1::RUNX1T1, to suppress differentiation and maintain stemness. Silencing RUNX1::RUNX1T1 expression using siRNA-loaded lipid nanoparticles induces substantial changes in chromatin accessibility, thereby redirecting the leukemia-associated transcriptional network towards a myeloid differentiation program. Single-cell analyses reveal that this transcriptional reprogramming is associated with the depletion of immature stem and progenitor-like cell populations, accompanied by an expansion of granulocytic and eosinophilic/mast cell-like populations with impaired self-renewal capacity. These findings underscore the essential role of RUNX1::RUNX1T1 in sustaining AML and highlight the therapeutic potential of targeting fusion gene expression in primary AML cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5469845f04d542fe9db7d0ec4a2180ccb6b7c697" target='_blank'>
              Fusion Gene Depletion Eliminates Stemness and Induces Bidirectional Differentiation of Acute Myeloid Leukemia.
              </a>
            </td>
          <td>
            Polina K Derevyanko, L. Swart, L. D. Mata Casimiro, Anita van Oort, M. Du Plessis, Luca van den Brink, Minoo Ashtiani, C. Zwaan, A. Krippner-Heidenreich, Constanze Bonifer, Raymond Schiffelers, Josef Vormoor, S. Kellaway, O. Heidenreich
          </td>
          <td>2025-09-24</td>
          <td>Blood</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b9752e134e3e7566b741e6bb0305e92d0896d6a" target='_blank'>
              Timing the onset of homologous recombination deficiency before cancer diagnosis
              </a>
            </td>
          <td>
            Michail Andreopoulos, Muchun Niu, Yang Zhang, V. Viswanadham, D. Gulhan, Hu Jin, Felipe Batalini, Gerburg Wulf, Chenghang Zong, Peter J. Park, Dominik Glodzik
          </td>
          <td>2025-10-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="A population of cells within a tumor can be described as antifragile (the opposite of fragile) if they derive a benefit from fluctuations or perturbations in environmental conditions induced by treatment. We hypothesized that treatment fluctuations may either promote (antifragile tumor) or inhibit (fragile tumor) the evolution of treatment resistance to targeted therapies. Analysis of the convexity of dose response curves provides a direct prediction of response to prescribed fluctuations in treatment. Theory predicts that continuous treatment protocols (i.e. zero prescribed fluctuations in treatment) will outperform uneven protocols when dose response is convex. We apply this theory to predict in vivo response to targeted therapy and validate these predictions by experimentally testing high/low intermittent dosing (uneven dosing) and continuous dosing (even dosing) schedules. Convexity (and its inverse, concavity) explains two phenomena: dose response is a convex (fragile) function but resistance onset rate is a concave (antifragile) function. Thus, we design, propose, and validate alternative treatment schedules which maximize response while maintaining prolonged sensitivity to treatment. These analyses provide supporting evidence that fluctuations alter the evolutionary trajectory of tumors in response to targeted therapy, even without altering the cumulative dose. This insight is used to design alternative, switching treatment protocols to limit the evolution of resistance, a process which we call evolutionary antifragile therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce8cf7a63052092f4dae35998f972484b69a62e3" target='_blank'>
              Evolutionary antifragile therapy
              </a>
            </td>
          <td>
            J. West, Bina Desai, M. Strobl, J. Gallaher, M. Robertson-Tessi, A. Marusyk, Alexander R. A. Anderson
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/090419ca37781f8fabe58e144378760a6083ab49" target='_blank'>
              Rapid clonal selection within early hematopoietic cell compartments presages outcome to ivosidenib combination therapy.
              </a>
            </td>
          <td>
            Sven Turkalj, Felix A. Radtke, B. Stoilova, R. Mecklenbrauck, Angus J Groom, N. A. Jakobsen, Curtis A. Lachowiez, M. Metzner, Batchimeg Usukhbayar, Mirian Angulo Salazar, Z. Zeng, S. Loghavi, Jennifer Marvin-Peek, Verena Körber, F. Ravandi, G. Issa, T. Kadia, Vasiliki Symeonidou, Anne P. de Groot, H. Kantarjian, Koichi Takahashi, Marina Y. Konopleva, C. DiNardo, Paresh Vyas
          </td>
          <td>2025-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>230</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c00ef580d1c2e048fccb2889a7c92151af7fd5d7" target='_blank'>
              Probability Distribution for Rare Neutral Mutations in Cancers and Application to Dynamic Precision Medicine of Cancer
              </a>
            </td>
          <td>
            Wei He, Matthew D. McCoy, Chen-Hsiang Yeang, Rebecca B. Riggins, Robert A. Beckman
          </td>
          <td>2025-10-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Chemoresistance is a major obstacle in cancer treatment, and it often results in treatment failure and disease progression. Among the plethora of factors contributing to chemoresistance, the transcription factor Krüppel-like factor 4 (KLF4) has emerged as a pivotal player. This review discusses the role of KLF4 in orchestrating various mechanisms underlying cancer chemoresistance. KLF4, originally identified as a regulator of cell differentiation and proliferation, has recently gained attention for its role in modulating cellular responses to chemotherapeutic agents. Through complex regulatory networks, KLF4 modulates the process of drug efflux, DNA repair, apoptotic signaling, tumor heterogeneity, and cancer cell stemness, leading to the development of cancer chemoresistance. Additionally, tissue or cell types specific post-translational modification (PTM) of KLF4 plays a significant role in the development of cancer chemoresistance. The review explores emerging possibilities and available information that can be utilised to understand the mechanism of chemoresistance mediated by KLF4 in cancer. In conclusion, understanding the complex mechanisms through which KLF4 orchestrates cancer chemoresistance opens promising avenues for developing more effective therapeutic interventions to combat treatment-resistant cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26e91a9cb65a0f14dcdac5b83f903eda917dba7f" target='_blank'>
              KLF4 in cancer chemoresistance: molecular mechanisms and therapeutic implications
              </a>
            </td>
          <td>
            Suresh Singh Yadav, Punita Kalia, Navneet Kaur, Sourav Sharma, Shambhavi Thakur, Soni Kumari, R. R. Nair
          </td>
          <td>2025-09-22</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de278a01c7636edead18a0789115129a52c7207e" target='_blank'>
              Centromere instability links genome damage to immune activation in systemic sclerosis
              </a>
            </td>
          <td>
            Azait Imtiaz, Mohammad Waseem, Hudson O’Neill, Bo-Ruei Chen, Wioletta Czaja, Rafael Contreras-Galindo
          </td>
          <td>2025-10-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Cancer is a deadly disease affecting millions of people worldwide. Circulating tumor cells (CTCs) represent a critical link between primary malignancies and metastasis, acting as key players in cancer dissemination, progression, and recurrence. Although rare, CTCs offer a valuable, non-invasive window into tumor biology and the evolution of disease in patients. CTCs can exist as single cells in the circulation, but some are shed and travel in larger groups, referred to as CTC clusters. These clusters possess a greater oncogenic potential compared to individual CTCs. In this review, we aim to provide insight into the dynamic biological processes underlying CTC generation, biology, and survival, with a focus on epithelial-to-mesenchymal transition (EMT) and beyond like cancer stem cells (CSCs), cellular plasticity, and senescence. A crucial aspect of CTC biology is EMT, a process that imparts cancer cells with increased motility, invasiveness, resistance to apoptosis, and the ability to intravasate and evade the immune system. Beyond EMT the cancer cells show further plasticity, allowing epithelial tumor cells to adopt mesenchymal or hybrid phenotypes, which enables adaptation to a changing microenvironment and enhances therapy resistance. Moreover, a subset of cancer cells can acquire stem cell-like properties, including self-renewal and tumor-initiating capacity. EMT, along with processes such as dedifferentiation, contributes to the generation of cancer stem cells. In recent years, studies have also highlighted the complex and paradoxical role of senescence in CTC biology. While senescence typically results in permanent cell cycle arrest, in cancer cells it may be reversible and can promote tumor cell dormancy, immune evasion, and metastatic reactivation. By exploring the connections between CTCs, EMT, CSCs, plasticity, and senescence, we aim to shed light on the unique biology of CTCs, their metastatic potential, and their contributions to tumor heterogeneity. We hope that a better understanding of these processes will help advance the development of novel biomarkers and therapeutic targets for solid tumors beyond EMT.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a99a639294b1ce88e2f796dfc138b9bd8dbb8609" target='_blank'>
              Circulating tumor cells: indicators of cancer progression, plasticity and utility for therapies
              </a>
            </td>
          <td>
            Tamás Richárd Linkner, Z. Nagy, Alexandra Kalmár, Eszter Farkas, Fruzsina Bányai, Nikolett Szakállas, István Takács, Béla Molnár
          </td>
          <td>2025-10-20</td>
          <td>Pathology and Oncology Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) are a key constituent of the tumor microenvironment. CAFs may affect the development of tumor cells. The critical role of CAFs in the tumor microenvironment is linked to their epigenetic modifications, as a stable yet reversible regulation of cellular phenotypes. Current evidence indicates that their formation and function are closely linked to epigenetic mechanisms. Existing research indicates that the epigenetic alteration abnormalities are triggered by metabolic cues and stabilize the acquired phenotype of CAFs. This process is associated with transcriptional changes and patient outcomes in various tumors, providing a biological rationale and translational potential for reprogramming CAFs. Understanding of epigenetic modifications in CAFs remain insufficient, while DNA methylation in CAFs can alter CAF states through multiple pathways and thereby influence tumor progression. It is necessary to investigate the unique, identifiable epigenetic signatures of CAF. As an epigenetic reader couple histone acetylation to high-output oncogenic transcription; meanwhile, noncoding RNAs modulate CAF formation and therapeutic responses via bidirectional crosstalk between tumor cells and stroma. The interactions between different epigenetic modifications and their underlying regulatory logic may play a crucial role in developing new therapeutic strategies. This review focuses on the roles of DNA methylation, histone acetylation, and enhancer reprogramming in CAFs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e18a6e7e1cda4729b61b9135d92b6bc6bb89be7e" target='_blank'>
              Significance of Epigenetic Alteration in Cancer-Associated Fibroblasts on the Development of Carcinoma
              </a>
            </td>
          <td>
            Hongdong Gao, Hinano Nishikubo, Dongheng Ma, Juncheng Pan, Tomoya Sano, Daiki Imanishi, Takashi Sakuma, Canfeng Fan, Masakazu Yashiro
          </td>
          <td>2025-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Pancreatic adenocarcinoma (PAAD) remains one of the most lethal malignancies, primarily due to its robust resistance to genotoxic therapies, such as chemotherapy and radiotherapy. Understanding the mechanisms underlying this resistance is essential to improve clinical outcomes. Here, we identified trimethylguanosine synthase 1 (TGS1), previously known for its role in RNA modification, as a critical mediator of homologous recombination (HR) repair that specifically contributes to resistance in PAAD. TGS1 was significantly overexpressed in PAAD tissues, correlating strongly with advanced disease stages, therapy resistance, and poor patient prognosis. Following DNA damage, ATM kinase phosphorylated TGS1 at serine residues S389 and S531, which mediated its direct interaction with BRCA1 and subsequent recruitment of BRCA1 to DNA damage sites. The phosphorylation-dependent interaction enhanced HR repair efficiency, enabling cancer cells to survive genotoxic stress. Depletion or pharmacological inhibition of TGS1 induced HR deficiency and markedly enhanced sensitivity to DNA-damaging agents, particularly PARP inhibitors, in PAAD cell lines in vitro and in both cell line-derived and patient-derived xenograft models in vivo. Collectively, these findings uncover an ATM-TGS1-BRCA1 signaling axis that promotes DNA repair and resistance to genotoxic therapies in pancreatic cancer, positioning TGS1 as a promising predictive biomarker and therapeutic target to enhance treatment efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/abe0d36d758071ca95a0b7efd96da1ea51c63685" target='_blank'>
              Targeting the ATM-TGS1-BRCA1 Axis Overcomes Genotoxic Therapy Resistance in Pancreatic Adenocarcinoma.
              </a>
            </td>
          <td>
            Changying Li, Xuran Zhao, Xinyan Li, Chunyan Wang, Zhenyu Huo, Xin Xu, Wenjuan Kang, S. Nowsheen, Khaled Aziz, Guangyi Sun, Zhihua Liu, Zhenkun Lou, Min Deng
          </td>
          <td>2025-11-03</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9bb8a507427a3e0cfe9ea50f66dabfc18ac940f2" target='_blank'>
              Cancer Metastasis: Therapeutic Challenges and Opportunities
              </a>
            </td>
          <td>
            Sri Sathya Sandilya Garemilla, Manisha Choudhary Kadambala, S. C. Gampa, Swetha Chinthala, S. Garimella
          </td>
          <td>2025-10-16</td>
          <td>Medical Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is the second leading cause of cancer death worldwide, largely due to the limited efficacy of current therapies in advanced stages of the disease. Most cases of HCC develop in the setting of chronic liver disease, particularly cirrhosis, where ongoing cycles of inflammation, hepatocyte death and regeneration foster the gradual accumulation of genetic and epigenetic alterations that promote malignant transformation. These molecular changes contribute to the high degree of tumour heterogeneity observed in HCC, a major factor underlying resistance to current treatments. As a result, sustained clinical responses to existing therapies, such as tyrosine kinase inhibitors, anti-angiogenic agents and immune checkpoint inhibitors, remain uncommon. In this context, a growing body of evidence has identified epigenetic dysregulation as a key driver of tumour progression and therapeutic resistance, highlighting a new frontier for intervention. This review provides clinicians and researchers with a comprehensive overview of the emerging field of epigenetic therapies in HCC, summarising results from both completed and ongoing clinical trials involving the so-called ‘epidrugs’. Importantly, we discuss how targeting epigenetic mechanisms may not only suppress tumour growth but also enhance the effectiveness of current therapies by reversing resistance pathways. By translating complex molecular insights into tangible therapeutic strategies, epigenetics is poised to reshape the future of HCC management, offering renewed hope for more durable and personalised treatment responses in a disease where progress is urgently needed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ca914bc3e6f4b196e21231ee5479dd3a9a6ded8" target='_blank'>
              Epigenetic therapies in hepatocellular carcinoma: emerging clinical tools and applications
              </a>
            </td>
          <td>
            Barbara Bueloni, Maite G Fernandez-Barrena, Esteban J. Fiore, Matías A. Avila, J. Bayo, Guillermo Mazzolini
          </td>
          <td>2025-09-23</td>
          <td>Gut</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="R-loops, three-stranded nucleic acid structures composed of an RNA:DNA hybrid and displaced single-stranded DNA, have emerged as important regulators of gene expression and genome maintenance. Although physiological R-loops participate in normal cellular processes, their dysregulation can threaten genomic integrity by inducing DNA damage and replication stress. The present review explores the role of R-loops in hepatocellular carcinoma (HCC), a malignancy characterized by marked genomic instability. In the present review, the formation mechanisms of R-loops, their dual functions in transcriptional regulation and DNA damage, and their specific implications for HCC pathophysiology were discussed. HCC cells exhibit altered R-loop homeostasis with aberrant accumulation linked to hepatitis B virus infection, inflammatory signaling and oncogene activation. The present review highlighted how HCC cells exploit or manage R-loops to promote tumor progression, particularly through the epigenetic silencing of differentiation genes and modulation of replication stress responses. Furthermore, emerging therapeutic strategies targeting R-loop biology were examined, including small molecules that induce synthetic lethality, gene-based interventions and combination approaches that exploit R-loop vulnerabilities. Challenges in targeting R-loops and future directions, including multi-omics profiling and biomarker development, were also addressed. Understanding the complex interplay between R-loops and HCC offers promising avenues for novel diagnostic and therapeutic approaches for this malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79c0998da0d4d0d06b4f6b914b08658a256dabc4" target='_blank'>
              R-loops in hepatocellular carcinoma: Bridging genomic instability and therapeutic opportunity (Review)
              </a>
            </td>
          <td>
            Ching-Hua Hsieh, Yueh-Wei Liu, Pei-Chin Chuang
          </td>
          <td>2025-10-15</td>
          <td>Molecular Medicine Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85d93739961372e0101853c9916cb4256544b480" target='_blank'>
              SenLect: a genetically encoded system to purify senescent cells
              </a>
            </td>
          <td>
            Hourieh Tousian, Rachael A Protzman, T. Sargeant, Julian M. Carosi
          </td>
          <td>2025-09-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Chemotherapy is a foundational element in cancer therapy; however, the development of resistance to antineoplastic agents presents a significant challenge to achieving enduring therapeutic triumphs. The acquisition of drug resistance by malignant cells is the result of a complex interplay between inherent and acquired mechanisms. This phenomenon leads to a reduction in pharmacological effectiveness and contributes to treatment failure. This review investigates the primary mechanisms contributing to chemoresistance in specific malignancies, namely lung, pancreatic, and thyroid cancers, along with non-Hodgkin lymphoma. The primary focus of this research is on genetic modifications, interactions within the tumor microenvironment, and cellular adaptations at the molecular level. The present study places particular emphasis on the contributions of drug efflux pumps, DNA repair pathways, epithelial-mesenchymal transition (EMT), microRNAs, and the suppression of apoptosis in mediating this resistance. In addition, we examine promising approaches to counteract drug resistance, encompassing multi-agent regimens, inhibitors targeting specific molecular pathways, and innovative therapeutic modalities. A comprehensive understanding of these fundamental mechanisms is imperative for the development of sophisticated therapeutic interventions that not only improve patient outcomes but also effectively address resistance. The resistance pathways and prospective translational approaches discussed are drawn from both preclinical and clinical investigations, providing a thorough perspective. Keywords: The following terms are relevant to the study: chemoresistance, tumor microenvironment, molecular pathways, targeted therapy, and resistance mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ec567f4501baf205432eef42dce8eb9f17d9a8c" target='_blank'>
              Mechanisms of Chemoresistance in Solid and Hematologic Malignancies: Challenges and Future Perspectives
              </a>
            </td>
          <td>
            Badiaa Batlamous, Boutaina Elgharbaoui, Imane Bensalim, Mohamed Khalis
          </td>
          <td>2025-09-24</td>
          <td>Middle Eastern Cancer and Oncology Journal</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Hypoxia is a recognized driver of transcriptional heterogeneity in high-grade glioblastoma (GBM), yet its functional significance in low-grade gliomas (LGGs) remains poorly understood. We hypothesized that hypoxia orchestrates a reversible quiescence-activation axis in LGGs via EGFR-dependent transcriptional reprogramming, thereby influencing tumor progression.



 We conducted integrative analyses of single-cell RNA sequencing (n=7 patients) and spatial transcriptomics (n=8 samples) across molecularly defined IDH-mutant LGGs to delineate hypoxia-associated transcriptional states. EGFR mutation, and clinical correlations with overall survival, were assessed in The Cancer Genome Atlas (TCGA) bulk RNA-seq cohorts. To functionally validate microenvironmental contributions, we established innovative co-culture models of LGG cells with hypoxia-conditioned primary microglia and astrocytes.



 We identified a hypoxia-associated, LGG-specific ribosomal-enriched (RE) transcriptional program characterizing a non-cycling subpopulation enriched in quiescence and activation stages. Quiescent RE cells exhibited glioma stem cell (GSC)-like features, with EGFR signaling dependency for transition to an activated phenotype. Spatial transcriptomics revealed distinct co-localization of hypoxia, RE program expression, and quiescent states, whereas EGFR-activated cells formed discrete micro-niches. Hypoxia-conditioned microglia significantly enhanced RE program activation and EGFR signaling, establishing a functional microenvironmental driver of phenotypic plasticity.



 Our findings delineate a novel hypoxia-EGFR signaling axis sustaining a quiescent yet activation-capable subpopulation in IDH-mutant LGGs. These insights nominate hypoxia-modulated EGFR pathways as potential therapeutic targets to disrupt tumor cell plasticity. Future studies will interrogate vulnerabilities of RE-programmed LGG cells under hypoxic conditions to inform next-generation therapeutic development.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/748507ebfe3412d3babe3bb777f9dae226192bc1" target='_blank'>
              P02.22.A HYPOXIA-DRIVEN EGFR SIGNALING PROMOTES A QUIESCENT-ACTIVATION AXIS IN IDH-MUTANT LOW-GRADE GLIOMAS
              </a>
            </td>
          <td>
            D. Das, V. Jariwala, A. Pandele, A. Mohammed, T. Das, N. Ghariwala, C. Krolla, S. Arosh, N. Hossain, I. Hosen, S. Smith, J. Beck, J. Grauvogel, I. Vasilikos, R. Roelz, R. Rahman, M. Börries, K. Joseph, V. Ravi, G. Andrieux, S. Chakraborty
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Complete or partial deletions of chromosome 7 (-7/del7q) represent the most frequent chromosomal abnormalities observed in myeloid neoplasms (MNs) and are associated with a poor prognosis. -7/del7q is observed in 10–15% of adult patients with myelodysplasia (MDS) or with acute myeloid leukemia (AML). The occurrence of -7/del7q is particularly frequent in pediatric MDS, often associated with germline mutations of GATA2 or SAMD9/SAMD9L genes. The disease biology of -7/del7q and the genes driving leukemic development have not been completely elucidated, but the haploinsufficiency of tumor suppressor genes located in chromosome 7 deleted regions seems to play a relevant role. The response to standard treatments based either on chemotherapy or hypomethylating agents plus Venetoclax is limited. No approved targeted therapies exist for patients with -7/del7q; however, some recent studies have discovered some vulnerabilities of these myeloid neoplasms than can be efficiently targeted.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a97a077b3eb4226178acf43f03df4eb2019c8ffd" target='_blank'>
              Molecular Complexity of MDS and AML with Aberrations of Chromosome 7
              </a>
            </td>
          <td>
            Ugo Testa, E. Pelosi, G. Castelli
          </td>
          <td>2025-09-18</td>
          <td>Hemato</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Identification of tumor driver genes in cancer has led to the discovery of specific allelic variants that are useful as predictive or prognostic markers. In colorectal cancer (CRC), the K-Ras GTPase is one of the most mutated oncoproteins; therefore, understanding the functional consequences of its mutations will allow us to exploit allele-specific vulnerabilities and propose potential mechanisms of resistance in cancer therapeutics. In this study, cellular phenotypic consequences of the K-Ras mutants K117I, K117N, A146T, E168K, and K172T identified in Filipino young-onset CRC (YO-CRC) patients were assessed in HCT116, Caco-2, and NIH3T3 cells. In assays assessing DNA doubling in 2D cultures and viability in 3D multicellular spheroids, only K117I, K117N, and A146T exhibited enhanced proliferation and supported a tumor-like property of increased size and viability. All mutants exhibited increased cellular migration in HCT116 and NIH3T3 but not in Caco-2 cells. None of them caused resistance to apoptosis. Compared to wild type, K117I, K117N, and A146T increased Elk1 reporter activation as well as phospho-Erk and phospho-Akt levels. All mutants induced extensive cytoskeletal remodeling including formation of migratory and invasive structures. Finally, GDP docking simulations suggested increased nucleotide exchange as a mechanism of activation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d480b5f5aee202dd438c3fe6977210301750ccd6" target='_blank'>
              Differential effects of KRAS G-domain and hypervariable region mutants on cancer phenotypes
              </a>
            </td>
          <td>
            J. A. D. de Borja, Kim Denyse J. Hao Lin, Daniel Angelo R. Mirador, Alexandra Isabelle D. Ang, Sidney Allen A. Chua, Joanne Marie D. Sytangco, D. Sacdalan, Reynaldo L. Garcia
          </td>
          <td>2025-11-03</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Recent technological advances have facilitated the assembly of telomere-to-telomere (T2T) genomes. The current T2T CHM13 showcases the complete architecture of the human genome, yet its use in functional experiments is limited by discrepancies with the actual genome of the specific biological system under study. Access to reference assemblies for experimentally relevant cell lines is therefore essential in advancing sequencing-based analyses and precise manipulation, particularly in highly variable regions such as centromeres. Here, we present RPE1v1.1, the near-complete diploid genome assembly of the hTERT RPE-1 cell line, a non-cancerous human retinal epithelial model with a stable karyotype. Using high-coverage Pacific Biosciences and Oxford Nanopore Technologies long-read sequencing, we generate a high-quality de novo assembly, validate it through multiple methods, and phase it by integrating high-throughput chromosome conformation capture (Hi-C) data. Our assembly includes chromosome-level scaffolds that span centromeres for all chromosomes. Comparing both haplotypes with the CHM13 genome, we detect haplotype-specific genomic variations, including the translocation between chromosome 10 and chromosome X t(X;10)(Xq28;10q21.2) characteristic of RPE-1 cells, and divergence peaking at centromeres. Altogether, the RPE1v1.1 genome provides a reference-quality diploid assembly of a widely used cell line, supporting high-precision genetic and epigenetic studies in this model system. The authors present a diploid reference-quality genome assembly of the widely used cell line RPE-1, revealing heightened variation between haplotypes and other human genomes at centromeres. The availability of a reference genome matching the cell line under study promises a new era of high-precision haplotype-resolved functional genomics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cb559c0519b9dfa48eac6893a79e9c8d2446468" target='_blank'>
              The reference genome of the human diploid cell line RPE-1
              </a>
            </td>
          <td>
            Emilia Volpe, A. Colantoni, Luca Corda, Elena Di Tommaso, Franca Pelliccia, Riccardo Ottalevi, A. Guarracino, Danilo Licastro, Luigi Faino, M. Capulli, G. Formenti, Evelyne Tassone, Simona Giunta
          </td>
          <td>2025-09-12</td>
          <td>Nature Communications</td>
          <td>2</td>
          <td>27</td>
        </tr>

        <tr id="Osteosarcoma is the most common primary malignant tumor of bone and occurs predominantly in children and adolescents during periods of accelerated skeletal growth. Within the tumor microenvironment (TME), vascular smooth muscle cells (VSMCs) may acquire osteogenic characteristics. Nuclear factor of activated T cells c1 (NFATc1) is central to skeletal biology, yet its behavior in VSMCs in osteosarcoma remains insufficiently defined. We profiled fresh osteosarcoma tissues by single-cell RNA sequencing on the 10x Genomics Chromium platform. Mechanical and enzymatic dissociation yielded single-cell suspensions; viable cells (> 85%) were enriched; libraries were prepared with Single Cell 3′ v3 reagents and sequenced on NovaSeq 6000. Reads were processed with Cell Ranger (GRCh38). Following log-normalization, 2,000 highly variable genes were retained. Harmony integration reduced batch structure. We computed 50 principal components, generated UMAP/t-SNE embeddings, and applied k-nearest neighbor/Louvain clustering (k = 20; resolution = 0.5). Cell types were assigned from differentially expressed markers cross-checked with CellMarker and PanglaoDB. NFATc1 distribution and co-expression patterns were assessed by Pearson correlation. Nine major cell classes were identified, including VSMCs, fibroblasts, macrophages, T/NK subsets, B cells, neutrophils, mast cells, and inflammatory myeloid cells. NFATc1 expression was most prominent in VSMCs, with a bimodal distribution within this lineage. In VSMCs, NFATc1 co-varied with SERGEF, APOE, and CCS, pointing to coordinated cytoskeletal remodeling, lipid handling, and copper homeostasis during osteogenic reprogramming. At single-cell resolution, NFATc1 marks a VSMC fraction with osteogenic features in osteosarcoma. The associated network suggests testable links to metabolism and matrix remodeling; functional perturbation will be required to establish causality.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e3638bf4253781f5bd9cf49aea68da6335438a6" target='_blank'>
              Single-cell RNA sequencing reveals NFATc1-mediated regulatory mechanisms of vascular smooth muscle cell osteogenic transformation in the osteosarcoma microenvironment
              </a>
            </td>
          <td>
            Xuejiao Yi, Shilin Fu, Chunliang Zhang, Tianle Zhao, Huan Feng, Dehui Qian
          </td>
          <td>2025-09-29</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f7c1744fc2f1ed50403a7b109097213e18ec402" target='_blank'>
              Design and Development of DNA Damage Chemical Inducers of Proximity (DD-CIP) for Targeted Cancer Therapy
              </a>
            </td>
          <td>
            Tian Qiu, Yeuan Ting Lee, Brendan G. Dwyer, Yi Jer Tan, Ting Chen, Bryan A. Romero, Yanlan Wang, Jiehui Deng, Tinghu Zhang, G. R. Crabtree, Stephen M. Hinshaw, , N. Gray
          </td>
          <td>2025-11-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Intra-tumor heterogeneity is a primary cause of therapeutic failure, driving tumor progression. Within tumors, diverse cell states coexist, maintained by a specific chromatin landscape that influences various cell functions, including cancer stemness. Among factors that induce chromatin changes affecting cell state fitness, DNA damage and its repair have emerged as significant contributors. This perspective examines recent advances that elucidate the interplay between DNA repair, epigenome, and cell plasticity. We discuss how epigenome affects DNA repair and, conversely, how DNA repair-induced chromatin changes influence cell plasticity. Finally, we discuss emerging concepts and highlight the therapeutic implications of these interconnected mechanisms. Cancer cell plasticity is often linked to treatment failure. Here, the authors discuss the role of epigenome in DNA repair and, how DNA repair-induced chromatin changes influence cancer cell plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb0841716149f708372bcf7e59a0f69e9d199da1" target='_blank'>
              The interplay of DNA damage, epigenetics and tumour heterogeneity in driving cancer cell fitness
              </a>
            </td>
          <td>
            Celia D Rouault, E. Charafe-Jauffret, C. Ginestier
          </td>
          <td>2025-09-30</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>56</td>
        </tr>

        <tr id="The MRE11 DNA nuclease plays central roles in the repair of DNA double-strand breaks (DSBs) as a core component of the MRE11-RAD50-NBS1 (MRN) complex. MRN localizes to chromosomal DSBs and recruits and activates the DSB repair protein kinase, ATM, which phosphorylates downstream substrates to elicit cellular DNA damage responses. Pathogenic variants in MRE11 cause the genome instability disorder ataxia-telangiectasia-like disorder (ATLD). The first ATLD patient allele identified, ATLD1, is a nonsense mutation that deletes 76 residues from the MRE11 C-terminus and markedly reduces levels of MRE11-ATLD1 and the entire MRN complex. The MRE11 C-terminus has been demonstrated to function in DNA binding, mediate protein interactions, and undergo post-translational modifications that regulate the MRE11 nuclease. We previously demonstrated that transgenic mice expressing reduced wildtype MRN levels exhibit severe phenotypes, including small body size, anemia, and DNA DSB repair defects. Thus, it is currently unknown whether low MRE11-ATLD1 levels, loss of the C-terminus, or both cause disease-associated phenotypes. In this study, we generated transgenic mouse models that express near endogenous or significantly reduced levels of MRE11-ATLD1 to determine the in vivo importance of the MRE11 C-terminus. We observe that low MRE11-ATLD1 expression leads to anemia, bone marrow failure, extramedullary hematopoiesis, and impaired lymphocyte development, similar to mice expressing low wildtype MRE11. In contrast, higher MRE11-ATLD1 expression results in a subset of moderate phenotypes, indicating that loss of the C-terminus has limited impact on MRN functions in vivo. These findings provide a foundation for predicting the clinical presentation and severity of ATLD patient phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10803b54e0a6ebf78334cad9c5698be48d5f5b38" target='_blank'>
              Differential expression of a disease-associated MRE11 variant reveals distinct phenotypic outcomes.
              </a>
            </td>
          <td>
            McKenna B DeFoer, Ahmed M Mostafa, A. Hartlerode, Steven K Orban, Keegan McDonough, Sophie Quirk, Brianna K L Ferguson, David O Ferguson, JoAnn M Sekiguchi
          </td>
          <td>2025-10-11</td>
          <td>Human molecular genetics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The cellular programs that mediate therapy resistance are often important drivers of metastasis, a phenomenon that needs to be understood better to improve screening and treatment options for patients with cancer. Although this issue has been studied extensively for chemotherapy, less is known about a causal link between resistance to radiation therapy and metastasis. We investigated this problem in triple-negative breast cancer and established that radiation-resistant tumor cells have enhanced metastatic capacity. Resistance to radiation increases the expression of integrin β3 (ITGB3), which promotes enhanced migration and invasion. Bioinformatic analysis and subsequent experimentation revealed an enrichment of RNA metabolism pathways that stabilize ITGB3 transcripts. Specifically, the RNA binding protein heterogeneous nuclear ribonucleoprotein L (HNRNPL), whose expression is regulated by Nrf2, mediates the formation of circular RNAs that sponge the family of let-7 microRNAs that target ITGB3. Collectively, our findings identify a mechanism of radiation-induced metastasis that is driven by alterations in RNA metabolism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8064b7e84c02e1178a653876a56f4f95d2d1c28" target='_blank'>
              Resistance to radiation enhances metastasis by altering RNA metabolism
              </a>
            </td>
          <td>
            Ayush Kumar, Kensei Kishimoto, H. Goel, Christi Silva, Rui Li, Brendan A Pacheco, L. J. Zhu, William A Flavahan, A. Mercurio
          </td>
          <td>2025-10-17</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Through most of medical history, treatments for metastatic cancers were ineffective, and rapid patient death was inevitable. Over the past five decades, a worldwide drug development effort has introduced a remarkable range of new cancer treatment strategies and agents so that virtually all metastatic cancers have one or more effective therapeutic options to prolong life. Yet most metastatic cancers remain fatal, and increasingly, the proximate cause of death is evolution. Local or systemic therapies applied to large, heterogeneous cancer populations elicit complex short- and long-term adaptive responses. Cells already possessing the molecular machinery of resistance obtain a stepwise fitness benefit relative to treatment-sensitive cells, allowing increased proliferation. Cells, otherwise sensitive to the treatment, may survive when in epigenetic states resistant to the treatment-induced death pathway or microenvironmental conditions that reduce drug delivery/efficacy, followed by a transition to "hard-wired" resistance allowing proliferation. These dynamics, enabled by the vast information content of the human genome, can produce diverse adaptive strategies in response to virtually all available treatments. Thus, oncology is rapidly approaching an era in which patient death is caused not by the absence of effective therapies but rather by eco-evolutionary dynamics that defeat initially successful treatments. Emerging evidence suggests that explicit integration of evolutionary principles to control or eliminate resistant populations can improve outcomes. In this issue of Cancer Research, Hockings and colleagues present an important evolutionary strategy to delay or prevent the evolution of resistance in ovarian cancer, with broad potential application. See related article by Hockings et al., p. 3503.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa08f9c64d9497ce73be1df571e19d588bc63780" target='_blank'>
              The Approaching Darwinian Era in Oncology.
              </a>
            </td>
          <td>
            Robert A. Gatenby
          </td>
          <td>2025-09-15</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The tumour suppressor protein p53 plays a central role in safeguarding genomic integrity through the regulation of DNA repair, cell cycle arrest, and apoptosis. Mutations in TP53, particularly within its DNA-binding domain, are among the most frequent genetic alterations in human cancers and are strongly associated with chemoresistance and poor prognosis. In this study, all TP53 mutations reported in the COSMIC database were systematically mapped onto all experimentally resolved TP53 three-dimensional structures available in the Protein Data Bank, supplemented with AlphaFold-predicted models to achieve full structural coverage. Mutations were classified according to their structural context—protein core, interface regions, ligand- and zinc-binding sites, and intrinsically disordered regions—and evaluated using complementary sequence- and structure-based predictive tools. The analysis revealed distinct mutational hotspots, differential distribution across structural regions, and context-dependent effects on stability and DNA-binding capacity. Notably, a subset of mutations exhibited consistent predictions of high destabilisation across all structural contexts, underscoring their potential as drivers of functional inactivation. By providing a comprehensive structural map of TP53 alterations, this work offers a valuable resource for understanding mutation-specific mechanisms of p53 dysfunction and for guiding the development of precision therapeutic strategies aimed at restoring its tumour-suppressive functions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85f3fc7b1e2b5e98df635979848fcd53d6ac30ab" target='_blank'>
              Mutational Disruption of TP53: A Structural Approach to Understanding Chemoresistance
              </a>
            </td>
          <td>
            Ali F. Alsulami
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Members of the fragile X protein (FXP) family (FMR1, FXR1 and FXR2) are differentially expressed in most types of cancer and major neurodegenerative diseases. While increased expression of FXR1 in cancer has been linked to senescence evasion and consequently tumor initiation and progression, decreased expression of FXPs in neurodegeneration may contribute to pathogenic protein aggregation and death of vulnerable neurons. However, due the causal role in fragile x syndrome, most data are available about loss of FMR1 in neurons while functions of FXR1 and especially FXR2 remain largely unexplored. To address this knowledge gap, and to directly compare functions of the FXPs, we used proteomics of CRISPR/Cas9 edited HAP1 cells carrying knockouts of the individual FXPs for identification of cellular mechanisms associated with these proteins. Further exploration of proteomic findings suggests roles of the FXPs in ribosome biogenesis, autophagy and mitochondrial health linked to organismal aging, and cellular senescence. Validation of FXP induced defects relevant for neurodegenerative diseases in neuroblastoma cell line SH-SY5Y upon FXP knockdown revealed high cell type specificity of individual FXP functions. Overall, we provide a comprehensive overview and comparison of cellular mechanisms related to the individual FXPs, as well as starting points for further studying this protein family in respective cell types of FXP associated diseases, and in aging in general.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31719b8163204b6368381730d4d32465032911f8" target='_blank'>
              Comparing loss of individual fragile X proteins suggests strong links to cellular senescence and aging
              </a>
            </td>
          <td>
            Sonja Menge, Inmaculada Segura, Max Hartmann, Lorena Decker, Selin Kiran, K. Danzer, S. Iben, A. Harbauer, P. Oeckl, A. Freischmidt
          </td>
          <td>2025-10-21</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Understanding therapy resistance requires deconvolving heterogeneous cell populations and tracking clonal trajectories. While CRISPR-based cellular barcoding is powerful for lineage tracing, many platforms suffer from low efficiency and limited compatibility with single-cell transcriptomics. We developed Oligo-CALL (Oligonucleotide-inducible CRISPR transcriptional activator–Assisted Lineage Labeling), an advanced barcoding system enabling precise lineage tracing, live clone isolation, and seamless integration with single-cell RNA sequencing. Applied to lung cancer cells treated with a KRASG12C inhibitor, Oligo-CALL identified clones consistently enriched posttreatment, supporting a model of predestined resistance. Oligo-CALL achieved >95% efficiency in linking lineage identity to transcriptomes, uncovering diverse clone-specific pathways with underlying resistance. Paired analysis of barcode-matched clones from naïve and resistant populations revealed transient and fixed resistance phenotypes. Notably, DNA repair pathways are recurrently altered in resistant clones, and inhibition of poly(adenosine 5′-diphosphate–ribose) polymerase synergizes with KRAS G12C inhibition to overcome resistance. Together, Oligo-CALL provides a versatile platform for dissecting lineage evolution and molecular dynamics of targeted therapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fc6bc5d7c1ef3b43ef612b4e6dbe8df9b69936e" target='_blank'>
              Oligo-CALL: A next-generation barcoding platform for studying resistance to targeted therapy
              </a>
            </td>
          <td>
            Yingzhuo Liu, Yi Ban, Dingcheng Gao
          </td>
          <td>2025-11-07</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90dbf82e0a483cd704102a3b6a055d355e22f224" target='_blank'>
              A mathematical model of drug-resistant cancer with cell cycle structure: stability, optimal control, and cost-effectiveness analysis
              </a>
            </td>
          <td>
            Tawfik Jaber, O. Balatif
          </td>
          <td>2025-09-29</td>
          <td>International Journal of Dynamics and Control</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Chronic lymphocytic leukemia (CLL) is a common neoplasm that carries the risk of transformation into Richter’s syndrome (RS), a highly aggressive B cell lymphoma with poor prognosis. Limited availability of animal models and cell lines hinders understanding of transformation mechanisms. Addressing this gap, we established the first in silico dynamic model of the disease. Our methodology integrates mathematical logic modeling with experimental data to identify disease drivers, mechanisms, and potential therapeutic targets. We validated the model by comparing the model’s readout with experimental data from different biological levels, such as single-cell RNA sequencing analyses and a CLL/RS patient formalin-fixed paraffin-embedded (FFPE) tissue cohort. Our analyses identified BMI1 proto-oncogene and TP53 loss as key RS progression regulators. In addition, we performed an in silico target screening to identify promising target combinations in a personalized fashion. Through the synergy of mathematical modeling with experimental readouts, our model provides a complementary approach to investigate the process of CLL transformation to RS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b0084b1cbd03c2692e1100a97e2d57fe76c34d4" target='_blank'>
              CLL to Richter syndrome: Integrating network strategies with experiments elucidating disease drivers and personalized therapies
              </a>
            </td>
          <td>
            Julia Maier, J. Schwab, Silke D. Werle, Ralf Marienfeld, S. Stilgenbauer, Peter Möller, Nensi Ikonomi, Hans A. Kestler
          </td>
          <td>2025-09-12</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="The combination of 5-fluorouracil and irinotecan (FOLFIRI) remains a standard-of-care treatment for metastatic colorectal cancer (mCRC), yet benefits only about half of patients. Using patient-derived xenografts (PDXs), we investigated the biological underpinnings of this heterogeneous response. FOLFIRI-resistant models showed transcriptional upregulation of innate immunity and mitochondrial metabolism genes, together with reduced expression of the DNA polymerase POLD1. Sensitive counterparts exhibited a BRCAness-like phenotype with genomic scars of homologous recombination (HR) deficiency, not caused by genetic or epigenetic loss of HR genes but by low abundance of the RAD51 recombinase. In tumoroids, forced RAD51 overexpression attenuated HR deficiency-related scars and chemotherapy-induced damage, while HR inhibition through ATM blockade enhanced drug sensitivity. The predictive relevance of key response determinants was validated in clinical samples. This work illuminates functional, non-genetic facets of BRCAness in mCRC and introduces actionable biomarkers and targets, offering prospects to improve clinical decision-making and broaden therapeutic options for chemorefractory patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fb783603c147c9c964fbfcc1a8e05ff7e8e4445" target='_blank'>
              The molecular and functional landscape of resistance to FOLFIRI chemotherapy in metastatic colorectal cancer.
              </a>
            </td>
          <td>
            Marco Avolio, S. Leto, F. Sassi, B. Lupo, Elena Grassi, Irene Catalano, E. Zanella, V. Vurchio, F. Cottino, Petros K Tsantoulis, L. Lazzari, Paolo Luraghi, Martina Ferri, Francesco Galimi, E. Berrino, S. Bellomo, Marco Viviani, A. Sogari, G. Mauri, F. Tosi, F. Cruciani, A. Sartore-Bianchi, S. Siena, Felice Borghi, V. Torri, E. Élez, J. Tabernero, Maria Nieva, C. Montagut, N. Tarazona, Andrés Cervantes, Sabine Tejpar, A. Bardelli, C. Marchiò, S. Marsoni, Andrea Bertotti, L. Trusolino
          </td>
          <td>2025-09-22</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Ribosomal proteins (RPs) are essential structural and functional components of the ribosome, and their disruption during embryogenesis generally results in embryonic lethality. Nevertheless, cancer genomes frequently harbor somatic missense mutations and gene deletions in RP genes in patterns that suggest selective advantages, often linked to inactivation of the tumor suppressor protein p53 pathway. This review discusses the landscape of RP mutations in cancer and their mechanistic consequences. RP mutations are detected across multiple malignancies, including glioblastoma, melanoma, T-cell acute lymphoblastic leukemia, as well as in congenital ribosomopathies, such as Diamond-Blackfan anemia, which confer an elevated lifetime risk of developing cancer. Cancer-associated RP mutations disturb ribosome homeostasis, compromise translational fidelity, and trigger proteotoxic and ribosomal stress, yet without halting tumor growth. Some RP mutants occupy structurally sensitive positions within the ribosome, altering mRNA selectivity and quality control. In turn, cancer cells may adapt through compensatory mechanisms, including the upregulation of RP paralogs, activation of proteostasis regulators, and rewiring of stress response pathways. Rather than a loss-of-function event, an RP mutation may create a persistent ribosomal disequilibrium that fundamentally alters cellular functions. Such changes in a cancer cell could generate interesting therapeutic vulnerabilities and targets, such as dependence on stress signaling, proteasome activity, or RP paralog expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b8cf8b6873dcfd04105fc54f804a07dc87de411" target='_blank'>
              Toward an understanding of cancer-associated ribosomal protein mutations
              </a>
            </td>
          <td>
            Mikael S. Lindström
          </td>
          <td>2025-10-03</td>
          <td>Tumor Discovery</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Mutations in RNA-binding proteins are increasingly identified in cancers through tumor sequencing and are correlated with disease progression, therapy response, and overall patient outcomes, underscoring the need to study them. Here, we focus on the RNA-binding protein Poly-C binding protein 1 (PCBP1), which binds target RNAs through K-homology (KH) domains to regulate RNA fate. PCBP1 is a tumor suppressor gene and hotspot missense mutations at leucine residues 100 and 102 are observed in colorectal cancer (CRC). PCBP1 mutations have been recurrently reported in CRC genome-wide mutation studies and are associated with poor clinical outcomes; however, their effects on PCBP1 expression and function remain largely unexplored. We show that cancer-associated mutations substituting leucine 100 and 102 with glutamine, proline, or arginine destabilize PCBP1, leading to increased protein turnover. The L100/L102 residues occur at the interface of the RNA-binding KH1 and KH2 domains, and our molecular dynamics simulations show that mutations at these residues disrupt the secondary structure of PCBP1. Additionally, these mutants display increased cytoplasmic localization. Importantly, mutant PCBP1 physically interacts with wild type PCBP1 and suppresses its expression through a dominant-negative mechanism. Together, our data demonstrate that CRC-associated PCBP1 mutations destabilize the protein and act as dominant-negative variants, revealing a novel mechanism of tumor suppressor inactivation in colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a7a507c9272e7ca6986d3ffe683c348245fff39" target='_blank'>
              Colorectal cancer-associated PCBP1 mutations disrupt protein stability in a dominant negative manner
              </a>
            </td>
          <td>
            Paige V. Blinkiewicz, Nicole C. Hoenes, Myra X. Afzal, Yihong Liu, Ulysses J. Hill, B. Bjarnadóttir, T. E. Burgin, P. Malaney
          </td>
          <td>2025-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Highlights What are the main findings? Emerging data from multidimensional and high-resolution phenotyping of disease-relevant tissues indicates that osteoarthritis (OA) is a polygenic disease, where genetic factors disrupt the chromatin landscape in disease-relevant cells leading to aberrant expression of effector genes that drive OA pathogenesis. According to the concept of a developmental origin for OA, the functional cooperation between chromatin dynamics and transcription factors (TFs) regulates the unfolding of a development-specific gene expression program, defines the outcome of skeletogenesis, and ultimately determines the articular biomechanics and the risk of OA. What is the implication of the main findings? Detailed mapping and functional characterization of the OA-associated chromatin conformation and epigenetic disruptions may accelerate the discovery of disease-modifying drugs for OA. Novel technologies pave the way for precise epigenetic editing at the desired genomic regions and may allow a targeted transcriptional regulation of OA-relevant genes in disease-relevant cells. Abstract Recent technological breakthroughs have enabled multidimensional phenotyping, with unprecedented single-cell resolution and genome-wide coverage, across multiple osteoarthritis (OA)-relevant tissues, such as articular cartilage, synovium, infrapatellar fat pad, and subchondral bone. The majority of the single nucleotide variations (SNVs) that have been associated with OA are located in non-protein coding regions and confer risk for disease by altering the expression level, instead of the amino acid sequence of the gene product. These data have shaped the concept of OA as a polygenic disease, where genetic factors disrupt the chromatin landscape in disease-relevant cells, leading to aberrant expression of effector genes. Pharmacologic manipulation of the OA-driving epigenetic landscape has recently emerged as an attractive path for the development of disease-modifying drugs. Novel clustered regulatory interspaced short palindromic repeats (CRISPR)-based technologies provide opportunities for precise epigenetic editing at the desired genomic regions and may allow a targeted transcriptional regulation of disease-relevant genes in disease-relevant cells. The aim of the present narrative review is to summarize the emerging data on the role of epigenetic factors and chromatin structure as calibrators of the risk for developing OA and to discuss the opportunities and challenges arising from the use of chromatin landscape to guide drug discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb518b2e515941622a9daa713244c8885ed8ed56" target='_blank'>
              The Articular Chromatin Landscape in Osteoarthritis
              </a>
            </td>
          <td>
            G. Kalliolias, E. Basdra, Athanasios G. Papavassiliou
          </td>
          <td>2025-10-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="The biological mechanisms driving the long survival in glioblastoma (GBM). Five‐year long‐term survival (LTS) and 10‐year survival very long‐term survival (VLTS) remain significantly understudied. Here we molecularly detailed two cases. AR10‐046 (VLTS) was affected by a giant cell‐GBM, classified as the pedHGG_RTK1a subtype according to the v12.5 Heidelberg brain tumor methylation classifier. Somatic and germline MSH6 mutations, typically in Lynch syndrome, and high tumour mutational burden were detected. The copy number variation plots showed chromosome 1q gain and chromosome 13 loss with no other typical GBM alterations. AR10‐037 (LTS) suffered from a classical GBM, identified as pedHGG_MYCN subclass. Apart from the canonical chromosome 7 gain and chromosome 10q loss, we observed MDM2 gene amplification and possible rearrangements on chromosome 12 and 18 with the typical aspect of chromothripsis, harbouring two putative new gene fusions: CPSF6::CPM and PTPRR::RAB3IP. We described two patients with peculiar tumour molecular profile, widening the scenario of clinical and molecular variability in such patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e8551ad58743a80e7a359a8ec1bcd9af76ef1c7" target='_blank'>
              Unique Genetic and Epigenetic Alterations in Glioblastoma Long‐Term Survivors: Insights From Two Clinical Cases
              </a>
            </td>
          <td>
            E. Anghileri, E. Miele, S. Patrizi, S. Barresi, E. Lazzarini, L. Maddaloni, M. Patanè, L. Pedace, R. Paterra, A. Silvani, Franco Locatelli, S. Indraccolo, B. Pollo
          </td>
          <td>2025-10-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/889c401af57de3889b95cf28e228d76bc1fc7c6d" target='_blank'>
              Repeated losses of self-fertility shaped heterozygosity and polyploidy in yeast evolution
              </a>
            </td>
          <td>
            Nina Vittorelli, C. Gómez-Muñoz, Irina Andriushchenko, Louis Ollivier, Nicolas Agier, Stéphane Delmas, Yann Corbeau, Guillaume Achaz, M. Cosentino Lagomarsino, Gianni Liti, Bertrand Llorente, G. Fischer
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f80032618ad0bd974f697fa45a97d919565be754" target='_blank'>
              Selective degradation of oncogenic extrachromosomal DNA by type I-E CRISPR
              </a>
            </td>
          <td>
            Ensieh Poursani, Vu v. Pham, A. Ehteda, Orazio Vittorio
          </td>
          <td>2025-09-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a660def78ace40512480c658f652835e9dad99a3" target='_blank'>
              The long non-coding RNA, CyKILRb, augments oncogenic phenotypes via induction of PIK3R2 and activation of the PI3K/AKT axis
              </a>
            </td>
          <td>
            Xiujie Xie, H. Macknight, Amy L. Lu, C. E. Chalfant
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Despite significant advances in cancer research, the complexity of signaling pathways remains a major challenge in precision oncology. Tumors harbor a diverse array of genetic alterations; however, these often converge onto a limited set of core signaling pathways, notably RAS/RAF/MEK/ERK (MAPK), PI3K/AKT/mTOR (PAM), and Wnt/β-catenin. Concurrently, these pathways diverge extensively downstream, driving therapeutic resistance through mechanisms such as epithelial-mesenchymal transition, immune evasion, and metabolic reprogramming. Unlike previous literature that largely provides descriptive accounts of pathway alterations, this review uniquely synthesizes convergent and divergent signaling into a clinically actionable diagnostic and therapeutic framework. It critically assesses current precision oncology strategies, identifies gaps, and proposes a dual-axis model that integrates static genomic profiling with dynamic signaling evolution to inform precision therapy. By highlighting opportunities for combination therapy informed by pathway interdependencies and adaptive resistance mechanisms, this perspective provides novel clinical insights and tangible directions for future research in overcoming resistance in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f1f096cbac949c4b15346a048f653260da1f8e6" target='_blank'>
              Convergent and Divergent Signaling Pathways in Cancer: A Dual-Axis Model for Adaptive Precision Oncology.
              </a>
            </td>
          <td>
            Aliasgar Shahiwala PhD
          </td>
          <td>2025-10-08</td>
          <td>Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Despite decades of progress in ovarian cancer treatment, therapeutic resistance remains a persistent and lethal challenge. Emerging evidence suggests polyploid giant cancer cells (PGCCs) as key drivers of this resistance. PGCCs are distinguished by multinucleation or a single oversized nucleus containing multiple chromosome sets. Polyploid cell populations are present in premalignant lesions and expand with disease progression and therapeutic stress. PGCCs demonstrate heightened resistance to treatment relative to non-PGCCs and can regenerate non-PGCCs via budding once therapeutic pressure is alleviated, leading to relapse. Supported by extensive preclinical and clinical evidence, we hypothesize that elimination of PGCCs may reduce therapeutic resistance in ovarian cancer. Given their significant role in resistance and relapse, PGCCs remain largely untargeted due to a lack of scalable detection methods. Standard viability assays such as MTT, XTT, and ATP-based screens fail to capture the rare but critical PGCC subpopulation. Current gold-standard methods, fluorescence-activated cell sorting (FACS) coupled with manual microscopy, while accurate, are time-intensive and unsuitable for high-throughput drug screening. Thus, a rapid and accurate technique for studying PGCCs is essential. In this pilot study, we have developed a high-throughput single-cell morphological analysis pipeline capable of precisely quantifying PGCC numbers and proportions. The pipeline accurately quantifies PGCCs based on nuclear and cellular features, including size and DNA content. Leveraging this platform, we screened a library of 2,726 FDA-approved compounds to identify agents that effectively kill PGCCs. Hits from this screen were validated in 3D spheroid cultures, revealing compounds with potent anti-PGCC activity. Complementary transcriptomic analysis (RNA-Seq) of flow-sorted PGCCs versus non-PGCCs unveiled profound dysregulation of the FOXM1 signaling axis, implicating this pathway as a key regulator of PGCC formation and survival. Consistent with this, the high-throughput screen identified Thiostrepton, a known FOXM1 inhibitor, as a selective anti-PGCC agent. Across multiple ovarian cancer cell lines, FOXM1 inhibitors outperformed standard-of-care agents (Paclitaxel, Cisplatin, Olaparib) in eliminating PGCCs. siRNA knockdown of FOXM1 significantly reduced PGCC numbers, particularly under Paclitaxel stress. Moreover, FOXM1 inhibition triggered apoptosis in PGCCs, an effect reversible with the pan-caspase inhibitor Z-VAD-FMK. Together, these findings underscore PGCCs as a therapeutically actionable cell state driving resistance in ovarian cancer. Our integrative single-cell morphological and transcriptomic platform enables rapid discovery of anti-PGCC agents and establishes FOXM1 inhibition as a compelling strategy to eradicate this resilient subpopulation. This pipeline offers broad utility for overcoming therapy resistance across diverse malignancies.



 Hsiao-Chun Chen, Yuan Zhang, Yushu Ma, Yu-Chih Chen. Combating ovarian cancer resistance via polyploid cancer cell targeted screening [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A023.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62f27ef6b0280aa103e9f9c96579bbfa985df28c" target='_blank'>
              Abstract A023: Combating ovarian cancer resistance via polyploid cancer cell targeted screening
              </a>
            </td>
          <td>
            Hsiao-Chun Chen, Yuan Zhang, Yushu Ma, Yu-Chih Chen
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Resistance to programmed cell death is a defining hallmark of cancer and a persistent barrier to successful therapy. Dual-function proteins such as p53, Ras, HIF-1α, BNIP3, and NF-κB act as molecular switches that determine cell fate between apoptosis and survival. In tumors, these proteins are deregulated not only by intrinsic mutations but also by extrinsic signals from the tumor microenvironment (TME). This Mini Review critically analyzes previous therapeutic approaches, emphasizing overlooked mechanisms such as Ras-mediated suppression of p53. It proposes a sequential therapeutic strategy: first, dismantling TME adaptations (hypoxia, inflammation, protective autophagy); second, inhibiting oncogenic Ras signaling; and third, restoring p53 activity. The phased approach integrates biomarker-guided patient stratification, recognizes tumor–microenvironment co-evolution, and highlights how resistance evolves over time. Although the concept does not resolve all challenges, it outlines a rational framework for restoring apoptotic competence and provides a pathway for translational and clinical testing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79ed9bca532a19e58792704a40e38539d4ea37e5" target='_blank'>
              Context-dependent rewiring of dual-function proteins in cancer: a sequential strategy to restore apoptosis
              </a>
            </td>
          <td>
            Leon Strzadala
          </td>
          <td>2025-10-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fa5a90466cca365ac23cb5111624676a9f4bea9" target='_blank'>
              Deciphering Cellular Ecosystems Driving Tumor Progression and Immune Escape from Spatial Transcriptomics and Single-Cell with COMPOTES
              </a>
            </td>
          <td>
            L. Herpin, Anaïs Chossegros, Roberta Codato, Josep Monserrat Sanchez, Jean El Khoury, Simon Grouard, V. Ducret, Alex Cornish, Baptiste Gross, Elodie Pronier, C. Hoffmann, Alberto Romagnoni, Eric Y. Durand, Almudena Espin Perez, Quentin Bayard
          </td>
          <td>2025-10-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/434dbd23a412e8c919acd85ad46264f7a360e2f6" target='_blank'>
              Long-read sequencing reveals telomere inheritance patterns from human trios
              </a>
            </td>
          <td>
            Yuxin Zhou, David R. Lougheed, Warren A. Cheung, I. Thiffault, T. Pastinen, Guillaume Bourque
          </td>
          <td>2025-10-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Genomic amplification may result in aberrant gene expression and support development of cancer, including chronic myeloid leukemia (CML). In CML cell line K-562, we recently reported overexpression of TBX1 located at chromosomal position 22q11, focally co-amplified together with BCR, part of the CML hallmark fusion gene BCR::ABL1. Here, we extended that study, by identifying genomically amplified and overexpressed MAPK1/ERK2 at 22q11 together with MCTS1 at Xq22. Using pharmacological inhibitors and siRNA-mediated knockdown assays, our data collectively revealed novel regulatory connections between TBX1, MAPK1 and MCTS1, which may play a role in drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11cfe5a18303d3c15ad4fb64c39efcc6da10d99d" target='_blank'>
              Aberrant expression of MAPK1 and MCTS1 in chronic myeloid leukemia (CML)
              </a>
            </td>
          <td>
            Leo Kortendick, C. Meyer, S. Nagel
          </td>
          <td>2025-09-18</td>
          <td>microPublication Biology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Aim: Cisplatin resistance in head and neck squamous cell carcinoma (HNSCC) is thought to involve both reduced drug uptake and altered molecular responses. However, the relative contribution of these mechanisms remains unclear. Methods: Two HNSCC cell lines with differing sensitivity (HNO97 and HNO41) were analyzed using cytotoxicity assays, atomic absorption spectroscopy-based quantification of intracellular cisplatin, caspase 3/7 assays, Western blotting, polymerase chain reaction (PCR)-based transcriptomic analysis of DNA damage response and cell cycle arrest pathways, and RNA-seq data from The Cancer Genome Atlas (TCGA) to characterize the resistance phenotype. Results: HNO97 (IC50 = 440 µM) was 7.6-fold more resistant to cisplatin than HNO41 (IC50 = 57.8 µM; P = 0.0286). After quantifying intracellular uptake (pg Pt/µg protein) and normalizing cytotoxicity to intracellular drug levels, HNO97 (IC50 = 778.9 pg Pt/µg protein) remained 5-fold more resistant than HNO41 (IC50 = 153.5 pg Pt/µg protein), indicating only a partial reduction in resistance (33% decrease, from 7.6-fold to 5-fold; P = 0.0286). At cisplatin concentrations yielding comparable intracellular exposure (HNO97: 440 µM; HNO41: 196 µM; both ≈ 725 pg Pt/µg protein), caspase 3/7 activation and induction of CDKN1A, GADD45A, GADD45G, and PPP1R15A were weaker in HNO97 than in HNO41. Notably, baseline expression of these genes was significantly higher in HNO97. In the TCGA cohort, multivariate analysis showed that high FANCD2 expression was associated with unfavorable recurrence-free survival in platinum-treated patients (hazard ratio = 4.0; P = 0.011), but not in those who did not receive platinum chemotherapy. Conclusion: Cisplatin resistance in HNSCC appears to be driven primarily by molecular mechanisms involving DNA damage response and cell cycle arrest pathways, rather than poor drug uptake.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba27336a14cf094e2bc0e409273ca8370a971da1" target='_blank'>
              Cisplatin resistance in head and neck squamous cell carcinoma is linked to DNA damage response and cell cycle arrest transcriptomics rather than poor drug uptake
              </a>
            </td>
          <td>
            Ketaki Sandu, R. Warta, Uddipta Biswas, Wang Zhang, P. Michl, Christel Herold-Mende, Johanna Weiss, Dirk Theile
          </td>
          <td>2025-09-19</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="


 Protein arginine methyltransferases (PRMTs) are critical regulators of cellular processes such as cell cycle progression, DNA repair, immune signaling, and transcriptional regulation. Among them, type I PRMTs, responsible for catalyzing asymmetric dimethylation of arginine residues, are frequently overexpressed in human cancers and represent compelling therapeutic targets. This study investigates the anticancer potential of UnAcH416 (P2), a novel peptoid-based inhibitor with selective activity against PRMT1. P2 exhibited dose-dependent antiproliferative effects across various cancer cell lines, with minimal cytotoxicity in non-malignant cells, indicating favorable tumor selectivity through cell viability and morphological analyses. Mechanistic studies demonstrated that P2 transiently inhibits PRMT1 activity and induces cell cycle arrest at the G1/S transition, as confirmed by Western blot analysis. In both MDA-MB-468 breast cancer and HCT116 colon cancer cells, P2 treatment led to elevated LC3-II levels, detected by Western blot and ELISA, suggesting the induction of autophagy as a component of its mechanism of action. Unexpectedly, P2 appeared to promote migratory behavior in cancer cells, as evidenced by accelerated wound closure in scratch assays. Additionally, combinatorial treatment with P2 and the chemotherapeutic agent paclitaxel (Taxol) resulted in reduced drug sensitivity, potentially due to altered cytoskeletal dynamics or autophagy-associated resistance mechanisms. However, gelatin zymography and MMP-9 activity assays revealed a concurrent suppression of extracellular matrix remodeling, suggesting a complex, possibly stress-related modulation of invasion pathways. Ongoing research aims to elucidate the relationship between P2-induced autophagy and epithelial-to-mesenchymal transition (EMT), as well as to explore synergy with anti-migratory agents and bioactive compounds such as doxycycline and saffron. Future studies will include in vivo validation, maximum tolerated dose (MTD) determination, hollow fiber assays, and pharmacodynamic profiling.



 Joseph Camps, Naomi Laird, Diana Soto, Corey Causey, Bryan Knuckley, Fatima K. Rehman. UnAcH416 (P2), a selective PRMT1 inhibitor, disrupts tumor cell growth and stress-response pathways [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr A089.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/caf474117df54b3326e86f5b540cdb9498419307" target='_blank'>
              Abstract A089: UnAcH416 (P2), a selective PRMT1 inhibitor, disrupts tumor cell growth and stress-response pathways
              </a>
            </td>
          <td>
            Joseph Camps, Naomi Laird, Diana Soto, Corey P. Causey, B. Knuckley, F. K. Rehman
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Cisplatin resistance presents a significant challenge in cancer therapy, emphasizing the necessity for identifying new regulatory elements that influence drug response. Recent research has revealed the importance of long noncoding RNAs (lncRNAs) in chemotherapy resistance, with LINC02381 identified as a potential regulatory factor. Through an in-depth bioinformatics analysis, we investigated the impact of LINC02381 on cisplatin resistance in ovarian cancer across various datasets. By conducting differential expression analysis, survival analysis, gene set enrichment analysis (GSEA), and constructing protein-protein interaction (PPI) networks, we identified key pathways associated with LINC02381 expression. The results indicated that the altered expression of LINC02381 in patients treated with cisplatin was associated with reduced survival. Functional studies and correlation analyses further demonstrated that this LncRNA influences critical pathways and genes related to apoptosis, efflux, DNA repair, and EMT. Lastly, through an examination of its interactions with microRNA and protein networks, we identified LINC02381 as a ceRNA implicated in cisplatin resistance. Our findings suggest that LINC02381 may influence cisplatin sensitivity in ovarian cancer and establish a basis for further experimental validation, including molecular assays or in vivo analyses, and suggest the potential therapeutic targeting of LINC02381 to combat chemoresistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35a8ae366b9c19996905f6b9e1ed9f603948832f" target='_blank'>
              The Role of LINC02381 in Modulating Cisplatin Resistance in Ovarian Cancer: A Bioinformatics Approach.
              </a>
            </td>
          <td>
            Zeinab Karbalaei Pazoki, B. M. Soltani, Mostafa Hosseini, Shiva Irani
          </td>
          <td>2025-09-16</td>
          <td>Iranian journal of allergy, asthma, and immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Breast cancer (BC) remains one of the most prevalent cancers worldwide and a significant cause of cancer-related mortality among women. Despite significant advancements in understanding the genetic foundations of BC, numerous research initiatives have historically focused on protein-coding genes, which constitute merely about 2% of the human genome. This focus has produced significant insights into oncogenes such as HER2, TP53, and BRCA1, along with tumor suppressor genes. Nonetheless, it has led to the non-coding portions of the genome garnering relatively less focus. Recent studies illuminate the crucial significance of non-coding DNA in cancer biology, highlighting its regulatory roles and influence on tumor formation, metastasis, and treatment resistance. This review examines the importance of non-coding DNA in BC. It provides an in-depth analysis of essential non-coding regions, their functions in gene regulation and chromatin structure, and their implications for various BC subtypes. Examining these facets, we seek to reveal the potential of non-coding DNA as a viable source of novel diagnostic markers and treatment approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/966867774ecd7f0803dece977fed36246e346fe6" target='_blank'>
              The Role of Non-Coding Regions in Breast Cancer: From Gene Regulation to Therapeutic Implications
              </a>
            </td>
          <td>
            Hussein Sabit, Sarah Sobhy, Shaimaa Abdel-Ghany, Al-Hassan Soliman Wadan, Olubukola Ayodele, Yasser Albrahim, Hirendra Banerjee, Ahmed Elhashash, B. Arneth
          </td>
          <td>2025-09-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ed2431373f3afb26d382f9fbc1e05e4679be852" target='_blank'>
              Activating NRF2E79Q mutation alters the differentiation of human non-small cell lung cancer.
              </a>
            </td>
          <td>
            S. Hamad, Hansa Joshi, T. Hess, S. Jefferys, Zena Saleh, N. Bendjilali, Rani S Sellers, G. Zhu, Travis Shrank, R. Moore, David Corcoran, Jeremy M. Simon, Francis R. Spitz, David Shersher, M. Major, Bernard Weissman
          </td>
          <td>2025-10-11</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="


 Colorectal cancer (CRC) remains a major global health burden, contributing significantly to morbidity and mortality worldwide. Despite advances in understanding the molecular landscape of CRC and the development of targeted therapies, there is still a critical need for novel treatment strategies that address specific molecular vulnerabilities in CRC. To address this challenge, we developed a robust and scalable platform for target discovery, centered on identifying synthetic lethal (SL) interactions linked to key oncogenic drivers such as APC, KRAS, and SMAD4. Using colorectal cancer (CRC) as a model, we integrated genomic and transcriptomic analyses across a diverse set of biologically relevant systems, including human intestinal stem cell (hISC)-derived isogenic cancer models that accurately replicate tumor genotypes, patient-derived cells (PDCs), and clinical tumor specimens. At the heart of our platform is CRISPR-based functional screening, enabling systematic interrogation of gene dependencies. To benchmark our engineered models, we applied machine learning to RNA sequencing data, comparing them against a large cohort of clinically annotated CRC transcriptomes. This approach enabled precise molecular subtyping, detailed characterization of gene expression programs, and the identification of novel, tractable vulnerabilities specific to genetically defined subgroups. Candidate targets were prioritized based on therapeutic relevance and selective efficacy in cancer cells, while sparing untransformed hISCs. Our findings not only advance early-stage target identification and validation but also support the initiation of precision drug discovery programs focused on CRC subtypes driven by APC, KRAS, and SMAD4 mutations. Moreover, the versatility of our platform enables its application to other malignancies. In summary, we present an integrated, high-resolution framework for uncovering novel molecular dependencies in CRC, providing a foundation for precision oncology and future therapeutic innovation.



 Eliza Zimoląg, Joanna Szuszkiewicz, Marcin Serocki, Marcin Duleba, Oleksii Bryzghalov, Izabella Więckowska, Marcelina Chmiel, Barbara Lipert, Katarzyna Sarad, Julia Wirkijowska, Szymon Worszyło, Dariusz Klonski, Michał Mikula, Rafał Dziadziuszko, Andrew Thomason, Krzysztof Brzózka, Andrzej Mazan. Identification of novel molecular vulnerabilities in colorectal cancer through integrated transcriptomic profiling and functional genomics [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr B050.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8a3fb0b893ce7123a2e4b7841128bf8eca7bc83" target='_blank'>
              Abstract B050: Identification of novel molecular vulnerabilities in colorectal cancer through integrated transcriptomic profiling and functional genomics
              </a>
            </td>
          <td>
            E. Zimoląg, J. Szuszkiewicz, M. Serocki, M. Dulęba, O. Bryzghalov, Izabella Więckowska, M. Chmiel, B. Lipert, Katarzyna Sarad, Julia Wirkijowska, Szymon Worszyło, Dariusz Klonski, M. Mikula, Rafal Dziadziuszko, Andrew Thomason, Krzysztof Brzózka, Andrzej Mazan
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="T-cell acute lymphoblastic leukemia (T-ALL) is an aggressive hematologic malignancy caused by mutation accumulation during hematopoiesis. The characterization of chromosomal abnormalities may provide significant insights into genetic mechanisms of malignant transformation in hematopoietic cells. However, T-ALL is genetically very heterogenous and driving mutations as well as clonal markers for the assessment of minimal residual disease are not always identifiable. Hence, there is a clinical need to further refine the genetic landscape of T-ALL including previously unrecognized fusion partners of commonly translocated genes in T-ALL of childhood. In this study, we screened n = 229 T-ALL cases by our targeted genomic capture high-throughput sequencing (gc-HTS) approach. In total, we identified n = 60 gene–gene fusions, present in n = 57 (25%) of the patients. Nine rare or even unrecognized translocations were identified and validated. Furthermore, owing to its interesting chromosomal structure, we studied the oncogenic potential of the complex rearrangement of chromosome 2 and 12, found in a near-early T-cell progenitor (ETP) ALL that leads to the fusion events PRKCE::ETV6 and ETV6::INO80D. Exogenous expression of PRKCE::ETV6 in Ba/F3 pro-B and D1 T-cells caused interleukin-independent proliferation and enhanced survival upon interleukin withdrawal, respectively. Our study underlines the heterogenous mutational landscape in T-ALL. The previously unrecognized PRKCE::ETV6 resulting from a complex rearrangement involving chromosome 2 and 12 demonstrated transforming potential in cytokine-dependent cellular models support the notion of a driver mutation in near ETP-ALL. Our data reconfirm the relevance of ETV6-fusion proteins in the pathogenesis of undifferentiated T-ALL. Importantly, genomic breakpoints at the ETV6 locus represent potentially robust MRD markers for (near) ETP-ALL that lack IG/TR rearrangements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b131db4b78b9de8c54f4c834443ee073ebb553a8" target='_blank'>
              Characterization of a PRKCE::ETV6 fusion as a potential oncogenic driver in T-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Esther L Monsees, U. Zur Stadt, Julia Strauss, Sabrina Schuster, N. Kleist, R. Hauch, Michael Spohn, Gerrit Wolters-Eisfeld, M. Horstmann, G. Escherich, Lena Behrmann
          </td>
          <td>2025-10-22</td>
          <td>Molecular and Cellular Pediatrics</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="BCL-2 is a central regulator of apoptosis and inhibits cell death by sequestering pro-apoptotic BH3 alpha-helices within a hydrophobic surface groove. While venetoclax, a BH3-mimetic drug, has transformed the treatment of BCL-2–driven malignancies, its efficacy is increasingly limited by acquired resistance mutations that disrupt small-molecule binding yet preserve anti-apoptotic function—reflecting a remarkable structural adaptation. Here, we employ hydrocarbon-stapled alpha-helices derived from the BAD BH3 motif as conformation-sensitive molecular probes to investigate this therapeutic challenge. The stapled peptides not only retain high-affinity binding to all BCL-2 variants but also show enhanced potency to select venetoclax-resistant mutants. Structural analyses, including X-ray crystallography and hydrogen-deuterium exchange mass spectrometry (HDX MS), demonstrate that these stapled helices restore native BH3 engagement by reversing the conformational consequences of resistance mutations. Notably, we identify a serendipitous interaction between the α3–α4 region of BCL-2 and hydrocarbon staple, which further compensates for altered groove conformation and contributes to mutant binding affinity. Together, these findings offer mechanistic insights into BCL-2 drug resistance and reveal a blueprint for designing next-generation inhibitors that overcome this clinically significant barrier to durable treatment responses. Resistance mutations in BCL-2 reduce the clinical efficacy of venetoclax. DeAngelo et al. show stapled BAD BH3 peptides can retain and even enhance binding to these mutants, offering a structurally informed strategy to overcome this mechanism of cancer drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c989e0417fad14adefa4b6808df4b60e2e5666a7" target='_blank'>
              Structural insights into chemoresistance mutants of BCL-2 and their targeting by stapled BAD BH3 helices
              </a>
            </td>
          <td>
            Thomas M. DeAngelo, Utsarga Adhikary, Kyle J. Korshavn, H. Seo, Clara R. Brotzen-Smith, Christina M Camara, S. Dhe-Paganon, Gregory H. Bird, T. Wales, L. Walensky
          </td>
          <td>2025-09-29</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Radiotherapy (RT) is a crucial treatment for hepatocellular carcinoma (HCC); however, resistance to radiation remains a major challenge to its effectiveness. Circular RNAs (circRNAs), traditionally considered stable non-coding RNAs, have recently gained attention for their potential to encode proteins or peptides that play crucial roles in HCC radioresistance. In this study, circSNX25 (has_circ_0004874) was identified as a novel circRNA with coding potential related to radiation resistance in HCC through circRNA sequencing, Northern blotting, and mass spectrometry (MS). We demonstrated that SNX25-215, a novel protein encoded by circSNX25, rather than the parental circSNX25 itself, promotes resistance to radiotherapy in HCC cells both in vitro and in vivo. Co-immunoprecipitation (co-IP) and molecular docking revealed that amino acids H207 and E214 of SNX25-215 are critical for binding to Golgi to ER traffic protein 4 homolog (GET4). This interaction inhibits the binding of BCL2-associated athanogene 6 (BAG6) to GET4, thereby exposing the nuclear localization sequence (NLS) of BAG6 and facilitating its nuclear translocation. This, in turn, enhances DNA damage repair, ultimately increasing resistance to ionizing radiation (IR) in HCC cells. Importantly, elevated levels of SNX25-215 lead to nuclear localization of BAG6, endowing HCC cells with radioresistant activity, which is further supported by clinical evidence. Our findings highlight the potential of circSNX25 as a prognostic biomarker and a therapeutic target for overcoming radioresistance in HCC. This study provides deeper insights into the roles that circRNA-encoded proteins play in HCC radioresistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16daa4d6e57dc11c1f72d49cb0e599242e4afab2" target='_blank'>
              Circular SNX25 encoded radioresistance augmenter facilitates DNA damage repair in hepatocellular carcinoma by targeting BAG6-GET4 interaction
              </a>
            </td>
          <td>
            Shuping Li, Ying Gao, Yuhao Tang, Aoran Dong, Shuifang Hu, Ruizhi Wang, Xiaoyong Gong, Zhenwei Peng
          </td>
          <td>2025-10-21</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The disruption of the epigenetic mechanisms of gene expression regulation due to the emergence of pathogenic variants in genes-encoding elements of epigenetic machinery leads to the development of chromatinopathies. This group of hereditary diseases includes 179 syndromes, some of which present with overlapping phenotypes. Despite the variety of approaches to molecular diagnostics of chromatinopathies, it is not always possible to establish the molecular diagnosis by traditional methods; thus, the issue of optimizing diagnostic algorithms remains relevant. One of the most rapidly expanding areas of post-genomic molecular diagnostics is episignature detection, which relies on genome-wide DNA methylation analysis. This article aims to represent an original approach to indirect diagnostics of chromatinopathies on the example of Rubinstein–Taybi syndrome 1, which is based on the analysis of the methylation level of a limited set of loci designed to reproduce its classic episignature. In the current study, we apply two methods of targeted quantitative analysis of DNA methylation, which are relatively accessible and can be integrated into diagnostic practice. We demonstrate that Rubinstein–Taybi syndrome 1 episignature may be successfully reduced to a single locus of human genome, and that quantitative bisulfite DNA methylation analysis at this locus allows accurate identification of the Rubinstein–Taybi syndrome 1 patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3537a5166624f1652437f23c675d33be54fbf812" target='_blank'>
              DNA Methylation at a Single Locus of Human Genome Accurately Recapitulates Episignature of CREBBP-Related Rubinstein–Taybi Syndrome
              </a>
            </td>
          <td>
            Olga A. Zemlianaia, A. Kalinkin, A. Tanas, Anna V. Efremova, I. Volodin, O. Ismagilova, Anton S. Smirnov, D. Zaletaev, Marina V Nemtsova, S. Kutsev, V. Strelnikov
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background Proto-oncogenes are abnormally activated in nearly all types of tumors. However, the epigenetic mechanism of proto-oncogene activation has not yet been well elucidated. Methods The present study involved the construction of a double-stranded cDNA library derived from gastrointestinal cancer cells, followed by high-throughput genome sequencing to select the c-MYC gene associated with colorectal cancer. Through RACE analysis, we identified the antisense RNA MYC-AS1 and its complete sequence. By investigating the cellular functions, expression of MYC-AS1, elucidating its interaction mechanism with the c-MYC gene, and exploring the impact of DNA methylation on MYC-AS1 expression, we uncover the fundamental principles and regulatory mechanisms underlying colorectal cancer development. Results Here, we show that a subset of proto-oncogenes,including c-MYC, possess antisense RNAs. Upregulation of c-MYC in cancer tissues was attributed to the silencing of its antisense RNA MYC-AS1 via DNA hypermethylation. MYC-AS1 RNA markedly inhibited the proliferation of cancer cells in vitro and impeded tumor growth in nude mice in vivo by repressing the expression of c-MYC via an RNAi mechanism. MYC-AS1 RNA bound directly to the HuR protein in the cytoplasm, enhancing the RNA stability of MYC-AS1. Furthermore, MYC-AS1 inhibited c-MYC-targeted gene LDHA expression. Unlike the well-characterized oncogenic long noncoding RNA PVT1, which is coamplified with MYC and enhances its stability, MYC-AS1 is epigenetically silenced and functions as a tumor suppressor through an RNAi mechanism, revealing a distinct layer of MYC regulation. Conclusion Our work provides a novel mechanism by which c-MYC is activated in cancer cells by epigenetic silencing of its antisense RNA, which functions as a tumor suppressor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c20b565d2f250856d2e173d09028c6873b6e2cb" target='_blank'>
              A novel mechanism by which c-MYC is aberrantly activated by epigenetic silencing of its antisense lncRNA in colon cancer
              </a>
            </td>
          <td>
            Xuming Hu, Ye Wei, Meiying Zhang, Chunfeng Dou, Liping Wang, Gul Zaib, Huixian Wu, Wang Guo, Xiaoyuan Wang, Shihao Chen, Qi Xu, Mingzhou Guo, Hengmi Cui
          </td>
          <td>2025-10-21</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Of the thousands of genes and substrates identified in KRAS-mutant signaling networks in pancreatic ductal adenocarcinoma (PDAC), more than 200 are transcription factors, implying extensive and complex transcriptional regulation. However, we observed that genetic suppression of the transcription factor MYC alone was sufficient to phenocopy the effect of KRAS suppression in signaling, growth, and metabolic processes in PDAC cells. We determined the gene transcription changes caused by acute suppression of MYC function in KRAS-mutant PDAC cell lines and performed dependency map and pathway analyses on the affected gene sets. The expression of 1685 genes was increased upon suppression of MYC, and this gene set may comprise the bulk of the MYC-regulated genes essential for PDAC growth. In contrast, the 1325 genes whose expression was inhibited may comprise a compensatory response to oncogenic stress, mediated in part by the GTPase RHO. MYC-dependent transcriptional activity was largely ERK dependent, and almost one-third of ERK-regulated genes were also regulated by MYC in PDAC cells. Furthermore, chemical proteomic profiling revealed MYC-regulated protein kinases that can be targeted therapeutically. Together, these data provide a molecular portrait of MYC-dependent signaling that encompasses potentially exploitable mechanisms for treating PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c2dfd12aa532b6fe57212e6285b3679bdee8785" target='_blank'>
              Defining the MYC-regulated transcriptome and kinome that support KRAS- and ERK-dependent growth of pancreatic cancer.
              </a>
            </td>
          <td>
            Priya S. Hibshman, Clint A. Stalnecker, Jeffrey A. Klomp, Kristina Drizyte-Miller, Jennifer E. Klomp, A. C. Edwards, Lily M Pita, Richard G Hodge, J. Diehl, Ryan D. Mouery, Brandon L. Mouery, Kayla R Snare, Andrew M. Waters, Sen Peng, Natalie K Barker, M. Pierobon, N. Rashid, Nhan L Tran, Laura A Herring, Lee M. Graves, Emanuel F. Petricoin, Kirsten L. Bryant, Adrienne D. Cox, C. Der
          </td>
          <td>2025-09-30</td>
          <td>Science signaling</td>
          <td>0</td>
          <td>124</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36d5b919ff4c0f1e65c48676350abe4ef162d1ad" target='_blank'>
              Pan-cancer single-cell and spatial transcriptomics implicate cancer-associated fibroblasts in neutrophil immunosuppressive phenotypic transitions and immunotherapy resistance
              </a>
            </td>
          <td>
            Zhiyu Guo, Xujia Li, Lingli Huang, Yue Yan, Mengge Gao, Jinsheng Huang
          </td>
          <td>2025-10-10</td>
          <td>Functional & Integrative Genomics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="BACKGROUND
Cyclin-dependent kinase 12 (CDK12) regulates general gene transcription elongation, and plays multiple roles in RNA splicing, DNA damage-response, cell cycle and genomic stability. However, transcriptional partners that guide CDK12-specific gene programs have not been identified. Genomic alterations in CDK12 have been observed in multiple cancers, exhibiting both pro-tumorigenic and tumor-suppressive functions, suggesting a context-dependent mechanism of action.


METHODS
CDK12 copy number alterations and gene expression levels were analyzed in matched primary and brain metastatic patient tumors. Clinical significance was assessed by immunohistochemistry in a large cohort of primary breast cancer patient tumors. RNA-sequencing, ChIP-sequencing and molecular studies were conducted to explore CDK12's mechanism of action, and pharmacological studies were performed both in vitro and in vivo using models of advanced (endocrine-resistant and metastatic) estrogen receptor positive (ER+) disease.


RESULTS
CDK12 amplifications and gene overexpression were observed in brain metastatic tumors. In ER+ primary patient tumors, high CDK12 protein expression significantly associated with poor overall survival, particularly within the ER+/HER2-negative group. In ER+ endocrine resistant models, CDK12 regulated estrogen signaling pathways, with ER/MED1 identified as the master transcriptional complex directing CDK12-specific pro-tumorigenic gene programs. Pharmacological inhibition of CDK12 significantly reduced viability in endocrine resistant and metastatic cell and organoid models in vitro, and decreased metastatic spread in vivo.


CONCLUSION
This work describes a novel mechanism for CDK12, suggesting a potential vulnerability in ER+ breast cancer. These findings provide a basis for further investigation into the role of CDK12 inhibition as a therapeutic approach, particularly in advanced disease settings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af116245324e2c76800300a12fb01f632badb6d3" target='_blank'>
              Targeting CDK12 disrupts estrogen-receptor chromatin recruitment and ER-MED1 transcription in advanced ER+ breast cancer.
              </a>
            </td>
          <td>
            Daniela Ottaviani, Mihaela Ola, A. Allotta, Yasmine Maati Chaibi, Seán Hickey, Petra Jagust, Nicola Cosgrove, S. Cocchiglia, F. Bane, Ramón Fallon, Gordon Daly, A. Hegarty, Lance Hudson, Katherine Sheehan, Shannon Kalsi, Stephen Shovlin, Aoibhín M Powell, Ashwani K. Bahl, Ed Ainscow, S. Oesterreich, Adrian V. Lee, F. Couch, Arnold D K Hill, D. Varešlija, Leonie S Young
          </td>
          <td>2025-10-15</td>
          <td>Journal of the National Cancer Institute</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="
 Clustered regularly interspaced palindromic repeats (CRISPR) gene editing, known for its precision, is revolutionizing tumor research and treatment. This article covers the evolution of the CRISPR system from Cas9, which produces DNA double‐strand breaks, to Cas12, Cas13, base editors, and prime editors. These breakthroughs allow DNA/RNA editing, transcriptional regulation, and epigenetic reprogramming. The study encompasses CRISPR's diverse applications in tumor research, including high‐throughput screening of cancer driver genes, identification of synthetic lethal targets, and analysis of drug resistance pathways across various cancer types. It highlights CRISPR's importance in therapeutic tactics, such as targeting oncogene inactivation, restoring tumor suppressor gene activity, and engineering immune effector cells, such as CAR T and NK cells, to increase persistence, cytotoxicity, and resistance to exhaustion. This review also examines CRISPR's role in modifying the tumor microenvironment (TME), encompassing the regulation of immune checkpoints, metabolic reprogramming, and the targeted eradication of specific immunosuppressive cells. Off‐target consequences for both delivery and efficiency are discussed, along with cutting‐edge technologies including high‐fidelity Cas variants, AI‐assisted sgRNA design, and stimuli‐responsive delivery systems. Finally, the paper examines how CRISPR can be integrated with combination medicines, multimodal editing methods, and single‐cell and multiomics data to deliver tailored precision‐based interventions. This review explains the molecular logic and translational routes of CRISPR, highlighting its promise to revolutionize precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7f333b854e7e8b8178a62bd98bd7d363744559a" target='_blank'>
              CRISPR Enabled Precision Oncology: From Gene Editing to Tumor Microenvironment Remodeling
              </a>
            </td>
          <td>
            Kailai Li, Peixin Huang, Yue Qian, Anqi Lin, Jingjun He, Junyi Shen, Li Chen, Kai Miao, Jian Zhang
          </td>
          <td>2025-11-05</td>
          <td>Med Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Many chemotherapeutic agents exert their cytotoxic effects primarily by inducing DNA damage. In response to DNA damage, cells activate a signaling cascade known as DNA damage response (DDR) to repair the damage and promote cell survival. Accordingly, the capacity of the DDR in cancer cells is a critical factor that influences their sensitivity to chemotherapy. Here, we identified a role for interferon γ-inducible protein 16 (IFI16) in modulating the DDR and chemosensitivity of breast cancer cells. Depletion of IFI16 in MDA-MB-231 cells conferred resistance to the DNA-damaging agents doxorubicin and 5-fluorouracil, as evidenced by increased cell viability and reduced caspase-3 cleavage compared to control cells. Mechanistically, IFI16 interacted with the MRE11-RAD50-NBS1 complex and disrupted the interaction between NBS1 and ataxia telangiectasia mutated (ATM), a critical step for ATM activation. In vivo, xenograft tumors derived from IFI16 knockout cells exhibited diminished responses to doxorubicin treatment, characterized by decreased apoptotic cell death and reduced expression of DSB marker proteins, such as γH2AX and 53BP1. Furthermore, analysis of breast cancer patient datasets revealed that high IFI16 expression correlated with an improved pathological complete response rate following chemotherapy. Our findings suggest that IFI16 could serve as both a predictive biomarker for chemotherapy response and a potential therapeutic target for enhancing the efficacy of DNA-damaging agents.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e97552038179200f37fe515ddaf27aa8bae8377e" target='_blank'>
              IFI16 Enhances Chemosensitivity of Breast Cancer Cells by Inhibiting DNA Damage Response
              </a>
            </td>
          <td>
            Na-Lee Ka, G. Lim, Seung-Su Kim, Mi-Ock Lee
          </td>
          <td>2025-10-02</td>
          <td>Biomolecules & Therapeutics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Chromosomal inversions shape evolution and are implicated in human disease, yet their effects on genomic variation and health outcomes remain poorly understood. We analyze genome-wide human inversion polymorphisms, contrasting single-event and recurrent loci. Inversion recurrence is validated using structured-coalescent simulations. We show that single-event inversions evolve in near-complete isolation: inverted haplotypes show ∼16-fold lower diversity and strong differentiation from direct haplotypes (median FST = 0.33). By contrast, recurrent inversions maintain gene flow, resulting in similar diversity across orientations and ∼4-fold lower differentiation. We further find marked differences in coding sequence conservation between single-event and recurrent inversions. Using the NIH All of Us biobank, we impute inversions and identify four inversions with significant disease associations. Notably, the 17q21 inversion is associated with reduced risk of cognitive decline (OR=0.919) and breast cancer (OR=0.910) but with increased obesity risk (OR=1.097), consistent with pleiotropic selection. These findings establish inversions as major drivers of human genetic diversity and disease, with evolutionary outcomes critically dependent on recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/219c7d990ca63e41240349f098c120c6ba543f70" target='_blank'>
              Genome-wide diversity of chromosomal inversions and their disease relationships
              </a>
            </td>
          <td>
            Scott C. Sauers, Bailey Huepenbecker, PingHsun Hsieh
          </td>
          <td>2025-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Poly(ADP-ribose) glycohydrolase (PARG), the primary de-PARylating enzyme, plays a critical role in resolving DNA damage. PARG inhibitors (PARGi) are currently in clinical development for patients with tumors exhibiting homologous recombination deficiency (HRD). PARGi induces toxic single-stranded DNA gaps (ssGAPs) in ovarian cancer models by exacerbating S-phase PARylation and causing Okazaki fragment processing (OFP) defects (https://www.pnas.org/doi/10.1073/pnas.2413954121). Similarly, inhibition of USP1, a deubiquitinating enzyme, promotes the accumulation of ssGAPs during replication in BRCA1-deficient cells. Notably, cellular sensitivity to either PARGi or USP1 inhibitor (USP1i) correlates strongly with ssGAP formation, identifying ssGAPs as a predictive biomarker of drug response. Recently CRISPR screens with USP1 inhibitors have identified OFP genes (e.g., PARG, PARP1, FEN1) as USP1i sensitizers. Given the clinical challenges associated with USP1i monotherapy, there is increasing interest in rational combination strategies that maintain efficacy while potentially improving tolerability. Here we show that co-targeting PARG and USP1 induces a novel synergistic interaction in ovarian cancer models by amplifying replication stress and DNA damage. USP1i augmented the PARGi-induced PARylation at DNA replication forks, leading to a greater increase in ssGAPs gaps, compared to treatment with either single agent alone. This dual inhibition led to accumulation of unresolved replication intermediates, culminating in mitotic catastrophe, as demonstrated by increased micronuclei formation and chromosomal aberrations. Importantly, the PARGi and USP1i combination also enhanced tumor cell killing in a subset of PARP-inhibitor-resistant BRCA1-deficient tumors. Mechanistically we identified RAD18-mediated PCNA ubiquitination as a key driver of synergistic lethality. Loss of RAD18 rescued fork-associated PARylation, reduced replication stress, and suppressed ssGAPs thereby mitigating the combined effects of USP1 and PARG inhibition and causing drug resistance. Consistently, USP1 knockout cells exhibited increased sensitivity to PARGi , which was reversed by RAD18 depletion, highlighting RAD18’s essential role in translesion synthesis-mediated ssGAP filling and combination-induced lethality. Notably, USP1i and PARGi also synergized with other ssGAP-inducing agents (e.g., FEN1 inhibitors), suggesting broader combinatorial potential. Finally, we observed synergistic activity in specific HRD patient derived ovarian cancer organoids (PDOs). Our data provides novel mechanistic insights into the therapeutic potential of combining these inhibitors, offering preclinical evidence to support its advancement for treating tumors with and without HRD and addressing PARPi resistance. Ovarian cancer PDOs enable rapid sensitivity testing, with ssGAP accumulation serving as a potential predictive biomarker. This combination strategy holds promise for enhancing treatment efficacy while reducing toxicity, paving the way for more targeted and effective therapeutic options.



 Ramya Ravindranathan, Nicholas William. Ashton, Alexandre Andre. Da Costa, Ozge Somuncu, Ben Lamarre, Sirisha Mukkavalli, Kalindi Parmar, Geoffrey Shapiro, Alan D'Andrea. Synergistic ovarian cancer killing by combination of PARG and USP1 inhibitors: Role of single stranded DNA gaps as a predictive biomarker [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr A010.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c44d105eb035ae20f8668c449943eff1251b5f67" target='_blank'>
              Abstract A010: Synergistic ovarian cancer killing by combination of PARG and USP1 inhibitors: Role of single stranded DNA gaps as a predictive biomarker
              </a>
            </td>
          <td>
            Ramya Ravindranathan, N. Ashton, A. A. D. Da Costa, Ozge Somuncu, Benjamin P Lamarre, S. Mukkavalli, K. Parmar, Geoffrey I. Shapiro, Alan D. D'Andrea
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background: The role of germline genetics in cancer is rapidly evolving beyond risk assessment. While pathogenic variants in high-penetrance genes are established risk factors, their influence on therapeutic response and long-term outcomes, alongside the enigmatic roles of variants of uncertain significance (VUS) and polygenic risk, is a critical area of investigation. This narrative review explores the expanding impact of the germline genome on cancer care.
Objective: This review aims to synthesize current evidence on how germline genetic factors, both established and potential, influence cancer outcomes, comparing trajectories between individuals with and without identified hereditary cancer syndromes.
Main Discussion Points: Key themes include the validated prognostic and predictive utility of high-penetrance genes like BRCA and DNA mismatch repair genes in guiding targeted therapy. The discussion also encompasses the clinical challenges posed by VUS, the emerging potential of polygenic risk scores, and the profound impact of germline genetics on somatic evolution and clonal selection. Significant methodological limitations and the lack of diversity in genomic research are critically analyzed as major constraints within the existing literature.
Conclusion: Evidence strongly supports integrating germline testing into oncology to guide treatment for specific syndromes. However, the broader application of genetics for prognostication requires further validation. Future efforts must prioritize inclusive, large-scale prospective studies to translate the full potential of germline information into equitable, personalized cancer care.`">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ec7d0e7c389a31fbefef9d4b96c4c76466f9cef" target='_blank'>
              INVESTIGATIONAL REVIEW OF GENETICS CANCER OUTCOMES BY DNA REFLECTION IN PEOPLE WITH AND WITHOUT GENETIC CANCER GENES GENETICALLY
              </a>
            </td>
          <td>
            Irfan Ishaque¹, Duaa Sheeraz, Muhammad Samoo², Mariam Sohaib Hassan³, Kamran⁴, Safia Ghulam, Abbas Rasool⁵, Arshad Aziz⁷ Ali⁶
          </td>
          <td>2025-11-04</td>
          <td>Insights-Journal of Health and Rehabilitation</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is one of the most common cancers among malignant tumors, and extensive evidence suggests that the prognosis of various human cancers is associated with DNA methylation. Therefore, we analyzed potential GBM prognostic biomarkers by combining GBM methylation sequencing data and transcriptome sequencing data. Biostatistical analysis of the expression of three RNA types (long non-coding RNA, microRNA, and mRNA) from The Cancer Genome Atlas (TCGA) was performed to determine the correlation between NACAD and GBM prognosis. Additionally, we predicted and validated a competitive endogenous RNA (ceRNA) regulatory network associated with epigenetic DNA methylation modifications. The ceRNA regulatory network (DCTN1-AS1–hsa-let-7a-5p–NACAD) was found to be related to overall survival (OS) and disease-specific survival (DSS) of GBM. Furthermore, we identified NACAD as an independent prognostic factor for both GBM OS and DSS. In summary, comprehensive analysis of methylation and transcriptome data led to the identification of genes and pathways in different GBM subtypes, highlighting prognostic genes and ceRNA regulatory networks, and proposing potential candidates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36d242277602456fb7dfcd53ca176ff548d65ded" target='_blank'>
              Epigenetic and transcriptomic analysis revealed ceRNA regulatory network in glioblastoma multiforme
              </a>
            </td>
          <td>
            Zhenghui Li, Yanqing Li, Jenna Z Liang, Tianxiang Geng, Jing Chen, Yangyang Yuan, Yufang Sui, Wanhong Zhang, Qian Chen, Guowei Tan, Qiao Shan
          </td>
          <td>2025-10-09</td>
          <td>European Journal of Medical Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Cancer remains a leading cause of mortality worldwide, characterized by cells that exhibit 'monster traits' such as uncontrolled proliferation, invasiveness, resistance to apoptosis, and immune evasion. Traditional therapies often target single hallmarks of cancer, leading to limited success and eventual resistance. This paper proposes a novel, integrative therapeutic framework inspired by evolutionary adaptations in species with remarkable cancer resistance, such as elephants (Loxodonta Africana). Elephants possess enhanced TP53-mediated cancer surveillance and DNA repair mechanisms, which contribute to their extremely low cancer incidence. We hypothesize that by synthetically recapitulating these 'anti-monster' factors—such as augmented p53 activity, enhanced DNA repair, and controlled inflammation—in human cells, we can reprogram malignant phenotypes towards benignity. Our proposed strategy combines synthetic biology, gene therapy (including CRISPR/Cas systems), and multi-targeted pharmacological approaches to simultaneously address multiple cancer hallmarks. By leveraging advanced *in vitro* organoid and *in vivo* models, this research aims to pioneer a new paradigm in oncology that moves beyond cytotoxic elimination towards cellular reprogramming and durable cancer prevention, potentially yielding a significant reduction in lifetime cancer risk.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c541ffbed17a7ff84c98c50ab50d257ba1c2dd4" target='_blank'>
              Engineering Cancer Resistance: A Novel Framework Inspired by Evolutionary Adaptations
              </a>
            </td>
          <td>
            Hafiz Muhammadismail Azher Syed
          </td>
          <td>2025-09-29</td>
          <td>Wah Academia Journal of Health and Nutrition</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Epigenetic dysregulation drives many pediatric tumors, including neuroblastoma, an aggressive malignancy that accounts for approximately 15% of childhood cancer deaths. Drugs targeting epigenetic regulators have emerged, with FDA approvals of tazemetostat for patients with metastatic or locally advanced epithelioid sarcoma with incomplete resection and for patients with relapsed or refractory follicular lymphoma with EZH2 mutations. More recently, the menin inhibitor revumenib was FDA approved for children and adults with relapsed/refractory KMT2A-rearranged acute myeloid leukemia. Unfortunately, these drugs are generally not curative, and resistance develops by multiple mechanisms suggesting that combination therapies are needed. Our laboratory recently identified some neuroblastomas, particularly MYCN amplified, that are dependent on the SAGA complex, a transcriptional coactivator complex that regulates gene expression by modifying histones. Knockout of TADA2B, a gene essential to the histone acetyltransferase function of SAGA, or treatment with GSK699, a selective degrader of KAT2A and KAT2B, the canonical catalytic acetyltransferases of SAGA, causes cell cycle arrest and delayed progression in xenograft models of neuroblastoma. In order to nominate combination strategies with SAGA inhibitors, we used orthogonal approaches: DepMap nearest neighbor analysis and a CRISPR sensitizer screen. Utilizing the Broad Institute’s Cancer Dependency Map (DepMap), we identified EZH2, a core member of the PRC2 complex, as a co-dependency with the SAGA complex in neuroblastoma. Furthermore, we conducted a CRISPR/Cas9 screen with a focused sgRNA library (1,350 genes) to identify additional epigenetic sensitizers and drivers of resistance in neuroblastoma cells treated with GSK699. EZH2 knockout scored as sensitizing to GSK699 while knockout of BRD2, a member of the BET family of proteins and an epigenetic reader, scored as rendering resistance to GSK699. The utilization of both in vitro and preclinical studies confirmed that pharmacological inhibition of KAT2A/B using GSK699 and EZH2 using tazemetostat induced a combinatorial inhibitory effect on neuroblastoma tumor proliferation compared to single agent treatment, with marked prolongation of survival in xenograft models with combination therapy. Transcriptional analysis of tumors treated with combination therapy showed increases in epithelial to mesenchymal transition (EMT) programs and myogenesis differentiation pathways, a surprising finding under active investigation. In conclusion, our data demonstrates that inhibition of the SAGA complex in combination with EZH2 inhibition effectively suppresses the proliferation of MYCN-amplified neuroblastoma supporting further evaluation of this combination in models of neuroblastoma, including deeper mechanistic studies.



 Melinda Soeung, Kenneth Ross, Lucy Merickel, Silvi Salhotra, Audrey Taillon, Allen Basanthakumar, Kimberly Stegmaier. Combination Therapeutic Targeting of Chromatin Complexes in Neuroblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr A043.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5216fb3aee2d0a8a7d9f8e5c04d43544aa31dd4b" target='_blank'>
              Abstract A043: Combination Therapeutic Targeting of Chromatin Complexes in Neuroblastoma
              </a>
            </td>
          <td>
            Melinda Soeung, Kenneth Ross, L. Merickel, S. Salhotra, A. Taillon, A. Basanthakumar, K. Stegmaier
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background: Medulloblastoma is an aggressive pediatric brain tumor characterized by marked molecular heterogeneity, which significantly impacts prognosis. The low frequency of genomic mutations in medulloblastoma suggests that alternative mechanisms, such as epigenetic regulation, may play a critical role in its pathogenesis. Methods: Using the EpiFactors database, we investigated the expression of epigenetic regulators in two independent RNA sequencing cohorts [Pediatric Brain Tumor Atlas (PBTA) and Williamson], stratified by molecular subgroups and clinical outcomes. We further analyzed expression heterogeneity at the single-cell level in malignant medulloblastoma cells using single-cell RNA sequencing. Results: Members of the SWI/SNF superfamily were dysregulated across all four molecular subtypes of medulloblastoma. Subtype-specific alterations were also observed: the acetyltransferase complex was shared between Group 3 (with SMARCD3 as a potential marker) and Group 4 (with RBM24 as a potential marker); SWR1, β-catenin/TCF, and protein–DNA complexes were specifically enriched in SHH-MB (with EYA1 and SATB2 as SHH markers); and RSC-type, PRC1, DNA polymerase complexes, and X-chromosome-related factors were enriched in WNT-MB (with FOXA1 and PIWIL4 as WNT markers). An epigenetic score (epi-score), linked to RNA metabolism and S-adenosyl-L-methionine pathways, was developed and identified as an independent adverse prognostic factor. High epi-scores were associated with proliferative, stem-like SHH malignant cells (characterized by G2/M phase, low pseudotime, and high entropy), exhibiting alterations in RNA splicing, DNA recombination, and nuclear division. Conclusions: Expression heterogeneity of epigenetic regulators is closely associated with molecular subgroups and clinical outcomes in medulloblastoma. These findings highlight the role of epigenetic dysregulation in RNA metabolism and tumor progression, particularly in SHH-driven proliferative cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/733b0f36e6a075919ebc25ace84fd8a8035e0b6e" target='_blank'>
              Single-Cell Heterogeneity of Epigenetic Factor Regulation Deciphers Alteration of RNA Metabolism During Proliferative SHH-Medulloblastoma
              </a>
            </td>
          <td>
            R. Francés, Jenny Bonifacio-Mundaca, Íñigo Casafont, Christophe Desterke, Jorge Mata-Garrido
          </td>
          <td>2025-10-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) remains a devastating disease with a median survival of 15 months. Its molecular definition illustrates underlying unstable genomes with copy number variation (chr 7+/10-), epigenetic modification in DNA repair enzymes (MGMT methylation), and changes to telomere maintenance genes (TERT or ATRX), all recognised as key factors in gliomagenesis and patient outcomes. Telomeres protect chromosome-ends from aberrant DNA damage responses (DDR), and telomere erosion has been shown to drive such genome instability and clonal evolution in several cancers. As more innovative platform trial designs emerge, identifying novel markers to better stratify patients and predict response to targeted therapies is a priority. We conducted single molecule telomere length (TL) analysis and long-read fusion sequencing in GBM patients and related this to clinical outcome.



 146 adults with molecular GBMs had tumour TL assessed with high-throughput single telomere length analysis. Assays for extrachromosomal c-circles and TERT promoter mutation status were performed, to assess underlying telomere maintenance mechanisms. Uni- and multi-variable analysis was conducted to assess tumour TL impact on OS and PFS. Single-molecule PCR was used to identify telomere fusions which were further characterised with long-read nanopore sequencing, including in patients with paired samples from index surgery and at recurrence.



 GBMs displayed shortened telomeres (median TL 4.0kb), unrelated to age at diagnosis or MGMT methylation status. Short TL was negatively predictive for OS on multivariable analysis (HR 4.81, p=0.009). This impact was more pronounced in patients with gross total resection at index surgery (HR 21.55, p=0.0074). Long read Fusion-Seq demonstrated widespread telomere fusions in GBMs and revealed microhomology, telomeric deletions and insertions of genomic loci, including from known protein-coding genes with driver mutations, long non-coding RNA, canonical telomeric repeats, and subtelomeric regions. This mutational spectrum suggests utilisation of specific DNA repair pathways in mediating telomere fusion.



 This study provides mechanistic evidence of dysfunctional telomeres in GBMs, with high resolution TL analysis together with MGMT methylation status providing powerful independent prognostic information. The presence of short telomeres in patients with unmethylated MGMT defines a subgroup of GBM patients that may benefit from alternative treatment regimens. The presence of telomere fusions offers direct evidence for significant genomic instability, potentially favouring targeted DDR inhibitors for patients exhibiting this signature. HB is supported by a WCAT fellowship and Brain Research UK. Work in the Baird Lab is supported by Cancer Research UK and US NIH. Thanks go to BRAIN UK and all our patients.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5786e68cc90683f81be8d14b8359cb26641b2fd7" target='_blank'>
              P05.18.B TELOMERE DYSFUNCTION AND FUSIONS IN GLIOBLASTOMA - DEFINING PROGNOSIS AND GENOME INSTABILITY
              </a>
            </td>
          <td>
            H. Bhatt, K. Norris, K. Cleal, M. Hodgson, C. Herbert, W. Gray, K. Kurian, D. Baird
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Telomere homeostasis serves as a key regulatory mechanism linking aging and cancer. While telomere attrition imposes a proliferative barrier by inducing cellular senescence, abnormal telomere elongation circumvents this constraint, thereby granting malignant cells unlimited replicative capacity. This study systematically explores the causal relationship between telomere length and cancer risk, with the goal of elucidating the molecular pathways involved in telomere-driven tumorigenesis. Mendelian randomization (MR) was employed to establish a causal link between telomere length and pan-cancer susceptibility. Multiple MR models were employed to ensure the robustness of the results. Telomere-associated genes were identified through SNPense analysis, followed by Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway enrichment. Key gene regulatory networks were identified and visualized using the MCODE and cytoHubba algorithms. The expression profile of TERT across diverse cancer types was analyzed using TCGA datasets, and its diagnostic potential was evaluated via receiver operating characteristic (ROC) curve analysis. Correlations between TERT expression and immune cell infiltration were further explored. Longer telomere length was significantly associated with an increased risk of 33 cancer types (IVW OR = 1.27–1.41, all P < 0.001). A total of 143 telomere-related genes were identified, with functional enrichment highlighting their involvement in genome integrity and telomere maintenance. TERT emerged as the most influential hub gene (MCC score = 169). Transcriptomic analyses from TCGA demonstrated widespread TERT overexpression in 16 cancers (e.g., CHOL, LIHC, LUAD), a finding corroborated by RT-qPCR validation. TERT exhibited high diagnostic performance (AUC > 0.85 in 16 cancers, peaking at AUC = 0.97 in LUSC). Immune infiltration analysis revealed a positive correlation with Th2 cells (r = 0.42) but negative correlations with dendritic cells (r = − 0.38) and macrophages (r = − 0.31). This study proposes a comprehensive framework linking telomere length regulation to cancer progression through the TERT axis. Telomere dysfunction contributes to tumorigenesis via two key mechanisms: promoting genomic instability and altering the immune microenvironment. These findings offer new insights into telomere-driven oncogenesis and lay a conceptual foundation for precision diagnostics and targeted therapies in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5fbec04f31e44d8506f3f60bd214523d80d9aa3" target='_blank'>
              TERT links telomere length to cancer risk by integrating genomic instability and immune modulation
              </a>
            </td>
          <td>
            Jingwei Zhao, Jiayun Zhu, Kangle Zhu, Mingyang Wang, Wei Gong, Jiandong Wang, Shilei Liu
          </td>
          <td>2025-09-30</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cardiac myxoma, the most common primary heart tumor, remains poorly understood at the molecular level. Here, we combined single-nucleus RNA sequencing, third-generation transcriptomics, and untargeted metabolomics to dissect its origin and pathology. Single-cell analyses demonstrate an endothelial origin driven by aberrant endothelial-to-mesenchymal transition (EndMT), with pseudotime and RNA-velocity tracing a continuum from endothelial-like to mesenchymal-like and metabolically active states. We identify two distinct myxoma subtypes: Subtype 1, marked by MAPK/WNT/EGFR pathway activation, and Subtype 2, characterized by ribosomal and oxidative phosphorylation signatures alongside immune-evasive programs. Third-generation data highlight extracellular matrix remodeling and endothelial signaling, while metabolomics reveal dysregulated purine, nicotinic acid, and nicotinamide metabolism. Notably, MET–PTK2 signaling emerges as a potential driver of tumor initiation and progression. These integrated findings define the cellular architecture and metabolic adaptations of cardiac myxoma and lay the foundation for future interventions. The effect of zingerone on postmenopausal osteoporosis, as modelled in mouse, was assessed The authors combine single-cell transcriptomics and bulk metabolomics to investigate cardiac myxoma. They identify Subtype 1 marked by MAPK/WNT/EGFR pathway activation and Subtype 2, characterized by ribosomal and oxidative phosphorylation signatures">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0702495e091268eccb0b45edf83576207a5dbb0c" target='_blank'>
              Multi-omics data reveal the origin of cardiac myxoma
              </a>
            </td>
          <td>
            Shengzhong Liu, Wanfeng Zhang, Huajun Sun, Chenqing Zheng, Keli Huang, Chengming Fan, Rensheng Lai, Mingzhu Yin, Jie Lan, Xiushan Wu, Longke Ran, Xiaoping Li
          </td>
          <td>2025-09-26</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Colibactin is a small genotoxic molecule of polyketide produced by a subset of enteric bacteria including certain Escherichia coli (E. coli) harbored in the human gut microbiota. Its biosynthesis is governed by a multistep enzymatic process encoded by the polyketide synthase (pks) gene cluster. Colibactin is thought to exert its carcinogenic potential primarily through the induction of DNA interstrand crosslinks (ICLs); however, the precise mechanisms underlying its genotoxicity remain largely unresolved. In this study, we focused on ICL formation and its associated repair pathways to investigate whether colibactin-induced ICLs play a central role in the induction of chromosomal aberrations and inhibition of cell proliferation. HAP1 cells deficient in FANCD2, a gene essential for ICL repair, and their wild-type counterparts were infected with colibactin producing (clb⁺) E. coli strains isolated from a Japanese colorectal cancer (CRC) patient. Following recovery culture, the frequency of micronucleated (MN) cells was assessed. The results showed that FANCD2-deficient cells exhibited a significantly higher frequency of MN cells compared to wild-type cells. Additionally, the cytotoxicity of the clb⁺ strains was evaluated using the XTT assay. FANCD2-deficient cells demonstrated higher sensitivity to the clb⁺ E. coli strains than wild-type cells. These findings suggest that colibactin, produced by clb⁺ E. coli, can play a role in the formation of ICLs, thereby contributing significantly to the induction of chromosomal aberrations and the inhibition of human cell proliferation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5434e698c009fe5d0b86871ed28e1115dcedc77" target='_blank'>
              DNA cross-link repair deficiency enhances human cell sensitivity to colibactin-induced genotoxicity
              </a>
            </td>
          <td>
            M. Kawanishi, Osamu Tsubohira, Ai Ueshima, Yuuta Hisatomi, Yoshimitsu Oda, Michio Sato, Noriyuki Miyoshi, M. Mutoh, Hideki Ishikawa, Keiji Wakabayashi, Takashi Yagi, Kenji Watanabe
          </td>
          <td>2025-09-22</td>
          <td>Genes and Environment</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="LINE-1 retrotransposition is common in human cancers and rearrangements at insertion sites can contribute to cancer-driving oncogene amplifications and promote genome instability. However, the mechanisms underlying rearrangements of L1 retrotransposition intermediates are poorly understood. To address this gap, we developed GFP-based recombination reporter assays to study the formation of L1 retrotransposition-mediated rearrangements. Using these reporters combined with long-read sequencing approaches, we find that L1 retrotransposition intermediates can recombine with distal DNA breaks to generate chromosomal rearrangements. We also find that two distinct L1 insertion intermediates can recombine with each other to generate chromosomal rearrangements. Both types of rearrangements depend on L1-encoded ORF2p endonuclease and reverse transcriptase activities. Using these reporters, we discover that L1 retrotransposition-mediated rearrangements are robustly induced when the recombining sequences share homology and that their formation requires the homologous recombination factor BRCA1. Given the repetitive nature of our genome, these findings highlight the risk of L1 insertion intermediates becoming substrates for aberrant recombination and promoting genome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb644439b03063ebcc926aea5075b191a155a103" target='_blank'>
              L1 insertion intermediates recombine with one another or with DNA breaks to form genome rearrangements
              </a>
            </td>
          <td>
            Carlos Mendez-Dorantes, Jupiter C. Kalinowski, Cheuk-Ting Law, Phillip Schofield, Aidan Burn, Kathleen H. Burns
          </td>
          <td>2025-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="


 Anaplastic lymphoma kinase (ALK) gene fusions are potent oncogenic drivers across diverse malignancies, including non-small cell lung cancer (NSCLC), anaplastic large cell lymphoma (ALCL), breast cancer, and glioma. These fusions lead to constitutive activation of the ALK tyrosine kinase domain (TKD), rendering tumors initially responsive to ALK tyrosine kinase inhibitors (TKIs)—primarily in NSCLC, but also in rare cases such as infant-type hemispheric glioma. However, resistance inevitably emerges, frequently due to acquired single nucleotide variants (SNVs) within the TKD. A significant proportion of these SNVs are classified as variants of uncertain significance (VUSs), highlighting the need for systematic functional annotation to inform precision therapy.



 We employed prime editing to introduce all possible SNVs across the ALK TKD (exons 20-28). High-throughput drug resistance screening was conducted against three generations of ALK TKIs: alectinib (second-generation), lorlatinib (third-generation), and zotizalkib (fourth-generation, previously evaluated in clinical trials). Resistance classification data were further assessed using Kaplan-Meier survival analyses to explore potential prognostic implications.



 A total of 2,953 SNVs (99.2% of all possible variants in exons 20-28) were successfully introduced and profiled. Resistance phenotypes were comprehensively mapped for each variant across the three TKIs. Canonical resistance mutations such as G1202R and L1196M were accurately recapitulated, validating our screening platform. Moreover, several previously uncharacterized VUSs were identified as conferring differential resistance profiles. Structural modeling revealed mechanistic insights into altered TKI binding. These findings aligned with existing clinical and preclinical datasets and expanded the known spectrum of clinically actionable ALK resistance mutations.



 This study presents the most comprehensive functional landscape to date of ALK TKD variants in the context of TKI resistance. Our data provide an invaluable resource for interpreting VUSs, optimizing TKI selection, and advancing personalized treatment in ALK fusion-positive cancers. Furthermore, the approach and insights are broadly applicable to other ALK-rearranged malignancies, including glioma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b6b913fdc6e574ca69abb2b1d6922fc64f2b203" target='_blank'>
              JS04.5.A HIGH RESOLUTION DRUG-RESISTANT MAPPING OF ALK FUSION VARIANTS USING PRIME EDITING
              </a>
            </td>
          <td>
            H. Oh
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c9f14cca351b80c4eaf8fa50610851b494729d1" target='_blank'>
              Noncanonical chromosomal-end-specific telomeric arrays in naturally telomerase-negative yeasts
              </a>
            </td>
          <td>
            Broňa Brejová, Viktória Hodorová, Hana Lichancová, A. Gafurov, Dominik Bujna, Filip Brázdovič, Filip Červenák, Tomáš Petrík, Eva Hegedűsová, M. Jakúbková, M. Neboháčová, Ľubomír Tomáška, Matthias Sipiczki, T. Vinař, J. Nosek
          </td>
          <td>2025-09-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56a3d728c459f75967661fea1d9baa92b4ef2796" target='_blank'>
              Age distinguishes selection from causation in cancer genomes
              </a>
            </td>
          <td>
            David Cheek, Martin Blohmer, Martin A. Nowak, Tibor Antal, Kamila Naxerova
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Genetic and epigenetic processes are intertwined in a complex manner in order to enhance the development of breast cancer by regulating the fundamental cellular processes including DNA repair, proliferation, differentiation and metastasis. This review provides an overview of the findings of animal model experiments to explain the causes of the significant changes in genetic aspect including RAD51C mutations and dys-regulation of the BAF chromatin remodeling complex and epigenetic alterations, such as DNA methylation, histone modification, and microRNA dys-regulation. The animal models are where the researcher can study the mechanism by which these changes of the molecules cause tumorigenesis, stimulate tumor growth and metastatic potential. The data received in the course of such studies denotes the dynamism and reversibility of the epigenetic processes which may be utilized in treating diseases by selectively regulating the expression of the said molecules with an aim of modifying DNA methylation, histone-modifying enzymes, and miRNAs. Integration of genetic and epigenetic understanding will not only enhance our understanding of breast cancer biology, but also design the precision therapies, and also expose potential biomarkers that can be utilized to identify and diagnose the disease at the earliest phase. In total, the significance of the animal-based research in the context of finding mechanistic information and design interventions to limit the occurrence of breast cancer is highlighted in this review as a foundation of the subsequent use of the translation research and clinical interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/615bf75efa81e4b14beb7b6d9c561e55199ae645" target='_blank'>
              Genetic and Epigenetic Factors in Breast Cancer Progression
              </a>
            </td>
          <td>
            Akash Anil Podutwar, Purva Chandorkar, Swati Changdeo Jagdale
          </td>
          <td>2025-10-18</td>
          <td>Journal of Pharmacology, Genetics and Molecular Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Essential epigenetic dependencies have become evident in many cancers. Based on functional antagonism between BAF/SWI/SNF and PRC2 in SMARCB1-deficient sarcomas, we and colleagues recently completed the clinical trial of the EZH2 inhibitor tazemetostat, leading to its FDA approval. However, the principles of tumor response to epigenetic therapy in general, and tazemetostat in particular, remain unknown. First, using functional genomics of patient tumors and diverse experimental models, we define molecular mechanisms of tazemetostat resistance in SMARCB1-deficient sarcomas and rhabdoid tumors. We found distinct classes of acquired mutations that converge on the RB1/E2F axis and decouple EZH2-dependent differentiation and cell cycle control. This allows tumor cells to escape tazemetostat-induced G1 arrest despite EZH2 inhibition, and suggests a general mechanism for effective EZH2 therapy. We provide prospective biomarkers for therapy stratification, including the cell cycle-regulated PRICKLE1 gene. To circumvent tazemetostat resistance and improve response in vitro and in vivo we develop the rational combination of tazemetostat with barasertib, to target the downstream cell cycle kinase AURKB. Second, through transcriptomic inference of SMARCB1-deficient tumor cells, we predicted ATR as a target to overcome resistance to EZH2 inhibition. We found that EZH2 inhibition induces DNA damage, at least in part via upregulation of the transposase-derived PGBD5 nuclease. Finally, we establish a second rational combination strategy for increased anti-tumor efficacy between tazemetostat and the ATR inhibitor, elimusertib, by leveraging the synthetic lethal interaction with PGBD5-dependent DNA damage repair signaling via ATR. Thus, the epigenetic therapy, EZH2 inhibition, induces a targetable collateral vulnerability through activation of the PGBD5-ATR axis. This establishes a mechanistically rational combination strategy for targeting EZH2 and ATR in SMARCB1-deficient cancers based on an EZH2-PGBD5-ATR synthetic lethal axis and is immediately translatable to clinical trials for patients with these aggressive malignancies.



 Helen S Mueller. Overcoming clinical resistance to EZH2 inhibition using rational epigenetic combination therapy [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr A048.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e20005fa9b7768828704bae3a839702d20b1fe7d" target='_blank'>
              Abstract A048: Overcoming clinical resistance to EZH2 inhibition using rational epigenetic combination therapy
              </a>
            </td>
          <td>
            Helen S. Mueller
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 The mechanisms governing the progression of IDH-wildtype glioblastoma (GBM) following standard-of-care therapy remain incompletely understood. While recurrence is nearly universal, the cellular and molecular trajectories that underpin this process - particularly the dynamics of tumor and microenvironmental compartments - have yet to be fully resolved.



 To dissect the longitudinal evolution of the GBM ecosystem, we analyzed paired primary and recurrent tumor specimens from 59 patients. We employed single-nucleus RNA sequencing to profile transcriptomic states at single-cell resolution and integrated these data with bulk DNA sequencing to assess genomic alterations. This approach enabled a comprehensive interrogation of cellular heterogeneity and malignant state transitions over time.



 Across the cohort, the most consistent feature at recurrence was a decreased proportion of malignant cells, accompanied by a reciprocal expansion of non-malignant glial and neuronal cell populations within the tumor microenvironment (TME). While the dominant malignant cell state often differed between primary and recurrent samples, no state was uniquely associated with a specific disease stage. Moreover, no singular evolutionary trajectory characterized the cohort as a whole. Instead, subsets of patients exhibited enriched and partially convergent state transitions. Notably, shifts in malignant cell states were mirrored by concurrent remodeling of the TME, implicating a tightly interwoven pattern of tumor-microenvironment co-evolution.



 These findings reveal diverse and patient-specific evolutionary trajectories in IDH-wildtype GBM, shaped by both therapeutic pressure and microenvironmental context. Our study provides a reference framework for understanding longitudinal GBM dynamics and highlights the importance of ecosystem-level interactions in driving recurrence.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e738ce34652886a4547b3d5a53d1b822511571f" target='_blank'>
              OS05.2.A DECIPHERING THE LONGITUDINAL TRAJECTORIES OF GLIOBLASTOMA ECOSYSTEMS BY INTEGRATIVE SINGLE-CELL GENOMICS
              </a>
            </td>
          <td>
            A. Spitzer, K. Johnson, M. Nomura, L. Garofano, D. Nehar-Belaid, N. Galili Darnell, A. Greenwald, L. Bussema, Y. Oh, F. Varn, F. D’Angelo, S. Gritsch, K. Anderson, S. Migliozzi, L. G. Gonzalez Castro, T. Chowdhury, N. Robine, C. Reeves, J. Park, A. Lipsa, F. Hertel, A. Gołȩbiewska, S. Niclou, L. Nusrat, S. Kellett, S. Das, H. Moon, S. Paek, F. Bielle, A. Laurenge, A. D. Di Stefano, B. Mathon, A. Picca, M. Sanson, S. Tanaka, N. Saito, D. M. Ashley, S. Keir, K. L. Ligon, J. Huse, W. Yung, A. Lasorella, A. Iavarone, R. Verhaak, I. Tirosh, M. Suvà
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Disruptor of Telomeric Silencing 1-Like (DOT1L) has emerged as a critical epigenetic regulator in cancer, primarily because of its role as the sole histone methyltransferase responsible for histone H3 at lysine 79 methylation. This modification affects transcriptional activation, DNA repair, and cell cycle progression. Its dysregulation is associated with both hematological and solid tumors. In Mixed-lineage leukemia (MLL)-rearranged leukemia, DOT1L maintains aberrant gene expression patterns at loci such as HOXA and MEIS1, supporting leukemic stem cell survival and driving oncogenesis. In solid tumors, DOT1L influences diverse processes, including epithelial-mesenchymal transition, angiogenesis, and cell cycle regulation, contributing to tumor growth and metastasis. Therapeutic strategies targeting DOT1L using inhibitors, such as EPZ-5676, have shown promise in preclinical and clinical studies, highlighting their potential as versatile targets for precision oncology. This review summarizes the recent findings on DOT1L’s involvement in cancer development and its potential as a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/652c9fcc6ab6b96caf0c41342c1311e10db0c889" target='_blank'>
              DOT1L as a Therapeutic Target: Insights into Epigenetic Regulation and Cancer Treatment
              </a>
            </td>
          <td>
            Hee-Joo Choi, Minh Tuan Nguyen, Boram Kim, Seung Hwan Kim, Jaehee Kim, Jeong‐Yeon Lee, Mikyung Park
          </td>
          <td>2025-10-31</td>
          <td>Biomolecules & Therapeutics</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Acute myeloid leukemias (AMLs) containing TP53 (p53) mutations are routinely treated with decitabine or 5-azacytidine which deplete DNA methyltransferase 1 (DNMT1)('hypomethylating agents', HMA). Unfortunately, resistance/relapse, characterized by preserved DNMT1, is rapid. HMA are pyrimidine analogs, and to deplete DNMT1, must compete with endogenous pyrimidines. These were substantially increased in HMA-resistant versus parental AML cells, together with upregulation of carbamoyl-phosphate-synthetase-2/aspartate transcarbamylase/dihydroorotase (CAD) that rate-limits de novo pyrimidine synthesis. Moreover, TP53-mutated AMLs appeared primed for such resistance with baseline higher CAD. Pyrimidine synthesis can be depowered by using the BCL2-inhibitor venetoclax to release BAX to depolarize mitochondrial membranes. However, BAX, a p53-target gene, was ~2-fold less expressed in TP53-mutated vs TP53-wildtype cells, and venetoclax impacts were correspondingly limited. Alternatively, pyrimidine synthesis can be inhibited directly at dihydroorotate dehydrogenase (DHODH) using the clinical drug teriflunomide. Contrasting with venetoclax, teriflunomide decreased pyrimidines several-fold, restored DNMT1-depletion, and cytoreduced HMA-resistant TP53-mutated AML cells via p53/apoptosis-independent terminal lineage-maturation. This non-cytotoxic pathway preserved viability and proliferation of normal hematopoietic stem/progenitor cells (HSPC). Inhibiting pyrimidine synthesis triggered automatic increases in pyrimidine salvage, such that schedules for teriflunomide combination with HMA, taken-up by salvage, mattered: in mice with TP53-mutated AML, teriflunomide scheduled day-before HMA was more efficacious than same-day or day-after. Chronic teriflunomide exposure paradoxically increased pyrimidines via sustained compensatory pyrimidine salvage, conferring resistance rather than sensitivity to HMA. In sum, DNMT1- and DHODH-targeting, administered by timed-intermittent (metronomic) schedules, can circumvent genetic-resistance caused by TP53-mutations, and adaptive-resistance caused by metabolic homeostasis, without cytotoxicity to normal HSPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b648fdff4610f9244486c913d8f3bc8335aa5ac" target='_blank'>
              Combining drugs that bypass p53 to treat TP53-mutated leukemias.
              </a>
            </td>
          <td>
            Sudipta Biswas, Zeinab Zahran, Xiaorong Gu, Lisa Cardone, Remuna Marti, Nour Mouannes, Maximillian Stich, Akriti G Jain, Kateryna Fedorov, Benjamin K Tomlinson, M. Goldfinger, Amit Verma, Yogenthiran Saunthararajah
          </td>
          <td>2025-10-08</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a9e3d7e8367b6848517e2b91b0fabe3f4c124a0" target='_blank'>
              Identification of a novel PML exon 6 splice variant in atypical PML::RARα transcripts in acute promyelocytic leukemia.
              </a>
            </td>
          <td>
            Haimin Chen, Meihong Chen, Zhongjie Yang, Shuzhen Liao, Yun Lin, Linlin Yan
          </td>
          <td>2025-09-26</td>
          <td>Annals of hematology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Phenotypic plasticity is a recognized mechanism of therapeutic resistance in prostate cancer (PCa), however current knowledge of driver mechanisms and therapeutic interventions are limited. Using genetically engineered mouse models (GEMMs) devoid of Pten and Rb1, we previously demonstrated the chromatin reprogramming factor enhancer of zeste homolog 2 (EZH2) as an important regulator of alternative transcription programs promoting phenotypic plasticity. Here, using a multi-omics approach we find that EZH2 regulates a multilineage cell state dependent on the activation of translation and regulation of RNA maturation. Combined chemical inhibition of EZH2 and PI3K/mTORC1 resulted in superior anti-tumor activity in murine and human phenotypic plastic models and was most significant when this combination was used with castration. Moreover, we elucidated that the regulation of cellular state transition by EZH2 inhibition requires activation of the RNA degrader, Tristetraprolin (TTP). Together, these data indicate phenotypic plasticity dependence on coordination between EZH2, TTP and mTORC1 signaling that represent novel therapeutic dependencies for this lethal PCa phenotype.



 Beatriz German, Leigh Ellis. EZH2 regulates cell identity in neuroendocrine-like prostate cancer by driving alternate transcription via activation of translation and RNA processing regulation [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C033.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e13a2ff65980383cdcce076c99675421b3f51723" target='_blank'>
              Abstract C033: EZH2 regulates cell identity in neuroendocrine-like prostate cancer by driving alternate transcription via activation of translation and RNA processing regulation
              </a>
            </td>
          <td>
            Beatriz German, Leigh Ellis
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Transposable elements (TEs), once regarded as genomic “junk,” are now recognized as powerful regulators of gene expression, genome stability, and innate immunity. In the context of neurodegeneration, particularly Amyotrophic Lateral Sclerosis (ALS), accumulating evidence implicates TEs as active contributors to disease pathogenesis. ALS is a fatal motor neuron disease with both sporadic and familial forms, linked to genetic, epigenetic, and environmental factors. While coding mutations explain a subset of cases, advances in long-read sequencing and epigenomic profiling have unveiled the profound influence of non-coding regions—especially retrotransposons such as LINE-1, Alu, and SVA—on ALS onset and progression. TEs may act through multiple mechanisms: generating somatic mutations, disrupting chromatin architecture, modulating transcriptional networks, and triggering sterile inflammation via innate immune pathways like cGAS-STING. Their activity is normally repressed by epigenetic regulators, including DNA methylation, histone modifications, and RNA interference pathways; however, these controls are compromised in ALS. Taken together, these insights underscore the translational potential of targeting transposable elements in ALS, both as a source of novel biomarkers for patient stratification and disease monitoring, and as therapeutic targets whose modulation may slow neurodegeneration and inflammation. This review synthesizes the current knowledge of TE biology in ALS; integrates findings across molecular, cellular, and systems levels; and explores the therapeutic potential of targeting TEs as modulators of neurodegeneration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69ff26f0b7276e91620b33b746e298b203260717" target='_blank'>
              The Other Side of the Same Coin: Beyond the Coding Region in Amyotrophic Lateral Sclerosis
              </a>
            </td>
          <td>
            Paola Ruffo, Benedetta Perrone, Francesco Perrone, F. De Amicis, Rodolfo Iuliano, Cecilia Bucci, Angela Messina, F. Conforti
          </td>
          <td>2025-10-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="


 SETD2 is a tumor suppressor gene responsible for trimethylation of H3K36, crucial for DNA mismatch repair and homologous recombination. Novel synthetic lethality agents, ATR inhibitors (ATRi), have demonstrated anti-tumor activity in preclinical models and early clinical trials for pan-tumors. SETD2 and H3K36 deficient cells have been demonstrated to display Micro Satellite Instability (MSI) with increased tumor mutational burden (TMB). Deleterious SETD2 alterations occur in 5-10% of brain tumor patients with glioblastoma, IDH wild type, WHO grade 4, (GBM). These patients have a poor prognosis. This study aims to evaluate targeted treatment strategies in central nervous system (CNS) tumors, by investigating the potential of ATRi for tumors harboring deleterious SETD2 alterations and immuno-therapy for tumors with increased mutational burden, with the objective of improving patient outcomes.



 This study analyzes genomic profiles from 434 patients diagnosed with central nervous system (CNS) tumors between January 1st, 2019, and February 28th, 2025, enrolled in the prospective Neurogenome and CoPPO studies. Rigshospitalet has one of the largest single-institution datasets with comprehensive data on SETD2 alterations derived from whole exome/genome sequencing including structural variants, copy number variations, and RNA expression.



 The study cohort included 434 patients with CNS tumors (166 female, 268 male) with a median age at diagnosis of 59 years (interquartile range: 49-66 years). GBM was the predominant tumor type (86%), followed by astrocytoma, IDH mutated (mut), WHO grade 2-4 (12%) and oligodendroglioma, IDH-mut, 1p/19q co-deletion, WHO-grade 2-3 (1.2%). Other CNS tumors were diagnosed in 6 (1.4%) patients. Deleterious SETD2 alterations were found in 13 (3%) patients: 11 with GBM, 1 astrocytoma IDH mut, and 1 oligodendroglioma. Twelve patients had mutations: 6 premature stop codons, 5 frameshift mutations and 1 splice site variant, and one case with biallelic SETD2 loss. Data on tumor mutational burden (TMB) was assessable for 12 SETD2-deficient patients, none exceeding 10mut/mb; mean TMB was 3.3 mut/mb vs. 6.2 mut/mb in the SETD2-wildtype (wt) group. No SETD2-deficient patients had received temozolomide (TMZ) treatment prior to sampling. By contrast, some samples in the SETD2-wt cohort were obtained at relapse, which may have inflated TMB due to TMZ treatment. COSMIC signatures were available for 10 SETD2-deficient patients with none having indications of MSI based on COSMIC signature evaluation.



 Loss of SETD2 function in primary CNS-tumors is relatively rare and was predominantly driven by mutations, with copy number analysis revealing only 1 biallelic deletion in 434 patients. SETD2 alterations was primarily seen in GBM patients. The SETD2 deficient patients did not exhibit indications of MSI or high TMB. However, the impact of patient relapse samples with TMZ induced mutations in the SETD2-wt cohort is being explored further together with pan-tumor markers for DNA damage repair.



 Tine Østergaard, Christoffer L. Halvorsen, Dorte S. Nørøxe, Vincent N. Fougner, Iben Spanggaard, Kristoffer S. Rohrberg, Martin Højgaard. Comprehensive genomic profiling in central nervous system tumors reveals low frequency of deleterious alterations in SETD2 without associated increase in mutational burden [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C037.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5615e3697959292b283838db14154a56c4e82cf6" target='_blank'>
              Abstract C037: Comprehensive genomic profiling in central nervous system tumors reveals low frequency of deleterious alterations in
 SETD2
 without associated increase in mutational burden
              </a>
            </td>
          <td>
            Tine Østergaard, Christoffer L. Halvorsen, D. Nørøxe, V. Fougner, I. Spanggaard, Kristoffer S. Rohrberg, M. Højgaard
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="


 Despite treatment advances, ovarian cancer (OC) remains the deadliest gynecological cancer, with a ∼70% mortality rate. OCs initially respond to platinum chemotherapy but often develop resistance. While PARP inhibitors improve relapse-free survival in responsive cases, they do not extend overall survival. This underscores the need for effective combination therapies beyond BRCA status. The lysosomal protein cathepsin L (CTSL) can traffic to the nucleus, where it can regulate DNA repair and serve as a chromatin modifier. We found that nucleoside analog clofarabine (CLF) induces CTSL nuclear trafficking in OC cells. Here we tested the hypothesis that accumulation nuclear CTSL (nCTSL) is a biomarker of sensitivity to DNA damaging therapy in OCs for improved therapeutic options.



 To determine the role of CLF induced nuclear CTSL in DNA damage and its synergy with PARP and CRM1 inhibitors in PARP inhibitor–sensitive and –resistant ovarian cancer (OC) cell lines, ex vivo cultures of ascites-derived patient OC cells (OVA-3D spheroids), primary ovarian tumors, and patient-derived xenograft (PDX) models and validated by colony formation assays. Nuclear CTSL localization was determined in subcellular fractionated lysates by western blots and immunofluorescence cytometry (IFC). Tumor regression was determined in vivo using ex vivo cultures of PDX derived cells.



 In ovarian cancer (OC) cell lines, CLF monotherapy induced nuclear CTSL (nCTSL) in a subset of models classified as CLF-responsive (CLF-r), sensitizing them to the PARP inhibitors olaparib and rucaparib. In CLF-non-responsive (CLF-nr) models, nCTSL induction and therapeutic synergy were observed only with the CLF+olaparib combination. Synergy was detected in 47% of CLF-r and 24% of CLF-nr patient-derived ascites samples (OVA). Mechanistically, CLF induced nuclear import of CTSL via KPNB1 in CLF-r cells, while this import was triggered by the CLF+olaparib combination in CLF-nr cells. CLF also downregulated the nuclear export protein CRM1 (XPO1) in both groups. Knockdown of CTSL in either CLF-r or CLF-nr cells abrogated sensitivity to the combination. Notably, in 29% of OVA samples classified as CLF-resistant (CLF-Res), CLF failed to suppress CRM1 expression. In these cases, CRM1 inhibition with KPT8602 restored CLF+PARP inhibitor synergy. In vivo, CLF-r and CLF-nr PDX models treated with the combination exhibited increased DNA damage, reduced tumor burden, and extended survival.



 These findings identify nuclear CTSL (nCTSL) induction as a potential biomarker of response to DNA-damaging therapies. The combination of CLF and olaparib represents a promising strategy to overcome drug resistance in ovarian cancer by promoting CTSL nuclear localization and enhancing DNA damage.



 Prabhu Thirusangu, Ling Jin, Upasana Ray, Anya Zhao, Julie Staub, Xinyan Wu, Xiaonan Hou, Jamison L. VanBlaricom, Syed Mohamed Musheer Aalam, Sun-Hee Lee, Ann Oberg, Nagarajan Kannan, John Weroha, Jeremy Chien, Scott H. Kaufmann, Jamie N. Bakkum-Gamez, Viji Shridhar. Role of drug induced nuclear CTSL (nCTSL) in DNA damage response in cancer- therapeutic implications [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A050.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f74528b6924a7e697a49ee1e41cc77afb923b83" target='_blank'>
              Abstract A050: Role of drug induced nuclear CTSL (nCTSL) in DNA damage response in cancer- therapeutic implications
              </a>
            </td>
          <td>
            P. Thirusangu, Ling Jin, Upasana Ray, Anya Zhao, J. Staub, Xinyan Wu, Xiaonan Hou, Ja Vanblaricom, Syed Mohammed Musheer Aalam, Sun-Hee Lee, A. Oberg, Nagarajan Kannan, John S Weroha, Jeremy Chien, Scott H. Kaufmann, Jamie N. Bakkum-Gamez, V. Shridhar
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="


 Cancer remains a leading cause of death for children and adolescents, underscoring the need for therapeutic advances. The PRISM assay enables rapid profiling of emerging anti-cancer agents by barcoding and pooling >900 genetically and functionally characterized cancer cell lines. The PRISM assay reveals not only drug potency and selectivity, but also genomic correlates of sensitivity and predictive biomarkers. To better represent pediatric, hematopoietic, and rare cancers in our cell collection, we expanded the PRISM panel in collaboration with the Pediatric Cancer Dependencies Accelerator project by adding 109 new cell lines, including 21 pediatric models, 60 hematopoietic lines, and 14 previously unrepresented subtypes. In many pediatric cancers, epigenetic proteins are frequently altered and are promising therapeutic targets; therefore, we screened clinically relevant compounds targeting epigenetic proteins at 8 concentrations for 5 days. The SMARCA2/4 inhibitor, BRM/BRG1 ATP Inhibitor-1, and the BRD9 degrader, CFT8634, showed strong activity in atypical teratoid rhabdoid tumor (ATRT) models. In contrast, the SMARCA2-selective degrader, ACBI2, was less active in ATRT, despite similar potency. Further work will investigate the mechanistic basis of this differential activity. As some small molecules developed against epigenetic targets may require a longer incubation period to enact their cytotoxic effects, we developed a 10-day PRISM assay, twice as long as our standard 5-day assay. We tested the EZH2 inhibitor, mevrometostat, in both the 5-day and 10-day assay and observed greater potency, selectivity, and stronger correlations with EZH2 dependency in the 10-day assay. Finally, given the prevalence and utility of combination treatments for pediatric cancers, we also wanted to probe the drug combination landscape. We screened a collection of combinations in a 5x5 dose matrix format, focusing on inhibitors targeting DNA repair proteins (ATR, ATM, WEE1, CHEK1, PolTheta, USP1, DNA-PK, Topo1) in combination with talazoparib or gemcitabine. We observed synergy in pediatric subtypes for many but not all of the combinations tested. We identified selective synergies in pediatric subtypes, including Ewing sarcoma with talazoparib-based combinations and B-ALL with low-dose talazoparib + ATM inhibitor AZD1390. Further analysis of this combination dataset will refine our understanding of specific contexts in which we observe synergy and reveal biomarkers of synergistic effects. Together, these advances position PRISM to accelerate pediatric cancer research by expanding disease-relevant models and enabling long-duration single-agent screening and matrix combination screening at scale. All data will eventually be made publicly available on depmap.org, providing a resource for the pediatric cancer research community to explore novel therapeutic opportunities.



 Colleen T. Harrington, Antonella Masciotti, Ursula Widocki, Ashish Bino George, Blanche Ip, Tenzin Sangpo, Li Wang, Laura Doherty, Aydin Golabi, Rachael Barry, Emily Reeves, John Davis, Melissa Ronan, Matthew G. Rees, Jennifer A. Roth. Defining the epigenetic drug landscape and rational combinations for pediatric cancer using the PRISM screening assay [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr A026.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b814cec2f64b38043334e1b8daf5cb66a0a058be" target='_blank'>
              Abstract A026: Defining the epigenetic drug landscape and rational combinations for pediatric cancer using the PRISM screening assay
              </a>
            </td>
          <td>
            Colleen T. Harrington, Antonella Masciotti, Ursula Widocki, Ashish Bino George, Blanche C. Ip, Tenzing Sangpo, Li Wang, Laura Doherty, Aydin Golabi, Rachael Barry, Emily Reeves, John F. Davis, Melissa Ronan, M. Rees, Jennifer Roth
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) is characterized by the expansion of hematopoietic stem and progenitor cells harboring somatic mutations, which confers an increased risk of hematologic malignancies and cardiovascular disease. Among CH-associated mutations, mutations affecting splicing factors (SFs), including splicing factor 3b subunit 1 (SF3B1), serine/arginine-rich splicing factor 2 (SRSF2), U2 small nuclear RNA auxiliary factor 1 (U2AF1), and zinc finger CCCH-type, RNA binding motif and serine/arginine rich 2 (ZRSR2), play a unique role in promoting clonal expansion and leukemogenesis. In this review, we summarize recent findings on the role of SF mutations in CH progression, their interplay with other mutations (e.g., DNA methyltransferase 3 alpha (DNMT3A), ten-eleven translocation methylcytosine dioxygenase 2 (TET2) and isocitrate dehydrogenase 2 (IDH2)), and their impact on hematopoietic homeostasis. Epidemiological studies have demonstrated that SF-mutant CH exhibits an accelerated clonal expansion compared to other CH clones. Furthermore, murine models suggest that SF mutations alone do not inherently confer a growth advantage for clonal expansion but rather enhance disease phenotypes when co-existing with epigenetic mutations, such as IDH2 and TET2. These findings suggest that SF mutations contribute to CH expansion and malignant transformation through a synergistic interplay with other mutations and external factors such as inflammation. Given the clinical significance of SF mutations, ongoing research is focused on developing targeted therapies that modulate aberrant RNA splicing and prevent CH-driven leukemogenesis. Understanding the mechanisms underlying mutant spliceosome-mediated CH expansion may provide novel insights into early detection, risk stratification, and therapeutic interventions in hematologic malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8d9014adbb748e6a4c3fdbe95052f49e04f07e8" target='_blank'>
              The Impact of Splicing Factor Mutations on Clonal Hematopoiesis and Myeloid Neoplasm Progression.
              </a>
            </td>
          <td>
            Takuya Izumi-Tamura, A. Kawachi, Akihide Yoshimi
          </td>
          <td>2025-10-29</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Childhood cancer is rare, with about 1 in 260 children developing cancer before age 20. However, it remains a leading cause of death for children and adolescents worldwide. The 5-year survival rate for childhood cancer in high-income countries exceeds 80%, but globally, the average survival rate is around 37%, highlighting significant disparities across the globe. Despite the life-saving impact of current treatment regimens, long-term side effects and risks are always concerns. Therefore, there is a continuing urgent need for novel therapies to overcome the limitations of existing approaches and improve patient outcomes. Targeted drug therapies that interfere with cancer-causing genes play a vital role in cancer treatment by specifically targeting cancer cells. TFs are primary drivers of gene expression that are critical in various pediatric cancers. Chromosomal rearrangements, involving changes in chromosome structure such as deletions, duplications, inversions, and translocations, can significantly alter TF activity and downstream gene expression. Dysregulation of TFs disrupts gene expression networks and has been strongly linked to the development and progression of many pediatric cancers, making them promising therapeutic targets. Several approaches targeting TFs, including small-molecule inhibitors designed to block TF-DNA binding, TF-cofactor interactions, or their epigenetic regulation, as well as RNA interference, have been developed. More recently, approaches like PROTACs (Proteolysis-Targeting Chimeras) and molecular glue degraders offer new therapeutic possibilities in pediatric cancers. These innovations represent a paradigm shift in pediatric oncology, offering hope for more targeted, less toxic treatment options. This review discusses the critical role of TFs in childhood cancers and emphasizes the need for evolving therapeutic strategies aimed at targeting these key regulators to improve outcomes for young patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68f0f98bb6cfaaa79c184289f21c5ba4f441b1a4" target='_blank'>
              Masters of Gene Expression: Transcription Factors in Pediatric Cancers
              </a>
            </td>
          <td>
            Anup S. Pathania
          </td>
          <td>2025-10-27</td>
          <td>Cancers</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Targeted therapy has been established as a therapeutic option for the treatment of metastatic melanoma. Despite initially being very efficient, many tumors develop resistance to targeted therapy, leading to its failure. We previously demonstrated that the neural crest (NC)-associated gene ERRFI1 is highly expressed in metastatic melanoma and correlates with a bad prognosis. Here, we show that the expression of ERRFI1 was upregulated in melanoma and negatively correlated with the expression of melanocytic differentiation markers, such as MITF and TYR. Downregulation of ERRFI1 with the help of siRNA increased the susceptibility of melanoma cells toward BRAF inhibition (BRAFi) and resensitized BRAFi-resistant melanoma cells to BRAFi. Mass spectrometry-based proteomic analysis revealed that ERRFI1 silencing diminished the activation of the mitogen-activated protein kinase (MAPK) and AKT signaling pathways, which usually contribute to drug resistance. Furthermore, we show that miR-200c targeted the 3'UTR of ERRFI1 and reduced its expression, resulting in the resensitization of BRAFi-resistant melanoma cells to BRAFi. Our study results suggest that ERRFI1 could be a potential therapeutic target for the treatment of metastatic melanoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a645774843921b1750028307066a9681c4de577" target='_blank'>
              The neural crest-associated gene ERRFI1 is involved in melanoma progression and resistance toward targeted therapy.
              </a>
            </td>
          <td>
            Nina Wang, Qian Sun, Daniel Novák, Lei Zhu, Juliane Poelchen, Tamara Steinfass, Yiman Wang, V. Umansky, Jochen Utikal
          </td>
          <td>2025-10-03</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="

 Extrachromosomal circular DNA (eccDNA) plays critical roles in cancer, yet its landscape in acute myeloid leukemia (AML) remains unexplored.



 We used CIRCLE-seq and RNA-seq to characterize eccDNA in 12 AML patients and 4 healthy controls.




 AML cells showed significantly increased eccDNA counts and gene involvement versus healthy controls, with distinct size peaks at 202 and 368 bp. eccDNAs localized non-randomly to chromosomes 1, 2, and 10–19, enriching near transcription start sites and regulatory regions. Functional analysis revealed activation of oncogenic pathways (e.g., MAPK, ErbB signaling) in AML-associated eccDNA. Integrative analysis identified 570 genes upregulated at both the eccDNA and mRNA levels, including myeloid leukemia-related genes (e.g.,
 FLT3
 ,
 RUNX1
 , and
 CD33
 ) and oncogenes. Prognostic analysis showed that high expression of these genes correlated with poor outcomes in AML.




 This study unveils the eccDNA landscape in AML by direct CIRCLE-seq, linking its accumulation to transcriptional dysregulation and leukemogenesis, and highlights eccDNA as a potential biomarker and therapeutic target.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1de89beb26c968490d1dcd9a8b5f72aeb3ffa8be" target='_blank'>
              Integrated CIRCLE-seq with RNA-seq to decipher the quantity, localization, and functional features of eccDNA in AML
              </a>
            </td>
          <td>
            Lijuan Gao, Qiongyu Lu, Hao Li, Changgen Ruan, Zhao Zeng, Suning Chen
          </td>
          <td>2025-11-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Triple-negative breast cancer (TNBC) is a highly aggressive and clinically challenging subtype of breast cancer, defined by the absence of estrogen receptor (ER), progesterone receptor (PR), and HER2 amplification. This molecular profile limits the use of targeted endocrine or HER2-directed therapies, leaving patients reliant on conventional chemotherapeutics such as doxorubicin. However, the frequent development of chemoresistance underscores the urgent need for new therapeutic strategies. Long noncoding RNAs (lncRNAs), a class of regulatory transcripts with emerging roles in cancer biology, remain largely unexplored in the context of TNBC pathogenesis and drug resistance. This study aimed to systematically identify lncRNAs that drive tumor survival and chemoresistance to uncover novel therapeutic targets and enhance the efficacy of doxorubicin-based treatment. We employed a high-throughput, dual-excision CRISPR-Cas9 screening system targeting the transcription start sites (TSSs) of approximately 1,000 lncRNAs. Functional screens were performed in both 2D monolayer and 3D organoid culture systems, with and without doxorubicin exposure, to closely mimic the tumor microenvironment and treatment stress. Integrative analysis of the screen data identified 22 high-confidence oncogenic lncRNAs essential for TNBC cell survival and resistance to doxorubicin, particularly under 3D conditions that recapitulate in vivo extracellular matrix architecture. Two of these lncRNAs were significantly overexpressed in patient-derived TNBC datasets, highlighting their clinical relevance. Functional validation using antisense oligonucleotides (ASOs) prioritized two top candidates—designated Candidate 100 and Candidate 305—whose knockdown resulted in marked inhibition of TNBC cell growth and viability in both 2D and 3D models. These effects were supported by impaired colony formation, reduced organoid growth, and AO/EtBr live-dead staining assays. This study presents the first comprehensive lncRNA essentiality map in TNBC under 3D culture conditions and reveals the context-dependent roles of lncRNAs in chemoresistance. Our findings establish Candidates 100 and 305 as actionable therapeutic targets, emphasizing the potential of RNA-based precision medicine approaches to overcome doxorubicin resistance in TNBC.



 Vishnubalaji Radhakrishnan, Ramesh Elango, Nehad M. Alajez. Systematic functional profiling of long noncoding RNAs identifies therapeutic vulnerabilities in doxorubicin-resistant triple-negative breast cancer [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C101.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76b88e1c44c29ba0bde4cc2f646df4f72b05fb33" target='_blank'>
              Abstract C101: Systematic functional profiling of long noncoding RNAs identifies therapeutic vulnerabilities in doxorubicin-resistant triple-negative breast cancer
              </a>
            </td>
          <td>
            Vishnubalaji Radhakrishnan, R. Elango, Nehad M. Alajez
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="
 Genomic drivers in glioblastoma (GB) are associated with specific transcriptional states. Within tumour microenvironment architecture however, malignant transcriptional states are largely determined by local cues such as hypoxia or immune infiltration. To what extent genomic drivers affect the TME architecture remains unknown. To investigate the effect of different genomic drivers on TME architecture in GB, we used a replication-competent avian-like sarcoma virus system to introduce EGFRvIII expression, NF1 loss and PDGFB overexpression in a tp53-/- mouse model with Nestin promoter-dependent tumor virus A expression. These genetic alterations recapitulated known classical, mesenchymal, and proneural transcriptomic GB subtypes. We leveraged single cell RNA sequencing and combined spatial transcriptomics and proteomics data at regional (NanoString GeoMx) and single cell (10x Visium HD) level to acquire an in-depth characterization of the differences in the TME organization. Our results revealed intricate dependencies of the GB TME architecture on genetic backgrounds. At single cell level, NF1- and EGFRvIII- driven tumours overrepresented mesenchymal- and astrocyte-like states, whereas PDGFB-driven tumours showed increased neuronal progenitor states. Regardless these evident transcriptional differences, cellular neighbourhoods within the TME locally showed conserved spatial compositions across all samples, with nuanced distinctions between genomic drivers; our integration of spatial transcriptomics and proteomics data showed that EGFRvIII-driven tumours contained complex perivascular immune neighbourhoods with high clustering of plasma B and antigen presenting cells. NF1 driven tumours contained strong perivascular mesenchymal tumour cell enrichment with wound healing features. PDGFB driven tumours increased neuronal signalling with markedly decreased immune infiltration. To extrapolate our findings to human glioblastoma, we leveraged a cohort of 50 spatial transcriptomics samples with known genomic alterations in EGFR, NF1 or PDGFB. Using a graph neural network approach, we were able to reliably predict which cellular neighbourhoods represented genomic alterations. Strikingly, both in mouse and human settings, EGFR, NF1 and PDGFB alterations showed enrichment of immune, tissue reactive and neurodevelopmental organizational structures. By employing sophisticated model systems alongside high dimensional spatial interrogations, we showed how the TME organization is dependent on a complex interplay between cancer cell intrinsic features and its TME. While spatial tumour architecture largely relies on external factors, genomic drivers impose preferred TME structures across mouse and human glioblastomas that could be important in future therapy designs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fd60449c2037e91ea85fb1a099b133b1ae3a448" target='_blank'>
              JS09.7.A GENETIC DRIVES PROMOTE DISTINCT MICROENVIRONMENT ORGANIZATION IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            J. Zhang, T. Joshi, D. Heiland, D. Hambardzumyan, L. van Hijfte
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Chromosome instability (CIN) and Genetic instability are hallmark features of malignant tumor cells, contributing to tumour progression, drug resistance and metastasis. KIF18A, a member of the kinesin superfamily of motor proteins, is essential for preserving the structural integrity of the bipolar spindle apparatus during cell division. Recent studies have demonstrated that genetic depletion of KIF18A impacts the chromosome alignment, elongation of mitotic spindles, and activates spindle assembly checkpoint in chromosomal instable tumour cells ultimately leading to cell death, while having negligible impact towards the survival of normal cells. These findings suggest that targeting KIF18A represents a promising therapeutic approach for selectively inhibiting the growth of chromosomally instable tumours, while sparing the normal division of euploid cells. We have designed novel macrocyclic KIF18A inhibitors based on our structural understanding from Cryo-EM structures of initial acyclic binders. Structure guided SAR led to the identification of highly potent and selective compounds with optimized properties. Lead compounds exhibited significant inhibition of KIF18A ATPase activity and demonstrated robust cellular anti-proliferative potency in chromosomally instable (CIN-high) high-grade serous ovarian carcinoma cells (HGSOC, OVCAR-3) while retaining a good selectivity over the CIN-low MCF7 cells. This in vitro potency was supported by a significant and dose-dependent upregulation of phospho-Histone H3 and gamma-H2AX, consistent with the induction of mitotic arrest and DNA damage. Further, the lead compounds exhibited encouraging drug-like properties, including acceptable in vitro ADMET profile and excellent in vivo exposure with low clearance resulting in good oral bioavailability in rodents. Additionally, these compounds showed significant anti-tumor efficacy that was associated with robust induction of phospho-histone H3 in the OVCAR3 CDX model. The macrocyclic lead compound is being extensively evaluated in advanced in vitro and in vivo assays to support further development.



 Kiran Aithal B, Sandip Sengupta, Raghavendra N R, Shailesh Vishwanath. Jadhav, Aditya Kiran. Gatta, Balasubramanyam D, Suresh D. Babu, Gokulakrishnan R, Srinivasu P, Samiulla D S, Amith A, Aravind A B, Girish C. Daginakatte, Kavitha Nellore, Kishore Narayanan, Chandrasekhar Abbineni, Murali Ramachandra, Susanta Samajdar. Discovery and preclinical characterization of novel macrocyclic KIF18A Inhibitors for treatment of chromosomally instable tumors [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr A030.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ab5f82e03ba5520dcf239a8a55ec749b43cc168" target='_blank'>
              Abstract A030: Discovery and preclinical characterization of novel macrocyclic KIF18A Inhibitors for treatment of chromosomally instable tumors
              </a>
            </td>
          <td>
            Kiran Aithal B, Sandip Sengupta, Raghavendra N R, Shailesh V. Jadhav, Aditya Kiran. Gatta, Balasubramanyam D, Suresh D. Babu, Gokulakrishnan R, Srinivasu P, S. S, A. A., A. B, G. Daginakatte, K. Nellore, Kishore Narayanan, C. Abbineni, Murali Ramachandra, S. Samajdar
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Ataxia-Telangiectasia (A-T) is caused by biallelic Ataxia-Telangiectasia Mutated (ATM) protein mutations and is hallmarked by neurodegeneration, immunodeficiency and cancer susceptibility. Emerging work suggests that non-coding RNAs are integral to the ATM driven DNA damage response, yet the scope of non-coding RNA perturbation in A-T has not systematically mapped. This study is done to chart and synthesize all primary evidence on ncRNA dysregulation in A-T and to identify mechanistic gaps and research priorities. The evidence base comprises a total of five studies focused exclusively on microRNAs and long non-coding RNAs. Baseline sequencing of 20 patients revealed consistent downregulation of miR-195-5p, miR-30a-5p and miR-342-3p. Low dose γ-irradiation of ATM-null lymphoblastoid cell lines produced an exaggerated bidirectional shift (8 microRNAs upregulated, 6 downregulated) versus wild-type, whereas primary T-cells showed largely ATM independent rises of miR-34a-5p and miR-182-5p. A lymphoblastoid cell lines study distinguished 22 recessive or dominant radiation-responsive microRNAs. Long non-coding RNAs profiling identified 149 transcripts that require functional ATM for induction and three uniquely upregulated in A-T cells. No circular RNA study in A-T is observed till date. The ncRNA research in A-T is nascent with five studies present a miRNA dysregulation and an ATM-dependent lncRNA surge that is lost in A-T, but leave circRNAs, neuronal tissues and functional validation unexplored. Intermediate dose stress models, total and small RNA-seq under DNA damage and oxidative stress, and CRISPR rescue/knockout screens in patient derived organoids are immediate priorities for exploring the role of ncRNAs as therapeutic targets in A-T.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/347f924a4e62687ed1a1c526f37bf32430e75d71" target='_blank'>
              Mapping the non-coding RNA landscape in ataxia telangiectasia: a scoping review of ATM dependent miRNA and lncRNA dysregulation
              </a>
            </td>
          <td>
            Muhammad Junaid Iqbal, Usman Ameen, Hamza Tanveer, Laraib Israr, Anastasia Ricci, Gianluca Morganti, Muhammad Jawad Khan, M. Menotta
          </td>
          <td>2025-10-09</td>
          <td>Molecular Biology Reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 TP53, often referred to as the guardian of the genome, is one of the most frequently mutated genes in human cancers. High grade serous ovarian cancer (HGSOC) has a TP53 mutation rate of up to 96%. Given its multi-faceted role in human cancers, there have been many efforts to therapeutically target mutant p53. APR-246 is a small molecule drug that is proposed to bind mutant p53 and promote its thermodynamic stabilization, allowing the tumor suppressor to regain wild-type-like conformation and functionality. However, the mechanism of this drug has remained a contentious topic, as it has been shown to act via various p53-independent mechanisms, including modulation of the redox system. To address this, we have performed a systematic functional genetic screening for the identification of APR-246 rescuable p53 mutants. Our approach leverages a pooled TP53 mutagenesis library containing ∼8,000 amino acid variants, spanning the entirety of the TP53 coding sequence. Cell based screenings against sub-lethal doses of APR-246 in two ovarian cancer cell lines, SKOV3 and OVCAR5, has revealed that sensitivity to this drug is cell-line dependent rather than mutant p53-dependent. Our results suggest that p53-independent factors such as intrinsic resistance to ferroptosis may play a more significant role in response to APR-246. We propose that resistance to ferroptosis provides cellular conditions which enable identification of p53 mutants that are sensitive to APR-246. This is evidenced by our results which show that genetic screen in OVCAR5, which is more intrinsically resistant to ferroptosis, uncovers a greater number of potentially sensitive mutants, including R175H, a mutant largely recognized as rescuable by APR-246. Further, we have identified GPX4, an anti-ferroptosis gene, as a key determinant of APR-246 sensitivity. Indeed, we demonstrate synergy between APR-246 and GPX4 inhibitor, ML-210. These studies support the hypothesis that APR-246 efficacy is largely attributable to p53-independent mechanisms and that attenuation of these off-target effect may reveal potentially sensitive p53 mutants.



 Anais Saunders, Caili Tong, Anthony N. Karnezis, Gary S. Leiserowitz, Jeremy R. Chien. Functional genetic screen of TP53 mutants reveals APR-246 sensitivity is mediated by p53 independent effects [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A046.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7beb6941bcf2a1d1e3d25e6917a37bc8794a91d" target='_blank'>
              Abstract A046: Functional genetic screen of TP53 mutants reveals APR-246 sensitivity is mediated by p53 independent effects
              </a>
            </td>
          <td>
            Anais Saunders, Caili Tong, A. Karnezis, G. Leiserowitz, Jeremy Chien
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="


 Pancreatic cancer is a lethal disease that has a five-year survival rate of ∼13%. Pancreatic cancer is marked by a number of significant alterations in its dense tumor microenvironment and metabolism. Our analyses of differential gene expression in multiple datasets point to HDAC1 as a consistently upregulated gene in pancreatic cancer and one that is significantly correlated with multiple metabolic pathways. Using CRISPR/Cas-9 gene editing, we generated HDAC1-knockout cell lines. Our current findings show when HDAC1 is deleted, pancreatic cancer cell lines display alternate growth phenotypes, differentially express metabolic targets, and have global metabolomics changes. Further, Seahorse analysis revealed a switch to oxidative phosphorylation in a subset of HDAC1 knockout cells. This distinct response to HDAC1 deletion demonstrates the importance of understanding the nuanced characteristics in order to effectively target metabolism in pancreatic cancer based on different patient subgroups.



 Don-Gerard B. Conde, Evan J. Zhou, Maximilian Farma, Soren Jensen, Laiba Sheikh, Matthew Cheung, Chiamaka J. Ezeh, Harshita N. Khedkar, Zeribe C. Nwosu. Deletion of HDAC1 demonstrates alternative metabolic response in pancreatic cancer [abstract]. In: Proceedings of the 18th AACR Conference on the Science of Cancer Health Disparities; 2025 Sep 18-21; Baltimore, MD. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2025;34(9 Suppl):Abstract nr B135.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01b243a07ac38b6a04c17b406fa31db8c75f6446" target='_blank'>
              Abstract B135: Deletion of HDAC1 demonstrates alternative metabolic response in pancreatic cancer
              </a>
            </td>
          <td>
            Don-Gerard Conde, Evan J. Zhou, Maximilian Farma, Soren Jensen, Laiba Sheikh, Matthew Cheung, Chiamaka J. Ezeh, H. N. Khedkar, Z. Nwosu
          </td>
          <td>2025-09-18</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9becd4af94c7faf26e4bbaab5024e42c7ccb439f" target='_blank'>
              Origins of chromosome instability unveiled by coupled imaging and genomics.
              </a>
            </td>
          <td>
            Marco R. Cosenza, Alice Gaiatto, Büşra Erarslan Uysal, Álvaro Andrades, Nina Luisa Sautter, Marina Šimunović, M. Jendrusch, Sonia Zumalave, Tobias Rausch, A. Halavatyi, Eva-Maria Geissen, Joshua Lucas Eigenmann, Thomas Weber, Patrick Hasenfeld, Eva Benito, Catherine Stober, I. Cortés-Ciriano, Andreas E Kulozik, Rainer Pepperkok, Jan O. Korbel
          </td>
          <td>2025-10-29</td>
          <td>Nature</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="


 Acute lymphoblastic leukemia (ALL) is the leading cause of cancer-related mortality in pediatrics. Between 15-30% of patients relapse, and current treatments are associated with acute and long-term toxicity. The immune component of the tumor microenvironment is an under-explored factor that could play a role in the establishment or progression of the disease, as well as modulate response to chemo- and immunotherapies. We characterized the molecular profile of leukemias from 32 patients with pediatric ALL treated under the ALLIC-GATLA 2010 clinical protocol in Argentina by transcriptome sequencing (Illumina). We estimated the immune component through digital cytometry (MIXTURE) and a cytolytic score based on the expression of five genes. We searched for single nucleotide variants (SNV) in the transcriptome (CTAT-Mutations) that could alter the interaction with immune cells in the tumor microenvironment. For this, we analyzed 843 genes from Reactome collections associated with the immune system. Variants were filtered and annotated using publicly available germline and somatic variant databases. The group of patients with higher abundance of cytotoxic cells (Z-score>1) was associated with an increased risk of relapse/death (HR=4.5; p=0.05), and higher levels of CX3CR1 (q-val<0.05) and TIM-3 (q-val=0.06). Analysis of an independent high-risk cohort (TARGET B-ALL phase II, N=88), confirmed that a higher cytolytic score was associated with the risk of relapse in an univariate (HR=2.66, p=0.001) and multivariate model including cytolytic score (p<0.006), protocol (p=0,07), ETV6::RUNX1 (p:>0.05) and TCF3::PBX1 status (p<0.001). Cytolytic score was not associated with molecular subtype, age, sex or early response to treatment (p>0.05). For SNV analysis, we developed a threshold value for the QUAL parameter to select for high-confidence variants, establishing a specificity of 95%. Eleven variants were found in nine patients, being 8/11 classified by Franklin as TIER3 in CEP290, UBE3B, CPNE3, DUSP4, DAPP1, RNF135, TBC1D10C and TLR1; 2/11 as TIER1 in TP53, KRAS; and 1/11 as TIER4 in BTN3A3. Patient harboring variants in these genes were significantly associated with a lower GSVA score in the Reactome dataset “MHC_Class_II_Antigen_Presentation” (q-val<0.01). We found a variant in UBE3B, located in the active site of the E3 ubiquitin ligase, and associated with a low GSVA score in the Reactome “Antigen presentation through MHC class I” pathway. A novel variant in CPNE3 gene was found, located in a highly conserved residue and predicted to be deleterious by multiple tools. Conclusion: the abundance of cytotoxic cells was independent of molecular subtype and associated with a worse prognosis in independent cohorts. Further analysis of the variants found in this study has the potential to identify novel mechanisms modulating the interaction between the immune system and leukemic cells in ALL.



 Ignacio Gomez Mercado, Daniel Avendaño, Sabrina Ledesma Bazan, Pablo Sanchis, Maria Cecilia Riccheri, Geraldine Gueron, Javier Cotignola, Maria Sol Ruiz. Characterization of cytotoxic cell abundance and genetic variants in immune related genes in pediatric acute lymphoblastic leukemia patients [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr A040-PR008.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc8f83b4eeb02fdcb7483db9e8c77e60a6f1fd7e" target='_blank'>
              Abstract A040-PR008: Characterization of cytotoxic cell abundance and genetic variants in immune related genes in pediatric acute lymphoblastic leukemia patients
              </a>
            </td>
          <td>
            Ignacio Gomez Mercado, Daniel Avendaño, Sabrina Ledesma Bazan, P. Sanchis, M. C. Riccheri, G. Gueron, J. Cotignola, Maria Sol Ruiz
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Cancer cells can acquire a reversible, dormant drug-tolerant persister state mimicking embryonic diapause to evade therapy pressure. Deciphering the precise mechanisms driving cancer cells into or out of a diapause-like persister cell-state could provide strategies to overcome resistance. Here, we showed that following chemotherapy, diverse therapeutic agents converge on WNT pathway activation to induce a de novo diapause-like cell state across various triple-negative breast cancer (TNBC) cell line, xenograft, and patient-derived organoid models. Among early persister cells, only transcriptionally WNT-active persisters exhibited the transcriptional and functional characteristics typical of diapause-like cells, including a negative correlation with MYC transcriptional activity and reversible restricted proliferation. The WNT signaling pathway functioned as both an inducer and biomarker of the diapause-like early persister cell state in both parental (chemo-naïve) and chemotherapy-treated cells. Entry into and exit from the diapause-like early persister cell state was triggered by the transcriptional upregulation of components essential for canonical WNT ligand secretion. A combinatorial treatment strategy inhibiting WNT ligand secretion alongside chemotherapy effectively targeted the early mechanisms underlying the acquisition and enrichment of a diapause-like cell phenotype. These findings reveal WNT pathway activation as an early event that leads to a reversible diapause-like persister state and highlight the potential of targeting this axis to prevent the development of drug-resistant populations before they are firmly established.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc9bf626a72b9d5ae7b70a19b1f05585d0584500" target='_blank'>
              Chemotherapy-Treated Breast Cancer Cells Activate the WNT Signaling Pathway to Enter a Diapause-Like Early Persister State.
              </a>
            </td>
          <td>
            Youssef El Laithy, Willy Antoni Abreu de Oliveira, A. Pabba, Alessandra Qualizza, Gwenny Cosemans, Paula Garcia-Diaz, François Richard, Paraskevi Athanasouli, Carla Rios Luci, Wout De Wispelaere, Larissa Mourao, Siân Hamer, Stijn Moens, A. de Jaime-Soguero, M. Baietti, S. Hutten, Jos Jonkers, Stephen-John Sammut, S. Soenen, C. Scheele, Alejandra Bruna, C. Desmedt, D. Annibali, F. Lluis
          </td>
          <td>2025-10-21</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="The initiation and progression of tumors are closely linked to aberrant regulation of multiple genes. Among these genes, sister chromatid cohesion acetyltransferase 2 (ESCO2) has emerged as an important target in cancer research. ESCO2 exhibits complex and heterogeneous expression patterns across various types of malignancies and tumor progression stages. Extensive studies have shown that ESCO2 primarily functions in chromosomal cohesion, significantly contributing to genomic stability maintenance. Dysregulated expression of ESCO2 in tumor cells is often correlated with malignant behaviors. Given its essential role in tumorigenesis, ESCO2 has increasingly been explored as a promising therapeutic target in recent years. This review article summarizes recent research progress regarding ESCO2 in malignant tumors, aiming to identify a novel target for their treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79b1b540beced4302a863f075307c9fa3cb6930a" target='_blank'>
              The expression of establishment of cohesion 1 homolog 2 (ESCO2) in tumor cells and its research progress as a therapeutic target
              </a>
            </td>
          <td>
            Chunmei Lu, Hanchi Wu, Huiyu Wang
          </td>
          <td>2025-10-06</td>
          <td>European Journal of Medical Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10f7f4234a610f853443cb0ed073e79f12207d8b" target='_blank'>
              Multiple patient-derived glioblastoma models reveal synthetic lethality through concurrent PI3K and CDK4/6 inhibition by blocking trans-active cooperation.
              </a>
            </td>
          <td>
            Jing Zhang, Xu Chen, Meng Cheng, Jingzhe Wang, Yueyao Wu, Han Xie, Chunyu Zhang, Honghao Wang, Ying Pang, Tongjie Ji, Yuntong Yang, Junyu Yang, Siyi Xu, Zhigang Wang, Qi Wang, Min Liu, Chunlong Zhong
          </td>
          <td>2025-09-18</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="BACKGROUND
Ageing is a general, intrinsic and progressively deleterious process that affects all cells, tissues and organs albeit at different extent and rate in each individual. The complexity and universality of its phenotypic manifestations suggest a multifactorial origin. The autosomal recessive disorder Werner syndrome likely represents a segmental progeroid disorder since patients show several, but not all phenotypes of premature ageing.


SUMMARY
Proliferative senescence of diploid cells in culture provided a model system in which ageing can be studied experimentally. Cultures of cells from patients with Werner syndrome experienced an extreme form of proliferative senescence and a clonal succession of translocations, known as variegated translocation mosaicism. In addition, Werner syndrome cells showed spontaneous deletion formation and a prolongation of and arrest in the S phase of the cell cycle. The WRN protein harbors a helicase, an exonuclease and a RecQ interaction domain. With the latter the WRN protein may interact with NBS1, RPA, MRE11, TREX1, MUTYH, POT1, TRF1, FEN-1, PAPRP-1, p97/VCP, TRF2, DNA polymerase(beta), Ku76/80, EXO-1, NEIL1, and p53, which are key to DNA damage response pathways including canonical NHEJ, homologous recombination, base excision repair and telomere maintenance. The WRN exonuclease domain is a target of WRNIP1 binding, which links WRN to resolution of stalled replication due to collision with transcription and the ATM-mediated cell cycle checkpoint. . Patients with an incomplete complement of Werner syndrome phenotypes, called atypical Werner syndrome patients, were found to carry variants in LMNA, POLD1, SPRTN, MDM2, CTC1, SAMHD1.


KEY MESSAGES
These findings broaden the genotypic landscape and the phenotypic spectrum of Werner syndrome. In this review potential molecular mechanisms underlying genomic instability in Werner syndrome, including chromothripsis due to asynchronous S phase traverse and telomere crises followed by bridge fusion breakage cycles are discussed. The participation of WRN in multiple gene networks is consistent with the multifactorial nature of ageing in general.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96c57a230c575d7a7f034afedf23d1a5bec7bbac" target='_blank'>
              Molecular mechanisms of proliferative senescence and genomic instability in Werner syndrome and the WRN gene network.
              </a>
            </td>
          <td>
            Martin Poot
          </td>
          <td>2025-09-19</td>
          <td>Cytogenetic and genome research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ec13fad98d61dff3c83ecb3a4a35f96c11ec562" target='_blank'>
              Targeting MORF4L1-mediated DNA repair potentiates RT-induced antitumor immunity via cGAS-STING activation in hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Si-wei Wang, Wei-Feng Hong, Yi-Lan Huang, Yi-Min Zheng, Qiu-Yi Zheng, Jun-Jie Cheng, Bu-Fu Tang, Gen-Wen Chen, , Jia-Cheng Lu, Li Yuan, Shujung Hsu, Yang Zhang, Xiao-Bin Zheng, Zhaochong Zeng, Jian Zhou, Jia Fan, Aiwu Ke, Chao Gao, Shi-Suo Du
          </td>
          <td>2025-11-05</td>
          <td>Cellular & molecular immunology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Metastatic colorectal cancer (mCRC), particularly microsatellite stable (MSS) cases, often exhibits limited responsiveness to immunotherapy, leaving chemotherapy as the primary treatment option. While chemotherapy effectively targets tumor cells, its impact on the broader mutational landscape, including passenger mutations in large genes such as Titin (TTN), remains poorly understood. Passenger mutations, traditionally deemed biologically inert, may reflect tumor mutational burden (TMB) and influence treatment outcomes. In our study involving whole exome sequencing of paired primary and metastatic tumor samples from 22 mCRC patients, recurrent driver mutations in APC, KRAS, and TP53 were consistently observed. However, passenger mutations in large genes, particularly TTN, were notably enriched in chemonaïve specimens and associated with higher TMB. Chemotherapy-treated samples exhibited a significant reduction in these mutations, suggesting selective depletion of hypermutated subclones. Our findings demonstrate that chemotherapy may selectively reduce passenger mutations in mCRC, potentially influencing the persistence of hypermutated subclones. This highlights the potential role of passenger mutation patterns and TMB as biomarkers for treatment response and raises the hypothesis that they could help guide immunotherapy considerations for patients with MSS mCRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b08b39b78f849a94033df24c4d40094e5f87ad7" target='_blank'>
              Effect of chemotherapy on passenger mutations in metastatic colorectal cancer.
              </a>
            </td>
          <td>
            Marium T Siddiqui, Matthew A. Cottam, Muhammad Bilal Mirza, Keeli B. Lewis, K. K. Ciombor, M. K. Washington, K. Idrees
          </td>
          <td>2025-11-05</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="


 Glioblastoma (GB) is the most common malignant adult brain tumor. Modern chemotherapies and immunotherapies have failed to improve the outcome of GB. Thus, there is a need for developing novel approaches to target GB. Programmable cell death (PCD) programs have been long hailed as potential targets for cancer. Ferroptosis, an iron dependent cell death characterized by lipid peroxidation, has been identified as a potential target for various cancers’ therapy, including GB. However, how GB responds to ferroptosis remains unknown, hindering effective targeting.



 Genome-wide negative CRISPR screening was conducted in U87MG glioma cell line using 2 different classes of ferroptosis inducers. CRISPR/Cas9 mediated KO of the identified target (EDF1) was conducted to validate the downstream effect. RNA-seq and Ribo-seq were used to study the translational impact of EDF1 on glioma cells. Downstream validation of dysregulated pathways was conducted using gene knockdown and various methods.



 Genome-wide CRISPR screening revealed that the top gene in 2 independent screens was EDF1, a sensor of ribosome collisions. Induction of Ferroptosis in glioma cells activated the ribotoxic stress pathway (RSR) and the integrated stress response pathway (ISR) and upregulated EDF1 translationally. Ribo-seq analysis revealed significant ribosome stalling at distinct codons and activation of a G/C-ending codon biased translation characteristic of stress response. EDF1 knockout (KO) led to reduced proliferation and stemness of glioma cells. EDF1 KO rendered glioma cells extremely sensitive to ferroptosis and interfered with the activation of the RSR in response to ferroptosis inducers. RNA-seq and Ribo-seq analysis revealed dysregulated transcription and translation in response to EDF1 KO with and without stress induction. In normal growth conditions, we observed severe ribosome stalling at hydrophobic amino acids and mismatch between transcription and translation. During stress, EDF1 KO cells were unable to properly mount the stress-responsive codon biased translational program and showed severe ribosome stalling at sublethal ferroptosis inducers concentrations. Sequencing analysis revealed an upregulation of mitochondria-related pathways at the translational level. Flux analysis revealed hyperactive mitochondria that were structurally different from normal glioma mitochondria. Further analysis of the sequencing data revealed the dysregulated expression of various genes linked to glioma oncogenesis and ferroptosis resistance. Of note, Androgen receptor (AR) showed a near complete disappearance of its protein levels. AR is known to promote oncogenesis and ferroptosis resistance



 our results indicate that targeting the ribosome quality control machinery that operates during oxidative stress is a viable target for GB therapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51144dff1941a16223529b910cf69c5bc0703557" target='_blank'>
              P01.01.A RIBOSOME COLLISION IS A TARGETABLE VULNERABILITY IN GLIOMA
              </a>
            </td>
          <td>
            S. Rashad, T. Zhang, A. Mousa, K. Niizuma
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d4e23f0e2eb9b451cd70e81ca9b082a2f8b6469" target='_blank'>
              Advancements in single-cell sequencing for cervical cancer research.
              </a>
            </td>
          <td>
            Congli Pu, Biyuan Xing, Shujie Wang, Zhao Liu, Yingchao Zhao
          </td>
          <td>2025-11-06</td>
          <td>Molecular and cellular biochemistry</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2927a64ad6f25a642a97a344c961d20f78a6873" target='_blank'>
              Neuroblastoma-associated ALK variants have distinct cellular and biochemical activities
              </a>
            </td>
          <td>
            Anna M. Wulf, Stuart Lutimba, Dylan Cameron, Jade Desjardins, Tanya J. Shaw, Leone Rossetti, Mohammad A. Mansour, Karen J. Liu
          </td>
          <td>2025-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0007d62f2b8c21bcd3de104015c7a577a0669693" target='_blank'>
              Defining treatment-resistant brain cancer: Genetic screening to identify oncogene-driven immunomodulation and therapy resistance.
              </a>
            </td>
          <td>
            Zachary David Myers, S. Matosevic
          </td>
          <td>2025-10-26</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>25</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [7],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>