<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2026-02-16 10:03:32 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="DHFR gene amplification is commonly present in methotrexate (MTX)-resistant colon cancer cells and acute lymphoblastic leukemia. In this study, we proposed an integrative framework to characterize the amplified region by using a combination of single-molecule real-time sequencing, next-generation optical mapping, and chromosome conformation capture (Hi-C). We identified an amplification unit spanning 11 genes, from the DHFR gene to the ATP6AP1L gene position, with high adjusted interaction frequencies on chromosome 5 (~2.2 Mbp) and a twenty-fold tandemly amplified region, and novel inversions at the start and end positions of the amplified region as well as frameshift insertions in most of the MSH and MLH genes were detected. These mutations might stimulate chromosomal breakage and cause the dysregulation of mismatch repair. Characterizing the tandem gene-amplified unit may be critical for identifying the mechanisms that trigger genomic rearrangements. These findings may provide new insight into the mechanisms underlying the amplification process and the evolution of drug resistance. Sequencing a large region of DNA containing many surplus copies of genes linked to drug resistance in colon cancer cells may illuminate how these genomic rearrangements arise. Such regions of gene amplification are highly repetitive, making them impossible to sequence using ordinary methods, and little is known about how they are generated. Using advanced methods, Jeong-Sun Seo at Seoul National University Bundang Hospital in South Korea and co-workers sequenced a region of gene amplification in colon cancer cells. The amplified region was approximately 20 times the length of that in healthy cells and contained many copies of an eleven-gene segment, including a gene implicated in drug resistance. The region also contained mutations in chromosomal repair genes which would disrupt repair pathways. These results illuminate the genetic changes that lead to gene amplification and drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental & Molecular Medicine, Experimental and Molecular Medicine</td>
          <td>5</td>
          <td>50</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum
                </a>
              </td>
          <td>
            Gustavo S. França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>94</td>
          <td>46</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bd68744cdbfac994a21e7c57ae21390703eb542" target='_blank'>
              Selection for targeted therapy resistance leads to an indirect selection for higher phenotypic plasticity and enhanced evolvability to orthogonal stressors
              </a>
            </td>
          <td>
            Alicia Bjornberg, Aobuli Xieraili, M. Froid, R. Barker-Clarke, Jeffrey Maltas, Robert J. Vander Velde, Jamir Riffas, Berkley Gryder, Jacob G. Scott, David Bassanta, Alexander R. A. Anderson, Virginia A Turati, A. Marusyk
          </td>
          <td>2026-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63674f265b1ac623907383e45f60b5724b528b86" target='_blank'>
              Recurrent Escape from Osimertinib-Induced Senescence Promotes Genomic Instability Associated with Therapeutic Resistance
              </a>
            </td>
          <td>
            H. McDaid, Nazia S Jamil, Qualia Hooker, Nadjet Cornejal, Dean Hosgood
          </td>
          <td>2026-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/774d717fb446d87b8fca81f2c265377058a6eee1" target='_blank'>
              On-target mutations confer resistance to WRN helicase inhibitors in Microsatellite Unstable Cancer Cells
              </a>
            </td>
          <td>
            G. Picco, Yanhua Rao, Angham Al Saedi, Samantha J. Walker, S. Bhosle, Yang Lee, Sara F. Vieira, Maria Garcia-Casado, Gilberto Valdes Garcia, Kieron May, F. Sassi, Inigo Barrio Hernandez, Mamta Sharma, Cansu Dinçer, Theo Bell, Anastasia Kavasakali, Ray Shenje, Cuthbert Martyr, Edward J. Brnardic, Nanhua Deng, Hristina Grigorova Dimitrova, Emre Karakoc, Sandeep Sundara Rajan, Nicola Chan, Emma Hitch, Katrina McCarten, Camille Fourneaux, Zoe Hewitson, H. Lightfoot, S. Barthorpe, James P. Phelan, Phil Landis, Brian T. Jones, Diana M. Muñoz, Jay Prakash, P. Barsanti, J. Taygerly, Michael P. DeMartino, Emanuel Gonçalves, A. Bertotti, L. Trusolino, Michael A. White, Geeta Sharma, Matthew A. Coelho, J. Houseley, Benjamin Schwartz, Mathew J. Garnett
          </td>
          <td>2025-12-27</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>67</td>
        </tr>

        <tr id="Cancer cell adaptation to their physical tumor microenvironment is a key driver of malignancy. Recent experimental evolution experiments show that the soft extracellular matrix (ECM) can impose a selection pressure on genetically variable tumor populations. Over months of sustained culture, the selection pressure leads to enrichment of specific genetic variants with high fitness, but the mechanisms underlying the high fitness of these soft-selected clones are not fully understood. Here, we used a combination of RNA-seq, ATAC-seq, and RRBS-seq to compare soft-selected populations with non-selected ancestral populations cultured on soft ECM. We demonstrate that ancestral populations grown on soft ECM for short durations are characterized by a stressed cell state with low fitness marked by cell cycle arrest and distinct metabolic shifts, whereas sustained culture selects for a robust proliferative phenotype. Mechanistically, selected cells exhibit a silenced ancestral stress response through epigenetic modifications, characterized by reduced chromatin accessibility and de novo DNA methylation, including CDH1 promoter hypermethylation. This repressive landscape supports a high-fitness state defined by elevated MYBL2 and FAK levels. An in-silico mechanism-based model shows that these molecular differences, together with high YAP1 nuclear localization in soft-selected cells, are salient features of genetic clones capable of FAK upregulation. These findings uncover a coordinated genetic and epigenetic mechanism driving cancer cell evolution in mechanically soft microenvironments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4db71dfbf014de56876df2b0949f05913ac70b1" target='_blank'>
              Multi-omics investigation reveals molecular determinants of cancer cell evolution on soft extracellular matrix
              </a>
            </td>
          <td>
            Sarthak Sahoo, S. Khanna, Ting-Ching Wang, Tanmay P. Lele, M. Jolly
          </td>
          <td>2025-12-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Mutations in epigenetic regulators are common in bladder cancer, yet their impact on therapeutic responses remains unclear. Here, we identify that loss-of-function mutations in KDM6A, a histone demethylase altered in about 26% of advanced bladder cancers, are associated with poor survival after cisplatin chemotherapy, whereas they correlate with improved outcomes with anti-PD-1 therapy. Using CRISPR-Cas9-engineered murine and human bladder cancer models, we show that KDM6A deficiency increases formation of extrachromosomal circular DNA carrying chemoresistance loci, promoting cisplatin resistance. In parallel, KDM6A loss impairs DNA repair and rewires tumor metabolism, reducing glycolysis and lactate output. This metabolic shift diminishes histone lactylation in regulatory T cells, suppressing immunoregulatory genes and limiting expansion of PD-1hi regulatory T cells. Collectively, our findings establish KDM6A mutation as a key regulator of therapeutic responses, providing a foundation for its use in guiding precision therapy in advanced bladder cancer. Genes encoding key epigenetic regulators, including Lysine Demethylase 6A (KDM6A), are frequently mutated in bladder cancer. Here, the authors show that loss of KDM6A promotes formation of extrachromosomal circular DNA (eccDNA), genomic instability, and metabolic reprogramming, driving resistance to cisplatin chemotherapy while simultaneously enhancing sensitivity to immune checkpoint inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9139b0ede19d5b7c3ea40131744831c88cd696ed" target='_blank'>
              Loss of KDM6A-mediated genomic instability and metabolic reprogramming regulates response to therapeutic perturbations in bladder cancer
              </a>
            </td>
          <td>
            Pratishtha Singh, R. D’Rozario, Bidisha Chakraborty, Swadhin Meher, D. Raychaudhuri, Aminah J. Tannir, Yang Li, Anurag Majumdar, Jessalyn Hawkins, Yun Xiong, Philip L. Lorenzi, Padmanee Sharma, K. Akdemir, P. Pilié, Abhinav K. Jain, Byron H Lee, Sangeeta Goswami
          </td>
          <td>2026-01-07</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63b55c41a895a4562f500627e7c41a6885170b27" target='_blank'>
              Clonal morphology-guided combination therapies overcome heterogeneity-driven drug tolerance
              </a>
            </td>
          <td>
            Xiaoxi Li, Ling Liu, Yuanyuan Yan, Mengfang Yang, Lingli Luo, Yong Jiang, Weijun Ren
          </td>
          <td>2026-02-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Molecular targeted therapies targeting KRAS signaling have significantly improved patient outcomes, but they have not achieved sufficient therapeutic efficacy in colorectal cancer (CRC). Here, we demonstrate that a subset of KRAS-mutant CRC cells transitions to a cellular state characterized by enhanced ribosome biogenesis upon KRAS signaling inhibition. The mitogen-activated protein kinase kinase inhibitor, trametinib, and AMG510 induce a cellular state characterized by a gene expression profile highly enriched for ribosome biogenesis. We find that they are vulnerable to the inhibition of RNA polymerase I, and they exhibit synergistic anti-tumor effects with trametinib in an autochthonous mouse model of intestinal tumors and human patient-derived organoids (PDOs). These observations demonstrate that high ribosome biogenesis induced by KRAS inhibition is indispensable to maintain this cellular state and is a potential therapeutic target. Overall, this study reveals novel mechanisms of drug tolerance to KRAS inhibition, thereby facilitating the development of new therapeutic strategies. Drug tolerance to KRAS inhibition remains a major challenge in cancer therapy. Here the authors find that KRAS inhibition induces a cellular state characterized by enhanced ribosome biogenesis in KRAS-mutant colorectal cancer cells and these cells are vulnerable to inhibition of RNA polymerase I.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92b7de6d8aeceaad9c8d57eb1d9c3d308c5a80e2" target='_blank'>
              Ribosome biogenesis as a potential therapeutic target in KRAS mutant colorectal cancer
              </a>
            </td>
          <td>
            Yui Tanaka, M. Sakahara, H. Yamanaka, Yasuko Natsume, Daisuke Kusama, Kohei Kumegawa, Harunori Yoshikawa, Yuich Abe, K. Okabayashi, Shimpei Matui, Y. Kitagawa, Naohiko Koshikawa, H. Osumi, E. Shinozaki, Satoshi Nagayama, Jun Adachi, R. Maruyama, Ryoji Yao
          </td>
          <td>2025-12-27</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Abstract Liquid biopsies can detect actionable mutations and infer broad tumor states from genome-wide cfDNA measurements, but quantitative transcriptome-like phenotyping at single gene resolution still largely requires tissue. Here, we asked whether 6-base whole-genome sequencing that jointly quantifies 5-methylcytosine (5mC) and 5-hydroxymethylcytosine (5hmC) could infer gene expression directly from plasma. We applied this framework to plasma from patients enrolled in a phase 2 clinical trial of the PARP inhibitor olaparib plus the PD-L1 inhibitor durvalumab for metastatic castration-resistant prostate cancer. Inferred plasma transcriptomes distinguished adenocarcinoma vs. neuroendocrine phenotypes and identified a noncanonical WNT5A-associated signature linked to poor clinical response. Integrating longitudinal cfDNA methylomic profiles with phylogenetic reconstruction further revealed two resistance trajectories: one featuring high tumor heterogeneity with persistent AR signaling, and another marked by an AR-independent, stem-like program with metabolic reprogramming. These findings demonstrate that ctDNA can inform phenotype-driven tumor biology at gene-level resolution, integrating epigenetic modifications, inferred transcriptional programs, and clonal dynamics as a function of treatment response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/577ed4ef29e6ee2ab69965179663102c2fe760c9" target='_blank'>
              Integrative dual ctDNA 5mC/5hmC methylomics and clonal reconstruction infer tumor transcription and resistance phenotypes in metastatic prostate cancer
              </a>
            </td>
          <td>
            A. Sowalsky, Chennan Li, A. Baj, Clara C Y Seo, N. Terrigino, J. Bright, S. Hennigan, I. King, S. Wilkinson, Tzu-Ting Huang, S. Trostel, William Figg, William L Dahut, David Y. Takeda, Jung-Min Lee, F. Karzai
          </td>
          <td>2026-02-05</td>
          <td>Research Square</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74d7c38058905c3f246edc0fb45f9435b0439528" target='_blank'>
              Efflux pump regulation and metabolic rewiring define the ceiling of efflux-mediated drug resistance in Mycobacteria
              </a>
            </td>
          <td>
            Shiksha Sharma, Sarika Mehra
          </td>
          <td>2026-01-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="A fundamental challenge in studying therapy resistance is understanding whether it results from pre-existing cellular states (“priming”) or drug-induced changes (“adaptation”). While lineage barcoding enables retrospective analysis of cells before and after treatment, current methods struggle to efficiently capture rare lineages in single-cell RNA sequencing (scRNA-seq) or isolate multiple specific lineages simultaneously for functional study. To overcome these limitations, we developed CloneSweeper, a multiplexed lineage tracking platform that pools enrichment libraries to isolate or enrich multiple rare lineages. CloneSweeper utilizes a dual-function barcode expressed as both a Cas9 gRNA for live-cell sorting and a 3’ UTR transcript for high-recovery detection in 10x Genomics scRNA-seq. We applied CloneSweeper to a model of BRAF V600E melanoma, where we identified that resistance to targeted therapy emerges from a polyclonal population of rare, pre-existing lineages. By simultaneously targeting and enriching 21 distinct rare lineages prior to treatment, we defined a heritable, primed state characterized by de-differentiation and elevated mesenchymal markers. We demonstrate that these primed cells are not quiescent but instead exhibit upregulated inflammatory and stress response signaling, specifically via the AP-1 and NF-κB1 pathways. CloneSweeper thus provides a robust framework for dissecting the molecular mechanisms of rare biological phenomena through simultaneous, multiplexed lineage isolation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76cfe1b1a36a95588a97f5cc3024b429ba771c00" target='_blank'>
              Multiplexed enrichment and tracking of lineages with CloneSweeper
              </a>
            </td>
          <td>
            Robert J. Vander Velde, Raymond W. S. Ng, Christopher Coté, Sydney M. Shaffer
          </td>
          <td>2026-01-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d81001c0250b1fa8d16724cb8f5b65fb66caff9" target='_blank'>
              Functional genomics identifies therapeutic options, biomarkers, and resistance mechanisms for high-grade gliomas
              </a>
            </td>
          <td>
            W. Lin, F. Kosari, James Smadbeck, Michael Barrett, Ryan W. Feathers, Jamie Hall, Dorsay Sadeghian, Sotiris Sotiriou, Sarah H. Johnson, F. Harris, Taylor Berry, A. Mccune, Stephen J. Murphy, Lindsey Kinsella, Lauren E. Haydu, Diogo Moniz-Garcia, Shannon P. Fortin Ensign, Lin Yang, Angela R. Emanuel, Leila A Jones, J. Schaefer-Klein, Cristiane M. Ida, Marcela A. Salomao, Wendy J. Sherman, Alyx B Porter, Steven S. Rosenfeld, S. Kizilbash, K. Jaeckle, Maciej M Mrugala, Aaron S. Mansfield, M. Borad, B. Bendok, Terry C Burns, A. Quiñones-Hinojosa, John C. Cheville, G. Vasmatzis, P. Anastasiadis
          </td>
          <td>2026-02-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Colorectal cancer (CRC) develops through evolutionary processes involving genomic alterations, epigenetic regulation, and microenvironmental interactions. While traditionally explained by the stepwise accumulation of driver mutations, contemporary evidence supports a ‘Big Bang’ model in which many early-arising clones expand simultaneously to establish extensive heterogeneity. We reviewed recent studies employing spatially resolved multi-omic sequencing of tumour glands combined with computational modelling. These approaches enable high-resolution reconstruction of clonal architecture, transcriptional states, and chromatin accessibility. Findings show that although early clonal mutations shape tumour expansion, gene expression variability can be independent of genetic ancestry and instead reflects phenotypic plasticity driven by microenvironmental cues. Epigenomic analyses identified recurrent somatic chromatin accessibility alterations in promotors and enhancers of oncogenic pathways, frequently in the absence of DNA mutations, suggesting alternative mechanisms of gene regulation. Immune-focused studies demonstrated that early silencing of antigen-presenting genes and loss of neoantigens facilitate immune escape despite active surveillance. CRC is shaped by an interplay of genome, epigenome, and immune evolution, with non-genetic mechanisms and tumour plasticity emerging as important drivers of progression and therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c00d0ede4db6fdd44f5a7c3b138a4161c7d8d0f7" target='_blank'>
              Advances in Colorectal Cancer Cell Biology and Clonal Evolution
              </a>
            </td>
          <td>
            Sopozme Toghey, Elizabeth J. Harvey-Jones, Jonathan D. Towler, Charlotte J. H. Hafkamp, Irene Y. Chong
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Phenotypic plasticity is a prominent cancer feature that contributes to metastatic potential and resistance to therapy across multiple cancer types. Cancer cell state transitions have been attributed to transcriptional programs, such as the AP1/TEAD-regulated gene network driving the mesenchymal-like (MES) phenotype. In addition, during dissemination, tumor cells are subjected to variable loads of physical mechanical pressure and constriction across transited tissue, which are thought to impact nuclear molecular crowding. How the interplay between mechanical pressure, global 3D nuclear architecture and transcriptional programs contributes to MES identity and metastatic adaptation remains unclear. Using cutaneous melanoma as a model for early dissemination, we integrate in vitro and in vivo epigenomic profiling with nanoscale imaging of cell lines and patient samples to investigate chromatin organization features underlying the MES phenotype. We find that in MES cells, CTCF is relocated from domain boundaries to regulatory regions of EMT-like genes, leading to reduced insulation, extended topological associated domains (TADs) and increased inter-domain contacts, and de novo formation of chromatin hubs. This conformational rewiring, along with loss of heterochromatin, supports nuclear deformability during invasion and dissemination. Conversely, physical constriction of melanocytic cells induces MES-like chromatin features —including CTCF repositioning and heterochromatin loss— and promotes metastasis in vivo. Similarly, pharmacological inhibition of the heterochromatin mark H3K9me3 triggers MES characteristics and increases invasiveness. These results demonstrate that metastatic competency involves both epigenetic and structural nuclear reprogramming, enabling shifts in gene networks and physical adaptability. Our findings reveal mechanistic links between nuclear architecture and aggressive tumor behavior, identifying potential biomarkers and therapeutic targets to intercept metastatic progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92c8bf70d1df18ff53fcb785d7213684e545662a" target='_blank'>
              Chromatin architecture and physical constriction cooperate in phenotype switching and cancer cell dissemination
              </a>
            </td>
          <td>
            Pietro Berico, Cody L Dunton, L. Almassalha, Amanda Flores-Yanke, Karla I Medina, Nicolas Acosta, Tara Muijlwijk, Catherine Do, Soobeom Lee, S. Edmiston, D. Corcoran, Allison Reiner, Caroline E. Kostrzewa, Kathy Dorsey, Milad Ibrahim, Ronglai Shen, Nancy E. Thomas, Amanda W. Lund, Ata S. Moshiri, Iman Osman, I. Aifantis, Jane A. Skok, Vadim Backman, Eva Hernando
          </td>
          <td>2026-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d73e2496f48eaf63e522263faed717c4d2cc83e" target='_blank'>
              Exploring the Role of APC in Modulating Chemotherapeutic Response in Triple-Negative Breast Cancer cells
              </a>
            </td>
          <td>
            T. Nair, M. K. Vanklompenberg, Jenifer R Prosperi
          </td>
          <td>2026-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce1dd81fa69ca7596c6e2f6203643899d73e28c1" target='_blank'>
              In vivo isogenic modelling unveils a TP53-mediated relapse phenotype in T-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Stéphanie Gachet, S. Quentin, L. Hernandez, Loïc Maillard, M. Passet, R. Kim, Hugo Bergugnat, Melha Benlebna, Maxime Boy, Véronique Parietti, P. Fenaux, André Baruchel, Hervé Dombret, N. Boissel, François Sigaux, Hugues de Thé, E. Clappier, J. Soulier
          </td>
          <td>2026-01-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="Recent advances in genomic technologies have greatly enhanced our understanding of genotype–phenotype relationships and improved diagnostic of genetic diseases. However, the dissection of complex structural variants remains challenging due to the limitations of current methods in resolving their breakpoint and interpreting phenotypes involving multiple disrupted genes. In this study, we demonstrate how an integrative approach—combining cytological, genomic, and transcriptomic methods—enables the detection, structural and functional characterization of a complex structural variants affecting the MBD5, USP34, and XPO1 genes. Our findings underscore the utility of the Exo-C, a modified chromosome conformation capture technique, in resolving complex rearrangements. We also report, for the first time, a composite neurodevelopmental phenotype resulting from the combined effects of MBD5-associated intellectual disability and 2p15p16.1 microdeletion syndromes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c987c3067a72472e130e25f49fdb579eae34de80" target='_blank'>
              Zooming into rearranged genome: applying pipeline of cytological, genomic, and transcriptomic methods for structural variant interpretation
              </a>
            </td>
          <td>
            M. Gridina, T. Lagunov, P. Belokopytova, N. Torgunakov, A. Nurislamov, Darya A Yurchenko, Zhanna G Markova, Tatiana V Markova, Yana Stepanchuk, Galina Koksharova, Pavel Orlov, Anna Subbotovskaia, O. Ryzhkova, Nadezhda V. Shilova, Veniamin Fishman
          </td>
          <td>2025-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Immune dysfunction in cancer is enacted by multiple programs, including tumor cell-intrinsic responses to distinct immune subpopulations. A subset of these immune evasion programs can be systematically recapitulated through direct tumor-immune interactions in vitro. Here, we present an integrated, high-throughput single-cell CRISPR screening framework focused on the protein kinome for mapping the tumor-intrinsic regulation of T cell-driven immune pressure in glioblastoma (GBM). We combine pooled CRISPR interference and activation (CRISPRi/a) with immune-matched NY-ESO-1 antigen-specific allogeneic GBM-T cell co-culture and massively multiplexed single-cell transcriptomics to systematically quantify how genetic perturbation reshapes baseline tumor state and adaptive responses across graded effector-to-target ratios. We further leverage deep generative models for analyzing pooled CRISPR screens to decipher the effects of genetic perturbations on the mechanisms of tumor resistance. This framework resolves distinct modules of immune evasion and survival, including the regulation of the antigen-presentation machinery, interferon/NF-κB signaling, oxidative stress resilience, and checkpoint/cytokine programs, while identifying perturbations that reroute the continuous tumor transcriptional trajectory induced by T cell engagement. A secondary chemical screen in patient-derived GBM cultures identified putative kinase targets of immune evasion phenotypes (e.g., EPHA2 and PDGFRA), whose inhibition leads to the blockade of evasive programs and enhances T cell-mediated GBM killing. Together, this workflow provides a scalable blueprint for comprehensive charting of the genetic control of tumor-immune interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e76a058bed6b1c1e4d91cd692a05e66fab6beee" target='_blank'>
              Mapping kinase-dependent tumor immune adaptation with multiplexed single-cell CRISPR screens
              </a>
            </td>
          <td>
            Lingting Shi, Ross M. Giglio, Qingyuan Cai, Mathini Vaikunthan, Justin Hong, Abdullah Naqvi, Marina Milea, Hannah Khanshali, Anna C. M. Schoonen, Nicholas Hou, Jonathan Guo, Melanie Fraidenburg, Xumin Shen, Seth Malinowski, K. L. Ligon, Raúl Rabadán, Elham Azizi, José L. McFaline-Figueroa
          </td>
          <td>2026-01-09</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b10c93153ed6d7f62167729a4cbefd8ea1ed9de" target='_blank'>
              Adaptive regulation of dNTP homeostasis confers osimertinib resistance in EGFR mutant non-small cell lung carcinoma
              </a>
            </td>
          <td>
            Qian Xie, Yingying Wang, Yi Lei, Anthony Fernandez, Jessica D. Hess, Lin Yang, Terence Williams, Li Zheng, Binghui Shen, Min Li
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Immune aging is being increasingly recognized as a critical barrier to effective cancer immunotherapy, as the aged tumor microenvironment (TME) drives T cell dysfunction and impairs immune control of cancer1–4. However, the key molecular drivers of this process as well as potential targets to rescue T cell dysfunction in aged tumors remain incompletely understood. Therefore, we performed in vivo single-cell CRISPR screens in CD8⁺ T cells within aged tumors and tumor-draining lymph nodes (tdLNs). We identified Dusp5 and Zfp219 as key regulators of T cell persistence and effector differentiation in aged hosts. Loss of Dusp5, a negative regulator of ERK signaling, increased ERK1/2 phosphorylation and enhanced T cell proliferation in both young and aged tumors. In contrast, loss of Zfp219, a transcriptional repressor, induced epigenetic reprogramming of cytotoxic gene programs, thereby increasing granzyme secretion and enhancing antitumor immunity. Moreover, expression of the human ortholog gene ZNF219 is increased within intratumoral CD8⁺ T cells in older cancer patients. High ZNF219 expression correlates with poorer survival following immune checkpoint blockade (ICB) and reduces persistence of human intratumoral T cells. Notably, Zfp219 ablation synergized with anti-PD-1 blockade in mice to expand effector-like CD8⁺ T cells, leading to significantly enhanced anti-tumor immunity and tumor clearance in aged hosts. Together, these findings highlight Dusp5 and Zfp219 as critical drivers of age-related T cell dysfunction and as potential therapeutic targets to rejuvenate T cell antitumor immunity in older cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/017b5a7413706a4c2c7fbdc538a3a98539127c42" target='_blank'>
              CRISPR screens identify targets to rescue age-related T cell dysfunction in cancer
              </a>
            </td>
          <td>
            Alex C. Y. Chen, Keely Ji, Cansu Yerinde, Nelson H. Knudsen, Kevin Bi, Daniela Martinez, Thomas J Carmona-LaSalle, Kazuhiro Taguchi, Katherine H Xu, Elizabeth M. Seider, Marc A. Schwartz, Maria Zschummel, Linda T. Nieman, Kathleen B. Yates, T. R. Mempel, R. Manguso, N. Hacohen, Debattama R. Sen
          </td>
          <td>2026-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>118</td>
        </tr>

        <tr id="Oncogene amplifications fuel some of the most lethal, therapy‑refractory cancers, yet remain clinically untargeted. We report a single‑guide CRISPR/Cas9 strategy that converts the sheer copy‑number excess of oncogene amplicons into an Achilles’ heel. A solitary intronic double‑strand break is innocuous in diploid genomes but collapses oncogene amplification‑positive cells across neuroblastoma, small‑cell lung and colorectal carcinoma models, driving > 90% loss of viability, G₂/M blockade and catastrophic DNA‑damage signalling. Amplified‑locus cleavage rewires transcription toward cell death activation, necroptosis and cGAS–STING–mediated immunogenic cell death, enabling dendritic‑cell cross‑priming and T‑cell activation and proliferation. In xenografts, delivery of the intronic sgRNA shrinks tumours by 90%, prolongs survival and remodels the innate tumour microenvironment. Deep sequencing confirms negligible off‑target editing, and combination with doxorubicin achieves supra‑additive killing. These findings establish amplification density, not sequence content, as a tractable, tumour‑exclusive target and unveil a dual‑action platform that is simultaneously cytotoxic and immunostimulatory. Editing of tumor amplifications therefore offers a blueprint for translating copy‑number aberrations into precision genome‑editing therapies for treatment‑resistant cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8784639587a78e80b9c245135bde519d167d493" target='_blank'>
              Selective genome editing of amplified oncogenes triggers immunogenic cell death and tumor remodeling
              </a>
            </td>
          <td>
            A. Nieto-Sanchez, M. Martinez-Lage, P. Puig-Serra, S. Carpintero, A. Alonso-Yanez, P. Ojeda-Walczuk, M. Ibañez-Navarro, G. Pita, FJ. Moya, C. Moreno, MC. Martin, R. Alonso, R. Nuñez-Torres, VJ. Sanchez-Arevalo Lobo, L. Alonso-Guirado, N. Malats, A. González-Neira, L. Fernandez, P. Roda-Navarro, R. Torres-Ruiz, S. Rodríguez-Perales
          </td>
          <td>2025-12-22</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="Highlights What are the main findings? We established and characterized a unique pair of isogenic cervical cancer cell lines (AdMer35/AdMer43) from a patient with squamous cell carcinoma of the cervix before and after chemoradiotherapy, revealing that resistance is not caused by a single factor but by complex reorganization of cancer cells. Single-cell analysis identified key resistance hallmarks: transcriptional reprogramming towards embryonic stemness (HOX, POU5F1), a shift from fibrillar to non-fibrillar collagen expression, and activation of interferon/inflammatory pathways. What are the implications of the main findings? The study maps a remodeled tumor ecosystem with a metabolically reprogrammed, pro-tumorigenic senescent-like cell population and a progenitor-like cell reservoir, identifying them as critical therapeutic targets for eliminating resistant cell pools. The identified key pathways, Wnt, TGFβ, BMP, and the specific ligands EREG and SEMA3C, along with the metabolic vulnerabilities of the senescent compartment, provide a robust framework for future therapeutic exploration. Abstract Cervical cancer remains a significant global health burden, with chemoradioresistance representing a major obstacle to successful treatment. To elucidate the mechanisms underlying this resistance, we established a unique pair of isogenic primary cervical cancer cell lines, AdMer35 and AdMer43, obtained from a patient with squamous cell carcinoma of the cervix before and after radiation therapy. The aim of our study was to characterize the transcriptomic and cellular heterogeneity of these cells. We conducted an in-depth comparative analysis using single-cell RNA sequencing. Analysis of this paired, patient-derived isogenic model suggests that chemoradioresistance can arise through coordinated multilevel cellular adaptations. Resistant AdMer43 cells demonstrated transcriptional reprogramming, with the upregulation of embryonic stemness factors (HOX, POU5F1, SOX2), a shift in extracellular matrix from fibrillar to non-fibrillar collagens, and activation of inflammatory pathways. We identified and characterized critical cell-state dynamics: resistant cells exhibited a remodeled ecosystem with a metabolically reprogrammed senescent-like cell population showing an enhanced pro-tumorigenic communication via EREG, SEMA3C, BMP, and WNT pathways. Furthermore, we identified a progenitor-like cell population with a minimal CNV burden, potentially serving as a reservoir for tumor persistence. These findings offer novel insights for developing targeted strategies to eliminate resistant cell pools and improve cervical cancer outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ccd2a5a0a730a290acb2a66420d2bdad799661d" target='_blank'>
              Single-Cell Transcriptomic Landscape of Cervical Cancer Cell Lines Before and After Chemoradiotherapy
              </a>
            </td>
          <td>
            Dmitriy V Semenov, Irina S Tatarnikova, A. S. Chesnokova, Vadim A. Talyshev, M. Zenkova, E. Logashenko
          </td>
          <td>2026-01-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d29acb3217c08dff3be7e7d47281f950232e5a4" target='_blank'>
              Identification of an epigenetically and phenotypically distinct peritumoral glioblastoma cell population linked to inferior patient outcome
              </a>
            </td>
          <td>
            I. Neves, Xi Lu, Veera Jokinen, Francesco Latini, N. Maturi, A. Sundström, Yonglong Dang, Irem Uppman, Tobias Bergström, Pengwei Xing, M. Ryttlefors, Xingqi Chen, F. Swartling, L. Uhrbom
          </td>
          <td>2026-01-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f09bfb27dbaed264c5656346c24d74a25e18488" target='_blank'>
              An Atlas of Extrachromosomal DNA Structures Illuminates Its Evolution and Biogenesis in Cancer
              </a>
            </td>
          <td>
            Chee Hong Wong, Kuo-Chen Wei, Kuan-Yu Kung, Li-Hao Yang, Yin-Cheng Huang, Chiung-Yin Huang, I-Jiuan Chung, Meng-Fan Huang, Pei-Hua Peng, Meihong Li, C. Ngan, Kou-Juey Wu, Chia-Lin Wei
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d9b190b3682be1c45bebfc6d6804813d536f4d9" target='_blank'>
              Cellular mechanisms of acquired drug resistance against regorafenib in colon cancer cells
              </a>
            </td>
          <td>
            Deepu Sharma, Fayyaz Rasool, Binayak kumar, R. Verma, Ashutosh Singh, Sri Krishna Jayadev Magani
          </td>
          <td>2026-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf830ded86f6aa48fcef1356b0883f8c090233e1" target='_blank'>
              Patterns of hypermutation shape tumorigenesis and immunotherapy response in mismatch-repair-deficient glioma
              </a>
            </td>
          <td>
            Nick Fernandez, Yuan Chang, N. Nunes, J. Dimayacyac, A. Levine, Amit Ringel, L. Negm, A. B. Ercan, J. Hess, Olfat Ahmad, Caitlin Lee, L. Stengs, V. Bianchi, M. Edwards, Sheradan Doherty, J. Chung, L. Nobre, Julie Bennett, A. Dodgshun, David T. W. Jones, S. Pfister, A. Villani, David Malkin, Vijay Ramaswamy, A. Huang, É. Bouffet, M. Aronson, Peter B. Dirks, A. Shlien, Gaddy Getz, Y. Maruvka, B. Ertl-Wagner, Cynthia E. Hawkins, Anirban Das, U. Tabori
          </td>
          <td>2025-12-22</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d6260543e09115d78b28611e811c1654d5ea522" target='_blank'>
              Mutant ASXL1 Drives Transcriptional Activation and Repression in Human Hematopoiesis
              </a>
            </td>
          <td>
            Madison L. Hall, Aliya Quintal, Samantha Worme, Thai T. Nguyen, Zachary Schonrock, Mitsuhiro Tsuchiya, Sonia N. Acharya, Hanqian L Carlson, Itallia V. Pacentine, Shawn B Shrestha, Jommel Macaraeg, Randall Armstrong, James McGann, Sara Evans-Dutson, Theresa A. Lusardi, Brendan L. O’Connell, Andrew C. Adey, G. Yardımcı, Julia E. Maxson, Theodore P. Braun
          </td>
          <td>2026-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="The efficacy of anti-cancer therapies is severely limited by the emergence of drug resistance. While genetic drivers are well-characterized, growing evidence suggests that non-genetic mechanisms, particularly those involving drug-tolerant persisters (DTPs), play a pivotal role in solid tumor relapse. To elucidate the evolutionary dynamics of DTP-induced resistance, we develop a stochastic agent-based model (ABM) of solid tumor evolution that couples macroscopic population dynamics with microscopic epigenetic state inheritance during the cell cycle. Our simulations accurately reproduce the temporal progression of relapse observed in experimental studies, capturing the dynamic transition from sensitive cells to DTPs, and ultimately to stable resistant phenotypes under prolonged therapy. By explicitly modeling the epigenetic plasticity of individual cells, our model bridges the gap between cellular heterogeneity and population-level tumor evolution. Furthermore, we performed \textit{in silico} clinical trials using virtual patient cohorts to evaluate therapeutic outcomes, demonstrating that optimized adaptive treatment strategies can significantly delay tumor relapse compared to standard dosing. This study provides a quantitative framework for dissecting DTP-driven resistance mechanisms and designing more effective, biologically informed therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5335957f9dd7ef1ab2ffbe1285dc0a351204f56d" target='_blank'>
              Epigenetic state inheritance drivers drug-tolerant persister-induced resistance in solid tumors: A stochastic agent-based model
              </a>
            </td>
          <td>
            Xiyin Liang, Jinzhi Lei
          </td>
          <td>2026-01-30</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b87e3c57fa65cdcf7393f7997f48f33e896d0061" target='_blank'>
              A cross-species single-cell epigenome kidney atlas identifies epithelial cells as a driver of epigenetic aging
              </a>
            </td>
          <td>
            Hyeonsoo Jeong, B. Lake, D. Diep, Xuwen Li, Qi Yan, D. Gisch, Stephanie Reinert, Michael T. Eadon, J. Gaut, Sanjay Jain, Kun Zhang
          </td>
          <td>2026-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Understanding the molecular mechanisms underlying T-cell acute lymphoblastic leukemia (T-ALL) is essential for developing more effective therapeutic strategies. Despite therapeutic advances, the role of RNA-binding proteins in the pathogenesis of T-ALL remains poorly understood. Here, we investigate the RNA-binding protein Quaking (QKI), identifying it as a key regulator of splicing with tumor-suppressive properties in T-ALL. Through the analysis of two independent pediatric T-ALL cohorts, we demonstrate that QKI expression is frequently reduced in T-ALL, particularly within the HOXA subtype, and this reduction correlates with poor overall and event-free survival. Using T-ALL cell lines, we show that QKI depletion induces widespread splicing alterations, with numerous events corroborated in patient samples. Transcriptome profiling indicates that QKI downregulation leads to broad changes in gene expression, notably affecting pathways related to cell cycle progression, cholesterol homeostasis, and epithelial-mesenchymal transition. Functional assays demonstrate that QKI overexpression in T-ALL cells significantly reduces cell proliferation, induces G0/G1 cell cycle arrest, and limits leukemia progression and dissemination, ultimately improving survival in xenograft models. Together, these findings provide compelling evidence that QKI functions as a regulator of RNA splicing with tumor-suppressive activity in T-ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca5ec1e411749bba69b71a539ea37c30a49dc190" target='_blank'>
              QKI dysregulation induces extensive splicing changes in T-cell acute lymphoblastic leukemia.
              </a>
            </td>
          <td>
            Bruno Palhais, Nitesh D. Sharma, Igor Fijalkowski, T. Pieters, D. Deforce, F. Nieuwerburgh, Pieter Mestdagh, Panagiotis Ntziachristos, K. Matlawska-Wasowska, P. V. Vlierberghe
          </td>
          <td>2026-01-15</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/130c768582fb49ab11418fbd8cb89d6bb16d6b81" target='_blank'>
              Single-cell perturbations decipher ribosomal stress-surveillance regulators in type 2 diabetes
              </a>
            </td>
          <td>
            J. Nan, Xianglong He, Xiaoping Liu, Jianrong Ran, Jiahuan Chen, Pengxiao Li, Dongxue Liu, Yanan Sun, Aijing Shan, Xiuli Jiang, Jing Xie, Weiqing Wang, Guang Ning, Yanan Cao
          </td>
          <td>2026-01-01</td>
          <td>Nature Metabolism</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) occurs as a complex, multifaceted event driven by the interplay of tumor-permissive genetic mutations, the nature of the cellular origin, and microenvironmental stress. In this study, using primary human pancreatic acinar 3D organoids, we performed a CRISPR-KO screen targeting 199 potential tumor suppressors curated from clinical PDAC samples. Our data revealed significant enrichment of a list of candidate genes, with neurofibromatosis type 2 associated gene (NF2) emerging as the top target. Functional validation confirmed that loss of NF2 promoted the transition of PDAC to an invasive state, potentially through extracellular matrix modulation. NF2 inactivation was found to enhance PDAC cell fitness under nutrient starvation. This adaptation not only reinforced the oncogenic state but also conferred therapeutic resistance. Additionally, we found that NF2 loss was associated with fibroblast heterogeneity and cancer-stroma communication in tumor evolution. These findings establish NF2 as a critical tumor suppressor in PDAC and uncover its role in mediating nutrient adaptation and drug resistance. Importantly, this study provides additional insights into drug resistance mechanisms and potential therapeutic targets in PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58c5d227985bf39716bd4f69d5a354d37cc46442" target='_blank'>
              NF2 loss malignantly transforms human pancreatic acinar cells and enhances cell fitness under environmental stress
              </a>
            </td>
          <td>
            Yi Xu, Michael H. Nipper, Angel A. Dominguez, Chenhui He, Francis E. Sharkey, Sajid Khan, Han Xu, Daohong Zhou, Lei Zheng, Yu Luan, Jun Liu, Pei Wang
          </td>
          <td>2026-01-02</td>
          <td>The Journal of Clinical Investigation</td>
          <td>1</td>
          <td>8</td>
        </tr>

        <tr id="Tumor cell plasticity drives metastasis and therapy resistance, yet its regulation by oncoprotein dosage dynamics remains poorly understood. In Ewing sarcoma (EwS), variations in EWS::FLI1 (EF) fusion oncoprotein activity have been associated with epithelial-mesenchymal plasticity (EMP). Using degron technology, we precisely modulated endogenous EF in EwS cells and linked phenotypic states to distinct oncoprotein dosages. Strikingly, modest EF depletion promoted a pro-metastatic phenotype that diminished upon near-complete EF loss, revealing a paradoxical effect of submaximal EF inhibition. Nascent RNA-sequencing uncovered distinct gene clusters with heterogenous transcriptional responses to graded EF loss. Genes most sensitive to subtle EF depletion harbored GGAA microsatellites within EF-bound enhancers, while chromatin profiling uncovered candidate cofactors regulating EF-repressed EMP programs. Transient EF depletion followed by rapid restoration, modelling oncoprotein fluctuations, caused persistent dysregulation of genes functionally linked to enhanced extravasation and metastatic burden in preclinical models. This study highlights the therapeutic challenge of incomplete EF elimination, serving a paradigm in which oncoprotein dosage dynamics act as non-genetic drivers of disease progression and reveal novel vulnerabilities of advanced disease. The partial and transient suppression of the EF oncoprotein was found to paradoxically promote Ewing sarcoma metastatic potential through persistent transcriptional dysregulation, establishing the oncoprotein dosage dynamics as a key regulator of metastatic plasticity and therapeutic vulnerability. Graded EF depletion using dTAGV-1 elicited a non-linear phenotypic response, with modest EF loss being sufficient to induce a pro-metastatic state. Integration of acute transcriptional responses with chromatin profiling led to the identification of EF co-regulators. EF restoration following transient depletion resulted in lasting dysregulation of pro-metastasis-related genes and emergence of hybrid epithelial-mesenchymal states, thereby enhancing extravasation and lung metastasis in vivo. Partial or transient EF inhibition may promote disease progression and worsen clinical outcomes, highlighting that therapeutic strategies targeting EF must achieve complete and sustained suppression. Graded EF depletion using dTAGV-1 elicited a non-linear phenotypic response, with modest EF loss being sufficient to induce a pro-metastatic state. Integration of acute transcriptional responses with chromatin profiling led to the identification of EF co-regulators. EF restoration following transient depletion resulted in lasting dysregulation of pro-metastasis-related genes and emergence of hybrid epithelial-mesenchymal states, thereby enhancing extravasation and lung metastasis in vivo. Partial or transient EF inhibition may promote disease progression and worsen clinical outcomes, highlighting that therapeutic strategies targeting EF must achieve complete and sustained suppression. The partial and transient suppression of the EF oncoprotein was found to paradoxically promote Ewing sarcoma metastatic potential through persistent transcriptional dysregulation, establishing the oncoprotein dosage dynamics as a key regulator of metastatic plasticity and therapeutic vulnerability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5051c5027b542c8e38511f6f4e8d71524906c31f" target='_blank'>
              Modelling EWS::FLI1 protein fluctuations reveal determinants of tumor plasticity in Ewing sarcoma
              </a>
            </td>
          <td>
            Veveeyan Suresh, Christoph Hafemeister, Andri Konstantinou, S. Grissenberger, C. Sturtzel, M. Zylka, F. Cidre‐Aranaz, A. Wenninger-Weinzierl, Karla Queiroz, D. Kurek, Martin Distel, Anna C. Obenauf, T. Grünewald, F. Halbritter, H. Kovar, Valerie Fock
          </td>
          <td>2026-01-03</td>
          <td>EMBO Molecular Medicine</td>
          <td>2</td>
          <td>32</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is a highly heterogeneous malignancy with a poor prognosis and limited response to current therapies. Cancer stem cells (CSCs) contribute to this heterogeneity, driving tumor progression, immune evasion, and therapeutic resistance. Epigenetic regulators have emerged as pivotal modulators of CSC phenotypes. The histone methyltransferase SETDB1, a key stemness factor in both embryonic and adult stem cells, catalyzes H3K9 trimethylation (H3K9me3) and acts as a major oncogenic driver in a wide range of solid and hematological malignancies. Yet, its specific role in maintaining stemness and contributing to therapeutic resistance in HCC remains poorly defined. Integrative analyses of bulk and single-cell transcriptomic datasets from public HCC cohorts were performed. Tumors were stratified by SETDB1 expression, followed by pathway enrichment, immune deconvolution, and stemness scoring. Associations with clinical outcomes, immune phenotypes, drug resistance signatures, and chromatin-binding partners (DNMT3A, TRIM28, HDAC1) were also explored. The role of SETDB1 was then examined in vitro, where a stem-like phenotype was induced in HCC cell lines under hypoxic conditions, and patterns of SETDB1 expression and stemness marker levels were assessed following paclitaxel treatment. SETDB1-low tumors were enriched in well-differentiated, immunologically active subclasses and were associated with increased predicted responsiveness to immune checkpoint blockade (ICB), sorafenib, and transarterial chemoembolization. Conversely, SETDB1-high tumors exhibited transcriptional features of dedifferentiation, elevated stemness scores, and enrichment in aggressive molecular subtypes. These tumors also displayed higher TP53 mutation rates, reduced immune infiltration, immune exclusion signatures, and activation of pathways linked to ferroptosis and ABC transporter dysfunction, consistent with resistance to ICB and conventional therapies. SETDB1 expression was also associated with hypoxia-related pathways, suggesting a role in maintaining CSC niches. In vitro, hypoxia-induced stemness coincided with increased SETDB1 levels, whereas paclitaxel treatment decreased both SETDB1 expression and stemness markers. A chromatin risk score incorporating SETDB1 and its partners predicted poor disease-free survival independently of clinical parameters. SETDB1 defines a stemness-enriched, immune-resistant HCC subtype associated with poor outcomes and therapeutic failure. Correlations between SETDB1 expression and stem-like features in vitro suggest a potential role in maintaining CSC phenotypes, supporting its relevance as a biomarker and candidate for epigenetic- and CSC-directed therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c393716a837e5b1ef0091b6385eb92c64be4ab4" target='_blank'>
              The epigenetic regulator SETDB1 as a key component of cancer stem cells and drug resistance in primary liver cancer
              </a>
            </td>
          <td>
            Maël Padelli, Christophe Desterke, Aurore Devocelle, Georges Uzan, Antoinette Lemoine, Julien Giron-Michel
          </td>
          <td>2026-01-06</td>
          <td>Cellular Oncology (Dordrecht, Netherlands)</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Alternative splicing (AS) is a fundamental mechanism governing transcriptomic diversity and cellular identity. Although 293T (human embryonic kidney) and A549 (human lung adenocarcinoma) cell lines are widely used, cell-type-specific splicing dynamics—including responses to receptor overexpression—remain incompletely characterized. To address this, we integrated Oxford Nanopore long-read sequencing with BGI short-read data to profile transcriptomes under both basal and GPCR-overexpressing conditions (ADORA3 in 293T; P2RY12 in A549). Full-length isoform analysis using FLAIR and SQANTI3 revealed extensive transcriptomic complexity, including 18.02% novel isoforms in 293T and 19.52% in A549 cells. We found that 293T cells exhibited a stable transcriptome architecture enriched in splicing-related pathways, whereas A549 cells underwent broader transcriptional remodeling linked to tumorigenic processes. These findings suggest that 293T cells may be a suitable model for investigating splicing regulation, while A549 cells could serve as a relevant system for exploring tumor-related transcriptome dynamics. Our work elucidates context-dependent AS regulation and underscores the value of integrating long-read sequencing with FLAIR/SQANTI3 for dissecting cell-state-specific transcriptome dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e55019c610b32f15ff9e78800a5fca309bcf3247" target='_blank'>
              Long-Read Sequencing Reveals Cell- and State-Specific Alternative Splicing in 293T and A549 Cell Transcriptomes
              </a>
            </td>
          <td>
            Xin Li, Hanyun Que, Zhaoyu Liu, Guoqing Xu, Yipeng Wang, Zhaotong Cong, Liang Leng, Sha Wu, Chunyan Chen
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background The complexities of nucleotide metabolism in breast cancer (BC) cells are not yet fully understood. A deeper exploration of the various tumor subpopulations and their interactions with the tumor microenvironment (TME) could provide important insights into these clinically relevant signaling pathways. Methods We integrated five distinct single-cell enrichment scoring methodologies to conduct a comprehensive enrichment analysis of BC cell populations. The analytical findings underwent subsequent validation using an independent single-cell cohort. Tumor cell clusters were categorized based on their average enrichment scores. Functional analyses were carried out using several tools, including CellChat, Monocle, CopyKAT, SCENIC, and CytoTRACE. The RCTD method was then employed to map the single-cell clusters onto spatial transcriptomics data, facilitating the evaluation of cellular dependencies and pathway activities to differentiate tumor cell subtypes. A prognostic framework was subsequently established using large-scale transcriptomic datasets, enabling prediction of immunotherapy responsiveness. Experimental validation further confirmed expression patterns of pivotal genes implicated in therapeutic outcomes. Results Tumor cells exhibit significantly upregulated nucleotide metabolic activity, enabling their classification into two distinct subgroups: NUhighepi and NUlowepi. Cells within the NUhighepi subgroup demonstrate pronounced malignant phenotypes. Intercellular communication analysis performed with the stLearn platform revealed robust interactions between NUhighepi cells and fibroblasts. Supporting this finding, spatial transcriptomic analysis via MISTy revealed a distinct dependency of NUhighepi on fibroblasts. A robust prognostic model, developed using various machine learning algorithms, was able to predict survival outcomes and responses to immunotherapy. Furthermore, targeted drugs were identified for both the high and low scoring groups. Experimental investigations confirmed the expression of core genes in different breast cancer cells. Discussion In conclusion, we developed a nucleotide metabolism-derived prognostic signature for BC, with DCTPP1 highlighted as a promising biomarker and therapeutic target. These findings provide a valuable framework for early clinical intervention and show promising potential for predicting responses to immunotherapy in BC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/502ca564395f021e293ba0e223c1f50472f85541" target='_blank'>
              Spatial transcriptome and single-cell sequencing reveal the role of nucleotide metabolism in breast cancer progression and tumor microenvironment
              </a>
            </td>
          <td>
            Yuejun Pan, Yong Xu, Ke Gong, Songlin Yuan
          </td>
          <td>2026-01-14</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The topoisomerase I inhibitor topotecan is an effective chemotherapeutic agent for retinoblastoma; however, treatment resistance remains a major clinical challenge, and its mechanisms remain elusive. Using genome-wide CRISPR-Cas9 knockout screening, we identified ELF2 as a key gene involved in topotecan resistance. Here, we show that surviving retinoblastoma cells exposed to topotecan showed progressively decreased ELF2 expression, accompanied by reduced apoptosis. In a mouse xenograft model, ELF2 disruption diminished the antitumor efficacy of topotecan, with ELF2-knockout cells exhibiting reduced topotecan-induced apoptosis. RNA sequencing further revealed that the MT-CYB pathway, associated with ATP synthesis, contributes to ELF2-mediated resistance. Importantly, clinical analysis demonstrated a correlation between ELF2 expression and tumor volume in retinoblastoma patients treated with topotecan. Together, these findings interrogate the mechanisms underlying topotecan resistance in retinoblastoma and suggest ELF2 as a potential therapeutic target to overcome drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dffc85985364a8e0c2b935cb6a150df86f47c5ef" target='_blank'>
              Loss of ELF2 drives topotecan resistance in retinoblastoma revealed by genome-wide CRISPR-Cas9 screening
              </a>
            </td>
          <td>
            Jingyi Jiang, Zihua Jiang, Qian-Jiang Luo, Xi Chen, Jiejie Zhuang, Jiaxin Chen, Qingqing Mu, Jin Qiu, Yan Li, Shuxia Chen, Ping Zhang, Keming Yu, Shuilian Chen, Guei-Sheung Liu, Zhuang Jing
          </td>
          <td>2025-12-23</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Canine osteosarcoma (OS) is an aggressive bone cancer with remarkable similarity to human OS. Despite its value as a comparative oncology model, the drivers of genomic instability remain poorly defined, hindering cross-species insights into disease progression. We applied long-read PacBio HiFi whole genome sequencing to matched tumor and blood samples from four purebred dogs to generate high-resolution maps of somatic structural variants (SVs). Tumor genomes showed extensive rearrangement complexity, with satellite repeat regions, particularly SAT1_CF, enriched near somatic SV breakpoints and occurring within genomic contexts marked by focal hypomethylation. We also identified multiple extrachromosomal DNA (ecDNA) elements carrying oncogenes indicating ecDNA-mediated oncogene amplification. Together, these data provide an integrated genomic and epigenomic view of canine OS, implicating satellite repeats as recurrent substrates for rearrangement and ecDNA as a prominent mode of genome remodeling. This work advances our understanding of OS genome dynamics and establishes a comparative framework for investigating repeat-driven instability and oncogene amplification across species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5300af19b1916e9abe3db71ff1fa34ffceb5e24d" target='_blank'>
              Satellite DNA fragility accompanies complex genome rearrangements and ecDNA oncogene amplification in canine osteosarcomas
              </a>
            </td>
          <td>
            F. Yilmaz, Wonyoung Kong, Sabriya A. Syed, Francis H. O’Neill, Jody T. Lombardi, P. K. Ng, Ching C. Lau, Charles Lee
          </td>
          <td>2025-12-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Cisplatin-based chemotherapy remains a mainstay for the treatment of bladder cancer (BLCA); however, its clinical efficacy is frequently compromised by the emergence of chemoresistance, which leads to poor patient outcomes. Although known mechanisms—such as alterations in drug efflux, DNA repair, and key signaling pathways—have been implicated, they fail to fully explain the clinical complexity of cisplatin resistance, indicating that additional molecular drivers remain undiscovered. Keratin 14 (KRT14), an intermediate filament protein associated with aggressive BLCA subtypes, is consistently upregulated in cisplatin-resistant tumors, yet its precise mechanistic role in resistance remains unclear. In this study, we elucidate the functional contribution of KRT14 to cisplatin resistance in BLCA using patient-derived tissues, established cell lines, xenograft mouse models, and a suite of molecular interaction assays. Our results demonstrate that KRT14 is significantly upregulated in BLCA tissues, correlates with poor clinical prognosis, and functionally drives cisplatin resistance both in vitro and in vivo. Mechanistically, we identify a novel and direct interaction between KRT14 and the translation initiation factor eIF4H, specifically through the N-terminal Head domain of KRT14. This interaction modulates the association of eIF4H with the core eIF4F complex, thereby selectively promoting the translation of Acyl-CoA Oxidase 2 (ACOX2) mRNA through its 5’ untranslated region. We further show that ACOX2 is essential for mediating the effects of KRT14 on lipid metabolism, cell proliferation, survival, and ultimately, cisplatin resistance. Collectively, our findings reveal that KRT14 contributes to chemoresistance in BLCA not only via its structural roles but also by directly regulating translational machinery through eIF4H, leading to upregulation of the metabolic enzyme ACOX2. The newly defined KRT14–eIF4H–ACOX2 axis orchestrates lipid metabolic reprogramming and cell survival, underscoring the KRT14–eIF4H interface as a promising therapeutic target for overcoming cisplatin resistance in BLCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec144141be685aac475058d09e8da4ccea2f5711" target='_blank'>
              Cytoskeletal protein KRT14 governs cisplatin resistance by modulating eIF4H-dependent ACOX2 translation and lipid metabolism in bladder cancer
              </a>
            </td>
          <td>
            Shenghua Liu, Tingting Liu, Chen Yang, Z. Mou, L. Zou, Haowen Jiang
          </td>
          <td>2025-12-24</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Colorectal cancer (CRC) progression and therapy resistance are driven in part by metabolic reprogramming and the persistence of cancer stem-like cells (CSCs). The seven in absentia homolog 2 (SIAH2)/with-no-lysine kinase 1 (WNK1) signaling axis has emerged as a potential regulator of these processes, yet its functional role in CRC metabolism and tumor–stroma crosstalk remains incompletely understood. Integrated analyses of The Cancer Genome Atlas–Colon Adenocarcinoma (TCGA-COAD) and Gene Expression Omnibus (GEO, GSE17538) datasets revealed significant upregulation of SIAH2 and WNK1 in CRC tissues, with strong positive correlations to glycolysis- and hypoxia-associated genes, including PFKP, LDHA, BPGM, ADH1A, ADH1B, and HIF-1α. Single-cell and clinical profiling further demonstrated preferential enrichment of SIAH2 in undifferentiated, stem-like tumor cell populations. Functional studies across multiple CRC cell lines showed that SIAH2 silencing suppressed proliferation, clonogenic growth, tumor sphere formation, and cell-cycle progression, whereas SIAH2 overexpression exerted opposite effects. Seahorse extracellular flux analyses established that SIAH2 promotes glycolytic capacity and metabolic flexibility. At the protein level, SIAH2 regulated glycolytic enzymes and WNK1/hypoxia-inducible factor-1α (HIF-1α) signaling, effects that were amplified by cancer-associated fibroblast (CAF)-derived conditioned medium. CAF exposure enhanced SIAH2 expression, CSC spheroid growth, and resistance to fluorouracil, leucovorin, and oxaliplatin (FOLFOX) chemotherapy, whereas SIAH2 depletion effectively abrogated these effects. Collectively, these findings identify the SIAH2/WNK1 axis as a central metabolic regulator linking glycolysis, CSC maintenance, and microenvironment-driven therapy resistance in CRC, highlighting its potential as a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb269b1f5618b40199d4b3cc75e161d29b2f08d9" target='_blank'>
              SIAH2–WNK1 Signaling Drives Glycolytic Metabolism and Therapeutic Resistance in Colorectal Cancer
              </a>
            </td>
          <td>
            Kee-Thai Kiu, Cheng-Ying Chu, Yi-Chiao Cheng, Min-Hsuan Yen, Ying-Wei Chen, N. W. Pikatan, Vijesh Kumar Yadav, Tung-Cheng Chang
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="ABSTRACT Aims Mitochondrial DNA copy number (mtDNA-CN) is associated with several age-related chronic diseases and is a predictor of all-cause mortality. Here, we examine site-specific differential nuclear DNA (nDNA) methylation and differential gene expression resulting from in vitro reduction of mtDNA-CN to uncover shared genes and biological pathways mediating the effect of mtDNA-CN on disease. Materials and methods Epigenome and transcriptome profiles were generated for three independent human embryonic kidney (HEK293T) cell lines harboring a mitochondrial transcription factor A (TFAM) knockout generated via CRISPR-Cas9, and matched control lines. Results We identified 2924 differentially methylated sites, 67 differentially methylated regions, and 102 differentially expressed genes associated with mtDNA-CN. Integrated analysis uncovered 24 Gene-CpG pairs. GABAA receptor genes and related pathways, the neuroactive ligand signaling pathway, ABCD1/2 gene activity, and cell signaling processes were overrepresented, providing insight into the underlying biological mechanisms facilitating these associations. We also report evidence implicating chromatin state regulatory mechanisms as modulators of mtDNA-CN effect on gene expression. Conclusions We demonstrate that mitochondrial DNA variation signals to the nuclear DNA epigenome and transcriptome and may lead to nuclear remodeling relevant to development, aging, and complex disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcbea560ce049511e5c017437908e0cfbb595232" target='_blank'>
              Mitochondrial DNA copy number reduction via in vitro TFAM knockout remodels the nuclear epigenome and transcriptome
              </a>
            </td>
          <td>
            Phyo W. Win, Julia Nguyen, Elly H. Shin, T. S. Nagano, Brent Selimi, Katie Hong, A. Meybodi, Bradley P Yates, Emma V Burke, David E Carter, Gregory A Newby, Charles E Newcomb, D. Arking, C. Castellani
          </td>
          <td>2025-12-23</td>
          <td>Epigenomics</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="To probe drug mechanism of action (MOA) and interrogate the genetic architecture of human cells, we carried out isogenic genome-wide CRISPR/Cas9 knockout screens against 310 diverse drugs, bioactive compounds, and stress conditions. Stringent statistical correction for gene knockout fitness defects yielded a large-scale matrix of >12,000 high confidence chemical-genetic interactions (CGIs). This dataset revealed many previously unappreciated off-target effects for well-characterized compounds and novel MOAs for uncharacterized compounds. The CGI matrix uncovered dense genetic modules that yielded new biological insights into phospholipidosis, mitotic regulation, metabolism, the DNA damage response, and mTOR signaling. The dataset allowed identification of multi-drug sensitization and resistance mechanisms, inference of gene function, elaboration of cross-process connectivity, evaluation of the cell type specificity of CGIs, prediction of chemical synergism, and extensive annotation of understudied genes. This resource provides a map of the genetic landscape in human cells and a framework to help guide drug discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54ea6f9c264b3644d442991366676c1b8529f455" target='_blank'>
              A Chemical-Genetic Interaction Matrix Reveals Drug Mechanism and Genetic Architecture
              </a>
            </td>
          <td>
            J. Coulombe-Huntington, Thierry Bertomeu, Caroline Huard, Andrew Chatr-aryamontri, Daniel J. St-Cyr, María Sánchez-Osuna, David Papadopoli, Karine Normandin, Mohammadjavad Paydar, Shannon Mclaughlan, C. St-Denis, Li Zhang, Henry Say, Roger Palou, Chris Stark, Bobby-Joe Breitkreutz, Almer M. van der Sloot, Sandhya Manohar, H. Lavoie, Katherine L. B. Borden, B. Raught, Damien D’Amours, Frank Sicheri, Alain Verreault, S. Mader, Sylvain Meloche, M. Therrien, Pierre Thibault, Brian T. Wilhelm, Peter B. Dirks, J. Aitchison, E. Patton, Randall W. King, Philippe P. Roux, Guy Sauvageau, Trang Hoang, A. Marinier, Lea Harrington, Benjamin Kwok, Vincent Archambault, I. Topisirovic, Mike Tyers
          </td>
          <td>2026-01-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Segmental copy-number gains are major contributors to human genetic variation and disease, but how these alterations arise remains incompletely understood. Here, based on the analyses of both experimental evolution and human disease genomes, we describe a general mechanism of segmental copy-number gain from a rearrangement process termed ‘breakage–replication/fusion’. The hallmark genomic feature of breakage–replication/fusion is adjacent parallel breakpoints: two or more rearrangement breakpoints derived from replication of a single ancestral DNA end. We show that adjacent parallel breakpoints are a widespread feature of DNA duplications in human disease genomes and experimental models of chromothripsis. In addition to adjacent parallel breakpoints, breakage–replication/fusion also explains two other patterns of complex rearrangements with unclear provenance: chains of short (≤1 kb) insertions and high-level amplification consisting of inverted segments. Together, these findings revise the mechanistic model for chromothripsis and provide a new conceptual framework for understanding the origin of segmental DNA duplication during genome evolution. This paper introduces breakage–replication/fusion, a genomic rearrangement process underpinning three patterns of copy-number gains found in cancer and other diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/943c5640094a2cf860368d879c898eeb29d9cb6e" target='_blank'>
              A breakage–replication/fusion process explains complex rearrangements and segmental DNA amplification
              </a>
            </td>
          <td>
            Cheng-Zhong Zhang, Carlos Mendez-Dorantes, Kathleen H. Burns, David Pellman
          </td>
          <td>2026-01-01</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Genome-wide association studies (GWAS) identified over 600 loci containing single-nucleotide polymorphisms (SNPs) associated with type 2 diabetes (T2D), most of which reside in non-coding regions. Among the set of T2D SNPs, linking causal genome variants to disease risk experimentally has remained a challenge; however, advances in synthetic mammalian genome writing techniques now enable the delivery of multiple haplotypes to human induced pluripotent stem cells (hiPSCs) to create a series of isogenic cell lines that can be differentiated and phenotyped in vitro. Here, to begin efforts in dissecting a T2D GWAS locus, we engineered an NKX6-3/ANK1 gene cluster knockout hiPSC line and introduced a landing pad facilitating the delivery of synthetic haplotype payloads. We built four haplotypes, including several that are not observed in nature, containing risk SNPs spanning the NKX6-3/ANK1 gene cluster using a method called “variant Switching Auxotrophic markers for Integration” (vSwAP-In), and integrated them precisely into hiPSCs. NKX6-3/ANK1 deletion blocked pancreatic progenitor and skeletal muscle differentiation, suggesting that NKX6-3 and ANK1 are required for early pancreatic and skeletal muscle development, and perhaps related to the existence of two nonoverlapping sets of SNPs in linkage disequilibrium that associate with the expression of the two adjacent genes. When NKX6-3/ANK1 T2D “Risk” haplotypes were reintroduced, skeletal muscle and pancreatic progenitor differentiation capabilities were restored. ANK1 expression was elevated in the ANK1 Risk and All-Risk haplotypes compared to the NKX6-3 Risk and Non-Risk haplotypes, establishing a functional experimental platform to examine risk SNP clusters in their native contexts. Overall, this work establishes a platform for the dissection of GWAS loci using synthetic haplotype genomics in hiPSCs. Significance Statement Genome-wide association studies have been used to identify disease-associated SNPs; however, most SNPs lie in non-coding regions, making functional experimentation difficult to perform. Using vSwAP-In, a yeast-based DNA variant-building method, and mSwAP-In, a mammalian genome engineering approach, we establish a platform for functional GWAS dissection in hiPSCs. This platform allows us to build DNA harboring virtually any combination of disease-risk SNPs, allowing for functional characterization of SNPs without the limitations of linkage disequilibrium. We demonstrate this approach using a Type 2 diabetes GWAS gene cluster, NKX6-3/ANK1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f19a7bbc6f3616a799aaa84ab7eb946f7cda112e" target='_blank'>
              Genome writing and Targeted Delivery of the NKX6-3/ANK1 gene cluster and its Type 2 Diabetes GWAS Variants to Human iPSCs
              </a>
            </td>
          <td>
            Noor Chalhoub, Arushi Varshney, Weimin Zhang, Skyler Uhl, J. M. Laurent, Colleen McLoughlin, Hannah J. Ashe, Xingrui Mou, Nathan Dale, Kiran Ramnarine, Dan Paull, Jordan Goldberg, Matthew T. Maurano, Ran Brosh, David Fenyö, Filippo Cipriani, Stephen C. J. Parker, J. Boeke
          </td>
          <td>2026-01-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>114</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/638cde0ab64b6b068ce716defb37f6b2cb4ab7b9" target='_blank'>
              Unraveling the molecular mechanisms underlying spontaneous multipolar mitosis through CIN-seq
              </a>
            </td>
          <td>
            Pin-Rui Su, Ting-Chun Chou, M. T. López-Cascales, Junfeng Hunag, Tsai-Ying Chen, Sjoukje van Wieren, Chan Li, Cecile Beerens, Li You, J. Storteboom, M. Chien
          </td>
          <td>2026-01-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Amino acid metabolism plays a crucial role in cancer progression and invasion by modulating immune responses within the tumor microenvironment and tumor immune response. Transmembrane transport of cellular amino acids is mediated by members of the cationic amino acid transporters SLC7 family, a group of cationic amino acid transporters. Dysregulation of SLC7 family genes can disrupt systemic homeostasis, contributing to a variety of diseases, including multiple cancer types. Additionally, SLC7 family members can alter amino acid metabolic plasticity, further influencing tumor adaptation and progression. However, the precise impact of SLC7 family genes within the cancer microenvironment remains largely unexplored. This study analyzed the expression patterns, genomic alterations (single nucleotide variants, copy number variations, methylation levels), pathway activations, and clinical and subtype associations of SLC7 family genes across The Cancer Genome Atlas (TCGA) pan-cancer cohort. SLC7 activity was quantified using the ssGSEA function of the “GSVA” package to generate the SLC7 score, which was found to correlate with oncogenic and immune-related pathways, including mTORC1 signaling, interferon-gamma response, and inflammatory pathways. In vitro validation was conducted using multiple cancer cell lines (A549, H1299, HCT116, FaDu, Panc-1) and counterparts (BxPC-3, SW480, H460, BEAS-2B, FHC, HaCaT). Gene knockdown was achieved by siRNA transfection, followed by quantitative PCR, Western blotting, and functional assays assessing proliferation, invasion, and ferroptosis (iron and GSH levels) under erastin treatment. SLC7 family genes showed significant associations with immune infiltration and poor prognosis across various cancers. The SLC7 score demonstrated strong positive correlations with malignant and immune-related pathways, such as mTORC1 signaling, interferon-gamma response, and inflammatory response across the pan-cancer cohort. Elevated SLC7 scores were also associated with enhanced immune-related characteristics, and SLC7 genes, particularly SLC7A8 and SLC7A9, emerged as promising predictive markers for response to immune checkpoint inhibitor therapy (Atezolizumab) in cancer patients. Among SLC7 family genes, SLC7A5 expression was markedly elevated in KRAS-mutant tumor cell lines and was required for KRAS-driven proliferation and invasion. SLC7A11 was highly expressed in multiple tumor types and protected cancer cells from ferroptosis by maintaining intracellular cystine uptake and redox homeostasis. Individuals exhibiting elevated SLC7 scores and aberrant expression of SLC7 family genes demonstrate significant immune-inflammatory tumor immune microenvironment (TIME) characteristics and predictive potential for immunotherapy. Thus, the SLC7 family genes have the potential to serve as valuable biomarkers and therapeutic targets for cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a15f37f25792e120e4c9d88c14c57b67424b4e09" target='_blank'>
              Integrative pan-cancer analysis reveals the prognostic and immunological significance of SLC7 family cationic amino acid transporters
              </a>
            </td>
          <td>
            Gang Peng, Hao Peng, Zhe Shao, Liangqi Jiang, Mingrui Li, Yang Li
          </td>
          <td>2026-01-08</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="High-grade serous ovarian carcinoma (HGSOC) is characterised by profound genomic instability and limited durable responses to standard therapy, leading to poor prognosis. The use of next-generation sequencing technologies has improved understanding of its molecular landscape, revealing consistent Tumour Protein p53 (TP53) mutations, homologous recombination defects, pathway alterations, and epigenetic dysregulation. Such genomic profiling now underpins the classification criteria between the ovarian cancer subtypes described by the Cancer Genome Atlas. Widespread chromosomal instability and pathogenic variants in multiple genes distinguish HGSOC from other subtypes of ovarian cancer and, further, from low-grade serous ovarian cancer. Importantly, the new-found understanding of the genomic landscape of HGSOC guides the use of platinum-based chemotherapies and Poly(ADP-ribose) Polymerase (PARP) inhibitors, with homologous recombination deficiency emerging as a cancer vulnerability that enhances treatment response. A combined multi-omics approach integrates transcriptomics, proteomics, metabolomics, and epigenomics to further the understanding of the characteristics, therapeutic targets and treatment resistance within HGSOC. Despite these advances, major challenges persist, including intratumoural heterogeneity and the poor diversity of genomic datasets. Artificial Intelligence (AI) technology, Clustered regularly interspaced short palindromic repeats (CRISPR)-based gene editing, neoantigen-guided immunotherapy and ovarian cancer vaccination indicate a promising future for genomics-guided interventions and support the integration of genomics within multi-omic approaches to improve HGSOC outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63375259d1ee4ee2c082f84da6b555791abc91c8" target='_blank'>
              High-Grade Serous Ovarian Carcinoma in the Genomics Era: Current Applications, Challenges and Future Directions
              </a>
            </td>
          <td>
            Molly Elizabeth Lewis, Chiara Caricato, Hannah Leigh Roberts, Subhasheenee Ganesan, Nadia Amel Seksaf, E. Maniati, Michail Sideris
          </td>
          <td>2026-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Autologous stem cell transplantation (ASCT) involves harvesting hematopoietic stem and progenitor cells (HSPCs) prior to chemotherapy and subsequent repopulation of the bone marrow. This process imposes a bottleneck, providing a framework to dissect the unresolved short- and long-term clonal dynamics during hematopoietic reconstitution. By integrating bulk error-corrected targeted sequencing of clonal hematopoiesis (CH)-associated genes with mitochondrial single-cell Assay for Transposase-Accessible Chromatin sequencing (mtscATAC-seq), we characterized mutational trajectories in frequently altered hematological genes and traced clonal evolution through somatic mitochondrial DNA variants, revealing post-transplant cellular heterogeneity and clonal architecture. Among 60 patients (multiple myeloma, n = 51; non-Hodgkin lymphoma, n = 6; Hodgkin lymphoma, n = 3), CH-associated mutations were identified in 53% pre-ASCT, predominantly involving DNMT3A. A transient increase in mutation counts and gene diversity occurred 10-25 days post-ASCT, with a gradual clonal expansion two years post-transplantation. Tandem ASCT amplified clonal complexity, with a twofold increase in mutation count and gene-level diversity, while preserving clonal trajectories across both transplant courses. Mitochondrial single-cell profiling in longitudinal samples of 3 patients showed patient-specific immune reconstitution and clonal dynamics, with balanced multilineage output from graft HSPCs. Collectively, our findings provide a firsthand comprehensive view of ASCT-induced clonal dynamics and immune reconstitution, paving the way for targeted gene-specific post-transplant monitoring.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fa98043059e18d9fd50446cb2c494c87e5e2054" target='_blank'>
              Dynamics of clonal hematopoiesis and cellular responses to stress-induced toxicity in autologous stem cell transplantation
              </a>
            </td>
          <td>
            Catarina M Stein, Raphael Hablesreiter, F. Christen, P. Löwe, Coral Fustero-Torre, Klara Kopp, B. Locher, Lena Nitsch, Robert Altwasser, Johanna Franziska Kerschbaum, L. Bullinger, Leif S. Ludwig, Paulina M Strzelecka, Frederik Damm
          </td>
          <td>2025-12-19</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Abstract Introduction Cancer cell proliferation occurs within the context of persistent genomic instability. In this review, we propose the RS–CCD–CIN axis as a systems-level framework in which replication stress (RS), cell cycle deregulation (CCD) and chromosomal instability (CIN) form an interdependent triad that shapes tumour evolution. This axis represents a constrained metastable state in which genomic instability is tolerated and buffered. The objective of this review is to synthesize the current understanding of how the RS–CCD–CIN axis contributes to tumour heterogeneity, adaptability and therapy response. Discussion Evidence indicates that RS, CCD and CIN operate as a dynamic, interconnected network rather than as independent processes. Replication stress induces DNA damage and mutagenesis, while partial checkpoint disruption permits cells with unresolved lesions to proliferate. Chromosomal instability generates both structural and numerical alterations, contributing to intratumoural heterogeneity. Together, these processes facilitate adaptation to environmental and therapeutic pressures. Extrachromosomal DNA, micronuclei formation and cytosolic DNA signalling, including the cGAS–STING pathway, connect genomic instability to adaptive responses and immune modulation. Single-cell and spatial profiling reveal temporal and spatial variability in RS, CCD and CIN states, highlighting the limitations of static biomarkers. Therapeutically, targeting individual components often yields limited durability, whereas approaches that simultaneously perturb multiple aspects of the RS–CCD–CIN axis may improve clinical outcomes. Conclusions This review highlights the RS–CCD–CIN axis as a fragile and metastable architecture that supports cancer evolution, while also being susceptible to collapse. A deeper understanding of this interconnected framework may inform the development of therapeutic strategies and enhance the management of resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/294582a14e94664c9f90d67cf0b0db4402660bea" target='_blank'>
              A system-level metastable model of cancer evolution: integrating replication stress, cell cycle deregulation and chromosomal instability
              </a>
            </td>
          <td>
            Dongjing Ma, Jinbin Wang, Yicong Chen, Ling Yao, Qianya Wei, Jianjun Wu
          </td>
          <td>2026-01-28</td>
          <td>Annals of Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="CRISPR-Cas9 has become a popular genome editing tool for biomedical research and drug development due to its capability to enable precise correction or integration of genetic mutations in the genome. However, precise genome editing competency varies dramatically between cell types depending on their capabilities for DNA damage. In this proof-of-concept study, we took the example of HepG2 and MCF7 to show that omics profiling identifies bottlenecks that are associated with poor precise knock-in (KI) efficiency in hard-to-engineer cells. These bottlenecks include previously described factors such as the predominance of non-homologous end joining (NHEJ) repair and impaired homologous recombination (HR) capability, but also reveals apoptotic priming status of the cells as a limiting factor. Upon further comparative analysis between HepG2 and MCF7 cells, we pinpointed and validated the proliferating cell nuclear antigen (PCNA) as a target to overexpress to enhance precise KI efficiency in MCF7. Overall, we describe how employing a multi-omics approach to characterize cell models of interest can facilitate an in-depth understanding of their editability molecular signature, empowering us to manipulate the activity of key pathways for precise editing, and therefore increase efficiency of desired editing outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a93f75f6b65fd744e6a6a08cd46d5295f2abfb1" target='_blank'>
              Omics-aided design genome editing strategy for challenging human immortalized cell models
              </a>
            </td>
          <td>
            Patricia Mendoza-Garcia, Benjamin Keith, Markus Nordberg, Ella Quist, Cristina Ferrás, Ghaith M. Hamza, Ramy Elgendy, Stephanie Ashenden, Jordi Chi, Natalie R. van Zuydam, Neil Hattersley, Xiang Zhang
          </td>
          <td>2026-02-12</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Genetic factors driving the development of pediatric Acute Lymphoblastic Leukemia (ALL) remain incompletely understood. While epigenetic dysregulation through histone methyltransferases and demethylases has emerged as a critical oncogenic mechanism, their specific contributions to pediatric ALL require systematic investigation. In this study, we systematically investigated genetic variants in histone methyltransferases and demethylase found in top 50 mutated genes from TARGET ALL Phase 2 cohort. Single nucleotide variations were functionally annotated using PredictSNP and I-Mutant2.0, followed by survival analysis and pathway enrichment studies comparing mutated versus wild-type patients. KMT2D, NSD2, SETD2, EZH2 (methyltransferases), and KDM6A (demethylase), ranked among the most frequently mutated genes. Multiple frameshift and nonsense mutations were identified in these genes, likely resulting in truncated proteins and loss of function. Critically, missense variants in EZH2 (F145C, R679C, P132T, D652G, A576D, S280C, Y728H, C566S), KDM6A (G1242D), NSD2 (E1099K, D1125H), and SETD2 (G2170D, R1592Q) were computationally predicted as deleterious and destabilizing. Although survival analysis revealed no statistically significant differences (p = 0.37), mutation carriers exhibited a 44% increased mortality risk, suggesting clinical relevance despite limited sample size (n = 35). Gene set enrichment analysis of differentially expressed genes revealed significant activation of hematopoietic cell lineage pathways in mutated patients. Our findings underscore the importance of histone methyltransferase and demethylase gene variants in pediatric ALL, particularly novel mutations affecting protein function and stability that remain poorly characterized in the literature. This study identifies candidate mutations that warrant further functional and clinical investigation for their role in disease progression and therapeutic targeting.
HIGHLIGHTS

KMT2D, NSD2, SETD2, EZH2, and KDM6A are identified among the top 50 most frequently mutated genes in pediatric ALL
KMT2D exhibits the highest mutational burden in pediatric ALL, predominantly harboring frameshift and nonsense alterations
KMT2D, NSD2, SETD2, EZH2, and KDM6A genes show consensus deleterious predictions across multiple in silico tools, with 18 mutations representing high-priority candidates for experimental validation
Structural stability analysis reveals critical destabilizing mutations in EZH2 (F145C, R679C, P132T, D652G, A576D, S280C, Y728H, C566S), KDM6A (G1242D), NSD2 (E1099K, D1125H), and SETD2 (G2170D, R1592Q)

GRAPHICAL ABSTRACT">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b15b6568cbd53c364a5d95d54d4bb823b4c767ac" target='_blank'>
              Histone Methyltransferase and Demethylase Gene in Pediatric Acute Lymphoblastic Leukemia: A Molecular Insights
              </a>
            </td>
          <td>
            Nur Aziz, Y. N. Patria, A. Wahdi, Eddy Supriyadi, Dewajani Purnomosari
          </td>
          <td>2026-01-01</td>
          <td>Trends in Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Current understanding of B cell heterogeneity in diffuse large B cell lymphoma (DLBCL) and its functional impact on disease progression remains incomplete. This study applies single-cell RNA sequencing to identify and characterize a distinct proliferation-related B cell subpopulation in DLBCL, aiming to address this knowledge gap. We utilized dataset GSE182434 from the Gene Expression Omnibus (GEO) database for the analysis, with the aim of identifying genes that are specifically highly expressed in different cell clusters. The copy number variation analysis was performed using B cells of normal samples as the control. Thereafter, the cell–cell communication analysis was implemented to reveal the potential ligand–receptor pairs, and the functional enrichment analysis was performed to uncover the enriched pathways. Further, the potential transcription factors–target genes networks were plotted via SCENIC analysis, and a series of validation assays were implemented using DLBCL cells. The single-cell landscape of DLBCL revealed a reduced proportion of B cells, CD8+ T cells, and naïve T cells. Malignant B cells exhibited chr1q amplification and chr6 deletion, with genes in these regions linked to immune suppression and impaired antigen presentation, respectively. Further subclustering identified a proliferative B cell subcluster (subcluster 2) enriched in nucleotide metabolism and cell cycle pathways. This subcluster was characterized by elevated XBP1 activity, regulating ER stress response, and downregulation of SPIB, RELB, and IRF factors involved in lymphocyte activation and interferon signaling. Functionally, XBP1 silencing suppressed DLBCL cells proliferation and invasion. Cell communication analysis revealed crosstalk between this B cell subpopulation and CD8+ T/NK cells via MIF-(CD74+CXCR4) and LTA-TNFRSF pathways. This analysis has identified and characterized the proliferation-related B cells in DLBCL, which may provide some ideas for the treatment strategies in immune-oncology and cellular therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/818cbad6bbfec8aa7d649791531b260b5a549418" target='_blank'>
              XBP1-driven proliferative B cell subcluster in Diffuse Large B Cell Lymphoma linked to altered nucleotide metabolism
              </a>
            </td>
          <td>
            Li Ma, Jing Wang, Jin Zhao, Jingrong Wang, Xiaolian Wen, M. Zheng, Liping Su
          </td>
          <td>2026-02-03</td>
          <td>European Journal of Medical Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) is highly lethal, often developing rapid chemo-resistance and progression after initial response to platinum-based chemotherapy. The role of DNA methylation in driving this progression remains poorly understood. Here, genome-wide methylation profiling via methylated DNA immuno-precipitation sequencing (MeDIP-seq) reveals hypermethylated and hypomethylated regions in both SCLC tissue DNA and plasma cell free DNA (cfDNA). A hypermethylated region in the promoter of RASA4 gene, common to SCLC tumor DNA and progression-associated cfDNA, is identified. Pharmacologic and, more specifically, targeted demethylation reactivate RASA4 expression in SCLC. The down-regulation of RASA4, a negative regulator of Ras signal, leads to the activation of Ras-MAPK pathway in SCLC. RASA4 overexpression inhibits SCLC growth, invasion and chemo-resistance, whereas its knockdown promotes these malignant phenotypes by enhancing epithelial-mesenchymal transition (EMT) and stemness. We further demonstrate that RASA4 negatively regulates the EMT-related protein SERPINE2, and directly interacts with it. Immunohistochemistry analysis of clinical specimens validates that RASA4 is significantly down-regulated in SCLC tumors, and its low expression correlates with SCLC patients’ poor survival. Taken together, the epigenetic silencing of RASA4 drives SCLC progression through the induction of EMT and stemness. Our findings underscore the potential of RASA4 as a diagnostic biomarker and therapeutic target. Promoter hypermethylation-mediated silencing of the tumor suppressor RASA4 activates Ras signaling in small cell lung cancer, thereby driving tumor proliferation, metastasis, and chemoresistance, and revealing its role as a prognostic biomarker.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3de429a7817141d17bab7346b3911bc733e9f80d" target='_blank'>
              Integrated methylome analysis identifies an epigenetically silenced tumor suppressor RASA4 in small cell lung cancer
              </a>
            </td>
          <td>
            Meng Fu, Qizhi Zhu, Jian Qi, Xiaopeng Zhang, Zhipeng Wang, Jialiang Wang, Xiang Huang, Zongtao Hu, Wei Wang, Bo Hong, Hongzhi Wang
          </td>
          <td>2025-12-31</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0dd4ab62e8dce20ab891fcaeb464ced704c0754" target='_blank'>
              PKMYT1 inhibition induces DNA damage and synergizes with immune checkpoint blockade in CCNE1-amplified gastroesophageal adenocarcinoma
              </a>
            </td>
          <td>
            S. Jang, Lawrence W Wu, Jimyung Park, Sho Hangai, Cameron Shapiro, C. G. Marshall, Luke L. Cai, Benjamin Izar, Adam J. Bass, Ryan H. Moy
          </td>
          <td>2025-12-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bf612b75850cf10e9bb17c10ec3f99a040c4e2d" target='_blank'>
              Lineage plasticity: a new dilemma in lung cancer treatment
              </a>
            </td>
          <td>
            Cainan Huo, Yongxuan Wang, Rujia Li, Xiaofei Xie, Weiqi Lyu, Dingke Chen, Yanqin Sun
          </td>
          <td>2026-02-04</td>
          <td>Molecular Biology Reports</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) exhibits pronounced intratumoral heterogeneity largely driven by cancer stem cells (CSCs). However, the molecular regulators that connect CSC stemness to tumor progression remain incompletely understood. We performed an integrative analysis combining bulk RNA sequencing (RNA-seq), single-cell RNA sequencing (scRNA-seq), and spatial transcriptomics to identify regulators potentially associated with CSC-related stemness in HCC. High-dimensional co-expression network analysis and regulatory network inference were applied to delineate stemness-associated modules, while cell–cell interaction analyses explored the tumor microenvironment. Functional validation was conducted through in vitro assays, and candidate compounds were identified using multi-platform drug screening followed by experimental testing. Dual-specificity phosphatase 9 (DUSP9) was identified as a potential regulator enriched in a malignant subpopulation characterized by elevated ERK activation, oxidative metabolism, and progenitor-like features. Co-expression and regulatory network analyses revealed a DUSP9⁺-specific module enriched for stemness- and invasion-related genes, with Forkhead Box Protein O3 (FOXO3) identified as a core downstream effector. Functional assays confirmed that DUSP9 enhances stemness through the ERK–FOXO3 axis. Spatial transcriptomics demonstrated that DUSP9⁺ cells localized within immunosuppressive niches enriched with cancer-associated fibroblasts (CAFs). Moreover, a candidate compound, AKOS000434153, inhibited HCC cell proliferation, migration, and sphere formation in vitro, accompanied by increased p-ERK and decreased p-FOXO3 levels. This study suggests that DUSP9 may serve as a potential regulator linking CSC-associated stemness with tumor progression in HCC. By integrating multi-omics datasets, our findings provide a comprehensive view of DUSP9-associated regulatory networks and highlight its promise as a biomarker for stemness-driven hepatocarcinogenesis. Further in vivo validation will be necessary to confirm these regulatory relationships.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c6c4f739a5c6cd5201bcfb964f55340c8b6780a" target='_blank'>
              Single-cell multi-omics reveals DUSP9 as a key regulator of cancer stemness and a potential therapeutic target in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Zhaoyuan Xu, Xurun Zhai, Mingxin Liu, Yi Zhang, Jiahui Cao, Junkai He, Haoyi Li, R. Liang, Weijie Wang, Rongtao Zhu, Yuling Sun
          </td>
          <td>2026-01-08</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is the second most frequently diagnosed cancer worldwide and represents a major challenge for public health. Despite advances in molecular profiling, important gaps remain in our understanding of tumorigenesis and the regulatory mechanisms underlying CRC progression. The most widely adopted classification system is the Consensus Molecular Subtypes (CMS), which stratifies CRC into four biologically distinct subtypes. We investigated the role of A-to-I RNA editing across CMS subtypes in a cohort of 100 CRC patients at various disease stages. Bulk RNA-seq data were analyzed using REDItools to detect editing events, focusing on both recoding sites and edits within repetitive elements, such as Alu sequences. Furthermore, expression levels of the ADAR enzyme family were assessed, and deconvolution analyses were performed on single-cell RNA-seq data from an independent cohort of stage II CRC patients to characterize editing activity within the tumor microenvironment (TME). Competitive endogenous RNA (ceRNA) networks, specific to each CMS subtype, were constructed based on editing events in repetitive elements. A multivariate Cox proportional hazards model was applied to evaluate associations with overall survival (OS). We observed statistically significant differences in ADARB1 expression across CMS subtypes. Single-cell RNA-seq data revealed subtype-specific distribution patterns of ADAR enzymes within the TME. Analysis of editing events showed subtype-specific signatures in both known cancer-related genes (e.g., COPA, CADPS, IGFBP7) and novel candidates (ZNF552, RALGPS1). Editing in repetitive elements informed the construction of distinct ceRNA networks for each CMS subtype, suggesting different post-transcriptional regulatory mechanisms. Survival analysis identified three variables significantly associated with OS, independent of CMS classification and clinical stage: ADARB1 expression, and editing events in NOP14-AS1 (chr4:2960236; p = 0.036; HR = 0.0069), previously linked to 5-FU sensitivity, and ST7-AS2 (chr4:117120557). This study underscores the biological relevance of RNA editing in CRC, highlighting its impact on chemoresistance, the tumor microenvironment, and subtype-specific gene regulation. Our findings suggest that RNA editing represents a critical post-transcriptional regulatory layer in CRC and holds potential as a biomarker and therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/289a3e685e9081e2037f2e82b68ea443c3971d6f" target='_blank'>
              Biological and prognostic relevance of A-to-I RNA editing across consensus molecular subtypes of colon cancer
              </a>
            </td>
          <td>
            Dario Monaco, D. Traversa, E. Mattioli, F. A. Zito, Grazia Cristiani, Francesca Buono, Sabina Delcuratolo, Tiziana Guarino, R. Pinto, Antonia Lasorella, R. Lacalamita, Ernesto Picardi, Stefania Tommasi, S. De Summa
          </td>
          <td>2026-01-06</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b06870643c50efed909af7d3fcdcbd286303425f" target='_blank'>
              Pharmacological induction of mitochondrial stress counteracts therapy resistance in glioblastoma stem-like cells
              </a>
            </td>
          <td>
            Kenji Miki, Maged T. Ghoche, Fanen Yuan, Fengping Li, Namya Manoj, Skyler S. Oken, Koji Yoshimoto, Pascal O. Zinn, S. Agnihotri, Samuel K. McBrayer, Kalil G. Abdullah
          </td>
          <td>2026-02-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Post-transcriptional RNA modifications have emerged as critical regulators of stemness, cellular plasticity, adaptation to stress and transformation. Amongst these the N6-methyladenosine (m6A) modification orchestrates a wide range of physiological processes; yet its contribution to metabolic regulation remains poorly addressed. In the present study, differential proteomics revealed enrichment of core components of the m6A machinery, viz. WTAP (writer) and IGF2BP3 (reader) in stem-like cells of the mesenchymal subtype of High-Grade Serous Ovarian Carcinoma (HGSC). Intriguingly, components of de novo cholesterol biosynthesis were also enriched. In investigating the suggested link between m6A regulation and sterol metabolism, we established a m6A-dependent stabilization of transcripts encoding rate-limiting de novo cholesterol biosynthetic enzymes by IGF2BP3 towards sustaining cholesterol production. Disruption of this cross-regulation by pharmacological inhibition impaired cell survival, self-renewal, and migration. Analyses of datasets from The Cancer Genome Atlas (TCGA) assigned a clinical significance to this regulatory axis through correlation of elevated expression of de novo cholesterol biosynthetic genes with poor progression-free survival of serous ovarian carcinoma patients. IGF2BP3 knock down, and chemical blockade of de novo cholesterol biosynthesis, both alone or in combination, achieved attenuated disease progression, in vivo. Effectively, our study links RNA modifications with metabolic reprogramming in the HGSC mesenchymal subtype through delineation of a m6A–IGF2BP3–cholesterol biosynthesis axis-mediated regulation of tumor cell stemness and aggression. This study identifies a novel m6A-dependent stabilization of key de novo cholesterol biosynthesis transcripts in the mesenchymal subtype of HGSC, which could be its therapeutic vulnerability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bab0fa270f495cb3a95155d8e68092a701a0b8fd" target='_blank'>
              Harnessing m6A-regulated cholesterol biosynthesis helps impede tumor burden and stemness in high-garde serous ovarian carcinoma mesenchymal phenotype
              </a>
            </td>
          <td>
            Aravindan Narayanan, Kiran Dhenge, S. Bapat
          </td>
          <td>2025-12-29</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Background: The RecQ DNA helicase family member WRN is an important protein for maintaining genome stability. The concept of attention as a synthetic lethal target for MSI-H tumors has garnered significant attention in recent years. However, the role of WRN in cancer development, diagnosis, and prognosis has not yet been systematically evaluated at the pan-cancer level. Methods: On the basis of multiple public cancer databases, we employed bioinformatics techniques to systematically assess WRN expression, variation, and interaction pathways across various cancers, along with the impact of WRN expression on immune profiling, drug sensitivity, and treatment, as a diagnostic tool. Additionally, we used three cancer cell lines to evaluate the suppressor function of WRN inactivation. Results: WRN is highly expressed in rapidly proliferating tissues and is dysregulated in a cancer-specific manner, particularly in tumors with hereditary DNA repair deficiencies and myeloid malignancies. WRN expression and variants are correlated with prognosis and immune activation potential in cancers. In digestive cancer and endometrial cancer with a high proportion of MSI-H tumors, WRN is positively associated with MSI/TMB signatures. Pharmacogenomic analyses revealed significant correlations between WRN expression levels and sensitivity to the DNA synthesis inhibitors PI3K, ALK, and IFG1R and other target agents and immunomodulators. In vitro validation using WRN inhibitors demonstrated potent suppression of malignant phenotypes (proliferation, clonogenicity, migration, invasion) in colorectal, endometrial, and ovarian cancer models. Conclusion: Our study suggests that WRN plays a role in cancer diagnosis and therapy, especially in cancers characterized by replicative stress or defective DNA damage repair, and that WRN can serve as a potential target for cancer immunotherapy or targeted therapies and as a prognostic marker for certain tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/788f258156dd8cd3c77e8d00e322c95984d7e751" target='_blank'>
              Pan-Cancer Analysis of WRN: From Multi-Omics Biomarker Discovery to Therapy-Guiding Functional Evidence
              </a>
            </td>
          <td>
            Qingqing Yao, Yue Qi, Peiqin Shi, Ziye Yang, Pengpeng Qu, Wen-Wen Zhang
          </td>
          <td>2026-01-01</td>
          <td>OncoTargets and Therapy</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Multiple myeloma (MM) remains incurable despite advances in treatment options. Although tumor subtypes and specific DNA abnormalities are linked to worse prognosis, the impact of immune dysfunction on disease emergence and/or treatment sensitivity remains unclear. We developed an Immune Atlas of MM by generating profiles of 1,397,272 single cells from the bone marrow (BM) of 337 newly diagnosed participants and characterized immune and hematopoietic cell populations. Cytogenetic risk-based analysis revealed heterogeneous associations with T cells of BM, with 17p13 deletion showing distinct enrichment of a type 1 interferon signature. The disease progression-based analysis revealed the presence of a proinflammatory immune senescence-associated secretory phenotype in rapidly progressing participants. Furthermore, signaling analyses suggested active intercellular communication involving a proliferation-inducing ligand and B cell maturation antigen, potentially promoting tumor growth and survival. Lastly, using independent discovery and validation cohorts, we demonstrated that integrating immune cell signatures with known tumor cytogenetics and individual characteristics significantly improves stratification for the prediction of survival. Pilcher et al. present a single-cell transcriptomics-based immune atlas of participants with newly diagnosed multiple myeloma, reporting on the association of cell types, gene expression and intercellular interactions with disease progression phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f88dd8011aa8686433c4d02b3a0e4dfd59a9255" target='_blank'>
              A single-cell atlas characterizes dysregulation of the bone marrow immune microenvironment associated with outcomes in multiple myeloma
              </a>
            </td>
          <td>
            William C. Pilcher, Lijun Yao, Edgar Gonzalez-Kozlova, Yered H. Pita-Juárez, Dimitra Karagkouni, Chaitanya R. Acharya, Marina E. Michaud, Mark Hamilton, Shivani Nanda, Yizhe Song, Kazuhito Sato, Julia Wang, Sarthak Satpathy, Yuling Ma, Jessica Schulman, D. D’souza, R. Jayasinghe, Denis J Ohlstrom, Katherine E. Ferguson, Giulia Cheloni, Mojtaba Bakhtiari, Nick Pabustan, K. Nie, J. Foltz, Isabella Saldarriaga, Rania Alaaeldin, Eva Lepisto, Rachel Chen, M. Fiala, Beena E. Thomas, April Cook, Junia Vieira Dos Santos, Chiang I-ling, Igor Figueiredo, J. Fortier, Michael Slade, Stephen T. Oh, Michael P. Rettig, Emilie I. Anderson, Ying Li, Surendra Dasari, Michael A. Strausbauch, Vernadette A. Simon, , I-ling Chiang, N. Kalavros, Jennifer Rogers, T. Dawson, Brian H. Lee, G. Kelly, Laura A Walker, Nicolas F. Fernandez, John Leech, Jarod Morgenroth-Rebin, K. Angeliadis, Matthew A. Wyczalkowski, Song Cao, Omar Ibrahim, Roderick Lin, T. Fehniger, Andrew J. Houston, E. Radkevich, Adeeb Rahman, Zhihong Chen, A. Laganà, John F. DiPersio, J. Rosenblatt, S. Kim-Schulze, S. Lonial, Shaji K. Kumar, Swati S. Bhasin, T. Kourelis, M. Dhodapkar, Ravi Vij, D. Avigan, H. Cho, George Mulligan, Li Ding, Sacha Gnjatic, Ioannis S. Vlachos, Manoj K. Bhasin
          </td>
          <td>2026-01-01</td>
          <td>Nature Cancer</td>
          <td>0</td>
          <td>112</td>
        </tr>

        <tr id="Highlights What are the main findings? The anti-tumor efficacy of nucleoside analogs is dependent on transport, enzymatic activation, nucleotide pools and metabolic responses, which are often aberrant in cancer cells, rendering these chemotherapeutic drugs ineffective. What are the implications of the main findings? Small molecule inhibitors are being tested/developed to target or downregulate specific mechanisms responsible for resistance to nucleoside analogs. The combination of small molecule inhibitors with nucleoside analogs has enhanced the efficacy of nucleoside analogs and improved patient outcomes. Abstract Nucleoside analogs remain central to the treatment of hematologic malignancies and solid tumors, yet resistance frequently occurs, contributing to relapse and disease-related mortality. Rather than arising from a single mechanism, effective nucleoside analog activity requires successful navigation of multiple biological barriers, including cellular uptake, intracellular activation, nucleotide pool balance, genome surveillance, and mitochondrial stress responses. This review integrates recent advances describing how alterations at each of these levels contribute to resistance to nucleoside analog therapies. We further highlight emerging therapeutic strategies centered on small-molecule inhibitors that exploit these vulnerabilities to enhance the efficacy of nucleoside analogs. Together, this integrative perspective supports the need for development of small molecule inhibitors and design of combination approaches aimed at restoring apoptotic competence and improving the use of nucleoside analog-based therapies for the treatment of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f48f64e40e1b94edba907600f2e063e159b5fa3" target='_blank'>
              Enhancing the Nucleoside Analog Response with Translational Therapeutic Approaches to Overcome Resistance
              </a>
            </td>
          <td>
            Jenna Thibodeau, Kian K Hershberger, Sai Samanvitha M. Ramakrishna, Yongwei Su, Lauren Timmer, Bryce Brophy, Katherine K Zhang, Holly Edwards, Jeffrey W. Taub, Yubin Ge
          </td>
          <td>2026-01-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Most colorectal cancers are resistant to immune checkpoint inhibitors due to an immunosuppressive tumor microenvironment. LINC00673, previously classified as a long non-coding RNA, has been implicated in tumor progression, but its role in antitumor immunity remains poorly understood. We used single-cell RNA sequencing, flow cytometry, and functional assays in Linc00673 knockout and wild-type mouse models treated with programmed death 1 (PD-1) blockade to dissect changes in immune landscape and tumor cell behavior. Linc00673 deletion enhanced the efficacy of PD-1 therapy, reducing tumor burden and prolonging survival. KO tumors showed increased infiltration of cytotoxic CD8⁺ T cells, reduced exhaustion, and improved antigen presentation. The tumor microenvironment shifted toward a pro-inflammatory state, with elevated dendritic cell function, reduced immunosuppressive macrophages, and improved T cell–tumor interactions. Linc00673 also promoted tumor cell migration and immune evasion independently of immune cells, suggesting dual tumor-intrinsic and -extrinsic roles. LINC00673 suppresses antitumor immunity and promotes tumor aggressiveness. Its deletion synergizes with PD-1 blockade, revealing a promising therapeutic target to overcome immune resistance in colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb9ef0a9a3f2685b8b34be389ae9c671f82a1666" target='_blank'>
              Single-cell RNA sequencing unveils enhanced antitumor immunity in colorectal cancer with PD-1 blockade and LINC00673 deletion
              </a>
            </td>
          <td>
            Yifei Zhu, Yanxi Yao, Yaxian Wang, Zhibing Qiu, Dingpei Zhou, Huixia Huang, Keji Chen, Qingyang Sun, Jiayu Chen, Yuxue Li, Jianqiang Tang, Dawei Li, Ping Wei
          </td>
          <td>2026-01-03</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Clonal hematopoiesis of indeterminate potential (CHIP) is a precursor condition characterized by the expansion of blood cell clones harboring somatic mutations originating in hematopoietic stem cells (HSCs). Since individuals with CHIP face a high risk of developing myeloid malignancies, targeting CHIP clones could provide a viable strategy for leukemia prevention. Despite its clinical significance, the mechanisms underlying CHIP predisposition and progression remain poorly understood. Recent genome wide association studies (GWAS) have identified several non-coding genetic loci that are strongly associated with CHIP; however, their underlying mechanisms still remain unknown. We hypothesize that risk variants in these non-coding loci modulate enhancer elements active in HSCs. To test this, we selected 1,374 non-coding variants from 51 loci associated for CHIP risk in the UK Biobank and screened them for regulatory activity using a Massively Parallel Reporter Assay (MPRA). We performed our lentiviral MPRA screen in MUTZ-3 cells, a human hematopoietic cell line relevant to HSCs, which express CD34 surface marker and are dependent on HSC-specific transcription factors. Using a MPRA library of ∼73,000 constructs in CD34+ fraction of MUTZ-3 cells, we identified 87 variants representing 32 GWAS loci. We used targeted genome editing to demonstrate endogenous enhancer activity across 3 MPRA variants that affect the transcription of NKD2, FLT3, and MSI2. Our functional studies on MSI2 indicate that presence of higher levels of MSI2 mediated by CHIP risk allele enhances the clonal expansion of TET2 knockout hematopoietic stem and progenitor cells, providing a mechanistic link whereby non-coding genetic variants can influence the expansion of mutant CHIP clones.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/103a0a6127003db01ce70e503d0d9d03c9b83f1d" target='_blank'>
              Systematic functional dissection of germline noncoding risk variants impacting clonal hematopoiesis
              </a>
            </td>
          <td>
            Trieu Nguyen, Jessica Jeejan, Takeshi Iwasaki, S. Kales, Joyeeta Chakraborty, Chie Yanase, Aruna Shekhar, Dominika Kwasniak, Aditi Hegde, Richard Voit, Kristy Stengel, Keisuke Ito, Ryan Tewhey, Satish K. Nandakumar
          </td>
          <td>2026-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bce9b91a05b2c305ecf72c3bf59327984c56dba6" target='_blank'>
              Functional inactivation of the telomerase chaperone TCAB1 primes cells for the activation of ALT in osteosarcoma
              </a>
            </td>
          <td>
            Joshua Keegan, Sydney Sorbello, Joakin O. Mori, Shugo Muratani, I. Leshchiner, Christopher M. Heaphy, R. L. Flynn
          </td>
          <td>2026-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79941e547de1ddb3fb2c22aa561602d507f6cf5a" target='_blank'>
              CRISPR-enhanced assessment of variants of unknown significance nominates oncology therapeutic targets and drug repositioning opportunities
              </a>
            </td>
          <td>
            A. Savino, Athanasios Oikonomou, Riccardo R. De Lucia, Miquel L Grau, Kathrina McCarten, H. Najgebauer, Umberto Perron, Luca Azzolin, Alexandra Livanova, Paolo Cremaschi, Núria López-Bigas, A. Sottoriva, F. Iorio
          </td>
          <td>2026-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Recent evidence highlights the significance of a new type of tumour suppressors, which are not frequently mutated but inhibited by metabolic cues in cancers. Here, we identify BATF2 as a tumour suppressor whose expression is epigenetically silenced by glutamine in Head and Neck Squamous Cell Carcinomas (HNSCC). BATF2 correlates with type-I interferon and Th1 signatures in human HNSCC, with correlation coefficients even stronger than those of the positive control, STING. The phosphorylation of BATF2 at serine 227 promotes the oligomerization of STING. BATF2 deficiency or high glutamine levels result in higher oxygen consumption rates and metabolic profiles unfavorable for type-I interferon production. An isocaloric glutamine-rich diet abolishes STING-mediated effector cell expansion in tumours, weakening STING agonist-induced tumour control. Cancer cell-specific BATF2 expression promotes an Id2-centered T-cell effector signature, reduces T-cell exhaustion, and triggers spontaneous HNSCC rejection in a type-I interferon-dependent fashion. Utilizing syngeneic subcutaneous, orthotopic, and 24-week-long cigarette smoke carcinogen-induced HNSCC models, we demonstrate that host Batf2 deficiency results in increased infiltration of CD206+ myeloid cells and reduced effector CD8+ T-cells, accelerating the initiation of cancers. Overall, we reveal a tumour suppressor BATF2 whose loss is mediated by unique metabolic cues in the TME and drives cancer immune escape. STING–type-I interferon pathway regulates the immunogenicity of several cancer types, including head and neck squamous cell carcinoma. Here the authors describe that glutamine metabolism in the tumour microenvironment dampens the STING–type-I interferon pathway by epigenetically silencing the expression of BATF2, which functions as a tumour suppressor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bbcd65a0903fb5047be655a3d9e696c27c3d717" target='_blank'>
              BATF2 is a glutamine-responsive tumour suppressor required for type-I interferon-dependent anti-tumour immunity
              </a>
            </td>
          <td>
            Wang Gong, Hülya F. Taner, Yuesong Wu, Yumin He, Xingwu Zhou, Zaiye Li, Xin Hu, Charisse Ursin, Kala Chand Debnath, Kohei Okuyama, Qiang Hu, Christopher R Donnelly, Felipe Nör, Chamila D Perera, E. Bellile, Arash Yunesi, Zhiqian Zhai, Mei Zhao, Wanqing Cheng, Zackary R Fitzsimonds, Luke Broses, Jiaqian Li, Shadhmer Demehri, Deepak Nagrath, Gregory T Wolf, Andrew G Sikora, Yanbao Yu, Haitao Wen, Lei Wei, Steven B Chinn, Jeffrey N. Myers, Shizuo Akira, Yuying Xie, James J. Moon, Y. L. Lei
          </td>
          <td>2025-12-29</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Fusion genes are pivotal drivers of tumorigenesis, often generating oncogenic chimeric RNAs and fusion circular RNAs. However, the mechanisms by which these transcripts synergistically contribute to cancer progression remain poorly understood. Here, we identified a lung cancer-specific chimeric RNA KANSL1-ARL17A (chKANSARL) and its circular variant fusion circular RNA KANSL1-ARL17 A (F-circKA), both derived from the fusion gene KANSARL. Functional assays revealed that overexpression of either chKANSARL or F-circKA significantly enhanced lung cancer cell proliferation, migration, and invasion, while their knockdown suppressed these malignant phenotypes. In vivo experiments demonstrated that chKANSARL overexpression accelerated tumor growth in immunodeficient mice. Notably, coexpression experiments uncovered a synergistic regulatory interaction between F-circKA and chKANSARL, amplifying oncogenic effects. Mechanistically, miRNA sequencing and dual-luciferase assays revealed that F-circKA acts as a molecular sponge for miR-6860, thereby derepressing chKANSARL expression. Rescue experiments further validated this regulatory axis, wherein miR-6860 inhibition reversed the tumor-suppressive effects of F-circKA knockdown. Collectively, our study identifies and characterizes a novel F-circKA/miR-6860/chKANSARL regulatory axis, revealing how dual transcriptional outputs from the KANSARL fusion gene can synergistically drive lung cancer progression. These findings highlight a previously unrecognized layer of cooperative regulation between linear and circular fusion RNAs in oncogenesis and provide a new framework for understanding fusion gene-mediated tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/261c7a04a77f3f0847188162738372118fe5132c" target='_blank'>
              The KANSL1-ARL17A fusion gene generates oncogenic chKANSARL and F-circKA RNAs that synergistically drive lung cancer progression via a novel F-circKA/miR-6860/chKANSARL axis
              </a>
            </td>
          <td>
            Xinchao Guan, Tao Liu, Sili Chen, Junwei Zhang, Huanliang Huang, Dexiong Chen, Qiaoyuan Yang
          </td>
          <td>2026-01-01</td>
          <td>The Journal of Biological Chemistry</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Cisplatin remains a cornerstone of chemotherapy for numerous cancers, despite the persistent challenges of toxicity and the development of drug resistance. Therefore, a deeper understanding of the mechanisms behind cisplatin resistance and the development of strategies to counter it are of critical importance. This review systematically examines the pivotal role of mitochondrial dynamics in cisplatin resistance and discusses emerging therapeutic strategies that target these processes. Mitochondrial dynamics regulate the structure and function of the mitochondrial network through a balance of fusion and fission. Dysregulation of this process directly contributes to cisplatin resistance by maintaining cellular energy homeostasis, inhibiting apoptosis, and enhancing oxidative phosphorylation (OXPHOS). Furthermore, mitophagy, metabolic reprogramming, and the tumor immune microenvironment converge on mitochondrial dynamics to drive the acquisition of drug resistance. Consequently, targeting mitochondrial dynamics presents a promising approach to overcome cisplatin resistance. Potential strategies include restoring the balance of fusion and fission, intervening in mitophagy, disrupting OXPHOS metabolism, and developing mitochondrial-targeted nanodrug delivery systems. However, despite this promising outlook, significant challenges remain, including the heterogeneity of resistance mechanisms, a lack of reliable biomarkers, and the need for selective targeting to minimize off-target effects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db056eb8c3db5d2ebbae460e0446276957e89344" target='_blank'>
              Mitochondrial dynamics in cisplatin resistance: molecular mechanisms and therapeutic targeting
              </a>
            </td>
          <td>
            Wei Huang, Wei Sun, Zhiyuan Yang, Yiwen Li, Ziliang Wang
          </td>
          <td>2026-01-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Deregulation of proteins involved in chromatin regulation is common in pancreatic ductal adenocarcinoma (PDAC). Lysine demethylase 4C (KDM4C) is one of the chromatin-modifying proteins frequently overexpressed across multiple solid cancers and is linked to chromatin instability, increased cell proliferation, and enhanced stem cell–like behavior. We observed upregulation of KDM4C protein in a panel of human PDAC cell lines and patient samples compared with nonneoplastic controls. CRISPR/Cas9-mediated deletion of KDM4C in human and murine PDAC cells reduced proliferation, clonogenicity, and increased survival of orthotopically implanted murine PDAC allografts. Transcriptomic and proteomic analyses revealed that loss of KDM4C in both human and murine PDAC cell lines was associated with the reduction of activated phospho-ERK, a pivotal effector downstream of mutant RAS. Using proximity labeling, we identified the histone deacetylase SIRT1 as a novel interacting protein with KDM4C via the latter’s Tudor reader domain. SIRT1-mediated deacetylation leads to repression of downstream targets, including the dual specificity phosphatase DUSP2, which is known to inactivate ERK via dephosphorylation. In vitro propagation of KDM4C-null PDAC lines eventually led to adaptation and restitution of ERK signaling, with rescue of the KDM4C loss induced growth suppression. To bypass this adaptive phenomenon, we tested a preclinical pan-KDM4 inhibitor TACH107 and confirmed its efficacy in in vitro and in vivo PDAC models. Our studies identify KDM4C as an oncogenic molecule that sustains ERK signaling in KRAS-mutant PDAC and can be broadly targeted via small-molecule inhibitors. Significance: Our data suggests that KDM4C is a novel regulator of ERK signaling, the main effector pathway downstream of mutant RAS. This is the first demonstration linking the requirement of sustained KDM4 activity to ERK signaling in cancer and presents an opportunity to leverage this oncogenic pathway for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac9ce02e321041b11b3d0c59cd917566ed5c1973" target='_blank'>
              The Lysine Demethylase KDM4C Is an Oncogenic Driver and Regulates ERK Activity in KRAS-Mutant Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Menna-T-Allah Shaheen, Sarah Dhebat, K. Rajapakshe, Bidyut Ghosh, Benson Chellakkan Selvanesan, Shariq S Ansari, C. Haymaker, Dorsay Sadeghian, Huamin Wang, Ching-Fei Li, H. Ying, Anirban Maitra
          </td>
          <td>2026-01-01</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="


 Prostate cancer (PCa) is the most commonly diagnosed cancer and the second leading cause of cancer-related death in American men. Men with germline BRCA2 mutations are at especially high risk (19%-61% by age 80) of developing PCa. Loss of BRCA2 impairs homologous recombination repair (HRR), resulting in genomic instability and enhanced tumor progression. Our laboratory previously performed a whole-genome CRISPR-Cas9 screen in a BRCA2 mutant cell line under cisplatin induced stress. This screen identified Chromodomain Helicase DNA-binding protein 1 (CHD1) as the top gene whose loss promoted cell survival under DNA damage stress. CHD1 is a chromatin remodeler that is deleted in approximately 15% PCas and is strongly linked to metastatic disease. However, how its loss drives tumor progression, particularly in BRCA2-deficient PCas, remains poorly understood. To address this, we carried out RNA-seq analysis in castration-resistant prostate cancer (CRPC) cell line PC-3 following knockdown of CHD1 and BRCA2, individually and in combination. Notably, depletion of CHD1 in BRCA2-deficient cells resulted in the upregulation of gene signatures associated with hypoxia and epithelial-mesenchymal-transition (EMT). Functional validation using wound healing and migration assays demonstrated that CHD1 loss significantly enhanced the migration ability of PC-3 cells (including BRCA2-deficient) compared to controls. In keeping with increased EMT, we also see high vimentin expression in CHD1-depleted PC3. Furthermore, Causal Inference Enrichment (CIE) analysis of the transcriptomic data pointed to significant activation of transcription factors (TFs) FOSL1 and SOX9 after CHD1 knockdown compared to control cells. To assess clinical relevance, we analyzed RNA-seq data from the Stand Up To Cancer (SU2C) database. This analysis revealed that CHD1-low tumors exhibited upregulation of multiple oncogenic pathways, especially EMT, angiogenesis, apical junction, and IL2-STAT5 signaling, consistent with our in vitro observations. Finally, genetically engineered mouse model (GEMM)-derived prostate organoids from B2PPt (Brca2flox/flox;P53flox/flox;Ptenflox/flox) and B2CPPt (Brca2flox/flox; Chd1flox/flox;P53flox/flox;Ptenflox/flox) mice, after Adeno-Cre infection, revealed higher Fosl1, along with EMT marker vimentin in B2CPPt organoids compared to B2PPt, supporting FOSL1-dependent lineage plasticity and activation of EMT programs upon CHD1 loss in BRCA2-mutant prostate cells. Collectively, these findings indicate that CHD1 loss, either independently or in the context of BRCA2 deficiency, promotes oncogenic phenotypes through activation of FOSL1-dependent lineage plasticity and EMT programs that favor metastasis. Ongoing work dissects how CHD1 loss reshapes chromatin accessibility to reprogram transcriptional networks and drives oncogenic signaling and metastatic pathways in PCa, with particular relevance to BRCA2-deficient PCa.



 Jingzhu Hao, Shailja Pathania, Miklos Diossy, Kourosh Zarringhalam, Changmeng Cai, Zoltan Szallasi. Dissecting the role of CHD1 as a key regulator of pro-metastatic transcriptional reprogramming in BRCA2-mutant prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr PR017.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a2ea6307c1138357462e3893b183854dd498126" target='_blank'>
              Abstract PR017: Dissecting the role of CHD1 as a key regulator of pro-metastatic transcriptional reprogramming in BRCA2-mutant prostate cancer
              </a>
            </td>
          <td>
            Jingzhu Hao, S. Pathania, M. Dióssy, Kourosh Zarringhalam, Changmeng Cai, Zoltán Szállási
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Genomic alterations are a hallmark of cancer, and extrachromosomal DNA (ecDNA) has emerged as a key source of oncogene selection, tumor growth, and drug resistance. The intratumor heterogeneity and clonal selection of ecDNA is, however, poorly understood. In this study, we pursue a computational approach that leverages allelic imbalance and outlier expression from standard single-cell RNA sequencing (scRNA-seq) to deconvolve the tumor heterogeneity of ecDNA at the single-cell level (ecSingle). Using this approach, we identify oncogene-carrying ecDNAs in tumor samples at the single-cell level, which we validate using genome sequencing. Moreover, we show the superiority of using single-molecule long-read sequencing in resolving ecDNA. ecDNAs displayed extensive intratumor heterogeneity, including subclonal oncogene-carrying ecDNA in primary tumor cells that segregate with distinct transcriptional cell states. Importantly, we show that a rare ecDNA+ clone in the primary tumor can expand to form dominant clones in relapse tumors. Our study introduces a novel approach to studying ecDNA at the single-cell level, enabling both clonal evolution and transcription cell state analysis. We apply this approach to cancer samples to gain deeper insights into the role of ecDNA in intratumor heterogeneity and cellular plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7df14db8ad5c3596e9b413772fed54a722b4bb2b" target='_blank'>
              Resolving clonal evolution and selection of extrachromosomal DNA at single-cell resolution
              </a>
            </td>
          <td>
            J. Hendriksen, A. Locallo, B. Schlotmann, Francisco Germán Rodríguez González, Jane Skjøth-Rasmussen, C. Yde, D. Nørøxe, H. S. Poulsen, U. Lassen, J. Weischenfeldt
          </td>
          <td>2026-01-29</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94cb28bdcaf79e7d74e6f5d2dbc03edf13bb5c93" target='_blank'>
              SMAD4 loss drives cancer chromosomal instability through transcription-replication conflict-induced replication stress
              </a>
            </td>
          <td>
            Atmika Paul, Gargee Joshi, Philipp Jungk, Simranjeet Kaur, Shengyu Yao, Ioannis Tsirkas, N. Böhly, Maren Sitte, Gabriela Salinas, Stephan Hamperl, Maik Kschischo, Holger Bastians
          </td>
          <td>2025-12-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Dysregulation of repair pathways, specifically the base excision repair (BER) axis, is frequently observed in cancer. This dysfunction is associated with accelerated tumor growth and reduced patient survival, suggesting that BER components may serve as potential targets for cancer therapy. This study focused on the expression of Uracil-DNA glycosylase (UNG), a key enzyme in BER, in colorectal tumor tissues and adjacent normal tissues. We systematically explored the clinical significance, biological functions, and molecular mechanisms of UNG in colorectal cancer (CRC). Our findings revealed that UNG was significantly upregulated in CRC, and its higher expression correlated with poorer patient outcomes. Functional assays demonstrated that silencing UNG inhibited cell proliferation, reduced migration and invasion capabilities, and induced apoptosis along with S/G2 phase cell cycle arrest. Mechanistically, UNG appears to interact with the AMPK, AKT/mTOR, and ERK signaling pathways. These results suggested that UNG exhibited oncogenic potential in CRC by regulating malignant behaviors and oncogenic signaling cascades, positioned it as a potential prognostic biomarker and therapeutic target for CRC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40071783d4dcf5a7a90c5996af36927f2f827940" target='_blank'>
              Silencing Uracil-DNA glycosylase inhibits colorectal cancer progression
              </a>
            </td>
          <td>
            Jingjing Yang, Li-zhou Shi, Yong-wei Hu, Hua Chen, Qing-Hua Wang, Wei Han
          </td>
          <td>2026-01-04</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The Werner syndrome helicase (WRN) is a promising target for cancers with microsatellite instability (MSI) leading to the initiation of at least five Phase I clinical trials. Acquired resistance is a substantial obstacle for obtaining lasting benefit from targeted therapies in oncology and may be particularly acute in the setting of mismatch repair deficient (dMMR) tumors, which can sample increased fitness landscapes owing to higher mutational burden. In this study, we characterized resistance mechanisms using the clinical candidate HRO761 and two novel inhibitors in MSI cell lines and xenograft models. We observed rapid emergence of resistance both in vitro and in vivo, with sequencing revealing clustered mutations within the WRN helicase domain. Computational structural analyses indicated these mutations either directly interfere with inhibitor binding or alter the protein conformation required for inhibitor engagement. Notably, while most mutations conferred broad resistance across all three compounds, we identified specific alterations (L528S, C727R, and F730L) that exhibited selectivity between chemical scaffolds. This chemotype-specific resistance profile suggests opportunities for developing next-generation inhibitors that retain activity against resistant variants and for implementing rational treatment strategies with existing inhibitors. Overall, our findings demonstrate that on-target resistance to WRN inhibitors emerges rapidly in dMMR backgrounds but also highlight potential approaches to overcome resistance, supporting continued development of WRN-targeted therapies for MSI cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef96a8c92cd506d4f971d917a09905bef7c0e526" target='_blank'>
              Microsatellite instable cancer cells acquire on-target resistance mutations to WRN helicase inhibitors.
              </a>
            </td>
          <td>
            Faith Fowler, Jessica Gajda, Amirhossein Mafi, Aileen Kelly, Cynthia L Jeffries, Phuong Le, Lei Shi, Mariam D George, Nate Elsen, Alla V. Korepanova, Sarah M Zerbs, Orville A Pemberton, Xiangdong Xu, Nathan J Gesmundo, Wei Qiu, Fei Han, Henry Tang, Sanjay C Panchal, Mick Dart, Jonathan Hickson, Chaohong Sun, Yunsong Tong, Ari J Firestone
          </td>
          <td>2026-02-09</td>
          <td>Molecular cancer therapeutics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e04b464c0e1b167410299c1bc664ca1e5464bb8" target='_blank'>
              cellSTAAR: incorporating single-cell-sequencing-based functional data to boost power in rare variant association testing of noncoding regions
              </a>
            </td>
          <td>
            Eric Van Buren, Yi Zhang, Xihao Li, M. Selvaraj, Zilin Li, Hufeng Zhou, Nicholette D. Palmer, Donna K. Arnett, J. Blangero, Eric Boerwinkle, B. Cade, Jenna C. Carlson, April P. Carson, Yii-DerIda Chen, J. Curran, R. Duggirala, Myriam Fornage, Nora Franceschini, M. Graff, C. Gu, Xiuqing Guo, Jiang He, Nancy Heard-Cosa, Lifang Hou, Yi-Jen Hung, Rita R. Kalyani, Sharon L. R. Kardia, Eimear E. Kenny, C. Kooperberg, B. Kral, Leslie Lange, Daniel Levy, Changwei Li, Simin Liu, D. Lloyd-Jones, R. Loos, A. Manichaikul, L. W. Martin, Rasika A Mathias, R. Minster, Braxton D. Mitchell, J. Mychaleckyj, T. Naseri, Kari E. North, Jeffry R. O'Connell, James A. Perry, P. Peyser, B. Psaty, L. Raffield, Ramachandran S. Vasan, S. Redline, Alexander P. Reiner, Stephen S. Rich, , Brian W. Spitzer, Hua Tang, Kent D. Taylor, Russell Tracy, S. Viali, Lisa R. Yanek, Wei Zhao, , Namiko Abe, Gonçalo R. Abecasis, F. Aguet, Christine Albert, L. Almasy, Alvaro Alonso, Seth Ament, Peter Anderson, Pramod Anugu, Deborah Applebaum-Bowden, K. Ardlie, D. Arking, A. Ashley-Koch, Stella Aslibekyan, T. Assimes, Paul L. Auer, D. Avramopoulos, N. Ayas, Adithya Balasubramanian, John Barnard, Kathleen Barnes, R. G. Barr, E. Barron-Casella, L. Barwick, Terri L Beaty, Gerald Beck, Diane M. Becker, Lewis C. Becker, Rebecca L. Beer, A. Beitelshees, Emelia Benjamin, Takis Benos, Marcos Bezerra, Larry Bielak, Joshua C. Bis, Thomas Blackwell, Nathan Blue, Don Bowden, Russell P Bowler, Jennifer Brody, U. Broeckel, J. Broome, Deborah L. Brown, Karen L. Bunting, Esteban G. Burchard, Carlos Bustamante, Erin J. Buth, J. Cardwell, V. Carey, J. Carrier, C. Carty, Richard Casaburi, , James M. Casella, Peter Castaldi, M. Chaffin, Christy Chang, Yi-Cheng Chang, Daniel I. Chasman, S. Chavan, Bo-Juen Chen, Wei-Min Chen, Michael Cho, Seung Hoan Choi, Lee-Ming Chuang, M. Chung, Ren-Hua Chung, Clary Clish, S. Comhair, M. Conomos, Elaine Cornell, Adolfo Correa, Carolyn Crandall, J. Crapo, L. Cupples, J. Curtis, B. Custer, Coleen Damcott, D. Darbar, S. David, Colleen P. Davis, M. Daya, Michael R. DeBaun, Dawn Demeo, R. Deka, Scott Devine, H. Dinh, H. Doddapaneni, Qing Duan, Shannon Dugan-Perez, J. P. Durda, S. Dutcher, Charles B. Eaton, L. Ekunwe, Adel Boueiz, Patrick T Ellinor, L. Emery, S. Erzurum, Charles Farber, J. Farek, T. Fingerlin, M. Flickinger, C. Frazar, Mao Fu, Stephanie M. Fullerton, L. Fulton, Stacey Gabriel, Weiniu Gan, Shanshan Gao, Yan Gao, Margery L S Gass, Heather Geiger, B. Gelb, M. Geraci, S. Germer, R. Gerszten, Auyon Ghosh, Richard Gibbs, Chris Gignoux, M. Gladwin, David E. Glahn, S. Gogarten, Da-Wei Gong, H. Goring, S. Graw, Kathryn J. Gray, Daniel Grine, Colin Gross, Yue Guan, Namrata Gupta, Jeffery Haessler, Michael Hall, Yi Han, Patrick J. Hanly, Daniel N Harris, Nicola L. Hawley, Benjamin D. Heavner, S. Heckbert, Ryan Hernandez, David M. Herrington, Craig Hersh, Bertha Hidalgo, James E Hixson, Brian Hobbs, John Hokanson, Elliott Hong, Karin F. Hoth, C. Hsiung, Jianhong Hu, Yi-Jen Hung, H. Huston, C. Hwu, M. Irvin, Rebecca D. Jackson, D. Jain, C. Jaquish, J. Johnsen, Andrew Johnson, Craig W Johnson, R. Johnston, Kimberly Jones, H. M. Kang, R. Kaplan, Shannon Kelly, Michael Kessler, Alyna T Khan, Ziad M. Khan, Wonji Kim, J. Kimoff, G. Kinney, Barbara A. Konkle, Holly Kramer, Christoph Lange, Ethan Lange, Cathy C. Laurie, C. Laurie, M. LeBoff, J. Lefaive, Jiwon Lee, Sandra Lee, Wen-Jane Lee, David Levine, Joshua Lewis, Xiaohui Li, Yun Li, Henry J. Lin, Honghuang Lin, Yongmei Liu, Yu Liu, S. Lubitz, K. Lunetta, James Luo, U. Magalang, Michael P. Mahaney, B. Make, Alisa K Manning, J. Manson, Melissa A Marton, S. Mathai, Susanne May, P. McArdle, Merry-Lynn N. McDonald, Sean K. McFarland, S. McGarvey, Daniel J. McGoldrick, Caitlin P. McHugh, B. McNeil, Hao Mei, J. Meigs, Vipin Menon, L. Mestroni, G. Metcalf, Deborah Meyers, Emmanuel Mignot, Julie Mikulla, N. Min, Mollie A Minear, M. Moll, Z. Momin, M. Montasser, Courtney G. Montgomery, Donna M Muzny, Girish N. Nadkarni, Rakhi Naik, Sergei Nekhai, Sarah C. Nelson, Bonnie Neltner, C. Nessner, Deborah Nickerson, Osuji Nkechinyere, T. O’Connor, H. Ochs-Balcom, Geoffrey O. Okwuonu, Allan Pack, David T. Paik, Nicholette D. Palmer, J. Pankow, G. Papanicolaou, Cora Parker, J. Peralta, Marco V. Perez, U. Peters, Lawrence S. Phillips, Jacob Pleiness, Toni Pollin, Wendy S. Post, Julia Powers Becker, M. Boorgula, Michael H. Preuss, P. Qasba, D. Qiao, Zhaohui Qin, N. Rafaels, M. Rajendran, , Laura Rasmussen-Torvik, A. Ratan, Robert M Reed, C. Reeves, E. Regan, M. Reupena, Kenneth M. Rice, Rébecca Robillard, N. Robine, Dan Roden, C. Roselli, I. Ruczinski, Alexi Runnels, Pamela H. Russell, S. Ruuska, Kathleen A. Ryan, E. Sabino, D. Saleheen, S. Salimi, Sejal Salvi, Steven Salzberg, Kevin Sandow, Vijay G Sankaran, J. Santibanez, K. Schwander, D. Schwartz, F. Sciurba, C. Seidman, J. Seidman, Vivien Sheehan, S. Sherman, Amol C. Shetty, A. Shetty, Wayne Hui-Heng Sheu, M. B. Shoemaker, Brian H. Silver, Edwin Silverman, Robert Skomro, A. V. Smith, Josh Smith, Nicholas L. Smith, Tanja Smith, S. Smoller, Beverly Snively, M. Snyder, Tamar Sofer, N. Sotoodehnia, A. Stilp, G. Storm, E. Streeten, Jessica Lasky Su, Y. Sung, Jody M Sylvia, Adam A. Szpiro, F. Sériès, D. Taliun, M. Taub, Matthew Taylor, Simeon Taylor, M. Telen, Timothy A. Thornton, M. Threlkeld, Lesley E. Tinker, D. Tirschwell, Sarah Tishkoff, Hemant K. Tiwari, Catherine Tong, Michael Tsai, Dhananjay Vaidya, David Van Den Berg, Peter Vandehaar, S. Vrieze, Tarik Walker, R. Wallace, A. Walts, Fei Fei Wang, Heming Wang, Jiongming Wang, Karol Watson, Jenn Watt, D. Weeks, J. Weinstock, B. Weir, Scott T. Weiss, L. Weng, Jennifer Wessel, Cristen J. Willer, Kayleen Williams, , Scott Williams, Carla Wilson, James G. Wilson, L. Winterkorn, Q. Wong, Baojun Wu, Joseph Wu, Huichun Xu, Lisa R. Yanek, Ivana V. Yang, Ketian Yu, S. Zekavat, Yingze Zhang, Snow Xueyan Zhao, Xiaofeng Zhu, Elad Ziv, Michael C. Zody, S. Zoellner, Mariza de Andrade, Paul de Vries, L. Fuentes, Jerome L. Rotter, G. Peloso, P. Natarajan, Xihong Lin
          </td>
          <td>2025-12-31</td>
          <td>Nature Methods</td>
          <td>0</td>
          <td>102</td>
        </tr>

        <tr id="Genomic instability is a hallmark of nearly all human cancers, yet the genetic regulators that govern this instability, their functional roles, and their therapeutic potential remain incompletely understood in breast cancer. In this study, we conducted a comprehensive analysis of the expression profiles of genomic instability-maintaining genes (GIMGs) in breast cancer and revealed that elevated expression of GIMGs was associated with adverse clinical outcomes and an altered tumor immune microenvironment. Through further analysis, we identified a core set of GIMGs and pinpointed RPGRIP1L as a critical driver of genomic instability in breast cancer. Notably, RPGRIP1L expression was significantly upregulated in breast cancer tissues and strongly correlated with poor patient prognosis. Functional studies demonstrated that RPGRIP1L knockdown reduced genomic instability in tumor cells, as evidenced by decreased Phosphorylated Histone H2AX (γH2AX) expression, and suppressed tumor growth in mouse models. Additionally, high RPGRIP1L expression was linked to elevated expression of immune checkpoint Programmed Death Ligand 1(PD-L1) in human breast cancer samples. Mechanistically, we found that RPGRIP1L promoted the activation of the Hedgehog signaling pathway, which in turn drived tumor proliferation and upregulated PD-L1 expression. Collectively, these findings highlight RPGRIP1L as a key genomic instability-maintaining gene in human breast cancer, offering critical insights into the molecular mechanisms underlying disease progression. Furthermore, targeting RPGRIP1L may represent a promising therapeutic strategy for breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c08de5eb400d357e336428d7f7a366a3a038cb3" target='_blank'>
              Identification of RPGRIP1L as an instability-maintaining gene to drive tumor growth and PD-L1 expression via Hedgehog signaling in breast cancer
              </a>
            </td>
          <td>
            Meng Lv, Yan’e Liu, Weihong Cao, Yongmei Wang, Qi Wang, Xue-Qiang Gao, Baowei Peng, Hai-bo Wang, Yan Mao
          </td>
          <td>2025-12-30</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Homologous recombination deficiency (HRD) impacts cancer treatment strategies, particularly effective utilization of PARP inhibitors. However, the variability of different HRD assays has hampered the selection of oncology patients who may benefit from these therapies. Our study aims to use the whole genome landscape to better define HRD in a pan-cancer cohort. We employed a whole genome sequencing HRD classifier that includes genome-wide signatures associated with HRD to analyze 580 tumor/normal paired samples. The HRD phenotype was correlated with genomic variants in BRCA1/2 and other homologous recombination repair genes. In this paper we show that the HRD phenotype is identified in various cancers including breast (21%), pancreaticobiliary (20%), gynecological (17%), prostate (9%), upper gastrointestinal (GI) (2%), and other cancers (1%). HRD cases are not confined to BRCA1/2 mutations; 24% of HRD cases are BRCA1/2 wild-type. A diverse range of gene alterations involved in HRD are elucidated, including biallelic mutations in FANCF, XRCC2, and FANCC, and deleterious structural variants. In a subset of cases, the whole genome sequencing-based classifier offers more insights and a better correlation to treatment response when compared to other assays. Although HRD is a biomarker used to determine which cancer patients would benefit from PARP inhibitors, a lack of harmonization of tests to determine HRD status makes it challenging to interpret their results. Our study highlights the use of comprehensive whole genome sequencing analysis to better predict HRD and elucidates genomic mechanisms associated with this phenotype. Homologous recombination deficiency is a condition in which a cancer cell cannot repair certain types of DNA damage. It causes genetic instability and is often due to changes in parts of the DNA called genes, such as BRCA1 and BRCA2. Cancers with this deficiency can be more readily killed by certain drugs that prevent DNA repair. Some of these drugs are approved for the treatment of several types of cancer, including ovarian, breast, pancreatic, and prostate cancers. To better identify tumors with this deficiency, we characterize the whole genome of cancer samples. We find that a comprehensive analysis of the entire genome improves the detection of homologous recombination deficiency. This type of analysis may provide a more accurate way to guide treatment decisions for people with cancer. Assaad, Hadi and Levine et al. develop a whole-genome sequencing classifier to improve the detection of homologous recombination deficiency (HRD) across a pan cancer cohort. The classifier detects HRD beyond BRCA1/2 mutations, reveals HRD-related genomic events, and correlates with treatment response in a subset of patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/108bfd65e5fc2e36aa5d310bd5ebd99b171e9621" target='_blank'>
              Whole genome sequencing approach to assess homologous recombination deficiency in a pan-cancer cohort
              </a>
            </td>
          <td>
            Majd Al Assaad, Kevin Hadi, Max F. Levine, Daniela Guevara, Minal Patel, Marvel Tranquille, Abigail King, John Otilano, Alissa Semaan, G. Gundem, J. Medina-Martinez, M. Sigouros, J. Manohar, Hui-Hsuan Kuo, D. C. Wilkes, Eleni Andreopoulou, E. Chapman-Davis, S. Tagawa, A. Sboner, Allyson J. Ocean, Manish A. Shah, E. Papaemmanuil, Cora N. Sternberg, Kevin M. Holcomb, D. Nanus, O. Elemento, Juan Miguel Mosquera
          </td>
          <td>2026-01-12</td>
          <td>Communications Medicine</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="Despite significant therapeutic advances, multiple myeloma (MM) remains incurable in most patients due to frequent tumor relapse. A major contributor to relapse is clonal heterogeneity, where subclones exhibit distinct mechanisms of therapy resistance, along with the presence of drug-resistant dormant cells. Eliminating these distinct populations, which often coexist in the tumor niche, is clinically challenging. Identifying survival mechanisms shared by drug-resistant proliferating and dormant cells holds potential for the simultaneous elimination of different tumor-repopulating clones. To identify shared mechanisms of therapeutic resistance, we analyzed clinical databases and drug-resistant myeloma cell lines. We employed pharmacologic approaches to target common candidates identified in our analysis and assessed their impact on tumor progression and survival in preclinical mouse models containing both therapy-resistant and dormant cells. We identified upregulation of several components of the Notch signaling pathway in both dormant and drug-resistant MM cells, which correlated with poor clinical outcomes in newly diagnosed MM patients. Selective blockade of NOTCH3 with a neutralizing antibody or pan-Notch inhibition with a bone-targeted inhibitor reduced tumor burden and eliminated coexisting dormant and bortezomib-resistant cells in clinically relevant models of MM disease. Our findings reveal that NOTCH3-dependent survival programs represent a shared vulnerability in both cells refractory to therapy and dormant cells. These programs can be exploited to overcome the diverse mechanisms by which cancer cells evade therapy, potentially preventing disease relapse and extending remission in patients with MM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f4eb8f9fdb1cae4aedbf1364db151a77768c687" target='_blank'>
              Targeting NOTCH3 to eradicate dormant and therapy-resistant multiple myeloma cells
              </a>
            </td>
          <td>
            H. Sabol, Bethany C. Paxton, A. Anloague, Japneet Kaur, Mattie R. Nester, Sharmin Khan, James Smith, Peter I Croucher, Michelle M McDonald, Corey O Montgomery, J.B. Stambough, C. L. Barnes, E. Ambrogini, F. H. Ebetino, C. Schinke, Cody Ashby, J. Delgado-Calle
          </td>
          <td>2026-01-05</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/369c5e33b0cad07e2b10ea8763123bd8af3e00d3" target='_blank'>
              TERT drives liver tumorigenesis beyond telomere elongation
              </a>
            </td>
          <td>
            Laura Braud, Julien Vernerey, Arnaud Guille, Pierre Cordier, Clémence Ginet, Tom Egger, Manuel Bernabe, D. Churikov, Quentin Da Costa, Aïda Meghraoui, C. Desdouets, Li Gu, F. Bertucci, Christophe Lachaud, Vincent Géli
          </td>
          <td>2026-02-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Despite its critical role in tumor evolution, a detailed quantitative understanding of the evolutionary dynamics of aneuploidy remains elusive. Here we introduce ALFA-K (Adaptive Local Fitness landscapes for Aneuploid Karyotypes), a method that infers chromosome-level karyotype fitness landscapes from longitudinal single-cell data. ALFA-K estimates fitness of thousands of karyotypes closely related to observed populations, enabling robust prediction of emergent karyotypes not yet experimentally detected. We validate ALFA-K’s performance using synthetic data from an agent-based model and empirical data from in vitro and in vivo passaged cell lines. Analysis of fitted landscapes suggests several key insights: (1) Whole genome doubling facilitates aneuploidy evolution by narrowing the spectrum of deleterious copy-number changes; (2) Environmental context and cisplatin treatment significantly modulate the fitness impact of these changes; (3) Fitness effects of copy-number changes depend on parental karyotype; and (4) Chromosome mis-segregation rates strongly influence the predominant karyotypes in evolving populations. The evolutionary dynamics of aneuploidy in solid tumors are challenging to study. Here the authors introduce a method, ALFA-K, which estimates karyotype fitness and predicts emergent karyotypes before experimental detection, and test its performance on synthetic and empirical data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e9f0c1aa74fa8dcd40ddabd793fc67f7d7cd6be" target='_blank'>
              ALFA-K: Local adaptive mapping of karyotype fitness landscapes
              </a>
            </td>
          <td>
            Richard J Beck, Tao Li, Noemi Andor
          </td>
          <td>2025-12-27</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC), a highly lethal disease often developing in the context of chronic liver disease, is marked by high relapse rates and low 5‐year survival. To investigate the multicellular ecosystem and molecular features of hepatocarcinogenesis and progression, a comprehensive analysis integrating single‐cell and spatial transcriptomics was conducted. Significant alterations were observed in various cell types, notably an increase in liver parenchymal and endothelial cells, which play critical roles in tissue repair and angiogenesis. Scissor+ cells, predominantly hepatocytes, were identified as potentially evolving into malignant cells with poor prognosis and enhanced immune evasion capabilities. Key genes associated with Scissor+ cells exhibited potential as cancer prognostic markers. Additionally, the disruption of cellular communication networks during malignant progression revealed the evolution of hepatocytes into cancerous phenotypes. The enhanced signaling pathways, such as MDK‐SDC4 and COL4A‐SDC, were identified as pivotal in driving the malignant transformation of hepatocytes. This transformative process, wherein Scissor+ cells acquire malignant features, highlights novel mechanisms of cancer progression. These findings provide deeper insights into hepatocarcinogenesis and offer potential avenues for targeted and personalized therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ea05c9d911e5acfb02ab472396d64afacfb09aa" target='_blank'>
              Spatiotemporal Evolution of Malignant Hepatocyte Clones Unveils Immune Evasion Features and Therapeutic Vulnerabilities in Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Shizheng Xiong, Dekang Ren, Jiaming Jin, Linjie Zhao, Chengjun Gong, Jiarui Liu, Wanjie Zheng, Zhiheng He, Haochuan Guo, Li Guo, Tingming Liang
          </td>
          <td>2026-01-01</td>
          <td>Biotechnology Journal</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The CRISPR-Cas9 gene editing system represents a powerful approach for precision targeting of genetic determinants involved in cancer cell survival. This study evaluated the ability of CRISPR-Cas9 to induce apoptosis in cancer cells through targeted disruption of anti-apoptotic genes. Acute myeloid leukemia (HL-60 and KG-1) and anaplastic thyroid cancer (8505C and C643) cell lines were used to knockout BIRC5 (survivin) and MADD using specific single-guide RNAs. Efficient genome editing was achieved, with indel formation exceeding 50% across all models. Targeting BIRC5 and MADD significantly reduced cell viability to 39.8–54.6% of control levels and induced substantial apoptosis. Total apoptotic populations reached 52.8–60.1% following BIRC5 knockout and 45.3–50.6% following MADD knockout. Apoptosis induction was confirmed by caspase-3/7 activation, increased Sub-G1 accumulation, and cleavage of apoptosis-related proteins, indicating activation of the intrinsic apoptotic pathway.Notably, BIRC5 disruption consistently produced stronger pro-apoptotic effects than MADD across all cancer models. Overall, these findings highlight the therapeutic potential of CRISPR-Cas9–mediated targeting of anti-apoptotic genes as a promising strategy for precision oncology and for overcoming resistance to conventional cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6c6b9d8f35f7365dc8f3a9d540781b27c8b647d" target='_blank'>
              Exploration of CRISPR-Cas9 Gene Editing System Utilization for Targeted Induction of Apoptosis in Cancer Cells
              </a>
            </td>
          <td>
            Resti Ariani, Eka Cahya Muliawati, Zola Efa Harnis
          </td>
          <td>2026-01-31</td>
          <td>Science Get Journal</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f2b7d842c50e19f140b3dba7c07392f64035b86" target='_blank'>
              Single-nucleus multi-omics delineates distinct epigenetic programs associated with tumor progression in lung adenocarcinoma.
              </a>
            </td>
          <td>
            Yeji Kim, Huiram Kang, Y. Hwang, Kyung Soo Kim, Seok Whan Moon, Jeonghan Kim, Seung Joon Kim, Hae-Ock Lee
          </td>
          <td>2026-02-11</td>
          <td>Clinical epigenetics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9fa5c1ee49d7d4c736812a4f10f208818145e4f" target='_blank'>
              Characterizing cellular subpopulations critical to treatment response in autoimmune diseases
              </a>
            </td>
          <td>
            Siqi Sun, Mulini Pingili, S. Yadav, Zhuoya Wan, Michael Macoritto, Kathleen Smith, Jing Wang, Dan Chang, N. Mahi
          </td>
          <td>2026-01-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Chemoresistance remains a major barrier in cancer therapy, frequently resulting in treatment failure and reduced patient survival. This multifaceted phenomenon arises from the interplay of well-established mechanisms such as genetic mutations, non-genetic adaptations, and tumor microenvironment (TME) mediated influences as well as newly emerging findings from recent research (2020-present). Key biochemical contributors include diminished intracellular drug accumulation through altered uptake or efflux, dysregulation of drug metabolism and bioactivation involving multiple Phase I and Phase II enzymes, genomic instability affecting DNA repair pathways, disruption of cell cycle control, and evasion of apoptosis. In addition, recent evidence highlights the roles of epigenetic reprogramming, metabolic reconfiguration, and TME-derived signaling in amplifying chemoresistance. This review integrates both foundational concepts and recent advancements in understanding drug resistance, with particular emphasis on updated insights into drug-metabolizing enzymes and their impact on therapeutic failure. It also evaluates current and emerging strategies to overcome resistance including targeting metabolic enzymes, modulating the TME, and implementing polytherapy's that address multiple resistance pathways. By synthesizing established knowledge with recent discoveries, this review highlights promising directions for improving the efficacy of cancer treatments and enhancing patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b92ce9c249ee2d1807438a427a4bdf1ac44ab032" target='_blank'>
              Molecular basis of cancer chemoresistance: biochemical insights.
              </a>
            </td>
          <td>
            R. Guneidy
          </td>
          <td>2026-01-26</td>
          <td>Drug metabolism reviews</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd77a79644efdb57765d420fb7be18f39a993472" target='_blank'>
              Cancer-associated synonymous mutations reveal stress signal-dependent mRNA folding that selectively modulates protein function
              </a>
            </td>
          <td>
            Sivakumar Vadivel Gnanasundram, Lixiao Wang, Sa Chen, Ondrej Bonczek, B. Vojtěšek, Robin Fåhraeus
          </td>
          <td>2026-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Chemoresistance remains a major obstacle to effective cancer treatment, often driven by enhanced DNA repair mechanisms that enable tumor cells to withstand genotoxic therapies. One such pathway involves the atypical DNA damage repair complex ALKBH3–ASCC, activated by the E3 ligase RNF113A in response to alkylation damage. We previously showed that SMYD3-dependent methylation of RNF113A stimulates this pathway, enhancing DNA repair and promoting resistance. Here, we identify KDM7B/PHF8 as the bona fide RNF113A demethylase, establishing one of the first functional examples of a dynamic, reversible non-histone methylation event regulating genome integrity. KDM7B antagonizes SMYD3 activity by maintaining low levels of methylated RNF113A, thereby limiting ASCC activation and sensitizing cancer cells to alkylating agents. To dissect this regulation in depth, we focused on small cell lung cancer (SCLC), a particularly aggressive malignancy characterized by limited therapeutic options and rapid acquisition of resistance. In SCLC, high KDM7B levels correlate with improved patient prognosis, whereas xenografts with reduced expression exhibit diminished responses to alkylating treatment. Moreover, CRISPR-based on/off modulation of KDM7B in genetically engineered SCLC mouse models demonstrates its central role in determining tumor response to chemotherapy. Our findings position the RNF113A–ASCC axis as a central modulator of chemoresistance, regulated through a post-translational methylation switch representing an innovative therapeutic vulnerability that could be exploited to enhance the efficacy of alkylating agents. Targeting this pathway may provide new opportunities to overcome chemoresistance, with KDM7B levels serving as a predictive biomarker to guide treatment in SCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fadb48eb1c5f8acecabf2f1b43e895eccc0dda31" target='_blank'>
              KDM7B-mediated demethylation of RNF113A regulates small cell lung cancer sensitivity to alkylation damage
              </a>
            </td>
          <td>
            Tanveer Ahmad, Xiaojie Yang, A. Foucher, Lingnan Ren, Ning Tsao, Natasha M. Flores, Jinkai Wan, L. Belmudes, E. Dubiez, Monika Chandan Bhowmik, Jessica Vayr, Simone Hausmann, Florent Chuffart, Xiaoyin Lu, Sandrine Blanchet, Tourkian Chasan, F. Boussouar, Yohann Couté, N. Mosammaparast, Fei Lan, Jan Kadlec, Pawel K. Mazur, N. Reynoird
          </td>
          <td>2026-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Abstract Cervical cancer remains a major global health burden. Despite standard-of-care therapies, 30–50% of locally advanced-stage patients develop treatment resistance, leading to recurrence and mortality. While tumour-intrinsic mechanisms (e.g., DNA methylation, cancer-associated fibroblasts) explain only partial resistance heterogeneity, emerging evidence identifies the microbiome as a critical modulator of therapeutic efficacy. This review synthesizes recent advances demonstrating that vaginal microbial dysbiosis, characterized by Lactobacillus iners enrichment and L. crispatus depletion, drives resistance through lactate-mediated metabolic rewiring, immune checkpoint stabilization and drug metabolism alteration. Longitudinal studies reveal dynamic microbiome trajectories during therapy, with geographic variations (notably HIV co-infection in sub-Saharan Africa) further modulating treatment responses. We critically evaluate microbiome-based interventions, from probiotics to engineered bacteria, including synthetic biology-driven precision microbiome therapies, and establishing standardized multi-centre trial protocols. Bridging mechanistic insights with clinical application represents a paradigm shift towards microbiome-informed cervical cancer management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a822ae4e52835707e7a0923e3a8688d6920b131" target='_blank'>
              Microbiome-driven resistance in cervical cancer therapy: from mechanistic dissection to clinical translation
              </a>
            </td>
          <td>
            Xiaoqi Wu, Dingjie Wang, Xiaodan Mao, Binhua Dong, Yue Wang
          </td>
          <td>2025-12-29</td>
          <td>Expert Reviews in Molecular Medicine</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background Intratumor heterogeneity (ITH), a critical driver of tumor evolution and immune evasion, remains inadequately characterized at the transcriptomic level in colorectal cancer (CRC), and its clinical implications are not yet fully understood. Methods We integrated transcriptomic datasets from TCGA-COAD/READ and two independent GEO cohorts (GSE40967 and GSE87211) to develop an RNA-seq–based ITH score using the DEPTH2 algorithm and to con0struct an ITH-related gene (ITRG) prognostic model. A unified cutoff value of 0.64 was established to stratify patients into high- and low-ITH groups. Using 52 survival-associated ITRGs, we generated a nine-gene prognostic signature selected from 101 distinct combinations of feature selection techniques and modeling algorithms and validated its performance in two external datasets. The tumor microenvironment and potential responsiveness to immune checkpoint inhibitors were evaluated using ssGSEA, ESTIMATE, and TIDE algorithms. SHAP analysis, together with in vitro and in vivo experiments, was employed to identify and functionally validate key regulatory genes. Results Patients in the high-ITH group had markedly poorer overall survival (OS) than those in the low-ITH group. The ITH score correlated strongly with aggressive clinical features, including T3/4 invasion depth, N1/2 nodal status, and AJCC stage III. The nine-gene prognostic signature demonstrated consistent predictive capability across the TCGA training cohort and both GEO validation cohorts. In TCGA, the model yielded time-dependent AUCs of 0.669, 0.664, and 0.645 for 1-, 3-, and 5-year OS, respectively, and retained its status as an independent prognostic indicator in multivariate Cox regression analysis; in GSE40967 and GSE87211, the corresponding AUCs ranged from 0.544–0.573 and 0.648–0.744. The high-risk subgroup was characterized by a stromal immune phenotype enriched with cancer-associated fibroblasts and macrophages, elevated expression of multiple immune checkpoints, increased TIDE scores, and higher tumor mutational burden. SHAP analysis identified IL20RB as the top risk-associated gene, whose knockdown significantly suppressed CRC cell proliferation, migration, invasion, and tumorigenicity in vitro and in vivo. Conclusion This study introduces and validates a transcriptomic ITH score and a nine-gene ITRG-based prognostic model that delineate the immune landscape and enables effective survival stratification in CRC, complementing the limitations of current staging systems. Additionally, IL20RB is highlighted as a promising therapeutic target, supporting the development of personalized immuno-targeted combination therapies in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/385eed67dbbc79210270a5630b0149ac9e500c8e" target='_blank'>
              Intratumor heterogeneity score reveals immune landscape and survival stratification in colorectal cancer
              </a>
            </td>
          <td>
            Zijing Wang, Liyuan Ma, Jinzhong Cao, Xi Chen, Xin Chang, Ruxue Ma, Hengyi Lv, Zixin Zhang, Hai Li, Tao Jiang
          </td>
          <td>2026-01-05</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="In a recent study published in Nature, Cosenza et al. 1 introduced MAGIC (machine-learning-assisted genomics and imaging convergence), a versatile platform that photolabels cells with micronuclei and other nuclear atypia, enabling direct quanti ﬁ cation of de novo chromosomal abnormality (CA) formation and ongoing chromosomal instability (CIN) at single-cell resolution. Using this framework, the authors show how spontaneous and induced lesions from pre-mitotic DNA damage and mitotic errors drive distinct CIN trajectories, with TP53 inactivation markedly amplifying CA burden and complexity, highlighting how CIN emergence and propagation prior to clonal expansion may inform tumour initiation, progression and the identi ﬁ cation of actionable biomarkers and therapeutic vulnerabilities. CIN is the ongoing process by which cells acquire new CAs through repeated chromosome missegregation and structural rearrangements, altering chromosome number and structure across divisions (Fig. 1a). CAs are pervasive across cancers and typically affect more genes than substitutions or small insertion/ deletion alterations, fuelling clonal diversi ﬁ cation and tumour adaptation. 2 – 5 Accurate chromosome segregation and genome integrity rely on coordinated processes such as DNA replication and repair, telomere maintenance, spindle assembly and mitotic exit. 2 When these are compromised and cells bypass senescence and apoptosis, concurrent pre-mitotic and mitotic defects drive ongoing CIN. 2,5 However, most CIN">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0c5eb4ff0382eff914e4d728cb5754b2a5b73b7" target='_blank'>
              It’s a kind of MAGIC: uncovering the origins of chromosomal instability
              </a>
            </td>
          <td>
            D. Gómez-Peregrina, César Serrano
          </td>
          <td>2026-02-05</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a27e17b141d07dfd3357f3ee906c1f3a2792dd3c" target='_blank'>
              SigDyn: single-cell mutational signature dynamics
              </a>
            </td>
          <td>
            Sara B. Costa, Joana P. Gonçalves
          </td>
          <td>2026-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Identifying the mutant genes that are selected during carcinogenesis is key to identifying candidates for intervention and understanding the processes that promote transformation. Here we applied two selection metrics to study the dynamics of mutational selection in a mouse model of ultraviolet light driven skin carcinogenesis in which multiple synchronous tumors develop in each animal. Sequencing normal skin and tumors over a time course revealed two genetic routes to squamous carcinoma. Nonsynonymous Trp53 mutants were positively selected in both epidermis and tumors and present in 90% of tumors. The remaining tumors carried other oncogenic mutants, including activating Kras mutations. However, other positively selected mutant genes lost their competitive advantage in heavily mutated epidermis and in tumors. We found ten mutant genes under negative selection in normal skin, one of which was also negatively selected in tumors. In addition one gene was negatively selected in tumors but not normal skin. We conclude that analysing selection in normal tissue alongside tumors may resolve the dynamics of selection in carcinogenesis and refine the identification of cancer drivers. Sequencing mutants in both normal skin and tumors that arise from it in a mouse model of ultraviolet light driven carcinogenesis reveals mutant selection changes as cancers develop. Only p53 mutants are selected throughout squamous carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4638b6cb2b56a741aa25d0049e47e04146d5e21" target='_blank'>
              The dynamics of mutational selection in cutaneous squamous carcinogenesis
              </a>
            </td>
          <td>
            G. Skrupskelyte, J. Fowler, S. Dentro, Carine Winkler, Irina Abnizova, Niklaus Beumer, R. Sood, Thomas Quarrell, C. King, J. Kamarashev, E. Guenova, M. Gerstung, Benjamin A. Hall, Liliane Michalik, P. Jones
          </td>
          <td>2026-01-12</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="The B‐cell lymphoma 2 (BCL‐2) inhibitor venetoclax (VEN) in combination with hypomethylating agents (HMAs) has improved treatment outcomes for acute myeloid leukemia (AML) patients unfit for intensive chemotherapy and is increasingly used in the relapsed/refractory setting. However, primary resistance remains a significant challenge, affecting 20%–35% of treatment‐naïve and around 50% of previously treated AML patients. To investigate the mechanisms driving primary resistance to VEN–HMA therapy, we analyzed genetic, transcriptomic, BCL‐2 family protein expression, and ex vivo drug sensitivity data from 101 AML patients and correlated these profiles with clinical outcomes to VEN–HMA. Our study found that blasts from refractory patients exhibit an elevated BCL‐XL/BCL‐2 protein expression ratio, an immature CD34+CD38− phenotype, and frequent TP53 mutations. Consistent with the high ratio of BCL‐XL/BCL‐2, resistant samples showed increased ex vivo sensitivity to the dual BCL‐2/BCL‐XL inhibitor navitoclax. In addition, SMAC mimetics were effective in refractory blasts, which correlated with high TNF gene expression in these cells. Ex vivo treatment with the combination of navitoclax and SMAC mimetics further enhanced the eradication of VEN–HMA refractory blasts, although toxicity was also observed in healthy CD34+ cells. In conclusion, our integrative analysis identifies molecular signatures associated with primary VEN–HMA resistance and highlights BCL‐2/BCL‐XL inhibition and SMAC mimetics as therapeutic strategies to target resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f404303a3f5b827792610f3342a27ad893ee2ef5" target='_blank'>
              Drivers of clinical resistance to venetoclax and hypomethylating agents in acute myeloid leukemia and strategies for improving efficacy
              </a>
            </td>
          <td>
            Ida Vänttinen, Joseph Saad, Tanja Ruokoranta, S. Kytölä, Guangrong Qin, Bahar Tercan, Pia Ettala, A. Partanen, M. Pyörälä, J. Rimpiläinen, T. Siitonen, Mikko Manninen, P. Valk, G. Huls, V. Thorsson, C. Heckman, Mika Kontro, H. Kuusanmäki
          </td>
          <td>2026-01-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95eb5193d6ef19193b98a8f6a8fdb80c9022ea9d" target='_blank'>
              Updating ACC preclinical models: characterization of two new patient-derived cell lines
              </a>
            </td>
          <td>
            A. Abate, M. Tamburello, Claudia Bonera, G. Tiberio, Sara Baldelli, Mattia Carini, Alessia Inglesi, Maria Scatolini, D. Cosentini, M. Laganà, Marta Leporati, P. L. Poliani, Domenico Cosseddu, D. Brugnoni, A. Berruti, S. Sigala
          </td>
          <td>2026-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="This study aimed to explore the potential genetic underpinnings of childhood acute lymphoblastic leukemia (ALL). Initially, an exome-wide association analysis of 89 ALL survivors and 356 controls was conducted to identify candidate genes, incorporating single-variant tests and annotation-weighted aggregation tests. Single-cell analyses characterized the candidate genes’ biological context, while aggregate testing linked these genes to clinical traits in survivors, along with bulk tissue transcriptomes profiling their expression patterns in relation to secondary adverse events (e.g., progression, relapse, secondary malignant neoplasms or death). Results from single-variant tests showed that rs57421986 of hematology-related gene SPTB exhibited a suggestive association with ALL [minor allele frequency = 0.029, odds ratio (OR) = 21.7, P = 5.86 × 10− 5]. Aggregation tests of low-frequency and rare variants identified MFAP3L (P = 3.97 × 10− 8), CARD11 (P = 4.18 × 10− 8), NOL8 (P = 2.46 × 10− 10), and OR7G2 (P = 2.20 × 10− 7) as ALL-associated genes. Notably, CARD11 and NOL8 demonstrated differential expression across diverse immune cells between incident cases and controls. Clinical phenome analyses linked CARD11 to mean corpuscular hemoglobin (P = 4.32 × 10− 2), alkaline phosphatase (P = 4.33 × 10− 2), and free triiodothyronine (P = 4.85 × 10− 2), NOL8 to red blood cell count (P = 2.83 × 10− 4) and mean corpuscular volume (P = 2.04 × 10− 3), and OR7G2 to thyroid-stimulating hormone (P = 9.56 × 10− 3) and alkaline phosphatase levels (P = 3.07 × 10− 2). Furthermore, a higher expression of OR7G2 at diagnosis showed a suggestive association with the occurrence of secondary adverse events (P = 0.034). Our findings provide initial clues regarding the shared genetic factors underlying both ALL onset and prognosis. These insights may offer potential guidance for refined risk stratification and personalized survivorship care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed68046e0425c98697fcedfc5d6cbfdc047e6e06" target='_blank'>
              Low frequency and rare coding variants affect susceptibility and progression of childhood acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Xiao Liu, Ru Zhang, Heng Zhang, Junyi Xin, Huiqin Li, Yongchu Pan, Liwen Zhu, Meiyun Kang, Hua Jiang, Yongjun Fang, Yao Xue, Mulong Du
          </td>
          <td>2026-01-02</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6c066b01c86150bb1cb77ee7e3befe19a5fcc47" target='_blank'>
              Orphan Non-Coding RNAs Drive Tumorigenesis and Enable Pre-Diagnostic Detection in HPV-Negative Head and Neck Cancer
              </a>
            </td>
          <td>
            Hoang C. B. Nguyen, Lishi Li, M. Karimzadeh, Jack Ghanam, Jeffrey Wang, Yana Al Inaya, G. Lumaj, F. Hormozdiari, B. Alipanahi, A. Feng, J. Richmon, Daniel G. Deschler, M. Varvares, D. Faden, Derrick Lin, Hani Goodarzi
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb5bb553e473f1fe2d088744a8155328de5c9b80" target='_blank'>
              Therapeutic synergies that overcome carboplatin resistance in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Julia E Altman, Aaron D Valentine, Nina M. Dashti-Gibson, Emily K. Zboril, David C. Boyd, Rachel K. Myrick, Amy L. Olex, Mikhail G. Dozmorov, J. Harrell
          </td>
          <td>2026-02-03</td>
          <td>Journal of experimental & clinical cancer research : CR</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Double-Negative T (DNT) cells, lacking both CD4 and CD8 expression, play critical roles in cancer immunology, and have garnered increasing attention in cancer research. However, their heterogeneity and functional diversity within the tumor microenvironment (TME) remain underexplored. In-house and publicly available single-cell RNA sequencing (scRNA-seq) data for different cancer types were integrated after quality control and batch effect correction, followed by DNT cells separation from CD3+ T cells subtypes. Functional characteristics, intercellular communication, differentiation trajectories, regulatory networks, and clinical relevance were analyzed among different DNT subsets. Key findings were validated using multiplex immunofluorescence and spatial transcriptomics to investigate the spatial localization of DNT subsets and their interactions within the TME. Impact of γδ T cells on immunotherapy response was also assessed using MC38-based murine tumor model. By integrating scRNA-seq data from 2,369 samples across 23 cancer types, we established a comprehensive single-cell atlas of 157,025 high-quality DNT cells. Fourteen distinct DNT subsets (6 αβ DNT and 8 γδ T cell subsets) were identified, demonstrating tumor both type-specific and shared distribution patterns, as well as unique cell-cell interaction network within the TME. These subsets displayed specialized functional profiles, including cytotoxicity, antigen presentation, and immune modulation, indicating that the functional diversity of DNT cells is largely subset-specific rather than a manifestation of multifunctionality within a single population. We also delineated divergent trajectories for αβ DNT and γδ T cell subsets, including the functional plasticity of gut-resident γδ T cells transitioning between cytotoxic and immunosuppressive states. Notably, several DNT subsets were significantly associated with favorable clinical treatment outcomes, including improved responses to cancer immunotherapy. Consistently, depletion of γδ T cells in the murine tumor model significantly decreased the efficacy of PD-1 blockade, underscoring their critical role in therapeutic response. Our study uncovers the previously underappreciated heterogeneity and functional diversity of DNT cells in the TME and demonstrates their profound impact on tumor progression and immunotherapy outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecf4d4eea61e2f155251ae235dac3fc5c5c43d18" target='_blank'>
              A pan-cancer single cell landscape reveals heterogeneity and functional diversity of double-negative T cells
              </a>
            </td>
          <td>
            Qing Hao, Tao Zhou, Huayun Yan, Zhixiang Ren, Wei Mao, Li-Bin Huang, Yue-Yun Chen, Yuelan Wang, Minqin Xiao, Yajiao He, Xuyang Xia, Jing Wu, Wei-Han Zhang, Hai-Ning Chen, Maoxiang Qian, Feng Zhang, Li Yang, Lunzhi Dai, Huiyuan Zhang, Canhua Huang, Yang Shu, Yu Xu, Yang Luo, Yong Peng, Heng Xu
          </td>
          <td>2026-01-03</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Human T-lymphotropic virus type 1 (HTLV-1), the first human retrovirus identified, is linked to adult T-cell leukemia and HTLV-1-associated myelopathy/tropical spastic paraparesis. However, its post-transcriptional regulation remains poorly understood. Here, we used Oxford Nanopore direct RNA sequencing to profile the HTLV-1 transcriptome and epitranscriptome in MT2 cells. We identified 23 transcript isoforms, encompassing canonical and novel splice variants. Polyadenylation analysis revealed a predominant poly(A) tail length of around 50–100 nucleotides with transcript-specific variations. Distinct RNA modifications, including pseudouridine, N6-methyladenosine, and 5-methylcytidine, were enriched near the 3′ end and varied among transcript classes, with generally lower modification ratios in viral transcripts. These findings provide a more comprehensive map of HTLV-1 RNA splicing, polyadenylation, and modifications in MT2 cells, offering new insights into viral gene regulation and pathogenic mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a1e4aff36600c0fd3356ace5c0413dcc761d3f3" target='_blank'>
              Transcriptomic and Epitranscriptomic Landscape of Integrated HTLV-1 in MT2 Cells
              </a>
            </td>
          <td>
            Shuanglong Wei, Bohan Zhang, Jingwan Han, Hanping Li, Yongjian Liu, Lei Jia, Jingyun Li, Xiaotian Huang, Lin Li
          </td>
          <td>2025-12-30</td>
          <td>Viruses</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Therapy resistance remains a major cause of relapse and poor outcomes in acute lymphoblastic leukemia (ALL). Recent multi-omics studies in ALL have revealed that resistance arises from a combination of leukemia-specific genetic lesions, treatment-driven clonal evolution, and adaptive non-genetic programs. Genomic analyses have identified recurrent alterations associated with resistance to chemotherapy, tyrosine kinase inhibitors, and immunotherapies, while single-cell profiling has uncovered heterogeneous cell states that persist during treatment and contribute to minimal residual disease. Emerging epigenetic, proteomic, and metabolic data further indicate that reversible regulatory and signaling changes play a central role in leukemic persistence. Integrative analyses are beginning to define convergent resistance pathways and clinically relevant biomarkers, although longitudinal sampling and clinical translation remain limited. This review summarizes the current multi-omics landscape of therapy resistance in ALL and discusses opportunities to improve risk stratification and therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98ef9b63ea903857bf083205a34581a4f1ebb5b7" target='_blank'>
              Multi-Omics Approaches to Understanding Therapy Resistance in Acute Lymphoblastic Leukemia
              </a>
            </td>
          <td>
            Xiuyun Wu, Jingxin Zhang
          </td>
          <td>2026-01-29</td>
          <td>Lymphatics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Dogs play a major role in sustaining transmission of Leishmania infantum to people, thus prevention and treatment of canine leishmaniosis (CanL) to reduce transmission represents an unmet public health need. Although advances have been made in understanding how immunopathology correlates to infectiousness as CanL progresses, the immune mechanisms underlying transition between disease stages and terminal decline remain ill-defined. To address this knowledge gap, we generated a comprehensive atlas of peripheral immune cells from control dogs and naturally-infected dogs using single-cell RNA sequencing. The LeishDog Atlas captures the cellular and transcriptional complexity underlying CanL, tracing shifts in immune composition and gene expression across progressive, well-defined disease stages. Notably, we identified distinct myeloid, CD4+ and CD8+ T cell phenotypes and transcriptional states associated with disease progression. This resource provides a valuable framework for understanding systemic immune dysregulation in CanL, including determinants of T cell exhaustion and myeloid cell activation, and establishes a foundation for comparative, translational, and mechanistic studies of CanL immunopathology and transmissibility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8412687f28e64e4d6b2558529c494e2ee4440eca" target='_blank'>
              A single-cell transcriptomic atlas of peripheral blood immune cells spanning progressive canine leishmaniosis
              </a>
            </td>
          <td>
            Danielle P. Uhl, Daniel J. Holbrook, Max C Waugh, S. Dey, Najmeeyah Brown, Karen I. Cyndari, Jacob J. Oleson, Paul M. Kaye, Christine A. Petersen
          </td>
          <td>2025-12-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Cancer cells exhibit remarkable plasticity, enabling them to survive therapeutic pressure by dynamically rewiring both their epigenetic landscape and metabolic circuitry. Emerging evidence reveals that epigenetic mechanisms, including DNA methylation, histone modifications, chromatin remodeling, and non-coding RNAs, are tightly coupled to metabolic pathways through key metabolites that function as cofactors or regulators of chromatin-modifying enzymes. This reciprocal interplay establishes self-reinforcing loops that sustain tumor growth, promote heterogeneity, and drive the emergence of drug-tolerant states. In this review, we summarize current knowledge on how epigenetic remodeling shapes metabolic reprogramming and, in turn, how altered metabolite pools influence chromatin states in cancer. We highlight the central role of long non-coding RNAs and other ncRNA species in coordinating epigenetic and metabolic adaptations that underpin therapy resistance. We further examine the contribution of metabolite-dependent post-translational modifications, such as acetylation, methylation, lactylation, and succinylation, to the regulation of tumor aggressiveness and treatment response. Finally, we discuss how multi-omics integration, computational network approaches, and AI-enabled modeling are accelerating the discovery of epigenetic–metabolic vulnerabilities and informing the development of precision therapeutic strategies. Understanding and targeting this epigenetic–metabolic axis holds substantial promise for overcoming drug resistance and improving the durability of cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe9160a0725796061efe9744e0d756322382fe38" target='_blank'>
              The impact of epigenetics on tumor metabolism: Friend or foe in drug response?
              </a>
            </td>
          <td>
            Serena Castelli, Melania Franchini, Luigi Ferraro, Nicolò Mangraviti
          </td>
          <td>2026-02-09</td>
          <td>Frontiers in Epigenetics and Epigenomics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The tumour microenvironment (TME) is now recognized as pivotal to cancer progression and treatment outcomes, alongside cancer cell-intrinsic factors. Among TME constitute, cancer-associated fibroblasts (CAFs) are abundant and actively shape tumour biology. They are heterogenous population arising from multiple origins, and engage in dynamic crosstalk with cancer cells and immune infiltrates to promote virtually all the hallmarks of cancer. By influencing and reinforcement of cancer hallmark traits, CAFs create a pro-tumorigenic niche that also protects cancer cells from therapy, in this way, conferring resistance to conventional therapy. Consequently, CAFs have emerged as central players linking cancer hallmarks to therapeutic resistance. This review discusses the origins and phenotypic diversity of CAFs, the mechanisms of their communication with tumour and immune cells, and how these interactions enforce hallmark capabilities of cancer that underlie treatment resistance. We also highlight preclinical and clinical evidence for CAF-driven drug resistance and examine current strategies to target CAFs or their effects. A deeper understanding of CAF heterogeneity and function could pave the way for novel combination therapies that dismantle the tumour-CAF alliance and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d10e61bcec209fefda855e7181e3c9e5807e7aba" target='_blank'>
              Cancer-associated fibroblasts: central players in cancer hallmarks and therapeutic resistance
              </a>
            </td>
          <td>
            Yuanjun Lu, Ya Wu, Xiaoyu Xu, Yibin Feng, Ning Wang
          </td>
          <td>2025-12-01</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fef918e1d4b64862104d2dc24b6aa7569489a94" target='_blank'>
              Genomic profiling implicates candidate genes and mutagenic pathways driving lung cancer recurrence
              </a>
            </td>
          <td>
            L. Luhari, A. Valter, A. Bahcheli, K. C. Cheng, M. Bayati, A. Ustav, A. Velthut-Meikas, K. Oselin, J. Reimand
          </td>
          <td>2026-01-01</td>
          <td>None</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Aim Breast cancer is the most prevalent form of cancer among females and carries a substantial societal impact. DSN1, a component of the MIS12 complex, plays a critical role in centromere assembly, distribution, and stability. While DSN1’s role in tumors has been investigated, its specific function in breast cancer remains unclear. Methods First, we utilized bioinformatics techniques to explore DSN1 expression in breast cancer and conducted functional enrichment and correlation analyses. Subsequently, we assessed the clinical relevance of DSN1 through immunohistochemistry. Furthermore, we examined how DSN1 affects the growth of breast cancer cells by conducting CCK8 and colony formation tests. Cell cycle and apoptosis changes were assessed using flow cytometry. Moreover, we examined key genes related to cell cycle and apoptosis to further elucidate the underlying mechanisms. Finally, we screened potential drugs targeting DSN1 by drug sensitivity and molecular docking analyses. Results Bioinformatics analysis revealed that DSN1 is highly expressed in breast cancer, making it a potential diagnostic marker. Functional enrichment analysis indicated that the DSN1- overexpressed group was enriched in cell proliferation-related pathways. Cellular experiments confirmed that DSN1 promotes breast cancer proliferation by affecting cell cycle pathways, involving key molecules such as CCNB1, CCND1, CKD1, CDK4, and CDK6. Drug sensitivity analysis showed that the DSN1 high expression group was resistant to drugs such as Epirubicin, Cyclophosphamide, Ribociclib, and Palbociclib, but relatively sensitive to tamoxifen and lapatinib. Conclusions DSN1 contributes to breast cancer progression by modulating cell cycle pathways, making it a potential diagnostic and therapeutic target with clinical applicability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3515d6bfa96b38426d6222c03e6a7fb688833cda" target='_blank'>
              DSN1 drives breast cancer progression via cell cycle regulation: diagnostic and therapeutic implications
              </a>
            </td>
          <td>
            Dongyang Liu, M. Luis, Djojomoenawi Sherilyn H. G., Xiaoxuan Zhu, Xiuqin Zhang, Yuan Fang, Xiaoyan Lin, Shasha Tang, Fengfeng Cai
          </td>
          <td>2026-01-20</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) continues to pose significant therapeutic challenges, with high relapse rates driven largely by leukemic stem cells (LSCs), a rare, therapy-resistant population with self-renewal capacity, niche adaptation, and the ability to re-initiate disease. In this state-of-the-art review, we synthesize recent advances in LSC biology, addressing (i) how LSCs differ functionally and phenotypically from normal hematopoietic stem cells (HSCs), (ii) practical approaches for LSC quantification using multiparameter flow cytometry and LSC-enriched marker panels, (iii) the dysregulated signaling, metabolic and epigenetic programs that enable LSC persistence under chemotherapy and contribute to measurable residual disease, and (iv) current therapeutic strategies targeting LSC eradication, including antibody-based therapies, apoptosis and metabolic inhibitors, and emerging epigenetic agents. We also examine the key translational barriers, particularly antigen overlap with normal progenitors, microenvironmental protection, and the need for assay harmonization, while proposing a practical framework for integrating LSC assessment into risk stratification and therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cd1cbd5980c8002b0d9b3730098739618a3c127" target='_blank'>
              Decoding Leukemic Stem Cells in AML: From Identification to Targeted Eradication
              </a>
            </td>
          <td>
            E. Apostolidou, Vasileios Georgoulis, Dimitrios Leonardos, Leonidas Benetatos, Eleni Kapsali, E. Hatzimichael
          </td>
          <td>2026-01-30</td>
          <td>Diseases</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Mutations in DNA mismatch repair (MMR) pathway genes (MSH2, MSH6, MLH1, and PMS2) are linked to acquired resistance to temozolomide (TMZ) and high tumor mutation burden (TMB) in high-grade gliomas (HGG), including glioblastoma (GBM). However, the specific roles of individual MMR genes in the initiation, progression, TMB, microsatellite instability (MSI), and resistance to TMZ in glioma remain unclear. Here, we developed de novo mouse models of germline and somatic MMR-deficient (MMRd) HGG. Surprisingly, loss of Msh2 or Msh6 does not lead to high TMB, MSI, nor confer response to anti-PD-1 in GBM. Similarly, human GBM shows discordance between MMR gene mutations and TMB/MSI.Germline MMRd leads to promoted progression from low-grade to HGG and reduced survival compared to MMR-proficient (MMRp) tumor-bearing mice. This effect is not tumor cell intrinsic but is associated with MMRd in the tumor immune microenvironment, driving immunosuppressive myeloid programs, reduced lymphoid infiltration, and CD8+ T cell exhaustion. Both MMR-reduced (MMRr) and MMRd GBM are resistant to temozolomide (TMZ), unlike MMRp tumors. Our study shows that KL-50, a imidazotetrazine-based DNA targeting agent inducing MMR-independent cross-link-mediated cytotoxicity, was effective against germline and somatic MMRr/MMRd GBM, offering a potential therapy for TMZ-resistant HGG with MMR alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f59eb07f5af2f7c1bb36a92e8c3ca00e2e992ebc" target='_blank'>
              Mismatch repair deficiency drives malignant progression and alters the tumor immune microenvironment in glioblastoma models.
              </a>
            </td>
          <td>
            Montse Puigdelloses Vallcorba, N. Soni, Seung-won Choi, Kavita Rawat, T. Joshi, Sam Friedman, Alice Buonfiglioli, Angelo Angione, Zhihong Chen, Gonzalo Piñero, Gabrielle Price, Mehek Dedhia, Raina Roche, E. Radkevich, Anne M. Bowcock, Deepti Bhatt, Winfried Edelmann, R. Samstein, Timothy E. Richardson, N. Tsankova, A. Tsankov, R. Bindra, Raul Rabadan, Juan C Vasquez, D. Hambardzumyan
          </td>
          <td>2025-12-23</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33d403636ddf059830232f0617d3f6cb6b77d1ff" target='_blank'>
              Genotype-phenotype mapping identifies fetal-like CD55+ immunoregulatory cancer cells as mediators of immune escape in colorectal cancer
              </a>
            </td>
          <td>
            M. Mastel, Aitana Guiseris Martinez, Gabriele Diamante, Jasmin Meier, Luzi Schuchmann, Nikolaos Georgakopoulos, Ioannis Chiotakakos, Laura K. Steffens, Carolin Artmann, Diego Benitez, Michael Günther, Vera Thiel, Roxana Pincheira, A. Trumpp, R. Offringa, S. Ormanns, Rene Jackstadt
          </td>
          <td>2026-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Mitochondria are highly dynamic organelles that integrate metabolic regulation, signal transduction, and programmed cell death with their canonical role in adenosine triphosphate (ATP) production. Their ability to undergo continuous remodeling through the opposing processes of fusion and fission is essential for maintaining cellular homeostasis, preserving organelle quality control, and enabling adaptive responses to metabolic and oxidative stress. Among the core regulators of mitochondrial dynamics, the dynamin-related guanosine triphosphatase (GTPase) OPA1 plays a central role in inner membrane fusion, cristae architecture maintenance, bioenergetic efficiency, and the modulation of redox balance and apoptotic signaling. Accumulating evidence indicates that dysregulation of OPA1 expression or activity contributes to the initiation and progression of multiple malignancies, underscoring its importance in tumor cell survival, proliferation, metabolic adaptation, and resistance to stress. Here, we summarize current knowledge on OPA1 dysregulation in cancer and, based on preliminary, unpublished in silico analyses, we highlight the growing relevance of OPA1 as a therapeutic target, particularly through its GTPase domain and the still understudied Interface 7. Overall, these findings outline how integrated computational approaches could potentially guide the identification of novel OPA1 modulators, offering a conceptual framework that highlights OPA1 as a promising, yet still largely underexplored, target in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01bbc85b07d21254bc1200e427724eda318625e7" target='_blank'>
              OPA1 as a Cancer Target: Molecular Mechanisms, Structural Insights, and Strategies for Drug Development
              </a>
            </td>
          <td>
            Antonio Curcio, Ludovica Ganino, Ilenia Valentino, Massimo Gentile, Stefano Alcaro, R. Rocca, A. Artese, N. Amodio
          </td>
          <td>2026-01-01</td>
          <td>Antioxidants</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Abstract Extrachromosomal DNA (ecDNA) amplification represents an emerging mechanism underlying oncogene amplification, tumor heterogeneity, and drug resistance in cancer. However, the biology of ecDNA remains poorly understood because tools to engineer ecDNAs and precisely monitor their dynamics are limited. In particular, genome engineering strategies have not been established for ecDNA, which exists in tens to hundreds of copies within a single cell. Here, we report a systematic validation of ecDNA editing using standard CRISPR-Cas9 system and optimized CRISPR-Cas9 system with safeguard single-guide RNAs (sgRNAs), in which the addition of cytosine extensions finely reduces excessive Cas9 activity. The conventional CRISPR-Cas9 system induced severe cytotoxicity and markedly reduced ecDNA copy number, together with frequent micronucleus formation. Knock-in efficiency was remarkably low, highlighting an intrinsic difficulty in editing ecDNA. In contrast, the safeguard sgRNA strategy not only alleviated cytotoxicity and ecDNA loss in a cytosine-length–dependent manner but also enabled efficient knock-in into multiple ecDNA per cell. Computational simulations suggested that the degree and temporal patterns of multiple DNA cleavage events shape cell death, micronucleus formation, and rapid expansion of knock-in ecDNA. Collectively, optimization of Cas9 activity using safeguard sgRNAs enables efficient and nondisruptive ecDNA engineering, providing a powerful tool to study ecDNA biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85607d9d15aeb32494567c03b21204776a014e41" target='_blank'>
              Optimized CRISPR-Cas9 system for efficient engineering of ecDNA in cancer cells
              </a>
            </td>
          <td>
            Yohei Sugimoto, Takeru Kachi, Yu Watanabe, Mei Kubokawa, Koichi Ogami, Masaki Kawamata, Seiko Yoshino, Hiroshi I. Suzuki
          </td>
          <td>2026-01-14</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a16bf776576d5a9b29010f431eccdc0d7ad27b5b" target='_blank'>
              Mutational signatures in cancer genomes alter protein sequence motifs in cellular signaling networks
              </a>
            </td>
          <td>
            Jigyansa Mishra, M. Bayati, Zoe P. Klein, K. C. Cheng, Omar M. Wagih, Nina Adler, Maria Farina-Morillas, A. Bahcheli, J. Reimand
          </td>
          <td>2026-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83abedd58342641982b4fa326716bab2f81320dd" target='_blank'>
              Coordinate Up-regulation of Both TK1 and TS Expression in Cycling Cells: Mechanistic Basis and Implications for Dual-Targeted Cancer Therapy
              </a>
            </td>
          <td>
            Wanxing Wang, Jiawei Zhang, Guangyou Duan, Yiyao Zhang, Guofeng Xu, Zhi Cheng, Shunmei Chen, Xue Yao, Xin Li, Shan Gao
          </td>
          <td>2026-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Cancer remains a major global health challenge, marked by profound heterogeneity and adaptability that frequently limit the success of conventional and targeted therapies. Among the diverse cellular populations within tumors, cancer stem cells (CSCs) have emerged as key drivers of initiation, progression, metastasis, and therapeutic resistance. Sharing core features with normal stem cells, including self-renewal and multipotency, CSCs sustain tumor growth and contribute to intratumoral heterogeneity. Importantly, the traditional hierarchical model has been revised with the recognition of CSC plasticity—the dynamic ability of cells to transition between CSC and non-CSC states or among distinct CSC subsets. This phenotypic flexibility, governed by intrinsic genetic and epigenetic mechanisms and strongly influenced by extrinsic cues from the tumor microenvironment (TME), underpins therapeutic resistance, immune evasion, and disease recurrence. The TME represents a complex, dynamic ecosystem composed of stromal cells, immune populations, extracellular matrix components, and soluble factors. Far from a passive scaffold, the TME actively engages in reciprocal crosstalk with CSCs, sustaining stemness, modulating plasticity, and shaping tumor progression. This bidirectional interplay reinforces CSC survival, fuels metastatic potential, and contributes to therapeutic failure. A comprehensive understanding of CSC plasticity and CSC–TME interactions is therefore essential for the design of durable therapeutic strategies. This review synthesizes current knowledge of the molecular and cellular mechanisms driving CSC plasticity and their microenvironmental regulation. It further evaluates emerging approaches—including small molecules, epigenetic modulators, and TME-normalizing therapies—that target these processes, and discusses challenges and future directions for translating these strategies into precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d23c1a983bab64bd96982765223df1542f5dbed" target='_blank'>
              Targeting cancer stem cell plasticity and tumor microenvironment crosstalk: a comprehensive review
              </a>
            </td>
          <td>
            Mutaz Jamal Al-Khreisat, Waleed K. Abdulsahib, Ihsan Khudhair Jasim, H. Malathi, Priya Priyadarshini Nayak, D. A. Anand, Gunjan Mukherjee, Aashna Sinha, Oybek Ruziyev
          </td>
          <td>2025-12-21</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Replication stress (RS) is a hallmark of cancer, largely driven by oncogene activation. Due to high levels of RS, cancer cells depend heavily on the RS response mechanisms to avoid DNA damage. This dependency creates a therapeutic opportunity that can be exploited for more effective cancer treatment. This review synthesizes current mechanistic understanding of RS and RS response and further describes how targeted disruption of RS response proteins (ATR, Chk1, Wee1, PARP, RPA) has been used in preclinical and clinical studies. We summarize preclinical and emerging clinical evidence for exploiting RS for radiosensitization, and outline candidate biomarkers and functional assays for patient selection. We also highlight the links between RS, therapy-induced senescence and innate immune activation via the cGAS–STING (cyclic GMP-AMP synthase—Stimulator of Interferon Genes) pathway, and address current challenges and future directions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/290040ba46caa91c5606fcee3d46699f73704969" target='_blank'>
              Replication Stress in Cancer: Mechanistic Insights and Therapeutic Opportunities for Radiosensitization
              </a>
            </td>
          <td>
            Spyridon N. Vasilopoulos, Ioanna Tremi, I. Kotta-Loizou, A. Gkikoudi, O. Tsitsilonis, Sophia Havaki, A. Georgakilas
          </td>
          <td>2026-01-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="The increasing complexity of disease mechanisms challenges accurate diagnosis, prevention, and early risk stratification. Beyond genetic predisposition, epigenetic regulation—particularly DNA methylation—represents a dynamic molecular interface linking environmental exposures, metabolic imbalance, inflammation, and disease development. DNA methylation is the most extensively studied epigenetic mechanism and plays a central role in controlling gene expression across physiological and pathological conditions. In this review, we provide an integrated overview of DNA methylation biology and its involvement in inflammatory, metabolic, and oncological diseases, with a specific focus on pathways related to chronic inflammation and oxidative stress. We summarize evidence demonstrating how aberrant methylation patterns contribute to disease initiation and progression, highlighting recurrent epigenetic signatures affecting key regulatory genes. In parallel, we discuss current and emerging technologies for DNA methylation analysis, ranging from targeted methylation-specific assays to next-generation sequencing-based approaches, including nanopore adaptive sampling. Finally, we explore the translational potential of DNA methylation-based tests as predictive and preventive tools, emphasizing their ability to identify disease-associated molecular alterations before clinical onset. Overall, this evidence supports the integration of epigenetic profiling into future precision medicine strategies aimed at early risk assessment, prognosis refinement, and personalized prevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99748d02c4116250f185f476ead0e0e3f150a9ca" target='_blank'>
              A New Complexity Layer: DNA Methylation and the Predictive Impact of Epigenetic Tests
              </a>
            </td>
          <td>
            Giorgio Ladisa, Francesca Montenegro, A. Picerno, Alessio Nigro, Antonella Cicirelli, A. Stasi, M. Fiorentino, P. Pontrelli, L. Gesualdo, Fabio Sallustio
          </td>
          <td>2026-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Abstract Cancer development is driven by somatic evolution and clonal selection. However, traditional selective pressure analysis methods have treated all sites within a gene equally, such a gene-level model oversimplifies the complexity of cancer evolution. In this study, we introduced CN/CS-calculator, a novel site-specific method that can capture selective pressures acting across different gene sites. By deciphering the interplay between the selection pattern and the function of a gene in oncogenesis, CN/CS-calculator uncovers a unique class of mini-driver genes, which exhibit weak positive selection, with certain critical sites providing context-dependent promoter effects on the fitness of cancer subclones while others are constrained by evolutionary conservation. Our method emphasizes the importance of site-specific analysis in uncovering how subtle evolutionary forces shape cancer biology. The refined understanding offers new insights into the mechanisms of cancer heterogeneity and molecular evolution, with potential implications for advancing therapeutic strategies and prognostic assessments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f1b33838c929960cc9e2d6ebf3f9c1d9939a9da" target='_blank'>
              Identification of cancer mini-drivers by deciphering selective landscape in the cancer genome
              </a>
            </td>
          <td>
            Xunuo Zhu, Wenyi Zhao, Siqi Wang, Jingwen Yang, Jingqi Zhou, Binbin Zhou, Ji Cao, Bo Yang, Zhan Zhou, Xun Gu
          </td>
          <td>2026-01-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="INTRODUCTION
Epithelial-mesenchymal transition (EMT) is a fundamental biological process. During EMT, epithelial cells transition to a mesenchymal phenotype, thereby contributing to embryonic development, tissue renewal, and cancer progression. EMT is a well-recognized key driver of tumor invasion and metastasis. However, the transcriptional differences between the physiological and cancer-associated EMT remain incompletely understood.


METHODS
In the present study, we applied an integrative framework that combined transcriptomic profiling, functional enrichment analysis, and machine learning. The analysis was performed on 89 RNA-sequencing datasets derived from mouse cell lines and tissues, encompassing both normal and malignant contexts. This approach aimed to identify and prioritize genes systematically and signaling pathways associated with EMT.


RESULTS
Differential gene expression and pathway enrichment analyses revealed an over-representation of shared core biological processes related to cell adhesion, cytoskeletal remodeling, and morphogenesis, in both normal and cancer-associated EMT. Nonetheless, cancer-associated EMT exhibited additional enrichment for developmental and neural-related programs, including neurogenesis and gliogenesis. Machine learning models consistently prioritized candidate EMT biomarkers, with greater transcriptional heterogeneity observed in cancer samples.


CONCLUSION
Collectively, this integrative analysis delineates distinct transcriptional profiles between malignant and physiological EMT. The enrichment of neural-related programs in cancer-associated EMT highlights potential mechanisms that contribute to malignant cellular plasticity. In addition, the analysis identifies candidate biomarkers for future investigation of EMT heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80a8493b3b761607ee71b80e8ef5993a0ef10cfe" target='_blank'>
              Uncovering EMT-Associated Molecular Mechanisms Through Integrative Transcriptomic and Machine Learning Analyses.
              </a>
            </td>
          <td>
            Şehriban Büyükkılıç, Hani Alotaibi, A. Georgakilas, Athanasia Pavlopoulou
          </td>
          <td>2026-01-22</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Abstract The development of biomarkers for population screening, early cancer detection, monitoring, and recurrence surveillance offers substantial potential to improve patient outcomes and save lives. Nullomers are short k-mers that are absent from a human genome, and neomers are the subset of nullomers that emerge recurrently due to somatic mutations during cancer development. Here, we have developed neomerDB, a database that encompasses a catalogue of neomers across cancer types and organs. We examined 10 000 whole exome sequencing and 2658 whole genome sequencing tumour-matched samples and identified the set of neomers associated with each cancer type and organ. We also analysed 76 215 whole genomes and 730 947 whole exomes of individuals from diverse ancestries, from which we removed nullomers and neomers that can arise due to germline variants in the population. Finally, we conducted a case study demonstrating that neomers can be utilized to detect glioblastoma from liquid biopsy samples (n = 38), utilizing cell-free DNA and cell-free RNA, achieving a Receiver Operating Characteristic - Area Under the Curve score of 0.98 and a precision-recall score of 0.99. neomerDB is a user-friendly database that enables advanced searches, provides interactive visualizations, and download options for neomer biomarkers. neomerDB is publicly available at https://neomerDB.com/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d61e31009fa3725049e1e6c9f385fb155746c12" target='_blank'>
              neomerDB: a comprehensive database of neomer biomarkers in cancer
              </a>
            </td>
          <td>
            Kimonas Provatas, Candace S. Y. Chan, Ioannis Kerasiotis, Eleftherios Bochalis, Akshatha Nayak, Brad E Zacharia, Georgios A. Pavlopoulos, Wei Li, IIias Georgakopoulos-Soares
          </td>
          <td>2026-01-15</td>
          <td>Database: The Journal of Biological Databases and Curation</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Tumor adaptation and therapeutic resistance represent major limitations to sustained treatment efficacy in modern oncology. Despite advances in targeted therapies and personalized approaches, disease progression and relapse remain frequent, reflecting the intrinsic biological complexity of malignant tumors. This review synthesizes high-impact evidence addressing the mechanisms that drive tumor adaptation under therapeutic pressure, with particular emphasis on intratumoral heterogeneity, evolutionary dynamics, and cellular plasticity. The analyzed literature supports the view of cancer as a dynamic and evolving ecosystem composed of genetically and phenotypically diverse subpopulations that respond heterogeneously to treatment. Resistance emerges as a predictable consequence of Darwinian selection, shaped by both genetic alterations and non-genetic adaptive processes, including reversible drug-tolerant states and microenvironmental influences. The findings further highlight the limitations of static, single-time-point molecular profiling in capturing tumor complexity and guiding long-term therapeutic decisions. Evolution-informed strategies, such as adaptive therapy and rational treatment sequencing, are discussed as biologically grounded approaches that may improve disease control by managing, rather than attempting to eradicate, heterogeneous tumor populations. This integrative perspective is particularly relevant for diverse healthcare settings, including those in Latin America, where conceptual frameworks can support clinical decision-making in variable resource contexts. Overall, understanding tumor adaptation and resistance as emergent evolutionary phenomena provides a coherent foundation for advancing personalized oncology, improving clinical reasoning, and strengthening oncology education">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb1ab93e274f88a42001d0b9860cca9a59037481" target='_blank'>
              Evolutionary Dynamics of Tumor Adaptation and Therapeutic Resistance in Personalized Oncology
              </a>
            </td>
          <td>
            Alexis Ramírez Hernández, José Arnulfo Perez Carrillo, Juan Diego Paloma Meza, Adrián Jesús Santoyo Rojas, Patricio Sebastian Ordóñez Montalvo, Wilmer Rene Cujilema Guillin, Jorge Che Enseñat, Zuleika Morales
          </td>
          <td>2026-01-26</td>
          <td>IECCMEXICO</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29746864ab4e71ce2bf45dd37a7db9f0d5365e74" target='_blank'>
              Spatial patterns of APOBEC mutagenesis in the tumour microenvironment of Asian breast cancer
              </a>
            </td>
          <td>
            Zi-Ching Tan, Yang Wu, Zhen Wei Neo, Mai Chan Lau, S. Teo, J. P. S. Yeong, S. Chang, Jia-Wern Pan
          </td>
          <td>2026-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background: Rapid advances in biotechnology provide researchers with the opportunity to integrate omics profiles (genomics, epigenomics, transcriptomics, proteomics, etc.) with multiple phenotypes or experimental conditions. In cancers such as acute myeloid leukemia (AML), where combination therapies are standard of care, identifying genetic drivers of drug resistance requires evaluating how genes are associated with multiple drug response phenotypes. Statistical analyses associating omics profiles with multiple phenotypes yield multiple significance values and rankings for each of many genes. There is a great need to consolidate these multiple rankings into a consensus ranking to prioritize specific genes for detailed follow-up wet-lab or clinical studies. Methods/Results: Here, we evaluate the well-known Fisher’s method, the sum of squared z-statistics (SSz), and the recently published cellMCD method as tools for gene prioritization. In simulation studies, cellMCD showed very similar or highly superior performance to the widely used Fisher’s and SSz methods. These advantages were also observed in an example application involving a CRISPR drug screen of an acute myeloid leukemia cell line. Conclusions: In summary, our results indicate that cellMCD should be more widely used for prioritizing discoveries from multiple omic association studies. These methods are available as an R package on github.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/936ca737a6856489d929417a1a5e1ec23e0e8b36" target='_blank'>
              cellMCD Effectively Discovers Drug Resistance and Sensitivity Genes for Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Dora Obodo, Nam H. K. Nguyen, Xueyuan Cao, P. K. Parcha, Christopher D. Vulpe, J. Lamba, S. Pounds
          </td>
          <td>2026-01-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Abstract Loss‐of‐function mutations in lysine methyltransferase 2 (KMT2C) are frequently observed in breast cancer and are significantly associated with enhanced tumour heterogeneity, aggressive clinical behaviour, and poor prognosis. Although KMT2C has been established as a tumour suppressor, the precise molecular mechanisms by which its deficiency drives metastasis and therapy resistance remain incompletely understood. This review comprehensively examines the multifaceted roles of KMT2C in breast cancer pathogenesis, with a particular emphasis on its interplay with critical oncogenic signalling networks and functional axes, including the IFNγ‐EMT/MET axis, the MAPK/ERK and RAS/PI3K/AKT/mTOR pathways, and the KDM6A–MMP3 regulatory circuit. Furthermore, we discuss emerging therapeutic strategies for KMT2C‐deficient breast cancers, such as epigenetic modulators (EZH2 inhibitors, KDM6A inhibitors, and BET inhibitors), DNA damage response (DDR)‐targeting agents, and pathway‐specific inhibitors. These insights not only elucidate the complex biological functions of KMT2C in breast cancer but also provide a rationale for developing precision therapies tailored to KMT2C‐mutant tumours.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f8b0c30c025d7f0171e5e3f66b18d5c0855e666" target='_blank'>
              Targeting lysine methyltransferase 2C deficiency: New frontiers in breast cancer therapy and prognosis
              </a>
            </td>
          <td>
            Shuangmei Tong, Qiu Wang, Yun Chen, Zunjie Bo, Ya Zhang, Yan Liu
          </td>
          <td>2026-02-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Genomic instability caused by defective DNA double-strand break (DSB) repair is a key determinant of cellular radiosensitivity. The Long–Evans cinnamon (LEC) rat is a rare naturally occurring model with marked radiosensitivity, and a major quantitative trait locus, X-ray hypersensitivity 1 (xhs1), has been mapped to rat chromosome 4; however, the causal mechanism has remained unclear. Here, we investigated the cellular and molecular basis of xhs1-associated radiosensitivity using LEA and LEC rat-derived cells and human cultured cells. Exploratory RNA-seq of pre-hepatitic liver tissue identified a sequence variant within the Hmces transcript in LEC rats. Consistently, HMCES protein levels were markedly reduced in multiple tissues and liver-derived cell lines from LEC rats. Functional analyses showed that reduced HMCES activity prolonged γH2AX signaling after X-ray irradiation, indicating delayed DSB resolution. Clonogenic survival assays demonstrated increased radiosensitivity in HMCES-deficient cells, which was partially rescued by restoring HMCES expression in stable LEA/LEC lines. Moreover, pimEJ5GFP reporter assays revealed significantly decreased end-joining repair activity in HMCES-knockout human cells. Together, these results establish HMCES as a critical mediator of DSB repair and cellular radioresistance, identify HMCES dysfunction as a core genetic determinant underlying xhs1-associated radiosensitivity, and provide mechanistic insight into radiation response architecture in a naturally occurring radiosensitive model.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30f515e4b3f8ad17b7c3e245f9057ce8bc125fc8" target='_blank'>
              Identification of HMCES as the Core Genetic Determinant Underlying the xhs1 Radiosensitivity Locus in LEA/LEC Rats
              </a>
            </td>
          <td>
            Eisuke Hishida, Masaki Watanabe, Takeru Sasaki, Tatsuya Ashida, Keisuke Shimada, Tadashi Okamura, Takashi Agui, Nobuya Sasaki
          </td>
          <td>2026-01-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4dbc11170b8c3915b19ca1579bf9e3cf87692a5a" target='_blank'>
              Spatio-temporal resource landscapes govern the confinement and escape of therapy-resistant mutants in structured populations
              </a>
            </td>
          <td>
            Nico Appold, Timon Citak, Auguste Palm, Jona Kayser
          </td>
          <td>2025-12-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The phenomenon of transgene silencing was first observed shortly after the generation of the initial transgenic plants. The vast body of experimental data accumulated since then constitutes an invaluable resource for dissecting the mechanisms of epigenetic gene regulation. Silencing operates at either the transcriptional (TGS) or post-transcriptional (PTGS) level and is predominantly mediated by small interfering RNAs (siRNAs). Although these two epigenetic pathways involve distinct sets of proteins and enzymes, they share fundamental mechanistic features: the generation of double-stranded RNA (dsRNA), its processing into siRNAs by DICER-LIKE (DCL) enzymes, and the assembly of an Argonaute-centered effector ribonucleoprotein complex (RISC). Guided by sequence-specific siRNAs, this complex identifies complementary target sequences with high precision. A comprehensive understanding of these regulatory pathways enables the targeted induction or suppression of specific plant genes. This review traces the history of experimental findings regarding the loss of recombinant gene activity in transformants and their progeny, which collectively established the foundation for elucidating the molecular mechanisms of transgene silencing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ca1645109162b7bd31d62f8f845655fb6172721" target='_blank'>
              Gene Inactivation in Transgenic Plants—A Unique Model for Studying Epigenetic Regulation of Gene Expression
              </a>
            </td>
          <td>
            T. Marenkova, A. Zagorskaya, I. Deyneko, Elena V. Deineko
          </td>
          <td>2026-01-01</td>
          <td>Plants</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Multidrug resistance (MDR) is a central cause of chemotherapy failure and tumor recurrence and metastasis, and its mechanism involves enhanced drug efflux, target mutation, upregulation of DNA repair and remodeling of the tumor microenvironment. ABC transporter protein (P-gp, MRP, and BCRP)-mediated efflux of drugs is the most intensively researched aspect of the study, but the first three generations of small-molecule reversal agents were stopped in the clinic because of toxicity or pharmacokinetic defects. Natural products are considered as the fourth generation of MDR reversal agents due to their structural diversity, multi-targeting and low toxicity. In this paper, we systematically summarize the inhibitory activities of monoterpenes, sesquiterpenes, diterpenes and triterpenes against ABC transporter proteins in in vitro and in vivo models and focus on the new mechanism of reversing drug resistance by blocking efflux pumps, modulating signaling pathways such as PI3K-AKT, Nrf2, NF-κB and remodeling the tumor microenvironment. For example, Terpenoids possess irreplaceable core advantages over traditional multidrug resistance (MDR) reversers: Compared with the first three generations of synthetic reversers, natural/semisynthetic terpenoids integrate low toxicity (mostly derived from edible medicinal plants, half-maximal inhibitory concentration IC50 > 50 μM), high target specificity (e.g., oleanolic acid specifically inhibits the ATP-binding cassette (ABC) transporter subtype ABCC1 without cross-reactivity with ABCB1), and multi-mechanistic synergistic effects (e.g., β-caryophyllene simultaneously mediates the dual effects of “ABCB1 efflux inhibition + apoptotic pathway activation”). These unique characteristics enable terpenoids to effectively circumvent key limitations of traditional synthetic reversers, such as high toxicity and severe drug–drug interactions. Among them, lupane-type derivative BBA and euphane-type sooneuphanone D (triterpenoids), as well as dihydro-β-agarofuran-type compounds and sesquiterpene lactone Conferone (sesquiterpenoids), have emerged as the core lead compounds with the greatest translational potential in current MDR reverser research, attributed to their potent in vitro and in vivo MDR reversal activity, low toxicity, and excellent druggable modifiability. At the same time, we point out bottlenecks, such as low bioavailability, insufficient in vivo evidence, and unclear structure–activity relationship and put forward a proposal to address these bottlenecks. At the same time, the bottlenecks of low bioavailability, insufficient vivo evidence and unclear structure–activity relationship have been pointed out, and future research directions such as nano-delivery, structural optimization and combination strategies have been proposed to provide theoretical foundations and potential practical pathways for the clinical translation research of terpenoid compounds, whose clinical application still requires further in vivo validation and translational research support.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc8e568cf68a8569a80453b9e89afe2909550d62" target='_blank'>
              Terpenoids: Emerging Natural Modulators for Reversing ABC Transporter-Mediated Multidrug Resistance in Cancer Chemotherapy
              </a>
            </td>
          <td>
            Lanfei Ma, Dina Mahemuti, Yuanhong Lan, Jianxiong Xu, Wenfang Li, Zhengding Su, Jinyao Li, Aytursun Abuduwaili, Ayitila Maimaitijiang
          </td>
          <td>2026-01-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The clinical implementation of BCR::ABL1 tyrosine kinase inhibitors (TKIs) for the treatment of chronic myeloid leukemia (CML) represents one of the big successes of mechanism-based cancer therapy. In 2025, survival of patients who start TKI therapy while in the chronic phase is approaching that of age-matched controls. Despite this paradigm shift, significant challenges remain. Some patients still develop overt TKI resistance and progress to bast phase, and the majority continue to harbor residual leukemia and require life-long TKI therapy. Growth and survival signals arising from the microenvironment or from within the leukemia cells confer various degrees of resistance to support a spectrum of leukemic activity ranging from overt acute leukemia in blast phase to persistence of minimal residual disease in patients with a deep molecular response. Here we review cell-intrinsic resistance, covering both reactivation of BCR::ABL1 kinase activity and the less welldefined mechanisms underlying BCR::ABL1-independent TKI resistance. We propose that the pathways used by CML to escape TKI effects reflect the potential and the constraints of BCR::ABL1- driven reprogramming of hematopoietic stem and progenitor cells and that the role of BCR::ABL1 functions other than kinase activity may be underappreciated, providing a rationale for the clinical development of BCR::ABL1 degraders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/185fbac39dc0a4541073b2d9679567a034e30319" target='_blank'>
              Intrinsic cellular resistance to BCR::ABL1 inhibitors.
              </a>
            </td>
          <td>
            Nataly Cruz-Rodriguez, Yulieth Torres-Llanos, M. Deininger
          </td>
          <td>2026-02-12</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>93</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) is a circular, double-stranded DNA molecule distinct from chromosomal DNA, primarily found in cancer cells as a subtype of extrachromosomal circular DNA (eccDNA). Unlike eccDNAs found in normal cells, cancer-associated ecDNAs are large, clonal, and carry complete oncogenes. These ecDNAs are increasingly recognized as crucial drivers of cancer pathogenesis, contributing significantly to the evolution of tumor heterogeneity and the acquisition of therapeutic resistance through mechanisms such as high-level gene amplification and altered gene regulation. Historically, the understanding of these mobile genetic elements in cancer has been limited. This review synthesizes current knowledge on ecDNA’s structural and functional features, formation mechanisms and roles in cancer initiation, progression and therapeutic resistance. Moreover, we summarize five emerging therapeutic approaches that target ecDNA to inform future cancer research and precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb27a6296222e1300898cc02edbec7c0adb86fa2" target='_blank'>
              Extrachromosomal DNA and cancer: function, formation, and clinical implications
              </a>
            </td>
          <td>
            Yucai Wu, Rui Rui, Tai Tian, Jiaying Zheng, Shi-ming He, Liqun Zhou, Xuesong Li, Yanqing Gong
          </td>
          <td>2026-01-19</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="ESR1 mutations are the leading cause of endocrine therapy resistance and progression in ER-positive metastatic breast cancer. ESR1 mutations are detected in ~50% of metastatic breast cancer patients, and identification of effective targeted therapeutics are critically needed. Here, we identified enrichment of dysregulated replication stress and DNA damage responses in multiple ESR1 mutant models. Targeting the replication stress response utilizing checkpoint inhibition in combination with PARP inhibition synergistically suppressed growth, induced cell cycle arrest, and attenuated DNA replication. PARP inhibition blocked metastatic dissemination in vivo and reduced both PARP1 and ER-regulated protein expression. PARP trapping by olaparib treatment with or without endocrine therapy resulted in a significant increase of co-localized DNA-bound PARP1 and ER protein in ESR1 mutant cells, indicating ER-PARP1 co-regulation in ESR1 mutant breast cancer. Long-term treatment with endocrine therapy plus the CDK4/6 inhibitor abemaciclib led to the emergence of a Y537S ESR1 mutation in a cell line, which exhibited dysregulation of replication stress response, enhanced DNA damage response, and synergistic responses to inhibitors of these pathways. PARP inhibition also synergized with clinically relevant endocrine therapy in ESR1 mutant models, reducing tumor growth both ex vivo and in vivo. Together, these results identify replication stress and DNA damage responses as key dysregulated pathways in ESR1 mutant breast cancer with significant clinical potential for PARP inhibition in this metastatic breast cancer subset.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b53eddcd306fde536181ab48bb570449ec8d1816" target='_blank'>
              Defective DNA Damage Response is a Targetable Therapeutic Vulnerability in ESR1 Mutant Breast Cancer.
              </a>
            </td>
          <td>
            Sarah K. Herzog, Jessica H Stevens, G. Gu, Sandra L. Grimm, Kloma Cardoza, Autumn G M Hawkins, Hangqing Lin, Daniela Ramos, Amanda R Beyer, David G Edwards, D. Dustin, Harry J. Yang, Nicole Liang, A. Khan, Tasneem Bawa-Khalfe, Daniel J. McGrail, Shiaw-Yih Lin, C. Coarfa, S. Fuqua
          </td>
          <td>2026-01-07</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Ewing sarcoma is an aggressive bone tumor of adolescence characterized by a hallmark EWSR1::ETS fusion oncogene. The resulting chimeric oncoprotein drives tumorigenesis by reshaping transcriptional and epigenetic landscapes. However, how it is transcriptionally regulated and whether additional master transcription factors (MTFs) form a core regulatory circuit (CRC) in Ewing sarcoma remain unclear. Using an extensive panel of Ewing sarcoma cell lines and primary tumors, we mapped super-enhancers and identified enrichment of GGAA microsatellites, confirming their specificity to Ewing sarcoma as compared to other pediatric cancers and normal tissues. Integrating transcriptomic, epigenetic, 3D chromatin conformation, and dependency data, we predicted a set of MTFs potentially forming a CRC. However, functional validation demonstrated that these MTFs neither establish auto-regulatory loops nor confer robust proliferative dependencies typical of CRCs in other pediatric tumors. Instead, EWSR1::FLI1 emerged as an “hegemonic” oncoprotein, regulating expression of these MTFs without reciprocal regulation. Knockdown of EWSR1::FLI1 strongly shifted H3K27ac profiles toward mesenchymal states, whereas silencing individual or combined MTFs did not alter cell growth or EWSR1::FLI1 expression. These findings highlight the absence of a classical CRC in Ewing sarcoma and emphasize EWSR1::FLI1 as the dominant oncoprotein and a major vulnerability in this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/941b3d130e41a819129bbb3b8a4a261f87cf3d37" target='_blank'>
              The hegemonic EWSR1::ETS oncoprotein overrules core regulatory circuitry principles in Ewing sarcoma
              </a>
            </td>
          <td>
            Sandrine Grossetête, Sakina Zaidi, M. Orth, Sarah Morice, Karine Laud, Caroline Louis-Brennetot, Anna Sole Ferre, Virginie Perrin, F. Cidre‐Aranaz, Virginie Raynal, M. Aynaud, Sylvain Baulande, Marco Wachtel, I. Janoueix-Lerosey, Erika Brunet, Thomas G P Grünewald, Olivier Delattre, Didier Surdez
          </td>
          <td>2025-12-25</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/213457505483d2de87291e80a391cdd8fe7c3867" target='_blank'>
              Linking somatic mutations in cancer to the electronic properties of DNA
              </a>
            </td>
          <td>
            Benoît de Witte, Cyril Karamaoun, Pauline Hermans, Maxime Tarabichi, F. Pucci, Marianne Rooman
          </td>
          <td>2026-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06b83eddceaa6549ab81ac78629425aa46eb6aed" target='_blank'>
              MSH2 Prevents Liver Tumorigenesis by Regulating Cell Cycle Checkpoints under Chronic Inflammation.
              </a>
            </td>
          <td>
            Shigeharu Nakano, Atsushi Takai, Eriko Iguchi, Yosuke Fujii, Haruka Amino, Masayuki Ueno, Takahiko Ito, Mari Teramura, M. Mishima, Ken Kumagai, Tadashi Inuzuka, Haruhiko Takeda, Y. Eso, Takahiro Shimizu, T. Maruno, Hiroshi Seno
          </td>
          <td>2026-01-29</td>
          <td>Carcinogenesis</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Targeting cell cycle checkpoints has emerged as a promising strategy in cancer therapy, yet single-agent inhibitors often fail because of compensatory mechanisms. Here, we demonstrate that coinhibition of ATR (RP-3500) and PKMYT1 (RP-6306) induces synthetic lethality in Rb1-deficient breast cancers by disrupting both S-G2 and G2-M checkpoints. This leads to replication stress, premature mitotic entry, and accumulation of DNA damage. In vitro, Rb1-deficient breast cancer cells exhibited marked apoptosis and loss of clonogenic survival, whereas Rb1-proficient models remained resistant to combination treatment. Genetic manipulation confirmed this dependency: Rb1 knockdown sensitized resistant models, whereas reexpression conferred protection. In vivo, patient-derived xenograft (PDX) models recapitulated these findings. Rb1-deficient tumors regressed after ATR/PKMYT1 coinhibition, whereas Rb1-proficient tumors showed only modest responses. Combination therapy was well tolerated without weight loss or measurable toxicity. Biomarker analysis revealed increased γH2AX and reduced Ki67 staining exclusively in Rb1-deficient PDX models, underscoring the specificity of this response. Mechanistically, Rb1 loss impaired double-strand DNA repair by attenuating homologous recombination and nonhomologous end joining, leading to replication fork collapse, chromosomal instability, and mitotic catastrophe. Proteogenomic analysis identified JNK/p38 stress response pathway activation as a key driver of apoptosis after ATR/PKMYT1 inhibition in Rb1-deficient cells. Clinically, a retrospective analysis of stage IV breast cancer datasets revealed that Rb1-low tumors display reduced DNA repair pathway activity in triple-negative and CDK4/6 inhibitor-resistant luminal breast cancers. These results identify Rb1 loss as a predictive biomarker for ATR/PKMYT1-targeted therapy, offering a potential precision treatment strategy for advanced breast cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd18d92004042bbcc1904fd2be3bc1b5cf617210" target='_blank'>
              Rb1 deficiency induces synthetic lethality with ATR and PKMYT1 coinhibition in breast cancer cell lines and patient-derived xenografts.
              </a>
            </td>
          <td>
            Xiao-Ting Jiang, Mei-Kuang Chen, Jiwang Lee, X. Rao, Yan Wang, Linjie Luo, Jing Wang, Kelly K. Hunt, K. Keyomarsi
          </td>
          <td>2025-12-24</td>
          <td>Science translational medicine</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Highlights What are the main findings? Only 2.3% (52 total) of all human antisense genes have been examined in head and neck cancer (HNC) thus far. HOXA10-AS, LEF1-AS1, MSC-AS1, and ZEB2-AS1 have clinical promise for future biomarker development. What are the implications of the main findings? Robust cross-validations on antisense gene aberrations will promote biomarker development and implementation in the HNC field, as genetic biomarkers are lacking. Antisense gene aberrations and treatment responses are underexplored and warrant future studies. Abstract Antisense genes (usually suffixed by -AS) represent a class of long non-coding RNAs (lncRNAs) transcribed from the opposite strand of annotated human genes or exon(s). A total of ~2236 human antisense genes exist in the human genome. Their genomic locations with respect to the corresponding sense genes, their dysregulated expression patterns in cancer specimens, and clinical associations with patient outcomes reveal their potential importance in clinical settings. As of today, there lacks a comprehensive review of HNC-associated antisense genes/transcripts to help move forward the antisense field for genetic biomarker development or future drug research. In total, 2.3% (52/2236 antisense genes) of all known human antisense genes have been investigated in head and neck cancer (HNC). Thus, we perform a comprehensive review of the genomic aberrations (mutations, copy number changes, RNA-expression dysregulation, and single nucleotide polymorphisms) associated with HNC patient prognosis, disease progression, cancer cell signaling, drug sensitivity, and radio-resistance. Four antisense genes, namely HOXA10-AS, LEF1-AS1, MSC-AS1, and ZEB2-AS1, have been clinically cross-validated and have consistently demonstrated to be associated with patient outcomes in multiple independent cohorts by different research teams, with clear evidence for the prioritization of clinical biomarker development in HNC. Single nucleotide polymorphisms (SNPs) of antisense genes with evidence for HNC risk or outcomes should be further validated in different ethnic groups, for potential global HNC applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb801fbeee9cde00558b26b3b11efef6455c2950" target='_blank'>
              Genomic Aberrations of Antisense Gene Transcripts in Head and Neck Cancer
              </a>
            </td>
          <td>
            Jishi Ye, S. M. Abbang, Yuen-Keng Ng, V. Lui
          </td>
          <td>2025-12-19</td>
          <td>Cells</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fa4dc4e0545f448a9b7a462aef101f7b3480c9c" target='_blank'>
              Genomic alterations and their correlation with metabolic-related genes in lung cancer.
              </a>
            </td>
          <td>
            Gauri Gaur, N. Jha, Lokesh Gambhir, Seema V Acharya, Dhruv Kumar
          </td>
          <td>2026-02-11</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="
 Tumour heterogeneity, encompassing genetic, epigenetic, and microenvironmental diversity, remains a fundamental obstacle in precision oncology. Traditional bulk sequencing captures only averaged molecular profiles, thereby masking rare yet functionally critical subpopulations that drive malignant progression and therapeutic resistance. Recently, the emergence of single‐cell sequencing technologies has overcome the limitations of bulk approaches, enabling high‐resolution analyses of the genome, transcriptome, epigenome and proteome at the single‐cell level. These advances have enabled detailed mapping of tumour ecosystems, identification of key cellular subtypes, reconstruction of evolutionary trajectories and elucidation of intercellular communication networks within the tumour microenvironment. Accumulating evidence demonstrates that single‐cell technologies elucidate fundamental aspects of tumour biology and reveal potential diagnostic and therapeutic targets. This review systematically summarises the recent advances and applications of single‐cell sequencing in the field of precision oncology, with particular emphasis on its applications in mechanistic discovery, diagnosis, therapy, and prognosis. Furthermore, we discuss current challenges related to technology, data analysis, and clinical translation, and outline future research directions. In summary, single‐cell sequencing has profoundly reshaped our understanding of tumour biology and is propelling oncology into a new era of precision, prediction, and personalisation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11cbf7015a15504645fc0387a8b9ed997f9b6eef" target='_blank'>
              Single‐cell sequencing for cancer precision medicine: From mechanism discovery to diagnosis and therapeutics
              </a>
            </td>
          <td>
            Yue Zhao, Hancong Li, Huanzuo Yang, Gongshuang Zhang, Xinyue Fu, Peiheng Li, Puxing He, Shuang Wu, Han Luo
          </td>
          <td>2026-01-29</td>
          <td>Clinical and Translational Discovery</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82b68b7fa61f9edb29d61947a73ec839e1cdd7c8" target='_blank'>
              CRISPR-Cas9 technology: a breakthrough in cancer gene therapy
              </a>
            </td>
          <td>
            Z. I. Jameel
          </td>
          <td>2026-01-12</td>
          <td>Egyptian Journal of Medical Human Genetics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Approximately 5–10% of all cancer types are hereditary cancer syndromes, which are caused by pathogenic mutations in cancer susceptibility genes. In this investigation, a hereditary cancer pedigree was collected from a province in southern China, and the proband was a 31-year-old woman with breast cancer. Utilizing blood whole exome sequencing technology and bioinformatics analysis, the sole heterozygous missense mutation in PIBF1 that exhibits trait segregation was identified: PIBF1 (p.R405Q). The pedigree also included two other mutations that may be linked to carcinogenesis: RAD51D (p.K91Ifs*13) and BRCA2 (p.G3134Afs*29). This research concentrated on PIBF1 (p.R405Q) and employed breast cancer as a tumor model. In vitro and in vivo experiments showed that PIBF1-WT suppressed breast cancer cell proliferation, colony formation, invasive ability, and tumorigenesis. However, PIBF1 (p.R405Q) attenuated or inhibited the function of PIBF1-WT. Mechanistically, PIBF1-WT resisted cisplatin-induced DNA damage, significantly down-regulated the expression of γ-H2AX, and affected DNA damage repair, thus exerting a cancer inhibitory function. Interestingly, PIBF1 (p.R405Q) affects protein stability, thereby mitigating or eliminating the inhibitory effect of PIBF1. Furthermore, this family’s polygenic risk factors for cancer are analyzed, and there is speculation about potential synergistic effects between PIBF1 and DNA damage repair genes like BRCA2 and RAD51D. In conclusion, PIBF1 regulates the cell cycle and DNA damage repair, PIBF1(p.R405Q) increases susceptibility to cancer, multiple DNA damage repair gene mutations may synergistically promote cancer progression in this cancer family lineage, and PIBF1(p.R405Q) may be one of the polygenic risk factors for familial hereditary cancer syndromes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cccf48b367531a228006163c115c6d6e4b0e36b" target='_blank'>
              PIBF1 (p.R405Q) germline variant identified in cancer susceptibility family impairs protein stability and function
              </a>
            </td>
          <td>
            Lili Shao, Liang Wen, Qiao Xu, Jianing Zhao, Rongji Yu, Yang Wang, Xinchun Liu, Jinzheng Wu, Shubing Zhang
          </td>
          <td>2025-12-19</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Cancer cell characteristics are determined by gene expression, influenced by genomic, epigenetic, and transcriptional modifications. Genomic rearrangements and transcriptional splicing can result in the formation of fusion genes. BCR-ABL1 is an established fusion gene employed as a biomarker in leukemia. A single gene can amalgamate with several other genes and may impact cellular fate. Ethnicity-specific variants of fusion genes have been identified, such as the TMPRSS2-ERG variation observed in prostate malignancies among African-American, Caucasian, and Japanese populations in research studies. Next-generation sequencing has provided a new method for predicting genomic and transcriptomic changes. We aim to identify fusion genes in the Indian population using cancer samples to enhance diagnostic outcomes. This study performed a meta-analysis of tumor-specific RNA sequencing data for liver, tongue, and ovarian cancers, which are available online. It identified known fusion genes, including TRO-MAGED2, KRT14-S100A9, RNASE10-CD38, ACTN4-ACTN1, RGPD1-RANBP2, CTSC-RAB38, C15orf57-CBX3, AMBRA1-CKAP5, ATP2B3-ATP2B4, CNKSR3-IPCEF1, E2F4-RPL14, and MZT2A-MZT2B, along with 101 novel fusion genes. Novel fusion genes GABRP_SCGB3A2 and WWOX_FUT1 were identified in all three tumor tissues. GABRP acts as a tumor inducer, whereas SCGB3A2 functions as a tumor suppressor. WWOX2 serves as a tumor suppressor, whereas FUT1 functions as a promoter of malignancy. The interplay between tumor inducers and suppressors may serve as a survival mechanism for cancer cells, a subject that has received limited research attention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4406b3ded6f3886093045dfec09d59265a20dd7e" target='_blank'>
              RNAseq-based meta-analyses revealed tumor suppressor-inducer fusion events in liver, oral, and ovarian cancer in the Indian population: a cancer cell surviving mechanism.
              </a>
            </td>
          <td>
            Rahul Yadav, Hafsa Khan, Poonam Singh, Pramod Kumar, Dinesh Kumar Singhal
          </td>
          <td>2026-02-09</td>
          <td>Nucleosides, nucleotides & nucleic acids</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="This review examines how biotechnology advances (CRISPR/Cas9, next-generation targeted therapies, nanotechnology-based drug delivery, and immunotherapies) can be applied to address cancer drug resistance worldwide. It also considers the economic burden of resistance, inequities in access to biotechnology solutions, and ethical concerns surrounding rapid innovation, particularly in low-resource settings. A narrative review synthesized evidence from basic science studies, clinical trials, translational research, and policy analyses. Evidence was prioritized for 2015-2025 publications. The synthesis highlights resistance biology and evaluates how precision medicine, biomarker-guided treatment, and high-throughput drug screening can inform individualized regimens and rational combinations. Breakthroughs in gene editing, targeted inhibitors, nanocarriers, and immune engineering can counter key resistance mechanisms, including resistance-conferring mutations, altered drug transport, immune evasion, and tumor microenvironment-mediated protection. Despite progress, implementation barriers remain substantial: high drug and development costs, limited molecular diagnostics and manufacturing capacity, and regulatory and governance challenges that can delay adoption and widen disparities, particularly in low- and middle-income countries. Integrating biotechnology innovations within precision medicine frameworks may improve treatment selection and patient outcomes. Maximizing public health impact requires affordability and financing strategies, robust ethical oversight, timely regulatory pathways, and coordinated global collaboration to ensure access to effective therapies across health systems worldwide.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac60bfada39476a6a86007db1f4232c219cedf76" target='_blank'>
              Applying biotechnology to overcome cancer drug resistance and improve public health outcomes.
              </a>
            </td>
          <td>
            Franklin Adjei, Bernard Frempong, Augustine Afriyie
          </td>
          <td>2026-02-04</td>
          <td>Osong public health and research perspectives</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains one of the leading causes of cancer-related mortality worldwide. Compared with traditional two-dimensional (2D) models, patient-derived CRC organoids more faithfully preserve the genomic, transcriptomic, and architectural features of primary tumors, making them a powerful intermediate platform bridging basic discovery and clinical translation. Over the past several years, organoid systems have rapidly expanded beyond conventional epithelial-only cultures toward increasingly complex architectures, including immune-organoid co-culture models and mini-colon systems that enable long-term, spatially resolved tracking of tumor evolution. These advanced platforms, combined with high-throughput technologies and clustered regularly interspaced short palindromic repeats (CRISPR)-based functional genomics, have substantially enhanced our ability to dissect CRC mechanisms, identify therapeutic vulnerabilities, and evaluate drug responses in a physiologically relevant context. However, current models still face critical limitations, such as the lack of systemic physiology (e.g., gut–liver or gut–brain axes), limited standardization across platforms, and the need for large-scale, prospective clinical validation. These gaps highlight an urgent need for next-generation platforms and computational frameworks. The development of high-throughput multi-omics, CRISPR-based perturbation, drug screening technologies, and artificial intelligence-driven predictive approaches will offer a promising avenue to address these challenges, accelerating mechanistic studies of CRC, enabling personalized therapy, and facilitating clinical translation. In this perspective, we propose a roadmap for CRC organoid research centered on two major technical pillars: advanced organoid platforms, including immune co-culture and mini-colon systems, and mechanistic investigations leveraging multi-omics and CRISPR-based functional genomics. We then discuss translational applications, such as high-throughput drug screening, and highlight emerging computational and translational strategies that may support future clinical validation and precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ed98ccf8eb7ee4178076cc0e4368c14e7e399d9" target='_blank'>
              An Integrative Roadmap for Advancing Colorectal Cancer Organoid
              </a>
            </td>
          <td>
            Youqing Zhu, Ke He, Zhi Shi
          </td>
          <td>2026-01-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf877f758acfe7b3824ac8b63584f4eb82346275" target='_blank'>
              Mdig shapes 3D chromatin architecture of immune checkpoint genes and limits metastasis of triple negative breast cancer cells
              </a>
            </td>
          <td>
            Ziwei Wang, Ziqi Liu, Chitra Thakur, Yiran Qiu, Jun Wang, Mehdi Damaghi, John D Haley, Fei Chen
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Tumor evolution involves genetic, transcriptional, and phenotypic alterations that shape cancer cell behavior and interactions with the microenvironment. While single-cell technologies have advanced our understanding of this process, spatial dynamics remain incompletely characterized. Here, whole-exome sequencing (WES), imaging mass cytometry (IMC), and spatial transcriptomics (ST) were integrated to study molecular evolution and immune responses in two lung adenocarcinoma (LUAD) mouse models: a genetically engineered model (129S4/Sv-KrasLSL-G12D, termed 129S4 K) and a carcinogen-induced precancer model (129S4 U). Compared to 129S4 K, 129S4 U tumors exhibited higher mutational, neoantigen but lower copy number variation (CNV) burdens at matched developmental timepoints, consistent with findings of higher mutational burden in human smoking-related LUAD than nonsmoking LUAD. We profiled over 1.4 million spatial single cells from 284 IMC regions of interest and 51,531 spatial transcriptomic spots from 156 lesions across 141 mice. Macrophage abundance increased with tumor progression, while CD8 T-cell and B-cell densities declined in late-stage LUAD. 129S4 U showed greater immune infiltration in both tumor and adjacent normal tissue, higher T-cell cytotoxicity signature score in line with its higher mutational and neoantigen burdens. LUAD progression was marked by early morphological shifts and late-stage changes in cell states and interactions. These data define spatial and genetic landscapes of LUAD development and provide a framework for investigating immune evolution and therapeutic strategies in early carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a0e7ad7c0953d4cab7fb1f9dcb251447c108ead" target='_blank'>
              Spatial Profiling Reveals Distinct Molecular and Immune Evolution of Mouse Lung Adenocarcinoma Precancers with or Without Carcinogen Exposure.
              </a>
            </td>
          <td>
            Bo Zhu, M. Aminu, Pingjun Chen, Jian-Rong Li, Chuanpeng Dong, Chenyang Li, Yanhua Tian, Shao-Wei Lu, Hong Chen, Chenxi Ma, Xin Hu, Jie-Tong Ye, Andrew Y Liu, Beibei Huang, Frank R. Rojas, Parra Cuentas Edwin Roger, Ou Shi, M. Nilsson, A. Poteete, Khaja B Khan, Wei Lu, Luisa M. Solis Soto, Junya Fujimoto, C. Haymaker, Ignacio I. Wistuba, Zhubo Wei, Linghua Wang, Don L. Gibbons, Ken Chen, Alexandre Reuben, Jason M Schenke, J. Heymach, Chao Cheng, Jia Wu, Jianjun Zhang
          </td>
          <td>2026-01-25</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Simple Summary In many types of cancer, cells double their entire set of chromosomes, a state known as tetraploidy. This change can accelerate the cancer’s growth, spread, and resistance to drugs. We wanted to understand how this chromosome doubling changes the cell’s mitochondria. We compared cells with a normal versus a doubled set of chromosomes from human cancers, yeast, and fungi. We found that in all cases, cells with doubled chromosomes were larger and contained more mitochondria, which were also more active. By identifying this shared trait, our research points to these adapted mitochondria as a potential new target for treating cancers with unstable chromosomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9e2f77cdf8945e8a66c23348025539c8ac24aa7" target='_blank'>
              Mitochondrial Adaptations Underlying Tetraploidization in Human Cancer, Fungal, and Yeast Models
              </a>
            </td>
          <td>
            Mohamed Jemaà, Ameni Bedoui, Nihel Ammous, Ali Gargouri, Mohamed Guerfali
          </td>
          <td>2026-01-01</td>
          <td>Biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Breast cancer is a leading cause of cancer-related mortality among women, with hormone receptor-positive (HR + ) disease representing the majority of cases. Despite advances in targeted therapies and antibody-drug conjugates, resistance to treatment remains a significant challenge, necessitating innovative therapeutic strategies. Epigenetic dysregulation, particularly involving histone acetylation, plays a crucial role in tumorigenesis and therapy resistance. Lysine acetyltransferase 6 (KAT6A/B), key regulators of chromatin structure and gene expression, have emerged as promising therapeutic targets in HR+ breast cancer. This review highlights the molecular mechanisms by which KAT6A/B modulate histone acetylation and gene regulation, emphasizing their roles in oncogenic pathways and cellular processes such as transcriptional activation, chromatin remodeling, and regulation of cellular senescence. We discuss preclinical evidence supporting the therapeutic potential of KAT6 inhibition and the challenges faced in epigenetic drug development, including the need for robust biomarker-driven approaches. Finally, we explore the prospects for integrating KAT6 inhibitors with existing therapies to overcome resistance and improve patient outcomes in breast cancer">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/882962d718736529869b6bb59db5c43e6a4e7b49" target='_blank'>
              Exploring KAT6 as a therapeutic target in breast cancer: epigenetic approaches for precision medicine
              </a>
            </td>
          <td>
            Cyril Roussel-Simonin, Aranzazu Fernandez-Martinez, S. Postel-Vinay, F. Mosele, Alessandro Viansone, C. Massard, Adrien Mouren, Julien Vibert, B. Pistilli
          </td>
          <td>2025-12-01</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b30edb27f07cff97af34aacf8166230cf5b8614" target='_blank'>
              Unsupervised Identification of Cancer Attractor States through the Lens of Embryonic Origin and Cancer Hallmarks
              </a>
            </td>
          <td>
            P. Deshpande, Prabal Deb, Ibrahim Al Haddabi, B. Itkin
          </td>
          <td>2026-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer remains a major global health challenge, and although immunotherapy has achieved remarkable breakthroughs, its efficacy is often limited by tumor-induced immunosuppression within the tumor microenvironment (TME). Emerging evidence indicates that metabolic reprogramming plays a pivotal role in shaping the TME and regulating antitumor immune responses. Targeting tumor and immune cell metabolism has therefore become a promising strategy to enhance the effectiveness of cancer immunotherapy. This review first summarizes the metabolic reprogramming that occurs within the TME, including alterations in glucose, lipid, and amino acid metabolism in tumor cells, as well as the metabolic adaptation of immune cells. We then highlight recent advances in natural products that modulate key metabolic pathways and their potential to reshape the immunosuppressive TME. Special emphasis is placed on natural compounds that not only inhibit tumor cell metabolism but also restore the metabolic fitness of immune cells, thereby improving antitumor immunity. In addition, advances in delivery strategies, including nanocarrier-based and stimuli-responsive systems, are reviewed for their roles in improving the bioavailability, stability, and tumor targeting of natural metabolism-regulating agents. Finally, we discuss the current status and challenges of translating natural metabolism-regulating agents into clinical applications, including issues of dose optimization, safety evaluation, and patient selection. Despite these hurdles, precision targeting of metabolic pathways, interdisciplinary collaboration, and the discovery of novel compounds—particularly immune-sensitizing agents derived from traditional medicine—are expected to accelerate progress. Collectively, natural products represent promising adjuvant strategies for cancer immunotherapy, with great potential to overcome current therapeutic limitations and improve clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8c77bd01b532b4a1633bcd32fd11764fdfa763e" target='_blank'>
              Natural products as metabolic modulators to enhance cancer immunotherapy: reprogramming the tumor microenvironment
              </a>
            </td>
          <td>
            Xiaoyu Fan, Shushu Guo, Wanfang Li, Yingyin Wang, Jie Bao, Jiuming He, Hongtao Jin
          </td>
          <td>2026-01-14</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Aim: Cancer cells divide excessively, resulting in overpopulation and hypoxia. Tumor hypoxia drives tumor development and therapeutic resistance. Hypoxia dominates pancreatic tumor microenvironments. This study aimed to ascertain alterations in gene expression linked to chronic hypoxia in the pancreatic cancer cell (PANC-1) line. Methods: PANC-1 had eight-hour hypoxic events with oxygen levels below 1%. 40 episodes were exposed three times a week. Real-time PCR arrays were used to analyze gene expression changes. This investigation compared cells treated with 20 and 40 hypoxia episodes to normoxic cells. MTT cell proliferation assays assessed hypoxic cell doxorubicin resistance. Wound-healing assays measured cell migration. 20 and 40 hypoxia exposures altered gene expression patterns significantly. Results: No alterations were observed in either stage, as most genes exhibited a resurgence after 40 episodes. Following exposure to 20 episodes of hypoxia, the expression levels of the genes HIF1AN, HMOX-1, and PKM were significantly increased by factors of 6.9, 4.5, and 3.4, respectively. The IC50 value of Doxorubicin in PANC-1 cells exhibited a 2.7-fold increase and an approximately 3.6-fold increase after 20 and 40 episodes, respectively, compared to normoxic cells. This study demonstrates that subjecting cells to extended durations of hypoxia results in distinct alterations in gene expression compared to those induced by short-term hypoxia, specifically lasting less than 72 hours. Furthermore, it sanctioned the facilitation of chemotherapy resistance through prolonged exposure. Conclusions: The study suggests that HIF1AN, HMOX-1, and PKM may serve as promising biomarkers for hypoxia in pancreatic cancer and contributors to the cellular response to prolonged hypoxia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea72b67800b6ab127a692f8378384c4dd8f3f5c4" target='_blank'>
              The gene expression alterations in chronic hypoxic PANC-1 pancreatic cancer cell line
              </a>
            </td>
          <td>
            Malek A. Zihlif, Heba A Khader, L. Younis, Ahmad Sharab, Farah Tahboub, L. Hasoun, Hamzah Hajaj, Zaid Alawneh, A. Alsayed
          </td>
          <td>2026-02-04</td>
          <td>Pharmacy Practice</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The stable and safe integration of exogenous DNA into the genome is crucial to both genetic engineering and gene therapy. Traditional transgenesis approaches, such as those using retroviral vectors, result in random genomic integration, posing the risk of insertional mutagenesis and transcriptional dysregulation. Safe harbor sites (SHSs), genomic loci that support reliable transgene expression without compromising endogenous gene function, genomic integrity, or cellular physiology, have been identified and characterized across various model organisms. Well-established SHSs such as AAVS1, ROSA26, and CLYBL are routinely utilized for targeted transgene integration in human cells. Recent advances in genome architecture, gene regulation, and genome editing technologies are driving the discovery of novel SHSs for precise and safe genetic modification. This review aims to provide a comprehensive overview of SHSs and their applications that will guide investigators in the choice of SHS, especially when complementary sites are needed for more than one transgene integration. First, it outlines safety and functional criteria that qualify a genomic site as a safe harbor site. It then discusses the two primary strategies for identifying SHSs: i) traditional lentiviral-based random transgenesis, and ii) modern genome-wide in silico screening followed by CRISPR-based validation. This review also provides an updated catalogue of currently known SHSs in the human genome, detailing their characteristics, uses, and limitations. Additionally, it discusses the diverse applications of SHSs in basic research, gene therapy, CAR T cell-based therapy, and biotechnological production systems. Finally, it concludes by highlighting challenges in identifying universally applicable SHSs and outlines future directions for their refinement and validation across biological systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/600741ca6d853466b695798a17050f6f9f6dbe8e" target='_blank'>
              Human Genome Safe Harbor Sites: A Comprehensive Review of Criteria, Discovery, Features, and Applications
              </a>
            </td>
          <td>
            Amer Ahmed, D. Di Molfetta, G. N. Iaconisi, Antonello Caponio, Ansu Singh, A. Bibi, V. Dolce, Luigi Palmieri, Vincenzo Coppola, G. Fiermonte
          </td>
          <td>2026-01-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c2f5392a871706eceb6c7ad5228a2907ac09eeb" target='_blank'>
              Association of copy number alterations with the immune transcriptomic landscape in cancer.
              </a>
            </td>
          <td>
            S. Loipfinger, A. Bhattacharya, C. G. Urzúa-Traslaviña, M. V. van Vugt, M. de Bruyn, Rudolf S. Fehrmann
          </td>
          <td>2026-01-17</td>
          <td>NPJ systems biology and applications</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Non-homologous end joining (NHEJ) is a critical DNA double-strand break (DSB) repair pathway that operates throughout the cell cycle to maintain the genomic stability of the cell. Unlike homologous recombination (HR), NHEJ is capable of repairing DSBs without the need for a homologous template, making it a rapid response mechanism, but potentially prone to errors. Central to NHEJ function and essential for the ligation through the recruitment and activation of additional repair factors, such as Artemis, XRCC4, and DNA ligase IV, is the DNA-dependent protein kinase (DNA-PK) complex. Dysregulation in the NHEJ pathway contributes to genomic instability, oncogenesis, and resistance to genotoxic therapies. Consequently, inhibitors of DNA-PK have emerged as promising therapeutic agents to sensitize tumor cells to radiation and DNA-damaging chemotherapeutics. Inhibiting the DNA-PK ability to recruit the protein complex needed for successful DSB repair promotes cell death through apoptosis or mitotic catastrophe. While inhibitors of DNA-PK can be used to enhance the effects of genotoxic therapies, the field still struggles to address critical problems: how to best exploit the differential DNA repair capacities among tumor subtypes, how to maximize radiosensitization of cancerous cells while sparing normal tissues, and how to translate preclinical studies into clinical benefits. Given that NHEJ constitutes the primary line of defense against radiation-induced damage, rapidly repairing the majority of double-strand breaks throughout the cell cycle, this review concentrates on targeting the DNA-PK complex, as the master regulator of this rapid-response mechanism, highlighting why its inhibition represents a strategic action to overcome intrinsic radioresistance. The implementation of DNA-PK inhibitors into medical practice can enable the stratification of oncologic patients into two categories, based on the tumors’ vulnerability to NHEJ disruptions. Thus, the therapeutic pathways of patients with NHEJ tumors could branch, combining traditional genotoxic therapies (radiation and DNA-damaging chemotherapeutics) with DNA-PK inhibitors to achieve an enhanced effect and improved survival outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/212ac44d554e98203d51f1eb9fd409f026d7de87" target='_blank'>
              Impeding the NHEJ Pathway for Overcoming Radioresistance in the Context of Precision Radiotherapy of Cancer
              </a>
            </td>
          <td>
            D. Niculae, R. Șerban, D. Niculae, D. Drăgănescu
          </td>
          <td>2026-01-01</td>
          <td>Pharmaceutics</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b3ae1ba137c707051b533b78e7831f38e7fb1ac" target='_blank'>
              Metabolic reprogramming-driven resistance to multi-kinase inhibitors in hepatocellular carcinoma: molecular mechanisms and therapeutic opportunities.
              </a>
            </td>
          <td>
            Junxin Li, Yu Huang, Jiawei Li, Ming Shi, Yi Xiao, Fei Du, Gongli Hu
          </td>
          <td>2026-01-27</td>
          <td>Molecular cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Oral squamous cell carcinoma (OSCC) is a prevalent and aggressive malignancy with a persistently high mortality rate, largely attributable to therapy resistance and tumor recurrence. This review comprehensively explores the critical interplay between epigenetic dysregulation and the tumor microenvironment (TME) in driving OSCC progression. We detail how key epigenetic mechanisms, including DNA methylation, histone modifications, and non-coding RNAs (ncRNAs), intrinsically transform cancer cells and actively orchestrate pro-tumorigenic TME. These alterations substantially contribute to resistance against conventional therapies. Furthermore, we discuss the therapeutic potential of targeting these pathways using epigenetic drugs (epi-drugs), such as DNA methyltransferase (DNMT) inhibitors and histone deacetylase (HDAC) inhibitors, as well as engineered extracellular vesicles (EVs). The primary objective of this review is to synthesize current knowledge on the epigenetic-TME axis, thereby providing a mechanistic foundation for developing novel therapeutic strategies. We emphasize that rational combinations of epigenetic-targeting agents with conventional treatments or immunotherapy hold significant promise for overcoming drug resistance and improving clinical outcomes in OSCC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96aaa3d5aaab2d3886a4c338f86161bbd9cbefef" target='_blank'>
              Dissecting the role of epigenetic regulation in oral squamous cell carcinoma microenvironment: mechanisms and therapeutics
              </a>
            </td>
          <td>
            Xuechao Li, Yifei Ren, Shenghua Pei, Kai Zhao, Guanyu Chen, Zhenglin He
          </td>
          <td>2026-01-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b59601a1a811d20ae0b8430613e168e0ec52452b" target='_blank'>
              Multimodal antigenic escape to GPRC5D-targeted T cell engagers in multiple myeloma.
              </a>
            </td>
          <td>
            Holly Lee, Sungwoo Ahn, Gerone A. Gonzales, Noémie Leblay, E. Barakat, Dylan Greening, Pina Colarusso, Johnathan Canton, S. Benaoudia, Elham Hasheminasabgorji, Mansour Poorebrahim, David Jung, M. Truger, Jihong Jeong, C. Riedhammer, H. Einsele, K. M. Kortüm, Jan Eckmann, Jitka Somandin, Sara-Sheena Engel, L. Boise, J. Frenking, Niels Weinhold, Konstantina Taouxi, E. Kastritis, Sheri Skerget, D. Vishwamitra, Yunje Cho, F. Maura, Marc-Steffen Raab, J. Corre, L. Rasche, Hervé Avet-Loiseau, Paola Neri, Nizar J. Bahlis
          </td>
          <td>2026-01-15</td>
          <td>Nature medicine</td>
          <td>1</td>
          <td>76</td>
        </tr>

        <tr id="Long-read whole-transcriptome sequencing (WTS) has the potential to precisely characterize fusion oncogenes that drive leukemia and other cancers. While there are a variety of general-purpose fusion detection algorithms that use modern long-read sequencing data, they show poor sensitivity for precision diagnostics in B-ALL and do not robustly assess technical and analytical parameters (ex. sequencing depth) to reliably detect fusion transcripts. FUSILLI (FUSions In Leukemia Long-read sequencing Investigator) is a novel long-read fusion detection algorithm, with a focus on targeted genomic subtyping in B-ALL. FUSILLI was evaluated against extant methods using nanopore WTS from 51 pediatric B-ALL samples sequenced at high depth and 68 at low depth (mean of 11.2M and 1.4M reads, respectively). In the high-depth cohort, FUSILLI demonstrated increased sensitivity (0.81) compared to FusionSeeker, JAFFAL, and LongGF (0.63, 0.76, and 0.70, respectively), while maintaining high specificity (0.92). At lower sequencing depth, FUSILLI showed correspondingly lower sensitivity (0.27) but still outperformed the other fusion callers (sensitivities ranging from 0.09 to 0.16). Computational down sampling suggests that ten million reads is sufficient to sensitively detect B-ALL-relevant fusions using this approach. FUSILLI detects B-ALL fusions with high sensitivity at modest sequencing depth, supporting implementation of nanopore WTS as a low-cost and globally accessible sequencing-based molecular diagnostic platform for pediatric B-ALL and other fusion-driven cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/884c4daa4f67a077c5abf5eb62ebb345ad0e753d" target='_blank'>
              Long-read whole-transcriptome sequencing and selective gene panel profiling enable sensitive detection of fusion oncogenes in pediatric B-cell acute lymphoblastic leukemia.
              </a>
            </td>
          <td>
            John Lin, Kofi B. Opoku, M. Litzow, E. Paietta, Ching-Hon Pui, S. Jeha, K. Roberts, C. Mullighan, Thomas B Alexander, Jeremy R. Wang
          </td>
          <td>2026-02-01</td>
          <td>The Journal of molecular diagnostics : JMD</td>
          <td>0</td>
          <td>110</td>
        </tr>

        <tr id="


 Approximately 30% of Crohn’s disease (CD) patients develop a refractory form, usually marked by persistent inflammation resistant to conventional therapies. The cellular interactions and genetic factors driving this resistance remain poorly understood, limiting effective treatment options



 we performed single-cell RNA sequencing (scRNA-seq) on colonic and ileal biopsies from 45 CD patients (13 treatment-naïve, 17 in remission, 15 difficultto-treat) and 10 healthy controls. To dissect interaction changes beyond general inflammation, we first compared treatment-naïve (TN) and remission (RM) CD to catch the inflammation-associated alterations. Based on this, we adjusted for inflammation effects when analyzing difficult-totreat (DTT) CD patients, comparing inflammation-corrected DTT with remission to identify DTT-specific changes, calculated as Ratio = (DTT-(TN-RM))/RM. Transcriptomic data were further integrated with whole-exome sequencing (WES) in 27 families with refractory CD to identify mechanisms to treatment resistance.



 Our study comprises 289,272 high-quality single-cell transcriptomes from 55 gut samples. Subclustering of five immune, stromal and epithelial cell compartments yielded 49 distinct cell states. Notably, our analysis revealed substantial heterogeneity in disease state between the colon and ileum, both in cellular composition and in the expression patterns of CD risk genes (Figure 1). Then, distinct tissue-specific mechanisms were evident (Figure 2). In the colon, DTT-specific increases were observed in the fibroblast to ILC-like cell interactions. Notably, the PVR–CD226 axis was uniquely enriched in DTT, with PVR and CD226 highly expressed in inflammatory fibroblasts and ILC-like cells, respectively, leading to elevated TNF expression and enhanced TNF-α production and secretion. In the ileum, DTT-specific increases were identified in HES1+ epithelial cell to ILC-like cell interactions. The IL7–IL7R axis was uniquely enriched in DTT patients, with IL7R highly expressed in ILC-like cells, accompanied by increased IL7-mediated activation of the JAK–STAT signaling pathway. In addition, our WES analysis identified mutations in TTN and RNF213, which contribute to DTT by modulating the above mechanisms in the colon and ileum, respectively. Our findings indicate that DTT CD, often involving both colon and ileum, responds poorly to monotherapy but improves markedly with combined anti-TNFα and JAK inhibition.



 This study revealed distinct regional drivers of difficultto-treat CD, highlight the need for combined treatment with anti-TNFα and JAK inhibition to achieve improved outcomes in difficult-to-treat CD.
 Conflict of interest:
 Wang, Wei: No conflict of interest
 Shen, Mengping: No conflict of interest
 Lin, Jue: No conflict of interest
 Zhang, Qi: No conflict of interest
 Zhu, Ruixin: No conflict of interest
 Zhi, Min: No conflict of interest


">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7840d827d3966b65bc7de9201cbcd899d76ba34" target='_blank'>
              DOP021 Single-Cell Profiling Reveals Regional Drivers of Difficult-to-treat Crohn’s Disease
              </a>
            </td>
          <td>
            W. Wang, M. Shen, J. Lin, Q. Zhang, R. Zhu, M. Zhi
          </td>
          <td>2026-01-01</td>
          <td>Journal of Crohn’s and Colitis</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background

Immune checkpoint inhibitors have transformed the treatment landscape of non-small cell lung cancer (NSCLC); however, primary and acquired resistance remain major clinical challenges. Immune evasion in NSCLC is increasingly recognized as a complex, dynamic process driven by tumor heterogeneity and adaptive remodeling of the tumor microenvironment. Conventional bulk profiling approaches are limited in resolving this complexity, underscoring the need for higher-resolution methodologies.

Objective
This review aims to synthesize current evidence on how single-cell transcriptomic approaches have advanced the understanding of immune evasion mechanisms in treatment-resistant NSCLC, with a focus on tumor-intrinsic programs, immune cell dysfunction, and intercellular communication networks.

Methods

A structured narrative review with systematic elements was conducted, analyzing peer-reviewed studies employing single-cell RNA sequencing and related single-cell technologies in human NSCLC. Studies were evaluated for their insights into immune escape pathways, resistance to immunotherapy, and translational relevance.

Results

Single-cell transcriptomic analyses reveal profound intratumoral and immune heterogeneity in treatment-resistant NSCLC. Resistant tumors harbor distinct malignant cell states characterized by impaired antigen presentation, dysregulated interferon signaling, and lineage plasticity. Within the immune compartment, hierarchical T-cell exhaustion, depletion of progenitor-like exhausted T cells, and expansion of immunosuppressive myeloid populations—particularly tumor-associated macrophages—emerge as dominant features of resistance. Ligand–receptor interaction analyses further demonstrate coordinated immunosuppressive communication networks that sustain immune escape at a systems level. Several single-cell–defined cellular states correlate with poor response to immune checkpoint blockade and adverse clinical outcomes.

Conclusion

Single-cell transcriptomics has fundamentally reshaped the conceptual framework of immune resistance in NSCLC, redefining it as a multicellular, dynamic ecosystem rather than a single-pathway phenomenon. These insights provide a strong biological rationale for developing biomarker strategies and rational combination therapies that target both tumor-intrinsic adaptations and the immunosuppressive microenvironment. Continued integration of single-cell technologies into translational and clinical research is essential to overcome immunotherapy resistance and improve patient outcomes in NSCLC.

Keywords: Non-small cell lung cancer; Single-cell RNA sequencing; Immune evasion; Immunotherapy resistance; Tumor microenvironment">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7495ff2bd82ff3331af5929ec1f5ca0f1ed50c9" target='_blank'>
              Single-Cell Transcriptomic Mapping of Immune Evasion Pathways in Treatment-Resistant Non-Small Cell Lung Cancer
              </a>
            </td>
          <td>
            Marco Rossi, Giulia Bianchi, Alessandro Conti
          </td>
          <td>2026-01-19</td>
          <td>Advanced Journal of Biomedicine &amp; Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Epigenetic reprogramming is a major contributor to therapy resistance in prostate cancer (PCa). Indeed, chromatin accessibility profiling classified PCa into distinct subtypes, predictive of clinical outcome. Histone modifications influence nucleosome positioning, ultimately dictating chromatin compactness. The histone methyltransferase G9a, frequently upregulated in advanced PCa, catalyzes a repressive histone modification (di-methylation of histone H3 lysine 9) that signals for gene transcriptional repression. CM-272 is a reversible and highly selective inhibitor against G9a activity, and we have previously shown in cell line models of PCa that it depicted strong anti-tumoral effects. However, fundamental aspects of altered transcriptomic programs remain unexplored. Here, we validated, in vivo, the CM-272 effects in advanced PCa, and explored the molecular mechanisms underscoring G9a inhibition-mediated effects. Specifically, the PCa patient-derived xenograft model BM18 was used to assess the mode of action of CM-272 in vivo. Moreover, RNA sequencing of tumor tissues was performed to further explore the underlying gene expression signature. Finally, chromatin immunoprecipitation followed by qPCR was used to investigate the underlying functional consequence of epigenetic targeting. Our results showed that CM-272 significantly delayed BM18 tumor growth, specifically impairing tumor cell G9a activity and proliferation. Transcriptomic profiling revealed 119 significantly upregulated and 99 downregulated genes in response to G9a inhibition. Notably, by integrating our gene expression dataset with published chromatin accessibility data, an increase in the androgen receptor (AR) signature and a concomitant decrease in the neuroendocrine (NE) features were observed in CM-272-treated samples. Additionally, pathway enrichment and master regulator analysis pinpointed both the androgen receptor and TP53 networks as significantly upregulated in treated tissues. In line with the reduction of the NE score, the MYC/MYCN were the most significantly downregulated transcription factors. Moreover, under the androgen pathway, upregulation of AZGP1, STK39, and DPP4 was observed. At the promoter level, a significant reduction in H3K9me2 levels was detected in treated samples, allowing, in turn, an increase in AR binding. Additionally, upon treatment, higher p53 binding was detected at the promoter of the p53-repressed genes AURKA, CDC25B, and CD44, whose expression was downregulated. Overall, targeting G9a with CM-272 might hold promising therapeutic potential for the management of advanced disease by modulating relevant tumoral transcriptional programs associated with core vulnerabilities in PCa.



 Filipa Moreira-Silva, Mark Zimmerli, Panagiotis Chouvardas, Beat Roth, Carmen Jerónimo, Marianna Kruithof-de Julio. G9a blockade uncovers therapeutic vulnerabilities in advanced prostate cancer, leading to tumor growth delay [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A045.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5911786a7c980e0c86fa5660d05b013ec96ea309" target='_blank'>
              Abstract A045: G9a blockade uncovers therapeutic vulnerabilities in advanced prostate cancer, leading to tumor growth delay
              </a>
            </td>
          <td>
            Filipa Moreira-Silva, Mark Zimmerli, P. Chouvardas, Beat Roth, Carmen Jerónimo, Marianna Kruithof-de Julio
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background/Objectives: Comprehensive genomic profiling (CGP) is a cornerstone of personalized oncology. However, large-scale, systematic data on the somatic mutation spectrum in Russian cancer patients are scarce. This study aimed to characterize the genomic landscape and assess the potential for matched therapy in a Russian cohort of patients with solid tumors. Methods: This retrospective study included 204 patients with various solid tumors. CGP was performed using the FoundationOne®CDx (FFPE tissue) and FoundationOne®Liquid CDx (cfDNA) platforms. The analysis assessed single-nucleotide variants, indels, copy number alterations, gene fusions, tumor mutational burden (TMB), microsatellite instability (MSI), and PD-L1 expression. Results: The most frequently mutated genes were TP53 (61.5%) and KRAS. The median TMB was 4.0 mut/Mb and was significantly lower in stage IV tumors. Significant co-occurrence was observed between KRAS and TP53 mutations, as well as between APC and KRAS mutations, which were particularly characteristic of colorectal cancer. KRAS mutations were associated with higher combined positive score (CPS) values in cases with lung cancer. Based on the CGP results, 44% of patients had findings that supported the use of an approved matched targeted therapy or immunotherapy for their tumor type. An additional 36% of patients had alterations indicating potential benefit from off-label targeted therapy. Conclusions: This study reveals the distinct genomic characteristics of solid tumors in a Russian cohort and confirms the high clinical utility of CGP for identifying actionable targets. Implementing CGP early in the diagnostic process is a necessary step towards realizing personalized treatment strategies for cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66aca90a2bac69b97335224b751826ae6cb88014" target='_blank'>
              Genetic Landscape of Solid Malignant Tumors in a Russian Cohort of Patients
              </a>
            </td>
          <td>
            Iurii K. Slepov, E.D. Kopylov, Anton A. Turchin, D. Khmelkova, V. Kaimonov, A. Isaev, R. Deev
          </td>
          <td>2025-12-19</td>
          <td>Diagnostics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26a70cf29a8102e7c49d2b80d42e52230a68a64c" target='_blank'>
              Integrative multi-omics reveals PARP14 as a key IFNγ-regulated mediator of metastatic progression in Ewing sarcoma
              </a>
            </td>
          <td>
            S. Sánchez-Serra, M. Chicón-Bosch, Núria Martorell-Villanueva, Eduardo Candeal, Ignasi Jarne-Sanz, J. Bech-Serra, D. Giménez-Llorente, Manuel Gris-Lorente, P. Monaco, L. Santana-Viera, Silvia Mateo-Lozano, Jaume Mora, Mario Niepel, Carolina de la Torre Gomez, Rebeca Sanz-Pamplona, Ana Cuadrado, Ana Losada, R. López-Alemany, Ò. M. Tirado
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Chemotherapy selection traditionally relies on tumor tissue of origin. However, since genetic alterations drive tumor behavior, it remains unclear if mutations can better predict response. We hypothesized that genetic aberrations might influence chemotherapy outcomes more than tissue origin. We retrospectively analyzed 15,474 Japanese patients with solid tumors who underwent comprehensive genomic profiling (CGP) and received cytotoxic chemotherapy. Genetic alterations and tumor origin were evaluated for objective response rate (ORR) and time to next treatment (TNT). Gene mutations were assessed across five chemotherapy classes: platinum-based, alkylating agents, antimetabolites, microtubule inhibitors, and topoisomerase inhibitors. Genomic alteration data alone did not surpass organ-based models in predicting response. For platinum-based agents, the gene-only model had an AUC of 0.575 versus 0.604 for the organ-only model. A combined gene-organ model yielded an AUC of 0.618 (P < 0.01). Certain gene-organ interactions were associated with improved outcomes. For example, APC-mutated colorectal cancer showed higher ORR and prolonged TNT (hazard ratio, 0.82; 95% CI, 0.73–0.92; P < 0.001) for platinum-based drugs. While genetic alterations alone did not outperform tumor origin as a predictor, incorporating both may improve exploratory predictions of chemotherapy response. These exploratory findings require prospective validation before any clinical application.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1934ddadd51140930510ad12ba5b67db9adbcd8d" target='_blank'>
              Exploratory analysis of gene aberrations and chemotherapy response: findings from a real-world database in Japan
              </a>
            </td>
          <td>
            N. Ishibashi, T. Kamatani, Satoru Aoyama, Masanori Tokunaga, Yusuke Kinugasa, Sadakatsu Ikeda
          </td>
          <td>2025-12-22</td>
          <td>British Journal of Cancer</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Genetic interactions can reveal gene function and identify cancer-relevant synthetic lethals, but systematic mapping in human cells is constrained by inefficient reagents, vast combinatorial search space, and diversity of cell types. Here, we leverage principles from yeast genetic networks to identify human gene modules enriched for genetic interactions. Using our Cas12a-based In4mer combinatorial knockout platform, we screen all pairwise interactions within receptor tyrosine kinase and DNA damage response modules across eight diverse cancer cell lines. We identify hundreds of unreported synthetic lethals, including a dense network within the protein glycosylation machinery, and confirm that interactions in 2D cell culture are maintained in more physiologically relevant models. Our targeted modules show up to 16-fold enrichment of interaction density, providing a scalable strategy for systematic interaction mapping.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/081a137f8aa4f89541a558bf1cd83f7f679eee7b" target='_blank'>
              Functional modules predict cancer-relevant genetic interactions in mammalian cells
              </a>
            </td>
          <td>
            Chenchu Lin, Veronica Gheorghe, Juihsuan Chou, Sabriyeh Alibai, Subin Kim, N. Anvar, Yixin Xu, Xingdi Ma, Lori L. Wilson, Russell Moser, Christopher J. Kemp, Junjie Chen, Scott Kopetz, Traver Hart
          </td>
          <td>2025-12-21</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>30</td>
        </tr>

        <tr id="Malignant transformation of mature cystic teratoma (MTMCT) of the ovary is a rare but aggressive malignancy for which no standardized chemotherapy or effective targeted therapies currently exist. To identify therapeutic vulnerabilities in MTMCT, we performed a genome-wide CRISPR-Cas9 knockout screen using the MTMCT-derived NOSCC1 cell line. Two parallel selective pressures were applied: in vivo tumorigenicity in immunodeficient mice and cisplatin exposure in vitro. From this screen, 67 negatively selected genes were identified, among which SOD1 and NDUFB4 emerged as top candidates based on high basal expression levels and clinical relevance. Integration with spatial transcriptomic data from three independent MTMCT patient tumors further supported the prioritization of these targets. SOD1 was selected for further investigation due to the availability of known pharmacological inhibitors. Both siRNA-mediated knockdown and small-molecule inhibition of SOD1 using LCS-1 significantly suppressed MTMCT cell proliferation in vitro by inducing oxidative stress and impairing cell cycle progression. This antiproliferative effect was reversed by co-treatment with N-acetylcysteine, a reactive oxygen species scavenger. In vivo validation using patient-derived xenograft models demonstrated that oral administration of LCS-1 led to significant tumor growth suppression and increased expression of apoptotic and DNA damage markers, including cleaved caspase-3 and γH2AX. These findings establish SOD1 as a critical vulnerability in MTMCT and provide preclinical evidence supporting redox modulation as a therapeutic strategy for this highly chemoresistant and understudied ovarian cancer subtype. Our integrative approach combining functional genomics, spatial transcriptomics, and pharmacologic validation offers a framework for the discovery of novel targets in rare gynecologic malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e67036000a59a86b2e3dd28f5c317d050f6d45c" target='_blank'>
              In-Tumor CRISPR-Cas9 Knockout Screening and Novel Therapy Development for Malignant Transformation of Ovarian Teratoma.
              </a>
            </td>
          <td>
            S. Tamauchi, Kosuke Yoshida, Xinyuan Wang, Atsushi Nakagawa, Akira Yokoi, N. Yoshikawa, K. Niimi, Yusuke Yamamoto, Hiroaki Kajiyama
          </td>
          <td>2026-01-13</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The tumor infiltrating T cells determine whether a patient would respond positively to immunotherapy. This paper has analyzed an mRNA scRNA-seq dataset of melanoma in the context of mapping heterogeneity in T-cells and the question of whether clinical features can shape cell clustering. Unsupervised clustering with k-means based on quality control followed by a description of the major T-cell groups by canonical immune markers. The identification of the differentially expressed genes that were identified was then done by a posteriori information based ranking approach. This study have created a confidence-weighted approach, which is able to merge Spearman and cosine similarity measures with marker-gene rules, to enhance cell-type annotation. Metadata analysis indicated that treatment status is an element that causes the state changes in / T cells pre- and post-checkpoint therapy. Combining graph-based Leiden clustering and k-means, this study refined the definition of the fine-grained T-cell states and proved that, despite the significant heterogeneity of T cells, the strong definition of subclass depends on the combination of several methods of clustering and the use of orthogonal knowledge of experiments. Pseudotime analysis found evidence in gradual transition between naive or early-activated states to cytotoxic or dysfunctional phenotypes that indicate intratumoral T-cell diversity captures a reflection not only of inherent developmental trajectories, but also of the effects due to therapy. Lastly, survival analysis revealed that there was a distinct relationship between clinical response and overall survival. These findings combined narrow the scope of the T-cell picture and offer understanding in the form of transcriptional programs connected with treatment outcomes. Refer to this link for supplementary materials: https://drive.google.com/drive/folders/1UYW-6EMjSUMVOwHxvuYdFSZwREqrGp89">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b245ce41e96da9548625adf2b4b3d69d756bc71f" target='_blank'>
              Single-Cell Transcriptomics Reveals T-Cell State Transitions and Their Association with Immunotherapy Response in the Melanoma Tumor Microenvironment
              </a>
            </td>
          <td>
            Shangyu Ning
          </td>
          <td>2025-12-24</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Simple Summary MELK is a serine/threonine kinase that regulates cell cycle progression, proliferation, apoptosis, migration, and stemness. MELK is overexpressed in many solid and hematologic malignancies. Furthermore, high MELK levels correlate with aggressive disease, poor survival, treatment resistance, and cancer stem-like features. Several small-molecule inhibitors and RNAi-targeting MELK have suppressed the development and progression of various cancers. OTSSP167 has shown promising results in phase I and II clinical trials. However, a study employing CRISPR/Cas9 knockout in breast cancer cell lines reported that loss of MELK did not impair proliferation. The review concludes that MELK is a robust prognostic biomarker of proliferation and aggressive behavior across multiple cancers but is not a universal driver of tumor growth. Discrepancies may arise from off-target effects of RNAi or inhibitors, context differences across cancer types, and differences in study settings. MELK is proposed as a conditional oncogenic vulnerability, especially in p53-deficient triple-negative breast cancer (TNBC) and glioma stem cells. Further research is needed to clarify the target selectivity of inhibitors, context-specific dependencies, and the optimal use of MELK-focused agents in combination therapies rather than as standalone treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0e62df49d62d42a4c4ea852f26c65300bd99df5" target='_blank'>
              Maternal Embryonic Leucine Zipper Kinase (MELK) in Cancer: Biological Functions, Therapeutic Potential, and Controversies
              </a>
            </td>
          <td>
            Alaeddin M. Alzeer, Saad Al-Lahham
          </td>
          <td>2026-01-01</td>
          <td>Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="ABSTRACT Introduction Chromosomal instability (CIN), a hallmark of over 90% of solid tumors, presents both a vulnerability and an opportunity in cancer therapy. KIF18A, a kinesin motor protein essential for mitotic fidelity, becomes critical for CIN cancer cells, where it suppresses lethal mitotic errors. This selective dependency makes KIF18A an attractive therapeutic target. Areas covered This review explores the structural and functional biology of KIF18A, its role in maintaining chromosomal alignment, and how its inhibition leads to mitotic failure specifically in cancer cells exhibiting high CIN. We summarize the development of small-molecule inhibitors, from early compounds like BTB-1 to second-generation agents including Sovilnesib and VLS-1488, the latter currently in Phase I/II trials. Clinical data show manageable toxicity and promising antitumor activity, particularly in high-grade serous ovarian cancer. New entrants like ATX-295 and GH2616 further expand the landscape. We also highlight efforts to refine KIF18A targeting through structural studies and emerging AI-driven drug discovery. Expert opinion KIF18A inhibition represents a mechanistically grounded, tumor-selective strategy for treating CIN-driven cancers. As clinical trials progress, optimizing patient stratification and exploring synergistic combinations will be crucial. Broadening the range of druggable sites on KIF18A, especially beyond the motor domain, could mitigate resistance and help expand therapeutic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b172472dda432c5dc196b15477d3bfabc81ddaf1" target='_blank'>
              KIF18A inhibition and its rising role in cancer therapy trials: from bench to bedside
              </a>
            </td>
          <td>
            Jia Q Truong, Tal Galper, Jessica K. Holien
          </td>
          <td>2026-01-02</td>
          <td>Expert Opinion on Investigational Drugs</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The mechanisms underlying cancer cell drug resistance have not yet been fully elucidated. Numerous processes hinder effective tumor treatment, such as cancer cell epigenetic alterations, DNA repair, drug target modification, and epithelial–mesenchymal transition. Recent studies have highlighted the ability of cancer cells to transiently enter the resting phase, namely, the G0 phase of the cell cycle under treatment conditions; this is associated with an elevated level of cell death. Cells in the G0 phase are characterized by a specific phenotype, the exact features of which remain to be fully elucidated. Cell division and cell cycle dynamics are closely associated with cell adhesion, i.e., interaction with the extracellular matrix in normal cells, whereas cancer cells are characterized by the dys-coordination of these processes. However, the precise patterns of cell adhesion alterations in association with the cell cycle remain unclear. Knowledge on the interactions of cells in the G0 phase with the extracellular matrix during the entry, the exit, and the sustenance during the resting phase is limited. At the same time, focal adhesion-mediated molecules may be an attractive goal for cancer cells in the G0 phase, targeting to overcome tumor dissemination and drug resistance. Drug resistance and immune system response evasion are specific features of nonproliferating cancer cells that call for more detailed and accurate characterization of this population of malignant cells to optimize anticancer treatment and control tumor dissemination. The present review represents the recent data on the association of focal adhesion and cancer cell surveillance due to cell cycle alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16149b6070228042071985c6c4e07c2cd03bc235" target='_blank'>
              Focal Adhesion and Cancer Cell Cycle-Associated Resistance
              </a>
            </td>
          <td>
            T. Ruksha, N. Palkina, Ekaterina Z. Lapkina, Kangdong Liu
          </td>
          <td>2025-12-25</td>
          <td>ACS Omega</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="BACKGROUND
Medullary thyroid carcinoma (MTC) is a rare neuroendocrine malignancy with a 10-year mortality rate up to 50%. Current therapies for metastatic MTC comprise various tyrosine-kinase inhibitors, but resistance often appears due to the need for lifelong treatments. Like in other tumors, genetic, epigenetic, post-transcriptional, post-translational, and cell-cell interaction events influence drug response. However, progress in understanding MTC biology is limited by the lack of reliable in vivo and in vitro models. This study aims to develop a patient-derived model faithfully reproducing the microenvironmental alterations present in MTC.


METHODS
We applied a two-step protocol consisting of a first phase in which primary cells are cultivated as multicellular spheroids and a second phase in which they are switched to adherent cultures. After evaluation of the genetic background by targeted Next Generation Sequencing, we characterized our cells phenotype by examining a panel of stem/progenitor-related markers, the secretory abilities by ELISAs, the drug response by proliferation assays, the in vivo angiogenic and invasiveness by the use of zebrafish model, the in vitro invasivity by Matrigel Dome assays and the spatial variation of stem/progenitor marker in both 3D cell models and tissue samples by confocal microscopy.


RESULTS
Our model allowed the establishment of eight MTC patient-derived cell lines with different genetic backgrounds. The cultures faithfully reproduced the changes in stem and progenitor markers that we detected in our cohort of MTC tissue samples and could be successfully xenotransplanted in zebrafish model, showing both angiogenetic and invasive properties. Drug screening assays revealed the potential of our model for the study of patient-specific responses, as we were able to identify different candidate regulators of the sensitivity to currently available therapies for MTC.


CONCLUSIONS
Our two-step protocol successfully generated primary MTC lines that maintain high plasticity, can be cultivated for several passages, and recreate the heterogeneity observed in patients' tissues. Our model will offer a robust platform for preclinical drug testing and mechanistic studies, addressing a longstanding gap in MTC research. It enables exploration of tumor microenvironment interactions and personalized therapeutic responses, supporting progress beyond current genomic-driven frameworks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/efc387a5fab7de78625be6d806bb5f60287b5f7b" target='_blank'>
              Patient-Derived in Vitro Models Reveal Insights into Medullary Thyroid Cancer Microenvironment and Resistance to Tyrosine Kinase Inhibitors.
              </a>
            </td>
          <td>
            E. Grassi, V. Ghiandai, G. Gaudenzi, E. Massardi, V. Cirello, Erika Carbone, Davide Gentilini, Sarah Uraghi, G. Gazzano, G. Dionigi, C. Colombo, Giovanni Vitale, L. Fugazzola, L. Persani
          </td>
          <td>2026-02-06</td>
          <td>Thyroid : official journal of the American Thyroid Association</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74e725a85a73539cf9bc922a83eb56481b485413" target='_blank'>
              Pan-cancer silencer transcription atlas reveals leukemia silencer hijacking FOXP1 to enhance MYC oncogene
              </a>
            </td>
          <td>
            M. Guo, Liutao Chen, Shengcheng Deng, Jingkai Zhang, Zhijie Hu, Weiwen Wang, Songyang Zhou, Yuanyan Xiong
          </td>
          <td>2026-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Cellular memory, or epigenetic memory, represents the capacity for cells to retain information beyond the underlying DNA sequence. This heritable characteristic is primarily governed by epigenetic mechanisms which enable cells to maintain specialized characteristics across divisions. This persistent cellular state is essential for fundamental biological processes, such as maintaining tissue identity and facilitating cell differentiation, especially embryonic cells. Early-stage perturbations such as assisted reproductive technologies (ART) and nutritional stress links embryonic exposures to adult health and disease within the Developmental Origins of Health and Disease (DOHaD) framework. Crucially, memory established during early embryogenesis links these epigenetic modifications to adult long-term phenotypes related to metabolic disorders. These modifications—including DNA methylation, histone modifications, and non-coding RNAs—support cellular memory transmission across cell divisions, and in certain organisms, can be transmitted across generations without alterations to the DNA sequence. This review synthesizes recent advances in epigenetic pathways that mediate cellular memory, highlights critical preimplantation windows of vulnerability and outlines gaps necessary for mammalian developing interventions that safeguard future generations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0d6f5c2aa2e2cb595481c0a26f1959e68f5a807" target='_blank'>
              How Early-Life Programming During Embryogenesis Imprints Cellular Memory
              </a>
            </td>
          <td>
            Norermi Firzana Alfian, Kei Uechi, Yoshiya Morishita, Kaname Sato, Maruhashi Yui, Jannatul Ferdous Jharna, Md Wasim Bari, Shiori Ishiyama, Kazuki Mochizuki, Satoshi Kishigami
          </td>
          <td>2025-12-23</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Mitochondria function as the primary energy hubs of cells and possess semi-autonomous genetic characteristic. Mutations in mitochondrial DNA (mtDNA) frequently lead to severe illness and even premature death. The rapid advancement of gene therapy offers promising potential for correcting such disorders. This review first aims to delineate the mechanisms of gene therapy strategies applicable to mitochondrial diseases, including the allotopic expression of mtDNA in the nucleus, mitochondrial-targeted nuclease cleavage, and mtDNA-targeted base editing. It also discusses in detail the clinical efficacy of mtDNA allotopic expression and the preclinical progress of other strategies. Furthermore, the unique physiological features of mitochondria, such as heteroplasmy and independent molecular transport mechanisms, pose distinct challenges for the clinical implementation of mitochondrial gene therapy strategies. Accordingly, this review elaborates on the current limitations of each approach. Finally, it highlights potential optimization directions to address these challenges, emphasizing that understanding heteroplasmy dynamics and their corresponding phenotypes, ensuring the safe delivery and tissue-specific expression of therapeutic elements, and maintaining long-term therapeutic specificity and efficiency are essential for the clinical translation of mitochondrial gene therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19af52e15b8608097010e37fd7cc07ef155c45a1" target='_blank'>
              Advances in gene therapy for mitochondrial genetic disorders: current status and clinical implementation challenges
              </a>
            </td>
          <td>
            Lei Lyu, Beibei Qie, Yanjie He, Feilong Chen, Bao Liu
          </td>
          <td>2025-12-01</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Antimicrobial resistance (AMR) is escalating worldwide, posing a serious threat to global public health by driving infections that are no longer treatable with conventional antibiotics. CRISPR–Cas technology offers a programmable and highly specific therapeutic alternative by directly targeting the genetic determinants responsible for resistance. Various CRISPR systems can restore antibiotic susceptibility and induce selective bactericidal effects by eliminating resistance genes, disrupting biofilm formation, and inhibiting virulence pathways. Moreover, CRISPR can suppress horizontal gene transfer (HGT) by removing mobile genetic elements such as plasmids, thereby limiting the ecological spread of AMR across humans, animals, and the environment. Advances in delivery platforms—including conjugative plasmids, phagemids, and nanoparticle-based carriers—are expanding the translational potential of CRISPR-based antimicrobial strategies. Concurrent progress in Cas protein engineering, spatiotemporal activity regulation, and AI-driven optimization is expected to overcome current technical barriers. Collectively, these developments position CRISPR-based antimicrobials as next-generation precision therapeutics capable of treating refractory bacterial infections while simultaneously suppressing the dissemination of antibiotic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3f0f21606a1f66b09cc62a141dac0135cf1b347" target='_blank'>
              CRISPR–Cas-Mediated Reprogramming Strategies to Overcome Antimicrobial Resistance
              </a>
            </td>
          <td>
            Byeol Yoon, Jang Ah Kim, Yoo Kyung Kang
          </td>
          <td>2026-01-01</td>
          <td>Pharmaceutics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/951a7c06637898ecdcaad9d7afa3a28d2b55c4a5" target='_blank'>
              Prolyl-isomerase Pin1 drives platinum resistance by regulating Notch3 stability and function in ovarian cancer.
              </a>
            </td>
          <td>
            M. V. Giuli, Angelica Mancusi, Bianca Natiello, S. Di Cristofano, Rebecca Reali, M. G. Pignataro, Daniel D'Andrea, Laura Di Magno, Carmine Nicoletti, Alessandra Giorgi, Alberto Macone, S. Camerini, Marialuisa Casella, Giovanna Peruzzi, Sabrina Zema, G. Canettieri, F. Tomao, I. Palaia, Angelina Pernazza, A. Rustighi, R. Palermo, Domenico Raimondo, A. Monti, N. Doti, Giulia d'Amati, Giannino Del Sal, Isabella Screpanti, C. Talora, D. Bellavia, S. Checquolo
          </td>
          <td>2026-02-11</td>
          <td>Journal of experimental & clinical cancer research : CR</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Genomic imprinting disorders (IDs) arise from the disruption of parent-of-origin-specific gene expression, a process governed by heritable epigenetic marks. Conventional therapies are largely symptomatic and fail to correct the underlying defect in dynamic gene regulation. Epigenome editing, using programmable DNA-targeting tools to rewrite epigenetic information, has emerged as a powerful therapeutic strategy that directly addresses this molecular pathology. This review synthesizes the transformative progress of the past decade, arguing that epigenome editing is poised to revolutionize the treatment of these conditions. We detail key breakthroughs in both editor design and in vivo delivery. These include the development of next-generation editors, such as compact “hit-and-run” systems that establish durable epigenetic memory after only transient expression—a critical feature for clinical safety. We then examine parallel advances in delivery platforms, including engineered adeno-associated virus (AAV) vectors with enhanced central nervous system (CNS) tropism and programmable lipid nanoparticles (LNPs) for precise extra-hepatic targeting. The therapeutic potential of combining these technologies is underscored by convincing preclinical data in models of Angelman (AS) and Prader-Willi syndromes (PWS), where the targeted rewriting of epigenetic marks has successfully reactivated silenced alleles and rescued disease-relevant phenotypes. Finally, we discuss the remaining challenges—including long-term durability and off-target safety—that must be overcome to translate this scientific potential into transformative medicines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51a215ce891005fe8fac9034ff5a7f3671a19208" target='_blank'>
              Advances and challenges of precision epigenetic therapy in treating genomic imprinting diseases
              </a>
            </td>
          <td>
            Ying Lv, Mingyan Li, Chai Ji
          </td>
          <td>2026-01-01</td>
          <td>Translational Pediatrics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Mechanisms driving the increase in cell growth in developing leukemia are not fully understood. We focused on epigenomic regulation of this process by analyzing the changes of chromatin marks and gene expression in leukemic cell clones as they progressed toward increased proliferation in a mouse model of acute myeloid leukemia (AML). This progression was characterized by gradual modulation of chromatin states and gene expression across the genome, with a surprising preferential trend of reversing the prior changes associated with the origins of leukemia. Our analyses of this modulation in independently developing clones predicted a small set of potential growth regulators whose transcriptomic and epigenomic progression was consistent between clones and maintained both in vivo and ex vivo. We selected three of these genes as candidates (Irx5 and Plag1 as growth suppressors and Smad1 as a driver) and successfully validated their causal growth effects by overexpression in mouse leukemic cells. Overexpression of the IRX5 gene in human MOLM13 leukemic cells suppressed cell growth both in vitro and in mouse xenografts. Public patient data confirmed expression levels of PLAG1 and SMAD1 as markers of AML status and survival, suggesting that multiomic analysis of evolving clones in a mouse model is a valuable predictive approach relevant to human AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e782878493e2646d7e50204cd904395a449780e" target='_blank'>
              Multiomic analysis of clonal development reveals new regulators of leukemic cell growth.
              </a>
            </td>
          <td>
            Gracia Bonilla, Alexander Morris, Sharmistha Kundu, Anthony Ducasse, Grace Kirkpatrick, Jelena Milosevic, N. E. Jeffries, Kashish Chetal, Emma E Yvanovich, Ting Zhao, Jun Xia, Rana Barghout, David T. Scadden, Michael K. Mansour, Robert E. Kingston, David B. Sykes, François E. Mercier, Ruslan I. Sadreyev
          </td>
          <td>2026-02-04</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da7a9e1a9e3edf4edb1c29327ece5ed4b0de7ab1" target='_blank'>
              Colorectal cancer risk variants in the 11q13.4 locus are associated with variable POLD3 transcript expression which may promote DNA damage and telomere shortening in colorectal cancer cells
              </a>
            </td>
          <td>
            E. Clarkson, N. DeLeoN, S. Aldulaimi, J. Fernández-Tajes, T. Roberts, I. Tomlinson, A. Lewis
          </td>
          <td>2025-12-27</td>
          <td>None</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract Somatic mutations, key alterations in cancer development, exert differential effects across tissues and biological layers, such as transcriptomes, proteomes, and post-translational modifications (PTMs). Although previous pan-cancer studies have characterized the molecular landscape of cancer, the effects of individual somatic mutations across different tissues remain insufficiently explored. Here, we developed Panorama to evaluate the oncogenic potential of single somatic mutations across all cancer types. We collected cancer proteogenomics or multiomics data from over 10 000 individuals across 19 cancer types. Based on five evaluation criteria, we assessed whether a specific mutation affects the abundance of a particular gene’s transcriptome, proteome, or phosphoproteome; the tumor microenvironment; specific RNA- or protein-based signaling pathways; and outlier-level overexpression of PTMs, aiding in potential drug target identification. By leveraging five oncogenic metrics, Panorama quantifies the oncogenic potential of individual somatic mutations and provides a framework for identifying driver mutations by incorporating their downstream effects. With Panorama, researchers can integrate cancer proteogenomics data, providing a comprehensive approach that enhances our understanding of single somatic mutations in specific tissues. Finally, Panorama was developed as a web-based database to ensure easy access for researchers and is freely available at http://139.150.65.64:8080/or https://github.com/prosium/panorama.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3988e7434c0ad5c1c94f629876b4c62c68b08126" target='_blank'>
              Panorama: a database for the oncogenic evaluation of somatic mutations in pan-cancer
              </a>
            </td>
          <td>
            Seung-Jin Park, Seon-Young Kim
          </td>
          <td>2026-01-15</td>
          <td>Database: The Journal of Biological Databases and Curation</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2879e795e7efcebd514454af1fc1a736f45a29d8" target='_blank'>
              Somatic and germline mutational processes across the tree of life
              </a>
            </td>
          <td>
            Sangjin Lee, Yichen Wang, Haynes Heaton, Emily Mitchell, M. Maddison, Liam M. Crowley, Patrick Adkins, N. Mieszkowska, Mark Blaxter, R. Durbin, P. Campbell
          </td>
          <td>2025-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="BACKGROUND
Cancer cells are heterogeneous, each harboring distinct molecular aberrations and being dependent on different genes for their survival and proliferation. While targeted therapies based on driver DNA mutations have shown success, many tumors lack druggable mutations, limiting treatment options. We hypothesize that new precision oncology targets may be identified through "expression-driven dependency," where cancer cells with high expression of specific genes are more vulnerable to the knockout of those same genes.


RESULTS
We developed BEACON, a Bayesian approach to identify expression-driven dependency targets by analyzing global transcriptomic and proteomic profiles alongside genetic dependency data from cancer cell lines across 17 tissue lineages. BEACON successfully identified known druggable genes, including BCL2, ERBB2, EGFR, ESR1, and MYC, while revealing novel targets confirmed by both mRNA and protein-expression driven dependency. The identified genes showed a 3.8-fold enrichment for approved drug targets and a 7 to 10-fold enrichment for druggable oncology targets. Experimental validation demonstrated that depletion of GRHL2, TP63, and PAX5 effectively reduced tumor cell growth and survival in their dependent cells.


CONCLUSIONS
Our approach provides a systematic method to identify precision oncology targets based on expression-driven dependency patterns. By integrating multi-omics data with genetic dependency screens, we've created a comprehensive catalog of potential therapeutic targets that may expand treatment options for cancer patients lacking druggable mutations. This resource offers new opportunities for precision oncology target discovery beyond mutation-based approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39d9df8d8c54b9f6f78829d05c5029fee1f72554" target='_blank'>
              Expression-Driven Genetic Dependency Reveals Targets for Precision Oncology.
              </a>
            </td>
          <td>
            Abdulkadir Elmas, Hillary M. Layden, Jacob D Ellis, Luke N. Bartlett, Xian Zhao, Reika Kawabata-Iwakawa, Zishan Wang, Hideru Obinata, Scott W. Hiebert, Kuan-lin Huang
          </td>
          <td>2026-01-29</td>
          <td>GigaScience</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="DNMT1 variants are linked to complex neurodegenerative syndromes, yet their variant-specific functional and epigenomic consequences remain poorly defined. DNMT1 variants were identified in eight patients using gene-panel or whole-exome sequencing. Functional effects were assessed by site-directed mutagenesis and transient expression in HEK293T cells. Genome-wide methylation profiling of peripheral blood leukocyte DNA was performed using Nanopore sequencing, enabling direct quantification of 5-methylcytosine (5mC). CpG island-level differential methylation and gene set enrichment analysis (GSEA) were conducted. Variants in the replication foci targeting sequence (RFTS) domain (p.Y511H, p.Y540C, p.H569R) exhibited reduced DNMT1 protein expression, decreased enzymatic activity, and cytosolic aggregation. Variants in the C-terminal catalytic domain (p.A1334V and p.P1546S) showed reduced protein expression with relatively mild enzymatic impairment. Patients carrying the p.Y511H variant demonstrated a significant reduction in global 5mC levels compared with controls. Principal component analysis revealed distinct methylomic profiles separating most patients from controls, with marked intra- and inter-familial heterogeneity. CpG island-level analysis identified a single significantly hypomethylated region in p.Y511H carriers, and GSEA revealed differential enrichment of multiple Gene Ontology biological pathways. This study defines domain-dependent functional effects of DNMT1 variants and provides the first nanopore-based methylome analysis, revealing variant-specific and heterogeneous epigenomic alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b08782723cf1505854fccf87fe2c66f251748e64" target='_blank'>
              Functional and Epigenomic Consequences of DNMT1 Variants in Inherited Neurological Disorders
              </a>
            </td>
          <td>
            Junhui Yuan, Y. Higuchi, M. Ando, A. Yoshimura, S. Nozuma, Y. Sakiyama, Takashi Kanda, Masahiro Nomoto, Takeshi Nakamura, Yasuyuki Nobuhara, Hiroshi Takashima
          </td>
          <td>2026-01-26</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32c4664b6580edc429e59cb7fc3bb0cee1ddce94" target='_blank'>
              LISA: A Comprehensive R Package for Lentiviral Integration Site Analysis in Gene Therapy Safety Assessment
              </a>
            </td>
          <td>
            Shuai Ni, Ning Wu, Yun Kang, Fengjiao Zhu, Jueyu Wang, Lina Wang, Wei Wang, Heng Mei, Kejia Kan
          </td>
          <td>2025-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cutaneous T-cell lymphoma (CTCL) remains a challenging disease due to its significant heterogeneity, therapy resistance, and relentless progression. Multi-omics technologies offer the potential to provide uniquely precise views of disease progression and response to therapy. We present here a comprehensive multi-omics view of CTCL clonal evolution, incorporating exome, whole genome, epigenome, bulk, single cell (sc) TCR, and scRNA sequencing of 99 clinically annotated serial skin, peripheral blood, and lymph node samples from 34 CTCL patients. We leveraged this extensive dataset to define the molecular underpinnings of CTCL progression in individual patients at single cell resolution with the goal of identifying clinically useful biomarkers and therapeutic targets. Our studies identified recurrent progression-associated clonal genomic alterations; we highlight mutation of CCR4, PI3K signaling, and PD-1 checkpoint pathways as evasion tactics deployed by malignant T-cells. We identified a gain of function mutation in STAT3 (D661Y) and demonstrated by CUT&RUN- and RNA-seq that it enhances binding to and transcription of genes in Rho GTPase pathways. With our previous work implicating this pathway in HDACi-resistant CTCL, these data provide further support for a previously unrecognized role for Rho GTPase pathway dysregulation in CTCL progression. Recurrent progression-associated mutations were common in the epigenetic modifier EZH2, suggesting that EZH2 inhibition may benefit patients with CTCL. Our findings support an approach in which genomic analysis is widely utilized for improved disease monitoring, biomarker-informed clinical trial design, and genome-guided therapeutic decision making. Moreover, these molecular changes present new opportunities for therapeutic targeting in this challenging and incurable cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80a0bc9d50b98f66d82985f089d8566527ff9738" target='_blank'>
              Multi-omic Study of Cutaneous T-Cell Lymphoma Reveals Single Cell Clonal Evolution in Progression and Therapy Resistance.
              </a>
            </td>
          <td>
            Hannah K. Dorando, Jared M. Andrews, Oam Khatavkar, N. Borcherding, Yulia Korshunova, Gabriela Hakeman, Rodrigo Borgiani Panigassi, Phuong Mn Vo, Ruei-Yuan Tu, Diep Tran, Chaz C. Quinn, J. Schmidt, Jahnavi Aluri, Michael T Harmon, Marcus P Watkins, Anastasia Frank, M. Cooper, Amy C. Musiek, N. Mehta-Shah, Jacqueline E. Payton
          </td>
          <td>2026-02-09</td>
          <td>Blood</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af7630e9ff15d614dc5c29742755271640ed25f5" target='_blank'>
              KMT2A-Mediated transcriptional regulation in stemness and cancer: molecular mechanisms and therapeutic opportunities
              </a>
            </td>
          <td>
            Md Rashedunnabi Akanda, Md Shiblee, Sadik Sabuj, Tanvir Ahmed, Md Jamilur Rahman, S. M. A. Salam, Byung-Yong Park
          </td>
          <td>2025-12-24</td>
          <td>Medical Oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="


 Somatic structural variations (SVs) are common in cancer. Although a small fraction of SVs in breast and ovarian cancers can be attributed to homologous recombination deficiency, the underlying molecular mechanisms for the vast majority of somatic SVs remain unclear. Here, we focus on the roles of transcription and DNA replication collisions in genomic instability in cancer. Such collisions are unavoidable in cells since both transcription and replication use the same DNA as template. We hypothesized that transcription replication collisions (TRCs), if not properly repaired, would lead to collapsed replication forks and result in SVs. To this end, we studied somatic SVs in 6193 high-coverage whole-genome sequenced primary and metastatic tumors from three independent pan-cancer cohorts. A total of 13 conserved SV signatures, representing independent molecular mechanisms, were deconvoluted from these cohorts using non-negative matrix factorization approach. We detected replicated-strand bias, the expected footprint of transcription-replication collision, in large tandem duplications (TDs) across multiple cohorts. This bias was only observed in expressed genes, consistent with TRCs depending on transcription activity. Large TDs were abundant in female-specific (breast, ovarian and uterus), upper gastric-intestinal tract, and prostate cancers. They were associated with CDK12 mutations and worse patient survival. CDK12 is a cyclin-dependent kinase, a key regulator of transcription elongation and termination. Deleting or suppressing CDK12 using CRISPR-Cas9 in prostate cell lines increased RNA:DNA hybrids (R-loops), promoted TRCs, and ultimately led to large TDs. Finally, we found that cells lacking CDK12 were sensitive to WEE1, CHK1 and ATR inhibitors. In summary, our data suggest that large TDs in cancer form due to impaired TRC repair and can be used as a biomarker for prognosis and treatment.



 Lixing Yang. A078: Transcription and DNA replication collisions lead to large tandem duplications and expose targetable therapeutic vulnerabilities in CDK12-mutant prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A078.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a771b9c6597f1781d70e2b6d5df9ff54196c4d85" target='_blank'>
              Abstract A078: Transcription and DNA replication collisions lead to large tandem duplications and expose targetable therapeutic vulnerabilities in CDK12-mutant prostate cancer
              </a>
            </td>
          <td>
            Lixing Yang
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4773775e6ecfd7cb45043c77dfe433bc00bd2ae5" target='_blank'>
              Epigenetic Intelligence: How Organisms Track Their Environment Through Molecular Memory
              </a>
            </td>
          <td>
            Holly V. Moeller, H. Putnam, Ross Cunning, Steven B. Roberts, J. Eirín-López, R. Nisbet
          </td>
          <td>2026-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Regulatory T cells (Tregs) are central mediators of immune tolerance, yet within tumors they adopt specialized phenotypes that confer the potent suppression of anti-tumor immune responses. Emerging evidence indicates that this functional plasticity is not driven by genetic alterations but instead arises from dynamic and context-dependent epigenetic reprogramming. While individual epigenetic mechanisms controlling Treg development and stability have been described, how tumor-derived cues reshape Treg epigenetic states, how these programs differ across cancer types, and which features distinguish tumor-infiltrating Tregs from their peripheral counterparts remain incompletely understood. In this review, we synthesize recent advances in DNA methylation, histone modifications, chromatin accessibility, and non-coding RNA regulation that govern Treg identity and function with a particular emphasis on tumor-specific epigenetic adaptations. We highlight emerging epigenetic hallmarks of intratumoral Tregs, discuss unresolved mechanistic questions, and evaluate the therapeutic potential and limitations of targeting epigenetic pathways to selectively modulate Tregs in cancer. By integrating mechanistic, cancer-specific, and translational perspectives, this review aims to provide a conceptual framework for understanding how epigenetic regulation shapes Treg behavior in the tumor microenvironment and how it may be exploited for cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db0543fbf7345f9e8f2d1c52296f9e40aff347be" target='_blank'>
              Unraveling the Epigenetic Regulation of Regulatory T Cells in Cancer Immunity
              </a>
            </td>
          <td>
            K. Subedi, N. Parajuli, X. Solone, Jeffrey Cruz, Sahil Kapur, Deyu Fang, Q-S Mi, Li Zhou
          </td>
          <td>2026-01-25</td>
          <td>Cells</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="

 IEIs are a heterogeneous group of primarily inherited genetic disorders characterized by impaired immune system function. While most genetic alterations are single nucleotide variants (SNVs), some cases are caused by larger genomic changes such as insertions or deletions that lead to copy number variations (CNVs). To identify potential large deletions in the WAS gene, we developed a molecular strategy combining a novel own design multiplex ligation-dependent probe amplification (MLPA) assay specific to WAS with a droplet digital PCR (ddPCR) assay.



 Three patients with suspected Wiskott–Aldrich syndrome (WAS) were studied. Initial analysis involved Sanger sequencing. When a CNV was suspected, additional methodologies were incorporated: a WAS-specific MLPA test (covering exons 1, 2, 3, 4, 5, 6, and 8) designed according to the manufacturer’s recommendations and a ddPCR assay targeting designed exons 7, 8, 9, and 12.



 None of the patients showed amplification of individual WAS exons by PCR. MLPA analysis revealed a complete absence of all exons in two patients, while the third patient showed absence of exon 8 by MLPA and absence of exons 7, 8, 9, and 12 by ddPCR. Both methodologies allowed for the analysis of women carriers of this X-linked hereditary pathology, leading to the identification of a carrier.



 These patients clearly demonstrate the importance of having more than one screening strategy for efficient diagnosis and family genetic counseling, mostly in cases where the clinical suspicion is very strong but conventional diagnostic strategies yield negative results. In many IEIs, bioinformatic algorithms in next-generation sequencing studies detect potential CNVs that need to be confirmed. Both MLPA and ddPCR are complementary techniques that would allow for a definitive diagnosis in these cases.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8ac509b654f89f2624ce5daf8a49bbf716d811e" target='_blank'>
              MLPA+Ddpcr: Dual Molecular Strategy to Detect Copy Number Variations (CNVs) in Inborn Error of Immunity (IEI) Genes: A Case Study of the WAS Gene
              </a>
            </td>
          <td>
            V. Goris, E. Prieto, Mariana Villa, M. Oleastro, J. Yancoski
          </td>
          <td>2025-12-22</td>
          <td>Journal of Human Immunity</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Background: The profound heterogeneity of glioblastoma and the often-limited efficacy of conventional treatments, including arsenic trioxide (ATO), underscore the urgent and critical demand for innovative combination strategies specifically designed to overcome treatment resistance. Methods: We evaluated the therapeutic effects of ATO as a single agent and in combination with the MNK1 inhibitor AUM001 across patient-derived xenograft (PDX) models and investigated molecular determinants of sensitivity and synergy. Our results demonstrated that GBM models resistant to ATO, particularly those of the mesenchymal subtype, are more likely to show synergistic cytotoxicity when AUM001 is added. The combination significantly reduces the frequency of glioblastoma stem cells (GSCs) compared to either drug alone, especially in ATO-resistant models. Results: These observations suggest that targeting the MNK1 pathway in conjunction with ATO is a promising strategy to specifically eradicate GSCs, which are major drivers of GBM recurrence and therapeutic failure. Transcriptomic analyses revealed that ATO sensitivity correlated with activated translation-related pathways and cell cycle processes, while synergistic responses to the combination were driven by distinct molecular signatures in different GBM subtypes. Overall, synergistic response to the combination therapy is more associated with cellular organization, amino acid transmembrane transporter activity, ion channels, extracellular matrix organization and collagen formation. Conclusions: Our findings highlight that specific molecular pathways and their activities, including those involving translation, cell cycle and ion transport, appear to modulate the synergistic efficacy of the ATO and AUM001 combination, thereby offering potential biomarkers for improved patient stratification in future GBM clinical trials of such ATO-based treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed3a011ab153f12fa9045338da4a46a0d67f877a" target='_blank'>
              Arsenic Trioxide and the MNK1 Inhibitor AUM001 Exert Synergistic Anti-Glioblastoma Effects by Modulating Key Translational, Cell Cycle, and Transmembrane Transport Pathways
              </a>
            </td>
          <td>
            Yue Hao, Charles Shaffer, N. Tang, Valerie DeLuca, Angela Baker, Michael E Berens
          </td>
          <td>2026-01-23</td>
          <td>Brain Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="CRISPR/Cas systems have transformed molecular medicine, yet the field still lacks principled guidance on when transient editing suffices versus when sustained exposure through in vivo viral delivery is necessary and how to keep prolonged exposure safe. Notably, EDIT-101 was designed for a permanent edit in post-mitotic photoreceptors with lifelong Cas9 persistence. This review addresses this gap by defining the biological and therapeutic conditions that drive benefit from extended Cas activity while minimizing risk. We will (i) examine relationships between expression window and efficacy across Cas9/Cas12/Cas13 modalities, (ii) identify genome-wide off-target liabilities alongside orthogonal assays, and (iii) discuss controllable, self-limiting, and recallable editor platforms. By separating durable edits from persistent nuclease exposure, and by providing validated control levers, this work establishes a generalizable framework for safe, higher-efficacy CRISPR medicines. Furthermore, we highlight key studies in cell lines, murine models, non-human primates, and humans that examine the long-term effects of sustained expression of CRISPR/Cas systems and discuss the safety and efficacy of such approaches. Current evidence demonstrates promising therapeutic outcomes with manageable safety profiles, although there is a need for continued monitoring as CRISPR/Cas therapies are increasingly applied in clinical contexts and therapies are developed for broader clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93ffced530e1e86e12ebc967a34622935497f148" target='_blank'>
              Chronic In Vivo CRISPR-Cas Genome Editing: Challenges, Long-Term Safety, and Outlook
              </a>
            </td>
          <td>
            Caroline Bao, Catherine I. Channell, Yi Hsuan Tseng, J. Bailey, Naeem Sbaiti, A. Demirkol, Stephen H. Tsang
          </td>
          <td>2026-01-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Lung cancer persists as the leading cause of cancer-related mortality globally, largely due to late detection, genomic complexity, and limited durability of existing therapeutic interventions. The integration of CRISPR/Cas9 gene-editing systems with biomarker-driven therapeutic strategies represents a transformative advance in precision oncology. Biomarkers—including genetic mutations, epigenetic alterations, protein signatures, and circulating analytes—enable early detection, patient stratification, and dynamic monitoring of therapeutic response. CRISPR/Cas9 offers a unique opportunity to directly reprogram these biomarkers or the pathways regulating them, enhancing tumor immunogenicity, reversing immune evasion mechanisms, and strengthening anti-tumor immune responses. Preclinical models demonstrate that CRISPR-mediated biomarker editing can restore antigen presentation, augment T-cell cytotoxicity, sensitize resistant tumors to immunotherapy, and improve tumor regression. Early clinical trials further validate the feasibility and safety of CRISPR-engineered immune cells in patients. 
However, major challenges persist, including off-target editing, inefficient delivery to solid tumors, tumor microenvironment–mediated suppression, and ethical considerations linked to genome manipulation. Rapid advancements in editing fidelity, lipid nanoparticle systems, viral vectors, engineered vesicles, high-throughput biomarker discovery, and artificial intelligence–assisted CRISPR design are expected to accelerate clinical translation. This review synthesizes the current landscape, mechanistic underpinnings, emerging applications, and future directions of CRISPR/Cas9-enabled biomarker engineering for lung cancer immunotherapy, positioning this technology as a cornerstone of next-generation personalized oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d80a98d71ca868cfed559df1a6401f5e2e8b0702" target='_blank'>
              Precision Reprogramming of Lung Cancer Biomarkers via CRISPR/Cas9: A Paradigm Shift in Personalized Immunotherapy
              </a>
            </td>
          <td>
            M. R. Jameel, Amar Arora, Prem Shankar Mishra, Hina Jameel, M. Makhdoomi, Abid Ali Sheikh
          </td>
          <td>2025-12-30</td>
          <td>Journal of Analytical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="CRISPR-based therapeutics represent a transformative advancement in molecular medicine by enabling precise, programmable modification of genomic sequences for the treatment of genetic and acquired diseases. Unlike conventional pharmacological interventions that primarily modulate protein function, CRISPR technologies directly target the underlying genetic determinants of pathology, offering the potential for durable or permanent therapeutic effects. This paper examines the molecular mechanisms of CRISPR-mediated gene-targeted drug action, focusing on DNA recognition, cleavage, repair pathways, and emerging gene-editing modalities such as base editing and prime editing. By synthesizing developments in genome engineering, delivery systems, and off-target mitigation strategies, the study evaluates the clinical promise and technical limitations of CRISPR-based interventions. The findings underscore the growing role of gene-editing therapeutics in precision medicine while highlighting ethical, regulatory, and safety considerations that accompany their translational deployment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/601d94e7bfe27f0434bb1469e7b781cc59019561" target='_blank'>
              CRISPR-Based Therapeutics: Molecular Mechanisms of Gene-Targeted Drug Action
              </a>
            </td>
          <td>
            Kaan Karakas
          </td>
          <td>2026-02-07</td>
          <td>Next Frontier For Life Sciences and AI</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b89d2858599b4236f22769f3a4af65ec9fd4eb7" target='_blank'>
              Deciphering a common code: Unitary gene profile factor enabling Spontaneous Tumor Regression
              </a>
            </td>
          <td>
            Arushi Misra, B. Kumari, Prasun K. Roy
          </td>
          <td>2025-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Non-AIDS-defining cancers (NADCs) have emerged as an increasingly prominent cause of non-AIDS-related morbidity and mortality among people living with HIV (PLWH). However, the scarcity of NADC clinical samples, compounded by privacy and security constraints, continues to present formidable obstacles to advancing pathological and clinical investigations. In this study, we adopted a joint analysis strategy and deeply integrated and analyzed transcriptomic data from 12,486 PLWH and cancer patients to systematically identify potential key regulators for 23 NADCs. This effort culminated in NADCdb—a database specifically engineered for NADC pathological exploration, structured around three mechanistic frameworks rooted in the interplay of immunosuppression, chronic inflammation, carcinogenic viral infections, and HIV-derived oncogenic pathways. The “rNADC” module performed risk assessment by prioritizing genes with aberrant expression trajectories, deploying bidirectional stepwise regression coupled with logistic modeling to stratify the risks for 21 NADCs. The “dNADC” module, synergized patients’ dysregulated genes with their regulatory networks, using Random Forest (RF) and Conditional Inference Trees (CITs) to identify pathogenic drivers of NADCs, with an accuracy exceeding 75% (in the external validation cohort, the prediction accuracy of the HIV-associated clear cell renal cell carcinoma model exceeded 90%). Meanwhile, “iPredict” identified 1905 key immune biomarkers for 16 NADCs based on the distinct immune statuses of patients. Importantly, we conducted multi-dimensional profiling of these key determinants, including in-depth functional annotations, phenotype correlations, protein–protein interaction (PPI) networks, TF-miRNA-target regulatory networks, and drug prediction, to deeply dissect their mechanistic roles in NADC pathogenesis. In summary, NADCdb serves as a novel, centralized resource that integrates data and provides analytical frameworks, offering fresh perspectives and a valuable platform for the scientific exploration of NADCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/066c9074f988825c38d9ba8748ab7561dd1f4d89" target='_blank'>
              NADCdb: A Joint Transcriptomic Database for Non-AIDS-Defining Cancer Research in HIV-Positive Individuals
              </a>
            </td>
          <td>
            Jiajia Xuan, Chunhua Xiao, Runhao Luo, Yonglei Luo, Qing-yu He, Wanting Liu
          </td>
          <td>2026-01-23</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Oncofetal reprogramming has recently emerged as a critical concept in translational cancer research, particularly for its role in driving therapeutic resistance across a variety of malignancies. This biological process refers to a pattern of gene expression that is restricted to embryogenesis, but becomes expressed again in a subpopulation of cancer cells. These genes are typically suppressed after embryogenesis, and their aberrant re-expression in tumors endows cancer cells with stem-like properties and enhanced adaptability. The goal of this review is the following: (i) comprehensively examine the multifaceted nature of oncofetal reprogramming; (ii) elucidate its underlying molecular mechanisms, including its regulators and effectors; and (iii) evaluate its consequences for the therapeutic response in different cancer types. We comprehensively integrate the latest findings from colorectal, breast, lung, liver, and other cancers to provide a detailed understanding of how oncofetal programs interfere with tumor response to treatment. Among the candidates, YAP1 and AP-1 have emerged as central transcriptional drivers of this reprogramming process, especially in colorectal and breast cancers. We also explore the distinct expression patterns of oncofetal genes across different tumor types and how these patterns correlate with treatment outcomes and patient survival. Lastly, we propose a dual-targeting therapeutic strategy that simultaneously targets both cancer stem cells and oncofetal-reprogrammed populations as a more effective approach to overcome resistance and limit recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a262b7681553fb103b5d82c8b95c6fdfada4bf5" target='_blank'>
              Oncofetal Reprogramming: A New Frontier in Cancer Therapy Resistance
              </a>
            </td>
          <td>
            Anh L Nguyen, Molly Lausten, Bruce M. Boman
          </td>
          <td>2026-01-29</td>
          <td>International Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/719b7b9d009213c7ffc5a98ac8b16a699f1f4455" target='_blank'>
              Anchored random reverse primer sequencing for quantitative detection of novel gene fusions.
              </a>
            </td>
          <td>
            Xuehao Xiu, Yi Wu, Jiangxue Li, Dongfa Lin, Xiao Sun, Xinglei Su, Zhi Weng, Xiaolei Zuo, Xiurong Yang, Chunhai Fan, Yudong Wang, David Yu Zhang, Ping Song
          </td>
          <td>2026-01-02</td>
          <td>Nature biomedical engineering</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="


 Clonal hematopoiesis (CH) is exemplified by the expansion of genomically altered hematopoietic stem and progenitor cells that may give rise to altered immune repertoires. Driven largely by age and chronic inflammation, CH has been historically linked to myeloid malignancies. Although emerging evidence points to a modest prevalence of CH within solid tumors, the impact of these naturally occurring alterations on anti-tumor immunity and therapeutic responses remains less defined. Here, we employed somatic mutation calling algorithms to detect single nucleotide variants (SNVs) in both epithelial and immune compartments within tumors. Our preliminary analyses of single-cell RNA-sequencing (scRNA-seq) of 32 primary, therapy-naïve prostate tumors encompassing 11 patients identified conventional mutations representative of both CH and prostate cancer. Within the tumor-epithelial compartment, we identified recurrent FOXA1 class 1C mutations in 27% (3/11), alongside NCOR1 mutations in 18% (2/11), IDH1 in 9% (1/11), and SPOP in 9% (1/11) of patients. Of note, these SNVs were detected across multiple samples for the same patient. Interestingly, we also detected germline HOXB13 G84E variants, associated with increased risk for prostate cancer, in 2 cases at a high clonality within epithelial cells (>75%). CH alterations were identified in 27% (3/11) of patients based on the detection of exonic, nonsynonymous SNVs within myeloid cells. Specifically, these variants affected SF3B2 (variant allele frequency; VAF = 0.30), UBE2A (VAF = 0.30), and KMT2C (VAF = 0.43) that are predicted deleterious across multiple classifier algorithms. Patients with detectable CH alterations displayed immunologically hotter tumor microenvironments, with CH+ patients having significantly higher proportions of tumor-infiltrating monocytes (16.22% vs 7.13%, Fisher’s exact test p<0.0001), macrophages (6.49% vs 2.88%, p<0.0001), CD8+ T cells (10.35% vs 3.33%, p<0.0001), CD4+ T cells (2.81% vs 0.74%, p<0.0001), and B cells (1.81% vs 0.59%, p<0.0001), and lower fractions of epithelial cells (50.94% vs 69.81%, p<0.0001). In summary, we leveraged de novo mutational algorithms to identify somatic alterations using scRNA-seq. To our knowledge, our study is the first to detect CH alterations in prostate cancer at the single-cell resolution. Further work will integrate our mutational signatures with transcriptomics to elucidate the impact of both cancer- and immune-intrinsic alterations on anti-tumor immunity. Additionally, we will incorporate mitochondrial DNA-based lineage tracing algorithms on matched peripheral blood, primary, and metastatic tumor samples to deconvolute the spatiotemporal regulation of naturally occurring CH alterations across cancer progression. We envision our findings will illuminate novel drivers of immune function and regulation within prostate cancer that may be exploited for future immunomodulatory therapies.



 John R. Lozada, Christine Luo, Devin A. Schmeck, Jeffrey S. Miller, Akash Patnaik, Emmanuel S. Antonarakis, Frank Cichocki, Justin H. Hwang. Interrogating the Genomic Co-Evolution of the Immune System and Prostate Cancer Along the Time and Treatment Continuum [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A040.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03587209005d8227c89ae44e47e8117ddd350a30" target='_blank'>
              Abstract A040: Interrogating the Genomic Co-Evolution of the Immune System and Prostate Cancer Along the Time and Treatment Continuum
              </a>
            </td>
          <td>
            John R Lozada, Christine Luo, Devin A. Schmeck, Jeffrey S. Miller, Akash Patnaik, Emmanuel S. Antonarakis, Frank Cichocki, Justin H. Hwang
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Pediatric leukemia, primarily comprising acute lymphoblastic leukemia (ALL) and acute myeloid leukemia, represents approximately 30% of childhood cancers worldwide. Over the past decade, advances in molecular and genetic profiling, including next-generation sequencing and minimal residual disease monitoring, have refined risk stratification and enabled individualized treatment strategies. Integrating these approaches into clinical practice has improved survival rates, particularly for high-risk patients, by guiding therapy intensity and informing targeted interventions. Targeted therapies, such as tyrosine kinase inhibitors for Philadelphia chromosome-positive ALL, Chimeric Antigen Receptor T cell therapy for relapsed or refractory cases, and monoclonal antibodies like blinatumomab and inotuzumab ozogamicin, have transformed treatment outcomes while reducing chemotherapy-related toxicity. Despite these advances, challenges remain, including the development of therapy resistance (e.g., BCR-ABL1 and FLT3 mutations) and long-term adverse effects such as cardiotoxicity and secondary malignancies. This narrative review summarizes recent innovations in risk assessment and targeted therapies, highlights current challenges, and discusses future directions to optimize personalized pediatric leukemia care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e88e19df34880ed4268d38202fdb59c43c3fe8e" target='_blank'>
              Redefining pediatric leukemia care – innovations in risk assessment and targeted treatment: a narrative review
              </a>
            </td>
          <td>
            E. Obeagu
          </td>
          <td>2026-01-21</td>
          <td>Annals of Medicine and Surgery</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Precancers, defined as normal-appearing or morphologically altered tissues with a risk of oncogenesis, exhibit various detectable manifestations across anatomical sites, including epithelial dysplasia, metaplasia, hyperplasia, and stromal fibrosis. Considering the prevailing assumption that most cancers arise from precancers, early intervention at the precancerous stage has immense potential to reduce cancer-related morbidity and mortality. However, the complex signaling networks governing precancer initiation and progression remain elusive, hampering the development of effective targeted interventions. This review synthesizes three critical dimensions of precancer biology: historical foundations tracing the conceptual evolution of precancer research over the past century; mechanisms underlying the multistep progression of precancer biology, encompassing epithelial and macro/microenvironmental remodeling; and signaling networks cataloging dysregulated pathways and their therapeutic potential. Over 10 signaling pathways, including the transforming growth factor-β (TGF-β), p53, Wnt, phosphatidylinositol 3-kinase (PI3K), and mitogen-activated protein kinase (MAPK) pathways, drive multistep malignant transformation. We further synthesize emerging evidence supporting microenvironmental dominance, proposing the novel “soil degeneration” hypothesis. This paradigm shift underscores the necessity for dual-window intervention in which early-phase microenvironmental normalization prevents the establishment of precancerous lesions and advanced-phase treatment concurrently addresses epithelial malignancy and stromal degeneration. This review bridges foundational molecular discoveries with translational clinical potential and advocates for precision intervention frameworks that extend from biomarker-guided risk assessment to synergistic remodeling of the precancer microenvironment, thereby redefining precancer intervention in the molecularly targeted era.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67dd61437ebeffa1b547c7834fa24572f4d7328c" target='_blank'>
              Signaling pathways and targeted interventions for precancers
              </a>
            </td>
          <td>
            Jin Yang, Shimeng Wang, Xin Li, Hongdan Xu, Tongxu Sun, Tao Hu, Jingjing Luo, Hongmei Zhou
          </td>
          <td>2026-01-07</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/423b9ad510934f726aebaeeb465a2ff10426e96b" target='_blank'>
              A new cancer progression model: from synthetic tumors to real data and back
              </a>
            </td>
          <td>
            Daniela Volpatto, S. G. Contaldo, Simone Pernice, Marco Beccuti, Francesca Cordero, R. Sirovich
          </td>
          <td>2026-02-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="This paper delves into the role of epigenetics in complementing and challenging Darwinian evolution theory within the field of evolution. Epigenetics has revealed new sources of genetic variation, with epigenetic variations such as DNA methylation and histone modifications capable of influencing gene expression without altering DNA sequences. These variations can spread and accumulate within populations, exhibiting reversibility and dynamism, enabling species to rapidly adapt to environmental changes. Additionally, epigenetics expands the scope of natural selection, revises the patterns of genetic information flow, and highlights the need for updated evolutionary predictive models. Researchers have proposed theoretical integration pathways such as the Extended Evolutionary Synthesis framework, multilevel selection theory, and gene-epigenome coevolution models, offering new perspectives and directions for evolutionary biology research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b730d3e4497777825de2252ebd00338b352de17" target='_blank'>
              Epigenetics Complements and Challenges Darwinian Evolution: From Gene Regulation to the Study of Species Adaptation
              </a>
            </td>
          <td>
            Meichen Zhou
          </td>
          <td>2025-12-19</td>
          <td>MedScien</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Drosophila melanogaster, a well-established model organism in biomedical research, has significantly advanced our understanding of genetic mechanisms underlying human diseases. Its genetic similarity to humans, with approximately 75% of human disease-related genes having orthologues in the fly genome, renders it a valuable tool for studying complex conditions like cancer. The organism’s low cost, short life cycle, and manipulable genome make it ideal for high-throughput studies. Recent advances have enabled the use of Drosophila as a model for different cancer types, including colorectal, thyroid, lung, and brain cancers. These models replicate key tumor characteristics such as cell proliferation, invasion, and metastasis. Drosophila’s utility in drug screening has been exemplified through heterochromatin-promoting assays and personalized medicine approaches using patient-specific gene expression data. Its role in testing therapeutic combinations, such as MEK inhibitors with statins for lung cancer, underscores its translational potential. Despite its utility, the model has notable limitations. Differences in tumor biology between flies and humans, especially regarding organ systems and immune responses, limit its applicability in certain cancer types and metastatic studies. Furthermore, environmental influences and the simplified genetic background of Drosophila may not accurately represent human cancer complexity. Ethical concerns and translational challenges also exist, warranting cautious interpretation of results. The recent advancements in CRISPR-based genome editing, particularly cytosine and adenine base editors, have improved gene-editing efficiency in Drosophila, supporting its ongoing use in precision oncology. Future directions point toward integrating Drosophila models with other in vivo and in silico systems to optimize personalized drug discovery and better emulate human tumor heterogeneity. While the use of Drosophila melanogaster has intrinsic limitations, its genetic tractability and compatibility with cutting-edge genomic tools position it as a complementary model to mammalian systems. Continued innovations in genetic engineering and model integration will further strengthen its role in translational oncology and precision medicine. This review explores the applications of the model in human cancer research and highlights the major challenges with its use. This is crucial for the fight against cancer, as it helps inform researchers of the potential areas that require further research, especially with the rapid evolution of personalised medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b860b652fc843a37123c4a5f30311be7939a37a1" target='_blank'>
              Applications and challenges of Drosophila melanogaster as a laboratory model in human cancer research: a narrative review
              </a>
            </td>
          <td>
            R. Abdullateef, Jonas Paul Ibekwe, Henry Oyoyo, Samuel Ogbodo
          </td>
          <td>2025-12-19</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer is a heterogeneous disease at the cellular level and analyzing the genetic and molecular profile is essential for targeted therapy. Cancer cells continue to mutate, often resulting in drug resistance. In addition, cancers such as triple-negative breast cancer (TNBC) lack the target proteins used in some of the most effective therapies. This necessitates the identification of novel target proteins and biomarkers for effective treatment strategies. Ubiquitin E3 ligases are often differentially expressed in cancer cells, and numerous anticancer agents have been developed to inhibit them. SMURF2 is an E3 ligase that is differentially expressed in multiple cancer types. Although inhibiting upregulated SMURF2 may be strategically straightforward, enhancing the downregulated gene is often difficult. In addition, because E3 ligases ubiquitinate a variety of substrate proteins, targeting SMURF2 requires detailed analysis to achieve anticancer effect. This review discusses the dual role of SMURF2 in carcinogenesis and addresses the complex context-dependent function of SMURF2 in the various cellular pathways. In addition, resistance to existing cancer therapy related to SMURF2 and sensitivity mechanisms is discussed. Lastly, theranostic strategies for anticancer agents and biomarker development are suggested.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85f6a4efa5a333f3bf6aa2ee9217bfa27f7bea38" target='_blank'>
              SMURF2 in Anticancer Therapy: Dual Role in Carcinogenesis and Theranostics
              </a>
            </td>
          <td>
            Joy Eom, Yejin Chun, Hae Ryung Chang
          </td>
          <td>2026-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Ewing sarcoma is a pediatric bone and soft tissue cancer thought to arise from mesenchymal stem cells (MSCs). It is characterized by fusion of the RNA-binding protein EWS to an ETS family transcription factor, most often FLI1. EWS::FLI1 induces changes in gene expression that have been well-studied and have been linked to its ability to bind to and alter chromatin state at GGAA repeat sequences within the genome. In addition to these effects, EWS::FLI1 alters mRNA splicing; however, the mechanism of this activity remains unclear.



 We curated a sarcoma-focused transcriptome from published long-read RNA sequencing of Ewing sarcoma, osteosarcoma, and synovial sarcoma patient tumors and matched adjacent normal tissue (215M reads). We supplemented this with published long-read RNA sequencing data from relevant normal cell types, osteosarcoma cell lines, and Ewing sarcoma lines with or without EWS::FLI1 knockdown (105M reads). We used this transcriptome as a reference to evaluate expression and splicing from short-read RNA-seq of Ewing sarcoma cell lines, MSCs, and MSC-based models of Ewing sarcoma and established Ewing-associated expression and splicing signatures.



 We generated scores based on these signatures in published knockdown-rescue experiments in which endogenous EWS::FLI1 was silenced by shRNA and replaced with exogenous shRNA-resistant mutants. Expression and splicing scores were highly correlated within these datasets (r2 = 0.89), suggesting that similar regions of the fusion protein might be required for both its functions. In contrast, the scores had limited correlation within an internal dataset of RNA-binding protein perturbations (r2 = 0.02). We next examined the effects of pharmacological agents thought to alter EWS::FLI1 activity. Interestingly, we found that lurbinectedin which disrupts EWS::FLI1’s DNA-binding could partially revert both expression and splicing scores. In contrast, YK-4-279 which disrupts the fusion’s protein-protein interactions and MS0621 which reverts a Ewing-associated chromatin signature had limited effects on these scores. To further interrogate the potential role of EWS::FLI1 DNA-binding in splicing regulation, we silenced EWS::FLI1 in TC-32 cells and replaced it with either the wildtype fusion or a DNA-binding null mutant. Unlike the wildtype construct, the DNA-binding null mutant could not rescue expression or splicing. Finally, we examined the effects of the previously published KRAB-ZFA7 repressor which silences chromatin at GGAA repeats while sparing single GGAA sites and found that it could revert both scores.



 Together, these data establish a potential connection between EWS::FLI1 activity at GGAA repeats and its effects on mRNA splicing. Additionally, we find that lurbinectedin, an FDA-approved agent currently in a phase 1/2 trial for Ewing sarcoma, suppresses both the splicing and expression effects of EWS::FLI1. Emerging compounds designed to target EWS::FLI1 DNA-binding directly will likely have similar effects.



 David V. Allegakoen, Jeffrey A. Toretsky. EWS::FLI1 activity at GGAA repeats is essential for maintenance of Ewing-like splicing [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr B013.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6934f37ed326d496d296a4806fccea6f323a4f63" target='_blank'>
              Abstract B013: EWS::FLI1 activity at GGAA repeats is essential for maintenance of Ewing-like splicing
              </a>
            </td>
          <td>
            D. Allegakoen, J. Toretsky
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Background: RASAL1 is a prominent tumor suppressor gene through inactivating RAS, whose functional loss has been widely found to play an important role in thyroid cancer and occurs usually through its genetic and epigenetic inactivation. In this study, we intended to explore aberrant alternative splicing (AS) as an important novel mechanism for the oncogenic inactivation of RASAL1 in thyroid cancer. Methods: We used comprehensive bioinformatic and molecular experimental approaches to identify and characterize aberrant alternative splice variants of RASAL1. This included structural and functional investigation of the potentially oncogenic splice variants of RASAL1, with a focus on exploring the molecular mechanisms and clinical impacts on thyroid cancer. Results: We identified an aberrant alternative splice variant of RASAL1, known as RASAL1-004, that commonly compromised the function of RASAL1 in thyroid cancer. Specifically, we found common skipping of exon 18.1 in RASAL1, leading to the abundant formation of transcript RASAL1-004 in cancer, which was significantly associated with poor survival of patients with thyroid cancer. Mechanistically, Argonaute2 regulates exon 18.1 splicing by binding to the response element in exon 17 containing CCAGCC motif, promoting RASAL1-004 formation. The exon 18.1 skipping caused a conformational change in the RNA structure of RASAL1-004 at the junction of exons 17 and 18, resulting in ribosome stalling, halting RASAL1 translation. This reduced RAS GTPase-activating-like protein 1 (RASAL1 protein) synthesis, consequently leading to the functional loss of RASAL1. Compared with RASAL1-001, the canonical wild-type RASAL1 transcript, the absence of exon 18.1 in RASAL1-004 also conformationally altered the pleckstrin homology domain of RASAL1 protein, which, as we demonstrated, led to the loss of the ability of RASAL1 to localize with cell membrane, thereby impairing its RAS-inactivating function. We further demonstrated that compared with RASAL1-001, RASAL1-004 displayed impaired RAS-signaling pathway-suppressing and cancer cell-suppressing functions. Conclusions: This study identified a novel RASAL1-impairing mechanism, alternative to the classically known genetic and epigenetic mechanisms, for the inactivation of the tumor suppressor gene RASAL1 through aberrant AS to form RASAL1-004 with impaired protein translation. This represents a new oncogenic mechanism in thyroid cancer, with novel cancer biological, prognostic, and therapeutic-targeting implications in thyroid cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5cb95b9e95603f8cd6fd23f4253e107361cda23d" target='_blank'>
              Functional Loss of the Tumor Suppressor Gene RASAL1 Through Formation of Aberrant Splice Variant RASAL1-004 as a Novel Oncogenic Mechanism in Thyroid Cancer
              </a>
            </td>
          <td>
            Jing Liao, Junwei Song, Zhuo Wang, Mingzhao Xing
          </td>
          <td>2025-12-22</td>
          <td>Thyroid</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Immunotherapies have transformed cancer treatment by harnessing the immune system to recognize and eliminate malignant cells, offering durable clinical benefit across diverse tumor types. Despite successes with immune checkpoint inhibitors (ICIs) and other strategies like cytokines, oncolytic viruses, cancer vaccines, bispecific antibodies, and adoptive cell therapies, substantial fractions of patients still fail to respond or develop resistance. The oncogene MYC, deregulated in ~70% of human cancers, has emerged as a central driver of immune evasion and a key contributor to immunotherapy failure. MYC regulates broad transcriptional networks controlling proliferation, metabolism, angiogenesis, and cell survival, while also orchestrating profound remodeling of the tumor microenvironment (TME). Mechanistically, oncogenic MYC suppresses antigen processing and presentation, attenuates interferon signaling, and upregulates immune checkpoints such as PD-L1 and CD47. Concurrently, MYC stimulates secretion of immunosuppressive cytokines and chemokines that recruit regulatory T cells, myeloid-derived suppressor cells, and M2 macrophages, while driving metabolic reprogramming that fosters nutrient competition, hypoxia, and acidosis, impairing effector T- and NK-cell function. Through these pathways, MYC promotes primary, adaptive, and acquired resistance to immunotherapy. Targeting MYC, directly or indirectly, holds promise to restore immune surveillance and potentiate immunotherapeutic efficacy. This review highlights MYC as a master regulator of tumor–immune interactions and underscores the therapeutic potential of MYC inhibition to overcome resistance and expand the clinical impact of cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/437ab7ab32dc49430fcd2b151a4005d089060a97" target='_blank'>
              MYC at the tumor–immune interface: mechanisms of immune escape and immunotherapy resistance
              </a>
            </td>
          <td>
            Íñigo González-Larreategui, Manrique Valdés-Bango Martín, S. Casacuberta-Serra, Laura Soucek
          </td>
          <td>2026-01-23</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The advent of next-generation high-throughput sequencing and precision medicine has allowed the identification of recurrent genetic events of vital interest for leukemic cells and for the management of patients with leukemia, since it has a direct impact on prognosis and therapeutic stratification. The precise cytogenetic characterization of these diseases has rapidly enabled the development of innovative therapeutic strategies to improve outcomes and limit toxicities. New targeted therapies, including small molecule inhibitors, monoclonal antibodies, bispecific T cell engagers, antibody–drug conjugates, recombinant immunotoxins, and chimeric antigen receptor T cells (CAR-T cells) represent a broad therapeutic arsenal now available in everyday practice. The growing number of treatments means that many possibilities of drugs and combinations can be used to offer new lines of therapy in cases of refractory disease or relapse, but also in frontline treatment. This review provides a broad overview of pathogenic mechanisms and therapies (including targeted therapies, immunotherapy and RNA-based therapy) and gathers the main research progress in the field of leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9ad84d56b4eca59a338789eb4fc5c1bd3ecb571" target='_blank'>
              Molecularly targeted therapy and immunotherapy in leukemias
              </a>
            </td>
          <td>
            Elie Cousin, Agathe Picard, Maria Oundjian, Nina Tardif, Anaïs M. Quéméner, Anne-Gaëlle Rio, M. Galibert, Virginie Gandemer, Frédéric Mazurier
          </td>
          <td>2026-01-11</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/094fb602aadcf523532bb1ca21592ff9ebaa966e" target='_blank'>
              Follicular Helper T Cells and B Cell Maturation in Patients with 22q11.2 Deletion Syndrome and Recurrent Infections.
              </a>
            </td>
          <td>
            N. Alsaati, K. Beigel, K. Maurer, Sarah E. Henrickson, Montana S. Knight, Audrey Green, V. Giunta, D. McGinn, Bekah Wang, T. B. Crowley, Donna M. McDonald-McGinn, Kathleen E. Sullivan
          </td>
          <td>2026-02-03</td>
          <td>Journal of clinical immunology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) represent a highly plastic subpopulation within tumors that drives tumor initiation, progression, metastasis, therapeutic resistance, and disease relapse. Although CSC traits were initially attributed to stable genetic hierarchies, growing evidence suggests that cancer stemness is a dynamic and reversible state regulated by precise molecular events at the protein level. This literature review aims to synthesize experimental evidence regarding the impact of single amino acid mutations and alterations in regulatory proteins on the stemness properties of cancer cells. A systematic literature search was conducted using PubMed and Google Scholar to identify original research articles published between 2020 and 2025 that examined protein-level mutations, post-translational modifications, epigenetic regulators, and chromatin remodeling factors in diverse cancer models. The reviewed studies consistently demonstrate that single amino acid substitutions in oncogenic, tumor suppressor, or regulatory proteins can profoundly affect protein conformation, stability, enzymatic activity, and protein–protein interactions. These alterations frequently result in sustained activation of key stemness-associated signaling pathways, including Wnt/β-catenin, Notch, Hedgehog, and STAT3, thereby enhancing self-renewal capacity, stem cell marker expression, tumor-initiating potential, and resistance to conventional therapies. In parallel, dysregulation of post-translational modification systems, such as phosphorylation, ubiquitination, acetylation, and methylation, promotes epigenetic reprogramming and reactivation of embryonic transcriptional programs that favor cellular plasticity. Moreover, protein-level alterations strengthen cancer cell interactions with the tumor microenvironment, increasing responsiveness to hypoxia, inflammatory signals, and metabolic stress. Collectively, these findings highlight protein-level regulation as a central determinant of cancer stem cell maintenance and plasticity, and suggest that effective therapeutic strategies should target aberrant protein stability, post-translational modification machinery, and stemness-related signaling networks to achieve durable clinical responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f961c9b291a85047c8455458a035edcd22afcac" target='_blank'>
              IMPACT OF SINGLE AMINO ACID MUTATIONS OR ALTERATIONS IN REGULATORY PROTEINS ON THE STEMNESS PROPERTIES OF CANCER CELLS
              </a>
            </td>
          <td>
            Dewi Sartika, Irmatul Laili, Sri Utami
          </td>
          <td>2025-12-30</td>
          <td>Journal of Stem Cell Research and Tissue Engineering</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="MicroRNAs (miRNAs) are tiny, non-coding RNA molecules that are pivotal in regulating cell growth, development, cell cycle, and programmed cell death. Alterations in miRNA expression, caused by various mechanisms such as amplification, deletion, mutation, or epigenetic changes, can lead to either an increase or decrease in their levels in cancerous cells. Since their initial discovery in 1993, a wealth of research has established miRNAs as key players in oncogenesis and tumor suppression across various human cancers. The study of miRNAs is critical for predicting cancer outcomes and aiding in its detection. Numerous miRNAs have demonstrated potential as markers for cancer prognosis and diagnosis, as well as targets for therapy. Their profound influence on tumor development and spread is becoming increasingly acknowledged. This article presents a summary of how miRNAs are synthesized and generated, and explores their dual function as tumor suppressors and oncogenes. Additionally, we delve into the promising applications of miRNAs as instruments for cancer diagnosis">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/874dc41eef7a91dbf277f54075fa7796a49bfc9e" target='_blank'>
              MicroRNA Regulation in Cancer: Insights into Diagnosis, Prognosis, and Treatment
              </a>
            </td>
          <td>
            Ibtihaj Alrehaili, Norah Alrashoud, Rawaeb Almutairi, Lama Alqahtani, Nour Alhatm, A. Almutairi, Asma Alghamdi, Azhar Barnawi, Kharsan Alyami, H. Alghamdi, Jamilah Algasem, Reham Alshahrani, Lubna Aljaddua, M. Alahmari
          </td>
          <td>2026-02-04</td>
          <td>Pharmacy Practice</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f6c40c7ea1ba98d656117fae44f7bccd796a048" target='_blank'>
              Incomplete developmental silencing of cancer-testis antigen BORIS in humanized mouse model promotes cancer susceptibility
              </a>
            </td>
          <td>
            Emma Price, E. Pugacheva, D. N. Bhatt, Yon Ji, Sheila Yeboah, Arielle Scott, D. Loukinov, Victor V. Lobanenkov
          </td>
          <td>2025-12-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d21384971b0ced62ebc3c05bc7a513b76dd2fb8" target='_blank'>
              Five dominant amino acid substitution signatures shape tumour immunity.
              </a>
            </td>
          <td>
            Szilvia Juhász, B. Papp, Anna Tácia Fülöp, Z. Farkas, Dávid Kókai, Dóra Alexandra Gyémánt, Franciska Tóth, Zsófia Nacsa, Dóra Spekhardt, Balázs Koncz, Péter Burkovics, Csaba Pál, Máté Manczinger
          </td>
          <td>2026-01-28</td>
          <td>Molecular systems biology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9b6108bbfc8477081a30cd15de6db71d5ddd4a1" target='_blank'>
              Exploiting synthetic lethality in epithelial ovarian cancer: multi-dimensional approaches beyond DNA damage repair.
              </a>
            </td>
          <td>
            Mandana Bigdeli, Elizabeth Tremblay, Diane Provencher, A. Mes-Masson, Francis Rodier
          </td>
          <td>2026-02-10</td>
          <td>Molecular cancer</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Background: A significant challenge in oncology is the development of therapies that selectively eliminate cancer cells while engaging the immune system for durable protection. Oncogene-driven transformation has been shown to rewire cellular metabolism and organelle function, creating unique dependencies. Lysosomes, traditionally considered mere degradative centers, are emerging as critical regulators of cell survival and death. We hypothesized that oncogenic transformation, particularly by drivers like Her2, creates a lysosomal "Achilles' heel," priming these organelles for permeabilization and offering a novel therapeutic target. Methods: Using the MCF-10A-neoT model of oncogene-driven transformation, we first characterized lysosomal alterations. We then conducted a high-throughput screen for compounds inducing Lysosomal Membrane Permeabilization (LMP), focusing on Reactive Oxygen Species (ROS) inducers. Lead compounds were evaluated for their ability to trigger established markers of Immunogenic Cell Death (ICD), including surface calreticulin (CRT), ATP release, and HMGB1 release. Selectivity and efficacy were further validated in co-culture models and immunocompetent mouse models. Results: Oncogene-transformed cells exhibited significantly enlarged lysosomes, altered subcellular distribution, and increased cathepsin B/L activity compared to their non-transformed counterparts. Our screen identified several ROS-generating compounds that selectively induced LMP in cancer cells. Treatment with these lead compounds resulted in the robust exposure of CRT, and the extracellular release of ATP and HMGB1, confirming the induction of ICD. In co-culture, the compounds demonstrated a clear therapeutic window, preferentially killing cancer cells. In vivo studies in immunocompetent mice confirmed tumor regression and the establishment of a protective anti-tumor immune memory. Conclusion: Our findings validate that targeted LMP induction is a viable strategy for selectively killing oncogene-addicted cancer cells. Moreover, by triggering ICD, this approach not only directly eliminates tumor cells but also stimulates a systemic immune response, addressing the critical need for therapies that provide long-term anti-cancer immunity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cde312e8a563ef0269b386c82e442a967bac8f91" target='_blank'>
              Exploiting Lysosomal Vulnerabilities: A Novel Therapeutic Strategy to Induce Immunogenic Cell Death in Oncogene-Addicted Cancers
              </a>
            </td>
          <td>
            Hafiz Muhammadismail, Azher Syed
          </td>
          <td>2025-12-30</td>
          <td>Wah Academia Journal of Health and Nutrition</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7af085ce75faa396459c7f42299a926d7da2cc0b" target='_blank'>
              L-pentahomoserine correlates with therapy outcome in esophageal cancer and promotes metabolic adaptations that support cell survival under nutrient-deprived conditions
              </a>
            </td>
          <td>
            Mark E. Kelly, Alejandro Huerta-Uribe, Michaela Cerná, Keira B. Hughes, Hasnain Ahmed, Marco F Fernandes, Kevin M. Rattigan, Ruhi Deshmukh, David S. Moura, Swati Arya, Stephanie May, Miryam Müller, Thomas M Drake, Anastasia Georgakopoulou, Thomas G. Bird, Michael Hisbergues, Guillaume Piessen, Alan J. Stewart, Allan J. B. Watson, D. Sumpton, Victor H. Villar
          </td>
          <td>2026-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Abstract Background Chordoma is a rare bone tumor that is considered to have originated from the remnants of notochord by the general consensus in the field. Chordoma patients suffer from local invasion and recurrence of the disease, along with high resistance to conventional therapies. Despite the scarcity of the disease, new biomarkers must be discovered to develop targeted therapeutics. Methods This study aimed at analyzing the whole exome sequencing data of a chordoma cell line, CH22, which held more than 3,000 missense variants in the genome. These missense-carrying genes were investigated in terms of their association with chordoma pathogenesis. Results Out of them, neurotrophic tyrosine kinase receptor type 3 (NTRK3) and nuclear factor erythroid 2-related factor 2 were found to be associated with the altered pathways involved in chordoma. Conclusion Further examination through structural modeling, molecular docking, and phylogenetic analyses related to the missense variants of NTRK3 was conducted in the scope of this study.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b2fe9308814058a063733e1cf9abe4023d9644d" target='_blank'>
              Deciphering the Structural and Molecular Effects of a Novel Genetic Mutation by In Silico Analysis in Chordoma Cell Line
              </a>
            </td>
          <td>
            Muhammed Ümeyr Fettah, Nur Ekimci-Gürcan, Emre Yorulmaz, Batuhan Oğuz, A. Köseoğlu, Furkan Ayaz, Esra Aydemir
          </td>
          <td>2025-09-30</td>
          <td>Journal of Neurological Surgery Part B: Skull Base</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Gastric cancer (GC) persists as a leading cause of global cancer morbidity and mortality, with its pathogenesis intricately linked to epigenetic dysregulation. Emerging research specifies the novelty of these mechanisms—including DNA methylation, histone modifications, non-coding RNAs (ncRNAs), and RNA modifications—in GC initiation, progression, and therapeutic resistance. This review systematically examines key epigenetic mechanisms in GC, dissect the therapeutic implications as diagnostic biomarkers and therapeutic targets. Key insights include (1) aberrant methylation of tumor suppressor genes (e.g., CDH1, RUNX3): in early carcinogenesis; (2) histone lactylation and acetylation modulating immune evasion (3) ncRNAs (e.g., miR-21, HOTAIR); as promising biomarkers; and (4) m6A RNA modification in chemotherapy resistance. We further discuss translational applications of epigenetic biomarkers in liquid biopsies and targeted therapies (e.g., DNMT/HDAC inhibitors). Integrating multi-omics and epigenetic editing technologies may advance precision medicine in GC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b58e571395e3737fdd50502e583400d42b89b97" target='_blank'>
              Advances in epigenetics of gastric cancer
              </a>
            </td>
          <td>
            Rihua Zeng, Jianning Chen
          </td>
          <td>2026-01-28</td>
          <td>Oncology Reviews</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The Japanese medaka, Oryzias latipes, has become an important vertebrate model organism for addressing research questions across a broad range of disciplines, including developmental and evolutionary biology, stem cells, gene–environment interactions, behavioral neuroscience, disease modeling, and drug discovery. The medaka community took advantage of the successful completion of the ERC Synergy Grant project IndiGene to gather once again in the beautiful Heidelberg (July 22–25, 2025). Building on the opportunities created by the IndiGene project, which leverages the medaka inbreed panel Medaka Inbred Kiyosu–Karlsruhe as a unique resource for dissecting complex phenotype–genotype relationships, the meeting offered an outstanding update on emerging concepts, technologies, and community resources. By summarizing the content of each session, this report provides an overview of a vibrant and highly productive event that highlights the continued growth and vitality of medaka research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03d233ac5889c75a856a0044d1f780508929da83" target='_blank'>
              Insights and emerging perspectives from the 2025 Medaka PI/ERC IndiGene meeting: Heidelberg July 22–24, 2025
              </a>
            </td>
          <td>
            R. D. Acemel, Julien Bobe, Lázaro Centanin, Cielo Centola, Sapna Chhabra, F. Loosli, R. Bhandari, Javier Vázquez-Marín, L. Zilova, E. Birney, Jochen Wittbrodt, J. Martínez-Morales
          </td>
          <td>2026-01-21</td>
          <td>Developmental Dynamics</td>
          <td>0</td>
          <td>130</td>
        </tr>

        <tr id="


 Prostate adenocarcinomas are the most common form of the disease. Targeting the androgen receptor (AR) is the principal treatment strategy. However, resistance is common. Lineage plasticity (LP)—change in differentiation state—is increasingly recognized as a mechanism of resistance to AR-targeting therapies. LP is a continuum ranging from amphicrine to neuroendocrine prostate cancer (NEPC). Despite the increasing incidence of LP due to more widespread use of new and potent AR inhibitors, mechanisms by which LP emerges after therapy have been understudied due to paucity of matched patient tumors. To address this deficit, we performed multi-omic profiling on matched biopsies from 16 patients, including seven patients that underwent LP after treatment.



 We used DNA sequencing to identify mutations and copy number alterations present in tumors at baseline and at progression. We also performed RNA sequencing and transcriptional subtyping of all tumors.



 Analysis of RNA sequencing revealed two distinct subtypes of LP in our cohort—most consistent with amphicrine or NEPC. DNA sequencing identified TP53 and RB1 alteration patterns in baseline tumors that eventually underwent specific LP trajectories. Pathways linked to proliferation and inflammatory signaling were highly activated in baseline tumors that eventually underwent LP.



 Our analysis identified distinct LP trajectories linked with specific genomic and transcriptomic features at baseline. Profiling tumors with this approach may provide clues about which patients may be at greatest risk for undergoing specific LP trajectories after therapy.



 Anbarasu Kumaraswamy, Visweswaran Ravikumar, Ryan J. Rebernick, Eva Rodansky, Amina Tanweer, Joel A. Yates, Aaron M. Udager, Marcin P. Ceislik, Arvind Rao, Zachery R. Reichert, Arul M. Chinnaiyan, Joshi J. Alumkal. Multi-Omic Profiling Clarifies Mechanisms of Therapy-Induced Prostate Cancer Lineage Plasticity [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A036.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d201d414071181c06c4abf5de951c2f23846da37" target='_blank'>
              Abstract A036: Multi-Omic Profiling Clarifies Mechanisms of Therapy-Induced Prostate Cancer Lineage Plasticity
              </a>
            </td>
          <td>
            A. Kumaraswamy, V. Ravikumar, Ryan J. Rebernick, Eva Rodansky, Amina Tanweer, Joel A. Yates, Aaron M. Udager, Marcin P. Ceislik, Arvind Rao, Z. Reichert, A. Chinnaiyan, J. Alumkal
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>171</td>
        </tr>

        <tr id="Gene therapy has evolved into a sophisticated field encompassing diverse precision editing platforms and advanced delivery systems capable of addressing complex genetic disorders and age-related pathologies. This comprehensive review examines the current landscape of gene therapeutic technologies, including CRISPR-based genome editing, base editing systems, prime editing platforms, and emerging DNA polymerase-based editors alongside their corresponding delivery methodologies. The review encompasses viral vectors, including tissue-specific adeno-associated virus serotypes, non-viral delivery systems such as ionizable lipid nanoparticles and virus-like particles, and innovative platforms, including exosome-based delivery and the SEND system. We examine therapeutic applications spanning nuclear genome editing, mitochondrial genome modification, RNA editing, and epigenetic modulation, demonstrating the expanding scope of gene therapy beyond traditional monogenic disorders. Critical analysis reveals that while fundamental technological capabilities have been established, significant challenges remain in manufacturing scalability, long-term safety assessment, delivery across physiological barriers, and optimization of editing efficiency in post-mitotic tissues. The integration of artificial intelligence approaches for predictive analysis and rational vector design represents a promising avenue for addressing current limitations. This review concludes that successful clinical implementation requires systematic resolution of manufacturing, safety, and delivery challenges alongside the development of standardized protocols for patient stratification and robust regulatory frameworks that accommodate rapid technological innovation while ensuring patient safety.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24cc3f28029393deaa5073b92c5af359263d8da2" target='_blank'>
              Gene Therapy Techniques and Delivery Methods (Review)
              </a>
            </td>
          <td>
            E. I. Shchukina, Junior Researcher, PhD I.O. Mazunin, MD I.I. Eremin, DSc A.A. Moskalev
          </td>
          <td>2025-12-29</td>
          <td>Modern Technologies in Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Acute lymphoblastic leukemia (ALL) is a heterogeneous malignancy characterized by various genomic alterations playing a crucial role in disease classification, prognosis, and response to treatment. However, molecular diagnosis and effective management of this hematological malignancy remain a major challenge, particularly in developing countries, including Tunisia. In this study, we aimed to conduct a detailed analysis of copy number alterations (CNAs) associated with ALL in a cohort of 60 primary samples from Tunisian patients. Using multiplex ligation-dependent probe amplification (MLPA), major genetic lesions, including IKZF1, CDKN2A/2B, PAX5, ETV6, BTG1, and genes located in the PAR1 region, were analyzed and their associations with clinical and laboratory features, as well as survival outcomes, were also evaluated. Our analysis revealed that 70% of patients had deletions and/or amplifications in at least one gene. The most frequently observed deletions were in CDKN2A/2B (33.3%, n = 20), IKZF1 (30%, n = 18), and PAX5 genes (25%, n = 15). BTG1 deletions were significantly associated with female gender, IKZF1 deletions were more frequent in adult patients, in those with elevated white blood cell (WBC) counts, and in cases involving the BCR::ABL1 translocation, while duplications of the PAR1 region were significantly associated with hyperdiploïdy. Regarding treatment response, cases of IKZF1 deletions showed a significant association with poor glucocorticoid response (GC) at day 8 of treatment and positive minimal residual disease (MRD) rates at days 33 and 63, particularly in B-ALL cases. Furthermore, patients with IKZF1 deletions were associated with significantly lower survival rates in both univariate and multivariate analyses compared to those without these deletions. Additionally, the integration of IKZF1 deletion status into risk stratification models revealed markedly different survival outcomes, highlighting its potential interest in developing new stratification algorithms. These results underscore the critical importance of molecular profiling, particularly IKZF1 status, for improving outcomes in ALL patients in Tunisia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd7521499fd8c33c0af8b7b43eac8773db943a64" target='_blank'>
              Genomic characterization and prognostic significance of copy number alterations in Tunisian patients with acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Ameni Bedoui, W. Ayadi, Nour Louati, Imen Frikha, Yosra Fakhfakh, Fahmi Smaoui, Ali Gargouri, Ikram Ben Amor, Moez Elloumi, Moez Mdhaffar, Raja Mokdad Gargouri
          </td>
          <td>2026-02-03</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 The paradigm of the
 Hallmarks of Cancer
 , updated by Douglas Hanahan in 2022, represents one of the most influential syntheses for understanding the functional capabilities that sustain neoplastic transformation. However, its traditional interpretation, often reductionist and fragmentary, does not capture the non-linear, emergent, and adaptive dynamics of tumor behavior. This review proposes a reinterpretation of the hallmarks through the lens of complexity theory, conceptualizing colorectal cancer (CRC) as a self-organizing, open system operating far from equilibrium. Using an integrative conceptual approach, we map the ten classical hallmarks and the new dimensions proposed in 2022 (phenotypic plasticity, non-mutational epigenetic reprogramming, polymorphic microbiomes, and senescence) onto the fundamental properties of complex systems: nonlinearity, emergence, feedback, openness, and historical dependence. We argue that CRC should not be understood as a simple sum of molecular alterations but as a dynamic network of interactions among cells, tissues, and microenvironments where global organization emerges from local rules. This systems-based perspective provides a conceptual foundation for translational models and integrative methodologies in oncology.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0199a76d1009c249f4505d66d97a40ec8df69d8" target='_blank'>
              Colorectal cancer as a complex adaptive system: integrating the hallmarks of cancer with complexity theory
              </a>
            </td>
          <td>
            Luis R. Basbus
          </td>
          <td>2026-02-03</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Chimeric antigen receptor (CAR) T cell therapies have demonstrated remarkable efficacy in several hematological malignancies, yet their success has not been fully replicated in solid tumors. Moreover, even in hematological cancers, relapse after CAR T cell infusion continues to compromise long-term outcomes. These challenges highlight the urgent need to develop strategies that enhance CAR T cell efficacy, persistence, overcoming tumor and microenvironment-mediated resistance. Clustered regularly interspaced short palindromic repeats (CRISPR)-Cas9-based screening platforms provide a powerful approach to systematically identify genes that regulate CAR T cell function. By linking genetic perturbations to phenotypic outcomes, these assays enable the discovery of pathways controlling activation, proliferation, memory formation, and cytotoxicity. Standard workflows involve transduction of substantial numbers of cells with a single guide RNA (sgRNA) library, Cas9-mediated editing, selection of edited cells, and PCR amplification of sgRNA cassettes from genomic DNA (gDNA) prior to sequencing. However, PCR amplification using large amounts of gDNA poses significant challenges and often fails to selectively amplify and retrieve sgRNAs. Here, we describe an optimized CRISPR-Cas9 knockout screening protocol, which we have tested on primary human CAR T cells. The method here incorporates an intermediate step during sgRNA library preparation that reduces gDNA carryover through enzymatic digestion and selective pulldown of the sgRNA cassette, thereby increasing the efficiency of the first PCR amplification. This modification allowed us to retrieve sgRNA information across our CAR T cell screens, which had remained elusive in our previous attempts using traditional 1 and 2-step PCR amplification protocols. In conclusion, this optimized workflow facilitates CRISPR screening library preparation in challenging samples and enables the identification of key genetic determinants that can be targeted to improve therapeutic efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecca94f275145ee359153517eba4093f85ededd3" target='_blank'>
              Improving CRISPR-Cas9 Screens in CAR T Cells: A Refined Method for Library Preparation.
              </a>
            </td>
          <td>
            Maider Garnica, Patxi San Martín-Úriz, Paula Rodriguez-Marquez, M. E. Calleja-Cervantes, Saray Rodríguez-Díaz, Rebeca Martinez-Turrillas, Mikel Hernaez, Felipe Prósper, J. R. Rodríguez-Madoz
          </td>
          <td>2026-01-02</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Despite breakthroughs in molecularly targeted and immune therapies, the prognosis for patients with urinary bladder cancer (UBC) has remained unsatisfactory over the past few decades. Understanding the molecular underpinnings of UBC treatment refractoriness is crucial for identifying novel therapeutic targets and strategies. Cancer stemness plays a pivotal role in the oncogenesis and treatment resistance of UBC, while the underlying molecular regulatory mechanisms are poorly understood. We identified isoform 1 of ASPM (ASPM-i1) as the most upregulated stemness-associated factor in tumorigenic UBC cells, predominantly expressed by ALDH1+ stem-like cancer cells. Pairing genetic ASPM-i1 inhibition with standard chemotherapeutic agents used in the treatment of UBC, including cisplatin and gemcitabine, circumvents the treatment resistance of tumorigenic and stem-like UBC cells. Mechanistically, ASPM-i1 interacts with the Disheveled (DVL) and intracellular NOTCH proteins, thereby attenuating CUL3- or FBXW7-mediated ubiquitination and the subsequent proteasomal degradation. The regulatory module concomitantly enhances the activities and ligand responsiveness of the Wnt and Notch signaling pathways in UBC cells. As a result, ASPM-i1 inhibition sensitized tumorigenic UBC cells to chemotherapy in a NOTCH- and DVL-dependent manner. In human UBC tissues, ASPM-i1 shows substantial cell-to-cell heterogeneity and is upregulated in a subset (46.4%) of tumors, correlating with poor clinical prognosis. This study reveals a crucial co-regulatory module of Notch and Wnt signaling that mediates stemness and chemotherapy resistance in tumorigenic UBC cells; its inhibition provides a novel approach to enhance chemosensitivity and improve therapeutic outcome in human UBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/076780e38121b3f25a600b0e4d535709ef262847" target='_blank'>
              Chemotherapy resistance of urinary bladder cancer mediated by a Notch and Wnt co-regulatory module of stemness.
              </a>
            </td>
          <td>
            Lin-Hsin Cheng, Che-Wei Hsu, Pei-Ming Yang, Tai-Yan Liao, Fang-Chi Liu, Wan-Wen Chen, Chin-Chen Pan, Chung-Chi Hsu, Kuan-Chou Chen, K. K. Tsai
          </td>
          <td>2026-01-10</td>
          <td>Stem cells</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Uveal melanoma (UM) is a highly aggressive intraocular malignancy with limited therapeutic options for metastatic disease. Existing transgenic UM mouse models inadequately recapitulate human disease progression, while transplant models lack immune competence for studying the tumor immune microenvironment and therapeutic interventions. To address these limitations, we developed a genetically engineered mouse model incorporating stepwise genetic alterations implicated in human UM progression. Spatiotemporally controlled expression of mutant GNAQQ209L from the endogenous locus induced choroidal nevi with limited penetrance. Concomitant BAP1 deletion enhanced nevus formation, while further MYC activation led to fully penetrant intraocular tumors with the potential to disseminate. Single-cell RNA sequencing revealed malignant cells segregated into melanocytic and neural crest-like subpopulations characterized by distinct transcriptional and biosynthetic programs. Trajectory analyses inferred dedifferentiation from the melanocytic toward the neural crest-like state during tumor progression. Comparison to human UM revealed commonalities with highly aggressive class 2 UM, including gene expression signatures and copy number gains affecting genes that map to human chromosome 8q beyond the activated MYC allele, suggesting cooperative effects of multiple drivers in this chromosomal region. The tumor microenvironment featured immunosuppressive macrophage populations and exhausted T cells, closely resembling human UM. This physiologically relevant, immune-competent model provides a platform for investigating UM biology, functionally characterizing candidate driver genes, and developing immune-based therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8fdbc1108ba636781e58c18b409a85d4d4db19d" target='_blank'>
              A Multi-Step Immune-Competent Genetically Engineered Mouse Model Reveals Phenotypic Plasticity in Uveal Melanoma.
              </a>
            </td>
          <td>
            Xiaonan Xu, Xiaoxian Liu, Vinesh Jarajapu, Sathya Neelature Sriramareddy, Filip Konecny, Benjamin Posorske, J. Dollar, Xiao Liu, Neel Jasani, Kaizhen Wang, Nicol Mecozzi, Zulaida Soto-Vargas, Shaaron L. Ochoa-Rios, Harini Murikipudi, Nhan Phan, Manon Chadourne, J. Kuznetsoff, John Sinard, Richard L Bennett, Jonathan D. Licht, K. Smalley, J. W. Harbour, Xiaoqing Yu, F. Karreth
          </td>
          <td>2026-02-09</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Traditionally, scientists tend to approach cancer research in a reductionistic way: aiming at uncovering underlying, separate components in malignant processes. And indeed, great progress has been made by reducing the development of a tumor to single, specific genes and mutations. For instance, familial adenomatous polyposis (FAP) could be reduced to a germline mutation in the Adenomatous Polyposis Coli (APC) gene. The escape of tumor cells from immune surveillance could be reduced to the tumor expression of immune checkpoints, resulting in new approaches in tumor therapy by applying immune checkpoint inhibitors. However, a germline mutation in APC is not 1:1 related to colorectal cancer (CRC), and only some patients respond to immune checkpoint inhibitors. The point here is that biological systems, also comprising cancer, have properties that cannot be reduced to single components. The cooperation of the single components results in new, emergent properties. The outcome of an interaction in a complex network, like the immune system, depends on the many cell types involved and the numerous molecules that interact and activate or inhibit pathways. The way the composing elements are organized is a causal factor in itself for any emergent property. The rise of genomic analysis at the end of the previous century, enabling us to sequence a full genome at the DNA and RNA levels, has initiated an awareness of the need for ‘systems biology’: to consider a full system and how it is organized, in all of its aspects, to understand biological pathways and their outcomes. In this review, we outline the prospects and limitations of systems biology in cancer research and propose a causal framework that integrates upward and downward causation and multiple realizability to understand the emergent properties of tumors that determine the dynamics of tumor development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bad26730d321f49013a748603a1b42d130f54226" target='_blank'>
              The Need for a Systems Biology Approach in Cancer Explained
              </a>
            </td>
          <td>
            Hehuan Zhu, Xi Zhang, E. Nazemalhosseini-Mojarad, J. Roelands, L. D. A. N. de Muynck, C. Ravensbergen, Rachel Hoorntje, I. Stouten, Marianne Hokland, A. L. Vahrmeijer, Rob A. E. M. Tollenaar, Edwin Koster, Peter J K Kuppen
          </td>
          <td>2025-12-22</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Neuroendocrine prostate cancer (NEPC) is an aggressive, therapy-resistant subtype of prostate cancer characterized by lineage plasticity. While metabolic and signaling molecules are increasingly recognized as modulators of tumor progression, their role in cell fate transition remains unclear. NE tumors produce and accumulate serotonin, a neurotransmitter that regulates diverse physiological processes. Here, we identify a tumor-intrinsic serotonin axis as key driver of NEPC lineage commitment and progression. NEPC endogenously synthesize serotonin via aromatic L-amino acid decarboxylase (DDC) and reuptake through the transporter SLC6A4. Mechanistically, high level of intracellular serotonin promotes histone serotonylation at H3K4me3Q5, reconfiguring the H3K4me3 chromatin landscape and downstream gene expression, which drives induced NE differentiation and is associated with suppressed androgen receptor signaling. Pharmacological inhibition of 5-HT synthesis using the FDA-approved DDC inhibitor carbidopa significantly impairs tumor growth and prolongs survival in both genetically engineered and patient-derived xenograft models, highlighting histone serotonylation as a druggable vulnerability in NEPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09ed69f6cdfb6f3b5c5e09b3796f47388a50957e" target='_blank'>
              Serotonin modulates lineage plasticity in neuroendocrine prostate cancer via epigenetic reprogramming.
              </a>
            </td>
          <td>
            Yiyi Ji, Chengwei Ju, Lei Chen, Kai Shen, Ruopeng Su, Ang Li, Xinyu Liu, Bo Liu, Xinran Zhang, Ruitu Lyu, Peng Xia, Han Li, Yiqian Pan, Yunzheng Liu, Man Hin Tse, Yizheng Xue, Hongyang Qian, Na Jing, Helen He Zhu, Liangliang Wang, Lisheng Zhang, Shuheng Jiang, Weiwei Zhang, Liang Dong, Zejun Yan, Jiahua Pan, Yinjie Zhu, Jiangbo Wei, Qi Wang, Wei Xue
          </td>
          <td>2025-12-19</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Metastasis is still the leading cause of cancer-related death. It happens when disseminated tumor cells (DTCs) successfully navigate a series of steps and adapt to the unique conditions of distant organs. In this review, key molecular and immune mechanisms that shape metastatic spread, long-term survival, and eventual outgrowth are examined, with a focus on how tumor-intrinsic programs interact with extracellular matrix (ECM) remodeling, angiogenesis, and immune regulation. Gene networks that sustain tumor-cell plasticity and invasion are described, including EMT-linked transcription factors such as SNAIL and TWIST, as well as broader transcriptional regulators like SP1. Also, how epigenetic mechanisms, such as EZH2 activity, DNA methylation, chromatin remodeling, and noncoding RNAs, lock in pro-metastatic states and support adaptation under therapeutic pressure. Finally, proteases and matrix-modifying enzymes that physically and biochemically reshape tissues, including MMPs, uPA, cathepsins, LOX/LOXL2, and heparinase, are discussed for their roles in releasing stored growth signals and building permissive niches that enable seeding and colonization. In parallel, immune-evasion strategies that protect circulating and newly seeded tumor cells are discussed, including platelet-mediated shielding, suppressive myeloid populations, checkpoint signaling, and stromal barriers that exclude effector lymphocytes. A major focus is metastatic dormancy, cellular, angiogenic, and immune-mediated, framed as a reversible survival state regulated by stress signaling, adhesion cues, metabolic rewiring, and niche constraints, and as a key determinant of late relapse. Tumor-specific metastatic programs across mesenchymal malignancies (osteosarcoma, chondrosarcoma, and liposarcoma) and selected high-burden cancers (melanoma, hepatocellular carcinoma, glioblastoma, and breast cancer) are highlighted, emphasizing shared principles and divergent organotropisms. Emerging therapeutic strategies that target both the “seed” and the “soil” are also discussed, including immunotherapy combinations, stromal/ECM normalization, chemokine-axis inhibition, epigenetic reprogramming, and liquid-biopsy-enabled minimal residual disease monitoring, to prevent reactivation and improve durable control of metastatic disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a5ad40c8765430fb4e35e8df4878d707b5a6318" target='_blank'>
              Molecular and Immune Mechanisms Governing Cancer Metastasis, Including Dormancy, Microenvironmental Niches, and Tumor-Specific Programs
              </a>
            </td>
          <td>
            Dae Joong Kim
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="An early childhood onset neurodegenerative disorder, ataxia telangiectasia (AT), affects one in 40,000 to 100,000 individuals worldwide and is caused by mutations in the ataxia telangiectasia mutated (ATM) threonine–serine kinase, which regulates the DNA damage response (DDR). While the cause of AT has been known for years, the exact molecular mechanisms underlying disease progression, particularly at the transcriptomic level, remain poorly understood. Three stranded structures, known as R-loops, have recently emerged as important players in the DDR via regulating key gene expression. Here, we utilized neuronal progenitor cells (NPCs) derived from induced pluripotent stem cells reprogrammed from patient-derived somatic cells to identify how loss of ATM impacts R-loop and transcriptional dynamics, both at baseline and in response to acute DNA damage. AT-derived NPCs (AT-NPCs) exhibited elevated spontaneous R-loop levels compared with control-NPCs, as well as a strong positive correlation between R-loop accumulation and gene expression on a subset of dysregulated genes. Upon acute damage, loss of ATM resulted in an attenuated response, characterized by the impaired R-loop and transcriptional response to irradiation. Both control- and AT-NPCs underwent a similar cell cycle arrest, but AT-NPCs displayed an attenuated R-loop and transcriptional response, failing to activate a proper DDR response. Importantly, R-loop formation is required for many key genes to properly respond to DNA damage, supporting a direct and causal role in this process. Overall, our data reveal an underappreciated mechanistic link between ATM, R-loop regulation, and transcription, the disruption of which may contribute to the impaired DDR observed in AT-NPCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29977be637336b9d941d50d03bb41249687d1b5a" target='_blank'>
              Loss of ATM causes R-loop–associated transcriptional dysregulation and attenuates the related response to DNA damage
              </a>
            </td>
          <td>
            Katherine R. Westover, Yingzi Hou, Feng Wang, Yilin Wang, Yangping Li, Rachel Seong, Jie Xu, Zhexing Wen, Bing Yao
          </td>
          <td>2026-01-01</td>
          <td>The Journal of Biological Chemistry</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97e8ef7d28e9d10b44890e1b7be7668548430edd" target='_blank'>
              CHEK2 loss endows chemotherapy resistance to hematopoietic stem cells.
              </a>
            </td>
          <td>
            Jing Zhou, Tianyuan Hu, Dian Li, Sanming Li, Minhua Li, Xiangguo Shi, Daisuke Nakada
          </td>
          <td>2026-01-12</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Introduction.
 The development of therapy resistance and relapses of acute myeloid leukemia (AML), especially in the intermediate prognosis group, may be due to the molecular genetic heterogeneity of tumor cells. Chromosomal microarray analysis (CMA) can detect microdeletions, duplications, and copy-neutral loss of heterozygosity (cnLOH) which may be associated with a response to therapy.


 Aim:
 to evaluate tthe frequency of copy number aberrations and cnLOH in leukemogenesis-associated genes in patients with intermediate-stage AML and and their relationship to survival and response to treatment.


 Materials and methods
 . The study included 35 patients with de novo AML from the intermediate prognosis group for ELN2017. Copy number analysis by CMA was performed for a panel of 36 genes associated with leukemogenesis. The reference group included 102 healthy individuals without oncohematological disorders who also underwent comparable CMA testing.


 Results.
 Genomic aberrations were detected in 91.18 % of patients, most often in the genes of chromatin modifiers (64.7 % patients) and tumor suppressor genes (64.7% patients). The cnLOH type (PHF6, SMC1A, BKORL1) prevailed. KMT2A duplications occurred only in AML patients — 14.3 % (p < 0.001) and were associated with worse survival (log-rank P = 0.05). Combinations of genomic alterations involving 4–7 functional gene groups were found in 20.6% of patients.


 Conclusion.
 Driver gene aberrations, especially KMT2A duplications, are associated with an unfavorable clinical outcome in AML with an intermediate prognosis.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4336f89da62f27fbefbd6b50cf91ca51d0c03af7" target='_blank'>
              Structural aberrations of genes associated with leukemogenesis in patients with acute myeloid leukemia of intermediate prognosis
              </a>
            </td>
          <td>
            D. K. Bessmertnyy, S. Starchenko, N. Risinskaya, S. Kulikov, U. A. Chabaeva, V. Surimova, A. S. Ponamoreva, I. Kanivets, Z. T. Fidarova, I. Lukianova, A. Kashlakova, E. V. Romanyuk, N. I. Balaeva, V. Troitskaya, A. Sudarikov, E. Parovichnikova
          </td>
          <td>2025-12-21</td>
          <td>Russian journal of hematology and transfusiology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Tissue morphogenesis and homeostasis occur at the interface of a fine coordination between cell cycle and differentiation events. The fundamental role of Hox genes during embryonic development is well established. However, less is known about their role in adult tissues and in contexts other than cellular differentiation. In this review, we focus on the role of Hox proteins in cell cycle-related events in adult tissues and highlight two main cell cycle-permissive and repressive roles for these genes. Given the significance of Hox proteins as a therapeutic target in cancer treatment, we will further discuss the existing approaches in this domain highlighting the challenges that impact the success prospects. For the successful development and application of Hox-oriented anticancer therapies, a comprehensive image of Hox roles in carcinogenic events including their mechanism of function as well as the networks/cascades within which they exert their effects is an inevitable requirement. Moreover, the functional variations of Hox genes in response to changes in the tumour microenvironment along with possible alternations of the mutational landscape in tumours during disease progression should also be taken into account as factors that might impact the therapeutic outcomes of Hox-targeting approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8ece325e31ec25f81ab6186f05250cdd8b92ab2" target='_blank'>
              The diverse roles of Hox proteins in the regulation of the cell cycle and their therapeutic value in cancer treatment
              </a>
            </td>
          <td>
            Lupeuea Vakafua, Lachlan Clifton-Bligh, Naisana Seyedasli
          </td>
          <td>2026-01-15</td>
          <td>Cancer Metastasis Reviews</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="BACKGROUND Synthetic messenger RNA (mRNA) vaccines have raised concerns regarding prolonged spike protein expression, immune activation, and potential off-target effects. AIM To investigate transcriptomic alterations in individuals with new-onset adverse events or cancer following mRNA coronavirus disease 2019 vaccination. METHODS Bulk RNA sequencing was performed on peripheral blood from two patient groups: (1) Individuals with new-onset nonmalignant adverse events; and (2) Individuals newly diagnosed with cancer post-vaccination. A control group of normal individuals was used for comparison. Differential gene expression was analyzed using DESeq2, and Gene Set Enrichment Analysis was conducted using the MSigDB database and custom gene sets. RESULTS Both vaccine patient groups displayed widespread transcriptional dysregulation. In the nonmalignant adverse event group, hallmark enrichments included mitochondrial dysfunction, proteasome-mediated stress, transcriptomic instability, and systemic inflammation. The cancer group exhibited additional hallmarks of genomic instability and epigenetic reprogramming. Nonsense-mediated decay, ribosomal stress, and myelocytomatosis oncogene activation were prominent in both groups, while immune signaling via toll-like receptors and type I interferons was particularly elevated in cancer patients. The observed transcriptomic profiles indicate cellular stress responses, mitochondrial dysfunction, and immune dysregulation following exposure to mRNA vaccines, potentially in susceptible individuals. CONCLUSION Shared and distinct molecular signatures in both cohorts demonstrate underlying mechanisms contributing to post-vaccine symptomatology and complications, including oncogenesis and or progression of malignant disease. These findings underscore the need for a deeper investigation into the long-term safety of mRNA vaccines and host response variability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a28e301bd6430c7a6e743e763b659faf3780026d" target='_blank'>
              Synthetic messenger RNA vaccines and transcriptomic dysregulation: Evidence from new-onset adverse events and cancers post-vaccination
              </a>
            </td>
          <td>
            Natalia Lidmar Von Ranke, Wei Zhang, Philipp Anokhin, Nicolas Hulscher, Kevin McKernan, Peter Mccullough, J. Catanzaro
          </td>
          <td>2025-12-20</td>
          <td>World Journal of Experimental Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="BACKGROUND
Pancreatic cancer (PC) is among the most aggressive and lethal malignancies, characterized by development within a complex tumor microenvironment (TME) that includes a desmoplastic stroma composed of extracellular matrix (ECM) and various cellular components.


OBJECTIVES
This study aims to elucidate the cellular and molecular mechanisms regulating PC progression through an integrated analysis of single-cell pseudotime trajectories and intercellular communication.


MATERIAL AND METHODS
We constructed pseudotime trajectories using single-cell RNA sequencing (scRNA-seq) data from PC tissues to trace the developmental progression of cancer cells. Transitional cell states and critical genes involved in the shift from early-to-advanced disease stages were identified. Through a comprehensive analysis, we pinpointed key transcription factors and signaling pathways implicated in tumor progression. Expression of stemness-associated genes in pancreatic stellate cells (PSC) was validated using immunofluorescence and transmission electron microscopy (TEM). Additionally, cell-cell communication analysis was performed to examine interactions within the TME, with particular emphasis on ligand-receptor pairings.


RESULTS
Our analysis identified key transcription factors and signaling pathways that drive the cellular transitions associated with cancer progression. The findings revealed extensive intercellular crosstalk between cancer cells, stromal fibroblasts, and diverse immune cell subpopulations. Notably, the study underscored the distinct functional contributions of these cell populations to tumor development, immune evasion and metastatic dissemination.


CONCLUSIONS
The study uncovers the complex cellular diversity and intercellular crosstalk in PC, providing novel avenues for therapeutic interventions and early predictive markers in diagnosis. These findings support the potential for more targeted, personalized treatment strategies in combating PC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1072a0d2e360d78c1789524d36e7d5c1df78fbc" target='_blank'>
              Single-cell pseudotime and cell communication analysis of pancreatic cancer.
              </a>
            </td>
          <td>
            Chengming Ni, Xiaohang Wang, Zhensheng Cai, Yang Chen, Huan Wang, Qianqian Wang, Hao Lin, Yunting Zhou, Yang Yuan, Bo Sun, Zilin Sun
          </td>
          <td>2026-02-09</td>
          <td>Advances in clinical and experimental medicine : official organ Wroclaw Medical University</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) are the main constituents of the tumor microenvironment. Several studies have delineated CAF heterogeneity in different types of tumors, however, it is still unknown how the distinct CAF transcriptional profiles are established during tumor progression. We reanalyzed a previously published single-cell RNA-sequencing dataset of MMTV-PyMT tumors at higher resolution using Seurat , and CytoTRACE to characterize CAF subtypes and their differentiation states. Wilcoxon rank sum test was applied for differential gene expression. Multiplex immunostaining (Akoya PhenoImager HT) was performed on 38 murine mammary tumors from MMTV-PyMT mice to identify the distinct CAF subtypes. Whole-slide imaging and spatial analysis were conducted using QuPath and Cellpose , followed by neighborhood clustering and interaction mapping with CytoMAP . Cellular distances from CAFs to immune, tumor, and endothelial cells were quantified using SPIAT and Wilcoxon tests for comparisons. In parallel, human spatial transcriptomics data from the 10X Genomics Xenium platform were integrated for cross-species validation. Here, by single cell RNA-sequencing and multiplex immunostaining, we identify six CAF substates. Spatial analysis on immunostained murine mammary tumors and human spatial transcriptomics data outlined temporal changes in stromal composition and the existence of distinct functional niches enriched with different CAF substates. Immunomodulatory CAFs co-localized with immune cells while myofibroblastic CAFs formed a shield around the tumor core, thus preventing immune infiltration. Our work supports the idea that distinct spatial locations dictate different CAF transcriptional programs. Targeting specific functional niches will ultimately hinder tumor progression by inhibiting signaling between distinct CAF substates and the surrounding tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1d802e2b1323bfcdbf953412180c241803eb33a" target='_blank'>
              Spatial and temporal dynamics of cancer-associated fibroblast niches in breast cancer
              </a>
            </td>
          <td>
            Jessica Pantaleo, J. Sjölund, P. Bolivar, M. Bocci, B. Phung, Maria Malmberg, G. Jönsson, Kristian Pietras
          </td>
          <td>2026-01-11</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/683398225f12a992fd68b945ba84e877fc29b6de" target='_blank'>
              Interpreting cancer genetics through a two-step "evolutionary cascade hypothesis": bridging neutral and selective perspectives.
              </a>
            </td>
          <td>
            A. Ottaiano, M. Santorsola, Francesco Sabbatino, Roberto Sirica, Francesco Caraglia, Anna Ceccarelli, Vincenza Granata, Ines Simeone, S. Zappavigna, M. Berretta, Giovanni Savarese, Michele Caraglia
          </td>
          <td>2026-02-13</td>
          <td>Journal of translational medicine</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="The advent of precision medicine has marked a new Head and neck in cancer treatment, where therapeutic approaches are customized of each patient tumor individual genetics and molecular based on the characteristics. Standard practice now involves genomic profiling, which allows clinicians to pinpoint specific genetic mutations and alterations responsible for driving cancer growth. Armed with this information, targeted therapies can be chosen to disrupt these particular pathways. Additionally, liquid biopsies provide a noninvasive method for monitoring genetic changes over time, assisting in treatment adjustments and the identification of resistance mechanisms. The research significance lies not only in these individual advancements but in their synergistic integration. Comprehensively understanding the genomic landscape allows for the identification of actionable targets, which can then be exploited by targeted therapies. The interplay between genomics and immunotherapy, as well as the potential additive or synergistic effects of combined treatments, are areas of active investigation. Furthermore, the incorporation of personalized therapies into clinical practice represents a shift towards more individualized and effective treatment strategies. This is particularly crucial in head and neck cancer, where heterogeneity in tumor biology necessitates tailored approaches for optimal outcomes. As these advancements progress, ongoing clinical trials play a pivotal role in validating their efficacy, safety, and long-term impact on patient survival and quality of life. Personalized Medicine and Genomic Profiling, Public Health Initiatives and Prevention, Epidemiology and Risk Factors, Clinical Presentation and Diagnosis and Treatment Modalities. the DEMATEL-based ranking of Advancements in Personalized Therapies for Head and Neck Cancer, Personalized Medicine and Genomic Profiling secure the top position, while Clinical Presentation and Diagnosis obtain the lowest rank.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef221fdbd8dd2ea3c3c4ca93bd06a6ddefdf3e44" target='_blank'>
              Computational Modeling of Personalized Cancer Treatments Integrating Genomics, Immunotherapy, and Targeted Treatments using DEMATEL Method
              </a>
            </td>
          <td>
            Chandrasekar Raja
          </td>
          <td>2026-02-12</td>
          <td>Computer Science, Engineering and Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Chromosomal instability (CIN), impaired telomere biology, and aberrant DNA methylation are implicated in colorectal cancer (CRC) development. Tracking these alterations from precancerous lesions through tumours to metastases may reveal biomarkers of CRC initiation and progression. Tissue samples from 44 patients with either high-grade colorectal dysplasia (HGA; n = 13) or advanced metastatic CRC (n = 31) were analyzed. CIN was assessed in all patients using either low-coverage whole-exome sequencing or microarray-based comparative genomic hybridization. In a subset of patients, genome-wide CpG methylation profiling (n = 19) and telomere length measurements (n = 15) were performed. CIN was detected in 85% of HGA patients, spanning focal CNVs in MALAT1 (46%) to recurrent alterations on chromosomes 11, 13, and 20, with PTK6 being the most frequently amplified (61%). CIN was comparable between primary tumours and synchronous metastases but was significantly elevated in metachronous cases. DEK was amplified in all metastases but the aberration was absent in primaries, irrespective of tissue chronicity. Methylation profiling distinguished HGA from adjacent non-dysplastic mucosa (9,859 differentially methylated CpGs) and unrelated tumour tissues (17,638 CpGs), whereas primary tumours and metastases differed at only five CpG sites. Both primary tumours and metastases appeared epigenetically younger than colonic mucosa. Metastases exhibited significantly shorter telomeres than both primary tumours (p = 0.019) and colonic mucosa (p = 0.001). The amplification of PTK6 may serve as an early biomarker detectable at the HGA stage, while DEK amplification appears crucial for metastatic progression and may represent a therapeutic target. Further validation is needed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eac4e7ee54e068b1f7e50d381ee4cd9677cd3421" target='_blank'>
              Patterns of Chromosomal Instability and Epigenetic Alterations in Colorectal Cancer Progression: From High-Grade Dysplasia to Liver Metastases.
              </a>
            </td>
          <td>
            Kateřina Šašková, M. Landfors, V. Hlaváč, K. Šeborová, Jan Brůha, Katerina Honkova, Kristyna Tomasova, Saba Selvi, Jana Drabova, Fernanda S Hackenhaar, Torbjörn K Nilsson, P. Vodicka, Anna Valickova, Václav Liška, Sanja A. Farkas, S. Degerman, Michal Kroupa
          </td>
          <td>2026-02-13</td>
          <td>Mutagenesis</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e961f589165d98aa8aff64f0539527c2012d419" target='_blank'>
              Pooled single-cell screen in colorectal cancer defines transcriptional modules linked to oncogenes.
              </a>
            </td>
          <td>
            Viola Hollek, Francisca Böhning, Catalina Florez Vargas, A. Sieber, M. Morkel, Nils Blüthgen
          </td>
          <td>2026-01-19</td>
          <td>Molecular systems biology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Oral squamous cell carcinoma (OSCC) remains a highly prevalent and aggressive malignancy with limited improvements in survival rates. One of the major obstacles to successful treatment is the development of chemoresistance, which contributes to recurrence, metastasis, and treatment failure. This narrative review aims to integrate current evidence on the molecular and cellular mechanisms that drive chemoresistance in OSCC and to delineate how these processes converge under therapeutic pressure. A structured search was performed to identify relevant studies addressing chemoresistance in OSCC, focusing on preclinical and translational evidence. Multiple interconnected mechanisms have been implicated in driving resistance in OSCC, including epigenetic alterations, deregulated signaling pathways, cancer stem cell plasticity, epithelial–mesenchymal transition (EMT), interactions with the tumor microenvironment (TME), drug efflux mediated by ATP-binding cassette (ABC) transporters, and enhanced DNA damage response. In combination, these mechanisms support tumor persistence and limit effective antitumor immunity. Emerging strategies such as epigenetic modulators, signaling pathway inhibitors, immunomodulation, and nanomedicine-based delivery systems have shown promising results in preclinical models. By highlighting convergent resistance networks, this integrative perspective supports the rational design of combination therapies and biomarker-guided strategies aimed at overcoming chemoresistance in OSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25af512c23cbe217030a9a034876fead83f7c218" target='_blank'>
              Molecular Mechanisms of Chemoresistance in Oral Squamous Cell Carcinoma: A Narrative Review with Present and Future Perspectives
              </a>
            </td>
          <td>
            E. Morais, L. Q. R. de Oliveira, Cintia Eliza Marques, H. Morais, Deborah Gondim Lambert Moreira, Lucas de Araújo Albuquerque, J. R. V. Silva, R. Freitas, Ricardo Della Coletta
          </td>
          <td>2026-01-05</td>
          <td>Applied Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22b544cab2c7f5ac6e8301325b2bf082f8d3ca21" target='_blank'>
              TRMT112 drives a tumor growth and metastasis-promoting program in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Amr R. Elhamamsy, Brandon J. Metge, M. H. Elbahoty, Bhavya Papineni, Heba M. Alsheikh, Dongquan Chen, Rajeev S. Samant, L. Shevde
          </td>
          <td>2026-01-08</td>
          <td>Cell death and differentiation</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3333bb3dc7aa46ec7dea00023d761c89be2478c2" target='_blank'>
              Large-scale single-cell analysis and in silico perturbation reveal dynamic evolution of HCC: from initiation to therapeutic targeting.
              </a>
            </td>
          <td>
            Peng Xia, Shuang Si, Daosen Fu, Luyao Liu, Dongliang Yang, Yanan Guo, Yixiao Tian, Pengfei Ji, Xinyi Yuan, Yingxia Tian, Rong Shen, Degui Wang
          </td>
          <td>2026-01-30</td>
          <td>NPJ precision oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Prostate cancer depends on androgen receptor (AR) signaling for growth, which is why androgen deprivation (castration) therapy is effective at early stages. However, many tumors eventually progress to a lethal form known as castration-resistant prostate cancer (CRPC). A subset of CRPC tumors bypass dependency on AR signaling by acquiring lineage plasticity, where prostate cancer cells transdifferentiate into alternate cellular states through epigenetic reprogramming. Neuroendocrine (NE) prostate cancer represents one well-known lineage plasticity phenotype. Nevertheless, most AR-independent tumors do not exhibit NE features and are defined as AR-negative/NE-negative or “double-negative prostate cancer” (DNPC). In a collaboration with Dr. Ekta Khurana’s computational genomics lab at Weill Cornell Medicine, we recently classified CRPC into four epigenetic subtypes, including the well-established 1) AR and 2) NE, as well as the novel DNPC subgroups 3) WNT and 4) stem cell-like (SCL) (PMID: 35617398). We focused on the SCL subtype as it is the second most common group in CRPC patients and lacks therapeutic targets. Using functional genomic approaches, we found that YAP/TAZ/TEAD cooperates with FOSL1 to drive the SCL lineage and growth of SCL models. We therefore hypothesize that the heightened dependency on the YAP/TAZ/TEAD/FOSL1 transcriptional program represents a therapeutic vulnerability in CRPC-SCL. To test this, we exposed CRPC models to TEAD inhibitors and found robust growth suppression in SCL cells compared to non-SCL cells in vitro. To evaluate whether the TEAD inhibitors are on-target, we performed transcriptomic profiling in SCL models and observed downregulation of YAP/TAZ gene signature as well as FOSL1 expression, which phenocopies the effects of YAP/TAZ double knockdown. To define the cistromes of these factors upon TEAD inhibition, we performed ChIP-seq and observed reduced co-occupancy at consensus sites, suggesting the disruption of the YAP/TAZ/TEAD/FOSL1 transcriptional circuit by the small molecule compound. To determine whether these phenotypes are recapitulated in vivo, we will treat mice harboring CRPC-SCL xenografts with TEAD inhibitors to assess growth response and evaluate epigenetic and transcriptional response using single-nucleus Multiome (ATAC+RNA). These studies will establish whether small molecule inhibition of TEAD is a promising strategy for the treatment of CRPC-SCL and allow high-resolution analysis of cell state transitions, with a focus on loss of SCL-specific signatures and potential emergence of AR/NE programs as adaptive resistance mechanisms.



 Chen Khuan Wong, Dan Li, Hongsu Wang, Marjorie Roskes, Weiling Li, Shipra Shukla, Dana Schoeps, Ekta Khurana, Yu Chen. Defining and targeting drivers of lineage plasticity in stem cell-like prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A074.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/912fdd784be0fd097e91b40d9e4c6933dd2c1b14" target='_blank'>
              Abstract A074: Defining and targeting drivers of lineage plasticity in stem cell-like prostate cancer
              </a>
            </td>
          <td>
            C. K. Wong, Dan Li, Hongsu Wang, M. Roskes, Weiling Li, Shipra Shukla, Dana M. Schoeps, E. Khurana, Yu Chen
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Somatic mosaicism is pervasively observed in human aging, with clonal expansions of cells harboring mutations in recurrently mutated driver genes. Bulk sequencing of tissues captures mutation frequencies, but cannot reconstruct clonal architectures nor delineate how driver mutations impact cellular phenotypes. We developed single-cell Genotype-to-Phenotype sequencing (scG2P) for high-throughput, highly-multiplexed, joint capture of genotyping of mutation hotspots and mRNA markers. We applied scG2P to aged esophagus samples from six individuals and observed large numbers of clones with a single driver event, accompanied by rare clones with two driver mutations. NOTCH1 mutants dominate the clonal landscape and are linked to stunted epithelial differentiation, while TP53 mutants promote clonal expansion through both differentiation biases and increased cell cycling. Thus, joint single-cell highly multiplexed capture of somatic mutations and mRNA transcripts enables high resolution reconstruction of clonal architecture and associated phenotypes in solid tissue somatic mosaicism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e453d2ebb8e890d0f12017626a25469ee8fcb34" target='_blank'>
              Genotype-to-phenotype mapping of somatic clonal mosaicism via single-cell co-capture of DNA mutations and mRNA transcripts.
              </a>
            </td>
          <td>
            Dennis J. Yuan, John Zinno, Theo Botella, Dalia Dhingra, Shu Wang, Allegra G Hawkins, Ariel D Swett, Jesus Sotelo, Ramya Raviram, Clayton Hughes, Catherine Potenski, Katharine D Godfrey, Kara M Ainsworth, Shuzhen Xu, Jianwen Que, Julian A Abrams, Akira Yokoyama, N. Kakiuchi, Seishi Ogawa, D. Landau
          </td>
          <td>2025-12-31</td>
          <td>Cancer discovery</td>
          <td>1</td>
          <td>30</td>
        </tr>

        <tr id="RNA‐targeted therapy is reshaping molecular medicine by shifting the traditional “protein‐centric” view toward an “RNA‐regulatory network” paradigm. Beyond carrying genetic information, RNA plays essential roles in posttranscriptional regulation, signaling pathways, and epigenetic modulation. Advances in high‐throughput sequencing, structural biology, and delivery technologies have accelerated the development of diverse RNA therapeutics, including antisense oligonucleotides (ASOs), small interfering RNA (siRNA), microRNA (miRNA) modulators, messenger RNA (mRNA) therapeutics, aptamers, short hairpin RNA, and CRISPR/Cas‐guided single‐guide RNAs. However, a concise comparison of these major RNA modalities and the translational barriers that limit their broader clinical application is still lacking. This review outlines the mechanisms and representative applications of these RNA‐based strategies in gene silencing, editing, protein replacement, immune activation, and targeted drug delivery. Special emphasis is placed on ASOs and siRNAs for neurological, metabolic, and infectious diseases, as well as mRNA therapeutics that are transforming vaccine development. Common challenges‐such as in vivo stability, delivery efficiency, and immune activation‐are also discussed. Finally, we highlight how chemical modification, nanotechnology, and artificial intelligence‐assisted design are enhancing the specificity, stability, and safety of RNA therapeutics, providing a framework for advancing next‐generation precision RNA medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a8a098f4a76e1439134aee84974e5e6a557a0fc" target='_blank'>
              Progress in RNA‐Targeted Therapeutics for Human Diseases
              </a>
            </td>
          <td>
            Wangzheqi Zhang, Aimin Jiang, Bin‐Kui Jia, Yuming Jin, Yinghu Chen, Zhaoyu Li, Yan Liao, Haoling Zhang, Zhiheng Lin, Xiao Fang, Linhui Wang
          </td>
          <td>2026-02-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18e77f40b4649c94b742354f1776a0273d7ae6d5" target='_blank'>
              A Hallmark-Integrated, Agent-Based Framework for Intratumor Heterogeneity in Melanoma Evolution
              </a>
            </td>
          <td>
            Khola Jamshad, Trachette L. Jackson
          </td>
          <td>2026-01-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Epilepsy is a heterogeneous neurological disorder with a strong genetic basis, yet recent evidence underscores the critical role of epigenetic mechanisms in its pathogenesis. This review synthesizes current knowledge on how chromatin remodeling, histone modifications, DNA methylation, and transcriptional regulation intersect with classical channelopathies and signaling pathways. We emphasize how epigenetic dysregulation contributes to neuronal excitability and network plasticity, particularly through interactions with mTOR, PI3K-AKT, and GABAergic signaling cascades. The convergence of genetic mutations and epigenetic modifications creates a dynamic landscape in which environmental factors can modify gene expression and contribute to the development of epilepsy. Emerging therapeutic strategies—including epigenetic drugs (HDAC inhibitors, DNMT inhibitors), CRISPR/dCas9-based epigenome editing, and multi-omics approaches—offer promising avenues for precision medicine. This review provides a comprehensive synthesis of genetic and epigenetic mechanisms in epilepsy, examining how these layers interact to produce disease phenotypes and discussing the therapeutic implications of this multilayered regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90ba558c94ca490e7d26b00448d8a20bafb88d57" target='_blank'>
              Genetic–Epigenetic Interplay in Epilepsy: Pathways, Biomarkers, and Epigenome-Targeted Therapies
              </a>
            </td>
          <td>
            A. Zaruha, P. Codreanu, Mădălin-Codruț Coman, Monica Andreea Novac II, S. Duță-Ion, I. Jugănaru, Iulian-Andrei Hotinceanu, Andra Dan, Livia-Mălina Burtavel, Anca-Elena Eftenoiu, D. Barca, A. Ionescu, Cerasela Paraschiv, V. Rădoi
          </td>
          <td>2026-02-10</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The effects of cis-regulatory alterations in prostate cancer (PCa) are insufficiently characterized, presenting an opportunity to discover driver genes and therapeutic targets. To comprehensively study these effects, we identify genes undergoing allele-specific expression (ASE) in localized PCa and metastatic castration-resistant PCa (mCRPC) samples. By defining recurrent ASE events across prostate tissue and tumor-enriched ASE, we develop CASEDI, a computational framework for prioritizing cancer drivers by integrating ASE and clinical data. CASEDI reveals genes showing recurrent ASE and altered expression in PCa, including AR-regulated and oncogenic ACSM1. mCRPC samples show enrichment of ASE in DNA repair, resistance pathways, and oncogenes and increased frequency of monoallelic expression (MAE) compared to localized tumors. We define an mCRPC gene signature based on MAE status that identifies a subgroup of localized patients with worse prognosis. Using ASE analysis, we expand the landscape of cis-regulatory events in PCa to inform the identification of additional therapeutic targets. Implications: This study develops a framework for identifying cancer drivers using prostate cancer allele-specific expression (ASE) data, generates a comprehensive dataset of ASE in prostate cancer and highlights candidate targets in tumorigenesis and metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/616d957dd08b5a8335edc5a1298ef89197981ba1" target='_blank'>
              Landscape of allele-specific expression in prostate cancer reveals recurrent, stage-specific events in AR signaling and resistance pathways.
              </a>
            </td>
          <td>
            Maggie Tsui, Kevin Hu, Hanbing Song, Sarah C Hsu, Yih-An Chen, Lorraine Nuniz, Julia H Pham, Chih-Hao Chang, Keliana S F Hui, David A. Quigley, Jingjing Li, Franklin W. Huang
          </td>
          <td>2026-01-16</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9627eeea0e72cf88ceb4ce602d15c8920df0c909" target='_blank'>
              Splicing-driven post-translational dysregulation: a new frontier for precision cancer medicine and immunotherapy.
              </a>
            </td>
          <td>
            Sael Alatawi
          </td>
          <td>2026-01-24</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="An emerging paradox in cancer metabolism is that identical oncogenic mutations produce profoundly different metabolic phenotypes depending on tissue context, with many mutations exhibiting striking tissue-restricted distributions. Here we introduce metabolic permissiveness as the inherent capacity of a tissue to tolerate, adapt to, or exploit metabolic disruptions, providing a unifying framework for explaining this selectivity. We examine tissue-specific metabolic rewiring driven by canonical oncogenes (MYC and KRAS), tumor suppressors (p53, PTEN, and LKB1), and tricarboxylic acid (TCA) cycle enzymes (FH, SDH, and IDH), demonstrating that baseline metabolic architecture, nutrient microenvironment, redox buffering, and compensatory pathways determine whether mutations confer a selective advantage or metabolic crisis. We further discuss how the tumor microenvironment shapes metabolic adaptation and therapeutic vulnerability. This framework reveals shared principles of tissue-specific metabolic vulnerability in cancer and provides a mechanistic basis for precision metabolic therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/072e3d9ea119dddf236045289b733a4bb98c1497" target='_blank'>
              Metabolic permissiveness: how tissue context shapes cancer.
              </a>
            </td>
          <td>
            Chrysanthi Moschandrea, C. Frezza
          </td>
          <td>2026-02-09</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Leukemia is a heterogeneous group of hematologic malignancies characterized by the dysfunctional proliferation of white blood cells in the bone marrow. Genetic alterations are important risk factors for the development and progression of leukemia, and their detection is crucial. Although many genetic techniques, including karyotyping, fluorescence in situ hybridization (FISH), and polymerase chain reaction (PCR), have provided valuable information, they all have the limitation of incomplete genomic coverage. The evolution of genomic technologies, including Next Generation Sequencing (NGS) and Third Generation Sequencing (TGS), has enabled a more comprehensive and detailed characterization of the genetic landscape of leukemia. NGS technology has revolutionized leukemia diagnosis, treatment, and minimal residual disease monitoring, and its integration into routine leukemia care will enhance patient outcomes and pave the way for truly personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f067f109a8a6e2615bdfa692706a81a96d161ae" target='_blank'>
              The Application of Next-Generation Sequencing in Leukemia
              </a>
            </td>
          <td>
            Konstantinos Agiannitopoulos, E. Kouvidi
          </td>
          <td>2025-12-25</td>
          <td>OBM Genetics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Methyl-donor nutrients, including folate, vitamin B12, vitamin B6, choline, betaine, and methionine, play indispensable roles in one-carbon metabolism and govern key processes such as DNA methylation, nucleotide synthesis, and genomic maintenance. Yet despite decades of research, their relationship with cancer remains paradoxical and frequently misunderstood. Much of the confusion arises from an overreliance on epidemiological studies that use cancer incidence as a late-stage endpoint, thereby obscuring how the biological actions of methyl donors differ fundamentally across the continuum from precancerous lesions to established tumors. By synthesizing evidence from mechanistic studies, precancerous lesion research, and early-stage carcinogenic models, this review suggests that adequate methyl-donor availability may be protective during the earliest phases of cancer development. However, these same nutrients may later become substrates hijacked by neoplastic cells to fuel rapid proliferation, maintain oncogenic methylation programs, and enhance tumor progression in established malignancies and high-risk populations. Therefore, this review proposes a reframing that methyl donors may not be evaluated merely as protective or harmful, but rather as context-dependent modifiers whose influence is shaped by timing, metabolic status, and the underlying biology of the target tissue. Such a shift is promising for advancing precision nutrition and the prevention or targeted suppression of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdf401262f6b61b60e458c86d532862cdfdc9412" target='_blank'>
              The Double-Edged Nature of Methyl Donors in Cancer Development from Prevention to Progression
              </a>
            </td>
          <td>
            Da Pan, Shaokang Wang, Guiju Sun
          </td>
          <td>2025-12-28</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Nonsense mutations, responsible for ~11% of gene lesions causing human monogenic diseases, introduce premature termination codons (PTCs) that lead to truncated proteins and nonsense-mediated mRNA decay (NMD). In the central nervous system (CNS), these mutations drive severe, progressive neurological conditions such as spinal muscular atrophy, Rett syndrome, and Duchenne muscular dystrophy. Readthrough therapies—strategies to override PTCs and restore full-length protein expression—have evolved from early aminoglycosides to modern precision tools including suppressor tRNAs, RNA editing, and CRISPR-based platforms. Yet clinical translation remains hampered by inefficient CNS delivery, variable efficacy, and the absence of personalized stratification. In this review, we propose a translational framework—the 4 Ds of Readthrough Therapy—to systematically address these barriers. The framework dissects the pipeline into Detection (precision patient identification and biomarker profiling), Delivery (engineered vectors for CNS targeting), Decoding (context-aware molecular correction), and Durability (long-term safety and efficacy). By integrating advances in machine learning, nanocarriers, base editing, and adaptive trial designs, this roadmap provides a structured strategy to bridge the translational gap. We advocate that a synergistic, modality-tailored approach will transform nonsense suppression from palliative care to durable, precision-based cures for once-untreatable neurological disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95770f1e8f192732dea25c73c53d310819d06b6a" target='_blank'>
              A Translational Roadmap for Neurological Nonsense Mutation Disorders
              </a>
            </td>
          <td>
            Jiaqing Li, Zhenyun Zhu, Sanqing Xu
          </td>
          <td>2026-01-30</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="BACKGROUND
Mutant IDH1 (mIDH1) defines a therapeutically-targetable subtype of intrahepatic cholangiocarcinoma (ICC), with the mIDH1 inhibitor ivosidenib approved for advanced disease. A subset of patients experiences prolonged disease stabilization, although the molecular basis for eventual progression remains poorly defined.


EXPERIMENTAL DESIGN
We performed molecular profiling of matched baseline and post-progression circulating tumor DNA (ctDNA) samples from patients with mIDH1 ICC enrolled in the ClarIDHy phase III trial. Functional studies were conducted to characterize candidate resistance mechanisms.


RESULTS
Longitudinal ctDNA analysis of 18 patients treated with ivosidenib for >6 months revealed emergent genomic alterations in multiple cases. Acquired mutations in MAPK-pathway genes (KRAS, NRAS, MAP2K1, NF1) were identified in five cases, with instances of concurrent alterations and/or high variant allele fractions (VAF). Additional candidate resistance events included a secondary IDH1 mutation and a hotspot IDH2 mutation, detected at low VAF in the same patient. Functional studies demonstrated that these IDH mutations conferred sustained 2-hydroxyglutarate (2HG) production and ivosidenib resistance, whereas MAPK activation blunted gene expression induced by ivosidenib plus interferon-γ, a key therapeutic output of mIDH1 inhibition. In parallel, baseline ctDNA profiling of 81 patients linked ARID1A mutations and elevated mIDH1 VAF to reduced clinical benefit.


CONCLUSIONS
MAPK-pathway alterations represent a recurrent mechanism of resistance to mIDH1 inhibition in ICC, while emergent IDH1/IDH2 mutations appear infrequent. Functional data suggest that MAPK-mediated resistance may involve impaired IFN signaling. These results support MAPK-directed combination strategies and highlight the utility of ctDNA profiling to identify predictive and resistance biomarkers in mIDH1-driven ICC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4aefbd8f840b00705379daf10a2c077d305123a7" target='_blank'>
              MAPK Pathway Mutations Emerge in Mutant-IDH1 Inhibitor-Resistant Cholangiocarcinoma and Attenuate the Interferon Response.
              </a>
            </td>
          <td>
            J. Wan, H. Saatcioglu, H. Ellis, E. Aguado-Fraile, Qin Xu, Hiroshi Kondo, R. Manguso, I. Gritti, B. Norden, Carolina Noble, R. Corcoran, Sam J Lubner, M. Javle, G. Abou-Alfa, James M Cleary, R. Kelley, M. Borad, T. Macarulla, Do-Youn Oh, Scott Daigle, C. Gliser, Susan Pandya, Sung Choe, A. Tron, N. Bardeesy
          </td>
          <td>2025-12-29</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>105</td>
        </tr>

        <tr id="Sickle cell anemia (SCA) is a monogenic inherited disease caused by a single point mutation in the HBB gene. Traditional treatments are ineffective in fundamentally correcting the genetic defect. In recent years, CRISPR-Cas9, an emerging gene editing technology, has shown great potential in the treatment of SCA. This article systematically reviews the mechanism of action of CRISPR-Cas9 and compares it with other editing tools. We focus on its application in SCA treatment, including strategies such as repairing the HBB gene mutation or targeting the HBG1/2 promoter to activate fetal hemoglobin. Furthermore, based on data from cell-based experiments, animal models, and clinical trials, we discuss the efficacy and limitations of representative therapies such as CTX001, Casgevy, and Lyfgenia. Results have shown that CRISPR-Cas9 can significantly improve red blood cell function in patients, and some therapies have received clinical approval, marking a significant breakthrough in its transition from laboratory research to clinical application. However, challenges remain, such as off-target effects, insufficient delivery efficiency, high treatment costs, and transplant-related complications, which need to be addressed. In the future, emerging strategies such as RNA editing are expected to expand into broader areas, including tumor treatment, while improving safety, providing safer and more accessible treatment options for monogenic diseases such as SCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/732f63e17e30e7bab825eb7a3cacdb558c0d1f25" target='_blank'>
              CRISPR-Cas9 Technology in the Treatment of Sickle Cell Anemia (SCA)
              </a>
            </td>
          <td>
            Zihao Wang
          </td>
          <td>2025-12-28</td>
          <td>Academic Journal of Science and Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 The progression of prostate cancer (PCa) to metastatic disease remains a critical clinical challenge, making it essential to gain a detailed understanding of the factors that enable certain subclones to invade and spread. Building on our previous study, which examined genomic variations in benign and malignant prostate tissues by inferring copy number alterations (CNA) from spatial transcriptomics data1, this study aims to trace the spatial evolutionary trajectories of PCa subclones from the primary tumor to matched lymph node metastases and to identify alterations in associated microenvironments and early diagnostic markers. To achieve this, we performed spatial transcriptomics on entire prostate axial disks and patient-matched lymph node metastases from 10 individuals. Standard spatial transcriptomics (55 µm) was used for tissue profiling; from the spatial transcriptomics data, copy number alterations (CNAs) were inferred to identify distinct tumor subclones, and a clone-tree was constructed to describe their evolutionary relationships. Additionally, a high-resolution spatial transcriptomics (2 µm) technology was employed to gain a cellular-level view of the immediate tumor-microenvironment (TME) surrounding specific subclones. Analysis of over 1,000,000 barcoded regions identified many distinct tumor subclones, enabling us to trace their evolutionary trajectories spatially. We observed notable subclonal events within the lymph nodes, including polyclonal colonization, indicating multiple dissemination events during the evolution of the primary disease. Exploration of the immediate TME revealed significant cellular heterogeneity and upregulation of genes related to antigen presentation and inflammatory pathways concentrated near ancestral tumor clones, specifically at the tumor border. By focusing on the inferred CNA profiles of the metastasizing clone across patients, we identified several common features. Additionally, we were able to identify potential lethal disease as early as the diagnostic core needle biopsy stage by detecting features of metastasis and pinpointing the metastasizing clone. In summary, our study provides a detailed spatial map of PCa clonal evolution and dissemination, linking primary tumors to nodal metastases and revealing altered cell composition and gene expression around tumor clone borders. Importantly, it demonstrates subclonal events within lymph nodes, polyclonal colonization, and the potential to identify metastasizing clones as early as the diagnostic phase, which holds strong implications for precision oncology and early intervention.
 1. Erickson, A., He, M., Berglund, E. et al. Spatially resolved clonal copy number alterations in benign and malignant tissue. Nature 608, 360–367 (2022). https://doi.org/10.1038/s41586-022-05023-2



 Mengxiao He, Sandy Figiel, Max Beesley, Jintong Shi, Wencheng Yin, Emmanouela Perisynaki, Richard Colling, Ian G. Mills, Joakim Lundeberg, Alastair D. Lamb. Comprehensive spatial profiling of prostate cancer metastasis: Mapping clonal evolution, microenvironment dynamics, and early metastatic markers [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr PR013.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/604426087ba878274c47234a7ce00263acf256a4" target='_blank'>
              Abstract PR013: Comprehensive spatial profiling of prostate cancer metastasis: Mapping clonal evolution, microenvironment dynamics, and early metastatic markers
              </a>
            </td>
          <td>
            Mengxiao He, S. Figiel, Max Beesley, Jintong Shi, Wencheng Yin, Emmanouela Perisynaki, Richard Colling, I. Mills, J. Lundeberg, Alastair D. Lamb
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Chronic myeloid leukemia is a hematopoietic stem cell malignancy characterized by excessive proliferation of white blood cells, primarily driven by the Philadelphia chromosome (t 9;22). Despite the availability of tyrosine kinase inhibitors, disease progression and therapy resistance remain major challenges, often linked to additional somatic mutations. Objectives: To find out the mutational status and expression variations of the ASXL1 gene hotspot region in CML patients treated by TKIs. Methods: This retrospective cross-sectional study was conducted at the Department of Zoology, Division of Science and Technology, University of Education, Lahore, Pakistan. DNA and mRNA of 50 CML patients were analyzed alongside 10 healthy controls and statistically assessed using SPSS. Results: Results showed that sequence analysis of exon 13 identified hotspot mutations, including two novel missense alterations, G659S and D667E, within the coding region of ASXL1. In-silico analysis also suggested potential structural and functional relevance of these variants. The ASXL1 gene expression showed a progressive but statistically non-significant down-regulation in CML patients across disease phases (p=0.662), disease duration (p=0.42), treatment with second-line versus first-line TKIs (p=0.412), and wild-type and mutant (p=0.544). Conclusions:  It is concluded that two novel missense alterations, G6559S and D667E, were identified in the hotspot region, and ASXL1 gene expression was downregulated non-significantly in CML patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14ee16b9a16245af7fb856280afba55f65850e27" target='_blank'>
              Identification of Novel Alterations in the ASXL1 Gene and Its Expression Profile in CML Patients
              </a>
            </td>
          <td>
            R. Mehmood, Khuram Shehzad Khan, A. Akram, Sikandar Hayat, Amjad Zafar
          </td>
          <td>2025-12-31</td>
          <td>Pakistan BioMedical Journal</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Multiple myeloma (MM) is an incurable hematologic malignancy characterized by the clonal expansion of plasma cells in the bone marrow. Despite advances in therapeutic agents, including proteasome inhibitors, immunomodulatory drugs and immunotherapies, relapse driven by treatment resistance remains a major clinical challenge. This underscores the critical need to elucidate additional molecular mechanisms that drive MM pathogenesis and therapeutic failure. The emerging field of epitranscriptomics, which studies post‑transcriptional RNA modifications, offers a promising perspective. Among these modifications, N6‑methyladenosine (m6A), the most abundant internal mRNA modification, has been implicated in regulating nearly every aspect of RNA metabolism. Growing evidence indicates that dysregulation of the m6A modification machinery plays a pivotal role in MM heterogeneity, disease progression and drug resistance. The present review synthesized current knowledge on how specific m6A regulators contribute to MM oncogenesis by modulating key signaling pathways, interactions with the bone marrow microenvironment and responses to therapy. It also discussed the potential of targeting m6A pathways as a therapeutic strategy to overcome treatment resistance and improve patient outcomes. By highlighting recent advances and future directions, the present review underscored m6A modification as an important frontier in the battle against MM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cc90def1e6d756ae29b54123b091a9ffa3e51a7" target='_blank'>
              Emerging roles of RNA m6A modification in multiple myeloma pathogenesis and treatment resistance (Review).
              </a>
            </td>
          <td>
            Yasen Maimaitiyiming, Shuoyang Hu, Die Bai, Yingchao Guan, Na Bu, Wenhui Hao, Mayila Maimaiti
          </td>
          <td>2026-02-10</td>
          <td>International journal of oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46cb1762556f83822b8fcd8773e27d5e3004fd71" target='_blank'>
              Circular RNAs: key regulators in cancer stem cell dynamics
              </a>
            </td>
          <td>
            Puneet Jain, Varsha Kaushik, Ravindresh Chhabra
          </td>
          <td>2026-02-03</td>
          <td>Molecular Genetics and Genomics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dde5fce21e0ea399ec39e2d1188eb639ee8561aa" target='_blank'>
              Assessing off-target effects in CRISPR/Cas9: challenges and strategies for precision DNA editing
              </a>
            </td>
          <td>
            Abdul Basit, Jianzhong Zhu, Wanglong Zheng
          </td>
          <td>2026-01-12</td>
          <td>Archives of Microbiology</td>
          <td>1</td>
          <td>20</td>
        </tr>

        <tr id="Lung cancer is a highly malignant disease, posing a significant threat to global health. The presence of tumor heterogeneity results in substantial variations in prognosis and therapeutic responses among patients. Advances in bulk RNA sequencing and single-cell RNA sequencing have facilitated the identification of driver gene mutations and the exploration of cellular diversity within tumors. However, tumors are complex ecosystems comprising both tumor cells and their microenvironment, where interactions among different cell types give rise to specific functional structural units that collectively drive tumorigenesis and progression. The emergence of spatial omics technologies has allowed for the analysis of tumor ecosystems, providing unprecedented insights into tumor heterogeneity. This review aims to present updates on spatial omics technologies and data analysis algorithms, discuss current technical limitations, and explore potential future developments. Furthermore, we summarize the latest applications of spatial omics in elucidating lung cancer heterogeneity, investigating mechanisms of lung cancer progression and drug resistance, and identifying novel biomarkers. Based on these findings, we propose strategies for integrating spatial omics into lung cancer research, offering new perspectives for precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eca909ba170b3407f8c473dadaa8f67180cd2619" target='_blank'>
              The Evolution of Spatial Omics Technologies Introduces A Novel Avenue for Lung Cancer Research.
              </a>
            </td>
          <td>
            Yue He, Zifan Li, Wenxiang Wang, Xu Liu, Shanshan Lu, Jing Bai, Lin Weng, Qingna Zhang, Jun Wang, Kezhong Chen
          </td>
          <td>2026-01-30</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) is defined by the clonal expansion of hematopoietic stem and progenitor cells harboring somatic mutations that confer a fitness advantage. CH is common with advancing age and becomes nearly ubiquitous in middle age. Although typically asymptomatic, CH is associated with an increased risk of hematologic malignancies particularly myeloid neoplasms (MN), diverse non-malignant conditions, and all-cause mortality. Over the past decade, research has provided major insights into the origins of CH. In addition to aging, CH is promoted by environmental exposures, inherited genetic predisposition, and acquired conditions. Large-scale population and longitudinal sequencing studies have identified determinants of clonal behavior. Characterization of the natural history of CH has enabled the development of risk stratification models to identify individuals with CH at high risk for progression to MN, thereby providing a rationale for selecting patient populations best suited for therapeutic intervention trials. Emerging strategies include targeting mutation-specific vulnerabilities, modulating inflammatory pathways, reducing genotoxic therapy-induced clonal selection, and repurposing agents with efficacy in MN. In this review, we summarize current knowledge of the risk factors underlying CH development, highlight recent advances in understanding the determinants of clonal behavior including progression to MN, and discuss emerging therapeutic approaches for preventing malignant transformation and clinical trial design considerations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e8f2ddfaf7617522bdf1484770e1e9f235abbf2" target='_blank'>
              Clonal hematopoiesis and its progression to myeloid neoplasms: insights into risk, biology, and therapeutic strategies.
              </a>
            </td>
          <td>
            Scott J Beeler, Matthew J. Walter, Kelly L. Bolton
          </td>
          <td>2026-02-05</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Chronic myeloid leukaemia (CML) is a clonal myeloproliferative cancer caused by the constant activity of the BCR–ABL 1 fusion protein' s tyrosine kinase, resulting from the Philadelphia chromosome translocation, which leads to abnormal cell growth, survival, and disease progression. While tyrosine kinase inhibitors (TKIs) have greatly improved patient outcomes, issues like drug resistance and persistent leukemic stem cells highlight the need for alternative therapies. This study used a structure- guided CRISPR- Cas 9 genome editing approach to identify highly specific single- guide RNAs (sgRNAs) that can disrupt the human ABL 1 gene, a key part of the BCR–ABL 1 fusion. The high- resolution crystal structure of the ABL 1 kinase domain (PDB ID: 8 I 7 S) helped identify essential functional regions, including catalytic and ATP- binding sites, for precise CRISPR targeting. Computational design and filtering of sgrnas were performed using E- CRISP and CHOPCHOP, focusing on criteria like PAM site accessibility, targeting conserved kinase regions in exons, GC content, predicted efficiency, and low off- target risk. In silico analyses, including specificity scores, mismatch profiles, and sequence alignment across ABL 1 transcript variants, ensured high selectivity and broad coverage. Genomic visualization confirmed accurate targeting within exons encoding vital kinase functions. Protein–protein interaction analysis via STRING showed strong links between ABL 1 and key oncogenic regulators such as BCR, STAT 5, and MAPK pathway components. KEGG pathway analysis further indicated ABL 1' s involvement in chronic myeloid leukaemia, PI 3 K–AKT, MAPK signaling, and other cancer- related pathways, emphasising its importance in CML development. This combined computational approach demonstrates that structure- guided CRISPR- Cas 9 targeting of ABL 1 can effectively disrupt BCR–ABL 1 driven cancer signals. The results provide a strong theoretical basis for future experimental validation and genome editing therapies aimed at overcoming TKI resistance and achieving long- lasting molecular remission in CML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea06e417904b01587a2a61ab158e9f0d860c3941" target='_blank'>
              Structure-guided CRISPR CAS9 Targeting of ABL1 for Functional Disruption of BCR-ABL1 Fusion in Chronic Myeloid Leukaemia
              </a>
            </td>
          <td>
            Sumita Katal, Shivangi Koundal, Uma Kumari
          </td>
          <td>2026-02-04</td>
          <td>Computational Biology and Bioinformatics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Acute myeloid leukemia-myelodysplasia related (AML-MR) is a biologically and clinically distinct subtype of AML that arises in the context of prior dysplasia. It is characterized by adverse cytogenetics and poor prognosis compared to other AML subtypes. Several genetic mechanisms underpin the pathogenesis of AML-MR; however, additional findings are likely to come to light over time with advanced genomic technologies, enhancing our understanding of their evolution. This report details a case of AML-MR involving unreported gene fusion. A 59-year-old female with multiple comorbidities presented with slurred speech. Pathological evaluation and DNA-based next-generation sequencing results were consistent with AML-MR. AML fluorescence in situ hybridization (FISH) panel revealed an extra signal for RUNX1. G-banding karyotype revealed a solitary rare t(X;21)(q26.1;q22.12) in 18 out of 20 cells analyzed. Optical genome mapping (OGM) was performed to precisely localize the breakpoints and identify the specific genes or gene fusions created by the translocation. OGM identified a novel fusion involving ENOX2 (Xq26.1) and RUNX1 (21q22.12), which was subsequently confirmed by a retrospective custom FISH probe targeting ENOX2. The identification of an ENOX2::RUNX1 fusion in AML-MR expands the spectrum of rare RUNX1 fusions. High-resolution approaches such as OGM enable precise delineation of fusion partners and breakpoints beyond the resolution of conventional cytogenetics. While the biological and clinical significance of this fusion remains to be determined, this finding highlights the value of OGM in the identification of novel and rare genomic rearrangements in leukemia and other malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a1ef03479e0b598417cb6526e465be7236182fc" target='_blank'>
              Optical genome mapping identifies novel ENOX2::RUNX1 t(X;21)(q26.1;q22.12) rearrangement in acute myeloid leukemia-myelodysplasia related: first case report
              </a>
            </td>
          <td>
            G. V. George, Sarmad Ali, Chauncey R. Syposs, M. Iqbal
          </td>
          <td>2025-12-19</td>
          <td>Molecular Cytogenetics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Retrotransposons are genomic parasites frequently reactivated in cancers, where their mobility can cause genetic alterations. However, it remains unclear whether their gene products contribute to cancer beyond mutagenesis. Here, we uncover a chromatin-associated function of RNAs from Long Interspersed Element-1 (LINE-1), the only autonomous retrotransposon in the human genome. Subcellular-resolved transcriptomics revealed that LINE-1 RNAs are primarily nascent transcripts produced by full-length, cell-type-specific genomic copies of evolutionarily young subfamilies. Using a long-read chromosome conformation assay, we identified a class of highly interactive LINE-1 loci required for gene expression across cancer subtypes, revealing an unexpected regulatory role for LINE-1 locus transcription in oncogenic gene control. LINE-1 RNA depletion disrupted LINE-1-centric chromatin interactions and downregulated associated genes, whereas genomic insertion of an inducible LINE-1 generated de novo chromatin interactions in a transcription-dependent manner. Therefore, beyond their mutagenic potential, retrotransposons also regulate cancer gene expression by nucleating chromatin architecture through their transcriptional activity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e9e6bb366cb35c566b3052102db69c89368901c" target='_blank'>
              LINE-1 Locus Transcription Nucleates Oncogenic Chromatin Architecture.
              </a>
            </td>
          <td>
            Michael Lee, Yuannyu Zhang, Jun Yi Stanley Lim, Tao Dai, James S Ye, Margaret B Cervantes, Varun Sondhi, Sisi Zheng, Yoon Jung Kim, Brandon Chen, Ralph J. DeBerardinis, Jian Xu
          </td>
          <td>2026-01-05</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Epigenetic marks such as DNA methylation, histone modifications, and non-coding RNAs play a central role in regulating gene expression without altering the DNA sequence. These dynamic and reversible modifications contribute to cellular differentiation, genomic stability, and the fine-tuning of transcriptional activity in response to environmental and metabolic cues. Beyond genetic variation alone, epigenetic mechanisms provide a critical regulatory layer linking environmental factors, cellular context, and disease susceptibility. Alterations in epigenetic marks can disrupt normal gene expression patterns, contributing to a wide range of pathological conditions, including cancer, neurodegenerative, metabolic, and immune-mediated diseases. This review places particular emphasis on cancer, cardiovascular diseases, and autoimmune disorders, where epigenetic dysregulation has been most extensively characterized and therapeutically explored. Because epigenetic modifications are reversible and influenced by environmental and lifestyle factors, they represent highly promising targets for therapeutic intervention. In recent years, epigenetic-based drugs—such as DNA methyltransferase inhibitors, histone deacetylase inhibitors, and agents targeting non-coding RNAs—have demonstrated significant potential in preclinical and clinical studies. However, their clinical application remains limited by challenges including non-selectivity, systemic toxicity, and limited tissue specificity. Emerging technologies such as CRISPR/dCas9-mediated epigenome editing offer opportunities for precise regulation of disease-associated genes without permanent genomic alterations. This review summarizes current knowledge on major epigenetic mechanisms and highlights recent advances in epigenetic therapies, including FDA-approved agents, combinatorial strategies, and next-generation epigenetic editing approaches. A deeper understanding of these mechanisms is expected to enable targeted, reversible, and patient-specific therapeutic strategies within the framework of precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e63889ba19a59ab3f3b40aa0ebd9c404687b05f" target='_blank'>
              Epigenetic marks and promising epigenetic therapies
              </a>
            </td>
          <td>
            Selin Kankaya, Yildiz Dincer
          </td>
          <td>2026-02-10</td>
          <td>Academia Molecular Biology and Genomics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Enhancers are distal cis-regulatory elements that orchestrate spatiotemporal gene expression patterns in response to developmental cues and environmental stimuli. Genetic and epigenetic alterations in enhancers are associated with the initiation and progression of human diseases, including cancers. Over the past few decades, accumulating evidence has revealed that a class of nascent RNA transcripts, known as enhancer RNAs (eRNAs), is broadly transcribed from active enhancers. These eRNA species contribute to complex and dynamic gene regulatory networks under both physiological and pathological conditions through diverse mechanisms. Notably, dysregulated eRNA expression has been reported across various cancer types and is often correlated with patient survival outcomes. Consequently, eRNAs are emerging as promising biomarkers and therapeutic targets for cancer treatment. This review provides a comprehensive summary of the current understanding of eRNAs and their mechanisms of action in gene regulation. We discuss the critical roles of eRNAs in both health and disease and highlight their diagnostic and prognostic value, as well as their therapeutic potential in cancer. Additionally, we review current strategies for targeting RNA transcripts, including eRNAs, and discuss the major challenges in developing eRNA-targeted therapies. Finally, we propose future directions for advancing eRNA-based interventions in the treatment of human diseases, including cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9397eec4991d568fddc2093214f80298c06bfee4" target='_blank'>
              Deciphering the regulatory landscape of enhancer RNAs in health and disease
              </a>
            </td>
          <td>
            Qian Wang, P. ten Dijke, Chuannan Fan
          </td>
          <td>2026-01-29</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="S i c k l e C e l l D i s e a s e ( S C D ) i s a n i n h e r i t e d hemoglobinopathy caused by a point mutation (GAG→GTG) at the 6th codon of the β-globin gene (HBB). Among traditional treatments, hydroxyurea requires lifelong administration and some patients show poor response; although hematopoietic stem cell transplantation (HSCT) can cure the disease, only 18% of patients can find HLA-matched donors, and there is also the risk of graft-versus-host disease (GVHD). The CRISPR-Cas9 gene editing technology provides a potential curative option for SCD by precisely modifying hematopoietic stem/progenitor cells (HSPCs). This article systematically reviews the clinical application strategies, experimental evidence, and existing challenges of this technology, and discusses its translational prospects in combination with the latest research findings. It covers the clinical effects of strategies such as targeting the BCL11A gene enhancer to activate fetal hemoglobin (HbF), directly correcting the E6V mutation of the HBB gene via homology-directed repair (HDR), and mimicking hereditary persistence of fetal hemoglobin (HPFH), as well as safety and accessibility challenges (e.g., off-target effects, preconditioning toxicity, and high treatment costs) and their corresponding solutions, aiming to provide comprehensive references for the research and application of gene therapy in SCD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/890a8f128bb576259bd63b951a1f15ea5bf4e0b1" target='_blank'>
              Application and Breakthrough of CRISPR-Cas9 Gene Editing Technology in the Clinical Treatment of Sickle Cell Disease
              </a>
            </td>
          <td>
            Boxun Sun
          </td>
          <td>2025-12-19</td>
          <td>MedScien</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Large-scale single-cell genomics projects have revolutionized our understanding of human immune variation. Yet most studies to date have been Eurocentric, limited in cell-type resolution, or restricted to a single data modality. The newly published Chinese Immune Multi-Omics Atlas helps address these gaps by profiling 428 healthy Chinese adults using a multiomics single-cell approach that combines single-cell RNA sequencing and single-cell chromatin accessibility sequencing across over 10 million immune cells. This integrated strategy enabled the identification of 73 distinct immune cell subsets and the construction of cell-type–specific gene regulatory networks linking noncoding enhancers to target genes. The atlas delineated hundreds of enhancer modules (eRegulons), highlighting both established and novel regulators of immune cell identity. By aligning transcriptomic and epigenomic maps, Yin et al. show how expanding both the ancestral diversity and data modalities of immune cell genomics can reveal new biology and provide a valuable addition to global reference cell atlases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95dd47b2f0f754d8765c9bb6a6fd440e0d251aa3" target='_blank'>
              Charting immune variation through genetics and single-cell genomics
              </a>
            </td>
          <td>
            J. E. Powell
          </td>
          <td>2025-12-30</td>
          <td>GigaScience</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="BACKGROUND
Neuroblastoma is the most common extracranial solid tumor in children and exhibits substantial clinical heterogeneity. Although key genetic alterations such as N-Myc proto-oncogene protein amplification and anaplastic lymphoma kinase mutations are known drivers of neuroblastoma, their clinical utility is limited by population-specific genetic diversity. To address this, we propose a "function-over-gene" strategy by evaluating extracellular matrix (ECM)-related gene sets as molecular biomarkers, aiming to overcome the limitations posed by genetic heterogeneity and provide novel prognostic insights.


METHODS
We integrated data from a single-center cohort and the TARGET database. Cox regression models were used to assess the prognostic value of ECM gene alterations and their association with clinical outcomes. Gene set enrichment analysis (GSEA) was employed to identify ECM-related gene sets, followed by transcriptomic analysis to explore downstream regulatory pathways.


RESULTS
In the single-center cohort, ECM gene mutations were potentially associated with bone and lymph node metastases and emerged as an independent predictor of poor prognosis (HR=2.7, P=0.02) in multivariate analysis. Validation using the TARGET cohort confirmed the prognostic relevance of the ECM gene set (HR=1.55, P=0.0083) and revealed its involvement in modulating the tumor microenvironment via immune and complement pathways.


CONCLUSION
ECM gene signatures serve as robust prognostic markers across populations. This function-based approach offers a novel perspective to address genetic heterogeneity and provides a theoretical foundation for ECM-targeted combination therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d011b508eee694993a004e089261b1d23283417" target='_blank'>
              Dysregulation of Extracellular Matrix Genes Identifies Neuroblastoma Patients at High Risk of Recurrence and Poor Outcome.
              </a>
            </td>
          <td>
            Xiaoyu Jing, Shijing Ge, Guo-qian He, Sijia He, Ju Gao, Chaoban Wang, Xia Guo
          </td>
          <td>2026-01-15</td>
          <td>Journal of pediatric hematology/oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Fluoropyrimidine anticancer agents, exemplified by 5-fluorouracil (5-FU), induce severe adverse effects-such as myelosuppression, emesis, diarrhea, and hand-foot syndrome-in approximately 10-30% of patients. As such toxicities can result in delays or discontinuation of therapy, accurate prediction of drug response prior to treatment initiation is of critical importance. The metabolic degradation of 5-FU is primarily mediated by the drug-metabolizing enzymes dihydropyrimidine dehydrogenase (DPD) and dihydropyrimidinase (DHPase). These enzymes are encoded by the DPYD and DPYS genes, respectively; polymorphisms in these genes that reduce or abolish enzymatic activity lead to elevated systemic concentrations of 5-FU, thereby increasing the risk of severe toxicity. In Caucasian populations, four DPYD polymorphisms have been identified as predictive markers of adverse drug reactions. However, these variants are rarely observed in Japanese individuals, and reliable pharmacogenomic biomarkers remain largely unreported in this population. To address this gap, we conducted a comprehensive in vitro functional analysis of DPD and DHPase activity of DPYD and DPYS variants identified through large-scale whole-genome sequencing databases. This review summarizes our findings and elucidates the underlying mechanisms affecting the function of 5-FU-metabolizing enzymes, as revealed by our prior research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42d05830affe95465f834cdd71289f75658fb239" target='_blank'>
              [Characterization of Genetic Polymorphisms Related to 5-FU Metabolizing Enzymes in Japanese Populations].
              </a>
            </td>
          <td>
            Eiji Hishinuma
          </td>
          <td>2026-01-01</td>
          <td>Yakugaku zasshi : Journal of the Pharmaceutical Society of Japan</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Familial cancers are caused by inherited mutations in specific genes that regulate cell growth, division, and repair. Approximately 5–10% of all cancer cases have a hereditary component, where germline mutations in certain genes increase an individual’s susceptibility to developing cancer. Two major categories of genes are involved in cancer development: tumour suppressor genes and oncogenes. Both play critical roles in regulating normal cell behaviour, and when mutated, they can contribute to uncontrolled cell proliferation and tumour formation. In addition to genetic mutations, epigenetic alterations also play a significant role in familial cancer. Epigenetics refers to changes in gene expression due to DNA methylation, histone modifications, and the dysregulation of non-coding RNAs without alter the underlying DNA sequence. Familial cancer syndromes follow various inheritance patterns, including autosomal dominant, autosomal recessive, X-linked, and mitochondrial inheritance, each with distinct characteristics. Identifying genetic mutations associated with familial cancers is a cornerstone of genetic counselling, which helps individuals and families navigate the complex intersection of genetics, cancer risk, and prevention. Early identification of mutations enables personalized strategies for risk reduction, early detection, and, when applicable, targeted treatment options, ultimately improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69f296f3ae0afec7c6382eb8b634a83d98dd9138" target='_blank'>
              Genetic Basis of Familial Cancer Risk: A Narrative Review
              </a>
            </td>
          <td>
            Eman Fares Sabik
          </td>
          <td>2026-01-13</td>
          <td>DNA</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/722d55e2fce47cc8557185b3a62a306898fce381" target='_blank'>
              Genotoxicity of cancer therapies and the risk of secondary malignancies: toward personalized prevention.
              </a>
            </td>
          <td>
            O. Molehin, Amos Tomiwa Afolabi
          </td>
          <td>2026-02-04</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Autoimmune diseases (AIDs) are chronic disorders driven by the breakdown of immune tolerance, where genetic susceptibility and environmental triggers converge to promote aberrant T and B cell activation, inflammatory cytokine release, and progressive tissue damage. Current therapeutic strategies, including corticosteroids, immunosuppressants, and biologics, can alleviate symptoms but remain limited by adverse effects, incomplete remission, and variable patient responses. CRISPR interference (CRISPRi), a novel gene regulation platform based on catalytically inactive Cas9 and guide RNAs, achieves precise, reversible suppression of target genes without inducing DNA double-strand breaks, thereby offering improved safety and controllability over conventional genome editing. Recent studies demonstrate that CRISPRi can effectively silence multiple inflammatory mediators and immune checkpoint molecules, enabling immune microenvironment remodeling and providing a feasible strategy for complex autoimmune conditions. Advances in multiplex CRISPRi, including the design of multi-sgRNA constructs and delivery via viral or non-viral systems, expand its capacity to simultaneously repress synergistic cytokine networks. Despite ongoing challenges in delivery efficiency, off-target risk, and long-term safety, CRISPRi holds considerable promise for network-level therapeutic intervention. This review aims to summarize the principles, technological progress, and potential applications of CRISPRi in autoimmune diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1138bca3d3877bd7932621af6651b3b14d329c9b" target='_blank'>
              Potential Applications and Research Exploration of CRISPRi in the Treatment of Autoimmune Diseases
              </a>
            </td>
          <td>
            Shaopu Huang
          </td>
          <td>2025-12-28</td>
          <td>Academic Journal of Science and Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Simple Summary The incidence of neuroendocrine neoplasms is increasing over the years, and therapy effectiveness is still relatively limited by the development of resistance mechanisms. Given the low rate mutation of these neoplasia, there is evidence of an emerging role of epigenetic modifications in neuroendocrine tumor pathogenesis and therapy resistance, leading to a need to investigate more deeply how these mechanisms are altered in these tumor types, how they can affect therapeutic efficacy, and if they can mediate the crosstalk between tumor cells, tumor microenvironment components, and tumor-stroma crosstalk that support tumoral development and progression. In this review we aim to describe all the recent evidence involving epigenetic mechanisms and the tumor microenvironment components that support tumor onset, development and resistance to treatment in the context of neuroendocrine neoplasm. The goal is to highlight potential new path to improve treatment response and outcomes in patients affected by neuroendocrine neoplasms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a66ba50c801aa6fd2102a018fc8123d71e68a47" target='_blank'>
              Epigenetics and the Tumor Microenvironment in Neuroendocrine Tumors
              </a>
            </td>
          <td>
            Alice Castenetto, Teresa Gagliano
          </td>
          <td>2025-12-25</td>
          <td>Cancers</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/074b0a39466afd2855b2f02374d1bb3bcc283ce0" target='_blank'>
              Natural and Artificial Hematopoietic Cell Reprogramming.
              </a>
            </td>
          <td>
            Dong Yu, Wenpei Liu, Feng Liu, Lin Cheng
          </td>
          <td>2026-02-12</td>
          <td>Cellular reprogramming</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Myelofibrosis (MF) is a myeloproliferative neoplasm (MPN) characterized by splenomegaly, constitutional symptoms, bone marrow fibrosis and potential progression to a blast phase. This review provides a comprehensive overview of the current molecular landscape of MF beyond canonical driver mutations (JAK2, MPL or CALR), emphasizing insights gained from murine models that served as valuable tools for understanding disease mechanisms. High-throughput next-generation sequencing (NGS) has markedly enhanced our understanding of the molecular basis of MF, identifying numerous mutations beyond the canonical driver genes JAK2, MPL, and CALR, which are present in about 80% of patients. Additional mutations affect genes involved in DNA methylation (TET2, DNMT3A, IDH1, IDH2), histone modification (ASXL1, EZH2), mRNA splicing (SF3B1, SRSF2, U2AF1, ZRSR2), signaling pathways (CBL, NRAS, KRAS), and key transcription factors (RUNX1, NFE2, TP53). The presence and combination of these alterations influence clinical presentation, prognosis, and therapeutic response. This review offers an updated synthesis of the evolving molecular landscape of MF, highlighting how the intricate interplay among genetic alterations has deepened our understanding of disease heterogeneity, allowing refined risk stratification and therapeutic planning. Advances emerging from molecular research and experimental models are progressively translating into clinical practice, promoting more personalized and targeted approaches to the management of MF.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfa1e50ab4c23ee2312532232e5a53b22f4357a8" target='_blank'>
              The Role and Impact of Non-driver Gene Mutations in Myelofibrosis
              </a>
            </td>
          <td>
            V. Boldrini, P. Guglielmelli, A. Vannucchi
          </td>
          <td>2026-02-10</td>
          <td>Current Hematologic Malignancy Reports</td>
          <td>0</td>
          <td>113</td>
        </tr>

        <tr id="PURPOSE
Gene fusions are molecular rearrangements with oncogenic potential. However, their role in disease progression and therapy resistance remains largely unexplored, particularly in breast cancer.


EXPERIMENTAL DESIGN
In this study, we utilized multi-omics data from the AURORA program to characterize gene fusions in metastatic breast cancer. We analyzed RNA sequencing data from 325 primary tumors and 350 metastatic lesions across 476 patients to develop a high-confidence catalog of gene fusions.


RESULTS
In a subset of 398 matched samples from 199 patients, we observed a higher burden of gene fusions in metastatic tumors compared to their corresponding primary tumors. Metastatic-specific (acquired) gene fusions were characterized by few recurrent rearrangements and were generally absent in patients with de novo metastatic disease. We observed a colocalization of gene fusions with subtype-specific copy-number gains. Various scores indicative of genomic instability were found to be associated with the gene fusion burden. Fusions involving genes located within the same topologically associating domain (TAD) were common in HER2-positive tumors and frequently acquired in metastatic triple-negative breast cancer. The presence of gene fusions, particularly those that were acquired or involved genes within the same TAD, was associated with poor prognosis in ER-positive/HER2-negative tumors. Patients with acquired ESR1 fusions exhibited a more aggressive disease course, characterized by shorter treatment response and poorer clinical outcomes.


CONCLUSIONS
The AURORA gene fusion catalog may serve as a valuable resource for identifying targetable genetic alterations, thereby supporting the development of more effective therapeutic strategies for breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b67d22cb70a54186d4c79975c42b05e6e536bd92" target='_blank'>
              Molecular and clinical analyses of gene fusions identify therapeutic targets in paired primary and metastatic breast cancer from the AURORA program (BIG 14-01).
              </a>
            </td>
          <td>
            C. Biagioni, D. Fimereli, Alexandre Irrthum, A. Guerrero-Zotano, M. Piccart, D. Cameron, C. Caballero, Alice Raimbault, D. Martins-Branco, Dario Romagnoli, E. Agostinetto, D. Venet, Arnau Llinas-Bertran, Jose A Seoane, G. Zoppoli, Ana Vivancos, Andrea Vingiani, Mafalda Oliveira, G. Curigliano, Thayane Crestani, C. Sotiriou, E. Nili Gal-Yam, F. Hilbers, Philippe Aftimos, M. Benelli
          </td>
          <td>2025-12-22</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92529c21b1d903977db7b6f99cc016a10cc689ad" target='_blank'>
              Molecular analysis of tumor recurrence using established cancer stem cell-line and drug discovery.
              </a>
            </td>
          <td>
            Kiyotaka Nakano, E. Oki, Masaki Yamazaki, Masami Suzuki, S. Kawai, Y. Zaitsu, Chiyoko Nishime, K. Ando, G. Nagae, N. Harimoto, Mitsuhiko Ota, T. Kawazoe, Kentaro Nonaka, Keita Natsugoe, Sachie Omori, H. Saeki, Hiroyuki Aburatani, Tatsumi Yamazaki, Yoshihiko Maehara
          </td>
          <td>2026-01-20</td>
          <td>International journal of clinical oncology</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a54d931016d849a45a82192eedf06a276e9b6b73" target='_blank'>
              Rescuing the Function of Missense-Mutated Tumor Suppressor VHL using Stabilizing Small Molecules
              </a>
            </td>
          <td>
            Mariam Ahmed Fouad, Christopher S. Parry, Sven A. Miller, Shipra Malhotra, Glenn A. Doyle, Ezra Shimabenga, Mashhura Nurilloeva, Elena Bondarenko, Grigorii V. Andrianov, Wayne Childers, Benjamin E. Blass, Petr Makhov, Johnathan R. Whetstine, M. Clapper, E. Golemis, John Karanicolas
          </td>
          <td>2026-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="Prostate cancer is one of the major causes of cancer morbidity and mortality in men, especially at advanced stages where treatment fails. Despite the fact that the existing treatment options, such as androgen deprivation therapy, chemotherapy, radiotherapy, PARP inhibitors, and targeted agents, can successfully manage the disease progression, resistance and recurrence are frequent. Recent evidence suggests that therapy-induced senescence is dual and context dependent in prostate cancer. Although senescence at first limits tumor cell growth by halting cellular proliferation, senescent cells may nonetheless survive the treatment and cause tumor progression, immune evasion, and therapeutic resistance through the senescence-associated secretory phenotype (SASP). In addition, the senescent tumor cells can avoid the growth arrest through epigenetic reprogramming and metabolic adaptation, re-enter the cell cycle, and develop more aggressive and recurrent disease. This is a comprehensive review of the existing body of knowledge on how standard treatments induce senescence in prostate cancer cells. We cover the biological processes of senescence entry, maintenance, and escape, and the interactions between senescent tumor cells, the tumor microenvironment, and immune modulation. Also, we discuss the role of senescence in minimal residual disease, drug-tolerant persister cells, and dormant disseminated tumor cells, with a particular focus on their role in late relapse and metastasis. Lastly, the review highlights future therapeutic approaches to tackle senescent cells, such as senolytics and senomorphics, and addresses the current challenges of toxicity, biomarker discovery, and translation into clinical use. Future endeavors in the field of therapy-induced senescence can facilitate in devising treatment plans with prudent combination of senescence-based strategies to overcome therapy resistance in prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ae5697664dff1040b844f855f0aa51560e5dfd1" target='_blank'>
              Targeting Cellular Senescence in Prostate Cancer: Molecular Landscape and Therapeutic Avenues
              </a>
            </td>
          <td>
            Haneen Hossam, Bandar Alattaibi
          </td>
          <td>2026-02-07</td>
          <td>annals of urologic oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Immune dysregulation arises from a complex interplay between genetic predisposition, environmental exposures, and dynamic epigenetic modifications that collectively shape immune cell development, signaling, and responsiveness. Over the past two decades, rapid advances in genomics and epigenomics have elucidated key mechanisms by which inherited variants, somatic mutations, DNA methylation, histone modifications, microRNAs, and chromatin remodeling contribute to abnormal immune activation or suppression. These mechanisms underlie a wide spectrum of disorders, including autoimmunity, chronic inflammation, immunodeficiency, cancer-associated immune escape, and aberrant responses to infection. This review synthesizes current knowledge on major genetic and epigenetic determinants of immune dysregulation, highlighting the molecular pathways involved, their clinical significance, and the interplay between developmental and environmental influences. Emerging insights into gene-environment interactions, including the effects of diet, stress, xenobiotics, and the microbiome on immune epigenetics, reveal new therapeutic opportunities. Understanding how diverse layers of regulation converge on immune homeostasis provides a foundation for precision medicine approaches aimed at diagnosing, predicting, and treating diseases associated with immune imbalance.

Keywords: Immune dysregulation, genetic variants, epigenetic regulation, DNA methylation, microRNAs">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe02e272c25cf60f72783170e93ac5975c2b37f8" target='_blank'>
              Genetic and Epigenetic Modifiers of Immune Dysregulation: A Comprehensive Review
              </a>
            </td>
          <td>
            Bwanbale Geoffrey David
          </td>
          <td>2026-01-07</td>
          <td>IDOSR JOURNAL OF SCIENCE AND TECHNOLOGY</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Haemoglobinopathies, including β-thalassaemia and sickle cell disease (SCD), are among the most common monogenic disorders worldwide and remain major causes of morbidity and early mortality. Historically, management of these life-altering diseases has relied on supportive treatment and symptom management and, although these treatments reduce symptoms and ease disease burden, they do not correct the underlying genetic defect. Allogenic haematopoietic stem cell transplantation (HSCT) has been the only established curative option; however, it comes with substantial risks that significantly restrict its applicability. Over the past two decades, haematopoietic stem cell (HSC) gene therapy for haemoglobinopathies has rapidly progressed from experimental proof-of-concept to approved therapies. Lentiviral gene addition approaches have demonstrated durable expression of functional β-like globin transgenes, achieving transfusion independence in β-thalassaemia patients and significant reductions in vaso-occlusive events in SCD patients. Alternative therapeutic approaches to promote HbF expression have proved to be highly successful. Gene silencing strategies targeting BCL11A have been successful clinically and, more recently, gene editing technologies such as CRISPR/Cas9 have enabled precise disruption of regulatory elements controlling γ-globin repression, leading to the approval of the first CRISPR-based therapy for SCD and β-thalassaemia. Emerging base editing technologies promise even more precise genetic modification and are advancing through clinical evaluation. Despite these advances, access to gene therapy remains restricted due to the need for highly specialised manufacturing, toxic myeloablative conditioning regimens, and high treatment costs. Ongoing improvements and adaptations in these areas are essential to ensure that gene therapies fulfil their potential as accessible, curative treatments for patients suffering from haemoglobinopathies worldwide.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/649eef2598b223310fa16262b913a4e8a3810022" target='_blank'>
              Advances in Gene Therapy for Inherited Haemoglobinopathies
              </a>
            </td>
          <td>
            Anna B. Gaspar, H. B. Gaspar
          </td>
          <td>2025-12-27</td>
          <td>Hematology Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Enzalutamide (ENZ), a next-generation androgen receptor (AR) inhibitor, is a cornerstone treatment for metastatic prostate cancer. However, resistance to ENZ inevitably develops in these patients, and the mechanisms underlying this resistance remain poorly understood. This study reveals that UFL1 is dysregulated in ENZ-resistant cells, xenograft models, and prostate tumors. UFL1 deficiency enhances prostate cancer cell resistance to ENZ both in vitro and in vivo. Mechanistically, UFL1 loss decreases METTL16 UFMylation, thereby reducing its ubiquitination level and increasing its protein stability. Additionally, METTL16-mediated m6A modification of EEF1A1 mRNA activates the m6A-IGF2BP1 axis, resulting in increased EEF1A1 protein levels and enhanced resistance to ENZ-induced apoptosis. These findings uncover a novel UFL1-METTL16-EEF1A1 signaling pathway that drives ENZ resistance, suggesting that targeting this cascade may offer a promising therapeutic strategy for overcoming ENZ resistance in prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e59ab91b0a64a781c8e28a17966a8d7d91fb1255" target='_blank'>
              Loss of UFL1 confers enzalutamide resistance of prostate tumors by regulating METTL16-mediated m6A modification of EEF1A1 mRNA
              </a>
            </td>
          <td>
            Xu Wu, Hui Gao, Qianfeng Zhuang, Feng Li, Xingliang Feng, Naiyuan Shao, Wei Zhang, Yuyang Zhang, Xiansheng Zhang
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Rituximab has profoundly improved outcomes for patients with Diffuse large B-cell lymphoma, yet the development of drug resistance continues to pose a serious clinical challenge, leading to treatment failure in 30–40% of cases. Previous studies have revealed several key resistance mechanisms underlying rituximab resistance, such as CD20 antigen expression and conformation, functional impairments in immune effector mechanisms such as antibody-dependent cellular cytotoxicity (ADCC) and complement-dependent cytotoxicity (CDC), the establishment of an immunosuppressive tumor microenvironment (TME), and the activation of intrinsic tumor cell survival pathways. Despite these advances, the resistance landscape of rituximab in diffuse large B-cell lymphoma is not fully mapped; for instance, the roles of ferroptosis and non-coding RNAs remain to be elucidated. This review will summarize these mechanisms to help provide insights that will aid in the development of novel therapeutic strategies and facilitate more personalized approaches to overcome resistance and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/463291dad96bbffedffd98a9639e5b0b55368969" target='_blank'>
              Unraveling the Landscape of Rituximab Resistance in Diffuse Large B-Cell Lymphoma
              </a>
            </td>
          <td>
            Xiaoting Liao, Ting Wei, Qingshan Li
          </td>
          <td>2025-12-25</td>
          <td>International Journal of Biology and Life Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="A large number of reports have suggested that the cancer is associated with the gene mutations and chromosomal aberrations, and thus it has been a focus of research. Today new insights are proposed by analyzing relevant literature, suggesting that the cancer is not caused by the gene mutations and chromosomal aberrations, but by the regulatory dysfunction of gene expression resulted from the nuclear damage, among which, the gene mutations and chromosomal aberrations are only an accompany phenomena also resulting from the nuclear damage. The gene mutations generally do not cause diseases, but if they do, they mainly cause the single-gene diseases. The chromosomal aberrations usually refer to the abnormalities in chromosome structure and number, often leading to cell death or inducing various chromosomal diseases. Some chronic refractory diseases, such as the cancer, Alzheimer's disease, degenerative diseases, are probably not caused by the gene mutations and chromosomal abnormalities, but rather by the abnormal regulation of gene expression and the dysfunction of DNA transcription-protein synthesis resulted from the nuclear damage. These diseases are all the diseases of nuclear dysfunction and collapse of differentiation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4826bbb55a85cfdd2750ad0771ebe3b7d95aef3d" target='_blank'>
              Differences and Pathogenicity Between the Gene Mutation and Chromosomal Aberration
              </a>
            </td>
          <td>
            Maojin Li
          </td>
          <td>2025-12-29</td>
          <td>Medical Research Frontiers</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Gallbladder cancer (GBC) is a rare but highly aggressive malignancy with poor prognosis, largely due to the pervasive challenge of therapeutic resistance. Conventional chemotherapy, targeted therapy, and immunotherapy have shown limited efficacy, as tumor cells rapidly acquire resistance through diverse mechanisms. These include drug metabolism reprogramming, enhanced DNA damage repair, signaling pathway rewiring, epigenetic and non-coding RNA regulation, programmed cell death modulation, immune evasion, and remodeling of the TME. Increasing evidence indicates that resistance in GBC is not driven by a single factor but rather by the dynamic interplay between intrinsic tumor heterogeneity and extrinsic microenvironmental influences. To overcome these barriers, emerging strategies such as multi-target combination regimens, functionalized nanodelivery systems, tumor penetration enhancers, and microbiota-based interventions have demonstrated encouraging potential in preclinical and translational studies. This review provides a comprehensive overview of the multidimensional mechanisms of resistance in GBC and highlights current and emerging strategies aimed at reversing or bypassing these processes. A deeper understanding of key resistance nodes and the rational design of cross-pathway interventions will be essential for improving therapeutic efficacy and long-term survival in patients with GBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20fd1ec6b58d8560f5f295bb0281db2b94db3bb6" target='_blank'>
              Multifaceted Mechanisms and Precision Strategies to Overcome Therapeutic Resistance in Gallbladder Cancer.
              </a>
            </td>
          <td>
            Qingliang Jiang, Gang Li, Hengyu Li, Xiaoqing Jiang, Daimin Xiang
          </td>
          <td>2026-01-01</td>
          <td>Critical reviews in oncology/hematology</td>
          <td>1</td>
          <td>18</td>
        </tr>

        <tr id="Breast cancer is a highly heterogeneous disease characterized by diverse molecular subtypes and complex pathogenesis. Recent advances in epigenetics have unveiled the crucial roles of lysine demethylases (KDMs) in modulating gene expression and chromatin dynamics, thereby influencing breast cancer progression, including metastasis, and therapeutic resistance. KDMs, which remove methyl groups from histone lysine residues, are mainly categorized into seven subfamilies (KDM1-7) based on their catalytic mechanisms and substrate specificities. Meanwhile, each subfamily exhibits distinct roles in breast cancer, ranging from transcriptional regulation and chromatin remodeling to interactions with non-histone proteins. Notably, KDMs exhibit subtype-specific functions in breast cancer. KDMs are also implicated in various hallmarks of breast cancer, including DNA damage response, cell cycle regulation, stemness maintenance, metabolic reprogramming, and modulation of the tumor microenvironment. KDMs represent promising targets for overcoming therapeutic resistance in breast cancer. Inhibitors targeting KDMs have shown potential to enhance the efficacy of endocrine therapy, chemotherapy, and targeted therapy by modulating oncogenic signaling pathways. The KDM family members are intricately involved in the molecular pathogenesis of breast cancer, offering a rich landscape for therapeutic intervention. This review summarizes the multifaceted molecular mechanisms and biological functions of KDMs in breast cancer, highlighting their potential as therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45894d19dd17aa69313c016b62abaafcaa3e9588" target='_blank'>
              Histone lysine demethylases in breast cancer: molecular mechanisms, biological functions, and therapeutic intervention
              </a>
            </td>
          <td>
            Anqi Wang, Dianjun Qi, Yi Ma, Mozhi Wang, Haoran Dong, Chenxin Wang, Yingfan Zhang, Zheyuan Zhang, Lingwei Li, Jiayi Xu, Litong Yao, Yingying Xu
          </td>
          <td>2025-12-23</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Sickle cell anemia represents a cornerstone in molecular medicine, where a single gene mutation gives rise to complex pathophysiological cascades encompassing hemolysis, vaso-occlusion, and organ dysfunction. Although conventional therapies such as hydroxyurea and blood transfusions have improved survival, they remain largely palliative. The advent of molecular diagnostics and therapeutic innovation has transformed the landscape of SCA care, ushering in an era of precision medicine. This narrative review explores the expanding repertoire of emergingbiomarkers—including inflammatory mediators, oxidative stress markers, and genetic modifiers—that provide insights into disease heterogeneity and therapeutic responsiveness. Concurrently, moleculartherapies, ranging from gene addition and CRISPR-based gene editing to RNA-targeted interventions and pharmacologic fetal hemoglobin inducers, are redefining curative potential. These advancements signal a shift from symptomatic management toward molecular correction and individualized treatment. Integrating biomarker discovery with gene-directed therapies holds the promise of achieving long-term remission and functional cure, positioning SCA at the forefront of translational hematology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d582de2f5097a85f5ccdea63221a5417404ee855" target='_blank'>
              EMERGING BIOMARKERS AND MOLECULAR THERAPIES: THE NEXT FRONTIER IN SICKLE CELL ANEMIA CARE
              </a>
            </td>
          <td>

          </td>
          <td>2026-01-01</td>
          <td>Journal of Bio Innovation</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Epigenetic editing has emerged as a promising approach in the treatment of neurological and neuropsychiatric disorders, enabling the precise and enduring modification of genes associated with these conditions. Interventions that focus on chromatin, such as programmable systems like CRISPR/dCas9, zinc-finger proteins, and TALEs linked to epigenetic effector domains, enable the modification of DNA methylation, histone modifications, and noncoding RNA control at specific loci. This work integrates current progress in understanding the epigenetic landscape of neurological neuropsychiatric disorders, highlighting the functions of DNA methylation (de novo vs maintenance, active versus passive demethylation), histone remodeling, and context-dependent gene regulation. We emphasize that the dysregulation of these processes is essential to diseases such as Alzheimer’s disease (AD), Parkinson’s disease (PD), Huntington’s disease (HD), and major psychiatric disorders. Innovative therapeutic approaches, including KRAB- and TET-based repressors, “hit-and-run” epigenome editing, and targeted noncoding RNA regulation, are analyzed alongside translational methodologies that utilize gene therapy vectors, nanoparticle delivery systems, and inducible expression mechanisms. We also examine proof-of-concept studies that demonstrate how to prevent gene expression and alter the transcriptional networks of diseased cells in living organisms. We identify current challenges, including off-target effects, delivery issues, inadequate understanding of long-term stability, and the need for reliable diagnostics, while highlighting the translational promise of combining epigenetic clearance with biogenesis and repair. This review is aimed at providing a comprehensive and critical examination of the molecular principles, therapeutic strategies, and translational obstacles associated with epigenetic editing in neurological and neuropsychiatric disorders, thereby facilitating the development of next-generation precision therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8da6771a9dde7d1eca945df355bc0ba5bae7ba5" target='_blank'>
              Epigenetic Editing in Neurological and Neuropsychiatric Disorders: Pioneering Next-Gen Therapeutics for Precision Gene Control
              </a>
            </td>
          <td>
            Hany E. Marei
          </td>
          <td>2025-12-28</td>
          <td>Molecular Neurobiology</td>
          <td>2</td>
          <td>6</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025', '2026'],
    y: [5, 8],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>