<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2026-02-02 10:01:11 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="DHFR gene amplification is commonly present in methotrexate (MTX)-resistant colon cancer cells and acute lymphoblastic leukemia. In this study, we proposed an integrative framework to characterize the amplified region by using a combination of single-molecule real-time sequencing, next-generation optical mapping, and chromosome conformation capture (Hi-C). We identified an amplification unit spanning 11 genes, from the DHFR gene to the ATP6AP1L gene position, with high adjusted interaction frequencies on chromosome 5 (~2.2 Mbp) and a twenty-fold tandemly amplified region, and novel inversions at the start and end positions of the amplified region as well as frameshift insertions in most of the MSH and MLH genes were detected. These mutations might stimulate chromosomal breakage and cause the dysregulation of mismatch repair. Characterizing the tandem gene-amplified unit may be critical for identifying the mechanisms that trigger genomic rearrangements. These findings may provide new insight into the mechanisms underlying the amplification process and the evolution of drug resistance. Sequencing a large region of DNA containing many surplus copies of genes linked to drug resistance in colon cancer cells may illuminate how these genomic rearrangements arise. Such regions of gene amplification are highly repetitive, making them impossible to sequence using ordinary methods, and little is known about how they are generated. Using advanced methods, Jeong-Sun Seo at Seoul National University Bundang Hospital in South Korea and co-workers sequenced a region of gene amplification in colon cancer cells. The amplified region was approximately 20 times the length of that in healthy cells and contained many copies of an eleven-gene segment, including a gene implicated in drug resistance. The region also contained mutations in chromosomal repair genes which would disrupt repair pathways. These results illuminate the genetic changes that lead to gene amplification and drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental and Molecular Medicine, Experimental & Molecular Medicine</td>
          <td>5</td>
          <td>50</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum
                </a>
              </td>
          <td>
            Gustavo S. França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>90</td>
          <td>46</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Recurrent endometrial cancer (EC) has limited therapeutic options beyond platinum-based chemotherapy, highlighting the need to identify exploitable molecular vulnerabilities. Tumors with high genomic instability, including microsatellite instability-high (MSI-h) or copy-number-high (CNH) ECs, rely on the ATR-CHK1 signaling pathway to tolerate replication stress and maintain genome integrity, making this pathway an attractive therapeutic target. However, acquired resistance to ATR and CHK1 inhibitors (ATRi/CHK1i) often develops, and the transcriptomic basis of this resistance in EC remains unknown. Here, we established isogenic ATRi- and CHK1i-resistant cell line models from MSI-h (HEC1A) and CNH (ARK2) EC lineages and performed baseline transcriptomic profiling to characterize stable resistance-associated states. MSI-h-derived resistant clones adopted a unified transcriptional state enriched for epithelial-mesenchymal transition, cytokine signaling, and interferon responses, while ATRi-resistant models showing additional enrichment of developmental and KRAS/Notch-associated pathways. In contrast, CNH-derived resistant clones diverged by inhibitor class, with ATRi resistance preferentially enriching proliferation-associated pathways and CHK1i resistance inducing interferon signaling. Notably, THBS1, EDN1, and TENM2 were consistently upregulated across all resistant models relative to parental lines. Together, these findings demonstrate that acquired resistance to ATRi and CHK1i in EC is shaped by both lineage and inhibitor class and provide a transcriptomic framework that may inform future biomarker development and therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a904f9e3ffa981fe8595206fc4a5bb873d2992a4" target='_blank'>
              Transcriptional Profiling Reveals Lineage-Specific Characteristics in ATR/CHK1 Inhibitor-Resistant Endometrial Cancer
              </a>
            </td>
          <td>
            Tzu-Ting Huang, Jung-Min Lee
          </td>
          <td>2026-01-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Gallbladder cancer (GBC) is an aggressive malignancy with limited therapeutic options, primarily due to the frequent emergence of resistance to gemcitabine-based chemotherapy. Uncovering molecular mechanisms underlying this resistance is essential for developing more effective treatments. Gemcitabine-resistant GBC cell lines were generated and subjected to transcriptomic sequencing to identify resistance-associated genes. A genome-wide CRISPR-Cas9 knockout screen was used to pinpoint key genetic regulators. Functional validation was performed through gene knockdown and overexpression, cell viability and apoptosis assays, colony formation, and DNA damage analysis. A high-throughput virtual screening (HTVS) approach was applied to identify small-molecule inhibitors targeting the E2F8-DNA interaction. The efficacy of selected compounds was tested in vitro and in xenograft mouse models, and further validated using patient-derived organoids (PDOs) established from primary and recurrent gallbladder cancers. The transcription factor E2F8 was identified as a driver of gemcitabine resistance via upregulation of RRM2, a gene involved in DNA repair. Knockdown of E2F8 enhanced sensitivity to poly (ADP-ribose) polymerase (PARP) inhibitors in resistant GBC cells by impairing DNA repair. HTVS yielded HIT-4, a small-molecule inhibitor that binds to E2F8 and disrupts its interaction with DNA, leading to reduced RRM2 expression. HIT-4 significantly increased apoptosis and DNA damage when combined with PARP inhibitors. In vivo and in PDO models, HIT-4 and PARP inhibitor co-treatment markedly suppressed tumor growth, extended survival, and showed minimal toxicity. This study identifies the E2F8-RRM2 axis as a key regulator of gemcitabine resistance in GBC and establishes E2F8 as a druggable target. The novel compound HIT-4, in combination with PARP inhibitors, represents a promising therapeutic strategy to overcome chemoresistance and warrants further clinical investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/addfb38a71ef32140b6545dcfdbe09fb2f12f556" target='_blank'>
              Targeting E2F8 sensitizes gemcitabine-resistant gallbladder cancer to PARP inhibitors by disrupting RRM2-driven DNA repair
              </a>
            </td>
          <td>
            Xiaojian Ni, Min Li, Wenqing Qiu, Jichang Han, Meng Yu, Shulong Zhang, Bohao Zheng, Jiaqi He, Houbao Liu
          </td>
          <td>2025-12-14</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/774d717fb446d87b8fca81f2c265377058a6eee1" target='_blank'>
              On-target mutations confer resistance to WRN helicase inhibitors in Microsatellite Unstable Cancer Cells
              </a>
            </td>
          <td>
            G. Picco, Yanhua Rao, Angham Al Saedi, Samantha J. Walker, S. Bhosle, Yang Lee, Sara F. Vieira, Maria Garcia-Casado, Gilberto Valdes Garcia, Kieron May, F. Sassi, Inigo Barrio Hernandez, Mamta Sharma, Cansu Dinçer, Theo Bell, Anastasia Kavasakali, Ray Shenje, Cuthbert Martyr, Edward J. Brnardic, Nanhua Deng, Hristina Grigorova Dimitrova, Emre Karakoc, Sandeep Sundara Rajan, Nicola Chan, Emma Hitch, Katrina McCarten, Camille Fourneaux, Zoe Hewitson, H. Lightfoot, S. Barthorpe, James P. Phelan, Phil Landis, Brian T. Jones, Diana M. Muñoz, Jay Prakash, P. Barsanti, J. Taygerly, Michael P. DeMartino, Emanuel Gonçalves, A. Bertotti, L. Trusolino, Michael A. White, Geeta Sharma, Matthew A. Coelho, J. Houseley, Benjamin Schwartz, Mathew J. Garnett
          </td>
          <td>2025-12-27</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>67</td>
        </tr>

        <tr id="Advanced prostate cancer (PCa) remains therapeutically challenging due to heterogeneous mechanisms of resistance to androgen receptor (AR)-targeting agents. While AR signaling persists in castration-resistant PCa (CRPC), emerging evidence suggests AR-independent survival pathways may contribute to therapeutic escape. This study integrates transcriptomic data and clinical profiling to dissect AR dependency and resistance mechanisms in PCa, aiming to identify subtype-specific vulnerabilities and therapeutic targets. We performed CRISPR-Cas9 screens in AR-dependent (VCaP, LNCaP, 22Rv1) and AR-independent (DU145, PC-3, WPE1-NA22, P4E6, Shmac5) cell lines to identify core essential genes. RNA sequencing data from TCGA-PRAD, Changhai, and DKFZ cohorts were integrated to define molecular subtypes using consensus clustering. Spatial transcriptomics (ST) and single-cell RNA sequencing (scRNA-seq) were employed to validate gene expression patterns in primary tumors and metastatic samples. Temporal expression dynamics were analyzed using fuzzy clustering to identify resistance mediators, with a focus on MCL1. Drug sensitivity analysis revealed that AR-dependent cells were more sensitive to MCL1 inhibitor UMI-77, and MCL1 expression was higher in Enzalutamide-resistant cell lines. Functional validation via MCL1 knockdown confirmed its role in supporting the proliferation and inhibiting apoptosis of resistant cells. CRISPR screening identified 952 shared essential genes in prostate cancer, with 157 AR-high essential signature and 130 AR-low essential signature genes. AR-high essential signature genes enriched in cell cycle/polycomb pathways, while AR-low essential signature genes correlated with oxidative phosphorylation/mTOR signaling. Consensus clustering of TCGA-PRAD data revealed three molecular subtypes (Clusters 1–3); Cluster 3 showed worst prognosis (shorter PFI/OS) and advanced clinical features (higher T/N stage, Gleason grade). External validation confirmed Cluster 3’s aggressive phenotype and independent prognostic value (meta-cohort HR = 1.98, 95% CI: 1.19–3.27). Cluster 3 signature genes were upregulated in metastatic/CRPC tissues and spatially enriched in CRPC epithelium. Notably, Cluster 3 shared essential gene expression decreased after Enzalutamide treatment, whereas AR-high essential signature genes remained stable. MCL1 emerged as a key resistance driver, demonstrating persistent upregulation in Enzalutamide-resistant cells and CRPC models. This study elucidates distinct AR dependency landscapes in PCa, revealing AR-independent survival pathways and a clinically actionable molecular subtype (Cluster 3) linked to therapy resistance. MCL1 emerges as a critical mediator of adaptive resistance, highlighting the need for combination therapies targeting both AR-driven and AR-independent programs to improve outcomes in advanced PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fcb6fe4f0aa305df9d56910a479d95b04266697" target='_blank'>
              Integrative transcriptomic profiling reveals subtype-specific therapeutic vulnerabilities and resistance mechanisms in prostate cancer
              </a>
            </td>
          <td>
            Wei Liu, Weiyu Kong, Silin Jiang, Yong Wei, Zijie Yu, Xiaoyang Cao, Jian Yang, Xiyi Wei, Luming Shen, Chao Qin, Qingyi Zhu
          </td>
          <td>2026-01-21</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Cancer cell adaptation to their physical tumor microenvironment is a key driver of malignancy. Recent experimental evolution experiments show that the soft extracellular matrix (ECM) can impose a selection pressure on genetically variable tumor populations. Over months of sustained culture, the selection pressure leads to enrichment of specific genetic variants with high fitness, but the mechanisms underlying the high fitness of these soft-selected clones are not fully understood. Here, we used a combination of RNA-seq, ATAC-seq, and RRBS-seq to compare soft-selected populations with non-selected ancestral populations cultured on soft ECM. We demonstrate that ancestral populations grown on soft ECM for short durations are characterized by a stressed cell state with low fitness marked by cell cycle arrest and distinct metabolic shifts, whereas sustained culture selects for a robust proliferative phenotype. Mechanistically, selected cells exhibit a silenced ancestral stress response through epigenetic modifications, characterized by reduced chromatin accessibility and de novo DNA methylation, including CDH1 promoter hypermethylation. This repressive landscape supports a high-fitness state defined by elevated MYBL2 and FAK levels. An in-silico mechanism-based model shows that these molecular differences, together with high YAP1 nuclear localization in soft-selected cells, are salient features of genetic clones capable of FAK upregulation. These findings uncover a coordinated genetic and epigenetic mechanism driving cancer cell evolution in mechanically soft microenvironments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4db71dfbf014de56876df2b0949f05913ac70b1" target='_blank'>
              Multi-omics investigation reveals molecular determinants of cancer cell evolution on soft extracellular matrix
              </a>
            </td>
          <td>
            Sarthak Sahoo, S. Khanna, Ting-Ching Wang, Tanmay P. Lele, M. Jolly
          </td>
          <td>2025-12-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74d7c38058905c3f246edc0fb45f9435b0439528" target='_blank'>
              Efflux pump regulation and metabolic rewiring define the ceiling of efflux-mediated drug resistance in Mycobacteria
              </a>
            </td>
          <td>
            Shiksha Sharma, Sarika Mehra
          </td>
          <td>2026-01-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Drug resistance remains a major obstacle to successful chemotherapy, leading to treatment failure and tumor recurrence. Recent studies indicate that mutations in FAT Atypical Cadherin 1 (FAT1) contribute to drug resistance in cancer cells. However, the precise role and underlying mechanisms of FAT1 in breast cancer (BC) remain insufficiently explored. Here, we conducted a comprehensive genomic and transcriptomic analysis, identifying FAT1 as a crucial tumor suppressor gene in BC. Our study demonstrates that genomic alterations in FAT1 are associated with the Wnt/β-catenin pathway activation. We further show that FAT1 loss induces cyclophosphamide (CTX) resistance and leads to the upregulation of the Wnt signaling cascade, accompanied by the accumulation of CTNNB1 transcription factors. Notably, combination therapy effectively alleviates drug resistance by suppressing the Wnt pathway. These findings highlight the critical role of FAT1 loss in mediating CTX resistance in BC and provide insights into potential therapeutic strategies targeting the Wnt pathway.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/895ec95cba808fe5124a26c08b47ffde05186f13" target='_blank'>
              Loss of FAT1 drives cyclophosphamide resistance in breast cancer via the Wnt/β-Catenin pathway
              </a>
            </td>
          <td>
            Lijing Zhong, Dongyan Cao, Chen Zheng, Liyi Zhang, Yuxuan Xu, Shasha Zhao, Xiao Liu, Guiying Wei, Gengming Niu, Heming Xu, Xuan Tang, Jingyuan Wen, Paul W R Harris, Jian Zhang, Aina He, Dongxi Xiang
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Neuroblastoma accounts for approximately 15% of all pediatric cancer-related deaths, largely due to disease relapse following intensive multimodal therapy. A critical barrier to cure neuroblastoma is the emergence of therapy-resistant tumor cells. Neuroblastoma comprises two major cell states, adrenergic (ADRN) and mesenchymal (MES), which are believed to interconvert and contribute to therapeutic resistance through lineage plasticity. To investigate the mechanisms underlying this plasticity, we subjected human and murine neuroblastoma models to repeated treatment with indisulam, a molecular glue compound that selectively degrades the splicing factor RBM39, until full drug resistance emerged. We then generated datasets from these models, including bulk transcriptomic data, ATAC-seq, and H3K27ac CUT&Tag. These data comprehensively characterize transcriptomic and epigenetic landscapes of resistant ADRN and MES neuroblastoma cell states. We present this resource to facilitate reuse by the scientific community. These datasets may support efforts to decipher lineage switching, identify regulators of therapy resistance, and discover potential therapeutic vulnerabilities in resistant neuroblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09e91ffccc5754debd09b64e59ef74200817ed97" target='_blank'>
              Transcriptomic and epigenetic profiling of neuroblastoma states in response to RBM39 degrader
              </a>
            </td>
          <td>
            Hongjian Jin, Jie Fang, Jason Myers, Shivendra Singh, Jun Yang
          </td>
          <td>2025-12-09</td>
          <td>Scientific Data</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Acute myeloid leukaemia (AML) is a genetically heterogeneous malignancy associated with poor prognosis and limited treatment options. To identify molecular programs conserved across AML subtypes and perturbations, we analysed three RNA sequencing datasets that captured venetoclax treatment under metabolic stress and the knockdown of chromatin regulators (PSPC1, JMJD1C, and RUNX1). Differential expression analysis was performed using DESeq2, followed by functional enrichment and network analyses. An independent AML cell line dataset was used to validate results. We identified a conserved 73‐gene transcriptional signature that is consistently dysregulated across perturbations, characterised by the overexpression of CDKN1A, PHGDH, and ALDH1L2, and the downregulation of MYC and E2F targets. Functional analyses implicated cell cycle arrest, metabolic reprogramming, oxidative stress responses, and suppression of proliferative and biosynthetic pathways. PSPC1 emerged as a central hub linking chromatin remodelling to metabolic adaptation. Translational validation in the TCGA‐LAML cohort revealed that higher 73‐gene enrichment scores were associated with inferior overall survival, and stratification by hub gene expression recapitulated adverse prognostic trends. Collectively, these findings define a stress‐adaptive transcriptional program conserved across diverse AML perturbations, providing mechanistic insights into the coupling of metabolism and the cell cycle, and potential therapeutic vulnerabilities. Incorporation of this 73‐gene program into patient stratification frameworks may guide biomarker‐driven therapies and combination strategies targeting metabolic and apoptotic stress responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e81b6ab2fec05cc25f828d26f849df6993fc23c" target='_blank'>
              Integrative Bioinformatics Reveals Novel Molecular Mechanisms and Therapeutic Targets in Acute Myeloid Leukaemia
              </a>
            </td>
          <td>
            Muteb Muyey Daniel, Gradel Holel Andwey
          </td>
          <td>2026-01-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Colorectal cancer (CRC) develops through evolutionary processes involving genomic alterations, epigenetic regulation, and microenvironmental interactions. While traditionally explained by the stepwise accumulation of driver mutations, contemporary evidence supports a ‘Big Bang’ model in which many early-arising clones expand simultaneously to establish extensive heterogeneity. We reviewed recent studies employing spatially resolved multi-omic sequencing of tumour glands combined with computational modelling. These approaches enable high-resolution reconstruction of clonal architecture, transcriptional states, and chromatin accessibility. Findings show that although early clonal mutations shape tumour expansion, gene expression variability can be independent of genetic ancestry and instead reflects phenotypic plasticity driven by microenvironmental cues. Epigenomic analyses identified recurrent somatic chromatin accessibility alterations in promotors and enhancers of oncogenic pathways, frequently in the absence of DNA mutations, suggesting alternative mechanisms of gene regulation. Immune-focused studies demonstrated that early silencing of antigen-presenting genes and loss of neoantigens facilitate immune escape despite active surveillance. CRC is shaped by an interplay of genome, epigenome, and immune evolution, with non-genetic mechanisms and tumour plasticity emerging as important drivers of progression and therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c00d0ede4db6fdd44f5a7c3b138a4161c7d8d0f7" target='_blank'>
              Advances in Colorectal Cancer Cell Biology and Clonal Evolution
              </a>
            </td>
          <td>
            Sopozme Toghey, Elizabeth J. Harvey-Jones, Jonathan D. Towler, Charlotte J. H. Hafkamp, Irene Y. Chong
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4eb287f4be34ac61e66b972c8521500710a820fd" target='_blank'>
              Dual inhibition of the nonsense mediated mRNA decay enhances tumour immunogenicity, drives immunoediting, and potentiates checkpoint blockade
              </a>
            </td>
          <td>
            Ivan Zadra, Davor Orsolic, Dimitri Kounis, Mario Acera, Guillermo Palou Márquez, Etna Abad, Laura Ortet, Anna Benito, Y. Kharraz, Carlos M. Martínez, Elisabetta Mereu, Fran Supek, Ana Janic
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d73e2496f48eaf63e522263faed717c4d2cc83e" target='_blank'>
              Exploring the Role of APC in Modulating Chemotherapeutic Response in Triple-Negative Breast Cancer cells
              </a>
            </td>
          <td>
            T. Nair, M. K. Vanklompenberg, Jenifer R Prosperi
          </td>
          <td>2026-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce1dd81fa69ca7596c6e2f6203643899d73e28c1" target='_blank'>
              In vivo isogenic modelling unveils a TP53-mediated relapse phenotype in T-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Stéphanie Gachet, S. Quentin, L. Hernandez, Loïc Maillard, M. Passet, R. Kim, Hugo Bergugnat, Melha Benlebna, Maxime Boy, Véronique Parietti, P. Fenaux, André Baruchel, Hervé Dombret, N. Boissel, François Sigaux, Hugues de Thé, E. Clappier, J. Soulier
          </td>
          <td>2026-01-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Oncogenic KRAS mutations drive metabolic reprogramming in pancreatic ductal adenocarcinoma (PDAC). Src-homology 2 domain-containing phosphatase 2 (SHP2) is essential for full KRAS activity, and promising dual SHP2/mitogen-activated protein kinase (MAPK) inhibition is currently being tested in clinical trials. Exploitable metabolic adaptations may contribute to invariably evolving resistance. To understand the metabolic changes induced by dual inhibition, we comprehensively tested human and murine PDAC cell lines, endogenous tumor models, and patient-derived organoids, which are representative of the full spectrum of PDAC molecular subtypes. We found that dual SHP2/mitogen-activated protein kinase kinase (MEK1/2) inhibition induces major alterations in mitochondrial mass and function, impacts reactive oxygen species (ROS) homeostasis and triggers lipid peroxidase dependency. Anabolic pathways, autophagy and glycolysis were also profoundly altered. However, most strikingly, mitochondrial remodeling was evident, persisting into a therapy-resistant state. The resulting vulnerability to the induction of ferroptotic cell death via the combination of vertical SHP2/MEK1/2 with glutathione peroxidase (GPX4) inhibition was largely independent of the PDAC molecular subtype and was confirmed with direct targeting of RAS. The triple combination of SHP2/MEK1/2 inhibition and the ferroptosis-inducing natural compound withaferin A suppressed tumor progression in an endogenous PDAC tumor model in vivo. Our study offers a metabolic leverage point to reinforce RAS pathway interference for targeted PDAC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ece64da9d6dcfbff96e0c83595844cf2b2f5547" target='_blank'>
              Vertical RAS pathway inhibition in pancreatic cancer drives therapeutically exploitable mitochondrial alterations
              </a>
            </td>
          <td>
            P. Hafner, S. Keller, Xun Chen, Asma Alrawashdeh, H. Jumaa, Friederike I. Nollmann, S. Besson, Judith Kemming, Oliver Gorka, T. Das, Bismark Appiah, Ariane Lehmann, M. Li, Petya Apostolova, Bertram Bengsch, R. Zeiser, S. Tholen, Oliver Schilling, Olaf Groß, Andreas Vlachos, U. Wittel, Dominik von Elverfeldt, W. Reichardt, M. Boerries, G. Andrieux, Guus Heynen, S. Fichtner-Feigl, Luciana Hannibal, D. Ruess
          </td>
          <td>2026-01-16</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07c7ce9a407bf29f4b537e0cbead424db44e5f61" target='_blank'>
              Targeting glucocorticoid-induced CD20 activation in preclinical models of B-ALL
              </a>
            </td>
          <td>
            J. Bigot, Mathieu Bouttier, Vincent Fregona, Clémence Rouzier, Manon Bayet, Sylvie Hebrard, N. Prade, Stéphanie Lagarde, Christine Didier, Laetitia Largeaud, F. Vergez, A. Quillet-Mary, Loic Ysebaert, C. Broccardo, André Baruchel, Marion Strullu, E. Clappier, Marlène Pasquet, Elodie Lainey, Eric Delabesse, Bastien Gerby
          </td>
          <td>2025-12-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Mutations in tumour suppressor p53 are frequently implied in aggressive progression and metastasis in colorectal cancer. But the distinct phenotypes exhibited by site-specific mutations of p53 are not well elucidated. Here, we investigate the epigenetic and transcriptional impact of three major p53 hotspot mutations (R175H, R273H and R282W), through DNA methylation changes and single cell transcriptomics. We observed that the p53 R273H mutation is associated with a partial epithelial–mesenchymal transition (pEMT) and increased metastatic progression. Analysis of DNA methylation patterns revealed a distinct epigenetic landscape in R273H-mutant tumours, with hypomethylated regions correlating with enhanced transcriptional activation of YAP/TAZ target genes thus promoting pEMT and EMT-like phenotype in CRC tumours. In vitro ChIP-seq experiments in colorectal cancer cells expressing the R273H mutant p53 (HT29) showed enrichment of mutant p53 at the promoters of YAP/TAZ target genes suggesting EMT/pEMT like states with R273H mutation. Further, simulations from a gene regulatory network incorporating the interactions of p53R273H with EMT regulators explain how this mutation shapes the phenotypic landscape accessible to cancer cells. Our analysis of single-cell transcriptomes of colorectal tumours reveals R273H-linked enrichment of partial and mesenchymal EMT phenotypes across tumour subpopulations in CRC. We identified a distinct epigenetic signature associated with the p53 R273H mutation, characterised by hypomethylation of YAP/TAZ signalling genes that drives partial EMT and aggressive tumour behaviour. These findings highlight the importance of mutation-specific epigenetic regulation in shaping colorectal cancer progression and the need for developing therapeutic strategies tailored to p53 mutation status.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89ad1c85957c8764e20b8788eabb8e8a09e388db" target='_blank'>
              Gain of function p53 mutant R273H confers distinct methylation profiles and consequent YAP/TAZ signaling mediated activation of partial or full EMT states to colon tumours
              </a>
            </td>
          <td>
            H. Rani, Seemadri Subhadarshini, M. Jolly, V. Mahadevan
          </td>
          <td>2025-12-13</td>
          <td>Epigenetics & Chromatin</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Immune dysfunction in cancer is enacted by multiple programs, including tumor cell-intrinsic responses to distinct immune subpopulations. A subset of these immune evasion programs can be systematically recapitulated through direct tumor-immune interactions in vitro. Here, we present an integrated, high-throughput single-cell CRISPR screening framework focused on the protein kinome for mapping the tumor-intrinsic regulation of T cell-driven immune pressure in glioblastoma (GBM). We combine pooled CRISPR interference and activation (CRISPRi/a) with immune-matched NY-ESO-1 antigen-specific allogeneic GBM-T cell co-culture and massively multiplexed single-cell transcriptomics to systematically quantify how genetic perturbation reshapes baseline tumor state and adaptive responses across graded effector-to-target ratios. We further leverage deep generative models for analyzing pooled CRISPR screens to decipher the effects of genetic perturbations on the mechanisms of tumor resistance. This framework resolves distinct modules of immune evasion and survival, including the regulation of the antigen-presentation machinery, interferon/NF-κB signaling, oxidative stress resilience, and checkpoint/cytokine programs, while identifying perturbations that reroute the continuous tumor transcriptional trajectory induced by T cell engagement. A secondary chemical screen in patient-derived GBM cultures identified putative kinase targets of immune evasion phenotypes (e.g., EPHA2 and PDGFRA), whose inhibition leads to the blockade of evasive programs and enhances T cell-mediated GBM killing. Together, this workflow provides a scalable blueprint for comprehensive charting of the genetic control of tumor-immune interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e76a058bed6b1c1e4d91cd692a05e66fab6beee" target='_blank'>
              Mapping kinase-dependent tumor immune adaptation with multiplexed single-cell CRISPR screens
              </a>
            </td>
          <td>
            Lingting Shi, Ross M. Giglio, Qingyuan Cai, Mathini Vaikunthan, Justin Hong, Abdullah Naqvi, Marina Milea, Hannah Khanshali, Anna Schoonen, Nicholas Hou, Jonathan Guo, Melanie Fraidenburg, Xumin Shen, Seth Malinowski, K. L. Ligon, Raúl Rabadán, Elham Azizi, José L. McFaline-Figueroa
          </td>
          <td>2026-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Abstract Genetic instability is a hallmark of cancer, often arising from mutations to DNA damage repair and response (DDR) genes. Classical genetic, biochemical, and structural approaches elucidated the foundational mechanisms of DDR pathways and provided a scientific understanding of their involvement in repair of lesions induced by broad classes of DNA-damaging agents (DDAs). However, given the chemical diversity of DDAs and resultant DNA lesions, along with the multitude of interconnected DDR factors, the chemogenomic landscape of DDA–DDR interactions remains incompletely mapped. To this end, we developed a DDR-targeted, CRISPR knockout screening approach and assessed relationships amongst 353 DNA repair genes and 15 DDAs in LN229 glioma cells. Within this dataset of 5295 DDR-related chemogenomic interactions, we identified many established interactions and discovered novel ones. For example, we observed a specific role of transcription-coupled nucleotide excision repair in the repair of adducts generated by monofunctional alkylating agents, a role for the Fanconi anemia pathway in addressing methyl lesions, overt differences in DSB repair following treatment with topoisomerase I versus II poisons, and repair dependencies associated with the imidazotetrazines temozolomide, mitozolomide, and KL-50. Future directions will continue to investigate the mechanisms of novel chemogenomic interactions that we have uncovered as well as work to identify chemogenomic interactions amenable to clinical translation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/018770119a2496ee321bc85a8d0504634bbf54c9" target='_blank'>
              Targeted CRISPR knockout screening identifies known and novel chemogenomic interactions between DNA damaging agents and DNA repair genes
              </a>
            </td>
          <td>
            Collin D. Heer, James L. Elia, Vijay Menon, Spenser S. Johnson, Sofia R. Arbelaez, Sam Friedman, Francesc Lopez-Giraldez, R. Sundaram, Seth B Herzon, R. Bindra, Susan E. Gueble
          </td>
          <td>2026-01-09</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b10c93153ed6d7f62167729a4cbefd8ea1ed9de" target='_blank'>
              Adaptive regulation of dNTP homeostasis confers osimertinib resistance in EGFR mutant non-small cell lung carcinoma
              </a>
            </td>
          <td>
            Qian Xie, Yingying Wang, Yi Lei, Anthony Fernandez, Jessica D. Hess, Lin Yang, Terence Williams, Li Zheng, Binghui Shen, Min Li
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Despite advances in cancer therapies, treatment failure from resistance and recurrence remains a major clinical challenge. Therapy-induced senescence (TIS), a state of stable cell cycle arrest with sustained metabolic activity, has emerged as a driver of inflammation, tumor persistence, and relapse. However, the heterogeneity of TIS complicates its detection and targeting. Here, we applied a multi-modal strategy to characterize metabolic alterations in senescent cancer cells induced by doxorubicin or γ-irradiation across three tumor cell lines: MCF7, HeLa, and TPC-1. Mitochondrial dysfunction was assessed using MitoTracker and JC-1 staining, while two-photon excitation fluorescence (TPEF) microscopy enabled label-free visualization of intracellular NAD(P)H and FAD distribution. Lipid remodeling was evaluated by MALDI mass spectrometry imaging, and RNA sequencing was performed on control, senescent, and engulfing-senescent MCF7 cells to identify differentially expressed genes and enriched pathways. Senescent cells displayed mitochondrial dysfunction, with altered NAD(P)H/FAD distribution and decreased membrane potential. TPEF confirmed redistribution of coenzymes, reflecting redox changes. Lipidomics revealed consistent remodeling, notably involving cardiolipin precursors. Transcriptomic profiling showed engulfing-senescent MCF7 cells possess a distinct signature marked by increased lipid metabolism, endocrine signaling, and cell–cell communication. Overall, our findings reveal conserved and cell type–specific metabolic traits of TIS, highlighting metabolic vulnerabilities for senolytic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4f815c5ea5ff8465ffcedac66d5a28bbf1f8878" target='_blank'>
              Metabolic profiling of therapy-induced senescent cancer cells via TPEF, MALDI-MS, and RNA-sequencing
              </a>
            </td>
          <td>
            S. Ghislanzoni, F. Padelli, Matteo Niero, A. Bertolotti, A. Belfiore, Simone Torelli, A. Bresci, Andrea Masella, Silvia Betti, Dario Polli, Luca Agnelli, I. Bongarzone
          </td>
          <td>2025-12-17</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Immune aging is being increasingly recognized as a critical barrier to effective cancer immunotherapy, as the aged tumor microenvironment (TME) drives T cell dysfunction and impairs immune control of cancer1–4. However, the key molecular drivers of this process as well as potential targets to rescue T cell dysfunction in aged tumors remain incompletely understood. Therefore, we performed in vivo single-cell CRISPR screens in CD8⁺ T cells within aged tumors and tumor-draining lymph nodes (tdLNs). We identified Dusp5 and Zfp219 as key regulators of T cell persistence and effector differentiation in aged hosts. Loss of Dusp5, a negative regulator of ERK signaling, increased ERK1/2 phosphorylation and enhanced T cell proliferation in both young and aged tumors. In contrast, loss of Zfp219, a transcriptional repressor, induced epigenetic reprogramming of cytotoxic gene programs, thereby increasing granzyme secretion and enhancing antitumor immunity. Moreover, expression of the human ortholog gene ZNF219 is increased within intratumoral CD8⁺ T cells in older cancer patients. High ZNF219 expression correlates with poorer survival following immune checkpoint blockade (ICB) and reduces persistence of human intratumoral T cells. Notably, Zfp219 ablation synergized with anti-PD-1 blockade in mice to expand effector-like CD8⁺ T cells, leading to significantly enhanced anti-tumor immunity and tumor clearance in aged hosts. Together, these findings highlight Dusp5 and Zfp219 as critical drivers of age-related T cell dysfunction and as potential therapeutic targets to rejuvenate T cell antitumor immunity in older cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/017b5a7413706a4c2c7fbdc538a3a98539127c42" target='_blank'>
              CRISPR screens identify targets to rescue age-related T cell dysfunction in cancer
              </a>
            </td>
          <td>
            Alex C. Y. Chen, Keely Ji, Cansu Yerinde, Nelson H. Knudsen, Kevin Bi, Daniela Martinez, Thomas J Carmona-LaSalle, Kazuhiro Taguchi, Katherine H Xu, Elizabeth M. Seider, Marc A. Schwartz, Maria Zschummel, Linda T. Nieman, Kathleen B. Yates, T. R. Mempel, R. Manguso, N. Hacohen, Debattama R. Sen
          </td>
          <td>2026-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>118</td>
        </tr>

        <tr id="Chronic lymphocytic leukemia (CLL) is the most common leukemia in Western countries, and B-cell receptor (BCR) pathway inhibitors such as idelalisib and ibrutinib are currently established therapies for CLL. Although effective, these drugs frequently lead to resistance, but the mechanisms are still not fully understood. Activation-induced cytidine deaminase (AID) is a B-cell enzyme essential for antibody diversification. However, it can also introduce off-target mutations, leading to genomic instability. This study investigates whether treatment with BCR pathway inhibitors increases AID activity in CLL and whether this activity contributes to the development of drug resistance. Peripheral blood samples from CLL patients were collected before and after treatment with idelalisib or ibrutinib. Targeted sequencing was used to identify mutations in known AID off-target genes. Concurrently, AID-wild type (AID-WT) and AID-knockout (AID-KO) CLL cell lines were established and subsequently exposed to escalating doses of BCR pathway inhibitors to develop drug-resistant models. In patient samples, treatment with BCR pathway inhibitors was associated with an increase in AID-dependent mutations in off-target genes, including BCL2, MYC, and IRF8. The in vitro models efficiently recapitulated the patients’ data, as only AID-WT CLL cells accumulated mutations in the same AID off-target genes after drug exposure. However, no mutations were detected in genes that could mediate drug resistance. We conclude that BCR pathway inhibitors enhance AID mutational activity in CLL, but this does not appear to be directly involved in driving drug resistance. AID-targeted loci may nonetheless serve as biomarkers for monitoring genomic instability during treatment and inform further study.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1309ce3a611716e263bdb953a559c6610a8cb773" target='_blank'>
              Evaluation of the Role of AID-Induced Mutagenesis in Resistance to B-Cell Receptor Pathway Inhibitors in Chronic Lymphocytic Leukemia
              </a>
            </td>
          <td>
            Chiara Pighi, Alessandro Gasparetto, E. Genuardi, Jianli Tao, Qi Wang, C. Vitale, V. Griggio, R. Piazza, S. Ciavarella, M. Coscia, S. Ferrero, A. Zamò, C. Voena, Roberto Chiarle
          </td>
          <td>2025-12-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Glioblastoma (GBM) remains the most common and lethal adult malignant primary brain cancer with few treatment options. A significant issue hindering GBM therapeutic development is intratumor heterogeneity and plasticity. GBM tumors contain neoplastic cells within a fluid spectrum of diverse transcriptional states. Identifying effective therapeutics requires a platform that predicts the differential sensitivity and resistance of these states to various treatments. Here, we develop scFOCAL (Single-Cell Framework for -Omics Connectivity and Analysis via L1000), to quantify the cellular drug sensitivity and resistance landscape. Using single-cell RNA sequencing of newly diagnosed and recurrent GBM tumors, we identify compounds from the LINCS L1000 database with transcriptional response signatures selectively discordant with distinct GBM cell states, and leverage this capability to predict combination synergy. We validate the significance of these findings in vitro, ex vivo, and in vivo, and identify a combination of an OLIG2 inhibitor and Depatux-M for the treatment of GBM. Our studies suggest that scFOCAL identifies cell states that are sensitive and resistant to targeted therapies in GBM using a measure of cell and drug connectivity, which can be applied to identify new synergistic combinations. Glioblastoma is characterised by high levels of intratumoural heterogeneity and plasticity, hindering treatment. Here, the authors develop an analytical framework, scFOCAL, to predict the sensitivity of glioblastoma cell subpopulations to therapies based on reversal of disease transcriptional signatures to identify synergistic therapeutic combinations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ef55e3b53c83b7969434910c5b4838c55ce9672" target='_blank'>
              Drug and single-cell gene expression integration identifies sensitive and resistant glioblastoma cell populations
              </a>
            </td>
          <td>
            Robert K. Suter, Anna M. Jermakowicz, Rithvik Veeramachaneni, Matthew D’Antuono, Longwei Zhang, Rishika Chowdary, Simon Kaeppeli, Madison Sharp, Pravallika Palwai, Vasileios Stathias, Grace Baker, Luz Ruiz, Winston Walters, Maria Cepero, Danielle M. Burgenske, Edward B Reilly, Anatol Oleksijew, Mark G. Anderson, S. Williams, Michael E. Ivan, R. Komotar, Macarena I. de la Fuente, Gregory Stein, Alexandre Wojcinski, Santosh Kesari, J. Sarkaria, Stephan C. Schürer, Nagi G. Ayad
          </td>
          <td>2026-01-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="Highlights What are the main findings? We established and characterized a unique pair of isogenic cervical cancer cell lines (AdMer35/AdMer43) from a patient with squamous cell carcinoma of the cervix before and after chemoradiotherapy, revealing that resistance is not caused by a single factor but by complex reorganization of cancer cells. Single-cell analysis identified key resistance hallmarks: transcriptional reprogramming towards embryonic stemness (HOX, POU5F1), a shift from fibrillar to non-fibrillar collagen expression, and activation of interferon/inflammatory pathways. What are the implications of the main findings? The study maps a remodeled tumor ecosystem with a metabolically reprogrammed, pro-tumorigenic senescent-like cell population and a progenitor-like cell reservoir, identifying them as critical therapeutic targets for eliminating resistant cell pools. The identified key pathways, Wnt, TGFβ, BMP, and the specific ligands EREG and SEMA3C, along with the metabolic vulnerabilities of the senescent compartment, provide a robust framework for future therapeutic exploration. Abstract Cervical cancer remains a significant global health burden, with chemoradioresistance representing a major obstacle to successful treatment. To elucidate the mechanisms underlying this resistance, we established a unique pair of isogenic primary cervical cancer cell lines, AdMer35 and AdMer43, obtained from a patient with squamous cell carcinoma of the cervix before and after radiation therapy. The aim of our study was to characterize the transcriptomic and cellular heterogeneity of these cells. We conducted an in-depth comparative analysis using single-cell RNA sequencing. Analysis of this paired, patient-derived isogenic model suggests that chemoradioresistance can arise through coordinated multilevel cellular adaptations. Resistant AdMer43 cells demonstrated transcriptional reprogramming, with the upregulation of embryonic stemness factors (HOX, POU5F1, SOX2), a shift in extracellular matrix from fibrillar to non-fibrillar collagens, and activation of inflammatory pathways. We identified and characterized critical cell-state dynamics: resistant cells exhibited a remodeled ecosystem with a metabolically reprogrammed senescent-like cell population showing an enhanced pro-tumorigenic communication via EREG, SEMA3C, BMP, and WNT pathways. Furthermore, we identified a progenitor-like cell population with a minimal CNV burden, potentially serving as a reservoir for tumor persistence. These findings offer novel insights for developing targeted strategies to eliminate resistant cell pools and improve cervical cancer outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ccd2a5a0a730a290acb2a66420d2bdad799661d" target='_blank'>
              Single-Cell Transcriptomic Landscape of Cervical Cancer Cell Lines Before and After Chemoradiotherapy
              </a>
            </td>
          <td>
            Dmitriy V. Semenov, Irina S. Tatarnikova, A. S. Chesnokova, Vadim A. Talyshev, M. Zenkova, E. Logashenko
          </td>
          <td>2026-01-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d29acb3217c08dff3be7e7d47281f950232e5a4" target='_blank'>
              Identification of an epigenetically and phenotypically distinct peritumoral glioblastoma cell population linked to inferior patient outcome
              </a>
            </td>
          <td>
            I. Neves, Xi Lu, Veera Jokinen, Francesco Latini, N. Maturi, A. Sundström, Yonglong Dang, Irem Uppman, Tobias Bergström, Pengwei Xing, M. Ryttlefors, Xingqi Chen, F. Swartling, L. Uhrbom
          </td>
          <td>2026-01-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f09bfb27dbaed264c5656346c24d74a25e18488" target='_blank'>
              An Atlas of Extrachromosomal DNA Structures Illuminates Its Evolution and Biogenesis in Cancer
              </a>
            </td>
          <td>
            Chee Hong Wong, Kuo-Chen Wei, Kuan-Yu Kung, Li-Hao Yang, Yin-Cheng Huang, Chiung-Yin Huang, I-Jiuan Chung, Meng-Fan Huang, Pei-Hua Peng, Meihong Li, C. Ngan, Kou-Juey Wu, Chia-Lin Wei
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd5ef1091f544d26b38d75e4b4f01629d3fcc603" target='_blank'>
              VARS1 fuels prostate cancer evolution via codon-selective translational rewiring
              </a>
            </td>
          <td>
            Qinju He, Rongrong He, Yingying Li, Miaomiao Xu, Shuchao Ren, C. Zou, Zhenyu Li, Wenchao Li, Yuanzhen Zhang, Lanxin Hu, Bin Xu, Baobing Zhao, Zhipeng Zhou, Dingxiao Zhang
          </td>
          <td>2025-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf830ded86f6aa48fcef1356b0883f8c090233e1" target='_blank'>
              Patterns of hypermutation shape tumorigenesis and immunotherapy response in mismatch-repair-deficient glioma
              </a>
            </td>
          <td>
            Nick Fernandez, Yuan Chang, N. Nunes, J. Dimayacyac, A. Levine, Amit Ringel, L. Negm, A. B. Ercan, J. Hess, Olfat Ahmad, Caitlin Lee, L. Stengs, V. Bianchi, M. Edwards, Sheradan Doherty, J. Chung, L. Nobre, Julie Bennett, A. Dodgshun, David T. W. Jones, S. Pfister, A. Villani, David Malkin, Vijay Ramaswamy, A. Huang, É. Bouffet, M. Aronson, Peter B. Dirks, A. Shlien, Gaddy Getz, Y. Maruvka, B. Ertl-Wagner, Cynthia E. Hawkins, Anirban Das, U. Tabori
          </td>
          <td>2025-12-22</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19e1efee6c0706220f7b18885b5b5a4638edf605" target='_blank'>
              A single-nucleus multimodal framework reveals epigenomic priming of chemoresistant states in ovarian cancer
              </a>
            </td>
          <td>
            Y. Landais, J. Bertorello, M. Puerto, Agathe Kahn, B. Simon, A. Roehrig, Adeline Durand, M. Quatredeniers, C. Lamy, E. Laas, Nicolas Pouget, Simon Dumas, J. Féron, Virginie Fourchotte, C. Bonneau, Vincent Cockenpot, Marianne Delville, Magali Richard, M. Kamal, F. Lécuru, C. L. Tourneau, C. Vallot
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d440f96f3272f5794b9ae0c0eebed83d3b36971" target='_blank'>
              Direct detection of CRISPR mutations and transcriptional responses at single cell resolution in vivo
              </a>
            </td>
          <td>
            John A. Hawkins, Siamak Redhai, Svenja Leible, Mireia Osuna Lopez, Hilal Ozgur, Tianyu Wang, Michaela Holzem, M. Boutros, Oliver Stegle
          </td>
          <td>2025-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4fd78fdc6e7168376ba52f3c22a1f28b8371667" target='_blank'>
              Transcriptomic profiling of epigenetic regulators and metabolic reprogramming in human cholangiocarcinoma
              </a>
            </td>
          <td>
            Amaya López-Pascual, Jasmin Elurbide, Emiliana Valbuena-Goiricelaya, M. Latasa, Elena Anaya, Elena Adán-Villaescusa, Borja Castelló-Uribe, L. A. Martínez-Pérez, I. Uriarte, M. Arechederra, Sergio Ciordia, Fernando Corrales, Sona Frankova, Eva Sticová, Ondřej Fabián, L. Colyn, Patricia Inacio, Robert Arnes-Benito, J. Bayo, Meritxell Huch, C. Berasain, M. G. Fernández-Barrena, Matías A. Avila
          </td>
          <td>2025-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d6260543e09115d78b28611e811c1654d5ea522" target='_blank'>
              Mutant ASXL1 Drives Transcriptional Activation and Repression in Human Hematopoiesis
              </a>
            </td>
          <td>
            Madison L. Hall, Aliya Quintal, Samantha Worme, Thai T. Nguyen, Zachary Schonrock, Mitsuhiro Tsuchiya, Sonia N. Acharya, Hanqian L Carlson, Itallia V. Pacentine, Shawn B Shrestha, Jommel Macaraeg, Randall Armstrong, James McGann, Sara Evans-Dutson, Theresa A. Lusardi, Brendan L. O’Connell, Andrew C. Adey, G. Yardımcı, Julia E. Maxson, Theodore P. Braun
          </td>
          <td>2026-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Molecular heterogeneity in head and neck squamous cell carcinoma (HNSCC) is well recognized, yet existing subtype frameworks remain largely descriptive and have not translated into therapeutic decision-making. Here, we establish a mechanistic platform that converts transcriptomic diversity into drug-actionable tumor states. Integrating multi-cohort RNA-seq from 727 tumors across five independent datasets, genome-scale CRISPR dependency maps, and pharmacologic screening, we define distinct tumor survival circuits across HPV-negative HNSCC and nominate subtype-matched therapeutic strategies. These circuits encompass a proliferative axis (MYC, MET/FAK, inflammatory and translational programs), an epithelial-differentiated/adhesion program, an EMT-like state with stromal activation, and mitochondrial/oxidative metabolic states, each mapping to selective liabilities (e.g., mitotic/autophagy control, ERBB/PI3K and cadherin signaling, OXPHOS/mitochondrial translation, and G2/M-integrin-Notch pathways, respectively). We then develop a transcriptomic predictor of EGFR-inhibitor response using machine learning and validate it in prospectively collected, fresh patient-derived 3D microtumors. The resulting 13-gene signature identifies erlotinib-responsive tumors (R = 0.93) and maps biologically to an epithelial-differentiated state, outperforming EGFR expression alone. Our study establishes a subtype-to-dependency-to-therapy framework, enabling precision stratification and providing a clinically feasible path for prospective biomarker deployment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a29377070a0d6fb125a75d4b7cf89275cb55faab" target='_blank'>
              Subtype-Specific Dependencies and Drug Vulnerabilities Enable Precision Therapeutics in Head and Neck Cancer
              </a>
            </td>
          <td>
            Joel M Vaz, Songli Zhu, Mateo Useche, Lauren Shih, Emily Marchiano, Slobodan Beronja, Bruce E Clurman, Cristina P. Rodriguez, Brittany Barber, Marina Chan, Taranjit S. Gujral
          </td>
          <td>2025-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Muscle-invasive bladder cancer (MIBC) treatment depends on histological subtypes. While urothelial carcinoma (UC) benefits from novel therapies, options beyond radical cystectomy for rare subtypes such as squamous cell carcinoma (SCC) remain limited. Since we previously demonstrated ATR inhibitor (ATRi) enhanced radiation sensitivity in vitro, we aimed to further decipher the therapeutic impact of ATRi and compensatory pathways bypassing ATRi-resistance in patient-derived ex vivo cultures (PDCs). PDCs (p-SCC, p-UC, n = 6) were established and characterized by immunohistochemistry, qPCR, and whole-exome sequencing. Independent ATRi-resistant cell models (p-SCCATRi) were generated through long-term ATRi treatment (Ceralasertib) and characterized by multi-dimensional profiling. Drug responses were analyzed via cell viability (IC50) and clonogenic survival assays ± ionizing radiation (IR). DNA repair capacity was measured via γH2AX immunofluorescence, comet assays, qPCR, and immunoblotting. ATR siRNA knockdown and ATRi short-term studies validated the ATR-RAD51 axis. RAD51 inhibitor (RAD51i) B02 was tested in p-SCCATRi by cell cycle analysis and in ovo tumor growth of chorioallantoic membrane (CAM) xenografts, complemented by apoptosis staining. ATRi treatment sensitized cells to IR, reducing IC50 values up to 2.5-fold (at 8 Gy: 0.52 µM in SCC, 0.82 µM in UC). Clonogenic assays and γH2AX staining confirmed impaired DNA repair (γH2AX foci at 8 Gy: 11-fold in SCC, 15-fold in UC). In resistant p-SCCATRi models, ATRi-adaptation triggered various compensatory and potentially epigenetic regulated DNA repair pathways, particularly homologous recombination (HR) repair involving genes like BRCA1 and RAD51. Downstream consequences of functional ATR loss also affected non-DNA repair processes such as cell cycle, chromatin reorganization, and immunomodulation. As a therapeutic strategy, RAD51i overcame resistance by lowering IC50 by 40–80%, increasing DNA damage (2.2-fold γH2AX foci), and inducing G2/M arrest (2.4-fold). Finally, in ovo, RAD51i significantly induced apoptosis impairing tumor growth in p-SCCATRi xenografts by up to 37%. Our results propose ATR as a promising target in bladder cancer by (1) enhancing radio-sensitivity through classical ATR inhibition and (2) exploiting resistant ATRi-adaptation as a vulnerability by targeting compensatory HR repair reliance through RAD51 inhibition. These findings on the ATR–HR axis suggest novel strategies to improve bladder cancer treatment and addressing therapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1ecb3f8d7d2eb227866ebe473b957e2ec2ac743" target='_blank'>
              Targeting ATR offers multifaceted treatment strategies involving RAD51-mediated compensatory DNA repair in bladder cancer
              </a>
            </td>
          <td>
            Julia Pannhausen, A. Chughtai, Cem-Louis Yüce, M. Melzer, Yanchun Ma, L. Seillier, E. P. C. Van Der Vorst, G. Andrieux, J. Wirtz, Sophie Leypold, Mark P Kühnel, Per Hoffmann, S. Heilmann-Heimbach, M. Boerries, Alexander Kleger, Matthias Saar, Michael J. Eble, Danny D Jonigk, N. Gaisa, Michael Rose
          </td>
          <td>2025-12-12</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Understanding the molecular mechanisms underlying T-cell acute lymphoblastic leukemia (T-ALL) is essential for developing more effective therapeutic strategies. Despite therapeutic advances, the role of RNA-binding proteins in the pathogenesis of T-ALL remains poorly understood. Here, we investigate the RNA-binding protein Quaking (QKI), identifying it as a key regulator of splicing with tumor-suppressive properties in T-ALL. Through the analysis of two independent pediatric T-ALL cohorts, we demonstrate that QKI expression is frequently reduced in T-ALL, particularly within the HOXA subtype, and this reduction correlates with poor overall and event-free survival. Using T-ALL cell lines, we show that QKI depletion induces widespread splicing alterations, with numerous events corroborated in patient samples. Transcriptome profiling indicates that QKI downregulation leads to broad changes in gene expression, notably affecting pathways related to cell cycle progression, cholesterol homeostasis, and epithelial-mesenchymal transition. Functional assays demonstrate that QKI overexpression in T-ALL cells significantly reduces cell proliferation, induces G0/G1 cell cycle arrest, and limits leukemia progression and dissemination, ultimately improving survival in xenograft models. Together, these findings provide compelling evidence that QKI functions as a regulator of RNA splicing with tumor-suppressive activity in T-ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca5ec1e411749bba69b71a539ea37c30a49dc190" target='_blank'>
              QKI dysregulation induces extensive splicing changes in T-cell acute lymphoblastic leukemia.
              </a>
            </td>
          <td>
            Bruno Palhais, Nitesh D. Sharma, Igor Fijalkowski, T. Pieters, D. Deforce, F. Nieuwerburgh, Pieter Mestdagh, Panagiotis Ntziachristos, K. Matlawska-Wasowska, P. V. Vlierberghe
          </td>
          <td>2026-01-15</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ed06fb384ee497b6337be9253bfa34f2d9608de" target='_blank'>
              The sensitivity of acute myeloid leukemia to CDK8/19 inhibitors is determined by their metabolic profile
              </a>
            </td>
          <td>
            Varlamova Ekaterina Antonovna, Andreeva Daria Vladislavovna, Dalina Alexandra Alexandrovna, Misnik Elena Vladimirovna, Pavlenko Natalia Germanovna, Bruter Alexandra Vladimirovna, Tatarskiy Victor Vyacheslavovich
          </td>
          <td>2025-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The BCL-2 inhibitor venetoclax (VEN) has emerged as an important therapeutic backbone for hematological malignancies, but secondary resistance is a major challenge. In acute lymphoblastic leukemia (ALL), early clinical trials promise high efficacy as well. However, relapse is observed frequently, and so far, only a few resistance-inducing mechanisms have been discussed. We employed KMT2A-rearranged ALL cell lines and xenograft models to elucidate mechanisms of VEN resistance. Targeted DNA and mRNA sequencing, single-cell mRNAseq, as well as protein expression analyses were conducted. All models initially responded well but finally displayed secondary resistance. Novel as well as previously known pathogenic variants in tumor suppressor TP53 as well as pro-apoptotic molecule BAX, but not BCL2, were observed. Gene and protein expression studies demonstrated multifarious changes in resistant cells, with high inter- and intra-model heterogeneity. Finally, single-cell RNA sequencing revealed a likely contribution of the tumor microenvironment in the development of VEN resistance, as indicated by modulation of genes involved in cell-cell interaction and humoral signaling in resistance-specific clusters. Our data demonstrate and characterize the rise of VEN resistance in KMT2A-rearranged ALL models, suggesting that relapse must be expected in the clinical setting and that multifactorial processes are involved in this process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11ad7042e90c4e9abc4265e52191b4c0932e542a" target='_blank'>
              Venetoclax resistance in preclinical KMT2A-rearranged acute lymphoblastic leukemia models is characterized by high inter- and intra-model heterogeneity
              </a>
            </td>
          <td>
            Anna Richter, Lea Kinsky, Sandra Lange, N. Trakooljul, F. Hadlich, Anett Sekora, Gudrun Knuebel, S. Krohn, Christian Schmidt, Michelle Busch, Tim Schreiber, Simone Kumstel, K. Wimmers, H. Murua Escobar, Christian Junghanss
          </td>
          <td>2025-12-29</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/336dc5f971c5d12ee9a5e10fb186f6e15e48f01e" target='_blank'>
              TET2 and TP53 Mutations Cooperatively Modulate the Response to Inflammation to Promote Leukemic Transformation
              </a>
            </td>
          <td>
            Pablo Sánchez Vela, Brianna Kelly, Claudia Brady, J. Foox, J. Glass, R. Koche, Matthew Wereski, Isabelle Csete, Sebastian El Ghaity-Beckley, Bridget K. Marcellino, R. L. Levine, A. Shih
          </td>
          <td>2026-01-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Abstract Flap endonuclease 1 (FEN1) is a structure-specific nuclease with critical functions in DNA replication and repair. FEN1 has been proposed as an anti-cancer drug target because of its synthetic lethal interaction with homologous recombination deficiency (HRD). However, the exploration of FEN1 in this context has been hampered by the lack of suitable small molecule tools. Here, we describe MSC778, a highly potent, specific and selective small molecule inhibitor of FEN1 nuclease activity. MSC778 directly engages FEN1 in cells, enhances its retention on chromatin, and selectively kills BRCA-deficient cells. Mechanistically, MSC778 suppresses DNA replication causing S-phase accumulation, DNA damage, and ultimately, cell death. Cancer cell panel screening identified Ewing sarcoma (EWS) cells as sensitive to MSC778, which is driven by the expression of SLFN11. In addition to HRD, CRISPR screening revealed a spectrum of synthetic lethal interactions between MSC778 and DNA damage response (DDR) factors, such as PARP1, USP1, PARG, and ATR. Furthermore, we demonstrate that combined inhibition of these factors with MSC778 induces synergistic killing of cancer cells. Together these data highlight FEN1 inhibition as an attractive precision oncology strategy either as monotherapy or as a combination therapy with a broad range of current and next generation DDR-targeting agents.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d720ebfea477ebcfaccacc0ad325d640a78165dc" target='_blank'>
              Mechanistic insights into the monotherapy and combination potential of FEN1 inhibition in cancer therapy
              </a>
            </td>
          <td>
            E. Rajendra, Claudio A Lademann, Bethany Mason, Balca R. Mardin, Lucy Armstrong, Silvia Peripolli, Julian Kreis, Timothea Konstantinou, U. Pehl, J. Bomke, Stefanie Scharrelmann, David Perera, B. Leuthner, Maria Filipa Pinto, Owen A Davis, A. Galbiati, Ana Toste Rêgo, Claire L McWhirter, E. Elinati, Ada Sala-Hojman, Julien Lefranc, Sam E Mann, Robert A Heald, F. Zenke, Graeme C M Smith, Helen M R Robinson, L. Burgdorf
          </td>
          <td>2025-11-26</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f796949ca22238ba707f7eb2bf4bbb66b809b4f" target='_blank'>
              Single-cell whole-genome sequencing reveals convergent evolution in Burkitt lymphoma
              </a>
            </td>
          <td>
            A. Steemers, M. V. van Roosmalen, Rico Hagelaar, L. Trabut, M. Verheul, Siem Jongsma, J. D. de Kanter, F. Meyer-Wentrup, R. van Boxtel
          </td>
          <td>2025-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a29acdf562d4b29c5b781b23b586561ef03a93de" target='_blank'>
              Pan-cancer analysis reveals genomic fidelity and evolution of patient-derived organoids
              </a>
            </td>
          <td>
            Linoy Raz, Tal Ben-Yishay, Haia Khoury, Ron Saad, Amjad Zoabi, Giulia Orlandi, S. Donzelli, Giovanni Blandino, U. Ben-David
          </td>
          <td>2025-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is a highly heterogeneous malignancy with a poor prognosis and limited response to current therapies. Cancer stem cells (CSCs) contribute to this heterogeneity, driving tumor progression, immune evasion, and therapeutic resistance. Epigenetic regulators have emerged as pivotal modulators of CSC phenotypes. The histone methyltransferase SETDB1, a key stemness factor in both embryonic and adult stem cells, catalyzes H3K9 trimethylation (H3K9me3) and acts as a major oncogenic driver in a wide range of solid and hematological malignancies. Yet, its specific role in maintaining stemness and contributing to therapeutic resistance in HCC remains poorly defined. Integrative analyses of bulk and single-cell transcriptomic datasets from public HCC cohorts were performed. Tumors were stratified by SETDB1 expression, followed by pathway enrichment, immune deconvolution, and stemness scoring. Associations with clinical outcomes, immune phenotypes, drug resistance signatures, and chromatin-binding partners (DNMT3A, TRIM28, HDAC1) were also explored. The role of SETDB1 was then examined in vitro, where a stem-like phenotype was induced in HCC cell lines under hypoxic conditions, and patterns of SETDB1 expression and stemness marker levels were assessed following paclitaxel treatment. SETDB1-low tumors were enriched in well-differentiated, immunologically active subclasses and were associated with increased predicted responsiveness to immune checkpoint blockade (ICB), sorafenib, and transarterial chemoembolization. Conversely, SETDB1-high tumors exhibited transcriptional features of dedifferentiation, elevated stemness scores, and enrichment in aggressive molecular subtypes. These tumors also displayed higher TP53 mutation rates, reduced immune infiltration, immune exclusion signatures, and activation of pathways linked to ferroptosis and ABC transporter dysfunction, consistent with resistance to ICB and conventional therapies. SETDB1 expression was also associated with hypoxia-related pathways, suggesting a role in maintaining CSC niches. In vitro, hypoxia-induced stemness coincided with increased SETDB1 levels, whereas paclitaxel treatment decreased both SETDB1 expression and stemness markers. A chromatin risk score incorporating SETDB1 and its partners predicted poor disease-free survival independently of clinical parameters. SETDB1 defines a stemness-enriched, immune-resistant HCC subtype associated with poor outcomes and therapeutic failure. Correlations between SETDB1 expression and stem-like features in vitro suggest a potential role in maintaining CSC phenotypes, supporting its relevance as a biomarker and candidate for epigenetic- and CSC-directed therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c393716a837e5b1ef0091b6385eb92c64be4ab4" target='_blank'>
              The epigenetic regulator SETDB1 as a key component of cancer stem cells and drug resistance in primary liver cancer
              </a>
            </td>
          <td>
            Maël Padelli, Christophe Desterke, Aurore Devocelle, Georges Uzan, Antoinette Lemoine, Julien Giron-Michel
          </td>
          <td>2026-01-06</td>
          <td>Cellular Oncology (Dordrecht, Netherlands)</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7247a45454e42ff687acafe0ee75660b9dd942d" target='_blank'>
              A modular lentiviral system for multiplexed gene perturbation and functional analysis reveals interdependence of hormone receptors in breast cancer growth in vivo
              </a>
            </td>
          <td>
            Seher Alam, Daria Matvienko, Johan Boström, Maria J. Pires, Aikaterini Papadimitropoulou, Saeed Eshtad, Kumar Sanjiv, Hong Qian, Nicholas C. K. Valerie, C. Brisken, Mikael Altun
          </td>
          <td>2025-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="The topoisomerase I inhibitor topotecan is an effective chemotherapeutic agent for retinoblastoma; however, treatment resistance remains a major clinical challenge, and its mechanisms remain elusive. Using genome-wide CRISPR-Cas9 knockout screening, we identified ELF2 as a key gene involved in topotecan resistance. Here, we show that surviving retinoblastoma cells exposed to topotecan showed progressively decreased ELF2 expression, accompanied by reduced apoptosis. In a mouse xenograft model, ELF2 disruption diminished the antitumor efficacy of topotecan, with ELF2-knockout cells exhibiting reduced topotecan-induced apoptosis. RNA sequencing further revealed that the MT-CYB pathway, associated with ATP synthesis, contributes to ELF2-mediated resistance. Importantly, clinical analysis demonstrated a correlation between ELF2 expression and tumor volume in retinoblastoma patients treated with topotecan. Together, these findings interrogate the mechanisms underlying topotecan resistance in retinoblastoma and suggest ELF2 as a potential therapeutic target to overcome drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dffc85985364a8e0c2b935cb6a150df86f47c5ef" target='_blank'>
              Loss of ELF2 drives topotecan resistance in retinoblastoma revealed by genome-wide CRISPR-Cas9 screening
              </a>
            </td>
          <td>
            Jingyi Jiang, Zihua Jiang, Qian-Jiang Luo, Xi Chen, Jiejie Zhuang, Jiaxin Chen, Qingqing Mu, Jin Qiu, Yan Li, Shuxia Chen, Ping Zhang, Keming Yu, Shuilian Chen, Guei-Sheung Liu, Zhuang Jing
          </td>
          <td>2025-12-23</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Canine osteosarcoma (OS) is an aggressive bone cancer with remarkable similarity to human OS. Despite its value as a comparative oncology model, the drivers of genomic instability remain poorly defined, hindering cross-species insights into disease progression. We applied long-read PacBio HiFi whole genome sequencing to matched tumor and blood samples from four purebred dogs to generate high-resolution maps of somatic structural variants (SVs). Tumor genomes showed extensive rearrangement complexity, with satellite repeat regions, particularly SAT1_CF, enriched near somatic SV breakpoints and occurring within genomic contexts marked by focal hypomethylation. We also identified multiple extrachromosomal DNA (ecDNA) elements carrying oncogenes indicating ecDNA-mediated oncogene amplification. Together, these data provide an integrated genomic and epigenomic view of canine OS, implicating satellite repeats as recurrent substrates for rearrangement and ecDNA as a prominent mode of genome remodeling. This work advances our understanding of OS genome dynamics and establishes a comparative framework for investigating repeat-driven instability and oncogene amplification across species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5300af19b1916e9abe3db71ff1fa34ffceb5e24d" target='_blank'>
              Satellite DNA fragility accompanies complex genome rearrangements and ecDNA oncogene amplification in canine osteosarcomas
              </a>
            </td>
          <td>
            F. Yilmaz, Wonyoung Kong, Sabriya A. Syed, Francis H. O’Neill, Jody T. Lombardi, P. K. Ng, Ching C. Lau, Charles Lee
          </td>
          <td>2025-12-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Oncocytic (Hürthle cell) carcinoma of the thyroid (OCT) is characterized by widespread loss of heterozygosity (LOH), mitochondrial accumulation and recurrent mitochondrial DNA mutations leading to impairment of complex I. Here, we establish and characterize a novel OCT cell line, UT946, which displays severe mitochondrial electron transport chain dysfunction and a Warburg metabolic phenotype. Using a series of cytoplasmic hybrids, we establish that the complex I defect in UT946 stems from a nuclear-encoded loss of function mutation in the complex I subunit NDUFS1. To our surprise, the mutation in NDUFS1 was inherited as a recessive germline allele that underwent LOH in the tumor to expose functional loss of complex I. A re-analysis of 91 OCT tumor genomes revealed that LOH-driven exposure of recessive germline mutations in complex I subunits was a recurrent mechanism underlying complex I inactivation in OCT. These findings unveil a new germline-driven mechanism of complex I loss and metabolic reprogramming in cancer, and provide further evidence of the strong selective pressure for complex I impairment in OCT. Teaser Germline mutations in complex I induce aerobic glycolysis in oncocytic carcinoma of the thyroid through somatic loss of heterozygosity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ec415c1026a19fc77b2f19d7d97c4e52057a5cf" target='_blank'>
              Loss of heterozygosity exposes germline mutations in complex I and drives Warburg metabolism in oncocytic carcinoma of the thyroid
              </a>
            </td>
          <td>
            Celia de la Calle Arregui, A. Frank, Kelvin Mun, Jiwoong Kim, Kuntal Majmudar, Justin A. Bishop, C. Lewis, Yang Xie, David G. McFadden
          </td>
          <td>2025-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Colorectal cancer (CRC) progression and therapy resistance are driven in part by metabolic reprogramming and the persistence of cancer stem-like cells (CSCs). The seven in absentia homolog 2 (SIAH2)/with-no-lysine kinase 1 (WNK1) signaling axis has emerged as a potential regulator of these processes, yet its functional role in CRC metabolism and tumor–stroma crosstalk remains incompletely understood. Integrated analyses of The Cancer Genome Atlas–Colon Adenocarcinoma (TCGA-COAD) and Gene Expression Omnibus (GEO, GSE17538) datasets revealed significant upregulation of SIAH2 and WNK1 in CRC tissues, with strong positive correlations to glycolysis- and hypoxia-associated genes, including PFKP, LDHA, BPGM, ADH1A, ADH1B, and HIF-1α. Single-cell and clinical profiling further demonstrated preferential enrichment of SIAH2 in undifferentiated, stem-like tumor cell populations. Functional studies across multiple CRC cell lines showed that SIAH2 silencing suppressed proliferation, clonogenic growth, tumor sphere formation, and cell-cycle progression, whereas SIAH2 overexpression exerted opposite effects. Seahorse extracellular flux analyses established that SIAH2 promotes glycolytic capacity and metabolic flexibility. At the protein level, SIAH2 regulated glycolytic enzymes and WNK1/hypoxia-inducible factor-1α (HIF-1α) signaling, effects that were amplified by cancer-associated fibroblast (CAF)-derived conditioned medium. CAF exposure enhanced SIAH2 expression, CSC spheroid growth, and resistance to fluorouracil, leucovorin, and oxaliplatin (FOLFOX) chemotherapy, whereas SIAH2 depletion effectively abrogated these effects. Collectively, these findings identify the SIAH2/WNK1 axis as a central metabolic regulator linking glycolysis, CSC maintenance, and microenvironment-driven therapy resistance in CRC, highlighting its potential as a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb269b1f5618b40199d4b3cc75e161d29b2f08d9" target='_blank'>
              SIAH2–WNK1 Signaling Drives Glycolytic Metabolism and Therapeutic Resistance in Colorectal Cancer
              </a>
            </td>
          <td>
            Kee-Thai Kiu, , Yi-Chiao Cheng, Min-Hsuan Yen, Ying-Wei Chen, N. W. Pikatan, Vijesh Kumar Yadav, Tung-Cheng Chang
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="ABSTRACT Aims Mitochondrial DNA copy number (mtDNA-CN) is associated with several age-related chronic diseases and is a predictor of all-cause mortality. Here, we examine site-specific differential nuclear DNA (nDNA) methylation and differential gene expression resulting from in vitro reduction of mtDNA-CN to uncover shared genes and biological pathways mediating the effect of mtDNA-CN on disease. Materials and methods Epigenome and transcriptome profiles were generated for three independent human embryonic kidney (HEK293T) cell lines harboring a mitochondrial transcription factor A (TFAM) knockout generated via CRISPR-Cas9, and matched control lines. Results We identified 2924 differentially methylated sites, 67 differentially methylated regions, and 102 differentially expressed genes associated with mtDNA-CN. Integrated analysis uncovered 24 Gene-CpG pairs. GABAA receptor genes and related pathways, the neuroactive ligand signaling pathway, ABCD1/2 gene activity, and cell signaling processes were overrepresented, providing insight into the underlying biological mechanisms facilitating these associations. We also report evidence implicating chromatin state regulatory mechanisms as modulators of mtDNA-CN effect on gene expression. Conclusions We demonstrate that mitochondrial DNA variation signals to the nuclear DNA epigenome and transcriptome and may lead to nuclear remodeling relevant to development, aging, and complex disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcbea560ce049511e5c017437908e0cfbb595232" target='_blank'>
              Mitochondrial DNA copy number reduction via in vitro TFAM knockout remodels the nuclear epigenome and transcriptome
              </a>
            </td>
          <td>
            Phyo W. Win, Julia Nguyen, Elly H. Shin, T. S. Nagano, Brent Selimi, Katie Hong, A. Meybodi, Bradley P Yates, Emma V Burke, David E Carter, Gregory A Newby, Charles E Newcomb, D. Arking, C. Castellani
          </td>
          <td>2025-12-23</td>
          <td>Epigenomics</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b82ec4f52026d1218d8a25c579cd530acc17905" target='_blank'>
              Widespread remodeling of the RNA editome underlies transcriptional and clinical heterogeneity in pediatric acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Sara Rosenthal, Thomas Whisenant, Roman Sasik, Maria Rivera, Geena Il Defonso, Charles-Alexandre Roy, Sabina Enlund, R. Shraim, Kathleen M. Fisch, D. Teachey, Frida Holm, D. Kuo, Qingfei Jiang
          </td>
          <td>2025-12-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/638cde0ab64b6b068ce716defb37f6b2cb4ab7b9" target='_blank'>
              Unraveling the molecular mechanisms underlying spontaneous multipolar mitosis through CIN-seq
              </a>
            </td>
          <td>
            Pin-Rui Su, Ting-Chun Chou, M. T. López-Cascales, Junfeng Hunag, Tsai-Ying Chen, Sjoukje van Wieren, Chan Li, Cecile Beerens, Li You, J. Storteboom, M. Chien
          </td>
          <td>2026-01-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Alternative splicing (AS) is one of the principal mechanisms of post-transcriptional regulation that confers transcriptomic plasticity and proteomic diversity in cancer, thereby enabling tumor adaptation to therapeutic pressure. However, two obstacles impede the translation of these findings into clinical benefit: The absence of systematic functional annotation of the numerous splice variants associated with drug resistance and the paucity of biomarkers capable of distinguishing de novo from acquired splice-mediated resistance. In the present review, the current mechanistic understanding of AS-driven drug resistance was briefly synthesized, and it was evaluated how existing strategies address these challenges. It was also described how knowledge of dysregulated splicing networks, due to mutations in cis-regulatory elements such as ESS, overexpression of trans-acting factors such as SRSF1, as well as mechanisms such as alternative trans-splicing, in which the spliceosome interacts with splice sites on two distinct RNA molecules and which can be driven by complementary sequences or other trans-acting factors, could be used to more accurately identify tumors dependent on aberrant splicing for survival. In addition, it was outlined how targeting aberrant splice variants to overcome therapeutic resistance can be achieved, such as through spliceosome inhibition (for example, H3B-8800) or antisense oligonucleotides directed to a specific exon or splice junction (for example, targeting exon 2 of MET, which is implicated in cis-regulated AS isoforms, or alternatively spliced isoforms of BCL2L1, BRAF and CDYL). However, therapeutic strategies to target adaptive resistance mechanisms such as AS remain limited, as intratumoral heterogeneity may facilitate the emergence of resistant subpopulations, and as most spliceosome inhibitors are not spliceosome-specific, they exhibit off-target effects. Importantly, it was also discussed how pan-cancer splicing databases and single-cell isoform expression profiling can be integrated with deep-learning models, thereby informing the design of therapeutic strategies to overcome splicing-mediated adaptive drug resistance. Notably, such integration will enable the rational design of isoform-specific combination regimens to dismantle drug-resistance circuits. It is anticipated that the present review will assist the scientific community, including both basic and translational researchers, in translating these findings into interventions that mitigate therapeutic failure in recalcitrant cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1f0892d48c3453d0a0ca7ac5b857f619a850559" target='_blank'>
              Alternative splicing in cancer drug resistance: Mechanisms and therapeutic prospects (Review)
              </a>
            </td>
          <td>
            Wenpu Zhu, Zikun Wu, Chai Luv, Banghe Bao, Weijie Zhao, Guanglong Chen, Hang Yang, Huihan Ai, Faqiang Liu, Zhi Li
          </td>
          <td>2025-12-05</td>
          <td>Oncology Reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The tumour suppressor TP53 (tumor protein p53) is a master regulator of cell cycle, DNA repair, and apoptosis, and its mutation is a hallmark of cancer, with individual mutations exerting distinct effects on tumour biology. Despite accounting for ~7% of all TP53 variants, splice site mutations remain the least studied class, and their functional and clinical consequences are poorly understood. We analyzed 25,058 TP53 variants (18,562 somatic; 6496 germline) to characterize the frequency, molecular impact, transcriptional effects, genomic instability, and clinical outcomes of splice mutations. These alterations showed distinct distributions and substitution patterns between germline and somatic contexts and were frequently associated with copy number alterations, reduced TP53 mRNA, and variable protein expression. Transcriptomic profiling identified two transcriptional phenotypes: one with global suppression of canonical p53 target genes and another with mixed activation and repression independent of tumour type. Genomic instability was elevated in a subset of splice-mutant tumours, correlating with increased relapse risk, while other splice mutations showed lower instability but divergent clinical outcomes, including unexpectedly poor prognoses. Our findings fill a critical knowledge gap, defining the biological and clinical spectrum of TP53 splice site mutations and highlighting their potential as prognostic biomarkers and therapeutic targets in precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9df743368c74812062cf79c30615a68f562fc982" target='_blank'>
              TP53 Splice Mutations Have Tumour-Independent Effects on Genomic Stability and Prognosis: An In Silico Study
              </a>
            </td>
          <td>
            Apeksha Arun Bhandarkar, Noah Ethan Kelly-Foleni, D. Sarkar, Aaron Jeffs, T. Slatter, Antony Braithwaite, Sunali Y Mehta
          </td>
          <td>2025-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e50ce208a78ff6653e59e288e486f934aca2d39" target='_blank'>
              Three-Dimensional Epigenome Roadmap of Human B-cell Differentiation Uncovers Mechanisms of Humoral Immunity and Oncogenesis
              </a>
            </td>
          <td>
            Raúl de Haro-Blázquez, Laureano Tomás-Daza, Lucía Fanlo-Escudero, Paula López-Martí, Nicolás Byrne-Álvarez, Llorenç Rovirosa, Blanca Valero-Martínez, Juan Ochoteco, Maria Rigau, Galina Medvedeva, Lucía Álvarez-González, Blanca Urmeneta, Ainoa Planas-Riverola, Jose Carbonell, Teresa Robert-Finestra, A. Alqahtani, Nicholas Brittain, José Falcon-Bermejo, Marta Kulis, J. Martín-Subero, María Dolores Guerrero-Gilabert, Emilio Amilibia, Anna Costa, Sara Pérez, Lisa J. Rusell, Daniel Rico, Alfonso Valencia, B. Javierre
          </td>
          <td>2025-12-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Long Interspersed Nuclear Elements-1 (LINE-1, L1) are transposable elements that make up roughly 17% of the human genome. These elements can copy and insert themselves into new genomic locations (Kazazian and Moran, N Engl J Med 377:361–370, 2017). Typically, LINE-1 is repressed in healthy tissues but may become activated in various human diseases. LINE-1 expression has been associated with aging (Simon, et al., Cell Metab 29:871–885.e5, 2019; De Cecco et al. Nature 566:73–78, 2019; Della Valle et al. Nat Rev Genet 26:1–12, 2025), neurodegenerative disorders (Roy et al., Acta Neuropathol 148:75, 2024;Frost and Dubnau, Annu Rev Neurosci 47:123–143, 2024; Ravel-Godreuil et al. FEBS Lett 595:2733–2755, 2021), cancer (Rodriguez-Martin et al., Nat Genet 52:306-319, 2020; Taylor et al. Cancer Discov 13:2532–2547, 2023; Solovyov et al. Nat Commun 16:2049, 2025), and autoimmune diseases (Rice et al., N Engl J Med 379:2275–2277, 2018), (Carter et al., Arthritis Rheumatol 72:89–99, 2020). Despite the strong association between LINE-1 expression and disease, the regulatory mechanisms controlling the expression of LINE-1-encoded ORF1p and ORF2p and the link between LINE-1 activity and cancer cell survival remain poorly understood. Gaining insights into these regulatory pathways may help elucidate how LINE-1 contributes to disease pathogenesis. To identify upstream regulators of LINE-1 and genes associated with LINE-1 activity-dependent lethality, we developed a dual-reporter system that simultaneously monitors the protein levels of LINE-1-encoded ORF1p and ORF2p (wild-type or catalytically inactive EN/RT mutant). Using genome-wide CRISPR/Cas9-based screens with this system, we identified candidate genes that may influence LINE-1 regulation at multiple levels, including RNA and protein expression. Alongside known factors such as the HUSH complex, the screens revealed additional genes not previously linked to LINE-1 regulation, suggesting possible new regulatory mechanisms for ORF1p and ORF2p expression. We also identified genes whose loss correlated with reduced viability in a manner dependent on LINE-1 activity. These findings collectively provide a broad resource for exploring cellular factors that may modulate LINE-1 expression and activity. This study provides a resource for investigating the cellular regulation of LINE-1, highlighting distinct candidate factors that may modulate ORF1p and ORF2p expression and influence LINE-1 activity-associated cytotoxicity. While functional validation of these candidate regulators remains necessary, the findings offer a foundation for future studies aimed at experimentally confirming their roles and elucidating the molecular mechanisms underlying LINE-1 regulation and its potential contributions to disease contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85dc75de6176e9aaa531a302da776789c267f8ce" target='_blank'>
              Identification of novel regulators of LINE-1 expression via CRISPR/Cas9 screening
              </a>
            </td>
          <td>
            Ozgur Oksuz, Chong Chu, Cedric Arisdakessian, Liyang Diao, Dennis M. Zaller, Kimberly K Long, Heike Keilhack, Sarah Knutson
          </td>
          <td>2025-12-05</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7edbcbe66a598eaa27dab20a758b907c70d1442c" target='_blank'>
              Adaptive Responses Directed by CREB Control Epithelial-Mesenchymal Plasticity in Cancer
              </a>
            </td>
          <td>
            Kshitij Parag-Sharma, Harish Bharambe, John J. Powers, W. Gong, Daniel J. Purcell, Caleb Mensah, Chloe Twomey, Jay H. Mehta, Samy Lamouille, Antonio L. Amelio
          </td>
          <td>2025-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fa98043059e18d9fd50446cb2c494c87e5e2054" target='_blank'>
              Dynamics of clonal hematopoiesis and cellular responses to stress-induced toxicity in autologous stem cell transplantation.
              </a>
            </td>
          <td>
            Catarina M Stein, Raphael Hablesreiter, F. Christen, P. Löwe, Coral Fustero-Torre, Klara Kopp, B. Locher, Lena Nitsch, Robert Altwasser, Johanna Franziska Kerschbaum, L. Bullinger, Leif S. Ludwig, Paulina M Strzelecka, Frederik Damm
          </td>
          <td>2025-12-19</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Osteosarcoma (OS), the most common malignant bone tumor in adolescents, exhibits marked genetic and cellular heterogeneity, contributing to its resistance to conventional therapies. Although surgical and chemotherapeutic interventions have advanced, survival rates remain stagnant, especially for patients with metastatic or treatment-refractory disease. Recent multi-omic studies have uncovered distinct molecular subtypes characterized by unique genomic, epigenomic, transcriptomic, and tumor microenvironmental signatures. The integration of bulk and single-cell transcriptomics, spatial profiling, epigenetic mapping, and proteomic analyses has provided unprecedented insights into the pathogenesis of OS and has facilitated the identification of novel biomarkers. This review comprehensively summarizes the current advances in the molecular subtyping of OS, highlighting subtype-specific oncogenic drivers, their implications in tumor progression and therapeutic resistance, and the potential for precision medicine strategies tailored to these molecular profiles.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ade8a8b9df4408ea89e61a0e1bd36268d586bf5" target='_blank'>
              Decoding osteosarcoma from heterogeneity to precision therapy
              </a>
            </td>
          <td>
            Yu Xu, Yanfu Bai, Fan Yang, Binbin Yang, Zhisheng Huang, Lingling Su, Kuntao Peng
          </td>
          <td>2025-12-11</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cholangiocarcinoma (CCA) is a highly aggressive cancer of the biliary tract, distinguished by significant intratumoral heterogeneity (ITH), which contributes to therapy resistance and unfavorable clinical outcomes. Traditional genome profiling has revealed recurring driver changes in CCA; yet, genomic data alone fails to elucidate functional pathway activation, adaptive signaling, and the diverse treatment responses reported among tumor locations and disease subtypes. This review analyses the use of integrated sequencing technologies, proteogenomics, and phosphoproteomics to systematically characterize intratumoral heterogeneity in cholangiocarcinoma and convert molecular diversity into therapeutically applicable discoveries. We present evidence that the combination of genomic sequencing and mass spectrometry–based proteomics facilitates the direct correlation of genetic mutations with protein expression, post-translational modifications, and signaling system activity. Phosphoproteomic profiling specifically offers functional insights into kinase-driven networks that dictate tumor aggressiveness, therapeutic susceptibility, and adaptive resistance mechanisms, which cannot be anticipated only from DNA-level analysis. We propose that integrating proteogenomic and phosphoproteomic analyses into diagnostic and therapeutic assessments can enhance molecular classification, reveal subtype- and region-specific therapeutic dependencies, and guide rational combination treatment strategies, based on recent extensive proteogenomic studies and functional proteomic investigations in CCA. Pathway-level analysis of intratumoral heterogeneity provides a framework for selecting targeted medicines, predicting resistance, and informing personalized treatment strategies in CCA. The combination of sequencing, proteogenomics, and phosphoproteomics is essential for advancing precision oncology in cholangiocarcinoma. The implementation of this multi-layered analytical approach may better patient classification, refine therapy choices, and eventually improve clinical outcomes for individuals with this particular heterogeneous cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a96c2c6b9428d020026e895b0e1d0644f1ab872b" target='_blank'>
              Integrative Sequencing and Proteogenomic Approaches to Intratumoral Heterogeneity in Cholangiocarcinoma: Implications for Precision Diagnosis and Therapy
              </a>
            </td>
          <td>
            Sirinya Sitthirak, Arporn Wangwiwatsin, Apinya Jusakul, N. Namwat, Poramate Klanrit, Sitthirug Roytrakul, Hasaya Dokduang, Thitinat Duangchan, Yanisa Rattanapan, Attapol Titapun, Apiwat Jareanrat, Vasin Thanasukarn, Natcha Khuntikeo, Teh Bin Tean, Luke Boulter, Yoshinori Murakami, W. Loilome
          </td>
          <td>2026-01-07</td>
          <td>Medical Sciences</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Breast cancer (BRCA) remains one of the most prevalent malignancies and a major threat to women’s health worldwide. The biological role and mechanistic basis of the long non-coding RNA (lncRNA) RERE-AS1 in BRCA remain unclear. Clinical and RNA sequencing data from BRCA patients were analyzed to determine RERE-AS1 expression levels. RNA fluorescence in situ hybridization (FISH) was used to visualize its distribution in tumor tissues. Functional assays were conducted in MCF-7 and T47D cells to assess the effects of RERE-AS1 on cellular proliferation, migration, invasion, and apoptosis using Cell Counting Kit-8 (CCK-8), Transwell assays, and flow cytometry. Cell cycle progression was also evaluated. A co-culture system of BRCA cells and immune cells was established to examine immune cell migration, lactate dehydrogenase (LDH) release, and immune cytotoxicity. Cytokine secretion was quantified via enzyme-linked immunosorbent assay (ELISA). Furthermore, a xenograft mouse model was utilized to confirm the impact of RERE-AS1 on tumor growth and its response to anti-PD-1 therapy. RERE-AS1 expression was markedly decreased in BRCA tissues and correlated with overall survival (OS), tumor stage, and immune infiltration levels. Functional experiments demonstrated that RERE-AS1 overexpression inhibited BRCA cell proliferation, migration, and invasion while inducing apoptosis and cell cycle arrest. Additionally, RERE-AS1 strengthened the interaction between BRCA and immune cells, reduced immune escape, and enhanced tumor sensitivity to immune checkpoint blockade therapy. RERE-AS1 acts as a promising prognostic biomarker and potential therapeutic target in BRCA. Its expression is closely associated with disease progression and patient outcomes and plays a regulatory role in tumor cell behavior and immune microenvironment modulation. This study identifies RERE-AS1 as a critical regulator of tumor-immune interactions, suggesting its potential for guiding precision immunotherapy and prognostic assessment in BRCA patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cfc475fca3f9b9a9e702a4972797d2d3e37f83a" target='_blank'>
              RERE-AS1 as a regulator of immune modulation and therapeutic response in breast cancer
              </a>
            </td>
          <td>
            Shaohui Jiang, Weijie He, Zhimin Li, Haodong He, Yuehua Li, Meiling Wen, Hongbo Zhu, Pengfei Luo
          </td>
          <td>2025-12-19</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Loss of isozyme diversity (LID) refers to the selective dependency on a single isozyme following the functional collapse of its redundant counterparts, uncovering a metabolic vulnerability. This metabolic liability establishes LID as a novel framework for precision targeting strategy in cancer therapy. We integrated a large-scale breast cancer multi-omics cohort to systematically delineate isozyme expression patterns and identify therapeutically promising LID targets across intrinsic molecular subtypes. Genomic and epigenetic data were employed to decipher the underlying causes of LID. Subsequent experiments were conducted to validate the collateral metabolic vulnerabilities arising from identified LID targets and investigate the associated mechanisms. Our analysis delineated the isozyme utilization patterns for metabolic reactions in breast cancer, which revealed widespread and significant LID phenomena. We further decoded the enrichment characteristics of these LID reactions and identified a set of potential clinically relevant LID targets across different subtypes. Further investigation revealed that copy number loss and epigenetic silencing predominantly drive these LIDs, as exemplified by luminal subtype-specific N-myristoyltransferase 2 (NMT2) deficiency caused by hypermethylation. Mechanistic studies demonstrated that NMT2 deficiency confers synthetic lethality with NMT1 inhibition, potentially through impaired myristoylation of the oncogenic effector FMR1 autosomal homolog 2 (FXR2). These findings underscored the potential of NMT inhibitors as viable therapeutic strategies for NMT2-deficient tumors. In conclusion, this study constructed a comprehensive LID landscape across all breast cancer subtypes, advancing precision oncology through innovative metabolic targeting approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48d7510a6c9d424472c6f680b4af6eada3b51b02" target='_blank'>
              Collateral metabolic vulnerabilities unveiled by loss of isozyme diversity in breast cancer
              </a>
            </td>
          <td>
            Rui Ding, Tianjian Yu, Yizhou Jiang, Yi Xiao, Zhi-Ming Shao
          </td>
          <td>2025-12-12</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Genetic factors driving the development of pediatric Acute Lymphoblastic Leukemia (ALL) remain incompletely understood. While epigenetic dysregulation through histone methyltransferases and demethylases has emerged as a critical oncogenic mechanism, their specific contributions to pediatric ALL require systematic investigation. In this study, we systematically investigated genetic variants in histone methyltransferases and demethylase found in top 50 mutated genes from TARGET ALL Phase 2 cohort. Single nucleotide variations were functionally annotated using PredictSNP and I-Mutant2.0, followed by survival analysis and pathway enrichment studies comparing mutated versus wild-type patients. KMT2D, NSD2, SETD2, EZH2 (methyltransferases), and KDM6A (demethylase), ranked among the most frequently mutated genes. Multiple frameshift and nonsense mutations were identified in these genes, likely resulting in truncated proteins and loss of function. Critically, missense variants in EZH2 (F145C, R679C, P132T, D652G, A576D, S280C, Y728H, C566S), KDM6A (G1242D), NSD2 (E1099K, D1125H), and SETD2 (G2170D, R1592Q) were computationally predicted as deleterious and destabilizing. Although survival analysis revealed no statistically significant differences (p = 0.37), mutation carriers exhibited a 44% increased mortality risk, suggesting clinical relevance despite limited sample size (n = 35). Gene set enrichment analysis of differentially expressed genes revealed significant activation of hematopoietic cell lineage pathways in mutated patients. Our findings underscore the importance of histone methyltransferase and demethylase gene variants in pediatric ALL, particularly novel mutations affecting protein function and stability that remain poorly characterized in the literature. This study identifies candidate mutations that warrant further functional and clinical investigation for their role in disease progression and therapeutic targeting.
HIGHLIGHTS

KMT2D, NSD2, SETD2, EZH2, and KDM6A are identified among the top 50 most frequently mutated genes in pediatric ALL
KMT2D exhibits the highest mutational burden in pediatric ALL, predominantly harboring frameshift and nonsense alterations
KMT2D, NSD2, SETD2, EZH2, and KDM6A genes show consensus deleterious predictions across multiple in silico tools, with 18 mutations representing high-priority candidates for experimental validation
Structural stability analysis reveals critical destabilizing mutations in EZH2 (F145C, R679C, P132T, D652G, A576D, S280C, Y728H, C566S), KDM6A (G1242D), NSD2 (E1099K, D1125H), and SETD2 (G2170D, R1592Q)

GRAPHICAL ABSTRACT">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b15b6568cbd53c364a5d95d54d4bb823b4c767ac" target='_blank'>
              Histone Methyltransferase and Demethylase Gene in Pediatric Acute Lymphoblastic Leukemia: A Molecular Insights
              </a>
            </td>
          <td>
            Nur Aziz, Y. N. Patria, A. Wahdi, Eddy Supriyadi, Dewajani Purnomosari
          </td>
          <td>2026-01-01</td>
          <td>Trends in Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fab0829fa668d3f68087f51374ae0be2ba943151" target='_blank'>
              Perturbational single-cell profiling of patient tumors defines lineage- and context-specific programs of innate immune resistance
              </a>
            </td>
          <td>
            C. Perry, A. Frey, Y. Fei, J. Wang, P. Costa, H. N. Amin, S. Ghadermarzi, D. Levine, W. Tong, A. Koda, M. Mackie, M. He, Y. Nie, K. Clulo, F. Ouerghi, J. Wei, T. Cordero Dumit, M. Yaskolko, M. Ding, A. Caldera, O. Kyrysyuk, L. Lum, J. W. Allen, W. Guo, A. Naqash, A. Elliot, A. Vanderwalde, M. Capelletti, T. Adeyelu, S. Barry, M. Hugaboom, A. Bacchiocchi, H. Kluger, M. Bosenberg, A. Iwasaki, D. Braun, J. Clune, D. van Dijk, K. Olino, J. Ishizuka
          </td>
          <td>2025-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0dd4ab62e8dce20ab891fcaeb464ced704c0754" target='_blank'>
              PKMYT1 inhibition induces DNA damage and synergizes with immune checkpoint blockade in CCNE1-amplified gastroesophageal adenocarcinoma
              </a>
            </td>
          <td>
            S. Jang, Lawrence W Wu, Jimyung Park, Sho Hangai, Cameron Shapiro, C. G. Marshall, Luke L. Cai, Benjamin Izar, Adam J. Bass, Ryan H. Moy
          </td>
          <td>2025-12-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is the second most frequently diagnosed cancer worldwide and represents a major challenge for public health. Despite advances in molecular profiling, important gaps remain in our understanding of tumorigenesis and the regulatory mechanisms underlying CRC progression. The most widely adopted classification system is the Consensus Molecular Subtypes (CMS), which stratifies CRC into four biologically distinct subtypes. We investigated the role of A-to-I RNA editing across CMS subtypes in a cohort of 100 CRC patients at various disease stages. Bulk RNA-seq data were analyzed using REDItools to detect editing events, focusing on both recoding sites and edits within repetitive elements, such as Alu sequences. Furthermore, expression levels of the ADAR enzyme family were assessed, and deconvolution analyses were performed on single-cell RNA-seq data from an independent cohort of stage II CRC patients to characterize editing activity within the tumor microenvironment (TME). Competitive endogenous RNA (ceRNA) networks, specific to each CMS subtype, were constructed based on editing events in repetitive elements. A multivariate Cox proportional hazards model was applied to evaluate associations with overall survival (OS). We observed statistically significant differences in ADARB1 expression across CMS subtypes. Single-cell RNA-seq data revealed subtype-specific distribution patterns of ADAR enzymes within the TME. Analysis of editing events showed subtype-specific signatures in both known cancer-related genes (e.g., COPA, CADPS, IGFBP7) and novel candidates (ZNF552, RALGPS1). Editing in repetitive elements informed the construction of distinct ceRNA networks for each CMS subtype, suggesting different post-transcriptional regulatory mechanisms. Survival analysis identified three variables significantly associated with OS, independent of CMS classification and clinical stage: ADARB1 expression, and editing events in NOP14-AS1 (chr4:2960236; p = 0.036; HR = 0.0069), previously linked to 5-FU sensitivity, and ST7-AS2 (chr4:117120557). This study underscores the biological relevance of RNA editing in CRC, highlighting its impact on chemoresistance, the tumor microenvironment, and subtype-specific gene regulation. Our findings suggest that RNA editing represents a critical post-transcriptional regulatory layer in CRC and holds potential as a biomarker and therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/289a3e685e9081e2037f2e82b68ea443c3971d6f" target='_blank'>
              Biological and prognostic relevance of A-to-I RNA editing across consensus molecular subtypes of colon cancer
              </a>
            </td>
          <td>
            Dario Monaco, D. Traversa, E. Mattioli, F. A. Zito, Grazia Cristiani, Francesca Buono, Sabina Delcuratolo, Tiziana Guarino, R. Pinto, Antonia Lasorella, R. Lacalamita, Ernesto Picardi, Stefania Tommasi, S. De Summa
          </td>
          <td>2026-01-06</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a genetically heterogeneous malignancy where traditional DNA sequence-based classification fails to fully explain clinical outcomes. This review explores the dynamic and multifaceted role of the epigenome as a critical driver of leukemogenesis, progression, and therapeutic resistance. We detail how disruptions in DNA methylation, histone modifications, chromatin architecture, and non-coding RNAs create a complex regulatory landscape that transcends genetic alterations. We highlight the emergence of single-cell multi-omics technologies, which are resolving intratumoral heterogeneity and uncovering the epigenetic signatures of leukemic stem cells. Furthermore, we discuss the significant translational potential of these discoveries, including the development of epigenetic biomarkers for refined risk stratification, minimal residual disease (MRD) monitoring, and predicting response to novel epigenetic therapies like menin and LSD1 inhibitors. Integrating these dynamic epigenetic layers into clinical decision-making promises to usher in a new era of precision medicine, ultimately improving prognostic accuracy and therapeutic outcomes for AML patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d3853f768df9c1d2f10b0fd3237fd7635474d26" target='_blank'>
              Beyond the DNA sequence: mapping the dynamic epigenetic landscape for risk stratification and therapeutic intervention in acute myeloid leukemia
              </a>
            </td>
          <td>
            Jiqian Xie, Hamed Soleimani Samarkhazan
          </td>
          <td>2025-12-16</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e9f264c9beccf9f16f96fab6b533051457f1ceb" target='_blank'>
              Acquisition of drug-resistance induces cytokine reprogramming in a human leukemic cell line. Role of p38 MAPK.
              </a>
            </td>
          <td>
            Paula Cadenas-Garrido, A. García-Serna, Antonio Parrado, Javier Martínez-García, Marcela Herrera-Suarez, M. Canovas, Teresa de Diego Puente, Miguel Saceda, Pablo Pelegrin, M. S. Díaz-Carrasco, E. Martín-Orozco
          </td>
          <td>2026-01-29</td>
          <td>Cell communication and signaling : CCS</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Recent progress in single-cell RNA sequencing has led to mechanistic and clinically actionable insight into genetic heterogeneity and tumor progression via transcriptome profiling at single-cell level, applied to identification of cell types, gene expression patterns, and signaling pathways involved in cancer development. In this work, we review the use of single-cell RNA sequencing (scRNA-seq) applications to gain insights into tumor molecular and cellular characteristics, such as cellular heterogeneity, rare cell populations, characteristic pathogenic cell populations, cells of the immune tumor microenvironment, and information regarding clonal evolution, none of which can be observed using bulk RNA-seq. We describe how this set of methods facilitates a better understanding of tumor heterogeneity, interactions between the tumor cells and the cells of the tumor microenvironment (TME), and can elucidate potential therapeutic targets. From the applied clinical perspective, we summarise the ability of scRNA-seq data to identify molecular indicators for diagnosis, outcome, and prediction of response to therapy. This is particularly relevant due to the low response rate to therapy of non-small cell lung cancer (NSCLC) and acquired resistance, including in immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d91b19b2652e7118bd597f31b2a5c69fb2aa994e" target='_blank'>
              Tumor heterogeneity assessment using single-cell RNA sequencing (scRNA-seq): applications in lung cancer for diagnosis and treatment
              </a>
            </td>
          <td>
            C. Bica, Oana Zanoaga, L. Pop, C. Ciocan, L. Raduly, A. Nutu, I. Berindan‐Neagoe, Andreas Bender
          </td>
          <td>2026-01-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Simple Summary Given the clinical challenge of advanced, therapy-resistant prostate cancer (PCa), this study aimed to identify novel molecular drivers. Using transcriptomic data from the TCGA and GEO databases, combined with WGCNA, differential expression analysis, and LASSO regression, APOBEC3C (A3C) was identified as a key candidate, whose downregulation in PCa tumors correlated with advanced T stage, higher Gleason scores, and poor survival. Bioinformatic analysis linked high A3C expression to an anti-tumor immune microenvironment (e.g., increased CD8+ T cell infiltration and reduced M2 macrophages). In vitro assays confirmed that A3C overexpression suppressed PCa cell proliferation, migration, and invasion, while its knockdown promoted these malignant phenotypes. Mechanistically, A3C enhances the expression levels of STING1 and its downstream molecules, including Caspase1, IL-18, and IL-1β, upregulating DNA damage protective genes (GSTP1 and GPX3) and enhancing cell cycle regulator GAS1 expression. Collectively, this study establishes A3C as a PCa suppressor that impedes tumor progression via multiple key pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cec045d2e0d2008c26bed3ce43878ca6a3ed384d" target='_blank'>
              APOBEC3C Suppresses Prostate Cancer by Regulating Key Molecules Involved in Cellular Inflammation, Cell Cycle Arrest, and DNA Damage Response
              </a>
            </td>
          <td>
            Zhongqi Pang, Jian-she Wang, Yidan Xu, Bo Ji, Ming-hua Ren, Bei-chen Ding
          </td>
          <td>2026-01-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Introduction Philadelphia chromosome-positive acute lymphoblastic leukemia (Ph+ ALL) continues to pose a major clinical challenge. Although tyrosine kinase inhibitor (TKIs)-based regimens form the cornerstone of therapy, treatment outcomes are compromised by difficulties in achieving remission, high relapse rates, and the critical issue of drug resistance. This study aims to elucidate the role of the purinergic receptor P2RX1 in the pathogenesis of and therapeutic response in Ph+ ALL. Methods By analyzing P2RX1 expression levels in an online patient database and its association with prognosis, we evaluated the potential significance of this receptor in clinical outcomes. A P2RX1 overexpression model was established in the SUP-B15 Ph+ ALL cell line, and a series of functional experiments were conducted to assess its impact on tyrosine kinase inhibitor-induced apoptosis and proliferation capacity. To further elucidate its mechanism of action, we monitored changes in intracellular calcium ion concentration, mitochondrial membrane potential, and ATP production levels. Additionally, RT-PCR and Western blot analysis were employed to assess the activation status of the PI3K/Akt signaling pathway, CaMKII, and apoptosis-related proteins BAX, BAD, cytochrome C, and caspases Results Analysis of the online patient database revealed that high expression of P2RX1 was significantly associated with poor clinical outcomes. Functional experiments demonstrated that overexpression of P2RX1 in SUP-B15 cells markedly enhanced their sensitivity to apoptosis induced by tyrosine kinase inhibitors; conversely, treatment with the CaMKII inhibitor KN-62 significantly suppressed cell proliferation. Mechanistically, excessive P2RX1 activation disrupts intracellular calcium homeostasis, leading to reduced mitochondrial membrane potential and ATP depletion, thereby activating the intrinsic apoptotic pathway. This process involves inhibition of the PI3K/Akt signaling pathway, hyperactivation of CaMKII, and upregulation of pro-apoptotic proteins such as BAX, Bad, cytochrome C, and cleaved caspase-3 and cleaved caspase-9. Conclusion This study reveals the role of P2RX1 as a calcium-regulated tumor suppressor in Ph+ ALL, promoting apoptosis by disrupting mitochondrial function and inhibiting the PI3K/Akt survival signaling pathway, thereby providing a novel therapeutic approach to overcome TKI resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d84d7432474b5fe37d16f090e0022f4a6ea5c784" target='_blank'>
              P2RX1 promotes mitochondrial apoptosis via calcium/CaM KII-mediated suppression of PI3K/Akt signaling in Philadelphia chromosome-positive acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Xiao bing Li, Qin Ren, Xiangmei Ye, Jing lie Li, Leiguang Feng
          </td>
          <td>2025-12-12</td>
          <td>Frontiers in Pediatrics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Variants of uncertain significance represent the biggest challenge for genomics-based precision oncology. Activated fibroblast growth factor receptors (FGFRs) frequently drive tumorigenesis by different genetic aberrations. However, it remains unknown which of the many point mutations affecting FGFR1, FGFR2, FGFR3 or FGFR4 in cancer are druggable, that is, activating signaling while not mediating FGFR inhibitor resistance. Here we implemented a saturation mutational scanning platform to screen all 11,520 possible point mutations covering the kinase domains of FGFR1–4. Pooled positive selection screens identified 474 activating and 738 mutations mediating resistance to the FGFR inhibitors pemigatinib and futibatinib, together revealing 301 druggable FGFR mutations analogous to a strong PS3/BS3 evidence level. The screens also identified loss-of-function mutations and an association of gain-of-function mutations with hydrophobic changes. The functional screens identified 97% of acquired resistance mutations in clinical trials. Our comprehensive catalog of every druggable mutation in the FGFR kinase domains is readily available for clinical decision support. Saturation mutagenesis screening examines 11,520 point mutations in the kinase domains of FGFR1, FGFR2, FGFR3 and FGFR4, identifying their activating and resistance properties to the FGFR inhibitors pemigatinib and futibatinib.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/091d9948ae5e0367c92c6fee36f39d1751918d53" target='_blank'>
              Saturation mutagenesis identifies activating and resistance-inducing FGFR kinase domain mutations
              </a>
            </td>
          <td>
            Carla Tangermann, Avantika Ghosh, Martin Ziegler, Francesco Facchinetti, Jannis Stappenbeck, Yagmur Oyku Carus Sahin, M. Riester, Luise Carmina Viardot, Tobias Zundel, L. Friboulet, Antoine Hollebecque, J. J. Naveja, Angela Wanninger, M. Hess, Tilman Brummer, M. Boerries, S. Loges, Y. Loriot, A. Illert, Sven Diederichs
          </td>
          <td>2025-12-08</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Background: Long INterspersed Element-1 (LINE-1) retrotransposons comprise 17–20% of the human genome. These retroelements are normally silenced early in embryonic development through epigenetic mechanisms and reawakened during oncogenesis, leading to transcriptional dysregulation, genomic instability, and immune evasion. Methods: In the present study, we categorized LINE-1 transcripts across 121 non-small cell lung cancer (NSCLC) cell lines from the Cancer Cell Line Encyclopedia (CCLE) by subfamily, length, orientation, chromosomal origin, and distribution. In addition, high-prevalence insertions were mapped to nearby genes to assess potential functional interactions. Results: LINE-1 transcript abundance and length in NSCLC were dominated by evolutionarily young subfamilies, particularly L1HS and L1PA2 through L1PA5. Chromosomal patterns were conserved across NSCLC subtypes, with modest enrichment of L1HS activity on Chromosome 4 and the X Chromosome. The lung squamous cell carcinoma (LSQCC) subtype exhibited the highest total levels of L1HS expression relative to other NSCLC subtypes. Race modestly influenced LINE-1 transcript abundance, with cell lines derived from self-identified African American individuals showing elevated overall LINE-1 and L1HS expression. Age showed a weak positive correlation with total LINE-1 abundance. Integrative analysis revealed recurrent hotspots at 22q12.1 and 20p11.21 that were transcriptionally active across subtypes and coincided with previously reported intact LINE-1 elements active in epithelial cancers. Recurrent insertions were located near cancer-associated genes, including RB1, NEDD4, FTO, LAMA2, NOD1, and KCNB2, implicating LINE-1 activity in cis-regulatory remodeling of oncogenic pathways. Conclusions: Together, these findings indicate that LINE-1 transcript heterogeneity in NSCLC is shaped by host genomic architecture and conserved functional hotspots, providing new insights into the mechanisms of genetic and epigenetic dysregulation associated with LINE-1 retroelements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a76a2793974308abd948a4e791ee9224ae6f4f7f" target='_blank'>
              LINE-1 Transcript Heterogeneity in Non-Small Cell Lung Cancers Is Driven by Host Genomic Context and Conserved Functional Hotspots
              </a>
            </td>
          <td>
            Yingshan Wang, Kenneth S. Ramos
          </td>
          <td>2026-01-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Myeloid sarcoma, an aggressive extramedullary subtype of acute myeloid leukemia (AML), occurs in ∼10% of patients, and has not yet been included in large-scale genomic studies. The critical biological changes that drive tumor evolution are unknown, its detection in asymptomatic patients remains a clinical challenge, and treatment options are limited as patients are often excluded from clinical trials, rendering it a neglected disease entity. Based on comprehensive multi-omic profiling, we demonstrate that myeloid sarcoma evolves from medullary AML with distinct sitespecific clonal evolution patterns. Additionally, we show that circulating tumor DNA sequencing can serve as a non-invasive method for molecular profiling of myeloid sarcoma, offering a novel avenue in molecular diagnostics. We characterize unique transcriptional profiles of myeloid sarcoma, reflecting immune evasion and adaptation to an extramedullary microenvironment. We provide evidence for a key role of RAS pathway activation and demonstrate in murine models of myeloid sarcoma that RAS inhibition effectively reduces tumor burden. Overall, our data highlight key differences between medullary AML and myeloid sarcoma including universal molecular evolution and RAS pathway activation as hallmarks of the disease and nominate RAS inhibition as a promising therapeutic strategy for patients with myeloid sarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10582cdaf7507df9370133688fbcd15ae5208a8b" target='_blank'>
              Multiomic characterization, early detection, and therapeutic targeting of myeloid sarcoma
              </a>
            </td>
          <td>
            B. Nadorp, Audrey Lasry, S. Loghavi, Ravi Patel, Hager Mansour, Benjamin J. Kelly, Christopher J. Walker, Jill Buss, Isaiah Boateng, Wafa Al-Santli, Zoe Ciantra, Rebecca Austin, Adrienne Heyrosa, Helee Desai, Andrea Laganson, Hasan Abaza, Linda Procell, Tejas Patel, Benjamin Kaffenberger, Saranga Wijeratne, Maria Guillamot, Maria Velegraki, Luis Chiriboga, Zihai Li, L. Abruzzo, Daniel A. Pollyea, Christine M McMahon, A. Shana'ah, John C. Byrd, Alan H Shih, R. L. Levine, E. Papapetrou, A. Tsirigos, Elaine R. Mardis, Alice S. Mims, I. Aifantis, Ann-Kathrin Eisfeld
          </td>
          <td>2025-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="Highlights What are the main findings? The anti-tumor efficacy of nucleoside analogs is dependent on transport, enzymatic activation, nucleotide pools and metabolic responses, which are often aberrant in cancer cells, rendering these chemotherapeutic drugs ineffective. What are the implications of the main findings? Small molecule inhibitors are being tested/developed to target or downregulate specific mechanisms responsible for resistance to nucleoside analogs. The combination of small molecule inhibitors with nucleoside analogs has enhanced the efficacy of nucleoside analogs and improved patient outcomes. Abstract Nucleoside analogs remain central to the treatment of hematologic malignancies and solid tumors, yet resistance frequently occurs, contributing to relapse and disease-related mortality. Rather than arising from a single mechanism, effective nucleoside analog activity requires successful navigation of multiple biological barriers, including cellular uptake, intracellular activation, nucleotide pool balance, genome surveillance, and mitochondrial stress responses. This review integrates recent advances describing how alterations at each of these levels contribute to resistance to nucleoside analog therapies. We further highlight emerging therapeutic strategies centered on small-molecule inhibitors that exploit these vulnerabilities to enhance the efficacy of nucleoside analogs. Together, this integrative perspective supports the need for development of small molecule inhibitors and design of combination approaches aimed at restoring apoptotic competence and improving the use of nucleoside analog-based therapies for the treatment of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f48f64e40e1b94edba907600f2e063e159b5fa3" target='_blank'>
              Enhancing the Nucleoside Analog Response with Translational Therapeutic Approaches to Overcome Resistance
              </a>
            </td>
          <td>
            Jenna Thibodeau, Kian K Hershberger, Sai Samanvitha M. Ramakrishna, Yongwei Su, Lauren Timmer, Bryce Brophy, Katherine K Zhang, Holly Edwards, Jeffrey W. Taub, Yubin Ge
          </td>
          <td>2026-01-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Temozolomide (TMZ) remains foundational in the management of adult-type diffuse gliomas in general, and glioblastoma specifically. However, its efficacy harbors an evolutionary trade-off. TMZ drives its cytotoxicity through generating O6-methylguanine lesions, especially active in MGMT-silenced, mismatch repair (MMR)-proficient tumors. By selecting for acquired MMR-deficient subclones, often via MSH6 inactivation, this process escalates into a hypermutator phenotype, generating thousands of de novo alterations. This is a hallmark of the mutational signature known as SBS11, characterized by C>T transitions, which is associated with TMZ treatment. The hypermutator phenotype drives heterogeneity, therapeutic resistance, spatial diversification, and distant recurrence. Despite harboring a mutational burden comparable to melanoma and lung cancer, TMZ-induced hypermutation does not sensitize gliomas to immune checkpoint blockade. This resistance reflects the profoundly immunosuppressive brain microenvironment, impaired antigen presentation, marked transcriptional plasticity, and perhaps also the frequent use of corticosteroids. Emerging strategies aim to exploit vulnerabilities created by TMZ-mediated genomic instability, including PARP, ATR, WEE1, and AURKA inhibition; alternative alkylators; metabolic rewiring; and G-quadruplex stabilization. Notably, the real-time detection of evolving mutational signatures via CSF-based liquid biopsies may enable adaptive therapy before radiographic progression. By reframing TMZ as a potent evolutionary agent rather than a conventional chemotherapy, this review synthesizes recent mechanistic insights and translational opportunities to guide a next-generation, evolution-informed treatment paradigm for glioma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f781ca89daad9cc99ecacd68837541210aedbb9" target='_blank'>
              The Temozolomide Mutational Signature: Mechanisms, Clinical Implications, and Therapeutic Opportunities in Primary Brain Tumor Management
              </a>
            </td>
          <td>
            Adar Yaacov, Roni Gillis, Jaber Salim, Daniela Katz, Noam Asna, Iddo Paldor, A. Grinshpun
          </td>
          <td>2025-12-29</td>
          <td>Cells</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Most colorectal cancers are resistant to immune checkpoint inhibitors due to an immunosuppressive tumor microenvironment. LINC00673, previously classified as a long non-coding RNA, has been implicated in tumor progression, but its role in antitumor immunity remains poorly understood. We used single-cell RNA sequencing, flow cytometry, and functional assays in Linc00673 knockout and wild-type mouse models treated with programmed death 1 (PD-1) blockade to dissect changes in immune landscape and tumor cell behavior. Linc00673 deletion enhanced the efficacy of PD-1 therapy, reducing tumor burden and prolonging survival. KO tumors showed increased infiltration of cytotoxic CD8⁺ T cells, reduced exhaustion, and improved antigen presentation. The tumor microenvironment shifted toward a pro-inflammatory state, with elevated dendritic cell function, reduced immunosuppressive macrophages, and improved T cell–tumor interactions. Linc00673 also promoted tumor cell migration and immune evasion independently of immune cells, suggesting dual tumor-intrinsic and -extrinsic roles. LINC00673 suppresses antitumor immunity and promotes tumor aggressiveness. Its deletion synergizes with PD-1 blockade, revealing a promising therapeutic target to overcome immune resistance in colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb9ef0a9a3f2685b8b34be389ae9c671f82a1666" target='_blank'>
              Single-cell RNA sequencing unveils enhanced antitumor immunity in colorectal cancer with PD-1 blockade and LINC00673 deletion
              </a>
            </td>
          <td>
            Yifei Zhu, Yanxi Yao, Yaxian Wang, Zhibing Qiu, Dingpei Zhou, Huixia Huang, Keji Chen, Qingyang Sun, Jiayu Chen, Yuxue Li, Jianqiang Tang, Dawei Li, Ping Wei
          </td>
          <td>2026-01-03</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/103a0a6127003db01ce70e503d0d9d03c9b83f1d" target='_blank'>
              Systematic functional dissection of germline noncoding risk variants impacting clonal hematopoiesis
              </a>
            </td>
          <td>
            Trieu Nguyen, Jessica Jeejan, Takeshi Iwasaki, S. Kales, Joyeeta Chakraborty, Chie Yanase, Aruna Shekhar, Dominika Kwasniak, Aditi Hegde, Richard Voit, Kristy Stengel, Keisuke Ito, Ryan Tewhey, Satish K. Nandakumar
          </td>
          <td>2026-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79941e547de1ddb3fb2c22aa561602d507f6cf5a" target='_blank'>
              CRISPR-enhanced assessment of variants of unknown significance nominates oncology therapeutic targets and drug repositioning opportunities
              </a>
            </td>
          <td>
            A. Savino, Athanasios Oikonomou, Riccardo R. De Lucia, Miquel L Grau, Kathrina McCarten, H. Najgebauer, Umberto Perron, Luca Azzolin, Alexandra Livanova, Paolo Cremaschi, Núria López-Bigas, A. Sottoriva, F. Iorio
          </td>
          <td>2026-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Abstract Acute lymphoblastic leukemia (ALL) is the most common childhood cancer. Despite improved therapies, refractory and relapsed ALL remain the leading cause of cancer-related mortality in children. There is a need for accessible biomarkers for frequent, minimally invasive disease monitoring and prompt intervention. MicroDNA is a novel extrachromosomal DNA that preferentially originates from gene segments with high transcriptional activity and/or increased chromatin accessibility. We investigated whether microDNA-producing genes repertoire changes in a disease-dependent manner. We characterized microDNAs in 52 paired bone marrow (BM) and plasma samples from pediatric patients with ALL at diagnosis, relapse, and remission. No difference in the length or number of microDNA was noted across stages, but comparative analysis of microDNA profiles led to the identification of microDNA gene panels associated with active disease. The relative distribution of these genes was significantly different from that expected by chance (P < 0.0001). Analyses of BM samples identified a signature comprising 289 distinct microDNA-producing genes present in multiple patients at diagnosis and relapse but absent in remission. The best biomarker candidates were 11 microDNA-producing genes identified also in plasma samples at diagnosis and overrepresented in patients who relapsed (P = 0.006). MicroDNA from the same genes was confirmed in relapse plasma samples. All signature genes are known to be involved in cancer proliferation or drug response. MicroDNA seems to be a candidate for a novel class of biomarkers for ALL, with the potential to improve precision diagnostics, particularly through their identification in plasma samples. Further validation in an independent cohort of patients is warranted. Significance: Despite high cure rates, 10% to 15% of pediatric patients with ALL experience relapse. We identified a plasma-detectable microDNA signature from 11 genes that persists from diagnosis through relapse but disappears in remission. These findings demonstrate the potential of microDNA profiles as prognostic biomarkers in pediatric ALL, enabling noninvasive monitoring of disease status and risk stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9503e4bfe97b36cabeed806790d4fca481f09664" target='_blank'>
              Extrachromosomal microDNA Signature as a Candidate Biomarker in Pediatric Acute Lymphoblastic Leukemia
              </a>
            </td>
          <td>
            Ivan Brukner, V. Gagné, Alex Richard-St-Hilaire, P. Tremblay-Dauphinais, Claire Fuchs, Henrique Bittencourt, Teodor Veres, D. Sinnett, Maja Krajinovic
          </td>
          <td>2025-12-11</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Although two-thirds of cancers arise from loss-of-function mutations in tumor suppressor genes, there are few approved targeted therapies linked to these alterations. Synthetic lethality offers a promising strategy to treat such cancers by targeting vulnerabilities unique to cancer cells with these mutations. To identify clinically relevant synthetic lethal interactions, we analyzed genome-wide CRISPR/Cas9 knock-out (KO) viability screens from the Cancer Dependency Map and evaluated their clinical relevance in patient tumors through mutual exclusivity, a pattern indicative of synthetic lethality. Indeed, we found significant enrichment of mutual exclusivity for interactions involving cancer driver genes compared to non-driver mutations. To identify therapeutic opportunities, we integrated drug sensitivity data to identify inhibitors that mimic the effects of CRISPR-mediated KO. This approach revealed potential drug repurposing opportunities, including BRD2 inhibitors for bladder cancers with ARID1A mutations and SIN3A-mutated cell lines showing sensitivity to nicotinamide phosphoribosyltransferase (NAMPT) inhibitors. However, we discovered that pharmacological inhibitors often fail to phenocopy KO of matched drug targets, with only a small fraction of drugs inducing similar effects. This discrepancy reveals fundamental differences between pharmacological and genetic perturbations, emphasizing the need for approaches that directly assess the interplay of loss-of-function mutations and drug activity in cancer models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4924d51007baa9dac5b3c8eec35c04f94156ddd0" target='_blank'>
              Challenges and opportunities for oncology drug repurposing informed by synthetic lethality
              </a>
            </td>
          <td>
            Michael Vermeulen, Andrew W. Craig, Tomas Babak
          </td>
          <td>2025-12-01</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52aead1bb294c5c8ab3425b1373986d70ceb1c0f" target='_blank'>
              Blocking Minor Intron Splicing Disrupts DNA Repair and Overcomes Therapy Resistance in Prostate and Breast Cancer
              </a>
            </td>
          <td>
            Anke Augspach, P. Francica, Kaitlin N. Girardini, Muriel Jaquet, Marika Lehner, A. Rodriguez-Calero, Charlotte Komarek, Martín González-Fernández, Hannah L. Williams, Xiaoyue Deng, Lea Lingg, Cristina Zivko, Vera Fuchs, Seynabou Diop, S. Maletti, Simone de Brot, A. Ewe, Elena Perugini, Nigel B Jamieson, Y. Doncheva, Claire K Dietrich, Phillip Thienger, Dilara Akhoundova, Andrej Benjak, A. Aigner, R. Kanadia, Sven Rottenberg, Mark A. Rubin
          </td>
          <td>2025-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Medulloblastoma (MB) is one of the most prevalent malignant brain tumors in children, with tremendous cognitive and neuroendocrine disability among survivors. Group 3 MB (G3MB) has poor overall survival at <50%, high frequencies of metastases, and no targeted therapies. Amplification of MYC and activation of TGFβ signaling occur frequently in G3MB. Many tumors have no reported mutations, suggesting epigenetic drivers. We here describe novel humanized models for G3MB from human induced pluripotent stem cells (hiPSC). By transducing hiPSC-derived neuroepithelial stem cells (NESC), we determined that: 1) both MYC and TGFβ effectors drove tumors in vivo; 2) MYC/TGFβR1 in combination led to more aggressive tumors and resistance to clinical inhibitors of TGFβ, and 3) NESC-derived tumors clustered with human G3MB. To decipher mechanisms, we integrated RNA-sequencing with CUT&RUN (for MYC genomic localization and post-translational modification of histones). MYC-bound neural developmental genes were repressed in MYC/TGFβR1 co-driven lines. Gene signatures associated with the Polycomb Repressive Complex (PRC) demarcated with H3K27me3; the histone mark directly regulated by PRC. We identified JARID1B, a MYC binding partner and H3K4me3 demethylase, as a regulator of repressed neural genes. Primary G3MB also showed increased levels of H3K27me3 concurrent with higher expression of JARID1B. Knockdown of JARID1B in human G3MB cell lines reduced growth, supporting potential as a therapeutic target. We conclude that a MYC-TGFβ-JARID1B axis represses target genes to drive G3MB and present new humanized models for G3MB to understand epigenetic dysregulation in G3MB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd58fafdd674bdec3d7247a90b13d164f167e167" target='_blank'>
              Human stem cell models for group 3 medulloblastoma uncover JARID1B as a regulator of the chromatin landscape
              </a>
            </td>
          <td>
            Zulekha A. Qadeer, Elise Hou, Dara Siegel, Samantha Westelman, B. Gudenas, Kyle S Smith, Wazim Mohammed Ismael, Linyu Wang, Mackenzie O Johnson, Shane Grele, Ronald Phua, Liam D. Hendrikse, Ekin Guney, Miller Huang, Alexandre Gaspar-Maia, P. Northcott, Michael D. Taylor, William A. Weiss
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3e5486bf9671c3b3ce7043325de2e14e73fe8c9" target='_blank'>
              Connecting mitochondrial metabolism and mitotic fidelity to control vulnerability of high grade serous ovarian cancer patients to taxane-based chemotherapy
              </a>
            </td>
          <td>
            H. Moindjie, Morgane Morin, Evanthia Pangou, Diego Chianese, Cynthia Seiler, S. Rodrigues-Ferreira, Maria M. Haykal, Christine Péchoux, F. Dingli, D. Loew, Y. Kieffer, Géraldine Gentric, Patricia Pautier, Philippe Dessen, R. Argüello, Paolo Pinton, Fatima Mechta-Grigoriou, Catherine Brenner, Massimo Bonora, Clara Nahmias
          </td>
          <td>2026-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Cisplatin remains a cornerstone of chemotherapy for numerous cancers, despite the persistent challenges of toxicity and the development of drug resistance. Therefore, a deeper understanding of the mechanisms behind cisplatin resistance and the development of strategies to counter it are of critical importance. This review systematically examines the pivotal role of mitochondrial dynamics in cisplatin resistance and discusses emerging therapeutic strategies that target these processes. Mitochondrial dynamics regulate the structure and function of the mitochondrial network through a balance of fusion and fission. Dysregulation of this process directly contributes to cisplatin resistance by maintaining cellular energy homeostasis, inhibiting apoptosis, and enhancing oxidative phosphorylation (OXPHOS). Furthermore, mitophagy, metabolic reprogramming, and the tumor immune microenvironment converge on mitochondrial dynamics to drive the acquisition of drug resistance. Consequently, targeting mitochondrial dynamics presents a promising approach to overcome cisplatin resistance. Potential strategies include restoring the balance of fusion and fission, intervening in mitophagy, disrupting OXPHOS metabolism, and developing mitochondrial-targeted nanodrug delivery systems. However, despite this promising outlook, significant challenges remain, including the heterogeneity of resistance mechanisms, a lack of reliable biomarkers, and the need for selective targeting to minimize off-target effects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db056eb8c3db5d2ebbae460e0446276957e89344" target='_blank'>
              Mitochondrial dynamics in cisplatin resistance: molecular mechanisms and therapeutic targeting
              </a>
            </td>
          <td>
            Wei Huang, Wei Sun, Zhiyuan Yang, Yiwen Li, Ziliang Wang
          </td>
          <td>2026-01-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Deregulation of proteins involved in chromatin regulation is common in pancreatic ductal adenocarcinoma (PDAC). Lysine demethylase 4C (KDM4C) is one of the chromatin-modifying proteins frequently overexpressed across multiple solid cancers and is linked to chromatin instability, increased cell proliferation, and enhanced stem cell–like behavior. We observed upregulation of KDM4C protein in a panel of human PDAC cell lines and patient samples compared with nonneoplastic controls. CRISPR/Cas9-mediated deletion of KDM4C in human and murine PDAC cells reduced proliferation, clonogenicity, and increased survival of orthotopically implanted murine PDAC allografts. Transcriptomic and proteomic analyses revealed that loss of KDM4C in both human and murine PDAC cell lines was associated with the reduction of activated phospho-ERK, a pivotal effector downstream of mutant RAS. Using proximity labeling, we identified the histone deacetylase SIRT1 as a novel interacting protein with KDM4C via the latter’s Tudor reader domain. SIRT1-mediated deacetylation leads to repression of downstream targets, including the dual specificity phosphatase DUSP2, which is known to inactivate ERK via dephosphorylation. In vitro propagation of KDM4C-null PDAC lines eventually led to adaptation and restitution of ERK signaling, with rescue of the KDM4C loss induced growth suppression. To bypass this adaptive phenomenon, we tested a preclinical pan-KDM4 inhibitor TACH107 and confirmed its efficacy in in vitro and in vivo PDAC models. Our studies identify KDM4C as an oncogenic molecule that sustains ERK signaling in KRAS-mutant PDAC and can be broadly targeted via small-molecule inhibitors. Significance: Our data suggests that KDM4C is a novel regulator of ERK signaling, the main effector pathway downstream of mutant RAS. This is the first demonstration linking the requirement of sustained KDM4 activity to ERK signaling in cancer and presents an opportunity to leverage this oncogenic pathway for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac9ce02e321041b11b3d0c59cd917566ed5c1973" target='_blank'>
              The Lysine Demethylase KDM4C Is an Oncogenic Driver and Regulates ERK Activity in KRAS-Mutant Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Menna-T-Allah Shaheen, Sarah Dhebat, K. Rajapakshe, Bidyut Ghosh, Benson Chellakkan Selvanesan, Shariq S Ansari, C. Haymaker, Dorsay Sadeghian, Huamin Wang, Ching-Fei Li, H. Ying, Anirban Maitra
          </td>
          <td>2026-01-01</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="


 Prostate cancer (PCa) is the most commonly diagnosed cancer and the second leading cause of cancer-related death in American men. Men with germline BRCA2 mutations are at especially high risk (19%-61% by age 80) of developing PCa. Loss of BRCA2 impairs homologous recombination repair (HRR), resulting in genomic instability and enhanced tumor progression. Our laboratory previously performed a whole-genome CRISPR-Cas9 screen in a BRCA2 mutant cell line under cisplatin induced stress. This screen identified Chromodomain Helicase DNA-binding protein 1 (CHD1) as the top gene whose loss promoted cell survival under DNA damage stress. CHD1 is a chromatin remodeler that is deleted in approximately 15% PCas and is strongly linked to metastatic disease. However, how its loss drives tumor progression, particularly in BRCA2-deficient PCas, remains poorly understood. To address this, we carried out RNA-seq analysis in castration-resistant prostate cancer (CRPC) cell line PC-3 following knockdown of CHD1 and BRCA2, individually and in combination. Notably, depletion of CHD1 in BRCA2-deficient cells resulted in the upregulation of gene signatures associated with hypoxia and epithelial-mesenchymal-transition (EMT). Functional validation using wound healing and migration assays demonstrated that CHD1 loss significantly enhanced the migration ability of PC-3 cells (including BRCA2-deficient) compared to controls. In keeping with increased EMT, we also see high vimentin expression in CHD1-depleted PC3. Furthermore, Causal Inference Enrichment (CIE) analysis of the transcriptomic data pointed to significant activation of transcription factors (TFs) FOSL1 and SOX9 after CHD1 knockdown compared to control cells. To assess clinical relevance, we analyzed RNA-seq data from the Stand Up To Cancer (SU2C) database. This analysis revealed that CHD1-low tumors exhibited upregulation of multiple oncogenic pathways, especially EMT, angiogenesis, apical junction, and IL2-STAT5 signaling, consistent with our in vitro observations. Finally, genetically engineered mouse model (GEMM)-derived prostate organoids from B2PPt (Brca2flox/flox;P53flox/flox;Ptenflox/flox) and B2CPPt (Brca2flox/flox; Chd1flox/flox;P53flox/flox;Ptenflox/flox) mice, after Adeno-Cre infection, revealed higher Fosl1, along with EMT marker vimentin in B2CPPt organoids compared to B2PPt, supporting FOSL1-dependent lineage plasticity and activation of EMT programs upon CHD1 loss in BRCA2-mutant prostate cells. Collectively, these findings indicate that CHD1 loss, either independently or in the context of BRCA2 deficiency, promotes oncogenic phenotypes through activation of FOSL1-dependent lineage plasticity and EMT programs that favor metastasis. Ongoing work dissects how CHD1 loss reshapes chromatin accessibility to reprogram transcriptional networks and drives oncogenic signaling and metastatic pathways in PCa, with particular relevance to BRCA2-deficient PCa.



 Jingzhu Hao, Shailja Pathania, Miklos Diossy, Kourosh Zarringhalam, Changmeng Cai, Zoltan Szallasi. Dissecting the role of CHD1 as a key regulator of pro-metastatic transcriptional reprogramming in BRCA2-mutant prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr PR017.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a2ea6307c1138357462e3893b183854dd498126" target='_blank'>
              Abstract PR017: Dissecting the role of CHD1 as a key regulator of pro-metastatic transcriptional reprogramming in BRCA2-mutant prostate cancer
              </a>
            </td>
          <td>
            Jingzhu Hao, S. Pathania, M. Dióssy, Kourosh Zarringhalam, Changmeng Cai, Zoltán Szállási
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Background CD47 is a key innate immune checkpoint that enables tumor cells to evade macrophage-mediated clearance. Methods/Results To systematically identify genetic regulators of CD47 surface expression, we performed FACS-based genome-wide CRISPR screens in three murine cancer cell lines B16 (melanoma), MC38 (colon adenocarcinoma), and EMT6 (breast carcinoma). Results Comparative analysis of cells with high or low CD47 surface expression using DrugZ revealed CD47 itself as the top hit, validating the screens. Notably, DNAJC13 emerged as a consistent and robust regulator of CD47 expression across all three cell lines. Functional validation using DNAJC13-knockout cells confirmed a significant reduction in CD47 surface levels. Furthermore, in co-culture assays with macrophages, DNAJC13-deficient tumor cells exhibited increased susceptibility to phagocytosis, supporting a functional role for DNAJC13 in innate immune evasion. Finally, we verify that DNAJC13-knockout decrease tumor burden when treated with CD47 blockade. Conclusions Overall, this study highlights a previously unrecognized regulator of CD47 and demonstrates the utility of high-throughput FACS-based CRISPR screening to uncover modulators of key immune checkpoint pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/333becc09fbebbb08ac6ada5d4961c0def934201" target='_blank'>
              FACS-based genome-wide CRISPR screening platform identifies modulators of CD47
              </a>
            </td>
          <td>
            Ling Yin, Wei He, Yifan Wang, Huimin Zhang, Min Huang, Yuelong Yan, Siting Li, Xu Feng, Francisco Saenz, Jie Zhang, Dandan Zhu, Chang Yang, Tiantian Ma, Jialing Fu, Junjie Chen
          </td>
          <td>2026-01-12</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a hematologic malignancy characterized by the malignant proliferation of myeloid progenitor cells. Although the introduction of the B‐cell lymphoma‐2 (BCL‐2) inhibitor Venetoclax (VEN) has significantly improved patient outcomes and established it as a first‐line treatment, high rates of drug resistance and relapse remain major clinical challenges. We integrated RUNX3 chromatin immunoprecipitation sequencing (ChIP‐seq) data with the GAPIA2 database to identify CRTC2 as a key candidate gene. Subsequently, we employed qRT‐PCR to compare CRTC2 expression levels between donors and AML patients. The role of CRTC2 in apoptosis was further validated through knockdown and overexpression experiments in various cell lines. To investigate the impact of CRTC2 on AML progression, we established a cell line‐derived xenograft (CDX) model. The proportion of human CD45‐positive (hCD45+) cells in the bone marrow and liver was assessed, and histological examination was conducted using HE staining, along with peripheral blood smear analysis. In addressing VEN resistance, we analyzed CRTC2 expression patterns in clinical samples and explored the synergistic therapeutic effect of a CRTC2 inhibitor in combination with VEN. To further elucidate the underlying molecular mechanisms, we performed mitochondrial function assays and analyzed mitochondrial translation‐related proteins. Clinical analyses have demonstrated that elevated expression levels of CRTC2, a downstream target of RUNX3, are significantly correlated with poor prognosis in patients with AML. Functional experiments have shown that CRTC2 plays a role in disease progression by modulating apoptosis in AML cells. The knockdown of CRTC2 was observed to delay disease progression in CDX mouse models. Additional investigations revealed a positive correlation between CRTC2 expression and resistance to VEN in AML cells, with CRTC2 inhibition synergistically enhancing VEN's cytotoxic effects. Mechanistic studies suggest that increased mitochondrial activity contributes to VEN resistance, thereby identifying a potential molecular target for overcoming drug resistance. CRTC2 is a key regulator in AML, with high expression levels promoting disease progression and resistance to VEN. Inhibiting CRTC2 reduces mitochondrial translation and energy reserves, increasing AML cell sensitivity to VEN. These results highlight CRTC2 as a promising therapeutic target and suggest a new strategy to overcome VEN resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4e5700675fb469182a4657676fd57383406877e" target='_blank'>
              CRTC2 Promotes AML Progression and Mediates Mitochondrially‐Driven Venetoclax Resistance
              </a>
            </td>
          <td>
            Xiaomei Liang, Yilu Zheng, Bangxue Jiang, Yuling Huang, Jing Tang, Haimei Deng, Chenxing Zhang, Minyi Zhao, Dongjun Lin
          </td>
          <td>2025-12-09</td>
          <td>The FASEB Journal</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5bdb6dd873fe4b6bf8ca46144bcdfa66e477ac0" target='_blank'>
              Beyond one-size-fits-all: single-cell transcriptomic signatures predict drug efficacy and reveal responder subgroups in endometriosis
              </a>
            </td>
          <td>
            R. Pérez-Moraga, C. Bafligil, S. Harden, Santiago García‐Martín, M. J. Jiménez-Santos, Ioanna Tiniakou, Sophie Ribeiro-Volturo, Line Gies, María Teresa Pérez Zaballos, Cristina Fernández-Molina
          </td>
          <td>2025-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="This study aimed to investigate the molecular heterogeneity, accurately evaluate the prognosis, and identify those patients who would benefit most from immunotherapy in the hypermutated colorectal cancer (CRC) cohort. The unsupervised clustering analysis of CRC somatic mutation data from the Cancer Genome Atlas database was performed by combining tumor mutational burden (TMB) and indel burden to identify the subtypes within the hypermutated CRC cohort. Then, the somatic mutations, transcriptome, and gene mutations in the DNA damage repair signaling pathway of these subtypes were compared to obtain the corresponding molecular features. The proportion of 22 immune cells infiltrated in the samples was calculated using the CIBERSORT algorithm, and immunotherapeutic sensitivity was predicted for the patients using immunophenoscores (IPS). Five distinct clusters were identified from the hypermutated CRC sample, each exhibiting different characteristics regarding somatic mutations, gene expression, immune infiltration, and prognosis. The proportion of M2 macrophage infiltrated in Cluster 6 was significantly higher than in Cluster 1 (P = .018) and Cluster 3 (P = .003). The proportion of infiltrating M1 macrophages was significantly higher in Cluster 3 than in Cluster 1 (P = .026) and Cluster 6 (P = .016). Cluster 1 had a longer OS time compared to Cluster 3 (P = .015) and Cluster 6 (P = .005). In all 4 IPS types, Cluster 3 demonstrated higher IPS than Cluster 6 (P < .05), Cluster 2 demonstrated lower IPS than Cluster 5 (P < .05). In the IPS-CTLA4 type, the IPS of Cluster 1 was significantly higher than that of Cluster 6 (P = .038). In the non-POLE mutant cohort, the patients with the lowest TMB and indel burden had longer overall survival. The patients with higher levels of M1 macrophage infiltration showed higher sensitivity to anti-PD-1/PD-L1 immunotherapy or combined anti-PD-1/PD-L1 and anti-CTLA4 immunotherapy. In the POLE mutant cohort, the patients from the low TMB cohort may be better candidates for immunotherapy than those from the higher TMB cohort.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ec135f15fc8160351aa36efc3ceee1240577672" target='_blank'>
              Comprehensive analysis reveals the molecular heterogeneity and the differences in prognosis and immunotherapy sensitivity in hypermutated colorectal cancer
              </a>
            </td>
          <td>
            Gaobo Bai, Xiaojun Yang, Liang Xiao, Chunhui Xiong
          </td>
          <td>2026-01-09</td>
          <td>Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Myeloid malignancies, including acute myeloid leukemia (AML) and myeloproliferative neoplasms (MPN), exhibit overlapping pathophysiology. Chronic MPNs can transform into secondary AML (sAML), which is associated with poor prognosis and limited treatment options. However, the process and prognostic significance of leukemic transformation remain incompletely understood. Through a two-sample bidirectional Mendelian randomization (MR) analysis, we showed that genetic liability to MPN significantly predicts the risk of developing AML, establishing MPN as the precursor to leukemia. To identify mediators of this risk, we integrated population-level plasma proteomics data, identifying 55 proteins associated with MPN. Upon integrative analysis with the BEAT-AML cohort, we developed a prognostic proteogenomic gene signature, showing that higher expression of CDCP1, CRISP3, and DCXR, alongside lower MPO levels, correlates with worse AML outcomes. We further performed pharmacogenomic analysis to identify vulnerability to PI3K/AKT/mTOR signaling pathway inhibition in high-risk AML. In vitro and in vivo experiments validated the efficacy of mTOR inhibition in myeloid malignancies. This gene signature effectively stratified patients by risk, with significant survival differences across the BEAT-AML and TCGA-LAML cohorts, and revealed immune alterations in high-risk groups, including elevated monocyte prevalence and cytokine signaling activity. Single-cell RNA sequencing (scRNA-seq) further suggested enrichment of these genes in progenitor cells and AML blasts. Drug sensitivity predictions suggested that high-risk AML patients may be particularly responsive to PI3K/AKT/mTOR signaling pathway inhibitors. Consistently, we observed upregulation of the genes in cell line models harboring MPN and AML mutations, which was suppressible via dual PI3K/mTOR inhibitor omipalisib. The efficiency of PI3K/mTOR inhibition in myeloid malignancies was further corroborated by results from multiple in vivo models. Together, our findings revealed shared molecular features across MPN and AML, identified a prognostic gene signature for risk stratification, and provided rationale for PI3K/mTOR inhibition as a promising therapeutic strategy in myeloid malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e3ab2eec986fab416d322c9846be08264b6919b" target='_blank'>
              A proteogenomic gene signature defines prognostic subgroups highlighting PI3K/AKT/mTOR signaling pathway as a therapeutic vulnerability in myeloid malignancies
              </a>
            </td>
          <td>
            Fan He, Shuyang Lin, Bei Gao, Varun Ramesh, Alexander Kim, Tim Kong, Daniel A. C. Fisher, Christopher T. Letson, Molly Brakhane, M. Fulbright, Yanbo Yu, Marco Sardiello, Jorge Di Paola, Stephen M Sykes, Stephen T. Oh
          </td>
          <td>2025-12-16</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Acute Myeloid Leukemia (AML) remains challenging to treat, especially in cases with mutations in the BCL-6 co-repressor (BCOR), which are associated with poor prognosis and chemo-resistance. In this study, we reveal a synthetic lethal interaction between BCOR and dihydroorotate dehydrogenase (DHODH). We demonstrate that BCOR-deficient cells have a heightened sensitivity to DHODH inhibitors such as brequinar and leflunomide, that are already in clinical use. We confirm that DHODH inhibition selectively induces cell death in BCOR-mutant cells in multiple cellular models, in malignant and non-malignant cells, through chemical and genetic manipulation. Interestingly, we find that the dependency on DHODH does not stem from its role in de novo pyrimidine biosynthesis disruption. Rather, DHODH’s role in the electron transport chain, essential for mitigating reactive oxygen species, may be the physiological vulnerability that pushes BCOR-mutant cells toward cell death when DHODH is inhibited. DHODH inhibitors could be repurposed as targeted therapies for BCOR-mutant tumors, offering a promising strategy for precision medicine in AML and other cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1feaf1e5a859257ecbbc92a0a9f0dffd57f185be" target='_blank'>
              BCOR mutations define a therapeutic vulnerability to DHODH Inhibition in acute myeloid leukemia
              </a>
            </td>
          <td>
            Florian Robert, Cherif Badja, Soraya Boushaki, Andrea Degasperi, Yasin Memari, Sophie E. Momen, T. Roumeliotis, Z. Kozik, Malgorzata Gozdecka, J. Choudhary, George Vassiliou, Gene CC Koh, S. Nik-Zainal
          </td>
          <td>2026-01-01</td>
          <td>Annals of Hematology</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="Despite its critical role in tumor evolution, a detailed quantitative understanding of the evolutionary dynamics of aneuploidy remains elusive. Here we introduce ALFA-K (Adaptive Local Fitness landscapes for Aneuploid Karyotypes), a method that infers chromosome-level karyotype fitness landscapes from longitudinal single-cell data. ALFA-K estimates fitness of thousands of karyotypes closely related to observed populations, enabling robust prediction of emergent karyotypes not yet experimentally detected. We validate ALFA-K’s performance using synthetic data from an agent-based model and empirical data from in vitro and in vivo passaged cell lines. Analysis of fitted landscapes suggests several key insights: (1) Whole genome doubling facilitates aneuploidy evolution by narrowing the spectrum of deleterious copy-number changes; (2) Environmental context and cisplatin treatment significantly modulate the fitness impact of these changes; (3) Fitness effects of copy-number changes depend on parental karyotype; and (4) Chromosome mis-segregation rates strongly influence the predominant karyotypes in evolving populations. The evolutionary dynamics of aneuploidy in solid tumors are challenging to study. Here the authors introduce a method, ALFA-K, which estimates karyotype fitness and predicts emergent karyotypes before experimental detection, and test its performance on synthetic and empirical data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e9f0c1aa74fa8dcd40ddabd793fc67f7d7cd6be" target='_blank'>
              ALFA-K: Local adaptive mapping of karyotype fitness landscapes
              </a>
            </td>
          <td>
            Richard J Beck, Tao Li, Noemi Andor
          </td>
          <td>2025-12-27</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="ABSTRACT Aging is a dominant risk factor for chronic diseases characterized by the functional decline of tissues and organs. During aging, the hematopoietic system declines in regenerative capacity—seemingly attributable to increases in DNA damage, replicative stress, and autophagic flux—resulting in skewing towards a myeloid lineage and away from a lymphoid lineage. Here, we characterized the transcriptomic and cellular landscape of the aged C57Bl/6J mouse hematopoietic system using a combination of bulk RNAseq and single cell RNAseq (scRNAseq). We show that aging leads to global transcriptional alterations in bulk peripheral blood mononuclear cells (PBMCs), lineage marker‐depleted bone marrow cells (Lin‐BM), and in hematopoietic stem and progenitor cells (HSPCs), immunophenotypically lineage marker negative (Lin‐) Sca1+ cKit+ (LSK+). These changes indicate widespread activation of inflammatory processes, namely in PBMCs and Lin‐BM cells. Interestingly, there is also a downregulation of cell cycle genes in HSPCs during aging. ScRNAseq across 39 hematopoietic cell types revealed age‐related skewing in cell composition. Aged PBMCs showed significant decreases in CD4 and CD8 naïve cells concomitant with increases in CD4/8 memory and CD8 exhausted T cell populations. Lin‐BM cells showed significant myeloid skewing in common myeloid progenitor (CMP) cells, as well as in the HSC population. We also identified a unique HSC population marked by increased Vwf, Wwtr1, and Clca3a1 expression that does not exist in young HSCs, thus likely marking true aged HSCs. Collectively, this work should serve as a useful resource for understanding and therapeutically targeting the aged hematopoietic system.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a19cc88f7b5c2f02b7b252356ac23e9d207486aa" target='_blank'>
              A Cellular and Transcriptomic Atlas of the Aged Mouse Hematopoietic System
              </a>
            </td>
          <td>
            Ryan R. White, Kun Xiong, Matthew H. Wakai, Allison Surian, Christina Adler, Nicole Negron, Min Ni, T. Shavlakadze, Yu Bai, David J. Glass
          </td>
          <td>2026-01-28</td>
          <td>Aging Cell</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC), a highly lethal disease often developing in the context of chronic liver disease, is marked by high relapse rates and low 5‐year survival. To investigate the multicellular ecosystem and molecular features of hepatocarcinogenesis and progression, a comprehensive analysis integrating single‐cell and spatial transcriptomics was conducted. Significant alterations were observed in various cell types, notably an increase in liver parenchymal and endothelial cells, which play critical roles in tissue repair and angiogenesis. Scissor+ cells, predominantly hepatocytes, were identified as potentially evolving into malignant cells with poor prognosis and enhanced immune evasion capabilities. Key genes associated with Scissor+ cells exhibited potential as cancer prognostic markers. Additionally, the disruption of cellular communication networks during malignant progression revealed the evolution of hepatocytes into cancerous phenotypes. The enhanced signaling pathways, such as MDK‐SDC4 and COL4A‐SDC, were identified as pivotal in driving the malignant transformation of hepatocytes. This transformative process, wherein Scissor+ cells acquire malignant features, highlights novel mechanisms of cancer progression. These findings provide deeper insights into hepatocarcinogenesis and offer potential avenues for targeted and personalized therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ea05c9d911e5acfb02ab472396d64afacfb09aa" target='_blank'>
              Spatiotemporal Evolution of Malignant Hepatocyte Clones Unveils Immune Evasion Features and Therapeutic Vulnerabilities in Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Shizheng Xiong, Dekang Ren, Jiaming Jin, Linjie Zhao, Chengjun Gong, Jiarui Liu, Wanjie Zheng, Zhiheng He, Haochuan Guo, Li Guo, Tingming Liang
          </td>
          <td>2026-01-01</td>
          <td>Biotechnology Journal</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d39ef025a1bb316bde2c6dad77b6828031b8946" target='_blank'>
              Functional characterization of the 9q34.13 locus identifies RAPGEF1 as modulating risk for melanoma and nevi via RAS activation
              </a>
            </td>
          <td>
            Rohit Thakur, Mai Xu, Alexandra M. Thornock, Joshuah Yon, Samuel Anyaso-Samuel, M. Lauss, Thomas Rehling, Linh Bui-Raborn, Hayley A. Sowards, Gerard Duncan, Lea Jessop, Timothy Myers, Raj Chari, Erping Long, K. Funderburk, Jinhu Yin, Rebecca Hennessey, Emory Hseih, Hannah Levin, Mitchell John Machiela, Tongwu Zhang, Goran Jonsson, D. T. Bishop, Julia Newton-Bishop, Jérémie Nsengimana, M. Iles, M. Landi, M. Law, T. Andresson, Jiyeon Choi, Leonard I. Zon, Jianxin Shi, Kevin M. Brown
          </td>
          <td>2025-12-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99817029794cbefbdaf247b4f46b15275fd31679" target='_blank'>
              Metabolic reprogramming in non-small cell lung cancer: unraveling therapeutic vulnerabilities and resistance mechanisms
              </a>
            </td>
          <td>
            T. Addissouky
          </td>
          <td>2026-01-17</td>
          <td>Journal of Rare Diseases</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25f989a17f3320271cfb93b4f26cc7fc5ffbb463" target='_blank'>
              YAP/TEAD drives treatment-induced adaptive immunosuppression in EGFR-mutant lung cancer
              </a>
            </td>
          <td>
            Meija Honkanen, Jonas Baumgarten, Nikol Dibus, Jenna H. Rannikko, Milos Gojkovic, Paloma Cejas, Yingtian Xie, Zsuzsanna Ortutay, Johannes A. M. Merilahti, Mari Tienhaara, Iman Farahani, Steffen Boettcher, P. Jänne, Otto Kauko, Raphael A Nemenoff, Lynn E. Heasley, Heidi M. Haikala, M. Hollmén, Kari J. Kurppa
          </td>
          <td>2026-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>152</td>
        </tr>

        <tr id="Cancer cells rewire their metabolism to sustain the high proliferative rate. Metabolism is therefore a common vulnerability of cancer cells, successfully exploited for therapeutic purposes. Intrinsic tumor characteristics and adaptive responses of cancer cells can however reduce the short and long-term efficacy of such a strategy. Understanding the determinants of therapy response and the mechanisms of chemoresistance is crucial to maximize therapy efficacy. In cancer, lysosomes undergo massive changes in their localization, size, and composition that support tumor progression. Additionally, lysosomes are one of the crucial drivers of chemoresistance via the drug sequestration or by facilitating adaptations to stress conditions. In the last decades, several reports have shown that lysosomal membrane proteins, such as the lysosome-associated membrane proteins 1 and 2 (LAMP1 and LAMP2), are deregulated in different cancer types and their expression has been correlated to drug efficacy. We performed an in silico gene essentiality and drug sensitivity screenings, revealing that LAMP2 expression is one of the determinants of resistance to inhibitors of de novo purine synthesis. In vitro experiments confirmed the in silico data and also showed that purine synthesis inhibitors trigger a ROS- and transcriptional-dependent increase of LAMP2. Our results identify the upregulation of LAMP2 expression as an adaptive response to purine synthesis inhibition to preserve cell viability and, in those tumors showing high LAMP2 levels, could also be an indicator of intrinsic resistance to these drugs that may be taken into consideration during the selection of the most appropriate therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b41a50acd50b88a1b8f2629c5c9719051133237" target='_blank'>
              The inhibition of de novo purine synthesis increases LAMP2 expression to preserve cell viability
              </a>
            </td>
          <td>
            Angela De Cristofaro, Serena Castelli, Federica Felice, M. Ciriolo, Enrico Desideri
          </td>
          <td>2025-12-11</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e06f5f17cb5b3f515140e594c820a578d4d25816" target='_blank'>
              The landscape of allele-specific expression in human kidneys
              </a>
            </td>
          <td>
            Ana C. Onuchic-Whitford, Junmo Sung, Eric D. Sakkas, M. McNulty, Christopher L. O’Connor, Anya Greenberg, Jihoon G. Yoon, Sowmya Badina, L. Mariani, Markus Bitzer, Matthew G. Sampson, Dongwon Lee
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9932f70712f8bef30c70a5b989c4e744cf379f36" target='_blank'>
              Harnessing transcriptomics for discovery of natural products to overcome acquired cancer resistance.
              </a>
            </td>
          <td>
            Heerim Yeo, Sang-Yun Kim, Sang-Min Park
          </td>
          <td>2025-12-08</td>
          <td>Archives of pharmacal research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Introduction Cancer origin patterns significantly influence cancer phenotypes and hallmark characteristics. Method In the present review, four distinct carcinogenesis trajectories that contribute to malignant transformation: mutator phenotype, chromosomal instability, dysmetabolism, and stemness, are explored. Results and Discussion In the mutator phenotype trajectory, deficiencies in DNA repair or synthesis systems lead to hypermutation and accumulation of oncogenic alterations. The chromosomal instability trajectory involves aneuploidy-induced copy number alterations in oncogenes and tumor suppressor genes. Dysmetabolic carcinogenesis is driven by the accumulation of oncometabolites due to alterations in metabolic genes. The stemness trajectory refers to the malignant transformation of cells possessing stem-like properties under oncogenic stimuli. Each trajectory independently promotes carcinogenesis and endows cancer cells with distinct characteristics. Notably, the primary oncogenic drivers in each trajectory can self-reinforce and form spontaneous-reinforcing loops that amplify oncogenic signals. Although crosstalk exists among trajectories, evidence suggests they are mutually exclusive during cancer origin. Therefore, targeting specific carcinogenesis trajectories and disrupting the self-reinforcing oncogenic loops may represent novel therapeutic strategies. Understanding carcinogenesis trajectories provides a framework for future cancer research and treatment approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c92c0ec788118fc8fac54bdd52f8df0eef75667c" target='_blank'>
              Carcinogenesis trajectories
              </a>
            </td>
          <td>
            Rui Wang, Zhaopeng Yan
          </td>
          <td>2025-12-09</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Ribosomal protein L5 (RPL5) is considered a haplo-insufficient tumor suppressor by upregulating p53 expression or promoting the inactivation of c-Myc in solid tumors upon tumor initiation. However, its detailed effect and mechanism in tumor maintenance were more complicated. Particularly, the specific role of RPL5 in acute myeloid leukemia (AML) remains unclear. In this study, we found that RPL5 expression was increased in primary AML blasts compared with normal controls, regardless of TP53 mutation. RPL5 knockdown reduced the survival and colony-forming ability of AML cells in vitro, as well as inhibited the engraftment of leukemia stem cells (LSCs) in vivo. It indicated that RPL5 was required for the survival of AML cells, especially for maintaining the stemness of LSCs. The data analysis of RNA-seq and proteomics revealed that RPL5 deficiency eradicated LSC by inducing a cellular stress response, i.e., ribosomal stress, rather than a specific function relating to RPL5. The inhibited PI3K-Akt-mTOR signaling pathway played a central role in the ribosomal stress induced by RPL5 deficiency. To investigate the selectivity of RPL5 depletion in eradicating LSCs, we found that RPL5 expression was highest in LSCs compared to AML cells and healthy controls. Moreover, ribosomal stress specifically affected transcripts with longer exon lengths and proteins with a lower isoleucine (Ile) to valine (Val) ratio. Ile and Val are glycogenic branched-chain amino acids (BCAAs) that regulate fundamental cell processes by affecting mTOR activation through BCAA metabolism. In conclusion, RPL5 depletion-induced ribosomal stress disrupted stemness maintenance by affecting BCAA metabolism in AML, specifically inhibiting the PI3K-Akt-mTOR signaling pathway, which resulted in LSC eradication.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fe87710ebba440d0ff11f095a1b7ea630473288" target='_blank'>
              RPL5 deficiency-induced ribosomal stress targets a select subset of proteins and inhibits the PI3K-Akt-mTOR signaling pathway to eradicate leukemia stem cells
              </a>
            </td>
          <td>
            Xiaofei He, Daijiao Ye, Jiayao He, Gangfeng Ding, Zihan Lin, Yaqian Qin, Bin Zhou, Liuzhi Shi, Xuanyu Yu, Chen Meng, Min Li, Chongyun Xing, Shenmeng Gao
          </td>
          <td>2025-12-18</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Chemoresistance remains a major barrier in cancer therapy, frequently resulting in treatment failure and reduced patient survival. This multifaceted phenomenon arises from the interplay of well-established mechanisms such as genetic mutations, non-genetic adaptations, and tumor microenvironment (TME) mediated influences as well as newly emerging findings from recent research (2020-present). Key biochemical contributors include diminished intracellular drug accumulation through altered uptake or efflux, dysregulation of drug metabolism and bioactivation involving multiple Phase I and Phase II enzymes, genomic instability affecting DNA repair pathways, disruption of cell cycle control, and evasion of apoptosis. In addition, recent evidence highlights the roles of epigenetic reprogramming, metabolic reconfiguration, and TME-derived signaling in amplifying chemoresistance. This review integrates both foundational concepts and recent advancements in understanding drug resistance, with particular emphasis on updated insights into drug-metabolizing enzymes and their impact on therapeutic failure. It also evaluates current and emerging strategies to overcome resistance including targeting metabolic enzymes, modulating the TME, and implementing polytherapy's that address multiple resistance pathways. By synthesizing established knowledge with recent discoveries, this review highlights promising directions for improving the efficacy of cancer treatments and enhancing patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b92ce9c249ee2d1807438a427a4bdf1ac44ab032" target='_blank'>
              Molecular basis of cancer chemoresistance: biochemical insights.
              </a>
            </td>
          <td>
            R. Guneidy
          </td>
          <td>2026-01-26</td>
          <td>Drug metabolism reviews</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Chronic myeloid leukemia (CML) is a myeloproliferative malignancy marked through the excessive proliferative growth of functional myeloid cells. Approximately 95% of CML patients harbor the BCR-ABL fusion gene (Philadelphia chromosome), key driver of leukemogenesis. Despite therapeutic advancements with tyrosine kinase inhibitors (TKIs) such as imatinib, challenges including drug resistance, tumor cell senescence, and minimal residual disease (MRD) persist, primarily due to leukemic stem cells (LSCs) and senescent cells that evade TKI-mediated eradication. Musashi-2 (MSI2), an RNA-binding protein, is elevated in aggressive CML and is associated with poor disease prognosis. While its role in LSCs is emerging, its contribution to resistance and senescence remains unclear. This study investigated the role of MSI2 in proliferation and senescence by using pharmacological and biochemical approaches in imatinib sensitive (K562 S) and resistant (K562 R) models. Proliferation was assessed by MTT and colony formation assays. Apoptosis and expression analysis were performed using DNA fragmentation assay and Real time polymerase chain reaction (RT-PCR) respectively. MSI2 was significantly overexpressed in both models (p = 0.0004 and p < 0.001). Pharmacological targeting of MSI2 reduced cell viability and colony formation in both sensitive (p < 0.001) and resistant (p < 0.001) cells. Combined therapy with imatinib and MSI2 inhibition had an additive effect on K562 S (p = 0.002) and a synergistic effect in K562 R (p = 0.0017). Mechanistically, MSI2 inhibition moderately induced apoptosis and significantly upregulated p53 (p = 0.004), indicating activation of apoptotic pathways. Notably, the levels of senescence-associated cell cycle regulators, p16, p21, and p27, were significantly decreased (p = 0.0025, 0.0052, and 0.0015, respectively), suggesting that growth suppression was not due to classical cell cycle arrest. MSI2 inhibition also disrupted Wnt/β-catenin signaling by downregulating Eya3 (p = 0.0008), c-Myc (p = 0.0128), and HIF-1α (p = 0.0050) expression. Imatinib monotherapy increased p16, p21, and p27 expression, indicating therapy-induced senescence (TIS). Subsequent MSI2 inhibition reduced the levels of these markers and IL-6, highlighting its role in regulating TIS at the transcriptional level. In conclusion, MSI2 inhibition, in combination with TKI therapy, has shown to overcome drug resistance and mitigate senescence in preclinical CML models, and suggesting a potential strategy to target CML LSCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/938338a82e540d0fbcc0c7bb146cd8d1a816a51a" target='_blank'>
              Targeting Musashi-2 to counteract senescence and resistance in chronic myeloid leukemia: enhancing the efficacy of imatinib therapy
              </a>
            </td>
          <td>
            Khadija Arif, Maryam Yousaf, Dilawar Khan
          </td>
          <td>2025-12-07</td>
          <td>BMC Cancer</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/433ea7c0b515392b57a3190c314827cc2576b6c4" target='_blank'>
              A Novel Patient-Derived Mutation in Glucocorticoid Receptor Reveals an Essential Role for DNA binding in Development and Endocrine Regulation
              </a>
            </td>
          <td>
            T. Tettey, Qizong Lao, Sonal Vaid, Sohyoung Kim, L. Varticovski, Vipula Kolli, Deborah P. Merke, Gordon L. Hager
          </td>
          <td>2025-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd77a79644efdb57765d420fb7be18f39a993472" target='_blank'>
              Cancer-associated synonymous mutations reveal stress signal-dependent mRNA folding that selectively modulates protein function
              </a>
            </td>
          <td>
            Sivakumar Vadivel Gnanasundram, Lixiao Wang, Sa Chen, Ondrej Bonczek, B. Vojtěšek, Robin Fåhraeus
          </td>
          <td>2026-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Abstract Cervical cancer remains a major global health burden. Despite standard-of-care therapies, 30–50% of locally advanced-stage patients develop treatment resistance, leading to recurrence and mortality. While tumour-intrinsic mechanisms (e.g., DNA methylation, cancer-associated fibroblasts) explain only partial resistance heterogeneity, emerging evidence identifies the microbiome as a critical modulator of therapeutic efficacy. This review synthesizes recent advances demonstrating that vaginal microbial dysbiosis, characterized by Lactobacillus iners enrichment and L. crispatus depletion, drives resistance through lactate-mediated metabolic rewiring, immune checkpoint stabilization and drug metabolism alteration. Longitudinal studies reveal dynamic microbiome trajectories during therapy, with geographic variations (notably HIV co-infection in sub-Saharan Africa) further modulating treatment responses. We critically evaluate microbiome-based interventions, from probiotics to engineered bacteria, including synthetic biology-driven precision microbiome therapies, and establishing standardized multi-centre trial protocols. Bridging mechanistic insights with clinical application represents a paradigm shift towards microbiome-informed cervical cancer management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a822ae4e52835707e7a0923e3a8688d6920b131" target='_blank'>
              Microbiome-driven resistance in cervical cancer therapy: from mechanistic dissection to clinical translation
              </a>
            </td>
          <td>
            Xiaoqi Wu, Dingjie Wang, Xiaodan Mao, Binhua Dong, Yue Wang
          </td>
          <td>2025-12-29</td>
          <td>Expert Reviews in Molecular Medicine</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="This review aims to synthesize current knowledge on Cyclin-dependent kinase 12 (CDK12) and Cyclin-dependent kinase 13 (CDK13), two transcriptional kinases with distinct and overlapping roles in gene regulation, genome stability, and cancer biology. These kinases control transcription elongation and co-transcriptional processing, including splicing and polyadenylation, thereby linking RNA polymerase II activity with DNA repair and replication stress response. CDK12 and CDK13 play context-specific roles across malignancies, where amplification, truncating mutations, fusions, and deletions variably alter their transcriptional output and downstream DNA repair programs. Their dysregulation contributes to oncogenesis, metastasis, and resistance to conventional therapies. There is growing therapeutic interest in CDK12/13 inhibition, especially as selective inhibitors of single DNA repair nodes have shown limited success. Selective CDK12/13-targeting agents have shown efficacy in preclinical models by inducing synthetic lethality, restoring sensitivity to PARP inhibitors, and are being explored to potentiate immune checkpoint blockade. This review highlights the biological rationale for targeting CDK12 and CDK13 in cancer, summarizes emerging therapeutic strategies, and identifies key opportunities for integrating these approaches into precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c6aa4a8b6800b768181f51eaa1d8b4bd48846cd" target='_blank'>
              CDK12 and CDK13 in oncology: from RNA regulation to therapeutic targeting
              </a>
            </td>
          <td>
            Julia Dudkiewicz-Garbicz, Paweł K Włodarski
          </td>
          <td>2026-01-05</td>
          <td>Cellular Oncology (Dordrecht, Netherlands)</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer therapy remains an active field of investigation, particularly in understanding and overcoming therapy resistance. Small non-coding RNAs, such as microRNAs (miRNAs), are emerging as key regulators of cancer survival, progression, proliferation, invasion, migration, and metastasis. Although many studies have linked miRNAs to cancer therapy outcomes, significant questions remain regarding their precise molecular and cellular roles in therapy resistance. Increasing evidence shows that miRNAs influence critical pathways such as apoptosis, immune evasion, and other signaling cascades. However, there have been many setbacks because of the limitations in knowledge of each specific miRNA’s function. A deeper understanding of miRNA expression and function may enhance the development of more effective cancer therapeutics and improve overall survival of patients. This review explores the role of miRNA expression as a key regulator of therapeutic resistance in cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be2eecd6c06795aaec7cf5be8c8739a720372740" target='_blank'>
              MicroRNAs as key regulators of cancer drug resistance: insights and future directions in chemotherapy, targeted-therapy, radiotherapy, and immunotherapy
              </a>
            </td>
          <td>
            Michael A Attathikhun, A. Jurj, George A. Calin
          </td>
          <td>2025-12-10</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fc0251706b1f215af0fc6511724feaeecccbbd1" target='_blank'>
              Wakhan: reconstruction of chromosome-scale copy number profiles of tumor genomes with long-read sequencing
              </a>
            </td>
          <td>
            T. Ahmad, A. Keskus, S. Aganezov, A. Goretsky, I. Rodriguez, B. Yoo, L. Lansdon, E. Repnikova, L. Zhang, Y. Liu, A. Donmez, A. Bryant, S. Tulsyan, J. Park, J. Gardner, B. McNulty, S. Sacco, J. Shetty, Y. Zhao, B. Tran, S. Malikić, C.-P. Day, K. Miga, B. Paten, C. Sahinalp, M. Farooqi, M. Dean, M. Kolmogorov
          </td>
          <td>2025-12-15</td>
          <td>None</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Simple Summary Acute leukemias are difficult-to-treat blood cancers which many patients will eventually die from despite intensive chemotherapies and bone marrow transplantation. Antibodies that recognize proteins on leukemia cells and deliver a cell toxin (so-called antibody–drug conjugates) have been developed to improve these outcomes. Two such drugs, gemtuzumab ozogamicin (GO, for acute myeloid leukemia) and inotuzumab ozogamicin (IO, for acute lymphoblastic leukemia), have been approved for use in patients but are not always effective. In our laboratory research, we undertook genome-wide screening to discover genes that are associated with response or resistance to the toxin (calicheamicin) that both GO and IO contain. Our studies revealed the importance of several DNA damage pathway regulation genes for the anti-leukemia activity of calicheamicin, including TP53, ATM, and MDM2. Building on these data, we then identified several small-molecule inhibitors that increased GO/IO efficacy—findings that support further evaluation of these combination therapies with clinical testing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e01e84d0ae6489ed9f585be97591dbaf9334e876" target='_blank'>
              DNA Damage Sensing and TP53 Function as Modulators of Sensitivity to Calicheamicin-Based Antibody–Drug Conjugates for Acute Leukemia
              </a>
            </td>
          <td>
            Camryn M. Pettenger-Willey, George S. Laszlo, Margery Gang, Frances M. Cole, Colin D. Godwin, Sarah Erraiss, Pritha Chanana, Allie R. Kehret, Junyang Li, Jacob W. Barton, Meghann M. Yochim, E. Rodríguez-Arbolí, R. Walter
          </td>
          <td>2025-12-25</td>
          <td>Cancers</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Identifying the mutant genes that are selected during carcinogenesis is key to identifying candidates for intervention and understanding the processes that promote transformation. Here we applied two selection metrics to study the dynamics of mutational selection in a mouse model of ultraviolet light driven skin carcinogenesis in which multiple synchronous tumors develop in each animal. Sequencing normal skin and tumors over a time course revealed two genetic routes to squamous carcinoma. Nonsynonymous Trp53 mutants were positively selected in both epidermis and tumors and present in 90% of tumors. The remaining tumors carried other oncogenic mutants, including activating Kras mutations. However, other positively selected mutant genes lost their competitive advantage in heavily mutated epidermis and in tumors. We found ten mutant genes under negative selection in normal skin, one of which was also negatively selected in tumors. In addition one gene was negatively selected in tumors but not normal skin. We conclude that analysing selection in normal tissue alongside tumors may resolve the dynamics of selection in carcinogenesis and refine the identification of cancer drivers. Sequencing mutants in both normal skin and tumors that arise from it in a mouse model of ultraviolet light driven carcinogenesis reveals mutant selection changes as cancers develop. Only p53 mutants are selected throughout squamous carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4638b6cb2b56a741aa25d0049e47e04146d5e21" target='_blank'>
              The dynamics of mutational selection in cutaneous squamous carcinogenesis
              </a>
            </td>
          <td>
            G. Skrupskelyte, J. Fowler, S. Dentro, Carine Winkler, Irina Abnizova, Niklaus Beumer, R. Sood, Thomas Quarrell, C. King, J. Kamarashev, Emmanuella Guenova, M. Gerstung, Benjamin A. Hall, Liliane Michalik, P. Jones
          </td>
          <td>2026-01-12</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) and myelodysplastic neoplasms (MDS) are clonal hematopoietic malignancies in which next-generation sequencing (NGS) has become integral for diagnosis, classification, risk stratification, and measurable residual disease (MRD) monitoring. Traditional cytogenetic and PCR-based assays remain useful, but targeted NGS panels now represent the standard of care, providing rapid and sensitive detection of recurrent gene mutations, structural variants, and gene fusions. Whole-genome, whole-exome, and RNA sequencing and long-read platforms expand the spectrum of detectable alterations, though targeted panels remain most practical for routine diagnostics. Bioinformatic pipelines and quality metrics—including read length, sequencing depth, and coverage—are critical for accurate variant calling, with validation often required for variants of uncertain significance or those near detection thresholds. NGS is now embedded in diagnostic frameworks, including the WHO 2022 and ICC classifications, which incorporate recurrently mutated genes such as TP53, ASXL1, RUNX1, and FLT3. These data inform prognostic models, with ELN-2022 defining adverse-risk AML subgroups for patients treated with intensive chemotherapy, ELN-2024 AML for patients treated with less-intensive therapies, and the IPSS-M refining MDS risk categories by integrating mutational data. NGS also enables MRD monitoring, with gene panels and PCR-NGS hybrid approaches (e.g., for FLT3-ITD) showing increasing clinical utility, though standardization is still lacking. Furthermore, diagnostic NGS frequently uncovers germline predisposition syndromes (e.g., DDX41, GATA2), with significant implications for treatment decisions and donor selection in transplantation. In this manuscript, we review the advantages, limitations, and future perspectives of NGS in the clinical management of AML and MDS with a particular emphasis on the biological and technical principles underlying its use in these diseases. Furthermore, we discuss how NGS findings may influence diagnosis, prognostic classification, and therapeutic decision-making within current clinical frameworks. Our aim is to provide a comprehensive overview of NGS fundamentals to support clinicians in navigating the increasing complexity of molecular data in daily practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36f92e8c1514cc884abf1a1cf442e829de555deb" target='_blank'>
              How to Read a Next-Generation Sequencing Report for AML and MDS? What Hematologists Need to Know
              </a>
            </td>
          <td>
            Salvatore Perrone, C. Tresoldi, Silvia Rigamonti, Matteo Molica, Nadezda Zhdanovskaya, Laura Cicconi
          </td>
          <td>2025-12-01</td>
          <td>Journal of Clinical Medicine</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Human T-cell leukemia virus type 1 (HTLV-1) infects mainly CD4+ T lymphocytes and causes both malignant and inflammatory diseases: the aggressive malignancy known as Adult T-cell leukemia/lymphoma (ATL) and several chronic inflammatory syndromes. HTLV-1 infection is established by integration of proviral DNA (~9000 bp) into the host genome. In HTLV-1, two viral genes, tax and HBZ, play critical roles in viral transcription and promotion of T-cell proliferation, respectively. The present study was undertaken to test the hypothesis that the higher-order structure of proviral chromatin regulates its transcription on both the plus strand and the minus strand. ATAC-seq analysis identified an open chromatin region in the pol gene of proviral DNA, which we name IPOR, in many ATL cases. Using reporter assays, it was found that the sequence of IPOR suppresses the transcription of the plus strand and activates that of the minus strand. Binding motifs of Eomes and TEAD proteins were predicted in this region, and we confirmed recruitment of the transcription factors to their respective motifs by ChIP-qPCR. A mutant of IPOR which cannot bind the transcription factors weakened the transcriptional activating effects compared with the wild type, suggesting that the IPOR and those transcription factors suppress the 5' long terminal repeat (LTR) but activate the 3'LTR. In addition, a mutant HTLV-1 molecular clone, which possesses the IPOR mutant, produced a higher titer of virus than the wild type. RNA-seq analysis of HTLV-1-infected cell lines, in which Tax expression can be traced after induction, revealed that EOMES expression decreases during the tax transcriptional burst and resumes following termination of the burst. These findings suggested that the expression dynamics of Eomes affect the transient expression of Tax. The IPOR sequence appears to regulate the transcription from both LTRs, suppressing the 5'LTR but activating the 3'LTR. Recruitment of Eomes to the IPOR is likely to influence the expression of Tax and HBZ. Since both viral genes are involved in diverse mechanisms for viral replication, cellular proliferation, and immune regulation, the IPOR may play a role in fine-tuning the modes of viral persistence in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68b2f1e6ae01e0a8f9b5438f95ad7675208ffd69" target='_blank'>
              Intra-pol proviral open region of HTLV-1 controls the transcription from both long terminal repeats
              </a>
            </td>
          <td>
            Miyu Sonoda, Azusa Tanaka, Xueda Chen, Wenyi Zhang, Helen Kiik, S. Ramanayake, K. Nosaka, N. Takenouchi, Masanori Nakagawa, Akihiro Fujimoto, C. Bangham, Masao Matsuoka, Jun-ichirou Yasunaga
          </td>
          <td>2025-12-08</td>
          <td>Retrovirology</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="TP53 mutations are among the worst prognostic factors in acute myeloid leukemia (AML), with affected patients facing relapse-free survival of just five-to-six months compared to TP53 wild-type patients. A major barrier to improving outcomes lies in the dearth of effective therapies, as TP53 mutant patients remain refractory to conventional cytotoxic chemotherapies, targeted therapies, and even allogeneic stem cell transplantation. In this review, we first summarize current clinical strategies and the major setbacks of p53 activators, MDM2/X regulators, and immunotherapy, highlighting the disconnect between promising pre-clinical studies and limited durable clinical responses. We next discuss the mechanisms of therapy resistance in TP53 mutant AML, with specific emphasis on dysfunction in the mitochondrial apoptotic pathway and clonal evolution of TP53 mutant hematopoietic stem cells. We then outline a roadmap for developing tailored therapies that may finally redefine prognosis for this high-risk patient population, including apoptotic activators, cell-cycle modulators, and immune- and metabolic-based therapies. We lastly call attention to new biomarker-driven approaches that can improve patient stratification and optimize identification of responders. By connecting mechanistic understanding with translational insights, this review underscores both the formidable challenges and the emerging opportunities in TP53 mutant AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2b2b9dab76dc863b017965cd1c9f9debef78076" target='_blank'>
              Pandora’s Box of AML: How TP53 Mutations Defy Therapy and Hint at New Hope
              </a>
            </td>
          <td>
            E. Olesinski, Shruti Bhatt
          </td>
          <td>2025-12-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Nucleotide substitutions are common in cancer cells, and they occur in both protein coding regions and non-coding regions (5′ and 3′ UTRs and introns). Although substitutions in non-coding regions have the potential to alter gene expression, it is the alteration of coding regions that affects protein function and has the most drastic effect on cellular transformation. Mutations in certain genes (e.g., TP53, KRAS) are common to nearly all cancers, but most cancers are characterized by specific gene mutation signatures. In this report, we investigated nucleotide substitution signatures in coding regions of the top 25 most frequently mutated genes in multiple human cancers. The goal was to examine whether unique nucleotide substitution biases are associated with various cancers. A pan-cancer analysis showed that the most altered nucleotide is guanine, which is biased towards G->A transitions. A per-cancer analysis identified ten cancers with biased substitutions in certain genes. Some of these biases were expected (e.g., KRAS in gastrointestinal cancers or JAK2 in blood cancers). Our analysis revealed biased signature substitutions in 17 genes, of which 14 were characterized as drivers and constituted a closely related set of cell cycle regulators. We conclude that nucleotide substitution biases contribute to specific alterations in cancer genes that produce cellular transformation. Principle component analysis of nucleotide substitutions shows that most cancers cluster together, meaning that they have similar nucleotide changes. However, certain cancers, most notably lung, pancreas, and blood cancers, can be differentiated from each other based on specific nucleotide signatures. Thus, nucleotide substitution patterns can be used to differentiate between some cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/baa7149b8cff77e69abfdaa8112e2209417108ee" target='_blank'>
              Nucleotide Substitution Biases in Related Cancer Driver Genes
              </a>
            </td>
          <td>
            Adam Khadre, Yifan Dou, Golrokh Mirzaei, Ruben C. Petreaca
          </td>
          <td>2025-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract Understanding DNA double-strand break (DSB) repair is crucial for the development of targeted anticancer therapies, yet the roles of many genes remain unclear. Recent studies show that disruption of known DSB repair genes can alter the sequence-specific distribution of mutations arising after DSB repair, suggesting that genome-wide perturbation screens could be leveraged to identify new DSB genes leading to distinct deviations from the expected wild-type distribution. Given the challenges of designing controls for a genome-wide screen, we explore the high gene throughput to forgo the use of traditional controls by reframing the analysis as an outlier detection problem, assuming that most genes have minimal influence on DSB repair outcomes. We propose MUSICiAn (Mutational Signature Catalogue Analysis), a compositional data analysis method that ranks gene perturbation impact on mutational spectra without controls by measuring deviations from the central tendency considering the distribution of all spectra. We show that MUSICiAn effectively estimates pseudo-controls for the Repair-seq screen, yielding 476 genes and 60 nontargeting controls. We further apply MUSICiAn to the first genome-wide screen of 18 406 genes with mutational spectra readout, MUSIC, reporting that MUSICiAn successfully recovers known DSB repair genes, highlights the spliceosome as a lesser-appreciated player, and reveals candidates for further investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16a1d3962960ca885e8c3efafcd749c2130bf3c7" target='_blank'>
              MUSICiAn: genome-wide identification of genes involved in DNA repair via control-free mutational spectra analysis
              </a>
            </td>
          <td>
            Colm Seale, Marco Barazas, R. van Schendel, M. Tijsterman, Joana P. Gonçalves
          </td>
          <td>2026-01-06</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54cabefaf605ac94e03143b5c4643d22a1076d5d" target='_blank'>
              miRNA-mediated cell-to-cell communications boost DNA repair during the Radioadaptative Response
              </a>
            </td>
          <td>
            María del Carmen Domínguez-Pérez, María Jesús Fernández-Ávila, Lourdes González-Vinceiro, L. Zannini, Héctor Peinado, Román González-Prieto, Néstor García-Rodríguez, Pablo Huertas
          </td>
          <td>2025-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Head-and-neck squamous cell carcinoma (HNSCC) remains a lethal malignancy with stagnant survival despite advances in surgery, radiotherapy, and systemic therapy. Beyond cetuximab and PD-1 inhibitors, there are only a few targeted options, which benefit only a minority of patients, underscoring the need for new biomarkers and druggable dependencies. Genome-wide clustered regularly interspaced short palindromic repeats (CRISPR) Cas9 screening now enables systematic, high-specificity investigation of gene function to reveal determinants of tumor proliferation, survival, and therapy response. Compared with RNA interference, CRISPR provides cleaner on-target knockout and more interpretable phenotypes, allowing efficient discovery of essential genes and synthetic-lethal interactions. Although the Cancer Dependency Map profiled 89 OSCC/HNSCC lines to nominate baseline dependencies, drug-perturbed states critical for understanding platinum resistance remain underexplored. Only a handful of HNSCC studies have applied genome-wide CRISPR cas9 screening: two mapped core essential genes; two mapped cisplatin resistance and radiation resistance; and others uncovered synthetic-lethal targets, including vulnerabilities to mTOR inhibition, EGFR inhibition, glutamine metabolism inhibition, and host determinants of oncolytic HSV-1 efficacy. This review synthesizes these findings, highlights methodological considerations (library design, coverage, and treatment duration), and integrates complementary functional data to prioritize targets for rational combinations. This review also provides information on the TCGA database and in vivo CRISPR screening that can accelerate precision therapeutics for patients with HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a31c18994958d899b77bdbb4cbe1e355e7819ca3" target='_blank'>
              Target Discovery in Head-and-Neck Squamous Cell Carcinoma: Genome-Wide CRISPR Screens Illuminate Therapeutic Resistance and Actionable Dependencies
              </a>
            </td>
          <td>
            V. K. Vincent-Chong
          </td>
          <td>2025-12-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Therapy resistance remains a major cause of relapse and poor outcomes in acute lymphoblastic leukemia (ALL). Recent multi-omics studies in ALL have revealed that resistance arises from a combination of leukemia-specific genetic lesions, treatment-driven clonal evolution, and adaptive non-genetic programs. Genomic analyses have identified recurrent alterations associated with resistance to chemotherapy, tyrosine kinase inhibitors, and immunotherapies, while single-cell profiling has uncovered heterogeneous cell states that persist during treatment and contribute to minimal residual disease. Emerging epigenetic, proteomic, and metabolic data further indicate that reversible regulatory and signaling changes play a central role in leukemic persistence. Integrative analyses are beginning to define convergent resistance pathways and clinically relevant biomarkers, although longitudinal sampling and clinical translation remain limited. This review summarizes the current multi-omics landscape of therapy resistance in ALL and discusses opportunities to improve risk stratification and therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98ef9b63ea903857bf083205a34581a4f1ebb5b7" target='_blank'>
              Multi-Omics Approaches to Understanding Therapy Resistance in Acute Lymphoblastic Leukemia
              </a>
            </td>
          <td>
            Xiuyun Wu, Jingxin Zhang
          </td>
          <td>2026-01-29</td>
          <td>Lymphatics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Outcomes for patients with esophageal squamous cell carcinoma (ESCC) remain poor, partly due to treatment resistance, particularly to DNA-damaging therapies. Poly(ADP‑ribose) polymerase 1 (PARP1) plays a critical role in repairing single‑strand DNA breaks (SSBs). Unrepaired SSBs can be converted into double‑strand breaks (DSBs) during DNA replication, potentially leading to cell death. Genomic amplification of the distal portion of chromosome 3q (3q26-q29) is a frequent copy‑number alteration in ESCC, which harbors genes encoding several oncoproteins. However, whether long noncoding RNAs (lncRNAs) from this region contribute to ESCC pathogenesis and treatment resistance remains poorly understood. In situ hybridization and qPCR were used to assess RNA expression. Protein PARylation was evaluated by immunoprecipitation followed by Western blotting. Cellular phenotypes were quantified using the cell counting kit-8, Annexin V/Propidium iodide staining, and clonogenic assays. DNA damage was monitored by immunofluorescence staining for phosphorylated histone H2AX (γH2AX) and p53-binding protein 1 (53BP1) and by comet assays. RNA-protein interactions were assessed through RNA pulldown coupled with mass spectrometry and RNA immunoprecipitation. Chromatin fractionation and detergent pre-extraction immunofluorescence were conducted to examine PARP1 chromatin association. ESCC growth and responses to treatments were evaluated using xenograft models. The lncRNA LINC00885, hereafter referred to as PARylator, was the most upregulated lncRNA encoded within the 3q26-q29 amplicon in ESCC. PARylator was predominantly nuclear and interacted with PARP1. Knockdown of PARylator increased γH2AX and 53BP1 foci and comet tail moment, triggered apoptosis, reduced clonogenicity, sensitized ESCC cells to cisplatin and ionizing radiation. In vivo, PARylator knockdown impaired tumor growth and increased cisplatin sensitivity in ESCC xenografts. Mechanistically, PARylator promoted PARP1 recruitment to chromatin and catalytic activation, thereby increasing PARP1 auto-PARylation and enhancing the PARylation of X-ray repair cross-complementing 1 (XRCC1). PARylator was further upregulated in response to DNA damage. The DNA damage-responsive, 3q26-q29 amplicon-encoded lncRNA PARylator promotes PARP1‑mediated PARylation and SSB repair, thereby limiting DSB accumulation and supporting ESCC cell survival and resistance to DNA-damaging therapies. Targeting PARylator, alone or in combination with DNA-damaging agents, may represent a novel avenue for ESCC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ecafccdcfd377c547198645eff6618f1001f7c3" target='_blank'>
              The lncRNA PARylator promotes PARP1 activation and resistance to DNA‑damaging therapy in esophageal squamous cell carcinoma
              </a>
            </td>
          <td>
            Teng Fei Qi, Yadong Liu, Jinkun Yang, Yi Meng Yue, Man Man Han, Hongtao Liu, Jing Li, Min Liu, Yuwei Zhang, Jia Hui Kou, Wen Jin Li, Xiaoying Liu, Ting La, Tao Liu, Song Chen, Xu Dong Zhang, Shundong Cang, Liu Teng, Tianli Fan
          </td>
          <td>2025-12-31</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Over 15% of cancers worldwide are caused by viruses. Merkel cell polyomavirus (MCPyV) is the most recently discovered human oncovirus and is the only polyomavirus that drives malignant tumors in humans. Here, we show that MCPyV+ Merkel cell carcinoma is defined by neuroendocrine-lineage core regulatory (CR) transcription factors (TFs) (ATOH1, INSM1, ISL1, LHX3, POU4F3, and SOX2) that were essential for tumor survival and that co-bound chromatin with the viral small T antigen at super enhancers. Moreover, MCPyV integration sites were enriched at these neuroendocrine super enhancers. We further discovered that the MCPyV noncoding control region contained a homeodomain binding motif absent in other polyomaviruses that bound ISL1 and LHX3 and depended on them for T antigen expression. To therapeutically target the CR factors, we used histone deacetylase (HDAC) inhibitors to collapse the chromatin architecture and induce topological blurring of superenhancer loops, abrogating core TF expression and halting tumor growth. To our knowledge, our study presents the first example of oncogenic cross-regulation between viral and human epigenomic circuitry to generate interlocking and essential transcriptional feedback circuits that explain why MCPyV causes neuroendocrine cancer and represent a tumor dependency that can be targeted therapeutically.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/adb14a3c17daae13a9f8ee1cd3546f3c3832ec32" target='_blank'>
              Interlocking host and viral cis-regulatory networks drive Merkel cell carcinoma
              </a>
            </td>
          <td>
            L. Miao, D. Milewski, Amy Coxon, T. Gelb, K. Garman, Jadon Porch, Arushi Khanna, L. Collado, Natasha T. Hill, Kenneth Daily, S. Vilasi, Danielle Reed, Tiffany Alexander, G. Starrett, Maharshi Chakraborty, Young Song, Rachel Choi, Vineela Gangalapudi, Josiah D. Seaman, Andrew Morton, K. Busam, C. Vakoc, Daniel J. Urban, Min Shen, Matthew D. Hall, Richard Sallari, Javed Khan, B. Gryder, Isaac Brownell
          </td>
          <td>2025-12-15</td>
          <td>The Journal of Clinical Investigation</td>
          <td>2</td>
          <td>98</td>
        </tr>

        <tr id="Loss of the tumor suppressor PTEN is common in T-cell acute lymphoblastic leukemia (T-ALL), and is associated with poor prognosis. PTEN-loss drives robust activation of AKT/mTORC1 signaling to promote leukemic cell growth. We find that PTEN-loss in T-ALL confers dependence on the guanylate nucleotide synthesis enzyme inosine 5’-monophosphate dehydrogenase (IMPDH) for cell growth and viability. This metabolic vulnerability is dependent on sustained mTORC1 signaling and can be exploited using clinically approved IMPDH inhibitors to selectively kill PTEN-deficient T-ALL cells, and extend survival in genetic and xenograft T-ALL models in mice. Mechanistically, IMPDH inhibitors cause early DNA replication stress, followed by DNA damage. In contrast to treatment with mTORC1 inhibitors, these events culminate in robust and selective cell death in PTEN-deficient T-ALL cells. These findings reveal a targetable metabolic vulnerability in T-ALL, which could provide rationale for repurposing clinically approved IMPDH inhibitors. Statement of Significance We find that the IMPDH inhibitors mycophenolic acid and mizoribine, which are currently used as well-tolerated immunosuppressants, exert anti-leukemic activity in an aggressive molecular subset of T-ALL cells that are associated with poor prognosis. The use of clinically approved compounds to exploit this vulnerability could lead to rapid drug repurposing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78b58170f74952bf740dfcc24ed5ab165c769edf" target='_blank'>
              PTEN-loss confers dependence on the guanylate synthesis enzyme IMPDH in T-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            R. A. Padder, Thu Le Le, Emily Hyde, Maria E. D. Rubio, E. Chiles, Namratha Sheshadri, Kelly Mulraney, Wei-Xing Zong, Xiaoyang Su, Daniel Herranz, Alexander J. Valvezan
          </td>
          <td>2025-12-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fef918e1d4b64862104d2dc24b6aa7569489a94" target='_blank'>
              Genomic profiling implicates candidate genes and mutagenic pathways driving lung cancer recurrence
              </a>
            </td>
          <td>
            L. Luhari, A. Valter, A. Bahcheli, K. C. Cheng, M. Bayati, A. Ustav, A. Velthut-Meikas, K. Oselin, J. Reimand
          </td>
          <td>2026-01-01</td>
          <td>None</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb493837934e32a1361cf64825c6efc8b0b26882" target='_blank'>
              Pan-cancer RNA editing activity reveals complex editing functions and cancer immunotherapy biomarkers
              </a>
            </td>
          <td>
            M. Guo, Yuanyan Xiong
          </td>
          <td>2026-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a heterogeneous hematologic malignancy with poor overall survival (OS). Resistance to chemotherapeutic drugs such as idarubicin (IDA) remains a major cause of treatment failure. This study investigated the anti-leukemic activity of halofuginone (HF) a synthetic derivative of the natural compound from hydrangea Dichroa febrifuge and its potential to overcome IDA resistance in AML cells. Apoptosis, proliferation, cell cycle, and colony formation were assessed in AML cells treated with HF. RNA sequencing (RNA-seq) was performed to identify the potential molecular targets of HF. The anti-leukemic efficacy of HF was further assessed in NOD/SCID-IL2Rγ (NSG) mice xenografted with human relapsed/refractory (R/R) AML samples. HF treatment significantly inhibited cell proliferation, reduced colony formation, and induced apoptosis in AML cells. By RNA-seq analysis, S100A8 and S100A9 (S100A8/A9) were identified as potential targets of HF, and HF treatment markedly suppressed their expression. Overexpression of S100A8/A9 abrogated the anti-leukemic effects of HF, indicating that S100A8/A9 are critical mediators of HF activity. Mechanistically, HF activated the amino acid starvation response (AAR), leading to phosphorylation of eukaryotic translation initiation factor 2 subunit alpha (p-eIF2α), subsequent downregulation of S100A8/A9, and elevation of cytoplasmic Ca2⁺ levels. Knockdown of eIF2α prevented HF-induced downregulation of S100A8/A9, confirming that HF regulates S100A8/A9 expression via the eIF2α pathway. Furthermore, HF treatment inhibited global protein synthesis, enhanced the cytotoxicity of chemotherapeutic drugs, and reversed IDA resistance by suppressing S100A8/A9 expression. Finally, HF inhibits leukemic infiltration and extended OS in MLL-AF9-transduced AML mice and enhanced IDA-induced anti-leukemic effects in R/R AML-xenografted NSG mice model. These findings reveal that HF exerts anti-leukemic effects by modulating the p-eIF2α–S100A8/A9–Ca2⁺ signaling axis in AML cells. HF represents a promising therapeutic candidate for AML, particularly for patients with IDA-resistant disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33130edb35192d5e46cf41861d141416fe885561" target='_blank'>
              Halofuginone exerts broad-spectrum cytotoxic effects by regulating p-eIF2α-S100A8/A9-calcium signaling, inhibiting global protein synthesis, and reversing the resistance of idarubicin in acute myeloid leukemia
              </a>
            </td>
          <td>
            Liuzhi Shi, Min Zhao, Chen Meng, Min Li, Xuanyu Yu, Shixin Zhang, Shirui Yu, Xinyao Chen, Bin Zhou, Chongyun Xing, Jinjun Jia, Jingying Zhou, Shenmeng Gao
          </td>
          <td>2026-01-06</td>
          <td>Chinese Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The MTAP (methylthioadenosine phosphorylase) gene, located on chromosome 9p21, plays a crucial role in the methionine salvage pathway and is frequently co-deleted with CDKN2A in various malignancies. Loss of MTAP expression leads to the accumulation of methylthioadenosine (MTA), which selectively inhibits protein arginine methyltransferase 5 (PRMT5) and creates a unique metabolic vulnerability in MTAP-deficient tumors. These alterations have emerged as promising therapeutic targets in precision oncology. Recent advances highlight the potential of exploiting MTAP loss through synthetic lethality approaches using PRMT5 and methionine adenosyltransferase 2A (MAT2A) inhibitors. Preclinical and early clinical data indicate that targeting these pathways can selectively impair tumor growth while sparing MTAP-proficient cells. Moreover, MTAP deletion has been associated with specific molecular and immunologic profiles that may influence treatment response and tumor microenvironment characteristics. This review summarizes current knowledge on the biological functions of MTAP, the mechanisms linking its loss to oncogenesis, and the evolving landscape of therapeutic strategies targeting MTAP-deficient cancers. Understanding these molecular dependencies offers novel opportunities for the development of precision-based therapies across diverse tumor types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40708c2db41533a7db6997687c20695f0be103e4" target='_blank'>
              MTAP Deletion as a Therapeutic Vulnerability in Cancer: From Molecular Mechanism to Clinical Targeting
              </a>
            </td>
          <td>
            P. Krawczyk, K. Wojas-Krawczyk
          </td>
          <td>2025-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Despite substantial progress in the field of bladder cancer management, the disease continues to represent an important health issue characterized by increased recurrence and progression rates. This is largely attributed to cancer stem cells (CSCs), a unique cell subpopulation capable of self-renewal, differentiation and resistance to conventional anti-cancer therapies. At the same time, our understanding of cancer biology has been revolutionized by the identification of non-coding RNAs (ncRNAs), a heterogeneous group of RNA molecules that do not translate into proteins yet function as pivotal regulators of gene expression. Emerging evidence demonstrates that ncRNAs modulate key hallmarks of CSCs, including self-renewal, epithelial–mesenchymal transition and drug resistance. This review investigates the intricate interplay between ncRNAs and the core signaling pathways that underlie bladder CSC biology. Unravelling the nexus between CSCs and ncRNAs is crucial for developing novel diagnostic biomarkers, better prognostic tools and innovative therapeutic strategies for patients with bladder cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13091e7103acf49fcdc024b48e3b7a3f0f4f481d" target='_blank'>
              The Crossroads of Cancer Regulation: Discussing the Role of Non-Coding RNAs in Bladder Cancer Stem Cells
              </a>
            </td>
          <td>
            Alexandros Georgiou, Dimitrios Triantis, M. Goulielmaki, V. Zoumpourlis
          </td>
          <td>2025-12-11</td>
          <td>Uro</td>
          <td>1</td>
          <td>28</td>
        </tr>

        <tr id="Mutations in DNA mismatch repair (MMR) pathway genes (MSH2, MSH6, MLH1, and PMS2) are linked to acquired resistance to temozolomide (TMZ) and high tumor mutation burden (TMB) in high-grade gliomas (HGG), including glioblastoma (GBM). However, the specific roles of individual MMR genes in the initiation, progression, TMB, microsatellite instability (MSI), and resistance to TMZ in glioma remain unclear. Here, we developed de novo mouse models of germline and somatic MMR-deficient (MMRd) HGG. Surprisingly, loss of Msh2 or Msh6 does not lead to high TMB, MSI, nor confer response to anti-PD-1 in GBM. Similarly, human GBM shows discordance between MMR gene mutations and TMB/MSI.Germline MMRd leads to promoted progression from low-grade to HGG and reduced survival compared to MMR-proficient (MMRp) tumor-bearing mice. This effect is not tumor cell intrinsic but is associated with MMRd in the tumor immune microenvironment, driving immunosuppressive myeloid programs, reduced lymphoid infiltration, and CD8+ T cell exhaustion. Both MMR-reduced (MMRr) and MMRd GBM are resistant to temozolomide (TMZ), unlike MMRp tumors. Our study shows that KL-50, a imidazotetrazine-based DNA targeting agent inducing MMR-independent cross-link-mediated cytotoxicity, was effective against germline and somatic MMRr/MMRd GBM, offering a potential therapy for TMZ-resistant HGG with MMR alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f59eb07f5af2f7c1bb36a92e8c3ca00e2e992ebc" target='_blank'>
              Mismatch repair deficiency drives malignant progression and alters the tumor immune microenvironment in glioblastoma models.
              </a>
            </td>
          <td>
            Montse Puigdelloses Vallcorba, N. Soni, Seung-won Choi, Kavita Rawat, T. Joshi, Sam Friedman, Alice Buonfiglioli, Angelo Angione, Zhihong Chen, Gonzalo Piñero, Gabrielle Price, Mehek Dedhia, Raina Roche, E. Radkevich, Anne M. Bowcock, Deepti Bhatt, Winfried Edelmann, R. Samstein, Timothy E. Richardson, N. Tsankova, A. Tsankov, R. Bindra, Raul Rabadan, Juan C Vasquez, D. Hambardzumyan
          </td>
          <td>2025-12-23</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33d403636ddf059830232f0617d3f6cb6b77d1ff" target='_blank'>
              Genotype-phenotype mapping identifies fetal-like CD55+ immunoregulatory cancer cells as mediators of immune escape in colorectal cancer
              </a>
            </td>
          <td>
            M. Mastel, Aitana Guiseris Martinez, Gabriele Diamante, Jasmin Meier, Luzi Schuchmann, Nikolaos Georgakopoulos, Ioannis Chiotakakos, Laura K. Steffens, Carolin Artmann, Diego Benitez, Michael Günther, Vera Thiel, Roxana Pincheira, A. Trumpp, R. Offringa, S. Ormanns, Rene Jackstadt
          </td>
          <td>2026-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a29ad767de6b1b60629bc324b66e333d5c7c4a2b" target='_blank'>
              A Human Genetics Framework for De-risking Gene Editing Targets for Hematopoietic Cell and Gene Therapy
              </a>
            </td>
          <td>
            A. Dhall, Alyssa Pyclik, Matthew H. Ung, Julia Etchin, Y. Keschner, Huanying Gary Ge, Tirtha Chakraborty, J. Lydeard
          </td>
          <td>2025-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="This review examines the role of METTL3, a core RNA methyltransferase, in therapeutic resistance in acute myeloid leukemia (AML) and discusses emerging strategies to address this challenge. METTL3 regulates N6-methyladenosine (m6A) modifications on transcripts involved in key cellular processes, including apoptosis (BCL2, MCL1), metabolism (PGC-1α, CSRP1), proliferation (MYC), autophagy (FOXO3), and bone marrow microenvironmental interactions (ITGA4, AKT1). These modifications enhance the stability and translation of resistance-associated genes, supporting leukemic cell survival under treatment pressure. Pharmacological targeting of METTL3 has shown efficacy in preclinical AML models. Inhibitors such as STM2457, METTL3-directed PROTACs, and rational drug combinations with agents including venetoclax, anthracyclines, and ATRA, have reversed resistance phenotypes and impaired leukemic cell fitness. Beyond canonical resistance mechanisms, METTL3 also regulates noncoding RNAs, autophagy, and metabolic–epigenetic crosstalk, including histone lactylation, linking epitranscriptomic regulation to broader resistance pathways. By integrating molecular, cellular, and microenvironmental evidence, this review underscores METTL3 as a central driver of drug resistance and a promising therapeutic target in relapsed or refractory AML. Unlike previous summaries, it highlights the convergence of METTL3-mediated m6A modifications with noncoding RNA regulation, autophagy, and niche adaptation, and critically evaluates emerging therapeutic approaches, including catalytic inhibitors, PROTACs, and natural compounds.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/778d6b30e769b0ee3df71fb1d4bc535c7e793070" target='_blank'>
              The role of m6A RNA methyltransferase METTL3 in drug resistance mechanisms in acute myeloid leukemia
              </a>
            </td>
          <td>
            Suresh Prajapati, Charmi Jyotishi, Mansi Patel, Reeshu Gupta
          </td>
          <td>2026-01-30</td>
          <td>Blood Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Minimal residual disease (MRD) has become a significant predictor of relapse and survival in acute myeloid leukemia (AML), indicating the extent of remission beyond traditional morphological evaluation. Although multicolor flow cytometry and quantitative PCR are essential methodologies in minimal residual disease identification, both are constrained by immunophenotypic variability, the necessity for stable molecular targets, and limited sensitivity. Advancements in next-generation sequencing (NGS) have revolutionized the minimal residual disease (MRD) field by enabling highly sensitive, mutation-driven identification of leukemic clones across a broad genomic landscape. Contemporary error-suppressed next-generation sequencing techniques—such as unique molecular identifiers, duplex sequencing, and single-molecule molecular inversion probes—have enhanced analytical sensitivity to the 10⁻⁵ to 10⁻⁶ range, enabling the detection of ultra-low-frequency variations with greater specificity. These techniques improve clinical risk classification, refine prognostication within genetically defined AML subtypes, and guide therapeutic options, including post-remission therapy, targeted inhibition, and the timing and intensity of allogeneic stem cell transplantation. Innovative applications, such as single-cell sequencing, cell-free DNA studies, and integrative multi-omic MRD evaluation, enhance the capabilities of genomics-based monitoring. Nonetheless, obstacles remain, such as differentiating cancer mutations from clonal hematopoiesis, standardizing analytical pipelines, establishing clinically relevant thresholds, and incorporating NGS MRD into standardized treatment protocols. This review encapsulates contemporary NGS methods for AML MRD diagnosis, assesses their clinical ramifications and constraints, and suggests future pathways necessary for comprehensive clinical integration. With advancements in the area, NGS-based MRD is set to become a pivotal element of precision-guided AML control.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95498e2323c0e771bb1cadf3c3f61d8cda3d3bb4" target='_blank'>
              Next-Generation Sequencing for Minimal Residual Disease Detection in AML: Current Technologies and Clinical Implications.
              </a>
            </td>
          <td>
            Chidinma Gab-Obinna, Ikemefula Oriaku, Oluchi Okechukwu, A. Olowookere, N. N. Onyedum, Precious Deborah Ojetunde ‎, Kwesi Egyin Taylor
          </td>
          <td>2025-12-10</td>
          <td>Oncology, Nuclear Medicine and Transplantology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90832a3d9b2f9576b36552899274c52bd58fcd98" target='_blank'>
              RBM8A confers oxaliplatin resistance in gastric cancer by maintaining EGFR mRNA stability
              </a>
            </td>
          <td>
            Yichun Ma, Ying Xiang, Mei Yang, Yanmei Zhu, Yani Pan, Nannan Zhang, Xueni Fu, Ping Jiang, Yinya Pan, Hai Wu, Qiong Yan, Wenjun Li, Hongji Tao, Shangtao Mao, Yufei Tao, Jiale Li, Linzhe Su, Guangtao Gao, Fangmei An, Zhangding Wang, Guifang Xu
          </td>
          <td>2025-12-07</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Mitochondria are highly dynamic organelles that integrate metabolic regulation, signal transduction, and programmed cell death with their canonical role in adenosine triphosphate (ATP) production. Their ability to undergo continuous remodeling through the opposing processes of fusion and fission is essential for maintaining cellular homeostasis, preserving organelle quality control, and enabling adaptive responses to metabolic and oxidative stress. Among the core regulators of mitochondrial dynamics, the dynamin-related guanosine triphosphatase (GTPase) OPA1 plays a central role in inner membrane fusion, cristae architecture maintenance, bioenergetic efficiency, and the modulation of redox balance and apoptotic signaling. Accumulating evidence indicates that dysregulation of OPA1 expression or activity contributes to the initiation and progression of multiple malignancies, underscoring its importance in tumor cell survival, proliferation, metabolic adaptation, and resistance to stress. Here, we summarize current knowledge on OPA1 dysregulation in cancer and, based on preliminary, unpublished in silico analyses, we highlight the growing relevance of OPA1 as a therapeutic target, particularly through its GTPase domain and the still understudied Interface 7. Overall, these findings outline how integrated computational approaches could potentially guide the identification of novel OPA1 modulators, offering a conceptual framework that highlights OPA1 as a promising, yet still largely underexplored, target in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01bbc85b07d21254bc1200e427724eda318625e7" target='_blank'>
              OPA1 as a Cancer Target: Molecular Mechanisms, Structural Insights, and Strategies for Drug Development
              </a>
            </td>
          <td>
            Antonio Curcio, Ludovica Ganino, Ilenia Valentino, Massimo Gentile, Stefano Alcaro, R. Rocca, A. Artese, N. Amodio
          </td>
          <td>2026-01-01</td>
          <td>Antioxidants</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/062e9996a43c3e3095b1126f2396b4c755a5c6ba" target='_blank'>
              Expanding the genetic landscape of inherited metabolic diseases using long-read sequencing and transcriptomic profiling.
              </a>
            </td>
          <td>
            Alejandro Soriano-Sexto, Obdulia Sánchez-Lijarcio, Leonardo Beccari, Natalia Castejón-Fernández, F. Leal, P. Alcaide, Belén de la Morena-Barrio, M. Bahíllo‐Curieses, Patricia Correcher, Rafael Hencke-Tresbach, Laura López, Elena Martín-Hernández, Raquel Yahyaoui, Magdalena Ugarte, P. Rodríguez-Pombo, Belén Pérez
          </td>
          <td>2026-01-26</td>
          <td>European journal of human genetics : EJHG</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Globally, breast cancer (BC) is the most prevalent cancer in women, and drug resistance is a foremost challenge in BC management that contributes to 90% of BC-related deaths. Drug resistance arises as cellular adaptation during treatment, but the underlying mechanisms remain complex and are not yet fully understood. Research studies have shown that most anticancer drugs are effective against the mass of tumor cells rather than Cancer Stem Cells (CSCs). They have the ability to become dormant upon exposure to anticancer drugs and acquire drug resistance over a period of time after silently modifying themselves to resist the drug environment. The maintenance of the stem cell phenotype is mostly driven by stem cell transcription factors (e.g., KLFs, Myc, ZEB2, RUNX1, TWIST1, OCT4, NANOG, FOXO3, SOX2, E2Fs, etc.), which are mediated by epigenetic changes in tumor cells at many levels. These recognized factors/pathways have been demonstrated to be promising therapeutic targets. Therefore, knowledge of epigenetics in regulating cancer stem cells via stem cell transcription factors may be a promising solution for the reversal of therapy-resistant BC. In this review, we emphasize stem cell transcription factors, cancer stem cells, and epigenetic regulation as critical drivers of drug resistance in BC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8bb0cff2c052008da468be3376723f26d8ad1c7" target='_blank'>
              Interplay between Epigenetic and Transcription Factors Mediating Drug Resistance via Stem Cells in Breast Cancer
              </a>
            </td>
          <td>
            Komal Barhela, A. Chaudhary, Richa Mishra, Nupur Nupur, Sri Krishna Jayadev Magani, H. Rudayni, Sanjay Kumar, Binayak Kumar
          </td>
          <td>2025-12-09</td>
          <td>ACS Omega</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Abstract Extrachromosomal DNA (ecDNA) amplification represents an emerging mechanism underlying oncogene amplification, tumor heterogeneity, and drug resistance in cancer. However, the biology of ecDNA remains poorly understood because tools to engineer ecDNAs and precisely monitor their dynamics are limited. In particular, genome engineering strategies have not been established for ecDNA, which exists in tens to hundreds of copies within a single cell. Here, we report a systematic validation of ecDNA editing using standard CRISPR-Cas9 system and optimized CRISPR-Cas9 system with safeguard single-guide RNAs (sgRNAs), in which the addition of cytosine extensions finely reduces excessive Cas9 activity. The conventional CRISPR-Cas9 system induced severe cytotoxicity and markedly reduced ecDNA copy number, together with frequent micronucleus formation. Knock-in efficiency was remarkably low, highlighting an intrinsic difficulty in editing ecDNA. In contrast, the safeguard sgRNA strategy not only alleviated cytotoxicity and ecDNA loss in a cytosine-length–dependent manner but also enabled efficient knock-in into multiple ecDNA per cell. Computational simulations suggested that the degree and temporal patterns of multiple DNA cleavage events shape cell death, micronucleus formation, and rapid expansion of knock-in ecDNA. Collectively, optimization of Cas9 activity using safeguard sgRNAs enables efficient and nondisruptive ecDNA engineering, providing a powerful tool to study ecDNA biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85607d9d15aeb32494567c03b21204776a014e41" target='_blank'>
              Optimized CRISPR-Cas9 system for efficient engineering of ecDNA in cancer cells
              </a>
            </td>
          <td>
            Yohei Sugimoto, Takeru Kachi, Yu Watanabe, Mei Kubokawa, Koichi Ogami, Masaki Kawamata, Seiko Yoshino, Hiroshi I. Suzuki
          </td>
          <td>2026-01-14</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c863981147c795d5bd3bdf0f0373efc730fa086e" target='_blank'>
              Genetic identification of the RAS proteostatic machinery and its failure to regulate oncogenic variants
              </a>
            </td>
          <td>
            J. Bigenzahn, F. Kartnig, Manuela Vollert, V. Sedlyarov, Giulio Superti-Furga
          </td>
          <td>2025-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83abedd58342641982b4fa326716bab2f81320dd" target='_blank'>
              Coordinate Up-regulation of Both TK1 and TS Expression in Cycling Cells: Mechanistic Basis and Implications for Dual-Targeted Cancer Therapy
              </a>
            </td>
          <td>
            Wanxing Wang, Jiawei Zhang, Guangyou Duan, Yiyao Zhang, Guofeng Xu, Zhi Cheng, Shunmei Chen, Xue Yao, Xin Li, Shan Gao
          </td>
          <td>2026-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Resistance to androgen receptor (AR)-targeted therapies remains a major clinical challenge in the treatment of castration-resistant prostate cancer (CRPC). Emerging evidence suggests that Enzalutamide resistance is not solely due to the loss of AR dependency but can also arise from epigenomic reprogramming of the AR cistrome toward noncanonical gene networks. Recent studies have revealed that this reprogramming is mediated by previously unrecognized coactivators, including CXXC5, TET2, and EZH2, which cooperate with AR to establish a transcriptional landscape that supports lineage plasticity and therapeutic evasion. These noncanonical AR transcriptional programs enable CRPC cells to survive under continued AR blockade, acting as a transitional state towards neuroendocrine differentiation. Pharmacologic disruption of these coactivators abrogates noncanonical AR activity and suppresses tumor growth, highlighting a tractable vulnerability. These findings redefine AR signaling in advanced disease, suggesting that targeting noncanonical AR coactivators could offer a novel therapeutic paradigm to overcome resistance. Advances in single-cell and epigenomic profiling are poised to delineate further the heterogeneity and dynamics of AR cistrome remodeling in treatment-refractory prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2554d5c2e43e53c55e16f6ae494da51cebc744ec" target='_blank'>
              Coactivator networks orchestrating noncanonical AR programs in enzalutamide-resistant CRPC
              </a>
            </td>
          <td>
            Ephraim Gardner, Sasikumar Ponnusamy, Remi Adelaiye-Ogala
          </td>
          <td>2026-01-12</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Replication stress (RS) is a hallmark of cancer, largely driven by oncogene activation. Due to high levels of RS, cancer cells depend heavily on the RS response mechanisms to avoid DNA damage. This dependency creates a therapeutic opportunity that can be exploited for more effective cancer treatment. This review synthesizes current mechanistic understanding of RS and RS response and further describes how targeted disruption of RS response proteins (ATR, Chk1, Wee1, PARP, RPA) has been used in preclinical and clinical studies. We summarize preclinical and emerging clinical evidence for exploiting RS for radiosensitization, and outline candidate biomarkers and functional assays for patient selection. We also highlight the links between RS, therapy-induced senescence and innate immune activation via the cGAS–STING (cyclic GMP-AMP synthase—Stimulator of Interferon Genes) pathway, and address current challenges and future directions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/290040ba46caa91c5606fcee3d46699f73704969" target='_blank'>
              Replication Stress in Cancer: Mechanistic Insights and Therapeutic Opportunities for Radiosensitization
              </a>
            </td>
          <td>
            Spyridon N. Vasilopoulos, Ioanna Tremi, I. Kotta-Loizou, A. Gkikoudi, O. Tsitsilonis, Sophia Havaki, A. Georgakilas
          </td>
          <td>2026-01-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Abstract Cancer development is driven by somatic evolution and clonal selection. However, traditional selective pressure analysis methods have treated all sites within a gene equally, such a gene-level model oversimplifies the complexity of cancer evolution. In this study, we introduced CN/CS-calculator, a novel site-specific method that can capture selective pressures acting across different gene sites. By deciphering the interplay between the selection pattern and the function of a gene in oncogenesis, CN/CS-calculator uncovers a unique class of mini-driver genes, which exhibit weak positive selection, with certain critical sites providing context-dependent promoter effects on the fitness of cancer subclones while others are constrained by evolutionary conservation. Our method emphasizes the importance of site-specific analysis in uncovering how subtle evolutionary forces shape cancer biology. The refined understanding offers new insights into the mechanisms of cancer heterogeneity and molecular evolution, with potential implications for advancing therapeutic strategies and prognostic assessments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f1b33838c929960cc9e2d6ebf3f9c1d9939a9da" target='_blank'>
              Identification of cancer mini-drivers by deciphering selective landscape in the cancer genome
              </a>
            </td>
          <td>
            Xunuo Zhu, Wenyi Zhao, Siqi Wang, Jingwen Yang, Jingqi Zhou, Binbin Zhou, Ji Cao, Bo Yang, Zhan Zhou, Xun Gu
          </td>
          <td>2026-01-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="The myelodysplastic syndromes (MDS) are clonal hematopoietic stem cell disorders characterized by cytopenia and a high risk of transformation to acute myeloid leukemia. In recent years, next-generation sequencing (NGS) has revealed common hotspot gene mutations in MDS, which are not only involved in disease progression, but also affect the responsiveness of different therapeutic strategies. Current research has revealed that ASXL1 mutations in MDS predict demethylating agents (HMAs) resistance, the combination of HMAs and Venetoclax (VEN) achieved an ORR of 87%. DNMT3A R882 mutations induce decitabine sensitivity via hemi-methylated enhancer trapping, and TET2 mutations enhance HMAs efficacy only in ASXL1 wild-type contexts (ORR 62.1% vs. co-mutated 19%). RUNX1 aberrations reduce chemotherapy responses (18.9% ORR in high-risk MDS) through DNA repair impairment, while BCOR/EZH2 loss drives cytarabine resistance. TP53 multi-hit lesions correlate with poor survival (OS <12 months) but respond to eprenetapopt-azacitidine (ORR 73%), and IDH1/2 inhibitors achieve durable remissions (ivosidenib ORR 83.3%, mOS 35.7 months). In this paper, we systematically illustrate the correlation between key gene mutations and the response to HMAs, chemotherapy and targeted therapies in MDS patients. This article summarizes the current evidence on gene mutations as predictive biomarkers and discusses the direction of individualized therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec81be0184a702014697cea757cfc35eac9ff937" target='_blank'>
              Hotspot gene mutations and treatment response in myelodysplastic syndromes (MDS): predictive biomarkers and targeted strategies
              </a>
            </td>
          <td>
            Rui Zhang, Wei Kou, Tao Wu, Rui Zhou
          </td>
          <td>2026-01-05</td>
          <td>Frontiers in Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Molecular profiling of human primary cell types is essential for understanding human biology. We present a transcriptome and proteome map of 28 primary human cell types. Three major clusters of epithelial, endothelial, and mesenchymal cell types were observed in both the transcriptome and proteome levels along with the discovery of cell type enriched molecules including GRAP and C1orf116. The epithelial cell specific protein C1orf116 was further validated using immunohistochemistry across various human tissues. An exhaustive protein database search considering 39 post-translational modifications (PTMs) revealed novel insights into the PTM landscape including identification of understudied PTMs such as serine O-acetylation and histidine methylation. This also enabled comprehensive characterization of proteins with diverse PTMs. Interestingly, an unexpectedly higher frequency of dioxidation on tryptophan compared to methionine led to the identification of oxidative mitochondria complex subunit proteins. Further, a search strategy accounting for alternative translational start sites, splice junctions and translational readthrough refined genome annotation using proteomic evidence. For example, peptides from translational readthrough including extended sequence of LDHB and MDH1 were detected representing the first peptide-level evidence of these protein readthrough isoforms. Our comprehensive transcriptome and proteome data revealed cell type-specific molecular cues and heterogeneity, offering new insights into disease mechanisms often overlooked by tissue proteomics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58bc593b57c9117036851d0c30cf78e5e938882b" target='_blank'>
              A transcriptomic and proteomic map of primary human cell types
              </a>
            </td>
          <td>
            Dong-Gi Mun, Anil K. Madugundu, S. Renuse, R. Nirujogi, Chan Hyun Na, Min-Sik Kim, Mayank Saraswat, Smrita Singh, M. G. Ramarajan, Shivani Tiwary, Jürgen Cox, Amol Prakash, Marc K. Halushka, Kathleen H. Burns, Richard K. Kandasamy, Akhilesh Pandey
          </td>
          <td>2026-01-14</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Simple Summary Neuroblastoma (NB) is a childhood cancer that arises when certain nerve-related cells fail to mature properly and become malignant. Our study focuses on a gene called INSM1, which is normally active during early cell development but is abnormally elevated in NB tumors. We aimed to understand how INSM1 contributes to cancer’s aggressiveness and poor differentiation. We discovered that INSM1 keeps tumor cells in an immature, stem-like state and is closely linked to a chemical pathway called the methionine cycle, which affects gene regulation through epigenetic changes. Disrupting this cycle alters INSM1 activity and may help push cancer cells toward normal development. We also found that retinoic acid, a compound known to promote cell differentiation, reduces INSM1 and another cancer-driving gene, N-Myc. These findings suggest that targeting INSM1 and its metabolic regulation could be a promising strategy to treat NB by encouraging tumor cells to mature and lose their malignant properties. This research provides new insights into the biology of NB and may lead to more effective therapies for children affected by this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cbb4304aef088d414eab3a9ced76da697e7e9f2" target='_blank'>
              Regulation of INSM1 Gene Expression and Neuroendocrine Differentiation in High-Risk Neuroblastoma
              </a>
            </td>
          <td>
            Chiachen Chen, Siyuan Cheng, Xiuping Yu, Yisheng Lee, Michael S. Lan
          </td>
          <td>2025-12-22</td>
          <td>Biology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Tumor adaptation and therapeutic resistance represent major limitations to sustained treatment efficacy in modern oncology. Despite advances in targeted therapies and personalized approaches, disease progression and relapse remain frequent, reflecting the intrinsic biological complexity of malignant tumors. This review synthesizes high-impact evidence addressing the mechanisms that drive tumor adaptation under therapeutic pressure, with particular emphasis on intratumoral heterogeneity, evolutionary dynamics, and cellular plasticity. The analyzed literature supports the view of cancer as a dynamic and evolving ecosystem composed of genetically and phenotypically diverse subpopulations that respond heterogeneously to treatment. Resistance emerges as a predictable consequence of Darwinian selection, shaped by both genetic alterations and non-genetic adaptive processes, including reversible drug-tolerant states and microenvironmental influences. The findings further highlight the limitations of static, single-time-point molecular profiling in capturing tumor complexity and guiding long-term therapeutic decisions. Evolution-informed strategies, such as adaptive therapy and rational treatment sequencing, are discussed as biologically grounded approaches that may improve disease control by managing, rather than attempting to eradicate, heterogeneous tumor populations. This integrative perspective is particularly relevant for diverse healthcare settings, including those in Latin America, where conceptual frameworks can support clinical decision-making in variable resource contexts. Overall, understanding tumor adaptation and resistance as emergent evolutionary phenomena provides a coherent foundation for advancing personalized oncology, improving clinical reasoning, and strengthening oncology education">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb1ab93e274f88a42001d0b9860cca9a59037481" target='_blank'>
              Evolutionary Dynamics of Tumor Adaptation and Therapeutic Resistance in Personalized Oncology
              </a>
            </td>
          <td>
            Alexis Ramírez Hernández, José Arnulfo Perez Carrillo, Juan Diego Paloma Meza, Adrián Jesús Santoyo Rojas, Patricio Sebastian Ordóñez Montalvo, Wilmer Rene Cujilema Guillin, Jorge Che Enseñat, Zuleika Morales
          </td>
          <td>2026-01-26</td>
          <td>IECCMEXICO</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="How endogenous retroviral elements (ERVs), a family of transposable elements, may promote tumor progression is not well understood. Tripartite motif-containing 28 (TRIM28/TIF1b/KAP1) is a key transcriptional co-repressor protein that represses ERV expression in many cell types including embryonic stem cells, neural progenitor cells, differentiated adult cells, and cancer cells. In this study, we investigated the effect of Trim28 deletion on the expression of ERVs using an immune competent genetically engineered mouse model for prostate cancer. We found Trim28 deletion in prostate tumors led to the expression of ERVs in prostates from both hormonally intact and castrated mice. ERVs can regulate the expression of neighboring genes, and we detected increased expression of several protein-coding genes near overexpressed ERVs. Our data suggest that Trim28 deletion in prostate tumor epithelial cells may promote an innate immune response. However, Trim28 deletion also led to excessive deposition of tumor extracellular matrix (ECM). Our findings suggest that ECM alterations downstream of ERV derepression could affect immune cells in the tumor microenvironment and may promote tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f0f3ac90397a0b7a69b88c44582035578d15374" target='_blank'>
              Expression of endogenous retroviral elements is associated with extracellular matrix remodeling in prostate cancer
              </a>
            </td>
          <td>
            Emily C. Williams, Dewanga R. Mayarata, Anelia Horvath, Katherine B. Chiappinelli, Maho Shibata
          </td>
          <td>2026-01-08</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Bladder cancer is a highly heterogeneous malignant tumor of the urinary system with high recurrence rates, posing significant challenges in its diagnosis and treatment. Advances in multi-omics research have elucidated the molecular mechanisms underlying the pathogenesis and progression of bladder cancer, including driver gene mutations (e.g., FGFR3, TP53/RB1), dysregulation of signaling pathways (such as PI3K/AKT/mTOR and RAS-MAPK), epigenetic alterations, non-coding RNA networks, tumor microenvironment remodeling, and metabolic reprogramming. This review systematically summarizes recent progress in translational research bridging molecular mechanisms to breakthroughs in precision therapy, covering the clinical applications and challenges of FGFR inhibitors, immune checkpoint inhibitors, antibody-drug conjugates, and gene therapies. Translational efforts are increasingly relying on molecular subtyping to develop subtype-specific treatment strategies. Although significant advances have been made in precision therapy for bladder cancer, critical research gaps remain, including tumor heterogeneity, therapy resistance, and insufficient validation of biomarkers. Future research directions emphasize the potential of liquid biopsy for non-invasive diagnosis and dynamic monitoring, rational combination therapies, multi-omics data integration, and artificial intelligence in advancing personalized treatment, providing a systematic and forward-looking perspective on precision medicine in bladder cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d14a6737ae3e58aea00cf12ac1c7b98be1125652" target='_blank'>
              Molecular mechanisms and translational advances in bladder cancer: from driver genes to precision therapy
              </a>
            </td>
          <td>
            Jinjie Xiang, Yuhui Luo, Baiyu Zhang, Kunbin Ke, Hao Li
          </td>
          <td>2025-12-08</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29746864ab4e71ce2bf45dd37a7db9f0d5365e74" target='_blank'>
              Spatial patterns of APOBEC mutagenesis in the tumour microenvironment of Asian breast cancer
              </a>
            </td>
          <td>
            Zi-Ching Tan, Yang Wu, Zhen Wei Neo, Mai Chan Lau, S. Teo, J. P. S. Yeong, S. Chang, Jia-Wern Pan
          </td>
          <td>2026-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background: Rapid advances in biotechnology provide researchers with the opportunity to integrate omics profiles (genomics, epigenomics, transcriptomics, proteomics, etc.) with multiple phenotypes or experimental conditions. In cancers such as acute myeloid leukemia (AML), where combination therapies are standard of care, identifying genetic drivers of drug resistance requires evaluating how genes are associated with multiple drug response phenotypes. Statistical analyses associating omics profiles with multiple phenotypes yield multiple significance values and rankings for each of many genes. There is a great need to consolidate these multiple rankings into a consensus ranking to prioritize specific genes for detailed follow-up wet-lab or clinical studies. Methods/Results: Here, we evaluate the well-known Fisher’s method, the sum of squared z-statistics (SSz), and the recently published cellMCD method as tools for gene prioritization. In simulation studies, cellMCD showed very similar or highly superior performance to the widely used Fisher’s and SSz methods. These advantages were also observed in an example application involving a CRISPR drug screen of an acute myeloid leukemia cell line. Conclusions: In summary, our results indicate that cellMCD should be more widely used for prioritizing discoveries from multiple omic association studies. These methods are available as an R package on github.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/936ca737a6856489d929417a1a5e1ec23e0e8b36" target='_blank'>
              cellMCD Effectively Discovers Drug Resistance and Sensitivity Genes for Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Dora Obodo, Nam H. K. Nguyen, Xueyuan Cao, P. K. Parcha, Christopher D. Vulpe, J. Lamba, S. Pounds
          </td>
          <td>2026-01-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Resistance to anti-cancer drugs remains a major challenge in chemotherapy. Combination therapy using sensitizers has emerged as a promising strategy to restore drug sensitivity in resistant cancer cells. However, experimental screening of sensitizers is laborious and costly, highlighting the need for computational methods that enable systematic and efficient prediction. We developed SOCAR, a network-based computational framework that predicts sensitizer drugs by integrating transcriptome profiles with molecular interaction networks. SOCAR identifies resistance-associated genes and network modules, and quantifies each drug's potential to reverse these resistance mechanisms. Applied to 4,009 drugs, SOCAR accurately predicted candidate sensitizers for tamoxifen-resistant breast cancer (AUROC = 0.90). In vitro assays validated that all twelve top-ranked candidates significantly reduced cell viability (p < 0.005) when co-administered with tamoxifen. Furthermore, protein activity analyses showed that resistance-module proteins were markedly altered after acquiring resistance but were restored to normal levels following combined treatment (p < 0.05). Collectively, SOCAR provides a systems-level framework for discovering novel sensitizers and elucidating mechanisms of resistance reversal.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab1acf5805cb5dd4152a1362e84cfaff70e9641d" target='_blank'>
              SOCAR: Network-based Computational Framework to Overcome Acquired Tamoxifen Resistance of MCF7 Cells.
              </a>
            </td>
          <td>
            Mijin Kwon, Young Min Woo, Woochang Hwang, Jong-Won Kim, Doheon Lee
          </td>
          <td>2025-12-05</td>
          <td>IEEE transactions on computational biology and bioinformatics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4dbc11170b8c3915b19ca1579bf9e3cf87692a5a" target='_blank'>
              Spatio-temporal resource landscapes govern the confinement and escape of therapy-resistant mutants in structured populations
              </a>
            </td>
          <td>
            Nico Appold, Timon Citak, Auguste Palm, Jona Kayser
          </td>
          <td>2025-12-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Multidrug resistance (MDR) is a central cause of chemotherapy failure and tumor recurrence and metastasis, and its mechanism involves enhanced drug efflux, target mutation, upregulation of DNA repair and remodeling of the tumor microenvironment. ABC transporter protein (P-gp, MRP, and BCRP)-mediated efflux of drugs is the most intensively researched aspect of the study, but the first three generations of small-molecule reversal agents were stopped in the clinic because of toxicity or pharmacokinetic defects. Natural products are considered as the fourth generation of MDR reversal agents due to their structural diversity, multi-targeting and low toxicity. In this paper, we systematically summarize the inhibitory activities of monoterpenes, sesquiterpenes, diterpenes and triterpenes against ABC transporter proteins in in vitro and in vivo models and focus on the new mechanism of reversing drug resistance by blocking efflux pumps, modulating signaling pathways such as PI3K-AKT, Nrf2, NF-κB and remodeling the tumor microenvironment. For example, Terpenoids possess irreplaceable core advantages over traditional multidrug resistance (MDR) reversers: Compared with the first three generations of synthetic reversers, natural/semisynthetic terpenoids integrate low toxicity (mostly derived from edible medicinal plants, half-maximal inhibitory concentration IC50 > 50 μM), high target specificity (e.g., oleanolic acid specifically inhibits the ATP-binding cassette (ABC) transporter subtype ABCC1 without cross-reactivity with ABCB1), and multi-mechanistic synergistic effects (e.g., β-caryophyllene simultaneously mediates the dual effects of “ABCB1 efflux inhibition + apoptotic pathway activation”). These unique characteristics enable terpenoids to effectively circumvent key limitations of traditional synthetic reversers, such as high toxicity and severe drug–drug interactions. Among them, lupane-type derivative BBA and euphane-type sooneuphanone D (triterpenoids), as well as dihydro-β-agarofuran-type compounds and sesquiterpene lactone Conferone (sesquiterpenoids), have emerged as the core lead compounds with the greatest translational potential in current MDR reverser research, attributed to their potent in vitro and in vivo MDR reversal activity, low toxicity, and excellent druggable modifiability. At the same time, we point out bottlenecks, such as low bioavailability, insufficient in vivo evidence, and unclear structure–activity relationship and put forward a proposal to address these bottlenecks. At the same time, the bottlenecks of low bioavailability, insufficient vivo evidence and unclear structure–activity relationship have been pointed out, and future research directions such as nano-delivery, structural optimization and combination strategies have been proposed to provide theoretical foundations and potential practical pathways for the clinical translation research of terpenoid compounds, whose clinical application still requires further in vivo validation and translational research support.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc8e568cf68a8569a80453b9e89afe2909550d62" target='_blank'>
              Terpenoids: Emerging Natural Modulators for Reversing ABC Transporter-Mediated Multidrug Resistance in Cancer Chemotherapy
              </a>
            </td>
          <td>
            Lanfei Ma, Dina Mahemuti, Yuanhong Lan, Jianxiong Xu, Wenfang Li, Zhengding Su, Jinyao Li, Aytursun Abuduwaili, Ayitila Maimaitijiang
          </td>
          <td>2026-01-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Acute myeloid leukemia (AML) is characterized by a complex interplay between genomic alterations, aberrant hematopoiesis, and immune evasion. The aryl hydrocarbon receptor (AHR) pathway is a critical player in this phenomenon determining the fate of stem cell differentiation as well as dictating immune cell development and function. Despite this critical connection, little is known about how AHR regulates the immune microenvironment in AML. Methods We performed a retrospective study examining the pre-treatment effect of immune cell numbers (T and NK cells) in the bone marrow and their impact on overall survival in AML patients undergoing 7+3 induction chemotherapy. Utilizing flow cytometry and both bulk and single-cell RNA sequencing of AML patient samples, we characterized the immune signature of blast cells and the influence of AHR on the immune microenvironment. Lastly, we performed functional studies to determine impact of pharmacological and genomic AHR inhibition on NK cell function. Results Higher bone marrow NK cell percentage in ND-AML correlated with poorer OS and expression of HLA-E on leukemic blasts. AHR upregulation was associated with HLA-E expression on blasts and an innate immune resistant signature defined by upregulation of key cytokine pathways, interferon gamma (IFN-g) pathway, and MHC class I/II as well as impaired NK cell profiles. High AHR expression in AML was associated with monocytic maturation and discrepant MHC class I/II profiles. Pre-treatment of blasts with an AHR inhibitor (AHRi) prior to NK cell killing assay downregulated key checkpoint molecules, including HLA-E, and key IFN-g signaling transcription factors (STAT1, IRF1) and led to enhanced NK cell killing among multiple FAB subsets in AML. Conclusion The data support targeting the AHR pathway as a dual tumor intrinsic and immune targeting therapeutic strategy for AML, particularly in combination with NK cellular therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7893eab44f440eeeac15c67dc9e41e9217e3fc7f" target='_blank'>
              The aryl hydrocarbon receptor is associated with monocytic AML and innate immune resistance reversible with an AHR inhibitor
              </a>
            </td>
          <td>
            Jennifer N Saultz, D. Bottomly, Faith Burns, K. Byrd, Yoko Kosaka, Bernhard Alber, Daniel J Chandra, Stephen Kurtz, Guang Fan, A. Kaempf, Nicola Long, Marta Sanchez-Martin, Lei Wang, Karen McGovern, D. S. Kaufman, Shannon K. McWeeney, B. Druker, Jeffrey W. Tyner, Evan F. Lind
          </td>
          <td>2025-12-09</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>128</td>
        </tr>

        <tr id="Esophageal carcinoma (EC), comprising esophageal squamous cell carcinoma (ESCC) and adenocarcinoma (EAC), remains a deadly malignancy predominantly affecting individuals over 60 years of age. Although advances in diagnosis and treatment have improved outcomes for some cancers, EC still carries a poor prognosis, particularly in the elderly. Aging contributes significantly to EC pathogenesis via accumulated genetic mutations, metabolic deregulation, immunosenescence, and a pro-tumorigenic microenvironment. In this review, we synthesize current understanding of the molecular alterations that drive EC, with particular emphasis on age-related factors. We examine key genomic mutations (e.g., TP53, chromosomal instability), metabolic reprogramming (including glycolysis, glutaminolysis, and lipid metabolism), and their interplay with the tumor microenvironment (TME). We also highlight the influence of aging processes such as clonal hematopoiesis, senescence-associated secretory phenotype (SASP), and immune exhaustion in modulating tumor behavior and therapeutic resistance. We propose that aging-associated metabolic and immunologic alterations represent promising targets for therapeutic intervention. Greater integration of aging biology into EC research and clinical strategy is needed to advance personalized care for elderly patients. This review provides a conceptual foundation for future translational and clinical studies at the intersection of oncology and aging.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfdcaff8428584b8741ff99971d31c4c36a210c4" target='_blank'>
              Aging-associated mechanisms and metabolic vulnerabilities in esophageal carcinoma: an integrative review
              </a>
            </td>
          <td>
            Qiwei Liu, Zhiyu Xie, Wei Li, Mengxiang Li, Yijun Qi
          </td>
          <td>2025-12-16</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06b83eddceaa6549ab81ac78629425aa46eb6aed" target='_blank'>
              MSH2 Prevents Liver Tumorigenesis by Regulating Cell Cycle Checkpoints under Chronic Inflammation.
              </a>
            </td>
          <td>
            Shigeharu Nakano, Atsushi Takai, Eriko Iguchi, Yosuke Fujii, Haruka Amino, Masayuki Ueno, Takahiko Ito, Mari Teramura, M. Mishima, Ken Kumagai, Tadashi Inuzuka, Haruhiko Takeda, Y. Eso, Takahiro Shimizu, T. Maruno, Hiroshi Seno
          </td>
          <td>2026-01-29</td>
          <td>Carcinogenesis</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Cancer driver mutations alone are often insufficient to fully explain tumorigenesis. We demonstrate that these mutations cooperate with somatic copy number variations (CNVs) in a tissue-specific pattern of genomic epistasis. Analyzing 93,462 tumors, we identified 54 gene-cancer type pairs with significant co-occurrence of somatic mutations and CNVs. Our new Binoculars algorithm, which resolved phased DNA/RNA reads, revealed frequent preferential amplification in oncogenic mutation alleles, including AKT1 p.E17K, BRAF p.V600E, KRAS p.G12C/D/V, NRAS p.Q61K, and a fraction of gain-of-function TP53 p.R175H. Conversely, deletions selectively targeted the reference alleles, leading to loss of heterozygosity of IDH1 p.R132H and tumor suppressor mutations, including CDKN2A and TP53 truncations. Lung cancer patients carrying co-occurrences of somatic mutation-CNVs in TP53 and KRAS showed poorer survival than those carrying the same gene mutations. These findings reveal epistasis of cancer mutations and CNVs at an allelic resolution, suggesting specific genomic events to enhance patient stratification and therapeutic targeting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c67a7d5e6bdabbcb4e758c260b7827b1f0b38f4" target='_blank'>
              Widespread Epistasis between Cancer Driver Mutations and Allele-Specific Copy Number Variations
              </a>
            </td>
          <td>
            Serge Merzliakov, Guanlan Dong, Andrea Castro, Mahad Bihie, Yve Nichols-Evans, Hanna Carter, Kuan-lin Huang
          </td>
          <td>2025-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancer therapy resistance emerges from highly integrated molecular systems that enable tumor cells to evade cell death and survive cytotoxic therapeutic stress. High Mobility Group Box 1 (HMGB1) is increasingly gaining recognition as a central coordinator of these resistance programs. This review delineates how HMGB1 functions as a molecular switch that dynamically redistributes between cellular compartments in response to stress, with each localization enabling a distinct layer of resistance. In the nucleus, HMGB1 enhances chromatin accessibility and facilitates the recruitment of DNA repair machinery, strengthening resistance to radio- and chemotherapeutic damage. Cytosolic HMGB1 drives pro-survival autophagy, maintains redox stability, and modulates multiple regulated cell death pathways, including apoptosis, ferroptosis, and necroptosis, thereby predominantly shifting cell-fate decisions toward survival under therapeutic pressure. Once released into the extracellular space, HMGB1 acts as a damage-associated molecular pattern (DAMP) that activates key pro-survival and inflammatory signaling pathways, establishing microenvironmental circuits that reinforce malignant progression and therapy escape. HMGB1 further intensifies resistance through upregulation of multidrug resistance transporters, amplifying drug efflux. Together, these compartmentalized functions position HMGB1 as a central node in the networks of cancer therapy resistance. Emerging HMGB1-targeted agents, ranging from peptides and small molecules to receptor antagonists and nanoformulations, show promise in reversing resistance, but clinical translation will require precise, context- and redox-informed HMGB1 targeting to overcome multifactorial resistance program in refractory cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bcd9023ae13f0952b85c73d6ba319988bfa2cf22" target='_blank'>
              HMGB1: A Central Node in Cancer Therapy Resistance
              </a>
            </td>
          <td>
            Bashar A. Alhasan, Boris A. Margulis, Irina V. Guzhova
          </td>
          <td>2025-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="Myeloid leukemias, diseases marked by aggressiveness and poor outcomes, are frequently triggered by oncogenic translocations. In the case of chronic myelogenous leukemia (CML), the BCR-ABL fusion initiates chronic phase disease with second hits allowing progression to blast crisis. Although Gleevec has been transformative for CML, blast crisis CML remains relatively drug resistant. Here, we show that MSI2-HOXA9, a translocation with an unknown role in cancer, can serve as a second hit in driving bcCML. Compared to BCR-ABL, BCR-ABL/MSI2-HOXA9 led to a more aggressive disease in vivo with decreased latency, increased lethality, and a differentiation blockade that is a hallmark of blast crisis. Domain mapping revealed that the MSI2 RNA binding domain RRM1 had a preferential impact on growth and lethality of bcCML relative to RRM2 or the HOXA9 domain. Mechanistically, MSI2-HOXA9 triggered global downstream changes with a preferential upregulation of mitochondrial components. Consistent with this, BCR-ABL/MSI2-HOXA9 cells exhibited a significant increase in mitochondrial respiration. These data suggest that MSI2-HOXA9 acts, at least in part, by increasing expression of the mitochondrial polymerase POLRMT and augmenting mitochondrial function and basal respiration in blast crisis. Collectively, our findings demonstrate for the first time that translocations involving the stem and developmental signal MSI2 can be oncogenic and suggest that MSI, which we found to be a frequent partner for an array of translocations, could also be a driver mutation across solid cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a0a174e1fc7002e60c4dfa2851f21e71b1198a5" target='_blank'>
              Identification of a Musashi2 translocation as a novel oncogene in myeloid leukemia
              </a>
            </td>
          <td>
            Kyle Spinler, Michael Hamilton, Jeevisha Bajaj, Yutaka Shima, Emily Diaz, M. Kritzik, T. Reya
          </td>
          <td>2026-01-07</td>
          <td>eLife</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="
 Tumour heterogeneity, encompassing genetic, epigenetic, and microenvironmental diversity, remains a fundamental obstacle in precision oncology. Traditional bulk sequencing captures only averaged molecular profiles, thereby masking rare yet functionally critical subpopulations that drive malignant progression and therapeutic resistance. Recently, the emergence of single‐cell sequencing technologies has overcome the limitations of bulk approaches, enabling high‐resolution analyses of the genome, transcriptome, epigenome and proteome at the single‐cell level. These advances have enabled detailed mapping of tumour ecosystems, identification of key cellular subtypes, reconstruction of evolutionary trajectories and elucidation of intercellular communication networks within the tumour microenvironment. Accumulating evidence demonstrates that single‐cell technologies elucidate fundamental aspects of tumour biology and reveal potential diagnostic and therapeutic targets. This review systematically summarises the recent advances and applications of single‐cell sequencing in the field of precision oncology, with particular emphasis on its applications in mechanistic discovery, diagnosis, therapy, and prognosis. Furthermore, we discuss current challenges related to technology, data analysis, and clinical translation, and outline future research directions. In summary, single‐cell sequencing has profoundly reshaped our understanding of tumour biology and is propelling oncology into a new era of precision, prediction, and personalisation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11cbf7015a15504645fc0387a8b9ed997f9b6eef" target='_blank'>
              Single‐cell sequencing for cancer precision medicine: From mechanism discovery to diagnosis and therapeutics
              </a>
            </td>
          <td>
            Yue Zhao, Hancong Li, Huanzuo Yang, Gongshuang Zhang, Xinyue Fu, Peiheng Li, Puxing He, Shuang Wu, Han Luo
          </td>
          <td>2026-01-29</td>
          <td>Clinical and Translational Discovery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The discovery of lactylation, a post-translational modification derived from lactate, has fundamentally altered the perception of cancer metabolism. Once regarded as a metabolic waste product, lactate is now recognized as a central fuel source, a signaling molecule, and an epigenetic substrate capable of reprogramming gene expression and cellular function. Lactylation integrates metabolic reprogramming, tumor plasticity, and immune suppression, thereby orchestrating cancer initiation, progression, and resistance to therapy. This review provides a critical and integrative commentary on recent advances in lactylation biology, drawing from biochemical, epigenetic, and immunological perspectives. It synthesizes mechanistic insights into lactylation, highlights its role in tumorigenesis and the tumor microenvironment (TME), and evaluates therapeutic strategies that target lactate production, transport, and lactylation machinery. By dissecting consensus, controversies, and unresolved questions, we argue that lactylation represents both a hallmark of tumor adaptation and a potential Achilles’ heel for intervention. We further discuss future research directions, including comprehensive lactylome mapping, structural biology of lactylated proteins, microbiome-derived lactate, and clinical translation. Ultimately, lactylation is not merely a byproduct of glycolysis but a metabolic language that tumors employ to communicate, adapt, and thrive. Decoding this language may open new frontiers in cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f11df3d617ec11389a48bfdc33dfa42dd8ab08b2" target='_blank'>
              Lactylation in cancer: mechanistic insights, tumor microenvironment, and therapeutic horizons
              </a>
            </td>
          <td>
            Qiang Yang, Zhibo Yang, Hai Zhao
          </td>
          <td>2026-01-09</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Antibody-based therapeutics targeting tumor surface markers have transformed cancer treatment; however, their efficacy is frequently limited by tumor escape mechanisms such as antigen loss, phenotypic switching, and heterogeneous target expression. Beyond genetic or transcriptional changes, RNA alternative splicing (AS) has emerged as a central post-transcriptional mechanism driving antigenic diversity and immune escape. This review outlines how AS-generated isoforms remodel surface antigen structure and function across key therapeutic targets—including CD/19/CD20/CD22, EGFR/HER2, VEGF, and PD-1/PD-L1—thereby promoting resistance to monoclonal antibodies, antibody–drug conjugates, and immune checkpoint inhibitors. The aberrant activity of splicing regulators disrupts canonical exon selection, leading to altered receptor signaling or the secretion of soluble decoy isoforms that evade immune recognition. Emerging therapeutic strategies aim to counteract these processes through antisense oligonucleotide-mediated splicing correction, pharmacologic modulation of splicing regulators, and isoform-selective antibody or CAR-T designs. Collectively, understanding splicing-driven antigenic plasticity reveals an additional, dynamic layer of resistance regulation and provides a framework for developing RNA-informed precision antibody therapies designed to restore antigen expression, overcome immune escape, and enhance durable clinical responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f436ace9769f00110e75a91e51a1642377c416e" target='_blank'>
              Alternative Splicing-Mediated Resistance to Antibody-Based Therapies: Mechanisms and Emerging Therapeutic Strategies
              </a>
            </td>
          <td>
            Sanga Choi, Jieun Kang, Jung-Hyun Kim
          </td>
          <td>2025-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac5062560ab2a4d1c7e4c81040ac2034b6307511" target='_blank'>
              Synthetic Hybrid Receptors for Safer and Programmable T Cell Therapy
              </a>
            </td>
          <td>
            Maxwell G. Foisey, Julie M. Garcia, Xun Li, Xiaoyu Yang, Claire Hilburger, Chloë Thienpont, Alejandro Chaves-Martinez, Serkan Belkaya, Tina Truong, Iowis Zhu, Raymond Liu, Axel Hyrenius-Wittsten, Ricardo Almeida, Brian R. Shy, Greg M. Allen, Sarah Wyman, Kole T. Roybal
          </td>
          <td>2026-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Aneuploidy is near-ubiquitous in cancer and contributes to tumor biology. However, the temporal evolutionary dynamics that select for aneuploidy remain uncharacterized. We performed longitudinal genomic analysis of 755 samples from 167 patients with colorectal-derived neoplasias from different stages through metastasis and treatment. Adenomas had few copy number alterations (CNA) and most were subclonal, whereas cancers had many clonal CNAs, suggesting that progression goes through a CNA bottleneck. Individual colorectal cancer glands from the same tumor had similar karyotypes, despite evidence of ongoing instability at the cell level. CNAs in metastatic lesions, after therapy, and in late recurrences were similar to the primary. Mathematical modeling indicated that these data are consistent with the action of negative selection on CNAs that "trap" cancer genomes on a fitness peak characterized by specific CNAs. Hence, progression to colorectal cancer requires traversing a rugged fitness landscape, whereas subsequent CNA evolution is constrained by negative selection.


SIGNIFICANCE
We profiled 167 long-term responders longitudinally (755 samples), documenting long-term cancer evolution. We found that a genetic bottleneck is required for progression and is associated with dramatic increase in CNAs but decrease in clonal diversity. After initiation, copy number evolution is constrained by negative selection through metastasis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95c71eacb567a19cb84e29c03543a7fbd15b5975" target='_blank'>
              Negative Selection Maintains Grossly Altered but Broadly Stable Karyotypes in Metastatic Colorectal Cancer.
              </a>
            </td>
          <td>
            William Cross, S. Nowinski, George D. Cresswell, M. Mossner, Abhirup Banerjee, B. Lu, Marc J. Williams, G. Vlachogiannis, Laura J Gay, Ann-Marie Baker, C. Kimberley, F. J. Whiting, Hayley L. Belnoue-Davis, Pierre Martinez, Maria Traki, Viola Walther, Kane Smith, J. Fernández-Mateos, Erika Yara-Romero, Erica A Oliveira, Salvatore Milite, G. Caravagna, Chela T James, G. Elia, A. Berner, Chang-Ho Ryan Choi, Pradeep Ramagiri, Ritika Chauhan, Nik Matthews, Jamie Murphy, A. Antoniou, Susan K Clark, M. Mitchison, J. Chin Aleong, Enric Domingo, Inma Spiteri, S. McDonald, Darryl Shibata, M. Laclé, L. Wang, Morgan Moorghen, Ian P.M. Tomlinson, Marco Novelli, Marnix Jansen, A. Watson, Nicola Valeri, Nicholas A. Wright, John Bridgewater, M. Rodriguez-Justo, C. Barnes, Hemant M Kocher, Simon J. Leedham, A. Sottoriva, Trevor A. Graham
          </td>
          <td>2026-01-21</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Introduction Prostate cancer (PC) is one of the most prevalent malignancies in men, with rising incidence and mortality rates globally. Despite advances in therapeutic options such as androgen deprivation therapy and chemotherapy, effective cures, especially for advanced stages of the disease, remain limited. Recent research has highlighted the significant roles of alternative splicing (AS) and noncoding RNAs in tumor progression and drug resistance. This study aims to investigate the role of circIMP3, derived from the IMP3 gene, in prostate cancer development. Methods In this study, we employed quantitative PCR, RNA sequencing, and immunoblotting to identify and characterize circIMP3 in prostate cancer tissues and patient blood samples. Functional assays, including cell proliferation and in vivo tumorigenicity assays, were conducted to assess the biological role of circIMP3 in PC cells. RNA immunoprecipitation sequencing (RIP-seq) was used to identify alternative splicing events regulated by circIMP3. Additionally, exosome isolation and uptake assays were performed to explore the paracrine signaling function of circIMP3 within the tumor microenvironment (TME). Results We identified circIMP3, which is significantly upregulated in both prostate cancer tissues and peripheral blood of patients. CircIMP3 contains an internal ribosome entry site (IRES) and encodes a previously uncharacterized 288-amino-acid protein, circIMP3_288aa. Functional assays revealed that circIMP3_288aa promotes cell proliferation in vitro and accelerates tumor growth in vivo. Mechanistically, circIMP3_ 288aa regulates the alternative splicing of FBXW7, leading to impaired c-Myc ubiquitination and stabilization, which enhances oncogenic signaling. RIP-seq analysis identified over 2,000 alternative splicing events regulated by IMP3, with a notable enrichment in pathways related to ubiquitin-mediated proteolysis. Furthermore, circIMP3 is secreted into the TME via exosomes, where it is taken up by recipient cells, contributing to their proliferation. Discussion Our findings demonstrate that circIMP3 acts as a key regulator of both intracellular alternative splicing and extracellular paracrine signaling within the TME. The ability of circIMP3 to influence FBXW7 splicing and stabilize c-Myc provides a mechanistic basis for its role in promoting oncogenesis in prostate cancer. Clinically, high expression levels of circIMP3 correlate with poorer event-free survival in prostate cancer patients, suggesting its potential as a prognostic biomarker. Additionally, the detection of circIMP3 in peripheral blood positions it as a promising target for liquid biopsy applications in PC diagnosis and monitoring.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c44898723fc23dea0100abadafda89694a83449" target='_blank'>
              A novel protein encoded by circIMP3 promotes prostate cancer progression by regulating alternative splicing and tumor microenvironment
              </a>
            </td>
          <td>
            Wenren Zuo, Weizhou Huang, Haojie Chen, Yan Xu, Yang Zhang
          </td>
          <td>2026-01-05</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Tumor evolution involves genetic, transcriptional, and phenotypic alterations that shape cancer cell behavior and interactions with the microenvironment. While single-cell technologies have advanced our understanding of this process, spatial dynamics remain incompletely characterized. Here, whole-exome sequencing (WES), imaging mass cytometry (IMC), and spatial transcriptomics (ST) were integrated to study molecular evolution and immune responses in two lung adenocarcinoma (LUAD) mouse models: a genetically engineered model (129S4/Sv-KrasLSL-G12D, termed 129S4 K) and a carcinogen-induced precancer model (129S4 U). Compared to 129S4 K, 129S4 U tumors exhibited higher mutational, neoantigen but lower copy number variation (CNV) burdens at matched developmental timepoints, consistent with findings of higher mutational burden in human smoking-related LUAD than nonsmoking LUAD. We profiled over 1.4 million spatial single cells from 284 IMC regions of interest and 51,531 spatial transcriptomic spots from 156 lesions across 141 mice. Macrophage abundance increased with tumor progression, while CD8 T-cell and B-cell densities declined in late-stage LUAD. 129S4 U showed greater immune infiltration in both tumor and adjacent normal tissue, higher T-cell cytotoxicity signature score in line with its higher mutational and neoantigen burdens. LUAD progression was marked by early morphological shifts and late-stage changes in cell states and interactions. These data define spatial and genetic landscapes of LUAD development and provide a framework for investigating immune evolution and therapeutic strategies in early carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a0e7ad7c0953d4cab7fb1f9dcb251447c108ead" target='_blank'>
              Spatial Profiling Reveals Distinct Molecular and Immune Evolution of Mouse Lung Adenocarcinoma Precancers with or Without Carcinogen Exposure.
              </a>
            </td>
          <td>
            Bo Zhu, M. Aminu, Pingjun Chen, Jian-Rong Li, Chuanpeng Dong, Chenyang Li, Yanhua Tian, Shao-Wei Lu, Hong Chen, Chenxi Ma, Xin Hu, Jie-Tong Ye, Andrew Y Liu, Beibei Huang, Frank R. Rojas, Parra Cuentas Edwin Roger, Ou Shi, M. Nilsson, A. Poteete, Khaja B Khan, Wei Lu, Luisa M. Solis Soto, Junya Fujimoto, C. Haymaker, Ignacio I. Wistuba, Zhubo Wei, Linghua Wang, Don L. Gibbons, Ken Chen, Alexandre Reuben, Jason M Schenke, J. Heymach, Chao Cheng, Jia Wu, Jianjun Zhang
          </td>
          <td>2026-01-25</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9139b0ede19d5b7c3ea40131744831c88cd696ed" target='_blank'>
              Loss of KDM6A-mediated genomic instability and metabolic reprogramming regulates response to therapeutic perturbations in bladder cancer.
              </a>
            </td>
          <td>
            Pratishtha Singh, R. D’Rozario, Bidisha Chakraborty, Swadhin Meher, D. Raychaudhuri, Aminah J. Tannir, Yang Li, Anurag Majumdar, Jessalyn Hawkins, Yun Xiong, Philip L. Lorenzi, Padmanee Sharma, K. Akdemir, P. Pilié, Abhinav K. Jain, Byron H Lee, Sangeeta Goswami
          </td>
          <td>2026-01-07</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="As a type of cell with self-renewal ability and multi-directional differentiation potential, stem cells are closely related to their functions, such as reprogramming transcription factors, histone modifications, and energy metabolism. m6A (N6-methyladenosine modification) is one of the most abundant modifications in RNA, and dynamic reversible m6A modification plays an important role in regulating stem cell function. This review moves beyond listing isolated functions and instead adopts an integrated perspective, viewing m6A as a temporal regulator of cellular state transitions. We discuss how m6A dynamically regulates stem cell pluripotency, coordinates epigenetic and metabolic reprogramming, and serves as a central hub integrating key signaling pathways (Wnt, PI3K-AKT, JAK-STAT, and Hippo). Finally, using somatic reprogramming as an example, we elucidate the stage-specific role of m6A in complex fate transitions. This comprehensive exposition not only clarifies the context-dependent logic of m6A regulation but also provides a precise framework for targeting the m6A axis in regenerative medicine and cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e397129d10e01c5414b01cf61047665a5c7f45a" target='_blank'>
              The Regulatory Role of m6A Modification in the Function and Signaling Pathways of Animal Stem Cells
              </a>
            </td>
          <td>
            Xiaoguang Yang, Yongjie Xu, Suaipeng Zhu, Mengru Wang, Hongguo Cao, Lizhi Lu
          </td>
          <td>2026-01-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b30edb27f07cff97af34aacf8166230cf5b8614" target='_blank'>
              Unsupervised Identification of Cancer Attractor States through the Lens of Embryonic Origin and Cancer Hallmarks
              </a>
            </td>
          <td>
            P. Deshpande, Prabal Deb, Ibrahim Al Haddabi, B. Itkin
          </td>
          <td>2026-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background: Medulloblastoma is a highly malignant primary neuroembryonic tumor characterized by its unique anatomical location, cellular origin, and clinical manifestations, serving as a major contributor to childhood mortality. The disease exhibits biological heterogeneity, with significant variability in cells of origin, genetic mutation profiles, and prognoses across different subgroups, posing challenges for effective treatment. Aberrant DNA methylation has been identified as a promoter of tumorigenesis, influencing the tumor microenvironment and patient prognosis. Objective: This study aimed to investigate epigenetic changes in different medulloblastoma subtypes by integrating genomics, transcriptomics, epigenetics, and clinicopathological data to identify potential therapeutic targets and drugs that could significantly improve patient outcomes. Methods: We obtained medulloblastoma transcriptomic and methylation data (GSE85217 and GSE85212) from the NCBI GEO database and performed differential gene expression and methylation analysis using the limma and ChAMP packages. Functional pathway enrichment was assessed via GO and KEGG analysis. A prognostic model was constructed using LASSO regression, while WGCNA and GSEA were employed to analyze key gene modules and signaling pathways. In addition, CIBERSORT and Gene set variation analysis (GSVA) were used to evaluate the immune microenvironment and drug sensitivity. Results: We identified 1135 differentially expressed genes and 2582 differentially methylated sites, with inhibin beta B and ubiquitin-specific peptidase 2 significantly upregulated in Group 3/Group 4 subtypes. The prognostic model comprised 25 genes, and risk stratification effectively distinguished high- and low-risk patients (AUC 0.76–0.78). Immune analysis revealed decreased plasma cells and monocytes in the high-risk group, alongside increased naïve B cells and M0 macrophages. Drug prediction suggested that atovaquone and embelin may reverse tumor progression. Conclusion: Inhibin beta B and ubiquitin-specific peptidase 2 are key marker genes for predicting medulloblastoma patient prognosis stratification, with their expression patterns closely linked to molecular subtypes and the immune microenvironment. This study provides novel molecular targets and strategies for precision therapy and prognostic assessment in medulloblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06be784b4c82f0645bbe92b36a19fa4077518d9f" target='_blank'>
              Exploring the prognostic molecular mechanisms of medulloblastoma through methylation–transcriptome integration
              </a>
            </td>
          <td>
            Kaisai Tuerxun, Wenyu Ji, Turtuohut Tukebai, Dong Liu, Junhong Zhao, Yongxin Wang
          </td>
          <td>2026-01-01</td>
          <td>SAGE Open Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancer remains a major global health challenge, and although immunotherapy has achieved remarkable breakthroughs, its efficacy is often limited by tumor-induced immunosuppression within the tumor microenvironment (TME). Emerging evidence indicates that metabolic reprogramming plays a pivotal role in shaping the TME and regulating antitumor immune responses. Targeting tumor and immune cell metabolism has therefore become a promising strategy to enhance the effectiveness of cancer immunotherapy. This review first summarizes the metabolic reprogramming that occurs within the TME, including alterations in glucose, lipid, and amino acid metabolism in tumor cells, as well as the metabolic adaptation of immune cells. We then highlight recent advances in natural products that modulate key metabolic pathways and their potential to reshape the immunosuppressive TME. Special emphasis is placed on natural compounds that not only inhibit tumor cell metabolism but also restore the metabolic fitness of immune cells, thereby improving antitumor immunity. In addition, advances in delivery strategies, including nanocarrier-based and stimuli-responsive systems, are reviewed for their roles in improving the bioavailability, stability, and tumor targeting of natural metabolism-regulating agents. Finally, we discuss the current status and challenges of translating natural metabolism-regulating agents into clinical applications, including issues of dose optimization, safety evaluation, and patient selection. Despite these hurdles, precision targeting of metabolic pathways, interdisciplinary collaboration, and the discovery of novel compounds—particularly immune-sensitizing agents derived from traditional medicine—are expected to accelerate progress. Collectively, natural products represent promising adjuvant strategies for cancer immunotherapy, with great potential to overcome current therapeutic limitations and improve clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8c77bd01b532b4a1633bcd32fd11764fdfa763e" target='_blank'>
              Natural products as metabolic modulators to enhance cancer immunotherapy: reprogramming the tumor microenvironment
              </a>
            </td>
          <td>
            Xiaoyu Fan, Shushu Guo, Wanfang Li, Yingyin Wang, Jie Bao, Jiuming He, Hongtao Jin
          </td>
          <td>2026-01-14</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The stable and safe integration of exogenous DNA into the genome is crucial to both genetic engineering and gene therapy. Traditional transgenesis approaches, such as those using retroviral vectors, result in random genomic integration, posing the risk of insertional mutagenesis and transcriptional dysregulation. Safe harbor sites (SHSs), genomic loci that support reliable transgene expression without compromising endogenous gene function, genomic integrity, or cellular physiology, have been identified and characterized across various model organisms. Well-established SHSs such as AAVS1, ROSA26, and CLYBL are routinely utilized for targeted transgene integration in human cells. Recent advances in genome architecture, gene regulation, and genome editing technologies are driving the discovery of novel SHSs for precise and safe genetic modification. This review aims to provide a comprehensive overview of SHSs and their applications that will guide investigators in the choice of SHS, especially when complementary sites are needed for more than one transgene integration. First, it outlines safety and functional criteria that qualify a genomic site as a safe harbor site. It then discusses the two primary strategies for identifying SHSs: i) traditional lentiviral-based random transgenesis, and ii) modern genome-wide in silico screening followed by CRISPR-based validation. This review also provides an updated catalogue of currently known SHSs in the human genome, detailing their characteristics, uses, and limitations. Additionally, it discusses the diverse applications of SHSs in basic research, gene therapy, CAR T cell-based therapy, and biotechnological production systems. Finally, it concludes by highlighting challenges in identifying universally applicable SHSs and outlines future directions for their refinement and validation across biological systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/600741ca6d853466b695798a17050f6f9f6dbe8e" target='_blank'>
              Human Genome Safe Harbor Sites: A Comprehensive Review of Criteria, Discovery, Features, and Applications
              </a>
            </td>
          <td>
            Amer Ahmed, D. Di Molfetta, G. N. Iaconisi, Antonello Caponio, Ansu Singh, Aasia Bibi, V. Dolce, Luigi Palmieri, Vincenzo Coppola, G. Fiermonte
          </td>
          <td>2026-01-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Cancer has become a leading cause of mortality worldwide, with alarming increases in incidence and mortality rates. Emerging evidence suggests that tRNA modification enzymes play a crucial role in cancer development by modulating codon-specific translation. In this review, we focus on 18 tRNA modification enzymes and elucidate their mechanisms of action and roles in disease. We highlight the functions and mechanisms of seven tRNA regulators that mediate favorable tRNA translation in tumorigenesis and cancer progression, providing deeper insights into their clinical potential as cancer-related biomarkers and prognostic indicators. These findings emphasize the need for further investigation into the therapeutic potential of tRNA modification enzymes in cancer management and their potential application in personalized cancer therapy and diagnostics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39e019c44f4ec4dbe9eac3e87c51a69ce21d81f4" target='_blank'>
              Emerging roles of tRNA modification-mediated codon-specific translational reprogramming in cancer biology
              </a>
            </td>
          <td>
            Hanwei Wang, Junsi Zhang, Cen Jiang, Sunwang Xu
          </td>
          <td>2026-01-07</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b3ae1ba137c707051b533b78e7831f38e7fb1ac" target='_blank'>
              Metabolic reprogramming-driven resistance to multi-kinase inhibitors in hepatocellular carcinoma: molecular mechanisms and therapeutic opportunities.
              </a>
            </td>
          <td>
            Junxin Li, Yu Huang, Jiawei Li, Ming Shi, Yi Xiao, Fei Du, Gongli Hu
          </td>
          <td>2026-01-27</td>
          <td>Molecular cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Whole‐genome sequencing (WGS) has greatly expanded researchers' ability to study structural variants (SVs), that is, the variation in the presence, number, orientation or position of a DNA sequence. This has paved the way to study the eco‐evolutionary dynamics of SVs across the tree of life and within a population genomics framework. In this review, we provide the necessary fundamentals to help researchers generate and analyse population‐level SV data. We discuss the unique properties of different SV groups and how these fundamental differences interact with important biological and evolutionary processes using both empirical results and theory. This includes discussion of unresolved issues around SVs, such as technical difficulties in identification, accounting for diversity and evaluating functional effects. We explicitly integrate into this discussion transposable elements, which are an important component of SVs often identified in population‐level variant data. Finally, we focus on the practical side of SV analysis, offering a framework for SV identification and data analysis. In particular, we examine the heterogeneous nature of SV properties (type, length, sequence identity) that should be considered when studying them in ecology and evolution. This review aims to provide resources and guidelines to help researchers navigate the complexities of a relatively new field of eco‐evolutionary genomics research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4e7fdcc4f309779079c0cb9dffdc986139d5472" target='_blank'>
              A Beginner's Guide to Structural Variants in Eco‐Evolutionary Population Genomics
              </a>
            </td>
          <td>
            Katarina C. Stuart, Rebekah A. Oomen, Anna Tigano, Maren Wellenreuther, Jana Wold, David L Field, C. Mérot
          </td>
          <td>2026-01-01</td>
          <td>Molecular Ecology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="In pediatric acute myeloid leukemia (pAML) relapse/refractory (R/R) disease occurs frequently, but underlying mechanisms are unclear and effective second-line therapeutic options remain limited. While genomic characterization has advanced targeted treatments, their clinical implementation, particularly in the pediatric field, is hindered by toxicities or resistance due to poor correlation between preclinical and clinical studies. The use of more robust preclinical models is crucial for developing effective therapies. We established 26 pAML patient derived xenografts (PDXs) representing 14 high-risk genetic subtypes. These PDXs faithfully recapitulated the molecular complexity and heterogeneity of primary AML, and preserved the immunophenotypic, genomic, and transcriptomic profiles. Clonal and transcriptomic dynamics were tracked from patient samples to their matched PDXs, and vulnerabilities that could be exploited as therapeutic targets were identified, facilitating the evaluation of multi-faceted therapeutic strategies. We selected druggable variants and aberrantly activated pathways in KMT2A-rearranged AML and performed an in vitro drug screening of actionable targets exploiting a predictive three-dimensional co-culture model. Promising compounds were tested in vivo in AML-PDXs. Among new drugs targeting variants and pathways, we demonstrated the combination of IACS-010759, a mitochondrial complex I inhibitor, and Venetoclax, a BCL-2 inhibitor, reducing AML progression in KMT2A-r PDXs modeling both disease onset and relapse. The combination of Venetoclax+IACS-010759 with a stromal targeting drug slowed AML progression in a resistant model. Overall, our study highlights the power of AML-PDXs as a translational platform for novel targeted therapy identification. Our preclinical results testing Venetoclax+IACS-010759 in KMT2A-r AML strongly support mitochondrial targeting in this genetic AML subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41d42cd805ae2b01f54ab75804733bb0fe20182d" target='_blank'>
              Advancing Precision Therapy in Pediatric Acute Myeloid Leukemia Through PDX Models and Mitochondrial Targeting.
              </a>
            </td>
          <td>
            A. Da Ros, Alberto Peloso, Giorgia Longo, M. Benetton, V. Indio, S. Cairo, Monica Sandri, B. Buldini, Silvia Bresolin, Antonio Rosato, Andrea Pession, C. Tregnago, F. Locatelli, M. Pigazzi
          </td>
          <td>2026-01-23</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Blood cancers such as leukemia, lymphoma, and myeloma remain refractory in many patients due to immune escape, antigen heterogeneity, and therapy‑related toxicities. To address these challenges, we review recent strategies that harness CRISPR‑engineered gut commensals as precision “living therapeutics” to modulate host immunity and directly target malignant clones. We frame this review around three principal themes: (1) mechanistic strategies whereby CRISPR-engineered commensals modulate host immunity and directly antagonize malignant clones; (2) the enabling technologies and delivery/containment platforms, CRISPR variants, phage/LNP delivery, genetic circuits and biocontainment, that make living therapeutics feasible; and (3) translational progress, outstanding technical and safety barriers, and ethical/regulatory challenges that must be addressed for clinical deployment. To illustrate these themes, we discuss three concrete therapeutic modalities: engineered microbial secretion of immunomodulators, targeted delivery of tumor-lytic payloads, and engineered production of anticancer metabolites, and how these are enabled by contemporary CRISPR and synthetic-biology toolkits. Selected preclinical models report substantial antitumor effects, often >60% tumor reduction in rodent studies, and restoration of CAR-T cell function in controlled settings; however, effect sizes vary across models, and human translation remains unproven. We also analyze key technical barriers, strain stability, biocontainment, off‑target effects, and propose solutions, including auxotrophic kill-switches and AI‑guided strain optimization. Finally, we outline future directions, from in situ phage delivery to multi‑omics–driven patient stratification. CRISPR‑microbiome editing represents a paradigm shift in hematologic oncology, offering localized, sustained therapy with reduced systemic toxicity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e49361664a6596c9e706abe9f5db117dba2b21fd" target='_blank'>
              CRISPR-engineered microbiome: living therapeutics revolutionize blood cancer immunotherapy
              </a>
            </td>
          <td>
            Fang Cheng, Hamed Soleimani Samarkhazan, Yeganeh Khazaei
          </td>
          <td>2025-12-13</td>
          <td>NPJ Biofilms and Microbiomes</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Ovarian cancer (OC) remains a highly lethal gynecologic malignancy characterized by substantial molecular heterogeneity and diagnostic challenges. Although many reviews examine specific aspects of OC biology or treatment, few attempt to connect recent findings into a broader and coherent picture. This review provides an integrated and forward-looking synthesis of emerging mechanisms and therapeutic trends in OC, understanding OC progression and therapeutic resistance. Unique contributions of this review include providing a unified interpretation of emerging multi-omics evidence that reshapes the biological understanding of individual OC histotypes, delineating key metabolic dependencies such as glutamine addiction, aberrant lipid remodeling, glycolytic plasticity and purine pathway rewiring as potential therapeutic entry points, and elucidating how ascites-driven immune dysfunction remodels antitumor immunity to ultimately influence the effectiveness of cancer vaccines, immune checkpoint blockade and adoptive cell transfer therapies. Additionally, the review articulates underexplored therapeutic intersections, including nanotechnology-enhanced immunotherapy, gene-editing nanocarriers, and microbiome-mediated modulation of drug sensitivity and immune activation. By integrating these diverse yet interconnected domains, this review proposes updated conceptual models and cross-therapeutic strategies aimed at overcoming chemoresistance and advancing precision, personalized treatment paradigms in OC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d3677c50969edbedb78929193a3490430e71897" target='_blank'>
              Advances in ovarian cancer: biological insights, therapeutic innovations, and future perspectives
              </a>
            </td>
          <td>
            Xiao Ma, Linzhuo Qu, Qunying Wu, Qingqing Yu, Zhenhua Lin, Yang Yang, Xianchun Zhou
          </td>
          <td>2025-12-07</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b59601a1a811d20ae0b8430613e168e0ec52452b" target='_blank'>
              Multimodal antigenic escape to GPRC5D-targeted T cell engagers in multiple myeloma.
              </a>
            </td>
          <td>
            Holly Lee, Sungwoo Ahn, Gerone A. Gonzales, Noémie Leblay, E. Barakat, Dylan Greening, Pina Colarusso, Johnathan Canton, S. Benaoudia, Elham Hasheminasabgorji, Mansour Poorebrahim, David Jung, M. Truger, Jihong Jeong, C. Riedhammer, H. Einsele, K. M. Kortüm, Jan Eckmann, Jitka Somandin, Sara-Sheena Engel, L. Boise, J. Frenking, Niels Weinhold, Konstantina Taouxi, E. Kastritis, Sheri Skerget, D. Vishwamitra, Yunje Cho, F. Maura, Marc-Steffen Raab, J. Corre, L. Rasche, Hervé Avet-Loiseau, Paola Neri, Nizar J. Bahlis
          </td>
          <td>2026-01-15</td>
          <td>Nature medicine</td>
          <td>1</td>
          <td>76</td>
        </tr>

        <tr id="Background

Immune checkpoint inhibitors have transformed the treatment landscape of non-small cell lung cancer (NSCLC); however, primary and acquired resistance remain major clinical challenges. Immune evasion in NSCLC is increasingly recognized as a complex, dynamic process driven by tumor heterogeneity and adaptive remodeling of the tumor microenvironment. Conventional bulk profiling approaches are limited in resolving this complexity, underscoring the need for higher-resolution methodologies.

Objective
This review aims to synthesize current evidence on how single-cell transcriptomic approaches have advanced the understanding of immune evasion mechanisms in treatment-resistant NSCLC, with a focus on tumor-intrinsic programs, immune cell dysfunction, and intercellular communication networks.

Methods

A structured narrative review with systematic elements was conducted, analyzing peer-reviewed studies employing single-cell RNA sequencing and related single-cell technologies in human NSCLC. Studies were evaluated for their insights into immune escape pathways, resistance to immunotherapy, and translational relevance.

Results

Single-cell transcriptomic analyses reveal profound intratumoral and immune heterogeneity in treatment-resistant NSCLC. Resistant tumors harbor distinct malignant cell states characterized by impaired antigen presentation, dysregulated interferon signaling, and lineage plasticity. Within the immune compartment, hierarchical T-cell exhaustion, depletion of progenitor-like exhausted T cells, and expansion of immunosuppressive myeloid populations—particularly tumor-associated macrophages—emerge as dominant features of resistance. Ligand–receptor interaction analyses further demonstrate coordinated immunosuppressive communication networks that sustain immune escape at a systems level. Several single-cell–defined cellular states correlate with poor response to immune checkpoint blockade and adverse clinical outcomes.

Conclusion

Single-cell transcriptomics has fundamentally reshaped the conceptual framework of immune resistance in NSCLC, redefining it as a multicellular, dynamic ecosystem rather than a single-pathway phenomenon. These insights provide a strong biological rationale for developing biomarker strategies and rational combination therapies that target both tumor-intrinsic adaptations and the immunosuppressive microenvironment. Continued integration of single-cell technologies into translational and clinical research is essential to overcome immunotherapy resistance and improve patient outcomes in NSCLC.

Keywords: Non-small cell lung cancer; Single-cell RNA sequencing; Immune evasion; Immunotherapy resistance; Tumor microenvironment">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7495ff2bd82ff3331af5929ec1f5ca0f1ed50c9" target='_blank'>
              Single-Cell Transcriptomic Mapping of Immune Evasion Pathways in Treatment-Resistant Non-Small Cell Lung Cancer
              </a>
            </td>
          <td>
            Marco Rossi, Giulia Bianchi, Alessandro Conti
          </td>
          <td>2026-01-19</td>
          <td>Advanced Journal of Biomedicine &amp; Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Epigenetic reprogramming is a major contributor to therapy resistance in prostate cancer (PCa). Indeed, chromatin accessibility profiling classified PCa into distinct subtypes, predictive of clinical outcome. Histone modifications influence nucleosome positioning, ultimately dictating chromatin compactness. The histone methyltransferase G9a, frequently upregulated in advanced PCa, catalyzes a repressive histone modification (di-methylation of histone H3 lysine 9) that signals for gene transcriptional repression. CM-272 is a reversible and highly selective inhibitor against G9a activity, and we have previously shown in cell line models of PCa that it depicted strong anti-tumoral effects. However, fundamental aspects of altered transcriptomic programs remain unexplored. Here, we validated, in vivo, the CM-272 effects in advanced PCa, and explored the molecular mechanisms underscoring G9a inhibition-mediated effects. Specifically, the PCa patient-derived xenograft model BM18 was used to assess the mode of action of CM-272 in vivo. Moreover, RNA sequencing of tumor tissues was performed to further explore the underlying gene expression signature. Finally, chromatin immunoprecipitation followed by qPCR was used to investigate the underlying functional consequence of epigenetic targeting. Our results showed that CM-272 significantly delayed BM18 tumor growth, specifically impairing tumor cell G9a activity and proliferation. Transcriptomic profiling revealed 119 significantly upregulated and 99 downregulated genes in response to G9a inhibition. Notably, by integrating our gene expression dataset with published chromatin accessibility data, an increase in the androgen receptor (AR) signature and a concomitant decrease in the neuroendocrine (NE) features were observed in CM-272-treated samples. Additionally, pathway enrichment and master regulator analysis pinpointed both the androgen receptor and TP53 networks as significantly upregulated in treated tissues. In line with the reduction of the NE score, the MYC/MYCN were the most significantly downregulated transcription factors. Moreover, under the androgen pathway, upregulation of AZGP1, STK39, and DPP4 was observed. At the promoter level, a significant reduction in H3K9me2 levels was detected in treated samples, allowing, in turn, an increase in AR binding. Additionally, upon treatment, higher p53 binding was detected at the promoter of the p53-repressed genes AURKA, CDC25B, and CD44, whose expression was downregulated. Overall, targeting G9a with CM-272 might hold promising therapeutic potential for the management of advanced disease by modulating relevant tumoral transcriptional programs associated with core vulnerabilities in PCa.



 Filipa Moreira-Silva, Mark Zimmerli, Panagiotis Chouvardas, Beat Roth, Carmen Jerónimo, Marianna Kruithof-de Julio. G9a blockade uncovers therapeutic vulnerabilities in advanced prostate cancer, leading to tumor growth delay [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A045.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5911786a7c980e0c86fa5660d05b013ec96ea309" target='_blank'>
              Abstract A045: G9a blockade uncovers therapeutic vulnerabilities in advanced prostate cancer, leading to tumor growth delay
              </a>
            </td>
          <td>
            Filipa Moreira-Silva, Mark Zimmerli, P. Chouvardas, Beat Roth, Carmen Jerónimo, Marianna Kruithof-de Julio
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0feedb6dd227338600bf4a18ec50e34de570d13c" target='_blank'>
              Modulation of Network Plasticity Opens Novel Therapeutic Possibilities in Cancer, Diabetes, and Neurodegeneration.
              </a>
            </td>
          <td>
            Márk Kerestély, István Narozsny, Levente Szarka, D. Veres, Peter Csermely, D. Keresztes
          </td>
          <td>2026-01-31</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e7ea627495d26a3f7b3d82b351b84bb5f1b7889" target='_blank'>
              Genomics of Acute Myeloid Leukemia at Diagnosis and Remission
              </a>
            </td>
          <td>
            Kai Yu, Laura W. Dillon, Jesse M Tettero, Gege Gui, Rasha W Al-Ali, Michael R Grunwald, Elizabeth F Krakow, Elizabeth A. Griffiths, Alexandra Gomez-Arteaga, R. Vedula, M. Solh, A. Salhotra, N. Bejanyan, Lori S. Muffly, A. Jimenez-Jimenez, M. Drazer, Yi-Bin Chen, Aaron C. Logan, Reena V. Jayani-Kosarzycki, Sophia R Balderman, J. Blachly, Brian C Shaffer, L. Druhan, C. Yeung, Vanessa Kennedy, Amir T Fathi, H. Carraway, Sandeep Gurbuxani, M. Tjota, Farah Sahoo, Max Smith, Dylan Barfield, Jim Guo, James Han, Jason Hu, Heejon Jo, Vidya Kudlingar, Wilfred Li, Y. Qiu, P. Sathyan, Sean Truong, S. Catreux, Sam Ng, Khai Luong, Yunjiao Zhu, Reem Bahr, Jamie Diemer, C. Ferrone, Allison J Getker, Jackie Bjerke, Stephanie Bo-Subait, Steven M. Devine, Bergetta Dietel, Gabrielle Giammarino, Emily Heying Chihak, Jianqun Kou, Emily Kolb, Danielle O'Donnell Vitali, Stephen R. Spellman, Brenna K Tesch, Jenny Vogel, Stephanie Waldvogel, Syreeta Weatherspoon, Jeffery J. Auletta, Christopher S. Hourigan
          </td>
          <td>2025-12-07</td>
          <td>medRxiv : the preprint server for health sciences</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="


 The CBFA2T3–GLIS2 (C/G) fusion defines an exceptionally aggressive, epigenetically driven subtype of pediatric AML (pAML) confined to infants (<3 years). Arising from a pericentric inversion of chromosome 16, it is the most frequent lesion in acute megakaryoblastic leukemia (AMKL; ∼30%) and also occurs in non-AMKL subtypes, indicating lineage-independent pathogenicity. Clinically, C/G+ leukemia shows relapse rates > 90% and < 15% five-year survival, making it among the most treatment-refractory pediatric cancers. Lacking cooperating mutations, C/G acts as a singular oncogenic driver that reprograms the enhancer landscape, yet how it sustains epigenetic programs enforcing leukemic identity and chemoresistance remains unknown. We integrated multi-omic and functional epigenetic profiling across patient samples, C/G+ AML lines, and a developmentally faithful cord-blood CD34+ HSPC co-culture model. DNA methylation (DNAm) was mapped by meEM-seq, chromatin states by CUT&RUN and ATAC-seq, and transcription by RNA-seq. Functional studies included DNMT3B knockout, gRNA-resistant rescue, and inducible dCas9–TET1 demethylation, complemented by promoter-capture Hi-C and C/G+ xenograft validation. C/G+ pAML exhibited a promoter-biased hypermethylation signature (Δβ ≥ 0.2 vs NBM), with ∼50% of altered CpGs at promoters. The fusion bound the DNMT3B promoter, inducing its transcription without global 5mC gain but driving focal hypermethylation at active promoters linked via long-range enhancer loops. These hypermethylated promoters paradoxically stabilized expression of CRE-residing genes (BMP2, MED12, ITGB2) enriched for EMT, KRAS, heme-metabolism, and apoptotic pathways. In the fetal C/G–UCB-HSPC model, these signatures were faithfully recapitulated, confirming that the fusion alone reprograms the fetal hematopoietic epigenome. Motif analysis identified E2F2 and E2F4 as transcription factors stabilizing methylated yet transcriptionally active CREs. DNMT3B loss reduced locus-specific 5mC and target-gene expression but upregulated DNMT1 and UHRF1, suggesting a compensatory maintenance shunt that preserves DNAm at apoptotic loci and raises the apoptotic threshold. Targeted demethylation of PMAIP1 (NOXA) restored its transcription and apoptosis, confirming that DNAm stabilizes but constrains inducibility of pro-apoptotic genes. In vivo, DNMT3B-deficient xenografts regained Venetoclax sensitivity and extended survival, phenocopying DNMT inhibition (DNMTi). This study identifies C/G as an epigenetic architect converting DNAm from a repressive to a stabilizing signal, locking leukemic enhancers into an active yet apoptosis-resistant state. By defining the C/G–DNMT3B–DNMT1/UHRF1 axis and transcription-factor-mediated maintenance of methylated CREs, we reveal a unified mechanism linking developmental origin, transcriptional persistence, and therapy resistance in pAML. These findings position aberrant DNAm as a druggable epigenetic vulnerability and provide the mechanistic rationale for DNMTi + BCL-2 blockade in high-risk, treatment-refractory pAML.



 Samrat Roy Choudhury, Arundhati Chavan, Rhonda E. Ries, Giselle Almeida Gonzalez, Soheil Meshinchi, Jason Farrar. CBFA2T3–GLIS2 fusion reprograms enhancer-linked DNA methylation to enforce apoptotic resistance and defines an epigenetic vulnerability in pediatric AML [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr PR006.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e01e458a3959338b14b992ffa07b3d02aa22cbfe" target='_blank'>
              Abstract PR006: CBFA2T3–GLIS2 fusion reprograms enhancer-linked DNA methylation to enforce apoptotic resistance and defines an epigenetic vulnerability in pediatric AML
              </a>
            </td>
          <td>
            S. R. Choudhury, Arundhati Chavan, R. Ries, Giselle Almeida Gonzalez, S. Meshinchi, Jason Farrar
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f4eb8f9fdb1cae4aedbf1364db151a77768c687" target='_blank'>
              Targeting NOTCH3 to eradicate dormant and therapy-resistant multiple myeloma cells.
              </a>
            </td>
          <td>
            H. Sabol, Bethany C. Paxton, A. Anloague, Japneet Kaur, Mattie R. Nester, Sharmin Khan, James Smith, Peter I Croucher, Michelle M McDonald, Corey O Montgomery, J.B. Stambough, C. L. Barnes, E. Ambrogini, F. H. Ebetino, C. Schinke, Cody Ashby, J. Delgado-Calle
          </td>
          <td>2026-01-05</td>
          <td>Journal of experimental & clinical cancer research : CR</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Pancreatic cancer is a highly aggressive malignancy characterised by its invasive nature and poor therapeutic outcomes. These characteristics are closely associated with its complex biological characteristics and significant heterogeneity. Post-translational modifications (PTMs) have been identified as critical regulatory mechanisms through which cells respond to environmental changes and play a pivotal role in signal transduction. The various types of PTMs and their intricate regulatory mechanisms have a profound influence on multiple stages of pancreatic cancer progression. Research has demonstrated that PTMs modulate protein stability, activity, subcellular localization, and protein–protein interactions. The present review focuses on recent advances in our understanding of PTMs in pancreatic cancer, with a particular emphasis on phosphorylation, ubiquitination, SUMOylation, acetylation, lactylation, and O-GlcNAcylation. This study illuminates the molecular mechanisms and functional regulatory networks of PTMs within the distinctive tumour microenvironment of pancreatic cancer. Moreover, we summarise targeted therapeutic strategies directed at PTMs in pancreatic cancer to provide insights for future research and treatment development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6745c40730496d44cbedf3105b4de63f7bcb0" target='_blank'>
              Post-Translational Modifications: Key “Regulators” of Pancreatic Cancer Malignant Phenotype—Advances in Mechanisms and Targeted Therapies
              </a>
            </td>
          <td>
            Yongkang Shi, Renyi Qin, Yiming Li
          </td>
          <td>2025-12-01</td>
          <td>Biomedicines</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="Genetic interactions can reveal gene function and identify cancer-relevant synthetic lethals, but systematic mapping in human cells is constrained by inefficient reagents, vast combinatorial search space, and diversity of cell types. Here, we leverage principles from yeast genetic networks to identify human gene modules enriched for genetic interactions. Using our Cas12a-based In4mer combinatorial knockout platform, we screen all pairwise interactions within receptor tyrosine kinase and DNA damage response modules across eight diverse cancer cell lines. We identify hundreds of unreported synthetic lethals, including a dense network within the protein glycosylation machinery, and confirm that interactions in 2D cell culture are maintained in more physiologically relevant models. Our targeted modules show up to 16-fold enrichment of interaction density, providing a scalable strategy for systematic interaction mapping.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/081a137f8aa4f89541a558bf1cd83f7f679eee7b" target='_blank'>
              Functional modules predict cancer-relevant genetic interactions in mammalian cells
              </a>
            </td>
          <td>
            Chenchu Lin, Veronica Gheorghe, Juihsuan Chou, Sabriyeh Alibai, Subin Kim, N. Anvar, Yixin Xu, Xingdi Ma, Lori L. Wilson, Russell Moser, Christopher J. Kemp, Junjie Chen, Scott Kopetz, Traver Hart
          </td>
          <td>2025-12-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Malignant transformation of mature cystic teratoma (MTMCT) of the ovary is a rare but aggressive malignancy for which no standardized chemotherapy or effective targeted therapies currently exist. To identify therapeutic vulnerabilities in MTMCT, we performed a genome-wide CRISPR-Cas9 knockout screen using the MTMCT-derived NOSCC1 cell line. Two parallel selective pressures were applied: in vivo tumorigenicity in immunodeficient mice and cisplatin exposure in vitro. From this screen, 67 negatively selected genes were identified, among which SOD1 and NDUFB4 emerged as top candidates based on high basal expression levels and clinical relevance. Integration with spatial transcriptomic data from three independent MTMCT patient tumors further supported the prioritization of these targets. SOD1 was selected for further investigation due to the availability of known pharmacological inhibitors. Both siRNA-mediated knockdown and small-molecule inhibition of SOD1 using LCS-1 significantly suppressed MTMCT cell proliferation in vitro by inducing oxidative stress and impairing cell cycle progression. This antiproliferative effect was reversed by co-treatment with N-acetylcysteine, a reactive oxygen species scavenger. In vivo validation using patient-derived xenograft models demonstrated that oral administration of LCS-1 led to significant tumor growth suppression and increased expression of apoptotic and DNA damage markers, including cleaved caspase-3 and γH2AX. These findings establish SOD1 as a critical vulnerability in MTMCT and provide preclinical evidence supporting redox modulation as a therapeutic strategy for this highly chemoresistant and understudied ovarian cancer subtype. Our integrative approach combining functional genomics, spatial transcriptomics, and pharmacologic validation offers a framework for the discovery of novel targets in rare gynecologic malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e67036000a59a86b2e3dd28f5c317d050f6d45c" target='_blank'>
              In-Tumor CRISPR-Cas9 Knockout Screening and Novel Therapy Development for Malignant Transformation of Ovarian Teratoma.
              </a>
            </td>
          <td>
            S. Tamauchi, Kosuke Yoshida, Xinyuan Wang, Atsushi Nakagawa, Akira Yokoi, N. Yoshikawa, K. Niimi, Yusuke Yamamoto, Hiroaki Kajiyama
          </td>
          <td>2026-01-13</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="CRISPR-based gene editing has become a transformative tool to enhance immune cell therapies. In particular, engineering natural killer (NK) cells with CRISPR/Cas systems has gained traction due to their ability to mediate strong anti-tumor responses in an MHC-unrestricted, non-alloreactive manner. Early trials show the feasibility and safety of allogeneic NK cells, paving the way as scalable “off-the-shelf” products. CRISPR/Cas9 edits genomes by inducing DNA double-strand breaks (DSBs), mainly repaired through non-homologous end joining (NHEJ) or homology-directed repair (HDR). While effective, CRISPR carries risks of off-target (OT) activity that may disrupt essential genes, cause chromosomal rearrangements, or trigger oncogenic changes - posing threats to product integrity and patient safety. These concerns intensify with multiplex editing, where multiple loci are modified to improve function, persistence, and immune evasion. Since unmodified NK cells are typically short-lived, many clinical-stage products are engineered to express IL-15 or related constructs, extending their half-life and amplifying risks associated with unintended changes. This underscores the urgent need for robust safety assessments. In this review, we summarize the current landscape of safety assessment platforms for evaluating gene edited NK cells. We highlight predictive in silico tools, biochemical in vitro assays, and emerging cell-based detection systems to identify and quantify CRISPR-induced OT events. Particular attention is given to their suitability, limitations, and practical use in primary NK cells and multiplex editing strategies. Our aim is to support the design of safe, effective editing workflows for NK cell therapies - ensuring rigor as the field advances rapidly toward clinical application.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33b3e4984629f09ed0d85f8cdda8885ffc85a748" target='_blank'>
              Engineering with care: safety assessment platforms for CRISPR-modified natural killer cells
              </a>
            </td>
          <td>
            Alieh Fazeli, Evelyn Ullrich, Toni Cathomen, Tobias Bexte
          </td>
          <td>2025-12-05</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="In the past two decades, research on driver mutations has revolutionized lung cancer treatment with the emergence of targeted therapies as a new therapeutic strategy that significantly improves the prognosis of lung cancer. Targeted therapies are designed to recognize and disrupt specific proteins or pathways involved in the growth, spread and survival of cancer cells with high effectiveness. The use of targeted therapies has been shown to provide better progression-free survival and overall survival compared to traditional chemotherapy in NSCLC patients with targeted mutations. However, most patients eventually develop resistance regardless of the type and line of targeted therapy used. Resistance can occur in patients who initially respond targeted therapy. This is due to adaptive changes in tumor cells and the tumor microenvironment during drug exposure, through genetic and epigenetic processes forming secondary resistance. Understanding targeted therapies and their resistance mechanisms is essential to manage effective treatment for patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/227b94b42562030c2cb0c1fb5d9164cfac1c3274" target='_blank'>
              Mechanisms of target therapy resistance in non-small cell lung cancer
              </a>
            </td>
          <td>
            Nadia Amalia Haq, Andika Chandra, Fariz Putra, Jamal Nurwidya, Elisna Zaini, Syahruddin
          </td>
          <td>2025-12-27</td>
          <td>The Indonesian Journal of Cancer Control</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Simple Summary MELK is a serine/threonine kinase that regulates cell cycle progression, proliferation, apoptosis, migration, and stemness. MELK is overexpressed in many solid and hematologic malignancies. Furthermore, high MELK levels correlate with aggressive disease, poor survival, treatment resistance, and cancer stem-like features. Several small-molecule inhibitors and RNAi-targeting MELK have suppressed the development and progression of various cancers. OTSSP167 has shown promising results in phase I and II clinical trials. However, a study employing CRISPR/Cas9 knockout in breast cancer cell lines reported that loss of MELK did not impair proliferation. The review concludes that MELK is a robust prognostic biomarker of proliferation and aggressive behavior across multiple cancers but is not a universal driver of tumor growth. Discrepancies may arise from off-target effects of RNAi or inhibitors, context differences across cancer types, and differences in study settings. MELK is proposed as a conditional oncogenic vulnerability, especially in p53-deficient triple-negative breast cancer (TNBC) and glioma stem cells. Further research is needed to clarify the target selectivity of inhibitors, context-specific dependencies, and the optimal use of MELK-focused agents in combination therapies rather than as standalone treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0e62df49d62d42a4c4ea852f26c65300bd99df5" target='_blank'>
              Maternal Embryonic Leucine Zipper Kinase (MELK) in Cancer: Biological Functions, Therapeutic Potential, and Controversies
              </a>
            </td>
          <td>
            Alaeddin M. Alzeer, Saad Al-Lahham
          </td>
          <td>2026-01-01</td>
          <td>Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The survival mechanisms of circulating tumor cells (CTCs) remain poorly understood while these rare cell populations transit through the blood stream and colonize distant organs including the brain. Using single-cell RNA-seq of microfluidically-enriched CTCs and patient-matched brain metastatic tumor cells of lung adenocarcinoma (LUAD), we revealed CTC-selective upregulation of core gene signatures associated with ferroptosis. The malignancy of CTCs was confirmed through comparative genomic instability and copy number variation (CNV) analyses of paired CTCs and metastatic tumor cells using both transcriptomic and whole-exome sequencing. Among the transcription factors elevated in CTCs, Chromobox 3 (CBX3) was the top hit, which was tightly correlated with GPX4 expression. Functionally, CBX3 co-operated with EP300 to protect CTCs from ferroptosis by upregulating GPX4 expression. Genetic depletion of CBX3 triggered ferroptosis and substantially reduced tumor cell survival and invasiveness in two independent cancer cell models. Conversely, CBX3 overexpression was sufficient to promote tumor growth, migration, and invasion in vitro, and metastatic progression in vivo. Clinically, CBX3 expression was significantly correlated with TNM stage and was predictive of both progression-free interval and overall survival in LUAD. In a prospective cohort of LUAD and melanoma patients, CTCs co-expressing CBX3 and GPX4 were selectively elevated in the blood samples of metastatic cases compared to the non-metastatic group, highlighting the clinical association between ferroptosis-resistant CTCs and metastatic progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7fe0bbf1d19b9968391a14825a2e7d78f15cc38" target='_blank'>
              CBX3 confers ferroptosis resistance during blood-borne metastasis
              </a>
            </td>
          <td>
            Chun Wu, Xuefei Liu, Boxi Zhao, Mao Zhao, Binyu Zhang, Guanyin Huang, Yixin Cheng, Shuqian Zheng, Jianyang Hu, Ling Guo, Wei-Jie Guo, Jun Tan, Xin Hong
          </td>
          <td>2026-01-15</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f31378489089eb1239e625480e1522a21f193cf" target='_blank'>
              Chromosome 3p deletion leads to extensive genomic alterations in diverse cancers and confers synthetic lethality in uveal melanoma
              </a>
            </td>
          <td>
            Mitchell C. Cutler, P. B. Howland, M. Hejna, 
          </td>
          <td>2026-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Breast cancer (BC) remains a major contributor to cancer-related morbidity and mortality among women globally. While systemic therapies have significantly advanced, the emergence of drug resistance continues to hinder durable clinical benefit, leading to treatment failure and disease relapse. Long non-coding RNAs (lncRNAs) have gained recognition as key regulators in BC biology and therapeutic response, with growing evidence implicating them in resistance to chemotherapy, endocrine therapy, targeted agents, and immunotherapy. This review provides a detailed overview of the molecular mechanisms through which specific lncRNAs promote resistance, emphasizing their roles in modulating processes such as programmed cell death, epithelial-mesenchymal transition, and remodeling of the tumor microenvironment. We further highlight common regulatory axes - such as competing endogenous RNA networks and the phosphatidylinositol 3-kinase/protein kinase B pathway - that serve as converging points across distinct resistance mechanisms. In addition, we discuss recent progress in harnessing lncRNAs as liquid biopsy biomarkers and examine the key challenges and opportunities in translating lncRNA-based targets into therapeutic strategies. Together, these insights offer a foundation for future research aimed at overcoming drug resistance in BC through novel lncRNA-centered interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfac8b955e147276484ab116353ca6e5f90be4a2" target='_blank'>
              The role of long non-coding RNAs in breast cancer drug resistance: a comprehensive review
              </a>
            </td>
          <td>
            Yingkun Xu, Jingyu Ma, Changyuan Ding, Weili Liang, Q. Zeng, Bin Lv
          </td>
          <td>2025-12-10</td>
          <td>Journal of Translational Genetics and Genomics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="The mechanisms underlying cancer cell drug resistance have not yet been fully elucidated. Numerous processes hinder effective tumor treatment, such as cancer cell epigenetic alterations, DNA repair, drug target modification, and epithelial–mesenchymal transition. Recent studies have highlighted the ability of cancer cells to transiently enter the resting phase, namely, the G0 phase of the cell cycle under treatment conditions; this is associated with an elevated level of cell death. Cells in the G0 phase are characterized by a specific phenotype, the exact features of which remain to be fully elucidated. Cell division and cell cycle dynamics are closely associated with cell adhesion, i.e., interaction with the extracellular matrix in normal cells, whereas cancer cells are characterized by the dys-coordination of these processes. However, the precise patterns of cell adhesion alterations in association with the cell cycle remain unclear. Knowledge on the interactions of cells in the G0 phase with the extracellular matrix during the entry, the exit, and the sustenance during the resting phase is limited. At the same time, focal adhesion-mediated molecules may be an attractive goal for cancer cells in the G0 phase, targeting to overcome tumor dissemination and drug resistance. Drug resistance and immune system response evasion are specific features of nonproliferating cancer cells that call for more detailed and accurate characterization of this population of malignant cells to optimize anticancer treatment and control tumor dissemination. The present review represents the recent data on the association of focal adhesion and cancer cell surveillance due to cell cycle alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16149b6070228042071985c6c4e07c2cd03bc235" target='_blank'>
              Focal Adhesion and Cancer Cell Cycle-Associated Resistance
              </a>
            </td>
          <td>
            T. Ruksha, N. Palkina, Ekaterina Z. Lapkina, Kangdong Liu
          </td>
          <td>2025-12-25</td>
          <td>ACS Omega</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74e725a85a73539cf9bc922a83eb56481b485413" target='_blank'>
              Pan-cancer silencer transcription atlas reveals leukemia silencer hijacking FOXP1 to enhance MYC oncogene
              </a>
            </td>
          <td>
            M. Guo, Liutao Chen, Shengcheng Deng, Jingkai Zhang, Zhijie Hu, Weiwen Wang, Songyang Zhou, Yuanyan Xiong
          </td>
          <td>2026-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Abstract Somatic mutations, key alterations in cancer development, exert differential effects across tissues and biological layers, such as transcriptomes, proteomes, and post-translational modifications (PTMs). Although previous pan-cancer studies have characterized the molecular landscape of cancer, the effects of individual somatic mutations across different tissues remain insufficiently explored. Here, we developed Panorama to evaluate the oncogenic potential of single somatic mutations across all cancer types. We collected cancer proteogenomics or multiomics data from over 10 000 individuals across 19 cancer types. Based on five evaluation criteria, we assessed whether a specific mutation affects the abundance of a particular gene’s transcriptome, proteome, or phosphoproteome; the tumor microenvironment; specific RNA- or protein-based signaling pathways; and outlier-level overexpression of PTMs, aiding in potential drug target identification. By leveraging five oncogenic metrics, Panorama quantifies the oncogenic potential of individual somatic mutations and provides a framework for identifying driver mutations by incorporating their downstream effects. With Panorama, researchers can integrate cancer proteogenomics data, providing a comprehensive approach that enhances our understanding of single somatic mutations in specific tissues. Finally, Panorama was developed as a web-based database to ensure easy access for researchers and is freely available at http://139.150.65.64:8080/or https://github.com/prosium/panorama.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3988e7434c0ad5c1c94f629876b4c62c68b08126" target='_blank'>
              Panorama: a database for the oncogenic evaluation of somatic mutations in pan-cancer
              </a>
            </td>
          <td>
            Seung-Jin Park, Seon-Young Kim
          </td>
          <td>2026-01-15</td>
          <td>Database: The Journal of Biological Databases and Curation</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f8fae2bcd154582c2189d775e6e129baa231cb3" target='_blank'>
              A multi-omics analysis of human fibroblasts overexpressing an Alu transposon reveals widespread disruptions in aging-associated pathways.
              </a>
            </td>
          <td>
            Juan I. Bravo, Eyael Tewelde, Christina D. King, J. Bons, Samah Shah, Jacob Rose, Judith Campisi, Birgit Schilling, B. Benayoun
          </td>
          <td>2025-12-11</td>
          <td>GeroScience</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Despite improvements in the molecular profiling of pancreatic neuroendocrine tumors (PanNETs), predicting their clinical behavior and response to specific therapies remains challenging. We sought to elucidate the molecular basis underlying the broad phenotypic variations in these neoplasms through a genetic characterization of primary and metastatic PanNETs. Our findings revealed an enrichment of CDKN2A homozygous deletions and TSC2 somatic mutations in metastatic PanNETs when compared to non-metastatic lesions. Tumor evolution analysis further revealed the acquisition of such genetic alterations as late events in the progression of these neoplasms, conferring poor survival outcomes to the affected patients. Biallelic loss of DNA damage repair genes, ATRX and/or DAXX, was associated with a high fraction of the genome altered in PanNETs, with pathogenic alterations affecting those genes also being associated with a homologous recombination deficiency signature. These findings highlight molecular mechanisms driving PanNET progression and underscore the need for further molecular characterization and tumor evolution studies to evaluate targeted therapies for such a challenging disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db2469c6632fab23ebab1e0e71bc53bea256e7dd" target='_blank'>
              CDKN2A homozygous deletions and TSC2 somatic mutations in metastatic pancreatic neuroendocrine tumors
              </a>
            </td>
          <td>
            Tito Teles Jesus, Lorenzo Ferrando, Lia Rodrigues, R. Martins, Luís Cardoso, José Manuel Lopes, Paula Soares, Arnaud Da Cruz Paula, J. Vinagre
          </td>
          <td>2025-12-05</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2879e795e7efcebd514454af1fc1a736f45a29d8" target='_blank'>
              Somatic and germline mutational processes across the tree of life
              </a>
            </td>
          <td>
            Sangjin Lee, Yichen Wang, Haynes Heaton, Emily Mitchell, M. Maddison, Liam M. Crowley, Patrick Adkins, N. Mieszkowska, Mark Blaxter, R. Durbin, P. Campbell
          </td>
          <td>2025-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32c4664b6580edc429e59cb7fc3bb0cee1ddce94" target='_blank'>
              LISA: A Comprehensive R Package for Lentiviral Integration Site Analysis in Gene Therapy Safety Assessment
              </a>
            </td>
          <td>
            Shuai Ni, Ning Wu, Yun Kang, Fengjiao Zhu, Jueyu Wang, Lina Wang, Wei Wang, Heng Mei, Kejia Kan
          </td>
          <td>2025-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Chromosomal aberrations are frequent in cancer and play a pivotal role in disease initiation and progression. In prostate cancer (PCa), deletions at chromosome 13q14 are common and enriched in advanced disease, suggesting selective advantage in tumor progression. Beyond BRCA2, this region harbors other tumor suppressor genes, including RB1 and RNASEH2B, which impact sensitivity to androgen receptor inhibitors (ARi) and PARP inhibitors (PARPi). This study aims to elucidate the effect of individual vs concomitant loss of RB1 and RNASEH2B in response to dual AR and PARP inhibition.



 To explore the prevalence of deletions in 13q we interrogated copy number alterations in the Hartwig Medical Foundation (HMF) PCa WGS dataset (n=486). In vitro, we generated CRISPR-Cas9-mediated knock-outs (KO) of RB1, RNASEH2B, or both genes in PCa cell lines (LNCaP, C4-2, 22Rv1, DU145). Gene editing was confirmed by Sanger, and western blot analysis validated protein loss. Antitumor activity of enzalutamide (ARi), and olaparib or saruparib (PARPi) was determined using CCK8 kit and colony formation assays.



 RB1 deletions were present in 42% of HMF PCa samples (205/486), being mostly heterozygous deletions (200/205). RB1 deletions co-occurred with RNASEH2B loss in 94% of cases (192/205) and with BRCA2 deletions in 42% of cases (87/205). In vitro, RB1-KO increased resistance to ARi and PARPi in hormone-sensitive lines LNCaP and C4-2, while no change in sensitivity was detected in AR-independent models 22Rv1 and DU145. Loss of RNASEH2B increased sensitivity to PARPi in all models. Next, we generated RB1 and RNASEH2B double KO models in LNCaP and C4-2 by sequentially editing single KO populations. After single-cell seeding, we obtained: (1) RB1-KO (homozygous) clones with complete protein loss; (2) RNASEH2B-heterozygous (RNASEH2B-Het) models with partial protein expression; and (3) RB1-KO/RNASEH2B-Het models. No clones with complete RNASEH2B loss were obtained, suggesting that RNASEH2B-loss population is transient and that clones with complete loss do not proliferate. RB1-KO models maintained resistance to both treatments. RNASEH2B-Het cells showed increased sensitivity to PARPi, although to lower extent than the population with complete RNASEH2B loss. Importantly, introducing RB1-loss in RNASEH2B-Het cells restored PARPi resistance, and combined AR/PARP inhibition did not overcome resistance in RB1-KO or RB1-KO/RNASEH2B-Het cells.



 Our findings indicate that 13q14 deletions in PCa predominantly occur as heterozygous events encompassing key genes influencing ARi and PARPi responses. Heterozygous loss of RNASEH2B confers sensitivity to PARPi, although to a lesser extent than in models with complete RNASEH2B loss. RB1-loss confers higher resistance to ARi and PARPi, restoring resistance to PARPi in RNASEH2B heterozygous models. These results highlight the need for models that better mirror the heterozygous nature of 13q14 deletions to clarify their biological effects and refine therapeutic strategies for advanced PCa.



 Victor Esquefa, Pablo Cresta Morgado, Richard Norris, Lara de Llobet, Julian Brandariz, Sara Arce-Gallego, Teresa Casals, Manuel Ramos, Daniel Aguilar, Gisela Mir, Irene Casanova-Salas, Francisco M. Barriga, Joaquin Mateo. Impact of 13q.14 loss in prostate cancer progression and sensitivity to targeted agents [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr B018.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15430fc4c6c79184273233030845f22a6f4ecd5a" target='_blank'>
              Abstract B018: Impact of 13q.14 loss in prostate cancer progression and sensitivity to targeted agents
              </a>
            </td>
          <td>
            Victor Esquefa, P. C. Morgado, Richard Norris, L. D. De Llobet, Julian Brandariz, Sara Arce-Gallego, Teresa Casals, Manuel J. Ramos, Daniel Aguilar, Gisela Mir, I. Casanova-Salas, Francisco M. Barriga, Joaquin Mateo
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af7630e9ff15d614dc5c29742755271640ed25f5" target='_blank'>
              KMT2A-Mediated transcriptional regulation in stemness and cancer: molecular mechanisms and therapeutic opportunities
              </a>
            </td>
          <td>
            Md Shiblee, Sadik Sabuj, Tanvir Ahmed, Md Jamilur Rahman, S. M. A. Salam, Byung-Yong Park, Md Rashedunnabi Akanda
          </td>
          <td>2025-12-24</td>
          <td>Medical Oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="


 Somatic structural variations (SVs) are common in cancer. Although a small fraction of SVs in breast and ovarian cancers can be attributed to homologous recombination deficiency, the underlying molecular mechanisms for the vast majority of somatic SVs remain unclear. Here, we focus on the roles of transcription and DNA replication collisions in genomic instability in cancer. Such collisions are unavoidable in cells since both transcription and replication use the same DNA as template. We hypothesized that transcription replication collisions (TRCs), if not properly repaired, would lead to collapsed replication forks and result in SVs. To this end, we studied somatic SVs in 6193 high-coverage whole-genome sequenced primary and metastatic tumors from three independent pan-cancer cohorts. A total of 13 conserved SV signatures, representing independent molecular mechanisms, were deconvoluted from these cohorts using non-negative matrix factorization approach. We detected replicated-strand bias, the expected footprint of transcription-replication collision, in large tandem duplications (TDs) across multiple cohorts. This bias was only observed in expressed genes, consistent with TRCs depending on transcription activity. Large TDs were abundant in female-specific (breast, ovarian and uterus), upper gastric-intestinal tract, and prostate cancers. They were associated with CDK12 mutations and worse patient survival. CDK12 is a cyclin-dependent kinase, a key regulator of transcription elongation and termination. Deleting or suppressing CDK12 using CRISPR-Cas9 in prostate cell lines increased RNA:DNA hybrids (R-loops), promoted TRCs, and ultimately led to large TDs. Finally, we found that cells lacking CDK12 were sensitive to WEE1, CHK1 and ATR inhibitors. In summary, our data suggest that large TDs in cancer form due to impaired TRC repair and can be used as a biomarker for prognosis and treatment.



 Lixing Yang. A078: Transcription and DNA replication collisions lead to large tandem duplications and expose targetable therapeutic vulnerabilities in CDK12-mutant prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A078.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a771b9c6597f1781d70e2b6d5df9ff54196c4d85" target='_blank'>
              Abstract A078: Transcription and DNA replication collisions lead to large tandem duplications and expose targetable therapeutic vulnerabilities in CDK12-mutant prostate cancer
              </a>
            </td>
          <td>
            Lixing Yang
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4773775e6ecfd7cb45043c77dfe433bc00bd2ae5" target='_blank'>
              Epigenetic Intelligence: How Organisms Track Their Environment Through Molecular Memory
              </a>
            </td>
          <td>
            Holly V. Moeller, H. Putnam, Ross Cunning, Steven B. Roberts, J. Eirín-López, R. Nisbet
          </td>
          <td>2026-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Background: The profound heterogeneity of glioblastoma and the often-limited efficacy of conventional treatments, including arsenic trioxide (ATO), underscore the urgent and critical demand for innovative combination strategies specifically designed to overcome treatment resistance. Methods: We evaluated the therapeutic effects of ATO as a single agent and in combination with the MNK1 inhibitor AUM001 across patient-derived xenograft (PDX) models and investigated molecular determinants of sensitivity and synergy. Our results demonstrated that GBM models resistant to ATO, particularly those of the mesenchymal subtype, are more likely to show synergistic cytotoxicity when AUM001 is added. The combination significantly reduces the frequency of glioblastoma stem cells (GSCs) compared to either drug alone, especially in ATO-resistant models. Results: These observations suggest that targeting the MNK1 pathway in conjunction with ATO is a promising strategy to specifically eradicate GSCs, which are major drivers of GBM recurrence and therapeutic failure. Transcriptomic analyses revealed that ATO sensitivity correlated with activated translation-related pathways and cell cycle processes, while synergistic responses to the combination were driven by distinct molecular signatures in different GBM subtypes. Overall, synergistic response to the combination therapy is more associated with cellular organization, amino acid transmembrane transporter activity, ion channels, extracellular matrix organization and collagen formation. Conclusions: Our findings highlight that specific molecular pathways and their activities, including those involving translation, cell cycle and ion transport, appear to modulate the synergistic efficacy of the ATO and AUM001 combination, thereby offering potential biomarkers for improved patient stratification in future GBM clinical trials of such ATO-based treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed3a011ab153f12fa9045338da4a46a0d67f877a" target='_blank'>
              Arsenic Trioxide and the MNK1 Inhibitor AUM001 Exert Synergistic Anti-Glioblastoma Effects by Modulating Key Translational, Cell Cycle, and Transmembrane Transport Pathways
              </a>
            </td>
          <td>
            Yue Hao, Charles Shaffer, N. Tang, Valerie DeLuca, Angela Baker, Michael E Berens
          </td>
          <td>2026-01-23</td>
          <td>Brain Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background/Objectives: Neuroblastoma cells are phenotypically plastic, transitioning between mesenchymal and adrenergic states. Core functional genes (e.g., YAP1) mark the mesenchymal state, which is linked to unfavorable prognosis. We and others previously demonstrated relapse-specific Hippo-YAP pathway activation in matched primary/relapsed neuroblastomas. Here we explored the role of YAP1 in neuroblastoma aggressiveness and response to therapy. Methods: RT-qPCR and immunoblotting assessed YAP1 expression in neuroblastoma cell lines. RNA-sequencing detected YAP1-dependent gene signatures in Tet-ON SK-N-AS and SH-EP neuroblastoma cell models expressing wildtype YAP1 or constitutively activated YAP1S127A. Data from cell models were compared with our published YAP1 expression data from neuroblastomas. Efficacy of commonly used chemotherapeutics was comparatively analyzed in the cell models. Results: YAP1 expression showed marked variability across a panel of neuroblastoma cell lines, assessed by mRNA analysis in 10 cell lines and protein analysis in a subset of 9 cell lines. RNA sequencing in constitutively activated YAP1S127A mutant and wildtype YAP1 models detected 2162 and 1837 significantly differentially expressed genes in the SK-N-AS and SH-EP backgrounds, respectively. Continuously activating YAP1 in SK-N-AS cells upregulated mesenchymal signature genes and mesenchymal-associated transcription factors. Gene expression influenced by YAP1 activity in the cell models significantly overlapped with YAP1-associated genes (e.g., CYR61 and SPRY4) in published tumor data. Functionally, YAP1S127A expression rendered neuroblastoma cells resistant to chemotherapy. Conclusions: Findings corroborate the idea of a mechanistic role for YAP1 in neuroblastoma adrenergic to mesenchymal reprogramming and therapy resistance. The YAP1-mediated plastic switch towards a mesenchymal expression state in neuroblastoma cells may provide the molecular basis for novel therapeutic avenues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d8df35b4e9326d49eb3d2876e0fc3b02be6fd46" target='_blank'>
              YAP1 Enhances Mesenchymal-Type Gene Expression in Human Adrenergic-Type Neuroblastoma Cells
              </a>
            </td>
          <td>
            Marius Ludwig, Kerstin Ahrens, A. Winkler, Jasmin Wünschel, Peris Ruka, M. Lodrini, F. Hertwig, Sveva Castelli, Theresa M. Thole-Kliesch, Jan F. Hollander, Steffen Fuchs, A. Künkele, Marvin Jens, S. Mamlouk, Steven W. Warmann, K. Astrahantseff, Angelika Eggert, J. H. Schulte, A. Szymansky, H. Deubzer
          </td>
          <td>2026-01-26</td>
          <td>Cancers</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b89d2858599b4236f22769f3a4af65ec9fd4eb7" target='_blank'>
              Deciphering a common code: Unitary gene profile factor enabling Spontaneous Tumor Regression
              </a>
            </td>
          <td>
            Arushi Misra, B. Kumari, Prasun K. Roy
          </td>
          <td>2025-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b370747cb8a1720741d860f50d6f1043c88dbdfc" target='_blank'>
              Cancer stem cell-driven drug resistance in colorectal carcinoma: molecular aspects and therapeutic potentials.
              </a>
            </td>
          <td>
            Minfeng Zhou, Huifang Niu, Dandan Cui, Menghao Xu, Jinxiao Li, Guichen Huang, Minquan Zhou, Chutong Xiong, Yunya Liu, Xiaojuan Xu, Hongxing Zhang, Fengxia Liang, Rui Chen
          </td>
          <td>2026-01-22</td>
          <td>Molecular cancer</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Oncofetal reprogramming has recently emerged as a critical concept in translational cancer research, particularly for its role in driving therapeutic resistance across a variety of malignancies. This biological process refers to a pattern of gene expression that is restricted to embryogenesis, but becomes expressed again in a subpopulation of cancer cells. These genes are typically suppressed after embryogenesis, and their aberrant re-expression in tumors endows cancer cells with stem-like properties and enhanced adaptability. The goal of this review is the following: (i) comprehensively examine the multifaceted nature of oncofetal reprogramming; (ii) elucidate its underlying molecular mechanisms, including its regulators and effectors; and (iii) evaluate its consequences for the therapeutic response in different cancer types. We comprehensively integrate the latest findings from colorectal, breast, lung, liver, and other cancers to provide a detailed understanding of how oncofetal programs interfere with tumor response to treatment. Among the candidates, YAP1 and AP-1 have emerged as central transcriptional drivers of this reprogramming process, especially in colorectal and breast cancers. We also explore the distinct expression patterns of oncofetal genes across different tumor types and how these patterns correlate with treatment outcomes and patient survival. Lastly, we propose a dual-targeting therapeutic strategy that simultaneously targets both cancer stem cells and oncofetal-reprogrammed populations as a more effective approach to overcome resistance and limit recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a262b7681553fb103b5d82c8b95c6fdfada4bf5" target='_blank'>
              Oncofetal Reprogramming: A New Frontier in Cancer Therapy Resistance
              </a>
            </td>
          <td>
            Anh L Nguyen, Molly Lausten, Bruce M. Boman
          </td>
          <td>2026-01-29</td>
          <td>International Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/719b7b9d009213c7ffc5a98ac8b16a699f1f4455" target='_blank'>
              Anchored random reverse primer sequencing for quantitative detection of novel gene fusions.
              </a>
            </td>
          <td>
            Xuehao Xiu, Yi Wu, Jiangxue Li, Dongfa Lin, Xiao Sun, Xinglei Su, Zhi Weng, Xiaolei Zuo, Xiurong Yang, Chunhai Fan, Yudong Wang, David Yu Zhang, Ping Song
          </td>
          <td>2026-01-02</td>
          <td>Nature biomedical engineering</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Cancer is increasingly recognized not as a fixed genetic condition but as a dynamic and plastic disease state driven by reversible epigenetic, transcriptional and microenvironmental cues. This evolving understanding supports a therapeutic paradigm shift: from eradicating malignant cells to reprogramming them toward quiescence, differentiation or functional normalization. This review explores diverse strategies for redirecting cancer cell fate, including differentiation therapy, epigenetic remodeling, lineage reprogramming, senescence induction and tumor microenvironment resetting. These approaches exploit intrinsic cellular plasticity and contextual adaptability, offering novel avenues to contain malignancy and overcome resistance. By reframing cancer as a potentially reversible phenotype, this therapeutic strategy demands redefinition of clinical endpoints, incorporation of dynamic biomarkers, and development of integrative treatment frameworks. Ultimately, reprogramming‑based oncology holds the promise of transforming aggressive malignancies into manageable conditions while minimizing the collateral damage associated with conventional cytotoxic therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99bfdfe1c6957af978b8fb4415d04fd96bdf7b85" target='_blank'>
              Beyond eradication: Therapeutic reprogramming strategies for cancer normalization (Review).
              </a>
            </td>
          <td>
            Aliasgar F. Shahiwala
          </td>
          <td>2025-12-04</td>
          <td>Oncology reports</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="


 Clonal hematopoiesis (CH) is exemplified by the expansion of genomically altered hematopoietic stem and progenitor cells that may give rise to altered immune repertoires. Driven largely by age and chronic inflammation, CH has been historically linked to myeloid malignancies. Although emerging evidence points to a modest prevalence of CH within solid tumors, the impact of these naturally occurring alterations on anti-tumor immunity and therapeutic responses remains less defined. Here, we employed somatic mutation calling algorithms to detect single nucleotide variants (SNVs) in both epithelial and immune compartments within tumors. Our preliminary analyses of single-cell RNA-sequencing (scRNA-seq) of 32 primary, therapy-naïve prostate tumors encompassing 11 patients identified conventional mutations representative of both CH and prostate cancer. Within the tumor-epithelial compartment, we identified recurrent FOXA1 class 1C mutations in 27% (3/11), alongside NCOR1 mutations in 18% (2/11), IDH1 in 9% (1/11), and SPOP in 9% (1/11) of patients. Of note, these SNVs were detected across multiple samples for the same patient. Interestingly, we also detected germline HOXB13 G84E variants, associated with increased risk for prostate cancer, in 2 cases at a high clonality within epithelial cells (>75%). CH alterations were identified in 27% (3/11) of patients based on the detection of exonic, nonsynonymous SNVs within myeloid cells. Specifically, these variants affected SF3B2 (variant allele frequency; VAF = 0.30), UBE2A (VAF = 0.30), and KMT2C (VAF = 0.43) that are predicted deleterious across multiple classifier algorithms. Patients with detectable CH alterations displayed immunologically hotter tumor microenvironments, with CH+ patients having significantly higher proportions of tumor-infiltrating monocytes (16.22% vs 7.13%, Fisher’s exact test p<0.0001), macrophages (6.49% vs 2.88%, p<0.0001), CD8+ T cells (10.35% vs 3.33%, p<0.0001), CD4+ T cells (2.81% vs 0.74%, p<0.0001), and B cells (1.81% vs 0.59%, p<0.0001), and lower fractions of epithelial cells (50.94% vs 69.81%, p<0.0001). In summary, we leveraged de novo mutational algorithms to identify somatic alterations using scRNA-seq. To our knowledge, our study is the first to detect CH alterations in prostate cancer at the single-cell resolution. Further work will integrate our mutational signatures with transcriptomics to elucidate the impact of both cancer- and immune-intrinsic alterations on anti-tumor immunity. Additionally, we will incorporate mitochondrial DNA-based lineage tracing algorithms on matched peripheral blood, primary, and metastatic tumor samples to deconvolute the spatiotemporal regulation of naturally occurring CH alterations across cancer progression. We envision our findings will illuminate novel drivers of immune function and regulation within prostate cancer that may be exploited for future immunomodulatory therapies.



 John R. Lozada, Christine Luo, Devin A. Schmeck, Jeffrey S. Miller, Akash Patnaik, Emmanuel S. Antonarakis, Frank Cichocki, Justin H. Hwang. Interrogating the Genomic Co-Evolution of the Immune System and Prostate Cancer Along the Time and Treatment Continuum [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A040.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03587209005d8227c89ae44e47e8117ddd350a30" target='_blank'>
              Abstract A040: Interrogating the Genomic Co-Evolution of the Immune System and Prostate Cancer Along the Time and Treatment Continuum
              </a>
            </td>
          <td>
            John R Lozada, Christine Luo, Devin A. Schmeck, Jeffrey S. Miller, Akash Patnaik, Emmanuel S. Antonarakis, Frank Cichocki, Justin H. Hwang
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Background: Myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML) are clonal stem cell disorders in which disrupted post-transcriptional regulation contributes to aberrant hematopoiesis and leukemic transformation. The miRNA biogenesis machinery, which comprises Drosha, DGCR8, Dicer, TARBP2, and AGO1, ensures the precise maturation of miRNAs that control lineage commitment and proliferation. However, the extent to which alterations in this pathway reshape hematopoietic gene networks during myeloid disease evolution remains largely unexplored. Methods: Bone marrow samples from newly diagnosed, untreated MDS and AML patients and matched healthy controls were analyzed for the expression of five key miRNA biogenesis genes using quantitative real-time PCR. Statistical comparisons, correlation matrices, and ROC analyses were performed to characterize gene-expression differences. These results were integrated with multigene logistic modeling, decision-curve analysis, and exploratory random forest/SHAP approaches to evaluate molecular interactions and diagnostic relevance. Results: DROSHA, DICER1, and TARBP2 were significantly downregulated in both MDS and AML, suggesting impaired miRNA maturation and a loss of global post-transcriptional control. DGCR8 expression increased across higher-risk MDS groups, suggesting compensatory activation of the Microprocessor complex, whereas AGO1 levels remained relatively stable, consistent with partial maintenance of RISC function. Correlation analyses revealed a co-regulated DROSHA–TARBP2–AGO1 module. ROC, logistic, and machine learning models identified DGCR8 and DICER1 as the strongest diagnostic discriminators. The integrated five-gene signature achieved high discriminative performance (AUC ≈ 0.98) and showed promise but remains preliminary potential for clinical application. Conclusions: Our findings suggest that defects in miRNA biogenesis disrupt hematopoietic homeostasis, reflecting common mechanisms in MDS and AML. The dysregulation of DICER1, DGCR8, and TARBP2 offers insights into miRNA-driven leukemogenesis and may pave the way for miRNA-based diagnostic and therapeutic strategies, pending validation in larger cohorts. Although transcript-level data are provided, future studies should include functional validation to determine the impact on downstream miRNA processing and hematopoietic pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d47acba4f826476e2de9cf5f5fedcb821bc1d6f0" target='_blank'>
              Disrupted miRNA Biogenesis Machinery Reveals Common Molecular Pathways and Diagnostic Potential in MDS and AML
              </a>
            </td>
          <td>
            Kenan Çevik, M. Ay, Anıl Tombak, Özlem İzci Ay, Ümit Karakaş, Mehmet Emin Erdal
          </td>
          <td>2025-12-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Colorectal cancer (CRC) has a high incidence and mortality rate globally, with approximately 40% of patients harboring Kirsten rat sarcoma viral oncogene homolog (KRAS) mutations. Patients with KRAS mutations exhibit poorer prognoses compared to those with KRAS wild-type tumors, often showing resistance to targeted therapies and chemotherapy. There is an acute need for innovative therapeutic strategies, particularly as KRASG12C inhibitors have been approved by the U.S. Food and Drug Administration (FDA) for second-line therapy in treating non-small cell lung cancer (NSCLC). Yet, their efficacy in CRC remains suboptimal due to the emergence of drug resistance. Current research on KRAS inhibitor resistance in CRC focuses mainly on single biological processes, leaving the complex interplay between cellular systems underexplored. This review synthesizes evidence across four key dimensions — metabolism, epigenetics, immunology, and proteostasis — to reveal multidimensional mechanisms of resistance. How these interconnected pathways work in synergy with signaling aberrations, genetic alterations, and translational modifications to reinforce resistance was investigated. The research also integrates recent advances in multi-targeted strategies, including combinations of KRAS inhibitors with metabolic or epigenetic modulators or immune checkpoint blockade. These findings provide potential strategies for overcoming resistance in KRAS-mutant CRC, addressing a critical gap in precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de09a3f40e67f360dfd11b639fa71c10cdd13842" target='_blank'>
              Unravelling Resistance: Integrating Metabolism, Epigenetics, Immunology, and Proteostasis in Strategies against Kirsten Rat Sarcoma Viral Oncogene Homolog-mutant Colorectal Cancer
              </a>
            </td>
          <td>
            Jingyi Li, Na Song, Rui Ma, Xiujuan Qu
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16c6776876b0ba015273baad473a4e391867ab9a" target='_blank'>
              CRISPR screens identify PRMT7 as a therapeutic target to enhance T cell-mediated killing in breast cancer.
              </a>
            </td>
          <td>
            Wei Shi, Yi Luo, Yizhuo Wang, Jacqueline M. Burrows, Debra Black, Andrew Civitarese, Laura Perlaza-Jiménez, Ping Zhang, Murray Manning, Natasha Tuano, Miguel E Rentería, Christos Xiao, Siok-Keen Tey, Joseph Rosenbluh, Corey Smith, G. Chenevix-Trench, Jonathan Beesley
          </td>
          <td>2026-01-21</td>
          <td>NPJ breast cancer</td>
          <td>0</td>
          <td>110</td>
        </tr>

        <tr id="Hematologic malignancies remain among the most difficult cancers to treat, challenged by profound heterogeneity, treatment-induced immune dysfunction, and the frequent emergence of drug resistance. Beyond tumor-intrinsic mechanisms, dysbiosis of the gut microbiome is increasingly recognized as a critical determinant of therapeutic outcomes, shaping hematopoiesis, immune responses, and drug metabolism. Bacteriophage (phage) therapy has re-emerged as a precision tool capable of selectively eradicating pathogenic taxa while preserving commensal short-chain fatty acid–producing communities. Preclinical and early human studies demonstrate that phages can recalibrate microbial ecosystems, disrupt antibiotic-tolerant biofilms, and enrich metabolites such as butyrate that support mucosal integrity and immune balance. Mechanistically, phage DNA enriched with CpG motifs engages Toll-like receptor 9, activating dendritic cells and enhancing cytotoxic T lymphocyte responses, suggesting dual benefits in infection control and anti-tumor immunity. Emerging applications extend further, with engineered phages serving as vectors for CRISPR–Cas gene editing, targeted cytokine delivery, and nanocarrier platforms for leukemia therapy. Despite translational promise, major hurdles persist, including immunogenicity, horizontal gene transfer, resistance evolution, and regulatory uncertainty. Addressing these challenges through GMP-compliant manufacturing, metagenomics-guided personalization, and AI-optimized cocktail design could establish phage therapy as a microbiome-informed adjunct to overcome drug resistance in blood cancers. However, direct clinical evidence of phage therapy efficacy in hematologic malignancies remains limited, and current data are largely derived from preclinical and compassionate-use contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc9ece11723039e03ff5895ac5fce518e0f2ca3a" target='_blank'>
              Phage therapy and the microbiome in hematologic malignancies: opportunities, mechanisms, and early evidence
              </a>
            </td>
          <td>
            Juanwen Zhang, Jin Liu, Alireza Bayani
          </td>
          <td>2025-12-12</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Mutations in the global transcriptional activator EP300/KAT3B are being reported in aggressive malignancies. However, the mechanistic contribution of EP300 dysregulation to cancer is currently unknown. While EP300 has been implicated in regulating cell cycle and DNA replication, the role of EP300 in maintaining replication fork integrity has not been studied. Here, using EP300-mutated adult T-cell leukemia/lymphoma cells and an EP300-selective degrader, we reveal that EP300 loss leads to pronounced dysregulations in DNA replication dynamics and persistent genomic instability. Aberrant DNA replication in EP300-mutated cells is characterized by elevated replication origin firing due to replisome pausing. EP300 deficiency results in a prominent defect in fork protection resulting in the accumulation of single-stranded DNA gaps. Importantly, we find that the loss of EP300 results in decreased expression of BRCA2 protein leading to sensitivity to treatments that are cytotoxic to BRCA-deficient cancers. Overall, we demonstrate that EP300-mutated cells recapitulate features of BRCA-deficient cancers. Loss-of-function mutations in EP300/KAT3B and CBP/KAT3A have been implicated in the pathogenesis of cancer. Here, the authors reveal that the EP300 protein has a role in mediating replication fork protection at sites of replication stalling and show that EP300-mutated cells recapitulate features of BRCA-deficient cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c1df38e22e1ee22880d25154078c1bc6f87527f" target='_blank'>
              EP300 deficiency leads to chronic replication stress mediated by defective replication fork protection
              </a>
            </td>
          <td>
            Angelica Barreto-Galvez, Mrunmai Niljikar, J. Gagliardi, Carolina Plasencia Guzman, Ranran Zhang, Vasudha Kumar, Aastha Juwarwala, Archana Pradeep, Ankit Saxena, Cristina Montagna, Priya Mittal, Jeannine Gerhardt, Bing Xia, Jian Cao, Keisuke Kataoka, A. Durbin, Jun Qi, B. H. Ye, Advaitha Madireddy
          </td>
          <td>2025-12-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Lenvatinib serves as a first-line therapy for advanced hepatocellular carcinoma (HCC), but its clinical efficacy is severely limited by acquired drug resistance. Consequently, the identification of therapeutic targets to reverse lenvatinib resistance may offer novel strategies to achieve enhanced and durable treatment responses. In this study, a global elevation in the levels of m5C modification is observed in lenvatinib-resistant cells compared with their parental counterparts. DNA methyltransferase 2(DNMT2), functioning as an RNA methyltransferase, is markedly upregulated in lenvatinib-resistant specimens and correlated with poor patient survival outcomes. Both in vitro and in vivo experiments indicated that DNMT2 downregulation effectively overcame lenvatinib resistance by reducing HCC cell proliferation and promoting apoptosis, thereby restoring drug sensitivity. Mechanistically, DNMT2 functioned together with Y-box binding protein 1 (YBX1) to stabilize downstream adenosine triphosphate citrate lyase (ACLY) mRNA through m5C modification. This process activated the Notch signaling pathway by increasing intracellular acetyl-CoA levels and promoting histone acetylation, driving the progression of lenvatinib resistance. Critically, pharmacological inhibition of ACLY combined with lenvatinib treatment enhanced the therapeutic efficacy against HCC and could reduce the tumor burden in multiple preclinical models. Collectively, the findings indicate the importance of the DNMT2-ACLY-NOTCH signaling axis in lenvatinib resistance and propose novel combinatorial therapies to improve HCC treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29f48bc69f970768eff1c6a6e3857996952c92eb" target='_blank'>
              DNMT2-m5C-ACLY Axis Promotes Lenvatinib Resistance in Hepatocellular Carcinoma Through Histone Acetylation-Mediated Notch Pathway.
              </a>
            </td>
          <td>
            Shiguang Yang, Jie Li, Shengwei Mao, Xuhui Zhao, Douwaner Liu, Jiafeng Chen, Jialu Fu, Yichao Bu, Xiaoling Wu, Shaoqing Liu, Qi Yu, Qiongzhu Dong, Zheng Tang, Yuan Fang, Yi Shi
          </td>
          <td>2025-12-19</td>
          <td>Advanced science</td>
          <td>1</td>
          <td>20</td>
        </tr>

        <tr id="Precancers, defined as normal-appearing or morphologically altered tissues with a risk of oncogenesis, exhibit various detectable manifestations across anatomical sites, including epithelial dysplasia, metaplasia, hyperplasia, and stromal fibrosis. Considering the prevailing assumption that most cancers arise from precancers, early intervention at the precancerous stage has immense potential to reduce cancer-related morbidity and mortality. However, the complex signaling networks governing precancer initiation and progression remain elusive, hampering the development of effective targeted interventions. This review synthesizes three critical dimensions of precancer biology: historical foundations tracing the conceptual evolution of precancer research over the past century; mechanisms underlying the multistep progression of precancer biology, encompassing epithelial and macro/microenvironmental remodeling; and signaling networks cataloging dysregulated pathways and their therapeutic potential. Over 10 signaling pathways, including the transforming growth factor-β (TGF-β), p53, Wnt, phosphatidylinositol 3-kinase (PI3K), and mitogen-activated protein kinase (MAPK) pathways, drive multistep malignant transformation. We further synthesize emerging evidence supporting microenvironmental dominance, proposing the novel “soil degeneration” hypothesis. This paradigm shift underscores the necessity for dual-window intervention in which early-phase microenvironmental normalization prevents the establishment of precancerous lesions and advanced-phase treatment concurrently addresses epithelial malignancy and stromal degeneration. This review bridges foundational molecular discoveries with translational clinical potential and advocates for precision intervention frameworks that extend from biomarker-guided risk assessment to synergistic remodeling of the precancer microenvironment, thereby redefining precancer intervention in the molecularly targeted era.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67dd61437ebeffa1b547c7834fa24572f4d7328c" target='_blank'>
              Signaling pathways and targeted interventions for precancers
              </a>
            </td>
          <td>
            Jin Yang, Shimeng Wang, Xin Li, Hongdan Xu, Tongxu Sun, Tao Hu, Jingjing Luo, Hongmei Zhou
          </td>
          <td>2026-01-07</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="This paper delves into the role of epigenetics in complementing and challenging Darwinian evolution theory within the field of evolution. Epigenetics has revealed new sources of genetic variation, with epigenetic variations such as DNA methylation and histone modifications capable of influencing gene expression without altering DNA sequences. These variations can spread and accumulate within populations, exhibiting reversibility and dynamism, enabling species to rapidly adapt to environmental changes. Additionally, epigenetics expands the scope of natural selection, revises the patterns of genetic information flow, and highlights the need for updated evolutionary predictive models. Researchers have proposed theoretical integration pathways such as the Extended Evolutionary Synthesis framework, multilevel selection theory, and gene-epigenome coevolution models, offering new perspectives and directions for evolutionary biology research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b730d3e4497777825de2252ebd00338b352de17" target='_blank'>
              Epigenetics Complements and Challenges Darwinian Evolution: From Gene Regulation to the Study of Species Adaptation
              </a>
            </td>
          <td>
            Meichen Zhou
          </td>
          <td>2025-12-19</td>
          <td>MedScien</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Drosophila melanogaster, a well-established model organism in biomedical research, has significantly advanced our understanding of genetic mechanisms underlying human diseases. Its genetic similarity to humans, with approximately 75% of human disease-related genes having orthologues in the fly genome, renders it a valuable tool for studying complex conditions like cancer. The organism’s low cost, short life cycle, and manipulable genome make it ideal for high-throughput studies. Recent advances have enabled the use of Drosophila as a model for different cancer types, including colorectal, thyroid, lung, and brain cancers. These models replicate key tumor characteristics such as cell proliferation, invasion, and metastasis. Drosophila’s utility in drug screening has been exemplified through heterochromatin-promoting assays and personalized medicine approaches using patient-specific gene expression data. Its role in testing therapeutic combinations, such as MEK inhibitors with statins for lung cancer, underscores its translational potential. Despite its utility, the model has notable limitations. Differences in tumor biology between flies and humans, especially regarding organ systems and immune responses, limit its applicability in certain cancer types and metastatic studies. Furthermore, environmental influences and the simplified genetic background of Drosophila may not accurately represent human cancer complexity. Ethical concerns and translational challenges also exist, warranting cautious interpretation of results. The recent advancements in CRISPR-based genome editing, particularly cytosine and adenine base editors, have improved gene-editing efficiency in Drosophila, supporting its ongoing use in precision oncology. Future directions point toward integrating Drosophila models with other in vivo and in silico systems to optimize personalized drug discovery and better emulate human tumor heterogeneity. While the use of Drosophila melanogaster has intrinsic limitations, its genetic tractability and compatibility with cutting-edge genomic tools position it as a complementary model to mammalian systems. Continued innovations in genetic engineering and model integration will further strengthen its role in translational oncology and precision medicine. This review explores the applications of the model in human cancer research and highlights the major challenges with its use. This is crucial for the fight against cancer, as it helps inform researchers of the potential areas that require further research, especially with the rapid evolution of personalised medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b860b652fc843a37123c4a5f30311be7939a37a1" target='_blank'>
              Applications and challenges of Drosophila melanogaster as a laboratory model in human cancer research: a narrative review
              </a>
            </td>
          <td>
            R. Abdullateef, Jonas Paul Ibekwe, Henry Oyoyo, Samuel Ogbodo
          </td>
          <td>2025-12-19</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Ewing sarcoma is a pediatric bone and soft tissue cancer thought to arise from mesenchymal stem cells (MSCs). It is characterized by fusion of the RNA-binding protein EWS to an ETS family transcription factor, most often FLI1. EWS::FLI1 induces changes in gene expression that have been well-studied and have been linked to its ability to bind to and alter chromatin state at GGAA repeat sequences within the genome. In addition to these effects, EWS::FLI1 alters mRNA splicing; however, the mechanism of this activity remains unclear.



 We curated a sarcoma-focused transcriptome from published long-read RNA sequencing of Ewing sarcoma, osteosarcoma, and synovial sarcoma patient tumors and matched adjacent normal tissue (215M reads). We supplemented this with published long-read RNA sequencing data from relevant normal cell types, osteosarcoma cell lines, and Ewing sarcoma lines with or without EWS::FLI1 knockdown (105M reads). We used this transcriptome as a reference to evaluate expression and splicing from short-read RNA-seq of Ewing sarcoma cell lines, MSCs, and MSC-based models of Ewing sarcoma and established Ewing-associated expression and splicing signatures.



 We generated scores based on these signatures in published knockdown-rescue experiments in which endogenous EWS::FLI1 was silenced by shRNA and replaced with exogenous shRNA-resistant mutants. Expression and splicing scores were highly correlated within these datasets (r2 = 0.89), suggesting that similar regions of the fusion protein might be required for both its functions. In contrast, the scores had limited correlation within an internal dataset of RNA-binding protein perturbations (r2 = 0.02). We next examined the effects of pharmacological agents thought to alter EWS::FLI1 activity. Interestingly, we found that lurbinectedin which disrupts EWS::FLI1’s DNA-binding could partially revert both expression and splicing scores. In contrast, YK-4-279 which disrupts the fusion’s protein-protein interactions and MS0621 which reverts a Ewing-associated chromatin signature had limited effects on these scores. To further interrogate the potential role of EWS::FLI1 DNA-binding in splicing regulation, we silenced EWS::FLI1 in TC-32 cells and replaced it with either the wildtype fusion or a DNA-binding null mutant. Unlike the wildtype construct, the DNA-binding null mutant could not rescue expression or splicing. Finally, we examined the effects of the previously published KRAB-ZFA7 repressor which silences chromatin at GGAA repeats while sparing single GGAA sites and found that it could revert both scores.



 Together, these data establish a potential connection between EWS::FLI1 activity at GGAA repeats and its effects on mRNA splicing. Additionally, we find that lurbinectedin, an FDA-approved agent currently in a phase 1/2 trial for Ewing sarcoma, suppresses both the splicing and expression effects of EWS::FLI1. Emerging compounds designed to target EWS::FLI1 DNA-binding directly will likely have similar effects.



 David V. Allegakoen, Jeffrey A. Toretsky. EWS::FLI1 activity at GGAA repeats is essential for maintenance of Ewing-like splicing [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr B013.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6934f37ed326d496d296a4806fccea6f323a4f63" target='_blank'>
              Abstract B013: EWS::FLI1 activity at GGAA repeats is essential for maintenance of Ewing-like splicing
              </a>
            </td>
          <td>
            D. Allegakoen, J. Toretsky
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="TP53 mutation-driven gene expression programs define oncogenic phenotypes. While extensive studies have concentrated on the transcriptome and proteome, post-transcriptional processes, particularly translational variation, remain underexplored. This study presents a comprehensive analysis of the transcriptomics, translatiomics, and proteomics dynamics in the ovarian cancer cell line SKOV3, with a focus on the effects of p53 missense mutations (R175H, R273H, and Y220C) on gene dosage fluctuations. Despite clear transcriptional differences between wild-type and mutant p53, we find that extensive translational and post-translational buffering processes attenuate these discrepancies, yielding comparatively stable protein abundances. Moreover, we delineate that the relative contributions of transcription output, translation engagement, and protein stability collectively shape the final protein abundance in the context of p53 mutations. Clinical proteomic analysis of platinum-resistant ovarian cancer tissues reveals tumor-specific factors and acquired resistance pathways linked to p53 mutations. Our findings elucidate the multilayered regulatory landscape of p53 mutations and identify potential risk factors for platinum resistance associated with these mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/765c815e9b25f95d1edceba753e2ba1a94641e62" target='_blank'>
              Multilayered Regulatory Dynamics of p53 Mutations and Platinum Resistance in Ovarian Cancer.
              </a>
            </td>
          <td>
            Liling Hu, Hanchen Zou, LvYing Peng, Fan Li, Danya Liu, Jiangli Lu, Yuying Li, Chris Zhiyi Zhang, Qiu-Hong Tian
          </td>
          <td>2025-12-16</td>
          <td>Journal of proteome research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Oncofusions, cancer-causing chimeric proteins formed by chromosomal rearrangements, are key drivers in approximately 16% of all cancer cases. These fusion proteins are associated with a wide range of malignancies, including varieties of leukemia, lung cancer, and sarcoma. Thousands of unique oncofusions have been identified in recent years, yet most of these fusion proteins lack mechanistic annotation. Many oncofusions dysregulate chromatin architecture in complex ways that are not yet fully elucidated. Investigating these chromatin alterations is both labor-intensive and costly, limiting the feasibility of large-scale studies. This lack of mechanistic insight has hindered the development of targeted therapies, with only a handful of efficacious small-molecule inhibitors directed toward oncofusions currently in clinical trials or approved for patient use. To address this critical gap, we adapted our recently developed PROD-ATAC method into a high-throughput, scalable platform to systematically elucidate the effects of dozens of candidate inhibitory small molecules on the chromatin dysregulation profiles induced by a pooled library of diverse oncofusions. In the PROD-ATAC scheme, a pooled protein-coding variant library is expressed in a single disease-agnostic cell line, and scATAC-seq (single-cell assay for transposase accessible chromatin) catalogs each variant’s effect on the chromatin landscape. We applied PROD-ATAC to simultaneously assay the chromatin regulatory impact of >40 small molecule inhibitors against a panel of tens of oncofusions (including the majority of patient-identified fusion sequences causal for Ewing sarcoma) in a single assay, generating genome-wide chromatin accessibility datasets representative of >1,000 unique drug - fusion interactions. In this manner, we characterized several small molecules as previously unidentified inhibitors of multiple oncofusions. Certain inhibitors were found to abrogate the chromatin dysregulation caused by an oncofusion either globally or in a targeted cell signaling pathway-specific manner. By profiling multiple rare fusions causal for the same cancers, we identified separate instances of homogenous and heterogenous modes of chromatin dysregulation associated with distinct fusion proteins. In this work, PROD-ATAC chromatin accessibility readouts coupled the identification of novel oncofusion inhibitors with the characterization of each inhibitor’s impact on the epigenetic landscape of oncofusion-expressing cells.



 James E. Corban, Max Frenkel, Srivatsan Raman. High-throughput characterization of inhibitor impacts on chromatin dysregulation induced by oncofusions [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr A024.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ff99b547f5b85ca4c657ff5fd20f9c13db3f972" target='_blank'>
              Abstract A024: High-throughput characterization of inhibitor impacts on chromatin dysregulation induced by oncofusions
              </a>
            </td>
          <td>
            James E. Corban, Max Frenkel, Srivatsan Raman
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="PURPOSE OF REVIEW
Despite advances in targeted therapy and immunotherapy, glioblastoma (GBM) remains a therapeutic challenge because of its intrinsic adaptability, driven by intratumoural heterogeneity, cell-state plasticity, tumour-stroma crosstalk, and an immunosuppressive tumour microenvironment. The aggressive mesenchymal subtype, linked to treatment resistance and poor prognosis, exemplifies these adaptive mechanisms. Recent studies identify tractable oncogenic dependencies within these processes, opening routes to subtype-informed, multiaxis therapies.


RECENT FINDINGS
GBM exploits extrachromosomal DNA (ecDNA) and structural variants to rewire enhancer networks, reinforcing therapy-resistant mesenchymal states with distinct kinase dependencies (e.g. p38 MAPK signalling, STAT3). Tumour cells further hijack neuronal activity, glutamate/GABA, and long-range neuromodulatory (e.g. cholinergic) inputs, to promote growth, with synaptically enriched regions exhibiting immune suppression and mesenchymal enrichment. Myeloid-derived ligands (e.g. TNF and Oncostatin M) can drive proneural to mesenchymal transition, while Thrombospondin-1-mediated synaptic remodelling suppresses T-cell function, mechanistically coupling neural connectivity to immune evasion.


SUMMARY
The resilience of GBM arises from the interplay of epigenetic plasticity, neural circuit co-option, and myeloid-skewed immunosuppression. A coordinated strategy, kinome-guided targeting plus circuit disruption and myeloid reprogramming, offers a credible path to contain adaptation and improve outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/477ee36d35bf0a27b85e21d8ea8f32e1953a4901" target='_blank'>
              Interrupting glioblastoma malignant circuitry: plasticity traps, synaptic blockade, and myeloid reprogramming.
              </a>
            </td>
          <td>
            Hong Sheng Cheng, Carol Tang, Nguan Soon Tan
          </td>
          <td>2026-01-08</td>
          <td>Current opinion in clinical nutrition and metabolic care</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The advent of next-generation high-throughput sequencing and precision medicine has allowed the identification of recurrent genetic events of vital interest for leukemic cells and for the management of patients with leukemia, since it has a direct impact on prognosis and therapeutic stratification. The precise cytogenetic characterization of these diseases has rapidly enabled the development of innovative therapeutic strategies to improve outcomes and limit toxicities. New targeted therapies, including small molecule inhibitors, monoclonal antibodies, bispecific T cell engagers, antibody–drug conjugates, recombinant immunotoxins, and chimeric antigen receptor T cells (CAR-T cells) represent a broad therapeutic arsenal now available in everyday practice. The growing number of treatments means that many possibilities of drugs and combinations can be used to offer new lines of therapy in cases of refractory disease or relapse, but also in frontline treatment. This review provides a broad overview of pathogenic mechanisms and therapies (including targeted therapies, immunotherapy and RNA-based therapy) and gathers the main research progress in the field of leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9ad84d56b4eca59a338789eb4fc5c1bd3ecb571" target='_blank'>
              Molecularly targeted therapy and immunotherapy in leukemias
              </a>
            </td>
          <td>
            Elie Cousin, Agathe Picard, Maria Oundjian, Nina Tardif, Anaïs M. Quéméner, Anne-Gaëlle Rio, M. Galibert, Virginie Gandemer, Frédéric Mazurier
          </td>
          <td>2026-01-11</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Genome-wide association studies (GWAS) of melanoma have identified numerous susceptibility loci. However, causal genes and variants underlying risk have yet to be established for most. It is becoming apparent that many functional variants underlying complex traits act via cis-regulation that may be context-specific, dependent on availability of specific transcription factors/complexes in specific cell types and cell-states. To characterize a risk locus on chromosome band 2q33.1 associated with melanoma, breast cancer, and keratinocyte cancers, we integrated fine-mapping, cell-type specific expression quantitative trait locus (eQTL) analysis, a massively parallel reporter assay, individual luciferase assays, and SNP-based proteomics. Integrated analysis implicates the presence of multiple functional variants lying primarily within a promoter for CASP8. A haplotype containing rs3769823 appeared have the largest effect on expression. Strikingly, both tumor/normal context and this risk-associated haplotype play critical roles in mediating allelic cis-regulatory activity. Quantitative mass spectrometry for rs3769823 identified both E4F1, a transcriptional repressor, and IRF2, a transcriptional activator, as binding preferentially to risk-associated rs3969823-A. The binding of these transcription factors was validated via EMSA, supershift, and chromatin immunoprecipitation (ChIP) assays. The relative levels of E4F1 and IRF2 differ by cell-type and play a role in mediating transcriptional activity in a cell-type specific manner. Our results indicate that the top credible causal set variant rs3769823 likely influences expression of CASP8 and FLACC1 in a cell-type specific manner and may be a relevant functional variant for multiple cancers associated with this locus.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d0bd03d9d9be45c3f139097e5cecc84ff17cc6a" target='_blank'>
              Functional characterization of a multi-cancer risk locus on chromosome band 2q33.1 near CASP8
              </a>
            </td>
          <td>
            Hyunkyung Kong, Jiyeon Choi, Tongwu Zhang, Cathrin Gräwe, Mai Xu, Rohit Thakur, Hayley A. Sowards, Rebecca Hennessey, Andrew Vu, Jianxin Shi, T. Bishop, Julia Newton-Bishop, Jérémie Nsengimana, M. Iles, M. Landi, M. Vermeulen, M. Law, L. Amundadottir, Melanoma Meta-7 Analysis, Consortium, Kevin M. Brown
          </td>
          <td>2026-01-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) represent a highly plastic subpopulation within tumors that drives tumor initiation, progression, metastasis, therapeutic resistance, and disease relapse. Although CSC traits were initially attributed to stable genetic hierarchies, growing evidence suggests that cancer stemness is a dynamic and reversible state regulated by precise molecular events at the protein level. This literature review aims to synthesize experimental evidence regarding the impact of single amino acid mutations and alterations in regulatory proteins on the stemness properties of cancer cells. A systematic literature search was conducted using PubMed and Google Scholar to identify original research articles published between 2020 and 2025 that examined protein-level mutations, post-translational modifications, epigenetic regulators, and chromatin remodeling factors in diverse cancer models. The reviewed studies consistently demonstrate that single amino acid substitutions in oncogenic, tumor suppressor, or regulatory proteins can profoundly affect protein conformation, stability, enzymatic activity, and protein–protein interactions. These alterations frequently result in sustained activation of key stemness-associated signaling pathways, including Wnt/β-catenin, Notch, Hedgehog, and STAT3, thereby enhancing self-renewal capacity, stem cell marker expression, tumor-initiating potential, and resistance to conventional therapies. In parallel, dysregulation of post-translational modification systems, such as phosphorylation, ubiquitination, acetylation, and methylation, promotes epigenetic reprogramming and reactivation of embryonic transcriptional programs that favor cellular plasticity. Moreover, protein-level alterations strengthen cancer cell interactions with the tumor microenvironment, increasing responsiveness to hypoxia, inflammatory signals, and metabolic stress. Collectively, these findings highlight protein-level regulation as a central determinant of cancer stem cell maintenance and plasticity, and suggest that effective therapeutic strategies should target aberrant protein stability, post-translational modification machinery, and stemness-related signaling networks to achieve durable clinical responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f961c9b291a85047c8455458a035edcd22afcac" target='_blank'>
              IMPACT OF SINGLE AMINO ACID MUTATIONS OR ALTERATIONS IN REGULATORY PROTEINS ON THE STEMNESS PROPERTIES OF CANCER CELLS
              </a>
            </td>
          <td>
            Dewi Sartika, Irmatul Laili, Sri Utami
          </td>
          <td>2025-12-30</td>
          <td>Journal of Stem Cell Research and Tissue Engineering</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d21384971b0ced62ebc3c05bc7a513b76dd2fb8" target='_blank'>
              Five dominant amino acid substitution signatures shape tumour immunity.
              </a>
            </td>
          <td>
            Szilvia Juhász, B. Papp, Anna Tácia Fülöp, Z. Farkas, Dávid Kókai, Dóra Alexandra Gyémánt, Franciska Tóth, Zsófia Nacsa, Dóra Spekhardt, Balázs Koncz, Péter Burkovics, Csaba Pál, Máté Manczinger
          </td>
          <td>2026-01-28</td>
          <td>Molecular systems biology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebed292326a57b5e447307df550aebc051db233e" target='_blank'>
              RNA-guided STAT3 modification fine tunes the epigenetic and epitranscriptomic regulation of CD4 + T helper cell differentiation during non-small cell lung cancer (NSCLC)
              </a>
            </td>
          <td>
            Roshni Bibi, Melvin George, Koustav Sarkar
          </td>
          <td>2025-12-27</td>
          <td>Medical Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Most intracellular proteins undergo degradation by proteasomes, which exist in constitutive, immune, and intermediate forms. The diversity arises from the presence of different catalytic β subunits incorporated into the 20S core particle. Constitutive proteasomes contain β1, β2 and β5 subunits, whereas immunoproteasomes integrate β1i, β2i and β5i proteins. Intermediate proteasomes comprise combinations of constitutive and immune subunits, but typically lack the β2i subunit. Distinct proteasome configurations have widespread impact on cellular metabolism, gene expression, immune signaling, stress adaptation and tumorigenesis. However, the biological functions of proteasome subtypes remain incompletely defined, underscoring the need for refined experimental systems to uncover their specific activities. Methods Previously, we generated cancer cell lines expressing mCherry-tagged β5i subunit under the control of endogenous regulatory elements. Using CRISPR/Cas9 nickase technology, we further modified the genomes of the colon adenocarcinoma SW620B8-mCherry and cervical adenocarcinoma TZM-blB8-mCherry cells. In SW620B8-mCherryB5-GFP cells, the eGFP gene was inserted at the 3’ end of the PSMB5 gene, which encodes the β5 subunit. Similarly, the β2i-encoding gene (PSMB10) was fused with the photo-switchable cyan fluorescent protein gene (PS-CFP2) in TZM-blB8-mCherryB10-CFP cells. Results Efficient integration of fluorescent protein-encoding sequences into target genomes provided robust expression and integration of chimeric subunits into proteasomes. In TZM-blB8-mCherryB10-CFP cells, photo-conversion of PS-CFP2 allowed visualization of both intermediate and immunoproteasomes within the cellular nuclei following treatment with IFN-γ/TNF. In SW620B8-mCherryB5-GFP cells, analysis of intracellular proteasome localization revealed discrete regions enriched in β5i-containing proteasomes. Moreover, IFN-γ/TNF exposure induced a fluorescence shift from green to red, accompanied by redistribution of intracellular proteasomes in SW620B8-mCherryB5-GFP cells. Multiphoton microscopy of tumors grafted into immunocompromised mice showed the formation of β5i-containing, proteasome-enriched inclusions and suggested their potential release from cancer cells. Collectively, the TZM-blB8-mCherryB10-CFP cell line provides a tool to interrogate immunoproteasome functions independently of intermediate proteasomes, whereas SW620B8-mCherryB5-GFP cells enable visual discrimination between constitutive and β5i-containing proteasomes both in vitro and in vivo. Conclusions Generated cell lines provide a novel platform for dissecting proteasome functions, addressing the distinct properties of individual proteasome forms, and assessing the impact of diverse compounds on the proteasome repertoire in cultured cells and in tumor xenografts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8f185dc9839ab4abec820b173f8f7dd6f377d1a" target='_blank'>
              Next-generation cell lines for profiling different proteasome forms and their implications in cancer
              </a>
            </td>
          <td>
            A. Burov, Anastasia Yakovleva, Georgy Linovskiy, Andrei Fedotov, V. Popenko, Olga Leonova, Aleksandr Lanin, Marina Drutskaya, Vadim Karpov, Alexey Morozov
          </td>
          <td>2025-12-16</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="In the past 20 years, advances in genomic technologies have greatly improved our understanding of pediatric acute myeloid leukemia (AML). Today, cytogenetic tests can detect structural changes in approximately 75% of cases and remain a main tool for assessing risk. Recent technologies, such as next-generation sequencing, are revealing additional structural alterations (cryptic fusions) and mutations that often cooperate to influence disease biology and treatment response. This evolving genetic landscape has identified unique childhood subtypes of AML defined by specific fusions, such as NUP98::NSD1, CBFA2T3::GLIS2, and varied KMT2A rearrangements, which are linked to distinct clinical outcomes. Emerging data also point to the poor prognosis associated with certain subtypes of NPM1, like the NPM1-D isoform. Additionally, mutations in genes like WT1, DNMT3A, and TP53, the latter of which are rare in childhood AML, may influence patients' outcomes, particularly when occurring in combination. Targeted therapies, including FLT3, BCL2, and menin inhibitors, are beginning to reshape treatment, offering more personalized approaches. However, integrating these drugs effectively into the patient's treatment strategy remains challenging due to the genetic complexity and rarity of pediatric AML. Key issues ahead include identifying which genetic features truly affect outcomes, using this information to personalize therapy, predicting who will benefit from targeted drugs, and choosing the best markers to track disease response over time. Looking forward, collaborative efforts are urgently needed to validate pediatric-specific biomarkers, test novel drug combinations, and link genetic data to clinical outcomes to design trials and future treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03552c285ee4b593766096eaba70240ea7a0a020" target='_blank'>
              Using genomics to refine pediatric AML risk stratification.
              </a>
            </td>
          <td>
            M. Pigazzi, S. Meshinchi, Franco Locatelli
          </td>
          <td>2025-12-05</td>
          <td>Hematology. American Society of Hematology. Education Program</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/291cf6a8bf33099b2cf453177158ffc80d04fda4" target='_blank'>
              Immunotherapy in Melanoma: A Dynamic Frontier in Cancer Treatment
              </a>
            </td>
          <td>
            Azhar Ahmed, M. Alahmadi, Z. Khawaji, Ahmad Eissa, S. Alghamdi
          </td>
          <td>2026-01-19</td>
          <td>Current Treatment Options in Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The Methyl CpG Binding Protein 2 (MECP2) gene has copy number gain in a number of human cancers and functions as an oncogene through an unusual epigenetic mechanism. We explored the possibility that MECP2 might be a therapeutic target, and whether its epigenetic mode of action is amenable to specific therapy. Constitutively expressed or inducible lentiviral short hairpin RNA (shRNA) directed at MECP2 in human triple negative breast cancer (TNBC) cell lines with high or low level of MECP2 protein were grown as xenografts to assess oncogene addiction. We next evaluated the effect of DNA methylation inhibitors or histone deacetylase inhibitors on monolayer or soft agar growth of isogenic human mammary epithelial cells with or without MECP2 overexpression, and xenograft growth of MECP2-dependent or MECP2-independent human TNBC or lung cancer cell lines. We then investigated the mechanism of MECP2-induced activation of MAPK pathway and assessed the effect of drug treatment. Human cancer cell lines with MECP2 overexpression show MECP2 dependence, and epigenetic drugs are effective in these models. Activated RAS and other activators of the MAPK pathway caused resistance to these therapies, giving insight into their novel mode of action and demonstrating specificity. The kinase PAK3 is important for MECP2-mediated induction of MAPK pathway, is modulated by epigenetic drugs affecting MECP2 action as expected and may itself be a therapeutic target for MECP2-driven cancers. These preclinical studies show that tumors overexpressing MECP2 might benefit from epigenetic therapy targeting MECP2 function and demonstrate a novel mechanism of action for these drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4802b6674f8b3e0a4e440436dd1fc69dc98c09c" target='_blank'>
              MECP2-Dependent Cancers Can Be Targeted by Epigenetic Drugs: A New Role for Epigenetic Cancer Therapy.
              </a>
            </td>
          <td>
            Manish Neupane, Mukesh Kumar, Malela A Mwamufiya, Colleen M Donnelly, Alsu Ibragimova, Brian Curcio, Inna Chervoneva, Irina Vasilevskaya, Daniel P Silver
          </td>
          <td>2025-12-11</td>
          <td>Molecular cancer therapeutics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a malignant blood disease with a poor prognosis. With the development of targeted therapies, many refractory/relapsed or unfit AML patients have achieved better survival rates. However, primary or acquired resistance frequently occurs, and some targets become unavailable. This study aimed to identify and verify novel targets for AML using Mendelian Randomization (MR) analysis. The association between plasma protein quantitative trait loci (pQTL), expression quantitative trait loci (eQTL), and AML was examined via MR. Bayesian co-localization analysis was used to detect common segments of TCL1A and AML. Bioinformatics technology was used to explore the relationship between TCL1A and AML, and quantitative real-time PCR (RT-qPCR) analysis was used to detect the expression of TCL1A in various cell lines. Furthermore, high-throughput virtual screening (HTVS) and molecular dynamics (MD) simulation was used to identify potential TCL1A inhibitors. Further verification was carried out by AML cell experiments. Our study found a strong association between plasma TCL1A protein levels and AML (OR: 1.66; 95% CI: 1.46–1.88), and rs78986913 was the same variant in both TCL1A and AML. High TCL1A expression was associated with low overall survival rates in AML patients (P < 0.001), especially in the FAB M0 and M2 subgroups. Additionally, TCL1A was abnormally highly expressed in many AML cell lines (including MOLM-13, MV4-11, THP-1) compared to normal bone marrow stromal cells (HS-5) and other solid tumor cells. We also identified 19 small molecules, finding that Bletilloside A (id number: 2292159-89-6) is one of the most fitting small molecules for the TCL1A protein pocket. MD simulation confirmed the good stability of the Bletilloside A-TCL1A complex. And Bletilloside A could inhibit the proliferation of AML by inhibiting TCL1A. There is a causal relationship between TCL1A and AML. The expression level of TCL1A deserves further monitoring in clinical settings, and the benefits of targeting TCL1A should be further validated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5cf4dfa88d970f81cc960282fc2f28f7a5e6b5a2" target='_blank'>
              New drug targets for acute myeloid leukemia identified through a comprehensive analysis of the plasma protein
              </a>
            </td>
          <td>
            Yaonan Hong, Xu Wang, Jingjie Dong, Qi Liu, Yingying Shen, Xiawan Yang, Peicheng Wang, Wenjie Li, Baodong Ye, Keding Shao, Ya-Huei Kuo, Dijiong Wu
          </td>
          <td>2025-12-10</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Persistent fetal gene expression in childhood neoplasms is usually explained by a maturation block originating in the prenatal phase. In contrast, reactivation of fetal genes in adult malignancies is considered a consequence of oncofetal reprogramming (OFR) and is associated with aggressive disease. By reconstructing epigenetic ontogeny in juvenile myelomonocytic leukemia (JMML), we identified a postnatal maturation state of JMML stem cells with high transcriptional plasticity indicative of OFR in high-risk disease. Similarly, post-natal activation of oncogenic signaling by inducible Ptpn11E76K mutation in mice, triggered molecular plasticity and reactivation of fetal gene expression. Integrative multi-omics analysis revealed aberrant CD52 expression as a feature of high-risk JMML stem cells. Anti-CD52 treatment depleted JMML stem cells and blocked disease propagation in xenograft models. Our results challenge the prevailing maturation-block model of pediatric leukemogenesis and establish RAS-associated stem-cell plasticity as a determinant of OFR and potential therapeutic vulnerabilities in high-risk JMML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c603c5b4c0adc417c6958428e24696c22fabe88d" target='_blank'>
              Molecular Plasticity Results in Oncofetal Reprogramming and Therapeutic Vulnerabilities in Juvenile Myelomonocytic Leukemia.
              </a>
            </td>
          <td>
            M. Hartmann, M. Schönung, J. Rajak, Valentin J. Maurer, Ling Hai, K. Bauer, M. Hakobyan, Sina Stäble, Jens Langstein, Laura Jardine, R. Roelz, S. Bohler, Eleonora Khabirova, Abdul-Habib Maag, Dominik Vonficht, D. Lebrecht, K. Bernt, Kai Tan, Changya Chen, Fatemeh Alikarami, Julia A. Meyer, Jun Wang, Tobias Boch, Viktoria Flore, P. Lutsik, M. Milsom, S. Raffel, Christian Buske, Simon Haas, M. Haniffa, Jan-Philipp Mallm, S. Behjati, M. Bonder, Stefan Frohling, Elliot Stieglitz, Charlotte M. Niemeyer, Joschka Hey, C. Flotho, C. Plass, M. Erlacher, Matthias Schlesner, D. Lipka
          </td>
          <td>2025-12-05</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="Cancer reversion therapy represents a paradigm shift in oncology, focusing on reprogramming malignant cells to a non-malignant state rather than destroying them. This narrative review synthesizes current evidence, emerging technologies, and future directions in this promising field. Cancer reversion is founded on key biological observations: somatic cell reprogramming, spontaneous cancer regression, and microenvironmental influences on malignant behavior. Current approaches include epigenetic reprogramming using HDAC inhibitors and DNA methyltransferase inhibitors; microenvironmental modulation through extracellular matrix manipulation and vascular normalization; differentiation therapy exemplified by all-trans retinoic acid in acute promyelocytic leukemia; and targeting oncogene addiction as demonstrated in BCR-ABL-driven leukemias. Emerging technologies accelerating progress include single-cell analyses that reveal cancer heterogeneity and cellular state transitions; CRISPR-based approaches enabling precise genetic and epigenetic manipulation; patient-derived organoids that model tumor complexity; and artificial intelligence applications that identify novel reversion-inducing agents. Critical evaluation reveals that many reported “reversion” phenomena represent stimulus-dependent plasticity or transient growth arrest rather than stable phenotypic normalization. True cancer reversion requires durable, heritable phenotypic changes that persist after treatment withdrawal, with evidence of epigenetic consolidation and functional restoration. Despite promising advances, significant challenges remain: cancer cell plasticity facilitating therapeutic escape, difficulties in establishing stable reversion states, delivery challenges for solid tumors, and the need for combination approaches to address tumor heterogeneity. Future directions include integrated multi-omics analyses to comprehensively map cellular state transitions, studies of natural regression phenomena to identify reversion mechanisms, advanced nanodelivery systems for targeted therapy, and synthetic biology approaches creating intelligent therapeutic systems. By redirecting rather than destroying cancer cells, reversion therapy offers the potential for reduced toxicity and resistance, potentially transforming cancer from a deadly disease to a manageable condition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68ef16378f672c1c6324bdae132a2624ad963a92" target='_blank'>
              Cancer Reversion Therapy: Prospects, Progress and Future Directions
              </a>
            </td>
          <td>
            Emmanuel O. Oisakede, D. Olawade, Oluwakemi Jumoke Bello, Claret Chinenyenwa Analikwu, Eghosasere Egbon, Oluwaseun Fapohunda, S. Boussios
          </td>
          <td>2025-12-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>1</td>
          <td>47</td>
        </tr>

        <tr id="Abstract Background Chordoma is a rare bone tumor that is considered to have originated from the remnants of notochord by the general consensus in the field. Chordoma patients suffer from local invasion and recurrence of the disease, along with high resistance to conventional therapies. Despite the scarcity of the disease, new biomarkers must be discovered to develop targeted therapeutics. Methods This study aimed at analyzing the whole exome sequencing data of a chordoma cell line, CH22, which held more than 3,000 missense variants in the genome. These missense-carrying genes were investigated in terms of their association with chordoma pathogenesis. Results Out of them, neurotrophic tyrosine kinase receptor type 3 (NTRK3) and nuclear factor erythroid 2-related factor 2 were found to be associated with the altered pathways involved in chordoma. Conclusion Further examination through structural modeling, molecular docking, and phylogenetic analyses related to the missense variants of NTRK3 was conducted in the scope of this study.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b2fe9308814058a063733e1cf9abe4023d9644d" target='_blank'>
              Deciphering the Structural and Molecular Effects of a Novel Genetic Mutation by In Silico Analysis in Chordoma Cell Line
              </a>
            </td>
          <td>
            Muhammed Ümeyr Fettah, Nur Ekimci-Gürcan, Emre Yorulmaz, Batuhan Oğuz, A. Köseoğlu, Furkan Ayaz, Esra Aydemir
          </td>
          <td>2025-09-30</td>
          <td>Journal of Neurological Surgery Part B: Skull Base</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background/Aim: Prostate cancer features profound transcriptional dysregulation within the androgen receptor (AR) signaling axis. The pioneer factor FOXA1, which facilitates AR binding to chromatin, is recurrently altered in 10-40% of tumors. Recent studies classify FOXA1 mutations as Class 1 Wing 2 mutations, which enhance AR-dependent tumorigenesis, and Class 2 C-terminal truncations, which promote lineage plasticity and therapy resistance. The interplay of FOXA1 alterations with TMPRSS2-ERG fusions and PROX1 remains incompletely understood. Materials and Methods: Data from The Cancer Genome Atlas (TCGA) Prostate Adenocarcinoma (PRAD) cohort (n=492) were analyzed via UCSC Xena and cBioPortal. FOXA1 mutations were categorized following Eyunni et al. Copy number was assessed by log2(tumor/normal) ratios. Mutual exclusivity and co-occurrence were evaluated using Fisher’s exact test with false-discovery-rate correction. Associations between FOXA1 status and genomic instability were assessed using the fraction genome altered (FGA) metric. Results: FOXA1 was broadly expressed, with subsets showing elevation. Class 1 mutations localized to the Wing 2 region, while Class 2 truncations clustered in the C-terminal domain. Copy number changes were infrequent, indicating mutation-driven reprogramming as the main oncogenic mechanism. TMPRSS2 and ERG strongly co-occurred (log2 OR >3, q<0.001), whereas FOXA1 was mutually exclusive with both TMPRSS2 and ERG (q<0.001). Although FOXA1 alterations showed no significant Pearson correlation with FGA (r=−0.01, p=0.76), a moderate Spearman correlation (ρ=0.52, p<0.001) suggested enrichment in genomically unstable tumors. Conclusion: FOXA1 defines a major oncogenic axis in prostate cancer, distinct from TMPRSS2-ERG fusion and PROX1 induction. Class 1 and 2 FOXA1 mutations drive alternative transcriptional programs leading to therapy resistance, highlighting FOXA1 as a critical biomarker and target for chromatin-directed interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/962282da85200138085414df96f69e7ea38e7831" target='_blank'>
              FOXA1 Alterations in Prostate Cancer: Expression, Mutation Classes, and Copy Number Changes
              </a>
            </td>
          <td>
            S. Lehrer, P. Rheinstein
          </td>
          <td>2026-01-01</td>
          <td>In Vivo</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Rhabdomyosarcoma (RMS), the most common pediatric soft tissue sarcoma, exhibits marked clinical heterogeneity driven by poorly understood molecular mechanisms. Identifying the molecular characteristics of different RMS subtypes and the molecular pathways influencing the RMS treatment response and recurrence is an urgent clinical need. Here, we perform deep proteomic profiling of 19 RMS tumors (8 alveolar [ARMS], 11 embryonal [ERMS]) and matched normal tissues, integrating bioinformatics with functional validation to delineate subtype-specific pathways, therapy resistance drivers, and actionable targets. ARMS tumors are characterized by ubiquitination pathway activation (UBE2R2, UBE2J2), while ERMS exhibits spliceosome dysregulation. Chemo- and radio-resistant tumors both show significant enrichment in the ribosome pathway. Relapsed cases show phosphonate and phosphinate metabolism pathway enrichment, suggesting metabolism reliance. Unsupervised clustering reveals ribosome- and glycolysis-driven subtypes with distinct metabolic dependencies. Functional studies implicate MED18─a core component of the Mediator complex─in mediating therapy resistance possibly via promoting DNA damage repair. Our study establishes proteomics as a tool to decode RMS heterogeneity, proposing subtype-tailored strategies targeting ubiquitination, splicing, and metabolism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/864be8bb8da57a2925ca6df4d25ae9855e5c9074" target='_blank'>
              Proteomic Profiling Reveals Candidate Proteins and Pathways Associated with Chemo-Radio-Sensitivity and Relapse in Rhabdomyosarcoma.
              </a>
            </td>
          <td>
            Zhiyuan Zhou, Ying Ye, Wenbin Guan, Chuanying Zhu, Lu Wen
          </td>
          <td>2026-01-15</td>
          <td>Journal of proteome research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Prostate adenocarcinomas are the most common form of the disease. Targeting the androgen receptor (AR) is the principal treatment strategy. However, resistance is common. Lineage plasticity (LP)—change in differentiation state—is increasingly recognized as a mechanism of resistance to AR-targeting therapies. LP is a continuum ranging from amphicrine to neuroendocrine prostate cancer (NEPC). Despite the increasing incidence of LP due to more widespread use of new and potent AR inhibitors, mechanisms by which LP emerges after therapy have been understudied due to paucity of matched patient tumors. To address this deficit, we performed multi-omic profiling on matched biopsies from 16 patients, including seven patients that underwent LP after treatment.



 We used DNA sequencing to identify mutations and copy number alterations present in tumors at baseline and at progression. We also performed RNA sequencing and transcriptional subtyping of all tumors.



 Analysis of RNA sequencing revealed two distinct subtypes of LP in our cohort—most consistent with amphicrine or NEPC. DNA sequencing identified TP53 and RB1 alteration patterns in baseline tumors that eventually underwent specific LP trajectories. Pathways linked to proliferation and inflammatory signaling were highly activated in baseline tumors that eventually underwent LP.



 Our analysis identified distinct LP trajectories linked with specific genomic and transcriptomic features at baseline. Profiling tumors with this approach may provide clues about which patients may be at greatest risk for undergoing specific LP trajectories after therapy.



 Anbarasu Kumaraswamy, Visweswaran Ravikumar, Ryan J. Rebernick, Eva Rodansky, Amina Tanweer, Joel A. Yates, Aaron M. Udager, Marcin P. Ceislik, Arvind Rao, Zachery R. Reichert, Arul M. Chinnaiyan, Joshi J. Alumkal. Multi-Omic Profiling Clarifies Mechanisms of Therapy-Induced Prostate Cancer Lineage Plasticity [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A036.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d201d414071181c06c4abf5de951c2f23846da37" target='_blank'>
              Abstract A036: Multi-Omic Profiling Clarifies Mechanisms of Therapy-Induced Prostate Cancer Lineage Plasticity
              </a>
            </td>
          <td>
            A. Kumaraswamy, V. Ravikumar, Ryan J. Rebernick, Eva Rodansky, Amina Tanweer, Joel A. Yates, Aaron M. Udager, Marcin P. Ceislik, Arvind Rao, Z. Reichert, A. Chinnaiyan, J. Alumkal
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>170</td>
        </tr>

        <tr id="Chimeric antigen receptor (CAR) T cell therapies have demonstrated remarkable efficacy in several hematological malignancies, yet their success has not been fully replicated in solid tumors. Moreover, even in hematological cancers, relapse after CAR T cell infusion continues to compromise long-term outcomes. These challenges highlight the urgent need to develop strategies that enhance CAR T cell efficacy, persistence, overcoming tumor and microenvironment-mediated resistance. Clustered regularly interspaced short palindromic repeats (CRISPR)-Cas9-based screening platforms provide a powerful approach to systematically identify genes that regulate CAR T cell function. By linking genetic perturbations to phenotypic outcomes, these assays enable the discovery of pathways controlling activation, proliferation, memory formation, and cytotoxicity. Standard workflows involve transduction of substantial numbers of cells with a single guide RNA (sgRNA) library, Cas9-mediated editing, selection of edited cells, and PCR amplification of sgRNA cassettes from genomic DNA (gDNA) prior to sequencing. However, PCR amplification using large amounts of gDNA poses significant challenges and often fails to selectively amplify and retrieve sgRNAs. Here, we describe an optimized CRISPR-Cas9 knockout screening protocol, which we have tested on primary human CAR T cells. The method here incorporates an intermediate step during sgRNA library preparation that reduces gDNA carryover through enzymatic digestion and selective pulldown of the sgRNA cassette, thereby increasing the efficiency of the first PCR amplification. This modification allowed us to retrieve sgRNA information across our CAR T cell screens, which had remained elusive in our previous attempts using traditional 1 and 2-step PCR amplification protocols. In conclusion, this optimized workflow facilitates CRISPR screening library preparation in challenging samples and enables the identification of key genetic determinants that can be targeted to improve therapeutic efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecca94f275145ee359153517eba4093f85ededd3" target='_blank'>
              Improving CRISPR-Cas9 Screens in CAR T Cells: A Refined Method for Library Preparation.
              </a>
            </td>
          <td>
            Maider Garnica, Patxi San Martín-Úriz, Paula Rodriguez-Marquez, M. E. Calleja-Cervantes, Saray Rodríguez-Díaz, Rebeca Martinez-Turrillas, Mikel Hernaez, Felipe Prósper, J. R. Rodríguez-Madoz
          </td>
          <td>2026-01-02</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Adult T-cell leukemia/lymphoma (ATLL) is an aggressive T-cell malignancy caused by persistent infection with human T-cell leukemia virus type 1 (HTLV-1). ATLL remains difficult to treat despite intensive chemotherapy, antiviral therapy, and hematopoietic stem cell transplantation. The limited durability of current treatment strategies highlights the need for mechanism-based therapeutic approaches. Protein arginine methyltransferase 5 (PRMT5) is a type II arginine methyltransferase that regulates transcription, RNA splicing, DNA damage responses, and immune signaling through symmetric dimethylation of histone and non-histone substrates. PRMT5 is frequently overexpressed across hematologic and solid tumors. Preclinical studies indicate that PRMT5 expression is elevated during HTLV-1-mediated T-cell transformation and that pharmacologic inhibition of PRMT5 selectively impairs the survival and transformation of infected T cells in vitro and in vivo. In this review, we highlight the current understanding of PRMT5 biology in cancer, summarize preclinical studies supporting PRMT5 as a therapeutic target in ATLL, and discuss key challenges to future clinical translation. We also discuss emerging approaches such as rational combination therapies and tumor-selective PRMT5 inhibitors as potential paths toward treatment for ATLL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b9bb0acc4e1b3cbc80b42591847784cba408015" target='_blank'>
              Targeting PRMT5 in Adult T-Cell Leukemia/Lymphoma: Opportunities and Challenges
              </a>
            </td>
          <td>
            Kyle Ernzen, A. Panfil
          </td>
          <td>2026-01-01</td>
          <td>Viruses</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="


 Pediatric acute leukemia is the most common childhood cancer, and acute myeloid leukemia (AML) represents the second most frequent subtype, with overall survival rates of approximately 70% and relapse rates near 30%. The purpose of this study is to define molecular mechanisms essential for leukemogenesis and uncover potential vulnerabilities in a high-risk subtype of pediatric AML driven by the fusion oncoprotein PICALM::MLLT10. Our laboratory recently performed an integrated transcriptomic and genomic analysis of 887 pediatric AML cases, identifying 23 distinct molecular categories that differ in HOXA/HOXB expression, co-occurring mutations, and clinical outcomes. Among these, PICALM::MLLT10 AML is notable for its HOXA expression pattern similar to KMT2A-rearranged cases, yet it forms a distinct cluster with other immature AMLs on a UMAP projection, suggesting a unique transcriptional program. Leveraging genome-wide CRISPR screening data from the Cancer Dependency Map (DepMap) across 64 AML cell lines, we identified the RNA helicase DDX19A, but not its paralog DDX19B, as a selective genetic dependency in PICALM::MLLT10 models (U937, P31/FUJ) with an average Chronos score of -1.753 compared to -0.278 in all other AML models. Using a dTAG-induced degradation system, acute DDX19A depletion significantly impaired proliferation in PICALM::MLLT10-expressing cells and led to increased apoptosis as measured by Annexin V staining, and overexpression of DDX19A rescued the proliferation defect induced by dTAG13-mediated degradation. Domain-specific mutagenesis revealed that the DDX19A helicase and RNA-binding domains are essential for proliferation, while the NUP214-binding domains are dispensable in the PICALM::MLLT10 context. Next, we sought to define the transcriptional consequences of DDX19A loss in PICALM::MLLT10 cells. RNA-seq was performed on U937 cells expressing a FKBP12F36V-DDX19A allele after treatment with either DMSO or dTAG-13 for 24 hours, and pathway enrichment analysis revealed upregulation of cytoplasmic translation in DDX19A depleted cells. To functionally assess this effect, we measured nascent protein synthesis using O-propargyl-puromycin (OPP) incorporation after 24 hours of either DMSO or dTAG-13 treatment and observed a global reduction of protein synthesis in DDX19A depleted cells. These data establish DDX19A as a selective vulnerability and uncover the role of DDX19A in regulating protein synthesis in PICALM::MLLT10 driven pediatric AML. This work highlights a novel mechanistic vulnerability that may be leveraged for the development of targeted therapeutic strategies for this high-risk pediatric AML subtype.



 Lauren J. Ezzell, Ryan Hiltenbrand, Elizabeth Arnold, Michael P. Walsh, Melvin E. Thomas III, Tamara Westover, Masayuki Umeda, Jing Ma, Shondra M. Pruett-Miller, Jeffery M. Klco. Uncovering novel molecular vulnerabilities in PICALM::MLLT10 driven pediatric acute myeloid leukemia [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr LT005.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f08fe0a493aa37ad7908724e233549343842d3e" target='_blank'>
              Abstract LT005: Uncovering novel molecular vulnerabilities in PICALM::MLLT10 driven pediatric acute myeloid leukemia
              </a>
            </td>
          <td>
            Lauren Ezzell, R. Hiltenbrand, Elizabeth D Arnold, Michael P Walsh, Melvin E Thomas, Tamara Westover, Masayuki Umeda, Jing Ma, S. Pruett-Miller, J. Klco
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Despite breakthroughs in molecularly targeted and immune therapies, the prognosis for patients with urinary bladder cancer (UBC) has remained unsatisfactory over the past few decades. Understanding the molecular underpinnings of UBC treatment refractoriness is crucial for identifying novel therapeutic targets and strategies. Cancer stemness plays a pivotal role in the oncogenesis and treatment resistance of UBC, while the underlying molecular regulatory mechanisms are poorly understood. We identified isoform 1 of ASPM (ASPM-i1) as the most upregulated stemness-associated factor in tumorigenic UBC cells, predominantly expressed by ALDH1+ stem-like cancer cells. Pairing genetic ASPM-i1 inhibition with standard chemotherapeutic agents used in the treatment of UBC, including cisplatin and gemcitabine, circumvents the treatment resistance of tumorigenic and stem-like UBC cells. Mechanistically, ASPM-i1 interacts with the Disheveled (DVL) and intracellular NOTCH proteins, thereby attenuating CUL3- or FBXW7-mediated ubiquitination and the subsequent proteasomal degradation. The regulatory module concomitantly enhances the activities and ligand responsiveness of the Wnt and Notch signaling pathways in UBC cells. As a result, ASPM-i1 inhibition sensitized tumorigenic UBC cells to chemotherapy in a NOTCH- and DVL-dependent manner. In human UBC tissues, ASPM-i1 shows substantial cell-to-cell heterogeneity and is upregulated in a subset (46.4%) of tumors, correlating with poor clinical prognosis. This study reveals a crucial co-regulatory module of Notch and Wnt signaling that mediates stemness and chemotherapy resistance in tumorigenic UBC cells; its inhibition provides a novel approach to enhance chemosensitivity and improve therapeutic outcome in human UBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/076780e38121b3f25a600b0e4d535709ef262847" target='_blank'>
              Chemotherapy resistance of urinary bladder cancer mediated by a Notch and Wnt co-regulatory module of stemness.
              </a>
            </td>
          <td>
            Lin-Hsin Cheng, Che-Wei Hsu, Pei-Ming Yang, Tai-Yan Liao, Fang-Chi Liu, Wan-Wen Chen, Chin-Chen Pan, Chung-Chi Hsu, Kuan-Chou Chen, K. K. Tsai
          </td>
          <td>2026-01-10</td>
          <td>Stem cells</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Traditionally, scientists tend to approach cancer research in a reductionistic way: aiming at uncovering underlying, separate components in malignant processes. And indeed, great progress has been made by reducing the development of a tumor to single, specific genes and mutations. For instance, familial adenomatous polyposis (FAP) could be reduced to a germline mutation in the Adenomatous Polyposis Coli (APC) gene. The escape of tumor cells from immune surveillance could be reduced to the tumor expression of immune checkpoints, resulting in new approaches in tumor therapy by applying immune checkpoint inhibitors. However, a germline mutation in APC is not 1:1 related to colorectal cancer (CRC), and only some patients respond to immune checkpoint inhibitors. The point here is that biological systems, also comprising cancer, have properties that cannot be reduced to single components. The cooperation of the single components results in new, emergent properties. The outcome of an interaction in a complex network, like the immune system, depends on the many cell types involved and the numerous molecules that interact and activate or inhibit pathways. The way the composing elements are organized is a causal factor in itself for any emergent property. The rise of genomic analysis at the end of the previous century, enabling us to sequence a full genome at the DNA and RNA levels, has initiated an awareness of the need for ‘systems biology’: to consider a full system and how it is organized, in all of its aspects, to understand biological pathways and their outcomes. In this review, we outline the prospects and limitations of systems biology in cancer research and propose a causal framework that integrates upward and downward causation and multiple realizability to understand the emergent properties of tumors that determine the dynamics of tumor development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bad26730d321f49013a748603a1b42d130f54226" target='_blank'>
              The Need for a Systems Biology Approach in Cancer Explained
              </a>
            </td>
          <td>
            Hehuan Zhu, Xi Zhang, E. Nazemalhosseini-Mojarad, J. Roelands, L. D. A. N. de Muynck, C. Ravensbergen, Rachel Hoorntje, I. Stouten, Marianne Hokland, A. L. Vahrmeijer, Rob A. E. M. Tollenaar, Edwin Koster, Peter J K Kuppen
          </td>
          <td>2025-12-22</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Metastasis is still the leading cause of cancer-related death. It happens when disseminated tumor cells (DTCs) successfully navigate a series of steps and adapt to the unique conditions of distant organs. In this review, key molecular and immune mechanisms that shape metastatic spread, long-term survival, and eventual outgrowth are examined, with a focus on how tumor-intrinsic programs interact with extracellular matrix (ECM) remodeling, angiogenesis, and immune regulation. Gene networks that sustain tumor-cell plasticity and invasion are described, including EMT-linked transcription factors such as SNAIL and TWIST, as well as broader transcriptional regulators like SP1. Also, how epigenetic mechanisms, such as EZH2 activity, DNA methylation, chromatin remodeling, and noncoding RNAs, lock in pro-metastatic states and support adaptation under therapeutic pressure. Finally, proteases and matrix-modifying enzymes that physically and biochemically reshape tissues, including MMPs, uPA, cathepsins, LOX/LOXL2, and heparinase, are discussed for their roles in releasing stored growth signals and building permissive niches that enable seeding and colonization. In parallel, immune-evasion strategies that protect circulating and newly seeded tumor cells are discussed, including platelet-mediated shielding, suppressive myeloid populations, checkpoint signaling, and stromal barriers that exclude effector lymphocytes. A major focus is metastatic dormancy, cellular, angiogenic, and immune-mediated, framed as a reversible survival state regulated by stress signaling, adhesion cues, metabolic rewiring, and niche constraints, and as a key determinant of late relapse. Tumor-specific metastatic programs across mesenchymal malignancies (osteosarcoma, chondrosarcoma, and liposarcoma) and selected high-burden cancers (melanoma, hepatocellular carcinoma, glioblastoma, and breast cancer) are highlighted, emphasizing shared principles and divergent organotropisms. Emerging therapeutic strategies that target both the “seed” and the “soil” are also discussed, including immunotherapy combinations, stromal/ECM normalization, chemokine-axis inhibition, epigenetic reprogramming, and liquid-biopsy-enabled minimal residual disease monitoring, to prevent reactivation and improve durable control of metastatic disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a5ad40c8765430fb4e35e8df4878d707b5a6318" target='_blank'>
              Molecular and Immune Mechanisms Governing Cancer Metastasis, Including Dormancy, Microenvironmental Niches, and Tumor-Specific Programs
              </a>
            </td>
          <td>
            Dae Joong Kim
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5df76d59f3a1b2aa9161ce2bf7becbe9cb09edf1" target='_blank'>
              Uncovering the role of LINE-1 in the evolution of lung adenocarcinoma.
              </a>
            </td>
          <td>
            Tongwu Zhang, Wei Zhao, Christopher Wirth, M. Díaz-Gay, Jinhu Yin, M. Cecati, F. Marchegiani, P. Hoang, Charles Leduc, M. Baine, William D. Travis, L. Sholl, P. Joubert, Jian Sang, John P McElderry, Michelle Antony, Alyssa Klein, Azhar Khandekar, Caleb Hartman, J. Rosenbaum, Frank J Colón-Matos, M. Miraftab, Monjoy Saha, Olivia W. Lee, Kristine M. Jones, N. E. Caporaso, M. Wong, Kin Chung Leung, C. A. Hsiung, Chih-Yi Chen, E. S. Edell, Jacobo Martínez Santamaría, M. Schabath, Sai S. Yendamuri, Marta Mańczuk, J. Lissowska, B. Świątkowska, A. Mukeria, O. Shangina, D. Zaridze, I. Holcatova, D. Mates, Sasa Milosavljevic, Milan Savic, Y. Bossé, B. G. Rothberg, David C. Christiani, V. Gaborieau, Paul Brennan, Geoffrey Liu, P. Hofman, R. Homer, S. Yang, A. Pesatori, D. Consonni, Lixing Yang, Bin Zhu, Jianxin Shi, Kevin M. Brown, Nathaniel Rothman, S. Chanock, Ludmil B. Alexandrov, Jiyeon Choi, Maurizio Cardelli, Qing Lan, Martin A. Nowak, D. Wedge, M. Landi
          </td>
          <td>2025-12-10</td>
          <td>Nature</td>
          <td>1</td>
          <td>109</td>
        </tr>

        <tr id="KRAS is one of the most frequently mutated genes in all human cancers, and its oncogenic mutation hotspots are glycine 12 (G12), glycine 13 (G13), glutamine 61 (Q61) and alanine 146 (A146). Among these hotspot mutations, A146 substitution mutations (A146X) occur relatively infrequently, except for haematopoietic and lymphoid cancers, suggesting that A146X causes intrinsically distinct KRAS signalling compared to other KRAS oncogenic alleles. However, due to the absence of model A146X cell lines derived from haematopoietic sources, the cellular mechanisms that cause the differences between KRAS.A146X and other common KRAS mutants, such as KRAS.G12X, remain largely unexplored. In this study, we developed a set of isogenic model haematopoietic cell lines expressing KRAS.A146T, KRAS.G12V and KRAS.G12G (non-mutated) from the endogenous locus by genetically modifying the human lymphoblastoid TK6 cell line. We found that TK6 cells carrying KRASA146T/+ or KRASG12V/+ exhibited increased replication stress compared to KRAS wild-type cells. Strikingly, KRASA146T/+ cells strongly rely on PrimPol for maintaining cellular survival upon replication stress. In contrast, KRASG12V/+ cells exhibited hypersensitivity to inhibitors for the ATR-Chk1 checkpoint signalling axis and to nucleoside analogues commonly used to treat cancers and viral infections. Our findings demonstrate that the endogenously expressed oncogenic KRAS mutations exacerbate the replication stress and reveal KRAS allele-specific replication phenotypes, facilitating the development of effective chemotherapies tailored to specific oncogenic KRAS mutation alleles and types of cancer. Moreover, our study offers valuable model cell lines for investigating mechanisms underlying replication vulnerability in cancers harbouring oncogenic KRAS mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5113da5b492954d75a6c3e0082f785f965a4200" target='_blank'>
              Generation of Human Haematopoietic Model Cell Lines Revealed Distinct Replication Stress Tolerance Between Two Oncogenic KRAS Mutations, G12V and A146T
              </a>
            </td>
          <td>
            Mone Okuda, Ryotaro Kawasumi, Kayoko Tanaka, Kouji Hirota
          </td>
          <td>2026-01-28</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97e8ef7d28e9d10b44890e1b7be7668548430edd" target='_blank'>
              CHEK2 loss endows chemotherapy resistance to hematopoietic stem cells.
              </a>
            </td>
          <td>
            Jing Zhou, Tianyuan Hu, Dian Li, Sanming Li, Minhua Li, Xiangguo Shi, Daisuke Nakada
          </td>
          <td>2026-01-12</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 Prostate cancer lineage plasticity, characterized by histologic transformation of prostate adenocarcinoma (PRAD) to neuroendocrine prostate cancer (NEPC), is an emerging mechanism of treatment resistance. NEPC accounts for up to 15% of treatment-resistant prostate cancers and is associated with poor prognosis, highlighting an unmet need for new therapies. NEPC lineage reprogramming is primarily driven by epigenetic dysregulation including differential activity of histone and DNA methyltransferases. EZH2, the catalytic component of the Polycomb repressive complex 2 (Polycomb), is overexpressed in most treatment-resistant prostate cancers and is implicated as a driver of disease progression. In this study, we define the differential, lineage-specific action of Polycomb in both PRAD and NEPC subtypes to better understand its role in modulating differentiation and lineage plasticity, and to identify novel targetable drivers of NEPC. Epigenetic H3K27me3 CUT&Tag profiling of treatment-resistant PRAD (n=9) and NEPC (n=9) rapid autopsy clinical samples revealed that Polycomb targets, including NE-lineage transcription factors, are de-repressed in NEPC. Mechanistically, Polycomb modulates H3K4/K27me3 bivalent promoters, leading to the upregulation of NEPC-associated transcriptional drivers (e.g., ASCL1) and neuronal gene programs which facilitate forward differentiation following EZH2 targeting in NEPC patient-derived organoid/xenograft models. Notably, we identified prospero-homeobox 1 (PROX1) as an understudied cell-fate determining transcription factor that is epigenetically de-repression by differential Polycomb activity. Integrative H3K27ac CUT&RUN and Capture Hi-C analyses revealed potential enhancers of PROX1 in NEPC that may be regulated by ASCL1. We sought to functionally characterize the role of PROX1 in NEPC. An unbiased CRISPR screen in two NEPC patient-derived organoid models demonstrated high cellular dependency for PROX1; knockout of PROX1 impeded tumor growth in NEPC models, while overexpression of PROX1 in PRAD promoted tumor growth and spontaneous metastases. Transcriptomic and cistromic analyses across models of CRPC and NEPC pointed to PROX1 regulation of neuroendocrine-lineage transcriptional programs. Immunoprecipitation followed by mass spectrometry identified three novel phosphorylated sites in the DNA-binding domain of PROX1 that are critical for its stability and function. In silico analyses of these phosphorylation sites predicted CHEK1 as a potential upstream kinase which could be exploited for therapeutic targeting of PROX1. Our findings provide insights into the potential role for bivalent promoters in Polycomb-mediated lineage reprogramming, which may facilitate forward differentiation in NEPC upon EZH2 inhibition. Further investigation of de-repressed candidates defines the role of PROX1 as a driver of NEPC and a potential therapeutic target.



 Varadha Balaji Venkadakrishnan, Nathaniel C. E. Voss, Nicole Traphagen, Richa Singh, James Neiswender, Keira Prenza. Sosa, Kenny Weng, Francisca Vazquez, David S. Rickman, Myles Brown, Himisha Beltran. Polycomb dysregulation shapes chromatin bivalency critical for prostate cancer lineage plasticity [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr PR022.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d16b90ea02984495a11ed149a37089c1519a17fc" target='_blank'>
              Abstract PR022: Polycomb dysregulation shapes chromatin bivalency critical for prostate cancer lineage plasticity
              </a>
            </td>
          <td>
            V. B. Venkadakrishnan, Nathaniel C E Voss, Nicole A. Traphagen, Richa Singh, James Neiswender, Keira Prenza Sosa, Kenny Weng, Francisca Vazquez, D. Rickman, Myles Brown, H. Beltran
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="
 Introduction.
 The development of therapy resistance and relapses of acute myeloid leukemia (AML), especially in the intermediate prognosis group, may be due to the molecular genetic heterogeneity of tumor cells. Chromosomal microarray analysis (CMA) can detect microdeletions, duplications, and copy-neutral loss of heterozygosity (cnLOH) which may be associated with a response to therapy.


 Aim:
 to evaluate tthe frequency of copy number aberrations and cnLOH in leukemogenesis-associated genes in patients with intermediate-stage AML and and their relationship to survival and response to treatment.


 Materials and methods
 . The study included 35 patients with de novo AML from the intermediate prognosis group for ELN2017. Copy number analysis by CMA was performed for a panel of 36 genes associated with leukemogenesis. The reference group included 102 healthy individuals without oncohematological disorders who also underwent comparable CMA testing.


 Results.
 Genomic aberrations were detected in 91.18 % of patients, most often in the genes of chromatin modifiers (64.7 % patients) and tumor suppressor genes (64.7% patients). The cnLOH type (PHF6, SMC1A, BKORL1) prevailed. KMT2A duplications occurred only in AML patients — 14.3 % (p < 0.001) and were associated with worse survival (log-rank P = 0.05). Combinations of genomic alterations involving 4–7 functional gene groups were found in 20.6% of patients.


 Conclusion.
 Driver gene aberrations, especially KMT2A duplications, are associated with an unfavorable clinical outcome in AML with an intermediate prognosis.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4336f89da62f27fbefbd6b50cf91ca51d0c03af7" target='_blank'>
              Structural aberrations of genes associated with leukemogenesis in patients with acute myeloid leukemia of intermediate prognosis
              </a>
            </td>
          <td>
            D. K. Bessmertnyy, S. Starchenko, N. Risinskaya, S. Kulikov, U. A. Chabaeva, V. Surimova, A. S. Ponamoreva, I. Kanivets, Z. T. Fidarova, I. Lukianova, A. Kashlakova, E. V. Romanyuk, N. I. Balaeva, V. Troitskaya, A. Sudarikov, E. Parovichnikova
          </td>
          <td>2025-12-21</td>
          <td>Russian journal of hematology and transfusiology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08f4bc063f465f57cf5929952bda615af3f67fa3" target='_blank'>
              KIF5B-driven unfolded protein response reprograms breast cancer immunosuppressive microenvironment for single-cell guided therapeutic targeting.
              </a>
            </td>
          <td>
            Yue Liu, Shuyu Li, Zhiyong Luo, Yun Xia, Panshi Zhang
          </td>
          <td>2026-01-26</td>
          <td>Discover oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cervical cancer remains a major global public health concern, with persistent infection by high-risk human papillomavirus (hrHPV) types recognized as the primary etiological factor. This review explores the multifactorial nature of the disease, focusing on the complex interplay between host genetic susceptibility and epigenetic alterations that drive cervical carcinogenesis. Evidence from genome-wide association studies (GWAS) is discussed, highlighting the contribution of specific genetic loci, predominantly within the HLA region, to susceptibility to HPV infection and disease progression. In parallel, the review examines the molecular mechanisms by which the viral oncoproteins E6 and E7 promote genetic instability and epigenetic reprogramming, including histone modifications and dysregulation of non-coding RNAs. Particular emphasis is placed on DNA methylation, affecting both the viral genome and host genes such as FAM19A4, CADM1, PAX1, and MAL, as a promising biomarker for triage and detection of high-grade intraepithelial lesions (CIN2+). Finally, the review evaluates currently available methylation-based assays and self-sampling devices, highlighting their potential to enhance diagnostic accuracy and increase participation in cervical cancer screening programs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91df97fd2742e76a0b3c768e22567f076cd3a39f" target='_blank'>
              HPV-Driven Cervical Carcinogenesis: Genetic and Epigenetic Mechanisms and Diagnostic Approaches
              </a>
            </td>
          <td>
            E. Legaki, Theofania Lappa, Konstantina-Lida Prasoula, Zoi Kardasi, E. Kalampokas, Theodoros Kalampokas, M. Roubelakis, Ekaterina Charvalos, M. Gazouli
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="OTUD3 (Ovarian Tumor Domain-Containing Protein 3), a deubiquitinating enzyme, has emerged as a pivotal and context-dependent regulator in cancer pathogenesis, exhibiting a striking functional duality as either a tumor suppressor or oncoprotein across different cancer types. Acting as a dual regulator, it influences key cellular processes—including proliferation, apoptosis, immune evasion, and metabolic reprogramming—by stabilizing specific substrates such as PTEN, p53, GRP78, YY1, and PD-L1. Its role extends to modulating inflammatory signaling, antiviral immunity, and metabolic homeostasis, highlighting its broad functional versatility. The development of OTUD3-targeted inhibitors like Rolapitant, Rupatadine, and OTUDin3 shows promise in preclinical models, particularly in combination with immunotherapy. However, its tissue-specific duality poses both challenges and opportunities for therapeutic intervention. Further research is needed to elucidate OTUD3’s mechanistic networks and advance its clinical translation as a prognostic biomarker and therapeutic target in precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fad3ab1cf92aac3d1c51f41c5a7f881d1bb5c1b3" target='_blank'>
              The dual role of OTUD3 in cancer: mechanisms and therapeutic implications
              </a>
            </td>
          <td>
            Jialin Zhang, Lu Lu, Wenchang Xu, Na Zhou, Yongfeng Song
          </td>
          <td>2025-12-23</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="A decade since immune checkpoint inhibitors made a stride in the clinical landscape of oncology, there has been a substantial focus on understanding the response heterogeneity following these therapies. Insights gained from clinical data identified the primary and secondary resistance mechanisms that escape the upfront therapy pressure. Beyond PD-1 and CTLA-4, new checkpoints averting this pressure are under clinical development. Adenosinergic pathways are actively engaged in oncogenic signaling. The main protagonists, CD73, A2AR, and A2BR, span diverse immune subsets of lymphoid and myeloid lineages and have emerged as alternative checkpoints. This review discusses the latest update on immune regulation dynamics of adenosine receptor signaling and their complex interplay with hypoxia in a heterogeneous tumor immune microenvironment (TIME). In this spectrum, we also review the plasticity of A2AR and A2BR in designing new drug candidates, tracing their complex metabolic roots in inducing immune dysfunction. Beyond the existing modalities, the ENT1 and MTAP-loss-MTA axis shows scope for alternative perturbations. The CD39-CD73-A2AR axis plays a central role in the terminal exhaustion of T cells. We highlight the interventions that disrupt the mechanistic context of A2AR and its cooperativity with other suppressors to restore anti-tumor immune functions following inhibition of their multilayered signaling. We capture the ongoing clinical trials and predictive biomarker landscape, along with novel delivery methods, to illustrate the evolving trends in this field. From these perspectives, we discuss how the adenosine axis can widen this new therapeutic avenue and boost the efficacy of CAR-T therapies. Therapeutic cancer vaccines are a new modality in this premise. Finally, an integrated overview of this pathway, along with TIME dynamics, illustrates the barriers and opportunities of combining adenosine signaling inhibitors in clinical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6937364b61b4a093c90b664d5b38994db470fe4" target='_blank'>
              Adenosine receptors on the immuno-oncology expressway: TIME, perspectives, and translation
              </a>
            </td>
          <td>
            Biswanath Majumder, Santanu Datta
          </td>
          <td>2026-01-08</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="

 Breast cancer is the most prevalent form of cancer among females and carries a substantial societal impact. DSN1, a component of the MIS12 complex, plays a critical role in centromere assembly, distribution, and stability. While DSN1’s role in tumors has been investigated, its specific function in breast cancer remains unclear.



 First, we utilized bioinformatics techniques to explore DSN1 expression in breast cancer and conducted functional enrichment and correlation analyses. Subsequently, we assessed the clinical relevance of DSN1 through immunohistochemistry. Furthermore, we examined how DSN1 affects the growth of breast cancer cells by conducting CCK8 and colony formation tests. Cell cycle and apoptosis changes were assessed using flow cytometry. Moreover, we examined key genes related to cell cycle and apoptosis to further elucidate the underlying mechanisms. Finally, we screened potential drugs targeting DSN1 by drug sensitivity and molecular docking analyses.



 Bioinformatics analysis revealed that DSN1 is highly expressed in breast cancer, making it a potential diagnostic marker. Functional enrichment analysis indicated that the DSN1- overexpressed group was enriched in cell proliferation-related pathways. Cellular experiments confirmed that DSN1 promotes breast cancer proliferation by affecting cell cycle pathways, involving key molecules such as CCNB1, CCND1, CKD1, CDK4, and CDK6. Drug sensitivity analysis showed that the DSN1 high expression group was resistant to drugs such as Epirubicin, Cyclophosphamide, Ribociclib, and Palbociclib, but relatively sensitive to tamoxifen and lapatinib.



 DSN1 contributes to breast cancer progression by modulating cell cycle pathways, making it a potential diagnostic and therapeutic target with clinical applicability.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3515d6bfa96b38426d6222c03e6a7fb688833cda" target='_blank'>
              DSN1 drives breast cancer progression via cell cycle regulation: diagnostic and therapeutic implications
              </a>
            </td>
          <td>
            Dongyang Liu, M. Luis, Djojomoenawi Sherilyn H. G., Xiaoxuan Zhu, Xiuqin Zhang, Yuan Fang, Xiaoyan Lin, Shasha Tang, Fengfeng Cai
          </td>
          <td>2026-01-20</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c691f3f7f0a81f36b7c4733212366543fe923c8f" target='_blank'>
              Deciphering chromosome fusion in D. miranda’s neo-sex chromosome through single-copy and repetitive oligo probes
              </a>
            </td>
          <td>
            Henry Bonilla Bruno, Isabela Almeida, Maria Dulcetti Vibranovski
          </td>
          <td>2026-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/745490de31a317b3b5c109c205a2bdf3115f7580" target='_blank'>
              Structural Maintenance of Chromosomes 5/6 complex dysfunction enables tumor mutagenesis
              </a>
            </td>
          <td>
            T. Tran, J. Fan, X. Zhao, A. M. Green
          </td>
          <td>2025-12-05</td>
          <td>medRxiv : the preprint server for health sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e961f589165d98aa8aff64f0539527c2012d419" target='_blank'>
              Pooled single-cell screen in colorectal cancer defines transcriptional modules linked to oncogenes.
              </a>
            </td>
          <td>
            Viola Hollek, Francisca Böhning, Catalina Florez Vargas, A. Sieber, M. Morkel, Nils Blüthgen
          </td>
          <td>2026-01-19</td>
          <td>Molecular systems biology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Oral squamous cell carcinoma (OSCC) remains a highly prevalent and aggressive malignancy with limited improvements in survival rates. One of the major obstacles to successful treatment is the development of chemoresistance, which contributes to recurrence, metastasis, and treatment failure. This narrative review aims to integrate current evidence on the molecular and cellular mechanisms that drive chemoresistance in OSCC and to delineate how these processes converge under therapeutic pressure. A structured search was performed to identify relevant studies addressing chemoresistance in OSCC, focusing on preclinical and translational evidence. Multiple interconnected mechanisms have been implicated in driving resistance in OSCC, including epigenetic alterations, deregulated signaling pathways, cancer stem cell plasticity, epithelial–mesenchymal transition (EMT), interactions with the tumor microenvironment (TME), drug efflux mediated by ATP-binding cassette (ABC) transporters, and enhanced DNA damage response. In combination, these mechanisms support tumor persistence and limit effective antitumor immunity. Emerging strategies such as epigenetic modulators, signaling pathway inhibitors, immunomodulation, and nanomedicine-based delivery systems have shown promising results in preclinical models. By highlighting convergent resistance networks, this integrative perspective supports the rational design of combination therapies and biomarker-guided strategies aimed at overcoming chemoresistance in OSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25af512c23cbe217030a9a034876fead83f7c218" target='_blank'>
              Molecular Mechanisms of Chemoresistance in Oral Squamous Cell Carcinoma: A Narrative Review with Present and Future Perspectives
              </a>
            </td>
          <td>
            E. Morais, L. Q. R. de Oliveira, Cintia Eliza Marques, H. Morais, Deborah Gondim Lambert Moreira, Lucas de Araújo Albuquerque, J. R. V. Silva, R. Freitas, Ricardo Della Coletta
          </td>
          <td>2026-01-05</td>
          <td>Applied Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22b544cab2c7f5ac6e8301325b2bf082f8d3ca21" target='_blank'>
              TRMT112 drives a tumor growth and metastasis-promoting program in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Amr R. Elhamamsy, Brandon J. Metge, M. H. Elbahoty, Bhavya Papineni, Heba M. Alsheikh, Dongquan Chen, Rajeev S. Samant, L. Shevde
          </td>
          <td>2026-01-08</td>
          <td>Cell death and differentiation</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3333bb3dc7aa46ec7dea00023d761c89be2478c2" target='_blank'>
              Large-scale single-cell analysis and in silico perturbation reveal dynamic evolution of HCC: from initiation to therapeutic targeting.
              </a>
            </td>
          <td>
            Peng Xia, Shuang Si, Daosen Fu, Luyao Liu, Dongliang Yang, Yanan Guo, Yixiao Tian, Pengfei Ji, Xinyi Yuan, Yingxia Tian, Rong Shen, Degui Wang
          </td>
          <td>2026-01-30</td>
          <td>NPJ precision oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Prostate cancer depends on androgen receptor (AR) signaling for growth, which is why androgen deprivation (castration) therapy is effective at early stages. However, many tumors eventually progress to a lethal form known as castration-resistant prostate cancer (CRPC). A subset of CRPC tumors bypass dependency on AR signaling by acquiring lineage plasticity, where prostate cancer cells transdifferentiate into alternate cellular states through epigenetic reprogramming. Neuroendocrine (NE) prostate cancer represents one well-known lineage plasticity phenotype. Nevertheless, most AR-independent tumors do not exhibit NE features and are defined as AR-negative/NE-negative or “double-negative prostate cancer” (DNPC). In a collaboration with Dr. Ekta Khurana’s computational genomics lab at Weill Cornell Medicine, we recently classified CRPC into four epigenetic subtypes, including the well-established 1) AR and 2) NE, as well as the novel DNPC subgroups 3) WNT and 4) stem cell-like (SCL) (PMID: 35617398). We focused on the SCL subtype as it is the second most common group in CRPC patients and lacks therapeutic targets. Using functional genomic approaches, we found that YAP/TAZ/TEAD cooperates with FOSL1 to drive the SCL lineage and growth of SCL models. We therefore hypothesize that the heightened dependency on the YAP/TAZ/TEAD/FOSL1 transcriptional program represents a therapeutic vulnerability in CRPC-SCL. To test this, we exposed CRPC models to TEAD inhibitors and found robust growth suppression in SCL cells compared to non-SCL cells in vitro. To evaluate whether the TEAD inhibitors are on-target, we performed transcriptomic profiling in SCL models and observed downregulation of YAP/TAZ gene signature as well as FOSL1 expression, which phenocopies the effects of YAP/TAZ double knockdown. To define the cistromes of these factors upon TEAD inhibition, we performed ChIP-seq and observed reduced co-occupancy at consensus sites, suggesting the disruption of the YAP/TAZ/TEAD/FOSL1 transcriptional circuit by the small molecule compound. To determine whether these phenotypes are recapitulated in vivo, we will treat mice harboring CRPC-SCL xenografts with TEAD inhibitors to assess growth response and evaluate epigenetic and transcriptional response using single-nucleus Multiome (ATAC+RNA). These studies will establish whether small molecule inhibition of TEAD is a promising strategy for the treatment of CRPC-SCL and allow high-resolution analysis of cell state transitions, with a focus on loss of SCL-specific signatures and potential emergence of AR/NE programs as adaptive resistance mechanisms.



 Chen Khuan Wong, Dan Li, Hongsu Wang, Marjorie Roskes, Weiling Li, Shipra Shukla, Dana Schoeps, Ekta Khurana, Yu Chen. Defining and targeting drivers of lineage plasticity in stem cell-like prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A074.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/912fdd784be0fd097e91b40d9e4c6933dd2c1b14" target='_blank'>
              Abstract A074: Defining and targeting drivers of lineage plasticity in stem cell-like prostate cancer
              </a>
            </td>
          <td>
            C. K. Wong, Dan Li, Hongsu Wang, M. Roskes, Weiling Li, Shipra Shukla, Dana M. Schoeps, E. Khurana, Yu Chen
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Somatic mosaicism is pervasively observed in human aging, with clonal expansions of cells harboring mutations in recurrently mutated driver genes. Bulk sequencing of tissues captures mutation frequencies, but cannot reconstruct clonal architectures nor delineate how driver mutations impact cellular phenotypes. We developed single-cell Genotype-to-Phenotype sequencing (scG2P) for high-throughput, highly-multiplexed, joint capture of genotyping of mutation hotspots and mRNA markers. We applied scG2P to aged esophagus samples from six individuals and observed large numbers of clones with a single driver event, accompanied by rare clones with two driver mutations. NOTCH1 mutants dominate the clonal landscape and are linked to stunted epithelial differentiation, while TP53 mutants promote clonal expansion through both differentiation biases and increased cell cycling. Thus, joint single-cell highly multiplexed capture of somatic mutations and mRNA transcripts enables high resolution reconstruction of clonal architecture and associated phenotypes in solid tissue somatic mosaicism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e453d2ebb8e890d0f12017626a25469ee8fcb34" target='_blank'>
              Genotype-to-phenotype mapping of somatic clonal mosaicism via single-cell co-capture of DNA mutations and mRNA transcripts.
              </a>
            </td>
          <td>
            Dennis J. Yuan, John Zinno, Theo Botella, Dalia Dhingra, Shu Wang, Allegra G Hawkins, Ariel D Swett, Jesus Sotelo, Ramya Raviram, Clayton Hughes, Catherine Potenski, Katharine D Godfrey, Kara M Ainsworth, Shuzhen Xu, Jianwen Que, Julian A Abrams, Akira Yokoyama, N. Kakiuchi, Seishi Ogawa, D. Landau
          </td>
          <td>2025-12-31</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18e77f40b4649c94b742354f1776a0273d7ae6d5" target='_blank'>
              A Hallmark-Integrated, Agent-Based Framework for Intratumor Heterogeneity in Melanoma Evolution
              </a>
            </td>
          <td>
            Khola Jamshad, Trachette L. Jackson
          </td>
          <td>2026-01-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9627eeea0e72cf88ceb4ce602d15c8920df0c909" target='_blank'>
              Splicing-driven post-translational dysregulation: a new frontier for precision cancer medicine and immunotherapy.
              </a>
            </td>
          <td>
            Sael Alatawi
          </td>
          <td>2026-01-24</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Leukemia is a heterogeneous group of hematologic malignancies characterized by the dysfunctional proliferation of white blood cells in the bone marrow. Genetic alterations are important risk factors for the development and progression of leukemia, and their detection is crucial. Although many genetic techniques, including karyotyping, fluorescence in situ hybridization (FISH), and polymerase chain reaction (PCR), have provided valuable information, they all have the limitation of incomplete genomic coverage. The evolution of genomic technologies, including Next Generation Sequencing (NGS) and Third Generation Sequencing (TGS), has enabled a more comprehensive and detailed characterization of the genetic landscape of leukemia. NGS technology has revolutionized leukemia diagnosis, treatment, and minimal residual disease monitoring, and its integration into routine leukemia care will enhance patient outcomes and pave the way for truly personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f067f109a8a6e2615bdfa692706a81a96d161ae" target='_blank'>
              The Application of Next-Generation Sequencing in Leukemia
              </a>
            </td>
          <td>
            Konstantinos Agiannitopoulos, E. Kouvidi
          </td>
          <td>2025-12-25</td>
          <td>OBM Genetics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 Androgen pathway inhibitors have transformed advanced prostate cancer (PCa) treatment yet remain susceptible to acquired resistance. Progression of PCa to androgen receptor (AR)-independent subtypes such as neuroendocrine prostate cancer (NEPC) confers poor prognosis. This lineage transformation is mediated by epigenetic mechanisms. NEPC frequently inactivates the tumor suppressor and cell cycle regulators RB1 and TP53, suggesting crosstalk between reprogramming factors and the cell cycle. While this bridge has been characterized in multiple cell types, it is understudied in NEPC. Salient examples include the pluripotency transcriptional factor (TF) SOX2, which regulates CCND1, and INSM1, which directly binds Cyclin D1 protein. Here, we investigate these interactions using isogenic inducible models and in-silico analysis of patient tumors, to identify therapeutic vulnerabilities exposed by neuroendocrine (NE) differentiation.



 Data were obtained from published datasets (Beltran et al, 2016; Labreque et al, 2019; DepMap) and the Caris CodeAI database. Patient and cell line data was analyzed for survival (where applicable) and differential mRNA expression between groups stratified by mRNA quartile. Inducible systems of TFs SOX2, ASCL1, FOXA2, and INSM1 were established in AR-positive LNCaP, and AR-negative DU145 and PC3 cell lines. RB1 and TP53 knockout (KO) LNCaP lines were generated by Cas9 electroporation, and Cyclin D1-HA (Addgene 174154) was stably overexpressed. TFs were induced with 200ng/mL doxycycline for 24h up to 28d, and protein expression was assessed via western blot. Cell viability and proliferation were surveyed by CellTiter-Glo® assay and nuclear counts, respectively.



 NEPC patient data analysis revealed INSM1, ASCL1, FOXA2, and SOX2 expression to be negatively correlated with Cyclin D1 and positively correlated with Cyclin E1/2. High SOX2, ASCL1, FOXA2, or INSM1 correlated with worse prognosis in NEPC, while high SOX2 or INSM1 correlated with improved survival in prostate adenocarcinoma (PRAD). Elevated Cyclin D1 predicted improved outcomes in NEPC. INSM1 correlated with RB1 mutation and expression of E2F family members, CDK inhibitors, and E-/A-type cyclins in NEPC. Inducible INSM1, SOX2, ASCL1, and FOXA2 models showed robust induction 48h post-doxycycline addition, accompanied by expression of NE lineage markers and diminution of proliferation in the PRAD model LNCaP. Isogenic RB1 and TP53 KO LNCaP models were generated and validated, along with a CCND1 overexpression model. Ongoing work examines whether these models respond differently to TF induction.



 Our analyses reveal significant correlations between expression of NE-lineage TFs and cell cycle-regulators in clinical datasets. Novel in vitro studies suggest these TFs suppress proliferation. Future work will uncover whether RB1/TP53 loss and Cyclin D1 expression affect tolerance of NE programs, and uncover vulnerabilities such as G2/M checkpoint reliance, CDK inhibitor upregulation, and metabolic alterations.



 Connor Purcell, Anais Sidonia, Yumiko Imai, Audrey Su, Tyler Roady, Praveen R. Srinivasan, Maximilian Pinho-Schwermann, Benedito A. Carneiro, Wafik S. El-Deiry. Interactions between prostate cancer lineage plasticity drivers and the RB1/E2F axis in AR-independent acquired resistance to AR-pathway inhibitors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr B062.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57c93c6275e3e0d94b8cea988ad7669996dfac91" target='_blank'>
              Abstract B062: Interactions between prostate cancer lineage plasticity drivers and the RB1/E2F axis in AR-independent acquired resistance to AR-pathway inhibitors
              </a>
            </td>
          <td>
            Connor Purcell, Anais Sidonia, Yumiko Imai, Audrey Su, Tyler J. Roady, Praveen R Srinivasan, Maximilian Pinho-Schwermann, B. Carneiro, W. El-Deiry
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="BACKGROUND
Approximately 10% of cancers achieve replicative immortality through a telomerase-independent mechanism of telomere maintenance, termed Alternative Lengthening of Telomeres (ALT). ALT is particularly prevalent in certain subtypes of malignant gliomas, such as IDH-mutant astrocytoma and pediatric glioblastoma, and frequently co-occurs with ATRX inactivating mutations. Although ALT is an adaptive mechanism through which cancer cells achieve proliferative immortality, the elevated levels of replication stress observed in ALT tumors constitute a potential therapeutic vulnerability.


METHODS
Leveraging CRISPR/Cas9 screening data from the Cancer Dependency Mapping Project, coupled with patient-derived cell lines and xenografts, we identified SMARCAL1 as a novel synthetic lethal vulnerability in ATRX-deficient glioma models that engage ALT. Using complementary molecular assays for DNA damage, telomere maintenance, and telomeric replication stress, we define the mechanisms underlying cytotoxicity induced by SMARCAL1 depletion in ALT-positive glioma cells.


RESULTS
Our data demonstrate the annealing helicase SMARCAL1 is a highly specific synthetical lethal vulnerability in cancers that use ALT. SMARCAL1 localizes to ALT-associated PML bodies in ALT-positive glioma cell lines, including IDH-mutant astrocytomas. SMARCAL1 depletion, via doxycycline-induced RNAi, led to a hyperactivation of the ALT phenotype, high levels of DNA double-strand breaks in G2 phase, and cell death via mitotic catastrophe. In mice bearing intracranial xenografts derived from high-grade IDH-mutant astrocytoma, inducible SMARCAL1 depletion prolonged animal survival.


CONCLUSIONS
Our findings demonstrate that the molecular processes orchestrating ALT-mediated telomere maintenance constitute a targetable synthetic lethal vulnerability that can be exploited by SMARCAL1 inhibition, thus supporting the future development of small molecule inhibitors of SMARCAL1 as anti-cancer therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c97a1417417751706742470fa54409032d4aaa86" target='_blank'>
              SMARCAL1 is a targetable synthetic lethal therapeutic vulnerability in ATRX-deficient gliomas that use Alternative Lengthening of Telomeres.
              </a>
            </td>
          <td>
            Alexandrea Brown, Laura Strickland, Elise N Erman, Christopher J. Pirozzi, Justin T. Low, B. Diplas, Emiley A Gibson, Mariah Shobande, Taher Khambati, Marharyta Krylova, Heng Liu, R. McLendon, Zachary J. Reitman, S. Keir, Lee Zou, David M. Ashley, Matthew S. Waitkus
          </td>
          <td>2026-01-10</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>111</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6103c1444bcfcdc5f97533157cff885266554af" target='_blank'>
              Single-cell transcriptomics uncover RNF130-mediated TNF-α pathway activation and worenine synergy with paclitaxel in breast cancer.
              </a>
            </td>
          <td>
            Mi Hu, Liangbin Huang, Hongzhuan Deng, Zhifeng Chen, Guanghui Cheng, Xinchun Liu
          </td>
          <td>2026-01-24</td>
          <td>Clinical epigenetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Aggressive variant prostate cancers (AVPC) are a poorly understood subset of prostate cancers characterized by high mortality and extensive molecular heterogeneity. To address a lack of AVPC representative models for investigating underlying molecular and physiological mechanisms, we established NCI-LYM-1, a novel AVPC organoid/patient derived xenograft (PDX) derived from a patient with metastatic prostate cancer with small cell morphology displaying a unique clinical history and defined genomic features. It was hypothesized that this model would be able to accurately characterize biological underpinnings of the patient’s tumor profile and allow for comparative analysis to similar prostate cancers.



 NCI-LYM-1 was developed from a patient-derived AR-negative, ASCL1/Synaptophysin (SYP)-positive lymph node metastasis with small cell morphology. Short-read whole-genome sequencing (WGS) was used to identify genomic alterations and long-read WGS was used to identify epigenetic modifications. RNA-seq was used to characterize gene expression profiles. The resulting genomic, epigenetic, and transcriptomic data from NCI-LYM-1 was compared to molecular profiles from the LuCaP series of PDX models. In-vitro drug sensitivity assays were also conducted using organoid cultures.



 NCI-LYM-1 harbored biallelic inactivation of known neuroendocrine prostate cancer (NEPC) drivers: PTEN, TP53, RB1 and BRCA2, in concordance with patient diagnostic history. In addition, epigenetic evidence supported a consistent phenotype between NCI-LYM-1 and the donor tumor, including greater gene expression of neuroendocrine marker ASCL1 with decreased AR expression. Comparisons between NCI-LYM-1 and LuCaP models further supported transcriptional similarities between the model and a subset of AVPC cancer subtypes. Drug sensitivity assays were also consistent with the patient’s prior treatment exposure, displaying PARP inhibitor sensitivity but a lack of sensitivity to platinum treatments.



 NCI-LYM-1 faithfully recapitulates the phenotypic and genomic characteristics of aggressive metastatic AVPCs, including the display of neuroendocrine features. NCI-LYM-1 will be a useful model system to study molecular frameworks of understudied cancers like those of AVPC, including aspects such as drug sensitivity, disease progression, and lineage plasticity.



 Ana S. Jaramillo, Chennan Li, JuanJuan Yin, Melissa L. Abel, Dana S. Vargas, Kinjal Bhadresha, Sumeyra Kartal, Joseph Twohig, Tri M. Truong, Anish Thomas, Cindy H. Chau, Kathleen Kelly, William D. Figg, Adam G. Sowalsky. NCI-LYM-1 molecularly and phenotypically recapitulates aggressive variant prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A025.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/483313e79c5e529d10092ffd1001e71580f4403a" target='_blank'>
              Abstract A025: NCI-LYM-1 molecularly and phenotypically recapitulates aggressive variant prostate cancer
              </a>
            </td>
          <td>
            Ana S. Jaramillo, Chennan Li, JuanJuan Yin, Melissa L Abel, Dana S. Vargas, Kinjal Bhadresha, Sumeyra Kartal, Joseph Twohig, Tri M Truong, Anish Thomas, C. Chau, Kathleen Kelly, W. Figg, A. Sowalsky
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Ataxia telangiectasia and Rad3-related (ATR) kinase is a crucial regulator of the DNA damage response, supporting replication fork stability, enforcing cell-cycle checkpoints, and coordinating repair mechanisms. Tumor cells, which often experience oncogene-induced replication stress, rely more heavily on ATR signaling, presenting a potential therapeutic target for anticancer drug discovery and development. Over the last decade, intensive medicinal chemistry efforts have generated a broad pipeline of ATR inhibitors, including ceralasertib, elimusertib, camonsertib, berzosertib, ART0380, and gartisertib, many of which are in Phase I/II clinical trials. These compounds effectively disrupt checkpoints, induce replication catastrophe, and work synergistically with PARP inhibitors, topoisomerase poisons, platinum-based chemotherapies, radiotherapy, and immunotherapy. Although promising, challenges such as hematologic toxicities and resistance mechanisms persist. Future research aims to improve patient selection through biomarkers like replication-stress signatures, RAD51 foci tests, and liquid biopsy DNA damage markers; develop advanced modalities including brain-penetrant scaffolds, PROTAC degraders, and dual ATR/PARP or ATR/HDAC inhibitors; and optimize intermittent dosing to expand therapeutic windows. Incorporating these strategies into adaptive platform trials with pharmacodynamic markers and patient-centered outcomes will speed up translation. Overall, ATR inhibitors highlight progress in DNA damage response therapies, from understanding mechanisms to biomarker-driven clinical use, with the potential to revolutionize treatment across various cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/353226c771fd154371deec186a84ef0cd1fb5b04" target='_blank'>
              Recent advances in small molecule ATR kinase inhibitors as anticancer agents.
              </a>
            </td>
          <td>
            Gurpreet Singh, Ram Sharma, Vinod Gautam, A. Dhingra, Sung-Po Hsu, Kunal Nepali
          </td>
          <td>2026-01-28</td>
          <td>Future medicinal chemistry</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Conventional cancer therapies, including radiation therapy and chemotherapy, rely on inflicting DNA damage, yet they inevitably affect normal cells, leading to severe adverse effects. The advent of precision chemotherapy exploiting tumor-specific DNA repair defects has validated the effectiveness of this approach. The first successful example is PARP inhibitors, which selectively kill homologous recombination (HR) defective cancers, such as familial breast cancer possessing HR deficiency due to BRCA gene mutations. However, the broader landscape of DNA maintenance-including DNA replication, repair, and checkpoint pathways-harbors numerous mutations in tumors that remain untargeted. Here, we propose repurposing chain-terminating nucleoside analogs (CTNAs) to target such cancers' vulnerabilities. CTNAs, long utilized as anti-cancers and anti-viral drugs, inhibit replication and thereby suppress growth, but their activity has never been systematically aligned with specific cancer mutations associated with DNA maintenance defects. Based on our recent studies, we demonstrate that CTNAs elicit synthetic lethality in cells deficient for distinct DNA maintenance systems, amplifying replication stress, leading to cell death. We highlight the spectrum of CTNA-induced lesions and repair pathways required for cellular tolerance. This framework presents a versatile "repair-defect-guided" chemotherapy that expands the clinical utility of CTNAs and improves therapeutic effect by reducing side effects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e63be07db0a569b7816efc4ad2f977abe094978" target='_blank'>
              Targeting Genome Maintenance Defects of Cancers Using Chain-Terminating Nucleoside Analogs.
              </a>
            </td>
          <td>
            Ryotaro Kawasumi, Rubaiat E Tabassum, Kouji Hirota
          </td>
          <td>2025-12-10</td>
          <td>Cancer science</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="


 Acquired resistance to androgen receptor (AR)-targeted therapies and the progression to metastatic-castrate resistant prostate cancer (CRPC) remains a significant clinical problem. Mechanisms of acquired resistance include lineage plasticity, by which CRPC tumors can become AR-negative and progress to neuroendocrine prostate cancer (NEPC). Clinical prognoses for patients with NEPC are poor and more work is needed to understand the underlying molecular mechanisms driving lineage plasticity and CRPC to NEPC progression. FOXA2, a key lineage-determining pioneer transcription factor (TF), is significantly upregulated in subsets of CRPC and the majority of NEPC patient tumors. However, little is known about the role of FOXA2 in regulating lineage plasticity and progression from CRPC to NEPC.



 We queried single-cell RNAseq data from CRPC patient tumors and performed RNAseq and immunohistochemical assessment in patient tumors from 2 independent cohort of patient tumors. In CRPC and NEPC patient-derived organoids, we assessed transcriptional, chromatin accessibility changes and TF/pioneer factor and SWI-SNF chromatin occupancy changes. We performed unbiased approach of rapid immunoprecipitation mass spectrometry of endogenous protein for FOXA2 in adenocarcinoma and NEPC models to identify chromatin-bound, FOXA2-interacting proteins. We validated FOXA2-co-factor protein-protein interacting using co-immunoprecipitation. Lastly, we generated knock-in degron alleles for FOXA2 in NEPC patient-derived organoids to understand how FOXA2 is required for NEPC maintenance.



 In the CRPC tumors, high FOXA2 expression was correlated with high ONECUT2 and SOX2 expression, high NEPC score, low AR signaling score, and poorer patient prognosis. We also found that FOXA2 overexpression suppressed androgen signaling and reprograms the chromatin accessibility landscape to close AR binding sites and open chromatin related to neural reprogramming. Additionally, we found that FOXA2 interacts with the SWI-SNF complex, AR, and HOXB13 to redirect AR and HOXB13 DNA binding away from canonical AR-regulated sites. We found that FOXA2 is bound at regulatory regions of known NEPC driver genes and epigenetic modifiers. Lastly, we discovered that FOXA2 physically interacts with key NEPC TFs and epigenetic regulators, suggesting that these FOXA2 physical interactions are required for NEPC progression. Loss of FOXA2 was sufficient to suppress key NEPC genes.



 Overall, our data from murine and human models indicate that FOXA2 functions to suppress androgen signaling by redirecting AR and AR cofactor binding. Our data also shows that FOXA2 functions as a pioneer TF to redirect chromatin accessibility and, through its interaction with key NEPC-associated TFs and a chromatin-modifying protein complex, drives the NEPC-associated molecular program. These findings provide novel mechanistic insights underlying how FOXA2 regulates lineage plasticity and NEPC progression.



 Richard Garner, Nicholas Brady, Xuanrong Chen, Alessandra Ferri, Richa Singh, Kate Dunmore, Filippo Pederzolli, Hubert Pakula, Jagpreet Singh Nanda, Michael Freeman, Michael Haffner, Colm Morrissey, Brian Robinson, Antonio Marzio, Massimo Loda, Christopher Barbieri, David S. Rickman. FOXA2 reprograms AR signaling to promote lineage plasticity in advanced prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr PR033.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48d4e3ecda41ade1554b8e57c8aef3f6245ecae6" target='_blank'>
              Abstract PR033: FOXA2 reprograms AR signaling to promote lineage plasticity in advanced prostate cancer
              </a>
            </td>
          <td>
            Richard Garner, Nicholas J. Brady, Xuanrong Chen, Alessandra Ferri, Richa Singh, Kate Dunmore, Filippo Pederzolli, H. Pakula, Jagpreet S Nanda, Michael R. Freeman, M. Haffner, C. Morrissey, Brian D. Robinson, A. Marzio, Massimo Loda, C. Barbieri, D. Rickman
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Chronic myeloid leukemia is a hematopoietic stem cell malignancy characterized by excessive proliferation of white blood cells, primarily driven by the Philadelphia chromosome (t 9;22). Despite the availability of tyrosine kinase inhibitors, disease progression and therapy resistance remain major challenges, often linked to additional somatic mutations. Objectives: To find out the mutational status and expression variations of the ASXL1 gene hotspot region in CML patients treated by TKIs. Methods: This retrospective cross-sectional study was conducted at the Department of Zoology, Division of Science and Technology, University of Education, Lahore, Pakistan. DNA and mRNA of 50 CML patients were analyzed alongside 10 healthy controls and statistically assessed using SPSS. Results: Results showed that sequence analysis of exon 13 identified hotspot mutations, including two novel missense alterations, G659S and D667E, within the coding region of ASXL1. In-silico analysis also suggested potential structural and functional relevance of these variants. The ASXL1 gene expression showed a progressive but statistically non-significant down-regulation in CML patients across disease phases (p=0.662), disease duration (p=0.42), treatment with second-line versus first-line TKIs (p=0.412), and wild-type and mutant (p=0.544). Conclusions:  It is concluded that two novel missense alterations, G6559S and D667E, were identified in the hotspot region, and ASXL1 gene expression was downregulated non-significantly in CML patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14ee16b9a16245af7fb856280afba55f65850e27" target='_blank'>
              Identification of Novel Alterations in the ASXL1 Gene and Its Expression Profile in CML Patients
              </a>
            </td>
          <td>
            R. Mehmood, Khuram Shehzad Khan, A. Akram, Sikandar Hayat, Amjad Zafar
          </td>
          <td>2025-12-31</td>
          <td>Pakistan BioMedical Journal</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Acute lymphoblastic leukemia (ALL) driven by KMT2A rearrangements (KMT2A-r) is an aggressive hematologic malignancy with poor prognosis and a high incidence in infants. While KMT2A fusion proteins drive leukemogenesis through transcriptional dysregulation, recent discoveries have highlighted the pivotal role of non-coding RNAs (ncRNAs) in shaping the molecular and epigenetic landscape of this disease. These key regulators of gene expression influence chromatin dynamics, transcriptional activation, and post-transcriptional control. Circular RNAs (circRNAs) contribute to genome instability and facilitate chromosomal translocations, while some fusion-derived circRNAs (f-circRNAs) sustain oncogenic signaling and promote chemoresistance. Long non-coding RNAs (lncRNAs) orchestrate transcriptional programs that maintain leukemic stem cell properties and reinforce aberrant self-renewal pathways. MicroRNAs (miRNAs) modulate critical oncogenic networks by regulating KMT2A fusion transcripts and downstream effectors, thereby impacting drug resistance, apoptosis, and proliferation. Meanwhile, enhancer RNAs (eRNAs) fine-tune transcriptional activity and epigenetic regulation, influencing KMT2A target gene expression and chromatin accessibility. Collectively, these ncRNAs integrate into the complex regulatory circuits of KMT2A-r ALL, revealing their potential as biomarkers for disease classification, risk stratification, and treatment response prediction. Understanding their interplay with KMT2A fusion proteins not only provides new insights into leukemogenesis but also highlights promising opportunities for therapeutic intervention and precision medicine in this high-risk leukemia subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19b46cc5750ec443e7b12863d8785ed1d738e7da" target='_blank'>
              The hidden regulators: Non-coding RNAs in KMT2A-rearranged acute lymphoblastic leukemia.
              </a>
            </td>
          <td>
            Maria Augusta Poersch, Ana Carolina Rodrigues, Priscila Elias Ferreira Stricker, Alexandre Luiz Korte Azevedo, D. Bruschi, J. D. de Oliveira
          </td>
          <td>2025-12-08</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84024b37acab9255360ec615cb67be5b4af0c139" target='_blank'>
              A T-cell receptor targeting RUNX1 frameshift mutations in acute myeloid leukemia.
              </a>
            </td>
          <td>
            Nadine E. Struckman, Georgia Koutsoumpli, Rob C M de Jong, Dyantha I van der Lee, D. Remst, Sophie-Anne I Smith, M. Honders, R. Hagedoorn, A. D. de Ru, Masashi Matsuda, Fumihiko Ishikawa, Tamar Tak, Peter J. Valk, M. H. Heemskerk, P. V. van Veelen, J. H. Falkenburg, M. Griffioen
          </td>
          <td>2025-12-12</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eca909ba170b3407f8c473dadaa8f67180cd2619" target='_blank'>
              The Evolution of Spatial Omics Technologies Introduces A Novel Avenue for Lung Cancer Research.
              </a>
            </td>
          <td>
            Yue He, Zifan Li, Wenxiang Wang, Xu Liu, Shanshan Lu, Jing Bai, Lin Weng, Qingna Zhang, Jun Wang, Kezhong Chen
          </td>
          <td>2026-01-30</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4b00e19dbeba7049a53aa78186bf5b19a36b34d" target='_blank'>
              A genome-wide CRISPR/Cas9 screen reveals novel positive regulators of FTY720 sensitivity in acute lymphoblastic leukemia cells.
              </a>
            </td>
          <td>
            Jaira Ferreira de Vasconcellos, Leah Friedman, Isha Satapathy, Nicole Cubbage, Jasmin Palmer, Saurav Majumder, Mari Kono
          </td>
          <td>2026-01-23</td>
          <td>BMC research notes</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Enhancers are distal cis-regulatory elements that orchestrate spatiotemporal gene expression patterns in response to developmental cues and environmental stimuli. Genetic and epigenetic alterations in enhancers are associated with the initiation and progression of human diseases, including cancers. Over the past few decades, accumulating evidence has revealed that a class of nascent RNA transcripts, known as enhancer RNAs (eRNAs), is broadly transcribed from active enhancers. These eRNA species contribute to complex and dynamic gene regulatory networks under both physiological and pathological conditions through diverse mechanisms. Notably, dysregulated eRNA expression has been reported across various cancer types and is often correlated with patient survival outcomes. Consequently, eRNAs are emerging as promising biomarkers and therapeutic targets for cancer treatment. This review provides a comprehensive summary of the current understanding of eRNAs and their mechanisms of action in gene regulation. We discuss the critical roles of eRNAs in both health and disease and highlight their diagnostic and prognostic value, as well as their therapeutic potential in cancer. Additionally, we review current strategies for targeting RNA transcripts, including eRNAs, and discuss the major challenges in developing eRNA-targeted therapies. Finally, we propose future directions for advancing eRNA-based interventions in the treatment of human diseases, including cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9397eec4991d568fddc2093214f80298c06bfee4" target='_blank'>
              Deciphering the regulatory landscape of enhancer RNAs in health and disease
              </a>
            </td>
          <td>
            Qian Wang, P. ten Dijke, Chuannan Fan
          </td>
          <td>2026-01-29</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="S i c k l e C e l l D i s e a s e ( S C D ) i s a n i n h e r i t e d hemoglobinopathy caused by a point mutation (GAG→GTG) at the 6th codon of the β-globin gene (HBB). Among traditional treatments, hydroxyurea requires lifelong administration and some patients show poor response; although hematopoietic stem cell transplantation (HSCT) can cure the disease, only 18% of patients can find HLA-matched donors, and there is also the risk of graft-versus-host disease (GVHD). The CRISPR-Cas9 gene editing technology provides a potential curative option for SCD by precisely modifying hematopoietic stem/progenitor cells (HSPCs). This article systematically reviews the clinical application strategies, experimental evidence, and existing challenges of this technology, and discusses its translational prospects in combination with the latest research findings. It covers the clinical effects of strategies such as targeting the BCL11A gene enhancer to activate fetal hemoglobin (HbF), directly correcting the E6V mutation of the HBB gene via homology-directed repair (HDR), and mimicking hereditary persistence of fetal hemoglobin (HPFH), as well as safety and accessibility challenges (e.g., off-target effects, preconditioning toxicity, and high treatment costs) and their corresponding solutions, aiming to provide comprehensive references for the research and application of gene therapy in SCD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/890a8f128bb576259bd63b951a1f15ea5bf4e0b1" target='_blank'>
              Application and Breakthrough of CRISPR-Cas9 Gene Editing Technology in the Clinical Treatment of Sickle Cell Disease
              </a>
            </td>
          <td>
            Boxun Sun
          </td>
          <td>2025-12-19</td>
          <td>MedScien</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="BACKGROUND
Neuroblastoma is the most common extracranial solid tumor in children and exhibits substantial clinical heterogeneity. Although key genetic alterations such as N-Myc proto-oncogene protein amplification and anaplastic lymphoma kinase mutations are known drivers of neuroblastoma, their clinical utility is limited by population-specific genetic diversity. To address this, we propose a "function-over-gene" strategy by evaluating extracellular matrix (ECM)-related gene sets as molecular biomarkers, aiming to overcome the limitations posed by genetic heterogeneity and provide novel prognostic insights.


METHODS
We integrated data from a single-center cohort and the TARGET database. Cox regression models were used to assess the prognostic value of ECM gene alterations and their association with clinical outcomes. Gene set enrichment analysis (GSEA) was employed to identify ECM-related gene sets, followed by transcriptomic analysis to explore downstream regulatory pathways.


RESULTS
In the single-center cohort, ECM gene mutations were potentially associated with bone and lymph node metastases and emerged as an independent predictor of poor prognosis (HR=2.7, P=0.02) in multivariate analysis. Validation using the TARGET cohort confirmed the prognostic relevance of the ECM gene set (HR=1.55, P=0.0083) and revealed its involvement in modulating the tumor microenvironment via immune and complement pathways.


CONCLUSION
ECM gene signatures serve as robust prognostic markers across populations. This function-based approach offers a novel perspective to address genetic heterogeneity and provides a theoretical foundation for ECM-targeted combination therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d011b508eee694993a004e089261b1d23283417" target='_blank'>
              Dysregulation of Extracellular Matrix Genes Identifies Neuroblastoma Patients at High Risk of Recurrence and Poor Outcome.
              </a>
            </td>
          <td>
            Xiaoyu Jing, Shijing Ge, Guoqian He, Sijia He, Ju Gao, Chaoban Wang, Xia Guo
          </td>
          <td>2026-01-15</td>
          <td>Journal of pediatric hematology/oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Recessively inherited loss-of-function mutations in Excision Repair Cross-Complementing 6 like 2 (ERCC6L2) cause a bone marrow failure (BMF) syndrome characterized by moderate cytopenias, frequent somatic TP53 mutations, and a propensity to develop myeloid malignancies. The pathophysiology and molecular mechanisms underlying the BMF syndrome as well as its association with TP53-mutant clonal hematopoiesis (CH) and myeloid malignancies have remained poorly understood. Using novel preclinical in vitro and in vivo model systems, we demonstrate that Ercc6l2 maintains the competitive ﬁtness of hematopoietic stem and progenitor cells (HSPCs) by mitigating replication stress. Sustained replication stress and DNA damage in Ercc6l2-deficient HSPCs cause p53 pathway activation followed by cell cycle arrest and apoptosis. Moreover, Ercc6l2 deficiency results in decreased expression of master hematopoietic regulators Runx1 and Gata1 in HSPCs. Altogether, loss of Ercc6l2 leads to reduced HSPC numbers, bone marrow hypocellularity, and cytopenias. Notably, somatic Trp53 mutations restore cellular fitness of Ercc6l2-deficient HSPCs by abrogating p53 pathway activation and restoring Runx1 and Gata1 expression, thereby correcting the BMF phenotype. However, p53 loss fails to normalize replication stress, allowing for the accumulation of DNA damage over time which increases the likelihood for leukemic transformation. Our data uncover the pathogenesis of ERCC6L2 disease and provide a prototypic example of clonal compensation in BMF syndromes, where somatic mutations in leukemia-associated genes - in this case TP53 - transiently improve blood cell production at, however, the expense of increasing leukemogenic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e59817653a3cd48c2acdccbd610db0ade20bab3" target='_blank'>
              Bone marrow failure, somatic rescue by p53 inactivation, and enhanced leukemogenesis in germline ERCC6L2 disease.
              </a>
            </td>
          <td>
            R. Schimmer, Nancy Klemm, Jonas Fullin, Ebru Topçu, Milena Treacy, Karolina Zielinska, Cyril Doerdelmann, Daphne Devesa-Serrano, Melissa Lock, Francisco Caiado, Christian Koch, Nadjia Dietliker, Rahel Schwotzer, Marco Matteo Buehler, Mikko Myllymäki, Kari J. Kurppa, M. Manz, Massimo Lopes, Steffen Boettcher
          </td>
          <td>2026-01-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f9d687935c22333fba1c02521a2fc46ab87e52a" target='_blank'>
              Single-cell profiling of immunogenic cell death in melanoma reveals prognostic signatures and therapeutic targets.
              </a>
            </td>
          <td>
            Jingwen Huang, Taoxuan Jiang, Yiyang Han, Chunsheng Yang
          </td>
          <td>2026-01-16</td>
          <td>Discover oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cure rates for childhood malignancies using established therapy protocols have increased to an average of 80% but have reached a plateau. Moreover, survival rates are particularly low for some pediatric tumors-such as high-risk group 3 medulloblastomas, osteosarcomas, Ewing sarcomas, high-risk neuroblastomas, and high-grade gliomas-and dismal for patients with relapsed malignancies. A functional drug response profiling platform for pediatric solid and brain tumors has been established within the INFORM program to identify patient-specific vulnerabilities and biomarkers and to unravel molecular mechanisms associated with drug response profiles for clinical translation. In this study, we performed a multiomics analysis using drug sensitivity profiles, as well as genomic and transcriptomic data, of 81 pediatric solid tumor samples. The integrative analysis suggested two multiomics signatures associated with drug sensitivity. One signature distinguished neuroblastoma samples with sensitivity to navitoclax, a BCL2 family inhibitor. A second signature was specific to a subset of Wilms tumors harboring the SIX1 (Q177R) hotspot mutation that displayed high expression of MGAM, PTPN14, STAT4, and KDM2B and high sensitivity to MEK inhibitors. A patient-specific causal interaction network analysis suggested possible molecular interactions between MEK inhibitors and the SIX1 mutation in Wilms tumor samples. In conclusion, the integration of drug sensitivity profiling and multiomics data revealed potential biomarkers that may be associated with drug sensitivity in pediatric solid tumors. Patient-specific causal interaction network analysis further elucidated the interaction between inhibitors and signature biomarkers, providing insights that may inform clinical translation.


SIGNIFICANCE
The combination of multiomics analysis and drug sensitivity profiling identified two signatures related to drug sensitivity in pediatric solid tumors, contributing to the advancement of functional precision medicine and personalized treatment strategies. This article is part of a special series: Driving Cancer Discoveries with Computational Research, Data Science, and Machine Learning/AI .">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea339e646dcb66c61feed8e7a43ccdda96232499" target='_blank'>
              Integrative Multiomics and Drug Sensitivity Profiling Reveal Potential Biomarkers and Therapeutic Strategies in Pediatric Solid Tumors.
              </a>
            </td>
          <td>
            Dina ElHarouni, Rosa Hernansaiz-Ballesteros, H. Peterziel, G. Balasubramanian, C. Previti, Kathrin Schramm, Mirjam Blattner-Johnson, Rolf Kabbe, Barbara C Jones, S. Oppermann, David T. W. Jones, S. Pfister, Olaf Witt, Julio Saez‐Rodriguez, Ina Oehme, N. Jäger, Matthias Schlesner
          </td>
          <td>2025-12-19</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Most deaths from cancer are caused by their metastases. Along with the ability to evade the immune system and to invade and disrupt very different tissue environments, metastases usually develop resistance to whatever therapeutic approaches are tried. Their extreme adaptability requires a diversity of traits from which natural selection can choose, but current models, such as the epithelial-to-mesenchymal transition (EMT), do not directly address how this diversity is generated. We observed that single cell clones of Panc1 human pancreas cancer cells that had had crispR knock outs (KO) of the gene for activin-like kinase 4 (ALK4) had developed a markedly diverse morphology. Time-lapse and fluorescence microscopy, FACS analysis and mitotic chromosome squashes provided evidence that the cells diversify profoundly in size, behavior and chromosome number. This diversity appears to develop through cytokinesis failure, the generation of very large multinuclear cells that exhibit a high range of nuclear configurations, coupled with continued cell divisions that sometimes generate three, four, and more daughter cells in one cell division. This astounding activity often resulted in cells that were much smaller than the normal Panc1 cells, and indirect evidence suggests they were significantly sub-ploidy, yet underwent regular cell divisions. A subset of the smaller cells were highly motile, and were observed to sometimes merge together or even into larger cells. This phenomenon could provide an unappreciated venue for transferring mutated genes, chromosome fragments, whole chromosomes or sets of chromosomes into an invaded cell. We suggest that these observations may be a serendipitous illustration of the missing link between the genetic mutations implicated in the development of primary carcinoma, and the extreme genomic diversity that characterizes malignant metastases. Graphical Summary Single cell clones of Panc 1 ALK4 KO cultures changed from largely uniformly epithelioid in character (A) to cultures composed of an unexpectedly wide variety of cells, and that this variety is not explained by the traditional “EMT” (epithelial-to-mesenchymal-transition). These cells range from very small to very large, have varying degrees of motility, chromosome number, and cell division behavior. We describe the new morphologies and behaviors, and present time lapse movies and microscopic evidence that help explain the evolution of this variety. These new cell types integrate into colonies where the smallest cells gather into spherical masses (D). We believe that the heterogeneity in genomic content and behavior provides the variety needed for natural selection to select cells that are resistant to most treatments for metastatic cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba6b16627321b67c2cf57a151fe0aa1bc2c09235" target='_blank'>
              A Model for the Diversity Explosion Fundamental to Metastasis
              </a>
            </td>
          <td>
            Edward Tim O’Brien
          </td>
          <td>2025-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92b7de6d8aeceaad9c8d57eb1d9c3d308c5a80e2" target='_blank'>
              Ribosome biogenesis as a potential therapeutic target in KRAS mutant colorectal cancer.
              </a>
            </td>
          <td>
            Yui Tanaka, M. Sakahara, H. Yamanaka, Yasuko Natsume, Daisuke Kusama, Kohei Kumegawa, Harunori Yoshikawa, Yuich Abe, K. Okabayashi, Shimpei Matui, Y. Kitagawa, Naohiko Koshikawa, H. Osumi, E. Shinozaki, Satoshi Nagayama, Jun Adachi, R. Maruyama, Ryoji Yao
          </td>
          <td>2025-12-27</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Micronuclei (MN) are small extranuclear chromosomal fragments that arise from genomic instability and serve as established biomarkers for genotoxicity and disease susceptibility. Once considered only markers of disease, they are now recognized as active, causative drivers of disease progression, driven by emerging evidence of epigenetic regulation within these structures. The micronucleus assay is recognized as a cost-effective and minimally invasive method for monitoring genotoxicity resulting from both chronic and early exposure to environmental factors such as arsenic and lead, as well as from genetic instability associated with cancer progression. This review critically examines the expanding role of MN beyond traditional cytogenetic endpoints, with particular emphasis on recent insights into their epigenetic landscape. Mass spectrometry-based studies have demonstrated that MN possess distinct histone posttranslational modification signatures compared to primary nuclei, including alterations in H3K27ac, H3K9ac, and H3K18ac. These modifications affect chromatin structure, gene expression, and DNA repair mechanisms. In the context of xenobiotic exposures, MN-associated epigenetic changes may function as early indicators of disease progression. Additionally, rupture of the MN envelope can activate innate immune responses through the cGAS-STING pathway or result in chromothripsis, both of which contribute to cancer progression. The concept of "Micronuclear Epigenetics" is highlighted, with its potential application in high-throughput diagnostic platforms, particularly liquid biopsy, discussed. This approach may enhance early detection and risk stratification in exposure-induced toxicity and diseases such as cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a963e7666be290f5a7f2a006e9788007c4d9584f" target='_blank'>
              Potential of Micronuclear Epigenetic Signatures in Analyses of Toxicity and Genomic Instability.
              </a>
            </td>
          <td>
            Somnath Paul, Ankita Das, A. Bandyopadhyay, Ashok K. Giri
          </td>
          <td>2026-01-27</td>
          <td>Cell biology international</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Familial cancers are caused by inherited mutations in specific genes that regulate cell growth, division, and repair. Approximately 5–10% of all cancer cases have a hereditary component, where germline mutations in certain genes increase an individual’s susceptibility to developing cancer. Two major categories of genes are involved in cancer development: tumour suppressor genes and oncogenes. Both play critical roles in regulating normal cell behaviour, and when mutated, they can contribute to uncontrolled cell proliferation and tumour formation. In addition to genetic mutations, epigenetic alterations also play a significant role in familial cancer. Epigenetics refers to changes in gene expression due to DNA methylation, histone modifications, and the dysregulation of non-coding RNAs without alter the underlying DNA sequence. Familial cancer syndromes follow various inheritance patterns, including autosomal dominant, autosomal recessive, X-linked, and mitochondrial inheritance, each with distinct characteristics. Identifying genetic mutations associated with familial cancers is a cornerstone of genetic counselling, which helps individuals and families navigate the complex intersection of genetics, cancer risk, and prevention. Early identification of mutations enables personalized strategies for risk reduction, early detection, and, when applicable, targeted treatment options, ultimately improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69f296f3ae0afec7c6382eb8b634a83d98dd9138" target='_blank'>
              Genetic Basis of Familial Cancer Risk: A Narrative Review
              </a>
            </td>
          <td>
            Eman Fares Sabik
          </td>
          <td>2026-01-13</td>
          <td>DNA</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Two non-synonymous single-nucleotide polymorphisms (SNPs) rs61752561 (D95N substitution) and rs17632542 (I163T substitution) in the KLK3 gene encoding prostate-specific antigen (PSA), a chymotrypsin-like serine protease, are associated with prostate cancer risk and have been shown to reduce the activity of PSA. However, the structural impact of these SNPs on PSA, which may underlie the observed risk associations and functional alterations, has not been fully explored. Computational modelling predicted that the variants D95N and I163T do not cause drastic structural changes in PSA. However, molecular dynamics simulations suggested that while the two prominent loops of wild-type PSA remain tethered to their initial conformations over 500 ns of simulation, they are disrupted in both variants, leading to increased loop dynamics. Frustration analysis, normal mode analysis (NMA) and perturbation response scanning identified dynamic links between mutation sites and increased loop dynamics that trigger long-range conformational changes, disrupting the active site and potentially hindering catalytic activity. Thermal denaturation stability assays using recombinant protein show the impact of D95N and I163T substitution on the protein stability. These data show that KLK3 SNPs disrupt dynamic communication of the key loops required for proteolytic activity of PSA, which may explain the association of these SNPs with prostate cancer risk and/or progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/387e91e1f338cbf07e62b9545288e8b8401fb288" target='_blank'>
              Prostate cancer risk-associated single-nucleotide polymorphisms impact the conformational dynamics of prostate-specific antigen
              </a>
            </td>
          <td>
            Srilakshmi Srinivasan, Brooke K Hayes, Mauricio G S Costa, Blake T Riley, Emily Wilson, Emilia M. Marijanovic, Itamar Kass, Hannu Koistinen, D. Hoke, Judith A. Clements, Ashley M Buckle, J. Batra
          </td>
          <td>2025-12-17</td>
          <td>BMC Biology</td>
          <td>0</td>
          <td>24</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025', '2026'],
    y: [11, 1],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>