<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-04-14 09:29:08 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="DHFR gene amplification is commonly present in methotrexate (MTX)-resistant colon cancer cells and acute lymphoblastic leukemia. In this study, we proposed an integrative framework to characterize the amplified region by using a combination of single-molecule real-time sequencing, next-generation optical mapping, and chromosome conformation capture (Hi-C). We identified an amplification unit spanning 11 genes, from the DHFR gene to the ATP6AP1L gene position, with high adjusted interaction frequencies on chromosome 5 (~2.2 Mbp) and a twenty-fold tandemly amplified region, and novel inversions at the start and end positions of the amplified region as well as frameshift insertions in most of the MSH and MLH genes were detected. These mutations might stimulate chromosomal breakage and cause the dysregulation of mismatch repair. Characterizing the tandem gene-amplified unit may be critical for identifying the mechanisms that trigger genomic rearrangements. These findings may provide new insight into the mechanisms underlying the amplification process and the evolution of drug resistance. Sequencing a large region of DNA containing many surplus copies of genes linked to drug resistance in colon cancer cells may illuminate how these genomic rearrangements arise. Such regions of gene amplification are highly repetitive, making them impossible to sequence using ordinary methods, and little is known about how they are generated. Using advanced methods, Jeong-Sun Seo at Seoul National University Bundang Hospital in South Korea and co-workers sequenced a region of gene amplification in colon cancer cells. The amplified region was approximately 20 times the length of that in healthy cells and contained many copies of an eleven-gene segment, including a gene implicated in drug resistance. The region also contained mutations in chromosomal repair genes which would disrupt repair pathways. These results illuminate the genetic changes that lead to gene amplification and drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental and Molecular Medicine, Experimental & Molecular Medicine</td>
          <td>4</td>
          <td>49</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum.
                </a>
              </td>
          <td>
            Gustavo S França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>31</td>
          <td>42</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="In eukaryotic cells, gene-distal regulatory elements (REs) facilitate long-range gene regulation, ensuring cell type-specific transcriptional programs. This mechanism is frequently disrupted in cancer, often driven by transcription factors (TFs) that serve as targets for cancer therapy. However, targeting these TFs can lead to acquired resistance mechanisms that are not fully understood. We demonstrate that mesothelioma cancer cells, dependent on the oncogenic driver TF family TEAD, develop resistance to a pan-TEAD inhibitor and revert to an evolutionarily ancient, promoter-centric gene regulatory mechanism to recover gene expression following TEAD inhibition. Base-pair-resolution 3D chromatin conformation mapping reveals that RE-promoter interactions are disrupted in resistant cells, despite epigenetic and transcriptomic recovery. Mechanistically, in resistant cells, TF complexes, including resistance-specific FOSL1 and KLF4, preferentially bind and enhance promoter activity to recover gene expression, rendering distal REs dispensable. Our findings highlight promoter elements and promoter-specific TFs as potential therapeutic targets using a model of drug-resistant cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8fe445b4115593745bb7783131bb805e9f85aeb" target='_blank'>
              Promoter-centric gene regulation in drug-resistant cancer
              </a>
            </td>
          <td>
            Vasumathi Kameswaran, Sayantanee Paul, Daniel Le, Jonathan Hoover, Luke Y. Zhao, Alissa D. Guarnaccia, Thijs J. Hagenbeek, Jessica M. Lund, Ana Xavier-Magalhães, Minyi Shi, Julia Lau, Marco De Simone, Yuxin Liang, Anwesha Dey, Z. Modrusan, Bence Daniel
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="Identifying genetic dependencies in human colon cancer could help identify effective treatment strategies. Genome-wide CRISPR-Cas9 dropout screens have the potential to reveal genetic dependencies, some of which could be exploited as therapeutic targets using existing drugs. In this study, we comprehensively characterized genetic dependencies present in a colon cancer organoid avatar, and validated tumor-specific selectivity of select pharmacologic agents. We conducted a genome-wide CRISPR dropout screen to elucidate the genetic dependencies that interacted with select driver somatic mutations. We found distinct genetic dependencies that interacted with WNT, MAPK, PI3K, TP53, and mismatch repair pathways and validated targets that could be exploited as treatments for this specific subtype of colon cancer. These findings demonstrate the utility of functional genomic screening in the context of personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6f08c27268b9a856764ce30d936babdccabacfc" target='_blank'>
              Comprehensive genomic dependency landscape of a human colon cancer organoid
              </a>
            </td>
          <td>
            Sanam Khalili, Atefeh Mohseninia, Changlong Liu, Carolyn E. Banister, Paige Heine, M. Khazan, Sidney E. Morrison, P. Gokare, Glenn S Cowley, Barbara A. Weir, David Pocalyko, Kurtis E. Bachman, Phillip J. Buckhaults
          </td>
          <td>2025-03-14</td>
          <td>Communications Biology</td>
          <td>1</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06c5022e821728aae9de62cff2d23519c700c306" target='_blank'>
              Genomic Insights into Opioid Addiction: Identification of genomics variants from Gene Expression data
              </a>
            </td>
          <td>
            Swati Ajmeriya, Biswadip Chatterjee, S. Karmakar, ID Orcid
          </td>
          <td>2025-02-23</td>
          <td>None</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="This article reviews the impact of single-cell sequencing (SCS) on cancer biology research. SCS has revolutionized our understanding of cancer and tumor heterogeneity, clonal evolution, and the complex interplay between cancer cells and tumor microenvironment. SCS provides high-resolution profiling of individual cells in genomic, transcriptomic, and epigenomic landscapes, facilitating the detection of rare mutations, the characterization of cellular diversity, and the integration of molecular data with phenotypic traits. The integration of SCS with multi-omics has provided a multidimensional view of cellular states and regulatory mechanisms in cancer, uncovering novel regulatory mechanisms and therapeutic targets. Advances in computational tools, artificial intelligence (AI), and machine learning have been crucial in interpreting the vast amounts of data generated, leading to the identification of new biomarkers and the development of predictive models for patient stratification. Furthermore, there have been emerging technologies such as spatial transcriptomics and in situ sequencing, which promise to further enhance our understanding of tumor microenvironment organization and cellular interactions. As SCS and its related technologies continue to advance, they are expected to drive significant advances in personalized cancer diagnostics, prognosis, and therapy, ultimately improving patient outcomes in the era of precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b38b69885e5e53fa9260297634ff329303ec057" target='_blank'>
              Single-Cell Sequencing: Genomic and Transcriptomic Approaches in Cancer Cell Biology
              </a>
            </td>
          <td>
            Ana Ortega-Batista, Yanelys Jaén-Alvarado, Dilan Moreno-Labrador, Natasha Gómez, Gabriela García, Erika N. Guerrero
          </td>
          <td>2025-02-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b47a232bd7c224e77a90b7f42bb716c42af55b0" target='_blank'>
              Molecular mechanisms altering cell identity in cancer.
              </a>
            </td>
          <td>
            A. Zippo, Sven Beyes
          </td>
          <td>2025-02-26</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Objective Ovarian cancer (OC) ranks among the foremost causes of mortality in gynecological malignancies, with chemoresistance being the primary factor contributing to unfavorable prognosis. This work seeks to clarify the mechanisms of resistance-related lactylation in OC, intending to offer novel theoretical foundations and therapy strategies for addressing chemoresistance. Methods Through the combined analysis of bulk RNA-seq and single-cell RNA-seq data, we initially found lactylation genes linked to chemoresistance. Subsequently, we employed differential expression analysis, survival analysis, enrichment analysis, and other methodologies to further investigate the roles and molecular mechanisms of these genes in tumor resistance. Ultimately, we investigated the differential expression of these genes in resistant and non-resistant tissues and cells via experimentation. Results We found two candidate genes associated with lactylation chemoresistance, ALDH1A1 and S100A4. Analysis of single-cell data indicated that tumor cells represent the primary cell subpopulation relevant to resistance studies. Subpopulation analysis indicated that several tumor cell subtypes were markedly linked to resistance, with elevated expression levels of ALDH1A1 and S100A4 in the resistant subpopulation, notably correlating with various immunological and metabolic pathways. Analysis of metabolic pathways indicated that oxidative phosphorylation and glycolysis activity was elevated in the resistant subpopulation, and lactic acid buildup was associated with chemoresistance. The investigation of the marker gene protein-protein interaction network in the resistant subgroup elucidated the intricate interactions among these genes. The expression levels of ALDH1A1 and S100A4 in the OC tissues of the platinum-resistant cohort were markedly elevated compared to the sensitive cohort, with a considerable rise in S100A4 expression observed in resistant OC cells, demonstrating co-localization with lactylation. Conclusion This work elucidates the significant function of lactylation in OC chemoresistance and identifies ALDH1A1 and S100A4 as possible genes associated with drug resistance. These findings enhance our comprehension of the mechanisms behind chemoresistance in OC and offer critical insights for the formulation of novel therapeutic options.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e8903cd68465d50c11566b876c6f8d732f1ba01" target='_blank'>
              Integration of scRNA-seq and bulk RNA-seq to reveal the association and potential molecular mechanisms of metabolic reprogramming regulated by lactylation and chemotherapy resistance in ovarian cancer
              </a>
            </td>
          <td>
            Fang Ren, Xiaoao Pang, Feng Jin, Nannan Luan, Houhua Guo, Liancheng Zhu
          </td>
          <td>2025-02-28</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91c35578020ea8ee2f21fa56db96b33c568e8752" target='_blank'>
              Structural patterns and transcriptional effects of integrated Epstein-Barr virus revealed by long-read sequencing and RNA-sequencing in Nasopharyngeal Carcinoma
              </a>
            </td>
          <td>
            Zahra Kardan, Adway Kadam, A. Chai, S. Cheong, Sampath K Loganathan
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a4cbce0b7fb1d47370357ef7f4f5d368e5b4543" target='_blank'>
              Genetic Profiling of Acute and Chronic Leukemia via Next-Generation Sequencing: Current Insights and Future Perspectives
              </a>
            </td>
          <td>
            Laras Pratiwi, Fawzia Hanum Mashudi, Mukti Citra Ningtyas, H. Sutanto, P. Z. Romadhon
          </td>
          <td>2025-03-28</td>
          <td>Hematology Reports</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="BACKGROUND
Bronchopulmonary dysplasia (BPD) is the most common chronic respiratory disease in extremely premature infants. This study aims to identify gene expression dysregulation and explore various molecular pathways implicated in BPD.


METHODS
This study integrated BPD genome-wide association study (GWAS), single-cell transcriptomics (scRNA-seq), and Mendelian randomization (MR) analysis to investigate the causal relationship between gene expression and BPD.


RESULTS
Cell annotation and ligand-receptor analysis highlighted myofibroblasts as the most interactive cell type. Key genes, including CDH4, ENC1, and PAM, were identified as protective factors against BPD, while GRB10 was associated with increased disease risk. Immune metabolism-related pathways showed elevated activity of PAM, GRB10, and ENC1 in epithelial-mesenchymal transition. The Drug-Gene Interaction Database (DGIdb) predicted three drugs-LM10, navoximod, and ziprasidone-that potentially interact with these key genes.


CONCLUSION
This integrative genome-wide analysis provides valuable insights into the genetic mechanisms underlying BPD. The findings facilitate the identification of novel therapeutic targets and pave the way for personalized treatment strategies for affected neonates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3359b1453795210ae254e3906690102cf9db041c" target='_blank'>
              Novel therapeutic targets uncovered by genome-wide integrative analysis in bronchopulmonary dysplasia.
              </a>
            </td>
          <td>
            Zhenyu Xiong, Qingxiong Zhu, Lei Hang
          </td>
          <td>2025-02-23</td>
          <td>The journal of maternal-fetal & neonatal medicine : the official journal of the European Association of Perinatal Medicine, the Federation of Asia and Oceania Perinatal Societies, the International Society of Perinatal Obstetricians</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Transposable elements (TEs), which constitute nearly half of the human genome, have long been regarded as genomic "dark matter". However, their reactivation in tumor cells, resulting in the production of TE-chimeric transcripts (TCTs), has emerged as a potential driver of cancer progression. The complexity and full extent of these transcripts remain elusive, largely due to the limitations of short-read next-generation sequencing technologies. These methods have struggled to comprehensively capture the diversity and structure of TCTs, particularly those involving short interspersed nuclear elements (SINEs) or closely co-transcribed TEs. Methods Leveraging full-length cDNA sequencing technology based on nanopore sequencing platform, we developed a customized pipeline for identifying and quantifying TCTs in 19 lung adenocarcinoma (LUAD) cell lines. The short-read RNA-seq dataset from a LUAD corhort (~ 200 tumor samples) was employed to validate the identified TCTs and explore their association with tumor progression. To assess the functional roles of a specific TCTs, cell migration and cell proliferation assays were performed. Results We uncovered 208 unique TCT candidates in the LUAD cell lines. Our approach allowed for the identification of cryptic promoters and terminators within non-transposing TEs. Notably, we identified a chimeric transcript involving MIR_HKDC1, which appears to play a significant role in the progression of LUAD. Furthermore, the expression of these TCTs were associated with poor clinical outcomes in a cohort of LUAD patients, suggesting their potential as novel biomarkers for both LUAD progression and prognosis. Conclusions Our study underscores the application of long-read sequencing to unravel the complex landscape of TCTs in LUAD. We provide a comprehensive characterization of TCTs in LUAD, exploring their potential regulatory roles in cancer progression. These findings contribute to a deeper understanding of the genomic intricacies underlying cancer, and offer new directions for the development of targeted therapies and personalized treatment strategies for LUAD. This research highlights the potential of TCTs as both biomarkers and therapeutic targets in the oncogenesis, offering new insights into the interplay between transposable elements and gene regulation in cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-13888-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f1fcc8488427a1fecd1258b2070d8c49656eecc" target='_blank'>
              Long-read RNA sequencing enables full-length chimeric transcript annotation of transposable elements in lung adenocarcinoma
              </a>
            </td>
          <td>
            Yang Li, Yahui Liu, Yingxin Xie, Yaxuan Wang, Jing Wang, Huan Wang, Lin Xia, Dan Xie
          </td>
          <td>2025-03-15</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fbd7856e741845eb17452c67c39e622695a90ba" target='_blank'>
              Advances in DNA replication research for precision medicine
              </a>
            </td>
          <td>
            Md. Mosharraf Hossen
          </td>
          <td>2025-03-08</td>
          <td>Journal of Bioscience and Environment Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Traditionally, the evolutionary perspective of cancer has been understood as gradual alterations in passenger/driver genes that lead to branching phylogeny. However, in cases of prostate adenocarcinoma and kidney renal cell carcinoma, macroevolutionary landmarks like chromoplexy and chromothripsis are frequently observed. Unfortunately, short-read sequencing techniques often miss these significant macroevolutionary changes, which involve multiple translocations and deletions at the chromosomal level. To resolve such genomic dark matters, we provided high-fidelity long-read sequencing data (78–92 Gb of ~Q30 reads) of six genitourinary tumour cell lines (one benign kidney tumour and two kidney and three prostate cancers). Based on these data, we obtained 12 high-quality, partially phased genome assemblies (Contig N50 1.85–29.01 Mb; longest contig 2.02–171.62 Mb), graph-based pan-genome variant sets (11.57 M variants including 60 K structural variants), and 5-methylcytosine sites (14.68%–27.05% of the CpG sites). We also identified several severe chromosome aberration events, which would result from chromosome break and fusion events. Our cancer genome assemblies will provide unprecedented resolution to understand cancer genome instability and chromosomal aberration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39aabbe27003a6111c9e819cce1949d58b416548" target='_blank'>
              Genome assembly resources of genitourinary cancers for chromosomal aberration at the single nucleotide level
              </a>
            </td>
          <td>
            Hyunho Han, H. Lee, Min Gyu Kim, Yoo Sub Shin, Jin Soo Chung, Jun Kim
          </td>
          <td>2025-04-01</td>
          <td>Scientific Data</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/398c45e50f2a25bf4d4f179535f4ed528664ab8c" target='_blank'>
              Genomic profiling reveals molecular heterogeneity in patients with Richter transformation (RT) and chronic lymphocytic leukemia (CLL)
              </a>
            </td>
          <td>
            Shulan Tian, Hanyin Wang, S. Parikh, Yuanhang Liu, Helen Jin-4 Lee, Erik Jessen, Eric W. Klee, Yucai Wang, Fan Leng, Min Shi, Cinthya J Zepeda-Mendoza, Rong He, Saad J. Kenderian, Linda B. Baughn, D. L. Dyke, P. Hampel, Neil E. Kay, Esteban Braggio, S. Slager, Huihuang Yan, W. Ding
          </td>
          <td>2025-03-18</td>
          <td>None</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="Introduction A hallmark of small cell lung cancer (SCLC) is its recalcitrance to therapy. While most SCLCs respond to frontline therapy, resistance inevitably develops. Identifying phenotypes potentiating chemoresistance and immune evasion is a crucial unmet need. Previous reports have linked upregulation of the DNA damage response (DDR) machinery to chemoresistance and immune evasion across cancers. However, it is unknown if SCLCs exhibit distinct DDR phenotypes. Methods To study SCLC DDR phenotypes, we developed a new DDR gene analysis method and applied it to SCLC clinical samples, in vitro, and in vivo model systems. We then investigated how DDR regulation is associated with SCLC biology, chemotherapy response, and tumor evolution following therapy. Results Using multi-omic profiling, we demonstrate that SCLC tumors cluster into three DDR phenotypes with unique molecular features. Hallmarks of these DDR clusters include differential expression of DNA repair genes, increased replication stress, and heightened G2/M cell cycle arrest. SCLCs with elevated DDR phenotypes exhibit increased neuroendocrine features and decreased “inflamed” biomarkers, both within and across SCLC subtypes. Clinical analyses demonstrated treatment naive DDR status was associated with different responses to frontline chemotherapy. Using longitudinal liquid biopsies, we found that DDR Intermediate and High tumors exhibited subtype switching and coincident emergence of heterogenous phenotypes following frontline treatment. Conclusions We establish that SCLC can be classified into one of three distinct, clinically relevant DDR clusters. Our data demonstrates that DDR status plays a key role in shaping SCLC phenotypes and may be associated with different chemotherapy responses and patterns of tumor evolution. Future work targeting DDR specific phenotypes will be instrumental in improving patient outcomes. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02291-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/671674f412412e8378b84a598f7d2a87a83c70ce" target='_blank'>
              DNA damage response signatures are associated with frontline chemotherapy response and routes of tumor evolution in extensive stage small cell lung cancer
              </a>
            </td>
          <td>
            Benjamin B. Morris, Simon Heeke, Yuanxin Xi, L. Diao, Qi Wang, Pedro Rocha, Edurne Arriola, Myung Chang Lee, Darren R. Tyson, K. Concannon, K. Ramkumar, C. Stewart, R. Cardnell, Runsheng Wang, Vito Quaranta, Jing Wang, J. Heymach, B. Nabet, D. Shames, C. Gay, L. Byers
          </td>
          <td>2025-03-20</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>107</td>
        </tr>

        <tr id="Background: Breast cancer is the most prevalent malignancy among females worldwide. Extensive research has highlighted cancer stem cells (CSCs) as critical drivers of tumor initiation, progression, recurrence, and therapy resistance. However, the heterogeneity of breast cancer stem cells (BCSCs) and their dynamic roles within the tumor microenvironment remain inadequately understood. Methods: This study utilized the single-cell RNA sequencing dataset to categorize BCSCs into two subgroups within the breast cancer microenvironment and investigate their pseudo-time developmental dynamics. Bulk transcriptomic data from TCGA-BRCA were integrated to assess the prognostic significance and infiltration abundance of the BCSCs-2 subgroup. Functional enrichment, co-expression network analysis, and somatic mutation profiling were performed to elucidate key biological pathways and genetic features. Additionally, drug sensitivity analyses were conducted using the Connectivity Map database to identify potential therapeutic strategies. Results: A total of 459 BCSCs were identified and further classified into two distinct subpopulations: BCSCs-1 and BCSCs-2. High infiltration of BCSCs-2 was associated with poor prognosis and an immunosuppressive tumor microenvironment. Co-expression network analysis identified 16 key genes linked to BCSCs-2, while somatic mutation analysis revealed distinct mutation patterns associated with its infiltration. Drug sensitivity analysis suggested that patients with high BCSCs-2 infiltration could benefit from classical chemotherapy agents, such as Cisplatin, and other novel therapeutic compounds. Conclusions: This study offers novel insights into the heterogeneity and functional roles of BCSCs in breast cancer. The findings highlight the prognostic and therapeutic importance of the BCSCs-2 subgroup, providing potential biomarkers and therapeutic targets for precision medicine in breast cancer management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/491a724da75389432860fa6c4bc953c58c7363a6" target='_blank'>
              Multi-Omics Profiling Identifies a High-Risk Subgroup of Breast Cancer Stem Cells for Prognostic Stratification and Personalized Treatment
              </a>
            </td>
          <td>
            Guixin Wang, Ziyi Chen, Yao Tian, Yuxin Zhu, Shuo Wang, Wenbin Song, Xin Wang, Yingxi Li
          </td>
          <td>2025-02-28</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Glioblastoma (GBM) patients frequently develop resistance to temozolomide (TMZ), the standard chemotherapy. While targeting cancer metabolism shows promise, the relationship between metabolic perturbation and drug resistance remains poorly understood. Methods We performed high-throughput CRISPR interference screens in GBM cells to identify genes modulating TMZ sensitivity. Findings were validated using multiple GBM cell lines, patient-derived glioma stem cells, and clinical data. Molecular mechanisms were investigated through transcriptome analysis, metabolic profiling, and functional assays. Results We identified phosphoglycerate kinase 1 (PGK1) as a key determinant of TMZ sensitivity. Paradoxically, while PGK1 inhibition suppressed tumor growth, it enhanced TMZ resistance by inducing metabolic stress. This activated AMPK and HIF-1α pathways, leading to enhanced DNA damage repair through 53BP1. PGK1 expression levels correlated with TMZ sensitivity across multiple GBM models and patient samples. Conclusions Our study reveals an unexpected link between metabolic stress and chemoresistance, demonstrating how metabolic adaptation can promote therapeutic resistance. These findings caution against single-agent metabolic targeting and suggest PGK1 as a potential biomarker for TMZ response in GBM. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06261-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8773538db860e37609b781e5433e56f0729eca07" target='_blank'>
              Large-scale CRISPRi screens link metabolic stress to glioblastoma chemoresistance
              </a>
            </td>
          <td>
            Xing Li, Wansong Zhang, Yitong Fang, Tianhu Sun, Jian Chen, Ruilin Tian
          </td>
          <td>2025-03-06</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4df1cf0d85a605bfa799ff58abdd0710b0b2a957" target='_blank'>
              Optical Pooled Screening for the Discovery of Regulators of the Alternative Lengthening of Telomeres Pathway
              </a>
            </td>
          <td>
            Isabel Quintanilla, Benura Azeroglu, M. A. Sagar, Travis H. Stracker, Eros Lazzerini Denchi, Gianluca Pegoraro
          </td>
          <td>2025-02-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/019e491b785ffb0bdd2ceea7ba005ed8f5742762" target='_blank'>
              Relapse-specific genetic patterns in the exomic mutational landscape in acute myeloid leukemia
              </a>
            </td>
          <td>
            Wardah Qureshi, Aisha Jamal, Atia Gohar, Marium Imran Mansuri, Muhammad Irfan, I. Khan, Muhammad Shakeel
          </td>
          <td>2025-03-25</td>
          <td>Egyptian Journal of Medical Human Genetics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Metastasis is the leading cause of cancer deaths; nonetheless, how tumor cells adapt to vastly different organ contexts is largely unknown. To investigate this question, we generated a transcriptomic atlas of primary tumor and diverse metastatic samples from a patient with pancreatic ductal adenocarcinoma who underwent rapid autopsy. Unsupervised archetype analysis identified both shared and site-specific gene programs, including lipid metabolism and gastrointestinal programs prevalent in peritoneum and stomach wall lesions, respectively. We developed a probabilistic approach for inferring clonal phylogeny from single-cell and matched whole-exome data. Distantly related genetic clones in the peritoneum express the lipid metabolism program, likely due to signaling by the adipocyte-rich peritoneum environment, and cells in most clones express multiple programs, suggesting that transcriptomic plasticity is a prevalent feature of metastatic cells. These deeply annotated analyses using a patient-centric platform provide a model for investigating metastatic mechanisms and plasticity in advanced cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57b77f53eedac93ceda8af4b898f6071abded0e4" target='_blank'>
              Transcriptomic plasticity is a hallmark of metastatic pancreatic cancer
              </a>
            </td>
          <td>
            Alejandro Jiménez-Sánchez, S. Persad, Akimasa Hayashi, Shigeaki Umeda, Roshan Sharma, Yubin Xie, A. Mehta, Wungki Park, I. Masilionis, T. Chu, Feiyang Zhu, Jungeui Hong, R. Chaligné, Eileen M. O’Reilly, L. Mazutis, T. Nawy, I. Pe’er, Christine A. Iacobuzio-Donahue, D. Pe’er
          </td>
          <td>2025-03-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Aggressive tumors pose ultra-challenges to drug resistance. Anti-cancer treatments are often unsuccessful, and single-cell technologies to rein drug resistance mechanisms are still fruitless. The National Cancer Institute defines aggressive cancers at the tissue level, describing them as those that spread rapidly, despite severe treatment. At the molecular, foundational level, the quantitative biophysics discipline defines aggressive cancers as harboring a large number of (overexpressed, or mutated) crucial signaling proteins in major proliferation pathways populating their active conformations, primed for their signal transduction roles. This comprehensive review explores highly aggressive cancers on the foundational and cell signaling levels, focusing on the differences between highly aggressive cancers and the more treatable ones. It showcases aggressive tumors as harboring massive, cancer-promoting, catalysis-primed oncogenic proteins, especially through certain overexpression scenarios, as predisposed aggressive tumor candidates. Our examples narrate strong activation of ERK1/2, and other oncogenic proteins, through malfunctioning chromatin and crosslinked signaling, and how they activate multiple proliferation pathways. They show the increased cancer heterogeneity, plasticity, and drug resistance. Our review formulates the principles underlying cancer aggressiveness on the molecular level, discusses scenarios, and describes drug regimen (single drugs and drug combinations) for PDAC, NSCLC, CRC, HCC, breast and prostate cancers, glioblastoma, neuroblastoma, and leukemia as examples. All show overexpression scenarios of master transcription factors, transcription factors with gene fusions, copy number alterations, dysregulation of the epigenetic codes and epithelial-to-mesenchymal transitions in aggressive tumors, as well as high mutation loads of vital upstream signaling regulators, such as EGFR, c-MET, and K-Ras, befitting these principles.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbd64f2509541e7332b94c3a04278ee5d7e930fb" target='_blank'>
              Molecular principles underlying aggressive cancers
              </a>
            </td>
          <td>
            Ruth Nussinov, B. R. Yavuz, Hyunbum Jang
          </td>
          <td>2025-02-17</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>3</td>
          <td>47</td>
        </tr>

        <tr id="Over 80% of children with acute lymphoblastic leukemia (pALL) can be cured by treating them with multiple chemotherapeutic agents administered over several years, whereas pALL is incurable with 1-3 medications, suggesting significant variation in drug susceptibility across clonal populations. While bulk sequencing studies indicate that pALL cells contain relatively few genetic variants compared to other cancers, the true extent of genetic diversity at the single-cell level remains unknown. Here, we used three complementary approaches to investigate pALL genetic heterogeneity: error-corrected bulk sequencing, single-cell exome sequencing, and primary template-directed amplification (PTA)-enabled single-cell genome sequencing. We discovered that some ETV6-RUNX1 samples harbor multiple independent ras clones and that individual pALL cells harbor substantially more mutations (mean 3,553 per cell) than detected in bulk samples (mean 965 mutations), with variant signatures suggesting both early and late APOBEC-driven mutagenesis in ETV6-RUNX1 patients. Using PTA-based phylogenetic analysis of over 150 single-cell genomes from four pALL patients, we identified heritable phenotypes associated with specific genetic alterations, including some low-frequency clones that are preferentially selected for during chemotherapy treatment. Our findings reveal previously undetected genetic diversity in pALL and suggest that pre-existing mutations influence treatment response, with implications for future therapeutic strategies. This study provides a high-resolution framework for understanding cancer clonal evolution during treatment, yielding important new insights for developing more effective therapeutic approaches for pALL. Key Points Increasing the accuracy and resolution of sequencing from bulk to error-corrected bulk to single clone to single-cell sequencing reveals increasing levels of genetic diversity in pALL, including the presence of clone-specific driver and treatment resistance-associated mutations. Using PTA-enabled scWGS, single pALL cells contain several fold more mutations per cell than detected in bulk sequencing, which, when multiplied across billions of cells, reveal hidden population-scale genetic complexity. Some phenotypes in pALL are heritable, including treatment resistance, where we identify pre-existing rare clones with relapse-associated mutations that preferentially survive four weeks of standard treatment in patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cfebdd69571f854dae1eb84acf1f1884a093ad9" target='_blank'>
              Measuring Longitudinal Genome-wide Clonal Evolution of Pediatric Acute Lymphoblastic Leukemia at Single-Cell Resolution
              </a>
            </td>
          <td>
            Yakun Pang, Tamara Prieto, V. Gonzalez-Pena, Athena Aragon, Yuntao Xia, Sheng Kao, Sri Rajagopalan, John Zinno, Jean Quentin, Julien Laval, Dennis Yuan, Nathaniel D. Omans, David Klein, Matthew MacKay, I. De Vlaminck, John Easton, William Evans, D. Landau, Charles Gawad
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Genome-wide human genetic studies have identified inherited cis-regulatory loci variants that predispose to cancers. However, the mechanisms by which these germline variants influence cancer progression, particularly through gene expression and proteostasis control, remain unclear. By analyzing genomic data from a gastric cancer (GC) case-control study (2,117 individuals), focusing on the ubiquitin-specific protease (USP) family, we identify the single nucleotide polymorphism (SNP) rs72856331 (G>A) in the promoter region of the proto-oncogene USP47 as a putative susceptibility allele for GC (OR = 0.78, P = 0.015). Mechanistically, the risk allele G is associated with enhanced USP47 expression, mediated by altered recruitment of the transcription factor GLI3 and changes in the epigenetic status at promoter. CRISPR/Cas9-mediated single-nucleotide conversion into risk allele G results in increased GLI3 binding and subsequent USP47 upregulation. The depletion of GLI3 results in a reduction of cancer-related phenotypes, similar to those observed following USP47 knockdown. Furthermore, we identify Snai1 as a deubiquitination target of USP47, explaining USP47-dependent activation of epithelial-mesenchymal transition pathway and tumor progression. Our findings identify an important genetic predisposition that implicates the perturbation of transcription and proteostasis programs in GC, offering insights into prevention and therapeutic strategies for genetically stratified patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc1084b39e6f7bd79c603ae5a0dd8fa214a3f635" target='_blank'>
              An inherited predisposition allele promotes gastric cancer via enhancing deubiquitination-mediated activation of epithelial-to-mesenchymal transition signaling.
              </a>
            </td>
          <td>
            Bolin Tao, Zhenning Wang, Xuanyi Wang, Aixia Song, Jiaxian Liu, Jianan Wang, Qin Zhang, Zhaolin Chen, Zixian Wang, Wenjie Xu, Menghong Sun, Yanong Wang, Ping Zhang, Tao Xu, Gong-Hong Wei, F. X. Chen, Mengyun Wang
          </td>
          <td>2025-02-25</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background and Objective: The discovery of novel molecular biomarkers via next-generation sequencing technologies has revolutionized how glioblastomas (GBMs) are classified nowadays. This has resulted in more precise diagnostic, prognostic, and therapeutic approaches to address this malignancy. The present work examines the applications of single-cell RNA sequencing (scRNA-seq) in GBM, focusing on its potential to address tumor complexity and therapeutic resistance and improve patient outcomes. Methods: A scoping review of original studies published between 2009 and 2024 was conducted using the PUBMED and EMBASE databases. Studies in English or Spanish related to single-cell analysis and GBM were included. Key Findings: The database search yielded 453 publications. Themes related to scRNA-seq applied for the diagnosis, prognosis, treatment, and understanding of the cancer biology of GBM were used as criteria for article selection. Of the 24 studies that were included in the review, 11 focused on the tumor microenvironment and cell subpopulations in GBM samples, 5 investigated the use of sequencing to elucidate the GBM cancer biology, 3 examined disease prognosis using sequencing models, 3 applied translational research through scRNA-seq, and 2 addressed treatment-related problems in GBM elucidated by scRNA-seq. Conclusions: This scoping review explored the various clinical applications of scRNA-seq technologies in approaching GBM. The findings highlight the utility of this technology in unraveling the complex cellular and immune landscapes of GBM, paving the way for improved diagnosis and personalized treatments. This cutting-edge approach might strengthen treatment strategies against tumor progression and recurrence, setting the stage for multi-targeted interventions that could significantly improve outcomes for patients with aggressive, treatment-resistant GBMs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50f255f1e2df6321b7a03cd03e70b813bcc99665" target='_blank'>
              Current Applications of Single-Cell RNA Sequencing in Glioblastoma: A Scoping Review
              </a>
            </td>
          <td>
            Edgar G. Ordóñez-Rubiano, Nicolás Rincón-Arias, William J. Shelton, A. F. Salazar, María Alejandra Sierra, Raphael Bertani, D. Gómez-Amarillo, Fernando Hakim, Matías Baldoncini, César Payán-Gómez, A. Cómbita, Sandra C. Ordonez-Rubiano, Rafael Parra-Medina
          </td>
          <td>2025-03-01</td>
          <td>Brain Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1e688482cb691231e9728515c865331942ee770" target='_blank'>
              Integrating gene expression, mutation and copy number data to identify driver genes of recurrent chromosome-arm losses
              </a>
            </td>
          <td>
            Ron Saad, Ron Shamir, Uri Ben-David
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Chimeric Antigen Receptor T (CAR-T) cells offer a promising strategy for cancer treatment. These CAR-T cells are either autologous or allogeneic T cells that are genetically modified to express a chimeric antigen receptor targeting a specific tumor antigen. Ongoing research aims to optimize the CAR-T cell efficacy, including strategies to modulate their metabolism. One such approach involves inducing transgene expression by activating the GCN2 kinase signaling pathway through dietary deprivation of an essential amino acid. In this study, we investigated the general impact of a 6-hour leucine deprivation on primary activated human T cells using RNA-seq technology. Our analysis identified 3,431 differentially expressed genes between T cells cultured in regular medium and those cultured in leucine-deprived medium. Gene Set Enrichment Analysis revealed that “TNFα signaling via NFκB”, “interferon-γ response”, and “unfolded protein response” gene sets were positively enriched, while “mTORC1 signaling”, “Myc targets”, and “oxidative phosphorylation” gene sets were negatively enriched. To further evaluate the involvement of GCN2 kinase in regulating the differential gene expression during the 6-hour leucine deprivation, T cells were cultured with or without a GCN2 inhibitor. We found that 59% of the differentially expressed genes in our dataset were dependent on the kinase GCN2 (n = 2028), with 1,140 up-regulated and 888 down-regulated genes. These findings suggest a promising strategy to enhance CAR-T cell efficacy by combining short amino acid starvation with transient overexpression of a target gene.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7518ff0dd3a5481cf6365679f81c8157f814beb" target='_blank'>
              Transcriptomic analysis of human primary T cells after short-term leucine-deprivation and evaluation of kinase GCN2’s role in regulating differential gene expression
              </a>
            </td>
          <td>
            Aurore Dougé, Gwendal Cueff, Céline Keime, V. Carraro, C. Jousse, Paul Rouzaire, A. Bruhat
          </td>
          <td>2025-02-18</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbfea92c86debe9c5d855419c64075ab48f3f011" target='_blank'>
              Single cell transcriptional evolution of myeloid leukaemia of Down syndrome
              </a>
            </td>
          <td>
            Mi K Trinh, Konstantin Schuschel, Hasan Issa, Rebecca Thomas, Conor Parks, Agnes Oszlanczi, Toochi Ogbonnah, Di Zhou, L. Mamanova, E. Prigmore, Emilia Robertson, A. Hodder, Anna Wenger, Nathaniel D Anderson, Holly J. Whitfield, T. Treger, J. Gonçalves-Dias, Karin C Straathof, David O’Connor, , Laura Jardine, Stuart Adams, J. Klusmann, Jack Bartram, Sam Behjati
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/996e0da402cc341f0ff28812c9362a9660b3a867" target='_blank'>
              Abstract A026: Deciphering natural and synthetic genetic programs governing human T cell function with combinatorial CRISPR activation and interference screens
              </a>
            </td>
          <td>
            Zachary Steinhart, Jeffrey Perera, Dmytro Dorovskyi, Luke Workley, Carinna Tran, Rosmely Hernandez, Zhongmei Li, Max Foisey, Daniel Goodman, Kole T. Roybal, B. Shy, Alexander Marson
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37c650c9b7245da58253a96046c65fd7d628305a" target='_blank'>
              Understanding DNA and PARP in Cancer: Tackling Inhibitor Resistance
              </a>
            </td>
          <td>
            Hardha Balachandran, Gowramma Byran, V. Karri, Senthil Kumar Murugesan, K. Rajagopal
          </td>
          <td>2025-03-13</td>
          <td>Drug Metabolism and Bioanalysis Letters</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/895b7aa47450acfbdeb8f8f1748f0e8b623eb4a8" target='_blank'>
              Extrachromosomal DNA dynamics contribute to intratumoural receptor tyrosine kinase genetic heterogeneity and drug resistance in gastric cancer.
              </a>
            </td>
          <td>
            Kazuki Kanayama, Hiroshi Imai, Ryotaro Hashizume, Chise Matsuda, Eri Usugi, Y. Hirokawa, Masatoshi Watanabe
          </td>
          <td>2025-02-19</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="InfoScan is a novel bioinformatics tool designed for the comprehensive analysis of full-length single-cell RNA sequencing (scRNA-seq) data. It enables the identification of unannotated transcripts and rare cell populations, providing a powerful platform for transcriptome characterization. In this study, InfoScan was applied to glioblastoma multiforme (GBM), identifying a rare “neoplastic-stemness” subpopulation exhibiting cancer stem cell-like features. Functional analyses suggested that tumor-associated macrophages (TAMs) secrete SPP1, which binds to CD44 on neoplastic-stemness cells, activating the PI3K/AKT pathway and driving lncRNA transcription to promote metastasis. Integration of TCGA and CGGA datasets further supported these findings, highlighting key mutations associated with the neoplastic-stemness subpopulation. Drug sensitivity assays indicated that neoplastic-stemness cells might be sensitive to omipalisib, a PI3K inhibitor, pointing to a potential therapeutic target. InfoScan offers a robust framework for exploring complex transcriptomic landscapes and characterizing rare cell populations, providing valuable insights into GBM biology and advancing precision cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d19fc728a932329a1c6b81702c24f21513cb2f5" target='_blank'>
              InfoScan: A New Transcript Identification Tool Based on scRNA-Seq and Its Application in Glioblastoma
              </a>
            </td>
          <td>
            Shiqiang Mei, Jinjin Huang, Zhen Zhang, Haotian Lei, Qiaojuan Huang, Lianghu Qu, Lingling Zheng
          </td>
          <td>2025-02-28</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancer stem cells (CSCs), are a critical subpopulation within tumours, and are defined by their capacity for self-renewal, differentiation, and tumour initiation. These unique traits contribute to tumour progression, metastasis, and resistance to conventional treatments like chemotherapy and radiotherapy, often resulting in cancer recurrence and poor patient outcomes. As such, CSCs have become focal points in developing advanced cancer therapies. This review highlights progress in CSC-targeted treatments, including chimeric antigen receptor T-cell (CAR-T) therapy, immunotherapy, molecular targeting, and nanoparticle-based drug delivery systems. Plant-derived compounds and gene-editing technologies, such as clustered regularly interspaced short palindromic repeats (CRISPR), are explored for their potential to enhance precision and minimize side effects. Metabolic pathways integral to CSC survival, such as mitochondrial dynamics, mitophagy (regulated by dynamin-related protein 1 [DRP1] and the PINK1/Parkin pathway), one-carbon metabolism, amino acid metabolism (involving enzymes like glutaminase (GLS) and glutamate dehydrogenase (GDH]), lipid metabolism, and hypoxia-induced metabolic reprogramming mediated by hypoxia-inducible factors (HIF-1α and HIF-2α), are examined as therapeutic targets. The adaptability of CSCs through autophagy, metabolic flexibility, and epigenetic regulation by metabolites like α-ketoglutarate, succinate, and fumarate is discussed. Additionally, extracellular vesicles and nicotinamide adenine dinucleotide (NAD⁺) metabolism are identified as pivotal in redox balance, DNA repair, and epigenetic modifications. Addressing challenges such as tumour heterogeneity, immune evasion, and treatment durability requires interdisciplinary collaboration. Advancing CSC-targeted therapies is essential for overcoming drug resistance and preventing cancer relapse, paving the way for transformative cancer treatments. This review underscores the importance of leveraging innovative technologies and fostering collaboration to revolutionize cancer treatment. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdfd29a8c49b166f1e1f2b3241c057ea76b860a3" target='_blank'>
              Unveiling the future of cancer stem cell therapy: a narrative exploration of emerging innovations
              </a>
            </td>
          <td>
            Joseph Nhyira Obisi, Abike Ndidiamaka Josephine Abimbola, O. Babaleye, Peter Kwame Atidoglo, Saviour Godswealth Usin, Eudora Nwanaforo, Faith Sutu Patrick-Inezi, I. V. Fasogbon, Joseph Chimezie, Christianah Adebimpe Dare, Oluwadoyinsayemi Oluwadamilare Kuti, D. E. Uti, Humphrey Chukwudi Omeoga
          </td>
          <td>2025-03-22</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21b08a5b6171c80002da0a1517c54be1999c70c6" target='_blank'>
              MED12 dysregulation: insights into cancer and therapeutic resistance.
              </a>
            </td>
          <td>
            R. Bhole, Jagruti Shinkar, Sonali Labhade, Pawan N Karwa, Harshad S. Kapare
          </td>
          <td>2025-03-19</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Lung cancer exhibits substantial inter- and intra-tumor heterogeneity, with features that present significant challenges in advancing biomarker discovery and the development of targeted therapeutics. To fill this gap, we employed single-cell RNA sequencing (scRNA-seq) and advanced bioinformatics tools to evaluate the transcriptomic heterogeneity of immortalized, non-transformed (BEAS2B) and transformed (H460) lung epithelial cell lines and their responses to carcinogen challenge. Gene expression profiles resolved four primary clusters further discretized into unique subclusters based on genetic signatures and phenotypic profiles. Profiles of long non-coding RNAs (lncRNAs) identified microRNA host genes, antisense RNA genes, divergent transcript, and long intergenic non-coding RNAs as contributors to cellular heterogeneity. These findings indicate that distinct patterns of gene expression, remarkably in lncRNAs, define cellular heterogeneity in non-transformed versus transformed cells. These features can be exploited for the development of therapies directed at specific cell subpopulations in precancerous lesions and within lung tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/233c01bff81dc65b1be1e7c0ebbee44d4a90ef36" target='_blank'>
              Single-Cell RNA Sequencing Reveals Extensive Heterogeneity and Unique Gene Trajectories in Non-Transformed and Transformed Human Lung Epithelial Cells: Insights into the Role of LncRNAs in Tumor Heterogeneity
              </a>
            </td>
          <td>
            Sokviseth Moeng, Andres D. Chamorro-Parejo, Minsun S. Jeon, James J. Cai, Kenneth S Ramos
          </td>
          <td>2025-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>1</td>
          <td>6</td>
        </tr>

        <tr id="Aneuploidy is one of the most common mechanisms of adaptation to environmental selection in vegetatively growing cells. However, it is still unclear what advantages aneuploidy provides over other types of genomic alterations. In this study, we aimed to determine if beneficial aneuploidy phenotypes are driven primarily by combinatorial expression changes of multiple genes using budding yeast as a model system. To determine the impact of complex aneuploidy on cellular fitness and resistance to a variety of drug treatments, we generated yeast collections with nearly every possible combination of two chromosome gains or losses. In addition, we genetically dissected aneuploid chromosomes using partial deletions to identify chromosomal regions that contribute to aneuploidy-driven drug resistance. Strong resistance phenotypes consistently came from combinations of chromosomes or chromosomal regions. We identified up to five regions on a single chromosome that contribute to resistance to a single drug treatment. Finally, we found that the strongest aneuploidy phenotypes came from synergistic effects between copy number changes of different chromosomes or chromosomal regions. These results provide insights into how combining subtle gene expression changes across the genome impacts aneuploidy phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/288386141655e3d2d8bd580753d7c57e3ca32011" target='_blank'>
              Combinatorial effects of multiple genes contribute to beneficial aneuploidy phenotypes
              </a>
            </td>
          <td>
            M. S. Koller, Claudia Himmelbauer, Sarah Fink, Madhwesh C. Ravichandran, Christopher S. Campbell
          </td>
          <td>2025-03-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Non-coding RNAs (ncRNAs) play crucial roles in colorectal cancer (CRC) development and progression. Recent developments in single-cell transcriptome profiling methods have revealed surprising levels of expression variability among seemingly homogeneous cells, suggesting the existence of many more cell types than previously estimated. This review synthesizes recent advances in ncRNA research in CRC, emphasizing single-cell bioinformatics approaches for their analysis. We explore computational methods and tools used for ncRNA identification, characterization, and functional prediction in CRC, with a focus on single-cell RNA sequencing (scRNA-seq) data. The review highlights key bioinformatics strategies, including sequence-based and structure-based approaches, machine learning applications, and multi-omics data integration. We discuss how these computational techniques can be applied to analyze differential expression, perform functional enrichment, and construct regulatory networks involving ncRNAs in CRC. Additionally, we examine the role of bioinformatics in leveraging ncRNAs as diagnostic and prognostic biomarkers for CRC. We also discuss recent scRNA-seq studies revealing ncRNA heterogeneity in CRC. This review aims to provide a comprehensive overview of the current state of single-cell bioinformatics in ncRNA CRC research and outline future directions in this rapidly evolving field, emphasizing the integration of computational approaches with experimental validation to advance our understanding of ncRNA biology in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df0d488fffc7d97d7489f557a3f2dfce15740d80" target='_blank'>
              Single-Cell Transcriptomic Approaches for Decoding Non-Coding RNA Mechanisms in Colorectal Cancer
              </a>
            </td>
          <td>
            M. Gondal, H. Farooqi
          </td>
          <td>2025-03-10</td>
          <td>Non-Coding RNA</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da95ac3ec1b6b3cbe7baf6551a5457106bc739f1" target='_blank'>
              DNA damage response in cancer: New treatment strategies
              </a>
            </td>
          <td>
            Hang-Hao Zhang
          </td>
          <td>2025-03-31</td>
          <td>New Cell</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/330a5593d326f97de03e4da3203d107ef7927e48" target='_blank'>
              Extending differential gene expression testing to handle genome aneuploidy in cancer
              </a>
            </td>
          <td>
            Katsiaryna Davydzenka, G. Caravagna, Guido Sanguinetti
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancer genomes have a complicated landscape of mutations, including large-scale rearrangements known as structural variants (SVs). These SVs can disrupt genes or regulatory elements, playing a critical role in cancer development and progression. Despite their importance, accurate identification of somatic structural variants (SVs) remains a significant bottleneck in cancer genomics. Long-read sequencing technologies hold great promise in SV discovery, and there is an increasing number of efforts to develop new tools to detect them. In this study, we employ eight widely used SV callers on paired tumor and matched normal samples from both the NCI-H2009 lung cancer cell line and the COLO829 melanoma cell line, the latter of which has a well-established somatic SV truth set. Following separate variation detection in both tumor and normal DNA, the VCF merging procedure and a subtraction method were used to identify candidate somatic SVs. Additionally, we explored different combinations of the tools to enhance the accuracy of true somatic SV detection. Our analysis adopts a comprehensive approach, evaluating the performance of each SV caller across a spectrum of variant types and numbers in finding cancer-related somatic SVs. This study, by comparing eight different tools and their combinations, not only reveals the benefits and limitations of various techniques but also establishes a framework for developing more robust SV calling pipelines. Our findings highlight the strengths and weaknesses of current SV calling tools and suggest that combining multiple tools and testing different combinations can significantly enhance the validation of somatic alterations. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-92750-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6b50dd05db294032c45ba6780a2c802ad31817a" target='_blank'>
              Benchmarking long-read structural variant calling tools and combinations for detecting somatic variants in cancer genomes
              </a>
            </td>
          <td>
            Safa Kerem Aydin, Kübra Yılmaz, Ahmet Acar
          </td>
          <td>2025-03-13</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The LRRK2 G2019S mutation is known to have a high penetrance rate associated with Parkinson’s disease (PD), prevalent across both familial and sporadic PD cases and implicated in neurodegenerative mechanisms. This mutation disrupts several key cellular processes, particularly affecting the endoplasmic reticulum and mitochondrial functions in neural stem cells (NSCs), which are crucial for protein homeostasis and energy metabolism. Although aging is a major risk factor for PD, the complex interplay between LRRK2 G2019S and aging-related cellular dysfunction in NSCs remains poorly understood. In this study, we performed a comprehensive transcriptomic analysis to characterize the temporal transcriptional changes in LRRK2 G2019S-carrying NSCs across sequential passages, resembling cellular aging. BAC DNA-mediated correction of the LRRK2 mutation significantly restored dysregulated cellular processes, including endoplasmic reticulum-associated protein processing, mitochondrial function, and vesicular trafficking pathways, thereby restoring cellular homeostasis in NSCs. Notably, aged NSCs harboring the LRRK2 G2019S mutation exhibited pronounced alterations in epithelial-mesenchymal transition or TGF-β signaling, exacerbating declines in NSC function. Our findings elucidate the molecular mechanisms underlying LRRK2 G2019S-mediated pathogenesis in aging NSCs and highlight the therapeutic potential of genetic correction strategies for PD treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcdb3c9416e9267424a15e0bf88185953f8c0ed8" target='_blank'>
              Transcriptome signatures of human neural stem cells derived from LRRK2 gene therapeutic cells
              </a>
            </td>
          <td>
            Doyeong Kim, , , 
          </td>
          <td>2025-04-10</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Profiling circulating cell-free DNA (cfDNA) has become a fundamental practice in cancer medicine, but the effectiveness of cfDNA at elucidating tumor-derived molecular features has not been systematically compared to standard single-lesion tumor biopsies in prospective cohorts of patients. The use of plasma instead of tissue to guide therapy is particularly attractive for patients with small cell lung cancer (SCLC), due to the aggressive clinical course of this cancer, which makes obtaining tumor biopsies exceedingly challenging. Methods In this study, we analyzed a prospective cohort of 49 plasma samples obtained before, during, and after treatment from 20 patients with recurrent SCLC. We conducted cfDNA low-pass whole genome sequencing (0.1X coverage), comparing it with time-point matched tumor characterized using whole-exome (130X) and transcriptome sequencing. Results A direct comparison of cfDNA and tumor biopsy revealed that cfDNA not only mirrors the mutation and copy number landscape of the corresponding tumor but also identifies clinically relevant resistance mechanisms and cancer driver alterations not detected in matched tumor biopsies. Longitudinal cfDNA analysis reliably tracks tumor response, progression, and clonal evolution. Sequencing coverage of plasma DNA fragments around transcription start sites showed distinct treatment-related changes and captured the expression of key transcription factors such as NEUROD1 and REST in the corresponding SCLC tumors. This allowed for the prediction of SCLC neuroendocrine phenotypes and treatment responses. Conclusions cfDNA captures a comprehensive view of tumor heterogeneity and evolution. These findings have significant implications for the non-invasive stratification of SCLC, a disease currently treated as a single entity. Supplementary Information The online version contains supplementary material available at 10.1186/s13073-025-01438-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a75bdacf15f78eabb80ad03155a6f7ee66350fd9" target='_blank'>
              Genomic alterations and transcriptional phenotypes in circulating free DNA and matched metastatic tumor
              </a>
            </td>
          <td>
            N. Takahashi, L. Pongor, Shivam P. Agrawal, Mariya Shtumpf, Ankita Gurjar, V. Rajapakse, Ahmad Shafiei, Christopher W. Schultz, Sehyun Kim, Diana Roame, Paula Carter, R. Vilimas, Samantha Nichols, Parth Desai, W. Figg, Mohammad Bagheri, V. B. Teif, Anish Thomas
          </td>
          <td>2025-02-25</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdd747492529d370113dee3945f74bf5ec74e07f" target='_blank'>
              High levels of DNA replication initiation factors indicate ATRi sensitivity via excessive origin firing
              </a>
            </td>
          <td>
            A. Lumeau, PL Pfuderer, S. De Angelis, JA Scarth, MA Guscott, N. Shaikh, FB Copley, H. Gerdes, Emilie L Alard, PR Cutillas, F. Mardakheh, MA Boemo, JV Forment, SE McClelland
          </td>
          <td>2025-02-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Objective RNA-seq delivers valuable insights both to transcriptional patterns and mutational landscapes for transcribed genes. However, as tumour cell lines frequently lack their matched-normal counterpart, variant calling without the paired normal sample is still challenging. In order to exclude variants of common genetic variation without a matched-normal control, filtering strategies need to be developed to identify tumour relevant variants in cell lines. Results Here, variants of 29 breast cancer cell lines were called on RNA-seq data via HaplotypeCaller. Low read depth sites, RNA-edit sites, and low complexity regions in coding regions were excluded. Common variants were filtered using 1000 genomes, gnomAD, and dbSNP data. Starting from hundred thousands of single nucleotide variants and small insertions and deletions, about thousand variants remained after filtering for each sample. Extracted variants were validated against the Catalogue of Somatic Mutations in Cancer (COSMIC) for 10 cell lines included in both data sets. Approximately half of the COSMIC variants were successfully called. Importantly, missing variants could mainly be attributed to sites with low read depth. Moreover, filtered variants also included all 10 cancer gene census COSMIC variants, a condensed hallmark variant set. Supplementary Information The online version contains supplementary material available at 10.1186/s13104-025-07140-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5417aa6262174df782473083114e78a8cbd8f677" target='_blank'>
              Refined variant calling pipeline on RNA-seq data of breast cancer cell lines without matched-normal samples
              </a>
            </td>
          <td>
            Sonja Eberth, Julia Koblitz, Laura Steenpass, C. Pommerenke
          </td>
          <td>2025-02-15</td>
          <td>BMC Research Notes</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Background Fibroblast growth factor 1-4 (FGFR1-4) are well-known oncogenic drivers in many cancer types. Here, we studied the role of FGFRs in uterine leiomyoma (UL) that is a benign neoplasm arising from the myometrium and the most common tumour in women. Although ULs can be classified to molecular subtypes based on genetic drivers, potential secondary drivers are not well characterised. Methods We performed mutation analysis of RNA-sequencing data of ULs, followed by screening of FGFR alterations in our Finnish (n = 2677) and Swedish (n = 372) UL collections, utilising Sanger-, next-generation and Nanopore sequencing and SNP array data. The role of FGFR genes in UL predisposition was examined by GWAS. Results We identified FGFR activation in a subset of ULs on both genetic and epigenetic levels. In addition to single-nucleotide mutations in FGFR1/2, we detected an FGFR2-ERC1 fusion gene, FGFR1 gains and hypomethylation of regulatory regions of FGFR2/3. FGFR alterations were enriched in molecularly similar HMGA2, HMGA1 and PLAG1 UL subtypes. We also unveil a UL predisposing variant upstream of FGFR4 associated with increased expression in both normal myometrium and ULs. Conclusions Our results establish the role of FGFR signalling in the genesis of UL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee3a4ef449036c9f0b9b5728bbd660366041a80a" target='_blank'>
              Activation of FGFR genes by genetic and epigenetic alterations in uterine leiomyomas
              </a>
            </td>
          <td>
            Vilja Jokinen, Aurora Taira, Åsa Kolterud, Isa Ahlgren, Kimmo Palin, Riku Katainen, Maritta Räisänen, Eevi Kaasinen, Sini Ilves, A. Raitila, Helena Kopp Kallner, Emma Siili, Ralf Bützow, O. Heikinheimo, A. Pasanen, A. Karhu, Niko Välimäki, L. Aaltonen
          </td>
          <td>2025-02-27</td>
          <td>BJC Reports</td>
          <td>0</td>
          <td>111</td>
        </tr>

        <tr id="ABSTRACT The non‐coding genome, constituting 98% of human DNA, remains largely unexplored, yet holds potential for identifying new biomarkers and therapeutic targets in acute lymphoblastic leukemia (ALL). In this study, we conducted a systematic analysis of recurrent somatic non‐coding single nucleotide variants (SNVs) in pediatric B‐cell precursor (BCP) ALL. We leveraged whole genome sequencing (WGS) data from 345 pediatric BCP ALL cases, representing all major genetic subtypes and identified 346 mutational hotspots that harbored somatic SNVs in at least three cases. Through the integration of paired RNA sequencing along with published ChIP‐seq and ATAC‐seq data, we found 128 non‐coding hotspots associated with differentially expressed genes nearby, which were enriched for cis‐regulatory elements, demonstrating the effectiveness of multi‐omics integration in distinguishing pathogenic mutations from passengers. We identified one mutational hotspot that was associated with increased expression of the leukemia‐associated gene NRAS in three primary ALLs. Micro‐C analysis in the leukemia cell line demonstrated interactions between the hotspot region and NRAS regulatory elements. Dual luciferase assays indicated that the mutations disrupted regulatory interactions and CRISPR‐mediated deletion of the region significantly upregulated NRAS, confirming the hypothesized regulatory link. Altogether, we provide new insights into the functional roles of non‐coding mutations in leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6089d599cb2b181049865efd98f322d1bb52759" target='_blank'>
              Discovery of Cis‐Regulatory Mechanisms via Non‐Coding Mutations in Acute Lymphoblastic Leukemia
              </a>
            </td>
          <td>
            Efe Aydın, Eleanor L Woodward, Gladys Telliam Dushime, Rebeqa Gunnarsson, H. Lilljebjörn, L. Moura-Castro, T. Fioretos, B. Johansson, K. Paulsson, Minjun Yang
          </td>
          <td>2025-03-01</td>
          <td>Genes, Chromosomes & Cancer</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Abstract Despite the development of HER2‐targeting drugs such as trastuzumab and T‐DXd, treatment resistance is a substantial challenge, often leading to relapse and distant metastasis. Tumor heterogeneity in HER2‐positive breast cancer drives the evolution of resistant clones following therapeutic stress. However, the targetable drivers of anti‐HER2 treatment resistance are not thoroughly identified. This study aims to use neoadjuvant‐targeted therapy cohorts and a patient‐derived organoid in vitro treatment model to uncover the potential targetable drivers of anti‐HER2 treatment resistance. it is found that LINC01235 significantly enhances DNA replication licensing and chromosomal instability, fostering clonal expansion and evolution, and ultimately increasing resistance to therapeutic interventions. LINC01235 regulates global H3K27ac, H3K9ac, and H3K36me3 modifications, promotes H2A.Z expression in regulatory regions, and increases the accessibility of DNA licensing factors to their promoter regions. XRCC5 is identified as a key component for maintaining genomic stability, crucial for LINC01235's role in replication licensing. Furthermore, therapeutic strategies targeting LINC01235, including the use of antisense oligonucleotides or ATR inhibitors, which showed promise in overcoming treatment resistance are explored. These findings underscore the pivotal role of LINC01235 in driving resistance mechanisms and highlight novel avenues for targeted therapies to improve the outcomes of patients with HER2‐positive breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fde95930ed4b5d2e1c4571f86786e616e7f2cf6" target='_blank'>
              LINC01235 Promotes Clonal Evolution through DNA Replication Licensing‐Induced Chromosomal Instability in Breast Cancer
              </a>
            </td>
          <td>
            Qi Zhang, Xuliren Wang, Zhibo Shao, Yi Zhang, Li-Ying Zhang, Ming Chen, Xujie Zhou, Han Zhu, Yue Zhou, Xinya Lu, Pei Li, W. Chi, Lun Li, Zhibo Shao, Shenglin Huang, J. Xue, Y. Chi, Jiong Wu, B. Xiu
          </td>
          <td>2025-02-14</td>
          <td>Advanced Science</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (ecDNA) are commonly produced within the nucleus to drive genome dynamics and heterogeneity, enabling cancer cell evolution and adaptation. However, the mechanisms underlying ecDNA biogenesis remain poorly understood. Here using genome-wide CRISPR screening in human cells, we identified the BRCA1-A and the LIG4 complexes mediate ecDNA production. Following DNA fragmentation, the upstream BRCA1-A complex protects DNA ends from excessive resection, promoting end-joining for circularization. Conversely, the MRN complex, which mediates end resection and thus antagonizes the BRCA1-A complex, suppresses ecDNA formation. Downstream, LIG4 conservatively catalyzes ecDNA production in Drosophila and mammals, with patient tumor ecDNA harboring junctions marked by LIG4 activity. Notably, disrupting LIG4 or BRCA1-A in cancer cells impairs ecDNA-mediated adaptation, hindering resistance to both chemotherapy and targeted therapies. Together, our study reveals the roles of the LIG4 and BRCA1-A complexes in ecDNA biogenesis, and uncovers new therapeutic targets to block ecDNA-mediated adaptation for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65472598115e4d60921b909d76b9f2af808705a0" target='_blank'>
              BRCA1-A and LIG4 complexes mediate ecDNA biogenesis and cancer drug resistance
              </a>
            </td>
          <td>
            Oliver W. Chung, Shun Yao, Fu Yang, Ling Wang, Christian G Cerda-Smith, Haley M Hutchinson, Kris C. Wood, Weijia Su, Mustafa Khasraw, Lee Zou, Dale A. Ramsden, Z. Zhang
          </td>
          <td>2025-02-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="All cancer cells must adopt a telomere maintenance mechanism to achieve replicative immortality. Most human cancer cells utilize the enzyme telomerase to maintain telomeres. Alternative splicing of TERT regulates the amount and function of telomerase, however many alternative splicing isoforms of TERT have unknown functions. Single molecule long read RNA/cDNA sequencing of TERT revealed 45 TERT mRNA variants including 13 known and 32 novel variants. Among the variants, TERT Delta 2–4, which lacks exons 2–4 but retains the original open reading frame, was selected for further study. Induced pluripotent stem cells and cancer cells express higher levels of TERT Delta 2–4 compared to primary human bronchial epithelial cells. Overexpression of TERT Delta 2–4 enhanced clonogenicity and resistance to cisplatin-induced cell death. Knockdown of endogenous TERT Delta 2–4 in Calu-6 cells reduced clonogenicity and resistance to cisplatin. Our results suggest that TERT Delta 2–4 enhances cancer cells’ resistance to cell death. RNA sequencing following knockdown of Delta 2–4 TERT indicates that translation is downregulated and that mitochondrial related proteins are upregulated compared to controls. Overall, our data indicate that TERT produces many isoforms that influence the function of TERT and the abundance and activity of telomerase.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31160a8616dac1a93a1309bf2063539c61bdfcd2" target='_blank'>
              Discovery and characterization of a novel telomerase alternative splicing isoform that protects lung cancer cells from chemotherapy induced cell death
              </a>
            </td>
          <td>
            Jeongjin J. Kim, Alexander Ahn, Jeffrey Y Ying, Andrew T. Ludlow
          </td>
          <td>2025-02-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Two genes are synthetically lethal if loss of function of either one of the two genes does not result in cell death, whereas loss of function of both genes together results in being detrimental to cell survival. This concept has been the basis for developing personalized, precision treatments, which can selectively damage tumor cells and minimize toxicity to normal tissues. Tumor cells often harbor mutations in genes involved in DNA repair pathways, forcing them to switch to alternative repair pathways, leading to chemotherapeutic resistance. These interactions, if targeted, could be synthetically lethal. We aimed to summarize synthetically lethal gene pairs that could be utilized to selectively target cancer cells and minimize side effects on normal tissues. The objective of this review is to study druggable synthetically lethal gene pairs for targeted cancer therapy that have been identified through various genetic screens and functional studies. Methods A systematic literature search will be conducted to extract synthetically lethal gene pairs that can be specifically targeted to cancer cells. Owing to the relatively recent research pertaining to this field, the literature search will incorporate data from 1956. The search will be conducted on PubMed, Web of Science, Embase, and Scopus. The narrative approach will guide the analysis and synthesis of the results. Discussion This review highlights scientific articles that report druggable synthetically lethal gene pairs by testing the efficacy of targeted inhibitors in clonogenic assays. These include research studies that identify synthetically lethal gene pairs detected through CRISPR screens by knocking out one or two genes within the same cell and testing the potency of inhibitors to specifically kill malignant cells. Systematic review registration 10.17605/OSF.IO/5BCW6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a6415589cc03c514ef18895752ecf172aaaab60" target='_blank'>
              Synthetic lethality in cancer: a protocol for scoping review of gene interactions from synthetic lethal screens and functional studies
              </a>
            </td>
          <td>
            Raashi Chauhan, R. Damerla, V. Dhyani
          </td>
          <td>2025-04-08</td>
          <td>Systematic Reviews</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Heterogeneity is inherent to living organisms and it determines cell fate and phenotypic variability. Despite its ubiquity, the underlying molecular mechanisms and the genetic basis linking genotype to-phenotype heterogeneity remain a central challenge. Here we construct a yeast knockout library with a clone and genotype RNA barcoding structure suitable for genome-scale analyses to generate a high-resolution single-cell yeast transcriptome atlas of 3500 mutants under control and stress conditions. We find that transcriptional heterogeneity reflects the coordinated expression of specific gene programs, generating a continuous of cell states that can be responsive to external insults. Cell state plasticity can be genetically modulated with mutants that act as state attractors and disruption of state homeostasis results in decreased adaptive fitness. Leveraging on intra-genetic variability, we establish that regulators of transcriptional heterogeneity are functionally diverse and influenced by the environment. Our multimodal perturbation-based single-cell Genotype-to-Transcriptome Atlas in yeast provides insights into organism-level responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89465a1373d3260110cc521e62c1032cd2c67144" target='_blank'>
              A single-cell resolved genotype-phenotype map using genome-wide genetic and environmental perturbations
              </a>
            </td>
          <td>
            Mariona Nadal-Ribelles, Carme Solé, Anna Díez-Villanueva, Camille Stephan-Otto Attolini, Yaima Matas, Lars M. Steinmetz, Eulàlia de Nadal, F. Posas
          </td>
          <td>2025-03-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) has the highest mortality rate of all breast cancer subtypes and currently lacks effective targeted therapies. LARP6 is an RNA-binding protein associated with cancer promotion, but its mechanism of action in TNBC remains unclear. We conducted RNA sequencing (RNA-seq) and improved RNA immunoprecipitation and sequencing (iRIP-seq) to identify the differentially expressed genes (DEGs) and alternative splice sites bound and regulated by LARP6 in MDA-MB-231 cells. Finally, both RT-qPCR and RIP-qPCR were employed for verification. Our study revealed that LARP6 overexpression altered the expression levels of 171 genes and that the number of regulated alternative splicing events (RASEs) exceeded 1000. The regulated alternative splicing genes (RASGs) corresponding to RASEs were enriched in biological processes such as DNA repair, the cell cycle, and the cellular response to DNA damage stimulus. In addition, we found that LARP6 tends to bind the CGACGAG motif. The intersection of peak-related genes with RASGs suggested that LARP6 can bind to 16 genes and regulate their alternative splicing (AS), thus playing an important role in TNBC progression. Our research indicated that LARP6 may promote the proliferation and invasion of TNBC cells by directly regulating the AS of related genes, providing new clues for targeted therapy for TNBC. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-92351-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7eb087ff58a5d637f585f09e70365a21011c3f17" target='_blank'>
              The RNA-binding protein LARP6 regulates the alternative splicing of related genes in MDA-MB-231 cells
              </a>
            </td>
          <td>
            Li Guo, Yaobang Liu, Shuxun Yan, Hong Li, Kai Zhang, Jinping Li
          </td>
          <td>2025-03-06</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Therapy resistance in breast cancer is increasingly attributed to polyploid giant cancer cells (PGCCs), which arise through whole genome doubling and exhibit heightened resilience to standard treatments. Characterized by enlarged nuclei and increased DNA content, these cells tend to be dormant under therapeutic stress, driving disease relapse. Despite their critical role in resistance, strategies to effectively target PGCCs are limited, largely due to the lack of high-throughput methods for assessing their viability. Traditional assays lack the sensitivity needed to detect PGCC-specific elimination, prompting the development of novel approaches. To address this challenge, we developed a high-throughput single-cell morphological analysis workflow designed to differentiate compounds that selectively inhibit non-PGCCs, PGCCs, or both. Using this method, we screened a library of 2726 FDA Phase 1-approved drugs, identifying promising anti-PGCC candidates, including proteasome inhibitors, FOXM1, CHK, and macrocyclic lactones. Notably, RNA-Seq analysis of cells treated with the macrocyclic lactone Pyronaridine revealed AXL inhibition as a potential strategy for targeting PGCCs. Although our single-cell morphological analysis pipeline is powerful, empirical testing of all existing compounds is impractical and inefficient. To overcome this limitation, we trained a machine learning model to predict anti-PGCC efficacy in silico, integrating chemical fingerprints and compound descriptions from prior publications and databases. The model demonstrated a high correlation with experimental outcomes and predicted efficacious compounds in an expanded library of over 6,000 drugs. Among the top-ranked predictions, we experimentally validated five compounds as potent PGCC inhibitors using cell lines and patient-derived models. These findings underscore the synergistic potential of integrating high-throughput empirical screening with machine learning-based virtual screening to accelerate the discovery of novel therapies, particularly for targeting therapy-resistant PGCCs in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0482d6424427dcdf25109ef995747da0a47de141" target='_blank'>
              High-Throughput Empirical and Virtual Screening To Discover Novel Inhibitors of Polyploid Giant Cancer Cells in Breast Cancer
              </a>
            </td>
          <td>
            Yushu Ma, Chien-Hung Shih, Jinxiong Cheng, Hsiao-Chun Chen, Li-Ju Wang, Yanhao Tan, Yuan Zhang, Daniel D Brown, S. Oesterreich, Adrian V Lee, Yu-Chiao Chiu, Yu-Chih Chen
          </td>
          <td>2025-03-04</td>
          <td>Analytical Chemistry</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Ongoing mutagenesis in cancer drives genetic diversity throughout the natural history of cancers. As the activities of mutational processes are dynamic throughout evolution, distinguishing the mutational signatures of ‘active’ and ‘historical’ processes has important implications for studying how tumors evolve. This can aid in understanding mutagenic states at the time of presentation, and in associating active mutational process with therapeutic resistance. As bulk sequencing primarily captures historical mutational processes, we studied whether ultra-low-coverage single-cell whole-genome sequencing (scWGS), which measures the distribution of mutations across hundreds or thousands of individual cells, could enable the distinction between historical and active mutational processes. While technical challenges and data sparsity have limited mutation analysis in scWGS, we show that these data contain valuable information about dynamic mutational processes. To robustly interpret single nucleotide variants (SNVs) in scWGS, we introduce ArtiCull, a method to identify and remove SNV artifacts by leveraging evolutionary constraints, enabling reliable detection of mutations for signature analysis. Applying this approach to scWGS data from pancreatic ductal adenocarcinoma (PDAC), triple-negative breast cancer (TNBC), and high-grade serous ovarian cancer (HGSOC), we uncover temporal and spatial patterns in mutational processes. In PDAC, we observe a temporal increase in mismatch repair deficiency (MMRd). In cisplatin-treated TNBC patient-derived xenografts, we identify therapy-induced mutagenesis and inactivation of APOBEC3 activity. In HGSOC, we show distinct patterns of APOBEC3 mutagenesis, including late tumor-wide activation in one case and clade-specific enrichment in another. Additionally, we detect a clone-specific increase in SBS17 activity, in a clone previously linked to recurrence. Our findings establish ultra-low-coverage scWGS as a powerful approach for studying active mutational processes that may influence ongoing clonal evolution and therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed6d1d9f289244977b2dfb8fbeaff86f943a227d" target='_blank'>
              Inferring active mutational processes in cancer using single cell sequencing and evolutionary constraints
              </a>
            </td>
          <td>
            Gryte Satas, Matthew A. Myers, Andrew McPherson, Sohrab P. Shah
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="The chromosome 5p15.33 region, which encodes telomerase reverse transcriptase (TERT), harbors multiple germline variants identified by genome-wide association studies (GWAS) as risk for some cancers but protective for others. Here, we characterize a variable number tandem repeat within TERT intron 6, VNTR6-1 (38-bp repeat unit), and detect a strong link between VNTR6-1 alleles (Short: 24-27 repeats, Long: 40.5-66.5 repeats) and GWAS signals rs2242652 and rs10069690 within TERT intron 4. Bioinformatics analyses reveal that rs10069690-T allele increases intron 4 retention while VNTR6-1-Long allele expands a polymorphic G-quadruplex (G4, 35-113 copies) within intron 6, with both variants contributing to variable TERT expression through alternative splicing and nonsense-mediated decay. In two cell lines, CRISPR/Cas9 deletion of VNTR6-1 increases the ratio of TERT-full-length (FL) to the alternative TERT-β isoform, promoting apoptosis and reducing cell proliferation. In contrast, treatment with G4-stabilizing ligands shifts splicing from TERT-FL to TERT-β isoform, implicating VNTR6-1 as a splicing switch. We associate the functional variants VNTR6-1, rs10069690, and their haplotypes with multi-cancer risk and age-related telomere shortening. By regulating TERT splicing, these variants may contribute to fine-tuning cellular longevity and replicative potential in the context of stress due to tissue-specific endogenous and exogenous exposures, thereby influencing the cancer risk conferred by this locus.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3d3384fa00e95094541c65edba3285e24efeab9" target='_blank'>
              Genetic regulation of TERT splicing affects cancer risk by altering cellular longevity and replicative potential
              </a>
            </td>
          <td>
            Oscar Florez-Vargas, Michelle Ho, Maxwell H Hogshead, B. Papenberg, Chia-Han Lee, Kaitlin Forsythe, Kristine Jones, Wen Luo, Kedest Teshome, Cornelis Blauwendraat, Kimberly J Billingsley, M. Kolmogorov, Melissa Meredith, B. Paten, Raj Chari, Chi Zhang, John S Schneekloth, M. Machiela, S. Chanock, S. Gadalla, S. Savage, Sam M Mbulaiteye, Ludmila Prokunina-Olsson
          </td>
          <td>2025-02-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="Glioblastoma, one of the most aggressive and heterogeneous malignant tumors, presents significant challenges for clinical management due to its cellular and metabolic complexity. This review integrates recent advancements in single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics to elucidate glioblastoma’s cellular heterogeneity and metabolic reprogramming. Diverse cellular subpopulations, including malignant proliferative cells, stem-like cells, mesenchymal-like cells, and immune-related cells, contribute to tumor progression, treatment resistance, and microenvironmental interactions. Spatial transcriptomics has further revealed distinct spatial distributions of these subpopulations, highlighting differences in metabolic activities between the tumor core and periphery. Key metabolic adaptations, such as enhanced glycolysis, fatty acid oxidation, and glutamine metabolism, play critical roles in supporting tumor growth, immune evasion, and therapeutic resistance. Targeting these metabolic pathways, especially in combination with immunotherapy, represents a promising avenue for glioblastoma treatment. This review emphasizes the importance of integrating single-cell and spatial multi-omics technologies to decode glioblastoma’s metabolic landscape and explore novel therapeutic strategies. By addressing current challenges, such as metabolic redundancy and spatiotemporal dynamics, this work provides insights into advancing precision medicine for glioblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea52e3e92b1a5c4c3caddeca60f128feaaaf952b" target='_blank'>
              Single-cell and spatial transcriptomic insights into glioma cellular heterogeneity and metabolic adaptations
              </a>
            </td>
          <td>
            Yixin Fu, Yong Yi, Yongxiang Shao, Jingcheng Jiang, Qingshan Deng
          </td>
          <td>2025-04-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Hepatocellular carcinoma is the most prevalent primary liver cancer and arises from hepatocytes, which are very metabolically active and contain abundant mitochondria. Despite extensive characterisation of nuclear driver genes and pathways, the role of crosstalk between the mitochondrial and nuclear genomes in mitochondrial mutagenesis remains poorly understood. To address this, we leveraged a chemical carcinogenesis model of liver cancer in four mouse strains. We analysed hundreds of tumours with paired whole-genome sequencing and total RNA sequencing to comprehensively characterise mitochondrial mutations and expression. Following de novo assembly of three new mitochondrial genomes, we devised an accurate and efficient heteroplasmy detection approach and analysed mitochondrial DNA (mtDNA) content and transcription. Across strains, there was high concordance of the number of heteroplasmies, mutation signatures, and variant allele frequencies. In contrast, the heteroplasmy rate and genic locations differed between strains, suggesting strain-specificity in DNA repair mechanisms in early tumour development. Independent of age or environmental factors, tumours had lower mtDNA content than adjacent normal liver, suggesting mitochondrial loss during tumourigenesis. Additionally, within individual strains, mtDNA content differences were associated with nuclear driver gene choice. Our comprehensive characterisation of mitochondrial genomes and expression patterns demonstrates that both germline and somatic genetic variation influence tumour mtDNA content, mutation patterns, and gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7507e40e7369ffa2dd7aab3b39dffb6074ecd240" target='_blank'>
              Germline and somatic variation influence nuclear-mitochondrial crosstalk in tumourigenesis
              </a>
            </td>
          <td>
            Maëlle Daunesse, Vasavi Sundaram, Elissavet Kentepozidou, John Connelly, Craig J. Anderson, Núria López-Bigas, Colin A. Semple, D. T. Odom, Martin S. Taylor, Maša Roller, Sarah J. Aitken, Paul Flicek
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>94</td>
        </tr>

        <tr id="Cancer is a complex and highly lethal disease marked by unchecked cell proliferation, aggressive behavior, and a strong tendency to metastasize. Despite significant advancements in cancer diagnosis and treatment, challenges such as early detection difficulties, drug resistance, and adverse effects of radiotherapy or chemotherapy continue to threaten patient survival. MicroRNAs (miRNAs) have emerged as critical regulators in cancer biology, with miR-506 being extensively studied and recognized for its tumor-suppressive effects across multiple cancer types. This review examines the regulatory mechanisms of miR-506 in common cancers, focusing on its role in the competing endogenous RNA (ceRNA) network and its effects on cancer cell proliferation, apoptosis, and migration. We also discuss the potential of miR-506 as a therapeutic target and its role in overcoming drug resistance in cancer treatment. Overall, these insights underscore the therapeutic potential of miR-506 and its promise in developing novel cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06c1c0de824cb3b6994467264a7d64c1f9bef206" target='_blank'>
              Therapeutic potential of microRNA-506 in cancer treatment: mechanisms and therapeutic implications
              </a>
            </td>
          <td>
            Shuzhen Mao, Junyan Li, Jiahui Huang, Lili Lv, Qilian Zhang, Qing Cheng, Xiaojing Liu, Zhiwei Bi, Jing Yao
          </td>
          <td>2025-04-03</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9f68c71be9a5f4d5ac85f15c79176d7904112ca" target='_blank'>
              Long-read single-cell RNA sequencing enables the study of cancer subclone-specific genotypes and phenotypes in chronic lymphocytic leukemia.
              </a>
            </td>
          <td>
            Gage S. Black, Xiaomeng Huang, Yi Qiao, Philip Moos, Deepa Sampath, Deborah M. Stephens, J. Woyach, Gabor T. Marth
          </td>
          <td>2025-02-18</td>
          <td>Genome research</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Abstract Adenosine base editors (ABEs) facilitate A·T to G·C base pair conversion with significant therapeutic potential for correcting pathogenic point mutations in human genetic diseases, such as sickle cell anemia and β-thalassemia. Unlike CRISPR–Cas9 systems that induce double-strand breaks, ABEs operate through precise deamination, avoiding chromosomal instability. However, the off-target editing effects of ABEs remain inadequately characterized. In this study, we present a biochemical method Selict-seq, designed to evaluate genome-wide off-target editing by ABEs. Selict-seq specifically captures deoxyinosine-containing single-stranded DNA and precisely identifies deoxyadenosine-to-deoxyinosine (dA-to-dI) mutation sites, elucidating the off-target effects induced by ABEs. Through investigations involving three single-guide RNAs, we identified numerous unexpected off-target edits both within and outside the protospacer regions. Notably, ABE8e(V106W) exhibited distinct off-target characteristics, including high editing rates (>10%) at previously unreported sites (e.g. RNF2 and EMX1) and out-of-protospacer mutations. These findings significantly advance our understanding of the off-target landscape associated with ABEs. In summary, our approach enables an unbiased analysis of the ABE editome and provides a widely applicable tool for specificity evaluation of various emerging genome editing technologies that produce intermediate products as deoxyinosine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c7f86cbbb409d5234a83d13231df0efe7d59590" target='_blank'>
              Selict-seq profiles genome-wide off-target effects in adenosine base editing
              </a>
            </td>
          <td>
            Kexin Yuan, Xin Xi, Shaoqing Han, Jingyu Han, Bin Zhao, Qi Wei, Xiang Zhou
          </td>
          <td>2025-04-10</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60dc48b94558a55ad5ff2dbcdf85170057c59b44" target='_blank'>
              Natural resistance to cancers in long-lived mammals: genomic mechanisms and experimental evidence to explain Peto's paradox.
              </a>
            </td>
          <td>
            Linxia Sun, Zhikang Xu, Mengqi Shuai, Chengxu Li, Guang Yang, Shixia Xu
          </td>
          <td>2025-03-21</td>
          <td>Science China. Life sciences</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Simple Summary Proto-oncogenes and tumor-suppressor genes play important roles in preventing the development of tumors. The former supports healthy cell growth and division, while the latter regulates cell division. Mutations in proto-oncogenes, converting them into oncogenes, and mutations in tumor-suppressor genes can lead to uncontrolled cell proliferation and cancer development. This review explores key molecular pathways associated with these genes, highlighting their influence on cancer progression and treatment resistance, which are crucial for creating effective, personalized, and targeted treatments for patients. This comprehensive review also discusses the recent advancements and emerging technologies that enhance the analysis, diagnosis, prevention, and treatment of cancer at the genomic level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4acb0abcdb89e4becb0be9ae1ca6990d3d6cdf49" target='_blank'>
              Translational Advances in Oncogene and Tumor-Suppressor Gene Research
              </a>
            </td>
          <td>
            R. Stojchevski, Edward Agus Sutanto, Rinni Sutanto, N. Hadzi-Petrushev, M. Mladenov, Sajal Raj Singh, J. K. Sinha, Shampa Ghosh, Bhuvaneshwar Yarlagadda, Krishna Kumar Singh, Prashant Verma, Sonali Sengupta, Rakesh Bhaskar, Dimiter B Avtanski
          </td>
          <td>2025-03-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Background Schlafen11 (SLFN11) is a key gene in p53-independent apoptosis through ribosome stalling; however, systematic research has been conducted on its role in the tumor immune microenvironment, clinical application, and immunotherapy response across pan-cancer. Method Public data were downloaded and multi-omics approaches were used to investigate the relationship between the expression level of SLFN11 and spatial position, biological function, immune landscape, and clinical application values. Cell Counting Kit-8 assay and quantitative real-time PCR were used to validate the expression level of SLFN11 and drug sensitivity in colorectal cancer samples. Result Our study revealed that SLFN11 was downregulated in most cancers and correlated with DNA repair, the P53 pathway and immune response in tumor development progress by multi-omics analysis. Dysregulated SLFN11 is accompanied by several immune cell infiltrations and immune-related regulators, which can be a promising screening and prognostic biomarker and chemotherapy predictive target for clinical application. In vitro experiments proved that downregulated SLFN11 is a useful diagnostic biomarker and is linked to imatinib resistance in colorectal cancer. Conclusion The expression level of SLFN11 has a substantial promise as a valuable biomarker for diagnosis and a predictive indicator for assessing the effectiveness of chemotherapy and immunotherapy in human cancers, which deserves further additional basic experiments and clinical trials to prove.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c098d42efd569f7e477b717e0186198035c93eb9" target='_blank'>
              Comprehensive pan-cancer analysis indicates key gene of p53-independent apoptosis is a novel biomarker for clinical application and chemotherapy in colorectal cancer
              </a>
            </td>
          <td>
            Jianing Yan, Jingzhi Wang, Min Miao, Y. Shao
          </td>
          <td>2025-03-27</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Background Acute myeloid leukemia (AML) is the most common type of acute leukemia, accounting for 20% of cases in children and adolescents. Genome-wide studies have identified genes that are commonly mutated in AML, including many epigenetic regulators involved in either DNA methylation (DNMT3A, TET2, IDH1/2) or histone post-translational modifications (ASXL1, EZH2, MLL1). Several cell lines derived from AML patients are widely used in cancer research. Whether important differences in these cell lines exist remains poorly characterised. Results Here, we used RNA sequencing (RNA-Seq) to contrast the transcriptome of four commonly used AML-derived cell lines: THP-1, NOMO-1, MOLM-13 bearing the common initiating t(9;11) translocation, and MV4.11 bearing the t(4;11) translocation. Gene set enrichment analyses and comparison of key transcription and epigenetic regulator genes revealed important differences in the transcriptome, distinguishing these AML models. Among these, we found striking differences in the expression of clusters of genes located on chromosome 19 encoding Zinc Finger (ZNF) transcriptional repressors. Low expression of many ZNF genes within these clusters is associated with poor survival in AML patients. Conclusion The present study offers a valuable resource by providing a detailed comparative characterisation of the transcriptome of cell lines within the same AML subtype used as models for leukemia research. Supplementary Information The online version contains supplementary material available at 10.1186/s12864-025-11415-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a934cae686db7528797f8dea84b54367fe76eeb" target='_blank'>
              Transcriptomic characterisation of acute myeloid leukemia cell lines bearing the same t(9;11) driver mutation reveals different molecular signatures
              </a>
            </td>
          <td>
            Elise Georges, William Ho, Miren Urrutia Iturritza, L. Eory, Kamila Malysz, U. Sobhiafshar, Alan L. Archibald, Daniel J. Macqueen, Barbara Shih, David Garrick, Douglas Vernimmen
          </td>
          <td>2025-03-25</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56c8bae667d5d467ee9b1edf8b1b3bbcf92dc895" target='_blank'>
              Single-cell sequencing for lung cancer research: Progress and prospects
              </a>
            </td>
          <td>
            Zhongsha Li, Wei Su, Bingbing Bai, Yuesong Wu, Baoyin Zhang, Dejuan Xiang, Gilwa Borjigin, Jinhai Deng, Jian Sun, Mingzhu Yin
          </td>
          <td>2025-03-12</td>
          <td>Eurasian Journal of Medicine and Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Purpose Small cell carcinoma of the ovary, hypercalcemic type (SCCOHT) is a rare and lethal cancer lacking effective treatment. Its genomic mutations and tumor microenvironment need further exploration. Methods We performed whole-exome sequencing or gene panel test to explore the SMARCA4 mutation spectrum in SCCOHT (15 samples). Single-cell RNA sequencing was conducted on one primary lesion with matched normal ovarian tissue and one recurrent lesion to investigate the intra-tumoral heterogeneity and immune microenvironment. Multiplex immunofluorescence staining validated T cell infiltration and PD-1 expression. Results 13/15 (86.7%) patients harbored SMARCA4 mutations. The loss of heterozygosity (LOH) occurred in 10/15 (66.7%) patients. Cancer cells and immune cells were observed in SCCOHT tumors. Cancer cells were further divided into seven subtypes and one from recurrent lesion exhibited the highest stemness accompanied by high expression of genes related to cell mitosis (AURKB, CHEK2, CCNB1, WEE1), DNA repair (BRCA1, RAD51) and epigenetic (EZH2, DNMT1). Immune cells mainly included macrophages and T cells. Lipid-associated tumor-associated macrophages (TAMs) was mainly in primary lesion while inflammatory cytokine-enriched TAMs in recurrent lesion. CD4+/ CD8+ T cell infiltration was observed in SCCOHT tumor and a certain proportion of T cells expressed PD-1. Conclusions SCCOHT exhibits universal SMARCA4 LOH and significant intra-tumoral heterogeneity, suggesting potential therapeutic targets, including CHEK2, CCNB1, and WEE1. Exhausted T cells and distinct TAM subsets infiltrate tumors. Targeting macrophage polarization or cytokine signaling may also be promising. These findings provide insights for developing novel therapies to improve outcomes in SCCOHT. Clinical trial number Not applicable. Supplementary Information The online version contains supplementary material available at 10.1186/s13048-025-01649-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdba27a2b17d1b242308fa85ab562553932c973d" target='_blank'>
              Single-cell RNA sequencing reveals the intra-tumoral heterogeneity and immune microenvironment of small cell carcinoma of the ovary, hypercalcemic type
              </a>
            </td>
          <td>
            Yijun Gao, Kewei Zheng, Haowen Tan, Mingyi Kang, Bingjian Lu, Ling Chen, Jing Xu, Chong Lu, Ranran Chai, Congjian Xu, Yu Kang
          </td>
          <td>2025-04-09</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Aim: As intrinsic resistance - often driven by concurrent genomic alterations in tumor suppressor genes or oncogenes - remains a major challenge in oncology, this work aimed to comprehensively analyze BRAF somatic alterations across cancer types and identify new potential therapeutic strategies to overcome drug resistance. Methods: We conducted an extensive analysis of genomics, transcriptomics, and clinical data retrieved from public repositories, including cBioPortal. Our comprehensive analysis examined BRAF alterations [point mutations, structural variants (SVs) and copy number alteration] in more than 217,000 tumor samples across 120 distinct tumor types from primary and metastatic sites in both adult and pediatric cohorts, focusing on mutual exclusivity and co-occurrence of mutations in other oncogenes or tumor suppressors. The work also explores the association of BRAF somatic alterations with survival, clinical and molecular features. Results: Analysis of mutation frequencies across cancer types revealed that BRAFV600E represents approximately 90% of all BRAF alterations. While melanoma and thyroid carcinoma show the highest prevalence of BRAF mutations, followed by colorectal and non-small cell lung cancer in terms of absolute number of patients harboring BRAF mutations worldwide, notably high mutation frequencies were identified in rare malignancies, including hairy-cell leukemia, ganglioglioma, and serous borderline ovarian tumors. The comprehensive analysis of genomic profiling data across these tumors uncovered distinct patterns of co-occurring and mutually exclusive alterations in oncogenes and tumor suppressor genes, illuminating resistance mechanisms and suggesting novel therapeutic combinations. Conclusion: Comprehensive genomic profiling is critical for optimizing targeted therapy and overcoming drug resistance in BRAF-mutated cancers. The identification of co-occurring alterations provides opportunities for rational combination therapies, emphasizing the importance of detailed mutation profiling in developing effective treatment strategies across diverse cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7fa7f528d2835416489339d23ab2a839922e3b0" target='_blank'>
              Unraveling BRAF alterations: molecular insights to circumvent therapeutic resistance across cancer types
              </a>
            </td>
          <td>
            Caterina Perfetto, Marianna Aprile, Simona Cataldi, Elisa Giovannetti, Valerio Costa
          </td>
          <td>2025-03-24</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/572fd6e1dc6d845d6923c7249ae200e2136415b1" target='_blank'>
              Genome-wide transcriptomic response of whole blood to radiation
              </a>
            </td>
          <td>
            Ahmed Salah, D. Wollschläger, Maurizio Callari, H. Schmidberger, Federico Marini, S. Zahnreich
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Our study aimed to explore how DEK, a carcinogenic protein with chromatin architectural function, genome-widely binds to RNA and affects the alternative splicing in cancer cells to decipher its molecular functions. To achieve this goal, cell phenotype experiments, RNA sequencing (RNA-seq), and improved RNA immunoprecipitation sequencing (iRIP-seq) were conducted to identify the function and regulated targets of DEK in HeLa cells. The results showed DEK overexpression promoted cell proliferation and invasion of HeLa cells. Meanwhile, DEK hardly affected transcript level expression of those high expressed genes, but splicing pattern of 411 genes was regulated by DEK in HeLa cells, which were enriched in Hippo signaling pathway. Moreover, DEK broadly bind the RNA of a total of 11, 112 genes, with a biased binding the 5’ splice site (5’SS) consensus GGUAA motifs at the CDS and intronic regions. In addition, 297 DEK-binding genes showed different splicing pattern after DEK overexpression in HeLa cells. These genes were enriched in Hippo signaling pathway including CSNK1D. The RT-qPCR and RIP-PCR confirmed that DEK can bind to CSNK1D to regulate its alternative splicing in HeLa cells. In summary, our results indicated DEK could broadly bind and regulate the pre-mRNA splicing process, which provide new insights of mechanisms that DEK functions in various biological processes including cancer. Supplementary Information The online version contains supplementary material available at 10.1007/s00438-025-02226-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5d5217672991275c9b6fa039999b503a8b6fb11" target='_blank'>
              Proto-oncogene DEK binds to pre-mRNAs and regulates the alternative splicing of Hippo signaling genes in HeLa cells
              </a>
            </td>
          <td>
            Dongbo Liu, Wei Sun, Jing Han, Cong Wang, Dong Chen, Yunfei Wu, Yongjie Chang, Bin Yang
          </td>
          <td>2025-03-13</td>
          <td>Molecular Genetics and Genomics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD) remains a leading cause of cancer-related mortalities, characterized by substantial genetic heterogeneity that challenges a comprehensive understanding of its progression. This study employs next-generation sequencing data analysis to transform our comprehension of LUAD pathogenesis. Integrating epigenetic and transcriptomic data of LUAD patients, this approach assessed the critical regulatory occurrences, identified therapeutic targets, and offered profound insights into cancer molecular foundations. We employed the DNA methylation data to identify differentially methylated CpG sites and explored the transcriptome profiles of their adjacent genes. An intersectional analysis of gene expression profiles uncovered 419 differentially expressed genes (DEGs) influenced by smoke-induced differential DNA methylation, among which hub genes, including mitochondrial ribosomal proteins (MRPs), and ribosomal proteins (RPs) such as MRPS15, MRPS5, MRPL33, RPL24, RPL7L1, MRPL15, TUFM, MRPL22, and RSL1D1, were identified using a network-based approach. These hub genes were overexpressed and enriched to RNA processing, ribosome biogenesis, and mitochondrial translation, which is critical in LUAD progression. Enhancer Linking Methylation/Expression Relationship (ELMER) analysis revealed transcription factor (TF) binding motifs, such as JUN, NKX23, FOSB, RUNX3, and FOSL1, which regulated these hub genes through methylation-dependent enhancer dynamics. Predominant hypomethylation of MRPs and RPs disrupted mitochondrial function, contributed to oxidative phosphorylation (OXPHOS) and metabolic reprogramming, favoring cancer cell survival. The survival analysis validated the clinical relevance of these hub genes, with high-expression cohorts exhibiting poor overall survival (OS) outcomes enlightened their relevance in LUAD pathogenesis and presented the potential for developing novel targeted therapeutic strategies. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-93769-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f19b44fa19b901c527db0c43e3ead70766aaf9b" target='_blank'>
              Unraveling the intricate molecular landscape and potential biomarkers in lung adenocarcinoma through integrative epigenomic and transcriptomic profiling
              </a>
            </td>
          <td>
            Arnab Mukherjee, Manon Boonbangyang, Mukunthan K.S.
          </td>
          <td>2025-03-17</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract CRISPR/Cas9-mediated homology-directed repair (HDR) allows precise gene editing, but its efficiency remains low for certain cell types, such as human induced pluripotent stem cells (hiPSCs). In this study, we aimed to introduce the CTNNA1: c.2023C>T (p.Q675*) genetic alteration, which is associated with Hereditary Diffuse Gastric Cancer, into hiPSCs using CRISPR/Cas9. We designed a single-guide RNA targeting the alteration site and a single-stranded oligonucleotide donor DNA template for HDR-based repair. Herein, we report the successful introduction of the CTNNA1: c.2023C>T homozygous alteration in one hiPSC line, which resulted in severe phenotypic changes, including impaired colony formation and cell proliferation. Additionally, we established a straightforward protocol to assess hiPSCs karyotype integrity, ensuring the chromosomal stability required for the gene-editing process. This protocol involves routine G-banding analysis that is required for regular quality controls during handling of hiPSCs. This study demonstrates an efficient approach to precisely edit hiPSCs by CRISPR/Cas9 and highlights the essential role of CTNNA1 expression in maintaining hiPSC viability. Our methodology provides a valuable framework for modeling disease-associated alterations in human-derived cellular models that can be reproduced for other genes and other types of cell lines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77b6de2d94791247728d12901bea9b29daa88c32" target='_blank'>
              A protocol for karyotyping and genetic editing of induced pluripotent stem cells with homology-directed repair mediated CRISPR/Cas9
              </a>
            </td>
          <td>
            Silvana Lobo, Rita Barbosa-Matos, Sofia Dória, A. Pedro, Ana Brito, D. Ferreira, Carla Oliveira
          </td>
          <td>2025-02-28</td>
          <td>Biology Methods & Protocols</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Mapping genetic interactions (GIs) is crucial for understanding genetic network complexity. In this study, we investigate the utility of Cas13d, a CRISPR system targeting RNA, for GI mapping and compare it to Cas9 and Cas12a, two DNA nucleases commonly used for GI mapping. We find that Cas13d induces faster target gene perturbation and generates more uniform cell populations with double perturbations than Cas9 or Cas12a. We then encounter Cas13d gRNA-gRNA interference when concatenating gRNAs targeting different genes into one gRNA array, which we overcome by a dual promoter gRNA expression strategy. Moreover, by concatenating three gRNAs targeting the same gene into one array, we are able to maximize the Cas13d-mediated knockdown effects. Combining these strategies enhances proliferation phenotypes while reducing library size and facilitates reproducible quantification of GIs in oncogenic signaling pathways. Our study highlights the potential of Cas13d for GI mapping, promising advancements in understanding therapeutically relevant drug response pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ef2ae6c7368d6a4d9d50c4f417829a563bfed3b" target='_blank'>
              Evaluation of Cas13d as a tool for genetic interaction mapping
              </a>
            </td>
          <td>
            Ghanem El Kassem, Jasmine Hillmer, Michael Boettcher
          </td>
          <td>2025-02-14</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b66fc40e62ecd1d0cde61c3ba23696dc636af88" target='_blank'>
              Longitudinal Profiling of Circulating Tumor DNA Reveals the Evolutionary Dynamics of Metastatic Prostate Cancer During Serial Therapy.
              </a>
            </td>
          <td>
            Yuehui Zhao, Naveen Ramesh, Ping Xu, Emi Sei, Min Hu, S. Bai, Patricia Troncoso, Ana M Aparicio, Christopher J. Logothetis, Paul G Corn, Nicholas E Navin, A. Zurita
          </td>
          <td>2025-02-24</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="ABSTRACT ALK fusions, such as the classic EML4‐ALK, are known drivers of lung cancer and effective therapeutic targets. However, variant ALK fusions, including intergenic fusions like LOC388942‐ALK (LA), have been detected in increasing numbers of patients, with their roles in tumorigenesis and ALK inhibitor resistance remaining unclear. Using CRISPR/Cas9, we generated the LA fusion in A549 and H441 cells, confirming elevated ALK expression via qRT‐PCR and immunohistochemistry (IHC) staining. Functional analyses showed that LA significantly promoted tumor growth in vitro and in vivo while conferring increased resistance to alectinib. RNA‐seq revealed upregulation of the FOS pathway in LA tumors, identifying FOS as a potential therapeutic target. Subsequently, we demonstrated that FOS disruption and inhibition sensitized LA tumors to treatment. RNA‐seq profiling demonstrated that FOS depletion in LOC388942‐ALK tumor significantly downregulated multiple oncogenic pathways related to cell cycle progression, DNA replication fidelity, and extracellular matrix remodeling, suggesting a pivotal role of FOS in maintaining tumor growth. These findings establish LOC388942‐ALK as a novel oncogenic driver in lung cancer, highlighting its role in tumor growth and ALK inhibitor resistance. Targeting FOS may provide a promising therapeutic strategy for tumors harboring this intergenic fusion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/459d6322ba2d77e523434dfb3f269f42f6deaabb" target='_blank'>
              Identifying the Intergenic ALK Fusion LOC388942‐ALK as a Driver of Non–Small Cell Lung Cancer
              </a>
            </td>
          <td>
            X. Zhai, Manli Wang, Qi Zhang, Donglin Li, Yanmou Wu, Zuoyu Liang, Jiewei Liu, Weiya Wang, Yu Liu, Guowei Che, Qinghua Zhou, Chong Chen
          </td>
          <td>2025-03-27</td>
          <td>MedComm</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background Lung cancer is one of the most commonly diagnosed cancers. N6-methyladenosine (m6A) modification has a profound impact on RNA translation, splicing, transportation, and stability. Aims This research aimed to identify and verify m6A-modified signatures for Lung adenocarcinoma (LUAD) tumorigenesis. Objective Our previous mRNA-seq and m6A-seq data from 26 pairs of LUAD samples and tumor-adjacent normal tissues are used. Methods Univariate Cox regression analysis and the least absolute shrinkage and selection operator (LASSO) analysis were used to estimate the significance of 37 collected m6A regulators. WGCNA was constructed to identify the genes correlated with LUAD tumorigenesis. Pearson correlation analysis between mRNA-seq and m6A-seq data was used to identify the m6A-correlation genes. Results LASSO-Cox analysis identified 18 m6A significant regulators. The top 3 regulators, including METTL16, FTO, and SRSF10, and their downstream genes which were reported in the literature were analysed to confirm their role in LUAD tumorigenesis. Blue and brown coexpression modules were chosen as key modules for LUAD tumorigenesis. At last, we intersected Lasso-downstream genes, m6A-correlation genes, with blue or brown module genes. As a result, 56 m6A-modified gene signatures were obtained. Among them, AKAP9, PLXNB2, BRPF3, HPS4, EXOC7, and KLF6 have an inconsistent expression in protein and mRNA levels, probably due to m6A modification. In addition, these genes may be involved in regulating drug resistance. Conclusions 56 m6A-modified gene signatures for LUAD tumorigenesis were obtained from Pearson correlation analysis between mRNA-seq and m6A-seq data, along with LASSO and WGCNA analysis. Among them, AKAP9, PLXNB2, BRPF3, HPS4, EXOC7 and KLF6 play a crucial role in LUAD tumorigenesis in an m6A modification-dependent manner. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-02106-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8a1734ebf3b7cc16ea04d4fbcf6ec4bc799a0c4" target='_blank'>
              Identification of m6A-modified gene signatures in lung adenocarcinoma tumorigenesis and their potential role in drug resistance
              </a>
            </td>
          <td>
            Xiaomin Han, Qiang Ma, Ruyi Chang, Siyuan Xin, Guojun Zhang, Ruilong Wang, Yukun Wang
          </td>
          <td>2025-03-25</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/654c576507b39c0969a5f79ea0daae0b983731a4" target='_blank'>
              Riding the Wave of Progress: Examining the Current Landscape and Future Potential of MicroRNAs in Cancer Gene Therapy.
              </a>
            </td>
          <td>
            Saleem Ahmad, Hanif Khan, Njoka Irene Muthoni, Mohammed Alissa, Awaji Y. Safhi, F. Y. Sabei, Osama Abdulaziz, Khadijah Hassan Khan, Essam H Ibrahim, S. Khan
          </td>
          <td>2025-03-24</td>
          <td>Current gene therapy</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c926ae9ba30483b00d605482594a078fa10068fc" target='_blank'>
              The landscape of cell lineage tracing.
              </a>
            </td>
          <td>
            Ye Feng, Guang Liu, Haiqing Li, Lin Cheng
          </td>
          <td>2025-02-28</td>
          <td>Science China. Life sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Somatic mutations are particularly relevant for long-lived organisms. Sources of somatic mutations include imperfect DNA repair, replication errors, and exogenous damage such as ultraviolet radiation. A previous study estimated a surprisingly low number of somatic mutations in a 234-year-old individual of the pedunculate oak (Quercus robur), known as the Napoleon Oak. It has been suggested that the true number of somatic mutations was underestimated due to gaps in the reference genome and too conservative filtering of potential mutations. We therefore generated new high-fidelity long-read data for the Napoleon Oak (n = 12) to produce both a pseudo-haploid genome assembly and a partially phased diploid assembly. The high heterozygosity allowed for complete reconstruction of phased and gapless centromeres for 22 of the 24 chromosomes. On the other hand, the high heterozygosity posed challenges for short-read alignments. Use of only the pseudo-haploid assembly as a reference led to potential misalignments, while use of only the diploid assembly reduced variant detection sensitivity. Since most somatic mutations are layer-specific, the observed frequency is expected relatively low, even where all cells in a single layer contain a specific mutation. To address this challenge, we employed a read assignment strategy, selecting the appropriate reference sequence (pseudo-haploid or diploid) based on alignment score and mapping quality. Ultimately, we identified 198 high-confidence somatic mutations, compared with 17 somatic mutations identified before with the same set of short reads. Our approach thus increased the total estimated annual mutation rate by a factor of five.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c62a02d2eaee60312a4cfa06b2def598845faf3" target='_blank'>
              Minimizing detection bias of somatic mutations in a highly heterozygous oak genome
              </a>
            </td>
          <td>
            Wenfei Xian, P. Carbonell-Bejerano, Fernando A. Rabanal, I. Bezrukov, P. Reymond, Detlef Weigel
          </td>
          <td>2025-02-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Brain metastasis represents a significant challenge in oncology, driven by complex molecular and epigenetic mechanisms that distinguish it from primary tumors. While recent research has focused on identifying genomic mutation drivers with potential clinical utility, these strategies have not pinpointed specific genetic mutations responsible for site-specific metastasis to the brain. It is now clear that successful brain colonization by metastatic cancer cells requires intricate interactions with the brain tumor ecosystem and the acquisition of specialized molecular traits that facilitate their adaptation to this highly selective environment. This is best exemplified by widespread transcriptional adaptation during brain metastasis, resulting in aberrant gene programs that promote extravasation, seeding, and colonization of the brain. Increasing evidence suggests that epigenetic mechanisms play a significant role in shaping these pro-brain metastasis traits. This review explores dysregulated chromatin patterns driven by chromatin remodeling, histone modifications, DNA/RNA methylation, and other epigenetic regulators that underpin brain metastatic seeding, initiation, and outgrowth. We provide novel insights into how these epigenetic modifications arise within both the brain metastatic tumor and the surrounding brain metastatic tumor ecosystem. Finally, we discuss how the inherent plasticity and reversibility of the epigenomic landscape in brain metastases may offer new therapeutic opportunities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fe358dc3b7575e837baed5ca2b106b75b4f36a3" target='_blank'>
              The epigenetic landscape of brain metastasis.
              </a>
            </td>
          <td>
            Aoibhín M Powell, Louise Watson, L. Luzietti, S. Prekovic, L. Young, D. Varešlija
          </td>
          <td>2025-02-27</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8354bb2cbdecdc87317ac3be8bce15b2494319c2" target='_blank'>
              Emerging Roles of ACTL6A as an Oncogenic Hub: Transcriptional Regulation and Beyond.
              </a>
            </td>
          <td>
            Kelvin K Tsai, Lin-Hsin Cheng, , Pei-Ming Yang, Chin-Pin Chuu
          </td>
          <td>2025-04-07</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Genome sequencing of cancer and normal tissues, alongside single-cell transcriptomics, continues to produce findings that challenge the idea that cancer is a 'genetic disease', as posited by the somatic mutation theory (SMT). In this prevailing paradigm, tumorigenesis is caused by cancer-driving somatic mutations and clonal expansion. However, results from tumor sequencing, motivated by the genetic paradigm itself, create apparent 'paradoxes' that are not conducive to a pure SMT. But beyond genetic causation, the new results lend credence to old ideas from organismal biology. To resolve inconsistencies between the genetic paradigm of cancer and biological reality, we must complement deep sequencing with deep thinking: embrace formal theory and historicity of biological entities, and (re)consider non-genetic plasticity of cells and tissues. In this Essay, we discuss the concepts of cell state dynamics and tissue fields that emerge from the collective action of genes and of cells in their morphogenetic context, respectively, and how they help explain inconsistencies in the data in the context of SMT.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f9b3a10a05d5290d45b66f194c7384d1f7517bb" target='_blank'>
              The end of the genetic paradigm of cancer.
              </a>
            </td>
          <td>
            Sui Huang, Ana M Soto, C. Sonnenschein
          </td>
          <td>2025-03-01</td>
          <td>PLoS biology</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Head and neck squamous cell carcinoma (HNSCC) remains a prevalent and lethal malignancy, with a five-year survival rate of just 50% for cases of locally advanced disease. Chromosomal aberrations, particularly the deletion of the short arm of chromosome 3 (3p), have been strongly associated with poor prognosis and more aggressive tumor phenotypes. The tumor microenvironment (TME) plays a pivotal role in tumor progression and resistance to therapy. This study aims to elucidate the impact of 3p deletion on the TME, immune cell infiltration, and treatment response in HNSCC, to identify novel therapeutic targets to improve patient outcomes. We analyzed single-cell RNA sequencing (scRNA-seq) data from the Gene Expression Omnibus (GEO) and bulk transcriptome data from The Cancer Genome Atlas (TCGA). Pseudo-time trajectory and cell-cell communication analyses were performed with the Monocle and CellChat packages. The Wilcoxon test was used to evaluate the differential gene expression between wild-type (wt) and mutant (mut) groups. Prognostic models were developed using the Least Absolute Shrinkage and Selection Operator (LASSO) algorithm and Cox regression analyses to find the genes related to survival, with survival analysis conducted via Kaplan-Meier curves. Gene set enrichment analysis (GSEA) was employed to investigate pathway dysregulation, and immune cell infiltration was assessed using various immune scoring methodologies to explore the differences immune environment. The Tumor Immune Dysfunction and Exclusion (TIDE) database was utilized to predict the potential efficacy of immune checkpoint inhibitors. mRNA and protein expression levels of SPP1 were examined by RT-qPCR and Western blotting, while cell proliferation was assessed using the CCK8 assay. The mut group demonstrated significant alterations in cellular composition, characterized by increased endothelial cells and macrophages and decreased fibroblasts and CD8 + T cells, indicative of an immunosuppressive TME. Differential expression analysis revealed downregulation of immune pathways, including antigen processing and presentation, T cell receptor signaling, and B cell receptor signaling pathways in the mut group, along with enhanced metabolic activity in glycolysis and lipid metabolism. The prognostic model identified nine key genes associated with poor survival in HNSCC. The mut group exhibited poorer overall survival and a more immunosuppressive microenvironment compared to the wt group, which correlated with the outcomes observed in high-risk versus low-risk groups. High-risk patients also showed a diminished response to immunotherapy compared to low-risk patients. Additionally, SPP1 emerged as a critical gene associated with chemotherapy resistance and macrophage M2 polarization. This study demonstrates that 3p deletion significantly reshapes the TME, contributing to poor prognosis in HNSCC by fostering an immunosuppressive environment and enhancing chemoresistance. These findings highlight the potential for developing targeted therapies that address the genetic and immunological landscape of HNSCC. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-92078-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55bd28583d34995b3cfd1d2547de4bdf67a2acf7" target='_blank'>
              Integrative single cell transcriptomic analysis reveals 3p deletion associated tumor microenvironment and chemoresistance in head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Xiaochuan Chen, Siqi Xu, Junping Pan, Wenqian Xu, Hanxuan Yang, Xin Chen, Ronghui Chen, Yuan Wang, Sufang Qiu
          </td>
          <td>2025-03-10</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Abstract Functional characterization of genetic variants has the potential to advance the field of precision medicine by enhancing the efficacy of current therapies and accelerating the development of new approaches to combat genetic diseases. MUTYH is a DNA repair enzyme that recognizes and repairs oxidatively damaged guanines [8-oxoguanine (8-oxoG)] mispaired with adenines (8-oxoG·A). While some mutations in the MUTYH gene are associated with colorectal cancer, most MUTYH variants identified in sequencing databases are classified as variants of uncertain significance. Convoluting clinical classification is the absence of data directly comparing homozygous versus heterozygous MUTYH mutations. In this study, we present the first effort to functionally characterize MUTYH variants using precision genome editing to generate heterozygous and homozygous isogenic cell lines. Using a MUTYH-specific lesion reporter in which we site-specifically incorporate an 8-oxoG·A lesion in a fluorescent protein gene, we measure endogenous MUTYH enzymatic activity and classify them as pathogenic or benign. Further, we modify this reporter to incorporate the MUTYH repair intermediate (8-oxoG across from an abasic site) and validate it with co-immunoprecipitation experiments to demonstrate its ability to characterize the mechanism by which MUTYH mutants are defective at DNA repair.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2e7c0e491d2ef8489374e03b9d4bf701b388bba" target='_blank'>
              Precision genome editing and in-cell measurements of oxidative DNA damage repair enable functional and mechanistic characterization of cancer-associated MUTYH variants
              </a>
            </td>
          <td>
            Carlos A Vasquez, Nicola R B Osgood, Marcanthony U Zepeda, Dominika K Sandel, Quinn T. Cowan, Malalage N. Peiris, Daniel J Donoghue, Alexis C. Komor
          </td>
          <td>2025-03-20</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dd1c02bf99dc94856478bd198e131107c59bd8c" target='_blank'>
              KDM5 demethylases suppress R-loop-mediated “viral mimicry” and DNA damage in breast cancer cells
              </a>
            </td>
          <td>
            Lena Lau, Kurt W Henderson, Ahu Turkoz, Sara Linker, Dörte Schlesinger, Brad Townsley, Brian Egan, Shoba Ragunathan, Robert Rollins, Xianju Bi, Zhijian J. Chen, Oleg Brodsky, Clifford Restaino, Murali Gururajan, Kristen Jensen-Pergakes, A. Malarstig, Chames Kermi, Paul C Moore, Marie Classon
          </td>
          <td>2025-03-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Advancements in genomics have revolutionized human genetics by defining the genetic architecture of disease. However, identifying causal variants and their mechanisms of action remains a challenge in translating genetics into therapeutic interventions. Here, we developed Sensitive Transcriptomics And Genotyping by sequencing (STAG-seq), a high-throughput platform designed to define mechanistic genotype-phenotype relationships through simultaneous single-cell measurements of genomic DNA and RNA transcripts. Combined with base-editing, STAG-seq enables functionalization of variants in relevant cellular contexts. We demonstrate the applicability of this approach in several settings. First, we screened genetic perturbations to identify monoallelic and biallelic variant effects in primary human macrophages treated with innate immune stimuli. Next, we phenotyped clinically relevant missense variants associated with immunodeficiency and autoimmunity. Finally, we defined a noncoding variant in a pleiotropic autoimmunity locus that governs TNRC18 expression in primary T cells. STAG-seq thus enables variant phenotyping at scale to advance functional genomics and disease biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47496c4a5205c2d4eec3b8f1bd7dea12812365cb" target='_blank'>
              Sensitive Transcriptomics and Genotyping reveals function of genetic variants in immunity
              </a>
            </td>
          <td>
            Kai Liu, Anders Rasmussen, Wenkai Han, Qiyu Gong, Seneca Bohley, Dawn Chen, M. Kanai, Regan Maronick, Zhangyuan Yin, Chenlei Hu, Tanvi Jain, J. D. Martin-Rufino, Vijay G. Sankaran, Orr Ashenberg, Mark J. Daly, Fei Chen, Daniel B. Graham, Ramnik J. Xavier
          </td>
          <td>2025-03-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e945cd17bb6878accfa941b57523d14ed0d229c" target='_blank'>
              Genomic and transcriptomic sequencing in prostate cancer.
              </a>
            </td>
          <td>
            Safiullah Rifai, Azimullah Rifai, Xiaolei Shi, M. A. Khan, Wei Guang, Linbo Wang, Luke Tallon, Arif Hussain
          </td>
          <td>2025-02-28</td>
          <td>Current opinion in oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6cc4a68bb9eb32114e3b611e03801c7a052fee55" target='_blank'>
              Rational design of alternative treatment options for radioresistant rectal cancer using patient-derived organoids.
              </a>
            </td>
          <td>
            D. Andel, A. J. Nouwens, S. Klaassen, J. Laoukili, B. Viergever, A. Verheem, M. P. W. Intven, M. Zandvliet, J. Hagendoorn, I. Borel Rinkes, O. Kranenburg
          </td>
          <td>2025-04-10</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="ABSTRACT The transcription factor p53 is the most frequently impaired tumor suppressor in human cancers. In response to various stress stimuli, p53 activates transcription of genes that mediate its tumor-suppressive functions. Distinctive characteristics of p53 outlined here enable a well-defined program of genes involved in cell cycle arrest, apoptosis, senescence, differentiation, metabolism, autophagy, DNA repair, anti-viral response, and anti-metastatic functions, as well as facilitating autoregulation within the p53 network. This versatile, anti-cancer network governed chiefly by a single protein represents an immense opportunity for targeted cancer treatment, since about half of human tumors retain unmutated p53. During the last two decades, numerous compounds have been developed to block the interaction of p53 with the main negative regulator MDM2. However, small molecule inhibitors of MDM2 only induce a therapeutically desirable apoptotic response in a limited number of cancer types. Moreover, clinical trials of the MDM2 inhibitors as monotherapies have not met expectations and have revealed hematological toxicity as a characteristic adverse effect across this drug class. Currently, combination treatments are the leading strategy for enhancing efficacy and reducing adverse effects of MDM2 inhibitors. This review summarizes efforts to identify and test therapeutics that work synergistically with MDM2 inhibitors. Two main types of drugs have emerged among compounds used in the following combination treatments: first, modulators of the p53-regulated transcriptome (including chromatin modifiers), translatome, and proteome, and second, drugs targeting the downstream pathways such as apoptosis, cell cycle arrest, DNA repair, metabolic stress response, immune response, ferroptosis, and growth factor signaling. Here, we review the current literature in this field, while also highlighting overarching principles that could guide target selection in future combination treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3895fe8ca5319763195e6c375cf525ba1915bfa" target='_blank'>
              Harnessing p53 for targeted cancer therapy: new advances and future directions
              </a>
            </td>
          <td>
            Z. Andrysik, Joaquin M Espinosa
          </td>
          <td>2025-03-03</td>
          <td>Transcription</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="The hierarchical organization of the eukaryotic genome is crucial for nuclear activities and cellular development. Genetic aberrations can disrupt this 3D genomic architecture, potentially driving oncogenesis. However, current research often lacks a comprehensive perspective, focusing on specific mutation types and singular 3D structural levels. Here, pathological changes from chromosomes to nucleotides are systematically cataloged, including 10 789 interchromosomal translocations (ICTs), 18 863 structural variants (SVs), and 162 769 single nucleotide polymorphisms (SNPs). The multilayered analysis reveals that fewer than 10% of ICTs disrupt territories via potent 3D interactions, and only a minimal fraction of SVs disrupt compartments or intersect topologically associated domain structures, yet these events significantly influence gene expression. Pathogenic SNPs typically show reduced interactions within the 3D genomic space. To investigate the effects of variants in the context of 3D organization, a two-phase scoring algorithm, 3DFunc, is developed to evaluate the pathogenicity of variant-gene pairs in cancer. Using 3DFunc, IGHV3-23's critical role in chronic lymphocytic leukemia is identified and it is found that three pathological SNPs (rs6605578, rs7814783, rs2738144) interact with DEFA3. Additionally, 3DGAtlas is introduced, which provides a highly accessible 3D genome atlas and a valuable resource for exploring the pathological effects of genetic mutations in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/daa65f99b6475aca863361e2ed2a67fe83692271" target='_blank'>
              A 3D Genome Atlas of Genetic Variants and Their Pathological Effects in Cancer.
              </a>
            </td>
          <td>
            Li Tang, M. C. Hill, Mingxing He, Junhao Chen, Zirui Wang, Patrick T. Ellinor, Min Li
          </td>
          <td>2025-03-25</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40c8b8c8967239c4ebdf5921b795eefe1341f9d3" target='_blank'>
              Single-cell eQTL mapping reveals cell subtype-specific genetic control and mechanism in malignant transformation of colorectal cancer.
              </a>
            </td>
          <td>
            Can Chen, Yimin Cai, Wenlong Hu, Kai Tan, Zequn Lu, Xu Zhu, Ziying Liu, Chunyi He, Guangping Xu, Ruizhe Zhang, Caibo Ning, Shuheng Ruan, Jiayan Gao, Xiaojun Yang, Yongchang Wei, Xu Zhu, Xiangpan Li, Faxi Wang, Fubing Wang, Jiaoyuan Li, Meng Jin, Bin Li, Ying Zhu, Jianbo Tian, Xiaoping Miao
          </td>
          <td>2025-03-04</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29817e74ca4bd94e1017f077991d98ef4dbbdc5b" target='_blank'>
              De novo detection of somatic variants in high-quality long-read single-cell RNA sequencing data.
              </a>
            </td>
          <td>
            Arthur Dondi, Nico Borgsmüller, Pedro F. Ferreira, Brian J. Haas, Francis Jacob, V. Heinzelmann-Schwarz, N. Beerenwinkel
          </td>
          <td>2025-03-19</td>
          <td>Genome research</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Gene network models provide a foundation for graph theory approaches, aiding in the novel discovery of drug targets, disease genes, and genetic mechanisms for various biological functions. Disease genetics must be interpreted within the cellular context of disease-associated cell types, which cannot be achieved with datasets consisting solely of organism-level samples. Single-cell RNA sequencing (scRNA-seq) technology allows computational distinction of cell states which provides a unique opportunity to understand cellular biology that drives disease processes. Importantly, the abundance of cell samples with their transcriptome-wide profile allows the modeling of systemic cell-type-specific gene networks (CGNs), offering insights into gene-cell-disease relationships. In this review, we present reference-based and de novo inference of gene functional interaction networks that we have recently developed using scRNA-seq datasets. We also introduce a compendium of CGNs as a useful resource for cell-type-resolved disease genetics. By leveraging these advances, we envision single-cell network biology as the key approach for mapping the gene-cell-disease axis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9aa7b9ecda0e5aabd9c3870f99cf01184de548a3" target='_blank'>
              Single-cell network biology enabling cell-type-resolved disease genetics
              </a>
            </td>
          <td>
            Junha Cha, Insuk Lee
          </td>
          <td>2025-03-27</td>
          <td>Genomics & Informatics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Simple Summary Cancer arises as an abnormality in genomic control, either due to the activation of oncogenes or loss of functioning tumor-suppressor genes. Oncogenes result in an uncontrolled increase in cell proliferation; tumor-suppressor genes act as a brake, leading the cell cycle to stop and promote cell death when necessary. This review seeks to tease out the complex interactions between these genes, throwing light on key signaling pathways such as MAPK/ERK, PI3K/AKT/mTOR, and Wnt/β-catenin, all of which play a role in the progression of the tumor. We discuss major genetic theories, especially Knudson’s two-hit hypothesis, and how modern technologies like next-generation sequencing have advanced our understanding of cancer genetics. We also underscore the most current therapeutic approaches involving oncogenes and the reactivation of tumor-suppressor gene function, with challenges including drug resistance and heterogeneity of the tumor. In this regard, understanding the balance between oncogenes and tumor-suppressor genes is critical in developing effective and personalized cancer treatment, thereby bettering patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b78e398fe4233b0d93ca9c44f0387aa4d32d4ab" target='_blank'>
              Exploring the Genetic Orchestra of Cancer: The Interplay Between Oncogenes and Tumor-Suppressor Genes
              </a>
            </td>
          <td>
            Sajal Raj Singh, Rakesh Bhaskar, Shampa Ghosh, Bhuvaneshwar Yarlagadda, Krishna Kumar Singh, Prashant Verma, Sonali Sengupta, M. Mladenov, N. Hadzi-Petrushev, R. Stojchevski, J. K. Sinha, Dimiter B Avtanski
          </td>
          <td>2025-03-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The cellular programs that mediate therapy resistance are often important drivers of metastasis, a phenomenon that needs to be understood better to improve screening and treatment options for cancer patients. Although this issue has been studied extensively for chemotherapy, less is known about a causal link between resistance to radiation therapy and metastasis. We investigated this problem in triple-negative breast cancer (TNBC) and established that radiation resistant tumor cells have enhanced metastatic capacity, especially to bone. Resistance to radiation increases the expression of integrin β3 (ITGβ3), which promotes enhanced migration and invasion. Bioinformatic analysis and subsequent experimentation revealed an enrichment of RNA metabolism pathways that stabilize ITGβ3 transcripts. Specifically, the RNA binding protein heterogenous nuclear ribonucleoprotein L (HNRNPL), whose expression is regulated by Nrf2, mediates the formation of circular RNAs (circRNAs) that function as competing endogenous RNAs (ceRNAs) for the family of let-7 microRNAs that target ITGβ3. Collectively, our findings identify a novel mechanism of radiation-induced metastasis that is driven by alterations in RNA metabolism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14903a27d305650e2051ca5d1898fdc092718345" target='_blank'>
              Resistance to Radiation Enhances Metastasis by Altering RNA Metabolism
              </a>
            </td>
          <td>
            Ayush Kumar, Kensei Kishimoto, H. Goel, C. Wisniewski, Rui Li, Brendan Pacheco, L. J. Zhu, William A. Flavahan, A. Mercurio
          </td>
          <td>2025-02-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25fcf14d7421a285bda69ccae261660da8cc8fcc" target='_blank'>
              Gene Therapy and Gene Editing: Current Trends and Future Prospects of Molecular Medicine.
              </a>
            </td>
          <td>
            Aniruddha Sen, Vijay Singh, Shailendra Dwivedi, Ruchika Agrawal, Akash Bansal, Shashank Shekhar, Sudhir Shyam Kushwaha
          </td>
          <td>2025-02-18</td>
          <td>Current gene therapy</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="


 Recent progress in parasite genomics has is enabled for greater understanding of genetic structure of parasites, including the species of Plasmodium, Leishmania, and Trypanosoma. Cutting-edge next-generation sequencing (NGS) techniques, such as high-throughput sequencing, has enabled the discovery of essential genes linked to the metabolic pathways, drug resistance, and life cycle adaptations. Advancements in the field of genomics have enabled the deep understanding of biological processes and evolutionary adaptations of parasites.



 Investigating the genomes in various parasite strains has enhanced our understanding of their evolutionary adaptations, enhancing our capacity to predict epidemics and develop therapies that effectively act against various parasitic strains. Synthetic biology has also proposed novel treatment approaches, including the gene therapies and bioengineered microbes, that shows potential in fighting or inhibiting parasite illnesses. Revolutionary genome-editing methods, such as CRISPR-Cas9, have enabled the accurate genetic alterations, expediting the progress of sophisticated medicinal therapies, that are specifically designed for parasite management and eradication.



 Microbiome engineering, an emerging area, provides a novel opportunities for disease prevention by integrating the techniques such as transplanting faecal microbiota and genetically tailored bacteria to restore microbial equilibrium and decrease parasite populations. Nevertheless, the enduring stability of modified microbiomes, possible environmental hazards, and ethical concerns related to gene editing emphasise the necessity for stringent safety measures and monitoring by regulatory authorities. Using these technologies responsibly and ethically is crucial to guarantee the continuous therapeutic advancement.



">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a096b660a945d20f83a8eed57efbfc62f1dfd3e3" target='_blank'>
              Synthetic biology and parasite genomics: engineering parasite-resistant human microbiomes for sustainable disease prevention
              </a>
            </td>
          <td>
            E. S. Al-Malki
          </td>
          <td>2025-02-18</td>
          <td>Beni-Suef University Journal of Basic and Applied Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Single nucleotide polymorphisms (SNPs) in DNA repair genes can impair protein structure and function, contributing to disease development, including cancer. Non-synonymous SNPs (nsSNPs) in the LIG3 gene are linked to genomic instability and increased cancer risk, particularly acute myeloid leukemia (AML). This study aims to identify the most deleterious nsSNPs in the LIG3 gene and potential therapeutic targets for DNA repair restoration in AML. We employed in-silico computational methods to analyze LIG3 nsSNPs, using PredictSNP and Mutation3D to assess pathogenicity. Subsequently, molecular docking and dynamics simulations were conducted to evaluate ligand-binding affinities and protein stability. Out of the 12,191 mapped SNPs, 132 were nsSNPs located in the coding region. Among these, 18 nsSNPs were identified as detrimental including 12 destabilizing and 6 stabilizing nsSNPs. Nine cancer-associated nsSNPs, including L381R and R528C, were predicted due to their structural and functional impacts. Further analysis revealed key phosphorylation and methylation sites, such as 529S and 224R. Molecular dynamics simulations highlighted stable interactions of compounds AHP-MPC and DM-BFC with wild-type and R528C mutant LIG3 proteins, while R671G and V781M mutants showed instability. Protein-protein interaction networks and functional enrichment linked LIG3 to DNA repair pathways. Kaplan-Meier analysis associated high LIG3 expression with improved survival in breast cancer and AML, suggesting its role as a prognostic biomarker. This study emphasizes the mutation-specific effects of LIG3 nsSNPs on protein stability and ligand interactions. We recommend identifying DM-BFC to advance personalized medicine approaches for targeting deleterious variants, following in vitro and in vivo validation for AML treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87ba18559370ff8c79b7d669e449bdf299eda4ad" target='_blank'>
              Computational investigation unveils pathogenic LIG3 non-synonymous mutations and therapeutic targets in acute myeloid leukemia
              </a>
            </td>
          <td>
            Md. Arif Hossen, Umme Mim Sad Jahan, Md. Arju Hossain, Khalid Hossain Asif, Ahona Rahman, Sabbir Ahmed, Moin Uddin, Md Faisal Amin, Muhammad Abdul Bari, M. Islam, Mohammad Kamruzzaman, Soharth Hasnat, Mohammad Nasir Uddin, Tofazzal Islam, M. Hoque
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56b7ab64f4c6c0ffff56fec877ba1f8433465976" target='_blank'>
              Single-cell RNA sequencing revealed changes in the tumor microenvironment induced by radiotherapy for cervical cancer and the molecular mechanism of mast cells in immunosuppression.
              </a>
            </td>
          <td>
            L. Xue, Linzhi Gao, Shimin Zhou, Chaofan Yan, Xian Zhang, Wei Lin, Hu Li, Yuan Shen, Xiaoyu Wang
          </td>
          <td>2025-03-14</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background The ataxia-telangiectasia mutated (ATM) kinase phosphorylates and activates several downstream targets that are essential for DNA damage repair, cell cycle inhibition and apoptosis. Germline biallelic inactivation of the ATM gene causes ataxia-telangiectasia (A-T), and heterozygous pathogenic variant (PV) carriers are at increased risk of cancer, notably breast cancer. This study aimed to investigate whether DNA methylation profiling can be useful as a biomarker to identify tumors arising in ATM PV carriers, which may help for the management and optimal tailoring of therapies of these patients. Methods Breast tumor enriched DNA was prepared from 2 A-T patients, 27 patients carrying an ATM PV, 6 patients carrying a variant of uncertain clinical significance and 484 noncarriers enrolled in epidemiological studies conducted in France and Australia to investigate genetic and nongenetic factors involved in breast cancer susceptibility. Genome-wide DNA methylation analysis was performed using the Illumina Infinium HumanMethylation EPIC and 450K BeadChips. Correlation between promoter methylation and gene expression was assessed for 10 tumors for which transcriptomic data were available. Results We found that the ATM promoter was hypermethylated in 62% of tumors of heterozygous PV carriers compared to the mean methylation level of ATM promoter in tumors of noncarriers. Gene set enrichment analyses identified 47 biological pathways enriched in hypermethylated genes involved in neoplastic, neurodegenerative and metabolic-related pathways in tumor of PV carriers. Among the 327 differentially methylated promoters, promoters of ARHGAP40, SCGB3A1 (HIN-1), and CYBRD1 (DCYTB) were hypermethylated and associated with a lower gene expression in these tumors. Moreover, using three different deep learning algorithms (logistic regression, random forest and XGBoost), we identified a set of 27 additional biomarkers predictive of ATM status, which could be used in the future to provide evidence for or against pathogenicity in ATM variant classification strategies. Conclusions We showed that breast tumors that arise in women who carry an ATM PV display a specific genome-wide DNA methylation profile. Specifically, the methylation pattern of 27 key gene promoters was predictive of ATM PV status of the women. These genes may also represent new medical prevention and therapeutic targets for these women. Supplementary Information The online version contains supplementary material available at 10.1186/s13058-025-01988-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bebfd1b3f4c2e2d684ea0f216e88d7bab9a7fec5" target='_blank'>
              Breast tumors from ATM pathogenic variant carriers display a specific genome-wide DNA methylation profile
              </a>
            </td>
          <td>
            Nicolas M Viart, Anne-Laure Renault, Séverine Eon-Marchais, Yue Jiao, L. Fuhrmann, Sophia Murat El Houdigui, Dorothée Le Gal, Eve Cavaciuti, Marie-Gabrielle Dondon, Juana Beauvallet, Virginie Raynal, D. Stoppa-Lyonnet, A. Vincent-Salomon, Nadine Andrieu, M. Southey, F. Lesueur
          </td>
          <td>2025-03-11</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>118</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e83d84b8f1cfdb09485b80edd7f374acbe1eeac9" target='_blank'>
              Molecular mechanisms of Hippo pathway in tumorigenesis: therapeutic implications.
              </a>
            </td>
          <td>
            M. Saadh, Hanan Hassan Ahmed, Radhwan Abdul Kareem, A. K. Bishoyi, R. Roopashree, Debasish Shit, Renu Arya, Abhishek Sharma, Kakhramon Khaitov, Hayder Naji Sameer, Ahmed Yaseen, Zainab H. Athab, Mohaned Adil
          </td>
          <td>2025-02-27</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background Acute myelocytic leukemia (AML) is a common hematological malignancy in adults. Although several risk stratifications based on cytogenetic and molecular abnormalities are available to guide the indications for allogeneic hematopoietic cell transplantation (allo-HCT), determining optimal treatment strategies for AML remains challenging. In this study, using transcriptome datasets, we investigated the association between event-free survival (EFS) in intensively treated AML patients and the aberrant expression of endogenous viral element (EVE)-derived open reading frames (ORFs), which have been reported to be associated with the pathophysiology of various malignancies and have the potential to serve as neoantigens in specific cancers. Results The expression levels of human endogenous retrovirus family K9 (HERVK9) ORFs were associated with EFS, independent of conventional risk stratification. Furthermore, AML cells with higher levels of HERVK9 expression exhibited enhanced antigen processing and presentation, along with increased expression of genes associated with adaptive immune responses and apoptosis, indicating that aberrant HERVK9 expression may initiate an anti-neoplastic immune response via increased antigen presentation. Conclusions HERVK9 expression may have serve as a crucial prognostic indicator that could aid in determining the indications for upfront allo-HCT in AML patients. Supplementary Information The online version contains supplementary material available at 10.1186/s12977-025-00661-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5f29628a7105d4d628b72c6ef996a0f86eb8dca" target='_blank'>
              Aberrant expression of human endogenous retrovirus K9-derived elements is associated with better clinical outcome of acute myelocytic leukemia
              </a>
            </td>
          <td>
            Ryo Yanagiya, So Nakagawa, M. Onizuka, Ai Kotani
          </td>
          <td>2025-04-01</td>
          <td>Retrovirology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Transposable elements (TEs) drive genome evolution and can affect gene expression through diverse mechanisms. In breast cancer, disrupted regulation of TE sequences may facilitate tumor-specific transcriptomic alterations. We examine 142,514 full-length isoforms derived from long-read RNA sequencing (LR-seq) of 30 breast samples to investigate the effects of TEs on the breast cancer transcriptome. Approximately half of these isoforms contain TE sequences, and these contribute to half of the novel annotated splice junctions. We quantify splicing of these LR-seq derived isoforms in 1,135 breast tumors from The Cancer Genome Atlas (TCGA) and 1,329 healthy tissue samples from the Genotype-Tissue Expression (GTEx), and find 300 TE-overlapping tumor-specific splicing events. Some splicing events are enriched in specific breast cancer subtypes – for example, a TE-driven transcription start site upstream of ERBB2 in HER2 + tumors, and several TE-mediated splicing events are associated with patient survival and poor prognosis. The full-length sequences we capture with LR-seq reveal thousands of isoforms with signatures of RNA editing, including a novel isoform belonging to RHOA; a gene previously implicated in tumor progression. We utilize our full-length isoforms to discover polymorphic TE insertions that alter splicing and validate one of these events in breast cancer cell lines. Together, our results demonstrate the widespread effects of dysregulated TEs on breast cancer transcriptomes and highlight the advantages of long-read isoform sequencing for understanding TE biology. TE-derived isoforms may alter the expression of genes important in cancer and can potentially be used as novel, disease-specific therapeutic targets or biomarkers. One sentence summary: Transposable elements generate alternative isoforms and alter post-transcriptional regulation in human breast cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00341-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/279ce0f97c7d66cc42d068594f84880c4fe69c7b" target='_blank'>
              Alternative splicing of transposable elements in human breast cancer
              </a>
            </td>
          <td>
            Alex Nesta, Diogo F. T. Veiga, Jacques Banchereau, Olga Anczuków, Christine R. Beck
          </td>
          <td>2025-02-22</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Immune dysfunctions are believed to contribute to bipolar disorder (BD), yet their mechanistic basis remains unclear. To address this, we systematically characterize BD-associated epigenomic and genetic variation in peripheral blood immune cells by profiling and integrating 833 genome-wide maps of five histone modification marks across 180 individuals (88 Type I BD patients, 92 controls), coupled with whole-genome sequencing data and rich medical records. We annotate 450k candidate cis-regulatory elements (CREs) and identify differential CREs (dCREs) in BD patients, suggesting down-regulated adaptive and up-regulated innate immune response. We predict candidate BD driver genes in the circulating immune system, which frequently show matched brain activity mainly related to calcium signaling and endoplasmic reticulum (ER) transport, suggesting dysregulated synaptic transmission, neuronal plasticity, and ER stress. We find that candidate driver genes are often linked to BD GWAS variants through blood-specific eQTLs not found in any brain cell types, indicating potential causal roles of circulating immune cells in bipolar disorder. We then infer 24 latent factors of BD-differential CRE variation and use them to group the patients into five epigenomic subtypes, which also show distinct disease phenotypes, including infection and inflammation, osmotic laxative use and glucose intolerance, quetiapine use, and hypertension. We next associate immune-partitioned BD polygenic risk scores with patient epigenomic subtypes, revealing the genetic basis of BD patient heterogeneity captured by blood epigenomics. Lastly, by analyzing transcriptional responses to known pharmacological interventions in hematopoietic cells that enrich BD patient group-specific dysregulated genes, we identify drugs/compounds that could be repurposed for ameliorating BD-associated immune dysfunction in a patient group-dependent manner. Overall, based on our study of genotype-epigenome-phenotype integration, we infer a potentially causal role of immune cells in BD, offering insights into biomarkers, subtypes, and precision medicine interventions targeting peripheral immune dysfunction and thus advancing precision medicine in BD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae408f6dbcd8637b77675c5c1327b34b01ef6d58" target='_blank'>
              Genotype-Epigenome-Phenotype Integration Reveals the Contributions of Peripheral Immune Cells to Bipolar Disorder Pathogenesis, Phenotypic Heterogeneity, and Therapy
              </a>
            </td>
          <td>
            L. Hou, Y. Li, X. Xiong, Y. Tanigawa, Y. Park, S. W. Lenz, A. Grayson, J.-H. Lee, E. Ryu, J. E. Olson, J. M. Biernacka, M. A. Frye, T. Ordog, M. Kellis
          </td>
          <td>2025-03-18</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3df1af238c5c0bc411539014f0c361b61ef1f54" target='_blank'>
              Single Cell Resolution Tracking of Cutaneous T-Cell Lymphoma Reveals Clonal Evolution in Disease Progression
              </a>
            </td>
          <td>
            Hannah K. Dorando, Jared M. Andrews, Nicholas Borcherding, Chaz C. Quinn, J. Schmidt, Oam Khatavkar, Jahnavi Aluri, Michael T. Harmon, Marcus P. Watkins, Anastasia Frank, Megan A. Cooper, Amy C. Musiek, Neha Mehta-Shah, Jacqueline E. Payton
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background The tumor suppressor protein, p53, which is mutated in half of human tumors, plays a critical role in cellular responses to DNA damage and maintenance of genome stability. Therefore, increasing our understanding of the p53 pathway is essential for improving cancer treatment and diagnosis. Methods This study, which aimed to identify genes and pathways that mediate resistance to p53 upregulation, used genome-wide CRISPR-Cas9 loss-of-function screening done with Nutlin-3a, which inhibits p53-MDM2 interaction, resulting in p53 accumulation and apoptotic cell death. We used bioinformatics analysis for the identification of genes and pathways that are involved in the p53 pathway and cell survival assays to validate specific genes. In addition, we used RNA-seq to identify differentially expressed p53 target genes in gene knockout (KO) cell lines. Results Our screen revealed three significantly enriched pathways: The heparan sulfate glycosaminoglycan biosynthesis, diphthamide biosynthesis and Hippo pathway. Notably, TRIP12 was significantly enriched in our screen. We found that TRIP12 is required for the p53-dependent transcription of several pro-apoptotic genes. Conclusion Our study has identified two novel pathways that play a role in p53-mediated growth restriction. Moreover, we have highlighted the interaction between the Hippo and the p53 pathways. Interestingly, we have shown that TRIP12 plays an important function in the p53 pathway by selectively affecting its role as a transcription factor. Supplementary Information The online version contains supplementary material available at 10.1186/s10020-025-01141-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61f2f128a395ad66b24bd497a265456014e21a60" target='_blank'>
              Genome-wide screening in human embryonic stem cells identifies genes and pathways involved in the p53 pathway
              </a>
            </td>
          <td>
            Amir Haddad, T. Golan‐lev, N. Benvenisty, Michal Goldberg
          </td>
          <td>2025-03-13</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Simple Summary Nilotinib and other tyrosine kinase inhibitors (TKIs) target the BCR-ABL1 oncoprotein in Chronic Myeloid Leukemia (CML), reducing the growth of leukemic stem cells (LSCs) and promoting their death. However, some LSCs persist in the bone marrow (BM), which may lead to relapse and disease progression. We analyzed the gene expression profiling of BM CD34+/lin− cells from 79 Chronic-Phase CML patients at diagnosis and after 12 months of nilotinib treatment, comparing them to the healthy cells from 10 donors (CTRLs). Our study identified key genes involved in eight crucial biological pathways: CML, cell cycle, JAK-STAT, PI3K-Akt, MAPK, Ras, NF-kB, and ABC transporters. We observed a progressive down-regulation of the expression levels of several genes across these pathways from diagnosis to 12 months of nilotinib treatment and to CTRLs. This could suggest that nilotinib effects create selective pressure, potentially supporting the survival and self-renewal of LSCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01d3a05e9350f9c4633e6a38d029b612b50597a4" target='_blank'>
              Bone Marrow CD34+/lin− Cells of Patients with Chronic-Phase Chronic Myeloid Leukemia (CP-CML) After 12 Months of Nilotinib Treatment Exhibit a Different Gene Expression Signature Compared to the Diagnosis and the Corresponding Cells from Healthy Subjects
              </a>
            </td>
          <td>
            Alessandra Trojani, E. Pungolino, Barbara Di Camillo, L. E. Bossi, Cassandra Palumbo, M. D’adda, A. Perego, Mauro Turrini, C. Elena, L. Borin, A. Iurlo, Simona Malato, Francesco Spina, M. L. Latargia, P. Spedini, Salvatore Artale, M. Anghilieri, M. Carraro, C. Bucelli, A. Beghini, R. Cairoli
          </td>
          <td>2025-03-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a prevalent malignant tumor characterized by high global incidence and mortality rates. Furthermore, it is imperative to comprehend the molecular mechanisms underlying its development and to identify effective prognostic markers. These efforts are crucial for pinpointing potential therapeutic targets and enhancing patient survival rates. Therefore, we develop a novel prognostic model aimed at providing new theoretical support for clinical prognosis evaluation and treatment. We downloaded data from the Gene Expression Omnibus (GEO) and The Cancer Genome Atlas (TCGA) databases. Subsequently, we performed single-cell analysis and developed a prognostic model associated with colorectal cancer. We divided the scRNA-seq dataset (GSE221575) into 19 cell clusters and classified these clusters into 11 distinct cell types using marker genes. Using univariate Cox regression and LASSO (Least Absolute Shrinkage and Selection Operator) analyses, we developed a prognostic model consisting of 9 genes. Based on our 9-gene model, we divided patients into high-risk and low-risk groups using the median risk score. The high-risk group demonstrated significant positive correlations with M0 macrophages, CD8+ T cells, and M2 macrophages. The enrichment analyses indicate significant enrichment of immune-related pathways in the high-risk group, including HEDGEHOG_SIGNALING, Wnt signaling pathway, and cell adhesion molecules. Drug sensitivity analysis revealed that the low-risk group was sensitive to 5 chemotherapeutic drugs, while the high-risk group was sensitive to only 1. Additionally, we developed a highly reliable nomogram for clinical application. This suggests that the risk score derived from our modeling analysis is highly effective for stratifying colorectal cancer samples. This study comprehensively applied bioinformatics methods to construct a risk score model. The model showed good predictive performance, offering potential guidance for individualized treatment of colorectal cancer patients. Furthermore, it may provide valuable insights into the disease’s pathogenesis and identify potential therapeutic targets for further research. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-91761-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a4ee3ba4be7a7e3b5e63b304d989c55900905b7" target='_blank'>
              Integrated analysis of single-cell and bulk RNA-sequencing to predict prognosis and therapeutic response for colorectal cancer
              </a>
            </td>
          <td>
            Liyang Cai, Xin Guo, Yucheng Zhang, Huajie Xie, Yongfeng Liu, Jianlong Zhou, Huolun Feng, Jiabin Zheng, Yong Li
          </td>
          <td>2025-03-07</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="EGFR is a proto-oncogene that is mutationally activated in a variety of cancers. Small molecule inhibitors targeting EGFR can be effective in slowing the progression of disease, and in some settings these drugs even cause dramatic tumor regression. However, responses to EGFR inhibitors are rarely durable, and the mechanisms contributing to response variation remain unclear. In particular, several distinct mechanisms have been proposed for how EGFR inhibition activates cell death, and a consensus has yet to emerge. In this study, we use functional genomics with specialized analyses to infer how genetic perturbations effect the drug-induced death rate. Our data clarify that inhibition of PI3K signaling drives the lethality of EGFR inhibition. Inhibition of other pathways downstream of EGFR, including the RAS-MAPK pathway, promote growth suppression, but not the lethal effects of EGFR inhibitors. Taken together, our study reveals the first “reference map” for the genome-wide genetic dependencies of lethality for EGFR inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ecf934bb91c2c002402ace642323761657b953a" target='_blank'>
              Genome-wide profiling identifies the genetic dependencies of cell death following EGFR inhibition
              </a>
            </td>
          <td>
            Sydney A Porto, Gavin A. Birdsall, Nicholas W. Harper, Megan E. Honeywell, Michael J. Lee
          </td>
          <td>2025-04-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="To maintain cell fitness, deleterious genetic alterations are buffered by compensatory changes in additional genes. In cancer, buffering processes could be targeted by synthetic lethality. However, despite the large-scale identification of synthetic lethal effects in preclinical models, evidence that these operate clinically is limited. This impedes the application of synthetic lethal approaches. By integrating molecular profiling data from >9,000 cancers with synthetic lethal screens, we show that transcriptomic buffering of tumor suppressor gene (TSG) loss by hyperexpression of synthetic lethal partners is a common phenomenon, extending to multiple TSGs and histotypes. Transcriptomic buffering is also notable in cancers that phenocopy TSG loss, such as BRCAness cancers, where expression of BRCA1/2 synthetic lethal genes correlates with clinical outcome. Synthetic lethal genes that exhibit transcriptomic buffering also represent more robust synthetic lethal effects. These observations have implications for understanding how tumor cells tolerate TSG loss, in part explain transcriptomic architectures in cancer and provide insight into target selection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29a52e19b56a0e26f2474ec866322bdccd6f6854" target='_blank'>
              The transcriptomic architecture of common cancers reflects synthetic lethal interactions
              </a>
            </td>
          <td>
            S. Haider, R. Brough, Santiago Madera, Jacopo Iacovacci, Aditi Gulati, Andrew J. Wicks, J. Alexander, S. Pettitt, Andrew N J Tutt, C. J. Lord
          </td>
          <td>2025-03-03</td>
          <td>Nature Genetics</td>
          <td>1</td>
          <td>33</td>
        </tr>

        <tr id="Coronary artery disease (CAD) is the leading cause of death worldwide. Recently, hundreds of genomic loci have been shown to increase CAD risk, however, the molecular mechanisms underlying signals from CAD risk loci remain largely unclear. We sought to pinpoint the candidate causal coding and non-coding genes of CAD risk loci in a cell type-specific fashion. We integrated the latest statistics of CAD genetics from over one million individuals with epigenetic data from 45 relevant cell types to identify genes whose regulation is affected by CAD-associated single nucleotide variants (SNVs) via epigenetic mechanisms. Applying two statistical approaches, we identified 1,580 genes likely involved in CAD, about half of which have not been associated with the disease so far. Enrichment analysis and phenome-wide association studies linked the novel candidate genes to disease-specific pathways and CAD risk factors, corroborating their disease relevance. We showed that CAD-SNVs are enriched to regulate gene expression by affecting the binding of transcription factors (TFs) with cellular specificity. Of all the candidate genes, 23.5% represented non-coding RNAs (ncRNA), which likewise showed strong cell type specificity. We conducted a proof-of-concept biological validation for the novel CAD ncRNA gene IQCH-AS1. CRISPR/Cas9-based gene knockout of IQCH-AS1, in a human preadipocyte strain, resulted in reduced preadipocyte proliferation, less adipocyte lipid accumulation, and atherogenic cytokine profile. The cellular data are in line with the reduction of IQCH-AS1 in adipose tissues of CAD patients and the negative impact of risk alleles on its expression, suggesting IQCH-AS1 to be protective for CAD. Our study not only pinpoints CAD candidate genes in a cell type-specific fashion but also spotlights the roles of the understudied ncRNA genes in CAD genetics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/894c0b3eed40de06b0fd932a3214f3acdff103b5" target='_blank'>
              Cell type-specific epigenetic regulatory circuitry of coronary artery disease loci
              </a>
            </td>
          <td>
            Dennis Hecker, Xiaoning Song, Nina Baumgarten, A. Diagel, N. Katsaouni, Ling Li, Shuangyue Li, Ranjan Kumar Maji, Fatemeh Behjati Ardakani, Lijiang Ma, Daniel Tews, M. Wabitsch, J. Björkegren, H. Schunkert, Zhifen Chen, Marcel H. Schulz
          </td>
          <td>2025-02-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) are a heterogeneous group of tumor cells that play a significant role in tumorigenesis, therapeutic resistance, and recurrence in liver hepatocellular carcinoma (LIHC). This study combines clinical data sets from The Cancer Genome Atlas (TCGA) and the International Cancer Genome Consortium (ICGC) with bulk RNA sequencing data. This study also features the GSE156625 single-cell RNA sequencing (scRNA) data set from the GEO to explore the prognostic significance of CSC biomarkers (BCSCs) in LIHC. In this research, we introduce a developed prognostic risk model that relies on nine specific BCSCs, including ADM, CCL5, CD274, DLGAP5, HOXD9, IGF1, S100A9, SOCS2, and TNFRSF11B. It was found that high-risk patients experience shorter overall survival rates when compared to low-risk patients. Additionally, the study characterized the composition of immune cells within the tumor microenvironment (TME) and revealed significant variations in gene-expression levels and mutation rates between different risk groups. The model suggests that liver cancer progression might be driven by immune evasion independent of PD-L1 and highlights the potential of the low-risk BCSC group being sensitive to various treatments. Our findings offer a promising foundation for personalized LIHC therapy and highlight the need for further experimental validation of the roles of these CSCs in disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ccc91ecd266e7bcb8a4e400c1045bb16c558c70" target='_blank'>
              Identification of a Cancer Stem Cell-Related Gene Signature in Hepatocellular Carcinoma Based on Single-Cell RNA-Seq and Bulk RNA-Seq Analysis
              </a>
            </td>
          <td>
            Jing Wu, Xu Liu, Sheng Huang, Wei Liu
          </td>
          <td>2025-03-24</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The mutational landscape of phylogenetically ultraconserved elements (UCEs), especially those in noncoding DNAs (ncUCEs), and their functional relevance in cancers remain poorly characterized. Here, we perform a systematic analysis of whole-genome and in-house targeted UCE sequencing datasets from more than 3000 patients with cancer of 13,736 UCEs and demonstrate that ncUCE somatic alterations are common. Using a multiplexed CRISPR knockout screen in colorectal cancer cells, we show that the loss of several altered ncUCEs significantly affects cell proliferation. In-depth functional studies in vitro and in vivo further reveal that specific ncUCEs can be enhancers of tumor suppressors (such as ARID1B) and silencers of oncogenic proteins (such as RPS13). Moreover, several miRNAs located in ncUCEs are recurrently mutated. Mutations in miR-142 locus can affect the Drosha-mediated processing of precursor miRNAs, resulting in the down-regulation of the mature transcript. These results provide systematic evidence that specific ncUCEs play diverse regulatory roles in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d50d62f517b45fea38f094fa47dd4b743a633f64" target='_blank'>
              The mutational landscape and functional effects of noncoding ultraconserved elements in human cancers
              </a>
            </td>
          <td>
            R. Bayraktar, Yitao Tang, M. Dragomir, Cristina Ivan, Xinxin Peng, L. Fabris, Jianhua Zhang, A. Carugo, S. Aneli, Jintan Liu, M. Chen, Sanjana Srinivasan, Iman Sahnoune, E. Bayraktar, K. Akdemir, Meng Chen, P. Narayanan, Wilson Huang, L. Ott, A. K. Eterovic, O. Villarreal, M. M. Mohammad, M. Peoples, Danielle M Walsh, J. A. Hernandez, Margaret B Morgan, K. Shaw, Jennifer S Davis, David Menter, Constantine S Tam, Paul Yeh, S. Dawson, L. Rassenti, T. Kipps, Tanja Kunej, Z. Estrov, S. Joosse, Luca Pagani, C. Alix-Panabières, Klaus Pantel, Alessandra Ferajoli, Andrew Futreal, I. Wistuba, Milan Radovich, S. Kopetz, Michael J Keating, Giulio F. Draetta, John S. Mattick, Han Liang, George A Calin
          </td>
          <td>2025-02-19</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>99</td>
        </tr>

        <tr id="Somatic mobilization of LINE-1 (L1) has been implicated in cancer etiology. We analyzed a recent TCGA data release comprised of nearly 5000 pan-cancer paired tumor-normal whole-genome sequencing (WGS) samples and ~9000 tumor RNA samples. We developed TotalReCall an improved algorithm and pipeline for detection of L1 retrotransposition (RT), finding high correlation between L1 expression and “RT burden” per sample. Furthermore, we mathematically model the dual regulatory roles of p53, where mutations in TP53 disrupt regulation of both L1 expression and retrotransposition. We found those with Li-Fraumeni Syndrome (LFS) heritable TP53 pathogenic and likely pathogenic variants bear similarly high L1 activity compared to matched cancers from patients without LFS, suggesting this population be considered in attempts to target L1 therapeutically. Due to improved sensitivity, we detect over 10 genes beyond TP53 whose mutations correlate with L1, including ATRX, suggesting other, potentially targetable, mechanisms underlying L1 regulation in cancer remain to be discovered.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09c16a8e22a8e26a1832c9dc60f1f0a97f3c5163" target='_blank'>
              Pan-cancer multi-omic model of LINE-1 activity reveals locus heterogeneity of retrotransposition efficiency
              </a>
            </td>
          <td>
            Alexander Solovyov, Julie M. Behr, D. Hoyos, Eric Banks, Alexander W Drong, Bryan Thornlow, Jimmy Z Zhong, Enrique Garcia-Rivera, Wilson McKerrow, Chong Chu, Cédric Arisdakessian, Dennis M Zaller, J. Kamihara, Liyang Diao, M. Fromer, Benjamin D. Greenbaum
          </td>
          <td>2025-02-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Retrospective data support overall survival (OS) advantage to high clonal tumor mutation burden (cTMB), high clonal neoantigen load (cNEO) and low intratumor heterogeneity (ITH) in cancer patients who receive immunotherapy. In order to explore this relationship prospectively with Vigil, a triple function targeted immunotherapy involving ovarian cancer patients in long term follow up of the Phase 2b VITAL trial, we developed an exome sequencing procedure and associated bioinformatics pipeline to determine clonal signal patterns. DNA libraries containing exome sequences tagged with unique molecular identifiers (UMI) were prepared from paired samples and sequenced on Illumina sequencers to high coverage depths of ~ 930X (tumor) and ~ 130X (normal). Raw sequence reads were processed into optimized binary alignment map (BAM) files, using the UMI information. The BAM files were inputted into modules for calling MHC-I alleles, annotating single nucleotide variants (SNVs) and small insertions/deletions (InDels), and for determination of allelic copy number. The outputs were used to predict the sequence of peptide neoantigens and to perform clonality analysis in order to assign each SNV and InDel in a patient tumor sample to a primary clone or subclone. The Clonal Neoantigen pipeline was further assessed using whole exome Illumina sequencing data from three previously published studies. Evaluation of the pipeline using synthetic sequencing data from a sub-clonal deconvolution tool benchmarking study, showed positive predictive value (PPV) and positive percent agreement (PPA) of > 97.5% and > 96.5%, respectively, for SNV and InDel detection with minimum requirements for variant density and allele fraction. Haplotype calls from the Clonal Neoantigen pipeline MHC-I/ MHC-II typing module matched a published benchmark for 91.5% of the calls in a sample of 99 patients. Analysis of exome sequencing data from 14 patients with advanced melanoma revealed a strong correlation between cTMB values determined by the Clonal Neoantigen pipeline as compared to those calculated from the published data (R2 = 0.99). Following validation, the wet lab process and Clonal Neoantigen pipeline was applied to a set of matched normal, tumor, and Vigil product samples from 9 (n = 27 samples) ovarian cancer subjects entered into the VITAL (CL-PTL-119) trial. Results demonstrated marked correlation (R2 = 0.98) of cTMB between tumor used to construct Vigil and Vigil product. Correlation between tumor and Vigil for the cNEO and ITH metrics, showed R2 values of 0.95 and 0.87, respectively. The consistency of the Clonal Neoantigen pipeline results with previously published data as well as the agreement between results for tumor and Vigil for the entire system provide a strong basis of support for utilization of this pipeline for prospective determination of cTMB, cNEO, and ITH values in clinical tumor tissue in order to explore possible correlative relationships with clinical response parameters.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16718d2277420c09078544ae1939c2c13bd37fd2" target='_blank'>
              Exome sequencing shows same pattern of clonal tumor mutational burden, intratumor heterogenicity and clonal neoantigen between autologous tumor and Vigil product
              </a>
            </td>
          <td>
            David Willoughby, E. Bognar, L. Stanbery, Casey Nagel, G. Wallraven, Aman Pruthi, Nicholas Bild, Ericca Stamper, Donald Rao, Adam Walter, John Nemunaitis
          </td>
          <td>2025-03-13</td>
          <td>Scientific Reports</td>
          <td>1</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcd871168cb54fad2bfcadb7ada6ba23bd54105e" target='_blank'>
              The Farm Animal Genotype-Tissue Expression (FarmGTEx) Project.
              </a>
            </td>
          <td>
            L. Fang, J. Teng, Qing Lin, Zhonghao Bai, Shuli Liu, Dailu Guan, Bingjie Li, Yahui Gao, Yali Hou, Mian Gong, Zhangyuan Pan, Ying Yu, Emily L. Clark, Jacqueline Smith, K. Rawlik, Ruidong Xiang, A. Chamberlain, Michael E. Goddard, Mathew Littlejohn, G. Larson, D. MacHugh, John F. O’Grady, Peter Sørensen, G. Sahana, M. Lund, Zhihua Jiang, Xiangchun Pan, Wen Gong, Haihan Zhang, Xi He, Yuebo Zhang, Ning Gao, Jun He, G. Yi, Yuwen Liu, Zhonglin Tang, P. Zhao, Yang Zhou, Liangliang Fu, Xiao Wang, Dan Hao, Lei Liu, Siqian Chen, Robert S. Young, Xia Shen, Charley Xia, Hao Cheng, Li Ma, John B. Cole, R. Baldwin, Cong-Jun Li, C. P. Van Tassell, Benjamin D. Rosen, Nayan Bhowmik, Joan Lunney, Wansheng Liu, Leluo Guan, Xin Zhao, E. Ibeagha-Awemu, Yonglun Luo, Lin Lin, O. Canela‐Xandri, M. Derks, R. Crooijmans, M. Gòdia, Ole Madsen, MAM. Groenen, J. Koltes, Christopher K Tuggle, Fiona M. McCarthy, Dominique Rocha, Elisabetta Giuffra, M. Amills, A. Clop, Maria Ballester, G. Tosser-Klopp, Jing Li, Chao Fang, Ming Fang, Qishan Wang, Zhuocheng Hou, Qin Wang, Fuping Zhao, Lin Jiang, Guiping Zhao, Zhengkui Zhou, Rong Zhou, He-he Liu, Juan Deng, Long Jin, Mingzhou Li, Delin Mo, Xiaohong Liu, Yaosheng Chen, Xiaolong Yuan, Jiaqi Li, Shuhong Zhao, Yi Zhang, Xiangdong Ding, D. Sun, Hui-Zeng Sun, Cong-Jun Li, Yu Wang, Yu Jiang, Dongdong Wu, Wenwen Wang, Xinzhong Fan, Qin Zhang, Kui Li, Hao Zhang, N. Yang, Xiaoxiang Hu, Wen Huang, Jiuzhou Song, Yang Wu, Jian Yang, Weiwei Wu, C. Kasper, Xinfeng Liu, Xiaofei Yu, Leilei Cui, Xiang Zhou, Seyoung Kim, Wei Li, Hae Kyung Im, E. Buckler, Bing Ren, Michael C. Schatz, Jingyi Jessica Li, A. Palmer, L. Frantz, Huaijun Zhou, Zhe Zhang, George E. Liu
          </td>
          <td>2025-03-17</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Understanding the genetic basis of gene expression can shed light on the regulatory mechanisms underlying complex traits and diseases. Single-cell resolved measures of RNA levels and single-cell expression quantitative trait loci (sc-eQTLs) have revealed genetic regulation that drives sub-tissue cell states and types across diverse human tissues. Here, we describe the first phase of TenK10K, the largest-to-date dataset of matched whole-genome sequencing (WGS) and single-cell RNA-sequencing (scRNA-seq). We leverage scRNA-seq data from over 5 million cells across 28 immune cell types and matched WGS from 1,925 individuals. This provides power to detect associations between rare and low-frequency genetic variants that have largely been uncharacterised in their impact on cell-specific gene expression. We map the effects of both common and rare variants in a cell type specific manner using SAIGE-QTL. This newly developed method increases power by modelling single cells directly using a Poisson model rather than relying on aggregated 'pseudobulk' counts. We identify putative common regulatory variants for 83% of all 21,404 genes tested and cumulative rare variant signals for 47% of genes. We explore how genetic effects vary across cell type and state spectra, develop a framework to determine the degree to which sc-eQTLs are cell type specific, and show that about half of the effects are observed only in one or a few cell types. By integrating our results with functional annotations and disease information, we further characterise the likely molecular modes of action for many disease-associated variants. Finally, we explore the effects of genetic variants on gene expression across different cell states and functions, as well as effects that directly vary cell state abundance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69c58c57797887ff73f102df82f585e2cdd85a57" target='_blank'>
              Impact of Rare and Common Genetic Variation on Cell Type-Specific Gene Expression
              </a>
            </td>
          <td>
            A. Cuomo, Eleanor Spenceley, Hope A. Tanudisastro, Blake Bowen, Albert Henry, Hao Lawrence Huang, A. Xue, Wei Zhou, M. Welland, Arthur S. Lee, Kristof Wing, Owen Tang, Michael P. Gray, Michael Franklin, Michael Harper, Michael Silk, Katalina Bobowik, Alexander Stuckey, John Marshall, Vivian Bakiris, Caitlin Uren, B. Madala, Amy Miniter, Caitlin Bartie, Drew R Neavin, Zhen Qiao, Eyal Ben-David, Ling Chen, Kyle Kai-How, Farh, Stuart M. Grieve, T. Nguyen, Jennifer A. Piscionere, O. Siggs, Hannah R. Nicholas, Katrina M. de Lange, Alex H. Hewitt, G. A. Figtree, Daniel G. MacArthur, J. E. Powell
          </td>
          <td>2025-03-21</td>
          <td>None</td>
          <td>1</td>
          <td>35</td>
        </tr>

        <tr id="The phenotype of a mutation often differs across genetically distinct individuals. In the most extreme case, a gene can be essential for viability in one genetic background, but dispensable in another. Although genetic context-dependency of mutant phenotypes is frequently observed, the underlying causes often remain elusive. Here, we investigated the genetic changes responsible for differences in gene essentiality across 18 genetically diverse natural yeast strains. First, we identified 39 genes that were essential in the laboratory reference strain but not required for viability in at least one other genetic background, suggesting that the natural strain contained suppressor variants that could bypass the need for the essential gene. We then mapped and validated the causal bypass suppressor variants using bulk segregant analysis and allele replacements. Bypass suppression was generally driven by a single modifier gene that tended to differ between genetic backgrounds. The suppressors often indirectly counteracted the effect of deleting the essential gene, for instance by changing the transcriptome of a cell. Context-dependent essential genes and their bypass suppressors were frequently co-mutated across 1,011 yeast isolates and identified naturally occurring evolutionary trajectories. Overall, our results highlight the relatively high frequency of bypass suppression in natural populations, as well as the underlying variants and mechanisms. A thorough understanding of the causes of genetic background effects is crucial for the interpretation of genotype-to-phenotype relationships, including those associated with human disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3807dbb2ff664332cc4f31ac739a72b593b98e5b" target='_blank'>
              The modifiers that cause changes in gene essentiality
              </a>
            </td>
          <td>
            Amandine Batté, Núria Bosch-Guiteras, Carles Pons, Marina Ota, Maykel Lopes, Sushma Sharma, Nicolò Tellini, Claire Paltenghi, Michelle Conti, Kwan Ting Kan, Uyen Linh Ho, Michaël Wiederkehr, Jonas Barraud, Mark Ashe, Patrick Aloy, Gianni Liti, A. Chabes, Leopold Parts, Jolanda van Leeuwen
          </td>
          <td>2025-03-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03fe48b40d4aaf66299fa294f97de6fcfcaa3b59" target='_blank'>
              Integrative Bioinformatic Analysis of Cellular Senescence Genes in Ovarian Cancer: Molecular Subtyping, Prognostic Risk Stratification, and Chemoresistance Prediction
              </a>
            </td>
          <td>
            Ailian Li, Dian-shuang Xu
          </td>
          <td>2025-04-04</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Human genome sequencing efforts in healthy and diseased individuals continue to identify a broad spectrum of genetic variants associated with predisposition, progression, and therapeutic outcomes for diseases like cancer1–6. Insights derived from these studies have significant potential to guide clinical diagnoses and treatment decisions; however, the relative importance and functional impact of most genetic variants remain poorly understood. Precision genome editing technologies like base and prime editing can be used to systematically engineer and interrogate diverse types of endogenous genetic variants in their native context7–9. We and others have recently developed and applied scalable sensor-based screening approaches to engineer and measure the phenotypes produced by thousands of endogenous mutations in vitro10–12. However, the impact of most genetic variants in the physiological in vivo setting, including contextual differences depending on the tissue or microenvironment, remains unexplored. Here, we integrate new cross-species base editing sensor libraries with syngeneic cancer mouse models to develop a multiplexed in vivo platform for systematic functional analysis of endogenous genetic variants in primary and disseminated malignancies. We used this platform to screen 13,840 guide RNAs designed to engineer 7,783 human cancer-associated mutations mapping to 489 endogenous protein-coding genes, allowing us to construct a rich compendium of putative functional interactions between genes, mutations, and physiological contexts. Our findings suggest that the physiological in vivo environment and cellular organotropism are important contextual determinants of specific gene-variant phenotypes. We also show that many mutations and their in vivo effects fail to be detected with standard CRISPR-Cas9 nuclease approaches and often produce discordant phenotypes, potentially due to site-specific amino acid selection- or separation-of-function mechanisms. This versatile platform could be deployed to investigate how genetic variation impacts diverse in vivo phenotypes associated with cancer and other genetic diseases, as well as identify new potential therapeutic avenues to treat human disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2f71f3527a2a35f380d7482a2555f7e6eeef4ab" target='_blank'>
              Multiplexed in vivo base editing identifies functional gene-variant-context interactions
              </a>
            </td>
          <td>
            Jonuelle Acosta, Grace A. Johnson, Samuel I. Gould, Kexin Dong, Yovel Lendner, Diego Detrés, Ondine Atwa, Jari Bulkens, Samuel Gruber, Manuel E. Contreras, Alexandra N. Wuest, Varun K. Narendra, Michael T. Hemann, Francisco J. Sánchez-Rivera
          </td>
          <td>2025-02-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Background Understanding the intricate tumor microenvironment (TME) is crucial for elucidating the mechanisms underlying the progression of cervical squamous cell carcinoma (CSCC) and its response to anti-PD-1 therapy. Methods In this study, we characterized 50,649 cells obtained from the CSCC for single-cell RNA sequencing and integrated bulk sequencing data from The Cancer Genome Atlas (TCGA) and clinical samples to explore their cell composition, metabolic processes, signaling pathways, specific transcription factors, lineage tracking and response to immunotherapy. In vivo experiments were performed to validate the function of key cell subsets. Results We identified ten major cell type and 35 subsets of stromal and immune cells in TME and observed distinct patterns in the metabolic processes and signaling pathways of these cells between tumor and normal tissues. Furthermore, PCNA clamp-associated factor (PCLAF)+ tumor-associated epithelial cell (TAEpis) was negatively correlated with the number of C-X-C motif chemokine ligand 13 (CXCL13)+ CD8+ T cells, overall survival, and response to anti-programmed cell death-1(PD-1) therapy in patients with CSCC. Both in vivo and in vitro experiments demonstrated that PCLAF+ TAEpis promotes the apoptosis of CD8+ T and tumor growth, while also inhibiting T cell infiltration and function. Conclusion Our findings illuminate the heterogeneity of the complex TME in CSCC and offer evidence supporting PCLAF+ TAEpis as a promising therapeutic target. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-03725-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c01c23a0f55159d7cf4e2578f437c46b0c10e1b" target='_blank'>
              Single-cell transcriptomic analyses reveal heterogeneity and key subsets associated with survival and response to PD-1 blockade in cervical squamous cell carcinoma
              </a>
            </td>
          <td>
            Xia Li, Zhao Zhao, Yanmei Cheng, Jiaqin Yan, Fang Ren, Yanyan Jia, Juanhua Li, Binhui Wang, Junqi Liu, Chenyin Wang, Meimei Gao, Hao Gu, Mingliang Fan, Huirong Shi, M. Ji, Qitai Zhao
          </td>
          <td>2025-03-13</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background Environmental stressors can induce heritable traits in organisms across phyla, with distinct epigenetic alterations in gametes and phenotypic outcomes across several generations. However, the mechanisms underlying such intergenerational inheritance, mainly from the germline to the germline and from the germline to the soma, are enigmatic, given that postfertilization embryos and germline cells reprogram the epigenome in each generation to gain their cellular identity. Here, we report stable germline transmission of differential DNA methylation alterations (epimutations) and their associations with nonalcoholic fatty liver disease (NAFLD) in medaka exposed to a model estrogenic chemical but a ubiquitous environmental contaminant, bisphenol A (BPA). Results Ancestral BPA exposure in the F0 generation led to advanced NAFLD in the unexposed grandchildren generation (F2) of medaka. The F2 liver transcriptome and histopathology revealed a severe NAFLD phenotype in females. Whole-genome bisulfite sequencing of the sperm and liver revealed a gradual shift in promoter methylation from F0 sperm (hypomethylated) to F1 sperm (mix of hypo- and hypermethylated) and F2 liver (predominantly hypermethylated). Many differentially methylated promoters (DMPs) overlapped in F0 sperm, F1 sperm, and F2 liver, regardless of sex. In females, stable transmission of 1511 DMPs was found across three generations, which are associated with protein-coding genes, miRNAs, and others and linked to NAFLD and nonalcoholic steatohepatitis (NASH). Among them, 27 canonical genes maintained consistently hypermethylated promoters across three generations, with significant downregulation of their expression and enrichment in NAFLD-related pathways, mainly fat digestion, glycerolipid metabolism, and steroid biosynthesis. Conclusions The present results demonstrate stable inter- and transgenerational germline-to-germline and germline-to-soma transmission of environmentally induced DNA epimutations with F0 and F1 gametic epimutations, predicting the F2 liver phenotype—a clear transgenerational passage of the disease phenotype in medaka.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c849a6fb22aa22762cf7232ec88c3cc765b79a2" target='_blank'>
              Stable Transmission of DNA Methylation Epimutations from Germlines to the Liver and Their Association with Fatty Liver Disease in Medaka
              </a>
            </td>
          <td>
            Sourav Chakraborty, Santosh Anand, Xuegeng Wang, R. Bhandari
          </td>
          <td>2025-02-14</td>
          <td>Research Square</td>
          <td>1</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f6c4d310ebc720bef67a3e653d0a6beb661d459" target='_blank'>
              Historical Evolution and CRISPR-Mediated Gene Editing Applications in NK Cell Therapy
              </a>
            </td>
          <td>
            Ziqing Yan
          </td>
          <td>2025-02-21</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The second and third most frequently diagnosed cancers worldwide are breast (2.3 million new cases) and colorectal (1.9 million new cases), respectively. Although advances in cancer therapies and early detection have improved the overall survival of patients, patients still develop resistance or cancer recurrence. Thus, the development of novel therapies that can affect multiple mechanisms of drug resistance and cell survival is ideal for the treatment of advanced and metastatic cancers. CHD1L is a novel oncogenic protein involved in regulating chromatin remodeling, DNA damage repair, epithelial-mesenchymal transition (EMT), and programmed cell death via PARthanatos. Herein, we assess in real-time how the CHD1L inhibitor (CHD1Li) OTI-611 modulates cell cycle progression in Colo678, SUM149PT, and SW620 cell lines. By utilizing a cell cycle reporter, we tracked the real-time cell cycle progression of cancer cells treated with OTI-611 alone and in combination with standard-of-care (SOC) therapies. Our results indicate that OTI-611 causes G1 phase cell cycle arrest through a CHD1L-mediated mechanism that regulates Cyclin D1 expression and localization. As a result of this mechanism, OTI-611 can reprogram the cell cycle effects of other antitumor agents to modulate and arrest cells in G1 when used in combination, including agents commonly known to arrest cells in the G2/M phase. Therefore, we conclude that OTI-611-induced G1 arrest represents a critical component of its unique mechanism of action, contributing significantly to its anticancer activity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/775d92956dd4575de627e0b27e219c4a04b6d817" target='_blank'>
              Targeted Inhibition of CHD1L by OTI-611 Reprograms Chemotherapy and Targeted Therapy-Induced Cell Cycle Arrest and Suppresses Proliferation to Produce Synergistic Antitumor Effects in Breast and Colorectal Cancer
              </a>
            </td>
          <td>
            Hector Esquer, Qiong Zhou, Daniel V. LaBarbera
          </td>
          <td>2025-02-20</td>
          <td>Cells</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="ABSTRACT Background In B‐cell precursor acute lymphoblastic leukemia (BCP‐ALL), chromosomal translocations are strongly associated with prognoses. RNA sequencing (RNA‐seq) is a powerful technology that reveals a close correlation between types of translocation and patterns of gene expression in clinical samples of BCP‐ALL. Cancer cell lines are powerful research tools, and thus, we built a larger series of BCP‐ALL cell lines and performed RNA‐seq analysis to confirm their utility as a model system. Methods We performed RNA‐seq in a total of 94 BCP‐ALL cell lines, including 80 cell lines with 8 representative types of translocations. Results In the UMAP visualization, a close association was confirmed between the types of fusion genes and patterns of gene expression. In the cluster analysis of the gene expression profile, each type of fusion gene showed a clear association with the expression profile in the top 51 variable genes. Of clinical importance, the majority of the top variable genes in the BCP‐ALL cell lines also showed a significant association with the types of fusion genes in the clinical samples. When an association of 125 cell cycle‐related genes with the percentage of S and G2/M phases in 67 cell lines was evaluated, a significant positive correlation with cell cycle progression was confirmed in 10 cell cycle‐related genes (HDAC2, CDC23, YWHAG, MAD2L1, CCNH, ANAPC7, CDC6, ANAPC5, ORC3, andRBX1). Moreover, significant upregulation and downregulation of 40 and 10 genes, respectively, were observed in the cell lines established at relapse compared with those established at diagnosis. Four (SP6, CCNE1, HIST1H2BH, and DECR2) and two (EVI2B and SYN1) of these genes were also significantly higher and lower, respectively, in the clinical samples at relapse than in those at diagnosis. Conclusion Large series of BCP‐ALL cell lines is a powerful research tool for studying the mechanisms of leukemogenesis and the disease progression of BCP‐ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c14bf4771e63afa160dc9961de41a9da67d0a47d" target='_blank'>
              Utility of a Large Series of B‐Cell Precursor Acute Lymphoblastic Leukemia Cell Lines as a Model System
              </a>
            </td>
          <td>
            Minori Tamai, Chiaki Komatsu, K. Kagami, S. Kasai, K. Akahane, K. Goi, Kanji Sugita, C. Tomoyasu, Toshihiko Imamura, Hiroaki Goto, Takeshi Inukai
          </td>
          <td>2025-03-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Whole genome doubling (WGD) is a frequent event in cancer evolution associated with chromosomal instability, metastasis, and poor prognosis. While the genomic consequences of WGD are well documented, the effects of non-genetic alterations that accompany WGD, such as changes to cell and nuclear size, on tetraploid (4N) cancer cell physiology are less understood. Here, we show that cell and nuclear volume do not always scale with DNA content after WGD in cancer cells, resulting in 4N cells that differ in size. We find that small size is associated with enhanced cell fitness, mitotic fidelity, and tumorigenicity in 4N cancer cells and with poor patient survival in WGD-positive human cancers. Overall, these results suggest that cell and nuclear size contribute to the tumorigenic potential of 4N cancer cells and could be an important prognostic marker in human tumors that undergo WGD. Statement of Significance We report that WGD generates tetraploid cancer cells that vary in size, with larger cells displaying high chromosomal instability and smaller cells exhibiting high fitness and tumorigenicity. Furthermore, WGD status and cancer cell nuclear size in human tumors correlated with patient survival, demonstrating the clinical relevance of this association.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49f68f3dcdb4dd414fdeb5e81fa097ba1603de66" target='_blank'>
              Cell and nuclear size are associated with chromosomal instability and tumorigenicity in cancer cells that undergo whole genome doubling
              </a>
            </td>
          <td>
            Mathew Bloomfield, Sydney Huth, Daniella McCausland, Ron Saad, Nazia Bano, Tran Chau, Megan Sweet, N. Baudoin, Andrew McCaffrey, Kai Fluet, Eva M. Schmelz, Uri Ben-David, Daniela Cimini
          </td>
          <td>2025-04-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cancer cell lines are valuable models for studying tumor biology, yet their genomic evolution during culture can compromise experimental reproducibility. We conducted a detailed genomic analysis of the triple-negative breast cancer cell line MDA-MB-231, examining sublines obtained from different sources, at various time points, and across distinct passages. We introduce the concept of intraline heterogeneity (ILH) to highlight the genomic variability observed among these sublines. Our analyses revealed extensive genomic diversity, including differences in single nucleotide variants (SNVs) and copy number alterations (CNAs). In particular, CNAs exhibited remarkable heterogeneity, with pronounced chromosomal gains and losses between sublines, underscoring the impact of genomic instability on ILH. These findings suggest that ILH may influence experimental outcomes, emphasizing the importance of considering passage-specific genomic characterization to ensure consistency and reliability in cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c7a90a295650f938dfb30675bf97324372367f9" target='_blank'>
              Intraline genomic heterogeneity of the triple-negative breast cancer MDA-MB-231 cell line
              </a>
            </td>
          <td>
            Nair Varela-Rouco, N. Estévez-Gómez, C. Fernández-Santiago, Laura Tomás, Miriam Pérez, Daniel García-Souto, J. Pasantes, R. Piñeiro, J. M. Alves, D. Posada
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Epigenetics is currently considered the investigation of inheritable changes in gene expression that do not rely on DNA sequence alteration. Significant epigenetic procedures are involved, such as DNA methylations, histone modifications, and non-coding RNA actions. It is confirmed through several investigations that epigenetic changes are associated with the formation, development, and metastasis of various cancers, such as colorectal cancer (CRC). The difference between epigenetic changes and genetic mutations is that the former could be reversed or prevented; therefore, cancer treatment and prevention could be achieved by restoring abnormal epigenetic events within the neoplastic cells. These treatments, consequently, cause the anti-tumour effects augmentation, drug resistance reduction, and host immune response stimulation. In this article, we begin our survey by exploring basic epigenetic mechanisms to understand epigenetic tools and strategies for treating colorectal cancer in monotherapy and combination with chemotherapy or immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/719dc00daf58fbc383533083eb81aecec4828ea4" target='_blank'>
              Epigenetic marvels: exploring the landscape of colorectal cancer treatment through cutting-edge epigenetic-based drug strategies
              </a>
            </td>
          <td>
            A. Tahghighi, Effat Seyedhashemi, Javad Mohammadi, Arash Moradi, Aria Esmaeili, Majid Pornour, Kimia Jafarifar, S. M. Ganji
          </td>
          <td>2025-02-22</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f39bd0ddc99a69058c299e72e0d1b4296584ccd0" target='_blank'>
              Therapeutic targeting of chromatin alterations in leukemia and solid tumors.
              </a>
            </td>
          <td>
            Florian Perner, Tobias Berg, D. Sasca, Sophie-Luise Mersiowsky, Jayant Y. Gadrey, Johanna Thomas, Michael W M Kühn, Michael Lübbert
          </td>
          <td>2025-04-03</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b338d775526840d18d525abb3d293c825f24385" target='_blank'>
              Abstract B011: Harnessing epigenetic and metabolic biomarkers for targeted immunotherapy in colorectal cancer: Advances in precision medicine for solid tumors
              </a>
            </td>
          <td>
            Ji Min Lee
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17971509b5a82b8103a0ae0d9b891a0480387fd5" target='_blank'>
              Multiplexed epigenetic memory editing using CRISPRoff sensitizes glioblastoma to chemotherapy.
              </a>
            </td>
          <td>
            K. Lin, C. Zou, Akane Hubbard, Sasha Sengelmann, L. Goudy, I-Ching Wang, Rohit Sharma, J. Pak, K. Foster, T. Ozawa, J. D. de Groot, Joanna J. Phillips, H. Vasudevan, D. Raleigh, Alexander Marson, Niren Murthy, Luke A. Gilbert, Mitchel S Berger, S. J. Liu
          </td>
          <td>2025-02-25</td>
          <td>Neuro-oncology</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="A previous study investigated a genomic region on chromosome 1 associated with reduced human immunodeficiency virus type 1 (HIV) set-point viral load, implicating CHD1L as a novel HIV inhibitory factor. However, given that regulatory variants can influence expression of multiple nearby genes, further work is necessary to determine the impact of genetic variants on other genes in the region. This study evaluates the potential for genetic regulation of PRKAB2, a gene located upstream of CHD1L and encoding the β2 regulatory subunit of the AMPK complex, and for downstream impacts on HIV pathogenesis. Using genotype and gene expression data from the Gene Expression Omnibus repository and Genotype-Tissue Expression database, we observed cell-type-specific correlations between CHD1L and PRKAB2 expression, with a strong positive association in whole blood and negative correlation in monocytes. Notably, we found that individuals with HIV set-point viral load associated variants exhibited significantly reduced PRKAB2 expression in imputed whole blood models and ex vivo monocytes. Functional analyses using PRKAB2 −/− induced pluripotent stem cells suggest that PRKAB2 loss-of-function may influence CHD1L expression, and genes regulating cytokine activity, growth factor signaling, and pluripotency pathways associated with HIV infection. These results suggest that gene expression changes driven by HIV set-point viral load associated variants in the chromosome 1 impact multiple genes and, by influencing expression of PRKAB2, may result in altered expression of critical immune signaling processes. These findings advance our understanding of the contribution of host genetics on HIV pathogenesis and identifies new targets for ex vivo functional studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fe7a9ea737912b94d19eda24d178570eca32c1c" target='_blank'>
              Chromosome 1 variants associated with decreased HIV set-point viral load correlate with PRKAB2 expression changes
              </a>
            </td>
          <td>
            Riley H. Tough, Paul J. McLaren
          </td>
          <td>2025-03-06</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Malignant pleural mesothelioma (MPM) is a rare yet highly aggressive malignancy with a severe prognosis. Compounded by the lack of effective treatment modalities, MPM remains a formidable health challenge. Therefore, the identification of actionable liabilities is critical for advancing precision medicine to combat this lethal disease. Here, we exploit an unbiased genome-wide CRISPR screen, integrating and cross-comparing three MPM cell lines with nonmalignant mesothelial cells, to selectively map the gene targets whose depletion indicates a common dependency in MPM cells. This systematic approach unveils a cohort of verifiable genes, among which BUB1, a mitotic checkpoint serine/threonine kinase, emerges as a high-confidence hit in cancer cells. Cellular and molecular studies demonstrate that genetic depletion or pharmacological inhibition of BUB1 profoundly impairs MPM cell survival and growth while inducing G2/M cell cycle arrest, cellular senescence, and apoptosis, and attenuating functional hallmarks of aggressive cancer cells. Transcriptomic profiling of BUB1-depleted cells discloses differential gene expression signatures congruent with cell fate phenotypes, including the reprogramming of mitotic network genes. Mechanistically, BUB1 is indispensable for the proper localization of essential mitotic regulators MAD1, MAD2, and Shugoshin (SGO1), thereby ensuring the functionality of the spindle assembly checkpoint (SAC). Furthermore, BUB1 ablation leads to cytokinesis failure and multinucleation, a phenotype characterized by the downregulation of CDC20, Cyclin A, and Cyclin B, and a reciprocal upregulation of the cyclin-dependent kinase inhibitor p21. Clinically, MPM tumors exhibit elevated levels of BUB1, and high BUB1 expression is associated with shorter patient survival. Our novel findings accentuate comparative CRISPR screens as a powerful platform to explore tumor cell-selective gene essentiality and propose BUB1 kinase as a potential marker and druggable vulnerability with therapeutic implications for MPM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/147c280cc9a37e6a39ce87d2c82cecf0b12eda26" target='_blank'>
              Genome-wide CRISPR screen identifies BUB1 kinase as a druggable vulnerability in malignant pleural mesothelioma
              </a>
            </td>
          <td>
            E. Cakiroglu, Sude Eris, Ozden Oz, G. Karakülah, Serif Senturk
          </td>
          <td>2025-04-03</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/632a6b07fed31db9bd0bc2462cc1bb896521dfbb" target='_blank'>
              Timing and clustering co-occurring genome amplifications in cancers
              </a>
            </td>
          <td>
            Sara Cocomello, Giovanni Santacatterina, R. Bergamin, Alice Antonello, G. Caravagna
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d82c852aa82f168fc01790167512bfdf3a4c0023" target='_blank'>
              RNA-sequencing: A reliable tool to unveil transcriptional landscape of paediatric B-other acute lymphoblastic leukaemia.
              </a>
            </td>
          <td>
            Clara Vicente-Garcés, Guerau Fernández, Elena Esperanza-Cebollada, Mercè Richarte-Franqués, Alba Crespo-Carrasco, Sara Montesdeoca, Ignacio Isola, Edurne Sarrate, Esther Cuatrecasas, Susana Rives, J. Dapena, Mireia Camós, Nerea Vega-García
          </td>
          <td>2025-03-30</td>
          <td>British journal of haematology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is an aggressive blood cancer characterized by poor survival outcomes. Further, due to the extreme molecular heterogeneity of the disease, drug treatment response varies from patient to patient. The variability of drug response can cause unnecessary treatment in more than half of the patients with no or partial therapy responses leading to severe side effects, monetary as well as time loss. Understanding the genetic risk factors underlying the drug response in AML can help with improved prediction of treatment responses and identification of biomarkers in addition to mechanistic insights to monitor treatment response. Here, we report the results of the first Exome-Wide Association Study (EWAS) of ex-vivo drug response performed to date with 175 AML cases and 47 drugs. We used information from 55,423 germline exonic SNPs to perform the analysis. We identified exome-wide significant (p < 9.02 × 10− 7) associations for rs113985677 in CCIN with tamoxifen response, rs115400838 in TRMT5 with idelalisib response, rs11878277 in HDGFL2 with entinostat, and rs2229092 in LTA associated with vorinostat response. Further, using multivariate genome-wide association analysis, we identified the association of rs11556165 in ATRAID, and rs11236938 in TSKU with the combined response of all 47 drugs and 29 nonchemotherapy drugs at the genome-wide significance level (p < 5 × 10− 8). Additionally, a significant association of rs35704242 in NIBAN1 was associated with the combined response for nonchemotherapy medicines (p = 2.51 × 10− 8), and BI.2536, gefitinib, and belinostat were identified as the central traits. Our study represents the first EWAS to date on ex-vivo drug response in AML and reports 7 new associated loci that help to understand the anticancer drug response in AML patients. Supplementary Information The online version contains supplementary material available at 10.1186/s12920-025-02130-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f625e5d452eb4083063d649ecd47887fbc68449c" target='_blank'>
              Exome-wide association study reveals 7 functional variants associated with ex-vivo drug response in acute myeloid leukemia patients
              </a>
            </td>
          <td>
            Anil K. Giri, Jake Lin, Konstantinos Kyriakidis, Garima Tripathi, H. Almusa
          </td>
          <td>2025-04-04</td>
          <td>BMC Medical Genomics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ca43f615285dd30f7033e316e2f7208e0ef3058" target='_blank'>
              A novel cancer-associated lncRNA, LINC01123, participates in tumor progression, metabolism, immune escape, and resistance
              </a>
            </td>
          <td>
            Qiang Liu, He Huang, Shuwen Zhang, Fangteng Liu, Ting Lou
          </td>
          <td>2025-04-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8599e8314fc9026e2db49b39f76e5203a6d7080" target='_blank'>
              Allele-specific genomics decodes gene targets and mechanisms of the non-coding genome
              </a>
            </td>
          <td>
            Tim P. Hasenbein, Sarah Hoelzl, S. Engelhardt, Daniel Andergassen
          </td>
          <td>2025-03-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Non-coding RNAs (ncRNAs) are finely tuned cellular regulators important for human cell growth and cancer progression. DUBR (Dppa2 upstream binding RNA, also known as linc00883) is a nuclear ncRNA first discovered in mice for its role in regulating myoblast differentiation through interactions with chromatin and DNA methyltransferases. High expression levels of this ncRNA are predictive of poor patient outcome in colon adenocarcinoma, suggesting that DUBR may be involved in controlling cancer growth. To elucidate its function, we used RAP-MS and RNA immunoprecipitation techniques which revealed its interaction with epigenetic maintenance proteins in the human colon cancer cell line HCT116. Further, ATAC-seq and RNA-seq were used to address its function in regulating the epigenome and transcriptome of HCT116 cells. Here we report that DUBR is a regulator of human colon cancer cell line HCT116 survival. Additionally, we find that the ncRNA DUBR regulates AP-1 transcription factor binding site accessibility at enhancers of genes involved in differentiation and morphogenesis through interactions with epigenetic proteins such as NuRD complex members HDAC1 and CHD4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee7c5cbf5b3479ea3f3880a032c93a1d4cc7dfa8" target='_blank'>
              DUBR non-coding RNA regulates gene expression by affecting AP-1 enhancer accessibility
              </a>
            </td>
          <td>
            Simone D Hall, Khoa Tran, Jonathan Zhu, Tong Su, Colleen A. McHugh
          </td>
          <td>2025-03-21</td>
          <td>Functional & Integrative Genomics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The use of CRISPR to knockdown or knockout genes is a powerful tool for understanding the specific role of a gene in disease development. However, it can cause many unanticipated changes to the transcriptome that are not detected by DNA amplification and Sanger sequencing of the target site. Various RNA-sequencing techniques can be used to identify these changes and effectively gauge the full impact of the CRISPR knockout, thereby providing a means of selecting appropriate clones for further experimentation. Background/Objectives: RNA-seq data from 4 CRISPR knockout experiments were analyzed and techniques developed to both confirm the success of the CRISPR modifications and identify potential issues. Methods: A broad-based analysis of RNA-sequencing data identified many CRISPR-based changes not identified by PCR amplification of DNA around the CRISPR target site. These changes included an inter-chromosomal fusion event, exon skipping, chromosomal truncation, and the unintentional transcriptional modification and amplification of a neighboring gene. Conclusions: The inadvertent modifications identified by the evaluation of 4 CRISPR experiments highlight the value of using RNA-seq to identify transcriptional changes to cells altered by CRISPR, many of which cannot be recognized by evaluating DNA alone. Specific guidelines are presented for designing and analyzing CRISPR experiments using RNA-seq data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/220a9a7ebf2665e1a3e4bd65d2ca79919ea760ce" target='_blank'>
              Techniques for Validating CRISPR Changes Using RNA-Sequencing Data
              </a>
            </td>
          <td>
            S. Rathe, Tracy A. Marko, Elizabeth N. Edwards, Paige Hazelton Ridder, Jyotika Varshney, Kyle B. Williams, James E. Johnson, B. Moriarity, D. A. Largaespada
          </td>
          <td>2025-03-24</td>
          <td>Genes</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e707dd4d2590e8deca1a1e653d9b70d94049240c" target='_blank'>
              Suppress the proliferation process of glioblastoma by breaking off the tumor - microenvironment interaction by CRIPSR -mediated technology
              </a>
            </td>
          <td>
            Wei Wang
          </td>
          <td>2025-02-27</td>
          <td>MedScien</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Genes and genetic variants associated with human disease are continually being discovered, but validating their causative roles and mechanisms remains a significant challenge. CRISPR/Cas9 genome editing in model organisms like zebrafish can enable phenotypic characterization of founder generation (F0) knockouts (Crispants), but existing approaches are not amenable to high-throughput genetic screening due to high variability, cost, and low phenotype penetrance. To overcome these challenges, here we provide guide RNA (gRNA) selection rules that enable high phenotypic penetrance of up to three simultaneous knockouts in F0 animals following injection of 1–2 gRNAs per gene. We demonstrate a strong transcriptomic overlap in our F0 knockouts and stable knockout lines that take several months to generate. We systematically evaluated this approach across 324 gRNAs targeting 125 genes and demonstrated its utility in studying epistasis, characterizing paralogous genes, and validating human disease gene phenotypes across multiple tissues. Applying our approach in a high-throughput manner, we screened and identified 10 novel neurodevelopmental disorders and 50 hearing genes not previously studied in zebrafish. Altogether, our approach achieves high phenotypic penetrance using low numbers of gRNAs per gene in F0 zebrafish, offering a robust pipeline for rapidly characterizing candidate human disease genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d442e32931385bdcd94b5a33eaea8e6b64c86c8" target='_blank'>
              Optimizing gRNA selection for high-penetrance F0 CRISPR screening for interrogating disease gene function
              </a>
            </td>
          <td>
            Sheng-Jia Lin, Kevin Huang, C. Petree, Wei Qin, Pratishtha Varshney, Gaurav K Varshney
          </td>
          <td>2025-02-27</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Chronic lymphocytic leukemia (CLL) is characterized by the accumulation of monoclonal mature B lymphocytes in peripheral blood, bone marrow, lymphoid tissues, and extranodal sites. Genes involved in RNA splicing such as SF3B1 and U1 are frequently mutated in CLL, leading to altered splicing and generation of tumor neoepitopes. To study the impact of these mutations on the tumor microenvironment (TME), we have developed a comprehensive single-cell atlas of unmutated CLL encompassing 26 bone marrow and lymph node tumor samples from 23 U-CLL patients with mutations in U1 (n=7), SF3B1 (n=8), or without mutations in splicing genes (n=10). We observed high intra-tumor heterogeneity, discerning 12 transcriptional programs, one linked to the U1 g.A3>C mutation and characterized by NFKB hyperactivation. T cell and NK compartments exhibited site- and mutation-specific enrichment, with increased CD4+ regulatory cells (Treg) and CD8+ exhausted cells in lymph nodes, while U1-mutant tumors showed increased CD8+ cytotoxic activity, with a predominance of effector-like CD8+ cells. Single-cell T cell receptor sequencing revealed clonotype expansion in U1-mutated tumors, particularly in CD8+ effector and exhausted cells, suggesting a neoantigen-driven immune response. Cell-to-cell interaction analysis identified CD44 as a key mediator in U1-mutated tumors, showing pro-B survival interactions as those involving MIF-CD44-CD74. Furthermore, interactions between CD80 on CLL cells and CTLA4 on Tregs and CD8+ exhausted were upregulated, reflecting an immunosuppressive phenotype associated with U1 mutated CLL. These findings highlight the complex interplay between mutations in CLL and the TME, offering novel avenues for alternative therapeutic strategies for U-CLL with mutations in U1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/338007fb60dcc4b1652c081c06c04390110d3e38" target='_blank'>
              U1 snRNA mutation reshapes tumor microenvironment in chronic lymphocytic leukemia: a role for CD44-mediated signaling
              </a>
            </td>
          <td>
            Sara López-Tamargo, Javier Fernández-Mateos, Pablo Bousquets-Muñoz, Laura Llaó-Cid, F. Nadeu, Ares M. Farran, Cristina Olivar-Fernández, Ana de la Fuente-González, Andrea Aran, Roberto Martínez-Soler, Europa Azucena González, Manel Juan, J. Martín-Subero, D. Colomer, E. Campo, A. Gutiérrez-Fernández, Xose S Puente
          </td>
          <td>2025-04-01</td>
          <td>None</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="Tongue squamous cell carcinoma (TSCC), a subtype of head and neck squamous cell carcinoma, is characterized by frequent chemoresistance. Genetic mutations commonly observed in TSCC play a critical role in malignant progression; thus, elucidating their functional significance is essential for developing effective treatment strategies. To more accurately investigate the relationship between mutations and chemoresistance, we established low-passage TSCC cells, CTSC-1, obtained from a chemoresistant patient, and CTSC-2, from a treatment-naïve patient. Sanger sequencing revealed a specific TP53 mutation (Q331*) in CTSC-1, leading to the loss of the tetramerization and C-terminal regulatory domains. Notably, CTSC-1 cells harboring TP53-Q331* and CTSC-2 cells with TP53 knockout that have been engineered to ectopically express TP53-Q331* exhibit enhanced chemoresistance and increased cancer stem cell-like properties. Mechanistically, TP53-Q331* upregulates the expression of inhibitor of DNA binding 2 (ID2), which is crucial for maintaining the stemness of TSCC cells. Subsequently, ID2 activates the expression of nucleotide excision repair (NER) pathway-related genes ERCC4 and ERCC8, thereby enhancing the chemoresistance in TSCC. In conclusion, our study demonstrates that the TP53-Q331* mutation enhances TSCC chemoresistance through an ID2-mediated NER pathway, providing a potential prognostic marker and therapeutic target for TSCC chemotherapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9563aecd404844781094bd5fefafcbc9075af8dc" target='_blank'>
              A Truncated Mutation of TP53 Promotes Chemoresistance in Tongue Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            Xiaoyun Du, Junheng Zheng, Xiangwan Lu, Yan Zhang
          </td>
          <td>2025-03-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95b9504bba9cb27a525f77f6f19beab513280064" target='_blank'>
              Ribosome Profiling Reveals Translational Reprogramming via mTOR Activation in Omacetaxine Resistant Multiple Myeloma.
              </a>
            </td>
          <td>
            Z. Walker, Katherine F. Vaeth, Amber Baldwin, Denis J Ohlstrom, Lauren T. Reiman, Kady A Dennis, Kate Matlin, Beau M. Idler, Brett M Stevens, Neelanjan Mukherjee, Daniel W. Sherbenou
          </td>
          <td>2025-03-06</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43a61455bde5450ac8c919b6a07222c913fc3cde" target='_blank'>
              Aberrant DNA methylation as a key modulator of cell death pathways: insights into cancer progression and other diseases.
              </a>
            </td>
          <td>
            Ambreen Zahoor, Rafia Khazer, Insha Mehraj, Ubaid Gani, Falah Fayaz, F. Khanday, S. S. Bhat
          </td>
          <td>2025-03-01</td>
          <td>Functional & integrative genomics</td>
          <td>1</td>
          <td>23</td>
        </tr>

        <tr id="Anti-PD-1/PD-L1 immune checkpoint blockade (ICB) therapy has revolutionized clinical cancer treatment, while abnormal PD-L1 or HLA-I expression in patients can significantly impact the therapeutic efficacy. Somatic mutations in cancer cells that modulate these critical regulators are closely associated with tumor progression and ICB response. However, a systematic interpretation of cancer immune-related mutations is still lacking. Here, we harnessed the ABEmax system to establish a large-scale sgRNA library encompassing approximately 820,000 sgRNAs that target all feasible serine/threonine/tyrosine residues across the human genome, which systematically unveiled thousands of novel mutations that decrease or augment PD-L1 or HLA-I expression. Beyond residues associated with phosphorylation events, our screens also identified functional mutations that affect mRNA or protein stability, DNA binding capacity, protein-protein interactions, and enzymatic catalytic activity, leading to either gene inactivation or activation. Notably, we uncovered certain mutations that concurrently modulate PD-L1 and HLA-I expression, represented by the clinically relevant mutation SETD2_Y1666. We demonstrated that this mutation induces consistent phenotypic effects across multiple cancer cell lines and enhances the efficacy of immunotherapy in different tumor models. Our findings provide an unprecedented resource of functional residues that regulate cancer immunosurveillance, offering valuable guidance for clinical diagnosis, ICB therapy, and the development of innovative drugs for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0151a36f604dedfea97d97f1dfa3c6ae2a57eb97" target='_blank'>
              Massively parallel interrogation of human functional variants modulating cancer immunosurveillance
              </a>
            </td>
          <td>
            Ying Liu, Yongshuo Liu, Xuran Niu, Ang Chen, Yizhou Li, Ying Yu, Binrui Mo, Zhiheng Liu, Tao Xu, Jie Cheng, Zeguang Wu, Wensheng Wei
          </td>
          <td>2025-03-19</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 Cancer remains a significant global health challenge, claiming numerous lives annually and posing complex hurdles in management. The disease's multifaceted nature, combined with disparities in healthcare infrastructure and socioeconomic factors worldwide, complicates effective treatment. With alarming mortality rates, there is an urgent need to enhance prevention and early detection strategies. Early detection not only enables less invasive treatments but also improves the chances of successful recovery, highlighting the importance of proactive approaches. Lung cancer ranks highest in mortality rates, followed by breast, prostate, and colon cancers. Cancer is often seen as both a "genetic disease," involving gene mutations, and an "environmental disease," influenced by various environmental factors affecting gene expression. Metabolic alterations are a hallmark of cancer, and disruptions in cellular stability and function significantly contribute to its progression. Non-coding RNAs (ncRNAs) play crucial regulatory roles across cancers, with diverse structures and functions. They act as oncogenes or tumor suppressors depending on the cancer type, intricately modulating genetic and epigenetic processes. Recent studies challenge the perception of ncRNAs' insignificance, revealing their substantial impact on regulatory mechanisms. Indeed, ncRNAs intricately connect genetic networks, influencing vital protein effectors that dictate cellular responses and fate. Consequently, dysregulated ncRNAs play a pivotal role in disease pathogenesis, acting as either oncogenic drivers or tumor suppressors through aberrant expression in various cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e52de2b0fb9cae18b101db41cf959bce17a1986" target='_blank'>
              The role of non-coding RNAs (ncRNAs) and their potential connection with cancer
              </a>
            </td>
          <td>
            Safura Absalan, Hamidreza Vaziri
          </td>
          <td>2025-03-22</td>
          <td>Egyptian Journal of Medical Human Genetics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a genetically and cellularly heterogeneous disease. We characterized 120 AMLs using genomic and transcriptomic analyses, including single-cell RNA sequencing. Our results reveal an extensive cellular heterogeneity that distorts the bulk transcriptomic profiles. Selective examination of the transcriptional signatures of >90,000 immature AML cells identified four main clusters, thereby extending current genomic classification of AML. Notably, NPM1 mutated AML could be stratified into two novel, clinically relevant classes, with NPM1class I associated with downregulation of MHC class II and excellent survival following hematopoietic stem cell transplantation (HSCT). NPM1class II was instead associated with resistance to allogeneic T cells in an ex vivo co culture assay, and importantly, dismal survival following HSCT. These findings provide new insights into the cellular state space of AML, define new diagnostic entities, and highlight potential therapeutic intervention points. Key Points The bulk transcriptional profiles of AML are mainly driven by a diverse set of cellular signatures. Single cell RNA-sequencing of the most common AML subtypes reveals marked heterogeneity extending beyond current genomic classification schemes. NPM1-mutated AML can be divided into two new classes, with distinct immune evasion mechanisms and survival after transplantation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15ce425737814eabb8f9dbfc7144c4c613998394" target='_blank'>
              The cellular state space of AML unveils novel NPM1 subtypes with distinct clinical outcomes and immune evasion properties
              </a>
            </td>
          <td>
            H. Lilljebjörn, Pablo Peña-Martínez, Hanna Thorsson, Rasmus Henningsson, M. Rissler, N. Landberg, Noelia Puente-Moncada, Sofia von Palffy, Vendela Rissler, Petr Stanek, Jonathan Desponds, Xiangfu Zhong, Gunnar Juliusson, Vladimir Lazarevic, Sören Lehmann, Magnus Fontes, H. Ågerstam, Carl Sandén, Christina Orsmark-Pietras, T. Fioretos
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/344c45deee3ee4d6c328b84a6e73872831228859" target='_blank'>
              Abstract B019: Cyclin E1 amplification promotes chromosomal instability and distinct vulnerabilities in gastroesophageal adenocarcinoma
              </a>
            </td>
          <td>
            Zhong Wu, Ankur Nagaraja, Chunyang Bao, Jin Zhou, A. Bass
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Neoantigen vaccines are under investigation for various cancers, including epidermal growth factor receptor (EGFR)-driven lung cancers1,2. We tracked the phylogenetic history of an EGFR mutant lung cancer treated with erlotinib, osimertinib, radiotherapy and a personalized neopeptide vaccine (NPV) targeting ten somatic mutations, including EGFR exon 19 deletion (ex19del). The ex19del mutation was clonal, but is likely to have appeared after a whole-genome doubling (WGD) event. Following osimertinib and NPV treatment, loss of the ex19del mutation was identified in a progressing small-cell-transformed liver metastasis. Circulating tumour DNA analyses tracking 467 somatic variants revealed the presence of this EGFR wild-type clone before vaccination and its expansion during osimertinib/NPV therapy. Despite systemic T cell reactivity to the vaccine-targeted ex19del neoantigen, the NPV failed to halt disease progression. The liver metastasis lost vaccine-targeted neoantigens through chromosomal instability and exhibited a hostile microenvironment, characterized by limited immune infiltration, low CXCL9 and elevated M2 macrophage levels. Neoantigens arising post-WGD were more likely to be absent in the progressing liver metastasis than those occurring pre-WGD, suggesting that prioritizing pre-WGD neoantigens may improve vaccine design. Data from the TRACERx 421 cohort3 provide evidence that pre-WGD mutations better represent clonal variants, and owing to their presence at multiple copy numbers, are less likely to be lost in metastatic transition. These data highlight the power of phylogenetic disease tracking and functional T cell profiling to understand mechanisms of immune escape during combination therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82bc21840fb421694721da36c78c3833f6d5089d" target='_blank'>
              Clonal driver neoantigen loss under EGFR TKI and immune selection pressures
              </a>
            </td>
          <td>
            M. Al Bakir, J. Reading, Samuel Gamble, R. Rosenthal, Imran Uddin, Andrew Rowan, J. Przewrocka, Amber Rogers, Y. Wong, A. Bentzen, S. Veeriah, S. Ward, Aaron T Garnett, Paula Kalavakur, C. Martínez-Ruiz, C. Puttick, A. Huebner, Danielle Cook, David A Moore, C. Abbosh, C. Hiley, C. Naceur-Lombardelli, T. Watkins, Marina Petković, Roland F Schwarz, F. Gálvez-Cancino, K. Litchfield, P. Meldgaard, B. Sørensen, L. Madsen, Dirk Jäger, Martin D Forster, T. Arkenau, C. Domingo-Vila, T. Tree, M. Kadivar, S. Hadrup, Benjamin M Chain, Sergio A Quezada, N. Mcgranahan, C. Swanton
          </td>
          <td>2025-02-19</td>
          <td>Nature</td>
          <td>1</td>
          <td>69</td>
        </tr>

        <tr id="The chemical modification of biological molecules is a critical regulatory mechanism for controlling molecular functions. Although research has long focused on DNA and proteins, RNA modifications have recently attracted substantial interest with the advancement in detection technologies. In oncology, many studies have identified dysregulated RNA modifications including m6A, m1A, m5C, m7G, pseudouridylation and A to I editing, leading to disrupted downstream pathways. As the concept of the tumor microenvironment has gained prominence, studies have increasingly examined the role of RNA modifications in this context, focusing on interactions among cancer cells, immune cells, stromal cells, and other components. Here we review the RNA modifications in the tumor microenvironment through the perspective of the Cancer-Immunity Cycle. The extracellular RNA modifications including exosomes and influence of microbiome in RNA modifications are potential research questions. Additionally, RNA modifying enzymes including FTO, ALKBH5, METTL3, PUS7 are under investigation as potential biomarkers and targets for combination with immunotherapies. ADCs and mimetics of modified RNA could be potential novel drugs. This review discusses the regulatory roles of RNA modifications within the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95bb55c6fe724814b8270d97b3e10e6a1daa2920" target='_blank'>
              RNA modifications in the tumor microenvironment: insights into the cancer-immunity cycle and beyond
              </a>
            </td>
          <td>
            You-Peng Ding, Cui-Cui Liu, Ke-Da Yu
          </td>
          <td>2025-04-02</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The metabolic enzyme aldehyde dehydrogenase 1A1 (ALDH1A1), a cancer stem cell marker associated with poor outcomes in breast cancer, has emerged as a promising therapeutic target in TNBC. The aim of this study was to investigate the role of ALDH1A1 in radiation resistance and redox stress in triple negative breast cancer (TNBC). Functional knockouts of ALDH1A1 were generated by the CRISPR/Cas9-mediated deletion of ALDH1A1 in the SUM159 cell line, and three distinct clonal populations were isolated. Genetic targeting was confirmed by Sanger sequencing, and the loss of ALDH1A1 protein expression was validated by Western blotting. Functional assays assessed ALDEFLUOR activity, cell viability, self-renewal capacity, and reactive oxygen species (ROS) levels with or without radiation in both the bulk population and clonal lines. Interestingly, ALDEFLUOR activity was uniformly lost across all clonal lines; however, functional effects of ALDH1A1 loss on redox stress, survival, and radiation sensitivity were observed in only one clonal population. These findings highlight significant variability in the role of ALDH1A1 among clonal populations, reflecting the complexity of tumor heterogeneity. This underscores the importance of accounting for tumor heterogeneity when targeting ALDH1A1, as certain TNBC subpopulations may rely more heavily on ALDH1A1 function. These insights are critical for developing effective ALDH1A1-targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce39f8fe1789f12e83b81bf503275e5a974f1d83" target='_blank'>
              CRISPR/Cas9 Targeting of Aldehyde Dehydrogenase 1A1 Reveals Heterogeneous Roles in Radiation Response and Redox Stress Across Clonal Lines in Triple-Negative Breast Cancer
              </a>
            </td>
          <td>
            Grace O. Ajayi, Aihui Ma, Shirin R. Modarai, Lynn M. Opdenaker, Jennifer Sims-Mourtada
          </td>
          <td>2025-03-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64abcfa81e97e4ccd6aa92a436d6301cbd6a137c" target='_blank'>
              CRISPR-Cas9: Transforming Functional Genomics, Precision Medicine, and Drug Development - Opportunities, Challenges, and Future Directions.
              </a>
            </td>
          <td>
            Zuber Khan, Mumtaz, Sumedha Gupta, Sidharth Mehan, Tarun Sharma, Manjeet Kumar, Pankaj Kumar Maurya, Arun Kumar Sharma, Ghanshyam Das Gupta, A. Narula
          </td>
          <td>2025-03-21</td>
          <td>Current gene therapy</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Continuous directed evolution is a powerful Synthetic Biology tool to engineer proteins with desired functions in vivo. Mimicking natural evolution, it involves repeated cycles of high-frequency mutagenesis, selection, and replication within platform cells, where the function of the target gene is tightly linked to the host cell’s fitness. However, cells might escape the selection pressure due to the inherent flexibility of their metabolism, which allows for adaptation. Whole-proteome analysis as well as targeted proteomics offer valuable insights into global and specific cellular changes. They can identify modifications in the target protein and its interactors to help understand its evolution and network integration. Using the continuous evolution of the Arabidopsis methionine synthases AtMS1 and AtMS2 as an example, we demonstrate how mass spectrometry-based proteomics can be applied in CDE, propose specific checkpoints for its integration and illustrate its role in informed decision making.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ef76b52b5701cadb64d2fef7237a2cb3da56d6d" target='_blank'>
              Harnessing Mass Spectrometry-Based Proteomics for Continuous Directed Evolution
              </a>
            </td>
          <td>
            Katharina Belt, David Obe, Mark A. Wilson, A. H. Millar, U. Bathe
          </td>
          <td>2025-03-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a6f1a197237e49a8b6045fada2a0bb94a7b46f0" target='_blank'>
              Gene Expression Profiling in Acute Myeloid Leukemia Patient Subgroups With High and Low Sensitivity Toward SYK Inhibitors.
              </a>
            </td>
          <td>
            Marte Karen Brattås, Franziska Görtler, Silje Johansen, K. P. Rye, K. Hatfield, H. Reikvam
          </td>
          <td>2025-03-01</td>
          <td>Hematological oncology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Mutations in the DNA can affect cancer development and progression not only by changing the amino acid chain but also by affecting different regulatory elements such as transcription factors. This study introduces a novel pipeline to identify “mutation blocks” - small genomic areas with high mutation rates that potentially influence transcription factors binding. By analyzing GCB lymphoma patient data, mutations blocks were identified that correlated with gene expression changes and were linked to transcription factor activity. These mutation blocks suggest a selection for mutations that alter gene regulation, contributing to lymphoma development. The analysis identified 56 mutation blocks in germinal center B-cell like diffuse large B-cell lymphoma (GCB DLBCL) patients’ genomes, affecting genes such as BCL2, MYC, SGK1, and PIM1, and linked to transcription factors including MSC, TCFL5, HOXB7, FOXP3, and ZBTB6. A machine learning model that used gene ontology data suggested further potential transcription factors-gene pairs influencing cancer. These findings highlight the role of synonymous and silent mutations in altering transcription factors binding and gene expression, offering insights into the mechanisms of GCB lymphoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76d3b64e91469debfcea770e479a146974e05ae2" target='_blank'>
              Novel Driver Mutations in GCB Lymphoma Patients That Affect Transcription Factors Binding
              </a>
            </td>
          <td>
            Ofek Shami-Schnitzer, T. Tuller
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b52e6b9621db88b32d01f7e93fc6430a88ce3be" target='_blank'>
              Comprehensive interrogation of synthetic lethality in the DNA damage response.
              </a>
            </td>
          <td>
            John Fielden, Sebastian M. Siegner, Danielle N Gallagher, Markus S Schröder, Maria Rosaria Dello Stritto, Simon Lam, L. Kobel, Moritz F. Schlapansky, S. P. Jackson, Petr Cejka, Marco Jost, Jacob E. Corn
          </td>
          <td>2025-04-09</td>
          <td>Nature</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Long noncoding RNAs (lncRNAs) play numerous roles in cellular biology and alterations in lncRNA expression profiles have been implicated in a variety of cancers. Here, we identify and characterize a lncRNA, TRIM28 Interacting DNA damage repair Enhancing Noncoding Transcript (TRIDENT), whose expression is induced upon epithelial growth factor receptor (EGFR) activation, and which exerts pro-oncogenic functions in EGFR-driven non-small cell lung cancer. Knocking down TRIDENT leads to decreased tumor-cell proliferation in both in vitro and in vivo model systems and induces sensitization to chemotherapeutic drugs. Using ChIRP-MS analysis we identified TRIM28 as a protein interactor of TRIDENT. TRIDENT promotes phosphorylation of TRIM28 and knocking down TRIDENT leads to accumulation of DNA damage in cancer cells via decreased TRIM28 phosphorylation. Altogether, our results reveal a molecular pathway in which TRIDENT regulates TRIM28 phosphorylation to promote tumor cell growth and drug resistance. Our findings suggest that TRIDENT can be developed as a biomarker or therapeutic target for EGFR mutant non-small cell lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80bdf03e598e81f8c991cd3c3dc15b156d2a5764" target='_blank'>
              EGFR-induced lncRNA TRIDENT promotes drug resistance in non-small cell lung cancer via phospho-TRIM28-mediated DNA damage repair.
              </a>
            </td>
          <td>
            Tanvi Saxena, Anan Quan, Erica Chan, Nina Kozlova, Latika Matai, Jonathan D. Lee, R. Rupaimoole, Francisco Beca, Taru Muranen, Frank J. Slack
          </td>
          <td>2025-03-03</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55c3773603d3c9014e1c679ae73200f35f924d76" target='_blank'>
              Single-cell SNP-DNA sequencing precisely maps genotoxic events in CRISPR-edited primary cells
              </a>
            </td>
          <td>
            J. Boutin, S. Fayet, Victor Marin, C. Berges, M. Riandière, Jérôme Toutain, I. Lamrissi‐Garcia, C. Thibault, David Cappellen, S. Dabernat, Arthur Poulet, Maëla Francillette, Nathalie Droin, F. Moreau-Gaudry, A. Bedel
          </td>
          <td>2025-04-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Background Acute myeloid leukemia (AML) is a prevalent malignancy of the hematologic system. Despite advancements in therapeutic approaches, significant heterogeneity and therapeutic resistance pose substantial challenges to treatment. Tumors driven by core transcription factors through super-enhancers can establish core transcriptional regulatory circuits (CRCs) that modulate oncogene expression programs. Identifying CRC is crucial for understanding disease-related transcriptional regulation. This study sought to predict and establish a CRC model for AML, identify genes critical for AML survival and explore their regulatory mechanisms in AML progression. Methods The dbCoRC tool was used for predictive analysis of H3K27ac ChIP-seq data from 11 AML samples to construct and validate the CRC model in AML patients. To elucidate the functional role of the CRC member IRF1, we utilized short hairpin RNA (shRNA) to knock down IRF1 in AML cells. RNA-seq, CUT&Tag and lipidomics technologies were subsequently used to investigate the regulatory roles and downstream mechanisms of IRF1 in AML. Results This study established a core transcriptional regulatory circuit consisting of IRF1, ELF1, ETV6, RUNX2, and MEF2D, which formed an interconnected autoregulatory loop. Further investigations revealed up-regulated expression of IRF1 in AML patients, which was associated with poor prognosis. Inhibition of IRF1 expression resulted in decreased AML cell proliferation and induced apoptosis, indicating its essential role in the survival of AML cells. Additionally, this study revealed that IRF1 directly regulates the transcription of key genes such as FASN, SCD, and SREBF1 for lipid synthesis, thereby affecting lipid metabolism in AML cells. Conclusion In summary, this study identified IRF1 as a novel core transcription factor involved in AML pathogenesis. IRF1 collaborates with ELF1, ETV6, RUNX2, and MEF2D to form a core transcriptional regulatory circuit that promotes AML progression. Furthermore, we demonstrated that IRF1 directly regulates the expression of key genes involved in lipid metabolism, influencing the synthesis of diverse lipid molecules crucial for AML survival. Supplementary Information The online version contains supplementary material available at 10.1186/s40164-025-00612-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fff624cce1bb80e7ca5736d4165b2d6247abf0ec" target='_blank'>
              IRF1 is a core transcriptional regulatory circuitry member promoting AML progression by regulating lipid metabolism
              </a>
            </td>
          <td>
            Fenli Zhang, Zhiheng Li, Fang Fang, Yixin Hu, Zhixu He, Yanfang Tao, Yizhen Li, Zimu Zhang, Bi Zhou, Ying Yang, Yumeng Wu, Yijun Wu, Zhongling Wei, Ailian Guo, Ling Xu, Yongping Zhang, Xiaolu Li, Yan Li, Chunxia Yang, Man Zhou, Jian Pan, Shaoyan Hu, Xiaoyan Yang
          </td>
          <td>2025-03-01</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/391b9899dd79924e706f8b48664a0903bd6932bd" target='_blank'>
              Abstract A011: ATM inhibitors exploit checkpoint defects and ATM-dependent double strand break repair to induce mitotic cell death in TP53-mutant glioblastoma
              </a>
            </td>
          <td>
            D. Laverty, Shiv K Gupta, Gary Bradshaw, Jiajia Chen, B. Carlson, Nery Matias Calmo, Shulan Tian, J. Sarkaria, Zachary D. Nagel
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18a99a90f864b231f78785ccac2d5080c6a00e33" target='_blank'>
              Explainable AI-driven prediction of APE1 inhibitors: enhancing cancer therapy with machine learning models and feature importance analysis.
              </a>
            </td>
          <td>
            Aga Basit Iqbal, Tariq Ahmad Masoodi, A. Bhat, M. Macha, Assif Assad, Syed Zubair Ahmad Shah
          </td>
          <td>2025-02-21</td>
          <td>Molecular diversity</td>
          <td>1</td>
          <td>35</td>
        </tr>

        <tr id="Pancreatic adenocarcinoma (PAAD) is a fatal malignant tumor of the digestive system, and immunotherapy has currently emerged as a key therapeutic approach for treating PAAD, with its efficacy closely linked to T-cell subsets and the tumor immune microenvironment. However, reliable predictive markers to guide clinical immunotherapy for PAAD are not available. We analyzed the single-cell RNA sequencing (scRNA-seq) data focused on PAAD from the GeneExpressionOmnibus (GEO) database. Then, the information from the Cancer Genome Atlas (TCGA) database was integrated to develop and validate a prognostic risk model derived from T-cell marker genes. Subsequently, the correlation between these risk models and the effectiveness of immunotherapy was explored. Analysis of scRNA-seq data uncovered six T-cell subtypes and 1837 T-cell differentially expressed genes (DEGs). Combining these data with the TCGA dataset, we constructed a T-cell prognostic risk model containing 16 DEGs, which can effectively predict patient survival and immunotherapy outcomes. We have found that patients in the low-risk group had better prognostic outcomes, increased immune cell infiltration, and signs of immune activation compared to those in the high-risk group. Additionally, analysis of tumor mutation burden showed higher mutation rates in patients with PAAD in the high-risk group. Risk scores with immune checkpoint gene expression and drug sensitivity analysis provide patients with multiple therapeutic targets and drug options. Our study constructed a prognostic risk model for PAAD patients based on T-cell marker genes, providing valuable insights into predicting patient prognosis and the effectiveness of immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2eccd9b99a66967879131e3edf663927ebf7fa1" target='_blank'>
              Construction of T-Cell-Related Prognostic Risk Models and Prediction of Tumor Immune Microenvironment Regulation in Pancreatic Adenocarcinoma via Integrated Analysis of Single-Cell RNA-Seq and Bulk RNA-Seq
              </a>
            </td>
          <td>
            Ding-ya Sun, Yijie Hu, Jun Peng, Shan Wang
          </td>
          <td>2025-03-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The appearance of cancer progresses through a multistep process that includes genetic, epigenetic, mutational, inflammatory and metabolic disturbances to signaling pathways within an organ. The combined influence of these changes will dictate the growth properties of the cells; the direction of further malignancy depends on the severity of these “disturbances”. The molecular mechanisms driving abnormal inflammation and metabolism are beginning to be identified and, in some cases, are quite prominent in pre-condition states of cancer and are significant drivers of the malignant phenotype. As such, utilizing signaling pathways linked to inflammation and metabolism as biomarkers of cancer is an emerging method and includes pathways beyond those well characterized to drive metabolism or inflammation. In this review, we will discuss several emerging elements influencing proliferation, inflammation and metabolism that may play a part as drivers of the cancer phenotype. These include AMPK and leptin (linked to metabolism), NOD2/RIPK2, TAK1 (linked to inflammation), lactate and pyruvate transporters (monocarboxylate transporter [MCT], linked to mitochondrial biogenesis and metabolism) and RASSF1A (linked to proliferation, cell death, cell cycle control, inflammation and epigenetics). We speculate that the aforementioned elements are important drivers of carcinogenesis that should be collectively referenced as being involved in “ImmunoMET Oncogenesis”, a new tripartite description of the role of elements in driving cancer. This term would suggest that for a better understanding of cancer, we need to understand how proliferation, inflammation and metabolic pathways are impacted and how they influence classical drivers of malignant transformation in order to drive ImmunoMET oncogenesis and the malignant state.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f600df6865b40197b8b8a33d72a983d145a378e4" target='_blank'>
              ImmunoMet Oncogenesis: A New Concept to Understand the Molecular Drivers of Cancer
              </a>
            </td>
          <td>
            Reshma Sirajee, Sami El Khatib, L. Dieleman, Mohamed Salla, S. Baksh
          </td>
          <td>2025-02-27</td>
          <td>Journal of Clinical Medicine</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Background Recurrent/metastatic head and neck squamous cell carcinoma (R/M-HNSCC) is a severe, frequently lethal condition. Oncogene addiction to epidermal growth factor receptor (EGFR) is a hallmark of HNSCC, but the clinical efficacy of EGFR-targeted therapies remains low. Understanding molecular networks governing EGFR-driven progression is paramount to the exploration of (co)-treatment targets and predictive markers. Methods We performed function-based mapping of differentially expressed genes in EGFR-mediated local invasion (fDEGs) using photoconvertible tracers and RNA-sequencing (RNA-seq) in a cellular 3D-model. Results Upon alignment with public single-cell RNA-seq (scRNA-seq) datasets and HNSCC-specific regulons, a gene regulatory network of local invasion (invGRN) was inferred from gene expression data, which was overrepresented in budding tumors. InvGRN comprises the central hubs inhibin subunit beta alpha (INHBA) and snail family transcriptional repressor 2 (SNAI2), and druggable fDEGs integrin subunit beta 4 (ITGB4), laminin 5 (LAMB3/LAMC2), and sphingosine kinase 1 (SPHK1). Blockade of INHBA repressed local invasion and was reverted by activin A, laminin 5, and sphingosine-1-phosphate, demonstrating a functional interconnectivity of the invGRN. Epithelial-to-mesenchymal transition (EMT) of malignant cells and the invGRN are induced by newly defined EGFR-activity subtypes with prognostic value that are promoted by amphiregulin (AREG) and epiregulin (EREG). Importantly, co-inhibition of SPHK1 showed synthetic effects on Cetuximab-mediated invasion blockade and high expression of selected fDEGs was associated with response to Cetuximab in patient-derived xenotransplantation (PDX) and R/M-HNSCC patients. Conclusions We describe an actionable network of EGFR-mediated local invasion and define druggable effectors with predictive potential regarding the response of R/M-HNSCC to Cetuximab. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02290-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a3a765a8bcbfd4bc6d82c05747a49dc01756024" target='_blank'>
              EGFR-mediated local invasiveness and response to Cetuximab in head and neck cancer
              </a>
            </td>
          <td>
            Jiefu Zhou, Min He, Qiong Zhao, Enxian Shi, Hairong Wang, Vaidehi Ponkshe, Jiahang Song, Zhengquan Wu, Dongmei Ji, Gisela Kranz, Anna Tscherne, S. Schwenk‑Zieger, Nilofer Abdul Razak, J. Hess, C. Belka, H. Zitzelsberger, I. Ourailidis, F. Stögbauer, M. Boxberg, J. Budczies, Christoph A. Reichel, M. Canis, Philipp Baumeister, Hongxia Wang, Kristian Unger, Andreas Mock, O. Gires
          </td>
          <td>2025-03-22</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b3a2a7fdf1a317ed911bfc6a066f5d8abc14563" target='_blank'>
              Exploring the Function of METTL3 in Lung Cancer: Understanding Drug Resistance and Exploring Innovative Therapeutic Approaches
              </a>
            </td>
          <td>
            Shiqing Huang, Ye Li, F. Liew
          </td>
          <td>2025-03-12</td>
          <td>Current Cancer Therapy Reviews</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20366f8a8e956d20a8164d2739ca733da77de50e" target='_blank'>
              RUNX1::RUNX1T1 Depletion Eliminates Stemness and Induces Bidirectional Differentiation of Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Polina K Derevyanko, L. Swart, L. Daniel, Mata Casimiro, Luca van den Brink, Anita T. van Oort, Minoo Ashtiani, C. Zwaan, Anja Krippner-Heidenreich, Constanze Bonifer, Raymond Schiffelers, H. J. Vormoor, S. Kellaway, Olaf Heidenreich
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eed4652faa332d57f89c15c04ea896c21df38766" target='_blank'>
              Individualized pseudogenes networks for survival prognosis in B-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Arturo Kenzuke Nakamura-García, M. Kuijjer, J. Espinal-Enríquez
          </td>
          <td>2025-03-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0045e60f5f09ff48941f2c2a8009799ec50aa5d7" target='_blank'>
              Stem Cell-Parenchymal Fusion: Communication and Gene Regulation
              </a>
            </td>
          <td>
            JoAnne Huang, Fateme Nazaryabrbekoh, Joohyun Kim, Jangwook P. Jung
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Introduction: Leukocyte adhesion deficiency type 1 (LAD1) is a severe inborn error of immunity caused by mutations in the ITGB2 gene, which encodes the beta-2 integrin subunit (CD18). These mutations lead to the absence or deficiency of CD18/CD11a, b, and c heterodimers, crucial for leukocyte adhesion and immune function. CRISPR-Cas9 Gene editing technology represents a promising approach for correcting these genomic defects restore the stable expression of CD18 and reverse the disease. Methods: We developed a CRISPR-Cas9-based gene correction strategy using Jurkat cells and patient-derived lymphoblastoid cell lines as surrogates for hematopoietic progenitor cells. Three candidate gRNAs were first predicted in silico using CRISPOR and experimentally tested in wild-type ITGB2-expressing Jurkat cells to identify the gRNA with the highest genomic DNA cleavage efficiency. The most efficient gRNA was then paired with espCas9 and used alongside five homology-directed repair templates (HDRs) (single-stranded donor oligonucleotides, ssODNs) to repair ITGB2 defects in patient-derived lymphoblastoid cell lines. CD18 expression levels in edited cells were quantified via flow cytometry, and whole-genome sequencing (WGS) was conducted to assess off-target effects and insertion accuracy. Results: Among the three candidate gRNAs, 2-rev gRNA exhibited the highest genomic cleavage rate in Jurkat cells. Using this gRNA with espCas9 and HDR-2, we achieved a 23% restoration of CD18 expression in LAD1 patient-derived cells, a level sufficient to change the disease course from severe to moderate. Whole-genome sequencing confirmed the absence of off-target mutations or undesired DNA insertions, demonstrating high specificity and precision in gene correction. Discussion: This CRISPR-Cas9-based method provides a precise and effective approach for correcting ITGB2 mutations in LAD1 patients. The high-fidelity gene editing process, validated through WGS, supports its potential for future applications in CD34+ hematopoietic stem cell therapies. The approach can be further optimized for clinical translation, offering a path toward a stable and long-term cure for LAD1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af64b4b32aef04c80bbc3b2f45c925d4b04fb25b" target='_blank'>
              Lymphoblastoid and Jurkat cell lines are useful surrogate in developing a CRISPR-Cas9 method to correct leukocyte adhesion deficiency genomic defect
              </a>
            </td>
          <td>
            Ahmad R. Ramadan, Noureddine Ben Khalaf, Khaled Trabelsi, H. Bakheit, I. Ben‐Mustapha, M. Barbouche, M. Fathallah
          </td>
          <td>2025-03-21</td>
          <td>Frontiers in Bioengineering and Biotechnology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background: Clear cell renal cell carcinoma (ccRCC) is a prevalent and aggressive subtype of kidney cancer, often associated with metastasis and recurrence. Identifying key genes involved in ccRCC progression is critical for improving treatment strategies and patient outcomes. Methods: We performed a large-scale genome-wide CRISPR screening to identify genes crucial to ccRCC progression using the DepMap database. For discovery and validation, we integrated multi-omics data from The Cancer Genome Atlas (TCGA), GEO, and the NJMU-ccRCC clinical cohort. Bioinformatics analyses, including differential expression, pathway enrichment, and protein-protein interaction network analysis, were conducted to elucidate the biological functions. To validate our findings, we employed immunohistochemistry, qRT-PCR, and various cellular assays to investigate the role of PRC1 in ccRCC. Results: CRISPR screening identified PRC1 as a key gene significantly overexpressed in ccRCC tissues from the DepMap database. Elevated PRC1 expression was associated with poor overall survival, disease-specific survival, and progression-free interval. Silencing PRC1 in ccRCC cell lines inhibited cell proliferation, migration, and colony formation. Functional enrichment analyses revealed that PRC1 is involved in essential processes such as cell cycle regulation, mitosis, and cytokinesis. Additionally, PRC1 expression was correlated with the activation of the Wnt/β-catenin pathway, suggesting that PRC1 plays a pivotal role in tumor progression. Conclusion: PRC1 emerges as a promising biomarker and therapeutic target for ccRCC. Elevated PRC1 expression is associated with poor prognosis, and its inhibition suppresses ccRCC cell proliferation and migration. Our findings underscore the crucial role of PRC1 in ccRCC progression and highlight the need for further investigation into its molecular mechanisms and therapeutic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f7e34d29eb95f0f25970817abd06150852dbc62" target='_blank'>
              Large-scale genomic-wide CRISPR screening revealed PRC1 as a tumor essential candidate in clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            Baochao Li, Yongsheng Pan, Jiajin Wu, Chenkui Miao, Zengjun Wang
          </td>
          <td>2025-03-03</td>
          <td>International Journal of Medical Sciences</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b71c8a20f34594db8b2adcb5dc95c1f155e6425" target='_blank'>
              A Transcriptome-Wide Mendelian Randomization Study in Isolated Human Immune Cells Highlights Risk Genes Involved in Viral Infections and Potential Drug Repurposing Opportunities for Schizophrenia.
              </a>
            </td>
          <td>
            David Stacey, PhD Liam Gaziano, PhD Preethi Eldi, PhD Catherine Toben, PhD Beben Benyamin, PhD S Hong Lee, PhD Elina Hyppönen
          </td>
          <td>2025-03-24</td>
          <td>American journal of medical genetics. Part B, Neuropsychiatric genetics : the official publication of the International Society of Psychiatric Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Human T-cell lymphotropic virus type 1 (HTLV-1) infection causes the uncommon and deadly cancer known as adult T-cell leukemia/lymphoma (ATLL), which affects mature T cells. Its clinical appearance is varied, and its prognosis is often miserable. Drug resistance to conventional therapies confers significant therapeutic challenges in the management of ATLL. This review discusses the emerging role of epigenetic medical advances in the treatment of ATLL, focusing on DNA methyltransferase inhibitors, histone deacetylase inhibitors, histone methyltransferase inhibitors, and BET inhibitors. Indeed, several classes of epigenetic therapies currently exhibit trailed efficacy in preclinical and clinical studies: DNA methyltransferase inhibitors like azacitidine and decitabine reexpression of silenced tumor suppressors; histone deacetylase inhibitors like vorinostat and romidepsin induce cell cycle arrest and apoptosis; bromodomain and extra-terminal inhibitors like JQ1 disrupt oncogenic signaling pathways. Whereas preclinical and early clinical data indicate modest to good efficacy for such treatments, significant challenges remain. Here, we discuss the current state of understanding of epigenetic dysregulation in ATLL and appraise the evidence supporting the use of these epi-drugs. However, despite the opened doors of epigenetic treatment, much more research is required with regard to showing the best combinations of drugs and their resistance mechanisms, the minimization of adverse effects, and how this hope will eventually be translated into benefit for the patient with ATLL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a1ce69e97a9adaccd0f62087c29dd1f173f2a27" target='_blank'>
              Advances in epigenetic treatment of adult T-cell leukemia/lymphoma: a comprehensive review
              </a>
            </td>
          <td>
            Arash Letafati, Rabeeh Mehdigholian Chaijani, Fahimeh Edalat, Nazila Eslami, Hanieh Askari, Farideh Askari, Sara Shirvani, Hamed Talebzadeh, Mahdiyeh Tarahomi, Nila MirKhani, Faeze Karimi, Mehdi Norouzi, Sayed-Hamidreza Mozhgani
          </td>
          <td>2025-03-01</td>
          <td>Clinical Epigenetics</td>
          <td>1</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ee52870e604baf41aae12c6d88c6bacc4d5a388" target='_blank'>
              Updates on DNA Repair Gene Deficiency in Colorectal Cancer (dMMR)
              </a>
            </td>
          <td>
            Gustavo Sevá-Pereira, C. S. R. Coy, Carlos Augusto Real Martinez
          </td>
          <td>2025-03-01</td>
          <td>Journal of Coloproctology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Investigating the pivotal role of CTHRC1 in the tumor microenvironment of triple-negative breast cancer (TNBC). Method The RNA transcriptomic data obtained from the Cancer Genome Atlas and single-cell sequencing data from TNBC in Gene Expression Omnibus (GEO) were acquired and subjected to analysis. A comprehensive investigation was conducted with a specific focus on characterizing CTHRC1 in TNBC and its correlation with invasive genes. Furthermore, additional analyses were performed to explore the relationship between CTHRC1, tumor immune cell infiltration, and immunotherapy in TNBC. The expression of CTHRC1 in the tumor microenvironment, cellular differentiation, and cellular communication was systematically analyzed using single-cell data from TNBC. Result The expression of CTHRC1 in patients with TNBC gradually increases concomitantly with the progression of tumor T-stage and N-stage. Simultaneously, there is a concurrent increase in the expression of most invasive gene sets. Furthermore, there is a significant augmentation in both infiltration abundance and activity of M2-type macrophages associated with elevated levels of CTHRC1 expression. Single-cell data reveal an upregulated expression of the invasive gene set in CTHRC1-positive cancer associated fibroblasts (CAFs), thereby modulating their interaction with M2-type macrophages. Multiple immunofluorescence analyses confirmed that CTHRC1 modulates immune cell infiltration and tumor cell invasion through the mediation of CAFs. Conclusion CTHRC1 was a molecule that exhibits characteristic expression in TNBC. CTHRC1 positive CAFs exert regulatory effects within the immunosuppressive microenvironment of TNBC by modulating M2-type macrophages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4be4e81b525de2c62cbaf620445907b577884546" target='_blank'>
              Single-cell and transcriptome analyses revealed CTHRC1 a potential therapeutic target mediating invasion and tumor microenvironment in TNBC: experimental validation
              </a>
            </td>
          <td>
            Hong Wan, Zichen Ling, Yuwei Xie, Han Jiang, Zhifan Ruan, Dashuai Yang, Xiaowei Yang, Jing Pei
          </td>
          <td>2025-03-11</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="MYC plays a pivotal role in the biology of various sarcoma subtypes, acting as a key regulator of tumor growth, proliferation, and metabolic reprogramming. This oncogene is frequently dysregulated across different sarcomas, where its expression is closely intertwined with the molecular features unique to each subtype. MYC interacts with critical pathways such as cell cycle regulation, apoptosis, and angiogenesis, amplifying tumor aggressiveness and resistance to standard therapies. Furthermore, MYC influences the tumor microenvironment by modulating cell–extracellular matrix interactions and immune evasion mechanisms, further complicating therapeutic management. Despite its well-established centrality in sarcoma pathogenesis, targeting MYC directly remains challenging due to its “undruggable” protein structure. However, emerging therapeutic strategies, including indirect MYC inhibition via epigenetic modulators, transcriptional machinery disruptors, and metabolic pathway inhibitors, offer new hope for sarcoma treatment. This review underscores the importance of understanding the intricate roles of MYC across sarcoma subtypes to guide the development of effective targeted therapies. Given MYC’s central role in tumorigenesis and progression, innovative approaches aiming at MYC inhibition could transform the therapeutic landscape for sarcoma patients, providing a much-needed avenue to overcome therapeutic resistance and improve clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b3e9fb9dbedb9674622b99a03bb647022b7fb21" target='_blank'>
              Untangling the Role of MYC in Sarcomas and Its Potential as a Promising Therapeutic Target
              </a>
            </td>
          <td>
            Fabio Sias, Stefano Zoroddu, Rossana Migheli, L. Bagella
          </td>
          <td>2025-02-25</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="CRISPR-Cas technology has revolutionized biology by enabling precise DNA and RNA edits with ease. However, significant challenges remain for translating this technology into clinical applications. Traditional protein engineering methods, such as rational design, mutagenesis screens, and directed evolution, have been used to address issues like low efficacy, specificity, and high immunogenicity. These methods are labor-intensive, time-consuming, and resource-intensive and often require detailed structural knowledge. Recently, computational strategies have emerged as powerful solutions to these limitations. Using artificial intelligence (AI) and machine learning (ML), the discovery and design of novel gene-editing enzymes can be streamlined. AI/ML models predict activity, specificity, and immunogenicity while also enhancing mutagenesis screens and directed evolution. These approaches not only accelerate rational design but also create new opportunities for developing safer and more efficient genome-editing tools, which could eventually be translated into the clinic.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6cb55c796a93b1a1e6ebc7882f1d2a726253cbc6" target='_blank'>
              Engineering a New Generation of Gene Editors: Integrating Synthetic Biology and AI Innovations
              </a>
            </td>
          <td>
            B. Chia, Yu Fen Samantha Seah, Bolun Wang, Kimberle Shen, Diya Srivastava, W. Chew
          </td>
          <td>2025-02-25</td>
          <td>ACS Synthetic Biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Tissues and organs develop from single founder cells, which give rise to distinct cell lineages that contribute to regeneration and maintenance of homeostasis in the adult. Tracing the genealogical relationships between individual cells and their gene expression signatures is an important step towards understanding how these processes are regulated in human health and disease. Here, we present mt-SCITE, a computational method for inferring the evolutionary history of dividing cells based on mitochondrial mutations detected in single cells. We show that mt-SCITE outperforms existing methods in accurately estimating the correct mitochondrial mutation tree from simulated data. To validate our method, we applied mt-SCITE on in vitro expanded T-cells profiled with bulk ATAC sequencing, where the clonal relationships were determined independently from T cell receptor sequences. We used our method to reconstruct the division histories and transcriptional heterogeneity of clonally related CD8+ T cells from single-cell RNA sequencing data obtained from a healthy human donor. This allowed us to track the in vivo development of CD8+ T cell clones post vaccination and resolve subclonal lineage relationships without the need for genetic barcoding. Our analysis revealed that T cell clones can adopt predominantly memory-like or effector-like states, while some exhibit mixed identities. Taken together, our method represents a novel approach to study cell lineage development in humans.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b46463e933e01c11da7ed98772b7d2ff4e8ab033" target='_blank'>
              Cell lineage inference from mitochondrial mutations
              </a>
            </td>
          <td>
            Joanna Hård, S. Seidel, Pedro F. Ferreira, Pietro Berkes, Katharina Jahn, Carl-Johan Eriksson, Jeff E. Mold, Jakob Michaelsson, Jack Kuipers, N. Beerenwinkel
          </td>
          <td>2025-03-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/471fa22c73f7c1a8083ff473dd937339ee492d0c" target='_blank'>
              Abstract A022: Decoding aneuploidy: Identifying drivers and therapeutic targets in recurrent breast cancer copy number alterations
              </a>
            </td>
          <td>
            Khalid N. Al-Zahrani, Ellen Langille, A. Obersterescu, Christopher Lowden, Katie Teng, Lauren Caldwell, David Cook, Miguel Pérez-Castro, Cynthia H Chiu, Alec Bahcheli, R. Tsai, Jacob Berman, Kin Chan, Linkang Zhang, K.W. Annie Bang, Michael Parsons, Adele Lopes, Jocelyn Nurtanto, E. I. Temel, Iosifina Fotiadou, Julien Dessapt, Hartland Jackson, Sean E Egan, J. Reimand, Jeffrey Wrana, Daniel Schramek
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="ABSTRACT Human endogenous retroviruses (HERVs) are proviral relics of infections that affected primates' germ line. Many HERV elements retain a residual capacity to encode transcripts and proteins that have been occasionally domesticated for the host physiology. In addition, HERV transcriptional modulation is of great interest to clarify the etiology of complex disorders such as cancer, even if a few studies assessed the specific HERV loci modulated in tumor tissues. In the present work, we used a transcriptomic approach to investigate the specific expression of ~3300 HERV loci in paired tumor and normal tissues of 7 colorectal cancer (CRC) patients. A total of 102 HERVs were significantly modulated in CRC, with a general tendency towards downregulation. Of note, among the 42 upregulated HERVs 23 belonged to the HERV‐H group, that is the most investigated in CRC. De novo transcriptome reconstruction and qPCR validation allowed to identify a transcript from a HERV‐H locus on chromosome Xp22.3 with high specific expression in CRC samples, potentially encoding for a partial Pol protein. These results provide a detailed description of HERV transcriptional variations in CRC and its interindividual variability, identifying a HERV‐H transcript that deserves further investigation for its possible impact on tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8d57225f528a23471e9e5e65e4532aff3132bd2" target='_blank'>
              HERV Modulation in Colorectal Carcinoma Patients: A Snapshot of Endogenous Retroviral Transcriptome
              </a>
            </td>
          <td>
            Nicole Grandi, Ching-Hsuan Liu, Saili Chabukswar, Daniele Carta, Yun Yen, Liang-Tzung Lin, E. Tramontano
          </td>
          <td>2025-02-24</td>
          <td>Journal of Medical Virology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Background Tuberculosis, a persistent global health threat, necessitates a comprehensive understanding of the genes and pathways crucial for the survival and virulence of the causative pathogen, Mycobacterium tuberculosis. Working with M. tuberculosis (M.tb) presents significant challenges; therefore, the use of M. smegmatis as a surrogate system for conducting genetic studies of M.tb has proven to be highly valuable. Development of novel genetic tools to probe cellular processes accelerates the progress in the field of drug development and also helps in understanding the basic physiology of the bacterium. Results This study reports the successful implementation and evaluation of the CRISPR-Cas12a system for gene repression in Mycobacterium smegmatis, a surrogate for M. tuberculosis. We engineered a Cas12a-based CRISPR interference (CRISPRi) system and assessed its functionality. Targeting 45 genes with a single sgRNA per gene, we achieved efficient gene repression, leading to marked phenotypic changes. Each knockdown strain was evaluated individually for growth phenotypes, and a comparison of the results with the reported essential gene library probed with dCas9 demonstrated congruous results across diverse gene categories. The study shows that CRISPR/Cas12a system can be effectively utilised with a single gene specific target for efficient silencing of the gene and highlights the importance of subsequent growth assays required to evaluate the vulnerability of targeted gene silencing. Conclusion Our findings reveal the robustness and versatility of the dCas12a-CRISPRi system in M. smegmatis, providing a valuable tool for functional genomics research. This work showcases the potential of the dCas12a-CRISPRi system in investigating essential genes, enabling a deeper understanding of the biology and potential therapeutic targets in mycobacterium species. Supplementary Information The online version contains supplementary material available at 10.1186/s13036-025-00490-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fd956491f5700a578d8ac40b6d89d08cd7f2d58" target='_blank'>
              CRISPR/Cas12a-mediated gene silencing across diverse functional genes demonstrates single gene-specific spacer efficacy in Mycobacterium smegmatis
              </a>
            </td>
          <td>
            Megha Sodani, C. S. Misra, Savita Kulkarni, D. Rath
          </td>
          <td>2025-02-28</td>
          <td>Journal of Biological Engineering</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5712c0323ff04880c8d582570b76deb2d3982f10" target='_blank'>
              Basic-Leucine-Zipper Transcription Factors Regulate Selective Molecular Phenotypes in Regulatory T Cells During IL-2-Induced Activation
              </a>
            </td>
          <td>
            Joyce Tse, Xuan Liu, Ittai Eres, Tracy Yamawaki, Matt Kanke, Marisela Killian, Junedh M. Amrute, Ashutosh Chaudhry, Anupama Sahoo, Wenjun Liu, Cheng-Yuan Kao, Xin Luo, Jiamiao Lu, Daniel R Lu, Songli Wang, Scott Martin, Chi-Ming Li
          </td>
          <td>2025-02-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background Cell-free DNA (cfDNA) in plasma carries epigenetic signatures specific to tissue or cell of origin. Aberrant methylation patterns in circulating cfDNA have emerged as valuable tools for noninvasive cancer detection, prenatal diagnostics, and organ transplant assessment. Such epigenetic changes also hold significant promise for the diagnosis of neurodegenerative diseases, which often progresses slowly and has a lengthy asymptomatic period. However, genome-wide cfDNA methylation changes in neurodegenerative diseases remain poorly understood. Results We used whole-genome bisulfite sequencing (WGBS) to profile age-dependent and ALS-associated methylation signatures in cfDNA from 30 individuals, including young and middle-aged controls, as well as ALS patients with matched controls. We identified 5,223 age-related differentially methylated loci (DMLs) (FDR < 0.05), with 51.6% showing hypomethylation in older individuals. Our results significantly overlapped with age-associated CpGs identified in a large blood-based epigenome-wide association study (EWAS). Comparing ALS patients to controls, we detected 1,045 differentially methylated regions (DMRs) in gene bodies, promoters, and intergenic regions. Notably, these DMRs were linked to key ALS-associated pathways, including endocytosis and cell adhesion. Integration with spinal cord transcriptomics revealed that 31% of DMR-associated genes exhibited differential expression in ALS patients compared to controls, with over 20 genes significantly correlating with disease duration. Furthermore, comparison with published single-nucleus RNA sequencing (snRNA-Seq) data of ALS demonstrated that cfDNA methylation changes reflects cell-type-specific gene dysregulation in the brain of ALS patients, particularly in excitatory neurons and astrocytes. Deconvolution of cfDNA methylation profiles suggested altered proportions of immune and liver-derived cfDNA in ALS patients. Conclusions cfDNA methylation is a powerful tool for assessing age-related changes and ALS-specific molecular dysregulation by revealing perturbed locus, genes, and the proportional contributions of different tissues/cells to the plasma. This technique holds promise for clinical application in biomarker discovery across a broad spectrum of neurodegenerative disorders. Supplementary Information The online version contains supplementary material available at 10.1186/s13578-025-01366-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f50dea3e9f7300d7800974584eb194f23dc33f9" target='_blank'>
              Whole-genome bisulfite sequencing of cell-free DNA unveils age-dependent and ALS-associated methylation alterations
              </a>
            </td>
          <td>
            Yulin Jin, Karen N Conneely, Wenjing Ma, Robert K Naviaux, Teepu Siddique, Emily G. Allen, Sandra Guingrich, Robert M Pascuzzi, Peng Jin
          </td>
          <td>2025-02-20</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is characterized by several recurrent mutations that affect disease biology and phenotype, response to therapy and risk of subsequent relapse. Though tyrosine kinase inhibitors have gained regulatory approval for the treatment of AML, it is unclear whether single drugs targeting a specific genomic alteration will be sufficient to eradicate disease. Fortuitously, kinase/bromodomain inhibitors allow targeting of downstream transcriptional effectors of oncogenic pathways, allowing impediment of drug resistance at the transcriptional level. Successful development of combinatorial therapeutic strategies to inhibit both upstream oncogenic pathways and their downstream effectors could thus impede the onset of resistant disease. By using a combination of high-throughput cell-based screening assays and structure-based design, we have developed a novel anti-proliferative 3i-compound scaffold with a diverse range of single and dual FLT3/TAF1(2) activity against AML. Our novel approach to target both FLT3 kinase and TAF1(2) bromodomain efficiently maintained potency against haematological cancers. However, reference compounds and in vitro cell viability and cytotoxicity assays in cancer cell lines demonstrated superior effects of high affinity tyrosine kinase inhibition compared to inhibition of the TAF1 bromodomain. Our results highlight the feasibility of dual tyrosine kinase-bromodomain targeting to overcome disease mechanisms while also revealing the increased efficacy of FLT3-targeted compounds in AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d69a0e511ff64b3061edc6086da48a66db93673d" target='_blank'>
              Development and comparison of single FLT3-inhibitors to dual FLT3/TAF1-inhibitors as an anti-leukemic approach
              </a>
            </td>
          <td>
            R. Leigh, Bogac L. Kaynak, H. Ruskoaho, M. Välimäki
          </td>
          <td>2025-03-28</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Cellular senescence, a stable state of cell cycle arrest induced by various stressors or genomic damage, is recognized as a hallmark of cancer. It exerts a context-dependent dual role in cancer initiation and progression, functioning as a tumor suppressor and promoter. The complexity of senescence in cancer arises from its mechanistic diversity, potential reversibility, and heterogeneity. A key mediator of these effects is the senescence-associated secretory phenotype (SASP), a repertoire of bioactive molecules that influence tumor microenvironment (TME) remodeling, modulate cancer cell behavior, and contribute to therapeutic resistance. Given its intricate role in cancer biology, senescence presents both challenges and opportunities for therapeutic intervention. Strategies targeting senescence pathways, including senescence-inducing therapies and senolytic approaches, offer promising avenues for cancer treatment. This review provides a comprehensive analysis of the regulatory mechanisms governing cellular senescence in tumors. We also discuss emerging strategies to modulate senescence, highlighting novel therapeutic opportunities. A deeper understanding of these processes is essential for developing precision therapies and improving clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/286d205296f9a0a8d06235700589eef3dccbbc34" target='_blank'>
              Regulation of cellular senescence in tumor progression and therapeutic targeting: mechanisms and pathways
              </a>
            </td>
          <td>
            Bowei Liu, Zhigang Peng, Hao Zhang, Nan Zhang, Zao-bin Liu, Zhiwei Xia, Shaorong Huang, Peng Luo, Quan Cheng
          </td>
          <td>2025-04-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The landscapes of somatic mutation in normal cells inform on the processes of mutation and selection operative throughout life, permitting insight into normal ageing and the earliest stages of cancer development. Here, by whole-genome sequencing of 238 microdissections from 30 individuals, including 18 with gastric cancer, we elucidate the developmental trajectories of normal and malignant gastric epithelium. We find that gastric glands are units of monoclonal cell populations which accrue ∼28 somatic single nucleotide variants per year, predominantly attributable to endogenous mutational processes. In individuals with gastric cancer, glands often show elevated mutation burdens due to acceleration of mutational processes linked to proliferation and oxidative damage. These hypermutant glands were primarily detected in the gastric antrum and were mostly associated with chronic inflammation and intestinal metaplasia, known cancer risk factors. Unusually for normal cells, gastric epithelial cells often carry recurrent trisomies of specific chromosomes, which are highly enriched in a subset of individuals. Surveying approximately 8,000 gastric glands by targeted sequencing, we found somatic driver mutations in a distinctive repertoire of known cancer genes, including ARID1A, CTNNB1, KDM6A and ARID1B. Their prevalence increases with age to occupy approximately 5% of the gastric epithelial lining by age 60 years. Our findings provide insights into the intrinsic and extrinsic influences on somatic evolution in the gastric epithelium, in healthy, precancerous and malignant states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a80de4630550f795baa857c6ee76f35822aa4bc" target='_blank'>
              The somatic mutation landscape of normal gastric epithelium
              </a>
            </td>
          <td>
            Tim H. H. Coorens, G. Collord, Hyungchul Jung, Yichen Wang, L. Moore, Y. Hooks, Krishnaa Mahbubani, K. Saeb-Parsy, P. J. Campbell, I. Martincorena, Suet Yi Leung, Michael R Stratton
          </td>
          <td>2024-03-19</td>
          <td>Nature</td>
          <td>1</td>
          <td>88</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1ae2ca598b5cdc041c4c163b222a0b0f6d55027" target='_blank'>
              Tumor suppressor collateral damage screens reveal mRNA homeostasis protein HBS1L as a novel vulnerability in ch9p21 driven FOCAD deleted cancer
              </a>
            </td>
          <td>
            Hongxiang Zhang, M. Tonini, Lauren Martires, Charlotte B Pratt, Helena N. Jenkins, Eden N. Gordon, Shanchuan Zhao, Ashley H Choi, Samuel R. Meier, Tenzing Khendu, Shangtao Liu, B. Shen, Hannah Stowe, Katerina Pashiardis, Xuewen Pan, Madhavi Bandi, Minjie Zhang, Yi Yu, Chengyin Min, A. Huang, Jannik N Andersen, Hilary E. Nicholson, T. Teng
          </td>
          <td>2025-02-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background Gastric cancer (GC) remains a leading cause of cancer-related mortality, with over one million new cases and 769,000 deaths reported in 2020. Despite advancements in chemotherapy, surgery, and targeted therapies, delayed diagnosis due to overlooked early symptoms leads to poor prognosis. Methods We integrated bulk RNA sequencing and single-cell RNA sequencing datasets from TCGA, GEO, and OMIX001073, employing normalization, batch effect correction, and dimensionality reduction methods to identify key cell populations associated with GC invasion and epithelial-mesenchymal transition (EMT), as well as analyze the tumor immune microenvironment. Results Our analysis identified the MUC5AC+ malignant epithelial cell cluster as a significant player in GC invasion and EMT. Cluster 1, representing this cell population, exhibited higher invasion and EMT scores compared to other clusters. Survival analysis showed that high abundance in cluster 0 correlated with improved survival rates (P=0.012), whereas cluster 1 was associated with poorer outcomes (P=0.045). A prognostic model highlighted ANXA5 and GABARAPL2 as two critical genes upregulated in GC tumors. High-risk patients demonstrated increased immune cell infiltration and worse prognosic. Analysis of tumor mutation burden (TMB) indicated that patients with low TMB in the high-risk group had the worst prognosis. Wet-lab validation experiments confirmed the oncogenic role of ANXA5, showing its facilitation of cell proliferation, invasion, and migration while suppressing apoptosis. Conclusion This study offers novel insights into the subpopulations of malignant epithelial cells in GC and their roles in tumor progression. It provides a prognostic model and potential therapeutic targets to combat GC, contributing crucial understanding to the fundamental mechanisms of drug resistance in gastrointestinal cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f9fbc569ffcfa0009c2b9e523878c6293cec12b" target='_blank'>
              Integrative analysis of single-cell and bulk RNA sequencing reveals the oncogenic role of ANXA5 in gastric cancer and its association with drug resistance
              </a>
            </td>
          <td>
            Deng-Gang Chen, Peng Zhang, Li Gong, Hailang Wei, Guanghui Yu, Tingting Zhang, Chen Bai
          </td>
          <td>2025-03-07</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Accumulating evidence links pediatric cancers to prenatal transformation events, yet the influence of the developmental stage on oncogenesis remains elusive. We investigated how hematopoietic stem cell developmental stages affect leukemic transformation, disease progression, and therapy response using a novel, humanized model of NUP98::NSD1-driven pediatric acute myeloid leukemia, that is particularly aggressive with WT1 co-mutations. Fetal-derived hematopoietic stem cells readily transform into leukemia, and WT1 mutations further enhance stemness and alter lineage hierarchy. In contrast, stem cells from later developmental stages become progressively resistant to transformation. Single-cell analyses revealed that fetal-origin leukemia stem cells exhibit greater quiescence and reliance on oxidative phosphorylation than their postnatal counterparts. These differences drive distinct therapeutic responses, despite identical oncogenic mutations. In patients, onco-fetal transcriptional programs correlate with worse outcomes. By targeting key vulnerabilities of fetal-origin leukemia cells, we identified combination therapies that significantly reduce aggressiveness, highlighting the critical role of ontogeny in pediatric cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca97e870a1822de5485cb1751fdbfdd64fadf250" target='_blank'>
              Ontogeny Dictates Oncogenic Potential, Lineage Hierarchy, and Therapy Response in Pediatric Leukemia
              </a>
            </td>
          <td>
            Ke Wang, Shayan Saniei, Nikita Poddar, Subrina Autar, Saul Carcamo, Meghana Sreenath, Jack H Peplinski, R. Ries, Isabella G Martinez, Clifford Chao, Anna Huo-Chang Mei, Noshin Rahman, Levan Mekerishvili, Miguel Quijada-álamo, Grace Freed, Mimi Zhang, Katherine Lachman, Zayna Diaz, Manuel M Gonzalez, Jing Zhang, Giang Pham, Dan Filipescu, Mirela Berisa, Tommaso Balestra, J. Reisz, A. D’Alessandro, Daniel J Puleston, Emily Bernstein, Jerry E. Chipuk, M. Wunderlich, S. Tasian, Bridget K. Marcellino, Ian A. Glass, Christopher M Sturgeon, D. Landau, Zhihong Chen, E. Papapetrou, Franco Izzo, S. Meshinchi, Dan Hasson, Elvin Wagenblast
          </td>
          <td>2025-03-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6192f93a04789d7b2aabf4cf16ca2d7d42091ce4" target='_blank'>
              Innovative CRISPR-Cas9 enhances cell rejuvenation, addressing disease
              </a>
            </td>
          <td>
            Linfeng Zhang
          </td>
          <td>2025-03-20</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer development is characterized by chromosomal instability, manifesting in frequent occurrences of different genomic alteration mechanisms ranging in extent and impact. Mathematical modeling can help evaluate the role of each mutational process during tumor progression, however existing frameworks can only capture certain aspects of chromosomal instability (CIN). We present CINner, a mathematical framework for modeling genomic diversity and selection during tumor evolution. The main advantage of CINner is its flexibility to incorporate many genomic events that directly impact cellular fitness, from driver gene mutations to copy number alterations (CNAs), including focal amplifications and deletions, missegregations and whole-genome duplication (WGD). We apply CINner to find chromosome-arm selection parameters that drive tumorigenesis in the absence of WGD in chromosomally stable cancer types from the Pan-Cancer Analysis of Whole Genomes (PCAWG, n=718). We found that the selection parameters predict WGD prevalence among different chromosomally unstable tumors, hinting that the selective advantage of WGD cells hinges on their tolerance for aneuploidy and escape from nullisomy. Analysis of inference results using CINner across cancer types in The Cancer Genome Atlas (n=8207) further reveals that the inferred selection parameters reflect the bias between tumor suppressor genes and oncogenes on specific genomic regions. Direct application of CINner to model the WGD proportion and fraction of genome altered (FGA) in PCAWG uncovers the increase in CNA probabilities associated with WGD in each cancer type. CINner can also be utilized to study chromosomally stable cancer types, by applying a selection model based on driver gene mutations and focal amplifications or deletions (chronic lymphocytic leukemia in PCAWG, n=95). Finally, we used CINner to analyze the impact of CNA probabilities, chromosome selection parameters, tumor growth dynamics and population size on cancer fitness and heterogeneity. We expect that CINner will provide a powerful modeling tool for the oncology community to quantify the impact of newly uncovered genomic alteration mechanisms on shaping tumor progression and adaptation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a271cd04acf8979b5f4cf725c9c1e047db0d2529" target='_blank'>
              CINner: Modeling and simulation of chromosomal instability in cancer at single-cell resolution
              </a>
            </td>
          <td>
            K. Dinh, I. Vázquez-García, Andrew Chan, Rhea Malhotra, A. Weiner, Andrew McPherson, Simon Tavaré
          </td>
          <td>2025-04-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="The transcription factor Prdm14 is a potent oncogene implicated in the initiation of many cancers. PRDM14 resets and maintains the pluripotent state in normal cells, but the molecular mechanisms through which PRDM14 drives oncogenesis are poorly understood. Here, we interrogated the heterogeneity of Prdm14-expressing cells in a T cell lymphoblastic leukemia/lymphoma mouse model. Using mass cytometry (CyTOF) of bone marrow at a pre-leukemic timepoint, an unexpected abnormal progenitor B cell population was identified. Prdm14-expressing progenitor B cells demonstrated short-term self-renewal and a block in differentiation when transferred to syngeneic hosts. Consistently, aged host mice succumb to a highly penetrant B-LL. Single-cell RNA-seq analyses suggests that the expression signature of these pre-leukemia cells is more consistent with that of B-1 cells than B-2 cells. B-1 cells are a self-renewing population of unconventional B cells established during embryonic development. Overlaying the chromatin binding of transcriptional marks H3K4me1 and H3K4me3 with PRDM14 suggests that PRDM14 initiates cancers through promiscuous DNA binding, activating oncogenic pathways and skewing development towards a self-renewing B-1-like phenotype. Together, our data show that Prdm14 can initiate premature T and B cell cancer programs when expressed in hematopoietic progenitor cells. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-93043-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d78c4f8c37e578dfe03b8165770afdb4f53da3e6" target='_blank'>
              Functional and molecular single-cell analyses implicate PRDM14 in the initiation of B cell leukemia in mice
              </a>
            </td>
          <td>
            Lauren J. Tracey, Mahmoud El-Maklizi, Dustin J. Sokolowski, Miki S Gams, Travis Brooke-Bisschop, Julie Ruston, Christine Taylor, Alexandra Khozin, S. Rajakumar, Michael D. Wilson, C J Guidos, Monica J. Justice
          </td>
          <td>2025-03-14</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f250f1f9ca61e747d3f45521c173e479dc5a6f63" target='_blank'>
              Chemical enhancement of DNA repair in aging
              </a>
            </td>
          <td>
            J. C. Macedo, Maria M da Silva, Joana M. Magalhães, Carlos Sousa-Soares, M. Inês Ala, Mafalda Galhardo, Rui Ribeiro, Monika Barroso-Vilares, Mafalda Sousa, Paula Sampaio, E. Logarinho
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Advancements in biological and medical science are intricately linked to the biological central dogma. In recent years, gene editing techniques, especially CRISPR/Cas systems, have emerged as powerful tools for modifying genetic information, supplementing the central dogma and holding significant promise for disease diagnosis and treatment. Extensive research has been conducted on the continuously evolving CRISPR/Cas systems, particularly in relation to challenging diseases, such as cancer and HIV infection. Consequently, the integration of CRISPR/Cas-based techniques with contemporary medical approaches and therapies is anticipated to greatly enhance healthcare outcomes for humans. This review begins with a brief overview of the discovery of the CRISPR/Cas system. Subsequently, using CRISPR/Cas9 as an example, a clear description of the classical molecular mechanism underlying the CRISPR/Cas system was given. Additionally, the development of the CRISPR/Cas system and its applications in gene therapy and high-sensitivity disease diagnosis were discussed. Furthermore, we address the prospects for clinical applications of CRISPR/Cas-based gene therapy, highlighting the ethical considerations associated with altering genetic information. This brief review aims to enhance understanding of the CRISPR/Cas macromolecular system and provide insight into the potential of genetic macromolecular drugs for therapeutic purposes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec8f7ea9677d768cf517a5b73f3afb4dfe1f5d1d" target='_blank'>
              From Origin to the Present: Establishment, Mechanism, Evolutions and Biomedical Applications of the CRISPR/Cas-Based Macromolecular System in Brief
              </a>
            </td>
          <td>
            Zheng Yuan
          </td>
          <td>2025-02-01</td>
          <td>Molecules</td>
          <td>2</td>
          <td>3</td>
        </tr>

        <tr id="Background: Ovarian cancer (OC) is the deadliest malignant tumor in the female reproductive system. Sphingolipid metabolism (SM) is crucial for cellular function and has been linked to OC progression. Dysregulation of sphingolipid pathways contributes to tumor growth, chemoresistance, and metastasis in OC. Currently, investigations into the relationship between sphingolipid-related genes (SRGs) and OC prognosis in their initial stages. Our study aimed to develop a novel molecular subtyping based on SRGs and construct a signature to predict the prognosis of patients with OC, immune cell infiltration characteristics, and chemotherapy sensitivity. Methods: Bulk and single-cell RNA-sequencing data of OC was analyzed primarily from the TCGA and GEO databases. The gene set related to the sphingolipid pathway (hsa00600) was selected from the SM pathway, and the enrichment of SRGs was analyzed in the annotated single-cell sequencing data. The Scanpy function was used to score the gene features of each cell and further identify differentially expressed genes. By intersecting with the genes most closely related to SM activity identified through Weighted Gene Co-expression Network Analysis (WGCNA) based on bulk RNA sequencing data, and after performing univariate COX, multivariate COX and LASSO regression, three SRGs were identified. Subsequently, the SRGs-related prognostic signature was constructed. The analysis was further extended to clinical feature correlation, GSEA, tumor microenvironment (TME) analysis and chemotherapy sensitivity analysis. Finally, the expression and function of the key gene GBP5 in the model were validated through in vitro experiments. Results: Compared to other sites, SRG scores were highest in ascites, and among different cell types, SRG scores were highest in T cells. By integrating scRNA-seq and bulk RNA-seq analysis, three SRGs (C5AR1, GBP5, and MARCHF3) were ultimately selected to develop a prognostic model for SRGs. In this model, patients with higher risk scores had shorter overall survival, which was validated in the testing cohort. Immune infiltration analysis revealed that the risk score was negatively correlated with the abundance of CD8+ T cell infiltration and positively correlated with the abundance of M2 macrophage infiltration. Chemotherapy sensitivity analysis showed that the high-risk group exhibited increased resistance to Oxaliplatin, Gemcitabine, and Sorafenib. In vitro, we demonstrated that knockdown of the protective gene GBP5 in HEYA8 and SKOV3 cells enhanced cell viability, proliferation, and invasiveness, reduced apoptosis, and increased IC50 values for chemotherapy drugs. Conclusion: Our model effectively identifies high-risk patients and provides a reference for prognosis prediction using SRG signature. Moreover, hub gene GBP5 acts as a tumor inhibitory factor and regulates the chemosensitivity of oxaliplatin, gemcitabine, and sorafenib in OC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9a308520b9264d0d3f7ebb80152563d51018699" target='_blank'>
              Integrating Single-Cell and Bulk RNA Sequencing Data to Explore Sphingolipid Metabolism Molecular Signatures in Ovarian Cancer Prognosis: an Original Study
              </a>
            </td>
          <td>
            Xu Huang, Xiaoyu Li, Wulin Shan, Yingyu Dou, Qiongli Yu, Yao Chen, Zengying Wang, Haomin Zhang, Yumeng Wang, Xiaofei Lu, Wenju Peng, B. Xia
          </td>
          <td>2025-03-24</td>
          <td>International Journal of Medical Sciences</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/523e30ee8e9194593b3c69011f8c12e3823f9a16" target='_blank'>
              Pan-cancer transcriptional regulatory network analysis reveals key drivers and epigenetic modulators in tumorigenesis
              </a>
            </td>
          <td>
            Yaluan Yin, Junhua Zhang
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The expression of cytosolic aldehyde dehydrogenases (ALDHs), which mediate the last step in the pathway of the synthesis of all-trans retinoic acid, is dysregulated in various types of human cancer, and has been associated with the development of cancer stem cells (CSCs) in solid tumors and hematological malignancies. CSCs are considered a minor fraction of cancer cells with the capacity to initiate neoplastic tumors. ALDH1A1 serves a crucial role in the emergence of the CSC phenotype, induces the malignant behavior of cancer cells and promotes treatment resistance. Notably, ALDH1A1-induced therapy resistance is not exclusive to just one group of drugs, but affects diverse types of drugs that use different mechanisms to kill cells. This diversity of drug resistance-inducing effects is associated with the stemness-supporting functions of ALDH1A1. The inhibition of ALDH1A1 activity using chemicals or the depletion of ALDH1A1 via genetic approaches, such as the use of small interfering RNA, can overcome diverse pathways of therapy resistance. In the context of breast cancer, it is critical that only a fraction of malignant cells are expected to manifest stem-like features, which include increased expression of ALDH1A1. From the angle of disease prognosis, the extent of the association of ALDH1A1 with increased malignant behavior and drug resistance remains to be determined through the application of cutting-edge methods that detect the expression of tracked biomarkers within tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/378f86e62d74f35111b7a740e827c0ac03143fb8" target='_blank'>
              ALDH1A1 in breast cancer: A prospective target to overcome therapy resistance (Review)
              </a>
            </td>
          <td>
            L. Varışlı, P. Zoumpourlis, Demetrios A. Spandidos, V. Zoumpourlis, S. Vlahopoulos
          </td>
          <td>2025-03-04</td>
          <td>Oncology Letters</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Individual aging is a complex biological process involving multiple levels, with molecular changes existing in heterogeneity across different cell types and tissues, being regulated by both internal and external factors. Traditional senescence markers, including p16, cell morphological changes, and cell cycle arrest, can only partially reflect the complexity of senescence. Single-cell omics technology facilitates the integration of multi-faceted data, including gene expression profiles, spatial dynamics, chromatin accessibility and metabolic pathways. This comprehensive approach enhances the development of biomarkers, granting us a more profound insight into the heterogeneity inherent within senescent cell populations. In this review, we summarize the application of single cell multi-omics approaches in analyzing senescence mechanisms and potential intervention targets from the perspectives of transcriptomics, epigenetics, metabolomics, and proteomics, explore the potential of developing new senescence markers at the cellular level using machine learning algorithms and artificial intelligence in bioinformatics analysis. Finally, we further discuss the challenges and prospective trajectories within this research domain to provide a more comprehensive perspective on dissecting the regulatory networks of senescence cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb05feca31b360e2a4184cb1c6c1adbe6ecbd8d5" target='_blank'>
              Towards Precision Aging Biology: Single-Cell Multi-Omics and Advanced AI-Driven Strategies.
              </a>
            </td>
          <td>
            Sijia Xie, Xinwei Luo, Feitong Hong, Yijie Wei, Yuduo Hao, Xueqin Xie, Xiaolong Li, Guangbo Xie, Fuying Dao, Hao Lyu
          </td>
          <td>2025-03-17</td>
          <td>Aging and disease</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Gastric cancer (GC) remains a global health challenge due to its high mortality rate and the lack of specific diagnostic methods. Tumor heterogeneity significantly hinders effective treatment, necessitating advanced techniques to dissect its complexity. Artificial intelligence (AI) offers transformative potential in uncovering intricate cellular dynamics and gene regulatory networks. This study leverages single-cell RNA sequencing (scRNA-seq) combined with transcriptome analysis, enhanced by AI-driven analytics, to explore the tumor microenvironment and identify novel prognostic markers and therapeutic targets in GC. Methods scRNA-seq and transcriptome datasets of GC patients were analyzed using AI-enhanced methodologies to unravel tumor heterogeneity and microenvironmental dynamics. Macrophage subsets were identified as critical components within the GC microenvironment. High-variance gene screening in these subsets pinpointed apolipoprotein E (APOE) as a hub gene. Experimental validation of APOE expression in GC samples and functional studies in GC cell lines were conducted. Results Bioinformatics and AI-enabled analyses confirmed the immunosuppressive role of APOE in GC. An immune-related survival model was developed to predict immunotherapy responses and patient prognoses. Mechanistically, APOE was found to induce immunosuppression through M2 macrophage polarization, promoting tumor progression and leading to poorer outcomes in GC patients. Conclusion This study highlights the potential of AI-driven approaches in elucidating the role of APOE in GC progression. APOE’s regulatory effects on M2 macrophages underscore its value as a prognostic marker and therapeutic target, paving the way for precision medicine in GC management. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-02100-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/987e5761ad8244faf71073710e21f1e10a5477f1" target='_blank'>
              Artificial intelligence-driven integration of single-cell RNA sequencing and transcriptome analysis to decipher APOE’s role in gastric cancer prognosis and therapy
              </a>
            </td>
          <td>
            Ni Li, Jinquan Lin, Chunyu Wang, Xiao Ran, Zhi Zhao
          </td>
          <td>2025-03-13</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af5f5ff700aa0f25c87024a04fa36fa40cb85cb4" target='_blank'>
              Abstract A055: Dissecting mechanisms of tumor anti-PD1 resistance due to host genetic variation within ADAM17
              </a>
            </td>
          <td>
            James P Heiserman, R. Akhurst, Tanuja Desai, Sharon Chen
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdb015a26b60d1395883653385851a0c2cbe11b8" target='_blank'>
              Abstract IA02: Developing and manufacturing next generation cell therapies in the academic setting
              </a>
            </td>
          <td>
            B. Shy
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c906c5a3862aab5cd84c1716303e1ffc00c0344c" target='_blank'>
              NSD3: A Promising Target for Cancer Therapy.
              </a>
            </td>
          <td>
            Ting Huang, Bowen Zhang, Yifan Yang, Qiong Lin, Genbao Shao
          </td>
          <td>2025-03-26</td>
          <td>Cell biochemistry and function</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="One-carbon metabolism (1CM) has been reported to promote cancer progression across various malignancies. While 1CM is critical for cell proliferation by enhancing nucleotide synthesis, its physiological roles within different cell types in the tumor immune microenvironment (TIME) still remain unclear. In this study, we analyzed bulk-RNA sequencing and single-cell RNA sequencing (scRNA-seq) data from lung adenocarcinoma (LUAD) patients to elucidate the functional roles of 1CM within the TIME. Moreover, we examined scRNA-seq data from patients treated with immunotherapy across various cancers, including LUAD, glioblastoma, renal cell carcinoma, colorectal cancer, and triple-negative breast cancer. Compared to other cell types, 1CM gene profiles are significantly enriched in a specific subset of T cells. Intriguingly, these high-1CM T cells are identified as proliferative intermediate exhausted T cells (Texint). Furthermore, these proliferative Texint received the most robust CD137 signaling. Consistently, analysis of scRNA-seq data from LUAD patients undergoing anti-PD1 immunotherapy demonstrated that proliferative Texint exhibited higher 1CM scores and increased CD137 signaling. This observation was particularly pronounced in non-responders to immunotherapy, where the Texint population was significantly expanded. We further established that 1CM is a prominent signaling pathway in proliferative Texint in patients resistant to immunotherapy across multiple cancer types. Collectively, we identify CD137 signaling as a distinctive pathway in proliferative Texint of LUAD patients who do not respond to immunotherapy. These findings propose that targeting 1CM may represent a novel therapeutic strategy to enhance the efficacy of immunotherapy by mitigating Texint proliferation in diverse cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a799bb85faaaac3fa04136a525f7e0a65fbc4560" target='_blank'>
              One-carbon metabolism is distinct metabolic signature for proliferative intermediate exhausted T cells of ICB-resistant cancer patients
              </a>
            </td>
          <td>
            Ye-Chan Park, Yesueng Hwang, Jae Woong Jeong, Chae Min Lee, Minki Kim, Sugyeong Jo, Seyeon Joo, Nahee Hwang, Sungsoon Fang
          </td>
          <td>2025-02-14</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09704648420ca313fc3a94dc7cc40706df2fbbe8" target='_blank'>
              Blood-Based Epigenetic Instability Linked to Human Aging and Disease
              </a>
            </td>
          <td>
            Salman Basrai, Ido Nofech-Mozes, Rajesh Detroja, Fernando L. Scolari, Mehran Bakhtiari, Andrea Arruda, Tracy Murphy, Scott V. Bratman, Steven M. Chan, Mark D. Minden, , D. D. H. Kim, R. Kridel, Filio Billia, Sagi Abelson
          </td>
          <td>2025-03-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8677e695bbefb8bee72d8f6205f8c94375119315" target='_blank'>
              Developmental Origins and Oncogenesis in Medulloblastoma.
              </a>
            </td>
          <td>
            K. Kiang, Yogesh K H Wong, Soma Sengupta, Martine F. Roussel, Q. R. Lu
          </td>
          <td>2025-02-14</td>
          <td>Annual review of neuroscience</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Abstract For genome editing, the use of CRISPR ribonucleoprotein (RNP) complexes is well established and often the superior choice over plasmid-based or viral strategies. RNPs containing dCas9 fusion proteins, which enable the targeted manipulation of transcriptomes and epigenomes, remain significantly less accessible. Here, we describe the production, delivery, and optimization of second generation CRISPRa RNPs (dRNPs). We characterize the transcriptional and cellular consequences of dRNP treatments in a variety of human target cells and show that the uptake is very efficient. The targeted activation of genes demonstrates remarkable potency, even for genes that are strongly silenced, such as developmental master transcription factors. In contrast to DNA-based CRISPRa strategies, gene activation is immediate and characterized by a sharp temporal precision. We also show that dRNPs allow very high-target multiplexing, enabling undiminished gene activation of multiple genes simultaneously. Applying these insights, we find that intensive target multiplexing at single promoters synergistically elevates gene transcription. Finally, we demonstrate in human stem and differentiated cells that the preferable features of dRNPs allow to instruct and convert cell fates efficiently without the need for DNA delivery or viral vectors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ae64466d50f495675fc82663a1b7dac6851bcc6" target='_blank'>
              Efficient DNA- and virus-free engineering of cellular transcriptomic states using dCas9 ribonucleoprotein (dRNP) complexes
              </a>
            </td>
          <td>
            Tobias Schmidt, Maximilian F. Wiesbeck, Luisa Eg er t, Thi-T ram T ruong, Anna Danese, Lukas Voshagen, Simon Imhof, Matilde Iraci Borgia, Deeksha, Andrea M Neuner, Anna Köf er le, Ar ie Geer lof, André Santos, Dias Mourão, Stefan H Stricker
          </td>
          <td>2025-03-20</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="To gain insight into biological mechanisms that cause resistance to DNA damage, we performed parallel pooled genetic CRISPR-Cas9 screening for survival in high risk HNSCC subtypes. Surprisingly, and in addition to ATM, DNAPK, and NFKB signaling, JAK1 was identified as a driver of tumor cell radiosensitivity. Knockout of JAK1 in HNSCC increases cell survival by enhancing the DNA damage-induced G2 arrest, and both knockout and JAK1 inhibition with abrocitinib prevent subsequent formation of radiation-induced micronuclei. Loss of JAK1 function does not affect canonical CDK1 signaling but does reduce activation of PLK1 and AURKA, kinases that regulate both G2 and M phase progression. Correspondingly, JAK1 KO was found to cause mitotic defects using both EdU labeling and live cell imaging techniques. Given this insight, we evaluated Kif18a inhibition as an approach to exacerbate mitotic stress and enhance the efficacy of radiation. These studies establish Kif18a inhibition as a novel strategy to counteract therapeutic resistance to DNA damage mediated by G2 cell cycle arrest.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eddfa8407309bcd878a23af220d2c4449530b55c" target='_blank'>
              Loss of JAK1 Function Causes G2/M Cell Cycle Defects Vulnerable to Kif18a Inhibition
              </a>
            </td>
          <td>
            Vanessa Kelley, Marta Baro, William Gasperi, Nicholas Ader, Hannah Lea, Hojin Lee, C. Phoomak, Lilian Kabeche, Megan C. King, Joseph N. Contessa
          </td>
          <td>2025-02-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Alternative Lengthening of Telomeres (ALT) is a telomerase-independent mechanism deployed by several aggressive cancers to maintain telomere length. This contributes to their malignancy and resistance to conventional therapies. In prior studies, we have identified key proteins linked to the ALT process using multi-omic data integration strategies. In this work, we combined metabolomic datasets with our earlier results to identify targetable metabolic pathways for ALT-positive tumors. 39 ALT-related proteins were found to interact with 42 different metabolites in our analysis. Additional networking analysis revealed a complex interaction between metabolites and ALT-related proteins, suggesting that pan-cancer oncogenes may have an impact on these pathways. Three metabolic pathways have been primarily related with the ALT mechanism: purine metabolism, cysteine and methionine metabolism, and nicotinate and nicotinamide metabolism. Lastly, we prioritized FDA-approved drugs (azathioprine, thioguanine, and mercaptopurine) that could target ALT-positive tumors through purine metabolism. This work provides a wide perspective of the metabolomic pathways associated with ALT and reveals potential therapeutic targets that require further experimental validation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef209c063fddf47ab0ca2823c1da99d2d523acd4" target='_blank'>
              Metabolic pathways of Alternative Lengthening of Telomeres in pan-carcinoma
              </a>
            </td>
          <td>
            I. Armendáriz-Castillo, J. García-Cárdenas, Pamela Espinosa, Katherine Hidalgo-Fernández, Lizbeth Peña-Zúñiga, Ronie Martínez, Juan Moromenacho, Andrés Herrera-Yela, Jonathan Cruz-Varela, Anilú Saucedo-Sariñana, María-Esperanza Cerdán, A. López-Cortés, Santiago Guerrero
          </td>
          <td>2025-02-21</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ea4c6f0326f442ed8f79641ba270dbbfd88e4e5" target='_blank'>
              Locking the gates of immortality: targeting alternative lengthening of telomeres (ALT) pathways.
              </a>
            </td>
          <td>
            Apurwa Mishra, Trupti N Patel
          </td>
          <td>2025-02-18</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="MYC-driven (MYC+) cancers are aggressive and often fatal. MYC dysregulation is a key event in these cancers, but overexpression of MYC alone is not always enough to cause cancer. Plasmocytoma Variant Translocation 1 (PVT1), a long non-coding RNA (lncRNA) adjacent to MYC on chromosome 8 is a rearrangement hotspot in many MYC+ cancers. In addition to being co-amplified with MYC, the genomic rearrangement at PVT1 involves translocation, which has had obscure functional consequences. We report that translocation at the PVT1 locus cause asymmetric enrichment of 5’-PVT1 and loss of 3’-PVT1. Despite being classified as a non-coding RNA, the retained 5’ region of PVT1 generates a circular RNA (CircPVT1) that codes for the novel peptide we call Firefox (FFX). FFX augments AKT signaling and synergistically activates MYC and mTORC1 in these cells. Further, the 3’ end of PVT1, which is lost during the translocation, codes for a tumor-suppressing micropeptide we named as Honeybadger (HNB). We demonstrate that HNB interacts with KRAS and disrupts the activation of KRAS effectors. Loss of HNB leads to activation of RAS/MAPK signaling pathway, and enhances MYC stability by promoting phosphorylation of MYC at Ser62. These findings identify PVT1 as a critical node that synchronizes MYC, AKT, and RAS-MAPK activities in cancer. Our study thus identifies a key mechanism by which rearrangements at the PVT1 locus activate additional oncogenic pathways that synergize with MYC to exacerbate the aggressiveness of MYC+ cancers. This newfound understanding explains the poor prognosis associated with MYC+ cancers and offers potential therapeutic targets that could be leveraged in treatment strategies for these cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e076955c4c71d3d95422db7f1fcfc733c5d61e3" target='_blank'>
              Synergistic RAS-MAPK and AKT Activation in MYC-Driven Tumors via Adjacent PVT1 Rearrangements
              </a>
            </td>
          <td>
            Ashutosh Tiwari, Utkarsha Paithane, Jordan Friedlein, Kojiro Tashiro, Olivier Saulnier, Karina Barbosa, Quang Trinh, Bryan Hall, Shrawantee Saha, Aditi Soni, Takuma Nakashima, Andrey Bobkov, Lynn Fujimoto, Rabi Murad, Svetlana Maurya, Mayank Saraswat, Shahab Sarmashghi, Joshua T. Lange, Sihan Wu, M. Masihi, Srija Ghosh, Gazal Hemmati, Owen S. Chapman, Liam D Hendrikse, Brian James, J. Luebeck, Tanja Eisemann, Theophilos Tzaridis, Deepak Rohila, Robyn Leary, Jyotika Varshney, Badrinath R. Konety, S. Dehm, Yasuhiko Kawakami, R. Beroukhim, D. Largaespada, Lincoln Stein, Lukas Chavez, Hiromichi Suzuki, William A. Weiss, Jianhua Zhao, Aniruddha J. Deshpande, R. Wechsler-Reya, Michael D Taylor, Anindya Bagchi
          </td>
          <td>2025-02-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>148</td>
        </tr>

        <tr id="Cell division cycle 25 (CDC25) phosphatases serve as crucial regulators of cell cycle phase transitions and essential components of the checkpoint machinery involved in DNA damage response. Emerging evidence indicates the oncogenic potential of CDC25 family members across various cancers. However, comprehensive insights into the expression pattern and function of the CDC25 family in diverse cancers remain unexplored. In our study, we investigated CDC25 family using multiple databases, including gene expression levels, molecular signatures, diagnosis value, and prognostic value in pan-cancer. Furthermore, we focused on melanoma and systematically explored CDC25A expression and its clinical correlations. As a result, the expression of CDC25 family members is significantly abnormal in most cancers, correlating with poorer prognosis. CDC25 family members are differently regulated by DNA methylation and genetic alterations across various cancers. In addition, CDC25 family plays a critical role in the malignant progression of melanoma. Functional investigation reveals that CDC25A inhibition suppresses the proliferation of melanoma cells and sensitizes melanoma cells to chemotherapy and NK cell therapy. In conclusion, our study suggests that CDC25 family may serve as a significant biomarker for diagnosis and prognosis across multiple cancers, with CDC25A as a promising therapeutic target for melanoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac2d9665c88d909bf6dd5f9c6702e4a774f6a7f4" target='_blank'>
              CDC25A inhibition sensitizes melanoma cells to doxorubicin and NK cell therapy
              </a>
            </td>
          <td>
            Xinyue Gao, Feichang Liu, Bo Zhang, Tianyi Ren, You Zheng, Z. Niu, He Ren, Chenyu Liu, Chengzuo Jiang, Chenxi Wang, Hongyan Huang, Li Ma, Qiang Sun
          </td>
          <td>2025-04-11</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Resistance to DNA damage is one of the primary mechanisms by which tumor cells evade the effects of standard chemotherapeutic agents and radiotherapy. Dynamic and complex interactions between the tumor microenvironment (TME) and tumor cells critically influence the DNA damage response. Interleukin-33 (IL-33) is a multifunctional cytokine secreted at high levels in response to cellular damage and stress. Recently, increasing evidence has suggested that IL-33 plays a key role in promoting the therapeutic resistance of tumors. However, the actual source of IL-33 during cancer therapy and how IL-33 contributes to a resistant TME remain incompletely understood. In this study, we found that both cancer-associated fibroblasts (CAFs) and tumor cells treated with DNA damage-inducing agents expressed and secreted high levels of IL-33, subsequently leading to enhanced DNA damage repair efficacy. Mechanistically, nuclear IL-33 primarily functions as a transcriptional co-activator of homologous recombination repair (HRR) genes, whereas the active form of IL-33 can drive the non-homologous end joining (NHEJ) pathway via the canonical IL-33/ST2 axis. Overall, we demonstrated that IL-33 plays a key role in mediating a DNA damage-resistant TME, which could represent a potential therapeutic vulnerability in chemoresistant cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e0fd58212c64247926e93799486ccc1e7786024" target='_blank'>
              Interleukin-33 (IL-33) promotes DNA damage-resistance in lung cancer
              </a>
            </td>
          <td>
            Haoge Luo, Liping Liu, Xiaoping Liu, Yingdong Xie, Xin Huang, Ming Yang, Chen Shao, Dong Li
          </td>
          <td>2025-04-11</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/faa65e1fd6a4ddcb57e0eaf994498d400b2eb5d2" target='_blank'>
              Tissue-specific mutagenesis from endogenous guanine damage is suppressed by Polκ and DNA repair
              </a>
            </td>
          <td>
            Yang Jiang, M. Przybilla, Linda Bakker, Foster C. Jacobs, Dylan Mckeon, Roxanne van der Sluijs, Juliette Wezenbeek, Joeri van Strien, Jeroen Willems, Alex E E Verkennis, Jamie Barnett, Adrian Baez Ortega, Federico Abascal, Peter W. Villalta, P. Knipscheer, I. Martincorena, S. Balbo, Juan Garaycoechea
          </td>
          <td>2025-03-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="Background Inducible gene expression circuits enable precise control over target gene activation and are widely used in direct reprogramming. However, their usability is often compromised by DNA methylation-induced silencing, especially in iPSCs. This deactivates genetic circuits in engineered iPSCs preventing them from being used for long-term scalable expansion of desired cell types. A2-ubiquitous chromatin opening elements (A2UCOE) have been recognized for their anti-silencing properties, but they have not been used in human iPSCs with inducible systems for direct reprogramming. This study investigates the role of A2UCOE in inducible systems and identifies strategies to eliminate associated gene leakage enabling long-term use of engineered human iPSCs. Results We developed a compact all-in-one gene circuit — containing a doxycycline-inducible Tet-On system, 863 bp of A2UCOE, and FOXN1, a transcription factor critical for thymic epithelial cell (TEC) differentiation — easily deployed to new genomic sites. However, we observed significant FOXN1 gene leakage even without doxycycline, which is a novel limitation of A2UCOE. This leakage resulted in premature differentiation of iPSCs into TECs, limiting its continued use. To further investigate the relationship between A2UCOE and gene leakage, we generated A2UCOE fragments of varying lengths (1337 bp, 749 bp, and 547 bp) and found that all fragments, regardless of length, caused significant gene leakage. To solve this issue, we tested different spacer sequences between A2UCOE and the inducible promoter and found that the SV40 poly-A terminator fully eliminated FOXN1 leakage, and we show this effect is not due to AT- or GC-content. Unexpectedly, this architecture further enhanced anti-silencing effects > 60% providing prolonged stability for at least 30 days. Conclusions This study reveals a novel limitation of A2UCOE in inducible systems, specifically its contribution to gene leakage, which compromise sensitive systems like direct reprogramming of iPSCs. The inclusion of an SV40 poly-A sequence provides a practical solution and genomic architecture to improve the functionality of A2UCOE-based circuits. It also suggests investigating how termination of transcription modulates gene silencing as a novel design parameter. These findings have significant implications for the design of robust gene circuits, particularly in applications involving iPSCs, regenerative medicine, and cell therapy. Graphical abstract Supplementary Information The online version contains supplementary material available at 10.1186/s13036-025-00499-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84879df7e731c4525c04dff7de1fe6d2d28682c4" target='_blank'>
              Termination sequence between an inducible promoter and ubiquitous chromatin opening element (UCOE) reduces gene expression leakage and silencing
              </a>
            </td>
          <td>
            Tomoki Yanagi, Shean Fu Phen, Jonah Ayala, Deniz Ece Aydin, Susanna Jaramillo, David M. Truong
          </td>
          <td>2025-04-09</td>
          <td>Journal of Biological Engineering</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc7d674d8d38151120d4e80910c15408aca814f3" target='_blank'>
              Transcriptional stress induces the overexpression of novel lncRNAs that regulate the BRCA1 locus
              </a>
            </td>
          <td>
            Samantha Cruz-Ruiz, Raphael Vidal, Mayra Furlan-Magaril, John T. Lis, Mario Zurita
          </td>
          <td>2025-03-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73299a1254c80e9af5d25fe381a681e9866a35bc" target='_blank'>
              Studying the cellular efficacy and tolerability of using CRISPR-gRNA ribonucleoprotein (RNP) complex for in-vitro knockdown of TRIB2 in acute myeloid leukaemia (AML) cells as preliminary clinical evaluations.
              </a>
            </td>
          <td>
            Josephine You Pheng Tan, Ramesh Thevendran, Yek Song Quek, Solayappan Maheswaran
          </td>
          <td>2025-04-09</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Tumors exhibit highly heterogeneous genomic and transcriptomic landscapes. Characterization of this intratumor heterogeneity (ITH) and its implications for the cancer therapies has been one of the focal points of computational cancer biology in the past years. However, while ITH at the level of genomic changes, such as copy number aberrations, has been well characterized, the downstream impact of ITH on the gene regulatory landscape remains underexplored. Disruptions in gene regulatory programs can lead to emergence of more aggressive or drugresistant tumor clones. In order to better understand whether the evidence for ITH at the gene regulatory level can be inferred from scRNA-seq data we have conducted a series of experiments that explore the levels of gene regulatory network (GRN) discrepancies between the clonal sub-populations of tumor cells. In particular, we quantify the extent to which GRNs inferred from different tumor clones diverge from each other in three settings: triple negative breast cancer, colorectal cancer, and a longitudinal dataset from a patient undergoing chemotherapy for breast cancer. Our analyses indicate that a substantial (15-25%) proportion of GRN edges can be attributed to clone-specific activity. Furthermore, we observe differential network patterns across outlier transcription factors implicated in tumorogenesis and tumor progression. Our findings suggest that differential network patterns between clonal subpopulations can be reliably identified, and that such patterns offer a complementary insight that is not captured by differential gene analyses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86413a2b4bf81ee584f265b2f474b3de585e8771" target='_blank'>
              Intratumor Heterogeneity Through the Lens of Gene Regulatory Networks
              </a>
            </td>
          <td>
            Narein Rao, Nicolae Sapoval, Hamim Zafar, L. Nakhleh
          </td>
          <td>2025-04-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/055214dbdb29cbe5585cc333e89737a58f2ba504" target='_blank'>
              The Epigenetic Landscape: From Molecular Mechanisms to Biological Aging.
              </a>
            </td>
          <td>
            Rachel Evangelina, Subhashree Ganesan, Melvin George
          </td>
          <td>2025-03-17</td>
          <td>Rejuvenation research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Prognostic transcriptomic signatures for prostate cancer (PCa) often overlook the cellular origin of expression changes, an important consideration given the heterogeneity of the disorder. Current clinicopathological factors inadequately predict biochemical recurrence, a critical indicator guiding post-treatment strategies following radical prostatectomy. To address this, we conducted a meta-analysis of four large-scale PCa datasets and found 33 previously reported PCa-associated genes to be consistently up-regulated in prostate tumours. By analysing single-cell RNA-sequencing data, we found these genes predominantly as markers in epithelial cells. Subsequently, we applied 97 advanced machine-learning algorithms across five PCa cohorts and developed an 11-gene epithelial expression signature. This signature robustly predicted biochemical recurrence-free survival (BCRFS) and stratified patients into distinct risk categories, with high-risk patients showing worse survival and altered immune cell populations. The signature outperformed traditional clinical parameters in larger cohorts and was overall superior to published PCa signatures for BCRFS. By analysing peripheral blood data, four of our signature genes showed potential as biomarkers for radiation response in patients with localised cancer and effectively stratified castration-resistant patients for overall survival. In conclusion, this study developed a novel epithelial gene-expression signature that enhanced BCRFS prediction and enabled effective risk stratification compared to existing clinical- and gene-expression-derived prognostic tools. Furthermore, a set of genes from the signature demonstrated potential utility in peripheral blood, a tissue amenable to minimally invasive sampling in a primary care setting, offering significant prognostic value for PCa patients without requiring a tumour biopsy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0bf6f0e9c87d76cfdec8216eb4d7e07bdd3f760" target='_blank'>
              Integration of single-cell and bulk RNA-sequencing data reveals the prognostic potential of epithelial gene markers for prostate cancer.
              </a>
            </td>
          <td>
            Z. Mou, Lorna W Harries
          </td>
          <td>2025-02-19</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5392c951f2de30ee4a1335350f2dfe3bff927f11" target='_blank'>
              Circular RNAs: driving forces behind chemoresistance and immune evasion in bladder cancer.
              </a>
            </td>
          <td>
            M. Saadh, Wael Sheet Hussein, Ali Fawzi Al-Hussainy, A. K. Bishoyi, M. M. Rekha, Mayank Kundlas, V. Kavitha, Zafar Aminov, Sada Ghalib Taher, Mariem Alwan, M. Jawad, Hiba Mushtaq
          </td>
          <td>2025-03-25</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Overexpression of the H3K36 histone methyltransferase NSD2 in t(4;14) multiple myeloma (MM) is an early, oncogenic event, and understanding its impact on genomic organisation and expression is relevant to understanding MM biology. We performed epigenetic, transcriptional and phenotypic profiling of the t(4;14) KMS11 myeloma cell line and its isogenic translocation knock out (TKO) to characterise the sequelae of NSD2 overexpression. We found a marked global impact of NSD2 on gene expression and DNA organisation implicating cell identity genes; notably the early lymphocyte regulator, LAIR1 and MM cell surface markers, including CD38, a classical marker of plasma cells which was reduced in TKO cells. Plasma cell transcription factors such as PRDM1, IRF4 and XBP1 were unaffected, suggesting a downstream direct gene effect of NSD2 on cell identity. Changes in cell surface markers suggest an altered surface immunophenotype. Our findings suggest a role for NSD2 in maintaining MM cell identity, with potential implications for future therapeutic strategies based on targeting of NSD2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3900ea300ad687a680fc22372f0a01df92719adc" target='_blank'>
              NSD2-epigenomic reprogramming and maintenance of plasma cell phenotype in t(4;14) myeloma
              </a>
            </td>
          <td>
            Andrea Gunnell, Scott T. Kimber, R. Houlston, Martin Kaiser
          </td>
          <td>2025-03-21</td>
          <td>Oncotarget</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e607ac4077bd0584584c78f861a3a32f882b386" target='_blank'>
              Chemical Strategies to Modulate and Manipulate RNA Epigenetic Modifications.
              </a>
            </td>
          <td>
            Liang Cheng
          </td>
          <td>2025-03-18</td>
          <td>Accounts of chemical research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91ba15cf2f6390a1ec99834e397f27bbbddac7ca" target='_blank'>
              PFDN1 silencing disrupts critical cancer pathways in triple-negative breast cancer: investigating migration, cell cycle, and apoptosis as a new target therapy.
              </a>
            </td>
          <td>
            Abdulrahman M Alokda, A. Soffar, A. I. Yousef, Fawziya A R Ibrahim, Tarek S El-Sewedy, Alaa Elmetwalli
          </td>
          <td>2025-03-10</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9113b4cede2cf7486243e4f5408b0e991ee33d2d" target='_blank'>
              High-throughput single-cell DNA methylation and chromatin accessibility co-profiling with SpliCOOL-seq
              </a>
            </td>
          <td>
            Qingmei Shen, Enze Deng, Jingna Zhang, Qifeng Yang, Dan Su, Xiaoying Fan
          </td>
          <td>2025-03-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Glucose metabolism reprogramming provides significant insights into the development and progression of malignant tumors. This study aims to explore the temporal-spatial evolution of the glucose metabolism in HCC using single-cell sequencing and spatial transcriptomics (ST), and validates G6PD as a potential therapeutic target for HCC. Methods We collected single-cell sequencing data from 7 HCC and adjacent non-cancerous tissues from the GSE149614 database, and ST data from 4 HCC tissues from the HRA000437 database. Pseudotime analysis was performed on the single-cell data, while ST data was used to analyze spatial metabolic activity. High-throughput sequencing and experiments, including wound healing, CCK-8, and transwell assays, were conducted to validate the role and regulatory mechanisms of G6PD in HCC. Results Our study identified a progressive upregulation of PPP-related genes during tumorigenesis. ST analysis revealed elevated PPP metabolic scores in the central and intermediate tumor regions compared to the peripheral zones. High-throughput sequencing and experimental validation further suggested that G6PD-mediated regulation of HCC cell proliferation, migration, and invasion is likely associated with glutathione metabolism and ROS production. Finally, Cox regression analysis cofirmed G6PD as an independent prognostic factor for overall survival in HCC patients. Conclusion Our study provides novel insights into the changes in glucose metabolism in HCC from both temporal and spatial perspectives. We experimentally demonstrated that G6PD regulates proliferation, migration, and invasion in HCC and propose G6PD as a prognostic marker and therapeutic metabolic target for the HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92d4060fcfc333045f5d44b00ea2e74dde3d1885" target='_blank'>
              Single-cell sequencing and spatial transcriptomics reveal the evolution of glucose metabolism in hepatocellular carcinoma and identify G6PD as a potential therapeutic target
              </a>
            </td>
          <td>
            Deyang Xi, Yinshuang Yang, Jiayi Guo, Mengjiao Wang, Xuebing Yan, Chunyang Li
          </td>
          <td>2025-03-25</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3095fb742b0c6045977fafeab8ed1d06964edc6e" target='_blank'>
              FXYD5 regulates gastric cancer cell metastasis and drug resistance by EMT modulation.
              </a>
            </td>
          <td>
            Yuning Mao, Yaohua Hu, Han Meng, Jing Qin, Qingling An, Caiqin Zhang, Chenbo Guo, Yong Zhao, Dengxu Tan, Xu Ge, Changhong Shi
          </td>
          <td>2025-02-21</td>
          <td>Cancer gene therapy</td>
          <td>1</td>
          <td>13</td>
        </tr>

        <tr id="Delving into cancer dormancy has been an inherent task that may drive the lethal recurrence of cancer after primary tumor relief. Cells in quiescence can survive for a short or long term in silence, may undergo genetic or epigenetic changes, and can initiate relapse through certain contextual cues. The state of dormancy can be induced by multiple conditions including cancer drug treatment, in turn, undergoes a life cycle that generally occurs through dissemination, invasion, intravasation, circulation, immune evasion, extravasation, and colonization. Throughout this cascade, a cellular machinery governs the fate of individual cells, largely affected by gene regulation. Despite its significance, a precise view of cancer dormancy is yet hampered. Revolutionizing advanced single cell and long read sequencing through analysis methodologies and artificial intelligence, the most recent stage in the research tool progress, is expected to provide a holistic view of the diverse aspects of cancer dormancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cb4233622be9dcbd1fb28cce4bd3a5ada680860" target='_blank'>
              Towards understanding cancer dormancy over strategic hitching up mechanisms to technologies
              </a>
            </td>
          <td>
            Sumin Yang, Jieun Seo, Jeonghyeon Choi, Sung-Hyun Kim, Yunmin Kuk, Kyung Chan Park, Mingon Kang, Sangwon Byun, Jae-Yeol Joo
          </td>
          <td>2025-02-14</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Transcriptional activation of otherwise repressed endogenous retroelements (RTEs) is a hallmark of cancer, shaping tumour progression and immunogenicity by multifaceted, yet incompletely understood mechanisms. Methods We used an extended pan-cancer transcriptome assembly to identify potential effects of RTEs on the genes near or within which they have integrated. These were subsequently verified in test cases by further analysis of transcriptional profiles in cancer patient data, and by in vitro studies involving restoration of gene activity, and proliferation and migration assays in cancer cell lines. Results We report that cancer-specific transcriptional activation of RTEs causes frequent reduction or loss of gene function. Exonisation and alternative splicing of RTEs creates non-functional RNA and protein isoforms and derepressed RTE promoter activity initiates antisense transcription, both at the expense of the canonical isoforms. Contrary to theoretical expectation, transcriptionally activated RTEs affect genes with established tumour-promoting function, including the common essential RNGTT and the lung cancer-promoting CHRNA5 genes. Furthermore, the disruptive effect of RTE activation on adjacent tumour-promoting genes is associated with slower disease progression in clinical data, whereas experimental restoration of gene activity enhances tumour cell in vitro growth and invasiveness Conclusions These findings underscore the gene-disruptive potential of seemingly innocuous germline RTE integrations, unleashed only by their transcriptional utilisation in cancer. They further suggest that such metastable RTE integrations are co-opted as sensors of the epigenetic and transcriptional changes occurring during cellular transformation and as executors that disrupt the function of tumour-promoting genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf39ff907245282864eb90768249daa905476758" target='_blank'>
              Retroelement co-option disrupts the cancer transcriptional programme
              </a>
            </td>
          <td>
            Jane Loong, Rachael Thompson, Callum Hall, Laura Doglio, Judith Pape, G. Kassiotis
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Epigenetic dysregulation is prevalent in human cancers, affecting gene expression and metabolic patterns to meet the demands of malignant evolution and abnormal epigenetic processes, and resulting in a protumor immune microenvironment. Tumors require a steady supply of methionine for maintaining epigenetic flexibility, which is the only exogenous precursor of methyl donor S-adenosylmethionine for methylation, crucial for their resistance to therapies and survival in a nutrient-deficient microenvironment. Thus, tumor cells upregulate the Lat4 transporter to compete and deprive methionine in the microenvironment, sustaining their malignant phenotypes and also impairing immune cell functions. Addressing this methionine addiction is the key to overcoming drug resistance and improving immune response. Despite the challenge of lacking specific Lat4 inhibitors, an oxaliplatin prodrug crosslinked fluorinated polycation/anti-Lat4 small interfering RNA complex nanoregulator (AS-F-NP) has been designed and developed here. This nanoregulator restricted the greedy methionine uptake of tumor cells by knocking down Lat4, which in turn inhibited the malignant evolution of the tumor while restoring the viability and function of tumor-infiltrating immune cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96e01c550a22a629f238ced405d904af38f03862" target='_blank'>
              A Methionine Allocation Nanoregulator for the Suppression of Cancer Stem Cells and Support to the Immune Cells by Epigenetic Regulation.
              </a>
            </td>
          <td>
            Boyu Su, Qinjun Chen, Xuwen Li, Mingzhu Fang, Yu Wang, Haolin Song, Haoyu You, Zhen-yong Zhou, Yuxing Wu, Zhenhao Zhao, Yun Chen, Hongrui Fan, Chufeng Li, Chen Jiang, Tao Sun
          </td>
          <td>2025-02-22</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab9164c2b57024d5a00b47fc6698b2e2fcdfba04" target='_blank'>
              Mapping the Future: A Content Analysis of the Evolution of Gene Therapy in Urological Cancer
              </a>
            </td>
          <td>
            Muhammad Sidharta Krisna, Muhammad Alfi Reza, Bobby Aksanda Putra, Bukhari Muslim Siregar, Muhammad Alif Adhani
          </td>
          <td>2025-02-14</td>
          <td>International Journal of Health and Pharmaceutical (IJHP)</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Exonic enhancers (EEs) occupy an under-appreciated niche in gene regulation. By integrating transcription factor binding, chromatin accessibility, and high-throughput enhancer-reporter assays, we demonstrate that many protein-coding exons possess enhancer activity across species. These EEs exhibit characteristic epigenomic signatures, form long-range interactions with gene promoters, and can be altered by both nonsynonymous and synonymous variants. CRISPR–mediated inactivation demonstrated the involvement of EEs in the cis-regulation of host and distal gene expression. Through large-scale cancer genome analyses, we reveal that EE mutations correlate with dysregulated target-gene expression and clinical outcomes, highlighting their potential relevance in disease. Evolutionary comparisons show that EEs exhibit both strong sequence constraint and lineage-specific plasticity, suggesting that they serve ancient regulatory functions while also contributing to species divergence. Our findings redefine the landscape of functional elements by establishing EEs as a component of gene regulation, while revealing how coding regions can simultaneously fulfil both protein-coding and cis-regulatory roles.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/163da6e03d81a31e9fe14936d274ce9d2b480af7" target='_blank'>
              Exonic enhancers are a widespread class of dual-function regulatory elements
              </a>
            </td>
          <td>
            Jean-Christophe Mouren, Magali Torres, Antoinette F. van Ouwerkerk, Iris Manosalva, F. Gallardo, Salvatore Spicuglia, Benoît Ballester
          </td>
          <td>2025-04-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/255726480b1fba8b96c7a797db393035b828db59" target='_blank'>
              Abstract B022: Decipher actionable tumor suppressor defects for personalized cancer medicine
              </a>
            </td>
          <td>
            Feiyu Chen, Di Zhao
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Lung squamous cell carcinoma (LUSC), accounting for 30% of lung cancer cases, lacks adequate research due to limited understanding of its molecular abnormalities. Our study analyzed public LUSC datasets to explore the tumor microenvironment (TME) composition using scRNA-seq from two cohorts. Applying non-negative matrix factorization, we identified unique malignant cell phenotypes, or meta-programs (MPs), based on gene expression patterns. Survival analysis revealed the clinical relevance of these MPs. Findings illuminated a TME landscape enriched with immune cells—CD8 + T, exhausted T, CD4 + T, and naive T cells—and suggested roles for myeloid cells, like cDC1 and pDCs, in LUSC progression. Different MPs highlighted the heterogeneity of malignant cells and their clinical implications. Targeting MP-specific genes may enable personalized therapy, especially for early-stage LUSC. This study offers insights into immune cell function in tumor dynamics, identifies MPs, and paves the way for novel LUSC strategies, enhancing early intervention, personalized treatment, and prognosis, ultimately improving patient outcomes. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-93916-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a302b866a51ae52b8b8d7352c3feab56f45e3d0e" target='_blank'>
              Single-cell RNA sequencing reveals potential therapeutic targets in the tumor microenvironment of lung squamous cell carcinoma
              </a>
            </td>
          <td>
            Junda Fan, Yu Chen, Yue Gong, Hongmei Sun, Rui Hou, Xiaoya Dou, Yanping Zhang, Cheng Huo
          </td>
          <td>2025-03-26</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c5574ba347be751b5260edd71a2fcb0e3909c08" target='_blank'>
              Integrative single-cell and bulk RNA-seq analysis identifies lactylation-related signature in osteosarcoma.
              </a>
            </td>
          <td>
            Zhou Xie, Xiao Qu, Jun Zhang, Yanran Huang, Runhan Zhao, Dagang Tang, Ningdao Li, Zhule Wang, Xiaoji Luo
          </td>
          <td>2025-03-12</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Telomeres are dynamic complexes at the ends of chromosomes that are made up of protective proteins and tandem repeating DNA sequences. In the large majority of cancer cells, telomere length is maintained by telomerase, an enzyme that elongates telomeres. Telomerase activation is seen in the majority of cancer, which permits uncontrol cell proliferation. About 90% of human malignancies show telomere dysfunction and telomerase reactivation; as a result, telomerase activation plays a special role as a practically universal stage on the way to malignancy. This review understands the structural and functional of telomere and telomerase, mechanisms of telomerase activation in oncogenesis, biomarkers and therapeutic targets. Therapeutic strategies targeting telomerase, including antisense oligonucleotides, G-quadruplex stabilizers, immunotherapy, small-molecule inhibitors, gene therapy, Telomerase-Responsive Drug Release System, have shown promise in preclinical and clinical settings. Advances in telomere biology not only illuminate the complex interplay between telomeres, telomerase, and cancer progression but also open avenues for innovative, targeted cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0a88e3f76f3fb38bb6a5ed80e16846195631c78" target='_blank'>
              The role of telomere and telomerase in cancer and novel therapeutic target: narrative review
              </a>
            </td>
          <td>
            Temesgen Baylie, Mohammed Jemal, Gelagay Baye, Mamaru Getinet, G. Amare, Adane Adugna, D. Abebaw, Zigale Hibstu, B. Tegegne, Endalkachew Gugsa, Tadegew Adane, Gedefaw Getie, Baye Ashenef, Deresse Sinamaw
          </td>
          <td>2025-02-14</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Background Pseudogene‐derived lncRNAs are widely dysregulated in cancer. Technological advancements have facilitated the functional characterization of increasing pseudogenes in cancer progression. However, the association between pseudogenes and RNA N6‐methyladenosine (m6A) modification in cancer, as well as the underlying mechanisms, remains largely unexplored. Methods We analyzed the expression of 12 146 pseudogenes and comprehensively examined the m6A modification of RNAs derived from them and their paralogs. Through integrative analysis of multi‐omics data, we explored the associations between pseudogene dysregulation and m6A, identifying critical pseudogenes involved in HGSOC progression. Tumour promotion role of RPS15AP12 and its cognate parent gene was characterized by cell proliferation, transwell assays, and scratch assays in ovarian cells and xenograft nude mice. RNA decay assays were used to reveal the participation of m6A in decreasement of RPS15AP12 lncRNA stability. Luciferase reporter assays were performed to verify that RPS15AP12 enhances RPS15A expression by competitively binding to miR‐96‐3p. Western blot and phosphorylation assays were performed to investigate the impairment of RPS15AP12 towards the sensors of MAVS (RIG‐I and MDA5), and downstream p‐TBK1 and p‐IRF3. Finally, ELISA assays were performed to explore the regulatory role of RPS15AP12 in IFN‐β expression. Results M6A is distributed across over a thousand pseudogenes, and hypomethylation leads to their upregulation in HGSOC. We identified a processed pseudogene, RPS15AP12, upregulated by FTO‐mediated m6A demethylation. RPS15AP12 enhances the growth ability and metastatic capabilities of ovarian cancer (OC) cells via functioning as a competitive endogenous RNA (ceRNA) for its host gene, RPS15A, through the sequestration of miR‐96‐3p. Importantly, the deletion of RPS15AP12 diminishes the expression of RPS15A, leading to the upregulation of anti‐tumour immune responses by activating RIG‐I and MDA5 and downstream p‐TBK1 and p‐IRF3 as well as IFN‐β levels. Conclusion Our findings expand the understanding of m6A‐modulated pseudogenes in tumour growth and anti‐tumour innate immunity in OC. Key Points Genome‐wide profiling reveals the redistribution of m6A modification on pseudogene‐derived lncRNAs and m6A redistribution‐relevant dysregulation of pseudogenes in HGSOC. RPS15AP12, as a representative processed pseudogene, is up‐regulated by FTO‐mediated demethylation and acts as a miRNA sponge to promote RPS15A expression via competitively binding to miR‐96‐3p. RPS15AP12/RPS15A axis inhibits MAVS sensors (RIG‐I and MDA5) and downstream IFN‐β levels in ovarian cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7f32a6bc1ac1ae8503c9d4c32bcdde0ae2bb380" target='_blank'>
              Genome‐wide profiling of N6‐methyladenosine‐modified pseudogene‐derived long noncoding RNAs reveals the tumour‐promoting and innate immune‐restraining function of RPS15AP12 in ovarian cancer
              </a>
            </td>
          <td>
            Jie Xu, Yifei Ren, Jiayi Lu, Fengjiang Qin, Dan Yang, Chunyan Tang, Yu Yang, Jing Xu, Tao Liu, Ping Yi
          </td>
          <td>2025-02-25</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Activating PIK3CA mutations, present in up to 40% of hormone receptor-positive (HR+), human epidermal growth factor receptor 2-negative (Her2−) breast cancer (BC) patients, can be effectively targeted with the alpha isoform-specific PI3K inhibitor Alpelisib. This treatment significantly improves outcomes for HR+, Her2−, and PIK3CA-mutated metastatic BC patients. However, acquired resistance, often due to aberrant activation of the mTOR complex 1 (mTORC1) pathway, remains a significant clinical challenge. Our study, using in vitro and orthotopic xenograft mouse models, demonstrates that constitutively active mTORC1 signaling renders PI3K inhibitor-resistant BC exquisitely sensitive to various drugs targeting cancer metabolism. Mechanistically, mTORC1 suppresses the induction of autophagy during metabolic perturbation, leading to energy stress, a critical depletion of aspartate, and ultimately cell death. Supporting this mechanism, BC cells with CRISPR/Cas9-engineered knockouts of canonical autophagy genes showed similar vulnerability to metabolically active drugs. In BC patients, high mTORC1 activity, indicated by 4E-BP1T37/46 phosphorylation, correlated with p62 accumulation, a sign of impaired autophagy. Together, these markers predicted poor overall survival in multiple BC subgroups. Our findings reveal that aberrant mTORC1 signaling, a common cause of PI3K inhibitor resistance in BC, creates a druggable metabolic vulnerability by suppressing autophagy. Additionally, the combination of 4E-BP1T37/46 phosphorylation and p62 accumulation serves as a biomarker for poor overall survival, suggesting their potential utility in identifying BC patients who may benefit from metabolic therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c69e4dbaba941d1cd07858705c4708f9bc19fb8f" target='_blank'>
              Targeting PI3K inhibitor resistance in breast cancer with metabolic drugs
              </a>
            </td>
          <td>
            N. Gremke, I. Besong, Alina Stroh, Luise von Wichert, Marie Witt, S. Elmshäuser, M. Wanzel, Martin F Fromm, R. V. Taudte, S. Schmatloch, T. Karn, M. Reinisch, N. Hirmas, S. Loibl, Thomas Wündisch, Anne-Sophie Litmeyer, Paul Jank, Carsten Denkert, Sebastian Griewing, U. Wagner, Thorsten Stiewe
          </td>
          <td>2025-03-21</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Despite hundreds of genetic risk loci identified for neuropsychiatric disorders (NPD), most causal variants/genes remain unknown. A major hurdle is that disease risk variants may act in specific biological contexts, e.g., during neuronal activation, which is difficult to study in vivo at the population level. Here, we conducted a single-cell multiomics study of neuronal activation (stimulation) in human iPSC-induced excitatory and inhibitory neurons from 100 donors, and uncovered abundant neuronal stimulation-specific causal variants/genes for NPD. We surveyed NPD-relevant transcriptomic and epigenomic landscape of neuronal activation and identified thousands of genetic variants associated with activity-dependent gene expression (i.e., eQTL) and chromatin accessibility (i.e., caQTL). These caQTL explained considerably larger proportions of NPD heritability than the eQTL. Integrating the multiomic data with GWAS further revealed NPD risk variants/genes whose effects were only detected upon stimulation. Interestingly, multiple lines of evidence support a role of activity-dependent cholesterol metabolism in NPD. Our work highlights the power of cell stimulation to reveal context-dependent “hidden” genetic effects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a148d962f25c310bd7b436ba7fe1187d99a87de2" target='_blank'>
              Single-cell multiomics of neuronal activation reveals context-dependent genetic control of brain disorders
              </a>
            </td>
          <td>
            Lifan Liang, Siwei Zhang, Zicheng Wang, Hanwen Zhang, Chuxuan Li, Alexandra C. Duhe, Xiaotong Sun, Xiaoyuan Zhong, Alena Kozlova, Brendan Jamison, Whitney Wood, Zhiping P. Pang, Alan R. Sanders, Xin He, J. Duan
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Background/Objectives: Fragile X syndrome (FXS) is a disease of pathologic epigenetic silencing induced by RNA. In FXS, an expanded CGG repeat tract in the FMR1 gene induces epigenetic silencing during embryogenesis. FMR1 silencing can be reversed with 5-aza-deoxyctidine (5-aza-dC), a nonspecific epigenetic reactivator; however, continuous administration of 5-aza-dC is problematic due to its toxicity. We describe an approach to restore FMR1 expression in FXS neurons by transient treatment with 5-aza-dC, followed by treatment with 2HE-5NMe, which binds the CGG repeat expansion in the FMR1 mRNA and could block the resilencing of the FMR1 gene after withdrawal of 5-aza-dC. Methods: This study uses immunofluorescence and fluorescent in situ hybridization (FISH) to measure whether FMR1 expression is maintained in FXS post-mitotic neurons treated with 2HE-5NMe. Genome-wide profiling of histone marks was used to monitor epigenetic changes and drug selectivity in response to 5-aza-dC followed by 2HE-5NMe treatment. Changes to dendritic morphology were visualized using confocal microscopy. Results: In this study, we find that 2HE-5Nme maintains FMR1 in a reactivated state after reactivation using 5-aza-dC in post-mitotic neurons. FMR1 reactivation in neurons results in the re-expression of FMRP and reversal of FXS-associated dendritic spine defects. Conclusions: These results demonstrate that an RNA-binding small molecule can achieve gene-specific epigenetic control and provide an approach for the restoration of FMRP in FXS neurons.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ddaf9f6479660c108f91bf86393ce4507c8e607" target='_blank'>
              Sustained Epigenetic Reactivation in Fragile X Neurons with an RNA-Binding Small Molecule
              </a>
            </td>
          <td>
            Christina W. Kam, Jason G Dumelie, G. Ciceri, Wang-Yong Yang, Matthew D Disney, Lorenz Studer, S. Jaffrey
          </td>
          <td>2025-02-25</td>
          <td>Genes</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f1f61ef381af091b6db128907436bc9e8a1a514" target='_blank'>
              Directed evolution expands CRISPR-Cas12a genome editing capacity
              </a>
            </td>
          <td>
            E. Ma, Kai Chen, Honglue Shi, Kevin Wasko, Isabel Esain-Garcia, M. Trinidad, Kaihong Zhou, Jinjuan Ye, Jennifer A. Doudna
          </td>
          <td>2025-03-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35da01c8609a7028b2e2ec26d0b8011e3cf58489" target='_blank'>
              A single-cell transcriptomic comparison between small intestinal neuroendocrine tumors and their progenitor
              </a>
            </td>
          <td>
            Fredrik Axling, Elham Barazeghi, Per Hellman, O. Norlén, S. Backman, P. Stålberg
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/840081d94faa4f97800b809929ce77b526951304" target='_blank'>
              Enhancement of colorectal cancer therapy through interruption of the HSF1-HSP90 axis by p53 activation or cell cycle inhibition.
              </a>
            </td>
          <td>
            Tamara Isermann, Kim Lucia Schneider, F. Wegwitz, Tiago De Oliveira, Lena-Christin Conradi, Valery Volk, F. Feuerhake, Björn Papke, Sebastian Stintzing, Bettina Mundt, Florian Kühnel, Ute M. Moll, Ramona Schulz-Heddergott
          </td>
          <td>2025-04-09</td>
          <td>Cell death and differentiation</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The primary objective of this study is to conduct a pan-cancer analysis of CHRDL1 expression, to determine its correlation with patient survival rates, immune cell infiltration, and drug sensitivity. Additionally, the study aimed to further validate the mechanistic role of CHRDL1 in lung adenocarcinoma (LUAD), clarifying its contribution to tumorigenesis and evaluating its potential as a therapeutic target for LUAD.We employed bioinformatics strategies to analyze CHRDL1 expression using data from The Cancer Genome Atlas (TCGA) and the Genotype-Tissue Expression Project (GTEx). Survival analysis was executed with GEPIA2, while drug sensitivity to chemotherapeutic agents was evaluated via the CellMiner database. Mutational profiles were examined using cBioPortal, and the immune microenvironment was assessed through the TIMER database. To substantiate our findings, we conducted in vitro cellular assays and in vivo animal models to validate the mechanistic actions of CHRDL1 in LUAD.CHRDL1 expression levels showed significant variation across different cancer types, with tumor tissues typically demonstrating lower expression compared to their normal counterparts. In certain cancers, elevated CHRDL1 expression was linked to poorer survival outcomes, whereas in LUAD, it was associated with improved survival. Furthermore, CHRDL1 expression correlated with the IC50 values of multiple chemotherapeutic drugs and played a role in modulating the immune microenvironment. We discovered that CHRDL1 inhibits the epithelial-mesenchymal transition (EMT) in LUAD through the TGF-β pathway.CHRDL1 exerts a complex influence on cancer development and progression, particularly in LUAD, by impacting tumor progression, immune regulation, chemosensitivity, and EMT regulation. This research offers valuable insights into the overarching mechanisms of cancer progression and aids in the discovery of innovative therapeutic strategies for LUAD treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd79eedadd08697bb7daedaf44fd4b0c2b964ff9" target='_blank'>
              Pan-cancer analysis of CHRDL1 expression and its mechanistic role in inhibiting EMT via the TGF-β pathway in lung adenocarcinoma
              </a>
            </td>
          <td>
            Honghong Dong, Yahui Tian, Shaowei Xin, Yujie Guo, Suxin Jiang, Zitong Wan, Huai-Yu Wang, Yong Han
          </td>
          <td>2025-03-31</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fc9176f3b57d689386be0c2303a9e42c03c1797" target='_blank'>
              Unraveling the hidden complexity of cancer through long-read sequencing.
              </a>
            </td>
          <td>
            Qiuhui Li, Ayse G Keskus, Justin Wagner, Michal B Izydorczyk, W. Timp, F. Sedlazeck, A. Klein, J. Zook, M. Kolmogorov, Michael C. Schatz
          </td>
          <td>2025-03-20</td>
          <td>Genome research</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="CRISPR-Cas9 systems have revolutionized genome editing, but the off-target effects of Cas9 limit its use in clinical applications. Here, we systematically evaluate FrCas9, a variant from Faecalibaculum rodentium, for cell and gene therapy (CGT) applications and compare its performance to SpCas9 and OpenCRISPR-1. OpenCRISPR-1 is a CRISPR system synthesized de novo using large language models (LLMs) but has not yet undergone systematic characterization. Using AID-seq, Amplicon sequencing, and GUIDE-seq, we assessed the on-target activity and off-target profiles of these systems across multiple genomic loci. FrCas9 demonstrated higher on-target efficiency and substantially fewer off-target effects than SpCas9 and OpenCRISPR-1. Furthermore, TREX2 fusion with FrCas9 reduced large deletions and translocations, enhancing genomic stability. Through screening of 1903 sgRNAs targeting 21 CGT-relevant genes using sequential AID-seq, Amplicon sequencing, and GUIDE-seq analysis, we identified optimal sgRNAs for each gene. Our high-throughput screening platform highlights FrCas9, particularly in its TREX2-fused form, as a highly specific and efficient tool for precise therapeutic genome editing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6808f0ec0ae3c51721dc4169f97a812e9c172d0f" target='_blank'>
              Systematic high-throughput evaluation reveals FrCas9’s superior specificity and efficiency for therapeutic genome editing
              </a>
            </td>
          <td>
            Rui Tian, Xun Tian, Meiying Yang, Yuping Song, Tingting Zhao, Chaoyue Zhong, Wei Zhu, Ping Zhou, Zhiqiang Han, Zheng Hu
          </td>
          <td>2025-03-26</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a73dc13e33e938dc26c7fd4908040282b9950aea" target='_blank'>
              Advances in CRISPR-enabled genome-wide screens in yeast.
              </a>
            </td>
          <td>
            Nicholas R. Robertson, Sangcheon Lee, Aida Tafrishi, I. Wheeldon
          </td>
          <td>2025-03-20</td>
          <td>FEMS yeast research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68278946b31fce3c9e63062ccd0dab791b5f86e0" target='_blank'>
              The miRNA-9 Isoform Story in Cancer: An OncomiR or Tumor Suppressor?
              </a>
            </td>
          <td>
            Mahmoud Ibrahim, Maryam Abdul-Kader, Ava Azizi, Evan Schneider, Gabriel Salyer, Jill Ivory, Nora Reece-Orr, Heather O’Donnell, Linda Gerace, Zoe Foster, Alan Mariño del Puerto, Yvonne Chen, Brian D. Adams
          </td>
          <td>2025-04-08</td>
          <td>Journal of Cancer Biomoleculars and Therapeutics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Objectives/Goals: Large-scale tumor sequencing efforts have led to annotations of novel cancer hotspot mutations that may underlie driver or cooperative function. We have sought to define the molecular consequences of such hotspots associated with pediatric DICER1 syndrome cancers, with the ultimate goal of revealing novel targets that may inform new standards of care. Methods/Study Population: We have performed genomic analysis to identify tumor types (in TCGA and MSK-IMPACT patient data) for which mutations in the Dicer1 gene (encoding Dicer protein) emerge as the dominant signature of driver function. As Dicer is a critical RNA processing factor responsible for the generation of microRNAs, which are posttranscriptional gene regulatory molecules, we have modeled these mutations in human embryonic stem cells in order to study the direct effects on miRNAs and their target genes in an isogenic background. In addition to providing the required setting for unambiguous attribution of function to specific mutations, clonal human ES cells offer an opportunity for modeling of both developmental and cancer requirements associated with altered Dicer function. Results/Anticipated Results: Through generation of genomics and functional datasets from matched genotypes in Dicer mutated human ES cells, we have identified specific alterations in miRNAs and their effects on target genes. Unexpectedly, we found direct evidence for both loss of function and gain of function attributable to Dicer mutations. In addition, through integrated analysis of genomic data from tumor sequencing datasets and our human ES cell models, we have identified potential miRNA and target gene alterations that underlie tumorigenic potential, nominating gene candidates for targeted therapy in DICER1 syndrome. Direct mouse modeling of such candidate gene targets has revealed evidence for driver function of identified miRNA and their targets. Discussion/Significance of Impact: DICER1 syndrome cancers comprise a wide variety of rare pediatric tumor types. Presently, we still lack an effective standard of care. Furthermore, the previous lack of molecular profiling precluded targeted therapy opportunities. Our precise knock-in modeling of Dicer hotspots and deep profiling of relevant tumors now provide candidate targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/597fce8f39319f3d39a09a21bf59ed3021208522" target='_blank'>
              518 Modeling of cancer mutations found in pediatric DICER1 syndrome informs novel therapeutic targeting strategies
              </a>
            </td>
          <td>
            David Jee, Seungjae Lee, Dapeng Yang, Robert W. Rickert, Renfu Shang, Danwei Huangfu, Eric C. Lai
          </td>
          <td>2025-03-26</td>
          <td>Journal of Clinical and Translational Science</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fb803b3e058115ac9f37e42f7b7a8e8748c0c02" target='_blank'>
              Next-Generation Sequencing-Based In Silico Transgene Integration Profiling Tool Accelerates Cell Line Genetic Characterizations.
              </a>
            </td>
          <td>
            Zhenqiu Huang, Hyo-Young Jeong, Luke Nelson, Caitlyn Devine, Guanghua Benson Li, Lin Zhang
          </td>
          <td>2025-03-01</td>
          <td>Biotechnology journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="CRISPR/Cas9 based genome editing has advanced our understanding of a myriad of important biological phenomena. Important challenges to multiplex genome editing in maize include assembly of large complex DNA constructs, few genotypes with efficient transformation systems, and costly/labor-intensive genotyping methods. Here we present an approach for multiplex CRISPR/Cas9 genome editing system that delivers a single compact DNA construct via biolistics to Type I embryogenic calli, followed by a novel efficient genotyping assay to identify desirable editing outcomes. We first demonstrate the creation of heritable mutations at multiple target sites within the same gene. Next, we successfully created individual and stacked mutations for multiple members of a gene family. Genome sequencing found off-target mutations are rare. Multiplex genome editing was achieved for both the highly transformable inbred line H99 and Illinois Low Protein1 (ILP1), a genotype where transformation has not previously been reported. In addition to screening transformation events for deletion alleles by PCR, we also designed PCR assays that selectively amplify deletion or insertion of a single nucleotide, the most common outcome from DNA repair of CRISPR/Cas9 breaks by non-homologous end-joining. The Indel-Selective PCR (IS-PCR) method enabled rapid tracking of multiple edited alleles in progeny populations. The ‘end to end’ pipeline presented here for multiplexed CRISPR/Cas9 mutagenesis can be applied to accelerate maize functional genomics in a broader diversity of genetic backgrounds. Supplementary Information The online version contains supplementary material available at 10.1186/s13007-025-01365-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de431ff0e73b17a2417c342121bd7ddd4aecfa44" target='_blank'>
              Efficient mutagenesis and genotyping of maize inbreds using biolistics, multiplex CRISPR/Cas9 editing, and Indel-Selective PCR
              </a>
            </td>
          <td>
            Maruti Nandan Rai, Brian Rhodes, Stephen Jinga, Praveena Kanchupati, Edward Ross, Shawn R. Carlson, Stephen P. Moose
          </td>
          <td>2025-03-25</td>
          <td>Plant Methods</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Cancer stem cells (CSCs) are crucial for lung adenocarcinoma (LUAD). This study investigates tumor stem cell gene signatures in LUAD using single-cell RNA sequencing (scRNA-seq) and bulk RNA sequencing (RNA-seq), aiming to develop a prognostic tumor stem cell marker signature (TSCMS) model. Methods LUAD scRNA-seq and RNA-seq data were analyzed. CytoTRACE software quantified the stemness score of tumor-derived epithelial cell clusters. Gene Set Variation Analysis (GSVA) identified potential biological functions in different clusters. The TSCMS model was constructed using Lasso-Cox regression, and its prognostic value was assessed through Kaplan–Meier, Cox regression, and receiver-operating characteristic (ROC) curve analyses. Immune infiltration was evaluated using the Cibersortx algorithm, and drug response prediction was performed using the pRRophetic package. TAF10 functional investigations in LUAD cells involved bioinformatics analysis, qRT-PCR, Western blot, immunohistochemistry, and assays for cell proliferation. Results Seven distinct cell clusters were identified by CytoTRACE, with epithelial cell cluster 1 (Epi_C1) showing the highest stemness potential. The TSCMS model included 49 tumor stemness-related genes; high-risk patients exhibited lower immune and ESTIMATE scores and increased tumor purity. Significant differences in immune landscapes and chemotherapy sensitivity were observed between risk groups. TAF10 positively correlated with RNA expression-based stemness scores in various tumors, including LUAD. It was over-expressed in LUAD cell lines and clinical tumor tissues, with high expression linked to poor prognosis. Silencing TAF10 inhibited LUAD cell proliferation and tumor sphere formation. Conclusions This study demonstrates the TSCMS model's prognostic value in LUAD, reveals insights into immune infiltration and therapeutic response, and identifies TAF10 as a potential therapeutic target. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06243-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5653c962ff1a70fb0bb0f0fb2592a34ab549e9a8" target='_blank'>
              Integration of single-cell and bulk RNA sequencing to identify a distinct tumor stem cells and construct a novel prognostic signature for evaluating prognosis and immunotherapy in LUAD
              </a>
            </td>
          <td>
            Fengyun Zhao, Mengting Chen, Tianjiao Wu, M. Ji, Fugui Li
          </td>
          <td>2025-02-22</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/498b068e38445ba010a97c92197f812fab2d458c" target='_blank'>
              Abstract PR010: Genome-wide CRISPR screens in CD8+ T cells identify cullin-RING E3 ubiquitin ligase complex as a negative regulator of long-term effector function
              </a>
            </td>
          <td>
            Justin D. Saco, Daniel Karin, Nataly Naser Al Deen, Katie M. Campbell, Justin Patten, Yichen Fan, Egmidio Medina, Antoni Ribas
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Rationale: Despite substantial advancement in the treatment of B-cell acute lymphoblastic leukemia (B-ALL), it remains a leading cause of cancer mortality in children due to the high relapse rate. Moreover, the long-term survival rates for adult B-ALL patients are still less than 40%. The B-ALL patients carrying MLL rearrangements or BCR-ABL fusion represent high-risk B-ALL subtypes that face particularly dismal prognoses. This study aims to identify innovative therapeutic vulnerability for high-risk B-ALL. Methods: The CRISPR-Cas9 screen was conducted to pinpoint genes essential for high-risk B-ALL cell survival/growth. Both in vitro and in vivo models were then employed to investigate the pathological role of ZNF217 in high-risk B-ALL. To characterize the downstream functionally essential targets of ZNF217, we performed RNA-seq and CUT&RUN-seq, followed by integrative bioinformatics analysis and experimental validation. Results: Through the focused CRISPR-Cas9 screening, ZNF217 emerged as the most essential gene for the cell survival/growth of B-ALL driven by MLL rearrangement or BCR-ABL. Through in vitro gain- and loss-of-function assays, we demonstrated that ZNF217 is indeed required for B-ALL cell survival/growth. Moreover, we established the B-ALL xenograft model and patient-derived xenograft (PDX) model and demonstrated that ZNF217 depletion significantly suppressed B-ALL progression and substantially extended the survival of recipient mice. Through integrative multiple-omics analysis, we elucidated that ZNF217 exerts its oncogenic role in B-ALL through both CoREST-dependent and CoREST-independent mechanisms. Furthermore, we characterized FOS as a functionally essential downstream target of ZNF217, and ZNF217 inhibited FOS expression in a CoREST-independent manner. Conclusions: Our findings highlight ZNF217 as a promising therapeutic target for the treatment of high-risk B-ALL, such as those carrying MLL-rearrangements or BCR-ABL fusion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a506db79238d620427652c02a2a5ec31786dbecb" target='_blank'>
              CRISPR screening reveals ZNF217 as a vulnerability in high-risk B-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Xi Qin, Keren Zhou, Lei Dong, Lu Yang, Wei Li, Zhenhua Chen, Chao Shen, Li Han, Yangchan Li, A. Chan, S. Pokharel, Y. Qing, Meiling Chen, K. Wang, Keith Leung, Lillian Sau, C. Chen, Xiaolan Deng, R. Su, Jianjun Chen
          </td>
          <td>2025-02-18</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="ABSTRACT Epigenetics and epigenomics are captivating fields of molecular biology, dedicated to the exploration of heritable alterations in gene expression and cellular phenotypes, which transpire devoid of any discernible modifications to the fundamental DNA sequence. This intricate regulatory apparatus encompasses multiple mechanisms, prominently featuring DNA methylation, histone modifications, and the involvement of non-coding RNA molecules in pivotal roles. To achieve a comprehensive grasp of these diverse mechanisms, it is imperative to conduct research employing animal models as proxies for human studies. Since experimental animal models like mice and rats struggle to replicate the diverse environmental conditions experienced by humans, this review focuses on comparing common epigenetic alterations in naturally occurring tumors in canine models, which share the human environment, with those in humans. Through this, we emphasize the importance of an epigenetic regulation in the comparative medical approach to a deeper understanding of cancers and further development of cancer treatments. Additionally, we elucidate epigenetic modifications pertinent to specific developmental stages, the ageing process, and the progression of various diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17bfc3c89b9e9910bb56cc9d2d52428ce2d674ca" target='_blank'>
              Comparative epigenomics to clinical trials in human breast cancer and canine mammary tumor
              </a>
            </td>
          <td>
            Su-Jin Jeong, Kang-Hoon Lee, Je-Yoel Cho
          </td>
          <td>2025-03-19</td>
          <td>Animal Cells and Systems</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b20395b5511b0ca628f7388834f767639654823" target='_blank'>
              Conserved and Unique Features of Terminal Telomeric Sequences in ALT-Positive Cancer Cells
              </a>
            </td>
          <td>
            Benura Azeroglu, Wei Wu, R. Pavani, Ranjodh Sandhu, Tadahiko Matsumoto, A. Nussenzweig, E. L. Denchi
          </td>
          <td>2025-03-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db3af9ddf4c19e3b5f84a20f4bfe51b337d970b1" target='_blank'>
              Circularization of Single-Stranded DNA Donor Template Unleashes the Power of Non-Viral Gene Delivery for Long-Term HSCs editing
              </a>
            </td>
          <td>
            Gil Letort, A. Duclert, D. L. Clerre, I. Chion-Sotinel, Roger Salvatori, Emilie Dessez, Margaux Sevin, Marco Rotondi, Cosimo Ducani, Philippe Duchateau, J. Valton
          </td>
          <td>2025-02-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Enhancer RNAs (eRNAs), a subclass of non-coding RNAs transcribed from enhancer regions, have emerged as critical regulators of gene expression; however, their functional roles in prostate cancer remain largely unexplored. In this study, we performed integrated chromatin accessibility and transcriptomic analyses using ATAC-seq and RNA-seq on twenty pairs of prostate cancer and matched benign tissues. By incorporating chromatin immunoprecipitation sequencing data, we identified a subset of differentially expressed eRNAs significantly associated with genes involved in prostate development and oncogenic signaling pathways. Among these, lactotransferrin-eRNA (LTFe) was markedly downregulated in prostate cancer tissues, with functional analyses revealing its tumor-suppressive role. Mechanistically, LTFe promotes the transcription of its target gene, lactotransferrin (LTF), by interacting with heterogeneous nuclear ribonucleoprotein F (HNRNPF) and facilitating enhancer-promoter chromatin interactions. Furthermore, we demonstrate that the LTFe-LTF axis facilitates ferroptosis by modulating iron transport. Notably, androgen receptor (AR) signaling disrupts LTFe-associated chromatin looping, leading to ferroptosis resistance. Therapeutically, co- administration of the AR inhibitor enzalutamide and the ferroptosis inducer RSL3 significantly suppressed tumor growth, offering a promising strategy for castration-resistant prostate cancer. Collectively, this study provides novel insights into the mechanistic role of eRNAs in prostate cancer, highlighting the LTFe-LTF axis as a critical epigenetic regulator and potential therapeutic target for improved treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c68bec7303a475440a96327db2c0e7d483706690" target='_blank'>
              Enhancer transcription profiling reveals an enhancer RNA-driven ferroptosis and new therapeutic opportunities in prostate cancer
              </a>
            </td>
          <td>
            Sheng Ma, Zixian Wang, Zezhong Xiong, Yue Ge, Meng-Yao Xu, Junbiao Zhang, Yuzheng Peng, Qin Zhang, Jiaxue Sun, Zirui Xi, Hao Peng, Wenjie Xu, Yanan Wang, Le Li, Chunyu Zhang, Zheng Chao, Baojun Wang, Xu Gao, Xu Zhang, Gong-Hong Wei, Zhihua Wang
          </td>
          <td>2025-03-14</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d6e9e6ea94c0b9c3a4b09ce35477edf5061c010" target='_blank'>
              RNA Transcripts Serve as a Template for Double-Strand Break Repair in Human Cells
              </a>
            </td>
          <td>
            Manisha Jalan, Alessandra Brambati, Hina Shah, Niamh McDermott, Juber Patel, Yingjie Zhu, Ahmet Doymaz, Julius Wu, Kyrie S Anderson, Andrea M Gazzo, F. Pareja, Takafumi N. Yamaguchi, Theodore Vougiouklakis, Sana Ahmed-Seghir, Philippa Steinberg, Anna Neiman-Golden, Benura Azeroglu, Joan Gomez-Aguilar, E. D. da Silva, Suleman Hussain, Daniel Higginson, P. Boutros, N. Riaz, J. Reis-Filho, Simon N. Powell, Agnel Sfeir
          </td>
          <td>2025-02-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Epithelial Ovarian Cancer (EOC) is the most lethal gynecologic cancer and those affected are in urgent need of new therapeutic strategies. Standard treatment is surgery followed by taxane- and platinum-based chemotherapy, but the rate of relapse is high and the 5-year survival is only 45%. Oncolytic viruses (OV) are a promising approach to EOC therapy through remodeling the immune composition of the tumor microenvironment (TME). Treatment response in EOC tumors can differ based on the presence of key tumorigenic mutations. This study evaluated the impact of specific tumor mutations on the response to the current standard of care carboplatin, two promising OV candidates VSV∆M51 and MG1, an infected cell vaccine (ICV-MG1) regimen, and the anti-angiogenic drug Fc3TSR. Mice with tumors harboring constitutive K-Ras activation showed an enhanced response to carboplatin and VSV∆M51 treatment. Additionally, VSV∆M51 treatment prolonged survival of syngeneic mice bearing tumors with mutations in Pten and Kras, Pten and Trp53, or Trp53 and Brca2 with increased activation of CD4+ and CD8+ T lymphocytes in the peritoneal TME. To enhance OV potency, an MG1-based infected cell vaccine inducing expression of IL-21 or IL15+21 was developed and found to enable strong and long-lasting antitumoral immunity in two carboplatin refractory syngeneic models, ID8 Trp53-/- and STOSE. VSV∆M51 combined with the anti-angiogenic Fc3TSR enhanced efficacy in the ID8 model. In summary, OV-based immunotherapy has shown promise in diverse murine models of EOC-bearing clinically relevant mutations, thus laying the foundation for developing new OV-based strategies to target a large spectrum of EOC genotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0be8c226447376238aa375a0e7bc3031f54f154" target='_blank'>
              Specific Genetic Mutations Impact Chemotherapy Resistance and Therapeutic Efficacy of Oncolytic Viruses in Ovarian Cancer.
              </a>
            </td>
          <td>
            Alison O Cudmore, G. Rodriguez, Vincent Maranda, Salar Farokhi Boroujeni, Humaira Murshed, Elizabeth A Macdonald, Melanie Grondin, K. Garson, K. Matuszewska, J. Diallo, James J Petrik, B. Vanderhyden
          </td>
          <td>2025-03-10</td>
          <td>Molecular cancer therapeutics</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f37dc921f83b90daf61b10b8456e0c63c032e710" target='_blank'>
              [Pathogenic large duplication in TP53 as a hereditary predisposing factor in breast cancer].
              </a>
            </td>
          <td>
            Viktória Kovács, H. Butz, János Papp, T. Pócza, Kornél Vince Grolmusz, Petra Nagy, Attila Patócs, A. Bozsik
          </td>
          <td>2025-03-21</td>
          <td>Magyar onkologia</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b21a0c4249364393ca01747743514c9c587e8691" target='_blank'>
              Cryptic Rearrangement of the KMT2A Gene in a B-cell Acute Lymphoblastic Leukemia.
              </a>
            </td>
          <td>
            M. Brunetti, K. Andersen, S. Spetalen, G. E. Tjønnfjord, Sverre Heim, F. Micci
          </td>
          <td>2025-03-01</td>
          <td>Anticancer research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Long noncoding RNAs (lncRNAs) have been recognized as significant modulators of gene expression and are essential for various biological functions, even though they don’t appear to have the ability to encode proteins. Originally considered dark matter, lncRNAs have been recognized as being dysregulated and contributing to the onset, progression, and resistance to treatment of acute myeloid leukemia (AML). AML is a prevalent type of leukemia characterized by the disruption of myeloid cell differentiation, leading to an increased number of immature myeloid progenitor cells. Currently, the need for novel biomarkers and treatment targets to enhance therapeutic alternatives has led to a focus on lncRNAs as possible indicators for prognostic, therapeutic, and diagnostic systems in various human cancers, including AML. Recent research has recognized a limited set of lncRNAs as possible prognostic biomarkers or diagnoses in AML. This review evaluates the key research that highlights the significance of lncRNAs in AML and discusses their roles and impacts on the disease. Furthermore, we intend to underscore the importance of lncRNAs as new and trustworthy markers for the diagnosis, prediction, drug resistance, and targets for treatment in AML. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-03763-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aba77b991995100b921ea3744ff805cd3967204" target='_blank'>
              Long noncoding RNAs in acute myeloid leukemia: biomarkers, prognostic indicators, and treatment potential
              </a>
            </td>
          <td>
            Maryam Farajzadeh, Mehrdad Fathi, Pooya Jalali, Armin Mahmoudsalehi Kheshti, Shahla Khodayari, Mohammad Hojjat-Farsangi, Farhad Jadidi
          </td>
          <td>2025-04-05</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c32ba0e230de094bf228cc062386115a9ca1917" target='_blank'>
              Integrated multi-omics analysis reveals the functional and prognostic significance of lactylation-related gene PRDX1 in breast cancer
              </a>
            </td>
          <td>
            Qinqing Wu, Heng Cao, Jiangdong Jin, Dongxu Ma, Yixiao Niu, Yanping Yu, Xiang Wang, Yiqin Xia
          </td>
          <td>2025-04-04</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Expression levels of ATP-binding cassette (ABC) transporters are known to be increased in various tumor cells, including in breast cancer, and they are responsible for mediating drug resistance, leading to treatment failure. In the present study, gene expression array analysis revealed that among ABC transporter subtypes, ABC subfamily G member 8 (ABCG8) was one of the most increased in radiotherapy-resistant triple-negative breast cancer (RT-R-TNBC) cells compared with in TNBC cells. ABCG8 is involved in sterol efflux; however, its role in cancer is not well known. Therefore, the present study investigated the effect of ABCG8 on tumor progression in RT-R-TNBC cells. Gene expression profiling was conducted using the QuantiSeq 3′ mRNA-Seq Service, followed by western blotting to confirm protein levels. Loss-of-function assays using small interfering RNA (si) transfection were performed to assess the roles of ABCG8 and its regulatory signaling pathways. RT-R-MDA-MB-231 cells exhibited increased cholesterol levels in both cells and the surrounding media via induction of sterol regulatory element binding protein 1 (mature form) and fatty acid synthase. siABCG8 transfection increased intracellular cholesterol levels but decreased cholesterol levels in the media, indicating an accumulation of cholesterol inside cells. Additionally, RT-R-MDA-MB-231 cells exhibited increased levels of β-catenin compared with MDA-MB-231 cells, which was significantly reduced by ABCG8 knockdown. Furthermore, ABCG8 knockdown led to cell cycle arrest in the G2/M phase in RT-R-MDA-MB-231 cells by reducing Polo-like kinase 1 (PLK1) and Cyclin B1 expression. RT-R-MDA-MB-231 cells also exhibited increased phosphorylated-low-density lipoprotein (LDL) receptor-related protein 6 (LRP6) levels compared with MDA-MB-231 cells, and these were decreased by siABCG8 transfection. LRP6 siRNA transfection decreased β-catenin, PLK1 and Cyclin B1 expression. In addition, feedback mechanisms such as liver X receptor and inducible degrader of LDL were decreased in RT-R-MDA-MB-231 cells under normal conditions compared with in MDA-MB-231 cells. To the best of our knowledge, the present study was the first to suggest that the cholesterol exported by ABCG8, not inside the cells, may affect cancer progression via the LRP6/Wnt/β-catenin signaling pathway in RT-R-TNBC. The regulation of this pathway may offer a potential therapeutic strategy for the treatment of RT-R-TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a057625eb93a3173c7a3b1671e3d5105799112eb" target='_blank'>
              ABCG8-mediated sterol efflux increases cancer cell progression via the LRP6/Wnt/β-catenin signaling pathway in radiotherapy-resistant MDA-MB-231 triple-negative breast cancer cells
              </a>
            </td>
          <td>
            Y. Ko, Ju Yeong Won, Hana Jin, Nam Binh Nguyen, Yaeram Won, Vedaste Nsanzimana, S. Yun, Sang Won Park, Hye Jung Kim
          </td>
          <td>2025-03-20</td>
          <td>International Journal of Molecular Medicine</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03e8bbeec84df302b9adf6fafec7b5584f0a9749" target='_blank'>
              Advancement of Gene Therapy in Cancer
              </a>
            </td>
          <td>
            Kahkashan Sultana, Ayesha Fatima
          </td>
          <td>2025-03-15</td>
          <td>Journal of Advanced Scientific Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d3959b933b4c238cc534f4bd3bbf4f10f6618c5" target='_blank'>
              Recent Advances in Anti-Aging Therapeutic Strategies Targeting DNA Damage Response and Senescence-Associated Secretory Phenotype-Linked Signaling Cascade.
              </a>
            </td>
          <td>
            Jawad Nadeem, Razia Sultana, Amna Parveen, Sun Yeou Kim
          </td>
          <td>2025-02-26</td>
          <td>Cell biochemistry and function</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The ataxia-telangiectasia mutated (ATM) protein kinase plays a critical role in activating the cellular response to DNA double-strand breaks and promoting homology-directed repair. ATM is frequently mutated in cancer, contributing to an accumulation of DNA damage that drives genomic instability. To exploit cancer cells’ inherent vulnerability to DNA damage, various small molecule inhibitors have been developed that target ATM. ATM inhibitors have shown great versatility in preclinical studies and increasing use in the clinic. Here, we review the development of ATM inhibitors and their role in cancer therapy. We describe their limitations and the advances that have led to increases in both the number and diversity of active clinical trials targeting ATM. We also discuss ATM’s role in personalized medicine and the current challenges to more widespread use of ATM inhibitors in the clinic.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c1ae4df6ed1675fe0ee6d9bf7493ed3c92eb628" target='_blank'>
              The Development of ATM Inhibitors in Cancer Therapy
              </a>
            </td>
          <td>
            Elizabeth A. Ampolini, Judit Jimenez-Sainz, David T Long
          </td>
          <td>2025-03-01</td>
          <td>Targeted Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Myelodysplastic neoplasms (MDS) are stem cell disorders characterized by ineffective hematopoiesis and risk of transformation to acute myeloid leukemia (AML). Chromosomal alterations are frequent in MDS, with interstitial deletion of chromosome 5q (del(5q)) being the most common. Lenalidomide is the current first-line treatment for del(5q) MDS and its efficacy relies on degradation of CK1α which is encoded by the CSNK1A1 gene located in the commonly deleted region (CDR) of chromosome 5q. However, lenalidomide-resistance is common, often secondary to loss-of-function mutations in TP53 or RUNX1. The CDR in del(5q) harbors several genes, including noncoding miRNAs, the loss of which contribute to disease phenotypes. miR-143 and miR-145 are located within the del(5q) CDR, but precise understanding of their role in human hematopoiesis and in the pathogenesis of del(5q) MDS is lacking. Here we provide evidence that deficiency of miR-143 and miR-145 plays a role in clonal expansion of del(5q) MDS. We show that insulin-like growth factor 1 receptor (IGF-1R) is a direct target of both miR-143 and miR-145. Our data demonstrate that IGF-1R inhibition reduces proliferation and viability of del(5q) cells in vitro and in vivo, and that lenalidomide-resistant del(5q) MDS cells depleted of either TP53 or RUNX1 are sensitive to IGF-1R inhibition. Resistant del(5q) MDS-L cells, as well as primary MDS marrow cells, are also sensitive to targeting of IGF-1R-related dependencies in del(5q) MDS, which include the Abl and MAPK signaling pathways. This work thus provides potential new therapeutic avenues for lenalidomide-resistant del(5q) MDS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9bf67f74cd301ce09a925c27d079752c62cf556b" target='_blank'>
              Haploinsufficiency of miR-143 and miR-145 reveal targetable dependencies in resistant del(5q) myelodysplastic neoplasm
              </a>
            </td>
          <td>
            Nadia Gharaee, J. Wegrzyn-Woltosz, Jihong Jiang, Vijay Suresh Akhade, Josh Bridgers, Ryan J. Stubbins, D. Hiwase, M. Kutyna, O. Chan, R. Komrokji, Eric Padron, Yu Deng, Gary Cole, P. Umlandt, Megan E Fuller, Ada Kim, Aly Karsan
          </td>
          <td>2025-02-25</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Background Outcomes under anti-PD-(L)1 therapy have been variable in advanced non-small cell lung cancer (NSCLC) without reliable predictive biomarkers so far. Targeted next-generation sequencing (NGS) of circulating tumor DNA (ctDNA) has demonstrated potential clinical utility to support clinical decisions, but requires prior tumor genetic profiling for proper interpretation, and wide adoption remains limited due to high costs. Methods Tumor-agnostic low-coverage ctDNA whole genome sequencing (lcWGS) was used to longitudinally track genome-wide copy number variations (CNVs) and fragmentation features in advanced NSCLC patients (n = 118 samples from 49 patients) and healthy controls (n = 57). Tumor PD-L1 expression was available for comparison. Findings Fragmentation features and CNVs were complementary indicators, whose combination significantly increased ctDNA detection compared to single-marker assessments (+ 20.3% compared to CNV analysis alone). Baseline fragment length alterations, but not CNVs, were significantly associated with subsequent progression-free survival (PFS; hazard ratio [HR] = 4.10, p = 6.58e-05) and could improve PFS predictions based on tumor PD-L1 expression alone (HR = 2.70, p = 0.019). Residual CNVs or aberrant fragmentation of ctDNA under ongoing therapy could stratify patients according to the subsequent response duration (median 5.8 vs. 47.0 months, p = 1.13e-06). The integrative analysis of ctDNA fragment characteristics at baseline, tumor PD-L1 expression, and residual ctDNA under ongoing treatment constituted the strongest independent predictor of PFS (p = 6.25e-05) and overall survival (p = 1.3e-03) in multivariable analyses along with other clinicopathologic variables. Interpretation This study demonstrates the feasibility and potential clinical utility of lcWGS for the tumor-agnostic stratification and monitoring of advanced NSCLC under PD-(L)1 blockade based on CNV and fragmentomic profiling. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03348-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/263d81db230c34f72ebf729f4d1766f8b109dae5" target='_blank'>
              Low-coverage whole genome sequencing of cell-free DNA to predict and track immunotherapy response in advanced non-small cell lung cancer
              </a>
            </td>
          <td>
            F. Janke, Mateo Gasser, A. Angeles, A. Riediger, Magdalena Görtz, Louise Appenheimer, Astrid K. Laut, Simon J. Ogrodnik, Sabrina Gerhardt, A. Stenzinger, M. Schneider, M. Thomas, P. Christopoulos, Holger Sültmann
          </td>
          <td>2025-03-08</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Abstract The current treatment of triple‐negative breast cancer (TNBC) is still primarily based on platinum‐based chemotherapy. However, TNBC cells frequently develop resistance to platinum and experience relapse after drug withdrawal. It is crucial to specifically target and eliminate cisplatin‐tolerant cells after platinum administration. Here, it is reported that upregulated N 6‐methyladenosine (m6A) modification drives the development of resistance in TNBC cells during cisplatin treatment. Mechanistically, histone deacetylase 2 (HDAC2) mediates delactylation of methyltransferase‐like 3 (METTL3), facilitating METTL3 interaction with Wilms’‐tumor‐1‐associated protein and subsequently increasing m6A of transcript‐associated DNA damage repair. This ultimately promotes cell survival under cisplatin. Furthermore, pharmacological inhibition of HDAC2 using Tucidinostat can enhance the sensitivity of TNBC cells to cisplatin therapy. This study not only elucidates the biological function of lactylated METTL3 in tumor cells but also highlights its negative regulatory effect on cisplatin resistance. Additionally, it underscores the nonclassical functional mechanism of Tucidinostat as a HDAC inhibitor for improving the efficacy of cisplatin against TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c5ff71ca33fdaff595792bb621bbec0ae0038c4" target='_blank'>
              HDAC2‐Mediated METTL3 Delactylation Promotes DNA Damage Repair and Chemotherapy Resistance in Triple‐Negative Breast Cancer
              </a>
            </td>
          <td>
            Xiaoniu He, Yuanpei Li, Jian Li, Yu Li, Sijie Chen, Xia Yan, Zhangrong Xie, Jiangfeng Du, Guoan Chen, Jianbo Song, Qi Mei
          </td>
          <td>2025-02-14</td>
          <td>Advanced Science</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="Most genetic variants associated with complex traits and diseases occur in non-coding genomic regions and are hypothesized to regulate gene expression. To understand the genetics underlying gene expression variability, we characterize 14,324 ancestrally diverse RNA-sequencing samples from the NHLBI Trans-Omics for Precision Medicine (TOPMed) program and integrate whole genome sequencing data to perform cis and trans expression and splicing quantitative trait locus (cis-/trans-e/sQTL) analyses in six tissues and cell types, most notably whole blood (N=6,454) and lung (N=1,291). We show this dataset enables greater detection of secondary cis-e/sQTL signals than was achieved in previous studies, and that secondary cis-eQTL and primary trans-eQTL signal discovery is not saturated even though eGene discovery is. Most TOPMed trans-eQTL signals colocalize with cis-e/sQTL signals, suggesting many trans signals are mediated by cis signals. We fine-map European UK BioBank GWAS signals from 164 traits and colocalize the resulting 34,107 fine-mapped GWAS signals with TOPMed e/sQTL signals, finding that of 10,611 GWAS signals with a colocalization, 7,096 GWAS signals colocalize with at least one secondary e/sQTL signal. These results demonstrate that larger e/sQTL analyses will continue to uncover secondary e/sQTL signals, and that these new signals will benefit GWAS interpretation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93d4d05c38b07b5509f977fc72067af450ca02ed" target='_blank'>
              Cross-cohort analysis of expression and splicing quantitative trait loci in TOPMed
              </a>
            </td>
          <td>
            Peter Orchard, T. Blackwell, L. Kachuri, Peter J. Castaldi, M. Cho, S. Christenson, P. Durda, Stacey B. Gabriel, C. Hersh, S. Huntsman, Seungyong Hwang, R. Joehanes, Mari Johnson, Xingnan Li, Honghuang Lin, Ching-Ti Liu, Yongmei Liu, Angel C. Y. Mak, A. Manichaikul, David T. Paik, A. Saferali, Joshua D. Smith, K. Taylor, Russell P. Tracy, Jiongming Wang, Mingqiang Wang, J. Weinstock, Jeffrey Weiss, Heather E. Wheeler, Ying Zhou, S. Zoellner, Joseph Wu, Luisa Mestroni, S. Graw, Matthew Taylor, Victor E. Ortega, Craig W Johnson, Weiniu Gan, Gonçalo R. Abecasis, Deborah A. Nickerson, Namrata Gupta, K. Ardlie, Prescott G. Woodruff, Yinan Zheng, Russell P. Bowler, D. Meyers, Alexander P Reiner, C. Kooperberg, E. Ziv, Vasan S Ramachandran, Martin G Larson, L. Cupples, E. G. Burchard, Edwin K. Silverman, Steve Rich, N. Heard-Costa, Hua Tang, Jerome I. Rotter, Albert V Smith, Daniel Levy, F. Aguet, Laura Scott, L. Raffield, Stephen C. J. Parker
          </td>
          <td>2025-02-21</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Objectives/Goals: Investigating the B-cell acute lymphoblastic leukemia (B-ALL)-associated germline SNP rs7090445, located in intron 3 of ARID5B, which is more frequently observed in individuals of Hispanic/Latino descent. Investigating the mechanisms behind this inherited single nucleotide polymorphisms (SNP) that may contribute to the higher incidence of B-ALL in this population. Methods/Study Population: Specific Aim 1: We hypothesize ARID5B SNP rs7090445 disrupts intrinsic enhancer function. Identification of critical DNA looping events impacted by ARID5B variants using Capture C. Affinity purification-mass spectrometry to identify potential ARID5B transcription mediators. Specific Aim 2: We hypothesize the B-ALL-associated SNP leads to a partial human B-cell differentiation block. Utilize Cas9-mediated homology-directed repair to create ARID5B SNP in primary human hematopoietic stem cells. Gene-edited HSCs will be differentiated into B cells using an ex vivo system. Fluorescence-activated cell sorting to sort our pool of cells into stages of B-cell development spectrum. Amplicon sequencing and variant allele frequency of rs7090445 SNP to evaluate its impact on B-cell development. Results/Anticipated Results: This proposal is conceptually innovative as it seeks to understand the mechanism by which the B-ALL-associated SNP rs7090445 in intron 3 of ARID5B disrupts enhancer function and investigates its impact on human B-cell development. Future research will investigate a tumor-suppressive role of ARID5B and whether it constitutes a “first-hit” of leukemogenesis. Discussion/Significance of Impact: Successful completion of this research will elucidate the critical role of the B-ALL-associated ARID5B SNP rs7090445 in human B-cell development and leukemogenesis. As this SNP is more prevalent in Hispanic/Latino populations, it will also provide crucial insights into the genetic factors behind the elevated incidence of B-ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4345d4a2e4bf5156f2152b887906b55727b5a5a1" target='_blank'>
              71 Investigating gene regulatory mechanisms associated with B-cell acute lymphoblastic leukemia incidence in Hispanic/Latino populations
              </a>
            </td>
          <td>
            Clarissa Garcia, Julian Grandvallet, Illaria Iacobucci, Jun Yang, C. Mullighan, Tzu Phang, Matthew T. Witkowski
          </td>
          <td>2025-03-26</td>
          <td>Journal of Clinical and Translational Science</td>
          <td>0</td>
          <td>103</td>
        </tr>

        <tr id="Tumorigenesis in mammals is driven by inherited genetic variants, environmental factors and random errors during normal DNA replication that lead to cancer-causing mutations. These factors initiate uncontrolled cellular proliferation and disrupt the regulation of critical checkpoints. A few mammalian species possess unique protective mechanisms that enable them to resist widespread cancer development and achieve longevity. Tissue-specific tumor protection adds another layer of complexity to this diversity. Breast cancer is a leading cause of human mortality, particularly among females. Driven by the need for new strategies in treatment and prevention, this opinion article explores and supports the idea that herbivores are more resistant to mammary cancer than carnivores and omnivores. This diversity has occurred despite the remarkably similar basic mammary biology. Herbivores’ meatless diet cannot explain the differences in cancer resistance, which have accompanied species segregation since the Jurassic era. To investigate the causes of this diversity, the characteristics of tumorigenesis in the human breast—and to a lesser extent in other carnivores—have been compared with data from retrospective analyses of bovine mammary tumor development across various locations over the past century. Well-established genomic, cellular, and systemic triggers of breast cancer exhibit different, or less pronounced tissue-specific activity in the bovine mammary gland, accompanied by novel bovine-specific protective mechanisms. Together, these factors contribute to the near absence of breast cancer in bovines and offer a basis for developing future anticancer strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0cb9fafc9e4ae8646c09de8af8d088ea1242ab7" target='_blank'>
              Mammalian Species-Specific Resistance to Mammary Cancer
              </a>
            </td>
          <td>
            I. Barash
          </td>
          <td>2025-03-06</td>
          <td>Journal of Mammary Gland Biology and Neoplasia</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0687d954e3a7ce3f1c8da610c5b175b62bd0c491" target='_blank'>
              Single-cell multiomics reveals a gene regulatory circuit driving leukemia cell differentiation.
              </a>
            </td>
          <td>
            Xin Tian, Liuqingqing Zhang, Guiqiyang Xiang, Yijia Tang, Ping Zhu, Shuting Yu, Fangying Jiang, Shuai Wang, Jinzeng Wang, Yao Dai, Desheng Zheng, Jianbiao Wang, Xiangqin Weng, Shengyue Wang, Yun Tan, Feng Liu
          </td>
          <td>2025-02-22</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background TP53 mutations are linked to aggressive progression and chemoresistance in gastric cancer (GC). Frameshift mutation is the second most common mutation type of TP53. However, the consequences of this mutation type in GC were not well understood, and targeted therapies for cancer patients harboring frameshift mutations were also not established. Histone methylation significantly influences tumorigenesis in TP53-mutated cancers, and related inhibitors are emerging as specific therapeutic strategies. Methods and results By treating GC cell lines harboring various TP53 mutation types with a library of histone demethylase inhibitors, we identified that GSK690, a reversible inhibitor of lysine-specific demethylase 1 (LSD1), selectively inhibits GC cells harboring TP53 frameshift mutations without nuclear localization sequence (NLS) (termed TP53 Frameshift NLS), which accounts for 89% TP53 frameshift mutations in GC patients. GSK690 showed significant specific inhibition in vitro and in vivo against this subtype by inducing G1/S cell cycle arrest via the LSD1-CCNA2 axis. Importantly, dual-luciferase assays and ChIP-qPCR confirmed that the loss of transcriptional repression activities of p53 in drives LSD1 upregulation in TP53 Frameshift NLS cancer cells. Conclusions In summary, our results indicate that the nuclear localization deficiency of p53 accounts for increased expression of LSD1 in TP53 Frameshift NLS GCs. GSK690 inhibits cell cycle progression and tumor growth by suppressing aberrantly activated LSD1-CCNA2 signaling in this GC subtype, counteracting malignant proliferation and thereby providing a precise therapeutic strategy for GC patients with TP53 Frameshift NLS. Supplementary Information The online version contains supplementary material available at 10.1186/s13148-025-01829-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1998881bbe977bcf7672e918b2a1b2bf09e4b12e" target='_blank'>
              LSD1 is a targetable vulnerability in gastric cancer harboring TP53 frameshift mutations
              </a>
            </td>
          <td>
            Suzeng Wang, Chunyu Yang, Junhui Tang, Kaiqing Wang, Hao Cheng, Surui Yao, Zhaohui Huang, Bo-jian Fei
          </td>
          <td>2025-02-18</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Abstract Oncogene activation in normal untransformed cells induces DNA replication stress and creates a dependency on DNA damage response (DDR) mechanisms for cell survival. Different oncogenic stimuli signal via distinct mechanisms in every cancer setting. The DDR is also pathologically reprogrammed and deployed in diverse ways in different cancers. Because mutant KRAS is the driver oncogene in 90% of pancreatic ductal adenocarcinomas (PDACs), here we have investigated DDR mechanisms by which KRAS-induced DNA replication stress is tolerated in normal human pancreatic epithelial cells [human pancreatic nestin-expressing (HPNE) cells]. Using a candidate screening approach, we identify TRIP13 as a KRASG12V-induced messenger RNA that is also expressed at high levels in PDAC relative to normal tissues. Using genetic and pharmacological tools, we show that TRIP13 is necessary to sustain ongoing DNA synthesis and viability specifically in KRASG12V-expressing cells. TRIP13 promotes survival of KRASG12V-expressing HPNE cells in a homologous recombination (HR)-dependent manner. KRASG12V-expressing HPNE cells lacking TRIP13 acquire hallmark HR deficiency phenotypes, including sensitivity to inhibitors of translesion synthesis and poly-ADP ribose polymerase. Established PDAC cell lines are also sensitized to intrinsic DNA damage and therapy-induced genotoxicity following TRIP13 depletion. Taken together, our results expose TRIP13 as an attractive new and therapeutically tractable vulnerability of KRAS-mutant PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c52be837d7841e9eb90ef56e65d2eb6df322017a" target='_blank'>
              TRIP13 protects pancreatic cancer cells against intrinsic and therapy-induced DNA replication stress
              </a>
            </td>
          <td>
            J. R. Anand, G. Droby, Sayali Joseph, Urvi Patel, Xingyuan Zhang, Jeffrey A. Klomp, C. Der, Jeremy E. Purvis, Samuel C. Wolff, Jessica Bowser, Cyrus Vaziri
          </td>
          <td>2025-03-01</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>122</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb7e063d7f27e7ff5ba777247b0efc2ec0bcfdb4" target='_blank'>
              Functional analysis of cancer-associated germline risk variants.
              </a>
            </td>
          <td>
            Laura N. Kellman, Poornima H. Neela, Suhas Srinivasan, Z. Siprashvili, Ronald L. Shanderson, Audrey W. Hong, Deepti S Rao, Douglas F Porter, D. Reynolds, Robin M. Meyers, M. Guo, Xue Yang, Yang Zhao, Glenn G Wozniak, L. Donohue, R. Shenoy, Lisa A Ko, Duy T. Nguyen, Smarajit Mondal, Omar S. Garcia, L. Elcavage, Ibtihal Elfaki, Nathan S. Abell, Shiying Tao, Christopher M Lopez, Stephen B. Montgomery, Paul A. Khavari
          </td>
          <td>2025-02-17</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a58a37d56520c02804366c8628d09a8d85713cd6" target='_blank'>
              Acute high-dose irradiation of human primary ovarian cells reveals a shift in transcriptomics profile and impairment in cell-cell adhesion
              </a>
            </td>
          <td>
            S. P. Deligiannis, Tianyi Li, Elisabeth Moussaud-Lamodière, Á. Végvári, A. Damdimopoulos, D. Lavogina, K. Papaikonomou, Roman Zubarev, Ganesh Acharya, A. Velthut-Meikas, P. Damdimopoulou, A. Salumets, Valentina Di Nisio
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Endogenous retroviruses (ERVs) shape human genome functionality and influence disease pathogenesis, including cancer. ERVK-7, a significant ERV, acts as an immune modulator and prognostic marker in lung adenocarcinoma (LUAD). Although ERVK-7 overexpression has been linked to the amplification of the 1q22 locus in approximately 10% of LUAD cases, it predominantly arises from alternative regulatory mechanisms. Our findings indicate that the canonical 5’ long terminal repeat (LTR) of ERVK-7 is methylated and inactive, necessitating the use of alternative upstream promoters. We identified two novel transcripts, ERVK-7.long and ERVK-7.short, arising from distinct promoters located 2.8 kb and 13.8 kb upstream of the 5’LTR of ERVK-7, respectively. ERVK-7.long is predominantly overexpressed in LUAD. Through comprehensive epigenetic mapping and single-cell transcriptomics, we demonstrate that ERVK-7.long activation is predetermined by cell lineage, specifically in small airway epithelial cells (SAECs), where its promoter displays tumor-specific H3K4me3 modifications. Single-cell RNA sequencing further reveals a distinct enrichment of ERVK-7.long in LUAD tumor cells and alveolar type 2 epithelial cells, underscoring a cell-type-specific origin. Additionally, inflammatory signaling significantly influences ERVK-7 expression; TNF-α enhances ERVK-7.long, while interferon signaling preferentially augments ERVK-7.short by differential recruitment of NF-κB/RELA and IRF to their respective promoters. This differential regulation clarifies the elevated ERVK-7 expression in LUAD compared to lung squamous cell carcinoma (LUSC). Our study elucidates the complex regulatory mechanisms governing ERVK-7 in LUAD and proposes these transcripts as potential biomarkers and therapeutic targets, offering new avenues to improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f86f3682460f118255d9b277164814fad089c123" target='_blank'>
              A tumor necrosis factor-α responsive cryptic promoter drives overexpression of the human endogenous retrovirus ERVK-7
              </a>
            </td>
          <td>
            Sojung Lee, Yin Yee Ho, Suyu Hao, Yingqi Ouyang, Ling Liew, Ashish Goyal, Stephen Li, J. Barbour, Mu He, Yuanhua Huang, J. W. Wong
          </td>
          <td>2025-03-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb7435e798afc5c276c5bcf5f266a5cdfa975999" target='_blank'>
              Abstract B008: Targeting DNA damage responsive pathways in cancer therapy
              </a>
            </td>
          <td>
            Junjie Chen
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Ovarian cancer remains one of the most lethal gynecological malignancies owing to its high recurrence rate and chemotherapeutic resistance. MYC is a well-known proto-oncogene that is frequently amplified in ovarian cancer and has been implicated in drug resistance. Previously, we established a new promoter–reporter system combined with a CRISPR activation library to identify unknown MYC regulators, and M1AP was identified as a novel MYC regulator. However, considering the insufficient explanation for the absence of guide RNA (gRNA) of MYC, this present study explored methods to prevent the gRNA of MYC itself from binding. This study first modified the promoter–reporter vector to improve its quality, then conducted CRISPR screening and analyzed candidate genes as MYC promoter regulators using next-generation sequencing in OVSAHO ovarian cancer cells. Eighty-six genes had ≥ 1000 reads, and Pearson’s correlation coefficient analysis was performed on the cBioPortal of the Cancer Genomics database. Fourteen genes were identified as candidate MYC regulators with positive and significant correlations with MYC. Seven genes, including CYP4v2, ASPH, ANP32D, PCED1A, ABI1, FUZ, and HOOK2, demonstrated significantly higher luciferase activity than the control genes. Four genes, including ABI1, PCED1A, HOOK2, and CYP4v2, activated the MYC promoter, which showed over twofold higher activity than the control when overexpressed using a vector. In conclusion, four genes that activate MYC promoters were identified in an ovarian cancer cell line using the CRISPR library system with a modified promoter–reporter tool. These results will prove helpful in the development of novel treatment strategies for ovarian cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s13048-025-01644-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9220d2085ff0b29849fe6d91dbca5cdaecc789b" target='_blank'>
              Modified screening of MYC promotor region elements using the CRISPR library in ovarian cancer
              </a>
            </td>
          <td>
            Akiko Yamamoto, Yosuke Tanaka, S. Ishibashi, Masumi Ikeda, Keisuke Sugita, Masanori Ono, Hirotaka Nishi, Morito Kurata
          </td>
          <td>2025-04-02</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a71da6481387936a0712c7963a43d9bba36fd469" target='_blank'>
              Defeating lethal cancer: Interrupting the ecologic and evolutionary basis of death from malignancy.
              </a>
            </td>
          <td>
            Kenneth J. Pienta, Patrick L Goodin, S. Amend
          </td>
          <td>2025-03-09</td>
          <td>CA: a cancer journal for clinicians</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee23eae46a9ddd2433ad4acafc75ff3551fa2d98" target='_blank'>
              ANGEL2 modulates wildtype TP53 translation and doxorubicin chemosensitivity in colon cancer.
              </a>
            </td>
          <td>
            C. A. Lucchesi, S. Mantrala, Darren Tran, N. Batra, Avani Durve, Conner Suen, Jin Zhang, Paramita M Ghosh, Xinbin Chen
          </td>
          <td>2025-03-07</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Cancers that retain wild type TP53 presumably harbor other clonal alterations that permitted their precursors to bypass p53-mediated growth suppression. Consequently, studies that employ TP53-wild type cancer cells and their isogenic derivatives may systematically fail to appreciate the full scope of p53 functionality. Several TP53 phenotypes are known to be absent in the widely used isogenic HCT116 colorectal cancer (CRC) model, which originated from a tumor that had retained wild type TP53. In contrast, we show that restoration of p53 in the TP53-mutant CRC cell line DLD-1 impeded cell proliferation, increased levels of senescence and sensitized cells to ionizing radiation (IR). To study p53 in a non-cancer context, we disrupted TP53 in hTERT-RPE1 cells. Derived from primary cells that were immortalized in vitro, hTERT-RPE1 expressed striking p53-dependent phenotypes and appeared to select for p53 loss during routine culture. hTERT-RPE1 expressed a p53-responsive transcriptome that was highly representative of diverse experimental systems. We discovered several novel downstream p53 targets of potential clinical relevance including ALDH3A1, which is involved in the detoxification of aldehydes and the metabolism of reactive oxygen species, and nectin cell adhesion molecule 4 (NECTIN4) which encodes a secreted surface protein that is overexpressed in many tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56bfbd25a9a6de3dee6da6ae82f799aa171d1483" target='_blank'>
              Robust p53 phenotypes and prospective downstream targets in telomerase-immortalized human cells
              </a>
            </td>
          <td>
            Jessica J. Miciak, Lucy Petrova, Rhythm Sajwan, Aditya Pandya, Mikayla Deckard, Andrew J Munoz, Fred Bunz
          </td>
          <td>2025-02-18</td>
          <td>Oncotarget</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9c07b4e618fbc35eed479906ebc22d62dccc859" target='_blank'>
              496 Multi-Omic Profiling of Epigenetic Heterogeneity in Glioblastoma at Single-Cell Resolution Using MscCUT&TAG
              </a>
            </td>
          <td>
            Andrew Hardigan, E. Blandford, Aditya Mohan, Eliese Moelker, Steven Shen, P. Fecci, A. Patel
          </td>
          <td>2025-04-01</td>
          <td>Neurosurgery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Genomic screens and GWAS are powerful tools for identifying disease-modifying genes, but it is often challenging to understand the pathways by which these genes function. Here, we take an integrated approach that combines network analysis and an imaging-based pooled genetic perturbation study to examine modifiers of Huntington’s disease (HD). The computational analysis highlighted several genes in a subnetwork enriched for modifiers of neuronal development and morphology. To test the functional roles of these genes, we developed an experimental pipeline that allows pooled CRISPRi KD of 21 genes in human iPSC-derived neurons followed by optical analysis of genotypes, neuronal arborization, multiplexed pathway activity and morphological fingerprint readout. This approach recovered known genes involved in morphology and confirmed unexpected links from the network between several genetic modifiers of HD and morphology. Our approach overcomes challenges in pooled measurement of neuronal function and health and could be adapted for other phenotypes in HD and other neurological diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0647b0a177080271b9c6a93015c279a2d9cdfa6" target='_blank'>
              CellFIE: Integrating Pathway Discovery With Pooled Profiling of Perturbations Uncovers Pathways of Huntington’s Disease, Including Genetic Modifiers of Neuronal Development and Morphology
              </a>
            </td>
          <td>
            Byunguk Kang, Michael Murphy, Christopher W. Ng, M. J. Leventhal, Nhan Huynh, Egun Im, Serwah Danquah, D E Housman, R. Nehme, Samouil L. Farhi, Ernest Fraenkel
          </td>
          <td>2025-02-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89e38976a47d12226a011902bfacb13e2fb36e59" target='_blank'>
              Genome-wide CRISPR screen in human T cells reveals regulators of FOXP3.
              </a>
            </td>
          <td>
            Kelvin Y. Chen, Tatsuya Kibayashi, Ambre Giguelay, Mayu Hata, Shunsuke Nakajima, N. Mikami, Yusuke Takeshima, Kenji Ichiyama, Ryusuke Omiya, Leif S. Ludwig, Kunihiro Hattori, Shimon Sakaguchi
          </td>
          <td>2025-03-26</td>
          <td>Nature</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="CAR-T cell therapies are revolutionizing the treatment of refractory and relapsed haematological malignancies, but many patients do not exhibit long-term responses, and these therapies are less effective against solid tumours. Poor persistence of CAR-T cells in patients is associated with therapeutic failure, highlighting the need to identify genes that promote in vivo expansion. Here, we developed an accessible in vivo competitive screening method to identify genes whose inactivation provides a selective advantage to CAR-T cells. Inactivation of 50 genes in a heterogeneous population of T cells expressing an EGFR-targeting CAR revealed that disruption of REGNASE-1, SOCS1, PTPN2, and P16INK4A conferred a selective advantage to CAR-T cells in human lung tumour-bearing mice. Coherently, inactivation of these genes resulted in improved tumour eradication by CAR-T cells. Interestingly disruption of other genes, described to improve CAR-T cell function against other tumours, had a negative impact in this orthotropic lung tumour model. Further evaluation of late effects, within a subcutaneous model, highlighted SOCS1 gene inactivation as the most promising strategy for in vivo CAR-T cell persistence. These results support the importance of evaluating CAR-T cell editing strategies in specific tumours models and the screening method proposed provides a versatile pre-clinical tool for assessing persistence and function in context-specific settings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d302a39d274058dcb02936cf79573cbdf1bc9133" target='_blank'>
              An in vivo CRISPR screen unveils top target genes to efficiently improve CAR-T cell persistence in a solid tumour model of lung adenocarcinoma
              </a>
            </td>
          <td>
            Mattia Fumagalli, Dongjie An, Luca Simula, Camille Combe, Lisa Aziez, Yannick Simoni, M. Alves-Guerra, Andrea Valentini, Maude Marchais, Anaïs Vermare, Nadège Bercovici, E. Donnadieu, Frédéric Pendino
          </td>
          <td>2025-04-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/daf61c5fa6e9e754aae5ca89efec749dc7047206" target='_blank'>
              Pathway Coessentiality Mapping Reveals Complex II is Required for de novo Purine Biosynthesis in Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Amy E. Stewart, Derek K. Zachman, Pol Castellano-Escuder, Lois M. Kelly, Ben Zolyomi, Michael D.I. Aiduk, Christopher D. Delaney, Ian C. Lock, Claudie Bosc, John Bradley, Shane T. Killarney, Olga R. Ilkayeva, Christopher B. Newgard, Navdeep S. Chandel, Alexandre Puissant, Kris C. Wood, Matthey D. Hirschey
          </td>
          <td>2025-03-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Immunotherapy, particularly immune checkpoint inhibitor therapy, has demonstrated clinical benefits in solid tumours. Despite its satisfactory clinical efficacy, it still faces several issues, such as limited eligibility, low response rates and cytotoxicity. Cancer epigenetics implies that tumour cells exhibit unique phenotypes because of their unique characteristics, thus reprogramming of the epigenome holds promise for cancer therapy. Epigenetic regulation plays an important role in regulating gene expression during tumour development and maintenance. Epigenetic regulators induce cancer cell cycle arrest, apoptosis and differentiation of cancer cells, thereby exerting anti-tumour effects. Recent studies have revealed a significant correlation between epigenetic regulatory factors and immune checkpoint therapy. Epigenetics can modulate various aspects of the tumour immune microenvironment and immune response to enhance the sensitivity of immunotherapy, such as lowering the concentration required and mitigating cytotoxicity. This review primarily discusses DNA methyltransferase inhibitors, histone deacetylase inhibitors, enhancer of zeste homolog 2 inhibitors and lysine-specific demethylase 1 inhibitors, which are associated with transcriptional repression. This repression alters the expression of genes involved in the immune checkpoint, thereby enhancing the effectiveness of immunotherapy. We also discuss the potential and challenges of tumour immunotherapy and highlight its advantages, application challenges and clinical research on integrating epigenetic regulatory factors with tumour immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ded24f4b8584549c62a69b47b4fbedd7863bbb9" target='_blank'>
              Epigenetic regulators combined with tumour immunotherapy: current status and perspectives
              </a>
            </td>
          <td>
            Huan Zhang, Yutong Pang, L. Yi, Xiao-jue Wang, Panjian Wei, Haichao Wang, Shuye Lin
          </td>
          <td>2025-03-21</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="X-chromosome inactivation (XCI) is a fundamental mechanism in placental mammals that compensates for gene dosage differences between sexes. Using methylation levels of genes under XCI, we establish defective levels of XCI as a new source of interindividual variation among cancer types in females, characterized by a significant and consistent lowering of XIST expression and enrichment of differentially expressed genes under XCI. We show that defective XCI is an additive factor to the cancer risk of XCI escape deregulation in women. Defective XCI of more than 10% has an attributable risk of 40% among 12 different cancers from The Cancer Genome Atlas. Validations between independent studies of breast cancer samples show that defective XCI increases triple-negative subtype frequency, decreases survival rates, and is reduced by chemotherapy treatment. Mechanistically, it is associated with somatic mutations at TP53 and top MYC gains. In independent studies, defective XCI is detectable in blood and increases with aging, menopause, and cancer diagnosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/303ef18264231a4b1a297a874c3c158036c417de" target='_blank'>
              Defective X-chromosome inactivation and cancer risk in women
              </a>
            </td>
          <td>
            A. Cáceres, L. Pérez-Jurado, Albert Alegret-García, Varun B. Dwaraka, Ryan Smith, Juan R. González
          </td>
          <td>2025-02-22</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6eff96c5b33b1fd18b969ab18137992e335a5be4" target='_blank'>
              CRISPR/Cas9-mediated elimination of the LMNA c.745C>G pathogenic mutation enhances survival and cardiac function in LMNA-associated congenital muscular dystrophy
              </a>
            </td>
          <td>
            Déborah Gómez-Domínguez, Carolina Epifano, Iván Hernández, Borja Vilaplana-Martí, Sergi César, Antonio de Molina-Iracheta, Miguel Sena-Esteves, G. Sarquella-Brugada, Ignacio Pérez de Castro
          </td>
          <td>2025-02-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c19557776012bc520b8b5ecf4614f5b4aaa6f641" target='_blank'>
              The transcription factor SRF regulates MERVL retrotransposons and gene expression during zygotic genome activation.
              </a>
            </td>
          <td>
            C. Hermant, Carlos Michel Mourra-Díaz, Marlies E. Oomen, Luis Altamirano-Pacheco, Mrinmoy Pal, Tsunetoshi Nakatani, M. Torres-Padilla
          </td>
          <td>2025-02-27</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Abstract Background Chemoresistance remains a major hurdle in ovarian cancer (OC) treatment, as many patients eventually develop resistance to platinum‐based chemotherapy and/or PARP inhibitors (PARPi). Methods We performed transcriptome‐wide analysis by RNA sequencing (RNA‐seq) data of platinum‐resistant and ‐sensitive OC tissues. We demonstrated the role of LINC02776 in platinum resistance in OC cells, mice models and patient‐derived organoid (PDO) models. Results We identify the long noncoding RNA LINC02776 as a critical factor of platinum resistance. Elevated expression of LINC02776 is observed in platinum‐resistant OC and serves as an independent prognostic factor for OC patients. Functionally, silencing LINC02776 reduces proliferation and DNA damage repair in OC cells, thereby enhancing sensitivity to platinum and PARPi in both xenograft mouse models and patient‐derived organoid (PDO) models with acquired chemoresistance. Mechanistically, LINC02776 binds to the catalytic domain of poly (ADP‐ribose) polymerase 1 (PARP1), promoting PARP1‐dependent polyADP‐ribosylation (PARylation) and facilitating homologous recombination (HR) restoration. Additionally, high HIF‐1α expression in platinum‐resistant tissues further stimulates LINC02776 transcription. Conclusions Our findings suggest that targeting LINC02776 represents a promising therapeutic strategy for OC patients who have developed resistance to platinum or PARPi. Key points LINC02776 promotes OC cell proliferation by regulating DNA damage and apoptosis signaling pathways. LINC02776 binds PARP1 to promote DNA damage‐triggered PARylation in OC cells. LINC02776 mediates cisplatin and olaparib resistance in OC cells by enhancing PARP1‐mediated PARylation activity and regulating the PARP1‐mediated HR pathway. The high expression of LINC02776 is induced by HIF‐1α in platinum‐resistant OC cells and tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f154627a6e9fbb35a9c2baccde3a2a62d9723cc8" target='_blank'>
              HIF‐1α‐induced long noncoding RNA LINC02776 promotes drug resistance of ovarian cancer by increasing polyADP‐ribosylation
              </a>
            </td>
          <td>
            Yangjun Wu, Yu Zeng, Yong Wu, Xinyu Ha, Zheng Feng, Chaohua Liu, Ziqi Liu, Jiajia Wang, X. Ju, Shenglin Huang, Linhui Liang, Bin Zheng, Lulu Yang, Jun Wang, Xiaohua Wu, Shengli Li, H. Wen
          </td>
          <td>2025-03-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cdcaafe55411db23c30a2e5da0f6e25b45d3a4b" target='_blank'>
              Pan-Cancer Analysis Reveals SKA3 as a Potential Diagnostic and Prognostic Biomarker.
              </a>
            </td>
          <td>
            Chunlin Li, Min Gao, Hua Huang, Nashunbayaer Zha, Gang Guo
          </td>
          <td>2025-02-25</td>
          <td>APMIS : acta pathologica, microbiologica, et immunologica Scandinavica</td>
          <td>0</td>
          <td>3</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [1, 20],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>