<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-12-08 09:37:05 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="DHFR gene amplification is commonly present in methotrexate (MTX)-resistant colon cancer cells and acute lymphoblastic leukemia. In this study, we proposed an integrative framework to characterize the amplified region by using a combination of single-molecule real-time sequencing, next-generation optical mapping, and chromosome conformation capture (Hi-C). We identified an amplification unit spanning 11 genes, from the DHFR gene to the ATP6AP1L gene position, with high adjusted interaction frequencies on chromosome 5 (~2.2 Mbp) and a twenty-fold tandemly amplified region, and novel inversions at the start and end positions of the amplified region as well as frameshift insertions in most of the MSH and MLH genes were detected. These mutations might stimulate chromosomal breakage and cause the dysregulation of mismatch repair. Characterizing the tandem gene-amplified unit may be critical for identifying the mechanisms that trigger genomic rearrangements. These findings may provide new insight into the mechanisms underlying the amplification process and the evolution of drug resistance. Sequencing a large region of DNA containing many surplus copies of genes linked to drug resistance in colon cancer cells may illuminate how these genomic rearrangements arise. Such regions of gene amplification are highly repetitive, making them impossible to sequence using ordinary methods, and little is known about how they are generated. Using advanced methods, Jeong-Sun Seo at Seoul National University Bundang Hospital in South Korea and co-workers sequenced a region of gene amplification in colon cancer cells. The amplified region was approximately 20 times the length of that in healthy cells and contained many copies of an eleven-gene segment, including a gene implicated in drug resistance. The region also contained mutations in chromosomal repair genes which would disrupt repair pathways. These results illuminate the genetic changes that lead to gene amplification and drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental & Molecular Medicine, Experimental and Molecular Medicine</td>
          <td>5</td>
          <td>50</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum
                </a>
              </td>
          <td>
            Gustavo S. França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>72</td>
          <td>46</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c21e29753393c78c9a7147383291954089d02fd" target='_blank'>
              KDM5-driven transcriptional noise fuels plasticity-led awakening and relapse in paediatric cancer
              </a>
            </td>
          <td>
            Alejandro Allo Anido, Cecilia Roux, Emilia Chen, Siân Hamer, Abigail Shea, Ayeh Sadat Sadr, Christie English, Charlotte Butterworth, Harvey Che, Angella Bellini, Birgit Geoerger, Gudrun Schleiermacher, L. Chesler, Michael David Morgan, Alejandra Bruna
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Glioblastoma (GBM), the most common primary malignant brain tumor in adults, remains uniformly fatal due to the lack of effective targeted therapies. The epidermal growth factor receptor (EGFR) is the most frequently altered receptor tyrosine kinase oncogene in GBM with most alterations impacting the receptor ectodomain function, including gene amplification, mutation, rearrangement, and splicing site changes, which occur in approximately 50% of GBM tumors. Depatuxizumab mafodotin (Depatux-M; ABT-414), an antibody-drug conjugate composed of an EGFR-specific antibody (ABT-806) that recognizes the EGFR ectodomain linked to the cytotoxic agent monomethyl auristatin F, initially showed clinical promise. However, it failed to improve survival in phase III trials, highlighting an urgent need to understand mechanisms of resistance. We generated in vivo ABT-414 resistant GBM models using patient-derived xenografts (PDXs) and performed genomics and transcriptomic profiling, including whole exome sequencing, bulk RNA sequencing, and single-cell RNA sequencing. ABT-414-resistant tumors exhibited transcriptional reprogramming characterized by upregulation of synaptic and developmental gene networks and downregulation of biosynthetic processes, indicative of a plastic, therapy-adaptive state. Whole-exome sequencing revealed novel mutations exclusive to resistant tumors, including a recurrent TEK (TIE2) S466I point mutation present in all ABT-414 resistant GBM12 PDX tumors. Functional validation demonstrated that ectopic expression of TEK S466I and TEK WT in PDX models reduced EGFR levels, suggesting a novel feedback mechanism linking TEK signaling to EGFR downregulation and contributes to resistance. Our findings demonstrate that resistance to ABT-414 arises through both adaptive transcriptional remodeling and newly acquired genetic alterations. TEK-mediated suppression of EGFR represents a previously unrecognized mechanism of resistance, with potential implications for overcoming antibody-drug conjugate failure in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2178575c5331764ee967eeeb61b931918d25454a" target='_blank'>
              Glioblastoma resistance to EGFR antibody-drug conjugate is driven by transcriptional reprogramming and TEK-induced EGFR suppression
              </a>
            </td>
          <td>
            M. Blomquist, T. Noviello, C. Sereduk, Dustin Grief, F. Caruso, Ritin Sharma, K. Garcia-Mansfield, P. Pirrotte, J. Kloss, F. D’Angelo, B. Marin, K. Porath, Sonia Jain, Benjamin Rabichow, S. Ensign, A. Iavarone, Antonella Paladino, J. Sarkaria, Joseph C Loftus, M. Ceccarelli, Nhan L Tran
          </td>
          <td>2025-10-21</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="Multiple myeloma (MM) is a hematologic cancer marked by clonal expansion of plasma cells in the bone marrow. Although its genomic landscape has been extensively characterized, the transcriptional mechanisms that govern malignant progression and long-term tumor cell survival remain incompletely understood. We integrated single-cell RNA sequencing (scRNA-seq) data from healthy donors (HD), monoclonal gammopathy of undetermined significance (MGUS), smoldering multiple myeloma (SMM), and newly diagnosed multiple myeloma (NDMM) patients, retrieved from GEO datasets GSE124310 and GSE271107, to construct a comprehensive transcriptional landscape of plasma cell differentiation. Pseudotime and enrichment analyses identified PDIA4 as a terminal-state–associated gene. The prognostic significance of PDIA4 was validated using the multiple myeloma research foundation (MMRF) CoMMpass cohort. Functional analyses were performed in vitro and in vivo to validate the role of PDIA4 in MM cell survival and therapeutic response. Pseudotime trajectory analysis revealed progressive upregulation of genes involved in protein processing in the endoplasmic reticulum (ER), with PDIA4 identified as a top candidate in terminal-stage plasma cells. Survival analysis in the MMRF CoMMpass cohort further demonstrated that high PDIA4 expression correlated with poor overall survival. In RPMI-8226 cells, PDIA4 knockout activated the IRE1α/XBP1s branch of the unfolded protein response (UPR), impaired proliferation, and induced G1-phase arrest. PDIA4 depletion also sensitized cells to bortezomib. In vivo, sg-PDIA4 suppressed tumor growth in RPMI-8226 xenografts. PDIA4 is a key regulator of the unfolded protein response and MM cell survival. Targeting PDIA4 may enhance the efficacy of proteasome inhibitors and offers a potential strategy to overcome therapeutic resistance in multiple myeloma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/881fab1a35c430a7450d4d3fe7ea84a60cfe6f7c" target='_blank'>
              Single-cell transcriptomics identifies PDIA4 as a marker of progression and therapeutic vulnerability in multiple myeloma
              </a>
            </td>
          <td>
            Wenjie Yu, Jiamin Zhan, Yanxia Wang, Xu Cao, Aoyang Yu, Yuexuan Rao, Can Huang, Xuan Zhang, Shunv Lu, Yanhua Liu, Chong Chen, Zhenyu Li
          </td>
          <td>2025-10-21</td>
          <td>Journal of Translational Medicine</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="Understanding the complex molecular mechanisms driving drug resistance in glioblastoma (GBM) is crucial to develop effective therapeutic strategies. While prior studies have identified resistance mechanisms tied to specific drugs or pathways, a multi-modal molecular analysis of resistance across a range of drug classes is lacking. We identified highly drug-resistant (n = 5) and drug-sensitive (n = 4) cultures from a cohort of 32 patient-derived glioblastoma stem cell (GSC) cultures screened against a broad panel of ~ 500 anti-cancer drugs. To elucidate the key drivers of drug resistance, we performed integrative profiling of stemness, differentiation capacity, global gene expression, mutation profiles, and DNA methylation patterns between the two groups. Despite heterogeneous gene expression profiles, drug-resistant GSCs showed consistent upregulation of ATP-binding cassette (ABC) drug efflux transporters, stemness, and extracellular matrix (ECM)-related genes. Compared to drug-sensitive GSCs, drug-resistant GSCs exhibited more pronounced stem-like properties and reduced differentiation capacity. Notably, genes linked to axonogenesis displayed significant CpG island hypomethylation in drug-resistant GSCs. This study suggests a pivotal role for GSC plasticity, stemness maintenance, and ECM-mediated drug evasion in GBM treatment resistance. Our findings highlight the adaptive and dynamic nature of resistance mechanisms in GSCs, emphasizing the need for comprehensive molecular insights to inform targeted therapeutic strategies in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f39de902ec6cfcaa0e52251d953e6e404ad13d38" target='_blank'>
              Drug resistant glioblastoma stem cells exhibit enriched stemness signatures and share extracellular matrix overexpression
              </a>
            </td>
          <td>
            Lance L Estabillo, E. Skaga, S. Halldorsson, Einar O Vik-Mo, C. Sandberg
          </td>
          <td>2025-10-27</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Acute myeloid leukaemia (AML) with isolated trisomies (ITs) represents a distinct cytogenetic subgroup with heterogeneous clinical behaviour and incompletely defined molecular features. To explore its genomic and transcriptomic landscape, we performed next‐generation sequencing (NGS) on 14 AML patients harbouring isolated trisomies of chromosomes 8, 9, 10, 13, 14, 21 and 22. RNA sequencing (RNA‐Seq) was conducted on 15 samples, including 12 with IT and 3 cytogenetically normal AML cases (normal karyotype, NK‐AML) serving as controls. Trisomy 8 was most frequent, followed by chromosomes 13, 14 and 21. Recurrent mutations were identified in epigenetic regulators (DNMT3A, IDH1/2, ASXL1, TET2). Transcriptomic profiling stratified cases into IT‐8, IT‐21 and IT‐13+22 subgroups. Gene set enrichment analysis (GSEA) revealed shared downregulation of cell cycle‐related pathways (e.g., G2M checkpoint) and subgroup‐specific patterns: oxidative stress and unfolded protein response in IT‐8; epithelial‐mesenchymal transition and oxidative phosphorylation in IT‐21; inflammatory signalling (IL‐6/JAK/STAT, TNF‐α/NF‐κB) in IT‐13+22. A core set of 60 differentially expressed genes (DEGs) was shared, with nine hub genes related to cell cycle (MCM4, CDC7, CDC25A, DHFR), proteostasis (HSPA5, DNAJC3, CALR, HSP90B1) and inflammation. Drug sensitivity profiling revealed subgroup‐specific vulnerabilities: IT‐8 to DNA damage checkpoint inhibitors, IT‐21 to PLK/mTOR inhibitors and IT‐13+22 to BRAF/EGFR‐targeted agents. These findings highlight AML‐IT heterogeneity and therapeutic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34a07ac5d29af86fd0590c16e27dfff205a7ea13" target='_blank'>
              Integrated Genomic and Transcriptomic Profiling of Isolated Trisomies in AML Reveals Cell Cycle Dysregulation and Therapeutic Vulnerabilities
              </a>
            </td>
          <td>
            J. H. Maués, Bruno Kosa L Duarte, M. Svidnicki, Herton Luiz A S Filho, F. Niemann, Adriana da Silva S Duarte, Paula de Melo Campos, P. Moraes-Vieira, S. T. Olalla Saad
          </td>
          <td>2025-11-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/840b437f838fbeb4893166e383606b0c1b14d251" target='_blank'>
              Integrated Metabolic Complex Genetic Interaction Network of Chromosome 4p Loss in Basal Breast Cancer
              </a>
            </td>
          <td>
            Lynn Karam, Alain Pacis, Rohan Dandage, Michael Schwartz, Gabriel Alzial, Anastasia Gherghi, H. Djambazian, Gerardo Zapata, Marina A Schapfl, Paria Asadi, Mayerly Castrillón, Alysh Orr, Belen Hernandez, Pegah Kargar, Michelle Vandeloo, Sylvia Santosa, A. Villunger, Traver Hart, Jiannis Ragoussis, Geneviève Deblois, Andreas Bergdahl, Morag Park, Guillaume Bourque, Elena Kuzmin
          </td>
          <td>2025-10-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="


 Despite the remarkable success of targeted therapy, the emergence of acquired resistance remains a major obstacle to cure acute myeloid leukemia (AML) patients. While existing studies have predominantly focused on genetic heterogeneity as a driver for therapy resistance, recent evidence suggests that non-genetic persister state is equally important, but such studies are lacking in AML. Here, we define the temporal transcriptome and chromatin landscape along with lineage trajectory of individual genetically identical AML persister clones to existing anti-AML therapeutics.
 To model persister state in AML, we utilized watermelon library (Oren et al., 2021), that expresses lineage barcode with inducible Histone-2B-mCherry to trace clonal evolution and cell division. We first evaluated whether slow-cycling, quiescent persisters emerge in AML lines after exposure to standard therapy. Our data showed the emergence of slow cycling persisters with cytarabine (AraC) in all 4 cell lines, and gilteritinib (GLT) in MOLM-13 and MV4-11 cells while BH3 mimetics (venetoclax (VEN), S63845 and navitoclax) had fast-cycling persisters. Integrating proliferative-index profiles with cell cycle profiles depicted extensive heterogeneity in AML persisters imposed by drug target rather than genetic state. Drug holiday experiments exhibited a reversible drug tolerance state in both slow and fast-cycling persisters. Longitudinal drug sensitivity and BH3 profiling assays showed that persisters emerged with adaptation to reduced mitochondrial priming that reversed after drug holiday. However, transitioning from persister to stable resistant cells resulted in a sustained loss in apoptotic priming, mirroring observations made in PDX models and primary tumors.
 To understand trajectory of persister emergence and underlying mechanisms, we performed single-cell multiomics on drug-naïve and persister populations (day 9) in MV4-11 (GLT), and OCI-AML2 (VEN and AraC). We expanded 10,000 unique barcoded cells and administered the drug treatment to derive persisters. Using lineage barcode analysis, we report significant reduction in the dominant lineage of day 0 and survival of multiple small lineages indicating an acquired state of persistence than selection of pre-existing resistant clone in both the lines. Further trajectory analysis defined three fates of the lineages as: expanding (log2FC>1), shrinking (log2FC <1) and stable. AraC persisters showed higher expanding and stable lineages, while VEN and GLT persisters resulted in higher shrinking and stable clones, revealing heterogeneity in lineage expansion. We next asked whether persisters resulting from different therapies arised from the same parental lineage. To test this, we performed lineage tracing on VEN and AraC persisters in OCI-AML2 and compared surviving lineages at Day 9 vs Day 0 cells. This analysis identified 52 parental lineages that were able to produce both VEN and AraC persisters in AML2, providing evidence of branched evolution of persisters from the same parental clone.
 We finally defined transcriptional and epigenetic heterogeneity in baseline and persister cells by performing UMAP on single-cell RNAseq and ATACseq data. We identified persisters to all 3 drugs clusters distinctly from their drug naïve counterparts at both transcriptome and chromatin accessibility. Next, we investigated differentially expressed pathways in persisters and observed higher inflammatory response pathways such as TNFα signaling in GLT, enrichment of MYC targets and oxidative phosphorylation in VEN and high mTORC1 signalling in AraC persisters. We observed higher chromatin accessibility in GLT and VEN whereas condensed chromatin in AraC persisters. We next asked which motifs are highly enriched in each persister and observed the enrichment of various KLF and SP family of motifs in each persister driving multiple gene signatures in each persister type. We also observed enrichment of LSC-primed and cDC like cells in GLT persisters but with AraC and VEN persisters a uniform pattern of all states was observed, indicated stemness may not be common mechanism to persister evolution.
 Collectively, our integrated single-cell multiomics and lineage tracing approach reveals that AML persisters emerge through stochastic drug-specific, non-genetic adaptive mechanisms characterized by distinct transcriptional and chromatin remodeling programs rather than deterministic Darwinian selection of pre-existing resistant cells.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b912c83b14f52bda36c8d81fafb30ac8e86a906c" target='_blank'>
              Dissection of non-genetic resistance mechanisms by lineage tracing and single cell multiomics reveals drug-specific persister programs in Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            K. Bhatia, Xiao Xian Lin, Ankit Jana, Emanuelle I. Grody, Ryan Jeremy Lim Jing Wei, Shawn Koh Jia Jun, Elisha Teh Jia Yu, Cin thet Kyi, Prasad Iyer, Greg Kellogg, Yogesh Goyal, Lawrence Boise, Yaara Oren, Shruti Bhatt
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Despite the outstanding achievements of precision medicine in hematology, many targeted therapies eventually fail due to the emergence of resistance mechanisms. Traditionally, a genocentric approach has been adopted to uncover the molecular underpinnings of treatment resistance. This has contributed to identifying resistance gene mutations and designing novel therapeutic molecules with increased potency for the mutant target. However, over the last five years, additional non-genetic adaptations have become increasingly recognized as crucial promoters of treatment resistance. In parallel, emerging works in the field of evolutionary biology suggest that advantageous phenotypic traits appear most often due to cell-intrinsic phenotypic plasticity and can arise independently of gene mutations. In selected cases, single genetic abnormalities such as those involving TP53 can prime human cells for plasticity and facilitate phenotypic variability. In this narrative review, we retrace the resistance mechanisms to targeted therapies in the framework of these novel evolutionary concepts. We highlight the dichotomy between genes-first and phenotypes-first pathways of treatment adaptation, with the former being driven by traditional single-point mutations and the latter initiated by the phenotypic diversity and the high-level plasticity of cancer cells. Focusing on resistance mechanisms to kinase inhibitors and BH3 mimetics in leukemias and lymphomas, we describe how each drug can trigger both escape routes, which may even coexist within the tumor bulk of individual patients. Lastly, we provide a three-step translational perspective on how to counteract phenotypes-first resistance mechanisms, with the aim of prolonging disease control in hematological malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/326e4b504dccc956ee299598bcd494d4433bc387" target='_blank'>
              Genes-first and phenotypes-first paths to treatment resistance in hematological malignancies
              </a>
            </td>
          <td>
            Edoardo Tamellini, Cristina Frusteri, Isacco Ferrarini
          </td>
          <td>2025-11-10</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background: Doxorubicin (DOX) is a widely used chemotherapeutic agent, but its efficacy is often limited by cancer cell resistance. Although multiple DOX resistance mechanisms have been characterized, the global transcriptomic alterations underlying this phenomenon remain poorly understood. The aim of this work was to determine whether a common transcriptional response associated with DOX desensitization exists across tumor cells of different origins and to identify the core elements of this response. Methods: We performed an integrated bioinformatics analysis, including: analysis of independent transcriptomic datasets (comparing DOX-resistant neuroblastoma, breast, and cervical carcinoma cells to their DOX-sensitive counterparts), functional annotation of differentially expressed genes, reconstruction and topology analysis of gene networks, text mining, and survival analysis. The findings were validated through in vitro functional tests, RT-PCR, and analysis of the Cancer Therapeutics Response Portal and The Cancer Genome Atlas. Results: We showed that DOX resistance in cancer cells is associated with cytoskeletal reorganization, modulation of cell adhesion, cholesterol biosynthesis, and dysregulation of mTORC1, Wnt, and Gβγ signaling pathways. Network analysis identified a conserved regulome of 37 resistance-linked genes, with GJA1, SEH1L, TCF3, TUBA4A, and ZYX emerging as central hubs (mean degree: 8.7–19.7; mean fold change: 2.4–21.3). Experimental validation in DOX-resistant KB-8-5 cervical carcinoma cells and their sensitive counterparts (KB-3-1) confirmed enhanced cellular adhesion and reduced intracellular cholesterol levels associated with chemoresistance, supporting our in silico findings. A detailed follow-up analysis verified the upregulation of these hub genes in chemoresistant cells and their correlation with poor clinical outcomes across multiple cancer types. Conclusions: This integrative analysis identifies conserved transcriptomic signatures of DOX resistance and highlights hub genes GJA1, SEH1L, TCF3, TUBA4A, and ZYX with potential as predictive biomarkers and therapeutic targets. Targeting these pathways may help overcome chemoresistance and improve treatment outcomes in cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f2b259a82bfdbeb3c8cb64a5419e6a90ac8b1c5" target='_blank'>
              Novel Cross-Cancer Hub Genes in Doxorubicin Resistance Identified by Transcriptional Mapping
              </a>
            </td>
          <td>
            A. D. Moralev, O. Markov, M. Zenkova, A. Markov
          </td>
          <td>2025-10-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/566fc9fd9c1a3c438b8fdc4f08cbb7f2ae09af61" target='_blank'>
              Joint analysis of expression and variation at single cell resolution by scVar
              </a>
            </td>
          <td>
            Wei Zhao, Fei Yang, Wenting Zong, Xinchang Zheng, Yiming Bao
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1e72f57d35ee3dac82999a8b9759c889a0508af" target='_blank'>
              Genetic and transcriptomic alterations underlying aggressiveness in 5-fluorouracil-resistant HCT116 cells
              </a>
            </td>
          <td>
            Pornchai Sooksaen, A. Thim-uam, Ratsada Praphasawat, Chutamas Thepmalee, Kanokkarn Phromnoi, Damratsamon Surangkul, Surisak Prasarnpun, Chakkraphong Khonthun
          </td>
          <td>2025-10-10</td>
          <td>Medical Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/181bda14a56f6f64f5ac4fffa1ec57650a41f5a9" target='_blank'>
              Cooperative and antagonistic interactions between sub-clones favour the co-existence of multiple resistance mechanisms in melanoma
              </a>
            </td>
          <td>
            Karin Schlegelmilch, Viola Hollek, S. Hooper, Giovanni Giangreco, Sasha Bailey, Sarah C. Macfarlane, Alexandrine Carminati, Amy Bowes, S. Strohbuecker, Xiao Fu, E. Sahai
          </td>
          <td>2025-10-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Mantle cell lymphoma (MCL) responds to frontline therapy but is susceptible to relapse. While Bruton’s tyrosine kinase inhibitors (BTKi) achieve high response rates, most patients eventually experience disease progression. Predicting responses to subsequent treatments remains challenging due to the lack of an established platform. Heterogeneity in gene alterations and cellular pathways contribute to resistance, complicating treatment approaches. Here, we present a multi-modal profiling platform, targeting key pathways rather than focusing on singular DNA-associated lesions. We identified dysregulated signaling pathways by performing gene expression profiling on 20 MCL samples using a custom MCL MATCH gene set and analyzed the data with gene-set variation analysis. We also screened 22 therapeutics in vitro to assess their efficacy. Whole exome sequencing was conducted to identify prevalent and actionable mutations linked to enriched pathways. Finally, we tested selected therapeutics in patient-derived xenograft mouse models to predict potential response in corresponding patients. Based on our integrative profiling, we identified the top therapeutic candidates for three patients in this study. This integrative platform may help identify targeted therapies for BTKi-relapsed/refractory MCL patients in clinical settings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3929c1302d66ca38df0d5f835571ca5c0772a36" target='_blank'>
              Integrative profiling strategies to guide personalized therapy in mantle cell lymphoma: a pilot study
              </a>
            </td>
          <td>
            Yang Liu, H. Hill, Yijing Li, J. McIntosh, V. Jiang, F. Yan, Yixin Yao, Yue Fei, Jared R Zhang, Lawrence Qu, Jun Yao, P. Jain, Kent Chen, Michael Wang
          </td>
          <td>2025-11-21</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee85598703608ccef540311d86a0a9e7d9f4dae2" target='_blank'>
              Genetic barcoding of individual cells links cancer evolutionary trajectories and prognostic outcomes
              </a>
            </td>
          <td>
            Candice Merle, Ivan Di Terlizzi, M. Huyghe, Danae Welboren, Wenjie Sun, Cécile Conrad, Leïla Perié, S. Rulands, Silvia Fre
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 Despite the clinical success of the BCL2 inhibitor venetoclax in acute myeloid leukemia (AML), treatment resistance frequently emerges, leading to relapse. Mounting evidence suggests that metabolic reprogramming contributes to venetoclax resistance, yet the underlying mechanisms remain poorly defined. This study aimed to develop and characterize venetoclax-resistant AML models and explore metabolic vulnerabilities that could be exploited therapeutically.
 For this purpose, we established two venetoclax-resistant AML cell lines (MV4-11VR and MOLM-13VR) through repeated intermittent exposure to increasing doses of venetoclax. Resistance was assessed via cell viability assays, annexin V-based apoptosis analysis, mitochondrial membrane potential, cell cycle profiling, and colony formation assays. Metabolic reprogramming was evaluated using measurements of oxygen consumption rate (OCR) and extracellular acidification rate (ECAR). Molecular and genomic alterations were explored through transcriptomic profiling, global proteomics, and targeted sequencing. In addition, we tested the therapeutic potential of combining venetoclax with metabolic inhibitors, metformin, a mitochondrial complex I inhibitor, and KPT-9274, a NAMPT inhibitor, by calculating synergy scores and quantifying apoptotic responses.
 Venetoclax-resistant MV4-11VR and MOLM-13VR cells exhibited markedly elevated IC50 values (more than 200-fold compared to parental cells), impaired apoptosis induction, preserved proliferative capacity, and enhanced colony formation under venetoclax exposure. Notably, resistant cells showed distinct metabolic rewiring: MV4-11VR cells relied on enhanced glycolysis, whereas MOLM-13VR cells displayed increased oxidative phosphorylation (all p < 0.05). These findings were supported by functional assays and proteomic signatures, which showed enrichment of carbohydrate metabolism pathways in MV4-11VR and aerobic energy production in MOLM-13VR (all FDR < 0.05).
 At the molecular level, venetoclax resistance was accompanied by cell line-specific modulation of BCL2-family gene expression. MV4-11VR cells exhibited downregulation of multiple pro-apoptotic genes, including BAX, BCL2L11, BBC3, BIK, and BNIP3, while MOLM-13VR cells showed increased expression of MCL1 and BAD, along with reduced expression of BID, PMAIP1, and BMF. At the protein level, both models demonstrated reduced PARP1 cleavage and lower γH2AX levels upon venetoclax exposure, indicating suppression of apoptosis and DNA damage signaling. Furthermore, total BCL2 levels remained unchanged, while BCL-XL expression was stable in resistant cells but downregulated in parental cells upon drug exposure. A shared hallmark in both models was the hyperactivation of the PI3K/AKT/mTOR pathway, as evidenced by increased phosphorylation of the downstream effector RPS6. Additionally, MV4-11VR cells showed reduced ERK1/2 phosphorylation, whereas MOLM-13VR cells exhibited increased ERK1/2 activation.
 Genomic profiling revealed additional mechanisms contributing to resistance. MV4-11VR cells acquired a novel TP53 mutation (R158H) and showed clonal expansion of a pre-existing TP53 R248W mutation, alongside a mosaic deletion of the TP53 locus. In MOLM-13VR cells, sequencing revealed the emergence of a truncating mutation in the SPEN gene (I1326*) and acquisition of trisomy 5, further supporting distinct genetic trajectories of resistance.
 Therapeutically, venetoclax-resistant cells showed partial sensitivity to the metabolic inhibitors metformin and KPT-9274. Importantly, combination therapy with venetoclax significantly increased apoptosis in resistant cells. The venetoclax + KPT-9274 combination yielded strong synergy (ZIP scores up to 13.86) and eliminated over 95% of resistant cells, demonstrating the feasibility of targeting metabolic dependencies to restore drug sensitivity.
 Acquired venetoclax resistance in AML arises through heterogeneous yet targetable metabolic reprogramming. Distinct metabolic and molecular adaptations in resistant MV4-11 and MOLM-13 cells point to the complexity of resistance mechanisms. Nonetheless, the shared activation of survival pathways and metabolic plasticity creates vulnerabilities that can be exploited using mitochondrial or NAD+ biosynthesis inhibitors. These findings support a combinatorial therapeutic strategy targeting both apoptotic and metabolic pathways to overcome acquired venetoclax resistance in AML. Supported by FAPESP, CAPES, and CNPq.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c98bf25f48d9830834e2992fc3cf2049960a971f" target='_blank'>
              Metabolic reprogramming drives acquired venetoclax resistance in Acute Myeloid Leukemia and reveals targetable vulnerabilities
              </a>
            </td>
          <td>
            K. Lima, Gustavo Nery de Queiroz, Marcella Cipelli, Victória Tomaz, L. G. Cortes, Marina Basto Silva, R. Guedes, P. Campregher, Niels Câmara, L. Costa-Lotufo, E. Rego, J. Machado-Neto
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Metastatic colorectal cancer is commonly treated with oxaliplatin. However, patients may develop resistance to treatment over time, and currently, there are no validated predictive biomarkers for this resistance. A differential analysis of the transcriptome and DNA methylome of colorectal cancer cell lines, classified by their varying IC50 values for oxaliplatin, revealed that genes associated with resistance were enriched for interferon regulatory factor pathways. In contrast, sensitive genes showed enrichment for transcription, amino acid metabolism, development, and binding motifs of c-MyC:Max, AP1 and others. In univariate Cox’s proportional hazard model analysis, it was found that UBE2H expression is linked to shorter survival time in the TCGA dataset and was further validated across five GEO datasets of colorectal cancer. The transcription of UBE2H, along with its gene body methylation, and copy number variation was found to be higher in resistant cell lines compared to sensitive ones. Additionally, UBE2H levels were higher in cancer samples than in control samples, while the promoter methylation is lower in cancer samples than in control samples. In groups with high UBE2H, there was an increased infiltration of eight cell types, including CD8+ T cells and type 2 T helper cells. Conversely, only T helper 17 cells showed reduced infiltration. Moreover, UBE2H expression was positively correlated with checkpoint inhibitors, CTLA4 and PDCD1, along with immune regulatory genes, such as FOXP3, IL10, IGFB1, CD274, and LAG3, etc. Analysis of single cell RNA-sequencing data revealed that UBE2H expression is elevated in undifferentiated and proliferative cells located at the base of intestinal crypts in normal colon tissue. Our findings suggest that UBE2H could serve as a resistance marker to oxaliplatin, as it is associated with methylation, the presence of proliferative and undifferentiated cancer cells, and immune exhaustion. The proposed analytical pipeline may also be applicable to other cancers and diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/853b5cb3a79ef305fa7eb4ff1e22fd4ca52bed15" target='_blank'>
              An integrative analysis of transcriptome, methylome and single-cell RNA sequencing data identifies UBE2H as a marker of oxaliplatin resistance in colorectal cancer
              </a>
            </td>
          <td>
            Jinman Fang, Qizhi Zhu, Bo Hong, Xueling Li, Hongzhi Wang
          </td>
          <td>2025-10-31</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Oncogenesis involves widespread genetic and epigenetic alterations, yet the full spectrum of somatic variation genome-wide remains unresolved. We generated a near-telomere-to-telomere (T2T) diploid assembly of a donor paired with deep short- and long-read sequencing of their melanoma. This revealed that 16% of somatic variants occur in sequences absent from GRCh38, with satellite repeats acting as hotspots for UV-induced damage due to sequence-intrinsic mutability and inefficient repair. Centromere kinetochore domains emerged as focal sites of structural, genetic, and epigenetic variation, leading to remodeling of centromere kinetochore binding domains during tumor evolution. Single-molecule telomere reconstructions uncovered cycles of attrition, deletion, and telomerase-mediated extension that shape cancer telomeres. Finally, diploid chromatin maps exposed that copy number alterations and epimutations, rather than point mutations, predominate in rewiring cancer regulatory programs. These findings define the full landscape of a cancer’s somatic variation and their functional impact, establishing a blueprint for T2T studies of mosaicism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bd95123f481a9cef05baf3ace1238c5ad7fb915" target='_blank'>
              A telomere-to-telomere map of somatic mutation burden and functional impact in cancer
              </a>
            </td>
          <td>
            Min-Hwan Sohn, Danilo Dubocanin, Mitchell R. Vollger, Youngjun Kwon, Anna Minkina, Katherine M. Munson, Samuel FM Hart, Jane E. Ranchalis, Nancy L. Parmalee, Adriana E. Sedeño-Cortés, Jeffrey Ou, N. Y. Au, Stephanie Bohaczuk, Brianne Carroll, C. D. Frazar, William T. Harvey, Kendra Hoekzema, Meng-Fan Huang, Caitlin N Jacques, Dana M. Jensen, J. T. Kolar, R. Lee, Jiadong Lin, Kelsey Loy, Taralynn Mack, Yizi Mao, Meranda M. Pham, E. Ryke, Josh Smith, Lila M. Sutherlin, Elliott G. Swanson, Jeffrey M. Weiss, S. Wg, C. Carvalho, Tim H. H. Coorens, Kelley Harris, Chia-Lin Wei, E. Eichler, Nicolas Altemose, James T. Bennett, Andrew B. Stergachis
          </td>
          <td>2025-10-13</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>35</td>
        </tr>

        <tr id="Breast cancer recurrence remains a major clinical challenge, often associated with therapy resistance and altered metabolic states. To define metabolic vulnerabilities of recurrent disease, we performed a CRISPR knockout screen targeting 421 metabolic genes in paired primary and recurrent HER2-driven breast cancer cell lines. While both primary and recurrent tumors shared dependencies on core metabolic pathways, recurrent tumors exhibited selective essentiality for the de novo pyrimidine synthesis pathway, including Cad, Dhodh, and Ctps. Pharmacologic inhibition of the rate-limiting enzyme DHODH with BAY-2402234 selectively impaired the growth of recurrent tumor cells, while primary tumor cells were relatively resistant. BAY treatment robustly inhibited pyrimidine synthesis in all lines, but only recurrent cells underwent iron-dependent lipid peroxidation and ferroptotic cell death. Lipidomic profiling revealed enrichment of polyunsaturated ether phospholipids in recurrent cells, which may predispose them to ferroptosis. A sensitizer CRISPR screen in primary cells further identified nucleotide salvage and lipid metabolic pathways as modifiers of DHODH inhibitor sensitivity. Stable isotope tracing and nutrient depletion experiments showed that primary cells can compensate for DHODH inhibition through nucleotide salvage, whereas recurrent cells exhibit impaired salvage capacity, likely due to reduced expression of Slc28/Slc29 nucleoside transporters. Together, these findings reveal that breast cancer recurrence is associated with increased dependence on de novo pyrimidine synthesis to suppress ferroptosis, highlighting a therapeutically actionable metabolic vulnerability in recurrent disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83ca3ef9a21a542254ed4d0723e6bc92c26d2a66" target='_blank'>
              Recurrent Breast Cancer Cells Depend on De novo Pyrimidine Biosynthesis to Suppress Ferroptosis
              </a>
            </td>
          <td>
            Kelley R. McCutcheon, Josh Wu, Yasemin Ceyhan Ozdemir, Brock J McKinney, Sharan Srinivasan, Ayaha Itokawa, Oliver J. Newsom, Anna-Lena B G Vigil, Douglas B. Fox, Lucas B. Sullivan, James V. Alvarez
          </td>
          <td>2025-11-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Head and neck squamous cell carcinoma (HNSCC) remains a prevalent and aggressive malignancy, characterized by a lack of targeted therapies and limited clinical benefits. Here, we conducted an optimized whole-genome CRISPR screen across five HNSCC cell lines aimed at identifying actionable genetic vulnerabilities for rapid preclinical evaluation as novel targeted therapies. Given their critical role in cancer, cyclin-dependent kinases (CDKs) were prioritized for further investigation. Among these, CDK7 was identified as an essential and targetable gene across all five cell lines, prompting its selection for in-depth functional and molecular characterization. Genetic and pharmacological inhibition of CDK7 significantly and consistently reduced tumor cell proliferation due to generalized cell cycle arrest and apoptosis induction. Additionally, CDK7 knockout (KO) and selective inhibitors (YKL-5-124 and samuraciclib) demonstrated potent antitumor activity, effectively suppressing tumor growth in HNSCC patient-derived organoids (PDOs), as well as in both cell line- and patient-derived xenograft (PDX) mouse models with minimal toxicity. Mechanistically, CDK7 inhibition led to a broad downregulation of gene sets related to cell cycle progression and DNA repair, and significantly reduced the transcription of essential genes and untargetable vulnerabilities identified by our CRISPR screen. These findings highlight CDK7 as a promising therapeutic target for HNSCC. Our study provides strong evidence of the robust antitumor activity of CDK7-selective inhibition in disease-relevant preclinical models, strongly supporting its progression to clinical testing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4085230d5db63fd39c16115dd67e53fe6113079" target='_blank'>
              CDK7-targeted therapy effectively disrupts cell cycle progression and oncogenic signaling in head and neck cancer
              </a>
            </td>
          <td>
            María Otero-Rosales, M. Álvarez-González, Irene Pazos, Beatriz de Luxán-Delgado, Sonia Del Marro, Esperanza Pozo-Agundo, Mar Rodríguez-Santamaría, Ana López-Fernández, D. Corte-Torres, R. Granda-Díaz, S. Álvarez-Teijeiro, Iván Fernández-Vega, Corina Lorz, Ramón García-Escudero, J. P. Rodrigo, Konstantinos Tzelepis, G. Vassiliou, Irene Ferrer, Mónica Álvarez-Fernández, J. García-Pedrero, F. Hermida-Prado
          </td>
          <td>2025-11-06</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) play a crucial role in colorectal cancer by sustaining intratumoral heterogeneity, therapeutic resistance, and metastatic potential. CD133 (PROM1) is among the most frequently used surface markers for CSC identification, whereas TRIM28, a versatile epigenetic regulator, has been implicated in controlling CD133 expression and stem-like features. In this study, we performed a detailed molecular and functional analysis of Caco2 colorectal cancer cell clones with individual knockouts of CD133 or TRIM28. Elimination of CD133 neither altered global gene expression, as confirmed by transcriptome profiling, nor affected key cellular properties. In contrast, loss of TRIM28 led to a marked reduction in CD133 protein abundance and induced extensive molecular and phenotypic remodeling. TRIM28 knockout was associated with broad transcriptomic changes involving more than 500 differentially expressed genes, decreased proliferative activity monitored by time-lapse imaging, and reduced sensitivity to paclitaxel, cisplatin, and curcumin. Furthermore, immune evasion molecules CD24 and CD47 (“don’t eat me” signals) were strongly upregulated in TRIM28-deficient cells, consistently confirmed by both RNA-Seq and flow cytometry analyses. At the same time, imaging flow cytometry and mitochondrial activity assays indicated that these effects were not due to major shifts in mitotic index or bioenergetic status. Altogether, our results demonstrate that TRIM28, rather than CD133, functions as a central regulator of CSC-associated phenotypes in colorectal cancer. These findings highlight the importance of epigenetic context in CSC biology and may inform the development of more effective therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82a3d3eb05c9db6b57d94b54e8bdad26455b3ea6" target='_blank'>
              Contrasting Impacts of Targeted Disruption of the Cancer Stem Cell Marker CD133 and Its Epigenetic Regulator TRIM28 in Colorectal Cancer Cells
              </a>
            </td>
          <td>
            I. Kholodenko, A. Saidova, D. Potashnikova, V. A. Arzumanian, D. Romashin, A. Tvorogova, E. Poverennaya, K. Yarygin, Yan S. Kim
          </td>
          <td>2025-11-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Melanoma plasticity drives immune evasion and therapy resistance through dynamic cell-state transitions beyond genetic alterations. Epigenetic remodeling critically influences such processes, yet its role in reshaping the tumor ecosystem under therapeutic pressure remains unresolved. Here, we profiled longitudinal biopsies from melanoma patients treated in the phase Ib NIBIT-M4 epi-immunotherapy clinical trial (NCT02608437), testing the combination of a DNMT1 inhibitor with anti-CTLA4 using single-cell multiome and high-resolution spatial transcriptomics. Integrated analyses resolved seven malignant meta-programs, including a rare Wnt/β-catenin–driven melanocytic state and a de-differentiated neural crest–like state enriched in non-responders. Spatial modeling revealed that homotypic clustering stabilizes resistant programs, with neural crest–like cells forming compact, centrally localized niches, whereas Wnt/β-catenin subpopulations displayed a bimodal architecture, either cohesive clusters sustained by adhesion or dispersed, transcriptionally plastic cells. Responders exhibited progressive enrichment of an antigen presentation/interferon program and coordinated remodeling of the tumor microenvironment with T and B cell expansion, whereas tumors from non-responder patients maintained stable composition of neural crest–like clusters. Epigenetic therapy reactivated transposable elements, providing both regulatory signals that prime innate immunity within microenvironment and generating antigens that drive immunoediting and immunogenicity of Antigen presentation/interferon cell states in responders. Finally, NFATC2 emerged as a master regulator of neural crests–like transcriptional phenotypes and promoter of resistance to therapeutic interventions in melanoma patients. NFATC2 perturbation was able to shift tumor cells towards more differentiated and immunogenic states. These findings reveal how epigenetic-based immunotherapy reshapes melanoma ecosystems, provide mechanistic insights into how multiple transcriptional programs promote tumor plasticity and resistance to both combinatorial therapies and immune checkpoint blockade, identify spatial clustering as a principle stabilizing resistant niches, and highlight β-catenin and NFATC2 as actionable vulnerabilities to overcome resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc9320984fb7ca80afa8b2d49d724dd86390a378" target='_blank'>
              Transposable Elements and Homotypic Niches Drive Immune Dynamics and Resistance in Melanoma Epigenetic-based immunotherapy
              </a>
            </td>
          <td>
            Erika Ciervo, Francesco Ceccarelli, A. D. Di Giacomo, Piera Grisolia, A. Covre, Z. M. Besharat, Antonio De Falco, F. Caruso, Luigi Laezza, Luigi Ferraro, Gloria Mas Martin, M. F. Lofiego, Tommaso Sani, E. Ferretti, Yan Guo, Sean B. Holden, R. Mortarini, A. Anichini, M. Maio, T. Noviello, M. Ceccarelli
          </td>
          <td>2025-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25d53c86924fae7e6d4164c1b616a447b8ef85b5" target='_blank'>
              Comprehensive and accelerated mapping of driver mutations through single-nucleotide random mutagenesis of target genes
              </a>
            </td>
          <td>
            Keisuke Yoshida, Elena Solovieva, N. Miura, Shigeyuki Shichino, Michiaki Hamada, M. Muratani, Takanori Amano, Kazufumi Honda
          </td>
          <td>2025-11-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Highly accessible genomic super-enhancers often drive tumor-promoting programs, yet the impact of genomic variation within these regulatory elements remains unclear. Here, we identified a bifunctional super-enhancer that regulates the expression of cancer-promoting genes LINC00636 and CD47 in breast cancer. We discovered that a common germline insertion variant within the super-enhancer is associated with reduced chromatin accessibility at the super-enhancer locus. Deletion of the insertion in breast cancer cells increased chromatin accessibility, leading to upregulation of LINC00636 and CD47, enhanced resistance to nutrient-deprivation–induced apoptosis (mediated by CD47), activation of senescence (driven by elevated LINC00636), delayed cell death, and reduced infiltration of CD80⁺ pro-inflammatory macrophages, changes that represent tumor-promoting features. Together, our findings uncover a common insertion-deletion variant that fine-tunes the regulatory activity of a bifunctional super-enhancer, suggest a protective role for the insertion allele, and establish a novel function for LINC00636 in senescence and breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b015f1ff776d845bca2d14843c5dc6acc6da18a2" target='_blank'>
              An InDel Genomic Variant within a Bifunctional Super-Enhancer for LINC00636 and CD47 Regulation in Breast Cancer
              </a>
            </td>
          <td>
            Carolina Di Benedetto, Amelia Tsark, D. Acenas, Alysia Thach, Anmol Singhal, Valentina Opazo-Mellado, Anthony Rodriguez Lemus, Mustapha El Zeini, Hui Zhang, Catherine Park, Hugo Gonzalez Velozo, Irving Weissmann, Paola Betancur
          </td>
          <td>2025-11-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Aim: This study aimed to elucidate the role of regulator of chromosome condensation 1 (RCC1) in colorectal cancer (CRC) progression, as well as its involvement in chemoresistance. We specifically examined how RCC1 knockdown modulates cellular responses, including cell cycle, apoptosis, and senescence induced by 5-fluorouracil (5-FU) or doxorubicin (Doxo) in both parental and drug-resistant CRC cell lines. Additionally, we assessed the potential of RCC1 inhibition as an adjuvant therapeutic strategy to enhance the efficacy of chemoradiotherapy in CRC. Methods: The expression of RCC1 in colon cancer tissues and corresponding adjacent non-cancerous tissues was evaluated through tissue microarrays, and its correlation with characteristics and patient prognosis was also examined. Subsequently, a series of in vivo and in vitro experiments based on parental and drug-resistant CRC cell lines were conducted to assess the impact of RCC1 knockdown on sensitivity to 5-FU or Doxo. Finally, transcriptomic analysis and subsequent validation assays were performed to explore the underlying molecular mechanisms. Results: RCC1 knockdown significantly enhanced the antitumor efficacy of 5-FU and Doxo in both CRC and drug-resistant CRC cells. In xenograft models, RCC1 knockdown in combination with 5-FU or Doxo suppressed tumor growth with no evident systemic toxicity observed. Transcriptomic profiling and experimental verification revealed that RCC1 knockdown may impair DNA repair by downregulating key repair proteins, thereby leading to more severe and sustained DNA damage. Conclusion: Our results indicate that RCC1 downregulation enhances the responsiveness of both parental and drug-resistant CRC cells to 5-FU and Doxo, highlighting its potential as a therapeutic adjunct to improve clinical outcomes in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b8afbd5b9e059c22aa8009a12c6871018f16b21" target='_blank'>
              RCC1 knockdown sensitizes drug-resistant colorectal cancer to 5-fluorouracil or doxorubicin by impairing DNA repair
              </a>
            </td>
          <td>
            Jing Li, Ya Meng, Xumei Ouyang, Xiaowen Lin, Yangzhe Wu, Hang Fai Kwok
          </td>
          <td>2025-11-10</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Acute myeloid leukemia is a genetically and cellularly heterogeneous disease. We characterize 120 AMLs using genomic and transcriptomic analyses, including single-cell RNA sequencing. Our results reveal an extensive cellular heterogeneity that distorts the bulk transcriptomic profiles. Selective examination of the transcriptional signatures of >90,000 immature AML cells identifies four main clusters, thereby extending current genomic classification of AML. Notably, NPM1-mutated AML can be stratified into two clinically relevant classes, with NPM1class I associated with downregulation of MHC class II and excellent survival following hematopoietic stem cell transplantation. NPM1class II is instead associated with resistance to allogeneic T cells in an ex vivo co-culture assay, and importantly, dismal survival following hematopoietic stem cell transplantation. These findings provide insights into the cellular state space of AML, define diagnostic entities, and highlight potential therapeutic intervention points. The clinical outcomes and treatment responses of acute myeloid leukemia (AML) patients are heterogeneous. Here, the authors use bulk and single-cell sequencing approaches and identify two transcriptomic subtypes within NPM1-mutated AML with distinct immune evasion properties and responses to hematopoietic stem cell transplantation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0674b44d7df57e733fca27fc81a60b0eeaf92970" target='_blank'>
              The AML cellular state space unveils NPM1 immune evasion subtypes with distinct clinical outcomes
              </a>
            </td>
          <td>
            H. Lilljebjörn, Pablo Peña-Martínez, Hanna Thorsson, Rasmus Henningsson, M. Rissler, N. Landberg, Noelia Puente-Moncada, Sofia von Palffy, Vendela Rissler, Petr Stanek, Jonathan Desponds, Xiangfu Zhong, Gunnar Juliusson, Vladimir Lazarevic, Sören Lehmann, Magnus Fontes, H. Ågerstam, Carl Sandén, Christina Orsmark-Pietras, T. Fioretos
          </td>
          <td>2025-11-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82ab0daa25a475849fd61806981974de4d78a423" target='_blank'>
              A clinical-stage oncology compound selectively targets drug-resistant cancers
              </a>
            </td>
          <td>
            Kaitlin Long, Debanjan Bhattacharjee, Samuel H. Newman-Stonebraker, Simon C B Suhr, Brandon Q. Mercado, Anthony Tighe, Luciano Romero, Sarah L. Thompson, Erin L. Sausville, Kristen M. John, Linda Julian, Sanat Mishra, Olaf Klingbeil, Punya Gupta, Uditi Bhatt, Allen C. Gao, Sara Ricardo, C. Vakoc, B. Bornhauser, S. M. Corsello, Stephen S. Taylor, Patrick L. Holland, J. Sheltzer
          </td>
          <td>2025-11-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e0d7b578f97dbff41a41bb90de3b031320cfe41" target='_blank'>
              Unravelling drug resistant proteotypes through phenotype-resolved proteomics of single-cell derived colonies
              </a>
            </td>
          <td>
            Di Qin, Ridhima Das, S. Schallenberg, K. Riecken, I. Tinhofer, Fabian Coscia
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Somatic mosaicism is increasingly recognized as a fundamental feature of human biology, yet the detection of somatic mutations remains challenging. The SMaHT Network conducted four large-scale benchmarking experiments to evaluate sequencing technologies, experimental approaches, and computational methods for detecting diverse somatic mutations. Cumulative sequencing coverage exceeded 1,000× with short reads and 100-400× with long reads for each of nine analyzed samples. We defined optimal strategies for integrating bulk short- and long-read sequencing for mutation detection and demonstrated that using donor-specific assemblies and human pangenome improved variant calling and extended mutation catalogs to challenging genomic regions. We benchmarked six duplex-seq technologies and showed that single-cell sequencing resolves cell type-specific mutational patterns and heterogeneity. Our results indicate that bulk, single-cell, and duplex analyses are complementary – and leveraging all three provides comprehensive characterization of mosaicism within a tissue. Together, these findings provide a roadmap for accurate, genome-wide somatic mutation discovery and analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2f0d2163eec9b7645db30c84ca9253a02faf7da" target='_blank'>
              Comprehensive benchmarking of somatic mutation detection by the SMaHT Network
              </a>
            </td>
          <td>
            A. Abyzov
          </td>
          <td>2025-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Medulloblastoma (MB) is the most prevalent malignant brain tumor in children, exhibiting clinical and genomic heterogeneity. Of the four major subgroups, Group 3 tumors (MYC-MB), display high levels of MYC and metastasis rates. Despite treatment with surgery, radiation, and chemotherapy, patients with Group 3 MB are more likely to develop aggressive recurrent tumors with poor survival. To examine resistance mechanisms in this study, we perform single nuclei multiome analysis of matched primary and recurrent tumors. Therapy resistant Medulloblastoma demonstrates an expanded persistent progenitor population. Additionally, distinct chromatin landscapes link to altered transcription and correspond with metabolic reprogramming. In vivo modeling of radiation resistance exhibits similar chromatin-based metabolic reprogramming focused on wild-type isocitrate dehydrogenase (IDH1) activity. IDH1 inhibition reverses resistance-mediated chromatin changes and enables radiation re-sensitization. Ultimately, these findings demonstrate the efficacy of single-cell multiome analysis in elucidating resistance mechanisms and identifying targetable pathways for MYC-driven medulloblastoma. MYC-driven medulloblastoma is highly aggressive and associated with poor survival. Here, the authors perform single nuclei multi-omic analysis of matched primary and recurrent tumours and identify potential resistance mechanisms and targetable pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fba49308e533a39e32c7d3dcd263f21e4a690d4" target='_blank'>
              Single-cell multi-omics identifies metabolism-linked epigenetic reprogramming as a driver of therapy-resistant medulloblastoma
              </a>
            </td>
          <td>
            Bethany L. Veo, Dong Wang, J. DeSisto, Angela M Pierce, Breauna Brunt, P. Bompada, A. Donson, Andrew Goodspeed, Kiara Smart, Nicholas Foreman, Nathan A. Dahl, Rajeev Vibhakar
          </td>
          <td>2025-11-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Transcriptional regulation is mediated by enhancers, yet how genetic perturbations alter enhancer activity and gene expression remains poorly understood. We developed UDI-UMI-STARR-seq, which integrates dual indexes and unique molecular identifiers, and combined it with RNA-seq to profile the effects of perturbations on enhancer activity and target gene expression. We applied this approach to a library of 253,632 fragments representing 46,142 cell type–specific candidate enhancers and assessed the impact of CRISPR/Cas9-mediated deletion of six transcription factors (or TFs; ATF2, CTCF, FOXA1, LEF1, TCF7L2, and SCRT1) with diverse regulatory roles. Across knockout lines, we identified responsive enhancers that were either repressed or induced, often through motifs such as the p53 family of TFs. Enhancer–gene mapping revealed TF-specific programs, including repression of Wnt/p53 targets with ATF2 or LEF1 loss, downregulation of the FIRRE locus with CTCF loss, and compensatory upregulation of RNA polymerase II regulators following FOXA1 depletion. A deep learning model trained on enhancer sequences recapitulated core principles of enhancer grammar, including cooperative motif syntax and the influence of flanking sequence context. Applying this framework to the neurodevelopmental disorder-associated 16p12.1 deletion identified responsive enhancers linked to genes involved in axon guidance, synaptic plasticity, and translational control, providing a scalable readout of enhancer dynamics generalizable to genetic mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e599253ce970a00d4735ed417209ae82c988f97" target='_blank'>
              Quantifying the impact of genetic mutations on enhancer dynamics
              </a>
            </td>
          <td>
            Maitreya Das, Deepro Banerjee, Ayaan Hossain, Matthew Jensen, S. Mogre, Jiawan Sun, Joseph Mao, Adam B Glick, Howard M Salis, S. Girirajan
          </td>
          <td>2025-10-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="One of the seminal discoveries from genetic studies of autism spectrum disorder and related neurodevelopmental disorders (NDD) has been that loss-of-function (LoF) mutations in many genes that impact chromatin and transcriptional regulation confer substantial liability to NDD. Haploinsufficiency of the epigenetic regulator POGZ represents one of the strongest such associations; however, little is known about the direct or indirect regulatory targets of POGZ, or the mechanisms by which loss of this chromatin modifier alters early neuronal development and synaptic functions. Here, we created an allelic series of CRISPR-engineered human induced pluripotent stem cell (hiPSC) clones harboring mono- and biallelic POGZ deletions. In hiPSC-derived neural stem cells (NSC) and Neurogenin 2-induced neurons (iN), POGZ LoF altered the expression of genes associated with critical cellular processes and neuronal functions, including synaptic and intracellular signaling and extracellular matrix organization. Our multiomics profiling also showed altered footprinting of critical transcription factors (e.g., activator protein 1 complexes) that were enriched at promoters of differentially expressed genes associated with synaptic function. To further interrogate the shared molecular changes in neuronal development associated with NDD and POGZ regulation, we compared our results to deletions of the transcription factor MEF2C and the sodium channel gene SCN2A that we generated in these same isogenic iN. These analyses revealed strong enrichment of extracellular matrix and intracellular signaling disruption associated with POGZ and MEF2C deletion, whereas POGZ and SCN2A haploinsufficiency exhibited shared transcriptional effects on gene modules enriched for NDD-associated genes with opposing regulatory effects. Notably, we also observed alterations to synaptic firing rate and neurite extension with biallelic deletions, but not heterozygous lines, suggesting subtle effects in neuronal development associated with haploinsufficiency. Overall, these shared molecular consequences suggest key points of convergence that connect epigenetic regulation to neuronal function in the etiology of neurodevelopmental pathologies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eed6138b4f1663b7339e409062cbd4f7a6fbbd84" target='_blank'>
              CRISPR-engineered deletion of POGZ alters transcription factor binding at promoters of genes involved in synaptic signaling
              </a>
            </td>
          <td>
            M. Moyses-Oliveira, Yating Liu, Serkan Erdin, Dadi Gao, Riya Bhavsar, Kiana Mohajeri, Kathryn O’Keefe, Philip M. Boone, Gabriela Xavier, Calwing Liao, Aiqun Li, Rachita Yadav, M. Salani, D. Lucente, Benjamin Currall, C. D. de Esch, D. J. Tai, D. Ruderfer, Kristen J. Brennand, James F. Gusella, M. Talkowski
          </td>
          <td>2025-10-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="Background Triple-negative breast cancer (TNBC) is the most aggressive breast cancer subtype; however, clinically approved prognostic biomarkers and therapeutic options remain limited. This study aimed to investigate tumor dependency genes to identify novel therapeutic targets for TNBC. Methods Tumor dependency genes for TNBC were identified using the The Cancer Dependency Map (DEPMAP) database. The TCGA-BRCA dataset was utilized to analyze the expression, survival associations, and pathway enrichment of these genes. Single-cell datasets were employed to explore cellular trajectories and biological functions within tumor dependency gene-associated cell subpopulations. Genomic sequencing was used to investigate the somatic mutational landscape influencing the infiltration abundance of the tumor dependency-associated subpopulation. The METABRIC dataset assessed the impact of the tumor dependency-associated subpopulation on radiotherapy, chemotherapy, and combination therapy outcomes. Potential drugs were identified using the Connectivity Map (CMAP). Colony formation experiment and the CCK-8 experiment were performed to validate the biological function of gene. Results Four tumor dependency genes (TDGs) were identified. These genes were highly expressed in TNBC and associated with poor prognosis. Enrichment analysis revealed their significant involvement in cell cycle-related pathways. Single-cell analysis demonstrated that the tumor dependency-associated subpopulation (TDAS), defined by these four genes, resided at the differentiation terminus of epithelial/tumor cells and was linked to energy metabolism and cell proliferation pathways. Crucially, patients with high TDAS infiltration abundance were found to be unsuitable for surgery alone and should receive combined radiotherapy or chemotherapy. Potential therapeutic agents targeting the TDAS were screened. And in vitro experiments confirmed the cell proliferation role of candidate genes. Conclusion This study identifies four potential TNBC biomarkers for assessing TDAS abundance, providing novel insights and strategies for personalized TNBC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f04c8aa1f75b0d7e0c49aabd5f19b6bf7360c1cf" target='_blank'>
              Integrating single-cell transcriptomics and whole-genome CRISPR CAS9 screen identifies a cell cluster associated with tumor dependency in triple-negative breast cancer
              </a>
            </td>
          <td>
            Su Liu, Shuo Wang, Guixin Wang, Yingxi Li, Zhigang Zhao, Yao Tian, Jun Cao
          </td>
          <td>2025-10-17</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Neoadjuvant chemotherapy (NAC) is a critical component of breast cancer treatment, but the molecular mechanisms underlying resistance remain poorly understood. This study aimed to identify transcriptomic changes associated with NAC resistance across four breast cancer subtypes: Luminal A, Luminal B/HER2-positive, Luminal B/HER2-negative, and Triple-Negative Breast Cancer (TNBC). RNA-seq analysis was performed on paired pre- and post-NAC breast cancer samples from 32 non-responders. Differentially expressed genes (DEGs) were identified, and functional enrichment analyses were conducted. Protein-protein interaction (PPI) networks were constructed to identify hub genes. Tumor microenvironment (TME) infiltration was estimated using deconvolution algorithms. The results revealed distinct gene expression profiles between pre- and post-NAC samples, with FOS and NR4A1 being common DEGs across all subtypes. Enriched pathways varied among subtypes, including signal transduction, estrogen biosynthesis, extracellular matrix organization, dendritic cell activation, and B cell activation. TME analysis showed increased infiltration of specific immune cell populations after NAC, including CD4 memory T cells, regulatory T cells, neutrophils, macrophages, and mast cells, varying by subtype. These findings suggest that NAC modulates gene expression, cellular activity, and TME interactions, potentially contributing to treatment resistance. Understanding the molecular determinants of NAC resistance is crucial for developing targeted therapeutic strategies and improving outcomes for breast cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e52a24e6d865b78f25fc1cec6097a262050bb3b1" target='_blank'>
              Molecular determinants of neoadjuvant chemotherapy resistance in breast cancer: An analysis of gene expression and tumor microenvironment
              </a>
            </td>
          <td>
            H. M. Guevara-Nieto, Carlos Orozco-Castaño, R. Parra-Medina, Jenny Nathaly Poveda-Garavito, J. Garai, Jovanny Zabaleta, Liliana Lopez-Kleine, A. Cómbita
          </td>
          <td>2025-10-14</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Abstract Evidence suggests that alternative RNA splicing (AS) plays a critical role in tumor biology and may contribute to the generation of tumor antigens. Here, we develop a method to detect AS in short-read single-cell 5′-RNA-sequencing data, allowing us to uniquely characterize the heterogeneity and dynamic changes in AS in individual cell types within the tumor microenvironment. We identify numerous splicing events specific to either cancer cells or stromal cell types or for triple-negative versus estrogen receptor-positive breast cancers (BCs). By correlating these splice events with expression of splicing regulators in individual cells, we also identify their potential mediators. For instance, we identify and functionally validate the Epithelial Splicing Regulatory Protein-1 (ESRP1) to drive AS in BCs responding to immune checkpoint blockade (ICB). Prioritization of splicing events based on their likelihood to represent tumor antigens reveals that their aggregated load also correlates with high immune activity in multiple cancers, while also predicting expansion of T cells in BCs receiving ICB and prolonging long-term survival of cancer patients treated with ICB. Collectively, our method provides a framework for analyzing AS in single-cell data and defines a key role for AS in the response to ICB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b79be77027ec8fdf591f4f79253b80b26a945fe0" target='_blank'>
              Single-cell mapping of alternative splicing linked to checkpoint immunotherapy response
              </a>
            </td>
          <td>
            Jieyi Xiong, O. Bricard, I. Arijs, J. Demeulemeester, Chen Gu, Bernard Thienpont, Danie Daaboul, A. Bassez, Oliver Bechter, J. Pozniak, H. Vos, I. Nevelsteen, Sofie Torfs, Sara Aibar Santos, Qing Lan, Yong Hou, Lore Van Oudenhove, G. Boons, Junbin Qian, Stein Aerts, Ann Smeets, J. Marine, Diether Lambrechts
          </td>
          <td>2025-11-13</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Glioblastoma (GBM) remains a formidable challenge in clinical settings due to limited treatments available. The surface protein CD133 marks glioblastoma stem cells (GSCs), cells capable of overcoming therapeutic pressures and correlate with more aggressiveness tumor phenotypes. In this study, we employed a CRISPR-Cas9 functional screen to deconvolute CD133 dynamics in tumors. This led us to establish that SOX2 is a key player in controlling the PROM1 gene, which in turn influences how cells react to stress factors, including those induced by chemoradiation treatment. The discoveries in this study shed light on the complex web of mechanisms that control the survival and resistance of GSCs, offering promising new avenues for targeting and potentially overcoming therapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2f508110c54ab3c78e02eb486850e272c3a091d" target='_blank'>
              CRISPR screen reveals SOX2 as a critical regulator of CD133 and cellular stress response in glioblastoma
              </a>
            </td>
          <td>
            N. Savage, Etienne Danis, C. Chokshi, Stefan Custers, Muhammad Vaseem Shaikh, P. Miletic, C. Venugopal, Kevin R. Brown, Rajeev Vibhakar, Jason Moffat, Sheila K. Singh
          </td>
          <td>2025-10-16</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Genome-wide association studies (GWAS) have identified nearly 100 loci associated with metabolic dysfunction-associated steatotic liver disease (MASLD), but the molecular functions of these variant alleles remain elusive, particularly when they occur in non-coding regions. Here we profiled the chromatin accessibility landscape of liver nuclei from MASLD individuals, and demonstrated these accessible genomic sites were bound by cell type-specific transcription factors (TFs) and enriched for MASLD risk variants, highlighting lineage- and disease state-specific regulation. Using a massively parallel reporter assay (MPRA), we identified hundreds of differential activity variants (DAVs) that operate in a cell type-specific manner or in a stimulus-dependent context by disrupting liver pathogenesis-associated transcriptional regulatory network. Integrative analyses combining liver eQTLs, chromatin looping, and single-cell CRISPRi screening linked these DAVs to functional target genes. Notably, we demonstrated that DAVs located near SLC22A3 and core regulators of triglyceride metabolism (APOA5, ANGPTL3, and LPL) loci modulate their gene expression and contribute to altered lipid metabolism and hepatic stellate cell activation. Furthermore, these DAVs exhibit predictive power in distinguishing MASLD disease risk. Together, these multimodal integration analyses provide insights into the regulatory mechanisms of MASLD progression driven by noncoding genetic risks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5934e58f3e02312c2db36030491cd91459f528f6" target='_blank'>
              Integrative functional genomics reveals transcriptional regulatory function of risk alleles for metabolic liver disease
              </a>
            </td>
          <td>
            Wenxiang Hu, Biying Zhu, Na He, Yang Xiao, Bin Chen, Chen Li, Ravi Mandla, Yifan Liu, Jiayu Zhang, Xiao Chang, Fulong Yu, M. Vujkovic, J. Lynch, Kyong-Mi Chang, V. M. V. Program, Bogdan Pasaniuc, Daniel J. Rader, Mitchell A. Lazar
          </td>
          <td>2025-10-23</td>
          <td>Research Square</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Whole-genome doubling (WGD) is a macro-evolutionary event that is both prevalent and prognostically significant in human cancers, particularly in high-grade serous ovarian carcinoma (HGSOC). Historically, WGD has been viewed as a consequence of widespread genomic instability, but recent advancements in single-cell sequencing (sc-seq) have reframed its role as a central, dynamic driver of tumor evolution. This review summarizes cutting-edge findings, demonstrating how WGD acts as a catalyst for a distinct evolutionary trajectory characterized by the rapid accumulation of chromosomal losses and the selection of highly adaptable clones. A key finding is the resolution of a biological paradox: WGD-driven chromosomal instability, which should provoke an immune response, is instead correlated with a profoundly immunosuppressive phenotype via the repression of key innate immune pathways. Finally, this review discusses the clinical implications of these discoveries, highlighting WGD’s potential as a predictive biomarker and a source of unique therapeutic vulnerabilities, paving the way for targeted strategies in advanced HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5292890396c5c0d49191edc0b87d5766e4f3e3ca" target='_blank'>
              Genome doubling as a dynamic driver of ovarian cancer evolution: insights from single-cell sequencing
              </a>
            </td>
          <td>
            Tianjiao Zhao, Tianshi Zhao, Dengyue Dong, Dengyue Dong
          </td>
          <td>2025-11-19</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Extensive studies have demonstrated the relationship between metabolic reprogramming and the tumor microenvironment. Here, we characterized the head and neck squamous cell carcinoma (HNSCC) evolutionary landscape using spatial metabolomics/transcriptomics, single-cell transcriptomics, and bulk multi-omics. Metabolic heterogeneity during HNSCC malignant transformation was identified, with significant enrichment in the purine metabolism. Integrating single-cell and bulk data, we developed a robust ligand-receptor-based signature (LRS) linked to NT5E, a key upstream regulator of purine metabolism, which served as an independent prognostic indicator. The low LRS subtype was associated with a high proportion of immune cell infiltration and improved response to immunotherapy. Notably, in vitro and in vivo experiments demonstrated that AMIGO2, a core molecule within the LRS, regulates tumor-associated purine metabolism, and that its downregulation suppresses tumor cell invasion and migration, inhibits myofibroblast differentiation, and promotes immune effector cell infiltration. Moreover, combining AMIGO2 targeting with anti-PD-1 therapy yielded superior efficacy. Consistent validation was also obtained in a clinical cohort of HNSCC and premalignancy patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2ca668a38f0d67d558807d21d034edd4de991a2" target='_blank'>
              Metabolomic and transcriptomic profiling of HNSCC identifies AMIGO2 as a therapeutic target modulating tumor microenvironment
              </a>
            </td>
          <td>
            Gan Liu, Xinfeng Yao, Yuchen Hou, Wentao Deng, Liyu Zhang, Shutong Li, Panpan Huang, Wenjiao Chang, Haiyan Huang, Lan Shi, Qianqian Zhou, Yanqing Wang, Jia Chen, Wenchao Gu, Xiaoling Ma, Yu Zhou, Honghong Liu, Shanshan Zhang, Zongcheng Yang
          </td>
          <td>2025-11-18</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea5bbf3daa9e6f41e402a48c9bc2bd5b7f13849c" target='_blank'>
              Systematic elucidation and pharmacologic targeting on non-oncogene dependencies in imatinib-resistant gastrointestinal stromal tumor
              </a>
            </td>
          <td>
            Prabhjot S. Mundi, A. Grunn, A. Kojadinovic, Charles Karan, Ronald B. Realubit, Cristina I. Caescu, H. Hibshoosh, Mahalaxmi Aburi, Mariano J. Alvarez, M.A. Ingham, D. Evans, S. Rothschild, Gary K. Schwartz, Andrea Califano
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Although genome sequencings have recognized many cancer-associated genes in hepatocellular carcinoma (HCC), distinguishing their functional effect remains challenging. Leveraging on a genome-wide CRISPR knockout (KO) screening, we uncovered spliceosome factors as major survival essential genes in HCC and up-regulations of ferroptosis suppressors [particularly glutamate-cysteine ligase catalytic subunit (GCLC)] in lenvatinib resistance. Our KO screen in patient-derived HCC organoid showed splicing factor 3b subunit 4 (SF3B4) to be top-ranked, conferring prosurvival signal in HCC organoid and driving tumorigenic potentials in both hepatic progenitor organoids and hydrodynamic tail vein injection HCC murine model. The combined RNA immunoprecipitation sequencing, long-read isoform sequencing, and transcriptome revealed characteristic splicing landscape regulated by SF3B4 and identified T-box transcription factor 3 (TBX3) variant TBX3+2a as a potent downstream effector. Our findings highlighted vital roles of SF3B4 in HCC cell survival and tumor progression, and the phenomenon of ferroptosis resistance in patients unresponsive to first-line agent lenvatinib.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ee6f7f5b50ee5eb699e0b5d0e4977d78a15779e" target='_blank'>
              Genome-wide CRISPR screen identifies splicing factor SF3B4 in driving hepatocellular carcinoma
              </a>
            </td>
          <td>
            Yue Guo, Mingjing Xu, Hua Xue, Xiaofan Ding, Alissa M. Wong, Na Lin, Dandan Pu, A. M. Wong, Xin Wang, Hui Zhao, Nathalie Wong
          </td>
          <td>2025-10-10</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/137a07d5d39a7f965a9829b9a10c7a07db82c57e" target='_blank'>
              Multi-omics-driven kinetic modeling reveals metabolic adaptations and vulnerabilities in BRCA1-deficient ovarian cancer
              </a>
            </td>
          <td>
            Ilias Toumpe, M. Masid, V. Hatzimanikatis, L. Miskovic
          </td>
          <td>2025-11-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc219a64ec3d466c88c4dfd11649123286e2032b" target='_blank'>
              Cross-Species Insights from ART-D to Uncover Evolutionarily Conserved Oncogenic Mechanisms
              </a>
            </td>
          <td>
            Yifan Guo, Jiadong Zheng, Yixin Wu, Kun Zhao, Dian Lv, Wenhan Liu, Mengling Wang, Jin-Yu Lu, Wenyan Xu, Xianping Wang, Xianjue Ma
          </td>
          <td>2025-10-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Targeting cancer cell plasticity through chromatin organization is an emerging research area, yet the molecular mechanisms that govern chromatin loop formation remain unclear. Here, we develop a CRISPR screen based on our engineered live-cell CTCF-cohesin contact reporters to identify regulators of chromatin loops. Our findings reveal that tousled-like kinase 2 (TLK2) functions as a key regulator of chromatin loop formation during the cancer stemness transition. Mechanistically, TLK2 phosphorylates DYNLL1, enhancing its interaction with CTCF to promote CTCF-cohesin hub formation at the KLF4 locus. Suppressing TLK2 impairs cancer stemness plasticity, sensitizes cancer cells to cytotoxic stress in vitro, and reduces lung metastases and enhances immunotherapy response in breast cancer mouse models. Clinically, elevated TLK2 expression correlates with poor prognosis in breast cancer patients. Collectively, these findings identify TLK2 as a potential therapeutic target for mitigating cancer stemness plasticity, highlighting chromatin loop-targeting therapy as a promising strategy to eradicate cancer stem cells. Aberration of chromatin organisation is linked to cancer progression and cancer cell plasticity. Here, the authors investigate changes in chromatin state during the transition of cancer cells to cancer stem cells and, using a CTCF-cohesin contact reporter CRISPR screen, identify TLK2 as a driver of plasticity via regulation of chromatin loop formation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/040d9374168c648d367bfc7319dadc6870704688" target='_blank'>
              Chromatin looping-based CRISPR screen identifies TLK2 as chromatin loop formation regulator in cancer stemness plasticity
              </a>
            </td>
          <td>
            Zifeng Wang, Fang Liu, Nana Chen, Jingjing Wu, Xinhao Li, Mouxiang Fang, Min Yan, Ji Zhang, Bing Deng, Lulu Wang, Xuan Wang, Meiling Liu, Deshun Zeng, Zhengzhi Zou, Bo Wang, Songyang Zhou, Bin He, Quentin Liu
          </td>
          <td>2025-10-21</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a8fc3f37c54ae77fa1e33d4bf2768934a457fd9" target='_blank'>
              Single-cell transcriptomic profiling of bam mutant tumor reveals germline heterogeneity and gcrf1 as a modulator in Drosophila germ cells
              </a>
            </td>
          <td>
            Zhipeng Sun, Yujun Zeng, T. Nystul, Guohua Zhong
          </td>
          <td>2025-10-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ecd9aa898a5f744ae181ea0c4ce6e138f5e70b4" target='_blank'>
              Comprehensive pan-cancer characterization of cancer-testis genes at single-cell resolution
              </a>
            </td>
          <td>
            Chunyang Fu, Shumin Yin, Linjin Li, Xueying Liu, Xiuyuan Jin, Ke Liu
          </td>
          <td>2025-11-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54aa0aad15817f7ad062fae9183727fb28323813" target='_blank'>
              Insights from pooled CRISPRi single-cell screens in K562 cells reveal gene functions, regulatory networks, and highlight opportunities and limitations
              </a>
            </td>
          <td>
            Hong Zhang, Peifen Zhang, E. Bindels, Eskeatnaf Mulugeta
          </td>
          <td>2025-10-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df4375fd9e9cdc4563ab3b711dfac8983eeba0ca" target='_blank'>
              Genome-Wide CRISPR/Cas9 Screening Reveals Lipid Metabolism and Inflammatory Signalling as Modulators of Ganoderic Acid DM Cytotoxicity
              </a>
            </td>
          <td>
            Nimo Abdullah, Jacqueline Lewis, Prakash Arumugam
          </td>
          <td>2025-10-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b84948bcfa1aec61d5686e9537d4f4d0902383f" target='_blank'>
              ALKBH1 drives codon-biased, pro-oncogenic translation and tumor microenvironment remodeling in glioma via tRNA wobble oxidation
              </a>
            </td>
          <td>
            A. Nakayashiki, Arata Nagai, Yuko Iwasaki, Abdulrahman Mousa, Moe Kumai, Munehiro Kimura, Shadi Al-Mesitef, Hiroshi Kanno, S. Yamashita, Daisuke Ando, Yoshiteru Shimoda, Masayuki Kanamori, Keisuke Goda, Hidenori Endo, Sherif Rashad, Kuniyasu Niizuma
          </td>
          <td>2025-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Therapeutic resistance remains a defining challenge in oncology, limiting the durability of current therapies and contributing to disease relapse and poor patient outcomes. This review systematically integrates recent progress in understanding the molecular, cellular, and ecological foundations of drug resistance across chemotherapy, targeted therapy, and immunotherapy. We delineate how genetic alterations, epigenetic reprogramming, post-translational modifications, and non-coding RNA networks cooperate with metabolic reprogramming and tumor microenvironment remodeling to sustain resistant phenotypes. The influence of the microbiome is highlighted as an emerging determinant of therapeutic response through immune modulation and metabolic cross-talk. By summarizing key regulatory circuits, We establishe a unified framework linking clonal evolution, metabolic adaptability, and tumor ecological dynamics. We further synthesizes novel therapeutic strategies that convert resistance mechanisms into therapeutic vulnerabilities, including synthetic lethality approaches, metabolic targeting, and disruption of stem cell and stromal niches. Advances in single-cell and spatial omics, liquid biopsy, and artificial intelligence are emphasized as transformative tools for early detection and real-time prediction of resistance evolution. This review also identifies major translational gaps in preclinical modeling and proposes precision oncology frameworks guided by evolutionary principles. By bridging mechanistic understanding with adaptive clinical design, this work provides an integrated roadmap for overcoming therapeutic resistance and achieving sustained, long-term cancer control.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5732b68dc3b5afcc6ec00f3b3bd38479653a277" target='_blank'>
              Drug resistance in cancer: molecular mechanisms and emerging treatment strategies
              </a>
            </td>
          <td>
            Jinxin Li, Jiatao Hu, Yiren Yang, Hanzhong Zhang, Ying Liu, Yu Fang, Le Qu, Anqi Lin, Peng Luo, Aimin Jiang, Linhui Wang
          </td>
          <td>2025-11-17</td>
          <td>Molecular Biomedicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 Background and Objective: Acute myeloid leukemia (AML) is a genetically heterogeneous disease, but even isogenic leukemic populations exhibit phenotypic variability. To determine how non-genetic, biological heterogeneity influences growth, therapy response, and adaptation to metabolic stress, we generated and deeply characterized isogenic human and mouse AML clones. By longitudinally tracking single-cell clones, we found that metabolic plasticity, not newly acquired mutations or changes in transcriptional programs, dictates cell growth and therapeutic response to venetoclax. These data highlight the central importance of biological heterogeneity in AML that is not dictated by genetic changes.
 Methods: We isolated 29 single-cell clones from the MLL-rearranged AML cell line MOLM13 and assessed longitudinal differences in baseline growth, adaptation to nutrient stress, and drug response. Clones were tested under glutamine or serum restriction and treated with cytarabine, daunorubicin, venetoclax, and azacitidine. Mitochondrial respiration was measured using Seahorse assays. To explore the genetic basis for phenotypic divergence, we performed RNA-seq and whole-exome sequencing on the most venetoclax-sensitive and -resistant clones. We validated findings in a primary mouse MLL-AF9 AML using both a barcoding approach to tracking clonal dynamics by NGS and with individually isolated and cultured clones.
 Results: Even without exogenous stress, baseline growth rates varied >3-fold among MOLM13 clones and remained stable over time. Nutrient stress further amplified heterogeneity: glutamine deprivation suppressed growth by 40–70% across clones, whereas 0.1% FBS had a more uniform suppressive effect. Glutamine restriction reduced basal oxygen consumption rates (OCR) in all clones but preserved maximal respiration after FCCP uncoupling, indicating metabolic adaptation via alternative substrates. Faster-growing clones displayed higher basal OCR and relative resistance to venetoclax, while slower-growing clones were more venetoclax-sensitive. Cytarabine responses showed an opposite trend, suggesting that drug sensitivity is growth rate-dependent. The IC50 of venetoclax varied by ~12-fold, compared to ~2-fold for daunorubicin, indicating a context-dependent influence of clonal heterogeneity on therapy response. Next, we compared the genetic and transcriptional profile of clones that displayed the most sensitivity and resistance to venetoclax. RNA-seq showed no differential expression of BCL2-family genes between sensitive and resistant clones, and whole-exome sequencing revealed hundreds of unique, non-overlapping non-synonymous variants per clone, but no recurrent driver mutations were linked to the phenotype. These results suggest non-genetic mechanisms underlie observed phenotypic diversity. Finally, we extended our findings using the primary mouse MLL-AF9 AML. Using a lentiviral barcoding approach, we again observed interclonal heterogeneity and changes in clonal representation under glutamine restriction and venetoclax exposure. We then isolated and grew individual MLL-AF9 clones, following a similar approach to our work with the MOLM13 cells, and observed interclonal variability in OCR and drug sensitivity, confirming our findings in an independent, primary AML.
 Conclusions: Our study reveals substantial biological heterogeneity among isogenic AML clones, with stable differences in growth, metabolic plasticity, and drug response. These phenotypes were not attributable to known driver mutations, which indicate that non-genetic factors may be involved in clonal adaptation. Clonal traits were context-dependent: venetoclax and glutamine restriction revealed the most heterogeneity, suggesting that selective pressures shape phenotypic diversity. These findings highlight the need to consider isogenic clonal variability when interpreting experimental results or designing therapies and highlight that functional heterogeneity may be an underappreciated driver of therapeutic response in AML.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffb7687231df66f3626536a54d60f11393391a61" target='_blank'>
              Clonal heterogeneity of acute myeloid leukemia under nutrient and chemotherapy stress
              </a>
            </td>
          <td>
            Christina Mayerhofer, Eve Crompton, Ernst Mayerhofer, Trine A. Kristiansen, Tsuyoshi Fukushima, Sam Keyes, A. Sharda, Dan Li, Carmen Da Silva, Deumaya Shrestha, Anna Kiem, T. Feuchtinger, David T. Scadden, P. Miller
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20a5274e29358d261ef02cb6d42094a3acbdab40" target='_blank'>
              Signatures in CRISPR Mutational Spectra Reveal Role and Interplay of Genes in DNA Repair
              </a>
            </td>
          <td>
            Colm Seale, Marco Barazas, R. van Schendel, M. Tijsterman, Joana P. Gonçalves
          </td>
          <td>2025-11-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="


 Introduction: Extensive characterization of the genetic landscape in AML has revealed a set of driver mutations sufficient to initiate leukemic transformation. However, some cases do not contain any of these known drivers and thus pose a clinical challenge for prognostication and targeted therapy. To address this, we employed a novel, proximity ligation based, linked-read whole genome NGS assay to identify single nucleotide variants (SNVs), short Insertions/Deletions (InDels), copy number variation (CNV), structural variants (SVs) and 3D genome contact maps in AML. By employing this approach, we investigate whether comprehensive detection of genetic alterations in conjunction with insights into the epigenetic state of leukemic cells can reveal important biological information in cases lacking canonical driver mutations.
 Methods: We curated a cohort of patient samples where concomitant targeted sequencing by NGS and conventional karyotyping showed no detectable driver mutations. Further, we included control specimens with either known AML drivers as well as remission specimens after allogeneic transplantation. We utilized a multi-purpose NGS assay (Dovetail® LinkPrep™) to detect SNVs, InDels, CNVs, SVs and 3D genome contact maps. Peripheral blood or bone marrow cells were thawed and immediately fixed, subjected to crosslinking and in situ tagmentation, followed by library prep. Libraries were sequenced on Illumina sequencers with a target of 400M reads per library. Data analysis was performed using both published and custom tools to call SNVs/InDels (DeepVariant), Copy Number estimates (Purple), Structural Variants (HiC-Breakfinder, DTG-SELVA) and 3D contact maps (Juicebox).
 Results: We sequenced whole genome libraries to achieve an average coverage of 50.4x (range 41.1x - 58.1x) showing a long-range linkage percentage of 26.2% (range 17.9% - 31.8%). SNVs and short InDels, such as likely pre-leukemic mutations in DNMT3A, TET2, and mutations in GATA2 were detected reliably. Of note, the linked-read nature of this assay allowed us to perform phasing in a case with two TET2 mutations and confirm the compound heterozygous nature of these two TET2 mutations at a distance of 24kb. Further, known AML drivers in control specimens in genes such as NPM1, FLT3 and KRAS were reliably detected with variant allele frequencies as detected by targeted sequencing. Surprisingly, however, we found new structural variants in half of our cohort, including the diagnostic specimens with no detectable driver mutations, despite all specimens having no chromosomal abnormalities by conventional karyotyping. These new SVs included large intrachromosomal inversions and unbalanced translocations. We validated the presence of all SVs in sorted leukemic blasts and absence in sorted T cell fractions by PCR and Sanger sequencing. We subsequently examined the 3D genome contact maps and performed RNA-Sequencing to determine the epigenetic state and potential biological consequences across these novel breakpoints. We found several neoloops connecting enhancers with promoters across these breakpoints in 3D contact maps, and gene expression dysregulation proximal to the SVs by RNA-Seq. When we evaluated the expression of these putative driver genes across a cohort of healthy donors and AML patients, we found lack of expression in healthy cells but significantly increased expression in 9-10% of patients with AML (n=777 patients). Finally, we also identified an undetected structural variant in engrafted donor cells from a patient with myelodysplastic neoplasms (MDS) in remission after allogeneic transplantation, highlighting the potential to assay SVs in the context of non-leukemic hematopoiesis as well.
 Conclusion: In this exploratory cohort, we find several otherwise undetected structural variants in AML cases without known driver lesions using a proximity ligation based, linked-read whole genome NGS assay while maintaining our ability to detect recurrent, known driver lesions. This approach allowed us to interrogate the biological consequences of these genomic alterations by leveraging the ability to not only detect genomic variants but also analyze the epigenetic state through 3D genome contact maps. Our results imply that our current understanding of the genomic landscape in AML might be underestimating a sizable number of structural variants that are difficult to detect using standard assays, implying putative novel driver genes in AML pathogenesis.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f033c80701d9a0710865b0dce2b79cfc9a9b774" target='_blank'>
              Sensitive detection of novel structural variants and 3D chromosome conformation reveals likely novel drivers including enhancer hijacking in AML
              </a>
            </td>
          <td>
            Thomas Koehnke, A. Ediriwickrema, Mital S. Bhakta, Alexander Fortuna, Charu Tiwari, Zackary Sanborn, Tian Zhang, Lisa Munding, Ravi Majeti
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Metastatic uveal melanomas are highly resistant to all existing treatments. To identify actionable vulnerabilities, we conducted a CRISPR-Cas9 knockout screen using a library composed of chromatin regulators. We revealed that the lysine methyltransferase, SETDB1, plays a critical role in metastatic uveal melanoma cell proliferation and survival. Functionally, SETDB1 deficiency induces a DNA damage response, senescence-like state and growth arrest. Knockdown of SETDB1 is associated with a decreased expression of genes related to replication and cell cycle. Moreover, deficiency in CDC6, an essential regulator of DNA replication, phenocopies SETDB1 inhibition. Using a pre-clinical model, we further demonstrated that anti-SETDB1 therapy impairs tumor growth in vivo. Therefore, we not only provide evidence that SETDB1 plays a critical role in metastatic uveal melanoma cell growth, but we also identify SETDB1 as a novel relevant therapeutic target for the treatment of metastatic uveal melanoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f8f6bc0103a97e7d0bc4a345b868d6b8a6d6854" target='_blank'>
              SETDB1 is critically required for uveal melanoma growth and represents a promising therapeutic target
              </a>
            </td>
          <td>
            Imène Krossa, Céline Pisibon, Y. Cheli, K. Bille, Mélanie Dalmasso, Sabah Hamadat, Chrystel Husser, Marie Irondelle, Julien Cherfils-Vicini, Frédéric Soysouvanh, S. Nahon-Estève, Arnaud Martel, Sandra Lassalle, J. Caujolle, C. Maschi, Stéphanie Baillif, Dan Hasson, Saul Carcamo, Andrew E. Aplin, Irwin Davidson, Emily Bernstein, Valeria Naim, R. Ballotti, C. Bertolotto, Thomas Strub
          </td>
          <td>2025-10-24</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Cancer-associated fibroblasts (CAFs) are central architects of the prostate cancer (PCa) microenvironment, yet their phenotypic diversity and druggable vulnerabilities remain largely uncharted. Here, we present an integrative multi-omics characterization of primary ex vivo CAFs from seven treatment-naïve PCa patients. Using single-cell RNA sequencing (scRNA-seq), we uncover substantial transcriptional heterogeneity among CAFs, with distinct gene expression programs related to extracellular matrix remodeling, inflammation, immune modulation, and metabolic reprogramming. This phenotypic diversity was further supported by variable expression of canonical stromal markers, including FAP, SULF1, VIM, CAV1, and αSMA. Transcription factor network analysis revealed SOX, FOX, and STAT3 family members as key regulators of pro-tumorigenic CAF states. To probe therapeutic vulnerabilities, we performed high-throughput drug sensitivity and resistance testing (DSRT) across 396 oncology compounds. CAFs exhibited broad sensitivity to multikinase inhibitors, with dasatinib, midostaurin, and FGFR inhibitors (AZD4547, erdafitinib) emerging as top stromal-directed candidates. These findings underscore the plasticity of prostate CAFs and reveal actionable vulnerabilities, supporting the development of targeted stromal therapies to disrupt tumor-stroma interactions in PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0342abe6ecbf94b88ae457a3f680a1e8d054b454" target='_blank'>
              Molecular profiling of ex vivo prostate cancer CAF models captures stromal heterogeneity and drug vulnerabilities
              </a>
            </td>
          <td>
            Frida Rantanen, A. Murumägi, Mariliina Arjama, K. Välimäki, Elina Multamäki, T. Mirtti, Antti S. Rannikko, T. Pellinen, Daniela Ungureanu, Olli Kallioniemi
          </td>
          <td>2025-11-06</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Background Acetyl-CoA carboxylase alpha (ACACA), a crucial rate-limiting enzyme governing de novo biosynthesis of fatty acids, drives oncogenic metabolic reprogramming in diverse malignancies. However, the multiomics investigation and immunological implications of ACACA across cancers remain unclear. Methods We performed a comprehensive pan-cancer analysis of ACACA via transcriptomic, proteomic, and clinical data from The Cancer Genome Atlas (TCGA), Clinical Proteomic Tumor Analysis Consortium (CPTAC), and the Human Protein Atlas (HPA) databases. Then, single-cell RNA sequencing acquired from the Gene Expression Omnibus (GEO) database was employed to map the expression pattern of ACACA in the tumor microenvironment (TME). Subsequently, functional validation experiments were conducted in lung cancer and sarcoma cells. Results High ACACA expression was associated with poor survival in various cancers, particularly those exhibiting dysregulated lipid metabolism. Immune profiling revealed that elevated ACACA expression was associated with low infiltration of CD8+ T cells and activated natural killer (NK) cells. Single-cell analysis of lung adenocarcinoma revealed that ACACA was expressed predominantly within malignant cells and contributed to an immunosuppressive microenvironment through migration inhibitory factor (MIF) signaling and the extracellular matrix (ECM) remodeling pathway. Furthermore, in vitro studies demonstrated that ACACA inhibition suppresses fatty acid synthesis and tumor growth in lung cancer and sarcoma cells. Conclusions Our study establishes ACACA as a key metabolic regulator that links lipid metabolism to immune evasion and drug resistance, highlighting its potential as a promising therapeutic target across cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ef9b3c72fa2e6ad7dbbe3729933ef811be46587" target='_blank'>
              Integrated single-cell and bulk RNA sequencing analysis reveals ACACA as a potential prognostic and immunotherapeutic biomarker across cancers
              </a>
            </td>
          <td>
            Haihua He, Zhen Zhang, Leifeng Chen, Fushan Gao, Yuze Wu, Lina Yi, Fei Shao, Yibo Gao, Jie He
          </td>
          <td>2025-10-15</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="T-prolymphocytic leukemia (T-PLL) is a rare mature T-cell neoplasm with poor prognosis that mainly affects elderly people. Alemtuzumab is widely considered as first-line therapy, but almost all patients relapse within one year, if not consolidated by an allogeneic stem cell transplantation. The improved understanding of T-PLL-specific molecular pathomechanisms gained over the last years suggested new potential therapeutic targets (epigenetic dysregulation, defective DNA damage response, aberrant cell cycle regulation, dysregulated prosurvival pathways), which were recently evaluated in a preclinical drug response study. In addition, existing genomewide T-PLL gene expression profiles enabled the identification of three robust T-PLL gene expression subgroups differing in cellular processes like immune response, cellular respiration, cell proliferation, apoptosis, or migration. So far, these T-PLL gene expression subgroups were not considered in preclinical drug response analyses, but recently published drug response screens and corresponding already publicly available gene expression profiles offer the great opportunity to integrate both data types. Therefore, we computationally analyzed samples from 34 T-PLL patients of two comparable cohorts for their response to in total 11 drugs. No T-PLL subgroup-specific differences or sex differences in response to the tested drugs were found. With respect to the underlying drug dosage schemes, venetoclax and cladribine were most effective in erasing T-PLL cancer cells among both cohorts. Also dinaciclib, idasanutlin, romidepsin, and KRT-232, which were only tested in one of both cohorts, were very effective for all or most of the T-PLL patient samples. For the three drugs bendamustine, cladribine, and fludarabine, which were only effective in a subset of the T-PLL samples, an exploratory differential gene expression analysis predicted drug-specific genes that distinguished between strongly and not strongly responding samples. In-depth annotation and literature analyses showed that many of these genes are known to play a role in leukemias or other types of cancer. Many of these genes were also confirmed by a direct correlation analysis between gene expression levels and drug responses. The absence of T-PLL gene expression subgroup-specific drug responses across the tested drugs can be important for the design of future drug screens and may ease potential clinical trials. Genes associated with drug-specific responses could be useful for improved patient stratification and may help to characterize molecular settings associated with effective responses. Not applicable.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/660651553560497bd01926f03d350d88fe9fa7de" target='_blank'>
              Comparative preclinical drug response analyses of T-prolymphocytic leukemia reveal no differences between known gene expression subgroups
              </a>
            </td>
          <td>
            N. Mikhaylenko, T. Braun, S. Timonen, S. Mustjoki, M. Herling, Michael Seifert
          </td>
          <td>2025-10-27</td>
          <td>Biology Direct</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f0d2028082631983321af009576821353c38710" target='_blank'>
              Pharmacological Induction of Irreversible Senescence in Drug-Tolerant Persister Cells Prevents Tumor Relapse
              </a>
            </td>
          <td>
            Bowen Wang, Yun Zhi, Pengqi Wang, Wenbo Guo, Jin Gu, Hanqing He, Kezhang He, Wei Zhou, Ting Wang, Nana Huang, Haixia Yang, Dan Wang, Tianhua Ma, Sheng Ding
          </td>
          <td>2025-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b81b381c14e5b1230a77d230dfeaa5698f06f2cc" target='_blank'>
              Characterizing resistance in prostate cancer at a single cell level with hormonal treatment and epigenetic inhibitors.
              </a>
            </td>
          <td>
            Morgan Howells, Priyadarsini Gangadharannambiar, Calum McMullen, Francesco Crea, W. Bacon, Colette Christiansen
          </td>
          <td>2025-11-03</td>
          <td>The Journal of pharmacy and pharmacology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a highly aggressive brain tumor with limited treatment options and poor survival. Mitochondrial dysfunction and metabolic reprogramming, particularly the Warburg effect, are increasingly recognized as critical drivers of GBM progression. Here, we integrated single-cell (scRNA-seq) and bulk RNA-sequencing (bulk RNA-seq) data to comprehensively examine mitochondria-associated genes in GBM. We identified differentially expressed mitochondrial genes with prognostic significance and constructed a 9-gene risk signature—ACOT7, THEM5, MTHFD2, ABCB7, PICK1, PDK3, ARMCX6, GSTK1, and SSBP1—using LASSO and Cox regression. This signature robustly stratified patients into high- and low-risk groups in both The Cancer Genome Atlas (TCGA) and Chinese Glioma Genome Atlas (CGGA) cohorts, remaining an independent prognostic factor in multivariate analyses. Time-dependent ROC AUCs were 0.729, 0.813, and 0.828 at 1, 2, and 3 years in TCGA, and 0.597, 0.650, and 0.546 in CGGA. A nomogram integrating the signature with clinical variables achieved AUCs of 0.649, 0.820, and 0.854 at 1, 3, and 5 years, with good 1/2/3-year calibration. Functional enrichment and clustering analyses revealed distinct metabolic phenotypes and survival differences between subtypes. Single-cell pseudotime analysis showed a transition from oxidative phosphorylation to glycolysis in malignant cells, aligning with the Warburg effect and implicating metabolic reprogramming in immune modulation. Our findings underscore the prognostic value of mitochondria-associated genes and suggest potential therapeutic targets for disrupting GBM metabolism. Overall, these results establish a mitochondria-centric prognostic model supported by single-cell context; findings are hypothesis-generating and warrant prospective and experimental validation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1def4820c25938da49857764d61ab2d3606bd5c5" target='_blank'>
              Comprehensive analysis of mitochondria-associated genes in glioblastoma via single-cell and bulk RNA sequencing
              </a>
            </td>
          <td>
            Fangyuan Wang, Ao Huo, Minghe Li, Chenglin Zhao, Yupei Guo, Zhou Jing, Qingyi Wang, Bingqi Lu, Hao Li, Wulong Liang, Weihua Hu, Xudong Fu
          </td>
          <td>2025-10-27</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 Background Acute myeloid leukemia (AML) is the most common adult acute leukemia. The traditional “3+7” induction regimen with anthracycline and cytarabine remains a standard therapy, but 30%-40% of patients develop drug resistance to anthracyclines, leading to relapse. Refractory/relapsed AML has dismal survival rates, posing a major challenge. Elucidating mechanisms of therapy resistance and relapse is crucial for improving outcomes in high-risk AML.
 Methods ATAC-seq analyzed chromatin accessibility of anthracycline-sensitive and -resistant AML cells, focusing on METTL3 locus. RNA-seq, qRT-PCR and Western blot measured METTL3 expression in cell lines and primary cells. METTL3 was regulated by small-molecule inhibition, shRNA knockdown and overexpression to study its effect on m6A modification and phenotypes in resistant cells. MeRIP-seq and RNA-seq identified METTL3 target genes in shMETTL3-transfected resistant cells. MeRIP-qPCR validated m6A-modified mRNAs and actinomycin D assays tested RNA stability. BMI1 and METTL3 inhibitors were used to explore their functional relationship in drug resistance.
 Results
 In anthracycline-sensitive AML patients, over 50% of the global chromatin accessible regions are enriched in promoter regions. Conversely, in anthracycline-resistant AML patients, promoter-accessible regions account for less than 25% of the total. Treatment with anthracyclines induces changes in promoter chromatin accessibility in only approximately 5% of the genome in sensitive patients, whereas this proportion reaches approximately 25% in resistant patients. Notably, METTL3 exhibits significantly elevated chromatin accessibility in anthracycline-resistant AML cells. Following anthracycline exposure, METTL3 accessibility is further enhanced in resistant cells but decreased in sensitive cells. m6A modification was significantly upregulated in AML drug-resistant cells, accompanied by consistently higher METTL3 expression. Integrative multi-omics analyses identified HOXA9 as a key transcriptional regulator of METTL3, with HOXA9 positively modulating METTL3 expression. Clinical datasets (TCGA and GSE6891) revealed that elevated HOXA9 expression was significantly associated with poor prognosis in AML patients. HOXA9 levels were markedly higher in drug-resistant and relapsed/refractory (R/R) AML cells. HOXA9 knockdown in resistant cells suppressed proliferation and induced apoptosis. Dual-luciferase reporter assays confirmed that HOXA9 specifically bound to the “TAACGTG” motif within CpG island 3 of the METTL3 promoter.
 Mechanistically, reducing METTL3 expression in drug-resistant cells decreased m6A modification, inhibited cell proliferation, and triggered apoptosis. Conversely, METTL3 overexpression in sensitive cells enhanced m6A modification, promoted proliferation, suppressed apoptosis, and augmented doxorubicin resistance. METTL3 promoted m6A modification of BMI1 mRNA, stabilizing BMI1 transcripts and elevating BMI1 expression in anthracycline-resistant cells.
 BMI1 was overexpressed in anthracycline-resistant cells and correlated with poor prognosis in AML patients. Inhibiting BMI1 in resistant cells induced dose-dependent proliferation arrest and apoptosis, while restoring sensitivity to anthracyclines.
 Conclusion Our study demonstrates that chromatin accessibility in the promoter regions of anthracycline-resistant AML cells is significantly decreased compared to anthracycline-sensitive counterparts. Anthracycline elicits more pronounced changes in promoter chromatin accessibility in resistant cells than in sensitive cells, underscoring a heightened involvement of promoter-level epigenetic regulation in anthracycline-resistant AML. METTL3 exhibits both upregulated expression and enhanced chromatin accessibility in anthracycline-resistant AML cells relative to sensitive cells. Furthermore, anthracycline exposure leads to a further increase in chromatin accessibility at the METTL3 promoter in resistant cells, suggesting dynamic epigenetic modulation of this key m6A regulator during drug resistance. Our study demonstrated that The HOXA9/METTL3/BMI1 axis promotes anthracycline resistance in acute myeloid leukemia through mRNA m6A modification and suggested the therapeutic potential of targeting the METTL3/BMI1 axis in the treatment of anthracycline-resistant AML.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/266257a13a97162936fed07815814a1d0db44fc4" target='_blank'>
              HOXA9/METTL3/BMI1 axis is required for promoting anthracycline resistance in acute myeloid leukemia cells by modulating BMI1 expression through  mRNA m6A modification
              </a>
            </td>
          <td>
            Hao Wang, Daihong Liu, Liping Dou
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Therapeutic resistance remains a critical barrier in oncology, frequently leading to cancer relapse after initial treatment response. Growing evidence suggests the presence of drug-tolerant persisters (DTPs), a rare subpopulation of cancer cells that survives chemotherapy by entering a reversible specific adaptation. Unlike classical cell resistance, the DTP phenotype is independent of genetic changes and maintained through dynamic regulatory mechanisms. DTPs are phenotypically heterogeneous and can exhibit stem-like and quiescent cell phenotypes, non- or slow proliferation, and remarkable plasticity due to a di-pause-like state and executing epithelial–mesenchymal transition (EMT) or transdifferentiation processes. Despite advances in research, the molecular mechanisms underlying DTPs’ biology and their role in cancer relapse remain only partially understood. The review summarizes the current progress in processes that lead to the acquisition of cellular persistence status, which, in turn, constitute areas of vulnerability that can be exploited in cancer therapy. We highlight anti-DTP therapeutic strategies, including epigenetic modification, cell signaling and transcriptional regulation, metabolic reprogramming, and modification of cell interactions within the tumor microenvironment. Furthermore, we focus on the potential role of nanomaterials in the combat against DTPs. Nanoparticles not only act as part of the drug delivery process, enabling precise DTP targeting and enhancing intracellular drug accumulation, but their intrinsic properties can also be used to eradicate DTPs directly or by enhancing the effectiveness of other therapeutic strategies. The integrated approach offers strong potential to eliminate tumor persistence, prevent recurrence, and improve long-term patient outcomes beyond conventional therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/495b98c4c7bc3661dae9aa58d4006e54132b7ae3" target='_blank'>
              Targeting Drug-Tolerant Persister Cancer Cells: Can Nanomaterial-Based Strategies Be Helpful for Anti-DTP Therapies?
              </a>
            </td>
          <td>
            Prachi Ghoderao, Eliza Kwiatkowska-Borowczyk, Sanjay Sahare, H. Dams-Kozlowska
          </td>
          <td>2025-11-01</td>
          <td>Pharmaceutics</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Background The disease burden of acute myeloid leukemia (AML) continues to pose a significant public health challenge globally. Mitochondria play a critical role in tumor development and progression by influencing bioenergetics, biosynthesis, and signaling pathways. However, the prognostic significance and therapeutic implications of mitochondrial function in AML warrant further investigation. Methods We integrated mitochondrial gene expression data with bulk RNA sequencing to identify key mitochondrial genes associated with AML. A total of fourteen machine learning algorithms were employed, yielding 148 unique combinations. The best-performing model was utilized to develop a MitoScore, which was then combined with clinical variables to establish a MitoScore-based nomogram. Additionally, single-cell sequencing data were analyzed to assess the impact of key mitochondrial genes on immune cells. Samples were classified into low-risk and high-risk groups based on MitoScore, allowing for a comparative analysis of clinical features, biological mechanisms, copy number variations, tumor burden, immune infiltration, immune function, and drug sensitivity between the two groups. Results Specific expression patterns of mitochondrial genes were observed in T cell subsets and at various developmental stages of AML. Samples were classified into low-risk and high-risk groups based on MitoScore. The high-risk MitoScore group exhibited a worse prognosis, with enriched biological processes and molecular pathways associated with immune response, a higher frequency of gene mutations linked to poor outcomes, increased immune cell infiltration, enhanced immune function, upregulated immune checkpoint gene expression, and greater sensitivity to cyclophosphamide and venetoclax. Conclusions This robust machine learning framework underscores the potential of MitoScore as a tool for stratified prognostic assessment and personalized treatment planning in AML patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fffed46a555b2478189a76e93265c850b1501265" target='_blank'>
              Integrated bioinformatic analysis and machine learning developed a prognostic model based on mitochondrial function for acute myeloid leukemia
              </a>
            </td>
          <td>
            Xing-biao Chen, Weijun Ling, Zhehan Yang, Xinyi Chen, Zi-yuan Lu
          </td>
          <td>2025-10-23</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="CRISPR-based gene activation (CRISPRa) has emerged as a promising therapeutic approach for neurodevelopmental disorders (NDD) caused by haploinsufficiency. However, scaling this cis-regulatory therapy (CRT) paradigm requires pinpointing which candidate cis-regulatory elements (cCREs) are active in human neurons, and which can be targeted with CRISPRa to yield specific and therapeutic levels of target gene upregulation. Here, we combine Massively Parallel Reporter Assays (MPRAs) and a multiplex single cell CRISPRa screen to discover functional human neural enhancers whose CRISPRa targeting yields specific upregulation of NDD risk genes. First, we tested 5,425 candidate neuronal enhancers with MPRA, identifying 2,422 that are active in human neurons. Selected cCREs also displayed specific, autonomous in vivo activity in the developing mouse central nervous system. Next, we applied multiplex single-cell CRISPRa screening with 15,643 gRNAs to test all MPRA-prioritized cCREs and 761 promoters of NDD genes in their endogenous genomic contexts. We identified hundreds of promoter- and enhancer-targeting CRISPRa gRNAs that upregulated 200 of the 337 NDD genes in human neurons, including 91 novel enhancer-gene pairs. Finally, we confirmed that several of the CRISPRa gRNAs identified here demonstrated selective and therapeutically relevant upregulation of SCN2A, CHD8, CTNND2 and TCF4 when delivered virally to patient cell lines, human cerebral organoids, and a humanized mouse model of hTcf4. Our results provide a comprehensive resource of active, target-linked human neural enhancers for NDD genes and corresponding gRNA reagents for CRT development. More broadly, this work advances understanding of neural gene regulation and establishes a generalizable strategy for discovering CRT gRNA candidates across cell types and haploinsufficient disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c894b7f9e1d5db5f69bf147d03e982a27eebec4" target='_blank'>
              Large-scale discovery of neural enhancers for cis-regulation therapies
              </a>
            </td>
          <td>
            Troy A. McDiarmid, Nicholas F. Page, Florence M. Chardon, R. Daza, George T. Chen, Michael Kosicki, Lucas M. James, Hannah C. Nourie, Dianne Laboy-Cintrón, Arthur S. Lee, , Diego Calderon, Jean‐Benoît Lalanne, Beth K. Martin, Kyle D. Fink, Michael Talkowski, A. Muotri, Benjamin D. Philpot, Len A. Pennacchio, D. Geschwind, Stephan J. Sanders, N. Ahituv, Jay Shendure
          </td>
          <td>2025-11-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="HPV-negative (−) head and neck squamous cell carcinoma (HNSCC) is a highly heterogeneous cancer characterized by high mutational burden, an immunosuppressive microenvironment, and poor response to standard therapies. These features highlight the urgent need for novel and more effective treatment strategies. Drug sensitivity prediction was performed using integrated datasets from TCGA, GDSC, and CCLE. To assess the therapeutic potential and underlying mechanisms of combining the CDK inhibitor AZD5438 with the MEK1/2 inhibitor PD0325901, we employed a comprehensive panel of HNSCC models, including established cell lines, orthotopic mouse tumor models, and patient-derived organoids (PDOs). Lipid nanoparticles (LNPs) were engineered to co-deliver both agents into the same cancer cell populations. The tumor secretome was profiled using biotinylation coupled with liquid chromatography-mass spectrometry (LC-MS). Molecular alterations were examined by immunofluorescence, immunohistochemistry, ELISA, flow cytometry, and Western blot. Our bioinformatics analysis identified AZD5438 and PD0325901 as two of thirteen candidate drugs whose sensitivity is consistently associated with the five most frequently mutated genes in HPV (−) HNSCC. Notably, among these candidates, AZD5438 and PD0325901 exhibited the lowest correlation in their sensitivity profiles, suggesting complementary mechanisms of action. In experimental models, the combination of AZD5438 and PD0325901 not only outperformed either monotherapy in suppressing tumor growth but also augmented CD8⁺ T cell-mediated antitumor immunity by promoting caspase-8/gasdermin E-dependent pyroptosis. Furthermore, in both orthotopic tumor-bearing mice and PDOs, the LNP-encapsulated drug combination produced significantly greater therapeutic efficacy compared with the free drug formulation. Our findings indicate that the combination of AZD5438 and PD0325901 holds therapeutic potential for the treatment of HPV (−) HNSCC, particularly in tumors with a high mutational burden. By targeting complementary pathways, this combination may improve treatment outcomes in this aggressive cancer subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37c2bb6bc0e733d8972e63698f07f227976cd16a" target='_blank'>
              Dual CDK and MEK Inhibition potentiates CD8+ T cell-mediated antitumor immunity by inducing pyroptotic cell death in high-mutational head and neck cancer
              </a>
            </td>
          <td>
            Fanghui Chen, Fan Yang, David O. Popoola, Jianqiang Yang, Chris Tang, Alexis Payne, Lynn Zhang, N. C. Schmitt, Jin Xie, Nabil F. Saba, Yamin Li, Yong Teng
          </td>
          <td>2025-11-06</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Bladder cancer (BLCA) is the predominant type of urothelial carcinoma in urinary system, and resistance to cisplatin-based chemotherapy substantially worsens clinical outcomes, presenting a major therapeutic obstacle. In this study, we integrated genome-wide association study (GWAS) data with expression quantitative trait loci (eQTL) analyses, and applied Mendelian randomization (MR) to assess the causal effects of eQTLs from 19,942 genes on BLCA. By incorporating scRNA-seq data, our study also identifies differentially expressed genes (DEGs) in cisplatin-resistant BLCA cells and examined their causal associations with BLCA, aiming to elucidate genetic drivers of chemoresistance and tumor progression. Through this integrated approach, we identified the eQTL of the ARHGEF12 gene as a key mediator of cisplatin resistance. Bioinformatic analysis revealed that elevated ARHGEF12 expression was strongly associated with activation of the PI3K/Akt signaling pathway. To define ARHGEF12’s role in cisplatin resistance, we established a cisplatin-resistant UM-UC-3/DDP model. Silencing ARHGEF12 markedly reduced chemoresistance, increased apoptotic cell death, and induced pronounced morphological changes. Pharmacological modulation with the ROCK inhibitor Y-27632 and a rescue assay with the Akt activator SC79 supported a model in which ARHGEF12 drives chemoresistance via RhoA/ROCK-dependent activation of the PI3K/AKT axis. This study is the first to integrate MR with single-cell transcriptomics to explore the genetic contribution to cisplatin resistance in BLCA. Our results uncover a novel mechanistic role of ARHGEF12 in BLCA progression and chemoresistance and suggest it as a potential therapeutic target for precision treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/542b443be27492e921709c204aeb3560eab221c4" target='_blank'>
              Integrative multi-omics and single-cell transcriptomics reveal ARHGEF12 driving chemoresistance in bladder cancer
              </a>
            </td>
          <td>
            Kunyao Zhu, Zhejiao Zhang, Tinghao Li, Yan Sun, Junlong Zhu, Linfeng Wu, Xiaoyu Zhang, Hang Tong, Zijia Qin, Aimin Wang, Weiyang He
          </td>
          <td>2025-11-27</td>
          <td>Hereditas</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/648afec99f4d26f113081261fc3d480301dc1dab" target='_blank'>
              Transcriptional reprogramming of tumor-infiltrating T cells during PD-1 blockade revealed through gene regulatory network and trajectory inference in squamous cell carcinoma
              </a>
            </td>
          <td>
            Roger Casals-Franch, Jordi Villà-Freixa, Lara Nonell, Adrián López García de Lomana
          </td>
          <td>2025-10-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is characterized by complex genomic background and treatment resistance. We first defined “Titin (TTN) inactivation”, a state of TTN expression deficiency or mutation, affecting tumor progression. However, how TTN inactivation regulates immune escape and affects therapeutic resistance remains unclear. Using whole-exome sequencing, single-cell transcriptome sequencing, and spatial transcriptome sequencing, we screened the clinical features of TNBC patients with TTN inactivation who accepted neoadjuvant therapy. Meanwhile, we used CRISPR-Cas9 technology to construct various mutant TNBC cell lines. Lentiviral vector carrying TTN and delta-like ligand 4 (DLL4) was validated in vivo to verify potential mechanism. Myeloid-derived suppressor cells (MDSCs) metabolic function was measured using glycolysis-related molecular experiments. Immunotherapeutic agents against TNBC with TTN inactivation were explored in the orthotopic MCT4fl/fl genetically modified mouse model. DLL4-regulated ecological niche was established in TNBC with TTN inactivation. Mechanistically, TTN deficiency and mutation led to DLL4 secretion in TNBC. DLL4 enhanced MCT4-mediated glycolysis via MDSCs-derived NOTCH2 signaling pathway, driving the malignant function and lactate acid excretion of MDSCs. DLL4-derived MDSCs promoted stemness-mediated drug resistance by inducing histone lactate modification in TNBC, suppressing the anti-tumor activities of CD8+T cells. Blocking the DLL4-MCT4 axis stimulated anti-tumor immunity and synergized with anti-PD-1, improving response rates for first-line neoadjuvant therapy in TNBC. Our study revealed intrinsic mechanism by which TTN regulates the tumor immune microenvironment and provided a potential target for immunotherapy in TNBC with TTN inactivation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf824ac3335bab02348fceec44a41f866d2add7b" target='_blank'>
              Delta-like ligand 4 mediated myeloid-derived suppressor cell metabolic reprogramming promotes neoadjuvant therapy resistance in titin-inactivated triple-negative breast cancer
              </a>
            </td>
          <td>
            Yanfang Yang, Ziyun Liu, Parhat Kaysar, Yuxi Han, Bo Ni, Linwei Li, Lina Zhang, Xiaobin Shang, Yaoyao Zhou, Yongjie Xie, Zhansheng Jiang
          </td>
          <td>2025-12-01</td>
          <td>Molecular Biomedicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a185da472d5e884ad916a2af2cce445855a3e882" target='_blank'>
              Single-cell lineage trajectory defines CDK inhibitor-sensitive cells-of-origin in esophageal squamous cell cancer
              </a>
            </td>
          <td>
            Kyung-Pil Ko, Jie Zhang, Sohee Jun, Jae-Ll Park
          </td>
          <td>2025-10-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Cancer cell plasticity, the dynamic ability of tumor cells to switch phenotypic states, drives tumor progression, metastasis, and therapy resistance, posing a formidable challenge to effective cancer management. This review comprehensively explores the genetic and epigenetic mechanisms underlying plasticity, highlighting how oncogenic mutations, chromosomal instability, and pathways like PI3K/AKT and RAS/MAPK enable adaptive behaviors such as epithelial-mesenchymal transition and stemness. Epigenetic regulators, including DNA methylation, histone modifications, and non-coding RNAs, amplify this adaptability, allowing rapid responses to therapeutic pressures. We examine the profound clinical implications, with plasticity fueling relapse in breast, lung, and prostate cancers, exacerbated by tumor microenvironment interactions. Challenges in detecting plastic states due to intratumor heterogeneity underscore the need for advanced diagnostics like single-cell genomics. Emerging interventions, from CRISPR-based gene editing and epigenetic therapies to patient-derived organoids and AI-driven combination strategies, offer transformative potential to disrupt plasticity and overcome resistance. This review illuminates the intricate genetic landscape of cancer cell plasticity, advocating for integrated approaches to revolutionize personalized oncology and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a56a74b7b33fc882b2d862736314940182017fc5" target='_blank'>
              The Genetic Landscape of Cancer Cell Plasticity: Mechanisms, Implications for Therapy Resistance and Emerging Interventions
              </a>
            </td>
          <td>
            I. N. Monago, Nyerovwo Charity Okei, Chibuike Stephen Nzereogu, Taiwo Emmanuel Olowodasa, Tamasanni Omeiza Salawu, Ahmad Muhammad Garba, Doris Ada Ojum
          </td>
          <td>2025-10-31</td>
          <td>International Journal of Biological and Pharmaceutical Sciences Archive</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72837cc909b2a1ccbe8a1449f9aafc89b0a2cfa3" target='_blank'>
              Epigenetic reprogramming as the nexus of cancer stemness and therapy resistance: implications for biomarker discovery.
              </a>
            </td>
          <td>
            Chu Xin Ng, Shin Yuh Lee, Xin Yi Yap, Yong Hui Wong, Jian Sheng Loh, Kuan Ping Ang, Wen Hwei Lee, Wei Xu, Phelim Voon Chen Yong, Sau Har Lee
          </td>
          <td>2025-11-16</td>
          <td>Discover oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Cis-regulatory elements (CREs) control oncogene expression and malignant phenotypes. The high clinicopathological heterogeneity of cancer cannot be explained by gene expression alone, being attributed to CRE heterogeneity. However, characterizing cancer-associated CREs is challenging. To address this issue, we performed a single-cell epigenomic analysis of clinical specimens. To map the multicellular ecosystem of breast cancer (BC) and identify candidate CREs (cCREs), we performed a single-cell assay for transposase-accessible chromatin with sequencing (scATAC-seq) of 38 prospectively collected BC samples with various clinicopathological characteristics. First, we performed single-cell chromatin accessibility profiling of a high-quality set of 22,775 cells from BC samples. Cells were annotated using marker gene accessibility and integration with existing single-cell RNA sequencing data. The chromatin accessibility patterns exhibited by cancer cells were consistent with the clinicopathological features of each tumor. We identified 224,585 cCREs across the BC ecosystem. By identifying cluster-specific differentially accessible cCREs (DA-cCREs) and constructing a putative enhancer–promoter network, we mapped the cis-regulatory landscape for cancer cells and the tumor microenvironment. The accessibility of putative enhancers targeting the same gene differed within or between tumors, highlighting intra- and inter-cis-regulatory tumor heterogeneity. This study provides a valuable resource for future epigenetic research on BC and highlights the diverse regulatory landscape within or among tumor(s), suggesting that cCREs regulate intra- and intertumor heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9a987c29ce941aae8e905c09b2ed8fd2ff4315f" target='_blank'>
              Characterizing heterogeneous cis-regulatory elements in gene regulatory programs associated with breast cancer
              </a>
            </td>
          <td>
            Chisa Hori, Kohei Kumegawa, S. Saeki, Yoko Takahashi, Liying Yang, T. Nakadai, K. Otsuji, Chikako Takahata, Y. Ozaki, N. Uehiro, Yurie Haruyama, Tomo Osako, T. Takano, Seiichi Mori, Tetsuo Noda, Satoshi Fujii, Shinji Ohno, T. Ueno, Reo Maruyama
          </td>
          <td>2025-11-27</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34b22389536004a2e5e3234be8036e5614f2a3ae" target='_blank'>
              A blueprint for mutation-defined hallmark vulnerabilities across human cancers
              </a>
            </td>
          <td>
            Ran Xu, Rebecca San Gil, Xin Tracy Liu, Yanfei Qi, Derek Tran, Chun Xu, Justin J-L Wong, Lenka Munoz, Graham J. Mann, Yuchen Feng
          </td>
          <td>2025-10-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Metastatic breast cancer (MBC) remains the primary cause of mortality in breast cancer patients, driven by tumor heterogeneity, cellular evolution, and therapy-resistant clones. Traditional bulk transcriptomics, although informative, fail to capture rare subpopulations and context-specific gene expression, which are crucial for understanding disease progression. Single-cell transcriptomics (SCT) has emerged as a transformative approach, enabling high-resolution analysis of individual cells to reveal tumor composition, lineage dynamics, and transcriptional plasticity. This review highlights how SCT reshapes our understanding of MBC by mapping tumor evolution, identifying cancer stem-like cells, and characterizing states of epithelial-mesenchymal transition. We explore how SCT reveals clonal and spatial heterogeneity, and how tumor microenvironment components, including immune, stromal, and endothelial cells, interact with cancer cells to support immune evasion and the formation of a metastatic niche. SCT also uncovers mechanisms of therapeutic resistance, including transcriptional reprogramming and the survival of drug-tolerant subpopulations. Integrating SCT with spatial transcriptomics and multi-omics platforms offers a comprehensive view of the MBC ecosystem and may uncover novel therapeutic targets. We further discuss the translational potential of SCT for biomarker discovery, liquid biopsy development, and precision oncology. We address current technical challenges and future directions for clinical application. SCT is poised to transform MBC research and guide next-generation therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf97dd9ea39bc843d16cb57d14bc4af2cfce4592" target='_blank'>
              Single-cell transcriptomics in metastatic breast cancer: mapping tumor evolution and therapeutic resistance
              </a>
            </td>
          <td>
            Xu Han, Xin Li, Ling Bai, Gangling Zhang
          </td>
          <td>2025-10-23</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cf3f4a8479510c49ff95777ff501eb3f2838138" target='_blank'>
              Epigenetic priming promotes tyrosine kinase inhibitor resistance and oncogene amplification.
              </a>
            </td>
          <td>
            Rebecca M. Starble, Eric G Sun, Rana Gbyli, Jonathan S. D. Radda, Jiuwei Lu, Tyler B Jensen, Ning Sun, Nelli Khudaverdyan, Ting Zhao, Bomiao Hu, M. Melnick, Shuai Zhao, Nitin Roper, G. Wang, A. Tackett, Yinsheng Wang, Jikui Song, Katerina Politi, Siyuan Wang, Andrew Z Xiao
          </td>
          <td>2025-10-23</td>
          <td>Nature structural & molecular biology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="The human somatic genome evolves throughout our lifespan, producing mosaic individuals comprising clones harboring different mutations across tissues. While clonal expansions in the hematopoietic system have been extensively characterized and reported to be nearly ubiquitous, clonal mosaicism (CM) has more recently also been described across multiple solid tissues. However, outstanding questions remain about the parameters and processes of human somatic evolution in non-cancerous solid human tissues, including when clones arise, how they evolve over time, and what mechanisms lead to their expansion. Questions of timing and clonal dynamics can be addressed through phylogenetic reconstruction, which serves as a ‘temporal microscope’, while uncovering the mechanisms of expansion necessitates simultaneous phenotypic profiling. To address this gap, here we develop Single-cell Miniaturized Automated Reverse Transcription and Primary Template-directed Amplification (SMART-PTA) for joint single-cell whole-genome and whole-transcriptome sequencing for large scale and cost efficient interrogation of solid tissue CM. We established a workflow that generates hundreds of matched single-cell whole genome and transcriptome libraries within a week. We profiled phenotypically normal esophagus tissue from four aged donors and used somatic variants to build high-resolution single-cell lineages from >2,700 cells with accompanying transcriptomic information, reconstructing >70 years of somatic evolution. T cell expansions identified from T cell receptor (TCR) sequences validated the clonal structure of the single-nucleotide variant (SNV)-based phylogenies and phylogenetic cross-correlation analysis showed that epithelial cells had higher degrees of shared ancestry by spatial location compared to immune cells. Mapping mutation signatures to the phylogenetic tree revealed the emergence of tobacco/alcohol exposure-related signatures later in life, consistent with the donors’ exposure histories. We identified variants in driver genes that were previously reported in the phenotypically normal esophagus, detecting clonal expansions harboring mutations in genes including TP53 and FAT1. We mapped the evolution of clones with both monoallelic as well as biallelic TP53 loss, including a clone associated with high expression of cell cycling genes and higher chromosome instability. Leveraging the matched transcriptome data, we uncovered cell type biases in mutant clones, with a higher proportion of TP53 or FAT1-mutant cells in an earlier basal epithelial cell state compared to wild-type cells. We further observed copy-neutral loss of heterozygosity (CNLOH) events on chromosome 9q that spanned the NOTCH1 locus in up to ∼35% of epithelial cells. Mapping CNLOH events to the phylogenetic tree revealed a striking pattern in which CNLOH was separately acquired many times, reflecting convergent evolution. Cells with CNLOH events were biased towards the earlier basal epithelial state, suggestive of a selective advantage that leads to prevalent recurrence of chr9q CNLOH. Together, we demonstrate that SMART-PTA is an efficient, scalable approach for single-cell whole-genome and whole-transcriptome profiling to build phenotypically annotated single-cell phylogenies with enough throughput and power for application to normal tissue somatic evolution. Moreover, we reconstruct the evolutionary history of the esophageal epithelium at high scale and resolution, providing a window into the dynamics and processes that shape clonal expansions in phenotypically normal tissues throughout a lifespan.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57a93b5a70ed710c9aacd9cdfba3788cae5ad699" target='_blank'>
              Large-scale single-cell phylogenetic mapping of clonal evolution in the human aging esophagus
              </a>
            </td>
          <td>
            Tamara Prieto, Dennis J. Yuan, John Zinno, Clayton Hughes, Nicholas Midler, Sheng Kao, Jani Huuhtanen, Ramya Raviram, Foteini Fotopoulou, Neil Ruthen, Srinivas Rajagopalan, Joshua S. Schiffman, Andrew R. D’Avino, Sang-Ho Yoon, Jesus Sotelo, Nathaniel D. Omans, Noelle Wheeler, Alejandro Garces, Barun Pradhan, A. P. Cheng, N. Robine, Catherine Potenski, Katharine Godfrey, N. Kakiuchi, Akira Yokoyama, Seishi Ogawa, Julian A. Abrams, Ivan Raimondi, D. Landau
          </td>
          <td>2025-10-13</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a78192449266cbe137f9e2b3c247baae65185e3f" target='_blank'>
              CRISPR screens reveal YAP/TEAD axis as a mediator of drug-tolerant cells in EGFR-mutant NSCLC.
              </a>
            </td>
          <td>
            Muhammed Yusuf
          </td>
          <td>2025-11-12</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Elucidating mechanisms of drug resistance is key for overcoming resistance, guiding drug design, and enabling accurate resistance prediction. Recently, disease metabolic reprogramming has emerged as a novel mechanism of resistance, which enables disease cells to adapt to therapeutic resistance by altering energy production pathways, cellular signaling, and biosynthesis processes. Moreover, protein structure alterations also play a pivotal role in resistance study, facilitating mechanistic understanding, and structure-based target discovery. In other words, integrating these recently accumulated critical data is essential for enriching the landscape of drug resistance data. Therefore, in this study, DRESIS was a significant update by providing (i) 236 molecules that drive metabolic reprogramming and confer resistance to 168 drugs, together with a detailed mechanism, (ii) 2228 protein structural variants implicated in resistance to 671 drugs across 238 diseases, and (iii) greatly expanded landscapes of drug resistance information, now featuring 398 newly added key drug-resistant molecules, 356 drugs with the latest published resistance mechanisms, and 81 new drug-resistant disease categories. All in all, DRESIS 2.0 is expected to serve as a valuable resource for the scientific community and provide important support in tackling the global challenge of drug resistance, which is now publicly accessible at https://idrblab.org/dresis/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cafc45a66260eff1536a05a8101c61e45dec038b" target='_blank'>
              DRESIS 2.0: the comprehensive landscape of drug resistance information.
              </a>
            </td>
          <td>
            Xiuna Sun, Zhangle Wei, Xinyuan Yu, Kaixuan Liu, Shun Yao, Zheng Ni, Hanbing Wang, Ziming Zhao, Yinpeng Zhang, Yuxuan Liu, Hanlu Ding, Yintao Zhang, Zhiguo Liu, Mang Xiao, Feng Zhu
          </td>
          <td>2025-11-20</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cf856a29d7dc82e41e89d6c3e46ad0a67ee3c3a" target='_blank'>
              Cell-stereotyped DNA repair outcomes are widespread during genome editing
              </a>
            </td>
          <td>
            Moritz F. Schlapansky, Markus S. Schröder, Antonio J. Santinha, Tanja Rothgangl, Eleonora I Ioannidi, G. Cullot, Bohdan Lewków, G. C. Ortega, Ahmed Nouraiz, Dominic Mailänder, Louisa Selbert, Simon Lam, Aitana Egea, Jiyung J. Shin, Matteo Bordi, Mark DeWitt, Ana Gvozdenovic, Stephen P. Jackson, T. Schroeder, Helmuth Gehart, Gerald Schwank, Randall J. Platt, Jacob E. Corn
          </td>
          <td>2025-10-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="


 Fusion transcription factors (TF) are hallmark oncogenic drivers in pediatric B cell acute lymphoblastic leukemia (B-ALL), contributing to leukemia initiation and transformation. Despite advances in risk-adapted therapies guided by clinical and molecular features, genetic heterogeneity and variable outcomes underscore the need to uncover driver-specific vulnerabilities to expand personalized treatment strategies, particularly in the relapsed/refractory context. Here, we apply a systematic approach to identify rational combination strategies informed by integrating identification of driver-specific downstream targets of the fusion TF TCF3::PBX1 – a common driver in pediatric B-ALL – with an unbiased high-throughput drug combination screen.
 To dissect fusion-specific druggable genes we knocked down (KD) TCF3::PBX1 by short hairpin RNA (shRNA) in the 697 cell line and filtered for transcriptional targets with available approved and clinical-phase candidate drugs. Among top fusion-associated genes we identified ornithine decarboxylase (ODC1), the rate-limiting enzyme of synthesizing polyamines, small polycations with high affinity for nucleic acids controlling key cellular processes like transcription, translation, DNA replication and cell cycle progression. Public datasets confirmed highest mRNA expression of ODC1 in TCF3::PBX1+ patient samples across B-ALL subtypes (St. Jude) and selective dependency on ODC1 in TCF3::PBX1+ cell lines (DepMap) reinforcing its role as a specific fusion-driven vulnerability. This subtype-specific overexpression of ODC1 led us to investigate its regulation, and public ChIP-seq data revealed binding peaks of TCF3::PBX1 at the ODC1 promoter supporting its role as a novel regulator alongside MYC. Genetic interference by shRNA-mediated ODC1 KD and pharmacological inhibition by Difluoromethylornithine (DFMO) reduced intracellular polyamine levels of putrescine, spermidine and spermine and demonstrated cytostatic effects with decreased proliferation with a reduction of cells in S-Phase in TCF3::PBX1+ cell lines. In patient-derived xenograft (PDX) models, ex vivo sensitivity to DFMO was highest in TCF3::PBX1+ PDX compared to other subtypes. Notably, PDXs from TCF3::PBX1+ relapses exhibited sensitivity levels comparable to those observed in TCF3::PBX1+ PDX with favorable outcomes. Transcriptomic and proteomic analysis of DFMO-treated 697 cells uncovered changes in gene and protein expression associated with amino acid and polyamine transport, mitochondrial metabolism, global translation and cell cycle. Strikingly, treatment with DFMO induced regulation of key B cell identity TF and B cell receptor signaling components, alongside an enrichment of transcriptional signatures indicative of more mature B cell states.
 To uncover synthetic lethal vulnerabilities of this induced rewired cell state we performed an unbiased high-throughput combination screen (n = 155 drugs) in 697 cells with or without ODC1 KD. Among known synergistic targets like AMXT-1501, a polyamine transport inhibitor, BCL-XL inhibitors emerged as potent synergistic partners to genetic and pharmacological ODC1 inhibition, further validated in PDX models ex vivo. To evaluate this combination in vivo, we employed a dual-targeting strategy against polyamine metabolism using DFMO to inhibit synthesis and AMXT-1501 to block uptake, as ex vivo data revealed pronounced metabolic plasticity and compensatory upregulation of polyamine transporters following DFMO monotherapy. In vivo combination of DFMO + polyamine-transport inhibitor AMXT-1501 and BCL-XL PROTAC DT2216 in TCF3::PBX1+ PDX models demonstrated promising antileukemic efficacy seen by reduced leukemic burden in spleen and bone marrow at the end of treatment.
 Collectively, our findings identify the fusion TF TCF3::PBX1 as a previously unrecognized positive regulator of polyamine synthesis, acting via ODC1 upregulation, positioning ODC1 as a promising target in this subtype. We further demonstrated that targeting polyamine synthesis induces transcriptional changes in essential B-cell developmental and metabolic programs, uncovering opportunities to target synthetic lethal principles through BCL-XL. The recent approval of DFMO for neuroblastoma maintenance therapy highlights its translational potential not only in TCF3::PBX1+ B-ALL but warrants exploration of this rational combination in other MYC-driven and/or polyamine-addicted B-ALL, especially in the relapsed/refractory context.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60ff0281ccf3599cc3a7d33b49b4b28f1ec98b96" target='_blank'>
              Fusion-specific regulation of polyamine synthesis sensitizes to BCL-XL inhibition in TCF3::PBX1+ B-ALL
              </a>
            </td>
          <td>
            James Kim, Sarah Cherkaoui, Simona Ulrich, Bartimée Galvan, Chengkang Li, Caroline Mussak, Samanta Carbajal, Shannon Mezger, B. Bornhauser, Raphael J. Morscher, Jean-Pierre Bourquin
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="


 Glioma, the most common primary brain tumor, shows poor prognosis and resistance to first-line therapies, like temozolomide (TMZ), lomustine (LOM) and panobinostat-marizomib (PM), complicating treatment and diminishing disease-free survival. It is critical to investigate the molecular mechanisms that underlie chemoresistance to develop effective therapies. This study examines common transcriptional networks that facilitate chemoresistance to identify novel therapeutic targets that may overcome resistance.



 The pediatric high-grade glioma KNS42 cell line was chronically treated with increasing doses of TMZ, LOM, or PM to induce drug resistance (RES); control cells received vehicle (DMSO) treatment. Bulk-RNA sequencing was performed followed by differential gene expression analyses (PartekFlow-ANOVA/DESeq2) followed by pathway and comparison analysis (QIAGEN IPA, |FDR|<0.05).



 RNA-sequencing analysis followed by comparison-analysis revealed differential gene expression in treatment-resistant glioma cells as compared to control cells, including 3,839 differentially expressed genes (DEGs) in LOM RES, 3,641 in TMZ RES, and 3,445 in PM RES groups compared to control group. Among these, 374, 417, and 360 DEGs were downregulated in LOM, TMZ, and PM RES cells, respectively, while 265 (LOM and TMZ RES) and 216 (PM RES) DEGs were upregulated (log2FC > 2; p < 0.05). Interestingly, when the top 10 upregulated genes from each resistant group were compared, six genes (VCAN, FOXA1, SOX2, PDGFRA, ERBB4, and COL4A) were found to be commonly upregulated in all 3 resistant groups. We further confirmed that SOX2, PDGFRA, and ERBB4 are upregulated at the protein level. Ongoing studies will test whether these 6 DEGs are required for mediating resistance against diverse drugs.



 RNA-sequencing identified common genes linked to TMZ, LOM, and PM chemoresistance, revealing key genetic networks involved in acquiring and maintaining drug resistance. Targeting these pathways, along with epigenetic silencing and/or combinatorial therapies, may resensitize chemoresistant cells and allow for effective treatment of glioma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5824551e4b1f9c0b4ac4e374b69dbac7316d823" target='_blank'>
              DDDR-78. Identifying common transcriptional mechanisms in treatment-resistant glioma
              </a>
            </td>
          <td>
            N. B. Rao, Aaron R Johnson, Subrahmanya D. Vallabhapurapu, Tracy A Miller, Ian F Pollack
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The development and progression of gastrointestinal (GI) cancers not only depend on the malignancy of the tumor cells, but is also defined by the complex and adaptive nature of the tumor microenvironment (TME). The TME in GI cancers exhibits a complex internal structure, typically comprising cancer cells, cancer stem cells, cancer-associated fibroblasts, immune cells, and endothelial cells, all embedded within a dynamic extracellular matrix. This intricate ecosystem fuels tumor initiation, progression, metastasis, recurrence and therapy response through the heterogeneity and plasticity. Recent advances in single-cell sequencing have provided unprecedented resolution in profiling the cellular diversity and interactions within the TME. These technologies have uncovered previously unknown cell subtypes and intricate communication networks that drive therapy resistance and tumor relapse. In this review, we summarize and discuss the latest findings from single-cell sequencing of key cellular players and their interactions within the TME of GI cancers. We highlight single cell insights that are reshaping our understanding of tumor biology, with particular focus on their implications for overcoming therapy resistance and improving clinical outcomes. We believe that a deeper understanding of TME heterogeneity and plasticity at the single-cell level promises to transform the landscape of precision treatment in GI cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45ef919b2bb866835d8195a594df10437839f0cc" target='_blank'>
              Single-cell insights into tumor microenvironment heterogeneity and plasticity: transforming precision therapy in gastrointestinal cancers
              </a>
            </td>
          <td>
            Jialei Weng, Feng Ju, Zicheng Lyu, N. Fan, Daniel J. Smit, Wenxin Xu, Xiaolin Wu, Philip Becker, Yinan Xu, Michal R. Schweiger, Axel M. Hillmer, Ralf Harwig, Sheraz Gul, Alexander Link, Lydia Meder, Nan Fang, Qiongzhu Dong, Christiane J. Bruns, Ning Ren, Yue Zhao
          </td>
          <td>2025-11-28</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Chimeric antigen receptor (CAR)-T cell therapy has shown remarkable success in hematologic malignancies but faces significant challenges in solid tumors due to the immunosuppressive tumor microenvironment (TME). Among these, hypoxia plays a vital role, yet the molecular mediators that link hypoxia to CAR-T dysfunction remain incompletely understood. Anti-mesothelin (MSLN) CAR-T cells were cultured under normoxic (21% O2) and hypoxic (1% O2) conditions for six days. We assessed cell expansion, phenotypes, cytotoxicity, and metabolic features. RNA sequencing was conducted to identify key gene expression changes induced by hypoxia. Findings were further validated using anti-CD70 CAR-T cells. Hypoxia reduced CAR-T proliferation, increased apoptosis, lowered memory phenotypes, raised exhaustion, and weakened cytotoxicity in short-term and long-term assays. Transcriptomic and metabolic analyses showed metabolic reprogramming with increased glycolysis and reduced oxidative phosphorylation. Among the dysregulated genes, the serine/threonine-protein kinase PIM3 emerged as a previously underexplored mediator of hypoxia-driven dysfunction. Genetic or pharmacologic inhibition of PIM3 counteracted hypoxia-induced impairment, enhancing memory phenotypes of CAR-T cells, and improving their anti-tumor activity both in vitro and in vivo. This work identifies PIM3 as a previously underexplored target that links hypoxia to CAR-T cell dysfunction and demonstrates that PIM3 inhibition can reverse these effects. These findings provide a mechanistic rationale for incorporating PIM3 inhibition into CAR-T cell manufacturing or engineering to improve their therapeutic potential in hypoxic solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2adcb89e4cbae164c14c49d77ccdb838d38a89aa" target='_blank'>
              Metabolic reprogramming through PIM3 inhibition reverses hypoxia-induced CAR-T cell dysfunction in solid tumors
              </a>
            </td>
          <td>
            Muya Zhou, Luxia Xu, Jinhua Hu, Wenwen Chen, Jialu Hong, Mufeng Wang, Zhi-Cheng Guo
          </td>
          <td>2025-11-06</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD), a subtype of non-small cell lung cancer (NSCLC), has high incidence and poor prognosis. Although anti-programmed cell death protein 1 (PD1) therapy and epidermal growth factor receptor (EGFR) inhibitors benefit some patients, many remain unresponsive. Genome-wide association studies (GWAS) can identify risk loci, while single-cell transcriptomics enables exploration of genetic variations and mechanisms in LUAD. We integrated GWAS data from FinnGen, IEU OpenGWAS, and GWAS Catalog with single-cell and spatial transcriptomics from GEO to investigate LUAD genetics and pathology. GWAS analyses identified SNP loci, risk factors, LD Score, and SMR results, revealing a novel apoptosis-related locus, Serine/Threonine Kinase 24 (STK24) (SMR P < 0.05). Single-cell analysis identified STK24-positive epithelial cells (STK24posEpi) and their transcription factors (SCENIC), developmental trajectory (Monocle), and tumor microenvironment roles (CellCall). Spatial mapping (RCTD) and trajectory analysis (stlearn) were combined with cell communication (CellChat), spatial signal flow (Commot), and co-localization (Misty). Bayesian deconvolution linked STK24posEpi to TCGA-LUAD bulk transcriptomes, assessing overall survival (OS), immune infiltration, and clinical traits. A prognostic model (STK24SuperPC, mean c-index 0.61) was built using multiple machine learning algorithms to predict survival and immunotherapy response. Laboratory experiments tested STK24’s effects on LUAD cell proliferation and apoptosis. STK24 was identified as a LUAD risk factor (pSMR < 0.05). STK24posEpi communicated with fibroblasts, endothelial cells, and others via MIF and PDGF pathways, and was associated with apoptosis (Importance > 0.5), metastasis (Importance > 2), DNA repair (Importance > 1), and DNA damage (Importance > 2). Located at the origin of the trajectory, STK24posEpi correlated with poor prognosis (HR > 0, P < 0.05) and reduced lymphocyte infiltration (Pearson’s r < 0, P < 0.05). STK24 overexpression enhanced LUAD cell proliferation, migration, and colony formation, while reducing apoptosis, increasing BCL-2 and Caspase-3, and decreasing Cleaved-Caspase-3 expression. STK24, an apoptosis-related gene, promotes LUAD progression by enhancing proliferation and inhibiting apoptosis, with STK24posEpi interacting in the tumor microenvironment through PDGF and MIF pathways. These findings offer potential therapeutic targets and support personalized LUAD treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f038883aea606036c62cbeb3f18e89605908569d" target='_blank'>
              Genome-wide association, single-cell, and spatial transcriptomics analyses reveal the role of the STK24-expressing positive cells in LUAD progression and the tumor microenvironment, identifying STK24 as a potential therapeutic target
              </a>
            </td>
          <td>
            Junzhi Liu, Huimin Li, Yuheng Jiao, Meng Hou
          </td>
          <td>2025-10-30</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74bd0c1fdd61877244de8ec8adaa6272cd0c1476" target='_blank'>
              Loss of PIMREG impairs double-strand break signaling and repair
              </a>
            </td>
          <td>
            Maria Vitória de Rizzo Gasparini, Laís do Carmo, Giovanna Gonçalves de Almeida, Mariana Siqueira Lacerda Mamede, Isabela Spido Dias, Jennifer Ann Black, V. C. Arfelli, Yun-Chien Chang, S. Heinzlmeir, Bernhard Kuster, Philipp A. Greif, Leticia Fröhlich Archangelo
          </td>
          <td>2025-10-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="


 Background Drug-tolerant persister (DTP) cells are increasingly recognized as a key contributor to therapy resistance in cancer patients. In mantle cell lymphoma (MCL), the mechanisms by which DTP cells adapt to treatments and develop resistance remain poorly understood. This study investigates the presence and development of pirtobrutinib-tolerant persister cells in MCL and aims to elucidate their underlying mechanisms of resistance. By identifying these mechanisms, we seek to provide insights into novel therapeutic strategies for targeting DTP cells in MCL patients.
 Methods A non-stochastic drug-tolerant persister (DTP) cell line model was developed using mantle cell lymphoma (MCL) cells with acquired resistance to therapies. This model includes Mino cells resistant to venetoclax (Mino-VEN-R) and JeKo-1 cells resistant to ibrutinib (JeKo-1-IBN-R). Both bulk and single-cell RNA sequencing were employed to characterize upregulated and downregulated pathways. Metabolite profiling was conducted using ultra-high-resolution mass spectrometry (HRMS). Oxygen consumption was measured with the Seahorse XF96 analyzer (Agilent Technologies). Anti-CD19 CAR T-cells were generated from human primary pan-T cells isolated from healthy donors, transduced with lentivirus expressing a CD19-directed CAR (FMC63 scFv, 4-1BB costimulatory domain, and CD3ζ stimulatory domain). The presence of pirtobrutinib-tolerant persister cells in MCL was validated using patient-derived organoid and xenograft models. Transcriptome profiles from 62 primary MCL patient samples were analyzed via RNA sequencing.
 Results MCL cells with abnormal morphology have been recognized as an indicator of aggressive diseases with poor response to therapies. However, less was known about their origin and mechanism. Here, we observed that pirtobrutinib treatment induces anabolic growth rather than proliferation in Mino-VEN-R and JeKo-1-IBN-R mantle cell lymphoma (MCL) cells, resulting in enlarged cells with pleomorphism, designated as “Giant cells.” Upon pirtobrutinib withdrawal, these cells resume proliferation, indicating a reversible phenotypic transition characteristic of a drug-tolerant persister (DTP) state, with Giant cells serving as a transitional state for pirtobrutinib-tolerant persister cells. RNA sequencing revealed that Giant cells exhibit a dedifferentiated state under pirtobrutinib treatment, marked by significant loss of B-cell markers, such as CD19, conferring resistance to anti-CD19 CAR T-cell therapy in NSG mice. Mechanistically, an anabolic tricarboxylic acid (TCA) cycle fuels DTP cell development, driven by elevated acetyl-CoA levels mediated by increased ATP-citrate lyase (ACL) activity, which profoundly alters gene expression profiles. GOT2, a key enzyme, supports biosynthesis, including nucleotide production, while ribosome biogenesis is a hallmark of the Giant cell state. Notably, the EMT modulator SNAI1 is a key transcription factor potentially driving dedifferentiation. Upon drug withdrawal, the TCA cycle shifts to a catabolic mode, as validated in patient-derived organoid (PDO) cultures. Additionally, DTP cells in the Giant cell state were identified in patient-derived xenograft (PDX) tumor samples from an MCL patient resistant to pirtobrutinib and CAR T-cell therapy. More evidence from primary MCL patients with resistance to pirtobrutinib and CAR T-cell therapies showed the potential presence of DTP cells, including results from IHC imaging and RNA sequencing. These indicate that DTP cells can be more prevalent in therapy-resistant MCL patients compared to therapy-sensitive patients.
 Conclusions We conclude that non-genetic mechanisms, particularly metabolic reprogramming, drive the fate of drug-tolerant persister (DTP) cells in mantle cell lymphoma (MCL). The tricarboxylic acid (TCA) cycle dynamically switches between anabolic and catabolic modes, enabling tumor cells to adapt and evade therapeutic stress. Acetyl-CoA is a critical metabolite that reshapes the epigenetic landscape of tumor cells. This phenotypic plasticity of DTP cells contributes to tumor heterogeneity following prolonged therapies. DTP cells are prevalent in MCL patients with therapy resistance. Targeting the dedifferentiated state of DTP cells, rather than oncogenic lesions, offers a novel strategy to overcome stable therapy resistance in MCL.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89437df3f4f40ad405722c379a0b55564e1288a0" target='_blank'>
              The switch of TCA cycle mode determines the fate of pirtobrutinib-tolerant persister cells
              </a>
            </td>
          <td>
            Michael Wang, Yang Liu, Qingsong Cai, L. Nie, , Fangfang Yan, Yue Fei, Yijing Li, Lin Tan, Philip Lorenzi, , Wei Wang
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a heterogeneous disease that arises from dysregulated myeloid proliferation. We performed a high-throughput CRISPR interference (CRISPRi) screen in the THP-1 monocytic cancer cell line to identify long noncoding RNAs (lncRNAs) that play a role in contributing to cell proliferation. Our screen identified INSTAR (Intergenic Nuclear Suppressor lncRNA Targeting Adjacent Regulator SFMBT2) as a top candidate. RNA-seq on INSTAR deficient THP-1 cells revealed transcriptional changes in genes involved in cell proliferation as well as other cellular processes. Loss of INSTAR selectively reduced expression of its neighboring gene, SFMBT2. Functional assays confirmed that both genes suppress cell growth, revealing a cis-regulatory mechanism in which INSTAR regulates SFMBT2 expression to control monocyte proliferation. Here, we leverage high-throughput screening to rapidly pinpoint functional lncRNAs providing novel insights into a key regulatory locus consisting of INSTAR and SFMBT2 which could be critical for better understanding dysregulation contributing to acute myeloid leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78f9f6afd97add9714e69419ecf796522c175c3f" target='_blank'>
              CRISPRi Screen Identifies a Novel Growth Suppressor lncRNA, INSTAR , in Human Monocytes
              </a>
            </td>
          <td>
            Christy Montano, Eric Malekos, S. Covarrubias, Sol Katzman, Lisa Sudek, Jillian Ward, S. Carpenter
          </td>
          <td>2025-10-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) is a prevalent and devastating form of oncogene amplification in cancer1,2. Circular megabase-sized ecDNAs lack centromeres and segregate stochastically during cell division3–6 yet persist over many generations. EcDNAs were first observed to hitchhike on mitotic chromosomes into daughter cell nuclei over 40 years ago with unknown mechanism3,7. Here we identify a family of human genomic elements, termed retention elements, that tether episomes to mitotic chromosomes to increase ecDNA transmission to daughter cells. We develop Retain-seq, a genome-scale assay that reveals thousands of human retention elements conferring generational persistence to heterologous episomes. Retention elements comprise a select set of CpG-rich gene promoters and act additively. Live-cell imaging and chromatin conformation capture show that retention elements physically interact with mitotic chromosomes at regions which are mitotically bookmarked by transcription factors and chromatin proteins, intermolecularly recapitulating promoter-enhancer interactions. Multiple retention elements are co-amplified with oncogenes on individual ecDNAs in human cancers and shape their sizes and structures. CpG-rich retention elements are focally hypomethylated; targeted cytosine methylation abrogates retention activity and leads to ecDNA loss, suggesting that methylation-sensitive interactions modulate episomal DNA retention. These results highlight the DNA elements and regulatory logic of mitotic ecDNA retention. Amplifications of retention elements promote the maintenance of oncogenic ecDNA across generations of cancer cells, revealing the principles of episome immortality intrinsic to the human genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6479f6922125476f4549c4aaa1b85996bf2c096" target='_blank'>
              Genetic elements promote retention of extrachromosomal DNA in cancer cells
              </a>
            </td>
          <td>
            Venkat Sankar, King L. Hung, Aditi Gnanasekar, Ivy Tsz-Lo Wong, Quanming Shi, Katerina Kraft, Matthew G. Jones, B. J. He, Xiaowei Yan, J. Belk, Kevin J. Liu, Sangya Agarwal, Sean K. Wang, A. Henssen, P. Mischel, Howard Y. Chang
          </td>
          <td>2025-10-12</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>29</td>
        </tr>

        <tr id="Primary prostate cancer presents with multifocal lesions and unpredictable clinical behavior, posing significant challenges for effective prognosis. To address this, we investigate the epigenomic landscape of prostate tumor biopsies from 25 treatment-naïve male patients by analyzing chromatin compartmentalization patterns. Our analysis reveals two distinct molecular subtypes: one with a Low Degree of Decompartmentalization (LDD) and another with a High Degree of Decompartmentalization (HDD). Here we show that the HDD subgroup exhibits extensive chromatin reorganization associated with diminished oncogenic potential. This subtype shows repression of molecular pathways involved in extracellular matrix remodeling and cellular plasticity. From this distinction, we derive an 18-gene transcriptional signature capable of differentiating HDD from LDD cases. Importantly, we validate the prognostic relevance of this signature in multiple independent cohorts totaling over 900 patients. Our findings suggest that epigenetic-derived signature at the time of diagnostic biopsy can offer a powerful tool for risk stratification in prostate cancer. Prostate cancer (PCa) evaluation remains challenging due to its heterogenous and multifocal nature. Here, the authors reconstruct the nuclear chromatin compartmentalization of PCa patients-derived biopsies distinguishing two epigenetic subtypes and deriving an 18-gene signature with prognostic value on TCGA samples that is further validated with multiple independent PCa cohorts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/696cb38bd331720bed277dcfa9f4701c78f389c7" target='_blank'>
              Chromatin remodeling restrains oncogenic functions in prostate cancer
              </a>
            </td>
          <td>
            V. Rosti, Giovanni Lembo, C. Petrini, Francesca Gorini, Roberto Quadri, C. Cordiglieri, Margherita Mutarelli, Elisa Salviato, Elisabetta Casari, Emanuele Di Patrizio Soldateschi, Emanuele Montanari, G. Albo, Francesco Ripa, Alessandra Fasciani, Mariacristina Crosti, E. De Lorenzis, M. Maggioni, V. Vaira, Maria Vivo, Francesco Ferrari, Chiara Lanzuolo
          </td>
          <td>2025-10-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28b535784b00dc4f23ce87d1f10a7f009abae40d" target='_blank'>
              Extensive and differential platinum chemotherapy mutagenesis in children
              </a>
            </td>
          <td>
            Anna Wenger, H. Lee-Six, Manas Dave, M. Layeghifard, A. Lawson, Federico Abascal, Pantelis A. Nicola, T. D. Treger, Toochi Ogbonnah, Conor Parks, Thomas R. W. Oliver, Jonathan Kennedy, A. Hodder, Nathaniel D. Anderson, Felipe Luz Torres Silva, Mi K. Trinh, Thomas Dowe, Marwo Habarwaa, James J. Sun, Sergio Assia-Zamora, M. Cortes-Cerisuelo, W. Jassem, Charlotte Town, Anil Dhawan, Vandana Jain, Karin Straathof, M. Deheragoda, I. Martincorena, Liina Palm, J. C. Hutchinson, Tim H H Coorens, Claire Trayers, Nigel Heaton, A. Shlien, Yoh Zen, Foad J. Rouhani, Sam Behjati
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="
 MAPK pathway alterations occur in 10-20% of diffuse midline glioma, H3K27-altered (DMG-H3K27), resulting in a proposed novel subtype of the disease with distinct clinical features. In a previous co-clinical trial platform, we showed an increased sensitivity to MEK inhibitors in DMG-MAPK patient-derived models, though resistance occurred in vitro, in vivo and clinically. To understand the nature of these resistance mechanisms, we used sophisticated coupling of retrievable high-complexity barcoding and single cell RNA sequencing in DMG-MAPK cells. Lineage tracing conducted in several technical replicate experiments showed a subset of clones is consistently enriched across replicates when treated with a GI90 concentration of trametinib for 11 weeks. Interrogation of clonal dynamics using mathematical modelling showed barcodes commonly enriched across replicates exhibit significantly higher clonal fitness compared to replicate-specific enriched barcodes, suggesting a deterministic pattern of resistance. Serial single cell RNA sequencing analysis at weeks 5, 8 and 11 post-treatment initiation showed some resistant H3.3K27M/BRAF-mutant cells adopted astrocyte-like and neural progenitor-like metaprograms in which barcodes were commonly enriched across two cell states, suggesting a subset of clones circumvent trametinib treatment by phenotypic switching; simultaneously, a late-emerging resistant population of cells activate a distinct mesenchymal program. Furthermore, we determine the previously identified MAP2K1 (MEK1) mutations to arise specifically from a small population of radial glia-like stem cells that appear acutely predisposed to the acquisition of these mutations. Together, our data suggest two clonally and mechanistically distinct resistance mechanisms can be active in DMG-MAPK in response to MEK inhibition, involving both transcriptional plasticity and a progenitor-like state primed for the acquisition of genetic determinants which circumvent drug response. With trametinib-resistant cells being highly sensitive to selective SRC inhibition, evolutionary-guided combinatorial treatment schedules may provide for strategies to significantly lengthen time taken for resistance development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c54926c3fb10216fcf5c70e35738abc3a296f32" target='_blank'>
              DDDR-39. Untangling MEK inhibitor resistance in DMG-MAPK using coupled lineage tracing and single cell RNA sequencing
              </a>
            </td>
          <td>
            Molina Das, Y. Grabovska, Haider Tari, A. Mackay, Rebecca F Rogers, Jared Collins, Jiin Song, Laura Bevington, A. Burford, Drenusha Sejdiu, R. Beroukhim, P. Bandopadhayay, Chris Jones
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>152</td>
        </tr>

        <tr id="Germ cell tumors (GCTs) are rare, heterogeneous neoplasms derived from primordial germ cells. Although they typically develop in the gonads, they may also arise in extragonadal locations along the midline of the body. Approximately 90% of patients respond well to cisplatin-based chemotherapy; however, ~30% exhibit treatment resistance. Epithelial-mesenchymal plasticity (EMP), a recognized hallmark of cancer, has been implicated in promoting metastasis and chemoresistance. Nonetheless, studies investigating the specific role of EMP in GCT treatment resistance remain limited. The present review compiles key studies on GCTs, EMP markers and cisplatin resistance using both in vitro and in vivo models; it highlights the roles of associated genes, transcription factors and proteins, identifying potential therapeutic targets. Advancing our understanding of EMP and identifying novel therapeutic targets may support the development of treatment strategies that complement or replace cisplatin. This, in turn, could improve survival outcomes and create new avenues for molecular research and clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/731201ecd287c2222a955e4ad4ca52b43518aba0" target='_blank'>
              Epithelial-mesenchymal plasticity and cisplatin resistance in germ cell tumors: Mechanisms and emerging therapeutic strategies (Review)
              </a>
            </td>
          <td>
            Eleni Solange De Brito Gomes, Tainá De Souza, Ana Laura Oliveira, Ana Flavia Bezerra, Ingridy Izabella Vieira Cardoso, Lenilson Silva, Luiz Fernando Lopes, Marcela Nunes Rosa, Mariana Tomazini Pinto
          </td>
          <td>2025-10-13</td>
          <td>International Journal of Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Objectives Immunotherapies, including chimeric antigen receptor T-cell (CAR-T) therapy, represent a pivotal approach in the treatment of multiple myeloma (MM). However, the complex immunosuppressive tumor microenvironment (TME) poses significant challenges to their efficacy. Among the immunosuppressive cells in the MM TME, granulocytic myeloid-derived suppressor cells (G-MDSCs) are predominant, yet their functions remain incompletely understood. This study aimed to comprehensively analyze the role of G-MDSCs in MM patients undergoing CAR-T therapy. Methods Single-cell transcriptomic data from seven MM patients before and after CAR-T therapy were analyzed to characterize G-MDSCs. Functional enrichment and gene set enrichment analysis (GSEA) were performed to identify signaling pathways. A risk prediction model was constructed using RNA-seq and survival data from 713 MM patients, and validated by Kaplan–Meier analysis. In vitro experiments were conducted to assess the immunosuppressive functions of G-MDSCs. Results The findings, though exploratory due to limited sample size and inter-patient heterogeneity, suggested pathological activation and immunosuppressive roles of G-MDSCs potentially linked to patient prognosis. G-MDSCs were identified as key modulators of immune responses within the TME, with GSEA indicating regulation via IFN-α/γ signaling. They may facilitate immune evasion of MM cells by promoting proliferation through the IGF1–IGF1R axis and inhibiting T cells and other immune cells via the SIRPA–CD47 pathway. The risk prediction model based on G-MDSC gene signatures demonstrated high prognostic accuracy (AUC = 0.94). PTGS1 was identified as a key marker associated with high-risk groups, and functional assays confirmed its role in mediating G-MDSC immunosuppressive activity. Conclusions This study provides preliminary insights into the functional role of G-MDSCs in the MM TME and highlights their impact on CAR-T therapy outcomes. The findings suggest potential therapeutic strategies, including targeting PTGS1, to enhance CAR-T efficacy. Larger and more diverse cohorts are required to substantiate these observations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e8ff33894cad405d661ae879b8aae448cec342e" target='_blank'>
              Functional role of granulocytic myeloid-derived suppressor cells in CAR-T therapy: insights from single-cell RNA sequencing in multiple myeloma
              </a>
            </td>
          <td>
            Chaoyu Zhang, Fang An
          </td>
          <td>2025-11-02</td>
          <td>Immunopharmacology and Immunotoxicology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Acute myeloid leukemia (AML) remains a clinical challenge with suboptimal long-term survival. While circular RNAs derived from the RNF220 host gene have been implicated in AML pathogenesis, the functional role and regulatory mechanisms of RNF220 itself in AML are poorly understood. Methods We integrated bioinformatics analyses of public databases (TCGA-LAML, TARGET-LAML) and local cohort with in vitro functional assays. RNF220 was knocked down and overexpressed in AML cell lines using lentivirus. Transcriptomic profiling (RNA-seq), metabolic pathway enrichment (GSVA, GSEA), and immune microenvironment deconvolution (xCELL, CIBERSORT, MCP-counter) were performed. Transcription factor binding sites were predicted across five databases (JASPAR, ENCODE, GTRD, etc.). Validation of transcriptional regulation was performed using ChIP-PCR and luciferase reporter assays. Results RNF220 overexpression correlated with poor prognosis in AML, drove an immunosuppressive microenvironment characterized by reduced CD8+ T cells, and inhibited NK activity and M2 polarization of macrophage. RNF220 promoted tumor proliferation by suppressing apoptosis and preventing G1 arrest. Knockdown of RNF220 dysregulated metabolic pathways, notably suppressing glycolysis and phenylalanine metabolism. Mechanistically, FOXA1 was identified as an upstream negative regulator of RNF220, where high FOXA1 predicted favorable survival and inversely correlated with RNF220-associated metabolic reprogramming. Conclusion NF220 acts as an oncogenic ubiquitin ligase in AML by coordinating dual pro-leukemic mechanisms: cell-intrinsic metabolic rewiring (glycolysis/phenylalanine) and immune evasion via microenvironment suppression. Targeting the FOXA1–RNF220 axis may offer novel therapeutic strategies for high-risk AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4ff73f3a35554c15311058c2d83eb2628a0504d" target='_blank'>
              Dual oncogenic role of RNF220 in AML: linking metabolic rewiring to cell proliferation and immune evasion
              </a>
            </td>
          <td>
            Bi-xia Li, Shi Jiang, Yao Xu, Xiao Yan, Qitian Mu, Guifang Ouyang
          </td>
          <td>2025-10-30</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Anaplastic lymphoma kinase (ALK), a member of the receptor tyrosine kinase family, plays a central oncogenic role in the initiation and progression of diverse malignancies. Aberrant ALK activation generally results from structural alterations or dysregulated expression, leading to persistent activation of downstream signaling pathways that drive tumor cell proliferation, survival, and metastasis. ALK gene abnormalities predominantly encompass fusions, point mutations, and amplifications, with EML4-ALK-positive non–small cell lung cancer representing a canonical example. The advent of ALK-targeted inhibitors has constituted a major therapeutic milestone for ALK-positive tumors. From first-generation Crizotinib to third-generation Lorlatinib, successive agents have been refined for target selectivity, central nervous system penetration, and coverage of resistance-associated mutations, substantially improving patient survival and intracranial disease control. Nonetheless, the emergence of acquired resistance remains an overarching challenge, mediated by secondary kinase domain mutations, activation of bypass signaling pathways, and tumor phenotypic transformation. This review presents an integrative synthesis of ALK-targeted therapeutic developments, elucidates underlying resistance mechanisms, and surveys emerging strategies, providing a comprehensive perspective on current advances and future directions in precision management of ALK-driven malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74ec5cc87a27a59b857a6b6506e0949481a4fc3f" target='_blank'>
              ALK-Targeted Therapy: Resistance Mechanisms and Emerging Precision Strategies
              </a>
            </td>
          <td>
            Ya‐kun Zhang, Jianbo Tong, , Zhi‐Peng Qin, Rong Wang
          </td>
          <td>2025-11-27</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Reconstructing and understanding intra-tumor heterogeneity, the coexistence of multiple genetically distinct subclones within the tumor of a patient, and tumor development is essential for resolving carcinogenesis and for identifying mechanisms of therapy resistance. While bulk sequencing can provide a broad view on tumoral complexity/heterogeneity of a patient, single-cell analysis remains essential to identify rare subclones that might drive chemotherapy resistance. In this study, we performed an integrated analysis of bulk and single-cell DNA sequencing data of core-binding factor acute myeloid leukemia patients, defined by the presence of a RUNX1::RUNX1T1 or CBFB::MYH11 fusion gene. By single-cell sequencing, we inferred tumor phylogenies for 8 patients at diagnosis including patient-specific somatic variants, somatic copy-number alterations and fusion genes, and studied clonal evolution under the pressure of chemotherapy for 3 patients. As a result, we developed an approach to reliably integrate subclonal somatic copy number alterations into phylogenetic trees and clonal evolution analysis, obtaining unprecedented resolution of intra-tumor heterogeneity in CBF AML. We were able to show that the fusion gene is among the earliest events of leukemogenesis at single-cell level. We identified remaining tumor clones in 6 patients with complete remission samples indicating incomplete eradication of the tumor clones. Here, we show that identifying the order of mutation acquisition can provide valuable insights into evolutionary history, offering a framework to improve drug selection in the era of targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f79dc8d9b0526060e06d2b2413c27f6af3f0f4c7" target='_blank'>
              Resolving intra-tumor heterogeneity and clonal evolution of core-binding factor acute myeloid leukemia patients with single-cell resolution
              </a>
            </td>
          <td>
            Raphael Hablesreiter, Paulina M. Strzelecka, Klara Kopp, Natalia Estrada, A. Dolnik, Marlon Tilgner, Coral Fustero-Torre, Felicitas R. Thol, F. Heidel, M. Heuser, L. Haghverdi, Lars Bullinger, F. Christen, Frederik Damm
          </td>
          <td>2025-10-28</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="


 Over 75% of PTCL-NOS patients fail to achieve long-term control with current therapies, highlighting the need for novel therapeutic targets. Genetic and non-genetic drivers in cancer cells require external cues from other cancer cells and from the lymphoma microenvironment (LME), which enable cancer cell adaptation to metabolic stress, immune mediated stress and treatment exposure. The resulting signaling crosstalk propels the co-evolution of cancer and LME cells with emergence of diverse cancer-supporting cell states. Identifying these and their maintenance mechanisms may therefore reveal novel therapeutic targets.
 We characterized the LME of PTCL using transcriptional signature deconvolution and multiparametric imaging in 845 patient samples. We identified four biologically and clinically distinct categories named Depleted (DP), B-cell rich (BR), Mesenchymal (MS) and Inflammatory (IN) based on the proportion of different LME cells. RNA-seq analysis showed that SMA+ cancer associated fibroblasts (CAF) are present in DP, MS and IN LMEs each with distinct phenotypes. To functionally analyze the contribution of CAFs to PTCL maintenance, we established multiple patient-derived tumor xenografts (PDXs) that retain the same LME categories of primary tumors. We then tested lymphoma cell survival in co-cultures of CAFs and PTCL cells from matched (e.g., IN-CAF + IN-PTCL) and unmatched PDXs (e.g., IN-CAF + DP-PTCL). Matched co-cultures sustained lymphoma cell proliferation to a higher extent than unmatched, independently of LME subtype. This suggests that CAF are distinctly reprogrammed within each LME.
 We hypothesize that pro-tumoral CAF phenotypes require adaptive gene expression in coordination with lymphoma cells' phenotypes. RNA-Seq of matched IN-CAF co-cultured with IN-PTCL revealed activation of pathways like “Transcription”, “DNA replication” and “Translation” including the upregulation of CDK7. Although CDK7 plays a minor role in cell cycle progression, its phosphorylation of the RNA polymerase II as part of the TFIIH complex is critical in activating and coordinating “cell state”-defining transcription.
 To assess CDK7's role in establishing lymphoma-supportive CAF states, IN-PTCL PDX mice were treated with the covalent selective CDK7 inhibitor YKL-5-124 for 12 h followed by transcriptomic and proteomic profiling of IN-PTCL and IN-CAFs. In IN-CAFs, CDK7 inhibition altered gene and protein expression of several collagens and cytokines, including COL1A1, COL1A2, MMP8, and CXCL12. Ligand-receptor analysis identified CXCL12-CXCR4 as the key CDK7-dependent interaction between IN-CAFs and IN-PTCL cells. IN-PTCL cells showed activation of DNA damage repair pathways, including upregulation of ALYREF and EIF4E, likely as compensatory mechanisms for decreased CDK7-dependent transcription. Since these may represent acquired therapeutic vulnerabilities, we conducted viability screenings in co-cultures of IN-PTCL and IN-CAF isolated from YKL-5-124 (and vehicle) tretated PDX mice. Cells were subsequently exposed for 48 h, individually and in co-cultures, to a drug library containing 40 clinical-phase compounds. This approach led to the identification of 14 compounds with synergistic anti-lymphoma effects. Among these, ten drugs including the XPO1 inhibitor Selinexor, the HSP90 inhibitor PU-H71 and the translation inhibitor Omacetaxine exhibited enhanced anti-lymphoma activity specifically in the co-culture setting, whereas four compounds showed reduced efficacy. This was not seen in mismatched co-cultures and indicates a potential protective role of CAFs and, importantly, that this mechanism is CDK7 dependent.
 To address the limitations of PDX models established in immunodeficient mice with limited ability to capture the immune-mediated effects of drug treatments, we developed an IN-PTCL syngeneic model by intrasplenic implantation of the murine T-cell lymphoma cell line EL4. Treatment with YKL-5-124 significantly reduced the tumor burden compared to vehicles. Cytokine profiling (immunoblotting array) and ELISA of culture medium of YKL-5-124 treated EL4-CAFs showed reduced levels of pro-inflammatory cytokines such as CXCL12, IL6, CCL2, ICAM-1, and CXCL1 indicating that CDK7 is necessary in establishing pro-lymphomagenic IN-CAF phenotypes.
 In conclusion, CDK7 activity is required to sustain specific pro-tumoral crosstalk interactions of matched CAFs and PTCL cells, which can be exploited therapeutically.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/386e7a67f6fb5e2e6ca873d1aaf4d401efdcd89c" target='_blank'>
              CDK7 enables adaptive gene transcription required by stromal cells to support distinct T-cell lymphoma phenotypes and represents a novel microenvironment-directed therapeutic target
              </a>
            </td>
          <td>
            K. Kouidri, D. Fiore, M. Revuelta, N. Di Siervi, Federico Ruiz Moreno, Subhashini Joshi, C. Kayembe, Abigail Taylor, G. Inghirami, L. Cerchietti, N. Zamponi
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="


 Acute myeloid leukemia (AML) is an aggressive and biologically heterogeneous hematologic malignancy. It is characterized by the clonal expansion of aberrant myeloid progenitor cells and predominantly affects older adults. As many elderly patients are not suitable for intensive chemotherapy or allogeneic hematopoietic stem cell transplantation, the five-year overall survival rate remains below 10%.
 The emergence of venetoclax, a novel, orally bioavailable selective BCL-2 inhibitor, provides a promising therapeutic option for elderly AML patient population. However, resistance to venetoclax develops in approximately 30% of patients, posing a major clinical challenge. Although accumulating evidence implies metabolic reprogramming in venetoclax resistance, effective strategies to reverse this process remain elusive.
 To recapitulate the stepwise acquisition of venetoclax resistance, we first generated MOLM-13 sublines exhibiting intermediate and complete resistance. Whole-exome sequencing of the derived lines identified two BCL-2 mutations. Interestingly, the progressively increasing variant allele frequencies correlate with the resistance severity, implicate these mutations as potential molecular drivers of venetoclax resistance. Remarkably, a relapsed AML patient after allogeneic HSC transplantation, went under venetoclax treatment for the relapse, became venetoclax-resistant while developed the same two BCL-2 mutations in patient serial leukemic cell samples, underscoring the clinical relevance of these findings.
 To elucidate the molecular alterations associated with venetoclax resistance, we firstly performed transcriptomic and proteomic profiling on the resistant MOLM-13 sublines. These analyses revealed a marked upregulation of lipid metabolism pathways in resistant cells compared to wild-type. This finding prompted us to further investigate whether metabolic reprogramming functionally contributes to resistance. To this end, we conducted a metabolism-focused CRISPR-Cas9 screen in wild-type MOLM-13 cells under venetoclax pressure. The screen identified 175 and 191 negatively selected genes at day 8 and 16, respectively, with 30 genes overlapping at both time points. Pathway enrichment analysis highlighted lipid metabolism as a key pathway, with PTEN prominently enriched, suggesting a potential role in mediating venetoclax resistance via metabolic regulation.
 We next sought to validate the functional role of PTEN in modulating venetoclax response. In vitro, genetic ablation of PTEN using CRISPR/Cas9 or shRNA significantly increased venetoclax-induced apoptosis in both parental and resistant AML cell lines. Pharmacological inhibition of PTEN with small-molecule inhibitors (SF1670 or bpV(HOpic)) produced similar sensitizing effects. Importantly, these findings were corroborated in primary AML samples, where PTEN suppression enhanced venetoclax responsiveness. In vivo, PTEN inhibition in xenograft models led to increased apoptotic response upon venetoclax treatment, further confirming its critical regulatory role in venetoclax response.
 Mechanistically, PTEN loss led to pronounced lipid droplet accumulation, as confirmed by MS/MS-based lipidomic profiling showing elevated fatty acid levels. 13C-labeled palmitate tracing showed reduced incorporation of fatty acids into the TCA cycle, indicating impaired β-oxidation. Concomitantly, upregulation of FASN and SREBP suggested increased lipid synthesis. This dual effect of reduced lipid degradation and increased synthesis resulted in intracellular lipid accumulation, which caused mitochondrial damage, as evidenced by elevated ROS levels. This mitochondrial stress may contribute to reduced resistance to venetoclax by promoting apoptosis, providing a mechanistic link between PTEN loss, lipid metabolism reprogramming, and enhanced drug sensitivity.
 In conclusion, we identify PTEN as a key lipid metabolic regulator of venetoclax sensitivity in AML. By linking lipid metabolic reprogramming to apoptotic vulnerability, we uncover a mechanistic basis for venetoclax sensitization. Our findings highlight the potential of targeting lipid metabolic pathways as a strategy to overcome venetoclax resistance in AML, particularly in the context of BCL-2 mutations.
 MT.C, LZ.C and YJ.J contributed equally to this work.
 Corresponding authors: WP.Y, EL.J and YJ.C.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c622b6cf0575b845ded4dfa8ce291137a9afa72e" target='_blank'>
              PTEN loss reprograms lipid metabolism to modulate venetoclax resistance in acute myeloid leukemia
              </a>
            </td>
          <td>
            Mutian Cao, Lizhen Cong, Yujiao Jia, Yifei He, Mingyue Hao, Yuanyi Ling, Wanting Li, Yajing Chu, E. Jiang, Weiping Yuan
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="


 Acute myeloid leukemia (AML) is driven by diverse genetic alterations in hematopoietic precursor cells and has a high rate of relapse. Among the most common changes are mutations in NPM1, encoding nucleophosmin. Recent work suggests that mutant NPM1 sustains leukemogenic transcription of HOXA/B genes and MEIS1 by coordinating with RNA Polymerase II and the MLL/menin complex as well as by inhibiting histone deacetylases. It is hence critical to better understand the epigenetic mechanisms governing leukemogenesis and treatment resistance. Here, we sought to comprehensively characterize the gene regulatory landscape of NPM1-mutantAML at diagnosis and relapse by analyzing the chromatin accessibility, gene expression, and surface protein abundance of AML cells at single-cell resolution – together with NPM1 mutational status.
 We developed GoT-DOGMA-seq, a single-cell method integrating Genotyping of Transcriptomes (Nam et al., 2019)for NPM1 genotyping with DOGMA-seq (Mimitou et al., 2021), which captures chromatin accessibility, gene expression, and surface protein abundance information from the same individual cells. We applied this method to paired bone marrow mononuclear cells (BMMCs) from three patients at diagnosis of AML (n = 16,793 cells) and after relapse following chemotherapy or venetoclax (n = 22,191 cells) and compared to BMMCs from healthy controls (n = 33,375 cells across 7 individuals).
 Malignant cells, identified through NPM1 genotyping, immunophenotyping, copy number alterations, and assessment of AML gene expression signatures, exhibited pronounced heterogeneity at diagnosis and relapse, reflecting varied myeloid cell types. Cell states characterized by chromatin accessibility and gene expression were correlated, but there were distinctions. For example, HSC/MPP-like malignant cells subdivided into states related to differential stemness, myeloid chromatin priming, and transcription factor (TF) activity, as well as copy number alterations (e.g., chr17p loss), inflammatory signaling (e.g., TNF-alpha and IL-1 signaling), and mitochondrial energetics (e.g., oxidative phosphorylation).
 Overall, chromatin accessibility data revealed 167,959 candidatecis-regulatory elements (cCREs). These cCREs covered 88% of ENCODE elements, suggesting consistency with the database as well as recognition of novel cCREs. 98,698 cCREs (59%) were uniquely accessible in HSC/MPP-like malignant cells and not in control cells. Nearly 51% of these elements were located in introns, 27% were in distal intergenic loci, ~11% were in promoters, ~7% were in exons, and the remainder were in untranslated regions or other downstream loci. We hypothesized that these identified cCREs help to regulate myeloid leukemogenesis or pathways of treatment resistance.
 Of the leukemia-associated cCREs, diagnosis-specific cCREs (13%; n = 12,957) were enriched for motifs of transcription factor (TF) regulators of normal and abnormal myeloid differentiation, including HOXA/B, MEIS1, as well as CEBP, RUNX, and AP-1 TFs. Chemotherapy relapse-specific cCREs (12%; n = 11,980) were enriched for IRF, STAT1, ZEB1, and SNAI1 motifs, and linked to the expression of genes involved in interferon signaling, altogether suggesting a shift toward inflammatory, resistance, and self-renewal pathways in relapse. cCREs significantly more highly accessible in venetoclax relapse cells compared to controls were linked to the expression of genes in the NFkB, RAS-RAF-MEK-ERK, and PI3K-AKT-mTOR signaling pathways, including MCL-1, which are known to mediate acquired resistance to venetoclax.
 We have analyzed NPM1-mutantAML samples at single-cell resolution and shown that AML cells at diagnosis and relapse are highly heterogeneous both epigenetically and transcriptionally. We have demonstrated that integrating chromatin accessibility and gene expression data enables more precise annotation of malignant cell states. We have also identified cCREs that may help to coordinate gene expression programs associated with leukemogenesis and treatment resistance, highlighting potential regulatory vulnerabilities.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e038464d40880886ad86d66843aa02811f821f96" target='_blank'>
              Single-cell multiomics reveals candidate cis-regulatory elements in NPM1-mutant AML
              </a>
            </td>
          <td>
            Elliot O. Eton, S. Ganesan, JangKeun Kim, Kai Beattie, D. Barisic, Christopher Mason, D. Landau
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cancer cells rapidly induce PD-L1 expression in response to inflammatory cytokines such as IFNγ from cytotoxic T cells. Increased surface PD-L1 is a primary mechanism of cancer cells evading cytotoxic T-cell-mediated immune clearance. Identifying how cancer cells increase PD-L1 expression may yield clinically relevant immune checkpoint regulators. However, the key regulators and molecular mechanisms mediating rapid PD-L1 induction are yet to be understood entirely. To identify targetable mechanisms controlling cytokine-induced PD-L1 expression, we performed functional CRISPR gene KO screening with a custom-designed sgRNA library that targets “druggable” genes. We performed the screening in 6 different cancer lines: 3 ovarian (OVCAR4, CaOV3, and SKOV3) and three pancreatic cancer (MiaPaca2, ASPC1 and KP4) cell lines. The screening recovered the known regulators of PD-L1 expression and uncovered several novel regulators of PD-L1 that control its expression in all cell lines or in a cancer-type-specific fashion. For example, while genetic or pharmacological depletion of CSNK1A1 results in reduced PD-L1 expression in ovarian cancer cells, CDK1 depletion modulates PD-L1 in pancreatic cancer cell lines. Significantly, we discovered that KEAP1 depletion or pharmacological inhibition diminishes PD-L1 in all cell lines tested (n = 6). Mechanistically, KEAP1 depletion-mediated reduced PD-L1 is due to transcriptional repression of the PD-L1 gene by NRF2 activation. As such, depletion of NRF2 restores PD-L1 expression, while its overexpression leads to diminished PD-L1 expression. Supporting this, pharmacological NRF2 activation resulted in significant antitumor immunity with increased cytotoxic effector T cell infiltration and reduced exhausted T cells, resulting in smaller xenografted tumors. These findings establish the KEAP1/NRF2 axis as a novel and potentially druggable mechanism of IFNγ-meditated PD-L1 expression in cancer cells. CRISPR screening of druggable genes identified KEAP1/NRF2 axis as a novel regulator of PD-L1 expression. KEAP1 depletion or pharmacological inhibition reduces PD-L1 transcription via NRF2 activation, enhancing antitumor immunity in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8d42b79a60457a15da5b80564f464751ddc6948" target='_blank'>
              Druggable genome CRISPR screening identifies the KEAP1/NRF2 axis as a mediator of PD-L1 expression
              </a>
            </td>
          <td>
            Fidan Seker-Polat, Magdalena Rogozinska, Youngho Ban, Fatih Abdula, Kadir Buyukcelebi, Ping Xie, Jie Fan, Neda Abbaszadeh, Yasemin Kingham, Seyedehzahra Paylakhi, Bin Zhang, M. Adli
          </td>
          <td>2025-11-19</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Cutaneous squamous cell carcinoma (cSCC) is the second most common form of non-melanoma skin cancer (NMSC), with a steadily increasing global incidence, especially in populations with prolonged ultraviolet (UV) exposure. Advanced or metastatic cSCC carries a poor prognosis, while the application of targeted therapies and immunotherapies remains in the exploratory stage, due to limited understanding of the underlying mechanisms of cSCC occurrence and progression. Epigenetic dysregulation plays important roles in the progression of cSCC. However, how multi-dimensional regulatory networks reshape the epigenetic landscape and contribute to dysregulated gene expression in cSCC remains unclear. In this study, we performed parallel RNA m6A sequencing, 850K DNA methylation arrays, whole transcriptome sequencing, and ATAC-seq chromatin accessibility profiling on samples from normal skin, actinic keratosis (AK), and cSCC. We analyzed the regulatory networks and pathways of epigenetic modifications on gene expression. We further explored the crosstalk of epigenetic regulatory networks by correlation analysis. By integrating single-cell RNA-seq data, we identified epigenetically upregulated candidate genes and confirmed their expression and functions with experimental methods. Our integrated multi-omics analysis provides a comprehensive and dynamic epigenetic map of cSCC progression. Further analysis revealed that DNA methylation and m6A modification jointly regulate gene expression through independent and synergistic ways. The identified epigenetically upregulated candidate genes IDO1, IFI6, and OAS2 were validated to be overexpressed in cSCC tissues and cell lines, and functional assays confirmed their potential key roles in regulating the processes of cell proliferation, migration and invasion in cSCC. By integrating multi-omics data, this study systematically highlights the multi-layered epigenetic alterations and regulatory mechanisms involved in cSCC development. This multi-stage, multi-omics, and multi-resolution integrated analysis provides a theoretical basis and new insights for future personalized treatment strategies for cSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdeb7e8c78575b1d7e37855326ebd04eb68da2c1" target='_blank'>
              Multi-omics analysis reveals the crosstalk of epigenetic regulatory networks in cutaneous squamous cell carcinoma progression
              </a>
            </td>
          <td>
            Ya-Zhou Sun, Dan-Dan Zou, Xin-Jie Li, Xiao-Li Wang, Jia-Hao Feng, Jia-Ying Liu, Ke He, Yu-Tong He, Xian-Xin Lai, Xin Li, Xuan Jiang, Tian-Shun Gao, Li He
          </td>
          <td>2025-11-13</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Leukemia evolution is driven by a complex interplay between somatic mutations and epigenetic changes. However, our understanding of these processes is limited by a lack of scalable and cost-effective methods to co-assay genetic and epigenetic changes in single cells. Here, we present scGEM-seq (single-cell Genotyping and Methylome sequencing), a flexible multi-omics platform that permits simultaneous characterization of genetic variants and DNA methylation profiles from single cells. scGEM-seq combines three core innovations that can be accomplished at ~3% the cost ($0.03 per cell) of competing technologies (>$1 per cell in snmC-seq3 and SciMETv2): (1) single-cell capture with semi-permeable capsules (Atrandi Biosciences), enabling multi-step enzymatic reactions to be performed efficiently with minimal loss of DNA; (2) a novel split-pool strategy (HexaDecimal barcoding) that efficiently generates ~1 million unique cell barcodes that are compatible with bisulfite-like enzymatic conversions; and (3) flexible library preparation that can be adapted for genome-wide or targeted sequencing.
 In cell-mixing studies, we show that scGEM-seq accurately resolves cell populations based on DNA methylation profiles and genetics. At modest sequencing depth (0.001X per cell for whole-genome DNA libraries, 15734.4±12123.1 CpGs per cell for methylation libraries), scGEM-seq resolved mouse and human cell lines at expected ratios (1:1:1 ratio of GM12878, K562 and YAC1 cells) with low doublet rates (~2.4% using species genome alignment). scGEM-seq also resolved human cancer cell lines of distinct lineages (1:1:1 ratio of SW480, A375 and OCIAML3 cells) according to methylation and genetics (cell-specific SNVs and copy-number changes) with modest sequencing depth (0.002X per cell for whole-genome DNA libraries, 20242.6±12808.2 CpGs per cell for methylation libraries) and low doublet rates (2.88% using cell-specific SNVs).
 To enhance genotyping efficiency, we also combined scGEM-seq with two different target enrichment strategies: hybrid capture (Pan-cancer hybridization panel, IDT; n=127 genes) and amplicon sequencing. In mixed human cancer cell lines, scGEM-seq achieved consistent single-cell genotyping for known cancer genes and hotspot mutations, including a mean of 13.23±5.30% of cells for hybrid capture and >98% of cells for amplicon sequencing. To determine the sensitivity of scGEM-seq for genotyping of rare cells, we prepared a cell mixture containing varying proportions of human cancer cell lines (GM12878: 92.85%, SW480: 5%, A375: 1.5%, OCIAML3: 0.5%, K562, 0.15%). scGEM-seq coupled with amplicon sequencing for known cell-specific variants recovered the expected cell proportions down to ~0.1-0.2%, representing fewer than 2 malignant cells per 1000 (SW480: 302/6294 cells, 4.80%; A375: 88/6294 cells, 1.40%; OCIAML3: 30/6294 cells, 0.48%; K562: 11/6294 cells, 0.17%).
 Finally, we applied scGEM-seq to primary bone marrow samples from 4 patients with myeloid neoplasms (MDS and AML) representing a range of blast counts (47%, 52%, 10% and 16%). Here, scGEM-seq revealed distinct populations of normal, premalignant, and malignant cells, including subclonal leukemia cells with KIT D816V (47/777 cells, 6.05%) and T417_D419delinsL (72/700 cells, 10.29%) mutations in a patient with inv(16) AML, and clonal (premalignant) hematopoietic cells harboring a DNMT3A S770L mutation (251/558 cells, 44.98%) and subclonal malignant blasts with distinct KRAS G12C (92/385 cells, 23.9%) and G12D (4/381 cells, 1.05%) mutations.
 In conclusion, scGEM-seq provides a flexible, cost-effective, and high-resolution method for elucidating the complex interplay between epigenetic and genetic cell states in cancer cells, including within rare cell populations. We envision multiple applications for scGEM-seq for the study of leukemia and other blood cancers, including to characterize (epi)clonal heterogeneity and dynamics, identify minimal residual disease, and capture rare stem/progenitor populations or drug-resistant clones that emerge post therapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42da31bde36d790d58830d2a574d2fd507a69d8f" target='_blank'>
              Single-cell genotyping and methylome sequencing (scGEM-seq) resolves cell-states and clonal hierarchies in leukemia
              </a>
            </td>
          <td>
            Kyung Lock Kim, Maria R. Capilla-Guerra, Evan Chen, Andrew Lane, Gabriel Griffin
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Targeted therapy has revolutionized the treatment landscape of hepatocellular carcinoma (HCC), offering unprecedented hope to patients. However, despite its promise, significant challenges arise in the form of drug resistance. Only a fraction of HCC patients respond to targeted therapy, and even those who respond often develop resistance over time. Sorafenib and lenvatinib, the sole first-line targeted therapeutic drugs for HCC, face severe clinical limitations due to drug resistance. Understanding the mechanisms underlying sorafenib/lenvatinib resistance is crucial for maximizing treatment efficacy. Recent studies have highlighted the pivotal role of epigenetic regulation in drug resistance. Cancer is recognized as both a genetic and epigenetic disease, with epigenetic factors influencing various aspects of tumor cell biology, especially drug resistance. This review systematically summarizes the mechanisms of epigenetic-mediated sorafenib/lenvatinib resistance, encompassing non-coding RNA (ncRNA) regulation, DNA methylation, RNA methylation, and histone modification. These abnormal epigenetic processes typically influence biological activities, including escaped programmed cell death, tumor metabolic reprogramming, formation and maintenance of drug-resistant cells, uncontrolled cell proliferation signaling pathways, and abnormal transport processes, ultimately culminating in profound drug resistance. By comprehensively summarizing the latest discoveries in epigenetic regulation mechanisms, this review highlights potential strategies to overcome drug resistance, paving the way for future advancements in HCC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c8f7f2274db91d56455b33e92f98f5f8697e415" target='_blank'>
              Resistance of first-line targeted drugs in hepatocellular carcinoma: the epigenetic regulation mechanisms
              </a>
            </td>
          <td>
            Ding Qi, Yongqing Qin, Haidong Zhu, Yong Li, Shisong Han
          </td>
          <td>2025-11-26</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Melanoma shows heterogeneity across body sites like skin, acral skin, and the uvea, driven by molecular characteristics and genetic variations. However, comparative studies exploring the heterogeneity of melanoma across different anatomical sites remain limited, hindering a comprehensive understanding of its underlying biology. We proposed a research framework through bioinformatics to analyze the tumor ecosystems of cutaneous, acral, and uveal melanoma, from molecular characteristics to genetic variations at single-cell resolution. We found that oxidative phosphorylation (OXPHOS) is a critical driver of tumor cell evolution, with abnormal ribosomal gene and tumor suppressor expression observed in uveal melanoma (UM). Additionally, we screened for potential drug targets and drugs against tumor cells. In the immune microenvironment, acral melanoma (AM) and UM exhibit stronger immunosuppressive characteristics compared to cutaneous melanoma (CM). OXPHOS contributes to T cell cytotoxicity dysregulation in CM and AM, while interferon-γ is crucial in UM. Tumor cells may also induce T cell dysfunction through biological signals such as MIF-CD74 and HLA-E-NKG2A. This study offers valuable insights into melanoma heterogeneity, providing a comprehensive research framework for understanding the distinct molecular and immune characteristics of CM, AM, and UM, and potentially guiding the development of therapeutic strategies tailored to each melanoma subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da658ce35ad37afa8afa0fc4adb1c52869ddb7f4" target='_blank'>
              Dissecting Melanoma Ecosystem Heterogeneity from Molecular Characteristics to Genetic Variation at Single-Cell Resolution
              </a>
            </td>
          <td>
            Congxue Hu, Liyuan Li, Tengyue Li, Baobin Qi, Wanqi Mi, He Yu, Kaiyue Yang, Qi Ou, Xia Li, Yunpeng Zhang
          </td>
          <td>2025-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Somatic evolution drives cancer progression and therapy resistance. Underlying phenotypic progressions is the development of marked clonal heterogeneity in most cancers. However, deciphering the downstream molecular effects of clonal heterogeneity in primary cancer has been limited due to the inability to isolate subclones by standard methods. While single-cell RNA-sequencing (scRNA-seq) provides high-resolution mapping of heterogeneous cell states, it cannot link transcriptional programs to cancer-driving mutations in these individual cells. We and others developed methods to detect genotypes and their transcriptional outputs in single cells (e.g., Genotyping of Transcriptomes; Nature, 2019). However, these approaches have technical limitations to broadly profile numerous mutations, which is nonetheless required for clonally complex neoplasms. The existing approaches are also restricted to fresh or frozen samples, limiting the use of the widely available pathology formalin fixed paraffin embedded (FFPE) tissues.
 To address these challenges, we developed Genotyping of Transcriptomes for Multiple Targets and Sample Types, GoT-Multi, a next generation single-cell multi-omics method compatible with FFPE tissues that integrates multiplexed genotyping with transcriptomic profiling. GoT-Multi adapts a probe-based scRNA-seq method (Flex Fixed RNA Profiling, 10x Genomics) by including custom mutation-specific probes along with the standard transcriptional probes. Furthermore, we developed an ensemble-based machine learning pipeline, GoT-Multi-ML, that optimizes the genotyping calls by denoising technical artifacts. GoT-Multi-ML leverages multiple machine learning models to learn the non-linear patterns of the genotyping data features (e.g., reads per genotyping transcript, GC content of probes), agnostic to the whole transcriptomic data. We validated GoT-Multi via a cell line-mixing experiment using SK-BR-3 and MCF-7 cell lines. GoT-Multi detected the targeted mutations (HIST1H1C A24T and S100A10 A77V) in the expected cell line (MCF-7) with an average genotyping rate of 72% and accuracy of 97%.To define the impact of clonal evolution on phenotypic progression, we focused on a prototype of cancer evolution, the progression of chronic lymphocytic leukemia (CLL) to therapy-resistant large B-cell lymphoma (LBCL) called Richter Transformation (RT). We applied GoT-Multi to a unique cohort of primary nodal cryopreserved samples containing both CLL and LBCL components (n = 5). We targeted 18 mutations (1-6 targets per sample) with a 98% median accuracy. We observed heterogeneous cancer cell states, including proliferating, stress response and inflammatory (n = 51,465 cells). LBCL was enriched in proliferating and cyclin D2-elevated states, whereas CLL was associated with a stress response signature. GoT-Multi enabled us to reconstruct the clonal architectures, up to 4 distinct subclones per lymphoma, and their associated cell states. Differential gene expression between mutated and wildtype cells revealed that genetically heterogenous subclones, including those with SRRM2 and JUNB mutations and therapy resistance-associated mutations in PLCG2 and BTK, displayed enrichment in inflammatory cell states, upregulating TNF and interferon signaling genes while suppressing cell cycle-related genes. Conversely, other subclones with mutations in, e.g., IRF8, POU2F2, PRKDC were associated with MYC activation and/or enhanced proliferation. These findings suggested that heterogeneous genotypes may converge on similar downstream transcriptional cell states. In addition, TNF itself was overexpressed in the proliferative subclones, suggesting that TNF production from these subclones may enhance inflammatory signaling of the TNF-responsive subclones, indicative of crosstalk between subclones. Finally, we demonstrated the applicability of GoT-Multi to pathology archived FFPE tissues. We profiled 9 mutations with an accuracy of 98% in an FFPE RT sample (n = 2,140 nuclei), allowing us to detect 5 subclones and their associated cell states. In summary, co-mapping of clonal and cell state heterogeneity at single cell resolution through GoT-Multi suggested that heterogeneous subclonal genotypes may converge on similar downstream oncogenic pathways to enhance overall tumor fitness. We envision that the broad applicability of GoT-Multi may help uncover the molecular underpinnings of cancer progression and therapy-resistance across oncology.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/954b694c778f5e54689dba8e5834b825525f28c2" target='_blank'>
              Co-mapping clonal and transcriptional heterogeneity in somatic evolution via got-multi.
              </a>
            </td>
          <td>
            Minwoo Pak, M. Saurty-Seerunghen, Kellie Wise, Tsega-Ab Abera, Chhiring Lama, Neelang Parghi, Xiaotian Sun, Qi Gao, Liming Bao, Mikhail Roshal, John N Allan, Richard Furman, Luciano G. Martelotto, Anna S. Nam
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM), the most aggressive and prevalent type of brain cancer, presents a formidable therapeutic challenge due to its high degree of drug resistance. Despite advances in chemotherapy, targeted therapy and immunotherapy, patient outcomes remain dismal, with a 5-year survival rate of only 7%. Resistance mechanisms are multifactorial, including the restrictive nature of the blood-brain barrier (BBB), tumor heterogeneity and adaptive responses within the tumor microenvironment (TME). The BBB limits drug delivery, while efflux transporters further reduce therapeutic efficacy. Additionally, the vast molecular genetic and cellular heterogeneity of a glioblastoma enables the survival of resistant subpopulations, such as glioblastoma stem cells, that evade treatment. Chemotherapy resistance in GBM, particularly to temozolomide, is driven by factors such as O6-methylguanine-DNA methyltransferase upregulation, defective mismatch repair, hypoxia-induced gene expression and activation of several signaling pathways, such as the NF-κB, Hippo and Wnt pathways. Targeted therapies have shown limited success due to activation of compensatory pathways and tumor plasticity, while immunotherapeutic approaches are hindered by an immunosuppressive TME. Recently identified resistance mechanisms, including exosomal transfer of noncoding RNAs and metabolic reprogramming, further complicate treatment. Future directions should emphasize overcoming these challenges through combination therapies, enhanced drug delivery systems and precision medicine approaches. Emerging strategies include targeting persister cells, leveraging metabolic vulnerabilities and integrating AI-driven drug discovery approaches and nanotechnology. Robust patient stratification and biomarker-driven interventions are critical for tailoring therapies and improving outcomes. The present review highlights the urgent need for innovative, multidisciplinary approaches to address the complexity of GBM resistance and advance therapeutic strategies for this lethal disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7873b8c2ed7832370f4bef19a80cb17ac426667" target='_blank'>
              Drug resistance in glioblastoma: Challenges, mechanisms and therapeutic strategies (Review)
              </a>
            </td>
          <td>
            Yash Durga, Raju Pusapati
          </td>
          <td>2025-11-06</td>
          <td>Molecular and Clinical Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Introduction: Acute Myeloid Leukemia (AML) is an aggressive hematologic malignancy characterized by the clonal expansion of undifferentiated myeloid precursors in the bone marrow and peripheral blood. This uncontrolled proliferation is primarily driven by a range of genetic mutations that disrupt normal hematopoiesis and impair the function of healthy leukocytes, erythrocytes, and platelets. Among emerging regulatory mechanisms, N6-methyladenosine (m6A) RNA modification plays a critical role in post-transcriptional gene regulation and cancer progression. The m6A methyltransferase complex (MAC), which catalyzes the methylation of adenosine residues in mRNA, is primarily composed of METTL3 and METTL14. METTL3 serves as the catalytic subunit responsible for methyltransferase activity, while METTL14 provides structural support and enhances substrate recognition and binding. Together, this heterodimeric complex efficiently installs m6A marks on mRNA, thereby influencing RNA stability, translation, and splicing. Dysregulation of m6A modification has been implicated in the pathogenesis of multiple cancers, including AML. This study investigated the regulatory role of METTL3 in AML by reanalyzing publicly available RNA-seq datasets from AML MOLM13 cells treated with ZW27941, a potent small-molecule degrader of METTL3.
 Methods: Human AML MOLM13 cells were treated with either DMSO (control) or ZW27941 for 24 hours. Following treatment, total RNA was extracted, and RNA-seq was performed. Transcriptome data (GSE273361) were reanalyzed to identify differentially expressed genes (DEGs) using R packages (false discovery rate < 0.05 and |log₂ fold-change| > 0.2). Gene ontology (GO) enrichment analysis was conducted to identify biological processes significantly altered upon METTL3 degradation.
 Results: ZW27941 treatment induced substantial transcriptomic changes, with 534 genes upregulated and 641 genes downregulated. The suppressed genes included key drivers of AML progression (e.g., TREM1, S100A8/A9, TLR4, CD84, CXCR3, ITPK1, PECAM1, and ADGRE1), indicating pathways related to innate immune signaling, cytokine-mediated activation, and inflammatory cell adhesion. Downregulation of genes involved in lipid metabolism (e.g., PPARGC1A, ELOVL6, CYP27A1/B1/W1, ABCA1/2/9, and SLC-family transporters) suggests a disruption in fatty acid synthesis and cholesterol homeostasis, pathways known to support leukemic cell survival and drug resistance. Additionally, the downregulation of transcription factors and developmental regulators (e.g., RUNX1, TNR, SOX18, NR2F2, and FOXM1) indicates interference with lineage identity, cell proliferation, and AML subtype specification. Conversely, genes and pathways involved in apoptosis, DNA damage repair, cell cycle arrest, and immune activation (e.g., IL-2, IL-6, TNFα, NF-κB, and JAK-STAT3 signaling), were significantly upregulated in response to METTL3 degradation.
 Conclusion: Our study demonstrates that METTL3 plays a pivotal role in sustaining AML cell survival by regulating genes associated with inflammation, lipid metabolism, proliferation, and immune evasion. Pharmacological degradation of METTL3 by ZW27941 not only suppresses oncogenic transcriptional programs but also activates apoptotic and immune responses. These findings demonstrate METTL3 as a promising therapeutic target in AML and support further exploration of m6A modulators in anti-leukemic strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4ab99e6d63a6b8558aeee4c091c181cedae6e6c" target='_blank'>
              Targeting METTL3 disrupts oncogenic transcriptional programs and activates immune and apoptotic pathways in Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Harshita Ravi, Qian Wang
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Tumor heterogeneity is a fundamental property of leukemia and lymphoma that drives disease progression, relapse, and therapy resistance. For decades, bulk genomic and transcriptomic analyses have provided a foundational understanding of these malignancies but have obscured the cellular diversity within the tumor ecosystem. The advent of high-throughput single-cell technologies, particularly single-cell RNA sequencing (scRNA-seq), has revolutionized our capacity to deconvolute this complexity. This review synthesizes how single-cell analyses are reshaping our understanding of tumor heterogeneity in hematological cancers. We explore the role of intra-clonal genetic diversity in Acute Myeloid Leukemia (AML) and B-cell Acute Lymphoblastic Leukemia (B-ALL), the functional and transcriptional plasticity of subpopulations, and the dynamic interplay between malignant cells and the immune microenvironment in both leukemia and lymphoma. We further highlight how the tumor microenvironment in lymphomas fosters heterogeneity and immune evasion. Furthermore, the integration of single-cell data with spatial transcriptomics and computational models is beginning to predict evolutionary trajectories, offering a window into future clinical behaviors. Critically, we discuss how single-cell approaches have identified pre-existing or emergent resistant subclones under therapeutic pressure, providing a mechanistic basis for treatment failure. The clinical implications are profound, paving the way for single-cell diagnostics for minimal residual disease (MRD) monitoring and the identification of novel therapeutic targets. We conclude that single-cell multi-omics is an indispensable tool for unraveling the multifaceted nature of tumor heterogeneity, with the potential to propel the field toward more precise and effective therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0baf5679635e6fcbb86a03a048b12087069db02" target='_blank'>
              Tumor Heterogeneity in Leukemia and Lymphoma: Insights from Single-Cell Analyses
              </a>
            </td>
          <td>
            Pang Hou, Wen
          </td>
          <td>2025-11-21</td>
          <td>Hematological Disorders in the Single-Cell Era</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Acquisition of resistance to anti-cancer therapies is a multistep process initiated by the survival of drug-tolerant persister cells. Accessibility of drug-tolerant persister cells in patients is limited, which has hindered understanding the mechanisms driving their emergence. Here, using multiple patient-derived models to isolate persister cells, we showed that these cells are transcriptionally plastic in vivo and return to a common treatment naïve-like state upon relapse, regardless of treatment. Hallmarks of the persister state in TNBC across treatment modalities included high expression of basal keratins together with activation of stress response and inflammation pathways. These hallmarks were also activated in HER2+ breast and lung cancer cells in response to targeted therapies. Analysis of gene regulatory networks identified AP-1, NF-κB and IRF/STAT as the key drivers of this hallmark persister state. Functionally, FOSL1, an AP-1 member, drove cells to the persister state by binding enhancers and reprogramming the transcriptome of cancer cells. On the contrary, cancer cells without FOSL1 had a decreased ability to reach the persister state. By defining hallmarks of TNBC persistence on multiple therapies, this study provides a resource to design effective combination therapeutic strategies that limit resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88dba0b22030a811fd28fd3f8f3324b7f9286705" target='_blank'>
              Characterization of Drug-Tolerant Persister Cells in Triple-Negative Breast Cancer Identifies a Shared Persistence Program across Treatments and Patients.
              </a>
            </td>
          <td>
            Léa Baudre, Gregoire Jouault, Pacôme Prompsy, Melissa Saichi, Sarah Gastineau, Christophe Huret, Laura Sourd, A. Dahmani, Elodie Montaudon, F. Dingli, D. Loew, Elisabetta Marangoni, J. Marsolier, Céline Vallot
          </td>
          <td>2025-11-06</td>
          <td>Cancer research</td>
          <td>1</td>
          <td>54</td>
        </tr>

        <tr id="Leukemia and lymphoma represent a significant burden in Nigeria, characterized by distinct molecular profiles and treatment responses compared to global patterns. This uniqueness can be attributed to several factors including genetic diversity within the Nigerian population, environmental exposures specific to the region, and healthcare access challenges that affect early diagnosis and treatment adherence. The comprehensive understanding of these hematological malignancies has been revolutionized by the advent of single-cell technologies, which have uncovered unprecedented resolution into tumor heterogeneity, evolution, and microenvironmental interactions. Unlike traditional bulk sequencing methods that average signals across thousands of cells, single-cell approaches allow researchers to examine individual cells, revealing rare but critical subpopulations that drive disease progression and therapy resistance.
This review synthesizes findings from Nigerian studies integrated with global advances in single-cell research, highlighting how clonal architecture, cellular ecosystems, and molecular signatures contribute to pathogenesis and treatment outcomes. We explore how single-cell multi-omics approaches—including transcriptomics, genomics, and proteomics—have identified novel therapeutic targets and resistance mechanisms in leukemia and lymphoma. Additionally, we discuss the potential for implementing these technologies in Nigeria to advance precision medicine, considering both the opportunities and challenges specific to the Nigerian healthcare context. The integration of single-cell analyses into clinical practice promises to improve risk stratification, therapy selection, and overall management of these malignancies in resource-limited settings through more targeted and effective treatment approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca65a47e4dd6298d29b7f5a8f9777fdb3fa1474c" target='_blank'>
              Unraveling Clonal Architecture and Microenvironmental Complexity of Leukemia and Lymphoma Using Singlecell Multi-Omics in Nigeria
              </a>
            </td>
          <td>
            Leon Oluci Kochukgu
          </td>
          <td>2025-11-06</td>
          <td>Hematological Disorders in the Single-Cell Era</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) remains a formidable therapeutic challenge, with high relapse rates rooted not only in genetic heterogeneity, but also in dynamic and reversible epigenetic dysregulation that drive resistance to conventional and targeted small-molecule therapies. Although small-molecule inhibitors targeting epigenetic regulators have demonstrated preclinical efficacy and initial clinical activity, therapeutic gains are often limited by transient responses, intra- and inter-patient heterogeneity, and an incomplete understanding of predictive biomarkers. Besides, clinical outcomes have often been less robust than preclinical data suggested, partly due to the complexity of AMLs genetic and epigenetic heterogeneity, the limitations of model systems, and intrinsic resistance mechanisms. Combination approaches, such as pairing epigenetic agents with chemotherapy, immunotherapy, or other targeted drugs are under investigation. However, clinical efficacy remains inconsistent and requires a mechanistic rationale to avoid antagonism. To move forward, it is essential to delineate specific epigenetic dependencies in resistance-prone AML subtypes, design rational, biomarker-guided combination regimens, and enhance specificity and pharmacodynamic profiles of epigenetic inhibitors. Such an integrated strategy holds promise for elevating the effectiveness of epigenetic therapies in AML. Continued research is essential to refine epigenetic-based precision medicine and ultimately improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8e1118798f14d99fa1bf83fc376d07fc1df6c5c" target='_blank'>
              Epigenetic mechanisms of drug resistance in acute myeloid leukemia: advances in small-molecule targeted therapy
              </a>
            </td>
          <td>
            Jialing Tang, Xibo Ye, Zhangyang Xu, Tianjiao Min, Zherun Tian, Li Ma, Ying He, Beier Jiang
          </td>
          <td>2025-10-20</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Introduction Colon cancer is a common disease, treated with few chemotherapeutic agents with similar treatment sequencing despite its heterogeneity. A significant proportion of patients are diagnosed with metastasis, and resistance to antineoplastic drugs is associated with disease progression and therapeutic failure. It is known that the tumor microenvironment plays an essential role in cancer progression, contributing to processes that may be associated with therapeutic resistance mechanisms in colon cancer. In this study, we aim to identify a gene expression signature and its relationship with immune cell infiltration in colon cancer, contributing to the identification of potential resistance biomarkers. Methods An in silico study was conducted using RNA-seq data from The Cancer Genome Atlas Program (TCGA) samples, subdivided into two groups (treatment-resistant and non-resistant), taking into account the molecular subgroups (CMS1, CMS2, CMS3, and CMS4). The following algorithms were used: i. Limma was applied to identify differentially expressed genes; ii. WGCNA was applied to construct co-expression networks; iii. CIBERSORT was applied to estimate the proportion of infiltrating immune cells; and iv. TIMER was applied to explore the relationship between core genes and immune cell content. Results Twenty differentially expressed genes (DEGs) were found, with 18 related to the group considered resistant to oncologic treatment and presenting poorer overall survival. T CD4 memory resting cells and M0 and M2 macrophages were found in more significant proportions in the analyzed samples and more infiltrated in the tumor microenvironment, the higher the expression of some of these resistance DEGs. Additionally, these genes correlate with biological aspects of neuronal differentiation, axogenesis, and synaptic transmission. Conclusion The gene expression signature suggests the presence of differentially expressed synaptic membrane genes, which may be involved in neuronal pathways that influence the tumor microenvironment, potentially serving as future biomarkers. Furthermore, the presence of M0 and M2 macrophages and T CD4 memory resting cells suggests a potential interaction that may play a role in therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd3d04147fc5526a1e2f859a73c8cdc7c99f7a9f" target='_blank'>
              Gene expression profile in colon cancer therapeutic resistance and its relationship with the tumor microenvironment
              </a>
            </td>
          <td>
            P. Doria, G. Rocha, Vanessa Dybal Bertoni, Roberto de Souza Batista dos Santos, Mariana Araújo-Pereira, Clarissa A Gurgel
          </td>
          <td>2025-10-29</td>
          <td>Frontiers in Bioinformatics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Glioblastoma is characterized by highly invasive behavior that contributes to therapeutic failure and recurrence, yet the stability and regulatory mechanisms of the invasive phenotype remain poorly understood. We established a serial selection model in patient-derived glioblastoma stem-like cells to generate an isogenic subpopulation with stable invasive potential (GBinv⁺) and compared it to the non-invasive counterpart (GBnaïve). GBinv⁺ cells showed a ~10-fold increased invasion in transwell and 3D spheroid assays, a phenotype maintained across passages and independent of proliferation. Integrative RNA sequencing and DNA methylation profiling identified 325 differentially expressed genes and ~6,700 differentially methylated CpG sites, enriched in promoters and enhancers. Invasive cells upregulated a distinct secretome enriched in ECM-modulating factors, including COL7A1, ICAM1, and P4HA2. Transcription factor activity was inferred using ISMARA, revealing increased activity of stress-related regulators (ATF4/6, IRF7) and reduced activity of MYC, E2F, and MYCN. Notably, 24 invasion-associated TFs contained CpGs within their core motifs, suggesting methylation-sensitive regulation. Motif enrichment at differentially methylated sites highlighted E-box motifs bound by bHLH factors (MYC, MAX, MNT), with hypermethylated regions showing reduced co-activator marks, whereas hypomethylated enhancers were enriched for MYC/MAX/E2F1 occupancy. Public ChIP-seq data confirmed that methylation states correlate with chromatin accessibility and TF binding. Integrated analysis demonstrated that although the global correlation between methylation and expression was weak, specific genes such as TNC and FLNC exhibited coordinated methylation-expression changes, identifying a subset of epigenetically modulated invasion effectors. These results support a model in which glioblastoma invasion is sustained by epigenetically reprogrammed transcription factor networks rather than transient environmental cues. Future work will use CUT&Tag to map genome-wide methylation-sensitive TF binding, focusing on MYC/MAX/MLX complexes in patient-derived glioma samples.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e0f700028bbeda988f83eab61ca9826e85aa85a" target='_blank'>
              ANGI-07. EPIGENETIC MODULATION OF TRANSCRIPTION FACTOR BINDING DRIVES INVASIVE GLIOBLASTOMA PHENOTYPES
              </a>
            </td>
          <td>
            T. Buhlmann, S. Schreiner, C. Cialini, Vera Laub, P. Nazarov, Arnaud Muller, K. Plate, J. Steinbach, M. Mittelbronn, A. Hau
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Chromosome 8 (chr8) gains are common in cancer, but their contribution to tumor heterogeneity is largely unexplored. Ewing sarcoma (EwS) is defined by FET::ETS fusions with few other recurrent mutations to explain clinical diversity. In EwS, chr8 gains are the second most frequent alteration, making it an ideal model to study their relevance in an otherwise silent genomic context. We report that chr8 gain-driven expression patterns correlate with poor overall survival of EwS patients. This effect is mainly mediated by increased expression of the translation initiation factor binding protein 4E-BP1, encoded by EIF4EBP1 on chr8. Among all chr8-encoded genes, EIF4EBP1 expression showed the strongest association with poor survival and correlated with chr8 gains in EwS tumors. Similar findings emerged across multiple TCGA cancer entities. Multi-omics profiling revealed that 4E-BP1 orchestrates a pro-proliferative proteomic network. Silencing 4E-BP1 reduced proliferation, clonogenicity, spheroidal growth in vitro, and tumor growth in vivo. Drug screens demonstrated that high 4E-BP1 expression sensitizes EwS to pharmacological CDK4/6-inhibition. Chr8 gains and elevated 4E-BP1 emerge as prognostic biomarkers in EwS, with poor outcomes driven by 4E-BP1-mediated pro-proliferative networks that sensitize tumors to CDK4/6 inhibitors. Testing for chr8 gains may enhance risk stratification and therapy in EwS and other cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76ccda8b7403d2005459919c19fc87c7bc32b5f4" target='_blank'>
              High 4E-BP-1 expression associates with chromosome 8 gain and CDK4/6 sensitivity in Ewing Sarcoma.
              </a>
            </td>
          <td>
            C. Funk, Anna C. Ehlers, M. Orth, Karim Aljakouch, Jing Li, Tilman Lb Hoelting, Rainer Will, Florian H. Geyer, A. K. Ceranski, F. Willis, Endrit Vinca, S. Ohmura, R. Imle, Jana Siebenlist, Angelina Yershova, M. Knott, Felina Zahnow, A. Sastre, Javier Alonso, F. Sahm, H. Peterziel, Anna Loboda, Martin Schneider, A. Banito, G. Leprivier, Wolfgang Hartmann, U. Dirksen, Olaf Witt, Ina Oehme, Stefan M. Pfister, L. Romero-Pérez, Jeroen Krijgsveld, F. Cidre‐Aranaz, T. Grünewald, J. Musa
          </td>
          <td>2025-10-16</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Background Extrachromosomal circular DNA (eccDNA) plays critical roles in cancer, yet its landscape in acute myeloid leukemia (AML) remains unexplored. Methods We used CIRCLE-seq and RNA-seq to characterize eccDNA in 12 AML patients and 4 healthy controls. Results AML cells showed significantly increased eccDNA counts and gene involvement versus healthy controls, with distinct size peaks at 202 and 368 bp. eccDNAs localized non-randomly to chromosomes 1, 2, and 10–19, enriching near transcription start sites and regulatory regions. Functional analysis revealed activation of oncogenic pathways (e.g., MAPK, ErbB signaling) in AML-associated eccDNA. Integrative analysis identified 570 genes upregulated at both the eccDNA and mRNA levels, including myeloid leukemia-related genes (e.g., FLT3, RUNX1, and CD33) and oncogenes. Prognostic analysis showed that high expression of these genes correlated with poor outcomes in AML. Conclusions This study unveils the eccDNA landscape in AML by direct CIRCLE-seq, linking its accumulation to transcriptional dysregulation and leukemogenesis, and highlights eccDNA as a potential biomarker and therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1de89beb26c968490d1dcd9a8b5f72aeb3ffa8be" target='_blank'>
              Integrated CIRCLE-seq with RNA-seq to decipher the quantity, localization, and functional features of eccDNA in AML
              </a>
            </td>
          <td>
            Lijuan Gao, Qiongyu Lu, Hao Li, Changgeng Ruan, Zhao Zeng, Suning Chen
          </td>
          <td>2025-11-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Background: Chronic myelomonocytic leukemia (CMML) is a clonal hematopoietic stem cell disorder that combines clinical and biological characteristics of myelodysplastic neoplasms and myeloproliferative neoplasms. Current treatment choices are limited by incomplete understanding of its transcriptional programs and clonal evolution. Clinical heterogeneity necessitates single-cell resolution to resolve subclonal diversity and identify actionable nodes. We applied single-cell transcriptome sequencing to delineate cellular and molecular heterogeneity and to uncover clonal evolution features that may inform actionable targets.
 Methods: Bone marrow aspirates from 5 newly diagnosed CMML patients and 4 healthy donors underwent single-cell transcriptome sequencing. Comprehensive analysis included dimensionality reduction and clustering, gene ontology (GO) and KEGG pathway enrichment of significantly different genes (p<0.05 and fold changes > 1.5), intercellular communication inference (CellChat), regulatory network reconstruction (SCENIC), gene set variation analysis (GSVA), consensus non-negative matrix factorization, and construction of explainable gene ontology fingerprints.
 Results: 1. We identified 13 distinct cellular clusters with significant compositional shifts in CMML versus healthy controls; clusters 2, 5, 7, 9, and 10–13 showed the most pronounced differences. We also identified marker genes for each of the 13 cell clusters, and the top 10 marker genes in each cluster were identified based on log fold change and adjusted P-value, including ALPL, MAL, KLRF1, SASH1, SLC2A5, CRISP3, XCL2, E2F7, PAX5, ITGB3, and P2RY6. 2. A total of 1,866 genes were differentially expressed(1,584 upregulated, 282 downregulated) in CMML patients compared to healthy volunteers. 3. GO enrichment analysis was performed separately for biological process (BP), molecular function (MF), and cellular component (CC). In BP, upregulated genes were significantly enriched for neutrophil degranulation, whereas downregulated genes were overrepresented in SRP-dependent cotranslational protein targeting to the membrane. In CC, upregulated genes localized predominantly to the cytosol, while downregulated genes were concentrated in ribosomal structures. In MF, upregulated genes were chiefly associated with protein binding, whereas downregulated genes were enriched for functions related to the structural constituent of the ribosome. 4. KEGG enrichment analysis revealed significant enrichment in pathways such as B cell receptor signaling, microRNAs in cancer, ribosomes, and antigen processing and presentation. 5. Protein interaction analysis using the STRING database generated a PPI network of the top 25 differentially expressed genes, highlighting a central hub that includes NAMPT, SOD2, CXCL8, LYN, LTF, VMP1, PDE4D, FNDC3B, DEFA3, and ACSL1.
 Conclusion: This single-cell transcriptome sequencing study delineated disease-specific transcriptional and clonal evolution features in CMML versus healthy controls, providing mechanistic insight into CMML pathogenesis and identifying candidate molecular targets for therapeutic development.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99d3add7a95a8334b2ea20e007ff803fc2eda72c" target='_blank'>
              Exploring molecular heterogeneity and clonal evolution in chronic myelomonocytic leukemia via single-cell transcriptome sequencing
              </a>
            </td>
          <td>
            Jie Zhou, Yuyuan Huang, Hao Wu, Bing Li, Shaoguang Li, Aibin Liang, Jianfei Fu
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Prostate cancer, ranking among the most prevalent malignancies in males worldwide, is undergoing a significant evolution in therapeutic paradigms from conventional approaches to precision medicine, with recent advances in targeted therapies offering novel strategic insights. This review delineates the molecular foundations of prostate carcinogenesis, elucidating pivotal domains including genetic mutations, hormonal regulation, tumor microenvironment dynamics, cell cycle dysregulation, epigenetic modifications, and tumor heterogeneity. Furthermore, we evaluate the clinical translation of targeted strategies such as AR signaling axis inhibition, PI3K/AKT/mTOR pathway modulation, DNA damage repair machinery exploitation, prostate-specific membrane antigen -directed interventions, and combinatorial immunotherapy. Concurrent challenges—AR-driven heterogeneity, adaptive drug resistance mechanisms, spliceosomal vulnerabilities, and scarcity of selective molecular targets—are critically analyzed. Notwithstanding these obstacles, targeted therapies exhibit considerable potential to enhance therapeutic efficacy while mitigating systemic toxicities, paving the way for more personalized and precision-oriented oncologic care. By underscoring the imperative to decode prostate cancer’s molecular architecture, this work outlines future research priorities and advances a robust scientific framework for innovation in therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6680208f5ea18e2a4fa84ced337f25d04b9c91c6" target='_blank'>
              Unravelling targeted therapy in prostate cancer: from molecular mechanisms to translational opportunities
              </a>
            </td>
          <td>
            Litong Wu, Junfeng Qiu, Zhi-ming Hong, Quan Wang, Qixin Li, Wenbin Zhou
          </td>
          <td>2025-11-14</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 The formation of cancer and responsiveness to therapy is governed by a constantly evolving balance between somatic variation and dynamic epigenetic regulation. This evolution has been recognized as a significant obstacle challenging the development of durable cancer therapies. Thus, targeting epigenetic reprogramming in human cancers is a key focus for the design of new drugs to potentially deliver robust and durable anti-tumor activity. Lysine acetyltransferases (KATs) are a promising class of epigenetic targets that operate within interchangeable multi-subunit complexes to regulate cell cycle, pluripotency, and lineage determination. Within this class, KAT6A and KAT7 have been recognized as mechanistically intertwined modulators of stem cell identity and lineage commitment through partially overlapping control of chromatin organization specified by histone H3 acetylation of lysine 23 (H3K23) and lysine 14 (H3K14). Multiple lines of orthogonal evidence point to a “paralog” genetic relationship between these enzymes to support cancer cell autonomous survival in lineage-enriched CCLE subsets. Moreover, recent preclinical findings in pediatric and adult leukemia indicate the need to inhibit both KAT6 and KAT7 to deliver durable anti-tumor responses. These observations prompted evaluation of concomitant inhibition of KAT6 and KAT7 as a therapeutic approach in biomarker-defined solid cancers. Here, we report an equipotent KAT6/KAT7 dual inhibitor with high selectivity versus the structurally similar KAT family members KAT5/KAT8. We observed synergistic impact on cell viability and pharmacodynamic (PD) modulation of H3K23/H3K14 acetylation in biomarker positive breast, lung, and ovarian cancers with dual KAT6/7 inhibition versus selective inhibition of KAT6 or KAT7. Quantitative transcriptomic and epigenomic profiling of cancer cell responses to dual KAT6/7i revealed global inhibition of H3K23 and H3K14 acetylation as well as local suppression of H3K27ac-dependent enhancer elements at gene promoters controlling lineage specification, stemness, and WNT signaling. In vitro single cell RNA sequencing and protein expression analysis of ER+ PDX derived breast cancer cells indicated that dual KAT6/7 inhibition can overcome KAT6i associated adaptive drug resistance, downregulating drug tolerant persister cell states and metabolic adaptation. In vivo efficacy studies in biomarker positive breast and lung CDX models confirmed enhanced tumor growth inhibition and PD modulation for dual KAT6/7 versus selective KAT6 inhibition. Notably, dual KAT6/7 inhibition promoted durable and stable anti-tumor responses in multiple standard-of-care resistant and ESR1-mutant breast cancer PDXs that could not be achieved with KAT6 inhibition alone. In vivo bulk transcriptomics analysis of dual KAT6/7 treated PDX tumor samples highlighted disruption of lineage identity and metabolic rewiring. These observations indicate equipotent inhibition of KAT7 and KAT6A/B has the potential to deliver deep and durable anti-tumor activity in biomarker-selected patient populations.



 Manav Gupta, Scott R. Tyler, Nour Ghaddar, Pranav Gupta, Steven L. Spivak, Katelyn N. Lukas, Carl Schultz, Anjali Bisaria, Zabrisky Roland, Kelly S. Trego, Mason Appel, Parker Y. Jameson, Jacob Burch-Konda, Brittany Cruzan, Angelica Gonzalez-Sanchez, Monah Khalaj, Jay Prakash. Jain, Michael E . Dalziel, Julie Deichert, Sunjay Sethi, Ivan G. Shabalin, Rebeca Choy, Diana M. Munoz, Christian R. Frey, Peter Teriete, Josh Taygerly, Yuchen Bai, Richard Zang, Xin Linghu, Claire L. Neilan, Paul A. Barsanti, Michael A . White, Brian T. Jones. Dual KAT6/7 inhibition delivers robust monotherapy anti-tumor activity in biomarker selected indications and disrupts emergence of drug-tolerant persister cell populations [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C022.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7cc963d6c0afb39c048f1457ab9b594f9c49864" target='_blank'>
              Abstract C022: Dual KAT6/7 inhibition delivers robust monotherapy anti-tumor activity in biomarker selected indications and disrupts emergence of drug-tolerant persister cell populations
              </a>
            </td>
          <td>
            Manav Gupta, Scott R. Tyler, N. Ghaddar, Pranav Gupta, Steven L. Spivak, Katelyn N. Lukas, Carl Schultz, Anjali Bisaria, Zabrisky Roland, K. Trego, Mason J. Appel, Parker Y. Jameson, Jacob Burch-Konda, Brittany Cruzan, Angelica Gonzalez-Sanchez, Monah Khalaj, J. Jain, Michael E. Dalziel, Julie Deichert, Sunjay Sethi, I. Shabalin, Rebeca M. Choy, Diana M. Muñoz, Christian R. Frey, P. Teriete, J. Taygerly, Yuchen Bai, Richard Zang, Linghu Xin, Claire L. Neilan, P. Barsanti, Michael A. White, Brian T. Jones
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cancer is extensively studied, yet its origins and progression remain unclear. A key question is why tumors of the same type vary in aggressiveness and treatment response. In colorectal cancer (CRC), the third most common cancer, this variability led to the identification of molecular subtypes (CMS). However, the tumor microenvironment remains poorly understood and may be crucial for understanding carcinogenesis and drug resistance. A promising approach is analyzing cell interactions through ligand-receptor expression. This study used bioinformatics to examine CRC in different anatomical locations, identify diagnostic and prognostic biomarkers, and propose targeted drugs. Expression data were obtained from the TCGA-COAD database. All samples were filtered based on the tumor’s region of origin and purity. RNA-seq expression analysis was then conducted to assess molecular differences according to tumor location and purity, identifying region-specific ligands and receptors using the secretome list as a reference. Once these differences were identified, an interactome was constructed to depict cell interactions within the tumor microenvironment. The most relevant genes were then evaluated for their prognostic potential through survival analysis, and their susceptibility to pharmacological modulation was assessed to identify potential new drug candidates for CRC treatment. The integration of secretome data and the construction of the interactome proved to be a valuable approach for detecting novel biomarkers specific to right- and left-sided CRC. Through this approach, FGFR4, FLT1, and WNT5A were identified as key biomarkers involved in tumor carcinogenesis, modulating distinct processes in each region, such as fibroblast recruitment and cell division. Based on these biomarkers, Dovitinib and Nintedanib were predicted as potential therapeutic agents, as they target multiple identified markers. This study highlights FGFR4, FLT1, and WNT5A as key diagnostic and therapeutic biomarkers for CRC, with their relevance varying based on the tumor’s site of origin. Leveraging these findings, we propose Dovitinib and Nintedanib as promising targeted therapies for CRC. These insights can enhance current treatment strategies and pave the way for future in vivo and in vitro studies, driving progress in CRC research and therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d86a3d9f8b9f730ccd2b60a1bbaaeddf8c7f0fad" target='_blank'>
              Integrating the secretome and interactome to identify novel biomarkers and therapeutic targets in colorectal cancer
              </a>
            </td>
          <td>
            Gabriel Henrique Caxali, Mirian Carolini Esgoti Aal, Catherine Wesselka Garcia Osvaldo, Jakeline Santos Oliveira, L. Bidinotto, Robson Francisco Carvalho, F. K. Delella
          </td>
          <td>2025-10-10</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Simple Summary Optical Genome Mapping (OGM) is an innovative technology that is transforming the genetic diagnosis of acute lymphoblastic leukemia (ALL). Thanks to its high resolution and direct DNA analysis, OGM enables the detection of complex rearrangements, deletions, and gene fusions that are often missed by conventional methods such as karyotyping, FISH, and PCR. In particular, it can identify clinically relevant alterations such as IKZF1 deletions, KMT2A rearrangements, and kinase gene fusions involving ABL1, PDGFRB, JAK2, and EPOR, thereby improving molecular classification and guiding targeted therapy. The integration of OGM with next-generation sequencing (NGS) provides a comprehensive genomic overview, supporting more precise diagnosis, risk stratification, and personalized treatment in ALL. Abstract Background: Acute lymphoblastic leukemia (ALL) is a genetically heterogeneous malignancy driven by structural variants (SVs) that impact diagnosis, prognosis, and treatment. Traditional methods such as karyotyping, FISH, and PCR often fail to detect cryptic or complex rearrangements, which are critical for accurate risk stratification. Methods: Optical Genome Mapping (OGM) is a technology that directly analyzes ultra-high-molecular-weight DNA, enabling the identification of balanced and unbalanced SVs, copy number variations (CNVs), and gene fusions with high resolution. This review compares the advantages and limitations of OGM versus standard techniques in ALL. Results: OGM improves ALL diagnosis by detecting clinically relevant alterations such as IKZF1 deletions, cryptic KMT2A rearrangements, and kinase fusions, especially in cases with normal or uninformative karyotypes. It reduces artifacts by eliminating cell culture and shortens reporting times. OGM resolves complex events like intrachromosomal amplifications and chromothripsis, enhancing classification and therapy decisions. Limitations include reduced sensitivity in repetitive regions, challenges in detecting Robertsonian translocations, difficulties with complex ploidies, and lower sensitivity for low-frequency subclones. Conclusions: Integrating OGM with next-generation sequencing (NGS) allows comprehensive genomic profiling, improving diagnosis, prognosis, and personalized treatment in ALL. Future advancements promise to further enhance the clinical utility of OGM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df8b9963a1fbf7c91e5fb4aba30093e1076d7ee6" target='_blank'>
              The Silent Revolution of the Genome: The Role of Optical Genome Mapping in Acute Lymphoblastic Leukemia
              </a>
            </td>
          <td>
            Claudia Simio, Matteo Molica, Laura De Fazio, Marco Rossi
          </td>
          <td>2025-10-27</td>
          <td>Cancers</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Background: Acute myeloid leukemia (AML) is a complex disease marked by a multitude of genetic mutations and dysregulated gene expression profiles stemming from genetic and epigenetic alterations. The disease is marked by significant biological heterogeneity, which complicates effective treatment and contributes to the emergence of drug resistance. The molecular profiles of gene mutations, chromosomal anomalies within AML considerably impacts disease progression, therapeutic response, and survival prognosis. Several predictive and prognostic models are used but present inaccuracy at individual level. Integrating multiple layers of biological data, including transcriptomic analyses (gene expression, mutational profiling, gene fusion identification) and population-level characterization through deconvolution and multiparametric flow cytometry, offers a powerful approach to better understand the complexity of AML and improve prognostic prediction and guide treatment strategies.
 Methods: We used a multi-omics data integration approach, including bulk RNAseq-based gene expression (n=45), single nucleotide variant and gene fusion data from bulk RNA sequencing (n=45), deconvolution of tumor microenvironment immune subtypes (n=45), immune subtype identification and blast characterization using multiparametric flow cytometry (MFC) (n=36), in vitro drug response (Venetoclax, n=23; 5-Azacytidine, n=23; Midostaurine, n=21, Gilteritinib, n=22), and relevant clinical metadata. This allowed us to thoroughly characterize tumor and its tumor microenvironment.
 Results: Applying Multiomics Factor Analysis (MOFA) identified 9 principal factors capturing the diversity of the disease. Factor 1 was significantly associated with Venetoclax response (p<0.05). Factors 2 and 4 showed significant associations with NPM1 mutations and DNMT3A mutations, while Factors 6 and 9 were significantly associated with 5-Azacytidine response (p<0.05). Factor 9 was also significantly associated with KIT mutations (p<0.05). The correlation analysis revealed strong associations between factors and key clinical covariates including timepoint and drug response groups.
 Next, we performed unsupervised clustering using the latent MOFA factors and identified four distinct clusters. Cluster 1 (C1, n=14) predominantly consisted of diagnostic samples (79%) with balanced immune infiltration. Cluster 2 (C2, n=13) showed moderate monocyte enrichment. Cluster 3 (C3, n=10) was characterized by significantly elevated regulatory T cell infiltration (p=0.017 vs C4, p=0.023 vs C1) and higher myeloid dendritic cell populations (p = 0.023 vs C2, p = 0.0051 vs C4), confirmed by both deconvolution (p=0.023 vs C2, p=0.0051 vs C4) and multiparametric flow cytometry analysis, consistent with an immunosuppressive microenvironment in this relapse-enriched cluster (60% relapsed samples). Cluster 4 was significantly enriched in monocytes (p < 0.001 vs C1 and C3).
 Finally, therapeutic response profiling across molecular clusters revealed distinct patterns of drug sensitivity. C1 demonstrated favorable responses to Venetoclax, Midostaurine, and Gilteritinib. C2 exhibited poor responses to all four tested drugs, indicating a globally resistant phenotype. C3 showed resistance specifically to Midostaurine and Gilteritinib, while remaining potentially responsive to other agents. Interestingly, C4 was characterized by poor sensitivity to Venetoclax and 5-Azacytidine but responded well to Midostaurine and Gilteritinib, highlighting subtype-specific vulnerabilities that could inform personalized therapeutic strategies. Interestingly, Gilteritinib demonstrated significant activity in specific molecular subgroups (C1 and C4) independently of FLT3-ITD status, suggesting broader therapeutic potential beyond canonical FLT3-driven AML. This unexpected sensitivity, possibly linked to off-target effects such as AXL inhibition, highlights novel vulnerabilities revealed by our integrative multi-omics framework.
 Conclusion: Our comprehensive multi-omics MOFA framework identified four clinically distinct AML subtypes with distinct therapeutic vulnerabilities, immune microenvironment characteristics, mutational profiles, and blast immunophenotypes. Our systematic approach demonstrates the potential of multi-omics integration for advancing personalized medicine in AML and provides a framework that will benefit from validation with larger cohorts to improve risk stratification personalized treatment approaches.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c9ac3a924304ce3dd2cf299a9f56500c5d7d76a" target='_blank'>
              Multi-omics integration reveals therapeutic vulnerabilities and immune microenvironment subtypes in Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            E. Alaterre, H. de Boussac, A. Kassambara, A. Steer, Amélie Machura, Julien Aityaya, Ouissem Karmous Gadacha, M. Abouladzé, C. Bret, Gaspar Aspas Requena, Ludovic Gabellier, C. Herbaux, A. Bruyer, Jérôme Moreaux
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="The search for novel tumor-specific markers and therapeutic targets is driving the development of more effective and personalized treatment strategies for cancer patients. This article focuses on investigating a promising new source of biomarkers—readthrough transcripts, or downstream-of-gene (DoG) transcripts. These transcripts are extended products of gene transcription that continue into intergenic regions and can overlap neighboring genes, sometimes giving rise to cis-splicing of adjacent gene (cis-SAGe) transcripts. Recent studies suggest that besides frequently being a “transcriptional noise”, DoG transcripts can perform regulatory functions, serve as a source of novel protein products, and act as prognostic markers of patient survival across various cancers. This article aims to investigate the regulatory mechanisms and functional significance of readthrough transcripts in tumors, to identify currently known tumor-specific variants with potential utility as cancer biomarkers or therapeutic targets, and to evaluate the most suitable approaches for their detection. The knowledge gained through this research may provide a foundation for the development of diagnostic test systems and the design of new anticancer drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/005f6133a4fcac56d4aa30b3f8401ba10e194f84" target='_blank'>
              Regulatory Mechanisms and Functional Roles of Readthrough Transcripts in Tumorigenesis
              </a>
            </td>
          <td>
            Aleksander Modestov, G. Zakharova, E. Poddubskaya, Anton Buzdin
          </td>
          <td>2025-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Homologous recombination deficiency (HRD) activates pro-inflammatory cGAS/STING signaling, positioning it as a biomarker for combining immune checkpoint blockade (ICB) and PARP inhibition (PARPi). However, the consequences of HRD on the immune landscape across cancers remain unclear. Here, we applied a pan-cancer HRD classifier to >10,000 tumors from The Cancer Genome Atlas and uncovered striking heterogeneity in immune activity. Compared to HR-proficient tumors, HRD tumors showed elevated inflammation in breast, ovarian, and endometrial cancers. These tumors exhibited robust activation of innate and adaptive immune pathways (IFN, NF-κB) and transcriptional hallmarks of senescence, angiogenesis, and adenosine signaling. In contrast, lung, head and neck, and melanoma HRD tumors displayed suppressed inflammation and evidence of immune escape through large-scale loss-of-heterozygosity (LOH) at IFNA/B, STING, and other loci. These tumors also frequently presented HLA LOH and oncogene amplifications, suggesting selection under immune pressure and replication stress. Together, our study resolves HRD tumors into two immune archetypes, immune-inflamed and immune-evasive, linked to chromosomal instability and lineage, informing biomarker-driven evaluation of immune checkpoint blockade/PARPi combinatorial therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b68316c5391afa0d45aa42e226e37099bf84d45" target='_blank'>
              Tissue specificity and chromosomal alterations shape divergent immune programs in HRD tumors
              </a>
            </td>
          <td>
            D. Gulhan, David Barras, Marco Mina, Eleonora Ghisoni, Yoo-Na Kim, V. Viswanadham, Hu Jin, Florian Huber, K. Homicsko, M. Bassani-Sternberg, G. Ciriello, Peter J. Park, Denarda Dangaj Laniti
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Therapy resistance in acute myeloid leukemia (AML) remains a major clinical obstacle, particularly due to the persistence of leukemia stem cells (LSCs) capable of metabolic adaptation. While venetoclax (Ven) inhibits oxidative phosphorylation (OXPHOS), we found that Ven-resistant LSCs undergo glycolytic reprogramming to bypass OXPHOS inhibition. This metabolic shift is supported by enhanced ribosome biogenesis, sustained by upregulated de novo guanine nucleotide biosynthesis. Abundant guanine nucleotides suppress the impaired ribosome biogenesis checkpoint (IRBC), leading to TP53 destabilization and persistent MYC expression. Inhibition of inosine monophosphate dehydrogenases (IMPDH1/2) depletes guanine nucleotides, activates IRBC, stabilizes TP53, represses MYC, and impairs the metabolic shift to glycolysis. This metabolic rewiring disrupts LSC stemness and suppresses the reconstitution of human AML cells in xenotransplantation experiments. Notably, the suppression of LSC stemness was observed regardless of Ven resistance or the TP53 mutational status of AML cells. These findings reveal that mutation-independent TP53 inactivation is involved in resistant AML and suggest that targeting guanine nucleotide biosynthesis may offer a clinically actionable strategy to eradicate therapy-resistant LSCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6613ff1ea81832476d4f80a2812043adb8207057" target='_blank'>
              Guanine nucleotides drive ribosome biogenesis and glycolytic reprogramming in acute myeloid leukemia stem cells.
              </a>
            </td>
          <td>
            Gentaro Kawano, Riichiro Ikeda, Daisuke Ishihara, T. Shima, Teppei Sakoda, Shunsuke Yamamoto, Yuichi Kochi, Yuichiro Semba, Sanae Ashitani, Y. Mori, Koji Kato, Takahiro Maeda, Toshihiro Miyamoto, Tomoyoshi Soga, Koichi Akashi, Y. Kikushige
          </td>
          <td>2025-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Next-generation sequencing of canine spontaneous cancer is a powerful approach in both comparative oncology and veterinary genomics. We encountered a unique case with concurrent mammary tumors. Using whole-genome sequencing (WGS), we profiled the tumor-specific landscape of somatic mutations across multiple tumor subtypes, providing unprecedented evidence within an identical genetic background. Of the seven mammary gland tumors (MGTs) isolated, two were diagnosed as benign and five as malignant. High-quality WGS (34.5X average sequencing depth, ≥ 20X coverage across 94.9% of the genome) on tumors and a blood sample revealed missense mutations in human breast cancer-related genes such as BRCA2 and TP53. Furthermore, we found evidence of canine-specific somatic mutations depending on the tumor subtypes, including HECTD4 in malignant tumors and NIPBL in epithelial-derived malignant tumors. This unique case study provides novel insights into the genomic heterogeneity, clonal evolution, and subtype-specific pathogenesis of naturally occurring canine MGTs. Despite some inherent limitations and potential for individual-specific variation, our results emphasize and guide future large-scale, longitudinal studies to further elucidate the clinical and biological significance of key somatic alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa805022d59e7e8ebaa159b214972178e6ca2e8d" target='_blank'>
              Comparative genomic analysis of multiple mammary tumors from a single dog: whole-genome sequencing study
              </a>
            </td>
          <td>
            Keon Kim, Tae-Hoon Shin, Sin-Wook Park, Sang-Ik Park, Yoon-Jung Do, Woong-Bin Ro, Chang-Min Lee
          </td>
          <td>2025-10-30</td>
          <td>Irish Veterinary Journal</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Breast cancer is the leading cause of death among women globally. Several genes have been found to be transcriptionally dysregulated in cancer, according to recent studies. TATA-box binding protein (TBP) and its two paralogs, TBPL1 and TBPL2, play roles in human transcription. The TBPL1 gene is implicated in colorectal carcinomas by suppressing the expression of miR-18a. However, its function in breast cancer remains undisclosed. TBPL1 is distantly related to TBP and possesses a 40% similarity with TBP’s core domain. In the present study, we explored the potential role of the TBPL1 gene in transcriptome regulation by knocking out the TBPL1 gene through the CRISPR/Cas9 method. Following the knockout of the TBPL1 gene, we examined the gene transcription patterns and compared them to wild-type cell lines. We observed disparate signatures of upregulated and downregulated genes in wild-type and mutated conditions. Healthy breast MCF-12F, and T47D, SKBR3, and MDA-MB-231 breast cancer cell lines were assessed, as these cancer cells exhibit overexpression of the TBPL1 gene. Next-generation sequencing data revealed distinct marker genes regulated by the TBPL1 gene and their potential involvement in cell migration, proliferation, anti-apoptosis, and metastasis. Additionally, we also discovered novel lncRNAs implicated in the transcriptome analysis of the TBPL1 knocked-out gene. Our investigation indicated that this gene might affect varied stages of breast cancer cell lines’ cellular properties, such as cell duplication, morphology, and growth. It might also contribute to tumor formation in more aggressive cell lines like MDA-MB-231 in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f903eef7750f7a042028b3e6a2d14c2b170de62d" target='_blank'>
              RNA-seq analysis of wild-type and mutated TBPL1 gene in breast cancer cells lines through CRISPR/Cas9 approach reveals novel molecular signatures
              </a>
            </td>
          <td>
            R. Mishal, Jorge Meléndez-Zajgla, Bertha Rueda-Zarazúa, María Luisa Labra-Barrios, Carlos A Castañón-Sánchez, Salvador Uribe Carvajal, Laura Padierna-Mota, José Manuel Hernández-Hernández, Gloria Leon-Avila, Armando Pérez Rangel, Edgar Hernández-Martínez, Erika Angeles-Morales, Ibrahim Khalil Albalawi, J. Luna-Arias
          </td>
          <td>2025-10-28</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Lung cancer remains one of the leading causes of cancer-related mortality worldwide, with tumor recurrence and metastasis posing significant challenges despite advances in targeted therapies and immunotherapy. Cellular dormancy, a reversible, quiescent state marked by cell cycle arrest, has emerged as a key driver of therapeutic resistance and disease relapse, particularly in small-cell lung cancer (SCLC) and non-small cell lung cancer (NSCLC). Multiple mechanisms, including autophagy, stress-adaptive signaling, microenvironmental cues, and epigenetic dysregulation, have been implicated in the regulation of dormancy and long-term cell survival. Among these, epigenetic modifications such as DNA methylation, histone modifications, and non-coding RNAs (ncRNAs) play pivotal roles in maintaining dormancy by repressing proliferative gene expression programs. Increasing evidence suggests that dormant tumor cells harbor distinct epigenomic signatures, which may serve as predictive biomarkers for minimal residual disease (MRD) and relapse risk. This review summarizes current advances in understanding the epigenetic regulation of cellular dormancy in lung cancer, with a particular emphasis on the interplay between epigenetic modifiers and oncogenic signaling pathways. Furthermore, emerging molecular targets and associated therapeutic agents currently under clinical evaluation are presented, emphasizing how a deeper understanding of the epigenetic landscape governing dormancy may inform the development of novel interventions to improve long-term clinical outcomes in lung cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f78e8931704f8d1c6b0d604cdcde3728bfe1b4a" target='_blank'>
              Unlocking Lung Cancer Cell Dormancy: An Epigenetic Perspective
              </a>
            </td>
          <td>
            F. Fabrizio
          </td>
          <td>2025-11-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Thyroid carcinomas are phenotypically heterogeneous malignancies. Advances in molecular and cellular technologies have revealed genetic, epigenetic, and nongenetic factors underlying this heterogeneity. Our study aimed to assess the impact of single and combined treatments with anticancer agents (Carboplatin, Doxorubicin, Paclitaxel, Avastin), natural compounds (Quercetin), and epigenetic modulators (suberoylanilide hydroxamic acid and 5-Azacytidine) on the expression of long noncoding RNAs, methylation regulators, and functional features in the human thyroid cancer cell line K1. Methods: Treated and untreated K1 cells were used throughout experiments to evaluate the drug-induced cytotoxicity, apoptosis, cell cycle distribution, cytokine release, gene expression, and global DNA methylation levels. Results: Some single- and combined-drug treatments modulated both cell cycle progression and apoptotic events, demonstrating anti-tumor activity of the tested compounds. Gene expression analysis showed treatment-specific regulation of target genes and lncRNAs, including both upregulation and downregulation across different drug combinations. All treatments resulted in increased global DNA methylation levels compared to the untreated controls. Several combinations significantly upregulated DNMT1 and DNMT3B, while concomitantly decreased EZH2 levels. Conclusions: These coordinated epigenetic changes highlight the therapeutic potential of combining epigenetic modulators with chemotherapeutic agents, suggesting a strategy to prevent or reverse treatment resistance and improve outcomes in thyroid cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85aa1eba8f910f8930ce4647663ca6adba36af18" target='_blank'>
              Epigenetic Remodeling in Thyroid Cancer: New Dimensions of Targeted Therapy Through lncRNA Modulation
              </a>
            </td>
          <td>
            A. Albulescu, A. Fudulu, M. Mihăilă, I. Iancu, Adriana Pleșa, M. Bostan, A. Botezatu, L. Brasoveanu, C. Hotnog
          </td>
          <td>2025-10-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The biological heterogeneity between left- and right-sided colorectal cancer (CRC) poses a significant clinical challenge and the underlying regulatory mechanisms remain elusive. As an emerging hallmark of cancer, the contribution of circadian rhythm disruption (CRD) to this side-specific heterogeneity is unclear. By integrating single-cell and spatial transcriptomic analyses, this study shows that tumor cells in right-sided CRC exhibit significantly higher CRD scores and identifies the emergence of a NONO-positive tumor-cell subpopulation (NONO⁺ TC) as a key molecular feature of this phenotype. Spatial analysis further confirms that this NONO⁺ TC forms a tightly co-localized microenvironment with fibroblasts in situ. Notably, cell–cell communication analyses indicate that NONO expression does not augment the signal-sending capacity of tumor cells but instead reprograms them into highly efficient signal receivers. These augmented incoming signals predominantly originate from the more pro-tumorigenic myofibroblastic cancer-associated fibroblast (myCAF) subtype, indicating that NONO⁺ TC are particularly sensitive to malignant stromal inputs. In conclusion, our study delineates the dual role of NONO as both a circadian regulator and a pro-tumorigenic signaling hub. By enhancing tumor-cell reception of CAF-derived signals, NONO links CRD to the formation of NONO⁺ TC niches and a malignant microenvironment in right-sided CRC, providing new mechanistic insight into the spatial heterogeneity of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/241b9e1e30a34229f91641257f9f4e9436ac45df" target='_blank'>
              NONO links circadian rhythm disruption and enhanced tumor-fibroblast crosstalk in right-sided colorectal cancer
              </a>
            </td>
          <td>
            Zhi-Hao Shang, Qin-Chang Zhang, Song-yang Xi, Kai Chen, Shao-Bo Guo, Zhou Zhou, Xin-Zhuo Zhan, Yun-Xia Wu, Xin-Yi Li, Hai-Bo Cheng, Xue-Jun Song, Gui-Hua Tian
          </td>
          <td>2025-10-31</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Glioblastoma remains one of the most treatment-refractory solid tumors due to rapid adaptive resistance across multiple convergent molecular pathways. In particular, persistent activation of the PI3K-AKT axis combined with mesenchymal transition (EMT) reprogramming contributes to rapid failure of standard therapeutic regimens, including temozolomide, radiation, and immunotherapeutics. To address this clinical gap, we developed a synthetic RNA-guided dual-targeting system designed to simultaneously suppress both canonical PI3K-AKT activation and mesenchymal escape signatures, using strand-engineered molecules optimized for combinatorial silencing. Using patient-derived glioblastoma stem cell lines and isogenic resistance models, we performed transcriptomic profiling to identify co-expressed resistance-driving nodes. Computational integration of CRISPR perturbation datasets, RNA interference screens, and synthetic lethality maps enabled selection of targetable PI3K-AKT intermediates and EMT regulators including SOX2, TWIST1, and ZEB1. Synthetic RNA molecules were designed to co-suppress both vertical (PI3K-AKT) and horizontal (EMT plasticity) resistance axes, with in vitro validation demonstrating >70% knockdown efficiency and significant synergistic reduction in proliferation (p<0.001) across diverse resistance phenotypes. Mechanistic assays revealed reversal of mesenchymal signatures with restored epithelial markers (E-cadherin upregulation; vimentin downregulation) alongside AKT dephosphorylation. Combination treatment with low-dose temozolomide further amplified cell death, suggesting sensitization to standard chemotherapy. In vivo murine xenograft models exhibited >60% reduction in tumor volume compared to controls (p<0.01), without evidence of off-target cytotoxicity or systemic toxicity. This study demonstrates the feasibility of multi-axis synthetic RNA-guided combinatorial targeting to suppress glioblastoma therapeutic resistance. These findings support future clinical translation of modular RNA-based therapeutics capable of overcoming the heterogeneity and plasticity that characterize high-grade gliomas.



 Shivi Kumar. Synthetic RNA-guided combinatorial targeting of glioblastoma adaptive resistance: Dual suppression of PI3K-AKT and mesenchymal transition escape pathways [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr A036.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/746db514f096cab5d3002088013773d14b8c8b84" target='_blank'>
              Abstract A036: Synthetic RNA-guided combinatorial targeting of glioblastoma adaptive resistance: Dual suppression of PI3K-AKT and mesenchymal transition escape pathways
              </a>
            </td>
          <td>
            Shivi Kumar
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Rare tumor cell populations can evade cancer therapies, ultimately driving drug resistance and relapse. These resilient populations, termed “persisters,” display distinct molecular states compared to the bulk tumor. While some persisters acquire genetic mutations that confer growth advantage, others undergo non-genetic remodeling that enhances cellular plasticity and fitness. Previous studies suggest that stochastic gene activation promotes persister survival (Shaffer et al., Nature 2017; Goyal et al., Nature 2023), yet the master regulators of this plasticity remain undefined. Using DNA barcoding, we demonstrate that pre-determined, non-stochastic persisters exist in KRAS-mutant non-small cell lung cancer (NSCLC) treated with the KRAS G12C inhibitor sotorasib. These clones repopulate the tumor without harboring recurrent genetic alterations. By isolating single clones destined to persist or go extinct, and profiling their transcriptomes and chromatin accessibility, we found that persister clones activate an interferon-γ (IFNγ) transcriptional signature accompanied by increased chromatin accessibility at sites enriched for the transcription factors IRF1 and ZNF384. Functionally, ectopic ZNF384 expression converted the entire cell population into a resistant state, while IRF1 knockout prevented persister formation. Cleavage Under Targets & Release Using Nuclease (CUT&RUN) assays of ZNF384 and IRF1 revealed their binding to promoter and enhancer regions, including loci regulating IFNγ-responsive genes, thereby contributing to persister plasticity and survival. Together, our study identifies IRF1 and ZNF384 as tumor-intrinsic regulators linking IFNγ signaling to transcriptional plasticity, providing mechanistic insight into the evolution of drug tolerance and resistance in KRAS-mutant NSCLC.



 Chendi Li, Anahita Nimbalkar, Christopher J. Graser, Melissa Vieira, Mohammed U. Syed, Qian Qin, Barbara Karakyriakou, Sarah Clark, Anne Y. Saiki, Paul E. Hughes, Christopher Ott, Luca Pinello, Franziska Michor, Aaron N. Hata. Transcription factors governing the evolution of drug-tolerant persisters in KRAS-mutant non-small cell lung cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B014.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/331fb37a264b2ec60d8af95499de6058539d7fa9" target='_blank'>
              Abstract B014: Transcription factors governing the evolution of drug-tolerant persisters in KRAS-mutant non-small cell lung cancer
              </a>
            </td>
          <td>
            Chendi Li, Anahita Nimbalkar, Christopher J Graser, Melissa Vieira, M. U. Syed, Qian Qin, Barbara Karakyriakou, Sarah Clark, A. Y. Saiki, Paul E Hughes, Christopher Ott, Luca Pinello, Franziska Michor, Aaron N. Hata
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="


 EGFR-mutant non-small cell lung cancers (NSCLCs) initially respond to EGFR tyrosine kinase inhibitors (TKIs) such as osimertinib, but tumors eventually develop resistance. While genetic mutations explain many instances of resistance, an increasing body of evidence points to transcriptional and cell state changes as another route by which tumor cells evade therapy. The dynamics that shape these adaptive responses and the non-genetic mechanisms involved in facilitating them remain incompletely understood. We investigated transcriptional programs activated in EGFR-mutant LUAD cells following TKI treatment and observed robust induction of SOX2, a transcription factor involved in promoting stemness and lineage plasticity, within a day of drug exposure. In parallel, we investigated the role of APOBEC cytidine deaminases (APOBEC3A and APOBEC3B), which have been linked to mutagenic processes in cancer. While both enzymes have been implicated in drug resistance, the individual contributions of the two enzymes remain ill studied. Using competition assays with single gene knockouts of either APOBEC3A or APOBEC3B, we observed that loss of APOBEC3A leads to a more profound reduction in the survival of drug-tolerant cells following TKI therapy. These results highlight a functional role for APOBEC activity in sustaining early drug-tolerant populations under therapeutic pressure. Over the course of long-term osimertinib treatment, we also identified distinct subpopulations of cells with elevated ΔNp63, an isoform of TP63 that serves as a marker of squamous cell trans-differentiation. These ΔNp63-high cells emerge during the acquisition of drug resistance, and evidence suggests that this subpopulation may rely on CDK4, a cell cycle-regulating kinase, to sustain growth under therapeutic pressure. Pharmacological inhibition of CDK4 with palbociclib reduced the survival of ΔNp63-high cells, highlighting a potential therapeutic vulnerability in this proliferative subpopulation. Together, our findings demonstrate that adaptive resistance in EGFR-mutant NSCLC is driven by several non-genetic resistance mechanisms, including SOX2 induction, ΔNp63-driven squamous cell transdifferentiation, and APOBEC-mediated promotion of drug-tolerant persister cells.



 Sharan Srinivasan, Jessica N. Becerra, Nina M. Garcia, James V. Alvarez. SOX2 induction and ΔNp63-high subpopulations mark adaptive responses to EGFR inhibition in lung adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B019.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a218a816c806ca36ea70bf5b2dadd14af3bb49f" target='_blank'>
              Abstract B019: SOX2 induction and ΔNp63-high subpopulations mark adaptive responses to EGFR inhibition in lung adenocarcinoma
              </a>
            </td>
          <td>
            Sharan Srinivasan, Jessica N Becerra, N. M. Garcia, 
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/182327417f0bfd0539120b0fcc3161f6ade537ce" target='_blank'>
              Dissecting neuroblastoma heterogeneity through single-cell multi-omics: insights into development, immunity, and therapeutic resistance.
              </a>
            </td>
          <td>
            , Si-Jia He, Xiao-yu Jing, , Xia Guo, Ju Gao, Wei Zhang
          </td>
          <td>2025-11-27</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Cutaneous malignant melanoma is a common cancer in adults but extremely rare in young children, affecting fewer than one child per million each year in Europe. Because of its rarity, most treatments for children are adapted from adult therapies, despite possible biological differences. This study aimed to explore the molecular features of a rare and aggressive melanoma in a 16-month-old patient to understand disease progression and treatment resistance. We studied the tumour and metastases of a patient with a melanoma carrying an NRAS mutation, who received chemotherapy and immune checkpoint inhibitor treatment. The patient died 10 months after diagnosis. We used DNA methylation analysis, single-nucleus RNA sequencing, and deep spatial transcriptomic profiling to examine genetic changes, gene activity, and their spatial distribution in both the primary tumour and lymph node metastases. Here, we show that the tumour displayed high genetic and transcriptomic diversity. We identified increases in MITF and BRAF gene copies as likely key drivers of the aggressive disease, which were not detected at diagnosis. We also found activation of biological pathways, including VEGFA and WNT signalling, and abnormal activity of several genes linked to immune therapy response, with marked variation between tumour regions. This case demonstrates that paediatric melanoma can harbour complex and spatially variable molecular changes that contribute to rapid disease progression and treatment failure. Our findings support incorporating detailed spatial transcriptional profiling into clinical assessment to better guide therapy in rare paediatric cancers. Melanoma is a type of skin cancer that is common in adults but extremely rare in very young children. Because it is so uncommon, children are usually treated with approaches designed for adults, which may not work as well. In this case study, we investigated an aggressive case of melanoma in a 16-month-old child to understand why the cancer progressed quickly and did not respond to treatment. We examined samples from the original tumour and its spread to the lymph nodes using advanced techniques to map genetic changes and patterns of gene activity. We found specific changes in key cancer-related genes and signals that likely drove the disease and made treatment ineffective. These results highlight the importance of detailed tumour analysis to guide better treatment strategies for rare childhood cancers. Mucha et al. analyse tumour and metastases samples from a rare and aggressive melanoma in a 16-month-old child using advanced genetic and spatial profiling. They identify key gene changes and signaling pathways that likely drive rapid disease progression and treatment resistance, highlighting the need for detailed tumour mapping in paediatric melanoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e22b875f5118b342e202377324b1e876661dddb" target='_blank'>
              Multi-modal omics analysis of a paediatric melanoma highlights mechanisms underlying treatment resistance
              </a>
            </td>
          <td>
            M. Mucha, Sebastian Bühner, Maurice Loßner, V. Fincke, N. Reitsam, Felix Dorn, Dajana Lobbes, Katharina Gastberger, Tobias Schuster, Sebastian Dintner, Christopher Schroeder, Ines B. Brecht, D. Schneider, B. Märkl, Michael C. Frühwald, Pascal D. Johann
          </td>
          <td>2025-10-30</td>
          <td>Communications Medicine</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Cancer is a multifaceted disease characterized by uncontrolled cell division resulting from substantial disruptions of normal biological processes. Central to its development is cellular transformation, which involves a dynamic sequence of events including chromosomal translocations, genetic mutations, abnormal DNA methylation, post-translational protein modifications, and other genetic and epigenetic alterations. These changes compromise physiological regulatory mechanisms and contribute to accelerated tumor growth. A critical factor in this process is the dysregulation of transcription factors (TFs) which regulate gene expression and DNA transcription. Dysregulation of TFs initiates a cascade of biochemical events, such as abnormal DNA replication, that further enhance cell proliferation and increase genomic instability. This microenvironment not only sustains tumor growth but also promotes the accumulation of somatic mutations, thereby fueling tumor evolution and heterogeneity. In this study, we employed an in silico approach to identify TFs regulating 622 key genes whose mutations are implicated in carcinogenesis. Transcriptional regulatory networks were analyzed through bioinformatics methods to elucidate molecular pathways involved in cancer development. A thorough understanding of these processes may help to clarify the function of dysregulated TFs and facilitate the development of novel therapeutic approaches designed to make cancer treatments personalized and efficacious.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/020eeb95aab1edda2cdedffd56c0eb8b8019a670" target='_blank'>
              From Transcription Factors Dysregulation to Malignancy: In Silico Reconstruction of Cancer’s Foundational Drivers—The Eternity Triangle
              </a>
            </td>
          <td>
            A. Cammarota, A. Carrizzo, M. De Marco, N. Bukvic, Francesco Jacopo Romano, Alessandra Rosati, M. Chetta
          </td>
          <td>2025-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Background Extracellular leucine rich repeat and fibronectin type III domain containing 1 (ELFN1), a transmembrane protein implicated in tumorigenesis and therapy resistance, remains mechanistically undefined as a pan-cancer target. In this study, we aimed to elucidate the function and potential mechanism of action of ELFN1 across cancers. Methods Through integrative analysis of TCGA and GTEx datasets, we systematically characterized ELFN1 across 33 cancer types, including its expression patterns, prognostic value, mutation landscape, methylation modifications, protein-protein interaction (PPI) networks, and the relationship between ELFN1 expression and immune infiltration. KEGG enrichment analysis was also performed to predict the functions and associated cellular pathways of ELFN1. In addition, the molecular docking tool was used to analyze the affinities between ELFN1 protein and drugs. Finally, we assessed the effect of ELFN1 knockdown on colorectal cancer (CRC) cells using in vitro experiments. Results Our study revealed significant dysregulation of ELFN1 across various cancer types, with notable diagnostic and prognostic utility in most cancers analyzed. Mechanistically, ELFN1 expression was associated with DNA methylation, DNA repair, genomic instability, and tumor microenvironment (TME) scores in multiple cancer types. Furthermore, Drug sensitivity profiling linked ELFN1 to ABT-737 susceptibility and benzaldehyde resistance through molecular docking. In CRC cells, ELFN1 knockdown significantly inhibited tumor proliferation, migration, and motility. Conclusion The expression level of ELFN1 may provide insights into tumor development and progression in multiple cancers, including CRC, highlighting its potential utility as an effective prognostic biomarkers and immunotherapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89c3fe475dcaffa41259e73c95a1f5f7e78a3e04" target='_blank'>
              Pan-cancer analysis reveals ELFN1 as a novel prognostic biomarker and immunotherapeutic target associated with tumor microenvironment remodeling and promoting malignant phenotypes in colorectal cancer
              </a>
            </td>
          <td>
            Sha-Sha Hu, Tian-Yuan Tan, Wei Yan, Yu Wu, Xin-Nian Li, Fu-Jin Liu, Bo Wang
          </td>
          <td>2025-11-20</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Endometrial cancer represents one of the most common gynecologic malignancies worldwide, with molecular profiling emerging as a transformative approach to its classification and treatment. The traditional histopathological model, based on tumor grade and morphology has evolved with the advent of genomic technologies, leading to the development of The Cancer Genome Atlas (TCGA) molecular subtypes: POLE ultra mutated, microsatellite instability-high (MSI-H) or mismatch repair-deficient (DMMR), copy-number low, and copy-number high. POLE-mutated tumors, characterized by ultrahigh mutational burden and microsatellite stability, generally have excellent prognosis, whereas MSI-H/ DMMR tumors respond favorably to immune checkpoint inhibitors. This review article was intended to underlie the importance of personal treatment of every patient because they exhibit unique molecular characteristics, leading to variations in prognosis and therapeutic response. Therefore, tailoring management to these molecular profiles is essential to achieve optimal outcomes. Integrating immunohistochemistry, PCR, and next-generation sequencing facilitates accurate molecular classification and individualized therapy. The convergence of molecular pathology and genomics represents a paradigm shift in endometrial cancer care, positioning POLE and MMR testing as cornerstone biomarkers for truly personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/916625046da400792441b823ecd6e0f73c749886" target='_blank'>
              Integrating POLE Mutations and Mismatch Repair Status in Personalized Medicine for Endometrial Cancer
              </a>
            </td>
          <td>
            Erin Nabila Putri, Willy Sandhika
          </td>
          <td>2025-10-31</td>
          <td>Magna Scientia Advanced Research and Reviews</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Dose–response curves (DRCs) are a standard protocol in experimental biology to assess the effect of anticancer agents in vitro. Yet, therapy itself can reshape tumor composition, driving the gradual evolution of the DRC. Monitoring such changes can reveal the emergence of resistance, as well as evolutionary trade-offs that may be exploited to enhance therapeutic efficacy. Here, we investigate the evolution of DRCs in response to three different ALK inhibitors (ALKi) in ALK-positive anaplastic large cell lymphoma (ALCL) using patient-derived cell lines. Over the course of six months, ALKi concentrations were gradually escalated, allowing ALCL cells to adapt and survive at higher drug levels. Strikingly, at the same time, these resistant populations exhibited reduced fitness in the absence of the drug, indicating an evolutionary cost. This phenomenon, known as “drug addiction” or dependence, has been previously observed in melanoma following resistance to MAPK inhibitors and suggests that treatment holidays could optimize tumor control in the long term. Guided by these findings, we developed a mathematical model to capture the temporal evolution of DRCs and to test how therapy design might exploit addiction in ALCL. Calibrated with experimental data, the model simulated tumor growth under alternative dosing regimens. Simulations revealed that continuous therapy rapidly selects for resistance, leading to tumor regrowth within ∼150 days. In contrast, intermittent dosing schedules maintained long-term control for more than 300 days. Moreover, the model identified an adaptive strategy: switching from continuous to intermittent dosing precisely when tumor burden reaches its minimum. This evolutionarily informed approach prevents resistant clones from dominating while leveraging the fitness disadvantage associated with drug addiction. Preliminary experiments with naïve and resistant ALCL cell lines, cultured under intermittent dosing in both monoculture and co-culture systems, support the model’s predictions. These experiments also enable the refinement of the model's parameters to incorporate tumor heterogeneity. While additional validation is required, our findings demonstrate that integrating mathematical modeling with experimental DRCs provides a quantitative framework for predicting the evolutionary dynamics of resistance and addiction. Collectively, this study highlights the potential of adaptive and evolution-based therapy in ALCL. Optimized treatment schedules informed by evolutionary trade-offs can reduce tumor burden, delay progression, and exploit vulnerabilities generated by resistance mechanisms.



 Franco Pradelli, Lucy Hare, Jamie Matthews, Roxane Chen, Aiindrila Dhara, G. A. Amos Burke, Suzanne D. Turner, Jeffrey West. Capturing dose-response adaptation to identify evolutionarily-informed treatment strategies [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B037.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86b1f1eb717bb47b50718cff89eef53948102c75" target='_blank'>
              Abstract B037: Capturing dose-response adaptation to identify evolutionarily-informed treatment strategies
              </a>
            </td>
          <td>
            Franco Pradelli, Lucy Hare, J. Matthews, Roxane Chen, Aiindrila Dhara, G. A. A. Burke, Suzanne D. Turner, J. West
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a38b8fdfa925b911ba77595f93672078904315e1" target='_blank'>
              The role of Alu elements in causing BRCA1 structural variation and breast cancer susceptibility.
              </a>
            </td>
          <td>
            Minjae Yu, Dahee Jo, Wonseok Shin, Kyudong Han
          </td>
          <td>2025-11-17</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 Erythropoiesis is a complex, tightly regulated process involving sequential changes in epigenetics, metabolism, and proliferation. While conventional immunophenotyping combined with transcriptomic and epigenomic profiling has provided valuable insights, these approaches lack the resolution to fully capture dynamic transitions. A comprehensive understanding of how these programs are coordinated remains incomplete.
 To address this, we integrated single-cell RNA sequencing (scRNA-seq), cellular indexing of transcriptomes and epitopes (CITE-seq), histone modification profiling, and chromatin accessibility analysis to dissect murine erythropoiesis at high resolution. scRNA-seq identified nine transcriptional clusters (C1–C9) along a continuous trajectory. Conventional R3 cells (CD71⁺Ter119high) were further resolved into three subclusters (C5–C7), exhibiting heterogeneity in ribosome biogenesis, heme metabolism, hemoglobin gene expression, chromatin condensation, and enucleation priming. Notably, C4 displayed the highest unique molecular identifier (UMI) counts and total RNA content, co-expressing markers from neighboring clusters alongside its own signature genes.
 Differential expression analysis of C4 versus neighboring clusters revealed three coordinated changes: (1) initiation of heme biosynthesis and hemoglobin gene expression, (2) a decline in ribosome biogenesis compared to C3, and (3) a metabolic switch from glycolysis to oxidative phosphorylation (OXPHOS), marked by a 98% reduction in Pkm (pyruvate kinase M) expression. To probe this metabolic shift, we FACS-sorted R2 (CD71⁺Ter119⁻), R2–3 (CD71⁺Ter119med, corresponding to C4), and R3 populations to assess intracellular pyruvate and ATP levels and chromatin accessibility. Consistent with reduced Pkm, R2–3 cells showed a sharp decline in pyruvate and diminished ATAC-seq signals at the Pkm locus, suggesting epigenetic repression. CUT&Tag data revealed increased H3K27me3 at Pkm in R2–3, implicating Polycomb Repressive Complex 2 (PRC2) in the regulation of this metabolic switch.
 To test PRC2's functional role, we generated Ezh2 conditional knockout mice using Vav-Cre. These mice exhibited defective erythropoiesis, with R2–3 cell accumulation and greatly reduced downstream R3/R4 populations, resulting in anemia. scRNA-seq of Ezh2 KO bone marrow precursors revealed premature chromatin opening at hemoglobin and heme synthesis gene loci (e.g., Alas2, Fech), even though these genes were not direct PRC2 targets in wild-type cells. Hb gene expression in KO R4 cells remained comparable to wild-type, but heme synthesis gene expression was suboptimal, suggesting impaired heme availability.
 Strikingly, Pkm expression increased ~160-fold in KO C4 cells, and glycolysis gene scores were elevated across KO clusters, indicating enhanced pyruvate-generating capacity. FACS-sorted KO R2–3 and R3 cells confirmed pyruvate accumulation, exhibited reduced ATP levels, and showed impaired proliferation and increased apoptosis.
 To test the effect of pyruvate accumulation directly, we treated G1ER cells (arrested at the R2–R2–3 transition) with increasing pyruvate (10–50 mM). Intracellular pyruvate increased with dose, while 40–50 mM pyruvate led to ATP reduction, impaired proliferation, and increased cell death. Similarly, in vitro–differentiated Lin⁻ bone marrow cells treated with 50 mM pyruvate exhibited reduced frequencies of R2–3 and R3 cells and elevated cell death. In contrast, Lin⁺ cells cultured under the same conditions were less affected, suggesting lineage-stage-specific sensitivity to pyruvate overload.
 In summary, we identify a metabolically distinct transitional population (C4) in which erythroid precursors undergo a PRC2-regulated shift from glycolysis to OXPHOS, coinciding with the onset of heme and hemoglobin production. PRC2 suppresses Pkm expression via H3K27me3 deposition, thereby restraining pyruvate production and coordinating energy metabolism with terminal differentiation. Loss of PRC2 disrupts this metabolic control, leading to pyruvate accumulation, ATP deficiency, and defective erythropoiesis culminating in anemia.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a64b2b511f581be6d16c4c74b3ebadad0ee9f0b5" target='_blank'>
              PRC2-regulated glycolysis-to-oxphos metabolic switch coordinates the onset of terminal erythropoiesis
              </a>
            </td>
          <td>
            Zhiling Shi, Jiazhuo Li, Panpan Zhu, Stuart H Orkin, Huafeng Xie
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Disclosure: S. Gadad: None. M. Sedano: None. Triple-negative breast cancer (TNBC) accounts for approximately 12-20% of all new breast cancer cases diagnosed annually and is known to be associated with poorer clinical outcomes compared to other molecular subtypes of breast cancer. Patients with TNBC typically present with larger tumors and higher tumor grades, resulting in decreased overall and recurrence-free survival rates compared to individuals with non-TNBC tumors. The strategies involved in treating TNBC include chemotherapy agents such as doxorubicin, an anthracycline chemotherapeutic agent; however, its efficacy may vary due to the development of resistance. The specific mechanisms behind the constantly evolving resistance of TNBC to chemotherapy remain unclear, emphasizing the need for further investigation. In our study, we utilized two TNBC cell lines engineered with gradually increasing concentrations of doxorubicin to develop resistance. We conducted integrative genomic analyses to uncover the cellular mechanisms involved, identified altered molecular pathways relevant to cancer, and ascertained the factors contributing to the phenotypic characteristics of doxorubicin-resistant TNBC. Our findings indicate that many evolutionarily young transcripts with coding or limited coding potential are differentially expressed compared to non-resistant TNBC cells. Furthermore, analyses of adjacent genomic regions containing aberrantly expressed genes revealed that these genes are located in areas implicated in cellular plasticity, allowing cancer cells to adapt to the presence of doxorubicin, potentially through transcriptional reprogramming. We are also investigating the genetic and epigenetic alterations that may have facilitated the evolution of TNBC cells into doxorubicin-resistant variants. Additionally, we are assessing the clinical implications of the identified factors by studying them in vivo using mouse models with genetically engineered chemo-sensitive and -resistant cells, linking the outcomes to changes observed in patient samples associated with clinical prognosis. By elucidating the specific mechanisms contributing to the development of doxorubicin resistance in TNBC, we aim to uncover the underlying processes driving resistance and recurrence, potentially paving the way for novel therapeutic strategies. S.S.G. is a CPRIT Scholar in Cancer Research. S.S.G. was supported by a First-time faculty recruitment award from the Cancer Prevention and Research Institute of Texas (CPRIT; RR170020). S.S.G. is supported by the The American Cancer Society (RSG-22-170-01-RMC), NIH 1R16GM149497 grants and CPRIT-TREC (RP230420). Presentation: Sunday, July 13, 2025">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48b4373af8394b05d1e7e23058cecf5a8e12738f" target='_blank'>
              SUN-210 The role of transcriptional reprogramming and genomic alterations in the transition from chemo-sensitive to chemoresistant triple-negative breast cancer
              </a>
            </td>
          <td>
            Shrikanth S. Gadad, Melina J. Sedano
          </td>
          <td>2025-10-01</td>
          <td>Journal of the Endocrine Society</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="


 Introduction: The t(4;14) translocation subtype of multiple myeloma (MM) is characterized by overexpression of NSD2, a histone methyltransferase which catalyzes di-methylation of histone H3 lysine 36 (H3K36me2). About 12% of MM patients have t(4;14) and this is associated with poor outcome. If and how NSD2 drives unique epigenetic vulnerabilities in this subtype of MM is not well understood.
 Methods: Primary MM samples from the MMRF CoMMpass trial (NCT01454296) were analyzed by Whole Genome Bisulfite Sequencing (WGBS). CRISPR/Cas9 was used to genetically ablate the NSD2 translocated allele in KMS11 and KMS18 (hereafter referred to as NSD2-high and NSD2-low). Multiple single cell clones were isolated and confirmed by immunoblot analysis and sequencing. KMS11 NTKO and TKO models (Kuo et al. Molecular Cell 2011) were also used for CUT&Tag and DNAm analysis. Cell responses to decitabine (DAC) and GSK-3685032 were evaluated by flow cytometry for Annexin V and live/dead staining on day 4. Dynamic BH3 profiling (DBP) was performed similar to previously described (Matulis, ASH 2024). Gene expression was assessed using RNAseq with ribosomal RNA depletion (Kapa).
 Results: Analysis of CoMMpass samples showed higher levels of DNAm in the t(4;14) subtype compared to other subtypes (p=1.17x10-12). Consistent with DNAm being connected to NSD2 overexpression, KMS11 NSD2-high cells had higher DNAm than isogenic NSD2-low cells (p<1x10-9). Elevated DNAm in NSD2-high cells co-occurred in genomic regions with higher levels of H3K36me2 (OR=1.85, p<1x10-15). Treating KMS11 and KMS18 NSD2 isogenic cell lines with the DNA hypomethylating agent DAC showed NSD2-high cells are significantly more sensitive to DNAm inhibition (p<0.01 at ≥100 nM for KMS11 and p<0.001 at ≥500 nM for KMS18). This sensitivity to DNAm inhibition was seen again using the DNMT1 enzymatic inhibitor GSK-3685032 (p<0.001 at ≥100 nM). DBP of KMS18 showed NSD2-high cells increased mitochondrial priming when treated with DAC (p<0.001) whereas NSD2-low cells did not significantly change, consistent with the increased apoptosis observed in NSD2-high cells.
 RNAseq of KMS11 NSD2-high and NSD2-low clones treated with 0, 100, or 500 nM DAC showed 3,599 differentially expressed genes (FDR<0.01, fold-change>2). Genes downregulated with DAC treatment were common between NSD2-high and NSD2-low cells and included genes involved in cell cycle and MYC targets as indicated by Gene Set Enrichment Analysis (GSEA). Genes upregulated with DAC-treatment were more distinct between NSD2-high and NSD2-low cells, with GSEA indicating interferon response and TNFa signaling via NFKB more significantly induced in NSD2-high cells (FDR<0.0023) than in the NSD2-low cells (FDR>0.13).
 Further investigation of DAC-induced RNAseq shows an increase in transcription of endogenous retroviruses (ERVs), specifically in the NSD2-high KMS11 cells. This suggests that hypomethylating agents induce transcription of these ERVs, which then induce an intracellular interferon response, and this is specific to NSD2-high MM. Consistent with this, these ERVs had higher levels of DNAm in isogenic KMS11 NSD2-high cells as compared to NSD2-low cells (FDR<0.01). This was also observed in CoMMpass samples where higher DNA methylation was found at these ERVs in t(4;14) MM.
 Conclusions: NSD2-mediated increases in H3K36me2 in t(4;14) MM correspond with higher DNAm as compared to other MM subtypes. This is consistent with data from other cell types suggesting H3K36me2 is recognized by the PWWP domains of the DNMT3 DNA methyltransferasaes. Multiple NSD2 isogenic models suggest this creates a preferential sensitivity to DNA hypomethylating agents in NSD2-high MM. This phenomenon was shown both with DAC, a cytosine analog that incorporates into DNA, as well as with GSK-3685032, a DNMT1 enzymatic inhibitor. Both Annexin V staining and DBP indicated increased apoptosis, specific to NSD2-high cells. In addition, RNAseq data suggests this increased apoptosis is due to re-expression of ERVs that are triggering a viral defense response and interferon signaling. Consistent with this, t(4;14) cell line models and patient samples had higher levels of DNAm at ERVs, suggesting these are silenced through DNAm in t(4;14), but by other mechanisms in non-t(4;14) MM. Together, these data indicate NSD2 remodels the epigenome of t(4;14) MM creating unique epigenetic dependencies that can be exploited with readily available DNA hypomethylating agents.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c372d03852ab31bfaf4fcce61c8d2085a69b5cdf" target='_blank'>
              NSD2 expression remodels DNA methylation, creating a targetable epigenetic dependency in t(4;14) multiple myeloma
              </a>
            </td>
          <td>
            Anna Baj, Jonathan C Patton, Robert M Chavez, Shannon M Matulis, Doris R. Powell, Mala Shanmugam, Arun Wiita, Constantine S Mitsiades, J. Keats, Jonathan Licht, S. Lonial, Lawrence Boise, Benjamin G. Barwick
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>110</td>
        </tr>

        <tr id="


 Background: Despite recent advancements in treatment for multiple myeloma (MM), most patients eventually relapse due to the emergence of resistant tumor subclones. Bulk DNA sequencing has identified recurrent resistance-associated mutations and can infer subclonal architecture, but it lacks the resolution to determine whether multiple mutations co-exist within the same malignant cell or arise in separate subclones. This distinction is critical, as multiple somatic variants within individual cells may confer resistance that does not arise from any single alteration alone. We hypothesized that targeted single-cell DNA sequencing of myeloma tumor cells would reveal the co-occurrence of mutations within individual cells that are missed by conventional bulk sequencing approaches.
 Methods: Bone marrow aspirates were collected from 46 patients, including 9 with newly diagnosed and 37 with relapsed or refractory MM. Two patients were sampled at multiple time points, yielding a total of 48 samples. CD138⁺ plasma cells were enriched using magnetic-activated cell sorting and subjected to single-cell encapsulation for simultaneous genomic and proteomic profiling. Libraries were prepared using the Mission Bio Tapestri Single-Cell Multiple Myeloma Multiomics Assay, which includes amplicons targeting single-nucleotide variants, insertions and deletions, and focal copy number aberrations commonly altered in the disease. In parallel, immunoglobulin heavy- and light-chain V(D)J clonotypes were identified, and cells were labeled with a 20-plex oligo-conjugated antibody panel to quantify surface protein markers associated with cell lineage and immunotherapy targets. Tumor cells were defined as those harboring the dominant immunoglobulin V(D)J sequence within each sample. Reference cells were identified as VDJ-negative and expressing high levels of non–plasma cell markers (CD3, CD19, or CD33). Somatic mutations were identified by comparing variant allele frequencies between tumor and reference cell populations.
 Results: Across all 48 samples, 146,946 cells were sequenced, with 68,052 of those being tumor cells. Hallmark MM copy number alterations, such as 17p deletion, 1q amplification, and 13q deletion, were identified using the copy number assay. The most frequent mutations identified were in KRAS, TP53, TENT5C, and CRBN. The two patients with CRBN mutations underwent 27 and 34 prior cycles of immunomodulatory (IMiD) therapy, respectively, compared to a median of 8.0 cycles in patients without CRBN mutations, raising the possibility of treatment-driven selection. In another case, we identified three mutually exclusive KRAS mutations (Q61H, G12V, and G12D) in distinct subclones within a single patient, consistent with convergent evolution toward activation of the MAPK pathway. We also resolved a true “double hit” in another patient, characterized by 58% of tumor cells carrying a deletion of one allele and a somatic mutation in the other, consistent with biallelic inactivation. Finally, we observed CDKN2A more frequently co-mutated with TP53, PTPRT, and TENT5C than as isolated events, suggesting multi-hit inactivation of key cell-cycle checkpoints and DNA-damage response pathways that may drive selective growth advantage or therapeutic resistance. CDKN2A and TP53 are canonical tumor suppressors that regulate G1/S transition and apoptosis, respectively, while PTPRT encodes a phosphatase that negatively regulates STAT3 signaling, and TENT5C is a tumor suppressor involved in mRNA stability and plasma cell homeostasis. These recurrent co-mutations may act synergistically by simultaneously disrupting transcriptional and signaling regulation, thereby conferring a selective tumor cell growth advantage under therapeutic pressure.
 Conclusions: Targeted single-cell DNA sequencing reveals co-occurring mutations within individual myeloma cells that are obscured by bulk approaches, providing critical insight into the cooperative and competitive genomic events driving disease progression and resistance. These findings support a model in which myeloma subclones accumulate synergistic alterations within the same cell. Work is ongoing to provide functional validation of key mutation pairs and their integration with transcriptomic data to refine the mechanistic understanding and identify candidates for combination therapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98368199e493cbd0fcaadb2df0e1e6f8e200c1a5" target='_blank'>
              Single-cell DNA sequencing uncovers synergistic co-mutations in multiple myeloma
              </a>
            </td>
          <td>
            Ava Amidei, B. Diamond, Elizabeth Lyubchenko, Stephanie Fernandes, Baharak G. Fard, L. Nousheen, Marc Arribas-Layton, Adam Sciambi, D. Kazandjian, James Hoffman, Abhishek Pandey, Marcella Kaddoura, Brian Walker, O. Landgren, D. Coffey
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="
 A major barrier in understanding adult diffuse gliomas lies in the inability to track tumor evolution in real time. To address this, we employed a novel method known as continuous lineage tracing, which integrates CRISPR/Cas9-based expressed DNA barcoding with single-cell RNA sequencing, enabling a phylogenetic approach to studying tumor development. Patient-derived glioma-initiating cell lines were engineered with expressed barcodes targeted by CRISPR/Cas9 and implanted into mice to create intracranial xenografts. Tumors underwent single-cell RNA sequencing; expressed barcodes were used to infer clonal relationships, and transcriptomic profiles enabled cell state classification. Phylogenetic lineage trees were reconstructed using lineage inference algorithms to characterize cell fitness, expansion, and plasticity. Our analysis uncovered extensive intra-clonal cell state heterogeneity, indicating active phenotype switching prior to therapy. We identified consistent transcriptional programs associated with tumor engraftment and in vivo clonal advantage. Lineage tracing revealed gene expression signatures linked to key phenotypes: fitness, enriched for neural-mesenchymal and injury-response pathways; expansion, associated with RNA splicing; and plasticity, correlated with cell cycle and DNA repair programs. Glioma stem cells appeared to span a transcriptional continuum from undifferentiated, high-fitness states to more differentiated, low-fitness states, with high-fitness cells potentially representing transitional phenotypes. We then validated these findings in a cohort of 185 surgically resected gliomas with matched bulk RNA and proteomic data. Phylogenetic gene signatures differed markedly between IDH-mutant and IDH-wildtype tumors. When stratified by tumor type, both fitness and expansion signatures were significantly associated with overall survival in GBM (median 12.6 months, p=0.041) and oligodendrogliomas (median 66 months, p=0.027). This study demonstrates the utility of continuous lineage tracing to reconstruct tumor evolution and identify transcriptional programs linked to tumor growth and prognosis. Our approach provides a powerful framework for dissecting glioma biology and identifying potential therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93d67c2559a40c369b3a39bc8fe0aad62196164a" target='_blank'>
              STEM-24. SINGLE CELL CRISPR/CAS-9 LINEAGE TRACING REVEALS EVOLUTIONARY FITNESS, EXPANSION AND PLASTICITY AXES IN ADULT DIFFUSE GLIOMAS
              </a>
            </td>
          <td>
            A. Ajisebutu, Xiaojun Fan, Chloe Gui, Jeff Liu, Vikas Patil, A. Landry, Y. Ellenbogen, L. Yefet, Phooja Persaud, F. Nassiri, G. Zadeh
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Accurate detection of oncogenic gene fusions is becoming increasingly important given the availability of highly effective targeted therapies. However, their identification in clinical practice remains challenging due to the rarity of individual events, diversity of partner genes, and variability of breakpoint locations. Conventional approaches such as immunohistochemistry (IHC) and fluorescence in situ hybridization (FISH) lack multiplexing capacity and demonstrate variable sensitivity and specificity, while direct identification of fusion transcripts in whole-transcriptome sequencing (RNA-seq) profiles provides broader applicability but limited sensitivity, as fusion junctions are frequently supported by a minimal number of reads or even no reads at all. In this study, a novel approach was employed to accurately detect clinically actionable RET (REarranged during Transfection) fusions. This approach entailed the measurement of the imbalance in RNA-seq read coverage of potential fusion oncogenes at their 3′ and 5′ exons. A total of 1327 experimental solid tumor RNA-seq profiles were screened, including 154 non-small cell lung cancer and 221 thyroid cancer samples. The RET status was validated in 78 selected cases by targeted NGS and Sanger sequencing. An analysis of the coverage imbalance was conducted, which enabled the accurate discrimination between true and false positive RET fusions. This approach outperformed other methods and yielded 100% sensitivity and specificity with optimized thresholds. The findings were validated using an independent cohort of 79 thyroid cancer cases, confirming the reliability of the results. Among the 18 RET fusion-positive samples, one was identified as an extremely rare case (RUFY3::RET), and two were determined to be novel fusions (FN1::RET, PPP1R21::RET). The findings of this study demonstrate that exon coverage imbalance analysis serves as a robust complement to computational RNA-seq analysis pipelines for the detection of clinically relevant RET fusions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7e82c8e2a86b2423abf1721592b4c4f068cbddf" target='_blank'>
              Accurate RET Fusion Detection in Solid Tumors Using RNA Sequencing Coverage Imbalance Analysis
              </a>
            </td>
          <td>
            Ivan Gaziev, Anna Khristichenko, Daniil V. Luppov, M. Suntsova, E. Bondarenko, Maria Reinberg, Alina Matrosova, N. Khilal, M. Sorokin, M. Sekacheva, E. Poddubskaya, Anton Buzdin, G. Zakharova
          </td>
          <td>2025-11-22</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Pancreatic cancer, recognized as a refractory tumor, has an overall survival rate of less than 10%, and its mortality rate continues to rise. Due to the low immune activity induced by its unique tumor microenvironment, pancreatic cancer is classified as a “cold” tumor and is insensitive to current immunotherapies. However, little is still known about the identification and functional mechanisms of key regulatory molecules in the formation of “cold” tumors. In this study, we identified the pancreatic ductal cell niche through single-cell sequencing. By calculating CNV scores using inferCNV to distinguish malignant from non-malignant cells and analyzing differences in transcriptional levels, we constructed a 15-gene model. The inhibitory effect of UBE2H on the immune microenvironment was studied through single-cell sequencing analysis, including the inhibition of T cell function, impairment of antigen presentation in macrophages and neutrophils, and activation of neutrophil extracellular traps (NETs). Our results revealed the presence of a UBE2H+ cell population within the pancreatic ductal cell niche, primarily observed in pancreatic cancer samples. Notably, single-cell sequencing analysis of UBE2H overexpression revealed an enhanced T cell PD-L1 interaction axis and a downregulation of antigen presentation pathways in macrophages and neutrophils (antigen processing and presentation, macrophage activation, and neutrophil activation). Further in vivo experiments confirmed that high UBE2H expression promotes tumor progression, leading to increased T cell exhaustion (PD1) and decreased activation (CD69, GZMB, IFNR) in the immune microenvironment, impaired antigen presentation of macrophages and neutrophils (H2Kb, I-Ab, I-A/E), and increased neutrophil extracellular trap formation (MPO, NE, CITH3). This study revealed the presence of a UBE2H+ cell population within the pancreatic ductal cell niche and analyzed the inhibition of UBE2H on the immune microenvironment of pancreatic cancer through single-cell sequencing and in vivo experiments, providing important clues for the formation of “cold” tumors in pancreatic cancer and opening new directions for exploring new treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49ecc8251593af0fcef99c0ebc48949b3c7fba9f" target='_blank'>
              Integrated scRNA-seq and transcriptome analyses uncover the effects of UBE2H on the immune microenvironment regulation in pancreatic cancer
              </a>
            </td>
          <td>
            Fuxin Huang, Zhongyan Zhang, Jike Fang, Yue Chen, Jinhui Wei, Quanzhang Li, Chuanzhao Zhang, Shanzhou Huang, B. Hou
          </td>
          <td>2025-11-24</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Resistance to chemotherapy, targeted agents, and particularly immunotherapy remains the principal challenge in the management of gastrointestinal malignancies. This review aims to comprehensively delineate the molecular and microenvironmental drivers of resistance, with emphasis on mechanisms impacting immunotherapy response, and evaluate emerging, mechanism−guided interventions (including immunotherapeutic combinations) for precision therapy. We first examine intrinsic mechanisms—including drug−target alterations, dysregulated drug metabolism and efflux, hyperactivation of DNA damage repair pathways, and epigenetic remodeling—and extrinsic influences stemming from the tumor microenvironment and extracellular matrix remodeling. We then highlight epithelial–mesenchymal transition (EMT) as a critical nexus that integrates stromal cues with cell−intrinsic survival programs, thereby promoting drug efflux and immune evasion. Next, we discuss how single−cell and spatial omics, liquid biopsy, patient−derived organoids, and AI−enabled analytics facilitate subclone−level mapping of resistance networks and real−time tracking of clonal evolution. Finally, we review mechanism−based strategies—including KRAS G12C inhibitors, efflux−pump antagonists, apoptosis reactivators, and epigenetic/autophagy modulators—and propose an integrated, multimodal regimen leveraging immunotherapy where appropriate, informed by real-time drug sensitivity data (e.g., from liquid biopsy), dynamic biomarkers and AI−driven optimization to overcome resistance and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aebd9ae366f20576b8816fc8c01cf3154ec38755" target='_blank'>
              Integrated molecular and microenvironmental drivers of drug resistance in gastrointestinal cancers: mechanisms, immunotherapy challenges, and precision strategies
              </a>
            </td>
          <td>
            Heng Xu, Jiaan Lu, Jiangying Wu, Kangling Zhang, Xuancheng Zhou, Ziqi Gao, Jingqi Feng, Ziye Zhuang, Xiaolin Zhong
          </td>
          <td>2025-10-28</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="TP53-mutated myelodysplastic syndrome (MDS) and acute myeloid leukemia (AML) comprise a distinct subgroup of myeloid neoplasms with unique biological and clinical features. The molecular alterations linked to TP53 mutations drive genomic instability and treatment resistance and ultimately lead to poor survival outcomes. The disease biology is further shaped by alterations in immune response within the bone marrow microenvironment and significant changes in cellular metabolism. Conventional treatments, including chemotherapy and hypomethylating agents +/− venetoclax, offer limited benefit, with high relapse rates and short remissions. Allogeneic bone marrow transplantation is the only curative approach, but the vast majority of patients relapse. Novel therapeutic approaches—ranging from p53 reactivation strategies to immunotherapy and targeted inhibition of specific signaling pathways—are under active investigation. Our review summarizes current knowledge on the molecular pathogenesis, prognostic implications, and therapeutic landscape of TP53-mutated MDS/AML and discusses ongoing challenges and opportunities for improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8ae9969de3f5d4620c0b91a004a3598a646cad6" target='_blank'>
              Molecular and Clinical Insights into TP53-Mutated MDS and AML
              </a>
            </td>
          <td>
            Erotokritos Georgantzinos, Theodoros Karantanos
          </td>
          <td>2025-11-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c038a3eb980cd4c3376d3d5fe3b978d2d8815e95" target='_blank'>
              A Machine Learning Approach Reveals CRISPR-Cas I-F as a Genomic Marker of Antibiotic Susceptibility in Uropathogenic E. coli
              </a>
            </td>
          <td>
            Alexandra M Young, P. Humburg, Fang Liu, Michael C. Wehrhahn, A. Tay, Stephen M. Riordan, Li Zhang
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Acute lymphoblastic leukemia (ALL) is the most common hematologic malignancy in children, posing challenges for early diagnosis, relapse prediction, and individualized treatment due to its high cellular and immune heterogeneity. This study addresses these challenges by integrating deep learning-based imaging analysis with single-cell transcriptomics and T-cell receptor sequencing (TCR-seq). A ResNet50-based deep learning model was developed to classify leukemia single-cell images, achieving an accuracy of ~ 85% and an AUC of 0.86. Gradient-weighted Class Activation Mapping (Grad-CAM) indicated that nuclear morphology was the key feature for cell identification. Concurrently, Louvain clustering of single-cell RNA sequencing data revealed 18 distinct cellular subpopulations, with Cluster 4 exhibiting strong immune activity and pronounced TCRβ rearrangement. Functional analyses highlighted the roles of UBE2C and HMGB2 in leukemogenesis. Integration with TCR-seq further uncovered immune microenvironment alterations and potential relapse biomarkers. These results demonstrate that combining deep learning with single-cell multi-omics is a powerful strategy for elucidating disease mechanisms, improving early diagnosis, and informing personalized therapies for pediatric ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d9acc0f4914f12bf8600b5ee6d644cccba2b026" target='_blank'>
              ResNet50 and Single-Cell Multi-Omics analysis identify key cellular and molecular features in pediatric acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Taoli Suo, Jing Zhang, Yanli Chen, Hongyu Shi, Haiyan Lu
          </td>
          <td>2025-10-23</td>
          <td>Annals of Hematology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Glioblastomas (GBM) are aggressive brain tumors characterized by significant heterogeneity and notable resistance to treatment, particularly to temozolomide (TMZ), a standard chemotherapeutic agent utilized in clinical practice. A key mechanism underlying this resistance is the upregulation of the MGMT (O6-methylguanine-DNA methyltransferase) gene. Prior studies have shown that alterations in MGMT promoter methylation status influence MGMT expression levels. However, the regulatory mechanisms governing this process remain unclear. To identify the molecular changes associated with TMZ resistance, we established eight TMZ-resistant cell lines derived from patient-derived glioma stem cells (PDGs). Quantitative PCR (qPCR) and Western blot (WB) analyses confirmed elevated MGMT expression in these resistant cells. RNA sequencing of both treatment-naïve and resistant cells revealed significant alterations in key epigenetic enzymes, including DNA methyltransferases (DNMTs) DNMT1 and DNMT3A, as well as various DNA demethylation enzymes and methyl-binding domains, such as TETs and MBDs. qPCR further validated the downregulation of UHRF1 and DNMT1 mRNA in resistant cells. Besides, treatment-naïve cells exhibiting high methyltransferase expression demonstrated increased susceptibility to 5-Azacytidine (5-AZA), a pan DNMT inhibitor, compared to their resistant counterparts, which displayed reduced methylation capacity. Moreover, 5-AZA-mediated epigenetic reprogramming enhanced the resistance of treatment-naïve cells by inhibiting cell proliferation and upregulating MGMT expression. Notably, withdrawal of 5-AZA reverted MGMT expression levels to those observed in untreated cells. Retrospective analysis of sequencing data from paired initial and recurrent tumors in adult diffuse glioma patients revealed a consistent negative correlation between DNMTs expression and MGMT expression, with DNMT1 emerging as the most significant factor. Finally, specific inhibition of DNMT1 resulted in a marked upregulation of MGMT expression. In summary, our work highlights the role of MGMT in glioma TMZ resistance and identifies the downregulation of DNMTs as a critical mechanism facilitating MGMT upregulation, thereby contributing to chemotherapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/184ff52797748eb27e56749cfa70525ad6478ba8" target='_blank'>
              DDDR-32. DNMTs epigenetically regulate MGMT expression and contribute to TMZ resistance in glioblastoma
              </a>
            </td>
          <td>
            Mengli Yan, Dong Song, Qi An, R. Chai, Jiguang Wang
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Ovarian cancer is a highly lethal malignancy, with treatment efficacy frequently compromised by the development of cisplatin resistance. This study investigates the molecular mechanisms driving cisplatin resistance, focusing on interferon-induced protein with tetratricopeptide repeats 1 (IFIT1), a key interferon-stimulated gene. Through comprehensive transcriptome profiling, we systematically compared gene expression patterns between cisplatin-sensitive and cisplatin-resistant ovarian cancer cells, identifying a consistent upregulation of IFIT1 in resistant cells. Functional assays provided evidence that IFIT1 plays a central role in mediating cisplatin resistance. Mechanistically, IFIT1 was found to be associated with activation of inflammatory signaling pathways, suggesting its involvement in resistance-related immune regulation. Clinically, elevated IFIT1 expression significantly correlated with an unfavorable prognosis in ovarian cancer patients. Moreover, we demonstrated that hypoxia-inducible factor-1 alpha transcriptionally regulates IFIT1, linking its expression to hypoxia response. Overall, these findings identify IFIT1 as a key driver of cisplatin resistance in ovarian cancer, shedding light on its clinical significance and the regulatory networks underpinning therapy resistance. IFIT1 holds promise both as a prognostic indicator and as a target for therapeutic intervention. However, its clinical translation will depend on rigorous validation and in-depth exploration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e3e993b5a5faff584d9a192dda2fe082dc88120" target='_blank'>
              Overcoming cisplatin resistance in ovarian cancer by targeting IFIT1-mediated inflammation responses
              </a>
            </td>
          <td>
            Min Hu, Xuejiao Zhao, Xiuming Li, Bin Liu, Chunluan Yuan
          </td>
          <td>2025-11-06</td>
          <td>Tumor Discovery</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Simple Summary Medulloblastoma is the most common malignant brain tumor in children and includes several molecular subtypes with distinct clinical outcomes. Because genetic mutations are relatively rare in this tumor, our study focused on understanding how epigenetic regulation—the chemical and structural changes that control gene activity without altering DNA—contributes to disease progression. Using data from large patient cohorts and single-cell sequencing, we identified specific epigenetic factors that distinguish tumor subtypes and correlate with prognosis. We also developed an epigenetic score that predicts patient survival and is particularly elevated in aggressive Sonic Hedgehog (SHH) tumors. These results suggest that dysregulated epigenetic programs affecting RNA metabolism and cell proliferation play a key role in medulloblastoma aggressiveness, highlighting new potential targets for therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/733b0f36e6a075919ebc25ace84fd8a8035e0b6e" target='_blank'>
              Single-Cell Heterogeneity of Epigenetic Factor Regulation Deciphers Alteration of RNA Metabolism During Proliferative SHH-Medulloblastoma
              </a>
            </td>
          <td>
            R. Francés, Jenny Bonifacio-Mundaca, Íñigo Casafont, Christophe Desterke, Jorge Mata-Garrido
          </td>
          <td>2025-10-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) is defined by the expansion of a lineage of genetically identical cells in blood. Genetic lesions that confer a fitness advantage, such as leukemogenic point mutations or mosaic chromosomal alterations (mCAs), are frequent mediators of CH. However, recent analyses of both single cell-derived colonies of hematopoietic cells and population sequencing cohorts have revealed CH frequently occurs in the absence of known driver genetic lesions. To characterize CH without known driver genetic lesions, we use 51,399 deeply sequenced whole genomes from the NHLBI TOPMed sequencing initiative to perform simultaneous germline and somatic mutation analyses among individuals without leukemogenic point mutations (LPM), which we term CH-LPMneg. We quantify CH by estimating the total mutation burden. Because estimating somatic mutation burden without a paired-tissue sample is challenging, we develop a novel statistical method, the Genomic and Epigenomic informed Mutation (GEM) rate, that uses external genomic and epigenomic data sources to distinguish artifactual signals from true somatic mutations. We perform a genome-wide association study of GEM to discover the germline determinants of CH-LPMneg. We identify seven genes associated with CH-LPMneg (TCL1A, TERT, SMC4, NRIP1, PRDM16, MSRA, SCARB1).Functional analyses of SMC4 and NRIP1 implicated altered hematopoietic stem cell self-renewal and proliferation as the primary mediator of mutation burden in blood. We then perform comprehensive multi-tissue transcriptomic analyses, finding that the expression levels of 404 genes are associated with GEM. Finally, we perform phenotypic association meta-analyses across four cohorts, finding that GEM is associated with increased white blood cell count, but is not significantly associated with incident stroke or coronary disease events. Overall, we develop GEM for quantifying mutation burden from WGS and use GEM to discover the genetic, genomic, and phenotypic correlates of CH-LPMneg. Here the authors develop a novel statistical method for quantifying mutation burden from whole genome sequencing data and use it to discover the genetic, genomic, and phenotypic correlates of clonal hematopoiesis without known driver genetic lesions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a90296a3758d552c4d460a056b5b1554e1d3253" target='_blank'>
              Genetic determinants and genomic consequences of non-leukemogenic somatic point mutations
              </a>
            </td>
          <td>
            J. Weinstock, Sharjeel A. Chaudhry, Maria Ioannou, Maria Viskadourou, Paula Reventun, Yasminka A. Jakubek, L. Liggett, C. Laurie, J. Broome, Alyna T Khan, K. Taylor, Xiuqing Guo, P. Peyser, Eric Boerwinkle, Nathalie Chami, Eimear E. Kenny, Ruth J. F. Loos, B. Psaty, R. Tracy, J. Brody, Jeong H Yun, M. Cho, Ramachandran S. Vasan, Sharon L. R. Kardia, Jennifer A Smith, L. Raffield, A. Bidulescu, Emily C O'Brien, Mariza de Andrade, Jerome I. Rotter, S. Rich, Yii-DerIda Chen, C. Gu, C. Hsiung, C. Kooperberg, Bernhard Haring, Rami Nassir, Rasika A Mathias, Alexander P. Reiner, Vijay G. Sankaran, Charlie Lowenstein, T. Blackwell, G. Abecasis, Albert V. Smith, H. Kang, P. Natarajan, S. Jaiswal, Alexander G. Bick, Wendy S. Post, P. Scheet, Paul L. Auer, Theodoros Karantanos, Alexis Battle, M. Arvanitis
          </td>
          <td>2025-10-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>188</td>
        </tr>

        <tr id="


 Acute myeloid leukemia (AML) is an aggressive hematological malignancy with 5-year survival rate of less than 32%, mainly due to treatment failure and disease relapse. Thus, a systematic framework that allows us to monitor treatment response is urgently needed. Previously, we applied a state-transition modeling framework to represent AML disease evolution as messenger RNA (mRNA) and microRNA (miRNA) transcriptome trajectories in their respective state-spaces, partitioning samples into health, transition, and leukemic states based on mathematically defined critical points. These state-spaces were constructed using time-series RNA-seq data collected from a murine model of AML which mimics the fusion geneCBFB::MYH11 (CM) created by chromosome 16 inversion.
 To test the hypothesis that response to chemotherapy could be predicted by mRNA and miRNA transcriptome dynamics using a state-transition framework, we performed an experiment and mathematical analysis of longitudinal sample collections and multiomic profiling of peripheral blood from CM knock-in mouse model (Cbfb56M/+/Mx1-Cre) before, during, and after chemotherapy, as follows. After detection of overt AML (circulating cKit+ > 20%), CM mice (n=9) were treated with a “5+3” combination of cytarabine (50mg/kg/day, 5 days) and daunorubicin (1.5mg/kg/day, 3 days), modeling the standard “7+3” chemotherapy. Peripheral blood samples were collected weekly and were subjected to bulk RNA-seq and miRNA-seq. We used PCA to project the samples into a reference state-space constructed using untreated CM mice and C57BL/6 WT controls. The mRNA and miRNA transcriptome trajectories were first analyzed independently and then together to form a 2D multiomic state-space.
 After chemotherapy, we observed movement of both mRNA and miRNA transcriptomes from a leukemia state back towards a healthy state, before relapsing back to overt AML. Interestingly, we observed a desynchronization of mRNA and miRNA transcriptome dynamics post-chemotherapy. Specifically, the mRNA transcriptome immediately moved from a leukemia state to a health state, with an average peak response at week 4 (n=7; before week 4, n=2), however, a delayed response of 2 or more weeks was observed in the miRNA transcriptome trajectories, which continued to move towards a disease state post-treatment before moving towards a health state and reached a maximum response at later time-points (4 weeks, n=3; 5 weeks, n=2; 6 weeks and after, n=4). These desynchronized dynamics post-chemotherapy were enabled by the state-transition critical points and appeared more evident when visualized in a 2D multiomic state-space.
 To identify genes, miRNAs, and pathways altered by chemotherapy, we performed differential gene expression and gene set enrichment analyses. These analyses revealed that metabolic pathways such as OXPHOS, fatty acid metabolism and glycolysis are downregulated after chemotherapy and upregulated during relapse (p<0.05).
 We also performed a weighted gene co-expression network analysis on mRNAs and miRNAs to identify features with similar expression dynamics. We found a set of 34 miRNAs which were significantly upregulated and responsible for the delayed response of the miRNA transcriptome dynamics post-chemotherapy. Out of these 34 miRNAs, 23 are found in the DLK1-DIO3 imprinted region, located on chromosomes 14q32 in humans and 12qF1 in mice, which has been associated with stress response, pathogenesis of APL, solid tumors, and type 2 diabetes. These miRNAs have not previously been associated with effects of standard-of-care chemotherapy in AML. The co-localization of these miRNAs to a region of the genome suggests a potential explanation for genome instability and chemotherapy resistance.
 In summary, this study provides a longitudinal multiomic characterization of genomic instability and desynchronization of patterns of mRNA and miRNA expression induced by standard-of-care “7+3” therapy in AML, guided by mathematical analysis and critical points.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d8bb9687ac3e42f4c8f48f1349be6b29eb33446" target='_blank'>
              From theory to therapy: Predicting response to chemotherapy with multiomic longitudinal profiling of peripheral blood in AML
              </a>
            </td>
          <td>
            Jennifer Rangel Ambriz, Ziang Chen, , D. Frankhouser, Denis O’Meally, S. Branciamore, Jihyun Irizarry, Bin Zhang, Lianjun Zhang, Ying-Chieh Chen, Guido Marcucci, Russell C. Rockne, Ya-Huei Kuo
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eab80b92dbc650636e5eb2f387901a782d6b51b0" target='_blank'>
              Synergistic inhibition of CHK1 and MUS81 to combat replication stress resistance in high-risk neuroblastoma.
              </a>
            </td>
          <td>
            E. Hilgert, Christophe Van Neste, Sarah-Lee Bekaert, F. Martens, Suzanne Vanhauwaert, E. Sanders, P. Verstraelen, Martijn Risseeuw, E. M. Westerhout, Mark A Grannetia, Serge Van Calenbergh, Nadine Van Roy, W. D. De Vos, Rob W F Wolthuis, Frank Speleman, L. Depestel, K. Durinck
          </td>
          <td>2025-11-27</td>
          <td>Scientific reports</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="
 Gastrointestinal (GI) tumours account for a significant proportion of cancer cases and deaths globally; GI tumours show similarities regarding their development, diagnosis and treatment. Genomic and transcriptome analyses have been used to describe cancers, revealing driver mutations, aberrant regulatory pathways and several disease subtypes linked to tumours. However, these studies have not considered patient heterogeneity (intra‐tumour heterogeneity). Genetic heterogeneity, developmental programmes, epigenetics and the tumour microenvironment influence intra‐tumour heterogeneity. These factors are critical for tumour growth, clinical diagnosis, treatment resistance, metastasis and other cancer characteristics. Single‐cell genomics can evaluate transcriptional features at a single‐cell resolution and analyse genetic and functional heterogeneity. In the present study, we employed cNMF to illustrate intratumoral heterogeneity and identify shared programmes in scRNA‐seq and utilised novel machine‐learning algorithms to construct a signature based on the TCGA database and the different independent tumour‐type datasets from the GEO database in RNA‐seq. In addition, we evaluated whether this signature could predict survival, development, neoadjuvant therapy and drug efficacy. We employed the DepMap programme to identify the deficiency of genes in the constructed signatures that can affect tumour development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ef16e7acbe11bf15f1be7a05835d107e03feb24" target='_blank'>
              Single‐Cell Transcriptomic Analysis Reveals Epithelial‐Mesenchymal Transition and Key Gene AGRN as a Universal Programme in Gastrointestinal Tumours by an Artificial Intelligence‐Derived Prognostic Index
              </a>
            </td>
          <td>
            Zhizhan Ni, Pengyuan Zhu, Li-ming Liu, Haiyang Wang, Jinzhe Zhou, Xianqing Wang, Song Wang, Liangchen Zhu, Songze Zhang, Yongxin Zhou, Bujun Ge, Qi Huang, Chenshen Huang
          </td>
          <td>2025-11-01</td>
          <td>Med Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="KRAS and BRAF mutations are critical oncogenic drivers in colorectal cancer (CRC), with distinct molecular and clinical implications. To uncover genes differentially expressed between these mutations, we compared KRAS G12D- and BRAF V600E-mutated CRC cell lines. Protein-protein interaction (PPI) network construction, pathway enrichment, survival analysis, and drug target screening were performed to identify therapeutic opportunities. Ten hub genes—TNF, IL1B, FN1, EGF, IFI44L, EPSTI1, AHR, COL20A1, CDH1, and SOX9—were identified as critical in KRAS-driven CRC. Enrichment analysis highlighted immune and inflammatory pathways, including “SARS-CoV-2 Signaling” and “Macrophage Stimulating Protein Signaling,” as well as cellular processes like “Positive Regulation of PI3K Signaling”. IL1B was the only hub gene significantly associated with overall survival, suggesting its role as a favorable prognostic marker. Drug screening identified selective inhibitors such as Canakinumab and Rilonacept targeting IL1B, with docking studies revealing the strongest interaction for Omeprazole with AHR, followed by Tapinarof with AHR and Donepezil with IL1B. This study sheds light on the molecular mechanisms of KRAS-mutated CRC, emphasizing IL1B as a prognostic marker. Among the identified therapeutic agents, Omeprazole demonstrated the strongest interaction with AHR, suggesting a potential for repurposing in CRC treatment, while IL1B-targeting inhibitors such as Canakinumab emerged as selective candidates for modulating tumor-promoting inflammation. Novel genes such as EPSTI1, COL20A1, CDH1, and SOX9 warrant further investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/347d7ffedc4eee87a046791d3b66c644b633f639" target='_blank'>
              A systems biology approach to identify key targets in KRAS/BRAF-mutated colorectal cancer
              </a>
            </td>
          <td>
            Shiva Shiravi, Negar Mottaghi-Dastjerdi, Behzad Shahbazi, Nahid Ahmadi, Mohammad Soltany-Rezaee-Rad, Abozar Ghorbani, Hamed Montazeri
          </td>
          <td>2025-11-19</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Drug resistance remains a significant challenge in achieving successful cancer treatment, often leading to disease recurrence and reduced patient survival. While traditional tissue biopsies provide valuable insights into tumor biology, they are invasive, infrequent, and may fail to capture the full complexity of tumor heterogeneity and dynamic molecular changes. In contrast, liquid biopsy has emerged as a minimally invasive, real-time approach for monitoring tumor evolution through the analysis of circulating biomarkers. Among these biomarkers, circular RNAs (circRNAs) - a distinct class of non-coding RNAs characterized by covalently closed-loop structures - have gained attention due to their remarkable stability, abundance in body fluids, and functional involvement in gene regulation. Increasing evidence supports the role of circRNAs in mediating drug resistance through mechanisms such as inhibition of apoptosis, epithelial-mesenchymal transition, autophagy, and drug efflux, largely via interactions with microRNAs or proteins. Advanced detection methods, including quantitative reverse transcription polymerase chain reaction, droplet digital polymerase chain reaction, and RNA sequencing, combined with computational tools, enable precise profiling of circRNAs in plasma or exosomes. CircRNA-based liquid biopsies offer a dynamic, non-invasive strategy for early detection of therapeutic resistance and may guide personalized treatment decisions. This review highlights the technological advancements, biological relevance, and clinical promise of circRNAs as circulating biomarkers, emphasizing their potential in precision oncology and future collaborative translational applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df152da0476d5f6e5b6795f9a52718cb169ae7e7" target='_blank'>
              Circular RNA-based liquid biopsy: a promising approach for monitoring drug resistance in cancer
              </a>
            </td>
          <td>
            Desh Deepak Singh, Dharmendra Kumar Yadav, Dongyun Shin
          </td>
          <td>2025-10-30</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="OBJECTIVE
Large-scale CRISPR screens have identified essential genes across cancer cell lines, but links between tumor functional properties and specific dependencies require investigation to reveal the mechanisms underlying dependencies and broaden understanding of targeted therapy.


METHODS
We selected 47 breast cancer cell lines from the Cancer Cell Line Encyclopedia (CCLE) with multi-omics data including gene dependency; somatic mutations; copy number alterations; and transcriptomic, proteomic, metabolomic, and methylation data. We established a dependency marker association (DMA) analytic pipeline by using linear regression modeling to assess associations between 3,874 representative gene dependencies and multi-omics markers. Additionally, we conducted non-negative matrix factorization clustering, to stratify breast cancer cell lines according to gene dependency features, and investigated cluster-specific DMAs.


RESULTS
We interpreted valuable DMAs according to two primary aspects. First, dependencies associated with gain-of-function alterations revealed addiction to lactate transporter SLC16A3, thus suggesting a promising therapeutic target. Second, dependencies associated with loss-of-function alterations included synthetic lethality (SL), collateral SL, and prioritized metabolic SL, encompassing paralog SL (e.g., IMPDH1 and IMPDH2), single pathway SL (e.g., GFPT1 and UAP1), and alternative pathway SL (e.g., GPI and PGD). DMA analysis of the two clusters with divergent dependency signatures demonstrated that cluster1 cell lines exhibited extensive metabolism with mitochondrial protein dependencies, whereas cluster2 displays enhanced cell signaling, and reliance on DNA replication and membrane organelle regulators.


CONCLUSIONS
We established a DMA analysis pipeline linking the gene dependencies of breast cancer cell lines to multi-omics characteristics, thus elucidating the underpinnings of tumor dependencies and offering a valuable resource for developing novel precision treatment strategies incorporating relevant markers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/753908ccf816b6d23f787d4f37906f4b14d28c45" target='_blank'>
              Comprehensive investigation of the molecular basis of cancer dependencies suggests therapeutic options for breast cancer.
              </a>
            </td>
          <td>
            Rui Ding, Zhimin Shao, Tianjian Yu
          </td>
          <td>2025-10-23</td>
          <td>Cancer biology & medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Background Since its inception, RNA sequencing has been pivotal in studying differential gene expression. Despite its extensive results in large-scale oncological studies, differential expression predominantly reflects a response to cancer. Therefore, we introduce differential allelic dispersion (AD) as a more effective measure. AD highlights consistent differences in expression between the 2 alleles of a gene that is, unlike cis-expression quantitative trait loci, independent of normal genetic variation. Such differences can, for example, arise from prevalent copy number alterations or epimutations occurring in the original cancer cell, which are mitotically expanded during cancer growth, making increased AD a marker for allele-specific dysregulation in early carcinogenesis. Findings We present the maelstRom R/C++ software package that enables (differential) AD analysis solely requiring large-scale RNA sequencing data. Using the The Cancer Genome Atlas renal clear cell carcinoma cohort as a case study, we successfully benchmark maelstRom’s AD modeling using known copy number alterations. We also detect increased AD for loci featuring normal random monoallelic expression, including the X chromosome, but demonstrate minimal interference with cancer-specific AD detection. Finally, we identify early dysregulated genes (e.g., FBP1, CCDC8, ECHS1, CLDN7) and pathways in renal cancer, often related to metabolism (e.g., pentose phosphate pathway). Strikingly, many of these genes are known causal contributors to renal carcinogenesis. Conclusions Differential AD clearly indicates early dysregulation in renal cancer, complementing basic differential expression analysis in cancer transcriptomics. AD is also relevant to study random monoallelic expression and may equally detect allele-specific (dys)regulation during early development or in noncancer diseases. maelstRom is available as an open-source software package at github.com/Biobix/maelstRom.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/457e775646c46e5a03c9c6239d832825d07a63f0" target='_blank'>
              Population-level allelic dispersion modeling by maelstRom yields genome-wide maps of allele-specific dysregulation during early carcinogenesis
              </a>
            </td>
          <td>
            Cedric Stroobandt, Louis Coussement, T. Goovaerts, Femke De Graeve, Jeroen Galle, Wim Van Criekinge, Tim De Meyer
          </td>
          <td>2025-01-06</td>
          <td>GigaScience</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 BackgroundThird-generation tyrosine kinase inhibitors (3G-TKIs), olverembatinib and ponatinib, improved outcomes for patients with resistant chronic myeloid leukemia (CML). However, nearly half still developed resistance or progression. The longitudinal molecular evolution during therapy, at both genomic and transcriptomic features, remains incompletely understood. We investigated the dynamic clonal architecture and transcriptional reprogramming during 3G-TKI therapy to elucidate mechanisms of therapeutic response and resistance.
 MethodsWe analyzed 202 CML patients treated with 3G-TKIs. Baseline and serial bone marrow or blood samples were collected for targeted DNA sequencing and RNA sequencing. Variant allele frequency (VAF) trajectories were used to classify mutational dynamics. Gene expression profiles (GEPs) were analyzed by multi-machine learning and unsupervised clustering. Pathway enrichment, immune deconvolution (xCell algorithm) and T cell state analysis (TcellSI algorithm) characterized biological transitions during therapy.
 ResultsDynamic mutational profiling revealed divergent clonal evolution. In responders (n = 49), the frequency and number of ABL1and other somatic mutations significantly declined (p < 0.05). In contrast, primary resistance cases (n = 62) showed persistent or newly emergent high-VAF mutation clones, particularly ASXL1G646Wfs*12, RUNX1, and PHF6. Unsupervised clustering identified four prognostically distinct mutational patterns: (1) persistent clones; (2) emergent/expanding clones; and (3 - 4) contracting or eradicated clones. Patterns 1 and 2 were significantly correlated with therapy resistance, whereas Patterns 3 and 4 were correlated with favorable responses.
 Transcriptomic remodeling during 3G-TKI therapy provided novel insights into the molecular basis of response and resistance. In responders, leukemic stem cells (LSCs), E2F, MYC and DNA replication related pathways were markedly down-regulated (FDR < 0.001) during therapy, while immune and apoptotic pathways (TNF-α and IFN-α) were significantly activated (FDR < 0.001 - 0.04). Immune infiltration, including T cells, NK cells and dendritic cells, significantly increased (p < 0.001). In contrast, resistant patients retained or more-activated LSCs and metabolic related pathways with minimal immune induction, reflecting an immune-cold, transcriptionally static state.
 Using multi-machine learning and unsupervised clustering, 3 molecular subtypes were defined: GEPC1 (immune-active, n = 42), GEPC2 (stromal-remodeling, n = 86) and GEPC3 (stemness-high and immune-cold, n = 21) with distinct therapy responses and outcomes (p < 0.001 - 0.006). During therapy, patients in GEPC1 showed transcriptomic features characterized by down-regulation of LSCs and cell cycle related pathways (FDR < 0.001), along with strong up-regulation of immune-related pathways, such as TNF-ɑ, interferon-ɑ pathways (FDR < 0.001). Immune infiltration, such as T cells, B cells and NK cells, increased significantly during therapy. GEPC2 exhibited a moderate but partial transition, with suppression of proliferation signatures and activation of extracellular matrix and stromal pathways, and enrichment of fibroblasts and M2 macrophages. Immune activation was present but attenuated. In contrast, GEPC3 showed only limited transcriptional remodeling during therapy. Although LSCs and proliferative signatures declined slightly, they remained markedly higher compared to GEPC1/2, and immune activation was absent. xCell and TcellSI showed no increase in effector T cells, dendritic cells or cytotoxic function, consistent with an immune-cold micro-environment. Importantly, these patients often experienced disease progression without significant new mutations, suggesting transcription mediated resistance independent of clonal evolution.
 Conclusions Our integrative longitudinal analyses revealed that 3G-TKI therapy induced divergent molecular trajectories in CML: durable responses were defined by mutation clonal clearance, LSCs suppression and immune activation, whereas resistance was driven by persistent or expanding mutation clones (e.g., ASXL1G646Wfs*12, RUNX1and PHF6), sustained stemness, and immune-cold states. Transcriptomic remodeling provided a powerful early indicator of therapeutic failure and supports dynamic molecular monitoring to guide personalized therapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/280ba844f9e6179dd23e23035a9361badf93983f" target='_blank'>
              Dynamic mutational evolution and transcriptomic remodeling on 3rd-generation TKI therapy in TKI-resistant patients with chronic myeloid leukemia
              </a>
            </td>
          <td>
            Xiao-shuai Zhang, Lu Yu, Zongru Li, Ya-Zhen Qin, Qian Jiang
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) is a highly aggressive malignancy characterized by early metastasis and poor prognosis due to the limited efficacy of current treatments. Although initially responsive to chemotherapy and radiotherapy, the majority of patients with SCLC develop resistance within a year, often succumbing to distant metastases. Historically, SCLC was considered a homogeneous disease, primarily driven by the deletion or inactivation of key tumor suppressor genes TP53 and RB1. However, recent advancements in genomics and single-cell sequencing have identified distinct molecular subtypes of SCLC, derived from studies on cell lines, animal models, and tumor tissues. The tumor’s complexity, marked by the coexistence of multiple dynamic subtypes, contributes to its pronounced heterogeneity. Notably, different subpopulations exhibit a complex spatial relationship characterized by both mutual exclusion and coexistence. Temporally, SCLC exhibits the ability to undergo subtype transformations through various molecular mechanisms, underscoring the tumor’s plasticity and offering novel perspectives for personalized treatment approaches. This review synthesizes recent discoveries regarding SCLC subtype classification, intratumor heterogeneity, plasticity-related signaling pathways, immune landscape, and emerging therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/907fb7c9dba1e9795c5cd6f2373dce198db54231" target='_blank'>
              Intratumoral heterogeneity and potential treatment strategies in small cell lung cancer
              </a>
            </td>
          <td>
            Yunke Yang, Nana Liu, Jiaji Wu, Qingzhe Wu, Jiangnan Zhao, Yueli Shi, Mo Shen, Zhiyong Xu, Yueli Shi, Jianghao Yu, Peng Yi, Jin Cheng, Junhui Sun, Yun Xu, Kai Wang
          </td>
          <td>2025-10-27</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 Radiation therapy is a crucial component of the standard of care for newly diagnosed glioblastoma patients. However, the majority of tumors inevitably recur, most within the prior radiation field. Despite its broad clinical application, the molecular mechanisms that mediate response to radiation therapy in glioblastoma remain poorly understood. To address this knowledge gap, we systematically profiled the radiation sensitivity of four glioblastoma cell lines (T98G, SF295, LN18, DBTRG) and four patient-derived neurospheres (BT112, BT145, BT359, CPDM0095). The neurospheres were characterized at the genomic, transcriptomic and clinical levels. These models exhibited a spectrum of responses to radiation, with LD50s ranging from 1.5 to 8 Gy. To identify molecular factors that sensitize glioblastoma cells to radiation, we conducted a genome-wide CRISPR inactivation screen in T98G cells treated with or without a sub-lethal dose of radiation. Cells were transduced with the Inzolia multiplex CRISPR/Cas12a knockout library, targeting 19,687 single genes, 4,435 paralog pairs, 376 paralog triples, and 100 paralog quads. Cells were then divided into a control condition and an irradiated condition (3 Gy). After 18 days in culture, genomic DNA was harvested, and the relative abundance of each guide RNA array was compared between the two conditions. Targets preferentially depleted in the irradiated condition included known radiation sensitizers such as ATM and PRKDC, validating the screen’s performance. Gene set enrichment analysis (KEGG) of the top 200 hits revealed enrichment of DNA repair pathways, including homologous recombination and non-homologous end joining, as well as pathways related to the spliceosome, cell cycle, endocytosis, ubiquitin-mediated proteolysis, and pyrimidine metabolism. Notably, three components of the translesion synthesis pathway (REV1, REV3L, MAD2L2), not previously implicated in radiation response, also emerged as top candidates. A mini-pool library of the top 300 hits was subsequently generated and is being used for validation in additional glioblastoma cell lines and neurospheres. These findings provide a foundation for uncovering novel mechanisms of radiation response and nominate new targets for potential radiosensitization strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15dea4ab418adfb3586fa12497a41869a83e604c" target='_blank'>
              DNAR-13. Uncovering novel mediators of radiation sensitivity in glioblastoma using functional genomic screening
              </a>
            </td>
          <td>
            E. Aquilanti, Vincent Bozinov, Matthew L. Meyerson
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The treatment of cancer has taken a remarkable new direction with the advent of innovative scientific and technological methods. Immunotherapy has revolutionized oncology by harnessing the body's immune system to identify and destroy cancerous cells, leading to long-term remissions in many cancer types. Precision medicine enhances treatment effectiveness by tailoring therapeutic interventions to individual patients based on genetic, environmental, and lifestyle factors, thereby reducing both efficacy and adverse effects. The development of liquid biopsy methods enables non-invasive, real-time cancer detection and monitoring through blood samples (circulating tumor DNA and other biomarkers), allowing for treatment adjustments based on early cancer diagnosis. Recent advancements in CRISPR-Cas9 gene editing have opened new avenues for correcting oncogenic mutations and developing gene-based treatments. The use of artificial intelligence (AI) in cancer diagnostics, prognostics, and drug discovery is becoming increasingly common, enhancing prediction accuracy and optimizing predictive treatment planning. Combination therapies have shown synergistic potential to boost therapeutic effects and overcome drug resistance mechanisms. Meanwhile, cancer metabolism studies aim to exploit the metabolic changes in cancerous cells for therapeutic benefit. Epigenetic therapeutics seek to reverse abnormal genomic developmental control by restoring normal cellular epigenetic patterns and functions. Additionally, research on microbiomes is uncovering the crucial role of gut and tissue microbiota in cancer initiation, progression, and therapy response, suggesting potential new prevention and treatment strategies. These interdisciplinary innovations represent a paradigm shift towards more effective, personalized, and sustainable cancer care, offering new hope for improved patient survival and quality of life.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b52ad1f09c9728280095d0d63e09d01947a45160" target='_blank'>
              A Review: Recent Advancements in treatment of Cancer Research
              </a>
            </td>
          <td>
            Rahul Kumar, Khalid Raza, Kamna Gupta, Komal Gupta, Lukman Ansari, Sneha Kumari, Md. Danish Shahid Raja, Manjar Ansari, Faizal Nawaz, Raja, Articles Info, IJPHI26102810292025
          </td>
          <td>2025-10-31</td>
          <td>International Journal of Pharmaceutical and Healthcare Innovation</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Uncontrolled proliferation, resistance to apoptosis, inability to maintain genome integrity, and, recently, epigenetic reprogramming are all hallmarks of cancer. A number of gene expression and cell signaling networks control these-often-interconnected processes, while the study of their deregulation is in the forefront of cancer research for decades. Here we present data from cells and patients indicating that KMT2C, one of the most frequently mutated proteins in solid malignancies, is involved in all these processes. Its loss, a bad prognosis marker in bladder cancer, is associated with activation of the PI3K/PDK/AKT oncogenic/antiapoptotic axis, and tolerance to DNA damage during cell cycle progression. On the other hand, these cells suffer from mitotic stress that can be therapeutically exploited. Treatment with a PLK1 inhibitor showed high efficacy in vivo, and was associated with mitotic catastrophe and cellular senescence, providing evidence that targeting genes that promote mitotic progression could be a promising therapeutic approach in the subset of tumors with KMT2C loss.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a45d418115a05c2f3fb12ff53ed7255c53656e4" target='_blank'>
              KMT2C inactivation leads to PTEN downregulation and tolerance to DNA damage during cell cycle progression
              </a>
            </td>
          <td>
            T. Rampias, Andreas Goutas, Dimitris Karagiannis, Z. Kanaki, Antigoni Makri, L. Hoxhallari, Fotini Ε. Koukouzeli, Varvara Paraskevopoulou, Dimitra Tsouraki, Nikolaos Paschalidis, M. Avgeris, Andreas Scorilas, A. Klinakis
          </td>
          <td>2025-10-28</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="


 Acute myeloid leukemia (AML) progresses through the sequential accumulation of somatic mutations, with signaling transducer mutation typically preceding the acquisition of epigenetic alterations. However, recent single-cell DNA sequencing (scDNA-seq) studies have identified cases that do not follow this canonical mutational hierarchy. The prevalence of these non-classical mutation orders, and the biological mechanisms by which mutation timing drives phenotypic heterogeneity remain unclear.
 To address this, we analyzed publicly available scDNA-seq data from >150 newly diagnosed normal karyotype AML patients to reconstruct their evolutionary trajectories (Morita et al., 2020; Miles et al., 2020). We observed that 78% of the signaling mutation events—particularly FLT3-ITD, NRAS, and PTPN11—were terminal subclonal events in branch evolution. Surprisingly, 22% of the signaling mutations serve as initiating events, following a linear evolutionary trajectory and patients harboring these initiating signaling mutations exhibited significantly higher white blood cell and blast counts. These data suggest that the timing of signaling mutation acquisition is strongly linked to the resulting clonal evolution pattern, with FLT3-ITD emerging as the most prominent driver among all signaling events.
 To investigate the pathogenic mechanisms of initiating FLT3-ITD and the characteristics of the resultant AML, we therefore performed multimodal profiling (scDNA+protein-seq, scRNA-seq and patient-derived xenograft models) on 16 FLT3-ITD positive AML cases. Eight cases had FLT3-ITD as the initiating mutation (Initiating FLT3-ITD AML, IFA), and eight as subclonal events (Subclonal FLT3-ITD AML, SFA).
 Comparing tumor cell proportions identified by scRNA-seq with the mutant clone fractions from scDNA-seq in the same patient, we confirmed that leukemic transformation preceded acquisition of FLT3-ITD in SFA. Integrated genotype-immunophenotype analysis of paired samples—adjusted for co-mutations—revealed that initiating FLT3-ITD drove an expansion of hematopoietic stem and progenitor cells, whereas subclonal FLT3-ITD did not. Bayesian modeling further showed that initiating FLT3-ITD selectively upregulated leukemia stem cell (LSC) markers CD69 and CD123, resulting in a significantly larger CD123+CD69+ population (74% vs 35%, P=0.04). Consistently, in PDX models, IFA samples exhibited higher engraftment rates (5/8 vs 3/8). ScRNA-seq further revealed expansion of multipotent progenitor (MPP) and lymphoid-primed multipotent progenitor (LMPP) populations (29% vs 13%, P=0.04) and significant enrichment of LSC transcriptional signatures in IFA compared to SFA. Unsupervised clustering identified progenitor-enriched clusters in IFA characterized by downregulation of G2M checkpoint and E2F target pathways, indicative of a quiescent, stem-like state. Unexpectedly, canonical FLT3-ITD downstream pathways showed no significant differences between IFA, SFA, and FLT3-WT AML patients. Instead, in IFA, we observed significant upregulation of Hedgehog pathway genes (GLI2, ALDH1A1), known regulators of cell cycle and LSC quiescence.
 We observed frequent emergence of chr13q (FLT3 locus) uniparental disomy (UPD) in Initiating FLT3-ITD AML patients (15/23). Notably, we found the acquisition or expansion of UPD clones in IFA PDX models. Inferred from scRNA-seq, these UPD clones displayed increased proliferative capacity without losing LSC traits, potentially via RB1 inactivation. Drug sensitivity testing in PDX models revealed that IFA samples responded to chemotherapy and gilteritinib with distinct patterns, with gilteritinib selectively reducing leukemic blasts and inducing differentiation. However, single-cell genotyping of residual cells showed a marked rise in UPD clone post-gilteritinib. Longitudinal follow-up of these models confirmed that relapse was driven by the expansion of UPD clones. These data suggested chr13q is a required evolutionary event in the development and drug resistant progression of IFA.
 Our study demonstrates that initiating FLT3-ITD mutations reprograms leukemic cells to a quiescent, stem-like state, enforcing a linear evolutionary pattern, compared to subclonal FLT3-ITD. Clonal dominance is ensured by non-canonical FLT3 signaling and late-emerging chr13q UPD. These findings underscore the critical role of clonal timing in shaping AML biology and highlight new therapeutic vulnerabilities in initiating FLT3-ITD-driven AML.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c74975f6b634c7f7ca88b37dc7ce75122c37169" target='_blank'>
              Clonal timing of FLT3-ITD dictates evolutionary trajectory and phenotypic heterogeneity in acute myeloid leukemia
              </a>
            </td>
          <td>
            , Mengfang Xia, Ruoyang Shao, Qifa Liu, Qian-Fei Wang
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 Recurrent GBM poses significant treatment challenges due to limited therapeutic options and reliance on primary tumor molecular profiles, often overlooking treatment-induced molecular and phenotypic changes in recurrent tumors. This study investigates genomic, DNA methylation, and transcriptomic differences between a primary GBM (HF3016) and its matched recurrent tumor post-temozolomide and radiotherapy (HF3016R), using neurosphere cultures and patient-derived xenografts (PDXs) to assess radiotherapy (RT) response. Genomic DNA was analyzed via whole-genome sequencing, whole-exome sequencing, and EPIC DNA methylation arrays. Bulk RNA sequencing was performed on HF3016 and HF3016R cancer stem cell (CSC) cultures (n=4) and orthotopic PDX models (n=3). Differential gene expression was analyzed using NOISeq (q=0.95, fold change >2) on TMM-normalized data, with pathway enrichment via Metascape 3.1. Tumor growth and RT sensitivity were evaluated in NCRNU athymic nude mice implanted intracranially with HF3016 or HF3016R cells (3x10^5 cells/mouse, N=9-10 per group), receiving 5 Gy RT daily for 3 days or no treatment. Survival was assessed using Kaplan-Meier curves and Log-rank tests. No genomic driver alterations (CDK4, EGFR, MYC ecDNA amplification, PTEN, p53 mutations) or DNA methylation mesenchymal subgroup changes were observed between primary and recurrent models. However, a transcriptional shift from proneural to mesenchymal subtype occurred in recurrent models, with enriched epithelial-mesenchymal transition and interferon response pathways in HF3016R, while nervous system development pathways were enriched in HF3016. HF3016R PDXs exhibited faster growth (p=0.002) and greater RT resistance (p=0.0021 for HF3016, p=0.061 for HF3016R) compared to HF3016. These findings highlight the importance of considering treatment-induced molecular changes in recurrent GBM for optimizing therapeutic strategies. This study reveals that while genomic and DNA methylation profiles remain stable between primary and recurrent GBM models, significant transcriptional reprogramming occurs. The enrichment of epithelial-mesenchymal transition and interferon response pathways in recurrent models is associated with their aggressive growth and increased radiotherapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3aec6b57a9c052c7471268a4341bec91fcb4af93" target='_blank'>
              EXTH-73. Identifying therapeutic targets for recurrent glioblastoma using patient-derived longitudinal models
              </a>
            </td>
          <td>
            Indrani Datta, O. Nuga, A. Berezovsky, Laila M. Poisson, Ana deCarvalho
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d26c14dd48a6b62469af22170a5f8f8158260be3" target='_blank'>
              Non-cell-autonomous mechanisms of tumor initiation and relapse by chromosomal instability
              </a>
            </td>
          <td>
            Simona Lafirenze, Bastiaan van Gerwen, Ajit I. Quirindongo, Aniek Janssen, B. Etemad, Pim W. Toonen, W.H.M. Hoevenaar, Natasja Costermans, S. Youssef, A. de Bruin, L. Brosens, N. Jelluma, G. Kops
          </td>
          <td>2025-11-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Colorectal cancer (CRC) poses a significant global health challenge, yet immune checkpoint blockade (ICB) therapy benefits only a small subset of patients with mismatch repair-deficient (dMMR) or microsatellite instability-high (MSI-H) tumours. Through analyses of public single-cell and spatial transcriptomic datasets, primary mouse cell sorting and adoptive transfer experiments, flow cytometry, multiplex immunofluorescence, immunohistochemistry, and coimmunoprecipitation, we revealed that sentrin-specific protease 7 (SENP7) promotes regulatory B-cell (Breg) differentiation and inhibits senescence by activating the expression of the NAD-dependent protein deacetylase sirtuin-1 (SIRT1) via deSUMOylation, thereby enhancing the expression of genes such as interleukin-10 (IL-10). Notably, targeting SENP7 in B cells improved the antitumour efficacy of anti-PD-1 therapy. These findings suggest that inhibiting SENP7 may offer a promising strategy to sensitize immunologically “cold” tumours to immune checkpoint blockade.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddf8df97840e103aaf23b0df7a5e6b5ffd3a725b" target='_blank'>
              A SENP7-SIRT1-IL-10 Axis Driven by DeSUMOylation Promotes Breg Differentiation and Immune Evasion in Colorectal Cancer
              </a>
            </td>
          <td>
            Yuhan Liao, Xinghua Zhuo, Yuan Huang, Huimeng Xu, Zhe Hao, Lanhui Huang, Haoxuan Zheng, Jun Zhou
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Background:
 Chronic lymphocytic leukemia (CLL) is a heterogeneous B-cell malignancy characterized by clonal expansion and immune dysfunction. Although peripheral blood (PB) is frequently used for genomic and transcriptomic profiling, it may not fully capture the biological activity of the disease. The lymph node (LN) microenvironment plays a central role in promoting malignant cell survival, proliferation, and immune evasion. Genomic alterations such as TP53, NOTCH1, and SF3B1 mutations are associated with adverse outcomes, but it remains unclear how these mutations interact with the local immune contexts across compartments. We investigated how clonal architecture and immune composition differ between PB and LN, and how these features relate to evolutionary dynamics.
 Methods:
 We performed whole exome sequencing (WES) and bulk RNA sequencing on paired LN biopsies and CD19⁺-purified PB samples from 26 patients with CLL. We selected samples from patients with known CLL driver mutations. Somatic variants, copy number alterations, and clonal architecture were inferred using SuperFreq, and cancer cell fraction (CCF) was used to define clonal distribution. Variants were filtered to include only those with >3 alternate reads, >20 total reads, a variant allele frequency (VAF) >0.02, and classified as non-synonymous mutations. CCF was considered equivalent to VAF if >0.6, otherwise CCF was calculated as VAF * 2. CCF was adjusted by dividing CCF by the ancestral or founding clone CCF as a proxy for tumor cell fraction. Clones were considered compartmentalized if the absolute CCF difference between LN and PB exceeded 0.25. Exome alteration was calculated by summing the length of exonic targets falling in copy-number segments with an absolute log2 copy-ratio >0.2 and dividing by the total captured exome length. LN immune composition was estimated from bulk RNA-seq data using CIBERSORTx.
 Results:
 Of the 26 patients, 12 were treatment-naive and 18 were IGHV unmutated. In the PB, mutational frequency was as follows: 46.1% NOTCH1, 42.3% TP53, and 26.9% SF3B1. Oncoplot analysis revealed the presence of multi-hit alterations and a predominance of missense and frameshift variants.
 Analysis of paired PB and LN samples revealed notable heterogeneity in clonal architecture. Using SuperFreq, we identified between 2-6 clones per patient (median: 4). The median number of subclones did not differ between LN and PB (4 vs. 4), treatment naive vs. relapsed/refractory status (4 vs. 4) or between IGHV unmutated vs mutated (4.5 vs. 4). We evaluated genomic instability and found significantly higher exome alteration in patients who were IGHV unmutated (P = .03). Clonal compartmentalization (CCF difference of |.25|) was observed in 18 of 26 (69%) patients, with 5 patients showing clonal evolution in the LN (28%), 6 in the PB (33%), and 7 with distinct clones evolving in both LN and PB (39%).
 SF3B1 mutations were common in clones with CCF trending higher in the LN compared to PB (median ΔCCF 0.16, P = .11). Interestingly, CCF of NOTCH1 mutations was balanced across compartments (median ΔCCF = -0.02; P = .36), and CCFof TP53 mutations was numerically slightly lower in LN than PB (median ΔCCF –0.05; P = .97).
 LN samples with stable clonal populations had higher proportions of CD4⁺ naïve T cells compared to those with evolving clones (P = .02), but no other significant changes in T cell populations were noted. Patients with NOTCH1 or SF3B1 mutations had reduced dendritic cell abundance in LN (P = .04 and P = .06, respectively) compared to those with TP53 mutations or no known driver mutations.
 Conclusions:
 This study demonstrates distinct clonal and immune features between PB and LN compartments in CLL, a heterogeneous clonal architecture across compartments. Differences in clonal composition across LN sites within individual patients suggest that localized immune contexts play a role in clonal selection. Evolving LN clones were more common in LNs with relative decreases in CD4⁺ T cell subsets, and the presence of common driver mutations was associated with reductions in dendritic cell content. Together these observations suggest an interplay of supportive vs restrictive roles of the LN TME in CLL progression. These findings highlight the added biological insight provided by LN sampling and suggest that PB alone may not fully capture the clonal heterogeneity of CLL.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/121dea9d74142004efb29c8ec3326a61ca23f8a3" target='_blank'>
              Role of the immune microenvironment in the clonal evolution of chronic lymphocytic leukemia
              </a>
            </td>
          <td>
            Victoria Rogness, Bharath Dileep Kumar, Chingiz Underbayev, Pia Nierman, Clare Sun, Christine Gruessner, A. Wiestner
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bfbe39384668fd86cf5e7bd9896fdd269b9c106" target='_blank'>
              Peptide-Based Therapeutic Platforms for Addressing Drug Resistance in Breast Cancer
              </a>
            </td>
          <td>
            Shreyasi Meur, Dipanjan Karati
          </td>
          <td>2025-10-23</td>
          <td>International Journal of Peptide Research and Therapeutics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Single-cell sequencing (SCS) techniques have the potential to offer precise insights into cellular diversity by revealing unique genetic and transcriptomic profiles at the single-cell level. This advanced technology has been used extensively in research, but it has still not translated to clinical diagnostics, despite its potential. SCS provides more accurate granular information about heterogeneous cell populations and the creation of personalized treatment strategies. However, the integration of SCS into clinical practice is challenging. This review discusses the potential of SCS technologies in improving clinical molecular diagnostics in various clinical areas including oncology, genetics and rare diseases, infectious diseases, and autoimmune disorders and inflammation. We review recent advances, current uses, integration challenges, and their contribution to the development of these fields. SCS provides significant opportunities in oncology to analyze tumor heterogeneity and develop personalized treatments. In autoimmune and rare diseases, SCS has helped to define novel biomarkers and understand complex immune pathways. SCS has also been used to understand pathogen diversity and host-pathogen interactions in the context of infectious diseases, leading to targeted therapeutic approaches. Despite this progress, there remain challenges in data analysis, standardization, and routine clinical application. The future of SCS in clinical molecular diagnostics is promising. Further technological and research developments in SCS are expected to increase the precision and personalization of medical diagnostics and treatment. To overcome current limitations, interdisciplinary cooperation and innovative approaches to data analysis are needed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18f80640c729e34d2cf6d8aaba32d154d9a103af" target='_blank'>
              Single-cell sequencing in molecular diagnostics: Transformative yet untapped potential
              </a>
            </td>
          <td>
            D. Elshiekh, Nour Farchoukh, Reema El Hassan, Sara Al-Marzooqi, W. Aldosari, Haissam Abou-Saleh, Amal Al-Haidose, H. Zayed, Atiyeh M. Abdallah
          </td>
          <td>2025-11-19</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains a formidable clinical challenge due to therapy resistance, metastasis, and relapse. Central to these processes are colorectal cancer stem cells (CCSCs), a dynamic subpopulation endowed with self-renewal capacity, plasticity, and heterogeneity. This review synthesizes recent advancements in understanding how CCSCs orchestrate tumor progression through intricate bidirectional crosstalk with the tumor immune microenvironment (TIME). We begin by elucidating the cellular origins of CCSCs, their profound intratumoral heterogeneity, and their remarkable phenotypic plasticity—driven by genetic, epigenetic, and metabolic reprogramming—which collectively serve as the root cause of therapeutic failure. A significant portion of our discussion is dedicated to deconstructing the immunosuppressive niche co-opted by CCSCs. We detail mechanisms of immune evasion and tolerance, highlighting how CCSCs modulate innate and adaptive immune cells—including NK cells, Tregs, dendritic cells, macrophages, neutrophils, and myeloid-derived suppressor cells—to foster an environment that supports stemness and suppresses cytotoxic attack. This reciprocal interaction forms a vicious cycle that perpetuates tumor survival and progression. Finally, we critically evaluate emerging therapeutic strategies that concurrently target CCSC-specific vulnerabilities and counteract immunosuppression. We explore the limitations of conventional chemotherapy and the promise of targeted therapies (e.g., Wnt inhibitors), immunotherapies (e.g., CAR-T, bispecific antibodies), and combination regimens designed to remodel the TIME and eradicate the CCSC reservoir. By integrating insights from single-cell omics and spatial biology, this review provides a comprehensive framework for overcoming therapy resistance and proposes novel precision medicine approaches for CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81b2fada714432c2d56f0013d3edefb668b69269" target='_blank'>
              Colorectal cancer stem cells crosstalk in tumor immune microenvironment and targeted therapeutic strategies
              </a>
            </td>
          <td>
            Yuxuan Duan, Anshu Li, Daojia Miao, Diaoyi Tan, Keshan Wang, Jian Shi
          </td>
          <td>2025-11-19</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Age-related clonal hematopoiesis (CH) and clonal cytopenia of undetermined significance (CCUS) represent pre-leukemic conditions associated with predisposition to hematological malignancies, particularly myeloid neoplasms. Frequently associated with mutations in epigenetic regulators, such as DNMT3A, TET2, and ASXL1, deregulated epigenomes are implicated in the aberrant gene expression involved in the pathogenesis of CH and CCUS, whose molecular mechanism, however, has not been investigated. The major difficulty lies in the technical limitation to simultaneously analyze gene mutation, transcriptome, and epigenetics in a small fraction of mutant cells in comparison with wild-type cells within the same patients, particularly in those cases where mutant cell fractions are very small.
 Aim/Methods In this study, we developed a novel platform that enabled simultaneous measurements of gene mutations and chromatin accessibility at the single-cell level based on the modified Mission Bio Tapestri system, which was applied to the analysis of 11 samples with TET2-mutated (TET2-MUT) CH and CCUS to investigate the impact of TET2 mutations on chromatin accessibility. In a subset of the samples (n = 3), we performed 10x Chromium single-cell ATAC+RNA-seq (Multiome) experiments and then predicted single-cell TET2 genotypes by employing ATAC peak accessibility-based logistic regression models trained on the paired Tapestri multimodal data, enabling the analysis of effects of TET2 mutation on gene regulation and transcriptional outputs at single-cell resolution.
 Results First, our platform was validated using a mixed cell line sample (HL-60 and SKM-1). ATAC data alone clearly separated the two cell lines, and genotyping accuracy for cell line-discriminative SNPs reached 94-99%, confirming robust integration of ATAC-seq and genotyping at the single-cell level.
 Next, to investigate the consequences of TET2 mutation on hematopoiesis and chromatin accessibility, we applied the platform to a total of 37,190 cells from TET2-MUT CH/CCUS patients, achieving successful genotyping in 83% (74–92%) of the cells. TET2-MUT cells exhibited a differentiation bias toward myeloid and plasmacytoid dendritic cell (pDC) lineages, with reduced mature lymphoid output. In line with this skewed differentiation, TET2-MUT hematopoietic stem/progenitor cells (HSPCs) showed an increased accessibility to the motifs of transcription factors (TFs) associated with myeloid/pDC differentiation (e.g., CEBPE, RUNX1, and IKZF1), compared to wild-type (TET2-WT) HSPCs, whereas TET2-MUT B cells had a reduced accessibility to the motifs of B cell development-associated TFs (e.g., BACH2, IRF4, and IRF8), compared with TET2-WT counterparts. In erythroblasts, TET2-MUT cells exhibited a decreased accessibility to erythroid maturation-related genes and an increased accessibility to genes involved in cell proliferation, including E2F and MYC targets, compared with TET2-WT cells, suggesting dysregulated maturation of TET2-MUT erythroblasts. Collectively, these results revealed TET2 mutation-driven alterations in chromatin accessibility affecting lineage maturation.
 Next, to investigate combined effects of TET2 mutation on TF-mediated gene regulation and transcriptional programs, we analyzed 10x Multiome data with predicted single-cell TET2 genotypes. We first analyzed TET2-MUT cell fractions within monocyte subsets and observed a depletion of mutants in the CD16(+) subset, suggesting impaired maturation of TET2-MUT cells. To uncover the underlying mechanisms, we then compared TF activity between genotypes, using SCENIC+, which integrates paired single-cell ATAC-seq and RNA-seq data to infer TF activity based on both accessibility at TF-binding enhancers and expression of their target genes. For both accessibility- and expression-based TF activities, TET2-MUT monocytes exhibited up-regulation of cell cycle-regulating TFs (e.g., TFDP1, TFDP2, and MYBL2) and down-regulation of key monocyte maturation-related TFs (e.g., CEBPB, CEBPD, and SPI1). These results suggest that TET2 mutation may impair monocyte development by epigenetically and transcriptionally suppressing maturation-related programs and sustaining proliferative states.
 ConclusionOur innovative single-cell multi-omics approaches revealed that TET2 mutations rewire chromatin accessibility and TF regulatory networks, leading to dysregulated lineage maturation and aberrant cell cycle control in CH/CCUS.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8043e74c1da9433c5daf135387ded1955d981b6" target='_blank'>
              Single-cell multiomic profiling of gene mutation, chromatin accessibility, and gene expression in TET2-mutant clonal hematopoiesis
              </a>
            </td>
          <td>
            Masanori Motomura, Y. Ochi, Y. Nannya, N. Kakiuchi, N. Sezaki, S. Kasahara, Tomoe Nakagawa, Lanying Zhao, Shuichi Matsuda, M. Nakagawa, Seishi Ogawa
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/832132e602fe2a8fad05d9648f87894306e0e923" target='_blank'>
              Lineage-determining transcription factors constrain cohesin to drive multi-enhancer oncogene regulation
              </a>
            </td>
          <td>
            Yeqiao Zhou, Atishay Jay, Noah Burget, Tobias Friedrich, So-Yoon Yoon, Jessica Alsing, Guy Nir, Rudolf Grosschedl, Golnaz Vahedi, R. Faryabi
          </td>
          <td>2025-10-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is paradigmatic for therapeutic resistance driven by genetic heterogeneity, epigenetic plasticity and microenvironmental protection. Over the past decade, six targeted or pathway-directed small molecules—midostaurin, gilteritinib, quizartinib, ivosidenib, enasidenib, venetoclax and glasdegib—have changed frontline and relapsed/refractory (R/R) practice in genomically defined subgroups or in patients unfit for intensive chemotherapy. Yet primary refractoriness and early relapse remain common, frequently via adaptive rewiring of apoptotic dependencies, clonal evolution and differentiation resistance. Here we integrate mechanistic insights with clinical evidence to: (i) map resistance biology onto targetable nodes (apoptosis control; signalling kinases; chromatin/lineage programmes; RNA splicing; DNA-damage response; nuclear export; niche adhesion and innate immune evasion); (ii) summarise the clinical trajectory and current limits of approved and emerging small molecules (including menin and LSD1 inhibitors); (iii) propose rules for rational doublets and triplets that are biologically orthogonal yet clinically tolerable; (iv) outline a regulatory timeline for key AML small molecules; and (v) prioritise where drug development should go next, including next-generation BH3 toolkits, clonal-pressure-aware designs, minimal residual disease (MRD)–adapted trials and therapy guided by dynamic functional profiling. The review closes with cross-platform challenges—myelosuppression, infectious risk, resistance monitoring and trial design—and a pragmatic framework for moving beyond incrementalism toward durable control and cure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37f83b4532e166032da2b89eea2b4afa25aa7b8b" target='_blank'>
              Acute myeloid leukemia drug resistance: targetable nodes and the clinical trajectory of small-molecule inhibitors
              </a>
            </td>
          <td>
            Xixi Zhang, Yao Peng, Yina Tian, Shichao Chen, Yijiu Jia, Mengtong Liu, Li Zhang
          </td>
          <td>2025-11-19</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background: Acute myeloid leukemia (AML) is a heterogeneous blood cancer that arises from transformed myeloid precursor cells in a compromised bone marrow microenvironment. This environment is essential for AML initiation, progression, and relapse. Alongside oncogenic changes in hematopoietic cells, immunological dysregulation also contributes to leukemogenesis. The present study is aimed to identify prognostic genes in stromal and immune cells associated with AML using the weighted gene co-expression network analysis (WGCNA). Methods: Gene expression profiles were retrieved from The Cancer Genome Atlas database, and immune and stromal cell scores were calculated using the ESTIMATE (Estimation of STromal and Immune cells in MAlignant Tumor tissues using Expression data) method. These scores helped identify differentially expressed genes (DEGs), which were then used to create gene clusters through WGCNA. To explore the functions of genes linked to AML subtypes, Gene Ontology and Kyoto Encyclopedia of Genes and Genomes enrichment analyses were performed. A protein–protein interaction network was developed to identify hub genes. The top 18 hub genes were identified using the cytoHubba plug-in in Cytoscape software, and survival analysis was conducted with the Gene Expression Profiling Interactive Analysis 2 online tool. Results: A total of 1097 DEGs were identified, with 601 being upregulated and 496 downregulated. WGCNA analysis indicated that the gray module, comprising 165 genes, had the strongest association with AML subtypes (Cor > 0.3; P < .05). Gene Ontology enrichment analysis demonstrated that the 18 identified hub genes were predominantly associated with neutrophil activation, immune response, secretory granule membrane, and pattern recognition receptor activity. Kyoto Encyclopedia of Genes and Genomes pathway enrichment analysis revealed that the DEGs were mainly involved in pathways related to phagosome, lysosome, tuberculosis, leishmaniasis, and neutrophil extracellular trap formation. Kaplan–Meier survival analysis of the top 18 hub genes indicated that ITGAM, IL10, and CD163 were significantly correlated with survival outcomes in AML. Conclusion: Key stromal and immune-related genes influencing AML patient outcomes were identified, highlighting their potential as therapeutic targets. These discoveries provide deeper insights into the molecular mechanisms driving AML pathogenesis and subtype differentiation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a79f22e3d761d20e5f5e8f2956100649dbf9de1" target='_blank'>
              Exploring prognostic genes in the immune microenvironment of acute myeloid leukemia via weighted gene co-expression network analysis
              </a>
            </td>
          <td>
            Xiao Hu, Ying Chen, Huan Li, Shifeng Lou, Fengxia Bai
          </td>
          <td>2025-10-17</td>
          <td>Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a52f82067df3b6fe8b772cfcc0c82d13aa9a9b1" target='_blank'>
              Integrative transcriptomic analysis of non-small cell lung cancer: Microenvironment, circulation, and metastasis
              </a>
            </td>
          <td>
            Adnan Khosravi, Sharareh Seifi, Babak Salimi, Maryam Mabani, Parsa Rostami, Masoumeh Nomani
          </td>
          <td>2025-11-11</td>
          <td>Network Modeling Analysis in Health Informatics and Bioinformatics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa756c05989530f16b2f4c4778c0bfc833ec875f" target='_blank'>
              Deep Learning links TP53 genotype to expression-defined transcriptional program in Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            M. Tsesmelis, Rebecca Andersson, Alba Ferrer Perez, Sara Montserrat Vazquez, M. Carolina Florian, M. Mulaw
          </td>
          <td>2025-11-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Background HOXA5 (homeobox A5) exhibits context-dependent roles in cancer, but its pan-cancer spatial immune regulatory functions and therapeutic potential remain poorly understood. Methods We integrated multi-omics data from 33 cancer types (TCGA, n=11,096; GTEx, n=7,469; TISCH2; spatial transcriptomics) to characterize HOXA5 expression, genomic alterations, and immune interactions. Functional validation was performed in AML cell lines (U937, KG-1; n=3 biological replicates per experiment). Results HOXA5 was significantly dysregulated across cancers, with elevated expression in AML and GBM, and reduced expression in BRCA and LUAD. In AML, high HOXA5 expression predicted poor overall survival (HR = 2.80, 95% CI: 1.60–4.89, p < 0.001) and was associated with FLT3/NPM1 mutations. Spatial transcriptomics revealed HOXA5+ malignant cells enhance fibroblast/endothelial crosstalk via IGFBP3-TMEM219. HOXA5 knockdown suppressed proliferation (p < 0.01) and induced G0/G1 arrest. Mechanistically, HOXA5 maintained AML through cholesterol biosynthesis and ECM remodeling. Mercaptopurine was identified as a potential therapeutic agent, and molecular docking predicted a potential stable interaction with HOXA5. Conclusions HOXA5 plays a dual role in solid versus hematologic malignancies and serves as a key spatial immune regulator. It is a robust prognostic biomarker and therapeutic target in AML, with mercaptopurine representing a promising repurposing candidate.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbaf2f57507e416c0e787238787727a2c2564a0c" target='_blank'>
              HOXA5-mediated spatial remodeling of tumor-immune interfaces across cancers promotes AML pathogenesis
              </a>
            </td>
          <td>
            Changling Zhang, Ping Wen, Yan-yan Zeng, Tao Chen, Qulian Guo, Chunyan Liu, Fangfang Zhong
          </td>
          <td>2025-10-23</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 While venetoclax (VEN) combined with hypomethylating agents (HMA) has greatly advanced outcomes for acute myeloid leukemia (AML), resistance remains a major clinical challenge. Prior studies revealed several resistance mechanisms, including monocytic differentiation and RAS/TP53 mutations, but the full spectrum of mechanisms is still elusive.
 To assess the cellular mechanisms underlying VEN resistance, we performed an integrative analysis using bulk RNA sequencing (RNA-seq) and single-cell multi-omics in samples collected from the AML patients treated with 10-day decitabine and venetoclax (DAC10+VEN) trial (NCT03404193). First, we performed RNA-seq on 81 baseline and 10 relapse samples. K-means clustering of baseline samples identified a cluster enriched with non-responders, marked by a higher frequency of KRAS mutations (31%, p=0.012) and the absence of NPM1 and IDH1/2 mutations. Using LASSO logistic regression, we developed a 25-gene VEN-resistance (“VEN-R”) score comprising genes associated with MAPK, NF-κB, and apoptosis pathways. The VEN-R score significantly distinguished treatment response to DAC10+VEN (p<0.0001) and was also elevated in relapse samples (p=0.00018), supporting its association with resistance. We further validated VEN-R in two external, ex vivo datasets, finding significantly higher VEN-R scores in VEN-resistant samples compared to VEN-sensitive samples (BeatAML: p<0.0001 and Hashimoto et al. (Nature Cancer 2021): p=0.0002).
 To study the association between cell composition and DAC10+VEN response, we performed computational deconvolution of the RNA-seq data using CIBERSORTx. Baseline samples had higher fractions of erythroid (p=0.026) and CD8 effector (p=0.034) cells in nonresponders, while hematopoietic stem/progenitor (p=0.0006) and late erythroid (p=0.032) cells were more abundant in responders. Furthermore, compared with baseline samples, relapse samples had decreased hematopoietic stem (p=0.018), lymphoid primed multipotent progenitor (p=0.028), and naïve B (p=0.013) cells, and increased early erythroid (p=0.048), late erythroid (p=0.020), and granulocyte monocyte progenitor (p=0.036) cells.
 To assess the relationship between cell composition and VEN resistance at the cellular level, we performed single-cell RNA sequencing (scRNA-seq) on 22 samples (117,540 cells; 6 baseline, 16 relapse) treated with DAC10+VEN. scRNA-seq results corroborated the enrichment of mature cells at relapse; specifically, we identified the enrichment of erythroid, monocytic, and cDC (“EMD”) populations and the loss of progenitor cells. BCL2-expressing cells were depleted at relapse, whereas cells expressing MCL1, BCL2L1, or BCL2A1 were enriched in monocytes, erythroid cells, and cDCs, respectively. The EMD cells also had one of the highest expression levels of the VEN-R score at the single-cell level.
 To examine whether phenotypic remodeling is driven by genetic changes, we then performed single-cell DNA-antibody sequencing (DAb-seq) on 75 longitudinal samples, including six baseline-relapse pairs. DAb-seq analysis substantiated the findings from scRNA-seq that relapse is marked by the loss of stem/progenitor cells and the enrichment of erythroid/monocytic cells. Phenotypic remodeling occurred in five of the six pairs. One monocytic shift was accompanied by the de novo acquisition of FLT3 and KRAS mutations, one erythroid shift by expansion of an NRAS-mutant subclone, and a second erythroid shift by loss of a GATA2-mutant subclone. In the remaining two cases, remodeling occurred without any detectable genetic alterations. Integrating scRNA-seq and DAb-seq from 22 relapse samples revealed four phenotypic patterns: erythroid-predominant (6 cases), monocytic-predominant (4), mixed (erythroid±monocytic±cDC; 8), and inconclusive (4).
 Finally, we examined whether baseline phenotypic diversity predicts relapse after DAC10+VEN. In the DAb-seq cohort, a higher Shannon diversity index was associated with a higher risk of not only relapse (p=0.043) but also poor response (p=0.057).
 In summary, DAC10+VEN resistance is frequently observed with the enrichment of mature AML cells. To our knowledge, this is the first study to implicate erythroid cells and cDCs as major contributors to VEN resistance in patient samples, extending prior ex vivo studies (Kuusanmäki et al., Blood 2023) to a clinical setting. Monitoring these phenotypes and combining VEN with agents that target both immature and mature clones may improve patient outcomes.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9212f63e36e106f299197c47a52ccce0eb0f7523" target='_blank'>
              Phenotypic heterogeneity and remodeling drive resistance to venetoclax treatment in acute myeloid leukemia
              </a>
            </td>
          <td>
            Y. J. Kim, Ken Furudate, C. Dinardo, A. Maiti, M. Swaminathan, Zongrui Li, Eva Ordoñez, O. Nwogbo, Minggao Liang, Yoriko Saito, Akiko Kaneko, Kiyomi Morita, Jairo Matthews, Hongli Tang, Erika J. Thompson, Tomoyuki Tanaka, H. Uryu, N. Daver, N. Pemmaraju, T. Kadia, F. Ravandi, N. Short, M. Ohanian, G. Borthakur, Nitin Jain, G. Issa, L. Masarova, S. Kornblau, W. Wierda, S. Loghavi, Sa A. Wang, G. Garcia-Manero, Hagop Kantarjian, E. Jabbour, C. Bueso-Ramos, Shanshan Pei, C. Jordan, A. Futreal, Fumihiko Ishikawa, M. Konopleva, Koichi Takahashi
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>140</td>
        </tr>

        <tr id="Cancer treatment is a core challenge in clinical oncology. Despite the continuous advancement of chemotherapy, targeted therapy, and immunotherapy, the emergence of tumor drug resistance remains a key factor leading to treatment failure and worsening patient prognosis. Clinical data indicate that approximately 90% of treatment failures in patients with advanced cancer are directly related to drug resistance. Therefore, systematically analyzing the mechanisms of drug resistance and developing targeted strategies to address them are crucial for improving the efficacy of cancer treatment. This article systematically expounds on the main mechanisms of tumor drug resistance from the perspectives of tumor cell biological characteristics, tumor microenvironment regulation, and epigenetic modifications. These include reduced intracellular drug accumulation, mutations or aberrant expression of drug targets, inactivation of apoptotic pathways, cancer stem cell (CSC)-mediated resistance, and the regulatory role of immunosuppressive cells and cytokines in the tumor microenvironment. Furthermore, based on clinical practice, this article summarizes currently used strategies, such as drug development targeting resistance-associated molecules, chemotherapy regimen optimization and combination therapy, synergistic application of immunotherapy and conventional therapies, personalized treatment plan development, and the clinical translation of resistance monitoring technologies. Research has shown that the development of tumor drug resistance is the result of multiple factors and pathways, and a single strategy is unlikely to effectively reverse drug resistance. However, multi-target, multi-modal combination therapy based on resistance mechanisms, combined with dynamic resistance monitoring technology, can significantly improve treatment outcomes for patients with drug-resistant tumors. This article aims to provide a theoretical basis for clinicians to formulate tumor treatment plans and to guide future research related to drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f859514bae2da8d810219f47530fe8e0b51bb170" target='_blank'>
              A Review of the Mechanisms of Tumor Drug Resistance and Clinical Strategies
              </a>
            </td>
          <td>
            Wenqing Tie
          </td>
          <td>2025-11-21</td>
          <td>International Journal of Public Health and Medical Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7ab6bec4cbb03839af3fabebb27d4a6903a92d9" target='_blank'>
              Genome-wide characterization of clonal hematopoiesis reveals extensive non-coding putative driver mutations
              </a>
            </td>
          <td>
            J. Weinstock, K. Conneely, J. Woo, M. Arvanitis, M. Machiela, C. Russell
          </td>
          <td>2025-10-14</td>
          <td>None</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="


 Background - Methods: Hundreds of cell lines have been examined by genome-sale CRISPR knockout (KO) studies to identify functional drivers of tumors, but only few lines have been systematically examined with CRISPR activation (CRISPRa). We reasoned that CRISPRa studies can identify genes whose activation may suppress tumor cell fitness and serve as negative regulators of tumor cell survival/proliferation (NRSPs). To this end, we performed genome-scale CRISPRa studies (dCas9-P65-HSF transcriptional activation system; Calabrese sgRNA library, typically 6 sgRNAs/gene) in 10 genotypically diverse multiple myeloma (MM) cell lines at multiple timepoints and compared results with similar publicly available or in-house CRISPRa studies in other cancers (n=17; including lymphoma, leukemia, NSCLC, melanoma, ovarian, prostate cancer; with up to 3 lines per tumor type).
 Results: Based on multiple quantitative metrics, hundreds of genes were recurrently identified as NRSPs (e.g. 500 NRSPs in 5 or more lines). Integrated analyses of our CRISPRa data with molecular annotation for the respective MM lines did not identify any obvious bias in the CRISPRa-based detection of NRSP that relates to e.g., DNA copy number status; chromatin accessibility; or baseline levels of transcript expression. Top recurrent and pronounced NRSPs included several known proapoptotic genes (e.g., death receptors; caspases; BH3 only Bcl-2 family members) or other tumor suppressive molecules (e.g. CDK inhibitors), providing reassuring positive controls. Moreover, many previously unrecognized NRSPs were identified including transcription factors (TFs)/cofactors; chromatin remodeling genes and epigenetic regulators; signaling molecules regulating survival/proliferation; RNA binding proteins; nucleic acid-sensing and antiviral restriction factors; metabolic regulators; E3 ligases and other proteostatic regulators; DNA repair genes; solute carrier transporters, among others. No correlation is currently observed between the pattern of NRSPs for individual MM cell lines and their respective molecular subtype (e.g., t(4;14), t(14;16), t(11;14), etc). Some clusters of NRSPs are shared between MM and non-MM lines; and others are more prominent in MM, e.g. certain plasma cell-related TFs or endoplasmic reticulum regulators. We validated the functional relevance of several NRSPs with individual sgRNAs for CRISPRa; using focused sgRNA libraries for CRISPRa in MM cells in vitro and in NSG mice in vivo. We also performed single cell RNA-seq of MM cells transduced with pools of sgRNAs (CROP-seq) or bulk RNA-seq of MM cells transduced with individual sgRNAs for select NRSPs. Many NRSPs have undetectable/low transcript levels, without recurrent deletions/mutations, in MM cell lines or patient-derived MM cells but are expressed in healthy and malignant cells from different non-hematopoietic tissues. Thus, the NRSP effect of these genes in MM cells may involve lineage-inappropriate activation of molecular programs that are critical for, or at least tolerated in, other lineages but suppress MM cell fitness. Among the top 200 NRSPs with recurrent expression in MM cells, analyses of MMRF CoMMpass data revealed downregulation of ≥1 NRSPs in ~50% of paired MM samples after relapse vs. before treatment, suggesting that suppression of such NRSP genes can contribute, alone or in concert, to enhanced MM cell fitness in clinical relapses. Some NRSPs defined by CRISPRa are potent dependencies by CRISPR KO, including the MM-preferential, lineage-defining, TF PRDM1. CROP-Seq and bulk RNA-seq of MM cells with CRISPRa of PRDM1 (vs. other NRSPs or control sgRNAs) revealed that PRDM1 overexpression is toxic to MM cells by suppressing other dependencies, including IRF4, another lineage-defining TF and master MM dependency.
 Conclusions: Collectively,our studies have identified a large collection of NRSP genes whose overexpression suppresses tumor cell fitness in MM, with several of them also being relevant for other hematologic neoplasias. Several canonical tumor suppressor genes are also identified by CRISPRa, but most NRSPs have not been previously examined for their potential to suppress tumor cell fitness in MM or other neoplasias. Our study highlights the concept that engineered overexpression of NRSPs has intriguing potential for therapeutic applications in MM and other hematologic neoplasias.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28bf8e777a2bb3a092a1281fdd85652dfe41d721" target='_blank'>
              Negative regulators of survival and proliferation (NRSPs) identified by CRISPR activation studies in multiple myeloma (MM) vs. other neoplasias: Therapeutic implications
              </a>
            </td>
          <td>
            O. Dashevsky, Ricardo de Matos Simoes, Huihui Tang, Torsten Steinbrunn, S. Yamano, Eleni Anastasakou, Benjamin G. Barwick, S. Gandolfi, Ryosuke Shirasaki, Vikas Gupta, Michal Sheffer, Jonathan Licht, Lawrence Boise, Matthew Oser, Constantine S Mitsiades
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="


 Introduction: Subclonal diversity in multiple myeloma (MM)—defined by genetically distinct populations of malignant plasma cells—plays a pivotal role in disease progression, therapeutic resistance, and relapse. Characterizing subclone-specific transcriptomic profiles, copy number alterations, somatic variants, and their interactions with the immune microenvironment is critical for understanding tumor heterogeneity and identifying novel therapeutic targets. In this study, we analyzed smoldering multiple myeloma (SMM) and newly diagnosed multiple myeloma (NDMM) samples to explore the genomic landscape of subclones and their crosstalk with immune cells.
 Methods: CD138+ scRNA-seq and scATAC-seq multiomic data from SMM patients (n=10) and NDMM patients (n=21) were obtained from the published study dbGAP phs003220 (PMC11099140). Matched CD138- scRNA-seq data of immune cells were sourced from Sudha et al., IMS 2023. Subclones for each tumor sample were identified using inferCNV (v1.8.1) and a custom-built integration pipeline that utilized both scRNA-seq and scATAC-seq data. Cell-cell communications between subclones and immune cell types were identified using CellChat (v1.6.0) using the ligand-receptor interactions from CellChatDB. Copy Number Abnormality (CNA) profiles of subclones from single cell were compared with whole genome sequencing (WGS) data (phs003220) and similarity scores were computed. Due to differing technologies (RNA-seq vs WGS) and sequencing depth (single cell vs. bulk), this score reflects how well single-cell multiomics captures major chromosomal changes seen in WGS. Subclone-to-immune cell interactions were quantified to identify statistically significant chromosomal abnormalities that change their prevalence. Survival analysis was performed on the most common MM abnormalities to identify if subclonal co-occurrence influences prognosis. For each subclone, BAM files were generated, and somatic variants (SNVs + Indels) were called using Strelka2 with WGS control samples from the same patient.
 Results: We identified 23 subclones in SMM with a mean of 2.30 (min= 1; max=4); and 79 subclones in NDMM with a mean of 3.59 (min=1; max=8). Subclone and WGS CNAs were comparable to one another with mean Jaccard index (JI) of 0.57 for SMM and 0.69 for NDMM. Samples with low JI tended to have more subclones and more complex CNAs as measured through ploidy (correlation coefficient (r) = -0.069 in SMM and -0.177 in NDMM). NDMM subclones tended to have more CNAs than in SMM. From the IU cohort, Del13 was observed in 60% of the SMM and 45% of the NDMM patients. Gain1q was identified in 50% of the SMM and NDMM patients. Patients with co-occurring Gain1q and Del13q in the same subclone had significantly shorter progression-free survival (PFS) than patients with at least one of these alterations (median PFS 285 days vs 1194 days; P = 0.004). Subclones with Gain1q had a lower average percentage of interactions with CD8T cells (Gain1q=25.435, Norm1q=33.26; P=0.005). In NDMM subclones with Gain1q, there was a lower percentage of MHC-I pathway interactions with CD8T cells (P=0.001) and MHC-II pathway interactions with CD4T cells (P=0.001). Subclones with Del13q showed a higher percentage of MIF pathway (MIF-CD74/CXCR4) interactions in NDMM. SMM subclones with Del13p tended to have a lower percentage of MHC-I interactions in CD8T, CD4T and NK cells. Gain6p subclones had a higher percentage of MK pathway (MDK-NCL/ITGA4+ITGB1) interactions in SMM subclones. Somatic variants were identified for each subclone from the scATAC-seq. When compared with WGS somatic variants in NDMM, a median of 253 variants was found both subclones and WGS while 2752 were unique to subclones. Numerous pathogenic somatic variants were identified in the myeloma driver genes from subclones.Conclusions: The identification of subclone-specific immune interaction networks provides new opportunities for targeted therapeutic intervention. These results establish subclonal CNA profiling as both a prognostic tool and guide for precision immunotherapy in multiple myeloma, particularly for high-risk disease characterized by specific genomic alterations co-occurring in subclones. This study underscores the importance of single-cell analysis to fully capture the clinically relevant genomic heterogeneity that drives disease progression and treatment resistance.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be4387fcdd106b6a1f94375e3c3e641c2dbb1f9a" target='_blank'>
              Subclonal genomic heterogeneity shapes immune crosstalk and clinical outcomes in multiple myeloma
              </a>
            </td>
          <td>
            Parvathi Sudha, Sylvia Robertson, Sihong Li, Sean D. McCabe, Kelvin Lee, R. Abonour, Brian Walker, A. Suvannasankha, Travis Johnson
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="


 Older patients with acute myeloid leukemia (AML) have significantly worse response rates and overall survival compared to younger patients, even when matched for cytogenetic risk. This discrepancy has previously been explained by decreased tolerance of therapy and higher incidence of adverse risk disease. However, when these factors are taken into consideration there remains a higher incidence of primary refractory disease and relapse in older individuals, suggesting an underlying non-mutational resistance to therapy. Recent studies have implicated epigenomic disorganization from DNA damage repair as an important driver of cellular aging. The biological basis for age-related resistance and whether epigenomic disorganization contributes remains unclear.
 To investigate the impact of age on AML biology, we generated murine AML models by transducing hematopoietic stem and progenitor cells (HSPCs) from fetal (E13.5), young adult (8-10 weeks), and aged (>24 months) mice with retroviruses expressing MLL-ENL and NRasG12D. All models developed monocytic AML, confirmed by morphology and immunophenotyping, and were lethal upon transplantation into syngeneic recipients. Additionally, gene expression profile differences between young and old murine models were consistent with differences seen between younger and older AML patient samples (FDR q < 0.001) supporting their relevance to the human disease. We assessed in vitro sensitivity to menin inhibition, azacitidine, venetoclax, doxorubicin, and cytarabine. Whole exome sequencing (WES), bulk RNA-seq, single-cell RNA-seq (scRNA-seq), and proteomic profiling were performed to identify molecular correlates of resistance.
 AML cells derived from aged mice HSPCs were significantly more resistant to all tested therapies compared to fetal and young derived AML cells with marked resistance to cytarabine, doxorubicin and menin inhibition (p<0.001). WES did not reveal any recurrent mutations to explain this resistance. Transcriptomic and proteomic data were analyzed by KEGG pathway analysis revealing significant down regulation of several DNA damage repair pathways in older AML cells including base excision repair, homologous recombination and mismatch repair. In contrast, older AML cells had significantly increased expression of anti-apoptotic genes including multiple BCL2 family members. RNA and proteomics data demonstrated that younger AML cells had significantly increased expression of genes/proteins involved in chromatin and epigenome maintenance as identified by Gene Ontology terms (FDR q < 0.01) suggesting older AML cells may have a more disordered epigenome. Consistent with this, scRNA-seq revealed significantly increased transcriptional heterogeneity in aged AML cells compared to young and fetal models. This finding was corroborated in primary human AML samples when compared by age (p=0.0004). This suggests that AML cells derived from older HSPCs have a more disordered epigenome likely as the result of repeated rounds of DNA damage repair displacing epigenetic marks. To directly test if repeated rounds of DNA damage repair from a non-carcinogenic source would result in AML therapy resistance, we constitutively expressed the restriction endonuclease PPOI or GFP in fetal derived AML cells. After 12 weeks PPOI expressing fetal AML cells were significantly more resistant to cytarabine (p<0.0001) and doxorubicin (p<0.0001) compared to GFP expressing cells, mimicking the aged phenotype. They also appeared more aggressive when injected into syngeneic recipients.
 These findings support a unifying model of epigenetic based therapy resistance in AML, wherein aged HSPCs after multiple years of DNA damage repair, have a disordered epigenome and increased transcriptional heterogeneity. This heterogeneity is passed on to the resultant AML and enables survival of resistant subclones under therapeutic pressure, contributing to poor outcomes in older adults. This would also explain why therapy related and relapsed AML, where repeated DNA damage repair cycles have taken place, also have a worse outcome within the same cytogenetic risk category. Finally, it identifies a potential source of confounding in preclinical models that utilize enzymes that repeatedly induce DNA strand breaks like CAS9. Most importantly, it suggests that clinical trials data should be interpreted with extreme caution when considering patients outside the age range of those enrolled in the study.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/399dae455f6a2d1caf558c73e5130583d602ef95" target='_blank'>
              Age-associated non-mutational resistance in acute myeloid leukemia is driven by DNA damage-induced transcriptional heterogeneity
              </a>
            </td>
          <td>
            Jordon Burton, K. Pladna, Lance Miller, Birgit Schilling, Timothy Pardee
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Phenotypic plasticity plays a pivotal role in cancer, enabling tumor cells to adapt to environmental pressures and evade therapeutic interventions by transitioning between distinct cellular states. However, the contribution of phenotypic plasticity to adaptive drug resistance in glioblastoma (GBM), one of the most lethal of all cancers, remains poorly understood. In this study, we identify that GBM tumor-initiating cells resembling normal radial glia (RG), which occupy the apex of normal neurodevelopment, are driven by aberrant epidermal growth factor receptor (EGFR) signaling. Using a suite of patient-derived GBM models, we demonstrate through global proteomics and single-cell RNA sequencing that pharmacological inhibition of EGFR triggers a lineage transition toward neuronal and oligodendrocyte progenitor (OPC)-like states. This shift is accompanied by activation of oncogenic RAS-MAPK signaling – despite robust and durable inhibition of EGFR activation – and is further modulated by brain microenvironmental cues, including synaptic and calcium-mediated signaling programs. Dual inhibition of EGFR and RAS-MAPK with novel, tumor-selective small molecules blocks these phenotypic transitions and enhances GBM cell death in EGFR-mutated GBM models. To determine the subpopulation dynamics of RAS-MAPK signaling within GBM neurodevelopmental lineages, we develop DENALI (Dual-Expression Nuclear reporter of ERK Activity and Lineage Identity) – a novel, high-complexity barcoded lentiviral vector and integrative fluorescence reporter system. Using DENALI, we investigate the clonal mechanisms driving lineage plasticity in GBM following oncogenic EGFR inhibition and couple adaptive RAS signaling programs to the emergence of neuronal and OPC-like states under EGFRi therapy. Together, our findings establish neurodevelopmental lineage plasticity as a key driver of adaptive resistance in GBM and support dual-inhibition strategies to improve therapeutic outcomes in patients with GBM tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/099a2a811b34aed9bceff625fc38dcd958e60df8" target='_blank'>
              STEM-23. GBM NEURODEVELOPMENTAL LINEAGE PLASTICITY DRIVES RAPID ADAPTATION TO ONCOGENE TARGETED THERAPY
              </a>
            </td>
          <td>
            Dimitri Cadet, Toby Harris, Elisa Fazzari, Cassidy Andrasz, Nicholas A. Bayley, Thomas G. Graeber, T. Cloughesy, A. Bhaduri, D. Nathanson
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Metabolic reprogramming is a defining hallmark of cancer, enabling malignant cells to sustain growth while adapting to nutrient scarcity and immune surveillance. Among the diverse substrates that fuel these processes, amino acids occupy a central position due to their essential roles in bioenergetics, biosynthesis, and signaling. Cancer cells frequently develop “amino acid addiction,” a state of heightened dependency on specific amino acids such as glutamine, serine, glycine, tryptophan, arginine, and methionine. This rewiring not only supports proliferation and redox balance but also remodels the tumor microenvironment (TME) by impairing immune surveillance and promoting therapeutic resistance. Pharmacological inhibition of metabolic enzymes, enzyme deprivation strategies, and dietary interventions have shown efficacy in selectively disrupting amino acid–dependent pathways. Moreover, integrating amino acid–directed therapies with immunotherapy or chemotherapy holds potential to overcome resistance and enhance clinical outcomes. Despite these advances, challenges remain, including tumor heterogeneity, adaptive resistance mechanisms, and the need for patient stratification based on metabolic profiles. This review summarizes the molecular mechanisms underlying amino acid metabolic reprogramming, delineates its interactions with the tumor microenvironment, and discusses potential therapeutic approaches aimed at exploiting amino acid vulnerabilities in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70e5765744907e4988197e7d8dfc8db54daf4be6" target='_blank'>
              Metabolic dependencies on amino acids in cancer: connecting the tumor microenvironment and emerging therapeutic strategies
              </a>
            </td>
          <td>
            Talia Rossi, Silas Whittaker
          </td>
          <td>2025-10-15</td>
          <td>Clinical and translational reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Resistance to anti-cancer drugs remains a major challenge in chemotherapy. Combination therapy using sensitizers has emerged as a promising strategy to restore drug sensitivity in resistant cancer cells. However, experimental screening of sensitizers is laborious and costly, highlighting the need for computational methods that enable systematic and efficient prediction. We developed SOCAR, a network-based computational framework that predicts sensitizer drugs by integrating transcriptome profiles with molecular interaction networks. SOCAR identifies resistance-associated genes and network modules, and quantifies each drug's potential to reverse these resistance mechanisms. Applied to 4,009 drugs, SOCAR accurately predicted candidate sensitizers for tamoxifen-resistant breast cancer (AUROC = 0.90). In vitro assays validated that all twelve top-ranked candidates significantly reduced cell viability (p < 0.005) when co-administered with tamoxifen. Furthermore, protein activity analyses showed that resistance-module proteins were markedly altered after acquiring resistance but were restored to normal levels following combined treatment (p < 0.05). Collectively, SOCAR provides a systems-level framework for discovering novel sensitizers and elucidating mechanisms of resistance reversal.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab1acf5805cb5dd4152a1362e84cfaff70e9641d" target='_blank'>
              SOCAR: Network-based Computational Framework to Overcome Acquired Tamoxifen Resistance of MCF7 Cells.
              </a>
            </td>
          <td>
            Mijin Kwon, Young Min Woo, Woochang Hwang, Jong-Won Kim, Doheon Lee
          </td>
          <td>2025-12-05</td>
          <td>IEEE transactions on computational biology and bioinformatics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/460afa1e59ac1ebccf24338fe1d7768bc6e79f7a" target='_blank'>
              Splicing Factor Mutations in Chronic Myelomonocytic Leukemia: Biological Consequences and Therapeutic Implications
              </a>
            </td>
          <td>
            Nickolas Steinauer, Mirinal S Patnaik
          </td>
          <td>2025-10-25</td>
          <td>Current Hematologic Malignancy Reports</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="ABSTRACT Genomic studies of neurodevelopmental disorders (NDDs) have identified several relevant genomic variants. EBF3 is a gene with an excess of protein-coding de novo variants and underlies Hypotonia, Ataxia, and Delayed Development Syndrome. We previously identified noncoding de novo variants in an enhancer of EBF3 and further found enrichment of deletions of this enhancer in NDDs. In this study, we generated a novel mouse line that deletes the highly conserved, orthologous mouse region within the Rr169617 regulatory region, and characterized the molecular and phenotypic aspects of this mouse model. We found a deviation from Mendelian expectation (P=0.02) with significant depletion of the deletion allele (P=5.8×10−4). Rr169617+/− mice had a reduction of Ebf3 expression by 10% and Rr169617−/− mice had a reduction by 20%. Differential expression analyses in E12.5 forebrain, midbrain, and hindbrain in Rr169617+/+ versus Rr169617−/− mice identified dysregulated genes including histone and brain development related genes. A priori phenotyping analysis (open field, hole board and light/dark transition) identified sex-specific differences in mobility only for Rr169617−/− mice across multiple behavioral assays with Rr169617−/− males less mobile than Rr169617−/− females. Furthermore, both sexes when homozygous for the enhancer deletion displayed body composition differences when compared to wildtype mice. Overall, we show that deletion within Rr169617 reduces expression of Ebf3 and results in phenotypic outcomes consistent with potential sex specific behavioral differences.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3a7bc694ad47a81467f490ff2c604b9dd6ee34a" target='_blank'>
              Generation and characterization of a knockout mouse of an enhancer of EBF3
              </a>
            </td>
          <td>
            Emily Cordova Hurtado, Janine M Wotton, Alexander Gulka, Crystal Burke, Jeffrey K. Ng, Ibrahim Bah, Juana G. Manuel, Hillary Heins, Stephen A. Murray, D. Gorkin, Jacqueline K White, Kevin Peterson, Tychele N. Turner
          </td>
          <td>2025-10-13</td>
          <td>Biology Open</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="ABSTRACT Allostatic load (AL) is a measure of the body’s multi-systemic physiological dysregulation in response to chronic stress and life events. High AL has been associated with poor long-term health outcomes such as cardiovascular disease and mortality. DNA methylation (DNAm) is an epigenetic mechanism involving both genes and environmental factors and contributes to gene expression regulation. Hence, changes in AL can possibly be reflected in DNAm and gene expression differences and leveraging epigenetic and transcriptomic data together can help elucidate the underlying biological processes involved. To assess differential DNAm and gene expression between high and low AL in a cell-type specific manner, bulk DNAm and transcriptome signals from whole blood samples of 429 individuals from the Swiss Kidney Project On Genes in Hypertension (SKIPOGH) cohort were first deconvoluted into cell-type specific signals for six blood cell types using tensor composition analysis (TCA) and the software CIBERSORTx. For each cell type, DNAm associated with gene expression changes was then determined in high (N = 126) vs low (N = 303) AL groups. A total of 263 CpG-gene pairs were identified across all cell types, corresponding to 250 unique CpGs and 138 unique differentially methylated genes (DMGs). Several immune processes were enriched among downregulated genes of CD8 T and B cells, suggesting an impairment of the immune response, which is compatible with high AL. These findings highlight the importance of using cell-specific signals in DNAm and transcriptome analyses and may contribute to identify AL biomarkers and/or potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd140f40fed7634ccd27c98ff5d4ab45d13d0d05" target='_blank'>
              Integrative DNA methylation and transcriptome analysis reveal cell-type specific patterns in response to elevated allostatic load
              </a>
            </td>
          <td>
            O. Emery, C. Carmeli, S. Gonseth-Nusslé, Rp Juster, C. Kinnaer, D. Nanchen, S. Nusslé, S. Stringhini, Jd Chamberlain
          </td>
          <td>2025-10-29</td>
          <td>Epigenetics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 BackgroundAcute myeloid leukemia (AML) remains a lethal hematologic malignancy characterized by clonal expansion of immature myeloid blasts and dismal long-term survival. Despite advances in genomics-guided therapy, relapse and chemoresistance are common, especially in older or refractory patients. Emerging evidence implicates metabolic reprogramming and bone marrow (BM) hypoxia as key drivers of leukemic stem cell (LSC) persistence and drug resistance. We previously identified FGF11—a member of the intracellular FGF subfamily—as markedly upregulated in primary AML blasts and cell lines. However, its mechanistic role in AML pathogenesis and therapeutic vulnerability remains undefined.MethodsWe analyzed FGF11 and HIF-1α expression in 173 de novo AML patient samples (TCGA-LAML and GTEx), 15 paired diagnosis/remission BM aspirates, and two AML cell lines (HL-60, HEL) using RNA-seq, qPCR, and immunohistochemistry. Subcellular localization and co-localization of FGF11 and HIF-1α were assessed by immunofluorescence and proximity ligation assays. Functional studies employed siRNA-mediated knockdown, and rhFGF11 supplementation. Seahorse metabolic flux analysis quantified glycolytic and oxidative phosphorylation (OXPHOS) rates. ResultsCompared with normal bone marrow, FGF11 mRNA was 4.3-fold higher in primary AML blasts (p<0.01) and predicted shorter event-free survival (EFS; HR=1.54, p=0.0028). Bioinformatic interrogation of the FGF11 promoter revealed hypomethylation at the CpG site cg00795277 that was tightly linked to adverse clinical outcomes, suggesting epigenetic activation of the locus.In HL-60 cells, immunofluorescence showed nuclear co-localization of FGF11 with HIF-1α, providing a spatial basis for functional interaction. Functional assays demonstrated that siRNA-mediated knockdown of FGF11 attenuated proliferation, enhanced apoptosis, and curtailed migration and invasion. Transcriptomic profiling of these cells disclosed down-regulation of HIF-1α signaling, alongside re-activation of mitochondrial and oxidative-phosphorylation programs, implying that FGF11 represses oxidative metabolism.To explore the reciprocal relationship, we exposed AML cells to CoCl2-induced hypoxia and observed a time-dependent increase in both FGF11 and HIF-1α mRNA and protein, confirming that HIF-1α can transcriptionally up-regulate FGF11. Proteomic analysis of hypoxic versus normoxic cells, annotated by COG classification, further enriched energy-metabolism pathway.Finally, exogenous rhFGF11 not only recapitulated the aggressive phenotype—enhancing migration and invasion—but also triggered a metabolic shift from oxidative phosphorylation to glycolysis. ConclusionsOur data uncover a novel FGF11–HIF-1α epigenetic-metabolic axis that orchestrates AML progression and chemoresistance through glycolytic reprogramming. Targeting this loop—either by disrupting FGF11/HIF-1α interaction or reversing metabolic addiction—represents a promising therapeutic strategy to overcome residual disease and improve outcomes in high-risk AML.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a40abdc907bf009e1fd38544864df761a97e31a" target='_blank'>
              FGF11–HIF-1α glycolytic epigenetic axis drives AML progression and chemoresistance via metabolic reprogramming
              </a>
            </td>
          <td>
            Hang Lv, Jiayao He, Zhuang Xin, Songfu Jiang, Xiaoying Man, Ruochen Wang, Rong Yang, Xiaoyu Zeng, Honglan Qian, Yifen Shi
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancer cells are distinguished from normal cells by their rapid rate of division. This high division rate can be explained by various factors, including unregulated cell cycle progression, which occurs when cancer cells bypass checkpoints; activation of growth signals in cancer cells; and override of growth suppressor factors, which generally regulate cell growth and division within physiological thresholds. Cancer cells bypass these controls due to genetic mutations or epigenetic changes. Additionally, changes in the tumor microenvironment (TME) can contribute to the accelerated growth and division of cancer cells. It is well established that cancer cells accumulate mutations more frequently than normal cells. This accumulation can, in part, explain their rapid rate of division. The cell cycle consists of four phases: G1, S, G2, and mitosis, with the first three phases categorized as interphase. When cancer cells experience division stress, their demand for nucleotide synthesis increases, which is essential for RNA and DNA synthesis. RNA synthesis primarily occurs during interphase, while DNA replication occurs in the S phase. One of the conserved enzymes involved in nucleotide synthesis is dihydrofolate reductase (DHFR). This enzyme’s role in purine and thymidylate synthesis is crucial in cancerous cells under conditions of division stress. Methotrexate, a well-known DHFR inhibitor, has been introduced for the treatment of cancers such as meningeal leukemia, lymphoma, and breast cancer. While effective in alleviating cancer symptoms, methotrexate can cause adverse effects, including hepatotoxicity, pulmonary complications, and renal impairment. Based on these considerations, we applied combinatorial studies, including molecular dynamics simulations alongside quantum mechanics, to design novel DHFR inhibitors for cancer cells using carbohydrate- and amino acid-based scaffolds. Additionally, our studies suggest that the designed inhibitors may exhibit fewer side effects than methotrexate.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/936636e4b144e2a81ac99e1bbcb66eabfd38e1da" target='_blank'>
              Combinatorial QM and MD in silico design of natural product-based DHFR inhibitors
              </a>
            </td>
          <td>
            Sepideh Kalhor, Amin Mohammad Shapouri, Alireza Fattahi
          </td>
          <td>2025-10-27</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract The incidence of colorectal cancer in young adults (age of diagnosis <50 years) has been rapidly increasing. Although ∼20% of early-onset colorectal cancer (EOCRC) cases are due to germline mutations, the etiology of the majority of EOCRC cases remains poorly understood. Nongenetic factors such as environmental exposure and lifestyle changes are likely to have a direct link to the increased incidence of sporadic EOCRC. We hypothesize that such factors may be observable as alterations in the epigenome, microbiome, and immunome. We characterized the DNA methylation (DNAm) signature and measured DNAm age in EOCRC by using The Cancer Genome Atlas (TCGA). Furthermore, we carefully identified intratumoral microbes from TCGA and the Oncology Research Information Exchange Network datasets and then related the microbes to deconvolved immune cell abundances in EOCRC. We observed that the DNAm age in the EOCRC cohort was 12 years older when compared with the average-onset colorectal cancer (AOCRC) cohort, using three different epigenetic clocks. Differentially methylated sites associated with gene expression include cAMP-responsive element binding protein signaling in neurons, G protein–coupled receptor signaling, phagosome formation, and S100 family signaling. These differences were validated in the gene expression data from TCGA and the Oncology Research Information Exchange Network. When comparing the intratumoral microbes between EOCRC and AOCRC, no consistent differences were observed. Interestingly, the most abundant microbes interacted with the immune systems differently between the EOCRC and AOCRC tumors, characterized by more and larger positive correlations in EOCRC. These data suggest that epigenetic modulation and accelerated aging may play a key role in the development of EOCRC. Significance: We investigated whether environmentally driven factors contribute to EOCRC. We observed accelerated epigenetic aging in EOCRC and epigenetic changes associated with chronic inflammation. Tumor immune cell abundances correlated more strongly with microbes in EOCRC than in AOCRC. These data suggest a dysregulation of the immune response in EOCRC, driving chronic inflammation and tissue aging.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45c12561621c15cb4ece79f124013162ea3c517b" target='_blank'>
              Epigenetic Modulation, Intratumoral Microbiome, and Immunity in Early-Onset Colorectal Cancer
              </a>
            </td>
          <td>
            N. Jin, R. Hoyd, Ayse S Yilmaz, Jiangjiang Zhu, Yunzhou Liu, Malvenderjit S Jagjit Singh, Dennis J. Grencewicz, Xiaokui Mo, Matthew F Kalady, Daniel W Rosenberg, C. Dravillas, Eric A Singer, J. Carpten, Carlos H F Chan, M. Churchman, N. Denko, Frances Di Clemente, Rebecca D. Dodd, Islam Eljilany, Naomi Fei, S. Hardikar, Alexandra P Ikeguchi, A. Ma, Qin Ma, Martin D McCarter, Afaf E G Osman, Gregory Riedlinger, Lary A. Robinson, Bryan P. Schneider, Ahmad A. Tarhini, G. Tinoco, Jane C Figueiredo, Yousef Zakharia, C. Ulrich, A. Tan, Daniel J. Spakowicz
          </td>
          <td>2025-10-24</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/323b823debf44456586f41e4e06b190055e610d9" target='_blank'>
              Genetic Ablation and Multi-Omics Profiling Reveal CEP55 as a Key Driver of Tumorigenesis in Diverse Cancer Models
              </a>
            </td>
          <td>
            Behnam Rashidieh, Yuanhao Yang, A. Bain, Phillip Theron, Catharina Vosloo, Parinaz Ahangar, Brydie Bowden, S. Tria, Ben B. Wang, Sandra Isenmann, Nicholas J. Westra van Holthe, Zherui Xiong, Hani Vu, Sowmya Sharma, A. Sengal, J. A. López, John Finnie, Antonella Papa, Pascal H. G. Duijf, Quan H. Nguyen, Pirjo M. Apaja, K. Khanna
          </td>
          <td>2025-10-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="Simple Summary Ewing sarcoma (EwS) is a highly malignant tumor of bone and soft tissue that predominantly affects children and young adults, with a high propensity for early metastasis to the lungs and bones. Large-scale sequencing and SNP array studies have concluded that, in addition to the typical EWSR1::ETS translocation, the genome of EwS is remarkably simple at the single-nucleotide level, but recurrent structural chromosome alterations are frequent. The current findings indicate that translocation-mediated EWSR1 haplo-insufficiency results in replication stress and potential loss of heterozygosity and emphasize that copy number dysregulation acts as a critical secondary driver of EwS biology, shaping tumor evolution and influencing clinical outcome. The preclinical results directed at replication stress and diminished repair capacity of EwS are promising and should significantly improve the therapy of EwS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/593ddbf60a24ebf76d603d5b1ea4998334fd3f0e" target='_blank'>
              The Puzzle of Genetic Stability and Chromosomal Copy Number Alterations for the Therapy of Ewing Sarcoma
              </a>
            </td>
          <td>
            Günther H. S. Richter, A. Ranft, Maximilian Kerkhoff, Marvin Jens, Ina E. Kirchberg, U. Dirksen
          </td>
          <td>2025-11-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="
 IDH-mutant gliomas are typically slow growing on initial presentation, however selective pressure during conventional upfront treatment with genotoxic radiation and chemotherapy can fuel the evolution of sub-populations of tumor cells. These subclones promote recurrence of IDH-mutant gliomas that are less likely to benefit from standard treatments. Therefore, novel tumor-specific therapeutic strategies with non-overlapping mechanisms are needed for durable tumor control. IDH mutant cancer cells are known to reprogram their metabolism to sustain rapid proliferation, which generate potentially targetable vulnerabilities in canonical metabolic pathways. Using the patient-derived, WHO grade 3 oligodendroglioma line TS603, we screened a small molecule library of 240 metabolism-related compounds, including both FDA-approved and non- approved drugs, over a range of 10 concentrations (1nM-20µM). Drug efficacy was assessed after 5-day incubation via cell viability assay (Cell-Titer Glo). We identified 53 compounds with IC50<2µM, including drugs that target previously reported IDH-mutant glioma targets, like NAMPT and DHODH. Notable target pathways include folate metabolism and de novo thymidylate synthesis (methotrexate, raltitrexed, 5-FU), p97 AAA ATPase/VCP chaperone (CB-5083, ML240, eeyarestatin I), mevalonate pathway (Ro 48-8071 fumarate, atorvastatin calcium), and the BCL-2 family (Navitoclax, ABT-737). We are validating these drugs across a panel of patient-derived IDH-mutant gliomasphere models, and examining the combinatorial effects of these agents with vorasidenib and CDK4/6 inhibitors. Our results provide insight into important metabolic pathway dependencies in recurrent IDH1-R132H mutant gliomas that have targeting potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99af98af377416756584e34e3b8b2129cefb73a2" target='_blank'>
              DDDR-40. Identifying metabolic vulnerabilities of recurrent IDH1-R132H mutant glioma by high throughput screening
              </a>
            </td>
          <td>
            Yutong Wang, Sayali Daware, D. Cahill, Julie Miller
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 Glioblastoma has a complex regulatory landscape that drives its cellular heterogeneity, plasticity, and resistance to therapy—features not explained by genetics alone. Chromatin profiling is essential for uncovering the epigenetic mechanisms underlying these behaviors and for identifying potential vulnerabilities that could be targeted therapeutically. Combinations of histone marks govern the activity of their associated DNA, including functional genomic elements such as enhancers or promoters. Single-cell technologies have only recently been adapted to the study of epigenetic cell states and significant technical challenges have limited their widespread adoption, especially in complex tissues. Here, we employ our recently optimized method for performing simultaneous single-cell epigenetic and transcriptomic analyses in multiple frozen glioblastoma (GBM) samples. Our objective was to implement our recently optimized single cell multi-omic sequencing method (“mscCUT&TAG”) and integrate with additional single cell modalities to evaluate epigenetic heterogeneity in primary glioblastoma samples. Glioblastoma (IDH WT, MGMT unmethylated) samples were collected from primary surgical resection in two patients. We applied our previously developed multi-omic single cell cleavage-under-target and tagmentation (“mscCUT&TAG”), 10X single-cell RNA-seq, 10X Multiome (RNA+ATAC-seq) and whole transcriptome Spatial Enhanced Resolution Omics-sequencing (Stereo-seq) to these samples. We comprehensively characterized 72,000 cells at single-cell resolution by integrating joint chromatin accessibility, transcriptional profiles, and five histone modification chromatin states associated with active (H3K27ac, H3K4me3), poised (H3K4me1) and repressive marks (H3K27me3, H3K9me3), We then combined this data with whole transcriptome Stereo-seq of these tumors, and generated a spatially resolved blueprint of epigenomic regulation in GBM. We identified specific tumor cell populations, cell-type specific coordinated epigenetic and transcriptional changes, and defined functional regulatory elements.We performed a comprehensive analysis of epigenetic heterogeneity in GBM at single-cell and spatially resolved resolution. We anticipate that further extension of this work will provide a greater understanding of fundamental biology and therapeutic avenues in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27612dab26c7b403075cd8f3dfbd065d9089efda" target='_blank'>
              EPCO-48. DECIPHERING THE EPIGENETIC LANDSCAPE OF GLIOBLASTOMA THROUGH MULTIOMIC SINGLE-CELL AND SPATIAL PROFILING
              </a>
            </td>
          <td>
            Andrew Hardigan, E. Blandford, Aditya Mohan, P. Fecci, A. Patel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Background and significance. Stemness in acute myeloid leukemia (AML) is determined by a clonal hierarchy with rare, phenotypically defined leukemia-initiating cell (LIC) at its apex (Lapidot 1994, Bonnet 1997). Given that LICs in AML are drug-resistant and initiate relapse of refractory disease, the identification of an LIC population in B-ALL would be consequential. However, unlike AML, B-ALL does not evolve from a clonal hierarchy and LIC populations have remained elusive in B-ALL (Kelly 2007, Le Viseur 2008, Rehe 2013).
 Results. Time-lapse studies of single patient-derived B-ALL cells revealed that most B-ALL cells were continuously proliferating (steady), while subpopulations underwent periodic transitions between quiescent and proliferative cell states (alternating). Gene expression studies of 'steady’ and 'alternating’ cells revealed a dominant and reciprocally repressive transcriptional program driven by MYC in 'steady’ and BCL6 in 'alternating’ B-ALL cells. To dissect how MYC and BCL6 dynamics govern transitions between proliferative and quiescent cell states in B-ALL, we engineered patient-derived B-ALL (PDX) by CRISPR and HDRT with dual-reporter knockin alleles expressing C-terminal MYC–mNeonGreen and BCL6–mScarletI fusion proteins. Using time-lapse imaging, we tracked MYC and BCL6 protein expression in single B-ALL cells over 20 hours. Matching their proliferative characteristics, 'steady’ B-ALL cells expressed MYC and no detectable BCL6. However, B-ALL PDX also included 'alternating’ cells that underwent multiple transitions between MYC+ BCL6- and MYC- BCL6+ cell states. PDX from BCR-ABL1 B-ALL included approximately 30% 'alternating’ cells, whereas 'alternating’ cells exceeded 50% in B-ALL PDX with RAS-pathway lesions. Among 'alternating’ cells, transitions occurred independently from cell divisions (every 36 hours) with shorter MYC-phases of three hours and longer BCL6-phases of six hours.
 Integrated ChIP-seq, RNA-seq, and untargeted metabolomics revealed that MYC-high cells are much larger, activated glycolysis and protein-synthesis pathways, whereas BCL6-high cells are smaller and enriched for phosphatidylethanolamine (PtdEtn) synthesis pathways. Consistent with the essential role of PtdEtn in autophagosome biogenesis, BCL6+ B-ALL cells exhibited substantially increased autophagy (LC3B flux). To directly compare MYC and BCL6 dynamics with cell growth trajectories, we combined quantitative phase microscopy and time-lapse fluorescence imaging to measure single-cell dry mass over 8 hours. Cells in the MYC-high state accumulated biomass at twice the rate of BCL6-high cells, producing a step-by-step progression of fast (MYC) and slow (BCL6) cell mass accumulation that aligned with each MYC/BCL6 state transition. Finally, multiplex immunofluorescence of FFPE biopsies from patient samples revealed spatially segregated niches of MYC-high and BCL6-high leukemia cells reminiscent of partitioning of MYC+ centrocyte and BCL6+ centroblasts in germinal centers.
 To experimentally induce cell state-transitions, we engineered patient-derived B-ALL cells by HDRT with dual-reporter knockin alleles expressing C-terminal MYC–dTAG fusion proteins. In addition to the MYC-dTAG degron system, we used the BCL6 PROTAC degrader ARV-393 for acute ablation of either MYC or BCL6, which was achieved within 90 minutes. Acute dTAG-mediated degradation of MYC induced cell shrinkage and decreased cellular dry mass, whereas BCL6 degradation via PROTAC decreased autophagic flux and increased dry mass, confirming opposing roles in anabolic versus catabolic metabolism.
 To functionally study 'steady’ (MYC-only) and 'alternating’ (MYC/BCL6) B-ALL populations, we developed a cell-sorting strategy that enriched each of the two populations to a purity of 85% as confirmed by subsequent time-lapse imaging. Extreme limiting dilution and serial transplant experiments revealed that 'steady’ (MYC-only) largely lacked LIC-capacity. In contrast, 'alternating’ (MYC/BCL6) B-ALL cells were highly enriched for LIC and initiated fatal leukemia after short latency.
 Conclusion: This study uncovers previously unrecognized cell state transitions in B-ALL that are controlled by MYC- and BCL6-dependent transcriptional programs. Alternating cycles of quiescent and proliferative cell states balance anabolic and catabolic metabolism, promote drug resistance, and enhance leukemia-initiating potential.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21ac412fe67d4caf3c95440feaedce483b7297e3" target='_blank'>
              Alternating cycles of quiescent and proliferative cell states determine stemness and leukemia-initiation capacity in acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Zhangliang Cheng, Kohei Kume, Ruoyi Shi, Mark Robinson, Kadriye N Cosgun, Yujin Bao, Siyi Chen, Pranav Mishra, Joerg Bewersdorf, Mina Xu, Markus Müschen
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="


 Myeloproliferative neoplasms (MPNs), including chronic myeloid leukemia (CML), are oncogene-driven blood cancers arising from hematopoietic stem cells (HSCs) that often progress from a ‘smouldering’ chronic phase to a drug-resistant, pre-terminal blast phase. In CML, recent studies have shown that stem and progenitor cells (SPCs) from chronic phase (CP) patients at high risk of blast crisis (BC) transformation exhibit a convergent gene expression signature that mirrors that of overt BC, independent of underlying somatic mutations (Ko et al., Blood 2020; Krishnan et al., Blood 2023). Collectively, these findings suggest the existence of a common drug-resistant SPC state. Identifying the therapeutic vulnerabilities of such states holds promise for transforming the management of high-risk MPNs.
 To identify these resistant states, we performed single-cell (SC) multi-omic profiling (scRNA-seq, scATAC-seq, and targeted mutational analysis) on pre-treatment bone marrow (BM) mononuclear cells from (n=24, CML) three clinical groups (optimal imatinib [IM] responders, suboptimal IM responders, IM failure with BC progression; n=5/5/6), normal BM (n=4), and BC samples (myeloid n=2, lymphoid n=2). scATAC-seq data annotated 28 cell types from 240,456 cells; we identified DARs with DESeq2, inferred TF activity with ChromVAR, integrated gene-to-peak and enrichment analyses, mapped epigenetic remodeller occupancy with HOMER, and built Gene Regulatory Networks (GRNs) using SCENIC+ algorithm.
 Across all prognostic groups, analysis of LSCs revealed a shared, CML-specific epigenetic state distinct from normal BM HSCs, characterized by 29,317 differentially accessible regions (DARs) among 421,985 peaks (6.95%), with ~75% located in intronic and intergenic regions. Consistent with the drug-resistant phenotype of LSCs, functional annotation of these DARs highlighted hallmark features of resistance, including increased inflammatory and proliferative signaling and diminished stemness. Importantly, these LSC states were closely linked to canonical BCR::ABL1 signaling pathways, with enrichment of AP-1, STAT1/3/5A/5B, and NF-κB TF binding activity, implicating these pathways in the establishment of the LSC state. Strikingly, among the top 10 CML-specific transcription factors driving the LSC program, only STAT binding activity correlated with increasing clinical resistance, specifically through enhanced IL2-STAT5 and TNFα signaling.
 Given the relative stability of the LSC state across prognostic groups, we next examined the progenitor population for cell state changes. Unlike LSCs, nearly 32% of the progenitor epigenome showed altered chromatin accessibility with BC progression more extensive in lymphoid BC (LBC) (94,245 DARs) than myeloid BC (MBC) (43,378 DARs). In CP patients who developed LBC, 29,167 LBC-specific DARs were enriched in lymphoid progenitors at diagnosis. Whereas MBC-associated DARs (1,278–5,569) were weakly detected across myeloid subsets in CP patients who developed MBC. These findings support the notion of a common epigenetically-primed state in diverse progenitor compartments, predisposing them to subsequent transformation. Subtype-specific TF analysis showed AP-1/CEBP enrichment in MBC, and IRF8/9 in LBC. GRN reconstruction from scRNA-seq showed these TFs drive resistance associated programs characteristic of full-blown BC (Ko et al. Blood 2020). Ongoing GWAS-linked analysis seeks germline or acquired variants tied to these primed states.
 Lastly, we identified shared upregulation of STAT and IRF pathways in both BC subtypes, TFs downstream of JAK1/2/3, indicating convergent regulatory programs. This led us to testt, an FDA-approved JAK inhibitor, to dismantle these cell states. Although tofacitinib alone showed minimal cytotoxicity, it markedly sensitized BC cells (n=4) to TKIs (~90% killing) while sparing normal progenitors (~25% killing). Thus, mapping the CML epigenetic landscape identified core transformation regulators and revealed actionable therapeutic opportunities.
 This study presents the first scATAC atlas of CML resistance and progression, revealing that: (1) Epigenetic reprogramming in CP precedes BC transformation; (2) Distinct TF networks drive BC subtypes; and (3) JAK inhibition is a novel strategy to dismantle JAK/STAT-IRF activation in high-risk SPCs. Our findings highlight the power of SC- epigenomics to identify actionable vulnerabilities and guide pre-emptive therapies for high-risk MPN patients.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e0b2fbf0807bd6dc58e419db5b166831585fd5e" target='_blank'>
              Single-cell chromatin profiling reveals early epigenetic reprogramming, subtype-specific regulatory networks, and JAK/STAT–IRF vulnerabilities in high-risk chronic myeloid leukemia
              </a>
            </td>
          <td>
            S. T. Ong, Pavithra Shyamsunder, Beijun Chen, Zahid Nawaz, P. Venkatesh, Xinyi Yang, Yiyang Lim, Charles Chuah, Shyam Prabhakar, Vaidehi Krishnan, John Ouyang
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="


 Cancer cells adapt to environmental and therapeutic pressures through karyotypic plasticity, including whole genome doubling, aneuploidy, and structural genome changes. Polyploid/polyaneuploid giant cancer cells (PGCCs/PACCs) represent an extreme state of this plasticity, arising through endoreplication as a common response to chemotherapy, irradiation, viral infection, hypoxia, or nutrient restriction. PGCCs act as drug-tolerant reservoirs that can later depolyploidize to repopulate tumors, underscoring their role in recurrence and therapy failure. Yet, how readily tumor cells access the PGCC state and how karyotype dynamics sculpt drug sensitivity remain poorly understood. To address this gap, we integrated (i) long-term evolution experiments (LTEEs), (ii) mathematical modeling, and (iii) computational analysis into a unified framework to define karyotype evolution as a predictive biomarker of therapy response. To study karyotype evolution under metabolic stress, we developed CLONEID, an LTEE platform that combines bright-field imaging, single-cell karyotyping and a neutral-drift framework distinguishing drift from selection. Its computer-vision module classifies PGCC versus proliferative states while PCA of other morphometrics revealed stress-specific phenotypes even without whole-genome doubling or overt ploidy change. LTEEs (>6 months) under glucose deprivation, phosphate restriction or hypoxia revealed stress- and ploidy-dependent trajectories. Glucose deprivation promoted whole-genome doubling whereas phosphate restriction and hypoxia drove chromosome loss toward near-diploid states. Strikingly, our in vivo studies confirmed hypoxia-induced ploidy reduction, which was predicted by LTEE trajectories within the 36 days and recapitulated by our model. Moreover, long-term adaptations also shifted therapy response. Glucose-deprived cultures gained resistance to gemcitabine and topotecan while phosphate-deprived cultures became more sensitive to taxanes and carboplatin. Furthermore, our functional assays showed that evolved cultures enhanced T-cell activation while control condition suppressed immunogenicity through tryptophan metabolism. To complement LTEEs, we built an ODE linear chain trick (LCT) model of the cell cycle, calibrated with isogenic diploid and tetraploid TNBC lines. Our model successfully captured ploidy-conditioned therapeutic responses. Specifically, gemcitabine, among 74 tested anticancer agents, eliminated near-diploid cells, but spared tetraploids that entered PGCC state and resumed proliferation after drug withdrawal. Overall, our findings identified that whole-genome doubling, polyploidy, and PGCC formation shape both long-term evolution and acute therapy responses. Integrating LTEEs with LCT modeling successfully links ploidy, karyotypic adaptation, and drug sensitivity. Our approach establishes the ground for “Drug–Karyotype” pairs as biomarkers and shows how metabolic niches sculpt KFLs, offering a path toward personalized, evolution-informed oncology.



 Vural Tagal, Jackson Cole, Richard J. Beck, Didem Ilter, Daria Myroshnychenko, Konstantin Martin, Andriy Marusyk, Ana P. Gomes, Noemi Andor. Modeling karyotype-driven adaptations to metabolic restrictions predicts therapeutic response and immunogenicity in cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B026.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06671e4b962908638a954d43578035d059b71aa3" target='_blank'>
              Abstract B026: Modeling karyotype-driven adaptations to metabolic restrictions predicts therapeutic response and immunogenicity in cancer
              </a>
            </td>
          <td>
            Vural Tagal, Jackson Cole, Richard Beck, Didem Ilter, D. Myroshnychenko, Konstantin Martin, A. Marusyk, , Noemi Andor
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Acidosis in tumors arises from reprogrammed metabolism and compromised vasculature, creating a harsh, acidic microenvironment that drives the evolutionary selection of acid-resistant cell phenotypes. A mathematical model is proposed to integrate phenotypic evolution, microenvironmental acidification, and tumor density dynamics. Three key mechanisms are incorporated in it: frequency-dependent selection favoring acid-resistant cells below a critical pH, stress-induced phenotypic switching, and a positive feedback loop where resistant cells produce excess acid that intensifies selection pressure. The well-posedness of the model is established. Through numerical simulations across biologically relevant parameter regimes, we identify two therapeutically targetable parameters: baseline acid clearance rate (a proxy for vascular perfusion) and a protection factor (representing acid-resistance"machinery"effectiveness) as critical bifurcation parameters for resistance evolution. The model exhibits qualitatively distinct dynamics depending on phenotypic plasticity levels. In low-plasticity tumors, both parameters exhibit sharp bifurcations with strong parameter interactions: clearance and protection effects are context-dependent, with therapeutic interventions effective only within specific parameter ranges. In high-plasticity tumors, both parameters produce continuous, monotonic responses with independent, additive effects. These regime-dependent dynamics suggest that treatment strategies should adapt to tumor plasticity: in the formar, targeting perfusion alone is typically sufficient, though sequential therapy may be required if the perfusion rate approaches or exceeds the bifurcation threshold, whereas in the latter treatment might benefit from combination therapies addressing both parameters simultaneously.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd0887c6d3e30f61231afcb895923c66fd10d672" target='_blank'>
              Evolutionary Dynamics of Acid Resistance in Tumors: A Mathematical Model
              </a>
            </td>
          <td>
            Prithvi Anickode, Fabio Milner
          </td>
          <td>2025-10-31</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Acute leukemias are highly aggressive hematologic malignancies that demand intensive chemotherapy regimens. However, drug toxicity remains a major barrier to treatment success and patient survival. In this context, pharmacogenomics offers a promising strategy by identifying single-nucleotide variants (SNVs) that influence drug metabolism, efficacy, and toxicity, ultimately impacting treatment outcomes. This study analyzed data from the ClinPGx/PharmGKB database to identify clinically annotated variants related to chemotherapy response in Acute Myeloid Leukemia (AML) and Acute Lymphoblastic Leukemia (ALL). A total of 24 variants were curated for AML and 57 for ALL. Among these, nonsynonymous variants were most frequent in ALL (31.6%), while synonymous variants predominated in AML (33.3%). Although traditionally considered neutral, synonymous and intronic variants may influence gene expression through regulatory or splicing mechanisms. The analysis revealed clinically significant variants associated with chemotherapy response, particularly in the ABCB1 gene, observed in 12.5% of AML and 10.5% of ALL cases. Several variants, particularly TPMT, NUDT15, ABCC1, SLC28A3, and RARG, were associated with severe adverse effects such as myelotoxicity, mucositis, cardiotoxicity, and hepatotoxicity. This study reinforces the importance of genetic variants in modulating the therapeutic response and toxicity to chemotherapy drugs in acute leukemias. Analysis of ClinPGx/PharmGKB data emphasizes ABCB1 as a potential resistance marker and supports pre-treatment genotyping of genes like TPMT and NUDT15 to prevent severe toxicities. Future advances should include the expansion of pharmacogenetic studies in underrepresented populations and the clinical validation of new markers in prospective trials, aiming to consolidate precision medicine as a routine part of the therapeutic management of acute leukemias.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41d398f2f78483d0f180028688c26b519950cb78" target='_blank'>
              Pharmacogenomics Applied to Acute Leukemias: Identifying Clinically Relevant Genetic Variants
              </a>
            </td>
          <td>
            Flávia Melo Cunha de Pinho Pessoa, Isabelle Magalhães Farias, B. Nogueira, C. B. Machado, Igor Valentim Barreto, Anna Karolyna da Costa Machado, Guilherme Passos de Morais, Leidivan Sousa da Cunha, Deivide de Sousa Oliveira, André Pontes Thé, Rodrigo Monteiro Ribeiro, Patrícia Maria Pontes Thé, Manoel Odorico de Moraes Filho, Maria Elisabete Amaral de Moraes, Caroline Aquino Moreira-Nunes
          </td>
          <td>2025-10-22</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Background Acute myeloid leukemia (AML) remains a challenge in treatment and mechanisms understanding in disease progression and therapeutic resistance. Through large-scale transcriptomic profiling integrated with 10 machine learning algorithms, we identified CPNE8, a calcium-dependent phospholipid-binding protein which is involved in signal transduction, is significantly upregulated in HOXA-driven AML subtypes, whose function and regulatory mechanisms in leukemia warrants further investigation.
 Methods We integrated clinical and transcriptomic profiles from a cohort of 655 AML patients to develop a machine learning–based prognostic model, with external validation performed in the TCGA-LAML (n = 151) and GSE16432 (n = 436) datasets. CPNE8 expression across hematopoietic and immune cell compartments was profiled using single-cell RNA sequencing (GSE116256). Functional assays following CPNE8 knockdown were performed to evaluate cell viability, apoptosis, cell cycle dynamics, and leukemogenic potential in vivo. CPNE8-interacting proteins were identified via Co-immunoprecipitation (Co-IP) and Mass Spectrum (MS), with downstream signaling examined by immunoblotting. CUT&Tag profiling and dual-luciferase reporter assays were used to elucidate transcriptional regulation. OCR and ECAR were measured to assess cellular mitochondrial respiration and glycolytic capacity. Chemoresistance was assessed through compound screening and CellTiter-Glo assays, and the therapeutic efficacy of Menin inhibition was tested both in vitro and in vivo using a CPNE8-high OCI-AML3 xenograft model.
 Results CPNE8 was markedly upregulated in HOXA9-driven AML and associated with poor prognosis and chemoresistance. Patients with high CPNE8 expression exhibited significantly lower complete remission (CR) rates following two cycles of standard induction chemotherapy compared to CPNE8-low patients (p < 0.001), with pronounced resistance to conventional “3+7” anthracycline-based regimens. CPNE8 knockdown inhibited leukemic cell proliferation, induced apoptosis, and caused G0/G1 cell cycle arrest. Mechanically, CPNE8 physically interacted with SmgGDS, as identified by Co-IP–mass spectrometry and further validated by Co-IP assays and immunofluorescence co-localization，thereby activating the Rap1 GTPase module and driving downstream PI3K-AKT and MAPK (JNK/p38) signaling. To further assess the role of CPNE8 in drug resistance, ex vivo drug sensitivity assays were conducted using PBMCs from 107 AML patients. CPNE8-high PBMCs showed significantly elevated IC50 values across a range of clinically relevant chemotherapeutics, including idarubicin, homoharringtonine, fludarabine, azacitidine, venetoclax, aclarubicin, all-trans retinoic acid (ATRA), and mitoxantrone. Additionally, CPNE8 depletion elevated intracellular and mitochondrial ROS, reduced mitochondrial membrane potential, and impaired both glycolysis and oxidative phosphorylation, thereby sensitizing AML cells to chemotherapy and enhancing their vulnerability to T cell–mediated cytotoxicity. To delineate the upstream regulatory network, CUT&Tag profiling identified HOXA9 as a direct transcriptional activator of CPNE8, positioning CPNE8 as a critical effector of HOXA9-driven leukemogenesis. To therapeutically target the HOXA9–CPNE8 axis in CPNE8-high AML, we evaluated Menin inhibition as a strategy to disrupt the HOXA9 transcriptional program. Treatment with a Menin inhibitor significantly reduced CPNE8 expression in vitro. Importantly, both Menin inhibition and CPNE8 knockdown prolonged survival in CPNE8-high AML xenograft models, underscoring their therapeutic potential.
 Conclusion CPNE8 functions as a critical downstream effector of HOXA9, driving AML progression and therapy resistance through activation of Rap1 signaling, reprogramming of mitochondrial metabolism, and reshaping of the immune microenvironment. Blocking the HOXA9–CPNE8–Rap1 pathway, especially with Menin inhibitors, provides a rationale for stratified treatment strategies in CPNE8-high AML.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7845759371b121eb83b0e1b1c2020de00b0cd246" target='_blank'>
              Targeting CPNE8 suppresses HOXA9-dependent AML progression and overcomes chemotherapy resistance
              </a>
            </td>
          <td>
            Wenjing Shu, Yuncan Cao, Xiaojin Lin, Xin-jie Chen, , Yang Shen
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Somatic mutations in SET binding protein 1 (SETBP1), a putative chromatin regulator, occur in ~30% of high-risk myeloid malignancies, including myelodysplastic syndromes (MDS), chronic myelomonocytic leukemia (CMML), and secondary acute myeloid leukemia (AML), and strongly correlate with adverse clinical outcomes. Despite this significance, the precise roles of SETBP1 mutations in blood cancer remain enigmatic, primarily due to the lack of physiologically relevant cell lines or animal models that recapitulate the clonal evolution. Indeed, our 23 attempts to establish SETBP1-mutated PDX models failed, as the clones did not show advantageous effects. While conventional wisdom suggests that SETBP1 mutations escape proteasomal degradation to drive oncogenesis, our Setbp1 knockout mice revealed that SETBP1 is dispensable for normal and malignant hematopoiesis (Tanaka et al. Leukemia 2023), challenging prevailing paradigms.
 To resolve these contradictions, we developed a blood-specific Setbp1 mutant locus knock-in (KI) mouse model using Vav1-iCre, corresponding to the most prevalent D868N mutation. Unlike retroviral overexpression that immortalizes murine hematopoietic stem and progenitor cells (HSPCs), our KI model did not perturb HSPC function or transcriptome in vivo, indicating that SETBP1 acts as a cooperative rather than an initiating driver. As ASXL1 mutations are the most frequent founder mutations in SETBP1-mutated MDS/AML, we created double KI mice with Asxl1/Setbp1 physiologic mutations for transplant studies. These mice exhibited striking monocytosis (~40% of peripheral white blood cells) and HSPC expansion, rescuing stemness defects imposed by ASXL1 mutations. scRNA-seq/ATAC-seq revealed unique transcriptional clusters enriched for Hoxa9/10, Eya1, Myb, and Myc pathway alongside increased chromatin accessibility, suggesting that both mutants cooperate to remodel chromatin and amplify oncogenic transcriptional programs. Over time, the transplant model with double KI developed a leukemia phenotype, including serially transplantable AML and CMML-like diseases, all spontaneously acquired RAS pathway-activating mutations in Nras, Flt3, and Ptpn11. This reflects findings from over 7,000 patient samples, showing a significant correlation between SETBP1, ASXL1, and RAS pathway mutations. Indeed, the presence of RAS mutations predicts poor outcomes in SETBP1-mutated cases. Functional validation confirmed that all three alterations are required for complete transformation toward AML, establishing a three-step model: ASXL1mutations initiate clonal evolution, SETBP1 mutations enhance oncogenic programs, and RAS pathway activation completes the transformation.
 Leveraging these unprecedented in vivo models and cell lines with physiologically relevant SETBP1 mutations, we conducted comprehensive therapeutic screens using whole-genome CRISPR/sgRNA and pan-cancer drug libraries. The integrated results revealed a critical dependency on the druggable Exportin 1 (XPO1), shown by a 20.8-fold reduction (p = 0.00004) in the 7-day CRISPR screen and a 0.0003-fold growth under Leptomycin B treatment compared to DMSO at day 4. This unique dependency was robustly validated through genetic and pharmacological approaches using shXpo1, Selinexor, and Leptomycin B, resulting in monocytic differentiation and apoptosis in Setbp1-mutated AML cells. Notably, Selinexor monotherapy achieved sustained remissions exceeding 110 days in vivo, profoundly extending survival compared to controls, which succumbed to leukemia with a median survival of 59.5 days (p = 0.0004), demonstrating its potential to eradicate SETBP1-mutated leukemia. Mechanistically, XPO1 physically interacts with SETBP1 through the nuclear export signal groove and acts as a chromatin regulator beyond its canonical nuclear-cytoplasmic transport role. XPO1 inhibition suppressed the transcription of leukemogenic genes, including Hoxacluster genes, Myb, Eya1, and, critically, Myc pathway, with altered XPO1-mediated chromatin accessibility, as shown by integrated ChIP/RNA/ATAC-seq. These results uncover a critical epigenetic vulnerability that represents a promising target for therapeutic intervention in SETBP1-mutated leukemia.
 In summary, our study elucidates the cooperative leukemogenic roles of SETBP1, ASXL1, and RAS pathway mutations and identifies XPO1 as a targetable epigenetic vulnerability. These findings support precision treatment strategies for aggressive myeloid malignancies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a6566631ba342b06e47eaa7af05b86f27fd8cd1" target='_blank'>
              Clonal evolution and therapeutic vulnerabilities in SETBP1-mutated leukemia: Insights from physiological mouse model and integrative functional screens
              </a>
            </td>
          <td>
            Wataru Saika, Koutarou Nishimura, Atsushi Tanaka, J. Takeda, A. Kon, A. Yoda, Kazunari Aoki, Masaki Nomura, Yui Koike, Hiromi Yamazaki, Seigi Oshima, Shinri Okada, Daichi Morii, Weijia Zang, Takaya Yamasaki, Hiromi Ito, Manabu Nakayama, Hiromi Tatsumi, Masashi Matsuda, Chien-Chin Lin, Kosuke Yusa, Haruhiko Koseki, Makoto Murata, Seishi Ogawa, Daichi Inoue
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="
 Medulloblastoma is molecularly characterized into 4 main subgroups consisting of WNT, SHH, Grp 3 and Grp 4, which are further divided into 13 subtypes. Medulloblastoma manifests as a clinically heterogeneous tumor with varying prognoses depending on the molecular subgroup. Treatments for high-risk patients include craniospinal irradiation (36Gy) after surgical resection and chemotherapy. Group 3 tumors with upfront radiation therapy relapse in a shorter amount of time at 1 year. Currently, there are limited therapeutic strategies that address medulloblastoma tumor recurrence which has detrimental effects on patient outcome with <5% long term survival. Subsequently, there is a critical need to investigate the molecular mechanisms underlying therapeutic resistance in medulloblastoma and identify the best strategies to combat it. To understand what causes radiation resistance in medulloblastoma, we analyzed matched primary and relapsed patient tumors by single cell ATAC seq + gene expression. We further evaluated radiation resistance in an orthotopic mouse model and analyzed gene expression changes by single-cell RNA sequencing. We identify a resistant cell population that exhibits a distinctive gene signature comprising stem and metabolic gene transcription programs. We observe elevated expression of metabolic marker genes such as wtIDH1 and stem cell markers, TEAD1 and SOX6, within the recurrent cell population. We speculate that radiation resistance in group 3 medulloblastoma is facilitated by wtIDH1 metabolic reprogramming, leading to epigenetic adaptation and cell persistence. Thus, we evaluate how IDH1 changes the epigenetic landscape of radiation-resistant and sensitive cells by examining histone modifications (H3K27ac, H3K27me3, H3K4me3) and the broader chromatin architecture in response to IDH1 inhibition by Cut and Run and ATAC-sequencing. We observe that the manipulation of wtIDH1 reduces MB cell growth, stemness characteristics, and changes the chromatin landscape. Further, we find inhibition of wtIDH1 enhances susceptibility to re-irradiation and provides a survival benefit. Our results highlight evidence of the development of radiation resistance and provide a potential therapeutic alternative for treatment of relapsed MB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9d6602a0e48e41dffae7101ad80db0948939a5f" target='_blank'>
              DDDR-30. Targeting wtIDH1 in radiation-resistant medulloblastoma
              </a>
            </td>
          <td>
            Bethany L. Veo, Dong Wang, J. DeSisto, Angela M Pierce, Breauna Brunt, Rajeev Vibhakar
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="


 CRISPR/Cas9 screening is a powerful tool used to delineate genes critical for AML progression and drug resistance. However, most of the studies have been performed in cell lines and PDX models. While cell lines fail to represent the genetic, epigenetic, and phenotypic heterogeneity of primary leukemic cells, PDX models are limited by variable engraftment efficiency, time-consuming processes, and high costs. To overcome these challenges, we pioneered an integrated approach combining bioinformatics and CRISPR/Cas9 screening to uncover genetic drivers and vulnerabilities in AML using patient-derived primary cells in culture. Insights from our study could inform the development of targeted therapies and improve patient outcomes.



 Custom CRISPR/Cas9 screen targeting 2,440 genes associated with AML, leukemic stemness, and the druggable genome was performed using mononuclear cells (MNCs) isolated from peripheral blood or bone marrow from 9 AML patients. Briefly, the cells were transduced with custom library and cultured for 24-30 days. After an initial few days in suspension, a subset of MNCs from 5 of 9 patients transitioned to an adherent state 5-7 days post-transduction. Both adherent and suspension cells were collected at different time points, sgRNA was quantified and sequenced using the Illumina NovaSeq X Plus. sgRNA abundance was analyzed using the RRA algorithm of the MAGeCK computational tool and genes with significant negative or positive enrichment in each patient was identified by comparing early versus late time points and adherent versus suspension cell fractions.



 In a subset of patient samples, AML cells transitioned from suspension to an adherent phenotype during long-term culture, a state previously associated with increased chemotherapy resistance due to interactions with endothelial niches. Our CRISPR/Cas9 screen revealed that this adherent cell population harbored distinct genetic dependencies, including consistent negative enrichment of EDNRA and SOD1.These genes were selectively essential in the adherent fraction but not in suspension cells, suggesting context-specific vulnerabilities. The identification of these targets offers new opportunities to therapeutically disrupt chemo resistant AML niches, potentially enhancing the efficacy of existing treatments and reducing relapse risk. EDNRA has been implicated in many cancers and in context of AML, it is a downstream target of HOXA9 and MEIS1. Our results indicate that EDNRA knockout inhibits AML cell proliferation and growth, highlighting its potential as a therapeutic target. Furthermore, EDNRA is overexpressed in AML compared to normal cells, and since FDA approved EDNRA inhibitors such as Macitentan, Bosentan, Ambrisentan, and Clazosentan are already being used to treat pulmonary arterial hypertension, this presents a promising opportunity to evaluate these inhibitors as potential chemotherapy-sensitizing agents for high-risk AML.SOD1 plays a critical role in eliminating toxic radicals generated within biological systems and has been associated with poor outcomes in AML. SOD1 has been identified as a synthetic lethal target in PPM1D-mutant leukemia cells, corroborating our findings. TRAPPC9, SUZ12, and ERG were significant genes with negative enrichment when comparing adherent and suspension fractions across 5 patient samples. These genes have a higher expression in primary AML cells compared to normal controls, implying potential roles in disease progression. SDHA was identified as a significant negatively enriched gene in the suspension cell fraction in 4 of 9 patients. It exhibited elevated expression in AML cells compared to controls, and its deficiency has been linked to the development of drug resistance in AML, emphasizing its role in both drug resistance and disease progression.



 This study demonstrates the establishment of a scalable platform for CRISPR screening in primary AML cells for identification of more conserved vulnerabilities that may be exploited therapeutically, with EDNRA as a potential target. Ongoing work includes validating key targets with the goal of advancing therapeutic strategies. Future studies will expand CRISPR screening to a broader set of patient samples with diverse cytogenetic profiles and incorporate CROP-seq technology to perform single-cell CRISPR screens, providing deeper insights into the molecular mechanisms underlying AML progression and relapse.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46eb706147f8e498d47ce576631d70d699b36b33" target='_blank'>
              A patient-derived CRISPR platform reveals selective dependencies in Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Aneesha Nath, Nam Nguyen, A. Tagmount, Nathan D. Seligson, Christopher Vulpe, Vivek Shastri, Dina Khalaf, Christopher Cogle, S. Pounds, J. Lamba
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="


 CML progression is defined by increased disease burden and potential for transformation to blastic phase (BP), yet the mechanisms and ability to predict this evolution remain elusive. We applied elements of state-transition theory to model transcriptional dynamics underlying leukemic progression, where individual cells' gene expression profiles represent microstates, and aggregated population-level gene expression profiles define macrostates that govern phenotypic transitions from health to disease initiation to overt leukemia. Previously, we demonstrated through bulk transcriptomic analyses that CML progression and therapeutic responses can be accurately predicted by modeling a leukemic potential landscape derived from transcriptional state-space geometry.
 To extend this framework to single-cell (sc) resolution and capture more granular information at the microstate level, we collected weekly peripheral blood samples from inducible chronic phase (CP) and BP CML mouse models to generate time-series scRNA-seq data tracking the transition from health to overt CP or BP. After performing quality control and cell labeling, we used the first (healthy) time point before the induction of BCR::ABL (T0) and the final leukemia time point (Tf), in addition to other intermediate time points, to assess changes in cell populations and gene expression. All analyses were first performed in CP mice, validated in BC mice, and corroborated using two independent human CML scRNA-seq datasets.
 By comparing sequential time points, we observed that progression to CP CML was associated with a significant decrease in B cells (p<0.01) and increases in both myeloid (p<0.01) and stem cell (p<0.001) populations. Differential gene expression (DEG) analysis revealed marked transcriptional changes across all major lineages (DEGs: B cells = 781; T cells = 2,149; Myeloid = 1,999; Stem cells = 194). Despite these changes, no single-cell transcriptional state at any time point was uniquely associated with health or disease, as transcriptional profiles of leukemic cells largely overlapped with those from healthy cells. However, when scRNA-seq data were computationally aggregated into pseudobulk (PsB) samples, mimicking bulk RNA-seq, a distinct CML state-space emerged, revealing a clear disease trajectory defined by three stable macrostates representing early, transitional, and late leukemia. Cell type–specific PsB analyses uncovered independent state-transition dynamics in B cell, T cell, myeloid, and stem cell compartments, highlighting previously unrecognized complexity in disease evolution involving multiple lineages. We further showed that the transition from health to leukemia at the PsB level could be reconstructed as a linear combination of transitions within individual cell subpopulations.
 To this end, we quantified each lineage's contribution to disease progression by performing computational simulations that subtracted the influence of each cell type in turn. Interestingly, B and myeloid populations contributed most and comparably to the global disease trajectory—despite B cells decreasing and myeloid cells expanding over time. This counterintuitive finding suggests that leukemic information is encoded not simply by abundance but by dynamic transcriptional shifts across compartments. These findings were validated in a BP mouse model and in two human CML datasets.
 In summary, we introduce a conceptual framework that distinguishes between microstates (individual cells) and macrostates (population-level transcriptional ensembles), showing that disease progression is not encoded at the single-cell level but instead emerges only when cells are aggregated. While this may appear intuitive, we were surprised to find that higher resolution sc data did not provide clearer insight than pseudobulk data. We speculate that this may reflect greater Shannon information entropy at the single-cell level, which introduces variability that obscures coherent state transitions. To our knowledge, this framework is novel and may inform future approaches to interpreting scRNA-seq data in leukemia and other diseases.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ef9c6ca07080f16386ecab84fd1ec67de21d807" target='_blank'>
              Chronic myeloid leukemia (CML) evolution is not encoded in single-cell microstates, but emerges from population-level macrostates
              </a>
            </td>
          <td>
            D. Frankhouser, Dandan Zhao, , Anupam Dey, Ziang Chen, Jihyun Irizarry, Jennifer Rangel Ambriz, S. Branciamore, Denis O’Meally, Jeffrey Trent, Stephen J Forman, Ya-Huei Kuo, Adam MacLean, Bin Zhang, Russell C. Rockne, Guido Marcucci
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="This comprehensive review investigates the complex interactions between oncogenesis and stem cells, underscoring the crucial importance of cancer stem cells (CSCs) in the mechanisms of tumorigenesis, metastatic dissemination, and resistance to therapeutic interventions. CSCs constitute a distinct subset within neoplasms, defined by unique characteristics such as the ability to self-renew, the potential to differentiate into various cellular phenotypes, and an inherent resistance to conventional treatment strategies. It is hypothesized that these properties are regulated by a multitude of signaling pathways, including Wnt/β-catenin, Notch, and Hedgehog signaling cascades. The tumor microenvironment (TME) significantly influences CSC dynamics by providing a nurturing niche that enhances CSC survival, enables immune evasion, and contributes to resistance against therapies. Key components of the TME, such as cancer-associated fibroblasts and immune-modulating exosomes, play a crucial role in augmenting CSC plasticity and metastatic capabilities. Furthermore, the hypoxic environment prevalent within the TME enhances CSC stemness through both metabolic and epigenetic alterations. Despite the existence of promising CSC-targeted therapeutic strategies, including immunotherapy, small-molecule inhibitors, and combinatorial approaches, notable challenges persist. These obstacles include CSC plasticity and heterogeneity, as well as the potential for immune rejection of CSC-targeted therapies. Advancements in precision medicine, genomics, and novel therapeutic interventions instill hope for the improvement of CSC-targeted strategies. A comprehensive understanding of the fluid interactions between CSCs, standard stem cells, and the TME is vital for advancing personalized cancer treatments that enhance patient success and reduce recurrence rates. This review highlights the need for continued research to bridge existing knowledge gaps and facilitate the translation of foundational scientific discoveries into clinical practices.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e65aea70f8623678ce094b97b023361a5efc723c" target='_blank'>
              Biology of stem cell paradox: a double-edged sword-implications for cancer therapy
              </a>
            </td>
          <td>
            B. Mengistu, Yitayew Demessie, M. Kinde, Kalkidan Getnet, A. Bitew, Kassahun Berrie, Wagaw Sendeku, Asnakew Mulaw Berihun, A. Abat, Atsede solomon Mebratu, Melaku Getahun Feleke, Nesibu Tilahun Yesist, M. M. Ferede, Melkie Dagnaw Fenta
          </td>
          <td>2025-10-28</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Activating KRAS mutations are highly relevant to various cancers, and KRAS is the most frequently altered oncogenic protein in solid tumors. While historically considered undruggable, two KRASG12C inactive state-selective inhibitors are currently approved for treating patients with non-small cell lung cancer. However, these agents only demonstrate a 30–40% response rate and a median progression-free survival of approximately 6 months, with the inevitable emergence of resistance mechanisms, hence remaining far from achieving a cure. Additionally, several cancers with poor prognostic outcomes, such as pancreatic adenocarcinoma, are driven by other non-G12C KRAS mutations and thus have no effective targeted therapies. Improvements in understanding RAS signaling, RNA, and nucleic acid chemistry, as well as the role of the tumor microenvironment, have sparked a paradigm shift in the approach to KRAS inhibition and suggested the potential for several novel combination therapies. In this review, we provide an overview of the RAS pathway and discuss the ongoing development and status of therapeutic strategies for targeting the oncogenic RAS. We further delve into the challenges of resistance mechanisms to better understand the rationale behind these developing strategies, describe their mechanisms of action, and offer insights into the current clinical trial status of each of these approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c697e95b8beed852f306ce44fd5e98ffcf2023be" target='_blank'>
              Targeting KRAS mutations: orchestrating cancer evolution and therapeutic challenges
              </a>
            </td>
          <td>
            Khalil Choucair, Hafsa Imtiaz, Md. Hafiz Uddin, M. Nagasaka, M. N. Al-Hallak, P. Philip, B. El-Rayes, Boris C. Pasche, A. Azmi
          </td>
          <td>2025-11-28</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Traditional population sequencing methods struggle to directly correlate gene perturbations with transcriptome changes at the single-cell level, due to limitations such as long cycle times and significant batch effects. Single-cell sequencing (SCS) can reveal the diversity and dynamics of cell populations, while CRISPR gene editing and screening technologies enable large-scale functional gene discovery. The combination of these two approaches has led to the development of ScCRISPR-seq. This article reviews the principles of CRISPR screening and SCS technologies, introduces the development of ScCRISPR-seq, and, drawing on research on lung cancer and intrahepatic cholangiocarcinoma (ICC), summarizes its key applications in cancer research, including deciphering tumor heterogeneity, elucidating mechanisms of drug resistance, and modulating the immune microenvironment. This article further explores challenges with this technology, including experimental workflow, data sparsity, and technical noise. Solutions are proposed, including data completion using AI algorithms, eliminating amplification bias using UMI tags, and optimizing gene-phenotype associations through network modeling. However, clinical translation is limited by unclear mechanisms of action and high barriers to entry in drug development. Future research may consider combining ScCRISPR-seq with network pharmacology and molecular docking techniques to improve the efficiency of new cancer drug development and advance precision immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1262cf91f44f77bd99618ed1bc278657dd28dee" target='_blank'>
              The Application and Prospect of Single-cell CRISPR Screening Technology in Tumor Research
              </a>
            </td>
          <td>
            Zineng Zhou
          </td>
          <td>2025-10-23</td>
          <td>MedScien</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Pediatric acute myeloid leukemias (AML) are genetically and molecularly diverse. Pediatric AML carry a range of different driver mutations, often including translocations that produce fusion oncoproteins. Furthermore, they arise from diverse progenitor cell populations and at different stages of fetal and childhood development. This molecular and contextual diversity complicates efforts to identify effective targeted therapies, in part because modeling strategies do not capture the genetic heterogeneity observed among patients.
 We have developed a multipronged strategy to model complex genetic and contextual features of pediatric AML in mice, with the overarching goal of identifying malignant self-renewal programs that afford targetable vulnerabilities. We first used Cellular Indexing of Transcriptomes and Epitopes by sequencing (CITE-seq) and paired functional assays to identify functionally distinct subpopulations of NUP98-rearranged (NUP98r) AML using a previously described NUP98::HOXD13/Flt3ITD transgenic model. Our approach identified a leukemia stem cell (LSC) subpopulation and gene expression signature that was conserved in human NUP98r AML. The LSC signature was also enriched in human UBTF-tandem duplication (UBTF-TD) and NPM1 mutated (NPM1c) AML, but not KMT2A-rearranged AML, suggesting mutation class-specific self-renewal mechanisms.
 To compare LSC self-renewal mechanisms across molecular subclasses of AML, we developed a novel strategy to generate inducible, genetically diverse AML models. Specifically, we generated induced pluripotent stem cells (iPSCs) and chimeric mice to express NUP98::HOXA9, NUP98::KDM5A, NUP98::NSD1, RUNX1::RUNX1T1, MNX1, or KMT2A::MLLT1 oncoproteins specifically in blood cells at defined stages of ontogeny. We used CRISPR/Cas9 gene editing to introduce cooperating mutations at endogenous loci, including Flt3 internal tandem duplication (FLT3ITD), Nf1 loss of function (NF1LOF), and NrasG12D. With this approach, we could induce expression of any desired oncoprotein at any desired stage of development along with relevant cooperating mutation, all while bypassing complex mouse breeding. We also used gene editing strategies to generate NPM1c/FLT3ITD and UBTF-TD/ FLT3ITD AML, again with mutations at endogenous loci. The resulting AML models are transplantable, genetically manipulable and in some cases gave rise to immortalized in vitro cell lines.
 To credential the models and identify fusion/cooperating mutation-specific vulnerabilities, we performed CITE-seq focusing on NUP98::HOXA9/FLT3ITD, NUP98::HOXA9/NF1LOF,RUNX1::RUNX1T1/NF1LOF, KMT2A::MLLT1/NRASG12D, UBTF-TD/ FLT3ITD, and NPM1c/FLT3ITD models. Cross species comparisons to human AML revealed faithful representation of heterogeneity and gene expression within the murine AML. Each genetic subclass exhibited distinct but reproducible differentiation trajectories and transcriptional profiles. NUP98::HOXA9/FLT3ITD, NUP98::HOXA9/NF1LOF, NPM1c/FLT3ITD, UBTF-TD/ FLT3ITD, and KMT2A::MLLT1/NRASG12DAML all displayed intraleukemic heterogeneity and had putative LSC populations that expressed known self-renewal genes. RUNX1::RUNX1T1/NF1LOF AML was homogeneous and lacked a discernible LSC population. Presumptive LSC populations from NUP98::HOXA9/FLT3ITD, NUP98::HOXA9/NF1LOF, NPM1c/FLT3ITD, and UBTF-TD/ FLT3ITD were all enriched for a shared gene expression signature. These highly tractable models thus capture key aspects of human pediatric AML biology.We sought to identify actionable, mutation class-specific vulnerabilities within putative LSC populations, focusing primarily on the NUP98r models. We found that genes encoding sphingosine transporters SPNS2 and SPNS3were highly expressed in NUP98r LSCs relative to non-LSCs, identifying them as potential therapeutic targets. SPNS2/3 were also expressed, to a lesser degree, in NPM1c/FLT3ITD and UBTF-TD/ FLT3ITD AML. Pharmacological SPNS2/3 inhibition severely impaired NUP98rAML growth in vitro, and it significantly extended survival in vivo. In some cases, mice with NUP98r AMLwere cured with SPNS2/3-directed monotherapy. Other genetic subclasses were less sensitive to SPNS2/3 inhibitors, as were normal human and murine hematopoietic progenitors. Our data indicate a therapeutic window for targeting SPNS2/3 in NUP98r AML. They raise the possibility of identifying other mutation class-specific vulnerabilities through the use of well-credentialed, iPSC-derived chimeric AML models.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97c884502e32c16eaf731d0261d136ef166db839" target='_blank'>
              NUP98-rearranged pediatric AML is sustained by a conserved leukemia stem cell program with SPNS2/3 as a targetable vulnerability
              </a>
            </td>
          <td>
            Jihye Yoon, Elisabeth D Denby, Wei Yang, Run Zhang, Masayuki Umeda, Tingting Hu, Jonny Mendoza-Castrejon, Yanan Li, Emily B. Casey, Riddhi Patel, , J. Klco, Jeffrey Magee
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b9752e134e3e7566b741e6bb0305e92d0896d6a" target='_blank'>
              Timing the onset of homologous recombination deficiency before cancer diagnosis
              </a>
            </td>
          <td>
            Michail Andreopoulos, Muchun Niu, Yang Zhang, V. Viswanadham, D. Gulhan, Hu Jin, Felipe Batalini, Gerburg Wulf, Chenghang Zong, Peter J. Park, Dominik Glodzik
          </td>
          <td>2025-10-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Glioblastoma, IDH wild-type (WHO grade 4) (GBM), is the most common primary brain tumor in adults with a 21-month median overall survival, despite surgical-resection and radio-chemotherapy. Bevacizumab, a monoclonal antibody towards vascular endothelial growth factor-A, is used to treat recurrent-GBM. To find predictors of poor-response, patient-derived xenograft (PDX)-tumors were treated with bevacizumab or vehicle and subsequently grouped based on survival-response; RNAseq expression was then compared by responder-status. Bioinformatic-analysis demonstrated differential gene expression in tumors from poor-responders (six-PDXs) as compared to tumors from good-responders (three-PDXs), along with upregulation of angiogenesis and collagen gene-sets in poor-responders. Within these gene-sets, multiple genes known to be regulated by the early growth response-1 (EGR1) transcription factor, which was also upregulated, were identified and CHRNA7 (α7-nicotinic-acetylcholine receptor, α7-nAChR) was selected for validation. In terms of protein/functional studies, in the bevacizumab-treated poor-responders, nuclear-EGR1 was elevated, Ki67-labeling was increased in EGR1high tumor, and there was increased angiogenesis. Expression of α7-nAChR and nuclear EGR1 was directly correlated, suggesting CHRNA7 is an EGR1 downstream target. Data-mining (GLASS-database) showed that recurrent GBM in females with an elevated EGR1 and methylated MGMT promoter had a shorter survival. In summary, GBM with increased EGR1 expression, Ki67-labeling in EGR1high tumor and angiogenesis demonstrated a poor-response to bevacizumab, suggesting EGR1 could be useful in predicting response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/270b0171fcf8b62df506c60230e03fbed1dd756f" target='_blank'>
              Transcriptomic analysis identifies a potential target for bevacizumab resistant glioblastoma
              </a>
            </td>
          <td>
            R. Lodha, Gaelle Muller-Greven, Muhammad Asad Maqbool, Lydia Guo, Graham Buchan, Kailash Singh, Maha A. Qadan, Vipin Shankar Chelakkot, Paul A. Decker, K. Bakken, B. Carlson, Danielle M. Burgenske, Amy S Nowacki, G. Vasmatzis, S. Kizilbash, J. Sarkaria, C. Gladson
          </td>
          <td>2025-10-22</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a highly resistant brain tumor with a dismal prognosis despite standard treatments like temozolomide (TMZ) and radiation therapy (RT). This review delineates the molecular mechanisms driving this resistance to inform future therapeutic strategies. A narrative review was conducted using PubMed, synthesizing studies from the year 2000 up to March 2025 on DNA repair enzymes, TMZ and RT resistance pathways, anti-apoptosis mechanisms, genetic heterogeneity, and cell cycle dysregulation in GBM. Five key resistance mechanisms were identified: (1) DNA repair enzymes (e.g., MGMT, APNG) counteract TMZ-induced damage, regulated by NF-κB and CHD4; (2) TMZ and RT resistance depends on MGMT methylation and MMR functionality; (3) anti-apoptotic pathways (e.g., Bcl-2, PI3K/Akt) protect GBM cells, mitigated by TRAIL sensitization; (4) genetic mutations (EGFR, PTEN, TP53) confer adaptability; and (5) quiescent and proliferative GBM stem cells (GSCs) sustain recurrence. Novel therapies targeting HK2 and neural stem cells show promise. GBM resistance is multifaceted, requiring personalized, multi-targeted approaches. Future research should prioritize epigenetic modulation and combination therapies to enhance patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d28a9446444f9c597df4c24c1f9700021704068b" target='_blank'>
              Molecular Insights into Glioblastoma Resistance and Implications for Treatment Advancements: A Systematic Review
              </a>
            </td>
          <td>
            Kaustov Chakrabarti, Amanda M. Mohabir, Mahesh Nukala, Sara M. Safiullah, Tierney R. Woitas
          </td>
          <td>2025-11-21</td>
          <td>Scholarly Research In Progress</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="F-box proteins have been well-established as important regulators of cancer development after decades of exploration. As the understanding of cancer biology continues to deepen, novel biological characteristics of cancer cells have been incorporated, and the perspective has expanded from cancer cells themselves to a more complex tumor microenvironment (TME). In this context, F-box proteins form an extensive and multi-layered regulatory network to fine-tune diverse biological events, not only within cancer cells, but also in the TME. Therefore, in this review, we summarize the functional and mechanistic paradigms of F-box proteins in both cancer cells and the TME, and underscore their adaptability and wide coverage on the basis of a sophisticated regulatory hierarchy, presenting the clinical prospects of F-box proteins as potential targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/950db0086d31d060f586b6c69912e1bbecc0d101" target='_blank'>
              F-box proteins in cancer: from cancer cells to the tumor microenvironment
              </a>
            </td>
          <td>
            Jingyi Cheng, Ousheng Liu, Xin Bin, Zhangui Tang
          </td>
          <td>2025-10-13</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Introduction: Acute Myeloid Leukemia (AML) is a heterogeneous malignancy characterized by the clonal expansion of immature myeloid precursor cells in the bone marrow and peripheral blood, leading to disrupted hematopoiesis and rapid disease progression. The incidence of AML increases with age, with a median age at diagnosis of approximately 68 years. Prognosis remains poor, particularly among older adults and those with adverse-risk genetic mutations, with 5-year overall survival rates often below 30%, even with intensive chemotherapy and hematopoietic stem cell transplantation (Ferrara et al., 2024; Ofran et al., 2023). With these clinical limitations, there remains a critical need for a better understanding of leukemogenic mechanisms and therapeutic resistance to guide novel and effective targeted treatments. In AML, long non-coding RNAs (lncRNAs) are now recognized as key players in leukemogenesis, disease progression, and therapeutic response. In particular, MALNC is a novel lncRNA recurrently overexpressed in AML subtypes harboring NPM1, IDH2 mutations, or the PML-RARA fusion gene, but its functional significance remains unclear (Esteller 2022). This study examined the role of MALNC in regulating transcriptomic response to all-trans retinoic acid (ATRA)–induced differentiation in acute promyelocytic leukemia (APL).
 Methods: A publicly available RNA-seq dataset GSE299161, originally designed to investigate the independent and ATRA-interactive effects of MALNC deletion on global gene expression, was reanalyzed. The dataset included RNA-seq data from two CRISPR-Cas9 knockout conditions, namely KOA (targeting MALNC exons 1.1 and 1.2), and KOB (extended deletion of exon 1.0) as well as untreated wild-type (WT) controls. Additionally, KOB and WT NB4 APL cells were treated with 1 µM ATRA for 72 hours. Differential expression was assessed with R packages (FDR < 0.05; |log₂FC| > 0.2).
 Results: No significant differentially expressed genes (DEGs) were identified between WT and KOA or between WT and KOB under untreated conditions, suggesting that deletion of MALNC exons 1.0, 1.1, and 1.2 does not independently perturb baseline gene expression in NB4 APL cells. By contrast, 72-hour ATRA exposure reprogrammed the transcriptome, generating 5,992 DEGs (2,973 up- and 3,019 downregulated genes) in WT and 5,202 ((2,670 up- and 2,732 downregulated)) in KOB cells. Among these, 2,907 transcripts (1,371 up- and 1,536 downregulated) showed significantly different log₂ fold changes between KOB and WT, demonstrating that MALNC modulates the magnitude of response to ATRA for a substantial subset of genes. A KOB-versus-WT comparison under ATRA uncovered 2,857 DEGs (1,463 up- and 1,394 downregulated); within this set, 237 genes (113 up-and 124 downregulated) were both ATRA-regulated and MALNC-dependent. This MALNC-sensitive cohort was enriched for canonical differentiation drivers (PML, ITGAM, CSF3R, ALDH1A2, SPI1, IRF1, GDF15, and TGM2, all markers of myeloid differentiation that are consistently induced during ATRA-mediated maturation of APL cells), while oncogenic or pro-survival transcripts (MYC, NPM1, BCL2, and POU2F2) were repressed, suggesting a MALNC-mediated amplification of pro-differentiation and anti-leukemic programs.
 Conclusions: MALNC deletion alone does not alter gene expression under steady-state conditions, indicating that it is dispensable for basal transcription, However, MALNC significantly influences the gene expression dynamics during ATRA-induced differentiation. These findings suggest that MALNC is a key modulator of retinoid-responsive gene networks and may serve as a potential therapeutic target or biomarker for treatment stratification in AML, particularly in subtypes exhibiting sensitivity to the retinoic acid pathway. Further functional studies are warranted to elucidate the molecular mechanisms by which MALNC interacts with ATRA signaling and its implications for overcoming therapeutic resistance in leukemia.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51b6e8550aab8996596e0130d08fe2b3a8f6f2ab" target='_blank'>
              Lncrna malnc modulates ATRA-induced transcriptional reprogramming in acute promyelocytic leukemia cells
              </a>
            </td>
          <td>
            Qian Wang, 
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 B-cell acute lymphoblastic leukemia (B-ALL) is the most common pediatric cancer, with Hispanic/Latino (H/L) children in the United States exhibiting the highest incidence rates. Despite this, the genetic basis of this susceptibility remains poorly understood. Genome-wide association studies identified several well-replicated single-nucleotide polymorphisms (SNPs) associated with childhood B-ALL risk. Recent analyses discovered a novel childhood B-ALL risk signal near the IKZF1 gene that is positively selected for in H/L populations. The putatively causal variant, rs1451367 C>T, resides downstream of IKZF1 in a 3'cis-regulatory element (3'CRE) defined by histone marks and chromatin accessibility. In addition, an independent B-ALL risk variant, rs17133807 G>A, found across ancestries lies 26 bp away in the same regulatory element. However, how this regulatory element mechanistically impacts IKZF1 expression remains unknown.
 To unravel the molecular role of the 3'CRE, we integrated chromatin accessibility data, 3D chromatin interaction mapping, computational transcription factor (TF)-binding predictions, and functional genomic assays. ATAC-seq profiling from 156 B-ALL patients revealed a distinct and variable accessible chromatin landscape at this 3'CRE in B cell progenitors and precursors, suggestive of highly-regulated and stage-specific activity. Of note, in the presence of the risk variants, allele-specific chromatin accessibility was reduced, implicating decreased enhancer activity. Further, Promoter Capture Hi-C data from human lymphoblastoid cells revealed strong interaction with the IKZF1 promoter, supporting the CRE's role as a bona fide IKZF1 enhancer. Having established its accessibility and promoter interaction in early B cells, we next sought to identify the TFs occupying the 3'CRE and determine how their binding is altered by the risk allele.
 Employing motifbreakR and delta SVM modeling, we predicted that rs1451367 would disrupt binding of several Erythroblast Transformation Specific family members, including ERG, ETV6, ELK3, in addition to altering a potential IKZF1 motif. Prioritizing these TFs with a negative differential binding score, we then performed chromatin immunoprecipitation (ChIP) coupled with quantitative real-time PCR (qPCR) to examine differential TF binding. To model the risk variants at this locus, we used CRISPR/Cas9 genome editing to introduce a 60 bp microdeletion spanning the 3'CRE variants in SEM cells, a human B-ALL cell line with a KMT2A-AFF1 translocation, in which we confirmed the CRE harbors accessible chromatin. ChIP qPCR revealed strong ERG occupancy at the 3'CRE, which was abolished in the presence of microdeletions of the risk variants. These findings implicate ERG as a key participant in orchestrating IKZF1 expression and demonstrate that the risk variant disrupts this interaction. Intriguingly, Nalm-6 cells, which harbor an IGH-DUX4 rearrangement, displayed a strikingly different pattern, with an absence of ERG binding at the 3'CRE. Western blotting detected an additional ERG isoform in these cells, which we identified as ERGalt – a dominant-negative isoform generated through alternative transcription driven by IGH-DUX4. ChIP across multiple B-ALL lines confirmed distinct binding profiles, underscoring the context-dependent regulation of this enhancer by ERG across leukemic subtypes. Notably, as IKZF1 itself is known to regulate ERG, our findings suggest a previously unrecognized feedback mechanism that might occur among these TFs. Also, we found the adjacent multi-ancestry risk variant to disrupt a TCF3 motif, with ongoing studies assessing its impact on IKZF1 transcription. Thus, to more finely dissect precise regulation at this element, we are conducting systematic mutagenesis studies at this CRE.
 Collectively, this work identifies a mechanistic model in which the Hispanic-specific risk variant disrupts ERG-mediated regulation of IKZF1, stalling B-cell differentiation at an early stage and priming cells for leukemic transformation. By identifying ERG as a novel transcriptional regulator at the IKZF1 3'CRE, our work reveals a previously unrecognized regulatory intersection between two key B-cell transcription factors. These insights not only elucidate how population-specific noncoding variation contributes to B-ALL susceptibility, but also highlight a regulatory axis that could be leveraged for enhancer-targeted or TF-modulating therapeutic interventions in the future.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/606e694bb6a2d590187674c0419d978017077562" target='_blank'>
              Dissecting mechanisms at an IKZF1 cis-regulatory element harboring ancestry-specific and cross-ancestry B-cell acute lymphoblastic leukemia risk variation
              </a>
            </td>
          <td>
            L. Messa, Lara Wahlster, Andrew Lee, Pardiss Mehrzad, Mateusz Antoszewski, Anna-Lena Neehus, , Alexis Caulier, Gaurav Agarwal, A. D. de Smith, Jonathan Paolino, Yana Pikman, Katia Georgopoulos, V. Sankaran
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Epidermal growth factor receptor exon 20 insertion mutations (EGFRex20ins) are a unique molecular subtype of non-small cell lung cancer (NSCLC) linked to resistance to EGFR tyrosine kinase inhibitors of the first and second generations. Until recently, these patients had few and frequently ineffective treatment options. Amivantamab, a bispecific antibody targeting both EGFR and mesenchymal–epithelial transition factor, has become a novel therapeutic strategy for this population. This review explores the mechanism of action of Amivantamab and its clinical efficacy and safety as demonstrated in clinical trials. Additionally, the clinical development of the subcutaneous formulation of amivan-tamab, real-world evidence and its regulatory status were evaluated. Lastly, we contex-tualise Amivantamab in the current treatment landscape by contrasting it with mobocer-tinib and highlighting current studies that aim to improve central nervous system activity and overcome resistance mechanisms. This review highlights the therapeutic benefit of Amivantamab in EGFRex20ins-mutated NSCLC and offers guidance for future research in this quickly developing area.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac73581dcae1cceea8c975ae407a4672d857585f" target='_blank'>
              Evolving treatment strategies for EGFRex20ins-mutated NSCLC: a comprehensive review of Amivantamab’s role and future directions
              </a>
            </td>
          <td>
            Rafael Alvim Pereira, M. Tumelero, W. Schwengber, G. Lenz
          </td>
          <td>2025-11-17</td>
          <td>ecancermedicalscience</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Blood serves as a valuable biospecimen due to its ability to provide clinical insights. Peripheral blood mononuclear cells (PBMCs) can be readily isolated and are frequently used to monitor clinical progression and explore mechanisms of disease. Multiomic analyses provide powerful insights by linking open chromatin regions accessible to regulatory elements with gene expression. Here, PBMCs were profiled from a healthy and a diseased Acute Lymphoblastic Leukemia (ALL) donor at two timepoints six months apart. Analysis used 10x Genomics’ Single Cell Multiome ATAC + Gene Expression platform, enabling single cell RNA (scRNA-seq) and single cell ATAC sequencing (scATAC-seq) to profile transcriptomic and epigenomic landscapes simultaneously. Full-length T-cell receptor (TCR) sequences linked to clinical progression were also characterized using 10x Genomics’ GEM-X Immune Profiling technology. Over six months, profiling over 15,000 PBMCs, we identified an increase in naive B cells and a reduction in CD8+ effector-memory T cells, with a notable decline in the two most prominent TCR clonotypes, highlighting an altered adaptive immune response during disease progression. Parallel multiomic profiling uncovered distinct regulatory networks in advanced disease involving exhaustion markers TIGIT and LAG3 within T and NK cell populations. This study, enabled by the integration of multiple single cell technologies, provides critical insights into immune mechanisms driving cancer.



 Technological Innovations in Immunology (TECH)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d25a74164968a5001d5db1796dcb3bfa680db0f" target='_blank'>
              Multiomic analysis identifies epigenome alterations and effector-memory CD8+ T cell clonotypes associated with ALL disease progression 4323
              </a>
            </td>
          <td>
            Kelly Martin, Nandhini Raman, Connor Kunihiro, Dagmar Walter, Corey Nemec, Veronica Rodriguez, Sarah Taylor, Jill Herschleb, Peter Smibert, Josephine Lee
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancers exhibit genetic diversity between individual cancer cells. Previous work shows greater diversity than heretofore expected (1) and that also increases more quickly during a patient’s clinical course than previously thought (1, 2). Rare subclones will harbor pre-existing resistance to any single agent and may cause medium to late term relapse (1), which may evolve further variants that have simultaneous resistance to non-cross resistant therapies (3, 4). We now present a probability distribution function (PDF) of the variant allele fraction (VAF) or prevalence of a rare subclone, derived from previous evolutionary theory (1, 2). We show that current clinical sequencing protocols fail to detect the vast majority of rare subclones. By the time of detection, simultaneous multiple resistance may evolve. We then apply the PDF to simulation of dynamic precision medicine (DPM) (3), an evolutionary guided precision medicine paradigm that attempts to proactively eliminate singly-resistant subclones before they evolve multiple resistance, with significant potential to extend survival. We show that the simulated benefit of DPM with perfect information is degraded by inability to detect rare subclones if they are assumed to be absent when undetectable. But this benefit is restored if the PDF is used to calculate the likelihood of the subclone being present below the level of detection and incorporated into the DPM simulation and therapy recommendations in a probabilistic fashion. Moreover, other common statistical distributions are less effective. This theoretical advance facilitates DPM and potentially other evolutionary guided approaches to precision cancer medicine in spite of the limitations of clinical sequencing. Significance Statement Cancers contain many cells, each genetically unique. These variations can include pre-existing resistance to therapy, enabling relapse. DNA sequencing cannot detect minority cellular populations (subclones) below a certain size, by which time they may have evolved simultaneous resistance to multiple therapies. We present a mathematical approach that enables assigning a risk that a subclone is present, and at what prevalence in the cellular population, even when undetected. We simulate applying this approach to dynamic precision medicine (DPM), which attempts to proactively eliminate singly resistant cells before they become resistant to multiple therapies. Using this probability distribution, we can retain the benefit of DPM even when most rare subclones are undetectable, in contrast to just assuming undetected subclones are absent.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c00ef580d1c2e048fccb2889a7c92151af7fd5d7" target='_blank'>
              Probability Distribution for Rare Neutral Mutations in Cancers and Application to Dynamic Precision Medicine of Cancer
              </a>
            </td>
          <td>
            Wei He, Matthew D. McCoy, Chen-Hsiang Yeang, Rebecca B. Riggins, Robert A. Beckman
          </td>
          <td>2025-10-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Osteosarcoma (OS), the most common primary malignant bone tumor, remains a therapeutic challenge because of its high metastatic potential, chemoresistance, and poor prognosis. Mutations in the TP53 tumor suppressor gene, including loss-of-function (LOF) and gain-of-function (GOF) mutations, play a central role in OS progression by disrupting cell cycle regulation, DNA repair, and apoptosis and promoting immune evasion and metabolic reprogramming. This review provides an in-depth analysis of p53 biology in OS, highlighting its impact on therapeutic resistance and tumor progression. We discuss advancements in radiotherapy, chemotherapy, and immunotherapy, emphasizing strategies targeting mutant TP53 and its associated pathways. Emerging approaches, including metabolic reprogramming, noncoding RNA regulation, and precision biomarkers such as miRNAs and histone modifications, offer promising tools for diagnosis, risk stratification, and treatment optimization. By linking the molecular mechanisms of p53 with novel therapeutic strategies, this review underscores opportunities for translational research aimed at improving the clinical outcomes of OS patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4de5949ca03e6677f6b90cc0b23d966a37a92dc" target='_blank'>
              Involvement of TP53 in osteosarcoma - challenges and prospects
              </a>
            </td>
          <td>
            Yue Shen, Shuzhou Huang, Geng Chen, Guangda Wang, L. Sui
          </td>
          <td>2025-11-13</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Cellular senescence is a dynamic cancer cell condition with both tumor-suppressive and relapse-promoting features. While terminating cell proliferation, it may sustain disease persistence through its senescence-associated secretory phenotype (SASP), immune alteration, and stem-like reprogramming. Here, we identify therapy-induced senescence (TIS) as a unifying, plastic state across newly diagnosed (nd), genetically diverse non-M3 acute myeloid leukemias (AML) that reprograms blasts towards an acute promyelocytic leukemia (APL)-like phenotype with distinct plasticity-related therapeutic vulnerabilities.
 We established a short-term ex vivo chemotherapy assay to quantify the patient-individual TIS capacity of primary AML blasts by fluorescent senescence-associated β-galactosidase activity, further characterized by p16INK4a expression, Ki67 loss, and H3K9me3 heterochromatin remodeling. Like daunorubicin or cytarabin, hypomethylating agents, hydroxyurea, and anti-CD33 antibody-drug conjugates also triggered TIS to varying extents, showing that diverse agents converge on senescence and prime AML cells for potential senolytic elimination.
 Probing baseline transcriptomes of nd AML samples by an AML TIS-high signature of the top-100 differentially expressed genes failed to stratify outcomes across TCGA (n=172), OHSU Beat AML (n=405), and MLL Munich Leukemia Laboratory (n=433) cohorts. Unexpectedly, a 13-gene subclassifier, determined by unsupervised cluster analysis of the top-100 genes and unrelated to the M3-typical t(15;17) PML::RARA translocation, identified 65 of 67 genetically defined APL cases across these nearly 1,000 all-subtype AML transcriptomes, suggesting that TIS rendered non-M3 AML APL-like. Multi-omic analyses including bulk and single-cell RNA sequencing plus PRC2/SUZ12 and H3K27me3 ChIP-seq unveiled TIS-related loss of polycomb repression, epigenetic remodeling, and transcriptional reprogramming, yet independent of PML::RARA, as the underlying molecular mechanism. Functionally, TIS-associated APL-like plasticity conferred sensitivity to differentiation therapies such as all-trans retinoic acid (ATRA) and histone deacetylase inhibitors, and to Bcl2-targeting agents with senolytic activity. Patient-derived xenograft models validated these vulnerabilities as sequential TIS followed by senescence targeting achieved durable control exclusively in models able to mount a senescence response.
 Given the biologically unifying and ATRA-sensitizing role of TIS-associated M3-like plasticity, we assessed its clinical relevance in three independent cohorts totaling 92 newly diagnosed patients. TIS capacity at diagnosis robustly stratified outcomes, predicted superior disease-free and overall survival, and was associated with favorable ELN classification, normal cytogenetics, and NPM1 or DNMT3A mutations, while being lowest in adverse-risk SRSF2-mutant AML.
 Our findings uncover TIS-related M3-like plasticity as a novel state exploitable by a two-punch strategy of senesence induction first followed by targeted interventions through pro-differentiation or senolytic approaches, establishing a new therapeutic paradigm with immediate clinical potential and broader relevance to other malignancies where therapy-induced plasticity dictates outcome.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/530f7791e9ce429389bda61c51512e84812d995e" target='_blank'>
              Therapy-induced senescence reprograms non-M3 AML into a drug-exploitable APL-like state
              </a>
            </td>
          <td>
            Martin Schönlein, W. Walter, Paulina Richter-Pechanska, Katharina Barbara Rösel, Diego Gilioli, Zhiting Shao, Jana Ihlow, Karsten Spiekermann, Katharina S. S. Götze, Soyoung Lee, Yong Yu, T. Haferlach, Hugues De Thé, R. Di Micco, Clemens A. Schmitt
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da24476f2b5b20f3dabd541d60d10f29d3bbfa00" target='_blank'>
              Rapid Autopsy Multi-Omic Analysis Identifies Divergent Evolutionary Trajectories and DNA Damage Resistance Mechanisms in FGFR2-Driven Cholangiocarcinoma
              </a>
            </td>
          <td>
            A. Sheel, A. Paruchuri, J. Reeser, M. Wing, R. Vella, E. Samorodnitsky, A. Smith, T. Dao, E. L. Hoskins, R. Bonneville, , C. Li, Z. Risch, P. Allenby, A. G. Freud, A. D. Fitzthum, W. Chen, S. Roychowdhury
          </td>
          <td>2025-12-04</td>
          <td>None</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="The cellular programs that mediate therapy resistance are often important drivers of metastasis, a phenomenon that needs to be understood better to improve screening and treatment options for patients with cancer. Although this issue has been studied extensively for chemotherapy, less is known about a causal link between resistance to radiation therapy and metastasis. We investigated this problem in triple-negative breast cancer and established that radiation-resistant tumor cells have enhanced metastatic capacity. Resistance to radiation increases the expression of integrin β3 (ITGB3), which promotes enhanced migration and invasion. Bioinformatic analysis and subsequent experimentation revealed an enrichment of RNA metabolism pathways that stabilize ITGB3 transcripts. Specifically, the RNA binding protein heterogeneous nuclear ribonucleoprotein L (HNRNPL), whose expression is regulated by Nrf2, mediates the formation of circular RNAs that sponge the family of let-7 microRNAs that target ITGB3. Collectively, our findings identify a mechanism of radiation-induced metastasis that is driven by alterations in RNA metabolism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8064b7e84c02e1178a653876a56f4f95d2d1c28" target='_blank'>
              Resistance to radiation enhances metastasis by altering RNA metabolism
              </a>
            </td>
          <td>
            Ayush Kumar, Kensei Kishimoto, H. Goel, Christi Silva, Rui Li, Brendan A Pacheco, L. J. Zhu, William A. Flavahan, A. Mercurio
          </td>
          <td>2025-10-17</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="The integration of multi-omics technologies is transforming the landscape of cancer management, offering unprecedented insights into tumor biology, early diagnosis, and personalized therapy. This review provides a comprehensive overview of the current state of omics approaches, with a particular focus on the application of genomics, NMR-based metabolomics, and radiomics in non-small cell lung cancer (NSCLC). Genomics currently represents one of the most established omics technologies in oncology, as it enables the identification of genetic alterations that drive tumor initiation, progression, and therapeutic response. Interestingly, genomic analyses have revealed that many tumors harbor mutations in genes encoding metabolic enzymes, thus establishing a tight connection between genomics and tumor metabolism. In parallel, metabolomics profiling—by capturing the metabolic phenotype of tumors—has, in recent years, identified specific biomarkers associated with tumor burden, progression, and prognosis. Such findings have catalyzed growing interest in metabolomics as a complementary approach to better characterize cancer biology and discover novel diagnostic and therapeutic targets. Moreover, radiomics, through the extraction of quantitative features from standard imaging modalities, captures tumor heterogeneity and contributes predictive information on tumor biology, treatment response, and clinical outcomes. As a non-invasive and widely available technique, radiomics has the potential to support longitudinal monitoring and individualized treatment planning. Both metabolomics and radiomics, when integrated with genomic data, could support a more comprehensive understanding of NSCLC and pave the way for the development of non-invasive, predictive models and personalized therapeutic strategies. In addition, we explore the specific contributions of these technologies in enhancing clinical decision-making for lung cancer patients, with particular attention to their potential in early diagnosis, treatment selection, and real-time monitoring.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3446e3e91ac7b3e6259e583b7f7fc73de7c2244f" target='_blank'>
              Advancing Non-Small-Cell Lung Cancer Management Through Multi-Omics Integration: Insights from Genomics, Metabolomics, and Radiomics
              </a>
            </td>
          <td>
            Martina Pierri, G. Ciani, M. C. Brunese, G. Lauro, S. Terracciano, Maria Iorizzi, Valerio Nardone, M. Chini, Giuseppe Bifulco, Salvatore Cappabianca, A. Reginelli
          </td>
          <td>2025-10-01</td>
          <td>Diagnostics</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="


 Mature T-cell lymphomas (TCL) are a heterogeneous group of non-Hodgkin lymphomas arising from the malignant transformation of post-thymic T and NK cells. These lymphomas are characterized by poor response to chemotherapy, high relapse rates, and dismal survival outcomes. Genomic analyses of TCL have revealed recurrent alterations affecting intracellular signaling pathways and epigenetic regulators. The frequent involvement of epigenetic modifiers suggests that epigenetic dysregulation is a critical step in TCL pathogenesis.
 To identify the epigenetic changes driving TCL transformation, we analyzed chromatin modifications and gene expression profiles in murine TCL models that closely recapitulate human disease. Our data revealed that Id2, a repressor of basic helix-loop-helix proteins, is consistently overexpressed at both RNA and protein levels across multiple murine lymphoma models. Supporting this, analysis of the Cancer Cell Encyclopedia identified TCL as one of the tumor types with the highest ID2 expression, which we confirmed by Western blot in human TCL-derived cell lines. Furthermore, analysis of single-nuclei RNAseq data from a cohort of primary angioimmunoblastic T-cell lymphoma (AITL) and peripheral T-cell lymphoma not otherwise specified (PTCL, NOS) patient samples generated in our laboratory, demonstrated significant ID2 overexpression in a subset of cases (12/27; 44%) compared to tonsil and lymph node controls. Interestingly, all relapsed TCL cases in our cohort showed significant overexpression of ID2. Based on our data, we hypothesize that ID2 is epigenetically regulated during transformation and contributes to T-cell lymphomagenesis.
 To define the mechanisms leading to ID2 overexpression in TCL, we performed H3K27ac and H3K4me1 ChIP-seq, ATAC-seq, reverse-ChIP coupled with mass spectrometry, and dual-luciferase reporter assays. These analyses identified a conserved region near the ID2 locus enriched for H3K27ac and H3K4me1 marks that co-localizes with ATAC-seq peaks and is ranked as a super-enhancer by the ROSE algorithm. This region is specifically activated in mature TCL cells compared to control CD4⁺ T cells. Reporter assays using the most conserved segment within the enhancer region showed more than twofold increased luciferase activity relative to a minimal promoter, confirming its enhancer function. Interestingly, reverse-ChIP identified CTCF and ASCL1 as candidate regulators of enhancer looping and transcriptional activation, respectively. To characterize the proteins mediating the epigenetic function of ID2 in TCL, we employed a biotin ligase proximity labeling assay (BioID) approach coupled with mass spectrometry. Our results identified known ID2 partners, including the E-proteins TCF3, TCF4 and TCF12, key regulators of T-cell differentiation, as well as novel partners including elements of the SWI/SNF chromatin-remodeling complex, suggesting that ID2 overexpression indirectly regulates transcriptional programs critical for lymphomagenesis.
 To determine the functional role of ID2 in TCL, we characterized the effects of both genetic and pharmacological inhibition of ID2, using shRNA and CRISPR-Cas9 mediated knockout and novel pan-ID inhibitors AGX51 and AGXA, respectively. Loss of ID2 leads to decreased cell viability and increased apoptosis in in vitro TCL models. Interestingly, RNA-seq analysis revealed enrichment in signatures associated with glucose metabolism, suggesting a novel role for ID2 in tumor metabolic regulation. Finally, in vivo treatment with the pan-ID inhibitor AGX51 in a murine lymphoma model induced a modest but consistent reduction in tumor burden, spleen weight, and activated CD69⁺ T cells population in AGX51-treated mice compared to vehicle-treated controls.
 In summary, we have identified a novel TCL-specific super-enhancer that drives ID2 overexpression in TCL and demonstrated an essential role for ID2 in supporting lymphoma proliferation and survival. More importantly, pharmacological targeting of ID2 reduces tumor burden in vitro and in vivo, highlighting its potential as a therapeutic target in mature T-cell lymphomas.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e31f2dc0fbeb09da6f80dbe4eff065e3af3490a0" target='_blank'>
              Inhibitor of DNA binding 2 (ID2) is epigenetically regulated during lymphomagenesis and constitutes a novel therapeutic target in mature T-cell lymphoma
              </a>
            </td>
          <td>
            Anouchka P. Laurent, Hannah Miller, Ryan D. Najac, R. Albero, Jose R. Cortés, Cindy Ma, W. Lin, Anqi Wang, Ruth Alonso Alonso, Laura Quevedo, Rajesh Soni, O. Ouerfelli, Raul Rabadan, R. Benezra, Teresa Palomero
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Platinum-based drugs (Pt drugs) are widely used in cancer chemotherapy, yet their genome-wide binding patterns remain incompletely understood. Here, we present Pt sequencing (Pt-seq), an antibody-assisted, genome-wide method for mapping Pt-DNA adducts at single-base resolution. By using exo- and endonucleases to remove background, Pt-seq enables robust and sensitive profiling of binding sites for cisplatin, oxaliplatin, lobaplatin, and a Pt(IV) complex. Using Pt-seq, we identified tens to hundreds of binding clusters that are 10 to 20 kilobases in median length and highly consistent among different drugs, predominantly localizing to centromeric and ribosomal DNA regions. In cisplatin-resistant cells, we found significantly reduced binding within these regions. Moreover, we found that mutations in cancer cells can generate previously unidentified binding sites. On this basis, we demonstrated that ICR-191, an acridine orange compound capable of inducing G insertions, enhances cisplatin-DNA binding and sensitizes cells to cisplatin. Collectively, Pt-seq sensitively profiles Pt–DNA interactions and deepens our understanding of the genome-wide effect of chemotherapeutic drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b8b8af89d2820c69ce2eeb05e85318ac412b23f" target='_blank'>
              Pt-seq unveils the genomic binding pattern of platinum-based drugs
              </a>
            </td>
          <td>
            Chuyun Shao, Mingyang Li, Liuyi Liu, Dongsheng Bai, Jinmin Yang, Cong Liu, Linxiang Cai, Xiaoyong Wang, Jinying Peng, Zijian Guo, C. Yi
          </td>
          <td>2025-10-29</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="High-throughput sequencing (HTS) has revolutionized tumor immunology by enabling precise dissection of tumor-immune interactions, directly informing the development of precision immunotherapies. This review highlights key advances in HTS technologies—including whole genome sequencing (WGS), RNA sequencing, assay for transposase-accessible chromatin using sequencing, and single-cell immunogenomics (scTCR-seq/scBCR-seq)—and their clinical translation in personalized cancer vaccines, engineered T-cell therapies, and combination regimens. We discuss how these tools decode tumor-specific mutations, immune evasion mechanisms, and therapeutic targets, while addressing challenges in data standardization, sample processing, and computational integration. Emerging breakthroughs such as spatial multiomics, real-time monitoring, and artificial intelligence-driven discovery are transforming the field by enabling dynamic, personalized treatment strategies. Finally, we outline future directions to overcome current barriers and expand equitable access to HTS-driven precision immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1666fd68dbba4d54d4ecfda7e481cfc47c3e80e1" target='_blank'>
              High-throughput sequencing unveils tumor-immune interactions: From genomic alterations to clinical translation
              </a>
            </td>
          <td>
            Ling Yin
          </td>
          <td>2025-11-07</td>
          <td>Global Translational Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The MRE11 DNA nuclease plays central roles in the repair of DNA double-strand breaks (DSBs) as a core component of the MRE11-RAD50-NBS1 (MRN) complex. MRN localizes to chromosomal DSBs and recruits and activates the DSB repair protein kinase, ATM, which phosphorylates downstream substrates to elicit cellular DNA damage responses. Pathogenic variants in MRE11 cause the genome instability disorder ataxia-telangiectasia-like disorder (ATLD). The first ATLD patient allele identified, ATLD1, is a nonsense mutation that deletes 76 residues from the MRE11 C-terminus and markedly reduces levels of MRE11-ATLD1 and the entire MRN complex. The MRE11 C-terminus has been demonstrated to function in DNA binding, mediate protein interactions, and undergo post-translational modifications that regulate the MRE11 nuclease. We previously demonstrated that transgenic mice expressing reduced wildtype MRN levels exhibit severe phenotypes, including small body size, anemia, and DNA DSB repair defects. Thus, it is currently unknown whether low MRE11-ATLD1 levels, loss of the C-terminus, or both cause disease-associated phenotypes. In this study, we generated transgenic mouse models that express near endogenous or significantly reduced levels of MRE11-ATLD1 to determine the in vivo importance of the MRE11 C-terminus. We observe that low MRE11-ATLD1 expression leads to anemia, bone marrow failure, extramedullary hematopoiesis, and impaired lymphocyte development, similar to mice expressing low wildtype MRE11. In contrast, higher MRE11-ATLD1 expression results in a subset of moderate phenotypes, indicating that loss of the C-terminus has limited impact on MRN functions in vivo. These findings provide a foundation for predicting the clinical presentation and severity of ATLD patient phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10803b54e0a6ebf78334cad9c5698be48d5f5b38" target='_blank'>
              Differential expression of a disease-associated MRE11 variant reveals distinct phenotypic outcomes.
              </a>
            </td>
          <td>
            McKenna B DeFoer, Ahmed M Mostafa, A. Hartlerode, Steven K Orban, Keegan McDonough, Sophie Quirk, Brianna K L Ferguson, David O Ferguson, JoAnn M Sekiguchi
          </td>
          <td>2025-10-11</td>
          <td>Human molecular genetics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Genomic analysis of multiple myeloma has advanced greatly in recent years, but traditional short-read sequencing has limited the power to examine complex or linked events across the genome. Linking markers together into large phase-blocks, and examining the interaction of different sequencing modalities, allows complex genomic and epigenomic states to be integrated to generate true multiomic haplotypes at the chromosomal level.



 Fourteen multiple myeloma patient-derived xenografts were used to generate multiomic data including short- (Illumina) and long-read (PacBio) whole genome sequencing (WGS, average depths 92x and 16x) to identify single nucleotide variations (SNVs), copy number (CN) abnormalities, structural variation (SV), and DNA methylation, as well as expression (>124 million reads), chromatin states (Cut&Tag-IT, >10M reads, H3K27ac, H3K4me1, H3K4me3, H3K9me3, H3K27me3, and H3K36me3), and Micro-C/LinkPrep (Dovetail Genomics; >900M reads) and HiChIP (Dovetail, H3K27ac mark) to identify 3D chromatin architecture/folding and enhancer-target gene interactions. Micro-C/LinkPrep interaction reads were also used for chromosome-scale haplotype phasing, onto which SNVs, SVs, CNVs, DNA methylation, expression, and chromatin marks were super-imposed based on heterozygous SNPs. Data integration resulted in haplotype-phased somatic events for genomic architecture, SNVs, SVs, epigenomic states, and expression markers.



 For the first time, complete haplotype-resolved assemblies at the chromosomal level have been generated in multiple myeloma. A 100x coverage genome Micro-C/LinkPrep libraries we were able to phase the long and short arms of each chromosome, resulting in an average 96.3% haplotype phasing per autosome – thereby generating chromosome length haplotypes up to 241.9 Mb in length. In comparison, HiFi long-read (15 kb reads) WGS (PacBio) was able to generate phase-blocks of up to 4.6 Mb (median 44.1 kb).
 We examined the prevalence of somatic mutations on each haplotype across all chromosomes to determine if one parental haplotype was more likely to be mutated than the other. In one sample, of 19,306 mutations 56% were phased to their respective haplotypes and were generally equally distributed across both haplotypes.
 Haplotype-specific interaction heatmaps were generated allowing us to examine the interaction of complex SVs across chromosomes. In one sample, we identified a primary t(11;14) and additional SV events linked to the t(11;14) including a t(3;14), t(11;17), and t(3;17) which created a cyclical pattern. Using the chromosome-scale haplotype maps we were able to determine the proportion of interacting reads for the four possible haplotype combinations between any two chromosomes. We found that 70-83.5% of reads support specific combinations of haplotype interactions, and that all four SV events were linked and involved the same haplotypes on each chromosome. The pattern of interactions combined with breakpoint analysis in this case indicated a four-way complex reciprocal translocation between chromosomes 3,11,14 and 17.
 Integration of epigenetic data in this complex SV, including DNA methylation (HiFi reads) and super-enhancer histone marks (CUT&Tag-IT), showed DNA hyper-methylation 17 kb upstream of CCND1 next to the t(11;14) breakpoint as well as increased H3K27ac marks on the same haplotype, indicating spreading of the activating broad domain from the IGH super-enhancer on chromosome 14. Equally, the hypomethylated DNA marks at the IGH promoter are spread to chromosome 3, via the t(3;14), resulting in over-expression of the proto-oncogene SKIL. The same is true for the t(3;17), which shows hypomethylation on both sides of the breakpoint, compared to the non-translocated allele. Overall, this exemplifies the underlying intricacy and impact of complex SVs across multiple linked chromosomes and their epigenetic states.



 We have generated the first chromosome scale haplotype-resolved genomes in multiple myeloma and integrated them with epigenetic states. We have shown that it is possible to identify interactions across chromosomes to resolve complex SVs as well as their epigenomic consequences to understand the intricate nature of how the genome is organized.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e6488520a62edae9da4095a747986d863374bdb" target='_blank'>
              Multiomic evidence of coordinated complex rearrangements, enhancer hijacking, and epigenomic signatures in the first whole-chromosome-phased myeloma genomes
              </a>
            </td>
          <td>
            Nathan Becker, Enze Liu, Zackary Sanborn, A. Suvannasankha, Kelvin Lee, D. Kazandjian, James Hoffman, B. Diamond, Abhishek Pandey, R. Abonour, O. Landgren, Lisa Munding, Aneta Mikulasova, Brian A. Walker
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e7ea627495d26a3f7b3d82b351b84bb5f1b7889" target='_blank'>
              Genomics of Acute Myeloid Leukemia at Diagnosis and Remission
              </a>
            </td>
          <td>
            , L. Dillon, J. Tettero, G. Gui, R. Al-Ali, M. Grunwald, E. Krakow, E. Griffiths, A. Gomez-Arteaga, R. Vedula, M. Solh, A. Salhotra, N. Bejanyan, L. Muffly, A. Jimenez-Jimenez, M. Drazer, Y.-B. Chen, A. Logan, R. Jayani-Kosarzycki, S. Balderman, J. Blachly, B. Shaffer, L. Druhan, C. Yeung, V. Kennedy, A. T. Fathi, H. Carraway, S. Gurbuxani, M. Tjota, F. Sahoo, , D. Barfield, J. Guo, J. Han, J. Hu, H. Jo, V. Kudlingar, , , P. Sathyan, S. Truong, S. Catreux, , K. Luong, Y. Zhu
          </td>
          <td>2025-12-07</td>
          <td>None</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Understanding therapy resistance requires deconvolving heterogeneous cell populations and tracking clonal trajectories. While CRISPR-based cellular barcoding is powerful for lineage tracing, many platforms suffer from low efficiency and limited compatibility with single-cell transcriptomics. We developed Oligo-CALL (Oligonucleotide-inducible CRISPR transcriptional activator–Assisted Lineage Labeling), an advanced barcoding system enabling precise lineage tracing, live clone isolation, and seamless integration with single-cell RNA sequencing. Applied to lung cancer cells treated with a KRASG12C inhibitor, Oligo-CALL identified clones consistently enriched posttreatment, supporting a model of predestined resistance. Oligo-CALL achieved >95% efficiency in linking lineage identity to transcriptomes, uncovering diverse clone-specific pathways with underlying resistance. Paired analysis of barcode-matched clones from naïve and resistant populations revealed transient and fixed resistance phenotypes. Notably, DNA repair pathways are recurrently altered in resistant clones, and inhibition of poly(adenosine 5′-diphosphate–ribose) polymerase synergizes with KRAS G12C inhibition to overcome resistance. Together, Oligo-CALL provides a versatile platform for dissecting lineage evolution and molecular dynamics of targeted therapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fc6bc5d7c1ef3b43ef612b4e6dbe8df9b69936e" target='_blank'>
              Oligo-CALL: A next-generation barcoding platform for studying resistance to targeted therapy
              </a>
            </td>
          <td>
            Yingzhuo Liu, Yi Ban, Dingcheng Gao
          </td>
          <td>2025-11-07</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 B-cell acute lymphoblastic leukemia (B-ALL) is the most common pediatric hematological malignancy. While advances in modern medicine have significantly improved survival – especially in children– treatment resistance, relapse, and refractory disease remain major challenges, highlighting the complexities in treating B-ALL and the need for ongoing research and development of novel therapeutic strategies.
 Our lab previously demonstrated that the SWI/SNF chromatin remodeling complex subunit SMARCD1 is essential for the development of the lymphoid lineage, as its inactivation in the adult hematopoietic system leads to lymphopenia while sparing the myeloid and erythroid lineages (Priam et al., Developmental Cell, 2024). In this study, we identify SMARCD1 as an essential gene in B-ALL. Specifically, SMARCD1 inactivation in B-ALL cell lines and patient-derived xenograft (PDX) cells results in profound defects in cellular proliferation and survival. However, the molecular mechanisms underlying its oncogenic role in this context remain to be elucidated.
 The SMARCD1 subunit does not directly bind DNA or chromatin; instead, it is recruited to specific genomic loci through interactions with various co-factors, including the tumor suppressor protein p53. Interestingly, although p53 is somatically mutated in over half of human cancers, it remains wild-type (WTp53) in approximately 75% of newly diagnosed adult ALL cases and 97–98% of pediatric ALL cases at diagnosis (Stengel et al., Blood, 2014, Leukemia, 2017). Given that genetic instability is a hallmark of cancer, understanding why some malignancies retain WTp53 despite its tumor-suppressive functions is a key question with important therapeutic implications. Emerging evidence suggests WTp53 may promote oncogenic functions in specific cancer contexts by regulating metabolic plasticity and pro-survival signaling pathways.
 In B-ALL cell lines, we confirmed a biochemical interaction between SMARCD1 and WTp53 using coimmunoprecipitation (Co-IP) and bimolecular fluorescence complementation (BiFC). We further demonstrated that the SMARCD1-dependent B-ALL cell lines are also reliant on WTp53, as p53 inactivation impairs proliferation and survival to a similar extent as SMARCD1 depletion. Notably, co-depletion of both proteins did not rescue the proliferative defect, suggesting a cooperative role in driving leukemia progression. Importantly, this dependency was not observed in p53-proficient non-hematopoietic cell lines, supporting a context-specific oncogenic role for this SMARCD1-WTp53 axis in B-ALL.
 To explore how the SMARCD1-WTp53 axis drives B-ALL progression, we performed bulk RNA sequencing of B-ALL cells depleted of either SMARCD1 or TP53. Differential expression and gene set enrichment analyses (GSEA) revealed overlapping downregulation of pathways linked to proliferation and metabolism, including hypoxia, fatty acid metabolism, glycolysis, and mTORC1 signaling. Notably, key metabolic enzymes and regulators - such as GLUT4/SLC2A4, HK2, and SDHA- were consistently downregulated upon loss of either SMARCD1 or TP53. These transcriptomic changes were supported by Seahorse XF metabolic flux analyses, which confirmed reduced metabolic activity in SMARCD1- and TP53-depleted B-ALL cells. Cleavage Under Targets and Tagmentation (CUT&Tag) analyses further showed that SMARCD1 and WTp53 predominantly co-occupy the promoter regions of common target genes. Ongoing studies aim to elucidate their cooperative roles in modulating chromatin accessibility and promoter/enhancer function at these loci.
 Collectively, our findings identify the SMARCD1-WTp53 axis as a key regulator of metabolic programming in B-ALL, supporting a high metabolic state and rapid proliferation through control of a distinct oncogenic transcriptional program. This work has important therapeutic implications, highlighting the potential of targeting chromatin-metabolism crosstalk in B-ALL via disruption of the SMARCD1-WTp53 axis. Furthermore, this study reveals a context-dependent oncogenic role for WTp53, emphasizing the need for precision therapies tailored to p53 status - whether by activating WTp53 or restoring its function in p53-mutant cancers.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/969dc91cb92875d724cae066a989cc5ef6e8a223" target='_blank'>
              SMARCD1 subunit of SWI/SNF chromatin remodeling complexes collaborates with p53 to exert an oncogenic role in B-ALL by maintaining high metabolic activity
              </a>
            </td>
          <td>
            Laurence Côté, Pierre Priam, Alexandre Polsinelli, Hugo Philippeau, Gabriel Alzial, Geneviève Deblois, Julie A. Lessard
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) progression and recurrence are often driven by therapy-resistant cellular states emerging within hypoxic tumor niches. While hypoxia is a hallmark of GBM, the spatial transcriptomic heterogeneity that underpins resistance in these regions remains poorly characterized. Here, we employ pimonidazole labeling in conjunction with high-resolution spatial and single cell transcriptomics across 29 primary, treatment-naive human GBM samples, syngeneic mouse models, and patient-derived glioma-initiating cells exposed to hypoxia to define the hypoxia-specific tumor microenvironment and its associated cellular programs. We reveal spatially organized sub-niches within hypoxic regions, each supporting diverse cellular programs yet coexisting tumor lineage states with minimal transitions between cell states. Phylogenetic reconstruction based on inferred copy number variations demonstrates consistent evolutionary trajectories from non-hypoxic to hypoxic tumor loci. Specific copy number alterations were enriched in hypoxic regions and associated with immune infiltration and poor prognosis in MGMT-methylated tumors. Cross-species analyses confirmed the conservation of hypoxia-associated gene programs and spatial architectures. Together, our study delineates the cellular, evolutionary, and spatial complexity of hypoxia in GBM, identifies hypoxia-induced genomic alterations that may promote tumor fitness, and offers insights into new therapeutic vulnerabilities within hypoxic niches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddec812f9197d64f5c26991c527a5afc02eb1748" target='_blank'>
              TMIC-96. Integrated multi-omics reveals distinct hypoxic sub-niches and associated epigenomic and genomic alterations in glioblastoma
              </a>
            </td>
          <td>
            S. Mansouri, Jefferey Liu, Vikas Patil, Ruth Isserlin, Phoebe Lombard, Ronald Wu, Julio Sosa, S. Gandhi, Y. Ellenbogen, O. Singh, Hafsah Ali, C. Velásquez, Bradly Wouters, G. Zadeh
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="


 Chromosomal translocations are key genomic events that frequently occur in hematopoietic cancers and solid tumors. In lymphoma and leukemia, such as diffuse large B cell lymphoma (DLBCL) and follicular lymphoma (FL), most DSBs that lead to oncogenic translocation are initiated by activation-induced cytidine deaminase (AID). AID is a B-cell-specific enzyme that targets immunoglobulin (Ig) genes, considered AID on-target genes, to initiate somatic hypermutation (SHM) and class switch recombination (CSR). However, AID also exhibits off-target activity at non-immunoglobulin regions, contributing to genomic instability by promoting oncogenic chromosomal translocations and mutations that drive the development and progression of B cell lymphomas and leukemias. The mechanisms that govern AID's selective targeting of a limited subset of genomic regions remain poorly understood and represent a longstanding, fundamental question in the field. By utilizing high-throughput genome-wide translocation sequencing (HTGTS) in B cells, we previously demonstrated that PI3Kδ inhibition upregulates AID expression, thereby increasing genomic instability and partially elucidating these mechanisms.
 To further delineate the mechanistic cascade of AID activity as a cytidine deaminase, we now developed dU-seq to map genome-wide AID-induced mutations and applied sBLISS to detect genome-wide AID-induced DNA double-strand breaks (DSBs) in both primary B cells and CH12F3 cells. By these approaches, we successfully captured AID-mediated mutations and AID-dependent DSBs at most known translocation hotspots in the whole genome, including AID on-targets and off-targets, validating the technical robustness of our approach. Surprisingly, we identified thousands of previously unrecognized AID-mediated mutations and DSB hotspots besides those known AID target regions, revealing a much broader genomic footprint of AID activity than previously appreciated.
 To provide a potential clinical relevance of these discovery approaches, we tested whether emerging epigenetic therapies could impact the patterns and distribution of AID-mediated translocations in B cell lymphoma. EZH2 inhibitors, such as tazemetostat and valemetostat, are epigenetic regulators approved by the FDA for the treatment of follicular lymphoma and adult T-cell leukemia/lymphoma, respectively. We found that EZH2 inhibitors alone did not significantly alter AID expression or AID-mediated translocation frequency in CH12F3 mouse B cells or MEC-1 human B cells. However, when combined with PI3Kδ inhibition, EZH2 inhibition markedly enhanced the frequency of chromosomal translocations compared to either treatment alone in both cell models. EZH2 inhibition also further enhanced translocation formation in mouse B cells that were DNA repair deficient, such as Ligase4 knock-out cells. Mechanistically, EZH2 inhibition in B cells depletes the repressive histone modification H3K27me3 while concurrently enhancing the active histone modification H3K27ac, thereby selectively increasing transcriptional activity and facilitating chromosomal translocation formation in the presence of high AID activity or Ligase4 deficiency.
 Overall, to our knowledge this work represents the most comprehensive mapping of AID-induced mutational and genotoxic activity, shedding light on the whole trajectory of AID activity from the very early initiation steps of cytidine deamination to the formation of DSB intermediates up to the final outcome of chromosomal translocations. The described approach can be exploited to functionally dissect the impact of novel drugs on AID-mediated genomic instability in B cell lymphoma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/822e3be74360b79fb622284e6b4772e766b81ff1" target='_blank'>
              Dissecting the whole trajectory of aid-induced genomic instability from mutational activity to chromosomal translocation formation
              </a>
            </td>
          <td>
            Jianli Tao, Luca Alessandri, Beatrice Nuvolari, Fabio Iannelli, Roberto Chiarle
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Metabolic reprogramming has emerged as a defining hallmark of cancer, reflecting the capacity of malignant cells to restructure bioenergetic and biosynthetic pathways in response to genetic alterations and microenvironmental pressures. Tumor cells exploit metabolic plasticity to sustain proliferation, maintain redox homeostasis, and adapt to nutrient fluctuations or hypoxia. Glucose metabolism is frequently diverted toward aerobic glycolysis, yielding both ATP and intermediates essential for macromolecule synthesis. In parallel, glutamine catabolism replenishes tricarboxylic acid cycle intermediates and supports nucleotide production, whereas lipid and amino acid metabolism undergo extensive remodeling to provide structural components and signaling molecules that promote tumor growth. Such multifaceted alterations integrate with oncogenic signaling networks, creating a finely tuned metabolic landscape that underlies malignancy. Metabolic reprogramming extends beyond intrinsic tumor processes to reshape the tumor microenvironment (TME). Excessive consumption of glucose and glutamine by cancer cells restricts nutrient access for effector immune populations, fostering immune suppression. Accumulated metabolites such as lactate, kynurenine, and adenosine acidify the extracellular milieu, polarize macrophages toward tumor-supportive phenotypes, and dampen cytotoxic T-cell activity. Cancer-associated fibroblasts further participate through metabolic symbiosis, providing energy-rich substrates that sustain tumor oxidative metabolism and angiogenesis. Hypoxia-driven stabilization of hypoxia-inducible factors reinforces glycolytic flux and vascular remodeling, perpetuating a self-sustaining cycle of metabolic and microenvironmental adaptation. Understanding these metabolic interactions can provide a deeper understanding of cancer pathophysiology and may also reveal exploitable vulnerabilities. Therapeutic strategies targeting metabolic enzymes or normalizing nutrient fluxes represent promising avenues for disrupting tumor–stroma cooperation and enhancing treatment efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97bfc00b58584c90f101c9f638dcfecf915deca1" target='_blank'>
              Metabolic reprogramming in cancer: Mechanisms and microenvironmental interactions
              </a>
            </td>
          <td>
            Miriam Goldberg, Iris Callahan
          </td>
          <td>2025-10-15</td>
          <td>Clinical and translational reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Despite recent advances in targeted chemotherapies for acute myeloid leukemia (AML), treatment of relapsed and refractory AML remains a significant challenge. Overexpression or rearrangement of the Myelodysplasia Syndrome (MDS) and Ecotropic Virus Integration Site 1 (EVI1) Complex locus (MECOM) is associated with particularly poor prognoses. High MECOM expression establishes a population of leukemia stem cells (LSCs) that have increased resistance to chemotherapy. Prior data from our group revealed a MECOM associated gene regulatory network that is essential in hematopoietic stem cell self-renewal and function, which may represent a therapeutic vulnerability in high-risk AML. Here, we investigated the association of the MECOM transcriptional network with chemoresistance in multiple AML models and found that perturbation of MECOM increased sensitivity to standard chemotherapies independent of cell state changes.



 To probe the interaction of MECOM with chemosensitivity, we engineered multiple high MECOM expressing human AML cell lines by targeting the endogenous MECOM locus with a FKBP12F36V degron that enabled rapid and complete MECOM protein degradation with exposure to the small molecule dTAG. MUTZ-3, an inv(3) AML cell line, was the primary system used as it can expand indefinitely and has a subpopulation of CD34+ progenitors that either self-renew or differentiate into CD14+ monocytes. Findings were later verified in the UCSD-AML-1 and HNT-34 cell lines. Using these models, we determined differential chemosensitivity of AML cells following MECOM protein degradation compared to control cells by characterizing phenotypic properties and differential gene expression profiles with etoposide and doxorubicin treatment, common backbones in current AML treatment regimens.
 In MUTZ-3 cells, etoposide treatment increased the CD34+ stem cell progenitor population from 59.5% to 67% compared to the control (p<0.05, n=3). CD34+ MUTZ-3 LSCs are less likely to undergo apoptosis following etoposide treatment (5.5%, n=3) compared to more differentiated CD14+ cells (60.6%, n=3), demonstrating that etoposide selects for LSCs. Hypothesizing that some of this chemoresistance of LSCs was driven by MECOM, we determined IC50 values of etoposide and doxorubicin after MECOM degradation. MECOM degradation led to a 2.2-fold, 1.6-fold, and 1.7-fold decrease in etoposide IC50 respectively in MUTZ-3, UCSD-AML-1, and HNT-34 cells (p<0.05, n=3). Similarly, MECOM degradation led to a 1.5-fold, 1.8-fold, and 1.5-fold decrease in doxorubicin IC50 in MUTZ-3, UCSD-AML-1, and HNT-34 cell lines (p<0.05, n=3), which indicates that MECOM degradation sensitizes AML cells to etoposide and doxorubicin.
 To examine the transcriptional signatures associated with chemotherapy treatment and MECOM degradation, we performed bulk RNA sequencing with the CD34+ subpopulation of MUTZ-3 cells. We observed significant enrichment of the MECOM gene network following treatment with etoposide (NES =1.81, p <0.001) and doxorubicin (NES=1.91, p <0.001) that is reversed with degradation of MECOM with both etoposide (NES = -2.610, p<0.001) and doxorubicin treatment (NES= -2.634, p<0.001).
 To determine the relevance of this MECOM gene set in primary AMLs, we evaluated RNA sequencing data from adult patients in the BEAT-AML data set. We compared samples at diagnosis (n=450) and relapse (n=37) and found enrichment of the MECOM gene network in relapsed samples compared to diagnostic samples (NES = 1.9, p <0.001). We additionally examined diagnostic samples from patients that were later found to have a complete (n=220) or refractory (n=103) response to first-line chemotherapy and found an enrichment of the MECOM gene network in patients with a refractory response (NES = 1.7, p <0.002).



 In summary, the MECOM gene network is associated with etoposide and doxorubicin resistance in AML human cell models, suggesting that chemotherapy resistance is not only mediated by cell state changes, but is also modified by transcriptional changes in LSCs. Additionally, this gene set is enriched in relapsed and refractory primary AML patient samples. Further study of the mechanisms underlying the observed chemotherapy resistance and ways to overcome it are desperately needed for the development of newer and more effective therapies for high-risk AML.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfccb6c8ddff1c95d1884bc510e150bea0e7adef" target='_blank'>
              Mecom gene network drives chemotherapy resistance in high-risk acute myeloid leukemia
              </a>
            </td>
          <td>
            Sanjana Shah, Travis J. Fleming, Richard Voit, V. Sankaran
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 The RAS/MAPK signaling pathway is hyperactivated in acute myeloid leukemia (AML). RAS/MAPK signaling converges on the extracellular signal-regulated kinase (ERK) which in turn promotes the transcription of immediate-early genes (IEGs) that mediate blast proliferation and survival. Temuterkib is an orally bioavailable small molecule inhibitor of ERK1/2. In preclinical studies, temuterkib downregulated ERK targets in AML cell lines and caused growth arrest (Weisberg et al., 2020), but itseffects in human leukemia in vivo have not been defined. We therefore performed multiomic studies in primary AML blasts from patients treated with temuterkib to characterize the acute and adaptive responses to ERK inhibition and identify potential mechanisms of intrinsic resistance.



 We performed a phase 1 dose escalation trial of temuterkib in patients with R/R AML using continuous daily dosing (NCT04081259). We evaluated bone marrow blast response and pre/post-treatment genetics using DNA sequencing. Blood samples were collected pre-dose; 1, 4, 8, and 24 hours after first dose, and weekly thereafter. Blasts were purified for bulk and single-cell RNA-seq with CITE-seq targeting 330 surface and intracellular proteins and 28 phospho-proteins to assess transcriptional and signaling changes over time.



 We enrolled 17 patients over 3 dose levels (3 at 200mg, 3 at 100mg with concurrent azole, 11 at 400mg). There were no dose-limiting toxicities. Ten patients had TP53 mutations, 7 had AML-MR mutations and 9 had RAS pathway mutations (FLT3, NRAS, KRAS, CBL, PTPN11). Among 15 patients evaluable for response, 7 had progressive disease (PD) and 8 had stable disease (SD) per IWG criteria. Median time on study was 32 days (range 15-121) with disease progression as the primary reason for discontinuation. Among patients treated with 200mg and 400mg, temuterkib reached peak concentrations by 4 hours and showed on-target inhibition, reflected by decreased ERK target gene expression.
 We evaluated baseline DNA and RNA-sequencing to define pre-treatment differences between patients. Baseline genotype (TP53, AML-MR, RAS mutations) did not correlate with measures of clinical activity, including PD vs. SD or time on study. We compared RAS mutant (n=9) vs. WT (n=8) leukemias and identified 300 differentially expressed genes (>1.5 fold change, padj <0.05). Sixty-two genes with greater expression in RAS mutant leukemias showed enrichment for peptide biosynthetic processes (GO:0043043) and TORC1 signaling (GO:0038242).
 We performed serial DNA and RNA-seq to assess genetic and transcriptional responses to ERK inhibition despite lack of clinical response. Paired pre/post treatment NGS showed no subclonal expansion or contraction, including of RAS mutations. Due to high inter-patient heterogeneity at baseline, we analyzed longitudinal changes in global transcription in each patient separately. ERK dependent IEGs (EGR1, JUN, and FOS) and negative feedback regulators (DUSP1, DUSP6, and SPRY1) were among the most downregulated transcripts at 4 and 8 hours across all patients. However, in RAS mutant leukemias, this effect was dampened with a lower magnitude of EGR1 (mean log2fc -0.9 vs. -2.2; p=0.07) and DUSP6 decrease (mean log2fc -0.5 vs. -1.2; p=0.04) at 4 hours. MYC expression decreased by 20% at 4 hours post dose, returned to baseline by 8 hours, and increased to 54% greater than baseline by day 15. This rebound was independent of RAS mutation status.
 To compare the acute and adaptive response to ERK inhibition, we performed longitudinal single-cell RNA-seq with CITE-seq on samples from 2 patients treated at 400mg. In a patient with NRAS and TP53 mutations, we found an acute increase in JAK-STAT signaling proteins (pSTAT5A, pSTAT5B) at 4 and 8 hours, followed by a sustained increase above baseline at day 15. In another patient with FLT3-TKD/ITD and RUNX1 mutations we observed an acute increase in MTOR signaling proteins (AKT1, pMTOR, pRPS6KA1). These findings indicate that compensatory activation of alternative signaling pathways occurs rapidly in response to ERK inhibition.Conclusions: Temuterkib produced acute on-target ERK suppression without objective clinical responses in R/R AML regardless of RAS mutation status. RAS mutations did not confer increased sensitivity to ERK inhibition and may instead attenuate downstream suppression. Rapid induction of JAK-STAT and MTOR signaling may underlie resistance to ERK inhibition in AML.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c49e162531d90421db2f4f88aa7897b4f93832a" target='_blank'>
              Temporal multiomic profiling reveals intrinsic resistance mechanisms to ERK inhibition in relapsed Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            R. Vedula, Hannah Karp, Danielle Morrow, Andrew Gehrke, Yiwen Liu, Ellen Weisberg, Carter Lombardi, Rebecca Leonard, I. Galinsky, Jacqueline Garcia, Marlise R. Luskin, Maximilian F Stahl, M. Wadleigh, Eric S. Winer, D. DeAngelo, James Griffin, Richard M. Stone, R. C. Lindsley
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="


 High-grade serous ovarian carcinoma (HGSC) is a leading cause of gynecologic cancer death, driven by frequent recurrence and resistance to platinum therapies. Uncovering the mechanisms behind this process requires integrating genomic and transcriptional changes in space and time. Here, we present an evolutionary analysis combining multisite phylogenetic trees with transcriptomics, focusing on copy-number (CN) and structural variation (SV) to identify events that shape HGSC progression and treatment resistance. To address the changes during chemotherapy, we first reconstructed tumor phylogenies from 236 whole-genome–sequenced samples derived from 60 patients with relapsed HGSC using PyClone. Patients were stratified according to changes in mutational burden between diagnosis and relapse, as well as by clonal selection patterns, distinguishing cases of monoclonal versus multiclonal relapses. To extend the analysis beyond point mutations, CN and SV profiles were decomposed into clonal profiles within the phylogenetic framework, using a customized implementation of ALPACA. In parallel, matched bulk RNA-seq was integrated to connect relapse-specific genomic alterations with transcriptional changes and pathway deregulation, thereby uncovering pathways and processes underlying chemotherapy resistance. We observed that patients with a single dominant clone at relapse had significantly worse post-relapse survival compared with those exhibiting intra-sample clonal heterogeneity (p = 0.015). Across the cohort, clonal complexity, quantified as the geometric mean of within-sample subclonal heterogeneity per timepoint, showed a marked decline at relapse (p < 0.001). Since this cancer is driven by chromosomal instability, we next characterized CN segmentation and SV breakpoints, which were both significantly increased in relapse samples (p_CN = 0.004, p_SV = 0.006). Together, these results demonstrate structural remodeling rather than mutational burden as a defining feature of treatment resistance that underlies aggressive recurrences. Accordingly, we identified relapse-specific subclonal events as putative drivers of tumor progression during therapy, with impacts on pathways such as MAPK. Integration with matched transcriptomic samples enabled assessment of the functional impact of these events on gene expression and pathway activity. These findings show structural variation as a key driver of HGSC treatment resistance. By linking clonal genomic alterations with transcriptomic consequences, our approach uncovered relapse-specific drivers and opens new avenues for biomarker discovery and therapeutic targeting.



 Giulia Micoli, Jaana Oikkonen, Kari Lavikka, Déborah Boyenval, Johanna Hynninen, Sampsa Hautaniemi. Clonal evolution and structural variation drive chemotherapy resistance in ovarian carcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B016.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f771a1df9cd571a45e30843fc11bc1edd48d299c" target='_blank'>
              Abstract B016: Clonal evolution and structural variation drive chemotherapy resistance in ovarian carcinoma
              </a>
            </td>
          <td>
            Giulia Micoli, J. Oikkonen, K. Lavikka, Déborah Boyenval, J. Hynninen, S. Hautaniemi
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="


 Loss-of-function studies (LOF) have historically been the main functional approach to identify and study genes which drive the biology of Multiple Myeloma (MM) or other neoplasias. We hypothesized that substantial complementary data can be derived from systematically examining the impact of transcriptional activation of endogenous loci of different genes. We thus performed genome-scale CRISPR gene activation (CRISPRa) screens in 10 genotypically diverse MM cell lines (dCas9-P65-HSF transcriptional activation system; Calabrese genome-scale sgRNA library) to identify genes with significant sgRNA enrichment reflecting enhanced tumor cell fitness. These experiments identified that MM driver genes are highly cell line-dependent, with no universal “hit” but with many recurrent (e.g. in ≥4/10 lines) drivers, including key transcription factors/cofactors (e.g. POU2F2, POU2AF1, IRF4, MYC, HOXA3); growth factor signaling mediators, (e.g. IGF1R, IRS1); Ras family members (e.g. KRAS in MM1S cells); NFκB signaling (e.g. RELA, TIFA), PKC pathway (PRKCE), cell surface receptors (e.g. CD48, HTR1D, IGF1R, UPK3B), solute carrier transporter genes (e.g. SLC38A1). No correlation is currently observed between the pattern of driver genes for the individual MM cell lines and their respective molecular subtype (e.g., t(4;14), t(14;16), t(11;14), etc). Some MM-preferential dependencies identified by CRISPR knockout (KO) studies (de Matos Simoes et al Nat Cancer 2023) were identified by CRISPRa as recurrent MM drivers (e.g. POU2AF1, IRF4) or drivers with limited role in individual line(s) (e.g., IKZF3 and IKZF1 in MM1S cells). Notably, many top MM drivers are not essential for MM cells based on CRISPR KO studies. Most top MM drivers were not identified in similar CRISPRa studies as recurrent drivers for 17 non-MM cell lines from other hematologic neoplasias or solid tumors. With some notable exceptions (e.g., IRF4, POU2AF1), most other MM drivers are not identified in MM patient samples among the molecules with most frequent mutations, gene amplification or overexpression (e.g. vs normal plasma cells) at transcript or protein level. However, transcript upregulation (log2FC≥1.0) of at least one MM driver gene is observed in ~85% of paired MM samples after relapse vs. before treatment (MMRF CoMMpass study), suggesting that many CRISPRa-driver genes can contribute, alone or in concert, to enhanced MM cell fitness in clinical relapses. We validated the functional relevance of several MM driver genes (e.g. POU2F2, EGLN1, SLC38A1) with individual sgRNAs for CRISPR activation and/or cDNA overexpression (vs. isogenic controls) in competition experiments. We further probed the mechanistic basis for the role of POU2F2 as one of the top positive regulators of MM cell fitness and performed RNA sequencing analyses of MM1S cells harboring CRISPRa-based upregulation of POU2F2 expression. We observed upregulation of a distinct cluster of genes (including HOXA3, USP32, HTR1D) that are identified in our genome-scale CRISPRa studies as driver genes specifically in MM1S cells or ≥4 MM lines of our analyses, as well as several genes that are essential for MM survival (e.g., U2AF1, ZNF492, HYPK, PAM16, H2BC11). These results indicate that genome-scale CRISPRa studies provide data which are complementary to those derived from LOF studies. Indeed, our CRISPRa studies validated that MM cell fitness can be enhanced by upregulation of some genes that are prominently essential for MM cells in LOF studies (e.g. IRF4, POU2AF1) but also identified many other promising, previously understudied, regulators of MM cell biology which are not essential for baseline survival/growth of MM cells but can induce growth when further activated. Upregulation of one or more MM driver genes is recurrently observed in MM clinical relapses, which raises intriguing hypotheses about how potential cooperative interactions between these genes contribute to the risk of MM relapses. More broadly, the results indicate that CRISPRa studies can provide novel insights into the biology of MM cells and identify previously understudied genes with therapeutic implications towards suppressing the transition of MM cells to states of advanced biological aggressiveness.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02015e88839b8028448eeed27364c1dfd71a4d2b" target='_blank'>
              Identification of novel “driver” genes of multiple myeloma through genome-scale CRISPR activation studies
              </a>
            </td>
          <td>
            Huihui Tang, Ricardo de Matos Simoes, Torsten Steinbrunn, Ryosuke Shirasaki, O. Dashevsky, Eleni Anastasakou, Oliver Bohorquez, S. Yamano, E. Papadopoulos, Lisa B Leypoldt, Arun Wiita, Olwyn Mahon, Vikas Gupta, Jonathan Licht, Benjamin G. Barwick, Lawrence Boise, Constantine S Mitsiades
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="


 Introduction: KMT2A-rearranged acute myeloid leukemias (KMT2A-r AML) are highly lethal leukemias caused by rearrangements of the KMT2Agene, which encode abnormal fusion proteins that activate pro-leukemogenic genes such as HOXAgenes andMEIS1. Prior studies focused on identifying drugs to target proteins that form complexes with KMT2A fusions led to the recent FDA approval of the first menin inhibitor for refractory KMT2A-r AML. However, resistance remains a significant clinical challenge. The gene encoding the High Mobility Group A1 (HMGA1) chromatin regulator is up-regulated in refractory hematologic malignancies and solid tumors where high levels portend adverse outcomes (Resar et al., Cancer Res 2018; Li et al., Blood 2022; Chia et al., JCI2023; Luo et al., JCI2025). In preclinical models, HMGA1 drives tumor progression and a highly plastic, stem-like state by modulating chromatin structure to activate stem cell gene networks in diverse tumors (Li et al., Blood 2022; Chia et al., JCI2023; Luo et al., JCI2025). HMGA1is also enriched in the leukemic stem cell signature of KMT2A-r mouse models (Somervaille et al., Cell Stem Cell 2009), and a recent study identified HMGA1 among the gene signature of menin inhibitor resistance in KMT2A-r AML patient-derived xenografts (Soto-Feliciano et al., Cancer Discov2023). We therefore hypothesized that: 1) HMGA1 drives refractory disease and menin inhibitor resistance in KMT2A-r AML by modulating transcriptional networks, 2) Elucidating HMGA1 function in KMT2A-r AML will reveal novel therapeutic vulnerabilities.
 Methods: To define HMGA1 function in KMT2A-r AML, we silenced HMGA1 using CRISPR or short hairpin RNA (shRNA) in human KMT2A-r AML cell lines (MOLM-14, THP-1, MV4;11). Leukemogenic properties were tested in vitroand in mouse implantation models. To elucidate mechanisms underlying HMGA1, we integrated multiomic studies (RNA/ATACseq). Artificial intelligence (AI) was used to predict drugs to target HMGA1 gene networks.
 Results: HMGA1 is overexpressed in primary KMT2A-r AML caused by diverse fusions in two large patient cohorts compared to age-matched, healthy bone marrow by RNAseq (P<0.0001). Moreover, HMGA1 transcripts correlate positively with downstream targets of KMT2A-r fusions, including HOXA9,HOXA10, andMEIS1 (r>0.4; P<0.0001). HMGA1 gene silencing in KMT2A-r AML cell lines disrupts proliferation (P<0.0001) and clonogenicity (P<0.0001) while inducing differentiation markers (CD11b, CD14; P<0.01). Strikingly, HMGA1deficiency also decreases leukemic engraftment (P<0.0001) and prolongs survival (P<0.0001) following implantation in immunodeficient mice. Intriguingly, AML cells that engraft after HMGA1 silencing express higher HMGA1, suggesting that the escape from gene silencing and a specific HMGA1 level are required for leukemogenesis in this model. RNAseq in KMT2A-r AML cell lines (MOLM-14, THP-1) followed by GSEA revealed that HMGA1 activates transcriptional networks that govern: 1) HOX gene expression, 2) DNA repair, and 3) cell cycle progression (E2F, G2M, Mitotic Spindle). Moreover, HMGA1 activates the signature of menin inhibitor-resistance genes. Conversely, HMGA1 represses MHC-II genes involved in antigen presentation and an anti-tumor immune attack. ATACseq revealed that HMGA1 reduces chromatin accessibility within the MHC-II locus. Leveraging drug prediction algorithms, we identified candidate compounds to target HMGA1 networks, which include the histone deacetylase inhibitor (HDACi) entinostat (ENT), that promotes chromatin opening. Treatment with ENT represses HMGA1 and cell-cycle genes while inducing MHC-II gene expression. Preliminary results suggest that ENT enhances sensitivity to menin inhibitors, supporting further investigation into potential combination strategies.
 Conclusions: HMGA1 is required for salient leukemogenic properties in KMT2A-r AML preclinical models. Mechanistically, HMGA1 activates genes governing cell cycle progression, particularly those implicated in menin inhibitor resistance, while repressing anti-tumor, immune attack genes. Further, these networks can be targeted with clinically available drugs. Together, these findings implicate HMGA1 as a novel epigenetic regulator and promising therapeutic target in KMT2A-r AML.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8259a1497613206067514a7b4107c8e1b548c23b" target='_blank'>
              HMGA1 chromatin regulators drive transcriptional networks governing cell cycle progression, immune escape, and menin-inhibitor resistance in KMT2A-r Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Bailey E. West, Li Luo, Zanshé Thompson, Yuze Du, Leslie Cope, Joseph Kim, Audrey-Ann Supreme, Faiza Shaik, Isha Maloo, Kathryn Mi, Alison Park, S. Meshinchi, R. Ries, Jung-Hyun Kim, Linda S. Resar
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Small-cell lung carcinoma (SCLC) remains one of the most aggressive lung cancers and continues to pose a major challenge for precision oncology. Despite its morphological uniformity, SCLC exhibits marked molecular heterogeneity with recurrent, potentially targetable genomic alterations. Comprehensive profiling is often hindered by limited tissue availability and the need for rapid therapeutic intervention. We performed genomic profiling of 55 primary and metastatic SCLC samples using a 324-gene hybrid-capture next-generation sequencing panel. Consistent with prior reports, nearly all tumors exhibited biallelic TP53 and RB1 inactivation. Recurrent alterations involved the PI3K/Akt/mTOR pathway (62%), chromatin regulators (42%), and NOTCH signaling genes (15%). PTEN mutations were enriched in brain metastases. Frequent copy-number gains affected SOX2, NKX2-1, MYC-family genes, and CCNE1. Two novel recurrent amplifications of potential clinical significance were identified: TYRO3 (33%) and SDHA (13%). TYRO3, a TAM family receptor tyrosine kinase, and SDHA, a mitochondrial enzyme involved in succinate metabolism, may contribute to tumor progression and represent emerging therapeutic vulnerabilities. These findings underscore the genomic diversity of SCLC and highlight the potential utility of broad next-generation sequencing in uncovering new molecular targets for precision therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c257de36865932aa9b4ccc1af139b71a5ba02b7" target='_blank'>
              Comprehensive Genomic Profiling of Small-Cell Lung Cancer Reveals Frequent Potentially Targetable Alterations
              </a>
            </td>
          <td>
            Dániel Schmalz, Zoltán Krabóth, Veronika Czoma, Péter Urbán, Attila Gyenesei, István Ruzsics, Veronika Sárosi, Á. Boronkai, Emőke Papp, B. Kajtár
          </td>
          <td>2025-11-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Despite significant advances in cancer research, the complexity of signaling pathways remains a major challenge in precision oncology. Tumors harbor a diverse array of genetic alterations; however, these often converge onto a limited set of core signaling pathways, notably RAS/RAF/MEK/ERK (MAPK), PI3K/AKT/mTOR (PAM), and Wnt/β-catenin. Concurrently, these pathways diverge extensively downstream, driving therapeutic resistance through mechanisms such as epithelial-mesenchymal transition, immune evasion, and metabolic reprogramming. Unlike previous literature that largely provides descriptive accounts of pathway alterations, this review uniquely synthesizes convergent and divergent signaling into a clinically actionable diagnostic and therapeutic framework. It critically assesses current precision oncology strategies, identifies gaps, and proposes a dual-axis model that integrates static genomic profiling with dynamic signaling evolution to inform precision therapy. By highlighting opportunities for combination therapy informed by pathway interdependencies and adaptive resistance mechanisms, this perspective provides novel clinical insights and tangible directions for future research in overcoming resistance in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f1f096cbac949c4b15346a048f653260da1f8e6" target='_blank'>
              Convergent and Divergent Signaling Pathways in Cancer: A Dual-Axis Model for Adaptive Precision Oncology.
              </a>
            </td>
          <td>
            Aliasgar Shahiwala PhD
          </td>
          <td>2025-10-08</td>
          <td>Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9d2645c176f5e0510e7c9fe1ad9d0e8b2a80929" target='_blank'>
              Stroma-driven horizontal transfer of TCA-related proteins mediates metabolic plasticity and imatinib resistance in chronic myeloid leukemia.
              </a>
            </td>
          <td>
            P. Chrościcki, Nikodem Kasak, D. Dymkowska, L. Turos-Korgul, Dominik Cysewski, Vira Chumak, Dawid Stępnik, Monika Kusio-Kobiałka, Agata Kominek, M. Lebiedzińska-Arciszewska, Alicja Krop, Joanna Szczepanowska, Mariusz Wieckowski, Tomasz Stoklosa, Krzysztof Zabłocki, K. Piwocka
          </td>
          <td>2025-12-02</td>
          <td>Cell communication and signaling : CCS</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Background/Objectives: Triple-negative breast cancer (TNBC) accounts for 15 to 20% of breast cancer cases and contributes to a disproportionate 35% of breast cancer deaths. Its resistance to chemotherapy presents a significant challenge. Methods: We firstly compared transcriptomic profiles between TNBC cell lines and patient samples and inferred the MDA-MB-231 cell line as the most representative model for TNBC with poor response to chemotherapy. We then conducted a genome-wide CRISPR-Cas9 screening and RNA-seq analysis in MDA-MB-231. Results: This analysis revealed 96 and 93 genes that could re-sensitize cisplatin and doxorubicin treatment, respectively, with 19 overlapping genes. Among these genes, 28 have been studied and published previously in chemoresistance research. MCM9 was found as a new TNBC chemoresistance target. Its target drug, KPT-185, showed an additive effect with cisplatin in treating TNBC cells. In the follow-up gene combination double-knockout experiment among 65 genes selected from cell death pathways, 242 synthetic lethal gene pairs were discovered to overcome chemoresistance in TNBC. Conclusions: In this study, we identified synthetic lethal targets in treating TNBC with cisplatin and doxorubicin through a genome-wide CRISPR-Cas9 screening and gene combination double-knockout screening.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/928a386f1b1e43d1649df91b696f9fa5ac9e4221" target='_blank'>
              CRISPR-Cas9 Genome and Double-Knockout Screening to Identify Novel Therapeutic Targets for Chemoresistance in Triple-Negative Breast Cancer
              </a>
            </td>
          <td>
            Shuai Shao, Shangjia Li, Yang Huo, Shanwu Tang, Birkan Gökbağ, Kunjie Fan, Yirui Huang, Lingling Wang, Gregory Nagy, Jeffrey Parvin, Daniel G. Stover, Lijun Cheng, Lang Li
          </td>
          <td>2025-12-03</td>
          <td>Cancers</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Long non-coding RNAs (lncRNAs) represent an important layer of gene regulation and have emerged as significant contributors to multiple myeloma (MM) progression (Malard, F., Neri, P., Bahlis, N.J. et al.,2024). The lncRNA NEAT1 is over-expressed in MM patients, supporting malignant plasma cells proliferation and promoting therapy resistance (Taiana E.et al., 2020). Besides its conventional function in the assembly of paraspeckles (PSs), we recently demonstrated that NEAT1 is directly involved in transcriptional control (Puccio N. et al., 2024). In this study, we deepen our understanding of NEAT1's function in shaping the activity of transcriptional bodies, providing the rationale for targeted therapeutic intervention.
 Starting from a transcriptomic approach in NEAT1-silenced (KD) and CRISPRa NEAT1-overexpressing MM cells, we derived a list of 378 targets, primarily involved in mitosis. 52% of these present the cell cycle homology region motif (CHR) in their promoter region, which synchronizes their activation during late cell cycle phases. We confirmed the clinical relevance of NEAT1's transcriptional program in the coMMpass cohort of almost 900 MM patients. Unsupervised clustering analysis, based on the expression of 378 genes NEAT1's target genes, identified two different groups of patients characterized by high or low NEAT1's expression program and different clinical outcomes. Notably, the cluster with the high NEAT1-CHR program showed reduced overall survival. To explore the mechanistic contribution of NEAT1 to mitotic gene expression, we performed computational analysis through the binding model of analysis of transcription (BART) and query of the ENCODE database. These analyses revealed that FOXM1 and CDK9 as upstream regulators of NEAT1 program. Notably, FOXM1 resulted to be highly expressed in R2-ISS stages II-III of myeloma disease as compared to stage I, and in secondary plasma cell leukemia (sPCL) of myeloma origin, based on coMMpass and GSE66293 datasets analysis. We supported these observations by performing in vitro assays. We described a co-localization of NEAT1 and FOXM1 condensates in MM cells, through combined RNA-FISH/immunofluorescence, and direct binding of NEAT1-FOXM1/NEAT1-CDK9 by targeted RNA-immunoprecipitation (RIP). Additionally, with Chromatin immunoprecipitation experiments (ChIP), we demonstrated that NEAT1 KD results in a reduced occupancy of FOXM1 at the promoters of essential CHR genes. Finally, by performing high-throughput drug screening, we identified CDK9 inhibitors to have a strong synergistic effect with NEAT1 KD, consistent with a synthetic lethality effect. These results were confirmed by live cell imaging approach, with two selective CDK9 inhibitors (CDK9i) on a panel of MM cell lines. Consistently, NEAT1 overexpression confers an increased resistance to CDK9i, confirming the participation of NEAT1 to the transcriptional complex that synchronizes mitotic genes and executes the cell division program.
 These data are in line with our recent findings that demonstrate that NEAT1 controls and cooperates with Aurora kinase A (AURKA) in ensuring proficient chromosomal segregation(Puccio N. et al., 2024).
 In conclusion, our data demonstrated that NEAT1 contributes to overt phases of MM and to the clinical aggressive behaviour of the disease, by orchestrating a mitotic transcriptional program. Furthermore, through the application of a multi-dimensional approach, we revealed that the dual inhibition of NEAT1 and transcriptional regulators as CDK9 may offer a promising strategy for novel combinatorial anti-MM therapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0423164211e0008033b3494329d936fcc2b1eb0a" target='_blank'>
              Disruption of NEAT1-transcriptional network identifies targetable non-genetic dependencies in multiple myeloma
              </a>
            </td>
          <td>
            N. Puccio, Christian Boni, Federica Torricelli, Emanuele Vitale, E. Taiana, D. Ronchetti, Valentina Fragliasso, Selene Mallia, Giulia Gambarelli, Roberto Piva, N. Bolli, S. Luminari, A. Ciarrocchi, A. Neri
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Epitranscriptomics, the study of dynamic chemical modifications on RNA mediated by “writers,” “erasers,” and “readers,” has emerged as a pivotal discipline in elucidating the intricate interplay between cancer and immune regulation. These reversible modifications (e.g. m6A, m5C, Ψ) govern RNA metabolism, stability, and translation, thereby exerting spatiotemporal control over immune cell differentiation, activation, and function. Dysregulation of RNA-modifying proteins disrupts immune surveillance, enhances tumor cell survival under stress, and promotes chemoradiotherapy resistance by altering RNA splicing, translation, and stress adaptation pathways. This review summarized the recent progress in the regulatory mechanisms profoundly influencing the tumor microenvironment (TME), modulating immune checkpoints, antigen presentation pathways, and the activity of immune cells. Furthermore, we discussed the therapeutic strategies and challenges in targeting epitranscriptomic regulators and epitranscriptomic editing technologies to enhance anti-tumor immune responses and overcome therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50c0776f1a44885afebca84621fac676e8beb717" target='_blank'>
              Epitranscriptomic control of cancer immunity and therapy resistance
              </a>
            </td>
          <td>
            Xingsen Zhao, Suhua Guan
          </td>
          <td>2025-11-07</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Over the past two decades, considerable progress has been made in cataloguing genes and active chromatin elements in humans. However, despite these efforts, less than a quarter of genes have been assigned a function in the context of human disease, which limits our ability to interpret clinical genome sequencing results. In the field of inherited retinal diseases, 30–40% of cases remain genetically undiagnosed. This may be partially due to our limited understanding of the function of most retina-expressed genes, which prevents the correct interpretation of their sequence variations. In the effort of elucidating retinal gene function, we aimed at developing a protocol for in-vivo CRISPR-based gene knock-out perturbation in the mouse retina. This methodology can be useful to study retinal biology in health and disease, to investigate the effects of ablation of novel uncharacterized genes, and to study possible genetic modifiers of retinal phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/619ae9c941402bd8584dffd71b1ba2adc7fdf7f6" target='_blank'>
              Elucidating photoreceptor gene function with in-vivo CRISPR knock-out perturbation assay
              </a>
            </td>
          <td>
            Riccardo Sangermano, E. Galdikaitė-Brazienė, K. Bujakowska
          </td>
          <td>2025-10-27</td>
          <td>Research Square</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="


 The clinical outcome of acute myeloid leukemia (AML) is determined by therapy resistance and relapse. AML-induced immunosuppression has emerged as one of many cofactors and seems to be caused by distinct mechanisms, e.g. immune checkpoint interactions, altered antigen presentation and dysregulation of humoral factors. With chemotherapy, these alterations are expected to be reversible, but the exact mechanisms remain unclear. Adressing this issue, we performed single-cell RNAseq and spatial assessment to identify therapy-induced perturbations of the bone marrow (BM) microenvironment in newly diagnosed AML patients prior to and after induction therapy.



 43,000 single-cell RNA sequencing profiles were generated from 12 paired samples in 6 AML patients using 10x Genomics 5'-scRNA/CITE-seq (140 Total-seq C antibodies) at initial diagnosis (T1) and after induction chemotherapy with either “7+3” or CPX-351 (T2). Analyses were performed using R v4.1.1 and Seurat v4.3.0. Signaling pathway activity was calculated using PROGENy. Stemness was estimated using CytoTrace. Potential cell-cell interactions were scored using CellChat. Spatial correlation was performed by histopathology and immunohistochemistry in matched trephine biopsies using the following stains: H&E, Giemsa, PAS, Gomori silver, Prussian Blue, Glycophorin C, MPO, CD61, CD34, CD117, CD14, CD68, CD20, CD3, CD138, VISTA. All slides were microscopically assessed by a trained hematopathologist.



 Pre-treatmentBM (T1) was characterized by reduction of normal hematopoiesis (p=0.009), in particular classical dendritic cells type 1 (cDC1) (p=0.066) and monocytes (p=0.034). High levels of AML cells were associated with significantly higher levels of mature naïve B cells (p=0.002), higher levels of class-switched B cells (p=0.030) and enrichment of inflammatory monocyte-like cells with MDSC phenotype. Regarding AML cells, 8 different transcriptional subtypes were identified across patients: 4 clusters with highly immature properties (undifferentiated and LMPP-like AML cells) and 4 clusters with signs of maturation (aberrant erythroid-like, EMP-like, promyelocyte-like and monocyte-like AML cells). The immature AML clusters showed high similarities in their transcriptional and signaling patterns, high stemness, quiescence, strong expression of homing factors (CD44, ITGA4), DNA instability and signs of chemoresistance such as upregulation of beta-Catenin, Trail and TGFb pathways. Furthermore, an upregulation of NFkB, TNFa and JAK/STAT signaling was observed in these AML cell clusters, indicating the formation of a highly inflammatory and dysfunctional immune microenvironment. This environment was characterized by a decrease of cDC1, presence of IFIT-high monocyte-like MDSCs, clonal expansion of senescent T cells as well as functional disturbation of antigen-presenting cells, T and NK cells with high expression of TGFb. Interestingly, each AML cell cluster showed a distinct expression profile of immune checkpoint markers, with most prominent expression of VISTA in monocyte-like AML cells. Spatially, VISTA-positive monocyte-like AML cells surrounded other AML cell populations in the BM niche. In addition, monocyte-like AML cells showed high expression of hypoxia- and metabolism-associated signatures (e.g. OXPHOS, fatty acid/cholesterol homeostasis), which might have supported the dysfunctional microenvironment. These microenvironmental changes were largely reversible after chemotherapy (T2).



 Our study reveals AML-induced dysregulation of the BM immune microenvironment, which seems reversible upon AML treatment with standard chemotherapy. Our data suggest that immature AML cells sustain the leukemic stem cell pool, whereas more mature EMP- and promyelocyte-like AML cells contribute to the bulk of AML. Furthermore, monocyte-like leukemic cells may shape the microenvironment to safeguard AML, possibly by VISTA-induced modulation of the immune system. This might explain limited therapeutic efficacy with currently available checkpoint inhibitors in AML. In the long run, identification of specific immune signatures in AML may improve our understanding of AML biology and inform therapeutic algorithms with regard to immunotherapeutic strategies in AML.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3da4a552fa669ffb6977b0dc73a6ab66c9e44bb5" target='_blank'>
              AML maturation stages associate with disease-permissive and immunosuppressive effects in newly diagnosed AML patients: An analysis at the single-cell level
              </a>
            </td>
          <td>
            Jana Ihlow, J. Schrezenmeier, A. Trinks, M. Milek, Seunghoon Kim, Julia Jesse, Eva Tranter, Christian Schneider, Denise Bachorska, Dieter Beule, L. Penter, Sandrine Sander, Simon Haas, M. Morkel, C. Sers, Nils Bluethgen, David Horst, Lars Bullinger, P. Bischoff, J. Westermann
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Therapy-induced senescence (TIS) is a reversible growth arrest induced by anticancer treatments, which may contribute to the development of long-term therapy resistance in tumor cells. Senotherapeutics, agents targeting senescent cells, are being tested in clinical trials to improve patient outcomes. Due to the transient nature of TIS, we hypothesized that senolytics would be most effective when administered at the appropriate time. We created a reliable TIS cell line model in triple-negative breast cancer (TNBC) using experimental drug YM155. We observed that a single dose of YM155 triggers a brief senescence, leading to a persistent drug-tolerant state that cannot be reversed by redosing. This reversibility is not limited to cancer cells. It extends to noncancerous human cells and live zebrafish larvae, suggesting a rapid adaptation mechanism against xenobiotics. We identified transforming growth factor-β (TGF-β), a cytokine linked to TNBC chemoresistance, as being expressed alongside the emergence of drug tolerance. We inhibited TGF-β signaling to eliminate the tolerant phenotype and promote the clearance of cancer cells by immune cells. However, this was most effective within a specific time window after TIS induction. We suggest that the timely use of senotherapeutics could improve the effectiveness of anticancer drugs in clinical settings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17ec0acb6fede0ff85bcba84d195e45333ad41d7" target='_blank'>
              Mapping the Progression of Therapy-Induced Senescence to Therapy Tolerance: An Evolutionarily Conserved Mechanism for Optimizing Cancer Treatment with Senotherapeutics.
              </a>
            </td>
          <td>
            Gargi Mukherjee, Neha Dutta, Muthumeena Ramanathan, Kankonika Bhattacharyya, Alabhya Das, Shaista Haider, Goutam Chowdhury, Anindita Chakrabarty
          </td>
          <td>2025-10-10</td>
          <td>ACS pharmacology & translational science</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 CAR-T cell therapy is a standard treatment for relapsed diffuse large B cell lymphoma (DLBCL); however, many patients fail to achieve a durable response. Poor T cell quality, including differentiated or exhausted phenotypes, is a mechanism of resistance to CAR-T (Locke, Blood Adv. 2020). Tumor microenvironment (TME) factors such as hypoxia, nutrient deprivation, and immunosuppressive oncometabolites may impact both T cells made into autologous CAR-T and their function after infusion. Single cell RNASeq of DLBCL patients' axicabtagene ciloleucel (axi-cel) CAR-T infusion products (n=39) revealed MTRNR2L8, encoding a 24-amino-acid peptide called Humanin-like 8 (HN8), upregulation in products of patients that achieved a durable response of >9 months (Yu, JITC 2025). The HN8 amino acid sequence is almost identical to Humanin, encoded by a mitochondrial gene MT-RNR2, known to promote mitochondrial functions including glycolysis and oxidative phosphorylation, and reduce apoptosis.
 We manufactured CD19 CAR-T with CD3z/CD28 co-stimulatory domains similar to axi-cel, which also overexpressed HN8 (19.28z.HN8). HN8 overexpression was confirmed by western blot using Humanin antibody mixture (Sigma H2414, Invitrogen PA1-41325 and PA1-41610) and Liquid Chromatography Tandem Mass Spectrometry (LC-MS/MS).
 We performed bulk RNA sequencing on 19.28z.HN8, and 19.28z CAR-T not expressing HN8, manufactured from healthy donor PBMCs (n=11). Differential gene expression analyses were done with DESeq2 analyzed via paired sample Wilcoxon tests of rlog normalized gene expression values, reporting FDR adjusted p-values. Pathway analyses were conducted via single sample gene set enrichment analysis (ssGSEA). 19.28z.HN8 CAR-T had downregulation of exhaustion-associated transcripts including TOX (p=6.1×10⁻4), TOX2 (p=0.012), PRDM1encoding BLIMP-1(p=0.003), TIGIT (p=8.2×10⁻9), ENTPD1-encoding CD39 (p=3.7×10⁻4), NT5E encoding CD73 (p=6.4×10⁻5), IKZF2 encoding HELIOS (p=2.5×10⁻5), and immune checkpoint CTLA4 (p=1.9×10⁻12) and upregulation of cytotoxicity gene GZMB (p=5.4×10⁻13) compared to 19.28z control. Next, for functional validation, T cells from two non-responding DLBCL patients leukapheresis products used to manufacture axi-cel, were flow cytometry sorted into unexhausted CD39- (UE) and pre-exhausted CD39+ (PE) fractions prior to 19.28z and 19.28z.HN8 CAR manufacture. Following manufacture from PE fractions, 19.28z.HN8 CD8+ and CD4+ had significantly lower percentages of PD-1 and TIGIT double positive CAR-T cells. CAR-T manufactured from UE fractions had fewer apoptotic cells upon activation. Notably, 19.28z.HN8 CAR-T were significantly more cytotoxic, and secreted more IFNg, than control in both UE and PE fractions.
 We hypothesized that decreased exhaustion and improved effector function was due to HN8 impact on metabolic and anti-apoptotic programs similar to the function of humanin. RNA Seq demonstrated 19.28z.HN8, vs 19.28z control, mediates upregulation of HIF1A(p=1.2×10⁻3) and STAT3 (p=8.4×10⁻7), which promote glycolysis, and downregulation of glycolysis-suppressing transcription factor KLF2 (p= 3.5×10⁻4) and to lesser degree anti-apoptotic genes, specifically increased BFL-1 (p =0.04) and the NF kappa B signaling pathway (p=0.014). To validate, functional metabolic assays mito stress (n=9) and glycolysis stress (n=7) tests were performed using Agilent Seahorse XF at baseline and after stimulation for 15 hours with plate bound CD19 peptide. Consistent with transcriptomic data, 19.28z.HN8 exhibited enhanced basal glycolysis (n=9, p=0.008) and compensatory glycolysis (p=0.027). No differences were observed after stimulation. Upon stimulation 19.28z.HN8 had enhanced OCR (p=0.027) and ECAR (p=0.002), ATP production (p=0.023), maximal respiration (p=0.039), and basal respiration (p=0.039).
 To evaluate apoptotic pathways, we performed BH3 profiling, revealing lower cytochrome C release in 19.28z.HN8 vs 19.28z. in a dose dependent fashion with FS1 mimetic (Biosynth), which inhibits BFL-1, an antiapoptotic member of the BCL2 family. Taken together with RNA seq this suggests higher BFL1 in 19.28z.HN8 contributes to prevent intrinsic apoptosis.In conclusion, HN8 overexpression in CAR-T cells drives a less exhausted and more functional phenotype, likely informed by HN8 enhancement of glycolysis and resistance to apoptosis. Ongoing studies aim to uncover how HN8 boosts CAR-T cell function within TME like conditions, and in vivo.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e99f547ec1f6dd4ee9538ce970ced8b7d46628c" target='_blank'>
              Humanin-like 8 regulates metabolic fitness and anti-apoptotic programming in CD19 CAR-T cells
              </a>
            </td>
          <td>
            Samira Naderinezhad, Jerald Noble, C. Savid-Frontera, Bin Fang, M. Menges, John M. Koomen, R. Atkins, Frederick L Locke
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf829c80ee16d8593d7d0383b2985d0175ddbfa6" target='_blank'>
              Cell passage number drives transcriptomic drift as an overlooked factor in experimental reproducibility.
              </a>
            </td>
          <td>
            Shuai Liu, Hai-juan Peng, Junyan Meng, Liyuan Zhuo, Shaoping Fu, Wei Yang
          </td>
          <td>2025-11-21</td>
          <td>Scientific reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Precision therapy for glioma remains a major challenge due to tumor heterogeneity. The Origin Recognition Complex Subunit 6 (ORC6) is a crucial regulator of DNA replication initiation. This study aims to investigate the expression of ORC6 in gliomas and its relationship with survival rates and malignancy, while screening potential drugs targeting its functional network. By integrating multiple bioinformatics approaches with structure-based virtual screening, retrospective RNA sequencing data analysis was performed using patients from the Chinese Glioma Genome Atlas (CGGA) and The Cancer Genome Atlas (TCGA) databases. A protein-protein interaction (PPI) network was constructed from ORC6-coexpressed genes to identify core hubs. Molecular docking was employed to screen a library of natural compounds and known drugs against these hub targets. Research has revealed that ORC6 is significantly upregulated in high-grade gliomas, with its elevated expression associated with poor survival outcomes and immune inflammatory responses. Network analysis identified five core hub genes (ORC1, ORC2, MCM2, MCM6, CDC45) central to DNA replication. Molecular docking revealed that several compounds, including the natural flavonoid Baicalein and the FDA-approved drug Palbociclib, exhibited high binding affinity to these hub targets. ORC6 represents a highly promising novel target for precision therapy in glioma. Potential approaches to target this pathway include disrupting the ORC6-replication axis using existing drugs (such as palbociclib) or natural products (such as baicalin).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/324196dc2699eff3ae974af550ed77475be4c5ee" target='_blank'>
              High ORC6 expression is a prognostic indicator of poor survival in glioma patients
              </a>
            </td>
          <td>
            Mengjie Wang, Song Feng, Chen Zhang, Feng Jin
          </td>
          <td>2025-11-19</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea0c9bba6958723f9dc44615a55714dd8d76cc78" target='_blank'>
              Understanding KRAS: mechanisms of resistance and future therapeutic strategies
              </a>
            </td>
          <td>
            Rana Mahmoud Khashaba, Nada Ali Refaat, Sama Elsayed Elmorsy, Fatma Ibrahim Ali, Raghda W. Magar
          </td>
          <td>2025-11-27</td>
          <td>Discover Applied Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Medulloblastoma (MB) is the most common malignant brain tumor in children, comprising ~20% of pediatric brain tumors. MB is a heterogeneous disease, with clinical outcomes heavily influenced by molecular subgroups. Among these, Group 3 MB, characterized by MYC amplification, exhibits the poorest prognosis, with a high propensity for metastasis and recurrence. Our previous research identified WEE1 as a critical regulator of MB cell survival and found that Myc-driven MB is particularly sensitive to the WEE1 inhibitor AZD1775. Although AZD1775 has demonstrated efficacy, especially when combined with genotoxic agents, resistance to small molecule inhibitors often emerges, limiting its therapeutic potential. CDK7, a kinase essential for transcriptional initiation via phosphorylation of RNA Pol II’s C-terminal domain, plays a critical role in Myc-driven malignancies and is frequently enriched at super-enhancers. AZD1775-resistant cells undergo chromatin reorganization to establish alternate cis-acting elements that promote transcription of DNA repair and proliferation genes, circumventing treatment. We hypothesized that targeting CDK7-mediated transcriptional reprogramming could overcome AZD1775 resistance. Using the CDK7 inhibitor THZ1, we performed western blot analysis of DNA repair proteins and observed inhibition of ATM pathway activation, a key component of the DNA damage response. Proliferation assays in both parental and AZD1775-resistant cell lines demonstrated that combining AZD1775 with THZ1 significantly suppressed cell growth. These findings suggest that THZ1 may reverse adaptive chromatin reorganization, sensitizing resistant MB cells to AZD1775 and enhancing the overall therapeutic response. Future studies will investigate the potential of combining CDK7 inhibitors with other epigenetic modulators to further disrupt resistance mechanisms and improve treatment outcomes in aggressive MB subtypes, particularly Group 3 MB. This approach offers a promising avenue for overcoming therapeutic resistance and advancing precision medicine in medulloblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb1c890009c33793158ec3602b7038265f5fb5f2" target='_blank'>
              CNSC-10. CDK7 INHIBITION MITIGATES AZD1775 RESISTANCE VIA CHROMATIN REPROGRAMMING IN MYC-MB
              </a>
            </td>
          <td>
            Kiara Smart, Rajeev Vibhakar, Bethany L. Veo
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="While FLT3 inhibitors have significantly improved the treatment of aggressive FLT3-mutated acute myeloid leukemia (AML), the emergence of resistance remains as a major challenge. Here, we applied our recently developed single-cell lineage tracing method ReSisTrace to identify cells that are intrinsically resistant or sensitive to the FLT3 inhibitors midostaurin and quizartinib in AML with FLT3-ITD mutations. Comparison of the gene expression profiles of these cells revealed transcriptional resistance signatures, including upregulation of GSPT1. Depletion of GSPT1 with CRISPR-Cas9-mediated knockout resulted in increased sensitivity of AML cells to quizartinib treatment. Further, targeting GSPT1 with the small molecule CC-90009 exhibited strong synergistic effects when combined with FLT3 inhibitors in the FLT3-ITD cell lines and primary AML patient samples. In addition, in an FLT3-ITD-positive AML patient-derived xenograft (PDX) mouse model, the CC-90009 and quizartinib combination showed significantly higher anti-tumor efficacy and prolonged overall survival compared to either treatment alone. Furthermore, compounds that induced transcriptomic changes opposite to the resistance signatures prompted cells to acquire FLT3 inhibitor-sensitive states. Vistusertib (mTOR inhibitor), linsitinib (IGF1R and insulin receptor inhibitor), and meisoindigo (IGF1R and Src family kinase inhibitor), all inhibiting pathways parallel to or downstream of oncogenic FLT3 signaling, were predicted and validated to sensitize FLT3-mutated cell lines and primary cells to FLT3 inhibitors. Collectively, these findings demonstrate the ability of ReSisTrace to unveil pre-existing transcriptional features of treatment vulnerability in hematological cancers and elucidate strategies for enhancing FLT3 inhibitor treatment efficacy in FLT3-ITD-mutated AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84189684c5cf7142729adb65c133047ed0e40a88" target='_blank'>
              Single-Cell Lineage Tracing Uncovers Resistance Signatures and Sensitizing Strategies to FLT3 Inhibitors in Acute Myeloid Leukemia.
              </a>
            </td>
          <td>
            Johanna Eriksson, Shuyu Zheng, M. Popa, Jie Bao, Jun Dai, Wenyu Wang, Emmet McCormack, Anna Vähärautio, Jing Tang
          </td>
          <td>2025-11-21</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Simple Summary Acute myeloid leukemia (AML) is the second most common leukemia in children. Modern genetic sequencing helps identify molecular alterations linked to prognosis and reveals genomic heterogeneity among individual patients. This study used whole-transcriptome sequencing to map the genetic landscape of pediatric AML in Southwest China and explored the prognostic value of gene fusions and mutations in early and long-term outcomes. We found that certain gene mutations were more common in males or in specific risk groups. Importantly, WT1 mutation was an independent risk factor for poorer long-term survival, while FLT3-ITD mutation was associated with a lower initial treatment response rate. These findings enhance our understanding of the genetic basis of pediatric AML in this population and may guide more personalized treatment approaches in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50903cc6b98f1e80b304ceb6efe23aa48b4824d5" target='_blank'>
              Unveiling the Genetic Mosaic of Pediatric AML: Insights from Southwest China
              </a>
            </td>
          <td>
            Lan Huang, Xingyu Peng, Wenjing Shu, Hui Shi, Li Xiao, Tao Liu, Yan Xiang, Yuxia Guo, X. Guan, Jiacheng Li, Jie Yu
          </td>
          <td>2025-10-30</td>
          <td>Current Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 Background Acute myeloid leukemia (AML) is a clinically and genetically heterogeneous disease. While traditional classification systems such as FAB relied on morphology and immunophenotype, recent classifications (WHO, ICC) emphasize genetic alterations for diagnosis and therapy. However, more directly involved in the phenotypic cellular inheritance, the epigenomic profile may offer an additional dimension for understanding AML heterogeneity. Previously, we performed ATAC-seq of 1,536 AML samples and identified 16 epigenetically defined AML subgroups (subgroups A-P), which were associated with distinct clinical, genetic, and transcriptional features (Ochi et al., ASH 2024). In the present study, we extend these findings by using single-cell RNA and ATAC-seq (scRNA/ATAC-seq) profiling to explore intra- and inter-tumor epigenetic heterogeneity, transcriptional regulation, and hierarchical differentiation trajectories across AML subgroups.
 Methods We performed multi-platform scRNA/ATAC-seq on 36 AML samples including all the 16 epigenomic subgroups and 4 remission samples (as controls), profiling a total of 281,167 mononuclear cells. Computational analyses included cell clustering, projection onto normal hematopoiesis reference maps, pseudotime inference, transcription factor (TF) activity analysis using SCENIC+, and leukemic stem cell (LSC) signature scoring.
 Results scATAC-based clustering showed that leukemic cells from individual AML samples formed distinct clusters largely separated from normal cells, typically co-clustered by subgroup, regardless of their differentiation status, indicating that leukemic cells within each subgroup share a distinct intrinsic epigenetic program. Projection onto a normal hematopoiesis reference map revealed divergent differentiation arrest among subgroups, even within genetically similar cases. For instance, all NPM1-mutant subgroups (D–F) were HOX-related but differed in differentiation states: subgroup E was arrested at the HSC stage, D at the GMP stage, and F contained both progenitors and mature cells. Single-cell analysis also refined differentiation states of subgroups F-H: while bulk ATAC-based deconvolution suggested monocyte enrichment in these subgroups, single-cell analysis revealed mature monocytes predominated in H, whereas F and G were enriched for immature promonocytes.
 Combined with LSC signature analysis based on gene expression, pseudotime analysis demonstrated that cells showing high LSC scores consistently mapped to early stages of the differentiation trajectory across all subgroups. These results indicate the presence of a conserved leukemic hierarchy, with stem-like cells positioned at the apex, irrespective of epigenetic subgroup, thereby underpinning AML pathogenesis. We further analyzed TF activity using the SCENIC+ program based on scRNA/ATAC-seq data, which identified key TFs enriched in each subgroup, many of which overlapped with those found in bulk RNA/ATAC-seq analysis. Pseudotime analysis further showed that TFs exhibited subgroup-specific activation dynamics. For example, HOXA9 was activated throughout the myeloid differentiation trajectory in HOX-related subgroups but peaked at different stages according to subgroups. In the RUNX1-enriched subgroup, IRF8 and BCL11A were both active but peaked at mature and immature stages, respectively. These findings indicate that key TFs exhibit subgroup-specific activation patterns at distinct stages along the myeloid differentiation trajectory.
 Conclusion Single-cell multi-omics analysis reveals that AML epigenetic subgroups exhibit distinct but hierarchically organized differentiation trajectories, driven by dynamically regulated TF programs. These findings refine our understanding of AML diversity and may inform more precise, differentiation-stage–specific therapeutic strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d92f306fb3417df7e3eff8b67fe8a062b478725" target='_blank'>
              Single-cell multi-omics uncovers differentiation-dependent transcriptional programs shaping epigenetic subgroups in AML
              </a>
            </td>
          <td>
            Y. Ochi, Markus Liew-Littorin, Y. Nannya, S. Bengtzén, Bénédicte Piauger, Stefan Deneberg, M. Jadersten, Vladimir Lazarevic, Jörg Cammenga, Anna Robelius, L. Wennström, Emma Olander, S. Kasahara, N. Hiramoto, N. Kanemura, N. Sezaki, M. Sakurada, M. Iwasaki, J. Kanda, Yasunori Ueda, Satoshi Yoshihara, Tom Erkers, Nona Struyf, Yu Watanabe, Masanori Motomura, M. Nakagawa, R. Saiki, Hidehito Fukushima, Koji Okazaki, S. Morimoto, A. Yoda, R. Okuda, Shintaro Komatsu, Guoxiang Xie, Albin Osterroos, A. Kon, Lanying Zhao, Yuichi Shiraishi, Takayuki Ishikawa, Satoru Miyano, Shuichi Matsuda, A. Takaori-Kondo, Hiroyuki Aburatani, Hiroshi Suzuki, Olli-P. Kallioniemi, Gunnar Juliusson, Martin Höglund, Sören Lehmann, Seishi Ogawa
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20ec8c67f8ff833438b56a407614424d68047c71" target='_blank'>
              Single-cell transcriptome analysis defines novel molecular subtypes and reveals therapeutic implications of T/myeloid mixed-phenotype acute leukemia.
              </a>
            </td>
          <td>
            Bin Huang, Wenjie Liu, Yuxin Du, Ping Liu, Zixing Lu, Shiyang Zhong, Xingfei Hu, Wanting Zhou, Yuzhu Shi, Runheng Huang, Xian Zhang, Jinning Shi, Chuanyang Lu, Chunling Wang, Liang Yu, Lingxiang Wu, Wei Wu, Peng Xia, Qian Sun, Liwei Zhu, Zhi Wang, Ruohan Zhang, Xinrui Lin, Sali Lv, Qianghu Wang, Sixuan Qian, Kening Li, Mingyao Hong
          </td>
          <td>2025-11-29</td>
          <td>Genome medicine</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="


 Colorectal cancer (CRC) remains a major global health burden, contributing significantly to morbidity and mortality worldwide. Despite advances in understanding the molecular landscape of CRC and the development of targeted therapies, there is still a critical need for novel treatment strategies that address specific molecular vulnerabilities in CRC. To address this challenge, we developed a robust and scalable platform for target discovery, centered on identifying synthetic lethal (SL) interactions linked to key oncogenic drivers such as APC, KRAS, and SMAD4. Using colorectal cancer (CRC) as a model, we integrated genomic and transcriptomic analyses across a diverse set of biologically relevant systems, including human intestinal stem cell (hISC)-derived isogenic cancer models that accurately replicate tumor genotypes, patient-derived cells (PDCs), and clinical tumor specimens. At the heart of our platform is CRISPR-based functional screening, enabling systematic interrogation of gene dependencies. To benchmark our engineered models, we applied machine learning to RNA sequencing data, comparing them against a large cohort of clinically annotated CRC transcriptomes. This approach enabled precise molecular subtyping, detailed characterization of gene expression programs, and the identification of novel, tractable vulnerabilities specific to genetically defined subgroups. Candidate targets were prioritized based on therapeutic relevance and selective efficacy in cancer cells, while sparing untransformed hISCs. Our findings not only advance early-stage target identification and validation but also support the initiation of precision drug discovery programs focused on CRC subtypes driven by APC, KRAS, and SMAD4 mutations. Moreover, the versatility of our platform enables its application to other malignancies. In summary, we present an integrated, high-resolution framework for uncovering novel molecular dependencies in CRC, providing a foundation for precision oncology and future therapeutic innovation.



 Eliza Zimoląg, Joanna Szuszkiewicz, Marcin Serocki, Marcin Duleba, Oleksii Bryzghalov, Izabella Więckowska, Marcelina Chmiel, Barbara Lipert, Katarzyna Sarad, Julia Wirkijowska, Szymon Worszyło, Dariusz Klonski, Michał Mikula, Rafał Dziadziuszko, Andrew Thomason, Krzysztof Brzózka, Andrzej Mazan. Identification of novel molecular vulnerabilities in colorectal cancer through integrated transcriptomic profiling and functional genomics [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr B050.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8a3fb0b893ce7123a2e4b7841128bf8eca7bc83" target='_blank'>
              Abstract B050: Identification of novel molecular vulnerabilities in colorectal cancer through integrated transcriptomic profiling and functional genomics
              </a>
            </td>
          <td>
            E. Zimoląg, J. Szuszkiewicz, M. Serocki, M. Dulęba, O. Bryzghalov, Izabella Więckowska, M. Chmiel, B. Lipert, Katarzyna Sarad, Julia Wirkijowska, Szymon Worszyło, Dariusz Klonski, M. Mikula, Rafal Dziadziuszko, Andrew Thomason, Krzysztof Brzózka, Andrzej Mazan
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Breast cancer (BC) remains a leading cause of cancer-related mortality among women worldwide, with treatment resistance posing a significant clinical challenge. Circular RNAs (circRNAs), a class of non-coding RNAs, have gained increasing attention as key regulators of gene expression, influencing BC pathogenesis, progression, and therapeutic response. This review explores the mechanistic insights into circRNA functions in BC, focusing on their involvement in tumor proliferation, metabolic reprogramming, epithelial-mesenchymal transition (EMT), angiogenesis, metastasis, and apoptosis. Additionally, we highlight the crosstalk between circRNAs and microRNAs, emphasizing their potential as diagnostic and prognostic biomarkers. Beyond their roles in tumor biology, circRNAs are implicated in drug resistance, modulating responses to chemotherapy, targeted therapy, and endocrine treatment. Despite their promising applications, challenges remain, including the complexity of circRNA interactions, and the development of robust preclinical models. Addressing these challenges through interdisciplinary research integrating genomics, transcriptomics, and functional studies will pave the way for circRNA-based therapeutic strategies and personalized medicine approaches in BC management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c3e2c40f929a3a40c435801c1f0dae7da3b0326" target='_blank'>
              Circular RNAs orchestrating breast cancer hallmarks: bridging tumor biology and therapy resistance
              </a>
            </td>
          <td>
            Abdelhamid M. Abdelhamid, Shafei Ahmed Mai, Mariam Gamaleldin, Alaa Bassam Heikal, Wesam H Khidr, Nabil Hazza Al-Saadi, Ganna Magdy Kandil, Mahmoud M Omarn, Maha Alhelf
          </td>
          <td>2025-10-13</td>
          <td>Functional & Integrative Genomics</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="Mutations may cause changes in the structure and function of immune-related proteins, thereby affecting the operation of the immune system. Deep mutational scanning combines saturation mutagenesis, functional selection, and high-throughput sequencing to evaluate the effects of mutations on a large scale and with high resolution. By systematically and comprehensively analyzing the impact of mutations on the functions of immune-related proteins, the immune response mechanism can be better understood. However, each stage in deep mutation scanning has its limits, and the approach remains constrained in several ways. These include data and selection biases that affect the robustness of effect estimates, insufficient library coverage and editability leading to uneven representation of sites and alleles, system-induced biased signals that deviate phenotypes from their true physiological state, and imperfect models and statistical processing that limit extrapolation capabilities. Therefore, this technology still needs further development. Herein, we summarize the principles and methods of deep mutational scanning and discuss its application in immunological research. The aim is to provide insights into the broader application prospects of deep mutational scanning technology in immunology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fab3ce7657843d2dcbc129fae5b583a583f6457" target='_blank'>
              Deep Mutational Scanning in Immunology: Techniques and Applications
              </a>
            </td>
          <td>
            Chengwei Shao, Siyue Jia, Yue Li, Jingxin Li
          </td>
          <td>2025-10-01</td>
          <td>Pathogens</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Large parallel genetic screens have been used to identified targets and regulators that enhance T cell antitumor capability and persistence in tumor microenvironment. We hypothesized that by combining the pooled screen data from multiple independent genetic screens we could provide a systematic, comprehensive, and robust analysis of the effect of gene perturbation on T-cell based immunotherapies. After collecting data from previously published T cell screens, including CRISPR-based and ORF-based screens, through Gene Expression Omnibus (GEO), we reprocessed the gene hits summary and conducted a pathway enrichment analysis. A T cell screen perturbation score (TPS) metric was employed to quantify the impact of a gene perturbation on T cell function. Additionally, gene expression data (both bulk RNA level and single-cell RNA level) from autoimmune disease cohort and T cell-derived cancer patients were incorporated to gain further insight on gene perturbations that potentially augment T cell proliferation. We integrated all data and analysis on 35 T cells screens into our state-of-the-art T cell perturbation genomics database (TCPGdb), which is accessible through our webserver (http://tcpgdb.sidichenlab.org/) and allows users to interactively explore the impact of query genes on T cell function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f39e4489bcd5040b80166991d7202bfd2ac7d128" target='_blank'>
              TCPGdb: A comprehensive T cell perturbation genomics database for identification of critical T cell regulators.
              </a>
            </td>
          <td>
            Chuanpeng Dong, Feifei Zhang, Kaiyuan Tang, Nipun Verma, Xinxin Zhu, Di Feng, James Cai, Hongyu Zhao, Sidi Chen
          </td>
          <td>2025-11-21</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="


 Acute myeloid leukemia (AML) relapse after allogeneic hematopoietic cell transplantation (allo-HCT) is increasingly recognized as a consequence of immune escape mechanisms by which cancer cells evade the graft-versus-tumor effect. Recent studies have shown that, in most relapses, these immune changes are driven by epigenetic rewiring of AML cells, leading to two distinct and mutually exclusive immune evasion strategies: downregulation of HLA class II molecules or upregulation of T-cell inhibitory ligands (e.g., PD-L1, B7-H3) (Christopher et al., NEJM, 2018; Toffalori et al., Nat Med, 2019).
 To investigate whether these relapse modalities are associated with distinct metabolic patterns, we analyzed 14 longitudinally collected sample pairs from AML patients at diagnosis and post-transplantation relapse, evenly distributed between the two modalities. Following AML blast purification, we conducted bulk RNA sequencing and liquid chromatography–mass spectrometry (LC-MS) profiling of intracellular metabolites (endometabolites) and extracellular metabolites from supernatants after short-term culture in a metabolically defined medium (exometabolites). In parallel, we functionally characterized these samples using the Single Cell ENergetIc metabolism by profilIng Translation inhibition (SCENITH) assay (Argüello et al., Cell Metabolism, 2020).
 In line with our previous studies, transcriptional profiling clearly separated the two patterns of relapse: downregulation of HLA class II molecules (downHLA, n=7), and upregulation of T cell inhibitory ligands (upInib, n=7).
 Notably, we observed that most transcriptional and metabolic alterations were exclusive to each modality, with more evident metabolic rewiring in upInib relapses. In particular, biological processes enriched in these relapses included hypoxia (p=0.035) and nitric oxide (NO) biosynthesis (p=0.029), and endometabolome analysis highlighted increased intracellular levels of citrulline (FC=2.22, raw p=0.015) and arginine metabolites (FC=1.44, raw p=0.046), which are known substrates in nitric oxide synthesis. Notably, we also detected marked increase in intracellular pyroglutamic acid (FC= 1.71, raw p=0.046), a key intermediate in the γ-glutamyl cycle, suggesting a transcriptional and metabolic shift towards a pro-oxidative/nitrosative state at relapse.
 These findings indicate that upInib relapses adapt to a pro-inflammatory and pro-oxidative microenvironment and may rely more heavily on tumor microenvironment (TME)-driven metabolic reprogramming than their downHLA counterparts.
 Our analysis also revealed metabolic changes common to both relapse modalities. For instance, SCENITH demonstrated similar alterations in bioenergetic pathways in the two types of relapse, with increased mitochondrial dependence and reduced glycolytic capacity (p = 0.015). Furthermore, independently from the immune escape modality, in the exometabolome we detected a significant increase in lactic acid (LA) at relapse (p=0.007), which was confirmed by a validation colorimetric assay performed using both primary patient samples and AML patient-derived xenografts. Besides its known activity as immunosuppressive oncometabolite, LA has recently been shown to function also as a direct epigenetic regulator (Zhang et al, Nature, 2019). For this reason, we investigated by immunofluorescence analysis lactylation of specific hystone sites in our relapses, detecting a significant increase in H4K8la (p=0.04). CUT&Tag sequencing is currently underway to identify differentially lactylated genes and thus the role of pan- and residue-specific lactylation in driving relapse-associated gene expression programs.Taken together, our data provide a detailed landscape of the metabolic alterations associated with AML relapse, highlighting that most of them are modality-specific. These findings may serve as a roadmap to test therapeutic approaches aimed to untangle the epigenetic/metabolic TME rewiring driven by AML relapse, especially for upInib cases in which direct targeting by checkpoint blockade has yet largely failed at improving patient outcome.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2210ec6a9b70ffb5e5005ce8aa49e5a98d749447" target='_blank'>
              Metabolic signatures mirror divergent modalities of leukemia immune evasion and relapse after transplantation
              </a>
            </td>
          <td>
            Annalisa Tameni, Emanuele Rondelli, Angela Gentile, Vittorio Niccolucci, Maria Chiara Di Bonito, Fatemeh Kordevani, Francesca Tortorelli, Simone Cardaci, R. Di Micco, D. Lazarević, Giovanni Tonon, D. Sumpton, Alejandro Huerta Uribe, Marco Punta, C. Toffalori, F. Ciceri, L. Vago
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Drug resistance poses a major global health challenge, necessitating the development of effective therapeutic strategies. The main challenge is to predict drug-resistant mutations and design drugs that retain efficacy against such evolving targets. Our previous effort in computing the vitality value has provided a framework in assessing drug resistance. While promising, the approach lacked accuracy due to insufficient information about mutation tendencies and protein stability. In this study, we used the Stanford University HIV Drug Resistance Database and observed that drug resistance, usually quantified as [Formula: see text], exhibits a positive correlation with the Maximum Entropy energy, [Formula: see text]. However, both drug resistance and vitality are also correlated with the number of mutations, indicating that the virus cannot easily gain resistance through specific mutational pathways and must sacrifice stability and function to escape inhibition. To overcome this number dependence, we looked for a system with less extensive mutagenesis and chose HCV protease. In this case, resistance substitutions cluster at low [Formula: see text] values, reflecting a limited mutational space. This restricted landscape enables [Formula: see text] to predict evolutionary pathways of resistance and to guide the identification of robust therapeutic candidates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fc38f3191fe3cd0d179e733fa7821cca3748a5b" target='_blank'>
              Exploring evolutionary trajectories of drug resistance.
              </a>
            </td>
          <td>
            Linfeng Hu, Aoxuan Zhang, A. Warshel
          </td>
          <td>2025-11-04</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>108</td>
        </tr>

        <tr id="CRISPR screening has revolutionized cancer research by enabling systematic, genome-wide interrogation of gene
function to identify therapeutic vulnerabilities, resistance mechanisms, and immunotherapy determinants. This review
examines methodological approaches for CRISPR screenings using the Cas9 systems, evaluating how system and methodology selection
profoundly influence which biological mechanisms are revealed across therapeutic resistance, metastasis regulation, and immunotherapy
response. Cas9-based screens-from simple one-cell-type (1CT) in vitro approaches to complex autochthonous in vivo systems-excel at
identifying cell-autonomous DNA-level dependencies. Two-cell-type (2CT) co-culture approaches validate immune-dependent mechanisms,
while in vivo screens capture physiological complexity and have identified clinically relevant targets including PTPN2, now in Phase 1 trials.
Critical challenges include incomplete gene editing, false-positive and false-negative hits, context-dependent essentiality, and system-specific
technical limitations. Emerging technologies integrating artificial intelligence, prime editing, single-cell profiling, and patient-derived models
promise to address limitations while enabling functional precision medicine. This review provides a framework for selecting appropriate
systems, interpreting results within methodological constraints, and integrating complementary approaches to maximize biological insight and
translational impact.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6e21f4ff0a121855b3f14e7fcc069197b5d3507" target='_blank'>
              CRISPR- CAS SCREENING IN CANCER RESEARCH: METHODOLOGIES, APPLICATIONS, AND TRANSLATIONAL CHALLENGES
              </a>
            </td>
          <td>
            Ahana Patel
          </td>
          <td>2025-11-01</td>
          <td>INDIAN JOURNAL OF APPLIED RESEARCH</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Large B-cell lymphoma (LBCL) is an aggressive hematological malignancy characterized by significant clinical and genetic heterogeneity, resulting in variable prognosis and treatment response.
 It has become increasingly evident that the elements of the lymphoma tumor microenvironment (TME) impact on cancer progression and drug resistance. The TME features may be heterogeneous in LBCL and related to specific molecular subtypes, such as those characterized by the dark-zone (DZ) gene signature. MicroRNAs (miRNAs), small non-coding RNAs that regulate gene expression, play a pivotal and multifarious role in cancer biology by directly influencing tumor cell behavior, regulating immune evasion, and shaping TME.
 In this study, we performed a transcriptome analysis by total RNA sequencing on tumor samples from 39 de novo LBCL patients treated with R-CHOP first-line treatment, 20 refractory/relapsed (R/R) patients, and 19 therapy-responders. Using the Molecular Functional Portraits bioinformatic tool, we classified tumors according to gene expression profiles into prognostically relevant TME subtypes. Notably, R/R patients were enriched for the “immune-desert” (ID) subtype, which is associated with the poorest clinical outcomes.
 Given that the high-risk DZ-signature subtype has been previously characterized by an immunosuppressive TME, we examined the overlap between the gene expression profiles of our ID samples and the DZ signature. Remarkably, 84 out of 104 genes composing the DZ signature showed similar modulation in ID samples, underscoring a strong correlation between the DZ signature and the ID phenotype.
 To investigate the role of miRNAs in shaping TME subtypes, a small-RNA sequencing was performed on samples from the same patient cohort. We found 124 miRNAs differentially expressed between ID patients and those from other TME groups. We observed a significant positive correlation (Spearman's R = 0.35; p=0.00012) between miRNA expression changes linked to TME subtypes (ID type vs other TME types) and those associated with treatment response (R/R vs responding patients), highlighting the connection between miRNA-mediated TME remodeling and clinical outcomes. Of note, four of the identified TME-related miRNAs—miR-130b-3p (upregulated in ID and R/R patients), miR-100-5p, miR-99b-5p, and miR-125a-5p (all downregulated in ID and R/R patients)— were also able to accurately discriminate R/R from responsive patients with an area under the ROC curve >0.67. These miRNAs have previously been implicated in modulating the tumor immune microenvironment across various solid and hematologic malignancies.
 Moreover, an anticorrelated target and pathways analysis shows that genes targeted by the identified TME-related miRNA signature were enriched in pathways critical for TME regulation and tumor immune evasion, such as the Notch and IL6-JAK-STAT3 signaling pathways. Deconvolution analyses are ongoing to identify which TME components differ between tumors with high versus low expression of this four-miRNA signature. This will allow a better understanding of the molecular mechanisms by which these miRNAs influence the tumor microenvironment and treatment response. We will further validate all these data in a larger cohort of LBCL patients as well as in cellular models of resistance to R-CHOP.
 Altogether, our results represent a comprehensive, multi-layered approach to understanding how microRNAs contribute to drug resistance by reshaping the TME in LBCL and may pave the way for novel combined therapeutic strategies, holding significant promise for advancing precision medicine.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd5e42361d62ed7960e9e705935b24b001e66c4e" target='_blank'>
              An immune desert phenotype underlies therapy resistance in LBCL through distinct microenvironment-related miRNA profiles
              </a>
            </td>
          <td>
            Giulia Regazzo, G. Vari, F. Marchesi, F. Palombi, Andrea Sacconi, Giulia Orlandi, M. Novello, E. Papa, Martina Tomassi, Francesco Bertoni, E. Maiolo, A. Mengarelli, S. Hohaus, M. Rizzo
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="


 β-hemoglobinopathies, the most common monogenetic blood disorders, result from defective function or inadequate production of β-globin. Unfortunately, accessible and efficient treatments remain limited. Previous studies demonstrated that β-hemoglobinopathies patients with increased naturally occurring β-like subunit, γ-globin, a component of fetal hemoglobin (HbF), exhibit milder disease severity. Investigating novel factors that upregulate γ-globin expression could provide new therapeutic avenues for patients affected with β-hemoglobinopathies.
 In this study, we performed an unbiased genome-wide pooled CRISPR activation (CRISPRa) screen to identify novel γ-globin regulators. First, we generated a clonal HUDEP-2 cell line stably expressing the MPH activation complex (MS2-P65-HSF1). This cell line was then transduced with a lentiviral library expressing a VP64 activation element fused to a catalytically dead Cas9, a Blasticidin resistant gene, and one of 3 sgRNAs targeting virtually every gene in the human genome. On day 8 of differentiation, intracellular HbF staining was performed, and the 10% highest and lowest HbF-expressing cells (F-Cells) were sorted. sgRNAs were deep sequenced and compared between high and low γ-globin expressing cells. Expectedly, sgRNAs targeting the previously reported γ-globin repressor, BCL11A, were enriched in low γ-globin expressing cells, while sgRNAs targeting the γ-globin promotor were enriched in the high γ-globin expressing cells. Interestingly, this screen also identified several previously unknown putative γ-globin activators.
 To validate our screen results, we first chose the top 25 candidate genes for which sgRNAs were enriched in high HbF-expressing cells. For each candidate gene, we generated lentiviral particles expressing an sgRNA that targets it, VP64 fused to a catalytically dead Cas9, and a Blasticidin resistant cassette and then transduced into HUDEP-2 cells expressing MPH. Differentiated HUDEP-2-MPH cells were analyzed for percent F-Cells by flow cytometry and HBG1/2 (which encode γ-globin) mRNA levels by RT-qPCR. Compared to control cells transduced with non-targeting sgRNAs, activation of HBG2, HIC1, CHCHD7, FXR2, ANKH and SKIL resulted in >2-fold increase in %F-Cells and HBG1/2 mRNA levels. Since HIC1 is paralogous to HIC2, we confirmed that the HIC1-activating sgRNA does not additionally activate HIC2. Furthermore, we showed that HIC1 activation results in reduced BCL11A mRNA, suggesting that HIC1 may induce HbF production by suppressing BCL11A, a hypothesis we are currently testing.
 Next, we sought to validate our findings in primary human erythroid cells. To do so, we co-transduced primary human hematopoietic stem and progenitor cells (HSPCs) undergoing erythroid differentiation with two lentiviral constructs, one expressing MPH and a hygromycin resistant cassette and another expressing an sgRNA targeting a gene of interest, VP64-dCas9, and a Blasticidin resistance cassette. We found that, compared to control cells transduced with non-targeting sgRNAs, activation of HBG2, HIC1, CHCHD7, FXR2, ANKH and SKIL led to >3-fold increase in γ-globin mRNA levels, validating our HUDEP-2 findings described above. In additional preliminary results, we found that HIC1 and CHCHD7 cDNA overexpression in HSPCs resulted in increased %F-Cells and γ-globin mRNA. Further studies aimed at defining the mechanisms by which activation of the genes identified by our screen result in HbF induction are ongoing.
 In summary, we identified novel regulators of γ-globin expression. Our findings have important implications for furthering our understanding of β-globin switching, with important implications for β-hemoglobinopathies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b5f338b0fc7d497ea0e52566fa71e75ee9058a9" target='_blank'>
              A genome-wide CRISPR activation screen identifies novel γ-globin regulators
              </a>
            </td>
          <td>
            Carea Mullin, Ginette Balbin-Cuesta, Greggory Myers, Claire Drysdale, Zesen Lin, Lei Yu, Beth McGee, Ann Friedman, R. Khoriaty
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="


 Introduction: ASXL1-mutant (mt) Chronic Myelomonocytic Leukemia (CMML) is an aggressive myeloid neoplasm characterized by increased proliferation, resistance to epigenetic therapies, and poor survival. Epigenetic remodeling and up-regulation of leukemogenic driver genes are hallmarks of ASXL1mt CMML. Truncating mutations in ASXL1 activate enhancers that drive the expression of key leukemogenic driver genes such as HOXA9 and its co-factor MEIS1. Here, we investigate the chromatin accessibility of these leukemogenic enhancers by ASXL1 genotype at single-cell resolution (GoT-ChA-seq).
 Methods: After IRB approval, we performed bulk RNA-seq and ChIP-seq (H3K4me1, H3K4me3, H3K27ac, H3K27me3, H2AK119ub, H2BK20ac) on bone marrow (BM) mononuclear cells from 40 patients with CMML (19 ASXL1mt, 21 -wildtype/wt). After quality control, differential analyses of gene expression and histone modification occupancy between ASXL1mt and wt patients were conducted. A consensus peak calling framework was used to define genomic regions with ASXL1mt-specific histone modifications. Canonical enhancers were defined as genomic regions with a co-occupancy of H3K4me1 and H3K27ac. We then added regions marked by H2BK20ac to capture regulatory regions associated with p300 (Narita et al. Nat Genet 2023). We used an in silico prediction strategy to associate regulatory regions marked by strong histone modification occupancy with highly up-regulated leukemogenic driver genes (HOXA6-10, MEIS1). The ReMap database (Hammal et al. Nucleic Acids Res 2022) was queried for transcription factor binding in the genomic regions of interest. We then performed GoT-ChA-seq (Izzo et al. Nature 2024) to investigate chromatin accessibility for ASXL1mt (n=3) and wildtype BM single cells from CMML patients. In vitro colony forming assays were conducted on primary CMML BM cells (11- ASXL1mt and 9 wt CMML), assessing the efficacy of EP31670, a novel, oral, dual inhibitor of p300/BRD4.
 Results: This study included 40 patients with CMML, 19 ASXL1mt and 21 wt. The variant allele frequencies for ASXL1 were compatible with heterozygosity (median 0.41) and TET2 mutations were balanced between the two groups. We identified 149 candidate regulatory regions correlating with the expression of up-regulated target genes of interest (HOXA6-10, MEIS1) by bulk ChIP- and RNA-seq. Publicly available ChIP-seq data from myeloid cell lines demonstrated enrichment of p300 (in THP-1 cells, enrichment effect size +2.58, p=0.002) and BRD4 (in K-562 cells, enrichment effect size +1.58, p=0.045) in these candidate regulatory regions. Next, we interrogated the chromatin accessibility of ASXL1mt (n=13620) and ASXL1wt (n=24269) single cells from 3 CMML patients using GoT-ChA-seq. Differential accessibility analysis revealed 144 genomic regions, all of which were preferentially accessible in ASXL1mt cells compared to ASXL1wt cells (p<0.05 for all regions). Four of these preferentially accessible genomic regions (p<0.001 for all regions) overlapped with the previously identified candidate regulatory regions for MEIS1. The H3K27ac and H2BK20ac occupancy levels in these regions was strongly associated with the expression of MEIS1 (median Pearson r=0.73, p=1.16x10-7). These candidate regulatory regions were annotated in ENCODE as distal and proximal enhancers. Targeting this biology with the dual p300/BRD4 inhibitor EP31670 in progenitor colony forming assays led to a preferential therapeutic effect in ASXL1mt BM samples (median IC50 = 25nM) compared to in ASXL1wt samples (median IC50 = 74nM, p=0.03). Treatment with EP31670 was also associated with increased differentiation (increased blast forming units-erythroid; BFU-E) and organization of colonies in ASXL1mt vs wt CMML.
 Conclusions: ASXL1mt CMML is characterized by the up-regulation of several leukemogenic driver genes including HOXA9 and its co-factor MEIS1, secondary to the activation of genotype-specific enhancers. The preferential accessibility of these enhancers can be defined between ASXL1mt and ASXL1wt cells within patients. The enhancers of interest interact with p300 and BRD4 in myeloid cell lines and targeting them with the novel dual p300/BRD4 inhibitor, EP31670, induces therapeutic responses at low nanomolar drug concentrations in vitro, preferentially in ASXL1-mutant cells. These data solidify the biological rationale for targeting p300/BRD4 in ASXL1mt chronic myeloid neoplasms for therapeutic benefit (NCT05488548).
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d58c09cc34747d2000f4bf32710a735358c8669" target='_blank'>
              Clone-specific epigenetic regulatory mechanisms in ASXL1-mutant chronic myelomonocytic leukemia
              </a>
            </td>
          <td>
            Moritz Binder, T. Lasho, Jenna A. Fernandez, Wazim Mohammed Ismail, Amelia Mazzone, C. Finke, Pankaj Pradeep, A. Mangaonkar, Nickolas Steinauer, Huihuang Yan, Liguo Wang, Alexandre Gaspar-Maia, Mrinal Patnaik
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fbb0b650183e1ab55b2143b3779619fab2e24dd" target='_blank'>
              aiAtlas: High-Fidelity Cell Simulations of Genetic Perturbations in Rare Diseases and Cancers
              </a>
            </td>
          <td>
            WR Danter
          </td>
          <td>2025-10-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer is increasingly recognized not as a fixed genetic condition but as a dynamic and plastic disease state driven by reversible epigenetic, transcriptional and microenvironmental cues. This evolving understanding supports a therapeutic paradigm shift: from eradicating malignant cells to reprogramming them toward quiescence, differentiation or functional normalization. This review explores diverse strategies for redirecting cancer cell fate, including differentiation therapy, epigenetic remodeling, lineage reprogramming, senescence induction and tumor microenvironment resetting. These approaches exploit intrinsic cellular plasticity and contextual adaptability, offering novel avenues to contain malignancy and overcome resistance. By reframing cancer as a potentially reversible phenotype, this therapeutic strategy demands redefinition of clinical endpoints, incorporation of dynamic biomarkers, and development of integrative treatment frameworks. Ultimately, reprogramming‑based oncology holds the promise of transforming aggressive malignancies into manageable conditions while minimizing the collateral damage associated with conventional cytotoxic therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99bfdfe1c6957af978b8fb4415d04fd96bdf7b85" target='_blank'>
              Beyond eradication: Therapeutic reprogramming strategies for cancer normalization (Review).
              </a>
            </td>
          <td>
            Aliasgar F. Shahiwala
          </td>
          <td>2025-12-04</td>
          <td>Oncology reports</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Hematopoietic stem cell (HSC) therapy remains essential in treating blood disorders, autoimmune diseases, neurodegenerative conditions, and cancers. Despite its potential, challenges arise from the inherent heterogeneity of HSCs and the complexity of their regulatory niche. Recent advancements in single-cell RNA sequencing and chromatin accessibility sequencing have provided deeper insights into HSC markers and chromatin dynamics, highlighting the intricate balance between intrinsic and extrinsic regulatory mechanisms. Zebrafish models have emerged as valuable tools in HSC research, particularly through live imaging and cellular barcoding techniques. These models have allowed us to describe critical interactions between HSCs and embryonic macrophages, involving reactive oxygen species and calreticulin signaling. These are essential for ensuring HSC quality and proper differentiation, with implications for improving HSC transplant outcomes. Furthermore, the review examines clonal hematopoiesis, with a focus on mutations in epigenetic regulators such as DNMT3A, TET2, and ASXL1, which elevate the risk of myelodysplastic syndromes and acute myeloid leukemia. Emerging technologies, including in vivo cellular barcoding and CRISPR-Cas9 gene editing, are being investigated to enhance clonal diversity and target specific mutations, offering potential strategies to mitigate these risks. Additionally, macrophages play a pivotal role in maintaining HSC clonality and ensuring niche localization. Interactions mediated by factors such as VCAM-1 and CXCL12/CXCR4 signaling are crucial for HSC homing and the stress response, opening new therapeutic avenues for enhancing HSC transplantation success and addressing clonal hematopoiesis. This review synthesizes findings from zebrafish models, cutting-edge sequencing technologies, and novel therapeutic strategies, offering a comprehensive framework for advancing HSC biology and improving clinical outcomes in stem cell therapy and the treatment of hematologic diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6985cdf565a94b5e4ce65dadd2a9ba4efa0f62f7" target='_blank'>
              Advancements in Hematopoietic Stem Cell Therapy: From Biological Pathways to Emerging Therapeutic Strategies
              </a>
            </td>
          <td>
            Viviana Cortiana, H. Chorya, R. H. Abbas, Jade Gambill, Adhith Theyver, Chandler Park, Yan Leyfman
          </td>
          <td>2025-10-15</td>
          <td>Therapeutics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Germline mutations in ETV6 cause thrombocytopenia and predisposition to malignancies. ETV6 (ETS Variant 6) is a transcriptional repressor essential for embryonic development and bone marrow hematopoiesis. Several variants have been identified in ETV6 and while mutations in the ETS DNA-binding domain of ETV6 impair direct DNA binding, the P214L mutation in the central domain leads to aberrant cellular localization of ETV6, decreasing transcriptional repression, and impairing megakaryocyte (MK) maturation. Despite its critical role in hematopoiesis, very few direct transcriptional targets of ETV6 have been identified in MK development. We sought to define ETV6's genome-wide binding sites and the impact of ETV6 P214L variant in differentiated MK.
 Using CRISPR/Cas9, we generated ETV6 knockout (KO) and P214L knock-in variants in human CD34⁺ hematopoietic stem cells, derived from cord blood, and then differentiated into megakaryocytes over 14 days. We performed Cleavage Under Target and Release Under Nuclease (CUT&RUN) to map ETV6 binding, assay for transposase-accessible chromatin sequencing (ATAC-seq) to assess chromatin accessibility, and bulk RNA-seq to profile transcriptional changes.
 ATAC-seq profiling demonstrated that the overall chromatin accessibility remained unchanged between controls and ETV6 KO cells, as well as between silent control and P214L mutants, indicating that ETV6 loss or mutation affects transcriptional regulation without altering global chromatin accessibility.
 CUT&RUN analysis revealed ETV6 binding activity in both wild-type and silent control (CRISPR/Cas9) megakaryocytes. In wild-type cells, we detected 1,757 peaks in replicate 1 and 3,570 peaks in replicate 2, with 974 target genes shared between both replicates. Similarly, silent control megakaryocytes identified 2,853 and 3,104 peaks in replicates 1 and 2, respectively, with 1,592 shared target genes. 722 genes were commonly bound by ETV6 in both WT and silent control megakaryocytes. ETV6 binding was drastically reduced in ETV6 KO cells (181 peaks in Rep 1 and 67 peaks in Rep 2) and in P214L mutants (326 peaks in Rep1 and 266 peaks in Rep 2). Over 60% of ETV6 peaks localized to promoters in controls, whereas ETV6 peaks in KO and mutants were largely in non-promoter regions. Motif analysis (MEME suite) of ETV6 binding sites revealed that ETS motifs were most significantly enriched in WT (E-value= 2.0e-424) and silent control sample (E-value= 4.2e-180), with ETV6 as the predominant ETS transcription factor. This ETS motif enrichment was lost in both ETV6 KO and P214L mutant cells. We also performed immunofluorescence and cell spreading assays in ETV6 knockout and wild-type megakaryocytes to assess their ability to form proplatelets. Both experiments showed that loss of ETV6 disrupts cytoskeletal organization in ETV6 KO megakaryocytes, preventing proper proplatelet formation indicating a critical role for ETV6 in late-stage megakaryocyte maturation.
 RNA-seq analysis revealed that among the common 722 target genes, transcripts for 44 of the genes were altered (41 upregulated and 3 downregulated) in the ETV6 KO; and 21 were altered in the P214L variant (18 upregulated and 3 down-regulated) with 18 of these up-regulated gene in P214L variant common among ETV6 KO as well. Pathway analysis identified significant enrichment of the estrogen signaling pathway (p = 0.0008), interleukin-2 signaling pathway (p = 0.0003), and MAP kinase pathway (p = 0.000008). A subset of genes, including ESR2, SHC1, SYK, and GNAI2, were commonly represented across these pathways, suggesting that disruption of ETV6 function may converge on these signaling networks to impair megakaryocyte development. We compared our data with previously published datasets on ETV6 mutations and, while we observed several overlapping genes, we also identified novel targets in our dataset including ESR2. When we analyzed its expression, ESR2 levels were ~7-fold higher in ETV6 KO megakaryocytes compared to wild type and ~5-fold higher in the P214L variant compared to the silent control. Prior studies have shown that estrogen supports megakaryocyte growth and maturation by activating GATA1 through ERβ, which in turn triggers STAT1 signaling. Our findings suggest that altered ESR2 expression may influence signaling pathway, offering new insights into how estrogen signaling might impact platelet production and how its disruption could impair megakaryocyte development.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b67bfb55ff0d7deeb31a92c2c0db80c89d1bc777" target='_blank'>
              Integration of Cut&Run, atac-seq and RNA-seq analyses defines the impact of germline ETV6 mutations on chromatin binding and gene expression in megakaryocytes
              </a>
            </td>
          <td>
            Hyder Mir, Jesse Rowley, Lilian Antunes Heck, Thomas J. Girard, Wei Yang, Abigail Ajanel, Jorge Di Paola
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Gliomas carry a high disease burden, particularly the aggressive IDH-wildtype subtype. However, some IDH-mutant gliomas can transition from a less aggressive G-CIMP-High (GCH) state to a more malignant G-CIMP-Low (GCL) phenotype, mimicking IDH-wildtype tumors. Understanding the cellular and molecular mechanisms behind this progression is vital for developing effective therapies.



 We propose that single-nuclei resolution analysis can reveal distinct neoplastic cell subtypes driving the GCH-to-GCL transition through activation of stemness- and proliferation-related gene programs.



 We analyzed 22 glioma specimens from 15 patients (including matched primary and recurrent tumors) using single-nuclei RNA- and ATAC-sequencing. After quality control and neoplastic cell identification via somatic CNVs, we applied non-negative matrix factorization (NMF) to identify cell subclusters based on gene expression patterns. We further characterized these clusters using gene set enrichment, protein-protein interaction networks, developmental trajectory analysis, and integration with external RNA-seq datasets. Chromatin accessibility was assessed via snATAC-seq to identify regulatory elements and transcription factor activity.



 We identified six major neoplastic cell subtypes: Astrocytic-Support, Pro-neural Housekeepers, Proteo-synthetic Cells, Mitotic Proliferators, Neural Guidance & Growth Cells, and Mesenchymal-mutant Cells. Each showed distinct gene expression signatures, pathway enrichments, and survival associations. Two subtypes—Mitotic Proliferators and Mesenchymal-mutant Cells—were linked to high-grade tumors and poor patient outcomes. Chromatin accessibility patterns corroborated transcriptional findings, highlighting differential TF regulation in aggressive subtypes.



 This study reveals two aggressive GCL-related neoplastic cell subtypes with distinct stem-like and proliferative features. These findings provide insight into glioma progression mechanisms and uncover potential molecular targets for therapeutic intervention in IDH-mutant gliomas.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd05421cc91e3cb0411e5625733c706631851341" target='_blank'>
              EPCO-21. SINGLE-NUCLEI ANALYSIS IDENTIFIES NOVEL NEOPLASTIC IDH MUTANT ASTROCYTOMA PROLIFERATIVE AND STEMNESS PHENOTYPES DRIVING POOR PATIENT OUTCOMES
              </a>
            </td>
          <td>
            H. Noushmehr, G. Herrgott, L. Garofano, Ian Lee, J. Snyder, T. Walbert, A. Castro, A. Iavarone
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="
 Clustered regularly interspaced palindromic repeats (CRISPR) gene editing, known for its precision, is revolutionizing tumor research and treatment. This article covers the evolution of the CRISPR system from Cas9, which produces DNA double‐strand breaks, to Cas12, Cas13, base editors, and prime editors. These breakthroughs allow DNA/RNA editing, transcriptional regulation, and epigenetic reprogramming. The study encompasses CRISPR's diverse applications in tumor research, including high‐throughput screening of cancer driver genes, identification of synthetic lethal targets, and analysis of drug resistance pathways across various cancer types. It highlights CRISPR's importance in therapeutic tactics, such as targeting oncogene inactivation, restoring tumor suppressor gene activity, and engineering immune effector cells, such as CAR T and NK cells, to increase persistence, cytotoxicity, and resistance to exhaustion. This review also examines CRISPR's role in modifying the tumor microenvironment (TME), encompassing the regulation of immune checkpoints, metabolic reprogramming, and the targeted eradication of specific immunosuppressive cells. Off‐target consequences for both delivery and efficiency are discussed, along with cutting‐edge technologies including high‐fidelity Cas variants, AI‐assisted sgRNA design, and stimuli‐responsive delivery systems. Finally, the paper examines how CRISPR can be integrated with combination medicines, multimodal editing methods, and single‐cell and multiomics data to deliver tailored precision‐based interventions. This review explains the molecular logic and translational routes of CRISPR, highlighting its promise to revolutionize precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7f333b854e7e8b8178a62bd98bd7d363744559a" target='_blank'>
              CRISPR Enabled Precision Oncology: From Gene Editing to Tumor Microenvironment Remodeling
              </a>
            </td>
          <td>
            Kailai Li, Peixin Huang, Yue Qian, Anqi Lin, Jingjun He, Junyi Shen, Li Chen, Kai Miao, Jian Zhang
          </td>
          <td>2025-11-05</td>
          <td>Med Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Cancer immunotherapy has revolutionized cancer treatment, offering durable responses in a subset of patients. Despite this progress, cancer cells have developed sophisticated mechanisms to evade immune surveillance, including genetic mutations, activation of immune escape pathways, alterations in the tumor microenvironment, and metabolic reprogramming. Additionally, the immune system’s ability to effectively target cancer cells can be compromised by immune tolerance and functional exhaustion. There is growing evidence that abnormal activation of cancer signaling pathways, such as the Wnt/β-catenin, Notch, JAK/STAT, p53, and PTEN pathways, acts independently or through crosstalk to form complex regulatory networks that influence tumor immune escape, the therapeutic response, and immune resistance. Consequently, elucidating the mechanisms by which these signaling pathways contribute to tumor immune escape is essential for optimizing current immunotherapy regimens and developing novel therapeutic strategies. Targeting these pathways in combination with immunotherapy represents a promising approach to overcome immune resistance and enhance treatment efficacy. The integration of pathway modulation with immunotherapeutic interventions may reveal new avenues for more effective and durable cancer treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9142ff6a5cec86bc497f410ddb74aab33a5d7d49" target='_blank'>
              From intracellular drivers to immune modulators: emerging paradigms in oncogenic pathway-directed immunotherapy optimization
              </a>
            </td>
          <td>
            Dean Rao, Tiantian Wang, Chenan Fu, Yiming Luo, Junli Lu, Zhoubin Sun, Wei Yan, Huifang Liang, Limin Xia, Wenjie Huang
          </td>
          <td>2025-10-22</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>1</td>
          <td>27</td>
        </tr>

        <tr id="Background Proto-oncogenes are abnormally activated in nearly all types of tumors. However, the epigenetic mechanism of proto-oncogene activation has not yet been well elucidated. Methods The present study involved the construction of a double-stranded cDNA library derived from gastrointestinal cancer cells, followed by high-throughput genome sequencing to select the c-MYC gene associated with colorectal cancer. Through RACE analysis, we identified the antisense RNA MYC-AS1 and its complete sequence. By investigating the cellular functions, expression of MYC-AS1, elucidating its interaction mechanism with the c-MYC gene, and exploring the impact of DNA methylation on MYC-AS1 expression, we uncover the fundamental principles and regulatory mechanisms underlying colorectal cancer development. Results Here, we show that a subset of proto-oncogenes,including c-MYC, possess antisense RNAs. Upregulation of c-MYC in cancer tissues was attributed to the silencing of its antisense RNA MYC-AS1 via DNA hypermethylation. MYC-AS1 RNA markedly inhibited the proliferation of cancer cells in vitro and impeded tumor growth in nude mice in vivo by repressing the expression of c-MYC via an RNAi mechanism. MYC-AS1 RNA bound directly to the HuR protein in the cytoplasm, enhancing the RNA stability of MYC-AS1. Furthermore, MYC-AS1 inhibited c-MYC-targeted gene LDHA expression. Unlike the well-characterized oncogenic long noncoding RNA PVT1, which is coamplified with MYC and enhances its stability, MYC-AS1 is epigenetically silenced and functions as a tumor suppressor through an RNAi mechanism, revealing a distinct layer of MYC regulation. Conclusion Our work provides a novel mechanism by which c-MYC is activated in cancer cells by epigenetic silencing of its antisense RNA, which functions as a tumor suppressor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c20b565d2f250856d2e173d09028c6873b6e2cb" target='_blank'>
              A novel mechanism by which c-MYC is aberrantly activated by epigenetic silencing of its antisense lncRNA in colon cancer
              </a>
            </td>
          <td>
            Xuming Hu, Ye Wei, Meiying Zhang, Chunfeng Dou, Liping Wang, Gul Zaib, Huixian Wu, Wang Guo, Xiaoyuan Wang, Shihao Chen, Qi Xu, Mingzhou Guo, Hengmi Cui
          </td>
          <td>2025-10-21</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a highly heterogeneous hematologic malignancy, characterized by complex molecular features and mechanisms of treatment resistance, which lead to a poor prognosis and high relapse rates. The complexity of multi-pathway interactions and the dysregulated dynamics of tumor cell death pathways may contribute to the wide range of clinical outcomes observed despite advancements in current therapies. Most current research focuses on a single form of cell death, neglecting the mechanisms of other death pathways and their synergistic interactions, which hinders the development of novel therapeutic approaches. The present review systematically integrates and compares the molecular features of key cell death modalities in AML, including autophagy, apoptosis, pyroptosis, necroptosis, ferroptosis and cuproptosis. The present review analyzes their specific triggers, signaling hubs and regulatory networks within the metabolic microenvironment, and discusses the dynamic crosstalk among these pathways. A key focus is the therapeutic potential of exploiting this crosstalk to design synergistic combination therapies. To overcome the limitations of conventional treatments and improve patient outcomes, it is essential to further investigate the transition mechanisms of various cell death modes in AML progression, drug resistance and relapse. Additionally, establishing a theoretical foundation for the development of innovative therapies that synergistically regulate multiple death pathways is crucial.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c85373acc9cc6f42ac985757b2e5e7f882fbf46" target='_blank'>
              Targeting cell death pathways in acute myeloid leukemia: Molecular mechanisms and clinical implications (Review)
              </a>
            </td>
          <td>
            Reaila Jianati, Haijing Chen, Xi Yang, Lixiang Yan, Yifei Guo, Chenyang Fan, Xiaogang Hao, Gengda Zhu, Zhexin Shi
          </td>
          <td>2025-10-14</td>
          <td>Oncology Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Brain tumors such as glioblastoma remain among the most lethal and immunologically resistant cancers, in large part due to epigenetic programs that sculpt the tumor–immune microenvironment. DNA methylation, histone modifications, and chromatin remodeling do not merely drive tumor-intrinsic changes; they also profoundly reprogram immune responses, shaping antigen presentation, cytokine signaling, and immune cell recruitment. At the center of this regulation are T cells, whose effector functions are suppressed through promoter hypermethylation of antigen-processing genes, silencing of interferon pathways, and the establishment of exhaustion-specific chromatin states. Mutations such as IDH1/2 and H3K27M further reinforce these epigenetic barriers, fostering immune-cold microenvironments that disable cytotoxic T-cell activity. Emerging evidence highlights both CNS-specific adaptations, including microglial and astrocytic epigenetic programs that reinforce immune privilege, and conserved features of T cell exhaustion that mirror those in peripheral cancers. This duality underscores the need for therapeutic strategies that dismantle CNS-specific barriers while leveraging shared exhaustion programs across tumor types. Epigenetic drugs, ranging from DNA methyltransferase and EZH2 inhibitors to BET degraders and CRISPR-based epigenome editors, are beginning to restore antigenicity, reverse T cell dysfunction, and sensitize tumors to checkpoint blockade. Yet these approaches carry the paradoxical risk of disrupting CNS immune tolerance, potentially triggering harmful neuroinflammation or autoimmunity. To our knowledge, this is among the first comprehensive reviews to integrate CNS-specific immune privilege mechanisms with peripheral exhaustion pathways, providing a unified perspective on how epigenetic regulation orchestrates immune dysfunction across central and peripheral contexts. By mapping the continuum between immune evasion and global immunosuppression, we propose a conceptual framework for tailoring epigenetic-immunotherapy combinations to achieve durable antitumor immunity in the CNS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d7202ed1c4ffc9d0ee982c0f1ad42403d845276" target='_blank'>
              Beyond the genome: epigenetic regulation of immune responses and T cells in brain tumors
              </a>
            </td>
          <td>
            Shuo Sun, Yu Han, Haiying Li, Chengyan Wang, Shu Zhou, Xiaowei Zhang, Shuhong Dai, Yao Peng, Zhuoqun Wang
          </td>
          <td>2025-11-18</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f662f4b181b3395cd096885302fa365b2511567e" target='_blank'>
              Exploring tumor heterogeneity and drug resistance in cervical cancer through single-cell and bulk transcriptomics.
              </a>
            </td>
          <td>
            Yinxia Yang, Sirong Cheng, Yuxuan Zhong, Pengyu Sun, Xiaoru Deng, Bohui Zhou
          </td>
          <td>2025-11-27</td>
          <td>Discover oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Tumor cells often display aberrant chromosome counts, or aneuploidy, which can be exacerbated by increased chromosome missegregation rates in a condition called chromosomal instability (CIN). Despite the prevalence of CIN in solid tumors and its correlation to poor patient prognosis, understanding how CIN status affects cancer aggression remains a contested issue. In this work, we present a novel mathematical model linking missegregation levels and degree of aneuploidy to the underlying fitness landscape, and examine how these fitness landscapes can affect drug efficacy.



 To model how chromosome counts can shift over time, we use a partial differential equation that contains a diffusion term corresponding to chromosome missegregation and a growth term proportional to the relative fitness of a particular chromosomal distribution. Using this framework, a fitness landscape that connects chromosome count to relative fitness can be inferred from a steady-state cellular population. Fluorescence in situ hybridization (FISH) was used to track chromosome counts in both CIN and non-CIN cell lines over the course of 2 months, with data collected for chromosome 10 individually and for the group of chromosomes 1, 5, and 19. Experimental results were then analyzed using our mathematical framework to determine fitness landscapes for both CIN and non-CIN cells.



 A partial differential equation was developed to model how chromosome counts change over time based on a diffusion-like term corresponding to lagging chromosome rate and a growth rate based on a chromosome count’s contribution to overall cellular fitness. The CIN lines exhibited chromosome count distributions that were statistically closer to normal distributions than non-CIN lines. When connected to our mathematical model, these distributions imply that the CIN lines have a flatter fitness landscape than their non-CIN counterparts. This finding reinforces the notion that CIN lines are tolerant of highly varying degrees of aneuploidy.



 Our work suggests an underlying mechanism for the clinical observation that CIN tumors are more resilient to treatment with taxanes and other microtubule or mitotic targeting drugs. The flatter fitness landscapes associated with CIN provide an explanation for why these drugs are less effective, since increasing or decreasing the number of chromosomes in a CIN line does not lead to a relatively large change in fitness.



 Kelly Brock, Marianna Kleyman, Madison Stoddard, T. Ryan Gregory, Andrew Chen, Arijit Chakravarty. Fitness landscape modeling indicates decreased effectiveness of microtubule assembly targeting drugs in tumors demonstrating chromosomal instability (CIN) [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C006.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d1879dcd4be6c24e75a32134afbe36fcff096e0" target='_blank'>
              Abstract C006: Fitness landscape modeling indicates decreased effectiveness of microtubule assembly targeting drugs in tumors demonstrating chromosomal instability (CIN)
              </a>
            </td>
          <td>
            Kelly Brock, Marianna Kleyman, M. Stoddard, T. R. Gregory, Andrew Chen, Arijit Chakravarty
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 The estimation of the in vitro potency of anticancer agents against tumor cell lines in tissue culture is known to be fraught with a high degree of variability. While quality control of experimental conditions has been demonstrated by others to reduce this variability, other sources of variability remain even after standardization of experimental conditions. In this poster, we assess two potential mechanisms for this residual variability: phenotypic stochasticity in response and heritable resistance to treatment.



 We developed three agent-based mathematical models representing different potential scenarios for cellular responses to anticancer therapy: (1) all cells have an equal probability of survival under treatment (phenotypic stochasticity) and future division; (2) sensitive cells deterministically die under treatment while resistant cells survive and may divide to produce resistant offspring (heritable resistance); (3) sensitive subclones have a greater probability of death under treatment compared to resistant subclones (both mechanisms). We implemented these models in MATLAB, running simulated in vitro experiments consisting of large (106) or small (500 or 5000) populations of virtual cells under treatment. The final size of these virtual cell populations serves as the readout of the virtual experiments. To assess the effect of phenotypic stochasticity and heritable resistance on variation in experimental outcomes, we simulated these experiments 10,000 times and report the coefficient of variation (CV).



 Variability in response to treatment was low under most simulated conditions. In the first model simulating phenotypic stochasticity alone, high coefficients of variation in assay outcome were observed only when the probability of survival under treatment was less than 10%. In simulations of deterministic heritable survival under treatment, the coefficient of variation was generally low. However, simulations incorporating both mechanisms exhibited substantial variability in experimental outcome, consistent with the observed phenomenon.



 Stochastic cellular response to treatment or heritable binary resistance alone are insufficient explanations for the variation in in vitro experimental results. On the other hand, the high degree of variation in experimental outcomes is consistent with a combination of heritable resistance and phenotypic stochasticity. Our findings are consistent with a role for Darwinian evolution in the response to anticancer treatment for cancer cells in tissue culture.



 Alexander L. Young, Liam Noronha, Madison Stoddard, T Ryan. Gregory, Arijit Chakravarty. Heritable resistance and phenotypic stochasticity are likely drivers of irreproducibility in experimental determination of in vitro potency of anticancer agents [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C073.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a885f4ecedc46db5c7868acd715fccb8dde4f83" target='_blank'>
              Abstract C073: Heritable resistance and phenotypic stochasticity are likely drivers of irreproducibility in experimental determination of
 in vitro
 potency of anticancer agents
              </a>
            </td>
          <td>
            Alexander L. Young, Liam Noronha, M. Stoddard, T. R. Gregory, Arijit Chakravarty
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Advanced prostate cancer (PrCa) remains a leading cause of cancer-related death among men due to disease progression in nearly all patients on standard of care therapy targeting the androgen receptor. An important mechanism driving therapeutic resistance is lineage plasticity, which enables PrCa cells to reprogram into lineage variants no longer dependent on androgen receptor signaling. As inhibitors of the histone methyltransferase EZH2 are being evaluated clinically for the treatment of advanced PrCa, we investigated here how EZH2 affects PrCa lineage plasticity. Data from genetically engineered mice and human clinical samples demonstrated that genetic or pharmacological suppression of EZH2 altered chromatin to expand active transcription factor programs. These changes in gene expression during PrCa progression increased the diversity of PrCa lineage variants that arose. EZH2 suppression did not inhibit disease progression or therapeutic resistance in this context. These findings advance the current understanding of PrCa lineage plasticity and suggest EZH2 inhibitors may be less effective in treating PrCa prone to lineage plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee66b00f883047cd2ed8c292d9f0f040e78b6ffe" target='_blank'>
              EZH2 Suppression Diversifies Prostate Cancer Lineage Variant Evolution and Lacks Efficacy in Inhibiting Disease Progression.
              </a>
            </td>
          <td>
            Justine J Jacobi, K. Wadosky, Neha Jaiswal, Xiaojing Zhang, Yanqing Wang, Prashant K Singh, Jie Wang, Eduardo Cortes Gomez, Jianmin Wang, Bo Xu, Mark D Long, Paloma Cejas, Shweta Kukreja, H. Long, David W. Goodrich
          </td>
          <td>2025-11-12</td>
          <td>Cancer research</td>
          <td>1</td>
          <td>17</td>
        </tr>

        <tr id="


 Introduction Acute myeloid leukaemia (AML) is a potentially fatal haematological malignancy characterized by the uncontrolled growth of leukemic cells (myeloid blasts), causing bone marrow failure. Despite recent advances in targeted therapeutics, high mortality rates persist across all age groups particularly the patients above the age of 60 years, emphasizing the need for novel and less toxic therapeutic strategies.
 KAT2A is a histone acetyltransferase involved in transcriptional regulation as part of two multisubunit complexes: Spt-Ada-Gcn5 acetyltransferase (SAGA) and Ada-Two-A-containing (ATAC), modulating lineage specificity and metabolic/cell-cycle pathways respectively. KAT2A has been shown to be upregulated in AML, where it plays a pivotal role in leukemic stem cell maintenance and chemotherapy resistance.
 Pharmacological inhibition of KAT2A has been shown to trigger differentiation in AML blasts while largely sparing normal haematopoietic progenitors, indicating that KAT2A blockade is a mechanistically selective therapeutic approach. We investigated the efficacy and mechanism of action of a novel proteolysis-targeting chimera (PROTAC) degrader; MDI0117218 which targets KAT2A as a differentiation-inducing therapeutic strategy in primary AML blasts.
 Methods AML cell lines (MOLM-13, KG1a, OCI-AML3, NB4, HL60) and a large cohort (n=50) of primary AML samples were assessed across a range of assays including: ATP-based proliferation assays (Cell-titre Glo), cytospin morphology, PKH26 cell division tracking and flow cytometric evaluation of differentiation-associated surface markers. KAT2A degradation and subsequent acetylation reduction was monitored by Western blot and intracellular flow for sub-population analysis. Long-term co-culture efficacy with HS5 stroma and colony assays were established to evaluate PROTAC effects on drug resistant adherent and putative stem cell fractions. Synergistic interaction with standard of care agents Cytarabine, Venetoclax and Azacitidine were assessed in combination and pretreatment-washout experiments.
 Results MDI0117218 showed nanomolar efficacy in AML cell lines (MOLM13 and OCI-AML3) and 48% of primary AML samples (n=50, Average EC50 33nM +/-324nM in sensitive primary AML samples, Ave 6.6uM +/- 1.4uM in resistant primary AML samples). This effect was observed across both differentiated and undifferentiated FAB types of AML.
 Western blotting in primary AML samples confirmed potent and rapid target engagement, with complete and sustained KAT2A degradation observed from 5nM within 30 minutes of incubation. Intracellular flow cytometry confirmed uniform reduction of KAT2A across blast sub-populations in sensitive patient samples compared to those with resistant EC50s. H3K9 acetylation reduction also correlated with KAT2A knockdown in sensitive samples providing potential clinical biomarker readouts. Assessment of MDI0117218 mechanism of action revealed variable cytostatic and differentiation responses with induction of CD11b +/- terminal CD14 differentiation in responders accompanied by cell cycle arrest without apoptotic induction consistent with myeloid differentiation which was confirmed by morphological analysis. Drug efficacy and differentiation was sustained in longer-term stromal co-culture and Colony forming assays, with significant reduction in all blast sub-populations including stromal adherent drug-resistant blasts and colony forming units (p<0.02, n=10). Combination drug assays in ex vivo primary AML blasts showed moderate to additive synergistic interaction between MDI0117218 and cytarabine (Average CI at ED50 0.9 +/- 0.85) and good synergy with Venetoclax (Average CI at ED50 of 0.25 +/- 0.09) across a range of doses. Pre-treatment and sequential washout protocols confirmed synergy to standard agents highlighting the rapid and sustained knockdown effects of MDI0117218.
 Conclusion MDI0117218 is an effective and rapid KAT2A degrader across multiple models of cell line and primary AML. Differentiation and anti-proliferative effects were observed across a wide range of differentiated and non-differentiated AML patients, highlighting the potential for a broad acting agent with potential for combination with both Cytarabine and Venetoclax; two mainstays of treatment for transplant-ineligible or relapsed AML. Taken together with favourable pre-clinical toxicology profiles, these data support the rationale for this agent as a strong candidate for early-phase clinical trials.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/733271b65f73cc3e215c90487cd2163804c35ba1" target='_blank'>
              Investigating KAT2A protac therapy for targeting leukemic blast differentiation in acute myeloid leukaemia.
              </a>
            </td>
          <td>
            Agisilaos Zerdelis, Claudia Chriches, Loren Waters, Laia Sadeghi, Darren Le Grand, Simon Ward, S. Knapper, J. Zabkiewicz
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 The advent of cancer immunotherapy, particularly immune checkpoint inhibitors, has revolutionised the treatment landscape for many malignancies. However, a significant proportion of patients exhibit primary, adaptive, or acquired resistance to these therapies, limiting their broad clinical benefit. Tumour microenvironment (TME), a complex and dynamic ecosystem of cancer cells, immune cells, stromal cells and extracellular matrix, has emerged as a critical determinant of immunotherapy response and a major driver of resistance.



 This review provides a comprehensive overview of the intricate mechanisms by which the TME fosters an immunosuppressive milieu, thereby impeding effective anti‐tumour immunity. We delineate the key cellular and molecular components of the TME that contribute to immunotherapy resistance, including tumour‐associated macrophages, myeloid‐derived suppressor cells, regulatory T cells, and cancer‐associated fibroblasts. We then discuss the diverse and innovative strategies being developed to reprogram the TME and overcome these resistance mechanisms. These emerging approaches include targeting immunosuppressive cell populations, modulating tumour metabolism, epigenetic reprogramming, normalising the tumour vasculature and employing engineered cell therapies such as chimeric antigen receptor T cells and oncolytic viruses. The potential of combination therapies to synergistically dismantle the immunosuppressive TME is also explored.



 Reprogramming the TME represents a paradigm shift in cancer treatment, moving beyond direct tumour cell targeting to modulate the entire tumour ecosystem. While significant challenges remain, including TME heterogeneity and the need for predictive biomarkers, the strategies discussed herein hold immense promise for overcoming immunotherapy resistance and improving patient outcomes. A deeper understanding of the complex interplay between the tumour and its microenvironment will be crucial for the development of next‐generation, personalised immunotherapeutic strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab5e861ba23ee597d8c992f1eb0a4c50564cbedf" target='_blank'>
              Reprogramming the tumour microenvironment: Emerging strategies to overcome immunotherapy resistance
              </a>
            </td>
          <td>
            Mohammed Elmujtba Adam Essa, Shahd Elbadri, Abdelkareem A. Ahmed, Hamid Noori
          </td>
          <td>2025-11-09</td>
          <td>Clinical and Translational Discovery</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) remains uniformly fatal because the standard temozolomide (TMZ) + radiotherapy regimen is quickly thwarted by epigenetic adaptations that reactivate proliferation and blunt DNA-damage responses; mounting evidence implicates simultaneous up-regulation of lysine-specific demethylase 1 (KDM1A/LSD1) and histone deacetylase 2 (HDAC2) as a central driver of this resistance, yet no brain-penetrant agent has been available to co-target both enzymes. Here we show that MPT0G521, a dual KDM1A/HDAC inhibitor able to cross the blood–brain barrier, reverses this adaptive program. Integrated analyses of TMZ- and radiation-resistant GBM lines, public bulk datasets and 66 000 single cells revealed concerted over-expression of KDM1A and HDAC2, tight co-regulation with 50 core cell-cycle genes, and enrichment of LSD1-high NPC/OPC-like populations at invasive margins; high expression of either gene predicted significantly worse overall survival in CGGA cohorts (log-rank p < 0.01). In both parental and TMZ-resistant A172 and patient-derived Pt#3 cells, MPT0G521 produced dose- and time-dependent growth inhibition, suppressed clonogenicity, induced G2/M arrest, and triggered caspase-3/PARP-mediated apoptosis. Mechanistically, treatment raised H3K4me1/2 and H3K9/K14ac without altering core histone levels, confirming effective blockade of KDM1A demethylase and HDAC deacetylase activities. These findings identify KDM1A/HDAC2 over-expression as a tractable epigenetic vulnerability in therapy-refractory GBM and position MPT0G521 as a promising lead compound for overcoming TMZ resistance and improving clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a64cf703dedf5e47c6085e0296a117bfd8d31b01" target='_blank'>
              DDDR-13. A Brain-Penetrant Epigenetic Strategy to Overcome Therapy-Refractory Glioblastoma
              </a>
            </td>
          <td>
            W. Lo, An-Chih Wu, Pin-Yuan Chen, J. Chuang
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background Breast cancer remains a major global health challenge because of limitations in early detection and therapeutic outcomes. This study employed bulk and single-cell RNA sequencing to investigate SUMOylation-associated molecular networks, aiming to identify prognostic biomarkers and potential therapeutic applications. Methods Transcriptomic profiling was performed on 1,445 breast cancer and 113 normal samples to identify differentially expressed genes. Four hub genes, NR3C2, CDCA8, AURKA, and PLK1, were prioritized using machine learning. Consensus clustering stratified patients into molecular subtypes based on the hub gene expression patterns. Differential immune infiltration analysis was used to evaluate 28 immune cell populations between the subtypes. Hub gene-immune cell interactions were visualized using bubble diagrams. Pharmacogenomic sensitivity profiling was performed using subtype-specific drug response data. Single-cell sequencing identified epithelial subclusters enriched for hub genes, and transcription factor networks were analyzed using SCENIC. Pan-cancer validation was performed to assess the oncogenic role of hub genes in 21 malignancies. Statistical significance was determined using the Student’s t-test (p < 0.0001). Results Tumor tissues exhibited significant upregulation of CDCA8, AURKA, and PLK1, whereas NR3C2 was notably downregulated (p < 0.0001). Consensus clustering identified two distinct molecular subtypes: Subtype1, characterized by NR3C2 upregulation and poorer prognosis, and Subtype2, distinguished by enhanced expression of CDCA8, AURKA, and PLK1, correlating with favorable outcomes. Notably, PIK3CA mutations were prevalent in Subtype1, whereas TP53 mutations dominated Subtype2. Immune infiltration profiles differed significantly between the two subtypes for most immune cell types. Pharmacogenomic assessments revealed distinct drug sensitivity profiles for each subtype in response to various therapeutic agents. A pan-cancer analysis of the four hub genes demonstrated consistent expression patterns, immune correlations, and prognostic associations across malignancies. Conclusion Our findings reveal that SUMOylation subtypes in breast cancer exhibit distinct prognostic, immunological and pharmacogenomic profiles. These insights may provide candidate biomarkers for future personalized treatment strategies for breast cancer and potentially for other malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b694520cfc56236ad597a9d4d5eb2e2e73faafb" target='_blank'>
              Significance of SUMOylation in breast cancer progression: a comprehensive investigation using single-cell analysis and bioinformatics
              </a>
            </td>
          <td>
            Wenxing He, Zhengkui Sun, Dongmei Li, Tenghua Yu
          </td>
          <td>2025-11-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Phenotypic plasticity is a recognized mechanism of therapeutic resistance in prostate cancer (PCa), however current knowledge of driver mechanisms and therapeutic interventions are limited. Using genetically engineered mouse models (GEMMs) devoid of Pten and Rb1, we previously demonstrated the chromatin reprogramming factor enhancer of zeste homolog 2 (EZH2) as an important regulator of alternative transcription programs promoting phenotypic plasticity. Here, using a multi-omics approach we find that EZH2 regulates a multilineage cell state dependent on the activation of translation and regulation of RNA maturation. Combined chemical inhibition of EZH2 and PI3K/mTORC1 resulted in superior anti-tumor activity in murine and human phenotypic plastic models and was most significant when this combination was used with castration. Moreover, we elucidated that the regulation of cellular state transition by EZH2 inhibition requires activation of the RNA degrader, Tristetraprolin (TTP). Together, these data indicate phenotypic plasticity dependence on coordination between EZH2, TTP and mTORC1 signaling that represent novel therapeutic dependencies for this lethal PCa phenotype.



 Beatriz German, Leigh Ellis. EZH2 regulates cell identity in neuroendocrine-like prostate cancer by driving alternate transcription via activation of translation and RNA processing regulation [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C033.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e13a2ff65980383cdcce076c99675421b3f51723" target='_blank'>
              Abstract C033: EZH2 regulates cell identity in neuroendocrine-like prostate cancer by driving alternate transcription via activation of translation and RNA processing regulation
              </a>
            </td>
          <td>
            Beatriz German, Leigh Ellis
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Genetic and epigenetic processes are intertwined in a complex manner in order to enhance the development of breast cancer by regulating the fundamental cellular processes including DNA repair, proliferation, differentiation and metastasis. This review provides an overview of the findings of animal model experiments to explain the causes of the significant changes in genetic aspect including RAD51C mutations and dys-regulation of the BAF chromatin remodeling complex and epigenetic alterations, such as DNA methylation, histone modification, and microRNA dys-regulation. The animal models are where the researcher can study the mechanism by which these changes of the molecules cause tumorigenesis, stimulate tumor growth and metastatic potential. The data received in the course of such studies denotes the dynamism and reversibility of the epigenetic processes which may be utilized in treating diseases by selectively regulating the expression of the said molecules with an aim of modifying DNA methylation, histone-modifying enzymes, and miRNAs. Integration of genetic and epigenetic understanding will not only enhance our understanding of breast cancer biology, but also design the precision therapies, and also expose potential biomarkers that can be utilized to identify and diagnose the disease at the earliest phase. In total, the significance of the animal-based research in the context of finding mechanistic information and design interventions to limit the occurrence of breast cancer is highlighted in this review as a foundation of the subsequent use of the translation research and clinical interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/615bf75efa81e4b14beb7b6d9c561e55199ae645" target='_blank'>
              Genetic and Epigenetic Factors in Breast Cancer Progression
              </a>
            </td>
          <td>
            Akash Anil Podutwar, Purva Chandorkar, Swati Changdeo Jagdale
          </td>
          <td>2025-10-18</td>
          <td>Journal of Pharmacology, Genetics and Molecular Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Recent breakthroughs in genome technologies such Paired-Seq and Pair-Tag can jointly profile transcriptome and chromatin accessibility or histone modifications in single cells, and these methods have been used to study cell-type specific regulatory networks in normal human brains. However this approach has not yet been applied to any complex disease or primary tumor. Here, for the first time, we performed single-nucleus paired-tag characterization of freshly resected tissues from three recurrent glioblastoma (GBM) patients, including tumor core and adjacent brain specimens, to understand the epigenetic mechanisms that facilitate GBM cell aggressiveness and evolution. Clustering of over 25,000 cells across modalities identified distinctive normal and neoplastic populations, the latter of which exhibited clonal copy number amplifications in established oncogenes such as EGFR, MDM2, and PDGFRA. Cell states were sharply defined by unique enhancer profiles, while repressive regions existed in a broader continuum with overlap of glial, neuronal, and tumor populations. Detection of cell-type specific H3K27ac and H3K27me3 peaks revealed lineage restricted cis regulatory elements (CRE) that strongly correlated with gene expression patterns, including potent enhancers and silencers associated with cell cycle regulation. Co-accessibility analyses of CREs with nearby promotors further elucidated transcriptional mediators, such as tumor-specific enhancers of the glioma invasion gene HMGA2. Stratification of GBM cells into previously established transcriptional states (MES, NPC, OPC, AC) identified key epigenetic facilitators associated with subgroup-specific expression patterns, as well as downstream evolutionary trajectories that drive clonal selection and therapy resistance. Our results leverage granular co-profiling of histone modifications, gene expression, and chromatin organization to derive novel insights into recurrent GBM biology, identifying new therapeutic vulnerabilities in these molecularly divergent lesions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b188a0338485b43a319b25ce06bce2564a32bf6a" target='_blank'>
              EPCO-25. SINGLE-NUCLEUS CO-PROFILING OF TRANSCRIPTOME, HISTONE MODIFICATIONS, AND CHROMATIN ACCESSIBILITY REVEALS DYNAMIC EVOLUTION OF RECURRENT GLIOBLASTOMA
              </a>
            </td>
          <td>
            Mark W. Youngblood, Shaojun Yu, Yoshii Ma, K. Habashy, A. Gould, Li Chen, Ye Hou, Yu Luan, Yihao Fu, A. Sonabend, Feng Yue
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Breast cancer stem cells (BCSCs) represent a small but critical subpopulation within breast tumors that drive tumor initiation, progression, metastasis, therapeutic resistance, and recurrence. BCSCs are characterized by cell surface markers such as CD44⁺/CD24⁻ and high aldehyde dehydrogenase 1 (ALDH1) activity, and are regulated by pluripotency-associated transcription factors including POU class 5 homeobox 1 (OCT4), SRY-Box transcription factor 2 (SOX2), and Nanog homeobox (NANOG). They exhibit remarkable plasticity and heterogeneity across different breast cancer subtypes, which is further enhanced by epithelial-mesenchymal transition and dynamic interactions with the tumor microenvironment, enabling immune evasion and adaptation to hostile conditions such as hypoxia. Functionally, BCSCs sustain tumorigenesis through self-renewal, contribute to metastatic dissemination as circulating tumor cells, and can enter dormancy in distant niches, leading to late recurrences. Their intrinsic resistance to chemotherapy, radiotherapy, and targeted therapies arises from quiescence, enhanced DNA damage repair, overexpression of drug efflux transporters, and metabolic reprogramming, posing major clinical challenges. Recent advances have improved BCSC identification and isolation through marker-based cell sorting, dye exclusion assays, mammosphere culture, and functional reporter systems. Conventional therapies often spare or enrich BCSCs, highlighting the need for novel treatments. Targeted strategies focusing on stemness-related signaling pathways such as Wnt, Notch, and Hedgehog, as well as regulators including NANOG, ALDH1 and programmed death-ligand 1 (PD-L1), are under investigation. Emerging approaches incorporating natural compounds, nanotechnology, and microenvironmental modulation show promise in overcoming therapeutic resistance. This review integrates current understanding of BCSC biology, heterogeneity, and microenvironmental interactions, providing valuable insights for breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c0824bd9fb688f1edb70aeb158a043b8788637c" target='_blank'>
              Breast cancer stem cells: Biological roles, microenvironmental interactions, and therapeutic targeting
              </a>
            </td>
          <td>
            Mou Yan
          </td>
          <td>2025-10-11</td>
          <td>New Cell</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Transposable elements (TEs), once regarded as genomic “junk,” are now recognized as powerful regulators of gene expression, genome stability, and innate immunity. In the context of neurodegeneration, particularly Amyotrophic Lateral Sclerosis (ALS), accumulating evidence implicates TEs as active contributors to disease pathogenesis. ALS is a fatal motor neuron disease with both sporadic and familial forms, linked to genetic, epigenetic, and environmental factors. While coding mutations explain a subset of cases, advances in long-read sequencing and epigenomic profiling have unveiled the profound influence of non-coding regions—especially retrotransposons such as LINE-1, Alu, and SVA—on ALS onset and progression. TEs may act through multiple mechanisms: generating somatic mutations, disrupting chromatin architecture, modulating transcriptional networks, and triggering sterile inflammation via innate immune pathways like cGAS-STING. Their activity is normally repressed by epigenetic regulators, including DNA methylation, histone modifications, and RNA interference pathways; however, these controls are compromised in ALS. Taken together, these insights underscore the translational potential of targeting transposable elements in ALS, both as a source of novel biomarkers for patient stratification and disease monitoring, and as therapeutic targets whose modulation may slow neurodegeneration and inflammation. This review synthesizes the current knowledge of TE biology in ALS; integrates findings across molecular, cellular, and systems levels; and explores the therapeutic potential of targeting TEs as modulators of neurodegeneration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69ff26f0b7276e91620b33b746e298b203260717" target='_blank'>
              The Other Side of the Same Coin: Beyond the Coding Region in Amyotrophic Lateral Sclerosis
              </a>
            </td>
          <td>
            Paola Ruffo, Benedetta Perrone, Francesco Perrone, F. De Amicis, Rodolfo Iuliano, Cecilia Bucci, Angela Messina, F. Conforti
          </td>
          <td>2025-10-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Disruptor of Telomeric Silencing 1-Like (DOT1L) has emerged as a critical epigenetic regulator in cancer, primarily because of its role as the sole histone methyltransferase responsible for histone H3 at lysine 79 methylation. This modification affects transcriptional activation, DNA repair, and cell cycle progression. Its dysregulation is associated with both hematological and solid tumors. In Mixed-lineage leukemia (MLL)-rearranged leukemia, DOT1L maintains aberrant gene expression patterns at loci such as HOXA and MEIS1, supporting leukemic stem cell survival and driving oncogenesis. In solid tumors, DOT1L influences diverse processes, including epithelial-mesenchymal transition, angiogenesis, and cell cycle regulation, contributing to tumor growth and metastasis. Therapeutic strategies targeting DOT1L using inhibitors, such as EPZ-5676, have shown promise in preclinical and clinical studies, highlighting their potential as versatile targets for precision oncology. This review summarizes the recent findings on DOT1L’s involvement in cancer development and its potential as a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/652c9fcc6ab6b96caf0c41342c1311e10db0c889" target='_blank'>
              DOT1L as a Therapeutic Target: Insights into Epigenetic Regulation and Cancer Treatment
              </a>
            </td>
          <td>
            Hee-Joo Choi, Minh Tuan Nguyen, Boram Kim, Seung Hwan Kim, Jaehee Kim, Jeong‐Yeon Lee, Mikyung Park
          </td>
          <td>2025-10-31</td>
          <td>Biomolecules & Therapeutics</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="


 Deletion of the long arm of chromosome 9, del(9q), is a recurrent cytogenetic abnormality in AML and encompasses the HNRNPK locus at 9q21.32. hnRNP K is a multifunctional RNA- and DNA-binding protein involved in chromatin remodeling, transcriptional regulation, and RNA metabolism. Monoallelic deletions or loss-of-function mutations in HNRNPK, collectively observed in ~5–6% of AML cases, directly contribute to leukemogenesis.
 We previously showed that Hnrnpk haploinsufficiency disrupts p53 and myeloid lineage-specifying pathways (e.g., C/EBPα and PU.1), impairing differentiation and expanding myeloid progenitors—recapitulating del(9q) AML features. While elucidating these pathways has aided our understanding of AML biology, they remain largely undruggable. Therefore, identifying alternative, targetable vulnerabilities in HNRNPK-deficient AML is of significant clinical interest. To this end, our current studies reveal that HNRNPK-deficient cells exhibit marked sensitivity to translational inhibition, indicating a synthetic lethal interaction driven by impaired protein synthesis.
 To delineate the molecular basis of this vulnerability, we used RIP-seq, proteomics (RPPA, IP-MS), and translational profiling. Here, we identified that HNRNPK deficiency caused dysregulated mTOR signaling, aberrant ribosomal protein S6 phosphorylation, and altered expression of key translation initiation/elongation factors. Critically, rRNA processing was impaired, disrupting 47S precursor maturation and leading to defective 40S and 60S subunit assembly and reduced active polysomes—suggesting that del(9q) AML may represent a novel ribosomopathy.
 Importantly, complete HNRNPK loss is embryonically lethal, and large-scale CRISPR datasets identify it as essential, underscoring its dosage sensitivity and potential for synthetic lethal therapeutic strategies. To uncover synthetic lethal interactions, we performed a genome-wide CRISPR-Cas9 dropout screen in HNRNPK-deficient cells, identifying dependencies in ribosome biogenesis, translation initiation, and metabolite synthesis as potential vulnerabilities. We are currently using primary cells harboring HNRNPK deletions and genetically engineered models with reduced hnRNP K expression to identify therapies that induce cell death in an hnRNP K-dependent manner.
 Collectively, our findings establish hnRNP K as a dosage-sensitive tumor suppressor and suggest del(9q) AML as a novel ribosomopathy. Moreover, the synthetic vulnerabilities identified in our study provide a strong rationale for developing targeted therapies that exploit HNRNPK haploinsufficiency and lay the foundation for the clinical development of agents that induce synthetic lethality in this subset of patients with AML.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a1f4b3ab63f8b9803758eab2816344752deed3f" target='_blank'>
              Hnrnpk haploinsufficiency in del(9q) Acute Myeloid Leukemia reveals targetable vulnerabilities
              </a>
            </td>
          <td>
            S. Post, Oscar Benitez, Ashley G Rodriguez, Marisa Aitken, P. Malaney, Miguel Gallardo, Hussein A. Abbas
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>26</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025', '2026'],
    y: [9, 0],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>