<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2026-01-26 09:43:57 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="DHFR gene amplification is commonly present in methotrexate (MTX)-resistant colon cancer cells and acute lymphoblastic leukemia. In this study, we proposed an integrative framework to characterize the amplified region by using a combination of single-molecule real-time sequencing, next-generation optical mapping, and chromosome conformation capture (Hi-C). We identified an amplification unit spanning 11 genes, from the DHFR gene to the ATP6AP1L gene position, with high adjusted interaction frequencies on chromosome 5 (~2.2 Mbp) and a twenty-fold tandemly amplified region, and novel inversions at the start and end positions of the amplified region as well as frameshift insertions in most of the MSH and MLH genes were detected. These mutations might stimulate chromosomal breakage and cause the dysregulation of mismatch repair. Characterizing the tandem gene-amplified unit may be critical for identifying the mechanisms that trigger genomic rearrangements. These findings may provide new insight into the mechanisms underlying the amplification process and the evolution of drug resistance. Sequencing a large region of DNA containing many surplus copies of genes linked to drug resistance in colon cancer cells may illuminate how these genomic rearrangements arise. Such regions of gene amplification are highly repetitive, making them impossible to sequence using ordinary methods, and little is known about how they are generated. Using advanced methods, Jeong-Sun Seo at Seoul National University Bundang Hospital in South Korea and co-workers sequenced a region of gene amplification in colon cancer cells. The amplified region was approximately 20 times the length of that in healthy cells and contained many copies of an eleven-gene segment, including a gene implicated in drug resistance. The region also contained mutations in chromosomal repair genes which would disrupt repair pathways. These results illuminate the genetic changes that lead to gene amplification and drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental and Molecular Medicine, Experimental & Molecular Medicine</td>
          <td>5</td>
          <td>50</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum
                </a>
              </td>
          <td>
            Gustavo S. França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>87</td>
          <td>46</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Cancer cell adaptation to their physical tumor microenvironment is a key driver of malignancy. Recent experimental evolution experiments show that the soft extracellular matrix (ECM) can impose a selection pressure on genetically variable tumor populations. Over months of sustained culture, the selection pressure leads to enrichment of specific genetic variants with high fitness, but the mechanisms underlying the high fitness of these soft-selected clones are not fully understood. Here, we used a combination of RNA-seq, ATAC-seq, and RRBS-seq to compare soft-selected populations with non-selected ancestral populations cultured on soft ECM. We demonstrate that ancestral populations grown on soft ECM for short durations are characterized by a stressed cell state with low fitness marked by cell cycle arrest and distinct metabolic shifts, whereas sustained culture selects for a robust proliferative phenotype. Mechanistically, selected cells exhibit a silenced ancestral stress response through epigenetic modifications, characterized by reduced chromatin accessibility and de novo DNA methylation, including CDH1 promoter hypermethylation. This repressive landscape supports a high-fitness state defined by elevated MYBL2 and FAK levels. An in-silico mechanism-based model shows that these molecular differences, together with high YAP1 nuclear localization in soft-selected cells, are salient features of genetic clones capable of FAK upregulation. These findings uncover a coordinated genetic and epigenetic mechanism driving cancer cell evolution in mechanically soft microenvironments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4db71dfbf014de56876df2b0949f05913ac70b1" target='_blank'>
              Multi-omics investigation reveals molecular determinants of cancer cell evolution on soft extracellular matrix
              </a>
            </td>
          <td>
            Sarthak Sahoo, S. Khanna, Ting-Ching Wang, Tanmay P. Lele, M. Jolly
          </td>
          <td>2025-12-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00cba125444257d124df7ba1ec49280e8dc284a3" target='_blank'>
              Single-cell transcriptomics reveals a conserved embryonic progenitor in different human cancers
              </a>
            </td>
          <td>
            A. Khozyainova, Vera G. Subrakova, M. Menyailo, D. Zhigalina, T. N. Kireeva, A. Galiakberova, Mikhail A. Berestovoy, Elena E. Kopantseva, A. Korobeynikova, E. Dashinimaev, Dmitry M. Loos, M. S. Tretyakova, U. A. Bokova, N. Skryabin, E. Denisov
          </td>
          <td>2025-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d73e2496f48eaf63e522263faed717c4d2cc83e" target='_blank'>
              Exploring the Role of APC in Modulating Chemotherapeutic Response in Triple-Negative Breast Cancer cells
              </a>
            </td>
          <td>
            T. Nair, M. K. Vanklompenberg, Jenifer R Prosperi
          </td>
          <td>2026-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee85598703608ccef540311d86a0a9e7d9f4dae2" target='_blank'>
              Genetic barcoding of individual cells links cancer evolutionary trajectories and prognostic outcomes
              </a>
            </td>
          <td>
            Candice Merle, I. Di Terlizzi, M. Huyghe, Danae Welboren, Wenjie Sun, Cécile Conrad, Leïla Perié, S. Rulands, Silvia Fre
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Oncogenic KRAS mutations drive metabolic reprogramming in pancreatic ductal adenocarcinoma (PDAC). Src-homology 2 domain-containing phosphatase 2 (SHP2) is essential for full KRAS activity, and promising dual SHP2/mitogen-activated protein kinase (MAPK) inhibition is currently being tested in clinical trials. Exploitable metabolic adaptations may contribute to invariably evolving resistance. To understand the metabolic changes induced by dual inhibition, we comprehensively tested human and murine PDAC cell lines, endogenous tumor models, and patient-derived organoids, which are representative of the full spectrum of PDAC molecular subtypes. We found that dual SHP2/mitogen-activated protein kinase kinase (MEK1/2) inhibition induces major alterations in mitochondrial mass and function, impacts reactive oxygen species (ROS) homeostasis and triggers lipid peroxidase dependency. Anabolic pathways, autophagy and glycolysis were also profoundly altered. However, most strikingly, mitochondrial remodeling was evident, persisting into a therapy-resistant state. The resulting vulnerability to the induction of ferroptotic cell death via the combination of vertical SHP2/MEK1/2 with glutathione peroxidase (GPX4) inhibition was largely independent of the PDAC molecular subtype and was confirmed with direct targeting of RAS. The triple combination of SHP2/MEK1/2 inhibition and the ferroptosis-inducing natural compound withaferin A suppressed tumor progression in an endogenous PDAC tumor model in vivo. Our study offers a metabolic leverage point to reinforce RAS pathway interference for targeted PDAC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ece64da9d6dcfbff96e0c83595844cf2b2f5547" target='_blank'>
              Vertical RAS pathway inhibition in pancreatic cancer drives therapeutically exploitable mitochondrial alterations
              </a>
            </td>
          <td>
            P. Hafner, S. Keller, Xun Chen, Asma Alrawashdeh, H. Jumaa, Friederike I. Nollmann, S. Besson, Judith Kemming, Oliver Gorka, T. Das, Bismark Appiah, Ariane Lehmann, M. Li, Petya Apostolova, Bertram Bengsch, R. Zeiser, S. Tholen, Oliver Schilling, Olaf Groß, Andreas Vlachos, U. Wittel, Dominik von Elverfeldt, W. Reichardt, M. Boerries, G. Andrieux, Guus Heynen, S. Fichtner-Feigl, Luciana Hannibal, D. Ruess
          </td>
          <td>2026-01-16</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e46bc8969d218f90b6869d6ad86ad7ec80c28324" target='_blank'>
              Loss of ESE3/EHF is sufficient to promote cell plasticity, transformation and androgen-independent status in the early stage of prostate carcinogenesis
              </a>
            </td>
          <td>
            D. Albino, G. Sandrini, C. Musumeci, Elisa Storelli, Giovanni Papa, Daniela Impellizzieri, A. Cacciatore, G. Cassanmagnago, Atik Balla, Simone Mosole, G. Civenni, Andrea Rinaldi, Carmen Jerónimo, Rui M. Henrique, Marco Bolis, C. Catapano, G. M. Carbone
          </td>
          <td>2025-11-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddfc626bda7c6db9e371ab841d5c95cbd7dff747" target='_blank'>
              RNAseq analysis reveals the recurrent loss of heterozygosity in lung cancer and associated transcription patterns
              </a>
            </td>
          <td>
            Ruslan Gumerov, Wangzhen He, Phong Luong, Filippo Dall’Olio, Y. Vassetzky, Anna Schwager
          </td>
          <td>2026-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e404655f34f2f760a22785dfe7969af3c340a1ca" target='_blank'>
              Genome-wide CRISPR screens identified C18orf32 as a novel regulator of lipid metabolism that mediates PFOA-induced toxicity
              </a>
            </td>
          <td>
            Chanhee Kim, A. Tagmount, Zhaohan Zhu, W. B. Barbazuk, Rhonda L. Bacher, Christopher D. Vulpe
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Mutations in tumour suppressor p53 are frequently implied in aggressive progression and metastasis in colorectal cancer. But the distinct phenotypes exhibited by site-specific mutations of p53 are not well elucidated. Here, we investigate the epigenetic and transcriptional impact of three major p53 hotspot mutations (R175H, R273H and R282W), through DNA methylation changes and single cell transcriptomics. We observed that the p53 R273H mutation is associated with a partial epithelial–mesenchymal transition (pEMT) and increased metastatic progression. Analysis of DNA methylation patterns revealed a distinct epigenetic landscape in R273H-mutant tumours, with hypomethylated regions correlating with enhanced transcriptional activation of YAP/TAZ target genes thus promoting pEMT and EMT-like phenotype in CRC tumours. In vitro ChIP-seq experiments in colorectal cancer cells expressing the R273H mutant p53 (HT29) showed enrichment of mutant p53 at the promoters of YAP/TAZ target genes suggesting EMT/pEMT like states with R273H mutation. Further, simulations from a gene regulatory network incorporating the interactions of p53R273H with EMT regulators explain how this mutation shapes the phenotypic landscape accessible to cancer cells. Our analysis of single-cell transcriptomes of colorectal tumours reveals R273H-linked enrichment of partial and mesenchymal EMT phenotypes across tumour subpopulations in CRC. We identified a distinct epigenetic signature associated with the p53 R273H mutation, characterised by hypomethylation of YAP/TAZ signalling genes that drives partial EMT and aggressive tumour behaviour. These findings highlight the importance of mutation-specific epigenetic regulation in shaping colorectal cancer progression and the need for developing therapeutic strategies tailored to p53 mutation status.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89ad1c85957c8764e20b8788eabb8e8a09e388db" target='_blank'>
              Gain of function p53 mutant R273H confers distinct methylation profiles and consequent YAP/TAZ signaling mediated activation of partial or full EMT states to colon tumours
              </a>
            </td>
          <td>
            H. Rani, Seemadri Subhadarshini, M. Jolly, V. Mahadevan
          </td>
          <td>2025-12-13</td>
          <td>Epigenetics & Chromatin</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Immune dysfunction in cancer is enacted by multiple programs, including tumor cell-intrinsic responses to distinct immune subpopulations. A subset of these immune evasion programs can be systematically recapitulated through direct tumor-immune interactions in vitro. Here, we present an integrated, high-throughput single-cell CRISPR screening framework focused on the protein kinome for mapping the tumor-intrinsic regulation of T cell-driven immune pressure in glioblastoma (GBM). We combine pooled CRISPR interference and activation (CRISPRi/a) with immune-matched NY-ESO-1 antigen-specific allogeneic GBM-T cell co-culture and massively multiplexed single-cell transcriptomics to systematically quantify how genetic perturbation reshapes baseline tumor state and adaptive responses across graded effector-to-target ratios. We further leverage deep generative models for analyzing pooled CRISPR screens to decipher the effects of genetic perturbations on the mechanisms of tumor resistance. This framework resolves distinct modules of immune evasion and survival, including the regulation of the antigen-presentation machinery, interferon/NF-κB signaling, oxidative stress resilience, and checkpoint/cytokine programs, while identifying perturbations that reroute the continuous tumor transcriptional trajectory induced by T cell engagement. A secondary chemical screen in patient-derived GBM cultures identified putative kinase targets of immune evasion phenotypes (e.g., EPHA2 and PDGFRA), whose inhibition leads to the blockade of evasive programs and enhances T cell-mediated GBM killing. Together, this workflow provides a scalable blueprint for comprehensive charting of the genetic control of tumor-immune interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e76a058bed6b1c1e4d91cd692a05e66fab6beee" target='_blank'>
              Mapping kinase-dependent tumor immune adaptation with multiplexed single-cell CRISPR screens
              </a>
            </td>
          <td>
            Lingting Shi, Ross M. Giglio, Qingyuan Cai, Mathini Vaikunthan, Justin Hong, Abdullah Naqvi, Marina Milea, Hannah Khanshali, Anna Schoonen, Nicholas Hou, Jonathan Guo, Melanie Fraidenburg, Xumin Shen, Seth Malinowski, K. L. Ligon, Raúl Rabadán, Elham Azizi, José L. McFaline-Figueroa
          </td>
          <td>2026-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Abstract Genetic instability is a hallmark of cancer, often arising from mutations to DNA damage repair and response (DDR) genes. Classical genetic, biochemical, and structural approaches elucidated the foundational mechanisms of DDR pathways and provided a scientific understanding of their involvement in repair of lesions induced by broad classes of DNA-damaging agents (DDAs). However, given the chemical diversity of DDAs and resultant DNA lesions, along with the multitude of interconnected DDR factors, the chemogenomic landscape of DDA–DDR interactions remains incompletely mapped. To this end, we developed a DDR-targeted, CRISPR knockout screening approach and assessed relationships amongst 353 DNA repair genes and 15 DDAs in LN229 glioma cells. Within this dataset of 5295 DDR-related chemogenomic interactions, we identified many established interactions and discovered novel ones. For example, we observed a specific role of transcription-coupled nucleotide excision repair in the repair of adducts generated by monofunctional alkylating agents, a role for the Fanconi anemia pathway in addressing methyl lesions, overt differences in DSB repair following treatment with topoisomerase I versus II poisons, and repair dependencies associated with the imidazotetrazines temozolomide, mitozolomide, and KL-50. Future directions will continue to investigate the mechanisms of novel chemogenomic interactions that we have uncovered as well as work to identify chemogenomic interactions amenable to clinical translation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/018770119a2496ee321bc85a8d0504634bbf54c9" target='_blank'>
              Targeted CRISPR knockout screening identifies known and novel chemogenomic interactions between DNA damaging agents and DNA repair genes
              </a>
            </td>
          <td>
            Collin D. Heer, James L. Elia, Vijay Menon, Spenser S. Johnson, Sofia R. Arbelaez, Sam Friedman, Francesc Lopez-Giraldez, R. Sundaram, Seth B Herzon, R. Bindra, Susan E. Gueble
          </td>
          <td>2026-01-09</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Loss-of-function mutations in the TP53 gene are the most frequent genetic events in human malignancies. However, the extent to which missense TP53 mutations confer oncogenic gain-of-function (GOF) properties remains incompletely understood. Using genetically defined pancreatic cancer organoid models, we demonstrated that mutant p53 provides a selective growth advantage under acute environmental stress and drives molecular transition toward a basal-like state. Transcriptomic analyses showed that under stress conditions, organoids expressing mutant p53 adopted a basal-like gene expression program, a characteristic of a poor prognosis subtype of pancreatic cancer. Furthermore, the molecular subtype was reversed to the classical subtype once it returned to normal growth conditions, suggesting that transcriptional plasticity arises from acute changes in the tissue microenvironment. Given these context-dependent oncogenic properties, our findings suggest that targeting mutant p53 may be a promising therapeutic strategy for pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f58263187bbd03e5cadcf959b6bde70a5832e40c" target='_blank'>
              Mutant p53 regulates reversible subtype plasticity in pancreatic cancer
              </a>
            </td>
          <td>
            June-Ha Shin, Hwa-Ryeon Kim, Jae-Seok Roe
          </td>
          <td>2025-12-01</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Abstract Acquired resistance presents a major challenge for targeted therapies, with initially responsive tumors eventually reverting underlying vulnerabilities. Our group recently reported that cancers harboring isocitrate dehydrogenase 1/2 (IDH1/2) mutations have defective recruitment of homology-directed repair (HDR) factors to sites of DNA damage and consequent sensitivity to poly(ADP-ribose) polymerase inhibitors (PARPi), a vulnerability that is being tested in clinical trials. To probe potential mechanisms by which resistance to PARPi might arise in this setting, we modeled PARPi resistance in IDH-mutant tumors via serial transplantation of patient-derived xenografts in mice treated with PARPi. An analysis of candidate DNA repair factors in these resistant tumor populations identified downregulation of two end protection factors that are negative regulators of HDR, 53BP1, and REV7. Knockout of these factors by CRISPR–Cas9 in IDH1-mutant cancer cells conferred robust resistance to PARPi and restored HDR capacity. To overcome this resistance, we found that treatment with the receptor tyrosine kinase inhibitor, cediranib, previously reported to suppress expression of downstream HDR factors, resensitizes 53BP1 and REV7-knockout cells to PARPi treatment. Our findings identify key pathways driving PARPi resistance in IDH1-mutant cancers and highlight potential therapeutic strategies to overcome this resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef230260548104420dc4aabb1c7dae9c604e7081" target='_blank'>
              PARP inhibitor resistance in IDH1-mutant cancers due to loss of end protection factors, 53BP1 and REV7
              </a>
            </td>
          <td>
            Daniel A. Colon-Rios, Jonathan Dow, Adam Krysztofiak, Yanfeng Liu, Faye A. Rogers, Peter M. Glazer
          </td>
          <td>2025-12-01</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d29acb3217c08dff3be7e7d47281f950232e5a4" target='_blank'>
              Identification of an epigenetically and phenotypically distinct peritumoral glioblastoma cell population linked to inferior patient outcome
              </a>
            </td>
          <td>
            I. Neves, Xi Lu, Veera Jokinen, Francesco Latini, N. Maturi, A. Sundström, Yonglong Dang, Irem Uppman, Tobias Bergström, Pengwei Xing, M. Ryttlefors, Xingqi Chen, F. Swartling, L. Uhrbom
          </td>
          <td>2026-01-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f09bfb27dbaed264c5656346c24d74a25e18488" target='_blank'>
              An Atlas of Extrachromosomal DNA Structures Illuminates Its Evolution and Biogenesis in Cancer
              </a>
            </td>
          <td>
            Chee Hong Wong, Kuo-Chen Wei, Kuan-Yu Kung, Li-Hao Yang, Yin-Cheng Huang, Chiung-Yin Huang, I-Jiuan Chung, Meng-Fan Huang, Pei-Hua Peng, Meihong Li, C. Ngan, Kou-Juey Wu, Chia-Lin Wei
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19e1efee6c0706220f7b18885b5b5a4638edf605" target='_blank'>
              A single-nucleus multimodal framework reveals epigenomic priming of chemoresistant states in ovarian cancer
              </a>
            </td>
          <td>
            Y. Landais, J. Bertorello, M. Puerto, Agathe Kahn, B. Simon, A. Roehrig, Adeline Durand, M. Quatredeniers, C. Lamy, E. Laas, Nicolas Pouget, Simon Dumas, J. Féron, Virginie Fourchotte, C. Bonneau, Vincent Cockenpot, Marianne Delville, Magali Richard, M. Kamal, F. Lécuru, C. L. Tourneau, C. Vallot
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d440f96f3272f5794b9ae0c0eebed83d3b36971" target='_blank'>
              Direct detection of CRISPR mutations and transcriptional responses at single cell resolution in vivo
              </a>
            </td>
          <td>
            John A. Hawkins, Siamak Redhai, Svenja Leible, Mireia Osuna Lopez, Hilal Ozgur, Tianyu Wang, Michaela Holzem, M. Boutros, Oliver Stegle
          </td>
          <td>2025-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68c56b991225823160872bb1206ac61c788f7b1c" target='_blank'>
              Scaling perturbations: beyond genome-scale CRISPR screens
              </a>
            </td>
          <td>
            Anran Tang, Rico C. Ardy, Rafaela E. Mendes, Thomas M. Norman
          </td>
          <td>2026-01-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The budding yeast Saccharomyces cerevisiae is a well-established model for studying the genetic basis of complex traits, and it is a powerful system for investigating mechanisms of aging. Here, we examine the genomic and transcriptomic factors contributing to increased replicative age in recombinant yeast populations harboring standing genetic variation. Using Fluorescence-Activated Cell Sorting (FACS), we isolated young and aged cohort pairs across twelve biological replicates and sequenced their progeny to assess patterns of differentiation at the nucleotide and transcription levels. Most differentiated alleles were located in coding regions, including significant variants within 132 unique genes. Transcriptomic analysis revealed 60 differentially expressed genes in aged populations, including 18 genes with increased expression in aged cohorts, and 42 genes with decreased expression. Although only two genes (RFA3 and WSC4) were implicated in both genomic and transcriptomic analyses, functional overlap associated with protein homeostasis, DNA repair, and cell cycle regulation was evident across datasets. Notably, we found no strong evidence that differentially expressed genes were more likely to occur in close proximity to significant gene variants. This suggests that late-life survival is not predominantly governed by local cis-regulatory interactions (e.g. variants within or near coding regions). These findings underscore the power of integrating genomic and transcriptomic data to elucidate the genetics of complex traits such as aging, demonstrating how multi-omics approaches can reveal functional relationships that may be overlooked by single-layer analyses. Significance Statement While many individual genes contributing to aging and lifespan have been identified, our understanding of the polygenic interactions and regulatory processes that contribute to phenotypic variation in these traits is much more limited. Using a recombinant population of yeast, we identify novel links between genetic variation and the phenotype of replicative age. Additionally, we find little evidence for local cis-regulatory interactions, suggesting that downstream regulation or trans-regulatory processes may serve more dominant roles in modulating aging. These results reveal new insights into the role of polygenicity in the evolution and regulation of age-associated phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5421275a5bb574745168cdf381917f8c4a541df8" target='_blank'>
              Age-specific genomic and transcriptomic variation reveals limited evidence for cis-regulatory interactions modulating aging in Saccharomyces cerevisiae
              </a>
            </td>
          <td>
            K. McHugh, Felipe S Barreto, M. Burke
          </td>
          <td>2025-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Berberine (BBR), a natural compound with diverse anticancer properties, exerts potent inhibitory effects on lung cancer cell proliferation. However, its particular molecular targets remain unknown. The present study aimed to identify the key genetic determinants that mediate the cellular response to BBR in lung cancer. To achieve this, genome-wide clustered regularly interspaced short palindromic repeats (CRISPR)/CRISPR-associated protein 9 (Cas9) knockout screening was conducted in A549 cells, followed by functional validation and mechanistic assays. In the study, the genome-wide CRISPR/Cas9 knockout screening in A549 lung cancer cells identified claudin-1 (CLDN1) as a critical modulator of BBR sensitivity. Notably, whilst CLDN1 was enriched in the positive selection screen, its knockout markedly increased the sensitivity of A549 cells to BBR, thus leading to enhanced G1-phase arrest and reduced proliferation. These findings suggest that CLDN1 could serve a dual role, promoting cellular resistance under selective pressure, while simultaneously demonstrating a therapeutic vulnerability when directly inhibited. Overall, the present study identified CLDN1 as a key mediator of the anticancer effects of BBR, thus providing a foundation for its potential development as a therapeutic target to optimize lung cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb4f6034fc6b67d610759292e57a0e8b62f65752" target='_blank'>
              Genome-wide CRISPR screening identifies CLDN1 as a central node in the anticancer mechanisms of berberine and as a therapeutic target in lung cancer
              </a>
            </td>
          <td>
            Xiaogang Wang, Benzhong Mi, Jinfeng Lv, Xin Tang, Jingyu Huang, Fei Liu, Yajing Xing, Xin-Yu Wen, Yongde Wang, Yadong Zhong
          </td>
          <td>2025-12-01</td>
          <td>Oncology Letters</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d983622827e375feb6c8bdb7968042a7608e5a98" target='_blank'>
              Genomic Flexibility Through Extrachromosomal Amplifications: A Leishmania Survival Strategy
              </a>
            </td>
          <td>
            Atia B. Amin, A. Ibarra-Meneses, Mathieu Blanchette, Christopher Fernández-Prada, David Langlais
          </td>
          <td>2025-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Understanding the molecular mechanisms underlying T-cell acute lymphoblastic leukemia (T-ALL) is essential for developing more effective therapeutic strategies. Despite therapeutic advances, the role of RNA-binding proteins in the pathogenesis of T-ALL remains poorly understood. Here, we investigate the RNA-binding protein Quaking (QKI), identifying it as a key regulator of splicing with tumor-suppressive properties in T-ALL. Through the analysis of two independent pediatric T-ALL cohorts, we demonstrate that QKI expression is frequently reduced in T-ALL, particularly within the HOXA subtype, and this reduction correlates with poor overall and event-free survival. Using T-ALL cell lines, we show that QKI depletion induces widespread splicing alterations, with numerous events corroborated in patient samples. Transcriptome profiling indicates that QKI downregulation leads to broad changes in gene expression, notably affecting pathways related to cell cycle progression, cholesterol homeostasis, and epithelial-mesenchymal transition. Functional assays demonstrate that QKI overexpression in T-ALL cells significantly reduces cell proliferation, induces G0/G1 cell cycle arrest, and limits leukemia progression and dissemination, ultimately improving survival in xenograft models. Together, these findings provide compelling evidence that QKI functions as a regulator of RNA splicing with tumor-suppressive activity in T-ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca5ec1e411749bba69b71a539ea37c30a49dc190" target='_blank'>
              QKI dysregulation induces extensive splicing changes in T-cell acute lymphoblastic leukemia.
              </a>
            </td>
          <td>
            Bruno Palhais, Nitesh D. Sharma, Igor Fijalkowski, T. Pieters, D. Deforce, F. Nieuwerburgh, Pieter Mestdagh, Panagiotis Ntziachristos, K. Matlawska-Wasowska, P. V. Vlierberghe
          </td>
          <td>2026-01-15</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ed06fb384ee497b6337be9253bfa34f2d9608de" target='_blank'>
              The sensitivity of acute myeloid leukemia to CDK8/19 inhibitors is determined by their metabolic profile
              </a>
            </td>
          <td>
            Varlamova Ekaterina Antonovna, Andreeva Daria Vladislavovna, Dalina Alexandra Alexandrovna, Misnik Elena Vladimirovna, Pavlenko Natalia Germanovna, Bruter Alexandra Vladimirovna, Tatarskiy Victor Vyacheslavovich
          </td>
          <td>2025-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The selection of cells that have acquired a desired gene edit is often done by the introduction of additional genes that confer drug resistance or encode fluorophores. However, such marker genes can have unintended physiological effects and are not compatible with editing of single nucleotides. Here, we present SNIPE, a method that allows the marker-free selection of edited cells based on single nucleotide differences to unedited cells. SNIPE drastically enriches for cells, which have been precisely edited (median 7-fold). We validate the approach for 42 different edits using Cas9 or Cas12a in different cell types and species. We use it to enrich for combinations of substitutions that change missense mutations carried by all people today back to the ancestral state seen in Neandertals and Denisovans. We also show that it can be used to kill cultured tumor cells with aberrant genotypes and to repair heterozygous tumorigenic mutations. Genome editing often requires marker genes for selection of edited cells. Here, the authors present SNIPE, a marker-free method that selects cells based on DNA sequence, enabling precise enrichment of edited cells and applications from evolutionary research to the elimination of cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d65e0d5b2b7147e135bb71766c56327e516ee0e4" target='_blank'>
              Search-and-remove genome editing allows selection of cells by DNA sequence
              </a>
            </td>
          <td>
            Luise Fast, Madina Omar, Philipp Kanis, Theresa Schaffer, Deepika Chowdhury, Eva Rakava, S. Pääbo, S. Riesenberg
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>153</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is the most aggressive breast cancer subtype, characterized by a lack of key hormone receptors in tumor cells. As there are limited treatment options for these patients, it is crucial to understand the underlying mechanisms by which TNBC constantly evolves and evades treatments. In this regard, the pervasive nature of transcription provides a potential reservoir of transcripts, including both coding and noncoding, that TNBC leverages to sustain a proliferative advantage and support tumor growth. TNBC is affected by energy sources such as glucose, which can have a profound impact on gene expression regulation mediated by various molecules, including noncoding RNAs, at the cellular level. In this study, we demonstrate that glucose modulates the gene expression profile mediated by the microRNA-503 host gene (MIR503HG), which has been previously implicated in TNBC. To comprehensively characterize the impact of glucose on MIR503HG-regulated genes and cellular pathways, we sequenced total RNA, performed gene set enrichment analyses, and determined the relation between gene expression and patient outcomes. Analysis of gene subsets specific to various glucose environments identified clinical outcomes for breast cancer patients across different molecular subtypes. Our findings indicate that MIR503HG has potential as a diagnostic marker and may be useful in the clinical management of TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6cc268d3d3ded7d83b55b98f7c7d7bf50879c780" target='_blank'>
              Molecular Effects of Glucose on MIR503HG-Regulated Genes in Triple-Negative Breast Cancer
              </a>
            </td>
          <td>
            Victoria A. Reid, Barbara Yang, Kyle Russo, Melina J. Sedano, Ramesh Choudhari, E. Ramos, Shrikanth S. Gadad
          </td>
          <td>2025-12-17</td>
          <td>British Journal of Biomedical Science</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Prostate cancer is a highly heterogeneous disease, driven by genomic and transcriptional changes that impact disease progression and metastatic potential. The interplay between clonal evolution, transcriptional plasticity, and tumour microenvironment is, however, poorly understood. Here, we leverage and integrate single-nuclei RNA sequencing and whole-genome sequencing from 43 spatially distinct tumour samples from five patients with locally advanced prostate cancer to reconstruct clonal evolution trajectories and transcriptional changes driving metastasis at single-cell resolution. We find extensive clonal heterogeneity, including both monophyletic and polyphyletic metastatic dissemination, and ongoing clonal evolution in the primary tumour after metastatic spread. Metastatic seeding converges on disease trajectories involving both genomic and transcriptional changes, including androgen receptor independence and activation of estrogen-, WNT- and JAK-STAT- pathway activity, in spatially distinct areas. Our findings suggest an intricate interplay between clonal evolution and cellular plasticity driving metastatic seeding and point toward more integrative prognostic markers for improved patient management. The impact of tumour heterogeneity on metastatic potential in prostate cancer remains poorly understood. Here, the analysis of single nuclei RNA sequencing and whole-genome sequencing from samples from five patients suggests an interplay between clonal evolution and cellular plasticity driving metastatic seeding.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ea384ac37d8c4c2acd2bd5fc9ab62cffd7ea81c" target='_blank'>
              Clonal evolution and transcriptional plasticity shape metastatic dissemination routes in prostate cancer
              </a>
            </td>
          <td>
            Migle Mikutenaite, Evdoxia Karadoulama, F. Favero, A. Locallo, Francisco German Rodriguez Gonzalez, Daria Kiriy, Tina Keshavarzian, K. Furlano, M. Graefen, Robert G. Bristow, Clarissa Gerhäuser, C. Plass, G. Sauter, R. Simon, M. Lupien, S. Minner, Thorsten Schlomm, Joachim Weischenfeldt
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>104</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f796949ca22238ba707f7eb2bf4bbb66b809b4f" target='_blank'>
              Single-cell whole-genome sequencing reveals convergent evolution in Burkitt lymphoma
              </a>
            </td>
          <td>
            A. Steemers, Markus J. van Roosmalen, Rico Hagelaar, L. Trabut, M. Verheul, Siem Jongsma, J. D. de Kanter, F. Meyer-Wentrup, R. van Boxtel
          </td>
          <td>2025-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a29acdf562d4b29c5b781b23b586561ef03a93de" target='_blank'>
              Pan-cancer analysis reveals genomic fidelity and evolution of patient-derived organoids
              </a>
            </td>
          <td>
            Linoy Raz, Tal Ben-Yishay, Haia Khoury, Ron Saad, Amjad Zoabi, Giulia Orlandi, S. Donzelli, Giovanni Blandino, U. Ben-David
          </td>
          <td>2025-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is a highly heterogeneous malignancy with a poor prognosis and limited response to current therapies. Cancer stem cells (CSCs) contribute to this heterogeneity, driving tumor progression, immune evasion, and therapeutic resistance. Epigenetic regulators have emerged as pivotal modulators of CSC phenotypes. The histone methyltransferase SETDB1, a key stemness factor in both embryonic and adult stem cells, catalyzes H3K9 trimethylation (H3K9me3) and acts as a major oncogenic driver in a wide range of solid and hematological malignancies. Yet, its specific role in maintaining stemness and contributing to therapeutic resistance in HCC remains poorly defined. Integrative analyses of bulk and single-cell transcriptomic datasets from public HCC cohorts were performed. Tumors were stratified by SETDB1 expression, followed by pathway enrichment, immune deconvolution, and stemness scoring. Associations with clinical outcomes, immune phenotypes, drug resistance signatures, and chromatin-binding partners (DNMT3A, TRIM28, HDAC1) were also explored. The role of SETDB1 was then examined in vitro, where a stem-like phenotype was induced in HCC cell lines under hypoxic conditions, and patterns of SETDB1 expression and stemness marker levels were assessed following paclitaxel treatment. SETDB1-low tumors were enriched in well-differentiated, immunologically active subclasses and were associated with increased predicted responsiveness to immune checkpoint blockade (ICB), sorafenib, and transarterial chemoembolization. Conversely, SETDB1-high tumors exhibited transcriptional features of dedifferentiation, elevated stemness scores, and enrichment in aggressive molecular subtypes. These tumors also displayed higher TP53 mutation rates, reduced immune infiltration, immune exclusion signatures, and activation of pathways linked to ferroptosis and ABC transporter dysfunction, consistent with resistance to ICB and conventional therapies. SETDB1 expression was also associated with hypoxia-related pathways, suggesting a role in maintaining CSC niches. In vitro, hypoxia-induced stemness coincided with increased SETDB1 levels, whereas paclitaxel treatment decreased both SETDB1 expression and stemness markers. A chromatin risk score incorporating SETDB1 and its partners predicted poor disease-free survival independently of clinical parameters. SETDB1 defines a stemness-enriched, immune-resistant HCC subtype associated with poor outcomes and therapeutic failure. Correlations between SETDB1 expression and stem-like features in vitro suggest a potential role in maintaining CSC phenotypes, supporting its relevance as a biomarker and candidate for epigenetic- and CSC-directed therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c393716a837e5b1ef0091b6385eb92c64be4ab4" target='_blank'>
              The epigenetic regulator SETDB1 as a key component of cancer stem cells and drug resistance in primary liver cancer
              </a>
            </td>
          <td>
            Maël Padelli, Christophe Desterke, Aurore Devocelle, Georges Uzan, Antoinette Lemoine, Julien Giron-Michel
          </td>
          <td>2026-01-06</td>
          <td>Cellular Oncology (Dordrecht, Netherlands)</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Identifying drivers of cancer progression to guide treatment selection is hindered by our limited understanding of tumor heterogeneity and its impact on tumor evolution. Here, we delineate the phenotypic variability across ~300,000 cells collected from multiple tumor loci in primary prostate and matched locoregional metastases using single-cell chromatin accessibility and gene expression sequencing. We find inter-patient heterogeneity to be confined to malignant populations. Within individual tumor loci, we see phenotypic heterogeneity among malignant cell populations despite a shared clonal genotypic architecture. We also observe that malignant cell populations disseminating to locoregional lymph nodes mirror the clonal architecture and phenotypic heterogeneity across primary tumor loci, while shifting from canonical prostate-cancer states to non-canonical inflammatory-like states. Our findings suggest a bottleneck imposed during the dissemination process, funneling prostate cancer cells toward an inflammatory-like cell state. These insights into the interplay between phenotypic identity and clonal architecture refine our understanding of prostate cancer progression and suggest that convergence of cancer cells towards an inflammatory-like state underlies dissemination to lymph nodes, offering a critical framework for future studies into prostate cancer metastatic potential. Understanding tumor heterogeneity and its impact on prostate cancer progression remains elusive. Here, single nucleus snATAC and snRNA sequencing of a multi-loci sampled cohort of advanced prostate cancer patients identifies an inflammatory-like state underlying metastatic dissemination to the lymph nodes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b9d84066de3071bee8ea7d75cdd27aba04e0e02" target='_blank'>
              Prostate cancer cells converge to an inflammatory-like state upon metastatic dissemination
              </a>
            </td>
          <td>
            Tina Keshavarzian, K. Furlano, Giacomo Grillo, Lisanne Mout, C. Arlidge, Faizan Hasan, Ankita Nand, Migle Mikutenaite, Evdoxia Karadoulama, Ashish Goyal, Elisabeth L Pezzuto, Jessica Heilmann, Jakub Sykora, S. Minner, Thorsten Schlomm, G. Sauter, R. Simon, H. He, Joachim Weischenfeldt, C. Plass, Clarissa Gerhäuser, M. Lupien
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Deletion of functional sequence is predicted to represent a fundamental mechanism of molecular evolution. Comparative genetic studies of primates have identified thousands of human-specific deletions (hDels), and the cis-regulatory potential of short (≤31 base pairs) hDels has been assessed using reporter assays. However, how structural variant-sized (≥50 base pairs) hDels influence molecular and cellular processes in their native genomic contexts remains unexplored. Here, we design genome-scale libraries of single-guide RNAs targeting 7.2 megabases of sequence in 6358 hDels and present a systematic CRISPR interference (CRISPRi) screening approach to identify hDels that modify cellular proliferation in chimpanzee pluripotent stem cells. By intersecting hDels with chromatin state features and performing single-cell CRISPRi (Perturb-seq) to identify their cis- and trans-regulatory target genes, we discovered 20 hDels controlling gene expression. We highlight two hDels, hDel_2247 and hDel_585, with tissue-specific activity in the brain. Our findings reveal a molecular and cellular role for sequences lost in the human lineage and establish a framework for functionally interrogating human-specific genetic variants. Deletion of functional sequence is predicted to represent a fundamental mechanism of molecular evolution. Here the authors use CRISPR-based perturbations and epigenetic profiling in chimpanzee cells to identify gene regulatory roles for genomic sequence lost in the human lineage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d81c8c2f5bb5b9ea0191569860cee91545d0b60" target='_blank'>
              Mapping cis- and trans-regulatory target genes of human-specific deletions
              </a>
            </td>
          <td>
            Tyler Fair, Bryan J. Pavlovic, Dani Swope, Octavio E. Castillo, Nathan K. Schaefer, Alex A. Pollen
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba1256bc5be1c15c5a52165e8dbea8c1e3c75014" target='_blank'>
              Integrative Single-cell and Spatial Transcriptomic Analysis of Osteosarcoma Reveals Conserved and Distinct Ecosystems Across Sites and Species
              </a>
            </td>
          <td>
            Yogesh Budhathoki, Matthew V. Cannon, T. A. McEachron, Anand G. Patel, Matthew J Gust, Jaime F Modiano, D.T. Ammons, Kathryn E Cronise, M. Macchietto, A. Sarver, Ruben Dries, B. Gryder, Daniel P. Regan, Heather L. Gardner, Ryan D. Roberts
          </td>
          <td>2026-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="CRISPR-Cas9 genome editing enables precise genetic modifications by introducing targeted DNA double-strand breaks (DSBs). While Cas9-induced DSBs are known to cause unintended on-target mutations, their impact on the epigenetic landscape remains unexplored. Here, we investigate how Cas9-induced DSBs affect DNA methylation patterns in human embryonic stem cells (hESCs). We induce DSBs at differentially methylated regions of imprinted genomic loci and perform high-coverage, long-read native DNA sequencing to simultaneously obtain genetic variant and base-resolution methylation data in a haplotype-resolved manner. Our findings reveal that DSBs cause significant changes in DNA methylation at target sites through mechanisms including homologous recombination, large structural variations, or defective methylation maintenance during DNA repair. Notably, these epigenetic changes can occur either together with or independently of genetic alterations. Beyond imprinted loci, Cas9-induced DSBs significantly disrupt DNA methylation patterns of the MLH1 epimutation alleles in colorectal cancer cells, and hypermethylated heterochromatin loci in hESCs. Clonal analysis indicates that the aberrant methylation changes are stable during in vitro passaging. Intriguingly, significant changes in DNA methylation levels are also detected around endogenous deletions in unedited genomic regions, suggesting that methylation alterations are not unique to Cas9 nuclease activity but represent a general outcome of DSB repair in human cells. This study underscores the importance of assessing and mitigating unintended epigenetic consequences in genome editing applications, as such changes can profoundly affect gene regulation and cellular function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f286fe9b90d8b6d662cbbd6ec5480ee68e1fcc17" target='_blank'>
              CRISPR-Cas9-induced double-strand breaks disrupt maintenance of epigenetic information
              </a>
            </td>
          <td>
            Mengge Wang, Yingzi Zhang, C. Bi, Mo Li
          </td>
          <td>2025-12-01</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Canine osteosarcoma (OS) is an aggressive bone cancer with remarkable similarity to human OS. Despite its value as a comparative oncology model, the drivers of genomic instability remain poorly defined, hindering cross-species insights into disease progression. We applied long-read PacBio HiFi whole genome sequencing to matched tumor and blood samples from four purebred dogs to generate high-resolution maps of somatic structural variants (SVs). Tumor genomes showed extensive rearrangement complexity, with satellite repeat regions, particularly SAT1_CF, enriched near somatic SV breakpoints and occurring within genomic contexts marked by focal hypomethylation. We also identified multiple extrachromosomal DNA (ecDNA) elements carrying oncogenes indicating ecDNA-mediated oncogene amplification. Together, these data provide an integrated genomic and epigenomic view of canine OS, implicating satellite repeats as recurrent substrates for rearrangement and ecDNA as a prominent mode of genome remodeling. This work advances our understanding of OS genome dynamics and establishes a comparative framework for investigating repeat-driven instability and oncogene amplification across species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5300af19b1916e9abe3db71ff1fa34ffceb5e24d" target='_blank'>
              Satellite DNA fragility accompanies complex genome rearrangements and ecDNA oncogene amplification in canine osteosarcomas
              </a>
            </td>
          <td>
            F. Yilmaz, Wonyoung Kong, Sabriya A. Syed, Francis H. O’Neill, Jody T. Lombardi, P. K. Ng, Ching C. Lau, Charles Lee
          </td>
          <td>2025-12-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Oncocytic (Hürthle cell) carcinoma of the thyroid (OCT) is characterized by widespread loss of heterozygosity (LOH), mitochondrial accumulation and recurrent mitochondrial DNA mutations leading to impairment of complex I. Here, we establish and characterize a novel OCT cell line, UT946, which displays severe mitochondrial electron transport chain dysfunction and a Warburg metabolic phenotype. Using a series of cytoplasmic hybrids, we establish that the complex I defect in UT946 stems from a nuclear-encoded loss of function mutation in the complex I subunit NDUFS1. To our surprise, the mutation in NDUFS1 was inherited as a recessive germline allele that underwent LOH in the tumor to expose functional loss of complex I. A re-analysis of 91 OCT tumor genomes revealed that LOH-driven exposure of recessive germline mutations in complex I subunits was a recurrent mechanism underlying complex I inactivation in OCT. These findings unveil a new germline-driven mechanism of complex I loss and metabolic reprogramming in cancer, and provide further evidence of the strong selective pressure for complex I impairment in OCT. Teaser Germline mutations in complex I induce aerobic glycolysis in oncocytic carcinoma of the thyroid through somatic loss of heterozygosity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ec415c1026a19fc77b2f19d7d97c4e52057a5cf" target='_blank'>
              Loss of heterozygosity exposes germline mutations in complex I and drives Warburg metabolism in oncocytic carcinoma of the thyroid
              </a>
            </td>
          <td>
            Celia de la Calle Arregui, A. Frank, Kelvin Mun, Jiwoong Kim, Kuntal Majmudar, Justin A. Bishop, C. Lewis, Yang Xie, David G. McFadden
          </td>
          <td>2025-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Genomic instability is a hallmark of cancer, encompassing both sequence and structural alterations that drive tumor evolution and heterogeneity. The APOBEC3 family of deoxycytidine deaminases has emerged as a major source of mutagenic activity in cancers. R-loops are RNA-DNA hybrids and structural barriers that interfere with replication and transcription. Among the APOBEC3 family, APOBEC3C (A3C) is particularly worthy of attention for its upregulation, driving the DNA replication stress tolerance in response to replication stress-inducing drug gemcitabine. However, the molecular mechanisms of gemcitabine resistance and regulatory circuitries mediated by A3C remain largely unknown, especially in checkpoint-deficient tumors. Initially, we screened that A3C was a putative transcriptional target of p53, and p53-deficient H1299 cells harboring A3C elicited a chemoresistant phenotype upon gemcitabine treatment both in vitro and in vivo. A3C expression enhanced Chk1-dependent S-phase checkpoint activation, thus slowing down replication fork progression and facilitating DNA repair. Pull-down assay and proteomic analysis identified that A3C had a specific interaction with the RNA helicase DDX5, which coordinately played critical roles in R-loop resolution. In contrast to A3C, DDX5 expression attenuated Chk1-dependent S-phase checkpoint activation. Knockdown of DDX5 in A3C-proficient H1299 cells attenuated gemcitabine-induced Chk1 activation and enhanced the therapeutic index of gemcitabine by promoting R-loop accumulation. Therefore, we conclude that A3C/DDX5/R-loop complex may impair the sensitivity of gemcitabine by modulating Chk1 dynamics and DNA replication/damage response machinery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3c0c1bd150bb81f122f34c4b4eaf7affdf65ea8" target='_blank'>
              APOBEC3C coordinates DDX5 in R-loop resolution and dynamic control of Chk1-mediated stress-responsive circuitry as a prerequisite for gemcitabine resistance in p53-deficient cells
              </a>
            </td>
          <td>
            Li Tao, Yang Zhao, Zhuangzhaung Jiang, Shujing Kong, Yanlin Ding, Tengyang Ni, Weimin Wang, Yanqing Liu
          </td>
          <td>2026-01-07</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="We evaluate the effect of most FDA-approved drugs (>7,000 conditions) on double-strand DNA break repair pathways by analyzing mutational outcomes in human induced pluripotent stem cells. We identify drugs that can be repurposed as inhibitors and enhancers of repair outcomes attributed to non-homologous and microhomology-mediated end joining (NHEJ, MMEJ), and homology-directed repair (HDR). We also identify functions of the proteins estrogen receptor 2 (ESR2) and aldehyde oxidase 1 (AOX1), affecting several key DNA repair proteins, such as ATM and 53BP1. Silencing of ESR2 can have a synergistic effect on increasing HDR when combined with NHEJ inhibition (mean 4.6-fold increase). We further identify drugs that induce synthetic lethality when NHEJ or HDR is blocked and may therefore be candidates for precision medicine. We anticipate that the ability to modulate the DNA repair outcomes with clinically safe drugs will help disease modeling, gene therapy, chimeric antigen receptor immunotherapy, and cancer treatment. DNA repair pathways shape CRISPR editing outcomes. Here, authors identified FDA approved drugs that can be repurposed as repair modulators or to induce synthetic lethality, and uncovered new roles for ESR2 and AOX1 in DNA repair, enhancing editing and offering potential therapeutic applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d6eea628e8acb5e08c2cd2c0b97b3bdae99bd71" target='_blank'>
              Repurposing clinically safe drugs for DNA repair pathway choice in CRISPR genome editing and synthetic lethality
              </a>
            </td>
          <td>
            Dominik Macak, Philipp Kanis, S. Riesenberg
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97d63cfbf90a2e544c2d24c8ab605e5d44e8ba24" target='_blank'>
              A Pan-Cancer Single-Cell Compendium of Intratumoural Heterogeneity
              </a>
            </td>
          <td>
            Ido Nofech-Mozes, Ashton Cook, Tom W. Ouellette, Sara Hafezi-Bakhtiari, Philip Awadalla, Sagi Abelson
          </td>
          <td>2026-01-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b84948bcfa1aec61d5686e9537d4f4d0902383f" target='_blank'>
              ALKBH1 drives codon-biased, pro-oncogenic translation and tumor microenvironment remodeling in glioma via tRNA wobble oxidation
              </a>
            </td>
          <td>
            A. Nakayashiki, Arata Nagai, Yuko Iwasaki, Abdulrahman Mousa, Moe Kumai, Munehiro Kimura, Shadi Al-Mesitef, Hiroshi Kanno, S. Yamashita, Daisuke Ando, Yoshiteru Shimoda, Masayuki Kanamori, Keisuke Goda, Hidenori Endo, Sherif Rashad, Kuniyasu Niizuma
          </td>
          <td>2025-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c23f53baca0dc3b384bd4e216c9b8503ff2be41" target='_blank'>
              Pooled single-cell CRISPRa/i screens for functional genomics in bacteria at scale
              </a>
            </td>
          <td>
            Jacob R. Brandner, Quoc Tran, Yujia Huang, Dmitry Sutormin, Karl D. Gaisser, Georg Seelig, J. Zalatan, James M. Carothers, A. Kuchina
          </td>
          <td>2025-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/638cde0ab64b6b068ce716defb37f6b2cb4ab7b9" target='_blank'>
              Unraveling the molecular mechanisms underlying spontaneous multipolar mitosis through CIN-seq
              </a>
            </td>
          <td>
            Pin-Rui Su, Ting-Chun Chou, M. T. López-Cascales, Junfeng Hunag, Tsai-Ying Chen, Sjoukje van Wieren, Chan Li, Cecile Beerens, Li You, J. Storteboom, M. Chien
          </td>
          <td>2026-01-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Alternative splicing (AS) is one of the principal mechanisms of post-transcriptional regulation that confers transcriptomic plasticity and proteomic diversity in cancer, thereby enabling tumor adaptation to therapeutic pressure. However, two obstacles impede the translation of these findings into clinical benefit: The absence of systematic functional annotation of the numerous splice variants associated with drug resistance and the paucity of biomarkers capable of distinguishing de novo from acquired splice-mediated resistance. In the present review, the current mechanistic understanding of AS-driven drug resistance was briefly synthesized, and it was evaluated how existing strategies address these challenges. It was also described how knowledge of dysregulated splicing networks, due to mutations in cis-regulatory elements such as ESS, overexpression of trans-acting factors such as SRSF1, as well as mechanisms such as alternative trans-splicing, in which the spliceosome interacts with splice sites on two distinct RNA molecules and which can be driven by complementary sequences or other trans-acting factors, could be used to more accurately identify tumors dependent on aberrant splicing for survival. In addition, it was outlined how targeting aberrant splice variants to overcome therapeutic resistance can be achieved, such as through spliceosome inhibition (for example, H3B-8800) or antisense oligonucleotides directed to a specific exon or splice junction (for example, targeting exon 2 of MET, which is implicated in cis-regulated AS isoforms, or alternatively spliced isoforms of BCL2L1, BRAF and CDYL). However, therapeutic strategies to target adaptive resistance mechanisms such as AS remain limited, as intratumoral heterogeneity may facilitate the emergence of resistant subpopulations, and as most spliceosome inhibitors are not spliceosome-specific, they exhibit off-target effects. Importantly, it was also discussed how pan-cancer splicing databases and single-cell isoform expression profiling can be integrated with deep-learning models, thereby informing the design of therapeutic strategies to overcome splicing-mediated adaptive drug resistance. Notably, such integration will enable the rational design of isoform-specific combination regimens to dismantle drug-resistance circuits. It is anticipated that the present review will assist the scientific community, including both basic and translational researchers, in translating these findings into interventions that mitigate therapeutic failure in recalcitrant cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1f0892d48c3453d0a0ca7ac5b857f619a850559" target='_blank'>
              Alternative splicing in cancer drug resistance: Mechanisms and therapeutic prospects (Review)
              </a>
            </td>
          <td>
            Wenpu Zhu, Zikun Wu, Chai Luv, Banghe Bao, Weijie Zhao, Guanglong Chen, Hang Yang, Huihan Ai, Faqiang Liu, Zhi Li
          </td>
          <td>2025-12-05</td>
          <td>Oncology Reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The tumour suppressor TP53 (tumor protein p53) is a master regulator of cell cycle, DNA repair, and apoptosis, and its mutation is a hallmark of cancer, with individual mutations exerting distinct effects on tumour biology. Despite accounting for ~7% of all TP53 variants, splice site mutations remain the least studied class, and their functional and clinical consequences are poorly understood. We analyzed 25,058 TP53 variants (18,562 somatic; 6496 germline) to characterize the frequency, molecular impact, transcriptional effects, genomic instability, and clinical outcomes of splice mutations. These alterations showed distinct distributions and substitution patterns between germline and somatic contexts and were frequently associated with copy number alterations, reduced TP53 mRNA, and variable protein expression. Transcriptomic profiling identified two transcriptional phenotypes: one with global suppression of canonical p53 target genes and another with mixed activation and repression independent of tumour type. Genomic instability was elevated in a subset of splice-mutant tumours, correlating with increased relapse risk, while other splice mutations showed lower instability but divergent clinical outcomes, including unexpectedly poor prognoses. Our findings fill a critical knowledge gap, defining the biological and clinical spectrum of TP53 splice site mutations and highlighting their potential as prognostic biomarkers and therapeutic targets in precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9df743368c74812062cf79c30615a68f562fc982" target='_blank'>
              TP53 Splice Mutations Have Tumour-Independent Effects on Genomic Stability and Prognosis: An In Silico Study
              </a>
            </td>
          <td>
            Apeksha Arun Bhandarkar, Noah Ethan Kelly-Foleni, Debina Sarkar, Aaron Jeffs, T. Slatter, Antony Braithwaite, Sunali Y Mehta
          </td>
          <td>2025-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c961a3f1619198d178054a670088d76568b3783" target='_blank'>
              Friend and Foe: Genome-Wide Analysis of the Tardigrade Dsup Protein Expressed in Yeast Reveals Trade-offs of DNA Protection
              </a>
            </td>
          <td>
            H. Gaikani, Marjan Barazandeh, Lewis Hitchens, G. Giaever, Corey Nislow
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Replicative immortality is a hallmark of cancer, driven by the activation of telomere maintenance mechanisms, that is yet to be therapeutically exploited. To expedite discoveries that will enable the development of therapeutics that target telomere maintenance mechanisms, this study provides a resource of telomere biology metrics for a pan-cancer panel of 976 cell lines. We generate proteomic data from data-independent-acquisition mass spectrometry for most of these cell lines and integrate pre-existing multi-omic, drug sensitivity, and molecular dependency data from CRISPR/Cas9 knock-out screens. The data illustrate a broad range and heterogeneity in telomere biology, including states that diverge from the binary model of telomere maintenance activation involving either telomerase or the Alternative Lengthening of Telomeres mechanism. Using the telomere biology metrics and multi-omic data, we derive proteomic and transcriptomic predictors of Alternative Lengthening of Telomeres and telomerase activity levels. Our investigations also reveal molecular vulnerabilities associated with the Alternative Lengthening of Telomeres mechanism and drug sensitivity correlating with telomerase activity levels. These findings illustrate opportunities for leveraging this resource to realize the potential for telomere biology-directed cancer therapeutics and companion diagnostics. The therapeutic relevance of telomere maintenance mechanisms in cancer, remains to be explored. Here, the authors integrate multi-omic data and functional readouts, generate a resource of telomere biology metrics and identify potential molecular vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecc8e08e7ae63f123dc686a0aa51e8c03fa2c2c5" target='_blank'>
              Large-scale drug sensitivity, gene dependency, and proteogenomic analyses of telomere maintenance mechanisms in cancer cells
              </a>
            </td>
          <td>
            Yangxiu Wu, Zhaoxiang Cai, Dale Cross, Jane R. Noble, Kelsy Prest, Jamie Littleboy, Scott B. Cohen, Baylee Edlundh, Jennifer M. S. Koh, Ran Xu, Zainab Noor, Milad Bastami, Sara Valentini, Laura Richardson, S. Barthorpe, Nader Arymanesh, Phillip J. Robinson, P. Hains, Mathew J. Garnett, Qing Zhong, R. Reddel, Karen L MacKenzie
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Insight into the genetic basis for many common autoimmune disorders has been uncovered by genome-wide association studies (GWAS), but this alone does not reveal causal variants, effector genes, or the cell types impacted by disease-associated variation. Here, we generate 3D genomic datasets consisting of promoter-focused Capture-C, Hi-C, ATAC-seq, and RNA-seq and integrate this data with GWAS of 16 autoimmune traits to physically map disease-associated variants to the effector genes they likely regulate in 57 human cell types. The majority of variants implicated by these cis-regulatory architectures are trait-specific, but nearly half of the target genes connected to these variants are shared across multiple autoimmune disorders in multiple cell types, leading to enrichment of similar biological networks. While this suggests a high level of genetic diversity and complexity that converges at the level of target gene and cell type, some trait-specific pathways representing potential areas for disease-specific intervention were identified. We pharmacologically validate squalene synthase, a cholesterol biosynthetic enzyme encoded by the FDFT1 gene implicated by our approach and supported by prior eQTL data in multiple sclerosis and systemic lupus erythematosus, as a novel immunomodulatory drug target controlling T cell inflammatory cytokine production and aiding B cell antibody production in a human lymphoid organoid model. These data represent a comprehensive resource for basic discovery of gene cis-regulatory mechanisms, and the analyses reported reveal mechanisms by which autoimmune-associated variants act to regulate gene expression, function, and pathology across multiple, distinct tissues and cell types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d1170723d11b8f757949a6590030c15749de06c" target='_blank'>
              3D chromatin-based variant-to-gene maps across 57 human cell types reveal the cellular and genetic architecture of autoimmune disease susceptibility
              </a>
            </td>
          <td>
            K. Trang, Prabhat Sharma, Laura Cook, Zachary Mount, Rajan M Thomas, Nikhil N Kulkarni, Emylette Cruz Cabrera, Suzanna Rachimi, M. Pahl, J. Pippin, Chun Su, Klaus H. Kaestner, Joan M. O’Brien, Y. Wagley, Kurt D. Hankenson, Ashley A. Jermusyk, Jason W. Hoskins, L. Amundadottir, Mai Xu, Kevin M. Brown, Stewart A Anderson, Wenli Yang, Paul M. Titchenell, Patrick Seale, B. Zemel, A. Chesi, N. Romberg, M. Levings, Struan F. A. Grant, A. Wells
          </td>
          <td>2025-12-01</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="Genetic variation drives phenotypic diversity and disease susceptibility. Trans-acting genetic variation coordinates genome-wide chromatin changes, yet the molecular mechanisms underlying this regulation remain unclear. Here, we use the power of mouse genetics to investigate how genetic variation at trans-acting loci regulates 3D chromatin interactions. Using HiChIP to map H3K27ac-associated regulatory elements in C57BL/6J and DBA/2J embryonic stem cells (ESCs), we identified 4,962 strain-differential interactions, 71% of which overlapped chromatin accessibility quantitative trait loci (QTL), establishing chromatin interaction variation is predominantly heritable. These differential interactions showed coordinated changes in chromatin state and gene expression, with stronger interactions associated with increased accessibility and transcription. Notably, loci regulated in trans exhibited a unique chromatin signature where weaker interactions were enriched for H3K9me3-marked heterochromatin. Analysis of F1 hybrids revealed dominant repressive effects, consistent with heterochromatin-mediated trans-regulation. To causally test this mechanism, we generated reciprocal congenic mouse strains carrying a Chr13 trans-QTL region. Integrated multiomic profiling of congenic ESC lines demonstrated that this single locus coordinates changes in H3K9me3, H3K27ac, chromatin accessibility, and 3D contact frequency at hundreds of distal targets. Consistent with our expectations, 73-83% of differential interactions at Chr13 trans-QTL targets changed in the predicted direction, demonstrating that heterochromatin-mediated trans-regulation coordinately regulates hundreds of regulatory loci. This work establishes heterochromatin formation as a mechanism by which genetic variation at trans-acting loci coordinates changes across chromatin accessibility, histone modifications, and 3D genome organization, providing a framework for understanding how early developmental chromatin states could generate phenotypic variation while preserving essential developmental programs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b2d7eeaa0c1099ab546496536bae3d803b0827e" target='_blank'>
              Trans-regulation of heterochromatin underlies genetic variation in 3D genome contacts
              </a>
            </td>
          <td>
            Haley J Fortin, Anna Z Struba, Christopher L. Baker
          </td>
          <td>2025-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Long Interspersed Nuclear Elements-1 (LINE-1, L1) are transposable elements that make up roughly 17% of the human genome. These elements can copy and insert themselves into new genomic locations (Kazazian and Moran, N Engl J Med 377:361–370, 2017). Typically, LINE-1 is repressed in healthy tissues but may become activated in various human diseases. LINE-1 expression has been associated with aging (Simon, et al., Cell Metab 29:871–885.e5, 2019; De Cecco et al. Nature 566:73–78, 2019; Della Valle et al. Nat Rev Genet 26:1–12, 2025), neurodegenerative disorders (Roy et al., Acta Neuropathol 148:75, 2024;Frost and Dubnau, Annu Rev Neurosci 47:123–143, 2024; Ravel-Godreuil et al. FEBS Lett 595:2733–2755, 2021), cancer (Rodriguez-Martin et al., Nat Genet 52:306-319, 2020; Taylor et al. Cancer Discov 13:2532–2547, 2023; Solovyov et al. Nat Commun 16:2049, 2025), and autoimmune diseases (Rice et al., N Engl J Med 379:2275–2277, 2018), (Carter et al., Arthritis Rheumatol 72:89–99, 2020). Despite the strong association between LINE-1 expression and disease, the regulatory mechanisms controlling the expression of LINE-1-encoded ORF1p and ORF2p and the link between LINE-1 activity and cancer cell survival remain poorly understood. Gaining insights into these regulatory pathways may help elucidate how LINE-1 contributes to disease pathogenesis. To identify upstream regulators of LINE-1 and genes associated with LINE-1 activity-dependent lethality, we developed a dual-reporter system that simultaneously monitors the protein levels of LINE-1-encoded ORF1p and ORF2p (wild-type or catalytically inactive EN/RT mutant). Using genome-wide CRISPR/Cas9-based screens with this system, we identified candidate genes that may influence LINE-1 regulation at multiple levels, including RNA and protein expression. Alongside known factors such as the HUSH complex, the screens revealed additional genes not previously linked to LINE-1 regulation, suggesting possible new regulatory mechanisms for ORF1p and ORF2p expression. We also identified genes whose loss correlated with reduced viability in a manner dependent on LINE-1 activity. These findings collectively provide a broad resource for exploring cellular factors that may modulate LINE-1 expression and activity. This study provides a resource for investigating the cellular regulation of LINE-1, highlighting distinct candidate factors that may modulate ORF1p and ORF2p expression and influence LINE-1 activity-associated cytotoxicity. While functional validation of these candidate regulators remains necessary, the findings offer a foundation for future studies aimed at experimentally confirming their roles and elucidating the molecular mechanisms underlying LINE-1 regulation and its potential contributions to disease contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85dc75de6176e9aaa531a302da776789c267f8ce" target='_blank'>
              Identification of novel regulators of LINE-1 expression via CRISPR/Cas9 screening
              </a>
            </td>
          <td>
            Ozgur Oksuz, Chong Chu, Cedric Arisdakessian, Liyang Diao, Dennis M. Zaller, Kimberly K Long, Heike Keilhack, Sarah Knutson
          </td>
          <td>2025-12-05</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33139fce83b86aee2cfa45adb9a16cd12cc3fbd6" target='_blank'>
              CBL mutations in pediatric solid and CNS tumors are a marker of receptor tyrosine kinase activation and a potential therapeutic target
              </a>
            </td>
          <td>
            Lauren M. Brown, Chelsea Mayoh, Katherine A. Camper, G. Tax, Pablo Acera Mateos, Wenyan Li, Sam El-Kamand, Patricia Sullivan, James Bradley, Paris I. Thompson, Jingwei Chen, Teresa Sadras, Robert Salomon, Marie Wong, Mark J. Cowley, Antoine de Weck, Loretta M. S. Lau, Neevika Manoharan, Paul G. Ekert
          </td>
          <td>2025-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="CRISPR/Cas9 technologies provide unique capabilities for modeling disease and understanding gene-to-phenotype connections. In cultured cells, chemical-mediated control of Cas9 activity can limit off-target effects and enable mechanistic study of essential genes. However, widely-used Tet-On systems often show leaky Cas9 expression, leading to unintended edits, as well as weak activity upon induction. Leakiness can be problematic in the context of Cas9 nuclease activity, which may result in cumulative DNA damage and degradation of the target cell genome over time. To overcome these deficiencies, we have established transgenic platforms that minimize Cas9 functionality in the OFF-state along with maximized and uncompromised ON-state gene editing efficiency. By combining conditional destabilization and inhibition of Cas9, we have developed an all-in-one (one or multiple guide RNAs and Cas9) ultra-tight, Tet-inducible system with exceptional dynamic range (ON vs. OFF-state) across various cell lines and targets. As an alternative to Tet-mediated induction, we have created a Branaplam-regulated splice switch module for low-baseline and robust Cas9 activity control. Lastly, for circumstances where DNA damage needs to be avoided, we have constructed a dual-control, Tet-inducible CRISPRi module for tight and potent transcriptional silencing. This upgraded suite of inducible CRISPR systems has broad applications for numerous cell types and experimental conditions. Achieving tight Cas9 regulation without sacrificing activity remains difficult. Here, the authors design multi-level circuits combining anti-CRISPRs, splice sites, chemical induction, and degron control to enable ultra-high dynamic range and precise, on-demand genome editing across contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4cc5c8aa5f90027fd142622f22700e2cb43c085" target='_blank'>
              Chemically-inducible CRISPR/Cas9 circuits for ultra-high dynamic range gene perturbation
              </a>
            </td>
          <td>
            Rajini Srinivasan, Tao Sun, Azalia Sandles, Diana Wu, Liang Wang, Hetal Patel, Ryan Pabalate, Maram Bader, A. Heidersbach, Clark Ho, Shiqi Xie, Andrew Ng, Benjamin Haley
          </td>
          <td>2025-12-12</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Lineage plasticity, the ability of cancer cells to alter their differentiated state through transcriptional and epigenetic reprogramming, has emerged as a key mechanism of therapeutic resistance across cancers. This adaptive process can manifest in multiple ways, including epithelial–mesenchymal transition, acquisition of stem-like features, and histological transformation, the most striking and clinically apparent example. In EGFR-mutant lung adenocarcinoma (LUAD), lineage plasticity is increasingly recognized as a prevalent mechanism of acquired resistance to tyrosine kinase inhibitors (TKIs). Among its visible manifestations, histologic transformation into small-cell lung cancer (SCLC) is the most frequent, while squamous transformation and other phenotypic shifts also occur. Transformed tumors typically retain the initiating EGFR mutation but lose EGFR dependence, acquire neuroendocrine features, and display aggressive clinical behavior with poor clinical outcomes compared with both de novo SCLC and non-transformed LUAD. Recent studies show that plasticity arises through combined genomic, transcriptomic, and epigenetic reprogramming, often foreshadowed by molecular alterations before overt histological change. Spatial and single-cell profiling reveal heterogeneous trajectories and intermediate states, while functional models and multi-omics approaches have begun to identify therapeutic vulnerabilities distinct from both de novo EGFR-mutated SCLC and classical EGFR-mutated LUAD. Thus, lineage plasticity, whether manifested as histologic transformation or through more subtle epigenetic reprogramming, represents a formidable resistance mechanism in NSCLC. Defining its molecular basis and temporal dynamics will be essential for early detection, prognostication, and the development of tailored therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99eda567a0f2b2f5e4b0555a0ae32377856dcb42" target='_blank'>
              Lineage Plasticity and Histologic Transformation in EGFR-TKI Resistant Lung Cancer
              </a>
            </td>
          <td>
            Li Yieng Eunice Lau, A. Skanderup, Aaron C. Tan
          </td>
          <td>2025-12-31</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Atypical teratoid rhabdoid tumor (ATRT) is a highly aggressive but genetically simple pediatric central nervous system tumor, defined by biallelic inactivation of the chromatin regulator SMARCB1 with remarkably few other cooperating mutations. Despite its genetic homogeneity, ATRT exhibits profound clinical and epigenetic heterogeneity, with three major subgroups (ATRT-TYR, ATRT-MYC, and ATRT-SHH) defined by DNA methylation and transcriptional signatures. Beyond these subgroup-defining features, we aimed to investigate epigenetic variability within tumors by applying whole-genome bisulfite sequencing and probabilistic modeling to quantify stochastic DNA methylation in primary ATRT samples encompassing all three subgroups. We show that ATRT exhibits a destabilized and increasingly stochastic methylome. While ATRT global methylation patterns diverge according to subgroup, some methylation perturbations, such as hypermethylation and increased methylation entropy over bivalent promoters, are consistent across subgroups. We find that methylation stochasticity alterations map onto potential drivers of ATRT, such as LIN28a, the HOXD cluster for ATRT-MYC, and OTX2 for ATRT-TYR, and identify actionable targets, such as hypermethylation of the tumor suppressor CDKN2a across all subgroups. We investigate the sensitivity of the aberrant DNA methylation landscape of ATRT to pharmacologic DNA methyltransferase inhibition (DNMTi) and histone deacetylase inhibition (HDACi). We show that decitabine leads to profound demethylation of patient-derived ATRT cell lines, including reversal of hypermethylation at bivalent promoters and the CDKN2a locus. The addition of HDACi leads to dramatic gene expression changes, including upregulation of innate immune signaling pathways, such as STING/interferon signaling, genes under the regulation of bivalent promoters, and reactivation of the tumor suppressor CDKN2A. The combination of DNMTi and HDACi synergistically reduces cell viability. Taken together, we show that ATRT has a highly stochastic methylome sensitive to epigenetic manipulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e14cbda965cb78ec2d086d8bbd22a534ed3d3a3" target='_blank'>
              Disordered DNA methylation leads to targetable transcriptional plasticity in ATRT
              </a>
            </td>
          <td>
            Ashley R Tetens, Tyler R Findlay, Jordyn Craig-Schwartz, Athanasia Liapodimitri, Oscar Camacho, Kegan O Skalitzky, Adrian Idrizi, Rakel Tryggvadottir, Kayleigh Lunsford, Eric H. Raabe, Michael A Koldobskiy
          </td>
          <td>2025-12-17</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Ovarian cancer (OC), the third most common gynecologic malignancy, is characterized by high mortality largely driven by chemotherapy resistance, leading to recurrence and metastasis. Using transcriptomic data from GSE73935, we constructed a weighted gene co-expression network and identified eight hub genes (IGF1R, CDH2, PDGFRA, CDKN1A, SHC1, SPP1, CAV1 and FGF18) associated with cisplatin resistance, among which CDH2 emerged as the most clinically relevant candidate. CDH2 demonstrated moderate diagnostic potential (AUC = 0.792) and was markedly upregulated in cisplatin-resistant A2780/CP70 cells. Independent validation using clinical single-cell RNA-seq data (GSE211956) confirmed its selective enrichment in resistant tumor cell subpopulations. Gene set enrichment analysis linked elevated CDH2 expression to p53 signaling, DNA replication, nucleotide excision repair, and Toll-like receptor pathways, with qPCR supporting upregulation of key downstream genes in resistant cells. Immune deconvolution further indicated that high CDH2 expression correlated with increased infiltration of NK cells, Tregs, macrophages, and neutrophils, and immunohistochemistry verified CDH2 overexpression in cisplatin-resistant tissues. In addition, virtual screening and drug sensitivity profiling identified several FDA-approved agents with potential relevance to CDH2-associated drug response. These findings indicate that CDH2 may serve as a candidate marker associated with cisplatin response in OC, and its association with immune cell infiltration provides further insight into mechanisms potentially underlying chemoresistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7b261349cbc2ce5423ecbd97a555c9bd93cba98" target='_blank'>
              Integrated Analyses Identify CDH2 as a Hub Gene Associated with Cisplatin Resistance and Prognosis in Ovarian Cancer
              </a>
            </td>
          <td>
            Jun-Yi Xu, Mao-Qi Tian, Rui Yang, Zi-Xuan Li, Zi-Heng Lin, Yu-Fei Wang, Yu-Hang Chu, Wei-Ning Sun, Ya-Mei Wang
          </td>
          <td>2026-01-10</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fab0829fa668d3f68087f51374ae0be2ba943151" target='_blank'>
              Perturbational single-cell profiling of patient tumors defines lineage- and context-specific programs of innate immune resistance
              </a>
            </td>
          <td>
            C. Perry, A. Frey, Y. Fei, J. Wang, P. Costa, H. N. Amin, S. Ghadermarzi, D. Levine, W. Tong, A. Koda, M. Mackie, M. He, Y. Nie, K. Clulo, F. Ouerghi, J. Wei, T. Cordero Dumit, M. Yaskolko, M. Ding, A. Caldera, O. Kyrysyuk, L. Lum, J. W. Allen, W. Guo, A. Naqash, A. Elliot, A. Vanderwalde, M. Capelletti, T. Adeyelu, S. Barry, M. Hugaboom, A. Bacchiocchi, H. Kluger, M. Bosenberg, A. Iwasaki, D. Braun, J. Clune, D. van Dijk, K. Olino, J. Ishizuka
          </td>
          <td>2025-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a genetically heterogeneous malignancy where traditional DNA sequence-based classification fails to fully explain clinical outcomes. This review explores the dynamic and multifaceted role of the epigenome as a critical driver of leukemogenesis, progression, and therapeutic resistance. We detail how disruptions in DNA methylation, histone modifications, chromatin architecture, and non-coding RNAs create a complex regulatory landscape that transcends genetic alterations. We highlight the emergence of single-cell multi-omics technologies, which are resolving intratumoral heterogeneity and uncovering the epigenetic signatures of leukemic stem cells. Furthermore, we discuss the significant translational potential of these discoveries, including the development of epigenetic biomarkers for refined risk stratification, minimal residual disease (MRD) monitoring, and predicting response to novel epigenetic therapies like menin and LSD1 inhibitors. Integrating these dynamic epigenetic layers into clinical decision-making promises to usher in a new era of precision medicine, ultimately improving prognostic accuracy and therapeutic outcomes for AML patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d3853f768df9c1d2f10b0fd3237fd7635474d26" target='_blank'>
              Beyond the DNA sequence: mapping the dynamic epigenetic landscape for risk stratification and therapeutic intervention in acute myeloid leukemia
              </a>
            </td>
          <td>
            Jiqian Xie, Hamed Soleimani Samarkhazan
          </td>
          <td>2025-12-16</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Variants of uncertain significance represent the biggest challenge for genomics-based precision oncology. Activated fibroblast growth factor receptors (FGFRs) frequently drive tumorigenesis by different genetic aberrations. However, it remains unknown which of the many point mutations affecting FGFR1, FGFR2, FGFR3 or FGFR4 in cancer are druggable, that is, activating signaling while not mediating FGFR inhibitor resistance. Here we implemented a saturation mutational scanning platform to screen all 11,520 possible point mutations covering the kinase domains of FGFR1–4. Pooled positive selection screens identified 474 activating and 738 mutations mediating resistance to the FGFR inhibitors pemigatinib and futibatinib, together revealing 301 druggable FGFR mutations analogous to a strong PS3/BS3 evidence level. The screens also identified loss-of-function mutations and an association of gain-of-function mutations with hydrophobic changes. The functional screens identified 97% of acquired resistance mutations in clinical trials. Our comprehensive catalog of every druggable mutation in the FGFR kinase domains is readily available for clinical decision support. Saturation mutagenesis screening examines 11,520 point mutations in the kinase domains of FGFR1, FGFR2, FGFR3 and FGFR4, identifying their activating and resistance properties to the FGFR inhibitors pemigatinib and futibatinib.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/091d9948ae5e0367c92c6fee36f39d1751918d53" target='_blank'>
              Saturation mutagenesis identifies activating and resistance-inducing FGFR kinase domain mutations
              </a>
            </td>
          <td>
            Carla Tangermann, Avantika Ghosh, Martin Ziegler, Francesco Facchinetti, Jannis Stappenbeck, Yagmur Oyku Carus Sahin, M. Riester, Luise Carmina Viardot, Tobias Zundel, L. Friboulet, Antoine Hollebecque, J. J. Naveja, Angela Wanninger, M. Hess, Tilman Brummer, M. Boerries, S. Loges, Y. Loriot, A. Illert, Sven Diederichs
          </td>
          <td>2025-12-08</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="C1orf50 encodes a small, evolutionarily conserved protein, the function of which remains unclear. Its significance across various human cancers, particularly its specific role in ovarian cancer within an immunogenomic context, is not yet fully understood. Utilizing The Cancer Genome Atlas and single-cell RNA sequencing (scRNA-seq) public datasets, we conducted a comprehensive profiling of C1orf50 across multiple cancer types, with a particular focus on ovarian cancer, to investigate its associations with copy-number status, genomic instability, tumor programs, and the immune microenvironment. Across cancer types, copy-number gain or amplification of C1orf50 was most frequent in ovarian cancer and closely tracked with higher messenger RNA levels. Higher C1orf50 expression was associated with a greater tumor mutational burden and homologous recombination deficiency, as indicated by gene-set patterns that suggested heightened cell-cycle and cellular stress responses accompanied by reduced oxidative phosphorylation, enrichment of regulatory T cells, and depletion of resting memory CD4 T cells. In ovarian cancer, focal events at chromosome 1p34.2 were accompanied by stepwise increases in C1orf50 expression by clinical stage and were linked to higher tumor mutational burden, homologous recombination deficiency, and greater loss of heterozygosity, together with more frequent gene alterations in BRCA1 or BRCA2. Immune composition clustered into profiles consistent with an immunosuppressive context in tumors with higher C1orf50 expression. The scRNA-seq data further revealed that cancer cells enhanced immune-suppressive interactions with various immune cell populations and diminished antigen-presentation signals. Analyses of genomic instability in ovarian cancer suggested mutational processes compatible with base-substitution patterns associated with cytidine deaminase activity and with insertion-deletion patterns characteristic of homologous recombination failure, while transcript-level patterns pointed to a broad downshift of canonical DNA repair activity with apparent compensatory adjustments in related pathways rather than a uniform change in any single pathway. The overexpression of C1orf50 characterizes an aggressive immunogenomic phenotype in ovarian cancer, distinguished by genomic instability, impaired DNA repair mechanisms, and extensive immunosuppression. These findings indicate that C1orf50 warrants consideration as a potential biomarker and a prospective target for therapeutic investigation. Furthermore, they advocate for the progression to prospective validation and functional studies to ascertain its clinical significance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aef46ec0fd0bf3582b841375c94b60ec801575fd" target='_blank'>
              Pan-cancer profiling links C1orf50 to DNA repair and immune modulation in ovarian cancer
              </a>
            </td>
          <td>
            Anna Rogachevskaya, Yusuke Otani, Akira Ohtsu, Vanessa D. Chin, Tirso Peña, Seiji Arai, Shinichi Toyooka, Atsushi Fujimura, Atsushi Tanaka
          </td>
          <td>2025-12-08</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Xeroderma Pigmentosum group C (XP-C) is an autosomal recessive disorder caused by mutations in the XPC gene, leading to defective nucleotide excision repair. This defect leads to genomic instability and a profound cancer predisposition. To model this disease, we generated induced pluripotent stem cells (iPSCs) from an XP-C patient carrying a novel homozygous nonsense mutation in the XPC gene (c.1830C>A). The resulting iPSCs demonstrated typical pluripotent characteristics, including expression of key markers and trilineage differentiation capability. However, genomic assessment revealed progressive karyotypic instability during extended culture. While initial whole-genome sequencing detected no major chromosomal abnormalities, subsequent G-banding analysis identified acquired trisomy 12 in two lines (CL12 and CL27) and a derivative X chromosome in a third line (CL30). These abnormalities were absent in early-passage analyses, indicating that they were acquired and selected for during extended culture. The acquisition of a derivative X chromosome in CL30, alongside recurrent trisomy 12, represents a unique cytogenetic signature likely attributable to the underlying XPC defect. We hypothesize that the loss of GG-NER creates a permissive genomic environment, accelerating the accumulation of DNA damage and chromosomal missegregation under replicative stress. This temporal divergence in genomic integrity highlights how culture pressures drive chromosomal evolution in XP-C iPSCs independently of initial reprogramming. Our findings emphasize that XP-C iPSCs require continuous genomic surveillance and provide a model for investigating how DNA repair deficiencies interact with in vitro culture stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6a1050cf9f3b317ce0a251707bdfe662b15e797" target='_blank'>
              Karyotypic Profiling of Induced Pluripotent Stem Cells Derived from a Xeroderma Pigmentosum Group C Patient
              </a>
            </td>
          <td>
            Almaqdad Alsalloum, Natalia Mingaleva, Ekaterina Gornostal, Z. Antysheva, P. Sparber, M. Skoblov, V. Pozhitnova, T. Belysheva, Aygun Levashova, Ekaterina Kuznetsova, Yulia Suvorova, J. Krupinova, V. Bogdanov, A. Abyzov, O. Mityaeva, P. Volchkov
          </td>
          <td>2025-12-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc51775c02383b3021d6f8784d1d014c736cc552" target='_blank'>
              Chromosome-arm 17p Loss Renders Breast Cancer Cells Vulnerable to AURKB Inhibition
              </a>
            </td>
          <td>
            Tom Winkler, Eran Sdeor, Ron Saad, Hajime Okada, Kathrin Laue, Gil Leor, Guy Wolf, S. S. Shachar, U. Ben-David
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Synovial sarcoma (SyS) is an aggressive soft-tissue malignancy that is characterised by a pathognomonic t(X;18)(p11.2;q11.2) translocation, which produces the fusion oncogene named SS18::SSX. Despite recent advancements in our understanding of synovial sarcoma biology, the cell-of-origin remains undefined. A mesenchymal stromal cell (MSC) specific CreERT2 line was employed to express SS18::SSX in fibroblasts and related cell types, resulting in 100% penetrant synovial sarcoma development in mice, with a median latency period of 16.2 ± 2.8 weeks. Murine tumours exhibited high concordance with human synovial sarcoma subtypes at the histological and molecular levels. Genetic refinement of the cell-of-origin revealed that synovial sarcomas derive from a rare Hic1+Pdgfra+Lgr5+ fibroblastic population. Furthermore, comparative transcriptomic analysis revealed the acquisition of a transformed phenotype initiated by the loss of a mature fibroblastic profile and subsequent unmasking of an epigenetically embedded embryonic MSC program. Adult and embryonic MSCs exhibited overlapping H2AK119ub and H3K4me3/H3K27me3 (bivalent) histone marks, while SS18::SSX-mediated transformation culminated in the widespread loss of H3K27me3 at these genes and their consequent transcription. Collectively, these studies define a rare MSC context, conducive for SS18::SSX-mediated transformation, and demonstrate that SyS tumorigenesis involves the induction and maintenance of an embryonic-like MSC phenotype. The cellular origin of soft-tissue cancers, such as synovial sarcoma (SyS), is unknown. Here, expression of the oncoprotein, SS18::SSX, in fibroblasts was sufficient to produce human-like SyS tumours, thereby identifying a cell of origin for SyS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bc176f3d1d35f0c70401edbc66ca480c84142be" target='_blank'>
              The fibroblast epigenome underlies SS18::SSX-mediated transformation in synovial sarcoma
              </a>
            </td>
          <td>
            Lesley A Hill, R. Wilder, Lauren A. Martin, Martin Arostegui, George Davenport, Marcos Vemon, Jakob Hofvander, Xue Qi Wang, Jinxiu Li, Torsten O. Nielsen, Kevin B. Jones, Martin Hirst, Michael Underhill, T. Michael
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>15</td>
        </tr>

        <tr id="Myeloid sarcoma, an aggressive extramedullary subtype of acute myeloid leukemia (AML), occurs in ∼10% of patients, and has not yet been included in large-scale genomic studies. The critical biological changes that drive tumor evolution are unknown, its detection in asymptomatic patients remains a clinical challenge, and treatment options are limited as patients are often excluded from clinical trials, rendering it a neglected disease entity. Based on comprehensive multi-omic profiling, we demonstrate that myeloid sarcoma evolves from medullary AML with distinct sitespecific clonal evolution patterns. Additionally, we show that circulating tumor DNA sequencing can serve as a non-invasive method for molecular profiling of myeloid sarcoma, offering a novel avenue in molecular diagnostics. We characterize unique transcriptional profiles of myeloid sarcoma, reflecting immune evasion and adaptation to an extramedullary microenvironment. We provide evidence for a key role of RAS pathway activation and demonstrate in murine models of myeloid sarcoma that RAS inhibition effectively reduces tumor burden. Overall, our data highlight key differences between medullary AML and myeloid sarcoma including universal molecular evolution and RAS pathway activation as hallmarks of the disease and nominate RAS inhibition as a promising therapeutic strategy for patients with myeloid sarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10582cdaf7507df9370133688fbcd15ae5208a8b" target='_blank'>
              Multiomic characterization, early detection, and therapeutic targeting of myeloid sarcoma
              </a>
            </td>
          <td>
            B. Nadorp, Audrey Lasry, S. Loghavi, Ravi Patel, Hager Mansour, Benjamin J. Kelly, Christopher J. Walker, Jill Buss, Isaiah Boateng, Wafa Al-Santli, Zoe Ciantra, Rebecca Austin, Adrienne Heyrosa, Helee Desai, Andrea Laganson, Hasan Abaza, Linda Procell, Tejas Patel, Benjamin Kaffenberger, Saranga Wijeratne, Maria Guillamot, Maria Velegraki, Luis Chiriboga, Zihai Li, L. Abruzzo, Daniel A. Pollyea, Christine M McMahon, A. Shana'ah, John C. Byrd, Alan H Shih, R. L. Levine, E. Papapetrou, A. Tsirigos, Elaine R. Mardis, Alice S. Mims, I. Aifantis, Ann-Kathrin Eisfeld
          </td>
          <td>2025-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="Purpose This study aimed to investigate the role of minichromosome maintenance complex component 6 (MCM6), a DNA replication licensing factor, in retinoblastoma progression and its impact on melphalan chemosensitivity. Methods MCM6 expression patterns were analyzed using single-cell RNA sequencing (scRNA-seq) of retinoblastoma and validated in patient tumors, including specimens obtained after failed melphalan therapy. Stable MCM6 knockdown cell lines were established for proliferation and cell-cycle assays, DNA damage analyses, and chemosensitivity testing. In vivo xenograft models were employed to evaluate the therapeutic efficacy of MCM6 knockdown combined with melphalan. Results The scRNA-seq revealed that MCM6 was highly expressed in retinoblastoma cells and embedded in a proliferation-associated gene network. Elevated expression was also confirmed in human retinoblastoma, particularly in tumors from patients with failed melphalan therapy. MCM6 knockdown suppressed cell proliferation and cell-cycle progression while enhancing melphalan-induced DNA damage, thereby sensitizing retinoblastoma cells to melphalan. In vivo, MCM6 depletion synergized with melphalan to significantly inhibit intraocular tumor growth. Conclusions MCM6 acts as a critical regulator of retinoblastoma growth and modulates response to melphalan. Targeting MCM6 may offer a therapeutic approach to improve outcomes of chemotherapy in retinoblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dde806685583a484cb5f310887b499f1a44ba7c5" target='_blank'>
              Targeting MCM6 Enhances Melphalan Chemosensitivity in Retinoblastoma by Modulating DNA Damage Response
              </a>
            </td>
          <td>
            Meng Wang, Junjie Tang, Jinmiao Li, Jianjie Lv, Zhihui Zhang, Yaoming Liu, Hetian Sun, Yang Gao, Shuxia Chen, L. Tang, Ping Zhang, Shicai Su, Rong Lu
          </td>
          <td>2025-12-01</td>
          <td>Investigative Ophthalmology & Visual Science</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Endometrial cancer (EC) harbors highly recurrent cell cycle pathway alterations, especially hyperactivation of the CCND1/CDK4/6 axis, raising the potential for use of CDK4/6 inhibitors in these cancers. In this study, we performed a genome-wide CRISPR-Cas9-based knockout screen to identify genes that modify the response to CDK4/6 inhibitors. We found that NIMA-related kinase-6 (NEK6) levels determine the anti-tumor effects of CDK4/6 inhibitors and that NEK6 was a synthetic lethal target of CDK4/6 inhibitors in EC. We further demonstrated that combined inhibition of NEK6 and CDK4/6 resulted in marked suppression of tumor growth in vitro and in vivo. Mechanistically, NEK6 bound to Y-box binding protein-1 (YBX1) in the cytoplasm and facilitated its phosphorylation at serine 102 (p-YBX1S102), which promoted YBX1 nuclear translocation and further activated CDK2 and bcl2 transcription. As such, these data provided robust pre-clinical mechanistic evidence of synergy between NEK6 and CDK4/6 inhibitors and delineated a path for translation of these findings to preliminary clinical studies in EC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a22ac5a8b88f2cf91c83604dca40e2720bb056d3" target='_blank'>
              Genome-wide CRISPR screen identified NEK6 as a determinant of sensitivity to CDK4/6 inhibitor in endometrial cancer
              </a>
            </td>
          <td>
            Fangfang Jian, Chongying Zhu, Yiwei Wang, Tianyu Zhou, Chenmin Yang, Weiwei Feng
          </td>
          <td>2026-01-05</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a330bae0129f86adaee3ff8e6b953e0c449188da" target='_blank'>
              Virus-like particles enable targeted gene engineering and pooled CRISPR screening in primary human myeloid cells
              </a>
            </td>
          <td>
            Hyuncheol Jung, Pascal Devant, Carter Ching, Mineto Ota, Jennifer Hamilton, Zachary Steinhart, Wayne Ngo, Luis R. Sandoval, Jae Hyung Jung, Da Xu, Meirui An, Esha Urs, P. Chen, Vincent Allain, Takuya Tada, James K. Nuñez, Nathaniel R Landau, David R. Liu, Justin Eyquem, Jennifer A. Doudna, Alex Marson, J. Carnevale
          </td>
          <td>2025-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52aead1bb294c5c8ab3425b1373986d70ceb1c0f" target='_blank'>
              Blocking Minor Intron Splicing Disrupts DNA Repair and Overcomes Therapy Resistance in Prostate and Breast Cancer
              </a>
            </td>
          <td>
            Anke Augspach, P. Francica, Kaitlin N. Girardini, Muriel Jaquet, Marika Lehner, A. Rodriguez-Calero, Charlotte Komarek, Martín González-Fernández, Hannah L. Williams, Xiaoyue Deng, Lea Lingg, Cristina Zivko, Vera Fuchs, Seynabou Diop, S. Maletti, Simone de Brot, A. Ewe, Elena Perugini, Nigel B Jamieson, Y. Doncheva, Claire K Dietrich, Phillip Thienger, Dilara Akhoundova, Andrej Benjak, A. Aigner, R. Kanadia, Sven Rottenberg, Mark A. Rubin
          </td>
          <td>2025-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Medulloblastoma (MB) is one of the most prevalent malignant brain tumors in children, with tremendous cognitive and neuroendocrine disability among survivors. Group 3 MB (G3MB) has poor overall survival at <50%, high frequencies of metastases, and no targeted therapies. Amplification of MYC and activation of TGFβ signaling occur frequently in G3MB. Many tumors have no reported mutations, suggesting epigenetic drivers. We here describe novel humanized models for G3MB from human induced pluripotent stem cells (hiPSC). By transducing hiPSC-derived neuroepithelial stem cells (NESC), we determined that: 1) both MYC and TGFβ effectors drove tumors in vivo; 2) MYC/TGFβR1 in combination led to more aggressive tumors and resistance to clinical inhibitors of TGFβ, and 3) NESC-derived tumors clustered with human G3MB. To decipher mechanisms, we integrated RNA-sequencing with CUT&RUN (for MYC genomic localization and post-translational modification of histones). MYC-bound neural developmental genes were repressed in MYC/TGFβR1 co-driven lines. Gene signatures associated with the Polycomb Repressive Complex (PRC) demarcated with H3K27me3; the histone mark directly regulated by PRC. We identified JARID1B, a MYC binding partner and H3K4me3 demethylase, as a regulator of repressed neural genes. Primary G3MB also showed increased levels of H3K27me3 concurrent with higher expression of JARID1B. Knockdown of JARID1B in human G3MB cell lines reduced growth, supporting potential as a therapeutic target. We conclude that a MYC-TGFβ-JARID1B axis represses target genes to drive G3MB and present new humanized models for G3MB to understand epigenetic dysregulation in G3MB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd58fafdd674bdec3d7247a90b13d164f167e167" target='_blank'>
              Human stem cell models for group 3 medulloblastoma uncover JARID1B as a regulator of the chromatin landscape
              </a>
            </td>
          <td>
            Zulekha A. Qadeer, Elise Hou, Dara Siegel, Samantha Westelman, B. Gudenas, Kyle S Smith, Wazim Mohammed Ismael, Linyu Wang, Mackenzie O Johnson, Shane Grele, Ronald Phua, Liam D. Hendrikse, Ekin Guney, Miller Huang, Alexandre Gaspar-Maia, P. Northcott, Michael D. Taylor, William A. Weiss
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3025c50ede358eca90cea554b244f6195b45b1bd" target='_blank'>
              Architecting cis-regulation to quantitatively tune gene expression in cereals
              </a>
            </td>
          <td>
            Evan D. Groover, David Ding, F. Z. Wang, Gonzalo Benegas, Joseph Rivera, Shahar Schwartz, Stephen Chen, Michael Moubarak, V. Georgieva, P. Lemaux, B. Staskawicz, Krishna K. Niyogi, Yun S. Song, David F. Savage
          </td>
          <td>2026-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="Type B1 and B2 thymomas are lymphocyte‐rich malignant tumours with few somatic mutations in protein‐coding regions of the nuclear genome; nonetheless, noncoding regions remain uncharacterized. Here, we developed a method to isolate pure thymoma cells from lymphocyte‐rich tissues, and then performed genome‐wide deep sequencing. The total number of somatic mutations was ~80 times higher in noncoding regions than in coding regions in type B12 thymomas (1,671.3 versus 21.1 per case). Coding mutations were identified in epigenetic regulators, DNA repair genes, and some other genes. Nevertheless, 40% of the cases exhibited fewer than four nonsynonymous mutations in coding regions. A systematic noncoding analysis identified 405.0 mutations per case in cis‐regulatory elements and detected six recurrent mutations: one interferon regulatory factor (IRF8), two E3 ubiquitin ligases (UBR2 and RNF213), and three intergenic regions. Tumour‐specific/enriched mitochondrial heteroplasmic shift was observed in 90% of cases, with a significant proportion of mutations located in the D‐loop region. When tracing the evolutionary lineage of mtDNA mutation, the majority of cases can be explained by a linear evolutionary model. This suggests that positive selection may be operating on the mitochondrial genome during thymoma development. In summary, numerous noncoding mutations and mitochondrial heteroplasmic shift were detected in type B1 and B2 thymomas, some of which may be functional. Given the paucity of coding mutations observed in this disease entity, other factors such as disruption of the noncoding landscape and tumour‐specific/enriched mitochondrial heteroplasmic shift, may contribute to the development of thymoma. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/983bfc8a32a72d21e775036e5fe74d52ea0ed5cc" target='_blank'>
              Genome‐wide analysis of somatic noncoding mutation patterns and mitochondrial heteroplasmic shift in type B1 and B2 thymomas
              </a>
            </td>
          <td>
            Kohei Fujikura, Isabel Correa, Susanne Heck, Kaoru Watanabe, Juliet King, Emma Mclean, Susan Ndagire, Yoshihisa Takahashi, Masahiko Kuroda, Andrea Billè, Daisuke Nonaka
          </td>
          <td>2025-12-04</td>
          <td>The Journal of Pathology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1b81384b510909f679a401421e801231c6a36b6" target='_blank'>
              A tailored in vivo CRISPR screen identifies BAP1 as a potent tumor suppressor of soft tissue sarcoma
              </a>
            </td>
          <td>
            Jianguo Huang, Xingliang Liu, Warren Floyd, William Haugh, Zhaoyu Sun, Melissa J. Kasiewicz, Yaping Wu, Brian D. Piening, John T. Welle, Wesley K. Rosales, V. Rajamanickam, So Young Kim, Eric S. Xu, Lixia Luo, Yan Ma, Rutulkumar Patel, Ziqiang Zhang, B. Bernard, William L Redmond, Walter J. Urba, R. B. Bell, D. Kirsch
          </td>
          <td>2025-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="


 Prostate cancer (PCa) is the most commonly diagnosed cancer and the second leading cause of cancer-related death in American men. Men with germline BRCA2 mutations are at especially high risk (19%-61% by age 80) of developing PCa. Loss of BRCA2 impairs homologous recombination repair (HRR), resulting in genomic instability and enhanced tumor progression. Our laboratory previously performed a whole-genome CRISPR-Cas9 screen in a BRCA2 mutant cell line under cisplatin induced stress. This screen identified Chromodomain Helicase DNA-binding protein 1 (CHD1) as the top gene whose loss promoted cell survival under DNA damage stress. CHD1 is a chromatin remodeler that is deleted in approximately 15% PCas and is strongly linked to metastatic disease. However, how its loss drives tumor progression, particularly in BRCA2-deficient PCas, remains poorly understood. To address this, we carried out RNA-seq analysis in castration-resistant prostate cancer (CRPC) cell line PC-3 following knockdown of CHD1 and BRCA2, individually and in combination. Notably, depletion of CHD1 in BRCA2-deficient cells resulted in the upregulation of gene signatures associated with hypoxia and epithelial-mesenchymal-transition (EMT). Functional validation using wound healing and migration assays demonstrated that CHD1 loss significantly enhanced the migration ability of PC-3 cells (including BRCA2-deficient) compared to controls. In keeping with increased EMT, we also see high vimentin expression in CHD1-depleted PC3. Furthermore, Causal Inference Enrichment (CIE) analysis of the transcriptomic data pointed to significant activation of transcription factors (TFs) FOSL1 and SOX9 after CHD1 knockdown compared to control cells. To assess clinical relevance, we analyzed RNA-seq data from the Stand Up To Cancer (SU2C) database. This analysis revealed that CHD1-low tumors exhibited upregulation of multiple oncogenic pathways, especially EMT, angiogenesis, apical junction, and IL2-STAT5 signaling, consistent with our in vitro observations. Finally, genetically engineered mouse model (GEMM)-derived prostate organoids from B2PPt (Brca2flox/flox;P53flox/flox;Ptenflox/flox) and B2CPPt (Brca2flox/flox; Chd1flox/flox;P53flox/flox;Ptenflox/flox) mice, after Adeno-Cre infection, revealed higher Fosl1, along with EMT marker vimentin in B2CPPt organoids compared to B2PPt, supporting FOSL1-dependent lineage plasticity and activation of EMT programs upon CHD1 loss in BRCA2-mutant prostate cells. Collectively, these findings indicate that CHD1 loss, either independently or in the context of BRCA2 deficiency, promotes oncogenic phenotypes through activation of FOSL1-dependent lineage plasticity and EMT programs that favor metastasis. Ongoing work dissects how CHD1 loss reshapes chromatin accessibility to reprogram transcriptional networks and drives oncogenic signaling and metastatic pathways in PCa, with particular relevance to BRCA2-deficient PCa.



 Jingzhu Hao, Shailja Pathania, Miklos Diossy, Kourosh Zarringhalam, Changmeng Cai, Zoltan Szallasi. Dissecting the role of CHD1 as a key regulator of pro-metastatic transcriptional reprogramming in BRCA2-mutant prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr PR017.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a2ea6307c1138357462e3893b183854dd498126" target='_blank'>
              Abstract PR017: Dissecting the role of CHD1 as a key regulator of pro-metastatic transcriptional reprogramming in BRCA2-mutant prostate cancer
              </a>
            </td>
          <td>
            Jingzhu Hao, S. Pathania, M. Dióssy, Kourosh Zarringhalam, Changmeng Cai, Zoltán Szállási
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Single-cell sequencing (scRNA-seq) has emerged as a pivotal technology for deciphering the complex cellular heterogeneity and tumor microenvironment (TME) of pancreatic ductal adenocarcinoma (PDAC), positioning it as a critical tool for informing novel therapeutic strategies. This review explores how scRNA-seq reveals diverse cellular subpopulations and their functional roles within the PDAC TME, including malignant epithelial cells with transitional phenotypes, heterogeneous cancer-associated fibroblasts (CAFs), functionally distinct immune cells such as tumor-associated neutrophils (TANs) and macrophages (TAMs), and actively participating neural components like Schwann cells. These cellular constituents form specialized functional units that drive tumor progression, immune evasion, neural invasion, and therapy resistance through metabolic reprogramming, immunosuppressive signaling, and cellular plasticity. The review further examines technological advances in single-cell sequencing from 2023 to 2025, focusing on sample preprocessing innovations, multi-omics integration (combining transcriptomics with epigenomics and proteomics), spatial resolution enhancements, and customized computational tools that address PDAC-specific challenges. Clinically, single-cell sequencing enables precise cellular subtyping, identification of novel biomarkers, and development of personalized therapeutic approaches, including combination therapies targeting specific cellular subpopulations and their interactions. Despite these advances, significant challenges remain in standardizing clinical applications such as liquid biopsy for early detection and tumor microenvironment assessment for diagnostic staging, validating biomarkers like CLIC4, GAS2L1, Cytokeratins, Vimentin and N-cadherin in circulating tumor cells, and comprehensively integrating multi-omics data. Future research focusing on both technology refinement and biological validation will be essential for translating single-cell insights into improved diagnostic and therapeutic outcomes for pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33d9576dbb6fe77bb5f04f2ad67b474589557936" target='_blank'>
              Single-Cell Sequencing Unravels Pancreatic Cancer: Novel Technologies Reveal Novel Aspects of Cellular Heterogeneity and Inform Therapeutic Strategies
              </a>
            </td>
          <td>
            Keran Chen, Zeyu Chen, Jinai Wang, Mo Zhou, Yun Liu, Bin Xu, Zhi-Jing Yu, Yiming Li, Guanhu Yang, Tiancheng Xu
          </td>
          <td>2025-12-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Tumor cell plasticity plays a key role in the progression of malignant neoplasms and therapy resistance, yet its spatial dynamics remain poorly understood. This study investigates the spatial distribution of epithelial–mesenchymal transition (EMT) and stemness, and their prognostic relevance in lung adenocarcinoma (LUAD). Single‐cell RNA sequencing was performed on 10 tumor fragments from three regions (core, core‐adjacent, and edge) of a single LUAD using a SURFSeq 5000 platform. Stemness potential was identified by CytoTRACE2, EMT by UCell and the Hallmark_epithelial_mesenchymal_transition signature, and tumor cells associated with disease prognosis by Scissor. LUAD displayed pronounced spatial heterogeneity of tumor cell plasticity. Stemness potential was heightened at the edge, while EMT was most prominent in the core. The correlation between EMT and stemness was slightly higher at the edge, though it remained negligible (R = 0.17, p = 5.5e−7). Higher EMT scores were observed in tumor cells associated with poor overall survival (r = 0.433, p < 2e−16), whereas stemness was prevalent in tumor cells associated with poor progression‐free survival (r = 0.251, p = 7.15e−6). Tumor cells associated with poor prognosis were enriched with the MYC‐targets v1 signature with PCBP1 and PA2G4 as the top contributing genes. Immunohistochemical data showed that PCBP1 and PA2G4 proteins predominantly localized within tumor cells of LUAD. Taken together, these findings highlight the spatial heterogeneity of tumor cell plasticity features in LUAD and uncover biomarkers associated with poor prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/badf7545e441ad3330c93dcdb325456db2158597" target='_blank'>
              Spatial Dynamics of Tumor Cell Plasticity in Lung Adenocarcinoma Revealed by Region‐Specific Single Cell Transcriptomics
              </a>
            </td>
          <td>
            A. Khozyainova, Maxim E. Menyailo, Vera G. Subrakova, Dmitry M. Loos, P. Iamshchikov, Anton A Fedorov, M. S. Tretyakova, Ustinia A Bokova, Evgeny O Rodionov, Sergey V Miller, Evgeny B Topolnitsky, D. V. Piankov, Sergei A Korostelev, Evgeny V. Denisov
          </td>
          <td>2025-12-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Homozygous deletion of the 9p21.3 genomic locus spanning the CDKN2A/B and MTAP genes is an event affecting 15% of cancers. While CDKN2A is a well-established tumor suppressor gene, the role of MTAP in tumorigenesis varies across cancer types. MTAP codes for methylthioadenosine phosphorylase, a key enzyme in the methionine salvage pathway, and its loss has been associated with several downstream synthetic vulnerabilities. Despite multiple efforts to exploit MTAP loss for targeted therapies, none of these efforts have yielded substantial results in clinical trials. In this review, we consolidate the existing literature along with our systematic analysis to provide an updated perspective on the incidence of MTAP loss in different cancers and elucidate its impact on metabolism, immune microenvironment, and tumor progression. In addition, we summarize the therapeutic strategies that have been investigated preclinically on MTAP-null tumors before and after the advent of functional genomic screening tools. We further assess the current landscape of clinical trials investigating MTAP-targeted inhibitors, evaluating their limitations and potential avenues for improvement. The insights gained from this review will inform future research directions beyond the promising PRMT5/MAT2A axis for rational combination therapies that would work synergistically to eradicate this devastating disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/846311f0527655d852715f68535274117ddc12d9" target='_blank'>
              MTAP-Null Tumors: A Comprehensive Review on Synthetic Vulnerabilities and Therapeutic Strategies
              </a>
            </td>
          <td>
            Bavani Subramaniam, Wai Chin Chong, Aylar Babaei, M. Bornhorst, Chunchao Zhang, R. Packer, Javad Nazarian
          </td>
          <td>2025-12-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Personalized assessment of immunocompetence and DNA double-strand break (DSB) repair requires methods that are sensitive to genetic and molecular complexity beyond the well-known monogenic disorders. Inspired by decades of research using B cells to study DNA repair processes, here we present SWIBRID (SWItch junction Breakpoint Repertoire IDentification), a tool to systematically profile genomic junctions generated in vivo during antibody class switch recombination (CSR) in B cells. As CSR junctions reflect immune diversity and DNA repair proficiency, SWIBRID detects phenotypic manifestations of deficiencies via a highly scalable, blood-based PCR followed by long-read sequencing and bioinformatic analysis. We show that specific DNA repair defects, including cancer-associated mutations, exhibit distinct CSR junction patterns. Notably, SWIBRID distinguishes different types of DSB repair knockouts and identifies the respective genetic defect in cell lines. In 68 patients, we detect immunodeficiencies and DNA repair defects with high accuracy (area under the curve 0.99 and 0.84, respectively), and identify previously uncharacterized patient groups as well as patient-specific CSR junction signatures. With SWIBRID, we seek to advance the identification of pathogenic defects, support early diagnosis, and address molecular heterogeneity that drives variable clinical outcomes. Class switch recombination (CSR) is a process contributing to antibody diversity and generating characteristic genomic junctions that reflect DNA double-strand break repair efficiency. Here authors present a platform to profile the repertoire of CSR junction patterns; long-read-sequenced PCR products from blood-derived lymphocytes are analysed by a dedicated bioinformatic pipeline to accurately detect immunodeficiencies and DNA repair defects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ba96a13f0bc51cc221f1e3cd935284f9970d0b5" target='_blank'>
              Recombination junctions from antibody isotype switching classify immune and DNA repair dysfunction
              </a>
            </td>
          <td>
            Clara Vázquez García, Benedikt Obermayer, Baerbel Keller, Mikhail Lebedin, C. Ratswohl, Hassan Abolhassani, Antonia Busse, M. Di Virgilio, Stephan Mathas, Dorothee Speiser, D. Beule, Q. Pan-Hammarström, K. Warnatz, K. de la Rosa
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7640f6b330e4e8f53761137586d430feb56fe8ed" target='_blank'>
              Synthetic Data to Explore Transcriptional Regulation of Differentially Expressed Genes in Ovarian Cancer
              </a>
            </td>
          <td>
            Zhang Shuo, Kumar Selvarajoo
          </td>
          <td>2026-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Mutations in the global transcriptional activator EP300/KAT3B are being reported in aggressive malignancies. However, the mechanistic contribution of EP300 dysregulation to cancer is currently unknown. While EP300 has been implicated in regulating cell cycle and DNA replication, the role of EP300 in maintaining replication fork integrity has not been studied. Here, using EP300-mutated adult T-cell leukemia/lymphoma cells and an EP300-selective degrader, we reveal that EP300 loss leads to pronounced dysregulations in DNA replication dynamics and persistent genomic instability. Aberrant DNA replication in EP300-mutated cells is characterized by elevated replication origin firing due to replisome pausing. EP300 deficiency results in a prominent defect in fork protection resulting in the accumulation of single-stranded DNA gaps. Importantly, we find that the loss of EP300 results in decreased expression of BRCA2 protein leading to sensitivity to treatments that are cytotoxic to BRCA-deficient cancers. Overall, we demonstrate that EP300-mutated cells recapitulate features of BRCA-deficient cancers. Loss-of-function mutations in EP300/KAT3B and CBP/KAT3A have been implicated in the pathogenesis of cancer. Here, the authors reveal that the EP300 protein has a role in mediating replication fork protection at sites of replication stalling and show that EP300-mutated cells recapitulate features of BRCA-deficient cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c1df38e22e1ee22880d25154078c1bc6f87527f" target='_blank'>
              EP300 deficiency leads to chronic replication stress mediated by defective replication fork protection
              </a>
            </td>
          <td>
            Angelica Barreto-Galvez, Mrunmai Niljikar, J. Gagliardi, Carolina Plasencia Guzman, Ranran Zhang, Vasudha Kumar, Aastha Juwarwala, Archana Pradeep, Ankit Saxena, Cristina Montagna, Priya Mittal, Jeannine Gerhardt, Bing Xia, Jian Cao, Keisuke Kataoka, A. Durbin, Jun Qi, B. H. Ye, Advaitha Madireddy
          </td>
          <td>2025-12-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Abstract The precise regulation of gene transcription relies on promoters, and the selection of specific promoters for a particular gene is a key determinant of transcript diversity. However, the regulatory mechanisms governing promoter selection are not fully understood. G-quadruplexes (G4s) are unique DNA noncanonical secondary structures that have emerged as important regulators of gene expression. In this study, we systematically analyzed the relationship between G4 structures and alternative promoters (APs) in two cancer cell lines, K562 and HepG2, by integrating native elongating transcript-cap analysis of gene expression and G4 ChIP-seq datasets. We identified 573 differentially utilized APs (|fold change| > 2, false discovery rate < 0.05), 26% of which being associated with G4 structures within 100 base pairs. Notably, G4-associated promoters predominantly exhibited increased activity, suggesting that G4s generally promote AP selection. Furthermore, treatment with G4 ligands induced the generation of APs, suggesting that the stabilization of G4 structures may modulate AP usage. Collectively, these findings provide new insights into the G4-based mechanisms that regulate transcript isoform diversity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71f9b0a27b21c75155bc9ac383f5f6932eb79428" target='_blank'>
              G-quadruplex structures as modulators of alternative promoter usage
              </a>
            </td>
          <td>
            Rongxin Zhang, Jean-Louis Mergny
          </td>
          <td>2025-12-01</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed1248ad827da125e00e0f24cf372ada5d8bfee2" target='_blank'>
              DNA replication errors drive genome-wide small inverted triplication dynamics
              </a>
            </td>
          <td>
            Yi Lei, Yu Zhou, Haitao Sun, Hang Yuan, Xinyu Pei, Jessica D. Hess, Yao Yan, Zunsong Hu, Mian Zhou, Zhaohui Gu, Li Zheng, Xiwei Wu, Binghui Shen
          </td>
          <td>2025-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Homologous recombination deficiency (HRD) impacts cancer treatment strategies, particularly effective utilization of PARP inhibitors. However, the variability of different HRD assays has hampered the selection of oncology patients who may benefit from these therapies. Our study aims to use the whole genome landscape to better define HRD in a pan-cancer cohort. We employed a whole genome sequencing HRD classifier that includes genome-wide signatures associated with HRD to analyze 580 tumor/normal paired samples. The HRD phenotype was correlated with genomic variants in BRCA1/2 and other homologous recombination repair genes. In this paper we show that the HRD phenotype is identified in various cancers including breast (21%), pancreaticobiliary (20%), gynecological (17%), prostate (9%), upper gastrointestinal (GI) (2%), and other cancers (1%). HRD cases are not confined to BRCA1/2 mutations; 24% of HRD cases are BRCA1/2 wild-type. A diverse range of gene alterations involved in HRD are elucidated, including biallelic mutations in FANCF, XRCC2, and FANCC, and deleterious structural variants. In a subset of cases, the whole genome sequencing-based classifier offers more insights and a better correlation to treatment response when compared to other assays. Although HRD is a biomarker used to determine which cancer patients would benefit from PARP inhibitors, a lack of harmonization of tests to determine HRD status makes it challenging to interpret their results. Our study highlights the use of comprehensive whole genome sequencing analysis to better predict HRD and elucidates genomic mechanisms associated with this phenotype. Homologous recombination deficiency is a condition in which a cancer cell cannot repair certain types of DNA damage. It causes genetic instability and is often due to changes in parts of the DNA called genes, such as BRCA1 and BRCA2. Cancers with this deficiency can be more readily killed by certain drugs that prevent DNA repair. Some of these drugs are approved for the treatment of several types of cancer, including ovarian, breast, pancreatic, and prostate cancers. To better identify tumors with this deficiency, we characterize the whole genome of cancer samples. We find that a comprehensive analysis of the entire genome improves the detection of homologous recombination deficiency. This type of analysis may provide a more accurate way to guide treatment decisions for people with cancer. Assaad, Hadi and Levine et al. develop a whole-genome sequencing classifier to improve the detection of homologous recombination deficiency (HRD) across a pan cancer cohort. The classifier detects HRD beyond BRCA1/2 mutations, reveals HRD-related genomic events, and correlates with treatment response in a subset of patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/108bfd65e5fc2e36aa5d310bd5ebd99b171e9621" target='_blank'>
              Whole genome sequencing approach to assess homologous recombination deficiency in a pan-cancer cohort
              </a>
            </td>
          <td>
            Majd Al Assaad, Kevin Hadi, Max F. Levine, Daniela Guevara, Minal Patel, Marvel Tranquille, Abigail King, John Otilano, Alissa Semaan, G. Gundem, J. Medina-Martinez, M. Sigouros, J. Manohar, Hui-Hsuan Kuo, D. C. Wilkes, Eleni Andreopoulou, E. Chapman-Davis, S. Tagawa, A. Sboner, Allyson J. Ocean, , E. Papaemmanuil, Cora N. Sternberg, Kevin M. Holcomb, D. Nanus, O. Elemento, Juan Miguel Mosquera
          </td>
          <td>2026-01-12</td>
          <td>Communications Medicine</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="Cis-regulatory elements are specific DNA sequences that control gene expression in a spatiotemporal manner, and variation within these elements represents a major source of phenotypic diversity and evolutionary innovation. Nevertheless, how regulatory elements evolve and shape gene expression remains poorly understood, particularly in plants. The well-resolved phylogeny of allopolyploid peanut (Arachis hypogaea) and its diploid progenitors, A. duranensis and A. ipaensis, provides an ideal system to investigate the regulatory evolution at a lineage-specific level. By integrating comparative analyses of sequence similarity, chromatin accessibility, histone modifications, conserved noncoding sequences (CNSs), and gene expression, we reconstructed the evolutionary trajectories of Accessible Chromatin Regions (ACRs), where regulatory elements typically reside, and revealed their distinct contributions to homeolog expression bias, unequal expressions between homeologs, in a polyploid genome. Most ACRs exhibited high sequence similarity, comparable chromatin accessibility, and conserved states for H3K4me3, H3K56ac, and H3K36me3, indicating regulatory stability after hybridization and polyploidization. However, a subset of novel ACRs emerged de novo or through sequence mutations in preexisting ACRs, arising at different rates and evolutionary stages. Notably, even highly sequence-conserved ACRs exhibited substantial variation in chromatin accessibility variation, consistent with CNS composition differences and minor sequence variation, although causal relationships remain to be demonstrated. Our analyses further revealed a complex spectrum of CNS dynamics across the diploid-polyploid framework. Overall, our study provides empirical insights into the fine-scale evolution of plant regulatory landscapes and complements previous large-scale comparisons across distant lineages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d50bb9e4155c441955e8609994d5be654a1b395" target='_blank'>
              Lineage-specific evolution of regulatory landscapes in polyploid plant and its diploid progenitors
              </a>
            </td>
          <td>
            Xiang Li, Xuan Zhang, Ziliang Luo, Hao Zhang, J. P. Mendieta, Robert J. Schmitz
          </td>
          <td>2025-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="This study aimed to investigate the molecular heterogeneity, accurately evaluate the prognosis, and identify those patients who would benefit most from immunotherapy in the hypermutated colorectal cancer (CRC) cohort. The unsupervised clustering analysis of CRC somatic mutation data from the Cancer Genome Atlas database was performed by combining tumor mutational burden (TMB) and indel burden to identify the subtypes within the hypermutated CRC cohort. Then, the somatic mutations, transcriptome, and gene mutations in the DNA damage repair signaling pathway of these subtypes were compared to obtain the corresponding molecular features. The proportion of 22 immune cells infiltrated in the samples was calculated using the CIBERSORT algorithm, and immunotherapeutic sensitivity was predicted for the patients using immunophenoscores (IPS). Five distinct clusters were identified from the hypermutated CRC sample, each exhibiting different characteristics regarding somatic mutations, gene expression, immune infiltration, and prognosis. The proportion of M2 macrophage infiltrated in Cluster 6 was significantly higher than in Cluster 1 (P = .018) and Cluster 3 (P = .003). The proportion of infiltrating M1 macrophages was significantly higher in Cluster 3 than in Cluster 1 (P = .026) and Cluster 6 (P = .016). Cluster 1 had a longer OS time compared to Cluster 3 (P = .015) and Cluster 6 (P = .005). In all 4 IPS types, Cluster 3 demonstrated higher IPS than Cluster 6 (P < .05), Cluster 2 demonstrated lower IPS than Cluster 5 (P < .05). In the IPS-CTLA4 type, the IPS of Cluster 1 was significantly higher than that of Cluster 6 (P = .038). In the non-POLE mutant cohort, the patients with the lowest TMB and indel burden had longer overall survival. The patients with higher levels of M1 macrophage infiltration showed higher sensitivity to anti-PD-1/PD-L1 immunotherapy or combined anti-PD-1/PD-L1 and anti-CTLA4 immunotherapy. In the POLE mutant cohort, the patients from the low TMB cohort may be better candidates for immunotherapy than those from the higher TMB cohort.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ec135f15fc8160351aa36efc3ceee1240577672" target='_blank'>
              Comprehensive analysis reveals the molecular heterogeneity and the differences in prognosis and immunotherapy sensitivity in hypermutated colorectal cancer
              </a>
            </td>
          <td>
            Gaobo Bai, Xiaojun Yang, Liang Xiao, Chunhui Xiong
          </td>
          <td>2026-01-09</td>
          <td>Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Large-scale genomic rearrangements in cancer cells can lead to the creation of fusion genes which alter the expression or function of the partner genes. Some fusion genes act as tumour drivers and the use of kinase inhibitors in patients with specific fusions have led to breakthroughs in cancer therapy. The large clinical and scientific interest in fusions has led to the development of software that use RNA sequencing data to identify fusion transcripts. Unfortunately, fusion transcript callers output many predictions which lack underlying genomic rearrangements and large datasets with validated fusions are scarce. This paper and the accompanying Fusions4U web application present a resource of validated and annotated gene fusions for 328 cell lines from the Cancer Cell Line Encyclopedia. Predicted fusion transcripts from Arriba and STAR-Fusion were analysed with our published validation pipeline that uses matched whole-genome sequencing data to identify discordantly mapped read pairs and candidate genomic breakpoints that support genuine fusion events. This resulted in 8,753 and 2,244 validated fusion transcripts for Arriba and STAR-Fusion predictions, respectively, with 1,596 fusions common to both. Additional layers of annotation include alternative splicing of fusion transcripts, kinases, microRNA host genes, genes in the COSMIC Cancer Gene Census, as well as known fusion gene pairs from the Mitelman Database of Chromosome Aberrations and Gene Fusions in Cancer and the TumorFusions dataset. We furthermore analysed information about fusion genes together with cell line data from the PRISM drug repurposing screening as an example of how this dataset can be used. This resource can be used to design experiments for functional studies and drug development, alone or in combination with publicly available information for the Cancer Cell Line Encyclopedia cell lines. The large collection of validated fusion transcripts with candidate genomic breakpoints can also be used in development and evaluation of bioinformatic tools for fusion transcript detection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1751293449b5956386c8f2f0dbda4aebaafaddec" target='_blank'>
              Fusions4U: a resource of validated and annotated gene fusions in 328 cancer cell lines
              </a>
            </td>
          <td>
            Arianna Alamshahi, Helena Persson
          </td>
          <td>2025-12-01</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Colorectal cancer (CRC) has traditionally been thought to develop through stepwise mutation of the APC tumour suppressor and other driver genes, coupled with expansion of positively selected clones. However, recent publications show that many premalignant lesions comprise multiple clones expressing different mutant APC proteins1, 2, 3–4. Here, by mediating transformation on different mouse backgrounds containing mutations in Kras or other common CRC driver genes, we establish that the presence of diverse priming events in the normal mouse intestinal epithelium can change the transformation and clonal-selection landscape, permitting the fixation of strong driver mutations in Apc and Ctnnb1 that are otherwise lost due to negative selection. These findings, combined with our demonstration of mutational patterns consistent with similar priming events in human CRC, suggest that the order in which driver mutations occur in intestinal epithelium can determine whether clones are positively or negatively selected and can shape subsequent tumour development. The order in which driver mutations of colorectal cancer occur in intestinal epithelium can determine whether clones are positively or negatively selected and can shape subsequent tumour development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85284f318689c76d83e7b6a1af9509e1766375cf" target='_blank'>
              Decay of driver mutations shapes the landscape of intestinal transformation
              </a>
            </td>
          <td>
            Filipe C. Lourenço, Iannish D. Sadien, K. Wong, Sam Adler, Ashley Sawle, Leonor Schubert Santana, Lee Hazelwood, Giada Giavara, Anna M Nicholson, Matthew Eldridge, N. Maka, Gerard Lynch, Stephen T McSorley, J. Edwards, Richard Kemp, D. Adams, Doug Winton
          </td>
          <td>2025-12-03</td>
          <td>Nature</td>
          <td>2</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f729ce791f41a9454d4b02d52f02d2067f40e331" target='_blank'>
              The role of chimeric FAM72B transcripts generated by intergenic mRNA trans-splicing in breast cancer
              </a>
            </td>
          <td>
            Gangotri Patra, Pok-Son Kim, Arne Kutzner, Klaus Heese
          </td>
          <td>2025-12-01</td>
          <td>Functional & Integrative Genomics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The present study aimed to systematically investigate the biological role of the long non-coding RNA (lncRNA) LINC00473 in the pathogenesis and progression of ovarian cancer and to explore its potential as a novel therapeutic target for clinical translation. LINC00473 expression was specifically silenced in SK-OV-3 and A2780 ovarian cancer cell lines using small interfering RNA technology. Enhanced Cell Counting Kit-8 and Transwell invasion assays were performed to evaluate the regulatory effects of LINC00473 on malignant phenotypes (proliferation, migration, invasion and apoptosis) and chemosensitivity to cisplatin (CDDP). Western blotting was utilized to detect changes in the expression levels of apoptosis-related proteins [poly (adenosine diphosphate-ribose) polymerase (PARP) and caspase-3] and stemness/drug resistance-associated proteins [Sox2 and Yes-associated protein 1 (YAP1)]. Knockdown of LINC00473 significantly inhibited ovarian cancer cell proliferation (SK-OV-3, 23.1%, P<0.001; A2780, 41.5%, P<0.001) and induced apoptosis (apoptosis rate 18.0–25.2%; P<0.001), while reducing migration and invasion by 30.7–55.4% and 31.9–54.5% (both P<0.001), respectively, accompanied by pseudopodia retraction and cellular rounding. At the molecular level, LINC00473 knockdown upregulated PARP/caspase-3 expression (1.7- and 2.3-fold) and downregulated Sox2/YAP1 (0.5- and 0.6-fold), with A2780 cells exhibiting 25% higher sensitivity to intervention compared with SK-OV-3 cells. Combined with CDDP treatment, the proliferation rate of A2780 further decreased to 35.5% (P<0.001). These findings indicate that LINC00473 may promote ovarian cancer progression by suppressing apoptosis and activating oncogenic pathways (Sox2/YAP1). Furthermore, silencing LINC00473 synergistically enhanced CDDP efficacy, particularly in A2780 cells exhibiting heightened sensitivity. These findings suggest that targeting LINC00473 may represent a novel therapeutic strategy for ovarian cancer; however further exploration of its molecular network and in vivo validation are warranted in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6523ad551e86aa69b080448d3c2def363b8e6a9d" target='_blank'>
              LINC00473 modulates protein expression to promote ovarian cancer progression and overcome cisplatin resistance
              </a>
            </td>
          <td>
            Cairong Zhang, Jie Ma, Wenling Wang, Min Guo, Cuiliu Han, Madinamu Sadike, Kaichun Zhu
          </td>
          <td>2026-01-12</td>
          <td>Oncology Letters</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Transposable elements (TEs) make up about half of the human and mouse genomes and play important regulatory roles in immune responses. However, the cis-regulatory contribution of TEs to immune cell development is less characterized. Here, we analyzed hundreds of chromatin accessibility, gene expression, transcription factor occupancy and DNA-DNA contact datasets in diverse mouse and human immune cells. We identified two rodent-specific TE subfamilies, ORR1E and ORR1D2 (collectively, ODE) that have transformed into cell type-specific enhancers across the mouse immune system. ODE loci acquired mutations post-insertion that enable differential binding of lineage-specifying transcription factors. ODEs show evidence of evolutionary sequence constraint and contact promoters of hundreds of genes in immune cells. ODE-targeted genes show cell type-specific and mouse-specific increases in expression compared to concordant human cell types. Thus, a single TE family can undergo functionalization after its genomic spread to generate batteries of cell-specific enhancers supporting a complex developmental cascade.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e1e19afca4d729187327d1a4fb4af7e47aba33d" target='_blank'>
              Pervasive cis-regulatory co-option of a transposable element family reinforces cell identity across the mouse immune system
              </a>
            </td>
          <td>
            Jason D. Chobirko, Cédric Feschotte, Andrew Grimson
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Importance of Study Protein phosphatase magnesium-dependent 1D (PPM1D) is frequently mutated in diffuse midline gliomas (DMGs). DMGs are rare pediatric brain tumors with limited treatment options. Due to the cancer’s rapid progression, patients usually survive 12-24 months after diagnosis. This underscores the critical need to better understand the molecular mechanisms driving DMGs. This study describes a novel mouse model that provides a powerful platform to investigate PPM1D-driven tumor biology and offers mechanistic insights into disease development and progression. Furthermore, it serves as a valuable preclinical system for evaluating therapeutic strategies and identifying translational opportunities to target Ppm1d-mutant tumors. Background Diffuse midline gliomas (DMGs) are incurable brain tumors with limited treatment options. Approximately 20% of DMGs harbor truncating mutations in exon 6 of phosphatase PPM1D, which stabilize the protein and deregulate p53 signaling. However, the consequences of these mutations for tumor initiation, progression, and therapy remain unclear. Methods We developed a conditional Ppm1d-loxP-exon6-loxP-exon6-E518X-tag mouse allele (Ppm1d-flex-6) that enables lineage-, spatial-, and temporal-specific expression of a DMG-derived truncated Ppm1d protein from its endogenous locus in the presence of Cre-recombinase. Ubiquitous activation of mutant Ppm1d was modeled using the Meox2-Cre driver, and primary gliomas were modeled using the RCAS/tv-a system to introduce Cre and PDGFB co-drivers into Nestin-positive neural stem cells. Complementary studies were performed in mouse embryonic fibroblasts (MEFs) expressing truncated Ppm1d following Cre recombination. Results While Meox2-Cre-driven ubiquitous recombination of Ppm1d-flex-6 produced muted phenotypes, Ppm1d-flex-6 recombination in Nestin+ neural stem cells accelerated gliomagenesis. Its oncogenic effect was weaker than complete p53 loss, and it did not accelerate tumorigenesis further in p53-null tumors. Single-cell RNA-sequencing revealed that Ppm1d-flex-6 gliomas adopt more progenitor-like transcriptional states and upregulate p53- and cell cycle associated pathways. In MEFs, Ppm1d-flex-6 enhanced proliferation and shifted transcriptomic programs toward MAPK and PI3K-Akt signaling, while impairing DNA damage responses, including reduced γ-H2AX induction after irradiation. These defects sensitized cells to radiation and decreased clonogenic survival after ionizing radiation and PARP inhibition. Conclusions Ppm1d mutations confer intermediate suppression of the p53 pathway, consistent with the clinical features of PPM1D-mutant DMGs and are associated with radiosensitivity and PARP inhibitor vulnerability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b526ced19406db51fd6c39018ffced7037a7861b" target='_blank'>
              Oncogenic Ppm1d mutations deregulate the p53 pathway in primary mouse gliomas
              </a>
            </td>
          <td>
            Vennesa Valentine, Abigail J. Groth, Joshua Tolliver, Avneesh Saravanapavan, Spencer M. Maingi, Loren B. Weidenhammer, Nerissa T. Williams, Lixia Luo, Matthew S. Waitkus, Sheeba Jacob, Zachary J. Reitman
          </td>
          <td>2025-11-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background KRAS is one of the most frequently mutated genes in colorectal cancer (CRC) and plays a crucial role in tumorigenesis, progression, immune evasion, and treatment resistance. The pronounced heterogeneity within KRAS-mutant CRC highlights the urgent need for more precise and personalized therapeutic approaches. Methods To investigate this heterogeneity, we employed single-cell RNA sequencing and spatial transcriptomics to comprehensively characterize the tumor microenvironment of KRAS-mutant CRC. Data preprocessing and clustering were performed using Scanpy. Spatial cell-type deconvolution was conducted via Cell2location, whereas intercellular communication and spatial dependencies were analyzed using CellChat, MISTy, and stLearn. Results Our analyses revealed that KRAS-mutant tumor epithelial cells recruit Mono_S100A8 monocytes via the MDK_SDC4 signaling axis. Concurrently, surrounding Fib_CTHRC1 fibroblasts secrete collagen, which interacts with integrin receptors on KRAS-mutant epithelial cells and contributes to the exclusion of lymphocyte infiltration. Conclusion These cellular components collaboratively established an immunosuppressive spatial niche. These findings offer novel theoretical insights and potential targets for the development of immunoregulatory strategies tailored to KRAS-mutant CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/059c9dfc013c2c370152c3fa0d39b4efc600e01e" target='_blank'>
              Single-cell and spatial transcriptome profiling identifies the immunosuppressive spatial niche in KRAS-mutant colorectal cancer
              </a>
            </td>
          <td>
            Sheng‐Li Yang, Chao Gu, Xinsheng Miao, Hao Zuo, Wei Xu, Yan Zhang, Wei Tang, Jianhua Zhu, Zheng Yuan, Xinhua Gu, Chenyi Zhong, Yueming Sun, Jiahui Zhou
          </td>
          <td>2025-12-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Simple Summary Structural variations in cancer predisposition genes significantly contribute to the pool of pathogenic variants underlying heritable cancer susceptibility. In this report, we describe and functionally characterize a novel transposon-mediated germline pathogenic insertion identified within the coding region of the BRCA1 gene. The inserted sequence comprises the entire processed transcript of RPL18A, a ribosomal protein-coding gene, and may have been generated through a target-primed reverse transcription event. Robust molecular testing confirmed the heritability of the variant and provided evidence of its correlation with the observed phenotype. This is the first documented case of a germline cancer susceptibility variant arising through this unique mechanism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eab6faf5664bd66e2e317ebc80d8ac230de46d4a" target='_blank'>
              Processed Transcript Insertion as a Novel Germline Mutational Mechanism in BRCA1-Associated Hereditary Breast Cancer
              </a>
            </td>
          <td>
            A. Bozsik, H. Butz, V. Grolmusz, Petra Nagy, T. Pócza, Erika Tóth, E. Csernák, Attila Patócs, János Papp
          </td>
          <td>2025-12-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Cancer cells rewire their metabolism to sustain the high proliferative rate. Metabolism is therefore a common vulnerability of cancer cells, successfully exploited for therapeutic purposes. Intrinsic tumor characteristics and adaptive responses of cancer cells can however reduce the short and long-term efficacy of such a strategy. Understanding the determinants of therapy response and the mechanisms of chemoresistance is crucial to maximize therapy efficacy. In cancer, lysosomes undergo massive changes in their localization, size, and composition that support tumor progression. Additionally, lysosomes are one of the crucial drivers of chemoresistance via the drug sequestration or by facilitating adaptations to stress conditions. In the last decades, several reports have shown that lysosomal membrane proteins, such as the lysosome-associated membrane proteins 1 and 2 (LAMP1 and LAMP2), are deregulated in different cancer types and their expression has been correlated to drug efficacy. We performed an in silico gene essentiality and drug sensitivity screenings, revealing that LAMP2 expression is one of the determinants of resistance to inhibitors of de novo purine synthesis. In vitro experiments confirmed the in silico data and also showed that purine synthesis inhibitors trigger a ROS- and transcriptional-dependent increase of LAMP2. Our results identify the upregulation of LAMP2 expression as an adaptive response to purine synthesis inhibition to preserve cell viability and, in those tumors showing high LAMP2 levels, could also be an indicator of intrinsic resistance to these drugs that may be taken into consideration during the selection of the most appropriate therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b41a50acd50b88a1b8f2629c5c9719051133237" target='_blank'>
              The inhibition of de novo purine synthesis increases LAMP2 expression to preserve cell viability
              </a>
            </td>
          <td>
            Angela De Cristofaro, Serena Castelli, Federica Felice, M. Ciriolo, Enrico Desideri
          </td>
          <td>2025-12-11</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9932f70712f8bef30c70a5b989c4e744cf379f36" target='_blank'>
              Harnessing transcriptomics for discovery of natural products to overcome acquired cancer resistance.
              </a>
            </td>
          <td>
            Heerim Yeo, Sang-Yun Kim, Sang-Min Park
          </td>
          <td>2025-12-08</td>
          <td>Archives of pharmacal research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Abstract Cancer immune evasion is orchestrated by tumor‐intrinsic molecular constraints that remain incompletely defined. Here, we performed an in vivo genome‐wide clustered regularly interspaced short palindromic repeats (CRISPR) loss‐of‐function screen to catalogue gene regulatory determinants of immune evasion in cancer cells. We identify C9ORF50 as a novel splicing regulator whose inhibition profoundly sensitizes cancer to immune surveillance. Integrated multi‐omics profiling reveals this intrinsically disordered protein exhibits liquid–liquid phase separation properties and forms nuclear condensates that colocalize with spliceosome components. Genetic ablation correlates with intron retention in multiple spliceosome components and cytoplasmic accumulation of double‐stranded RNA, which is associated with type I interferon activation and enhances chemokine‐mediated T cell recruitment. As a result, C9ORF50 inhibition amplifies tumor cell immunogenicity, enhancing T cell infiltration in poorly infiltrated tumors. Clinically, elevated C9ORF50 expression correlates with poor survival and diminished lymphoid infiltration across malignancies. Therapeutic targeting of C9ORF50 using RNA interference enhances T cell infiltration and suppresses tumor growth. Our work identifies C9ORF50 as a candidate therapeutic target that modulates RNA splicing and tumor immunity, suggesting splicing regulation as a potential strategy to enhance immunotherapy responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4776a77ccb7e8b56c7236aff620b0308713e23cc" target='_blank'>
              Genome‐wide CRISPR screen reveals an uncharacterized spliceosome regulator as new candidate immunotherapy target
              </a>
            </td>
          <td>
            Tong Shao, Chuanyang Liu, Jingyu Kuang, Sisi Xie, Ying Qu, Yingying Li, Lulu Zhang, Fangzhou Liu, Yanhua Qi, Tao Hou, Ming Li, Sujuan Zhang, Yu Liu, Zhixiang Yuan, Jiali Liu, Yanming Hu, Jingyang Wang, C. Song, Shaowei Zhang, Lingyun Zhu, Jianzhong Shao, Aifu Lin, Wenjun Mao, Guangchuan Wang, Lv-yun Zhu
          </td>
          <td>2025-11-28</td>
          <td>iMeta</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Ovarian cancer (OvCa) remains one of the most lethal gynecological malignancies, primarily due to late-stage diagnosis, chemoresistance, and substantial molecular heterogeneity. Emerging evidence highlights the pivotal roles of microRNAs (miRNAs) and circular RNAs (circRNAs) in tumor progression. Among them, miR-7-5p has been implicated in tumor suppression and the regulation of metabolic pathways. However, its functional interaction with circ_0001946 in OvCa has yet to be elucidated. This study aimed to investigate the interplay between miR-7-5p and circ_0001946 in OvCa, with a particular focus on their regulatory roles in apoptosis, proliferation, tumorigenesis, and glucose metabolism. Using integrated bioinformatics analyses of TCGA and GEO datasets, we identified glucose metabolism-related molecular subtypes. Functional enrichment, immune landscape profiling, and LASSO regression were employed to validate these subtypes and develop prognostic risk models. A series of in vitro and in vivo experiments–including qPCR, CCK-8 assay, flow cytometry, colony formation, dual-luciferase reporter assay, and xenograft tumor models in nude mice–were conducted to evaluate the expression and functional effects of miR-7-5p and circ_0001946. Two molecular subtypes, Subtype-A and Subtype-B, with distinct glucose metabolism and immune profiles were identified. Subtype-A correlated with advanced-stage disease and chemoresistance, while Subtype-B exhibited better prognosis, higher immune infiltration, and activation of metabolic pathways. Circ_0001946 was upregulated in OvCa tissues and cell lines and acted as an oncogene by sponging miR-7-5p. Functional assays revealed that downregulation of miR-7-5p or overexpression of circ_0001946 promoted cell proliferation, inhibited apoptosis, and accelerated tumor growth. Dual-luciferase reporter assays confirmed a direct interaction between miR-7-5p and circ_0001946. Clinical tissue analysis further showed elevated miR-7-5p expression in para-cancerous tissues. The circ_0001946/miR-7-5p regulatory axis contributes to OvCa progression by suppressing apoptosis and promoting cellular proliferation. Targeting this axis presents a promising therapeutic avenue. Additionally, molecular subtyping based on glucose metabolism-related genes offers valuable prognostic information and supports the advancement of personalized treatment strategies in OvCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83e2b269ff28488a64b585e398a19f1eba53bce8" target='_blank'>
              Integrated molecular subtyping and functional analysis of circ_0001946/miR-7-5p axis reveals a glucose metabolism-linked regulatory mechanism in ovarian cancer
              </a>
            </td>
          <td>
            Yanlei Dong, Guiyu Zhang
          </td>
          <td>2025-12-15</td>
          <td>World Journal of Surgical Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Introduction Cancer origin patterns significantly influence cancer phenotypes and hallmark characteristics. Method In the present review, four distinct carcinogenesis trajectories that contribute to malignant transformation: mutator phenotype, chromosomal instability, dysmetabolism, and stemness, are explored. Results and Discussion In the mutator phenotype trajectory, deficiencies in DNA repair or synthesis systems lead to hypermutation and accumulation of oncogenic alterations. The chromosomal instability trajectory involves aneuploidy-induced copy number alterations in oncogenes and tumor suppressor genes. Dysmetabolic carcinogenesis is driven by the accumulation of oncometabolites due to alterations in metabolic genes. The stemness trajectory refers to the malignant transformation of cells possessing stem-like properties under oncogenic stimuli. Each trajectory independently promotes carcinogenesis and endows cancer cells with distinct characteristics. Notably, the primary oncogenic drivers in each trajectory can self-reinforce and form spontaneous-reinforcing loops that amplify oncogenic signals. Although crosstalk exists among trajectories, evidence suggests they are mutually exclusive during cancer origin. Therefore, targeting specific carcinogenesis trajectories and disrupting the self-reinforcing oncogenic loops may represent novel therapeutic strategies. Understanding carcinogenesis trajectories provides a framework for future cancer research and treatment approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c92c0ec788118fc8fac54bdd52f8df0eef75667c" target='_blank'>
              Carcinogenesis trajectories
              </a>
            </td>
          <td>
            Rui Wang, Zhaopeng Yan
          </td>
          <td>2025-12-09</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d72b955b3aea1e2e08b44ecc1d42ae2576e649d" target='_blank'>
              Ionizing Radiation Shapes Genome Evolution Through Nuclear Abnormalities That Trigger Delayed Proliferative Death
              </a>
            </td>
          <td>
            Mian Zhao, Bejo Presila, Cheng-Zhong Zhang, Spektor Alexander
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="MYC-driven medulloblastoma (MB) is a highly aggressive brain tumor with poor prognosis and limited treatment options. Through CRISPR-Cas9 screening, we identify the Mediator-associated kinase CDK8 as a critical regulator of MYC-driven MB. Both genetic loss and pharmacological inhibition of CDK8 impair MB tumor growth. Moreover, we find that CDK8 cooperates with MYC to sustain the MYC-mediated translational program, as CDK8 depletion induces pronounced transcriptional changes in translation-associated gene sets, reduces ribosome biogenesis, and impairs protein synthesis. Mechanistically, CDK8 regulates the occupancy of RNA polymerase II at specific chromatin loci, facilitating epigenetic alterations that promote the transcription of ribosomal genes. Furthermore, combined inhibition of CDK8 and mTOR synergistically enhances therapeutic efficacy in vivo, leading to more pronounced tumor growth suppression. Overall, our findings establish a functional link between CDK8-mediated transcriptional regulation and mRNA translation, suggesting a promising therapeutic approach targeting protein synthesis for MYC-driven MB. MYC-driven medulloblastoma is an aggressive pediatric tumor with limited treatment options. Here, the authors show that CDK8 regulates ribosome biogenesis and that combined inhibition of CDK8 and mTOR demonstrates therapeutic efficacy in mouse models of this cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57987b24c602508744e2198d54727be7f386ebb5" target='_blank'>
              Transcriptional regulation of protein synthesis by mediator kinase represents a therapeutic vulnerability in MYC-driven medulloblastoma
              </a>
            </td>
          <td>
            Dong Wang, Caitlin Ritz, Yuhuan Luo, Ammu Suresh, Angela M Pierce, Bethany L. Veo, Breauna Brunt, Nathan A. Dahl, Natalie J Serkova, S. Venkataraman, Etienne P Danis, Kamil Kuś, Milena Mazan, T. Rzymski, Rajeev Vibhakar
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="The KEAP1/NRF2 pathway, a major regulator of the cellular oxidative stress response, is frequently activated in human cancers. Often mediated by loss-of-function mutations in KEAP1, this activation causes increased NRF2 transcriptional activity and constitutive activation of the antioxidant response. While KEAP1 mutations have been well documented in various cancers, their presence and role in thyroid carcinoma have remained largely unexplored. In this study, we sequenced pediatric thyroid tumors and analyzed publicly available datasets, identifying 81 KEAP1 mutations in tumors across a range of histologies. In these tumors, we further identified frequent biallelic loss of KEAP1 via 19p13.2 loss of heterozygosity (LOH). MAPK-activating alterations were found in a subset of KEAP1-mutant cases, but they were mutually exclusive with 19p13.2 LOH. Transcriptome analysis also revealed significant activation of the NRF2 pathway in KEAP1-mutant tumors. Four additional cases with similar transcriptional profiles but lacking mutational data were identified, likely representing putative KEAP1 mutants. Using in vitro cell line models, we then profiled the functional consequences of KEAP1 knockout in cells with and without known driver alterations. In these models, we show that KEAP1 loss leads to an NRF2-dependent upregulation of AKR1C3, GCLC, NQO1, and TXNRD1, along with increased proliferation and migration irrespective of MAPK mutational status. We also demonstrate that loss of KEAP1 reduces sensitivity of RET fusion-positive cells to selpercatinib, consistent with previous reports that these alterations promote drug resistance in other malignancies. In this study, we comprehensively profile KEAP1 mutations in thyroid tumors, showing that they are more prevalent and functionally significant than previously recognized. These findings position KEAP1 mutations as novel oncogenic variants in thyroid cancer and support the integration of KEAP1/NRF2 pathway profiling into future studies and clinical frameworks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61e3679e5f3a81da4bfabc2e262c1a4730ec5b1c" target='_blank'>
              KEAP1 mutations activate the NRF2 pathway to drive cell growth and migration, and attenuate drug response in thyroid cancer
              </a>
            </td>
          <td>
            Nicholas E. Bambach, J. Ricarte-Filho, E. R. Reichenberger, Christian Hernandez-Padilla, K. Hinkle, A. Isaza, A. J. Bauer, A. Franco
          </td>
          <td>2026-01-07</td>
          <td>Frontiers in Oncology</td>
          <td>1</td>
          <td>7</td>
        </tr>

        <tr id="Immune checkpoint inhibitors (ICI) have demonstrated prolonged efficacy in certain melanoma patients, yet a significant portion of patients do not experience clinical improvement, with the mechanisms underlying this resistance still not fully understood. Using established cell markers, we partitioned the single‐cell transcriptome into clusters, finding a notable link between NK cells and patient response to immunotherapy. We further identified four distinct subpopulations of NK cells, profiling marker gene sets and unique biological functions associated with each subpopulation. This analysis provides insights into the trajectory of NK cell development and differentiation, along with identifying the transcription factors driving these processes. The study pinpointed NK cluster 01 as pivotal in influencing patient sensitivity and prognosis during immunotherapy. Single‐cell transcriptome and spatial transcriptomics (ST) analysis revealed the proximity of NK cluster 01 cells to melanoma cells, hinting at a potential regulation of cell‐cell interaction via the IFN‐II signaling pathway network. ST analysis revealed the spatial arrangement and interaction of NK cluster 01 cells with melanoma cells. This study explores the feasibility of targeting NK cluster 01 cells with small molecule drugs via molecular docking, offering a promising approach to bolster the clinical utility of NK cell therapy. We comprehensively analyze the heterogeneity of NK cells within melanoma, elucidate the potential regulatory interactions between NK cells and other microenvironmental components, and establish a basis for the future clinical utilization of distinct NK cell subsets as therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73fbdc1aaba32b7600dd3221ae61c5e31ee96bd3" target='_blank'>
              Integrating Single‐Cell and Spatial Transcriptomics Reveals NK Cell Subpopulations Associated With Immunotherapy for Melanoma
              </a>
            </td>
          <td>
            Zhicheng Hu, Yongfei Chen, Hao Yang, Qiuming Pan, Hongrui Li, Shuting Li, Junxi Wang, Yudi Huang, Guanglong Huang, Shanqiang Qu
          </td>
          <td>2025-12-01</td>
          <td>Smart Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 EGFR-mutant non-small cell lung cancers (NSCLCs) initially respond to EGFR tyrosine kinase inhibitors (TKIs) such as osimertinib, but tumors eventually develop resistance. While genetic mutations explain many instances of resistance, an increasing body of evidence points to transcriptional and cell state changes as another route by which tumor cells evade therapy. The dynamics that shape these adaptive responses and the non-genetic mechanisms involved in facilitating them remain incompletely understood. We investigated transcriptional programs activated in EGFR-mutant LUAD cells following TKI treatment and observed robust induction of SOX2, a transcription factor involved in promoting stemness and lineage plasticity, within a day of drug exposure. In parallel, we investigated the role of APOBEC cytidine deaminases (APOBEC3A and APOBEC3B), which have been linked to mutagenic processes in cancer. While both enzymes have been implicated in drug resistance, the individual contributions of the two enzymes remain ill studied. Using competition assays with single gene knockouts of either APOBEC3A or APOBEC3B, we observed that loss of APOBEC3A leads to a more profound reduction in the survival of drug-tolerant cells following TKI therapy. These results highlight a functional role for APOBEC activity in sustaining early drug-tolerant populations under therapeutic pressure. Over the course of long-term osimertinib treatment, we also identified distinct subpopulations of cells with elevated ΔNp63, an isoform of TP63 that serves as a marker of squamous cell trans-differentiation. These ΔNp63-high cells emerge during the acquisition of drug resistance, and evidence suggests that this subpopulation may rely on CDK4, a cell cycle-regulating kinase, to sustain growth under therapeutic pressure. Pharmacological inhibition of CDK4 with palbociclib reduced the survival of ΔNp63-high cells, highlighting a potential therapeutic vulnerability in this proliferative subpopulation. Together, our findings demonstrate that adaptive resistance in EGFR-mutant NSCLC is driven by several non-genetic resistance mechanisms, including SOX2 induction, ΔNp63-driven squamous cell transdifferentiation, and APOBEC-mediated promotion of drug-tolerant persister cells.



 Sharan Srinivasan, Jessica N. Becerra, Nina M. Garcia, James V. Alvarez. SOX2 induction and ΔNp63-high subpopulations mark adaptive responses to EGFR inhibition in lung adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B019.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a218a816c806ca36ea70bf5b2dadd14af3bb49f" target='_blank'>
              Abstract B019: SOX2 induction and ΔNp63-high subpopulations mark adaptive responses to EGFR inhibition in lung adenocarcinoma
              </a>
            </td>
          <td>
            Sharan Srinivasan, Jessica N Becerra, N. M. Garcia, James V. Alvarez
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Rare tumor cell populations can evade cancer therapies, ultimately driving drug resistance and relapse. These resilient populations, termed “persisters,” display distinct molecular states compared to the bulk tumor. While some persisters acquire genetic mutations that confer growth advantage, others undergo non-genetic remodeling that enhances cellular plasticity and fitness. Previous studies suggest that stochastic gene activation promotes persister survival (Shaffer et al., Nature 2017; Goyal et al., Nature 2023), yet the master regulators of this plasticity remain undefined. Using DNA barcoding, we demonstrate that pre-determined, non-stochastic persisters exist in KRAS-mutant non-small cell lung cancer (NSCLC) treated with the KRAS G12C inhibitor sotorasib. These clones repopulate the tumor without harboring recurrent genetic alterations. By isolating single clones destined to persist or go extinct, and profiling their transcriptomes and chromatin accessibility, we found that persister clones activate an interferon-γ (IFNγ) transcriptional signature accompanied by increased chromatin accessibility at sites enriched for the transcription factors IRF1 and ZNF384. Functionally, ectopic ZNF384 expression converted the entire cell population into a resistant state, while IRF1 knockout prevented persister formation. Cleavage Under Targets & Release Using Nuclease (CUT&RUN) assays of ZNF384 and IRF1 revealed their binding to promoter and enhancer regions, including loci regulating IFNγ-responsive genes, thereby contributing to persister plasticity and survival. Together, our study identifies IRF1 and ZNF384 as tumor-intrinsic regulators linking IFNγ signaling to transcriptional plasticity, providing mechanistic insight into the evolution of drug tolerance and resistance in KRAS-mutant NSCLC.



 Chendi Li, Anahita Nimbalkar, Christopher J. Graser, Melissa Vieira, Mohammed U. Syed, Qian Qin, Barbara Karakyriakou, Sarah Clark, Anne Y. Saiki, Paul E. Hughes, Christopher Ott, Luca Pinello, Franziska Michor, Aaron N. Hata. Transcription factors governing the evolution of drug-tolerant persisters in KRAS-mutant non-small cell lung cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B014.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/331fb37a264b2ec60d8af95499de6058539d7fa9" target='_blank'>
              Abstract B014: Transcription factors governing the evolution of drug-tolerant persisters in KRAS-mutant non-small cell lung cancer
              </a>
            </td>
          <td>
            Chendi Li, Anahita Nimbalkar, Christopher J Graser, Melissa Vieira, M. U. Syed, Qian Qin, Barbara Karakyriakou, Sarah Clark, A. Y. Saiki, Paul E Hughes, Christopher Ott, Luca Pinello, Franziska Michor, Aaron N. Hata
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Simple Summary Cancer remains a leading cause of death worldwide, often because tumors can evade the immune system and become resistant to treatments. In this study, we investigated a protein called HMGA1, which is known to be present in high amounts in many cancers but whose broader role was not fully understood. By analyzing data from thousands of patients across numerous cancer types, we discovered that HMGA1 is frequently overactive due to specific chemical changes on its DNA. High levels of this protein were linked to poorer patient survival. We found that HMGA1 helps tumors hide from the body’s immune defenses and makes them resistant to a common class of drugs known as AKT inhibitors. We confirmed this drug resistance in breast cancer cells. Our work identifies HMGA1 as a key driver of cancer aggression and suggests that measuring its levels could help predict patient outcomes and guide more effective treatment strategies, particularly for immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3e03334f80f9abed34dd7f9721e333bfb33a958" target='_blank'>
              Promoter Hypomethylation Unleashes HMGA1 to Orchestrate Immune Evasion and Therapy Resistance Across Cancers
              </a>
            </td>
          <td>
            Iram Shahzadi, T. Ahsan, Shoaib Anwaar, Wajid Zaman, Houjun Xia
          </td>
          <td>2025-12-01</td>
          <td>Biology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b069e3e7f863bc3b24b70272c8b819a793918112" target='_blank'>
              Clastogenesis by nucleotide lesions requires the completion of two cell cycles
              </a>
            </td>
          <td>
            J. Jansen, Piya Temviriyanukul, Daniel de Groot, K. Szuhai, S. van Hees-Stuivenberg, Anastasia Tsaalbi-Shtylik, Heinz Jacobs, Niels de Wind
          </td>
          <td>2025-12-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="


 Dose–response curves (DRCs) are a standard protocol in experimental biology to assess the effect of anticancer agents in vitro. Yet, therapy itself can reshape tumor composition, driving the gradual evolution of the DRC. Monitoring such changes can reveal the emergence of resistance, as well as evolutionary trade-offs that may be exploited to enhance therapeutic efficacy. Here, we investigate the evolution of DRCs in response to three different ALK inhibitors (ALKi) in ALK-positive anaplastic large cell lymphoma (ALCL) using patient-derived cell lines. Over the course of six months, ALKi concentrations were gradually escalated, allowing ALCL cells to adapt and survive at higher drug levels. Strikingly, at the same time, these resistant populations exhibited reduced fitness in the absence of the drug, indicating an evolutionary cost. This phenomenon, known as “drug addiction” or dependence, has been previously observed in melanoma following resistance to MAPK inhibitors and suggests that treatment holidays could optimize tumor control in the long term. Guided by these findings, we developed a mathematical model to capture the temporal evolution of DRCs and to test how therapy design might exploit addiction in ALCL. Calibrated with experimental data, the model simulated tumor growth under alternative dosing regimens. Simulations revealed that continuous therapy rapidly selects for resistance, leading to tumor regrowth within ∼150 days. In contrast, intermittent dosing schedules maintained long-term control for more than 300 days. Moreover, the model identified an adaptive strategy: switching from continuous to intermittent dosing precisely when tumor burden reaches its minimum. This evolutionarily informed approach prevents resistant clones from dominating while leveraging the fitness disadvantage associated with drug addiction. Preliminary experiments with naïve and resistant ALCL cell lines, cultured under intermittent dosing in both monoculture and co-culture systems, support the model’s predictions. These experiments also enable the refinement of the model's parameters to incorporate tumor heterogeneity. While additional validation is required, our findings demonstrate that integrating mathematical modeling with experimental DRCs provides a quantitative framework for predicting the evolutionary dynamics of resistance and addiction. Collectively, this study highlights the potential of adaptive and evolution-based therapy in ALCL. Optimized treatment schedules informed by evolutionary trade-offs can reduce tumor burden, delay progression, and exploit vulnerabilities generated by resistance mechanisms.



 Franco Pradelli, Lucy Hare, Jamie Matthews, Roxane Chen, Aiindrila Dhara, G. A. Amos Burke, Suzanne D. Turner, Jeffrey West. Capturing dose-response adaptation to identify evolutionarily-informed treatment strategies [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B037.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86b1f1eb717bb47b50718cff89eef53948102c75" target='_blank'>
              Abstract B037: Capturing dose-response adaptation to identify evolutionarily-informed treatment strategies
              </a>
            </td>
          <td>
            Franco Pradelli, Lucy Hare, J. Matthews, Roxane Chen, Aiindrila Dhara, G. A. A. Burke, Suzanne D. Turner, J. West
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a27e17b141d07dfd3357f3ee906c1f3a2792dd3c" target='_blank'>
              SigDyn: single-cell mutational signature dynamics
              </a>
            </td>
          <td>
            , 
          </td>
          <td>2026-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Human T-cell leukemia virus type 1 (HTLV-1) infects mainly CD4+ T lymphocytes and causes both malignant and inflammatory diseases: the aggressive malignancy known as Adult T-cell leukemia/lymphoma (ATL) and several chronic inflammatory syndromes. HTLV-1 infection is established by integration of proviral DNA (~9000 bp) into the host genome. In HTLV-1, two viral genes, tax and HBZ, play critical roles in viral transcription and promotion of T-cell proliferation, respectively. The present study was undertaken to test the hypothesis that the higher-order structure of proviral chromatin regulates its transcription on both the plus strand and the minus strand. ATAC-seq analysis identified an open chromatin region in the pol gene of proviral DNA, which we name IPOR, in many ATL cases. Using reporter assays, it was found that the sequence of IPOR suppresses the transcription of the plus strand and activates that of the minus strand. Binding motifs of Eomes and TEAD proteins were predicted in this region, and we confirmed recruitment of the transcription factors to their respective motifs by ChIP-qPCR. A mutant of IPOR which cannot bind the transcription factors weakened the transcriptional activating effects compared with the wild type, suggesting that the IPOR and those transcription factors suppress the 5' long terminal repeat (LTR) but activate the 3'LTR. In addition, a mutant HTLV-1 molecular clone, which possesses the IPOR mutant, produced a higher titer of virus than the wild type. RNA-seq analysis of HTLV-1-infected cell lines, in which Tax expression can be traced after induction, revealed that EOMES expression decreases during the tax transcriptional burst and resumes following termination of the burst. These findings suggested that the expression dynamics of Eomes affect the transient expression of Tax. The IPOR sequence appears to regulate the transcription from both LTRs, suppressing the 5'LTR but activating the 3'LTR. Recruitment of Eomes to the IPOR is likely to influence the expression of Tax and HBZ. Since both viral genes are involved in diverse mechanisms for viral replication, cellular proliferation, and immune regulation, the IPOR may play a role in fine-tuning the modes of viral persistence in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68b2f1e6ae01e0a8f9b5438f95ad7675208ffd69" target='_blank'>
              Intra-pol proviral open region of HTLV-1 controls the transcription from both long terminal repeats
              </a>
            </td>
          <td>
            Miyu Sonoda, Azusa Tanaka, Xueda Chen, Wenyi Zhang, Helen Kiik, S. Ramanayake, K. Nosaka, N. Takenouchi, Masanori Nakagawa, Akihiro Fujimoto, C. Bangham, Masao Matsuoka, Jun-ichirou Yasunaga
          </td>
          <td>2025-12-08</td>
          <td>Retrovirology</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c702cdf51be4f8a5bb8e9bacfe163f51cdb987fa" target='_blank'>
              Mitochondrial genetics defines anti-tumour immunity through mitochondrial ROS and PD-1 signalling
              </a>
            </td>
          <td>
            Gennaro Prota, Raquel Justo-Méndez, U. Gileadi, Macarena de Andrés-Laguillo, J. Cabrera-Alarcon, María del Mar Muñoz-Hernández, Sofía C Khouili, Raquel Martínez-de-Mena, Juan Pellico, Rui Benedito, J. Ruíz-Cabello, D. Sancho, José Antonio Enríquez, A. V. Lechuga-Vieco
          </td>
          <td>2025-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Nucleotide substitutions are common in cancer cells, and they occur in both protein coding regions and non-coding regions (5′ and 3′ UTRs and introns). Although substitutions in non-coding regions have the potential to alter gene expression, it is the alteration of coding regions that affects protein function and has the most drastic effect on cellular transformation. Mutations in certain genes (e.g., TP53, KRAS) are common to nearly all cancers, but most cancers are characterized by specific gene mutation signatures. In this report, we investigated nucleotide substitution signatures in coding regions of the top 25 most frequently mutated genes in multiple human cancers. The goal was to examine whether unique nucleotide substitution biases are associated with various cancers. A pan-cancer analysis showed that the most altered nucleotide is guanine, which is biased towards G->A transitions. A per-cancer analysis identified ten cancers with biased substitutions in certain genes. Some of these biases were expected (e.g., KRAS in gastrointestinal cancers or JAK2 in blood cancers). Our analysis revealed biased signature substitutions in 17 genes, of which 14 were characterized as drivers and constituted a closely related set of cell cycle regulators. We conclude that nucleotide substitution biases contribute to specific alterations in cancer genes that produce cellular transformation. Principle component analysis of nucleotide substitutions shows that most cancers cluster together, meaning that they have similar nucleotide changes. However, certain cancers, most notably lung, pancreas, and blood cancers, can be differentiated from each other based on specific nucleotide signatures. Thus, nucleotide substitution patterns can be used to differentiate between some cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/baa7149b8cff77e69abfdaa8112e2209417108ee" target='_blank'>
              Nucleotide Substitution Biases in Related Cancer Driver Genes
              </a>
            </td>
          <td>
            Adam Khadre, Yifan Dou, Golrokh Mirzaei, Ruben C. Petreaca
          </td>
          <td>2025-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Sorex araneus, the Eurasian common shrew, has seasonal brain size plasticity (Dehnel's phenomenon) and abundant intraspecific chromosomal rearrangements. Genomic contributions to these traits, however, remain unknown. We couple a chromosome-scale genome assembly with seasonal brain transcriptomes to discover relationships between molecular changes and both traits. While positively selected genes enriched the Fanconi anemia DNA repair pathway (FANCI, FAAP100), which is likely involved in chromosomal rearrangements by preventing the accumulation of chromosomal aberrations, genes under positive selection or showing seasonal differential expression in the brain implicate neurogenesis (PCDHA6, SOX9, Notch signaling) and metabolic regulation (VEGFA, SPHK2) as key mechanisms underlying Dehnel's phenomenon. We also find that both positively selected and differentially expressed genes in the hippocampus are overrepresented near Sor. araneus evolutionary breakpoints. This relates both positive selection and differential expression to accessible chromatin configuration, suggesting that chromosomal rearrangements are integral to adaptive evolution and the regulation of brain size plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60e5d59bc3d794d733f5961db0a47844813d84f1" target='_blank'>
              Genomic comparisons shed light on the adaptive basis of brain size plasticity and chromosomal instability in the Eurasian common shrew.
              </a>
            </td>
          <td>
            William R. Thomas, Tanya M. Lama, Cecilia Baldoni, Laia Marín-Gual, Diana D Moreno Santillán, Marta Farré, Linelle Abueg, Jennifer R. Balacco, O. Fedrigo, G. Formenti, Nivesh Jain, J. Mountcastle, Tatiana Tilley, Ying Sims, Alan Tracey, Jonathan M. D. Wood, David A. Ray, Dominik von Elverfeldt, J. Nieland, Angelique P. Corthals, A. Ruiz-Herrera, D. Dechmann, Erich D. Jarvis, Liliana M. Dávalos
          </td>
          <td>2026-01-09</td>
          <td>Molecular biology and evolution</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Gene regulatory innovation underlies many phenotypic transitions. Transposable elements are an established mechanism for creating families of cis-acting elements with shared sequence features and the potential to establish co-regulatory networks. To understand additional mechanisms by which co-regulatory networks form, we define families of noncoding elements based on sequence similarity and cell type-specific activity. We apply this analysis framework to the human telomere-to-telomere genome assembly and embryonic stem cell chromatin accessibility data. We identify segmental duplications as the major mechanism establishing these families, creating over one thousand networks of elements with open chromatin in embryonic stem cells. We functionally validate a subset of these networks as families of regulatory elements with STARR-seq and identify their target genes with CRISPRi in embryonic stem cells. Following segmental duplication, we find that regulatory elements at times maintain their relationship to target genes, and at times rewire to form novel connections. During this rewiring, we observe proximal-acting elements gaining the ability to regulate distally-located genes and observe transcriptional enhancers rewiring to regulate genes present at the locus outside the segmental duplication. Many of these rewiring events are human specific. Finally, we find that segmental duplications have made outsized contributions to expanding regulatory element families functioning in immune cell types and specific brain regions, including the posterior cingulate gyrus. We speculate that placing regulatory elements in new genomic contexts primes regulatory elements for neofunctionalization, and that regulatory rewiring after segmental duplication was a common mechanism underlying gene regulatory change during human evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d5648f42bb808c07a76dd1ef4b18bdfc083196e" target='_blank'>
              Segmentally duplicated regulatory elements undergo human-specific rewiring
              </a>
            </td>
          <td>
            Seth Weaver, Craig B. Lowe
          </td>
          <td>2025-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6c066b01c86150bb1cb77ee7e3befe19a5fcc47" target='_blank'>
              Orphan Non-Coding RNAs Drive Tumorigenesis and Enable Pre-Diagnostic Detection in HPV-Negative Head and Neck Cancer
              </a>
            </td>
          <td>
            Hoang C. B. Nguyen, Lishi Li, M. Karimzadeh, Jack Ghanam, Jeffrey Wang, Yana Al Inaya, G. Lumaj, F. Hormozdiari, B. Alipanahi, A. Feng, J. Richmon, D. Deschler, M. Varvares, D. Faden, Derrick Lin, Hani Goodarzi
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Cellular plasticity, the ability of cells to dynamically alter their phenotypes, is a key driver of tumor evolution. This process is a hallmark of cancer which enables the acquisition of malignant traits, leading to metastasis, progression, and therapy resistance. It is governed by cell‐intrinsic factors, such as genomic instability and epigenetic reprogramming, and extrinsic stimuli from the tumor microenvironment. However, a unified framework is still needed to position plasticity as the central process that links these drivers to diverse cancer hallmarks. In this review, we first explore how plasticity enables key steps of tumor evolution, including tumorigenesis, metastasis driven by epithelial–mesenchymal plasticity (EMP), therapy resistance, and cancer stem cell (CSC) dynamics. We then summarize the intrinsic and extrinsic mechanisms that govern this adaptability. Finally, we discuss clinical advances in monitoring and targeting plasticity and highlight how new spatiotemporal technologies can address current research challenges. This review provides a framework positioning cellular plasticity as a central mechanism in cancer evolution, connecting its fundamental drivers to clinical translation. By synthesizing the latest advances, we offer perspectives for developing therapies that integrate prediction, monitoring, and targeting of plasticity to proactively guide cancer evolution toward manageable outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/683e6aa0ee59731275431238ec93cc43ffca8292" target='_blank'>
              Tumor Cell Plasticity in Cancer: Signaling Pathways and Pharmaceutical Interventions
              </a>
            </td>
          <td>
            Shangwei Sun, Yunwei Sun, Ling Lan, S. Luan, Jin Zhou, Jiehui Deng, Yong Yuan, Zhong Wu
          </td>
          <td>2025-12-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54cabefaf605ac94e03143b5c4643d22a1076d5d" target='_blank'>
              miRNA-mediated cell-to-cell communications boost DNA repair during the Radioadaptative Response
              </a>
            </td>
          <td>
            María del Carmen Domínguez-Pérez, María Jesús Fernández-Ávila, Lourdes González-Vinceiro, L. Zannini, Héctor Peinado, Román González-Prieto, Néstor García-Rodríguez, Pablo Huertas
          </td>
          <td>2025-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6532d2e7cd4f584634ccf28fbab489accbcdd3b5" target='_blank'>
              DNA repeats: origins, conservation, and role in human disease
              </a>
            </td>
          <td>
            A. K. Rana
          </td>
          <td>2026-01-06</td>
          <td>Genome Instability & Disease</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Understanding how chemotherapy perturbs cell cycle dynamics is critical for advancing cancer treatment. We develop a probabilistic, multi-generational framework based on a Bellman–Harris branching process to quantify treatment-induced shifts in tumor cell dynamics. The model incorporates key drug-responsive behaviors, including checkpoint activation, apoptosis, and checkpoint adaptation that propagates inherited DNA damage, enabling the characterization of heterogeneous survival outcomes after treatment. Biological parameters map directly onto DNA repair fidelity and cell-fate decisions, providing mechanistic insights beyond what is accessible from experiments alone. Dose-dependent extensions further allow exploration of treatment-induced perturbations. Model parameters were calibrated to empirical cell cycle measurements using the robust adaptive Metropolis algorithm. Global sensitivity analysis shows that scale parameters governing unfaithful DNA repair under G2/M- and S phase–specific agents exert major influence on model predictions, particularly at later time points. Across three chemotherapies, the framework reveals consistent dose-dependent alterations in cell cycle dynamics, with higher doses driving pronounced disruptions. Together, these results demonstrate how model-informed analyses can provide quantitative insight into treatment-induced cell cycle perturbations and support the refinement of therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/130ec8b5cedf3ce34952bca8649e510976ff8a38" target='_blank'>
              Probabilistic modeling of cell cycle dynamics in response to cell cycle targeting chemotherapy drugs to guide treatment strategies
              </a>
            </td>
          <td>
            Chenhui Ma, Evren Gurkan-Cavusoglu
          </td>
          <td>2025-12-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Serine/arginine-rich splicing factor 11 (SRSF11) is an RNA-binding regulator that modulates alternative splicing and RNA metabolism in a context-dependent manner across selected malignancies. Evidence from colorectal, hepatocellular, gastric, glioma, and a few other cancers indicates that SRSF11 participates in cell-cycle regulation, telomerase recruitment, and epithelial-mesenchymal transition (EMT) through specific signaling axes, including PAK5-SRSF11-HSPA12A in colorectal cancer, METTL3-SRSF11 in gastric and breast cancers, and SRSF11-CDK1/telomerase circuits in hepatocellular carcinoma. These mechanisms highlight SRSF11 as a candidate biomarker for diagnosis and prognosis rather than a universal oncogenic driver. We summarize the current mechanistic, post-translational, and non-coding RNA-mediated regulatory evidence, clarify the limitations of existing data, and propose future multi-omics and functional approaches to validate SRSF11-directed splicing therapy. This review integrates mechanistic insight with clinical evidence while emphasizing cancer-specific rather than generalized conclusions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c51f190a021e2902688fb2026499d28338fcc769" target='_blank'>
              Dysregulation of SRSF11 in Cancer: Mechanistic Insights and Biomarker Potential for Diagnosis and Therapy
              </a>
            </td>
          <td>
            Yang Jun, Shanshan Liu, Jingwen Xiao, He Yu, Jianlong Xiao, Qingfeng Shi
          </td>
          <td>2026-01-14</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="

 The complexities of nucleotide metabolism in breast cancer (BC) cells are not yet fully understood. A deeper exploration of the various tumor subpopulations and their interactions with the tumor microenvironment (TME) could provide important insights into these clinically relevant signaling pathways.



 We integrated five distinct single-cell enrichment scoring methodologies to conduct a comprehensive enrichment analysis of BC cell populations. The analytical findings underwent subsequent validation using an independent single-cell cohort. Tumor cell clusters were categorized based on their average enrichment scores. Functional analyses were carried out using several tools, including CellChat, Monocle, CopyKAT, SCENIC, and CytoTRACE. The RCTD method was then employed to map the single-cell clusters onto spatial transcriptomics data, facilitating the evaluation of cellular dependencies and pathway activities to differentiate tumor cell subtypes. A prognostic framework was subsequently established using large-scale transcriptomic datasets, enabling prediction of immunotherapy responsiveness. Experimental validation further confirmed expression patterns of pivotal genes implicated in therapeutic outcomes.



 Tumor cells exhibit significantly upregulated nucleotide metabolic activity, enabling their classification into two distinct subgroups: NUhighepi and NUlowepi. Cells within the NUhighepi subgroup demonstrate pronounced malignant phenotypes. Intercellular communication analysis performed with the stLearn platform revealed robust interactions between NUhighepi cells and fibroblasts. Supporting this finding, spatial transcriptomic analysis via MISTy revealed a distinct dependency of NUhighepi on fibroblasts. A robust prognostic model, developed using various machine learning algorithms, was able to predict survival outcomes and responses to immunotherapy. Furthermore, targeted drugs were identified for both the high and low scoring groups. Experimental investigations confirmed the expression of core genes in different breast cancer cells.



 In conclusion, we developed a nucleotide metabolism-derived prognostic signature for BC, with DCTPP1 highlighted as a promising biomarker and therapeutic target. These findings provide a valuable framework for early clinical intervention and show promising potential for predicting responses to immunotherapy in BC patients.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/502ca564395f021e293ba0e223c1f50472f85541" target='_blank'>
              Spatial transcriptome and single-cell sequencing reveal the role of nucleotide metabolism in breast cancer progression and tumor microenvironment
              </a>
            </td>
          <td>
            Yuejun Pan, Yong Xu, Ke Gong, Songlin Yuan
          </td>
          <td>2026-01-14</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Senescence is a pleiotropic phenotype that alternatively suppresses or promotes cancer. The tumor suppressive roles are linked to clearance of damaged cells by the immune system, while cells with tumor promoting functions resist apoptosis, evade immune clearance and either persist and support the tumor microenvironment, or escape and proliferate. What generates these diverse populations is unclear. We investigated this in preneoplastic zebrafish livers where the epigenetic regulator, UHRF1, is overexpressed in hepatocytes. Double strand breaks, DNA methylation repatterning, retrotransposon expression, cell cycle withdrawal and activation of Atm and Tp53 dependent senescence were early responses to UHRF1 overexpression. This evolved to generate diverse populations of senescent cells, some which expressed immune and senescence signatures plus anti-apoptotic markers, and others co-expressed proliferative genes. The fate of these populations was dictated by UHRF1 levels and Tp53, as Tp53 loss enabled proliferation of cells with reduced UHRF1 expression but not in cells expressing high UHRF1. The senolytic Navitoclax targeted only a subset of senescent cells. Thus, the diversity of senescent cells driven by epigenetic changes can generate divergent outcomes. Short summary UHRF1 overexpression in zebrafish hepatocytes induces DNA damage leading to Atm–tp53–dependent senescence with UHRF1 levels dictating whether hepatocytes remain senescent, escape, or undergo senolytic elimination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b990cf440846339c231eccc76316041249b0f0d" target='_blank'>
              UHRF1 Overexpression Generates Distinct Senescent States with Different Tp53 Dependencies
              </a>
            </td>
          <td>
            Elena Magnani, Charlene Chen, F. Macchi, Bhavani P Madakashira, Tijana Randic, Ian McBain, Kirsten C. Sadler
          </td>
          <td>2025-11-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7b4d870dc4ba763551ef81671814fc477f8046f" target='_blank'>
              CancerSTFormer enables multi-scale analysis of spot-resolution spatial transcriptomes and dissects gene and immune regulatory responses to targeted therapies
              </a>
            </td>
          <td>
            Benjamin S Strope, Dana Varghese, William Z Bowie, Stacy Wang, Qian Zhu
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/206d73725d5a48cf760cbd0de1a8e6d43ff31976" target='_blank'>
              Features affecting Cas9-Induced Editing Efficiency and Patterns in Tomato: Evidence from a Large CRISPR Dataset
              </a>
            </td>
          <td>
            Amit Cucuy, Daniela Ben-Tov, C. Melamed-Bessudo, Arik Honig, Barry A. Cohen, A. Levy
          </td>
          <td>2026-01-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Dogs play a major role in sustaining transmission of Leishmania infantum to people, thus prevention and treatment of canine leishmaniosis (CanL) to reduce transmission represents an unmet public health need. Although advances have been made in understanding how immunopathology correlates to infectiousness as CanL progresses, the immune mechanisms underlying transition between disease stages and terminal decline remain ill-defined. To address this knowledge gap, we generated a comprehensive atlas of peripheral immune cells from control dogs and naturally-infected dogs using single-cell RNA sequencing. The LeishDog Atlas captures the cellular and transcriptional complexity underlying CanL, tracing shifts in immune composition and gene expression across progressive, well-defined disease stages. Notably, we identified distinct myeloid, CD4+ and CD8+ T cell phenotypes and transcriptional states associated with disease progression. This resource provides a valuable framework for understanding systemic immune dysregulation in CanL, including determinants of T cell exhaustion and myeloid cell activation, and establishes a foundation for comparative, translational, and mechanistic studies of CanL immunopathology and transmissibility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8412687f28e64e4d6b2558529c494e2ee4440eca" target='_blank'>
              A single-cell transcriptomic atlas of peripheral blood immune cells spanning progressive canine leishmaniosis
              </a>
            </td>
          <td>
            Danielle P. Uhl, Daniel J. Holbrook, Max C Waugh, S. Dey, Najmeeyah Brown, Karen I. Cyndari, Jacob J. Oleson, Paul M. Kaye, Christine A. Petersen
          </td>
          <td>2025-12-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Over 15% of cancers worldwide are caused by viruses. Merkel cell polyomavirus (MCPyV) is the most recently discovered human oncovirus and is the only polyomavirus that drives malignant tumors in humans. Here, we show that MCPyV+ Merkel cell carcinoma is defined by neuroendocrine-lineage core regulatory (CR) transcription factors (TFs) (ATOH1, INSM1, ISL1, LHX3, POU4F3, and SOX2) that were essential for tumor survival and that co-bound chromatin with the viral small T antigen at super enhancers. Moreover, MCPyV integration sites were enriched at these neuroendocrine super enhancers. We further discovered that the MCPyV noncoding control region contained a homeodomain binding motif absent in other polyomaviruses that bound ISL1 and LHX3 and depended on them for T antigen expression. To therapeutically target the CR factors, we used histone deacetylase (HDAC) inhibitors to collapse the chromatin architecture and induce topological blurring of superenhancer loops, abrogating core TF expression and halting tumor growth. To our knowledge, our study presents the first example of oncogenic cross-regulation between viral and human epigenomic circuitry to generate interlocking and essential transcriptional feedback circuits that explain why MCPyV causes neuroendocrine cancer and represent a tumor dependency that can be targeted therapeutically.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d35113c8a7150319a63f0b69dd0c395aec4ab43" target='_blank'>
              Interlocking host and viral cis-regulatory networks drive Merkel cell carcinoma
              </a>
            </td>
          <td>
            L. Miao, D. Milewski, Amy Coxon, T. Gelb, K. Garman, Jadon Porch, Arushi Khanna, L. Collado, Natasha T. Hill, Kenneth Daily, S. Vilasi, Danielle Reed, Tiffany Alexander, G. Starrett, Maharshi Chakraborty, Young Song, Rachel Choi, Vineela Gangalapudi, Josiah D. Seaman, Andrew Morton, K. Busam, C. Vakoc, Daniel J. Urban, Min Shen, Matthew D. Hall, Richard Sallari, Javed Khan, B. Gryder, Isaac Brownell
          </td>
          <td>2025-12-15</td>
          <td>The Journal of Clinical Investigation</td>
          <td>1</td>
          <td>98</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fef918e1d4b64862104d2dc24b6aa7569489a94" target='_blank'>
              Genomic profiling implicates candidate genes and mutagenic pathways driving lung cancer recurrence
              </a>
            </td>
          <td>
            L. Luhari, A. Valter, A. Bahcheli, K. C. Cheng, M. Bayati, A. Ustav, A. Velthut-Meikas, K. Oselin, J. Reimand
          </td>
          <td>2026-01-01</td>
          <td>None</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="R-loops consist of double-stranded DNA-RNA hybrids and a complementary DNA strand that is displaced from the duplex. R-loops play important role in numerous normal physiological processes, including DNA methylation, chromatin remodeling, RNA editing, replication, DNA repair, immunoglobulin class switching, and chromosome segregation during cell division. However, excessive or untimely formation of R-loops can lead to replicative collapse and subsequent DNA damage, resulting in genomic instability. One type of genomic rearrangements that is strongly associated with cancer malignancy is the extrachromosomal amplification of genes on circular DNA molecules (ecDNA). These molecules are relieved of hereditary constraints and conventional segregation laws and can endow cancer cells with the ability to rapidly change their genome, thereby accelerating tumor evolution and the development of therapy resistance. Multiple lines of evidence indicate that upregulated transcription of a gene can increase its susceptibility to amplification. Although the mechanisms underlying these processes are not yet fully understood, R-loops may play an important role in initiating gene amplification. In this review, we highlight the role of R-loops in replicative collapse, double-strand breaks, and DNA damage repair. We also provide examples of gene amplifications that is known to be induced by R-loops. Finally, we discuss amplification mechanisms in which involvement of R-loops has not yet been demonstrated, but appears highly likely.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d383d8d8b2013ac53fe539c01740ab644c592c14" target='_blank'>
              R-loops as a trigger for intra- and extrachromosomal DNA amplification in cancer
              </a>
            </td>
          <td>
            Tatyana F. Kovalenko, Amal Abdurazakov, Nadezhda V. Antipova, Mikhail I. Shakhparonov, M. Pavlyukov
          </td>
          <td>2025-12-03</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>1</td>
          <td>18</td>
        </tr>

        <tr id="MYC performs a dual role in DNA Damage Response (DDR), promoting genomic instability through replication stress, R-loop formation, and topoisomerase-mediated damage, while simultaneously activating DNA repair pathways to maintain cell survival. This review provides a comprehensive analysis of how MYC inhibition affects DDR pathway dependencies. In fact, when MYC is inhibited, cancer cells lose both their proficient DNA repair capacity and their protective mechanisms against replication stress. This creates a therapeutic window in which combining MYC inhibitors with DDR-targeting agents may achieve synergistic anti-cancer effects. Central to this approach is the exploration of rational combination strategies that pair MYC inhibitors with various DDR modulators including Poly (ADP-ribose) polymerase (PARP) inhibitors, ATR/CHK1 inhibitors, and other DNA repair pathway disruptors. This review summarizes preclinical evidence demonstrating enhanced therapeutic efficacy when MYC inhibition is combined with DDR-targeting agents and discusses early clinical findings that support this promising therapeutic strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbbb71b7cb292258fb87c94bcd4055c9b05edb3f" target='_blank'>
              The leading role of MYC in DNA damage response: exploring opportunities for therapeutic inhibition
              </a>
            </td>
          <td>
            F. Giuntini, Jonathan R. Whitfield, D. Massó-Vallés, Laura Soucek
          </td>
          <td>2026-01-05</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Over the past 5–7 million years, humans and chimpanzees have diverged in brain size, structural complexity, and cognitive abilities despite high conservation of protein-coding genes. Notably, the endogenization and proliferation of retroviral infections within host genomes has introduced numerous species-specific regulatory elements that have the potential to influence gene regulation. However, the role of these endogenous retroviruses in hominoid brain evolution remains unclear. A burst of lineage-specific PTERV1 retroviruses recently invaded the chimpanzee genome but are absent in humans. We conducted an epigenomic analysis of PTERV1 insertions in chimpanzee neural organoids and found that they are heavily covered by DNA methylation, representing more than 150 species-specific heterochromatin domains with the capacity to influence gene regulatory networks. We identified one such chimpanzee-specific PTERV1 insertion on chromosome 19 that blocks the expression of the long noncoding RNA LINC00662, via DNA methylation spread to the adjacent genomic region. The expression of LINC00662 was restored in chimpanzee induced pluripotent stem cells when we deleted the PTERV1 insertion using CRISPR editing. We found that LINC00662, a human-specific RNA, is highly expressed in the developing brain and plays an important role in the posttranscriptional control of neuronal maturation, axon outgrowth, and neural organoid development. In summary, our findings describe how endogenous retroviral insertions contributed to the functional divergence of the human and chimpanzee brains. This provides a new mechanism by which retroviral pandemics influenced primate brain speciation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b7aafdaae0e68f3439418fdcde81bbb4ad6e768" target='_blank'>
              Retroviral insertions contributed to the divergence of human and chimpanzee brains
              </a>
            </td>
          <td>
            Patricia Gerdes, Ofelia Karlsson, R. Garza, Diahann A. M. Atacho, Pinghsun Hsieh, Bharat Prajapati, Chandramouli Muralidharan, Anita Adami, Carrie Davis-Hansson, Emily M. Johansson, Laura Castilla-Vallmanya, Wiktor West, Meghna Vinoud, Lucian Domitrovic, Pia A. Johansson, Jenny G. Johansson, C. Douse, C. Kanduri, Patric Jern, E. Eichler, J. Jakobsson
          </td>
          <td>2025-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Despite substantial progress in the field of bladder cancer management, the disease continues to represent an important health issue characterized by increased recurrence and progression rates. This is largely attributed to cancer stem cells (CSCs), a unique cell subpopulation capable of self-renewal, differentiation and resistance to conventional anti-cancer therapies. At the same time, our understanding of cancer biology has been revolutionized by the identification of non-coding RNAs (ncRNAs), a heterogeneous group of RNA molecules that do not translate into proteins yet function as pivotal regulators of gene expression. Emerging evidence demonstrates that ncRNAs modulate key hallmarks of CSCs, including self-renewal, epithelial–mesenchymal transition and drug resistance. This review investigates the intricate interplay between ncRNAs and the core signaling pathways that underlie bladder CSC biology. Unravelling the nexus between CSCs and ncRNAs is crucial for developing novel diagnostic biomarkers, better prognostic tools and innovative therapeutic strategies for patients with bladder cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13091e7103acf49fcdc024b48e3b7a3f0f4f481d" target='_blank'>
              The Crossroads of Cancer Regulation: Discussing the Role of Non-Coding RNAs in Bladder Cancer Stem Cells
              </a>
            </td>
          <td>
            Alexandros Georgiou, Dimitrios Triantis, M. Goulielmaki, V. Zoumpourlis
          </td>
          <td>2025-12-11</td>
          <td>Uro</td>
          <td>1</td>
          <td>28</td>
        </tr>

        <tr id="Cancer represents a disease in which genetic alterations reassert the dominance of the cell cycle over all other cellular processes. From the earliest stages of evolution, the coupling of energy utilization with nucleotide and DNA synthesis established replication as the central driver of cellular behavior. In cancer, this evolutionary logic is replayed in reverse. Hyperactivation of the cell cycle drives hyperactivation of its metabolic core, while the loss or inactivation of tumor suppressor genes, many of which are cell-type specific, links this accelerated proliferation with altered cell fate. With virtually unlimited energy available, malignant cells amplify nucleotide production and DNA replication without restraint. This Perspective proposes that anti-metabolites, long-standing pillars of cancer therapy, can be redesigned to target the main components of DNA metabolism. By rationally combining these anti-metabolites into synergistic triads (three anti-metabolites, well selected, administered together), therapy may dismantle the metabolic foundations of cancer and achieve more durable control across tumor types. The combinations that could yield meaningful progress are outlined and discussed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67e9bbd295e09ee137dd6241889d264b14d2be13" target='_blank'>
              Targeting energy, nucleotide, and DNA synthesis in cancer
              </a>
            </td>
          <td>
            Ben Zion Vider
          </td>
          <td>2025-12-04</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Colon cancer (CC) is a malignancy with high global incidence and mortality, and elucidating its underlying molecular mechanisms is critical for improving prognostic assessment and therapeutic strategies. In this study, transcriptomic data from a large cohort of CC samples and a limited number of normal controls from the TCGA database were used to construct a multigene prognostic risk model using univariate Cox and least absolute shrinkage and selection operator (LASSO) regression analyses. The expression of key prognostic genes, including immunoglobulin superfamily member 9 (IGSF9), was further validated in CC tissues by PCR and Western blotting. Functional assays were performed in HCT116 cells to investigate the biological effects of IGSF9 overexpression and its regulatory relationship with p53. The prognostic model identified IGSF9 as a gene significantly associated with patient survival. Although IGSF9 expression was reduced at both the mRNA and protein levels in CC tissues, its overexpression in vitro markedly promoted apoptosis, alleviated DNA damage, and suppressed cell migration and invasion. Notably, silencing of p53 partially reversed the tumor‐suppressive effects induced by IGSF9 overexpression, indicating that IGSF9 exerts its biological functions in a p53‐dependent manner. Collectively, these findings demonstrate that IGSF9 acts as a tumor suppressor in colorectal cancer and regulates DNA damage responses and apoptosis through a mechanism partially dependent on p53, highlighting its potential value as a prognostic biomarker and therapeutic target in CC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7ff548ce8387a244b0397236c92065d62ddf1f1" target='_blank'>
              P53 Inhibition Diminishes IGSF9 Gene Activity to Promote DNA Repair and Exacerbate the Progression of Colon Cancer
              </a>
            </td>
          <td>
            Huan-Yu Zhang, Dan Tian, Wan-Fu Zhang, Ying-Hui Zhang, Jia-Li Feng, Juan Sheng, Xue-Qin Shang
          </td>
          <td>2026-01-01</td>
          <td>Journal of Biochemical and Molecular Toxicology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a29ad767de6b1b60629bc324b66e333d5c7c4a2b" target='_blank'>
              A Human Genetics Framework for De-risking Gene Editing Targets for Hematopoietic Cell and Gene Therapy
              </a>
            </td>
          <td>
            A. Dhall, Alyssa Pyclik, Matthew H. Ung, Julia Etchin, Y. Keschner, Huanying Gary Ge, Tirtha Chakraborty, J. Lydeard
          </td>
          <td>2025-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background Characteristic genetic events underpin acute myeloid leukemia (AML) heterogeneity and enable precise risk stratification. However, prognostic assessment remains ambiguous in many patients due to inadequate integration of specific genetic information. Materials and Methods Eighty NPM1-mutated AML patients were enrolled. Copy number alterations (CNAs) were detected via shallow whole-genome sequencing (sWGS), and concurrent mutations via targeted deep sequencing of myeloid malignancy-associated genes. Clinical and laboratory parameters were integrated with genomic data for statistical analysis, with the aim of assessing the potential clinical significance of CNA profiles in prognostic stratification. Results NPM1 mutation subtypes A, B, and D were the most prevalent, with all patients harboring at least two concurrent mutations (4–5 mutations being the most frequent), and these mutations commonly co-occurred with those in FLT3, DNMT3A, TET2, IDH2, and NRAS. Forty-one samples (51%) exhibited CNAs across diverse genomic regions, with dup(18)(p11.23) identified as the most recurrent locus. No significant differences in FAB classification, hematologic parameters, demographic characteristics (gender, age), co-mutation profiles, complete remission (CR) rates, or survival outcomes were observed between the CNA-positive and CNA-negative groups. Univariate survival analysis revealed patients with ≥2 CNAs, or FLT3-internal tandem duplication (FLT3-ITD) had significantly shorter overall survival (OS). Notably, integrative analysis of CNAs with mutational profiles showed that patients harboring both FLT3-ITD and ≥2 CNAs had the poorest OS, followed by those with FLT3-ITD and <2 CNAs, and multivariate Cox regression analysis suggests a potential association between ≥3 CNAs and adverse outcomes; however, given the limited sample size of cases with high CNA burden, this result should be interpreted with caution. Conclusion This exploratory study suggests that combining CNAs and gene mutation profiles may potentially improve the existing prognostic evaluation system for NPM1-mutated AML patients. Confirmation of these results requires additional validation in larger prospective cohorts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46557e96e5c633f3844a681ce880aa92b69a8c33" target='_blank'>
              The Prognostic Value of Integrating Copy Number Alteration Profiles in NPM1-Mutated Acute Myeloid Leukemia: An Exploratory Study
              </a>
            </td>
          <td>
            Chun-xia Zhang, Li Gao, Ming Gong, Chundi Liu, Dongmei Zhang, Zhen-ling Li
          </td>
          <td>2025-12-01</td>
          <td>The Application of Clinical Genetics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Colorectal cancer (CRC) develops through evolutionary processes involving genomic alterations, epigenetic regulation, and microenvironmental interactions. While traditionally explained by the stepwise accumulation of driver mutations, contemporary evidence supports a ‘Big Bang’ model in which many early-arising clones expand simultaneously to establish extensive heterogeneity. We reviewed recent studies employing spatially resolved multi-omic sequencing of tumour glands combined with computational modelling. These approaches enable high-resolution reconstruction of clonal architecture, transcriptional states, and chromatin accessibility. Findings show that although early clonal mutations shape tumour expansion, gene expression variability can be independent of genetic ancestry and instead reflects phenotypic plasticity driven by microenvironmental cues. Epigenomic analyses identified recurrent somatic chromatin accessibility alterations in promotors and enhancers of oncogenic pathways, frequently in the absence of DNA mutations, suggesting alternative mechanisms of gene regulation. Immune-focused studies demonstrated that early silencing of antigen-presenting genes and loss of neoantigens facilitate immune escape despite active surveillance. CRC is shaped by an interplay of genome, epigenome, and immune evolution, with non-genetic mechanisms and tumour plasticity emerging as important drivers of progression and therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c00d0ede4db6fdd44f5a7c3b138a4161c7d8d0f7" target='_blank'>
              Advances in Colorectal Cancer Cell Biology and Clonal Evolution
              </a>
            </td>
          <td>
            Sopozme Toghey, Elizabeth J. Harvey-Jones, Jonathan D. Towler, Charlotte J. H. Hafkamp, Irene Y. Chong
          </td>
          <td>2026-01-18</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background: Non-homologous end joining (NHEJ) is a crucial pathway for repairing DNA double-strand breaks and a key contributor to chemoresistance in cancer. The assembly of the DNA Ligase IV (LIG4)–XRCC4 complex is essential for NHEJ fidelity, however, the regulatory mechanisms governing this complex in cancer remain poorly understood. This study aims to investigate whether and how lactate, a key metabolic byproduct of the Warburg effect, regulates the XRCC4–LIG4 complex and influences chemoresistance. Methods: The functional role of lactate in NHEJ was assessed using DNA repair reporter assays in ovarian cancer cells. Protein–protein interactions were examined through co-immunoprecipitation and pull-down assays. The molecular mechanism of lactate’s action was delineated using a combination of site-directed mutagenesis, in vitro binding assays, and molecular docking. Finally, the physiological relevance of lactate-mediated NHEJ was validated in a preclinical ovarian cancer mouse model treated with cisplatin. Results: We demonstrated that lactate enhances NHEJ repair efficiency and confers resistance to DNA-damaging chemotherapeutics. Mechanistically, lactate directly binds to XRCC4 at key residues, including Y66, E55, and S110, thereby strengthening the XRCC4–LIG4 association. This interaction is independent of protein lactylation. In vivo studies confirmed that lactate-driven NHEJ promotes chemoresistance in ovarian cancer. Conclusions: Our findings reveal lactate as a novel metabolic regulator of the NHEJ pathway by directly allosterically modulating the XRCC4–LIG4 complex. This work establishes a direct molecular link between the Warburg effect and DNA repair-driven chemoresistance, offering new insights into potential therapeutic strategies for ovarian cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85e9b949622bf51d02f40bafe00f73cb3235d18e" target='_blank'>
              Lactate Enhances Non-Homologous End Joining Repair and Chemoresistance Through Facilitating XRCC4–LIG4 Complex Assembly in Ovarian Cancer
              </a>
            </td>
          <td>
            Jingyi Lu, Jiayu Zhu, Huanxiao Zhang, Zhou Zhou, Haoyuan Li, Cuimiao Zheng, Xi Huang, Siqi Chen, Chaoyun Pan, Jie Li, Hao Tan
          </td>
          <td>2025-11-30</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="One of the most important changes in the transformation of normal cells into tumor cells is metabolism. In order to satisfy the more active proliferation, migration and metastasis of cancer cells, abnormal changes occur in various pathways and molecules involved in metabolism, which eventually lead to metabolic reprogramming of tumor cells. This process involves the uptake of nutrients and changes in major metabolic forms. As an important part of post-transcriptional epigenetics, RNA methylation modifications can regulate RNA processing and metabolism, while dynamically and reversibly influencing the expression of specific molecules, thereby ultimately affecting diverse biological processes and cellular phenotypes. In this review, various types of RNA methylation modifications involved in cancer are summarized. Subsequently, we systematically elucidate the mechanism of RNA modification for metabolic reprogramming in cancer, including glucose, lipid, amino acid and mitochondrial metabolism. Most importantly, we discuss in depth the clinical significance of RNA modification in metabolic targeted therapy and immunotherapy from mechanism to therapeutic application.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12e3fd9d374b6d8826edc1b140acc672b5b30712" target='_blank'>
              RNA Methylation in Cancer Metabolism: from Mechanisms to Therapeutic Opportunities
              </a>
            </td>
          <td>
            Zeyu Wu, Yuncan Xing, S. Mei, T. Xiao, Fangze Wei, Qian Liu
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Abstract Extrachromosomal DNA (ecDNA) amplification represents an emerging mechanism underlying oncogene amplification, tumor heterogeneity, and drug resistance in cancer. However, the biology of ecDNA remains poorly understood because tools to engineer ecDNAs and precisely monitor their dynamics are limited. In particular, genome engineering strategies have not been established for ecDNA, which exists in tens to hundreds of copies within a single cell. Here, we report a systematic validation of ecDNA editing using standard CRISPR-Cas9 system and optimized CRISPR-Cas9 system with safeguard single-guide RNAs (sgRNAs), in which the addition of cytosine extensions finely reduces excessive Cas9 activity. The conventional CRISPR-Cas9 system induced severe cytotoxicity and markedly reduced ecDNA copy number, together with frequent micronucleus formation. Knock-in efficiency was remarkably low, highlighting an intrinsic difficulty in editing ecDNA. In contrast, the safeguard sgRNA strategy not only alleviated cytotoxicity and ecDNA loss in a cytosine-length–dependent manner but also enabled efficient knock-in into multiple ecDNA per cell. Computational simulations suggested that the degree and temporal patterns of multiple DNA cleavage events shape cell death, micronucleus formation, and rapid expansion of knock-in ecDNA. Collectively, optimization of Cas9 activity using safeguard sgRNAs enables efficient and nondisruptive ecDNA engineering, providing a powerful tool to study ecDNA biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85607d9d15aeb32494567c03b21204776a014e41" target='_blank'>
              Optimized CRISPR-Cas9 system for efficient engineering of ecDNA in cancer cells
              </a>
            </td>
          <td>
            Yohei Sugimoto, Takeru Kachi, Yu Watanabe, Mei Kubokawa, Koichi Ogami, Masaki Kawamata, Seiko Yoshino, Hiroshi I. Suzuki
          </td>
          <td>2026-01-14</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Genome-wide association studies (GWAS) have identified over 60 autosomal risk loci associated with clear cell renal cell carcinoma (ccRCC), yet the functional mechanisms underlying these associations remain largely unclear. To establish connections between risk variants and their target genes, we applied the activity-by-contact (ABC) model, which integrates epigenomic data and Micro-C interactions, complemented with renal-specific quantitative trait loci, to predict enhancer-gene relationships. Our analyses implicate variation in hypoxia sensing, cell cycle regulation, and telomerase maintenance pathways as central mediators of ccRCC risk. These findings provide new insights into the molecular basis of ccRCC susceptibility and highlight potential therapeutic avenues for prevention and treatment. Integration of genome-wide association data with epigenomic and chromatin interaction data identifies target genes mediating clear cell renal cell carcinoma risk, implicating disrupted hypoxia signalling, cell cycle, and telomerase regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e26c65e87cfb98c7e3f6c3d6a8a7f0754996d3cf" target='_blank'>
              Deciphering genetic susceptibility to clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            Maria Mandelia, Philip J. Law, Charlie Mills, M. Went, J. Vijayakrishnan, R. Houlston
          </td>
          <td>2025-12-01</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80b7464169a3f17594b10023232756dfe06a0b90" target='_blank'>
              The causes and consequences of human-specific DNA methylation
              </a>
            </td>
          <td>
            Zhenzhen Ma, Alexander L. Starr, David Gokhman, Hunter B. Fraser
          </td>
          <td>2026-01-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c863981147c795d5bd3bdf0f0373efc730fa086e" target='_blank'>
              Genetic identification of the RAS proteostatic machinery and its failure to regulate oncogenic variants
              </a>
            </td>
          <td>
            J. Bigenzahn, F. Kartnig, Manuela Vollert, V. Sedlyarov, Giulio Superti-Furga
          </td>
          <td>2025-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83abedd58342641982b4fa326716bab2f81320dd" target='_blank'>
              Coordinate Up-regulation of Both TK1 and TS Expression in Cycling Cells: Mechanistic Basis and Implications for Dual-Targeted Cancer Therapy
              </a>
            </td>
          <td>
            Wanxing Wang, Jiawei Zhang, Guangyou Duan, Yiyao Zhang, Guofeng Xu, Zhi Cheng, Shunmei Chen, Xue Yao, Xin Li, Shan Gao
          </td>
          <td>2026-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Termination of DNA replication is a surprisingly complex process that contributes critically to genome stability and cell viability. And even though progress was made to establish the consequences that arise if termination is going awry, the precise molecular mechanisms of fork fusion events and the coordination with key factors that ensure that DNA replication is brought to a successful conclusion remain poorly understood. We therefore investigated replication termination in Escherichia coli, focusing specifically on the interplay between replication fork fusions and genomic stability, the Tus–ter replication fork trap, and key DNA-processing enzymes. By utilizing whole genome sequencing, immunoblotting, and recombination reporter assays, we demonstrate that local hyper-recombination is induced wherever forks meet and that the combined loss of factors such as RecG helicase and 3′ exonucleases causes extreme over-replication in the terminus region of the chromosome. Unexpectedly, cells lacking Tus exhibit elevated R-loop levels, revealing an unanticipated connection between the fork trap and R-loop metabolism. These findings underscore the complexity of replication termination and its central role in maintaining bacterial genome stability, while providing mechanistic insights with implications for understanding replication termination in more complex organisms and developing new antimicrobial strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7313e7e0c5ea82dfcde902877c06fc46c8baafab" target='_blank'>
              Termination of DNA replication drives genomic instability via multiple mechanisms
              </a>
            </td>
          <td>
            Daniel J. Goodall, J. U. Dimude, M. A. Hashemloo, Emma L Dunbar, Iren Grigoryan, Amy L. Upton, E. Bolt, Christian J. Rudolph
          </td>
          <td>2026-01-14</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Abstract Cancer development is driven by somatic evolution and clonal selection. However, traditional selective pressure analysis methods have treated all sites within a gene equally, such a gene-level model oversimplifies the complexity of cancer evolution. In this study, we introduced CN/CS-calculator, a novel site-specific method that can capture selective pressures acting across different gene sites. By deciphering the interplay between the selection pattern and the function of a gene in oncogenesis, CN/CS-calculator uncovers a unique class of mini-driver genes, which exhibit weak positive selection, with certain critical sites providing context-dependent promoter effects on the fitness of cancer subclones while others are constrained by evolutionary conservation. Our method emphasizes the importance of site-specific analysis in uncovering how subtle evolutionary forces shape cancer biology. The refined understanding offers new insights into the mechanisms of cancer heterogeneity and molecular evolution, with potential implications for advancing therapeutic strategies and prognostic assessments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f1b33838c929960cc9e2d6ebf3f9c1d9939a9da" target='_blank'>
              Identification of cancer mini-drivers by deciphering selective landscape in the cancer genome
              </a>
            </td>
          <td>
            Xunuo Zhu, Wenyi Zhao, Siqi Wang, Jingwen Yang, Jingqi Zhou, Binbin Zhou, Ji Cao, Bo Yang, Zhan Zhou, Xun Gu
          </td>
          <td>2026-01-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Most mammalian genes undergo alternative splicing. The splicing of some exons has been acquired or lost in specific mammalian lineages, but differences in splicing within the human population are poorly understood. Using GTEx tissue transcriptomes from 838 individuals, we identified 57,271 “naturally variable exons” (NVEs) – exons which are included in mRNAs in some individuals but entirely excluded from others (or vice versa). NVEs impact three quarters of protein-coding genes, occur at all population frequencies, and are often absent from reference annotations. NVEs are more abundant in genes depleted of genetic loss-of-function mutations and aid in the interpretation of causal genetic variants. Genetic variants modulate the splicing of many NVEs, and 5’ untranslated region and coding-region NVEs are often associated with increased and decreased gene expression, respectively. Together, our findings characterize abundant splicing variation in the human population, with implications for a range of human genetic analyses. Most human genes undergo alternative splicing, but population-level variation is poorly understood. Here, the authors map over 57,000 naturally variable exons across 838 individuals, revealing widespread splicing diversity and its impact on gene expression and genetic interpretation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02230641f54b7e0d8d49af8da5398b8bdcf445b3" target='_blank'>
              Widespread naturally variable human exons aid genetic interpretation
              </a>
            </td>
          <td>
            Hannah N Jacobs, Bram L Gorissen, Jérémy Guez, M. Kanai, Kavi Gupta, H. Finucane, K. Karczewski, Christopher B. Burge
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Background Sequence divergence within gene regulatory elements has been proposed to play an important role in the evolution of human-specific traits, including cortical expansion. However, the mutational processes that efficiently modify gene regulatory elements and the target genes upon which they act are poorly understood. We investigated the regulatory function and origins of the fastest evolved regions in the human genome, termed Human Ancestor Quickly Evolved Regions (HAQERs), in their native genomic context during human cerebral cortex development. Results We identified 50 HAQERs with accessible chromatin in developing human cortex, largely arising from previously unconstrained ancestral sequences. To test the necessity of these HAQERs for gene regulation, we established an all-in-one CRISPRi lentiviral vector and linked 26 HAQERs to nearby target genes across cell types and Wnt pathway activation contexts. Rapid gains of CpGs distinguished HAQERs active during cortical development and displaying human-specific epigenomic marks. As a high density of CpG sites can drive formation of permissive chromatin, we identified 107 HAQERs with at least 17 human-specific CpG gains per kb, termed HAQER CpG Beacons. These HAQERs emerged via contributions from GC-biased gene conversion (gBGC) with evidence for selection preferentially fixing CpG sites. Notably, the CHL1 and DPP10 loci, both implicated in human neurological disorders, each harbored two gene-linked HAQER CpG Beacons. Conclusions Our findings reveal HAQER target genes and support a model where gBGC and natural selection jointly drive regulatory-altering CpG variants to fixation, forging regulatory innovations in the human cortex.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71e64bca00a8a0022a427effe630eac24ac36a2d" target='_blank'>
              Interrogating the Regulatory Function of HAQERs during Human Cortical Development
              </a>
            </td>
          <td>
            Yashodara Abeykoon, Natalie Dzikowski, Yanting Luo, Enakshi Sinniah, Seth Weaver, Bryan J. Pavlovic, Jenelle L. Wallace, Riley J. Mangan, Craig B. Lowe, Alex A. Pollen
          </td>
          <td>2025-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a0f98bd86315e8e5aa3b48daf84b94980e7dd61" target='_blank'>
              Dynamic Supercoiling Sponsors Transcription Amplification by MYC
              </a>
            </td>
          <td>
            R. Jha, Fedor Kouzine, Bo Wang, James D. Phelan, Subhendu K Das, Brian A Lewis, D. Levens
          </td>
          <td>2026-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="


 Soft tissue sarcomas (STS) represent a paradigm of cancer evolution, where spatial and temporal heterogeneity creates dynamic ecosystems that rapidly adapt to therapeutic pressures. Through multi-region multi-omics profiling of 45 patients across primary, irradiated, and recurrent tumors, we reconstructed evolutionary trajectories using an integrated analytical framework. DNA methylation (RRBS) was processed via Bismark, with CAMDAC deconvolution distinguishing cell type-specific signals from copy number alterations. Transcriptomic data were analyzed using Salmon and edgeR, while copy number landscapes were inferred through TitanCNA and InferCNV. Evolutionary dynamics were reconstructed using PRISM, enabling methylation-based phyloepigenome reconstruction to trace subclonal epigenetic evolution. Our analysis reveals that STS evolution follows a branching phylogenetic pattern, with early “trunk” alterations establishing the foundational epigenome and subsequent “branch” events driving radiation adaptation. PRISM-based methylation phylogenies demonstrated conserved evolutionary trajectories across patients, with epigenetic divergence increasing through treatment. At diagnosis, widespread methylome remodeling systematically rewires cellular identity, with convergent hypermethylation-mediated silencing of tumor suppressive networks (SMARCA4, DICER1, FOXO3) and hypomethylation-driven activation of oncogenic pathways (CCND1/2, CDK6). Across patients, these events were recurrent and enriched for functional gene expression changes, representing positive selection for survival advantages. Under radiotherapy selection pressure, the evolutionary dynamics shift from broad remodeling to targeted adaptation. The epigenome stabilizes, focusing on DNA repair (DDB2), stemness (ZBTB16), and proliferation (CCND1) pathways—evidence of directional selection for resistance mechanisms. Single-cell phylogenetics of >150,000 cells reveals that radiation selects for pre-existing subclones rather than generating novel populations, with overall ecosystem architecture maintained despite cellular turnover. Spatial-temporal mapping demonstrates that evolutionary trajectories vary dramatically both between patients and within individual tumors, generating multi-focal ecosystems where distinct subclones coexist, compete, and occasionally converge on shared adaptive solutions. This evolutionary framework explains radiotherapy resistance as the inevitable outcome of selection acting upon pre-existing heterogeneity. The convergent emergence of DNA repair enhancement, stem-like properties, and immunosuppressive niches across patients represents parallel evolution under common selective pressures. Together, these findings argue for the development of evolution-informed therapeutic strategies that anticipate and intercept adaptive trajectories, with the potential to overcome the formidable challenge of heterogeneity in treatment-resistant STS.



 Shaghayegh Soudi, Nadia Silvia, Ajay Subramanian, Serey Nouth, Faith Ryu, Taryn Kaneko, Christin New, Deborah Kenney, Raffi Avedian, Robert Steffner, David Mohler, Anusha Kalbasi, Matt van de Rijn, Gregory Charville, Everett Moding. Decoding the evolutionary landscape of soft tissue sarcomas: From multiregion origins to therapy-driven adaptation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B018.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9dbc7c928e0902865ecc73b6cf6d1b8b370047f" target='_blank'>
              Abstract B018: Decoding the evolutionary landscape of soft tissue sarcomas: From multiregion origins to therapy-driven adaptation
              </a>
            </td>
          <td>
            Shaghayegh Soudi, Nadia Silvia, Ajay Subramanian, Serey C L Nouth, Faith Ryu, Taryn Kaneko, Christin New, Deborah Kenney, R. Avedian, R. Steffner, David Mohler, A. Kalbasi, Matt van de Rijn, Gregory Charville, E. Moding
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="The tumor suppressor RAD51D is essential for homologous recombination (HR). Pathogenic variants in RAD51D are associated with breast and ovarian cancers. However, most clinical missense variants are of unknown significance. We performed a multiplex assay of variant effect to test 6,888 RAD51D coding variants for loss-of-function. The resulting variant-to-function map perfectly separates known pathogenic and benign variants and is validated by orthogonal HR and biochemical assays across 70 clinical variants. Our screen shows that variants in the DNA-binding or ATPase core most severely compromise HR, and we identify the RAD51D-RAD51C interface within the BCDX2 complex as essential for regulating its ATPase activity. We hypothesize that, paradoxically, the primary function of RAD51D is to slow the ATPase activity of BCDX2, thereby allowing sufficient time and space for RAD51 filament assembly. Together, we identify hotspots of deleterious RAD51D variants and uncover the mechanisms by which variants compromise its biochemical functions. Highlights Used a multiplexed assay of functional effect (MAVE) to assess the functionality via olaparib sensitivity of 6,888 RAD51D coding variants, which can be used for variant classification Provided cellular functional analysis for 70 clinically-identified breast and ovarian cancer RAD51D variants Identified key regions and enzymatic activities of RAD51D critical for its function in the BCDX2 and the X3CDX2 complexes Determined mechanism of RAD51D-mediated regulation of BCDX2 ATPase activity">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b147f7b191823354d4eee4bfd6f57b8ab44617ab" target='_blank'>
              High-throughput mapping of 6,888 RAD51D variants identifies distinct biochemical functions needed for homologous recombination and olaparib response
              </a>
            </td>
          <td>
            Kristie E. Darrah, Shelby L. Hemker, Yashpal Rawal, Noah J. Goff, Phoebe S. Parker, Gayatri Ganesan, Caleb M. Stratton, Katherine Oppenheimer, Ella Roberts, Elena Glick, Nicole Banks, Arjun Kumar, Silvia Casadei, Matthew W Snyder, Katherine L Nathanson, S. Domchek, Lea M. Starita, Shaun K. Olsen, Patrick Sung, J. Kitzman, Kara A. Bernstein
          </td>
          <td>2026-01-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>105</td>
        </tr>

        <tr id="Ovarian cancers have been the most lethal gynecologic malignancies worldwide, with survival outcomes remaining poor despite therapeutic advances. Since TP53 is the most frequently mutated gene in ovarian cancers, it highlights the central role of p53 dysfunction in promoting tumor initiation, genomic instability, and progression. Beyond classical loss of transcriptional activity, mutant p53 may exert dominant‐negative effects on residual wild‐type protein and acquire oncogenic gain‐of‐function properties that promote invasion, metastasis, and resistance to chemotherapy. Nevertheless, despite extensive efforts, effective therapeutic agents targeting TP53‐mutant tumors remain elusive, representing a major unmet medical need.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ae95b37ac19ede6887e4eb26c6f0ef7fba06778" target='_blank'>
              Novel Approaches for Detecting TP53 Mutations and Targeted Therapeutic Strategies: Emerging Insights Into Cytoplasmic p53 Aggregates in Ovarian Cancer
              </a>
            </td>
          <td>
            Naoyuki Iwahashi, Kazuhiko Ino
          </td>
          <td>2025-11-30</td>
          <td>Journal of Obstetrics and Gynaecology Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Break-induced replication (BIR) is a primary homologous recombination pathway for repairing one-ended double-strand DNA breaks, including those arising from collapsed replication forks and eroded telomeres. BIR frequently leads to loss of heterozygosity, genetic mutations, and gross chromosomal rearrangements, all hallmarks of cancer. Here, we conducted a genome-wide screen that allowed us to identify and validate the involvement of 33 novel yeast genes in BIR. We report that, while DNA damage and spindle checkpoint machineries are both required to delay nuclear division and provide adequate time for BIR to complete, the spindle position checkpoint is required to coordinate between nuclear division and cytokinesis. Furthermore, we show that two nucleopore proteins play a sequential role during BIR: Nup84 acts before DNA synthesis, while Nup188 functions later to support repair completion. Given the conservation of BIR between yeast and humans and the role of BIR in cancer development, human homologs of the identified BIR proteins may represent promising targets for anti-cancer therapeutics. Break-induced replication (BIR) repairs broken DNA but can also destabilize genomes. The authors identify 33 new genes controlling BIR completion, showing that spindle assembly and spindle positioning checkpoints coordinate repair, and that nuclear pore proteins regulate BIR at multiple steps.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db012d324e0626e6663caa36aedc14163842b5a8" target='_blank'>
              Genome-wide screen reveals dependence of break induced replication on several distinct checkpoints
              </a>
            </td>
          <td>
            Liping Liu, Rosemary S. Lee, Jerzy M Twarowski, Timothy Emagbetere, Jessie Thomas, Jacob M Wells, Gabriel J Seuferer, K. Lobachev, A. Malkova
          </td>
          <td>2025-12-15</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f0d81edb495273502e26a305788d8fbd3e8ea5a" target='_blank'>
              Quantitative profiling of millions of nucleotides reveals sequence-encoded interactions that govern plasmid propagation
              </a>
            </td>
          <td>
            Tom Copeman, John H. C. Fong, Joshua Mayne, Thomas E. Gorochowski, Conde S Rodriguez, Tom Ellis, Francesca Ceroni
          </td>
          <td>2025-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Uveal melanoma (UM) is the most common intraocular tumor, and despite being rare, it accounts for nearly 13% of melanoma-related deaths. Indeed, patients with metastatic disease have typically survival rates of less than one year, with little improvement over the past few decades. Although TP53 mutations are uncommon in UM, recent findings highlight a dysfunctional p53 pathway in this cancer. Given its crucial role in mediating DNA damage responses, we analyzed the p53 protein functionality and downstream target activation in a panel of UM cell lines in response to standard-of-care treatments (i.e., cisplatin and proton-beam irradiation). Although most of the analyzed cells retained a wild-type p53, we observed a wide range of p53 protein stabilization and targets’ activation. Recently, p53 isoforms have been recognized as modifiers of p53 activity, and their biology and functions depend on cellular context. We observed that UM cells express a broad spectrum of p53 isoforms, including Δ160p53α and Δ133p53β and the longer variants Δ40p53β and p53β. Interestingly, the down-regulation of the short p53 isoforms (Δ133/Δ160) revealed their contribution to promoting cell growth and in mitigating cell death triggered by standard-of-care therapies. Moreover, we verified the wild-type p53 status in a panel of 32 UM cases and analyzed the expression levels of p53 isoforms. Our results indicated a correlation between higher expression levels of Δ40p53α or Δ133p53γ isoforms and the development of more aggressive cancers. Our findings suggest that shorter p53 isoforms can promote cancer aggressiveness and therapy resistance, thereby providing crucial insights into UM pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a90351c01a5b0b8d4397664241c06d4691042c1" target='_blank'>
              Exploring p53 isoforms: unraveling heterogeneous p53 tumor suppressor functionality in uveal melanoma
              </a>
            </td>
          <td>
            Laura Bartolomei, Y. Ciribilli, Samuele Brugnara, Francesco Reggiani, Gian Mario Moretta, Mariangela Petito, Elisa Marcaccini, Marianna Ambrosio, Carlo Mosci, Ulrich Pfeffer, A. Amaro, P. Monti, Alessandra Bisio
          </td>
          <td>2025-12-05</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e732ad2e253dbdf86a5a9ac4cf431aea3406d6e9" target='_blank'>
              Genome wide analysis of YBX1-mediated redistribution of JMJD6 in ER+ breast cancer cells
              </a>
            </td>
          <td>
            Aritra Gupta, Siddharth Bhardwaj, K. Desai
          </td>
          <td>2026-01-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Ovarian cancer remains one of the most lethal malignancies of the female reproductive system, with its high mortality rate largely driven by chemotherapy resistance and disease recurrence. Ovarian cancer stem cells (OCSCs), a small subpopulation within ovarian tumors, are characterized by their capacity for self-renewal, differentiation, and tumorigenic growth. They are recognized as central drivers of tumor initiation, metastasis, drug resistance, and relapse. Mounting evidence in recent years has highlighted the pivotal role of metabolic reprogramming in sustaining OCSC stemness and therapeutic resistance. In this review, we reported the major metabolic pathways engaged by cancer stem cells (CSCs), including glucose metabolism (glycolysis, the tricarboxylic acid cycle, oxidative phosphorylation, and reactive oxygen species regulation), lipid metabolism, and amino acid metabolism. These pathways function to meet the bioenergetic and biosynthetic requirements of CSCs. Particular emphasis is placed on the metabolic plasticity of OCSCs, which can transform between a relatively inactive quiescent state and a highly proliferative active state. This adaptability allows OCSCs to respond dynamically to microenvironmental changes, facilitate ovarian cancer implantation and metastasis, and evade chemotherapeutic stress. We further analyze the molecular networks governing OCSC metabolism, including key signaling cascades and transcription factors. From a therapeutic perspective, we discuss the anti-diabetic drug metformin, which has demonstrated potential in targeting CSC metabolism in both preclinical models and clinical studies. Finally, we outline future research directions aimed at exploiting the metabolic vulnerabilities of OCSCs. We highlight that combination strategies targeting metabolism hold significant potential for overcoming treatment resistance and preventing ovarian cancer recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a98ef6b05ff0d2a035206d3941ef2e78eccd31c9" target='_blank'>
              The metabolic landscape of ovarian cancer stem cells: how do they survive?
              </a>
            </td>
          <td>
            Jixue Tan, Lin Tang, Qian Zhang
          </td>
          <td>2026-01-16</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/213457505483d2de87291e80a391cdd8fe7c3867" target='_blank'>
              Linking somatic mutations in cancer to the electronic properties of DNA
              </a>
            </td>
          <td>
            Benoît de Witte, Cyril Karamaoun, Pauline Hermans, Maxime Tarabichi, F. Pucci, Marianne Rooman
          </td>
          <td>2026-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Viral transcriptional regulators (vTRs) reprogram host gene regulatory networks to promote replication, persistence, and immune evasion. Despite the identification of hundreds of vTRs in human viruses, how they rewire host pathways remains unclear. Here, we systematically profiled 95 vTRs from diverse human viruses across multiple functional assays. vTRs perturb immune, cell proliferation/death, and signaling pathways through various mechanisms; some bind DNA directly, others cooperate or antagonize human transcription factors (hTFs), and some remodel chromatin. vTRs can act as activators or repressors and recruit similar but not identical repertoires of proteins as hTFs. These findings reveal vTRs as versatile transcriptional modulators that converge on conserved host “pressure points” while diversifying across pathways to promote viral replication and persistence. Notably, many vTR dysregulate genes within autoimmune, neurological, and cardiovascular risk loci, revealing mechanistic links to disease. Together, we provide a comprehensive resource for understanding and targeting viral control of human transcription.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6deb8278c1abd256d3cb1730399ddf6f6f200174" target='_blank'>
              Viral transcriptional regulators extensively rewire host pathways through diverse mechanisms
              </a>
            </td>
          <td>
            Jaice T Rottenberg, Xing Liu, A. Berenson, L. Soto-Ugaldi, Mohamed Y. ElSadec, Clarissa S Santoso, James E. Corban, Phillip J. Dexheimer, Berkay Engin, Ryan Lane, Sakshi Shah, Kerstin Spirohn-Fitzgerald, Shubham Khetan, Cheng-Che Lee, George Muñoz-Esquivel, Zhaorong Li, Lucia Martinez-Cuesta, Yunwei Lu, Philipp Trollmann, Tong Hao, S. Yi, Nidhi Sahni, M. Bulyk, M. Calderwood, M. Weirauch, Marc Vidal, Srivatsan Raman, J. F. Fuxman Bass
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="


 Cancer cells adapt to environmental and therapeutic pressures through karyotypic plasticity, including whole genome doubling, aneuploidy, and structural genome changes. Polyploid/polyaneuploid giant cancer cells (PGCCs/PACCs) represent an extreme state of this plasticity, arising through endoreplication as a common response to chemotherapy, irradiation, viral infection, hypoxia, or nutrient restriction. PGCCs act as drug-tolerant reservoirs that can later depolyploidize to repopulate tumors, underscoring their role in recurrence and therapy failure. Yet, how readily tumor cells access the PGCC state and how karyotype dynamics sculpt drug sensitivity remain poorly understood. To address this gap, we integrated (i) long-term evolution experiments (LTEEs), (ii) mathematical modeling, and (iii) computational analysis into a unified framework to define karyotype evolution as a predictive biomarker of therapy response. To study karyotype evolution under metabolic stress, we developed CLONEID, an LTEE platform that combines bright-field imaging, single-cell karyotyping and a neutral-drift framework distinguishing drift from selection. Its computer-vision module classifies PGCC versus proliferative states while PCA of other morphometrics revealed stress-specific phenotypes even without whole-genome doubling or overt ploidy change. LTEEs (>6 months) under glucose deprivation, phosphate restriction or hypoxia revealed stress- and ploidy-dependent trajectories. Glucose deprivation promoted whole-genome doubling whereas phosphate restriction and hypoxia drove chromosome loss toward near-diploid states. Strikingly, our in vivo studies confirmed hypoxia-induced ploidy reduction, which was predicted by LTEE trajectories within the 36 days and recapitulated by our model. Moreover, long-term adaptations also shifted therapy response. Glucose-deprived cultures gained resistance to gemcitabine and topotecan while phosphate-deprived cultures became more sensitive to taxanes and carboplatin. Furthermore, our functional assays showed that evolved cultures enhanced T-cell activation while control condition suppressed immunogenicity through tryptophan metabolism. To complement LTEEs, we built an ODE linear chain trick (LCT) model of the cell cycle, calibrated with isogenic diploid and tetraploid TNBC lines. Our model successfully captured ploidy-conditioned therapeutic responses. Specifically, gemcitabine, among 74 tested anticancer agents, eliminated near-diploid cells, but spared tetraploids that entered PGCC state and resumed proliferation after drug withdrawal. Overall, our findings identified that whole-genome doubling, polyploidy, and PGCC formation shape both long-term evolution and acute therapy responses. Integrating LTEEs with LCT modeling successfully links ploidy, karyotypic adaptation, and drug sensitivity. Our approach establishes the ground for “Drug–Karyotype” pairs as biomarkers and shows how metabolic niches sculpt KFLs, offering a path toward personalized, evolution-informed oncology.



 Vural Tagal, Jackson Cole, Richard J. Beck, Didem Ilter, Daria Myroshnychenko, Konstantin Martin, Andriy Marusyk, Ana P. Gomes, Noemi Andor. Modeling karyotype-driven adaptations to metabolic restrictions predicts therapeutic response and immunogenicity in cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B026.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06671e4b962908638a954d43578035d059b71aa3" target='_blank'>
              Abstract B026: Modeling karyotype-driven adaptations to metabolic restrictions predicts therapeutic response and immunogenicity in cancer
              </a>
            </td>
          <td>
            Vural Tagal, Jackson Cole, Richard J Beck, Didem Ilter, D. Myroshnychenko, Konstantin Martin, A. Marusyk, Ana P. Gomes, Noemi Andor
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8b75da29edbdbad60fea18ff4b3049824f5017d" target='_blank'>
              Critical role for a high-plasticity cell state in lung cancer.
              </a>
            </td>
          <td>
            Jason E Chan, Chun-Hao Pan, Jonathan Rub, Gary Guzman, Klavdija Krause, Emma Brown, Zeda Zhang, Hannah Styers, Griffin Hartmann, Zhuxuan Li, Xueqian Zhuang, Scott W. Lowe, D. Betel, Yan Yan, Tuomas Tammela
          </td>
          <td>2026-01-21</td>
          <td>Nature</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Cancer driver mutations alone are often insufficient to fully explain tumorigenesis. We demonstrate that these mutations cooperate with somatic copy number variations (CNVs) in a tissue-specific pattern of genomic epistasis. Analyzing 93,462 tumors, we identified 54 gene-cancer type pairs with significant co-occurrence of somatic mutations and CNVs. Our new Binoculars algorithm, which resolved phased DNA/RNA reads, revealed frequent preferential amplification in oncogenic mutation alleles, including AKT1 p.E17K, BRAF p.V600E, KRAS p.G12C/D/V, NRAS p.Q61K, and a fraction of gain-of-function TP53 p.R175H. Conversely, deletions selectively targeted the reference alleles, leading to loss of heterozygosity of IDH1 p.R132H and tumor suppressor mutations, including CDKN2A and TP53 truncations. Lung cancer patients carrying co-occurrences of somatic mutation-CNVs in TP53 and KRAS showed poorer survival than those carrying the same gene mutations. These findings reveal epistasis of cancer mutations and CNVs at an allelic resolution, suggesting specific genomic events to enhance patient stratification and therapeutic targeting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c67a7d5e6bdabbcb4e758c260b7827b1f0b38f4" target='_blank'>
              Widespread Epistasis between Cancer Driver Mutations and Allele-Specific Copy Number Variations
              </a>
            </td>
          <td>
            Serge Merzliakov, Guanlan Dong, Andrea Castro, Mahad Bihie, Yve Nichols-Evans, Hanna Carter, Kuan-lin Huang
          </td>
          <td>2025-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7ef5b0807e681dc02a817ada526d9c5fc0ddd59" target='_blank'>
              RegEvol: detection of directional selection in regulatory sequences through phenotypic predictions and phenotype-to-fitness functions
              </a>
            </td>
          <td>
            Alexandre Laverré, T. Latrille, Marc Robinson-Rechavi
          </td>
          <td>2025-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed06b707f53d2e1de85e352e5c2852fcf93afdb7" target='_blank'>
              Multiplexed CRISPR/Cas9 Editing of Tumor Suppressor Genes Recapitulates Molecular and Morphological Features of High-Risk Endometrial Cancer
              </a>
            </td>
          <td>
            Maria Vidal-Sabanés, R. Navaridas, Núria Bonifaci, Ada Gay-Rua, Damià Ortega-Peinado, Joaquim Egea, Mario Encinas, Xavier Matias-Guiu, David Llobet-Navás, Xavier Dolcet
          </td>
          <td>2025-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3de429a7817141d17bab7346b3911bc733e9f80d" target='_blank'>
              Integrated methylome analysis identifies an epigenetically silenced tumor suppressor RASA4 in small cell lung cancer.
              </a>
            </td>
          <td>
            Meng Fu, Qizhi Zhu, Jian Qi, Xiaopeng Zhang, Zhipeng Wang, Jialiang Wang, Xiang Huang, Zongtao Hu, Wei Wang, Bo Hong, Hongzhi Wang
          </td>
          <td>2025-12-31</td>
          <td>Communications biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3e23ac93cedc319985d92525602e934cc95a0f6" target='_blank'>
              Whole-genome profiling of native 5-hydroxymethylation in human neurons with long-read sequencing
              </a>
            </td>
          <td>
            Dennis Klose, Mohammad Hossein Sepehri, Remi-André Olsen, Ha Vu, Jason Ernst, L. Kular, Maria Needhamsen, Maja Jagodic
          </td>
          <td>2026-01-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b6af067476c9f99776b29fefd89874e4b024b1b" target='_blank'>
              Comparative genomics of Tandem Repeat variation in apes
              </a>
            </td>
          <td>
            Carolina L. Adam, Joana L. Rocha, Peter H. Sudmant, R. Rohlfs
          </td>
          <td>2026-01-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eef0494e51b8128b410f9d5b4dc00c303aff4796" target='_blank'>
              Mapping lineage and functional diversity in the high-risk human mammary epithelium
              </a>
            </td>
          <td>
            Matthew Waas, Bowen Zhang, Meinusha Govindarajan, Pirashaanthy Tharmapalan, Abhijith Kuttanamkuzhi, Olivia Drummond Guy, Michael Woolman, H. Berman, Paul D. Waterhouse, Rama Khokha, T. Kislinger
          </td>
          <td>2026-01-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Human germline gene expression is normally constrained to the germ cells, responsible for the production of sperm and oocytes. Cancer-germline (CG) genes, a subset of germline genes involved in testis development, are frequently aberrantly activated in cancer cells. The present study investigates the broader hypothesis that epigenetic modifications, specifically DNA methylation, can modulate the expression profiles of several CG genes in cancer and germ cells. Breast cancer (BC), normal breast (NB), and chronic myelogenous leukemia (CML) cell lines were treated with the DNA methyltransferase inhibitor (DNMTi) 5-aza-2’-deoxycytidine for three days. The effects of this treatment on the transcriptional activation of CG genes (SYCP1, ADAD1, SYCE1, PRSS54, DMRTC2, and TEX101) were then evaluated. We comprehensively analyzed differential methylation, survival analysis (Kaplan-Meier), correlation (Spearman’s), and pathway enrichment analysis (GO/KEGG) of CG genes (SYCP1, ADAD1, SYCE1, PRSS54, DMRTC2, and TEX101) in BC and leukemia. Treatment with 5-aza-2’-deoxycytidine upregulated CG genes in BC cells but downregulated them in leukemia cells, highlighting tissue-specific epigenetic responses. Differential methylation analysis revealed cancer-specific patterns: ADAD1 was hypermethylated in both malignancies, while PRSS54 was hypomethylated in leukemia. Survival analysis linked SYCE1 and PRSS54 to prolonged survival in BC, whereas TEX101 and SYCP1 correlated with poorer outcomes. Functional enrichment identified ADAD1 and SYCP1 as key players in BC and leukemia pathways, respectively. Meta-analysis validated SYCP1 as a robust biomarker with consistent effect sizes across datasets. Methylation-expression correlations were stronger in tumors, with SYCE1 and DMRTC2 showing inverse relationships in leukemia. These findings demonstrate that the expression of a subset of CG genes is responsive to modulation by hypomethylating drugs in a tissue-specific manner, highlighting their promise as candidates for future investigation in cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef103bd1b026e3599a6151bd43c1dbb8d8e4301f" target='_blank'>
              Contrasting effects of DNA demethylation on cancer-germline gene expression in breast cancer and leukemia cells
              </a>
            </td>
          <td>
            M. Almutairi, T. M. Alrubie
          </td>
          <td>2025-12-18</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Simple Summary Heritable breast cancer (BC) is influenced by well-known high-penetrance genes such as BRCA1, BRCA2, PALB2, and CHEK2, but the contribution of many moderate- and low-penetrance genes remains unclear. This study used large, multi-ethnic genomic datasets, including the UK Biobank and FinnGen, to refine the set of genes associated with BC predisposition. By integrating multiple association approaches (GWAS, TWAS, and PWAS) and removing likely false positives, we reported on 38 high-confidence BC predisposition genes. These include established DNA repair genes and new candidate genes supported by independent evidence. PWAS also highlighted genes with potential recessive effects that are often missed by standard GWAS. Findings were most consistent in European ancestry populations. Overall, this work provides a conservative, gene-focused framework to clarify biological pathways and prioritize candidate genes for further functional study.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58f8ebf2ad2ff2b0876594246e484e2180346625" target='_blank'>
              Integrative Gene-Centric Analysis Reveals Cellular Pathways Associated with Heritable Breast Cancer Predisposition
              </a>
            </td>
          <td>
            R. Zucker, S. Schreiber, A. Stern, M. Linial
          </td>
          <td>2025-12-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Resistance to single-agent therapy in acute myeloid leukemia (AML) often arises from compensatory epigenetic circuits. We present a causal inference framework that treats therapy as an intervention (do-operator) and defines resistance as the deviation of dual perturbations from the additive expectation of monotherapies. By modeling the causal structure of the leukemia stem-cell epigenetic landscape that maintains identity and fitness, the framework shows that synergistic and antagonistic interactions can be inferred directly from single perturbations, suggesting interaction inference may be simpler than previously assumed. We applied this model to Perturb-seq of 16 epigenetic regulators in KMT2A-rearranged AML (>31,000 single-cell transcriptomes). From single-knockout profiles, the framework correctly predicted synergistic pairs (Menin+KAT6A, Menin+DOT1L) that enhanced differentiation and induced cell death, as well as an antagonistic pair (DOT1L + PCGF1) that conferred resistance. Predictions were validated by pharmacologic inhibition and bulk RNA-seq, confirming the model’s predictive accuracy. This work presents a mechanism-guided approach for rational prioritization of epigenetic drug combinations in AML and other cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/057014eca0b4705ea6ee698244a083b962878940" target='_blank'>
              Causal interaction inference of compensatory structures from single-cell perturb-seq in AML
              </a>
            </td>
          <td>
            Changde Cheng, Sajesan Aryal, Brittany Curtiss, Xinyue Zhou, Rui Lu
          </td>
          <td>2025-12-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Breast cancer progression is increasingly linked to disturbances in circadian rhythm genes, although the underlying molecular mechanisms remain poorly understood. Circadian rhythm genes help maintain normal biological processes and their disruption contributes to breast cancer development. Transcriptomic data from breast cancer (MCF-7) and normal breast (MCF-10A) cell lines from the GSE76370 dataset were analyzed using the limma R package to identify differentially expressed genes. Functional enrichment and network analyses using GO, KEGG, STRING and Cytoscape revealed 1,788 DEGs, including 1,008 upregulated genes involved in DNA replication, chromatin remodeling and PI3K-Akt signaling and 780 downregulated genes associated with cell adhesion and apoptosis. Disrupted expression of core circadian genes (BMAL1, CLOCK and PER3) and hub genes such as ACTB, GAPDH and CDK1 suggests that circadian gene dysregulation promotes breast cancer progression and represents a potential therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f54cf298483ced6cca5212ac65f53b37db09bc31" target='_blank'>
              Disrupted circadian control promotes oncogenesis in breast cancer
              </a>
            </td>
          <td>
            Y. Fatima, Mohammad Kashif, Prashant Ankur Jain
          </td>
          <td>2025-12-15</td>
          <td>Bioinformation</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95c71eacb567a19cb84e29c03543a7fbd15b5975" target='_blank'>
              Negative Selection Maintains Grossly Altered but Broadly Stable Karyotypes in Metastatic Colorectal Cancer.
              </a>
            </td>
          <td>
            William Cross, S. Nowinski, George D. Cresswell, M. Mossner, Abhirup Banerjee, B. Lu, Marc J Williams, G. Vlachogiannis, Laura J Gay, Ann-Marie Baker, C. Kimberley, F. J. Whiting, Hayley L. Belnoue-Davis, Pierre Martinez, Maria Traki, Viola Walther, Kane Smith, J. Fernández-Mateos, Erika Yara-Romero, Erica A Oliveira, Salvatore Milite, G. Caravagna, Chela T James, G. Elia, A. Berner, Chang-Ho Ryan Choi, Pradeep Ramagiri, Ritika Chauhan, Nik Matthews, Jamie Murphy, Anthony Antoniou, Susan K Clark, M. Mitchison, J. Chin Aleong, Enric Domingo, Inma Spiteri, S. McDonald, Darryl Shibata, M. Laclé, L. Wang, Morgan Moorghen, Ian P.M. Tomlinson, Marco Novelli, Marnix Jansen, A. Watson, Nicola Valeri, Nicholas A. Wright, John Bridgewater, M. Rodriguez-Justo, C. Barnes, Hemant M Kocher, Simon J. Leedham, A. Sottoriva, Trevor A. Graham
          </td>
          <td>2026-01-21</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Introduction Prostate cancer (PC) is one of the most prevalent malignancies in men, with rising incidence and mortality rates globally. Despite advances in therapeutic options such as androgen deprivation therapy and chemotherapy, effective cures, especially for advanced stages of the disease, remain limited. Recent research has highlighted the significant roles of alternative splicing (AS) and noncoding RNAs in tumor progression and drug resistance. This study aims to investigate the role of circIMP3, derived from the IMP3 gene, in prostate cancer development. Methods In this study, we employed quantitative PCR, RNA sequencing, and immunoblotting to identify and characterize circIMP3 in prostate cancer tissues and patient blood samples. Functional assays, including cell proliferation and in vivo tumorigenicity assays, were conducted to assess the biological role of circIMP3 in PC cells. RNA immunoprecipitation sequencing (RIP-seq) was used to identify alternative splicing events regulated by circIMP3. Additionally, exosome isolation and uptake assays were performed to explore the paracrine signaling function of circIMP3 within the tumor microenvironment (TME). Results We identified circIMP3, which is significantly upregulated in both prostate cancer tissues and peripheral blood of patients. CircIMP3 contains an internal ribosome entry site (IRES) and encodes a previously uncharacterized 288-amino-acid protein, circIMP3_288aa. Functional assays revealed that circIMP3_288aa promotes cell proliferation in vitro and accelerates tumor growth in vivo. Mechanistically, circIMP3_ 288aa regulates the alternative splicing of FBXW7, leading to impaired c-Myc ubiquitination and stabilization, which enhances oncogenic signaling. RIP-seq analysis identified over 2,000 alternative splicing events regulated by IMP3, with a notable enrichment in pathways related to ubiquitin-mediated proteolysis. Furthermore, circIMP3 is secreted into the TME via exosomes, where it is taken up by recipient cells, contributing to their proliferation. Discussion Our findings demonstrate that circIMP3 acts as a key regulator of both intracellular alternative splicing and extracellular paracrine signaling within the TME. The ability of circIMP3 to influence FBXW7 splicing and stabilize c-Myc provides a mechanistic basis for its role in promoting oncogenesis in prostate cancer. Clinically, high expression levels of circIMP3 correlate with poorer event-free survival in prostate cancer patients, suggesting its potential as a prognostic biomarker. Additionally, the detection of circIMP3 in peripheral blood positions it as a promising target for liquid biopsy applications in PC diagnosis and monitoring.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c44898723fc23dea0100abadafda89694a83449" target='_blank'>
              A novel protein encoded by circIMP3 promotes prostate cancer progression by regulating alternative splicing and tumor microenvironment
              </a>
            </td>
          <td>
            Wenren Zuo, Weizhou Huang, Haojie Chen, Yan Xu, Yang Zhang
          </td>
          <td>2026-01-05</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Malignant tumors developing in the conditions of chronic hypoxia, unlike normal tissues, quickly acquire resistance to decreased oxygen level. Hypoxia-resistant cells in malignant disease obtain various features promoting their survival: they alter metabolism to aerobic glycolysis, activate angiogenesis-stimulating programs, and rearrange signaling cascades to adapt to hypoxia. At the same time, tumor cells become resistant to chemotherapy and radiation therapy and effectively colonize metastatic niches. 
This review analyzes signaling pathways which activate in the conditions of chronic hypoxia and underlie tumor adaptation to insufficient oxygen. Apart from the main pathway associated with activation of hypoxia-inducible factor 1α (HIF-1α), such cascades as STAT3, Snail cascade associated with epithelial-mesenchymal transition, NRF2 factor responsible for adaptation to reactive oxygen species, and stemness factors OCT4, SOX2 and NANOG are also considered. Effective killing of tumor cells requires simultaneous inhibition of several parts of signaling pathways because disruption of just one will lead to switching of signaling proteins to bypass the blocked one. The article lists various inhibitors of signaling pathways patricipating in tumor cell adaptation to chronic hypoxia and describes their mechanisms of action and targets. Development of successful strategies for treatment of hypoxia-resistant tumors requires identification of an effective combination of the above-mentioned and other inhibitors of the key signaling cascades participating in adaptation to hypoxia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21710e96836bbca03c72f6e2b505d714f5b359ca" target='_blank'>
              Rearrangement of signaling pathways and adaptation of tumor cells to hypoxia
              </a>
            </td>
          <td>
            O. E. Andreeva, E. I. Mikhaevich, M. Krasil'nikov
          </td>
          <td>2025-12-14</td>
          <td>Advances in Molecular Oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Abstract Clustered regularly interspaced short palindromic repeats (CRISPR)–Cas (CRISPR-associated) adaptive immune systems provide sequence-specific mechanisms for targeting foreign DNA or RNA and have been widely used in genome editing and DNA detection. Type V CRISPR–Cas systems are characterized by a single RNA-guided RuvC domain-containing effector, Cas12. Here, through comprehensive mining of large-scale genomic and metagenomic data from microbial sources, we identified a new Class 2 CRISPR–Cas effector superfamily, designated Casδ, comprising three members with protein sizes ranging from 867 to 936 amino acids. Biochemical analyses revealed that Casδ-1 functions as a single RNA-guided endonuclease with specific recognition of 5′-RYR-3′ protospacer-adjacent motifs, where R represents A or G, and Y represents T or C. Casδ-1 exhibits robust double-stranded DNA cleavage activity and target-dependent trans-cleavage activity. Casδ-1 mediates efficient genome editing across species, achieving up to 60% indel rates in human cells while generating homozygous knockout lines in two agriculturally important monocot species (Oryza sativa and Zea mays) through stable transformation. Structural and evolutionary analyses reveal Casδ as an evolutionary transitional nuclease bridging Cas12n and canonical type V systems, featuring a C-terminal loop that is essential for activity. Collectively, Casδ is an evolutionarily distinct, compact (<1000 aa), tracrRNA-free CRISPR system enabling versatile cross-kingdom genome editing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d212dfa8931b032b8e112c79ba75e9ae204a8f50" target='_blank'>
              Casδ, an evolutionary transitional CRISPR system enables efficient genome editing across animals and plants
              </a>
            </td>
          <td>
            Zhijia Yang, Meixia Yu, Peiyang Li, Zhuoyang Li, Yunpeng Teng, Yueheng Zhou, Meiqi Zhao, Chang Liu, Zilong Zhao, Zijian Wang, Jianrui Li, Yilin Jing, Yingnan Li, Haiming Zhao, Weibin Song, Chao Bian, Hainan Zhao, Jian Chen, Beibei Xin, Jinsheng Lai
          </td>
          <td>2025-11-26</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9139b0ede19d5b7c3ea40131744831c88cd696ed" target='_blank'>
              Loss of KDM6A-mediated genomic instability and metabolic reprogramming regulates response to therapeutic perturbations in bladder cancer.
              </a>
            </td>
          <td>
            Pratishtha Singh, R. D’Rozario, Bidisha Chakraborty, Swadhin Meher, D. Raychaudhuri, Aminah J. Tannir, Yang Li, Anurag Majumdar, Jessalyn Hawkins, Yun Xiong, Philip L. Lorenzi, Padmanee Sharma, K. Akdemir, P. Pilié, Abhinav K. Jain, Byron H Lee, Sangeeta Goswami
          </td>
          <td>2026-01-07</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72271f0d34d2f8f24354ade5fddf8b10f1a093fc" target='_blank'>
              Whole-genome landscapes of 1,364 breast cancers.
              </a>
            </td>
          <td>
            Ryul Kim, J. Yu, Joonoh Lim, Brian Baek-Lok Oh, S. Nam, S. Kim, J. Lee, Byung-Joo Chae, Ji-Yeon Kim, Ga Eun Park, Bong Joo Kang, P. Paik, Soo Yeon Bae, C.I. Yoon, Young Joo Lee, D. Kim, K. Shin, Ji Eun Lee, Jun Kang, Ahwon Lee, E. Connolly-Strong, Sangmoon Lee, Bo-Rahm Lee, Yuna Lee, Kijong Yi, Youngoh Kwon, In Hwan Chun, Jung-Woon Park, Jihye Kim, Chahyun Choi, Jong-Yeon Shin, Hyungjung Lee, Minji Kim, H. Park, Ilecheon Jeong, Boram Yi, W. Lee, Jeongmin Lee, Woo-Chan Park, Sung Hun Kim, Yoon-La Choi, J. Lee, Young Seok Ju, Yeon Hee Park
          </td>
          <td>2025-12-03</td>
          <td>Nature</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b30edb27f07cff97af34aacf8166230cf5b8614" target='_blank'>
              Unsupervised Identification of Cancer Attractor States through the Lens of Embryonic Origin and Cancer Hallmarks
              </a>
            </td>
          <td>
            P. Deshpande, Prabal Deb, Ibrahim Al Haddabi, B. Itkin
          </td>
          <td>2026-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Human cells possess a complex network of regulatory systems——the DNA damage repair pathway (DDR), to deal with various DNA damage to prevent the inheritance of deleterious mutations. DDR perform its functions via cell cycle checkpoint activation. Mutations in the DDR gene are also the basis for tumor progression, metastasis, and therapeutic effect to treatments that cause DNA damage. Consequently, DDR has received high attention for its role in the therapy against cancer, and the related experiments have been carried out to further explore. Up to now, PARP inhibitors such as Olaparib, Rucaparib, Niraparib, Talazoparib, etc. have been approved for marketing and clinical use, other DDR inhibitors (DDRi) such as ATMi/ATRi/DNA-PKi, Chk1/2i, and WEE1i have partially entered clinical trials. Synthetic lethality has led to the exploration of combination of DDRi with other anti-tumor therapies, in particular, the combination of DDRi with immunotherapies such as chemoradiotherapy and immune checkpoint inhibitors (ICIs) has achieved potential efficacy. This article reviewed the DDR inhibitors that are currently in development or in clinical therapy, analyzed the limitations and future development prospects, and stated the mechanisms and clinical trials in antitumor and immunotherapy, to provide ideas for the subsequent development of DDRi and enhance the efficacy of antitumor immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9a6e3b57be514835a193a70ee6587199732e497" target='_blank'>
              Research progress of DNA damage repair (DDR) and DDR inhibitors in tumor immunotherapy
              </a>
            </td>
          <td>
            Qi Liu, Chunmei Zhang, Yixuan Gao, Dongmei Feng, Duo Deng, Yun Pan
          </td>
          <td>2025-12-01</td>
          <td>World Journal of Surgical Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Lethal prostate cancer is marked by tumor heterogeneity and resistance to androgen receptor signaling inhibitors (ARSIs). In this study we identify glycolysis as a driver of disease progression and therapy resistance. Using single-sample gene set enrichment analysis (ssGSEA) on the SU2C cohort, we demonstrate that elevated glycolysis activity is associated with poor progression-free and overall survival. The glycolysis-based prognostic score (GLY score) is derived from the HALLMARK_GLYCOLYSIS gene set which includes CLN6, SDHC, B4GALT2, RPE, NANP, and KIF20A, via LASSO-Cox regression. The GLY score effectively stratifies risk in the SU2C and WDCT cohorts, with higher scores predicting worse outcomes and increased SYNE1 mutation frequency. Pan-cancer analysis across TCGA datasets confirm its prognostic value. In vitro, enzalutamide-resistant prostate cancer cell lines exhibit heightened glycolysis, and 2-DG inhibition reverses this effect, restoring drug sensitivity. CLN6 knockdown reduces glycolytic activity and cell proliferation. The GLY score offers robust prognostic value, and CLN6 represents a promising therapeutic target for precision medicine in lethal prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87df0347ea6e869d416c054b5869afecc3690bd8" target='_blank'>
              Glycolysis reprogramming predicts poor prognosis and drives therapy resistance via CLN6 in lethal prostate cancer.
              </a>
            </td>
          <td>
            Zhouda Cai, Jianming Lu, Shanshan Mo, Jipu Liu, Chuanfan Zhong, Yongding Wu, Fen Zou, Jian-Heng Ye, Zhaodong Han, Yuxiang Liang, Le Zhang, Fengping Liu, Weide Zhong
          </td>
          <td>2026-01-01</td>
          <td>Acta biochimica et biophysica Sinica</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7113c729d031131a133b62ffaff3d0e421a576f7" target='_blank'>
              eTRex Reveals Oncogenic Transcriptional Regulatory Programs Across Human Cancers
              </a>
            </td>
          <td>
            Zeyu Lu, Yuqiu Yang, Qiang Zheng, Feng Gao, Lin Xu, Xinlei Wang
          </td>
          <td>2026-01-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Blood cancers such as leukemia, lymphoma, and myeloma remain refractory in many patients due to immune escape, antigen heterogeneity, and therapy‑related toxicities. To address these challenges, we review recent strategies that harness CRISPR‑engineered gut commensals as precision “living therapeutics” to modulate host immunity and directly target malignant clones. We frame this review around three principal themes: (1) mechanistic strategies whereby CRISPR-engineered commensals modulate host immunity and directly antagonize malignant clones; (2) the enabling technologies and delivery/containment platforms, CRISPR variants, phage/LNP delivery, genetic circuits and biocontainment, that make living therapeutics feasible; and (3) translational progress, outstanding technical and safety barriers, and ethical/regulatory challenges that must be addressed for clinical deployment. To illustrate these themes, we discuss three concrete therapeutic modalities: engineered microbial secretion of immunomodulators, targeted delivery of tumor-lytic payloads, and engineered production of anticancer metabolites, and how these are enabled by contemporary CRISPR and synthetic-biology toolkits. Selected preclinical models report substantial antitumor effects, often >60% tumor reduction in rodent studies, and restoration of CAR-T cell function in controlled settings; however, effect sizes vary across models, and human translation remains unproven. We also analyze key technical barriers, strain stability, biocontainment, off‑target effects, and propose solutions, including auxotrophic kill-switches and AI‑guided strain optimization. Finally, we outline future directions, from in situ phage delivery to multi‑omics–driven patient stratification. CRISPR‑microbiome editing represents a paradigm shift in hematologic oncology, offering localized, sustained therapy with reduced systemic toxicity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e49361664a6596c9e706abe9f5db117dba2b21fd" target='_blank'>
              CRISPR-engineered microbiome: living therapeutics revolutionize blood cancer immunotherapy
              </a>
            </td>
          <td>
            Fang Cheng, Hamed Soleimani Samarkhazan, Yeganeh Khazaei
          </td>
          <td>2025-12-13</td>
          <td>NPJ Biofilms and Microbiomes</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b59601a1a811d20ae0b8430613e168e0ec52452b" target='_blank'>
              Multimodal antigenic escape to GPRC5D-targeted T cell engagers in multiple myeloma.
              </a>
            </td>
          <td>
            Holly Lee, Sungwoo Ahn, Gerone A. Gonzales, Noémie Leblay, E. Barakat, Dylan Greening, Pina Colarusso, Johnathan Canton, S. Benaoudia, Elham Hasheminasabgorji, Mansour Poorebrahim, David Jung, M. Truger, Jihong Jeong, C. Riedhammer, H. Einsele, K. M. Kortüm, Jan Eckmann, Jitka Somandin, Sara-Sheena Engel, L. Boise, J. Frenking, Niels Weinhold, Konstantina Taouxi, E. Kastritis, Sheri Skerget, D. Vishwamitra, Yunje Cho, F. Maura, Marc-Steffen Raab, J. Corre, L. Rasche, Hervé Avet-Loiseau, Paola Neri, Nizar J. Bahlis
          </td>
          <td>2026-01-15</td>
          <td>Nature medicine</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="Background/Objectives: The RET (Rearranged during Transfection) gene encodes a receptor tyrosine kinase. RET plays a critical role in embryonic development and postnatal physiology. This review provides a comprehensive overview of RET-associated disorders, focusing on the molecular mechanisms of RET activation, associated clinical phenotypes and therapeutic implications. In addition, we present an updated RET mutation database. Methods: RET mutation database is built through the integration and curation of data from two major RET mutation repositories: the Leiden Open Variation Database (LOVD) and the Cancer Knowledge Base (CKB) as well as information derived from the ClinVar database. Results: To date, 78 pathogenic RET mutations have been identified, among these, 71 (91.0%) are single nucleotide substitutions (missense variants), 2 (2.6%) are deletions, 1 (1.3%) are indels, 2 (2.6%) are nonsense mutations and 1 (1.3%) mutation affecting the introns. A pronounced clustering was observed in exons 10–11, accounting for ~60% of cases, suggesting a potential mutational hotspot with structural or functional relevance. Conclusions: Aberrant RET activation, resulting from activating missense variants, gene fusions, or overexpression, underlies a wide spectrum of human diseases. These include multiple endocrine neoplasia type 2A (MEN2A), medullary thyroid carcinoma (MTC), Hirschsprung disease, and pheochromocytoma. The existence and use of a database classifying variants in the RET gene plays a fundamental role in molecular diagnostics and personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/abf390ac8f6c6ce13e6f1d602040b530d5fbd8fc" target='_blank'>
              RET Gene Alterations in Clinical Practice: A Comprehensive Review and Database Update
              </a>
            </td>
          <td>
            Claudio Ricciardi Tenore, Eugenia Tulli, Alessia Perrucci, Roberto Bertozzi, Ludovica Fortuna, Giulia Maneri, C. Santonocito, Andrea Urbani, M. De Bonis, A. Minucci
          </td>
          <td>2025-12-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background

Immune checkpoint inhibitors have transformed the treatment landscape of non-small cell lung cancer (NSCLC); however, primary and acquired resistance remain major clinical challenges. Immune evasion in NSCLC is increasingly recognized as a complex, dynamic process driven by tumor heterogeneity and adaptive remodeling of the tumor microenvironment. Conventional bulk profiling approaches are limited in resolving this complexity, underscoring the need for higher-resolution methodologies.

Objective
This review aims to synthesize current evidence on how single-cell transcriptomic approaches have advanced the understanding of immune evasion mechanisms in treatment-resistant NSCLC, with a focus on tumor-intrinsic programs, immune cell dysfunction, and intercellular communication networks.

Methods

A structured narrative review with systematic elements was conducted, analyzing peer-reviewed studies employing single-cell RNA sequencing and related single-cell technologies in human NSCLC. Studies were evaluated for their insights into immune escape pathways, resistance to immunotherapy, and translational relevance.

Results

Single-cell transcriptomic analyses reveal profound intratumoral and immune heterogeneity in treatment-resistant NSCLC. Resistant tumors harbor distinct malignant cell states characterized by impaired antigen presentation, dysregulated interferon signaling, and lineage plasticity. Within the immune compartment, hierarchical T-cell exhaustion, depletion of progenitor-like exhausted T cells, and expansion of immunosuppressive myeloid populations—particularly tumor-associated macrophages—emerge as dominant features of resistance. Ligand–receptor interaction analyses further demonstrate coordinated immunosuppressive communication networks that sustain immune escape at a systems level. Several single-cell–defined cellular states correlate with poor response to immune checkpoint blockade and adverse clinical outcomes.

Conclusion

Single-cell transcriptomics has fundamentally reshaped the conceptual framework of immune resistance in NSCLC, redefining it as a multicellular, dynamic ecosystem rather than a single-pathway phenomenon. These insights provide a strong biological rationale for developing biomarker strategies and rational combination therapies that target both tumor-intrinsic adaptations and the immunosuppressive microenvironment. Continued integration of single-cell technologies into translational and clinical research is essential to overcome immunotherapy resistance and improve patient outcomes in NSCLC.

Keywords: Non-small cell lung cancer; Single-cell RNA sequencing; Immune evasion; Immunotherapy resistance; Tumor microenvironment">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7495ff2bd82ff3331af5929ec1f5ca0f1ed50c9" target='_blank'>
              Single-Cell Transcriptomic Mapping of Immune Evasion Pathways in Treatment-Resistant Non-Small Cell Lung Cancer
              </a>
            </td>
          <td>
            Marco Rossi, Giulia Bianchi, Alessandro Conti
          </td>
          <td>2026-01-19</td>
          <td>Advanced Journal of Biomedicine &amp; Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Epigenetic reprogramming is a major contributor to therapy resistance in prostate cancer (PCa). Indeed, chromatin accessibility profiling classified PCa into distinct subtypes, predictive of clinical outcome. Histone modifications influence nucleosome positioning, ultimately dictating chromatin compactness. The histone methyltransferase G9a, frequently upregulated in advanced PCa, catalyzes a repressive histone modification (di-methylation of histone H3 lysine 9) that signals for gene transcriptional repression. CM-272 is a reversible and highly selective inhibitor against G9a activity, and we have previously shown in cell line models of PCa that it depicted strong anti-tumoral effects. However, fundamental aspects of altered transcriptomic programs remain unexplored. Here, we validated, in vivo, the CM-272 effects in advanced PCa, and explored the molecular mechanisms underscoring G9a inhibition-mediated effects. Specifically, the PCa patient-derived xenograft model BM18 was used to assess the mode of action of CM-272 in vivo. Moreover, RNA sequencing of tumor tissues was performed to further explore the underlying gene expression signature. Finally, chromatin immunoprecipitation followed by qPCR was used to investigate the underlying functional consequence of epigenetic targeting. Our results showed that CM-272 significantly delayed BM18 tumor growth, specifically impairing tumor cell G9a activity and proliferation. Transcriptomic profiling revealed 119 significantly upregulated and 99 downregulated genes in response to G9a inhibition. Notably, by integrating our gene expression dataset with published chromatin accessibility data, an increase in the androgen receptor (AR) signature and a concomitant decrease in the neuroendocrine (NE) features were observed in CM-272-treated samples. Additionally, pathway enrichment and master regulator analysis pinpointed both the androgen receptor and TP53 networks as significantly upregulated in treated tissues. In line with the reduction of the NE score, the MYC/MYCN were the most significantly downregulated transcription factors. Moreover, under the androgen pathway, upregulation of AZGP1, STK39, and DPP4 was observed. At the promoter level, a significant reduction in H3K9me2 levels was detected in treated samples, allowing, in turn, an increase in AR binding. Additionally, upon treatment, higher p53 binding was detected at the promoter of the p53-repressed genes AURKA, CDC25B, and CD44, whose expression was downregulated. Overall, targeting G9a with CM-272 might hold promising therapeutic potential for the management of advanced disease by modulating relevant tumoral transcriptional programs associated with core vulnerabilities in PCa.



 Filipa Moreira-Silva, Mark Zimmerli, Panagiotis Chouvardas, Beat Roth, Carmen Jerónimo, Marianna Kruithof-de Julio. G9a blockade uncovers therapeutic vulnerabilities in advanced prostate cancer, leading to tumor growth delay [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A045.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5911786a7c980e0c86fa5660d05b013ec96ea309" target='_blank'>
              Abstract A045: G9a blockade uncovers therapeutic vulnerabilities in advanced prostate cancer, leading to tumor growth delay
              </a>
            </td>
          <td>
            Filipa Moreira-Silva, Mark Zimmerli, P. Chouvardas, Beat Roth, Carmen Jerónimo, Marianna Kruithof-de Julio
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="SMARCA4 deficiency plays a critical role in the oncogenesis of various aggressive tumors that exhibit resistance to conventional chemotherapy and radiotherapy, posing significant challenges to clinical management. The pivotal role of SMARCA4 deficiency in tumorigenesis suggests the need for a paradigm shift from traditional tumor origin-based approaches to novel tumor-agnostic strategies focused on molecular alterations associated with SMARCA4 deficiency. This review explores potential targetable molecular changes and emerging therapeutic strategies for SMARCA4-deficient tumors. Molecular alterations related to SMARCA4 deficiency involve impaired genomic stability, defects in DNA mismatch repair, and elevated tumor mutation burdens, all of which suggest potential sensitivity to immune checkpoint inhibitors (ICIs). Recent studies indicate that combining ICIs with chemotherapy or anti-angiogenic agents as first-line treatments may offer clinical benefits for SMARCA4-deficient tumors. Furthermore, SMARCA4 deficiency epigenetically affects chromatin accessibility, alters the distribution of Polycomb group proteins on chromatin, and modulates histone acetylation, highlighting the potential efficacy of epigenetic regulators such as EZH2 and HDAC inhibitors. In addition, synthetic lethality strategies targeting vulnerabilities in SMARCA4-deficient tumors are promising therapeutic approaches, including inhibitors of SMARCA2, CDK4/6, ATR, CHK1, PARP, and the oxidative phosphorylation pathway. Based on current clinical evidence, ICI-based combination therapies represent the most promising first-line regimens for SMARCA4-deficient tumors. Although a theoretical basis supports the potential of tumor-agnostic therapy as a promising strategy for these tumors, several challenges remain in clinical practice. These include heterogeneous therapeutic responses across tumor types, safety concerns associated with synthetic lethality-based agents, and the absence of any histology-agnostic approved therapy for SMARCA4-deficient tumors. The continued development of novel therapeutics and further large-scale clinical evaluations are essential to overcoming these barriers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6ff8adc9ee03e886c2da7337f12fb44eb799835" target='_blank'>
              Tumor-agnostic therapy: a potential therapeutic approach for SMARCA4-deficient malignancies
              </a>
            </td>
          <td>
            Yang Liu, Zhi-Hui Liu, Qiong Zhang, Yin-Miao Bai, Xiao-Bo Guo, Hong-Chen Ji, Hong-Mei Zhang
          </td>
          <td>2025-12-01</td>
          <td>Therapeutic Advances in Medical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26a70cf29a8102e7c49d2b80d42e52230a68a64c" target='_blank'>
              Integrative multi-omics reveals PARP14 as a key IFNγ-regulated mediator of metastatic progression in Ewing sarcoma
              </a>
            </td>
          <td>
            S. Sánchez-Serra, M. Chicón-Bosch, Núria Martorell-Villanueva, Eduardo Candeal, Ignasi Jarne-Sanz, J. Bech-Serra, D. Giménez-Llorente, Manuel Gris-Lorente, P. Monaco, L. Santana-Viera, Silvia Mateo-Lozano, Jaume Mora, Mario Niepel, Carolina de la Torre Gomez, Rebeca Sanz-Pamplona, Ana Cuadrado, Ana Losada, R. López-Alemany, Ò. M. Tirado
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Pancreatic cancer is a highly aggressive malignancy characterised by its invasive nature and poor therapeutic outcomes. These characteristics are closely associated with its complex biological characteristics and significant heterogeneity. Post-translational modifications (PTMs) have been identified as critical regulatory mechanisms through which cells respond to environmental changes and play a pivotal role in signal transduction. The various types of PTMs and their intricate regulatory mechanisms have a profound influence on multiple stages of pancreatic cancer progression. Research has demonstrated that PTMs modulate protein stability, activity, subcellular localization, and protein–protein interactions. The present review focuses on recent advances in our understanding of PTMs in pancreatic cancer, with a particular emphasis on phosphorylation, ubiquitination, SUMOylation, acetylation, lactylation, and O-GlcNAcylation. This study illuminates the molecular mechanisms and functional regulatory networks of PTMs within the distinctive tumour microenvironment of pancreatic cancer. Moreover, we summarise targeted therapeutic strategies directed at PTMs in pancreatic cancer to provide insights for future research and treatment development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7b6745c40730496d44cbedf3105b4de63f7bcb0" target='_blank'>
              Post-Translational Modifications: Key “Regulators” of Pancreatic Cancer Malignant Phenotype—Advances in Mechanisms and Targeted Therapies
              </a>
            </td>
          <td>
            Yongkang Shi, Renyi Qin, Yiming Li
          </td>
          <td>2025-12-01</td>
          <td>Biomedicines</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Background The human mitochondrial genome (mtDNA) plays a key role in maintaining cellular functions. However, mtDNA is vulnerable to mutations due to the lack of protective histones and low efficiency in DNA repair mechanisms. Among these mutations, the 4977 bp mtDNA common deletion (mtDNA-CD) encompasses twelve essential genes that are essential for oxidative phosphorylation. Aim In previous studies, mtDNA-CD was identified in nasopharyngeal carcinoma (NPC) patient samples. However, bulk sequencing protocols employed in past studies were unable to distinguish whether those mtDNA-CD were specific to the cancer cells or from other non-cancer cell types in the samples. Single-cell RNA sequencing (scRNA-seq) enables the discovery of different cell types within a population and facilitates the identification of cell-type-specific mutation signatures. In this study, we propose to use scRNA-seq data to identify cell-type-specific deletion signatures in NPC samples. Methods We developed a custom computational pipeline to detect large-scale deletions using scRNA-seq data. The pipeline harnesses both clipped read profiles and secondary alignments to overcome the typical limitations of low sequencing depth and uneven read coverage in scRNA-seq data. We tested our pipeline on an NPC data set consisting of nine patients. Results Our result suggests the presence of mtDNA-CD mutations specific to malignant cancer cells in NPC tumour samples. This study proposes the use of scRNA-seq of cell-type-specific mtDNA-CD in NPC samples, offering new insights into the genetic and molecular pathogenesis of cancer and guiding the development of specific biomarkers for NPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb9ee5d5ecd8271f6680610b609d3e9e7cc092eb" target='_blank'>
              Discovery of cell-type-specific mtDNA common deletion in nasopharyngeal carcinoma
              </a>
            </td>
          <td>
            Mun Kay Ho, J. W. Ho
          </td>
          <td>2025-12-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Genetic interactions can reveal gene function and identify cancer-relevant synthetic lethals, but systematic mapping in human cells is constrained by inefficient reagents, vast combinatorial search space, and diversity of cell types. Here, we leverage principles from yeast genetic networks to identify human gene modules enriched for genetic interactions. Using our Cas12a-based In4mer combinatorial knockout platform, we screen all pairwise interactions within receptor tyrosine kinase and DNA damage response modules across eight diverse cancer cell lines. We identify hundreds of unreported synthetic lethals, including a dense network within the protein glycosylation machinery, and confirm that interactions in 2D cell culture are maintained in more physiologically relevant models. Our targeted modules show up to 16-fold enrichment of interaction density, providing a scalable strategy for systematic interaction mapping.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/081a137f8aa4f89541a558bf1cd83f7f679eee7b" target='_blank'>
              Functional modules predict cancer-relevant genetic interactions in mammalian cells
              </a>
            </td>
          <td>
            Chenchu Lin, Veronica Gheorghe, Juihsuan Chou, Sabriyeh Alibai, Subin Kim, N. Anvar, Yixin Xu, Xingdi Ma, Lori L. Wilson, Russell Moser, Christopher J. Kemp, Junjie Chen, Scott Kopetz, Traver Hart
          </td>
          <td>2025-12-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42eab2688ceba2ad3f606cde51c1958eb7bd27fd" target='_blank'>
              Decoding epigenetic and transcriptional landscapes: DNA methylome-transcriptome integration reveals novel drivers in 4NQO-Induced esophageal squamous cell carcinoma mouse model.
              </a>
            </td>
          <td>
            Yanli Qian, Runhua Lin, Zhiwei Liu, Xinxin Zhang, Dongming Lin, Min Su
          </td>
          <td>2025-12-27</td>
          <td>Clinical epigenetics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Malignant transformation of mature cystic teratoma (MTMCT) of the ovary is a rare but aggressive malignancy for which no standardized chemotherapy or effective targeted therapies currently exist. To identify therapeutic vulnerabilities in MTMCT, we performed a genome-wide CRISPR-Cas9 knockout screen using the MTMCT-derived NOSCC1 cell line. Two parallel selective pressures were applied: in vivo tumorigenicity in immunodeficient mice and cisplatin exposure in vitro. From this screen, 67 negatively selected genes were identified, among which SOD1 and NDUFB4 emerged as top candidates based on high basal expression levels and clinical relevance. Integration with spatial transcriptomic data from three independent MTMCT patient tumors further supported the prioritization of these targets. SOD1 was selected for further investigation due to the availability of known pharmacological inhibitors. Both siRNA-mediated knockdown and small-molecule inhibition of SOD1 using LCS-1 significantly suppressed MTMCT cell proliferation in vitro by inducing oxidative stress and impairing cell cycle progression. This antiproliferative effect was reversed by co-treatment with N-acetylcysteine, a reactive oxygen species scavenger. In vivo validation using patient-derived xenograft models demonstrated that oral administration of LCS-1 led to significant tumor growth suppression and increased expression of apoptotic and DNA damage markers, including cleaved caspase-3 and γH2AX. These findings establish SOD1 as a critical vulnerability in MTMCT and provide preclinical evidence supporting redox modulation as a therapeutic strategy for this highly chemoresistant and understudied ovarian cancer subtype. Our integrative approach combining functional genomics, spatial transcriptomics, and pharmacologic validation offers a framework for the discovery of novel targets in rare gynecologic malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e67036000a59a86b2e3dd28f5c317d050f6d45c" target='_blank'>
              In-Tumor CRISPR-Cas9 Knockout Screening and Novel Therapy Development for Malignant Transformation of Ovarian Teratoma.
              </a>
            </td>
          <td>
            S. Tamauchi, Kosuke Yoshida, Xinyuan Wang, Atsushi Nakagawa, Akira Yokoi, N. Yoshikawa, K. Niimi, Yusuke Yamamoto, Hiroaki Kajiyama
          </td>
          <td>2026-01-13</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8bb1ad1f1afa1b592836b21a838884cdc277c82" target='_blank'>
              Targeting EZH2 Oncogenic Splicing: Decoding the Regulatory Network and Antisense Correction
              </a>
            </td>
          <td>
            Md Rafikul Islam, Preeti Nagar, Naomi McNaughton, Shabiha Afroj Heeamoni, Md Mahbub Hasan, Shila Kandel, Panagiotis Tsakiroglou, W. Dalton, Omar Abdel-Wahab, A. Krainer, M. A. Rahman
          </td>
          <td>2026-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>109</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD) is one of the most common cancers and a leading cause of cancer-related mortality worldwide, highlighting the need for novel therapeutic strategies. Proteasome 26S Subunit, Non-ATPase 12 (PSMD12), a component of the proteasomal 19S regulatory particle, is associated with tumorigenesis; however, its role in LUAD remains poorly understood. Integrative bioinformatic analysis of The Cancer Genome Atlas (TCGA) and other publicly available LUAD datasets identified PSMD12 as a candidate driver gene on chromosome 17q, a region frequently amplified in LUAD. Clinicopathological and prognostic analyses revealed that PSMD12 was significantly upregulated in tumor tissues because of DNA copy number gain. High PSMD12 expression was associated with poor prognosis and advanced pathological stages. Gene set enrichment analysis of TCGA LUAD dataset demonstrated that samples with high PSMD12 expression were enriched for cell cycle-related pathways. Using CRISPR-Cas9-mediated PSMD12 knockout and lentivirus-mediated overexpression models, we demonstrated that PSMD12 promoted tumor cell proliferation by accelerating the G2/M cell cycle transition in vitro, and xenograft experiments confirmed its tumor-promoting effect in vivo. Mechanistically, PSMD12 overexpression reduced the ubiquitination of CDK1, a key regulator of mitotic entry. Cycloheximide chase and MG132 assays confirmed that PSMD12 stabilized CDK1 by inhibiting proteasome-mediated degradation. In conclusion, we identified PSMD12 as a novel driver gene and prognostic biomarker of LUAD. PSMD12 promoted LUAD progression by modulating CDK1 ubiquitination and enhancing cell cycle progression. These findings suggest that PSMD12 is a promising molecular target for future LUAD therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cb3bc3b62a546392f132deef63cdce1ec580a2a" target='_blank'>
              PSMD12 Overexpression Promotes Lung Adenocarcinoma Progression via Ubiquitin-Proteasome Pathway Dysregulation.
              </a>
            </td>
          <td>
            Yuya Ono, Hajime Otsu, Takaaki Masuda, K. Kosai, Shohei Shibuta, Kosuke Hirose, Takashi Ofuchi, Yuki Ando, Koto Kawata, Yasuo Tsuda, Yusuke Yonemura, Taro Tobo, T. Takenaka, T. Yoshizumi, Koshi Mimori
          </td>
          <td>2026-01-06</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="CRISPR-based gene editing has become a transformative tool to enhance immune cell therapies. In particular, engineering natural killer (NK) cells with CRISPR/Cas systems has gained traction due to their ability to mediate strong anti-tumor responses in an MHC-unrestricted, non-alloreactive manner. Early trials show the feasibility and safety of allogeneic NK cells, paving the way as scalable “off-the-shelf” products. CRISPR/Cas9 edits genomes by inducing DNA double-strand breaks (DSBs), mainly repaired through non-homologous end joining (NHEJ) or homology-directed repair (HDR). While effective, CRISPR carries risks of off-target (OT) activity that may disrupt essential genes, cause chromosomal rearrangements, or trigger oncogenic changes - posing threats to product integrity and patient safety. These concerns intensify with multiplex editing, where multiple loci are modified to improve function, persistence, and immune evasion. Since unmodified NK cells are typically short-lived, many clinical-stage products are engineered to express IL-15 or related constructs, extending their half-life and amplifying risks associated with unintended changes. This underscores the urgent need for robust safety assessments. In this review, we summarize the current landscape of safety assessment platforms for evaluating gene edited NK cells. We highlight predictive in silico tools, biochemical in vitro assays, and emerging cell-based detection systems to identify and quantify CRISPR-induced OT events. Particular attention is given to their suitability, limitations, and practical use in primary NK cells and multiplex editing strategies. Our aim is to support the design of safe, effective editing workflows for NK cell therapies - ensuring rigor as the field advances rapidly toward clinical application.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33b3e4984629f09ed0d85f8cdda8885ffc85a748" target='_blank'>
              Engineering with care: safety assessment platforms for CRISPR-modified natural killer cells
              </a>
            </td>
          <td>
            Alieh Fazeli, Evelyn Ullrich, Toni Cathomen, Tobias Bexte
          </td>
          <td>2025-12-05</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="(1) Background: The transcription factor p73 exists in multiple isoforms with divergent functions in cancer. While DNp73 promotes stemness, epithelial–mesenchymal transition (EMT), and metastasis, the tumor-suppressive isoform TAp73 can also switch to promoting cancer progression. How isoforms sharing the same DNA-binding domain produce divergent outcomes remains unclear. (2) Methods: Here, we performed CUT&RUN in combination with JASPAR, transcriptomics, proteomics, patient survival and gene expression data to map genome-wide and promoter-associated DNA-binding and coregulatory transcription factor (coTF) profiles of TAp73α and DNp73β in melanoma cells. (3) Results: Systematic screening for motif enrichment in cancer hallmark gene sets revealed TAp73- and DNp73-specific coTF repertoires with distinct functions. We identified a coregulator signature for EMT genes enriched for both isoforms that has tumor context-dependent effects on survival and correlates with unfavorable patient prognosis. Of these EMT-associated coTFs, PATZ1 was validated as a novel direct interactor of DNp73β. (4) Conclusions: Our results provide a comprehensive reference map of p73 isoform-specific binding and coregulator recruitment and establish a workflow to model their influence on cancer reprogramming with implications for AI-based individualized therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51d7357286e4d775a6aeae941e92117c88a7c586" target='_blank'>
              Integrative Multimodal Profiling of TAp73 and DNp73 Reveals Isoform-Specific Transcriptomic Coregulator Landscapes in Cancer Programs
              </a>
            </td>
          <td>
            Steffen Möller, A. Spitschak, Nico Murr, Brigitte M Pützer
          </td>
          <td>2025-12-31</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Nearly half of the human genome consists of transposable elements, among which endogenous retroviruses, remnants of ancient retroviral infections, represent some of the most evolutionarily intriguing due to their paradoxical functional duality. While research has documented functional ERV exaptation in key biological processes, these elements have also been associated with age-related diseases, particularly cancer. This apparent contradiction presents an evolutionary question: why would potentially disruptive elements persist in genomes over evolutionary time? Here we review the complex relationship between ERVs, aging and cancer to address this question. After reviewing the physiological roles of ERVs, we explore how the transcriptional activation of normally repressed ERVs may function as an evolutionary-conserved genomic surveillance system that, when triggered by cellular stressors, generates viral-like nucleic acids and proteins that activate pathways to potentially eliminate cancerous cells. Conversely, we discuss how cancer cells could appropriate ERV expression to distort cellular processes, promoting inflammation and senescence that ultimately facilitate tumor progression. Despite this duality, we advance a novel hypothesis that many ERVs have been exapted in mammalian genomes primarily as defense mechanisms against tumorigenesis. This evolutionary perspective provides a framework for understanding both the persistence of ERVs in our and other mammals’ genomes and their intriguing roles in cancer biology. Moreover, even after tumor development, ERVs can be exploited by immunotherapy due to their canonical function as regulators of the immune response, positioning them as emerging central elements in cancer treatment strategies. This work offers new insights into these endogenous retroviruses’ evolutionary significance and potential applications in cancer therapeutics and diagnostics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47cd2557a8bd78199bb0c2bedd92cbc9a68a2fb0" target='_blank'>
              Endogenous retroviruses in aging and cancer: from genomic defense to oncogenic activation
              </a>
            </td>
          <td>
            Gabriel Arantes dos Santos, Nathália Da Roz D’Alessandre, Gabriela Der Agopian Guardia, Rafael Loch Batista, Pedro A. F. Galante
          </td>
          <td>2025-12-01</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6aef2d09241345ec730cec2612f4e7bed8b0baea" target='_blank'>
              DNA2 variant analysis supports the nuclease activity as a preferred therapeutic target
              </a>
            </td>
          <td>
            Katherine E. Baillie, Sijie Zhang, Veena Mathew, Troy Nations, Balakrishna Koneru, Peter C. Stirling
          </td>
          <td>2026-01-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="The mechanisms underlying cancer cell drug resistance have not yet been fully elucidated. Numerous processes hinder effective tumor treatment, such as cancer cell epigenetic alterations, DNA repair, drug target modification, and epithelial–mesenchymal transition. Recent studies have highlighted the ability of cancer cells to transiently enter the resting phase, namely, the G0 phase of the cell cycle under treatment conditions; this is associated with an elevated level of cell death. Cells in the G0 phase are characterized by a specific phenotype, the exact features of which remain to be fully elucidated. Cell division and cell cycle dynamics are closely associated with cell adhesion, i.e., interaction with the extracellular matrix in normal cells, whereas cancer cells are characterized by the dys-coordination of these processes. However, the precise patterns of cell adhesion alterations in association with the cell cycle remain unclear. Knowledge on the interactions of cells in the G0 phase with the extracellular matrix during the entry, the exit, and the sustenance during the resting phase is limited. At the same time, focal adhesion-mediated molecules may be an attractive goal for cancer cells in the G0 phase, targeting to overcome tumor dissemination and drug resistance. Drug resistance and immune system response evasion are specific features of nonproliferating cancer cells that call for more detailed and accurate characterization of this population of malignant cells to optimize anticancer treatment and control tumor dissemination. The present review represents the recent data on the association of focal adhesion and cancer cell surveillance due to cell cycle alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16149b6070228042071985c6c4e07c2cd03bc235" target='_blank'>
              Focal Adhesion and Cancer Cell Cycle-Associated Resistance
              </a>
            </td>
          <td>
            T. Ruksha, N. Palkina, Ekaterina Z. Lapkina, Kangdong Liu
          </td>
          <td>2025-12-25</td>
          <td>ACS Omega</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Oxidative DNA damage is a major driver of genome instability and human disease. Among the various types of oxidative DNA base lesions, 8-oxo-7,8-dihydroguanine (8oxoG) is particularly prevalent due to guanine’s low oxidation potential and the abundance of guanine-rich (G-rich) sequences across the genome. Structure-forming repeat sequences, which are commonly G-rich, can adopt alternative DNA secondary structures that further expose nucleobases to oxidative damage. The base excision repair (BER) pathway is primarily responsible for the repair of 8oxoG lesions; however, the complex topologies and dynamic conformations formed by these repeat sequences present challenges for complete repair. Inefficient BER within these structures can lead to DNA strand breaks, mutations, and large chromosomal rearrangements, all of which are associated with human disease. Notably, structure-forming repeat sequences are often enriched at regulatory genomic regions, where BER can directly influence processes such as replication and transcription. This review summarizes current insights into BER activity within oxidatively damaged structure-forming repeat sequences and highlights how repair efficiency within these sequences impacts genome stability and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7a9b8ec92f7ce4226378783f87443780b182e66" target='_blank'>
              Base excision repair within structure-forming repeat sequences and its impact on cancer and other diseases
              </a>
            </td>
          <td>
            Carson B Cohen, Millie C Coombes, Christopher P Merlo, Chantal A Kontor, Riaz Meah, Amy M Whitaker
          </td>
          <td>2025-12-01</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) arises from the expansion of a single hematopoietic stem cell harboring somatic mutations that confer growth advantage. Recent studies highlight a substantial heritable component to CH, implicating germline mutations in DNA damage repair (DDR) genes. These genes are essential for maintaining genomic integrity and pathogenic variants in key DDR genes are well-established genetic underpinnings of several hereditary cancer syndromes. This review synthesizes current data linking germline DDR mutations – including ATM, CHEK2, TP53, PPM1D, BRCA1/2, and PARP1 – to CH and the development of myeloid malignancies. Emerging evidence suggests that germline perturbations in DDR pathway contribute to CH, though mechanisms remain incompletely defined. Large scale genome-wide association studies (GWAS) have identified strong associations between ATM and CHEK2 variants and CH. Assessing prevalence and CH risk in individuals with germline TP53 variants presents unique challenges, as distinguishing between somatic and constitutional lesions is often complex and requires careful tissue evaluation. The link between germline BRCA1/2 and CH remains inconclusive, confounded by concurrent diagnosis of solid malignancy and prior exposure to chemoradiation therapy in studied patient populations. Although germline mutations in PPM1D and PARP1 are rare, a potential germline predisposition to CH cannot be excluded. The totality of current evidence suggests that germline DDR pathway mutations not only predispose to well-established solid malignancy syndromes but also to CH, which independently increases the risk of hematologic malignancies. Recognizing germline contributions to CH has broad implications for risk assessment, surveillance strategies, and development of preventive strategies in myeloid neoplasia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7301780c127edf4d26a740d2302af1182d86f99" target='_blank'>
              Impact of Germline DNA Repair Mutations on Clonal Hematopoiesis and Myeloid Neoplasm Development
              </a>
            </td>
          <td>
            Kateryna Fedorov, Leo Y. Luo, Alexander G. Bick, M. Savona
          </td>
          <td>2025-12-01</td>
          <td>Current Hematologic Malignancy Reports</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c517ba39588758cc3e5fedf674d78132e58c9969" target='_blank'>
              Identification of full-length circular RNAs linked with therapy resistance of pediatric cancers
              </a>
            </td>
          <td>
            C. Bessiere, L. Babin, E. Andraos, J. M. Riepl, A. Szymansky, M. Lodrini, H. Deubzer, A. Eggert, C. Quivoron, C. Rigaud, V. Verge, S. Pyronnet, L. Lamant, F. Meggetto, C. Gaspin, S. Fuchs
          </td>
          <td>2025-12-04</td>
          <td>None</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="The intrinsic and acquired resistance of ovarian cancer to conventional platinum/taxane chemotherapy is approximately 80-85%, with a high recurrence rate, making it one of the most lethal gynecological cancers. Epigenetic dysregulation, a key factor in tumor growth and chemoresistance, includes abnormal DNA methylation and 5-hydroxymethylcytosine (5hmC) loss. The ten-eleven translocation (TET) family of dioxygenases (TET1/TET2/TET3) mediates DNA demethylation, causing oxidation of 5-methylcytosine to 5hmC, potentially altering gene expression due to cancer cell plasticity and impacting treatment responses. This review discusses the multiple effects of TETs in ovarian cancer, highlighting the regulation of epithelial mesenchymal transition (EMT), cancer stem cells (CSCs), and the Wnt/β-catenin and TGF-β signaling pathways by TET enzymes. TET1 plays a dual role, promoting chemoresistance via CSC enrichment and suppressing tumors by replenishing Wnt antagonists. TET2, primarily a tumor suppressor, reduces 5hmC; TET2 loss is associated with poor therapeutic results. Elevated expression of TET3, which controls EMT and miRNA expression, is linked to a worse prognosis. In addition, we reviewed the potential resensitization of resistant tumors to multiple modalities of treatment by reactivating/modulating TET activity and function via cofactors and epigenetic treatment. Regulation of the TET-5hmc axis appears promising to overcome chemoresistance and improve therapeutic outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cef73ef5bc008f144d7b42251311b87e823dba94" target='_blank'>
              Targeting TET enzymes in ovarian cancer: epigenetic regulation, chemoresistance, and therapeutic opportunities.
              </a>
            </td>
          <td>
            Padival Shruptha, Jagadeesha Poyya, T. G. Vasudevan, Kapaettu Satyamoorthy
          </td>
          <td>2025-11-30</td>
          <td>Epigenomics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f39b31297f4e813d518282b03522258c7ea79337" target='_blank'>
              Aging promotes escape from X chromosome inactivation in human tissues
              </a>
            </td>
          <td>
            Mélise Edwards, Clarissa Rocca, Zam Cing, Alex R. DeCasien
          </td>
          <td>2025-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4d2c494a1cd657524f51f9577b21d3baa396ea7" target='_blank'>
              Spatiotemporal regulation of cell cycle states within the complex tumor microenvironment
              </a>
            </td>
          <td>
            Gianni Zanardelli, Olivier Tassy, Maulik K. Nariya, Nacho Molina
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Somatic mutations, key alterations in cancer development, exert differential effects across tissues and biological layers, such as transcriptomes, proteomes, and post-translational modifications (PTMs). Although previous pan-cancer studies have characterized the molecular landscape of cancer, the effects of individual somatic mutations across different tissues remain insufficiently explored. Here, we developed Panorama to evaluate the oncogenic potential of single somatic mutations across all cancer types. We collected cancer proteogenomics or multiomics data from over 10 000 individuals across 19 cancer types. Based on five evaluation criteria, we assessed whether a specific mutation affects the abundance of a particular gene’s transcriptome, proteome, or phosphoproteome; the tumor microenvironment; specific RNA- or protein-based signaling pathways; and outlier-level overexpression of PTMs, aiding in potential drug target identification. By leveraging five oncogenic metrics, Panorama quantifies the oncogenic potential of individual somatic mutations and provides a framework for identifying driver mutations by incorporating their downstream effects. With Panorama, researchers can integrate cancer proteogenomics data, providing a comprehensive approach that enhances our understanding of single somatic mutations in specific tissues. Finally, Panorama was developed as a web-based database to ensure easy access for researchers and is freely available at http://139.150.65.64:8080/or https://github.com/prosium/panorama.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3988e7434c0ad5c1c94f629876b4c62c68b08126" target='_blank'>
              Panorama: a database for the oncogenic evaluation of somatic mutations in pan-cancer
              </a>
            </td>
          <td>
            Seung-Jin Park, Seon-Young Kim
          </td>
          <td>2026-01-15</td>
          <td>Database: The Journal of Biological Databases and Curation</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f8fae2bcd154582c2189d775e6e129baa231cb3" target='_blank'>
              A multi-omics analysis of human fibroblasts overexpressing an Alu transposon reveals widespread disruptions in aging-associated pathways.
              </a>
            </td>
          <td>
            Juan I. Bravo, Eyael Tewelde, Christina D. King, J. Bons, Samah Shah, Jacob Rose, Judith Campisi, Birgit Schilling, B. Benayoun
          </td>
          <td>2025-12-11</td>
          <td>GeroScience</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Despite improvements in the molecular profiling of pancreatic neuroendocrine tumors (PanNETs), predicting their clinical behavior and response to specific therapies remains challenging. We sought to elucidate the molecular basis underlying the broad phenotypic variations in these neoplasms through a genetic characterization of primary and metastatic PanNETs. Our findings revealed an enrichment of CDKN2A homozygous deletions and TSC2 somatic mutations in metastatic PanNETs when compared to non-metastatic lesions. Tumor evolution analysis further revealed the acquisition of such genetic alterations as late events in the progression of these neoplasms, conferring poor survival outcomes to the affected patients. Biallelic loss of DNA damage repair genes, ATRX and/or DAXX, was associated with a high fraction of the genome altered in PanNETs, with pathogenic alterations affecting those genes also being associated with a homologous recombination deficiency signature. These findings highlight molecular mechanisms driving PanNET progression and underscore the need for further molecular characterization and tumor evolution studies to evaluate targeted therapies for such a challenging disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db2469c6632fab23ebab1e0e71bc53bea256e7dd" target='_blank'>
              CDKN2A homozygous deletions and TSC2 somatic mutations in metastatic pancreatic neuroendocrine tumors
              </a>
            </td>
          <td>
            Tito Teles Jesus, Lorenzo Ferrando, Lia Rodrigues, R. Martins, Luís Cardoso, José Manuel Lopes, Paula Soares, Arnaud Da Cruz Paula, J. Vinagre
          </td>
          <td>2025-12-05</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32c4664b6580edc429e59cb7fc3bb0cee1ddce94" target='_blank'>
              LISA: A Comprehensive R Package for Lentiviral Integration Site Analysis in Gene Therapy Safety Assessment
              </a>
            </td>
          <td>
            Shuai Ni, Ning Wu, Yun Kang, Fengjiao Zhu, Jueyu Wang, Lina Wang, Wei Wang, Heng Mei, Kejia Kan
          </td>
          <td>2025-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Chromosomal aberrations are frequent in cancer and play a pivotal role in disease initiation and progression. In prostate cancer (PCa), deletions at chromosome 13q14 are common and enriched in advanced disease, suggesting selective advantage in tumor progression. Beyond BRCA2, this region harbors other tumor suppressor genes, including RB1 and RNASEH2B, which impact sensitivity to androgen receptor inhibitors (ARi) and PARP inhibitors (PARPi). This study aims to elucidate the effect of individual vs concomitant loss of RB1 and RNASEH2B in response to dual AR and PARP inhibition.



 To explore the prevalence of deletions in 13q we interrogated copy number alterations in the Hartwig Medical Foundation (HMF) PCa WGS dataset (n=486). In vitro, we generated CRISPR-Cas9-mediated knock-outs (KO) of RB1, RNASEH2B, or both genes in PCa cell lines (LNCaP, C4-2, 22Rv1, DU145). Gene editing was confirmed by Sanger, and western blot analysis validated protein loss. Antitumor activity of enzalutamide (ARi), and olaparib or saruparib (PARPi) was determined using CCK8 kit and colony formation assays.



 RB1 deletions were present in 42% of HMF PCa samples (205/486), being mostly heterozygous deletions (200/205). RB1 deletions co-occurred with RNASEH2B loss in 94% of cases (192/205) and with BRCA2 deletions in 42% of cases (87/205). In vitro, RB1-KO increased resistance to ARi and PARPi in hormone-sensitive lines LNCaP and C4-2, while no change in sensitivity was detected in AR-independent models 22Rv1 and DU145. Loss of RNASEH2B increased sensitivity to PARPi in all models. Next, we generated RB1 and RNASEH2B double KO models in LNCaP and C4-2 by sequentially editing single KO populations. After single-cell seeding, we obtained: (1) RB1-KO (homozygous) clones with complete protein loss; (2) RNASEH2B-heterozygous (RNASEH2B-Het) models with partial protein expression; and (3) RB1-KO/RNASEH2B-Het models. No clones with complete RNASEH2B loss were obtained, suggesting that RNASEH2B-loss population is transient and that clones with complete loss do not proliferate. RB1-KO models maintained resistance to both treatments. RNASEH2B-Het cells showed increased sensitivity to PARPi, although to lower extent than the population with complete RNASEH2B loss. Importantly, introducing RB1-loss in RNASEH2B-Het cells restored PARPi resistance, and combined AR/PARP inhibition did not overcome resistance in RB1-KO or RB1-KO/RNASEH2B-Het cells.



 Our findings indicate that 13q14 deletions in PCa predominantly occur as heterozygous events encompassing key genes influencing ARi and PARPi responses. Heterozygous loss of RNASEH2B confers sensitivity to PARPi, although to a lesser extent than in models with complete RNASEH2B loss. RB1-loss confers higher resistance to ARi and PARPi, restoring resistance to PARPi in RNASEH2B heterozygous models. These results highlight the need for models that better mirror the heterozygous nature of 13q14 deletions to clarify their biological effects and refine therapeutic strategies for advanced PCa.



 Victor Esquefa, Pablo Cresta Morgado, Richard Norris, Lara de Llobet, Julian Brandariz, Sara Arce-Gallego, Teresa Casals, Manuel Ramos, Daniel Aguilar, Gisela Mir, Irene Casanova-Salas, Francisco M. Barriga, Joaquin Mateo. Impact of 13q.14 loss in prostate cancer progression and sensitivity to targeted agents [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr B018.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15430fc4c6c79184273233030845f22a6f4ecd5a" target='_blank'>
              Abstract B018: Impact of 13q.14 loss in prostate cancer progression and sensitivity to targeted agents
              </a>
            </td>
          <td>
            Victor Esquefa, P. C. Morgado, Richard Norris, L. D. De Llobet, Julian Brandariz, Sara Arce-Gallego, Teresa Casals, Manuel Ramos, Daniel Aguilar, Gisela Mir, I. Casanova-Salas, Francisco M. Barriga, Joaquin Mateo
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Most of the disease associated genetic variants identified in genome wide association studies have been mapped to the non-coding regions of the genome. One of the leading mechanisms by which these variants are thought to affect disease susceptibility is by altering transcription factor (TF) binding. Even though inbred mouse strains have been commonly used to investigate polygenic diseases, less is known on how their genetic differences translate to the level of gene regulation and chromatin landscape. Here, we investigated how genetic variation affects chromatin accessibility in the epididymal white adipose tissue (eWAT) of C57BL/6J and 129S1/SvImJ mice, which are commonly used to study diet-induced obesity, fed either chow or high-fat diet. We show that differences in chromatin accessibility are almost exclusively strain-specific and driven by genetic variation. In addition, we integrate ATAC-seq (chromatin accessibility) and H3K27ac ChIP-seq (active regulatory regions) data to show that tissue-specific TF binding sites are commonly found in the active regulatory regions hosting TF motif altering variants in eWAT. Using footprint analysis, we also show that TF occupancy is consistent with TF binding motif scores at the genetically altered loci. In addition, we validate these findings by extending the analysis to ATAC-seq and H3K27ac ChIP-seq data obtained from the liver. We employ RNA-seq to show that differentially expressed genes are co-located with differentially accessible regions hosting genetic variants. Overall, our findings highlight the connection between differential chromatin accessibility and genetic variation across metabolically central tissues of a mouse model for polygenic obesity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/279c0869f46b532b730e2854a8bf3fa9eb563f59" target='_blank'>
              Genetic variation shapes the chromatin accessibility landscape and transcriptional responses in mouse adipose tissue.
              </a>
            </td>
          <td>
            Juho Mononen, Mari Taipale, M. Malinen, A. Levonen, Anna-Kaisa Ruotsalainen, Luke Norton, Sami Heikkinen
          </td>
          <td>2026-01-16</td>
          <td>PLoS genetics</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af7630e9ff15d614dc5c29742755271640ed25f5" target='_blank'>
              KMT2A-Mediated transcriptional regulation in stemness and cancer: molecular mechanisms and therapeutic opportunities
              </a>
            </td>
          <td>
            Md Shiblee, Sadik Sabuj, Tanvir Ahmed, Md Jamilur Rahman, S. M. A. Salam, Byung-Yong Park, Md Rashedunnabi Akanda
          </td>
          <td>2025-12-24</td>
          <td>Medical Oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="


 Somatic structural variations (SVs) are common in cancer. Although a small fraction of SVs in breast and ovarian cancers can be attributed to homologous recombination deficiency, the underlying molecular mechanisms for the vast majority of somatic SVs remain unclear. Here, we focus on the roles of transcription and DNA replication collisions in genomic instability in cancer. Such collisions are unavoidable in cells since both transcription and replication use the same DNA as template. We hypothesized that transcription replication collisions (TRCs), if not properly repaired, would lead to collapsed replication forks and result in SVs. To this end, we studied somatic SVs in 6193 high-coverage whole-genome sequenced primary and metastatic tumors from three independent pan-cancer cohorts. A total of 13 conserved SV signatures, representing independent molecular mechanisms, were deconvoluted from these cohorts using non-negative matrix factorization approach. We detected replicated-strand bias, the expected footprint of transcription-replication collision, in large tandem duplications (TDs) across multiple cohorts. This bias was only observed in expressed genes, consistent with TRCs depending on transcription activity. Large TDs were abundant in female-specific (breast, ovarian and uterus), upper gastric-intestinal tract, and prostate cancers. They were associated with CDK12 mutations and worse patient survival. CDK12 is a cyclin-dependent kinase, a key regulator of transcription elongation and termination. Deleting or suppressing CDK12 using CRISPR-Cas9 in prostate cell lines increased RNA:DNA hybrids (R-loops), promoted TRCs, and ultimately led to large TDs. Finally, we found that cells lacking CDK12 were sensitive to WEE1, CHK1 and ATR inhibitors. In summary, our data suggest that large TDs in cancer form due to impaired TRC repair and can be used as a biomarker for prognosis and treatment.



 Lixing Yang. A078: Transcription and DNA replication collisions lead to large tandem duplications and expose targetable therapeutic vulnerabilities in CDK12-mutant prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A078.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a771b9c6597f1781d70e2b6d5df9ff54196c4d85" target='_blank'>
              Abstract A078: Transcription and DNA replication collisions lead to large tandem duplications and expose targetable therapeutic vulnerabilities in CDK12-mutant prostate cancer
              </a>
            </td>
          <td>
            Lixing Yang
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4773775e6ecfd7cb45043c77dfe433bc00bd2ae5" target='_blank'>
              Epigenetic Intelligence: How Organisms Track Their Environment Through Molecular Memory
              </a>
            </td>
          <td>
            Holly V. Moeller, H. Putnam, Ross Cunning, Steven B. Roberts, J. Eirín-López, R. Nisbet
          </td>
          <td>2026-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Cancer progression is driven by somatic mutations, with alterations in driver genes such as tumor suppressors and oncogenes playing critical roles. In breast cancer (BRCA), mutations in MAP3K1 and MAP2K4 are recurrent, especially in estrogen receptor-positive (ER+) subtypes, yet their functional significance and mechanistic contributions remain incompletely understood. This study aims to elucidate the role of MAP3K1/MAP2K4 mutations in BRCA pathogenesis. We performed integrated genomic analyses using data from The Cancer Genome Atlas (TCGA) and Molecular Taxonomy of Breast Cancer International Consortium (METABRIC) cohorts. Functional validation was conducted in breast cancer cell lines (e.g., MCF-7, ZR-75-1) using shRNA-mediated knockdown, overexpression of dominant-negative MKK4 (MKK4DN), and western blotting. In vivo tumor growth and metastasis were assessed using a xenograft mouse model. Proteomic and phosphoproteomic data from Clinical Proteomic Tumor Analysis Consortium (CPTAC) were analyzed to evaluate JNK pathway activity and FOSL1 expression across multiple cancer types. MAP3K1 and MAP2K4 were identified as frequently mutated in BRCA, with mutation spectra dominated by loss-of-function alterations. These mutations exhibited mutual exclusivity with TP53 alterations and were enriched in ER + tumors. Mechanistically, MAP3K1/MAP2K4 loss led to reduced JNK2 phosphorylation, impaired p53 activation at Ser15, and subsequent upregulation of FOSL1 (encoding FRA1), promoting tumor proliferation and metastasis. In vivo, MKK4DN (dominant-negative MAP2K4) expression enhanced tumor growth and lung metastasis, accompanied by decreased phospho-JNK/p53 and increased FRA1. Pan-cancer analysis revealed that MAP3K1/MAP2K4 mutations compensate for TP53 loss in regulating FOSL1 expression, particularly in tumors with moderate TP53 mutation rates. Our findings establish MAP3K1 and MAP2K4 as key tumor suppressors in BRCA that operate via the JNK2–p53–FOSL1 axis. Their inactivation provides an alternative mechanism for p53 pathway disruption, adhering to the “minimal necessary alteration” principle in cancer signaling. This study highlights the dual regulatory mechanisms controlling FRA1 expression and offers insights into breast cancer heterogeneity, with potential implications for targeted therapy and patient stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc651b8d57047584a3f8e42d25fdec9c87aeefba" target='_blank'>
              MAP3K1/MAP2K4 mutations drive breast cancer progression by compensating for TP53 loss through inactivation of the JNK2-p53-FOSL1 axis
              </a>
            </td>
          <td>
            Sike Hu, Ailing Ji, Manxue Wang, Xia Li, Lili Sun, Ruifang Gao, Ying Zhang
          </td>
          <td>2025-12-16</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc2d1ddc89a7f99d7d73ea18fc63a369a387876b" target='_blank'>
              An optimised method for generation of murine CAR-T cells by CRISPR/Cas9
              </a>
            </td>
          <td>
            T. J. Jackson, Courtney Himsworth, Sophie Munning-Tomes, Farah Alam, Helena Brezovjakova, Laura K. Donovan, Amy K. Erbe, Paul M. Sondel, Louis Chesler, John Anderson
          </td>
          <td>2025-11-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Chromosome instability (CIN) remains among the most important problems in modern cancer research. In this study, we conducted a genome-wide RNAi screen to identify genes that contribute to CIN. To achieve this, we used a human artificial chromosome in a novel sensitized screen to measure CIN. We screened 18,658 genes for their roles in maintaining chromosomal stability and identified 834 candidates as potential CIN genes. A secondary RNAi screen identified 44 genes with the most pronounced CIN phenotypes. In guilt-by-association analysis using a published set of 8,498 proteins across a panel of 949 cancer cell lines, this cohort of 44 genes displayed a striking correlation with mitotic regulators. Furthermore, altered expression of these proteins was associated with a poor prognosis across multiple cancer types. Specifically, downregulation of AMY2B, ALAD, PDGFRA, PPIE, VEZ1, and TTC19 is associated with poor survival in two or more of the following: small cell lung cancer, lung adenocarcinoma, adrenocortical carcinoma, ovarian cancer, and breast cancer. The genes identified in this screen hold potential as prognostic markers for patient survival across several cancer types and could potentially serve as targets for the development of new therapeutic approaches aimed at mitigating CIN.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48402b9da158c79dbc9aec979b9f8a854d30f0b3" target='_blank'>
              A genome-wide RNAi screen for novel CIN genes using human artificial chromosome.
              </a>
            </td>
          <td>
            Mikhail Liskovykh, N. Y. Kochanova, Chih-Yuan Chiang, A. Dhall, V. Aksenova, Yu-Chi Chen, William C Reinhold, M. Dasso, Anish Thomas, K. Cheng, Y. Pommier, William C. Earnshaw, Vladimir Larionov, N. Kouprina
          </td>
          <td>2025-12-01</td>
          <td>PNAS nexus</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d9d9e99d312aee7d3e8959371325610d5ce9af0" target='_blank'>
              MIF-mediated reprogramming of myeloid lineage within the glioma tumor microenvironment impacts the efficacy of immune stimulatory gene therapy
              </a>
            </td>
          <td>
            Ziwen Zhu, Noah C. Kanis, Anthony E. George, Michael Albdewi, Yingxiang Li, Anzar A. Mujeeb, Brandon L. McClellan, Gurveer Singh, Jialin Liu, W. Al-Holou, J. Heth, Joshua D. Welch, Justin D. Lathia, Pedro Lowenstein, Richard Bucala, Maria G. Castro
          </td>
          <td>2026-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Ionizing radiation (IR) is used to treat more than half of cancer patients because it induces DNA double-strand breaks and triggers apoptosis. IR also damages other nucleic acids, lipids, proteins and cellular organelles, initiating additional complex cellular responses. Some of these responses are transient, while others can become permanent and lead to changes in cellular identity. This review focuses on cell fate plasticity, defined as the conversion of one cell type into another, during recovery after IR-induced damage. We recognize that this process likely occurs along a continuum and may be reversible. We will distinguish cell fate plasticity from molecular or phenotypic plasticity, such as epigenetic modifications, transcriptomic shifts, altered signalling, morphological changes or acquisition of migratory behaviour, all of which are clinically relevant but do not constitute a change in cell type for the purposes of this review. Importantly, cell fate plasticity can enable cancer cells to acquire stem-like properties, which has major implications for tumour progression and therapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcae9ab22121364b1772334438f8e401bf8921d7" target='_blank'>
              Radiation-induced cell fate plasticity.
              </a>
            </td>
          <td>
            Michael Shiferaw, T. T. Su
          </td>
          <td>2026-01-14</td>
          <td>Open biology</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aecbc95950c12cbf91340acad8d28e801a22a818" target='_blank'>
              tRNA-derived fragments (tRFs) as key non-coding players in the tumor transcriptome of colorectal cancer: introducing a new suspect responsible for the remaining unknowns of tumor pathology
              </a>
            </td>
          <td>
            Hamid Aria, Behnam Mansoori, Zahra Saadatian, Amin Moradi Hasan-Abad, Roumina Norouzi, Shayan Heidari, Masume Dehghani, Y. Mansoori, A. Daraei
          </td>
          <td>2025-12-01</td>
          <td>Medical Oncology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="The ongoing development of human artificial chromosomes (HACs) will permit investigation into essential centromere processes and the means to deliver large genetic cargoes to target cells. Starting with large (∼750 kb) yeast artificial chromosome (YAC)-based constructs limits the rampant multimerization that has complicated many prior types of HACs. Large YAC construction is accomplished using transformation-associated recombination (TAR) strategies that can become unwieldly when several functional modules are to be incorporated and tested. To address this issue, we developed an approach where modules are built using high-fidelity in vitro assembly strategies in a bacterial artificial chromosome (BAC) format. Then, the assembled modules are transferred in a simplified TAR step into a recipient YAC harboring the prokaryotic “stuffer” DNA that comprises a large portion of the final HAC construct. This approach is highly efficient with two-thirds of all screened yeast clones harboring the correct TAR product. Further, whole-genome Oxford Nanopore Technologies (ONT) sequencing/alignments, de novo assembly of the final YAC using a single ONT sequencing run, and close inspection of highly repetitive regions are all streamlined to rapidly validate clones that match the design. The fully sequenced, verified strain harboring a multi-module construct was then fused to human cells, where it efficiently formed functional HACs upon initial seeding with CENP-A-containing nucleosomes. We envision that the rapid assembly steps will be useful to quickly incorporate different functional modules, including diverse genetic cargoes, to engineer HACs with specific design features.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01c21e293126f24e1c3078b3f559128178306eb7" target='_blank'>
              Rapid assembly of functional modules for generating human artificial chromosome constructs compatible with epigenetic centromere seeding
              </a>
            </td>
          <td>
            Gabriel J. Birchak, Daniel G. Gibson, P. K. Allu, Prakriti Kashyap, John I. Glass, Ben E. Black
          </td>
          <td>2025-10-03</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>8</td>
        </tr>

        <tr id="ABSTRACT Replication stress involves the slowing or stalling of the replication fork as DNA is copied during S phase. This stress can drive genomic instability, a cancer hallmark. RNA:DNA hybrids, such as R-loops and single genome-embedded ribonucleotides, are significant sources of replication stress. RNA:DNA hybrid homeostasis must therefore be tightly regulated through prevention and removal. Ribonuclease H2 (RNase H2) functions both in R-loop removal and excision of single ribonucleotides from genomic DNA. Recent research has generated new mechanistic insights into the functions of RNase H2 in the replication stress response, and implicated both loss and overexpression of RNase H2 in cancer development and therapy response. These findings help generate new models but also raise new questions. This Review explores the contribution of RNA:DNA hybrids to replication stress, the involvement of RNase H2 in regulating these structures, and the emerging roles of RNase H2 in replication stress response and cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3660ad79bb728b1c188f68e1b705105f627767e4" target='_blank'>
              Emerging roles of RNA:DNA hybrid regulation by mammalian ribonuclease H2 in replication stress and cancer
              </a>
            </td>
          <td>
            Rosanna J. Wilkins, E. Petermann
          </td>
          <td>2025-12-01</td>
          <td>Journal of Cell Science</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b370747cb8a1720741d860f50d6f1043c88dbdfc" target='_blank'>
              Cancer stem cell-driven drug resistance in colorectal carcinoma: molecular aspects and therapeutic potentials.
              </a>
            </td>
          <td>
            Minfeng Zhou, Huifang Niu, Dandan Cui, Menghao Xu, Jinxiao Li, Guichen Huang, Minquan Zhou, Chutong Xiong, Yunya Liu, Xiaojuan Xu, Hongxing Zhang, Fengxia Liang, Rui Chen
          </td>
          <td>2026-01-22</td>
          <td>Molecular cancer</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="


 Clonal hematopoiesis (CH) is exemplified by the expansion of genomically altered hematopoietic stem and progenitor cells that may give rise to altered immune repertoires. Driven largely by age and chronic inflammation, CH has been historically linked to myeloid malignancies. Although emerging evidence points to a modest prevalence of CH within solid tumors, the impact of these naturally occurring alterations on anti-tumor immunity and therapeutic responses remains less defined. Here, we employed somatic mutation calling algorithms to detect single nucleotide variants (SNVs) in both epithelial and immune compartments within tumors. Our preliminary analyses of single-cell RNA-sequencing (scRNA-seq) of 32 primary, therapy-naïve prostate tumors encompassing 11 patients identified conventional mutations representative of both CH and prostate cancer. Within the tumor-epithelial compartment, we identified recurrent FOXA1 class 1C mutations in 27% (3/11), alongside NCOR1 mutations in 18% (2/11), IDH1 in 9% (1/11), and SPOP in 9% (1/11) of patients. Of note, these SNVs were detected across multiple samples for the same patient. Interestingly, we also detected germline HOXB13 G84E variants, associated with increased risk for prostate cancer, in 2 cases at a high clonality within epithelial cells (>75%). CH alterations were identified in 27% (3/11) of patients based on the detection of exonic, nonsynonymous SNVs within myeloid cells. Specifically, these variants affected SF3B2 (variant allele frequency; VAF = 0.30), UBE2A (VAF = 0.30), and KMT2C (VAF = 0.43) that are predicted deleterious across multiple classifier algorithms. Patients with detectable CH alterations displayed immunologically hotter tumor microenvironments, with CH+ patients having significantly higher proportions of tumor-infiltrating monocytes (16.22% vs 7.13%, Fisher’s exact test p<0.0001), macrophages (6.49% vs 2.88%, p<0.0001), CD8+ T cells (10.35% vs 3.33%, p<0.0001), CD4+ T cells (2.81% vs 0.74%, p<0.0001), and B cells (1.81% vs 0.59%, p<0.0001), and lower fractions of epithelial cells (50.94% vs 69.81%, p<0.0001). In summary, we leveraged de novo mutational algorithms to identify somatic alterations using scRNA-seq. To our knowledge, our study is the first to detect CH alterations in prostate cancer at the single-cell resolution. Further work will integrate our mutational signatures with transcriptomics to elucidate the impact of both cancer- and immune-intrinsic alterations on anti-tumor immunity. Additionally, we will incorporate mitochondrial DNA-based lineage tracing algorithms on matched peripheral blood, primary, and metastatic tumor samples to deconvolute the spatiotemporal regulation of naturally occurring CH alterations across cancer progression. We envision our findings will illuminate novel drivers of immune function and regulation within prostate cancer that may be exploited for future immunomodulatory therapies.



 John R. Lozada, Christine Luo, Devin A. Schmeck, Jeffrey S. Miller, Akash Patnaik, Emmanuel S. Antonarakis, Frank Cichocki, Justin H. Hwang. Interrogating the Genomic Co-Evolution of the Immune System and Prostate Cancer Along the Time and Treatment Continuum [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A040.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03587209005d8227c89ae44e47e8117ddd350a30" target='_blank'>
              Abstract A040: Interrogating the Genomic Co-Evolution of the Immune System and Prostate Cancer Along the Time and Treatment Continuum
              </a>
            </td>
          <td>
            John R Lozada, Christine Luo, Devin A. Schmeck, Jeffrey S. Miller, Akash Patnaik, Emmanuel S. Antonarakis, Frank Cichocki, Justin H. Hwang
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f2bac6b9fbac768b087852174f053e8622b039f" target='_blank'>
              Targeted editing of pericentromeric satellite DNA alters sensitivity to meiotic drive
              </a>
            </td>
          <td>
            D. Eickbush, J. Rahmat, M. Lindsay, J. Bowers, N. Fuda, A. Larracuente
          </td>
          <td>2026-01-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="XPD is a key nucleotide excision repair (NER) protein whose function is vital for genome integrity. During NER, XPD serves as a 5′−3′ single-strand DNA translocase that enables lesion scanning and verification in genomic DNA. Yet, its translocation mechanism is incompletely understood. Here we use molecular simulations and chain-of-replicas path optimization methods to model the ATP-driven translocation mechanisms of XPD and its bacterial homolog DinG, revealing all on-path metastable intermediates and corresponding kinetic rates. We identify the XPD(DinG) global domain motions that modulate the strength of DNA association at the opposing ends of the DNA-binding groove. During the ATP hydrolysis cycle, alternating weak and strong interactions at two defined groove constrictions enable DNA reptation and forward displacement of the ATPase. Moreover, we show that DNA- or ATP-binding residues directly involved in translocation are hotspots for genetic disease mutations. Thus, our findings shed light on the etiology of XPD-associated genetic syndromes. This computational modelling study unveils the detailed DNA translocation mechanisms of two archetypal SF2 family helicases, XPD and DinG, and sheds light on the functional impact of XPD-associated mutations on genetic disease etiology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/555132c702e8b55f9007aea31f9aea0e94f9efe6" target='_blank'>
              Translocation mechanism of xeroderma pigmentosum group D protein on single-stranded DNA and genetic disease etiology
              </a>
            </td>
          <td>
            Tanmoy Paul, Chunli Yan, Grant Derdeyn-Blackwell, I. Ivanov
          </td>
          <td>2025-11-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e9f0c1aa74fa8dcd40ddabd793fc67f7d7cd6be" target='_blank'>
              ALFA-K: Local adaptive mapping of karyotype fitness landscapes.
              </a>
            </td>
          <td>
            Richard J Beck, Tao Li, Noemi Andor
          </td>
          <td>2025-12-27</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Mutations in DNA mismatch repair (MMR) pathway genes (MSH2, MSH6, MLH1, and PMS2) are linked to acquired resistance to temozolomide (TMZ) and high tumor mutation burden (TMB) in high-grade gliomas (HGG), including glioblastoma (GBM). However, the specific roles of individual MMR genes in the initiation, progression, TMB, microsatellite instability (MSI), and resistance to TMZ in glioma remain unclear. Here, we developed de novo mouse models of germline and somatic MMR-deficient (MMRd) HGG. Surprisingly, loss of Msh2 or Msh6 does not lead to high TMB, MSI, nor confer response to anti-PD-1 in GBM. Similarly, human GBM shows discordance between MMR gene mutations and TMB/MSI.Germline MMRd leads to promoted progression from low-grade to HGG and reduced survival compared to MMR-proficient (MMRp) tumor-bearing mice. This effect is not tumor cell intrinsic but is associated with MMRd in the tumor immune microenvironment, driving immunosuppressive myeloid programs, reduced lymphoid infiltration, and CD8+ T cell exhaustion. Both MMR-reduced (MMRr) and MMRd GBM are resistant to temozolomide (TMZ), unlike MMRp tumors. Our study shows that KL-50, a imidazotetrazine-based DNA targeting agent inducing MMR-independent cross-link-mediated cytotoxicity, was effective against germline and somatic MMRr/MMRd GBM, offering a potential therapy for TMZ-resistant HGG with MMR alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f59eb07f5af2f7c1bb36a92e8c3ca00e2e992ebc" target='_blank'>
              Mismatch repair deficiency drives malignant progression and alters the tumor immune microenvironment in glioblastoma models.
              </a>
            </td>
          <td>
            Montse Puigdelloses Vallcorba, N. Soni, Seung-won Choi, Kavita Rawat, T. Joshi, Sam Friedman, Alice Buonfiglioli, Angelo Angione, Zhihong Chen, Gonzalo Piñero, Gabrielle Price, Mehek Dedhia, Raina Roche, E. Radkevich, Anne M. Bowcock, Deepti Bhatt, Winfried Edelmann, R. Samstein, Timothy E. Richardson, N. Tsankova, A. Tsankov, R. Bindra, Raul Rabadan, Juan C Vasquez, D. Hambardzumyan
          </td>
          <td>2025-12-23</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Targeted insertion (TIN) of transgenic trait cassettes has the potential to reduce timeline and cost for GM product development and commercialization. Precise genome engineering has made remarkable progress over the last several decades, particularly with the use of site-directed nucleases as core editing machinery. However, there are still many critical factors that can impact TIN efficiency including insertion site selection, nuclease optimization and expression, donor vector design, gene delivery, and stable event regeneration. Here, we develop workflows for target site sequence identification and gRNA screening for CRISPR-Cas12a system and demonstrate its successful application for TIN in maize with donor sequences up to 10 kilobase pairs (kb) in size. We first prioritize genomic regions for inserting transgenes in silico using bioinformatics tools and then test gRNA performance using a leaf protoplast transient assay. Despite its known low efficiency, we identify homology-directed repair (HDR) as the preferential pathway for directing targeted insertions of large sequences in immature embryos and demonstrate double-junction integrations at a rate of up to 4%. We further apply a molecular analysis workflow using large amplicon TaqMan assays and nanopore sequencing for streamlined identification and characterization of high-quality insertion events with intact large inserts. Analysis of TIN events across generations suggests that efficiency bottlenecks are not limited to donor targeted integration; attrition in efficiency also results from partial or additional donor insertion, chimerism, and close linkage with undesired sequence insertions such as those encoding the editing machinery. This work represents a major step forward in realizing the potential of precise genome engineering in maize for basic research and biotech trait development applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef8c6345ed05b236093324d1a96c625d7fc33416" target='_blank'>
              Development of efficient targeted insertion mediated by CRISPR-Cas12a and homology-directed repair in maize
              </a>
            </td>
          <td>
            Brenden Barco, Shujie Dong, Yuki Matsuba, Ashley Crook, Ruiji Xu, Yingxiao Zhang, Chengjin Zhang, Ryan Carlin, Kevin Potter, Stephen B. Rigoulot, Jeongmoo Park, Erin M. Seaberry, Allison Parrish, Sivamani Elumalai, Samson Nalapalli, Craig Schuller, Anna Prairie, Anna Mangum, Kangfeng Mei, Hao Wu, Melissa Murray, Kristin Setliff, Francine Johnson, Dawn Mcnamara, Ling Zhu, Mark Rose, W. Gu, Hao Hu, Yuanji Zhang, Yaping Jiang, Wenling Wang, Guozhu Tang, Lizhao Geng, Jianping Xu, Wan Shi, Jason Nichols, Tim Kelliher, Liang Shi, Ian Jepson, Q. Que
          </td>
          <td>2025-12-04</td>
          <td>Frontiers in Genome Editing</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Colorectal cancer (CRC) has a high incidence and mortality rate globally, with approximately 40% of patients harboring Kirsten rat sarcoma viral oncogene homolog (KRAS) mutations. Patients with KRAS mutations exhibit poorer prognoses compared to those with KRAS wild-type tumors, often showing resistance to targeted therapies and chemotherapy. There is an acute need for innovative therapeutic strategies, particularly as KRASG12C inhibitors have been approved by the U.S. Food and Drug Administration (FDA) for second-line therapy in treating non-small cell lung cancer (NSCLC). Yet, their efficacy in CRC remains suboptimal due to the emergence of drug resistance. Current research on KRAS inhibitor resistance in CRC focuses mainly on single biological processes, leaving the complex interplay between cellular systems underexplored. This review synthesizes evidence across four key dimensions — metabolism, epigenetics, immunology, and proteostasis — to reveal multidimensional mechanisms of resistance. How these interconnected pathways work in synergy with signaling aberrations, genetic alterations, and translational modifications to reinforce resistance was investigated. The research also integrates recent advances in multi-targeted strategies, including combinations of KRAS inhibitors with metabolic or epigenetic modulators or immune checkpoint blockade. These findings provide potential strategies for overcoming resistance in KRAS-mutant CRC, addressing a critical gap in precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de09a3f40e67f360dfd11b639fa71c10cdd13842" target='_blank'>
              Unravelling Resistance: Integrating Metabolism, Epigenetics, Immunology, and Proteostasis in Strategies against Kirsten Rat Sarcoma Viral Oncogene Homolog-mutant Colorectal Cancer
              </a>
            </td>
          <td>
            Jingyi Li, Na Song, Rui Ma, Xiujuan Qu
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16c6776876b0ba015273baad473a4e391867ab9a" target='_blank'>
              CRISPR screens identify PRMT7 as a therapeutic target to enhance T cell-mediated killing in breast cancer.
              </a>
            </td>
          <td>
            Wei Shi, Yi Luo, Yizhuo Wang, Jacqueline M. Burrows, Debra Black, Andrew Civitarese, Laura Perlaza-Jiménez, Ping Zhang, Murray Manning, Natasha Tuano, Miguel E Rentería, Christos Xiao, Siok-Keen Tey, Joseph Rosenbluh, Corey Smith, G. Chenevix-Trench, Jonathan Beesley
          </td>
          <td>2026-01-21</td>
          <td>NPJ breast cancer</td>
          <td>0</td>
          <td>110</td>
        </tr>

        <tr id="This paper delves into the role of epigenetics in complementing and challenging Darwinian evolution theory within the field of evolution. Epigenetics has revealed new sources of genetic variation, with epigenetic variations such as DNA methylation and histone modifications capable of influencing gene expression without altering DNA sequences. These variations can spread and accumulate within populations, exhibiting reversibility and dynamism, enabling species to rapidly adapt to environmental changes. Additionally, epigenetics expands the scope of natural selection, revises the patterns of genetic information flow, and highlights the need for updated evolutionary predictive models. Researchers have proposed theoretical integration pathways such as the Extended Evolutionary Synthesis framework, multilevel selection theory, and gene-epigenome coevolution models, offering new perspectives and directions for evolutionary biology research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b730d3e4497777825de2252ebd00338b352de17" target='_blank'>
              Epigenetics Complements and Challenges Darwinian Evolution: From Gene Regulation to the Study of Species Adaptation
              </a>
            </td>
          <td>
            Meichen Zhou
          </td>
          <td>2025-12-19</td>
          <td>MedScien</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) represent a resilient subpopulation within tumors, capable of driving progression, metastasis, and recurrence. One mechanism that enables this plasticity is asymmetric cell division (ACD), a process by which CSCs generate both a self-renewing stem cell and a differentiated daughter cell. While traditionally associated with tissue development, ACD is now recognized as a dynamic and regulated feature of cancer biology, particularly in response to stress conditions such as hypoxia and radiation. In this review, we provide a comprehensive and mechanistic synthesis of how intrinsic factors such as polarity complexes, cell fate determinants (CFDs), spindle orientation cooperate with extrinsic cues from the tumor microenvironment to orchestrate ACD in CSCs. We explore how this process contributes to tumor heterogeneity, therapy resistance, and the emergence of quiescent, drug-tolerant CSCs across multiple malignancies, including brain, breast, colorectal, and hematologic cancers. Importantly, we highlight recent efforts to pharmacologically disrupt or redirect ACD using inhibitors of NOTCH, WNT, AURORA kinases, and MSI1, presenting ACD as a therapeutic vulnerability rather than a static trait. By shifting the focus from CSC markers to division mode, this review introduces a novel conceptual framework for targeting tumor hierarchy and plasticity. Understanding and manipulating ACD offers a promising frontier in precision oncology - one where altering the balance of cell fate decisions could limit relapse, reduce intratumoral complexity, and enhance long-term treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b911960284166687ce9462f6da53a77bba5e7d79" target='_blank'>
              Targeting asymmetric division in cancer stem cells: implications for relapse and resistance
              </a>
            </td>
          <td>
            Mona Latifi, Mahsa Sani, F. Sani
          </td>
          <td>2025-12-01</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Ewing sarcoma is a pediatric bone and soft tissue cancer thought to arise from mesenchymal stem cells (MSCs). It is characterized by fusion of the RNA-binding protein EWS to an ETS family transcription factor, most often FLI1. EWS::FLI1 induces changes in gene expression that have been well-studied and have been linked to its ability to bind to and alter chromatin state at GGAA repeat sequences within the genome. In addition to these effects, EWS::FLI1 alters mRNA splicing; however, the mechanism of this activity remains unclear.



 We curated a sarcoma-focused transcriptome from published long-read RNA sequencing of Ewing sarcoma, osteosarcoma, and synovial sarcoma patient tumors and matched adjacent normal tissue (215M reads). We supplemented this with published long-read RNA sequencing data from relevant normal cell types, osteosarcoma cell lines, and Ewing sarcoma lines with or without EWS::FLI1 knockdown (105M reads). We used this transcriptome as a reference to evaluate expression and splicing from short-read RNA-seq of Ewing sarcoma cell lines, MSCs, and MSC-based models of Ewing sarcoma and established Ewing-associated expression and splicing signatures.



 We generated scores based on these signatures in published knockdown-rescue experiments in which endogenous EWS::FLI1 was silenced by shRNA and replaced with exogenous shRNA-resistant mutants. Expression and splicing scores were highly correlated within these datasets (r2 = 0.89), suggesting that similar regions of the fusion protein might be required for both its functions. In contrast, the scores had limited correlation within an internal dataset of RNA-binding protein perturbations (r2 = 0.02). We next examined the effects of pharmacological agents thought to alter EWS::FLI1 activity. Interestingly, we found that lurbinectedin which disrupts EWS::FLI1’s DNA-binding could partially revert both expression and splicing scores. In contrast, YK-4-279 which disrupts the fusion’s protein-protein interactions and MS0621 which reverts a Ewing-associated chromatin signature had limited effects on these scores. To further interrogate the potential role of EWS::FLI1 DNA-binding in splicing regulation, we silenced EWS::FLI1 in TC-32 cells and replaced it with either the wildtype fusion or a DNA-binding null mutant. Unlike the wildtype construct, the DNA-binding null mutant could not rescue expression or splicing. Finally, we examined the effects of the previously published KRAB-ZFA7 repressor which silences chromatin at GGAA repeats while sparing single GGAA sites and found that it could revert both scores.



 Together, these data establish a potential connection between EWS::FLI1 activity at GGAA repeats and its effects on mRNA splicing. Additionally, we find that lurbinectedin, an FDA-approved agent currently in a phase 1/2 trial for Ewing sarcoma, suppresses both the splicing and expression effects of EWS::FLI1. Emerging compounds designed to target EWS::FLI1 DNA-binding directly will likely have similar effects.



 David V. Allegakoen, Jeffrey A. Toretsky. EWS::FLI1 activity at GGAA repeats is essential for maintenance of Ewing-like splicing [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr B013.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6934f37ed326d496d296a4806fccea6f323a4f63" target='_blank'>
              Abstract B013: EWS::FLI1 activity at GGAA repeats is essential for maintenance of Ewing-like splicing
              </a>
            </td>
          <td>
            D. Allegakoen, J. Toretsky
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="TP53 mutation-driven gene expression programs define oncogenic phenotypes. While extensive studies have concentrated on the transcriptome and proteome, post-transcriptional processes, particularly translational variation, remain underexplored. This study presents a comprehensive analysis of the transcriptomics, translatiomics, and proteomics dynamics in the ovarian cancer cell line SKOV3, with a focus on the effects of p53 missense mutations (R175H, R273H, and Y220C) on gene dosage fluctuations. Despite clear transcriptional differences between wild-type and mutant p53, we find that extensive translational and post-translational buffering processes attenuate these discrepancies, yielding comparatively stable protein abundances. Moreover, we delineate that the relative contributions of transcription output, translation engagement, and protein stability collectively shape the final protein abundance in the context of p53 mutations. Clinical proteomic analysis of platinum-resistant ovarian cancer tissues reveals tumor-specific factors and acquired resistance pathways linked to p53 mutations. Our findings elucidate the multilayered regulatory landscape of p53 mutations and identify potential risk factors for platinum resistance associated with these mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/765c815e9b25f95d1edceba753e2ba1a94641e62" target='_blank'>
              Multilayered Regulatory Dynamics of p53 Mutations and Platinum Resistance in Ovarian Cancer.
              </a>
            </td>
          <td>
            Liling Hu, Hanchen Zou, LvYing Peng, Fan Li, Danya Liu, Jiangli Lu, Yuying Li, Chris Zhiyi Zhang, Qiu-Hong Tian
          </td>
          <td>2025-12-16</td>
          <td>Journal of proteome research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Genome-wide association studies (GWAS) of melanoma have identified numerous susceptibility loci. However, causal genes and variants underlying risk have yet to be established for most. It is becoming apparent that many functional variants underlying complex traits act via cis-regulation that may be context-specific, dependent on availability of specific transcription factors/complexes in specific cell types and cell-states. To characterize a risk locus on chromosome band 2q33.1 associated with melanoma, breast cancer, and keratinocyte cancers, we integrated fine-mapping, cell-type specific expression quantitative trait locus (eQTL) analysis, a massively parallel reporter assay, individual luciferase assays, and SNP-based proteomics. Integrated analysis implicates the presence of multiple functional variants lying primarily within a promoter for CASP8. A haplotype containing rs3769823 appeared have the largest effect on expression. Strikingly, both tumor/normal context and this risk-associated haplotype play critical roles in mediating allelic cis-regulatory activity. Quantitative mass spectrometry for rs3769823 identified both E4F1, a transcriptional repressor, and IRF2, a transcriptional activator, as binding preferentially to risk-associated rs3969823-A. The binding of these transcription factors was validated via EMSA, supershift, and chromatin immunoprecipitation (ChIP) assays. The relative levels of E4F1 and IRF2 differ by cell-type and play a role in mediating transcriptional activity in a cell-type specific manner. Our results indicate that the top credible causal set variant rs3769823 likely influences expression of CASP8 and FLACC1 in a cell-type specific manner and may be a relevant functional variant for multiple cancers associated with this locus.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d0bd03d9d9be45c3f139097e5cecc84ff17cc6a" target='_blank'>
              Functional characterization of a multi-cancer risk locus on chromosome band 2q33.1 near CASP8
              </a>
            </td>
          <td>
            Hyunkyung Kong, Jiyeon Choi, Tongwu Zhang, Cathrin Gräwe, Mai Xu, Rohit Thakur, Hayley A. Sowards, Rebecca Hennessey, Andrew Vu, Jianxin Shi, T. Bishop, Julia Newton-Bishop, Jérémie Nsengimana, M. Iles, M. Landi, M. Vermeulen, M. Law, L. Amundadottir, Melanoma Meta-7 Analysis, Consortium, Kevin M. Brown
          </td>
          <td>2026-01-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Simple Summary ADP-ribosylation is a post-translational modification that plays a crucial role in DNA repair, transcription, replication, and overall cell fitness. The ADP-ribose cycle represents one of the most tightly regulated biochemical pathways in the cell, as precise control of mono- and poly-ADP-ribose levels, together with the coordinated activity of the enzymes that synthesize and recycle ADP-ribose to NAD+, is essential for cellular homeostasis. Moving beyond the traditional cancer framework focused exclusively on oncogenes and tumor suppressor genes, ADP-ribose can be viewed as a central signaling molecule that regulates proteins involved in tumor progression and adaptation to stress conditions within the tumor microenvironment. In this context, proteins whose roles in ADP-ribosylation are relatively less characterized, such as mono-ADP-ribosyltransferases (MARTs) and poly(ADP-ribose) glycohydrolases (PARGs), are emerging as important regulators of these processes. As their functions and biological implications continue to be elucidated, these enzymes should be considered promising prognostic markers and potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90553f7630f85c06dd88b57a27b79fc887c602e0" target='_blank'>
              Targeting MARylation and DePARylation in Cancer Therapy: New Promising Therapeutic Opportunities
              </a>
            </td>
          <td>
            Vanesa Cabeza-Fernández, Francisco Javier Ríos-Sola, D. Martín-Oliva, Jerónimo Borrego-Pérez, F. J. Oliver, J. Yélamos, J. Rodríguez-Vargas
          </td>
          <td>2025-12-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Complex transcriptional programs and signaling pathways control early haematopoietic lineage specification. Many key regulators have been identified; however, a substantial portion of the genome remains functionally uncharacterized. Here, we investigated six uncharacterized 'Riken' genes identified through transcriptomic profiling of Flk-1+/Pdgfrα- (haematoendothelial-enriched) and Flk-1+/Pdgfrα+ (cardiac mesoderm-enriched) populations at day 4 of embryoid body (EB) differentiation. We generated knockouts in mouse embryonic stem cells and performed bulk RNA-sequencing at day 4. Three of these genes (C130074G19Rik, I830077J02Rik, A530016L24Rik) were selected for further investigation by single-cell RNA-sequencing at day 7 of differentiation, which provided novel insight for two of these genes. Knockout of C130074G19Rik (G19Rik) increased the abundance of megakaryocyte progenitors and reduced endothelial populations, with differentially expressed genes enriched for hemostasis and membrane trafficking pathways. The I830077J02Rik (J02Rik) knockout cells showed subtle changes in extracellular matrix and cell adhesion genes, with a shift toward haematoendothelial lineages. Both G19Rik and J02Rik genes encode (predicted) transmembrane proteins that modulate membrane-associated processes in early haematopoietic development. This work establishes a framework for the study of uncharacterized genes with potential roles in cell fate determination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ff2c9456a24e24d6ccaf029b245641c1cc3ca05" target='_blank'>
              Functional analysis of two uncharacterized genes, C130074G19Rik and I830077J02Rik, during early haematopoietic development.
              </a>
            </td>
          <td>
            Giulia Picco, Ridvan Cetin, E. Bindels, Gregory van Beek, W. V. van Ijcken, M. van den Hout, Eskeatnaf Mulugeta, D. Huylebroeck, Frank Grosveld
          </td>
          <td>2026-01-06</td>
          <td>Journal of cell science</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Gastric cancer remains a highly prevalent and accounts for a notable proportion of global cancer mortality. Both Intrinsic and exogenous agents can exacerbate reactive oxygen species (ROS) related oxidized DNA base lesions and single stranded DNA breaks (SSBs). Base excision repair (BER) serves as the primary defense mechanism for repairing DNA damage induced by oxidative stress. DNA polymerase beta (Pol β) plays a critical role in BER and non-homologous end joining repair pathways. The Pol β is the first perform gap-filling DNA synthesis by its polymerase activity and then cleave a 5′-deoxyribose-5-phosphate (dRP) moiety via its dRP lyase activity. Furthermore, defect in POLB promotes genetic liability of the cancer cells for different targeted and synthetic lethality-based treatment strategies. In this review, we have provided a potential example to illustrate the mechanistic insight how PARP1 inhibitor (Olaparib) induces replication associated double strand breaks in POLB deficient cells and DNA mediated innate immune signal activation that likely enhances immune based therapy. Based on our previously published data and the current recent findings, POLB status of the patient likely provide genetic indicators to stratify gastric cancer patient. Overall, in this review article, we presented a new direction to highlight the opportunity to exploit POLB genetic defect in cancer cells to enhance treatment response and to explore synergistic effect to target gastric cancer cells that harbor aberrant DNA polymerase beta function with immune based therapeutic strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0501b8820aaf9cfdd67e2f7a15f0de284236eed" target='_blank'>
              Harnessing DNA polymerase beta defect enhances synthetic lethality and treatment response in gastric cancer cells: implication for immunotherapy
              </a>
            </td>
          <td>
            Aashirwad Shahi, Shengyuan Zhao, Dawit Kidane
          </td>
          <td>2026-01-06</td>
          <td>Journal of Pharmacy & Pharmaceutical Sciences</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="OBJECTIVE
Glioma recurrence severely impacts patient prognosis, with current treatments showing limited efficacy. Traditional methods struggle to analyze recurrence mechanisms due to challenges in assessing tumor heterogeneity, spatial dynamics, and gene networks. Single-cell combined spatial transcriptomics (ST) offers innovative solutions.


METHODS
We analyzed glioma mRNA data from TCGA and single-cell and ST data from GEO. Following quality control, dimensionality reduction, clustering, and cell annotation of single-cell sequencing data, we identified cell types exhibiting significantly aberrant distributions between primary and recurrent samples by analyzing the deviation degree of Ro/e values. Fibroblasts demonstrating the greatest intergroup differences were subsequently selected as the key cellular population for further investigation. Key differentially expressed genes (DEGs) were identified via random survival forest analysis. Drug sensitivity was assessed using GDSC. Deconvolution algorithms mapped cellular spatial distribution, while PROGENy quantified pathway activity. MISTy modeling revealed cell-cell interactions.


RESULTS
Fibroblasts were the primary recurrence-associated subpopulation, with marker genes enriched in extracellular matrix and adhesion pathways. AEBP1, ZNF708, and TSHZ2 were identified as key genes: AEBP1/TSHZ2 correlated with poor prognosis, while ZNF708 showed an inverse trend. These genes were linked to chemosensitivity (Irinotecan, Carmustine, Vincristine, and Cisplatin). Recurrent tumors exhibited increased plasma cell infiltration, with key genes regulating IL-17, Notch, and Toll-like receptor pathways. Spatial analysis highlighted oligodendrocyte-astrocyte interactions in the tumor microenvironment.


INTERPRETATION
Fibroblasts drive glioma recurrence, with AEBP1, ZNF708, and TSHZ2 predicting recurrence and chemoresistance. These genes promote immune suppression (via plasma cells) and activate recurrence pathways. Oligodendrocyte-astrocyte interactions shape the recurrent microenvironment, suggesting new therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aec7c524a9377c361243faed8c72c15c26a0c97d" target='_blank'>
              Unraveling the Molecular Mechanisms of Glioma Recurrence: A Study Integrating Single-Cell and Spatial Transcriptomics.
              </a>
            </td>
          <td>
            Lei Qiu, Yinjiao Fei, Jiaxuan Ding, Kexin Shi, Jinyan Luo, Yuchen Zhu, Xingjian Sun, Gefei Jiang, Yuandong Cao, Weilin Xu, Shu Zhou
          </td>
          <td>2026-01-06</td>
          <td>Annals of clinical and translational neurology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) represent a highly plastic subpopulation within tumors that drives tumor initiation, progression, metastasis, therapeutic resistance, and disease relapse. Although CSC traits were initially attributed to stable genetic hierarchies, growing evidence suggests that cancer stemness is a dynamic and reversible state regulated by precise molecular events at the protein level. This literature review aims to synthesize experimental evidence regarding the impact of single amino acid mutations and alterations in regulatory proteins on the stemness properties of cancer cells. A systematic literature search was conducted using PubMed and Google Scholar to identify original research articles published between 2020 and 2025 that examined protein-level mutations, post-translational modifications, epigenetic regulators, and chromatin remodeling factors in diverse cancer models. The reviewed studies consistently demonstrate that single amino acid substitutions in oncogenic, tumor suppressor, or regulatory proteins can profoundly affect protein conformation, stability, enzymatic activity, and protein–protein interactions. These alterations frequently result in sustained activation of key stemness-associated signaling pathways, including Wnt/β-catenin, Notch, Hedgehog, and STAT3, thereby enhancing self-renewal capacity, stem cell marker expression, tumor-initiating potential, and resistance to conventional therapies. In parallel, dysregulation of post-translational modification systems, such as phosphorylation, ubiquitination, acetylation, and methylation, promotes epigenetic reprogramming and reactivation of embryonic transcriptional programs that favor cellular plasticity. Moreover, protein-level alterations strengthen cancer cell interactions with the tumor microenvironment, increasing responsiveness to hypoxia, inflammatory signals, and metabolic stress. Collectively, these findings highlight protein-level regulation as a central determinant of cancer stem cell maintenance and plasticity, and suggest that effective therapeutic strategies should target aberrant protein stability, post-translational modification machinery, and stemness-related signaling networks to achieve durable clinical responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f961c9b291a85047c8455458a035edcd22afcac" target='_blank'>
              IMPACT OF SINGLE AMINO ACID MUTATIONS OR ALTERATIONS IN REGULATORY PROTEINS ON THE STEMNESS PROPERTIES OF CANCER CELLS
              </a>
            </td>
          <td>
            Dewi Sartika, Irmatul Laili, Sri Utami
          </td>
          <td>2025-12-30</td>
          <td>Journal of Stem Cell Research and Tissue Engineering</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f6c40c7ea1ba98d656117fae44f7bccd796a048" target='_blank'>
              Incomplete developmental silencing of cancer-testis antigen BORIS in humanized mouse model promotes cancer susceptibility
              </a>
            </td>
          <td>
            Emma Price, E. Pugacheva, D. N. Bhatt, Yon Ji, Sheila Yeboah, Arielle Scott, D. Loukinov, Victor V. Lobanenkov
          </td>
          <td>2025-12-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="The Methyl CpG Binding Protein 2 (MECP2) gene has copy number gain in a number of human cancers and functions as an oncogene through an unusual epigenetic mechanism. We explored the possibility that MECP2 might be a therapeutic target, and whether its epigenetic mode of action is amenable to specific therapy. Constitutively expressed or inducible lentiviral short hairpin RNA (shRNA) directed at MECP2 in human triple negative breast cancer (TNBC) cell lines with high or low level of MECP2 protein were grown as xenografts to assess oncogene addiction. We next evaluated the effect of DNA methylation inhibitors or histone deacetylase inhibitors on monolayer or soft agar growth of isogenic human mammary epithelial cells with or without MECP2 overexpression, and xenograft growth of MECP2-dependent or MECP2-independent human TNBC or lung cancer cell lines. We then investigated the mechanism of MECP2-induced activation of MAPK pathway and assessed the effect of drug treatment. Human cancer cell lines with MECP2 overexpression show MECP2 dependence, and epigenetic drugs are effective in these models. Activated RAS and other activators of the MAPK pathway caused resistance to these therapies, giving insight into their novel mode of action and demonstrating specificity. The kinase PAK3 is important for MECP2-mediated induction of MAPK pathway, is modulated by epigenetic drugs affecting MECP2 action as expected and may itself be a therapeutic target for MECP2-driven cancers. These preclinical studies show that tumors overexpressing MECP2 might benefit from epigenetic therapy targeting MECP2 function and demonstrate a novel mechanism of action for these drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4802b6674f8b3e0a4e440436dd1fc69dc98c09c" target='_blank'>
              MECP2-Dependent Cancers Can Be Targeted by Epigenetic Drugs: A New Role for Epigenetic Cancer Therapy.
              </a>
            </td>
          <td>
            Manish Neupane, Mukesh Kumar, Malela A Mwamufiya, Colleen M Donnelly, Alsu Ibragimova, Brian Curcio, Inna Chervoneva, Irina Vasilevskaya, Daniel P Silver
          </td>
          <td>2025-12-11</td>
          <td>Molecular cancer therapeutics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a malignant blood disease with a poor prognosis. With the development of targeted therapies, many refractory/relapsed or unfit AML patients have achieved better survival rates. However, primary or acquired resistance frequently occurs, and some targets become unavailable. This study aimed to identify and verify novel targets for AML using Mendelian Randomization (MR) analysis. The association between plasma protein quantitative trait loci (pQTL), expression quantitative trait loci (eQTL), and AML was examined via MR. Bayesian co-localization analysis was used to detect common segments of TCL1A and AML. Bioinformatics technology was used to explore the relationship between TCL1A and AML, and quantitative real-time PCR (RT-qPCR) analysis was used to detect the expression of TCL1A in various cell lines. Furthermore, high-throughput virtual screening (HTVS) and molecular dynamics (MD) simulation was used to identify potential TCL1A inhibitors. Further verification was carried out by AML cell experiments. Our study found a strong association between plasma TCL1A protein levels and AML (OR: 1.66; 95% CI: 1.46–1.88), and rs78986913 was the same variant in both TCL1A and AML. High TCL1A expression was associated with low overall survival rates in AML patients (P < 0.001), especially in the FAB M0 and M2 subgroups. Additionally, TCL1A was abnormally highly expressed in many AML cell lines (including MOLM-13, MV4-11, THP-1) compared to normal bone marrow stromal cells (HS-5) and other solid tumor cells. We also identified 19 small molecules, finding that Bletilloside A (id number: 2292159-89-6) is one of the most fitting small molecules for the TCL1A protein pocket. MD simulation confirmed the good stability of the Bletilloside A-TCL1A complex. And Bletilloside A could inhibit the proliferation of AML by inhibiting TCL1A. There is a causal relationship between TCL1A and AML. The expression level of TCL1A deserves further monitoring in clinical settings, and the benefits of targeting TCL1A should be further validated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5cf4dfa88d970f81cc960282fc2f28f7a5e6b5a2" target='_blank'>
              New drug targets for acute myeloid leukemia identified through a comprehensive analysis of the plasma protein
              </a>
            </td>
          <td>
            Yaonan Hong, Xu Wang, Jingjie Dong, Qi Liu, Yingying Shen, Xiawan Yang, Peicheng Wang, Wenjie Li, Baodong Ye, Keding Shao, Ya-Huei Kuo, Dijiong Wu
          </td>
          <td>2025-12-10</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="The tumour suppressor TP53 is frequently mutated in breast cancer and drives poor outcomes. The impact of mutant p53 (mutp53) on subtype-specific gene and non-coding RNA networks, and their clinical significance, remains largely underexplored. Here, using TCGA-BRCA data, we have delineated subtype-specific mRNA, lncRNA, and microRNA signatures, pathways, co-expression/interaction networks, and prognosis associated with hotspot mutp53 or wildtype p53 tumours. Our study shows that mutp53 deregulates the genes related to EMT, chemoresistance, and prognosis in a subtype-specific manner. The EMT-associated signature was able to stratify HER2 and Basal patients by their p53 status. Construction of lncRNA-mRNA-miRNA interaction networks led to the identification of various feedback loops and hub genes with prognostic relevance that possess binding sites for p53 and EMT-TFs within their promoters. In the basal mutp53 tumours, we found Androgen Receptor (AR) to be a downregulated EMT-associated gene, with its higher levels linked to a better prognosis. We validated that mutp53 breast cancer cell lines show reduced levels of AR and its predicted transcriptional target, miR-196a-5p. Overexpression of WTp53 resulted in the upregulation of AR and miR-196a-5p, while mutp53 (R175H) suppressed their expression. Basal mutp53 tumours with low AR displayed higher EMT scores. Enforced expression of AR led to suppression of mesenchymal markers in basal cell lines. Overall, we have identified novel prognostically relevant RNA signatures and networks that may serve as attractive therapeutic targets in mutp53 breast cancer patients in a subtype-specific manner. Additionally, we have discovered a novel AR:mutp53 association that may be implicated in EMT and chemoresistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d44469e17ac42fb8123cb7e43e4cf7857cd2b3af" target='_blank'>
              Capturing the Variations in Mutant p53-Driven Regulatory Networks in Breast Cancer Subtypes, Its Clinical Relevance and a Novel Association With Androgen Receptor and EMT.
              </a>
            </td>
          <td>
            Aastha Singh, Rahul Gupta, Ritu Kulshreshtha
          </td>
          <td>2025-12-30</td>
          <td>IUBMB life</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background/Aim: Prostate cancer features profound transcriptional dysregulation within the androgen receptor (AR) signaling axis. The pioneer factor FOXA1, which facilitates AR binding to chromatin, is recurrently altered in 10-40% of tumors. Recent studies classify FOXA1 mutations as Class 1 Wing 2 mutations, which enhance AR-dependent tumorigenesis, and Class 2 C-terminal truncations, which promote lineage plasticity and therapy resistance. The interplay of FOXA1 alterations with TMPRSS2-ERG fusions and PROX1 remains incompletely understood. Materials and Methods: Data from The Cancer Genome Atlas (TCGA) Prostate Adenocarcinoma (PRAD) cohort (n=492) were analyzed via UCSC Xena and cBioPortal. FOXA1 mutations were categorized following Eyunni et al. Copy number was assessed by log2(tumor/normal) ratios. Mutual exclusivity and co-occurrence were evaluated using Fisher’s exact test with false-discovery-rate correction. Associations between FOXA1 status and genomic instability were assessed using the fraction genome altered (FGA) metric. Results: FOXA1 was broadly expressed, with subsets showing elevation. Class 1 mutations localized to the Wing 2 region, while Class 2 truncations clustered in the C-terminal domain. Copy number changes were infrequent, indicating mutation-driven reprogramming as the main oncogenic mechanism. TMPRSS2 and ERG strongly co-occurred (log2 OR >3, q<0.001), whereas FOXA1 was mutually exclusive with both TMPRSS2 and ERG (q<0.001). Although FOXA1 alterations showed no significant Pearson correlation with FGA (r=−0.01, p=0.76), a moderate Spearman correlation (ρ=0.52, p<0.001) suggested enrichment in genomically unstable tumors. Conclusion: FOXA1 defines a major oncogenic axis in prostate cancer, distinct from TMPRSS2-ERG fusion and PROX1 induction. Class 1 and 2 FOXA1 mutations drive alternative transcriptional programs leading to therapy resistance, highlighting FOXA1 as a critical biomarker and target for chromatin-directed interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/962282da85200138085414df96f69e7ea38e7831" target='_blank'>
              FOXA1 Alterations in Prostate Cancer: Expression, Mutation Classes, and Copy Number Changes
              </a>
            </td>
          <td>
            S. Lehrer, P. Rheinstein
          </td>
          <td>2026-01-01</td>
          <td>In Vivo</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="

 CD47 is a key innate immune checkpoint that enables tumor cells to evade macrophage-mediated clearance.



 To systematically identify genetic regulators of CD47 surface expression, we performed FACS-based genome-wide CRISPR screens in three murine cancer cell lines B16 (melanoma), MC38 (colon adenocarcinoma), and EMT6 (breast carcinoma).



 Comparative analysis of cells with high or low CD47 surface expression using DrugZ revealed CD47 itself as the top hit, validating the screens. Notably, DNAJC13 emerged as a consistent and robust regulator of CD47 expression across all three cell lines. Functional validation using DNAJC13-knockout cells confirmed a significant reduction in CD47 surface levels. Furthermore, in co-culture assays with macrophages, DNAJC13-deficient tumor cells exhibited increased susceptibility to phagocytosis, supporting a functional role for DNAJC13 in innate immune evasion. Finally, we verify that DNAJC13-knockout decrease tumor burden when treated with CD47 blockade.



 Overall, this study highlights a previously unrecognized regulator of CD47 and demonstrates the utility of high-throughput FACS-based CRISPR screening to uncover modulators of key immune checkpoint pathways.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/333becc09fbebbb08ac6ada5d4961c0def934201" target='_blank'>
              FACS-based genome-wide CRISPR screening platform identifies modulators of CD47
              </a>
            </td>
          <td>
            Ling Yin, Wei He, Yifan Wang, Huimin Zhang, Min Huang, Yuelong Yan, Siting Li, Xu Feng, Francisco Saenz, Jie Zhang, Dandan Zhu, Chang Yang, Tiantian Ma, Jialing Fu, Junjie Chen
          </td>
          <td>2026-01-12</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Central nervous system (CNS) metastasis is a major driver of morbidity in metastatic breast cancer, yet the molecular determinants of CNS tropism remain incompletely defined. LYN, a Src-family kinase integrating receptor tyrosine kinase and integrin signaling, is a biologically plausible mediator of metastatic traits. We performed a retrospective, multi-study analysis of publicly available breast cancer cohorts aggregated in cBioPortal. After harmonization and de-duplication, LYN status was determinable in 5,947 invasive carcinoma of no special type (NST) tumors across 29 studies. The primary endpoint was CNS metastasis at any time (Yes/No), harmonized via a prespecified controlled vocabulary (case-insensitive substring mapping). Somatic LYN variants (coding SNVs/indels) were collapsed to patient-level classes (missense-only; truncating if any nonsense/frameshift/splice). Variants with resolvable positions were mapped to Src-family modules (SH4/Unique, SH3, SH2, SH2-kinase linker, kinase). Two-group comparisons used two-sided Fisher’s exact tests with exact 95% CIs; domain screens used omnibus χ² and Benjamini–Hochberg FDR control. A prespecified Firth logistic model evaluated truncating vs. missense within LYN-mutant tumors. Public cancer genomics repositories (cBioPortal); multi-institutional cohorts. 5,947 tumors across multi-study cohorts with LYN status available. None. Primary: ever-CNS metastasis (yes/no). Secondary: distribution of LYN variant classes and domains (SH4/Unique, SH3, SH2, linker, kinase). CNS metastasis occurred in 5/46 (10.9%) LYN-mutated tumors vs. 110/5,901 (1.9%) LYN wild-type tumors (odds ratio (OR) = 6.42; 95% confidence interval (CI), 2.49–16.56; p = 0.0018). The endpoint was captured as ever vs. never CNS involvement (event dates unavailable), precluding time-to-event inference. Within LYN-mutant cases, an exploratory domain analysis indicated that distributions differed by CNS status (omnibus χ² p ≈ 0.014); a one-versus-rest signal at the SH4/Unique N-terminus was nominally significant and borderline after false discovery rate (FDR) (unadjusted p ≈ 0.010; q ≈ 0.052; small in-domain n = 3). By mutation class, truncating vs. missense showed a higher CNS-positive proportion (28.6% vs. 7.9%) but did not reach significance (Fisher p = 0.166; alternatively framed OR = 4.22; exact 95% CI, 0.58–30.75; p = 0.182). Firth estimates were directionally consistent with wide profile CIs under sparse counts. Across pooled cohorts, LYN mutation is associated with increased odds of CNS metastasis, and domain context appears informative, with a small-sample, FDR-borderline enrichment at the SH4/Unique N-terminus. The truncating-class signal is exploratory given limited power. Signals by domain (notably SH4/Unique) are exploratory and require independent validation in larger, uniformly annotated datasets. Given small mutant denominators and ever-CNS endpoint capture, findings are hypothesis-generating and not actionable for risk stratification or treatment selection. Results motivate domain-aware annotation in future validation studies and mechanistic work. What is already known: CNS metastasis is a major clinical problem in breast cancer; molecular determinants of brain tropism remain incompletely defined. What this study adds: Across pooled cohorts, any LYN mutation is associated with higher odds of CNS metastasis; domain context (notably SH4/Unique) may matter, albeit with small counts and FDR-borderline evidence. How this might affect research/practice/policy: Encourages domain-aware LYN annotation and targeted mechanistic work on membrane targeting/BBB traversal; not ready for risk stratification or treatment selection. What is already known: CNS metastasis is a major clinical problem in breast cancer; molecular determinants of brain tropism remain incompletely defined. What this study adds: Across pooled cohorts, any LYN mutation is associated with higher odds of CNS metastasis; domain context (notably SH4/Unique) may matter, albeit with small counts and FDR-borderline evidence. How this might affect research/practice/policy: Encourages domain-aware LYN annotation and targeted mechanistic work on membrane targeting/BBB traversal; not ready for risk stratification or treatment selection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9e375e78de0e51d3450906c5e0322f113edd318" target='_blank'>
              LYN mutations in breast cancer: hypothesis-generating evidence for an association with central nervous system metastasis and domain-level insights
              </a>
            </td>
          <td>
            Elif Kardelen Çağdaş, Berkay Çağdaş
          </td>
          <td>2025-12-05</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Prostate adenocarcinomas are the most common form of the disease. Targeting the androgen receptor (AR) is the principal treatment strategy. However, resistance is common. Lineage plasticity (LP)—change in differentiation state—is increasingly recognized as a mechanism of resistance to AR-targeting therapies. LP is a continuum ranging from amphicrine to neuroendocrine prostate cancer (NEPC). Despite the increasing incidence of LP due to more widespread use of new and potent AR inhibitors, mechanisms by which LP emerges after therapy have been understudied due to paucity of matched patient tumors. To address this deficit, we performed multi-omic profiling on matched biopsies from 16 patients, including seven patients that underwent LP after treatment.



 We used DNA sequencing to identify mutations and copy number alterations present in tumors at baseline and at progression. We also performed RNA sequencing and transcriptional subtyping of all tumors.



 Analysis of RNA sequencing revealed two distinct subtypes of LP in our cohort—most consistent with amphicrine or NEPC. DNA sequencing identified TP53 and RB1 alteration patterns in baseline tumors that eventually underwent specific LP trajectories. Pathways linked to proliferation and inflammatory signaling were highly activated in baseline tumors that eventually underwent LP.



 Our analysis identified distinct LP trajectories linked with specific genomic and transcriptomic features at baseline. Profiling tumors with this approach may provide clues about which patients may be at greatest risk for undergoing specific LP trajectories after therapy.



 Anbarasu Kumaraswamy, Visweswaran Ravikumar, Ryan J. Rebernick, Eva Rodansky, Amina Tanweer, Joel A. Yates, Aaron M. Udager, Marcin P. Ceislik, Arvind Rao, Zachery R. Reichert, Arul M. Chinnaiyan, Joshi J. Alumkal. Multi-Omic Profiling Clarifies Mechanisms of Therapy-Induced Prostate Cancer Lineage Plasticity [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A036.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d201d414071181c06c4abf5de951c2f23846da37" target='_blank'>
              Abstract A036: Multi-Omic Profiling Clarifies Mechanisms of Therapy-Induced Prostate Cancer Lineage Plasticity
              </a>
            </td>
          <td>
            A. Kumaraswamy, V. Ravikumar, Ryan J. Rebernick, Eva Rodansky, Amina Tanweer, Joel A. Yates, Aaron M. Udager, Marcin P. Ceislik, Arvind Rao, Z. Reichert, A. Chinnaiyan, J. Alumkal
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>170</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d978d923d3df7e502b397062dfadf85de9aa31d6" target='_blank'>
              DNA methylation and lncRNA control asynchronous DNA replication at specific imprinted gene domains.
              </a>
            </td>
          <td>
            Yui Imaizumi, François Charon, Caroline Surcis, Christel Picard, Pol Arnau-Romero, J. Andrau, D. Noordermeer, Benoit Moindrot, Jean-Charles Cadoret, Robert Feil
          </td>
          <td>2026-01-21</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Chimeric antigen receptor (CAR) T cell therapies have demonstrated remarkable efficacy in several hematological malignancies, yet their success has not been fully replicated in solid tumors. Moreover, even in hematological cancers, relapse after CAR T cell infusion continues to compromise long-term outcomes. These challenges highlight the urgent need to develop strategies that enhance CAR T cell efficacy, persistence, overcoming tumor and microenvironment-mediated resistance. Clustered regularly interspaced short palindromic repeats (CRISPR)-Cas9-based screening platforms provide a powerful approach to systematically identify genes that regulate CAR T cell function. By linking genetic perturbations to phenotypic outcomes, these assays enable the discovery of pathways controlling activation, proliferation, memory formation, and cytotoxicity. Standard workflows involve transduction of substantial numbers of cells with a single guide RNA (sgRNA) library, Cas9-mediated editing, selection of edited cells, and PCR amplification of sgRNA cassettes from genomic DNA (gDNA) prior to sequencing. However, PCR amplification using large amounts of gDNA poses significant challenges and often fails to selectively amplify and retrieve sgRNAs. Here, we describe an optimized CRISPR-Cas9 knockout screening protocol, which we have tested on primary human CAR T cells. The method here incorporates an intermediate step during sgRNA library preparation that reduces gDNA carryover through enzymatic digestion and selective pulldown of the sgRNA cassette, thereby increasing the efficiency of the first PCR amplification. This modification allowed us to retrieve sgRNA information across our CAR T cell screens, which had remained elusive in our previous attempts using traditional 1 and 2-step PCR amplification protocols. In conclusion, this optimized workflow facilitates CRISPR screening library preparation in challenging samples and enables the identification of key genetic determinants that can be targeted to improve therapeutic efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecca94f275145ee359153517eba4093f85ededd3" target='_blank'>
              Improving CRISPR-Cas9 Screens in CAR T Cells: A Refined Method for Library Preparation.
              </a>
            </td>
          <td>
            Maider Garnica, Patxi San Martín-Úriz, Paula Rodriguez-Marquez, M. E. Calleja-Cervantes, Saray Rodríguez-Díaz, Rebeca Martinez-Turrillas, Mikel Hernaez, Felipe Prósper, J. R. Rodríguez-Madoz
          </td>
          <td>2026-01-02</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afc8fd0b9c370fcd3fe478613bb2d770e9cf4d40" target='_blank'>
              Clonal Hematopoiesis Associated with TP53 and DNMT3A Mutations Promotes Tissue Repair in Acute Cardiovascular Diseases
              </a>
            </td>
          <td>
            Yi Pan, XiaDuo Meng, Chen Wang, WenXuan Zhou, Qin Zhou, Jingshu Chi, Breton Barrier, Luis A Martinez-Lemus, De-Pei Li, Zhenguo Liu, Xunlei Kang
          </td>
          <td>2025-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Despite breakthroughs in molecularly targeted and immune therapies, the prognosis for patients with urinary bladder cancer (UBC) has remained unsatisfactory over the past few decades. Understanding the molecular underpinnings of UBC treatment refractoriness is crucial for identifying novel therapeutic targets and strategies. Cancer stemness plays a pivotal role in the oncogenesis and treatment resistance of UBC, while the underlying molecular regulatory mechanisms are poorly understood. We identified isoform 1 of ASPM (ASPM-i1) as the most upregulated stemness-associated factor in tumorigenic UBC cells, predominantly expressed by ALDH1+ stem-like cancer cells. Pairing genetic ASPM-i1 inhibition with standard chemotherapeutic agents used in the treatment of UBC, including cisplatin and gemcitabine, circumvents the treatment resistance of tumorigenic and stem-like UBC cells. Mechanistically, ASPM-i1 interacts with the Disheveled (DVL) and intracellular NOTCH proteins, thereby attenuating CUL3- or FBXW7-mediated ubiquitination and the subsequent proteasomal degradation. The regulatory module concomitantly enhances the activities and ligand responsiveness of the Wnt and Notch signaling pathways in UBC cells. As a result, ASPM-i1 inhibition sensitized tumorigenic UBC cells to chemotherapy in a NOTCH- and DVL-dependent manner. In human UBC tissues, ASPM-i1 shows substantial cell-to-cell heterogeneity and is upregulated in a subset (46.4%) of tumors, correlating with poor clinical prognosis. This study reveals a crucial co-regulatory module of Notch and Wnt signaling that mediates stemness and chemotherapy resistance in tumorigenic UBC cells; its inhibition provides a novel approach to enhance chemosensitivity and improve therapeutic outcome in human UBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/076780e38121b3f25a600b0e4d535709ef262847" target='_blank'>
              Chemotherapy resistance of urinary bladder cancer mediated by a Notch and Wnt co-regulatory module of stemness.
              </a>
            </td>
          <td>
            Lin-Hsin Cheng, Che-Wei Hsu, Pei-Ming Yang, Tai-Yan Liao, Fang-Chi Liu, Wan-Wen Chen, Chin-Chen Pan, Chung-Chi Hsu, Kuan-Chou Chen, K. K. Tsai
          </td>
          <td>2026-01-10</td>
          <td>Stem cells</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Traditionally, scientists tend to approach cancer research in a reductionistic way: aiming at uncovering underlying, separate components in malignant processes. And indeed, great progress has been made by reducing the development of a tumor to single, specific genes and mutations. For instance, familial adenomatous polyposis (FAP) could be reduced to a germline mutation in the Adenomatous Polyposis Coli (APC) gene. The escape of tumor cells from immune surveillance could be reduced to the tumor expression of immune checkpoints, resulting in new approaches in tumor therapy by applying immune checkpoint inhibitors. However, a germline mutation in APC is not 1:1 related to colorectal cancer (CRC), and only some patients respond to immune checkpoint inhibitors. The point here is that biological systems, also comprising cancer, have properties that cannot be reduced to single components. The cooperation of the single components results in new, emergent properties. The outcome of an interaction in a complex network, like the immune system, depends on the many cell types involved and the numerous molecules that interact and activate or inhibit pathways. The way the composing elements are organized is a causal factor in itself for any emergent property. The rise of genomic analysis at the end of the previous century, enabling us to sequence a full genome at the DNA and RNA levels, has initiated an awareness of the need for ‘systems biology’: to consider a full system and how it is organized, in all of its aspects, to understand biological pathways and their outcomes. In this review, we outline the prospects and limitations of systems biology in cancer research and propose a causal framework that integrates upward and downward causation and multiple realizability to understand the emergent properties of tumors that determine the dynamics of tumor development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bad26730d321f49013a748603a1b42d130f54226" target='_blank'>
              The Need for a Systems Biology Approach in Cancer Explained
              </a>
            </td>
          <td>
            Hehuan Zhu, Xi Zhang, E. Nazemalhosseini-Mojarad, J. Roelands, L. D. A. N. de Muynck, C. Ravensbergen, Rachel Hoorntje, I. Stouten, Marianne Hokland, A. L. Vahrmeijer, Rob A. E. M. Tollenaar, Edwin Koster, Peter J K Kuppen
          </td>
          <td>2025-12-22</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5df76d59f3a1b2aa9161ce2bf7becbe9cb09edf1" target='_blank'>
              Uncovering the role of LINE-1 in the evolution of lung adenocarcinoma.
              </a>
            </td>
          <td>
            Tongwu Zhang, Wei Zhao, Christopher Wirth, M. Díaz-Gay, Jinhu Yin, M. Cecati, F. Marchegiani, P. Hoang, Charles Leduc, M. Baine, William D. Travis, L. Sholl, P. Joubert, Jian Sang, John P McElderry, Michelle Antony, Alyssa Klein, Azhar Khandekar, Caleb Hartman, J. Rosenbaum, Frank J Colón-Matos, M. Miraftab, Monjoy Saha, Olivia W. Lee, Kristine M. Jones, N. E. Caporaso, M. Wong, Kin Chung Leung, C. A. Hsiung, Chih-Yi Chen, E. S. Edell, Jacobo Martínez Santamaría, M. Schabath, Sai S. Yendamuri, Marta Mańczuk, J. Lissowska, B. Świątkowska, A. Mukeria, O. Shangina, D. Zaridze, I. Holcatova, D. Mates, Sasa Milosavljevic, Milan Savic, Y. Bossé, B. G. Rothberg, David C. Christiani, V. Gaborieau, Paul Brennan, Geoffrey Liu, P. Hofman, R. Homer, S. Yang, A. Pesatori, D. Consonni, Lixing Yang, Bin Zhu, Jianxin Shi, Kevin M. Brown, Nathaniel Rothman, S. Chanock, Ludmil B. Alexandrov, Jiyeon Choi, Maurizio Cardelli, Qing Lan, Martin A. Nowak, D. Wedge, M. Landi
          </td>
          <td>2025-12-10</td>
          <td>Nature</td>
          <td>0</td>
          <td>109</td>
        </tr>

        <tr id="Age-related hearing loss (ARHL) is the most common type of hearing loss. Genetic factors are considered to play important roles in the development of ARHL. To identify novel susceptibility genes and cell types relevant to ARHL, we performed a two-stage single-cell transcriptome-wide association study (scTWAS) on ARHL in 96,372 cases and 141,590 controls of European descent. In the discovery stage, we identified 1034 gene-cell pairs that showed suggestive associations with ARHL (P < 1.0 × 10-5), representing 450 genes across various cell types. These genes were enriched in multiple pathways, including the immune-related, estrogen signaling, and oxidative damage response pathways. Besides, we provided prominent genetic evidence for putative drug repurposing, highlighting several genes as potential targets, including NR3C2, CHRM4 and SHBG. Further, we validated the significant association of 41 genes with ARHL in the replication stage of scTWAS, including previously reported genes such as HLA-DRA, as well as novel candidates such as TNF, ZC3HAV1, and SLC44A4. Among these novel candidates, several are highly biologically plausible in the development of ARHL. In conclusion, this scTWAS broadens our understanding of the genetic susceptibility to ARHL, which might be helpful in developing new strategies for the treatment and prevention of ARHL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1879737c8dd65842b6c44cd9451b29e4f4073aa3" target='_blank'>
              A Single-cell Transcriptome-wide Association Study Reveals Susceptibility Genes for Age-related Hearing Loss.
              </a>
            </td>
          <td>
            Yuanfeng Li, Tao Zeng, Wenyu Song, Yahui Wang, Lili Ren, Chenning Yang, Gangqiao Zhou, Yuguang Niu
          </td>
          <td>2026-01-06</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97e8ef7d28e9d10b44890e1b7be7668548430edd" target='_blank'>
              CHEK2 loss endows chemotherapy resistance to hematopoietic stem cells.
              </a>
            </td>
          <td>
            Jing Zhou, Tianyuan Hu, Dian Li, Sanming Li, Minhua Li, Xiangguo Shi, Daisuke Nakada
          </td>
          <td>2026-01-12</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="During a lifetime, normal cells accumulate thousands of changes in their genome sequence. These changes, termed somatic mutations, have mostly been studied in the context of cancer, but their presence in normal tissues is ubiquitous and widespread. Somatic mutation accompanies the aging process and is influenced by genetic and environmental factors. Differently from gene expression or imaging data, which fluctuate over time, somatic variants are non-reversible marks in the genome and accumulate over time. This property can be exploited to track the history of a cell, from conception to old age, providing information that cannot be acquired via classical histological tissue inspection nor other types of omics data. Mutations can track embryonic development, measure how clones compete in a tissue over time, or report the mutational processes active in cells and tissues throughout life. We discuss selected examples and emphasize how somatic mutation analysis can enable expanding applications at the service of physiology and cell biology, as well as a deeper understanding of the aging process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e1754e49deb7eef5ac28d0fb31ba15c272ffd93" target='_blank'>
              Reconstructing the lifelong history of cells and tissues via somatic mutation analysis
              </a>
            </td>
          <td>
            Sipontina Faienza, J. P. Margaria, Irene Franco
          </td>
          <td>2025-12-01</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="
 Introduction.
 The development of therapy resistance and relapses of acute myeloid leukemia (AML), especially in the intermediate prognosis group, may be due to the molecular genetic heterogeneity of tumor cells. Chromosomal microarray analysis (CMA) can detect microdeletions, duplications, and copy-neutral loss of heterozygosity (cnLOH) which may be associated with a response to therapy.


 Aim:
 to evaluate tthe frequency of copy number aberrations and cnLOH in leukemogenesis-associated genes in patients with intermediate-stage AML and and their relationship to survival and response to treatment.


 Materials and methods
 . The study included 35 patients with de novo AML from the intermediate prognosis group for ELN2017. Copy number analysis by CMA was performed for a panel of 36 genes associated with leukemogenesis. The reference group included 102 healthy individuals without oncohematological disorders who also underwent comparable CMA testing.


 Results.
 Genomic aberrations were detected in 91.18 % of patients, most often in the genes of chromatin modifiers (64.7 % patients) and tumor suppressor genes (64.7% patients). The cnLOH type (PHF6, SMC1A, BKORL1) prevailed. KMT2A duplications occurred only in AML patients — 14.3 % (p < 0.001) and were associated with worse survival (log-rank P = 0.05). Combinations of genomic alterations involving 4–7 functional gene groups were found in 20.6% of patients.


 Conclusion.
 Driver gene aberrations, especially KMT2A duplications, are associated with an unfavorable clinical outcome in AML with an intermediate prognosis.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4336f89da62f27fbefbd6b50cf91ca51d0c03af7" target='_blank'>
              Structural aberrations of genes associated with leukemogenesis in patients with acute myeloid leukemia of intermediate prognosis
              </a>
            </td>
          <td>
            D. K. Bessmertnyy, S. Starchenko, N. Risinskaya, S. Kulikov, U. A. Chabaeva, V. Surimova, A. S. Ponamoreva, I. Kanivets, Z. T. Fidarova, I. Lukianova, A. Kashlakova, E. V. Romanyuk, N. I. Balaeva, V. Troitskaya, A. Sudarikov, E. Parovichnikova
          </td>
          <td>2025-12-21</td>
          <td>Russian journal of hematology and transfusiology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Rearranged T and B-cell receptor loci serve as critical clonal markers for studying adaptive immune system functioning during normal immune response and abnormal expansion in lymphoid malignancies. Analyses of both complete and partial TCR/BCR gene rearrangements provide valuable insights into the clonal proliferation of malignant T and B lymphocytes. Our earlier work identified novel partial rearrangements between two D (diversity) genes in the human TRB locus in normal T cells from peripheral blood and thymus. In this study, we demonstrate the presence of these novel rearrangements in leukemic T cells, explore their properties for clonality assessment and potential use in minimal residual disease (MRD) monitoring in acute lymphoblastic leukemia, and lay the foundation for further clinical validation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a36c5f22f8b794213fc21de30e3f965afef99616" target='_blank'>
              Exploring coding and signal joints of novel TCR beta D-D rearrangements for tracking T-cell leukemia clonality
              </a>
            </td>
          <td>
            A. M. Miroshnichenkova, V. K. Ruppel, A. O. Smirnova, A. Borkovskaia, B. E. Minasian, A. V. Pavlova, P. V. Shelyakin, A. N. Kazakova, E. A. Zerkalenkova, A. Karachunsky, M. Maschan, Y. Ventura-Carmenate, D. M. Chudakov, Y. Olshanskaya, A. Y. Komkov
          </td>
          <td>2025-12-01</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="BACKGROUND Although the relationship between somatic DNA polymerase epsilon (POLE) exonuclease domain mutations (EDMs) and colorectal cancer (CRC) is well established, the role of POLE non-EDMs in CRC remains unclear. AIM To identify POLE non-EDMs and EDMs in CRC, and to determine their associations with accompanying mutations and microsatellite instability (MSI). METHODS In this retrospective study, next-generation sequencing was performed using a targeted colon cancer panel (Qiagen, DHS-003Z) on 356 CRC patients. Of these, 191 patients were found to carry POLE mutations. For these patients, MSI status was assessed using both real-time PCR (EasyPGX® Ready MSI kit) and immunohistochemistry, and accompanying somatic mutations were investigated. RESULTS POLE mutations were identified in 53.65% of the CRC patients. Among the POLE-mutant patients, 87.96% were classified as pMMR (MSI-L), and 12.04% as dMMR (MSI-H). The most frequently observed POLE non-EDM variant was exon 34 c.4337_4338delTG p.V1446fs*3. The POLE EDMs were present in exon 14, with two specific variants p.Y458F (0.52%) and p.Y468N (0.52%). The most common pathogenic variants accompanying the POLE mutations were in MLH3, MSH3, KRAS, PIK3CA, and BRAF genes. POLE mutations were associated with a high mutational burden and MSI in CRC, particularly in the dMMR phenotype. This association suggests that POLE mutations may serve as important biomarkers for understanding the genetic profile of the disease and may be used in the clinical management of CRC. CONCLUSION POLE mutations, especially non-EDMs, are frequent in MSI-L CRC and often co-occur with MLH3, MSH3, KRAS, PIK3CA, and BRAF, highlighting their potential role in tumor biology and as biomarkers for personalized treatment. Functional validation and multicenter studies are needed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b2e749fce823c98ace9d60bbfaa0c391d7f0a6e" target='_blank'>
              DNA polymerase epsilon-mutant colorectal cancers: Insights into non-exonuclease domain mutation variants, microsatellite instability status, and co-mutation profiles
              </a>
            </td>
          <td>
            Ismail Taskiran, Seda Orenay-Boyacioglu, Olcay Boyacıoğlu, I. Erdogdu, N. Çulhacı, Ibrahim Meteoglu
          </td>
          <td>2025-11-28</td>
          <td>World Journal of Gastroenterology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Cell metabolism has a profound impact on maintaining genomic stability. AMP-activated protein kinase (AMPK) is a crucial regulator of cell metabolism and the maintenance of genomic stability. There is increasing evidence that AMPK plays a crucial role in the efficient response to DNA damage (DDR). However, the underlying mechanism is still unclear. Here, we show that glucose deprivation rapidly reduces γH2AX levels, a hallmark of DNA damage. We then found that WIP1, rather than PP2A or PP4C, is the primary phosphatase responsible for dephosphorylating γH2AX under both normal and damaged conditions. Molecular studies have revealed that AMPK directly binds and phosphorylates WIP1 at Thr25 (T25). This action enhances protein stability and the binding ability of WIP1 with γH2AX, likely promoting the enzyme activity of WIP1 and subsequently reducing the level of γH2AX. These processes facilitate DNA damage repair and contribute to the radioresistance of tumor cells. The findings provide experimental evidence of a novel link between metabolic stress and DDR, suggesting that AMPK may promote the resistance of tumor cells to radiation therapy by phosphorylating WIP1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ded97f76015b39710193c6311700750806454773" target='_blank'>
              AMPK phosphorylates WIP1 to promote DNA repair and radioresistance in cancer cells
              </a>
            </td>
          <td>
            Manman Lu, Xiaochuan Dong, Chunrui Wu, Guisong Wang, Haiyang Wang, Yingli Pan, Yali Qin, Yushuai Song, Hongming Pan, Shenzhi Liu, Kun Zhang, Xuewu Zhang, Jing Qu, Zhenhua Yang
          </td>
          <td>2025-11-28</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background: Genomic integrity is crucial to the cellular life cycle, which involves a tightly regulated process where cells progress through specific phases to ensure that fully replicated, undamaged DNA is inherited by daughter cells. Any dysfunction in this process or unrepaired DNA damage leads to cell cycle arrest and programmed cell death. Cancer cells are known to exploit these mechanisms to continue dividing. Usually, DNA damage arrests replication, allowing the DNA Damage Response (DDR) pathway to activate, which repairs the DNA or bypasses the damage to support cell survival and preserve genome integrity. For DNA damage bypass or translesion synthesis (TLS), a group of low-fidelity polymerases perform error-prone DNA synthesis opposite damaged bases, where REV1 functions as the main scaffolding protein. Previously, we reported non-TLS functions of REV1, including its role in triggering DNA damage-dependent specific DNA metabolic processes. Methods and Results: In this study, we demonstrate that REV1 plays a significant role in cell cycle progression and that its loss causes arrest at the G2/M phase in flow cytometry analysis. This unexpected phenotype includes dysregulation of G2/M regulators, such as Cyclin B1 and tubulins, in REV1-deficient cells compared to controls, as quantified by Western blot. Additionally, phosphorylation of histone H3 at serine 28 was significantly reduced in these REV1-deficient cells. These G2/M arrest features were even more pronounced in REV1-deficient cells treated with the tubulin inhibitor colchicine. Conclusions: Overall, this study reveals a previously unrecognized link between REV1 TLS polymerase inhibition and the G2/M cell cycle arrest.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/349f4407e91b1d8708aa93b1aa069a7dcc5ae83b" target='_blank'>
              REV1 Loss Triggers a G2/M Cell-Cycle Arrest Through Dysregulation of Mitotic Regulators
              </a>
            </td>
          <td>
            Brailey Buntin, Madison Guyette, Vihit Gupta, Kanayo Ikeh, Sombodhi Bhattacharya, Erica N. Lamkin, Allison Lafuze, Roxana del Rio-Guerra, Jiyong Hong, Pei Zhou, Nimrat Chatterjee
          </td>
          <td>2025-12-31</td>
          <td>Genes</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/745490de31a317b3b5c109c205a2bdf3115f7580" target='_blank'>
              Structural Maintenance of Chromosomes 5/6 complex dysfunction enables tumor mutagenesis
              </a>
            </td>
          <td>
            T. Tran, J. Fan, X. Zhao, A. M. Green
          </td>
          <td>2025-12-05</td>
          <td>medRxiv : the preprint server for health sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e961f589165d98aa8aff64f0539527c2012d419" target='_blank'>
              Pooled single-cell screen in colorectal cancer defines transcriptional modules linked to oncogenes.
              </a>
            </td>
          <td>
            Viola Hollek, Francisca Böhning, Catalina Florez Vargas, A. Sieber, M. Morkel, Nils Blüthgen
          </td>
          <td>2026-01-19</td>
          <td>Molecular systems biology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22b544cab2c7f5ac6e8301325b2bf082f8d3ca21" target='_blank'>
              TRMT112 drives a tumor growth and metastasis-promoting program in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Amr R. Elhamamsy, Brandon J. Metge, M. H. Elbahoty, Bhavya Papineni, Heba M. Alsheikh, Dongquan Chen, Rajeev S. Samant, L. Shevde
          </td>
          <td>2026-01-08</td>
          <td>Cell death and differentiation</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="


 Prostate cancer depends on androgen receptor (AR) signaling for growth, which is why androgen deprivation (castration) therapy is effective at early stages. However, many tumors eventually progress to a lethal form known as castration-resistant prostate cancer (CRPC). A subset of CRPC tumors bypass dependency on AR signaling by acquiring lineage plasticity, where prostate cancer cells transdifferentiate into alternate cellular states through epigenetic reprogramming. Neuroendocrine (NE) prostate cancer represents one well-known lineage plasticity phenotype. Nevertheless, most AR-independent tumors do not exhibit NE features and are defined as AR-negative/NE-negative or “double-negative prostate cancer” (DNPC). In a collaboration with Dr. Ekta Khurana’s computational genomics lab at Weill Cornell Medicine, we recently classified CRPC into four epigenetic subtypes, including the well-established 1) AR and 2) NE, as well as the novel DNPC subgroups 3) WNT and 4) stem cell-like (SCL) (PMID: 35617398). We focused on the SCL subtype as it is the second most common group in CRPC patients and lacks therapeutic targets. Using functional genomic approaches, we found that YAP/TAZ/TEAD cooperates with FOSL1 to drive the SCL lineage and growth of SCL models. We therefore hypothesize that the heightened dependency on the YAP/TAZ/TEAD/FOSL1 transcriptional program represents a therapeutic vulnerability in CRPC-SCL. To test this, we exposed CRPC models to TEAD inhibitors and found robust growth suppression in SCL cells compared to non-SCL cells in vitro. To evaluate whether the TEAD inhibitors are on-target, we performed transcriptomic profiling in SCL models and observed downregulation of YAP/TAZ gene signature as well as FOSL1 expression, which phenocopies the effects of YAP/TAZ double knockdown. To define the cistromes of these factors upon TEAD inhibition, we performed ChIP-seq and observed reduced co-occupancy at consensus sites, suggesting the disruption of the YAP/TAZ/TEAD/FOSL1 transcriptional circuit by the small molecule compound. To determine whether these phenotypes are recapitulated in vivo, we will treat mice harboring CRPC-SCL xenografts with TEAD inhibitors to assess growth response and evaluate epigenetic and transcriptional response using single-nucleus Multiome (ATAC+RNA). These studies will establish whether small molecule inhibition of TEAD is a promising strategy for the treatment of CRPC-SCL and allow high-resolution analysis of cell state transitions, with a focus on loss of SCL-specific signatures and potential emergence of AR/NE programs as adaptive resistance mechanisms.



 Chen Khuan Wong, Dan Li, Hongsu Wang, Marjorie Roskes, Weiling Li, Shipra Shukla, Dana Schoeps, Ekta Khurana, Yu Chen. Defining and targeting drivers of lineage plasticity in stem cell-like prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A074.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/912fdd784be0fd097e91b40d9e4c6933dd2c1b14" target='_blank'>
              Abstract A074: Defining and targeting drivers of lineage plasticity in stem cell-like prostate cancer
              </a>
            </td>
          <td>
            Chen Khuan Wong, Dan Li, Hongsu Wang, M. Roskes, Weiling Li, Shipra Shukla, Dana M. Schoeps, E. Khurana, Yu Chen
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Somatic mosaicism is pervasively observed in human aging, with clonal expansions of cells harboring mutations in recurrently mutated driver genes. Bulk sequencing of tissues captures mutation frequencies, but cannot reconstruct clonal architectures nor delineate how driver mutations impact cellular phenotypes. We developed single-cell Genotype-to-Phenotype sequencing (scG2P) for high-throughput, highly-multiplexed, joint capture of genotyping of mutation hotspots and mRNA markers. We applied scG2P to aged esophagus samples from six individuals and observed large numbers of clones with a single driver event, accompanied by rare clones with two driver mutations. NOTCH1 mutants dominate the clonal landscape and are linked to stunted epithelial differentiation, while TP53 mutants promote clonal expansion through both differentiation biases and increased cell cycling. Thus, joint single-cell highly multiplexed capture of somatic mutations and mRNA transcripts enables high resolution reconstruction of clonal architecture and associated phenotypes in solid tissue somatic mosaicism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e453d2ebb8e890d0f12017626a25469ee8fcb34" target='_blank'>
              Genotype-to-phenotype mapping of somatic clonal mosaicism via single-cell co-capture of DNA mutations and mRNA transcripts.
              </a>
            </td>
          <td>
            Dennis J. Yuan, John Zinno, Theo Botella, Dalia Dhingra, Shu Wang, Allegra G Hawkins, Ariel D Swett, Jesus Sotelo, Ramya Raviram, Clayton Hughes, Catherine Potenski, Katharine D Godfrey, Kara M Ainsworth, Shuzhen Xu, Jianwen Que, Julian A Abrams, Akira Yokoyama, N. Kakiuchi, Seishi Ogawa, D. Landau
          </td>
          <td>2025-12-31</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Transcriptional reprogramming through induced proximity has emerged as a powerful strategy for modulating the expression of oncogenic and tumor-suppressive genes. Inspired by transcriptional reprogramming approaches such as transcriptional/epigenetic chemical inducers of proximity (TCIPs) that link BCL6 inhibitors to transcriptional regulators, we sought to develop covalent ligands that rewire BCL6 proximity to selectively suppress MYC transcriptional output while derepressing BCL6 target loci. Through a chemistry-driven and chemoproteomics-enabled design strategy, we generated a panel of BCL6-based electrophile-bearing hybrid ligands and identified a nondegradative molecular glue, ZD-1-186, that potently suppresses MYC and robustly induces CDKN1A (p21) in diffuse large B-cell lymphoma cells. ZD-1-186 downregulates MYC more effectively than BCL6 inhibitors or degraders, while strongly derepressing canonical BCL6 targets, including p21. Through BCL6 pulldown proteomics, ZD-1-186 induced a selective recruitment of the noncanonical BAF complex subunit BRD9 to BCL6 and covalently modified BRD9 at C288. Pharmacologic inhibition or genetic knockdown of BRD9 attenuated ZD-1-186-mediated MYC suppression and blunted p21 induction. Transcriptomic profiling of ZD-1-186 showed simultaneous derepression of BCL6-repressive loci and suppression of MYC transcriptional programs. These findings demonstrated that ZD-1-186 acted as a transcriptional rewiring glue, recruiting BRD9 to BCL6-repressive loci to activate tumor-suppressive transcription, while also potentially redirecting BCL6 to BRD9-bound oncogenic loci. Overall, our work provides a blueprint for the rational discovery and design of electrophile-enabled, nondegradative molecular glues for targeted transcriptional rewiring.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01e1cf0470c87b88a6512470358d48dec81cbfa4" target='_blank'>
              Discovery of Non-Degradative Covalent Molecular Glues for Transcriptional Reprogramming
              </a>
            </td>
          <td>
            Tuong Nghi Duong, Edward Pandji, Qian Shao, Daniel K. Nomura
          </td>
          <td>2025-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e04b464c0e1b167410299c1bc664ca1e5464bb8" target='_blank'>
              cellSTAAR: incorporating single-cell-sequencing-based functional data to boost power in rare variant association testing of noncoding regions.
              </a>
            </td>
          <td>
            Eric Van Buren, Yi Zhang, Xihao Li, M. Selvaraj, Zilin Li, Hufeng Zhou, Nicholette D. Palmer, Donna K. Arnett, J. Blangero, Eric Boerwinkle, B. Cade, Jenna C. Carlson, April P. Carson, Yii-DerIda Chen, J. Curran, R. Duggirala, Myriam Fornage, Nora Franceschini, M. Graff, C. Gu, Xiuqing Guo, Jiang He, Nancy Heard-Cosa, Lifang Hou, Yi-Jen Hung, Rita R. Kalyani, Sharon L. R. Kardia, Eimear E. Kenny, C. Kooperberg, B. Kral, Leslie Lange, Daniel Levy, Changwei Li, Simin Liu, D. Lloyd-Jones, R. Loos, A. Manichaikul, L. W. Martin, Rasika A Mathias, R. Minster, Braxton D. Mitchell, J. Mychaleckyj, T. Naseri, Kari E. North, Jeffry R. O'Connell, James A. Perry, P. Peyser, B. Psaty, L. Raffield, Ramachandran S. Vasan, S. Redline, Alexander P. Reiner, Stephen S. Rich, Jennifer A. Smith, Brian W. Spitzer, Hua Tang, Kent D. Taylor, Russell Tracy, S. Viali, Lisa R. Yanek, Wei Zhao, Jerome L. Rotter, G. Peloso, P. Natarajan, Xihong Lin
          </td>
          <td>2025-12-31</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="BACKGROUND
Mutant IDH1 (mIDH1) defines a therapeutically-targetable subtype of intrahepatic cholangiocarcinoma (ICC), with the mIDH1 inhibitor ivosidenib approved for advanced disease. A subset of patients experiences prolonged disease stabilization, although the molecular basis for eventual progression remains poorly defined.


EXPERIMENTAL DESIGN
We performed molecular profiling of matched baseline and post-progression circulating tumor DNA (ctDNA) samples from patients with mIDH1 ICC enrolled in the ClarIDHy phase III trial. Functional studies were conducted to characterize candidate resistance mechanisms.


RESULTS
Longitudinal ctDNA analysis of 18 patients treated with ivosidenib for >6 months revealed emergent genomic alterations in multiple cases. Acquired mutations in MAPK-pathway genes (KRAS, NRAS, MAP2K1, NF1) were identified in five cases, with instances of concurrent alterations and/or high variant allele fractions (VAF). Additional candidate resistance events included a secondary IDH1 mutation and a hotspot IDH2 mutation, detected at low VAF in the same patient. Functional studies demonstrated that these IDH mutations conferred sustained 2-hydroxyglutarate (2HG) production and ivosidenib resistance, whereas MAPK activation blunted gene expression induced by ivosidenib plus interferon-γ, a key therapeutic output of mIDH1 inhibition. In parallel, baseline ctDNA profiling of 81 patients linked ARID1A mutations and elevated mIDH1 VAF to reduced clinical benefit.


CONCLUSIONS
MAPK-pathway alterations represent a recurrent mechanism of resistance to mIDH1 inhibition in ICC, while emergent IDH1/IDH2 mutations appear infrequent. Functional data suggest that MAPK-mediated resistance may involve impaired IFN signaling. These results support MAPK-directed combination strategies and highlight the utility of ctDNA profiling to identify predictive and resistance biomarkers in mIDH1-driven ICC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4aefbd8f840b00705379daf10a2c077d305123a7" target='_blank'>
              MAPK Pathway Mutations Emerge in Mutant-IDH1 Inhibitor-Resistant Cholangiocarcinoma and Attenuate the Interferon Response.
              </a>
            </td>
          <td>
            Jinkai Wan, H. Saatcioglu, H. Ellis, E. Aguado-Fraile, Qin Xu, Hiroshi Kondo, R. Manguso, I. Gritti, B. Norden, Carolina Noble, R. Corcoran, Sam J Lubner, M. Javle, G. Abou-Alfa, James M Cleary, R. Kelley, M. Borad, T. Macarulla, Do-Youn Oh, Scott Daigle, C. Gliser, Susan Pandya, Sung Choe, A. Tron, N. Bardeesy
          </td>
          <td>2025-12-29</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>105</td>
        </tr>

        <tr id="Conventional cancer therapies, including radiation therapy and chemotherapy, rely on inflicting DNA damage, yet they inevitably affect normal cells, leading to severe adverse effects. The advent of precision chemotherapy exploiting tumor-specific DNA repair defects has validated the effectiveness of this approach. The first successful example is PARP inhibitors, which selectively kill homologous recombination (HR) defective cancers, such as familial breast cancer possessing HR deficiency due to BRCA gene mutations. However, the broader landscape of DNA maintenance-including DNA replication, repair, and checkpoint pathways-harbors numerous mutations in tumors that remain untargeted. Here, we propose repurposing chain-terminating nucleoside analogs (CTNAs) to target such cancers' vulnerabilities. CTNAs, long utilized as anti-cancers and anti-viral drugs, inhibit replication and thereby suppress growth, but their activity has never been systematically aligned with specific cancer mutations associated with DNA maintenance defects. Based on our recent studies, we demonstrate that CTNAs elicit synthetic lethality in cells deficient for distinct DNA maintenance systems, amplifying replication stress, leading to cell death. We highlight the spectrum of CTNA-induced lesions and repair pathways required for cellular tolerance. This framework presents a versatile "repair-defect-guided" chemotherapy that expands the clinical utility of CTNAs and improves therapeutic effect by reducing side effects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e63be07db0a569b7816efc4ad2f977abe094978" target='_blank'>
              Targeting Genome Maintenance Defects of Cancers Using Chain-Terminating Nucleoside Analogs.
              </a>
            </td>
          <td>
            Ryotaro Kawasumi, Rubaiat E Tabassum, Kouji Hirota
          </td>
          <td>2025-12-10</td>
          <td>Cancer science</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="Background/Objectives: Current genomics research equates the genome with DNA sequence and treats the epigenome as a regulatory layer. This DNA-centric view obscures the fact that genomic identity arises through epigenomic processes. The objective of this article is to reinterpret published findings into a new theoretical framework: the EpG2 (Epigenome–Genome) system. Methods: This work develops a new conceptual framework by integrating published evidence from diverse domains—including enhancer biology, overlapping genomic functions, alternative coding frames, zygotic genome activation, and disease-associated loci—and reinterpreting these findings through the lens of epigenomic processes. Results: Evidence shows that enhancers emerge only through the interplay of sequence, transcription factors, and chromatin environment. At fertilization, paternal and maternal genomes remain separate, and a new genome emerges through coordinated epigenomic reprogramming or zygote genome emergence (ZGE). DNA sequence risk variants illustrate the concept of contextual risk alleles, whose effects shift across tissues and developmental stages as epigenomic contexts change. Conclusions: The EpG2 system reframes the genome as a processual, emergent entity generated and regulated by epigenomic processes, offering a paradigm for understanding genomic variation beyond DNA sequence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2af39b725984a653ec4ca7bda8e349ac7f79a7de" target='_blank'>
              Introducing the EpG2 System: Epigenomic Processes and the Emergent Genome
              </a>
            </td>
          <td>
            E. Ruiz-Narváez
          </td>
          <td>2025-12-01</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Chronic myeloid leukemia is a hematopoietic stem cell malignancy characterized by excessive proliferation of white blood cells, primarily driven by the Philadelphia chromosome (t 9;22). Despite the availability of tyrosine kinase inhibitors, disease progression and therapy resistance remain major challenges, often linked to additional somatic mutations. Objectives: To find out the mutational status and expression variations of the ASXL1 gene hotspot region in CML patients treated by TKIs. Methods: This retrospective cross-sectional study was conducted at the Department of Zoology, Division of Science and Technology, University of Education, Lahore, Pakistan. DNA and mRNA of 50 CML patients were analyzed alongside 10 healthy controls and statistically assessed using SPSS. Results: Results showed that sequence analysis of exon 13 identified hotspot mutations, including two novel missense alterations, G659S and D667E, within the coding region of ASXL1. In-silico analysis also suggested potential structural and functional relevance of these variants. The ASXL1 gene expression showed a progressive but statistically non-significant down-regulation in CML patients across disease phases (p=0.662), disease duration (p=0.42), treatment with second-line versus first-line TKIs (p=0.412), and wild-type and mutant (p=0.544). Conclusions:  It is concluded that two novel missense alterations, G6559S and D667E, were identified in the hotspot region, and ASXL1 gene expression was downregulated non-significantly in CML patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14ee16b9a16245af7fb856280afba55f65850e27" target='_blank'>
              Identification of Novel Alterations in the ASXL1 Gene and Its Expression Profile in CML Patients
              </a>
            </td>
          <td>
            R. Mehmood, Khuram Shehzad Khan, A. Akram, Sikandar Hayat, Amjad Zafar
          </td>
          <td>2025-12-31</td>
          <td>Pakistan BioMedical Journal</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/131d5075adfc8a892d5a30b581d5ddea0c0afd87" target='_blank'>
              Single-cell transcriptomic analysis reveals cellular heterogeneity and prognostic subtypes in colorectal cancer.
              </a>
            </td>
          <td>
            Huizhen Han, Yuan Gao, Yuxing Zhao, Jiahong Wu, Jia Chen, Jinling He, Hong Bai, Liang Qiao, Yali Ren, Lei Sun
          </td>
          <td>2025-12-22</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Genomic imprinting, an epigenetic process resulting in parent-specific gene expression, is essential for normal development and growth. Disruption of imprinting leads to various developmental disorders and cancers, yet our understanding of the full repertoire of imprinted genes in humans remains incomplete. Here, we utilised androgenetic, parthenogenetic and biparental human embryonic stem cells and their neural derivatives to identify novel imprinted genes by analysing their methylome and transcriptome profiles. Our analysis revealed 12 novel putative imprinted genes distributed across four distinct loci, with six of them clustered in an uncharacterised imprinted region on chromosome 19. We identified potential imprinting control regions regulating this novel cluster, suggesting a coordinated regulatory mechanism. Notably, these imprinted genes are enriched in cancer-related pathways, with several showing isoform-specific imprinting patterns. Our analysis also revealed consistent DNA methylation aberrations in pluripotent stem cells at specific imprinted loci, highlighting potential epigenetic instability during culturing. These findings contribute to our understanding of genomic imprinting regulation in human development and highlight potential genomic regions for further investigation of imprinting-related disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac9fb51dc7090ae00e411986f95ea22a3b88f723" target='_blank'>
              Analysis of Human Uniparental Embryonic Stem Cells Reveals New Putative Imprinted Loci.
              </a>
            </td>
          <td>
            Shay Kinreich, N. Benvenisty
          </td>
          <td>2025-12-02</td>
          <td>Cell proliferation</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4b00e19dbeba7049a53aa78186bf5b19a36b34d" target='_blank'>
              A genome-wide CRISPR/Cas9 screen reveals novel positive regulators of FTY720 sensitivity in acute lymphoblastic leukemia cells.
              </a>
            </td>
          <td>
            Jaira Ferreira de Vasconcellos, Leah Friedman, Isha Satapathy, Nicole Cubbage, Jasmin Palmer, Saurav Majumder, Mari Kono
          </td>
          <td>2026-01-23</td>
          <td>BMC research notes</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Retrotransposons are genomic parasites frequently reactivated in cancers, where their mobility can cause genetic alterations. However, it remains unclear whether their gene products contribute to cancer beyond mutagenesis. Here, we uncover a chromatin-associated function of RNAs from Long Interspersed Element-1 (LINE-1), the only autonomous retrotransposon in the human genome. Subcellular-resolved transcriptomics revealed that LINE-1 RNAs are primarily nascent transcripts produced by full-length, cell-type-specific genomic copies of evolutionarily young subfamilies. Using a long-read chromosome conformation assay, we identified a class of highly interactive LINE-1 loci required for gene expression across cancer subtypes, revealing an unexpected regulatory role for LINE-1 locus transcription in oncogenic gene control. LINE-1 RNA depletion disrupted LINE-1-centric chromatin interactions and downregulated associated genes, whereas genomic insertion of an inducible LINE-1 generated de novo chromatin interactions in a transcription-dependent manner. Therefore, beyond their mutagenic potential, retrotransposons also regulate cancer gene expression by nucleating chromatin architecture through their transcriptional activity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e9e6bb366cb35c566b3052102db69c89368901c" target='_blank'>
              LINE-1 Locus Transcription Nucleates Oncogenic Chromatin Architecture.
              </a>
            </td>
          <td>
            Michael Lee, Yuannyu Zhang, Jun Yi Stanley Lim, Tao Dai, James S Ye, Margaret B Cervantes, Varun Sondhi, Sisi Zheng, Yoon Jung Kim, Brandon Chen, R. DeBerardinis, Jian Xu
          </td>
          <td>2026-01-05</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Cellular differentiation is driven by epigenetic modifiers and readers, including the methyl CpG binding protein 2 (MeCP2), whose level and mutations cause the neurological disorder Rett syndrome. During differentiation, most of the genome gets densely packed into heterochromatin, whose function has been simplistically viewed as gene silencing. However, gene expression changes reported in mutations leading to Rett syndrome have failed to be a predictor of disease severity. Here we show that MeCP2 increases nuclear stiffness in a concentration-dependent manner and dependent on its ability to cluster heterochromatin during differentiation. MeCP2-dependent stiffness increase could not be explained by changes in the expression of mechanobiology-related genes, but we found that it is disrupted by Rett syndrome mutations and correlated with disease severity. Our results highlight the impact of chromatin organization on the mechanical properties of the cell as an alternative or complementary mechanism to changes in cytoskeleton components. A correlative AFM–fluorescence approach elucidates how epigenetic control of chromatin organization contributes to nuclear stiffness and its dysregulation in Rett syndrome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d330823c212a15c08be21ea0eb068997201f78c8" target='_blank'>
              MeCP2-driven chromatin organization controls nuclear stiffness
              </a>
            </td>
          <td>
            Hector Romero, Anahid Amiri, M. K. Pabba, Hui Zhang, Veronika Berg, Maria Arroyo, Paulina Prorok, Andreas Zhadan, Marah Mahmoud, Nina Trautwein, Bodo Laube, Christian Dietz, R. Stark, Cristina Cardoso
          </td>
          <td>2025-12-08</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 A crucial adaptability trait of melanoma cells that persist under all selection pressures, including therapies, is their ability to efficiently switch from proliferation to quiescence. With the goal of improving therapeutic strategies for preventing melanoma recurrence, herein we describe an approach to modeling this trait in cell culture. Body imposes a variety of selection pressures and bottlenecks ranging from different components of immunity to metabolic challenges to enforce quiescence and influence cancer evolution. The rationale for the choice of model as cell culture is that it is ideal for modeling the realistic depth of quiescence and for evaluating therapeutic strategies that may inhibit highly abnormal but adaptable cancer cells. As it is not possible to replicate all such bottlenecks in cell culture, our approach is to apply as severe a bottleneck as possible, thus eliminating all proliferative (non-adaptable) cells, leaving behind rare cells surviving in deep quiescence. We hypothesize that if the quiescent cell state being modeled can survive a severe bottleneck, it can also survive other bottlenecks of similar or lower severity. A lack of glutamine proved to be a strong and prolonged selection pressure for the highly metastatic human melanoma cell line A375SM, killing more than 99% of cells and selecting rare cells based on their ability to survive in deep quiescence. After 4 weeks, cells gradually exited quiescence and proliferated indefinitely. Furthermore, we modeled even deeper quiescence lasting longer than 4 weeks by increasing the severity of the selection pressure using dialyzed serum in medium. The cells selected in this manner were much more resistant to paclitaxel than was their parental cell line. We obtained similar results with the highly metastatic mouse melanoma cell line B16-BL6. We conclude that our approach is suitable for modeling abnormal deep quiescence in melanoma that drives the disease, including recurrence and metastasis. The main strength of the approach is that, besides modeling genetic alterations, it incorporates suitable selection pressures and the time element that are important in cancer evolution. Although there is a limitation of lack of body-like microenvironments in cell culture, we believe that abnormal quiescent cancer cell state being modeled here is capable of overcoming restraints imposed by the body. We will also discuss our published and new observations with rare cells (0.01% in population) surviving in quiescence for several weeks in aggressive triple-negative breast cancer cell lines and their functional characteristics. Our results obtained with adaptable cells from SUM149 TNBC cell line, modeled as above for melanoma cells, suggest that they are stem-like quiescent cells. We will also discuss how we choose and evaluate therapies that would likely inhibit quiescent and slow-growing cancer cells prior to recurrence. This approach may significantly improve how we investigate deep intrinsic resistance and strategies to overcome it.



 Balraj Singh, Vanessa N. Sarli, Nikil Erry, Anthony Lucci. A cell culture model of quiescent melanoma cell state that governs cancer evolution, therapy resistance, and recurrence/metastasis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr A010.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56b4de6eb92fbb6a26e62be45c5389b5a53529d5" target='_blank'>
              Abstract A010: A cell culture model of quiescent melanoma cell state that governs cancer evolution, therapy resistance, and recurrence/metastasis
              </a>
            </td>
          <td>
            Balraj Singh, V. Sarli, Nikil Erry, Anthony Lucci
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="BACKGROUND
Neuroblastoma is the most common extracranial solid tumor in children and exhibits substantial clinical heterogeneity. Although key genetic alterations such as N-Myc proto-oncogene protein amplification and anaplastic lymphoma kinase mutations are known drivers of neuroblastoma, their clinical utility is limited by population-specific genetic diversity. To address this, we propose a "function-over-gene" strategy by evaluating extracellular matrix (ECM)-related gene sets as molecular biomarkers, aiming to overcome the limitations posed by genetic heterogeneity and provide novel prognostic insights.


METHODS
We integrated data from a single-center cohort and the TARGET database. Cox regression models were used to assess the prognostic value of ECM gene alterations and their association with clinical outcomes. Gene set enrichment analysis (GSEA) was employed to identify ECM-related gene sets, followed by transcriptomic analysis to explore downstream regulatory pathways.


RESULTS
In the single-center cohort, ECM gene mutations were potentially associated with bone and lymph node metastases and emerged as an independent predictor of poor prognosis (HR=2.7, P=0.02) in multivariate analysis. Validation using the TARGET cohort confirmed the prognostic relevance of the ECM gene set (HR=1.55, P=0.0083) and revealed its involvement in modulating the tumor microenvironment via immune and complement pathways.


CONCLUSION
ECM gene signatures serve as robust prognostic markers across populations. This function-based approach offers a novel perspective to address genetic heterogeneity and provides a theoretical foundation for ECM-targeted combination therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d011b508eee694993a004e089261b1d23283417" target='_blank'>
              Dysregulation of Extracellular Matrix Genes Identifies Neuroblastoma Patients at High Risk of Recurrence and Poor Outcome.
              </a>
            </td>
          <td>
            Xiaoyu Jing, Shijing Ge, Guoqian He, Sijia He, Ju Gao, Chaoban Wang, Xia Guo
          </td>
          <td>2026-01-15</td>
          <td>Journal of pediatric hematology/oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92b7de6d8aeceaad9c8d57eb1d9c3d308c5a80e2" target='_blank'>
              Ribosome biogenesis as a potential therapeutic target in KRAS mutant colorectal cancer.
              </a>
            </td>
          <td>
            Yui Tanaka, M. Sakahara, H. Yamanaka, Yasuko Natsume, Daisuke Kusama, Kohei Kumegawa, Harunori Yoshikawa, Yuich Abe, K. Okabayashi, Shimpei Matui, Y. Kitagawa, Naohiko Koshikawa, H. Osumi, E. Shinozaki, Satoshi Nagayama, Jun Adachi, R. Maruyama, Ryoji Yao
          </td>
          <td>2025-12-27</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="The IFNG gene encodes Interferon-gamma (IFN-γ), a critical mediator that coordinates both innate and adaptive immune mechanisms. Polymorphisms in this gene can substantially affect immunological performance and predisposition to pathological conditions. This study systematically assessed the biological impact of missense mutations in the human IFNG gene using an integrated computational methodology. From the NCBI database, 170 nonsynonymous single-nucleotide polymorphisms (nsSNPs) were extracted and subjected to analysis through eight distinct bioinformatics platforms. The functional consequences were predicted using SIFT, PROVEAN, PolyPhen-2, and PANTHER computational frameworks. Disease-related significance was determined via SNPs&GO, PhD-SNP, and Meta-SNP analytical tools, whereas MutPred2 elucidated the molecular mechanisms underlying variant deleterious effects. The analysis identified eight high-risk variants (V45E, L51P, W59R, S70I, I72N, D125Y, L136H, V139E, and L143P) showing consistent pathogenic predictions across multiple algorithms. Different numbers of potentially deleterious mutations were identified by the various analysis platforms, namely, SIFT identified 87 potentially deleterious mutations, PROVEAN identified 68 mutations, PolyPhen-2 identified 94 mutations, and PANTHER detected 91 potentially pathogenic mutations. Disease association analysis revealed 14 additional mutations that are consistently classified as pathogenic, in addition to the eight prioritized variants that demonstrated the strongest clinical relevance. The research findings predict potential biomarkers for immune-related disease susceptibility and provide a robust computational framework for identifying pathogenic variants. The identified high-risk variants represent promising targets that require urgent experimental validation to confirm their functional and clinical impact before considering any therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4470109088cb8fb24abdafa69118b673f90cd75b" target='_blank'>
              Integrative computational analysis of IFNG gene variants: Identifying high-risk mutations affecting interferon-gamma structure and function
              </a>
            </td>
          <td>
            A. R. Al-Zaalan, Hussam Saadi Aziz, Noor Fadhil Abdulabbas
          </td>
          <td>2025-12-17</td>
          <td>Asia-Pacific Journal of Molecular Biology and Biotechnology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background/Aim: Breast cancer (BCa) remains one of the most prevalent malignancies and a leading cause of cancer-related deaths globally. Understanding the genetic underpinnings of BCa is critical for advancing precision medicine, including the development of predictive biomarkers and repurposed therapies. This study aimed to identify and functionally annotate BCa-associated single nucleotide polymorphisms (SNPs) to identify biological risk genes and assess drug repositioning opportunities. Materials and Methods: We extracted BCa-related SNPs from the GWAS Catalog, applying a genome-wide significance threshold (p-value <10−8) to identify 1,219 SNPs. From these, 14 missense variants were prioritized and evaluated using six complementary tools: missense annotation, cis-expression quantitative trait loci (eQTL) mapping, combined annotation dependent depletion (CADD), sorting intolerant from tolerant (SIFT), polymorphism phenotyping v2 (PolyPhen-2) and AlphaMissense. Genes were scored across these criteria, with those scoring ≥2 considered biologically relevant. GTEx data was used to assess tissue-specific gene expression. Allele frequencies across populations were obtained from the Ensembl database, and druggability was evaluated using DrugBank. Results: We identified nine genes achieving the maximum score of 4 as the highest-priority candidates SLCO1B1 (rs4149056), ARHGEF38 (rs61751053), EXO1 (rs4149909), KDELC2 (rs74911261), MAPT (rs63750417), PHLDA3 (rs35383942), AKAP9 (rs6964587), ATXN7 (rs1053338) and DCLRE1B (rs11552449). SLCO1B1 (rs4149056) and ARHGEF38 (rs61751053), supported by functional and regulatory evidence. SLCO1B1, predominantly expressed in the liver, may influence BCa metastasis and drug metabolism; its variant shows population-specific allele distribution, being particularly higher in Europeans (16%). ARHGEF38, though variably expressed across tissues, may play regulatory roles relevant to tumorigenesis. Among the prioritized genes, MAPT was identified as the only druggable target, with existing therapeutics such as paclitaxel and docetaxel indirectly linked to its function, suggesting potential for drug repurposing. These findings provide a foundation for further studies on SNP-guided biomarkers and repositioned therapies targeting key BCa-related genes. Conclusion: This integrative bioinformatics approach prioritized functionally significant BCa-associated SNPs and identified promising candidates for biomarker development and drug repositioning. The nine high-scoring variants, including SLCO1B1 and ARHGEF38 emerge as biologically impactful genes, while MAPT’s known drug interactions highlight its translational potential in repurposing existing anticancer agents.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9312e199b9466799d1e9fb1fc384373810795e6" target='_blank'>
              Leveraging Genome-wide Association Studies to Identify Pathogenic Variants for Breast Cancer Among Multiple Continents
              </a>
            </td>
          <td>
            Putri Permata Suka Admanegara, Rista Yulianti, Desti Rahmawati, Siska Widiastuti, W. Adikusuma, Brilliant Citra Wirashada, D. Amukti, D. Darmawi, Baik Heni Rispawati, Benni Iskandar, Rockie Chong, I. Ates, L. Irham
          </td>
          <td>2025-11-29</td>
          <td>AntiCancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edfd2702367051da70ddb6fa2941183d106513bc" target='_blank'>
              The expanding roles of homologous recombination proteins in genome stability.
              </a>
            </td>
          <td>
            Lorenzo Sassi, Andrea Martinez Marroquin, Salli Waked, Alessandra Ardizzoia, Vincenzo Costanzo
          </td>
          <td>2026-01-03</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Two non-synonymous single-nucleotide polymorphisms (SNPs) rs61752561 (D95N substitution) and rs17632542 (I163T substitution) in the KLK3 gene encoding prostate-specific antigen (PSA), a chymotrypsin-like serine protease, are associated with prostate cancer risk and have been shown to reduce the activity of PSA. However, the structural impact of these SNPs on PSA, which may underlie the observed risk associations and functional alterations, has not been fully explored. Computational modelling predicted that the variants D95N and I163T do not cause drastic structural changes in PSA. However, molecular dynamics simulations suggested that while the two prominent loops of wild-type PSA remain tethered to their initial conformations over 500 ns of simulation, they are disrupted in both variants, leading to increased loop dynamics. Frustration analysis, normal mode analysis (NMA) and perturbation response scanning identified dynamic links between mutation sites and increased loop dynamics that trigger long-range conformational changes, disrupting the active site and potentially hindering catalytic activity. Thermal denaturation stability assays using recombinant protein show the impact of D95N and I163T substitution on the protein stability. These data show that KLK3 SNPs disrupt dynamic communication of the key loops required for proteolytic activity of PSA, which may explain the association of these SNPs with prostate cancer risk and/or progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/387e91e1f338cbf07e62b9545288e8b8401fb288" target='_blank'>
              Prostate cancer risk-associated single-nucleotide polymorphisms impact the conformational dynamics of prostate-specific antigen
              </a>
            </td>
          <td>
            Srilakshmi Srinivasan, Brooke K Hayes, Mauricio G S Costa, Blake T Riley, Emily Wilson, Emilia M. Marijanovic, Itamar Kass, Hannu Koistinen, D. Hoke, Judith A. Clements, Ashley M Buckle, J. Batra
          </td>
          <td>2025-12-17</td>
          <td>BMC Biology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="


 Androgen receptor (AR) signaling is critical for the survival and proliferation of prostate cancer (PCa) cells, thus making androgen deprivation therapy (ADT; e.g., castration) the mainstay for treatment. Notably, the oncogenic transcriptional functions of AR rely on a host of chromatin-binding regulatory proteins, which includes a pioneer transcription factor called FOXA1. FOXA1 de-compacts chromatin to enable DNA binding of AR and activation of target gene expression. Our lab found FOXA1 alterations to recur within three distinct structural classes in over 35% of metastatic castration-resistant PCa (mCRPC) cases in Caucasian White men. Subsequent studies reported that FOXA1 mutations are prevalent in over 40% of primary PCa in Chinese patients, thus positioning FOXA1 as a principal oncogene in this disease. Yet, the tumorigenic potential and pathobiology of FOXA1 alterations remain unexplored in vivo. Here, we have developed and characterized the first in-field transgenic mouse models that conditionally overexpress FOXA1 mutants in the prostate luminal epithelia. Our findings reveal that truncal FOXA1 class1 mutations (i.e., wing 2 alterations) in a Trp53-null background triggered high-grade adenocarcinoma mimicking human disease, characterized by abnormal induction of NSD2 expression, extensive AR reprogramming to chimeric AR-half neo-enhancers, and sensitivity to androgen deprivation therapies. In contrast, FOXA1 class 2 mutations (i.e., C-terminal truncations) that are acquired in human mCRPC do not drive prostate luminal transformation in mice. Instead, these mutations induce intra-luminal plasticity by reprogramming differentiated cells into a progenitor stem-like state within androgen-replete normal tissues—a phenomenon otherwise induced upon castration. Class 2 mutants lead to a 20-fold expansion of Ar+/Ck8+ luminal epithelia that gain expression of stemness markers such as Trop2, Ck4, and Psca. These stem cells are similar to the Club cells detected in the human prostate gland that have been implicated in driving resistance to ADT. Mechanistically, we found the cistromically-dominant Class 2 mutants to pioneer over 40,000 neo-enhancer elements that were bound by stemness-associated transcription factors, like KLF5 and AP-1, which together instruct an androgen-insensitive luminal progenitor cell fate. Consistently, we found that Class 2-mutant mouse prostates showed minimal atrophy upon castration, with immunohistological assessment revealing a higher density of Ki67+ luminal epithelial cells relative to both wild-type and Class 1-mutant tissues. Class 2-mutant organoids also showed greater subcutaneous grafting ability in limiting-dilution assays in mice. Collectively, our data establish FOXA1 as a multifaceted oncogene in AR-dependent prostate cancers, in which divergent evolution of FOXA1 mutational classes distinctly drives either cancer formation or therapy-resistant, intra-luminal plasticity during disease progression. These findings also represent the first report of FOXA1-driven prostate adenocarcinoma in mice.



 Sanjana Eyunni, Rahul Mannan, Yuping Zhang, Eleanor Young, Qiuyang Zhang, Jie Luo, Matthew Pang, Somnath Mahapatra, Jean Ching-Yi Tien, James George, Mustapha Jaber, Hamzah Hakkani, Sandra E. Carson, Abigail J. Todd, Noshad Hosseini, Mahnoor Gondal, Ryan J. Rebernick, Xuhong Cao, Fengyun Su, Rui Wang, Rohit Mehra, Jing Li, Marcin Cieslik, Arul M. Chinnaiyan, Abhijit Parolia. Divergent FOXA1 mutations drive prostate tumorigenesis and therapy-resistant cellular plasticity [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr PR012.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e838d76f934b647aed2d9fe44ea43b83b285eb1c" target='_blank'>
              Abstract PR012: Divergent FOXA1 mutations drive prostate tumorigenesis and therapy-resistant cellular plasticity
              </a>
            </td>
          <td>
            Sanjana Eyunni, R. Mannan, Yuping Zhang, E. Young, Qiuyang Zhang, Jie Luo, Matthew Pang, Somnath Mahapatra, Jean Ching-Yi Tien, James M. George, Mustapha Jaber, Hamzah Hakkani, Sandra E Carson, Abigail J Todd, Noshad Hosseini, M. Gondal, Ryan J. Rebernick, Xuhong Cao, Fengyun Su, Rui Wang, Rohit Mehra, Jing Li, Marcin Cieslik, A. Chinnaiyan, A. Parolia
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>170</td>
        </tr>

        <tr id="Autoimmune diseases (AIDs) are chronic disorders driven by the breakdown of immune tolerance, where genetic susceptibility and environmental triggers converge to promote aberrant T and B cell activation, inflammatory cytokine release, and progressive tissue damage. Current therapeutic strategies, including corticosteroids, immunosuppressants, and biologics, can alleviate symptoms but remain limited by adverse effects, incomplete remission, and variable patient responses. CRISPR interference (CRISPRi), a novel gene regulation platform based on catalytically inactive Cas9 and guide RNAs, achieves precise, reversible suppression of target genes without inducing DNA double-strand breaks, thereby offering improved safety and controllability over conventional genome editing. Recent studies demonstrate that CRISPRi can effectively silence multiple inflammatory mediators and immune checkpoint molecules, enabling immune microenvironment remodeling and providing a feasible strategy for complex autoimmune conditions. Advances in multiplex CRISPRi, including the design of multi-sgRNA constructs and delivery via viral or non-viral systems, expand its capacity to simultaneously repress synergistic cytokine networks. Despite ongoing challenges in delivery efficiency, off-target risk, and long-term safety, CRISPRi holds considerable promise for network-level therapeutic intervention. This review aims to summarize the principles, technological progress, and potential applications of CRISPRi in autoimmune diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1138bca3d3877bd7932621af6651b3b14d329c9b" target='_blank'>
              Potential Applications and Research Exploration of CRISPRi in the Treatment of Autoimmune Diseases
              </a>
            </td>
          <td>
            Shaopu Huang
          </td>
          <td>2025-12-28</td>
          <td>Academic Journal of Science and Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Normal somatic cells possess innate defense mechanisms that confer resistance to cancer-driving mutations, including cellular senescence activation upon oncogene overexpression. However, oncogenic mutations are detected in normal somatic tissues, suggesting that some cells are able to escape these defense mechanisms. We investigated whether all cells become senescent upon overexpressing oncogenic KRAS G12D in primary human keratinocytes and found that while most cells entered a non-dividing state consistent with oncogene-induced senescence, 10-30% escaped these defenses and continued dividing. Cellular barcoding revealed that sister cells typically shared the same fate upon oncogene overexpression, suggesting heritable, non-genetic differences determine a cell’s ability to escape senescence programs. Using single-cell RNA sequencing prior to oncogene overexpression, we identified transcriptional states associated with cells that ultimately escape senescence, thus marking a subpopulation of cells most susceptible to cancer-driving mutations. We also observed transcriptional variation amongst escaper cells that may interact with subsequent mutational events. Our work demonstrates that a cell's initial transcriptional state influences its response to cancer-driving mutations and suggests that non-genetic variation interacts with genetic mutations to drive somatic evolution and cancer progression, challenging the genetics-only paradigm of cancer initiation.



 Grant Kinsler, Arjun Raj. Pre-existing non-genetic cellular states determine susceptibility to oncogenic transformation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr A001.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95adbe47a20bb5a9feae6f4d099365519435f00f" target='_blank'>
              Abstract A001: Pre-existing non-genetic cellular states determine susceptibility to oncogenic transformation
              </a>
            </td>
          <td>
            Grant Kinsler, Arjun Raj
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="DNA double-strand breaks (DSB) represent one of the most severe forms of genomic damage. Thus, cells have evolved a complex network of DSB repair pathways, including homologous recombination, classical and alternative end joining, and single-strand annealing, which are tightly regulated by genetic and epigenetic factors. The selection and efficiency of these pathways influence genome integrity, oncogenesis, and therapeutic response. This comprehensive review synthesizes recent findings on the genetic regulation of DSB repair, with emphasis on pathway-specific regulators, chromatin context, and post-translational modifications. Moreover, this review integrates primary research from mammalian systems, including CRISPR-based studies, proteomics, and imaging, with a focus on publications from 2020 to 2025. We discuss the role of key players, such as MRE11-RAD50-NBS1 (MRN), ataxia telangiectasia mutated (ATM), mediator tumor suppressor p53-binding protein 1 (53BP1), breast cancer type 1 susceptibility protein (BRCA1), anti-silencing function 1 (ASF1), ring finger protein (RNF)8/168, DNA-dependent protein kinase catalytic subunit (DNA-PKcs), and RAD51 recombinase (RAD51), in orchestrating the associated pathway choice. Epigenetic modifications, RNA-mediated mechanisms, and chromatin remodeling dynamically influence the efficiency and fidelity of repair. Particular attention is provided to emerging regulators, including thyroid hormone receptor interactor 13 (TRIP13), ubiquitin-like with plant homeodomain (PHD) and RING finger domains 1 (UHRF1), Shieldin, and polymerase theta. This review highlights novel insights into transcription-associated DSB repair, the interplay of replication stress with repair pathway engagement, and context-dependent synthetic lethality. We also examine implications for cancer biology, including therapy resistance and biomarker development. Ultimately, understanding the genetic regulation of DSB repair pathways can provide critical insights into genome stability maintenance and reveal new therapeutic opportunities in cancer. Future work should focus on pathway crosstalk, phase-specific regulation, and integrating repair modulation into personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/940ba623f1d9b48456fecd2cfe27231108b1711a" target='_blank'>
              Genetic Regulation of DNA Double-Strand Breaks and Repair Pathways.
              </a>
            </td>
          <td>
            Lucián Zastko
          </td>
          <td>2025-12-17</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dee4eda01b3e677e9209b06281628a8f8ffa74f2" target='_blank'>
              Human infectious disease susceptibility: key perspectives
              </a>
            </td>
          <td>
            Geetha Bharathi, Avantika Bharathi, Kousik Saravana
          </td>
          <td>2026-01-09</td>
          <td>Egyptian Journal of Medical Human Genetics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a0b876781bf210db756c35db260709c192ea56b" target='_blank'>
              Genomic evolution of pancreatic cancer at single-cell resolution.
              </a>
            </td>
          <td>
            Haochen Zhang, P. Sashittal, Elias-Ramzey Karnoub, Akhil Jakatdar, Shigeaki Umeda, Jungeui Hong, Anne Marie Noronha, Agustin Cardenas, Amanda Erakky, Caitlin A McIntyre, Akimasa Hayashi, Nicolas Lecomte, Marc Hilmi, Wungki Park, Nan Pang, Eileen M. O’Reilly, Alice C. Wei, Benjamin J. Raphael, Christine A. Iacobuzio-Donahue
          </td>
          <td>2026-01-22</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Urothelial carcinoma of the bladder (UBC) is a heterogeneous disease from the clinicopathologic and molecular standpoint and presents clinical management problems for its range in progression, frequent recurrence and complicated genetic/epigenetic architecture. Not only traditional genetic changes, but also epigenetic dysregulation (e.g., alterations of chromatin remodeling, histone modification and DNA methylation) has been increasingly recognized to play a pivotal role in the process of tumor initiation, progression, metastasis and resistance to therapy. It is a dissapointing result that mutations of the chromatin remodeler such as ARID1A, KMT2C, KMT2D, and histone modifying enzymes (EP300, CREBBP and EZH2) not only cause loss-of-nucleosome positioning in addition lead to lack of enhancer function and perturbation ofT program contributing to shaping up intratumoral heterogeneity as well as transcriptional plasticity resulting in failure in immunoevasion. These changes sculpt the tumor microenvironment and impact the response to chemotherapy, immunotherapy, and targeted therapy.Epigenetic biomarkers based on mutation profiles, chromatin accessibility and DNA methylation signatures present new non-invasive strategies that can be followed for early detection, monitoring of disease and stratification of patients usage; which are all available in urine or plasma. Therapeutically, inhibitors of EZH2, HDACs, BET proteins, DNMTs and certain chromatin remodeling factors are currently being heavily investigated in preclinical studies or clinical trials as monotherapies or rational combinations with ICIs, chemotherapy or targeted therapies.Advances in tech nologies, such as single- and spatial genomics combined with AI-facilitated multi-omics integration make it possible to realize high-resolution maps of tumor heterogeneity, enhancer d namics, and epigenetic vulnerabilities. This review aims to present the current data on UBC epigenetic modifications and their mechanistic, biomarker and therapeutic implications. By combining mechanistic understanding with translational perspectives, we outline future directions to move precision oncology forward, optimize our treatment approach and improve clinical outcomes for UBC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c58b19d1ac81843818571ae3d302c08f8f24348" target='_blank'>
              Epigenetic Alterations and Chromatin Landscape Remodeling in Urothelial Bladder Cancer: Molecular Drivers, Biomarker Potential, and Emerging Therapeutic Strategies
              </a>
            </td>
          <td>
            Amr Abd-Elraheem Abdo, Ahmed Mohamed Kamel
          </td>
          <td>2025-12-10</td>
          <td>Annals of urologic oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The phenomenon of transgene silencing was first observed shortly after the generation of the initial transgenic plants. The vast body of experimental data accumulated since then constitutes an invaluable resource for dissecting the mechanisms of epigenetic gene regulation. Silencing operates at either the transcriptional (TGS) or post-transcriptional (PTGS) level and is predominantly mediated by small interfering RNAs (siRNAs). Although these two epigenetic pathways involve distinct sets of proteins and enzymes, they share fundamental mechanistic features: the generation of double-stranded RNA (dsRNA), its processing into siRNAs by DICER-LIKE (DCL) enzymes, and the assembly of an Argonaute-centered effector ribonucleoprotein complex (RISC). Guided by sequence-specific siRNAs, this complex identifies complementary target sequences with high precision. A comprehensive understanding of these regulatory pathways enables the targeted induction or suppression of specific plant genes. This review traces the history of experimental findings regarding the loss of recombinant gene activity in transformants and their progeny, which collectively established the foundation for elucidating the molecular mechanisms of transgene silencing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ca1645109162b7bd31d62f8f845655fb6172721" target='_blank'>
              Gene Inactivation in Transgenic Plants—A Unique Model for Studying Epigenetic Regulation of Gene Expression
              </a>
            </td>
          <td>
            T. Marenkova, A. Zagorskaya, I. Deyneko, Elena V. Deineko
          </td>
          <td>2026-01-13</td>
          <td>Plants</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e3ce0e5eeb35d45bde98218a3b940e6aa2d9ace" target='_blank'>
              Cell and Nuclear Size are Associated with Chromosomal Instability and Tumorigenicity in Cancer Cells that Undergo Whole Genome Doubling.
              </a>
            </td>
          <td>
            Mathew Bloomfield, Sydney M Huth, Daniella S McCausland, Ron Saad, Nazia Bano, Tran N Chau, Megan L Sweet, N. Baudoin, Andrew McCaffrey, Kai Fluet, Eva M. Schmelz, U. Ben-David, Daniela Cimini
          </td>
          <td>2026-01-22</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92529c21b1d903977db7b6f99cc016a10cc689ad" target='_blank'>
              Molecular analysis of tumor recurrence using established cancer stem cell-line and drug discovery.
              </a>
            </td>
          <td>
            Kiyotaka Nakano, E. Oki, Masaki Yamazaki, Masami Suzuki, S. Kawai, Y. Zaitsu, Chiyoko Nishime, K. Ando, G. Nagae, N. Harimoto, Mitsuhiko Ota, T. Kawazoe, Kentaro Nonaka, Keita Natsugoe, Sachie Omori, H. Saeki, Hiroyuki Aburatani, Tatsumi Yamazaki, Yoshihiko Maehara
          </td>
          <td>2026-01-20</td>
          <td>International journal of clinical oncology</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="The evolutionary history of cancers is encoded in molecular patterns that provide critical insights into the mechanisms of tumor progression. However, reconstructing cancer lineages in patient tumors remains challenging, as conventional sequencing captures only static snapshots of an ongoing evolutionary process. DNA methylation at CpG sites is a heritable epigenetic mark that accumulates stochastic errors - termed epimutations - at rates far exceeding those of somatic DNA mutations. These epimutations serve as a 'molecular clock' and can be harnessed for retrospective lineage tracing. When applied at single-cell resolution, methylation-based lineage tracing enables the reconstruction of cancer phylogenies and provides a powerful framework for studying the dynamics of treatment resistance, cell-state heritability, and metastasis directly in human tumors. In this review, we outline the strengths and challenges of single-cell DNA methylation-based lineage tracing, highlight key insights gained from its application with an emphasis on hematological cancers where relevant, and discuss its future potential in advancing our understanding of cancer evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1728d59760495508f83df1d25fc69028b0e41513" target='_blank'>
              Methylation-based lineage tracing in cancer.
              </a>
            </td>
          <td>
            Jiaoyi Chen, Benson Wu, Federico Gaiti
          </td>
          <td>2025-12-15</td>
          <td>Blood</td>
          <td>0</td>
          <td>17</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025', '2026'],
    y: [11, 1],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>