<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-02-24 09:26:48 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental and Molecular Medicine, Experimental & Molecular Medicine</td>
          <td>4</td>
          <td>49</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum.
                </a>
              </td>
          <td>
            Gustavo S França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>20</td>
          <td>41</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Prostate cancer is a common malignancy that in 5%-30% leads to treatment-resistant and highly aggressive disease. Metastasis-potential and treatment-resistance is thought to rely on increased plasticity of the cancer cells-a mechanism whereby cancer cells alter their identity to adapt to changing environments or therapeutic pressures to create cellular heterogeneity. To understand the molecular basis of this plasticity, genomic studies have uncovered genetic variants to capture clonal heterogeneity of primary tumors and metastases. As cellular plasticity is largely driven by non-genetic events, complementary studies in cancer epigenomics are now being conducted to identify chromatin variants. These variants, defined as genomic loci in cancer cells that show changes in chromatin state due to the loss or gain of epigenomic marks, inclusive of histone post-translational modifications, DNA methylation and histone variants, are considered the fundamental units of epigenomic heterogeneity. In prostate cancer chromatin variants hold the promise of guiding the new era of precision oncology. In this review, we explore the role of epigenomic heterogeneity in prostate cancer, focusing on how chromatin variants contribute to tumor evolution and therapy resistance. We therefore discuss their impact on cellular plasticity and stochastic events, highlighting the value of single-cell sequencing and liquid biopsy epigenomic assays to uncover new therapeutic targets and biomarkers. Ultimately, this review aims to support a new era of precision oncology, utilizing insights from epigenomics to improve prostate cancer patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8f247f6de2138b02c15cb85166ee95210b75593" target='_blank'>
              Epigenomics-guided precision oncology: Chromatin variants in prostate tumor evolution.
              </a>
            </td>
          <td>
            K. Furlano, Tina Keshavarzian, Nadine Biernath, Annika Fendler, M. De Santis, Joachim Weischenfeldt, Mathieu Lupien
          </td>
          <td>2025-01-23</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Cancer cells within tumors exhibit a wide range of phenotypic states driven by non-genetic mechanisms in addition to extensively studied genetic alterations. Conversions among cancer cell states can result in intratumoral heterogeneity which contributes to metastasis and development of drug resistance. However, mechanisms underlying the initiation and/or maintenance of such phenotypic plasticity are poorly understood. In particular, the role of intercellular communications in phenotypic plasticity remains elusive. In this study, we employ a multiscale inference-based approach using single-cell RNA sequencing (scRNA-seq) data to explore how intercellular interactions influence phenotypic dynamics of cancer cells, particularly cancers undergoing epithelial-mesenchymal transition. Our inference approach reveals that signaling interactions between cancerous cells in small cell lung cancer (SCLC) result in seemingly contradictory behaviors—reinforcing the cellular phenotypes and maintaining population-level intratumoral heterogeneity. Additionally, we find a recurring signaling pattern across multiple types of cancer in which the mesenchymal-like subtypes utilize signals from other subtypes to reinforce its phenotype, further promoting the intratumoral heterogeneity. We use a mathematical model based on ordinary differential equations to show that inter-subtype communication accelerates the development of heterogeneous tumor populations. Our work highlights the critical role of intercellular signaling in sustaining intratumoral heterogeneity, and our approach of computational analysis of scRNA-seq data can infer inter- and intra-cellular signaling networks in a holistic manner. Significance Single cell-based inference approach reveals a key role of intercellular signaling in maintaining intratumoral heterogeneity. Cell-cell communications stabilize newly acquired cell states and diverse phenotypes of cell populations in multiple cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fa5cf6c7e9bc3e2c2f27d636cb203f597857db1" target='_blank'>
              Intercellular signaling stabilizes single-cell level phenotypic transitions and accelerates the reestablishment of equilibrium of heterogeneous cancer cell populations
              </a>
            </td>
          <td>
            Daniel Lopez, Darren R Tyson, Tian Hong
          </td>
          <td>2025-01-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Intra tumor heterogeneity complicates cancer therapy by providing tumors with the ability to alter their phenotypes and become more therapy resistant. Here, we tested the hypothesis that identifying and modulating expression of key state-specific transcription factors could be used as a strategy for driving cells to a more therapy-sensitive state. Recent single-cell studies have explored the inter and intra tumoral heterogeneity of osteosarcoma and identified gene pathways enriched in specific cell states. For example, metastatic tumors are characterized by an expression of genes in the TNF-α, PI3K, TGFß and mTOR pathways. We identified similar profiles in osteosarcoma patient-derived xenograft-derived cell lines and potential transcription factor drivers of these states. We then used perturb-seq to downregulate expression of key transcription factors and evaluated the effect of these modulations on single cell RNA profiles and drug responses. Knockdown of NFE2L3 or NR0B1 increased the proportion of cells sensitive to targeted therapy. This approach, which could potentially be applied to other cancers, could be used as a strategy to increase the response to targeted therapies by increasing the proportion of cells in a drug-sensitive state. Highlights Distinct transcriptomic states were identified in osteosarcoma cell lines using single-cell RNA sequencing. Lineage tracing identified states with differential sensitivity to therapy. Using perturb-seq, we identified transcription factors that drive cells towards a more sensitive state. The transcription factors NR0B1 and NFE2L3 were identified as targets capable of reprogramming cells to a sensitive state.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6bc16f1a1ca4653fbc0122d06703e74482e021a" target='_blank'>
              Targeting Osteosarcoma heterogeneity to improve therapeutic response
              </a>
            </td>
          <td>
            Brenda Melano, Truc Dinh, Bahar Zirak, Mehran Karimzadeh, Brian J Woo, Benedict Choi, Hani Goodarzi, E. A. Sweet-Cordero, San Francisco
          </td>
          <td>2025-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Copy number variants (CNVs) are an important source of genetic variation underlying rapid adaptation and genome evolution. Whereas point mutation rates vary with genomic location and local DNA features, the role of genome architecture in the formation and evolutionary dynamics of CNVs is poorly understood. Previously, we found the GAP1 gene in Saccharomyces cerevisiae undergoes frequent amplification and selection in glutamine-limitation. The gene is flanked by two long terminal repeats (LTRs) and proximate to an origin of DNA replication (autonomously replicating sequence, ARS), which likely promote rapid GAP1 CNV formation. To test the role of these genomic elements on CNV-mediated adaptive evolution, we evolved engineered strains lacking either the adjacent LTRs, ARS, or all elements in glutamine-limited chemostats. Using a CNV reporter system and neural network simulation-based inference (nnSBI) we quantified the formation rate and fitness effect of CNVs for each strain. Removal of local DNA elements significantly impacts the fitness effect of GAP1 CNVs and the rate of adaptation. In 177 CNV lineages, across all four strains, between 26% and 80% of all GAP1 CNVs are mediated by Origin Dependent Inverted Repeat Amplification (ODIRA) which results from template switching between the leading and lagging strand during DNA synthesis. In the absence of the local ARS, distal ones mediate CNV formation via ODIRA. In the absence of local LTRs, homologous recombination can mediate gene amplification following de novo retrotransposon events. Our study reveals that template switching during DNA replication is a prevalent source of adaptive CNVs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/488ecf1ac91854034846373133d03867e6680f28" target='_blank'>
              Template switching during DNA replication is a prevalent source of adaptive gene amplification
              </a>
            </td>
          <td>
            Julie N. Chuong, Nadav Ben Nun, Ina Suresh, J. Matthews, Titir De, Grace Avecilla, Farah Abdul-Rahman, Nathan Brandt, Yoav Ram, David Gresham
          </td>
          <td>2025-02-03</td>
          <td>eLife</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Cancer is a major cause of death worldwide. Next-generation sequencing (NGS) has dramatically increased the sequencing data output and transformed biomedical investigations. NGS enables the generations of genetic data specific to patients from tumor tissue samples so that targeted therapies can be used. The obtained data further allows the prioritization of effective therapies based on the tumor-specific genotype. Practitioners in the field of clinical genomics can make the best use of testing facilities while lessening the possible off-targets by choosing a priori gene set. Therefore, targeted sequencing has arisen as a more affordable technique for the genomic profiling of tumors. Drug resistance is commonly observed in cancer because of mutations. Thus, precise genetic and molecular profiling of tumors ought to be routinely done prior to the use of targeted therapy or precision cancer therapy. NGS already has the capacity to ameliorate genetic screening in families with previous histories of the high occurrence of various cancer-associated genes, including TP53, APC, BRCA2, and BRCA1. By using NGS system, researchers detected increased variants in cancer cells with greater specificity and sensitivity than conventional diagnostic approaches, which suggest the potential of NGS in diagnosis. The field of precision cancer therapy is continuously growing and because of their specificity at the molecular level has improved the management and treatment of numerous cancers. These therapies are less toxic and more efficient compared to conventional chemotherapies used in cancer treatment. The field of precision cancer therapy is likely to significantly expand as NGS system advances. This review provides extensive information regarding current advances in the NGS field in terms of methods, clinical applications, genomic profiling, and the role of NGS of precision cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e42985c9d2d1f73b46baa3e0ccc12da6c1288b30" target='_blank'>
              Recent advances in the methods and clinical applications of next-generation sequencing in genomic profiling and precision cancer therapy
              </a>
            </td>
          <td>
            A. Alsaiari
          </td>
          <td>2025-01-03</td>
          <td>EXCLI Journal</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d50475b8002866903aae4806efbfc187f7cb121" target='_blank'>
              Canonical and noncanonical NOTCH signaling in the nongenetic resistance of cancer: distinct and concerted control.
              </a>
            </td>
          <td>
            Xianzhe Huang, Wenwei Chen, Yanyan Wang, Dmytro Shytikov, Yanwen Wang, Wangyi Zhu, Ruyi Chen, Yuwei He, Yanjia Yang, Wei Guo
          </td>
          <td>2025-01-02</td>
          <td>Frontiers of medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="KRAS was long deemed undruggable until the discovery of the switch-II pocket facilitated the development of specific KRAS inhibitors. Despite their introduction into clinical practice, resistance mechanisms can limit their effectiveness. Initially, tumors rely on mutant KRAS, but as they progress, they may shift to alternative pathways, resulting in intrinsic resistance. This resistance can stem from mechanisms like epithelial-to-mesenchymal transition (EMT), YAP activation, or KEAP1 mutations. KRAS inhibition often triggers cellular rewiring to counteract therapeutic pressure. For instance, feedback reactivation of signaling pathways such as MAPK, mediated by receptor tyrosine kinases, supports tumor cell survival. Inhibiting KRAS disrupts protein homeostasis, but reactivation of MAPK or AKT can restore it, aiding tumor cell survival. KRAS inhibition also causes metabolic reprogramming and protein re-localization. The re-localization of E-cadherin and Scribble from the membrane to the cytosol causes YAP to translocate to the nucleus, where it drives MRAS transcription, leading to MAPK reactivation. Emerging evidence indicates that changes in cell identity, such as mucinous differentiation, shifts from alveolar type 2 to type 1 cells, or lineage switching from adenocarcinoma to squamous cell carcinoma, also contribute to resistance. In addition to these nongenetic mechanisms, secondary mutations in KRAS or alterations in upstream/downstream signaling proteins can cause acquired resistance. Secondary mutations in the switch-II pocket disrupt drug binding, and known oncogenic mutations affect drug efficacy. Overcoming these resistance mechanisms involves enhancing the efficacy of drugs targeting mutant KRAS, developing broad-spectrum inhibitors, combining therapies targeting multiple pathways, and integrating immune checkpoint inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83c1ae3fc98dfd6ef9151275fdf77ddcf612521a" target='_blank'>
              Mechanisms of Resistance to KRAS Inhibitors: Cancer Cells' Strategic Use of Normal Cellular Mechanisms to Adapt.
              </a>
            </td>
          <td>
            Noritaka Tanaka, Hiromichi Ebi
          </td>
          <td>2024-12-27</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Breast cancer is the most common malignancy affecting women, manifesting as a heterogeneous disease with diverse molecular characteristics and clinical presentations. Recent studies have elucidated the role of epigenetic modifications in the pathogenesis of breast cancer, including drug resistance and efflux characteristics, offering potential new diagnostic and prognostic markers, treatment efficacy predictors, and therapeutic agents. Key modifications include DNA cytosine methylation and the covalent modification of histone proteins. Unlike genetic mutations, reprogramming the epigenetic landscape of the cancer epigenome is a promising targeted therapy for the treatment and reversal of drug resistance. Epidrugs, which target DNA methylation and histone modifications, can provide novel options for the treatment of breast cancer by reversing the acquired resistance to treatment. Currently, the most promising approach involves combination therapies consisting of epidrugs with immune checkpoint inhibitors. This review examines the aberrant epigenetic regulation of breast cancer initiation and progression, focusing on modifications related to estrogen signaling, drug resistance, cancer progression, and the epithelial–mesenchymal transition (EMT). It examines existing epigenetic drugs for treating breast cancer, including agents that modify DNA, inhibitors of histone acetyltransferases, histone deacetylases, histone methyltransferases, and histone demethyltransferases. It also delves into ongoing studies on combining epidrugs with other therapies and addresses the upcoming obstacles in this field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b1d287b99832f39586fe0fd89b9b4c10e300e3c" target='_blank'>
              Cellular Epigenetic Targets and Epidrugs in Breast Cancer Therapy: Mechanisms, Challenges, and Future Perspectives
              </a>
            </td>
          <td>
            Ibrahim S. Alalhareth, Saleh M. Alyami, Ali H. Alshareef, Ahmed O. Ajeibi, M. A. Al Munjem, Ahmad A. Elfifi, Meshal M. Alsharif, S. A. Alzahrani, Mohammed A. Alqaad, Marwa B. Bakir, Basel A. Abdel-Wahab
          </td>
          <td>2025-02-03</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40aee1d94d2e0752a7ed0c107794f2527da8d5a1" target='_blank'>
              Metabolic reprogramming induced by PSMA4 overexpression facilitates bortezomib resistance in multiple myeloma.
              </a>
            </td>
          <td>
            Han Yu, Chengli Wu, Jie He, Yajun Zhang, Qiqi Cao, Hongyan Lan, Hongshan Li, Chengyang Xu, Chen Chen, Rong Li, Bo Zheng
          </td>
          <td>2025-01-04</td>
          <td>Annals of hematology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Despite efficacy of FLT3 and BCL2 inhibition in acute myeloid leukemia (AML), relapse limits survival. Mutation status and AML monocytic differentiation are implicated in resistance. On-treatment tumor evolution may select for genetically distinct clones or shifts in differentiation not resolvable by bulk sequencing. We performed multiomic single cell (SC) DNA/protein and RNA/protein profiling of patients treated on a clinical trial of the BCL2 inhibitor venetoclax and the FLT3 inhibitor gilteritinib (Ven/Git) to characterize immunophenotypic, transcriptional, and genetic clonal evolution on therapy. We found that while Ven/Gilt effectively eliminated FLT3 mutant clones, it selected for RAS mutations, RAS pathway activation and RAS-associated monocytic differentiation. In an in vitro model of monocytic differentiation associated with heightened RAS pathway activation, we demonstrated that MEK inhibition re-sensitized to Ven/Gilt. These data indicate RAS signaling is central to FLT3 and BCL2 inhibitor resistance, is tightly coupled to monocytic differentiation and can be overcome by RAS pathway inhibition. COI C.C.S. has provided educational talks for Astellas Pharma, served on advisory boards for Genentech/Abbvie and received research funding from Abbvie. B.C., Y.S. and J.H. are employees of Abbvie. M.S. and H.H. are or were previously employees of Genentech. Funding This work was supported in part by Abbvie. C.C.S. is a Leukemia & Lymphoma Society Scholar in Clinical Research and a Damon Runyon-Richard Lumsden Foundation Clinical Investigator supported (in part) by the Damon Runyon Cancer Research Foundation (CI-99–18). Statement of Significance: Mutational and non-mutational RAS signaling activation drives clonal selection, monocytic differentiation and treatment resistance to FLT3 and BCL2 inhibition in AML. MEK inhibition can resensitize resistant AML cells, suggesting therapeutic potential for combined FLT3, BCL2 and RAS pathway inhibition in AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/044e6c5d837777d0153b7a0fb95ff71f6ce12ad5" target='_blank'>
              RAS pathway activation drives clonal selection and monocytic differentiation in FLT3 and BCL2 inhibitor resistance
              </a>
            </td>
          <td>
            Vanessa E. Kennedy, C. Peretz, Anushka Walia, Brenda Chyla, Yan Sun, Jason Hill, Elaine Tran, Andrew Koh, Timothy T. Ferng, Samantha M. Pintar, Matthew F. Jones, Bogdan Popescu, N. Murad, Ritu Roy, Adam Olshen, Sunil Joshi, E. Traer, Monique Dail, Habib Hamidi, Jessica Altman, Naval G Daver, M. Levis, James K. McCloskey, Alexander Perl, Catherine C Smith
          </td>
          <td>2025-02-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="The interplay between genetics and cancer has been a focal point of research for decades, leading to profound understandings into the molecular mechanisms driving tumorigenesis. In this comprehensive review article, we explore the genetic basis of cancer, encompassing the diverse array of alterations that underline oncogenic transformation. From oncogenes to tumor suppressor genes, and from point mutations to chromosomal rearrangements, we delve into the molecular hallmarks of cancer and their implications for diagnosis, treatment, and prevention. Drawing on recent advancements in genomic technologies, we discuss the role of next-generation sequencing, single-cell sequencing, and computational modeling in unraveling the complexity of cancer genetics. Furthermore, we examine the clinical implications of genetic predisposition to cancer, highlighting the importance of genetic testing and counselling in cancer risk assessment and management. Through an exploration of tumor heterogeneity, clonal evolution, and therapeutic resistance, we underscore the challenges and opportunities in precision oncology. Finally, we discuss future directions in cancer genetics research, including precision prevention strategies and ethical considerations.Keywords: Cancer Genetics; Oncogenes; Tumor Suppressor Genes; Genetic Alterations; Precision Oncology; Clonal Evolution   ">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b6286c0dc712fbb40ada9cedccccb4c8abdd42a" target='_blank'>
              Understanding the Role of Genetics in Tumour and Cancer Biology
              </a>
            </td>
          <td>
            A. Asiri, Amer Al Ali, Mohammed H. Abu-Alghayth
          </td>
          <td>2025-01-29</td>
          <td>Advancements in Life Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/241144354fceff41346aae0d566101a01ccaa823" target='_blank'>
              Advances in cancer genomics and precision oncology.
              </a>
            </td>
          <td>
            Yonjong Heo, Woo-Jin Kim, Yong-Joon Cho, Jae-Won Jung, Nam-Soo Kim, Ik-Young Choi
          </td>
          <td>2025-01-23</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Mismatch repair (MMR) is a crucial DNA repair pathway that maintains genomic integrity by correcting replication errors and various forms of DNA damage. MMR deficiency (MMRd) leads to increased mutation rates, microsatellite instability, and contributes to tumorigenesis in multiple cancer types. Using a CRISPR-Cas9-mediated knockout assay in human isogenic cell lines, we characterised mutational profiles in MMR-deficient cells. Our findings revealed expected increases in mutation burden and the emergence of known MMR-associated mutational signatures. Notably, we identified a previously unconnected process, SBS57, and linked it to germline single-nucleotide polymorphisms and MMR-driven indels in MMRd cells, establishing its association with tensor signature TS27. Comparative analyses of in vitro MMRd profiles and in vivo tumour data uncovered key differences in mutational signatures, highlighting the biological context dependence of MMR-associated mutations. Furthermore, we provide direct experimental evidence that MMR plays a role in repairing 5-methylcytosine deamination, a repair process previously inferred from tumour sequencing data. These findings offer novel insights into MMR deficiency, shedding light on previously uncharacterised mutational mechanisms and their implications in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85a293399722ef8f7c62c0c36dd67ff9d94d28e7" target='_blank'>
              Dissecting DNA-mismatch-repair-driven mutational processes in human cells
              </a>
            </td>
          <td>
            Michel Owusu, Jörg Menche, Joanna Loizou, Donate Weghorn
          </td>
          <td>2025-02-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df4d7f1f1862a0b62b5b5df29d2416fddab0290c" target='_blank'>
              lncRNAs: the unexpected link between protein synthesis and cancer adaptation
              </a>
            </td>
          <td>
            Mila Gugnoni, Manoj Kumar Kashyap, Kishore K. Wary, A. Ciarrocchi
          </td>
          <td>2025-01-31</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="New therapeutic approaches that antagonize tumour-promoting phenotypes in lung cancer are needed to improve patient outcomes. Chromosomal instability (CIN) is a hallmark of lung cancer characterized by the ongoing acquisition of genetic alterations that include the gain and loss of whole chromosomes or segments of chromosomes as well as chromosomal rearrangements during cell division. Although it provides genetic diversity that fuels tumour evolution and enables the acquisition of aggressive phenotypes like immune evasion, metastasis, and drug resistance, too much CIN can be lethal because it creates genetic imbalances that disrupt essential genes and induce severe proteotoxic and metabolic stress. As such, sustaining advantageous levels of CIN that are compatible with survival is a fine balance in cancer cells, and potentiating CIN to levels that exceed a tolerable threshold is a promising treatment strategy for inherently unstable tumours like lung cancer. Kinesins are a superfamily of motor proteins with many members having functions in mitosis that are critical for the correct segregation of chromosomes and, consequently, maintaining genomic integrity. Accordingly, inhibition of such kinesins has been shown to exacerbate CIN. Therefore, inhibiting mitotic kinesins represents a promising strategy for amplifying CIN to lethal levels in vulnerable cancer cells. In this review, we describe the concept of CIN as a therapeutic vulnerability and comprehensively summarize studies reporting the clinical and functional relevance of kinesins in lung cancer, with the goal of outlining how kinesin inhibition, or “targeting kinesins”, holds great potential as an effective strategy for treating lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/994a7c70e4769153ecb42042c5f04caa4a869a11" target='_blank'>
              Targeting Kinesins for Therapeutic Exploitation of Chromosomal Instability in Lung Cancer
              </a>
            </td>
          <td>
            Christopher Zhang, Benson Z. Wu, Kelsie L. Thu
          </td>
          <td>2025-02-18</td>
          <td>Cancers</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background/Aim Bile tract cancer (BTC) is a malignant tumor with a poor prognosis. Recent studies have reported the heterogeneity of the genomic background and gene alterations in BTC, but its genetic heterogeneity and molecular profiles remain poorly understood. Whole-genome sequencing may enable the identification of novel actionable gene mutations involved in BTC carcinogenesis, malignant progression, and treatment resistance. Patients and Methods We performed whole-genome sequencing of six BTC samples to elucidate its genetic heterogeneity and identify novel actionable gene mutations. Somatic mutations, structural variations, copy number alterations, and their associations with clinical factors were analyzed. Results The average number of somatic mutations detected in each case was 53,705, with SNVs accounting for most of these mutations (85.02%). None of the 331 mutations related to BTC in The Cancer Genome Atlas (TCGA) database were found in the mutations identified in our study. A higher prevalence of gene mutations was observed in samples without vascular invasion than in those with vascular invasion. Several genes with differences in mutation accumulation between groups were identified, including ADAMTS7, AHNAK2, and CAPN10. Conclusion Our study provides novel insights into the genomic landscape of BTC and highlights the potential of whole-genome sequencing analysis to identify actionable gene mutations and understand the molecular mechanisms underlying this malignancy. The high mutational burden, structural variations, and copy number alterations observed in BTC samples in this study underscore the genetic complexity and heterogeneity of this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34204b05b4cfe3c5397ce50291dbbf864d51918d" target='_blank'>
              Whole-genome Sequencing Analysis of Bile Tract Cancer Reveals Mutation Characteristics and Potential Biomarkers
              </a>
            </td>
          <td>
            T. Kokuryo, M. Sunagawa, J. Yamaguchi, Taisuke Baba, S. Kawakatsu, N. Watanabe, S. Onoe, T. Mizuno, T. Ebata
          </td>
          <td>2024-12-27</td>
          <td>Cancer Genomics & Proteomics</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="PAX3-FOXO1, an oncogenic transcription factor, drives a particularly aggressive subtype of rhabdomyosarcoma (RMS) by enforcing gene expression programs that support malignant cell states. Here we show that PAX3-FOXO1+ RMS cells exhibit altered pyrimidine metabolism and increased dependence on enzymes involved in de novo pyrimidine synthesis, including dihydrofolate reductase (DHFR). Consequently, PAX3-FOXO1+ cells display increased sensitivity to inhibition of DHFR by the chemotherapeutic drug methotrexate, and this dependence is rescued by provision of pyrimidine nucleotides. Methotrexate treatment mimics the metabolic and transcriptional impact of PAX3-FOXO1 silencing, reducing expression of genes related to PAX3-FOXO1-driven malignant cell states. Accordingly, methotrexate treatment slows growth of multiple PAX3-FOXO1+ tumor xenograft models, but not fusion-negative counterparts. Taken together, these data demonstrate that PAX3-FOXO1 induces cell states characterized by altered pyrimidine dependence and nominate methotrexate as an addition to the current therapeutic arsenal for treatment of these malignant pediatric tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d50231dc8772bfbbe71ed944e599efde5187dc0b" target='_blank'>
              PAX3-FOXO1 drives targetable cell state-dependent metabolic vulnerabilities in rhabdomyosarcoma
              </a>
            </td>
          <td>
            Katrina I. Paras, Julia S. Brunner, Jacob A. Boyer, Angela M. Montero, Benjamin T. Jackson, Sangita Chakraborty, Abigail Xie, Kristina Guillan, Armaan Siddiquee, Lourdes Pajuelo Torres, Joshua D. Rabinowitz, Andrew L Kung, Daoqi You, Filemon Dela Cruz, Lydia W. S. Finley
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Colorectal cancer (CRC) represents the third-leading cause of cancer-related deaths. Knowledge covering diverse cellular and molecular data from individual patients has become valuable for diagnosis, prognosis, and treatment selection. Here, we present an in-depth comparative mRNA-seq and microRNA-seq analysis of tissue samples from 32 CRC, pairing tumors with adjacent healthy tissues. The differential expression gene (DEG) analysis revealed an interconnection between nutrients, metabolic programs, and cell cycle pathways. We focused on the impact of overexpressed SLC7A11 (xCT) and SLC3A2 genes which compose the cystine/glutamate transporter (Xc-) system. To assess the oncogenic potency of the Xc-system in a cellular setting, we applied a knowledge-based approach for analyzing gene perturbations from CRISPR screens across various cell types as well as using a variety of functional assays in five primary patient-derived organoid cell models to functionally verify our hypothesis. We identified a previously undescribed cell surface protein signature predicting chemotherapy resistance and further highlighted the causality and potential of pharmacological blockage of ferroptosis as promising avenue for cancer therapy. Biological processes such as redox homeostasis, ion/amino acid transporters and de novo nucleotide synthesis were associated with these co-dependent genes in patient specimens. This study highlighted a number of overlooked genes as potential clinical targets for CRC and promotes stem cell-based, patient-individual in vitro model systems as a versatile partner platform to functionally validate in silico predictions, with focus on SLC7A11 and its associated genes in tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06923ea47f54b55109b8b50bd2b097751afa1aed" target='_blank'>
              Patient-Specific Pharmacogenomics unveils xCT as a Key Regulator for Druggable Metabolic and Proliferation Pathways in Colon Cancer
              </a>
            </td>
          <td>
            Marco Strecker, Keren Zohar, Martin Böttcher, Thomas Wartmann, Henry Freudenstein, Maximilian Doelling, Mihailo Andric, W. Shi, O. Kakhlon, Katrin Hippe, Beatrix Jahnke, Dimitrios Mougiakakos, Franziska Baenke, Daniel Stange, R. Croner, M. Linial, U. D. Kahlert
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13edecae9b7bef7cd629efeb486e0288bd13530b" target='_blank'>
              Tumour hypoxia in driving genomic instability and tumour evolution.
              </a>
            </td>
          <td>
            Alexandru Suvac, Jack Ashton, Robert G Bristow
          </td>
          <td>2025-01-28</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Chromosomal instability in gastric cancer cells is associated with the amplification of oncogenes that encode receptor tyrosine kinases (RTKs), such as HER2 and FGFR2; such gene amplification varies from cell to cell and manifests as genetic heterogeneity within tumours. The intratumoural genetic heterogeneity of RTK gene amplification causes heterogeneity in RTK protein expression, which has been suggested to be associated with therapeutic resistance to RTK inhibitors; however, the underlying mechanism is not fully understood. Here, we show that extrachromosomal DNA (ecDNA) causes intratumoural genetic heterogeneity in RTKs and drug resistance due to diverse dynamic changes. We analysed the dynamics of FGFR2 and MYC ecDNA in a gastric cancer cell line after single-cell cloning. Similar to those in parental cells, the copy numbers of FGFR2 and MYC in subclones differed significantly between cells, indicating intraclonal genetic heterogeneity. Furthermore, the ecDNA composition differed between subclones, which affected FGFR2 protein expression and drug sensitivity. Interestingly, clone cells that were resistant to the FGFR2 inhibitor AZD4547 presented diverse changes in ecDNA, including chimeric ecDNA, large ecDNA and increased ecDNA numbers; these changes were associated with high expression and rephosphorylation of FGFR2. Conversely, when resistant clone cells were cultured under conditions that excluded AZD4547, the ecDNA status became similar to that of the original clone cells, and the inhibitory effect on cell growth was restored. Implications: Our results show that dynamic quantitative and qualitative changes in ecDNA can drive the intratumoural genetic heterogeneity of RTKs and resistance to RTK inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/895b7aa47450acfbdeb8f8f1748f0e8b623eb4a8" target='_blank'>
              Extrachromosomal DNA dynamics contribute to intratumoural receptor tyrosine kinase genetic heterogeneity and drug resistance in gastric cancer.
              </a>
            </td>
          <td>
            Kazuki Kanayama, Hiroshi Imai, Ryotaro Hashizume, Chise Matsuda, Eri Usugi, Y. Hirokawa, Masatoshi Watanabe
          </td>
          <td>2025-02-19</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Breast cancer is the most prevalent cancer and the leading cause of cancer-related mortality among women. Recent breakthroughs in breast cancer therapeutics have significantly enhanced outcomes for hormone receptor-positive and HER2-negative subtypes. However, the emergence of drug resistance, particularly in triple-negative breast cancer, presents a formidable challenge. The intricate interplay of genetic and metabolic diversity within breast cancer cells is pivotal to its development. By reprogramming metabolic pathways, cancer cells can adapt and thrive, meeting the demands of survival, growth, and invasion. These metabolic shifts also play a key role in the development of resistance to conventional therapies. This review explores the genetic and metabolic complexities of breast cancer, emphasizing the diverse subtypes and their unique profiles. We examine how genetic variations and metabolic alterations contribute to breast cancer development and progression, influencing both treatment efficacy and resistance. By integrating insights into the genetic background and metabolic reprogramming of breast cancer subtypes, this review aims to highlight the genetic variations and metabolic alterations that contribute to the pathogenesis of breast cancer, with a vision of advancing more precise and effective targeted therapies as well as discovering of novel diagnostic and prognostic markers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd8d2c71ef9a50520221d0171aab11b20229faa4" target='_blank'>
              Genetic heterogeneity and metabolic reprogramming in breast cancer
              </a>
            </td>
          <td>
            Yarkın Dolaş, Ayşe Buruş, B. Çeltikçi
          </td>
          <td>2024-12-29</td>
          <td>Acta Medica</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Multiple myeloma (MM) is a complex and heterogeneous hematologic malignancy characterized by clonal evolution, genetic instability, and interactions with a supportive tumor microenvironment. These factors contribute to treatment resistance, disease progression, and significant variability in clinical outcomes among patients. This review explores the mechanisms underlying MM progression, including the genetic and epigenetic changes that drive clonal evolution, the role of the bone marrow microenvironment in supporting tumor growth and immune evasion, and the impact of genomic instability. We highlight the critical insights gained from single-cell technologies, such as single-cell transcriptomics, genomics, and multiomics, which have enabled a detailed understanding of MM heterogeneity at the cellular level, facilitating the identification of rare cell populations and mechanisms of drug resistance. Despite the promise of advanced technologies, MM remains an incurable disease and challenges remain in their clinical application, including high costs, data complexity, and the need for standardized bioinformatics and ethical considerations. This review emphasizes the importance of continued research and collaboration to address these challenges, ultimately aiming to enhance personalized treatment strategies and improve patient outcomes in MM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b4ede18b0be075e37d640173950681509758ae8" target='_blank'>
              Multiple Myeloma Insights from Single-Cell Analysis: Clonal Evolution, the Microenvironment, Therapy Evasion, and Clinical Implications
              </a>
            </td>
          <td>
            Sihong Li, Jiahui Liu, Madeline Peyton, Olivia Lazaro, Sean D. McCabe, Xiaoqing Huang, Yunlong Liu, Zanyu Shi, Zhiqi Zhang, Brian A. Walker, Travis S. Johnson
          </td>
          <td>2025-02-14</td>
          <td>Cancers</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Developing resistance to androgen receptor (AR) signaling inhibitors is a significant challenge in the treatment of castration-resistant prostate cancer. Prolonged use of inhibitors like enzalutamide can cause prostate cancer cells to undergo lineage reprogramming, transitioning to neuroendocrine subtypes that no longer rely on AR signaling. These neuroendocrine subtypes are among the most aggressive forms of prostate cancer. During this process of lineage plasticity, cancer cells experience extensive transcriptional rewiring and acquire stem-like properties characterized by increased stemness. Research has shown that prostate cancer cells gain these stem-like traits through the expression of stem cell-associated proteins such as NANOG, particularly under stable and accumulating conditions. The post-translational modification of NANOG at specific sites is critical for maintaining its stability, which in turn enhances the tumorigenic potential of the cells. This review discusses the mechanisms by which NANOG phosphorylation promotes stemness and lineage plasticity in prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d23842d78f2e67e6ef1863e8dd082e1ac56f105a" target='_blank'>
              Molecular mechanisms driving lineage plasticity in prostate cancer: NANOG and beyond
              </a>
            </td>
          <td>
            Hamed Maasoumyhaghighi, Mansoureh Nouri, Jinghui Liu, Xiaoqi Liu
          </td>
          <td>2024-12-27</td>
          <td>Cancer Heterogeneity and Plasticity</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The discovery of noncoding RNAs (ncRNAs) revolutionised our understanding of gene regulation, marking a paradigm shift in cancer biology. Initially considered transcriptional noise, ncRNAs, including microRNAs (miRNAs) and long noncoding RNAs (lncRNAs), have emerged as critical regulators of gene expression, tumour progression, and therapy resistance. Their role in modulating invasion, metastasis, and tumour heterogeneity highlights their significance in cancer pathophysiology. Advances in transcriptomics have unveiled the potential of ncRNAs as biomarkers and therapeutic targets, exemplified by miRNA-based therapies and lncRNA-targeted interventions. However, challenges remain, particularly in clinical translation, where delivery mechanisms and off-target effects must be addressed. Emerging technologies such as single-cell RNA sequencing and bioinformatics-driven analyses promise to describe ncRNA functions further, improving the precision oncology framework. This editorial highlights the journey of ncRNAs from discovery to their integration into cancer research, emphasising their transformative potential in diagnostics and therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b69c4670efb4dc2591bcf884f4e7478cd4e2c883" target='_blank'>
              Few Thoughts about Noncoding RNAs in Cancer
              </a>
            </td>
          <td>
            I. Berindan-Neagoe
          </td>
          <td>2024-12-31</td>
          <td>Journal of Medical and Radiation Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8a65325c104ad2755155bebd8b8e82939a9a6fc" target='_blank'>
              Transposon mediated functional genomic screening for BRAF inhibitor resistance reveals convergent Hippo and MAPK pathway activation events
              </a>
            </td>
          <td>
            Li Chen, Iulian Pruteanu-Malinici, A. Dastur, Xunqin Yin, Dennie Frederick, R. I. Sadreyev, Cyril H. Benes
          </td>
          <td>2025-01-24</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="INTRODUCTION
Globally, prostate cancer (CaP) is a leading cause of death and disability among men and a substantial public health burden. Despite advancements in cancer treatment, chemoresistance remains a significant issue in cancer therapy, accounting for the majority of patient relapses and poor survival. Cancer stem cells (CSCs) are considered the main cause of cancer recurrence, chemoresistance, and poor survival of patients. These CSCs acquire stemness and chemoresistance by certain mechanisms such as enhanced DNA repair processes, increased expression of drug efflux pumps, resistance to apoptosis, and altered cell cycle and tumor microenvironment (TME).


AREA COVERED
We cover the latest developments in this field and give an overview of future research directions.


EXPERT OPINION
CSCs show dysregulation of several signaling pathways, mostly related to conferring chemoresistance phenotype, such as high drug efflux, apoptotic resistance, quiescent cell cycle, tumor microenvironment, and DNA repair. There are several research articles published on this topic. However, still, this field warrants further investigations to identify the therapeutic molecule that can either chemosensitize CSCs or kill them effectively. This can only be possible when we know the complete mechanisms to comprehend the fundamental causes of cancer stemness and therapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/756692242b3daaa338fbef439d9939f7beaddfd4" target='_blank'>
              An update understanding of stemness and chemoresistance of prostate cancer.
              </a>
            </td>
          <td>
            Afiya Wahab, Hifzur R Siddique
          </td>
          <td>2025-02-11</td>
          <td>Expert review of anticancer therapy</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Immune escape is a critical hallmark of cancer progression and underlies resistance to multiple immunotherapies. However, it remains unclear when the genetic events associated with immune escape occur during cancer development. Here, we integrate functional genomics studies of immunomodulatory genes with a tumor evolution reconstruction approach to infer the evolution of immune escape across 38 cancer types from the Pan-Cancer Analysis of Whole Genomes dataset. Different cancers favor mutations in different immunomodulatory pathways. For example, the antigen presentation machinery is highly mutated in colorectal adenocarcinoma, lung squamous cell carcinoma, and chromophobe renal cell carcinoma, and the protein methylation pathway is highly mutated in bladder transitional cell carcinoma and lung adenocarcinoma. We also observe different timing patterns in multiple immunomodulatory pathways. For instance, mutations impacting genes involved in cellular amino acid metabolism were more likely to happen late in pancreatic adenocarcinoma. Mutations in the glucocorticoid receptor regulatory network pathway tended to occur early, while mutations in the TNF pathways were more likely to occur late in B-cell non-Hodgkin lymphoma. Mutations in the NOD1/2 signaling pathway and DNA binding transcription factor activity tended to happen late in breast adenocarcinoma and ovarian adenocarcinoma. Together, these results delineate the evolutionary trajectories of immune escape in different cancer types and highlight opportunities for improved immunotherapy of cancer. Significance Despite its critical role in cancer progression, the evolution of immune escape is poorly understood. We integrate functional genomics and tumor evolution reconstruction and infer immune escape trajectories across cancer types. Our results have important implications for developing biomarkers for immunoprevention and treatment strategies for immune escape of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a491b543872787c05b0f657359563c86a9093f6" target='_blank'>
              Evolutionary trajectories of immune escape across cancers
              </a>
            </td>
          <td>
            Wenjie Chen, Toby M. Baker, Zhihui Zhang, Huw A. Ogilvie, P. Van Loo, S. Gu
          </td>
          <td>2025-01-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="The link between the “stress phenotype”-a well-established hallmark of cancer-and its role in tumor progression and intratumor heterogeneity remains poorly defined. The integrated stress response (ISR) is a key adaptive pathway that enables tumor survival under oncogenic stress. While ISR has been implicated in promoting tumor growth, its precise role in driving tumor evolution and heterogeneity has not been elucidated. In this study, using a genetically engineered mouse models, we demonstrate that ISR activation—indicated by elevated levels of phosphorylated eIF2 (p-eIF2) and ATF4—is essential for the emergence of dedifferentiated, therapy-resistant cell states. ISR, through the coordinated actions of ATF4 and MYC, facilitates the development of tumor cell populations characterized by high plasticity, stemness, and an epithelial-mesenchymal transition (EMT)-prone phenotype. This process is driven by ISR-mediated expression of genes that maintain mitochondrial integrity and function, critical for sustaining tumor progression. Importantly, genetic, or pharmacological inhibition of the p-eIF2–ATF4 signaling axis leads to mitochondrial dysfunction and significantly impairs tumor growth in mouse models of lung adenocarcinoma (LUAD). Moreover, ISR-driven dedifferentiation is associated with poor prognosis and therapy resistance in advanced human LUAD, underscoring ISR inhibition as a promising therapeutic strategy to disrupt tumor evolution and counteract disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/984621d182e32177c7e7be128d21ab0e1df4a135" target='_blank'>
              Lineage plasticity of the integrated stress response is a hallmark of cancer evolution
              </a>
            </td>
          <td>
            Shiqi Diao, Jia Yi Zou, Shuo Wang, N. Ghaddar, Jason E. Chan, Hyungdong Kim, Nicolas Poulain, C. Koumenis, M. Hatzoglou, Peter Walter, Nahum Sonenberg, John Le Quesne, Tuomas Tammela, A. Koromilas
          </td>
          <td>2025-02-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="
 The molecular mechanisms underlying estrogen receptor (ER)-positive breast carcinogenesis and drug resistance remain incompletely understood. Elevated expression of CCND1 is linked to enhanced invasiveness, poorer prognosis, and resistance to drug therapies in ER-positive breast cancer. In this study, we identify a highly expressed circular RNA (circRNA) derived from
 FOXK2
 , called circFOXK2, which plays a key role in stabilizing
 CCND1
 mRNA, thereby promoting cell cycle progression, cell growth, and endocrine therapy resistance in ER-positive breast cancer cells. Mechanistically, circFOXK2 binds directly to
 CCND1
 mRNA via RNA–RNA pairing and recruits the RNA-binding protein ELAVL1/HuR, stabilizing the
 CCND1
 mRNA and enhancing CCND1 protein levels. This results in activation of the CCND1–CDK4/6–p-RB–E2F signaling axis, driving the transcription of downstream E2F target genes and facilitating the G1/S transition during cell cycle progression. Notably, targeting circFOXK2 with antisense oligonucleotide (ASO-circFOXK2) suppresses ER-positive breast cancer cell growth both in vitro and in vivo. Moreover, combination therapy with ASO-circFOXK2 and tamoxifen exhibits synergistic effects and restores tamoxifen sensitivity in tamoxifen-resistant cells. Clinically, high circFOXK2 expression is positively correlated with
 CCND1
 levels in both ER-positive breast cancer cell lines and patient tumor tissues. Overall, our findings reveal the critical role of circFOXK2 in stabilizing the oncogene
 CCND1
 and promoting cancer progression, positioning circFOXK2 as a potential therapeutic target for ER-positive breast cancer in clinical settings.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a572f11006892aa8b4e142fead4939f8a023665b" target='_blank'>
              A circRNA–mRNA pairing mechanism regulates tumor growth and endocrine therapy resistance in ER-positive breast cancer
              </a>
            </td>
          <td>
            Jia Yi, Jiao Du, Xue Chen, Rui-chao Nie, Guo-Sheng Hu, Lei Wang, Yue-ying Zhang, Shang Chen, , , Hua He, Wen Liu
          </td>
          <td>2025-02-18</td>
          <td>Proceedings of the National Academy of Sciences</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Cancer evolves continuously over time through a complex interplay of genetic, epigenetic, microenvironmental, and phenotypic changes. This dynamic behavior drives uncontrolled cell growth, metastasis, immune evasion, and therapy resistance, posing challenges for effective monitoring and treatment. However, today's data-driven research in oncology has primarily focused on cross-sectional analysis using data from a single modality, limiting the ability to fully characterize and interpret the disease's dynamic heterogeneity. Advances in multiscale data collection and computational methods now enable the discovery of longitudinal multimodal biomarkers for precision oncology. Longitudinal data reveal patterns of disease progression and treatment response that are not evident from single-timepoint data, enabling timely abnormality detection and dynamic treatment adaptation. Multimodal data integration offers complementary information from diverse sources for more precise risk assessment and targeting of cancer therapy. In this review, we survey methods of longitudinal and multimodal modeling, highlighting their synergy in providing multifaceted insights for personalized care tailored to the unique characteristics of a patient's cancer. We summarize the current challenges and future directions of longitudinal multimodal analysis in advancing precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a35a93de0536a7f3b11801216a34e22a7ecf3075" target='_blank'>
              Advancing Precision Oncology Through Modeling of Longitudinal and Multimodal Data
              </a>
            </td>
          <td>
            Luoting Zhuang, Stephen H. Park, Steven J. Skates, A.E. Prosper, Denise R. Aberle, William Hsu
          </td>
          <td>2025-02-11</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67f873b54c9840f07bddb284c4e229ec7800362c" target='_blank'>
              Genetic advancements in breast cancer treatment: a review
              </a>
            </td>
          <td>
            Marzieh Shokoohi, Sadaf Sedaghatshoar, Homaira Arian, Milad Mokarami, Fatemeh Habibi, Fatemeh Bamarinejad
          </td>
          <td>2025-02-07</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Synthetic lethality is defined as a type of genetic interaction where the combination of two genetic events results in cell death, whereas each of them separately does not. Synthetic lethality can be a useful tool in personalised oncology. MLH1 is a cancer-related gene that has a central role in DNA mismatch-repair and TP53 is the most frequently mutated gene in cancer. To identify genetic events that can lead to tumour death once either MLH1 or TP53 is mutated, a genome-wide genetic screening was performed. Thus, mutations in all protein-coding genes were introduced into haploid human embryonic stem cells (hESCs) with and without loss-of-function mutations in the MLH1 or TP53 genes. These experiments uncovered a list of putative hits with EXO1, NR5A2, and PLK2 genes for MLH1, and MYH10 gene for TP53 emerging as the most promising candidates. Synthetic lethal interactions of these genes were validated genetically or chemically using small molecules that inhibit these genes. The specific effects of SR1848, which inhibits NR5A2, ON1231320 or BI2536, which inhibits PLK2, and blebbistatin, which inhibits MYH10, were further validated in cancer cell lines. Finally, animal studies with CCL xenografts showed the selective effect of the small molecule BI2536 on MLH1-null tumours and of blebbistatin on TP53-mutated tumours. Thus, demonstrating their potential for personalised medicine, and the robustness of genetic screening in haploid hESCs in the context of cancer therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf8258266a9c323d57d9c42f16e5c72059537055" target='_blank'>
              Genome-Wide Screening in Haploid Stem Cells Reveals Synthetic Lethality Targeting MLH1 and TP53 Deficient Tumours.
              </a>
            </td>
          <td>
            R. Cashman, Guy Haim-Abadi, Elyad Lezmi, Hagit Philip, J. Nissenbaum, Ruth Viner-Breuer, Chen Kozulin, T. Golan‐lev, Aseel Gadban, Shiri Spinner-Potesky, O. Yanuka, O. Kopper, N. Benvenisty
          </td>
          <td>2025-01-15</td>
          <td>Cell proliferation</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Autophagy is a critical cellular process that maintains homeostasis by recycling damaged or aberrant components. This process is orchestrated by a network of proteins that form autophagosomes, which engulf and degrade intracellular material. In cancer, autophagy plays a dual role: it suppresses tumor initiation in the early stages but supports tumor growth and survival in advanced stages. Chronic myeloid leukemia (CML), a hematological malignancy, is characterized by the Philadelphia chromosome, a chromosomal abnormality resulting from a translocation between chromosomes 9 and 22. Autophagy has emerged as a key factor in CML pathogenesis, promoting cancer cell survival and contributing to resistance against tyrosine kinase inhibitors (TKIs), the primary treatment for CML. Targeting autophagic pathways is being actively explored as a therapeutic approach to overcome drug resistance and enhance cancer cell death. Recent research highlights the intricate interplay between autophagy and CML progression, underscoring its role in disease biology and treatment outcomes. This review aims to provide a comprehensive analysis of the molecular and cellular mechanisms underlying CML, with a focus on the therapeutic potential of targeting autophagy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca600b2dcbd0c4958d816d4cfd021e7ec09edd48" target='_blank'>
              Autophagy and Cancer: Insights into Molecular Mechanisms and Therapeutic Approaches for Chronic Myeloid Leukemia
              </a>
            </td>
          <td>
            M. A. Kausar, Sadaf Anwar, Yusuf Saleem Khan, Ayman A. Saleh, Mai Ali Abdelfattah Ahmed, Simran Kaur, Naveed Iqbal, W. Siddiqui, M. Najm
          </td>
          <td>2025-02-02</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Breast cancer (BC) is considered as the most frequent cancer among women and a significant contributor to mortality. The CRISPR/Cas9 gene-editing tool has promising applications for BC drug resistance. It is a unique and creative approach that has lately attracted attention and can be used to fix gene alterations related to multidrug resistance. Recent research has effectively investigated and targeted particular genes linked to BC treatment resistance using CRISPR/Cas9 gene editing, including those linked to hormone receptor signaling, drug efflux transporters, and DNA repair pathways. The CRISPR/Cas9 technology's selective disruption or mutation of these genes provides valuable information about their role in resistance and paves the path for cutting-edge treatment options. CRISPR/Cas9 gene editing can overcome BC treatment resistance by identifying crucial genetic variables and revealing new therapeutic targets Despite the advantages, there are limitations in the study on CRISPR/Cas9-based gene editing for BC treatment resistance, for example, off-target effects and the improvement of delivery techniques are still major issues. Successful clinical translation depends on methods to improve the specificity and effectiveness of CRISPR/Cas9 editing and to solve these constraints. This review aims to explore the possibility of CRISPR/Cas9 gene editing as an innovative method of combating BC medication resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4162c264e1c96e6bc11bcded7869da58b56f7ba" target='_blank'>
              Molecular Mechanisms of Breast Cancer Drug Resistance and CRISPR/Cas9 Strategies to Overcome
              </a>
            </td>
          <td>
            B. Hussen, Bnar Saleh Ismael, Saman S. Abdulla, Noor Haval Jamal, S. A. Mustafa, Zana Baqi Najmalddin, Mohammed Fatih Rasul
          </td>
          <td>2024-12-30</td>
          <td>BioMed Target Journal</td>
          <td>1</td>
          <td>27</td>
        </tr>

        <tr id="Metabolic reprogramming, particularly lipid metabolism, is a hallmark of cancer progression and a critical vulnerability in prostate cancer (PCa). Two pivotal studies, one by Lu et al. and the other by Dairo et al., have illuminated the roles of HOXB13, a transcriptional regulator, and fatty acid synthase (FASN), a key enzyme in lipid biosynthesis, in the metabolic dysregulation of PCa. The Lu et al. study highlights HOXB13’s role in androgen receptor (AR)-independent PCa, where it regulates lipid metabolism via epigenetic mechanisms involving histone deacetylase HDAC3. A G84E mutation in HOXB13 disrupts this regulation, leading to increased lipid synthesis and a pro-metastatic phenotype. The Dairo et al. study demonstrates that FASN hypomethylation, coupled with increased expression, drives lipid biosynthesis critical for tumor growth. Both studies establish a link between HOXB13 mutations and FASN dysregulation, underscoring their interplay in PCa biology. Therapeutically, pharmacological inhibition of FASN mitigates the aggressive features of HOXB13-deficient or mutant PCa, highlighting lipid metabolism as a promising target. Despite their strengths, including robust methodologies, limitations include reliance on preclinical models and the need for broader patient diversity. These studies collectively emphasize the potential of metabolic and epigenetic interventions for precision medicine in PCa, paving the way for novel therapies targeting lipid metabolism in patients with specific genetic and epigenetic profiles.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc58c2230658f37cc79a630362aeb84c369fdaa5" target='_blank'>
              Metabolic Reprogramming in Prostate Cancer: The Roles of HOXB13 and FASN in Tumor Progression and Therapy
              </a>
            </td>
          <td>
            Faisal Saeed, Lara Harik
          </td>
          <td>2025-01-14</td>
          <td>Serican Journal of Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Breast cancer is characterized by genetic and epigenetic deregulations, leading to aberrant expression of tissue-specific genes that are normally silent in healthy breast tissue. Our previous work identified the embryonic stem cell-specific gene DNMT3B, a de novo DNA methyltransferase, as aberrantly activated in breast cancer, correlating with aggressive tumour behaviour and high relapse risk, regardless of molecular subtype. Through integrative bioinformatic analyses of DNA methylation and transcriptomic data, we identified 154 genes downregulated via DNMT3B-driven promoter hypermethylation, many of which are associated with high relapse risk. Notably, the tumour suppressor gene GATA3 emerged as a primary target of functional inactivation through either loss-of-function mutations or DNMT3B-controlled hypermethylation, in a mutually exclusive manner. Both mechanisms of GATA3 inactivation were associated with similar molecular signatures linked to tumour progression, increased malignancy, and poorer prognosis. However, distinct differences were observed, with immune- and inflammation-related genes enriched in GATA3 hypermethylation cases but depleted in mutation-driven silencing. Additionally, our analysis uncovered other potential tumour suppressor genes epigenetically repressed in aggressive breast cancers. These findings underscore a broader role of GATA3 inactivation beyond genetic alterations and suggest therapeutic opportunities to target epigenetically silenced tumour suppressors in aggressive breast tumours.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c122f003440fc5017c72299634c64a7c4caded88" target='_blank'>
              Discovery of Epigenetically Silenced Tumour Suppressor Genes in Aggressive Breast Cancer Through a Computational Approach
              </a>
            </td>
          <td>
            Anne-Laure Vitte, Florent Chuffart, Emmanuelle Jacquet, Eleni Nika, Mireille Mousseau, Ina Jung, S. Tabone-Eglinger, Thomas Bachelot, Sophie Rousseaux, Saadi Khochbin, Ekaterina Bourova-Flin
          </td>
          <td>2025-02-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Comparative genomic studies of Marek’s disease virus (MDV) have suggested that attenuated and virulent strains share >98% sequence identity. However, these estimates fail to account for variation in regions of the MDV genome harboring tandem repeats. To resolve these loci and enable assessments of intrapopulation diversity, we used a PacBio Sequel II platform to sequence MDV strains CVI988/Rispens (attenuated), HPRS-B14 (virulent), Md5 (very virulent) and 675A (very virulent plus). This approach enabled us to identify patterns of variation in tandem repeat regions that are consistent with known phenotypic differences across these strains. We also found CVI988/Rispens variants showing a 4.3-kb deletion in the Unique Short (US) region, resulting in the loss of five genes. These findings support a potential link between MDV tandem repeats and phenotypic traits like virulence and attenuation, and demonstrate that DNA viruses can harbor high levels of intrapopulation diversity in tandem repeat regions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9048b732c74cf2e0b72ef6607e53274d6fd6385" target='_blank'>
              High-Fidelity Long-Read Sequencing of an Avian Herpesvirus Reveals Extensive Intrapopulation Diversity in Tandem Repeat Regions
              </a>
            </td>
          <td>
            Alejandro Ortigas-Vásquez, Christopher D. Bowen, Daniel W. Renner, S. Baigent, Yaoyao Zhang, Yongxiu Yao, Venugopal Nair, Dave A. Kennedy, Moriah L. Szpara
          </td>
          <td>2025-02-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Cancer’s epigenetic landscape, a labyrinthine tapestry of molecular modifications, has long captivated researchers with its profound influence on gene expression and cellular fate. This review discusses the intricate mechanisms underlying cancer epigenetics, unraveling the complex interplay between DNA methylation, histone modifications, chromatin remodeling, and non-coding RNAs. We navigate through the tumultuous seas of epigenetic dysregulation, exploring how these processes conspire to silence tumor suppressors and unleash oncogenic potential. The narrative pivots to cutting-edge technologies, revolutionizing our ability to decode the epigenome. From the granular insights of single-cell epigenomics to the holistic view offered by multi-omics approaches, we examine how these tools are reshaping our understanding of tumor heterogeneity and evolution. The review also highlights emerging techniques, such as spatial epigenomics and long-read sequencing, which promise to unveil the hidden dimensions of epigenetic regulation. Finally, we probed the transformative potential of CRISPR-based epigenome editing and computational analysis to transmute raw data into biological insights. This study seeks to synthesize a comprehensive yet nuanced understanding of the contemporary landscape and future directions of cancer epigenetic research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf231d1889782a99759e5b17e3a47e9d23349015" target='_blank'>
              Mechanisms and technologies in cancer epigenetics
              </a>
            </td>
          <td>
            Z. Sherif, O. Ogunwobi, H. Ressom
          </td>
          <td>2025-01-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Neuroendocrine prostate cancer (NEPC) arises primarily through neuroendocrine transdifferentiation (NEtD) as an adaptive mechanism of therapeutic resistance. Models to define the functional effects of putative drivers of this process on androgen receptor (AR) signaling and NE cancer lineage programs are lacking. We adapted a genetically defined strategy from the field of cellular reprogramming to directly convert AR-active prostate cancer (ARPC) to AR-independent NEPC using candidate factors. We delineated critical roles of the pioneer factors ASCL1 and NeuroD1 in NEtD and uncovered their abilities to silence AR expression and signaling by remodeling chromatin at the somatically acquired AR enhancer and global AR binding sites with enhancer activity. We also elucidated the dynamic temporal changes in the transcriptomic and epigenomic landscapes of cells undergoing acute lineage conversion from ARPC to NEPC which should inform future therapeutic development. Further, we distinguished the activities of ASCL1 and NeuroD1 from the inactivation of RE-1 silencing transcription factor (REST), a master suppressor of a major neuronal gene program, in establishing a NEPC lineage state and in modulating the expression of genes associated with major histocompatibility complex class I (MHC I) antigen processing and presentation. These findings provide important, clinically relevant insights into the biological processes driving NEtD of prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b450ad259d614ef1f0c3aeece63e1bbfcda82938" target='_blank'>
              Defined cellular reprogramming of androgen receptor-active prostate cancer to neuroendocrine prostate cancer
              </a>
            </td>
          <td>
            Shan Li, Kai Song, Huiyun Sun, Yong Tao, Arthur Huang, Vipul Bhatia, B. Hanratty, Radhika A Patel, Henry W. Long, C. Morrissey, Michael C Haffner, Peter S. Nelson, Thomas G. Graeber, John K. Lee
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a0977541bd5b827628ce81b0938c0df52c0a295" target='_blank'>
              Proteogenomic analysis reveals adaptive strategies for alleviating the consequences of aneuploidy in cancer.
              </a>
            </td>
          <td>
            Jan-Eric Bökenkamp, Kristina Keuper, Stefan Redel, Karen Barthel, Leah Johnson, Amelie Becker, A. Wieland, M. Räschle, Zuzana Storchová
          </td>
          <td>2025-02-10</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Despite the development of HER2-targeting drugs such as trastuzumab and T-DXd, treatment resistance is a substantial challenge, often leading to relapse and distant metastasis. Tumor heterogeneity in HER2-positive breast cancer drives the evolution of resistant clones following therapeutic stress. However, the targetable drivers of anti-HER2 treatment resistance are not thoroughly identified. This study aims to use neoadjuvant-targeted therapy cohorts and a patient-derived organoid in vitro treatment model to uncover the potential targetable drivers of anti-HER2 treatment resistance. it is found that LINC01235 significantly enhances DNA replication licensing and chromosomal instability, fostering clonal expansion and evolution, and ultimately increasing resistance to therapeutic interventions. LINC01235 regulates global H3K27ac, H3K9ac, and H3K36me3 modifications, promotes H2A.Z expression in regulatory regions, and increases the accessibility of DNA licensing factors to their promoter regions. XRCC5 is identified as a key component for maintaining genomic stability, crucial for LINC01235's role in replication licensing. Furthermore, therapeutic strategies targeting LINC01235, including the use of antisense oligonucleotides or ATR inhibitors, which showed promise in overcoming treatment resistance are explored. These findings underscore the pivotal role of LINC01235 in driving resistance mechanisms and highlight novel avenues for targeted therapies to improve the outcomes of patients with HER2-positive breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fde95930ed4b5d2e1c4571f86786e616e7f2cf6" target='_blank'>
              LINC01235 Promotes Clonal Evolution through DNA Replication Licensing-Induced Chromosomal Instability in Breast Cancer.
              </a>
            </td>
          <td>
            Qi Zhang, Xuliren Wang, Zhibo Shao, Yi Zhang, Li-Ying Zhang, Ming Chen, Xujie Zhou, Han Zhu, Yue Zhou, Xinya Lu, Pei Li, W. Chi, Lun Li, Zhibo Shao, Shenglin Huang, J. Xue, Y. Chi, Jiong Wu, B. Xiu
          </td>
          <td>2025-02-14</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28cbf9783c62fb0f01914152c3e6d88182c98411" target='_blank'>
              LKB1 inactivation promotes epigenetic remodeling-induced lineage plasticity and antiandrogen resistance in prostate cancer
              </a>
            </td>
          <td>
            Fei Li, Pengfei Dai, Huili Shi, Yajuan Zhang, Juan He, Anuradha Gopalan, Dan Li, Yu Chen, Yarui Du, Guoliang Xu, Weiwei Yang, Chao Liang, Dong Gao
          </td>
          <td>2025-01-01</td>
          <td>Cell Research</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b99a1f2cb92c3a6d34aa29596c928af0010bece5" target='_blank'>
              Single-cell sequencing reveals the mechanisms of multiple myeloma progression: clarity or confusion?
              </a>
            </td>
          <td>
            Yunhui Xiang, G. Sun, Lvbo Tian, Pinpin Xiang, Chunbao Xie
          </td>
          <td>2025-02-07</td>
          <td>Annals of hematology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Osteosarcoma (OS) is a prevalent primary malignant bone tumor, typically managed through a combination of neoadjuvant chemotherapy and surgical interventions. Recent advancements in early detection and the use of novel chemotherapeutic agents have significantly improved the 5-year survival rate of OS patients. However, some patients fail to achieve the desired treatment outcomes despite undergoing intensive chemotherapy and surgicals procedures, with chemotherapy resistance emerging as a critical factor contributing to therapeutic failure in OS. Noncoding RNAs (ncRNAs) are a group of RNAs that lack protein-coding capacity but play a crucial role in tumor progression by modulating various biological characteristics of cancer cells, such as proliferation, apoptosis, migration, invasion, and drug resistance. Emerging evidence indicates that the dysregulated expression of numerous ncRNAs in OS cells can influence the response to chemotherapeutic agents by modulating processes such as cell apoptosis, signaling pathways, intracellular drug concentrations, and cell autophagy. This review aims to elucidate the roles and mechanisms of ncRNAs in mediating drug resistance in OS, offering new insights for investigating novel pathways underlying drug resistance, overcoming tumor resistance to therapeutics, and developing innovative chemotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbb08d17a2f1faca4973ae29ad1b2a3dacb83f65" target='_blank'>
              Unraveling the impact of noncoding RNAs in osteosarcoma drug resistance: a review of mechanisms and therapeutic implications.
              </a>
            </td>
          <td>
            Y. Pei, Shenglong Li
          </td>
          <td>2024-12-31</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Objectives Unlike other diseases, cancer is not just a genome disease but should broadly be viewed as a disease of the cellular machinery. Therefore, integrative multifaceted approaches are crucial to understanding the complex nature of cancer biology. Bcl-2 (B-cell lymphoma 2), encoded by the human Bcl-2 gene, is a critical anti-apoptotic protein that regulates cell death pathways, primarily by inhibiting apoptosis. It plays a pivotal role in maintaining cellular homeostasis by preventing premature or excessive cell death. Genetic variations and dysregulation of Bcl-2 are particularly significant in cancer, as they disrupt the normal apoptotic machinery, enabling cancer cells to evade programmed cell death. Single nucleotide polymorphisms (SNPs) are considered viable diagnostic and therapeutic biomarkers for various cancers. Therefore, this study explores the association between SNPs in Bcl-2 and the structural, functional, protein-protein interactions (PPIs), drug binding and dynamic characteristics. Methods Comprehensive cross-validated bioinformatics tools and molecular dynamics (MD) simulations. Multiple sequence, genetic, structural and disease phenotype analyses were applied in this study. Results Analysis revealed that out of 130 mutations, approximately 8.5% of these mutations were classified as pathogenic. Furthermore, two particular variants, namely, Bcl-2G101V and Bcl-2F104L, were found to be the most deleterious across all analyses. Following 500 ns, MD simulations showed that these mutations caused a significant distortion in the protein conformational, protein-protein interactions (PPIs), and drug binding landscape compared to Bcl-2WT. Conclusion Despite being a predictive study, the findings presented in this report would offer a perspective insight for further experimental investigation, rational drug design, and cancer gene therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5f67dee38c9954b65ace573045ef349b37771a6" target='_blank'>
              Integrative genomic analyses combined with molecular dynamics simulations reveal the impact of deleterious mutations of Bcl-2 gene on the apoptotic machinery and implications in carcinogenesis
              </a>
            </td>
          <td>
            G. Elamin, Zhichao Zhang, Depika Dwarka, K. Kasumbwe, John Mellem, Nompumelelo P. Mkhwanazi, P. Madlala, Mahmoud E. S. Soliman
          </td>
          <td>2025-01-07</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87208750323d073805f3062a30f76b4849f02905" target='_blank'>
              Unveiling cell-type-specific microRNA networks through alternative polyadenylation in glioblastoma
              </a>
            </td>
          <td>
            Mert Cihan, Greta Schmauck, Maximilian Sprang, Miguel Andrade
          </td>
          <td>2025-01-21</td>
          <td>BMC Biology</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Oncogenic gene fusions are key drivers of cancer, yet most remain untargetable by current therapies. Here, we establish CRISPR-PspCas13b as a personalizable platform for systematic silencing of various fusion transcripts. We reveal that recognition and cleavage of the breakpoint sequence by PspCas13b disrupts the fusion transcript, resulting in unexpected RNA nicking and ligation near the cleavage site, which generates out-of-frame, translation-incompetent transcripts. This approach efficiently degrades canonical and drug-resistant BCR::ABL1 mutants (e.g., T315I), a primary cause of resistance to tyrosine kinase inhibitors (TKIs) and relapse in chronic myeloid leukemia (CML). Silencing T315I BCR::ABL1 mRNA in drug-resistant CML cells triggers extensive transcriptomic, proteomic, and phosphoproteomic remodelling, causing erythroid differentiation and apoptosis. Beyond BCR-ABL1 mutants, personalized design of PspCas13b effectively silences other undruggable fusions, including RUNX1::RUNX1T1 and EWSR1::FLI1, key drivers in acute myeloid leukemia and in Ewing sarcoma, respectively. Collectively, this study establishes a framework for systematic, precise, and personalizable targeting of otherwise undruggable or drug-resistant oncogenic transcripts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52d993b12a12216338adf29eefc996f4f14d40a3" target='_blank'>
              Systematic silencing of oncogenic fusion transcripts with ultra-precise CRISPR-Cas13b
              </a>
            </td>
          <td>
            Wenxin Hu, Honglin Chen, J. Casan, Carolyn Shembrey, Steven He, Lauren M Brown, Timothy P. Hughes, Deborah L. White, Antoine de Weck, I. Voskoboinik, Joseph A Trapani, Paul G Ekert, Teresa Sadras, Mohamed Fareh
          </td>
          <td>2025-02-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Genomes of clonally reproducing fungal pathogens are often compartmentalized into conserved core and lineage-specific accessory regions (ARs), enriched in transposable elements (TEs). ARs and TEs are thought to promote pathogen adaptation, but direct experimental evidence is sparse. Using an evolve and re-sequence approach, we found that serial passaging of the cross-kingdom fungal pathogen Fusarium oxysporum through tomato plants or axenic media rapidly increased fitness under the selection condition. TE insertions were the predominant type of mutations in the evolved lines, with a single non-autonomous hAT-type TE accounting for 63% of total events detected. TEs inserted preferentially at sites of histone H3 lysine 27 trimethylation, a hallmark of ARs. Recurrent evolutionary trajectories during plate adaptation led to increased proliferation concomitant with reduced virulence. Unexpectedly, adaptive mutations in accessory genes strongly impacted core functions such as growth, development, quorum sensing or virulence. Thus, TEs and ARs drive rapid adaptation in this important fungal pathogen.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbe5b9e3d56f1bc2cff4dbc0a29dd960a776205f" target='_blank'>
              Transposons and accessory genes drive adaptation in a clonally evolving fungal pathogen
              </a>
            </td>
          <td>
            Cristina López Díaz, Dilay Hazal Ayhan, Ana Rodríguez López, Lucía Gómez Gil, Li-Jun Ma, A. Di Pietro
          </td>
          <td>2025-02-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Genetic variation underpins evolutionary change, but accumulation of slightly deleterious mutations also increases mutation load. There are multiple factors affecting the extent of load such as population size and breeding system, yet other potential determinants remain unexplored. A common macromutation, whole-genome duplication (WGD) occurs broadly across Eukaryotes, yet we lack a clear understanding of how WGD impacts neutral and selective processes within a population. Using forward simulations and empirical analysis of 632 short- and 16 long-read sequenced individuals of Arabidopsis arenosa (23 diploid and 42 natural autotetraploid populations), we test for the effects of WGD on genome-wide diversity and mutation load. Our simulations show how genetic variation gradually rises in autotetraploids due to increase of mutational target size. Moreover, mutation load increases due to relaxed purifying selection when deleterious mutations are masked by additional chromosome copies. Empirical data confirm these patterns, showing significant increase in nucleotide diversity, ratios of non-synonymous to synonymous SNPs, and number of indels and large structural variants in A. arenosa autotetraploids. However, a rather modest increase in load proxies together with a broad distribution and niche of autotetraploids suggests load accumulation has not (yet) limited their successful expansion. Overall, we demonstrate a complex interplay between neutral processes and purifying selection in shaping genetic variation following WGD and highlight ploidy as an important determinant of genetic diversity and mutation load in natural populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86bbacd2db01892da50f5a93e5f2a705f35e9543" target='_blank'>
              Whole-genome duplication increases genetic diversity and load in outcrossing Arabidopsis
              </a>
            </td>
          <td>
            Jakub Vlček, Tuomas Hämälä, Cristina Vives Cobo, Emma Curran, G. Šrámková, T. Slotte, R. Schmickl, Levi Yant, Filip Kolář
          </td>
          <td>2025-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="In mammalian cells, gene copy number is tightly controlled to maintain gene expression and genome stability. However, a common molecular feature across cancer types is oncogene amplification, which promotes cancer progression by drastically increasing the copy number and expression of tumor-promoting genes. For example, in tyrosine kinase inhibitor (TKI)-resistant lung adenocarcinoma (LUAD), oncogene amplification occurs in over 40% of patients’ tumors. Despite the prevalence of oncogene amplification in TKI-resistant tumors, the mechanisms facilitating oncogene amplification are not fully understood. Here, we find that LUADs exhibit a unique chromatin signature demarcated by strong CTCF and cohesin deposition in drug-naïve tumors, which correlates with the boundaries of oncogene amplicons in TKI-resistant LUAD cells. We identified a global chromatin priming effect during the acquisition of TKI resistance, marked by a dynamic increase of H3K27Ac, cohesin loading, and inter-TAD interactions, which occurs before the onset of oncogene amplification. Furthermore, we have found that the METTL7A protein, which was previously reported to localize to the endoplasmic reticulum and inner nuclear membrane, has a novel chromatin regulatory function by binding to amplified loci and regulating cohesin recruitment and inter-TAD interactions. Surprisingly, we discovered that METTL7A remodels the chromatin landscape prior to large-scale copy number gains. Furthermore, while METTL7A depletion has little effect on the chromatin structure and proliferation of drug-naïve cells, METTL7A depletion prevents the formation and maintenance of TKI resistant-clones, highlighting the specific role of METTL7A as cells are becoming resistant. In summary, we discovered an unexpected mechanism required for the acquisition of TKI resistance regulated by a largely uncharacterized factor, METTL7A. This discovery sheds light into the maintenance of oncogene copy number and paves the way to the development of new therapeutics for preventing TKI resistance in LUAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/864e77e3ffbc341dd750e7bc0c5fd24e1c53b031" target='_blank'>
              Epigenetic priming promotes acquisition of tyrosine kinase inhibitor resistance and oncogene amplification in human lung cancer
              </a>
            </td>
          <td>
            Rebecca M. Starble, Eric G. Sun, Rana Gbyli, Jonathan S. D. Radda, Jiuwei Lu, Tyler B. Jensen, Ning Sun, Nelli Khudaverdyan, Bomiao Hu, M. Melnick, Shuai Zhao, Nitin Roper, Gang Greg Wang, Jikui Song, Katerina Politi, Siyuan Wang, Andrew Xiao
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Epigenetics plays a critical role relative to other branches of genetics, as it makes it possible to regulate gene expression without changing the nucleotide sequence of DNA molecules. This process allows cells to respond to external signals and adapt to  changes in  the  environment while keeping their genetic information intact. The  main mechanisms of  epigenetic regulation include DNA methylation, posttranslational modifications of  histones, chromatin remodeling and  regulation through non-coding RNAs. These processes play a key role in fundamental biological processes such as cellular differentiation, organismal development, and adaptation to environmental conditions.Disturbances in epigenetic mechanisms can lead to various pathologies, including cancer, neurological and  autoimmune diseases. Understanding the  mechanisms of  epigenetic regulation opens new avenues for  the  development of  targeted therapies that can correct abnormal epigenetic profiles without changing the DNA structure itself.In  recent years, the  development and  application of  innovative technologies, such  as  the  CRISPR/Cas9 genome editing system, have significantly expanded the ability to study epigenetic mechanisms and their relationship with diseases. These technologies allow not only a deeper understanding of epigenetic changes, but also the development of new therapeutic approaches, especially in the field of oncology. Research in  epigenetics is  also focusing on  the  interaction between epigenetic changes and the immune system, which opens new perspectives for the development of immunotherapies. The search for new markers of epigenetic disorders and therapeutic agents may lead to the development of individualized treatments that take into account the unique epigenetic profile of each patient.The role of epigenetic modifications in the development of diseases and the creation of new therapeutic strategies cannot be overestimated. Recent research in this area is revealing the potential of epigenetic approaches to treat a wide range of diseases, ushering in a new era in medicine where understanding and correcting epigenetic changes will be the key to effective treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d047bbd846068847a5bb8e163e24692ce5b5c48" target='_blank'>
              Epigenetic modulation in medicine: Regulation of gene expression in the context of pathogenesis and therapy
              </a>
            </td>
          <td>
            M. A. Omarov, A. R. Mulyukov, R. V. Khalitov, S. I. Safarov, G. U. Ayupova, O. N. Demianenko, V. V. Butenko, A. A. Safronova, R. R. Zainetdinova, O. Y. Ianturina, A. A. Miroshnikov
          </td>
          <td>2024-12-28</td>
          <td>Acta Biomedica Scientifica</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb6bea0ac205c045a3acd7f52f1841e20780dc8e" target='_blank'>
              Utilizing sc-linker to integrate single-cell RNA sequencing and human genetics to identify cell types and driver genes associated with non-small cell lung cancer
              </a>
            </td>
          <td>
            Yangfan Zhou, Liang Zhao, Meimei Cai, Doudou Luo, Y. Pang, Jianhao Chen, Qicong Luo, Qin Lin
          </td>
          <td>2025-01-23</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="CAR-T therapies utilizing T cells engineered with chimeric antigen receptors (CARs) have revolutionized the treatment of hematologic and immune-related malignancies. However, the anti-tumor capability of engineered CAR-T cells are often not persistent, which greatly dampen its clinical efficacy. To address this limitation, massive parallel genetic screens were widely used to identified novel targets and regulators that enhance T cell anti-tumor capability and persistence in tumor microenvironment. We hypothesized that by combining the pooled screen data from multiple independent genetic screens we could provide a systematic, comprehensive, and robust analysis of the effect of gene perturbation on T-cell based immunotherapies. After collecting data from previously published T cell screens, including CRISPR-based and ORF-based screens, through Gene Expression Omnibus (GEO), we reprocessed the gene hits summary and conducting a pathway enrichment analysis. A T cell screen perturbation score (TPS) was employed to quantifies the impact of the gene on T cell function. Additionally, gene expression data (both bulk RNA level and single-cell RNA level) from autoimmune disease and T cell-derived cancers were analyzed to pick up gene perturbations that potentially augment T cell proliferation. We integrated all data and analysis on 27 T-cells screens into our state-of-the-art T cell perturbation genomics database (TCPGdb), which is made easily accessible through our webserver and allows users to interactively explore the impact of query genes on T cell function based on prior screen data and our TPS scoring module. TCPGdb is publicly accessible at http://tcpgdb.sidichenlab.org/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a099e229f2c35d38ee6b66c8422aec7d483d003c" target='_blank'>
              TCPGdb: A comprehensive T cell perturbation genomics database for identification of critical T cell regulators
              </a>
            </td>
          <td>
            Chuanpeng Dong, Feifei Zhang, Kaiyuan Tang, Nipun Verma, Xinxin Zhu, Di Feng, James Cai, Hongyu Zhao, Sidi Chen
          </td>
          <td>2024-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Despite advancements in the treatment of advanced prostate cancer, including metastatic castration-resistant prostate cancer (mCRPC), the disease remains lethal, often developing resistance to conventional therapies such as androgen deprivation therapy (ADT) and chemotherapy. Therapy resistance poses a significant challenge, particularly in aggressive forms of prostate cancer. Data on TP53 mutations, especially gain-of-function (GOF) mutations, and their specific impact on prostate cancer metabolism remain limited, underscoring the need for a deeper understanding to uncover new therapeutic avenues. This study aimed to explore the metabolic consequences of TP53 GOF mutations (R175H and R273H) in prostate cancer progression using metabolomics analysis. Approximately 300 metabolites were significantly altered across PC3-GFP (KO), PC3-TP53WT (WT), and PC3-TP53 mutant (R175H & R273H) models. Key findings revealed dysregulation of essential metabolites, such as α-ketoglutarate and spermidine, which support cancer cell survival and progression. These insights not only enhance our understanding of the interplay between TP53 mutations and metabolic alterations but also identify potential biomarkers for prognosis and therapeutic targeting in therapy-resistant prostate cancer. In the TP53-R175H mutant, numerous essential metabolites were dysregulated (downregulated: glutamine, GAP/DHAP, and α-ketoglutarate; Upregulated: deoxyuridine, spermidine, and pseudouridine) when compared with the control (KO) thereby causing metabolic shifts that favor cancer cell survival. Conversely, metabolites such as GlaNAC/GlcNAC, adenosine, acetoacetic acid, acetylcholine, octopamine, itaconic acid, lactic acid, and NADP+ are all downregulated in the R273H mutant. While spermine, spermidine, N-acetyl-DL-serine, pseudouridine, deoxyguanosine, sucrose, and AMP/dGMP are upregulated by the R273H mutant but downregulated in GFP (KO). These findings showed that TP53 GOF mutations (R175H & R273H) cause metabolic alterations in numerous metabolites essential for tumor survival and progression. Furthermore, it may provide therapeutic benefits by identifying appropriate biomarkers that could be used as diagnostic and/or prognostic tools for prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6235a81e6d316a547376d1bb83acc9177dd3e97b" target='_blank'>
              Impact of TP53 Gain-of-Function Mutations on Metabolic Reprogramming in Prostate Cancer Progression
              </a>
            </td>
          <td>
            Farida Jazuli Usman, D. G. Gupta, S. Abdulkadir, Ibrahim Muhammad Usman, Adamu Jibrin Alhassan, Young A Yoo, Haris Jaafar Bello, Binta G. Kurfi, K. Babagana
          </td>
          <td>2025-01-10</td>
          <td>Asian Journal of Biochemistry, Genetics and Molecular Biology</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9aa45e1efe02786e2e4534cf1ae408e65c7ffadc" target='_blank'>
              To describe the subsets of malignant epithelial cells in gastric cancer, their developmental trajectories and drug resistance characteristics
              </a>
            </td>
          <td>
            Tingting Xu, Tianying Zhang, Yan Sun, Sijia Wu
          </td>
          <td>2025-01-27</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ef2ae6c7368d6a4d9d50c4f417829a563bfed3b" target='_blank'>
              Evaluation of Cas13d as a tool for genetic interaction mapping
              </a>
            </td>
          <td>
            Ghanem El Kassem, Jasmine Hillmer, Michael Boettcher
          </td>
          <td>2025-02-14</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="DNA methylation is an essential epigenetic modification that plays a crucial role in regulating gene expression and maintaining genomic stability. With the advancement in sequencing technology, methylation studies have provided valuable insights into the diagnosis of rare diseases through the various identification of episignatures, epivariation, epioutliers, and allele-specific methylation. However, current methylation studies are not without limitations. This mini-review explores the current understanding of DNA methylation in rare diseases, highlighting the key mechanisms and diagnostic potential, and emphasizing the need for advanced methodologies and integrative approaches to enhance the understanding of disease progression and design more personable treatment for patients, given the nature of rare diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30b0846fe63347313e7e0457ade21ff242a0edeb" target='_blank'>
              Expanding Upon Genomics in Rare Diseases: Epigenomic Insights
              </a>
            </td>
          <td>
            Jia W Tan, Emily J. Blake, Joseph D. Farris, Eric W. Klee
          </td>
          <td>2024-12-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Adaptation to challenging environmental conditions is crucial for the survival/fitness of all organisms. Alongside genetic mutations that provide adaptive potential during environmental challenges, epigenetic modifications offer dynamic, reversible, and rapid mechanisms for regulating gene expression in response to environmental changes in both evolution and daily life, without altering DNA sequences or relying on accidental favorable mutations. The widespread conservation of diverse epigenetic mechanisms - like DNA methylation, histone modifications, and RNA interference across diverse species, including plants - underscores their significance in evolutionary biology. Remarkably, environmentally induced epigenetic alterations are passed to daughter cells and inherited transgenerationally through germline cells, shaping offspring phenotypes while preserving adaptive epigenetic memory. Throughout anthropoid evolution, epigenetic modifications have played crucial roles in: i) suppressing transposable elements and viral genomes intruding into the host genome; ii) inactivating one of the X chromosomes in female cells to balance gene dosage; iii) genetic imprinting to ensure expression from one parental allele; iv) regulating functional alleles to compensate for dysfunctional ones; and v) modulating the epigenome and transcriptome in response to influence from the gut microbiome among other functions. Understanding the interplay between environmental factors and epigenetic processes may provide valuable insights into developmental plasticity, evolutionary dynamics, and disease susceptibility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f7f5656f1a791fd7b047008e717c5695b95280c" target='_blank'>
              Epigenetics in evolution and adaptation to environmental challenges: pathways for disease prevention and treatment.
              </a>
            </td>
          <td>
            H. Abdolmaleky, S. Nohesara, Jin-Rong Zhou, S. Thiagalingam
          </td>
          <td>2025-02-13</td>
          <td>Epigenomics</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="
 Introduction: Mutations in KEAP1 and NFE2L2 (NFR2) occur in approximately 20% of non-small cell lung cancer (NSCLC) patients and are associated with resistance to radiotherapy and systemic treatments. In previous research, we found that not all KEAP1 mutations found in NSCLC patients are pathogenic. These benign mutations exhibit a wild-type phenotype that does not confer therapy resistance. With only roughly 100 of 13,000 (<1%) potential single amino acid substitution mutations in KEAP1 characterized thus far, here we comprehensively evaluated all possible point mutations in KEAP1 using deep mutational scanning (DMS). Methods: The High-throughput Mutagenesis by Integrated TilEs (MITE) method was used to generate the KEAP1 DMS library. In short, the KEAP1 sequence was segmented into 21 variable regions (tiles), each comprising a 90-base pair variable region flanked by a 30-base pair constant region. Each codon position included the 20 possible natural amino acids plus a stop codon, for a total of 13,076 KEAP1 variants. The tiles were then assembled to ensure that each full-length KEAP1 construct contained only a single amino acid substitution. The KEAP1 DMS lentivirus library was transduced into H1299-KEAP1NULLcells, followed by selective pressure with H2O2, radiation, or vehicle control. The KEAP1 ORF was then amplified from surviving cell genomic DNA, and mutations were quantified using next-generation sequencing. Functional annotations were derived through mutation enrichment analysis. Results: Both H2O2 and radiation selective pressure of the KEAP1 DMS library induced a bimodal distribution of KEAP1 mutations, with cells harboring KEAP1 nonsense mutations being enriched and cells with silent mutations being depleted. By setting a threshold of 95% specificity for silent mutations to differentiate between benign and pathogenic mutations, we identified 94% of nonsense mutations and 88% of previously characterized pathogenic mutations as pathogenic. While most KEAP1 mutations were categorized as benign (64%), analysis of mutations in TCGA NSCLC patients showed pathogenic mutations were enriched (72%). Mapping mutation phenotypes onto a KEAP1 structural model revealed an enrichment of pathogenic mutations at critical functional sites, including the NRF2 binding pocket of the Kelch domain, the BTB homodimerization interface, and the hydrophobic core of the oxidative stress-sensing IVR domain. Conclusions: Through deep mutational scanning experiments, we generated a comprehensive list of pathogenic KEAP1 single amino acid substitutions. These results align with previously known biological functions and prior single amino acid substitutions characterized in the literature. These findings will facilitate the identification of benign and pathogenic KEAP1 mutations in clinical settings, enabling personalized radiotherapeutic and systemic therapy strategies for patients harboring KEAP1 mutations.
 Citation Format: Noah Kastelowitz, Shashank Shrishrimal, Soyeong Jun, Anni MY Zhang, Rui Wang, Ilayda Ilerten, Maximilian Diehn. Deep mutational scanning of KEAP1 to identify mutations resistant to radiation therapy and oxidative stress. [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr BO23.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f223327077f38ac09a18b5895edd5fce4b10c15a" target='_blank'>
              Abstract B023: Deep mutational scanning of KEAP1 to identify mutations resistant to radiation therapy and oxidative stress
              </a>
            </td>
          <td>
            Noah Kastelowitz, S. Shrishrimal, Soyeong Jun, Anni MY Zhang, Rui Wang, Ilayda Ilerten, Maximilian Diehn
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a602005403e19b0fe4fd439cfa9748d3ee799d93" target='_blank'>
              Identification of cancer cell-intrinsic biomarkers associated with tumor progression and characterization of SFTA3 as a tumor suppressor in lung adenocarcinomas
              </a>
            </td>
          <td>
            Yu Zhao, Chengcheng Zhou, Ling Zuo, Haoming Yan, Yuhan Gu, Hong Liu, Guiping Yu, Xiaorong Zhou
          </td>
          <td>2025-01-08</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Stemness-associated cell states are linked to chemotherapy resistance in AML. We uncovered a direct mechanistic link between expression of the stem cell transcription factor GATA2 and drug resistance. The GATA-binding protein 2 (GATA2) plays a central role in blood stem cell generation and maintenance. We find substantial intra- and inter-patient variability in GATA2 expression across AML patient samples. GATA2 expression varies by molecular subtype and has been linked to outcome. In a murine model, KMT2A-MLL3 driven AML originating from a stem cell or immature progenitor cell population have higher Gata2 expression and are more resistant to the standard AML chemotherapy agent doxorubicin. Deletion of Gata2 resulted in more robust induction of p53 following exposure to doxorubicin. ChIP-Seq, RNA-Seq and functional studies revealed that GATA2 regulates the expression of RASSF4, a modulator of the p53 inhibitor MDM2. GATA2 and RASSF4 are anti-correlated in human cell lines and AML patient cell bulk and single cell expression datasets. Knockdown of Rassf4 in Gata2 low cells resulted in doxorubicin or nutlin-3 resistance. Conversely, overexpression of Rassf4 results in sensitization of cells expressing high levels of Gata2. Finally, doxorubicin and nutlin-3 are synergistic in Gata2-high murine AML, as well as AML patient samples. We discovered a previously unappreciated role for GATA2 in dampening p53-mediated apoptosis via transcriptional regulation of RASSF4, a modulator of MDM2. This role for GATA2 directly links the expression of a stemness associated transcription factor to chemotherapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/149e4d0de77d706d929171b654f103344701ea6a" target='_blank'>
              GATA2 links stemness to chemotherapy resistance in acute myeloid leukemia.
              </a>
            </td>
          <td>
            Fatemeh Alikarami, H. Xie, Simone S. Riedel, Haley Goodrow, D. Barrett, Leila Mahdavi, Alexandra Lenard, Changya Chen, T. Yamauchi, Etienne Danis, Zhendong Cao, Vu L. Tran, M. Jung, Yapeng Li, Hua Huang, Junwei Shi, Kai Tan, D. Teachey, E. Bresnick, Tobias Neff, Kathrin M Bernt
          </td>
          <td>2025-01-22</td>
          <td>Blood</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="A pseudogene is a non-functional copy of a protein-coding gene. Processed pseudogenes, which are created by the reverse transcription of mRNA and subsequent integration of the resulting cDNA into the genome, being a major pseudogene class, represent a significant challenge in genome analysis due to their high sequence similarity to the parent genes and their frequent absence in the reference genome. This homology can lead to errors in variant identification, as sequences derived from processed pseudogenes can be incorrectly assigned to parental genes, complicating correct variant calling. In this study, we quantified the occurrence of variant calling errors associated with pseudogenes, generated by the most popular germline variant callers, namely GATK-HC, DRAGEN, and DeepVariant, when analysing 30x human whole-genome sequencing data (n = 13,307). The results show that the presence of pseudogenes can interfere with variant calling, leading to false positive identifications of potentially clinically relevant variants. Compared to other approaches, DeepVariant was the most effective in correcting these errors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1dbcef1abd35252b28627cdf942fe6a6b40cb02f" target='_blank'>
              Quantitative Analysis of Pseudogene-Associated Errors During Germline Variant Calling
              </a>
            </td>
          <td>
            Artem Podvalnyi, Arina Kopernik, Mariia Sayganova, M. Woroncow, G. Zobkova, A. Smirnova, Anton Esibov, A. Deviatkin, P. Volchkov, Eugene Albert
          </td>
          <td>2025-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="In this review, we explore the application of next-generation sequencing in liver cancer research, highlighting its potential in modern oncology. Liver cancer, particularly hepatocellular carcinoma, is driven by a complex interplay of genetic, epigenetic, and environmental factors. Key genetic alterations, such as mutations in TERT, TP53, and CTNNB1, alongside epigenetic modifications such as DNA methylation and histone remodeling, disrupt regulatory pathways and promote tumorigenesis. Environmental factors, including viral infections, alcohol consumption, and metabolic disorders such as nonalcoholic fatty liver disease, enhance hepatocarcinogenesis. The tumor microenvironment plays a pivotal role in liver cancer progression and therapy resistance, with immune cell infiltration, fibrosis, and angiogenesis supporting cancer cell survival. Advances in immune checkpoint inhibitors and chimeric antigen receptor T-cell therapies have shown potential, but the unique immunosuppressive milieu in liver cancer presents challenges. Dysregulation in pathways such as Wnt/β-catenin underscores the need for targeted therapeutic strategies. Next-generation sequencing is accelerating the identification of genetic and epigenetic alterations, enabling more precise diagnosis and personalized treatment plans. A deeper understanding of these molecular mechanisms is essential for advancing early detection and developing effective therapies against liver cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf941fd49f418f7109e7e4bbca5152a8ddaa54e7" target='_blank'>
              Advances and challenges in molecular understanding, early detection, and targeted treatment of liver cancer
              </a>
            </td>
          <td>
            Ji Shi, Xu Zhu, Jun-Bo Yang
          </td>
          <td>2025-01-27</td>
          <td>World Journal of Hepatology</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Essentiality studies have traditionally focused on coding regions, often overlooking other small genetic regulatory elements. To address this, we obtained a high-resolution essentiality map at near-single-nucleotide precision of the genome-reduced bacterium Mycoplasma pneumoniae, combining transposon libraries containing promoter or terminator sequences. By integrating temporal transposon-sequencing data, we developed a novel method of essentiality assessment based on k-means unsupervised clustering, which provides dynamic and quantitative information on the fitness contribution of different genomic regions. We compared the insertion tolerance and persistence of the two engineered libraries, assessing the local impact of transcription and termination on cell fitness. Essentiality assessment at the local base-level revealed essential protein domains and small genomic regions that are either essential or inaccessible to transposon insertion. We also identified structural regions within essential genes that tolerate transposon disruptions, resulting in functionally split proteins. Overall, this study presents a nuanced view of gene essentiality, shifting from static and binary models to a more accurate perspective. Additionally, it provides valuable insights for genome engineering and enhances our understanding of the biology of genome-reduced cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0403d822930ef2b08e4a3eb27b77353feeed21e9" target='_blank'>
              Quantitative essentiality in a reduced genome: a functional, regulatory and structural fitness map
              </a>
            </td>
          <td>
            Samuel Miravet-Verde, Raul Burgos, Eva Garcia-Ramallo, Marc Weber, Luis Serrano
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Genomic rearrangements play an important role in shaping genetic diversity, as they enable the generation of novel structural variations through specific genome manipulation tools. These variations contribute to phenotypic differences among individuals within a population, thereby serving as the foundation for natural selection and driving evolutionary processes. In recent years, Synthetic Chromosome Rearrangement and Modification by LoxP-mediated Evolution (SCRaMbLE) has emerged as a promising tool for studying genomic rearrangements. SCRaMbLE utilizes site-specific recombination mediated by loxPsym sites to induce targeted chromosomal rearrangements in yeast cells. In this review, we provide a comprehensive overview of recent advancements in optimization strategies of the SCRaMbLE system and discuss influential factors that affect its performance based on recent research findings. We demonstrate how the SCRaMbLE system can be employed for pathway engineering, phenotype improvement, genome minimization, and dissection of genotype-to-phenotype relationships. We highlight both the advantages and challenges associated with SCRaMbLE and envision its potential applications beyond yeast genetics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0314619e822b8ab25ecabe0ad0985b3e4d4d9a40" target='_blank'>
              SCRaMbLE: a versatile tool for genome manipulation.
              </a>
            </td>
          <td>
            Li Cheng, Jielin Li, Yaojun Chen, Junbiao Dai
          </td>
          <td>2025-02-12</td>
          <td>Physiology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58506d08da7a5b25a950b60c249aaba7b6397bbd" target='_blank'>
              Neutral drift upon threshold-like selection promotes variation in antibiotic resistance phenotype
              </a>
            </td>
          <td>
            A. N. Erdoğan, P. Dasmeh, Raymond D. Socha, Johnson Chen, Ben Life, Rachel Jun, Linda Kiritchkov, Dan Kehila, Adrian W. R. Serohijos, N. Tokuriki
          </td>
          <td>2024-12-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Gene therapy and genome editing have emerged as transformative approaches in the management of a diverse range of genetic and acquired diseases. This evaluation offers a thorough examination of the present state and prospects of these innovative technologies. Gene therapy is a prospective approach to the treatment and prevention of a variety of conditions, including complex cancers and inherited genetic disorders, which entail the introduction, removal, or modification of genetic material within a patient's cells. Genome editing, particularly through techniques such as CRISPR-Cas9, enables targeted corrections of genetic defects and opens new possibilities for personalized medicine by allowing for precise modifications at the DNA level. The review addresses the ethical implications, clinical applications, and significant advancements of these technologies. This article endeavors to underscore the substantial influence of gene therapy and genome editing on contemporary medicine by assessing the most recent research and clinical trials, thereby emphasizing their potential to revolutionize disease treatment and management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25fcf14d7421a285bda69ccae261660da8cc8fcc" target='_blank'>
              Gene Therapy and Gene Editing: Current Trends and Future Prospects of Molecular Medicine.
              </a>
            </td>
          <td>
            Aniruddha Sen, Vijay Singh, Shailendra Dwivedi, Ruchika Agrawal, Akash Bansal, Shashank Shekhar, Sudhir Shyam Kushwaha
          </td>
          <td>2025-02-18</td>
          <td>Current gene therapy</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Introduction Rice samples exposed to the space environment have generated diverse phenotypic variations. Miniature-inverted-repeat transposable elements (MITEs), often found adjacent to genes, play a significant role in regulating the plant genome. Herein, the contribution of MITEs in regulating space-mutagenic phenotypes was explored. Methods The space-mutagenic phenotype changes in the F3 to F5 generations of three space-mutagenic lines from the rice varieties Dongnong423 (DN423) and Dongnong (DN416) were meticulously traced. Rice leaves samples at the heading stage from three space-mutagenic lines were subjected to high coverage whole-genome bisulfite sequencing and whole-genome sequencing. These analyses were conducted to investigate the effects of MITEs related epigenetic and genetic variations on space-mutagenic phenotypes. Results and discussion Studies have indicated that MITEs within gene regulatory regions might contribute to the formation and differentiation of space-mutagenic phenotypes. The space environment has been shown to induce the transposable elements insertion polymorphisms of MITEs (MITEs-TIPs), with a notable preference for insertion near genes involved in stress response and phenotype regulation. The space-induced MITEs-TIPs contributed to the formation of space-mutagenic phenotype by modulating the expression of gene near the insertion site. This study underscored the pivotal role of MITEs in modulating plant phenotypic variation induced by the space environment, as well as the transgenerational stability of these phenotypic variants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93bd51f2bf12b147c09ffbe77911db1ad254e58e" target='_blank'>
              Miniature-inverted-repeat transposable elements contribute to phenotypic variation regulation of rice induced by space environment
              </a>
            </td>
          <td>
            Lishan Chen, Qing Yang, Yan Zhang, Yeqing Sun
          </td>
          <td>2025-01-08</td>
          <td>Frontiers in Plant Science</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69cfefb50db5bb84559d86b5f736278ac09bb363" target='_blank'>
              LINC00626 drives tamoxifen resistance in breast cancer cells by interaction with UPF1
              </a>
            </td>
          <td>
            Hui Yuan, Lianbang Zhou, Wei Hu, Min Yang
          </td>
          <td>2025-01-23</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Epigenetics encompasses heritable and stable changes in gene expression caused by external chromosomal modifications, without altering the underlying DNA sequence. Epigenetic modifications, established during early development and maintained through successive cell divisions, play a critical role in regulating gene expression. Post-translational modifications (PTMs) are a key aspect of epigenetics and are essential for modulating protein functionality, as well as regulatory cellular processes, including proliferation, differentiation, metabolic pathways, and tumorigenic events. Among these, the small ubiquitin-related modifier (SUMOylation) system is a reversible PTM mechanism that alters target protein interaction surfaces through covalent binding to lysine residues, thereby influencing protein structure and function. Acute myeloid leukemia (AML) is a highly aggressive malignancy characterized by the clonal expansion of primitive hematopoietic stem cells of the myeloid lineage in the bone marrow. Despite recent advancements in therapeutic strategies and an improved understanding of leukemogenic pathways, patient outcomes remain poor, particularly in elderly populations. Consequently, efforts have focused on developing novel agents, including co-targeting specific mutations or integrating targeted therapies into combinatorial chemotherapeutic regimens. Emerging evidence suggests that SUMOylation plays a significant role in AML pathogenesis and treatment response, representing a promising therapeutic target for advanced disease cases. This review provides a brief analysis of the functional role of the SUMOylation system in AML and highlights its potential as a therapeutic target. We also discuss current knowledge gaps and propose directions for future research to advance precision medicine approaches for AML treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/734b8acfb1af3f3bde64169ef0104b00e64873e6" target='_blank'>
              Diagnostic and Therapeutic Implications of the SUMOylation Pathway in Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Elena Chatzikalil, Konstantinos Arvanitakis, Filippos Filippatos, P. Diamantopoulos, T. Koufakis, Elena E Solomou
          </td>
          <td>2025-02-13</td>
          <td>Cancers</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7f6cf05d6f649a55a948cbad9eac9ac4b6467ec" target='_blank'>
              Modern biology of extrachromosomal DNA: A decade-long voyage of discovery
              </a>
            </td>
          <td>
            Qing-Lin Yang, Yipeng Xie, Kailiang Qiao, Jun Yi Stanley Lim, Sihan Wu
          </td>
          <td>2025-01-01</td>
          <td>Cell Research</td>
          <td>3</td>
          <td>2</td>
        </tr>

        <tr id="Abstract High hyperdiploid (HeH) B‐cell acute lymphoblastic leukemia (B‐ALL) is the most prevalent subtype of childhood ALL. This leukemia is characterized by trisomies and tetrasomies of specific chromosomes and additional point mutations. Here, we used single‐cell targeted DNA and antibody sequencing to determine the clonal evolution of HeH B‐ALL during development and chemotherapy treatment. Chromosomal copy number changes were mostly stable over all the leukemia cells, while mutations were typically subclonal. Within all 13 cases, at least one RAS mutant (KRAS or NRAS) subclone was detected (range: 1 to 4 subclones with RAS mutations), indicating the importance of RAS signaling in HeH B‐ALL development. NSD2 mutations were detected in 4 out of 13 cases and always in a subclone with RAS signaling mutations. Single‐cell DNA sequencing detected residual leukemia cells during chemotherapy treatment, and analysis of chromosomal copy number changes aided in the accurate detection of these cells. Our single‐cell data demonstrate that chromosomal changes are acquired prior to additional mutations and that RAS signaling mutations are present in all HeH cases, often as subclonal mutations. This single‐cell multi‐omics study enabled us to extensively characterize the genetic and surface protein heterogeneity in patients with HeH B‐ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc5be0cc4c5cf6db382e785e540535d41eade046" target='_blank'>
              Single‐cell DNA and surface protein characterization of high hyperdiploid acute lymphoblastic leukemia at diagnosis and during treatment
              </a>
            </td>
          <td>
            Margo Aertgeerts, Sarah Meyers, O. Gielen, Jochen Lamote, Barbara Dewaele, M. Tajdar, Johan Maertens, Jolien de Bie, K. De Keersmaecker, N. Boeckx, Lucienne Michaux, A. Uyttebroeck, S. Demeyer, Heidi Segers, Jan Cools
          </td>
          <td>2025-02-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c5a9f2f7d6e562bbd4a865f9dd5dc95d52e44f1" target='_blank'>
              Transformer-based modeling of Clonal Selection and Expression Dynamics reveals resistance mechanisms in breast cancer
              </a>
            </td>
          <td>
            Nathan D Maulding, Jun Zou, Wei Zhou, C. Metcalfe, Josh Stuart, Xin Ye, M. Hafner
          </td>
          <td>2025-01-10</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Tumor heterogeneity is the substrate for tumor evolution and the linchpin of treatment resistance. Cancer cell heterogeneity is largely attributed to distinct genetic changes within each cell population. However, the widespread epigenome repatterning that characterizes most cancers is also highly heterogenous within tumors and could generate cells with diverse identities and malignant features. We show that high levels of the epigenetic regulator and oncogene, UHRF1, in zebrafish hepatocytes rapidly induced methylome disordering, loss of heterochromatin, and DNA damage, resulting in cell cycle arrest, senescence, and acquisition of stemness. Reducing UHRF1 expression transitions these cells from senescent to proliferation-competent. The expansion of these damaged cells results in hepatocellular carcinomas (HCC) that have immature cancer cells intermingled with fibroblasts, immune and senescent cells expressing high UHRF1 levels, which serve as reservoirs for new cancer cells. This defines a distinct and heterogenous HCC subtype resulting from epigenetic changes, stemness and senescence escape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2a7fc1c9934592d995c7390a3ec0bd3e0344392" target='_blank'>
              Epigenetic Disordering Drives Stemness, Senescence Escape and Tumor Heterogeneity
              </a>
            </td>
          <td>
            Elena Magnani, F. Macchi, Tijana Randic, Charlene Chen, Bhavani P. Madakashira, Shashi Ranjan, Sema Elif Eski, S. P. Singh, Kirsten C. Sadler
          </td>
          <td>2024-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02c600bc41b490f1030b42efdd32168810e5ead4" target='_blank'>
              The dynamics of loss of heterozygosity events in genomes
              </a>
            </td>
          <td>
            Abhishek Dutta, J. Schacherer
          </td>
          <td>2025-01-02</td>
          <td>EMBO Reports</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a heterogeneous and aggressive brain tumour that is invariably fatal despite maximal treatment. Genetic or transcriptomic ‘biomarkers’ could be used to stratify patients for treatments, however, pairing biomarkers with appropriate therapeutic ‘targets’ is challenging. Consequently, therapeutics have not yet been optimised for specific GBM patient subsets. Here we integrate genome-wide CRISPR/Cas9 knockout screening and genetic-annotation data for 60 distinct patient-derived, IDHwildtype, adult GBM cell lines, quantifying the essentiality of 15,145 genes. We describe a novel method using Targeted Learning, to estimate the effect size of GBM-relevant biomarkers on context-dependent gene essentiality (GBM-CoDE). We derive multiple target-biomarker pair hypotheses, which we release in an accessible platform to accelerate translation to biomarker-stratified clinical trials. Two of these (WWTR1 with EGFR mutation/amplification, and VRK1 with VRK2 expression suppression) have been validated in GBM, implying that our additional novel findings may be valid. Our method is readily translatable to other cancers of unmet need.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1cf5308b3e1f097593cf8ac72787c4818c5257d" target='_blank'>
              Systematic identification of context-dependent gene-essentiality in Glioblastoma: The GBM-CoDE platform
              </a>
            </td>
          <td>
            Mitchell T Foster, Ailith Ewing, M. Frame, Paul M Brennan, A. Khamseh, S. Beentjes, Neil O. Carragher, Colin A. Semple
          </td>
          <td>2025-01-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/264c6d858496cf48cc3c25b9e46942a18e35bc4d" target='_blank'>
              A compendium of Amplification-Related Gain Of Sensitivity genes in human cancer
              </a>
            </td>
          <td>
            Verónica Rendo, Michael Schubert, Nicholas Khuu, Maria F Suarez Peredo Rodriguez, Declan Whyte, Xiao Ling, A. van den Brink, Kaimeng Huang, Michelle L Swift, Yizhou He, Johanna Zerbib, Ross Smith, J. Raaijmakers, P. Bandopadhayay, Lillian M. Guenther, Justin H Hwang, A. Iniguez, Susan Moody, J-H. Seo, Elizabeth H Stover, Levi A. Garraway, William C. Hahn, K. Stegmaier, René H. Medema, Dipanjan Chowdhury, Maria Colomé-Tatché, Uri Ben-David, R. Beroukhim, F. Foijer
          </td>
          <td>2025-01-27</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>148</td>
        </tr>

        <tr id="Breast cancer is the most prevalent neoplasm in women. ER+ (Luminal subtype), representing over 70% of breast tumors, is a genetically diverse group. Structural and Numerical-Chromosomal instability initiates tumor development and is recognized as the primary driver of genetic alteration in luminal breast tumors. Genomic instability refers to the increased tendency of cancer cells to accumulate genomic alterations during cell proliferation. The cell cycle check-point response to constant and stable genomic alterations in tumor cells drives this process. The impact of CNV patterns and aneuploidies in cell cycle and proliferation perturbation has recently been highlighted by scientists in Luminal breast tumors. The impact of chromosomal instability on cancer therapy and prognosis is not a new concept. Still, the degree of emerging genomic instability leads to prognosis alteration following cell cycle deregulation by chromosomal instability could be predicted by CNVs-based reclassification of breast tumors. In this review, we try to explain the effect of CIN in the cell cycle that ended with genomic instability and altered prognosis and the impact of CIN in decision-making for a therapy strategy for patients with luminal breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06bef9c6cdff104c948a88f734e3a20c0747386c" target='_blank'>
              A new insight into the impact of copy number variations on cell cycle deregulation of luminal-type breast cancer
              </a>
            </td>
          <td>
            Amir Mahdi Khamaneh, Davoud Jafari-Gharabaghlou, Khalil Ansarin, Pouya Pazooki, Zahra Akbarpour, B. Naghili, Nosratollah Zarghami
          </td>
          <td>2025-02-12</td>
          <td>Oncology Reviews</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background: Bladder cancer poses a significant threat to human health. In recent years, genetic circuit therapy has emerged as a novel alternative for precision tumor treatment, demonstrating promising potential for clinical application. Compared to traditional drugs, genetic circuits – typically carried by plasmids – offer advantages such as modularity, druggability, and shorter drug development cycles. These circuits can identify multiple molecular signals from tumors and integrate them through logic gates to specifically target tumor cells. Furthermore, by assembling effector modules, they can induce specific forms of cell death in tumor cells or alter malignant phenotypes, thereby reshaping the immune microenvironment to produce efficient, durable, and controllable antitumor effects. The urinary system serves as an ideal model for this new therapy due to its accessibility from the outside. Several genetic circuits have already been validated for the treatment of bladder cancer. This review outlined the effectiveness and potential value of genetic circuits in bladder cancer therapy. Objective: This review focused on the design principles of genetic circuits, their ability to recognize and convert signals, their therapeutic signal output, and the associated delivery vehicles. We also discussed the challenges and future prospects of genetic circuits as a novel form of “intelligent biotherapy.” Conclusion: The gene circuit can identify multiple signals, processing complex information, and generating multiple effects, thus providing a new approach for personalized treatment of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae601aa52d10ce092dba79e0fef244dcf8c04230" target='_blank'>
              Advancements in genetic circuits as part of intelligent biotherapy for the treatment of bladder cancer: A review
              </a>
            </td>
          <td>
            Chu Liu, Junlin Lu, Jiajian Lai, Kaiwen Jie, Tianxin Lin, Haifeng Wang, Xu Chen
          </td>
          <td>2025-02-04</td>
          <td>Bladder</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Plants host a range of DNA elements capable of self-replication. These molecules, usually associated to the activity of transposable elements or viruses, are found integrated in the genome or in the form of extrachromosomal DNA. The activity of these elements can impact genome plasticity by a variety of mechanisms, including the generation of structural variants, the shuffling of regulatory or coding DNA sequences across the genome, and DNA endoduplication. This plasticity can dynamically alter gene expression and genome stability, ultimately affecting plant development or the response to environmental changes. While the activation of these elements is often considered deleterious to the genome, their role in creating variation is important in adaptation and evolution. Moreover, the mechanisms by which mobile DNA proliferate have been exploited for plant engineering, or contributed to understand how desirable traits can be generated in crops. In this review, we discuss the origins and the roles of mobile DNA element activity on genome plasticity and plant biology, as well as their potential function and current application in plant biotechnology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b762e6df52387f3493832fc69614173777627231" target='_blank'>
              The role of mobile DNA elements in the dynamic of plants genome plasticity.
              </a>
            </td>
          <td>
            Robyn Emmerson, M. Catoni
          </td>
          <td>2025-01-10</td>
          <td>Journal of experimental botany</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="We lack tools to edit DNA sequences at scales necessary to study 99% of the human genome that is noncoding. To address this gap, we applied CRISPR prime editing to insert recombination handles into repetitive sequences, up to 1697 per cell line, which enables generating large-scale deletions, inversions, translocations, and circular DNA. Recombinase induction produced more than 100 stochastic megabase-sized rearrangements in each cell. We tracked these rearrangements over time to measure selection pressures, finding a preference for shorter variants that avoided essential genes. We characterized 29 clones with multiple rearrangements, finding an impact of deletions on expression of genes in the variant but not on nearby genes. This genome-scrambling strategy enables large deletions, sequence relocations, and the insertion of regulatory elements to explore genome dispensability and organization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a90f87137d367e42fa5bf7ac26c12b1be111991" target='_blank'>
              Randomizing the human genome by engineering recombination between repeat elements.
              </a>
            </td>
          <td>
            Jonas Koeppel, Raphael Ferreira, T. Vanderstichele, Lisa M. Riedmayr, Elin Madli Peets, Gareth Girling, Juliane Weller, Pierre Murat, Fabio Giuseppe Liberante, Tom Ellis, George M. Church, Leopold Parts
          </td>
          <td>2025-01-31</td>
          <td>Science</td>
          <td>2</td>
          <td>8</td>
        </tr>

        <tr id="Abstract Achieving targeted hypermutation of specific genomic sequences without affecting other regions remains a key challenge in continuous evolution. To address this, we evolved a T7 RNA polymerase (RNAP) mutant that synthesizes single-stranded DNA (ssDNA) instead of RNA in vivo, while still exclusively recognizing the T7 promoter. By increasing the error rate of the T7 RNAP mutant, it generates mutated ssDNA that recombines with homologous sequences in the genome, leading to targeted genomic hypermutation. This approach, termed T7 RNAP mutant-assisted continuous evolution (T7ACE), functions effectively in both typical prokaryotic and eukaryotic microorganisms (Escherichia coli and Saccharomyces cerevisiae), achieving targeted hypermutations at rates 2800- and 1200-fold higher than the genomic mutation rates, respectively. Using T7ACE, we successfully evolved an eight-fold increase in tigecycline resistance within 7 days and doubled the efficiency of a xylose utilization pathway in 10 days, demonstrating the efficiency and broad applicability of this single-component tool for continuous evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8bff78f49477a4ea5070aa46ecc36f6c1f66021" target='_blank'>
              An evolved, orthogonal ssDNA generator for targeted hypermutation of multiple genomic loci
              </a>
            </td>
          <td>
            Weiran Chu, Rongzhen Tian, Yaxin Guo, Yaokang Wu, Fabian B H Rehm, Long Liu, Jianghua Li, Guocheng Du, Jian Chen, Yanfeng Liu
          </td>
          <td>2025-01-24</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Cutaneous T-cell lymphoma (CTCL) remains a challenging disease due to its significant heterogeneity, therapy resistance, and relentless progression. Multi-omics technologies offer the potential to provide uniquely precise views of disease progression and response to therapy. We present here a comprehensive multi-omics view of CTCL clonal evolution, incorporating exome, whole genome, epigenome, bulk-, single cell (sc) VDJ-, and scRNA-sequencing of 114 clinically annotated serial skin, peripheral blood, and lymph node samples from 35 CTCL patients. We leveraged this extensive dataset to define the molecular underpinnings of CTCL progression in individual patients at single cell resolution with the goal of identifying clinically useful biomarkers and therapeutic targets. Our studies identified a large number of recurrent progression-associated clonal genomic alterations; we highlight mutation of CCR4, PI3K signaling, and PD-1 checkpoint pathways as evasion tactics deployed by malignant T cells. We also identified a gain of function mutation in STAT3 (D661Y) and demonstrated by CUT&RUN-seq that it enhances binding to transcription start sites of genes in Rho GTPase pathways, which we previously reported to have activated chromatin and increased expression in HDACi-resistant CTCL. These data provide further support for a previously unrecognized role for Rho GTPase pathway dysregulation in CTCL progression. A striking number of progression-associated mutations occurred in chromatin methylation modifiers, including EZH2, suggesting that EZH1/2 inhibition may also benefit patients with CTCL. Knowledge of these molecular changes should be leveraged for improved disease monitoring, biomarker-informed clinical trial design, and new therapeutic strategies in this challenging and incurable cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3df1af238c5c0bc411539014f0c361b61ef1f54" target='_blank'>
              Single Cell Resolution Tracking of Cutaneous T-Cell Lymphoma Reveals Clonal Evolution in Disease Progression
              </a>
            </td>
          <td>
            Hannah K. Dorando, Jared M. Andrews, Nicholas Borcherding, Chaz C. Quinn, J. Schmidt, Oam U. Khatavkar, Jahnavi Aluri, Michael T. Harmon, Marcus P. Watkins, Anastasia Frank, Megan Cooper, Amy C. Musiek, Neha Mehta-Shah, Jacqueline E. Payton
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Among the extensive genomic alterations in prostate cancer, the genomic deletion of PTEN stands out as one of the most consistently observed and confirmed alteration. PTEN loss in prostate tumors is primarily associated with cancer cell proliferation and survival through the activation of the PI3K-AKT-mTOR signalling pathway. However, its use as a robust biomarker in the clinical practice is hampered by the complex epigenetic, transcriptional and post-translational regulation. Here, we undertook an approach that combined in situ assessment of PTEN protein with transcriptional surrogates of its activity to gain insights into the downstream functional effects of PTEN loss in primary tumors. Our extensive bioinformatic analyses, including the integration with single-cell RNA-Seq approaches in a new clinical cohort, highlighted stroma remodeling as the major cancer cell-extrinsic process associated with PTEN loss. By applying similar computational strategy on the transcriptomic data generated from primary prostate tumors of genetically engineered Pten knock-out mouse models, we validated the causal role of Pten in the stromal reaction observed in clinical specimens. Mechanistically, we provide evidence for the activation of a paracrine program that encompasses enhanced TGF-β signalling and that is compatible with the secretome of PTEN-deficient senescent cancer cells. Our study provides relevant biological context to the cellular and molecular alterations unleashed upon PTEN protein loss.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c2b98ad80d259dc7a130daab0e45ee7b8d7d990" target='_blank'>
              Transcriptional network analysis of PTEN protein-deficient prostate tumors reveals robust stromal reprogramming and signs of senescent paracrine communication
              </a>
            </td>
          <td>
            Ivana Rondon-Lorefice, Jose I. Lopez, Aitziber Ugalde-Olano, Maite Zufiaurre, I. Astobiza, Saioa Garcia-Longarte, Amaia Zabala, Sofía Rey, Aida Santos-Martín, Miguel Unda, Ana Loizaga-Iriarte, M. Graupera, Paolo Nuciforo, A. Carracedo, Isabel Mendizabal
          </td>
          <td>2025-02-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Mitosis is a critical phase of the cell cycle and a vulnerable point where cancer cells can be effectively disrupted, leading to cell death and inhibition of tumor growth. However, challenges such as drug resistance remain significant in clinical applications. During mitosis, mRNA translation is generally downregulated, while non-canonical translation of specific transcripts proceeds. Here, we demonstrate that mitotic cancer cells redistribute ribosomes toward the 5’ untranslated region (5’ UTR) and the start of the coding sequence (CDS), enhancing the translation of thousands of upstream open reading frames (uORFs) and upstream overlapping open reading frames (uoORFs). This mitotic induction of uORF/uoORF enriches the presentation of immunopeptides at the surface of cancer cells following treatment with mitotic inhibitors. Functional assays indicate the potential of such neoepitopes to provoke cancer-cell killing by T cells. Altogether, our findings highlight the therapeutic potential of targeting uORF/uoORF-derived neoepitopes in combination with mitotic inhibitors to enhance immune recognition and tumor cell elimination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e58b9a8310537bb25bde77518c59ca5cd45e675f" target='_blank'>
              Upstream open reading frame translation enhances immunogenic peptide presentation in mitotically arrested cancer cells
              </a>
            </td>
          <td>
            Alexander Kowar, Jonas P. Becker, Rossella Del Pizzo, Zhiwei Tang, Julien Champagne, Pierré-René Körner, Jasmine Montenegro Navarro, Fiona Megan Tilghman, Hanan Sakeer, Angelika B. Riemer, R. Agami, Fabricio Loayza-Puch
          </td>
          <td>2025-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Polyploidization is a driving force of wheat evolution and speciation, yet its impact on epigenetic regulation and gene expression remains unclear. Here, we constructed a high-resolution epigenetic landscape across leaves, spikes, and roots of hexaploidy wheat and its tetraploid and diploid relatives. Inter-species stable-expression genes exhibited conserved amino acid sequences under strong purifying selection, while dynamic-expression genes were linked to species-specific adaptation. During hexaploidization, dominant D-subgenome homoeolog expression was suppressed via reduced activating epigenetic signals, converging expression with the A and B subgenomes. Proximal chromatin regions near genes were more stable, whereas distal regions, particularly enhancer-like elements mediated by H3K27ac and H3K4me3, exhibit higher dynamism. Sequence variations in these enhancers lead to differential gene regulation, influencing traits such as spike development. For instance, the two haplotypes of dCRE region of TaDEP-B1 resulted in significant differences in its expression and spikelet numbers. We also observed a coevolution of transcription factors and their binding sites, particularly within the expanded ERF family, which regulates spike morphology. This study highlights the interplay between sequence variation and epigenetic modifications in shaping transcriptional regulation during wheat speciation, offering valuable insights for genetic improvement.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3eb492b176e5f2e28bdd73ac3853852b58a5d2c" target='_blank'>
              Reshaping epigenomic landscapes in facilitating the speciation of bread wheat
              </a>
            </td>
          <td>
            Zhaoheng Zhang, Xuelei Lin, Jingjing Yue, Yong-Chang Xu, Lingfeng Miao, Wenqiang Tang, Weilong Guo, Jun Xiao
          </td>
          <td>2024-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="We examine the complex relationship between genomic copy number variation (CNV) and gene expression, highlighting the relevance to cancer biology and other biological contexts. By tracing the history of genometranscriptome correlations, we emphasize the complexity and challenges in understanding these interactions, particularly within the heterogeneous landscape of human cancers. Recent advances in computational algorithms and high-throughput single-cell multi-omic sequencing technologies are discussed, demonstrating their potential to refine our understanding of cancer biology and their limitations. The integration of genomic and transcriptomic analyses, which offers novel insights into tumor evolution and heterogeneity as well as therapeutic strategies, is presented as a crucial approach for advancing cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fea1466a2396cc40442863f3364565e95fd6a58" target='_blank'>
              Exploring genome-transcriptome correlations in cancer.
              </a>
            </td>
          <td>
            M. Ronemus, Daniel Bradford, Zachary Laster, Siran Li
          </td>
          <td>2025-02-05</td>
          <td>Biochemical Society transactions</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Cyclin-dependent kinase 4 and 6 (CDK4/6) inhibitors have transformed the therapeutic landscape for hormone receptor-positive, human epidermal growth factor receptor 2-negative breast cancer, demonstrating significant efficacy in both early and advanced stages of the disease. Combined with endocrine therapy, these inhibitors have dramatically improved survival outcomes. However, resistance to CDK4/6 inhibitors inevitably develops, posing a significant challenge in clinical management. Resistance to CDK4/6 inhibitors can develop through inherent and acquired fashions, and their mechanisms are explored in this comprehensive review. Inherent resistance arises from pre-existing genetic or signaling pathway alterations that diminish cancer cell sensitivity to CDK4/6 inhibitors. Acquired resistance, on the other hand, develops over time through mechanisms, such as the activation of alternative signaling pathways or changes in the tumor microenvironment. The review also examines potential biomarkers for predicting resistance, especially circulating tumor DNA markers, and discusses strategies to overcome resistance. These include combination therapies targeting multiple pathways simultaneously, sequential approaches to delay the onset of resistance, and the development of next-generation CDK4/6 inhibitors with improved efficacy and reduced resistance potential. Understanding resistance mechanisms and developing effective countermeasures are crucial for optimizing patient outcomes and extending the clinical benefits of CDK4/6 inhibitors in cancer therapy. Leveraging these insights will enable clinicians to personalize treatment strategies, ultimately enhancing the long-term effectiveness of CDK4/6 inhibitors in managing breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2dbd50afe53f28765713814f13749bfc2d847b6c" target='_blank'>
              Cyclin-dependent kinase 4/6 inhibitor resistance mechanisms and strategies for subsequent treatment in breast cancer: A comprehensive review
              </a>
            </td>
          <td>
            Yuling Zhang, Bingfeng Chen, Rendong Zhang, Jundong Wu, Chunfa Chen
          </td>
          <td>2025-02-05</td>
          <td>Tumor Discovery</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Oncogenesis and tumor progression are shaped by somatic alterations in the cancer genome and features of the tumor immune microenvironment (TME). How interactions of these two systems influence tumor development and clinical outcomes remains incompletely understood. To address this challenge, we developed the multi-omics analysis framework PACIFIC to systematically integrate genetic cancer drivers and infiltration profiles of immune cells with clinical information. In an analysis of 8500 cancer samples, we report 34 immunogenomic interactions (IGXs) in 13 cancer types in which context-specific combinations of genomic alterations and immune cell activities associate with disease outcomes. Risk associations of IGXs are potentially explained by tumor-intrinsic and microenvironmental metrics of immunogenicity and differential expression of therapeutic targets. In luminal-A breast cancer, MEN1 deletion combined with reduced neutrophils is associated with poor prognosis and deregulation of immune signalling pathways. These findings help elucidate how cancer drivers interact with TME to contribute to tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7541ff011e197e5ec1cc98d2bd31291176ffd633" target='_blank'>
              Combinations of genomic alterations and immune microenvironmental features associate with patient survival in multiple cancer types
              </a>
            </td>
          <td>
            Masroor Bayati, Zoe P. Klein, Alec Bahcheli, Mykhaylo Slobodyanyuk, Jeffrey To, Kevin C. L. Cheng, Jigyansa Mishra, Diogo Pellegrina, Kissy Guevara-Hoyer, Chris McIntosh, Mamatha Bhat, J. Reimand
          </td>
          <td>2024-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="BRAF inhibitor-resistant melanomas (BRAFiR) acquire (epi)genetic and functional alterations that enable them to evade alternative treatments. Identifying these alterations is critical to advancing treatment strategies. Here, we explored the effect of Chk1 inhibition (Chk1i) on BRAFiR cells, revealing higher sensitivity compared to treatment-naïve cells both in vitro and in vivo. Using FUCCI-labeling and time-lapse microscopy, we show that S phase progression is required for Chk1i-induced cytotoxicity in BRAFiR cells, but not in treatment-naïve cells. Replication stress markers, including reduced BrdU incorporation and increased phospho-RPA and γH2AX, were exclusive to BRAFiR cells exposed to Chk1i. Untreated BRAFiR cells exhibited upregulated DNA replication genes, reduced progressing forks and increased origin firing, suggesting intrinsic replication changes. MAPK pathway reactivation in treatment-naïve cells mimicked BRAFiR traits, increasing sensitivity to Chk1i. These findings indicate that Chk1i exploits elevated replication stress specifically in BRAFiR, highlighting its therapeutic potential in overcoming MAPK inhibitor resistance in melanoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1b5d8e2bacae8698be1cc41dc94d4d67c11c52a" target='_blank'>
              Resistance to BRAF inhibitors drives melanoma sensitivity to Chk1 inhibition
              </a>
            </td>
          <td>
            D. G. Carvalho, J. Kenski, Daniel A. Moreira, Matheus A. Rajão, O. Krijgsman, Carolina Furtado, M. Boroni, João P. B. Viola, D. Peeper, P. Possik
          </td>
          <td>2025-01-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Natural Killer (NK) cell therapy has emerged as a transformative approach in cancer immunotherapy, driven by decades of research since their discovery in the 1970s. NK cells possess a unique ability to target tumor cells through antigen-independent recognition, which has been expanded with the discovery of antigen-specific memory and licensing mechanisms. This review explores the evolution of NK cell biology, highlighting key findings such as the enhanced response of NK cells upon re-exposure to cytomegalovirus (CMV), defying conventional immune paradigms. Recent advancements in gene editing, particularly with CRISPR technology, have led to the development of "armored" NK cells. For example, knock-in strategies have enabled the insertion of chimeric antigen receptors (CAR), enhancing tumor specificity, while knock-out approaches have improved NK cell activation by removing inhibitory molecules like PD-1 and NKG2A. Early clinical trials demonstrate improved tumor-targeting efficacy and reduced resistance. However, challenges remain, including high R&D costs, limited clinical data, and technical uncertainties such as off-target effects. In China, NK cell therapy is in its nascent stages, with increasing interest but limited clinical applications due to regulatory and funding barriers. This review provides a comprehensive analysis of NK cell evolution, CRISPR-based advancements, and future directions for overcoming current limitations and optimizing therapeutic outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f6c4d310ebc720bef67a3e653d0a6beb661d459" target='_blank'>
              Historical Evolution and CRISPR-Mediated Gene Editing Applications in NK Cell Therapy
              </a>
            </td>
          <td>
            Ziqing Yan
          </td>
          <td>2025-02-21</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Based on the notion that hypomorphic germline genetic variants are linked to autoimmune diseases, we reasoned that novel targets for cancer immunotherapy might be identified through germline variants associated with greater T-cell infiltration into tumors. Here, we report that while investigating germline polymorphisms associated with a tumor immune gene signature, we identified PKCδ as a candidate. Genetic deletion of PKCδ in mice resulted in improved endogenous antitumor immunity and increased efficacy of anti-PD-L1. Single-cell RNA sequencing revealed myeloid cell expression of Prkcd, and PKCδ deletion caused a shift in macrophage gene expression from an M2-like to an M1-like phenotype. Conditional deletion of PKCδ in myeloid cells recapitulated improved tumor control that was augmented further with anti-PD-L1. Analysis of clinical samples confirmed an association between PRKCD variants and M1/M2 phenotype, as well as between a PKCδ KO-like gene signature and clinical benefit from anti-PD-1. Our results identify PKCδ as a candidate therapeutic target that modulates myeloid cell states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31ff12050b59b63f4c02e53380b59da37e0eb557" target='_blank'>
              PKCδ germline variants and genetic deletion in mice augment antitumor immunity through regulation of myeloid cells.
              </a>
            </td>
          <td>
            Kyle R Cron, A. Sivan, K. Aquino-Michaels, Andrea Ziblat, Emily F. Higgs, R.F. Sweis, Ruxandra Tonea, Seoho Lee, Thomas F Gajewski
          </td>
          <td>2025-01-14</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 226


 Background:
 Metastatic prostate cancer (PC) accumulates significant genomic alterations as the disease progresses, exhibiting the highest levels of chromosomal instability (CIN) among all metastatic tumors (Bakhoum, Nature 2018). Furthermore, tumor suppressor genes (TSG;
 RB1
 ,
 PTEN
 and
 TP53
 ) are frequently altered in metastatic PC and are associated with adverse outcomes. Here, we aim to investigate CIN in metastatic hormone-sensitive PC (mHSPC) harbouring alterations in TSG.
 Methods:
 TSG and whole transcriptome gene expression was assessed by nCounter platform (N=354) and RNA-seq (N=60) in mHPSC samples, respectively. TSG
 low
 was considered when ≥2 out of 3 TSG presented low expression of a previously stablished cut-off, and TSG
 wt
 in the remaining cases. Differential gene expression analysis between TSG
 low
 vs TSG
 wt
 tumors of our RNA-seq cohort (N=60) and TCGA cohort (N=333) was performed. Differential expressed genes (DEGs) were considered if FDR <0.05 and log2FoldChange (LFC)>0.5. DEGs were correlated with castration-resistance PC-free survival (CRPC-FS) and overall survival (OS) by Cox analysis. CIN gene signatures were obtained from literature and the rest of Hallmark signatures from MSigDB.
 Results:
 In our cohort of mHSPC patients treated with androgen deprivation therapy + docetaxel, TSG
 low
 tumors (16.7%) displayed higher levels of CIN compared with TSG
 wt
 (CIN25 normalized enrichment score [NES] 2.13, p<0.001 and CIN70 NES 2.54, p<0001). Cell cycle and DNA-repair signatures were also overexpressed in TSG
 low
 tumors, contrasting with a low expression of androgen receptor pathway. Results were validated in the TCGA cohort, where TSG
 low
 presented also overexpression of CIN signatures. In our RNA-seq cohort, we found 22 DEGs associated with CIN (29.3% of the total DEGs) in TSG
 low
 tumors compared with TSG
 wt
 . 51 genes were significantly correlated with CRPC-FS in a multivariate analysis and 16 of them (31,3%) were CIN-associated genes. Among these genes, 5 encode proteins that are targetable for metastatic PC, with specific inhibitors already developed:
 BUB1
 (LFC 1.06, FDR 0.034),
 CDC7
 (LFC 12.4 FDR 0.003),
 KIFC1
 , (LFC 11,7, FDR 0.01),
 PLK1
 (LFC 1.08, FDR <0.001) and
 WEE1
 (LFC 0.79, FDR 0.011).
 Conclusions:
 TSG-altered mHSPC tumors harbor high levels of CIN, and differentially expressed CIN-associated genes could potentially be targets in mHSPC.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/243ddfb9d9bb3f9a966b17fe378947695a450d86" target='_blank'>
              Chromosomal instability in tumor suppressor genes–altered prostate cancer.
              </a>
            </td>
          <td>
            M. Garcia de Herreros, Natalia Jiménez, J. Padrosa, Òscar Reig, L. Fernández-Mañas, L. Ferrer-Mileo, S. García-Esteve, A. Font Pous, V. Ruiz de Porras, Begoña Mellado
          </td>
          <td>2025-02-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The discovery of oncogenes and tumor suppressor genes led to a better understanding of tumorigenesis, and prompted the development of molecularly targeted therapy. Over the past 30 years, many new drugs, which are primarily aimed at activated oncogenic proteins in signal transduction pathways involved in cell proliferation and survival, have been introduced in the clinic. Despite its rational design, the overall efficacy of targeted therapy has been modest. Recently, the noncoding RNAs (ncRNAs) have emerged as key regulators of important cellular processes in addition to the known regulatory proteins. It now appears that dual epigenetic regulatory systems exist in higher eukaryotic cells: a ncRNA network that governs essential cell functions, like cell fate decision and maintenance of homeostasis, and a protein-based system that presides over core physiological processes, like cell division and genomic maintenance. Modifications of the ncRNA network due to altered ncRNAs can cause the cell to shift towards to neoplastic phenotype; this is cancer initiation. Mutations in the well-known cancer driver genes provide the incipient cancer cell with a selective growth advantage and fuel its consequent clonal expansion. Because of the crucial role of the altered ncRNAs in tumorigenesis, targeting them may be a reasonable therapeutic strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d4bbdeb17c896767d39cab33f2cc2532e16bb98" target='_blank'>
              Implications of noncoding RNAs for cancer therapy: Are we aiming at the right targets?
              </a>
            </td>
          <td>
            Amil Shah
          </td>
          <td>2025-01-16</td>
          <td>Exploration of Targeted Anti-tumor Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="A critical knowledge gap in prostate cancer research is understanding whether castration-tolerant progenitor-like cells that reside in treatment-naïve tumors play a direct role in therapy resistance and tumor progression. Herein, we reveal that the castration tolerance of LSCmed (Lin-, Sca-1+, CD49fmed) progenitor cells, the mouse equivalent of human prostatic Club cells, arises not from intrinsic properties but from significant transcriptional reprogramming. Utilizing single-cell RNA sequencing of LSCmed cells isolated from prostate-specific Pten-deficient (Ptenpc-/-) mice, we identify the emergence of castration-resistant LSCmed cells enriched in stem-like features, driven by the transcription factor Fosl1/AP-1. We demonstrate that cells exhibiting Ptenpc-/- LSCmed characteristics are prevalent in the aggressive mesenchymal stem-like prostate cancer (MSPC) subtype recently identified in human castration-resistant prostate cancer (CRPC). Furthermore, our findings show that the dual-targeting agents JQ-1 and CX-6258—focused on Fosl1/AP-1 and PIM kinases, respectively—effectively suppress both the progenitor properties and the growth of mouse and human MSPC surrogates in vitro and in vivo. Thus, early eradication of castration-tolerant Club-like cells presents a promising therapeutic strategy to mitigate prostate cancer progression toward CRPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7bdb2fa65420e548d3fff241438aeb80083554c5" target='_blank'>
              Targeting Pre-existing Club-Like Cells in Prostate Cancer Potentiates Androgen Deprivation Therapy
              </a>
            </td>
          <td>
            M. Baurès, Anne-Sophie Vieira Aleixo, E. Pacreau, Aysis Koshy, M. Diedisheim, Martin Raigel, Yichao Hua, Charles Dariane, F. Boutillon, Lukas Kenner, J. Marine, Gilles Laverny, Daniel Metzger, Florian Rambow, Jacques-Emmanuel Guidotti, Vincent Goffin
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Rationale: Leukemic evolution occurs through gradual clonal selection at early and late stages of disease development. The early stage of this evolutionary process is called clonal hematopoiesis (CH) and is typified by expansion, and increased self-renewal capacity, of non-malignant, somatically mutated hematopoietic stem and progenitor cells (HSPCs). CH-mutant genes result in either genomic hypomethylation through loss of DNMT3A, or hypermethylation either by genetic loss of TET2 or its’ inhibition by the oncometabolite 2-HG generated by mutant IDH1/IDH2. Leukemic transformation is often associated with subsequent mutagenesis in a broader range of genes including mitogenic signaling genes that confer increased proliferative capacity (FLT3, NRAS). The stepwise acquisition of mutations generates a polyclonal “ecosystem” of clones with increasing clonal diversity indicative of disease progression and poor prognosis. The goal of this study is to 1) identify genetic evolutionary trends associated with distinct CH-related mutations in DNMT3A and TET2 and 2) implement innovative single cell DNA sequencing protocols to identify genotype-selective DNA methylation alterations at single cell resolution. Approach: We compiled patients diagnosed with de novo AML at the University of Pennsylvania that had multiple genomic profiling studies, a cohort comprising 182 patients. All patients were profiled at diagnosis, with n=119 at first remission and n=76 at first relapse. 84.6% of patients received induction chemotherapy with combination anthracycline and nucleoside analog therapy. To investigate the genomic changes in AML disease course, we calculated the change in variant allele frequencies (VAFs) for all identified variants, evaluating matched samples from individual patients across time. We infer phylogenetic relationships from matched genomic profiles across timepoints in individual patients using CALDER. Finally, we performed single cell DNA methylation sequencing on 3 patients (n=4000 cells) using the Single Cell Methylation Kit from Scale Biosciences. This is a combinatorial indexing-based whole genome bisulfite sequencing based approach. Results: Comparing VAFs between diagnosis and relapse for DNMT3Amut vs TET2mut AML, we observed that FLT3 and NPM1 variants were similarly split between increased and decreased VAFs at relapse vs diagnosis. NRAS mutations were both gained and lost in DNMT3Amut samples between diagnosis and relapse, but no NRAS mutations were identified at all in TET2mut samples. When inferring phylogeny associations, we found DNMT3Amut samples were more likely to undergo subclonal swapping from diagnosis to relapse, whereas TET2mut samples relapsed with a more stable mutation profile. Analyses are currently underway for single cell DNA methylation data. Conclusions: Our analyses suggest that DNMT3A and TET2 have distinct sets of co-occurring genetic alterations at diagnosis and relapse. Early CH-associated mutations preceding transformation can shape the evolutionary trajectories of AML throughout disease development.
 Citation Format: Nisargbhai Shah, Ryan Cho, Priya Velu, Selina Luger, Martin Carroll, Jennifer Morrissette, Michael Bowman, Robert Bowman. Evolutionary trajectories and single cell methylation analysis of DNMT3A and TET2 mutant acute myeloid leukemia [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr A021.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da48ac3143bd55d667bb07036faa9f708e803fd9" target='_blank'>
              Abstract A021: Evolutionary trajectories and single cell methylation analysis of DNMT3A and TET2 mutant acute myeloid leukemia
              </a>
            </td>
          <td>
            Nisargbhai Shah, Ryan D Chow, Priya Velu, Selina M Luger, Martin P Carroll, Jennifer J D Morrissette, Michael Bowman, Robert L Bowman
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Prostate cancer is one of the most prevalent malignant tumors in men, particularly in regions with a high Human Development Index. While the long-term survival rate for localized prostate cancer is relatively high, the mortality rate remains significantly elevated once the disease progresses to advanced stages, even with various intensive treatment modalities. The primary obstacle to curing advanced prostate cancer is the absence of comprehensive treatment strategies that effectively target the highly heterogeneous tumors at both genetic and molecular levels. Prostate cancer development is a complex, multigenic, and multistep process that involves numerous gene mutations, alteration in gene expression, and changes in signaling pathways. Key genetic and pathway alterations include the amplification and/or mutation of the androgen receptor, the loss of Rb, PTEN, and p53, the activation of the WNT signaling pathway, and the amplification of the MYC oncogene. This review summarizes the mechanisms by which these genes influence the progression of prostate cancer and highlights the interactions between multiple genes and their relationship with prostate cancer. Additionally, we reviewed the current state of treatments targeting these genes and signaling pathways, providing a comprehensive overview of therapeutic approaches in the context of prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38fab9adcd074d173268fdd423c799e2e23c962b" target='_blank'>
              Interactions between key genes and pathways in prostate cancer progression and therapy resistance
              </a>
            </td>
          <td>
            Fan Wu, Heng Zhang, Miaomiao Hao
          </td>
          <td>2025-01-23</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a278e50ce1c75eb976a4bb58887b3881b23ccb24" target='_blank'>
              Deep CRISPR mutagenesis characterizes the functional diversity of TP53 mutations
              </a>
            </td>
          <td>
            Julianne S Funk, Maria Klimovich, Daniel Drangenstein, Ole Pielhoop, Pascal Hunold, Anna Borowek, Maxim Noeparast, Evangelos Pavlakis, M. Neumann, Dimitrios-Ilias Balourdas, Katharina Kochhan, N. Merle, Imke Bullwinkel, M. Wanzel, Sabrina Elmshäuser, J. Teply-Szymanski, Andrea Nist, Tara Procida, Marek Bartkuhn, Katharina Humpert, M. Mernberger, R. Savai, Thierry Soussi, A. Joerger, Thorsten Stiewe
          </td>
          <td>2025-01-07</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Alternative splicing (AS) is a mechanism that generates translational diversity within a genome. Equally important is the dynamic adaptability of the splicing machinery, which can give preference to one isoform over others encoded by a single gene. These isoform preferences change in response to the cell’s state and function. Particularly significant is the impact of physiological alternative splicing in T lymphocytes, where specific isoforms can enhance or reduce the cells’ reactivity to stimuli. This process makes splicing isoforms defining features of cell states, exemplified by CD45 splice isoforms, which characterize the transition from naïve to memory states. Two developments have accelerated the use of AS dynamics for therapeutic interventions: advancements in long-read RNA sequencing and progress in nucleic acid chemical modifications. Improved oligonucleotide stability has enabled their use in directing splicing to specific sites or modifying sequences to enhance or silence particular splicing events. This review highlights immune regulatory splicing patterns with potential significance for enhancing anticancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfaaff8ba550e599aee86ffb2e27658c31cc7d97" target='_blank'>
              Alternative splicing of modulatory immune receptors in T lymphocytes: a newly identified and targetable mechanism for anticancer immunotherapy
              </a>
            </td>
          <td>
            Shay Tzaban, Ori Stern, Elad Zisman, G. Eisenberg, Shiri Klein, S. Frankenburg, Michal Lotem
          </td>
          <td>2025-01-07</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Most rare diseases (RDs) encompass a diverse group of inherited disorders that affect millions of people worldwide. A significant proportion of these diseases are driven by functional haploinsufficiency, which is caused by pathogenic genetic variants. Currently, most treatments for RDs are limited to symptom management, emphasizing the need for therapies that directly address genetic deficiencies. Recent advancements in gene therapy, particularly with adeno-associated viruses (AAVs) and lipid nanoparticle-encapsulated messenger RNA (mRNA), have introduced promising therapeutic approaches. AAV vectors offer durable gene expression, extensive tissue tropism, and a safety profile that makes them a leading choice for gene delivery; however, limitations remain, including packaging size and immune response. In contrast, mRNA therapeutics, formulated in LNPs, facilitate transient protein expression without the risk of genomic integration, supporting repeated dosing and pharmacokinetic control, though with less long-term expression than AAVs. This review analyzes the latest developments in AAV and mRNA technologies for rare monogenic disorders, focusing on preclinical and clinical outcomes, vector design, and delivery challenges. We also address key regulatory and immunological considerations impacting therapeutic success. Together, these advancements in AAV and mRNA technology underscore a new era in RD treatment, providing innovative tools to target the genetic root of these diseases and expanding therapeutic approaches for patients who currently face limited medical options.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b36772b38705c1dece2a4426665b4ee92f6b8580" target='_blank'>
              Advances in Gene Therapy for Rare Diseases: Targeting Functional Haploinsufficiency Through AAV and mRNA Approaches
              </a>
            </td>
          <td>
            Nuria Bara-Ledesma, Adrian Viteri-Noel, Monica Lopez Rodriguez, Konstantinos Stamatakis, M. Fabregate, Almudena Vazquez-Santos, V. Gomez del Olmo
          </td>
          <td>2025-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Simple Summary KRAS mutations are key drivers of colorectal cancer progression and resistance to treatment, significantly limiting therapeutic options for affected patients. Found in 30–40% of cases, these mutations promote persistent activation of oncogenic pathways such as MAPK/ERK and PI3K/AKT, contributing to tumor growth, poor prognosis, and reduced responsiveness to anti-EGFR therapies. This study aims to elucidate the biological role, clinical significance, and therapeutic potential of targeting KRAS mutations. Recent breakthroughs include the development of targeted inhibitors, such as sotorasib and adagrasib for G12C mutations, and experimental therapies for G12D. However, therapeutic responses in colorectal cancer remain suboptimal compared to other malignancies, largely due to resistance mechanisms and tumor heterogeneity. Promising strategies, including combination therapies, vaccines, and nucleic acid-based treatments, offer hope for improved outcomes. These findings underscore the importance of advancing personalized approaches to enhance care for patients with KRAS-mutant colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2803fdbec8d915a47df49692d27cd1e6edb65e91" target='_blank'>
              The Role of KRAS Mutations in Colorectal Cancer: Biological Insights, Clinical Implications, and Future Therapeutic Perspectives
              </a>
            </td>
          <td>
            Mitsunobu Takeda, Shoma Yoshida, Takuya Inoue, Y. Sekido, Tsuyoshi Hata, Atsushi Hamabe, T. Ogino, N. Miyoshi, Mamoru Uemura, Hirofumi Yamamoto, Y. Doki, H. Eguchi
          </td>
          <td>2025-01-27</td>
          <td>Cancers</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="Background UCK2 (Uridine-Cytidine Kinase 2) is a promising prognostic marker for malignant tumors, but its association with immune infiltration and cancer stemness in pan-cancer remains to be fully understood. we find that gene UCK2 is closed related to RNA stemness scores (RNAss) and DNA stemness scores (DNAss), which is measured the tumor stemness. We also discover an association between UCK2 expression and immune cells by CIBERSORT algorithm, ESTIMATE algorithm and ssGSEA algorithm, especially, related to T cell, monocytes, mast cells, and macrophages. This study aims to shed light on the role and possible mechanism of UCK2 in pan-cancer. Methods We used the R programming language for pan-cancer bulk sequencing data analysis, which were obtained from the University of California, Santa Cruz (UCSC) datasets. UCSC database is a very useful for explore data from TCGA and other cancer genomics datasets, The data we explored at the UCK2 transcriptome level came from TCGA data in the UCSC database. We explored differential UCK2 expression between tumor and normal samples. Immunohistochemistry (IHC) was utilized to validate the expression of UCK2 in different types cancers using tumor tissue chips. The correlations of UCK2 with prognosis, genetic instability, DNA repair, cancer stem cell characteristics, and immune cell infiltration were investigated. Furthermore, single-cell datasets, acquired from the Gene Expression Omnibus (GEO) database, were used to validate the relationship between UCK2 and immune cells. GEO is a famous public genomics database supporting freely disseminates microarray data. Finally, we analyzed the correlation between UCK2 and drug sensitivity. Results UCK2 expression was observed to be high in most cancers and was remarkably related to the prognosis of pan-cancers. We found that the increased UCK2 expression was associated with higher genetic instability. Additionally, positive relationships were observed between UCK2 expression and mismatch repair genes, homologous recombination repair genes, and cancer stemness across different cancer types. There were significant correlations between UCK2 and T cells, monocytes, mast cells, and macrophages. Moreover, as expected, the immune checkpoint human leucocyte antigen (HLA) was found to be negatively related to UCK2. Similarly, UCK2 was also observed to have a negative association with major histocompatibility complex (MHC) genes. We noted that UCK2 had significant correlations with the sensitivity to various anti-cancer drug. Conclusion We have observed that UCK2 plays pivotal roles in prognosis and tumor immunity, and it is associated with DNA repair and cancer stemness. The UCK2 gene exhibits a strong correlation with the immune checkpoints HLA. This study highlights its potential impact on drug sensitivity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f3d615de6acd1c023835e11b2fb04b8e2aecd39" target='_blank'>
              Uridine-cytidine kinase 2 is correlated with immune, DNA damage repair and promotion of cancer stemness in pan-cancer
              </a>
            </td>
          <td>
            Jinlong Tian, Yanlei Li, Yu Tong, Yuan Zhang, Tingxiao Zhao, Yao Kang, Qing Bi
          </td>
          <td>2025-01-27</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="There are no reports of cancer in sponges, despite them having somatic cell turnover, long lifespans, and no specialized adaptive immune cells. In order to investigate whether sponges are cancer resistant, we exposed a species of sponge, Tethya wilhelma, to X-rays. We found that T. wilhelma can withstand 518 Gy of X-ray radiation. That is approximately 100 times the lethal dose for humans. A single high dose of X-rays did not induce cancer in T. wilhelma, providing the first experimental evidence of cancer resistance in the phylum Porifera. Following X-ray exposure, we found an overexpression of genes involved in DNA repair, signaling transduction pathways, and epithelial-to-mesenchymal transition. T. wilhelma has the highest level of radiation resistance that has yet been observed in animals that have sustained somatic cell turnover. This may make them an excellent model system for studying cancer resistance and developing new approaches for cancer prevention and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df59729e500f4d53564b7e1eb8c5ca399205442e" target='_blank'>
              Tethya wilhelma (Porifera) Is Highly Resistant to Radiation Exposure and Possibly Cancer
              </a>
            </td>
          <td>
            Angelo Fortunato, Jake Taylor, Jonathan Scirone, Sareh Seyedi, Athena C Aktipis, Carlo C. Maley
          </td>
          <td>2025-02-07</td>
          <td>Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Phenotypic outcomes can be heavily affected by environmental factors. In this study, we exploited the previously observed nutrient-dependency of cell biological phenotypic features, captured by a cross-condition image-based profiling assay of Escherichia coli deletion strains, to examine this in more detail. We identified several general principles, including the existence of a spectrum of deviating phenotypes across nutrient conditions (i.e., from nutrient- or feature-specific to pleiotropic phenotypic deviations), limited conservation of phenotypic deviations across nutrient conditions (i.e., limited phenotypic robustness), and a subset of nutrient-independent phenotypic deviations (indicative of consistent genetic determinants of specific phenotypic features). In a subsequent step, we used this cross-condition dataset to identify five genes of unknown function of which the deletion displayed either nutrient-independent phenotypic deviations or phenotypic similarities to genes of known function: yibN, yaaY, yfaQ, ybiJ, and yijD. These genes showed different levels of phylogenetic conservation, ranging from conserved across the tree of life (yibN) to only present in some genera of the Enterobacterales (yaaY). Analysis of the structural properties of the proteins encoded by these y-genes, identification of structural similarities to other proteins, and the examination of their subcellular localization yielded new insights into their contribution to E. coli cell morphogenesis, cell cycle progression and cell growth. Together, our approach showcases how bacterial image-based profiling assays and datasets can serve as a gateway to reveal the function of uncharacterized proteins. Importance Despite unprecedented access to genomic information, predicting phenotypes based on genotypes remains notoriously difficult. One major confounding factor is the environment and its ability to modulate phenotypic outcomes. Another is the fact that a large fraction of protein-coding genes in bacterial genomes remains uncharacterized and have no known function. In this work, we use a large-scale cross-condition image-based profiling dataset to characterize nutrient-dependent phenotypic variability of E. coli deletion strains and exploit it to provide insight into the cellular role of genes of unknown function. Through our analysis, we identified five genes of unknown function that we subsequently further characterized by examining their phylogenetic conservation, predicted structural properties and similarities, and their intracellular localization. Combined, this approach highlights the potential of cross-condition image-based profiling, which extracts many cell biological phenotypic readouts across multiple conditions, to better understand nutrient-dependent phenotypic variability and uncover protein function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e64fdb79587408307645dda0d8efcc1410cda8f" target='_blank'>
              Complex interplay between gene deletions and the environment uncovers cellular roles for genes of unknown function in Escherichia coli
              </a>
            </td>
          <td>
            Kaat Sondervorst, Kristina Nesporova, Matthew Herdman, Bart Steemans, Joëlle Rosseels, Sander K. Govers
          </td>
          <td>2025-02-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Recently revised brain tumor classification suggested a glioma treatment strategy that takes into consideration molecular variants in IDH1 and TP53 marker genes. While pathogenic variants of IDH1 and TP53 can be accompanied by chromosomal instability (CIN), the impact of IDH1 and TP53 mutations on genome stability remains unstudied. Elevated CIN might provide therapeutic targets, based on synergistic effects of chemotherapy with CIN-inducing drugs. Using an assay based on human artificial chromosomes, we investigated the impact of common glioma missense mutations in IDH1 and TP53 on chromosome transmission and demonstrated that IDH1R132H and TP53R248Q variants elevate CIN. We next found enhanced CIN levels and the sensitivity of IDH1R132H/WT and TP53R248Q/R248Q genotypes, introduced into U87 MG glioma cells by CRISPR/Cas9, to different drugs, including conventional temozolomide. It was found that U87 MG cells carrying IDH1R132H/WT exhibit dramatic sensitivity to paclitaxel, which was independently confirmed on cell cultures derived from patients with naturally occurring IDH1R132H/WT. Overall, our results suggest that the development of CIN-enhancing therapy for glioma tumors with the IDH1R132H/WT genotype could be advantageous for adjuvant treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/641df7ee46092e60a05501256dd4211c20f30f9f" target='_blank'>
              Therapy enhancing chromosome instability may be advantageous for IDH1R132H/WT gliomas
              </a>
            </td>
          <td>
            Nikolay V Goncharov, Ivan Baklanov, Valeriia S Gulaia, Anastasiia P Shuliak, Daria V Lanskikh, Valeriia Zhmenia, M.E. Shmelev, N. Shved, Jing Wu, Mikhail Liskovykh, Vladimir Larionov, N. Kouprina, Vadim Kumeiko
          </td>
          <td>2025-02-13</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/125e72b0b5ffa707da90fa52c6a8e151fa428862" target='_blank'>
              TRACERx analysis identifies a role for FAT1 in regulating chromosomal instability and whole-genome doubling via Hippo signalling
              </a>
            </td>
          <td>
            Wei-Ting Lu, L. Zalmas, Chris Bailey, James R. M. Black, C. Martínez-Ruiz, O. Pich, F. Gimeno-Valiente, Ieva Usaite, Alastair Magness, K. Thol, Thomas A. Webber, Ming Jiang, R. Saunders, Yun-Hsin Liu, D. Biswas, Esther O. Ige, Birgit Aerne, E. Grönroos, Subramanian Venkatesan, Georgia Stavrou, Takahiro Karasaki, M. Al Bakir, M. Renshaw, Hang Xu, Deborah Schneider-Luftman, Natasha Sharma, Laura Tovini, M. Jamal-Hanjani, S. McClelland, K. Litchfield, Nicolai J. Birkbak, Michael Howell, N. Tapon, K. Fugger, N. Mcgranahan, J. Bartek, N. Kanu, C. Swanton
          </td>
          <td>2024-12-30</td>
          <td>Nature Cell Biology</td>
          <td>1</td>
          <td>69</td>
        </tr>

        <tr id="Simple Summary Triple-negative breast cancer (TNBC) is a challenging-to-treat breast cancer subtype, characterized by high recurrence, increased risk of metastasis, and lower survival for patients compared to other breast cancer subtypes. Depending on the clinical advancement of the disease, surgery, radiotherapy, and chemotherapy remain the standard treatment, though they often have limited long-term efficacy. Chemoresistance in TNBC is closely related to the epithelial–mesenchymal transition (EMT), a process where tumor cells gain mesenchymal-like characteristics. This, in turn, increases their metastatic potential and resistance to standard chemotherapeutic treatments. There is a growing interest in small-molecule inhibitors targeting EMT as a potential strategy to overcome resistance and improve TNBC patient outcomes. This review discusses the TNBC progression and drug resistance within the context of EMT, highlighting molecular features, key EMT protein markers, and signaling pathways. It also explores other mechanisms and factors related to chemoresistance in TNBC with an emphasis on treatment advancements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4c6ca062ee5d60c17f8c0f2fefbe58118ce62bb" target='_blank'>
              Triple-Negative Breast Cancer Progression and Drug Resistance in the Context of Epithelial–Mesenchymal Transition
              </a>
            </td>
          <td>
            E. Błaszczak, Paulina Miziak, Adrian Odrzywolski, M. Baran, Ewelina Gumbarewicz, Andrzej Stepulak
          </td>
          <td>2025-01-01</td>
          <td>Cancers</td>
          <td>1</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e5ba64df098cc1a4c221cfe3104b836d138b7b9" target='_blank'>
              Genome-wide association studies are enriched for interacting genes
              </a>
            </td>
          <td>
            Peter T. Nguyen, Simon G. Coetzee, Irina Silacheva, Dennis J. Hazelett
          </td>
          <td>2025-01-15</td>
          <td>BioData Mining</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a genetically and cellularly heterogeneous disease. We characterized 120 AMLs using genomic and transcriptomic analyses, including single-cell RNA sequencing. Our results reveal an extensive cellular heterogeneity that distorts the bulk transcriptomic profiles. Selective examination of the transcriptional signatures of >90,000 immature AML cells identified four main clusters, thereby extending current genomic classification of AML. Notably, NPM1 mutated AML could be stratified into two novel, clinically relevant classes, with NPM1class I associated with downregulation of MHC class II and excellent survival following hematopoietic stem cell transplantation (HSCT). NPM1class II was instead associated with resistance to allogeneic T cells in an ex vivo co culture assay, and importantly, dismal survival following HSCT. These findings provide new insights into the cellular state space of AML, define new diagnostic entities, and highlight potential therapeutic intervention points. Key Points The bulk transcriptional profiles of AML are mainly driven by a diverse set of cellular signatures. Single cell RNA-sequencing of the most common AML subtypes reveals marked heterogeneity extending beyond current genomic classification schemes. NPM1-mutated AML can be divided into two new classes, with distinct immune evasion mechanisms and survival after transplantation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15ce425737814eabb8f9dbfc7144c4c613998394" target='_blank'>
              The cellular state space of AML unveils novel NPM1 subtypes with distinct clinical outcomes and immune evasion properties
              </a>
            </td>
          <td>
            H. Lilljebjörn, Pablo Peña-Martínez, Hanna Thorsson, Rasmus Henningsson, M. Rissler, N. Landberg, Noelia Puente-Moncada, Sofia von Palffy, Vendela Rissler, Petr Stanek, Jonathan Desponds, Xiangfu Zhong, Gunnar Juliusson, Vladimir Lazarevic, Sören Lehmann, Magnus Fontes, H. Ågerstam, Carl Sandén, Christina Orsmark-Pietras, T. Fioretos
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Mutations in the TP53 gene and chromosomal instability (CIN) are two of the most common alterations in cancer. CIN, marked by changes in chromosome numbers and structure, drives tumor development, but is poorly tolerated in healthy cells, where developmental and tissue homeostasis mechanisms typically eliminate cells with chromosomal abnormalities. Mechanisms that allow cancer cells to acquire and adapt to CIN remain largely unknown. Tumor suppressor protein p53, often referred to as the “guardian of the genome”, plays a critical role in maintaining genomic stability. In cancer, CIN strongly correlates with TP53 mutations, and recent studies suggest that p53 prevents the propagation of cells with abnormal karyotypes arising from mitotic errors. Furthermore, p53 dysfunction is frequent in cells that underwent whole-genome doubling (WGD), a process that facilitates CIN onset, promotes aneuploidy tolerance, and is associated with poor patient prognosis across multiple cancer types. This review summarizes current insights into p53’s role in protecting cells from chromosome copy number alterations and discusses the implications of its dysfunction for the adaption and propagation of cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ee772438cd2bfb17d973a17539b1ffdae477941" target='_blank'>
              The Role of p53 Mutations in Early and Late Response to Mitotic Aberrations
              </a>
            </td>
          <td>
            Anna Hertel, Zuzana Storchová
          </td>
          <td>2025-02-08</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Members of the DNA methyltransferase (DNMT) family have been recognized as major epigenetic regulators of altered gene expression during tumor development. They establish and maintain DNA methylation of the CpG island of promoter and non-CpG region of the genome. The abnormal methylation status of tumor suppressor genes (TSGs) has been associated with tumorigenesis, leading to genomic instability, improper gene silence, and immune evasion. DNMT1 helps preserve methylation patterns during DNA replication, whereas the DNMT3 family is responsible for de novo methylation, creating new methylation patterns. Altered DNA methylation significantly supports tumor growth by changing gene expression patterns. FDA-approved DNMT inhibitors reverse hypermethylation-induced gene repression and improve therapeutic outcomes for cancer. Recent studies indicate that combining DNMT inhibitors with chemotherapies and immunotherapies can have synergistic effects, especially in aggressive metastatic tumors. Improving the treatment schedules, increasing isoform specificity, reducing toxicity, and utilizing genome-wide analyses of CRISPR-based editing to create personalized epigenetic therapies tailored to individual patient needs are promising strategies for enhancing therapeutic outcomes. This review discusses the interaction between DNMT regulators and DNMT1, its binding partners, the connection between DNA methylation and tumors, how these processes contribute to tumor development, and DNMT inhibitors’ advancements and pharmacological properties.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6ccf781a8f63d70125dd8b377423575517766d9" target='_blank'>
              The Role of the DNA Methyltransferase Family and the Therapeutic Potential of DNMT Inhibitors in Tumor Treatment
              </a>
            </td>
          <td>
            Dae Joong Kim
          </td>
          <td>2025-02-05</td>
          <td>Current Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Significant progress in determining the molecular origins and resistance mechanisms of mantle cell lymphoma (MCL) has improved our understanding of the disease's clinical diversity. These factors greatly impact prognosis in MCL patients. Given the dynamic alterations in MCL clones and disease evolution, it is crucial to recognize high-risk prognostic factors at diagnosis and relapse. Clinical factors include a high MCL International Prognostic Index score with a high Ki-67 proliferation index; early disease progression within 24 months of first-line treatment; >3 prior lines of therapy at relapse; and an aggressive (blastoid or pleomorphic) histology. Molecular aberrations include dysregulated cyclin D1; an aberrant SOX11-CD70 axis; upregulated Musashi-2; MYC rearrangement; metabolic reprogramming; and epigenetic changes. Other factors contributing to high-risk MCL include an immune-depleted microenvironment and clone adaptability with complex chromosomal anomalies and somatic mutations in TP53, NSD2, CCND1, CDKN2A, BIRC3, SP140, KMT2D, NFkBIE, SMARCA4, and NOTCH2. Ultra-high-risk MCL is indicated by the coexistence of multiple high-risk prognostic factors in the relapse setting and can portend very short progression-free survival. As MCL treatments advance towards cellular therapies, resistance to anti-CD19 chimeric antigen receptor T-cell therapy is also observed. These findings necessitate revisiting the prognostic impact of high-risk factors, current management strategies, new bi- and tri-specific T-cell engagers, combination therapies, novel therapeutic targets, and next-generation clinical trials for high-risk MCL patients. This article provides a comprehensive update on recognizing and managing high-risk MCL, encompassing current practices and future directions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03364deb8d8c1c44077292ebb1c2031c1c0a3c37" target='_blank'>
              High-Risk MCL: Recognition and Treatment.
              </a>
            </td>
          <td>
            P. Jain, Michael L Wang
          </td>
          <td>2024-12-30</td>
          <td>Blood</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Gene duplication is the process of a gene copied via specific molecular mechanisms to form more duplicate genes. As an important approach to the origination of new genes, gene duplication contributes to around half of the genes in eukaryotic genomes, facilitating the adaptive evolution of species. Over the past fifty years, especially since entering the genomics era in the last two decades, there have been extensive and profound discussions on the mechanisms, evolutionary processes and forces behind the emergence of duplicate genes. Sequence similarity of duplicate genes often leads to functional redundancy, enhancing organismal robustness. Conversely, functional divergence can create novel functions and improve evolvability. In this review, we summarize the mechanism of gene duplication, the fate and the evolutionary models of duplicate genes. This article concludes by outlining how long-read sequencing technologies, gene editing, and various other high-throughput techniques will further advance our understanding of the role of duplicate genes in the genetics-development-evolution network.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11a8968d2fd67447569fbf8e6c7852ab9321d0f2" target='_blank'>
              Evolution by gene duplication: in the era of genomics.
              </a>
            </td>
          <td>
            Jie-Yu Shen, Tian-Han Su, Da-Qi Yu, Sheng-Jun Tan, Yong-E Zhang
          </td>
          <td>2025-02-01</td>
          <td>Yi chuan = Hereditas</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2fc69ff58eaa4bfa3b4fa917eb7dfeb9e6159b0" target='_blank'>
              Genomic and phenotypic stability of fusion-driven pediatric sarcoma cell lines
              </a>
            </td>
          <td>
            Merve Kasan, Florian H. Geyer, Jana Siebenlist, M. Sill, R. Öllinger, Tobias Faehling, Enrique de Álava, D. Surdez, U. Dirksen, Ina Oehme, Katia Scotlandi, O. Delattre, Martina Müller-Nurasyid, R. Rad, K. Strauch, T. G. Grünewald, F. Cidre-Aranaz
          </td>
          <td>2025-01-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa72f1de2d7ad2ee27bb41187db8bcfae6ef0577" target='_blank'>
              The integrative genomic and functional immunological analyses of colorectal cancer initiating cells to modulate stemness properties and the susceptibility to immune responses
              </a>
            </td>
          <td>
            I. Tout, Salim Bougarn, Mohammed Toufiq, Neha Gopinath, Ola Hussein, Abbirami Sathappan, E. Chin-Smith, Fazulur Rehaman, Rebecca Mathew, L. Mathew, Kun Wang, Li Liu, Abdulrahman Salhab, Oleksandr Soloviov, Sara Tomei, W. Hasan, Sahar I. Da’as, Yosra Bejaoui, Nady El Hajj, Karama Makni Maalej, Said Dermime, K. Rasul, P. Dellabona, G. Casorati, A. Turdo, M. Todaro, Giorgio Stassi, S. Ferrone, Xinhui Wang, Cristina Maccalli
          </td>
          <td>2025-02-17</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Constitutively active mutations of KRAS are prevalent in non-small cell lung cancer (NSCLC). However, the relationship between these mutations and resistance to platinum-based chemotherapy and the underlying mechanisms remain elusive. In this study, we demonstrated that KRAS mutants confer resistance to platinum in NSCLC. Mechanistically, KRAS mutants mediate platinum resistance in NSCLC cells by activating ERK/JNK signaling, which inhibits ALKBH5 m6A demethylase activity by regulating post-translational modifications (PTMs) of ALKBH5. Consequently, the KRAS mutant leads to a global increase in m6A methylation of mRNAs, particularly DDB2 and XPC, which are essential for nucleotide excision repair. This methylation stabilized the mRNA of these two genes, thus enhancing NSCLC cells' ability to repair platinum-induced DNA damage and avoid apoptosis, thereby contributing to drug resistance. Furthermore, blocking KRAS-mutant-induced m6A methylation, either by overexpressing a SUMOylation-deficient mutant of ALKBH5, or by inhibiting METTL3 pharmacologically, significantly sensitizes KRAS-mutant NSCLC cells to platinum drugs in vitro and in vivo. Collectively, our study uncovers a previously unrecognized mechanism that mediates KRAS mutant-induced chemoresistance in NSCLC cells by activating DNA repair through the modulation of the ERK/JNK/ALKBH5 PTMs-induced m6A modification in DNA damage repair-related genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b30cd92ed6de2a868acc5ae23a99dadaae25a64" target='_blank'>
              KRAS Mutants Confer Platinum Resistance by Regulating ALKBH5 Post-translational Modifications in Lung Cancer.
              </a>
            </td>
          <td>
            Fang Yu, Shikan Zheng, Chunjie Yu, Sanhui Gao, Zuqi Shen, Rukiye Nar, Zhexin Liu, Shuang Huang, Lizi Wu, Tongjun Gu, Zhijian Qian
          </td>
          <td>2025-02-04</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Purpose: Pancreatic cancer (PC) is an aggressive lethal tumor with an unmet need for novel therapeutic approaches. KRAS activating mutations occur in 90-95% of PC and contribute to tumor progression and resistance to therapy, including radiation. The mechanisms by which oncogenic KRAS promotes radiation resistance are critical to understand in order to identify novel therapies. Methods: We first analyzed the expression levels of DNA damage response and repair genes using Affymetrix RNA expression microarray in isogenic HCT116 and SW48 cells with KRAS wide-type and KRASG13D activating mutations to identify novel targets by which KRAS mutations may confer radiation resistance. We analyzed the expression of STN1 in pancreas normal and cancer tissues and assessed the correlation with PC clinical outcomes using TCGA dataset. Human tumor xenografts were generated to explore the role of STN1 on tumor growth in vivo. Radiation response was assessed through clonogenicity and gH2AX foci assays. Homologous recombination (HR) and non-homologous end joining (NHEJ) repair reporter assays, chromatin spreading assay, cell cycle analysis, mitotic catastrophe, Annexin-V assays were performed to investigate the mechanisms of radiation-induced cell death. Mass spectrometry analysis was performed to identify STN1 interacting proteins important in DNA damage response and further validated by immunoprecipitation and immunoblotting. Results: We find that KRAS activation increases STN1 expression to enhance DNA double strand break repair capacity in PC. STN1 is a component of the CST complex normally important for telomere duplication and maintenance. We find that STN1 is significantly upregulated in PC, especially in aggressive subtypes of PC, associates with KRAS oncogenic mutations, and correlates with poor patient clinical outcomes. Genetic silencing or pharmacologic inhibition of KRAS signaling decreases STN1 expression in PC cells, suggesting KRAS signaling positively regulates STN1 expression. Interestingly, STN1 depletion reduces tumor growth in a heterotopic model of KRAS mutant PC. Mechanistically, depletion of STN1 potentiates DNA damage, replication stress, and sensitizes PC cells to ionizing radiation independent of CTC1 and TEN1. In support of this finding, STN1 silencing reduces both HR and NHEJ repair of DSBs. Furthermore, knockdown of STN1 impairs cell cycle arrest at the G2/M phase in response to ionizing radiation, which is accompanied with increased mitotic catastrophe, radiation-induced apoptosis. Proteomic analysis reveals that STN1 physically interacts with many proteins important for DNA repair, replication and cell cycle progression, including ATM, DICER1, CEP164, and CEP250. Conclusion: Our findings have revealed a novel, potentially CST complex-independent role of STN1 in DSB repair after radiation. STN1 may function at one of the apical nodes in the DNA damage response pathway by interacting with ATM. Our findings suggest STN1 may be a promising target for improving genotoxic therapies in KRAS mutant cancers, including PC.
 Citation Format: Tiantian Cui, Changxian Shen, Linlin Yang, Ling Gui, Sergio Corrales-Guerrero, Sindhu Nair, Joanna M. Karasinska, James T. Topham, Xiaoli Ping, Jeremy M. Stark, Terence M. Williams.STN1 (OBFC1) promotes DNA double-strand break repair in a potentially CTC1-STN1-TEN1 (CST) complex-independent role in pancreatic cancer.[abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr P011">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ce1e95ed7e839e47ffa944834be9b3c39c5d903" target='_blank'>
              Abstract P011: STN1 (OBFC1) promotes DNA double-strand break repair in a potentially CTC1-STN1-TEN1 (CST) complex-independent role in pancreatic cancer
              </a>
            </td>
          <td>
            Tiantian Cui, Changxian Shen, Linlin Yang, Ling Gui, S. Corrales-Guerrero, Sindhu Nair, J. Karasinska, J. Topham, Xiaoli Ping, Jeremy M. Stark, Terence M. Williams
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Epigenetic modifications, which reversibly regulate gene expression without altering the DNA sequence, are increasingly described in the literature as essential elements in the processes leading to cancer development. SETDB1 regulates histone 3 (H3) K9 di- and trimethylation, promoting heterochromatin formation, and plays a key role in gene silencing. Epigenetic deregulation of SETDB1 expression appears to be involved in different cancers types, particularly in aggressive, relapsing or treatment-resistant subtypes. Despite advances in research, the full range of mechanisms through which this protein acts remains unclear; however, it is evident that SETDB1 has a pivotal role, particularly in the mesenchymal stem cells differentiation, tumor evasion and treatment resistance. Its role in genetically complex sarcomas, such as osteosarcoma, has not been fully explored, although recent Omics analyses suggest its presence and amplification in osteosarcoma. Given its involvement in osteoblastogenesis and adipogenesis, we discuss the potential of SETDB1 as a key target for new therapeutic strategies in osteosarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b378d2be6324f27ab95439dedf132da8d385c12" target='_blank'>
              SETDB1 amplification in osteosarcomas: Insights from its role in healthy tissues and other cancer types
              </a>
            </td>
          <td>
            Elodie Verdier, Nathalie Gaspar, Maria Eugénia Marques da Costa, A. Marchais
          </td>
          <td>2025-02-12</td>
          <td>Oncotarget</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acca60e9e5684fac9b42ec113ffc78d1e571a20d" target='_blank'>
              Pan-inhibition of super-enhancer-driven oncogenic transcription by next-generation synthetic ecteinascidins yields potent anti-cancer activity.
              </a>
            </td>
          <td>
            Max Cigrang, Julian Obid, Maguelone Nogaret, Léane Seno, Tao Ye, G. Davidson, Philippe Catez, P. Berico, Clara Capelli, Clara Marechal, Amélie Zachayus, Clémence Elly, Marie Jose Guillen Navarro, Marta Martínez Diez, Gema Santamaría Nuñez, Tsai-Kun Li, E. Compe, Pablo Aviles, Irwin Davidson, J. Egly, Carmen Cuevas, F. Coin
          </td>
          <td>2025-01-08</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Prostate cancer is the most common cancer among men worldwide, especially in those over 65, and is a leading cause of cancer-related mortality. The disease typically advances from an androgen-dependent state to castration-resistant prostate cancer (CRPC), which poses significant treatment challenges. The androgen receptor (AR) on the X chromosome is a central driver in this process, activating genes that govern proliferation and survival. Mutations and amplifications of the AR are closely associated with disease progression and treatment resistance. While traditional therapies such as androgen deprivation therapy (ADT) and AR antagonists like enzalutamide have been effective, resistance persists due to reactivation of AR signaling through mechanisms like ligand-independent activation. Recent research highlights the role of epigenetic modifications in enhancing AR activity and drug resistance. The tumor microenvironment, particularly interactions with cancer-associated fibroblasts (CAFs) and tumor-associated macrophages (TAMs), further complicates treatment by promoting aggressive tumor behavior and immune evasion. Future directions include developing next-generation AR antagonists, identifying AR-related biomarkers for personalized therapy, and exploring combinations with immune checkpoint inhibitors. Additionally, basal cell-lumen-derived organoids provide innovative models that can enhance understanding and treatment strategies in prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76cc795daea1ab497c79508ac8d30990e3cf3c4a" target='_blank'>
              Androgen receptor dynamics in prostate cancer: from disease progression to treatment resistance
              </a>
            </td>
          <td>
            Caihong Li, Dongkai Cheng, Pengya Li
          </td>
          <td>2025-02-11</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Colorectal cancer poses a major global health issue, profoundly affecting both mortality and morbidity rates across the world. A key obstacle in understanding the pathogenesis of colorectal cancer lies in its high inter-patient and spatial heterogeneity, making standard treatments ineffective. Commonly, the study on colorectal cancer relies on bulk RNA sequencing, offering an average gene expression profile for a heterogenous cell population. However, this approach obscures the heterogeneous characteristics of the cancer cells and hinders a comprehensive understanding of the complex interplay among different cell populations. Recently, the advent of single-cell RNA sequencing has been revolutionary, enabling researchers to analyze individual immune cells and overcome the limitations of bulk RNA sequencing. Through single-cell RNA sequencing, researchers have gained insights into the dynamic nature of the immune response to cancer and potential targets for immunotherapy. In this review, we discuss the technical aspects of single-cell RNA sequencing, the application of single-cell RNA sequencing in cancer immunology, and the potential of single-cell RNA in clinical settings. We believe that harnessing singlecell RNA sequencing in cancer research holds immense potential to drive the development of personalized immunotherapies, aiming to improve patient outcomes in colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddac82763361480546b876aad89a46e0a8e233fa" target='_blank'>
              Single Cell RNA Sequencing in Colorectal Cancer Immunology: Recent Updates, Application, and Emerging Challenges.
              </a>
            </td>
          <td>
            Sabrina George, Nor Adzimah Johdi
          </td>
          <td>2025-02-12</td>
          <td>Current molecular medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Chimeric antigen receptor T (CART) cell therapy is an innovative form of targeted immunotherapy that has revolutionized the treatment of cancer. However, the durable response remains limited. Recent studies have shown that the epigenetic landscape of preinfusion CART cell products can influence response to therapy, and gene editing has been proposed as a solution. However, more work needs to be done to determine the optimal gene editing strategy. Genome-wide CRISPR screens have become popular tools to both investigate mechanisms of resistance and optimize gene editing strategies. Yet their application to primary cells presents many challenges. Here we describe a method to complete a genome-wide CRISPR knockout screen in CART cells from healthy donors. As a proof-of-concept model, we designed this method to investigate the development of exhaustion in CART cells targeting the CD19 antigen. However, we believe that this approach can be used to address a variety of mechanisms of resistance to therapy in different CAR constructs and tumor models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/995da50c8649d8341a6a265386cac02363ea1e3b" target='_blank'>
              Performing an In Vitro Genome-Wide CRISPR Knockout Screen in Chimeric Antigen Receptor T Cells.
              </a>
            </td>
          <td>
            Carli M. Stewart, Claudia Manriquez Roman, Saad S. Kenderian
          </td>
          <td>2025-01-31</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Retrotransposons, multi-copy sequences that propagate via copy-and-paste mechanisms involving an RNA intermediate, occupy large portions of all eukaryotic genomes. A great majority of their manifold copies remain silenced in somatic cells, nevertheless, some are transcribed, often in a tissue specific manner, and a small fraction retains its ability to mobilize. Retrotransposon expression or mobility are increasingly recognized to contribute to normal development and tissue homeostasis, as well as to aging and disease. While it is well characterized that retrotransposon sequences may provide cis regulatory elements for neighboring genes, how their own expression and mobility are achieved in different somatic contexts is not well understood. Here, using long-read DNA sequencing, we characterize somatic retrotransposition in the Drosophila intestine. We show that retroelement mobility does not change significantly upon aging and is limited to very few active sub-families of retrotransposons. Importantly, we identify a polymorphic donor locus of an endogenous LTR retroviral element rover, active in the intestinal tissue. We reveal that gut activity of the rover donor copy depends on its genomic environment. Without affecting local gene expression, the copy co-opts its upstream enhancer sequence, rich in transcription factor binding sites, for somatic expression. Further we show that escargot, a snail-type transcription factor critical for gut progenitor cell function, can drive transcriptional activity of the active rover copy. These data provide new insights into how locus-specific features allow active retrotransposons to produce functional transcripts and mobilize in a somatic lineage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1e2218d35b486ab25b581d00df030a25e3f5364" target='_blank'>
              An endogenous retroviral element co-opts an upstream regulatory sequence to achieve somatic expression and mobility
              </a>
            </td>
          <td>
            Natalia Rubanova, Darshika Singh, Louis Barolle, Fabienne Chalvet, Sophie Netter, Mickael Poidevin, Nicolas Servant, Allison J. Bardin, Katarzyna Siudeja
          </td>
          <td>2025-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Originating from, but independent of, linear chromosomes, extrachromosomal DNA (ecDNA) exists in a more active state of transcription and autonomous replication. It plays a crucial role in the development of malignancies and therapy resistance. Since its discovery in eukaryotic cells more than half a century ago, the biological characteristics and functions of ecDNA have remained unclear due to limitations in detection methods. However, recent advancements in research tools have transformed ecDNA research. It is believed that ecDNA exhibits greater activity in the abnormal amplification of oncogenes, thereby driving cancer progression through their overexpression. Notably, compared to linear DNA, ecDNA can also function as a genomic element with regulatory roles, including both trans- and cis-acting functions. Its critical roles in tumorigenesis, evolution, progression, and drug resistance in malignant tumors are increasingly recognized. This review provides a comprehensive summary of the evolutionary context of ecDNA and highlights significant progress in understanding its biological functions and potential applications as a therapeutic target in malignant tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdf3896d81e701493b4d97e132a04a0cb6710489" target='_blank'>
              Insights into Extrachromosomal DNA in Cancer: Biogenesis, Methodologies, Functions, and Therapeutic Potential.
              </a>
            </td>
          <td>
            Xudong Mao, Guocheng Rao, Gonghui Li, Shihan Chen
          </td>
          <td>2025-02-13</td>
          <td>Advanced biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="High fidelity of replication is important to preserve genomic integrity and ensure healthy progeny. Perturbations of replication, also known as replication stress, is frequently observed in cancer cells and is considered a cancer cell-specific trait. Although replication stress drives genomic instability and tumor progression, it also generates a targetable cancer-specific vulnerability. In order to identify potential therapeutic targets in cancer cells that experience replication stress, we performed a genome wide genetic screen in human HAP1 cells challenged with low doses of replication stress-inducing drugs. We identified a large set of genes that specifically hamper cell survival in the context of replication stress. In addition to well-known players in the replication stress response and DNA repair, such as RNASEH1, BRIP1, and MDC1, we identified several genes with no prior described role in DNA replication, damage tolerance or repair. We validated that the loss of GIGYF2, HNRNPA2B1, and SUMO2 renders cells more vulnerable to replication stress. For GIGYF2 and SUMO2, we could implicate a role in homologous recombination. Taken together, our replication stress screen identified several known as well as some novel factors that protect against the toxic implications of replication stress. These factors could entail potential therapeutic targets for cancer cells experiencing replication stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddea308b1694526a4e6fc1d9af3b1fa983ba1e3c" target='_blank'>
              Identification of critical factors of the replication stress response in human cells
              </a>
            </td>
          <td>
            L. M. Janssen, Empar Baltasar Perez, C. Vaarting, Abdelghani Mazouzi, M. Raaben, T. Brummelkamp, René H. Medema, J. Raaijmakers
          </td>
          <td>2025-01-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3ef97a03979c9e266e81720f51438d4d5cd7b23" target='_blank'>
              BRCA1 is involved in sustaining rapid antler growth possibly via balancing of the p53/endoplasmic reticulum stress signaling pathway
              </a>
            </td>
          <td>
            Qianqian Guo, Zhen Wang, Jiping Li, Chao Ma, Junjun Zheng, H. Ba, Guokun Zhang, Chunyi Li
          </td>
          <td>2025-01-24</td>
          <td>Biology Direct</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Simple Summary Melanoma is an aggressive malignancy defined by significant intratumoral heterogeneity, driving its capacity for therapeutic resistance and recurrence. This study adopts a systems-level approach to dissect the melanoma microenvironment, focusing on intricate interactions between malignant cells and immune infiltrates. We identified critical regulatory networks and intercellular communication pathways that appear to influence disease progression. These findings highlight the dynamic interplay between tumor-intrinsic factors and the surrounding microenvironment, revealing potential mechanisms underlying immune evasion and therapy resistance. By mapping these complex interactions, the present study builds on our foundation for precision-based therapeutic strategies tailored to the unique biological landscape of melanoma, offering promise for improved clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f383353b5d7dd45f21551edcc7dd626f8651941" target='_blank'>
              Integrated Analysis of Single-Cell and Bulk RNA Data Reveals Complexity and Significance of the Melanoma Interactome
              </a>
            </td>
          <td>
            Michael J Diaz, J. T. Tran, Arthur M. Samia, M. Forouzandeh, Jane M. Grant-Kels, Marjorie E. Montanez-Wiscovich
          </td>
          <td>2025-01-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05abf56ac7febcd59ec911620962ed978222a49b" target='_blank'>
              Copy number amplification of FLAD1 promotes the progression of triple-negative breast cancer through lipid metabolism
              </a>
            </td>
          <td>
            Xiao-Qing Song, Tian-Jian Yu, Ou-Yang Yang, Jianlong Ding, Yizhou Jiang, Zhi-Ming Shao, Yi Xiao
          </td>
          <td>2025-02-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Deletion of the long q arm of chromosome 22 (22qDEL) is the most frequently identified recurrent somatic copy number alteration observed in papillary thyroid carcinoma (PTC). Since its role in PTC is not fully understood, we conducted a pooled analysis of genomic characteristics and clinical correlates in 1094 primary tumors from four published PTC genomic studies. The majority of PTC cases with 22qDEL exhibited arm-level loss of heterozygosity (86%); nearly all PTC cases with 22qDEL had losses in 22q12 and 13, which together constitute 70% of the q arm. Our analysis confirmed that 22qDEL occurs more frequently with RAS point mutations (50.4%), particularly HRAS (70.3%), compared with other PTC drivers (9.3%), supporting the conclusion that 22qDEL is unlikely to be a solitary driver of PTC but possibly an important co-factor in carcinogenesis, particularly in PTCs with RAS driver mutations. Differential RNA expression analyses revealed downregulation of most genes located on chromosome 22 in cases with 22qDEL compared to those without 22qDEL. Many differentially expressed genes are drawn from immune response and regulation pathways. These findings highlight the value of further investigations into the contributions of 22qDEL events to PTC, perhaps mediated through immune perturbations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97e0fd29e42888016677b1946cca20fdf0013b09" target='_blank'>
              Somatic copy number deletion of chromosome 22q in papillary thyroid carcinoma
              </a>
            </td>
          <td>
            Olivia W Lee, D. Karyadi, Stephen W Hartley, Weyin Zhou, M. Machiela, Shahriar A. Zamani, L. Zurnadzhy, John N Weinstein, Young Joo Park, Jeong-Sun Seo, Gerry A Thomas, Tetiana I Bogdanova, M. Tronko, Lindsay M. Morton, S. Chanock
          </td>
          <td>2025-01-01</td>
          <td>European Thyroid Journal</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Post-translational modifications (PTMs) of proteins refer to the process of adding chemical groups, sugars, or other molecules to specific residues of target proteins following their biosynthesis by ribosomes. PTMs play a crucial role in processes such as signal transduction, epigenetics, and disease development. Lactylation is a newly discovered PTM that, due to its close association with lactate—the end product of glycolytic metabolism—provides a new perspective on the connection between cellular metabolic reprogramming and epigenetic regulation. Studies have demonstrated that lactylation plays a significant role in tumor progression and is associated with poor clinical prognosis. Abnormal histone lactylation can influence gene expression in both tumor cells and immune cells, thereby regulating tumor progression and immunosuppression. Lactylation of non-histone proteins can also modulate processes such as tumor proliferation and drug resistance. This review summarizes the latest research progress in the field of lactylation, highlighting its roles and mechanisms in tumorigenesis, tumor development, the tumor microenvironment, and immunosuppression. It also explores the potential application value of lactylation in tumor-targeted therapy and combined immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfbb1c836619ec3b777921f3932ca667930b58e9" target='_blank'>
              Regulatory role and therapeutic prospect of lactate modification in cancer
              </a>
            </td>
          <td>
            Mengdi Sun, Kejing Wang, Fang Lu, Donghua Yu, Shumin Liu
          </td>
          <td>2025-02-17</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="EGFR tyrosine kinase inhibitors (EGFR-TKIs) have garnered substantial clinical success in treating nonsmall cell lung cancer (NSCLC) harboring epidermal growth factor receptor (EGFR) mutations. However, the inevitable emergence of drug resistance, frequently attributed to activation, mutation, or deletion of multiple signaling pathways, poses a significant challenge. Notably, the loss of PTEN protein expression has emerged as a pivotal mechanism fostering resistance in EGFR mutant lung cancers. Consequently, strategies aimed at upregulating PTEN expression hold great promise for restoring drug sensitivity. Leveraging the versatility, precision, and efficacy of nuclease-deactivated Cas9 (dCas9) as a transcriptional activator, we designed a CRISPR-dCas9 system to stimulate PTEN expression. To further enhance target specificity and drug delivery efficiency, we innovatively harnessed the tumor cell membrane (CCM) as a homologous targeting surface coating for our vector, thereby creating a targeted activation nanoplatform. Comprehensive in vitro and in vivo evaluations demonstrated that the synergistic interplay between gefitinib and the CRISPR-dCas9 system significantly enhanced drug sensitivity. The finding underscores the potential of our approach in addressing the issue of lung cancer resistance, offering a promising avenue for personalized and effective cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f86d44b9e09159a099ee094747765812cf22c69" target='_blank'>
              CRISPR-dCas9-Mediated PTEN Activation via Tumor Cell Membrane-Coated Nanoplatform Enhances Sensitivity to Tyrosine Kinase Inhibitors in Nonsmall Cell Lung Cancer.
              </a>
            </td>
          <td>
            Kaiyong Yang, Chunli Zhang, Zeyu Wang, Qiqing Huang, Jing Qian, Gaoyu Shi, Wenwen Sun, Jinqiu Wang, Yu Ji, Zhaorui Sun, Yanni Song, Xin Han
          </td>
          <td>2025-02-20</td>
          <td>ACS applied materials & interfaces</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4d5925d1e02fdeb24a29a2a8e41aed5142d614d" target='_blank'>
              Advances in A-to-I RNA editing in cancer
              </a>
            </td>
          <td>
            Yi Zhang, Lvyuan Li, Juana Jessica Mendoza, Dan Wang, Qijia Yan, Lei Shi, Zhaojian Gong, Zhaoyang Zeng, Pan Chen, Wei Xiong
          </td>
          <td>2024-12-27</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Molecular targeted drugs are frequently used in precision cancer therapies. However, the prolonged administration of these agents can result in drug resistance. Cancer cells exhibiting defects in homologous recombination are particularly susceptible to genotoxic stress, such as that induced by PARP inhibitors (PARPi), resulting in synthetic lethality. Here, we show that sustained treatment of such cancer cells with PARPi selectively upregulates YAP1-2α, a minor isoform of YAP1. Elevated YAP1-2α heterodimerizes with TAZ to induce liquid-liquid phase separation (LLPS) in the nucleus. This LLPS-driven process generates nuclear condensates that activate a super-enhancer comprising YAP1-2α, TAZ, TEAD, and BRD4. The super-enhancer reconfigures transcriptional networks to enhance cancer stem-like malignant properties, thereby potentiating cell resilience and augmenting cell resistance to adverse conditions, including precision cancer therapies. Our study reveals the presence of a super-enhancer that promotes malignant tumor progression. Targeting the YAP1-2α/TAZ-induced super-enhancer is a promising strategy for overcoming resistance to anticancer modalities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a0a7f11bb9d0a8943e8f9ed992501df776b3e17" target='_blank'>
              A super-enhancer actuated by heterodimerization of a specific YAP1 isoform, YAP1-2α, with TAZ endows cancer stemness and drug resistance
              </a>
            </td>
          <td>
            Takuya Ooki, Takeru Hayashi, Yui Yamamoto, Masanori Hatakeyama
          </td>
          <td>2025-01-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/067a6a45cdd25069d169d43fc6989270e4c626cb" target='_blank'>
              SMARCB1-driven EGFR-GLI1 epigenetic alterations in lung cancer progression and therapy are differentially modulated by MEOX2 and GLI-1.
              </a>
            </td>
          <td>
            Octavio A Trejo-Villegas, Priscila Pineda-Villegas, Leonel Armas-López, C. Mendoza-Milla, I. Peralta-Arrieta, Oscar Arrieta, I. Heijink, Joaquín Zúñiga, F. Ávila-Moreno
          </td>
          <td>2025-02-19</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74d077c0e04d5c20a662dd5e8ac2b16b1dbaf335" target='_blank'>
              Integrative pan-cancer genomic analysis highlights mitochondrial protein p32 as a potential therapeutic target in Myc-driven tumorigenesis.
              </a>
            </td>
          <td>
            Qiufen Bi, Jun Nie, Qiang Wu, Liang Sun, Shuang Zhu, Jin Bai, Yong Liu, Fang Huang, Keli Chai
          </td>
          <td>2025-02-01</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7c10b0a74142199e8c7ea18f31540a06dfe3ffe" target='_blank'>
              Aneuploidy-driven gene expression profiling in human blastocysts: insights from RNA-Seq analysis.
              </a>
            </td>
          <td>
            M. Bazrgar, F. Khajehoseini, P. Eftekhari-Yazdi, M. Bakhtiarizadeh, H. Gourabi, Parishad Saei, R. Pazhoomand, S. Hosseinishenatal, R. Mohammadi
          </td>
          <td>2025-01-03</td>
          <td>Journal of assisted reproduction and genetics</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfa0f09862c96ff9cec51a61499fdcc67366281e" target='_blank'>
              Analysis of microarray and single-cell RNA-seq identifies gene co-expression, cell–cell communication, and tumor environment associated with metabolite interconversion enzyme in prostate cancer
              </a>
            </td>
          <td>
            Danial Hashemi Karoii, Ali Shakeri Abroudi, Nadia Forghani, Sobhan Bavandi, Melika Djamali, Hamoon Baghaei, Sana Shafaeitilaki, Ehsan HasanZadeh
          </td>
          <td>2025-02-13</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Here, we have discussed the molecular mechanisms of p53-responsive microRNAs dysregulation in response to genotoxic stress in diffuse large B-cell lymphoma (DLBCL) patients. The role of micro ribonucleic acids (microRNAs) in p53-signaling cellular stress has been studied. MicroRNAs are the small non-coding RNAs, which regulate genes expression at post-transcriptional level. Many of them play a crucial role in carcinogenesis and may act as oncogenes or suppressor of tumor growth. The understanding of the effect of p53-responsive microRNA dysregulation on oncogenesis achieved in recent decades opens wide opportunities for the diagnosis, prediction and of microRNA-based cancer therapy. Development of new bioinformatics tools and databases for microRNA supports DLBCL research. We overview the studies on the role of miRNAs in regulating gene expression associated with tumorigenesis processes, with particular emphasis on their role as tumor growth-suppressing factors. The starting point is a brief description of the classical microRNA biogenesis pathway and the role of p53 in regulating the expression of these molecules. We analyze various molecular mechanisms leading to this dysregulation, including mutations in the TP53 gene, DNA methylation, changes in host-genes expression or microRNA gene copy number, mutations in microRNA and microRNA biogenesis genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/740d5b189850125f69ad77979cf0d447a7b59eef" target='_blank'>
              Deregulation mechanisms and therapeutic opportunities of p53-responsive microRNAs in diffuse large B-cell lymphoma
              </a>
            </td>
          <td>
            E. Voropaeva, Yuriy L. Orlov, Anastasia B. Loginova, O. Seregina, Vladimir N. Maksimov, T. I. Pospelova
          </td>
          <td>2025-01-07</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Colorectal cancer (CRC) patients with microsatellite-stable (MSS) tumors are mostly treated with chemotherapy. Clinical benefits of targeted therapies depend on mutational states and tumor location. Many tumors carry mutations in KRAS proto-oncogene, GTPase (KRAS) or B-Raf proto-oncogene, serine/threonine kinase (BRAF), rendering them more resistant to therapies. We performed whole-exome sequencing and RNA-Sequencing of 28 tumors of the Athens Comprehensive Cancer Center CRC cohort, and molecularly characterized CRC patients based on their microsatellite instability (MSI) status, single-nucleotide variations (SNVs)/copy number alterations (CNAs), and pathway/transcription factor activities at the individual patient level. Variants were classified using a computational score for integrative cancer variant annotation and prioritization. Complementing this with public multi-omics datasets, we identified activation of transforming growth factor beta (TGFβ) signaling to be more strongly activated in MSS patients, whereas Janus kinase (JAK)-signal transducer and activator of transcription (STAT) and mitogen-activated protein kinase (MAPK) molecular cascades were activated specifically in MSI tumors. We unraveled mechanisms consistently perturbed in the transcriptional and mutational circuits and identified Runt-related transcription factors (RUNX transcription factors) as putative biomarkers in CRC, given their role in the regulation of pathways involved in tumor progression and immune evasion. Assessing the immunogenicity of CRC tumors in the context of RAS/RAF mutations and MSI/MSS status revealed a critical impact that KRAS mutations have on immunogenicity, particularly in the MSS patient subgroup, with implications for diagnosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a74b1bcf5fa54757058aca72542a0424e2f6a831" target='_blank'>
              Molecular and functional profiling unravels targetable vulnerabilities in colorectal cancer.
              </a>
            </td>
          <td>
            E. Vlachavas, Konstantinos Voutetakis, Vivian Kosmidou, Spyridon Tsikalakis, Spyridon Roditis, Konstantinos Pateas, Ryangguk Kim, Kymberleigh Pagel, Stephan Wolf, Gregor Warsow, Antonia Dimitrakopoulou-Strauss, Georgios N Zografos, Alexander Pintzas, J. Betge, O. Papadodima, Stefan Wiemann
          </td>
          <td>2025-01-28</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Abstract The molecular basis of adaptive evolution and cancer progression are both complex processes that share many striking similarities. The potential adaptive significance of environmentally-induced epigenetic changes is currently an area of great interest in both evolutionary and cancer biology. In the field of cancer biology intense effort has been focused on the contribution of stress-induced non-coding RNAs (ncRNAs) in the activation of epigenetic changes associated with elevated mutation rates and the acquisition of environmentally adaptive traits. Examples of this process are presented and combined with more recent findings demonstrating that stress-induced ncRNAs are transferable from somatic to germline cells leading to cross-generational inheritance of acquired adaptive traits. The fact that ncRNAs have been implicated in the transient adaptive response of various plants and animals to environmental stress is consistent with findings in cancer biology. Based on these collective observations, a general model as well as specific and testable hypotheses are proposed on how transient ncRNA-mediated adaptive responses may facilitate the transition to long-term biological adaptation in both cancer and evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbfc2bb67d7c58db953ed76c627eb64e92c4eb6a" target='_blank'>
              Adaptive Significance of Non-coding RNAs: Insights from Cancer Biology
              </a>
            </td>
          <td>
            John F. McDonald
          </td>
          <td>2025-01-01</td>
          <td>Molecular Biology and Evolution</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Oncogenes are typically overexpressed in tumor tissues and often linked to poor prognosis. However, recent advancements in bioinformatics have revealed that many highly expressed genes in tumors are associated with better patient outcomes. These genes, which act as tumor suppressors, are referred to as “paradoxical genes.” Analyzing The Cancer Genome Atlas (TCGA) confirmed the widespread presence of paradoxical genes, and KEGG analysis revealed their role in regulating tumor metabolism. Mechanistically, discrepancies between gene and protein expression-affected by pre- and post-transcriptional modifications-may drive this phenomenon. Mechanisms like upstream open reading frames and alternative splicing contribute to these inconsistencies. Many paradoxical genes modulate the tumor immune microenvironment, exerting tumor-suppressive effects. Further analysis shows that the stage- and tumor-specific expression of these genes, along with their environmental sensitivity, influence their dual roles in various signaling pathways. These findings highlight the importance of paradoxical genes in resisting tumor progression and maintaining cellular homeostasis, offering new avenues for targeted cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3eab8741264f274ad63584e4f394278cc1295e34" target='_blank'>
              Discovery of paradoxical genes: reevaluating the prognostic impact of overexpressed genes in cancer
              </a>
            </td>
          <td>
            Dequan Liu, Lei Liu, Xiangyu Che, Guangzhen Wu
          </td>
          <td>2025-01-22</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Basal-like breast cancer (BLBC) is characterized by an aggressive clinical course, high genomic instability, and limited therapeutic options. The Inhibitor of Differentiation 4 (ID4) protein has been identified as a critical regulator of BLBC, where its overexpression correlates with poor prognosis. However, the mechanistic contributions of ID4 to BLBC tumorigenesis remain incompletely understood. In this study, we employed an integrative approach combining CRISPR-Cas9-mediated ID4 knockout, small-molecule inhibition, in vivo tumor modeling, and in silico transcriptional analyses to investigate the functional role of ID4 in BLBC. CRISPR-Cas9-mediated knockout of ID4 in MDA-MB-231 cells resulted in significant reductions in proliferation, colony formation, and Ki67 expression, indicating a loss of aggressive phenotypic traits. In vivo xenograft studies further revealed that ID4-silenced cells exhibited markedly delayed tumor formation and a significant reduction in metastatic potential compared to controls. Kaplan-Meier survival analysis of basal-like tumors from The Cancer Genome Atlas (TCGA) dataset demonstrated that patients with low ID4 expression had improved relapse-free survival. Gene set enrichment analysis (GSEA) of BLBC tumors stratified by ID4 expression revealed a shift toward luminal-like transcriptional programs in the ID4-low subgroup, including increased estrogen response and inflammatory signaling pathways. Furthermore, transcription factor activity analysis identified the activation of MYC, JUN, and STAT in ID4-low tumors, suggesting a transition toward a more differentiated phenotype. Finally, pharmacological inhibition of ID4 using the small-molecule degrader AGX51 effectively reduced proliferation in TNBC cells, highlighting ID4 as a potential therapeutic target. Together, these findings establish ID4 as a key driver of BLBC aggressiveness and suggest that its inhibition may represent a viable therapeutic strategy. This study provides compelling evidence supporting the development of ID4-targeted therapies for TNBC patients, with the potential to improve clinical outcomes in this challenging disease subset.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/147bfd57152033ce1c79fd9d00f386b5bffcdc1b" target='_blank'>
              Exploring ID4 as a Driver of Aggression and a Therapeutic Target in Triple-Negative Breast Cancer
              </a>
            </td>
          <td>
            C. Toro, S. Real, S. Laurito, MT Branham
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5668e09864551d73b8ef36f697d610eddc6e8f2b" target='_blank'>
              MircoRNAs predict and modulate responses to chemotherapy in leukemic patients.
              </a>
            </td>
          <td>
            D. Ellakwa, Mohamed Ahmed Abdelmalek, Mostafa M Mostafa, Takwa E. Ellakwa, Al-Hassan Soliman Wadan
          </td>
          <td>2025-01-14</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>2</td>
          <td>12</td>
        </tr>

        <tr id="Copy number variations of the human CNTN6 gene, resulting from megabase-scale microdeletions or microduplications in the 3p26.3 region, are frequently implicated in neurodevelopmental disorders such as intellectual disability and developmental delay. However, duplication of the full-length human CNTN6 gene presents with variable penetrance, resulting in phenotypes that range from neurodevelopmental disorders to no visible pathologies, even within the same family. Previously, we obtained a set of induced pluripotent stem cell lines derived from a patient with a CNTN6 gene duplication and from two healthy donors. Our findings demonstrated that CNTN6 expression in neurons carrying the duplication was significantly reduced. Additionally, the expression from the CNTN6 duplicated allele was markedly lower compared to the wild-type allele. Here, we first introduce a system for correcting megabase-scale duplications in induced pluripotent stem cells and secondly analyze the impact of this correction on CNTN6 gene expression. We showed that the deletion of one copy of the CNTN6 duplication did not affect the expression levels of the remaining allele in the neuronal cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53c9c747915f6b48c8d45b21bda1f171cd233c0f" target='_blank'>
              Targeted correction of megabase-scale CNTN6 duplication in induced pluripotent stem cells and impacts on gene expression
              </a>
            </td>
          <td>
            Maria Gridina, Polina Orlova, Oleg Serov
          </td>
          <td>2025-01-20</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f7b4b6b0cd06f7bc05e01f01febfa31643b31b8" target='_blank'>
              Prevalent integration of genomic repetitive and regulatory elements and donor sequences at CRISPR-Cas9-induced breaks
              </a>
            </td>
          <td>
            C. Bi, Baolei Yuan, Yingzi Zhang, Mengge Wang, Yeteng Tian, Mo Li
          </td>
          <td>2025-01-20</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 Immunotherapy has been a promising treatment for various cancer types. This therapy heavily relies on the efficiency of the targeted neoantigens for T cell recognition and downstream immune responses. Neoantigens derived from tumor-specific nonsynonymous somatic mutations have been a great resource and manifested remarkable therapeutic effects, yet with high patient specificity and high dependency on mutation rate. Here, we propose a novel strategy for discovering neoantigens derived from epigenetic mutations, rather than genetic mutations, from transposable elements (TEs) sequences. TEs make up 50% of the human genome but have been long considered to be “junk DNA” and therefore overlooked in medical genomics. However, recent works from our lab have shown that, despite being epigenetically silenced in somatic cells, certain TEs contribute significantly to the evolution of regulatory networks and shape the cancer transcriptome landscape, including functioning as promoters and producing novel, tumor-specific TE-derived transcripts (TSTETs) that can be translated into proteins. Given the plasticity of epigenome, neoantigens resulted from epigenetic mutations could be pan-cancer with high recurrence. Thus, we hypothesize that cryptic promoters embedded in TEs can be re-activated due to epigenetic dysregulation in cancer, leading to the generation of highly recurrent, pan-cancer, TE-derived, tumor-specific protein products that could be employed as neoantigens for immuno-therapy. In this study, we developed a computational pipeline termed TE Promoter Finder 3 (TEProf3) to precisely identify TE-derived promoters and transcripts genome-wide from transcriptomic data, including data from both long-read and short-read sequencing. We applied TEProf3 to analyze 12,015 tumor samples encompassing 33 cancer types from The Cancer Genome Atlas (TCGA) and Clinical Proteomic Tumor Analysis Consortium (CPTAC) datasets. Our analysis revealed 14,417 of TSTETs and hundreds of tumor-specific TE-derived antigens (TSTEAs) from the tumor transcriptomic and mass spectrometry data, respectively. Notably, most TSTETs exhibited high recurrence rates, with over 10% recurrence in the majority and some exceeding 90% recurrence. Each tumor sample displayed a median number of 19 TSTETs. Certain cancer types showed a comparable landscape of tumor-specific TE-derived promoters, potentially due to similarities in epigenetic mutations and cell types of origin. The number of TSTETs correlated with tumor progression in some cancer types, suggesting a link between progressive dysregulation of TE-derived promoters and tumorigenesis. We further assessed the efficiency of antigen presentation in cancer cell lines as a proof of concept. We successfully identified T cell receptors sequences from CD8+ T cells that exhibited strong binding affinity for TSTEAs and demonstrated cytotoxicity effects on the cancer cell lines. In summary, this work represents one of the first comprehensive catalogs of TSTEAs, providing a valuable resource for future immunotherapy strategies.
 Citation Format: Yonghao Liang, Xiaoyun Xing, Ting Wang. Transposable elements activation produces pan-cancer tumor-specific neoantigens [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B024.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a628523c6f85200cb9e347eacaf757c8785c94bb" target='_blank'>
              Abstract B024: Transposable elements activation produces pan-cancer tumor-specific neoantigens
              </a>
            </td>
          <td>
            Yonghao Liang, Xiaoyun Xing, Ting Wang
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/047a24a952e93dd146dc76725bcf8e221c664c94" target='_blank'>
              Targeting chromosomally unstable tumors with a selective KIF18A inhibitor
              </a>
            </td>
          <td>
            Aaron F Phillips, Rumin Zhang, Mia Jaffe, Ryan Schulz, Marysol Chu Carty, Akanksha Verma, Tamar Feinberg, Michael D Arensman, Alan Chiu, Reka Letso, Nazario Bosco, Katelyn A. Queen, Allison R Racela, Jason K. Stumpff, Celia Andreu-Agullo, Sarah E. Bettigole, R. Depetris, Scott Drutman, Shinsan M Su, Derek A Cogan, Christina H Eng
          </td>
          <td>2025-01-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Identifying genes involved in cancer is crucial for understanding the underlying molecular mechanisms of the disease and developing effective treatment strategies. Differential expression analysis (DEA) is the predominant method used to identify cancer-related genes. This approach involves comparing gene expression levels between different samples, such as cancerous and non-cancerous tissues, to identify genes that are significantly upregulated or downregulated in cancer. DEA is based on the commonly believed assumption that genes upregulated in cancerous tissues have the potential to function as oncogenes. Their expression levels often correlate with cancer advancement and unfavorable prognosis, whereas downregulated genes display the opposite correlation. However, contrary to the prevailing belief, our analysis utilizing The Cancer Genome Atlas (TCGA) databases revealed that the upregulated or downregulated genes in cancer do not always align with cancer progression or prognosis. These findings emphasize the need for alternative approaches for identifying cancer-related genes that may be more accurate and effective. To address this need, we compared the effectiveness of machine learning (ML) methods with that of traditional DEA in the identification of cancer-related genes. ML algorithms have the advantage of being able to analyze large-scale genomic data and identify complex patterns that may go unnoticed by traditional methods. Our results demonstrated that ML methods significantly outperformed DEA in the screening of cancer-related genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6167fa4f27157ad0473050eb8b2f5d919ef194a8" target='_blank'>
              Challenging Conventional Perceptions of Oncogenes and Tumor Suppressor Genes: A Comprehensive Analysis of Gene Expression Patterns in Cancer.
              </a>
            </td>
          <td>
            Mingyuan Zou, Li Qiu, Wentao Wu, Hui Liu, Han Xiao, Jun Liu
          </td>
          <td>2025-02-01</td>
          <td>Genes, chromosomes & cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d042ce055ab273191af06fc94f1a5e070d7aee0d" target='_blank'>
              The emerging roles of aberrant alternative splicing in glioma
              </a>
            </td>
          <td>
            Reda Ben Mrid, Sara El Guendouzi, Marco Mineo, Rachid El Fatimy
          </td>
          <td>2025-02-06</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Abstract RNA modifications are emerging as critical cancer regulators that influence tumorigenesis and progression. Key modifications, such as N6‐methyladenosine (m6A) and 5‐methylcytosine (m5C), are implicated in various cellular processes. These modifications are regulated by proteins that write, erase, and read RNA and modulate RNA stability, splicing, translation, and degradation. Recent studies have highlighted their roles in metabolic reprogramming, signaling pathways, and cell cycle control, which are essential for tumor proliferation and survival. Despite these scientific advances, the precise mechanisms by which RNA modifications affect cancer remain inadequately understood. This review comprehensively examines the role RNA modifications play in cancer proliferation, metastasis, and programmed cell death, including apoptosis, autophagy, and ferroptosis. It explores their effects on epithelial–mesenchymal transition (EMT) and the immune microenvironment, particularly in cancer metastasis. Furthermore, RNA modifications’ potential in cancer therapies, including conventional treatments, immunotherapy, and targeted therapies, is discussed. By addressing these aspects, this review aims to bridge current research gaps and underscore the therapeutic potential of targeting RNA modifications to improve cancer treatment strategies and patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d0f5a36dcee65c7337d6b6996dd54de083cb710" target='_blank'>
              RNA modifications in cancer
              </a>
            </td>
          <td>
            Han Wu, Shi Chen, Xiang Li, Yuyang Li, He Shi, Yiwen Qing, Bohe Shi, Yifei Tang, Zhuoyi Yan, Yang Hao, Dongxu Wang, Weiwei Liu
          </td>
          <td>2025-01-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background: The human FOXM1 transcription factor controls cell cycle progression and genome stability, and it has been correlated to the onset and progression of many tumor types. Methods: In our study, we collected all recent sequence and quantitative transcriptomics data about FOXM1, testing its presence across vertebrate evolution and its upregulation in cancer, both in bulk tissue contexts (by comparing the TCGA tumor dataset and the GTEx normal tissue dataset) and in single-cell contexts. Results: FOXM1 is significantly and consistently upregulated in all tested tumor types, as well as in tumor cells within a cancer microenvironment. Its upregulation reverberates in the upregulation of its target genes and can be used as a biomarker for poor cancer outcome in at least four tumor types. Conclusions: Despite its lack of cancer-related mutations and amplifications, the recurring upregulation of FOXM1 in all tumors puts a focusing lens on this gene as a candidate pan-cancer master regulator.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e19cd5cddb455ecb73607992ac0679d96d8dc34" target='_blank'>
              Pan-Cancer Upregulation of the FOXM1 Transcription Factor
              </a>
            </td>
          <td>
            Daniele Pozzobon, Arianna Bellezza, F. Giorgi
          </td>
          <td>2025-01-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Elevated anti-citrullinated protein antibodies (ACPA) levels in the peripheral blood are associated with an increased risk for developing rheumatoid arthritis (RA). Currently, no treatments are available that prevent progression to RA in these at-risk individuals. In addition, diverse pathogenic mechanisms underlying a common clinical phenotype in RA complicate therapy as no single agent is universally effective. We propose that a unifying set of transcription factor and their downstream pathways regulate a pro-inflammatory cell communication network, and that this network allows multiple cell types to serve as pathogenic drivers in at-risk individuals and in early RA. To test this hypothesis, we identified ACPA-positive at-risk individuals, patients with early ACPA-positive RA and matched controls. We measured single cell chromatin accessibility and transcriptomic profiles from their peripheral blood mononuclear cells. The datasets were then integrated to define key TF, as well as TF-regulated targets and pathways. A distinctive TF signature was enriched in early RA and at-risk individuals that involved key pathogenic mechanisms in RA, including SUMOylation, RUNX2, YAP1, NOTCH3, and β-Catenin Pathways. Interestingly, this signature was identified in multiple cell types, including T cells, B cells, and monocytes, and the pattern of cell type involvement varied among the at-risk and early RA participants, supporting our hypothesis. Similar patterns of individualized gene expression patterns and cell types were confirmed in single cell studies of RA synovium. Cell communication analysis revealed that the lineages displaying this RA TF signature deliver a common set of pro-inflammatory mediators to receiver cells that subsequently orchestrate rheumatoid inflammation. These cell-type-specific signature pathways could explain the personalized pathogenesis of RA and contribute to the diversity of clinical responses to targeted therapies. Furthermore, these data could provide opportunities for stratifying individuals at-risk for RA, and selecting therapies tailored for prevention or treatment of RA. Overall, this study supports a new paradigm to understand how a common clinical phenotype could arise from diverse pathogenic mechanisms and demonstrates the relevance of peripheral blood cells to synovial disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86677aefd5550e942dc49063256f0df11e1c06ea" target='_blank'>
              Multi-lineage transcriptional and cell communication signatures define pathways in individuals at-risk for developing rheumatoid arthritis that initiate and perpetuate disease
              </a>
            </td>
          <td>
            Cong Liu, E. B. Prideaux, Peiyao Wu, David L. Boyle, Amy Westermann, K. Nguyen, Vlad Tsaltskan, Leander Lazaro, Andrea Ochoa, Kevin D. Deane, M. Feser, M. K. Demoruelle, Kristine A. Kuhn, V. M. Holers, Fan Zhang, L. Moss, Megan A. L. Criley, Brian C. Hattel, Marguerite D. Siedschlag, L. Okada, Mark A. Gillespie, P. Genge, M. Weiss, Veronica Hernandez, Julian Reading, Lynne A. Becker, Jane H. Buckner, C. Speake, Thomas F. Bumol, Peter J. Skene, G. Firestein, Wei Wang
          </td>
          <td>2025-02-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>99</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93bae82e2a8f4a414d538a8210faf20fea3eff9d" target='_blank'>
              Molecular and pharmacological heterogeneity of ETV6::RUNX1 acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Zhenhua Li, Huanbin Zhao, Wenjian Yang, Maud Maillard, S. Yoshimura, Yu-Chih Hsiao, Xingyu Huang, Yoshihiro Gocho, L. Rowland, Anthony Brown, Landon Choi, K R Crews, C. Mullighan, Samuel Brady, Cheng Cheng, Ti-Cheng Chang, Gang Wu, M. Loh, A. E. J. Yeoh, Federico Antillon-Klussmann, S. Jeha, H. Inaba, Jiyang Yu, Ching-Hon Pui, Seth E. Karol, William E Evans, Jun J Yang
          </td>
          <td>2025-01-29</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>103</td>
        </tr>

        <tr id="The use of replication-competent viruses for selective tumor oncolysis while sparing normal cells marks a significant advancement in cancer treatment. HSV-1 presents several advantages that position it as a leading candidate for oncolytic virotherapies. Its large genome can accommodate insertions over 30 kb or deletions of multiple virulence genes without compromising lytic replication in tumor cells. Additionally, anti-herpes drugs can inhibit its replication during accidental infections. Importantly, HSV-1 does not integrate into the host genome and cause mutations. The HSV-1 genome can be modified through genetic engineering in two main ways: first, by reducing infectivity and toxicity to normal cells via limited replication and assembly, altered protein-virus receptor binding, and minimized immune evasion; second, by enhancing anticancer activity through disruption of tumor cell metabolism, induction of autophagy, improved immune recognition, and modification of the tumor microenvironment. In this mini-review, we systematically examine genetic modification strategies for oncolytic HSV-1 while highlighting advancements from these modifications. Certain genetic alterations have shown efficacy in improving clinical outcomes for HSV-1-based therapies. These modifications include silencing specific genes and inserting exogenous genes into the HSV-1 genome. The insertion of exogenous genes has increasingly been used to develop new oncolytic HSV-1 variants. Finally, we discuss limitations associated with oncolytic virotherapy at the conclusion of this review. As more clinical trials explore newly engineered therapies, they are likely to yield breakthroughs and promote broader adoption for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/182b0f8940cb24ae25b0fa04d28f451e16c714c9" target='_blank'>
              Advanced progress in the genetic modification of the oncolytic HSV-1 virus
              </a>
            </td>
          <td>
            Mi Zhou, Zhenyu Shen
          </td>
          <td>2025-01-21</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Currently, mankind is fiercely struggling with cancer. Recently, we have been winning the battle against cancer through precision medicine and accompanying diagnostic methods, and we are raising many hopes with blockbuster drugs. It would be even better if we could read the cancer nucleotide sequence, identify them in advance, and suggest treatments simultaneously. However, this may be an impossible dream because it takes a lot of time and effort to diagnose and ensure all the long gene sequences of cancer at once. Thus, victory will be even closer if a rapid and accurate diagnosis of the cancer-specific gene biomarkers that will soon be imprinted can be made. With the advent of nanotechnology, a new short cancer diagnostic toolkit has been proposed to achieve the goal. This review presents a small diagnostic device that detects genetic fragments (simply 'Gizmo') of certain cancers. The development of numerous diagnostic methods has focused on (1) directly detecting pre-selectively targeted genes using novel diagnostic systems, and (2) indirectly detecting substantial improvements in diagnostic sensitivity only through detection signal amplification without existing gene amplification steps. Our fight against cancer is not a dream, but the result of success, and it is assumed that victory will accelerate.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eccae0a3a741529fc24b2b02ce5646b06a22df12" target='_blank'>
              Engineering Gizmos for Short Cancer Genetic Fragments Discrimination.
              </a>
            </td>
          <td>
            Jisoo Yuk, Jeonghun Kim, Sunghwan Jung, Soong Ho Um
          </td>
          <td>2025-02-05</td>
          <td>Chembiochem : a European journal of chemical biology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="ABSTRACT The discovery at the end of the 20th century of genes that induce cell death revolutionised the biocontaintment of genetically manipulated bacteria for environmental or agricultural applications. These bacterial ‘killer’ genes were then assayed for their potential to target and control malignant cells in human cancers. The identification of the bacteriomes in different human organs and tissues, coupled with the observation that bacteria tend to accumulate near tumours, has opened new avenues for anti‐cancer strategies. This progress, along with recent insights into how cancer cells evade immune response, has prompted innovative therapeutic approaches. Tumour microenvironments are typically nutrient‐rich, characterised by low oxygen tensions and very resistant to immune responses. Two recent studies in MBT highlight the promise of using Salmonella typhimurium and Escherichia coli as vectors in novel cancer treatments. Engineered S. typhimurium strains can generate adjuvant flagellin‐antigen complexes that function as in situ vaccines, hence increasing the immunogenic responses within tumour environment. Similarly, gut E. coli can be used as vectors to targert tumour cells in colon cancer, enabling both diagnostic applications and localised treatments. Both approaches hold significant potential to improve patient survival outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bba9ffcefc81fe9e0a146a68f1324d903a6bef1" target='_blank'>
              Bacteria as Precision Tools for Cancer Therapy
              </a>
            </td>
          <td>
            C. Michán, José Prados, Juan-Luis Ramos
          </td>
          <td>2025-01-01</td>
          <td>Microbial Biotechnology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d60a3bedb4bf1a029a736f9136afc2a07aef47f" target='_blank'>
              Decoding the functional impact of the cancer genome through protein-protein interactions.
              </a>
            </td>
          <td>
            Haian Fu, Xiulei Mo, Andrey A. Ivanov
          </td>
          <td>2025-01-14</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51d1fd954b718bc92e2f3d26e9d0a2654e108d08" target='_blank'>
              Prostate cancer epigenetics - from pathophysiology to clinical application.
              </a>
            </td>
          <td>
            V. Constâncio, J. Lobo, J. P. Sequeira, Rui Henrique, Carmen Jerónimo
          </td>
          <td>2025-01-16</td>
          <td>Nature reviews. Urology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Monosomy 7 and deletion 7q are common chromosomal abnormalities in myeloid malignancies, and they are associated with a poor prognosis. The mechanism underlying their acquisition remains elusive. We identified a cohort of 24 patients exhibiting clones with different chromosome 7 abnormalities, such as deletion 7q, unstable derivatives (ring chromosomes or ‘naked’ centromeres), and monosomy 7. We designated this group as having cytogenetic clonal evolution of chromosome 7 abnormalities (CCE7). In some cases, CCE7 correlated with disease progression, suggesting that deletions or other derivatives involving the q-arm of chromosome 7 may arise early in the disease course. These abnormalities may be transient but can potentially evolve into monosomy 7. Within the CCE7 group, telomere loss or shortening may contribute to chromosomal instability and the emergence of unstable derivatives, as the chromosome 7 derivatives displayed loss or rearrangement of subtelomeric regions. Moreover, we identified variants in genes implicated in telomere biology disorders and observed specific genetic mutation profiles associated with different chromosome 7 abnormalities. These findings shed light on a potential mechanism leading to monosomy 7 through the evolution of chromosome 7q abnormalities. Identifying patients at risk of developing monosomy 7, based on the presence of unstable derivatives with telomere loss or a specific mutation profile, could potentially enhance patient management and guide the development of novel therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd97cfdef751d5eb8d87f19388c6ddb34eef62a2" target='_blank'>
              Emergence and Cytogenetic Clonal Evolution of Chromosome 7 Abnormalities in Myeloid Malignancies: Investigating the Role of Telomere Dysfunction
              </a>
            </td>
          <td>
            Carmen Baldazzi, L. Bandini, V. Robustelli, A. Patuelli, C. Venturi, Alessandra Grassi, G. Marzocchi, Angela Ielpo, V. Solli, M. Bochicchio, Stefania Paolini, Chiara Sartor, F. Zingarelli, Antonio Curti, E. Ottaviani, N. Testoni
          </td>
          <td>2025-01-29</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Despite rapid advances in mapping genetic drivers and gene expression changes in hematopoietic stem cells (HSCs), there is a relative paucity of studies at the protein level. Here, we perform a deep, multi-omic characterization (epigenome, transcriptome and proteome) of HSCs carrying a loss-of-function mutation in Tet2, a key driver of increased self-renewal in blood cancers. Using state-of-the-art, multiplexed, low-input mass spectrometry (MS)-based proteomics, we profile wildtype (WT) and TET2-deficient (Tet2-/-) HSCs and show that the proteome captures previously unrecognized molecular processes which define the pre-leukemic HSC molecular landscape. Specifically, we obtain more accurate stratification of WT and Tet2-/- HSCs than transcriptomic approaches and identify extracellular matrix (ECM) molecules as novel points of dysregulation upon TET2 loss. HSC expansion assays using ECM-functionalized hydrogels confirm a selective effect on the expansion of Tet2-mutant HSCs. Taken together, our study represents a comprehensive molecular characterization of Tet2-mutant HSCs and identifies a previously unanticipated role of ECM molecules in regulating self-renewal of disease-driving HSCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee30ff0c55cc7680e35335a72a060f31e9944948" target='_blank'>
              Selection of pre-leukemic hematopoietic stem cells driven by distinct extracellular matrix molecules
              </a>
            </td>
          <td>
            M. Jassinskaja, D. Bode, Monika Gonka, Theodoros I. Roumeliotis, Alexander J Hogg, Juan A Rubio Lara, Ellie Bennett, Joanna Milek, Bart Theeuwes, M. S. Vijayabaskar, Lilia Cabrera Cosme, James L C Che, Sandy MacDonald, Sophia Ahmed, Benjamin A. Hall, Grace Vasey, Helena Kooi, M. Belmonte, Mairi S. Shepherd, W. Brackenbury, I. Kucinski, Satoshi Yamazaki, Andrew N. Holding, Alyssa H Cull, Nicola K Wilson, B. Göttgens, J. Choudhary, David G. Kent
          </td>
          <td>2025-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>80</td>
        </tr>

        <tr id="Significance Understanding the multifaceted genetic disorder ataxia–telangiectasia (A-T) offers new insights into the functions of the ATM protein kinase, the loss of which underlies this disease. A-T exemplifies the connection between genome instability, cellular senescence, and aging. This study elucidates the pathways leading to a hallmark of A-T’s cellular phenotype: the premature senescence of primary fibroblasts derived from affected individuals. Our findings provide a mechanistic understanding of how inherited genome instability accelerates cellular senescence, offering potential explanations for several A-T symptoms. The work introduces murine primary lung fibroblasts growing under 3% oxygen as a valuable experimental system to explore the genome instability-senescence link and the molecular events leading to the neoplastic transformation of primary diploid cells without any directed manipulations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d92deb9c47fa1db095e272225b8766e094ed2771" target='_blank'>
              The cGAS–STING, p38 MAPK, and p53 pathways link genome instability to accelerated cellular senescence in ATM-deficient murine lung fibroblasts
              </a>
            </td>
          <td>
            M. Haj, Yann Frey, Amit Levon, Avishai Maliah, Tal Ben-Yishay, Rachel Slutsky, Riham Smoom, Y. Tzfati, Uri Ben-David, Carmit Levy, Ran Elkon, Y. Ziv, Yosef Shiloh
          </td>
          <td>2025-01-07</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>1</td>
          <td>30</td>
        </tr>

        <tr id="Prostate cancer (PCa) is a prevalent male cancer with high survival rates, except in advanced or metastatic stages, for which effective treatments are lacking. Metastatic PCa involves complex mechanisms including loss of tumor suppressor genes and DNA repair molecules, which impacts therapy responses. We have reanalyzed data from a CRISPR/Cas9 genome wide screening previously performed to identify essential regulators of invasive abilities of the metastatic cell line DU145 identifying SYCP3 as a regulator of metastatic invasion. Subsequent analyses of tumor samples demonstrated that SYCP3 expression is frequently upregulated in PCa tumors from patients in advanced stages. Furthermore, SYCP3 genetic depletion significantly reduced the invasive and migratory abilities of DU145 cells and increased their adhesion capacity. Additionally, and due to the implication of SYCP3 on DNA repair processes, we have analyzed the role of SYCP3 on the cellular response to radiotherapy (RT) and found that its depletion induced RT resistance, suggesting a role for SYCP3 in DNA damage response and genomic instability. All these data support a role for SYCP3 in PCa metastasis and provides opportunities for personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/016d79d4fcdaa6ad45de27d2da94398a7826f930" target='_blank'>
              CRISPR screening reveals SYCP3 as a key driver of metastasis in prostate cancer
              </a>
            </td>
          <td>
            Maria Rodrigo-Faus, Inés del Monte-García, Marina Hermosilla-Trespaderne, Alicia Gordo-Vega, Natalia Vidal, Javier Puente, Melchor Saiz-Pardo, Ángel M. Cuesta, Hui-Qi Qu, H. Hakonarson, Almudena Porras, Daniel Sanchez-Parcerisa, Paloma Bragado, Á. Gutierrez-Uzquiza
          </td>
          <td>2025-01-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Aggressive variant prostate cancer (AVPC) is characterized by a molecular signature involving combined defects in TP53, RB1, and/or PTEN (AVPC-TSGs), identifiable through immunohistochemistry or genomic analysis. The reported prevalence of AVPC-TSG alterations varies widely, reflecting differences in assay sensitivity, treatment pressure, and disease stage evolution. Although robust clinical evidence is still emerging, the study of AVPC-TSG alterations in prostate cancer (PCa) is promising. Alterations in TP53, RB1, and PTEN, as well as the combined loss of AVPC-TSGs, may have significant implications for prognosis and treatment. These biomarkers might help predict responses to various therapies, including hormonal treatments, cytotoxic agents, radiotherapy, and targeted therapies. Understanding the impact of these molecular alterations in patients with PCa is crucial for personalized management. In this review, we provide a comprehensive overview of the emerging prognostic and predictive roles of AVPC-TSG alterations across PCa stages. Moreover, we discuss the implications of different methods used for detecting AVPC-TSG alterations and summarize factors influencing their prevalence. As our comprehension of the genomic landscape of PCa disease deepens, incorporating genomic profiling into clinical decision making will become increasingly important for improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc76168bc6cc3f716d4ad9c939472b94406c6f4f" target='_blank'>
              The Emerging Predictive and Prognostic Role of Aggressive-Variant-Associated Tumor Suppressor Genes Across Prostate Cancer Stages
              </a>
            </td>
          <td>
            Martino Pedrani, Jessica Barizzi, Giuseppe Salfi, Alessandro Nepote, Irene Testi, Sara Merler, Luis Castelo-Branco, R. P. Mestre, Fabio Turco, Luigi Tortola, Jean-Philippe P. Theurillat, S. Gillessen, U. Vogl
          </td>
          <td>2025-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Background: CRISPR/Cas9 genome-editing technology has revolutionized human stem cell (HSC) research, offering novel therapeutic and diagnostic applications. HSCs play a crucial role in regenerative medicine and genetic therapies due to their ability to self-renew and differentiate into various blood cell lineages. The precise genome-editing capability of CRISPR/Cas9 allows for targeted gene modifications, enabling the correction of inherited disorders, disease modeling and the discovery of novel biomarkers. Despite significant advancements, challenges such as off-target effects, delivery efficiency, and ethical concerns persist, requiring further research and optimization.
Objective: This systematic review evaluates the therapeutic and diagnostic potential of CRISPR/Cas9 genome editing in HSCs, focusing on its efficacy in gene correction for hematologic disorders, disease modeling and biomarker discovery.
Methods: A systematic review was conducted following PRISMA guidelines, analyzing studies published between 2015 and 2024. Literature searches were performed in PubMed, Web of Science, and Scopus using MeSH-aligned keywords. The inclusion criteria encompassed peer-reviewed studies utilizing CRISPR/Cas9 for gene modification in HSCs for therapeutic and diagnostic applications. Exclusion criteria included studies that lacked experimental validation, did not focus on HSCs, or were non-English publications. Out of 85 initially retrieved studies, 40 met the inclusion criteria, and 15 were selected for final synthesis.
Results: CRISPR/Cas9 gene-editing strategies in HSCs were categorized as gene knockout (53%), gene activation (40%), and dual knockout/activation (7%). Hematological disorders, including sickle cell anemia and beta-thalassemia, accounted for 35% of studies, demonstrating up to 90% correction in β-globin mutations. Neurodegenerative diseases constituted 20% of studies, where knockout of amyloid precursor protein (APP) in Alzheimer’s models resulted in a 60% reduction in plaque accumulation. Muscular dystrophy studies (10%) showed 75% improvement in dystrophin expression through gene activation. High-throughput CRISPR screening was employed in 15% of studies for biomarker identification. Despite promising outcomes, off-target mutations were observed in 28% of studies, and viral vector-based delivery methods were used in 65%, raising safety concerns.
Conclusion: CRISPR/Cas9 genome editing in HSCs presents a ground-breaking approach for treating genetic disorders and enhancing precision medicine. Its potential to correct disease-causing mutations, model complex disorders, and identify novel therapeutic targets is substantial. However, challenges in delivery methods, long-term safety, and ethical considerations remain barriers to clinical translation. Future research should focus on optimizing high-fidelity Cas9 variants, improving non-viral delivery methods, and addressing ethical concerns to ensure the safe and effective application of CRISPR/Cas9 in regenerative medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d20400a815c119b74c31ff8ba0f49e856aea421b" target='_blank'>
              EVALUATION OF CRISPR/CAS9 GENOME-EDITING SYSTEM IN HUMAN STEM CELLS HSCS: THERAPEUTICS AND DIAGNOSTICS PROSPECTS
              </a>
            </td>
          <td>
            Ehsan ul Haq, Haseeb Khaliq, Ayesha Muddasser, Misha Aslam, Farwa Zafar, Naveera Mazhar, Amna Naheed Khan
          </td>
          <td>2025-01-20</td>
          <td>Insights-Journal of Health and Rehabilitation</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer cells display distinct, recurrent phenotypic cell states. Metastatic spreading correlates with tumor cell state evolution. However, the molecular mechanisms underlying metastasis remain elusive. Here, we demonstrate that the quantitative dosage of oncogenic KRAS drives lung adenocarcinoma progression and metastasis via the integration of external signaling and pioneer transcription factor dynamics into qualitative cell states. Combining mouse models, in vivo CRISPR activation screens, and fate mapping, we show that even mild transcriptional amplification of KRAS significantly fuels tumor progression and metastasis. Chromatin profiling and transcriptomics reveal that high and low KRAS dosages supersede and integrate inflammatory and TGFβ signaling to dictate mouse cancer cell states. Patient data show that KRAS dosages correlate with distinct survival outcomes, transcription factor activity, and cell states. Direct KRAS inhibition in xenografts limits the KRAS-high “proliferative” cell state but spares a minimal residual state mimicking the KRAS-low “ciliated-like” state. Thus, oncogenic KRAS dosage fuels tumor heterogeneity at the cell state level and drives a bimodal tumor evolution during metastasis, with implications for prognosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/688e49a0ef57c8cb9ef7efb0f9083ef2faa0f479" target='_blank'>
              Transcriptional Dosage of Oncogenic KRAS Drives Lung Adenocarcinoma Cell States, Progression and Metastasis
              </a>
            </td>
          <td>
            M. Serresi, Ali Osman Çetin, Yuliia Dramaretska, Sonia Kertalli, M. Schmitt, Heike Naumann, Maria Zschummel, Marie Liesse-Labat, Lucas F. Maciel, Jeroen Declercq, J. Marine, G. Gargiulo
          </td>
          <td>2024-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="
 11


 Background:
 Metastatic squamous cell carcinoma of the anus (SCCA) is associated with significant morbidity and mortality. Due to its relative rarity, there is limited evidence to support systemic therapy regimens informed by genomic data. This study aims to utilize whole-genome and transcriptome sequencing to enhance the understanding of the genetic alterations driving metastatic SCCA and to identify potentially actionable therapeutic targets.
 Methods:
 This report examines nine cases of metastatic SCCA in patients enrolled in the Personalized Oncogenomics (POG) Program at the BC Cancer Agency. Comprehensive genomic profiling, including whole-genome and transcriptome analysis (WGTA) was conducted on fresh tumor biopsy or formalin fixed paraffin embedded (FFPE) tissue. Tumor genomic alterations were analyzed in detail including: SNVs, indels, copy number alterations, structural variants, mutation signatures, viral presence, tumor mutation burden (TMB), expression outliers, and immune cell scores. The somatic alterations were integrated with existing knowledge of drug-target interactions to identify actionable therapeutic targets.
 Results:
 Of the nine patients, eight were found to be HPV-positive, with one exhibiting high tumor mutational burden (TMB>10mut/Mb). Mutation signatures included seven cases with AID/APOBEC signatures (commonly seen with HPV), and one case with an ID2 DNA replication slippage signature. The PI3K/AKT/mTOR pathway was the most commonly affected signaling pathway (n=4), followed by FGFR amplification and overexpression (n=3), and RAS/RAF alterations (
 BRAF
 mutation and
 KRAS
 overexpression, n=2). EGFR amplification was identified in two cases, one of which also exhibited
 EGFR
 overexpression. Additionally,
 FBXW7
 mutations were present in two cases. Immunotherapy was recommended for all nine patients: eight based on HPV positivity, either alone or in combination with other alterations, such as high TMB, SWI/SNF complex alterations, and presence of immune infiltrating cells, and the remaining case based on a SWI/SNF complex mutation. Only two patients ultimately received immunotherapy, both of whom derived clinical benefit. One patient, treated with atezolizumab as part of the phase II CAPTIV-8 trial (NCT04273061), demonstrated progression-free survival (PFS) of 10 months (ongoing), while the other, treated with nivolumab, achieved a PFS of 6 months.
 Conclusions:
 Genomic analysis using WGTA supported the recommendation of immunotherapy for all patients in this cohort. Although only two patients received immunotherapy, both experienced clinical benefit, underscoring the potential of personalized genomic-guided treatment in metastatic SCCA.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c65e46bc3b8c5ed99a2bc82c211bf31bd2460d61" target='_blank'>
              Personalized oncogenomic analysis of metastatic squamous cell carcinoma of the anus: Utilizing whole-genome sequencing to guide clinical decision-making.
              </a>
            </td>
          <td>
            R. D. Peixoto, Melissa McConechy, Howard Lim, M. Marra, Jing Xu, Kathleen Wee, E. Pleasance, J. Laskin
          </td>
          <td>2025-02-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56e458c5fd0e78f846bc042dbe472faae6943ee1" target='_blank'>
              CDC40 suppression induces CDCA5 splicing defects and anti-proliferative effects in lung cancer cells
              </a>
            </td>
          <td>
            Die Hu, Brigitte L Thériault, Vida Talebian, Laurent Hoffer, Julie Owen, Justin J Lim, B. Blencowe, Evelyne Lima-Fernandes, Punit Saraon, Richard Marcellus, Rima Al-awar
          </td>
          <td>2025-01-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>80</td>
        </tr>

        <tr id="The concept of genetic code expansion (GCE) has revolutionized the field of chemical and synthetic biology, enabling the site-specific incorporation of noncanonical amino acids (ncAAs) into proteins, thus opening new avenues in research and applications across biology and medicine. In this review, we cover the principles of GCE, including the optimization of the aminoacyl-tRNA synthetase (aaRS)/tRNA system and the advancements in translation system engineering. Notable developments include the refinement of aaRS/tRNA pairs, enhancements in screening methods, and the biosynthesis of noncanonical amino acids. The applications of GCE technology span from synthetic biology, where it facilitates gene expression regulation and protein engineering, to medicine, with promising approaches in drug development, vaccine production, and gene editing. The review concludes with a perspective on the future of GCE, underscoring its potential to further expand the toolkit of biology and medicine. Through this comprehensive review, we aim to provide a detailed overview of the current state of GCE technology, its challenges, opportunities, and the frontier it represents in the expansion of the genetic code for novel biological research and therapeutic applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10c8f20128f59397264338dcf03afc1ecef23ea9" target='_blank'>
              Genetic Code Expansion: Recent Developments and Emerging Applications
              </a>
            </td>
          <td>
            Yujia Huang, Pan Zhang, Haoyu Wang, Yan Chen, Tao Liu, Xiaozhou Luo
          </td>
          <td>2024-12-31</td>
          <td>Chemical Reviews</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Quantitative understanding of mitochondrial heterogeneity is necessary for elucidating the precise role of these multifaceted organelles in tumor cell development. We demonstrate an early mechanistic role of mitochondria in initiating neoplasticity by performing quantitative analyses of structure-function of single mitochondrial components coupled with single cell transcriptomics. We demonstrate that the large Hyperfused-Mitochondrial-Networks (HMNs) of keratinocytes promptly get converted to the heterogenous Small-Mitochondrial-Networks (SMNs) as the stem cell enriching dose of the model carcinogen, TCDD, depolarizes mitochondria. This happens by physical reorganization of the HMN nodes and edges, which enriches redox tuned SMNs with distinct network complexity. This leads to establishment of transcriptomic interaction between the upregulated redox relevant mtDNA genes and the lineage specific stemness gene, KRT15, prior to cell cycle exit. The SMN enrichment and related transcriptomic connections are sustained in the neoplastic cell population. Consistently, carcinogenic dose incapable of causing pronounced neoplastic stem cell enrichment fails to establish specific enrichment of SMNs and its linked mtDNA-KRT15(stemness) transcriptomic interaction prior to cell cycle exit. The mtDNA-KRT15 modulation is confirmed in cSCC tumors, while highlighting patient heterogeneity. Therefore, we propose that early enrichment of redox-tuned SMNs primes neoplastic transformation by establishing mtDNA-stemness transcriptomic interaction prior to cell cycle exit towards specifying quiescent neoplastic stem cells. Our data implies that redox-tuned SMNs, created by mitochondrial fission, would be sustained by tuning the balance of mitochondrial fission-fusion during neoplastic transformation. The proposed early role of mitochondria in cancer etiology is potentially relevant for designing precision strategies for cancer prevention and therapy. Significance Statement The challenges of understanding the complex role of the multifaceted and heterogenous cellular organelles, mitochondria, can be potentially overcome with their quantitative analyses. We use a combinatorial approach of quantitative analyses of single-mitochondrial-components and scRNA-seq to elucidate a mechanism of mitochondrial priming of cancer initiation by a model carcinogen. We propose that conversion of large Hyperfused-Mitochondrial-Networks (HMNs) to Small-Mitochondrial-Networks (SMNs) primes non-transformed keratinocytes towards their neoplastic transformation. Mechanistically, the SMNs, enriched by modulation of the physical nodes and edges of mitochondrial networks, tunes mitochondrial redox balance to establish transcriptomic interactions towards specifying a state of stemness. Further probing of our fundamental findings in the light of cancer heterogeneity may facilitate refinement of the various proposed mitochondria based targeted cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1a7a71505ee77a3481f995d4e8fb8a923538ee9" target='_blank'>
              Quantitative analyses of single mitochondrial components reveal an early role of small-mitochondrial-networks in priming neoplasticity
              </a>
            </td>
          <td>
            Mayank Saini, Swati Agarwala, Biratal Wagle, B. Spurlock, Bharti Golchha, D. Parker, Kasturi Mitra
          </td>
          <td>2025-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Mitochondria play key roles in maintaining cell life and cell function, and their dysfunction can lead to cell damage. Mitochondrial ribosomal proteins (MRPs) are encoded by nuclear genes and are assembled within the mitochondria. MRPs are pivotal components of the mitochondrial ribosomes, which are responsible for translating 13 mitochondrial DNA-encoded proteins essential for the mitochondrial respiratory chain. Recent studies have underscored the importance of MRPs in cancer biology, revealing their altered expression patterns in various types of cancer and their potential as both prognostic biomarkers and therapeutic targets. Herein, we review the current knowledge regarding the multiple functions of MRPs in maintaining the structure of the mitochondrial ribosome and apoptosis, their implications for cancer susceptibility and progression, and the innovative strategies being developed to target MRPs and mitoribosome biogenesis in cancer therapy. This comprehensive overview aims to provide insights into the role of MRPs in cancer biology and highlight promising strategies for future precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82ce4a5576b3adfb615ae7e49696b3d3c132a872" target='_blank'>
              Mitochondrial Ribosomal Proteins and Cancer
              </a>
            </td>
          <td>
            Huiyi Wu, Xiaowei Zhu, Huilin Zhou, Min Sha, Jun Ye, Hong-Jeng Yu
          </td>
          <td>2025-01-01</td>
          <td>Medicina</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Simple Summary Cancer cells arise from multiple complementary genomic and epigenomic abnormalities that deregulate the pathways controlling cell proliferation, cell survival, and tissue homeostasis. Epigenetic modifications of histones, DNA, chromatin structure, and non-coding RNA expression are emerging as critical determinants of gene expression and essential drivers of tumor cell heterogeneity and neoplastic cell phenotypes. Multipotent stem-like cells (also referred to as cancer stem cells, CSCs) are critical determinants of tumor propagation, therapeutic resistance, and recurrence following treatment. However, our understanding of how these multidimensional epigenetic modifications cooperatively drive cancer cell stemness and how to efficaciously target them is currently inadequate to positively impact patient outcomes. Abstract Background/Objectives: CSCs are critical drivers of the tumor and stem cell phenotypes of glioblastoma (GBM) cells. Chromatin modifications play a fundamental role in driving a GBM CSC phenotype. The goal of this study is to further our understanding of how stem cell-driving events control changes in chromatin architecture that contribute to the tumor-propagating phenotype of GBM. Methods: We utilized computational analyses to identify a subset of clinically relevant genes that were predicted to be repressed in a Polycomb repressive complex 2 (PRC2)-dependent manner in GBM upon induction of stem cell-driving events. These associations were validated in patient-derived GBM neurosphere models using state-of-the-art molecular techniques to express, silence, and measure microRNA (miRNA) and gene expression changes. Advanced Poly(β-amino ester) nanoparticle formulations (PBAEs) were used to deliver miRNAs in vivo to orthotopic human GBM tumor models. Results: We show that glioma stem cell (GSC) formation and tumor propagation involve the crosstalk between multiple epigenetic mechanisms, resulting in the repression of the miRNAs that regulate PRC2 function and histone H3 lysine 27 tri-methylation (H3K27me3). We also identified miR-217-5p as an EZH2 regulator repressed in GSCs and showed that miR-217-5p reconstitution using advanced nanoparticle formulations re-activates the PRC2-repressed genes, inhibits GSC formation, impairs tumor growth, and enhances the effects of ionizing radiation in an orthotopic model of GBM. Conclusions: These findings suggest that inhibiting PRC2 function by targeting EZH2 with miR-217-5p advanced nanoparticle formulations could have a therapeutic benefit in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a650a5c81b8e5180cf08ed0736baa5ed09de3eb0" target='_blank'>
              miR-217-5p NanomiRs Inhibit Glioblastoma Growth and Enhance Effects of Ionizing Radiation via EZH2 Inhibition and Epigenetic Reprogramming
              </a>
            </td>
          <td>
            Jack Korleski, Sweta Sudhir, Yuan Rui, Christopher Caputo, Sophie Sall, Amanda L Johnson, Harmon Khela, Tanmaya Madhvacharyula, Anisha Rasamsetty, Yunqing Li, B. Lal, Weiqiang Zhou, Karen Smith-Connor, Stephany Tzeng, Jordan Green, J. Laterra, H. Lopez-Bertoni
          </td>
          <td>2024-12-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="T-cell acute lymphoblastic leukaemia (T-ALL) is a rare aggressive haematological malignancy characterised by the clonal expansion of immature T-cell precursors. It accounts for 15% of paediatric and 25% of adult ALL. T-ALL is associated with the overexpression of major transcription factors (TLX1/3, TAL1, HOXA) that drive specific transcriptional programmes and constitute the molecular classifying subgroups of T-ALL. Although the dysregulation of transcription factor oncogenes is frequently associated with chromosomal translocations in T-ALL, epigenetic dysregulation resulting in changes to post-translational modifications of histones has also been reported. This includes non-coding intergenic mutations that form oncogenic neo-enhancers. This review will focus on the known epigenetically activating intergenic mutations reported in T-ALL, and will discuss the wider implications of neo-enhancer mutations in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12f059393054a743b5ecf3a79dba4386f2b175eb" target='_blank'>
              Neo-enhancers in T-cell acute lymphoblastic Leukaemia (T-ALL) and beyond.
              </a>
            </td>
          <td>
            Charlotte Smith, V. Asnafi, A. Touzart
          </td>
          <td>2025-01-03</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/743e24a828ab3b6aa71556c7a8a2010607a05574" target='_blank'>
              Classifications of triple-negative breast cancer: insights and current therapeutic approaches
              </a>
            </td>
          <td>
            Ziqi Chen, Yumeng Liu, Minchuan Lyu, Chi Ho Chan, Meiheng Sun, Xin Yang, Shuangying Qiao, Zheng Chen, Sifan Yu, Meishen Ren, Aiping Lu, Ge Zhang, Fangfei Li, Yuanyuan Yu
          </td>
          <td>2025-02-01</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="
 Kirsten Rat Sarcoma Virus mutation and its complications have been a major concern. Limitations to drawing its accurate configuration, its complications; mutates and thermodynamic properties, have limited their effectiveness as a molecular diagnostic tool. To counteract these challenges, over the years a different set of sophisticated Immunotherapeutic approaches have been adopted such as Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR), Caspase Protein System (CAS), and Chimeric Antigen Receptor-T cell (CAR-T). Some of them have already managed approval of the Food and Drug Administration. One of the primary goals of CRISPR/CAS technology is to correct mutations within the genetic sequence, thereby potentially repairing defective genes like KRAS. Meanwhile, CAR-T immunotherapies capitalise on genetically modified T cells. In comparison to traditional therapies, immunotherapies can offer greater safety and precision. However, their efficacy is still questionable while treating solid tumours. Here authors have covered the various key targets such as p53, p21, WNT, and RAS, their success stories over the years and have proposed the importance of targets like KRAS oncogene to manage a better output. KRAS is a RAS family oncoprotein responsible for the development of 85% of different cancers. The missense mutations at the various KRAS hotspots can lead to the development of lung and Pancreatic Cancers. Some of the most recently conducted revolutionary studies synthesize CAR-NK, CAR-M, and CRISPR-Cas9/3/10/Novel CAS systems, as well as their prospective molecular combination to target the KRAS. The purpose of this study is to provide a present status of these therapies, limitations, clinical trials, and a proposal to sketch a combinatorial impact via the PRISMA model for 62 different manuscripts (2001-2024). This format aligns and organizes everything in Preferred Reporting Items for Systematic Reviews and Meta-Analyses (PRISMA) guidelines, providing a clear and systematic infographic that represents the process, including stages of selection, screening, and documentation.
 Citation Format: SDanish Kadir. Harnessing the dual therapeutic approach of CRISPR/CAS9 and CAR-T cell technologies in KRAS mutated solid tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B023.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e14aedd9264aefb4bd305bff7eb38afd48f109be" target='_blank'>
              Abstract B023: Harnessing the dual therapeutic approach of CRISPR/CAS9 and CAR-T cell technologies in KRAS mutated solid tumors
              </a>
            </td>
          <td>
            SDanish Kadir
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Colorectal cancer is a common and fatal disease that affects many people globally. CRC is classified as the third most prevalent cancer among males and the second most frequent cancer among females worldwide. The purpose of this article is to examine how personalized medicine might be used to treat colorectal cancer. The classification of colorectal cancer based on molecular profiling, including the detection of significant gene mutations, genomic instability, and gene dysregulation, is the main topic of this discussion. Advanced technologies and biomarkers are among the detection methods that are explored, demonstrating their potential for early diagnosis and precise prognosis. In addition, the essay explores the world of treatment possibilities by providing light on FDA-approved personalized medicine solutions that provide individualized and precise interventions based on patient characteristics. This article assesses targeted treatments like cetuximab and nivolumab, looks at the therapeutic usefulness of biomarkers like microsatellite instability (MSI) and circulating tumor DNA (ctDNA), and investigates new approaches to combat resistance. Through this, our review provides a thorough overview of personalized medicine in the context of colorectal cancer, ultimately highlighting its potential to revolutionize the field and improve patient care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b80ad33c609a3090e37d7111c49bfa155aa4a9e" target='_blank'>
              Personalized medicine in colorectal cancer: a comprehensive study of precision diagnosis and treatment.
              </a>
            </td>
          <td>
            Fatemeh Gila, Somayeh Khoddam, Zahra Jamali, Mohmmad Ghasemian, Shayan Shakeri, Zeinab Dehghan, Jafar Fallahi
          </td>
          <td>2025-02-09</td>
          <td>Personalized medicine</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract Ovarian cancer is a deadly gynecologic disease with frequent recurrence. Current treatments for patients include platinum-based therapy regimens with PARP inhibitors specific for homologous recombination–deficient high-grade serous ovarian cancers (HGSOC). Despite initial effectiveness, patients inevitably develop disease progression as tumor cells acquire resistance. Toward the development of new therapeutic avenues, we describe a gene amplification involving both heat shock factor 1 (HSF1) and MYC, wherein these two genes are co-amplified in more than 30% of patients with HGSOC. We further found that HSF1 and MYC transcriptional activities were highly correlated with human HGSOC tumors and cell lines, suggesting that they may cooperate in the disease. CUT&RUN sequencing for HSF1 and MYC revealed overlapping HSF1 and MYC binding throughout the genome. Moreover, the binding peaks of both transcription factors in HGSOC cells were nearly identical, and a protein–protein interaction between HSF1 and MYC was detected, supporting molecular cooperation. Supporting a functional cooperation of these two transcription factors, the growth of HGSOC cells with the co-amplification was dependent on both HSF1 and MYC. To identify a therapeutic target that could take advantage of this unique HSF1 and MYC dependency, polo-like kinase 1 (PLK1) was correlated with HSF1 and MYC in HGSOC specimens. Targeting PLK1 with volasertib revealed a greater than 200-fold increased potency in HSF1–MYC co-amplified HGSOC cells compared with those with wild-type HSF1 and MYC copy numbers. Although the success of volasertib and other PLK1 inhibitors in clinical trials has been modest, the current study suggests that targeting PLK1 using a precision medicine approach based on HSF1–MYC co-amplification as a biomarker in HGSOC would improve therapy response and patient outcomes. Significance: We show that HSF1 and MYC genes are co-amplified in more than 30% of HGSOC and demonstrate that HSF1 and MYC functionally cooperate to drive the growth of HGSOC cells. This work provides the foundation for HSF1 and MYC co-amplification as a biomarker for treatment efficacy of the polo-like kinase 1 inhibitor volasertib in HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b37b1725a2f0ac79cd573a1bb5f9e94eafaf83ac" target='_blank'>
              MYC and HSF1 Cooperate to Drive Sensitivity to Polo-like Kinase 1 Inhibitor Volasertib in High-grade Serous Ovarian Cancer
              </a>
            </td>
          <td>
            Imade Williams, Matthew O'Malley, Haddie DeHart, Bobby Walker, Vrushabh Ulhaskumar, Pranav Jothirajah, Haimanti Ray, Lisa M Landrum, Joe R. Delaney, K. Nephew, Richard L. Carpenter
          </td>
          <td>2025-01-20</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c108f9b6c451482458b3d22e6cbf9ba4232a44ff" target='_blank'>
              Repeated ionizing radiation exposure induces TRIP13 expression, conferring radioresistance in lung cancer cells
              </a>
            </td>
          <td>
            Wenqing Liu, Qijing Lei, A. V. van Pelt, G. Hamer
          </td>
          <td>2025-01-06</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="DNA methylation is an epigenetic mechanism of gene silencing crucial to the regulation of gene expression in normal physiological events such as differentiation and X-inactivation. However, aberrant silencing of regulatory genes can contribute to oncogenic transformation, further perpetuated due to the heritability of these changes down the cell line. Though aberrant DNA methylation is implicated across cancer types, epigenetic therapy with DNA methyltransferase 1 (DNMT1) depleting drugs is only approved in the treatment of hematological cancers. This limitation is due to the drugs’ high toxicity, a byproduct of their mechanism as nucleoside analogs. Identifying less toxic nucleoside analogs or developing non-nucleoside analogs is necessary to expand the application of epigenetic therapy. This review examines the existing nucleoside and non-nucleoside DNMT1-inhibitors at various stages of preclinical and clinical development, and overviews prospective applications of epigenetic therapy in solid tumors as monotherapy and combined therapy. The list of drugs reported in this review is non-comprehensive as it excludes primary research on drugs last tested prior to the FDA approval of azacitidine and drugs not tested or inviable in human cells. This is to limit the survey to studies that intend to improve upon the pharmacological profile of the approved drugs. Of particular importance are the recently developed DNMT1-inhibitor (DNMT1-i) GSK analogs and the advancements in protein modeling that have elucidated their mechanism to the greatest precision yet. Not yet discussed in length in secondary literature, these findings provide a clearer model for the development of more specific DNMT-is. Furthermore, evidence showing enhanced efficacy of DNMT1 inhibitors when combined with DNA damaging agents identifies epigenetic combination therapy as a pertinent focus of future research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f220480bdc96c0e4bd612e7cc2d97304ce081345" target='_blank'>
              Nucleoside and Non-Nucleoside DNA Methyltransferase 1 Inhibitors in Epigenetic and Combination Therapies in Cancer: A Scoping Review
              </a>
            </td>
          <td>
            Nila Mohan
          </td>
          <td>2025-01-31</td>
          <td>Undergraduate Research in Natural and Clinical Science and Technology (URNCST) Journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c56d0e33616064b81832853efdda25a0062176f" target='_blank'>
              Exploring Epigenetic Complexity in Regulation of Hematopoietic Stem Cells Niche: A Mechanistic Journey from Normal to Malignant Hematopoiesis.
              </a>
            </td>
          <td>
            N. A. Yusoff, Zariyantey Abd Hamid, Izatus Shima Taib, Siti Razila Abdul Razak, S. B. Budin
          </td>
          <td>2025-01-23</td>
          <td>Advances in experimental medicine and biology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17bb4f23b9e1892130e632d422e0888edddfeef1" target='_blank'>
              Molecular heterogeneity and MYC dysregulation in triple-negative breast cancer: genomic advances and therapeutic implications.
              </a>
            </td>
          <td>
            Priya, Arun Kumar, Dhruv Kumar
          </td>
          <td>2025-01-01</td>
          <td>3 Biotech</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Chromosome aberrations (CAs) play an important role in tumor initiation, followed by metastasis. Short-term peripheral blood lymphocyte (PBL) cultures
using RPMI-1640 media with 5 % FBS were used to evaluate the frequency of structural and numerical chromosome aberrations in breast cancer patients.
Highest frequency of aneuploidy (11.53%) were observed followed by trisomy -21(8.93%) in karyotypes. De-novo mutation involving 12q21 with loss
of 16.38 Mbp DNA fragment is the most relevant finding in breast cancer (BC) patients and has not been reported earlier. Secondly, role of trisomy-21
(8.93%) might have increase genetic susceptibility of disease because of “giant satellites”, are the active sites of rRNA sub unit of 18S and 28S. These active
sites might have increase gene-expression of truncated protein during tumor Significant increase in the frequency of aneuploidy with increased number
of non-homologous chromosome was the striking feature due to increase of non-disjunction event followed by unequal crossing-over and synapse
formation by known environmental factors like arsenic. However, the present study is small, but interesting to explore the etiopathology of BC patients
associated chromosome instability. However, further study is required to confirm these changes, whether these mutations are either familial (inherited)
or spontaneous in nature.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3f684d4b189095eeab5e425ab3b1be62a314e19" target='_blank'>
              Genomic Instability with Non- Homologues Chromosomes in Aneuploid Cell Increase Risk Factor for the Development of Breast Cancer Patients
              </a>
            </td>
          <td>
            A. K. Saxena
          </td>
          <td>2025-01-31</td>
          <td>Journal of Medicine and Healthcare</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22203f23dc18f7e477336e6b95f3919f3954fe27" target='_blank'>
              Mechanisms and therapeutic implications of gene expression regulation by circRNA-protein interactions in cancer
              </a>
            </td>
          <td>
            Nan Zhang, Xinjia Wang, Yu Li, Yiwei Lu, Chengcheng Sheng, Yumeng Sun, Ningye Ma, Yisheng Jiao
          </td>
          <td>2025-01-17</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="PURPOSE
Previous research has shown, that ABC transporters gene expression level can predict the efficacy of therapy. However, other mechanisms of gene activity are rarely considered, especially in non-small cell lung cancer (NSCLC). Thus, the purpose of the work was to assess chromosomal aberrations of all 49 ABC transporters genes and the expression levels of some ABC genes, as well as their correlation with survival.


MATERIALS AND METHODS
The surgical material of 104 patients with NSCLC was used in this study. Treatment included surgery and 3 courses of adjuvant chemotherapy with "platinum doublets". DNA and RNA were isolated from the samples, followed by microarray analysis to assess the expression and chromosomal aberrations (deletions and amplifications) of ABC genes.


RESULTS
Metastatic-free survival (MFS) was higher with ABCC1, ABCC2, and ABCG1 hypoexpression at a statistically significant level (p=0.01). The presence of deletion in ABCB1 correlates with 100% MFS (p=0.001). The survival rates with ABCG1 amplification are not higher than 45% (p<0.0001). ABCA11 deletion is associated with a low MFS rate (38%) versus 91% with normal copy number (p=0.006). ABCB9 analysis showed opposite results, with survival rates of 55% and 91% in the presence of amplification and normal copy number, respectively (p=0.006). ABCC subfamily genes showed a similar result in the presence of amplification, where ABCC3 and ABCC10 account for 64% and 60% survival, respectively (p=0.005, p=0.01).


CONCLUSION
Thus, not only expression but also chromosomal aberrations were found to be associated with patient survival. These findings could be a potential marker of metastatic-free survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1581dfdb2ff097ba8f6291fda2517229cc004d6d" target='_blank'>
              Influence of DNA Copy Number Aberrations in ABC Transporter Family Genes on the Survival of Patients with Primary Operatable Non-Small Cell Lung Cancer.
              </a>
            </td>
          <td>
            Matvey Mikhailovich Tsyganov, M. Ibragimova, E. Rodionov, Anastavia Alekseevna Frolova, Irina Aleksandrovna Tsydenova, E. A. Lutzkaya, S. Miller
          </td>
          <td>2025-01-31</td>
          <td>Current cancer drug targets</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cd39dfdcf319f5d953557bf73c2a9a4d9323e2e" target='_blank'>
              Long-term safety of lentiviral or gammaretroviral gene-modified T cell therapies.
              </a>
            </td>
          <td>
            J. Jadlowsky, Elizabeth O Hexner, A. Marshall, Stephan A Grupp, N. Frey, James L Riley, E. Veloso, Holly McConville, Walter Rogal, Cory Czuczman, Wei-Ting Hwang, Yimei Li, R. Leskowitz, Olivia Farrelly, J. Karar, Shannon Christensen, Julie Barber-Rotenberg, A. Gaymon, Naomi Aronson, Wendy Bernstein, J. Melenhorst, A. Roche, John K Everett, Sonja A. Zolnoski, Alexander G McFarland, Shantan Reddy, Angelina Petrichenko, Emma J Cook, Carole Lee, Vanessa Gonzalez, Kathleen Alexander, I. Kulikovskaya, Ángel Ramírez-Fernández, Janna C Minehart, M. Ruella, Saar Gill, S. Schuster, Adam D. Cohen, A. Garfall, Payal D Shah, David L. Porter, S. Maude, Bruce L. Levine, Don L Siegel, A. Chew, Stephen McKenna, Lester Lledo, M. Davis, Gabriela Plesa, Friederike Herbst, Edward A. Stadtmauer, Pablo Tebas, Amanda M DiNofia, Andrew Haas, Naomi B. Haas, R. Myers, Donald M. O’Rourke, J. Svoboda, J. Tanyi, R. Aplenc, Jeffrey M. Jacobson, Andrew H Ko, Roger B. Cohen, Carl H. June, Frederic D. Bushman, J. Fraietta
          </td>
          <td>2025-01-20</td>
          <td>Nature medicine</td>
          <td>1</td>
          <td>54</td>
        </tr>

        <tr id="Productive infections of oncogenic human papillomaviruses (HPVs) are closely linked to the differentiation of host epithelial cells, a process that the virus manipulates to create conditions favorable to produce virion progeny. This viral interference involves altering the expression of numerous host genes. Among these, proprotein convertases (PCs) have emerged as potential oncogenes due to their central role in cellular functions. Using RT-qPCR, aberrant PC gene expression was detected across the progression from early HPV infection stages to cancer. These findings demonstrated a progressive disruption of normal PC expression profiles, with FURIN consistently downregulated and other PCs upregulated, transitioning from the episomal stage to neoplastic and carcinoma phenotypes. This pattern of dysregulation was distinct from the broader trends observed in a variety of cancer types through bioinformatic analysis of publicly available transcriptomic datasets, where FURIN expression was predominantly upregulated compared to other PCs. Further bioinformatic investigations revealed a correlation between PC gene expression and cancer phenotype diversity, suggesting a potential link between the loss of normal PC gene expression patterns and the progression of HPV infections toward malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f01d2dc341bbc570cc91f7701d59ca7b10ccc365" target='_blank'>
              Dysregulation of FURIN and Other Proprotein Convertase Genes in the Progression from HPV Infection to Cancer
              </a>
            </td>
          <td>
            Gonzalo Izaguirre, Natalia Zirou, Craig Meyers
          </td>
          <td>2025-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background/Objectives: Lung cancer ranks as the leading cause of cancer-related deaths globally and is highly associated with cisplatin resistance due to both intrinsic and extrinsic mechanisms. Proliferating Cell Nuclear Antigen (PCNA) plays a critical role in molecular processes, such as DNA replication and repair, chromatin structure maintenance, and cell cycle progression. PCNA is known as a molecular marker for proliferation and an excellent inhibition target to shut down highly proliferative cells. One of the mechanisms of cisplatin resistance is the increase in DNA repair, and studies have reported an association between PCNA, lung cancer, and cisplatin treatment. The present study aimed to characterize the absence of PCNA in A549 lung adenocarcinoma cells. Methods: Employing a CRISPR/Cas9 gene-editing approach, we generated a monoclonal cell culture, termed PKO (PCNA knockout). Results: PKO cells exhibited a residual PCNA expression, significantly decreased clonogenic potential and ubiquitylation at K164 residue. IC50 assay suggested that PKO cells could not acquire cisplatin resistance when compared to PX. After cisplatin treatment, PKO cells presented impaired ubiquitylation and did not have increased STAT3 phosphorylation (Tyr705), a previously characterized mechanism of cisplatin resistance. Conclusions: We suggest that PCNA participates in cisplatin resistance in A549, partially by DNA damage tolerance through failure on PCNA monoubiquitylation, and its inhibition may be an approach to circumvent cisplatin resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb4e12fe22d095e0fcb48503db24d12eb54e726a" target='_blank'>
              Partial Proliferating Cell Nuclear Antigen Functional Knockout Impairs Cisplatin Resistance and Clonogenic Potential in Lung Adenocarcinoma Cells
              </a>
            </td>
          <td>
            A. P. Morelli, Nathalia Quintero-Ruiz, M. Mancini, I. Pavan, Isabelle Lima Flores, Luiz Guilherme Salvino Silva, M. B. Severino, R. M. N. Bezerra, F. Simabuco
          </td>
          <td>2025-02-02</td>
          <td>DNA</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Post-translational modifications (PTMs) of proteins dynamically build the buffering and adapting interface between oncogenic mutations and environmental stressors, on the one hand, and cancer cell structure, functioning, and behavior. Aberrant PTMs can be considered as enabling characteristics of cancer as long as they orchestrate all malignant modifications and variability in the proteome of cancer cells, cancer-associated cells, and tumor microenvironment (TME). On the other hand, PTMs of proteins can enhance anticancer mechanisms in the tumoral ecosystem or sustain the beneficial effects of oncologic therapies through degradation or inactivation of carcinogenic proteins or/and activation of tumor-suppressor proteins. In this review, we summarized and analyzed a wide spectrum of PTMs of proteins involved in all regulatory mechanisms that drive tumorigenesis, genetic instability, epigenetic reprogramming, all events of the metastatic cascade, cytoskeleton and extracellular matrix (ECM) remodeling, angiogenesis, immune response, tumor-associated microbiome, and metabolism rewiring as the most important hallmarks of cancer. All cancer hallmarks develop due to PTMs of proteins, which modulate gene transcription, intracellular and extracellular signaling, protein size, activity, stability and localization, trafficking, secretion, intracellular protein degradation or half-life, and protein–protein interactions (PPIs). PTMs associated with cancer can be exploited to better understand the underlying molecular mechanisms of this heterogeneous and chameleonic disease, find new biomarkers of cancer progression and prognosis, personalize oncotherapies, and discover new targets for drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22300c2f6fc124353b32d558bbea365d47496670" target='_blank'>
              Post-Translational Modifications of Proteins Orchestrate All Hallmarks of Cancer
              </a>
            </td>
          <td>
            Pathea Bruno, Aneeta Arshad, Maria-Raluca Gogu, Natalie Waterman, Rylie Flack, Kimberly Dunn, Costel C. Darie, A. Neagu
          </td>
          <td>2025-01-01</td>
          <td>Life</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/adb0953d261751e4de7ab51c018f54f11e89daf6" target='_blank'>
              p53-loss induced prostatic epithelial cell plasticity and invasion is driven by a crosstalk with the tumor microenvironment
              </a>
            </td>
          <td>
            Darya Yanushko, Beatriz German Falcon, Rana El Bizri, Despoina Pervizou, Robin Dolgos, Céline Keime, Tao Ye, C. Thibault-Carpentier, Clémentine Le Magnen, S. Henri, Gilles Laverny, Daniel Metzger
          </td>
          <td>2025-01-26</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Gene therapy offers promising potential as an efficacious and long-lasting therapeutic option for genetic conditions, by correcting defective mutations using engineered vectors to deliver genetic material to host cells. Among these vectors, adeno-associated viruses (AAVs) stand out for their efficiency, versatility, and safety, making them one of the leading platforms in gene therapy. The enormous potential of AAVs has been demonstrated through their use in over 225 clinical trials and the FDA’s approval of six AAV-based gene therapy products, positioning these vectors at the forefront of the field. This review highlights the evolution and current applications of AAVs in gene therapy, focusing on their clinical successes, ongoing developments, and the manufacturing processes required for the rapid commercial growth anticipated in the AAV therapy market. It also discusses the broader implications of these advancements for future therapeutic strategies targeting more complex and multi-systemic conditions and biological processes such as aging. Finally, we explore some of the major challenges currently confronting the field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e13c57c4e988a0cda2f0afdf15f687ca2f4bac61" target='_blank'>
              Adeno-Associated Virus Vectors: Principles, Practices, and Prospects in Gene Therapy
              </a>
            </td>
          <td>
            Limor Zwi-Dantsis, Saira Mohamed, G. Massaro, Emad Moeendarbary
          </td>
          <td>2025-02-09</td>
          <td>Viruses</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22c8fd6c1c580a3737a18f3485f8040f1c52d76c" target='_blank'>
              TRIM28-dependent developmental heterogeneity determines cancer susceptibility through distinct epigenetic states.
              </a>
            </td>
          <td>
            Ilaria Panzeri, Luca Fagnocchi, S. Apostle, Megan Tompkins, Emily Wolfrum, Z. Madaj, G. Hostetter, Yanqing Liu, Kristen Schaefer, Chih-Hsiang Yang, Alexis Bergsma, Anne Drougard, Erez Dror, Darrell P Chandler, Daniel Schramek, Timothy J. Triche, J. Pospisilik
          </td>
          <td>2025-01-24</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1d734aed102ed089c1bedbb1c15be484c90edb1" target='_blank'>
              tRNA-derived small RNAs: their role in the mechanisms, biomarkers, and therapeutic strategies of colorectal cancer
              </a>
            </td>
          <td>
            Bo Zhang, Yanru Pan, Zhe Li, Kefeng Hu
          </td>
          <td>2025-01-13</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a highly heterogeneous gastrointestinal malignancy. Despite significant advances in molecular targeted therapies for CRC in recent years, the increase in the overall survival rates for CRC patients remains limited. Therefore, there is an urgent need to explore novel drug targets. Herein, we show that heparin binding growth factor (HDGF) is highly expressed in CRC, and that its overexpression is associated with a poor disease-free interval. Additionally, we reveal that HDGF knockout reduces proliferation, migration, and invasion, while enhancing apoptosis in CRC cells, thereby validating HDGF as a potential therapeutic target for CRC. Mechanistically, we found that HDGF modulates DNA damage response and, by recruiting C-terminal binding protein-interacting protein (CtIP), it facilitates homologous recombination repair to influence CRC drug sensitivity. Furthermore, we propose that HDGF may serve as a recognition protein for H3K36me3, participating in the repair of damaged transcriptionally active genes, thus maintaining genomic stability in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f394455983264a89c22ede32239ab6721086ed77" target='_blank'>
              HDGF Knockout Suppresses Colorectal Cancer Progression and Drug Resistance by Modulating the DNA Damage Response
              </a>
            </td>
          <td>
            Riya Su, Qin Wang, Qun Hu, Wendurige, Kexin Li, Changshan Wang, Liang Tao
          </td>
          <td>2025-02-14</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Motivation Human mitochondrial DNA (mtDNA) mutations are causally implicated in maternally inherited mitochondrial respiratory disorders; however, the role of somatic mtDNA mutations in both late-onset chronic diseases and cancer remains less clear. Although recent advances in mtDNA base editing technology have the potential to model and characterize many of these mutations, current editing approaches are complicated by the potential for multiple unintentional edits (bystanders) that are only identifiable through empirical ‘trial and error’, thereby sacrificing valuable time and effort towards suboptimal construct development. Results We developed MitoEdit, a novel tool that incorporates empirical base editor patterns to facilitate identification of optimal target windows and potential bystander edits. MitoEdit allows users to input DNA sequences in a text-based format, specifying the target base position and its desired modification. The program generates a list of candidate target windows with a predicted number of bystander edits and their functional impact, along with flanking nucleotide sequences designed to bind TALE (transcription activator-like effectors) array proteins. In silico evaluations indicate that MitoEdit can predict the majority of bystander edits, thereby reducing the number of constructs that need to be tested empirically. To the best of our knowledge, MitoEdit is the first tool to automate prediction of base edits. Availability and implementation MitoEdit is freely available at Kundu Lab GitHub (https://github.com/Kundu-Lab/mitoedit). Contact Corresponding email: Gang.Wu@stjude.org; Mondira.Kundu@stjude.org Supplementary information Supplementary data are available at Bioinformatics online.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0eb09ec051ad08fe864f50953db873f84673f691" target='_blank'>
              MitoEdit: a pipeline for optimizing mtDNA base editing and predicting bystander effects
              </a>
            </td>
          <td>
            Devansh Shah, K. McCastlain, Ti-Cheng Chang, Gang Wu, M. Kundu
          </td>
          <td>2025-01-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Kinase-related gene fusion and point mutations play pivotal roles as drivers in cancer, necessitating optimized, targeted therapy against these alterations. The efficacy of molecularly targeted therapeutics varies depending on the specific alteration, with great success reported for such therapeutics in the treatment of cancer with kinase fusion proteins. However, the involvement of actionable alterations in solid tumors, especially regarding kinase fusions, remains unclear. Therefore, in this study, we aimed to compare the number of actionable alterations in patients with tyrosine or serine/threonine kinase domain fusions, mutations, and copy number alterations (CNAs). We analyzed 613 patients with 40 solid cancer types who visited our division between June 2020 and April 2024. Furthermore, to detect alterations involving multiple-fusion calling, we performed comprehensive genomic sequencing using FoundationOne® companion diagnostic (F1CDx) and FoundationOne® Liquid companion diagnostic (F1LCDx). Patient characteristics and genomic profiles were analyzed to assess the frequency and distribution of actionable alterations across different cancer types. Notably, 44 of the 613 patients had fusions involving kinases, transcriptional regulators, or tumor suppressors. F1CDx and F1LCDx detected 13 cases with kinase-domain fusions. We identified 117 patients with kinase-domain mutations and 58 with kinase-domain CNAs. The number of actionable alterations in patients with kinase-domain fusion, mutation, or CNA (median [interquartile range; IQR]) was 2 (1–3), 5 (3–7), and 6 (4–8), respectively. Patients with kinase fusion had significantly fewer actionable alterations than those with kinase-domain mutations and CNAs. However, those with fusion involving tumor suppressors tended to have more actionable alterations (median [IQR]; 4 [2–9]). Cancers with kinase fusions exhibited fewer actionable alterations than those with kinase mutations and CNAs. These findings underscore the importance of detecting kinase alterations and indicate the pivotal role of kinase fusions as strong drivers of cancer development, highlighting their potential as prime targets for molecular therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e5249feb71c9f79a3c5d96e6debe6951cd92ec6" target='_blank'>
              Comparison of actionable alterations in cancers with kinase fusion, mutation, and copy number alteration
              </a>
            </td>
          <td>
            S. Suzuki, T. Akahane, A. Tanimoto, M. Higashi, I. Kitazono, M. Kirishima, M. Nishigaki, T. Ikeda, S. Kanemitsu, J. Nakazawa, E. Akahane, H. Nishihara, Kimiharu Uozumi, M. Yoshimitsu, K. Ishitsuka, S.-i. Ueno
          </td>
          <td>2025-01-23</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Simple Summary RAC1 is an oncogene. As of today, the biological importance and potential druggability of RAC1 genomic aberrations remain largely underexplored in head and neck cancer. Findings from this proof-of-principle study showed that RAC1-amplified head and neck cancer patient-derived xenografts (but not RAC1-diploidy tumors), and engineered RAC1-A159V HNSCC xenografts (but not engineered RAC1-P29S HNSCC xenografts) were both sensitive to Rac targeting in vivo with a preclinical Rac inhibitor, EHop-16. These findings provide evidence supportive of the precision drugging of RAC1-amp or RAC1-A159V-mutated head and neck cancer patients (~5% cases of HNSCC, based on the Cancer Genome Atlas (TCGA) data) with new Rac1 inhibitors in clinical settings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08f87fb2681bb35d44a456b0d36d08cad1ca1b27" target='_blank'>
              RAC1-Amplified and RAC1-A159V Hotspot-Mutated Head and Neck Cancer Sensitive to the Rac Inhibitor EHop-016 In Vivo: A Proof-of-Concept Study
              </a>
            </td>
          <td>
            H. H. Chan, Hoi-Lam Ngan, Yuen-Keng Ng, Chun-Ho Law, P. H. Y. Poon, Ray Wai Wa Chan, Kwok-Fai Lau, Wenyin Piao, Hui Li, Lan Wang, J. Chan, Yudan Su, Thomas Chung-Kit Yeung, E. Wong, Angela Wing Tung Li, K. Verhoeft, Yuchen Liu, Yukai He, S.K.W. Tsui, Gordon B. Mills, V. Lui
          </td>
          <td>2025-01-23</td>
          <td>Cancers</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e9558944d48747d281f714408c8c756ade188a3" target='_blank'>
              Genetic and epigenetic characterization of sarcoma stem cells across subtypes identifies EZH2 as a therapeutic target
              </a>
            </td>
          <td>
            Edmond O'Donnell, Maria Muñoz, Ryan Davis, Jessica Bergonio, R. L. Randall, Clifford Tepper, Janai R. Carr-Ascher
          </td>
          <td>2025-01-09</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="As a ubiquitous human pathogen, the Epstein–Barr virus (EBV) has established lifelong persistent infection in about 95% of the adult population. The EBV infection is associated with approximately 200,000 human cancer cases and 140,000 deaths per year. The presence of EBV in tumor cells provides a unique advantage in targeting the viral genome (also known as episome), to develop anti-cancer therapeutics. In this review, we summarize current strategies targeting the viral episome in cancer cells. We also highlight emerging technologies, such as clustered regularly interspersed short palindromic repeat (CRISPR)-based gene editing or activation, which offer promising avenues for selective targeting of the EBV episome for anti-cancer therapy. We discuss the challenges, limitations, and future perspectives associated with these strategies, including potential off-target effects, anti-cancer efficacy and safety.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f15b2a6c5daab5ecad16d68d037359039237312" target='_blank'>
              Targeting EBV Episome for Anti-Cancer Therapy: Emerging Strategies and Challenges
              </a>
            </td>
          <td>
            Febri Gunawan Sugiokto, Renfeng Li
          </td>
          <td>2025-01-01</td>
          <td>Viruses</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Chemotherapy often kills a large fraction of cancer cells but leaves behind a small population of drug- tolerant persister cells. These persister cells survive drug treatments through reversible, non-genetic mechanisms and cause tumour recurrence upon cessation of therapy. Here, we report a drug tolerance mechanism regulated by the germ-cell-specific H3K4 methyltransferase PRDM9. Through histone proteomic, transcriptomic, lipidomic, and ChIP-sequencing studies combined with CRISPR knockout and phenotypic drug screen, we identified that chemotherapy-induced PRDM9 upregulation promotes metabolic rewiring in glioblastoma stem cells, leading to chemotherapy tolerance. Mechanistically, PRDM9-dependent H3K4me3 at cholesterol biosynthesis genes enhances cholesterol biosynthesis, which persister cells rely on to maintain homeostasis under chemotherapy- induced oxidative stress and lipid peroxidation. PRDM9 inhibition, combined with chemotherapy, resulted in strong anti-cancer efficacy in preclinical glioblastoma models, significantly enhancing the magnitude and duration of the antitumor response by eliminating persisters. These findings demonstrate a previously unknown role of PRDM9 in promoting metabolic reprogramming that enables the survival of drug-tolerant persister cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44ca423a805e163b74f1186c734661fa71cbd4de" target='_blank'>
              Histone methyltransferase PRDM9 promotes survival of drug-tolerant persister cells in glioblastoma
              </a>
            </td>
          <td>
            George L. Joun, Emma G. Kempe, Brianna Chen, Jayden Sterling, Ramzi H. Abbassi, W. Daniel du Preez, A. Recasens, Teleri Clark, Tian Y. Du, Jason K.K. Low, Hani Kim, Pengyi Yang, Jasmine Khor, Monira Hoque, Dinesh C. Indurthi, M. Kuchibhotla, Ranjith Palanisamy, William T Jorgensen, Andrew P. Montgomery, Jennifer R. Baker, S. Higginbottom, E. Tomaskovic-Crook, J. M. Crook, Lipin Loo, Bryan W. Day, G. G. Neely, Ernesto Guccione, Terrance G. Johns, Michael Kassiou, A. S. Don, L. Munoz
          </td>
          <td>2025-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52a090853d43190aed49536f43b138c297c3e652" target='_blank'>
              Telomerase in cancer- ongoing quest and future discoveries.
              </a>
            </td>
          <td>
            Apurwa Mishra, Trupti N Patel
          </td>
          <td>2025-01-25</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Breast cancers of the IntClust-2 type, characterized by amplification of a small portion of chromosome 11, have a median survival of only five years. Several cancer-relevant genes occupy this portion of chromosome 11, and it is thought that overexpression of a combination of driver genes in this region is responsible for the poor outcome of women in this group. In this study we used a gene editing method to knock out, one by one, each of 198 genes that are located within the amplified region of chromosome 11 and determined how much each of these genes contributed to the survival of breast cancer cells. In addition to well-known drivers such as CCND1 and PAK1, we identified two different genes (SERPINH1 and P4HA3), that encode proteins involved in collagen synthesis and organization. Using both in vitro and in vivo functional analyses, we determined that P4HA3 and/or SERPINH1 provide a critical driver function on IntClust-2 basic processes, such as viability, proliferation, and migration. Inhibiting these enzymes via genetic or pharmacologic means reduced collagen synthesis and impeded oncogenic signaling transduction in cell culture models, and a small-molecule inhibitor of P4HA3 was effective in treating 11q13 tumor growth in an animal model. As collagen has a well-known association with tissue stiffness and aggressive forms of breast cancer, we believe that the two genes we identified provide an opportunity for a new therapeutic strategy in IntClust-2 breast cancers. Implications: Breast cancers with 11q13/14 chromosomal amplifications may be vulnerable to inhibitors of collagen synthesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bd01b0833b0dda4d17087283c1b41d82a5f1780" target='_blank'>
              Identifying and targeting key driver genes for collagen production within the 11q13/14 breast cancer amplicon.
              </a>
            </td>
          <td>
            D. Araiza-Olivera, T. Prudnikova, Cristina Uribe-Alvarez, Kathy Q Cai, J. Franco-Barraza, Jesus M Dones, Ronald T. Raines, J. Chernoff
          </td>
          <td>2025-01-21</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a63030f8f4a5318cca9879e32ba13ec7a2af3ed4" target='_blank'>
              Chromosome mis-segregation triggers cell cycle arrest through a mechanosensitive nuclear envelope checkpoint
              </a>
            </td>
          <td>
            Solène Hervé, Andrea Scelfo, Gabriele Bersano Marchisio, Marine Grison, Kotryna Vaidžiulytė, Marie Dumont, Annapaola Angrisani, Adib Keikhosravi, Gianluca Pegoraro, Mathieu Deygas, Guilherme P. F. Nader, Anne-Sophie Macé, Matteo Gentili, Alice Williart, N. Manel, Matthieu Piel, Y.A. Miroshnikova, D. Fachinetti
          </td>
          <td>2025-01-08</td>
          <td>Nature Cell Biology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Recently formed allopolyploid species offer unprecedented insights into the early stages of polyploid evolution. This review examines seven well-studied neopolyploids (we use 'neopolyploid' to refer to very recently formed polyploids, i.e. during the past 300 years), spanning different angiosperm families, exploring commonalities and differences in their evolutionary trajectories. Each neopolyploid provides a unique case study, demonstrating both shared patterns, such as rapid genomic and phenotypic changes, and unique responses to hybridization and genome doubling. While previous studies of these neopolyploids have improved our understanding of polyploidy, significant knowledge gaps remain, highlighting the need for further research into the varied impacts of whole-genome duplication on gene expression, epigenetic modifications, and ecological interactions. Notably, all of these neopolyploids have spontaneously arisen due to human activity in natural environments, underscoring the profound consequences of polyploidization in a rapidly changing world. Understanding the immediate effects of polyploidy is crucial not only for evolutionary biology but also for applied practices, as polyploidy can lead to novel traits, as well as stress tolerance and increased crop yields. Future research directions include investigating the genetic and epigenetic mechanisms underlying polyploid evolution, as well as exploring the potential of neopolyploids for crop improvement and environmental adaptation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac5a204f000a3e9de1ebd9bce03b0ead5230c52b" target='_blank'>
              Natural neopolyploids: a stimulus for novel research.
              </a>
            </td>
          <td>
            P. Edger, D. Soltis, Shunsuke Yoshioka, Mario Vallejo-Marín, Rie Shimizu‐Inatsugi, Kentaro K. Shimizu, A. Salmon, Simon Hiscock, Malika Ainouche, Pamela S Soltis
          </td>
          <td>2025-02-14</td>
          <td>The New phytologist</td>
          <td>0</td>
          <td>142</td>
        </tr>

        <tr id="IDH-mutant low-grade gliomas (LGGs) are slow-growing brain tumors that frequently progress to aggressive high-grade gliomas that have dismal outcomes. In a recent study, Wu and colleagues provide critical insights into the mechanisms underlying malignant progression by analyzing single-cell gene expression and chromatin accessibility across different tumor grades. Their findings support a two-phase model: in early stages, tumors are primarily driven by oligodendrocyte precursor-like cells and epigenetic alterations that silence tumor suppressors like CDKN2A and activate oncogenes such as PDGFRA. As the disease advances, the tumors become sustained by more proliferative neural precursor-like cells, where genetic alterations, including PDGFRA, MYCN, and CDK4 amplifications and CDKN2A/B deletion, drive tumor progression. The study further highlights a dynamic regulation of interferon (IFN) signaling during progression. In low-grade IDH-mutant gliomas, IFN responses are suppressed through epigenetic hypermethylation, which can be reversed with DNMT1 inhibitors or IDH inhibitors, leading to reactivation of the IFN pathway. In contrast, higher-grade gliomas evade IFN signaling through genetic deletions of IFN gene clusters. These findings emphasize a broader epigenetic-to-genetic shift in oncogenic regulation that drives glioma progression, provides a valuable framework for understanding the transition from indolent tumors to lethal malignancies, and has implications for therapy and clinical management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/598e5f4259516cb6e86cc401bbe6fe663aa489ab" target='_blank'>
              Switching Drivers: Epigenetic Rewiring to Genetic Progression in Glioma.
              </a>
            </td>
          <td>
            K. Drucker, Robert B. Jenkins, Daniel Schramek
          </td>
          <td>2024-12-30</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Comprehensive genetic analysis of tumors with exome or whole genome sequencing has enabled the identification of the genes that are recurrently mutated in cancer. This has stimulated a series of exciting advances over the past 15 years, guiding us to new molecular biomarkers and therapeutic targets among the common mature B-cell neoplasms. In particular, diffuse large B-cell lymphoma (DLBCL), follicular lymphoma (FL) and Burkitt lymphoma (BL) have each been the subject of considerable attention in this field. Currently, more than 850 genes have been reported as targets of protein-coding mutations in at least one of these entities. To reduce this to a manageable size, we describe a systematic approach to prioritize and categorize these genes, based on the quality and type of supporting data. For each entity, we provide a list of candidate driver genes categorized into Tier 1 (high-confidence genes), Tier 2 (candidate driver genes), or Tier 3 (lowest-confidence genes). Collectively, this reduces the number of high-confidence genes for these three lymphomas to a mere 144. This further affirms the substantial overlap between the genes relevant in DLBCL and each of FL and BL. These highly curated and annotated gene lists will continue to be maintained as a resource to the community. These results emphasize the extent of the knowledge gap regarding the role of each of these genes in lymphomagenesis. We offer our perspective on how to accelerate the experimental confirmation of drivers using a variety of model systems, using these lists as a guide for prioritizing genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dadf53884a55922c51a797521fda638d01af815c" target='_blank'>
              Audit of B-cell Cancer Genes.
              </a>
            </td>
          <td>
            K. Coyle, K. Dreval, Daniel J Hodson, Ryan D Morin
          </td>
          <td>2025-01-24</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Neuroblastoma (NB) is a highly metastatic pediatric cancer arising from the neural crest lineage. Genetic amplification of the MYCN proto-oncogene is a defining feature of NB, present in about 20% of all cases. The let-7 tumor suppressor microRNA targets the 3’ UTR of MYCN mRNA. We previously demonstrated that the 3’ UTR of MYCN mRNA acquires the ability to sequester let-7 in MYCN-amplified (MA) disease, thus inhibiting its function. This work established that a noncoding element within an oncogenic mRNA can contribute independently to disease pathology and genetic patterning. To further investigate the roles of noncoding RNA elements within the MYCN mRNA, we engineered cells expressing either MYCN-ORF (MYCN open reading frame only), MYCN-GL (full-length MYCN mRNA from the intact genetic locus), and Null-GL (full-length MYCN mRNA variant where EGFP replaces MYCN protein). We observe that all constructs enhance growth compared to controls in vitro and in vivo. MYCN-GL-expressing cells displayed the most robust growth in vitro despite containing multiple regulatory RNA elements. Remarkably, the Null-GL construct induces cells to grow as fast or faster than MYCN-ORF-expressing cells. Animal studies further confirmed these observations, where the Null-GL-driven tumors had the highest incidence and lowest latency, followed by MYCN-GL and then MYCN-ORF. Further, through NGS analysis, let-7, miR-101, and miR-34a targets are enriched in both MYCN-GL and Null-GL expressing cells. Thus, the 3’UTR of MYCN, which is also targeted by these microRNAs, may interact with them in MYCN-GL and Null-GL cells to deliver a protective effect for the mRNA targets of these microRNAs. We also observed more dynamic differential gene expression in the -GL constructs than in ORF and GFP-expressing cells. In addition, MYCN-GL and Null-GL expressing cells are similarly enriched in gene ontology pathways for cancer, RNA metabolism, and microRNA processing pathways as compared to ORF and GFP. Whole genome sequencing also revealed more similarities in copy number variation in MYCN-GL and Null-GL than in ORF and GFP, suggesting that these constructs may provide selective pressure to favor specific CNV patterns. These observations show that full-length MYCN mRNA containing noncoding regulatory elements are more robust drivers of cell growth and oncogenicity than MYCN protein alone and provide insights into the mechanisms of oncogenic contribution. These results open an exciting door for our understanding of NB pathology and genetic patterning and have broad implications for other oncogene-driven cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99c9a5c9ad241e7424dc126cfe9aacbf830aa7c8" target='_blank'>
              Noncoding elements within MYCN mRNA are autonomous drivers of oncogenesis in neuroblastoma
              </a>
            </td>
          <td>
            Vishwa Patel, Lorraine-Rana E. Benhamou, John T. Powers
          </td>
          <td>2025-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Renowned as a highly invasive and lethal tumor derived from neural stem cells in the central nervous system, glioblastoma (GBM) exhibits substantial histopathological variation and genomic complexity, which drive its rapid progression and therapeutic resistance. Alterations in mitochondrial DNA (mtDNA) copy number (CN) serve a crucial role in GBM development and progression, affecting various aspects of tumor biology, including energy production, oxidative stress regulation and cellular adaptability. Fluctuations in mtDNA levels, whether elevated or diminished, can impair mitochondrial function, potentially disrupting oxidative phosphorylation and amplifying reactive oxygen species generation, thereby fueling tumor growth and influencing treatment responses. Understanding the mechanisms of mtDNA-CN variations, and their interplay with genetic and environmental elements in the tumor microenvironment, is essential for advancing diagnostic and therapeutic strategies. Targeting mtDNA alterations could strengthen treatment efficacy, mitigate resistance and ultimately enhance the prognosis of patients with this aggressive brain tumor. The present review summarizes the existing literature on mtDNA alterations, specifically emphasizing variations in mtDNA-CN and their association with GBM by surveying articles published between 1996 and 2024, sourced from databases such as Scopus, PubMed and Google Scholar. In addition, the review provides a brief overview of mitochondrial genome architecture, knowledge regarding the regulation of mtDNA integrity and CN, and how mitochondria significantly impact GBM tumorigenesis. This review further presents information on therapeutic approaches for restoring mtDNA-CN that contribute to optimized mitochondrial function and improved health outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/608231a30fdc2050cf19ab35c688a4cbcbde93d3" target='_blank'>
              Mitochondrial DNA copy number alterations: Key players in the complexity of glioblastoma (Review)
              </a>
            </td>
          <td>
            Abdul Aziz Mohamed Yusoff, Siti Zulaikha Nashwa Mohd Khair, Siti Muslihah Abd Radzak
          </td>
          <td>2025-01-24</td>
          <td>Molecular Medicine Reports</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) represent a small, yet pivotal subpopulation of tumor cells that play significant roles in tumor initiation, progression and therapeutic resistance. Circular RNAs (circRNAs) are a distinct class of RNAs characterized by their closed-loop structures, lacking 5′ to 3′ends. There is growing evidence that circRNAs are integral to the development and regulation of CSCs. Aberrant expression of circRNAs in CSCs can contribute to oncogenic properties and drug resistance. Specifically, oncogenic circRNAs modulate CSC behavior via key signaling pathways, thereby promoting CSC self-renewal and maintenance, as well as tumor progression. This review summarizes the latest research on the functional roles and regulatory mechanisms of circRNAs in CSC behavior and discusses potential applications and challenges of targeting circRNAs in CSCs. Understanding the intricate interactions between circRNAs and CSCs may lead to novel therapeutic strategies that effectively combat treatment resistance and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c01b87d0f590a46f3144a9c75dc4396105c17dca" target='_blank'>
              Circular RNAs in cancer stem cells: Insights into their roles and mechanisms (Review)
              </a>
            </td>
          <td>
            Lunyu Yang, Yuling Yi, Zhu Mei, Dongmei Huang, Sitian Tang, Liyi Hu, Ling Liu
          </td>
          <td>2025-01-23</td>
          <td>International Journal of Molecular Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Horizontal gene transfer (HGT) is a major driver of bacterial evolution, allowing bacteria to acquire new traits from other organisms. It enables the transfer of advantageous traits, such as antibiotic resistance, which can rapidly spread across bacterial populations, affecting public health. Several studies have estimated how widespread HGT can be and suggested that environmental changes are associated with changes in the rate of HGT. It is often considered that environmental changes lead to an increase in HGT rates due to the adaptive benefits of HGT. However, previous studies have been limited to one or a few species and have often focused on specific genes of interest. Here, we systematically investigate the effect of environmental changes on the rate of HGT in a large number of bacterial genomes along a deep phylogeny. We extend the scope of use of existing state-of -the -art methods for HGT inference to infer the transition into new ecosystems and the gain of pathogenicity. Transitions to new environments are associated with a decrease in the rate of HGT, along with the strengthening of purifying selection. This decrease in HGT rates is consistent regardless of genomic relatedness between the donor and recipient. Our results are robust to the use of different methodological frameworks and have implications for understanding the evolution of bacteria and HGT rates in natural microbial communities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/509a20de1480698aa85c82ec7a8080e8f597bcfe" target='_blank'>
              Horizontal gene transfer rates decrease during transitions to new ecosystems
              </a>
            </td>
          <td>
            Swastik Mishra, Martin J. Lercher
          </td>
          <td>2024-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Objectives: The TGF-β signaling pathway is widely acknowledged for its role in various aspects of cancer progression, including cellular invasion, epithelial-mesenchymal transition, and immunosuppression. Immune checkpoint inhibitors (ICIs) and pharmacological agents that target TGF-β offer significant potential as therapeutic options for cancer. However, the specific role of TGF-β in prognostic assessment and treatment strategies for breast cancer (BC) remains unclear. Methods: The Cancer Genome Atlas (TCGA) database was utilized to develop a predictive model incorporating five TGF-β signaling-related genes (TSRGs). The GSE161529 dataset from the Gene Expression Omnibus was employed to conduct single-cell analyses aimed at further elucidating the characteristics of these TSRGs. Additionally, an unsupervised clustering algorithm was applied to categorize BC patients into two distinct groups based on the five TSRGs, with a focus on immune response and overall survival (OS). Further investigations were conducted to explore variations in pharmacotherapy and the tumor microenvironment across different patient cohorts and clusters. Results: The predictive model for BC identified five TSRGs: FUT8, IFNG, ID3, KLF10, and PARD6A. Single-cell analysis revealed that IFNG is predominantly expressed in CD8+ T cells. Consensus clustering effectively categorized BC patients into two distinct clusters, with cluster B demonstrating a longer OS and a more favorable prognosis. Immunological assessments indicated a higher presence of immune checkpoints and immune cells in cluster B, suggesting a greater likelihood of responsiveness to ICIs. Conclusion: The findings of this study highlight the potential of the TGF-β signaling pathway for prognostic classification and the development of personalized treatment strategies for BC patients, thereby enhancing our understanding of its significance in BC prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98f9fd471fec796da5a5b1dc52ea20516e6ffafd" target='_blank'>
              Identification of Molecular Subtypes and Prognostic Features of Breast Cancer Based on TGF-β Signaling-related Genes
              </a>
            </td>
          <td>
            Jia Qu, Meihuan Wang, Yuehua Gao, Hua-Wei Zhang
          </td>
          <td>2025-01-01</td>
          <td>Cancer Informatics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3815b96ede148fbe1798a5178e21eca65c618ab6" target='_blank'>
              FOXM1, a super enhancer-associated gene, is related to poorer prognosis and gemcitabine resistance in pancreatic cancer.
              </a>
            </td>
          <td>
            Jian Jiang, Tianci Shen, Dan Chen, Zihao Dai, Xuelong Wang, Qiang Meng, Zhuo Yang, Di Zhang, Xiaoyi Guo, Jianqiang Xu, Jiangning Gu, Changmiao Wang
          </td>
          <td>2025-02-03</td>
          <td>Cell biochemistry and biophysics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb7e063d7f27e7ff5ba777247b0efc2ec0bcfdb4" target='_blank'>
              Functional analysis of cancer-associated germline risk variants.
              </a>
            </td>
          <td>
            Laura N. Kellman, Poornima H. Neela, Suhas Srinivasan, Z. Siprashvili, Ronald L. Shanderson, Audrey W. Hong, Deepti S Rao, Douglas F Porter, D. Reynolds, Robin M. Meyers, M. Guo, Xue Yang, Yang Zhao, Glenn G Wozniak, L. Donohue, R. Shenoy, Lisa A Ko, Duy T Nguyen, Smarajit Mondal, Omar S. Garcia, L. Elcavage, Ibtihal Elfaki, Nathan S. Abell, Shiying Tao, Christopher M Lopez, Stephen B. Montgomery, Paul A. Khavari
          </td>
          <td>2025-02-17</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8641c47a99cb4e08fcabebc4fec5794fc9e23e2f" target='_blank'>
              Role of CD4+ T cells in cancer immunity: a single-cell sequencing exploration of tumor microenvironment
              </a>
            </td>
          <td>
            Qi An, Li Duan, Yuanyuan Wang, Fuxin Wang, Xiangyu Liu, Chao Liu, Qinyong Hu
          </td>
          <td>2025-02-14</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca8eb632e8dcc4d371aa3266d6271acffef83ed2" target='_blank'>
              The integrated stress response drives MET oncogene overexpression in cancers
              </a>
            </td>
          <td>
            M. Cerqua, Marco Foiani, C. Boccaccio, Paolo M Comoglio, D. Altintas
          </td>
          <td>2025-01-07</td>
          <td>The EMBO Journal</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="How nervous systems coordinate the generation of specific neuron types with gene expression plasticity and how these mechanisms impact cell type evolution is unknown. Here we use Caenorhabditis species to study neuron-type robustness, plasticity and evolution, using VC4 and VC5 cholinergic motoneurons as models. In C. elegans, we found that epigenetic silencing through histone 3 lysine 9 methylation (H3K9me) is necessary to suppress the expression of the serotonin reuptake gene mod-5/Sert and a serotonergic phenotype in these cells. In contrast, we observed that VC4 and VC5 neurons in the Angaria group of species of the Caenorhabditis genus have evolved an intense serotonergic staining. This phenotype is caused by the emergence of a new enhancer in the mod-5/Sert locus, which has been recruited to the ancestral neuron-type gene regulatory network. Enhancer transfer from C. angaria is sufficient to impose a constitutive serotonergic fate in C. elegans. Remarkably, acquiring this new trait modulates egg-laying responses to high levels of exogenous serotonin, which can be found in specific environments. Finally, we discovered that the repression of the serotonergic fate in C. elegans VC4 and VC5 neurons is indeed a plastic trait that can be adjusted in specific environmental growth conditions to elicit egg-laying behaviours similar to those observed in Angaria species. Our work identifies gene regulatory mechanisms that coordinate the generation of robust neuron-type-specific programs with plastic gene expression responses. These findings identify a gene regulatory framework underlying the evolution of neuron-type-specific features and the emergence of novel behaviours.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2315a11171225e8baca8060e06a35244519fbc76" target='_blank'>
              Gene regulatory mechanisms underlying evolutionary adaptations of homologous neuronal cell types
              </a>
            </td>
          <td>
            Andrea Millán-Trejo, Carlos Mora-Martínez, Adrián Tarazona-Sánchez, Carla Lloret-Fernández, Rafael Alis, Antonio Jordán, A. Barrios, Nuria Flames
          </td>
          <td>2025-01-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Breast cancer represents the most common type of cancer in females worldwide. The survival rates for breast cancer patients have been increasing since 1990. However, in 2023 breast cancer is still the second most common cause of malignancy-associated death in women. One decisive reason is the increase of treatment resistance and low therapy response. Therefore, new therapy targets and predictive markers for the response to treatment are needed. The present study analyzed the potential effects triggered by different breast cancer treatments on the transcriptional expression of 12 pre-selected long non-coding (lnc) RNAs and the proliferation markers Cyclin D1 and Ki-67 in six different breast cancer cell lines (BT-474, MDA-MB-231, BT-20, T-47D, SKBR-3 and MCF-7). The results revealed that lncRNA cytoskeleton regulator RNA may be an appropriate biomarker for the response to treatment with both epirubicin and gemcitabine (P<0.001). NF-ĸB interacting lnc RNA may be a marker for therapy response (P<0.001), while HOX transcript antisense RNA overexpression suggested resistance to treatment (P<0.001) with epirubicin. The transcriptional expression of lncRNA BC4 increased during treatment with epirubicin and gemcitabine, which indicated therapy response. Overall, the present data suggested that the aforementioned lncRNAs have a promising potential as biomarkers to detect early therapy response or resistance in and therefore should be analyzed in more detail.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/030e7011040033b6c4ff54f51d9cc9c5158b86f4" target='_blank'>
              Long non‑coding RNA signatures in breast cancer: Properties as biomarkers?
              </a>
            </td>
          <td>
            J. Asberger, Isabell Ge, Benjamin Schmidt, M. Jäger, D. Weiss, K. Berner, T. Erbes, I. Juhasz-Böss, Sebastian Mayer
          </td>
          <td>2025-01-22</td>
          <td>Experimental and Therapeutic Medicine</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Introduction Breast cancer is the most frequently diagnosed malignancy and a leading cause of cancer-related mortality among women worldwide. Although NADPH oxidase 4 (NOX4) has been implicated in various oncogenic processes, its exact function in breast cancer progression, metabolic reprogramming, and immune modulation remains unclear. Methods We used murine 4T1 and EO771 breast cancer models to generate NOX4 knockout (KO) cell lines via CRISPR/Cas9. In vitro assays (cell proliferation, colony formation, wound healing, and Seahorse metabolic analyses) and in vivo orthotopic tumor studies assessed the impact of NOX4 loss. Transcriptomic changes were identified through RNA sequencing and gene set enrichment analysis. We performed MYC knockdown in NOX4 KO cells to investigate its mechanistic role. Flow cytometry characterized tumor-infiltrating immune cells. Finally, NOX4-overexpressing cells were tested for survival benefit and response to dual-checkpoint immunotherapy (anti-PD-1/anti-CTLA-4). Results NOX4 deletion accelerated tumor growth in vivo and enhanced proliferation, colony formation, and migratory capacity in vitro. Metabolic profiling showed that NOX4 KO cells had elevated glycolysis and fatty acid oxidation, along with increased mitochondrial mass. Transcriptomic and enrichment analyses revealed MYC pathway activation in NOX4 KO cells; suppressing MYC reversed these hyperproliferative and metabolic changes. Immunologically, NOX4 KO reduced CD8+ T cell infiltration and function, partially due to lowered CCL11/CCL5 levels, while PD-L1 expression was upregulated. In contrast, NOX4 overexpression improved survival in mice and synergized with checkpoint blockade, demonstrating a positive effect on anti-tumor immunity. Discussion These findings show that NOX4 constrains breast cancer aggressiveness by limiting MYC-driven metabolic adaptations and supporting CD8+ T cell-mediated immunity. Loss of NOX4 promotes a more malignant phenotype and dampens T cell responses, whereas its overexpression prolongs survival and enhances checkpoint inhibitor efficacy. Therapeutically targeting the NOX4–MYC axis and leveraging NOX4’s immunomodulatory capacity could offer promising strategies for breast cancer management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b9114ae3608de23e0dbc2cbea2c8b3a88bf3557" target='_blank'>
              NOX4 modulates breast cancer progression through cancer cell metabolic reprogramming and CD8+ T cell antitumor activity
              </a>
            </td>
          <td>
            Yingying Xiong, Y. Weng, Shan Zhu, Jian Qin, Jia Feng, Xiaopeng Jing, Chao Luo, Wei Gong, Rui Sun, Min Peng
          </td>
          <td>2025-02-07</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Acute myeloid leukemias (AMLs) have an overall poor prognosis with many high-risk cases co-opting stem cell gene regulatory programs, yet the mechanisms through which this occurs remain poorly understood. Increased expression of the stem cell transcription factor, MECOM, underlies one key driver mechanism in largely incurable AMLs. How MECOM results in such aggressive AML phenotypes remains unknown. To address existing experimental limitations, we engineered and applied targeted protein degradation with functional genomic readouts to demonstrate that MECOM promotes malignant stem cell-like states by directly repressing pro-differentiation gene regulatory programs. Remarkably and unexpectedly, a single node in this network, a MECOM-bound cis-regulatory element located 42 kb downstream of the myeloid differentiation regulator CEBPA, is both necessary and sufficient for maintaining MECOM-driven leukemias. Importantly, targeted activation of this regulatory element promotes differentiation of these aggressive AMLs and reduces leukemia burden in vivo, suggesting a broadly applicable differentiation-based approach for improving therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c3c42a2a7df7281b8834a5ccf4f4a2d74201a60" target='_blank'>
              CEBPA repression by MECOM blocks differentiation to drive aggressive leukemias
              </a>
            </td>
          <td>
            Travis J. Fleming, Mateusz Antoszewski, Sander Lambo, Michael Gundry, Riccardo Piussi, Lara Wahlster, Sanjana Shah, Fiona E. Reed, Kevin Dong, Joao A. Paulo, Steve P Gygi, C. Mimoso, Seth R Goldman, Karen Adelman, Jennifer A. Perry, Y. Pikman, K. Stegmaier, Maria N. Barrachina, Kellie R. Machlus, Volker Hovestadt, A. Arruda, Mark D. Minden, Richard A. Voit, V. G. Sankaran
          </td>
          <td>2024-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Primary human cells cultured in organoid format have great promise as potential regenerative cellular therapies. However, their immunogenicity and mutational profile remain unresolved, impeding effective long-term translation to the clinic. In this study we report, for the first time, the generation of human leukocyte antigen (HLA)-I and HLA-II knock-out expandable human primary cholangiocyte organoids (PCOs) using CRISPR-Cas9 as a potential ‘universal’ low-immunogenic therapy for bile duct disorders. HLA-edited PCOs (ePCOs) displayed the same phenotypical and functional characteristics as parental un-edited PCOs. Despite minimal off-target edits, single-molecule DNA-sequencing demonstrated that ePCOs and PCOs acquire substantial mutations in culture at similar rates but without evident selection for cancer-driver mutations. ePCOs induced reduced T cell-mediated immunity and a donor-dependent NK cell cytotoxicity in vitro and evaded cytotoxic responses with increased graft survival in humanized mice in vivo. Our findings have important implications for assessment of safety and immunogenicity of organoid cellular therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e45d4053045048435e735430b496d5825010d393" target='_blank'>
              Immune and Mutational Profile of Gene-Edited Low-Immunogenic Human Primary Cholangiocyte Organoids
              </a>
            </td>
          <td>
            Sandra Petrus-Reurer, A. Baez-Ortega, Winnie Lei, Girishkumar Kumaran, James Williamson, Julia Jones, Daniel Trajkovski, A. Lawson, Krishnaa T. Mahbubani, Cara Brodie, Paul Lehner, Matthew J. Bottomley, I. Martincorena, K. Saeb‐Parsy
          </td>
          <td>2025-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="Cancers that retain wild type TP53 presumably harbor other clonal alterations that permitted their precursors to bypass p53-mediated growth suppression. Consequently, studies that employ TP53-wild type cancer cells and their isogenic derivatives may systematically fail to appreciate the full scope of p53 functionality. Several TP53 phenotypes are known to be absent in the widely used isogenic HCT116 colorectal cancer (CRC) model, which originated from a tumor that had retained wild type TP53. In contrast, we show that restoration of p53 in the TP53-mutant CRC cell line DLD-1 impeded cell proliferation, increased levels of senescence and sensitized cells to ionizing radiation (IR). To study p53 in a non-cancer context, we disrupted TP53 in hTERT-RPE1 cells. Derived from primary cells that were immortalized in vitro, hTERT-RPE1 expressed striking p53-dependent phenotypes and appeared to select for p53 loss during routine culture. hTERT-RPE1 expressed a p53-responsive transcriptome that was highly representative of diverse experimental systems. We discovered several novel downstream p53 targets of potential clinical relevance including ALDH3A1, which is involved in the detoxification of aldehydes and the metabolism of reactive oxygen species, and nectin cell adhesion molecule 4 (NECTIN4) which encodes a secreted surface protein that is overexpressed in many tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56bfbd25a9a6de3dee6da6ae82f799aa171d1483" target='_blank'>
              Robust p53 phenotypes and prospective downstream targets in telomerase-immortalized human cells
              </a>
            </td>
          <td>
            Jessica J. Miciak, Lucy Petrova, Rhythm Sajwan, Aditya Pandya, Mikayla Deckard, Andrew J Munoz, Fred Bunz
          </td>
          <td>2025-02-18</td>
          <td>Oncotarget</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Cancer initiation and progression are associated with numerous somatic mutations, genomic rearrangements, and structure variants. The transformation of a normal cell into a cancer cell involves spatio-temporal changes in the regulation of different gene networks. The accessibility of these genes within the cell nucleus is manipulated via nucleosome remodeling ATPases, comprising one of the important mechanisms. Here, we reviewed studies of an ATP-dependent chromatin remodulator, chromodomain helicase DNA-binding 4 (CHD4), in cancer. Multiple domains of CHD4 are known to take part in nucleosome mobilization and histone binding. By binding with other proteins, CHD4 plays a vital role in transcriptional reprogramming and functions as a key component of Nucleosome Remodeling and Deacetylase, or NuRD, complexes. Here, we revisit data that demonstrate the role of CHD4 in cancer progression, tumor cell proliferation, DNA damage responses, and immune modulation. Conclusively, CHD4-mediated chromatin accessibility is essential for transcriptional reprogramming, which in turn is associated with tumor cell proliferation and cancer development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2a438ec9f6a0d935eb8720173993e895011ee0d" target='_blank'>
              Chromatin Remodulator CHD4: A Potential Target for Cancer Interception
              </a>
            </td>
          <td>
            Krishnendu Goswami, K. Venkatachalam, , C. Rao, Venkateshwar Madka
          </td>
          <td>2025-02-15</td>
          <td>Genes</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d92332ae486c3b94c41f8d10e6867cf69c3624d" target='_blank'>
              Unveiling the molecular profile of a prostate carcinoma: implications for personalized medicine
              </a>
            </td>
          <td>
            Massimiliano Agostini, Erica Giacobbi, F. Servadei, Julia Bishof, Likas Funke, Giuseppe Sica, Valentina Rovella, Marco Carilli, V. Iacovelli, Yufang Shi, Jianquan Hou, E. Candi, G. Melino, Giulio Cervelli, M. Scimeca, A. Mauriello, Pierluigi Bove
          </td>
          <td>2024-12-31</td>
          <td>Biology Direct</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ba7e1fd60eab1467a858b79695c0c9e7c85c584" target='_blank'>
              Mapping chromatin remodelling in glioblastoma identifies epigenetic regulation of key molecular pathways and novel druggable targets
              </a>
            </td>
          <td>
            C. Vinel, James Boot, Weiwei Jin, N. Pomella, Alexandra Hadaway, C. Mein, N. Zabet, Silvia Marino
          </td>
          <td>2025-02-07</td>
          <td>BMC Biology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="LMNA-associated congenital muscular dystrophy is a currently incurable rare genetic disorder characterized by early-onset muscle weakness, dilated cardiomyopathy and respiratory failure, resulting from mutations in the LMNA gene. In this study, we assessed the potential of a CRISPR-mediated strategy to eliminate the mutant allele Lmna c.745C>T, p.R249W using a mutation specific guide (sg745T). Results from R249W-mutation-carrying cellular models showed specific activity of the Cas9/sg745T complex towards the mutant allele. This property varied depending on the concentration of CRISPR components, with a loss of specificity observed with increased dosage. We tested this strategy in vivo using adeno-associated virus delivery in LmnaR249W mice. Despite being associated with a modest CRISPR activity, this therapeutic approach resulted in a 10% increase in the survival of R249W homozygous mice. Interestingly, a similar CRISPR activity improved the cardiac pathology developed by Lmna+/R249W animals, significantly extending their median survival. These results represent the first therapeutic validation of a CRISPR/Cas9-mediated gene editing strategy for the treatment of LMNA-associated congenital muscular dystrophy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6eff96c5b33b1fd18b969ab18137992e335a5be4" target='_blank'>
              CRISPR/Cas9-mediated elimination of the LMNA c.745C>G pathogenic mutation enhances survival and cardiac function in LMNA-associated congenital muscular dystrophy
              </a>
            </td>
          <td>
            Déborah Gómez-Domínguez, Carolina Epifano, Iván Hernández, Borja Vilaplana-Martí, Sergi César, Antonio de Molina-Iracheta, Miguel Sena-Esteves, Georgia Sarquella-Brugada, Ignacio Pérez de Castro
          </td>
          <td>2025-02-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id=". The development of cancer involves an intri‑ cate mechanism, including oncogene activation and tumor suppressor gene inactivation, often driven by epigenetic altera‑ tions. Notably, these epigenetic modifications, such as DNA hypermethylation, are dynamic and reversible, and emerging research suggests that dietary factors can influence them. Recent findings have examined the intricate associations between dietary compounds and cancer development, with a particular focus on the mechanisms through which natural components affect epigenetic alterations. Natural compounds have gained significant attention for their potential in chemo‑ prevention and treatment due to their capacity to modulate DNA methylation patterns, particularly at promoter CpG islands. Chemotherapy and radiation therapy are effective cancer treatments and are often associated with severe adverse effects. As a result, there is increasing interest in exploring natural substances as alternative therapeutic options. The present review focuses on natural compounds that can reverse hypermethylation, providing a potentially safer approach for targeted cancer treatment. These bioactive agents, abundant in natural compounds, exhibit the potential to combat cancer by inhibiting metastasis, inducing cell cycle arrest and reversing DNA hypermethylation. The present review aimed to provide further in‑depth mechanistic insight into the mechanisms through which natural compounds, such as genistein, resvera‑ trol, quercetin and capsaicin modulate DNA hypermethylation in various types of cancer, including breast, cervical, prostate and neuroblastoma. By combining in silico , in vitro and in vivo approaches, it uniquely integrates computational and experimental evidence to demonstrate compound‑specific reversal of epigenetic marks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4eaa3ae9852c9314b773d0fad7daf6e3a6798eb1" target='_blank'>
              Natural compounds as epigenetic modulators: Reversing DNA hypermethylation in cancer (Review)
              </a>
            </td>
          <td>
            Farheen Farheen, Kantimahanti Shreya, R. Mathur, A. Jha
          </td>
          <td>2024-12-30</td>
          <td>World Academy of Sciences Journal</td>
          <td>1</td>
          <td>14</td>
        </tr>

        <tr id="Tumor-specific antigens, also known as neoantigens, have potential utility in anti-cancer immunotherapy, including immune checkpoint blockade (ICB), neoantigen-specific T cell receptor-engineered T (TCR-T), chimeric antigen receptor T (CAR-T), and therapeutic cancer vaccines (TCVs). After recognizing presented neoantigens, the immune system becomes activated and triggers the death of tumor cells. Neoantigens may be derived from multiple origins, including somatic mutations (single nucleotide variants, insertion/deletions, and gene fusions), circular RNAs, alternative splicing, RNA editing, and polymorphic microbiome. An increasing amount of bioinformatics tools and algorithms are being developed to predict tumor neoantigens derived from different sources, which may require inputs from different multi-omics data. In addition, calculating the peptide-major histocompatibility complex (MHC) affinity can aid in selecting putative neoantigens, as high binding affinities facilitate antigen presentation. Based on these approaches and previous experiments, many resources were developed to reveal the landscape of tumor neoantigens across multiple cancer types. Herein, we summarized these tools, algorithms, and resources to provide an overview of computational analysis for neoantigen discovery and prioritization, as well as the future development of potential clinical utilities in this field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/676b6cef04b3606a814455c2e9548b49b282ba61" target='_blank'>
              Characterization of Tumor Antigens from Multi-omics Data: Computational Approaches and Resources.
              </a>
            </td>
          <td>
            Yunzhe Wang, James Wengler, Yuzhu Fang, Joseph Zhou, Hang Ruan, Zhao Zhang, Leng Han
          </td>
          <td>2025-01-20</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Environmental factors are linked to aging and age-related diseases. Emerging evidence suggests that enhancing body's resistance to xenobiotics might be an anti-aging strategy. The constitutive androstane receptor (CAR) regulates drug-metabolizing enzymes and transporters, coordinating metabolism and immune responses to adapt to stress triggered by exogenous exposure. However, the impact of activating CAR on aging remains unknown. In this study, Caenorhabditis elegans (C. elegans), drug-induced premature aging mice, and senescence accelerated P8 (SAMP8) mice are used as models to explore the effects of CAR activation on lifespan and healthspan, along with the underlying mechanisms. The results showed that hCAR agonist CITCO and mCAR agonist TCPOBOP prolonged the lifespan and healthspan in model organism. The longevity effects of CITCO and TCPOBOP were attenuated in CAR homozygous nhr-8/daf-12 mutant C. elegans as well as CAR-/- mice. In C. elegans, CITCO activated both anti-stress and detoxification genes, and increased the resistance to environmental adversities. Additionally, the lifespan-extending and xenobiotic resistant effects of CITCO might be related to the regulation of age-related pathways. Furthermore, CITCO improved age-related neurodegeneration in C. elegans models. Taken together, the results suggest that the longevity effects of CAR agonists may be related to the enhancement of xenobiotic resistance of animals.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/960efc318b7833116a8e79bd8aa34344f0dc8fbe" target='_blank'>
              Activation of Nuclear Receptor CAR: A Pathway to Delay Aging through Enhanced Capacity for Xenobiotic Resistance.
              </a>
            </td>
          <td>
            Jing Yu, Xiaoyan Gao, Hang Shi, Li-jun Zhang, Wenlong Nie, Ruochen Zhang, Ming Fang, Ying Liu, Yingxuan Yan, Bingbing Fan, Chengyuan Wu, Cheng Huang, Shengjie Fan
          </td>
          <td>2025-01-31</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="BACKGROUND
 Colorectal cancer (CRC) is a prevalent malignant tumor characterized by a high mortality rate, with significant challenges persisting in the identification and management of its metastatic stage. The role of checkpoint kinase 1 (CHEK1), a cell cycle checkpoint kinase, in CRC has not been fully clarified. We hypothesize that the upregulation of CHEK1 may enhance the proliferation of CRC cells, indicating its potential as a novel therapeutic target for CRC therapy.
 AIM
 To investigate the expression and function of CHEK1 in CRC, this study utilizes single-cell RNA sequencing and tissue microarray data.
 METHODS
 Single-cell RNA sequencing technology was employed to analyze CRC cells from the GSE144735 dataset, and immunohistochemistry was conducted to confirm the expression of CHEK1 in CRC and adjacent tissues. We also integrated mRNA expression data from multiple public databases to assess global CHEK1 expression in CRC. Molecular docking experiments were performed to explore the interaction between CHEK1 and the potential drug nitidine chloride (NC), as well as to investigate the influence of CHEK1 on CRC cell proliferation.
 RESULTS
 We found comparatively elevated CHEK1 expression in the malignant epithelial cells of CRC, with marked upregulation of its mRNA levels in CRC tissues. Immunohistochemical analysis further confirmed the high expression of CHEK1 in CRC tissues, and the receiver operating characteristic curve demonstrated high accuracy (area under the curve = 0.964) for CHEK1 as a biomarker. Analysis of global datasets indicated a statistically significant overexpression of CHEK1 in CRC (standard mean difference = 1.81, P < 0.01), with summary receiver operating characteristic analysis yielding sensitivity and specificity values of 0.83 and 0.88, respectively. Molecular docking studies indicated that NC specifically targeted CHEK1, while clustered regularly interspaced short palindromic repeats knockout experiments demonstrated that CHEK1 promoted CRC cell proliferation.
 CONCLUSION
 Upregulation of CHEK1 promotes CRC cell proliferation. However, the dataset's diversity is limited, requiring further investigation into its specific mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a566889c9fc5fe520488439e6857e1a25ba38f4b" target='_blank'>
              Checkpoint kinase 1 in colorectal cancer: Upregulation of expression and promotion of cell proliferation
              </a>
            </td>
          <td>
            Yu-yan Pang, Zu-Yuan Chen, Da-Tong Zeng, Dong-Ming Li, Qi Li, Wan-Ying Huang, Bin Li, Jia-Yuan Luo, Bang-Teng Chi, Qiu Huang, Zhen-Bo Feng, Rong-Quan He
          </td>
          <td>2025-03-24</td>
          <td>World Journal of Clinical Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Cervical cancer is a significant global health threat, ranking as the fourth most common malignancy among women and resulting in over 300,000 deaths annually. Although screening and vaccination initiatives have led to a decline in incidence rates, treatment options for advanced or recurrent cervical cancer remain inadequate, often proving ineffective and costly. In this context, adenoviral therapy has emerged as a promising strategy to enhance therapeutic outcomes. Adenoviruses are non-enveloped viruses that can efficiently infect a wide range of cells, including tumor cells, while exhibiting a favorable safety profile, making them suitable candidates for clinical applications. Adenoviral vectors possess the unique ability to package large segments of therapeutic genes, allowing for diverse treatment approaches, including oncolytic virotherapy, which selectively targets and destroys tumor cells while stimulating robust immune responses. By engineering adenoviruses to express tumor suppressor genes such as p53, researchers can restore critical apoptotic pathways often disrupted in cervical cancer. Furthermore, genetic modifications to capsid proteins can enhance the targeting of tumor cells and reduce the immunogenicity associated with these viral vectors. Additionally, adenoviral vectors can serve as delivery systems for therapeutic vaccines against HPV oncogenes E6 and E7, promoting effective immune responses and potentially preventing disease progression. The combination of adenoviral therapy with immune checkpoint inhibitors offers a novel approach to overcoming the immunosuppressive tumor microenvironment, enhancing overall antitumor immunity. Overall, this review highlights the significant advancements in adenoviral therapy for cervical cancer, emphasizing the need for further research to optimize these strategies and translate preclinical successes into effective clinical applications. By harnessing the full potential of adenoviral vectors, we can improve treatment options for patients who have cervical cancer, paving the way for more personalized and effective therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c82e231d3c7785447bc8b4e9c1d3b38641d5352c" target='_blank'>
              Adenoviral Therapy for Cervical Cancer: From Targeted Modification to Immunotherapy.
              </a>
            </td>
          <td>
            Yufeng Li, Haibin Zhang, Wenhu Xin, Tiansheng Qin
          </td>
          <td>2025-01-27</td>
          <td>Anti-cancer agents in medicinal chemistry</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Nearly half of the human genome is composed of transposable elements (TEs), which were once considered “junk” DNA due in part to their repetitive and non-coding characteristics. However, TEs are now emerging as important regulators of gene activity, especially when they escape typical epigenetic silencing mechanisms like DNA methylation and histone modifications. When activated, TEs can contribute sequences that are exploited as disruptors of genomic stability or novel gene regulatory elements, making them important factors in both development and disease, including cancer. In cancer, these previously silenced elements can act as alternative promoters of oncogenes, leading to the production of TE-chimeric transcripts in a process called “onco-exaptation”. While these oncoexaptation events are reported to be widespread across many cancers, only a few TE-chimeric transcripts have been validated as active contributors to cancer progression. Additionally, it remains unclear whether these TE-chimeric transcripts are merely a consequence of the inherent genomic instability in cancer or whether they can initiate cancer. To address these gaps, we designed a cDNA overexpression library incorporating our previously identified pan-cancer TE-chimeric transcript candidates. This library is used in early passage (EP) BJ fibroblast, a cell line known for its transformation potential in the well-established Weinberg transformation model, which allows us to systematically investigate these onco-exaptation events during cancer progression. Our initial analyses indicate that most TE-chimeric oncogenes are not expressed in EP fibroblast, but are activated in subsequent transformation stages following hTERT, SV40 and HRas induction, underscoring the model viability. We are currently using this model to distinguish TE-chimeric transcripts that act as drivers (TE-chimeric oncogenes) from those requiring previous vulnerabilities, such as telomere inactivation by hTERT or p53 and Rb inactivation by SV40. Ultimately, this work will identify novel biomarkers and therapeutic targets, enhance insights into gene regulation, and provide opportunities for early detection and intervention. These findings have the potential to improve cancer treatment and patient outcomes.
 Citation Format: Wesley N Saintilnord, Yixin Wu, Yonghao Liang, Ting Wang. The functional role of transposable element-oncogene chimeric transcript during cancer progression [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B005.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9f7a8875ef14376de973322ef301e4c4ddf4aab" target='_blank'>
              Abstract B005: The functional role of transposable element-oncogene chimeric transcript during cancer progression
              </a>
            </td>
          <td>
            Wesley N. Saintilnord, Yixin Wu, Yonghao Liang, Ting Wang
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad33c5772a7c6f391f659ca36bbfaca426706c80" target='_blank'>
              Improving the odds of survival: transgenerational effects of infections.
              </a>
            </td>
          <td>
            V. Spanou, Theano Andriopoulou, E. Giamarellos‐Bourboulis, Mihai G Netea
          </td>
          <td>2025-01-22</td>
          <td>EMBO molecular medicine</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fc6572745a00744a0ffe9484307c354e0370541" target='_blank'>
              Short tandem repeats delineate gene bodies across eukaryotes
              </a>
            </td>
          <td>
            William B. Reinar, A. Krabberød, V. O. Lalun, M. Butenko, K.S. Jakobsen
          </td>
          <td>2024-12-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="
 239


 Background:
 Inactivating genomic alteration (GA) of the TP53 tumor suppressor gene leads to inactivation of p53 protein and is currently the most frequently associated GA across all cancers. Colorectal cancer is a common and lethal subtype of cancer, in the top 5 for incidence, prevalence and cancer-related deaths worldwide. TP53 is the second most frequent GA identified in clinically advanced CRC. It has been postulated that GA in TP53 not only affects the molecular biology of malignant cells but also impacts the tumor microenvironment in CRC. This GA is currently untargetable by approved drugs however, recently a novel approach designed to revert to wild type functioning of TP53 inactivated by Y220C base substitution has gained significant clinical interest.
 Methods:
 67,301 cases of clinically advanced CRC underwent hybrid capture based comprehensive genomic profiling to assess all classes of GA. Cases were sequenced to a mean coverage depth of 650X with microsatellite instability (MSI) status and tumor mutation burden (TMB) determined from the sequencing data and PD-L1 expression measured by immunohistochemistry using the DAKO tumor proportional staining system (low positive set at 1-49% staining and high positive set at ≥50% staining).
 Results:
 422 (0.6%) of the CRC featured the
 TP53
 Y220C GA (Y220C+). When compared with the CRC that were
 TP53
 Y220C negative (Y220C-), the Y220C+ cases were of similar age, gender and number of GA per tumor. The Y220C- cases has significantly higher frequencies of MSI status (6.1% vs 2.2%; p=.0007) and TMB levels greater >10 mutations/Mb (8.7% vs 4.7%; p=.004). The Y220C+ featured lower frequencies of GA in
 PIK3CA
 (12.9% vs 18.6%; p=.002),
 BRAF
 (8.8% vs 10.1%; NS) and
 PTEN
 (7.4% vs 8.3%, NS) and a higher frequency of
 ERBB2
 amplification (5.0% vs 2.7%; p=.004). GA in
 APC
 ,
 KRAS
 ,
 NRAS
 ,
 BRCA2
 ,
 EGFR
 and
 MET
 were similar in both Y220C+ and Y220C- CRC. PD-L1 low expression was similarly infrequent (12.1% to 16.7% range) in both groups.
 Conclusions:
 The potentially targetable
 TP53
 Y220C short variant mutation is uncommon in clinically advanced CRC but is associated with unique genomic and biomarker characteristics. Further research might elucidate subpopulations or CRC subtypes with a greater frequency of this mutation who might benefit from a novel targeted therapy.






 CRC
 TP53
 Y220C+(422 Cases)


 CRC
 TP53
 Y220C-(66,879 Cases)

 P Value




 Gender
 41.2% F/ 58.8% M
 44.4% F/55.6% M
 NS


 Median Age (range)
 62 (23-89+)
 61 (8-89+)
 NS


 GA/tumor
 6.1
 6.5
 NS



 TP53
 non-Y220C GA

 16.1%
 75.7%
 <.0001



 PIK3CA

 12.9%
 18.6%
 .002



 ERBB2

 6.2% (5.0% amp)
 5.1% (2.7% amp)
 .004 (amp)


 MSI High
 2.2%
 6.1%
 .0007


 TMB > 10 mut/Mb
 4.7%
 8.7%
 .004


 PD-L1 Low (1-49%TPS)/ PD-L1 High (>50% TPS)
 16.7% (132 cases)/2.3%
 12.1% (20,759 cases)/1.4%
 NS




">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99ba4427e7070f06aa4eaa87971edb3bcc24e315" target='_blank'>
              TP53
 Y220C mutations in colorectal carcinoma (CRC): A genomic landscape study.
              </a>
            </td>
          <td>
            Deevyashali Parekh, J. Ross, Nimisha Srivastava, Tamara Jamaspishvili, A. Basnet
          </td>
          <td>2025-02-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Historically considered downstream effects of tumorigenesis—arising from changes in DNA content or chromatin organization—nuclear alterations have long been seen as mere prognostic markers within a genome‐centric model of cancer. However, recent findings have placed the nuclear envelope (NE) at the forefront of tumor progression, highlighting its active role in mediating cellular responses to mechanical forces. Despite significant progress, the precise interplay between NE components and cancer progression remains under debate. In this review, we provide a comprehensive and up‐to‐date overview of how changes in NE composition affect nuclear mechanics and facilitate malignant transformation, grounded in the latest molecular and functional studies. We also review recent research that uses advanced technologies, including artificial intelligence, to predict malignancy risk and treatment outcomes by analyzing nuclear morphology. Finally, we discuss how progress in understanding nuclear mechanics has paved the way for mechanotherapy—a promising cancer treatment approach that exploits the mechanical differences between cancerous and healthy cells. Shifting the perspective on NE alterations from mere diagnostic markers to potential therapeutic targets, this review calls for further investigation into the evolving role of the NE in cancer, highlighting the potential for innovative strategies to transform conventional cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53ec969f050943713ac6f56187d24a844cc59674" target='_blank'>
              At the nucleus of cancer: how the nuclear envelope controls tumor progression
              </a>
            </td>
          <td>
            F. Paganelli, Alessandro Poli, Serena Truocchio, Alberto M. Martelli, Carla Palumbo, Giovanna Lattanzi, Francesca Chiarini
          </td>
          <td>2025-01-24</td>
          <td>MedComm</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="RAD18 is a conserved eukaryotic E3 ubiquitin ligase that promotes genome stability through multiple pathways. One of these is gap-filling DNA synthesis at active replication forks and in post-replicative DNA. RAD18 also regulates homologous recombination (HR) repair of DNA breaks; however, the current literature describing the contribution of RAD18 to HR in mammalian systems has not reached a consensus. To investigate this, we examined three independent RAD18-null human cell lines. Our analyses found that loss of RAD18 in HCT116, but neither hTERT RPE-1 nor DLD1 cell lines, resulted in elevated sister chromatid exchange, gene conversion, and gene targeting, i.e., HCT116 mutants were hyper-recombinogenic (hyper-rec). Interestingly, these phenotypes were linked to RAD18’s role in PCNA K164 ubiquitination, as HCT116 PCNAK164R/+ mutants were also hyper-rec, consistent with previous studies in rad18−/− and pcnaK164R avian DT40 cells. Importantly, the knockdown of UBC9 to prevent PCNA K164 SUMOylation did not affect hyper-recombination, strengthening the link between increased recombination and RAD18-catalyzed PCNA K164 ubiquitination, but not K164 SUMOylation. We propose that the hierarchy of post-replicative repair and HR, intrinsic to each cell type, dictates whether RAD18 is required for suppression of hyper-recombination and that this function is linked to PCNA K164 ubiquitination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ade4d531e6f454e305c916fc9c53a0c286d620e" target='_blank'>
              Cell Type Specific Suppression of Hyper-Recombination by Human RAD18 Is Linked to Proliferating Cell Nuclear Antigen K164 Ubiquitination
              </a>
            </td>
          <td>
            Colette B. Rogers, W. Leung, Ryan M. Baxley, Rachel E. Kram, Liangjun Wang, Joseph P. Buytendorp, Khoi Le, D. Largaespada, Eric A. Hendrickson, A. Bielinsky
          </td>
          <td>2025-01-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="BACKGROUND
Massively parallel sequencing (MPS) of nucleic acids has been a transformative technology for basic and applied genomic science, increasing efficiencies and decreasing costs to enable studies of unprecedented scope and impact. In clinical settings, these technological and scientific advances have led to the development of tests that are increasingly fast, comprehensive, and more frequently employed. Practitioners of genomic medicine have applied these tools across clinical settings, including diagnosis of inherited disorders and cancers and infectious disease detection and surveillance. In recent years, the commercial marketplace for MPS sequencers and reagents has been dominated by a few companies. The growing demand for sequencing has led to the recent emergence of several new sequencing platforms with techniques that may provide alternatives or improvements to existing workflows or allow the adoption of sequencing workflows in new settings. Clinical genomics laboratories will evaluate these platforms from a unique perspective, focusing on how technological advancements can improve patient care.


CONTENT
This review describes short-read sequencing platforms provided by Illumina, Element Biosciences, MGI, PacBio, Singular Genomics, Thermo Fisher Scientific, and Ultima Genomics. This review discusses their innovative approaches, principles, workflows, and applications.


SUMMARY
This review aims to inform laboratory geneticists, clinicians, and researchers about emerging short-read technologies and their applications in clinical genomics. By highlighting their principles and potential contributions, we aim to assist laboratories in selecting suitable solutions for their sequencing needs considering key factors such as applications, throughput, and integration with existing laboratory workflows.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc19721b678ba2d4cedee69c6b99d021887e0026" target='_blank'>
              Innovations in Short-Read Sequencing Technologies and Their Applications to Clinical Genomics.
              </a>
            </td>
          <td>
            Katarzyna Polonis, Joseph H. Blommel, Andrew E O Hughes, DH Spencer, Joseph A Thompson, Molly C Schroeder
          </td>
          <td>2025-01-01</td>
          <td>Clinical chemistry</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Viral vectors have successfully modified T cells to express chimeric antigen receptors (CAR), leading to clinical approvals. However, their high cost and regulatory challenges hinder rapid and broad clinical translation. Here, we demonstrate that our lentivirally (LV) manufactured R110-CAR T cells, targeting a leukemia neoepitope, can also be engineered using non-viral Sleeping Beauty (SB) transposition with minimal-sized DNA vectors. Flow cytometry and single-cell sequencing was used to compare the two production modes using healthy donor and CLL patient-derived T cells and a CD19-CAR T cell control. SB products were shifted towards CD8+ subsets with expression of activation/co-inhibition markers (CD69, LAG-3, TIM-3) despite their naïve-like phenotype and lack of antigenic challenge. The CAR binding moiety modulated these patterns with R110-CAR T cells showing more aberrant phenotypes. Moreover, SB engineering resulted in inflammatory signatures along with RIG-I-like and TOLL-like nucleotide sensing potentially resulting from the transfection procedure. Patient-derived products showed significantly fewer CAR-expressing cells, reduced proliferation clusters, and lower T cell diversity, particularly with SB manufacturing, pointing at potential challenges with this method when engineering CLL T cells. Together, our data suggest that the engineering mode may substantially influence T cell properties and that these are further modulated by the CAR binding moiety and the type of T cell donor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/091bcf82c5312a9b158fdb202c07c565ad9cabb7" target='_blank'>
              CAR T cell engineering impacts antigen-independent activation and co-inhibition
              </a>
            </td>
          <td>
            Simon Stücheli, Christoph Schultheiss, Paul Schmidt-Barbo, Andreas Zingg, Natascha Franz, Sarah Adamo, Claudia Fischer, H. Läubli, Mascha Binder
          </td>
          <td>2025-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Oncogene activation in normal untransformed cells induces DNA replication stress and creates a dependency on DNA Damage Response (DDR) mechanisms for cell survival. Different oncogenic stimuli signal via distinct mechanisms in every cancer setting. The DDR is also pathologically re-programmed and deployed in diverse ways in different cancers. Because mutant KRAS is the driver oncogene in 90% of Pancreatic Ductal Adenocarcinomas (PDAC), here we have investigated DDR mechanisms by which KRAS-induced DNA replication stress is tolerated in normal human pancreatic epithelial cells (HPNE). Using a candidate screening approach, we identify TRIP13 as a KRASG12V-induced mRNA that is also expressed at high levels in PDAC relative to normal tissues. Using genetic and pharmacological tools, we show that TRIP13 is necessary to sustain ongoing DNA synthesis and viability specifically in KRASG12V-expressing cells. TRIP13 promotes survival of KRASG12V-expressing HPNE cells in a Homologous Recombination (HR)-dependent manner. KRASG12V-expressing HPNE cells lacking TRIP13 acquire hallmark HR-deficiency (HRD) phenotypes including sensitivity to inhibitors of Trans-Lesion Synthesis (TLS) and Poly-ADP Ribose Polymerase (PARP). Established PDAC cell lines are also sensitized to intrinsic DNA damage and therapy-induced genotoxicity following TRIP13-depletion. Taken together our results expose TRIP13 as an attractive new and therapeutically-tractable vulnerability of KRAS-mutant PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89fa9b68b58350edbc77be5948633226c0fa98f1" target='_blank'>
              TRIP13 protects pancreatic cancer cells against intrinsic and therapy-induced DNA replication stress
              </a>
            </td>
          <td>
            J. R. Anand, G. Droby, Sayali Joseph, Urvi Patel, Xingyuan Zhang, Jeffrey A. Klomp, C. Der, Jeremy E. Purvis, Samuel C. Wolff, Jessica Bowser, Cyrus Vaziri
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>122</td>
        </tr>

        <tr id="Cellular senescence features a durable exit from the cell cycle triggered by stress or carcinogens. The INK4 locus is inactivated in various cancers, yet in senescence, p16Ink4a is activated. Whether senescence is tumor-suppressing or -promoting remains a conundrum. We discovered an evolutionally-conserved Vertebrata INK4-homolog. This ink4ab triggers senescence upon oxidative- and/or carcinogenic-stress. Adult Ink4ab-deficient animals failed to activate senescence and developed spontaneous cancers. Combined Ink4ab and Tp53 deficiency revealed a reciprocal senescence and apoptosis regulation, controlling tumorigenesis, including retinoblastoma. INK4-hematopoietic-deficient mice exhibited p19Arf-dependent enhanced senescence-like phenotypes, uncontrolled cell proliferation, defective stem cell differentiation, and splenomegaly, with single-splenocytes spatially-enriched in senescence-associated secretory profiles. Our studies reveal the evolutionary origin of paradigms co-regulating senescence and tumor suppression and offer strategies to exploit these reciprocal pathways for cancer prevention and therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9243e085293669a418053332a11c9df19226137a" target='_blank'>
              Evolutionary conserved reciprocal senescence and tumor suppressor signals limit lifetime cancer
              </a>
            </td>
          <td>
            Liqiong Liu, Stephani Davis, Shamila Yusuff, Ann Strange, Yonghua Zhuang, Prasanna Vaddi, Kathleen Flaherty, Kelly Jara, Joseph Kramer, Christine Archer, Schuyler Lee, Bifeng Gao, A. van Bokhoven, Wei Wang, Sharon R. Pine, Tetsuya Nakamura, Hatim E. Sabaawy
          </td>
          <td>2024-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is a highly invasive and highly lethal malignant tumor. Due to the difficulty in early diagnosis and limited treatment options, the 5-year survival rate is less than 10%. Traditional treatments such as surgery, chemotherapy, and radiotherapy have limited effects on PDAC, which is mainly attributed to its unique tumor microenvironment (TME), common gene mutations (such as KRAS, and TP53), and immunosuppressive properties. In recent years, with the advancement of molecular biology and genomics technologies, the molecular mechanisms of PDAC have been studied more deeply, revealing the gene mutations and regulatory networks associated with its pathogenesis. In particular, KRAS mutations have become an important research direction for targeted therapy. However, the effect of immunotherapy in PDAC is limited by the immune escape characteristics of TME. This article systematically summarizes the key gene mutations in PDAC and their regulatory mechanisms, including epigenetic regulation, microRNA, histone modification and other influencing factors. CRISPR-Cas9 shows great potential in correcting gene mutations and reshaping TME. In addition, combining immunotherapy with targeted drug delivery (such as nanotechnology) can improve the precision of treatment and reduce side effects. These advances provide a comprehensive reference for future personalized and effective treatment options for PDAC">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1538e4df893f0e562277858358833ccdc8a4f65" target='_blank'>
              Genetic Mutation and Regulation in Pancreatic Ductal Adenocarcinoma Cancer: Developing Relevant Molecular Biomarkers
              </a>
            </td>
          <td>
            Xiaomi Ai
          </td>
          <td>2025-01-15</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d57175017416aef8cf7ca32d649a39c06422fc2" target='_blank'>
              Circular RNAs as a novel class of potential therapeutic and diagnostic biomarkers in reproductive biology/diseases
              </a>
            </td>
          <td>
            Wanyu Song, Xiuli Chen, Haiying Wu, Neda Rahimian
          </td>
          <td>2024-12-31</td>
          <td>European Journal of Medical Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09816baa7e13a8a81394d1d42faa90a68058ed21" target='_blank'>
              Stochastic demethylation and redundant epigenetic suppressive mechanisms generate highly heterogeneous responses to pharmacological DNA methyltransferase inhibition
              </a>
            </td>
          <td>
            Mie K. Jakobsen, S. Traynor, A. Y. Nielsen, Christina Dahl, Mette Staehr, S. T. Jakobsen, M. S. Madsen, Rasmus Siersbaek, Mikkel G Terp, Josefine B. Jensen, C. B. Pedersen, Anup Shrestha, J. R. Brewer, P. H. Duijf, Odd L. Gammelgaard, Henrik J Ditzel, Alexei F Kirkin, Per Guldberg, Morten F. Gjerstorff
          </td>
          <td>2025-01-23</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Abstract Acute lymphoblastic leukemia (ALL) is the most common childhood cancer. The translocation t(1;19), encoding the TCF3‐PBX1 fusion, is associated with intermediate risk and central nervous system (CNS) infiltration at relapse. Using our previously generated TCF3‐PBX1 conditional knock‐in mice, we established a model to study relapsed clones after in vivo chemotherapy treatment, CNS infiltration, and clonal dynamic evolution of phenotypic diversity at the single cell‐level using next‐generation sequencing technologies and mass cytometry. Mice transplanted with TCF3‐PBX1 + leukemia cells and treated with vehicle succumbed to disease, whereas 40% of treated mice with prednisolone or daunorubicin survived. Bulk and single‐cell RNA sequencing of FACS‐sorted GFP+ cells from TCF3‐PBX1 + leukemias arising after chemotherapy treatment revealed that apoptosis, interleukin‐, and TGFβ‐signaling pathways were regulated in CNS‐infiltrating leukemic cells. Across tissues, upregulation of the MYC signaling pathway was detected in persisting leukemic cells and its downregulation by BRD3/4 inhibition increased sensitivity to chemotherapy. In TCF3‐PBX1+ leukemia cells collected after chemotherapy treatment, mass cytometry identified increased phosphorylation of STAT3/5 upon preBCR stimulation, which was susceptible to inhibition by the proteasome inhibitor bortezomib. In summary, we developed a TCF3‐PBX1+ ALL mouse model and characterized relapsed disease after in vivo chemotherapy and cell phenotype dependence on microenvironment. Transcriptomics and phospho‐proteomics revealed distinct pathways that may underlie chemotherapy resistance and might be suitable for pharmacological interventions in human ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a278da44c640128811ffe9cd7b7d31669827014f" target='_blank'>
              Dynamic evolution of TCF3‐PBX1 leukemias at the single‐cell level under chemotherapy pressure
              </a>
            </td>
          <td>
            Mira Kusterer, M. Lahnalampi, M. Voutilainen, Alexandra Brand, S. Pennisi, J. Norona, Gaia Gentile, Heike Herzog, G. Greve, Michael Lübbert, M. Sipola, Emma Kaartinen, R. Sankowski, Marco Prinz, S. Killmer, M. S. Lago, Bertram Bengsch, Stepan Cysar, K. Aumann, Martin Werner, J. Duyster, O. Lohi, M. Heinäniemi, J. Duque-Afonso
          </td>
          <td>2025-02-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82bc21840fb421694721da36c78c3833f6d5089d" target='_blank'>
              Clonal driver neoantigen loss under EGFR TKI and immune selection pressures.
              </a>
            </td>
          <td>
            M. Al Bakir, J. Reading, Samuel Gamble, R. Rosenthal, Imran Uddin, Andrew Rowan, J. Przewrocka, Amber Rogers, Y. Wong, A. Bentzen, S. Veeriah, S. Ward, Aaron T Garnett, Paula Kalavakur, C. Martínez-Ruiz, C. Puttick, A. Huebner, Danielle Cook, David A Moore, C. Abbosh, C. Hiley, C. Naceur-Lombardelli, T. Watkins, Marina Petković, Roland F Schwarz, F. Gálvez-Cancino, K. Litchfield, P. Meldgaard, B. Sørensen, L. Madsen, Dirk Jäger, Martin D Forster, T. Arkenau, C. Domingo-Vila, T. Tree, M. Kadivar, S. Hadrup, Benjamin M Chain, Sergio A Quezada, N. Mcgranahan, C. Swanton
          </td>
          <td>2025-02-19</td>
          <td>Nature</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34beca577def84b7592da411aab4f563850a4d1a" target='_blank'>
              Tumour-wide RNA splicing aberrations generate actionable public neoantigens.
              </a>
            </td>
          <td>
            Darwin W. Kwok, Nicholas O Stevers, Iñaki Etxeberria, Takahide Nejo, Maggie Colton Cove, Lee H. Chen, Jangham Jung, K. Okada, S. Lakshmanachetty, M. Gallus, Abhilash Barpanda, Chibo Hong, Gary Chan, Jerry Liu, Samuel H. Wu, Emilio Ramos, A. Yamamichi, P. Watchmaker, Hirokazu Ogino, A. Saijo, Aidan Du, Nadia Grishanina, James Woo, Aaron Diaz, Shawn L Hervey-Jumper, Susan Marina Chang, Joanna J Phillips, Arun P Wiita, Christopher A. Klebanoff, Joseph F Costello, Hideo Okada
          </td>
          <td>2025-02-19</td>
          <td>Nature</td>
          <td>1</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/501a2344c548bcfc786460170b1feb2e0d711776" target='_blank'>
              Sperm-borne small non-coding RNAs: potential functions and mechanisms as epigenetic carriers
              </a>
            </td>
          <td>
            Muhammad Naveed, Zhaokang Shen, Jianqiang Bao
          </td>
          <td>2025-01-17</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4211b81d0b19e4c7f2e0cfa7c734b541dc3fa943" target='_blank'>
              Epigenetics in the modern era of crop improvements.
              </a>
            </td>
          <td>
            Yan Xue, Xiaofeng Cao, Xiangsong Chen, Xian Deng, Xing Wang Deng, Yong Ding, Aiwu Dong, Cheng-Guo Duan, Xiaofeng Fang, L. Gong, Zhizhong Gong, Xiaofeng Gu, Chongsheng He, Hang He, Shengbo He, Xin-Jian He, Yan He, Yuehui He, Guifang Jia, Danhua Jiang, Jianjun Jiang, Jinsheng Lai, Zhaobo Lang, Chenlong Li, Qing Li, Xingwang Li, Bao Liu, Bing Liu, Xiao Luo, Yijun Qi, Weiqiang Qian, Guodong Ren, Qingxin Song, Xianwei Song, Zhixi Tian, Jia-Wei Wang, Yuan Wang, Liang Wu, Zhe Wu, Rui Xia, Jun Xiao, Lin Xu, Zhengyi Xu, Wenhao Yan, Hongchun Yang, J. Zhai, Yijing Zhang, Yusheng Zhao, Xuehua Zhong, Dao-Xiu Zhou, Ming Zhou, Yue Zhou, Bo Zhu, Jian-Kang Zhu, Qikun Liu
          </td>
          <td>2025-01-08</td>
          <td>Science China. Life sciences</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Lentiviral vector-transduced T cells were approved by the FDA as gene therapy anti-cancer medications. Little is known about the effects of host genetic variation on the safety and efficacy of the lentiviral vector gene delivery system. To narrow this knowledge gap, we characterized hepatic gene delivery by lentiviral vectors across the Collaborative Cross (CC) mouse genetic reference population. For 24 weeks, we periodically measured hepatic luciferase expression from lentiviral vectors in 41 CC mouse strains. Hepatic and splenic vector copy numbers were determined. We report that the CC mouse strains showed highly diverse outcomes following lentiviral gene delivery. For the first time, a moderate correlation between mouse-strain-specific sleeping patterns and transduction efficiency was observed. We associated two quantitative trait loci (QTLs) with intrastrain variations in transduction phenotypes, which mechanistically relates to the phenomenon of metastable epialleles. An additional QTL was associated with the kinetics of hepatic transgene expression. Genes found in the above QTLs are potential targets for personalized gene therapy protocols. Importantly, we identified two mouse strains that open new directions for characterizing continuous viral vector silencing and HIV latency. Our findings suggest that wide-range patient-specific outcomes of viral vector-based gene therapy should be expected. Thus, novel clinical protocols should be considered for non-fatal diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4970be4c0655da468396fb187c5303cf10c27ee8" target='_blank'>
              Analysis of Hepatic Lentiviral Vector Transduction: Implications for Preclinical Studies and Clinical Gene Therapy Protocols
              </a>
            </td>
          <td>
            Peirong Hu, Yajing Hao, Wei Tang, Graham H. Diering, Fei Zou, T. Kafri
          </td>
          <td>2025-02-17</td>
          <td>Viruses</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Approximately 97% of the human genome comprises non-coding sequences, with nearly half originating from transposable elements. Among these, retrotransposons represent a critical subclass that replicates via a "copy-and-paste" mechanism and significantly influences the regulation of host genomes. In both normal and pathological contexts, retrotransposons contribute a vast reservoir of regulatory elements that can modulate the expression of genes. If left unchecked, retrotransposons can substantially affect host transcriptional programs and genomic integrity. Therefore, various mechanisms, including epigenetic modifications, are employed to mitigate their potentially deleterious effects. In diseases such as cancers, the epigenome is often significantly reprogrammed, which can lead to retrotransposon dysregulation. Drawing insights from recent studies conducted in human and murine cells, this review examines how retrotransposons expand the complexity of mammalian genomes, describes the impact of their epigenetic dysregulation in cancer development, and highlights the potential of targeting these sequences for therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acf5c3f4d65d70d88355c0196d8616e76468f57c" target='_blank'>
              Epigenetic dysregulation of retrotransposons in cancer.
              </a>
            </td>
          <td>
            Danny Leung, Kwok Yu Liu
          </td>
          <td>2025-02-13</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Prostate cancer is a prevalent tumor type that, despite being highly curable, progresses to metastatic disease in a fraction of patients, thus accounting for more than 350.000 annual deaths worldwide. In turn, uncovering the molecular insights of metastatic disease is instrumental to improve the survival rate of prostate cancer patients. By means of gene expression metanalysis in multiple prostate cancer patient cohorts, we identified a set of genes that are differentially expressed in aggressive prostate cancer. Transcription factor 19 (TCF19) stood out as an unprecedented epithelial gene upregulated in metastatic disease, with prognostic potential and associated with the activity of androgen receptor. By combining computational and empiric approaches, our data revealed that TCF19 is required for full metastatic capacity and its depletion influences core cancer-related processes, such as vascular permeability, supporting the role of this gene in the dissemination of prostate tumor cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ecdf804c61fcb68669b4e697cde8e1d82f6075a" target='_blank'>
              Transcription factor 19 is an androgen responsive gene that modulates vessel homeostasis and sustains metastatic prostate cancer
              </a>
            </td>
          <td>
            Amaia Ercilla, Jana R. Crespo, Saioa Garcia-Longarte, Marta Fidalgo, Natalia Martín-Martín, Onintza Carlevaris, I. Astobiza, Sonia Fernández-Ruiz, M. Guiu, Laura Bárcena, Isabel Mendizabal, A. Aransay, M. Graupera, R. Gomis, Arkaitz Carracedo
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Long non-coding RNAs (lncRNAs) refer to a group of RNA molecules that exceed a length of 200 nucleotides and lack the ability to code proteins. Numerous studies suggest that lncRNAs significantly contribute to the onset and progression of various forms of cancers. A specific lncRNA, known as long non-coding RNA 01296 (LINC01296), is extensively expressed in human malignancies. The level of LINC01296 has been shown to correlate with the progression and prognosis of cancers. Moreover, numerous scientific investigations have provided evidence that the dysregulation of LINC01296 functioning as a competitive endogenous RNA (ceRNA) exerts a profound influence on various aspects of cancer cell behavior, including proliferation, apoptosis, invasion, metastasis, and cell cycle progression, by means of regulating target genes and signaling pathways. An increasing body of data strongly suggests that LINC01296 may serve as a valuable biomarker for predicting cancer prognosis and could represent a promising therapeutic target for cancer intervention. In this comprehensive review, we summarize the recent advancements in our understanding of the role, underlying mechanisms, and clinical significance of LINC01296 in malignant tumors. The findings suggest that LINC01296 may be both a reliable biomarker and a potential therapeutic target for cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bcfc7646d19e3cb4d25d1197357b2a9f9017093" target='_blank'>
              The Emerging Role of Long Non-Coding RNA 01296 in Human Malignancies.
              </a>
            </td>
          <td>
            Lei Luo, Fan Yang, Xiaoping Fu, Tingting Yu, Wenqian Tang, Juan Xue
          </td>
          <td>2025-01-31</td>
          <td>Current molecular medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b10a2ab1ac270b68e0530d0e78e9d779c180238b" target='_blank'>
              Histone Modifications and DNA Methylation in Psoriasis: A Cellular Perspective.
              </a>
            </td>
          <td>
            Jing Pan, Siji Chen, Xianzhen Chen, Yinjing Song, Hao Cheng
          </td>
          <td>2025-01-27</td>
          <td>Clinical reviews in allergy & immunology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The role of somatic mitochondrial DNA (mtDNA) mutations in leukemogenesis remains poorly characterized. To determine the impact of somatic mtDNA mutations on this process, we assessed the leukemogenic potential of hematopoietic progenitor cells (HPCs) from mtDNA mutator mice (Polg D257A) with or without NMyc overexpression. We observed a higher incidence of spontaneous leukemogenesis in recipients transplanted with heterozygous Polg HPCs and a lower incidence of NMyc-driven leukemia in those with homozygous Polg HPCs compared to controls. Although mtDNA mutations in heterozygous and homozygous HPCs caused similar baseline impairments in mitochondrial function, only heterozygous HPCs responded to and supported altered metabolic demands associated with NMyc overexpression. Homozygous HPCs showed altered glucose utilization with pyruvate dehydrogenase inhibition due to increased phosphorylation, exacerbated by NMyc overexpression. The impaired growth of NMyc-expressing homozygous HPCs was partially rescued by inhibiting pyruvate dehydrogenase kinase, highlighting a relationship between mtDNA mutation burden and metabolic plasticity in leukemogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a692877215b062ddb45f64e78542319dbb82acdd" target='_blank'>
              Somatic mtDNA mutation burden shapes metabolic plasticity in leukemogenesis
              </a>
            </td>
          <td>
            Xiujie Li-Harms, Jingjun Lu, Yu Fukuda, John Lynch, Aditya Sheth, Gautam Pareek, Marcin M Kamiński, Hailey S Ross, Christopher W. Wright, Amber L Smith, Huiyun Wu, Yong-Dong Wang, Marc Valentine, Geoffrey Neale, Peter Vogel, S. Pounds, J. Schuetz, Min Ni, M. Kundu
          </td>
          <td>2025-01-01</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="Glioblastoma (GBM) is described as a group of highly malignant primary brain tumors and stands as one of the most lethal malignancies. The genetic and cellular characteristics of GBM have been a focal point of ongoing research, revealing that it is a group of heterogeneous diseases with variations in RNA expression, DNA methylation, or cellular composition. Despite the wealth of molecular data available, the lack of transferable pre-clinic models has limited the application of this information to disease classification rather than treatment stratification. Transferring the patients' genetic information into clinical benefits and bridging the gap between detailed descriptions of GBM, genotype-phenotype associations, and treatment advancements remain significant challenges. In this context, we present an advanced human GBM organoid model, the Laboratory Engineered Glioblastoma Organoid (LEGO), and illustrate its use in studying the genotype-phenotype dependencies and screening potential drugs for GBM. Utilizing this model, we have identified lipid metabolism dysregulation as a critical milestone in GBM progression and discovered that the microsomal triglyceride transfer protein inhibitor Lomitapide shows promise as a potential treatment for GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3c5689a723411e5e090a7db347716f4db71706a" target='_blank'>
              Laboratory-Engineered Glioblastoma Organoid Culture and Drug Screening.
              </a>
            </td>
          <td>
            Changwen Wang, Nadja Stöffler, Haikun Liu
          </td>
          <td>2025-01-10</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Mutations drive evolution and genetic diversity, but the impact of transcription on germline mutagenesis remains poorly understood. Here, we identify a hypermutation phenomenon at transcription start sites in the human germline, spanning several hundred base pairs in both directions. We link this TSS mutational hotspot to divergent transcription, RNA polymerase II stalling, R-loops, and mitotic—but not meiotic—double-strand breaks, revealing a recombination-independent mechanism distinct from known processes. Notably, the hotspot is absent in de novo mutation data. We reconcile this by showing that TSS mutations are significantly enriched with early mosaic variants often filtered out in de novo mutation calls, indicating that the hotspot arises during early embryogenesis. Mutational signature analysis reinforces these findings and implicates alternative non-homologous end joining and maternal mutation clusters. Our study provides the first detailed description of a germline TSS mutation hotspot, with broad evolutionary and biomedical implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3da1619adeabbe4ded6c11c45076e2a69d9a307" target='_blank'>
              Transcription start sites experience a high influx of heritable variants fuelled by early development
              </a>
            </td>
          <td>
            Miguel Cortés Guzmán, David Castellano, Claudia Serrano Colome, V. Seplyarskiy, Donate Weghorn
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/937fb72f5c07d87edcc2493e093c8ba719c5c0a0" target='_blank'>
              TET2 cascade: a novel regulator of chromatin structure and leukaemogenesis
              </a>
            </td>
          <td>
            Wolfram C. M. Dempke, K. Fenchel
          </td>
          <td>2025-01-06</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background: In eukaryotes with a double-stranded linear DNA genome, the loss of terminal DNA during replication is inevitable due to an end-replication problem; here, telomeres serve as a buffer against DNA loss. Thus, the activation of the telomere maintenance mechanism (TMM) is a prerequisite for malignant transformation. Methods: We compared neurofibroma (NF, benign) and malignant peripheral nerve sheath tumors (MPNSTs) occurring in the same patient with type 1 neurofibromatosis, where each NF–MPNST pair shared the same genetic background and differentiation lineage; this minimizes the genetic bias and contrasts only those changes that are related to malignant transformation. A total of 20 NF–MPNST pairs from 20 NF1 patients were analyzed. Whole-transcriptome sequencing (WTS) was conducted to profile the transcriptional relationship, and whole-genome sequencing (WGS) was performed to measure the telomere length. Results: We identified 22 differentially expressed genes (DEGs) during the malignant transformation of MPNSTs. Among them, NELL2 activated PAX7, which sequentially activated RAD52, the recombinase of RAD52-dependent alternative lengthening of telomeres (ALT). RAD52 elongated MPNSTs–telomeres (p = 0.017). Otherwise, neither NELL2 nor PAX7 affected telomere length (p = 0.647 and p = 0.354, respectively). RAD52 increased MPNSTs–telomeres length, independently of NELL2 and PAX7 in multiple analyses (p = 0.021). The group with increased telomere length during the malignant transformation showed inferior overall survival (OS) (HR = 3.809, p = 0.038) to the group without increased telomere length. Accordingly, the group with increased PAX7 showed inferior OS (HR = 4.896, p = 0.046) and metastasis-free survival (MFS) (HR = 9.129, p = 0.007) in comparison to the group without increased PAX7; the group with increased RAD52 showed inferior MFS (HR = 8.669, p = 0.011) in comparison to the group without increased RAD52. Conclusions: We suggest that the NELL2-PAX7 transcriptional cascade activates RAD52-dependent ALT to increase telomere length during the malignant transformation of MPNSTs, resulting in a poor prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec682f73e8b37948b58b7e542b7ea83e47ad44c3" target='_blank'>
              NELL2-PAX7 Transcriptional Cascade Suggests Activation Mechanism for RAD52-Dependent Alternative Lengthening of Telomeres During Malignant Transformation of Malignant Peripheral Nerve Sheath Tumors: Elongation of Telomeres and Poor Survival
              </a>
            </td>
          <td>
            Jungwoo Lee, Eunji Choi, Hyoju Kim, Young-Joon Kim, Seung Hyun Kim
          </td>
          <td>2025-01-23</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Tumorigenesis and progression are complex process with multifarious molecules and related mechanisms, while most of them are still elusive. Most research into the mechanisms of cancer today are focusing on encoding RNAs. However, for the past few years as the emergence and development of epigenetics, a kind of new molecule: non-coding RNAs (ncRNAs), RNAs without protein-coding function, have gained prominence in recent scientific discussions. Since the first ncRNA was discovered in 1990s, extensive research has shown that ncRNAs significantly influence processes such as cell growth, differentiation, metabolic regulation, and programmed cell death at both transcriptional and post-transcriptional stages. By means of acting as the tumor suppression and oncogenesis, research has identified several ncRNAs with abnormal expression patterns in cancer cells, marking them as primary oncogenic factors or valuable targets in cancer treatment. This review focus on three kinds of ncRNA, MicroRNAs (miRNAs), long non-coding RNAs (lncRNAs), and circular RNAs (circRNAs), the molecule structure, generative mechanism, characteristic function and regulatory mechanism in tumor progression. In addition, we will discuss the application value and future expiration of the therapeutic that are focusing on ncRNA, that will open up a new idea and methods for the treatment of cancer for future investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/026f448482a66bffaadbd5d2efa692a3ff825932" target='_blank'>
              Non-coding RNAs in Tumor Biology: Exploring miRNAs, lncRNAs, and circRNAs Roles and Therapeutic Potentials
              </a>
            </td>
          <td>
            Tianhao Zhang
          </td>
          <td>2025-01-15</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Epigenetic therapies that induce global DNA hypomethylation activate transcriptionally repressed transposable elements (TEs). The activation of a class of TEs known as human endogenous retroviruses (HERVs) can trigger an innate antiviral response in cells called “viral mimicry”. Viral mimicry induced by aberrantly expressed HERVs stimulates interferon signaling genes resulting in an adaptive immune response that can ultimately lead to immunogenic cell death. This heightened immune activity can enhance the efficacy of immune checkpoint inhibitor therapies. Identifying cancers that are susceptible to a viral mimicry response or treatment regimens that induce aberrant transcription of TEs has not yet been fully explored. Recent computational tools enable the measurement of TE signal from short-read RNA-seq data and several large-scale reprocessing pipelines have uniformly processed RNA-seq data of cancer cell lines treated with epigenetic and cytotoxic drugs. However, these large-scale pipelines omit the quantification of TEs. To address this gap and fully utilize publicly available data, we have developed a computational pipeline and database of TE expression for more than 7,000 samples from 220 cancer cell lines treated with over 700 cytotoxic and epigenetic drugs, yielding over 2,100 differential expression experiments. To illustrate the usefulness of this database, we ranked drugs by %TE activation and found that the top three targets were DNMT (Decitabine), CDK9 (MC180295), and Topoisomerase II (Doxorubicin). We enable the exploration of this dataset by implementing several meta-analysis methods and visualizations made available as an interactive web application. This database and web application can serve as a valuable resource for the cancer and epigenetics community by enabling hypothesis generation and providing insights into how specific drugs may induce TE expression.
 Citation Format: Gennaro H Calendo, Morgan Chaunzwa, Iman Dehzangi, Jozef Madzo, Jean-Pierre Issa. TEDEdb: Transposable element differential expression database [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B022.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5443bedc86457ac044103f87e00c65e563c2339" target='_blank'>
              Abstract B022: TEDEdb: Transposable element differential expression database
              </a>
            </td>
          <td>
            G. Calendo, Morgan Chaunzwa, Iman Dehzangi, Jozef Madzo, Jean-Pierre J Issa
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="This research sought to explore the role of epigenetic changes in cancer initiation and progression, the therapeutic potential of epigenetic modulators, i.e., DNMT inhibitors and HDAC inhibitors, and the expression levels of non-coding RNAs, i.e., miRNAs, in controlling epigenetic changes in cancer cells. The research used a quantitative method, i.e., structured questionnaires and regression analysis to determine the role of epigenetic changes in cancer biology. The research validated that 65% of the respondents were familiar with DNA methylation, 60% with histone modification, and 55% with gene-editing technologies like CRISPR. The Chi-Square analysis validated the presence of significant disparity in awareness among demographic groups, with p-values of 0.02 and 0.05 for awareness of DNA methylation by age and awareness of histone modification by medical specialty, respectively. Regression analysis validated that DNMT inhibitors significantly caused cancer cell apoptosis (β = 0.55, p = 0.0002) and suppressed cancer cell migration (β = -0.30, p = 0.02), while HDAC inhibitors also caused apoptosis (β = 0.47, p = 0.005) and promoted migration inhibition (β = -0.25, p = 0.03). The Chi-Square test of expression of miRNAs further validated significant correlations between miR-21 expression and cancer type (χ² = 10.4, p = 0.02) and between miR-34a expression and DNA methylation (χ² = 12.1, p = 0.01). These results suggest the potential of epigenetic therapies and miRNA-based therapies in controlling cancer cell behavior and enhancing therapeutic efficacy. This study highlights epigenetic modifications in cancer, the potential of DNMT and HDAC inhibitors, and miRNAs, urging further research on optimization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/869a71d30138952ce61567951ce4b3e25c0e546d" target='_blank'>
              The Role of Epigenetic Modifications in Cancer Development and Progression: Potential Therapeutic Approaches
              </a>
            </td>
          <td>
            Imtiaz Ali Soomro, Quratulain Badar, Wajid Ali, Adil khan, Syed Rizvan Ali, Waqas Mahmood
          </td>
          <td>2024-12-31</td>
          <td>Indus Journal of Bioscience Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Aging is a process of systemic deterioration and the most significant risk factor for cancers. Clonal hematopoiesis (CH) commonly occurs with aging and links to higher mortality, leukemia risk, and cardiovascular diseases. Age-related CH involves the abnormal clonal expansion of hematopoietic stem cells (HSCs) bearing somatic mutations in genes frequently mutated in leukemia, including genes encoding epigenetic regulators, e.g., the DNA demethylase TET2. While such mutations are known to alter the HSC epigenome, the mechanisms through which these mutations drive HSC self-renewal and myeloid transformation preferentially in old age remain elusive. In this study, we hypothesize that aging and CH mutations cooperatively reshape the HSC transcriptomic landscape, enhancing HSC competitive advantage that facilitates clonal expansion. Using single-cell multi-omic analyses and flow cytometry, we demonstrated, for the first time, that Tet2 deficiency mitigates detrimental HSC aging processes at the transcriptomic, epigenomic, and cellular levels, thereby preserving HSC function and promoting age-dependent expansion. Age-related gene expression changes, including increased Eya4, Lars2, and Mical2 and decreased Cdk6, CD53, and Sox4, were mitigated in Tet2-deficient HSCs. These gene expression changes were confirmed by single-nucleus ATAC-seq. Chromatin accessibility analysis also revealed Tet2 deficiency predominantly mitigates epigenetic changes associated with HSC aging. Specifically, age-related increases in chromatin accessibility of binding motifs of transcription factors (TFs), such as E2f4, Ppard, Mycn, and Max, and decreases in binding motifs of chromatin accessibility for Gabpa, Runx3, Etv4, and Elf1, were also mitigated in Tet2-deficient HSCs. Furthermore, DNA methylation status of these TF binding motifs is linked to the regulation for the targets’ gene expression, and CTCF insulator binding during HSC aging are also mitigated by Tet2 deficiency. Together, Tet2 deficiency stabilizes TF-target gene pairs critical for maintaining HSC self-renewal during aging. In conclusion, our findings reveal that Tet2 deficiency significantly contributes to the mitigation of HSC aging and hijacks the HSC expansion strategy during HSC aging via epigenetic reprogramming, which contributes to age-related clonal hematopoiesis. By elucidating the transcriptomic and epigenomic alterations in Tet2-deficient HSCs, our study provides novel insights into how age and somatic mutations interact to promote the pathogenesis of age-related hematological diseases. These discoveries enhance our understanding of how aging and somatic mutations interact, offering insights into potential biomarkers and therapeutic targets for age-associated hematological disorders.
 Citation Format: Shilpita Karmakar, Efthymios Motakis, Yang Liu, Kaustav Sengupta, Lamis Naddaf, Travis Roeder, Brandon Vu, Kristina Mujica, Ming Xu, Eric Pietrias, James DeGregori, Jennifer Trowbridge, Hideyuki Oguro, Sheng Li. Tet2 deficiency mitigates epigenetic aging in clonal hematopoiesis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr A014.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b4766c6ad595902085d51a487e2edbf0d4dfa6d" target='_blank'>
              Abstract A014: Tet2 deficiency mitigates epigenetic aging in clonal hematopoiesis
              </a>
            </td>
          <td>
            Shilpita Karmakar, Efthymios Motakis, Yang Liu, Kaustav Sengupta, Lamis Naddaf, Travis Roeder, Brandon Vu, Kristina D. Mujica, Ming Xu, Eric Pietrias, James DeGregori, Jennifer J. Trowbridge, Hideyuki Oguro, Sheng Li
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Numerical abnormalities in chromosomal states, referred to as aneuploidy, is commonly observed in many cancer cells. Although numerous internal and external factors induce aneuploidy, the primary cause of aneuploidy in humans remains unclear. DNA damage is identified as a potential cause of aneuploidy by inducing chromosome segregation errors. However, a direct relationship between DNA damage and aneuploidy remains poorly understood. A major reason for this is the extremely low frequency of aneuploidy in cultured cells, making quantitative analyses challenging. In this study, we investigated the relationship between DNA damage and aneuploidy in cell lines containing minichromosomes. These chromosomes are more prone to loss than normal chromosomes, with the rate of loss substantially increased following exposure to various DNA-damaging agents. To determine whether damaged chromosomes were subjected to direct loss or whether chromosome loss occurred as an indirect consequence of a prolonged G2 phase or other factors, we used the CRISPR-Cas9 system to introduce a single DNA double-strand break (DSB) on a minichromosome. The rate of minichromosome loss increased by approximately seven-fold compared with that of the control. Furthermore, the loss rate was significantly elevated in the absence of KU70, a key factor in non-homologous end joining, and upon inhibition of ataxia telangiectasia mutated (ATM), a DNA damage checkpoint protein. Finally, two closely spaced nicks, believed to generate a 5’-overhang, were also shown to induce minichromosome loss. These findings indicated that a single DSB or two closely spaced nicks can cause aneuploidy if improperly repaired in vertebrates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/657f1468476733a77f4e32e0f0402d6f84ad1e95" target='_blank'>
              Quantitative analysis of the frequency of chromosome loss following DSB induction
              </a>
            </td>
          <td>
            Seiya Matsuno, Ryo Ishida, Ryotaro Kawasumi, Kouji Hirota, Takuya Abe
          </td>
          <td>2025-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="ABSTRACT Chromosomal aneuploidies are a major cause of developmental failure and pregnancy loss. To investigate the possible consequences of aneuploidy on early embryonic development in vitro, we focused on primed pluripotent stem cells that are relatable to the epiblast of post-implantation embryos in vivo. We used human induced pluripotent stem cells (iPSCs) as an epiblast model and altered chromosome numbers by treating with reversine, a small-molecule inhibitor of monopolar spindle 1 kinase (MSP1) that inactivates the spindle assembly checkpoint, which has been strongly implicated in chromosome mis-segregation and aneuploidy generation. Upon reversine treatment, we obtained cells with varied chromosomal content that retained pluripotency and potential to differentiate into cells of three germ lineages. However, these cells displayed lagging chromosomes, increased micronuclei content, high p53 expression and excessive apoptotic activity. Cell proliferation was not affected. Prolonged in vitro culture of these cells resulted in a selective pool of cells with supernumerary chromosomes, which exhibited cellular hypertrophy, enlarged nuclei, and overproduction of total RNAs and proteins. We conclude that increased DNA damage responses, apoptosis, and improper cellular mass and functions are possible mechanisms that contribute to abnormal epiblast development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d165ff1955e4cb3146937b90c98c9776c59f3b72" target='_blank'>
              Chromosome number alterations cause apoptosis and cellular hypertrophy in induced pluripotent stem cell models of embryonic epiblast cells
              </a>
            </td>
          <td>
            Althea Stella Anil Martis, Loshini Soundararajan, Pallavi Shetty, Syed Moin, Tejashree Vanje, Yogeshwaran Jai Sankar, Shagufta Parveen
          </td>
          <td>2025-01-15</td>
          <td>Biology Open</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background: CRISPR/Cas9 technology has rapidly advanced as a pivotal tool in cancer research, particularly in the precision targeting required for both detecting and treating malignancies. Its high specificity and low off-target effects make it exceptionally effective in applications involving Human Papillomavirus (HPV) related diseases, most notably cervical cancer. This approach offers a refined methodology for the rapid detection of viral infections and provides a robust platform for the safe and effective treatment of diseases associated with viral infections through gene therapy. Purpose: Gene therapy, within this context, involves the strategic delivery of genetic material into target cells via a vector. This is followed by the meticulous modulation of gene expression, whether through correction, addition, or suppression, specifically honed to target tumor cells while sparing healthy cells. This dual capacity to diagnose and treat at such a precise level underscores the transformative potential of CRISPR/Cas9 in contemporary medical science, particularly in oncology and virology. Research Design: This article provides an overview of the advancements made in utilizing the CRISPR-Cas9 system as a research tool for HPV-related treatments while summarizing its application status in basic research, diagnosis, and treatment of HPV. Data Collection: Furthermore, it discusses the future prospects for this technology within emerging areas of HPV research and precision medicine in clinical practice, while highlighting technical challenges and potential directions for future development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c69224d4406f31958e4148b5cda3be164a3c3d00" target='_blank'>
              Research Status of Clustered Regulary Interspaced Short Palindromic Repeats Technology in the Treatment of Human Papillomavirus (HPV) Infection Related Diseases
              </a>
            </td>
          <td>
            Minxue Tang
          </td>
          <td>2025-01-01</td>
          <td>Cancer Control : Journal of the Moffitt Cancer Center</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b52af98ba891142904f59178c596ff51575258f" target='_blank'>
              Rapid and scalable personalized ASO screening in patient-derived organoids
              </a>
            </td>
          <td>
            John C Means, A. Martinez-Bengochea, D. Louiselle, Jacqelyn Nemechek, John M Perry, Emily G Farrow, T. Pastinen, Scott T Younger
          </td>
          <td>2025-01-22</td>
          <td>Nature</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="The DREAM (dimerization partner, RB-like, E2F, and multi-vulval class B) complex is an evolutionarily conserved transcriptional repression complex that coordinates nearly one thousand target genes, primarily associated with the cell cycle processes. The formation of the DREAM complex consequently inhibits cell cycle progression and induces cellular quiescence. Given its unique role in cell cycle control, the DREAM complex has gained significant interest across various physiological and pathological contexts, particularly in conditions marked by dysregulated cell cycles, such as cancer. However, the specific cancer types most significantly affected by alterations in the DREAM complex are yet to be determined. Moreover, the possibility of restoring or pharmacologically targeting the DREAM complex as a therapeutic intervention against cancer remains a relatively unexplored area of research and is currently under active investigation. In this review, we provide an overview of the latest advances in understanding the DREAM complex, focusing on its role in cancer. We also explore strategies for targeting the DREAM complex as a potential approach for cancer therapeutics. Advances in understanding the precise role of the DREAM complex in cancer, combined with ongoing efforts to develop targeted therapies, may pave the way for new options in cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08f83080159a75138829f6897856a0dcb6a27d73" target='_blank'>
              Emerging Role of the DREAM Complex in Cancer and Therapeutic Opportunities
              </a>
            </td>
          <td>
            Ye-Jin Hwang, Moon Jong Kim
          </td>
          <td>2025-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Mutations found in AML such as DNMT3A, TET2 and ASXL1 can be found in the peripheral blood of healthy adults - a phenomenon termed clonal hematopoiesis (CH). These mutations are thought to represent the earliest genetic events in the evolution of AML. Genomic studies on samples acquired at diagnosis, remission, and at relapse have demonstrated significant stability of CH mutations following induction chemotherapy. Meanwhile, later mutations in genes such as NPM1 and FLT3, have been shown to contract at remission and in the case of FLT3 often are absent at relapse. We sought to understand how early CH mutations influence subsequent evolutionary trajectories throughout remission and relapse in response to induction chemotherapy. We assembled a retrospective cohort of patients diagnosed with de novo AML at our institution that underwent genomic sequencing at diagnosis, remission and/or relapse (total n=182 patients). FLT3 and NPM1 mutations were generally eliminated at complete remission but subsequently reemerged upon relapse, whereas DNMT3A, TET2 and ASXL1 mutations often persisted through remission. CH-related mutations exhibited distinct constellations of co-occurring genetic alterations, with NPM1 and FLT3 mutations enriched in DNMT3Amut AML, while CBL and SRSF2 mutations were enriched in TET2mut and ASXL1mut AML, respectively. In the case of NPM1 and FLT3 mutations, these differences vanished at the time of complete remission yet readily reemerged upon relapse, indicating the reproducible nature of these genetic interactions. Thus, CH-associated mutations that likely precede malignant transformation subsequently shape the evolutionary trajectories of AML through diagnosis, therapy, and relapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb0b511bf0bfbb32bc9fd1061ccf533beadb0c77" target='_blank'>
              Persistent post-remission clonal hematopoiesis shapes the relapse trajectories of acute myeloid leukemia.
              </a>
            </td>
          <td>
            Ryan D Chow, Priya Velu, Safoora Deihimi, Jonathan P Belman, Angela Youn, Nisargbhai Shah, Selina M Luger, Martin P Carroll, Jennifer J D Morrissette, Robert L. Bowman
          </td>
          <td>2025-02-12</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b2f332da3db2ea7ccd5f5a9efe0e7c8eb081b95" target='_blank'>
              TP53 mutations and MDM2 polymorphisms in breast and ovarian cancers: amelioration by drugs and natural compounds.
              </a>
            </td>
          <td>
            Rituraj Chakraborty, Anupam Dutta, Rupak Mukhopadhyay
          </td>
          <td>2025-01-11</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Senescence is a tumor suppressor mechanism triggered by oncogene expression and chemotherapy treatment. It orchestrates a definitive cessation of cell proliferation through the activation of the p53-p21 and p16-Rb pathways, coupled with the compaction of proliferative genes within heterochromatin regions. Some cancer cells have the ability to elude this proliferative arrest but the signaling pathways involved in circumventing senescence remain to be characterized. We have recently described that malignant cells capable of evading senescence have an increased expression of specific tRNAs, such as tRNA-Leu-CAA and tRNA-Tyr-GTA, alongside the activation of their corresponding tRNA ligases, namely LARS and YARS. We have previously shown that YARS promotes senescence escape by activating proliferation and cell cycle genes but its functions during this proliferative arrest remain largely unknown. In this study, we have continued to characterize the functions of YARS, describing non-canonical transcriptional functions of the ligase. Our results show that YARS is present in the nucleus of proliferating and senescent cells and interacts with the Trim28 transcriptional regulator. Importantly, YARS binds to the LIN9 promoter, a critical member of the Dream complex responsible for regulating cell cycle gene transcription. The ligase facilitates the binding and the phosphorylation of the type II RNA polymerase and promotes the deposition of activating epigenetic marks on the LIN9 promoter. Consequently, during senescence escape, YARS activates LIN9 expression and both proteins are necessary to induce the proliferation of emergent cells. These results underscore unconventional transcriptional functions of YARS in activating LIN9 expression in proliferating cells and during senescence escape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c36e3380440225cd14c09aec61c163d6f628c872" target='_blank'>
              A non-canonical role for the tyrosyl tRNA synthetase: YARS regulates senescence induction and escape and controls the transcription of LIN9.
              </a>
            </td>
          <td>
            H. Coquelet, Geraldine Leman, Amine Maarouf, Coralie Petit, Bertrand Toutain, Cécile Henry, Alice Boissard, Catherine Guette, Eric Lelièvre, Pierre Vidi, Jordan Guillon, O. Coqueret
          </td>
          <td>2025-01-05</td>
          <td>The FEBS journal</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="REV7, also known as MAD2B, MAD2L2, and FANCV, is a HORMA-domain family protein crucial to multiple genome stability pathways. REV7’s canonical role is as a member of polymerase ζ, a specialized translesion synthesis polymerase essential for DNA damage tolerance. REV7 also ensures accurate cell cycle progression and prevents premature mitotic progression by sequestering an anaphase-promoting complex/cyclosome activator. Additionally, REV7 supports genome integrity by directing double-strand break repair pathway choice as part of the recently characterized mammalian shieldin complex. Given that genome instability is a hallmark of cancer, it is unsurprising that REV7, with its numerous genome maintenance roles, is implicated in multiple malignancies, including ovarian cancer, glioma, breast cancer, malignant melanoma, and small-cell lung cancer. Moreover, high REV7 expression is associated with poor prognoses and treatment resistance in these and other cancers. Promisingly, early studies indicate that REV7 suppression enhances sensitivity to chemotherapeutics, including cisplatin. This review aims to provide a comprehensive overview of REV7’s myriad roles in genome maintenance and other functions as well as offer an updated summary of its connections to cancer and treatment resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67ab376b5318f868977c071b0b61bc16785632df" target='_blank'>
              REV7: a small but mighty regulator of genome maintenance and cancer development
              </a>
            </td>
          <td>
            Lara Maggs, Mitch McVey
          </td>
          <td>2025-01-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Spermatogenesis is a complex process that can be disrupted by genetic and epigenetic changes, potentially leading to male infertility. Recent research has rapidly increased the number of protein coding mutations causally linked to impaired spermatogenesis in humans and mice. However, the role of non-coding mutations remains largely unexplored. As a case study to evaluate the effects of non-coding mutations on spermatogenesis, we first identified an evolutionarily conserved topologically associated domain (TAD) boundary near two genes with important roles in mammalian testis function: Dmrtb1 and Lrp8. We then used CRISPR-Cas9 to generate a mouse line where 26 kb of the boundary was removed including a strong and evolutionarily conserved CTCF binding site. ChIP-seq and Hi-C experiments confirmed the removal of the CTCF site and a resulting mild increase in the DNA-DNA interactions across the domain boundary. Mutant mice displayed significant changes in testis gene expression, higher frequency of histological abnormalities, a drop of 47-52% in efficiency of meiosis, a 15-18% reduction in efficiency of spermatogenesis, and consistently, a 12-28% decrease in daily sperm production compared to littermate controls. Despite these quantitative changes in testis function, mutant mice show no significant changes in fertility. This suggests that non-coding deletions affecting testis gene regulation may have smaller effects on fertility compared to coding mutations of the same genes. Our results demonstrate that disruption of a TAD boundary can have a negative impact on sperm production and highlight the importance of considering non-coding mutations in the analysis of patients with male infertility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15762b1599e500696a63ae395a5142469b79eb41" target='_blank'>
              Deletion of an evolutionarily conserved TAD boundary impacts spermatogenesis in mice†.
              </a>
            </td>
          <td>
            Ana C. Lima, M. Okhovat, Alexandra M. Stendahl, Ran Yang, Jake VanCampen, K. Nevonen, Jarod Herrera, Weiyu Li, L. Harshman, Lev M. Fedorov, Katinka A. Vigh-Conrad, N. Ahituv, Donald F. Conrad, Lucia Carbone
          </td>
          <td>2025-02-04</td>
          <td>Biology of reproduction</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="In renewing tissues, mutations conferring selective advantage may result in clonal expansions. In contrast to somatic tissues, mutations driving clonal expansions in spermatogonia (CES) are also transmitted to the next generation. This results in an effective increase of de novo mutation rate for CES drivers. CES was originally discovered through extreme recurrence of de novo mutations causing Apert syndrome. Here, we develop a systematic approach to discover CES drivers as hotspots of human de novo mutation. Our analysis of 54,715 trios ascertained for rare conditions, 6,065 control trios and population variation from 807,162 mostly healthy individuals identifies genes manifesting rates of de novo mutations inconsistent with plausible models of disease ascertainment. We propose 15 genes hypermutable at loss-of-function (LoF) sites as candidate CES drivers. Additional 17 genes feature hypermutable missense mutations at individual positions, suggesting CES acting through gain-of-function (GoF). Among candidates are 5 of 13 known CES drivers, 11 cancer drivers, and 16 members of major signaling pathways; notably, 12 genes show CES evidence in direct sperm sequencing. CES increases the average mutation rate ~16-fold for LoF genes in both control trios and sperm and ~500-fold for pooled GoF sites in sperm. Positive selection in the male germline elevates the prevalence of genetic disorders and increases polymorphism levels, masking the effect of negative selection in human populations. Despite the excess of mutations in disease cohorts for 15 LoF CES driver candidates, only 5 show clear evidence of disease causality, suggesting that CES may lead to false-positive disease associations">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebbede9fcc8090e566844fd84a8257e56652782d" target='_blank'>
              Cohort-level analysis of human de novo mutations points to drivers of clonal expansion in spermatogonia
              </a>
            </td>
          <td>
            V. Seplyarskiy, M. A. Moldovan, E. M. Koch, P. Kar, M. D. Neville, R. Rahbari, S. Sunyaev
          </td>
          <td>2025-01-05</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Head and Neck Squamous Cell Carcinoma (HNSCC) is the sixth most common cancer worldwide, and local failure of disease following treatment with radiation therapy remains a major challenge for improving patient outcomes. We therefore designed a CRISPR-Cas9 pooled genetic screen to identify signaling mechanisms that regulate HNSCC radiosensitivity. Using a kinome gRNA library (763 genes with 8 gRNAs each), Cal27 and Detroit562 cells were screened using fractionated exposure to ionizing radiation as a selection pressure. Deep sequencing results identified significant enrichment or depletion of gRNAs for DNA damage signaling (ATM and DNAPK) and NFKB signaling (IRAK4, IRAK1, IKKA, IKKB), pathways known to regulate cell survival after radiation. Surprisingly, the screen also identified significant depletion of Janus kinase signaling (JAK1 and TYK2) to be a cause of radiation resistance. Independent knockout (KO) of Janus Kinase 1 (JAK1) in both Cal27 and Detroit562 cells caused radioresistance in vitro and in vivo. JAK1 KO prolongs G2 cell cycle arrest in HNSCC after treatment with radiation or after thymidine synchronization. JAK1 KO cells also have enhanced Rad51 foci formation indicative of homologous recombination usage for DNA repair. Consistent with an enhanced mitotic arrest, JAK1 KO causes the appearance of cells with 4N and 8N DNA content after exposure to radiation as measured by flow cytometry with propidium iodide or following 5-ethynyl-2’deoxyuridine (EdU) labeling. HNSCC cells with JAK1 KO, or treated with the JAK1 specific inhibitor Abrocitinib, formed significantly fewer micronuclei after radiation treatment consistent with a reduction in mitotic errors. Consistent with these findings, live cell imaging demonstrated cells with JAK1 KO undergo fewer mitotic catastrophe events. Loss of JAK1 function did not alter CDK1, but delayed activation of Aurora Kinase A (AURKA) and Polo-like Kinase 1 (PLK1), identifying a cell cycle signaling pathway regulated by JAK1. Live cell imaging also showed a more than doubling of the time required from chromatin condensation to sister chromatid separation, suggesting an associated defect in kinetochore assembly. The effects of inhibiting Kif18a, a kinesin that regulates kinetochore tension, in combination with radiation was therefore tested. Kif18 inhibition significant radiosensitized JAK1 KO HNSCC in clonogenic survival and xenograft models. In summary we have elucidated a JAK1-dependent mechanism for the regulation of AURKA/PLK1 signaling, a protective G2 arrest, and radioresistance. Furthermore, we provide a mechanistic and targeted pharmacologic approach for overcoming therapeutic resistance and enhancing radiation therapy in HNSCC.
 Citation Format: Vanessa Kelley, Marta Baro, William Gasperi, Chatchai Phoomak, Hojin Lee, Joseph Contessa. CRISPR-Cas9 screening reveals a novel JAK1 dependent mechanism of radioresistance in head and neck squamous cell carcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr B024.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab6adeb5ebe92c5746446bc4ffdb501c4fd87e05" target='_blank'>
              Abstract B024: CRISPR-Cas9 screening reveals a novel JAK1 dependent mechanism of radioresistance in head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Vanessa Kelley, Marta Baro, William Gasperi, C. Phoomak, Hojin Lee, Joseph Contessa
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Metabolism is the name given to all of the chemical reactions in the cell involving thousands of proteins, including enzymes, receptors, and transporters. Inborn errors of metabolism (IEM) are caused by defects in the production and breakdown of proteins, fats, and carbohydrates. Micro ribonucleic acids (miRNAs) are short non-coding RNA molecules, ⁓19-25 nucleotides long, hairpin-shaped, produced from DNA. They play key roles in regulating gene expression of target mRNAs at both transcriptional and post-transcriptional levels. Altered expression of miRNAs has emerged as an additional molecular mechanism implicated in the pathogenesis of many diseases, this altered miRNA expression is also present in inherited metabolic diseases. A single miRNA can regulate targets associated with similar cellular processes and pathways, making miRNAs powerful therapeutics to restore the impaired cellular functions seen in disease phenotypes by enhancing the cellular response. Although the miRNA research field has advanced significantly in recent years, studies in IEM are still limited. Further research on miRNA expression specifically related to IEM may allow the identification of new biomarkers for the diagnosis, progression, and prognosis of diseases. In this review, the literature studies between miRNAs and IEMs, diagnosis, prognosis, follow-up, and treatment possibilities of miRNAs were analyzed. We present recent advances on miRNAs in IEM and an overview of current miRNA therapeutics for the clinic.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6176f135941afbcf0385bbe268cda37e8759c7b8" target='_blank'>
              Effects of miRNAs in inborn error of metabolism and treatment strategies.
              </a>
            </td>
          <td>
            Harun Bayrak, Parisa Sharafi, A. Ç. Özketen, Mustafa Kılıç
          </td>
          <td>2025-01-07</td>
          <td>Postgraduate medical journal</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancer therapies have attempted to target the transcription factor p53, a gene also described as the “guardian of the genome,” for decades. However, the approach has faced numerous barriers to clinical efficacy due to several factors: mutations in p53 occur in almost half of all human cancers, mutations are cancer-specific, and the associated genomic changes grant mutant p53 with oncogenic potential unique from that of wild-type p53. A host of new therapeutic agents have emerged that work to target mutant p53. These agents can broadly be classified into six categories: the viral approach, direct modifiers of the p53 pathway, epigenetic modifiers of the p53 pathway, synthetic lethal agents, structural reactivators, and immune activating vaccines. Even these strategies have been met with limited success. Bypassing p53 entirely may be the next avenue in cancer therapeutics to kill tumor cells regardless of p53’s mutation pattern.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25b1fb68a093354533852b3fe76935ba3e2890db" target='_blank'>
              Side-stepping the guardian of the genome: current cancer therapeutics targeting mutant p53
              </a>
            </td>
          <td>
            I. Taritsa, Eric T. Fossel
          </td>
          <td>2025-01-29</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Background: Clonal hematopoiesis of indeterminate potential (CHIP) is characterized by somatic mutations (e.g. DNMT3A, TET2, and ASXL1I) in hematopoietic stem cells, and linked to age-related clonal expansions and inflammatory states. We previously showed that patients with CHIP receiving CAR-T cell therapy have a four-fold risk of developing clinically significant cytokine release syndrome (CRS) compared to those without CHIP (Transplant Cell Ther 2024). This prompted us to investigate the mechanisms through which CHIP mutations could contribute to CAR-T-related toxicities. Methods: We employed CRISPR-based gene editing to target CHIP-associated genes—DNMT3A, TET2, and ASXL1—in M1-like macrophages, which were differentiated from human monocytes in the presence of 20 ng/mL of GM-CSF. Gene perturbation was achieved with cell-penetrating phosphorothioated single-stranded DNA oligonucleotides to deliver the CRISPR-Cas9 and gRNA complex (PS-Cas9/gRNA). Editing was performed using multiple gRNAs for each gene: DNMT3A (2 gRNAs), TET2 (3 gRNAs), and ASXL1 (2 gRNAs), and decreased protein expression was confirmed post-editing. We co-cultured CS1 CAR-T cells targeting multiple myeloma (MM), MM.1S cells, and edited autologous macrophages at a 1:1:0.1 ratio for 24 hours to assess the inflammatory response. Cytokine secretion was quantified using the LEGENDplex™ Human Inflammation Panel 1 (13-plex). Results: Compared to non-edited macrophages, DNMT3A-, TET2-, and ASXL1-edited macrophages consistently secreted significantly higher levels of IL-6 (642, 920, 1338 pg/ml vs. 325 pg/ml respectively, P<0.05-0.001), MCP-1(90, 122, 151 pg/ml vs. 55 pg/ml, respectively, P<0.01-0.001), TNFα (2015, 1722, 1874 pg/ml vs. 1043 pg/ml, respectively P:ns - < 0.001), IL-1β (75, 99, 146 pg/ml vs. 38 pg/ml, respectively, P<0.05-0.001), and IL-8 (21, 19, 22 ng/ml vs. 15 ng/ml, respectively, P:ns- <0.001). Conclusions: These findings suggest CHIP mutations in human macrophages may exacerbate the inflammatory response in CAR-T therapy, highlighting a potential biomarker for stratifying patients at higher risk for toxicities such as CRS and guiding more personalized prophylactic interventions.
 Citation Format: Xiuli Wang, Scott R. Goldsmith, June-Wha Rhee, Stephen J. Forman, Chunyan Zhang, Saro Armenian. CRISPR-based investigation of CHIP mutations reveals their role in CAR-T therapy-related inflammatory toxicities [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr A019.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/622934218386f9dc97c13a0c545169d234cb7502" target='_blank'>
              Abstract A019: CRISPR-based investigation of CHIP mutations reveals their role in CAR-T therapy-related inflammatory toxicities
              </a>
            </td>
          <td>
            Xiuli Wang, S. Goldsmith, J. Rhee, Stephen J Forman, Chunyan Zhang, S. Armenian
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Lung cancer remains the leading cause of cancer-related deaths worldwide, driven by its complexity and the heterogeneity of its subtypes, which influence pathogenesis, tumor microenvironment, and genetic alterations. We developed a novel weighted gene regulatory network reconstruction method based on maximum entropy and Markov chain entropy principles, which integrates gene expression and DNA methylation data to generate biologically informed networks. Applied to LUAD and LUSC datasets, we define a network methylation index to determine whether gene methylation acts as oncogenic or tumor-suppressive. By revealing a stable core set of pathogenic genes, we identify not only genes with significant expression changes, such as CD74 and HGF, but also pathogenic genes with stable expression, such as BRAF and KDM6A. Additionally, we uncover potential driver genes, such as CORO2B and C20orf194, associated with disease stage, gender, and smoking status. This method offers a more comprehensive understanding of NSCLC mechanisms, paving the way for improved therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9cc71fe2bac06e5363ad9c2b40b91ef9226b861" target='_blank'>
              Weighted Gene Networks Derived from Multi-Omics Reveal Core Cancer Genes in Lung Cancer
              </a>
            </td>
          <td>
            Qingcai He, Zhilong Mi, Ziqiao Yin, Zhiming Zheng, Binghui Guo
          </td>
          <td>2025-02-20</td>
          <td>Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 The lysine methyltransferase KMT2A/MLL1 plays a crucial role in normal development and hematopoiesis. Rearrangement of KMT2A results in the formation of a KMT2A fusion protein with aberrant activity, which is a well-characterized driver of leukemogenesis and is therapeutically targetable. In contrast to the widely studied KMT2A translocation, the molecular consequences of other KMT2A alterations, including KMT2A amplification, remain unknown. KMT2A amplification is detected in highly aggressive and treatment-resistant myelodysplastic syndrome (MDS) and acute myeloid leukemia (AML). It strongly associates with TP53 mutations and complex karyotypes. Unlike KMT2A rearrangement, KMT2A amplification is reported mostly in patients older than 65 and treated with alkylating agents. Here, we conducted a comprehensive molecular analysis of KMT2A-amplified MDS/AML by long-read nanopore sequencing. We have revealed the unique epigenetic landscape, including DNA methylation, histone modification, and chromatin accessibility, associated with KMT2A amplification. We also explored the potential of targeting KMT2A or KMT2A-medicated pathways to block leukemogenesis. These results provide a foundation for targeting the KMT2A complex in KMT2A-amplified disease, representing a promising first step toward treating MDS/AML associated with KMT2A amplification.
 Citation Format: Jayme Ogino, Yang Liu, Wentao Yang, Madina Sukhanova, Sheng Li, Yali Dou. Epigenetic dynamics in KMT2A-amplified leukemia [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr A008.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/245971e55eeef33c3dd5df76ff7015895c3cd83a" target='_blank'>
              Abstract A008: Epigenetic dynamics in KMT2A-amplified leukemia
              </a>
            </td>
          <td>
            Jayme Ogino, Yang Liu, Wentao Yang, M. Sukhanova, Sheng Li, Yali Dou
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0cafa641073896bd378d089dc135baed57213473" target='_blank'>
              NLRP7 maintains the genomic stability during early human embryogenesis via mediating alternative splicing
              </a>
            </td>
          <td>
            Zhongliang Chen, Liangxia Jiang, Min Su, Qibing Zeng, Peng Luo, Liangzhao Chu
          </td>
          <td>2025-01-26</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 406


 Background:
 Specific gene fusions and their fusion products (chimeric transcripts and proteins) have served as ideal diagnostic markers and therapeutic targets for cancer. However, few systematic studies for chimeric transcriptome have been conducted in prostate cancer (PCa), and none in the Chinese populations.
 Methods:
 We analyzed raw RNA-seq data from PCa patients in the Chinese Prostate Cancer Genome Epigenome Atlas (CPGEA) to predict chimeric RNAs using the Eriscript. Fluorescence-activated cell sorting (FACS) was employed to isolate tumor cells, cancer-associated fibroblasts (CAFs), tumor-associated macrophages (TAMs), and T cells from fresh clinical specimens. Cellular functional assays, western blotting, FISH, electron microscopy, RNA pull-down, and immunofluorescence were employed to elucidate the underlying mechanisms by which the chimeric SFT2D2-TBX19 regulates prostate cancer progression.
 Results:
 Over 100,000 chimeric RNAs have been predicted in CPGEA. After several rounds of screening, we identified 301 of them with differential expression between cancerous and adjacent normal tissues, and 101 out of them were validated in clinical samples. About 20% showed prognosis correlation, indicating their potential biomarker values. Further experiments found differential expression of those chimeras among cancer epithelial cells, CAFs, TAMs, and T cells. We further examined the effect of these chimeric RNAs, and found examples of chimeric RNAs directly regulates the growth and motility of tumor cells, and regulate communications between tumor cells and their microenvironment, which jointly facilitate tumor progression. An important chimeric RNA,
 SFT2D2-TBX19
 , was identified as encoding the TBX19-202 protein. Both TBX19-202 and its parental TBX19, which share homologous amino acid sequences, enhance prostate cancer cell proliferation, migration and invasion. Additionally,
 SFT2D2-TBX19
 also can work as a lncRNA which could interact with the ATP synthase F1 subunit ATP5F1A, increasing ATP5F1A phosphorylation, which stabilizes the interaction between ATP5F1A and ATP5F1B. The region spanning 1801-2400bp of
 SFT2D2-TBX19
 and the intermediate structural domain of ATP5F1A are crucial functional areas. This stabilization of ATP5F1A and ATP5F1B enhances mitochondrial ATP synthase activity and ATP production, thus maintaining prostate cancer cell proliferation.
 Conclusions:
 These findings established the atlas of chimeric transcriptome in PCa from the Chinese populations, and provided a large list of validated chimeric RNAs that have the potentials of becoming novel biomarkers and/or treatment targets. Our research provides comprehensive evidence that the
 SFT2D2-TBX19
 promotes prostate cancer progression by encoding the TBX19-202 protein and stabilizing mitochondrial ATP synthase through ATP5F1A phosphorylation.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc18fdd9c089adba620555625ca0a5c806af9c3f" target='_blank'>
              Chimeric RNA landscape in the Chinese population and the role of
 SFT2D2-TBX19
 in prostate cancer progression.
              </a>
            </td>
          <td>
            Qiong Wang
          </td>
          <td>2025-02-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 For the last thirty years an increasing amount of evidence has shown the relevance of epigenetics in cell biology and tissue physiology, being DNA methylation aberrations in cancer the flag-ship for the recognition of its disturbance in human diseases. From the candidate gene approaches, new powerful technologies such as comprehensive DNA methylation microarrays and whole genome bisulfite sequencing has recently emerged that have reinforced the notion of epigenetic disruption in the crossroad of many sickness. From the poster-boy case of MGMT hypermethylation in the prediction of alkylating drug response to the personalized treatment of leukemia with small molecules targeted to fusion proteins involving histone modifiers, the field has walked a long path. The current talk will focus in the epigenetic profiling, basically at the level of DNA methylation and histone modifications, that is starting to provide clinical value in the diagnosis, prognosis and prediction of response to drug therapies. For cancer, we have already a wide view of the undergoing DNA methylation events that expand beyond classical promoter CpG islands of tumor suppressor genes and we have a growing list of mutated chromatin remodeler genes that contributes to the tumorigenesis process. It is time to apply this knowledge in practical clinical situations like the diagnosis of cancers of unknown primary, the screening of malignancies in high-risk populations or a biomarker selection of the patients that should receive treatment with anticancer drugs, including immunotherapy. Beyond cancer, DNA methylation is starting to be recognized as playing a major role in infectious diseases, and in this regard, the present lecture will also address the epigenomic component of COVID-19. Most important, Epigenomics can be added to Genomics, Transcriptomics and Proteomics plus Bioinformatics to create a Multomics view of human diseases.
 Citation Format: Manel Esteller. DNA methylation in cancer: From knowledge to applications [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr IA007">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/855f061aa62959677f73a3c7e17c250ce96d93c4" target='_blank'>
              Abstract IA007: DNA methylation in cancer: From knowledge to applications
              </a>
            </td>
          <td>
            M. Esteller
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Coordinated initiation of DNA replication is essential to ensure efficient and timely DNA synthesis. Yet, molecular mechanism describing how replication initiation is coordinated in eukar-yotic cells is not completely understood. Herein, we present data demonstrating a novel feature of RNAs transcribed in the proximity of actively replicating gene loci. We show that RNAs aN-Choring ORC1 (ANCORs) to the histone variant H2A.Z are licensors of the DNA replication process. This ANCOR-H2A.Z interaction is essential for cells to initiate duplication of their ge-netic material. Widespread and locus-specific perturbations of these transcripts correlate with anomalous replication patterns and a notable loss of the H2A.Z replicative marker at the origin site. Collectively, we present a previously undescribed RNA-mediated mechanism that is associ-ated with the generation of active replication origins in eukaryotic cells. Our findings delineate a strategy to modulate the origins of replication in human cells at a local and global level, with potentially broad biomedical implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3495588b0764b95573d71095d9ad11861909150" target='_blank'>
              RNAs anchoring replication complex control initiation and firing of DNA replication
              </a>
            </td>
          <td>
            A. D. Ruscio, Simone Ummarino, L. Poluben, Alexander Ebralidze, Ida Autiero, Yanzhou Zhang, Theodore Paniza, Madhura Deshpande, Lucrezia Rinaldi, Jonathan Lee, Mahmoud Bassal, B. Trinh, Steven Balk, R. Flaumenhaft, Jeannine Gerhardt, Sergei Mirkin, Daniel Tenen
          </td>
          <td>2025-01-28</td>
          <td>Research Square</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5889adf5def7b284cf9f7d22530eab68401de07f" target='_blank'>
              PlexinD1 is a driver and a therapeutic target in advanced prostate cancer
              </a>
            </td>
          <td>
            Jing Wei, Jing Wang, Wen Guan, Jingjing Li, Tianjie Pu, E. Corey, Tzu-Ping Lin, Allen C Gao, Boyang Jason Wu
          </td>
          <td>2025-01-02</td>
          <td>EMBO Molecular Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Highly recurrent somatic mutations in the gene encoding the core splicing factor SF3B1 are drivers of multiple cancer types. SF3B1 is a scaffold protein that orchestrates multivalent protein-protein interactions within the spliceosome that are essential for recognizing the branchsite (BS) and selecting the 3' splice site during the earliest stage of pre-mRNA splicing. In this review, we first describe the molecular mechanism by which multiple oncogenic SF3B1 mutations disrupt splicing. This involves perturbation of an early spliceosomal trimeric protein complex necessary for accurate BS recognition in a subset of introns, which leads to activation of upstream branchpoints and selection of cryptic 3' splice sites. We next discuss how specific transcripts affected by aberrant splicing in SF3B1-mutant cells contribute to the initiation and progression of cancer. Finally, we highlight the prognostic value and disease phenotypes of different cancer-associated SF3B1 mutations, which is critical for developing new targeted therapeutics against SF3B1-mutant cancers still lacking in the clinic.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba3aeca8883795216b447437aa1c8b2cba3a9442" target='_blank'>
              SF3B1: From core splicing factor to oncogenic driver.
              </a>
            </td>
          <td>
            Pedro Bak-Gordon, James L. Manley
          </td>
          <td>2025-01-07</td>
          <td>RNA</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Tumor suppressor gene TP53 is ubiquitously mutated in ovarian cancer precursor lesions that undergo pervasive accumulation of DNA damage. Prior literature provides evidence that the expression of mutant p53 protein in epithelial cells is associated with increased survival in response to DNA-damaging treatments. Hence, identification and understanding of the mechanisms that, in response to DNA damage accumulation, support survival of ovarian cancer precursor cells carrying p53 mutations might provide important information about the evolution of the disease. Here we used a combination of OC precursor cell models, biochemistry, microscopy, and flow cytometry to provide evidence that the taurine transporter, the SLC6A6 molecule, contributes to cell protection from DNA-damaging (cisplatin) treatment. We found that expression of mutant p53R175H in OC precursor cells, the fallopian tube non-ciliated epithelial (FNE) cells, induced resistance to the DNA-damaging agent cisplatin. Most importantly, shRNA-mediated targeting of SLC6A6 transcript re-sensitized FNE cells expressing mutant p53R175H to cisplatin treatment. Our studies are consistent with the model that the loss of SLC6A6 alters mechanisms involved in the regulation of cell survival in response to DNA damage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/258ec54986c78fd95aedc9a7b2b989006fafe5e2" target='_blank'>
              Mutant p53R175H-Associated Protection of Ovarian Cancer Precursor Cells from Cisplatin Requires Expression of SLC6A6 Taurine Transporter
              </a>
            </td>
          <td>
            Teagan Polotaye, Daniel Centeno, Marcin Iwanicki
          </td>
          <td>2024-08-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Despite the study of BCR::ABL1-positive and -negative myeloproliferative neoplasms (MPNs) providing seminal insights into cancer biology, tumor evolution and precision oncology over the past half century, significant challenges remain. MPNs are clonal hematopoietic stem cell-derived neoplasms with heterogenous clinical phenotypes and a clonal architecture which impacts the often-complex underlying genetics and microenvironment. The major driving molecular abnormalities have been well characterized, but debate on their role as disease-initiating molecular lesions continues. The introduction of the ABL1 tyrosine kinase inhibitors have been extremely successful in the treatment of chronic myeloid leukemia with most patients having a near-normal life expectancy. Similar success has, however, not been achieved for BCR::ABL1-negative MPNs in terms of disease course modification and most patients remain incurable. In both disease categories, genomic instability seems to increase the risk of disease progression to accelerated/blast phase, which is resistant/refractory to conventional treatment and associated with a poor prognosis. To address some of these issues, the late John Goldman and Tariq Mughal founded a scientific and clinical platform in 2006, the Post-American Society of Hematology (ASH) MPN workshop, to appraise novel cancer biology, candidate therapeutic targets, treatments and other clinical challenges and pay tribute to all the many scientists and clinicians around the world instrumental to the progress made and continuing advances being made. This paper summarizes some of the recent data discussed at the 18th edition of the workshop and includes reference to some data presented or published after the workshop, including the 26th John Goldman CML conference.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0649d6d751f0c7ea745500a8801328aa9032fd0" target='_blank'>
              Impact of Recent Translational and Therapeutic Developments on Clinical Course of BCR::ABL1-Positive and -Negative Myeloproliferative Neoplasms.
              </a>
            </td>
          <td>
            T. I. Mughal, J. Mascarenhas, R. Rampal, P. Bose, Thomas Lion, Helen Ajufo, Abdulraheem M. Yacoub, S. Meshinchi, L. Masarova, Ruben Mesa, C. Jamieson, T. Barbui, G. Saglio, R. V. van Etten
          </td>
          <td>2025-01-01</td>
          <td>Hematological oncology</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4925c81af84f202c2cbaa094b077120c44d57f50" target='_blank'>
              Long non-coding RNAs (lncRNAs) in cancer development: new insight from STAT3 signaling pathway to immune evasion
              </a>
            </td>
          <td>
            Lie Ma, XuQing Liu, R. Roopashree, Syeda Wajida Kazmi, Saade Abdalkareem Jasim, K. Phaninder Vinay, Ata Fateh, Fang Yang, Mansour Rajabivahid, Mahmoud Dehghani-Ghorbi, Reza Akhavan
          </td>
          <td>2025-02-11</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31a4f63acf43608df31e660727bdf65ebfa6ed35" target='_blank'>
              Concurrent RB1 and P53 pathway disruption predisposes to the development of a primitive neuronal component in high-grade gliomas depending on MYC-driven EBF3 transcription.
              </a>
            </td>
          <td>
            Francesca Pagani, Francesca Orzan, Sara Lago, Francesca De Bacco, Marta Prelli, Manuela Cominelli, Elena Somenza, Magdalena Gryzik, Piera Balzarini, Davide Ceresa, Daniela Marubbi, Claudio Isella, Giovanni Crisafulli, Maura Poli, Paolo Malatesta, Rossella Galli, Roberto Ronca, Alessio Zippo, Carla Boccaccio, P. L. Poliani
          </td>
          <td>2025-01-16</td>
          <td>Acta neuropathologica</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="This study delves into the intriguing world of extrachromosomal DNA (ecDNA) in breast cancer, uncovering its pivotal role in cancer's aggressiveness and genetic variability. ecDNA, a form of circular DNA found outside chromosomes, is known to play a significant role in cancer progression by increasing oncogene expression. Focusing on two contrasting cell lines, MDA-MB-231 (triple-negative) and MCF-7 (Luminal-A), we utilized advanced microscopy and fluorescence techniques to detect and characterize ecDNA. Our findings reveal a stark difference: MDA-MB-231 cells, known for their high metastatic potential, exhibit a striking abundance of ecDNA, manifested as double minutes and single form with intense fluorescence signals. In contrast, the less aggressive MCF-7 cells harbor significantly fewer ecDNA. This disparity highlights the potential of ecDNA as a key player in cancer progression and a promising target for novel therapies. This research sheds light on the unseen genetic forces driving breast cancer and opens the door to new strategies in cancer treatment. Further research is necessary to understand the mechanisms of ecDNA formation and its role in different breast cancer subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fb924ca3420181045f4a1ed603d86cd06599e30" target='_blank'>
              Extrachromosomal DNA in Breast Cancer Cell Lines: Detection and Characterization.
              </a>
            </td>
          <td>
            Shadira Anindieta Irdianto, Fadhillah Fadhillah, Retno Lestari, Fadilah Fadilah, A. Bowolaksono, A. Dwiranti
          </td>
          <td>2024-12-29</td>
          <td>Microscopy research and technique</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="The human fungal pathobiont Candida albicans displays extensive genomic plasticity, including large-scale chromosomal changes such as aneuploidy. Chromosome trisomy appears frequently in natural and laboratory strains of C. albicans. Trisomy of specific chromosomes has been linked to large phenotypic effects, such as increased murine gut colonization by strains trisomic for chromosome 7 (Chr7). However, studies of whole-chromosome aneuploidy are generally limited to the SC5314 genome reference strain, making it unclear whether the imparted phenotypes are conserved across C. albicans genetic backgrounds. Here, we report the presence of a Chr7 trisomy in the “commensal-like” oral candidiasis strain, 529L, and dissect the contribution of Chr7 trisomy to colonization and virulence in 529L and SC5314. These experiments show that strain background and homolog identity (i.e., AAB vs ABB) interact with Chr7 trisomy to alter commensal and virulence phenotypes in multiple host niches. In vitro filamentation was the only phenotype altered by Chr7 trisomy in similar ways across the two strain backgrounds. Oral colonization of mice was increased by the presence of a Chr7 trisomy in 529L but not SC5314; conversely, virulence during systemic infection was reduced by Chr7 trisomy in SC5314 but not 529L. Strikingly, the AAB Chr7 trisomy in the SC5314 background rendered this strain avirulent in murine systemic infection. Increased dosage of NRG1 failed to reproduce most of the Chr7 trisomy phenotypes. Our results demonstrate that aneuploidy interacts with background genetic variation to produce complex phenotypic patterns that deviate from our current understanding in the genome reference strain. AUTHOR SUMMARY Candida albicans is a clinically important fungus of humans that is also part of the microbiota that typically colonizes our bodies. Changes in the number of copies of chromosome 7 (Chr7) in the genome of the reference strain of C. albicans enhances host colonization. We identified a third copy of Chr7 in a strain classified as a good colonizer of humans that doesn’t cause disease, mimicking the increased colonization of the reference strain. Here, we demonstrate that the number of copies of Chr7 alters phenotypes differently between these strains, and three copies of Chr7 does not universally produce a strain that is only a better commensal strain. Additionally, which copy of Chr7 in this diploid species exists as the third copy influences commensal and pathogen phenotypes. Therefore, trisomic changes of the same chromosome in different C. albicans strains have substantially different effects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2332e9447e18d3247be54ad40b31742ed672d8fa" target='_blank'>
              Strain background interacts with chromosome 7 aneuploidy to determine commensal and virulence phenotypes in Candida albicans
              </a>
            </td>
          <td>
            Abhishek Mishra, N. Solis, Siobhan M Dietz, Audra L Crouch, S. Filler, Matthew Z. Anderson
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="Tyrosine kinase inhibitors (TKIs) have transformed the treatment of epidermal growth factor receptor (EGFR)-mutant non-small cell lung cancer. However, treatment resistance remains a major challenge in clinical practice. The tumor microenvironment (TME) is a complex system composed of tumor cells, immune and non-immune cells, and non-cellular components. Evidence indicates that dynamic changes in TME during TKI treatment are associated with the development of resistance. Research has focused on identifying how each component of the TME interacts with tumors and TKIs to understand therapeutic targets that could address TKI resistance. In this review, we describe how TME components, such as immune cells, fibroblasts, blood vessels, immune checkpoint proteins, and cytokines, interact with EGFR-mutant tumors and how they can promote resistance to TKIs. Furthermore, we discuss potential strategies targeting TME as a novel therapeutic approach.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26f6f1b01ac68ddc2dc3220a920e6b272e7041d2" target='_blank'>
              Targeting the Tumor Microenvironment in EGFR-Mutant Lung Cancer: Opportunities and Challenges
              </a>
            </td>
          <td>
            Jeong Uk Lim, Junyang Jung, Yeon Wook Kim, Chi Young Kim, Sang Hoon Lee, Dong Won Park, Sue In Choi, W. Ji, C. Yeo, Seung Hyeun Lee
          </td>
          <td>2025-02-14</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Background/Objectives: GFI1-36N represents a single-nucleotide polymorphism (SNP) of the zinc finger protein Growth Factor Independence 1 (GFI1), in which the amino acid serine (S) is replaced by asparagine (N). The presence of the GFI1-36N gene variant is associated with a reduced DNA repair capacity favoring myeloid leukemogenesis and leads to an inferior prognosis of acute myeloid leukemia (AML) patients. However, the underlying reasons for the reduced DNA repair capacity in GFI1-36N leukemic cells are largely unknown. Since we have demonstrated that GFI1 plays an active role in metabolism, in this study, we investigated whether increased levels of reactive oxygen species (ROS) could contribute to the accumulation of genetic damage in GFI1-36N leukemic cells. Methods: We pursued this question in a murine model of human AML by knocking in human GFI1-36S or GFI1-36N variant constructs into the murine Gfi1 gene locus and retrovirally expressing MLL-AF9 to induce AML. Results: Following the isolation of leukemic bone marrow cells, we were able to show that the GFI1-36N SNP in our model is associated with enhanced oxidative phosphorylation (OXPHOS), increased ROS levels, and results in elevated γ-H2AX levels as a marker of DNA double-strand breaks (DSBs). The use of free radical scavengers such as N-acetylcysteine (NAC) and α-tocopherol (αT) reduced ROS-induced DNA damage, particularly in GFI1-36N leukemic cells. Conclusions: We demonstrated that the GFI1-36N variant is associated with extensive metabolic changes that contribute to the accumulation of genetic damage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae3d1609ac86603afd42ec35dafbbc01b4f80c88" target='_blank'>
              Germline Single-Nucleotide Polymorphism GFI1-36N Causes Alterations in Mitochondrial Metabolism and Leads to Increased ROS-Mediated DNA Damage in a Murine Model of Human Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Jan Vorwerk, Longlong Liu, Theresa Helene Stadler, D. Frank, H. Ahmed, P. Patnana, Maxim Kebenko, Eva Dazert, Bertram Opalka, N. von Bubnoff, Cyrus Khandanpour
          </td>
          <td>2025-01-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>34</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [4, 14],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>