<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-04-21 09:27:55 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="DHFR gene amplification is commonly present in methotrexate (MTX)-resistant colon cancer cells and acute lymphoblastic leukemia. In this study, we proposed an integrative framework to characterize the amplified region by using a combination of single-molecule real-time sequencing, next-generation optical mapping, and chromosome conformation capture (Hi-C). We identified an amplification unit spanning 11 genes, from the DHFR gene to the ATP6AP1L gene position, with high adjusted interaction frequencies on chromosome 5 (~2.2 Mbp) and a twenty-fold tandemly amplified region, and novel inversions at the start and end positions of the amplified region as well as frameshift insertions in most of the MSH and MLH genes were detected. These mutations might stimulate chromosomal breakage and cause the dysregulation of mismatch repair. Characterizing the tandem gene-amplified unit may be critical for identifying the mechanisms that trigger genomic rearrangements. These findings may provide new insight into the mechanisms underlying the amplification process and the evolution of drug resistance. Sequencing a large region of DNA containing many surplus copies of genes linked to drug resistance in colon cancer cells may illuminate how these genomic rearrangements arise. Such regions of gene amplification are highly repetitive, making them impossible to sequence using ordinary methods, and little is known about how they are generated. Using advanced methods, Jeong-Sun Seo at Seoul National University Bundang Hospital in South Korea and co-workers sequenced a region of gene amplification in colon cancer cells. The amplified region was approximately 20 times the length of that in healthy cells and contained many copies of an eleven-gene segment, including a gene implicated in drug resistance. The region also contained mutations in chromosomal repair genes which would disrupt repair pathways. These results illuminate the genetic changes that lead to gene amplification and drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental & Molecular Medicine, Experimental and Molecular Medicine</td>
          <td>5</td>
          <td>49</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum.
                </a>
              </td>
          <td>
            Gustavo S França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>30</td>
          <td>42</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Identifying genetic dependencies in human colon cancer could help identify effective treatment strategies. Genome-wide CRISPR-Cas9 dropout screens have the potential to reveal genetic dependencies, some of which could be exploited as therapeutic targets using existing drugs. In this study, we comprehensively characterized genetic dependencies present in a colon cancer organoid avatar, and validated tumor-specific selectivity of select pharmacologic agents. We conducted a genome-wide CRISPR dropout screen to elucidate the genetic dependencies that interacted with select driver somatic mutations. We found distinct genetic dependencies that interacted with WNT, MAPK, PI3K, TP53, and mismatch repair pathways and validated targets that could be exploited as treatments for this specific subtype of colon cancer. These findings demonstrate the utility of functional genomic screening in the context of personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6f08c27268b9a856764ce30d936babdccabacfc" target='_blank'>
              Comprehensive genomic dependency landscape of a human colon cancer organoid
              </a>
            </td>
          <td>
            Sanam Khalili, Atefeh Mohseninia, Changlong Liu, Carolyn E. Banister, Paige Heine, M. Khazan, Sidney E. Morrison, P. Gokare, Glenn S Cowley, Barbara A. Weir, David Pocalyko, Kurtis E. Bachman, Phillip J. Buckhaults
          </td>
          <td>2025-03-14</td>
          <td>Communications Biology</td>
          <td>1</td>
          <td>14</td>
        </tr>

        <tr id="This article reviews the impact of single-cell sequencing (SCS) on cancer biology research. SCS has revolutionized our understanding of cancer and tumor heterogeneity, clonal evolution, and the complex interplay between cancer cells and tumor microenvironment. SCS provides high-resolution profiling of individual cells in genomic, transcriptomic, and epigenomic landscapes, facilitating the detection of rare mutations, the characterization of cellular diversity, and the integration of molecular data with phenotypic traits. The integration of SCS with multi-omics has provided a multidimensional view of cellular states and regulatory mechanisms in cancer, uncovering novel regulatory mechanisms and therapeutic targets. Advances in computational tools, artificial intelligence (AI), and machine learning have been crucial in interpreting the vast amounts of data generated, leading to the identification of new biomarkers and the development of predictive models for patient stratification. Furthermore, there have been emerging technologies such as spatial transcriptomics and in situ sequencing, which promise to further enhance our understanding of tumor microenvironment organization and cellular interactions. As SCS and its related technologies continue to advance, they are expected to drive significant advances in personalized cancer diagnostics, prognosis, and therapy, ultimately improving patient outcomes in the era of precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b38b69885e5e53fa9260297634ff329303ec057" target='_blank'>
              Single-Cell Sequencing: Genomic and Transcriptomic Approaches in Cancer Cell Biology
              </a>
            </td>
          <td>
            Ana Ortega-Batista, Yanelys Jaén-Alvarado, Dilan Moreno-Labrador, Natasha Gómez, Gabriela García, Erika N. Guerrero
          </td>
          <td>2025-02-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b47a232bd7c224e77a90b7f42bb716c42af55b0" target='_blank'>
              Molecular mechanisms altering cell identity in cancer.
              </a>
            </td>
          <td>
            A. Zippo, Sven Beyes
          </td>
          <td>2025-02-26</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a4cbce0b7fb1d47370357ef7f4f5d368e5b4543" target='_blank'>
              Genetic Profiling of Acute and Chronic Leukemia via Next-Generation Sequencing: Current Insights and Future Perspectives
              </a>
            </td>
          <td>
            Laras Pratiwi, Fawzia Hanum Mashudi, Mukti Citra Ningtyas, H. Sutanto, P. Z. Romadhon
          </td>
          <td>2025-03-28</td>
          <td>Hematology Reports</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68496fd9e9a9f934f03f0dd67116b1c071c33e67" target='_blank'>
              Diploid genome assembly of human fibroblast cell lines enables clone specific variant calling, improved read mapping and accurate phasing
              </a>
            </td>
          <td>
            T. R. Ranallo-Benavidez, Yue Hao, Emilia Volpe, Maryam Jehangir, Noelle Fukushima, Ogechukwu Mbegbu, Rebecca Reiman, Jessica Molnar, Danyael Murphy, Dorothy Marie Paredes, Shukmei Wong, Kara Karaniuk, Stephanie Buchholtz, Jonathan Keats, M. Machiela, M. Kolmogorov, B. Paten, Simona Giunta, Floris P Barthel
          </td>
          <td>2025-04-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="Familial adenomatous polyposis (FAP) is caused by pathogenic germline variants in the tumor suppressor gene APC. Confirmation of diagnosis was not achieved by cancer gene panel and exome sequencing or custom array-CGH in a family with suspected FAP across five generations. Long-read genome sequencing (PacBio), short-read genome sequencing (Illumina), short-read RNA sequencing, and further validations were performed in different tissues of multiple family members. Long-read genome sequencing resolved a 6 kb full-length intronic insertion of a heterozygous LINE-1 element between exons 7 and 8 of APC that could be detected but not fully resolved by short-read genome sequencing. Targeted RNA analysis revealed aberrant splicing resulting in the formation of a pseudo-exon with a premature stop codon. The variant segregated with the phenotype in several family members allowing its evaluation as likely pathogenic. This study supports the utility of long-read DNA sequencing and complementary RNA approaches to tackle unsolved cases of hereditary disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b2dfb493e47fa2c687a6c94a41bc29721dde1bb" target='_blank'>
              Long-read genome and RNA sequencing resolve a pathogenic intronic germline LINE-1 insertion in APC
              </a>
            </td>
          <td>
            Alexandra A Baumann, Lisanne I Knol, Marie Arlt, Tim Hutschenreiter, Anja Richter, Thomas J Widmann, Marcus Franke, K. Hackmann, S. Winkler, D. Richter, Isabel Spier, Stefan Aretz, Daniela Aust, J. Porrmann, D. William, E. Schröck, H. Glimm, A. Jahn
          </td>
          <td>2025-04-04</td>
          <td>NPJ Genomic Medicine</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fbd7856e741845eb17452c67c39e622695a90ba" target='_blank'>
              Advances in DNA replication research for precision medicine
              </a>
            </td>
          <td>
            Md. Mosharraf Hossen
          </td>
          <td>2025-03-08</td>
          <td>Journal of Bioscience and Environment Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Traditionally, the evolutionary perspective of cancer has been understood as gradual alterations in passenger/driver genes that lead to branching phylogeny. However, in cases of prostate adenocarcinoma and kidney renal cell carcinoma, macroevolutionary landmarks like chromoplexy and chromothripsis are frequently observed. Unfortunately, short-read sequencing techniques often miss these significant macroevolutionary changes, which involve multiple translocations and deletions at the chromosomal level. To resolve such genomic dark matters, we provided high-fidelity long-read sequencing data (78–92 Gb of ~Q30 reads) of six genitourinary tumour cell lines (one benign kidney tumour and two kidney and three prostate cancers). Based on these data, we obtained 12 high-quality, partially phased genome assemblies (Contig N50 1.85–29.01 Mb; longest contig 2.02–171.62 Mb), graph-based pan-genome variant sets (11.57 M variants including 60 K structural variants), and 5-methylcytosine sites (14.68%–27.05% of the CpG sites). We also identified several severe chromosome aberration events, which would result from chromosome break and fusion events. Our cancer genome assemblies will provide unprecedented resolution to understand cancer genome instability and chromosomal aberration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39aabbe27003a6111c9e819cce1949d58b416548" target='_blank'>
              Genome assembly resources of genitourinary cancers for chromosomal aberration at the single nucleotide level
              </a>
            </td>
          <td>
            Hyunho Han, H. Lee, Min Gyu Kim, Yoo Sub Shin, Jin Soo Chung, Jun Kim
          </td>
          <td>2025-04-01</td>
          <td>Scientific Data</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Introduction A hallmark of small cell lung cancer (SCLC) is its recalcitrance to therapy. While most SCLCs respond to frontline therapy, resistance inevitably develops. Identifying phenotypes potentiating chemoresistance and immune evasion is a crucial unmet need. Previous reports have linked upregulation of the DNA damage response (DDR) machinery to chemoresistance and immune evasion across cancers. However, it is unknown if SCLCs exhibit distinct DDR phenotypes. Methods To study SCLC DDR phenotypes, we developed a new DDR gene analysis method and applied it to SCLC clinical samples, in vitro, and in vivo model systems. We then investigated how DDR regulation is associated with SCLC biology, chemotherapy response, and tumor evolution following therapy. Results Using multi-omic profiling, we demonstrate that SCLC tumors cluster into three DDR phenotypes with unique molecular features. Hallmarks of these DDR clusters include differential expression of DNA repair genes, increased replication stress, and heightened G2/M cell cycle arrest. SCLCs with elevated DDR phenotypes exhibit increased neuroendocrine features and decreased “inflamed” biomarkers, both within and across SCLC subtypes. Clinical analyses demonstrated treatment naive DDR status was associated with different responses to frontline chemotherapy. Using longitudinal liquid biopsies, we found that DDR Intermediate and High tumors exhibited subtype switching and coincident emergence of heterogenous phenotypes following frontline treatment. Conclusions We establish that SCLC can be classified into one of three distinct, clinically relevant DDR clusters. Our data demonstrates that DDR status plays a key role in shaping SCLC phenotypes and may be associated with different chemotherapy responses and patterns of tumor evolution. Future work targeting DDR specific phenotypes will be instrumental in improving patient outcomes. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02291-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/671674f412412e8378b84a598f7d2a87a83c70ce" target='_blank'>
              DNA damage response signatures are associated with frontline chemotherapy response and routes of tumor evolution in extensive stage small cell lung cancer
              </a>
            </td>
          <td>
            Benjamin B. Morris, Simon Heeke, Yuanxin Xi, L. Diao, Qi Wang, Pedro Rocha, Edurne Arriola, Myung Chang Lee, Darren R. Tyson, K. Concannon, K. Ramkumar, C. Stewart, R. Cardnell, Runsheng Wang, Vito Quaranta, Jing Wang, J. Heymach, B. Nabet, D. Shames, C. Gay, L. Byers
          </td>
          <td>2025-03-20</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>107</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/158e0cac5c37d59febfcaeb1492ca3834b253583" target='_blank'>
              Expanding single-cell toolbox with a cost-effective full-length total RNA droplet-based sequencing technology
              </a>
            </td>
          <td>
            Xue Sun, Shir-Liya Dadon, Dena Ennis, Wenpeng Fan, Muhammad Awawdy, Eli Reuveni, A. Alajem, Oren Ram
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Cancer cells exhibit high heterogeneity and lineage plasticity, complicating studies of tumorigenesis and development of therapies. Recently, preneoplastic cells, although histologically normal, have been shown to possess high plasticity and early genetic alterations, yet their origins and lineage trajectories remain unclear. Herein, we introduce a lineage-tracing tool integrating genetic barcoding with single-cell RNA sequencing to map preneoplastic esophageal cell lineages. We identified preneoplastic precursor cells (PNPCs) as a distinct progenitor-like population with unique transcriptional profiles and high plasticity, contributing to proliferative and basal cell populations. To enhance lineage mapping, we developed the eXamined Ridge (XR) score, accurately identifying high-plasticity cells. Nfib and Qk emerged as conserved PNPC markers, peaking in early preneoplasia and declining after malignant transformation. These findings reveal PNPCs as key players in early tumorigenesis and highlight their potential as biomarkers for early cancer detection and therapeutic intervention, offering new strategies for preventing esophageal cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37dcd14a4aabf910a9a90379f13bfe9df5a44b29" target='_blank'>
              Deciphering Precursor Cell Dynamics in Esophageal Preneoplasia via Genetic Barcoding and Single-Cell Transcriptomics
              </a>
            </td>
          <td>
            Jinho Jang, Kyung-Pil Ko, Jie Zhang, Sohee Jun, Jae-Ll Park
          </td>
          <td>2025-02-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Metastasis is the leading cause of cancer deaths; nonetheless, how tumor cells adapt to vastly different organ contexts is largely unknown. To investigate this question, we generated a transcriptomic atlas of primary tumor and diverse metastatic samples from a patient with pancreatic ductal adenocarcinoma who underwent rapid autopsy. Unsupervised archetype analysis identified both shared and site-specific gene programs, including lipid metabolism and gastrointestinal programs prevalent in peritoneum and stomach wall lesions, respectively. We developed a probabilistic approach for inferring clonal phylogeny from single-cell and matched whole-exome data. Distantly related genetic clones in the peritoneum express the lipid metabolism program, likely due to signaling by the adipocyte-rich peritoneum environment, and cells in most clones express multiple programs, suggesting that transcriptomic plasticity is a prevalent feature of metastatic cells. These deeply annotated analyses using a patient-centric platform provide a model for investigating metastatic mechanisms and plasticity in advanced cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57b77f53eedac93ceda8af4b898f6071abded0e4" target='_blank'>
              Transcriptomic plasticity is a hallmark of metastatic pancreatic cancer
              </a>
            </td>
          <td>
            Alejandro Jiménez-Sánchez, S. Persad, Akimasa Hayashi, Shigeaki Umeda, Roshan Sharma, Yubin Xie, A. Mehta, Wungki Park, I. Masilionis, T. Chu, Feiyang Zhu, Jungeui Hong, R. Chaligné, Eileen M. O’Reilly, L. Mazutis, T. Nawy, I. Pe’er, Christine A. Iacobuzio-Donahue, D. Pe’er
          </td>
          <td>2025-03-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db41c8c1cf9654af3ad975bf6a786aaf0b60bc69" target='_blank'>
              Somatic repetitive element insertions Define New Biomarkers in Pan-cancer genome
              </a>
            </td>
          <td>
            Lin Xia, Kailing Tu, Huan Wang, Xuyan Liu, Yahui Liu, Qilin Zhang, Ziting Feng, Chunyan Yu, Maolin Li, Yan Huang, Jing Wang, Ranlei Wei, Tianfu Zeng, Xuenan Pi, Chen Ling, Qianwen Zhang, Yixin Qiao, Shengzhong Hou, Zhuoyuan Zhang, Ailin Wei, Haotian Liao, Wenhao Guo, Jifeng Liu, Zhong Wu, Zhoufeng Wang, Ang Li, Weimin Li, Dan Xie
          </td>
          <td>2025-03-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background and Objective: The discovery of novel molecular biomarkers via next-generation sequencing technologies has revolutionized how glioblastomas (GBMs) are classified nowadays. This has resulted in more precise diagnostic, prognostic, and therapeutic approaches to address this malignancy. The present work examines the applications of single-cell RNA sequencing (scRNA-seq) in GBM, focusing on its potential to address tumor complexity and therapeutic resistance and improve patient outcomes. Methods: A scoping review of original studies published between 2009 and 2024 was conducted using the PUBMED and EMBASE databases. Studies in English or Spanish related to single-cell analysis and GBM were included. Key Findings: The database search yielded 453 publications. Themes related to scRNA-seq applied for the diagnosis, prognosis, treatment, and understanding of the cancer biology of GBM were used as criteria for article selection. Of the 24 studies that were included in the review, 11 focused on the tumor microenvironment and cell subpopulations in GBM samples, 5 investigated the use of sequencing to elucidate the GBM cancer biology, 3 examined disease prognosis using sequencing models, 3 applied translational research through scRNA-seq, and 2 addressed treatment-related problems in GBM elucidated by scRNA-seq. Conclusions: This scoping review explored the various clinical applications of scRNA-seq technologies in approaching GBM. The findings highlight the utility of this technology in unraveling the complex cellular and immune landscapes of GBM, paving the way for improved diagnosis and personalized treatments. This cutting-edge approach might strengthen treatment strategies against tumor progression and recurrence, setting the stage for multi-targeted interventions that could significantly improve outcomes for patients with aggressive, treatment-resistant GBMs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50f255f1e2df6321b7a03cd03e70b813bcc99665" target='_blank'>
              Current Applications of Single-Cell RNA Sequencing in Glioblastoma: A Scoping Review
              </a>
            </td>
          <td>
            Edgar G. Ordóñez-Rubiano, Nicolás Rincón-Arias, William J. Shelton, A. F. Salazar, María Alejandra Sierra, Raphael Bertani, D. Gómez-Amarillo, Fernando Hakim, Matías Baldoncini, César Payán-Gómez, A. Cómbita, Sandra C. Ordonez-Rubiano, Rafael Parra-Medina
          </td>
          <td>2025-03-01</td>
          <td>Brain Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1e688482cb691231e9728515c865331942ee770" target='_blank'>
              Integrating gene expression, mutation and copy number data to identify driver genes of recurrent chromosome-arm losses
              </a>
            </td>
          <td>
            Ron Saad, Ron Shamir, Uri Ben-David
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Glioblastoma, one of the most aggressive and heterogeneous malignant tumors, presents significant challenges for clinical management due to its cellular and metabolic complexity. This review integrates recent advancements in single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics to elucidate glioblastoma’s cellular heterogeneity and metabolic reprogramming. Diverse cellular subpopulations, including malignant proliferative cells, stem-like cells, mesenchymal-like cells, and immune-related cells, contribute to tumor progression, treatment resistance, and microenvironmental interactions. Spatial transcriptomics has further revealed distinct spatial distributions of these subpopulations, highlighting differences in metabolic activities between the tumor core and periphery. Key metabolic adaptations, such as enhanced glycolysis, fatty acid oxidation, and glutamine metabolism, play critical roles in supporting tumor growth, immune evasion, and therapeutic resistance. Targeting these metabolic pathways, especially in combination with immunotherapy, represents a promising avenue for glioblastoma treatment. This review emphasizes the importance of integrating single-cell and spatial multi-omics technologies to decode glioblastoma’s metabolic landscape and explore novel therapeutic strategies. By addressing current challenges, such as metabolic redundancy and spatiotemporal dynamics, this work provides insights into advancing precision medicine for glioblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea52e3e92b1a5c4c3caddeca60f128feaaaf952b" target='_blank'>
              Single-cell and spatial transcriptomic insights into glioma cellular heterogeneity and metabolic adaptations
              </a>
            </td>
          <td>
            Yixin Fu, Yong Yi, Yongxiang Shao, Jingcheng Jiang, Qingshan Deng
          </td>
          <td>2025-04-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/996e0da402cc341f0ff28812c9362a9660b3a867" target='_blank'>
              Abstract A026: Deciphering natural and synthetic genetic programs governing human T cell function with combinatorial CRISPR activation and interference screens
              </a>
            </td>
          <td>
            Zachary Steinhart, Jeffrey Perera, Dmytro Dorovskyi, Luke Workley, Carinna Tran, Rosmely Hernandez, Zhongmei Li, Max Foisey, Daniel Goodman, Kole T. Roybal, B. Shy, Alexander Marson
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37c650c9b7245da58253a96046c65fd7d628305a" target='_blank'>
              Understanding DNA and PARP in Cancer: Tackling Inhibitor Resistance
              </a>
            </td>
          <td>
            Hardha Balachandran, Gowramma Byran, V. Karri, Senthil Kumar Murugesan, K. Rajagopal
          </td>
          <td>2025-03-13</td>
          <td>Drug Metabolism and Bioanalysis Letters</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6d647ad1e76680b2e0c017bae0688ab2ee0a334" target='_blank'>
              Signaling cascades are targets for breast cancer therapy in the light of genome – wide sequencing data
              </a>
            </td>
          <td>
            L. F. Gulyaeva, M. Filipenko, N. E. Kushlinskii
          </td>
          <td>2025-04-13</td>
          <td>Advances in Molecular Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Although two-thirds of cancers arise from loss-of-function mutations in tumor suppressor genes, there are few approved targeted therapies linked to these alterations. Synthetic lethality offers a promising strategy to treat such cancers by targeting vulnerabilities unique to cancer cells with these mutations. To identify clinically relevant synthetic lethal interactions, we analyzed genome-wide CRISPR/Cas9 knock-out (KO) viability screens from the Cancer Dependency Map and evaluated their clinical relevance in patient tumors through mutual exclusivity, a pattern indicative of synthetic lethality. Indeed, we found significant enrichment of mutual exclusivity for interactions involving cancer driver genes compared to non-driver mutations. To identify therapeutic opportunities, we integrated drug sensitivity data to identify inhibitors that mimic the effects of CRISPR-mediated KO. This approach revealed potential drug repurposing opportunities, including BRD2 inhibitors for bladder cancers with ARID1A mutations and SIN3A-mutated cell lines showing sensitivity to nicotinamide phosphoribosyltransferase (NAMPT) inhibitors. However, we discovered that pharmacological inhibitors often fail to phenocopy KO of matched drug targets, with only a small fraction of drugs inducing similar effects. This discrepancy reveals fundamental differences between pharmacological and genetic perturbations, emphasizing the need for approaches that directly assess the interplay of loss-of-function mutations and drug activity in cancer models. Author Summary Synthetic lethality is an emerging approach for targeting a biological dependency in cancer cells that does not harm normal cells. This strategy is particularly valuable for targeting loss-of-function mutations in tumor suppressor genes, which are more challenging to directly target. In an effort to accelerate treatments for cancer patients, we aimed to map out these dependencies and overlap them with responses to available drugs. We discovered different outcomes when a protein is targeted by a drug versus when that same target is disrupted genetically. Thus, if a drug is to be effectively repurposed as synthetic lethal agent, feasibility studies must capture drug biology, ideally by test the drug empirically in relevant cancer models. A second notable discovery is that in vitro synthetic lethal interactions involving cancer driver genes are significantly more likely to exhibit consistent patterns, such as mutual exclusivity in human tumor samples. This is important since selection of relevant cell lines is often critical in drug development to maximize potential for translation to clinical responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4c1469ac01d0657e51e2a370ba73f2735664c7f" target='_blank'>
              Challenges and opportunities for drug repurposing in cancers based on synthetic lethality induced by tumor suppressor gene mutations
              </a>
            </td>
          <td>
            Michael Vermeulen, Andrew W. Craig, Tomas Babak
          </td>
          <td>2025-03-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer stem cells (CSCs), are a critical subpopulation within tumours, and are defined by their capacity for self-renewal, differentiation, and tumour initiation. These unique traits contribute to tumour progression, metastasis, and resistance to conventional treatments like chemotherapy and radiotherapy, often resulting in cancer recurrence and poor patient outcomes. As such, CSCs have become focal points in developing advanced cancer therapies. This review highlights progress in CSC-targeted treatments, including chimeric antigen receptor T-cell (CAR-T) therapy, immunotherapy, molecular targeting, and nanoparticle-based drug delivery systems. Plant-derived compounds and gene-editing technologies, such as clustered regularly interspaced short palindromic repeats (CRISPR), are explored for their potential to enhance precision and minimize side effects. Metabolic pathways integral to CSC survival, such as mitochondrial dynamics, mitophagy (regulated by dynamin-related protein 1 [DRP1] and the PINK1/Parkin pathway), one-carbon metabolism, amino acid metabolism (involving enzymes like glutaminase (GLS) and glutamate dehydrogenase (GDH]), lipid metabolism, and hypoxia-induced metabolic reprogramming mediated by hypoxia-inducible factors (HIF-1α and HIF-2α), are examined as therapeutic targets. The adaptability of CSCs through autophagy, metabolic flexibility, and epigenetic regulation by metabolites like α-ketoglutarate, succinate, and fumarate is discussed. Additionally, extracellular vesicles and nicotinamide adenine dinucleotide (NAD⁺) metabolism are identified as pivotal in redox balance, DNA repair, and epigenetic modifications. Addressing challenges such as tumour heterogeneity, immune evasion, and treatment durability requires interdisciplinary collaboration. Advancing CSC-targeted therapies is essential for overcoming drug resistance and preventing cancer relapse, paving the way for transformative cancer treatments. This review underscores the importance of leveraging innovative technologies and fostering collaboration to revolutionize cancer treatment. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdfd29a8c49b166f1e1f2b3241c057ea76b860a3" target='_blank'>
              Unveiling the future of cancer stem cell therapy: a narrative exploration of emerging innovations
              </a>
            </td>
          <td>
            Joseph Nhyira Obisi, Abike Ndidiamaka Josephine Abimbola, O. Babaleye, Peter Kwame Atidoglo, Saviour Godswealth Usin, Eudora Nwanaforo, Faith Sutu Patrick-Inezi, I. V. Fasogbon, Joseph Chimezie, Christianah Adebimpe Dare, Oluwadoyinsayemi Oluwadamilare Kuti, D. E. Uti, Humphrey Chukwudi Omeoga
          </td>
          <td>2025-03-22</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Methylation processes in different molecular contexts (DNA, RNA, and histones) are controlled by different regulatory factors and serve as critical determinants in cancer development. However, the mechanistic links between these epigenetic modifications during malignant transformation, metastasis, disease relapse, and therapeutic resistance remain incompletely understood. In this research, we investigated the transcriptional and genetic alterations of regulators associated with 3 major types of methylation modifications in clear cell renal cell carcinoma. Utilizing ChIP/MeRIP-seq and 450K methylation array data, we identified genes regulated by multiple methylation modifications and constructed a scoring model to quantify the methylation patterns for each patient. Our findings indicate that patients with a low score may be more likely to respond to immunotherapy, whereas patients with a high score may be more sensitive to targeted therapy, such as RITA, Pazopanib, Irlotinib, SU-11274, BRD-K16762525, and FCCP. In conclusion, the score model can serve as a valuable biomarker to guide clinical selection of immunotherapy and targeted drugs and help to improve personalized clear cell renal cell carcinoma treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da4691220e407b79a42a6b06dcf7ad89ec71062f" target='_blank'>
              Investigating the efficacy of immune checkpoint inhibitors in clear cell renal cell carcinoma based on methylation cross talk scoring
              </a>
            </td>
          <td>
            Qinglong Du, Qiyuan Wang, Chen Yang, Yiping Wang, Huiyang Yuan, Bing Zhang, Hong Ji, Shuai Fu, Chunlei Xue
          </td>
          <td>2025-03-14</td>
          <td>Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21b08a5b6171c80002da0a1517c54be1999c70c6" target='_blank'>
              MED12 dysregulation: insights into cancer and therapeutic resistance.
              </a>
            </td>
          <td>
            R. Bhole, Jagruti Shinkar, Sonali Labhade, Pawan N Karwa, Harshad S. Kapare
          </td>
          <td>2025-03-19</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e261b7effc9021ce936ae363e0d5194e2c3f12d" target='_blank'>
              Integrated genomic analysis of small intestinal neuroendocrine tumors provides further insights into molecular subgroups and identifies putative driver genes
              </a>
            </td>
          <td>
            S. Backman, Elham Barazeghi, Per Hellman Olov Norlén, P. Stålberg, P. Stålberg
          </td>
          <td>2025-03-12</td>
          <td>None</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Aneuploidy is one of the most common mechanisms of adaptation to environmental selection in vegetatively growing cells. However, it is still unclear what advantages aneuploidy provides over other types of genomic alterations. In this study, we aimed to determine if beneficial aneuploidy phenotypes are driven primarily by combinatorial expression changes of multiple genes using budding yeast as a model system. To determine the impact of complex aneuploidy on cellular fitness and resistance to a variety of drug treatments, we generated yeast collections with nearly every possible combination of two chromosome gains or losses. In addition, we genetically dissected aneuploid chromosomes using partial deletions to identify chromosomal regions that contribute to aneuploidy-driven drug resistance. Strong resistance phenotypes consistently came from combinations of chromosomes or chromosomal regions. We identified up to five regions on a single chromosome that contribute to resistance to a single drug treatment. Finally, we found that the strongest aneuploidy phenotypes came from synergistic effects between copy number changes of different chromosomes or chromosomal regions. These results provide insights into how combining subtle gene expression changes across the genome impacts aneuploidy phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/288386141655e3d2d8bd580753d7c57e3ca32011" target='_blank'>
              Combinatorial effects of multiple genes contribute to beneficial aneuploidy phenotypes
              </a>
            </td>
          <td>
            M. S. Koller, Claudia Himmelbauer, Sarah Fink, Madhwesh C. Ravichandran, Christopher S. Campbell
          </td>
          <td>2025-03-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Lung cancer is a major cause of cancer-related deaths globally. Targeted therapies, specifically attacking cancer cells based on genetic mutations, offer promising alternatives. ALK (anaplastic lymphoma kinase) fusions result in aberrant proteins that drive cancer growth. Drugs like crizotinib and ceritinib have shown efficacy in treating ALK-positive NSCLC. Accurate detection of ALK fusions is crucial for guiding these therapies. We conducted a retrospective analysis of a Chinese cohort of 131 ALK rearrangement-positive patients detected by DNA NGS between January 2017 and December 2021. Among those 131 ALK fusions, RNA-NGS confirmed positive transcripts in 88% of canonical ALK fusions and 75% of ALK fusions with rare partners in samples sequenced by both DNA NGS and RNA NGS. The secondary classification approach increased transcript prediction accuracy to 95.4% when combining common breakpoints and inframe fusion analysis in canonical ALK fusions. Combining rare breakpoints and inframe fusion could increase transcript prediction accuracy to 100%. For ALK fusions with rare partners, combining common breakpoints and frameshift improved transcript prediction accuracy to 100%. Additionally, combining rare breakpoints with inframe or frameshift could enhance the prediction accuracy to 100%. Combining DNA NGS and RNA NGS with a secondary classification approach significantly enhances the transcript prediction accuracy at the RNA level. This method optimizes clinical diagnostic and therapeutic strategies for ALK-positive NSCLC, highlighting the importance of advanced sequencing techniques in precision oncology. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-92590-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34804a6772ac12af08ec23d6c1744145feb9ff1c" target='_blank'>
              Combined utility of genomic breakpoints and frame is a reliable predictor of ALK transcript function
              </a>
            </td>
          <td>
            Qiong Yang, Yumin Guo, Weihui Guo, Dongdong Liu, Haitao Wang, Xufan Cai
          </td>
          <td>2025-03-11</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Non-coding RNAs (ncRNAs) play crucial roles in colorectal cancer (CRC) development and progression. Recent developments in single-cell transcriptome profiling methods have revealed surprising levels of expression variability among seemingly homogeneous cells, suggesting the existence of many more cell types than previously estimated. This review synthesizes recent advances in ncRNA research in CRC, emphasizing single-cell bioinformatics approaches for their analysis. We explore computational methods and tools used for ncRNA identification, characterization, and functional prediction in CRC, with a focus on single-cell RNA sequencing (scRNA-seq) data. The review highlights key bioinformatics strategies, including sequence-based and structure-based approaches, machine learning applications, and multi-omics data integration. We discuss how these computational techniques can be applied to analyze differential expression, perform functional enrichment, and construct regulatory networks involving ncRNAs in CRC. Additionally, we examine the role of bioinformatics in leveraging ncRNAs as diagnostic and prognostic biomarkers for CRC. We also discuss recent scRNA-seq studies revealing ncRNA heterogeneity in CRC. This review aims to provide a comprehensive overview of the current state of single-cell bioinformatics in ncRNA CRC research and outline future directions in this rapidly evolving field, emphasizing the integration of computational approaches with experimental validation to advance our understanding of ncRNA biology in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df0d488fffc7d97d7489f557a3f2dfce15740d80" target='_blank'>
              Single-Cell Transcriptomic Approaches for Decoding Non-Coding RNA Mechanisms in Colorectal Cancer
              </a>
            </td>
          <td>
            M. Gondal, H. Farooqi
          </td>
          <td>2025-03-10</td>
          <td>Non-Coding RNA</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da95ac3ec1b6b3cbe7baf6551a5457106bc739f1" target='_blank'>
              DNA damage response in cancer: New treatment strategies
              </a>
            </td>
          <td>
            Hang-Hao Zhang
          </td>
          <td>2025-03-31</td>
          <td>New Cell</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9dec962af7d1ee05246dee3d378f9152a2d6c3e" target='_blank'>
              Genetic variants in NUDT15 gene their clinical implications in cancer therapy.
              </a>
            </td>
          <td>
            Y. Jarrar, Maria Ghishan, Fatima Khirfan, Nancy Hakooz
          </td>
          <td>2025-04-14</td>
          <td>Drug metabolism and personalized therapy</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/330a5593d326f97de03e4da3203d107ef7927e48" target='_blank'>
              Extending differential gene expression testing to handle genome aneuploidy in cancer
              </a>
            </td>
          <td>
            Katsiaryna Davydzenka, G. Caravagna, Guido Sanguinetti
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Fibroblast growth factor 1-4 (FGFR1-4) are well-known oncogenic drivers in many cancer types. Here, we studied the role of FGFRs in uterine leiomyoma (UL) that is a benign neoplasm arising from the myometrium and the most common tumour in women. Although ULs can be classified to molecular subtypes based on genetic drivers, potential secondary drivers are not well characterised. Methods We performed mutation analysis of RNA-sequencing data of ULs, followed by screening of FGFR alterations in our Finnish (n = 2677) and Swedish (n = 372) UL collections, utilising Sanger-, next-generation and Nanopore sequencing and SNP array data. The role of FGFR genes in UL predisposition was examined by GWAS. Results We identified FGFR activation in a subset of ULs on both genetic and epigenetic levels. In addition to single-nucleotide mutations in FGFR1/2, we detected an FGFR2-ERC1 fusion gene, FGFR1 gains and hypomethylation of regulatory regions of FGFR2/3. FGFR alterations were enriched in molecularly similar HMGA2, HMGA1 and PLAG1 UL subtypes. We also unveil a UL predisposing variant upstream of FGFR4 associated with increased expression in both normal myometrium and ULs. Conclusions Our results establish the role of FGFR signalling in the genesis of UL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee3a4ef449036c9f0b9b5728bbd660366041a80a" target='_blank'>
              Activation of FGFR genes by genetic and epigenetic alterations in uterine leiomyomas
              </a>
            </td>
          <td>
            Vilja Jokinen, Aurora Taira, Åsa Kolterud, Isa Ahlgren, Kimmo Palin, Riku Katainen, Maritta Räisänen, Eevi Kaasinen, Sini Ilves, A. Raitila, Helena Kopp Kallner, Emma Siili, Ralf Bützow, O. Heikinheimo, A. Pasanen, A. Karhu, Niko Välimäki, L. Aaltonen
          </td>
          <td>2025-02-27</td>
          <td>BJC Reports</td>
          <td>0</td>
          <td>111</td>
        </tr>

        <tr id="ABSTRACT The non‐coding genome, constituting 98% of human DNA, remains largely unexplored, yet holds potential for identifying new biomarkers and therapeutic targets in acute lymphoblastic leukemia (ALL). In this study, we conducted a systematic analysis of recurrent somatic non‐coding single nucleotide variants (SNVs) in pediatric B‐cell precursor (BCP) ALL. We leveraged whole genome sequencing (WGS) data from 345 pediatric BCP ALL cases, representing all major genetic subtypes and identified 346 mutational hotspots that harbored somatic SNVs in at least three cases. Through the integration of paired RNA sequencing along with published ChIP‐seq and ATAC‐seq data, we found 128 non‐coding hotspots associated with differentially expressed genes nearby, which were enriched for cis‐regulatory elements, demonstrating the effectiveness of multi‐omics integration in distinguishing pathogenic mutations from passengers. We identified one mutational hotspot that was associated with increased expression of the leukemia‐associated gene NRAS in three primary ALLs. Micro‐C analysis in the leukemia cell line demonstrated interactions between the hotspot region and NRAS regulatory elements. Dual luciferase assays indicated that the mutations disrupted regulatory interactions and CRISPR‐mediated deletion of the region significantly upregulated NRAS, confirming the hypothesized regulatory link. Altogether, we provide new insights into the functional roles of non‐coding mutations in leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6089d599cb2b181049865efd98f322d1bb52759" target='_blank'>
              Discovery of Cis‐Regulatory Mechanisms via Non‐Coding Mutations in Acute Lymphoblastic Leukemia
              </a>
            </td>
          <td>
            Efe Aydın, Eleanor L Woodward, Gladys Telliam Dushime, Rebeqa Gunnarsson, H. Lilljebjörn, L. Moura-Castro, T. Fioretos, B. Johansson, K. Paulsson, Minjun Yang
          </td>
          <td>2025-03-01</td>
          <td>Genes, Chromosomes & Cancer</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (ecDNA) are commonly produced within the nucleus to drive genome dynamics and heterogeneity, enabling cancer cell evolution and adaptation. However, the mechanisms underlying ecDNA biogenesis remain poorly understood. Here using genome-wide CRISPR screening in human cells, we identified the BRCA1-A and the LIG4 complexes mediate ecDNA production. Following DNA fragmentation, the upstream BRCA1-A complex protects DNA ends from excessive resection, promoting end-joining for circularization. Conversely, the MRN complex, which mediates end resection and thus antagonizes the BRCA1-A complex, suppresses ecDNA formation. Downstream, LIG4 conservatively catalyzes ecDNA production in Drosophila and mammals, with patient tumor ecDNA harboring junctions marked by LIG4 activity. Notably, disrupting LIG4 or BRCA1-A in cancer cells impairs ecDNA-mediated adaptation, hindering resistance to both chemotherapy and targeted therapies. Together, our study reveals the roles of the LIG4 and BRCA1-A complexes in ecDNA biogenesis, and uncovers new therapeutic targets to block ecDNA-mediated adaptation for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65472598115e4d60921b909d76b9f2af808705a0" target='_blank'>
              BRCA1-A and LIG4 complexes mediate ecDNA biogenesis and cancer drug resistance
              </a>
            </td>
          <td>
            Oliver W. Chung, Shun Yao, Fu Yang, Ling Wang, Christian G Cerda-Smith, Haley M Hutchinson, Kris C. Wood, Weijia Su, Mustafa Khasraw, Lee Zou, Dale A. Ramsden, Z. Zhang
          </td>
          <td>2025-02-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="All cancer cells must adopt a telomere maintenance mechanism to achieve replicative immortality. Most human cancer cells utilize the enzyme telomerase to maintain telomeres. Alternative splicing of TERT regulates the amount and function of telomerase, however many alternative splicing isoforms of TERT have unknown functions. Single molecule long read RNA/cDNA sequencing of TERT revealed 45 TERT mRNA variants including 13 known and 32 novel variants. Among the variants, TERT Delta 2–4, which lacks exons 2–4 but retains the original open reading frame, was selected for further study. Induced pluripotent stem cells and cancer cells express higher levels of TERT Delta 2–4 compared to primary human bronchial epithelial cells. Overexpression of TERT Delta 2–4 enhanced clonogenicity and resistance to cisplatin-induced cell death. Knockdown of endogenous TERT Delta 2–4 in Calu-6 cells reduced clonogenicity and resistance to cisplatin. Our results suggest that TERT Delta 2–4 enhances cancer cells’ resistance to cell death. RNA sequencing following knockdown of Delta 2–4 TERT indicates that translation is downregulated and that mitochondrial related proteins are upregulated compared to controls. Overall, our data indicate that TERT produces many isoforms that influence the function of TERT and the abundance and activity of telomerase.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31160a8616dac1a93a1309bf2063539c61bdfcd2" target='_blank'>
              Discovery and characterization of a novel telomerase alternative splicing isoform that protects lung cancer cells from chemotherapy induced cell death
              </a>
            </td>
          <td>
            Jeongjin J. Kim, Alexander Ahn, Jeffrey Y Ying, Andrew T. Ludlow
          </td>
          <td>2025-02-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Heterogeneity is inherent to living organisms and it determines cell fate and phenotypic variability. Despite its ubiquity, the underlying molecular mechanisms and the genetic basis linking genotype to-phenotype heterogeneity remain a central challenge. Here we construct a yeast knockout library with a clone and genotype RNA barcoding structure suitable for genome-scale analyses to generate a high-resolution single-cell yeast transcriptome atlas of 3500 mutants under control and stress conditions. We find that transcriptional heterogeneity reflects the coordinated expression of specific gene programs, generating a continuous of cell states that can be responsive to external insults. Cell state plasticity can be genetically modulated with mutants that act as state attractors and disruption of state homeostasis results in decreased adaptive fitness. Leveraging on intra-genetic variability, we establish that regulators of transcriptional heterogeneity are functionally diverse and influenced by the environment. Our multimodal perturbation-based single-cell Genotype-to-Transcriptome Atlas in yeast provides insights into organism-level responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89465a1373d3260110cc521e62c1032cd2c67144" target='_blank'>
              A single-cell resolved genotype-phenotype map using genome-wide genetic and environmental perturbations
              </a>
            </td>
          <td>
            Mariona Nadal-Ribelles, Carme Solé, Anna Díez-Villanueva, Camille Stephan-Otto Attolini, Yaima Matas, Lars M. Steinmetz, Eulàlia de Nadal, F. Posas
          </td>
          <td>2025-03-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Therapy resistance in breast cancer is increasingly attributed to polyploid giant cancer cells (PGCCs), which arise through whole genome doubling and exhibit heightened resilience to standard treatments. Characterized by enlarged nuclei and increased DNA content, these cells tend to be dormant under therapeutic stress, driving disease relapse. Despite their critical role in resistance, strategies to effectively target PGCCs are limited, largely due to the lack of high-throughput methods for assessing their viability. Traditional assays lack the sensitivity needed to detect PGCC-specific elimination, prompting the development of novel approaches. To address this challenge, we developed a high-throughput single-cell morphological analysis workflow designed to differentiate compounds that selectively inhibit non-PGCCs, PGCCs, or both. Using this method, we screened a library of 2726 FDA Phase 1-approved drugs, identifying promising anti-PGCC candidates, including proteasome inhibitors, FOXM1, CHK, and macrocyclic lactones. Notably, RNA-Seq analysis of cells treated with the macrocyclic lactone Pyronaridine revealed AXL inhibition as a potential strategy for targeting PGCCs. Although our single-cell morphological analysis pipeline is powerful, empirical testing of all existing compounds is impractical and inefficient. To overcome this limitation, we trained a machine learning model to predict anti-PGCC efficacy in silico, integrating chemical fingerprints and compound descriptions from prior publications and databases. The model demonstrated a high correlation with experimental outcomes and predicted efficacious compounds in an expanded library of over 6,000 drugs. Among the top-ranked predictions, we experimentally validated five compounds as potent PGCC inhibitors using cell lines and patient-derived models. These findings underscore the synergistic potential of integrating high-throughput empirical screening with machine learning-based virtual screening to accelerate the discovery of novel therapies, particularly for targeting therapy-resistant PGCCs in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0482d6424427dcdf25109ef995747da0a47de141" target='_blank'>
              High-Throughput Empirical and Virtual Screening To Discover Novel Inhibitors of Polyploid Giant Cancer Cells in Breast Cancer
              </a>
            </td>
          <td>
            Yushu Ma, Chien-Hung Shih, Jinxiong Cheng, Hsiao-Chun Chen, Li-Ju Wang, Yanhao Tan, Yuan Zhang, Daniel D Brown, S. Oesterreich, Adrian V Lee, Yu-Chiao Chiu, Yu-Chih Chen
          </td>
          <td>2025-03-04</td>
          <td>Analytical Chemistry</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Ongoing mutagenesis in cancer drives genetic diversity throughout the natural history of cancers. As the activities of mutational processes are dynamic throughout evolution, distinguishing the mutational signatures of ‘active’ and ‘historical’ processes has important implications for studying how tumors evolve. This can aid in understanding mutagenic states at the time of presentation, and in associating active mutational process with therapeutic resistance. As bulk sequencing primarily captures historical mutational processes, we studied whether ultra-low-coverage single-cell whole-genome sequencing (scWGS), which measures the distribution of mutations across hundreds or thousands of individual cells, could enable the distinction between historical and active mutational processes. While technical challenges and data sparsity have limited mutation analysis in scWGS, we show that these data contain valuable information about dynamic mutational processes. To robustly interpret single nucleotide variants (SNVs) in scWGS, we introduce ArtiCull, a method to identify and remove SNV artifacts by leveraging evolutionary constraints, enabling reliable detection of mutations for signature analysis. Applying this approach to scWGS data from pancreatic ductal adenocarcinoma (PDAC), triple-negative breast cancer (TNBC), and high-grade serous ovarian cancer (HGSOC), we uncover temporal and spatial patterns in mutational processes. In PDAC, we observe a temporal increase in mismatch repair deficiency (MMRd). In cisplatin-treated TNBC patient-derived xenografts, we identify therapy-induced mutagenesis and inactivation of APOBEC3 activity. In HGSOC, we show distinct patterns of APOBEC3 mutagenesis, including late tumor-wide activation in one case and clade-specific enrichment in another. Additionally, we detect a clone-specific increase in SBS17 activity, in a clone previously linked to recurrence. Our findings establish ultra-low-coverage scWGS as a powerful approach for studying active mutational processes that may influence ongoing clonal evolution and therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed6d1d9f289244977b2dfb8fbeaff86f943a227d" target='_blank'>
              Inferring active mutational processes in cancer using single cell sequencing and evolutionary constraints
              </a>
            </td>
          <td>
            Gryte Satas, Matthew A. Myers, Andrew McPherson, Sohrab P. Shah
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="In recent years, synthetic lethality has become an important theme in the field of targeted cancer therapy. Synthetic lethality refers to simultaneous defects in two or more genes leading to cell death, whereas defects in any single gene do not lead to cell death. Taking advantage of the genetic vulnerability that exists within cancer cells, it theoretically has no negative impact on healthy cells and has fewer side effects than non-specific chemotherapy. Currently, targeted cancer therapies focus on inhibiting key pathways in cancer. However, it has been found that over-activation of oncogenic-related signaling pathways can also induce cancer cell death, which is a major breakthrough in the new field of targeted therapies. In this review, we summarize the conventional gene targets in synthetic lethality (PARP, ATR, ATM, WEE1, PRMT) and provide an in-depth analysis of their latest potential mechanisms. We explore the impact of over-activation of pathways such as PI3K/AKT, MAPK, and WNT on cancer cell survival, and present the technical challenges of current research. Important theoretical foundations and insights are provided for the application of synthetic lethal strategies in cancer therapy, as well as future research directions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49a12057bb33fb82ecea65af89cddf52c4a9e3cd" target='_blank'>
              Perspectives on cancer therapy—synthetic lethal precision medicine strategies, molecular mechanisms, therapeutic targets and current technical challenges
              </a>
            </td>
          <td>
            Shixuan Peng, Mengle Long, Qisheng Chen, Zhijian Yin, Chang Zeng, Wanyong Zhang, Qingyang Wen, Xinwen Zhang, Weiqi Ke, Yongjun Wu
          </td>
          <td>2025-04-16</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Adenosine base editors (ABEs) facilitate A·T to G·C base pair conversion with significant therapeutic potential for correcting pathogenic point mutations in human genetic diseases, such as sickle cell anemia and β-thalassemia. Unlike CRISPR–Cas9 systems that induce double-strand breaks, ABEs operate through precise deamination, avoiding chromosomal instability. However, the off-target editing effects of ABEs remain inadequately characterized. In this study, we present a biochemical method Selict-seq, designed to evaluate genome-wide off-target editing by ABEs. Selict-seq specifically captures deoxyinosine-containing single-stranded DNA and precisely identifies deoxyadenosine-to-deoxyinosine (dA-to-dI) mutation sites, elucidating the off-target effects induced by ABEs. Through investigations involving three single-guide RNAs, we identified numerous unexpected off-target edits both within and outside the protospacer regions. Notably, ABE8e(V106W) exhibited distinct off-target characteristics, including high editing rates (>10%) at previously unreported sites (e.g. RNF2 and EMX1) and out-of-protospacer mutations. These findings significantly advance our understanding of the off-target landscape associated with ABEs. In summary, our approach enables an unbiased analysis of the ABE editome and provides a widely applicable tool for specificity evaluation of various emerging genome editing technologies that produce intermediate products as deoxyinosine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c7f86cbbb409d5234a83d13231df0efe7d59590" target='_blank'>
              Selict-seq profiles genome-wide off-target effects in adenosine base editing
              </a>
            </td>
          <td>
            Kexin Yuan, Xin Xi, Shaoqing Han, Jingyu Han, Bin Zhao, Qi Wei, Xiang Zhou
          </td>
          <td>2025-04-10</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is an aggressive brain tumor with poor prognosis. Recent advancements in bioinformatics have contributed to uncovering the genetic alterations that underlie the development and progression of GBM. Analysis of extensive genomic data led to the identification of significant pathways involved in GBM, such as the PI3K/AKT/mTOR and Ras/Raf/MEK/ERK signaling pathways, alongside key genes such as EGFR, TP53 and TERT. These findings have enhanced our understanding of GBM biology and led to the identification of new therapeutic targets. Bioinformatics has become an indispensable tool in pinpointing the genetic modifications that drive GBM, paving the way for innovative treatment strategies. This approach not only aids in comprehending the complexities of GBM but also holds promise for improving outcomes in patients suffering from this devastating disease. The ongoing integration of bioinformatics in GBM research continues to be vital for advancing therapeutic options.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9bc4bba3faa28b20f6c816df320e96742c008f6" target='_blank'>
              Key genes altered in glioblastoma based on bioinformatics (Review)
              </a>
            </td>
          <td>
            Marcelino Al Ghafari, Nour El Jaafari, Mariam Mouallem, Tala Maassarani, M. El-Sibai, Ralph Abi‑Habib
          </td>
          <td>2025-03-24</td>
          <td>Oncology Letters</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Simple Summary Proto-oncogenes and tumor-suppressor genes play important roles in preventing the development of tumors. The former supports healthy cell growth and division, while the latter regulates cell division. Mutations in proto-oncogenes, converting them into oncogenes, and mutations in tumor-suppressor genes can lead to uncontrolled cell proliferation and cancer development. This review explores key molecular pathways associated with these genes, highlighting their influence on cancer progression and treatment resistance, which are crucial for creating effective, personalized, and targeted treatments for patients. This comprehensive review also discusses the recent advancements and emerging technologies that enhance the analysis, diagnosis, prevention, and treatment of cancer at the genomic level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4acb0abcdb89e4becb0be9ae1ca6990d3d6cdf49" target='_blank'>
              Translational Advances in Oncogene and Tumor-Suppressor Gene Research
              </a>
            </td>
          <td>
            R. Stojchevski, Edward Agus Sutanto, Rinni Sutanto, N. Hadzi-Petrushev, M. Mladenov, Sajal Raj Singh, J. K. Sinha, Shampa Ghosh, Bhuvaneshwar Yarlagadda, Krishna Kumar Singh, Prashant Verma, Sonali Sengupta, Rakesh Bhaskar, Dimiter B Avtanski
          </td>
          <td>2025-03-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Background Schlafen11 (SLFN11) is a key gene in p53-independent apoptosis through ribosome stalling; however, systematic research has been conducted on its role in the tumor immune microenvironment, clinical application, and immunotherapy response across pan-cancer. Method Public data were downloaded and multi-omics approaches were used to investigate the relationship between the expression level of SLFN11 and spatial position, biological function, immune landscape, and clinical application values. Cell Counting Kit-8 assay and quantitative real-time PCR were used to validate the expression level of SLFN11 and drug sensitivity in colorectal cancer samples. Result Our study revealed that SLFN11 was downregulated in most cancers and correlated with DNA repair, the P53 pathway and immune response in tumor development progress by multi-omics analysis. Dysregulated SLFN11 is accompanied by several immune cell infiltrations and immune-related regulators, which can be a promising screening and prognostic biomarker and chemotherapy predictive target for clinical application. In vitro experiments proved that downregulated SLFN11 is a useful diagnostic biomarker and is linked to imatinib resistance in colorectal cancer. Conclusion The expression level of SLFN11 has a substantial promise as a valuable biomarker for diagnosis and a predictive indicator for assessing the effectiveness of chemotherapy and immunotherapy in human cancers, which deserves further additional basic experiments and clinical trials to prove.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c098d42efd569f7e477b717e0186198035c93eb9" target='_blank'>
              Comprehensive pan-cancer analysis indicates key gene of p53-independent apoptosis is a novel biomarker for clinical application and chemotherapy in colorectal cancer
              </a>
            </td>
          <td>
            Jianing Yan, Jingzhi Wang, Min Miao, Y. Shao
          </td>
          <td>2025-03-27</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Establishing DNA as the genetic material, deciphering the genetic code and defining the double helical structure of the genome provided a foundation for pursuing a mechanistic understanding of gene expression that is operative for biological control and pathology. Equally important are a series of discoveries that have established epigenetic mechanisms, defined as the control of gene expression by non-DNA-encoded regulation, mediated by an evolving histone code that supports chromatin states, as complementary and obligatory determinants for biological activity. Evidence is accruing for pivotal contributions by epigenetic parameters of control to gene expression that is required for development and differentiation as well as for transient and sustained responsiveness to physiological regulatory cues that support cell structure, function, cell survival, tissue repair and remodeling. Epigenetic control is emerging as a decisive contributor to the regulatory compliance for the highly integrative processes that balance phenotypic function and tumor suppression with aberrant gene expression that accompanies the onset and progression of disease, strikingly illustrated with tumorigenesis and skeletal disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be4c2c01eadaece3705383df803332330abc2445" target='_blank'>
              Reflections and perspectives on epigenetically mediated biological control: compromises in cancer and skeletal pathology
              </a>
            </td>
          <td>
            Gary S. Stein
          </td>
          <td>2025-04-11</td>
          <td>Academia Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56c8bae667d5d467ee9b1edf8b1b3bbcf92dc895" target='_blank'>
              Single-cell sequencing for lung cancer research: Progress and prospects
              </a>
            </td>
          <td>
            Zhongsha Li, Wei Su, Bingbing Bai, Yuesong Wu, Baoyin Zhang, Dejuan Xiang, Gilwa Borjigin, Jinhai Deng, Jian Sun, Mingzhu Yin
          </td>
          <td>2025-03-12</td>
          <td>Eurasian Journal of Medicine and Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Aim: As intrinsic resistance - often driven by concurrent genomic alterations in tumor suppressor genes or oncogenes - remains a major challenge in oncology, this work aimed to comprehensively analyze BRAF somatic alterations across cancer types and identify new potential therapeutic strategies to overcome drug resistance. Methods: We conducted an extensive analysis of genomics, transcriptomics, and clinical data retrieved from public repositories, including cBioPortal. Our comprehensive analysis examined BRAF alterations [point mutations, structural variants (SVs) and copy number alteration] in more than 217,000 tumor samples across 120 distinct tumor types from primary and metastatic sites in both adult and pediatric cohorts, focusing on mutual exclusivity and co-occurrence of mutations in other oncogenes or tumor suppressors. The work also explores the association of BRAF somatic alterations with survival, clinical and molecular features. Results: Analysis of mutation frequencies across cancer types revealed that BRAFV600E represents approximately 90% of all BRAF alterations. While melanoma and thyroid carcinoma show the highest prevalence of BRAF mutations, followed by colorectal and non-small cell lung cancer in terms of absolute number of patients harboring BRAF mutations worldwide, notably high mutation frequencies were identified in rare malignancies, including hairy-cell leukemia, ganglioglioma, and serous borderline ovarian tumors. The comprehensive analysis of genomic profiling data across these tumors uncovered distinct patterns of co-occurring and mutually exclusive alterations in oncogenes and tumor suppressor genes, illuminating resistance mechanisms and suggesting novel therapeutic combinations. Conclusion: Comprehensive genomic profiling is critical for optimizing targeted therapy and overcoming drug resistance in BRAF-mutated cancers. The identification of co-occurring alterations provides opportunities for rational combination therapies, emphasizing the importance of detailed mutation profiling in developing effective treatment strategies across diverse cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7fa7f528d2835416489339d23ab2a839922e3b0" target='_blank'>
              Unraveling BRAF alterations: molecular insights to circumvent therapeutic resistance across cancer types
              </a>
            </td>
          <td>
            Caterina Perfetto, Marianna Aprile, Simona Cataldi, Elisa Giovannetti, Valerio Costa
          </td>
          <td>2025-03-24</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a21c97e108e69ab358170dee175b96d9645c472" target='_blank'>
              Cancer Dependency Map: Past and Present
              </a>
            </td>
          <td>
            Rand Arafeh, T. Shibue, Francisca Vazquez, William C Hahn
          </td>
          <td>2025-04-11</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Advances in single-cell multi-omics technologies have deepened our understanding of cancer biology by integrating genomic, transcriptomic, epigenomic, and proteomic data at single-cell resolution. These single-cell multi-omics technologies provide unprecedented insights into tumour heterogeneity, tumour microenvironment, and mechanisms of therapeutic resistance, enabling the development of precision medicine strategies. The emerging field of single-cell multi-omics in genomic medicine has improved patient outcomes. However, most clinical applications still depend on bulk genomic approaches, which fail to directly capture the genomic variations driving cellular heterogeneity. In this review, we explore the common single-cell multi-omics platforms and discuss key analytical steps for data integration. Furthermore, we highlight emerging knowledge in therapeutic resistance and immune evasion, and the potential of new therapeutic innovations informed by single-cell multi-omics. Finally, we discuss the future directions of the application of single-cell multi-omics technologies. By bridging the gap between technological advancements and clinical implementation, this review provides a roadmap for leveraging single-cell multi-omics to improve cancer treatment and patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7137167161ba1e0d324611dc617b5e8bcaedde60" target='_blank'>
              Single-Cell Multi-Omics: Insights into Therapeutic Innovations to Advance Treatment in Cancer
              </a>
            </td>
          <td>
            Angel Guan, Camelia Quek
          </td>
          <td>2025-03-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d6303d9f75dded8dc24ee6a6e2566fac41f5275" target='_blank'>
              Comprehensive Analysis of Oncogenic Somatic Alterations of Mismatch Repair Gene in Breast Cancer Patients
              </a>
            </td>
          <td>
            Yin Yan, Yang Wang, Junjie Tang, Xiao-ran Liu, Jichuan Wang, Guohong Song, Huiping Li
          </td>
          <td>2025-04-18</td>
          <td>Bioengineering</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The use of AAV vectors for in vivo gene therapy has demonstrated the potential to permanently correct genetic diseases by delivering functional gene copies to the nuclei of affected tissues. AAV vectors, as tools for in vivo gene delivery, are particularly appealing and have shown safety and long-term efficacy in numerous organ-targeted experiments. Nevertheless, employing AAV vectors for gene therapy in the brain faces a notable hurdle in the shape of immune responses, chiefly instigated by the brain’s resident immune cells, microglia. Additionally, lower levels of AAV vector-neutralizing antibodies have been detected in the cerebrospinal fluid compared to the circulatory system. This research, leveraging transcriptomic and single-cell RNA sequencing (scRNA-seq) data in conjunction with Mendelian randomization analysis, has identified the potential role of the XBP1 protein in mediating B-cell immunosuppression in the brain via the MDK-NCL ligand-receptor pair and associated genes. Furthermore, it paves the way for further investigation into the regulatory factors and pathways within the immune modulation network, as well as their prospective beneficial implications in immunotherapeutic treatments. By employing various innovative approaches, the study seeks to recreate the immune environment generated by AAV in the brain and preliminarily explore the immune suppression mechanisms induced by AAV vectors in the brain. Supplementary Information The online version contains supplementary material available at 10.1007/s12026-025-09609-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fbe09900693cf8126b4ed7a1f742723962eeea9" target='_blank'>
              A comprehensive analysis of scRNA-Seq and RNA-Seq unveils B cell immune suppression in the AAV-loaded brain
              </a>
            </td>
          <td>
            Shunyu Wu, Lu Xue, Xiang Li, Yaoxuan Wang, Yuting Zhu, Yuanbo Luo, Jiayu Sun, Tingting Jin, Wenying Shu, Zhaoyan Wang
          </td>
          <td>2025-03-05</td>
          <td>Immunologic Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract CRISPR/Cas9-mediated homology-directed repair (HDR) allows precise gene editing, but its efficiency remains low for certain cell types, such as human induced pluripotent stem cells (hiPSCs). In this study, we aimed to introduce the CTNNA1: c.2023C>T (p.Q675*) genetic alteration, which is associated with Hereditary Diffuse Gastric Cancer, into hiPSCs using CRISPR/Cas9. We designed a single-guide RNA targeting the alteration site and a single-stranded oligonucleotide donor DNA template for HDR-based repair. Herein, we report the successful introduction of the CTNNA1: c.2023C>T homozygous alteration in one hiPSC line, which resulted in severe phenotypic changes, including impaired colony formation and cell proliferation. Additionally, we established a straightforward protocol to assess hiPSCs karyotype integrity, ensuring the chromosomal stability required for the gene-editing process. This protocol involves routine G-banding analysis that is required for regular quality controls during handling of hiPSCs. This study demonstrates an efficient approach to precisely edit hiPSCs by CRISPR/Cas9 and highlights the essential role of CTNNA1 expression in maintaining hiPSC viability. Our methodology provides a valuable framework for modeling disease-associated alterations in human-derived cellular models that can be reproduced for other genes and other types of cell lines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77b6de2d94791247728d12901bea9b29daa88c32" target='_blank'>
              A protocol for karyotyping and genetic editing of induced pluripotent stem cells with homology-directed repair mediated CRISPR/Cas9
              </a>
            </td>
          <td>
            Silvana Lobo, Rita Barbosa-Matos, Sofia Dória, A. Pedro, Ana Brito, D. Ferreira, Carla Oliveira
          </td>
          <td>2025-02-28</td>
          <td>Biology Methods & Protocols</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9b5519e4d897e90daae110bc4297e9d131ea31b" target='_blank'>
              Biobank of genetically defined murine prostate cancer tumoroids uncovers oncogenic pathways and drug vulnerabilities driven by PTEN-loss
              </a>
            </td>
          <td>
            Jessica Kalla, Thomas E. Dillinger, Zlata Pavlovicova, Reema Jacob, Emine Atas, Anil Baskan, Kristina Draganić, A. Tiefenbacher, T. Limberger, Theresia Mair, Gabriel Wasinger, Ludovica Villanti, S. Kubicek, Lukas Kenner, G. Egger
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/601b29cba678e1083184d2ddf34dbba867983f13" target='_blank'>
              Abstract B010: Exploring the synthetic lethal interactions between mRNA methyl-5-cytocine at sites of R-loops and mismatch repair deficiency in solid tumor
              </a>
            </td>
          <td>
            Li Lan
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="ABSTRACT ALK fusions, such as the classic EML4‐ALK, are known drivers of lung cancer and effective therapeutic targets. However, variant ALK fusions, including intergenic fusions like LOC388942‐ALK (LA), have been detected in increasing numbers of patients, with their roles in tumorigenesis and ALK inhibitor resistance remaining unclear. Using CRISPR/Cas9, we generated the LA fusion in A549 and H441 cells, confirming elevated ALK expression via qRT‐PCR and immunohistochemistry (IHC) staining. Functional analyses showed that LA significantly promoted tumor growth in vitro and in vivo while conferring increased resistance to alectinib. RNA‐seq revealed upregulation of the FOS pathway in LA tumors, identifying FOS as a potential therapeutic target. Subsequently, we demonstrated that FOS disruption and inhibition sensitized LA tumors to treatment. RNA‐seq profiling demonstrated that FOS depletion in LOC388942‐ALK tumor significantly downregulated multiple oncogenic pathways related to cell cycle progression, DNA replication fidelity, and extracellular matrix remodeling, suggesting a pivotal role of FOS in maintaining tumor growth. These findings establish LOC388942‐ALK as a novel oncogenic driver in lung cancer, highlighting its role in tumor growth and ALK inhibitor resistance. Targeting FOS may provide a promising therapeutic strategy for tumors harboring this intergenic fusion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/459d6322ba2d77e523434dfb3f269f42f6deaabb" target='_blank'>
              Identifying the Intergenic ALK Fusion LOC388942‐ALK as a Driver of Non–Small Cell Lung Cancer
              </a>
            </td>
          <td>
            X. Zhai, Manli Wang, Qi Zhang, Donglin Li, Yanmou Wu, Zuoyu Liang, Jiewei Liu, Weiya Wang, Yu Liu, Guowei Che, Qinghua Zhou, Chong Chen
          </td>
          <td>2025-03-27</td>
          <td>MedComm</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/654c576507b39c0969a5f79ea0daae0b983731a4" target='_blank'>
              Riding the Wave of Progress: Examining the Current Landscape and Future Potential of MicroRNAs in Cancer Gene Therapy.
              </a>
            </td>
          <td>
            Saleem Ahmad, Hanif Khan, Njoka Irene Muthoni, Mohammed Alissa, Awaji Y. Safhi, F. Y. Sabei, Osama Abdulaziz, Khadijah Hassan Khan, Essam H Ibrahim, S. Khan
          </td>
          <td>2025-03-24</td>
          <td>Current gene therapy</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f8ba9e3fcdae65a1005fa8694699f4a2467c8be" target='_blank'>
              Intratumour oxidative hotspots provide a niche for cancer cell dissemination.
              </a>
            </td>
          <td>
            Yoshifumi Ueda, Shigeki Kiyonaka, L. Selfors, Keisuke Inoue, Hiroshi Harada, Tomohiro Doura, Kunishige Onuma, Makoto Uchiyama, Ryuhei Kurogi, Yuji Yamada, Jiacheng H Sun, Reiko Sakaguchi, Yuki Tado, Haruki Omatsu, Harufumi Suzuki, M. Aoun, Takahiro Nakayama, Taketoshi Kajimoto, Tetsuya Yano, Rikard Holmdahl, I. Hamachi, Masahiro Inoue, Yasuo Mori, Nobuaki Takahashi
          </td>
          <td>2025-02-21</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c926ae9ba30483b00d605482594a078fa10068fc" target='_blank'>
              The landscape of cell lineage tracing.
              </a>
            </td>
          <td>
            Ye Feng, Guang Liu, Haiqing Li, Lin Cheng
          </td>
          <td>2025-02-28</td>
          <td>Science China. Life sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Significance Chemotherapeutics effectively kill cancer cells but can also induce mutations in surviving cells, leading to therapy resistance and secondary malignancies. We analyzed the DNA rearrangements caused by doxorubicin in murine HER2+ breast cancer organoids, focusing on genomic structural variants (SVs). Using single-cell sequencing, we revealed extensive therapy-induced SVs and genomic instability in a subset of posttreatment cells. Additionally, by advancing computational methods to analyze nucleosome occupancy (NO) in different cell types of the mammary lineage, we link SVs to gene expression changes that could potentially lead to breast cancer progression. This research underscores the need for strategies to minimize drug-induced genomic damage to improve treatment efficacy and patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95047a549d97c358dede6e2ed75d74511cbd0c06" target='_blank'>
              Structural variant and nucleosome occupancy dynamics postchemotherapy in a HER2+ breast cancer organoid model
              </a>
            </td>
          <td>
            Maja Starostecka, Hyobin Jeong, Patrick Hasenfeld, Eva Benito-Garagorri, Tania Christiansen, Catherine Stober Brasseur, Maise Gomes Queiroz, Marta Garcia Montero, Martin Jechlinger, J. Korbel
          </td>
          <td>2025-02-24</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>98</td>
        </tr>

        <tr id="Immune checkpoint inhibitors (ICIs) have transformed the treatment landscape across multiple cancer types achieving durable responses for a significant number of patients. Despite their success, many patients still fail to respond to ICIs or develop resistance soon after treatment. We sought to identify early treatment features associated with ICI outcome. We leveraged the MC38 syngeneic tumor model because it has variable response to ICI therapy driven by tumor intrinsic heterogeneity. ICI response was assessed based on the level of immune cell infiltration into the tumor – a well-established clinical hallmark of ICI response. We generated a spatial atlas of 48,636 transcriptome-wide spots across 16 tumors using spatial transcriptomics; given the tumors were difficult to profile, we developed an enhanced transcriptome capture protocol yielding high quality spatial data. In total, we identified 8 tumor cell subsets (e.g., proliferative, inflamed, and vascularized) and 4 stroma subsets (e.g., immune and fibroblast). Each tumor had orthogonal histology and bulk-RNA sequencing data, which served to validate and benchmark observations from the spatial data. Our spatial atlas revealed that increased tumor cell cholesterol regulation, synthesis, and transport were associated with a lack of ICI response. Conversely, inflammation and T cell infiltration were associated with response. We further leveraged spatially aware gene expression analysis, to demonstrate that high cholesterol synthesis by tumor cells was associated with cytotoxic CD8 T cell exclusion. Finally, we demonstrate that bulk RNA-sequencing was able to detect immune correlates of response but lacked the sensitivity to detect cholesterol synthesis as a feature of resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5d48556d03df2b88d1c06ed57c2bca71cfaf9b2" target='_blank'>
              Spatial genomics reveals cholesterol metabolism as a key factor in colorectal cancer immunotherapy resistance
              </a>
            </td>
          <td>
            Andrew J. Kavran, Yulong Bai, Brian Rabe, Anna Kreshock, Andrew Fisher, Yelena Cheng, Anne Lewin, Chao Dai, Matthew J. Meyer, Konstantinos J. Mavrakis, Anna Lyubetskaya, Eugene Drokhlyansky
          </td>
          <td>2025-03-18</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Brain metastasis represents a significant challenge in oncology, driven by complex molecular and epigenetic mechanisms that distinguish it from primary tumors. While recent research has focused on identifying genomic mutation drivers with potential clinical utility, these strategies have not pinpointed specific genetic mutations responsible for site-specific metastasis to the brain. It is now clear that successful brain colonization by metastatic cancer cells requires intricate interactions with the brain tumor ecosystem and the acquisition of specialized molecular traits that facilitate their adaptation to this highly selective environment. This is best exemplified by widespread transcriptional adaptation during brain metastasis, resulting in aberrant gene programs that promote extravasation, seeding, and colonization of the brain. Increasing evidence suggests that epigenetic mechanisms play a significant role in shaping these pro-brain metastasis traits. This review explores dysregulated chromatin patterns driven by chromatin remodeling, histone modifications, DNA/RNA methylation, and other epigenetic regulators that underpin brain metastatic seeding, initiation, and outgrowth. We provide novel insights into how these epigenetic modifications arise within both the brain metastatic tumor and the surrounding brain metastatic tumor ecosystem. Finally, we discuss how the inherent plasticity and reversibility of the epigenomic landscape in brain metastases may offer new therapeutic opportunities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fe358dc3b7575e837baed5ca2b106b75b4f36a3" target='_blank'>
              The epigenetic landscape of brain metastasis.
              </a>
            </td>
          <td>
            Aoibhín M Powell, Louise Watson, L. Luzietti, S. Prekovic, L. Young, D. Varešlija
          </td>
          <td>2025-02-27</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8354bb2cbdecdc87317ac3be8bce15b2494319c2" target='_blank'>
              Emerging Roles of ACTL6A as an Oncogenic Hub: Transcriptional Regulation and Beyond.
              </a>
            </td>
          <td>
            Kelvin K Tsai, Lin-Hsin Cheng, , Pei-Ming Yang, Chin-Pin Chuu
          </td>
          <td>2025-04-07</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Genome sequencing of cancer and normal tissues, alongside single-cell transcriptomics, continues to produce findings that challenge the idea that cancer is a 'genetic disease', as posited by the somatic mutation theory (SMT). In this prevailing paradigm, tumorigenesis is caused by cancer-driving somatic mutations and clonal expansion. However, results from tumor sequencing, motivated by the genetic paradigm itself, create apparent 'paradoxes' that are not conducive to a pure SMT. But beyond genetic causation, the new results lend credence to old ideas from organismal biology. To resolve inconsistencies between the genetic paradigm of cancer and biological reality, we must complement deep sequencing with deep thinking: embrace formal theory and historicity of biological entities, and (re)consider non-genetic plasticity of cells and tissues. In this Essay, we discuss the concepts of cell state dynamics and tissue fields that emerge from the collective action of genes and of cells in their morphogenetic context, respectively, and how they help explain inconsistencies in the data in the context of SMT.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f9b3a10a05d5290d45b66f194c7384d1f7517bb" target='_blank'>
              The end of the genetic paradigm of cancer.
              </a>
            </td>
          <td>
            Sui Huang, Ana M Soto, C. Sonnenschein
          </td>
          <td>2025-03-01</td>
          <td>PLoS biology</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Abstract Functional characterization of genetic variants has the potential to advance the field of precision medicine by enhancing the efficacy of current therapies and accelerating the development of new approaches to combat genetic diseases. MUTYH is a DNA repair enzyme that recognizes and repairs oxidatively damaged guanines [8-oxoguanine (8-oxoG)] mispaired with adenines (8-oxoG·A). While some mutations in the MUTYH gene are associated with colorectal cancer, most MUTYH variants identified in sequencing databases are classified as variants of uncertain significance. Convoluting clinical classification is the absence of data directly comparing homozygous versus heterozygous MUTYH mutations. In this study, we present the first effort to functionally characterize MUTYH variants using precision genome editing to generate heterozygous and homozygous isogenic cell lines. Using a MUTYH-specific lesion reporter in which we site-specifically incorporate an 8-oxoG·A lesion in a fluorescent protein gene, we measure endogenous MUTYH enzymatic activity and classify them as pathogenic or benign. Further, we modify this reporter to incorporate the MUTYH repair intermediate (8-oxoG across from an abasic site) and validate it with co-immunoprecipitation experiments to demonstrate its ability to characterize the mechanism by which MUTYH mutants are defective at DNA repair.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2e7c0e491d2ef8489374e03b9d4bf701b388bba" target='_blank'>
              Precision genome editing and in-cell measurements of oxidative DNA damage repair enable functional and mechanistic characterization of cancer-associated MUTYH variants
              </a>
            </td>
          <td>
            Carlos A Vasquez, Nicola R B Osgood, Marcanthony U Zepeda, Dominika K Sandel, Quinn T. Cowan, Malalage N. Peiris, Daniel J Donoghue, Alexis C. Komor
          </td>
          <td>2025-03-20</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dd1c02bf99dc94856478bd198e131107c59bd8c" target='_blank'>
              KDM5 demethylases suppress R-loop-mediated “viral mimicry” and DNA damage in breast cancer cells
              </a>
            </td>
          <td>
            Lena Lau, Kurt W Henderson, Ahu Turkoz, Sara Linker, Dörte Schlesinger, Brad Townsley, Brian Egan, Shoba Ragunathan, Robert Rollins, Xianju Bi, Zhijian J. Chen, Oleg Brodsky, Clifford Restaino, Murali Gururajan, Kristen Jensen-Pergakes, A. Malarstig, Chames Kermi, Paul C Moore, Marie Classon
          </td>
          <td>2025-03-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2804e296b978220c32264085aca8c1be9f4a7db0" target='_blank'>
              Potential and Optimization of Mammalian Splice Riboswitches for the Regulation of Exon Skipping-Dependent Gene Expression and Isoform Switching within the ALOX5 Gene.
              </a>
            </td>
          <td>
            Robin W Bruckhoff, Olga Becker, Dieter Steinhilber, Beatrix Suess
          </td>
          <td>2025-02-26</td>
          <td>ACS synthetic biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Advancements in genomics have revolutionized human genetics by defining the genetic architecture of disease. However, identifying causal variants and their mechanisms of action remains a challenge in translating genetics into therapeutic interventions. Here, we developed Sensitive Transcriptomics And Genotyping by sequencing (STAG-seq), a high-throughput platform designed to define mechanistic genotype-phenotype relationships through simultaneous single-cell measurements of genomic DNA and RNA transcripts. Combined with base-editing, STAG-seq enables functionalization of variants in relevant cellular contexts. We demonstrate the applicability of this approach in several settings. First, we screened genetic perturbations to identify monoallelic and biallelic variant effects in primary human macrophages treated with innate immune stimuli. Next, we phenotyped clinically relevant missense variants associated with immunodeficiency and autoimmunity. Finally, we defined a noncoding variant in a pleiotropic autoimmunity locus that governs TNRC18 expression in primary T cells. STAG-seq thus enables variant phenotyping at scale to advance functional genomics and disease biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47496c4a5205c2d4eec3b8f1bd7dea12812365cb" target='_blank'>
              Sensitive Transcriptomics and Genotyping reveals function of genetic variants in immunity
              </a>
            </td>
          <td>
            Kai Liu, Anders Rasmussen, Wenkai Han, Qiyu Gong, Seneca Bohley, Dawn Chen, M. Kanai, Regan Maronick, Zhangyuan Yin, Chenlei Hu, Tanvi Jain, J. D. Martin-Rufino, Vijay G. Sankaran, Orr Ashenberg, Mark J. Daly, Fei Chen, Daniel B. Graham, Ramnik J. Xavier
          </td>
          <td>2025-03-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="ABSTRACT Background Genetic testing and sequencing technologies offer a comprehensive understanding of cancer genetics, providing rapid and cost‐effective solutions. In particular, these advanced technologies play an important role in assessing the complexities of the rare cancer types affecting several systems including the bone, endocrine, digestive, vascular, and soft tissue. This review will explore how genetic testing and sequencing technologies have contributed to the identification of biomarkers across several rare cancer types in diagnostic, therapeutic, and prognostic stages, thereby advancing PM. Methods A comprehensive literature search was conducted across PubMed (MEDLINE), EMBASE, and Web of Science using keywords related to sequencing technologies, genetic testing, and cancer. There were no restrictions on language, methodology, age, or publication date. Both primary and secondary research involving humans or animals were considered. Results In practice, fluorescence in situ hybridization, karyotype, microarrays and other genetic tests are mainly applied to identify specific genetic alterations and mutations associated with cancer progression. Sequencing technologies, such as next generation sequencing, polymerase chain reaction, whole genome or exome sequencing, enable the rapid analysis of millions of DNA fragments. These techniques assess genome structure, genetic changes, gene expression profiles, and epigenetic variations. Consequently, they help detect main intrinsic markers that are crucial for personalizing diagnosis, treatment options, and prognostic assessments, leading to better patient prognosis. This highlights why these methods are now considered as primary tools in rare cancer research. However, these methods still face multiple limitations, including false positive results, limited precision, and high costs. Conclusion Genetic testing and sequencing technologies have significantly advanced the field of rare cancer research by enabling the identification of key biomarkers for precision diagnosis, treatment, and prognosis. Despite existing limitations, their integration into clinical and research fields continues to improve the development of personalized medicine strategies for rare and complex cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44e6c2844fd1beb610939541608801e2d5fb55e3" target='_blank'>
              Advancing Precision Medicine: The Role of Genetic Testing and Sequencing Technologies in Identifying Biological Markers for Rare Cancers
              </a>
            </td>
          <td>
            Joviana Farhat, Lara Alzyoud, M. Alwahsh, A. Acharjee, B. Al-Omari
          </td>
          <td>2025-04-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e945cd17bb6878accfa941b57523d14ed0d229c" target='_blank'>
              Genomic and transcriptomic sequencing in prostate cancer.
              </a>
            </td>
          <td>
            Safiullah Rifai, Azimullah Rifai, Xiaolei Shi, M. A. Khan, Wei Guang, Linbo Wang, Luke Tallon, Arif Hussain
          </td>
          <td>2025-02-28</td>
          <td>Current opinion in oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Hematologic malignancies (HMs) encompass a diverse spectrum of cancers originating from the blood, bone marrow, and lymphatic systems, with myeloid malignancies representing a significant and complex subset. This review provides a focused analysis of their classification, prevalence, and incidence, highlighting the persistent challenges posed by their intricate genetic and epigenetic landscapes in clinical diagnostics and therapeutics. The genetic basis of myeloid malignancies, including chromosomal translocations, somatic mutations, and copy number variations, is examined in detail, alongside epigenetic modifications with a specific emphasis on DNA methylation. We explore the dynamic interplay between genetic and epigenetic factors, demonstrating how these mechanisms collectively shape disease progression, therapeutic resistance, and clinical outcomes. Advances in diagnostic modalities, particularly those integrating epigenomic insights, are revolutionizing the precision diagnosis of HMs. Key approaches such as nano-based contrast agents, optical imaging, flow cytometry, circulating tumor DNA analysis, and somatic mutation testing are discussed, with particular attention to the transformative role of machine learning in epigenetic data analysis. DNA methylation episignatures have emerged as a pivotal tool, enabling the development of highly sensitive and specific diagnostic and prognostic assays that are now being adopted in clinical practice. We also review the impact of computational advancements and data integration in refining diagnostic and therapeutic strategies. By combining genomic and epigenomic profiling techniques, these innovations are accelerating biomarker discovery and clinical translation, with applications in precision oncology becoming increasingly evident. Comprehensive genomic datasets, coupled with artificial intelligence, are driving actionable insights into the biology of myeloid malignancies and facilitating the optimization of patient management strategies. Finally, this review emphasizes the translational potential of these advancements, focusing on their tangible benefits for patient care and outcomes. By synthesizing current knowledge and recent innovations, we underscore the critical role of precision medicine and epigenomic research in transforming the diagnosis and treatment of myeloid malignancies, setting the stage for ongoing advancements and broader clinical implementation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1633c8ceb2748c3ad6ca551f7099ac2f4cee0994" target='_blank'>
              Epigenomic insights and computational advances in hematologic malignancies
              </a>
            </td>
          <td>
            Carolyn Lauzon-Young, Ananília Silva, B. Sadikovic
          </td>
          <td>2025-04-12</td>
          <td>Molecular Cytogenetics</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/006d2b04fc1884e4bfd784cf684ee4d760f680ad" target='_blank'>
              Exploring the tumor suppressor role of RIN1 in familial thyroid carcinoma.
              </a>
            </td>
          <td>
            Luna Picello, Mattia Dalle Nogare, Daniele Puggina, Cecilia Salvoro, G. Pennelli, G. Gaudenzi, S. Carra, Monica Oldani, Davide Gentilini, Laura Fugazzola, A. Faggiano, Giovanni Vitale, Gianluca Occhi, Giovanni Vazza
          </td>
          <td>2025-03-01</td>
          <td>Endocrine-related cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="ABSTRACT The transcription factor p53 is the most frequently impaired tumor suppressor in human cancers. In response to various stress stimuli, p53 activates transcription of genes that mediate its tumor-suppressive functions. Distinctive characteristics of p53 outlined here enable a well-defined program of genes involved in cell cycle arrest, apoptosis, senescence, differentiation, metabolism, autophagy, DNA repair, anti-viral response, and anti-metastatic functions, as well as facilitating autoregulation within the p53 network. This versatile, anti-cancer network governed chiefly by a single protein represents an immense opportunity for targeted cancer treatment, since about half of human tumors retain unmutated p53. During the last two decades, numerous compounds have been developed to block the interaction of p53 with the main negative regulator MDM2. However, small molecule inhibitors of MDM2 only induce a therapeutically desirable apoptotic response in a limited number of cancer types. Moreover, clinical trials of the MDM2 inhibitors as monotherapies have not met expectations and have revealed hematological toxicity as a characteristic adverse effect across this drug class. Currently, combination treatments are the leading strategy for enhancing efficacy and reducing adverse effects of MDM2 inhibitors. This review summarizes efforts to identify and test therapeutics that work synergistically with MDM2 inhibitors. Two main types of drugs have emerged among compounds used in the following combination treatments: first, modulators of the p53-regulated transcriptome (including chromatin modifiers), translatome, and proteome, and second, drugs targeting the downstream pathways such as apoptosis, cell cycle arrest, DNA repair, metabolic stress response, immune response, ferroptosis, and growth factor signaling. Here, we review the current literature in this field, while also highlighting overarching principles that could guide target selection in future combination treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3895fe8ca5319763195e6c375cf525ba1915bfa" target='_blank'>
              Harnessing p53 for targeted cancer therapy: new advances and future directions
              </a>
            </td>
          <td>
            Z. Andrysik, Joaquin M Espinosa
          </td>
          <td>2025-03-03</td>
          <td>Transcription</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="The hierarchical organization of the eukaryotic genome is crucial for nuclear activities and cellular development. Genetic aberrations can disrupt this 3D genomic architecture, potentially driving oncogenesis. However, current research often lacks a comprehensive perspective, focusing on specific mutation types and singular 3D structural levels. Here, pathological changes from chromosomes to nucleotides are systematically cataloged, including 10 789 interchromosomal translocations (ICTs), 18 863 structural variants (SVs), and 162 769 single nucleotide polymorphisms (SNPs). The multilayered analysis reveals that fewer than 10% of ICTs disrupt territories via potent 3D interactions, and only a minimal fraction of SVs disrupt compartments or intersect topologically associated domain structures, yet these events significantly influence gene expression. Pathogenic SNPs typically show reduced interactions within the 3D genomic space. To investigate the effects of variants in the context of 3D organization, a two-phase scoring algorithm, 3DFunc, is developed to evaluate the pathogenicity of variant-gene pairs in cancer. Using 3DFunc, IGHV3-23's critical role in chronic lymphocytic leukemia is identified and it is found that three pathological SNPs (rs6605578, rs7814783, rs2738144) interact with DEFA3. Additionally, 3DGAtlas is introduced, which provides a highly accessible 3D genome atlas and a valuable resource for exploring the pathological effects of genetic mutations in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/daa65f99b6475aca863361e2ed2a67fe83692271" target='_blank'>
              A 3D Genome Atlas of Genetic Variants and Their Pathological Effects in Cancer.
              </a>
            </td>
          <td>
            Li Tang, M. C. Hill, Mingxing He, Junhao Chen, Zirui Wang, Patrick T. Ellinor, Min Li
          </td>
          <td>2025-03-25</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a306f36a45e59527afe8dc83f82d2317d23a26f" target='_blank'>
              Abstract A035: Functional profiling of chimeric RNA transcripts identifies new biomarkers and therapeutic targets for precision medicine in cancer
              </a>
            </td>
          <td>
            Samir Lalani, Mason Ingram, Sehajroop Gadh, Hui Li
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Gene network models provide a foundation for graph theory approaches, aiding in the novel discovery of drug targets, disease genes, and genetic mechanisms for various biological functions. Disease genetics must be interpreted within the cellular context of disease-associated cell types, which cannot be achieved with datasets consisting solely of organism-level samples. Single-cell RNA sequencing (scRNA-seq) technology allows computational distinction of cell states which provides a unique opportunity to understand cellular biology that drives disease processes. Importantly, the abundance of cell samples with their transcriptome-wide profile allows the modeling of systemic cell-type-specific gene networks (CGNs), offering insights into gene-cell-disease relationships. In this review, we present reference-based and de novo inference of gene functional interaction networks that we have recently developed using scRNA-seq datasets. We also introduce a compendium of CGNs as a useful resource for cell-type-resolved disease genetics. By leveraging these advances, we envision single-cell network biology as the key approach for mapping the gene-cell-disease axis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9aa7b9ecda0e5aabd9c3870f99cf01184de548a3" target='_blank'>
              Single-cell network biology enabling cell-type-resolved disease genetics
              </a>
            </td>
          <td>
            Junha Cha, Insuk Lee
          </td>
          <td>2025-03-27</td>
          <td>Genomics & Informatics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Simple Summary Cancer arises as an abnormality in genomic control, either due to the activation of oncogenes or loss of functioning tumor-suppressor genes. Oncogenes result in an uncontrolled increase in cell proliferation; tumor-suppressor genes act as a brake, leading the cell cycle to stop and promote cell death when necessary. This review seeks to tease out the complex interactions between these genes, throwing light on key signaling pathways such as MAPK/ERK, PI3K/AKT/mTOR, and Wnt/β-catenin, all of which play a role in the progression of the tumor. We discuss major genetic theories, especially Knudson’s two-hit hypothesis, and how modern technologies like next-generation sequencing have advanced our understanding of cancer genetics. We also underscore the most current therapeutic approaches involving oncogenes and the reactivation of tumor-suppressor gene function, with challenges including drug resistance and heterogeneity of the tumor. In this regard, understanding the balance between oncogenes and tumor-suppressor genes is critical in developing effective and personalized cancer treatment, thereby bettering patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b78e398fe4233b0d93ca9c44f0387aa4d32d4ab" target='_blank'>
              Exploring the Genetic Orchestra of Cancer: The Interplay Between Oncogenes and Tumor-Suppressor Genes
              </a>
            </td>
          <td>
            Sajal Raj Singh, Rakesh Bhaskar, Shampa Ghosh, Bhuvaneshwar Yarlagadda, Krishna Kumar Singh, Prashant Verma, Sonali Sengupta, M. Mladenov, N. Hadzi-Petrushev, R. Stojchevski, J. K. Sinha, Dimiter B Avtanski
          </td>
          <td>2025-03-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The cellular programs that mediate therapy resistance are often important drivers of metastasis, a phenomenon that needs to be understood better to improve screening and treatment options for cancer patients. Although this issue has been studied extensively for chemotherapy, less is known about a causal link between resistance to radiation therapy and metastasis. We investigated this problem in triple-negative breast cancer (TNBC) and established that radiation resistant tumor cells have enhanced metastatic capacity, especially to bone. Resistance to radiation increases the expression of integrin β3 (ITGβ3), which promotes enhanced migration and invasion. Bioinformatic analysis and subsequent experimentation revealed an enrichment of RNA metabolism pathways that stabilize ITGβ3 transcripts. Specifically, the RNA binding protein heterogenous nuclear ribonucleoprotein L (HNRNPL), whose expression is regulated by Nrf2, mediates the formation of circular RNAs (circRNAs) that function as competing endogenous RNAs (ceRNAs) for the family of let-7 microRNAs that target ITGβ3. Collectively, our findings identify a novel mechanism of radiation-induced metastasis that is driven by alterations in RNA metabolism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14903a27d305650e2051ca5d1898fdc092718345" target='_blank'>
              Resistance to Radiation Enhances Metastasis by Altering RNA Metabolism
              </a>
            </td>
          <td>
            Ayush Kumar, Kensei Kishimoto, H. Goel, C. Wisniewski, Rui Li, Brendan Pacheco, L. J. Zhu, William A. Flavahan, A. Mercurio
          </td>
          <td>2025-02-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Pediatric leukemia is one of the most prevalent childhood malignancies, and its pathogenesis involves complex molecular and genetic mechanisms. Among the critical players in leukemogenesis, the Janus kinase 2 (JAK2) gene has garnered significant attention due to its role in aberrant signal transduction and hematopoietic dysregulation. Mutations such as JAK2 V617F, while more commonly associated with myeloproliferative neoplasms, have also been implicated in pediatric leukemia subtypes, including acute lymphoblastic leukemia (ALL) and acute myeloid leukemia (AML). These mutations result in constitutive activation of the JAK-STAT pathway, promoting unchecked cell proliferation, survival, and resistance to apoptosis. The dysregulation of the JAK2 signaling pathway not only drives malignant transformation but also facilitates interactions with the bone marrow microenvironment, creating a niche that supports leukemia cell survival and immune evasion. Therapeutic advancements have focused on JAK2 inhibitors such as ruxolitinib and fedratinib, which show promise in preclinical and early clinical settings. However, challenges such as drug resistance and off-target effects limit their efficacy, necessitating the exploration of combination therapies and novel drug formulations. Current strategies include combining JAK2 inhibitors with chemotherapy, immune checkpoint inhibitors, or epigenetic modulators to achieve synergistic effects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b8c5bd5328b2c81b8552a0d00878e9d6a8e64d6" target='_blank'>
              JAK2 in pediatric leukemia: mechanisms of pathogenesis and drug development- a narrative review
              </a>
            </td>
          <td>
            E. Obeagu
          </td>
          <td>2025-03-18</td>
          <td>Annals of Medicine &amp; Surgery</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acd56a27bbf9de38ce63e14692f7247323ea5220" target='_blank'>
              Recent advances in targeted degradation in the RAS pathway.
              </a>
            </td>
          <td>
            Zhiming Ge, Zi-sheng Fan, Wei He, Guizhen Zhou, Yidi Zhou, Mingyue Zheng, Sulin Zhang
          </td>
          <td>2025-03-10</td>
          <td>Future medicinal chemistry</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background The ataxia-telangiectasia mutated (ATM) kinase phosphorylates and activates several downstream targets that are essential for DNA damage repair, cell cycle inhibition and apoptosis. Germline biallelic inactivation of the ATM gene causes ataxia-telangiectasia (A-T), and heterozygous pathogenic variant (PV) carriers are at increased risk of cancer, notably breast cancer. This study aimed to investigate whether DNA methylation profiling can be useful as a biomarker to identify tumors arising in ATM PV carriers, which may help for the management and optimal tailoring of therapies of these patients. Methods Breast tumor enriched DNA was prepared from 2 A-T patients, 27 patients carrying an ATM PV, 6 patients carrying a variant of uncertain clinical significance and 484 noncarriers enrolled in epidemiological studies conducted in France and Australia to investigate genetic and nongenetic factors involved in breast cancer susceptibility. Genome-wide DNA methylation analysis was performed using the Illumina Infinium HumanMethylation EPIC and 450K BeadChips. Correlation between promoter methylation and gene expression was assessed for 10 tumors for which transcriptomic data were available. Results We found that the ATM promoter was hypermethylated in 62% of tumors of heterozygous PV carriers compared to the mean methylation level of ATM promoter in tumors of noncarriers. Gene set enrichment analyses identified 47 biological pathways enriched in hypermethylated genes involved in neoplastic, neurodegenerative and metabolic-related pathways in tumor of PV carriers. Among the 327 differentially methylated promoters, promoters of ARHGAP40, SCGB3A1 (HIN-1), and CYBRD1 (DCYTB) were hypermethylated and associated with a lower gene expression in these tumors. Moreover, using three different deep learning algorithms (logistic regression, random forest and XGBoost), we identified a set of 27 additional biomarkers predictive of ATM status, which could be used in the future to provide evidence for or against pathogenicity in ATM variant classification strategies. Conclusions We showed that breast tumors that arise in women who carry an ATM PV display a specific genome-wide DNA methylation profile. Specifically, the methylation pattern of 27 key gene promoters was predictive of ATM PV status of the women. These genes may also represent new medical prevention and therapeutic targets for these women. Supplementary Information The online version contains supplementary material available at 10.1186/s13058-025-01988-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bebfd1b3f4c2e2d684ea0f216e88d7bab9a7fec5" target='_blank'>
              Breast tumors from ATM pathogenic variant carriers display a specific genome-wide DNA methylation profile
              </a>
            </td>
          <td>
            Nicolas M Viart, Anne-Laure Renault, Séverine Eon-Marchais, Yue Jiao, L. Fuhrmann, Sophia Murat El Houdigui, Dorothée Le Gal, Eve Cavaciuti, Marie-Gabrielle Dondon, Juana Beauvallet, Virginie Raynal, D. Stoppa-Lyonnet, A. Vincent-Salomon, Nadine Andrieu, M. Southey, F. Lesueur
          </td>
          <td>2025-03-11</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>118</td>
        </tr>

        <tr id="Pancreatic Ductal Adenocarcinoma (PDAC) poses a significant health threat characterized by poor clinical outcomes, largely attributable to late detection, chemotherapy resistance, and the absence of tailored therapies. Despite progress in surgical, radiation, and chemotherapy treatments, 80% of PDAC patients do not benefit optimally from systemic therapy, often due to asymptomatic presentation or disease regression upon diagnosis. The disease’s progression is influenced by complex interactions involving immunological, genetic, and environmental factors, among others. However, the precise molecular mechanisms underlying PDAC remain incompletely understood. A major challenge in elucidating PDAC’s origins lies in deciphering the genetic variations governing its network. PDAC exhibits heterogeneity, manifesting diverse genetic compositions, cellular attributes, and behaviors across patients and within tumors. This diversity complicates diagnosis, treatment strategies, and prognostication. Identification of “Differentially Expressed Genes” (DEGs) between PDAC and healthy controls is vital for addressing these challenges. These DEGs serve as the foundation for constructing the PDAC protein interaction network, with their network properties being assessed for further insights. Our analysis revealed five key hub genes (KHGs): EGF, SRC, SDC1, ICAM1 and CEACAM5. The KHGs were predominantly enriched in pathways such as: ErbB signaling pathway, Rap1 signaling pathway, etc. Acknowledging the therapeutic promise and biomarker importance of PDAC KHGs, we have also pinpointed approved medications for the identified key genes. Nevertheless, it is crucial to conduct experimental validation on KHGs to confirm their effectiveness within the PDAC context. Overall, this study identified potential key hub genes implicated in the progression of PDAC, offering significant guidance for personalized clinical decision-making and molecular-targeted therapy for PDAC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78a2c9cf06e25fb71d0969b6fd19506dac89644c" target='_blank'>
              Identification of key hub genes in pancreatic ductal adenocarcinoma: an integrative bioinformatics study
              </a>
            </td>
          <td>
            Kankana Bhattacharjee, Avik Sengupta, Rahul Kumar, Aryya Ghosh
          </td>
          <td>2025-03-28</td>
          <td>Frontiers in Bioinformatics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e83d84b8f1cfdb09485b80edd7f374acbe1eeac9" target='_blank'>
              Molecular mechanisms of Hippo pathway in tumorigenesis: therapeutic implications.
              </a>
            </td>
          <td>
            M. Saadh, Hanan Hassan Ahmed, Radhwan Abdul Kareem, A. K. Bishoyi, R. Roopashree, Debasish Shit, Renu Arya, Abhishek Sharma, Kakhramon Khaitov, Hayder Naji Sameer, Ahmed Yaseen, Zainab H. Athab, Mohaned Adil
          </td>
          <td>2025-02-27</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="It is well established that hematologic malignancies are often considerably radiosensitive, which enables usage of far lower doses of therapeutic radiotherapy. This review summarizes the currently known genomic landscape of hematologic malignancies, particularly as it relates to radiosensitivity and the field of radiation oncology. By tracing the historical development of the modern understanding of radiosensitivity, we focus on the discovery and implications of pivotal mutated genes in hematologic malignancies such as TP53, ATM, and other genes critical to DNA repair pathways. These genetic insights have contributed significantly to the advancement of personalized medicine, aiming to enhance treatment precision and outcomes, and there is an opportunity to extend these insights to personalized radiotherapy. We explore the transition from early discoveries to the current efforts in integrating comprehensive genomic data into clinical practice. Specific examples from Hodgkin lymphoma, non-Hodgkin lymphoma, and plasma cell neoplasms illustrate how genetic mutations could influence radiosensitivity and impact subsequent radiotherapeutic response. Despite the advancements, challenges remain in translating these genetic insights into routine clinical practice, particularly due to the heterogeneity of alterations and the complex interactions within cancer signaling pathways. We emphasize the potential of radiogenomics to address these challenges by identifying genetic markers that predict radiotherapy response and toxicity, thereby refining treatment strategies. The need for robust decision support systems, standardized protocols, and ongoing education for healthcare providers is critical to the successful integration of genomic data into radiation therapy. As research continues to validate genetic markers and explore novel therapeutic combinations, the promise of personalized radiotherapy becomes increasingly attainable, offering the potential to significantly improve outcomes for patients with hematologic malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e274aa44601f7df10924839e8b158bfac4f727a9" target='_blank'>
              The art of war: using genetic insights to understand and harness radiation sensitivity in hematologic malignancies
              </a>
            </td>
          <td>
            N. Wijetunga, J. Yahalom, B. Imber, Thomas FitzGerald, Lara Hilal, Ji Hyun Hong, Jonathan Leeman
          </td>
          <td>2025-03-21</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="Background The tumor suppressor protein, p53, which is mutated in half of human tumors, plays a critical role in cellular responses to DNA damage and maintenance of genome stability. Therefore, increasing our understanding of the p53 pathway is essential for improving cancer treatment and diagnosis. Methods This study, which aimed to identify genes and pathways that mediate resistance to p53 upregulation, used genome-wide CRISPR-Cas9 loss-of-function screening done with Nutlin-3a, which inhibits p53-MDM2 interaction, resulting in p53 accumulation and apoptotic cell death. We used bioinformatics analysis for the identification of genes and pathways that are involved in the p53 pathway and cell survival assays to validate specific genes. In addition, we used RNA-seq to identify differentially expressed p53 target genes in gene knockout (KO) cell lines. Results Our screen revealed three significantly enriched pathways: The heparan sulfate glycosaminoglycan biosynthesis, diphthamide biosynthesis and Hippo pathway. Notably, TRIP12 was significantly enriched in our screen. We found that TRIP12 is required for the p53-dependent transcription of several pro-apoptotic genes. Conclusion Our study has identified two novel pathways that play a role in p53-mediated growth restriction. Moreover, we have highlighted the interaction between the Hippo and the p53 pathways. Interestingly, we have shown that TRIP12 plays an important function in the p53 pathway by selectively affecting its role as a transcription factor. Supplementary Information The online version contains supplementary material available at 10.1186/s10020-025-01141-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61f2f128a395ad66b24bd497a265456014e21a60" target='_blank'>
              Genome-wide screening in human embryonic stem cells identifies genes and pathways involved in the p53 pathway
              </a>
            </td>
          <td>
            Amir Haddad, T. Golan‐lev, N. Benvenisty, Michal Goldberg
          </td>
          <td>2025-03-13</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="EGFR is a proto-oncogene that is mutationally activated in a variety of cancers. Small molecule inhibitors targeting EGFR can be effective in slowing the progression of disease, and in some settings these drugs even cause dramatic tumor regression. However, responses to EGFR inhibitors are rarely durable, and the mechanisms contributing to response variation remain unclear. In particular, several distinct mechanisms have been proposed for how EGFR inhibition activates cell death, and a consensus has yet to emerge. In this study, we use functional genomics with specialized analyses to infer how genetic perturbations effect the drug-induced death rate. Our data clarify that inhibition of PI3K signaling drives the lethality of EGFR inhibition. Inhibition of other pathways downstream of EGFR, including the RAS-MAPK pathway, promote growth suppression, but not the lethal effects of EGFR inhibitors. Taken together, our study reveals the first “reference map” for the genome-wide genetic dependencies of lethality for EGFR inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ecf934bb91c2c002402ace642323761657b953a" target='_blank'>
              Genome-wide profiling identifies the genetic dependencies of cell death following EGFR inhibition
              </a>
            </td>
          <td>
            Sydney A Porto, Gavin A. Birdsall, Nicholas W. Harper, Megan E. Honeywell, Michael J. Lee
          </td>
          <td>2025-04-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/459cd487cd2ba94650a35149058062f7d408b1cc" target='_blank'>
              Molecular Subtypes and Targeted Therapeutic Strategies in Small Cell Lung Cancer: Advances, Challenges, and Future Perspectives
              </a>
            </td>
          <td>
            Daoyuan Huang, Jingchao Wang, Li Chen, Weiwei Jiang, Hiroyuki Inuzuka, David K Simon, Wenyi Wei
          </td>
          <td>2025-04-12</td>
          <td>Molecules</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="To maintain cell fitness, deleterious genetic alterations are buffered by compensatory changes in additional genes. In cancer, buffering processes could be targeted by synthetic lethality. However, despite the large-scale identification of synthetic lethal effects in preclinical models, evidence that these operate clinically is limited. This impedes the application of synthetic lethal approaches. By integrating molecular profiling data from >9,000 cancers with synthetic lethal screens, we show that transcriptomic buffering of tumor suppressor gene (TSG) loss by hyperexpression of synthetic lethal partners is a common phenomenon, extending to multiple TSGs and histotypes. Transcriptomic buffering is also notable in cancers that phenocopy TSG loss, such as BRCAness cancers, where expression of BRCA1/2 synthetic lethal genes correlates with clinical outcome. Synthetic lethal genes that exhibit transcriptomic buffering also represent more robust synthetic lethal effects. These observations have implications for understanding how tumor cells tolerate TSG loss, in part explain transcriptomic architectures in cancer and provide insight into target selection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29a52e19b56a0e26f2474ec866322bdccd6f6854" target='_blank'>
              The transcriptomic architecture of common cancers reflects synthetic lethal interactions
              </a>
            </td>
          <td>
            S. Haider, R. Brough, Santiago Madera, Jacopo Iacovacci, Aditi Gulati, Andrew J. Wicks, J. Alexander, S. Pettitt, Andrew N J Tutt, C. J. Lord
          </td>
          <td>2025-03-03</td>
          <td>Nature Genetics</td>
          <td>1</td>
          <td>33</td>
        </tr>

        <tr id="Metabolic reprogramming is a hallmark of cancer, enabling tumor cells to adapt to and exploit their microenvironment for sustained growth. The liver is a common site of metastasis, but the interactions between tumor cells and hepatocytes remain poorly understood. In the context of liver metastasis, these interactions play a crucial role in promoting tumor survival and progression. This study leverages multiomics coverage of the microenvironment via liquid chromatography and high-resolution, high-mass-accuracy mass spectrometry-based untargeted metabolomics, 13C-stable isotope tracing, and RNA sequencing to uncover the metabolic impact of co-localized primary hepatocytes and a colon adenocarcinoma cell line, SW480, using a 2D co-culture model. Metabolic profiling revealed disrupted Warburg metabolism with an 80% decrease in glucose consumption and 94% decrease in lactate production by hepatocyte–SW480 co-cultures relative to SW480 control cultures. Decreased glucose consumption was coupled with alterations in glutamine and ketone body metabolism, suggesting a possible fuel switch upon co-culturing. Further, integrated multiomics analysis indicates that disruptions in metabolic pathways, including nucleoside biosynthesis, amino acids, and TCA cycle, correlate with altered SW480 transcriptional profiles and highlight the importance of redox homeostasis in tumor adaptation. Finally, these findings were replicated in three-dimensional microtissue organoids. Taken together, these studies support a bioinformatic approach to study metabolic crosstalk and discovery of potential therapeutic targets in preclinical models of the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d939bb36944c04a3083bae73bc1b5307c8fa9271" target='_blank'>
              Deciphering Colorectal Cancer–Hepatocyte Interactions: A Multiomics Platform for Interrogation of Metabolic Crosstalk in the Liver–Tumor Microenvironment
              </a>
            </td>
          <td>
            Alisa B. Nelson, Lyndsay E Reese, Elizabeth Rono, Eric D. Queathem, Yinjie Qiu, Braedan M. McCluskey, Alexandra Crampton, Eric Conniff, Katherine A Cummins, Ella Boytim, Senali Dansou, Justin H. Hwang, S. Safo, Patrycja Puchalska, David K. Wood, K. Schwertfeger, Peter A. Crawford
          </td>
          <td>2025-02-25</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a87c4098a87feebf01a217fad3de111af3e86862" target='_blank'>
              Abstract IA002: Functional omics for novel therapeutic combinations in myeloid malignancies
              </a>
            </td>
          <td>
            J. Tyner
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Somatic mobilization of LINE-1 (L1) has been implicated in cancer etiology. We analyzed a recent TCGA data release comprised of nearly 5000 pan-cancer paired tumor-normal whole-genome sequencing (WGS) samples and ~9000 tumor RNA samples. We developed TotalReCall an improved algorithm and pipeline for detection of L1 retrotransposition (RT), finding high correlation between L1 expression and “RT burden” per sample. Furthermore, we mathematically model the dual regulatory roles of p53, where mutations in TP53 disrupt regulation of both L1 expression and retrotransposition. We found those with Li-Fraumeni Syndrome (LFS) heritable TP53 pathogenic and likely pathogenic variants bear similarly high L1 activity compared to matched cancers from patients without LFS, suggesting this population be considered in attempts to target L1 therapeutically. Due to improved sensitivity, we detect over 10 genes beyond TP53 whose mutations correlate with L1, including ATRX, suggesting other, potentially targetable, mechanisms underlying L1 regulation in cancer remain to be discovered.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09c16a8e22a8e26a1832c9dc60f1f0a97f3c5163" target='_blank'>
              Pan-cancer multi-omic model of LINE-1 activity reveals locus heterogeneity of retrotransposition efficiency
              </a>
            </td>
          <td>
            Alexander Solovyov, Julie M. Behr, D. Hoyos, Eric Banks, Alexander W Drong, Bryan Thornlow, Jimmy Z Zhong, Enrique Garcia-Rivera, Wilson McKerrow, Chong Chu, Cédric Arisdakessian, Dennis M Zaller, J. Kamihara, Liyang Diao, M. Fromer, Benjamin D. Greenbaum
          </td>
          <td>2025-02-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Glioblastoma, the most common and aggressive primary malignant brain tumor, is characterized by a high rate of recurrence, disability, and lethality. Therefore, there is a pressing need to develop more effective prognostic biomarkers and treatment approaches for glioblastoma. Lactylation, an emerging form of protein post-translational modification, has been closely associated with lactate, a metabolite of glycolysis. Since the initial identification of lactylation sites in core histones in 2019, accumulating evidence has shown the critical role that lactylation plays in glioblastoma development, assessment of poor clinical prognosis, and immunosuppression, which provides a fresh angle for investigating the connection between metabolic reprogramming and epigenetic plasticity in glioblastoma cells. The objective of this paper is to present an overview of the metabolic and epigenetic roles of lactylation in the expanding field of glioblastoma research and explore the practical value of developing novel treatment plans combining targeted therapy and immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4de8e5b16e7caa0a9241085fd7820d5813bb6d8" target='_blank'>
              Lactylation in Glioblastoma: A Novel Epigenetic Modifier Bridging Epigenetic Plasticity and Metabolic Reprogramming
              </a>
            </td>
          <td>
            Qingya Qiu, Hui Deng, Ping Song, Yushu Liu, Mengxian Zhang
          </td>
          <td>2025-04-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Precision medicine has revolutionized the treatment of colorectal cancer by enabling a personalized approach tailored to each patient’s unique genetic characteristics. Genomic profiling allows for the identification of specific mutations in genes such as KRAS, BRAF, and PIK3CA, which play a crucial role in cell signaling pathways that regulate cell proliferation, apoptosis, and differentiation. This information enables doctors to select targeted therapies that inhibit specific molecular pathways, maximizing treatment effectiveness and minimizing side effects. Precision medicine also facilitates adaptive monitoring of tumor progression, allowing for adjustments in therapy to maintain treatment effectiveness. While challenges such as high costs, limited access to genomic technology, and the need for more representative genomic data for diverse populations remain, collaboration between researchers, medical practitioners, policymakers, and the pharmaceutical industry is crucial to ensure that precision medicine becomes a standard of care accessible to all. With continued advances and support, precision medicine has the potential to improve treatment outcomes, reduce morbidity and mortality rates, and enhance the quality of life for colorectal cancer patients worldwide.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3623ce8f60143b7348cbfaa508b5bbdc79c01b3a" target='_blank'>
              Precision medicine in colorectal cancer: genomics profiling and targeted treatment
              </a>
            </td>
          <td>
            Adi Muradi Muhar, A. J. Velaro, Arya Tjipta Prananda, S. E. Nugraha, Princella Halim, R. A. Syahputra
          </td>
          <td>2025-02-27</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Understanding the genetic basis of gene expression can shed light on the regulatory mechanisms underlying complex traits and diseases. Single-cell resolved measures of RNA levels and single-cell expression quantitative trait loci (sc-eQTLs) have revealed genetic regulation that drives sub-tissue cell states and types across diverse human tissues. Here, we describe the first phase of TenK10K, the largest-to-date dataset of matched whole-genome sequencing (WGS) and single-cell RNA-sequencing (scRNA-seq). We leverage scRNA-seq data from over 5 million cells across 28 immune cell types and matched WGS from 1,925 individuals. This provides power to detect associations between rare and low-frequency genetic variants that have largely been uncharacterised in their impact on cell-specific gene expression. We map the effects of both common and rare variants in a cell type specific manner using SAIGE-QTL. This newly developed method increases power by modelling single cells directly using a Poisson model rather than relying on aggregated 'pseudobulk' counts. We identify putative common regulatory variants for 83% of all 21,404 genes tested and cumulative rare variant signals for 47% of genes. We explore how genetic effects vary across cell type and state spectra, develop a framework to determine the degree to which sc-eQTLs are cell type specific, and show that about half of the effects are observed only in one or a few cell types. By integrating our results with functional annotations and disease information, we further characterise the likely molecular modes of action for many disease-associated variants. Finally, we explore the effects of genetic variants on gene expression across different cell states and functions, as well as effects that directly vary cell state abundance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69c58c57797887ff73f102df82f585e2cdd85a57" target='_blank'>
              Impact of Rare and Common Genetic Variation on Cell Type-Specific Gene Expression
              </a>
            </td>
          <td>
            A. Cuomo, Eleanor Spenceley, Hope A. Tanudisastro, Blake Bowen, Albert Henry, Hao Lawrence Huang, A. Xue, Wei Zhou, M. Welland, Arthur S. Lee, Kristof Wing, Owen Tang, Michael P. Gray, Michael Franklin, Michael Harper, Michael Silk, Katalina Bobowik, Alexander Stuckey, John Marshall, Vivian Bakiris, Caitlin Uren, B. Madala, Amy Miniter, Caitlin Bartie, Drew R Neavin, Zhen Qiao, Eyal Ben-David, Ling Chen, Kyle Kai-How, Farh, Stuart M. Grieve, T. Nguyen, Jennifer A. Piscionere, O. Siggs, Hannah R. Nicholas, Katrina M. de Lange, Alex H. Hewitt, G. A. Figtree, Daniel G. MacArthur, J. E. Powell
          </td>
          <td>2025-03-21</td>
          <td>None</td>
          <td>1</td>
          <td>35</td>
        </tr>

        <tr id="Human genome sequencing efforts in healthy and diseased individuals continue to identify a broad spectrum of genetic variants associated with predisposition, progression, and therapeutic outcomes for diseases like cancer1–6. Insights derived from these studies have significant potential to guide clinical diagnoses and treatment decisions; however, the relative importance and functional impact of most genetic variants remain poorly understood. Precision genome editing technologies like base and prime editing can be used to systematically engineer and interrogate diverse types of endogenous genetic variants in their native context7–9. We and others have recently developed and applied scalable sensor-based screening approaches to engineer and measure the phenotypes produced by thousands of endogenous mutations in vitro10–12. However, the impact of most genetic variants in the physiological in vivo setting, including contextual differences depending on the tissue or microenvironment, remains unexplored. Here, we integrate new cross-species base editing sensor libraries with syngeneic cancer mouse models to develop a multiplexed in vivo platform for systematic functional analysis of endogenous genetic variants in primary and disseminated malignancies. We used this platform to screen 13,840 guide RNAs designed to engineer 7,783 human cancer-associated mutations mapping to 489 endogenous protein-coding genes, allowing us to construct a rich compendium of putative functional interactions between genes, mutations, and physiological contexts. Our findings suggest that the physiological in vivo environment and cellular organotropism are important contextual determinants of specific gene-variant phenotypes. We also show that many mutations and their in vivo effects fail to be detected with standard CRISPR-Cas9 nuclease approaches and often produce discordant phenotypes, potentially due to site-specific amino acid selection- or separation-of-function mechanisms. This versatile platform could be deployed to investigate how genetic variation impacts diverse in vivo phenotypes associated with cancer and other genetic diseases, as well as identify new potential therapeutic avenues to treat human disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2f71f3527a2a35f380d7482a2555f7e6eeef4ab" target='_blank'>
              Multiplexed in vivo base editing identifies functional gene-variant-context interactions
              </a>
            </td>
          <td>
            Jonuelle Acosta, Grace A. Johnson, Samuel I. Gould, Kexin Dong, Yovel Lendner, Diego Detrés, Ondine Atwa, Jari Bulkens, Samuel Gruber, Manuel E. Contreras, Alexandra N. Wuest, Varun K. Narendra, Michael T. Hemann, Francisco J. Sánchez-Rivera
          </td>
          <td>2025-02-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38d11642e20fef657158a832ae45b3c22aa7b26f" target='_blank'>
              Abstract B025: Integrative clinical and molecular analysis of 665 next-generation in vitro cancer models generated by the the Human Cancer Models Initiative (HCMI) for advancing precision medicine and functional drug discovery
              </a>
            </td>
          <td>
            Dina Elharouni, Mushriq Al-Jazrawe, Seongmin Choi, Merve Dede, T. Hinoue, Sean A Misek, Heeju Noh, Luca Zanella, Moony Tseng, Hayley E. Francies, Priya Sridevi, Rachana Agarwal, Cindy W Kyi, Julyann Pérez-Mayoral, Megan J Stine, Eva Y. Tonsing-Carter, James M Clinton, The Hcmi Network, Peter W. Laird, Calvin J Kuo, O. Elemento, David L. Spector, A. Cherniack, Kyle Ellrott, M. Ferguson, R. Beroukhim, Katie Hoadley, N. Robine, Mathew Garnett, Andrea Califano, P. Spellman, D. Tuveson, Keith L. Ligon, Daniela S Gerhard, Louis M Staudt, Jesse S Boehm
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>148</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67a8da738511095a6b29a737fec7d84a93f6b5d7" target='_blank'>
              Bimodality in pan-cancer proteomics reveals new opportunities for biomarker discovery
              </a>
            </td>
          <td>
            Wen Jiang, Damian Bikiel, Jan Zaucha, Jixin Wang, Sanddhya Jayabalan, Yongjin Li, Matthew Sung, James Harper, Jimmy Zhao, K. Bulusu, Wenyan Zhong
          </td>
          <td>2025-03-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65af4b0d470be7b6d3b8d36aa5bfc0557940fe59" target='_blank'>
              The impact of long-read sequencing on human population-scale genomics.
              </a>
            </td>
          <td>
            Tobias Rausch, T. Marschall, Jan O. Korbel
          </td>
          <td>2025-04-01</td>
          <td>Genome research</td>
          <td>1</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f6c4d310ebc720bef67a3e653d0a6beb661d459" target='_blank'>
              Historical Evolution and CRISPR-Mediated Gene Editing Applications in NK Cell Therapy
              </a>
            </td>
          <td>
            Ziqing Yan
          </td>
          <td>2025-02-21</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Whole genome doubling (WGD) is a frequent event in cancer evolution associated with chromosomal instability, metastasis, and poor prognosis. While the genomic consequences of WGD are well documented, the effects of non-genetic alterations that accompany WGD, such as changes to cell and nuclear size, on tetraploid (4N) cancer cell physiology are less understood. Here, we show that cell and nuclear volume do not always scale with DNA content after WGD in cancer cells, resulting in 4N cells that differ in size. We find that small size is associated with enhanced cell fitness, mitotic fidelity, and tumorigenicity in 4N cancer cells and with poor patient survival in WGD-positive human cancers. Overall, these results suggest that cell and nuclear size contribute to the tumorigenic potential of 4N cancer cells and could be an important prognostic marker in human tumors that undergo WGD. Statement of Significance We report that WGD generates tetraploid cancer cells that vary in size, with larger cells displaying high chromosomal instability and smaller cells exhibiting high fitness and tumorigenicity. Furthermore, WGD status and cancer cell nuclear size in human tumors correlated with patient survival, demonstrating the clinical relevance of this association.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49f68f3dcdb4dd414fdeb5e81fa097ba1603de66" target='_blank'>
              Cell and nuclear size are associated with chromosomal instability and tumorigenicity in cancer cells that undergo whole genome doubling
              </a>
            </td>
          <td>
            Mathew Bloomfield, Sydney Huth, Daniella McCausland, Ron Saad, Nazia Bano, Tran Chau, Megan Sweet, N. Baudoin, Andrew McCaffrey, Kai Fluet, Eva M. Schmelz, Uri Ben-David, Daniela Cimini
          </td>
          <td>2025-04-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c2a9e1a9ae9c6282d886b43a42912fd5730bf65" target='_blank'>
              Revisiting the role of cancer-associated fibroblasts in tumor microenvironment
              </a>
            </td>
          <td>
            Xiaolei Lan, Wenyang Li, Kai Zhao, Jianpeng Wang, Shifang Li, Hai Zhao
          </td>
          <td>2025-04-17</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Pooled CRISPR screens are vital in the unbiased interrogation of gene function and are instrumental in uncovering therapeutic targets and biological processes. However, follow-up hit validation is critical to confirm observed results. Researchers need a simple and robust approach to rapidly verify putative hits and test resulting observations. Thus, we developed a CRISPR-based method for hit validation that tests the effect of a genetic perturbation on cell fitness. By editing target loci and monitoring the indel profiles over time, we have created a Cellular Fitness (CelFi) assay that can elucidate cellular vulnerabilities and verify hits from pooled CRISPR knockout screens. Unlike traditional cellular fitness assays that evaluate viability over time, the CelFi assay correlates changes in the indel profile at the target gene with a selective growth advantage or disadvantage in individual cells over time. Moreover, the CelFi assay can be utilized to evaluate gene dependencies and test new hypotheses, regardless of variations in single guide RNA optimization, ribonucleoprotein concentration, and gene copy number. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-96095-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7bfe3c00eb1f10374c9c0ac1481d5794cd3fb46b" target='_blank'>
              Rapid and robust validation of pooled CRISPR knockout screens using CelFi
              </a>
            </td>
          <td>
            Allister J Loughran, S. Narina, Jonathon Klein, Jamaica F Siwak, Jon P Connelly, S. Pruett-Miller
          </td>
          <td>2025-04-17</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Genetic diseases are serious health issues worldwide, caused by changes in gene structure due to mutations in single or multiple genes. Currently, no definitive treatments or medicines are available to cure these disorders. Gene therapy, a medical approach that utilizes genetic material, offers potential for preventing and treating diseases by addressing their underlying causes rather than relying on traditional methods like medication and surgery. This technique has shown promise in certain conditions by altering genetic material, enabling the body to produce desirable proteins or medications. However, gene therapy remains unavailable for most diseases and is still largely in the experimental stage for many conditions. This review investigates the current and future status of gene therapy not only for genetic disorders but also for life-threatening diseases such as cancer, cardiovascular disease (CVD), and diabetes. Additionally, the applications and limitations of gene therapy were evaluated. A review of recent research concluded that while rare or inherited genetic disorders could potentially be prevented or cured by precisely targeting their root causes, significant challenges remain. Rather than solely managing blood disorders, retinal diseases, and neuromuscular conditions, gene therapy can potentially eliminate these conditions. Furthermore, gene-based therapies offer expansive possibilities beyond rare genetic disorders, paving the way for broader therapeutic applications in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b52a192a46bd071e41e3a160b10b231a04116a32" target='_blank'>
              Advancement in Gene therapy: Current and Future of Gene Therapy
              </a>
            </td>
          <td>
            Shafee Ur Rehman
          </td>
          <td>2025-03-17</td>
          <td>Kashmir Journal of Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The alphabaculovirus Autographa californica multiple nucleopolyhedrovirus (AcMNPV) is the most commonly used virus in the Baculovirus Expression Vector System (BEVS) and has been utilized for the production of many human and veterinary biologics. AcMNPV has a large dsDNA genome that remains understudied, and relatively unmodified from the wild-type, especially considering how extensively utilized it is as an expression vector. Previously, our group utilized CRISPR-Cas9 genome engineering that revealed phenotypic changes when baculovirus genes are targeted using either co-expressed sgRNA or transfected sgRNA into a stable insect cell line that produced the Cas9 protein. Here, we describe a pipeline to sequence the recombinant AcMNPV expression vectors using shotgun sequencing, provide a set of primers for tiled-amplicon sequencing, show that untargeted baculovirus vector genomes remain relatively unchanged when amplified in Sf9-Cas9 cells, and confirm that AcMNPV gp64 gene disruption can minimize baculovirus contamination in cell cultures. Our findings provide a robust baseline for analyzing in process genome editing of baculoviruses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5cc3f1fc2cd30a998a1cee24f7f66a812c7add53" target='_blank'>
              Adapting Next-Generation Sequencing to in Process CRISPR-Cas9 Genome Editing of Recombinant AcMNPV Vectors: From Shotgun to Tiled-Amplicon Sequencing
              </a>
            </td>
          <td>
            Madhuja Chakraborty, Lisa Nielsen, Delaney Nash, Jozef I. Nissimov, Trevor C. Charles, M. Aucoin
          </td>
          <td>2025-03-01</td>
          <td>Viruses</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Epigenetics is currently considered the investigation of inheritable changes in gene expression that do not rely on DNA sequence alteration. Significant epigenetic procedures are involved, such as DNA methylations, histone modifications, and non-coding RNA actions. It is confirmed through several investigations that epigenetic changes are associated with the formation, development, and metastasis of various cancers, such as colorectal cancer (CRC). The difference between epigenetic changes and genetic mutations is that the former could be reversed or prevented; therefore, cancer treatment and prevention could be achieved by restoring abnormal epigenetic events within the neoplastic cells. These treatments, consequently, cause the anti-tumour effects augmentation, drug resistance reduction, and host immune response stimulation. In this article, we begin our survey by exploring basic epigenetic mechanisms to understand epigenetic tools and strategies for treating colorectal cancer in monotherapy and combination with chemotherapy or immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/719dc00daf58fbc383533083eb81aecec4828ea4" target='_blank'>
              Epigenetic marvels: exploring the landscape of colorectal cancer treatment through cutting-edge epigenetic-based drug strategies
              </a>
            </td>
          <td>
            A. Tahghighi, Effat Seyedhashemi, Javad Mohammadi, Arash Moradi, Aria Esmaeili, Majid Pornour, Kimia Jafarifar, S. M. Ganji
          </td>
          <td>2025-02-22</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f39bd0ddc99a69058c299e72e0d1b4296584ccd0" target='_blank'>
              Therapeutic targeting of chromatin alterations in leukemia and solid tumors.
              </a>
            </td>
          <td>
            Florian Perner, Tobias Berg, D. Sasca, Sophie-Luise Mersiowsky, Jayant Y. Gadrey, Johanna Thomas, Michael W M Kühn, Michael Lübbert
          </td>
          <td>2025-04-03</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b338d775526840d18d525abb3d293c825f24385" target='_blank'>
              Abstract B011: Harnessing epigenetic and metabolic biomarkers for targeted immunotherapy in colorectal cancer: Advances in precision medicine for solid tumors
              </a>
            </td>
          <td>
            Ji Min Lee
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17971509b5a82b8103a0ae0d9b891a0480387fd5" target='_blank'>
              Multiplexed epigenetic memory editing using CRISPRoff sensitizes glioblastoma to chemotherapy.
              </a>
            </td>
          <td>
            K. Lin, C. Zou, Akane Hubbard, Sasha Sengelmann, L. Goudy, I-Ching Wang, Rohit Sharma, J. Pak, K. Foster, T. Ozawa, J. D. de Groot, Joanna J. Phillips, H. Vasudevan, D. Raleigh, Alexander Marson, Niren Murthy, Luke A. Gilbert, Mitchel S Berger, S. J. Liu
          </td>
          <td>2025-02-25</td>
          <td>Neuro-oncology</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="Objectives/Goals: This study aims to explore transcriptional adaptation, where mutations in one gene trigger compensatory changes in related genes, and how this affects the variability in clinical manifestations of ALS. Our findings will provide insights for therapeutic strategies, while we also use gene editing to investigate correcting variants in ALS patients. Methods/Study Population: The presence of a premature termination codon triggers transcriptional activation. Therefore, we utilized CRISPR-Cas9 tool to generate a premature termination codon in CHCHD10 gene in multiple types of cells, including induced pluripotent stem cells derived from patient samples with known CHCHD10 mutations causative for Amyotrophic lateral sclerosis. CRISPR-Cas9 tool was delivered via ribonucleoprotein electroporation and transfected cell’s DNA was sequenced to validate gene editing. To confirm transcriptional adaption, changes in levels of protein and gene expression will be measured via immunoblot and quantification of CHCHD10 and CHCHCD2 from whole cell lysates of the edited cells. Results/Anticipated Results: Utilizing CRISPR-Cas9 tools, we anticipate that CHCHD2 gene can functionally compensate for the loss of function in the CHCHD10 locus through transcriptional adaptation. Additionally, employing single-stranded oligodeoxynucleotides (ssODNs) we aim to accurately correct the genetic aberrations in ALS patient cells, and study the pathomechanisms of aberrant CHCHD10. Discussion/Significance of Impact: The significance of this research lies in its potential to uncover transcriptional adaptation in humans, which could explain why patients with the same genetic variant experience different symptoms. By understanding this mechanism, we could pave the way for novel therapies, especially for CHCHD10-associated ALS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60e325e0bdd2a54149390c949c2bbd9ef99b9c77" target='_blank'>
              482 Translational gene editing strategies for understanding and treating CHCHD10-associated amyotrophic lateral sclerosis (ALS)
              </a>
            </td>
          <td>
            A. Gomez, Nathan P. Staff, S. Ekker
          </td>
          <td>2025-03-26</td>
          <td>Journal of Clinical and Translational Science</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72d25552df8222f9a3aedbc3a952a7bd4c3cfcc2" target='_blank'>
              Tracing genomic instability in induced mesenchymal stromal cell manufacture: an integration-free transfection approach.
              </a>
            </td>
          <td>
            Jong-Mi Lee, C. Lee, Binna Seol, Chan Kwon Jung, Yonggoo Kim, Dain Kang, Haein Yu, Yuna Hong, Cho Lok Song, Y. Cho, Myungshin Kim
          </td>
          <td>2025-04-14</td>
          <td>Experimental & molecular medicine</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Malignant pleural mesothelioma (MPM) is a rare yet highly aggressive malignancy with a severe prognosis. Compounded by the lack of effective treatment modalities, MPM remains a formidable health challenge. Therefore, the identification of actionable liabilities is critical for advancing precision medicine to combat this lethal disease. Here, we exploit an unbiased genome-wide CRISPR screen, integrating and cross-comparing three MPM cell lines with nonmalignant mesothelial cells, to selectively map the gene targets whose depletion indicates a common dependency in MPM cells. This systematic approach unveils a cohort of verifiable genes, among which BUB1, a mitotic checkpoint serine/threonine kinase, emerges as a high-confidence hit in cancer cells. Cellular and molecular studies demonstrate that genetic depletion or pharmacological inhibition of BUB1 profoundly impairs MPM cell survival and growth while inducing G2/M cell cycle arrest, cellular senescence, and apoptosis, and attenuating functional hallmarks of aggressive cancer cells. Transcriptomic profiling of BUB1-depleted cells discloses differential gene expression signatures congruent with cell fate phenotypes, including the reprogramming of mitotic network genes. Mechanistically, BUB1 is indispensable for the proper localization of essential mitotic regulators MAD1, MAD2, and Shugoshin (SGO1), thereby ensuring the functionality of the spindle assembly checkpoint (SAC). Furthermore, BUB1 ablation leads to cytokinesis failure and multinucleation, a phenotype characterized by the downregulation of CDC20, Cyclin A, and Cyclin B, and a reciprocal upregulation of the cyclin-dependent kinase inhibitor p21. Clinically, MPM tumors exhibit elevated levels of BUB1, and high BUB1 expression is associated with shorter patient survival. Our novel findings accentuate comparative CRISPR screens as a powerful platform to explore tumor cell-selective gene essentiality and propose BUB1 kinase as a potential marker and druggable vulnerability with therapeutic implications for MPM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/147c280cc9a37e6a39ce87d2c82cecf0b12eda26" target='_blank'>
              Genome-wide CRISPR screen identifies BUB1 kinase as a druggable vulnerability in malignant pleural mesothelioma
              </a>
            </td>
          <td>
            E. Cakiroglu, Sude Eris, Ozden Oz, G. Karakülah, Serif Senturk
          </td>
          <td>2025-04-03</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/632a6b07fed31db9bd0bc2462cc1bb896521dfbb" target='_blank'>
              Timing and clustering co-occurring genome amplifications in cancers
              </a>
            </td>
          <td>
            Sara Cocomello, Giovanni Santacatterina, R. Bergamin, Alice Antonello, G. Caravagna
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6bfccf3e169cfc64e8f5a384718db0d5b7282e59" target='_blank'>
              Abstract B092: Mechanisms of PARP7 inhibition-induced tumor cell death: A novel therapeutic for non-small cell lung cancer with tumor-intrinsic and immune-dependent pathways
              </a>
            </td>
          <td>
            Nalani Coleman, Ji-Ann Lee, C. Dumitras, Gauri Gusain, D. Conklin, Michael Palazzolo, Bitta Kahangi, Austin K. Rennels, J. Abascal, William P Crosson, M. Oh, E. P. Reyes, Carlos Botero, Hong Jiang, Alvaro Chumpitaz Lavalle, Shahed Tappuni, Emily Melik Aslanian, Jie Deng, R. Salehi-Rad, L. Tran, K. Krysan, Bin Liu, Steven M Dubinett
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Oncogenic gene fusions occur across a broad range of cancers and are a defining feature of some cancer types. Cancers driven by gene fusion products tend to respond well to targeted therapies, where available; thus, detection of potentially targetable oncogenic fusions is necessary to select optimal treatment. Detection methods include non-sequencing methods, such as fluorescence in situ hybridization and immunohistochemistry, and sequencing methods, such as DNA- and RNA-based next-generation sequencing (NGS). While NGS is an efficient way to analyze multiple genes of interest at once, economic and technical factors may preclude its use in routine care globally, despite several guideline recommendations. The aim of this review is to present a summary of oncogenic gene fusions, with a focus on fusions that affect tyrosine kinase signaling, and to highlight the importance of testing for oncogenic fusions. We present an overview of the identification of oncogenic gene fusions and therapies approved for the treatment of cancers harboring gene fusions, and summarize data regarding treating fusion-positive cancers with no current targeted therapies and clinical studies of fusion-positive cancers. Although treatment options may be limited for patients with rare alterations, healthcare professionals should identify patients most likely to benefit from oncogenic gene fusion testing and initiate the appropriate targeted therapy to achieve optimal treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46a05e8c035d2b5c9d05b3addb1c0208226f39c0" target='_blank'>
              Oncogenic gene fusions in cancer: from biology to therapy
              </a>
            </td>
          <td>
            Stephen V Liu, Misako Nagasaka, Judith Atz, Flavio Solca, Leonhard Müllauer
          </td>
          <td>2025-04-14</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0242c9047b84d4fc0e48d364c67e5fbd69f2f6c" target='_blank'>
              Genetics of Retinoblastoma - An Update.
              </a>
            </td>
          <td>
            Adwaita Nag, V. Khetan
          </td>
          <td>2025-04-16</td>
          <td>Seminars in ophthalmology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a279852f8f3e4dea6bf2c322ab3c789c81b3e6c" target='_blank'>
              How cancer emerges: Data-driven universal insights into tumorigenesis via hallmark networks
              </a>
            </td>
          <td>
            Jiahe Wang, Yan Wu, Yuke Hou, Yang Li, Dachuan Xu, Changjing Zhuge, Yue Han
          </td>
          <td>2025-02-27</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8599e8314fc9026e2db49b39f76e5203a6d7080" target='_blank'>
              Allele-specific genomics decodes gene targets and mechanisms of the non-coding genome
              </a>
            </td>
          <td>
            Tim P. Hasenbein, Sarah Hoelzl, S. Engelhardt, Daniel Andergassen
          </td>
          <td>2025-03-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0460f2bfed152b0841d1d7ccdc52c08315edba3" target='_blank'>
              In vivo CRISPR screening reveals epigenetic regulators of hepatobiliary plasticity.
              </a>
            </td>
          <td>
            Jonathan H Sussman, Hector Cure, Salina Yuan, Kenji Ito, Irfan A Asangani, Benjamin A Garcia, Ben Z. Stanger, Takeshi Katsuda
          </td>
          <td>2025-04-01</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="The use of CRISPR to knockdown or knockout genes is a powerful tool for understanding the specific role of a gene in disease development. However, it can cause many unanticipated changes to the transcriptome that are not detected by DNA amplification and Sanger sequencing of the target site. Various RNA-sequencing techniques can be used to identify these changes and effectively gauge the full impact of the CRISPR knockout, thereby providing a means of selecting appropriate clones for further experimentation. Background/Objectives: RNA-seq data from 4 CRISPR knockout experiments were analyzed and techniques developed to both confirm the success of the CRISPR modifications and identify potential issues. Methods: A broad-based analysis of RNA-sequencing data identified many CRISPR-based changes not identified by PCR amplification of DNA around the CRISPR target site. These changes included an inter-chromosomal fusion event, exon skipping, chromosomal truncation, and the unintentional transcriptional modification and amplification of a neighboring gene. Conclusions: The inadvertent modifications identified by the evaluation of 4 CRISPR experiments highlight the value of using RNA-seq to identify transcriptional changes to cells altered by CRISPR, many of which cannot be recognized by evaluating DNA alone. Specific guidelines are presented for designing and analyzing CRISPR experiments using RNA-seq data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/220a9a7ebf2665e1a3e4bd65d2ca79919ea760ce" target='_blank'>
              Techniques for Validating CRISPR Changes Using RNA-Sequencing Data
              </a>
            </td>
          <td>
            S. Rathe, Tracy A. Marko, Elizabeth N. Edwards, Paige Hazelton Ridder, Jyotika Varshney, Kyle B. Williams, James E. Johnson, B. Moriarity, D. A. Largaespada
          </td>
          <td>2025-03-24</td>
          <td>Genes</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Abstract Genes and genetic variants associated with human disease are continually being discovered, but validating their causative roles and mechanisms remains a significant challenge. CRISPR/Cas9 genome editing in model organisms like zebrafish can enable phenotypic characterization of founder generation (F0) knockouts (Crispants), but existing approaches are not amenable to high-throughput genetic screening due to high variability, cost, and low phenotype penetrance. To overcome these challenges, here we provide guide RNA (gRNA) selection rules that enable high phenotypic penetrance of up to three simultaneous knockouts in F0 animals following injection of 1–2 gRNAs per gene. We demonstrate a strong transcriptomic overlap in our F0 knockouts and stable knockout lines that take several months to generate. We systematically evaluated this approach across 324 gRNAs targeting 125 genes and demonstrated its utility in studying epistasis, characterizing paralogous genes, and validating human disease gene phenotypes across multiple tissues. Applying our approach in a high-throughput manner, we screened and identified 10 novel neurodevelopmental disorders and 50 hearing genes not previously studied in zebrafish. Altogether, our approach achieves high phenotypic penetrance using low numbers of gRNAs per gene in F0 zebrafish, offering a robust pipeline for rapidly characterizing candidate human disease genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d442e32931385bdcd94b5a33eaea8e6b64c86c8" target='_blank'>
              Optimizing gRNA selection for high-penetrance F0 CRISPR screening for interrogating disease gene function
              </a>
            </td>
          <td>
            Sheng-Jia Lin, Kevin Huang, C. Petree, Wei Qin, Pratishtha Varshney, Gaurav K Varshney
          </td>
          <td>2025-02-27</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa86b709f959a0cab24a9ee68d51374b7177f4bf" target='_blank'>
              Germline-Somatic Interactions in BRCA-Associated Cancers: Unique Molecular Profiles and Clinical Outcomes Linking ATM to TP53 Synthetic Essentiality.
              </a>
            </td>
          <td>
            Ali T Arafa, Siddhartha Yadav, C. H. Marshall, Elizabeth Mauer, Minxuan Huang, Binyam Yilma, Y. van der Pol, Stamatina Fragkogianni, Emily A. Teslow, Samuel Kellen, Ella Boytim, Christine Luo, Megan Ludwig, Weijie Zhang, Arockia Jayaraj, Deborah K Armstrong, William B Isaacs, J. Drake, Hai Dang Nguyen, R. S. Huang, Calvin Y Chao, Emil Lou, S. Dehm, F. Couch, Justin H. Hwang, E. Antonarakis
          </td>
          <td>2025-02-28</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="This study aimed to explore the potential correlation between the metabolic intermediate L-2-hydroxyglutarate (L-2-HG) and T cell exhaustion, as well as the underlying mechanisms involved. In this study, we investigated the presence of exhausted T (Tex) cells in patients under certain conditions: HIV infection, chronic leukemia, and hepatocellular carcinoma. To gain insights into the epigenetic signatures and transcriptome changes in Tex cells, we employed a combination of RNA-seq and ATAC-seq analyses. To evaluate the impact of L-2-HG on mitochondrial function, differentiation, and antitumor capacity of Tex cells, we utilized in vitro cell culture experiments and animal tumor models. We observed mitochondrial depolarization and metabolic dysfunction in Tex cells, accompanied by a significant reduction in L-2-HG levels. Moreover, altered epigenetic characteristics were observed in Tex cells, including a substantial increase in H3K27me3 abundance. Culturing Tex cells with L-2-HG demonstrated improved mitochondrial metabolism, reduced H3K27me3 abundance, and enhanced memory T cell differentiation. In a mouse melanoma tumor model, L-2-HG–treated CD8+ T cells for adoptive therapy led to significantly reduced tumor volume and significantly enhanced effector function of T cells. The study revealed that L-2-HG acted as an immune metabolite through epigenetic modifications of Tex cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/949077e42ee523669fc5e79909fcf9d866b45a14" target='_blank'>
              Immunometabolite L-2-HG promotes epigenetic modification of exhausted T cells and improves antitumor immunity
              </a>
            </td>
          <td>
            Yanying Yang, Xiaoyan Li, Fangming Liu, Mingyue Ma, Ying Yang, Chengchao Ruan, Yan Lu, Xiaoyan Li, Xiangdong Wang, Yinghong Shi, Zheng Zhang, Hua Wang, Zhou-Li Cheng, Duojiao Wu
          </td>
          <td>2025-03-04</td>
          <td>JCI Insight</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Anti-PD-1/PD-L1 immune checkpoint blockade (ICB) therapy has revolutionized clinical cancer treatment, while abnormal PD-L1 or HLA-I expression in patients can significantly impact the therapeutic efficacy. Somatic mutations in cancer cells that modulate these critical regulators are closely associated with tumor progression and ICB response. However, a systematic interpretation of cancer immune-related mutations is still lacking. Here, we harnessed the ABEmax system to establish a large-scale sgRNA library encompassing approximately 820,000 sgRNAs that target all feasible serine/threonine/tyrosine residues across the human genome, which systematically unveiled thousands of novel mutations that decrease or augment PD-L1 or HLA-I expression. Beyond residues associated with phosphorylation events, our screens also identified functional mutations that affect mRNA or protein stability, DNA binding capacity, protein-protein interactions, and enzymatic catalytic activity, leading to either gene inactivation or activation. Notably, we uncovered certain mutations that concurrently modulate PD-L1 and HLA-I expression, represented by the clinically relevant mutation SETD2_Y1666. We demonstrated that this mutation induces consistent phenotypic effects across multiple cancer cell lines and enhances the efficacy of immunotherapy in different tumor models. Our findings provide an unprecedented resource of functional residues that regulate cancer immunosurveillance, offering valuable guidance for clinical diagnosis, ICB therapy, and the development of innovative drugs for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0151a36f604dedfea97d97f1dfa3c6ae2a57eb97" target='_blank'>
              Massively parallel interrogation of human functional variants modulating cancer immunosurveillance
              </a>
            </td>
          <td>
            Ying Liu, Yongshuo Liu, Xuran Niu, Ang Chen, Yizhou Li, Ying Yu, Binrui Mo, Zhiheng Liu, Tao Xu, Jie Cheng, Zeguang Wu, Wensheng Wei
          </td>
          <td>2025-03-19</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 Cancer remains a significant global health challenge, claiming numerous lives annually and posing complex hurdles in management. The disease's multifaceted nature, combined with disparities in healthcare infrastructure and socioeconomic factors worldwide, complicates effective treatment. With alarming mortality rates, there is an urgent need to enhance prevention and early detection strategies. Early detection not only enables less invasive treatments but also improves the chances of successful recovery, highlighting the importance of proactive approaches. Lung cancer ranks highest in mortality rates, followed by breast, prostate, and colon cancers. Cancer is often seen as both a "genetic disease," involving gene mutations, and an "environmental disease," influenced by various environmental factors affecting gene expression. Metabolic alterations are a hallmark of cancer, and disruptions in cellular stability and function significantly contribute to its progression. Non-coding RNAs (ncRNAs) play crucial regulatory roles across cancers, with diverse structures and functions. They act as oncogenes or tumor suppressors depending on the cancer type, intricately modulating genetic and epigenetic processes. Recent studies challenge the perception of ncRNAs' insignificance, revealing their substantial impact on regulatory mechanisms. Indeed, ncRNAs intricately connect genetic networks, influencing vital protein effectors that dictate cellular responses and fate. Consequently, dysregulated ncRNAs play a pivotal role in disease pathogenesis, acting as either oncogenic drivers or tumor suppressors through aberrant expression in various cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e52de2b0fb9cae18b101db41cf959bce17a1986" target='_blank'>
              The role of non-coding RNAs (ncRNAs) and their potential connection with cancer
              </a>
            </td>
          <td>
            Safura Absalan, Hamidreza Vaziri
          </td>
          <td>2025-03-22</td>
          <td>Egyptian Journal of Medical Human Genetics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Glioblastoma (GBM) is an aggressive malignant brain tumor with almost inevitable recurrence despite multimodal management with surgical resection and radio-chemotherapy. While several genetic, proteomic, cellular, and anatomic factors interplay to drive recurrence and promote treatment resistance, the epigenetic component remains among the most versatile and heterogeneous of these factors. Herein, the epigenetic landscape of GBM refers to a myriad of modifications and processes that can alter gene expression without altering the genetic code of cancer cells. These processes encompass DNA methylation, histone modification, chromatin remodeling, and non-coding RNA molecules, all of which have been found to be implicated in augmenting the tumor’s aggressive behavior and driving its resistance to therapeutics. This review aims to delve into the underlying interactions that mediate this role for each of these epigenetic components. Further, it discusses the two-way relationship between epigenetic modifications and tumor heterogeneity and plasticity, which are crucial to effectively treat GBM. Finally, we build on the previous characterization of epigenetic modifications and interactions to explore specific targets that have been investigated for the development of promising therapeutic agents.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3dc0fa03a9f64a66bc6beac8c2a3a49bfa95cc71" target='_blank'>
              The epigenetic mechanisms involved in the treatment resistance of glioblastoma
              </a>
            </td>
          <td>
            Aanya Shahani, Hasan Slika, Ahmad Elbeltagy, Alexandra Lee, Christopher Peters, Toriyn Dotson, Divyaansh Raj, Betty Tyler
          </td>
          <td>2025-02-25</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/344c45deee3ee4d6c328b84a6e73872831228859" target='_blank'>
              Abstract B019: Cyclin E1 amplification promotes chromosomal instability and distinct vulnerabilities in gastroesophageal adenocarcinoma
              </a>
            </td>
          <td>
            Zhong Wu, Ankur Nagaraja, Chunyang Bao, Jin Zhou, A. Bass
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f7430000b0cae106ff398286a8c011c0d35dbd7" target='_blank'>
              Loss-of-Function genetic Screen Unveils Synergistic Efficacy of PARG Inhibition with Combined 5-Fluorouracil and Irinotecan Treatment in Colorectal Cancer
              </a>
            </td>
          <td>
            Cristina Queralt, Cristina Moreta-Moraleda, Marta Costa, Ferran Grau-Leal, J. Diesch, Carla Vendrell-Ayats, Eva Musulén, C. Bugés, José Luis Manzano, Sara Cabrero, Johannes Zuber, M. Buschbeck, Sonia Forcales, E. Martínez-Balibrea
          </td>
          <td>2025-04-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Simple Summary Pancreatic cancer is one of the most heterogeneous and highly aggressive malignancies. Traditionally, the gradual evolution model of pancreatic tumors has been based on the progressive acquisition of key genetic alterations. However, a complementary model based on rapid genetic evolution has also emerged, according to which driver gene inactivations can occur simultaneously. In both models, an activating mutation in the KRAS oncogene is the main initiating event in acinar pancreatic cells. However, in advanced stages of the disease, additional amplifications and major imbalances in the mutant KRAS allele are often detected. Here, we discuss how KRAS allelic imbalances can arise during tumor evolution. We also analyze the importance of KRAS allelic imbalances in pancreatic cancer biology and their possible therapeutic implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f5bfa462eb026da0cc351b1af5d1e8f34f671d8" target='_blank'>
              Unlocking the Genetic Secrets of Pancreatic Cancer: KRAS Allelic Imbalances in Tumor Evolution
              </a>
            </td>
          <td>
            Vasiliki Liaki, Blanca Rosas-Perez, Carmen Guerra
          </td>
          <td>2025-04-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Objective Active surveillance (AS) offers a viable alternative to surgical intervention for the management of indolent papillary thyroid carcinoma (PTC), helping to minimize the incidence of unnecessary treatment. However, the broader adoption of AS is hindered by the need for more reliable diagnostic markers. This study aimed to identify the differences between indolent and progressive PTC and find new targets for biomarker development and therapeutic strategies. Methods We used single-cell RNA sequencing (scRNA-seq) to analyze cellular differences in 10 early-stage PTC tumors. Findings were validated in an additional 25 tumors using cell co-culture, migration assays, immunofluorescence staining, flow cytometry, and analysis of data from The Cancer Genome Atlas (TCGA). Results Tumor-infiltrating B cells (TIL-B), particularly germinal center B cells (GC-B), were more abundant in indolent PTC. These cells suppressed thyroid cell proliferation in both indolent and progressive cases, though indolent PTC had a higher capacity to recruit peripheral B cells. In indolent cases, TIL-B cells showed increased proliferation and formed clusters within tertiary lymphoid structures (TLS). PTPRC-CD22 interactions were identified as potential drivers of TIL-B cell proliferation. Markers linked to GC-B cells, such as LMO2, were highlighted as potential diagnostic and prognostic indicators for indolent PTC. Conclusion This study provides insights into the cellular landscape of early-stage PTC, revealing distinct tumor and immune microenvironment features in indolent and progressive cases. These findings advance the understanding of indolent PTC biology and support the development of reliable diagnostic and prognostic biomarkers. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03341-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c059a59d61bc9cea405825a6d5d7499c3788b8e" target='_blank'>
              Single-cell transcriptomics analysis reveals that the tumor-infiltrating B cells determine the indolent fate of papillary thyroid carcinoma
              </a>
            </td>
          <td>
            Chunmei Li, Pei Wang, Zhizhong Dong, Weihan Cao, Yan-Jun Su, Jian-ming Zhang, Shuyan Zhao, Zhiyuan Wang, Zi Lei, Li Shi, Ruochuan Cheng, Wen Liu
          </td>
          <td>2025-03-11</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Breast cancer (BC) is the most prevalent malignancy among women, characterized by extensive heterogeneity stemming from molecular and genetic alterations. This review explores the intricate epigenetic landscape of BC, highlighting the significant role of epigenetic modifications—particularly DNA methylation, histone modifications, and the influence of non-coding RNAs—in the initiation, progression, and prognosis of the disease. Epigenetic alterations drive crucial processes, including gene expression regulation, cell differentiation, and tumor microenvironment interactions, contributing to tumorigenesis and metastatic potential. Notably, aberrations in DNA methylation patterns, including global hypomethylation and hypermethylation of CpG islands, have been associated with distinct BC subtypes, with implications for early detection and risk assessment. Furthermore, histone modifications, such as acetylation and methylation, affect cancer cell plasticity and aggressiveness by profoundly influencing chromatin dynamics and gene transcription. Finally, non-coding RNAs contribute by modulating epigenetic machinery and gene expression. Despite advances in our knowledge, clinical application of epigenetic therapies in BC is still challenging, often yielding limited efficacy when used alone. However, combining epi-drugs with established treatments shows promise for enhancing therapeutic outcomes. This review underscores the importance of integrating epigenetic insights into personalized BC treatment strategies, emphasizing the potential of epigenetic biomarkers for improving diagnosis, prognosis, and therapeutic response in affected patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67a596476c40bed6ddb55424b08c446ae8d78faf" target='_blank'>
              Decoding the Epigenome of Breast Cancer
              </a>
            </td>
          <td>
            Elisa Cortellesi, Isabella Savini, Matteo Veneziano, Alessandra Gambacurta, M. Catani, V. Gasperi
          </td>
          <td>2025-03-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="The chemical modification of biological molecules is a critical regulatory mechanism for controlling molecular functions. Although research has long focused on DNA and proteins, RNA modifications have recently attracted substantial interest with the advancement in detection technologies. In oncology, many studies have identified dysregulated RNA modifications including m6A, m1A, m5C, m7G, pseudouridylation and A to I editing, leading to disrupted downstream pathways. As the concept of the tumor microenvironment has gained prominence, studies have increasingly examined the role of RNA modifications in this context, focusing on interactions among cancer cells, immune cells, stromal cells, and other components. Here we review the RNA modifications in the tumor microenvironment through the perspective of the Cancer-Immunity Cycle. The extracellular RNA modifications including exosomes and influence of microbiome in RNA modifications are potential research questions. Additionally, RNA modifying enzymes including FTO, ALKBH5, METTL3, PUS7 are under investigation as potential biomarkers and targets for combination with immunotherapies. ADCs and mimetics of modified RNA could be potential novel drugs. This review discusses the regulatory roles of RNA modifications within the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95bb55c6fe724814b8270d97b3e10e6a1daa2920" target='_blank'>
              RNA modifications in the tumor microenvironment: insights into the cancer-immunity cycle and beyond
              </a>
            </td>
          <td>
            You-Peng Ding, Cui-Cui Liu, Ke-Da Yu
          </td>
          <td>2025-04-02</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cancer is a rising issue worldwide, and numerous studies have focused on understanding the underlying reasons for its occurrence and finding proper ways to defeat it. By applying technological advances, researchers are continuously uncovering and updating treatments in cancer therapy. Their vast functions in the regulation of cell growth and proliferation and their significant role in the progression of diseases, including cancer. This review provides a comprehensive analysis of ncRNAs in breast cancer, focusing on long non-coding RNAs such as HOTAIR, MALAT1, and NEAT1, as well as microRNAs such as miR-21, miR-221/222, and miR-155. These ncRNAs are pivotal in regulating cell proliferation, metastasis, drug resistance, and apoptosis. Additionally, we discuss experimental approaches that are useful for studying them and highlight the advantages and challenges of each method. We then explain the results of these clinical trials and offer insights for future studies by discussing major existing gaps. On the basis of an extensive number of studies, this review provides valuable insights into the potential of ncRNAs in cancer therapy. Key findings show that even though the functions of ncRNAs are vast and undeniable in cancer, there are still complications associated with their therapeutic use. Moreover, there is an absence of sufficient experiments regarding their application in mouse models, which is an area to work on. By emphasizing the crucial role of ncRNAs, this review underscores the need for innovative approaches and further studies to explore their potential in cancer therapy. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2d581bb8c9e5870e5b44e989456c39919368fe9" target='_blank'>
              Non-coding RNAs, a double-edged sword in breast cancer prognosis
              </a>
            </td>
          <td>
            Maryam Solaimani, Sahar Hosseinzadeh, M. Abasi
          </td>
          <td>2025-04-01</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The metabolic enzyme aldehyde dehydrogenase 1A1 (ALDH1A1), a cancer stem cell marker associated with poor outcomes in breast cancer, has emerged as a promising therapeutic target in TNBC. The aim of this study was to investigate the role of ALDH1A1 in radiation resistance and redox stress in triple negative breast cancer (TNBC). Functional knockouts of ALDH1A1 were generated by the CRISPR/Cas9-mediated deletion of ALDH1A1 in the SUM159 cell line, and three distinct clonal populations were isolated. Genetic targeting was confirmed by Sanger sequencing, and the loss of ALDH1A1 protein expression was validated by Western blotting. Functional assays assessed ALDEFLUOR activity, cell viability, self-renewal capacity, and reactive oxygen species (ROS) levels with or without radiation in both the bulk population and clonal lines. Interestingly, ALDEFLUOR activity was uniformly lost across all clonal lines; however, functional effects of ALDH1A1 loss on redox stress, survival, and radiation sensitivity were observed in only one clonal population. These findings highlight significant variability in the role of ALDH1A1 among clonal populations, reflecting the complexity of tumor heterogeneity. This underscores the importance of accounting for tumor heterogeneity when targeting ALDH1A1, as certain TNBC subpopulations may rely more heavily on ALDH1A1 function. These insights are critical for developing effective ALDH1A1-targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce39f8fe1789f12e83b81bf503275e5a974f1d83" target='_blank'>
              CRISPR/Cas9 Targeting of Aldehyde Dehydrogenase 1A1 Reveals Heterogeneous Roles in Radiation Response and Redox Stress Across Clonal Lines in Triple-Negative Breast Cancer
              </a>
            </td>
          <td>
            Grace O. Ajayi, Aihui Ma, Shirin R. Modarai, Lynn M. Opdenaker, Jennifer Sims-Mourtada
          </td>
          <td>2025-03-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64abcfa81e97e4ccd6aa92a436d6301cbd6a137c" target='_blank'>
              CRISPR-Cas9: Transforming Functional Genomics, Precision Medicine, and Drug Development - Opportunities, Challenges, and Future Directions.
              </a>
            </td>
          <td>
            Zuber Khan, Mumtaz, Sumedha Gupta, Sidharth Mehan, Tarun Sharma, Manjeet Kumar, Pankaj Kumar Maurya, Arun Kumar Sharma, Ghanshyam Das Gupta, A. Narula
          </td>
          <td>2025-03-21</td>
          <td>Current gene therapy</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4625a1647e0225fc85616b697ca8ae26c76a3c30" target='_blank'>
              Transformative CRISPR-Cas9 Technologies: A Review of Molecular Mechanisms, Precision Editing Techniques, and Clinical Progress in Sickle Cell Disease.
              </a>
            </td>
          <td>
            Komal, Prabhjot Kaur, Nidhi Arora, Jyotiram A. Sawale, Amandeep Singh
          </td>
          <td>2025-03-05</td>
          <td>Current drug metabolism</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="ABSTRACT Gene fusions represent important oncogenic driver mutations resulting in aberrant cellular signaling. In up to 17% of all solid tumors at least one gene fusion can be identified. Precision therapy targeting fusion gene signaling has demonstrated effective clinical benefit. Advancements in clinically relevant next-generation sequencing and bioinformatic techniques have enabled expansion of therapeutic opportunity to subpopulations of patients with fusion gene expression. Clinically, tyrosine inhibitors have shown efficacy in treating fusion gene expressing cancers. Fusion genes are also clonal mutations, meaning it is a personal cancer target involving all cancer cells of that patient, not just a subpopulation of cancer cells within the cancer mass. Thus, both fusion signal disruption and immune signal targeting are effective therapeutic directions. This review discusses fusion gene targeting, therapeutic resistance, and molecular biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a39d77dcb6dc4121a7fa7e989f8a588d3f3ba74" target='_blank'>
              Clinical characterization and therapeutic targeting of fusion genes in oncology
              </a>
            </td>
          <td>
            Susan Morand, Lauren Rager, Daniel Craig, Alexander Nemunaitis, Khalil Choucair, Donald Rao, L. Stanbery, Richard C Phinney, Adam Walter, Maurizio Ghisoli, John Nemunaitis
          </td>
          <td>2025-03-24</td>
          <td>Future Oncology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Mutations in the DNA can affect cancer development and progression not only by changing the amino acid chain but also by affecting different regulatory elements such as transcription factors. This study introduces a novel pipeline to identify “mutation blocks” - small genomic areas with high mutation rates that potentially influence transcription factors binding. By analyzing GCB lymphoma patient data, mutations blocks were identified that correlated with gene expression changes and were linked to transcription factor activity. These mutation blocks suggest a selection for mutations that alter gene regulation, contributing to lymphoma development. The analysis identified 56 mutation blocks in germinal center B-cell like diffuse large B-cell lymphoma (GCB DLBCL) patients’ genomes, affecting genes such as BCL2, MYC, SGK1, and PIM1, and linked to transcription factors including MSC, TCFL5, HOXB7, FOXP3, and ZBTB6. A machine learning model that used gene ontology data suggested further potential transcription factors-gene pairs influencing cancer. These findings highlight the role of synonymous and silent mutations in altering transcription factors binding and gene expression, offering insights into the mechanisms of GCB lymphoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76d3b64e91469debfcea770e479a146974e05ae2" target='_blank'>
              Novel Driver Mutations in GCB Lymphoma Patients That Affect Transcription Factors Binding
              </a>
            </td>
          <td>
            Ofek Shami-Schnitzer, T. Tuller
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Simple Summary Pharmacogenomics is a rapidly evolving field that is crucial in optimizing cancer treatments by tailoring therapies to individual patients’ genetic profiles. This review highlights the impact of key genes, such as CYP2D6, DPYD, and UGT1A1, which influence the metabolism of essential cancer drugs like tamoxifen, fluoropyrimidines, and irinotecan. Variations in these genes can affect drug efficacy or increase the risk of side effects, underscoring the need for genetic testing to guide treatment decisions. Despite its proven benefits, pharmacogenomic testing is not yet widely used. Ongoing research aims to improve the accessibility and cost effectiveness of these tests, especially in regions where testing is limited. In the future, combining pharmacogenomic data with emerging technologies, such as liquid biopsies and gene-editing tools, will improve personalized cancer therapies, leading to better treatment outcomes and fewer side effects for patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fb68894349371e2a391455ef95a044b32009e11" target='_blank'>
              Pharmacogenomics in Solid Tumors: A Comprehensive Review of Genetic Variability and Its Clinical Implications
              </a>
            </td>
          <td>
            R. Sánchez-Bayona, Camila Catalán, María Ángeles Cobos, Milana Bergamino
          </td>
          <td>2025-03-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b52e6b9621db88b32d01f7e93fc6430a88ce3be" target='_blank'>
              Comprehensive interrogation of synthetic lethality in the DNA damage response.
              </a>
            </td>
          <td>
            John Fielden, Sebastian M. Siegner, Danielle N Gallagher, Markus S Schröder, Maria Rosaria Dello Stritto, Simon Lam, L. Kobel, Moritz F. Schlapansky, S. P. Jackson, Petr Cejka, Marco Jost, Jacob E. Corn
          </td>
          <td>2025-04-09</td>
          <td>Nature</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55c3773603d3c9014e1c679ae73200f35f924d76" target='_blank'>
              Single-cell SNP-DNA sequencing precisely maps genotoxic events in CRISPR-edited primary cells
              </a>
            </td>
          <td>
            J. Boutin, S. Fayet, V. Marin, C. Berges, M. Riandière, Jérôme Toutain, I. Lamrissi‐Garcia, C. Thibault, David Cappellen, S. Dabernat, Arthur Poulet, Maëla Francillette, Nathalie Droin, F. Moreau-Gaudry, A. Bedel
          </td>
          <td>2025-04-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Immortalized cell lines are commonly used as proxies for primary cells in human biology research. For example, Jurkat leukemic T cells fundamentally contributed to uncovering T cell signaling, activation, and immune responses. However, the immortalization process can alter key cellular properties, and researchers widely believe this process could significantly change RNA modification machinery and modification sites. In this study, we focus on pseudouridine (Ψ), one of the most abundant mRNA modifications, and compare Ψ profiles in mRNA from primary and immortalized T cells using direct RNA sequencing (DRS). Surprisingly, 87% of Ψ-sites were shared between the two cell types, primarily in transcripts encoding proteins involved in essential cellular processes, including RNA-modification regulation. Furthermore, the analysis of the 13% of sites unique to each cell type reveals that Jurkat cells contained transcripts linked to immune activation and oncogenesis, while primary T cells contained transcripts associated with calcium signaling and intracellular trafficking. We provide a list of these genes, which should be considered when using immortalized cells to study RNA modifications in immunology contexts. Most differences were driven by whether the mRNA was present or absent in the immortalized or primary cell type. Interestingly, RNA-modification enzyme expression levels were highly conserved in both cell types. This suggests that site-specific differences in Ψ levels arise from regulatory processes acting in trans rather than differences in modification enzyme levels. Significance Statement It is widely believed that RNA modification machinery and modification sites could be significantly altered in immortalized cells, yet this has never been tested. Focusing on pseudouridine (Ψ), we map Ψ in the transcriptomes of primary and immortalized human T cells. Surprisingly, most sites are conserved in the primary and immortalized T cells, with several important examples of cases with cell type specificity and should be considered on a case-by-case basis. Furthermore, we evaluated RNA-modification machinery levels in primary and immortalized T cells, finding high conservation across the cell lines. Our findings demonstrate that RNA modifications are largely conserved between primary and immortalized cells, and the edge cases can be considered individually.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5b46d555b004bfc6033b490f3c93648df1820f4" target='_blank'>
              Direct RNA sequencing of primary human T cells reveals the impact of immortalization on mRNA pseudouridine modifications
              </a>
            </td>
          <td>
            Oleksandra Fanari, Dylan Bloch, Yuchen Qiu, Michele Meseonznik, Amr Makhamreh, M. Wanunu, S. Rouhanifard
          </td>
          <td>2025-03-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18a99a90f864b231f78785ccac2d5080c6a00e33" target='_blank'>
              Explainable AI-driven prediction of APE1 inhibitors: enhancing cancer therapy with machine learning models and feature importance analysis.
              </a>
            </td>
          <td>
            Aga Basit Iqbal, Tariq Ahmad Masoodi, A. Bhat, M. Macha, Assif Assad, Syed Zubair Ahmad Shah
          </td>
          <td>2025-02-21</td>
          <td>Molecular diversity</td>
          <td>1</td>
          <td>35</td>
        </tr>

        <tr id="Introduction The recombinant human p53 adenovirus (Ad-p53) offers a promising approach for cancer therapy, yet its chromatin structure and effects on host chromatin organization and gene expression are not fully understood. Methods In this study, we employed in situ ChIA-PET to investigate the colorectal cancer cell line HCT116 with p53 knockout, comparing them to cells infected with the adenovirus-vector expressing p53. We examined alterations in chromatin interactions and gene expression following treatment with the anti-cancer drug 5-fluorouracil (5-FU). Results Our results indicate that Ad-p53 forms a specific chromatin architecture within the vector and mainly interacts with repressive or inactive regions of host chromatin, without significantly affecting the expression of associated genes. Additionally, Ad-p53 does not affect topologically associating domains (TADs) or A/B compartments in the host genome. Discussion These findings suggest that while Ad-p53 boosts p53 expression, enhancing drug sensitivity without substantially altering host HCT116 chromatin architecture.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68b2495bcc5f798b26cfeb646dc7337f34c34784" target='_blank'>
              Chromatin structure and gene transcription of recombinant p53 adenovirus vector within host
              </a>
            </td>
          <td>
            Duo Ning, Yuqing Deng, S. Tian
          </td>
          <td>2025-02-28</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract The integration of liquid biopsy with epigenetic markers offers significant potential for early lung cancer detection and personalized treatment. Epigenetic alterations, including DNA methylation, histone modifications, and noncoding RNA changes, often precede genetic mutations and are critical in cancer progression. In this study, we explore how liquid biopsy, combined with epigenetic markers, can provide early detection of lung cancer, potentially predicting onset up to 4 years before clinical diagnosis. We discuss the challenges of targeting epigenetic regulators, which could disrupt cellular balance if overexploited, and the need for maintaining key gene expressions in therapeutic applications. This review highlights the promise and challenges of using liquid biopsy and epigenetic markers for early‐stage lung cancer diagnosis, with a focus on optimizing treatment strategies for personalized and precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec05c410e812fe06f571d18427676b8689248360" target='_blank'>
              Epigenetic modifications in early stage lung cancer: pathogenesis, biomarkers, and early diagnosis
              </a>
            </td>
          <td>
            Lingfeng Bi, Xin Wang, Jiayi Li, Weimin Li, Zhoufeng Wang
          </td>
          <td>2025-02-21</td>
          <td>MedComm</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="The appearance of cancer progresses through a multistep process that includes genetic, epigenetic, mutational, inflammatory and metabolic disturbances to signaling pathways within an organ. The combined influence of these changes will dictate the growth properties of the cells; the direction of further malignancy depends on the severity of these “disturbances”. The molecular mechanisms driving abnormal inflammation and metabolism are beginning to be identified and, in some cases, are quite prominent in pre-condition states of cancer and are significant drivers of the malignant phenotype. As such, utilizing signaling pathways linked to inflammation and metabolism as biomarkers of cancer is an emerging method and includes pathways beyond those well characterized to drive metabolism or inflammation. In this review, we will discuss several emerging elements influencing proliferation, inflammation and metabolism that may play a part as drivers of the cancer phenotype. These include AMPK and leptin (linked to metabolism), NOD2/RIPK2, TAK1 (linked to inflammation), lactate and pyruvate transporters (monocarboxylate transporter [MCT], linked to mitochondrial biogenesis and metabolism) and RASSF1A (linked to proliferation, cell death, cell cycle control, inflammation and epigenetics). We speculate that the aforementioned elements are important drivers of carcinogenesis that should be collectively referenced as being involved in “ImmunoMET Oncogenesis”, a new tripartite description of the role of elements in driving cancer. This term would suggest that for a better understanding of cancer, we need to understand how proliferation, inflammation and metabolic pathways are impacted and how they influence classical drivers of malignant transformation in order to drive ImmunoMET oncogenesis and the malignant state.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f600df6865b40197b8b8a33d72a983d145a378e4" target='_blank'>
              ImmunoMet Oncogenesis: A New Concept to Understand the Molecular Drivers of Cancer
              </a>
            </td>
          <td>
            Reshma Sirajee, Sami El Khatib, L. Dieleman, Mohamed Salla, S. Baksh
          </td>
          <td>2025-02-27</td>
          <td>Journal of Clinical Medicine</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcc63f157070e0fd70ef6717ad423137ac275d75" target='_blank'>
              Stochastically Emergent Tumors offer in vivo whole genome interrogation of cancer evolution from non-malignant precursors
              </a>
            </td>
          <td>
            Ruhollah Moussavi-Baygi, Matthew J. Ryan, Woogwang Sim, Samuel B. Hoelscher, V. Luga, Arun A. Chandrakumar, Lore Hoes, Junghwa Cha, Young Sun Lee, Katelyn Herm, Ben Doron, C.K. Cornelia Ding, Bradley A. Stohr, Peng Jin, Tejasveeta V. Nadkarni, Xiangyi Fang, Melita Haryono, An Nguyen, Wouter R. Karthaus, C.L. Sawyers, Felix Y. Feng, Hani Goodarzi, Rohit Bose
          </td>
          <td>2025-03-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background
 Identifying transcriptional cis-regulatory elements (CREs) and understanding their role in gene expression are essential for the precise manipulation of gene expression and associated phenotypes. This knowledge is fundamental for advancing genetic engineering and improving crop traits. Results We here demonstrate that CREs can be accurately predicted and utilized to precisely regulate gene expression beyond the range of natural variation. We firstly build two sequence-to-expression deep learning models to respectively identify distal and proximal CREs by combining them with interpretability methods in multiple crops. A large number of distal CREs are verified for enhancer activity in vitro using UMI-STARR-seq on 12,000 synthesized sequences. These comprehensively characterized CREs and their precisely predicted effects further contribute to the design of in silico editing schemes for precise engineering of gene expression. We introduce a novel concept of “editingplasticity” to evaluate the potential of promoter editing to alter expression of each gene. As a proof of concept, both exhaustive prediction and random knockout mutants are analyzed within the promoter region of ZmVTE4, a key gene affecting α-tocopherol content in maize. A high degree of agreement between predicted and observed expression is observed, extending the range of natural variation and thereby allowing the creation of an optimal phenotype. Conclusions Our study provides a robust computational framework that advances knowledge-guided gene editing for precise regulation of gene expression and crop improvement. By reliably predicting and validating CREs, we offer a tool for targeted genetic modifications, enhancing desirable traits in crops. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03516-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8f6a490bf2e01c278db6d61ce9c6c070c47d07c" target='_blank'>
              Precise engineering of gene expression by editing plasticity
              </a>
            </td>
          <td>
            Yang Qiu, Lifen Liu, Jiali Yan, Xianglei Xiang, Shouzhe Wang, Yun Luo, Kaixuan Deng, Jieting Xu, Minliang Jin, Xiaoyu Wu, Liwei Cheng, Ying Zhou, Weibo Xie, Hai-Jun Liu, A. Fernie, Xuehai Hu, Jianbing Yan
          </td>
          <td>2025-03-10</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>151</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f24aad6476cebed0b2038b1cccec32124148b6f1" target='_blank'>
              The perspective of targeting cancer cell metabolism: combination therapy approaches.
              </a>
            </td>
          <td>
            Arezu Karimpur-zahmatkesh, Mohammad Khalaj-Kondori
          </td>
          <td>2025-04-09</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Introduction: Leukocyte adhesion deficiency type 1 (LAD1) is a severe inborn error of immunity caused by mutations in the ITGB2 gene, which encodes the beta-2 integrin subunit (CD18). These mutations lead to the absence or deficiency of CD18/CD11a, b, and c heterodimers, crucial for leukocyte adhesion and immune function. CRISPR-Cas9 Gene editing technology represents a promising approach for correcting these genomic defects restore the stable expression of CD18 and reverse the disease. Methods: We developed a CRISPR-Cas9-based gene correction strategy using Jurkat cells and patient-derived lymphoblastoid cell lines as surrogates for hematopoietic progenitor cells. Three candidate gRNAs were first predicted in silico using CRISPOR and experimentally tested in wild-type ITGB2-expressing Jurkat cells to identify the gRNA with the highest genomic DNA cleavage efficiency. The most efficient gRNA was then paired with espCas9 and used alongside five homology-directed repair templates (HDRs) (single-stranded donor oligonucleotides, ssODNs) to repair ITGB2 defects in patient-derived lymphoblastoid cell lines. CD18 expression levels in edited cells were quantified via flow cytometry, and whole-genome sequencing (WGS) was conducted to assess off-target effects and insertion accuracy. Results: Among the three candidate gRNAs, 2-rev gRNA exhibited the highest genomic cleavage rate in Jurkat cells. Using this gRNA with espCas9 and HDR-2, we achieved a 23% restoration of CD18 expression in LAD1 patient-derived cells, a level sufficient to change the disease course from severe to moderate. Whole-genome sequencing confirmed the absence of off-target mutations or undesired DNA insertions, demonstrating high specificity and precision in gene correction. Discussion: This CRISPR-Cas9-based method provides a precise and effective approach for correcting ITGB2 mutations in LAD1 patients. The high-fidelity gene editing process, validated through WGS, supports its potential for future applications in CD34+ hematopoietic stem cell therapies. The approach can be further optimized for clinical translation, offering a path toward a stable and long-term cure for LAD1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af64b4b32aef04c80bbc3b2f45c925d4b04fb25b" target='_blank'>
              Lymphoblastoid and Jurkat cell lines are useful surrogate in developing a CRISPR-Cas9 method to correct leukocyte adhesion deficiency genomic defect
              </a>
            </td>
          <td>
            Ahmad R. Ramadan, Noureddine Ben Khalaf, Khaled Trabelsi, H. Bakheit, I. Ben‐Mustapha, M. Barbouche, M. Fathallah
          </td>
          <td>2025-03-21</td>
          <td>Frontiers in Bioengineering and Biotechnology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Glioblastoma (GBM) is one of the deadliest tumors due to its high aggressiveness and resistance to standard therapies, resulting in a dismal prognosis. This lethal tumor carries out metabolic reprogramming in order to modulate specific pathways, providing metabolites that promote GBM cells proliferation and limit the efficacy of standard treatments. Indeed, GBM remodels glucose metabolism and undergoes Warburg effect, fuelling glycolysis even when oxygen is available. Moreover, recent evidence revealed a rewiring in nucleotide, lipid and iron metabolism, resulting not only in an increased tumor growth, but also in radio- and chemo-resistance. Thus, while on the one hand metabolic reprogramming is an advantage for GBM, on the other hand it may represent an exploitable target to hamper GBM progression. Lately, a number of studies focused on drugs targeting metabolism to uncover their effects on tumor proliferation and therapy resistance, demonstrating that some of these are effective, in combination with conventional treatments, sensitizing GBM to radiotherapy and chemotherapy. However, GBM heterogeneity could lead to a plethora of metabolic alterations among subtypes, hence a metabolic treatment might be effective for proneural tumors but not for mesenchymal ones, which are more aggressive and resistant to conventional approaches. This review explores key mechanisms of GBM metabolic reprogramming and their involvement in therapy resistance, highlighting how metabolism acts as a double-edged sword for GBM, taking into account metabolic pathways that seem to offer promising treatment options for GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d51665808a8d2cd9bd7f18aa2c80f26d1323e55" target='_blank'>
              Targeting metabolic reprogramming in glioblastoma as a new strategy to overcome therapy resistance
              </a>
            </td>
          <td>
            S. D’Aprile, S. Denaro, Anna Gervasi, N. Vicario, R. Parenti
          </td>
          <td>2025-02-26</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Human T-cell lymphotropic virus type 1 (HTLV-1) infection causes the uncommon and deadly cancer known as adult T-cell leukemia/lymphoma (ATLL), which affects mature T cells. Its clinical appearance is varied, and its prognosis is often miserable. Drug resistance to conventional therapies confers significant therapeutic challenges in the management of ATLL. This review discusses the emerging role of epigenetic medical advances in the treatment of ATLL, focusing on DNA methyltransferase inhibitors, histone deacetylase inhibitors, histone methyltransferase inhibitors, and BET inhibitors. Indeed, several classes of epigenetic therapies currently exhibit trailed efficacy in preclinical and clinical studies: DNA methyltransferase inhibitors like azacitidine and decitabine reexpression of silenced tumor suppressors; histone deacetylase inhibitors like vorinostat and romidepsin induce cell cycle arrest and apoptosis; bromodomain and extra-terminal inhibitors like JQ1 disrupt oncogenic signaling pathways. Whereas preclinical and early clinical data indicate modest to good efficacy for such treatments, significant challenges remain. Here, we discuss the current state of understanding of epigenetic dysregulation in ATLL and appraise the evidence supporting the use of these epi-drugs. However, despite the opened doors of epigenetic treatment, much more research is required with regard to showing the best combinations of drugs and their resistance mechanisms, the minimization of adverse effects, and how this hope will eventually be translated into benefit for the patient with ATLL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a1ce69e97a9adaccd0f62087c29dd1f173f2a27" target='_blank'>
              Advances in epigenetic treatment of adult T-cell leukemia/lymphoma: a comprehensive review
              </a>
            </td>
          <td>
            Arash Letafati, Rabeeh Mehdigholian Chaijani, Fahimeh Edalat, Nazila Eslami, Hanieh Askari, Farideh Askari, Sara Shirvani, Hamed Talebzadeh, Mahdiyeh Tarahomi, Nila MirKhani, Faeze Karimi, Mehdi Norouzi, Sayed-Hamidreza Mozhgani
          </td>
          <td>2025-03-01</td>
          <td>Clinical Epigenetics</td>
          <td>1</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ee52870e604baf41aae12c6d88c6bacc4d5a388" target='_blank'>
              Updates on DNA Repair Gene Deficiency in Colorectal Cancer (dMMR)
              </a>
            </td>
          <td>
            Gustavo Sevá-Pereira, C. S. R. Coy, Carlos Augusto Real Martinez
          </td>
          <td>2025-03-01</td>
          <td>Journal of Coloproctology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8eb6952db5fdb7496be1946cac13627deae120ed" target='_blank'>
              Targeting replication stress in neuroblastoma by exploiting the synergistic potential of second generation RRM2 and CHK1 inhibitors
              </a>
            </td>
          <td>
            I. Nelen, Soetkin Leys, Sarah-Lee Bekaert, F. De Vloed, F. Martens, Angeline Praveena Enton Raj, S. Ohmura, Lasse Vleminckx, T. G. Grünewald, N. Roy, B. D. Wilde, Frank Speleman, Annelies Van Hemelryk, L. Depestel, K. Durinck
          </td>
          <td>2025-03-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="MYC plays a pivotal role in the biology of various sarcoma subtypes, acting as a key regulator of tumor growth, proliferation, and metabolic reprogramming. This oncogene is frequently dysregulated across different sarcomas, where its expression is closely intertwined with the molecular features unique to each subtype. MYC interacts with critical pathways such as cell cycle regulation, apoptosis, and angiogenesis, amplifying tumor aggressiveness and resistance to standard therapies. Furthermore, MYC influences the tumor microenvironment by modulating cell–extracellular matrix interactions and immune evasion mechanisms, further complicating therapeutic management. Despite its well-established centrality in sarcoma pathogenesis, targeting MYC directly remains challenging due to its “undruggable” protein structure. However, emerging therapeutic strategies, including indirect MYC inhibition via epigenetic modulators, transcriptional machinery disruptors, and metabolic pathway inhibitors, offer new hope for sarcoma treatment. This review underscores the importance of understanding the intricate roles of MYC across sarcoma subtypes to guide the development of effective targeted therapies. Given MYC’s central role in tumorigenesis and progression, innovative approaches aiming at MYC inhibition could transform the therapeutic landscape for sarcoma patients, providing a much-needed avenue to overcome therapeutic resistance and improve clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b3e9fb9dbedb9674622b99a03bb647022b7fb21" target='_blank'>
              Untangling the Role of MYC in Sarcomas and Its Potential as a Promising Therapeutic Target
              </a>
            </td>
          <td>
            Fabio Sias, Stefano Zoroddu, Rossana Migheli, L. Bagella
          </td>
          <td>2025-02-25</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="CRISPR-Cas technology has revolutionized biology by enabling precise DNA and RNA edits with ease. However, significant challenges remain for translating this technology into clinical applications. Traditional protein engineering methods, such as rational design, mutagenesis screens, and directed evolution, have been used to address issues like low efficacy, specificity, and high immunogenicity. These methods are labor-intensive, time-consuming, and resource-intensive and often require detailed structural knowledge. Recently, computational strategies have emerged as powerful solutions to these limitations. Using artificial intelligence (AI) and machine learning (ML), the discovery and design of novel gene-editing enzymes can be streamlined. AI/ML models predict activity, specificity, and immunogenicity while also enhancing mutagenesis screens and directed evolution. These approaches not only accelerate rational design but also create new opportunities for developing safer and more efficient genome-editing tools, which could eventually be translated into the clinic.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6cb55c796a93b1a1e6ebc7882f1d2a726253cbc6" target='_blank'>
              Engineering a New Generation of Gene Editors: Integrating Synthetic Biology and AI Innovations
              </a>
            </td>
          <td>
            B. Chia, Yu Fen Samantha Seah, Bolun Wang, Kimberle Shen, Diya Srivastava, W. Chew
          </td>
          <td>2025-02-25</td>
          <td>ACS Synthetic Biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="The ribosomal DNA (rDNA) plays a vital role in regulating protein synthesis by ribosome biogenesis, essential for maintaining cellular growth, metabolism, and more. Cancer cells show a high dependence on ribosome biogenesis and exhibit elevated rDNA transcriptional activity. CX-5461, also known as Pidnarulex, is a First-in-Class anticancer drug that has received 'Fast Track Designation' approval from the FDA. Initially reported to inhibit Pol I-driven rDNA transcription, CX-5461 was recently identified as a G-quadruplex structure (G4) stabilizer and is currently completed or undergoing multiple Phase I clinical trials in patients with breast and ovarian cancers harboring BRCA1/2, PALB2, or other DNA repair deficiencies. Additionally, preclinical studies have confirmed that CX-5461 demonstrates promising therapeutic effects against multifarious non-cancer diseases, including viral infections, and autoimmune diseases. This review summarizes the mechanisms of CX-5461, including its transcriptional inhibition of rDNA, binding to G4, and toxicity towards topoisomerase, along with its research status and therapeutic effects across various diseases. Lastly, this review highlights the targeted therapy strategy of CX-5461 based on nanomedicine delivery, particularly the drug delivery utilizing the nucleic acid aptamer AS1411, which contains a G4 motif to specifically target the highly expressed nucleolin on the surface of tumor cell membranes; It also anticipates the strategy of coupling CX-5461 with peptide nucleic acids and locked nucleic acids to achieve dual targeting, thereby realizing individualized G4-targeting by CX-5461. This review aims to provide a general overview of the progress of CX-5461 in recent years and suggest potential strategies for disease treatment involving ribosomal RNA synthesis, G4, and topoisomerase.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38252b99654ce0eee22dc4caf1a279f3bd484a1a" target='_blank'>
              The G-quadruplex ligand CX-5461: an innovative candidate for disease treatment
              </a>
            </td>
          <td>
            Hong-Xia Li, , Jing Fei, Man Guo, Chen Zeng, Pi-Jun Yan, Yong Xu, Gang Qin, Fang-Yuan Teng
          </td>
          <td>2025-04-18</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Tissues and organs develop from single founder cells, which give rise to distinct cell lineages that contribute to regeneration and maintenance of homeostasis in the adult. Tracing the genealogical relationships between individual cells and their gene expression signatures is an important step towards understanding how these processes are regulated in human health and disease. Here, we present mt-SCITE, a computational method for inferring the evolutionary history of dividing cells based on mitochondrial mutations detected in single cells. We show that mt-SCITE outperforms existing methods in accurately estimating the correct mitochondrial mutation tree from simulated data. To validate our method, we applied mt-SCITE on in vitro expanded T-cells profiled with bulk ATAC sequencing, where the clonal relationships were determined independently from T cell receptor sequences. We used our method to reconstruct the division histories and transcriptional heterogeneity of clonally related CD8+ T cells from single-cell RNA sequencing data obtained from a healthy human donor. This allowed us to track the in vivo development of CD8+ T cell clones post vaccination and resolve subclonal lineage relationships without the need for genetic barcoding. Our analysis revealed that T cell clones can adopt predominantly memory-like or effector-like states, while some exhibit mixed identities. Taken together, our method represents a novel approach to study cell lineage development in humans.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b46463e933e01c11da7ed98772b7d2ff4e8ab033" target='_blank'>
              Cell lineage inference from mitochondrial mutations
              </a>
            </td>
          <td>
            Joanna Hård, S. Seidel, Pedro F. Ferreira, Pietro Berkes, Katharina Jahn, Carl-Johan Eriksson, Jeff E. Mold, Jakob Michaelsson, Jack Kuipers, N. Beerenwinkel
          </td>
          <td>2025-03-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/471fa22c73f7c1a8083ff473dd937339ee492d0c" target='_blank'>
              Abstract A022: Decoding aneuploidy: Identifying drivers and therapeutic targets in recurrent breast cancer copy number alterations
              </a>
            </td>
          <td>
            Khalid N. Al-Zahrani, Ellen Langille, A. Obersterescu, Christopher Lowden, Katie Teng, Lauren Caldwell, David Cook, Miguel Pérez-Castro, Cynthia H Chiu, Alec Bahcheli, R. Tsai, Jacob Berman, Kin Chan, Linkang Zhang, K.W. Annie Bang, Michael Parsons, Adele Lopes, Jocelyn Nurtanto, E. I. Temel, Iosifina Fotiadou, Julien Dessapt, Hartland Jackson, Sean E Egan, J. Reimand, Jeffrey Wrana, Daniel Schramek
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Background Tuberculosis, a persistent global health threat, necessitates a comprehensive understanding of the genes and pathways crucial for the survival and virulence of the causative pathogen, Mycobacterium tuberculosis. Working with M. tuberculosis (M.tb) presents significant challenges; therefore, the use of M. smegmatis as a surrogate system for conducting genetic studies of M.tb has proven to be highly valuable. Development of novel genetic tools to probe cellular processes accelerates the progress in the field of drug development and also helps in understanding the basic physiology of the bacterium. Results This study reports the successful implementation and evaluation of the CRISPR-Cas12a system for gene repression in Mycobacterium smegmatis, a surrogate for M. tuberculosis. We engineered a Cas12a-based CRISPR interference (CRISPRi) system and assessed its functionality. Targeting 45 genes with a single sgRNA per gene, we achieved efficient gene repression, leading to marked phenotypic changes. Each knockdown strain was evaluated individually for growth phenotypes, and a comparison of the results with the reported essential gene library probed with dCas9 demonstrated congruous results across diverse gene categories. The study shows that CRISPR/Cas12a system can be effectively utilised with a single gene specific target for efficient silencing of the gene and highlights the importance of subsequent growth assays required to evaluate the vulnerability of targeted gene silencing. Conclusion Our findings reveal the robustness and versatility of the dCas12a-CRISPRi system in M. smegmatis, providing a valuable tool for functional genomics research. This work showcases the potential of the dCas12a-CRISPRi system in investigating essential genes, enabling a deeper understanding of the biology and potential therapeutic targets in mycobacterium species. Supplementary Information The online version contains supplementary material available at 10.1186/s13036-025-00490-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fd956491f5700a578d8ac40b6d89d08cd7f2d58" target='_blank'>
              CRISPR/Cas12a-mediated gene silencing across diverse functional genes demonstrates single gene-specific spacer efficacy in Mycobacterium smegmatis
              </a>
            </td>
          <td>
            Megha Sodani, C. S. Misra, Savita Kulkarni, D. Rath
          </td>
          <td>2025-02-28</td>
          <td>Journal of Biological Engineering</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcf67474bac596be1d2773e2079b4606a1ba650b" target='_blank'>
              Chromosomal Instability and IL6-Mediated Inflammatory Signaling in Human Lymphoma Cells
              </a>
            </td>
          <td>
            Veronika Müller
          </td>
          <td>2025-03-31</td>
          <td>McGill Science Undergraduate Research Journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d56061c904e32a44a57cf237e45520971f217c24" target='_blank'>
              The breast tumor immune microenvironment of DNA double-strand break repair pathogenic variant carriers is enriched with tumor-associated macrophages.
              </a>
            </td>
          <td>
            Rebecca L Kelly, Yuxi Liu, Alexandra R Harris, Cheng Peng, Yujing J Heng, Gabrielle M. Baker, D. Stover, R. Tamimi, P. Kraft
          </td>
          <td>2025-04-02</td>
          <td>Cancer epidemiology, biomarkers & prevention : a publication of the American Association for Cancer Research, cosponsored by the American Society of Preventive Oncology</td>
          <td>0</td>
          <td>84</td>
        </tr>

        <tr id="The sequencing of the human genome in 2003 marked a transformative shift from a one-size-fits-all approach to personalized medicine, emphasizing patient-specific molecular and physiological characteristics. Advances in sequencing technologies, from Sanger methods to Next-Generation Sequencing (NGS), have generated vast genomic datasets, enabling the development of tailored therapeutic strategies. Pharmacogenomics (PGx) has played a pivotal role in elucidating how the genetic make-up influences inter-individual variability in drug efficacy and toxicity discovering predictive and prognostic biomarkers. However, challenges persist in interpreting polymorphic variants and translating findings into clinical practice. Multi-omics data integration and bioinformatics tools are essential for addressing these complexities, uncovering novel molecular insights, and advancing precision medicine. In this review, starting from our experience in PGx studies performed by DMET microarray platform, we propose a guideline combining machine learning, statistical, and network-based approaches to simplify and better understand complex DMET PGx data analysis which can be adapted for broader PGx applications, fostering accessibility to high-performance bioinformatics, also for non-specialists. Moreover, we describe an example of how bioinformatic tools can be used for a comprehensive integrative analysis which could allow the translation of genetic insights into personalized therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d0ed0b57661f06afa1c6d9d81fc8ef07f868c7b" target='_blank'>
              Bioinformatic challenges for pharmacogenomic study: tools for genomic data analysis
              </a>
            </td>
          <td>
            M. Arbitrio, Marianna Milano, Maria Lucibello, E. Altomare, N. Staropoli, P. Tassone, P. Tagliaferri, Mario Cannataro, Giuseppe Agapito
          </td>
          <td>2025-04-11</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="The landscapes of somatic mutation in normal cells inform on the processes of mutation and selection operative throughout life, permitting insight into normal ageing and the earliest stages of cancer development. Here, by whole-genome sequencing of 238 microdissections from 30 individuals, including 18 with gastric cancer, we elucidate the developmental trajectories of normal and malignant gastric epithelium. We find that gastric glands are units of monoclonal cell populations which accrue ∼28 somatic single nucleotide variants per year, predominantly attributable to endogenous mutational processes. In individuals with gastric cancer, glands often show elevated mutation burdens due to acceleration of mutational processes linked to proliferation and oxidative damage. These hypermutant glands were primarily detected in the gastric antrum and were mostly associated with chronic inflammation and intestinal metaplasia, known cancer risk factors. Unusually for normal cells, gastric epithelial cells often carry recurrent trisomies of specific chromosomes, which are highly enriched in a subset of individuals. Surveying approximately 8,000 gastric glands by targeted sequencing, we found somatic driver mutations in a distinctive repertoire of known cancer genes, including ARID1A, CTNNB1, KDM6A and ARID1B. Their prevalence increases with age to occupy approximately 5% of the gastric epithelial lining by age 60 years. Our findings provide insights into the intrinsic and extrinsic influences on somatic evolution in the gastric epithelium, in healthy, precancerous and malignant states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a80de4630550f795baa857c6ee76f35822aa4bc" target='_blank'>
              The somatic mutation landscape of normal gastric epithelium
              </a>
            </td>
          <td>
            Tim H. H. Coorens, G. Collord, Hyungchul Jung, Yichen Wang, L. Moore, Y. Hooks, Krishnaa Mahbubani, K. Saeb-Parsy, P. J. Campbell, I. Martincorena, Suet Yi Leung, Michael R Stratton
          </td>
          <td>2024-03-19</td>
          <td>Nature</td>
          <td>1</td>
          <td>88</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a74f7e200ce532d827202c8faf344e8401edaf1" target='_blank'>
              Abstract B015: Assessment of apoptotic priming in thymic epithelial tumors to define hallmarks of pathogenesis and novel therapeutic strategies
              </a>
            </td>
          <td>
            Christopher S Nabel, Yin P. Hung, Brian Do, Xing-Lei Qin, Cameron S. Fraser, Yolonda L. Colson, Michael Lanuti, Uma Sachdeva, Cameron Wright, Kris Sarosiek
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Pluripotent stem cells (PSCs) possess the extraordinary capability to differentiate into a variety of cell types. This capability is tightly regulated by epigenetic mechanisms, particularly histone modifications. Moreover, the reprogramming of somatic or fate-committed cells into induced pluripotent stem cells (iPSCs) largely relies on these modifications, such as histone methylation and acetylation of histones. While extensive research has been conducted utilizing mouse models, the significance of histone modifications in human iPSCs is gaining increasing recognition. Recent studies underscore the importance of epigenetic regulators in both the reprogramming process and the regulation of cancer stem cells (CSCs), which are pivotal in tumor initiation and the development of treatment resistance. This review elucidates the dynamic alterations in histone modifications that impact reprogramming and emphasizes the necessity for a balance between activating and repressive marks. These epigenetic marks are influenced by enzymes such as DNA methyltransferases (DNMTs) and histone deacetylases (HDACs). Furthermore, this review explores therapeutic strategies aimed at targeting these epigenetic modifications to enhance treatment efficacy in cancer while advancing the understanding of pluripotency and reprogramming. Despite promising developments in the creation of inhibitors for histone-modifying enzymes, challenges such as selectivity and therapy resistance continue to pose significant hurdles. Therefore, future endeavors must prioritize biomarker-driven approaches and gene-editing technologies to optimize the efficacy of epigenetic therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d6f9fda52e0528a1ec82918f3f164c54bd8aa69" target='_blank'>
              Epigenetic regulation of reprogramming and pluripotency: insights from histone modifications and their implications for cancer stem cell therapies
              </a>
            </td>
          <td>
            Woori Bae, Eun A. Ra, Myon Hee Lee
          </td>
          <td>2025-03-03</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1767b38084076cdfaf10fe2b85fb296d8eecabbf" target='_blank'>
              Abstract B021: Saturation mutational scanning uncovers druggability of all FGFR kinase point mutations
              </a>
            </td>
          <td>
            Carla Tangermann, Avantika Ghosh, Martin Ziegler, Francesco Facchinetti, Jannis Stappenbeck, M. Riester, Luise Carmina Viardot, Tobias Zundel, L. Friboulet, A. Hollebecque, José Naveja Romero, Angela Wanninger, Maria Elena Hess, Tilman Brummer, Melanie Börries, Sonja Loges, Yohann Loriot, Lena Illert, Sven Diedrichs
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b9087cd192ee1d68f582f0206ef3d120de20494" target='_blank'>
              CRISPR/Cas9-mediated editing of the ADE2 locus in Saccharomyces cerevisiae: targeted mutagenesis and homology-directed repair
              </a>
            </td>
          <td>
            Essa Jarra
          </td>
          <td>2025-04-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="ABSTRACT We previously developed a genome engineering method (TAQing2.0) based on the direct delivery of DNA endonucleases into living cells, which induces genome rearrangements even in non‐sporulating nonconventional yeasts without introducing foreign DNA. Using TAQing2.0 and conventional mutagenesis (by nitrosoguanidine), we obtained mutant asexual Candida utilis strains capable of growing under highly acidic conditions (pH 1.8). Whole genome resequencing revealed that the genomic sequences of mutants generated by both methods contain a negligible small population of unmappable sequences, suggesting that both types of mutants can be regarded as equivalent to naturally occurring mutants. TAQing2.0 mutants exhibit multiple genome rearrangements with few point mutations, whereas conventional mutagenesis produces numerous point mutations. This feature enabled us to easily identify candidate genes (e.g., LYP 1 homolog) responsible for acid resistance. TAQing2.0 is a powerful and versatile tool for mutant production and gene hunting without invasion of foreign DNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a550fa9f1db57985ff3e541281a27ba3e3b9828" target='_blank'>
              Advantages of Mutant Generation by Genome Rearrangements of Non‐Conventional Yeast via Direct Nuclease Transfection
              </a>
            </td>
          <td>
            Arisa H. Oda, Taishi Yasukawa, Miki Tamura, Ayumu Sano, Naohisa Masuo, Kunihiro Ohta
          </td>
          <td>2025-03-01</td>
          <td>Genes to Cells</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6192f93a04789d7b2aabf4cf16ca2d7d42091ce4" target='_blank'>
              Innovative CRISPR-Cas9 enhances cell rejuvenation, addressing disease
              </a>
            </td>
          <td>
            Linfeng Zhang
          </td>
          <td>2025-03-20</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer development is characterized by chromosomal instability, manifesting in frequent occurrences of different genomic alteration mechanisms ranging in extent and impact. Mathematical modeling can help evaluate the role of each mutational process during tumor progression, however existing frameworks can only capture certain aspects of chromosomal instability (CIN). We present CINner, a mathematical framework for modeling genomic diversity and selection during tumor evolution. The main advantage of CINner is its flexibility to incorporate many genomic events that directly impact cellular fitness, from driver gene mutations to copy number alterations (CNAs), including focal amplifications and deletions, missegregations and whole-genome duplication (WGD). We apply CINner to find chromosome-arm selection parameters that drive tumorigenesis in the absence of WGD in chromosomally stable cancer types from the Pan-Cancer Analysis of Whole Genomes (PCAWG, n=718). We found that the selection parameters predict WGD prevalence among different chromosomally unstable tumors, hinting that the selective advantage of WGD cells hinges on their tolerance for aneuploidy and escape from nullisomy. Analysis of inference results using CINner across cancer types in The Cancer Genome Atlas (n=8207) further reveals that the inferred selection parameters reflect the bias between tumor suppressor genes and oncogenes on specific genomic regions. Direct application of CINner to model the WGD proportion and fraction of genome altered (FGA) in PCAWG uncovers the increase in CNA probabilities associated with WGD in each cancer type. CINner can also be utilized to study chromosomally stable cancer types, by applying a selection model based on driver gene mutations and focal amplifications or deletions (chronic lymphocytic leukemia in PCAWG, n=95). Finally, we used CINner to analyze the impact of CNA probabilities, chromosome selection parameters, tumor growth dynamics and population size on cancer fitness and heterogeneity. We expect that CINner will provide a powerful modeling tool for the oncology community to quantify the impact of newly uncovered genomic alteration mechanisms on shaping tumor progression and adaptation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a271cd04acf8979b5f4cf725c9c1e047db0d2529" target='_blank'>
              CINner: Modeling and simulation of chromosomal instability in cancer at single-cell resolution
              </a>
            </td>
          <td>
            K. Dinh, I. Vázquez-García, Andrew Chan, Rhea Malhotra, A. Weiner, Andrew McPherson, Simon Tavaré
          </td>
          <td>2025-04-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b45b1b9cc6476f30f9e99c9947f2d78f83ee6b9" target='_blank'>
              Crispr Chronicles: Pioneering Gene Editing in Cardiovascular Therapy
              </a>
            </td>
          <td>
            Yasmeen Yasmeen, Thota Vaishnavi
          </td>
          <td>2025-04-11</td>
          <td>International Journal of Current Science Research and Review</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f250f1f9ca61e747d3f45521c173e479dc5a6f63" target='_blank'>
              Chemical enhancement of DNA repair in aging
              </a>
            </td>
          <td>
            J. C. Macedo, Maria M da Silva, Joana M. Magalhães, Carlos Sousa-Soares, M. Inês Ala, Mafalda Galhardo, Rui Ribeiro, Monika Barroso-Vilares, Mafalda Sousa, Paula Sampaio, E. Logarinho
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="The expression of cytosolic aldehyde dehydrogenases (ALDHs), which mediate the last step in the pathway of the synthesis of all-trans retinoic acid, is dysregulated in various types of human cancer, and has been associated with the development of cancer stem cells (CSCs) in solid tumors and hematological malignancies. CSCs are considered a minor fraction of cancer cells with the capacity to initiate neoplastic tumors. ALDH1A1 serves a crucial role in the emergence of the CSC phenotype, induces the malignant behavior of cancer cells and promotes treatment resistance. Notably, ALDH1A1-induced therapy resistance is not exclusive to just one group of drugs, but affects diverse types of drugs that use different mechanisms to kill cells. This diversity of drug resistance-inducing effects is associated with the stemness-supporting functions of ALDH1A1. The inhibition of ALDH1A1 activity using chemicals or the depletion of ALDH1A1 via genetic approaches, such as the use of small interfering RNA, can overcome diverse pathways of therapy resistance. In the context of breast cancer, it is critical that only a fraction of malignant cells are expected to manifest stem-like features, which include increased expression of ALDH1A1. From the angle of disease prognosis, the extent of the association of ALDH1A1 with increased malignant behavior and drug resistance remains to be determined through the application of cutting-edge methods that detect the expression of tracked biomarkers within tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/378f86e62d74f35111b7a740e827c0ac03143fb8" target='_blank'>
              ALDH1A1 in breast cancer: A prospective target to overcome therapy resistance (Review)
              </a>
            </td>
          <td>
            L. Varışlı, P. Zoumpourlis, Demetrios A. Spandidos, V. Zoumpourlis, S. Vlahopoulos
          </td>
          <td>2025-03-04</td>
          <td>Oncology Letters</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Abstract The dual genetic control of mitochondrial respiratory function, combined with the high mutation rate of the mitochondrial genome (mtDNA), makes mitochondrial diseases among the most frequent genetic diseases in humans (1 in 5,000 in adults). With no effective treatments available, gene therapy approaches have been proposed. Notably, several studies have demonstrated the potential for nuclear expression of a healthy copy of a dysfunctional mitochondrial gene, referred to as allotopic expression, to help recover respiratory function. However, allotopic expression conditions require significant optimization. We harnessed engineering biology tools to improve the allotopic expression of the COX2-W56R gene in the budding yeast Saccharomyces cerevisiae. Through conducting random mutagenesis and screening of the impact of vector copy number, promoter, and mitochondrial targeting sequence, we substantially increased the mitochondrial incorporation of the allotopic protein and significantly increased recovery of mitochondrial respiration. Moreover, CN-PAGE analyses revealed that our optimized allotopic protein does not impact cytochrome c oxidase assembly, or the biogenesis of respiratory chain supercomplexes. Importantly, the most beneficial amino acid substitutions found in the second transmembrane helix (L93S and I102K) are conserved residues in the corresponding positions of human MT-CO2 (L73 and L75), and we propose that mirroring these changes could potentially help improve allotopic Cox2p expression in human cells. To conclude, this study demonstrates the effectiveness of using engineering biology approaches to optimise allotopic expression of mitochondrial genes in the baker's yeast.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71b719879ddbf80db7c9d88a52dbf9b4791e0038" target='_blank'>
              A new set of mutations in the second transmembrane helix of the Cox2p-W56R substantially improves its allotopic expression in Saccharomyces cerevisiae
              </a>
            </td>
          <td>
            Kewin Gombeau, Stefan A Hoffmann, Yizhi Cai
          </td>
          <td>2025-04-01</td>
          <td>Genetics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Background Allogeneic double-negative T-cell (DNT) therapy has emerged as a novel, off-the-shelf cellular treatment with clinical feasibility, safety, and promising efficacy against leukemia. However, the biology of DNTs is less well characterized, and how DNT therapy distinguishes from conventional γδ T-cell therapy remains unclear. Collectively, this hinders our ability to bolster DNT functionalities in cancer therapy. Here, we performed single-cell RNA sequencing with in vitro and in vivo functional analysis on DNTs. As a significant proportion of DNTs express Vγ9Vδ2 (Vδ2) TCR chain, we compared DNTs with donor-matched conventional Vδ2 T cells expanded with zoledronic acid. Methods Healthy donor-derived allogeneic DNTs and Vδ2 T cells were expanded ex vivo. Single-cell RNA sequencing analysis was performed on both cellular products to identify the transcriptional landscape and inferred cellular interactions within DNTs, followed by comparisons with donor-matched Vδ2 T cells. Unique cellular subsets found only in DNTs were depleted to identify their contributions to the overall efficacy of DNTs against acute myeloid leukemia. The anti-leukemic activity and in vivo persistence of DNTs and Vδ2 T-cells were explored using flow cytometry-based cytotoxicity assays, memory phenotyping, and xenograft models. Results Despite a shared Vδ2 expression between cellular products, we identified unique cellular compositions in DNTs that contribute to distinct transcriptional and cellular communication patterns relative to the donor-matched Vδ2 T cells, including higher expression of genes identified in chimeric antigen receptor T cells that persist in patients with durable cancer-remission. Vδ2– DNTs exhibited strong persistence characteristics, and their presence promoted the cytotoxic capabilities of Vδ2+ DNTs in repeated stimulation assays. This unique genetic signature and diverse cellular composition of DNTs resulted in better overall ex vivo expansion, prolonged persistence, and superior anti-leukemic activity compared with Vδ2 T cells in vitro and in vivo. Conclusions These results highlight the unique transcriptional, cellular, and functional profile of human DNTs and support the continued clinical investigation of allogeneic DNT therapy. The data also provide a reference gene signature that may help improve the efficacy of other types of allogeneic adoptive cellular therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d868758cb2da0c02a12654ab175d0d11e3cbc21" target='_blank'>
              Single-cell RNA sequencing of human double-negative T cells reveals a favorable cellular signature for cancer therapy
              </a>
            </td>
          <td>
            Enoch Tin, Ismat Khatri, Karen Fang, Yoosu Na, Michele Nawata, Juan Arteaga, Mark D. Minden, S. Rutella, JongBok Lee, Li Zhang
          </td>
          <td>2025-04-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Individual aging is a complex biological process involving multiple levels, with molecular changes existing in heterogeneity across different cell types and tissues, being regulated by both internal and external factors. Traditional senescence markers, including p16, cell morphological changes, and cell cycle arrest, can only partially reflect the complexity of senescence. Single-cell omics technology facilitates the integration of multi-faceted data, including gene expression profiles, spatial dynamics, chromatin accessibility and metabolic pathways. This comprehensive approach enhances the development of biomarkers, granting us a more profound insight into the heterogeneity inherent within senescent cell populations. In this review, we summarize the application of single cell multi-omics approaches in analyzing senescence mechanisms and potential intervention targets from the perspectives of transcriptomics, epigenetics, metabolomics, and proteomics, explore the potential of developing new senescence markers at the cellular level using machine learning algorithms and artificial intelligence in bioinformatics analysis. Finally, we further discuss the challenges and prospective trajectories within this research domain to provide a more comprehensive perspective on dissecting the regulatory networks of senescence cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb05feca31b360e2a4184cb1c6c1adbe6ecbd8d5" target='_blank'>
              Towards Precision Aging Biology: Single-Cell Multi-Omics and Advanced AI-Driven Strategies.
              </a>
            </td>
          <td>
            Sijia Xie, Xinwei Luo, Feitong Hong, Yijie Wei, Yuduo Hao, Xueqin Xie, Xiaolong Li, Guangbo Xie, Fuying Dao, Hao Lyu
          </td>
          <td>2025-03-17</td>
          <td>Aging and disease</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bab1918e950670abacd73729f396666b45fba3c8" target='_blank'>
              Optical genome mapping enables accurate testing of large repeat expansions.
              </a>
            </td>
          <td>
            B. van der Sanden, K. Neveling, Syukri Shukor, Michael D Gallagher, Joyce Lee, Stephanie L Burke, M. Pennings, R. van Beek, M. Oorsprong, E. Kater‐Baats, E. Kamping, Alide A Tieleman, N. Voermans, Ingrid E. Scheffer, Jozef Gécz, M. Corbett, L. Vissers, A. Pang, Alex Hastie, E. Kamsteeg, A. Hoischen
          </td>
          <td>2025-03-20</td>
          <td>Genome research</td>
          <td>1</td>
          <td>44</td>
        </tr>

        <tr id="Abstract Gene editing technology involves modifying target genes to alter genetic traits and generate new phenotypes. Beginning with zinc-finger nucleases (ZFN) and transcription activator-like effector nucleases (TALEN), the field has evolved through the advent of clustered regularly interspaced short palindromic repeats and CRISPR-associated protein (CRISPR-Cas) systems, and more recently to base editors (BE) and prime editors (PE). These innovations have provided deep insights into the molecular mechanisms of complex biological processes and have paved the way for novel therapeutic strategies for a range of diseases. Gene editing is now being applied in the treatment of both genetic and acquired kidney diseases, as well as in kidney transplantation and the correction of genetic mutations. This review explores the current applications of mainstream gene editing technologies in biology, with a particular emphasis on their roles in kidney disease research and treatment of. It also addresses the limitations and challenges associated with these technologies, while offering perspectives on their future potential in this field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af39194bb98716939bc4ae876ce4da54d1f76d7b" target='_blank'>
              Gene Editing: An Effective Tool for the Future Treatment of Kidney Disease
              </a>
            </td>
          <td>
            Meiling Cao, Rui-Yi Han, Si-Da Chen, Danyang Zhao, Mingyue Shi, J. Zou, Lei Li, Hongkun Jiang
          </td>
          <td>2025-03-01</td>
          <td>Journal of Inflammation Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Gastric cancer (GC) remains a global health challenge due to its high mortality rate and the lack of specific diagnostic methods. Tumor heterogeneity significantly hinders effective treatment, necessitating advanced techniques to dissect its complexity. Artificial intelligence (AI) offers transformative potential in uncovering intricate cellular dynamics and gene regulatory networks. This study leverages single-cell RNA sequencing (scRNA-seq) combined with transcriptome analysis, enhanced by AI-driven analytics, to explore the tumor microenvironment and identify novel prognostic markers and therapeutic targets in GC. Methods scRNA-seq and transcriptome datasets of GC patients were analyzed using AI-enhanced methodologies to unravel tumor heterogeneity and microenvironmental dynamics. Macrophage subsets were identified as critical components within the GC microenvironment. High-variance gene screening in these subsets pinpointed apolipoprotein E (APOE) as a hub gene. Experimental validation of APOE expression in GC samples and functional studies in GC cell lines were conducted. Results Bioinformatics and AI-enabled analyses confirmed the immunosuppressive role of APOE in GC. An immune-related survival model was developed to predict immunotherapy responses and patient prognoses. Mechanistically, APOE was found to induce immunosuppression through M2 macrophage polarization, promoting tumor progression and leading to poorer outcomes in GC patients. Conclusion This study highlights the potential of AI-driven approaches in elucidating the role of APOE in GC progression. APOE’s regulatory effects on M2 macrophages underscore its value as a prognostic marker and therapeutic target, paving the way for precision medicine in GC management. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-02100-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/987e5761ad8244faf71073710e21f1e10a5477f1" target='_blank'>
              Artificial intelligence-driven integration of single-cell RNA sequencing and transcriptome analysis to decipher APOE’s role in gastric cancer prognosis and therapy
              </a>
            </td>
          <td>
            Ni Li, Jinquan Lin, Chunyu Wang, Xiao Ran, Zhi Zhao
          </td>
          <td>2025-03-13</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af5f5ff700aa0f25c87024a04fa36fa40cb85cb4" target='_blank'>
              Abstract A055: Dissecting mechanisms of tumor anti-PD1 resistance due to host genetic variation within ADAM17
              </a>
            </td>
          <td>
            James P Heiserman, R. Akhurst, Tanuja Desai, Sharon Chen
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdb015a26b60d1395883653385851a0c2cbe11b8" target='_blank'>
              Abstract IA02: Developing and manufacturing next generation cell therapies in the academic setting
              </a>
            </td>
          <td>
            B. Shy
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c906c5a3862aab5cd84c1716303e1ffc00c0344c" target='_blank'>
              NSD3: A Promising Target for Cancer Therapy.
              </a>
            </td>
          <td>
            Ting Huang, Bowen Zhang, Yifan Yang, Qiong Lin, Genbao Shao
          </td>
          <td>2025-03-26</td>
          <td>Cell biochemistry and function</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background The tumor suppressor p53 (Trp53), also known as p53, is the most commonly mutated gene in cancer. Canonical p53 DNA damage response pathways are well characterized and classically thought to underlie the tumor suppressive effect of p53. Challenging this dogma, mouse models have revealed that p53-driven apoptosis and cell cycle arrest are dispensable for tumor suppression. Here, we investigated the inverse context of a p53 mutation predicted to drive the expression of canonical targets but is detected in human cancer. Methods We established a novel mouse model with a single base pair mutation (GAG>GAT, p53E221D) in the DNA-Binding domain that has wild-type function in screening assays, but is paradoxically found in human cancer in Li-Fraumeni syndrome. Using mouse p53E221D and the analogous human p53E224D mutants, we evaluated expression, transcriptional activation, and tumor suppression in vitro and in vivo. Results Expression of human p53E224D from cDNA translated to a fully functional p53 protein. However, p53E221D/E221D RNA transcribed from the endogenous locus is mis-spliced resulting in nonsense-mediated decay. Moreover, fibroblasts derived from p53E221D/E221D mice do not express a detectable protein product. Mice homozygous for p53E221D exhibited increased tumor penetrance and decreased life expectancy compared to p53WT/WT animals. Conclusions Mouse p53E221D and human p53E224D mutations lead to splice variation and a biologically relevant p53 loss of function in vitro and in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97a5ab888592bac1b40fc9b15ac14955958bd526" target='_blank'>
              Mis-splicing drives loss of function of p53E224D point mutation
              </a>
            </td>
          <td>
            Ian C. Lock, Nathan H Leisenring, Warren Floyd, Eric S. Xu, Lixia Luo, Yan Ma, Erin C. Mansell, Diana M. Cardona, Chang-Lung Lee, D. Kirsch
          </td>
          <td>2025-03-05</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6f03a05980baac4dbe599f109d605f3d1cea815" target='_blank'>
              EcDNA-borne PVT1 fusion stabilizes oncogenic mRNAs
              </a>
            </td>
          <td>
            Hyerim Yi, Shu Zhang, Jason Swinderman, Yanbo Wang, Vishnupriya Kanakaveti, King L. Hung, I. Wong, Suhas Srinivasan, Ellis J Curtis, Aarohi Bhargava-Shah, Rui Li, Matthew G Jones, J. Luebeck, Yanding Zhao, J. Belk, Katerina Kraft, Quanming Shi, Xiaowei Yan, Simon K. Pritchard, Frances M. Liang, D. Felsher, Luke A. Gilbert, V. Bafna, P. Mischel, Howard Y. Chang
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09704648420ca313fc3a94dc7cc40706df2fbbe8" target='_blank'>
              Blood-Based Epigenetic Instability Linked to Human Aging and Disease
              </a>
            </td>
          <td>
            Salman Basrai, Ido Nofech-Mozes, Rajesh Detroja, Fernando L. Scolari, Mehran Bakhtiari, Andrea Arruda, Tracy Murphy, Scott V. Bratman, Steven M. Chan, Mark D. Minden, Jae-Sook Ahn, D. D. H. Kim, R. Kridel, Filio Billia, Sagi Abelson
          </td>
          <td>2025-03-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Epigenomic modifications—such as DNA methylation, histone acetylation, and histone methylation—and their implications in tumorigenesis, progression, and treatment have emerged as a pivotal field in cancer research. Tumors undergo metabolic reprogramming to sustain proliferation and metastasis in nutrient-deficient conditions, while suppressing anti-tumor immunity in the tumor microenvironment (TME). Concurrently, immune cells within the immunosuppressive TME undergo metabolic adaptations, leading to alterations in their immune function. The complicated interplay between metabolites and epigenomic modulation has spotlighted the significance of epigenomic regulation in tumor immunometabolism. In this review, characteristics of the epigenomic modification associated with tumors are systematically summarized alongside with their regulatory roles in tumor metabolic reprogramming and immunometabolism. Classical and emerging approaches are delineated to broaden the boundaries of research on the crosstalk research on the crosstalk between tumor immunometabolism and epigenomics. Furthermore, we discuss potential therapeutic strategies that target tumor immunometabolism to modulate epigenomic modifications, highlighting the burgeoning synergy between metabolic therapies and immunotherapy as a promising avenue for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b7b18f960568f0266743f9b084b520cf67c3989" target='_blank'>
              Bridging epigenomics and tumor immunometabolism: molecular mechanisms and therapeutic implications
              </a>
            </td>
          <td>
            Xiaowen Xie, Weici Liu, Zhiyuan Yuan, Hanqing Chen, Wenjun Mao
          </td>
          <td>2025-03-08</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH), characterized by the expansion of hematopoietic stem and progenitor cells harboring somatic mutations, has emerged as a significant age-related phenomenon with profound implications for human health. While initially recognized in the 1960s, recent technological advances have revealed its complex nature and widespread prevalence, affecting up to 84% of individuals aged ≥ 70 years. The clinical significance of CH extends beyond its well-established role as a precursor to hematological malignancies, encompassing its association with cardiovascular diseases, chronic kidney disease, and other non-malignant disorders. This comprehensive review synthesizes the current understanding of CH, focusing on recent advances in genetic and molecular mechanisms, particularly the roles of commonly mutated genes such as DNMT3A, TET2, and ASXL1. We address the emerging distinction between myeloid and lymphoid CH, their differential impacts on disease progression, and the complex interplay between CH and inflammation. Special attention is given to newly identified genetic determinants of clonal expansion rates and their implications for disease progression. The review also examines the revolutionary concept of passenger-approximated clonal expansion rate and its utility in understanding CH dynamics. Furthermore, we discuss therapeutic strategies targeting inflammatory pathways and their potential in mitigating CH-associated complications. By integrating recent findings from genetic, molecular, and clinical studies, this review provides a framework for understanding CH as a systemic condition and highlights promising directions for therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7375f0846811e9250a784d2b67a30ac0f6b0da37" target='_blank'>
              Clonal hematopoiesis: elements associated with clonal expansion and diseases
              </a>
            </td>
          <td>
            Gangpyo Ryu, Youngil Koh, S. Jaiswal, S. Yoon
          </td>
          <td>2025-03-13</td>
          <td>Blood Research</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eed8e001dc646667003aedc35613a33f0517e9a4" target='_blank'>
              Common variation in meiosis genes shapes human recombination phenotypes and aneuploidy risk
              </a>
            </td>
          <td>
            S. A. Carioscia, A. Biddanda, M. R. Starostik, X. Tang, E. R. Hoffmann, Z. P. Demko, R. McCoy
          </td>
          <td>2025-04-04</td>
          <td>None</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Abstract For genome editing, the use of CRISPR ribonucleoprotein (RNP) complexes is well established and often the superior choice over plasmid-based or viral strategies. RNPs containing dCas9 fusion proteins, which enable the targeted manipulation of transcriptomes and epigenomes, remain significantly less accessible. Here, we describe the production, delivery, and optimization of second generation CRISPRa RNPs (dRNPs). We characterize the transcriptional and cellular consequences of dRNP treatments in a variety of human target cells and show that the uptake is very efficient. The targeted activation of genes demonstrates remarkable potency, even for genes that are strongly silenced, such as developmental master transcription factors. In contrast to DNA-based CRISPRa strategies, gene activation is immediate and characterized by a sharp temporal precision. We also show that dRNPs allow very high-target multiplexing, enabling undiminished gene activation of multiple genes simultaneously. Applying these insights, we find that intensive target multiplexing at single promoters synergistically elevates gene transcription. Finally, we demonstrate in human stem and differentiated cells that the preferable features of dRNPs allow to instruct and convert cell fates efficiently without the need for DNA delivery or viral vectors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ae64466d50f495675fc82663a1b7dac6851bcc6" target='_blank'>
              Efficient DNA- and virus-free engineering of cellular transcriptomic states using dCas9 ribonucleoprotein (dRNP) complexes
              </a>
            </td>
          <td>
            Tobias Schmidt, Maximilian F. Wiesbeck, Luisa Eg er t, Thi-T ram T ruong, Anna Danese, Lukas Voshagen, Simon Imhof, Matilde Iraci Borgia, Deeksha, Andrea M Neuner, Anna Köf er le, Ar ie Geer lof, André Santos, Dias Mourão, Stefan H Stricker
          </td>
          <td>2025-03-20</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Alternative Lengthening of Telomeres (ALT) is a telomerase-independent mechanism deployed by several aggressive cancers to maintain telomere length. This contributes to their malignancy and resistance to conventional therapies. In prior studies, we have identified key proteins linked to the ALT process using multi-omic data integration strategies. In this work, we combined metabolomic datasets with our earlier results to identify targetable metabolic pathways for ALT-positive tumors. 39 ALT-related proteins were found to interact with 42 different metabolites in our analysis. Additional networking analysis revealed a complex interaction between metabolites and ALT-related proteins, suggesting that pan-cancer oncogenes may have an impact on these pathways. Three metabolic pathways have been primarily related with the ALT mechanism: purine metabolism, cysteine and methionine metabolism, and nicotinate and nicotinamide metabolism. Lastly, we prioritized FDA-approved drugs (azathioprine, thioguanine, and mercaptopurine) that could target ALT-positive tumors through purine metabolism. This work provides a wide perspective of the metabolomic pathways associated with ALT and reveals potential therapeutic targets that require further experimental validation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef209c063fddf47ab0ca2823c1da99d2d523acd4" target='_blank'>
              Metabolic pathways of Alternative Lengthening of Telomeres in pan-carcinoma
              </a>
            </td>
          <td>
            I. Armendáriz-Castillo, J. García-Cárdenas, Pamela Espinosa, Katherine Hidalgo-Fernández, Lizbeth Peña-Zúñiga, Ronie Martínez, Juan Moromenacho, Andrés Herrera-Yela, Jonathan Cruz-Varela, Anilú Saucedo-Sariñana, María-Esperanza Cerdán, A. López-Cortés, Santiago Guerrero
          </td>
          <td>2025-02-21</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a429de2819c4c4deeab83af19866cf4b65fdd75e" target='_blank'>
              Recent Progress and Potential of G4 Ligands in Cancer Immunotherapy
              </a>
            </td>
          <td>
            Jiahui Lin, Zhu Gong, Yingyue Lu, Jiongheng Cai, Junjie Zhang, Jiaheng Tan, Zhishu Huang, Shuobin Chen
          </td>
          <td>2025-04-17</td>
          <td>Molecules</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="MYC-driven (MYC+) cancers are aggressive and often fatal. MYC dysregulation is a key event in these cancers, but overexpression of MYC alone is not always enough to cause cancer. Plasmocytoma Variant Translocation 1 (PVT1), a long non-coding RNA (lncRNA) adjacent to MYC on chromosome 8 is a rearrangement hotspot in many MYC+ cancers. In addition to being co-amplified with MYC, the genomic rearrangement at PVT1 involves translocation, which has had obscure functional consequences. We report that translocation at the PVT1 locus cause asymmetric enrichment of 5’-PVT1 and loss of 3’-PVT1. Despite being classified as a non-coding RNA, the retained 5’ region of PVT1 generates a circular RNA (CircPVT1) that codes for the novel peptide we call Firefox (FFX). FFX augments AKT signaling and synergistically activates MYC and mTORC1 in these cells. Further, the 3’ end of PVT1, which is lost during the translocation, codes for a tumor-suppressing micropeptide we named as Honeybadger (HNB). We demonstrate that HNB interacts with KRAS and disrupts the activation of KRAS effectors. Loss of HNB leads to activation of RAS/MAPK signaling pathway, and enhances MYC stability by promoting phosphorylation of MYC at Ser62. These findings identify PVT1 as a critical node that synchronizes MYC, AKT, and RAS-MAPK activities in cancer. Our study thus identifies a key mechanism by which rearrangements at the PVT1 locus activate additional oncogenic pathways that synergize with MYC to exacerbate the aggressiveness of MYC+ cancers. This newfound understanding explains the poor prognosis associated with MYC+ cancers and offers potential therapeutic targets that could be leveraged in treatment strategies for these cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e076955c4c71d3d95422db7f1fcfc733c5d61e3" target='_blank'>
              Synergistic RAS-MAPK and AKT Activation in MYC-Driven Tumors via Adjacent PVT1 Rearrangements
              </a>
            </td>
          <td>
            Ashutosh Tiwari, Utkarsha Paithane, Jordan Friedlein, Kojiro Tashiro, Olivier Saulnier, Karina Barbosa, Quang Trinh, Bryan Hall, Shrawantee Saha, Aditi Soni, Takuma Nakashima, Andrey Bobkov, Lynn Fujimoto, Rabi Murad, Svetlana Maurya, Mayank Saraswat, Shahab Sarmashghi, Joshua T. Lange, Sihan Wu, M. Masihi, Srija Ghosh, Gazal Hemmati, Owen S. Chapman, Liam D Hendrikse, Brian James, J. Luebeck, Tanja Eisemann, Theophilos Tzaridis, Deepak Rohila, Robyn Leary, Jyotika Varshney, Badrinath R. Konety, S. Dehm, Yasuhiko Kawakami, R. Beroukhim, D. Largaespada, Lincoln Stein, Lukas Chavez, Hiromichi Suzuki, William A. Weiss, Jianhua Zhao, Aniruddha J. Deshpande, R. Wechsler-Reya, Michael D Taylor, Anindya Bagchi
          </td>
          <td>2025-02-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>148</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccad37f8db740ee741e31d76d75cb5a07f7e923f" target='_blank'>
              Biology as vulnerability in follicular lymphoma: genetics, epigenetics, and immunogenetics.
              </a>
            </td>
          <td>
            F. Araujo-Ayala, W. Béguelin
          </td>
          <td>2025-03-16</td>
          <td>Blood</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Rare monogenic disorders are caused by mutations in single genes and have an incidence rate of less than 0.5%. Due to their low prevalence, these diseases often attract limited research and commercial interest, leading to significant unmet medical needs. In a therapeutic landscape where treatments are targeted to manage symptoms, gene editing therapy emerges as a promising approach to craft curative and lasting treatments for these patients, often referred to as “one-and-done” therapeutics. CRISPR-dependent base editing enables the precise correction of genetic mutations by direct modification of DNA bases without creating potentially deleterious DNA double-strand breaks. Base editors combine a nickase version of Cas9 with cytosine or adenine deaminases to convert C·G to T·A and A·T to G·C, respectively. Together, cytosine (CBE) and adenine (ABE) base editors can theoretically correct ∼95% of pathogenic transition mutations cataloged in ClinVar. This mini-review explores the application of base editing as a therapeutic approach for rare monogenic disorders. It provides an overview of the state of gene therapies and a comprehensive compilation of preclinical studies using base editing to treat rare monogenic disorders. Key considerations for designing base editing-driven therapeutics are summarized in a user-friendly guide for researchers interested in applying this technology to a specific rare monogenic disorder. Finally, we discuss the prospects and challenges for bench-to-bedside translation of base editing therapies for rare monogenic disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50941b3a230c6aba4e286e74fa37535f7cdb8846" target='_blank'>
              CRISPR-dependent base editing as a therapeutic strategy for rare monogenic disorders
              </a>
            </td>
          <td>
            Júlia-Jié Cabré-Romans, Raquel Cuella-Martin
          </td>
          <td>2025-04-02</td>
          <td>Frontiers in Genome Editing</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="In classic Hodgkin-lymphoma (cHL), only a few cases recur, and only a limited fraction of patients is primary-refractory to standard-polychemotherapy. Underlying genomic features of unfavorable clinical courses remain sparsely characterized. Here, we investigated the genomic characteristics of primary-refractory/relapsed cHL in contrast with responders. Therefore, ultra-deep next-generation panel-sequencing was performed on a total of 59 FFPE-samples (20 responders, 26 relapsed (rHL: 11 initial-diagnosis, 15 relapse) and 13 primary-refractory (prHL: 8 initial-diagnosis, 5 progression) from 44 cHL-patients applying a hybrid-capture approach. We compared samples associated with distinct disease courses concerning their oncogenic drivers, mutational signatures, and perturbed pathways. Compared to responders, mutations in genes such as PMS2, PDGFRB, KAT6A, EPHB1, and HGF were detected more frequently in prHL/rHL. Additionally, we observed that in rHL or prHL, BARD1-mutations occur, whereas ETV1, NF1, and MET-mutations were eliminated through clonal selection. A significant enrichment of non-synonymous variants was detected in prHL compared to responders and a significant selection process in favor of NOTCH-pathway mutations driving rHL or prHL was observed. However, our analysis revealed a negative selection process for non-synonymous variants affecting the hippo-pathway. This study delineates distinct mutational signatures between responders and rHL/prHL, whilst illustrating longitudinal dynamics in mutational profiles using paired samples. Further, several exploitable therapeutic vulnerabilities for rHL and prHL were identified.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e9cb6223e119f0dfa395cc6c6b37f41897377e2" target='_blank'>
              The mutational landscape and its longitudinal dynamics in relapsed and refractory classic Hodgkin lymphoma.
              </a>
            </td>
          <td>
            H. Witte, A. Künstner, Thomas Hahn, V. Bernard, S. Stölting, Kathrin Kusch, Kumar Nagarathinam, Cyrus Khandanpour, N. von Bubnoff, Arthur Bauer, Michael Grunert, Svenja Hartung, Annette Arndt, Konrad Steinestel, Hartmut Merz, Hauke Busch, Alfred C. Feller, N. Gebauer
          </td>
          <td>2025-02-24</td>
          <td>Annals of hematology</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/408d0e46d8aefed4371316d279839ada80b1b263" target='_blank'>
              Abstract A006: Differential therapeutic vulnerabilities and opportunities for smokers and never-smokers based on physiological and adaptive landscapes of lung adenocarcinoma
              </a>
            </td>
          <td>
            K. Dasari, Jorge A. Alfaro-Murillo, Jeffrey P. Townsend
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Cell division cycle 25 (CDC25) phosphatases serve as crucial regulators of cell cycle phase transitions and essential components of the checkpoint machinery involved in DNA damage response. Emerging evidence indicates the oncogenic potential of CDC25 family members across various cancers. However, comprehensive insights into the expression pattern and function of the CDC25 family in diverse cancers remain unexplored. In our study, we investigated CDC25 family using multiple databases, including gene expression levels, molecular signatures, diagnosis value, and prognostic value in pan-cancer. Furthermore, we focused on melanoma and systematically explored CDC25A expression and its clinical correlations. As a result, the expression of CDC25 family members is significantly abnormal in most cancers, correlating with poorer prognosis. CDC25 family members are differently regulated by DNA methylation and genetic alterations across various cancers. In addition, CDC25 family plays a critical role in the malignant progression of melanoma. Functional investigation reveals that CDC25A inhibition suppresses the proliferation of melanoma cells and sensitizes melanoma cells to chemotherapy and NK cell therapy. In conclusion, our study suggests that CDC25 family may serve as a significant biomarker for diagnosis and prognosis across multiple cancers, with CDC25A as a promising therapeutic target for melanoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac2d9665c88d909bf6dd5f9c6702e4a774f6a7f4" target='_blank'>
              CDC25A inhibition sensitizes melanoma cells to doxorubicin and NK cell therapy
              </a>
            </td>
          <td>
            Xinyue Gao, Feichang Liu, Bo Zhang, Tianyi Ren, You Zheng, Z. Niu, He Ren, Chenyu Liu, Chengzuo Jiang, Chenxi Wang, Hongyan Huang, Li Ma, Qiang Sun
          </td>
          <td>2025-04-11</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/faa65e1fd6a4ddcb57e0eaf994498d400b2eb5d2" target='_blank'>
              Tissue-specific mutagenesis from endogenous guanine damage is suppressed by Polκ and DNA repair
              </a>
            </td>
          <td>
            Yang Jiang, M. Przybilla, Linda Bakker, Foster C. Jacobs, Dylan Mckeon, Roxanne van der Sluijs, Juliette Wezenbeek, Joeri van Strien, Jeroen Willems, Alex E E Verkennis, Jamie Barnett, Adrian Baez Ortega, Federico Abascal, Peter W. Villalta, P. Knipscheer, I. Martincorena, S. Balbo, Juan Garaycoechea
          </td>
          <td>2025-03-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="Genome-wide human genetic studies have identified inherited cis-regulatory loci variants that predispose to cancers. However, the mechanisms by which these germline variants influence cancer progression, particularly through gene expression and proteostasis control, remain unclear. By analyzing genomic data from a gastric cancer (GC) case-control study (2,117 individuals), focusing on the ubiquitin-specific protease (USP) family, we identify the SNP rs72856331 (G>A) in the promoter region of the proto-oncogene USP47 as a putative susceptibility allele for GC. Mechanistically, the risk allele G is associated with enhanced USP47 expression, mediated by altered recruitment of the transcription factor GLI3 and changes in the epigenetic status at promoter. CRISPR/Cas9-mediated single-nucleotide conversion into risk allele G results in increased GLI3 binding and subsequent USP47 upregulation. The depletion of GLI3 results in a reduction of cancer-related phenotypes, similar to those observed following USP47 knockdown. Furthermore, we identify Snai1 as a deubiquitination target of USP47, explaining USP47-dependent activation of the epithelial-mesenchymal transition pathway and tumor progression. Our findings identify an important genetic predisposition that implicates the perturbation of transcription and proteostasis programs in GC, offering insights into prevention and therapeutic strategies for genetically stratified patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc1084b39e6f7bd79c603ae5a0dd8fa214a3f635" target='_blank'>
              An inherited predisposition allele promotes gastric cancer via enhancing deubiquitination-mediated activation of epithelial-to-mesenchymal transition signaling
              </a>
            </td>
          <td>
            Bolin Tao, Zhenning Wang, Xuanyi Wang, Aixia Song, Jiaxian Liu, Jianan Wang, Qin Zhang, Zhaolin Chen, Zixian Wang, Wenjie Xu, Menghong Sun, Yanong Wang, Ping Zhang, Tao Xu, Gong-Hong Wei, F. X. Chen, Mengyun Wang
          </td>
          <td>2025-02-25</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73299a1254c80e9af5d25fe381a681e9866a35bc" target='_blank'>
              Studying the cellular efficacy and tolerability of using CRISPR-gRNA ribonucleoprotein (RNP) complex for in-vitro knockdown of TRIB2 in acute myeloid leukaemia (AML) cells as preliminary clinical evaluations.
              </a>
            </td>
          <td>
            Josephine You Pheng Tan, Ramesh Thevendran, Yek Song Quek, Solayappan Maheswaran
          </td>
          <td>2025-04-09</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Tumors exhibit highly heterogeneous genomic and transcriptomic landscapes. Characterization of this intratumor heterogeneity (ITH) and its implications for the cancer therapies has been one of the focal points of computational cancer biology in the past years. However, while ITH at the level of genomic changes, such as copy number aberrations, has been well characterized, the downstream impact of ITH on the gene regulatory landscape remains underexplored. Disruptions in gene regulatory programs can lead to emergence of more aggressive or drugresistant tumor clones. In order to better understand whether the evidence for ITH at the gene regulatory level can be inferred from scRNA-seq data we have conducted a series of experiments that explore the levels of gene regulatory network (GRN) discrepancies between the clonal sub-populations of tumor cells. In particular, we quantify the extent to which GRNs inferred from different tumor clones diverge from each other in three settings: triple negative breast cancer, colorectal cancer, and a longitudinal dataset from a patient undergoing chemotherapy for breast cancer. Our analyses indicate that a substantial (15-25%) proportion of GRN edges can be attributed to clone-specific activity. Furthermore, we observe differential network patterns across outlier transcription factors implicated in tumorogenesis and tumor progression. Our findings suggest that differential network patterns between clonal subpopulations can be reliably identified, and that such patterns offer a complementary insight that is not captured by differential gene analyses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86413a2b4bf81ee584f265b2f474b3de585e8771" target='_blank'>
              Intratumor Heterogeneity Through the Lens of Gene Regulatory Networks
              </a>
            </td>
          <td>
            Narein Rao, Nicolae Sapoval, Hamim Zafar, L. Nakhleh
          </td>
          <td>2025-04-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/055214dbdb29cbe5585cc333e89737a58f2ba504" target='_blank'>
              The Epigenetic Landscape: From Molecular Mechanisms to Biological Aging.
              </a>
            </td>
          <td>
            Rachel Evangelina, Subhashree Ganesan, Melvin George
          </td>
          <td>2025-03-17</td>
          <td>Rejuvenation research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77cf64c5137ce4675e3a801e6fb302671875782c" target='_blank'>
              Using Cancer Profiles to Identify Synthetic Lethal Therapeutic Targets and Predictive Biomarkers in Cancer Gene Dependency Data
              </a>
            </td>
          <td>
            Laurence H. Pearl, Frances M. G. Pearl
          </td>
          <td>2025-04-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e607ac4077bd0584584c78f861a3a32f882b386" target='_blank'>
              Chemical Strategies to Modulate and Manipulate RNA Epigenetic Modifications.
              </a>
            </td>
          <td>
            Liang Cheng
          </td>
          <td>2025-03-18</td>
          <td>Accounts of chemical research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eba26ff57f58d982ea6db2c2b425446ece055593" target='_blank'>
              Exploring the Prognostic and Predictive Impact of Genomic Loss of Heterozygosity and Homologous Recombination Deficiency Alterations in Patients With Metastatic Colorectal Cancer.
              </a>
            </td>
          <td>
            R. Moretto, M. Germani, M. Carullo, V. Conca, A. Minelli, M. Giordano, Rossella Bruno, D. Rossini, Eleonora Gusmaroli, M. D. De Grandis, C. Antoniotti, L. Salvatore, A. Passardi, S. Tamberi, M. Scartozzi, F. Pietrantonio, S. Lonardi, Clara Ugolini, G. Masi, C. Cremolini
          </td>
          <td>2025-04-01</td>
          <td>JCO precision oncology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91ba15cf2f6390a1ec99834e397f27bbbddac7ca" target='_blank'>
              PFDN1 silencing disrupts critical cancer pathways in triple-negative breast cancer: investigating migration, cell cycle, and apoptosis as a new target therapy.
              </a>
            </td>
          <td>
            Abdulrahman M Alokda, A. Soffar, A. I. Yousef, Fawziya A R Ibrahim, Tarek S El-Sewedy, Alaa Elmetwalli
          </td>
          <td>2025-03-10</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9113b4cede2cf7486243e4f5408b0e991ee33d2d" target='_blank'>
              High-throughput single-cell DNA methylation and chromatin accessibility co-profiling with SpliCOOL-seq
              </a>
            </td>
          <td>
            Qingmei Shen, Enze Deng, Jingna Zhang, Qifeng Yang, Dan Su, Xiaoying Fan
          </td>
          <td>2025-03-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cedb6597be1303e7e75516814053b5e6b9747717" target='_blank'>
              Natural products targeting cancer stem cells: a promising therapeutic approach
              </a>
            </td>
          <td>
            Julia K. Opara, Sierra Sanchez, Sufi Mary Thomas, S. Anant
          </td>
          <td>2025-04-18</td>
          <td>Exploration of Drug Science</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Cancer is a complex and highly lethal disease marked by unchecked cell proliferation, aggressive behavior, and a strong tendency to metastasize. Despite significant advancements in cancer diagnosis and treatment, challenges such as early detection difficulties, drug resistance, and adverse effects of radiotherapy or chemotherapy continue to threaten patient survival. MicroRNAs (miRNAs) have emerged as critical regulators in cancer biology, with miR-506 being extensively studied and recognized for its tumor-suppressive effects across multiple cancer types. This review examines the regulatory mechanisms of miR-506 in common cancers, focusing on its role in the competing endogenous RNA (ceRNA) network and its effects on cancer cell proliferation, apoptosis, and migration. We also discuss the potential of miR-506 as a therapeutic target and its role in overcoming drug resistance in cancer treatment. Overall, these insights underscore the therapeutic potential of miR-506 and its promise in developing novel cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06c1c0de824cb3b6994467264a7d64c1f9bef206" target='_blank'>
              Therapeutic potential of microRNA-506 in cancer treatment: mechanisms and therapeutic implications
              </a>
            </td>
          <td>
            Shuzhen Mao, Junyan Li, Jiahui Huang, Lili Lv, Qilian Zhang, Qing Cheng, Xiaojing Liu, Zhiwei Bi, Jing Yao
          </td>
          <td>2025-04-03</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Epigenetic dysregulation is prevalent in human cancers, affecting gene expression and metabolic patterns to meet the demands of malignant evolution and abnormal epigenetic processes, and resulting in a protumor immune microenvironment. Tumors require a steady supply of methionine for maintaining epigenetic flexibility, which is the only exogenous precursor of methyl donor S‐adenosylmethionine for methylation, crucial for their resistance to therapies and survival in a nutrient‐deficient microenvironment. Thus, tumor cells upregulate the Lat4 transporter to compete and deprive methionine in the microenvironment, sustaining their malignant phenotypes and also impairing immune cell functions. Addressing this methionine addiction is the key to overcoming drug resistance and improving immune response. Despite the challenge of lacking specific Lat4 inhibitors, an oxaliplatin prodrug crosslinked fluorinated polycation/anti‐Lat4 small interfering RNA complex nanoregulator (AS‐F‐NP) has been designed and developed here. This nanoregulator restricted the greedy methionine uptake of tumor cells by knocking down Lat4, which in turn inhibited the malignant evolution of the tumor while restoring the viability and function of tumor‐infiltrating immune cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96e01c550a22a629f238ced405d904af38f03862" target='_blank'>
              A Methionine Allocation Nanoregulator for the Suppression of Cancer Stem Cells and Support to the Immune Cells by Epigenetic Regulation
              </a>
            </td>
          <td>
            Boyu Su, Qinjun Chen, Xuwen Li, Mingzhu Fang, Yu Wang, Haolin Song, Haoyu You, Zhen-yong Zhou, Yuxing Wu, Zhenhao Zhao, Yun Chen, Hongrui Fan, Chufeng Li, Chen Jiang, Tao Sun
          </td>
          <td>2025-02-22</td>
          <td>Advanced Science</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/255726480b1fba8b96c7a797db393035b828db59" target='_blank'>
              Abstract B022: Decipher actionable tumor suppressor defects for personalized cancer medicine
              </a>
            </td>
          <td>
            Feiyu Chen, Di Zhao
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Lung squamous cell carcinoma (LUSC), accounting for 30% of lung cancer cases, lacks adequate research due to limited understanding of its molecular abnormalities. Our study analyzed public LUSC datasets to explore the tumor microenvironment (TME) composition using scRNA-seq from two cohorts. Applying non-negative matrix factorization, we identified unique malignant cell phenotypes, or meta-programs (MPs), based on gene expression patterns. Survival analysis revealed the clinical relevance of these MPs. Findings illuminated a TME landscape enriched with immune cells—CD8 + T, exhausted T, CD4 + T, and naive T cells—and suggested roles for myeloid cells, like cDC1 and pDCs, in LUSC progression. Different MPs highlighted the heterogeneity of malignant cells and their clinical implications. Targeting MP-specific genes may enable personalized therapy, especially for early-stage LUSC. This study offers insights into immune cell function in tumor dynamics, identifies MPs, and paves the way for novel LUSC strategies, enhancing early intervention, personalized treatment, and prognosis, ultimately improving patient outcomes. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-93916-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a302b866a51ae52b8b8d7352c3feab56f45e3d0e" target='_blank'>
              Single-cell RNA sequencing reveals potential therapeutic targets in the tumor microenvironment of lung squamous cell carcinoma
              </a>
            </td>
          <td>
            Junda Fan, Yu Chen, Yue Gong, Hongmei Sun, Rui Hou, Xiaoya Dou, Yanping Zhang, Cheng Huo
          </td>
          <td>2025-03-26</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="DNA damage and repair have been central themes in cellular biology research. Broadly, DNA damage is understood as modifications to canonical nucleotides that disrupt their function during transcription and replication. A deeper biochemical understanding of DNA damage is essential, as the genome governs all cellular processes. We can classify DNA damage according to whether the modifications to the nucleic acid scaffold are chemically or enzymatically initiated. This distinction is important because chemical modifications are often irreversible, sometimes sparse, and difficult to detect or control spatially and replicate systematically. This can result in genomic damage or modifications to nucleotides in the nucleotide pool, which is less commonly studied. In contrast, enzymatic modifications are typically induced by the cell for specific purposes and are under strong regulatory control. Enzymatic DNA modifications also present a degree of sequence specificity and are often reversible. However, both types of DNA modifications contribute to cellular aging when poorly repaired and, as a result, remain incompletely understood. This review hopes to gather less studied mechanisms in nucleotide modifications and show research gaps in our current understanding of nucleotide biology. By examining the implications of these mechanisms on DNA modifications, in the nucleotide pool and genome, we may gain insights into innovative strategies for mitigating the effects of cellular aging.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4b3fc5571224db4c16f95a7fa9aaf2cfdba5201" target='_blank'>
              Chemical Versus Enzymatic Nucleic Acid Modifications and Genomic Stability
              </a>
            </td>
          <td>
            Jonathan R. Cortez, Marie E. Migaud
          </td>
          <td>2025-04-09</td>
          <td>DNA</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background/Objectives: Fragile X syndrome (FXS) is a disease of pathologic epigenetic silencing induced by RNA. In FXS, an expanded CGG repeat tract in the FMR1 gene induces epigenetic silencing during embryogenesis. FMR1 silencing can be reversed with 5-aza-deoxyctidine (5-aza-dC), a nonspecific epigenetic reactivator; however, continuous administration of 5-aza-dC is problematic due to its toxicity. We describe an approach to restore FMR1 expression in FXS neurons by transient treatment with 5-aza-dC, followed by treatment with 2HE-5NMe, which binds the CGG repeat expansion in the FMR1 mRNA and could block the resilencing of the FMR1 gene after withdrawal of 5-aza-dC. Methods: This study uses immunofluorescence and fluorescent in situ hybridization (FISH) to measure whether FMR1 expression is maintained in FXS post-mitotic neurons treated with 2HE-5NMe. Genome-wide profiling of histone marks was used to monitor epigenetic changes and drug selectivity in response to 5-aza-dC followed by 2HE-5NMe treatment. Changes to dendritic morphology were visualized using confocal microscopy. Results: In this study, we find that 2HE-5Nme maintains FMR1 in a reactivated state after reactivation using 5-aza-dC in post-mitotic neurons. FMR1 reactivation in neurons results in the re-expression of FMRP and reversal of FXS-associated dendritic spine defects. Conclusions: These results demonstrate that an RNA-binding small molecule can achieve gene-specific epigenetic control and provide an approach for the restoration of FMRP in FXS neurons.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ddaf9f6479660c108f91bf86393ce4507c8e607" target='_blank'>
              Sustained Epigenetic Reactivation in Fragile X Neurons with an RNA-Binding Small Molecule
              </a>
            </td>
          <td>
            Christina W. Kam, Jason G Dumelie, G. Ciceri, Wang-Yong Yang, Matthew D Disney, Lorenz Studer, S. Jaffrey
          </td>
          <td>2025-02-25</td>
          <td>Genes</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34833006929f8c53f4dd25f1eecd3e7760e96187" target='_blank'>
              Biochemical Mechanisms of Genetic Recombination and DNA Repair.
              </a>
            </td>
          <td>
            Vivek B. Raina, Aidan M. Jessop, Eric C Greene
          </td>
          <td>2025-03-28</td>
          <td>Annual review of biochemistry</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f1f61ef381af091b6db128907436bc9e8a1a514" target='_blank'>
              Directed evolution expands CRISPR-Cas12a genome editing capacity
              </a>
            </td>
          <td>
            E. Ma, Kai Chen, Honglue Shi, Kevin Wasko, Isabel Esain-Garcia, M. Trinidad, Kaihong Zhou, Jinjuan Ye, Jennifer A. Doudna
          </td>
          <td>2025-03-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fc9176f3b57d689386be0c2303a9e42c03c1797" target='_blank'>
              Unraveling the hidden complexity of cancer through long-read sequencing.
              </a>
            </td>
          <td>
            Qiuhui Li, Ayse G Keskus, Justin Wagner, Michal B Izydorczyk, W. Timp, F. Sedlazeck, A. Klein, J. Zook, M. Kolmogorov, Michael C. Schatz
          </td>
          <td>2025-03-20</td>
          <td>Genome research</td>
          <td>1</td>
          <td>49</td>
        </tr>

        <tr id="CRISPR-Cas9 systems have revolutionized genome editing, but the off-target effects of Cas9 limit its use in clinical applications. Here, we systematically evaluate FrCas9, a variant from Faecalibaculum rodentium, for cell and gene therapy (CGT) applications and compare its performance to SpCas9 and OpenCRISPR-1. OpenCRISPR-1 is a CRISPR system synthesized de novo using large language models (LLMs) but has not yet undergone systematic characterization. Using AID-seq, Amplicon sequencing, and GUIDE-seq, we assessed the on-target activity and off-target profiles of these systems across multiple genomic loci. FrCas9 demonstrated higher on-target efficiency and substantially fewer off-target effects than SpCas9 and OpenCRISPR-1. Furthermore, TREX2 fusion with FrCas9 reduced large deletions and translocations, enhancing genomic stability. Through screening of 1903 sgRNAs targeting 21 CGT-relevant genes using sequential AID-seq, Amplicon sequencing, and GUIDE-seq analysis, we identified optimal sgRNAs for each gene. Our high-throughput screening platform highlights FrCas9, particularly in its TREX2-fused form, as a highly specific and efficient tool for precise therapeutic genome editing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6808f0ec0ae3c51721dc4169f97a812e9c172d0f" target='_blank'>
              Systematic high-throughput evaluation reveals FrCas9’s superior specificity and efficiency for therapeutic genome editing
              </a>
            </td>
          <td>
            Rui Tian, Xun Tian, Meiying Yang, Yuping Song, Tingting Zhao, Chaoyue Zhong, Wei Zhu, Ping Zhou, Zhiqiang Han, Zheng Hu
          </td>
          <td>2025-03-26</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The malaria parasite has a complex lifecycle involving various host cell environments in both human and mosquito hosts. The parasite must tightly regulate gene expression at each stage in order to adapt to its current environment while continuing development. However, it is challenging to study gene function and regulation of essential genes across the parasite’s multi-host lifecycle. Thus, we adapted a recently developed a single-plasmid dimerizable Cre recombinase system for rapamycin-controllable expression of Cas9, allowing for conditional introduction of mutations. We explored rates of gene deletion using varying repair template lengths, showing functionality of donor templates under 250bp for homology-directed repair. As a proof of concept, we conditionally disrupted two uncharacterized genes in blood and gametocyte stages, identifying new stage-specific phenotypes. Importance As progress towards eliminating malaria has stalled, there is a pressing need for new antimalarials and vaccines. Genes essential to multiple stages of development represent ideal candidates for both antimalarials and vaccines. However, much of the parasite genome remains uncharacterized. Conditional gene perturbation approaches are needed in order to study gene function across the lifecycle. Currently available tools are limited in their ability to perturb genes at the scale required for large screens. We describe a tool that allows for conditional introduction of desired mutations by controlling Cas9 with the DiCre-loxP system. We demonstrate the accessibility of this approach by designing gRNA-donor pairs that can be commercially synthesized. This toolkit provides a scalable system for identifying new drug and vaccine candidates targeting multiple stages of the parasite lifecycle">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0bb6d5a5f56597242d2c3a0b197c97dddd6d786d" target='_blank'>
              A Conditional Cas9 System for Stage-Specific Gene Editing in P. falciparum
              </a>
            </td>
          <td>
            Sean T. Windle, M. L. Neal, Fred D. Mast, Stefan H. I. Kappe, J. Aitchison
          </td>
          <td>2025-03-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/739935357776c471010c005564687898973a29a6" target='_blank'>
              Epidemiology, pathogenesis, biology and evolving management of MSI-H/dMMR cancers.
              </a>
            </td>
          <td>
            M. Ambrosini, P. Manca, Vincenzo Nasca, C. Sciortino, F. Ghelardi, Jenny F. Seligmann, J. Taieb, F. Pietrantonio
          </td>
          <td>2025-04-03</td>
          <td>Nature reviews. Clinical oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Objectives/Goals: Large-scale tumor sequencing efforts have led to annotations of novel cancer hotspot mutations that may underlie driver or cooperative function. We have sought to define the molecular consequences of such hotspots associated with pediatric DICER1 syndrome cancers, with the ultimate goal of revealing novel targets that may inform new standards of care. Methods/Study Population: We have performed genomic analysis to identify tumor types (in TCGA and MSK-IMPACT patient data) for which mutations in the Dicer1 gene (encoding Dicer protein) emerge as the dominant signature of driver function. As Dicer is a critical RNA processing factor responsible for the generation of microRNAs, which are posttranscriptional gene regulatory molecules, we have modeled these mutations in human embryonic stem cells in order to study the direct effects on miRNAs and their target genes in an isogenic background. In addition to providing the required setting for unambiguous attribution of function to specific mutations, clonal human ES cells offer an opportunity for modeling of both developmental and cancer requirements associated with altered Dicer function. Results/Anticipated Results: Through generation of genomics and functional datasets from matched genotypes in Dicer mutated human ES cells, we have identified specific alterations in miRNAs and their effects on target genes. Unexpectedly, we found direct evidence for both loss of function and gain of function attributable to Dicer mutations. In addition, through integrated analysis of genomic data from tumor sequencing datasets and our human ES cell models, we have identified potential miRNA and target gene alterations that underlie tumorigenic potential, nominating gene candidates for targeted therapy in DICER1 syndrome. Direct mouse modeling of such candidate gene targets has revealed evidence for driver function of identified miRNA and their targets. Discussion/Significance of Impact: DICER1 syndrome cancers comprise a wide variety of rare pediatric tumor types. Presently, we still lack an effective standard of care. Furthermore, the previous lack of molecular profiling precluded targeted therapy opportunities. Our precise knock-in modeling of Dicer hotspots and deep profiling of relevant tumors now provide candidate targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/597fce8f39319f3d39a09a21bf59ed3021208522" target='_blank'>
              518 Modeling of cancer mutations found in pediatric DICER1 syndrome informs novel therapeutic targeting strategies
              </a>
            </td>
          <td>
            David Jee, Seungjae Lee, Dapeng Yang, Robert W. Rickert, Renfu Shang, Danwei Huangfu, Eric C. Lai
          </td>
          <td>2025-03-26</td>
          <td>Journal of Clinical and Translational Science</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fb803b3e058115ac9f37e42f7b7a8e8748c0c02" target='_blank'>
              Next-Generation Sequencing-Based In Silico Transgene Integration Profiling Tool Accelerates Cell Line Genetic Characterizations.
              </a>
            </td>
          <td>
            Zhenqiu Huang, Hyo-Young Jeong, Luke Nelson, Caitlyn Devine, Guanghua Benson Li, Lin Zhang
          </td>
          <td>2025-03-01</td>
          <td>Biotechnology journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background/Objectives: Genetics and epigenetics play an important role in the pathogenesis of cutaneous melanoma. The majority of cases harbor mutations in genes associated with the MAPK signaling pathway, i.e., BRAF, NRAS, or NF1. The remaining neoplasms, often located on acral sites, are condensed as the triple-wildtype subtype and are characterized by other molecular drivers. This study aimed to elucidate genetic and epigenetic differences within cutaneous melanoma and to compare it with melanocytic nevi. Methods: DNA was extracted from archived tissue samples of cutaneous melanoma (n = 19), melanocytic nevi (n = 11), and skin controls (n = 11) and subsequently analyzed by massive parallel (next generation) gene panel sequencing and genome-wide DNA methylation array analysis. The sample size was increased by including repository data from an external study. Results: There were major differences in the genomic landscape of MAPK-altered and triple-wildtype cutaneous melanoma, the latter presenting with a lower number of mutations, a different pattern of copy number variants, and a low frequency of TERT promoter mutations. Dimensional reduction of DNA methylation array analysis clearly separated cutaneous melanoma from melanocytic nevi but revealed no major differences between classical cutaneous melanoma and the triple-wildtype cases. However, it identified a possible biological subgroup characterized by intermediately methylated CpGs. Conclusions: Dimensional reduction of methylation array data is a useful tool for the analysis of melanocytic tumors to differentiate between malignant and benign lesions and may be able to identify biologically distinct subtypes of cutaneous melanoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd4730aeef318378ac4a266dcf9f011ba2c44628" target='_blank'>
              DNA Methylation Array Analysis Identifies Biological Subgroups of Cutaneous Melanoma and Reveals Extensive Differences with Benign Melanocytic Nevi
              </a>
            </td>
          <td>
            Simon Schwendinger, Wolfram Jaschke, Theresa Walder, J. Hench, Verena Vogi, Stephan Frank, Per Hoffmann, S. Herms, Johannes Zschocke, Van Anh Nguyen, Matthias Schmuth, E. Jukic
          </td>
          <td>2025-02-21</td>
          <td>Diagnostics</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="CRISPR/Cas9 based genome editing has advanced our understanding of a myriad of important biological phenomena. Important challenges to multiplex genome editing in maize include assembly of large complex DNA constructs, few genotypes with efficient transformation systems, and costly/labor-intensive genotyping methods. Here we present an approach for multiplex CRISPR/Cas9 genome editing system that delivers a single compact DNA construct via biolistics to Type I embryogenic calli, followed by a novel efficient genotyping assay to identify desirable editing outcomes. We first demonstrate the creation of heritable mutations at multiple target sites within the same gene. Next, we successfully created individual and stacked mutations for multiple members of a gene family. Genome sequencing found off-target mutations are rare. Multiplex genome editing was achieved for both the highly transformable inbred line H99 and Illinois Low Protein1 (ILP1), a genotype where transformation has not previously been reported. In addition to screening transformation events for deletion alleles by PCR, we also designed PCR assays that selectively amplify deletion or insertion of a single nucleotide, the most common outcome from DNA repair of CRISPR/Cas9 breaks by non-homologous end-joining. The Indel-Selective PCR (IS-PCR) method enabled rapid tracking of multiple edited alleles in progeny populations. The ‘end to end’ pipeline presented here for multiplexed CRISPR/Cas9 mutagenesis can be applied to accelerate maize functional genomics in a broader diversity of genetic backgrounds. Supplementary Information The online version contains supplementary material available at 10.1186/s13007-025-01365-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de431ff0e73b17a2417c342121bd7ddd4aecfa44" target='_blank'>
              Efficient mutagenesis and genotyping of maize inbreds using biolistics, multiplex CRISPR/Cas9 editing, and Indel-Selective PCR
              </a>
            </td>
          <td>
            Maruti Nandan Rai, Brian Rhodes, Stephen Jinga, Praveena Kanchupati, Edward Ross, Shawn R. Carlson, Stephen P. Moose
          </td>
          <td>2025-03-25</td>
          <td>Plant Methods</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="ERBB2 (HER2) is a well-studied oncogene with several driver mutations apart from the well-known amplification defect in some breast cancers. We used the GigaAssay to test the functional effect of HER2 missense mutations on its receptor tyrosine kinase function. The GigaAssay is a modular high-throughput one-pot assay system for simultaneously measuring molecular function of thousands of genetic variants at very high accuracy. The activities of 5,886 mutations were classified, significantly more than mutants previously reported. These variants include 112 new in vitro, 10 known, and 9 new in vivo gain-of-function (GOF) mutations. Many of the GOFs spatially cluster in sequence and structure, supporting the activation mechanisms of heterodimerization with EGFR and release of kinase inhibition by the juxtamembrane domain. Retrospective analysis of patient outcomes from the Genomic Data Commons predicts increased survival with the newly identified HER2 GOF variants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b136d3cab282e49a197992ab3575cc9958742822" target='_blank'>
              New Gain-of-Function Mutations Prioritize Mechanisms of HER2 Activation
              </a>
            </td>
          <td>
            C. Giacoletto, Liz J. Valente, Lancer Brown, Sara Patterson, Rewatee H Gokhale, S. Mockus, Wayne W. Grody, Hong-Wen Deng, Jerome I. Rotter, Martin R. Schiller
          </td>
          <td>2025-03-04</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc7fd173fd5669ad14660b53ec6364396f64ec51" target='_blank'>
              A modular CRISPRa system for molecular therapy of FOXG1 syndrome
              </a>
            </td>
          <td>
            J. Halmai, Casiana E Gonzalez, P. Deng, Jasmine L Carter, J. Waldo, D. Cameron, Angels Almenar-Queralt, Lauro Thiago Turaça, Alysson R Muotri, Kyle D Fink
          </td>
          <td>2025-04-17</td>
          <td>NAR Molecular Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/498b068e38445ba010a97c92197f812fab2d458c" target='_blank'>
              Abstract PR010: Genome-wide CRISPR screens in CD8+ T cells identify cullin-RING E3 ubiquitin ligase complex as a negative regulator of long-term effector function
              </a>
            </td>
          <td>
            Justin D. Saco, Daniel Karin, Nataly Naser Al Deen, Katie M. Campbell, Justin Patten, Yichen Fan, Egmidio Medina, Antoni Ribas
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="ABSTRACT Epigenetics and epigenomics are captivating fields of molecular biology, dedicated to the exploration of heritable alterations in gene expression and cellular phenotypes, which transpire devoid of any discernible modifications to the fundamental DNA sequence. This intricate regulatory apparatus encompasses multiple mechanisms, prominently featuring DNA methylation, histone modifications, and the involvement of non-coding RNA molecules in pivotal roles. To achieve a comprehensive grasp of these diverse mechanisms, it is imperative to conduct research employing animal models as proxies for human studies. Since experimental animal models like mice and rats struggle to replicate the diverse environmental conditions experienced by humans, this review focuses on comparing common epigenetic alterations in naturally occurring tumors in canine models, which share the human environment, with those in humans. Through this, we emphasize the importance of an epigenetic regulation in the comparative medical approach to a deeper understanding of cancers and further development of cancer treatments. Additionally, we elucidate epigenetic modifications pertinent to specific developmental stages, the ageing process, and the progression of various diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17bfc3c89b9e9910bb56cc9d2d52428ce2d674ca" target='_blank'>
              Comparative epigenomics to clinical trials in human breast cancer and canine mammary tumor
              </a>
            </td>
          <td>
            Su-Jin Jeong, Kang-Hoon Lee, Je-Yoel Cho
          </td>
          <td>2025-03-19</td>
          <td>Animal Cells and Systems</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Comprehensive and systematic proteome-wide experiments require financial and technical resources unavailable to most researchers. Here we describe a scalable CRISPR/Cas9-non-homologous end joining (NHEJ) based method for Pooled Recombinant Integration of Seamless Markers (PRISM) into protein coding genes. We created two gRNA libraries for 5’- and 3’-tagging of 18,804 human protein-coding genes. Selection for in-frame integration of the donor cassette can be guaranteed by fusing it to an antibiotic resistance enzyme (ARE) and P2A self-cleaving peptide, resulting in tagged proteins and free ARE to select for clones with inserted donor cassettes. We achieved a library integration rate of 19.75% and tagging of ∼80% for genes expressed in Hek293T cells and notably, 89.7% of essential genes, with donor DNA. Our strategy is scalable, specific, and selective, paving the way for genome-scale construction of human cell lines tagged with different types of reporter genes for protein functional characterization. Significance statement Here we report the first practical method to achieve near complete 5’- or 3’-end integration of reporter protein-coding sequences to express seamless fusions of human protein and reporter proteins that we call Pooled Recombinant Integration of Seamless Markers (PRISM). PRISM is independent of homology templates, yet it works with high efficiency and precision. Using gRNA-directed CRISPR/Cas9 genome editing and mini-plasmid donor cassettes we were able to tag about 80% of the protein-coding sequences of human genes with a green florescent protein at an integration efficiency of about 20% and most notably, of 90% of essential genes in the human cell line HEK 293T. This is important because the characterization of gene functions in a particular cell type usually begins with essential genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8834136786fe808caa704ff04cd6e336c885cd2d" target='_blank'>
              A strategy for genome-wide seamless tagging of human protein-coding genes
              </a>
            </td>
          <td>
            Minoo Karimi, Masih M. Saber, Nicolas Stifani, Louis Gauthier, Adrian Serohijos, S. Michnick
          </td>
          <td>2025-03-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b20395b5511b0ca628f7388834f767639654823" target='_blank'>
              Conserved and Unique Features of Terminal Telomeric Sequences in ALT-Positive Cancer Cells
              </a>
            </td>
          <td>
            Benura Azeroglu, Wei Wu, R. Pavani, Ranjodh Sandhu, Tadahiko Matsumoto, A. Nussenzweig, E. L. Denchi
          </td>
          <td>2025-03-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Philadelphia chromosome-like acute lymphoblastic leukemia (ALL) is a group of diseases that possess similar gene expression patterns to those of Philadelphia chromosome-positive ALL, but lack the BCR-ABL1 gene rearrangement and exhibit diverse genetic characteristics. These cases are associated with a poorer prognosis than that of other B-cell ALL cases and require novel therapeutic approaches. Philadelphia chromosome-like ALL can be categorized into ABL-activated, Janus kinase-signal transducer and activator of transcription (JAK-STAT) pathway-altered, and other subtypes based on genetic alterations. For the ABL-activated subtype, treatment with ABL-directed tyrosine kinase inhibitors may improve outcomes, whereas JAK-directed tyrosine kinase inhibitors have been proposed as a combination therapy for patients with JAK-STAT pathway mutations. In this review, we discuss treatment approaches for each subtype of Philadelphia chromosome-like ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/baccb2a6cea2dd838b8bfc81e802e8e35068f3e2" target='_blank'>
              Treatment Approaches for Philadelphia Chromosome-Like Acute Lymphoblastic Leukemia
              </a>
            </td>
          <td>
            Jeong Suk Koh, I. Song
          </td>
          <td>2025-04-01</td>
          <td>The Korean Journal of Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db3af9ddf4c19e3b5f84a20f4bfe51b337d970b1" target='_blank'>
              Circularization of Single-Stranded DNA Donor Template Unleashes the Power of Non-Viral Gene Delivery for Long-Term HSCs editing
              </a>
            </td>
          <td>
            Gil Letort, A. Duclert, D. L. Clerre, I. Chion-Sotinel, Roger Salvatori, Emilie Dessez, Margaux Sevin, Marco Rotondi, Cosimo Ducani, Philippe Duchateau, J. Valton
          </td>
          <td>2025-02-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d6e9e6ea94c0b9c3a4b09ce35477edf5061c010" target='_blank'>
              RNA Transcripts Serve as a Template for Double-Strand Break Repair in Human Cells
              </a>
            </td>
          <td>
            Manisha Jalan, Alessandra Brambati, Hina Shah, Niamh McDermott, Juber Patel, Yingjie Zhu, Ahmet Doymaz, Julius Wu, Kyrie S Anderson, Andrea M Gazzo, F. Pareja, Takafumi N. Yamaguchi, Theodore Vougiouklakis, Sana Ahmed-Seghir, Philippa Steinberg, Anna Neiman-Golden, Benura Azeroglu, Joan Gomez-Aguilar, E. D. da Silva, Suleman Hussain, Daniel Higginson, P. Boutros, N. Riaz, J. Reis-Filho, Simon N. Powell, Agnel Sfeir
          </td>
          <td>2025-02-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1dc5fbd4c7c95df929aa7b21084f47cc0960e08a" target='_blank'>
              Reinforced CRISPR interference enables reliable multiplex gene repression in phylogenetically distant bacteria
              </a>
            </td>
          <td>
            Joshua R. Elmore, R. Shrestha, Andrew Wilson, Elise Van Fossen, Andrew R. Frank, Ryan M. Francis, H. Baldino, Molly Stephenson, Bhavya Gupta, Jeane Rivera, R. Egbert
          </td>
          <td>2025-04-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Abstract The Philadelphia chromosome, the translocation between BCR and ABL genes, is seen in 95% of chronic myeloid leukemia (CML) patients. Although discovered >60 years ago, the molecular mechanism of BCR fragility is unclear. Here, we have identified several G4 DNA motifs at the BCR fragile region of CML patients. Various lines of experimentation revealed that the breakpoint regions could fold into multiple intramolecular G-quadruplex structures. The sodium bisulfite modification assay revealed single strandedness in the fragile region when present on a plasmid and human genome. Circular dichroism spectroscopy revealed the parallel G4 DNA formation, leading to polymerase arrest at the BCR breakpoints. Intracellular recombination assay revealed that DNA breakage at the BCR fragile region could join with the break generated by ISceI endonuclease. Finally, purified AID could bind and deaminate cytosines when present on single-stranded DNA generated due to G4 DNA, both in vitro and inside the cells. Therefore, our results suggest that AID binds to G4 DNA present at the BCR fragile region, resulting in the deamination of cytosines to uracil and induction of DNA breaks in one of the DNA strands, which can later get converted into a double-strand break, leading to t(9;22) chromosomal translocation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dad133901fa4a2888984a520b619646b9998e103" target='_blank'>
              Formation of multiple G-quadruplexes contributes toward BCR fragility associated with chronic myelogenous leukemia
              </a>
            </td>
          <td>
            Shivangi Sharma, Elizabeth Thomas, Sumedha Dahal, Sayak Das, Shefali Kothari, U. Roy, Nitu Kumari, V. Gopalakrishnan, S. Raghavan
          </td>
          <td>2025-03-20</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f37dc921f83b90daf61b10b8456e0c63c032e710" target='_blank'>
              [Pathogenic large duplication in TP53 as a hereditary predisposing factor in breast cancer].
              </a>
            </td>
          <td>
            Viktória Kovács, H. Butz, János Papp, T. Pócza, Kornél Vince Grolmusz, Petra Nagy, Attila Patócs, A. Bozsik
          </td>
          <td>2025-03-21</td>
          <td>Magyar onkologia</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53dcae9646767b55b828bf57d0d3c3835c11bb8d" target='_blank'>
              Abstract B035: Evaluating drug sensitivity heterogeneity in cancer organoids derived from circulating tumor cells
              </a>
            </td>
          <td>
            Y. Han, David Hsieh, Shian-Jiun Shih
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da0026f814610b80b221cc4995959120d34e1c0a" target='_blank'>
              Deciphering WNT signalling vulnerabilities in soft tissue sarcoma.
              </a>
            </td>
          <td>
            Marina Pérez-Capó, Esther Martinez-Font, Elena Prados, Rafael Ramos, Raúl Sánchez Morillas, Maria Rosa Martorell, Oliver Vögler, R. Alemany, A. Obrador-Hevia
          </td>
          <td>2025-03-20</td>
          <td>Pathobiology : journal of immunopathology, molecular and cellular biology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Background: Epigenetic biomarkers, particularly CpG methylation, are increasingly employed in clinical and forensic settings. However, we still lack a cost-effective, sensitive, medium-scale method for the analysis of hundreds to thousands of user-defined CpGs suitable for minute DNA input amounts (<10 ng). In this study, motivated by promising results in the genetics field, we investigated single-molecule molecular inversion probes (smMIPs) for simultaneous analysis of hundreds of CpGs by using an example set of 514 age-associated CpGs (Zhang model). Methods: First, we developed a novel smMIP design tool to suit bisulfite-converted DNA (Locksmith). Then, to optimize the capture process, we performed single-probe capture for ten selected, representative smMIPs. Based on this pilot, the full smMIP panel was tested under varying capture conditions, including hybridization and elongation temperature, smMIP and template DNA amounts, dNTP concentration and elongation time. Results: Overall, we found that the capture efficiency was highly probe-(and hence, sequence-) dependent, with a heterogeneous coverage distribution across CpGs higher than the 1000-fold range. Considering CpGs with at least 20X coverage, we yielded robust methylation detection with levels comparable to those obtained from the gold standard EPIC microarray analysis (Pearsons’s r: 0.96). Conclusions: The observed low specificity and uniformity indicate that smMIPs in their current form are not compatible with the lowered complexity of bisulfite-converted DNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/069c0dd8ea6b1c32ff7183c9249403b29e389f54" target='_blank'>
              Investigating Single-Molecule Molecular Inversion Probes for Medium-Scale Targeted DNA Methylation Analysis
              </a>
            </td>
          <td>
            Roy B. Simons, H. Adams, M. Kayser, A. Vidaki
          </td>
          <td>2025-03-01</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03e8bbeec84df302b9adf6fafec7b5584f0a9749" target='_blank'>
              Advancement of Gene Therapy in Cancer
              </a>
            </td>
          <td>
            Kahkashan Sultana, Ayesha Fatima
          </td>
          <td>2025-03-15</td>
          <td>Journal of Advanced Scientific Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d3959b933b4c238cc534f4bd3bbf4f10f6618c5" target='_blank'>
              Recent Advances in Anti-Aging Therapeutic Strategies Targeting DNA Damage Response and Senescence-Associated Secretory Phenotype-Linked Signaling Cascade.
              </a>
            </td>
          <td>
            Jawad Nadeem, Razia Sultana, Amna Parveen, Sun Yeou Kim
          </td>
          <td>2025-02-26</td>
          <td>Cell biochemistry and function</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Chromosomal alterations are frequent events in lung cancer progression. Although gains and losses of chromosomal position have been reported, the association between copy number alteration and lung cancer patient survival has not been extensively investigated. In this study, we performed a meta-analysis of public cBioPortal datasets spanning 25 lung cancer studies to identify putative cancer driver genes with copy number alterations associated with overall patient survival. Ten copy-number altered genes enriched in deceased lung cancer patients were identified. Seven of these putative driver genes were located in the 7p11.2 chromosomal location, and two were in the 9p21.3 cytoband. Among these genes, the mitochondrial ribosomal protein S17 (MRPS17) amplification was significantly associated with a lower patient survival rate (P = 1.47e-7). To investigate the functional role of MRPS17, small interfering RNA-mediated knockdown was performed in two non-small cell lung cancer cell lines, A549 and NCI-H460. MRPS17 knockdown significantly reduced cell proliferation, migration, invasion, and anchorage-independent growth in both cell lines. Furthermore, knockdown of MRPS17 decreased the activation of the phosphatidylinositol 3-kinase/protein kinase B signaling pathway, suggesting its role in driving lung cancer progression through this critical oncogenic pathway. Our findings highlight MRPS17 as a potential cancer therapy target and a prognostic biomarker that may improve the survival rates of lung cancer patients. Future studies should explore its inhibition as a therapeutic strategy as well as elucidate its molecular mechanisms in cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f1c0b37271452f619e3684f3550b14101ddebda" target='_blank'>
              Mitochondrial Ribosomal Protein S17 Silencing Inhibits Proliferation and Invasiveness of Lung Cancer Cells
              </a>
            </td>
          <td>
            Woo Rin Lee, Kook Sun Ha
          </td>
          <td>2025-03-30</td>
          <td>Journal of Cancer Prevention</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2050a0d48b53f00fa87bdf892317410211b4055f" target='_blank'>
              Harnessing State-of-the-Art Gene Therapy to Transform Oral Cancer Treatment.
              </a>
            </td>
          <td>
            Wei Zhang, Yan Zhang, Xiaowen Yang, Hongyan Chai
          </td>
          <td>2025-03-11</td>
          <td>Biochemical genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The ataxia-telangiectasia mutated (ATM) protein kinase plays a critical role in activating the cellular response to DNA double-strand breaks and promoting homology-directed repair. ATM is frequently mutated in cancer, contributing to an accumulation of DNA damage that drives genomic instability. To exploit cancer cells’ inherent vulnerability to DNA damage, various small molecule inhibitors have been developed that target ATM. ATM inhibitors have shown great versatility in preclinical studies and increasing use in the clinic. Here, we review the development of ATM inhibitors and their role in cancer therapy. We describe their limitations and the advances that have led to increases in both the number and diversity of active clinical trials targeting ATM. We also discuss ATM’s role in personalized medicine and the current challenges to more widespread use of ATM inhibitors in the clinic.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c1ae4df6ed1675fe0ee6d9bf7493ed3c92eb628" target='_blank'>
              The Development of ATM Inhibitors in Cancer Therapy
              </a>
            </td>
          <td>
            Elizabeth A. Ampolini, Judit Jimenez-Sainz, David T Long
          </td>
          <td>2025-03-01</td>
          <td>Targeted Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Gene fusions are one of the most important molecular biomarkers for tumor diagnosis, classification and targeted therapy. How to accurately detect them is a key issue in clinical work. In this study, a custom-designed integration of DNA and RNA-based next generation sequencing (NGS) assay including 16 targeted therapy related genes was developed and validated to identify gene fusions in solid tumors. This assay accurately identified all 10 different types of fusion in 8 commercial fusion spiked-in reference standards and 29 fusions including 16 different fusion forms in 60 clinical solid tumor samples previously identified by clinical testing methods. In addition, a TPM3::NTRK1 fusion was additionally identified and validated by Sanger sequencing, which showed a false-negative result for the previous result. Mutational abundance limit of detection for the assay was assessed with a series of dilution experiments. These fusions can be stably detected when the mutational abundance is down to 5% for DNA and 250–400 copies/100 ng for RNA. The intra-assay and inter-assay reproducibility was observed in three samples and three replicates. This integration of DNA and RNA-based NGS assay shows excellent performance on formalin-fixed, paraffin-embedded samples, results at different levels can complement each other, thereby facilitating precise diagnosis and treatment. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-91640-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bf99e2aad7ac70489c36434ebca41134cb4c9a6" target='_blank'>
              An accurate DNA and RNA based targeted sequencing assay for clinical detection of gene fusions in solid tumors
              </a>
            </td>
          <td>
            G. Ji, Qianlan Yao, Min Ren, Qianming Bai, Xiaoli Zhu, Xiaoyan Zhou
          </td>
          <td>2025-02-28</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Objectives/Goals: Investigating the B-cell acute lymphoblastic leukemia (B-ALL)-associated germline SNP rs7090445, located in intron 3 of ARID5B, which is more frequently observed in individuals of Hispanic/Latino descent. Investigating the mechanisms behind this inherited single nucleotide polymorphisms (SNP) that may contribute to the higher incidence of B-ALL in this population. Methods/Study Population: Specific Aim 1: We hypothesize ARID5B SNP rs7090445 disrupts intrinsic enhancer function. Identification of critical DNA looping events impacted by ARID5B variants using Capture C. Affinity purification-mass spectrometry to identify potential ARID5B transcription mediators. Specific Aim 2: We hypothesize the B-ALL-associated SNP leads to a partial human B-cell differentiation block. Utilize Cas9-mediated homology-directed repair to create ARID5B SNP in primary human hematopoietic stem cells. Gene-edited HSCs will be differentiated into B cells using an ex vivo system. Fluorescence-activated cell sorting to sort our pool of cells into stages of B-cell development spectrum. Amplicon sequencing and variant allele frequency of rs7090445 SNP to evaluate its impact on B-cell development. Results/Anticipated Results: This proposal is conceptually innovative as it seeks to understand the mechanism by which the B-ALL-associated SNP rs7090445 in intron 3 of ARID5B disrupts enhancer function and investigates its impact on human B-cell development. Future research will investigate a tumor-suppressive role of ARID5B and whether it constitutes a “first-hit” of leukemogenesis. Discussion/Significance of Impact: Successful completion of this research will elucidate the critical role of the B-ALL-associated ARID5B SNP rs7090445 in human B-cell development and leukemogenesis. As this SNP is more prevalent in Hispanic/Latino populations, it will also provide crucial insights into the genetic factors behind the elevated incidence of B-ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4345d4a2e4bf5156f2152b887906b55727b5a5a1" target='_blank'>
              71 Investigating gene regulatory mechanisms associated with B-cell acute lymphoblastic leukemia incidence in Hispanic/Latino populations
              </a>
            </td>
          <td>
            Clarissa Garcia, Julian Grandvallet, Illaria Iacobucci, Jun Yang, C. Mullighan, Tzu Phang, Matthew T. Witkowski
          </td>
          <td>2025-03-26</td>
          <td>Journal of Clinical and Translational Science</td>
          <td>0</td>
          <td>103</td>
        </tr>

        <tr id="Tumorigenesis in mammals is driven by inherited genetic variants, environmental factors and random errors during normal DNA replication that lead to cancer-causing mutations. These factors initiate uncontrolled cellular proliferation and disrupt the regulation of critical checkpoints. A few mammalian species possess unique protective mechanisms that enable them to resist widespread cancer development and achieve longevity. Tissue-specific tumor protection adds another layer of complexity to this diversity. Breast cancer is a leading cause of human mortality, particularly among females. Driven by the need for new strategies in treatment and prevention, this opinion article explores and supports the idea that herbivores are more resistant to mammary cancer than carnivores and omnivores. This diversity has occurred despite the remarkably similar basic mammary biology. Herbivores’ meatless diet cannot explain the differences in cancer resistance, which have accompanied species segregation since the Jurassic era. To investigate the causes of this diversity, the characteristics of tumorigenesis in the human breast—and to a lesser extent in other carnivores—have been compared with data from retrospective analyses of bovine mammary tumor development across various locations over the past century. Well-established genomic, cellular, and systemic triggers of breast cancer exhibit different, or less pronounced tissue-specific activity in the bovine mammary gland, accompanied by novel bovine-specific protective mechanisms. Together, these factors contribute to the near absence of breast cancer in bovines and offer a basis for developing future anticancer strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0cb9fafc9e4ae8646c09de8af8d088ea1242ab7" target='_blank'>
              Mammalian Species-Specific Resistance to Mammary Cancer
              </a>
            </td>
          <td>
            I. Barash
          </td>
          <td>2025-03-06</td>
          <td>Journal of Mammary Gland Biology and Neoplasia</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Abstract Oncogene activation in normal untransformed cells induces DNA replication stress and creates a dependency on DNA damage response (DDR) mechanisms for cell survival. Different oncogenic stimuli signal via distinct mechanisms in every cancer setting. The DDR is also pathologically reprogrammed and deployed in diverse ways in different cancers. Because mutant KRAS is the driver oncogene in 90% of pancreatic ductal adenocarcinomas (PDACs), here we have investigated DDR mechanisms by which KRAS-induced DNA replication stress is tolerated in normal human pancreatic epithelial cells [human pancreatic nestin-expressing (HPNE) cells]. Using a candidate screening approach, we identify TRIP13 as a KRASG12V-induced messenger RNA that is also expressed at high levels in PDAC relative to normal tissues. Using genetic and pharmacological tools, we show that TRIP13 is necessary to sustain ongoing DNA synthesis and viability specifically in KRASG12V-expressing cells. TRIP13 promotes survival of KRASG12V-expressing HPNE cells in a homologous recombination (HR)-dependent manner. KRASG12V-expressing HPNE cells lacking TRIP13 acquire hallmark HR deficiency phenotypes, including sensitivity to inhibitors of translesion synthesis and poly-ADP ribose polymerase. Established PDAC cell lines are also sensitized to intrinsic DNA damage and therapy-induced genotoxicity following TRIP13 depletion. Taken together, our results expose TRIP13 as an attractive new and therapeutically tractable vulnerability of KRAS-mutant PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c52be837d7841e9eb90ef56e65d2eb6df322017a" target='_blank'>
              TRIP13 protects pancreatic cancer cells against intrinsic and therapy-induced DNA replication stress
              </a>
            </td>
          <td>
            J. R. Anand, G. Droby, Sayali Joseph, Urvi Patel, Xingyuan Zhang, Jeffrey A. Klomp, C. Der, Jeremy E. Purvis, Samuel C. Wolff, Jessica Bowser, Cyrus Vaziri
          </td>
          <td>2025-03-01</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>122</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae8281399f0352c6d8c9ddd4ab50516eea1c7526" target='_blank'>
              CRISPR-Cas Systems: A Functional Perspective and Innovations
              </a>
            </td>
          <td>
            Carla Navarro, , Pablo Durán, Ana Castro, Andrea Díaz, Clímaco Cano, Ana-Karina Carbonell-Zabaleta, Donny-Sabrith Solano-Jimenez, Diego Rivera-Porras, J. Contreras-Velásquez, Valmore Bermúdez
          </td>
          <td>2025-04-12</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/902f148702ebadfdb43908d49c0509768bb206ea" target='_blank'>
              Expression of Human Endogenous Retrovirus env-gene product is a hallmark of sidedness in operable colorectal cancer.
              </a>
            </td>
          <td>
            M. Dolci, Ivan Civettini, Pietro Francesco Bagnoli, Wafa Toumi, L. Signorini, Roberto Crocchiolo, Kevin K Maina, Federica Perego, Pasquale Ferrante, Carolina Scagnolari, Marco Bregni, S. Delbue
          </td>
          <td>2025-03-11</td>
          <td>Oncology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb7435e798afc5c276c5bcf5f266a5cdfa975999" target='_blank'>
              Abstract B008: Targeting DNA damage responsive pathways in cancer therapy
              </a>
            </td>
          <td>
            Junjie Chen
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Ovarian cancer remains one of the most lethal gynecological malignancies owing to its high recurrence rate and chemotherapeutic resistance. MYC is a well-known proto-oncogene that is frequently amplified in ovarian cancer and has been implicated in drug resistance. Previously, we established a new promoter–reporter system combined with a CRISPR activation library to identify unknown MYC regulators, and M1AP was identified as a novel MYC regulator. However, considering the insufficient explanation for the absence of guide RNA (gRNA) of MYC, this present study explored methods to prevent the gRNA of MYC itself from binding. This study first modified the promoter–reporter vector to improve its quality, then conducted CRISPR screening and analyzed candidate genes as MYC promoter regulators using next-generation sequencing in OVSAHO ovarian cancer cells. Eighty-six genes had ≥ 1000 reads, and Pearson’s correlation coefficient analysis was performed on the cBioPortal of the Cancer Genomics database. Fourteen genes were identified as candidate MYC regulators with positive and significant correlations with MYC. Seven genes, including CYP4v2, ASPH, ANP32D, PCED1A, ABI1, FUZ, and HOOK2, demonstrated significantly higher luciferase activity than the control genes. Four genes, including ABI1, PCED1A, HOOK2, and CYP4v2, activated the MYC promoter, which showed over twofold higher activity than the control when overexpressed using a vector. In conclusion, four genes that activate MYC promoters were identified in an ovarian cancer cell line using the CRISPR library system with a modified promoter–reporter tool. These results will prove helpful in the development of novel treatment strategies for ovarian cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s13048-025-01644-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9220d2085ff0b29849fe6d91dbca5cdaecc789b" target='_blank'>
              Modified screening of MYC promotor region elements using the CRISPR library in ovarian cancer
              </a>
            </td>
          <td>
            Akiko Yamamoto, Yosuke Tanaka, S. Ishibashi, Masumi Ikeda, Keisuke Sugita, Masanori Ono, Hirotaka Nishi, Morito Kurata
          </td>
          <td>2025-04-02</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Coupling single-cell transcriptomics with tumor-initiating cell enrichment identified IFN response gene expression not previously reported in bulk RNA-sequencing–derived signatures and proposed IFN/STAT1 signaling as a candidate therapeutic target in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b49032c731c7f43ad3396d95549c8d805e477fd3" target='_blank'>
              Lineage Tracing and Single-Cell RNA Sequencing Reveal a Common Transcriptional State in Breast Cancer Tumor–Initiating Cells Characterized by IFN/STAT1 Activity
              </a>
            </td>
          <td>
            Eric P Souto, Ping Gong, John D. Landua, R. Srinivasan, Abhinaya Ganesan, L. Dobrolecki, S. C. Purdy, Xingxin Pan, Michael Zeosky, Anna Chung, , H. Ford, Michael T. Lewis, Connie Cepko, Dr. Kenneth L. Scott, Dr. Michael Davidson
          </td>
          <td>2025-02-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a71da6481387936a0712c7963a43d9bba36fd469" target='_blank'>
              Defeating lethal cancer: Interrupting the ecologic and evolutionary basis of death from malignancy.
              </a>
            </td>
          <td>
            Kenneth J. Pienta, Patrick L Goodin, S. Amend
          </td>
          <td>2025-03-09</td>
          <td>CA: a cancer journal for clinicians</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca1fa79a96dc2bc5104b7e66300caf2368b25848" target='_blank'>
              Ribosome specialization by cancer-associated ribosomal protein mutations: progress made and open questions.
              </a>
            </td>
          <td>
            Marino Caruso, K. De Keersmaecker
          </td>
          <td>2025-03-06</td>
          <td>Philosophical transactions of the Royal Society of London. Series B, Biological sciences</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee23eae46a9ddd2433ad4acafc75ff3551fa2d98" target='_blank'>
              ANGEL2 modulates wildtype TP53 translation and doxorubicin chemosensitivity in colon cancer.
              </a>
            </td>
          <td>
            C. A. Lucchesi, S. Mantrala, Darren Tran, N. Batra, Avani Durve, Conner Suen, Jin Zhang, Paramita M Ghosh, Xinbin Chen
          </td>
          <td>2025-03-07</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89e38976a47d12226a011902bfacb13e2fb36e59" target='_blank'>
              Genome-wide CRISPR screen in human T cells reveals regulators of FOXP3.
              </a>
            </td>
          <td>
            Kelvin Y. Chen, Tatsuya Kibayashi, Ambre Giguelay, Mayu Hata, Shunsuke Nakajima, N. Mikami, Yusuke Takeshima, Kenji Ichiyama, Ryusuke Omiya, Leif S. Ludwig, Kunihiro Hattori, Shimon Sakaguchi
          </td>
          <td>2025-03-26</td>
          <td>Nature</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Metastasis is an inefficient process requiring cancer cells to adapt metabolically for survival and colonization in new environments. The contributions of tumor metabolic reprogramming to lymph node (LN) metastasis and its underlying mechanisms remain elusive. Through single-cell RNA sequencing, we identified rare metastasis-initiating cells (MICs) with stem-like properties that drive early LN metastasis. Integrated transcriptome, lipidomic, metabolomic, and functional analyses demonstrated that MICs depend on oxidative phosphorylation (OXPHOS) fueled by fatty acid oxidation (FAO) in the lipid-rich LN microenvironment. Mechanistically, the NRF2-SLC7A11 axis promotes glutathione synthesis to mitigate oxidative stress, thereby enhancing stress resistance and metastatic potential of MICs. Inhibition of NRF2-SLC7A11 reduced LN metastasis and sensitized tumors to cisplatin. Clinically, elevated NRF2-SLC7A11 expression was observed in tumors, with high expression correlating with LN metastasis, chemoresistance, and poor prognosis in esophageal squamous cell carcinoma (ESCC). These findings highlight the pivotal roles of FAO-fueled OXPHOS and NRF2 in LN metastasis and suggest targeting these pathways as a promising therapeutic strategy for metastatic ESCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98f4385d990789c0b31148d9b9d56276aa03285e" target='_blank'>
              FAO-fueled OXPHOS and NRF2-mediated stress resilience in MICs drive lymph node metastasis.
              </a>
            </td>
          <td>
            Shan-Shan Li, Baifeng Zhang, Cuicui Huang, Yuying Fu, Yuying Zhao, Lanqi Gong, Yanan Tan, Huali Wang, Wenqi Chen, Jie Luo, Yu Zhang, Stephianie Ma, Li Fu, Chenli Liu, Jiandong Huang, Huai-qiang Ju, Anne Wing Mui Lee, Xin-Yuan Guan
          </td>
          <td>2025-04-11</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Background Prostate cancer is defined by the suppression of genes that suppress tumours and the activation of proto-oncogenes. These are the hallmarks of prostate cancer, and they have been linked to numerous genomic variations, which lead to unfavourable treatment outcomes. Prostate cancer can be categorised into various risk groups of tumour molecular subtypes grounded in the idea of genomic structural variations connected to TMPRSS2:ERG fusion and loss of PTEN. Research suggests that certain genomic alterations may be more prevalent or exhibit different patterns in prostate cancer tumours across populations. Studies have reported a higher frequency of PTEN loss and TMPRSS2:ERG fusion in prostate tumours of Black/African American men, which may contribute to the more aggressive nature of the disease in this population. Thus, therapeutically important information can be obtained from these structural variations, including correlations with poor prognosis and disease severity. Methods Peer-reviewed articles from 1998 to 2024 were sourced from PubMed and Google Scholar. During the review process, the following search terms were employed: “Tumour suppressor genes OR variations OR alterations OR oncogenes OR diagnostics OR ethnicity OR biomarkers OR prostate cancer genomics OR prostate cancer structural variations OR tumour and molecular subtypes OR therapeutic implications OR immunotherapy OR immunogenetics.” Results There was a total of 13,012 results for our search query: 5,903 publications from Google Scholar with the patent and citation unchecked filer options, and 7127 articles from PubMed with the abstract, free full text, and full-text options selected. Unpublished works were not involved. Except for four articles published between 1998 and 1999, all other selected articles published in 2000 and later were considered. However, papers with irrelevant information or redundant or duplicate content were not chosen for this review. Thus, 134 met the inclusion criteria and were ultimately retained for this review. Conclusion This review extracted 134 relevant articles about genomic structure variations in prostate cancer. Our findings demonstrate the importance of PTEN and TMPRSS2:ERG fusion and tumour molecular subtyping in prostate cancer precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5a2e35d6e52b6e77bd4c02088e02ccca569cc23" target='_blank'>
              Exploring therapeutic applications of PTEN, TMPRSS2:ERG fusion, and tumour molecular subtypes in prostate cancer management
              </a>
            </td>
          <td>
            Fidelis Charles Bugoye, R. Torrorey-Sawe, Richard Biegon, N. Dharsee, F. Mafumiko, Herry G. Kibona, Said Aboud, Kirtika Patel, Simeon K Mining
          </td>
          <td>2025-03-17</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03e021b5185f33e152b622ca1b885b9675c23a62" target='_blank'>
              The Endo-GeneScreen Platform Identifies Drug-Like Probes that Regulate Endogenous Protein Levels within Physiological Contexts
              </a>
            </td>
          <td>
            Preston Samowitz, Laszlo Radnai, T. Vaissiere, S. Michaelson, Camilo Rojas, Ryan Mitchell, Murat Kilinc, Austin Edwards, Justin Shumate, Richard Hawkins, V. Fernández-Vega, Timothy P. Spicer, Louis Scampavia, Theodore Kamenecka, Courtney A. Miller, Gavin Rumbaugh
          </td>
          <td>2025-03-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Background. Mutations in homologous recombination repair (HRR) genes (BRCA2, ATM, CHEK2, NBN, etc.) are found in 20–25 % of patients with metastatic prostate cancer (PC) and are an indication for prescription of PARP inhibitors. Sensitivity to these drugs results from homologous recombination deficiency (HRD) in the tumor. It is currently not fully elucidated which HRR genes besides BRCA1/2 cause HRD. The aim of the study was to evaluate the presence of homologous recombination deficiency in pc associated with mutations in different HRR genes. Material and methods. Paired tumor and normal DNA samples from 272 pc patients were examined using the HiSNP Ultra Panel v1.0 NGS panel (Nanodigmbio). Tumor copy number variation profiles were used to obtain the HRD score, which was determined as the unweighted sum of three characteristics: the number of chromosomal regions with loss of heterozygosity (LOH), large scale state transitions (LST), and telomeric allelic imbalances (TAI). HRD scores in different pc categories were compared using the Mann–Whitney test. Results. The studied case series included 58 pcs with pathogenic/ likely pathogenic mutations in at least one of 34 HRR genes, and 214 pcs without HRR mutations. The median HRD scores in the groups with BRCA2, ATM, CHEK2, other HRR mutations and without HRR mutations were 41, 22.5, 7.5, 7.5, 14 and 9, respectively. The HRD score was significantly higher in BRCA2-associated tumors than in other PC categories (p<0.01 for all comparisons), except in cases with atm mutations, where the difference did not reach formal significance (p=0.051). Homologous recombination deficiency, defined as HRD score ≥25, was observed in 19/58 (32.8 %) PCs with HRR mutations and 40/214 (18.7 %) tumors without HRR mutations (p=0.03). In tumors without HRR mutations, the HRD score was significantly higher in the presence of somatic TP53 mutations (p<0.0001). Conclusion. In contrast to BRCA2-associated PCs, most tumors with mutations in the CHEK2, NBN, BLM, FANCM, BRCA1 genes are not characterized by homologous recombination deficiency. in the case of ATM gene lesions, approximately half of pcs have a high HRD score. Testing for HRD score allows identification of a significant proportion (5–19 %, depending on the threshold chosen) of tumors with HRD among pcs that do not contain mutations in HRR genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/475a2bac32f3b3fba84f14504ff2c301042d61c8" target='_blank'>
              Analysis of homologous recombination deficiency in prostate cancer
              </a>
            </td>
          <td>
            A. Iyevleva, S. Aleksakhina, A. Sokolenko, E. A. Otradnova, A. S. Nikitina, K. A. Kashko, M. V. Syomina, N. S. Morozova, A. D. Shestakova, E. Kuligina, E. Imyanitov
          </td>
          <td>2025-03-18</td>
          <td>Siberian journal of oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="X-chromosome inactivation (XCI) is a fundamental mechanism in placental mammals that compensates for gene dosage differences between sexes. Using methylation levels of genes under XCI, we establish defective levels of XCI as a new source of interindividual variation among cancer types in females, characterized by a significant and consistent lowering of XIST expression and enrichment of differentially expressed genes under XCI. We show that defective XCI is an additive factor to the cancer risk of XCI escape deregulation in women. Defective XCI of more than 10% has an attributable risk of 40% among 12 different cancers from The Cancer Genome Atlas. Validations between independent studies of breast cancer samples show that defective XCI increases triple-negative subtype frequency, decreases survival rates, and is reduced by chemotherapy treatment. Mechanistically, it is associated with somatic mutations at TP53 and top MYC gains. In independent studies, defective XCI is detectable in blood and increases with aging, menopause, and cancer diagnosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/303ef18264231a4b1a297a874c3c158036c417de" target='_blank'>
              Defective X-chromosome inactivation and cancer risk in women
              </a>
            </td>
          <td>
            A. Cáceres, L. Pérez-Jurado, Albert Alegret-García, Varun B. Dwaraka, Ryan Smith, Juan R. González
          </td>
          <td>2025-02-22</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Variants of uncertain significance (VUS) limit the actionability of genetic testing. A prominent example is MUTYH, a base excision repair factor associated with polyposis and colorectal cancer, which has a pathogenic variant carrier rate approaching 1 in 50 individuals in some populations. To systematically interrogate variant function in MUTYH, we coupled deep mutational scanning with a DNA repair reporter containing its lesion substrate, 8OG:A. Our variant-to-function map covers >97% of all possible MUTYH point variants (n=10,941) and achieves 100% accuracy classifying the pathogenicity of known clinical variants (n=247). Leveraging a large clinical registry, we observe significant associations with colorectal polyps and cancer, with more severely impaired missense variants conferring greater risk. We recapitulate known functional differences between pathogenic founder alleles, and highlight sites of complete missense intolerance, including residues that intercalate DNA and coordinate essential Zn2+ or Fe-S clusters. This map provides a resource to resolve the 1,032 existing missense VUS and 90 variants with conflicting interpretations in MUTYH, and demonstrates a scalable strategy to interrogate other clinically relevant DNA repair factors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/123ddb79d358a7742980d1fc41247ef4a3c39e8c" target='_blank'>
              Saturation mapping of MUTYH variant effects using DNA repair reporters
              </a>
            </td>
          <td>
            Shelby L. Hemker, Ashley Marsh, Felicia Hernandez, Elena Glick, Grace Clark, Alyssa Bashir, Krystal Jiang, Jacob O. Kitzman
          </td>
          <td>2025-03-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdc6f9aae4bbf3240c3af412977bf68d0c88eb1d" target='_blank'>
              Non-viral intron knock-ins for targeted gene integration into human T cells and for T-cell selection.
              </a>
            </td>
          <td>
            Theodore L. Roth, Johnathan Lu, Alison McClellan, Courtney Kernick, Oliver Takacsi-Nagy, Ansuman T. Satpathy
          </td>
          <td>2025-03-07</td>
          <td>Nature biomedical engineering</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f347f7efa84f23c234288cd33445bd958b696c90" target='_blank'>
              Epigenetic modifications in breast cancer: from immune escape mechanisms to therapeutic target discovery
              </a>
            </td>
          <td>
            Ziyu Kang, Junlin Wang, Jiyan Liu, Li Du, Xiaofei Liu
          </td>
          <td>2025-04-17</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1013bf1014632de45f27d6f58428ba3cd06f739c" target='_blank'>
              Gene editing using CRISPR-Cas9 technology: potential implications in assisted reproduction.
              </a>
            </td>
          <td>
            Gizem Nur Sahin, Emre Seli
          </td>
          <td>2025-04-11</td>
          <td>Current opinion in obstetrics & gynecology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is a highly malignant tumor in women, characterized by high morbidity, mortality, and recurrence rates. Although surgical treatment, radiotherapy, and chemotherapy are the mainstays of current treatment methods, the high heterogeneity of TNBC results in unsatisfactory outcomes with low 5-year survival rates. Rapid advancements in omics technology have propelled the understanding of TNBC molecular biology. The emergence of single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics (ST) has significantly enhanced knowledge of tumor heterogeneity and the distribution, functionality, and intercellular interactions of various cell types within the tumor microenvironment, including tumor cells, T cells, B cells, macrophages, and fibroblasts. The present study provides an overview of the technical characteristics of scRNA-seq and ST, highlighting their applications in exploring TNBC heterogeneity, cell spatial distribution patterns, and intercellular interactions. This review aims to enhance the comprehension of TNBC at the cellular level for the development of effective therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9db2a16a0771f525b51537059ff8768102022813" target='_blank'>
              Analysis of single-cell and spatial transcriptomics in TNBC cell-cell interactions
              </a>
            </td>
          <td>
            Yan Xin, Qiji Ma, Qiang Deng, Tielin Wang, Dongxu Wang, Gang Wang
          </td>
          <td>2025-02-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2f7401161a6069ba8245d4505d68dfc98e30045" target='_blank'>
              The Feasibility of Using CRISPR-Cas9 to Knock Out IL-6 Receptor in CAR-T Cells to Mitigate Cytokine Release Syndrome in Cancer Therapy
              </a>
            </td>
          <td>
            Zhiyuan Wang
          </td>
          <td>2025-02-21</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Mosaic chromosomal alterations of the autosomes (aut-mCAs) are large structural somatic mutations which cause clonal hematopoiesis and increase cancer risk. Here, we detected aut-mCAs in 1,011,269 participants across four biobanks. Through integrative analysis of the minimum critical region and inherited genetic variation, we found that proto-oncogenes exclusively drive chromosomal gains, tumor suppressors drive losses, and copy-neutral events can be driven by either. We identified three novel inherited risk loci in CHI3L2, HLA class II, and TERT that modulate aut-mCA risk and ten novel aut-mCA-specific loci. We found specific aut-mCAs are associated with cardiovascular, cerebrovascular, or kidney disease incidence. High-risk aut-mCAs were associated with elevated plasma protein levels of therapeutically actionable targets: NPM1, PARP1, and TACI. Participants with multiple high-risk features such as high clonal fraction, more than one aut-mCA, and abnormal red cell morphology had a 50% cumulative incidence of blood count abnormalities over 2 years. Leveraging inherited variation, we causally established aut-mCAs as premalignant lesions for chronic lymphocytic leukemia. Together, our findings provide a framework integrating somatic mosaicism, germline genetics, and clinical phenotypes to identify individuals who could benefit from preventative interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f202aec9ed88d0660af9bd306eaa9b009384d1c" target='_blank'>
              Genetic drivers and clinical consequences of mosaic chromosomal alterations in 1 million individuals
              </a>
            </td>
          <td>
            K. Zhao, Y. Pershad, H. Poisner, X. Ma, K. Quade, C. Vlasschaert, T. Mack, N. K. Khankari, K. von Beck, J. Brogan, A. Kishtagari, R. Corty, Y. Li, Y. Xu, A. Reiner, P. Scheet, P. Auer, A. G. Bick
          </td>
          <td>2025-03-06</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="In cancer, metabolic adaptations such as fatty acid oxidation are frequently associated with immune suppression, oxidative stress and tumor promotion. Using unbiased metabolomics and quantitative PCR, we found that itaconate, the product of the decarboxylation of cis-aconitate, was among the most highly upregulated metabolites in tumor-associated macrophages of the peritoneal space. We show that itaconate is important for fatty acid beta-oxidation promotion of oxidative phosphorylation and generation of mitochondrial reactive oxygen species in macrophages that facilitate tumor growth. Genetic interference with itaconate production confirms a pro-tumoral role for itaconate and highlight itaconate as a therapeutic target for oncology. To evaluate this, we screened a library of over 21,000 compounds using a high throughput primary screen to identify compounds capable of interacting with Irg1, the protein which catalyzes itaconate production. Inhibitory or activating activity was next confirmed by mass spectrometry using both in vitro reactions and cell-based assays. Finally, tumor-bearing mice were treated with the most promising candidate compounds. Using this approach, we identified multiple compounds capable of significantly reducing peritoneal tumor burden and are evaluating them for dependency on itaconate inhibition. Our findings reveal metabolic-based pathways which highlight the feasibility of using newly identified compounds for the treatment of peritoneal cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ed1b8ac5a92acf08f4046e4ef4f073a4225ecae" target='_blank'>
              Targeting Metabolic Programming in the Tumor Microenvironment
              </a>
            </td>
          <td>
            Jonathan M. Weiss, Erika M. Palmieri, Marieli Gonzalez-Cotto, Ian A. Bettencourt, Shilpa Kurian, Barry O’Keefe, Daniel W. McVicar
          </td>
          <td>2025-03-25</td>
          <td>South East European Journal of Immunology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1178b1ac2d65f9262b22b808cb25c2ac38fa27ab" target='_blank'>
              Mutation of SMARCA4 Induces Cancer Cell-Intrinsic Defects in the Enhancer Landscape and Resistance to Immunotherapy.
              </a>
            </td>
          <td>
            Yawen Wang, Ismail M. Meraz, Md Qudratullah, Sasi Kotagiri, Yanyan Han, Yuanxin Xi, Jing Wang, Kadir C Akdemir, Jack A Roth, Yonathan Lissanu
          </td>
          <td>2025-03-13</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Patients with chronic, indolent myeloproliferative neoplasms (MPNs) are at risk for transformation to highly lethal leukemia, although targetable mechanisms driving progression remain elusive. We discovered that the High Mobility Group A1 (HMGA1) gene is up-regulated with MPN progression in patients and required for evolution into myelofibrosis (MF) or acute myeloid leukemia (AML) in preclinical models. HMGA1 encodes the HMGA1 epigenetic regulators that modulate the chromatin state during embryogenesis and tissue regeneration. While HMGA1 is silenced in most differentiated cells, it becomes aberrantly re-expressed in JAK2 mutant (JAK2-V617F) MPN, with the highest levels after transformation to secondary MF or AML. Here, we review recent work highlighting HMGA1 function in MPN progression. Though underlying mechanisms continue to emerge, increasing evidence suggests that HMGA1 functions as a “chromatin key” required to “unlock” regions of the genome involved in clonal expansion and progression in MPN. Together, these findings illuminate HMGA1 as a driver of MPN progression and a promising therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55614432dafb795932a39f2af22dcb87acd53c87" target='_blank'>
              High Mobility Group A1 Chromatin Keys: Unlocking the Genome During MPN Progression
              </a>
            </td>
          <td>
            Linda S. Resar, L. Luo
          </td>
          <td>2025-02-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Somatic alterations, like mutations and copy number changes, driver oncogenesis and cancer progression. Their inhibition has been exploited in the clinic, with several targeted therapies approved for patients with specific mutations or amplifications. However, the response rate of these treatments remains low. The causes are several, ranging from clonal heterogeneity to off target binding. For this reason, CRISPR assays have been developed to study the exact effect of a gene’s deletion. Still, the results from them are puzzling with the same alterations responding different to knockout even in the same cellular context. For this reason, we have developed SAEG, a novel deep learning architecture for somatic alterations in cancer. Our architecture is able to model mutations and copy number alterations and protein-protein interactions to predict if a cell will be susceptible to a gene knockout. SAEG outperforms other models and we show that it learns patterns that can be traced back to the biochemical and biological properties of genes and amino acids. Code Availability https://github.com/Luisiglm/SAEG Contact luis.iglesiarmatinez@ucd.ie">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/caba93fa83f15f4694f814ad4d5d269b25293aad" target='_blank'>
              SAEG: A Novel Deep Learning Architecture for Somatic Alterations
              </a>
            </td>
          <td>
            Luis F. Iglesias-Martinez, Dan Yang Wester, Walter Kolch
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e7cad934b860ca66c2574dc37104aa0e67005c2" target='_blank'>
              Current updates regarding biogenesis, functions and dysregulation of microRNAs in cancer: Innovative approaches for detection using CRISPR/Cas13‑based platforms (Review).
              </a>
            </td>
          <td>
            Abdulaziz A. Aloliqi, Abdullah M. Alnuqaydan, Aqel Albutti, Basmah F. Alharbi, A. Rahmani, Amjad Ali Khan
          </td>
          <td>2025-04-11</td>
          <td>International journal of molecular medicine</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8314a19e58c7781d684d1f0ff425f592ad18a516" target='_blank'>
              OmicsCam Enables Trimodal Profiling of Mitochondrial Genome Editing.
              </a>
            </td>
          <td>
            Guoqiang Zhou, Ting Li, Jingjing Du, Chengpeng He, Yu Yang, Guanju Chen, Jie Li, Bin Shen, Weilin Pu, Jingwei Zhang, Zhenglong Gu
          </td>
          <td>2025-03-25</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Transfer RNAs (tRNAs) are among the most highly conserved and frequently transcribed genes. Recent studies have identified transcription-associated mutagenesis (TAM) as a significant contributor to sequence variation around tRNA loci. However, the extent to which TAM drives allelic variation in tRNAs remains unclear, largely due to the confounding effects of strong selection pressures to maintain their structural integrity. This complexity arises because TAM-induced mutations primarily involve nucleotide transitions, which tend to preserve base-pairing stability. To address this dichotomy at the population level, we analyzed tRNA allelic variation in contemporary Caenorhabditis elegans strains. We propose a model of tRNA microevolution driven by TAM and demonstrate that the observed secondary structure characteristics align with our predicted TAM-biased patterns. Furthermore, we developed a continuous Markov substitution model that incorporates TAM-specific mutational biases. This TAM-biased model fits the C. elegans tRNA data more effectively than standard models, such as the general time-reversible (GTR) model. Based on these results, we conclude that TAM plays a significant role in shaping tRNA allelic variation within populations. This finding is consistent with recent experimental studies on tRNA fitness in yeast but challenges prior theoretical and computational analyses that emphasize RNA base-pairing as a primary determinant in genotype-phenotype systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e7c029829a4d467bf5e0a65940547deb03e8113" target='_blank'>
              How much is Transcription-associated Mutagenesis Driving tRNA Microevolution?
              </a>
            </td>
          <td>
            Hector Baños, Ling Wang, Corinne Simonti, Annalise B. Paaby, Christine Heitsch
          </td>
          <td>2025-03-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2eed7b2fdc5b17bdbf958e634178449e843736cc" target='_blank'>
              Genome-Wide CRISPR Screening Identifies Cellular Factors Controlling Nonviral Genome Editing Efficiency
              </a>
            </td>
          <td>
            Shivani Saxena, Meha Kabra, Amr Abdeen, Divya Sinha, Min Zhu, Ruosen Xie, Giovanni Hanstad, Maria A. Fernandez Zepeda, David M. Gamm, B. Pattnaik, Shaoqin Gong, Krishanu Saha
          </td>
          <td>2025-03-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Epidemiological studies have revealed significant sex differences in the incidence of tumors unrelated to reproductive functions, with females demonstrating a lesser risk and a better response to therapy than males. However, the reasons for these disparities are still unknown and cancer therapies are generally sex-unbiased. The tumor-suppressor protein p53 is a transcription factor that can activate the expression of multiple target genes mainly involved in the maintenance of genome stability and tumor prevention. It is encoded by TP53, which is the most-frequently mutated gene in human cancers and therefore constitutes an attractive target for therapy. Recently, evidence of sex differences has emerged in both p53 regulations and functions, possibly providing novel opportunities for personalized cancer medicine. Here, we will review and discuss current knowledge about sexual disparities in p53 pathways, their role in tumorigenesis and cancer progression, and their importance in the therapy choice process, finally highlighting the importance of considering sex contribution in both basic research and clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a84bb2275916188060a8234c6ff4a6afe8ab155d" target='_blank'>
              Sex Disparities in P53 Regulation and Functions: Novel Insights for Personalized Cancer Therapies
              </a>
            </td>
          <td>
            Miriana Cardano, G. Buscemi, L. Zannini
          </td>
          <td>2025-02-28</td>
          <td>Cells</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ad75c766d4e07168f2e27a4c036f888ffea4b32" target='_blank'>
              Integrating model systems and genomic insights to decipher mechanisms of cancer metastasis.
              </a>
            </td>
          <td>
            Michelle M Leung, C. Swanton, N. Mcgranahan
          </td>
          <td>2025-03-10</td>
          <td>Nature reviews. Genetics</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Despite their deleterious effects, small insertions and deletions (InDels) have received far less attention than substitutions. Here we generated isogenic CRISPR-edited human cellular models of postreplicative repair dysfunction (PRRd), including individual and combined gene edits of DNA mismatch repair (MMR) and replicative polymerases (Pol ε and Pol δ). Unique, diverse InDel mutational footprints were revealed. However, the prevailing InDel classification framework was unable to discriminate these InDel signatures from background mutagenesis and from each other. To address this, we developed an alternative InDel classification system that considers flanking sequences and informative motifs (for example, longer homopolymers), enabling unambiguous InDel classification into 89 subtypes. Through focused characterization of seven tumor types from the 100,000 Genomes Project, we uncovered 37 InDel signatures; 27 were new. In addition to unveiling previously hidden biological insights, we also developed PRRDetect-a highly specific classifier of PRRd status in tumors, with potential implications for immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63ec8a3709640e7f5608e28fbcbdbf6376d5a9c2" target='_blank'>
              A redefined InDel taxonomy provides insights into mutational signatures.
              </a>
            </td>
          <td>
            G. Koh, A. S. Nanda, Giuseppe Rinaldi, Soraya Boushaki, A. Degasperi, Cherif Badja, A. Pregnall, S. Zhao, Lucia Chmelova, Daniella Black, Laura Heskin, João M. L. Dias, Jamie Young, Yasin Memari, Scott Shooter, Jan Czarnecki, Matthew A Brown, Helen R Davies, Xueqing Zou, S. Nik-Zainal
          </td>
          <td>2025-04-10</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Background Targeted therapy is an effective strategy for the treatment of advanced and metastatic pancreatic cancer, one of the leading causes for cancer-related death worldwide. To address the limitations of existing targeted drugs, there is an urgently need to find novel targets and therapeutic strategies. Transcription factor FOS like 1 (FOSL1) is a potential therapeutic target for challenging pancreatic cancer, which contributes to the malignant progression and poor gnosis of pancreatic cancer. High mobility group A1 (HMGA1) is a nonhistone chromatin structural protein that contributes to malignant progression and poor prognosis of cancer. Methods Human FOSL1 complete RNA, shRNA against FOSL1 and shRNA against HMGA1 lentiviral recombination vectors were used to overexpress FOSL1 and knock down FOSL1 and HMGA1. RNA sequencing, Q-PCR and Western blots were used to investigate the mechanism of FOSL1 in regulating the proliferation of pancreatic cancer cells. The relationship between FOSL1 and HMGA1 were analyzed by co-immunoprecipitation Mass spectrometry, Q-PCR of chromatin immunoprecipitation and Western blots. The regulation of FOSL1 and HMGA1 in the invasion and migration, stemness, and multidrug efflux system were determined by transwell assay, sphere formation assay, immunofluorescence, Q-PCR and Western blots. Results We found that FOSL1 promoted the proliferation and progression of pancreatic cancer by trigging stemness, invasion and metastasis, and drug resistance. HMGA1 was a key downstream target regulated by FOSL1 at the transcriptional level and directly interacted with FOSL1. Knockdown of HMGA1 inhibited the proliferation of pancreatic cancer cells by regulating the expression of genes related to stemness, epithelial-mesenchymal transition and multidrug efflux system. Targeted inhibition of FOSL1 and HMGA1 expression significantly inhibited the proliferation of pancreatic cancer cells. Conclusion FOSL1 promote the malignant progression of pancreatic cancer by promoting HMGA1 expression. Targeting FOSL1 and HMGA1 in monotherapy or combination therapy is a promising strategy for the treatment of advanced and metastasis pancreatic cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06304-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cb9672d2fcbfed129bd061551ff24b1fe5d4981" target='_blank'>
              FOSL1 drives the malignant progression of pancreatic cancer cells by regulating cell stemness, metastasis and multidrug efflux system
              </a>
            </td>
          <td>
            Xiaolong Liu, Xueyan Zhang, Tingyu Zeng, Yali Chen, Liu Ye, Shuping Wang, Yulan Li
          </td>
          <td>2025-03-04</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cell death is critical in tumor biology. The common cancer therapies can cause cell death and alleviate tumor, while the cancer cells can develop a resistance to cell death and survive from the therapies. Thus, not only observing the alternative mechanisms of tumor cells resistant to cell death, but also understanding the intricate dynamics of cell death processes within the tumor microenvironment (TME), are essential for tailoring effective therapeutic strategies. High-throughput sequencing technologies have revolutionized cancer research by enabling comprehensive molecular profiling. Recent advances in single cell sequencing have unraveled the heterogeneity of TME components, shedding light on their complex interactions. In this review, we explored the interplay between cell death signaling and the TME, summarised the potential drugs inducing cell death in pre-clinical stage, reviewed some studies applying next-generation sequencing technologies in cancer death research, and discussed the future utilization of updated sequencing platforms in screening novel treatment methods targeted cell death. In conclusion, leveraging multi-omics technologies to dissect cell death signaling in the context of the TME holds great promise for advancing cancer research and therapy development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c40c61d7dcc3833421bbc3568f9b1a472c0c4795" target='_blank'>
              Cell death in tumor microenvironment: an insight for exploiting novel therapeutic approaches
              </a>
            </td>
          <td>
            Wenxin Wang, Tong Li, Kui Wu
          </td>
          <td>2025-03-10</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc7f2508afe51502faf6796ddc3154fedcbc86bc" target='_blank'>
              Gene therapy and gene therapy products introduced to market by 2022.
              </a>
            </td>
          <td>
            Cengiz Bereket, I. Kunter, Elaheh Ashrafian Bonab, Ghazal Footohi
          </td>
          <td>2025-04-10</td>
          <td>Nucleosides, nucleotides & nucleic acids</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Regulating the occurrence and progression of tumor cells at the epigenetic level is a new insight of clear cell renal cell carcinoma (ccRCC). Chromatin accessibility is an important pathway of epigenetic regulation, which may explain the mystery of tumor occurrence. Assay for transposase-accessible chromatin sequencing (ATAC-seq) provides insight into the epigenetic regulatory features of ccRCC, especially at the single-cell level. In this study, we performed scATAC-seq of 3 ccRCC samples and captured a total of 18,703 high-quality cell nuclei and 104,818 unique peaks. Our protocol for nuclear extraction was reliable and stable, which can be used to deal with fresh and frozen single-cell suspensions. We presented basic methods for scATAC-seq data analysis, such as cell clustering, gene activity scoring, cell subtype specific peaks, transcription factors, motif and motif footprinting analysis. Taken together, our data indicated the valuable epigenetic features of ccRCC, which will provide more references for the study of ccRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7f2a7adbad7b64718a49a79d5c94f7b319bf5e0" target='_blank'>
              Single-cell assay for transposase-accessible chromatin sequencing of human clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            Wenhao Lu, Yixuan Liu, Jiaping Li, Shengzhu Huang, Zheng Wen, Cheng Su, Zheng Lu, Zengnan Mo, Zhenyuan Yu
          </td>
          <td>2025-02-25</td>
          <td>Scientific Data</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="CAR-T cell therapies are revolutionizing the treatment of refractory and relapsed haematological malignancies, but many patients do not exhibit long-term responses, and these therapies are less effective against solid tumors. Poor persistence of CAR-T cells in patients is associated with therapeutic failure, highlighting the need to identify strategies promoting in vivo expansion. Here, we developed an in vivo competitive screening method to identify genes whose inactivation confers a selective advantage to CAR-T cells. Inactivation of 50 genes in a heterogeneous population of T cells expressing an EGFR-targeting CAR revealed that disruption of REGNASE-1, SOCS1, PTPN2, and P16/NK4A conferred a selective advantage to CAR-T cells in human lung tumor-bearing mice. Consistently, inactivation of these genes improved tumor eradication by CAR-T cells. Interestingly disruption of other genes, described to improve CAR-T cell function in other contexts, had a negative impact in this orthotopic lung tumor model. Further evaluation of long-term effects in a subcutaneous model, highlighted SOCS1 ablation as the most promising strategy for in vivo CAR-T cell amelioration. These results support the importance of evaluating CAR-T cell editing strategies in tumor-specific models and highlight the versatility of our screening approach as a pre-clinical tool for context-specific studies on CAR-T cells amelioration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b45e95cba718c0beef550e419bba24e1f199503b" target='_blank'>
              An in vivo CRISPR screen unveils top target genes to improve CAR-T cell efficacy in a solid tumor model
              </a>
            </td>
          <td>
            Mattia Fumagalli, Dongjie An, Luca Simula, Camille Combe, Lisa Aziez, Yannick Simoni, M. Alves-Guerra, Andrea Valentini, Maude Marchais, Anaïs Vermare, J. Moraly, Nadège Bercovici, E. Donnadieu, Frédéric Pendino
          </td>
          <td>2025-04-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fbfb7ef2eb024a4f3adfe9adb4228362e40780f" target='_blank'>
              The Influence of Epigenetics on Gene Expression: Mechanisms, Environmental Interactions and Clinical Applications in Disease Prevention and Treatment
              </a>
            </td>
          <td>
            Yun Ao
          </td>
          <td>2025-02-21</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0afd262ffd1c8e8c07bf5791b18064c23b8384ea" target='_blank'>
              Epigenetic Alterations in Cancer: The Therapeutic Potential of Epigenetic Drugs in Cancer Therapy
              </a>
            </td>
          <td>
            Preeti Gupta
          </td>
          <td>2025-04-05</td>
          <td>Drugs and Drug Candidates</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Genomic defects caused by truncating mutations or deletions in the Retinoblastoma tumor suppressor gene (RB1) are frequently observed in many cancer types leading to dysregulation of the RB pathway. Here, we propose an integrative proteogenomic approach that predicts cancers with dysregulation in the RB pathway. A subset of these cancers, which we term as “RBness,” lack RB1 genomic defects and yet phenocopy the transcriptional profile of RB1-defective cancers. We report RBness as a pan-cancer phenomenon, associated with patient outcome and chemotherapy response in multiple cancer types, and predictive of CDK4/6 inhibitor response in estrogen-positive breast cancer. Using RNA interference and a CRISPR-Cas9 screen in isogenic models, we find that RBness cancers also phenocopy synthetic lethal vulnerabilities of cells with RB1 genomic defects. In summary, our findings suggest that dysregulation of the RB pathway in cancers lacking RB1 genomic defects provides a molecular rationale for how these cancers could be treated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1b769a127908e214629b60c835bc205b258e11d" target='_blank'>
              Proteogenomic discovery of RB1-defective phenocopy in cancer predicts disease outcome, response to treatment, and therapeutic targets
              </a>
            </td>
          <td>
            Jacopo Iacovacci, R. Brough, Fatemeh Ahmadi Moughari, J. Alexander, H. Kemp, Andrew N J Tutt, R. Natrajan, C. J. Lord, Syed Haider
          </td>
          <td>2025-03-26</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Background The reactivation of developmental silenced γ-globin genes (HBG1/2) has shown promise as a therapeutic strategy for improving symptoms of β-hemoglobinopathies. Currently, the focus of therapeutic targets is primarily on the major fetal hemoglobin suppressors, such as BCL11A and ZBTB7A and of their binding sites on the proximal HBG promoter. However, the role of the distal HBG promoter in regulating gene expression remains to be explored. Methods We used CRISPR/Cas9 system to edit the distal HBG promoter. In vitro and in vivo assays, as well as engrafted NCG-Kit-V831M mice, were used for functional validation and mechanistic studies. Results We discovered an insertion of nucleotide A (insA) between − 1368 and − 1369 bp upstream of the TSS in HBG2 resulting in remarkable increase in γ-globin expression in HUDEP-2 cells. We also observed elevated γ-globin expression in human CD34+ erythroid progenitor cells from healthy individuals and those with β-thalassemia when introducing insA mutation. Similarly, engrafted NCG-Kit-V831M mice showed increased γ-globin expression. Importantly, neither did insA have any off-target effects nor did it affect the maturation of erythroid cells. Furthermore, we found that the insA mutation created a binding site for the transcription activator FOXO3, which was activated by AMPK. Additionally, introducing insA specifically demethylated the − 162 CpG site on HBG promoter by reducing the enrichment of DNA methyltransferase 3 A (DNMT3A). At the same time, it activated histone modifications and RNA polymerase II (Pol II) in both distal and proximal HBG promoter and might inhibit the binding of BCL11A and ZBTB7A on -115 and − 200 sites on the HBG promoter respectively. In addition, combination of insA and the − 115 or -200 editing targets resulted in an amplify effect in reactivating γ-globin genes expression. Conclusions Overall, we presented the preclinical data to support the role of insA on regulating γ-globin expression using CD34+ HSPC cells derived from healthy donors or patients with β-thalassemia, and subsequently engrafted mice. Our study suggests that introducing insA mutation leads to significantly boosted fetal globin levels and uncovers new safe therapeutic target or strategy for β-hemoglobinopathies. Supplementary Information The online version contains supplementary material available at 10.1186/s40164-025-00626-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e602d03d7e5d834d0278abcf5359fc05d4a105ed" target='_blank'>
              A one-base therapeutic insertion in the HBG2 distal promoter reactivates γ-globin expression
              </a>
            </td>
          <td>
            Xiuqin Bao, Yuanyi Gao, Xiaoyi Chen, Zhongju Wang, Xiaoqin Feng, Liren Wang, Jing Du, Yuhua Ye, Feijing Chen, Li Du, Aihua Yin, Xiangmin Xu
          </td>
          <td>2025-03-28</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22ab73f4c8cf8c6df493069f963fc3cc43cc9298" target='_blank'>
              The epigenetic basis of hepatocellular carcinoma - mechanisms and potential directions for biomarkers and therapeutics.
              </a>
            </td>
          <td>
            Hong-Yi Lin, Ah-Jung Jeon, Kaina Chen, Chang Jie Mick Lee, Lingyan Wu, S. Chong, C. Anene-Nzelu, R. Foo, P. Chow
          </td>
          <td>2025-03-08</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Background: Engineers seeking to generate natural product analogs through altering modular polyketide synthases (PKSs) face significant challenges when genomically editing large stretches of DNA. Results: We describe a CRISPR-Cas9 system that was employed to reprogram the PKS in Streptomyces venezuelae ATCC 15439 that helps biosynthesize the macrolide antibiotic pikromycin. We first demonstrate its precise editing ability by generating strains that lack megasynthase genes pikAI-pikAIV or the entire pikromycin biosynthetic gene cluster but produce pikromycin upon complementation. We then employ it to replace 4.4-kb modules in the pikromycin synthase with those of other synthases to yield two new macrolide antibiotics with activities similar to pikromycin. Conclusion: Our gene-editing tool has enabled the efficient replacement of extensive and repetitive DNA regions within streptomycetes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/645ccccd07878f419f6834738fe7805cc7df2ebb" target='_blank'>
              A CRISPR-Cas9 System for Knock-out and Knock-in of High Molecular Weight DNA Enables Module-Swapping of the Pikromycin Synthase in its Native Host
              </a>
            </td>
          <td>
            Zhe-Chong Wang, Hayden Stegall, Takeshi Miyazawa, A. Keatinge-Clay
          </td>
          <td>2025-03-27</td>
          <td>Research Square</td>
          <td>0</td>
          <td>31</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [1, 10],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>