<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-01-13 09:27:01 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental and Molecular Medicine, Experimental & Molecular Medicine</td>
          <td>4</td>
          <td>49</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum.
                </a>
              </td>
          <td>
            Gustavo S França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>14</td>
          <td>41</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Phenotypic plasticity, the capacity of cells to transition between distinct phenotypic and lineage states over time, is a genetically and epigenetically encoded trait essential for normal development and adult tissue homeostasis. In cancer, phenotypic plasticity programs can be deployed aberrantly to enable disease progression and acquired therapeutic resistance. Cancer phenotypic plasticity is a current barrier to achieving cures for advanced cancers using available molecularly targeted therapies. This review summarizes the complex and interconnected molecular pathways implicated in phenotypic plasticity, both in the context of normal tissue homeostasis and cancer. Molecular pathways convergent between these contexts are highlighted while pathways enabling plasticity are distinguished from those that specify the phenotype of already plastic cells. Key unresolved questions in the field are discussed along with emerging technologies that may be used to help answer them.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91d0b45e5336c08fda7f8b88d683482dae5405ff" target='_blank'>
              The molecular determinants of phenotypic plasticity in homeostasis and neoplasia
              </a>
            </td>
          <td>
            Bradley Balk, David W. Goodrich
          </td>
          <td>2024-12-13</td>
          <td>Cancer Heterogeneity and Plasticity</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1054dfecee5e5157b77728bea8bb0a9b8823603" target='_blank'>
              Single-cell multiomics analysis reveals dynamic clonal evolution and targetable phenotypes in acute myeloid leukemia with complex karyotype
              </a>
            </td>
          <td>
            Aino-Maija Leppä, Karen Grimes, Hyobin Jeong, Frank Y Huang, Alvaro Andrades, A. Waclawiczek, T. Boch, Anna Jauch, S. Renders, Patrick Stelmach, Carsten Müller-Tidow, Darja Karpova, Markus Sohn, F. Grünschläger, Patrick Hasenfeld, Eva Benito Garagorri, Vera Thiel, A. Dolnik, Bernardo Rodriguez-Martin, L. Bullinger, Krzysztof Mrózek, Ann-Kathrin Eisfeld, Alwin Krämer, A. Sanders, J. Korbel, A. Trumpp
          </td>
          <td>2024-11-25</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>97</td>
        </tr>

        <tr id="Nongenetic plasticity has emerged as a key driver of cancer drug resistance, yet its precise origins, nature, and consequences are not fully clarified. This review examines technological, computational, and conceptual developments in the nongenetic determinants of drug resistance. We begin by proposing refined definitions of cellular state, fate, and plasticity. We subsequently contextualize the findings from multimodal approaches to investigate plasticity, highlighting how new single-cell lineage-tracing methods provide opportunities for quantitatively capturing state transitions, their timescales and heritability, and how they contribute to resistance mechanisms. We also draw parallels with concepts from developmental biology and microbial persistence research. Next, we cover the role that computational approaches have played in revealing the otherwise latent patterns of heterogeneity that underlie plasticity from complex datasets. We conclude by emphasizing the need for standardized terminology in this rapidly evolving field and the path from bench to bedside.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e38fadd15ca85570a489d3058bda72d96573ce1d" target='_blank'>
              Navigating Nongenetic Plasticity in Cancer Drug Resistance
              </a>
            </td>
          <td>
            Hanxiao Sun, Nitu Kumari, Madeline E. Melzer, Vadim Backman, Yogesh Goyal
          </td>
          <td>2024-12-11</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Cancer cells within tumors exhibit a wide range of phenotypic states driven by non-genetic mechanisms in addition to extensively studied genetic alterations. Conversions among cancer cell states can result in intratumoral heterogeneity which contributes to metastasis and development of drug resistance. However, mechanisms underlying the initiation and/or maintenance of such phenotypic plasticity are poorly understood. In particular, the role of intercellular communications in phenotypic plasticity remains elusive. In this study, we employ a multiscale inference-based approach using single-cell RNA sequencing (scRNA-seq) data to explore how intercellular interactions influence phenotypic dynamics of cancer cells, particularly cancers undergoing epithelial-mesenchymal transition. Our inference approach reveals that signaling interactions between cancerous cells in small cell lung cancer (SCLC) result in seemingly contradictory behaviors—reinforcing the cellular phenotypes and maintaining population-level intratumoral heterogeneity. Additionally, we find a recurring signaling pattern across multiple types of cancer in which the mesenchymal-like subtypes utilize signals from other subtypes to reinforce its phenotype, further promoting the intratumoral heterogeneity. We use a mathematical model based on ordinary differential equations to show that inter-subtype communication accelerates the development of heterogeneous tumor populations. Our work highlights the critical role of intercellular signaling in sustaining intratumoral heterogeneity, and our approach of computational analysis of scRNA-seq data can infer inter- and intra-cellular signaling networks in a holistic manner. Significance Single cell-based inference approach reveals a key role of intercellular signaling in maintaining intratumoral heterogeneity. Cell-cell communications stabilize newly acquired cell states and diverse phenotypes of cell populations in multiple cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fa5cf6c7e9bc3e2c2f27d636cb203f597857db1" target='_blank'>
              Intercellular signaling stabilizes single-cell level phenotypic transitions and accelerates the reestablishment of equilibrium of heterogeneous cancer cell populations
              </a>
            </td>
          <td>
            Daniel Lopez, Darren R Tyson, Tian Hong
          </td>
          <td>2025-01-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer drug resistance represents one of the most significant challenges in oncology and manifests through multiple interconnected molecular and cellular mechanisms. Objective: To provide a comprehensive analysis of multilevel processes driving treatment resistance by integrating recent advances in understanding genetic, epigenetic, and microenvironmental factors. This is a systematic review of the recent literature focusing on the mechanisms of cancer drug resistance, including genomic studies, clinical trials, and experimental research. Key findings include the following: (1) Up to 63% of somatic mutations can be heterogeneous within individual tumors, contributing to resistance development; (2) cancer stem cells demonstrate enhanced DNA repair capacity and altered metabolic profiles; (3) the tumor microenvironment, including cancer-associated fibroblasts and immune cell populations, plays a crucial role in promoting resistance; and (4) selective pressure from radiotherapy drives the emergence of radioresistant phenotypes through multiple adaptive mechanisms. Understanding the complex interplay between various resistance mechanisms is essential for developing effective treatment strategies. Future therapeutic approaches should focus on combination strategies that target multiple resistance pathways simultaneously, guided by specific biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58a34f85a0c27106a786f617d3281a89988e8619" target='_blank'>
              Multilevel Mechanisms of Cancer Drug Resistance
              </a>
            </td>
          <td>
            Małgorzata Roszkowska
          </td>
          <td>2024-11-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Phenotypic plasticity is a hallmark of cancer and increasingly realized as a mechanism of resistance to androgen receptor (AR)-targeted therapy. Now that many prostate cancer (PCa) patients are treated upfront with AR-targeted agents, it's critical to identify actionable mechanisms that drive phenotypic plasticity, to prevent the emergence of resistance. We showed that loss of tristetraprolin (TTP, gene ZFP36) increased NF-κB activation, and was associated with higher rates of aggressive disease and early recurrence in primary PCa. We also examined the clinical and biological impact of ZFP36 loss with co-loss of PTEN, a known driver of PCa. Analysis of multiple independent primary PCa cohorts demonstrated that PTEN and ZFP36 co-loss was associated with increased recurrence risk. Engineering prostate-specific Zfp36 deletion in vivo, induced prostatic intraepithelial neoplasia, and, with Pten co-deletion, resulted in rapid progression to castration-resistant adenocarcinoma. Zfp36 loss altered the cell state driven by Pten loss, demonstrated by enrichment of EMT, inflammation, TNFα/NF-κB, IL6-JAK/STAT3 gene sets. Additionally, our work revealed that ZFP36 loss also induced enrichment of multiple gene sets involved in mononuclear cell migration, chemotaxis, and proliferation. Use of the NF-κB inhibitor, dimethylaminoparthenolide (DMAPT) induced marked therapeutic responses in tumors with PTEN and ZFP36 co-loss and reversed castration resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51a8debd5a52234a8e6e2eecf0d0e1cd5849b25a" target='_blank'>
              Low tristetraprolin expression activates phenotypic plasticity and primes transition to lethal prostate cancer in mice.
              </a>
            </td>
          <td>
            Katherine L. Morel, Beatriz German, Anis A Hamid, Jagpreet S Nanda, S. Linder, A. Bergman, Henk van der Poel, Ingrid Hofland, E. Bekers, S. Trostel, Deborah L. Burkhart, S. Wilkinson, Anson T. Ku, Minhyung Kim, Jina Kim, Duanduan Ma, Jasmine T. Plummer, Sungyong You, Xiaofeng A. Su, W. Zwart, A. Sowalsky, Christopher J. Sweeney, Leigh Ellis
          </td>
          <td>2024-11-19</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Aim: Resistance to hormonal and targeted therapies in breast cancer limits treatment efficacy. Epigenetic alterations, including changes mediated by DNA methyltransferases, play a key role in this process. Previously, we identified that resistance to tamoxifen and rapamycin is associated with the suppression of DNMT3A. This study aims to further explore the mechanisms underlying this suppression, with a focus on identifying NR6A1 as a novel regulatory factor. Methods: Acquisition of resistant breast cancer cell sublines, MTT-test, immunoblotting, transient transfection and reporter analysis, lentiviral infection, qRT-PCR, and analysis of methylation using bisulfite pyrosequencing. Results: Our findings indicate that the development of cross-resistance in breast cancer cells to hormonal and targeted therapies involves a shift in cell signaling to alternative AKT pathways, marked by a localized suppression of the NR6A1/DNMT3A axis and associated DNA methylation changes. We demonstrated the critical role of NR6A1 downregulation in resistance development. Additionally, we observed activation of Snail - a key regulator in the epithelial-mesenchymal transition - as a mediator of the effects of NR6A1 depletion, establishing a direct link between Snail expression and resistance formation. Conclusion: The coordinated suppression of NR6A1 and DNMT3A may contribute to sustaining the resistant phenotype in breast cancer cells. This pathway could serve as a predictive marker, helping guide the selection of optimal therapeutic strategies for breast cancer treatment in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d24584557816dae92ae33446c23b68005521f3e" target='_blank'>
              Breast cancer cell resistance to hormonal and targeted therapeutics is correlated with the inactivation of the NR6A1 axis
              </a>
            </td>
          <td>
            O. Andreeva, D. Sorokin, S. Vinokurova, P. Kopnin, N. V. Elkina, A. Katargin, R. S. Faskhutdinov, D. Salnikova, A. M. Scherbakov, M. Krasil'nikov
          </td>
          <td>2024-11-23</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae4bbd7ae49e10933acda8e20cbd7e3bb473c50d" target='_blank'>
              Endosomal pH is an evolutionarily conserved driver of phenotypic plasticity in colorectal cancer
              </a>
            </td>
          <td>
            Hari Prasad, H. Bv, A. R. Subbalakshmi, S. Mandal, M. Jolly, Sandhya S. Visweswariah
          </td>
          <td>2024-12-19</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Intra tumor heterogeneity complicates cancer therapy by providing tumors with the ability to alter their phenotypes and become more therapy resistant. Here, we tested the hypothesis that identifying and modulating expression of key state-specific transcription factors could be used as a strategy for driving cells to a more therapy-sensitive state. Recent single-cell studies have explored the inter and intra tumoral heterogeneity of osteosarcoma and identified gene pathways enriched in specific cell states. For example, metastatic tumors are characterized by an expression of genes in the TNF-α, PI3K, TGFß and mTOR pathways. We identified similar profiles in osteosarcoma patient-derived xenograft-derived cell lines and potential transcription factor drivers of these states. We then used perturb-seq to downregulate expression of key transcription factors and evaluated the effect of these modulations on single cell RNA profiles and drug responses. Knockdown of NFE2L3 or NR0B1 increased the proportion of cells sensitive to targeted therapy. This approach, which could potentially be applied to other cancers, could be used as a strategy to increase the response to targeted therapies by increasing the proportion of cells in a drug-sensitive state. Highlights Distinct transcriptomic states were identified in osteosarcoma cell lines using single-cell RNA sequencing. Lineage tracing identified states with differential sensitivity to therapy. Using perturb-seq, we identified transcription factors that drive cells towards a more sensitive state. The transcription factors NR0B1 and NFE2L3 were identified as targets capable of reprogramming cells to a sensitive state.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6bc16f1a1ca4653fbc0122d06703e74482e021a" target='_blank'>
              Targeting Osteosarcoma heterogeneity to improve therapeutic response
              </a>
            </td>
          <td>
            Brenda Melano, Truc Dinh, Bahar Zirak, Mehran Karimzadeh, Brian J Woo, Benedict Choi, Hani Goodarzi, E. A. Sweet-Cordero, San Francisco
          </td>
          <td>2025-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Significant advances in the development of new cancer therapies have given rise to multiple novel therapeutic options in chemotherapy, radiotherapy, immunotherapy, and targeted therapies. Although the development of resistance is often reported along with temporary disease remission, there is often tumor recurrence of an even more aggressive nature. Resistance to currently available anticancer drugs results in poor overall and disease-free survival rates for cancer patients. There are multiple mechanisms through which tumor cells develop resistance to therapeutic agents. To date, efforts to overcome resistance have only achieved limited success. Epitranscriptomics, especially related to m6A RNA modification dysregulation in cancer, is an emerging mechanism for cancer therapy resistance. Here, recent studies regarding the contributions of m6A modification and its regulatory proteins to the development of resistance to different cancer therapies are comprehensively reviewed. The promise and potential limitations of targeting these entities to overcome resistance to various anticancer therapies are also discussed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/477760476e17db056d04688fc6e1891cd8c161d2" target='_blank'>
              Epitranscriptomic RNA m6A Modification in Cancer Therapy Resistance: Challenges and Unrealized Opportunities.
              </a>
            </td>
          <td>
            M. Uddin, Zhishan Wang, Chengfeng Yang
          </td>
          <td>2024-12-11</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fa374ada53bb9d0014e3752d56d972702923921" target='_blank'>
              Genome-scale modeling identifies dynamic metabolic vulnerabilities during the epithelial to mesenchymal transition
              </a>
            </td>
          <td>
            Rupa Bhowmick, Scott Campit, Shiva Krishna Katkam, V. Keshamouni, Sriram Chandrasekaran
          </td>
          <td>2024-12-27</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Cancer cells exploit a mesenchymal-like transcriptional state (MLS) to survive drug treatments. Although the MLS is well characterized, few therapeutic vulnerabilities targeting this program have been identified. Here, we systematically identify the dependency network of mesenchymal-like cancers through an analysis of gene essentiality scores in ~800 cancer cell lines, nominating a poorly studied kinase, PKN2, as a top therapeutic target of the MLS. Co-essentiality relationships, biochemical experiments, and genomic analyses of patient tumors revealed that PKN2 promotes mesenchymal-like cancer growth through a PKN2-SAV1-TAZ signaling mechanism. Notably, pairing genetic PKN2 inhibition with clinically relevant targeted therapies against EGFR, KRAS, and BRAF oncogenes suppresses drug resistance by depleting mesenchymal-like drug-tolerant persister cells. These findings provide evidence that PKN2 is a core regulator of the Hippo tumor suppressor pathway and highlight the potential of PKN2 inhibition as a generalizable therapeutic strategy to overcome drug resistance driven by the MLS across cancer contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc877056e57a8e2b137d870c6f49bfae37eaa22e" target='_blank'>
              PKN2 is a dependency of the mesenchymal-like cancer cell state.
              </a>
            </td>
          <td>
            Shane T. Killarney, Gabriel Mesa, Rachel Washart, Benjamin Mayro, Kerry Dillon, S. Wardell, M. Newlin, Min Lu, Areej Abu Rmaileh, Nicky Liu, Donald P. McDonnell, A. Pendergast, Kris C Wood
          </td>
          <td>2024-11-19</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Ex vivo drug response profiling is emerging as a valuable tool for identifying drug resistance mechanisms and advancing precision medicine in hematological cancers. However, the functional impact of dysregulation of the epigenome and transcriptome in this context remains poorly understood. In this study, we combined ex vivo drug sensitivity profiling with transcriptomic and epigenomic analyses in diagnostic samples from 597 pediatric B-cell precursor acute lymphoblastic leukemia (BCP-ALL) patients. Ex vivo resistance to antimetabolites (e.g., cytarabine, thioguanine), glucocorticoids (e.g., dexamethasone, prednisolone), and doxorubicin was independently associated with reduced relapse-free survival (RFS, p < 0.05). Molecular profiling identified pre-treatment DNA methylation and gene expression patterns distinguishing resistant from sensitive cases, revealing key drug resistance signatures. These included aberrant expression of genes related to heme metabolism (e.g. ATPV06A) and KRAS signaling (e.g. ABCB1). Notably, we also observed atypical expression of genes usually restricted to T-cells and other immune cells (e.g., ITK) in resistant BCP-ALL cells. Our findings demonstrate that ex vivo drug response patterns are predictive of clinical outcomes and reflect intrinsic molecular states associated with drug tolerance. This integrative multi-omics approach highlights potential therapeutic targets and underscores the value of functional precision medicine in identifying treatment vulnerabilities in pediatric ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1b0bb7a98b1f284795f68f2172436eb0a2df655" target='_blank'>
              Ex Vivo Drug Responses and Molecular Profiles of 597 Pediatric Acute Lymphoblastic Leukemia Patients
              </a>
            </td>
          <td>
            Anna Pia Enblad, O. Krali, Henrik Gezelius, Anders Lundmark, Kristin Blom, Claes Andersson, J. Palle, B. Frost, S. Ryhänen, Trond Flægstad, Ó. Jónsson, Kjeld Schmiegelow, Arja Harila, Peter Nygren, Rolf Larsson, Gudmar Lönnerholm, J. Nordlund, Anna Pia
          </td>
          <td>2024-12-20</td>
          <td>None</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40aee1d94d2e0752a7ed0c107794f2527da8d5a1" target='_blank'>
              Metabolic reprogramming induced by PSMA4 overexpression facilitates bortezomib resistance in multiple myeloma.
              </a>
            </td>
          <td>
            Han Yu, Chengli Wu, Jie He, Yajun Zhang, Qiqi Cao, Hongyan Lan, Hongshan Li, Chengyang Xu, Chen Chen, Rong Li, Bo Zheng
          </td>
          <td>2025-01-04</td>
          <td>Annals of hematology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Colorectal cancer (CRC) arises through a combination of genetic and epigenetic alterations that affect key pathways involved in tumor growth and progression. This review examines the major molecular pathways driving CRC, including Chromosomal Instability (CIN), Microsatellite Instability (MSI), and the CpG Island Methylator Phenotype (CIMP). Key mutations in genes such as APC, KRAS, NRAS, BRAF, and TP53 activate signaling pathways like Wnt, EGFR, and PI3K/AKT, contributing to tumorigenesis and influencing responses to targeted therapies. Resistance mechanisms, including mutations that bypass drug action, remain challenging in CRC treatment. This review highlights the role of molecular profiling in guiding the use of targeted therapies such as tyrosine kinase inhibitors and immune checkpoint inhibitors. Novel combination treatments are also discussed as strategies to improve outcomes and overcome resistance. Understanding these molecular mechanisms is critical to advancing personalized treatment approaches in CRC and improving patient prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ceb2e831232c0aff4dac73fca5b5e905790ed261" target='_blank'>
              Redefining Therapeutic Approaches in Colorectal Cancer: Targeting Molecular Pathways and Overcoming Resistance
              </a>
            </td>
          <td>
            S. Duta-Ion, I. Juganaru, Iulian-Andrei Hotinceanu, Andra Dan, Livia-Mălina Burtavel, Madalin-Codrut Coman, I. Focsa, A. Zaruha, P. Codreanu, L. Bohiltea, V. Rădoi
          </td>
          <td>2024-11-21</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad9d32195a3dad4b1106ede5f2fdac86e111eb66" target='_blank'>
              Cellular and molecular aspects of drug resistance in cancers.
              </a>
            </td>
          <td>
            Rahaman Shaik, M. S. Malik, Sreevani Basavaraju, Jihan Qurban, Fatimah M M Al-Subhi, Sathvika Badampudi, Jagruthi Peddapaka, Azeeza Shaik, Ahmad Abd‐El‐Aziz, Z. Moussa, Saleh A. Ahmed
          </td>
          <td>2024-12-09</td>
          <td>Daru : journal of Faculty of Pharmacy, Tehran University of Medical Sciences</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/738f42b88303f919a1111f0bb3965a9352b45745" target='_blank'>
              Molecular mechanisms and therapeutic strategies in overcoming chemotherapy resistance in cancer
              </a>
            </td>
          <td>
            Yixiang Gu, Ruifeng Yang, Yang Zhang, Miaomiao Guo, Kyle Takehiro, Ming Zhan, Linhua Yang, Hui Wang
          </td>
          <td>2025-01-06</td>
          <td>Molecular Biomedicine</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The integration of small interfering RNA (siRNA) with traditional cancer therapies represents a promising frontier in oncology aimed at enhancing treatment effectiveness, reducing side effects, and overcoming drug resistance. This review highlights the potential of siRNA to selectively silence genes that are overexpressed or uniquely expressed in cancer cells, thereby disrupting critical pathways that support tumor growth and survival. Key target genes discussed include survivin, VEGF, EGFR, c-MET, HER2, MUC1, and Bcl-2, all of which play vital roles in tumor proliferation, angiogenesis, and resistance to therapies. Clinical trials investigating various siRNA candidates, such as EZN-3042 and ALN-VSP, indicate that these therapies are generally well-tolerated; however, significant challenges persist, including the effective delivery and stability of siRNA. Recent advancements in nanoparticle-based delivery systems have shown promise in addressing these issues. Future research will focus on optimizing siRNA delivery methods, personalizing therapies based on individual genetic profiles, and establishing clearer regulatory guidelines for approval. As the field evolves, siRNA-based combination therapies are poised to become an integral part of precision oncology, offering new therapeutic options and hope for patients with difficult-to-treat cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00e63d7c300a29f5bcdddcbfc0fecd6c316315ab" target='_blank'>
              Dual Approaches in Oncology: The Promise of siRNA and Chemotherapy Combinations in Cancer Therapies
              </a>
            </td>
          <td>
            Carolina Sousa, M. Videira
          </td>
          <td>2025-01-02</td>
          <td>Onco</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Abstract Carcinogenesis often involves significant alterations in the cancer genome, marked by large structural variants (SVs) and copy number variations (CNVs) that are difficult to capture with short-read sequencing. Traditionally, cytogenetic techniques are applied to detect such aberrations, but they are limited in resolution and do not cover features smaller than several hundred kilobases. Optical genome mapping (OGM) and nanopore sequencing [Oxford Nanopore Technologies (ONT)] bridge this resolution gap and offer enhanced performance for cytogenetic applications. Additionally, both methods can capture epigenetic information as they profile native, individual DNA molecules. We compared the effectiveness of the two methods in characterizing the structural, copy number and epigenetic landscape of a clear cell renal cell carcinoma tumor. Both methods provided comparable results for basic karyotyping and CNVs, but differed in their ability to detect SVs of different sizes and types. ONT outperformed OGM in detecting small SVs, while OGM excelled in detecting larger SVs, including translocations. Differences were also observed among various ONT SV callers. Additionally, both methods provided insights into the tumor’s methylome and hydroxymethylome. While ONT was superior in methylation calling, hydroxymethylation reports can be further optimized. Our findings underscore the importance of carefully selecting the most appropriate platform based on specific research questions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2a68c85c3d1dfc77fbfc1f8f772053fa84be16b" target='_blank'>
              Long-read structural and epigenetic profiling of a kidney tumor-matched sample with nanopore sequencing and optical genome mapping
              </a>
            </td>
          <td>
            Sapir Margalit, Zuzana Tulpová, Tahir Detinis Zur, Yael Michaeli, J. Deek, Gil Nifker, Rita Haldar, Y. Gnatek, Dorit Omer, Benjamin Dekel, H. Baris Feldman, Assaf Grunwald, Yuval Ebenstein
          </td>
          <td>2025-01-07</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Drug resistance is the chief cause of treatment failure for therapies targeting chronic and infectious diseases. Whether the emergence of resistance is accelerated by environmental exposure to low levels of therapeutics remains controversial. Here, we report a non-genetic mechanism of stress adaptation that promotes heritable resistance to the widely used antifungal drug fluconazole. In the human fungal pathogen Candida albicans, transient exposure to subtherapeutic fluconazole doses induces a protective response that we term para-resistance. Like conventional resistance mechanisms, para-resistance is heritable. However, it does not arise from genetic mutations and can revert spontaneously. Systematic analyses of para-resistant isolates suggest that its key regulators include the stress-activated MAP kinase Hog1, the histone deacetylase subunit Snt1, the chromatin regulator Rap1, and the Sko1 transcriptional factor. Notably, molecules that disrupt biomolecular condensation and prion propagation – crucial for the inheritance of protein assemblies – block the induction of para-resistance, whereas inhibiting histone deacetylases facilitates its induction. We find that para-resistance is common in clinical isolates and, remarkably, passage through the mammalian gut triggers its acquisition, compromising fluconazole’s therapeutic efficacy. Our work defines a pervasive, prion-like epigenetic mechanism of stress adaptation and highlights potential strategies to mitigate the rapid emergence of drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6a3a4969dd38cce29e6bd3dee1a41bff2be4085" target='_blank'>
              Stress-driven emergence of heritable non-genetic drug resistance
              </a>
            </td>
          <td>
            Jinglin L. Xie, Sifei Yin, Theodore S. Yang, Kiran Chandrasekher, Luke Hanson, Sang Hu Kim, Lucas Esqueda, Catherine A. Hogan, Niaz Banaei, June L. Round, Kyla S. Ost, Judith Berman, Dan Jarosz
          </td>
          <td>2024-12-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Epigenetic regulator genes play critical roles in controlling cell identity and are frequently disrupted in breast cancers, suggesting a key driver role in this disease and its associated phenotypes. However, specific epigenetic drivers (epidrivers) of mammary cell plasticity and their mechanistic contributions to this phenotype are poorly characterized. Methods To identify potential epidrivers of the emergence of mesenchymal breast cancer stem cell-like phenotypes in non-tumorigenic mammary cells, we employed a CRISPR/Cas9 loss-of-function screening strategy targeting epigenetic regulator genes. This approach was followed by an in-depth validation and characterization of epigenomic, transcriptomic, proteomic and phenotypic changes resulting from the disruption of the putative epidriver gene BAP1. Results Our investigation revealed that loss of the histone deubiquitinase BAP1 impacts cellular processes associated with breast cancer cell plasticity such as epithelial-to-mesenchymal transition (EMT) and actin cytoskeleton organization. In addition, we unveiled that BAP1 loss resulted in an overall less permissive chromatin and downregulated gene expression, impacting programs that control cellular glycosylation and leading to decreased glycan abundance and complexity. BAP1 rescue restored the expression of several deregulated genes in a catalytic activity-dependent manner, suggesting that BAP1-mediated cell identity and glycosylation regulation are largely dependent on its histone deubiquitinase activity. Conclusions Overall, our results point to BAP1 disruption as a driver of mammary cell plasticity and reveal a novel role of BAP1 as an epigenetic regulator of cellular glycosylation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/744135b0816e44d41619b9824fc988445e6f4363" target='_blank'>
              Disruption of the epigenetic regulator BAP1 drives chromatin remodeling leading to the emergence of cells with breast cancer stem cell properties and aberrant glycosylation
              </a>
            </td>
          <td>
            Mariana Gomes da Silva Araujo, Aurélie Sallé, Vincent Cahais, Claire Renard, C. Cuenin, Caroline Pires Poubel, Stéphane Keita, Thorsten Mosler, Christine Carreira, Gabrielle Goldman Levy, Lya Parres, Ekaterina Bourova-Flin, Sophie Rousseaux, Saadi Khochbin, A. Ghantous, Siniša Habazin, M. Pučić-Baković, Erika Cosset, G. Lauc, Z. Herceg, Rita Khoueiry
          </td>
          <td>2024-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e385b264839697ab3b154e181f427818a7fd2d8b" target='_blank'>
              Biological and genetic characterization of a newly established human primary multidrug-resistant distal cholangiocarcinoma cell line, CBC3T-6
              </a>
            </td>
          <td>
            Mingzhen Bai, Ruoshui Wang, Chongfei Huang, Ruyang Zhong, Ningzu Jiang, Wenkang Fu, Ningning Mi, Long Gao, Yuyao Jin, Haidong Ma, Jie Cao, Haiying Yu, Qiang Jing, Chao Zhang, Ping Yue, Yong Zhang, Yanyan Lin, Hengwei Zhang, Wenbo Meng
          </td>
          <td>2024-11-29</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Aim: Mutations in the mitochondrial (mt) genome contribute to metabolic dysfunction and their accumulation relates to disease progression and resistance development in cancer cells. This study explores the mutational status of the mt genome of cisplatin-resistant vs. -sensitive testicular germ cell tumor (TGCT) cells and explores its association with their respiration parameters, expression of respiratory genes, and preferences for metabolic pathways to reveal new markers of therapy resistance in TGCTs. Methods: Using Illumina sequencing with Twist Enrichment Panel, the mutations of mt genomes of sensitive 2102EP, H12.1, NTERA-2, T-cam and resistant 2102EP Cis, H12.1 ODM, 1411HP, 1777NRpmet, NTERA-2 Cis and T-cam Cis cell lines were identified. The mt respiration of the cells was assessed using high-resolution respirometry method (O2k-respirometer Oroboros) and the differential expression profiles of mt respiratory genes were determined using RT-qPCR. Associated preferences for metabolic pathways were compared using Glycolysis/OXPHOS assay. Results: In resistant TGCT cells, new mutations in mt genes MT-ND1-6, MT-RNR, MT-CO1-3, MT-ATP6, and MT-CYB were recognized. The respiratory rates of the 1777NRpmet cell line were the highest, while those of the 1411HP line the lowest; rates of the control and all other TGCT cell lines fell between these two lines. The statistically significant differences in gene expression of the respiratory genes were recorded only in NTERA-2 Cis and T-cam Cis cell lines. Sensitive cell lines NTERA-2 and 2102EP preferred oxidative phosphorylation (OXPHOS), while glycolysis was typical for resistant NTERA-2 Cis, 2102EP Cis and 1411HP cell lines. Metastatic 1777NRpmet cells seem to utilize both. An isogenic pair of cell lines H12.1 and H12.1ODM showed the opposite dependence, sensitive H12.1 preferring glycolysis, while resistant H12.1ODM OXPHOS. Conclusion: In summary, our study identified new mutations in mt genes of resistant TGCT cell lines that are associated with different mt respiration parameters, gene expression patterns and preferences for metabolic pathways, providing potential novel molecular biomarkers that distinguish the resistant TGCT phenotype or specify its histological classification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28ec03801d589beba72991767df646f63f72e7a8" target='_blank'>
              Mitochondrial genome variability and metabolic alterations reveal new biomarkers of resistance in testicular germ cell tumors
              </a>
            </td>
          <td>
            Pavlina Kabelikova, Danica Ivovic, Z. Sumbalová, M. Karhánek, Lucia Tatayova, Martina Skopkova, Michal Cagalinec, V. Bruderová, J. Roska, Dana Jurkovicova
          </td>
          <td>2024-12-18</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c2f707eacba8ac9d3c84de9541b146c76849cd9" target='_blank'>
              Exosome-delivered NR2F1-AS1 and NR2F1 drive phenotypic transition from dormancy to proliferation in treatment-resistant prostate cancer via stabilizing hormonal receptors
              </a>
            </td>
          <td>
            Wenbin Chen, Yiyou Mao, YiYuan Zhan, Wenfeng Li, Jun Wu, Xiangming Mao, Bin Xu, Fangpeng Shu
          </td>
          <td>2024-12-18</td>
          <td>Journal of Nanobiotechnology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/029b373b085751bc9a273803a2aea04199d3ea75" target='_blank'>
              Metabolic reprogramming and therapeutic resistance in primary and metastatic breast cancer
              </a>
            </td>
          <td>
            Shan Liu, Xingda Zhang, Wenzheng Wang, Xue Li, Xue Sun, Yuqian Zhao, Qi Wang, Yingpu Li, Fangjie Hu, He Ren
          </td>
          <td>2024-11-21</td>
          <td>Molecular Cancer</td>
          <td>2</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4ae77252749b1c986ef7ff79cc410ca86b264c4" target='_blank'>
              Comparison of baseline global gene expression profiles of prostate cancer cell lines LNCaP and DU145
              </a>
            </td>
          <td>
            Khalid Ahmed, Zhannur Omarova, Alisalman Sheikh, Gulzhan Abuova, K. Ghias, Syed Hani Abidi
          </td>
          <td>2024-12-31</td>
          <td>BMC Research Notes</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81cf24082e18b8a822679c8c6c602531a443fe70" target='_blank'>
              Nucleotide metabolism-associated drug resistance gene NDUFA4L2 promotes colon cancer progression and 5-FU resistance
              </a>
            </td>
          <td>
            Hongxin He, Shiyao Zheng, Shangkun Jin, Weijie Huang, Enhao Wei, Shen Guan, Chunkang Yang
          </td>
          <td>2025-01-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Abstract In a comprehensive study to decipher the multi-layered response to the chemotherapeutic agent temozolomide (TMZ), we analyzed 427 genomes and determined mutational patterns in a collection of ∼40 isogenic DNA repair-deficient human TK6 lymphoblast cell lines. We first demonstrate that the spontaneous mutational background is very similar to the aging-associated mutational signature SBS40 and mainly caused by polymerase zeta-mediated translesion synthesis (TLS). MSH2-/- mismatch repair (MMR) knockout in conjunction with additional repair deficiencies uncovers cryptic mutational patterns. We next report how distinct mutational signatures are induced by TMZ upon sequential inactivation of DNA repair pathways, mirroring the acquisition of chemotherapy resistance by glioblastomas. The most toxic adduct induced by TMZ, O6-meG, is directly repaired by the O6-methylguanine-DNA methyltransferase (MGMT). In MGMT-/- cells, MMR leads to cell death and limits mutagenesis. MMR deficiency results in TMZ resistance, allowing the accumulation of ∼105 C > T substitutions corresponding to signature SBS11. Under these conditions, N3-methyladenine (3-meA), processed by base excision repair (BER), limits cell survival. Without BER, 3-meA is read through via error-prone TLS, causing T > A substitutions but not affecting survival. Blocking BER after abasic site formation results in large deletions and TMZ hypersensitization. Our findings reveal potential vulnerabilities of TMZ-resistant tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdcf8efd6ed1b3006f0325ffb555aaabb03b08a5" target='_blank'>
              Comprehensive whole-genome sequencing reveals origins of mutational signatures associated with aging, mismatch repair deficiency and temozolomide chemotherapy
              </a>
            </td>
          <td>
            Taejoo Hwang, Lukasz Karol Sitko, Ratih Khoirunnisa, Fernanda Navarro-Aguad, David M Samuel, Hajoong Park, Banyoon Cheon, Luthfiyyah Mutsnaini, Jaewoong Lee, Burçak Otlu, Shunichi Takeda, Semin Lee, Dmitri Ivanov, Anton Gartner
          </td>
          <td>2024-12-05</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 Background: Prostate cancer is a prevalent malignancy in men, with treatment resistance posing a significant challenge to successful cure. Acquired Resistance to Therapy (ART) arises from two main components: (1) Tumor Microenvironment (TME) and (2) mutations or epigenetic changes at gene level within the tumor cells themselves. While most cancer therapies target the intrinsic biology of cancer cells, growing evidence indicates that these treatments can also harm the stromal cells within TME, potentially driving ART. The biology of ART may vary based on the types of treatments, cell types, and organ locations. Thus, this study aims to dissect ART components driven by TME-mediated cellular processes in a comprehensive manner. Hypothesis: We propose that cancer therapies induce stress responses in non-cancerous cells within the TME. These stress responses involve the secretion of cytokines, mediators, and growth factors, which we term Stress Response Secretory Programs (SRSPs). We further hypothesize that targeting SRSPs could overcome ART that helps to improve treatment specificity and efficacy. Materials and Methods: We employed comprehensive single-cell profiling of common metastatic sites included liver, prostate, lung, bone, lymph node and spleen in preclinical models to identify secreted proteins and dysregulated pathways caused by genotoxic therapies. In vitro, we evaluated treatment resistance in murine prostate epithelial cell lines (Myc-CAP) by exposing them to conditioned media (CM) from irradiated murine fibroblasts (F4M2), using cell growth assays. Results: Single-cell RNA sequencing of organ samples revealed significant expressions of SRSPs and increased inflammatory responses in liver and prostate fibroblasts as well as other stromal cells treated with carboplatin. In vitro, CM from irradiated F4M2 cells enhanced cell proliferation and survival of Myc-CAP cells in response to docetaxel treatment. Conclusion: Our findings highlight the crucial role of SRSPs in mediating ART through interactions within the TME, which alter tumor cell phenotypes. Targeting these SRSPs holds potential for overcoming treatment resistance and improving therapeutic outcomes, underscoring the need to consider TME complexity in cancer treatment strategies.
 Citation Format: Tony Lok Heng Chu, Armand Bankhead, Ilsa Coleman, Sander Frank, Peter S. Nelson, Tarana Arman. Identifying and targeting prostate cancer therapy resistance mechanisms driven by the tumor microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr C033.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c940709273c954da8cb4bd8042d98f46649ce8ee" target='_blank'>
              Abstract C033: Identifying and targeting prostate cancer therapy resistance mechanisms driven by the tumor microenvironment
              </a>
            </td>
          <td>
            Tony Lok Heng Chu, Armand Bankhead, Ilsa M Coleman, Sander Frank, Peter S. Nelson, Tarana Arman
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Background Osteosarcoma is a malignant tumor originating from mesenchymal bone tissue, characterized by high malignancy and poor prognosis. Despite progress in comprehensive treatment approaches, the five-year survival rate remains largely unchanged, highlighting the need to clarify its underlying mechanisms and discover new therapeutic targets. Methods This study utilized RNA sequencing data from multiple public databases, encompassing osteosarcoma samples and healthy controls, along with single-cell RNA sequencing data. Various methods were utilized, such as differential expression analysis of genes, analysis of metabolic pathways, and weighted gene co-expression network analysis (WGCNA), to pinpoint crucial genes. Using this list of genes, we developed and validated a prognostic model that incorporated risk signatures, and we evaluated the effectiveness of the model through survival analysis, immune cell infiltration examination, and drug sensitivity evaluation. Results We analyzed gene expression and metabolic pathways in nine samples using single-cell sequencing data. Initially, we performed quality control and clustering, identifying 21 statistically significant cell subpopulations. Metabolic analyses of these subpopulations revealed heterogeneous activation of metabolic pathways. Focusing on the osteoblastic cell subpopulation, we further subdivided it into six groups and examined their gene expression and differentiation capabilities. Differential expression and enrichment analyses indicated that tumor tissues were enriched in cytoskeletal and structural pathways. Through WGCNA, we identified core genes negatively correlated with four highly activated metabolic pathways. Using osteosarcoma patient data, we developed a risk signature model that demonstrated robust prognostic predictions across three independent cohorts. Ultimately, we performed a thorough examination of the model, which encompassed clinical and pathological characteristics, enrichment analysis, pathways associated with cancer markers, and scores of immune infiltration, highlighting notable and complex disparities between high-risk and low-risk populations. Conclusion This research clarifies the molecular mechanisms and metabolic features associated with osteosarcoma and how they relate to patient outcomes, offering novel perspectives and approaches for targeted therapy and prognostic assessment in osteosarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3c754f2161a5b2138f06ad7cb757b73f892978d" target='_blank'>
              Exploring the heterogeneity of osteosarcoma cell characteristics and metabolic states and their association with clinical prognosis
              </a>
            </td>
          <td>
            Sen Qin, YaoFeng Hu, RuCui Deng, Zhe Wang
          </td>
          <td>2024-12-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Simple Summary Chemotherapy-resistant and dormant cancer cells present significant hurdles in lung cancer treatment. Resistance to therapies such as chemotherapy and tyrosine kinase inhibitors is driven by factors including the tumor microenvironment and genetic mutations. Dormant cancer cells within minimal residual disease can evade detection, which contributes to cancer recurrence. Emerging strategies such as combination therapies, antibody–drug conjugates, and targeted protein degradation offer promising avenues to address these challenges. A deeper understanding of resistance mechanisms is crucial for the development of more effective, personalized treatment approaches. The lung cancer organoid model also holds immense potential to advance research and clinical applications in this field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba53f5fca0859828ac3bcb03b5bf97bbb1252798" target='_blank'>
              Navigating the Complexity of Resistance in Lung Cancer Therapy: Mechanisms, Organoid Models, and Strategies for Overcoming Treatment Failure
              </a>
            </td>
          <td>
            D. Kang, Jisoo Lee, Subin Im, C. Chung
          </td>
          <td>2024-11-28</td>
          <td>Cancers</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="This study aimed to generate Car- and Pac-resistant cell lines from the human lung adenocarcinoma H1792 cell line, designated as H1792/Car and H1792/Pac, and perform transcriptome sequencing to identify potential targets. Common differentially expressed genes (Co-DEGs) in both resistant cell lines were identified, followed by hub gene identification. Online validation was conducted through GEPIA and Kaplan-Meier Plotter platforms, with experimental validation performed using real-time quantitative PCR (RT-qPCR). After six months, the H1792/Car and H1792/Pac cell lines exhibited a 10.7-fold and 5.6-fold increase in resistance to Car and Pac, respectively. Flow cytometry analysis demonstrated that both resistant cell lines were resistant to cell cycle arrest and apoptosis induced by Car or Pac. Transcriptomic sequencing identified 123 Co-DEGs, including 72 upregulated and 51 downregulated genes, consistently expressed in both H1792/Car and H1792/Pac cell lines. Among these, 13 hub genes were identified, with colony-stimulating factor 3 (CSF3) uniquely associated with post-progression survival (PPS) in adenocarcinoma patients undergoing chemotherapy. Notably, CSF3 expression was significantly elevated in both H1792/Car and H1792/Pac compared to parental cells. These findings underscore the value of drug-resistant models in uncovering critical biomarkers. CSF3 emerges as a promising guiding marker or potential molecular target for optimizing Car- and Pac-based doublet regimens.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/319b42d1b63fc345fe5d5188434196b326a0a009" target='_blank'>
              Transcriptomic Profiling of Carboplatin- and Paclitaxel-Resistant Lung Adenocarcinoma Cells Reveals CSF3 as a Potential Biomarker for the Carboplatin Plus Paclitaxel Doublet Regimens.
              </a>
            </td>
          <td>
            Pritsana Raungrut, Suchanan Tanyapattrapong, Thipphanet Masjon, Saowanee Maungchanburi, P. Thongsuksai
          </td>
          <td>2024-12-11</td>
          <td>Current issues in molecular biology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Breast cancer is the most prevalent cancer and the leading cause of cancer-related mortality among women. Recent breakthroughs in breast cancer therapeutics have significantly enhanced outcomes for hormone receptor-positive and HER2-negative subtypes. However, the emergence of drug resistance, particularly in triple-negative breast cancer, presents a formidable challenge. The intricate interplay of genetic and metabolic diversity within breast cancer cells is pivotal to its development. By reprogramming metabolic pathways, cancer cells can adapt and thrive, meeting the demands of survival, growth, and invasion. These metabolic shifts also play a key role in the development of resistance to conventional therapies. This review explores the genetic and metabolic complexities of breast cancer, emphasizing the diverse subtypes and their unique profiles. We examine how genetic variations and metabolic alterations contribute to breast cancer development and progression, influencing both treatment efficacy and resistance. By integrating insights into the genetic background and metabolic reprogramming of breast cancer subtypes, this review aims to highlight the genetic variations and metabolic alterations that contribute to the pathogenesis of breast cancer, with a vision of advancing more precise and effective targeted therapies as well as discovering of novel diagnostic and prognostic markers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd8d2c71ef9a50520221d0171aab11b20229faa4" target='_blank'>
              Genetic heterogeneity and metabolic reprogramming in breast cancer
              </a>
            </td>
          <td>
            Yarkın Dolaş, Ayşe Buruş, B. Çeltikçi
          </td>
          <td>2024-12-29</td>
          <td>Acta Medica</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cell-cell fusion has been implicated in various physiological and pathological processes, including cancer progression. This study investigated the role of cell-cell fusion in non-small cell lung cancer (NSCLC), focusing on its contribution to chemoresistance and tumor evolution. By co-culturing drug-sensitive and drug-resistant NSCLC cell lines, we observed spontaneous cell-cell fusion events, particularly under gefitinib selection. These fused cells exhibited enhanced fitness and a higher degree of chemoresistance compared to parental lines across a panel of 12 chemotherapeutic agents. Further analysis, including fluorescence imaging and cell cycle analysis, confirmed nuclear fusion and increased DNA content in the fused cells. Bulk RNA sequencing revealed genomic heterogeneity in fused cells, including enrichment of gene sets associated with cell cycle progression and epithelial-mesenchymal transition, hallmarks of cancer malignancy. These findings demonstrate that cell-cell fusion contributes significantly to therapeutic resistance and the promotion of aggressive phenotypes in NSCLC, highlighting its potential as a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f78ae5e6d6b40de2fc2ee9f6021e6a3e7d5997a6" target='_blank'>
              Cell-Cell Fusion in NSCLC Confers a Fitness Benefit Under Drug Selection
              </a>
            </td>
          <td>
            Paulameena V. Shultes, D. Tadele, A. Durmaz, D. Weaver, R. Barker-Clarke, Mina N. Dinh, Siqing Liu, Endalkachew A. Alemu, Simon Rayner, Jacob G. Scott
          </td>
          <td>2024-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="While the key aspects of genetic evolution and their clinical implications in clear cell renal-cell carcinoma (ccRCC) are well-documented, how genetic features co-evolve with the phenotype and tumor microenvironment (TME) remains elusive. Here, through joint genomic-transcriptomic analysis of 243 samples from 79 patients recruited to the TRACERx Renal study, we identify pervasive non-genetic intratumor heterogeneity, with over 40% not attributable to genetic alterations. By integrating tumor transcriptomes and phylogenetic structures, we observe convergent evolution to specific phenotypic traits, including cell proliferation, metabolic reprogramming and overexpression of putative cGAS-STING repressors amid high aneuploidy. We also uncover a co-evolution between the tumor and the T cell repertoire, as well as a longitudinal shift in the TME from an anti-tumor to an immunosuppressive state, linked to the acquisition of recurrently late ccRCC drivers 9p loss and SETD2 mutations. Our study reveals clinically-relevant and hitherto underappreciated non-genetic evolution patterns in ccRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c84f2fcb6ee92e2f103bced79a769e625785b29b" target='_blank'>
              Tracking non-genetic evolution from primary to metastatic ccRCC: TRACERx Renal.
              </a>
            </td>
          <td>
            Ángel Fernández-Sanromán, A. Fendler, B. J. Tan, A. Cattin, C. Spencer, Rachael Thompson, L. Au, Irene Lobon, H. Pallikonda, Alice Martin, F. Byrne, Antonia Franz, Anna Mikolajczak, Haseeb Rahman, Z. Tippu, S. T. Shepherd, Hugang Feng, Daqi Deng, Andrew Rowan, Lisa M. Pickering, A. J. Furness, Kate Young, David Nicol, S. Rudman, Tim O’Brien, K. Edmonds, Ashish Chandra, S. Hazell, Kevin Litchfield, G. Kassiotis, J. Larkin, S. Turajlic
          </td>
          <td>2025-01-08</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Cancer continues to pose a formidable challenge in global health due to its incidence and increasing resistance to conventional therapies. A key factor driving this resistance is tumor hypoxia, characterized by reduced oxygen levels within cancer cells. This hypoxic environment triggers a variety of adaptive mechanisms, significantly compromising the efficacy of cancer treatments. Notably, hypoxia promotes metastasis and reshapes the tumor microenvironment (TME), thereby aggravating treatment resistance. Central to this process are hypoxia-inducible factors (HIFs), which mediate cellular adaptations such as metabolic shifts and enhanced survival pathways. These adaptations render therapies like chemotherapy, radiotherapy, and photodynamic therapy (PDT) less effective. Additionally, hypoxia-induced vascular irregularities further impede drug delivery, amplifying the therapeutic challenge. This review provides a comprehensive examination of the roles of hypoxia in cancer, its contributions to drug resistance, and its interplay with apoptosis and autophagy. By evaluating novel mechanistic and translational approaches to target hypoxia, this study highlights the potential to improve therapeutic outcomes and offers insights into overcoming treatment resistance in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25892c7388537dae1aafc1066de6a56e2191122e" target='_blank'>
              An Enhanced Scrutiny of Mechanistic and Translational Approaches to Extinguish Cancer Hypoxia.
              </a>
            </td>
          <td>
            Arun Radhakrishnan, Nikhitha K. Shanmukhan, Linda Christabel. S
          </td>
          <td>2024-12-26</td>
          <td>Pharmaceutical nanotechnology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Developing resistance to androgen receptor (AR) signaling inhibitors is a significant challenge in the treatment of castration-resistant prostate cancer. Prolonged use of inhibitors like enzalutamide can cause prostate cancer cells to undergo lineage reprogramming, transitioning to neuroendocrine subtypes that no longer rely on AR signaling. These neuroendocrine subtypes are among the most aggressive forms of prostate cancer. During this process of lineage plasticity, cancer cells experience extensive transcriptional rewiring and acquire stem-like properties characterized by increased stemness. Research has shown that prostate cancer cells gain these stem-like traits through the expression of stem cell-associated proteins such as NANOG, particularly under stable and accumulating conditions. The post-translational modification of NANOG at specific sites is critical for maintaining its stability, which in turn enhances the tumorigenic potential of the cells. This review discusses the mechanisms by which NANOG phosphorylation promotes stemness and lineage plasticity in prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d23842d78f2e67e6ef1863e8dd082e1ac56f105a" target='_blank'>
              Molecular mechanisms driving lineage plasticity in prostate cancer: NANOG and beyond
              </a>
            </td>
          <td>
            Hamed Maasoumyhaghighi, Mansoureh Nouri, Jinghui Liu, Xiaoqi Liu
          </td>
          <td>2024-12-27</td>
          <td>Cancer Heterogeneity and Plasticity</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01ace1a7b748e6be8dfa6d49964b342cccfe62af" target='_blank'>
              PCK1 as a target for cancer therapy: from metabolic reprogramming to immune microenvironment remodeling
              </a>
            </td>
          <td>
            Na Liu, Xiao-Ren Zhu, Chang-Ying Wu, Yuan-Yuan Liu, Minbin Chen, Jin-Hua Gu
          </td>
          <td>2024-11-22</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Medulloblastoma (MB) is the most prevalent malignant brain tumor in children, exhibiting clinical and genomic heterogeneity. Of the four major subgroups, Group 3 tumors (MYC-MB), display high levels of MYC and metastasis rates. Despite treatment with surgery, radiation, and chemotherapy, patients with Group 3 MB are more likely to develop aggressive recurrent tumors with poor survival. To examine resistance mechanisms, single nuclei multiome analysis of matched primary and recurrent tumors was performed in this study. A persistent progenitor population supporting resistance to therapy was identified. Additionally, distinct chromatin landscapes are linked to altered transcription and correspond to metabolic reprogramming. In vivo modeling of radiation resistance resolves similar chromatin-based metabolic reprogramming focused on wild-type isocitrate dehydrogenase (IDH1) activity. IDH1 inhibition reverses resistance-mediated chromatin changes and enables radiation re-sensitization. Ultimately, these findings demonstrate the efficacy of single-cell multiome analysis in elucidating resistance mechanisms and identifying novel target pathways for MYC-driven medulloblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/beeb800863a129b3f026b22eed6f7669390988ee" target='_blank'>
              Single-cell multi-omics analysis identifies metabolism-linked epigenetic reprogramming as a driver of therapy-resistant medulloblastoma
              </a>
            </td>
          <td>
            Rajeev Vibhakar, Bethany L. Veo, Dong Wang, J. DeSisto, Angela M Pierce
          </td>
          <td>2024-12-13</td>
          <td>Research Square</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/660334082207693e7b39f13fca8e1d97d42e1441" target='_blank'>
              Comprehensive review of drug resistance in mammalian cancer stem cells: implications for cancer therapy
              </a>
            </td>
          <td>
            B. Mengistu, Tirunesh Tsegaw, Yitayew Demessie, Kalkidan Getnet, A. Bitew, M. Kinde, Asnakew Mulaw Beirhun, Atsede solomon Mebratu, Yesuneh Tefera Mekasha, Melaku Getahun Feleke, Melkie Dagnaw Fenta
          </td>
          <td>2024-12-18</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31f970e4e2fc5fa77ad8483f91874195b589fe0b" target='_blank'>
              Single-cell RNA-seq reveals diverse molecular signatures associated with Methotrexate resistance in primary central nervous system lymphoma cells.
              </a>
            </td>
          <td>
            Rosuke Osako, A. Hayano, Atsushi Kawaguchi, R. Yamanaka
          </td>
          <td>2024-12-05</td>
          <td>Journal of neuro-oncology</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="
 Pharmacological inhibitors of oncogenic signaling, such as drugs that target ALK kinase (ALKi) in ALK+ lung cancers, can induce strong and durable clinical responses. However, targeted therapies are not curative in advanced cancers and even strong responders eventually develop resistance. While therapy resistance is typically attributed to cell-intrinsic (epi)mutational characteristics of tumor cells, it can also result from interactions of tumor cells with the tumor microenvironment (TME). In contrast to the detailed elucidation of the molecular mediators of therapy resistance, our knowledge of the evolutionary dynamics underlying the resistance emergence, including the impact of TME, remains poorly defined. A key open question in understanding the evolutionary dynamics of therapy resistance is whether relapse results from an expansion of pre-existing therapy-resistant subpopulations or from a bona fide gradual evolutionary process. To elucidate this question, we integrated experimental mouse studies with mathematical modeling, interrogating therapeutic responses of therapy-naïve experimental xenograft ALK+ tumors, spiked-in with differentially labeled resistant cells. We found that ALKi treatment induced a rapid expansion of resistant cells, which drastically reduced the magnitude and duration of remissions. Surprisingly, our in silico analyses pointed to the existence of a strong positive ecological interaction between therapy-resistant and therapy-sensitive competitors. Using a combination of spatial analyses, experimental studies, and mathematical modeling, we found that this interaction was mediated by peristromal niches that protected therapy-sensitive tumor cells. Specifically, by limiting tumor regression, the therapy- induced expansion of resistant cells limited the loss of protective peristromal niches, while the subsequent resumption of tumor growth created new peristromal niches capable of supporting the survival and proliferation of therapy-sensitive cells. While this niche-mediated interaction had only a marginal impact on the transition from remission to relapse, enhanced survival of therapy-sensitive cells potentiated their ability to adapt to ALKi, leading to a higher diversity of resistance phenotypes. In summary, our study has revealed a new type of indirect, niche-mediated ecological interaction between therapy-resistant and therapy-sensitive cells. The rapid expansion of rare pre-existent resistant cells and fast transition to relapse challenge a common assumption of the pre-existence of therapy resistance. Finally, our results highlight the essentiality of TME considerations in understanding the evolutionary dynamic underlying the development of therapy resistance.
 Citation Format: Mark Robertson-Tessi, Bina Desai, Tatiana Miti, Pragya Kumar, Sagnik Yarlagadda, Rishi Shah, Robert Vander Velde, Daria Miroshnychenko, David Basanta, Alexander Anderson, Andriy Marusyk. Therapy-protective peristromal niches mediate positive ecological interaction between therapy-sensitive and therapy-resistant cells, altering the evolutionary dynamics of acquired targeted therapy resistance in lung cancers [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr B008.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/832b711609a909524835a20edd34f9feaf6c94a2" target='_blank'>
              Abstract B008: Therapy-protective peristromal niches mediate positive ecological interaction between therapy-sensitive and therapy-resistant cells, altering the evolutionary dynamics of acquired targeted therapy resistance in lung cancers
              </a>
            </td>
          <td>
            M. Robertson-Tessi, Bina Desai, Tatiana Miti, Pragya Kumar, Sagnik Yarlagadda, Rishi Shah, R. V. Velde, Daria Miroshnychenko, D. Basanta, Alexander R. A. Anderson, A. Marusyk
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Mismatch repair deficiency leads to high mutation rates and microsatellite instability (MSI-H), associated with immune infiltration and responsiveness to immunotherapies. In early stages, MSI-H tumors generally have a better prognosis and lower metastatic potential than microsatellite-stable (MSS) tumors, especially in colorectal cancer. However, in advanced stages, MSI-H tumors lose this survival advantage for reasons that remain unclear. We developed a syngeneic mouse model of MSI cancer by knocking out the MMR gene Msh2 in the metastatic 4T1 breast cancer cell line. This model mirrored genomic features of MSI-H cancers and showed reduction in metastatic incidence compared to their MSS counterparts. In MSI-H tumors, we observed an enrichment of immune gene-signatures that negatively correlated with metastasis incidence. Importantly, a hybrid epithelial-mesenchymal signature, related to aggressiveness was detected only in metastatic MSI-H tumors which may explain the worse outcomes after recurrence of MSI tumors compared to MSS. Interestingly, we identified immature myeloid cells at primary and metastatic sites in MSI-H tumor-bearing mice, suggesting that MMR deficiency elicits specific immune responses beyond T-cell activation. Significance A novel syngeneic mouse model of MSI cancer demonstrates that the immune system regulates MSI cancer cell dissemination, offering an important tool to model advanced stages of human MSI-driven disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb4d06352a13d99fbaae03568dc6632e6da89c31" target='_blank'>
              Effect of MisMatch Repair Deficiency on metastasis occurrence modelized in a syngeneic mouse model
              </a>
            </td>
          <td>
            Pierre Laplante, Reginaldo Rosa, L. Nebot-Bral, Jordane Goulas, Sergey I. Nikolaev, Aymeric Silvin, P. Kannouche
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="In cancer research, the term epigenetics was used in the 1970s in its modern sense encompassing non-genetic events modifying the chromatin state, mainly to oppose the emerging oncogene paradigm. However, starting from the establishment of this prominent concept, the importance of these epigenetic phenomena in cancer rarely led to questioning the causal role of genetic alterations. Only in the last 10 years, the accumulation of problematic data, better experimental technologies, and some ambitious models pushed the idea that epigenetics could be at least as important as genetics in early oncogenesis. Until this year, a direct demonstration of epigenetic oncogenesis was still lacking. Now, Parreno, Cavalli and colleagues, using a refined experimental model in the fruit fly Drosophila melanogaster, enforced the initiation of tumors solely by imposing a transient loss of Polycomb repression, leading to a purely epigenetic oncogenesis phenomenon. Despite a few caveats that we discuss, this pioneering work represents a major breakpoint in cancer research. We are led to consider the theoretical and conceptual implications on oncogenesis and to search for links between this artificial experimental model and naturally occurring processes, while revisiting cancer theories that were previously proposed as alternatives to the oncogene-centered paradigm.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f2f2c2f1a7eb766c5cfd45527bf643538142c9e" target='_blank'>
              Evidence of Epigenetic Oncogenesis: A Turning Point in Cancer Research.
              </a>
            </td>
          <td>
            Jean-Pascal Capp, Benoît Aliaga, Vera Pancaldi
          </td>
          <td>2024-11-21</td>
          <td>BioEssays : news and reviews in molecular, cellular and developmental biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Despite the ability to induce strong clinical responses, therapies that target mutational drivers of oncogenic signaling are not curative in advanced cancers, as a subset of cancer cells is capable of surviving therapy and evolving resistance. While therapy resistance is commonly viewed as a cell-intrinsic phenomenon, multiple factors produced by tumors can also confer strong therapy resistance. In contrast to the detailed elucidation of molecular mediators of cell-intrinsic and stroma-mediated (SM) resistance, the relative importance of cell-intrinsic and SM resistance to in vivo therapeutic responses remains poorly defined. To address this gap of knowledge, we used a well-characterized experimental model of ALK+ lung cancer H3122 cell line that exemplifies a duality observed across common models of targetable cancers. Under standard stroma-free in vitro cultures, H3122 cells can survive under therapeutically relevant concentrations of ALK inhibitors (ALKi), eventually acquiring cell-intrinsic therapy resistance. At the same time, co-culture with stromal fibroblasts drastically reduces the sensitivity of H3122 cells to ALKi, indicating the relevance of SM resistance. To understand the impact of SM resistance on in vivo responses, we sought to identify the mechanism(s) responsible for the therapy-protective effects of stromal fibroblasts, subsequently interrogating the impact of the disruption of this mechanism on in vivo therapeutic responses. Our in vitro studies with a large panel of primary stromal fibroblast isolates identified the HGF-cMET axis as the major mechanism responsible for ALKi desensitization. Surprisingly, xenograft validation studies demonstrated a weak effect of HGF-cMET modulation. Histological analyses of tumor tissues revealed that this relative weakness is attributable to a strong, spatially limited HGF-independent component of SM resistance. Mechanistic follow-up demonstrated that the HGF-independent component of SM resistance integrates the effect of multiple well-known juxtacrine and paracrine-acting mechanisms. Whereas the multifactorial nature of stroma-mediated resistance prevented a clear assessment of its relative contribution to in vivo therapy responses, our spatial analyses indicate that SM resistance dominates the ability of tumors to avoid therapeutic elimination. Surprisingly, we found that SM resistance was also a substantial contributor to tumor relapse, indicating that in vivo therapy resistance can integrate both cell-intrinsic and cell-extrinsic effects. This duality of therapy resistance and the multifactorial underpinning of both cell-intrinsic and stroma-mediated resistance present a challenge to therapeutic strategies focused on identifying and targeting specific resistance mechanisms. Our studies suggest that this limitation can be overcome by shifting the therapeutic focus towards shared orthogonal therapeutic sensitivities of tumor cells within residual disease.
 Citation Format: Bina Desai, Tatiana Miti, Sandhya Prabhakaran, Daria Miroshnychenko, Menkara Henry, Viktoriya Marusyk, Chandler Gatenbee, Marilyn Bui, Jacob Scott, Philipp M. Altrock, Eric Haura, Alexander R.A. Anderson, David Basanta, Andriy Marusyk. Spatially limited stroma-mediated resistance potentiates targeted therapy resistance through an integration of multiple juxtacrine and paracrine mechanisms [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr C035.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/943bc0d90e7f0f13d1f5d2038bd76c206372d768" target='_blank'>
              Abstract C035: Spatially limited stroma-mediated resistance potentiates targeted therapy resistance through an integration of multiple juxtacrine and paracrine mechanisms
              </a>
            </td>
          <td>
            Bina Desai, Tatiana Miti, Sandhya Prabhakaran, Daria Miroshnychenko, Menkara-Chinua N Henry, Viktoriya Marusyk, Chandler D. Gatenbee, Marilyn M Bui, Jacob G. Scott, Philipp M. Altrock, E. Haura, Alexander R. A. Anderson, D. Basanta, A. Marusyk
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Cell-cell fusion is a tightly controlled process in the human body known to be involved in fertilization, placental development, muscle growth, bone remodeling, and viral response. Fusion between cancer cells results first in a whole-genome doubled state, which may be followed by the generation of aneuploidies; these genomic alterations are known drivers of tumor evolution. The role of cell-cell fusion in cancer progression and treatment response has been understudied due to limited experimental systems for tracking and analyzing individual fusion events. To meet this need, we developed a molecular toolkit to map the origins and outcomes of individual cell fusion events within a tumor cell population. This platform, ClonMapper Duo (‘CMDuo’), identifies cells that have undergone cell-cell fusion through a combination of reporter expression and engineered fluorescence-associated index sequences paired to random barcode sets. scRNA-seq of the indexed barcodes enables the mapping of each set of parental cells and fusion progeny throughout the cell population. In triple negative breast cancer cells CMDuo uncovered subclonal transcriptomic hybridization and unveiled distinct cell-states which arise in direct consequence of homotypic cell-cell fusion. CMDuo is a platform that enables mapping of cell-cell fusion events in high-throughput single cell data and enables the study of cell fusion in disease progression and therapeutic response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05fe68cd8b32acb61197d1f1f0a4cc878efa91c6" target='_blank'>
              Mapping cell-cell fusion at single-cell resolution
              </a>
            </td>
          <td>
            Andrea L. Gardner, Lan Zheng, Kennedy Howland, Andrew Saunders, Andrea Ramirez, Patrik Parker, Chisom Iloegbunam, Daylin Morgan, T. Jost, Amy Brock
          </td>
          <td>2024-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cancer is caused by mutations that drive aberrant growth, proliferation, and invasion, thus overriding regulatory mechanisms that normally link these processes to organismal needs and cellular physiology. This imposes demands for the production of energy and biomass and for survival in microenvironments that are often nonphysiologic and nutrient-poor, which are met by rewiring of cellular metabolism. The resultant dependence of tumor cells on altered metabolism can induce sensitivity to specific metabolic perturbations that can be exploited for cancer therapy. Some cancers are caused by mutations that impart a novel function to metabolic enzymes, leading to the production of a tumor-promoting metabolite that is dispensable in normal cells, representing an ideal therapeutic target. Tumors can also exploit metabolic regulation of cellular immunity to evade antitumor immune responses, and deciphering this biology has revealed potential targets for therapeutic intervention. Here, we discuss a number of illustrative examples highlighting the therapeutic potential and the challenges of targeting metabolism for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/096951e0d89ec39b14a1cc5abbb929dff9f4e14a" target='_blank'>
              Cancer Therapies Targeting Cellular Metabolism.
              </a>
            </td>
          <td>
            Benjamin Morris, Alejandro Gutierrez
          </td>
          <td>2024-11-25</td>
          <td>Cold Spring Harbor perspectives in medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Breast cancer (BC) is considered as the most frequent cancer among women and a significant contributor to mortality. The CRISPR/Cas9 gene-editing tool has promising applications for BC drug resistance. It is a unique and creative approach that has lately attracted attention and can be used to fix gene alterations related to multidrug resistance. Recent research has effectively investigated and targeted particular genes linked to BC treatment resistance using CRISPR/Cas9 gene editing, including those linked to hormone receptor signaling, drug efflux transporters, and DNA repair pathways. The CRISPR/Cas9 technology's selective disruption or mutation of these genes provides valuable information about their role in resistance and paves the path for cutting-edge treatment options. CRISPR/Cas9 gene editing can overcome BC treatment resistance by identifying crucial genetic variables and revealing new therapeutic targets Despite the advantages, there are limitations in the study on CRISPR/Cas9-based gene editing for BC treatment resistance, for example, off-target effects and the improvement of delivery techniques are still major issues. Successful clinical translation depends on methods to improve the specificity and effectiveness of CRISPR/Cas9 editing and to solve these constraints. This review aims to explore the possibility of CRISPR/Cas9 gene editing as an innovative method of combating BC medication resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4162c264e1c96e6bc11bcded7869da58b56f7ba" target='_blank'>
              Molecular Mechanisms of Breast Cancer Drug Resistance and CRISPR/Cas9 Strategies to Overcome
              </a>
            </td>
          <td>
            B. Hussen, Bnar Saleh Ismael, Saman S. Abdulla, Noor Haval Jamal, S. A. Mustafa, Zana Baqi Najmalddin, Mohammed Fatih Rasul
          </td>
          <td>2024-12-30</td>
          <td>BioMed Target Journal</td>
          <td>1</td>
          <td>26</td>
        </tr>

        <tr id="Extensive intratumor heterogeneity in glioblastoma (GBM) impedes successful treatment and complicates drug discovery as it is not obvious which cells a tumor is most dependent on. Here, we posit that single-cell-resolution transcriptomic data can be integrated with loss-of-function screens to identify the most critical cells to target within a tumor. We parsed CRISPR screen data from the Dependency Map (DepMap) Consortium and identified a GBM Dependency Signature (GDS) − 168 genes that are essential for GBM cell viability in vitro. Through similarity scoring of GDS transcriptomic profiles in single-cell RNA-sequencing (scRNA-seq) data and iterative hierarchical clustering, we identify and report here 3 single-cell vulnerability states (VS) characterized in 49 GBM tumors using both scRNA-seq and spatial transcriptomic data. These VS reflect single-cell gene dependencies and differ significantly in enrichment profiles and spatial distributions. Importantly, the proportion of VS in each GBM tumor is variable, suggesting a means of stratifying patients in clinical trials. Collectively, we have developed a novel computational pipeline to identify unique vulnerability states in GBM and other cancers, which can be used to identify existing or novel drugs for incurable diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4aef7e027ff56302092f68c8a779be84bebb597e" target='_blank'>
              Identification of disease-specific vulnerability states at the single-cell level
              </a>
            </td>
          <td>
            Matthew D’Antuono, Madison Sharp, Rishika Chowdary, Michael E. Ivan, Ricardo J Komotar, Robert K. Suter, N. Ayad
          </td>
          <td>2024-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Tumor heterogeneity is a significant factor influencing cancer treatment effectiveness and can arise from genetic, epigenetic, and phenotypic variations among cancer cells. Understanding how tumor heterogeneity impacts tumor evolution and therapy response can lead to more effective treatments and improved patient outcomes. Traditional bulk genomic approaches fail to provide insights into cellular-level events, whereas single-cell RNA sequencing (scRNA-seq) offers transcriptomic analysis at the individual cell level, advancing our understanding of tumor growth, progression, and drug response. However, implementing single-cell approaches in clinical trials involves challenges, such as obtaining high-quality cells, technical variability, and the need for complex computational analysis. Effective implementation of single-cell genomics in clinical trials requires a collaborative “Team Medicine” approach, leveraging shared resources, expertise, and workflows. Here, we describe key technical considerations in implementing the collection of research biopsies and lessons learned from integrating scRNA-seq into City of Hope’s clinical trial design, highlighting collaborative efforts between computational and clinical teams across breast, brain, and ovarian cancer studies to understand the composition, phenotypic state, and underlying resistance mechanisms within the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8e501cb60571ffc07f7277f4aa067130d5278a3" target='_blank'>
              Single-Cell Transcriptomics Sheds Light on Tumor Evolution: Perspectives from City of Hope’s Clinical Trial Teams
              </a>
            </td>
          <td>
            Patrick A. Cosgrove, Andrea H. Bild, T. Dellinger, B. Badie, J. Portnow, Aritro Nath
          </td>
          <td>2024-12-01</td>
          <td>Journal of Clinical Medicine</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Background Cellular plasticity, involving loss of lineage determination and emergence of hybrid cell states, plays a pivotal role in non-small cell lung cancer (NSCLC) disease progression and therapy resistance. However, the full spectrum of atypical states generated in human NSCLC and the pathways that regulate them are yet to be fully elucidated. Here we examine the role of developmental programmes, alveogenesis and branching morphogenesis (BM), in regulating phenotypic diversity in NSCLC. Methods Transcriptomic analysis of epithelial cells isolated from murine lungs at different stages of organogenesis were used to derive gene signatures for developmental programmes. Bulk tissue transcriptomic datasets from human NSCLC and non-neoplastic control samples were used to identify whether developmental programmes were associated with molecular, morphological, and clinical parameters. Single-cell RNA-sequencing was used to identify malignant cell states in human NSCLC (n = 16,621 epithelial cells from 72 samples) and protein level validation of these states was carried out using multiplexed immunohistochemistry (n = 40). Results Mutually antagonistic regulation of alveogenesis and BM was found to account for a significant proportion of transcriptomic variance in human NSCLC bulk tissue datasets. BM activation was associated with poor overall survival rates in five independent lung adenocarcinoma (LUAD) cohorts (p=2.04e-13); and was significantly prognostic for resistance to tyrosine kinase inhibitors (TKIs; p=0.003) and immune checkpoint blockade (ICBs; p=0.014), in pre-treatment biopsies. Single-cell RNA-sequencing analysis revealed that malignant LUAD cells with loss of alveolar lineage fidelity predominantly acquired inflamed or basal-like cellular states, which were variably persistent in samples from TKI and ICB recurrence. Conclusions Our results show LUAD tumours undergo reversion from an alveogenic to branching morphogenic phenotype during disease progression, generating inflamed or basal-like cell states that are variably persistent following TKI or ICB treatments. These findings identify prognostic biomarkers for therapy response and underscore the role of different cell states in resistance to multiple treatment modalities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/390cd095a97603dc458c49ab8aaa4b76e0155529" target='_blank'>
              Developmental programmes drive cellular plasticity, disease progression and therapy resistance in lung adenocarcinoma
              </a>
            </td>
          <td>
            Kamila J Bienkowska, Stephany Gallardo Y, Nur S Zainal, Matthew Ellis, Maria-Antoinette Lopez, Judith Austine, Sai Pittla, S. Chee, A. Alzetani, Emily C Shaw, Christian H Ottensmeier, Gareth J Thomas, Christopher Hanley
          </td>
          <td>2024-12-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="All forms of genetic variation originate from new mutations, making it crucial to understand their rates and mechanisms. Here, we use long-read PacBio sequencing to investigate de novo mutations that accumulated in 12 inbred mouse lines derived from three commonly used inbred strains (C3H, C57BL/6, and FVB) maintained for 8-15 generations in a mutation accumulation (MA) experiment. We built chromosome-level genome assemblies based on the MA line founders' genomes, and then employed a combination of read and assembly-based methods to call the complete spectrum of new mutations. On average, there are ~45 mutations per haploid genome per generation, about half of which (54%) are insertions and deletions shorter than 50 bp (indels). The remainder are single nucleotide mutations (SNMs, 44%) and large structural mutations (SMs, 2%). We found that the degree of DNA repetitiveness is positively correlated with SNM and indel rates, and that a substantial fraction of SMs can be explained by homology-dependent mechanisms associated with repeat sequences. Most (90%) indels can be attributed to microsatellite contractions and expansions, and there is a marked bias towards 4 bp indels. Among the different types of SMs, tandem repeat mutations have the highest mutation rate, followed by insertions of transposable elements (TEs). We uncover a rich landscape of active TEs, and notable differences in their spectrum among MA lines and strains, and a high rate of gene retroposition. Our study offers novel insights into mammalian genome evolution, and highlights the importance of repetitive elements in shaping genomic diversity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4966070a9e9e2ca28b9c82696887c588bee4c86" target='_blank'>
              The rate and spectrum of new mutations in mice inferred by long-read sequencing.
              </a>
            </td>
          <td>
            Eugenio López-Cortegano, Jobran Chebib, Anika Jonas, Anastasia Vock, Sven Künzel, P. Keightley, D. Tautz
          </td>
          <td>2024-12-02</td>
          <td>Genome research</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="Many agents that show promise in preclinical cancer models lack efficacy in patients due to patient heterogeneity that is not captured in traditional assays. To address this problem, we have developed GENEVA, a platform that measures the molecular and phenotypic consequences of drug perturbations within diverse populations of cancer cells at single-cell resolution, both in vitro and in vivo. Here, we apply GENEVA to study the KRAS G12C inhibitors, recapitulating known properties of these drugs and uncovering a previously unknown role for mitochondrial activation in cell death induced by KRAS inhibition. We demonstrate that this finding can be leveraged for the development of combination therapies with greater efficacy. Finally, we show that the application of GENEVA with in vivo mouse models revealed epithelial to mesenchymal transition (EMT) as a key mechanism for resistance to KRAS G12C inhibition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/198b25a5af544d70c25d15829b150e55c9e084e8" target='_blank'>
              Multiplexed mosaic tumor models reveal natural phenotypic variations in drug response within and between populations
              </a>
            </td>
          <td>
            Johnny X. Yu, Jung Min Suh, Katerina D. Popova, Kristle Garcia, Tanvi Joshi, Bruce Culbertson, Jessica B. Spinelli, Vishvak Subramanyam, Kevin Lou, K. Shokat, Jonathan Weissman, Hani Goodarzi
          </td>
          <td>2024-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>131</td>
        </tr>

        <tr id="The emergence of drug resistance leading to cancer recurrence is one of the challenges in the treatment of cancer patients. Several mechanisms can lead to drug resistance, including epigenetic changes. Histone deacetylases (HDACs) play a key role in chromatin regulation through epigenetic mechanisms and are also involved in drug resistance. The control of histone acetylation and the accessibility of regulatory DNA sequences such as promoters, enhancers, and super-enhancers are known mechanisms by which HDACs influence gene expression. Other targets of HDACs that are not histones can also contribute to resistance. This review describes the contribution of HDACs to the mechanisms that, in some cases, may determine resistance to chemotherapy or other cancer treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4960e67938604ec9bdc889e3cde099b5e416cd0e" target='_blank'>
              HDAC-driven mechanisms in anticancer resistance: epigenetics and beyond
              </a>
            </td>
          <td>
            Martina Minisini, Martina Mascaro, Claudio Brancolini
          </td>
          <td>2024-11-20</td>
          <td>Cancer Drug Resistance</td>
          <td>1</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e10dcc942b3981d9a72139061b995f341cc1437" target='_blank'>
              Protacs in cancer therapy: mechanisms, design, clinical trials, and future directions.
              </a>
            </td>
          <td>
            Akash Vikal, Rashmi Maurya, Brij Bihari Patel, Rajeev Sharma, Preeti Patel, Umesh K Patil, Balak Das Kurmi
          </td>
          <td>2024-11-29</td>
          <td>Drug delivery and translational research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Cancer cell lines are the most common in-vitro models for the evaluation of anti-cancer drug sensitivities. Past studies have been conducted to decipher and characterize the pharmacogenomic feature of cell lines based on other omics data, such as genomic mutation data and whole-genome RNA sequencing (RNA-seq) profiles. In particular, proteomic data is also an essential component for the characterization of tumours. However, different from RNA-seq datasets rich in numerous transcriptome profiles of cancer cell lines and cell viability assay of drug responses, the pharmacogenomic protein quantifications are relatively scarce. With the availability of the recently enriched proteomic dataset ProCan-DepMapSanger, we systematically evaluated the interplays among genomic mutations, transcription, and protein expressions across cancer cell lines. In general, blood cancers have higher RNA-protein correlations than those in solid cancers. The differential expression analysis on protein data helped identify more expressional and functional impact of genomic mutations of cancer genes. We also integrated the proteomic map with drug molecular chemical features to construct a bi-modal machine learning model to infer the drug sensitivities of cancer cell lines. Our results demonstrated that protein quantifications can lead to better drug response prediction performance than the model trained on transcriptome profiles. In addition, integrating protein data with drug chemical features, represented as molecular graphs and learned by Graph Neural Network, outperformed the state-of-the-art model DeepOmicNet for drug response prediction in proteomics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c89e8ade30b207677f590882fddb023a8cd320d3" target='_blank'>
              Drug Response Modeling across Cancers: Proteomics vs. Transcriptomics
              </a>
            </td>
          <td>
            Zetian Zheng, Lei Huang, Fuzhou Wang, Linjing Liu, Jixiang Yu, Weidun Xie, Xingjian Chen, Xiangtao Li, Ka-Chun Wong
          </td>
          <td>2024-12-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Adenosine-to-Inosine (A-to-I) RNA editing is a critical post-transcriptional modification that diversifies the transcriptome and influences various cellular processes. Despite its significance, the regulatory mechanisms controlling A-to-I editing remain largely unknown. In this study, we present two complementary CRISPR-based genetic screening platforms: CREDITS (CRISPR-based RNA EDITing regulator Screening), which enables genome-scale identification of editing regulators using an RNA recorder-based reporter system, and scCREDIT-seq (single-cell CRISPR-based RNA EDITing sequencing), which provides multiplexed single-cell characterization of transcriptome and editome changes for pooled perturbations on dozens of selected genes. Through screening 1,350 RNA-binding proteins, we identified a series of known and novel A-to-I editing regulators. Detailed mechanistic investigation revealed DDX39B as a global repressor of A-to-I editing, which functions by preventing double-stranded RNA accumulation through its helicase and ATPase activities. We demonstrate that targeting DDX39B significantly enhances the efficiency of RNA editing-based tools like CellREADR and LEAPER, and represents a promising strategy for anti-HDV therapy by modulating viral genome editing. These technological advances not only expand our understanding of RNA editing regulation but also provide powerful tools for exploring tissue-specific and context-dependent RNA modification mechanisms, with broad implications for therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d02f7d393c0da3066112a870364908fe61bba6dd" target='_blank'>
              Multimodal CRISPR screens uncover DDX39B as a global repressor of A-to-I RNA editing
              </a>
            </td>
          <td>
            Tianzi Wei, Jiaxuan Li, Xiang Lei, Risheng Lin, Qingyan Wu, Zhenfeng Zhang, Shimin Shuai, Ruilin Tian
          </td>
          <td>2024-11-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28cbf9783c62fb0f01914152c3e6d88182c98411" target='_blank'>
              LKB1 inactivation promotes epigenetic remodeling-induced lineage plasticity and antiandrogen resistance in prostate cancer
              </a>
            </td>
          <td>
            Fei Li, Pengfei Dai, Huili Shi, Yajuan Zhang, Juan He, Anuradha Gopalan, Dan Li, Yu Chen, Yarui Du, Guoliang Xu, Weiwei Yang, Chao Liang, Dong Gao
          </td>
          <td>2025-01-01</td>
          <td>Cell Research</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="Loss of function screens using shRNA (short hairpin RNA) and CRISPR (clustered regularly interspaced short palindromic repeats) are routinely used to identify genes that modulate responses of tumor cells to anti-cancer drugs. Here, by integrating GSEA (Gene Set Enrichment Analysis) and CMAP (Connectivity Map) analyses of multiple published shRNA screens, we identified a core set of pathways that affect responses to multiple drugs with diverse mechanisms of action. This suggests that these pathways represent “weak points” or “Achilles heels”, whose mild disturbance should make cancer cells vulnerable to a variety of treatments. These “weak points” include proteasome, protein synthesis, RNA splicing, RNA synthesis, cell cycle, Akt-mTOR, and tight junction-related pathways. Therefore, inhibitors of these pathways are expected to sensitize cancer cells to a variety of drugs. This hypothesis was tested by analyzing the diversity of drugs that synergize with FDA-approved inhibitors of the proteasome, RNA synthesis, and Akt-mTOR pathways. Indeed, the quantitative evaluation indicates that inhibitors of any of these signaling pathways can synergize with a more diverse set of pharmaceuticals, compared to compounds inhibiting targets distinct from the “weak points” pathways. Our findings described here imply that inhibitors of the “weak points” pathways should be considered as primary candidates in a search for synergistic drug combinations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d68c29dcb0705107711d0d05473f868fd7dbb129" target='_blank'>
              Cancer Cell’s Achilles Heels: Considerations for Design of Anti-Cancer Drug Combinations
              </a>
            </td>
          <td>
            Valid Gahramanov, Frederick S. Vizeacoumar, Alain Morejon Morales, Keith Bonham, M. Sakharkar, Santosh Kumar, Franco J Vizeacoumar, A. Freywald, Michael Y. Sherman
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fef73b59f7e05ebe6c2a352df52b047a092cb9e6" target='_blank'>
              State of the art CRISPR-based strategies for cancer diagnostics and treatment
              </a>
            </td>
          <td>
            E. Di Carlo, Carlo Sorrentino
          </td>
          <td>2024-12-18</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Sinonasal squamous cell carcinoma (SNSCC) is a rare and aggressive malignancy with limited treatment options, necessitating comprehensive molecular characterization to uncover actionable therapeutic targets. Here, we integrate bulk and single-cell transcriptomics, epigenomics, and DNA methylation profiling to construct a detailed molecular and cellular landscape of SNSCC. Our analysis identifies five transcriptionally and epigenetically distinct malignant subpopulations, including hypoxic (TC1) and proliferative (TC2) clusters, with TC1 showing significant association with poor clinical outcomes. We uncover a hypoxia-driven angiogenic axis, wherein TC1 cells secrete adrenomedullin (ADM), MIF, and VEGFA to activate endothelial tip cells, fostering tumor angiogenesis. Epigenetic reprogramming, characterized by alterations in DNA methylation and chromatin accessibility, underpins these transcriptional programs, with AP-1 transcription factors emerging as key regulators. Notably, ADM expression is epigenetically controlled, correlates with advanced clinical stage, and predicts reduced survival in head and neck cancer patients. This multi-omic study highlights the tumor heterogeneity and microenvironmental interactions driving SNSCC progression, revealing epigenetically regulated pathways as promising therapeutic targets for future intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0f1bcee9ecccbc4aae5987d7312ce58acb94e80" target='_blank'>
              Multi-omic profiling identifies malignant subpopulations and a hypoxia-driven angiogenic axis as therapeutic vulnerabilities in sinonasal squamous cell carcinoma
              </a>
            </td>
          <td>
            Chaelin You, Jaewoo Park, Jung Yeon Jang, Myeong S. Yu, Yoo-Sam Chung, Keunsoo Kang, Jihwan Park, Ji Heui Kim, Kyuho Kang
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Replication timing (RT) allows us to analyze temporal patterns of genome-wide replication, i.e., if genes replicate early or late during the S-phase of the cell cycle. RT has been linked to gene expression in normal and diseased acute and chronic states such as cancer. However, studies done to date focused on bulk cell populations that required tens of thousands of cells for RT analysis. Here, we developed an affordable novel single cell (sc)-multiomics approach to simultaneously analyze RT and gene expression from cells or nuclei. We used this approach to generate sc-RT profiles and sc-gene expression data from the well-established human liver cancer cell line, HepG2. We demonstrated that as few as 17 mid S-phase cells were sufficient to produce cell-type specific pseudo bulk RT profiles that had a high correlation to previously published HepG2 bulk RT profiles. The sc-RT profiles allowed us to visualize how individual cells progressed through genome replication. We were also able to demonstrate high-resolution correlations between RT and gene expression within each individual cell, which to our knowledge, has not been reported. We observed trends that were conserved between individual cells, as well as cell-to-cell variations, which were not possible to detect with the bulk RT studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36c8d64daee99beea62c15fd3842a66ac8960670" target='_blank'>
              Development of a Novel Single Cell Multiomics Approach for Simultaneous Analysis of Replication Timing and Gene Expression
              </a>
            </td>
          <td>
            Anala V. Shetty, Clifford J. Steer, Walter C. Low
          </td>
          <td>2025-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9043774f4d71a3f413b248306070648bf9c170e4" target='_blank'>
              Multi-omics approaches reveal that diffuse midline gliomas present altered DNA replication and are susceptible to replication stress therapy
              </a>
            </td>
          <td>
            Anastasia E. Hains, Kashish Chetal, Tsunetoshi Nakatani, Joana G. Marques, Andreas Ettinger, C. B. Junior, Adriana Gonzalez-Sandoval, Renjitha Pillai, Mariella G. Filbin, M. Torres-Padilla, Ruslan I. Sadreyev, Capucine Van Rechem
          </td>
          <td>2024-12-20</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Simple Summary KMT2A partial tandem duplication (PTD) involves intragenic duplications within the KMT2A gene and has been associated with acute myeloid leukemia (AML) and myelodysplastic syndrome (MDS). KMT2A PTD cannot be detected by conventional karyotyping or fluorescence in situ hybridization (FISH). In this study, we used optical genome mapping (OGM) to analyze the cytogenomic alterations in 1277 hematolymphoid neoplasms and identified KMT2A PTD exclusively in patients with myeloid neoplasms, including 35 (7%) with AML, 5 (2.2%) with MDS, and 5 (7.2%) with chronic myelomonocytic leukemia (CMML). Neoplasms with KMT2A PTD frequently exhibit a normal or non-complex karyotype and consistently harbor gene mutations involving epigenetic regulators, the FLT3/RAS signaling pathway, transcription factors, and spliceosome genes. Patients with KMT2A PTD are generally resistant to conventional chemotherapy, with the exception of those with de novo AML, which demonstrates a high remission rate. Patients with KMT2A PTD positive secondary AML and MDS had a poor outcome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fce3f21aa3468dded682d3ad1d494e6eb3d9bfdb" target='_blank'>
              Detection of KMT2A Partial Tandem Duplication by Optical Genome Mapping in Myeloid Neoplasms: Associated Cytogenetics, Gene Mutations, Treatment Responses, and Patient Outcomes
              </a>
            </td>
          <td>
            Qing Wei, Shimin Hu, Jie Xu, Sanam Loghavi, Naval G Daver, G. Toruner, Wei Wang, L. J. Medeiros, Guilin Tang
          </td>
          <td>2024-12-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Tubo-ovarian high-grade serous carcinoma (HGSC) is the most lethal gynecological malignancy and frequently responds to platinum-based chemotherapy because of common genetic and somatic impairment of DNA damage repair (DDR) pathways. The mechanisms of clinical platinum resistance are diverse and poorly molecularly defined. Consequently, there are no biomarkers or medicines that improve patient outcomes. Herein we use single cell mass cytometry (CyTOF) to systematically evaluate the phosphorylation and abundance of proteins known to participate in the DNA damage response (DDR). Single cell analyses of highly characterized HGSC cell lines that phenocopy human patients show that cells with comparable levels of intranuclear platinum, a proxy for carboplatin uptake, undergo different cell fates. Unsupervised analyses revealed a continuum of DDR responses. Decompositional methods were used to identify eight distinct protein modules of carboplatin resistance and sensitivity at single cell resolution. CyTOF profiling of primary and secondary platinum-resistance patient models shows that a complex DDR sensitivity module is strongly associated with response, suggesting it as a potential tool to clinically characterize complex drug resistance phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1700b92ee74968435b1e4e6c96fff15f73c54ccf" target='_blank'>
              Coordinated protein modules define DNA damage responses to carboplatin at single cell resolution in human ovarian carcinoma models
              </a>
            </td>
          <td>
            Jacob S. Bedia, Ying-Wen Huang, Antonio Delgado Gonzalez, Veronica D. Gonzalez, I. Funingana, Zainab Rahil, Alyssa Mike, Alexis Lowber, M. Vias, Alan Ashworth, James D. Brenton, W. Fantl
          </td>
          <td>2024-11-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Targeted therapies like Venetoclax have increased the options available to acute myeloid leukemia (AML) patients, but survival remains poor due to drug resistance and disease relapse. We found that the translation initiation factor EIF4A1, which unwinds complex mRNA structures in the 5’ UTR of oncogenic transcripts, is highly expressed in AML stem- and progenitor-like cells. Inhibiting eIF4A with the small molecule Zotatifin reduces translation of transcripts related to the cell cycle and survival. This results in downregulation of AKT, STAT-5, and MCL-1 and underlies synergy of Zotatifin with Venetoclax. The drug combination promotes apoptosis across AML genotypes, while the effect on healthy blood cells is limited. Using in vivo relapsed and refractory AML patient-derived xenograft models, the combination significantly suppressed tumor burden and prolonged survival of xenografted mice. These results support eIF4A-mediated protein translation as a therapeutic target in AML. SIGNIFICANCE Despite advances in targeted therapies, the 5-year survival rate for acute myeloid leukemia remains around 32%. The efficacy of existing treatments may be improved with the addition of Zotatifin, an inhibitor targeting the translation initiation factor eIF4A.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8672d4c71b8c253c15ee73780df3c698e781744" target='_blank'>
              Research Brief: Translation Initiation Represents an Acute Myeloid Leukemia Cell Vulnerability That Can Be Co-Targeted With BCL-2 Inhibition
              </a>
            </td>
          <td>
            Y. Lee, Jonathan D. Good, Noha Awais, Benjamin K. Eschle, Maia S. Lineberry, Jessica L. Teo, Sarah Bibeau, Morgan Wollman, George Karadimov, Nurefsan Sariipek, Jean Acosta, Jenny Noel, David Cucchi, E. DePasquale, Prafulla Gokhale, Fadi J. Najm, Richard M. Stone, Jacqueline S. Garcia, Andrew A. Lane, Peter van Galen
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background: Drug resistance is a major obstacle to the successful treatment of non-small cell lung cancer (NSCLC), the most common subtype of lung cancer. The purpose of this research was to look at possible treatment approaches and the processes behind drug resistance in non-small cell lung cancer.
Methods: From March 2022 to February 2024, a prospective observational research was carried out at Lady Reading Hospital in Peshawar. 128 NSCLC patients in all were enrolled. Next-generation sequencing, DNA methylation tests, and immunological profiling were used to examine the genetic, epigenetic, and environmental components linked to medication resistance. Multivariate logistic regression, t-tests, chi-square tests, and Kaplan-Meier survival analysis were among the statistical techniques used.
Results: T790M mutations contributed to resistance, with EGFR mutations found in 35% of patients. Ten percent of patients had MET amplifications, while seven percent of patients had ALK rearrangements. In resistant cancers, epigenetic changes such as hypoacetylation of histones and hypermethylation of CDKN2A were common. Immunosuppressive cells and cancer-associated fibroblasts were abundant in the tumor microenvironment. The median overall survival (OS) of patients with drug-resistant NSCLC was 14 months, a substantial decrease from the 22 months seen in patients without resistance (p < 0.001).
Conclusions: The research emphasizes how genetic, epigenetic, and microenvironmental variables all play a role in the complex process of medication resistance in non-small cell lung cancer. Molecular profiling and targeted medicines along with personalized treatment techniques are critical to overcome resistance and enhancing patient outcomes. Larger, more varied populations and the creation of cutting-edge treatment approaches should be the main topics of future study.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96863261655834ce5a911e129e2f5a3a8ee64f44" target='_blank'>
              Understanding the Mechanisms of Drug Resistance in Non-Small Cell Lung Cancer Potential Therapeutic Approaches
              </a>
            </td>
          <td>
            Afnan Ali Shah, Waqar Ahmad Tahir, Owais Nawaz Khan, A. Abid, Yasar Umar Khan, Talha Mazhar
          </td>
          <td>2024-12-31</td>
          <td>Innovative Research in Applied, Biological and Chemical Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Glioblastoma (GBM) cells are highly heterogeneous and invasive, leading to treatment resistance and relapse. However, the molecular regulation in and distal to tumors remains elusive. Here, we collected paired tissues from the tumor core (TC) and peritumoral brain (PTB) for integrated snRNA-seq and snATAC-seq analyses. Tumor cells infiltrating PTB from TC behave more like oligodendrocyte progenitor cells than astrocytes at the transcriptome level. Dual-omics analyses further suggest that the distal regulatory regions in the tumor genome and specific transcription factors are potential determinants of regional heterogeneity. Notably, while activator protein 1 (AP-1) is active in all GBM states, its activity declines from TC to PTB, with another transcription factor, BACH1, showing the opposite trend. Combined inhibition of AP-1 and BACH1 more efficiently attenuates the tumor progression in mice and prolongs survival than either single-target treatment. Together, our work reveals marked molecular alterations of infiltrated GBM cells and a synergy of combination therapy targeting intratumor heterogeneity in and distal to GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63c193d462d0c2dbf4aee7a761498e3e53b84808" target='_blank'>
              Single-cell multi-omics sequencing uncovers region-specific plasticity of glioblastoma for complementary therapeutic targeting
              </a>
            </td>
          <td>
            Xin Wang, Qian Sun, Tianbin Liu, Haoran Lu, Xuyi Lin, Weiwen Wang, Yang Liu, Yunting Huang, Guodong Huang, Haixi Sun, Qianxue Chen, Junmin Wang, Daofeng Tian, Fan-En Yuan, Longqi Liu, Bo Wang, Ying Gu, Baohui Liu, Liang Chen
          </td>
          <td>2024-11-22</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/444c4446513d7716773a0e9577f630bc02f3f28a" target='_blank'>
              Advances in Understanding Drug Resistance Mechanisms and Innovative Clinical Treatments for Melanoma.
              </a>
            </td>
          <td>
            Xiaoya He, Hao Deng, Wei Liu, Liling Hu, Xiao Tan
          </td>
          <td>2024-12-05</td>
          <td>Current treatment options in oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40243d83674d54172095c39c31dd740a67953065" target='_blank'>
              Comparative transcriptomics suggests a highly species-specific nature of the phenotypic plasticity associated with the outbreaks of the two main pest locusts
              </a>
            </td>
          <td>
            N. Bakkali, S. Saadi, A. Badih, M. Bakkali
          </td>
          <td>2024-12-19</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1aaea8ef96bcf757be9b672ce6a26fbe4a6d0c91" target='_blank'>
              Integration of single-cell sequencing and drug sensitivity profiling reveals an 11-gene prognostic model for liver cancer
              </a>
            </td>
          <td>
            Qunfang Zhou, Jingqiang Wu, Jiaxin Bei, Zixuan Zhai, Xiuzhen Chen, Wei Liang, Jing Meng, Mingyu Liu
          </td>
          <td>2024-11-25</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The P-glycoprotein efflux pump, encoded by the MDR1 gene, is an ATP-driven transporter capable of expelling a diverse array of compounds from cells. Overexpression of this protein is implicated in the multi-drug resistant phenotype observed in various cancers. Numerous studies have attempted to decipher the impact of genetic variants within MDR1 on P-glycoprotein expression, functional activity, and clinical outcomes in cancer patients. Among these, three specific single nucleotide polymorphisms—T1236C, T2677G, and T3435C - have been the focus of extensive research efforts, primarily through in vitro cell line models and clinical cohort analyses. However, the findings from these studies have been remarkably contradictory. In this study, we employ a computational, data-driven approach to systematically evaluate the effects of these three variants on principal stages of the gene expression process. Leveraging current knowledge of gene regulatory mechanisms, we elucidate potential mechanisms by which these variants could modulate P-glycoprotein levels and function. Our findings suggest that all three variants significantly change the mRNA folding in their vicinity. This change in mRNA structure is predicted to increase local translation elongation rates, but not to change the protein expression. Nonetheless, the increased translation rate near T3435C is predicted to affect the protein’s co-translational folding trajectory in the region of the second ATP binding domain. This potentially impacts P-glycoprotein conformation and function. Our study demonstrates the value of computational approaches in elucidating the functional consequences of genetic variants. This framework provides new insights into the molecular mechanisms of MDR1 variants and their potential impact on cancer prognosis and treatment resistance. Furthermore, we introduce an approach which can be systematically applied to identify mutations potentially affecting mRNA folding in pathology. We demonstrate the utility of this approach on both ClinVar and TCGA and identify hundreds of disease related variants that modify mRNA folding at essential positions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3513112218d83dd11f2ce2e7d98d7d9261c6c5e8" target='_blank'>
              Computational Analysis of MDR1 Variants Predicts Effect on Cancer Cells via their Effect on mRNA Folding
              </a>
            </td>
          <td>
            T. Gutman, T. Tuller
          </td>
          <td>2024-12-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Dysregulated epigenetic programs that restrict differentiation, reactivate fetal genes, and confer phenotypic plasticity are critical to colorectal cancer (CRC) development. By screening a small molecule library targeting epigenetic regulators using our dual reporter system, we found that inhibiting histone deacetylase (HDAC) 1/2 promotes CRC differentiation and anti-tumor activity. Comprehensive biochemical, chemical, and genetic experiments revealed that on-target blockade of the HDAC1/2 catalytic domain mediated the differentiated phenotype. Unbiased profiling of histone posttranslational modifications induced by HDAC1/2 inhibition nominated acetylation of specific histone lysine residues as potential regulators of differentiation. Genome-wide assessment of implicated marks indicated that H3K27ac gains at HDAC1/2-bound regions associated with open chromatin and upregulation of differentiation genes upon HDAC1/2 inhibition. Disrupting H3K27ac by degrading acetyltransferase EP300 rescued HDAC1/2 inhibitor-mediated differentiation of a patient-derived CRC model using single cell RNA-sequencing. Genetic screens revealed that DAPK3 contributes to CRC differentiation induced by HDAC1/2 inhibition. These results highlight the importance of specific chemically targetable histone modifications in governing cancer cell states and epigenetic reprogramming as a therapeutic strategy in CRC. BRIEF SUMMARY HDAC1/2 inhibition promotes colorectal cancer differentiation via gains in H3K27ac, which can be reversed by blocking its acetyltransferase EP300.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/207eb16a60f11154e81c963033e896c8385daaa7" target='_blank'>
              Chemical perturbations impacting histone acetylation govern colorectal cancer differentiation
              </a>
            </td>
          <td>
            Pornlada Likasitwatanakul, Zhixin Li, Paul Doan, S. Spisák, Akhouri Kishore Raghawan, Qi Liu, Priscilla Liow, Sunwoo Lee, David Chen, Pratyusha Bala, P. Sahgal, Daulet Aitymbayev, Jennifer S. Thalappillil, Malvina Papanastasiou, William Hawkins, Steven A. Carr, Haeseong Park, James M. Cleary, Jun Qi, Nilay S. Sethi
          </td>
          <td>2024-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27645ec5d75c6606ae8cc897c50e3164a6947fb9" target='_blank'>
              Exploring new mechanisms in cancer molecular pathways and pathogenic cell transformation: PIP4K2A as a prognostic marker and therapeutic target in cutaneous malignant melanoma
              </a>
            </td>
          <td>
            Wen-fei Luo, Ding-ming Song, Tong Shen, Yi-bo He, Hong-yang Du, Ming-Jue Si, Lu-Wei Fang
          </td>
          <td>2024-11-23</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Understanding the molecular pathogenesis of MLL fusion oncoprotein (MLL-FP) leukaemia has spawned epigenetic therapies that have improved clinical outcomes in this often-incurable disease. Using genetic and pharmacological approaches, we define the individual and combined contribution of KAT6A, KAT6B and KAT7, in MLL-FP leukaemia. Whilst inhibition of KAT6A/B is efficacious in some pre-clinical models, simultaneous targeting of KAT7, with the novel inhibitor PF-9363, increases the therapeutic efficacy. KAT7 interacts with Menin and the MLL complex and is co-localised at chromatin to co-regulate the MLL-FP transcriptional program. Inhibition of KAT6/KAT7 provides an orthogonal route to targeting Menin to disable the transcriptional activity of MLL-FP. Consequently, combined inhibition rapidly evicts the MLL-FP from chromatin, potently represses oncogenic transcription and overcomes primary resistance to Menin inhibitors. Moreover, PF-9363 or genetic depletion of KAT7 can also overcome acquired genetic/non-genetic resistance to Menin inhibition. These data provide the molecular rationale for rapid clinical translation of combination therapy in MLL-FP leukaemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f99055bf5d30190642cc04c1d1e10700ca75334b" target='_blank'>
              Catalytic inhibition of KAT6/KAT7 enhances the efficacy and overcomes primary and acquired resistance to Menin inhibitors in MLL leukaemia
              </a>
            </td>
          <td>
            Shellaina J. V. Gordon, Florian Perner, Laura MacPherson, Daniela V. Wenge, Wallace Bourgeois, Katie Fennell, Tabea Klaus, Jelena Petrovic, Jakub Horvath, Joan Cao, John J. Lapek, Sean Uryu, Jeffrey White, E. Lam, X. Mu, Yih-Chih Chan, Andrea Gillespie, Benjamin Blyth, M. Camerino, Ylyva E. Bozikis, Henrietta Holze, Kathy Knezevic, Jesse J Balic, P. Stupple, Ian P. Street, B. Monahan, Shikhar Sharma, Elanor N. Wainwright, Dane Vassiliadis, Thomas A. Paul, Scott A. Armstrong, Mark A Dawson
          </td>
          <td>2024-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc4ce21cb54a65498b2c72289ec6bd7eb2e7dc62" target='_blank'>
              Characterization of shared neoantigens landscape in Mismatch Repair Deficient Endometrial Cancer
              </a>
            </td>
          <td>
            E. De Paolis, C. Nero, E. Micarelli, Guido Leoni, A. Piermattei, R. Trozzi, Elisa Scarselli, A. D’Alise, Luciano Giacò, M. De Bonis, A. Preziosi, Gennaro Daniele, Diletta Piana, T. Pasciuto, Gianfranco Zannoni, Angelo Minucci, G. Scambia, Andrea Urbani, Francesco Fanfani
          </td>
          <td>2024-12-20</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Glioblastoma (GBM), the most prevalent primary malignant brain tumor, remains challenging to treat due to extensive inter- and intra-tumor heterogeneity. This variability demands combination treatments to improve therapeutic outcomes. A significant obstacle in treating GBM is the expression of O6-methylguanine-DNA methyltransferase, a DNA repair enzyme that reduces the efficacy of the standard alkylating agent, temozolomide, in about 50% of patients. This underscores the need for novel, more targeted therapies. Our study investigates the metabolic–epigenetic impact of combining SN-38, a novel topoisomerase inhibitor inducing DNA double-strand breaks, with rabusertib, a checkpoint kinase 1 inhibitor. We identified this synergistic combination through high-throughput drug screening across a panel of GBM cell lines using a cancer drug library combined with SN-38. A secondary metabolic screening with the PEDS algorithm demonstrated a synergistic modulation of purine, one-carbon, and redox metabolism. Furthermore, the combined treatment led to the significant depletion of epigenetically relevant metabolites such as 5-methyl-cytosine, acetyl-lysine, and trimethyl-lysine. Reduced intermediates of the glutathione cycle indicated increased cellular stress following combinatorial treatment. Overall, the combination of SN-38 and rabusertib synergistically disrupts metabolites associated with epigenetic adaptations, leading to cytotoxicity independent of O6-methylguanine-DNA methyltransferase status, thereby underpinning this combination as a promising candidate for combinatorial therapy in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ed32abd0bdaa8ba2724fd42a8df45084943d6e1" target='_blank'>
              Targeting Metabolic and Epigenetic Vulnerabilities in Glioblastoma with SN-38 and Rabusertib Combination Therapy
              </a>
            </td>
          <td>
            J. Chiou, Valeria Impedovo, Yen Bao Huynh, Ruggiero Gorgoglione, Luiz O. F. Penalva, A. Lodi, Andrew J. Brenner, Stefano Tiziani
          </td>
          <td>2025-01-08</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Next‐generation sequencing (NGS) has emerged as a transformative technology in oncology, revolutionizing cancer diagnostics and personalized treatment strategies. By providing comprehensive insights into the genetic landscape of tumors, NGS enables the identification of critical somatic and germline mutations, copy number variations (CNVs), and gene fusions. Over the past decade, advancements in NGS platforms have led to greater accuracy, speed, and cost‐effectiveness, making it an integral part of cancer research and clinical diagnostics. Despite its widespread adoption, significant challenges remain, including the need for improved methods to detect minimal residual disease (MRD) and accurately profile tumor heterogeneity. This review explores the evolution of NGS technologies and their pivotal role in cancer biology, from early diagnostics to therapeutic guidance. It delves into the application of NGS in identifying CNVs and gene fusions, monitoring MRD, and the increasing relevance of targeted NGS and spatial genomics. Furthermore, the integration of spatial transcriptomics is highlighted as a frontier in understanding the tumor microenvironment. By addressing these critical aspects, this review provides a comprehensive overview of how NGS is shaping the future of cancer research and treatment, offering a complete overview of potential NGS applications in scientific and clinical oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9ec16577d76cdb54f1382e600b551c2d7b5e57b" target='_blank'>
              Role of next‐generation sequencing in revolutionizing healthcare for cancer management
              </a>
            </td>
          <td>
            O. S. Sahoo, Hiya Aidasani, A. Nayek, Smita Tripathi, Joyeeta Talukdar, Anamta Gul, Deepak Kumar, Ruby Dhar, S. Karmakar
          </td>
          <td>2024-11-23</td>
          <td>MedComm – Future Medicine</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="High-risk multiple myeloma (MM) is genomically unstable, comprised of heterogeneous populations of tumor cells that evolve over time. Light chain escape (LCE) is a clinical phenomenon observed when light chains rise separately from M-spike values, implying divergent tumor evolution. We sought to understand LCE by performing high depth transcriptomic and phenotypic studies. We performed single cell RNA sequencing and ex vivo drug sensitivity profiling on serial bone marrow biopsies from a patient with LCE at diagnosis, first relapse and relapsed/refractory timepoints. Single cell RNA sequencing uncovered distinct transcriptomic subpopulations with phenotypes that could be tracked separately by clinical serum light chain and M-spike values. Genes differentially expressed between subpopulations were assessed for generalizable effects on prognosis from the MMRF CoMMpass and GSE24080 datasets. Notably, the LCE subpopulation exhibited gene expression profile featuring prominent LAMP5-overexpression, which was associated with risk for osteolytic bone lesions. Ex vivo drug sensitivity testing displayed differential sensitivity of the subpopulations. Copy Number Variant inference showed that the transcriptomic subpopulation underlying LCE was related to a genetic subclone that evolved over time. Our findings illustrate that malignant subpopulations underly light chain escape in multiple myeloma. These studies implied that LCE and LAMP5 gene overexpression portends for increased risk of osteolytic bone disease and adverse prognosis, findings that were confirmed in the subset of patients in the CoMMpass database with light chain escape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e33173187a2b1d3d0f9422813ed4bc1d3b6c7d8" target='_blank'>
              Single Cell RNA Sequencing Before and After Light Chain Escape Reveals Intra-patient Multiple Myeloma Subpopulations with Divergent Osteolytic Gene Expression.
              </a>
            </td>
          <td>
            Denis Ohlstrom, Z. Walker, Abhishek Pandey, Lorraine N Davis, Krysta Engel, Zenggang Pan, Peter A Forsberg, T. Mark, Austin E. Gillen, Daniel W. Sherbenou
          </td>
          <td>2024-12-19</td>
          <td>Cancer research communications</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ceb40aeff67c4162245cfc687f8adca56b59144a" target='_blank'>
              Integrated molecular and functional characterization of the intrinsic apoptotic machinery identifies therapeutic vulnerabilities in glioma
              </a>
            </td>
          <td>
            E. Fernández, Wilson X Mai, Kai Song, Nicholas A. Bayley, Jiyoon Kim, Henan Zhu, Marissa S Pioso, Pauline Young, Cassidy Andrasz, Dimitri Cadet, Linda M. Liau, Gang Li, William Yong, Fausto J Rodriguez, Scott J. Dixon, Andrew J Souers, Jingyi Jessica Li, Thomas Graeber, T. Cloughesy, David A. Nathanson
          </td>
          <td>2024-11-21</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="When a species invades a novel environment, it must bridge the environment-phenotype mismatch in its new range to persist. Contemporary invasion biology research has focused on the role that trait variation and adaptation, and their underlying genomic factors, play in a species' adaptive potential, and thus facilitating invasion. Empirical studies have provided valuable insights into phenotypes that persist and arise in novel environments, coupled with 'omics tools that further the understanding of the contributions of genomic architecture in species establishment. Particularly, the use of transcriptomics to explore the role of plasticity in the initial stages of an invasion is growing. Here, we assess the role of various mechanisms relating to regulation and functional adaptation (often measured via the transcriptome) that support trait-specific plasticity in invasive species, allowing phenotypic variability without directly altering genomic diversity. First, we present a comprehensive review of the studies utilising transcriptomics in invasion biology. Second, we collate the evidence for and against the role of a range of regulatory processes in contributing to invasive species plasticity. Finally, we pose open questions in invasion biology where the use of transcriptome data may be valuable, as well as discuss the methodological limitations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f9052d1e86599a8758cdb0f2f25e6c38b46b7d5" target='_blank'>
              Can Transcriptomics Elucidate the Role of Regulation in Invasion Success?
              </a>
            </td>
          <td>
            Amy L Vaughan, M. Dhami
          </td>
          <td>2024-11-15</td>
          <td>Molecular ecology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Patients with epidermal growth factor receptor mutant nonsmall cell lung cancer (NSCLC) often fail to treat gefitinib because of secondary drug resistance. The development of tumor drug resistance is closely related to variations in cancer cell metabolism. Single-cell metabolomics analysis can provide unique information about tumor drug resistance. Herein, we constructed a platform to study the secondary resistance of tumor cells based on single-cell metabolomics (sSRTC-scM). A gefitinib-resistant NSCLC cell line (PC9GR) was constructed by increasing the dose step by step. The metabolic profiles of parental PC9 cells and PC9GR cells with different drug resistance levels were detected by intact living-cell electrolaunching ionization mass spectrometry at the single-cell level. The data were analyzed by statistical methods such as t-SNE, variance, volcano plot, heat map, and metabolic pathway analysis. Using this platform, we found that the metabolic fingerprints of PC9GR cells can evaluate drug resistance degrees. The metabolic fingerprints continue to be altered with the increase of drug resistance. We revealed 19 metabolic markers of secondary resistance by variance analysis and clarified that the glycerophospholipid metabolic pathway of PC9GR cells changed significantly. In addition, we found that with the increase in drug resistance levels, the heterogeneity of single-cell metabolism became greater and the number of cells with weak drug resistance gradually decreased. This phenomenon can be utilized to illustrate the drug resistance degrees of PC9GR cells. This study provides diagnostic markers for evaluating the drug resistance of tumors and gives new insight into overcoming the secondary resistance of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84899e62659612e5b6164a21695dadcc8005bf4e" target='_blank'>
              Single-Cell Mass Spectrometry Studies of Secondary Drug Resistance of Tumor Cells.
              </a>
            </td>
          <td>
            Guizhen Zhu, Wenmei Zhang, Yaoyao Zhao, Guangyun Wang, Hanyu Yuan, Guangsheng Guo, Xiayan Wang
          </td>
          <td>2024-12-20</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e88d1d3e327b2658624144e333e3ed0c37f8a284" target='_blank'>
              CRISPR/Cas9-induced double-strand breaks in the huntingtin locus lead to CAG repeat contraction through DNA end resection and homology-mediated repair
              </a>
            </td>
          <td>
            P. Śledziński, Mateusz Nowaczyk, Marianna Śmiełowska, M. Olejniczak
          </td>
          <td>2024-12-03</td>
          <td>BMC Biology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Background: Most tumors initially respond to treatment, yet refractory clones subsequently develop owing to resistance mechanisms associated with cancer cell plasticity and heterogeneity. Methods: We used a chemical biology approach to identify protein targets in cancer cells exhibiting diverse driver mutations and representing models of tumor lineage plasticity and therapy resistance. An unbiased screen of a drug library was performed against cancer cells followed by synthesis of chemical analogs of the most effective drug. The cancer subtype target range of the leading drug was determined by PRISM analysis of over 900 cancer cell lines at the Broad Institute, MA. RNA-sequencing and enrichment analysis of differentially expressed genes, as well as computational molecular modeling and pull-down with biotinylated small molecules were used to identify and validate RPS6KB1 (p70S6K or S6K1) as an essential target. Genetic restoration was used to test the functional role of S6K1 in cell culture and xenograft models. Results: We identified a novel derivative of the antihistamine drug ebastine, designated Super-ebastine (Super-EBS), that inhibited the viability of cancer cells representing diverse KRAS and EGFR driver mutations and models of plasticity and treatment resistance. Interestingly, PRISM analysis indicated that over 95% of the diverse cancer cell lines tested were sensitive to Super-EBS and the predicted target was the serine/threonine kinase S6K1. S6K1 is upregulated in various cancers relative to counterpart normal/benign tissues and phosphorylated-S6K1 predicts poor prognosis for cancer patients. We noted that inhibition of S6K1 phosphorylation was necessary for tumor cell growth inhibition, and restoration of phospho-S6K1 rendered tumor cells resistant to Super-EBS. Inhibition of S6K1 phosphorylation by Super-EBS induced caspase-2 dependent apoptosis via inhibition of the Cdc42/Rac-1/p-PAK1 pathway that led to actin depolymerization and caspase-2 activation. The essential role of S6K1 in the action of Super-EBS was recapitulated in xenografts, and knockout of S6K1 abrogated tumor growth in mice. Conclusion: S6K1 is a therapeutic vulnerability in tumors exhibiting intrinsic and/or acquired resistance to treatment">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e4047ca455e4b4317a09e2a2f99cd529981915d" target='_blank'>
              S6K1 is a Targetable Vulnerability in Tumors Exhibiting Plasticity and Therapy Resistance
              </a>
            </td>
          <td>
            Saptadwipa Ganguly, R. Burikhanov, Vitaliy M. Sviripa, Sally Ellingson, Jieyun Jiang, Christian M. Gosser, David Orren, Eva M. Goellner, G.G. Shenoy, Mahadev Rao, John A. D’Orazio, C. F. Brainson, Chang-Guo Zhan, Peter H. Spielmann, David S. Watt, Vivek M. Rangnekar
          </td>
          <td>2025-01-01</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="The MCF-7R breast cancer cell line, developed by treating the parental MCF-7 cells with increasing doses of doxorubicin, serves as a model for studying acquired multidrug resistance (MDR). MDR is a major challenge in cancer therapy, often driven by overexpression of the efflux pump P-glycoprotein (P-gp) and epigenetic modifications. While many P-gp inhibitors show promise in vitro, their nonspecific effects on the efflux pump limit in vivo application. Curcumin, a natural compound with pleiotropic action, is a nontoxic P-gp inhibitor capable of modulating multiple pathways. To explore curcumin’s molecular effects on MCF-7R cells, we analyzed the expression of genes involved in DNA methylation and transcription regulation, including ABCB1/MDR1. Reduced representation bisulfite sequencing further unveiled key epigenetic changes induced by curcumin. Our findings indicate that curcumin treatment not only modulates critical cellular processes, such as ribosome biogenesis and cytoskeletal dynamics, but also reverses the resistant phenotype, toward that of sensitive cells. This study highlights curcumin’s potential as an adjuvant therapy to overcome chemoresistance, offering new avenues for pharmacological strategies targeting epigenetic regulation to re-sensitize resistant cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a25fe3bb8a31c48e3c18c599af20c640c2363916" target='_blank'>
              Epigenetic and Cellular Reprogramming of Doxorubicin-Resistant MCF-7 Cells Treated with Curcumin
              </a>
            </td>
          <td>
            P. Poma, S. Rigogliuso, M. Labbozzetta, Aldo Nicosia, Salvatore Costa, M. Ragusa, M. Notarbartolo
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Ionizing radiotherapy (RT) is a widely used palliative and curative treatment strategy for malignancies. In solid tumors, RT-induced double strand breaks lead to the accumulation of indels, and their repair by non-homologous end-joining has been linked to the ID8 mutational signature in resistant cells. However, the extent of RT-induced DNA damage in hematologic malignancies and its impact on their evolution and interplay with commonly used chemotherapies has not yet been explored. Here, we interrogated 580 whole genome sequencing (WGS) from patients with large B-cell lymphoma, multiple myeloma, and myeloid neoplasms and identified ID8 only in relapsed disease. Yet, it was detected after exposure to both RT and mutagenic chemotherapy (i.e., platinum). Using WGS of single-cell colonies derived from treated lymphoma cells, we revealed a dose-response relationship between RT and platinum and ID8. Finally, using ID8 as a genomic barcode we demonstrate that a single RT-resistant cell may seed systemic relapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7be6715a8b86cbf647052fda12a7d5d6ddae614" target='_blank'>
              Mutagenic impact and evolutionary influence of radiotherapy in hematologic malignancies
              </a>
            </td>
          <td>
            B. Diamond, Dhanvantri Chahar, M. Jain, A. Poos, Michael Durante, B. Ziccheddu, Marcella Kaddoura, M. Papadimitriou, K. Maclachlan, T. Jelínek, Faith Davies, N. Figura, Gareth Morgan, Elias K Mai, Katja C. Weisel, Roland Fenk, Marc-Steffen Raab, S. Usmani, O. Landgren, Frederick L. Locke, Hartmut Goldschmidt, Jonathan H. Schatz, Niels Weinhold, F. Maura
          </td>
          <td>2024-11-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Therapy resistance still constitutes a common hurdle in the treatment of many human cancers and is a major reason for treatment failure and patient relapse, concomitantly with a dismal prognosis. In addition to “intrinsic resistance”, e.g., acquired by random mutations, cancer cells typically escape from certain treatments (“acquired resistance”) by a large variety of means, including suppression of apoptosis and other cell death pathways via upregulation of anti-apoptotic factors or through inhibition of tumor-suppressive proteins. Therefore, ideally, the tumor-cell-restricted induction of apoptosis is still considered a promising avenue for the development of novel, tumor (re)sensitizing therapies. A growing body of evidence has highlighted the multifaceted role of tripartite motif 25 (TRIM25) in controlling different aspects of tumorigenesis, including chemotherapeutic drug resistance. Accordingly, overexpression of TRIM25 is observed in many tumors and frequently correlates with a poor patient survival. In addition to its originally described function in antiviral innate immune response, TRIM25 can play critical yet context-dependent roles in apoptotic- and non-apoptotic-regulated cell death pathways, including pyroposis, necroptosis, ferroptosis, and autophagy. The review summarizes current knowledge of molecular mechanisms by which TRIM25 can interfere with different cell death modalities and thereby affect the success of currently used chemotherapeutics. A better understanding of the complex repertoire of cell death modulatory effects by TRIM25 is an essential prerequisite for validating TRIM25 as a potential target for future anticancer therapy to surmount the high failure rate of currently used chemotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9eb1e40b262aff1ec5083769b832df042944b012" target='_blank'>
              TRIM25: A Global Player of Cell Death Pathways and Promising Target of Tumor-Sensitizing Therapies
              </a>
            </td>
          <td>
            W. Eberhardt, Usman Nasrullah, Josef Pfeilschifter
          </td>
          <td>2025-01-07</td>
          <td>Cells</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Abstract. Lung cancer is the most aggressive and prevalent malignancies worldwide, contributing significantly to global health challenges with over 1.8 million new cases reported annually. Non-small cell lung carcinoma (NSCLC) accounts for the majority of these cases and is characterized by a poor 5-year survival rate of only 10%. Traditional therapies, including surgery, chemotherapy, and radiotherapy, face limitations such as drug resistance and systemic toxicity. The advent of Clustered regularly interspaced short palindromic repeats (CRISPR)/CRISPR-associated (Cas) systems (CRISPR/Cas) technology has transformed the landscape of gene editing, enabling precise targeting of genes involved in NSCLC pathogenesis. This review explores the therapeutic potential of this technology in NSCLC, focusing on key genes such as AIFM1, ROCK2, TSPAN4, and the PD-1 receptor protein, which play crucial roles in tumor progression and immune evasion. By elucidating the mechanisms through which the genes above help with lung cancer, we highlight how CRISPR/Cas9-mediated interventions can pave the way for innovative and effective treatment strategies. Additionally, we address the current challenges and prospects of implementing CRISPR/Cas9 in clinical settings to enhance the efficacy and specificity of lung cancer therapies, in order to improve patient outcomes and reduce the global burden of this devastating disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f5695fa779ae7c0481e84826ad52efb38a86c56" target='_blank'>
              Advancements of CRISPR/Cas9 technology in non-small cell lung cancer research: Applications and current developments
              </a>
            </td>
          <td>
            Kongzhao Chen, Jingyi Chen
          </td>
          <td>2024-11-15</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6ef6a160eddfbbad312082d26af8e03b167d27f" target='_blank'>
              Polyploid cancer cells reveal signatures of chemotherapy resistance.
              </a>
            </td>
          <td>
            Michael J Schmidt, Amin Naghdloo, Rishvanth K. Prabakar, Mohamed Kamal, Radu M. Cadaneanu, I. Garraway, Michael Lewis, Ana Aparicio, A. Zurita-Saavedra, Paul Corn, Peter Kuhn, Kenneth J Pienta, Sarah R. Amend, James B. Hicks
          </td>
          <td>2024-11-22</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96442cd68e41fa673e831da90c37e37f248ca02b" target='_blank'>
              A multiomic atlas identifies a treatment-resistant, bone marrow progenitor-like cell population in T cell acute lymphoblastic leukemia.
              </a>
            </td>
          <td>
            Jason Xu, Changya Chen, Jonathan H Sussman, S. Yoshimura, T. Vincent, P. Pölönen, Jianzhong Hu, S. Bandyopadhyay, Omar Elghawy, Wenbao Yu, Joseph S. Tumulty, Chia-Hui Chen, Elizabeth Li, Caroline Diorio, R. Shraim, H. Newman, Lahari Uppuluri, A. Li, Gregory M. Chen, David W Wu, Yang-yang Ding, Jessica Xu, Damjan Karanfilovski, Tristan Lim, Miles Hsu, Anusha Thadi, Kyung Jin Ahn, Chi-Yun Wu, Jacqueline Peng, Yusha Sun, Alice Wang, Rushabh Mehta, David Frank, Lauren Meyer, M. Loh, E. Raetz, Zhiguo Chen, Brent L Wood, M. Devidas, K. Dunsmore, S. Winter, Ti-Cheng Chang, Gang Wu, S. Pounds, Nancy R. Zhang, William L Carroll, S P Hunger, Kathrin M Bernt, Jun J Yang, C. Mullighan, Kai Tan, D. Teachey
          </td>
          <td>2024-11-25</td>
          <td>Nature cancer</td>
          <td>1</td>
          <td>103</td>
        </tr>

        <tr id="Resistance to BRAF and MAPK inhibitors is a significant challenge in melanoma treatment, driven by adaptive and acquired mechanisms that allow tumour cells to evade therapy. Here, we examined early signalling responses to single and combined BRAF and MAPK inhibition in a BRAFV600E, drug-sensitive melanoma cell line and a drug-resistant ARID1A-knockout (KO) derivative. ARID1A, frequently mutated in melanoma, is associated with resistance and immune evasion. Using an innovative systems biology approach that integrates transcriptomics, proteomics, phosphoproteomics, and functional kinomics through matrix factorization and network analysis, we identified key signalling alterations and resistance mechanisms. We found that ARID1A-KO cells exhibited transcriptional rewiring, sustaining MAPK1/3 and JNK activity post-treatment, bypassing feedback sensitivity observed in parental cells. This rewiring suppressed PRKD1 activation, increased JUN activity—a central resistance network node—and disrupted PKC dynamics through elevated basal RTKs (e.g., EGFR, ROS1) and Ephrin receptor activity post-treatment. ARID1A mutations also reduced HLA-related protein expression and enriched extracellular matrix components, potentially limiting immune infiltration and reducing immunotherapy efficacy. Our graph-theoretical multi-omics approach uncovered novel resistance-associated signalling pathways, identifying PRKD1, JUN, and NCK1 as critical nodes. While receptor activation redundancies complicate single-target therapies, they also present opportunities for combination strategies. This study highlights ARID1A’s role in reshaping signalling and immune interactions, offering new insights into melanoma resistance mechanisms. By identifying actionable targets, including JUN and immune pathways, we provide a foundation for developing integrated therapeutic strategies to overcome resistance in BRAF/MAPK inhibitor-treated melanoma. One sentence summary This study reveals how ARID1A-mediated transcriptional rewiring drives resistance to MAPK inhibitors in melanoma by altering signalling pathways, immune interactions, and receptor dynamics, highlighting potential targets for combinatorial therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6bdb990bb1257653b7e587028f560dd7ef66b129" target='_blank'>
              ARID1A-induced transcriptional reprogramming rewires signalling responses to drug treatment in melanoma
              </a>
            </td>
          <td>
            Charlie George Barker, Sumana Sharma, Ana Mafalda Santos, Konstantinos-Stylianos Nikolakopoulos, Athanassios D. Velentzas, Franziska Völlmy, A. Minia, V. Pliaka, M. Altelaar, Gavin J Wright, Leonidas G. Alexopoulos, D. Stravopodis, E. Petsalaki
          </td>
          <td>2024-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Gene editing has emerged as a transformative approach in cancer immunotherapy. Several gene editing tools have been employed for precise modification of the DNA of immune cells, enhancing their ability to target and eliminate cancer cells. This review examines the evolution and applications of key gene-editing tools, such as CRISPR-Cas9, TALENs, ZFNs, and recent innovations like base and prime editing in the field of cancer immunotherapy. 
Promising results have been observed in therapies such as CAR-T and tumor-infiltrating lymphocyte (TIL) treatments, which have shown success in cancers like leukemia and lymphoma. These technologies improve immune system function by disrupting checkpoints, boosting cytokine production, and modifying tumor microenvironments. 
Significant clinical trials have demonstrated promising outcomes, such as CRISPR-Cas9-engineered T cells targeting refractory cancers, which showed improved efficacy and safety. However, despite these advancements, there have been limitations, including off-target effects, delivery inefficiencies, immunogenicity, and ethical concerns, alongside the high costs that hinder widespread adoption. 
Future directions for gene editing in cancer therapy include the integration of AI and machine learning to enhance target accuracy and guide RNA design, as well as novel gene-editing systems such as the I-C Cascade-Cas3, which facilitates large-scale genomic deletions. Furthermore, CRISPR-based epigenome editing holds promise for further advancing cancer therapies. These innovations, combined with optimized delivery methods, are expected to improve the precision, efficacy, and accessibility of gene editing in cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c53d457951919724a8c5ed4f27b1693452c6fc64" target='_blank'>
              Gene Editing in Cancer Immunotherapy: Mechanisms, Advancements, Limitations and Future Directions
              </a>
            </td>
          <td>
            Lawrence John Ajutor, Okabeonye Sunday Agbo, Christian Ajiri Adobor, Blessing Aitebiereme Iyoyojie, Kingsley Ugonna Ugoagwu, Oghenevware Manawa, Osaruese Faluyi, Emmanuel Oluwasegun Ismaila, Oluchi Esther Ugwuanyi, Adepeju Matilda Adekoya
          </td>
          <td>2024-12-21</td>
          <td>Asian Journal of Biochemistry, Genetics and Molecular Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d907c1f70a7e162db16222c621c3aec6a63202a" target='_blank'>
              Integrated single-cell analysis reveals heterogeneity and therapeutic insights in osteosarcoma
              </a>
            </td>
          <td>
            Dongan He, Xiaoqian Che, Haiming Zhang, Jiandong Guo, Lei Cai, Jian Li, Jinxi Zhang, Xin Jin, Jianfeng Wang
          </td>
          <td>2024-11-18</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The orchestration of DNA repair is of fundamental importance to the maintenance of genomic integrity and tumor suppression. DNA damage must be detected in the context of the varied chromatin landscape, its presence must be communicated throughout the cell to alter many ongoing processes, and the machinery that will mend the lesion must be recruited to the damage site. In my presentation, I will discuss our recent efforts in mapping genome maintenance pathways using genome-scale CRISPR/Cas9 screens in human cells. I will highlight how these screens can be used to identify new genome stability factors, characterize drug responses and provide new insights into the genetic architecture of the genome stability network. I will finally present how these screens can also identify potentially actionable synthetic lethal genetic interactions that could form the basis of new oncology drug discovery efforts.
 Citation Format: Daniel Durocher. Synthetic lethality: Pathways to therapeutic discovery. [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Optimizing Therapeutic Efficacy and Tolerability through Cancer Chemistry; 2024 Dec 9-11; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(12_Suppl):Abstract nr IA003">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c5b7589059a6134690d03df4a91c9f05b4284ea" target='_blank'>
              Abstract IA003: Synthetic lethality: Pathways to therapeutic discovery
              </a>
            </td>
          <td>
            Daniel Durocher
          </td>
          <td>2024-12-09</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/033d9d641f5cc9ef741b9f061b7709ec5c750d88" target='_blank'>
              Characterizing the evolutionary dynamics of cancer proliferation in single-cell clones with SPRINTER.
              </a>
            </td>
          <td>
            Olivia Lucas, S. Ward, Rija Zaidi, Abigail Bunkum, A. Frankell, David A Moore, M. Hill, Wing Kin Liu, Daniele Marinelli, E. Lim, S. Hessey, C. Naceur-Lombardelli, A. Rowan, Sukhveer Kaur Purewal-Mann, Haoran Zhai, M. Dietzen, Boyue Ding, G. Royle, Sam Aparicio, N. Mcgranahan, M. Jamal-Hanjani, N. Kanu, C. Swanton, S. Zaccaria
          </td>
          <td>2024-11-29</td>
          <td>Nature genetics</td>
          <td>1</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a602005403e19b0fe4fd439cfa9748d3ee799d93" target='_blank'>
              Identification of cancer cell-intrinsic biomarkers associated with tumor progression and characterization of SFTA3 as a tumor suppressor in lung adenocarcinomas
              </a>
            </td>
          <td>
            Yu Zhao, Chengcheng Zhou, Ling Zuo, Haoming Yan, Yuhan Gu, Hong Liu, Guiping Yu, Xiaorong Zhou
          </td>
          <td>2025-01-08</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Multiple myeloma (MM) is a hematological malignancy characterized by the clonal proliferation of malignant plasma cells within the bone marrow. The disease’s complexity is underpinned by a variety of genetic and molecular abnormalities that drive its progression. Methods This review was conducted through a state-of-The-art literature search, primarily utilizing PubMed to gather peer-reviewed articles. We focused on the most comprehensive and cited studies to ensure a thorough understanding of the genetic and molecular landscapes of MM. Results We detail primary and secondary alterations such as translocations, hyperdiploidy, single nucleotide variants (SNVs), copy number alterations (CNAs), gene fusions, epigenetic modifications, non-coding RNAs, germline predisposing variants, and the influence of the tumor microenvironment (TME). Our analysis highlights the heterogeneity of MM and the challenges it poses in treatment and prognosis, emphasizing the distinction between driver mutations, which actively contribute to oncogenesis, and passenger mutations, which arise due to genomic instability and do not contribute to disease progression. Conclusion & Future Perspectives We report key controversies and challenges in defining the genetic drivers of MM, and examine their implications for future therapeutic strategies. We discuss the importance of systems biology approaches in understanding the dependencies and interactions among these alterations, particularly highlighting the impact of double and triple-hit scenarios on disease outcomes. By advancing our understanding of the molecular drivers and their interactions, this review sets the stage for novel therapeutic targets and strategies, ultimately aiming to improve clinical outcomes in MM patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec433a1270b41378c5571d49631b7d920ddc502b" target='_blank'>
              The Genetic and Molecular Drivers of Multiple Myeloma: Current Insights, Clinical Implications, and the Path Forward
              </a>
            </td>
          <td>
            Meghana Ram, Molly Fraser, Júnia Vieira dos Santos, R. Tasakis, Ariana Islam, Jannah Abo-Donia, Samir Parekh, A. Laganà
          </td>
          <td>2024-12-01</td>
          <td>Pharmacogenomics and Personalized Medicine</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="In this review, we explore the application of next-generation sequencing in liver cancer research, highlighting its potential in modern oncology. Liver cancer, particularly hepatocellular carcinoma, is driven by a complex interplay of genetic, epigenetic, and environmental factors. Key genetic alterations, such as mutations in TERT , TP53 , and CTNNB1 , alongside epigenetic modifications such as DNA methylation and histone remodeling, disrupt regulatory pathways and promote tumorigenesis. Environmental factors, including viral infections, alcohol consumption, and metabolic disorders such as nonalcoholic fatty liver disease, enhance hepatocarcinogenesis. The tumor microenvironment plays a pivotal role in liver cancer progression and therapy resistance, with immune cell infiltration, fibrosis, and angiogenesis supporting cancer cell survival. Advances in immune checkpoint inhibitors and chimeric antigen receptor T-cell therapies have shown potential, but the unique immunosuppressive milieu in liver cancer presents challenges. Dysregulation in pathways such as Wnt/β-catenin underscores the need for targeted therapeutic strategies. Next-generation sequencing is accelerating the identification of genetic and epigenetic alterations, enabling more precise diagnosis and personalized treatment plans. A deeper understanding of these molecular mechanisms is essential for advancing early detection and developing effective therapies against liver cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf941fd49f418f7109e7e4bbca5152a8ddaa54e7" target='_blank'>
              Advances and challenges in molecular understanding, early detection, and targeted treatment of liver cancer
              </a>
            </td>
          <td>
            Ji Shi, Xu Zhu, Jun-Bo Yang
          </td>
          <td>2025-01-27</td>
          <td>World Journal of Hepatology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Determining the phenotypic effects of single nucleotide variants is critical for understanding the genome and interpreting clinical sequencing results. Base editors, including diversifying base editors that create C>N mutations, are potent tools for installing point mutations in mammalian genomes and studying their effect on cellular function. Numerous base editor options are available for such studies, but little information exists on how the composition of the editor (deaminase, recruitment method, and fusion architecture) affects editing. To address this knowledge gap, the effect of various design features, such as deaminase recruitment and delivery method (electroporation or lentiviral transduction), on editing was assessed across ∼200 synthetic target sites. The direct fusion of a hyperactive variant of activation-induced cytidine deaminase to the N-terminus of dCas9 (DivA-BE) produced the highest editing efficiency, ∼4-fold better than the previous CRISPR-X method. Additionally, DivA-BE mutagenized the DNA strand that anneals to the targeting sgRNA to create G>N mutations, which were absent when the deaminase was fused to the C-terminus of dCas9. The DivA-BE editors efficiently diversified their target sites, an ideal characteristic for discovering functional variants. These and other findings provide a comprehensive analysis of how design features influence the activity of several popular base editors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4879f4371c59083e5df3495887b2d3ce24001052" target='_blank'>
              Towards optimizing diversifying base editors for high-throughput studies of single- nucleotide variants
              </a>
            </td>
          <td>
            Carley I Schwartz, Nathan S. Abell, Amy Li, Aradhana, Josh Tycko, Alisa Truong, Stephen B. Montgomery, Gaelen T. Hess
          </td>
          <td>2024-11-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="RNA-based genetic therapies have emerged as a promising approach in the treatment of rare genetic diseases, offering a novel alternative to traditional therapeutic strategies. This study reviews the current literature on the effectiveness of RNA-based therapies in treating genetic disorders, focusing on advancements in mRNA, siRNA, and antisense oligonucleotides (ASOs) therapies. A qualitative, literature-based methodology was employed to analyze recent research articles, clinical trials, and case studies from the past five years. Key findings highlight the potential of RNA-based therapies to target the root causes of genetic diseases at the molecular level, such as correcting mutations, silencing defective genes, or replacing missing proteins. RNA therapies have shown particular promise in treating rare diseases like Duchenne muscular dystrophy, spinal muscular atrophy, and certain forms of inherited retinal diseases, with several therapies already reaching clinical approval. However, challenges such as delivery mechanisms, immune response, and long-term efficacy remain significant barriers to widespread adoption. This review emphasizes the need for further research to refine RNA delivery systems, assess long-term safety, and optimize therapeutic outcomes. The study concludes by suggesting that RNA-based therapies could revolutionize the treatment of rare genetic diseases, provided these challenges are addressed through continued innovation and clinical validation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4318feb40e0dfdfce825e0cca992e00b2fd570d3" target='_blank'>
              Effectiveness of RNA-Based Genetic Therapy in Treating Rare Diseases of a Genetic Nature
              </a>
            </td>
          <td>
            Suherman Jaksa, Anna Maria Nurhajati Widjaja, Neng Mira Atjo
          </td>
          <td>2024-12-05</td>
          <td>The Journal of Academic Science</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/557911aa9625d7bd95b52abd71f3273a6c915b9d" target='_blank'>
              Transcription and DNA replication collisions lead to large tandem duplications and expose targetable therapeutic vulnerabilities in cancer.
              </a>
            </td>
          <td>
            Yang Yang, Michelle L Badura, Patrick C O'Leary, Henry Delavan, Troy M Robinson, Emily A Egusa, Xiaoming Zhong, Jason Swinderman, Haolong Li, Meng Zhang, Minkyu Kim, Alan Ashworth, Felix Y. Feng, Jonathan Chou, Lixing Yang
          </td>
          <td>2024-11-18</td>
          <td>Nature cancer</td>
          <td>1</td>
          <td>8</td>
        </tr>

        <tr id="Population-scale resources of genetic, molecular, and cellular information form the basis for understanding human genomes, charting the heritable basis of disease, and tracing the effects of mutations. Pooled perturbation assays applied to cellular models, probing the effect of many perturbations coupled with an scRNA-seq readout (Perturb-seq), are especially potent references for interpreting disease-linked mutations or gene expression changes. However, the utility of existing maps has been limited by the comprehensiveness of perturbations possible, and the relevance of their cell line context. Here, we present the first genome-scale CRISPR interference (CRISPRi) perturbation map with single-cell RNA sequencing readout across many human genetic backgrounds in human pluripotent cells. To do so, we establish large-scale CRISPRi screening in human induced pluripotent stem cells from healthy donors, using over 20,000 guide RNAs to target 7,226 genes across 34 cell lines from 26 genetic backgrounds, and gather expression data from nearly 2 million cells. We comprehensively map trans expression changes induced by the target knockdowns, which complement co-expression patterns in unperturbed cells and facilitate the functional annotation of target genes to biological processes and complexes. Consistency of targeting protein complex members point to protein complexes as a nexus for aggregating transcriptional variation, revealing novel interaction partners. We characterise variation in perturbation effects across donors, with expression quantitative trait loci linked to higher genetic modulation of perturbation effects but overall low replication of trans effects due to knockdown of their corresponding cis regulators. This study pioneers population-scale CRISPR perturbations with single cell readouts that will fuel foundation models for the future of effective modulation of cellular disease phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5800ff9ee76e49e133d6ead67bfba5c5d87d6239" target='_blank'>
              A genome-scale single cell CRISPRi map of trans gene regulation across human pluripotent stem cell lines
              </a>
            </td>
          <td>
            Claudia Feng, Elin Madli Peets, Yan Zhou, Luca Crepaldi, S. Usluer, Alistair S. Dunham, Jana M. Braunger, Jing Su, Magdalena E Strauss, Daniele Muraro, Kimberly Ai Xian Cheam, M. Bonder, Edgar Garriga Nogales, Sarah Cooper, Andrew Bassett, Steven Leonard, Yong Gu, Bo Fussing, David Burke, Leopold Parts, O. Stegle, B. Velten
          </td>
          <td>2024-11-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>84</td>
        </tr>

        <tr id="Tumor heterogeneity is the substrate for tumor evolution and the linchpin of treatment resistance. Cancer cell heterogeneity is largely attributed to distinct genetic changes within each cell population. However, the widespread epigenome repatterning that characterizes most cancers is also highly heterogenous within tumors and could generate cells with diverse identities and malignant features. We show that high levels of the epigenetic regulator and oncogene, UHRF1, in zebrafish hepatocytes rapidly induced methylome disordering, loss of heterochromatin, and DNA damage, resulting in cell cycle arrest, senescence, and acquisition of stemness. Reducing UHRF1 expression transitions these cells from senescent to proliferation-competent. The expansion of these damaged cells results in hepatocellular carcinomas (HCC) that have immature cancer cells intermingled with fibroblasts, immune and senescent cells expressing high UHRF1 levels, which serve as reservoirs for new cancer cells. This defines a distinct and heterogenous HCC subtype resulting from epigenetic changes, stemness and senescence escape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2a7fc1c9934592d995c7390a3ec0bd3e0344392" target='_blank'>
              Epigenetic Disordering Drives Stemness, Senescence Escape and Tumor Heterogeneity
              </a>
            </td>
          <td>
            Elena Magnani, F. Macchi, Tijana Randic, Charlene Chen, Bhavani P. Madakashira, Shashi Ranjan, Sema Elif Eski, S. P. Singh, Kirsten C. Sadler
          </td>
          <td>2024-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0297441e9d4a24ced84d1909df04d7019094986" target='_blank'>
              Functional screening reveals genetic dependencies and diverging cell cycle control in atypical teratoid rhabdoid tumors
              </a>
            </td>
          <td>
            Daniel J. Merk, F. Tsiami, S. Hirsch, B. Walter, L. Haeusser, Jens D. Maile, Aaron Stahl, M. Jarboui, Anna Lechado-Terradas, Franziska Klose, S. Babaei, Jakob Admard, N. Casadei, C. Roggia, Michael Spohn, J. Schittenhelm, Stephan Singer, U. Schüller, F. Piccioni, N. Persky, Manfred Claassen, M. Tatagiba, Philipp J. Kahle, D. Root, M. Templin, Ghazaleh Tabatabai
          </td>
          <td>2024-12-02</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02c600bc41b490f1030b42efdd32168810e5ead4" target='_blank'>
              The dynamics of loss of heterozygosity events in genomes.
              </a>
            </td>
          <td>
            Abhishek Dutta, J. Schacherer
          </td>
          <td>2025-01-02</td>
          <td>EMBO reports</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="
 Cancer research has historically focused on the Somatic Mutation Theory (SMT), viewing cancer as a consequence of genetic mutations. However, this perspective has limitations in explaining phenomena like tumor reversion and cancer heterogeneity. This paper introduces an alternative approach: viewing cancer as a complex information processing system shaped by its microenvironment. By integrating historical data on tumor reversion and insights into evolutionary dynamics, I propose a reframing of cancer biology. This process-oriented perspective highlights the role of cellular plasticity and adaptive behaviors, offering new pathways for therapeutic development">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d33c3cc92fad268ef72893a64fd8d179664753cb" target='_blank'>
              Rethinking Cancer Evolution: From Genetic Mutations to Complex Information Systems in Tumor Reversion
              </a>
            </td>
          <td>
            M. Tez
          </td>
          <td>2024-12-13</td>
          <td>Evolution, Medicine, and Public Health</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Plants host a range of DNA elements capable of self-replication. These molecules, usually associated to the activity of transposable elements or viruses, are found integrated in the genome or in the form of extrachromosomal DNA. The activity of these elements can impact genome plasticity by a variety of mechanisms, including the generation of structural variants, the shuffling of regulatory or coding DNA sequences across the genome, and DNA endoduplication. This plasticity can dynamically alter gene expression and genome stability, ultimately affecting plant development or the response to environmental changes. While the activation of these elements is often considered deleterious to the genome, their role in creating variation is important in adaptation and evolution. Moreover, the mechanisms by which mobile DNA proliferate have been exploited for plant engineering, or contributed to understand how desirable traits can be generated in crops. In this review, we discuss the origins and the roles of mobile DNA element activity on genome plasticity and plant biology, as well as their potential function and current application in plant biotechnology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b762e6df52387f3493832fc69614173777627231" target='_blank'>
              The role of mobile DNA elements in the dynamic of plants genome plasticity.
              </a>
            </td>
          <td>
            Robyn Emmerson, M. Catoni
          </td>
          <td>2025-01-10</td>
          <td>Journal of experimental botany</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The study by Grossmann and colleagues uses single-nucleus RNA sequencing in a cohort of matched high-risk neuroblastoma primary tumor samples, obtained from the same patient at diagnosis and definitive surgery, to identify persister cells that survive induction chemotherapy. These persister cells utilize mechanisms of chemoresistance that are both tumor-intrinsic and tumor-extrinsic, are highly dependent on the original genetic profile of the tumor, and represent novel, patient-specific targets to precisely inhibit chemoresistance and disease recurrence. See related article by Grossmann et.al., p. 2387.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17960d164122a139c932c5adc28d8d07f55f0399" target='_blank'>
              Characterization of Persister Cells Provides Insights into Mechanisms of Therapy Resistance in Neuroblastoma.
              </a>
            </td>
          <td>
            Amber B. Wolf, C. P. Reynolds, Eveline Barbieri
          </td>
          <td>2024-12-02</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Aberrant epigenetic alterations are known to lead to cancer arising from normal non-cancerous cells. Unlike genetic mutations, these dysregulations can be reversible and are potential targets for anticancer drugs. Curcumin, a natural plant-derived compound, has been shown to have an anticancer effect via its influence on epigenetic regulation. However, the exact identity of those epigenetic changes, as well as their mechanisms of action, remain largely unknown. The goal of this study is to use a 3D network model to identify what histone codes are being modified by curcumin and hypothesize the enzymatic pathways through which those changes occur in breast cancer. We analyzed multi-omic data from the NIH LINCS program and identified two novel histone marks associated with curcumin that are influenced by twenty-three phosphoproteins involved in cell signaling which are profiled in this paper. These histone and phosphoprotein signatures can be used as potential biomarkers for future chromatin-based drug therapies in breast cancer. GitHub URL Source code is available at https://github.com/mollahlab/ECHC">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd80504373cad8ddc1f4f4900bd8a98fdd6b50ec" target='_blank'>
              Deciphering epigenetic influence of curcumin on histone signatures in breast cancer using 3D network
              </a>
            </td>
          <td>
            Tina Tang, Mikhail Y. Berezin, Shamim A. Mollah
          </td>
          <td>2024-11-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="To maintain genome stability, proliferating cells must enact a program of telomere maintenance. While most tumors maintain telomeres through the action of telomerase, a subset of tumors utilize a DNA-templated process termed Alternative Lengthening of Telomeres or ALT. ALT is associated with mutations in the ATRX/DAXX/H3.3 histone chaperone complex, which is responsible for deposition of non-replicative histone variant H3.3 at heterochromatic regions of the genome including telomeres. We wished to better understand the role DAXX plays in ALT suppression, and to determine which disease-associated DAXX mutations are unable to suppress ALT. To answer this question, we have leveraged the G292 cell line, in which ATRX is wild type but DAXX has undergone a fusion event with the non-canonical kinesin KIFC3. Restoration of wild-type DAXX in G292 localizes ATRX and abrogates ALT. Using this model system, we tested the ability of a panel of disease-associated DAXX missense variants to suppress ALT. Missense mutations in the ATRX binding domain, the histone binding domain, and the C-terminal SUMO interaction motif reduce the ability of DAXX to suppress ALT. Unexpectedly, we find that mutations in the DAXX histone binding domain lead to failure of ATRX localization. We conclude that a key function of DAXX in ALT suppression is the localization of ATRX to nuclear foci.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecce9b8b3920bbf0f0b14800927fbabcdf8182c8" target='_blank'>
              Cancer-associated DAXX mutations reveal a critical role for ATRX localization in ALT suppression
              </a>
            </td>
          <td>
            Sarah F. Clatterbuck Soper, Robert L. Walker, M. Pineda, Y. Zhu, James Dalgleish, Jasmine Wang, Paul S. Meltzer
          </td>
          <td>2024-11-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9638b372db45cf9d9c1035df13d83aeced5bf8e5" target='_blank'>
              Combination strategies with PARP inhibitors in BRCA-mutated triple-negative breast cancer: overcoming resistance mechanisms.
              </a>
            </td>
          <td>
            Aditi Jain, Alan Barge, Christopher N Parris
          </td>
          <td>2024-11-21</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Resistance to targeted therapies, particularly those inhibiting oncogenic tyrosine kinases (TKIs), presents a significant challenge in the treatment of advanced non-small cell lung cancer (NSCLC). Despite initial robust clinical responses, tumors often relapse due to the emergence of resistant cell populations. The origins of this resistance—whether it is pre-existing in a subset of tumor cells or arises de novo during treatment—remains a critical question in understanding and combating therapeutic failure. To investigate this, we developed in silico mathematical models that integrate both pre-existing and de novo resistance mechanisms within a framework that explicitly integrates the stromal niche. Our study focused on the evolutionary dynamics of NSCLC in response to therapy, using a hybrid agent-based spatial modeling framework (HAL) to simulate tumor behavior under treatment. These models were designed to simulate the complex interactions between resistant and sensitive tumor cells, mediated by the stroma, and to predict the timing of relapse based on these interactions. Our findings suggest that stroma-mediated interactions play a crucial role in the evolutionary dynamics of therapy responses, influencing the fitness and survival of both resistant and sensitive cells. Specifically, our models predict that pre-existing resistance is unlikely to be compatible with the long remission periods observed in a substantial subset of patients treated with advanced anaplastic lymphoma kinase (ALK) inhibitors. This conclusion was supported by the observed relapse times and volumetric tumor growth data from subsequent animal models. Moreover, our analyses highlight the importance of ecological factors within the tumor microenvironment in shaping the response to therapy. The spatial models we developed reveal that the stroma can foster positive interactions between resistant and sensitive cells, thereby accelerating the emergence of resistance. These insights underscore the need to consider the stromal niche when developing new therapeutic strategies and when interpreting the dynamics of drug resistance. In conclusion, our study provides new insights into the role of stroma in mediating resistance to TKIs in NSCLC. By combining mathematical modeling with experimental data, we have elucidated how stroma-mediated interactions between tumor cells can influence the evolution of resistance, challenging the assumption that pre-existing resistance alone drives relapses in TKI-treated patients. Our findings emphasize the need for adaptive therapeutic approaches that anticipate and disrupt these microenvironmental interactions to improve patient outcomes.
 Citation Format: Rishi M. Shah, Sagnik Yarlagadda, Mark Robertson-Tessi, Bina Desai, Tatiana Miti, Daria Miroshnychenko, David Basanta, Andriy Marusyk, Alexander Anderson. A novel stroma-mediated positive interaction between resistant and sensitive cells in non-small cell lung cancer facilitates drug resistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr A040.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f45a98f3cd8eb088bb6214ea6a11a3e5e391c9a" target='_blank'>
              Abstract A040: A novel stroma-mediated positive interaction between resistant and sensitive cells in non-small cell lung cancer facilitates drug resistance
              </a>
            </td>
          <td>
            Rishi Shah, Sagnik Yarlagadda, M. Robertson-Tessi, Bina Desai, Tatiana Miti, Daria Miroshnychenko, D. Basanta, A. Marusyk, Alexander R. A. Anderson
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="BACKGROUND
The majority of patients with prostate cancer (PCa) exhibit intrinsic resistance to immune checkpoint blockade (ICB) following radiotherapy (RT). This resistance is generally attributed to the limited antigen presentation of heterogeneous cells within tumors. Here, we aimed to isolate and characterize these diverse subgroups of tumor post-RT to understand the molecular mechanisms of their resistance to ICB.


METHODS
Single-cell RNA-sequencing (scRNA-seq) was used to profile senescent cancer cell clusters induced by RT in LNCaP cells. The expression and phosphorylation levels of ataxia telangiectasia and Rad3-related protein (ATR) were assessed by immunohistochemistry in clinical samples from patients with or without RT. Co-immunoprecipitation, mutagenesis, and Western blotting were used to measure the interactions between proteins. Xenograft experiments were performed to assess the tumor immune response in the mice.


RESULTS
We identified a subset of PCa cells that exhibited resistance to RT, characterized by a reduced antigen presentation capability, which enhanced their ability to evade immune detection and resist cytotoxic T-lymphocyte-associated protein 4 (CTLA-4) blockade. scRNA-seq revealed that the senescent state was a transient phase of PCa cells post-RT, particularly in CTLA-4 blockade treatment-resistant cells. This state was marked by increased cytosolic ATR level. Cytosolic ATR phosphorylated CD86 in its cytosolic domain and enhanced the interaction between CD86 and its E3 ligase MARCH1 through electrostatic attraction. Depletion or inhibition of Atr increased the sensitivity to immune attack and improved responses to anti-Ctla-4 antibody treatment in a mouse model.


CONCLUSIONS
Our findings indicate that the activation of cytosolic ATR, which is associated with cellular senescence, impedes the effectiveness of combined RT and ICB treatments. This discovery may provide valuable insights for improving the efficacy of combined RT and ICB therapies in PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54111804377a9c0bbc1679bc34e9106d5b8ef9e5" target='_blank'>
              Radiotherapy-resistant prostate cancer cells escape immune checkpoint blockade through the senescence-related ataxia telangiectasia and Rad3-related protein.
              </a>
            </td>
          <td>
            Chenyi Shao, Yingyi Zhang, Hang Li, Jiajia Chen, Ting Huang, Jiaze Li, Simeng Wen, Sen Wang, Saijun Fan, Yu Zhao
          </td>
          <td>2024-12-19</td>
          <td>Cancer communications</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is an aggressive malignancy that poses significant challenges due to high rates of relapse and resistance to treatment, particularly in older populations. While therapeutic advances have been made, survival outcomes remain suboptimal. The evolution of DNA and RNA sequencing technologies, including whole-genome sequencing (WGS), whole-exome sequencing (WES), and RNA sequencing (RNA-Seq), has significantly enhanced our understanding of AML at the molecular level. These technologies have led to the discovery of driver mutations and transcriptomic alterations critical for improving diagnosis, prognosis, and personalized therapy development. Furthermore, single-cell RNA sequencing (scRNA-Seq) has uncovered rare subpopulations of leukemia stem cells (LSCs) contributing to disease progression and relapse. However, widespread clinical integration of these tools remains limited by costs, data complexity, and ethical challenges. This review explores recent advancements in DNA/RNA sequencing in AML and highlights both the potential and limitations of these techniques in clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51e5d43f554c8fe98964b8130a7d1804ae561d5a" target='_blank'>
              Advances in DNA/RNA Sequencing and Their Applications in Acute Myeloid Leukemia (AML)
              </a>
            </td>
          <td>
            Fatimah Ahmed, Jiang Zhong
          </td>
          <td>2024-12-25</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer cells exhibit a remarkable resilience to cytotoxic stress, often adapting through transcriptional changes linked to alterations in chromatin structure. In several types of cancer, these adaptations involve epigenetic modifications and restructuring of topologically associating domains (TADs). However, the underlying principles by which chromatin architecture facilitates such adaptability across different cancers remain poorly understood. To investigate the role of chromatin in this process, we developed a physics-based mechanistic model that connects chromatin organization to cell fate decisions, specifically survival following chemotherapy. Our model builds on the observation that chromatin forms packing domains, which influence transcriptional efficiency through macromolecular crowding. The model accurately predicts chemoevasion in vitro, suggesting that changes in packing domains affect the likelihood of survival. Consistent results across diverse cancer types indicate that the model captures fundamental principles of chromatin-mediated adaptation, independent of the specific cancer or chemotherapy mechanisms involved. Based on these insights, we hypothesized that compounds capable of modulating packing domains, termed Transcriptional Plasticity Regulators (TPRs), could prevent cellular adaptation to chemotherapy. Using live-cell chromatin imaging, we conducted a compound screen that identified several TPRs which synergistically enhanced chemotherapyinduced cell death. The most effective TPR significantly improved therapeutic outcomes in a patient-derived xenograft (PDX) model of ovarian cancer. These findings underscore the central role of chromatin in cellular adaptation to cytotoxic stress and present a novel framework for enhancing cancer therapies, with broad potential across multiple cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f35f61f79981a14cfb46f8ab43779aba13110922" target='_blank'>
              Leveraging chromatin packing domains to target chemoevasion in vivo
              </a>
            </td>
          <td>
            J. Frederick, R. Virk, I. C. Ye, L. Almassalha, Greta Wodarcyk, D. VanDerway, Paola Carrillo Gonzalez, Rikkert J. Nap, Vasundhara Agrawal, Nicholas M. Anthony, John Carinato, Wing Shun Li, Cody L Dunton, Karla I Medina, Rivaan Kakkaramadam, Surbhi Jain, Shohreh Shahabi, Guillermo Ameer, I. Szleifer, Vadim Backman
          </td>
          <td>2024-11-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="
 Breast tumors comprise several sub-populations of cells, so-called “clones”, with specific cellular and molecular characteristics. Some cancer clones can be more ‘aggressive’ than others in particular tumor microenvironments (TMEs), contributing to metastatic progression and drug resistance. Understanding the specific genes/pathways deregulated in these aggressive cancer cells is essential for developing new tailored treatment strategies. To accomplish this, we optimized the use of a lentiviral-based barcoding technique (Berthelet, J., et.al., Sci Adv 2021, Serrano, A., et.al., Nat Rev Cancer 2022) where cancer cells from cell lines and patient-derived xenografts are tagged with optical barcodes and transplanted into the mammary fat pads of immunodeficient mice. Upon resection of tumors and collection of metastatic organs at the experimental endpoint, we analyzed clonal fate and fitness using flow cytometry and imaging. The subsequent molecular characterization of individual clones was determined by bulk and single-cell RNA sequencing. We also leverage the use of these barcoded models to investigate the mechanisms of innate and acquired resistance to Sacituzumab Govitecan (SG), a drug recently approved for the treatment of patients with advanced breast cancer. Our studies revealed unique gene signatures and pathways associated with preferential organ tropism (in the lungs, liver and brain TMEs) and SG resistance. This comprehensive analysis, currently validated using autopsy samples and patient databases, will support the design of new predictive markers and targeted therapies for metastatic breast cancer.
 Citation Format: Sreeja Reddy Gadipally, Jean Berthelet, Samuel C. Lee, Dharmesh D. Bhuva, Michael Roach, Farrah El-Saafin, Caroline Bell, Yunjian Wu, David Baloyan, Christphe Ginestier, Charafe Jauffret Emmanuelle, Melissa Davis, Robin L. Anderson, Luciano Martelotto, Belinda Yeo, Delphine Merino. Exploring breast cancer heterogeneity in the context of metastatic tumour microenvironments and therapy resistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr B005.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/528d6539434045b291c3a0faa67fa6d3c29fbc80" target='_blank'>
              Abstract B005: Exploring breast cancer heterogeneity in the context of metastatic tumour microenvironments and therapy resistance
              </a>
            </td>
          <td>
            Sreeja Gadipally, J. Berthelet, Samuel C. Lee, Dharmesh D. Bhuva, Michael Roach, Farrah El-Saafin, C. Bell, Yunjian Wu, David Baloyan, Christphe Ginestier, C. Emmanuelle, Melissa J Davis, Robin L Anderson, Luciano Martelotto, B. Yeo, Delphine Merino
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4cb54307571359f46baeb0ac619041e8a5efa2f" target='_blank'>
              Integrating single-cell RNA-seq and bulk RNA-seq to construct a neutrophil prognostic model for predicting prognosis and immune response in oral squamous cell carcinoma
              </a>
            </td>
          <td>
            Jinhang Wang, Zifeng Cui, Qiwen Song, Kaicheng Yang, Yanping Chen, Shixiong Peng
          </td>
          <td>2024-12-26</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec5ccdfbc4363bf23d6a45f74342f6b2cc8e760a" target='_blank'>
              CRISPR/Cas9 system: a novel approach to overcome chemotherapy and radiotherapy resistance in cancer.
              </a>
            </td>
          <td>
            Somaye Noruzi, R. Mohammadi, K. Jamialahmadi
          </td>
          <td>2024-11-19</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edb16603152832a7b5a7238cbeef6e092d7b2df9" target='_blank'>
              PRMT5-regulated splicing of DNA repair genes drives chemoresistance in breast cancer stem cells.
              </a>
            </td>
          <td>
            Matthew S Gillespie, Kelly Chiang, Gemma L Regan-Mochrie, Soo-Youn Choi, Ciara Ward, Debashish Sahay, Paloma Garcia, Roland Arnold, Clare C Davies
          </td>
          <td>2024-12-18</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a321392f82837178158946451c3a5d2c2f02daa" target='_blank'>
              Genomic and transcriptomic profiling of radioresistant prostate and head and neck cancers implicate a BAHD1-dependent modification of DNA damage at the heterochromatin
              </a>
            </td>
          <td>
            Chaw Yee Beh, Celestia P X Yeo, Boon Hao Hong, E. Tan, Kah M Tan, Dennis J J Poon, Pek L Chu, Dewi Susanti, Pei Ling Tai, Monica Ryu, James Proudfoot, E. Yeo, Khee Chee Soo, Melvin L. K. Chua
          </td>
          <td>2024-12-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is a leading cause of cancer-related deaths globally. The heterogeneity of the tumor microenvironment significantly influences patient prognosis, while the diversity of tumor cells shapes its unique characteristics. A comprehensive analysis of the molecular profile of tumor cells is crucial for identifying novel molecular targets for drug sensitivity analysis and for uncovering the pathophysiological mechanisms underlying CRC. Methods We utilized single-cell RNA sequencing technology to analyze 13 tissue samples from 4 CRC patients, identifying key cell types within the tumor microenvironment. Intercellular communication was assessed using CellChat, and a risk score model was developed based on eight prognostic genes to enhance patient stratification for immunotherapeutic approaches. Additionally, in vitro experiments were performed on DLX2, a gene strongly associated with poor prognosis, to validate its potential role as a therapeutic target in CRC progression. Results Eight major cell types were identified across the tissue samples. Within the tumor cell population, seven distinct subtypes were recognized, with the C0 FXYD5+ tumor cells subtype being significantly linked to cancer progression and poor prognosis. CellChat analysis indicated extensive communication among tumor cells, fibroblasts, and immune cells, underscoring the complexity of the tumor microenvironment. The risk score model demonstrated high accuracy in predicting 1-, 3-, and 5-year survival rates in CRC patients. Enrichment analysis revealed that the C0 FXYD5+ tumor cell subtype exhibited increased energy metabolism, protein synthesis, and oxidative phosphorylation, contributing to its aggressive behavior. In vitro experiments confirmed DLX2 as a critical gene associated with poor prognosis, suggesting its viability as a target for improving drug sensitivity. Conclusion In summary, this study advances our understanding of CRC progression by identifying critical tumor subtypes, molecular pathways, and prognostic markers that can inform innovative strategies for predicting and enhancing drug sensitivity. These findings hold promise for optimizing immunotherapeutic approaches and developing new targeted therapies, ultimately aiming to improve patient outcomes in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8647de1e0192ab64820f770599e15f3edf4689c3" target='_blank'>
              Innovative strategies to optimise colorectal cancer immunotherapy through molecular mechanism insights
              </a>
            </td>
          <td>
            Quanjun Lin, Zhiqiang Wang, Jue Wang, Ming Xu, Xinyi Zhang, Peng Sun, Yihang Yuan
          </td>
          <td>2024-12-09</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Simple Summary Cancer initiation and progression are controlled not only by intrinsic factors, such as genetic mutations and mesenchymal trait acquisition, but also by dynamic spatiotemporal shifts within the tumor microenvironment. These elements induce plasticity, and enhance invasiveness, metastasis, and treatment resistance, while promoting immune evasion through the reprogramming of the immune microenvironment. However, the molecular mechanisms driving this reprogramming remain unclear. In this review, we summarize the recent pivotal findings on pancreatic cancer, which is a highly malignant disease, and overview the roles of plasticity, metabolic reprogramming, and the creation of an immunosuppressive microenvironment in disease progression. We also present our research on two crucial signaling pathways, i.e., the dynamic interplay between mutational load and the tumor microenvironment, and the acquisition of cancer plasticity via inflammatory responses and metabolic reprogramming. A comprehensive understanding of these processes is expected to transform therapeutic strategies and improve the outcomes of pancreatic cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09d6b2b31d56874111eca289fe4cec7a0df598ab" target='_blank'>
              Plasticity and Tumor Microenvironment in Pancreatic Cancer: Genetic, Metabolic, and Immune Perspectives
              </a>
            </td>
          <td>
            A. Hashimoto, S. Hashimoto
          </td>
          <td>2024-12-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5889adf5def7b284cf9f7d22530eab68401de07f" target='_blank'>
              PlexinD1 is a driver and a therapeutic target in advanced prostate cancer.
              </a>
            </td>
          <td>
            Jing Wei, Jing Wang, Wen Guan, Jingjing Li, Tianjie Pu, E. Corey, Tzu-Ping Lin, Allen C Gao, Boyang Jason Wu
          </td>
          <td>2025-01-02</td>
          <td>EMBO molecular medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Oncogenesis and tumor progression are shaped by somatic alterations in the cancer genome and features of the tumor immune microenvironment (TME). How interactions of these two systems influence tumor development and clinical outcomes remains incompletely understood. To address this challenge, we developed the multi-omics analysis framework PACIFIC to systematically integrate genetic cancer drivers and infiltration profiles of immune cells with clinical information. In an analysis of 8500 cancer samples, we report 34 immunogenomic interactions (IGXs) in 13 cancer types in which context-specific combinations of genomic alterations and immune cell activities associate with disease outcomes. Risk associations of IGXs are potentially explained by tumor-intrinsic and microenvironmental metrics of immunogenicity and differential expression of therapeutic targets. In luminal-A breast cancer, MEN1 deletion combined with reduced neutrophils is associated with poor prognosis and deregulation of immune signalling pathways. These findings help elucidate how cancer drivers interact with TME to contribute to tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7541ff011e197e5ec1cc98d2bd31291176ffd633" target='_blank'>
              Combinations of genomic alterations and immune microenvironmental features associate with patient survival in multiple cancer types
              </a>
            </td>
          <td>
            Masroor Bayati, Zoe P. Klein, Alec Bahcheli, Mykhaylo Slobodyanyuk, Jeffrey To, , Jigyansa Mishra, Diogo Pellegrina, Kissy Guevara-Hoyer, Chris McIntosh, Mamatha Bhat, J. Reimand
          </td>
          <td>2024-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Multiple myeloma (MM), a clonal B-cell neoplasia, is an incurable and heterogeneous disease where survival ranges from a few months to more than 10 years. The clinical heterogeneity of MM arises from multiple genomic events that result in tumour development and progression. Recurring genomic abnormalities including cytogenetic abnormalities, gene mutations and abnormal gene expression profiles in myeloma cells have a strong prognostic power. With the advancement in technologies and the development of novel drugs, the prognostic factors and treatment paradigms of MM have been fast evolving over the past few years. Following the introduction of new highthroughput cytogenomic technologies such as array comparative genome hybridisation (aCGH) or single nucleotide polymorphism array (SNP array) and molecular techniques such as gene expression profiling (GEP) and massively parallel genomic sequencing, the prediction of survival in MM no longer solely depends on conventional cytogenetics and interphase fluorescence in situ hybridisation (iFISH) analysis findings. These new technologies enable screening for all possible chromosomal aberrations and other genomic alterations, identifying each aberration on a case-bycase basis and discovering new aberrations that are relevant in unraveling the tumor cells' complex biology. This in turn allows a better understanding of the disease complexity and heterogeneity. The objective of this review on the genetic architecture of MM is to discuss the latest developments on the cytogenetic/cytogenomic-based risk classification of MM that are currently in use and their prognostic implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f51a37b55e510ee5aeb00a67198a8c2f2e860c2a" target='_blank'>
              Genetic architecture of Multiple Myeloma and its prognostic implications - An updated review.
              </a>
            </td>
          <td>
            E. Foong, R. Ankathil
          </td>
          <td>2024-12-01</td>
          <td>The Malaysian journal of pathology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbb9159d119b846b192af86b098eda853ace7fad" target='_blank'>
              Orthogonal and multiplexable genetic perturbations with an engineered prime editor and a diverse RNA array
              </a>
            </td>
          <td>
            Qichen Yuan, Hongzhi Zeng, Tyler C Daniel, Qingzhuo Liu, Yongjie Yang, Emmanuel C Osikpa, Qiaochu Yang, Advaith Peddi, Liliana M Abramson, Boyang Zhang, Yong Xu, Xue Gao
          </td>
          <td>2024-12-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The CRISPR/Cas system has emerged as a transformative genome-editing technology, enabling precise modifications of genetic material. This paper explores the potential applications of CRISPR/Cas technology in the treatment of gliomas, particularly glioblastoma (GBM), the most aggressive and lethal brain tumor. With over 3,000 genes linked to disease-causing mutations, CRISPR/Cas offers a novel approach to address the underlying genetic alterations in cancer. Recent studies demonstrate the efficacy of CRISPR/Cas12a in knocking out miR-21, a microRNA inversely correlated with glioblastoma survival, resulting in reduced tumor growth in mouse models. Furthermore, we introduce a novel "genome shredding" approach using CRISPR/Cas9 to target repetitive DNA sequences in GBM, leading to extensive cell death, independent of the tumor's genetic profile. Despite these promising advancements, challenges remain in optimizing CRISPR's efficacy and delivery mechanisms to improve patient outcomes. This review highlights the significant potential of CRISPR/Cas systems in revolutionizing glioma therapy while addressing the need for continued research to mitigate off-target effects and enhance therapeutic delivery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/500664bef54808a00d4dd7947aa8294abce819eb" target='_blank'>
              CRISPR/Cas gene editing and its application in the treatment of Gliomas in the brain
              </a>
            </td>
          <td>
            Raymond Sun
          </td>
          <td>2024-12-24</td>
          <td>Highlights in Science, Engineering and Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancers exhibit a remarkable ability to develop resistance to a range of treatments, often resulting in relapse following first-line therapies and significantly worse outcomes for subsequent treatments. While our understanding of the mechanisms and dynamics of the emergence of resistance during cancer therapy continues to advance, many questions remain about which treatment strategies can minimize the probability that resistance will evolve, thereby improving long-term patient outcomes. In this study, we present an evolutionary simulation model of a clonal population of cells that can acquire resistance mutations to one or more treatments. We then leverage this model to examine the efficacy of a two-strike “extinction therapy” protocol—in which two treatments are applied sequentially in an effort to first contract the population to a vulnerable state and then push it to extinction—in comparison to that of a combination therapy protocol. We investigate the impact of parameters such as the timing of the switch between the two strikes, the rate of emergence of resistant mutations, the dose of the applied drugs, the presence of cross-resistance, and whether resistance is a binary or a quantitative trait. Our results indicate that the timing of switching from the first to the second strike has a marked effect on the likelihood of driving the population to extinction, and that extinction therapy outperforms combination therapy when cross-resistance is present. We conduct an in silico trial that reveals more detailed insight into when and why a second strike will succeed or fail. Finally, we demonstrate that modeling resistance as a quantitative rather than binary trait does not change our overall conclusions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f738c3dddcab46b26ff2c97c3888fefb1e9d2858" target='_blank'>
              Evolutionary rescue model informs strategies for driving cancer cell populations to extinction
              </a>
            </td>
          <td>
            Amjad Dabi, Joel S. Brown, Robert A Gatenby, Corbin D. Jones, Daniel R. Schrider
          </td>
          <td>2024-11-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Abstract Radiotherapy is a cornerstone in the treatment of various tumors, yet radioresistance often leads to treatment failure and tumor recurrence. Several factors contribute to this resistance, including hypoxia, DNA repair mechanisms, and cancer stem cells. This review explores the diverse elements that drive tumor radiotherapy resistance. Historically, resistance has been attributed to cellular repair and tumor repopulation, but recent research has expanded this understanding. The tumor microenvironment – characterized by hypoxia, immune evasion, and stromal interactions – further complicates treatment. Additionally, molecular mechanisms such as aberrant signaling pathways, epigenetic modifications, and non-B-DNA structures play significant roles in mediating resistance. This review synthesizes current knowledge, highlighting the interplay of these factors and their clinical implications. Understanding these mechanisms is crucial for developing strategies to overcome resistance and improve therapeutic outcomes in cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fed4d49a52ebd6a8d3980680b9bc6dfd6788002" target='_blank'>
              Myriad factors and pathways influencing tumor radiotherapy resistance
              </a>
            </td>
          <td>
            Lanjuan Mi, Hongquan Zhang
          </td>
          <td>2024-01-01</td>
          <td>Open Life Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f8a77c179e3e4f5ecf72af9fc5a5da53d8facac" target='_blank'>
              Overview of a comparative analysis of microsatellite instability and standard mismatch repair protein-deficiency tests in a large cancer cohort.
              </a>
            </td>
          <td>
            András Kiss, M. L. Nádorvári, J. Kulka, T. Barbai, E. Raso, István Kenessey, Gábor Lotz, József Tímár
          </td>
          <td>2024-12-04</td>
          <td>Pathologie</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a hematological malignancy characterized by a block in differentiation and accelerated proliferation of myeloid progenitor cells. Genes encoding for epigenetic regulators are among the most frequent targets for mutations and structural variations in AML, giving rise to profound epigenetic heterogeneity between and within tumors. Deletions of chromosome 5q [del(5q)] are among the most common copy number alterations in AML and are associated with extremely poor clinical outcome and therapy resistance, however the mechanisms linking del(5q) to leukemic progression are not understood. Analyzing DNA methylation profiles from 477 elderly AML patients using DNA methylome deconvolution, we discovered that del(5q) AML is an epigenetically distinct subgroup characterized by a signature of DNA hypermethylation, which we propose may be linked to dysregulation of H3K9me1/2 and overexpression of the leukemic stem cell marker, DNMT3B. Interrogation of the minimally deleted 5q region highlighted the H3K9me1/2 demethylase KDM3B as a likely target for haploinsufficiency in this subgroup. Our data suggest that del(5q) AML should be reconsidered as an epigenetically dysregulated subgroup, driven by heterozygous loss of KDM3B, and that the resulting imbalance of H3K9me1/2 may contribute to the progression of these aggressive leukemias.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8284d22674e6348a422c0aa420226cc0452e3a13" target='_blank'>
              Acute Myeloid Leukemia with deletion 5q is an epigenetically distinct subgroup defined by heterozygous loss of KDM3B
              </a>
            </td>
          <td>
            Katherine Kelly, Linda Welte, Etienne Sollier, Anna Riedel, Fiona Brown-Burke, Michael Scherer, Harold Keer, Mohammad Azab, E. Jahn, H. Döhner, Konstanze Döhner, P. Lutsik, C. Plass
          </td>
          <td>2024-11-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a278e50ce1c75eb976a4bb58887b3881b23ccb24" target='_blank'>
              Deep CRISPR mutagenesis characterizes the functional diversity of TP53 mutations.
              </a>
            </td>
          <td>
            Julianne S Funk, Maria Klimovich, Daniel Drangenstein, Ole Pielhoop, Pascal Hunold, Anna Borowek, Maxim Noeparast, Evangelos Pavlakis, M. Neumann, Dimitrios-Ilias Balourdas, Katharina Kochhan, N. Merle, Imke Bullwinkel, M. Wanzel, Sabrina Elmshäuser, J. Teply-Szymanski, Andrea Nist, Tara Procida, Marek Bartkuhn, Katharina Humpert, M. Mernberger, R. Savai, Thierry Soussi, A. Joerger, Thorsten Stiewe
          </td>
          <td>2025-01-07</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b26feb24c74a70e25f404994931c40c73dd0fe64" target='_blank'>
              Deciphering molecular landscape of breast cancer progression and insights from functional genomics and therapeutic explorations followed by in vitro validation
              </a>
            </td>
          <td>
            Bushra Khan, Rowaid M. Qahwaji, Mashael S. Alfaifi, Tanwir Athar, Abdullah Khan, Mohammad Mobashir, I. Ashankyty, K. Imtiyaz, Areej Alahmadi, M. M. Rizvi
          </td>
          <td>2024-11-20</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Abstract BACKGROUND Breast cancer is the most common malignancy in women; brain metastasis is a marked cause of mortality. Recent genomic studies have identified distinct genetic profiles in brain metastases, suggesting unique mechanisms underlying treatment resistance. This study aimed to compare gene expression in primary breast tumors and the corresponding brain metastases to identify potential biomarkers and therapeutic targets. METHODS Eight paired samples of primary breast tumors and brain metastases from patients who developed brain metastases after radical mastectomy were analyzed. RNA sequencing was used to comprehensively assess gene expression differences between the two tumor types. RESULTS RNA sequencing revealed 391 downregulated and 474 upregulated genes in brain metastases compared to primary tumors. Gene set enrichment analysis revealed that epithelial-mesenchymal transition (EMT)-associated genes (SPP, GREM1, and MMP3) were upregulated in primary breast tumors. Conversely, reactive oxygen species (ROS)-related genes (GPX4 and PRDX family members) were upregulated in brain metastases. CONCLUSION High expression of EMT-related genes in primary tumors indicates that these tumors prepare for metastasis by inducing EMT during radical resection, enabling tumors to invade the surrounding tissues and metastasize to distant organs. Additionally, ROS-related genes are highly expressed in metastatic brain tumors, suggesting that these genes contribute to tumor enhancement at the metastatic site. These findings could aid in development of biomarkers for predicting brain metastasis and for identifying novel therapeutic targets. In future, we aim to analyze tumor localization and functions of these gene products.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e41314c75680ba4a96e915950728faec65dcd87" target='_blank'>
              10168- GEN-11 GENETIC ANALYSIS OF PRIMARY BREAST CANCER TUMORS AND BRAIN METASTASIS TUMORS WITH RNA-SEQ
              </a>
            </td>
          <td>
            Ayaka Sakakibara, Masayo Kaneko, M. Shirahata, Tetsuya Sato, Masanori Yasuda, Akihiko Osaki, K. Mishima
          </td>
          <td>2024-11-29</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="The CRISPR/Cas genome editing approach in non-model organisms poses challenges that remain to be resolved. Here, we demonstrated a generalized roadmap for a de novo genome annotation approach applied to the non-model organism Macrobrachium rosenbergii. We also addressed the typical genome editing challenges arising from genetic variations, such as a high frequency of single nucleotide polymorphisms, differences in sex chromosomes, and repetitive sequences that can lead to off-target events. For the genome editing of M. rosenbergii, our laboratory recently adapted the CRISPR/Cas genome editing approach to embryos and the embryonic primary cell culture. In this continuation study, an annotation pipeline was trained to predict the gene models by leveraging the available genomic, transcriptomic, and proteomic data, and enabling accurate gene prediction and guide design for knock-outs. A next-generation sequencing analysis demonstrated a high frequency of genetic variations in genes on both autosomal and sex chromosomes, which have been shown to affect the accuracy of editing analyses. To enable future applications based on the CRISPR/Cas tool in non-model organisms, we also verified the reliability of editing efficiency and tracked off-target frequencies. Despite the lack of comprehensive information on non-model organisms, this study provides an example of the feasibility of selecting and editing specific genes with a high degree of certainty.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba9f9b90d707bd226bf9a4714aba878076437ac1" target='_blank'>
              Roadmap and Considerations for Genome Editing in a Non-Model Organism: Genetic Variations and Off-Target Profiling
              </a>
            </td>
          <td>
            Hanin Wattad, Jonathan Molcho, Rivka Manor, S. Weil, E. Aflalo, V. Chalifa-Caspi, Amir Sagi
          </td>
          <td>2024-11-22</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Simple Summary Adult B-cell acute lymphoblastic leukemia (B-ALL) is characterized by genetic heterogeneity and high relapse rate. Forty-four adult B-ALL patients were studied at both diagnosis and relapse by next-generation sequencing (NGS) in order to identify genetic alteration driving relapse. Four main genetic pathways leading to relapse in adults were identified: IKZF1plus genetic profile, RAS mutations and TP53 alterations in Philadelphia chromosome (Ph)-negative B-ALL and acquisition of ABL1 mutations in Ph-positive patients. In addition, three clonal evolution patterns were identified: the persistent clone trajectory (25%), the expanding clone trajectory (11%) and the therapy-boosted trajectory (48%). Our results reveal the presence of preferential biological pathways leading to relapse in adult B-ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54416c39b5f84bc7f01293f1564700b87fc9be40" target='_blank'>
              Preferential Genetic Pathways Lead to Relapses in Adult B-Cell Acute Lymphoblastic Leukemia
              </a>
            </td>
          <td>
            Josgrey Navas-Acosta, Alberto Hernández-Sánchez, Teresa González, Ángela Villaverde Ramiro, Sandra Santos, Cristina Miguel, J. Ribera, Isabe L Granada, M. Morgades, Ricardo Sánchez, E. Such, S. Barrena, Juana Ciudad, Julio Dávila, Natalia de Las Heras, Alfonso García-de Coca, J. Labrador, J. Queizán, Sandra Martín, Alberto Orfao, J. Ribera, Rocío Benito, J. Hernández-Rivas
          </td>
          <td>2024-12-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Background Acute myeloid leukemia (AML) is highly prevalent and heterogeneous among adult acute leukemias. Current chemotherapeutic approaches for AML often face the challenge of drug resistance, and AML immune cells play an important role in the regulation of AML drug resistance. Thus, it is of key significance to explore the regulatory mechanisms of immune cells in AML to alleviate chemotherapy resistance in AML. Methods Based on AML single-cell transcriptomic data, this study revealed the differences in the expression of immune cell subpopulations and marker genes in AML patients in the complete remission group (CR) compared to AML patients in the non-complete remission group (non-CR) after chemotherapy. Functional enrichment by clusterprofiler revealed the regulatory functions of differentially expressed genes (DEGs) in AML. AUCell enrichment scores were used to assess the immunoregulatory functions of immune cells. Pseudotime analysis was used to construct immune cell differentiation trajectories. CellChat was used for cellular communication analysis to elucidate the interactions between immune cells. Survival analysis with the R package “survival” revealed the role of immune cell marker genes on AML prognosis. Finally, the wound healing and trans-well assay were performed. Results Single-cell clustering analysis revealed that NK/T cells and macrophage cells subpopulations were significantly higher in non-CR AML patients than in CR AML. AUCell enrichment analysis revealed that FCAR+ and FCGR3A+ macrophages were significantly more active in the non-CR group and correlated with processes regulating cellular energy metabolism and immune cell activity. Differentially expressed NK cell marker genes between CR and non-CR groups mainly included HBA1, S100A8, and S100A9, which were associated with cancer drug resistance regulation, these marker genes of (FCAR, FCGR3A, PREX1, S100A8 and S100A9) were upregulated in human chronic myeloid leukemia cells (HAP1) and silencing of S100A8 affected migration and invasion of HAP1 cells. In particular, the differentiation pathways of macrophages and NK cells in non-CR differed from those of patients in the CR group. Cellular communication analyses showed that ligand-receptor pairs between NK cells and macrophage cells mainly included HLA-E-KLRK1, HLA-E-KLRC1, HLA-E-CD94:NKG2A, CLEC2B-KLRB1. In addition, LGALS9-CD45, CCL3L1- CCR1, CCL3-CCR1 between these two immune cells mainly regulate secreted signaling to mediate AML progression. Marker genes in NK/T cells and macrophage cells were significantly associated with AML prognosis. Conclusion This study reveals the potential role of NK cells and macrophages in AML chemoresistance through the analysis of single-cell RNA sequencing data. This provides new ideas and insights into the key mechanisms of immune cells in AML treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c53c451df2b0963907333145bbeb332073d75fc7" target='_blank'>
              Single-cell data revealed the function of natural killer cells and macrophage cells in chemotherapy tolerance in acute myeloid leukemia
              </a>
            </td>
          <td>
            Jing Gao, Xueqian Yan, Dan Fan, Yuanchun Li
          </td>
          <td>2024-11-21</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d49c3b89849fb97fd894e63e495575849c7b7545" target='_blank'>
              A multilineage screen identifies actionable synthetic lethal interactions in human cancers.
              </a>
            </td>
          <td>
            Samson H. Fong, Brent M Kuenzi, Nicole M Mattson, John Lee, Kyle S. Sanchez, A. Bojorquez-Gomez, Kyle Ford, Brenton P. Munson, Katherine Licon, Sarah Bergendahl, John Paul Shen, J. Kreisberg, Prashant Mali, Jeffrey H Hager, Michael A White, T. Ideker
          </td>
          <td>2024-11-18</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>80</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81af71d49f35f4ca08b391cf52780be4b4e726a9" target='_blank'>
              Integrated single-cell analysis reveals distinct epigenetic-regulated cancer cell states and a heterogeneity-guided core signature in tamoxifen-resistant breast cancer
              </a>
            </td>
          <td>
            Kun Fang, A. Ohihoin, Tianxiang Liu, Lavanya Choppavarapu, Bakhtiyor Nosirov, Qianben Wang, Xue-Zhong Yu, S. Kamaraju, Gustavo W. Leone, Victor X Jin
          </td>
          <td>2024-11-18</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15816c951b15b9cb72261b03981624db25adcb29" target='_blank'>
              Non-coding RNAs modulation in breast cancer radioresponse: mechanisms and therapeutic implications.
              </a>
            </td>
          <td>
            A. Moqadami, Sahar Ghafari, Mohammad Khalaj-Kondori
          </td>
          <td>2024-11-15</td>
          <td>Strahlentherapie und Onkologie : Organ der Deutschen Rontgengesellschaft ... [et al]</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The association between RET and multiple endocrine neoplasia type 2 was established in 1993 and remains one of the very few oncogenes for which distinct phenotypes (medullary thyroid cancer or phaeochromocytoma) are associated with the same hot-spot variants occurring either in germline or somatic DNA. Somatic RET fusion events have also been described in several cancers, including papillary thyroid cancer, non-small cell lung cancer, breast cancer, salivary gland cancer and pancreatic cancer. Highly selective RET inhibitors have improved outcomes in RET-altered cancers and have been well tolerated. Nevertheless, primary and acquired drug resistance has been observed, arising from distinct genomic alterations either in RET (on target resistance) or via alternate oncogenic pathways (bypass resistance). The same mechanisms of resistance have been observed across multiple cancer types, which implies RET-altered cancers evolve away from RET addiction via stochastic subclonal events. Understanding these mechanisms is crucial for identifying therapeutic opportunities to overcome resistance. Successful treatment targeting bypass oncogenes has been reported in several instances, at least for short-term outcomes; in contrast, although several compounds have been reported to overcome on-target RET alterations, none have yet been translated into routine clinical practice and this remains an area of urgent clinical need.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce4b80f3267e487a6f53e42db32a243722c94787" target='_blank'>
              Mechanisms of resistance to RET-directed therapies.
              </a>
            </td>
          <td>
            R. J. Clifton-Bligh
          </td>
          <td>2024-12-01</td>
          <td>Endocrine-related cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Although immune checkpoint inhibitors (ICIs) have revolutionized melanoma treatment, many individuals either show no response or eventually develop acquired resistance. This variability could be linked to differences in the tumor immune microenvironment, pre-existing drug-resistant cells or treatment induced changes in melanoma cell states. Using single cell RNA sequencing (scRNA-seq) coupled with heritable barcodes, this study aims to lineage trace changes in melanoma transcriptional cell states and map changes in the tumor immune microenvironment to uncover if preexisting and/or therapy-induced melanoma transcriptional cell states lead to resistance to ICIs. In this study, we used YUMMER1.7 mouse melanoma cells which were transduced with barcodes under conditions that allow delivery of a single unique barcode to each cell, this served as a cell lineage tag. Each barcode is heritable and stably transcribed into RNA molecules so that individual barcoded cells can be matched with a gene expression profile from scRNA-seq outputs. The barcoded cells were subcutaneously injected into C57BL/6 mice, and once tumors were established, mice were treated with 3 cycles of anti-PD-1 and anti-CTLA-4 therapy. Lineage tracing and scRNA-seq analyses were performed on tumors harvested prior to treatment, early on treatment (Day 6), during minimal residual disease (Day 13) and upon relapse. Early on treatment (Day 6)- two tumour groups were harvested, those that responded well and others that only partially responded. Our results showed that during the early on treatment phase (Day 6) tumors that responded well and those that only partially responded displayed distinct barcodes. Additionally, prevalent barcoded cells identified at the minimal residual disease phase were also dominant upon relapse. Analysis of the transcriptional states is underway to determine if ICI therapy induces transcriptional heterogeneity in sensitive and tolerant cells and if a particular or several transcriptional states lead to relapse. This study will potentially identify predictive response biomarkers and vulnerabilities of ICI naïve and resistant cells that could be targeted to improve outcomes for melanoma patients. We will also identify neoadjuvant approaches, to remove the inherent heterogeneity within the tumor cell population, facilitating a more uniform sensitivity to ICIs.
 Citation Format: Reem Saleh, Riyaben Patel, Dane Vassiliadis, Fayrouz Hammal, Benjamin Blyth, Xin Du, Katie Fennell, Mark A. Dawson, Grant A. McArthur, Karen Sheppard. Single cell RNA-sequencing coupled with lineage tracing identifies novel clonal populations associated with immunotherapy resistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr C044.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2046528ce162da3c656f0facbde8bbcbfa6faf03" target='_blank'>
              Abstract C044: Single cell RNA-sequencing coupled with lineage tracing identifies novel clonal populations associated with immunotherapy resistance
              </a>
            </td>
          <td>
            Reem Saleh, Riyaben P. Patel, Dane Vassiliadis, Fayrouz Hammal, Benjamin Blyth, Xin Du, Katie Fennell, Mark A. Dawson, Grant A. McArthur, Karen E. Sheppard
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Gene expression is coordinated by a multitude of transcription factors (TFs), whose binding to the genome is directed through multiple interconnected epigenetic signals, including chromatin accessibility and histone modifications. These complex networks have been shown to be disrupted during aging, disease, and cancer. However, profiling these networks across diverse cell types and states has been limited due to the technical constraints of existing methods for mapping DNA:Protein interactions in single cells. As a result, a critical gap remains in understanding where TFs or other chromatin remodelers bind to DNA and how these interactions are perturbed in pathological contexts. To address this challenge, we developed a transformative single-cell immuno-tethering DNA:Protein mapping technology. By coupling a species-specific antibody-binding nanobody to a cytosine base editing enzyme, this approach enables profiling of even weak or transient factor binding to DNA, a task that was previously unachievable in single cells. Thus, our Docking & Deamination followed by sequencing (D&D-seq) technique induces cytosine-to-uracil edits in genomic regions bound by the target protein, offering a novel means to capture DNA:Protein interactions with unprecedented resolution. Importantly, this technique can be seamlessly incorporated into common single-cell multiomics workflows, enabling multimodal analysis of gene regulation in single cells. We tested the ability of D&D-seq to record TF binding both in bulk and at the single-cell level by profiling CTCF and GATA family members, obtaining high specificity and efficiency, with clear identification of TF footprint and signal retention in the targeted cell subpopulations. Furthermore, the deamination reaction showed minimal off-target activity, with high concordance to bulk ChIP-seq reference data. Applied to primary human peripheral blood mononuclear cells (PBMCs), D&D-seq successfully identified CTCF binding sites and enabled integration with advanced machine-learning algorithms for predicting 3D chromatin structure. Furthermore, we integrated D&D-seq with single-cell genotyping to assess the impact of IDH2 mutations on CTCF binding in a human clonal hematopoiesis sample, uncovering altered binding and chromatin co-accessibility patterns in mutant cells. Altogether, D&D-seq represents an important technological advance enabling the direct mapping of TF or chromatin remodeler binding to the DNA in primary human samples, opening new avenues for understanding chromatin and transcriptional regulation in health and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c35b9d0d36121406dcb7489958fdbc0f064e833" target='_blank'>
              Single-cell mapping of regulatory DNA:Protein interactions
              </a>
            </td>
          <td>
            Wei-Yu Chi, Sang-Ho Yoon, Levan Mekerishvili, S. Ganesan, Catherine Potenski, Franco Izzo, Dan A Landau, Ivan Raimondi
          </td>
          <td>2025-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b1f5a7ee68cb6b5638219d48a2b0e5c5d417966" target='_blank'>
              A complex systems approach to mosaic loss of the Y chromosome.
              </a>
            </td>
          <td>
            Xihan Guo, Xueqin Dai
          </td>
          <td>2024-12-16</td>
          <td>GeroScience</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Radiation therapy holds a key position in the arsenal of cancer treatment methods. This non-invasive technique has been actively used for several decades and has demonstrated high effectiveness in combating various types of malignant tumors. Despite significant advancements in ionizing radiation delivery technologies and the introduction of targeted radiosensitizing drugs and immunotherapy, classical radiation therapy faces limitations related to the radioresistance of tumor cells. This resistance is caused by numerous factors, such as genetic mutations, the metabolic characteristics of cancer cells, their ability to repair DNA, the presence of a tumor microenvironment, and many others. Tumor radioresistance reduces the success of treatment, making it necessary to explore new approaches to enhance the effectiveness of radiation therapy. This review discusses the main principles of radiation therapy and the properties of cancer cells that affect their radiosensitivity. It examines both existing methods for overcoming the radioresistance of cancer cells and prospects for further development, which could significantly improve the effectiveness of cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/322cff994342678fd900b945239ecb7a27067dde" target='_blank'>
              Mechanisms of Cancer Cell Radioresistance: Modern Trends and Research Prospects
              </a>
            </td>
          <td>
            M. Sharapov, E. Karmanova, S. V. Gudkov
          </td>
          <td>2024-12-15</td>
          <td>Biofizika</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="It is thought that cells surviving ionizing radiation exposure repair DNA double-strand breaks (DSBs) and restore their genomes. However, the recent biochemical and genetic characterization of DSB repair pathways reveals that only homologous recombination (HR) can function in an error-free manner and that the non-homologous end joining (NHEJ) pathways canonical NHEJ (c-NHEJ), alternative end joining (alt-EJ), and single-strand annealing (SSA) are error-prone, and potentially leave behind genomic scars and altered genomes. The strong cell cycle restriction of HR to S/G2 phases and the unparalleled efficiency of c-NHEJ throughout the cell cycle, raise the intriguing question as to how far a surviving cell "reaches" after repairing the genome back to its pre-irradiation state. Indeed, there is evidence that the genomes of cells surviving radiation treatment harbor extensive genomic alterations. To directly investigate this possibility, we adopted next-generation sequencing (NGS) technologies and tested a normal human fibroblast cell line, 82-6 hTert, after exposure up to 6 Gy. Cells were irradiated and surviving colonies expanded and the cells froze. Sequencing analysis using the Illumina sequencing platform and comparison with the unirradiated genome detected frequent genomic alterations in the six investigated radiation survivor clones, including translocations and large deletions. Translocations detected by this analysis and predicted to generate visible cytogenetic alterations were frequently (three out of five) confirmed using mFISH cytogenetic analysis. PCR analysis of selected deletions also confirmed seven of the ten examined. We conclude that cells surviving radiation exposure tolerate and pass to their progeny a wide spectrum of genomic alterations. This recognition needs to be integrated into the interpretation of biological results at all endpoints, as well as in the formulation of mathematical models of radiation action. NGS analysis of irradiated genomes promises to enhance molecular cytogenetics by increasing the spectrum of detectable genomic alterations and advance our understanding of key molecular radiobiological effects and the logic underpinning DSB repair. However, further developments in the technology will be required to harness its full potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b3247759666806f5cf74d149fcc038397e2c814" target='_blank'>
              NGS Detects Extensive Genomic Alterations in Survivors of Irradiated Normal Human Fibroblast Cells.
              </a>
            </td>
          <td>
            A. Soni, Daniela Beisser, E. Mladenov, Matthias Höller, Inken Wohlers, Vladimir Nikolov, S. Magin, Tamara Mussfeldt, Ludger Klein-Hitpass, Michael N. Cornforth, B. Loucas, Sven Rahmann, G. Iliakis
          </td>
          <td>2024-12-27</td>
          <td>Radiation research</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Significant progress in determining the molecular origins and resistance mechanisms of mantle cell lymphoma (MCL) has improved our understanding of the disease's clinical diversity. These factors greatly impact prognosis in MCL patients. Given the dynamic alterations in MCL clones and disease evolution, it is crucial to recognize high-risk prognostic factors at diagnosis and relapse. Clinical factors include a high MCL International Prognostic Index score with a high Ki-67 proliferation index; early disease progression within 24 months of first-line treatment; >3 prior lines of therapy at relapse; and an aggressive (blastoid or pleomorphic) histology. Molecular aberrations include dysregulated cyclin D1; an aberrant SOX11-CD70 axis; upregulated Musashi-2; MYC rearrangement; metabolic reprogramming; and epigenetic changes. Other factors contributing to high-risk MCL include an immune-depleted microenvironment and clone adaptability with complex chromosomal anomalies and somatic mutations in TP53, NSD2, CCND1, CDKN2A, BIRC3, SP140, KMT2D, NFkBIE, SMARCA4, and NOTCH2. Ultra-high-risk MCL is indicated by the coexistence of multiple high-risk prognostic factors in the relapse setting and can portend very short progression-free survival. As MCL treatments advance towards cellular therapies, resistance to anti-CD19 chimeric antigen receptor T-cell therapy is also observed. These findings necessitate revisiting the prognostic impact of high-risk factors, current management strategies, new bi- and tri-specific T-cell engagers, combination therapies, novel therapeutic targets, and next-generation clinical trials for high-risk MCL patients. This article provides a comprehensive update on recognizing and managing high-risk MCL, encompassing current practices and future directions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03364deb8d8c1c44077292ebb1c2031c1c0a3c37" target='_blank'>
              High-Risk MCL: Recognition and Treatment.
              </a>
            </td>
          <td>
            P. Jain, Michael L Wang
          </td>
          <td>2024-12-30</td>
          <td>Blood</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Organisms regulate gene expression in response to environmental cues, a process known as plasticity, to adjust to changing environments. Research into natural variation and the evolution of plasticity frequently studies cis-regulatory elements with theory suggesting they are more important evolutionarily than trans-regulatory elements. Genome-wide association studies have supported this idea, observing a predominance of cis-loci affecting plasticity. However, studies in structured populations provide a contrasting image, raising questions about the genetic architecture of natural variation in plasticity. To circumvent potential statistical difficulties present in genome-wide association studies, we mapped loci underlying transcriptomic plasticity in response to salicylic acid using recombinant inbred lines generated from two random Arabidopsis thaliana accessions. We detected extensive transgressive segregation in the salicylic acid response, suggesting that plasticity to salicylate in Arabidopsis is polygenic. Most loci (>75%) underlying this variation act in trans, especially for loci influencing plasticity. Trans-acting loci were enriched in genome hotspots, with predominantly small effect sizes distributed across many genes. This could potentially explain their under-discovery in genome-wide association studies. This work reveals a potentially important role for trans-acting loci in plastic expression responses, with implications for understanding plant adaptation to different environments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6242103d645ac263c51e56f8b61d6b0403cdeb3b" target='_blank'>
              Trans-regulatory loci shape natural variation of gene expression plasticity in Arabidopsis
              </a>
            </td>
          <td>
            Mariele Lensink, Grey Monroe, Dan Kliebenstein
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74432db7e7bddd7e5f3b4f353ed49acd74ffe66d" target='_blank'>
              Getting the right combination to break the epigenetic code.
              </a>
            </td>
          <td>
            Seda S. Tolu, A. Viny, Jennifer E Amengual, Barbara Pro, Susan E Bates
          </td>
          <td>2024-12-02</td>
          <td>Nature reviews. Clinical oncology</td>
          <td>1</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2fc69ff58eaa4bfa3b4fa917eb7dfeb9e6159b0" target='_blank'>
              Genomic and phenotypic stability of fusion-driven pediatric sarcoma cell lines
              </a>
            </td>
          <td>
            Merve Kasan, Florian H. Geyer, Jana Siebenlist, Martin Sill, R. Öllinger, Tobias Faehling, Enrique de Álava, D. Surdez, U. Dirksen, Ina Oehme, Katia Scotlandi, O. Delattre, Martina Müller-Nurasyid, R. Rad, Konstantin Strauch, T. G. Grünewald, F. Cidre-Aranaz
          </td>
          <td>2025-01-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7c220b52569d4d73c77f6958bccc6ea49d4592f" target='_blank'>
              Increased translation driven by non-canonical EZH2 creates a synthetic vulnerability in enzalutamide-resistant prostate cancer
              </a>
            </td>
          <td>
            Shankha S Chatterjee, J. Linares, Tania Cid-Diaz, A. Duran, Mohd Imran K Khan, Marta Osrodek, Nicholas J Brady, Miguel Reina-Campos, A. Marzio, V. B. Venkadakrishnan, Martin K. Bakht, F. Khani, Juan Miguel Mosquera, Brian D. Robinson, Jenna Moyer, O. Elemento, Andrew C. Hsieh, David W. Goodrich, D. Rickman, H. Beltran, J. Moscat, Maria T. Diaz-Meco
          </td>
          <td>2024-11-20</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="Chronic myeloid leukemia (CML) is a myeloproliferative disorder caused by the reciprocal translocation of the BCR-ABL1 oncogene, which activates tyrosine kinase resulting in uncontrolled cell proliferation and apoptosis suppression. Although Imatinib (IM) is an effective treatment, IM resistance remains a significant concern. Therefore, microRNAs (miRNAs) have emerged as potential alternative therapies. These short, non-coding RNAs (20-22 nucleotides) regulate gene expression by binding to the 3' untranslated region (3'UTR) of target genes. The study aims to identify miRNAs linked to CML, determine miRNA’s target genes, construct a protein-protein interaction (PPI) network, and analyze pathways and gene ontology. A literature search using the keywords “microRNA”, and “chronic myeloid leukemia” yielded relevant papers, which were screened and categorized into three groups: 1) miRNA associated with TKI resistance, 2) miRNA as biomarkers or leukemogenesis, and 3) miRNA as therapy. Target genes for the identified miRNAs were determined using DIANA tools and miRTarBase. Gene ontology and pathway analysis were conducted using DAVID, while PPI and network visualization were performed using STRING, Cytoscape, and ClueGO. Thirteen miRNAs were selected, targeting 782 genes and forming 16 clusters. ClueGO identified clusters associated with key biological processes, including G1/S cell cycle transition, miRNA-mediated gene silencing, hematopoietic stem cell differentiation, and apoptosis regulation. Target genes are also significant in the CML pathway and other cancer pathways, such as p53, ErbB, FoxO, autophagy, apoptosis, VEGF, TNF, and microRNA. Notably, hsa-miR-16 emerged as the most promising therapeutic and biomarker candidate for CML, highlighting its role in critical pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/082a3893f78953cb45d6e871eb9d7b48eccbc043" target='_blank'>
              Role of MicroRNAs in Chronic Myeloid Leukemia (CML) Treatment, Biomarkers, and Resistance to Tyrosine Kinase Inhibitor: A Bioinformatics-Based Analysis
              </a>
            </td>
          <td>
            Syarifah Faezah binti Syed Mohamad, Rabiatul Adawiyah Ramlee, M. H. Elias
          </td>
          <td>2024-11-21</td>
          <td>Journal of Tropical Life Science</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Resistance to chemotherapeutic agents poses a significant challenge in cancer treatment, particularly with doxorubicin, a widely used drug for various cancers, including breast cancer, leukaemia, osteosarcoma, and gastrointestinal cancers. This review aims to elucidate the critical role of microRNAs (miRNAs) in the development of doxorubicin resistance, focusing on their interactions with ATP-binding cassette (ABC) transporters. Despite extensive research, the molecular mechanisms governing doxorubicin resistance still need to be completed, particularly regarding the regulatory influence of miRNAs on ABC transporter expression. By analyzing current literature, this review identifies a notable gap: the lack of comprehensive insight into how specific miRNAs modulate the expression and activity of ABC transporters in cancer cells, contributing to doxorubicin resistance. We systematically examine recent findings on the interplay between miRNAs and ABC transporters, providing a detailed assessment of potential therapeutic strategies that leverage miRNA modulation to overcome drug resistance. Ultimately, this review underscores the significance of integrating miRNA research into existing therapeutic frameworks to enhance the efficacy of doxorubicin in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0410d7419f2b9e676f6aa38b55c81e86e45dae4" target='_blank'>
              MiRNAs function in the development of resistance against doxorubicin in cancer cells: targeting ABC transporters
              </a>
            </td>
          <td>
            Xin-Yan Lu, Hongxu Jin
          </td>
          <td>2024-11-29</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="RNA velocity has emerged as a popular approach for modeling cellular change along the phenotypic landscape but routinely omits regulatory interactions between genes. Conversely, methods that infer gene regulatory networks (GRNs) do not consider the dynamically changing nature of biological systems. To integrate these two currently disconnected fields, we present RegVelo, an end-to-end dynamic, interpretable, and actionable deep learning model that learns a joint model of splicing kinetics and gene regulatory relationships and allows us to perform in silico perturbation predictions. When applied to datasets of the cell cycle, human hematopoiesis, and murine pancreatic endocrinogenesis, RegVelo demonstrates superior predictive power for interactions and perturbation simulations, for example, compared to methods that focus solely on dynamics or GRN inference. To leverage RegVelo’s full potential, we studied the dynamics of zebrafish neural crest development and underlying regulatory mechanisms using our Smart-seq3 dataset and shared gene expression and chromatin accessibility measurements. Using RegVelo’s in silico perturbation predictions, validated by CRISPR/Cas9-mediated knockout and single-cell Perturb-seq, we establish transcription factor tfec as an early driver and elf1 as a novel regulator of pigment cell fate and propose a gene-regulatory circuit involving tfec and elf1 interactions via the toggle-switch model.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb495aa52d71e37d301ee44c41f833b2bc276eae" target='_blank'>
              RegVelo: gene-regulatory-informed dynamics of single cells
              </a>
            </td>
          <td>
            Weixu Wang, Zhiyuan Hu, P. Weiler, Sarah Mayes, Marius Lange, Jingye Wang, Zhengyuan Xue, Tatjana Sauka-Spengler, F. Theis
          </td>
          <td>2024-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Resistance to chemotherapy remains a major hurdle to the cure of Acute Myeloid Leukemia (AML) patients. Recent studies indicate a minority of malignant cells, termed drug-tolerant persisters (DTPs), stochastically upregulate stress pathways to evade cell death upon acute exposure to chemotherapy without acquiring new genetic mutations. This chemoresistant state is transient and the cells return to baseline after removal of chemotherapy. Yet, the mechanisms employed by DTPs to resist chemotherapy are not well understood and it is largely unknown whether these mechanisms are also seen in patients receiving chemotherapy. Here, we used leukemia cell lines, primary AML patient samples and samples from patients with AML receiving systemic chemotherapy to study the DTP state. We demonstrated that a subset of AML cells transiently increases membrane rigidity to resist killing due to acute exposure to Daunorubicin and Ara-C. Upon removal of the chemotherapy, membrane rigidity returned to baseline and the cells regained chemosensitivity. Although resistant to chemotherapy, the increased membrane rigidity, rendered AML cells more susceptible to T-cell mediated killing. Thus, we identified a novel mechanism by which DTP leukemic cells evade chemotherapy and a strategy to eradicate these persistent cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e81267e4f27d8857a181f0de9d74c11042a73092" target='_blank'>
              Acute myeloid leukemia drug tolerant persister cells survive chemotherapy by transiently increasing plasma membrane rigidity, that also increases their sensitivity to immune cell killing.
              </a>
            </td>
          <td>
            Yael Morgenstern, Jongbok Lee, Yoosu Na, Brandon Y Lieng, Nicholas S Ly, William D Gwynne, R. Hurren, Li Ma, Dakai Ling, M. Gronda, A. Arruda, A. Frisch, T. Zuckerman, Y. Ofran, Mark D. Minden, Li Zhang, Catherine O'Brien, Andrew T. Quaile, J. R. Montenegro-Burke, Aaron D Schimmer
          </td>
          <td>2024-11-21</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Abstract Background Amplification of the programmed cell death-ligand 1 gene (CD274) is highly prevalent and associated with a high response rate to immune checkpoint inhibitors (ICIs) in lymphomas, and is also a potential biomarker for ICI treatment of solid tumors. However, the efficacy of ICIs for solid tumors with CD274 amplification identified by comprehensive genomic profiling (CGP) has been unclear. We here examined ICI efficacy for solid tumors with CD274 amplification identified by CGP in a national database. Methods We retrospectively analyzed data from the Center for Cancer Genomics and Advanced Therapeutics database containing 60,155 CGP test results for individuals with solid tumors. Only clinical data from patients treated with ICIs alone (not those undergoing concomitant therapy with molecularly targeted or cytotoxic chemotherapeutic agents) were evaluated. We matched 48 patients in the CD274 amplification-positive group with 170 patients in the CD274 amplification-negative group in a 1:4 ratio based on tumor type, histology, treatment, and age. Overall survival (OS), time to next treatment (TTNT), and response rate were evaluated as treatment outcomes in the two groups. Results OS was similar in the CD274-amplified and matched CD274-non-amplified groups (median of 22.1 vs 26.3 months, respectively; HR of 0.92 with a 95% CI of 0.55 to 1.54; p=0.075). TTNT tended to be longer in the CD274-amplified group than in the matched CD274-non-amplified group (median of 16.5 vs 14.0 months; HR of 0.63 with a 95% CI of 0.37 to 1.08; p=0.091). The objective response rate was 33.3% and 18.4% (difference of 14.9%, with a 95% CI of –0.2% to 31.6%), and the disease control rate was 63.9% and 41.1% (difference of 22.8%, with a 95% CI of 5.1% to 40.4%), in the CD274-amplified and matched CD274-non-amplified groups, respectively. Conclusions The number of patients with solid tumors positive for CD274 amplification in this analysis is the largest to date, and our results suggest that such gene amplification may be associated with the outcome of ICI treatment in such individuals. CD274 amplification identified by CGP may therefore be a predictor of ICI efficacy for solid tumors. Trial registration number UMIN000029779.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e69b62b239b4cbdbf362661323ab2f6c2f830f1f" target='_blank'>
              Evaluation of immune checkpoint inhibitor efficacy for solid tumors with CD274 (PD-L1 gene) amplification identified by comprehensive genomic profiling: retrospective study based on a nationwide database
              </a>
            </td>
          <td>
            Tomohiro Nakayama, Takayuki Takahama, Y. Chiba, Naoki Shiraishi, H. Kawakami, K. Yonesaka, Kazuhiko Nakagawa, H. Hayashi
          </td>
          <td>2024-12-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="The phenomenon of adaptive mutation in bacteria presents several challenges to classical models of evolution, particularly regarding the observed coordination of mutation patterns across populations. Here, we examine statistical evidence from evolved Escherichia coli populations showing mutation enrichment up to 256-fold above background rates in key metabolic genes, with remarkable temporal stability that appears to transcend known bacterial communication mechanisms. We propose a theoretical framework suggesting that bacteria may utilize quantum coherent oscillations, potentially mediated through synchronized membrane potential fluctuations, to achieve this degree of coordination in adaptive mutation. Building on Sprouffske et al.’s findings that high mutation rates can limit adaptive evolution in E. coli, we examined mutation patterns in four key metabolic genes (pykF, topA, cspC, and rpoC) previously identified as targets of selection. While Sprouffske found that extremely high mutation rates impaired adaptation, our analysis reveals that these genes show non-random enrichment patterns (p < 1.76×10-34 for pykF) that maintain remarkable temporal stability across multiple timepoints. Recent advances in quantum biology have demonstrated sustained quantum coherence in biological systems, we present a model for how quantum-mediated bacterial signaling could potentially contribute to adaptive mutation. The framework makes several experimentally testable predictions about mutation patterns, population dynamics, and coherence times in bacterial populations under stress conditions. While substantial experimental validation remains necessary, this work provides specific approaches for investigating potential quantum contributions to bacterial adaptation, with implications for understanding evolutionary mechanisms and bacterial stress responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c15202aa7837996110092bdd84ad8961fcfb080a" target='_blank'>
              Could Quantum-Mediated Bacterial Signaling Explain Adaptive Mutation?
              </a>
            </td>
          <td>
            Patrick Ross
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7ddc2e4963aa356dbc3de332034a633ed4d7780" target='_blank'>
              Antisense oligonucleotides as a targeted therapeutic approach in model of acute myeloid leukemia.
              </a>
            </td>
          <td>
            D. Maceckova, L. Vaňková, J. Bufka, Petr Hošek, Jiří Moravec, Pavel Pitule
          </td>
          <td>2024-12-18</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Pancreatic cancer is the result of mutations in crucial genetic markers like KRAS and TP53 that make treatment challenging. This article discusses how CRISPR Cas9 technology can be combined with these markers to create treatments. CRISPR allows for the alteration or repair of these mutations, with the aim of restoring gene function or blocking cancer-causing pathways. For instance, CRISPR has the potential to fix mutations in TP53 or CDKN2A genes and restore SMAD4 signaling or target the KRAS oncogene in the body’s cells. However promising, it may be that CRISPR encounters obstacles like unintentional effects and challenges in effectively delivering it to pancreatic tumor cells. Furthermore, ethical concerns, especially related to the editing of the germline, need consideration. As techniques based on CRISPR advance, there is a chance for them to transform the treatment landscape for cancer by offering personalized therapies. More studies are needed to enhance how treatments are administered accurately and safely through methods and targeted testing for effectiveness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d180ed2014a91157721c0376d7e2ca0a9fe65e1" target='_blank'>
              Integrating CRISPR Technology with Key Genetic Markers in Pancreatic Cancer: A New Frontier in Targeted Therapies
              </a>
            </td>
          <td>
            Dimitrios Stefanoudakis
          </td>
          <td>2025-01-06</td>
          <td>SynBio</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="BRCA1, a critical tumor suppressor gene, plays an essential role in maintaining genomic stability through its involvement in DNA double-strand break repair, particularly via homologous recombination. Loss or impairment of BRCA1 function disrupts this repair pathway, resulting in genomic instability and significantly increased susceptibility to breast and ovarian cancers. To elucidate the molecular mechanisms by which BRCA1 mutations contribute to tumorigenesis, we employed quantitative mass spectrometry-based proximity labeling and affinity purification to identify cancer-specific protein-protein interactions (PPIs). Our integrated omics and visualization analysis of interactors revealed that the BRCA1-Y1853ter mutant, through its interaction with BARD1, perturbs the interactome and impacts cellular processes within the cytoplasm and nucleoplasm. Structural data further corroborated these findings, showing enhanced binding between the mutant BRCA1 and specific interactors, suggesting an altered functional profile. Together, these observations raise the hypothesis that the BRCA1-Y1853ter mutant may exhibit gain-of-function characteristics, providing new insights into the molecular and cellular effects of mutations in the BRCA1 C-Terminal (BRCT) domain and their implications for the pathogenesis of breast and ovarian cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7081d3dc4a7fe94ebd83524e06e557bb05ca412" target='_blank'>
              Differential Remodeling of the BRCA1 Interactome by a Unique Minimal Clinical Truncation Mutation
              </a>
            </td>
          <td>
            Jingjing Chen, Mihaela E. Sardiu, M. Rekowski, Jules Nde, Zachary Clark, S. Anant, Michael P. Washburn, Roy A Jensen
          </td>
          <td>2024-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="SUMMARY DNA, is assaulted by endogenous and exogenous agents that lead to the formation of damage. In order to maintain genome integrity DNA repair pathways must be efficiently activated to prevent mutations and deleterious chromosomal rearrangements. Conversely, genome rearrangement is also necessary to allow genetic diversity and evolution. The antagonist interaction between maintenance of genome integrity and rearrangements determines genome shape and organization. Therefore, it is of great interest to understand how the whole linear genome structure behaves upon formation and repair of DNA damage. For this, we used long reads sequencing technology to identify and to characterize genomic structural variations (SV) of wild‐type Arabidopsis thaliana somatic cells exposed either to UV‐B, to UV‐C or to protons irradiations. We found that genomic regions located in heterochromatin are more prone to form SVs than those located in euchromatin, highlighting that genome stability differs along the chromosome. This holds true in Arabidopsis plants deficient for the expression of master regulators of the DNA damage response (DDR), ATM (Ataxia‐telangiectasia‐mutated) and ATR (Ataxia‐telangiectasia‐mutated and Rad3‐related), suggesting that independent and alternative surveillance processes exist to maintain integrity in genic regions. Finally, the analysis of the radiations‐induced deleted regions allowed determining that exposure to UV‐B, UV‐C and protons induced the microhomology‐mediated end joining mechanism (MMEJ) and that both ATM and ATR repress this repair pathway.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52b13004cb07afbb75c796371a93fdfc391d332b" target='_blank'>
              Characterization of radiations‐induced genomic structural variations in Arabidopsis thaliana
              </a>
            </td>
          <td>
            Salimata Ousmane Sall, Abdelmalek Alioua, Sébastien Staerck, Stéfanie Graindorge, M. Pellicioli, Jacky Schuler, Catherine Galindo, Quentin Raffy, Marc Rousseau, J. Molinier
          </td>
          <td>2024-12-01</td>
          <td>The Plant Journal</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Chronic antigenic stimulation is central to marginal zone lymphoma (MZL) development. While the pharmacological inhibition of the B-cell receptor (BCR) signaling is initially effective, secondary resistance frequently develops. We conducted a CRISPR interference (CRISPRi) screen investigating enhancer-associated long non-coding RNAs (elncRNAs) in MZL cells to identify transcripts crucial to shape their dependence on BCR pathway activation. We identified LOC730338, an elncRNA associated with A-to-I RNA editing, which we renamed ADARreg. Silencing ADARreg re-sensitized tumor cells to BCR pathway inhibition by modulating ADAR2 nuclear translocation and altering intronic RNA editing. The process preferentially affected genes that enhance immune responses, such as STING, IRF3, and p65. ADARreg knockdown also increased lymphoma cell sensitivity to NK cell-mediated cytotoxicity. Our results indicate that targeting elncRNAs like ADARreg represents a potential strategy to overcome drug resistance in lymphoma, also opening new therapeutic opportunities for immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53b0f5b32756fe2cd1cc1438cd5631c98fcc3016" target='_blank'>
              Enhancer lncRNA LOC730338 limits the innate immune response against lymphoma cells, inhibiting ADAR2-dependent alternative transcription
              </a>
            </td>
          <td>
            Luciano Cascione, F. Guidetti, S. Ramnarayanan, Andrea Rinaldi, F. Spriano, Alex Zadro, C. Tarantelli, Nicolas Munz, A. Arribas, Rory Johnson, Francesco Bertoni, S. Napoli
          </td>
          <td>2024-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fc92ee9eeeb11fdc0010923fa936390ffdc668a" target='_blank'>
              Multi-omic and single-cell profiling of chromothriptic medulloblastoma reveals genomic and transcriptomic consequences of genome instability
              </a>
            </td>
          <td>
            Petr Smirnov, M. Przybilla, Milena Simovic-Lorenz, R. G. Parra, H. Susak, Manasi Ratnaparkhe, John K. L. Wong, V. Körber, Jan-Philipp Mallm, George Philippos, M. Sill, Thorsten Kolb, Rithu Kumar, Nicola Casiraghi, K. Okonechnikov, David R. Ghasemi, K. Maass, K. Pajtler, Anna Jauch, A. Korshunov, Thomas Höfer, M. Zapatka, Stefan M Pfister, W. Huber, O. Stegle, Aurélie Ernst
          </td>
          <td>2024-11-23</td>
          <td>Nature Communications</td>
          <td>2</td>
          <td>109</td>
        </tr>

        <tr id="BACKGROUND/AIM
Chemotherapy resistance is an important problem in the treatment of patients with cholangiocarcinoma (CCA) who are not eligible for surgery. This study aimed to overcome gemcitabine (Gem) resistance in CCA by investigating and targeting Gem resistance-associated molecules.


MATERIALS AND METHODS
Three stable Gem-resistant CCA cell lines (CCA-GemR) were established by gradually exposing CCA cell lines to Gem. The cells were characterized in terms of growth, cross-resistance to chemotherapeutic drugs, cell cycle distribution, and colony formation. The molecular mechanisms related to Gem resistance were assessed using a phosphorylation array assay and protein expression was confirmed using western blotting analysis. The targeted molecules were subsequently analyzed using PanDrugs to identify potential targeted therapies. The drug was used to enhance Gem sensitivity.


RESULTS
The results demonstrated that CCA-GemR cells grow more slowly compared to their parental cell lines. Cell cycle analysis revealed an increase in KKU-213A-GemR and KKU-213B-GemR cell accumulation in the G1 phase. Moreover, cross-resistance to 5-FU and cisplatin was observed in all CCA-GemR cells. The Proteome Profiler Human Phospho-Kinase Array showed increased phosphorylation of EGFR in CCA-GemR cells. Erlotinib, a specific inhibitor of EGFR, significantly enhanced the anti-tumor activity of Gem with a synergistic effect (Combination index <1). Western blot analysis confirmed that phosphorylation of EGFR increased in cells treated with Gem, whereas the expression was significantly decreased in cells treated with either erlotinib alone or in combination with Gem.


CONCLUSION
EGFR is a potential target molecule for reducing Gem resistance and enhancing its anti-tumor effects in patients with CCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82ce64739e1e89d1067e4ae8b135012799efcbdf" target='_blank'>
              Targeting EGFR Activation to Overcome Gemcitabine Resistance in Cholangiocarcinoma.
              </a>
            </td>
          <td>
            Sonexai Kidoikhammouan, Worachart Lert-itthiporn, Raksawan Deenonpoe, Charupong Saengboonmee, S. Obchoei, S. Wongkham, Wunchana Seubwai
          </td>
          <td>2024-12-01</td>
          <td>Anticancer research</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Resistance of BRAF-mutant melanomas to targeted therapy arises from the ability of cells to enter a persister state, evade treatment with relative dormancy, and repopulate the tumor when reactivated. A better understanding of the temporal dynamics and specific pathways leading into and out of the persister state is needed to identify strategies to prevent treatment failure. Using spatial transcriptomics in patient-derived xenograft models, we captured clonal lineage evolution during treatment. The persister state showed increased oxidative phosphorylation, decreased proliferation, and increased invasive capacity, with central-to-peripheral gradients. Phylogenetic tracing identified intrinsic and acquired resistance mechanisms (e.g., dual specific phosphatases, reticulon-4, and CDK2) and suggested specific temporal windows of potential therapeutic susceptibility. Deep learning-enabled analysis of histopathological slides revealed morphological features correlating with specific cell states, demonstrating that juxtaposition of transcriptomics and histological data enabled identification of phenotypically distinct populations from using imaging data alone. In summary, this study defined state change and lineage selection during melanoma treatment with spatiotemporal resolution, elucidating how choice and timing of therapeutic agents will impact the ability to eradicate resistant clones.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d694b41718972dca1b5b8aa26a4498984d65d307" target='_blank'>
              Spatiotemporal Profiling Defines Persistence and Resistance Dynamics During Targeted Treatment of Melanoma.
              </a>
            </td>
          <td>
            Jill C. Rubinstein, Sergii Domanskyi, Todd B Sheridan, Brian Sanderson, SungHee Park, Jessica Kaster, Haiyin Li, Olga Anczuków, M. Herlyn, Jeffrey H. Chuang
          </td>
          <td>2024-12-19</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>114</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5361264ba7fa5b03a05e9caae598d056bebbf71c" target='_blank'>
              Multi-omic profiling highlights factors associated with resistance to immuno-chemotherapy in non-small-cell lung cancer.
              </a>
            </td>
          <td>
            Yilv Yan, Dongqing Sun, Junjie Hu, Yue Chen, Liangdong Sun, Huansha Yu, Yicheng Xiong, Zhida Huang, Haoran Xia, Xinsheng Zhu, D. Bian, Fenghuan Sun, Likun Hou, Chunyan Wu, Orion R Fan, Haiyang Hu, An Zeng, Lele Zhang, Yi Eve Sun, Chenfei Wang, Peng Zhang
          </td>
          <td>2024-12-10</td>
          <td>Nature genetics</td>
          <td>1</td>
          <td>16</td>
        </tr>

        <tr id="Human T-cell leukemia virus type 1 (HTLV-1) broadly impacts host genes, affecting the infected cell population and inducing the development of a disease with a poor prognosis, adult T-cell leukemia-lymphoma (ATL). This study aimed to provide a comprehensive epigenomic characterization of the infected cell population and evaluated the transcriptome and chromatin structures of peripheral blood cells in HTLV-1-infected individuals using RNA sequencing (RNA-seq) and assay for transposase-accessible chromatin sequencing (ATAC-seq). The infected cells showed significant changes in gene expression patterns from the polyclonal stage and before ATL onset while demonstrating similarities to tumor-forming ATL cells. These similarities were a result of large-scale open chromatin changes, supporting the independent early formation of epigenomic aberrations as an underlying mechanism for later clonal propagation. This study also demonstrated that HTLV-1 Tax directly affects the host chromatin structure, thereby developing fundamental epigenomic characteristics. Several Tax target genes, including the RASGRP3-ERK pathway, were recognized, indicating an impact on signaling pathways. This genome-wide variability in chromatin structural property is a novel feature of HTLV-1 infection and may contribute to pathogenic mechanisms. In addition, it has crucial implications for better understanding the impact of HTLV-1 on the host genome and identifying novel therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99f7c450fea8921cdae499b880a5d0560a4c49aa" target='_blank'>
              Rewired chromatin structure and epigenetic gene dysregulation during HTLV-1 infection to leukemogenesis.
              </a>
            </td>
          <td>
            Jun Mizuike, Kako Suzuki, Shu Tosaka, Yuta Kuze, Seiichiro Kobayashi, Makoto Nakashima, Koji Jimbo, Y. Nannya, Yutaka Suzuki, K. Uchimaru, Makoto Yamagishi
          </td>
          <td>2024-11-19</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="The molecular basis of adaptive evolution and cancer progression are both complex processes that share many striking similarities. The potential adaptive significance of environmentally-induced epigenetic changes is currently an area of great interest in both evolutionary and cancer biology. In the field of cancer biology intense effort has been focused on the contribution of stress-induced non-coding RNAs (ncRNAs) in the activation of epigenetic changes associated with elevated mutation rates and the acquisition of environmentally adaptive traits. Examples of this process are presented and combined with more recent findings demonstrating that stress-induced ncRNAs are transferable from somatic to germline cells leading to cross-generational inheritance of acquired adaptive traits. The fact that ncRNAs have been implicated in the transient adaptive response of various plants and animals to environmental stress is consistent with findings in cancer biology. Based on these collective observations, a general model as well as specific and testable hypotheses are proposed on how transient ncRNA-mediated adaptive responses may facilitate the transition to long-term biological adaptation in both cancer and evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbfc2bb67d7c58db953ed76c627eb64e92c4eb6a" target='_blank'>
              Adaptive Significance of Non-coding RNAs: Insights from Cancer Biology.
              </a>
            </td>
          <td>
            John F. McDonald
          </td>
          <td>2025-01-07</td>
          <td>Molecular biology and evolution</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/188a74494b56066cf4346e450a3d56405973c270" target='_blank'>
              Single-cell multi-omics analysis reveals candidate therapeutic drugs and key transcription factor specifically for the mesenchymal subtype of glioblastoma
              </a>
            </td>
          <td>
            Yufan Yang, Ziyuan Liu, Yerong Wei, Shuai He, Ancheng Gu, Zhiyong Li, Jianlong Li, Zhongyuan Xu, B. Cen
          </td>
          <td>2024-12-20</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="INTRODUCTION
Acute myeloid leukemia (AML) is an aggressive and poor-prognosis blood cancer. Despite a low mutation burden compared to other cancers, AML is heterogenous and identifying robust therapeutic targets has been difficult. Genomic profiling has greatly advanced our understanding of AML, and has revealed targets for AML therapy. However, only 50% of AML patients have gene mutations that are currently druggable, and relapse rates remain high. The addition of proteomic profiling is emerging to address these challenges.


AREAS COVERED
Using references collected through Pubmed, we review recent studies that have combined genomic and proteomic profiling (i.e. proteogenomic profiling), as well as studies that have additionally integrated other omics approaches, such as phosphoproteomics. We highlight how proteogenomic profiling promises to deconvolve the cellular pathways driving leukemogenesis, uncover novel therapeutic targets, and identify biomarkers of response to novel and existing therapies.


EXPERT OPINION
Proteogenomic profiling is providing unparalleled insight into AML, and is beginning to identify robust biomarkers. Standardization of workflows will be required before mass spectrometry-based proteomic assays can be integrated into routine clinical use. However, the demonstrated ability to adapt signatures into biomarker panels that can be assayed by existing clinical workflows is enabling current clinical translation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00da146263cc574dc14279fe5c3bf5fbe1f370e6" target='_blank'>
              Proteogenomic profiling of acute myeloid leukemia to identify therapeutic targets.
              </a>
            </td>
          <td>
            Heather C Murray, Jonathan R. Sillar, Maddison Chambers, N. Verrills
          </td>
          <td>2024-11-22</td>
          <td>Expert review of proteomics</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Immune checkpoint inhibitors (ICIs) are extremely effective in a subgroup of mismatch repair-deficient (MMRd) cancers, but ∼50% remain resistant to treatment. We have shown for the first time that this may be due to the differential regulation of factors linked to response to ICIs upon loss of the different MMR genes. Here, we show that increased PD-L1 expression is observed upon loss of the MMR genes MLH1, MSH2 and PMS2. However, this is not true upon loss of MSH6. Here, we show that this is due to a novel role for MSH6 as a direct regulator of PD-L1 transcription, dependent on recruitment by the histone trimethyltransferase SETD2. Next-generation sequencing of MLH1 and MSH6 knockout (KO) cells revealed that MSH6 KO cells have significantly lower microsatellite instability in comparison to MLH1 KO cells, despite MSH6 KO cells having a higher mutational burden. These findings emphasise the need for gene-specific stratification in the MMRd cohort.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d6ede7d9d243c2456000d9151a205cf93b9af95" target='_blank'>
              The DNA Mismatch repair protein, MSH6 is a novel regulator of PD-L1 expression
              </a>
            </td>
          <td>
            Kirsten Brooksbank, Charlotte Smith, E. Maniati, Amy Gibson, Wai Yiu Tse, Amy Kate Hall, Jun Wang, T. Sharp, Sarah A Martin
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Epigenetic modifications play a pivotal role in cellular differentiation and the onset of complex diseases by regulating gene expression without altering the underlying DNA sequence. Mechanisms such as DNA methylation, histone modification, and chromatin remodeling orchestrate the transition from pluripotent stem cells to specialized cell types, ensuring tissue-specific functionality. This research delves into the intricate interplay between these epigenetic regulators and their involvement in developmental processes. Moreover, it examines how aberrations in epigenetic landscapes contribute to the pathogenesis of diseases such as cancer, neurodegenerative disorders, and autoimmune conditions. By employing advanced techniques like single-cell epigenomics and CRISPR-based epigenetic editing, this study aims to uncover novel biomarkers and therapeutic targets. The findings highlight the dynamic and reversible nature of epigenetic marks, offering promising avenues for personalized medicine. Understanding the role of epigenetic modifications in cellular differentiation and disease development is essential for advancing diagnostics, therapeutics, and our broader understanding of human biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05dc2e90b2c758b79765cdc1b955a8ef36a6ccb9" target='_blank'>
              The Role of Epigenetic Modifications in Cellular Differentiation and the Development of Complex Diseases
              </a>
            </td>
          <td>
            Rea Barlas
          </td>
          <td>2024-11-28</td>
          <td>Next Frontier For Life Sciences and AI</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Metastatic melanoma remains a major clinical challenge. Large-scale genomic sequencing of melanoma has identified bona fide activating mutations in RAC1, which are associated with resistance to BRAF-targeting therapies. Targeting the RAC1-GTPase pathway, including the upstream activator PREX2 and the downstream effector PI3Kβ, could be a potential strategy for overcoming therapeutic resistance, limiting melanoma recurrence, and suppressing metastatic progression. Here, we used genetically engineered mouse models and patient-derived BRAFV600E-driven melanoma cell lines to dissect the role of PREX2 in melanomagenesis and response to therapy. While PREX2 was dispensable for the initiation and progression of melanoma, its loss conferred sensitivity to clinically relevant therapeutics targeting the MAPK pathway. Importantly, genetic and pharmacological targeting of PI3Kβ phenocopied PREX2 deficiency, sensitizing model systems to therapy. These data reveal a druggable PREX2/RAC1/PI3Kβ signaling axis in BRAF-mutant melanoma that could be exploited clinically.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d015227301e9822a2f9c55706223dee9a9babf44" target='_blank'>
              Targeting the PREX2/RAC1/PI3Kβ Signaling Axis Confers Sensitivity to Clinically Relevant Therapeutic Approaches in Melanoma.
              </a>
            </td>
          <td>
            Catriona A. Ford, Dana Koludrovic, Patricia P. Centeno, M. Foth, Elpida Tsonou, Nikola Vlahov, Nathalie Sphyris, K. Gilroy, Courtney Bull, Colin Nixon, Bryan Serrels, Alison F. Munro, John C Dawson, Neil O. Carragher, Valeria Pavet, David C Hornigold, Philip D Dunne, Julian Downward, Heidi C. E. Welch, Simon T Barry, Owen J. Sansom, A. Campbell
          </td>
          <td>2024-12-05</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Repeated antigen exposure leads to T-cell exhaustion, a transcriptionally and epigenetically distinct cellular state marked by loss of effector functions (e.g., cytotoxicity, cytokine production/release), up-regulation of inhibitory receptors (e.g., PD-1), and reduced proliferative capacity. Molecular pathways underlying T-cell exhaustion have been defined for CD8+ cytotoxic T cells, but which factors drive exhaustion in CD4+ T cells, that are also required for an effective immune response against a tumor or infection, remains unclear. Here, we utilize quantitative proteomic, phosphoproteomic, and metabolomic analyses to characterize the molecular basis of the dysfunctional cell state induced by chronic stimulation of CD4+ memory T cells. We identified a dynamic response encompassing both known and novel up-regulated cell surface receptors, as well as dozens of unexpected transcriptional regulators. Integrated causal network analysis of our combined data predicts the histone acetyltransferase p300 as a driver of aspects of this phenotype following chronic stimulation, which we confirmed via targeted small molecule inhibition. While our integrative analysis also revealed large-scale metabolic reprogramming, our independent investigation confirmed a global remodeling away from glycolysis to a dysfunctional fatty acid oxidation-based metabolism coincident with oxidative stress. Overall, these data provide both insights into the mechanistic basis of CD4+ T-cell exhaustion and serve as a valuable resource for future interventional studies aimed at modulating T-cell dysfunction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b40307de70e1191dcea42e4f0307585ecc9c75a" target='_blank'>
              Multiomic profiling of chronically activated CD4+ T cells identifies drivers of exhaustion and metabolic reprogramming
              </a>
            </td>
          <td>
            Matthew L. Lawton, Melissa M. Inge, B. Blum, Erika L Smith-Mahoney, Dante Bolzan, Weiwei Lin, Christina Mcconney, Jacob Porter, Jarrod Moore, Ahmed Youssef, Yashasvi Tharani, X. Varelas, Gerald V Denis, Wilson W Wong, D. Padhorny, D. Kozakov, Trevor Siggers, S. Wuchty, Jennifer Snyder-Cappione, Andrew Emili
          </td>
          <td>2024-12-01</td>
          <td>PLOS Biology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="5-Methylcytosine (5mC) is a common source of somatic mutations. Deamination of 5mC to thymine generates a G/T mismatch, which occurs spontaneously and must be repaired prior to DNA replication to avoid mutation. We generated genetically engineered mice and cell lines to define DNA repair pathways that protect against methylation damage. We observed a low background mutation rate in mouse bone marrow or colon, typically 0.2-0.5 CG>TG mutations/genome/day. This increased 3-7 fold in cells lacking the glycosylase Methyl-binding domain 4 (Mbd4), one of the few glycosylases capable of excising thymine from G/T mismatches. We found no role for Thymine DNA glycosylase (Tdg) in methylation damage repair. Instead, our results support cooperation between Mbd4 and the mismatch repair (MMR) complex MutSα (Msh6:Msh2), evident through elevated rates of methylation damage in Msh6-deficient cells; increasing to 2.6-4.8 CG>TG mutations/genome/day in primary cells and up to 13.9 CG>TG mutations/genome/day in cell lines. Our findings support the view that MutSα has DNA repair activity outside of replication. While loss of Mbd4 elevates methylation damage selectively, the broader functionality of MutSα explains why mutational signatures linked to Msh6-deficiency are variable and reflect the replicative history of the cell.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d89af6527eda652ec94285879400d97b8fdfca77" target='_blank'>
              Mbd4 and MutSα protect cells from spontaneous deamination of 5-methylcytosine
              </a>
            </td>
          <td>
            R. Bilardi, C. Flensburg, Zhen Xu, Emily B. Derrick, Andrew Kueh, I. Majewski
          </td>
          <td>2024-12-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Perturb-seq enabled the profiling of transcriptional effects of genetic perturbations in single cells but lacks the ability to examine the impact on tissue environments. We present Perturb-DBiT for simultaneous co- sequencing of spatial transcriptome and guide RNAs (gRNAs) on the same tissue section for in vivo CRISPR screen with genome-scale gRNA libraries, offering a comprehensive understanding of how genetic modifications affect cellular behavior and tissue architecture. This platform supports a variety of delivery vectors, gRNA library sizes, and tissue preparations, along with two distinct gRNA capture methods, making it adaptable to a wide range of experimental setups. In applying Perturb-DBiT, we conducted un-biased knockouts of tens of genes or at genome-wide scale across three cancer models. We mapped all gRNAs in individual colonies and corresponding transcriptomes in a human cancer metastatic colonization model, revealing clonal dynamics and cooperation. We also examined the effect of genetic perturbation on the tumor immune microenvironment in an immune-competent syngeneic model, uncovering differential and synergistic perturbations in promoting immune infiltration or suppression in tumors. Perturb-DBiT allows for simultaneously evaluating the impact of each knockout on tumor initiation, development, metastasis, histopathology, and immune landscape. Ultimately, it not only broadens the scope of genetic inquiry, but also lays the groundwork for developing targeted therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0da363c9d27ca3668a40dc28fec3eb35286a497" target='_blank'>
              Spatially Resolved in vivo CRISPR Screen Sequencing via Perturb-DBiT
              </a>
            </td>
          <td>
            Alev Baysoy, Xiaolong Tian, Feifei Zhang, Paul Renauer, Zhiliang Bai, Hao Shi, Haikuo Li, Bo Tao, Mingyu Yang, Archibald Enninful, Fu Gao, Guangchuan Wang, Wanqiu Zhang, Thao Tran, N. H. Patterson, Shuozhen Bao, Chuanpeng Dong, Shan Xin, Mei Zhong, Sherri Rankin, C. Guy, Yan Wang, Jon Connelly, S. Pruett-Miller, Hongbo Chi, Sidi Chen, Rong Fan
          </td>
          <td>2024-11-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42786039d5d90ecc6efc329111435b0ff4acfda0" target='_blank'>
              Homologous recombination deficiency (HRD) diagnostics: underlying mechanisms and new perspectives.
              </a>
            </td>
          <td>
            Andrey Kechin, M. Koryukov, Regina Mikheeva, M. Filipenko
          </td>
          <td>2024-12-26</td>
          <td>Cancer metastasis reviews</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22e6a9e6b43769b80db465f9dd9f678e6c40ef5c" target='_blank'>
              Single cell and spatial transcriptomics highlight the interaction of club-like cells with immunosuppressive myeloid cells in prostate cancer
              </a>
            </td>
          <td>
            A. Kiviaho, Sini K. Eerola, H. Kallio, M. K. Andersen, Miina Hoikka, A. Tiihonen, I. Salonen, Xander Spotbeen, A. Giesen, C. T. Parker, S. Taavitsainen, Olli Hantula, Mikael Marttinen, Ismaïl Hermelo, Mazlina Ismail, Elise Midtbust, M. Wess, W. Devlies, Abhibhav Sharma, S. Krossa, Tomi Häkkinen, Ebrahim Afyounian, K. Vandereyken, Sam Kint, J. Kesseli, T. Tolonen, T.L.J. Tammela, T. Viset, Øystein Størkersen, G. Giskeødegård, M. B. Rye, T. Murtola, A. Erickson, Leena Latonen, G. Bova, Ian G. Mills, S. Joniau, J. Swinnen, Thierry Voet, T. Mirtti, Gerhardt Attard, Frank Claessens, T. Visakorpi, K. Rautajoki, May-Britt Tessem, Alfonso Urbanucci, M. Nykter
          </td>
          <td>2024-11-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="Introduction: The tumor microenvironment (TME) plays a pivotal role in cancer progression, influencing tumor initiation, growth, invasion, metastasis, and response to therapies. This study explores the dynamic interactions within the TME, particularly focusing on self-organization—a process by which tumor cells and their microenvironment reciprocally shape one another, leading to cancer progression and resistance. Understanding these interactions can reveal new prognostic markers and therapeutic targets within the TME, such as extracellular matrix (ECM) components, immune cells, and cytokine signaling pathways. Methods: A comprehensive search method was employed to investigate the current academic literature on TME, particularly focusing on self-organization in the context of cancer progression and resistance across the PubMed, Google Scholar, and Science Direct databases. Results: Recent studies suggest that therapies that disrupt TME self-organization could improve patient outcomes by defeating drug resistance and increasing the effectiveness of conventional therapy. Additionally, this research highlights the essential of understanding the biophysical properties of the TME, like cytoskeletal alterations, in the development of more effective malignancy therapy. Conclusion: This review indicated that targeting the ECM and immune cells within the TME can improve therapy effectiveness. Also, by focusing on TME self-organization, we can recognize new therapeutic plans to defeat drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6fe20eef20df9b82cbb5db576695d6323f5f64" target='_blank'>
              The role of tumor microenvironment and self-organization in cancer progression: Key insights for therapeutic development
              </a>
            </td>
          <td>
            M. Asadi, V. Zafari, Sanam Sadeghi-Mohammadi, D. Shanehbandi, Ufuk Mert, Zahra Soleimani, Ayse Caner, H. Zarredar
          </td>
          <td>2024-12-07</td>
          <td>BioImpacts</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Pediatric acute myeloid leukemia (pAML) is a clonal disease with recurrent genetic alterations that affect epigenetic states. However, the implications of epigenetic dysregulation in disease progression remain unclear. Here, we interrogated single-cell and clonal level chromatin accessibility of bone marrow samples from 28 pAML patients representing multiple subtypes using mtscATAC-seq, which revealed distinct differentiation hierarchies and abnormal chromatin accessibility in a subtype-specific manner. Innate immune signaling was commonly enhanced across subtypes and related to improved advantage of clonal competition and unfavorable prognosis, with further reinforcement in a relapse-associated leukemia stem cell-like population. We identified a panel of 31 innate immunity related genes to improve the risk classification of pAML patients. By comparing paired diagnosis and post-chemotherapy relapse samples, we showed that primitive cells significantly reduced MHC class II signaling, suggesting an immune evasion mechanism to facilitate their expansion at relapse. Key regulators orchestrating cell cycle dysregulation were identified to contribute to pAML relapse in drug-resistant clones. Our work establishes the single-cell chromatin accessibility landscape at clonal resolution and reveals the critical involvement of epigenetic disruption, offering insights into classification and targeted therapies of pAML patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e993d1844c9aff473d0effdd0e0a49f56c123ecc" target='_blank'>
              Single-cell epigenetic and clonal analysis decodes disease progression in pediatric acute myeloid leukemia.
              </a>
            </td>
          <td>
            Boyu Cui, Lanlan Ai, Minghui Lei, Yongjuan Duan, Chao Tang, Jingliao Zhang, Yan Gao, Xuan Li, Caiying Zhu, Yingchi Zhang, Xiaofan Zhu, Tomoya Isobe, Wenyu Yang, Berthold Gottgens, Ping Zhu
          </td>
          <td>2024-12-11</td>
          <td>Blood</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Cancer is a multifactorial process involving genetic, epigenetic, physiological, and metabolic changes. The ability of tumours to regulate new reactive pathways is essential for their survival. A key aspect of this involves the decision-making process of cancer cells as they balance the exploitation of surrounding and distant tissues for their own benefit while avoiding the rapid destruction of the host. Nutrition plays a central role in these processes but is inherently limited. Understanding how tumour cells interact with non-tumoural tissues to acquire nutrients is crucial. In this review, we emphasise the utility of Drosophila melanogaster as a model organism for dissecting the complex oncogenic networks underlying these interactions. By studying various levels—from individual tumour cells to systemic markers—we can gain new insights into how cancer adapts and thrives. Moreover, developing innovative technologies, such as high-throughput methods and metabolic interventions, enhances our ability to explore how tumours adapt to different conditions. These technological advances allow us to explore tumour adaptations and open new opportunities for potential therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edc422f043ced142c0695662e09ec939b17ed8e3" target='_blank'>
              Metabolic Adaptations in Cancer and the Host Using Drosophila Models and Advanced Tools
              </a>
            </td>
          <td>
            Ernesto Saez-Carrion, Mario Aguilar-Aragon, Lucía García-López, Maria Dominguez, M. L. Uribe
          </td>
          <td>2024-11-29</td>
          <td>Cells</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="ABSTRACT Diffuse large B‐cell lymphoma (DLBCL) patients that fail to achieve a complete metabolic response with frontline immunochemotherapy have a poor prognosis. Genomic profiling has led to a broader understanding of the molecular drivers in DLBCL, but it is unknown how well current classifiers identify patients that will experience primary treatment resistance (PTR). Using whole exome and RNA sequencing data from newly diagnosed DLBCL patients, we evaluated the genomic landscape of PTR and compared it to that of non‐PTR DLBCL. We found a significant increase in the frequency of TP53 (34% vs. 15%, p = 0.005) and ARID1A mutations (21% vs. 7%, p = 0.007) in PTR cases, with pathway analysis further demonstrating a downregulation of TP53 and an increase in chromatin modifying pathways. These results suggest that TP53 and ARID1A may be key mediators of PTR and important pathways contributing to the poor outcomes. We found that the current molecular classifiers were unable to identify PTR cases at diagnosis. However, our newly identified high‐risk signature identified 46% of PTR cases at diagnosis. Overall, these results contribute to our understanding of the genomic landscape of patients with primary treatment resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8dad9b8db809f309807b25afc6dc6fdb4c12b2eb" target='_blank'>
              Molecular Features of Diffuse Large B‐Cell Lymphoma Associated With Primary Treatment Resistance
              </a>
            </td>
          <td>
            A. M. Bock, K. Wenzl, J. Novak, M. Stokes, M. Hopper, J. Krull, A. Dropik, V. Sarangi, María Ortiz, Nicholas Stong, C. Huang, M. Maurer, Rebecca L. King, U. Farooq, Yucai Wang, T. Witzig, S. Ansell, T. Habermann, J. Cerhan, A. K. Gandhi, Grzegorz Nowakowski, A. Novak
          </td>
          <td>2024-11-29</td>
          <td>Hematological Oncology</td>
          <td>0</td>
          <td>128</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bc96a135b65d18f94baeeec7bb6f83c6f3c5c8d" target='_blank'>
              Landscape of antisense genes in the human genome and identification of new human hepatic antisense RNAs by long-read sequencing
              </a>
            </td>
          <td>
            J. Rojo-Carrillo, Pedro Garrido-Rodríguez, Maria Llamas-López, Rosa Cifuentes-Riquelme, Jose Padilla, Bruno Ramos-Molina, M. L. Lozano, B. de la Morena-Barrio, M. E. de la Morena-Barrio, Javier Corral
          </td>
          <td>2024-11-27</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="In recent years, gene editing technologies have rapidly evolved to enable precise and efficient genomic modification. These strategies serve as a crucial instrument in advancing our comprehension of genetics and treating genetic disorders. Of particular interest is the manipulation of large DNA fragments, notably the insertion of large fragments, which has emerged as a focal point of research in recent years. Nevertheless, the techniques employed to integrate larger gene fragments are frequently confronted with inefficiencies, off-target effects, and elevated costs. It is therefore imperative to develop efficient tools capable of precisely inserting kilobase-sized DNA fragments into mammalian genomes to support genetic engineering, gene therapy, and synthetic biology applications. This review provides a comprehensive overview of methods developed in the past five years for integrating large DNA fragments with a particular focus on burgeoning CRISPR-related technologies. We discuss the opportunities associated with homology-directed repair (HDR) and emerging CRISPR-transposase and CRISPR-recombinase strategies, highlighting their potential to revolutionize gene therapies for complex diseases. Additionally, we explore the challenges confronting these methodologies and outline potential future directions for their improvement with the overarching goal of facilitating the utilization and advancement of tools for large fragment gene editing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05aaf5b1a8b083f9c6bd065129f6dea59d703d40" target='_blank'>
              CRISPR/Cas-Based Gene Editing Tools for Large DNA Fragment Integration.
              </a>
            </td>
          <td>
            Shuhan Yang, Guang Hu, Jianming Wang, Jie Song
          </td>
          <td>2024-12-16</td>
          <td>ACS synthetic biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Currently, numerous associations between genetic polymorphisms and various diseases have been characterized through Genome-Wide Association Studies. The majority of clinically significant polymorphisms are localized in non-coding regions of the genome. While modern bioinformatic resources make it possible to predict molecular mechanisms that explain the influence of non-coding polymorphisms on gene expression, such hypotheses require experimental verification. This review discusses the methods for elucidating the molecular mechanisms underlying the dependence of disease pathogenesis on specific genetic variants within non-coding sequences. A particular focus is on the methods to identify the transcription factors with binding efficiency contingent upon polymorphic variations. Despite remarkable progress in the bioinformatic resources enabling the prediction of the impact of polymorphisms on disease pathogenesis, the imperative for experimental approaches to this inquiry still persists.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0e300f57318f3e11948f7c4bd85a221591ff36c" target='_blank'>
              Methods for Functional Characterization of Genetic Polymorphisms of Non-Coding Regulatory Regions of the Human Genome (Review)
              </a>
            </td>
          <td>
            A. Uvarova, E. A. Tkachenko, E. M. Stasevich, E. A. Zheremyan, K. Korneev, D. V. Kuprash
          </td>
          <td>2024-12-25</td>
          <td>Biohimiâ</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Introduction Despite remarkable progress in Pediatric Acute Myeloid Leukemia (pAML) treatments, the relapsed disease remains difficult to treat, making it pertinent to identify novel biomarkers of prognostic/therapeutic significance. Material and methods Bone marrow samples from 21 pAML patients were analyzed using single cell RNA sequencing, functional assays with ARMH1 knockdown and overexpression were performed in leukemia cell lines to evaluate impact on proliferation and migration, and chemotherapy sensitivity. Mitochondrial function was assessed via Seahorse assay, ARMH1 interacting proteins were studied using co-immunoprecipitation. Bulk RNA-seq was performed on ARMH1knockdown and over expressing cell lines to evaluate the pathways and networks impacted by ARMH1. Results Our data shows that ARMH1, a novel cancer-associated gene, is highly expressed in the malignant blast cells of multiple pediatric hematologic malignancies, including AML, T/B-ALL, and T/B-MPAL. Notably, ARMH1 expression is significantly elevated in blast cells of patients who relapsed or have a high-risk cytogenetic profile (MLL) compared to standard-risk (RUNX1, inv (16)). ARMH1 expression is also significantly correlated with the pediatric leukemia stem cell score of 6 genes (LSC6) associated with poor outcomes. Perturbation of ARMH1 (knockdown and overexpression) in leukemia cell lines significantly impacted cell proliferation and migration. The RNA-sequencing analysis on multiple ARMH1 knockdown and overexpressing cell lines established an association with mitochondrial fatty acid synthesis and cell cycle pathways.The investigation of the mitochondrial matrix shows that pharmacological inhibition of a key enzyme in fatty acid synthesis regulation, CPT1A, resulted in ARMH1 downregulation. ARMH1 knockdown also led to a significant reduction in CPT1A and ATP production as well as Oxygen Consumption Rate. Our data indicates that downregulating ARMH1 impacts cell proliferation by reducing key cell cycle regulators such as CDCA7 and EZH2. Further, we also established that ARMH1 is a key physical interactant of EZH2, associated with multiple cancers. Conclusion Our findings underscore further evaluation of ARMH1 as a potential candidate for targeted therapies and stratification of aggressive pAML to improve outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f9c3fb31bae52e61839b92c250ccd940a2b13b6" target='_blank'>
              ARMH1 is a novel marker associated with poor pediatric AML outcomes that affect the fatty acid synthesis and cell cycle pathways
              </a>
            </td>
          <td>
            Mojtaba Bakhtiari, Sean C Jordan, Hope L. Mumme, Richa Sharma, Mala Shanmugam, Swati S. Bhasin, Manoj K. Bhasin
          </td>
          <td>2024-12-05</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1708204ce5d02d098867433430233edb7d10ec3" target='_blank'>
              Profiling the epigenome using long-read sequencing.
              </a>
            </td>
          <td>
            Tianyuan Liu, Ana Conesa
          </td>
          <td>2025-01-08</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Cellular differentiation is controlled by intricate layers of gene regulation, involving the modulation of gene expression by various transcriptional regulators. Due to the complexity of gene regulation, identifying master regulators across the differentiation trajectory has been a longstanding challenge. To tackle this problem, a computational framework, single-cell Boolean network inference and control (BENEIN), is presented. Applying BENEIN to human large intestinal single-cell transcriptome data, MYB, HDAC2, and FOXA2 are identified as the master regulators whose inhibition induces enterocyte differentiation. It is found that simultaneous knockdown of these master regulators can revert colorectal cancer cells into normal-like enterocytes by synergistically inducing differentiation and suppressing malignancy, which is validated by in vitro and in vivo experiments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19c7fb2686fa44d6df149bd4d9108d57834cc43d" target='_blank'>
              Control of Cellular Differentiation Trajectories for Cancer Reversion.
              </a>
            </td>
          <td>
            Jeong-Ryeol Gong, Chun-Kyung Lee, Hoon-Min Kim, Juhee Kim, Jaeog Jeon, Sunmin Park, Kwang-Hyun Cho
          </td>
          <td>2024-12-11</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Cancer is a complex and multifaceted group of diseases characterized by highly intricate mechanisms of tumorigenesis and tumor progression, which complicates diagnosis, prognosis, and treatment. In recent years, targeted therapies have gained prominence by focusing on specific mutations and molecular features unique to each tumor type, offering more effective and personalized treatment options. However, it is equally critical to explore the genetic commonalities across different types of cancer, which has led to the rise of pan-cancer studies. These approaches help identify shared therapeutic targets across various tumor types, enabling the development of broader and potentially more widely applicable treatment strategies. This review aims to provide a comprehensive overview of key concepts related to tumors, including tumorigenesis processes, the tumor microenvironment, and the role of extracellular vesicles in tumor biology. Additionally, we explore the molecular interactions and mechanisms driving tumor progression, with a particular focus on the pan-cancer perspective. To achieve this, we conducted an in silico analysis using publicly available datasets, which facilitated the identification of both common and divergent genetic and molecular patterns across different tumor types. By integrating these diverse areas, this review offers a clearer and deeper understanding of the factors influencing tumorigenesis and highlights potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a5b3ea0ad38a337f0df54ee493320c36455e356" target='_blank'>
              Exploring the Complexity of Pan-Cancer: Gene Convergences and in silico Analyses
              </a>
            </td>
          <td>
            Leandro Teodoro, Ana Claudia Carreira, M.C. Sogayar
          </td>
          <td>2024-12-01</td>
          <td>Breast Cancer : Targets and Therapy</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/103b75e315d591fa4510717dd10f578400ede160" target='_blank'>
              Central control of dynamic gene circuits governs T cell rest and activation.
              </a>
            </td>
          <td>
            Maya M. Arce, Jennifer M. Umhoefer, Nadia Arang, Sivakanthan Kasinathan, Jacob W. Freimer, Zachary Steinhart, Haolin Shen, Minh T N Pham, Mineto Ota, Anika Wadhera, Rama Dajani, Dmytro Dorovskyi, Y. Chen, Qi Liu, Yuan Zhou, D. Swaney, K. Obernier, B. Shy, J. Carnevale, Ansuman T. Satpathy, N. Krogan, Jonathan K. Pritchard, Alex Marson
          </td>
          <td>2024-12-11</td>
          <td>Nature</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7fa17db238d722caee80a374931a8a07aedd8d3" target='_blank'>
              ASXL1 truncating variants in BOS and myeloid leukemia drive shared disruption of Wnt-signaling pathways but have differential isoform usage of RUNX3
              </a>
            </td>
          <td>
            Isabella Lin, Zain Awamleh, Mili Sinvhal, Andrew Wan, Leroy Bondhus, Angela Wei, Bianca E Russell, Rosanna Weksberg, VA Arboleda
          </td>
          <td>2024-11-29</td>
          <td>BMC Medical Genomics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="DNA damage can lead to mutations that can alter the function of oncogenes or tumor suppressor genes, thus promoting the development of cancer. p53 plays a multifaceted and complex role in the DNA damage response and cancer progression and is known as the ‘guardian of the gene’. When DNA damage occurs, p53 is activated through a series of post-translational modifications, which stabilize the protein and enhance its function as a transcription factor. It regulates processes including cell cycle checkpoints, DNA repair and apoptosis, thereby preventing the spread of damaged DNA and maintaining genome integrity. On the one hand, p53 can initiate cell cycle arrest and induce cells to enter the G1/S and G2/M checkpoints, preventing cells with damaged DNA from continuing to proliferate and gaining time for DNA repair. At the same time, p53 can promote the activation of DNA repair pathways, including base excision repair, nucleotide excision repair and other repair pathways, to ensure the integrity of genetic material. If the damage is too severe to repair, p53 will trigger the apoptosis process to eliminate potential cancer risks in time. p53 also plays a pivotal role in cancer progression. Mutations in the p53 gene are frequently found in many cancers, and the mutated p53 not only loses its normal tumor suppressor function but may even acquire pro-cancer activity. Therefore, we also discuss therapeutic strategies targeting the p53 pathway, such as the use of small-molecule drugs to restore the function of wild-type p53, the inhibition of negative regulatory factors and synthetic lethality approaches for p53-deficient tumors. This review therefore highlights the important role of p53 in maintaining genomic stability and its potential in therapeutic strategies for cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b423291d0566fe3b6571def47ce2c8576918e9f7" target='_blank'>
              Unraveling the Guardian: p53’s Multifaceted Role in the DNA Damage Response and Tumor Treatment Strategies
              </a>
            </td>
          <td>
            Han Zhang, Jianxiong Xu, Yuxuan Long, Ayitila Maimaitijiang, Zhengding Su, Wenfang Li, Jinyao Li
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Aggressive variant prostate cancer (AVPC) is characterized by a molecular signature involving combined defects in TP53, RB1, and/or PTEN (AVPC-TSGs), identifiable through immunohistochemistry or genomic analysis. The reported prevalence of AVPC-TSG alterations varies widely, reflecting differences in assay sensitivity, treatment pressure, and disease stage evolution. Although robust clinical evidence is still emerging, the study of AVPC-TSG alterations in prostate cancer (PCa) is promising. Alterations in TP53, RB1, and PTEN, as well as the combined loss of AVPC-TSGs, may have significant implications for prognosis and treatment. These biomarkers might help predict responses to various therapies, including hormonal treatments, cytotoxic agents, radiotherapy, and targeted therapies. Understanding the impact of these molecular alterations in patients with PCa is crucial for personalized management. In this review, we provide a comprehensive overview of the emerging prognostic and predictive roles of AVPC-TSG alterations across PCa stages. Moreover, we discuss the implications of different methods used for detecting AVPC-TSG alterations and summarize factors influencing their prevalence. As our comprehension of the genomic landscape of PCa disease deepens, incorporating genomic profiling into clinical decision making will become increasingly important for improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc76168bc6cc3f716d4ad9c939472b94406c6f4f" target='_blank'>
              The Emerging Predictive and Prognostic Role of Aggressive-Variant-Associated Tumor Suppressor Genes Across Prostate Cancer Stages
              </a>
            </td>
          <td>
            Martino Pedrani, Jessica Barizzi, Giuseppe Salfi, Alessandro Nepote, Irene Testi, Sara Merler, Luis Castelo-Branco, R. P. Mestre, Fabio Turco, Luigi Tortola, Jean-Philippe P. Theurillat, S. Gillessen, U. Vogl
          </td>
          <td>2025-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ff4ae0eaa9128342a0dedd747e19622d92c7f71" target='_blank'>
              FDFT1 maintains glioblastoma stem cells through activation of the Akt pathway
              </a>
            </td>
          <td>
            Hui Mo, Jiajia Shao, Zhun Li, Peiting Zeng, Xinke Yin, Yongsheng Huang, Peng Wang, Jianwei Liao
          </td>
          <td>2024-12-20</td>
          <td>Stem Cell Research & Therapy</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dce415c1d6cd65bcfca677d71376e494061ba037" target='_blank'>
              Genome-wide profiling of DNA repair proteins in single cells
              </a>
            </td>
          <td>
            Kim L. de Luca, Pim M. J. Rullens, Magdalena A. Karpinska, Sandra S. de Vries, Agnieszka Gacek-Matthews, L. Pongor, G. Legube, Joanna W. Jachowicz, A. M. Oudelaar, Jop Kind
          </td>
          <td>2024-11-21</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Mutations in the Nucleophosmin-1 (NPM1) gene are among the most common molecular aberrations in acute myeloid leukemia (AML). Various studies have established mutant NPM1 (mNPM1) as a faithful molecular measurable residual disease (MRD) marker with prognostic significance. Assessment of prognostic mNPM1 is included in the European LeukemiaNet (ELN) recommendations on MRD detection in AML. Due to recent advancements of promising drugs targeting mNPM1 AML, monitoring of mNPM1 MRD has gained interest, and is generally done by reverse transcriptase quantitative PCR (RT-qPCR). However, these RT-qPCR assays use cDNA as input, are based on gene expression levels of mNPM1, and are generally limited to specific mNPM1 gene variants. The main advantages of next-generation sequencing (NGS) using genomic DNA as input are stability, independence of gene expression levels, and the ability to detect any NPM1 variant in a single assay. Here we comprehensively investigated the applicability of NGS on DNA to detect mNPM1 MRD in a cohort of 119 (cDNA) and 310 (DNA) mNPM1 AML patients in complete remission after two cycles of induction chemotherapy. We demonstrate high correlations in levels and prognostic value between RT-qPCR/cDNA and NGS/DNA approaches, postulating NGS/DNA as an attractive alternative to RT-qPCR. We report that the 2% mNPM1/ABL1 threshold by RT-qPCR/cDNA corresponds to a NGS/DNA variant allele frequency (VAF) of 0.01%. The NGS/DNA threshold of >0.01% after two cycles of induction chemotherapy identifies significantly more AML patients with an increased relapse risk than current RT-qPCR/cDNA assays. The prognostic significance of mNPM1 MRD appears greatest in FLT3-ITD AML patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29543a49300e06642677b500be09c4ca9fc87fd1" target='_blank'>
              Advantages of a genomic DNA-based NGS assay for detection of mutant NPM1 measurable residual disease in AML.
              </a>
            </td>
          <td>
            C. Vonk, T. Grob, M. Rijken, F. Kavelaars, J. Konijnenburg, Gert J. Ossenkoppele, M. Manz, Yngvar Floisand, Bob Löwenberg, Peter J M Valk
          </td>
          <td>2024-12-05</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Cancer stem cells are characterized by self-renewal, clonal tumor initiation capacity, and treatment resistance, which play essential roles in the tumor progression of prostate cancer (PCa). In this study, we aim to explore the features of cancer stemness and characterize the expression of stem cell-related genes for PCa. Methods We downloaded RNA-seq data and related clinical information from The Cancer Genome Atlas (TCGA) database. The mRNA stemness index (mRNAsi) was analyzed for various clinical features, overall survival (OS), and disease-free survival (DFS), and a weighted gene co-expression network analysis (WGCNA) was performed to identify crucial gene modules and key genes, which may play a role in CSCs. The key gene functions were verified using multiple databases, including the TCGA and Gene Expression Omnibus database (GEO). Next, we explored the potential function of the modules and genes obtained using WGCNA using an enrichment analysis. Finally, we performed in vitro experiments for further verification. Results We found that mRNAsi were higher in PCa tissues than in normal tissues, and the mRNAsi were closely related to the clinical characteristics of PCa. A total of 16 key genes associated with the mRNAsi scores were identified by WGCNA analysis, including NCAPG, NEK2, DLGAP5, CENPA, CENPF, TPX2, GTSE1, KIF4A, NEIL3, CDC25C, UBE2C, CDCA5, MELK, SKA3, NUF2, and BIRC5. These genes were explicitly highly expressed in PCa across TCGA cancers and were validated in 3 independent GEO PCa datasets. The functional annotations of the key genes were linked with the cell proliferation processes. NUF2 may be a potential biomarker for PCa. In vitro experiments showed that knockdown NUF2 reduced the proliferation and migration of PCa cells. Conclusion The 16 key genes identified in this study significantly correlate with PCa stem cell characteristics and showed prognosis-oriented effects in PCa patients. Further, the NUF2 gene may be used as a drug target for treating PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72bc9466fa18820a99adb15c098ec91082dee0a3" target='_blank'>
              NUF2 is associated with cancer stem cell characteristics and a potential drug target for prostate cancer
              </a>
            </td>
          <td>
            Dongxu Zhang, Pu Liang, Qi Wang, Bowen Xia, Liqian Yu, Xiaopeng Hu
          </td>
          <td>2024-12-05</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Colorectal cancer (CRC) has increased at an alarming rate amongst younger (< 50 years) individuals. Such early-onset colorectal cancer (EOCRC) has been particularly notable within the Hispanic/Latino population. Yet, this population has not been sufficiently profiled in terms of two critical elements of CRC -- the MYC proto-oncogene and WNT signaling pathway. Here, we performed a comprehensive multi-omics analysis on 30 early-onset and 37 late-onset CRC ([≥] 50 years) samples from Hispanic/Latino patients. Our analysis included DNA exome sequencing for somatic mutations, somatic copy number alterations, and global and local genetic similarity. Using RNA sequencing, we also assessed differential gene expression, cellular pathways, and gene fusions. We then compared our findings from early-onset Hispanic/Latino patient samples with publicly available data from Non-Hispanic White cohorts. Across all early-onset patients, which had a median 1000 Genomes Project Peruvian-in-Lima-like (1KG-PEL-like) genetic similarity proportion of 60%, we identified 41 WNT pathway genes with significant mutations. Six important examples were APC, TCF7L2, DKK1, DKK2, FZD10, and LRP5. Notably, patients with mutations in DKK1 and DKK2 had the highest 1KG-PEL-like proportion (79%). When we compared the Hispanic/Latino cohort to the Non-Hispanic White cohorts, four of these key genes -- DKK1, DKK2, FZD10, and LRP5 -- were significant in both risk association analyses and differential gene expression. Interestingly, early-onset tumors (vs. late-onset) exhibited distinct somatic copy number alterations and gene expression profiles; the differences included MYC and drug-targetable WNT pathway genes. We also identified a novel WNT gene fusion, RSPO3, in early-onset tumors; it was associated with enhanced WNT signaling. This integrative analysis underscores the distinct molecular features of EOCRC cancer in the Hispanic/Latino population; reveals potential avenues for tailored precision medicine therapies; and emphasizes the importance of multi-omics approaches in studying colorectal carcinogenesis. We expect this data to help contribute towards reducing cancer health disparities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62d0668a8b4dd0379f87ebaa20ad3d4dabb63bc8" target='_blank'>
              Assessment of MYC Gene and WNT Pathway Alterations in Early-Onset Colorectal Cancer Among Hispanic/Latino Patients Using Integrated Multi-Omics Approaches
              </a>
            </td>
          <td>
            F. Carranza, B. Waldrup, Y. Yin, Y. Amzaleg, M. Postel, D. Craig, J. Carpten, B. Salhia, D. Hernandez, N. Gutierrez, C. Ricker, J. Culver, C. Chavez, M. Stern, L. Baezconde-Garbanati, H.-J. Lenz, E. Velazquez-Villarreal
          </td>
          <td>2024-12-08</td>
          <td>None</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="A significant number of castration-resistant prostate cancer (CRPC) evolve into a neuroendocrine (NE) subtype termed NEPC, leading to resistance to androgen receptor (AR) pathway inhibitors and poor clinical outcomes. Through Hi-C analyses of a panel of patient-derived xenograft tumors, here we report drastically different 3D chromatin architectures between NEPC and CRPC samples. Such chromatin re-organization was faithfully recapitulated in vitro on isogenic cells undergoing NE transformation (NET). Mechanistically, neural transcription factor (TF) NKX2-1 is selectively and highly expressed in NEPC tumors and is indispensable for NET across various models. NKX2-1 preferentially binds to gene promoters, but it interacts with chromatin-pioneering factors such as FOXA2 at enhancer elements through chromatin looping, further strengthening FOXA2 binding at NE enhancers. Conversely, FOXA2 mediates regional DNA demethylation, attributing to NE enhancer priming and inducing NKX2-1 expression, forming a feed-forward loop. Single-cell multiome analyses of isogenic cells over time-course NET cells identify individual cells amid luminal-to-NE transformation, exhibiting intermediate epigenetic and transcriptome states. Lastly, NKX2-1/FOXA2 interacts with, and recruits CBP/p300 proteins to activate NE enhancers, and pharmacological inhibitors of CBP/p300 effectively blunted NE gene expression and abolished NEPC tumor growth. Thus, our study reports a hierarchical network of TFs governed by NKX2-1 in regulating the 2D and 3D chromatin re-organization during NET and uncovers a promising therapeutic approach to eradicate NEPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0f60ea05e69028e7c615b1e7620f9deafcd0cdd" target='_blank'>
              Epigenetic remodeling and 3D chromatin reorganization governed by NKX2-1 drive neuroendocrine prostate cancer
              </a>
            </td>
          <td>
            Xiaodong Lu, Viriya Keo, Irina Cheng, Wanqing Xie, Galina A Gritsina, Juan Wang, Q. Jin, Peng Jin, Feng Yue, Martin G. Sanda, Victor Corces, Nicolas Altemose, Jonathan C. Zhao, Jindan Yu
          </td>
          <td>2024-12-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Cancer arises through an evolutionary process in which somatic mutations, including single nucleotide variants (SNVs) and copy number aberrations (CNAs), drive the development of a malignant, heterogeneous tumor. Reconstructing this evolutionary history from sequencing data is critical for understanding the order in which mutations are acquired and the dynamic interplay between different types of alterations. Advances in modern whole genome single-cell sequencing now enable the accurate inference of copy number profiles in individual cells. However, the low sequencing coverage of these low pass sequencing technologies poses a challenge for reliably inferring the presence or absence of SNVs within tumor cells, limiting the ability to simultaneously study the evolutionary relationships between SNVs and CNAs. In this work, we introduce a novel tumor phylogeny inference method, Pharming, that jointly infers the evolutionary histories of SNVs and CNAs. Our key insight is to leverage the high accuracy of copy number inference methods and the fact that SNVs co-occur in regions with CNAs in order to enable more precise tumor phylogeny reconstruction for both alteration types. We demonstrate via simulations that Pharming outperforms state-of-the-art single-modality tumor phylogeny inference methods. Additionally, we apply Pharming to a triple-negative breast cancer case, achieving high-resolution, joint reconstruction of CNA and SNV evolution, including the de novo detection of a clonal whole-genome duplication event. Thus, Pharming offers the potential for more comprehensive and detailed tumor phylogeny inference for high-throughput, low-coverage single-cell DNA sequencing technologies compared to existing approaches. Availability https://github.com/elkebir-group/Pharming">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d64f3b951403db46ae5a215e58c613919dd9b93c" target='_blank'>
              Pharming: Joint Clonal Tree Reconstruction of SNV and CNA Evolution from Single-cell DNA Sequencing of Tumors
              </a>
            </td>
          <td>
            Leah L. Weber, Anna Hart, Idoia Ochoa, M. El-Kebir
          </td>
          <td>2024-11-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Cancer’s epigenetic landscape, a labyrinthine tapestry of molecular modifications, has long captivated researchers with its profound influence on gene expression and cellular fate. This review discusses the intricate mechanisms underlying cancer epigenetics, unraveling the complex interplay between DNA methylation, histone modifications, chromatin remodeling, and non-coding RNAs. We navigate through the tumultuous seas of epigenetic dysregulation, exploring how these processes conspire to silence tumor suppressors and unleash oncogenic potential. The narrative pivots to cutting-edge technologies, revolutionizing our ability to decode the epigenome. From the granular insights of single-cell epigenomics to the holistic view offered by multi-omics approaches, we examine how these tools are reshaping our understanding of tumor heterogeneity and evolution. The review also highlights emerging techniques, such as spatial epigenomics and long-read sequencing, which promise to unveil the hidden dimensions of epigenetic regulation. Finally, we probed the transformative potential of CRISPR-based epigenome editing and computational analysis to transmute raw data into biological insights. This study seeks to synthesize a comprehensive yet nuanced understanding of the contemporary landscape and future directions of cancer epigenetic research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf231d1889782a99759e5b17e3a47e9d23349015" target='_blank'>
              Mechanisms and technologies in cancer epigenetics
              </a>
            </td>
          <td>
            Z. Sherif, O. Ogunwobi, H. Ressom
          </td>
          <td>2025-01-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="
 Although various effective anti-cancer treatments have become available over the last decades, resistance to all available therapies remains the major cause of death of cancer patients with disseminated tumors. Striking examples are patients with triple-negative breast cancer (TNBC), which are frequently defective in the repair of DNA double strand breaks, e.g. due to loss of BRCA1 function. Because of this defect, the patients initially respond very well to DNA damage-inducing chemotherapy. Unfortunately, disseminated tumors are usually not eradicated and resistant tumor cells are eventually selected from residual primary or metastatic tumor sites. It is therefore crucial to understand the molecular mechanisms underlying the drug tolerance of the residual tumor cells. To study residual disease, we used the K14cre;Brca1 F/F ;p53 F/F (KB1P) mouse model for hereditary breast cancer, which provides the unique opportunity to explore and target those mechanisms in a fully immunocompetent model. The mammary tumors that spontaneously develop highly resemble their human counterparts, both morphologically and in their therapy response. For example, tumors shrink in response to poly(ADP-ribose) polymerase inhibition, platinum-based treatment or the commonly used doxorubicin, docetaxel and cyclophosphamide (TAC) combination therapy. But despite repeated drug sensitivity, the KB1P mammary tumors are not eradicated, not even by a frequent dosing schedule. By combining single-cell RNA sequencing, spatial transcriptomics and imaging mass cytometry, we dissected the intratumoral heterogeneity and alterations in the tumor microenvironment of residual disease. We identified specific subpopulations of tumor cells that have a survival benefit after treatment, and these occurred together with an altered microenvironment characterized by a highly reactive stroma infiltrated with both anti-inflammatory and pro-tumoral immune cells. Interestingly, those structural and transcriptional changes are reversed in the relapsed tumors, highlighting the plasticity of drug tolerance. To investigate the mechanisms of the altered tumor-stroma interactions that are relevant for drug tolerance, we have designed a custom-made CRISPR/Cas9 library based on differential gene expression of the residual tumor cells. This library enables us to functionally test relevant mechanisms of drug tolerance in vivo and we expect that these analyses will provide useful insights into the tumor-stroma interactions that contribute to residual disease. Taken together, our detailed analyses demonstrate the substantial remodeling of tumor cells in their microenvironment upon treatment. To develop new therapeutic approaches to eradicate drug-tolerant tumor cells and thereby circumvent tumor relapse, it is essential to better understand the heterogeneous cellular composition of residual tumors in its spatial context. With this project, we hope to provide comprehensive data to develop better therapeutic strategies that target the tumor-stroma interaction and eradicate residual tumors.
 Citation Format: Morgane Decollogny, Demeter Túrós, Astrid Chanfon, Myriam Siffert, Ismar Klebic, Sven Rottenberg. Investigating residual disease in its spatial context in BRCA1 p53-deficient mammary tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr C034.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ecf546e6bc5a8fa717c4c87fcb379e8ed3399d7" target='_blank'>
              Abstract C034: Investigating residual disease in its spatial context in BRCA1 p53-deficient mammary tumors
              </a>
            </td>
          <td>
            Morgane Decollogny, Demeter Túrós, Astrid Chanfon, Myriam Siffert, I. Klebic, Sven Rottenberg
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6a4f47822586a1fd8d669e38036044aa6da804e" target='_blank'>
              Chromothripsis in cancer.
              </a>
            </td>
          <td>
            Milena Simovic-Lorenz, Aurélie Ernst
          </td>
          <td>2024-11-15</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="An important direction in cancer prevention is the development of novel strategies that can be used in the treatment by affecting the mechanisms of the regulatory pathways and genomic elements involved in malignant transformation. Strategy proposed in this study is to initiate cell death mechanisms in response to the depletion of the energy resources within a cancer cell due to the uncontrolled spread of mobile genetic elements throughout its genome. The calculation results obtained using a preliminary mathematical model that simulates cellular bioenergetic balance, taking into account the energy consumption for retrotransposition of the mobile elements such as LINE-1 and SINE, show that this scenario is plausible. Intracellular resources undergo a critical redistribution when affecting the genomic defense mechanisms and the LINE-1 transcription rate. This leads to a sharp increase in energy consumption for retrotransposon transcription that causes a significant decrease in the pool of free ATPs in the cell.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05095e01db5773f04351486017740af52565a16e" target='_blank'>
              Control of the Activity of Mobile Elements in Cancer Cells as a Strategy for Anticancer Therapy
              </a>
            </td>
          <td>
            S. R. Pavlov, V. V. Gursky, M. Samsonova, A. Kanapin, A. A. Samsonova
          </td>
          <td>2024-12-15</td>
          <td>Biofizika</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e6b74de7021622cacaa229773949fb0310d8a9a" target='_blank'>
              A practice-oriented genome-profiling study for acute myeloid leukemia using the novel HANDLE system: HM-screen-JAPAN02.
              </a>
            </td>
          <td>
            H. Arai, N. Hosono, Sung-Gi Chi, Kentaro Fukushima, Daisuke Ikeda, Satoshi Iyama, Akihiko Gotoh, Takayuki Ikezoe, Chikashi Yoshida, Goichi Yoshimoto, Junya Kanda, Naoto Takahashi, E. Sakaida, Kensuke Usuki, Takahiro Yamauchi, Yosuke Minami
          </td>
          <td>2024-12-16</td>
          <td>International journal of hematology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Breast cancer is one of the leading causes of death among women, which implies an urgent need to explore further alternative therapeutic strategies than conventional surgery, chemotherapy, and radiotherapy treatments. Thus, prevention should be highlighted through early genetic screening, not only among patients with clear risk factors but also among the general population in order to discover predisposition and intervene earlier.
The current therapeutic innovations are focused on improving such treatments by devising less invasive and more specific approaches. Examples include gene repair, such as halting tumor growth by restoring functionality to damaged genes using CRISPR and similar technologies. MicroRNAs have also shown much promise in controlling major cancer processes such as metastasis and cell division. Besides, nanoparticle-based drug delivery allows for more effective release of drugs. Consequently, there is lesser development of side effects. Suicide gene therapy is also commanding more attention against breast cancer as a less aggressive and hence more effective therapy with the appearance of selective killing of cancerous cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8403af1baef50b0a9182d430ee9e9895b5ac636" target='_blank'>
              Emerging approaches to genetic therapy in treatment for breast cancer
              </a>
            </td>
          <td>
            Jimena Pagaza-Hernández, Jazmín E. Barrientos-Cabrales, Dayanara G. López, Julissa Carrillo-Villarreal, Aylin Meza-Sifuentes, Monica M. Nuñez-Vallez, Brissia Lazalde
          </td>
          <td>2024-11-30</td>
          <td>GSC Biological and Pharmaceutical Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The resistance of tumors to immune checkpoint inhibitors (ICI) may be intricately linked to cellular senescence, although definitive clinical validation remains elusive. In this study, comprehensive pan-cancer scRNA-seq analyses identify fibroblasts as exhibiting the most pronounced levels of cellular senescence among tumor-associated cell populations. To elucidate this phenomenon, a fibroblast senescence-associated transcriptomic signature (FSS), which correlated strongly with protumorigenic signaling pathways and immune dysregulation that fosters tumor progression, is developed. Leveraging the FSS, the machine learning (ML) framework demonstrates exceptional accuracy in predicting ICI response and survival outcomes, achieving superior area under curve (AUC) values across validation, testing, and in-house cohorts. Strikingly, FSS consistently outperforms established signatures in predictive robustness across diverse cancer subtypes. From an integrative analysis of 17 CRISPR/Cas9 libraries, CDC6 emerges as a pivotal biomarker for pan-cancer ICI response and prognostic stratification. Mechanistically, experimental evidence reveals that CDC6 in tumor cells orchestrates fibroblast senescence via TGF-β1 secretion and oxidative stress, subsequently reprogramming the tumor microenvironment and modulating ICI response. These findings underscore the translational potential of targeting fibroblast senescence as a novel therapeutic strategy to mitigate immune resistance and enhance antitumor efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83eb29659bd661ae208e4642398f8b3b7069e404" target='_blank'>
              A Distinguished Roadmap of Fibroblast Senescence in Predicting Immunotherapy Response and Prognosis Across Human Cancers.
              </a>
            </td>
          <td>
            Dongjie Chen, Pengyi Liu, Jiayu Lin, Longjun Zang, Yihao Liu, Shuyu Zhai, Xiongxiong Lu, Yuanchi Weng, Hongzhe Li
          </td>
          <td>2024-12-30</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Simple Summary Cancer cells arise from multiple complementary genomic and epigenomic abnormalities that deregulate the pathways controlling cell proliferation, cell survival, and tissue homeostasis. Epigenetic modifications of histones, DNA, chromatin structure, and non-coding RNA expression are emerging as critical determinants of gene expression and essential drivers of tumor cell heterogeneity and neoplastic cell phenotypes. Multipotent stem-like cells (also referred to as cancer stem cells, CSCs) are critical determinants of tumor propagation, therapeutic resistance, and recurrence following treatment. However, our understanding of how these multidimensional epigenetic modifications cooperatively drive cancer cell stemness and how to efficaciously target them is currently inadequate to positively impact patient outcomes. Abstract Background/Objectives: CSCs are critical drivers of the tumor and stem cell phenotypes of glioblastoma (GBM) cells. Chromatin modifications play a fundamental role in driving a GBM CSC phenotype. The goal of this study is to further our understanding of how stem cell-driving events control changes in chromatin architecture that contribute to the tumor-propagating phenotype of GBM. Methods: We utilized computational analyses to identify a subset of clinically relevant genes that were predicted to be repressed in a Polycomb repressive complex 2 (PRC2)-dependent manner in GBM upon induction of stem cell-driving events. These associations were validated in patient-derived GBM neurosphere models using state-of-the-art molecular techniques to express, silence, and measure microRNA (miRNA) and gene expression changes. Advanced Poly(β-amino ester) nanoparticle formulations (PBAEs) were used to deliver miRNAs in vivo to orthotopic human GBM tumor models. Results: We show that glioma stem cell (GSC) formation and tumor propagation involve the crosstalk between multiple epigenetic mechanisms, resulting in the repression of the miRNAs that regulate PRC2 function and histone H3 lysine 27 tri-methylation (H3K27me3). We also identified miR-217-5p as an EZH2 regulator repressed in GSCs and showed that miR-217-5p reconstitution using advanced nanoparticle formulations re-activates the PRC2-repressed genes, inhibits GSC formation, impairs tumor growth, and enhances the effects of ionizing radiation in an orthotopic model of GBM. Conclusions: These findings suggest that inhibiting PRC2 function by targeting EZH2 with miR-217-5p advanced nanoparticle formulations could have a therapeutic benefit in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a650a5c81b8e5180cf08ed0736baa5ed09de3eb0" target='_blank'>
              miR-217-5p NanomiRs Inhibit Glioblastoma Growth and Enhance Effects of Ionizing Radiation via EZH2 Inhibition and Epigenetic Reprogramming
              </a>
            </td>
          <td>
            Jack Korleski, Sweta Sudhir, Yuan Rui, Christopher Caputo, Sophie Sall, Amanda L. Johnson, Harmon Khela, Tanmaya Madhvacharyula, Anisha Rasamsetty, Yunqing Li, B. Lal, Weiqiang Zhou, Karen Smith-Connor, Stephany Tzeng, Jordan Green, J. Laterra, H. Lopez-Bertoni
          </td>
          <td>2024-12-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="DNA topoisomerases are essential molecular machines that manage DNA topology in the cell and play important roles in DNA replication and transcription. We find that knocking down the enzyme Topoisomerase Top2 or its mammalian homolog Top2b increases the life span of S. cerevisiae, C. elegans, and mice. Top2b reduction also extends the health span of mice and alleviates the pathologies of aging in multiple tissues. At the cellular/molecular level, Top2b reduction attenuates the major hallmarks of aging, such as cellular senescence, de-regulated nutrient sensing, epigenetic alteration, and lysosomal biogenesis. We observed that Top2b reduction significantly changes the epigenetic landscape in various mouse tissues toward those of the young animals, and differentially down-regulates genes with active promoter and high expression. Our observations suggest that Top2 reduction confers longevity effect across species via a conserved mechanism, and may be used as a novel therapeutic strategy for countering aging.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2158b0d6967f642c67566187862ea04490680967" target='_blank'>
              Reduction of DNA Topoisomerase Top2 reprograms the epigenetic landscape and extends health and life span across species
              </a>
            </td>
          <td>
            Man Zhu, Meng Ma, Lunan Luo, Feiyang Li, Jiashun Zhang, Yan Pan, Lu Yang, Ying Xiao, Ziteng Wang, Bo Xian, Yi Zheng, Hao Li, Jing Yang
          </td>
          <td>2024-11-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background: Cancer treatment remains a critical area of clinical research, with numerous approaches developed depending on tumor type and stage. Recent advances in genetic and immunotherapy, bioinformatics, and genetic science have revolutionized cancer diagnosis and treatment. Emerging technologies such as gene delivery, oncolytic virotherapy, suicide gene therapy, and CRISPR/Cas9 offer promising therapeutic avenues.
Methods: This review provides a detailed analysis of the latest techniques of Gene therapy, artificial intelligence, Nanocarrier Delivery Systems, Immunotherapy, CAR T-Cell Therapy, Epigenetics, Vaccines, and Clinical translation and assessing their therapeutic potential in cancer treatment. A review of early critical studies focused on the integration of immunotherapy, nanotechnology, and artificial intelligence (AI) in cancer therapies with emphasis on targeted delivery systems and precision medicine.
Results: The review highlights the synergistic effects of combining targeted therapies with immunotherapy, particularly immune checkpoint inhibitors and CAR-T cell therapy. Therapies such as CRISPR/Cas9 demonstrate significant potential in cancer targeting, while advancements in nanocarrier delivery systems offer enhanced precision with reduced side effects. AI's role in improving cancer diagnosis and personalized treatment is also underscored.
Conclusion: This comprehensive analysis of recent therapeutic approaches and technological advancements addresses gaps in previous reviews and offers updated insights into cutting-edge cancer treatments. The review emphasizes the evolving role of immunotherapy, nanotechnology, AI, and nanotechnology, providing clinicians and researchers with the most current and relevant information for optimizing cancer treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e4489de30cd4bf2c5d0db097c602c99f94e6e1f" target='_blank'>
              Recent and Advanced Trends in Cancer Treatments
              </a>
            </td>
          <td>
            RA Chithra, Mohamed Osman Elamin, Sucheta Karande
          </td>
          <td>2024-12-01</td>
          <td>National Journal of Community Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c580172e55394858107b5db74bf7f577e6d7f63" target='_blank'>
              DNMT1 prolonged absence is a tunable cellular stress that triggers cell proliferation arrest to protect from major DNA methylation loss
              </a>
            </td>
          <td>
            Salvatore Martino, Serena Gargano, P. Carollo, A. Di Leonardo, V. Barra
          </td>
          <td>2024-12-18</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="BACKGROUND
Acute lymphocytic leukemia (ALL), characterized by uncontrolled growth of abnormal lymphocytes, predominantly affects children. Genetic analysis focusing on genes and microRNAs reveals important information about the biology of ALL, enabling accurate diagnosis and treatment. This study examines gene and microRNA expression in B cell ALL to improve early diagnosis and personalized treatment.


METHODS
Bone marrow samples were collected from patients both before and after the induction phase of chemotherapy. Comprehensive diagnostic techniques including flow cytometry, molecular assays, real-time PCR for common translocations, karyotyping, and complete blood count (CBC) analysis were employed. These methods were utilized to determine the type and risk assessment of ALL, identify specific gene and microRNA expressions, and measure blood cell counts.


RESULTS
The study comprised 12 patients, all under the age of 18. Post-treatment RT-PCR analysis revealed significant reductions in the expression of the ABCB1 gene, miR-129-5p, and miR-9-5p following the induction phase of chemotherapy. Karyotype analysis indicated that two patients were hypodiploid; unfortunately, both of these patients did not survive.


CONCLUSION
MicroRNAs and ABC genes serve as predictive and prognostic biomarkers in Acute Lymphoblastic Leukemia (ALL) and should be carefully reconsidered. It is more accurate to state that while microRNAs and ABC genes may potentially influence treatment response in ALL, further research is crucial to fully understand their roles in determining treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36d6c6bf058c1b51d999c0b567643ee6cccfac4f" target='_blank'>
              ATP-Binding Cassette Transporter Genes and microRNAs in Pediatric B-Cell ALL: Expression Insights.
              </a>
            </td>
          <td>
            Reza Nekouian, Pooya Faranoush, Fatemeh Khesali, Parisa Shams, M. Foroughi‐Gilvaee, Negin Sadighnia, Dorsa Fallah Azad, MohammadAli Ehsani, Mohammad Faranoush
          </td>
          <td>2024-12-23</td>
          <td>Journal of clinical laboratory analysis</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8e4edf1ee1ccd4a382a2ceac11847327f800ad1" target='_blank'>
              Silencing of LINC01278 promotes sensitivity of non-small cell lung cancer cells to osimertinib by targeting miR-324-3p/ZFX axis.
              </a>
            </td>
          <td>
            Quan Lei, Ping Liu, Xinlei Guan, Li Liu, Wenjuan He
          </td>
          <td>2024-12-19</td>
          <td>Cytotechnology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Osteosarcoma is the most common primary malignant bone tumor, predominantly affecting children and adolescents. Despite advances in conventional therapies like chemotherapy and surgical resection, the survival rate has remained stagnant due to therapeutic resistance and high rates of metastasis. The tumor microenvironment (TME), a complex network of cellular and non-cellular components, plays a pivotal role in osteosarcoma progression, metastasis, and treatment resistance. Understanding the dynamics within the TME is crucial for developing novel therapeutic strategies that can overcome these challenges. This review explores the key elements of the osteosarcoma TME, including immune cells, endothelial cells, cancer-associated fibroblasts (CAFs), and extracellular matrix (ECM). It examines the roles of cytokines, growth factors, and exosomes secreted by osteosarcoma cells in modifying the TME to foster tumor growth, evade immune surveillance, and promote angiogenesis. Furthermore, the review critically assesses current therapeutic approaches that target TME components, with a focus on disrupting the interactions between the tumor and its microenvironment. This analysis includes a review of clinical trials and preclinical studies evaluating TME-targeting therapies. The findings highlight that the TME actively contributes to osteosarcoma progression by promoting immunosuppression, angiogenesis, and metastasis, while also enhancing resistance to standard treatments such as chemotherapy and immune checkpoint inhibitors. Various therapeutic strategies targeting the TME, such as inhibiting angiogenesis, modulating immune responses, and disrupting CAF and ECM interactions, have shown promise in preclinical models. However, clinical outcomes remain variable, underscoring the complexity of the TME and the need for more comprehensive approaches. Targeting the TME represents a promising pathway to overcome therapeutic resistance in osteosarcoma. While significant progress has been made in understanding the role of the TME in tumor development and resistance mechanisms, further research is required to optimize TME-targeting therapies. A deeper comprehension of the intricate interactions between osteosarcoma cells and their microenvironment may lead to more effective, personalized treatments, improving clinical outcomes for patients with osteosarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de846221faa142b04c72a554ab42e7526bfe9854" target='_blank'>
              Targeting the Tumor Microenvironment in Osteosarcoma: A Pathway to Overcome Therapeutic Resistance
              </a>
            </td>
          <td>
            Kaniga Pandi, B. Cheriyan, Rithika Sooriyaprakash, Pradhosh Sakthivel, Rajeshkumar Thirupathi
          </td>
          <td>2024-12-30</td>
          <td>Biomedical and Pharmacology Journal</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6666f30aff9bcd1de0a39ac7dc32869d01658a2" target='_blank'>
              Integrated analysis of single-cell and bulk-RNA sequencing for the cellular senescence in prognosis of lung adenocarcinoma
              </a>
            </td>
          <td>
            Fengqiang Yu, Liangyu Zhang, Xun Zhang, Jianshen Zeng, Fancai Lai
          </td>
          <td>2025-01-09</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Histones are key epigenetic factors for regulating the accessibility and compaction of eukaryotic genomes, affecting the replication, repair, and expression of DNA. Recent studies have demonstrated that histone missense mutations can perturb normal histone function, promoting the development of phenotypically distinguishable cancers. However, most histone mutations observed in cancer patients remain enigmatic in their potential to promote cancer development. To assess the oncogenic potential of histone missense mutations, we have gathered whole-exome sequencing data for the tumors of over 12,000 patients. Overall, histone mutations occurred in about 16% of cancer patients, although specific cancer types showed substantially higher rates. Using a combination of genomic, structural, and biophysical analyses, we found several predominant modes of action, where cancer missense mutations in histones affected acidic patches and protein binding interfaces in a cancer-specific manner and targeted interaction sites with specific DNA repair proteins. Consistent with this finding, we observed a high tumour mutational burden in patients with histone mutations affecting interactions of DNA repair proteins. We also identified potential cancer driver mutations in several histone genes, including histone H4-a highly conserved histone without previously documented driver mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00566c578efc8bbf332002783a58fe37b3053b70" target='_blank'>
              Cancer Histone Mutations Impact Binding and DNA Repair Processes, Leading to Increased Mutagenesis
              </a>
            </td>
          <td>
            Daniel Espiritu, Yiru Sheng, Yunhui Peng, Daria Ostroverkhova, Shuxiang Li, David Landsman, Maria J. Aristizabal, Anna R Panchenko
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Biological mechanisms are inherently dynamic, requiring precise and rapid gene manipulation for effective characterization. Traditional genetic perturbation tools such as siRNA and CRISPR knockout operate on timescales that render them unsuitable for exploring dynamic processes or studying essential genes, where chronic depletion can lead to cell death. Here, we compared four major inducible degron systems—dTAG, HaloPROTAC, and two auxin-inducible degron (AID) tools—in human pluripotent stem cells. We evaluated basal degradation levels, inducible degradation kinetics, and recovery dynamics for endogenously tagged genes. While the AID 2.0 system is the most efficient for rapid protein degradation, it exhibited higher basal degradation and slower recovery after ligand washout. To address these challenges, we applied directed protein evolution, incorporating base-editing-mediated mutagenesis and iterative functional selection and screening. We discovered novel OsTIR1 variants, including S210A, with significantly enhanced overall degron efficiency. The resulting system, designated as AID 3.0, demonstrates minimal basal degradation and rapid and effective target protein depletion and substantially rescues the cellular and molecular phenotypes due to basal degradation or slow target protein recovery in previous systems. We conclude that AID 3.0 represents a superior degron technology, offering a valuable tool for studying gene functions in dynamic biological contexts and exploring therapeutic applications. Additionally, the research strategy used here could be broadly applicable for improving other degron and biological tools.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a538e1075274bba3f3af7ca77c8e65e5a189520" target='_blank'>
              Comparative analysis and directed protein evolution yield an improved degron technology with minimal basal degradation, rapid inducible depletion, and faster recovery of target proteins
              </a>
            </td>
          <td>
            Mazhar Adli, De Xing, Tao Bai, Ozlem Neyisci, Seyedehzahra Paylakhi, Alexander Duval, Yasemin Tekin
          </td>
          <td>2024-11-15</td>
          <td>Research Square</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Philadelphia chromosome-like B-cell acute lymphoblastic leukemia (Ph-like ALL) is driven by genetic alterations that induce constitutive kinase signaling and is associated with chemoresistance and high relapse risk in children and adults. Preclinical studies in the most common CRLF2-rearranged/JAK pathway-activated Ph-like ALL subtype have shown variable responses to JAK inhibitor-based therapies, suggesting incomplete oncogene addiction and highlighting a need to elucidate alternative biologic dependencies and therapeutic vulnerabilities, while the ABL-class Ph-like ALL subtype appears preferentially sensitive to SRC/ABL- or PDGFRB-targeting inhibitors. Which patients may be responsive versus resistant to tyrosine kinase inhibitor (TKI)-based precision medicine approaches remains a critical knowledge gap. Using bulk and single-cell multiomics analyses, we profiled residual cells from CRLF2-rearranged or ABL1-rearranged Ph-like ALL patient-derived xenograft models treated in vivo with targeted inhibitors to identify TKI-resistant subpopulations and potential mechanisms of therapeutic escape. We detected a specific MYC dependency in Ph-like ALL cells and defined a new leukemia cell subpopulation with senescence-associated stem cell-like features regulated by AP-1 transcription factors. This dormant ALL subpopulation was effectively eradicated by dual pharmacologic inhibition of BCL-2 and JAK/STAT or SRC/ABL pathways, a clinically-relevant therapeutic strategy. Single cell-derived molecular signatures of this senescence and stem/progenitor-like subpopulation further predicted poor clinical outcomes associated with other high-risk genetic subtypes of childhood B-ALL and thus may have broader prognostic applicability beyond Ph-like ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b87bb41065d73fc1bc25a49641c54717955deab" target='_blank'>
              Targeting Senescent Stemlike Subpopulations in Philadelphia Chromosome-Like Acute Lymphoblastic Leukemia.
              </a>
            </td>
          <td>
            Yang-yang Ding, Jonathan H Sussman, Kellyn Madden, J. Loftus, Robert K Chen, C. D. Falkenstein, D. A. Barcenas Lopez, David A Hottman, Benjamin Mathier, Wenbao Yu, Jason Xu, Changya Chen, Chia-Hui Chen, Bing He, S. Bandyopadhyay, Zhan Zhang, DongGeun Lee, Hong Wang, Junmin Peng, Chi V. Dang, Kai Tan, S. Tasian
          </td>
          <td>2025-01-07</td>
          <td>Blood</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Cancer, a complex and heterogeneous disease, continues to be a major global health concern. Despite advancements in diagnostics and therapeutics, the aggressive nature of certain cancers remain a significant challenge, necessitating a deeper understanding of the underlying molecular mechanisms driving their severity and progression. Cancer severity and progression depend on cellular properties such as cell migration, cell division, cell shape changes, and intracellular transport, all of which are driven by dynamic cellular microtubules. Dynamic properties of microtubules, in turn, are regulated by an array of proteins that influence their stability and growth. Among these regulators, End Binding (EB) proteins stand out as critical orchestrators of microtubule dynamics at their growing plus ends. Beyond their fundamental role in normal cellular functions, recent research has uncovered compelling evidence linking EB proteins to the pathogenesis of various diseases, including cancer progression. As the field of cancer research advances, the clinical implication of EB proteins role in cancer severity and aggressiveness become increasingly evident. This review aims to comprehensively explore the role of microtubule-associated EB proteins in influencing the severity and aggressiveness of cancer. We also discuss the potential significance of EB as a clinical biomarker for cancer diagnosis and prognosis and as a target for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e8a843dc90163f1664e05266a959467a1810e6f" target='_blank'>
              End Binding Proteins: Drivers of Cancer Progression.
              </a>
            </td>
          <td>
            Dhakshmi Sasankan, Renu Mohan
          </td>
          <td>2024-12-19</td>
          <td>Cytoskeleton</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50b3b6f34b71d4b5f3b8e63dfe568f53d1e1b13e" target='_blank'>
              Stabilization of expandable DNA repeats by the replication factor Mcm10 promotes cell viability
              </a>
            </td>
          <td>
            Chiara Masnovo, Zohar Paleiov, Daniel Dovrat, Laurel K Baxter, Sofia Movafaghi, A. Aharoni, S. Mirkin
          </td>
          <td>2024-12-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="T-cell activation and clonal expansion are essential for the efficacy of immunotherapy in non-small cell lung cancer (NSCLC) patients. Since the distribution of T-cell clones might provide insights into immunogenic mechanisms, we determined the α/β TCR clonality using RNA-sequencing from frozen tumor tissue of 182 NSCLC patients and paired the results with extensive in situ image and sequence analyses of the immune microenvironment of NSCLC. TCR clonality (Gini index) patterns ranged from high T-cell clone diversity with high evenness (Gini index low) to clonal dominance with low evenness (Gini index high). TCR clonality in cancer tissue was lower than in matched normal lung (p=0.021). High Gini index correlated strongly with distinct mutations (EGFR, P53), tumor mutation burden (p<0.001), and inflamed tumor phenotypes (PRF1, GZMA, GZMB, INFG) with exhaustion signatures (LAG3, TIGIT, IDO1, PD-1, PD-L1). Correspondingly, PD-1+, CD3+, CD8A+, CD163+, and CD138+ immune cells infiltrated cancer tissue with high TCR clonality. In situ sequencing revealed that dominant T-cell clones were more often of CD8-subtype and tended to approximate the tumor cell compartment (p<0.03). In a checkpoint inhibitor-treated NSCLC patient cohort, high TCR clonality was associated with therapy response (p=0.016) and prolonged survival (p=0.003, median survival 13.8 vs 2.9 months). Our robust analysis pipeline revealed diverse TCR repertoires related to genotypes and immune phenotypes. The in situ positioning of expanded T-cell clones indicated functional impact, which was clinically confirmed in NSCLC patients receiving immunotherapy. One sentence summary T-cell clone expansion in NSCLC is associated with genetic mutations, immune phenotypes, immunotherapy response, and patient survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d627cf9e5819664ceb7a72e57e39ad1a7a4de3d3" target='_blank'>
              TCR clonality and TCR clonal expansion in the in situ microenvironment of non-small cell lung cancer
              </a>
            </td>
          <td>
            Hui Yu, Anastasia Magoulopoulou, Rose-Marie Amini, Maria Chatzinikolaou, Masafumi Horie, A. Lindberg, Artur Mezheyeuski, M. Backman, Andreas Metousis, Hans Brunström, Millaray Marincevic Zuniga, J. Botling, J. Mattsson, K. Kärre, Karin Leandersson, Mats Nilsson, Carina Strell, P. Micke
          </td>
          <td>2024-12-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="A significant portion of human cancers utilize a recombination-based pathway, Alternative Lengthening of Telomeres (ALT), to extend telomeres. To gain further insights into this pathway, we developed a high-throughput imaging-based screen named TAILS (Telomeric ALT In situ Localization Screen), to identify genes that either promote or inhibit ALT activity. Screening over 1000 genes implicated in DNA transactions, TAILS revealed both well-established and novel ALT modulators. We have identified new factors that promote ALT, such as the nucleosome-remodeling factor CHD4 and the chromatin reader SGF29, as well as factors that suppress ALT, including the RNA helicases DDX39A/B, the replication factor TIMELESS, and components of the chromatin assembly factor CAF1. Our data indicate that defects in histone deposition significantly contribute to ALT-associated phenotypes. Based on these findings, we demonstrate that pharmacological treatments can be employed to either exacerbate or suppress ALT-associated phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b124646ba82a460ac6aa3c7cf35a789c6739919" target='_blank'>
              Identification of Novel Modulators of the ALT Pathway Through a Native FISH-Based Optical Screen
              </a>
            </td>
          <td>
            Benura Azeroglu, Simran Khurana, Shih-Chun Wang, Gianna M Tricola, Shalu Sharma, Camille Jubelin, Ylenia Cortolezzis, Gianluca Pegoraro, Kyle M. Miller, Travis H. Stracker, Eros Lazzerini Denchi
          </td>
          <td>2024-11-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Prostate cancer (PC) progresses from benign epithelium through pre-malignant lesions, localized tumors, metastatic dissemination, and castration-resistant stages, with some cases exhibiting phenotype plasticity under therapeutic pressure. However, high-resolution insights into how cell phenotypes evolve across successive stages of PC remain limited. Here, we present the Prostate Cancer Cell Atlas (PCCAT) by integrating ∼710,000 single cells from 197 human samples covering a spectrum of tumor stages. This comprehensive analysis dissects the cellular landscape and characterizes key cell types and molecular features that associate with PC progression and prognosis. In malignant cells, we highlight a distinctive profile denoted by high Major Histocompatibility Complex (MHC) expression, low Androgen Receptor (AR) activity, and enhanced stemness programs associated with enzalutamide resistance. Moreover, we reveal several cell states strongly correlated with PC progression and adverse prognosis, including lineage plasticity-like malignant cells (LPCs), neuroendocrine tumor cells, pericytes, and matrix cancer-associated fibroblasts (mCAFs). Furthermore, we uncover shared cell states that underpin the immune suppressive tumor microenvironment in advanced PC, including activated regulatory T cells, exhausted CD8+ T cells, and SPP1-expressing macrophages. Lastly, we pinpoint a spatial niche composed of mCAFs and SPP1-expressing macrophages localized near the tumor boundary in aggressive PC, which correlates with poor prognosis. Overall, our work provides a valuable resource and offers deeper insights into the diverse cell states, dynamics, and functional characteristics involved in PC progression at single-cell resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1b731bdee7235cc5667d4336486935b5dd643fd" target='_blank'>
              Deciphering single-cell heterogeneity and cellular ecosystem dynamics during prostate cancer progression
              </a>
            </td>
          <td>
            Faming Zhao, Jianming Zeng, Canping Chen, Xiaofan Zhao, Tingting Zhang, George V. Thomas, Rosalie C. Sears, J. Alumkal, Amy E Moran, G. Mills, Peter S Nelson, Zheng Xia
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17bb4f23b9e1892130e632d422e0888edddfeef1" target='_blank'>
              Molecular heterogeneity and MYC dysregulation in triple-negative breast cancer: genomic advances and therapeutic implications.
              </a>
            </td>
          <td>
            Priya, Arun Kumar, Dhruv Kumar
          </td>
          <td>2025-01-01</td>
          <td>3 Biotech</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Glioblastoma (GBM) is uniformly lethal due to profound treatment resistance. Altered cellular metabolism is a key mediator of GBM treatment resistance. Uptake of the essential sulfur-containing amino acid methionine is drastically elevated in GBMs compared to normal cells, however, it is not known how this methionine is utilized or whether it relates to GBM treatment resistance. Here, we find that radiation acutely increases the levels of methionine-related metabolites in a variety of treatment-resistant GBM models. Stable isotope tracing studies further revealed that radiation acutely activates methionine to S-adenosyl methionine (SAM) conversion through an active signaling event mediated by the kinases of the DNA damage response. In vivo tumor SAM synthesis increases after radiation, while normal brain SAM production remains unchanged, indicating a tumor- specific metabolic alteration to radiation. Pharmacological and dietary strategies to block methionine to SAM conversion slowed DNA damage response and increased cell death following radiation in vitro. Mechanistically, these effects are due to depletion of DNA repair proteins and are reversed by SAM supplementation. These effects are selective to GBMs lacking the methionine salvage enzyme methylthioadenosine phosphorylase. Pharmacological inhibition of SAM synthesis hindered tumor growth in flank and orthotopic in vivo GBM models when combined with radiation. By contrast, methionine depletion does not reduce tumor SAM levels and fails to radiosensitize intracranial models, indicating depleting SAM, as opposed to simply lowering methionine, is critical for hindering tumor growth in intracranial models of GBM. These results highlight a new signaling link between DNA damage and SAM synthesis and define the metabolic fates of methionine in GBM in vivo. Inhibiting radiation-induced SAM synthesis slows DNA repair and augments radiation efficacy in GBM. Using MAT2A inhibitors to deplete SAM may selectively overcome treatment resistance in GBMs with defective methionine salvage while sparing normal brain.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11aa91d406c5f4b75a9a6b9dbcbaac218f6d1df0" target='_blank'>
              Reciprocal links between methionine metabolism, DNA repair and therapy resistance in glioblastoma
              </a>
            </td>
          <td>
            Navyateja Korimerla, Baharan Meghdadi, Isra Haq, Kari Wilder-Romans, Jie Xu, Nicole Becker, Ziqing Zhu, Peter Kalev, Nathan Qi, Charles Evans, Maureen Kachman, Zitong Zhao, Angelica Lin, A. Scott, Alexandra M O'Brien, Ayesha U. Kothari, Peter Sajjakulnukit, Li Zhang, Sravya Palavalasa, Erik R Peterson, Marc L Hyer, Katya Marjon, Taryn Sleger, Meredith A. Morgan, C. Lyssiotis, Everett M. Stone, S. Ferris, Theodore S. Lawrence, Deepak Nagrath, Weihua Zhou, Daniel R Wahl
          </td>
          <td>2024-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="T-cell acute lymphoblastic leukaemia (T-ALL) is a heterogeneous malignant disease with high relapse and mortality rates. To characterise the multiomics features of T-ALL, we conducted integrative analyses using single-cell RNA, TCR and chromatin accessibility sequencing on pre- and post-treatment peripheral blood and bone marrow samples of the same patients. We found that there is transcriptional rewiring of gene regulatory networks in T-ALL cells. Some transcription factors, such as TCF3 and KLF3, showed differences in activity and expression levels between T-ALL and normal T cells and were associated with the prognosis of T-ALL patients. Furthermore, we identified multiple malignant TCR clonotypes among the T-ALL cells, where the clonotypes consisted of distinct combinations of the same TCR α and β chain per patient. The T-ALL cells displayed clonotype-specific immature thymocyte cellular characteristics and response to chemotherapy. Remarkably, T-ALL cells with an orphan TCRβ chain displayed the strongest stemness and resistance to chemotherapy. Our study provided transcriptome and epigenome characterisation of T-ALL cells categorised by TCR clonotypes, which may be helpful for the development of novel predictive markers to evaluate treatment effectiveness for T-ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b35386d641d4da0518c18007554d1299175192fd" target='_blank'>
              Single-Cell Multiomics Reveals TCR Clonotype-Specific Phenotype and Stemness Heterogeneity of T-ALL Cells.
              </a>
            </td>
          <td>
            Songnan Sui, Xiaolei Wei, Yue Zhu, Qiuyue Feng, Xianfeng Zha, Lipeng Mao, Boya Huang, Wen Lei, Guobing Chen, Huien Zhan, Huan Chen, Ru Feng, Chengwu Zeng, Yangqiu Li, Oscar Junhong Luo
          </td>
          <td>2024-12-15</td>
          <td>Cell proliferation</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e5f657e821743bf515a8df904c0de712f1e97ea" target='_blank'>
              Understanding the biological processes of kidney carcinogenesis: an integrative multi-omics approach
              </a>
            </td>
          <td>
            Ricardo Cortez Cardoso Penha, Alexandra Sexton Oates, S. Senkin, Hanla A. Park, Joshua Atkins, I. Holcatova, A. Hornakova, S. Savic, Simona Ognjanovic, Beata Świątkowska, J. Lissowska, D. Zaridze, A. Mukeria, V. Janout, Amélie Chabrier, Vincent Cahais, C. Cuenin, G. Scelo, M. Foll, Z. Herceg, Paul Brennan, K. Smith-Byrne, N. Alcala, James D. McKay
          </td>
          <td>2024-11-26</td>
          <td>Molecular Systems Biology</td>
          <td>0</td>
          <td>105</td>
        </tr>

        <tr id="Background/Objectives: AML is an aggressive malignant disease characterized by aberrant proliferation and accumulation of immature blast cells in the patient’s bone marrow. Chemotherapeutic treatment can effectively induce remission and re-establish functional hematopoiesis. However, many patients experience chemoresistance-associated relapse and disease progression with a poor prognosis. The identification of molecular determinants of chemoresistance that could serve as potential targets for the therapeutic restoration of chemosensitivity has proven to be challenging. Methods: To address this, we have analyzed longitudinal changes in the expression of microRNAs during disease progression in a small set of four AML patients, combined with gene ontology (GO) pathway analysis and evaluation of gene expression data in patient databases. Results: MicroRNA profiling of bone marrow samples at diagnosis and after relapse revealed significant differential expression of a large number of microRNAs between the two time points. Subsequent GO pathway analysis identified 11 signal transduction pathways likely to be affected by the differential miRNA signatures. Exemplary validation of the FoxO signaling pathway by gene expression analysis confirmed significant upregulation of FOXO1 and the target genes GADD45 and SOD2. Conclusions: Here, we show how a microRNA-based pathway prediction strategy can be used to identify differentially regulated signaling pathways that represent potential targets for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3ad3edfd6af9770fc155f2d3924cf9e30fae927" target='_blank'>
              MicroRNA Screening Reveals Upregulation of FoxO-Signaling in Relapsed Acute Myeloid Leukemia Patients
              </a>
            </td>
          <td>
            Paula Reichelt, S. Bernhart, U. Platzbecker, Michael Cross
          </td>
          <td>2024-12-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e31d8028a34805d9bd64c4990406de1eae29e1a1" target='_blank'>
              Age-related genomic alterations and chemotherapy sensitivity in osteosarcoma: insights from cancer genome profiling analyses.
              </a>
            </td>
          <td>
            Hidetatsu Outani, Masachika Ikegami, Yoshinori Imura, Shou Nakai, Haruna Takami, Yuki Kotani, A. Inoue, S. Okada
          </td>
          <td>2024-12-17</td>
          <td>International journal of clinical oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e49e60b286ec304714eb4ff898b1356bb43dd043" target='_blank'>
              Understanding tumour growth variability in breast cancer xenograft models identifies PARP inhibition resistance biomarkers
              </a>
            </td>
          <td>
            D. Voulgarelis, J. Forment, A. Herencia Ropero, D. Polychronopoulos, J. Cohen-Setton, A. Bender, V. Serra, M. O'Connor, J. W. T. Yates, K. Bulusu
          </td>
          <td>2024-11-18</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/155fb0cbcac71baffd69e7cee3cc6ee3b9918fae" target='_blank'>
              A comprehensive study of Z-DNA density and its evolutionary implications in birds
              </a>
            </td>
          <td>
            Yu-Ren Wang, Shao-Ming Chang, Jinn-Jy Lin, Hsiao-Chian Chen, Lo-Tung Lee, Dien-Yu Tsai, Shih-Da Lee, Chung-Yu Lan, Chuang-Rung Chang, Chih-Feng Chen, Chen Siang Ng
          </td>
          <td>2024-11-21</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Acute myeloid leukemia (AML) is a hematologic tumor with poor prognosis and significant clinical heterogeneity. By integrating transcriptomic data, single-cell RNA sequencing data and independently collected RNA sequencing data this study aims to identify key genes in AML and establish a prognostic assessment model to improve the accuracy of prognostic prediction. Materials and methods We analyzed RNA-seq data from AML patients and combined it with single-cell RNA sequencing data to identify genes associated with AML prognosis. Key genes were screened by bioinformatics methods, and a prognostic assessment model was established based on these genes to validate their accuracy. Results The study identified eight key genes significantly associated with AML prognosis: SPATS2L, SPINK2, AREG, CLEC11A, HGF, IRF8, ARHGAP5, and CD34. The prognostic model constructed on the basis of these genes effectively differentiated between high-risk and low-risk patients and revealed differences in immune function and metabolic pathways of AML cells. Conclusion This study provides a new approach to AML prognostic assessment and reveals the role of key genes in AML. These genes may become new biomarkers and therapeutic targets that can help improve prognostic prediction and personalized treatment of AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fba51cf3073c5201e7804e4ac1e1ae1bb4b49f8" target='_blank'>
              Single-cell transcriptomics reveals heterogeneity and prognostic markers of myeloid precursor cells in acute myeloid leukemia
              </a>
            </td>
          <td>
            Guangfeng He, Lai Jiang, Xuancheng Zhou, Yuheng Gu, Jingyi Tang, Qiang Zhang, Qingwen Hu, Gang Huang, Ziye Zhuang, Xinrui Gao, Ke Xu, Yewei Xiao
          </td>
          <td>2024-12-16</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Microtubules play essential roles in diverse cellular processes and are important pharmacological targets for treating human disease. Here, we sought to identify cellular factors that modulate the sensitivity of cells to antimicrotubule drugs. We conducted a genome-wide CRISPR/Cas9-based functional genetics screen in human cells treated with the microtubule-destabilizing drug nocodazole or the microtubule-stabilizing drug paclitaxel. We further conducted a focused secondary screen to test drug sensitivity for ∼1,400 gene targets across two distinct human cell lines and to additionally test sensitivity to the KIF11 inhibitor, STLC. These screens defined gene targets whose loss enhances or suppresses sensitivity to antimicrotubule drugs. In addition to gene targets whose loss sensitized cells to multiple compounds, we observed cases of differential sensitivity to specific compounds and differing requirements between cell lines. Our downstream molecular analysis further revealed additional roles for established microtubule-associated proteins and identified new players in microtubule function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fe92d3ab4eb0f4d2bf377007f003eebe7871111" target='_blank'>
              Functional genetics reveals modulators of antimicrotubule drug sensitivity.
              </a>
            </td>
          <td>
            Kuan-Chung Su, Elena Radul, Nolan K Maier, Mary-Jane Tsang, Claire Goul, Brittania Moodie, Océane Marescal, Heather R Keys, I. Cheeseman
          </td>
          <td>2024-11-21</td>
          <td>The Journal of cell biology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Aging constitutes complex and dynamic alterations in molecular and physiological processes and is associated with numerous disorders, in part due to increased genetic instability. The aging population is projected to double by 2050, underscoring the urgent need to better understand the relationships between aging and age-related disorders. Repetitive DNA elements are intrinsic sources of genetic instability and have been found to co-localize with mutation hotspots in human cancer genomes. In this study, we explored the relationship between aging and DNA repeat-mediated genetic instability in vivo using an H-DNA-forming mirror-repeat sequence from the cancer-associated human c-MYC gene. Utilizing a unique mutation-reporter mouse model, we observed tissue-specific effects of aging on H-DNA-induced genetic instability, with mutation frequencies increasing in spleen tissues and remaining unchanged in testis tissues. Analysis of the mutation spectra revealed large deletion mutations as the primary contributor to increasing H-DNA-induced mutations, supported by increased cleavage activity of H-DNA structures in aged spleen tissues. Our findings demonstrate that aging has distinct tissue-specific effects on repeat-mediated, cancer-associated mutations, providing insights into the complex relationship between aging and cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8066191467ac61ad7ea1a2f5a55dbdea73e14d68" target='_blank'>
              Tissue-Specific Effects of Aging on Repeat-Mediated Mutation Hotspots In Vivo
              </a>
            </td>
          <td>
            Alexandra M. D’Amico, Tonia T. Li, Karen M. Vasquez
          </td>
          <td>2024-11-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15084ff422804e2f9ab2fbc7e8a0344ff82b8dbd" target='_blank'>
              High CD44 expression and enhanced E-selectin binding identified as biomarkers of chemoresistant leukemic cells in human T-ALL.
              </a>
            </td>
          <td>
            J. Calvo, Irina Naguibneva, Anthony Kypraios, Florian Gilain, B. Uzan, Baptiste Gaillard, Léa Bellenger, Laurent Renou, Christophe Antoniewski, Hélène Lapillonne, Arnaud Petit, P. Ballerini, Stéphane J. C. Mancini, Tony Marchand, Jean-François Peyron, Françoise Pflumio
          </td>
          <td>2024-11-24</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background Metabolic reprogramming is a hallmark of cancer, including alterations in the hexosamine biosynthesis pathway (HBP). Glutamine-fructose-6-phosphate transaminase 1 (GFPT1) is the key regulatory enzyme in the HBP; however, its role in invasive breast carcinoma remains underexplored. Methods This study utilized integrated data from The Cancer Genome Atlas (TCGA) to assess GFPT1 expression in breast cancer (BRCA) patients. Functional enrichment and mutational landscape analyses were performed, along with chemosensitivity predictions. In vitro experiments were conducted by silencing GFPT1 in malignant breast epithelial cells to evaluate changes in proliferation, migration, and apoptosis. Results Elevated GFPT1 expression was linked to advanced-stage breast cancer and identified as an independent prognostic marker for overall survival (OS). High GFPT1 levels were associated with increased cytoplasmic translation, activation of oncogenic pathways, and infiltration of M2 macrophages. The GFPT1-High group also showed a higher mutational burden, with frequent TP53 mutations. Chemosensitivity analysis revealed increased IC50 values for chemotherapy drugs in this group. GFPT1 silencing led to reduced cell proliferation and migration, along with enhanced apoptosis. Conclusion These findings indicate that GFPT1 is a novel prognostic biomarker and a predictive indicator of chemotherapy response in invasive breast carcinoma. GFPT1 influences mRNA translation, cell cycle regulation, and M2 macrophage infiltration, thereby promoting cancer cell proliferation and metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b6655d004010dcffbbdb0a2efe7ca8624f1e53e" target='_blank'>
              Bioinformatics insights into the role of GFPT1 in breast invasive carcinoma: implications for tumor prognosis, immune modulation, and therapeutic applications
              </a>
            </td>
          <td>
            Jianghui Liang, Xiaolian Deng, Yingyi Zhang, Tianchi Fei, Muzi Ouyang, Chengjie Yu, Yang Xiang, Dongwei Jia, Fangfang Duan
          </td>
          <td>2024-11-22</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Clinical medicine and biological sciences make extensive use of the gene editing method symbolized by CRISPR-Cas9. With the rapid development of molecular biology technology in recent years, the CRISPR-Cas systemwhich stands for clustered regularly interspaced short palindromic repeatshas become a potent tool for gene editing. Due to its high efficiency, precision, and flexibility, it has surpassed last two generations in its application within the field of molecular biology. This article provides a detailed overview of the development history, working mechanism, and applications of this technology. The CRISPR-Cas system achieves specific site modifications by introducing insertions, deletions, or single base substitutions at particular genomic sites. It has made significant contributions to research and treatment in areas such as the therapies of tumors and genetic diseases. However, challenges such as off-target effects need to be addressed before its practical clinical utility can be rigorously validated through further research and clinical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1ea967093b688e91d8c17ab505cb62e377bf9fa" target='_blank'>
              Progress in human diseases treatment CRISPR system
              </a>
            </td>
          <td>
            Yuanrong Wu
          </td>
          <td>2024-12-26</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Adaptive radiations are rich laboratories for exploring, testing, and understanding key theories in evolution and ecology because they offer spectacular displays of speciation and ecological adaptation. Particular challenges to the study of adaptive radiation include high levels of species richness, rapid speciation, and gene flow between species. Over the last decade, high-throughput sequencing technologies and access to population genomic data have lessened these challenges by enabling the analysis of samples from many individual organisms at whole-genome scales. Here we review how population genomic data have facilitated our knowledge of adaptive radiation in five key areas: (1) phylogenetics, (2) hybridization, (3) timing and rates of diversification, (4) the genomic basis of trait evolution, and (5) the role of genome structure in divergence. We review current knowledge in each area, highlight outstanding questions, and focus on methods that facilitate detection of complex patterns in the divergence and demography of populations through time. It is clear that population genomic data are revolutionising the ability to reconstruct evolutionary history in rapidly diversifying clades. Additionally, studies are increasingly emphasising the central role of gene flow, re-use of standing genetic variation during adaptation, and structural genomic elements as facilitators of the speciation process in adaptive radiations. We highlight hybridization-and the hypothesized processes by which it shapes diversification-and questions seeking to bridge the divide between microevolutionary and macroevolutionary processes as rich areas for future study. Overall, access to population genomic data has facilitated an exciting era in adaptive radiation research, with implications for deeper understanding of fundamental evolutionary processes across the tree of life.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6567f401893572605f0ef83ca062582a85587de" target='_blank'>
              Population Genomics of Adaptive Radiation.
              </a>
            </td>
          <td>
            Lucia L Combrink, Jimena Golcher-Benavides, Alexander L Lewanski, J. Rick, William C Rosenthal, C. Wagner
          </td>
          <td>2024-12-24</td>
          <td>Molecular ecology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="BACKGROUND
Acquired resistance to temozolomide (TMZ) chemotherapy due to DNA mismatch repair (MMR) enzyme deficiency is a barrier to improving outcomes for IDH wild-type glioblastoma (GBM) patients. KL-50 is a new imidazotetrazine-based therapeutic designed to induce DNA interstrand cross links, and subsequent double-stranded breaks, in an MMR-independent manner in cells with O-6-Methylguanine-DNA Methyltransferase (MGMT) deficiency. Previous research showed its efficacy against LN229 glioma cells with MMR and MGMT knockdown. Its activity against patient-derived GBM that model post-TMZ recurrent tumors is unclear.


METHODS
We created MMR-deficient GBM patient-derived xenografts through exposure to TMZ, followed by treatment with additional TMZ or KL-50. We also generated isogenic, MSH6 knockout patient-derived GBM and tested them for sensitivity to TMZ and KL-50.


RESULTS
KL-50 extended the median survival of mice intracranially engrafted with either patient-derived TMZ-naïve GBM6 or TMZ-naïve GBM12 by 1.75-fold and 2.15-fold, respectively (p<0.0001). A low dose (4 Gy) of fractionated RT further extended the survival of KL-50 treated GBM12 mice (median survival=80 days for RT+ KL-50 vs. 71 days KL-50 alone, P=0.018). KL-50 also extended the median survival of mice engrafted with post-TMZ, MMR-deficient GBM6R-m185 (140 days for KL-50 vs. 37 days for vehicle, p<0.0001). MSH6-KO increased TMZ IC50 for GBM6 and GBM12 cultures by >5-fold and >12-fold for cell death and live cell count outputs, respectively. In contrast, MSH6-KO actually decreased KL-50 IC50 by 10-80%.


CONCLUSION
KL-50-based compounds are a promising new strategy for the treatment of MGMT-deficient, MMR-deficient GBM that recurs after frontline TMZ.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03f3a0f339bf4219c2fc489415b10b402bba9dd1" target='_blank'>
              The novel DNA cross-linking agent KL-50 is active against patient-derived models of new and recurrent post-temozolomide mismatch repair-deficient glioblastoma.
              </a>
            </td>
          <td>
            Matthew McCord, Thomas Sears, Wenxia Wang, Rahul Chaliparambil, Shejuan An, Jann Sarkaria, C. James, Bruce Ruggeri, Susan E. Gueble, R. Bindra, C. Horbinski
          </td>
          <td>2024-12-10</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e9558944d48747d281f714408c8c756ade188a3" target='_blank'>
              Genetic and epigenetic characterization of sarcoma stem cells across subtypes identifies EZH2 as a therapeutic target
              </a>
            </td>
          <td>
            Edmond O'Donnell, Maria Muñoz, Ryan Davis, Jessica Bergonio, R. L. Randall, Clifford Tepper, Janai R. Carr-Ascher
          </td>
          <td>2025-01-09</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract RNA modification has emerged as a crucial area of research in epigenetics, significantly influencing tumor biology by regulating RNA metabolism. N‐acetyltransferase 10 (NAT10)‐mediated N4‐acetylcytidine (ac4C) modification, the sole known acetylation in eukaryotic RNA, influences cancer pathogenesis and progression. NAT10 is the only writer of ac4C and catalyzes acetyl transfer on targeted RNA, and ac4C helps to improve the stability and translational efficiency of ac4C‐modified RNA. NAT10 is highly expressed and associated with poor prognosis in pan‐cancers. Based on its molecular mechanism and biological functions, ac4C is a central factor in tumorigenesis, tumor progression, drug resistance, and tumor immune escape. Despite the increasing focus on ac4C, the specific regulatory mechanisms of ac4C in cancer remain elusive. The present review thoroughly analyzes the current knowledge on NAT10‐mediated ac4C modification in cancer, highlighting its broad regulatory influence on targeted gene expression and tumor biology. This review also summarizes the limitations and perspectives of current research on NAT10 and ac4C in cancer, to identify new therapeutic targets and advance cancer treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc21647bfb08f2bead5c258ee67888a30cbd41b0" target='_blank'>
              The role and mechanism of NAT10‐mediated ac4C modification in tumor development and progression
              </a>
            </td>
          <td>
            Zhuoran Gu, Libin Zou, Xinjian Pan, Yang Yu, Yongqiang Liu, Zhijin Zhang, Ji Liu, Shiyu Mao, Junfeng Zhang, Changcheng Guo, Wei Li, Jiang Geng, Wentao Zhang, Xudong Yao, Bing Shen
          </td>
          <td>2024-12-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Dysregulation of enhancer-promoter communication in the context of the three-dimensional (3D) nucleus is increasingly recognized as a potential driver of oncogenic programs. Here, we profiled the 3D enhancer-promoter networks of primary patient-derived glioblastoma stem cells (GSCs) in comparison with neuronal stem cells (NSCs) to identify potential central nodes and vulnerabilities in the regulatory logic of this devastating cancer. Specifically, we focused on hyperconnected 3D regulatory hubs and demonstrated that hub-interacting genes exhibit high and coordinated expression at the single-cell level and strong association with oncogenic programs that distinguish IDH-wt glioblastoma patients from low-grade glioma. Epigenetic silencing of a recurrent 3D enhancer hub—with an uncharacterized role in glioblastoma—was sufficient to cause concordant downregulation of multiple hub-connected genes along with significant shifts in transcriptional states and reduced clonogenicity. By integrating published datasets from other cancer types, we also identified both universal and cancer type-specific 3D regulatory hubs which enrich for varying oncogenic programs and nominate specific factors associated with worse outcomes. Genetic alterations, such as focal duplications, could explain only a small fraction of the detected hyperconnected hubs and their increased activity. Overall, our study provides computational and experimental support for the potential central role of 3D regulatory hubs in controlling oncogenic programs and properties. HIGHLIGHTS - 3D regulatory “hubs” in glioblastoma enrich for highly coregulated genes at a single-cell level and expand oncogenic regulatory networks. - Targeted perturbation of a highly recurrent 3D regulatory hub in GSCs results in altered transcriptional states and cellular properties. - 3D regulatory hubs across cancer types associate with tumor-specific and universal oncogenic programs and worse outcomes. - The majority of hyperconnected hubs do not overlap with structural variants, suggesting epigenetic mechanisms. eTOC Here we profile the 3D enhancer connectomes of primary patient-derived human glioblastoma stem cells (GSCs), identify hyperconnected 3D regulatory “hubs”, and examine the impact of 3D hub perturbation on the transcriptional program and oncogenic properties.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ef46a4bf29419cbc8b345c6cf8f3bc9f4287626" target='_blank'>
              Three-dimensional regulatory hubs support oncogenic programs in glioblastoma
              </a>
            </td>
          <td>
            Sarah L. Breves, Dafne Campigli Di Giammartino, James Nicholson, S. Cirigliano, Syed Raza Mahmood, UkJin Lee, Alexander Martínez-Fundichely, Johannes Jungverdorben, R. Singhania, Sandy Rajkumar, Raphael Kirou, L. Studer, Ekta Khurana, A. Polyzos, Howard A. Fine, E. Apostolou
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="Renal cell carcinoma (RCC) is characterized by recurrent somatic mutations in epigenetic regulators, which stratify patients into clinically significant subgroups with distinct prognoses and treatment responses. However, the cell type-specific epigenetic landscape of RCC—broadly and in the context of these mutations—is incompletely understood. To investigate these open questions, we integrated single nucleus ATAC sequencing data from RCC tumors across four independent cohorts. In clear cell RCC tumors, we identified four shared malignant epigenetic programs related to angiogenesis, proximal tubule-like features, interferon (IFN) signaling, and one that lacked distinct genomic regions with increased accessibility. Among the mutated epigenetic regulators, BAP1 mutation exhibited the most significant impact on chromatin accessibility in tumor cells, and the associated epigenetic changes were linked to IFN response. We identify multiple potential sources of elevated IFN signaling in these lesions, such as increased immune infiltration and increased accessibility and expression of an IFN-associated ERV, ERV3-16A3_LTR. We find that the expression of ERV3-16A3_LTR may itself be a negative prognostic biomarker in ccRCC. Our findings highlight the convergence of malignant epigenetic programs across ccRCC tumors and suggest that BAP1 loss, potentially through ERV3-16A3_LTR dysregulation, is associated with an IFN response-high epigenetic program.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f60a362700f5a511019adc2819b352bb15feca9" target='_blank'>
              Single-cell epigenetic profiling reveals an interferon response-high program associated with BAP1 deficiency in kidney cancer
              </a>
            </td>
          <td>
            Sabrina Y. Camp, M. X. He, Michael S. Cuoco, E. Saad, Erica Pimenta, Kevin Meli, Z. Bakouny, C. Labaki, Breanna M. Titchen, Yun Jee Kang, Jack Horst, Rachel Trowbridge, Erin Shannon, K. Helvie, A. Thorner, S. Vigneau, Angie Mayorga, Jahnavi Kodali, Hannah Lachmayr, Meredith Bemus, Jihye Park, T. Choueiri, Kevin Bi, E. V. Van Allen
          </td>
          <td>2024-11-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Chemotherapy often kills a large fraction of cancer cells but leaves behind a small population of drug- tolerant persister cells. These persister cells survive drug treatments through reversible, non-genetic mechanisms and cause tumour recurrence upon cessation of therapy. Here, we report a drug tolerance mechanism regulated by the germ-cell-specific H3K4 methyltransferase PRDM9. Through histone proteomic, transcriptomic, lipidomic, and ChIP-sequencing studies combined with CRISPR knockout and phenotypic drug screen, we identified that chemotherapy-induced PRDM9 upregulation promotes metabolic rewiring in glioblastoma stem cells, leading to chemotherapy tolerance. Mechanistically, PRDM9-dependent H3K4me3 at cholesterol biosynthesis genes enhances cholesterol biosynthesis, which persister cells rely on to maintain homeostasis under chemotherapy- induced oxidative stress and lipid peroxidation. PRDM9 inhibition, combined with chemotherapy, resulted in strong anti-cancer efficacy in preclinical glioblastoma models, significantly enhancing the magnitude and duration of the antitumor response by eliminating persisters. These findings demonstrate a previously unknown role of PRDM9 in promoting metabolic reprogramming that enables the survival of drug-tolerant persister cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44ca423a805e163b74f1186c734661fa71cbd4de" target='_blank'>
              Histone methyltransferase PRDM9 promotes survival of drug-tolerant persister cells in glioblastoma
              </a>
            </td>
          <td>
            George L. Joun, Emma G. Kempe, Brianna Chen, Jayden Sterling, Ramzi H. Abbassi, W. Daniel du Preez, A. Recasens, Teleri Clark, Tian Y. Du, Jason K.K. Low, Hani Kim, Pengyi Yang, Jasmine Khor, Monira Hoque, Dinesh C. Indurthi, M. Kuchibhotla, Ranjith Palanisamy, William T Jorgensen, Andrew P. Montgomery, , S. Higginbottom, E. Tomaskovic-Crook, J. M. Crook, Lipin Loo, Bryan W. Day, G. G. Neely, Ernesto Guccione, Terrance G. Johns, Michael Kassiou, A. S. Don, L. Munoz
          </td>
          <td>2025-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Background: RAD51C, a critical member of the RAD51 paralog family, is essential for homologous recombination (HR)-mediated DNA repair, a pathway crucial for maintaining genomic stability. Mutations in RAD51C have been linked to cancer susceptibility, particularly in breast and ovarian cancers, where impaired DNA repair mechanisms contribute to genomic instability and tumor progression. Despite its clinical significance, the functional impact of specific RAD51C variants remains poorly understood, necessitating a comprehensive investigation into their biological implications. 
Methods: This study classified RAD51C gene variants into damaging and tolerant categories using computational prediction tools, including SIFT, PolyPhen, CADD, MetaLR, and Mutation Assessor. Variants were prioritized based on consensus scores and classified as high-confidence damaging variants. Correlation and agreement among tools were analyzed to refine predictions. Principal Component Analysis (PCA) and clustering methods were employed to group variants based on prediction patterns. Protein-protein interaction (PPI) networks and pathway enrichment analyses were conducted to contextualize damaging variants within broader biological systems, with a focus on their roles in HR, DNA repair, and cellular processes. 
Results: A total of 2526 variants were analyzed, with damaging variants showing consistent patterns across tools. Consensus scores highlighted 302 high-confidence damaging variants, which were associated with disrupted biological processes, including double-strand break repair via homologous recombination, telomere maintenance, and regulation of cell cycle checkpoints. PPI analysis revealed an interconnected network with 11 nodes and 54 edges, with a clustering coefficient of 0.982, indicating tightly coordinated interactions among DNA repair proteins. Pathway enrichment analyses identified significant associations with homologous recombination (FDR = 2.55E-17) and the Fanconi anemia pathway (FDR = 2.96E-06). 
Conclusion: This study provides a comprehensive framework for assessing the functional impacts of RAD51C variants by integrating computational predictions with biological analyses. The findings underscore the importance of RAD51C in HR and DNA repair pathways, offering insights into its role in genomic stability and cancer progression. These results can inform the prioritization of variants for experimental validation and guide therapeutic strategies targeting DNA repair deficiencies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16240c1097223735006d241b04503aff291d133b" target='_blank'>
              Statistical Analysis of Gene Variants for Homologous Recombination Pathways of DNA Repair leading to Cancer Susceptibility
              </a>
            </td>
          <td>
            Usha Adiga, B. Jyoti, P. Reddemma, Alfred J. Augustine, S. Vasishta
          </td>
          <td>2024-12-27</td>
          <td>International Journal of Statistics in Medical Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="BACKGROUND
Despite recent advances in the biology of IDH-wildtype glioblastoma, it remains a devastating disease with median survival of less than 2 years. However, the molecular underpinnings of the heterogeneous response to the current standard-of-care treatment regimen consisting of maximal safe resection, adjuvant radiation, and chemotherapy with temozolomide remain unknown.


METHODS
Comprehensive histopathologic, genomic, and epigenomic evaluation of paired initial and recurrent glioblastoma specimens from 106 patients was performed to investigate the molecular evolution and cellular phenotypes underlying differential treatment responses.


RESULTS
While TERT promoter mutation and CDKN2A homozygous deletion were early events during gliomagenesis shared by initial and recurrent tumors, most other recurrent genetic alterations (eg, EGFR, PTEN, and NF1) were commonly private to initial or recurrent tumors indicating acquisition later during clonal evolution. Furthermore, glioblastomas exhibited heterogeneous epigenomic evolution with subsets becoming more globally hypermethylated, hypomethylated, or remaining stable. Glioblastoma that underwent sarcomatous transformation had shorter interval to recurrence and were significantly enriched in NF1, TP53, and RB1 alterations and the mesenchymal epigenetic class. Patients who developed somatic hypermutation following temozolomide treatment had significantly longer interval to disease recurrence and prolonged overall survival, and increased methylation at 4 specific CpG sites in the promoter region of MGMT was significantly associated with this development of hypermutation. Finally, an epigenomic evolution signature incorporating change in DNA methylation levels across 347 critical CpG sites was developed that significantly correlated with clinical outcomes.


CONCLUSIONS
Glioblastoma undergoes heterogeneous genetic, epigenetic, and cellular evolution that underlies prognostically different treatment responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce8c70c5b87e0a8ebd1138c27ed60e44214d7859" target='_blank'>
              Longitudinal multimodal profiling of IDH-wildtype glioblastoma reveals the molecular evolution and cellular phenotypes underlying prognostically different treatment responses.
              </a>
            </td>
          <td>
            Calixto-Hope G Lucas, N. Al-Adli, Jacob S. Young, Rohit Gupta, R. Morshed, Jasper Wu, Ajay Ravindranathan, Anny Shai, N. Oberheim Bush, Jennie W Taylor, John de Groot, J. Villanueva‐Meyer, M. Pekmezci, Arie Perry, Andrew W Bollen, P. Theodosopoulos, M. Aghi, E. Chang, Shawn L Hervey-Jumper, D. Raleigh, Annette M Molinaro, Joseph F Costello, Aaron A. Diaz, Jennifer L. Clarke, N. Butowski, Joanna J. Phillips, Susan M Chang, Mitchell S. Berger, D. A. Solomon
          </td>
          <td>2024-11-19</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="IDH-mutant low-grade gliomas (LGGs) are slow-growing brain tumors that frequently progress to aggressive high-grade gliomas that have dismal outcomes. In a recent study, Wu and colleagues provide critical insights into the mechanisms underlying malignant progression by analyzing single-cell gene expression and chromatin accessibility across different tumor grades. Their findings support a two-phase model: in early stages, tumors are primarily driven by oligodendrocyte precursor-like cells and epigenetic alterations that silence tumor suppressors like CDKN2A and activate oncogenes such as PDGFRA. As the disease advances, the tumors become sustained by more proliferative neural precursor-like cells, where genetic alterations, including PDGFRA, MYCN, and CDK4 amplifications and CDKN2A/B deletion, drive tumor progression. The study further highlights a dynamic regulation of interferon (IFN) signaling during progression. In low-grade IDH-mutant gliomas, IFN responses are suppressed through epigenetic hypermethylation, which can be reversed with DNMT1 inhibitors or IDH inhibitors, leading to reactivation of the IFN pathway. In contrast, higher-grade gliomas evade IFN signaling through genetic deletions of IFN gene clusters. These findings emphasize a broader epigenetic-to-genetic shift in oncogenic regulation that drives glioma progression, provides a valuable framework for understanding the transition from indolent tumors to lethal malignancies, and has implications for therapy and clinical management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/598e5f4259516cb6e86cc401bbe6fe663aa489ab" target='_blank'>
              Switching Drivers: Epigenetic Rewiring to Genetic Progression in Glioma.
              </a>
            </td>
          <td>
            K. Drucker, Robert B. Jenkins, Daniel Schramek
          </td>
          <td>2024-12-30</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Gene editing with CRISPR-Cas9 technology revolutionizes modern medicine by enabling precise DNA modifications. Initially discovered as a bacterial defense mechanism, the Cas9 enzyme, guided by RNA, can target and cut specific DNA sites, allowing for gene editing. Applications include genetic engineering, functional studies, and potential treatments for genetic diseases like cancer. Notably, in 2023, CRISPR-Cas9 was approved for treating sickle cell anaemia with significant results, despite challenges like long-term security and high costs. CRISPR-Cas9 technology allows for correcting genetic defects, treating diseases, and improving agricultural crops. It can regulate gene transcription through the CRISPRi system, using an inactive Cas9 to interfere with gene expression without permanently altering DNA. This gene-editing tool shows promise in gene therapy, potentially curing diseases like HIV-1, sickle cell disease, and haemophilia B. However, challenges include off-target mutations and efficient delivery of CRISPR/Cas9. Precise target site selection and dosage control are crucial, with tools like CasOT helping identify and prevent unwanted mutations. CRISPR-Cas9 requires a PAM sequence to function, narrowing its targets in the genome but increasing specificity. Production of gRNA faces challenges due to mRNA processing, with alternatives like the artificial gene RGR showing promise. Efficient delivery methods are still needed, with current techniques involving DNA and RNA injection. Future applications include treating genetic diseases and agricultural improvements, with ongoing research essential for overcoming challenges and ensuring safety and accuracy. Global collaboration is vital for the ethical use of this technology.
Contribution to evidence-based healthcare: CRISPR therapy represents an exciting frontier in genetic medicine, allowing for precise gene editing and opening up new possibilities for treating incurable diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb0ce03e8f26830137fb7f68ada78482dfe3faf6" target='_blank'>
              CRISPR therapy: A revolutionary breakthrough in genetic medicine
              </a>
            </td>
          <td>
            Geneci Da Silva Barreto, Jair Brito do Nascimento, Karin Cristina Santos de Almeida, José Carlos Marcolino Neto, Samuel Lucas Ferreira Luz da Silva
          </td>
          <td>2024-12-16</td>
          <td>International Healthcare Review (online)</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is the most aggressive breast cancer subtype with substantial disease heterogeneity, limited treatment options, and dismal clinical outcomes. Some TNBCs display homologous recombination deficiency (HRD), a phenotype with elevated genomic burden and worse prognosis if left untreated but chemotherapeutic sensitivity. While the molecular landscape of TNBC is distinct from other breast cancer subtypes, the TNBC-specific link between HRD and epigenome-wide methylation has not been established. This study reports two independent cohorts of TNBC tumors (n = 32 and n = 58) with HRD and epigenomic landscapes measured by the Multiplex Ligation-dependent Probe Amplification assay and the Illumina MethylationEPIC arrays, respectively. Genome-wide copy number and methylation alterations were significantly higher in HRD (all p <.05). Methylation of genome-wide repeat element Alu and transcriptional regulatory regions were significantly lower in HRD (all p <.05). An age-adjusted epigenome-wide association study of the continuous HRD probability scores revealed significant loci (all FDR <0.05) that were depleted from the CpG-rich "island" regions often seen in gene promoters but enriched in the CpG-poor "open sea" regions localized to gene enhancers. The significant loci implicated well-known candidate genes involved in the epithelial-to-mesenchymal transition, Wnt signaling, and DNA damage response. Supervised machine learning of HRD with nucleotide-specific methylation as the input enabled clinically relevant tumor stratification. Taken together, this study provides novel biological and translational insights into HRD in TNBCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5080511deac3bf2a96cf31c6d0433b409f61f60d" target='_blank'>
              Extensive epigenomic dysregulation is a hallmark of homologous recombination deficiency in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Youdinghuan Chen, Lucas A Salas, Jonathan D. Marotti, Nicole P. Jenkins, Chao Cheng, Todd W. Miller, A. Kettenbach, Brock C. Christensen
          </td>
          <td>2024-12-05</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Background. The management of pediatric acute myeloid leukemia (AML) is based on the prognostic risk classification of initial leukemia. Targeted next-generation sequencing (NGS) is a reliable method used to identify recurrently mutated genes of pediatric AML and associated prognosis.
Methods. In this study, we retrospectively evaluated the prognostic, and therapeutic utility of a targeted NGS panel covering twenty-five genes, in 21 children with de novo and 8 with relapsed or secondary AML.  
Results. Variants were detected in 44.8% of patients, and 63.2% of them were in the signaling pathway genes. The number of variants per patient and diversity increased with age. The panel results affected hematopoietic stem cell transplantation decisions, especially in core binding factor AML, and allowed the categorization of diseases according to current classifications. Panel results also pointed out predisposition to germline leukemia to the extent of the panel coverage. No targeted therapy was used based on the variants, and none of the variants were used to monitor minimal residual disease.
Conclusions. Targeted NGS results, along with well-known genetic aberrations and treatment responses, can guide treatment modalities. The coverage of the routine panels should include proven mutations of childhood AML and germline leukemia predisposition genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b4601dddbafa15e4c68929e64e4ba8e0770802f" target='_blank'>
              The importance of targeted next-generation sequencing based genomic profiling at diagnosis of childhood acute myeloid leukemia: a single center experience
              </a>
            </td>
          <td>
            D. Kaçar, B. Çavdarlı, Ayça Koca Yozgat, Melek Işık, F. Kurtipek, Fatma Tuba Yıldırım, T. Bayhan, Dilek Gürlek Gökçebay, Namık Yaşar Özbek, N. Yaralı
          </td>
          <td>2024-12-19</td>
          <td>The Turkish Journal of Pediatrics</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Genetic alterations alone cannot account for the diverse phenotypes of cancer cells. Even cancers with the same driver mutation show significant transcriptional heterogeneity and varied responses to therapy. However, the mechanisms underpinning this heterogeneity remain under-explored. Here, we find that novel enhancer usage is a common feature in acute lymphoblastic leukemia (ALL). In particular, KMT2A::AFF1 ALL, an aggressive leukemia with a poor prognosis and a low mutational burden, exhibits substantial transcriptional heterogeneity between individuals. Using single cell multiome analysis and extensive chromatin profiling, we reveal that much transcriptional heterogeneity in KMT2A::AFF1 ALL is driven by novel enhancer usage. Using high resolution Micro-Capture-C in primary patient samples, we also identify patient-specific enhancer activity at key oncogenes such as MEIS1 and RUNX2, driving high levels of expression of both oncogenes in a patient-specific manner. Overall, our data show that enhancer heterogeneity is highly prevalent in KMT2A::AFF1 ALL and may also be a mechanism that drives transcriptional heterogeneity in cancer more generally. Key Points Leukemia patients with the same driver mutations often display gene expression differences Using chromatin profiling and high resolution 3C methods we show that enhancer heterogeneity drives gene expression differences">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1af97c7a783137350d56568e2bae7a8a0304b710" target='_blank'>
              Enhancer heterogeneity in acute lymphoblastic leukemia drives differential gene expression between patients
              </a>
            </td>
          <td>
            Alastair L. Smith, N. Denny, Catherine M Chahrour, Kim Sharp, N. Elliott, Joe R Harman, T. Jackson, H. Geng, Owen Smith, Jonathan Bond, Irene Roberts, Ronald W. Stam, Nicholas Crump, James Oliver Davies, Anindita Roy, Thomas A. Milne
          </td>
          <td>2024-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Aneuploidy is highly detrimental to organisms due to genomic imbalance. However, the influence of parental unbalanced genome conditions on gene expression of their offspring remains unclear, particularly in animals. To further explore the molecular regulatory mechanisms, we firstly analyzed the expression patterns of aneuploid Drosophila offspring from different parents with unbalanced genomes via reciprocal crosses and studied the potential functions of male-specific lethal 2 (MSL2) in this process. The results showed that the ectopic expression of MSL2 in aneuploidy resulted in gene expression patterns closer to those of diploidy, including MSL2 target genes, maternal genes, mitochondrial genes, and transposable elements. In addition, it was also found that ERp60, the key target gene of MSL2, played a crucial role in regulating endoplasmic reticulum (ER) Ca2+ homeostasis through its interaction with the STIM1 protein. When it was overexpressed, ER Ca2+ levels and the survival of aneuploid females were significantly increased. Furthermore, we observed upregulated ER Ca2+ levels identified in aneuploid brains, which suggested that Ca2+ homeostasis may be involved in the regulation mediated by MSL2 in aneuploid genomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ee01eb4a8bef6630928776abc6349828f286edd" target='_blank'>
              Calcium Homeostasis Is Involved in the Modulation of Gene Expression by MSL2 in Imbalanced Genomes
              </a>
            </td>
          <td>
            Ruixue Wang, Shuai Zhang, Haizhu Qi, Liuqing Wang, Youjun Wang, Lin Sun
          </td>
          <td>2024-11-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e77d5a82f46b1391391ade8c249b0d27a2b059b" target='_blank'>
              Comprehensive genomic characterization of early-stage bladder cancer.
              </a>
            </td>
          <td>
            F. Prip, P. Lamy, S. Lindskrog, T. Strandgaard, I. Nordentoft, K. Birkenkamp-Demtröder, N. Birkbak, Nanna Kristjánsdóttir, Asbjørn Kjær, T. Andreasen, J. Ahrenfeldt, J. S. Pedersen, A. Rasmussen, Gregers G Hermann, K. Mogensen, Astrid C Petersen, Arndt Hartmann, M. Grimm, Marcus Horstmann, R. Nawroth, U. Segersten, D. Sikic, K. V. van Kessel, E. Zwarthoff, Tobias Maurer, Tatjana Simic, Per-Uno Malmström, N. Malats, J. B. Jensen, Francisco X. Real, L. Dyrskjøt
          </td>
          <td>2025-01-03</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40b5002680da603aefb2e7735c702ed9500dd550" target='_blank'>
              What have we learned about TP53-mutated acute myeloid leukemia?
              </a>
            </td>
          <td>
            Moazzam Shahzad, M. K. Amin, Naval G Daver, M. Shah, D. Hiwase, Daniel A Arber, M. Kharfan-Dabaja, T. Badar
          </td>
          <td>2024-11-19</td>
          <td>Blood Cancer Journal</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Gene therapy has recently become a reality in the treatment of cardiovascular diseases. Strategies to modulate gene expression using antisense oligonucleotides or small interfering RNA are proving to be safe and effective in the clinic. Adeno-associated viral vector-based gene delivery and CRISPR-Cas9-based genome editing have emerged as efficient strategies for gene delivery and repair in humans. Overall, gene therapy holds the promise not only of expanding current treatment options, but also of intervening in previously untackled causal disease mechanisms with little side effects. This scientific statement provides a comprehensive overview of the various modalities of gene therapy used to treat heart failure and some of its risk factors, and their application in the clinical setting. It discusses specifically the possibilities of gene therapy for hereditary heart diseases and (non)-genetic heart failure. Furthermore, it addresses safety and clinical trial design issues and challenges for future regulatory strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/019b14509c4c68e2ef22bb5a660b5682977dc75c" target='_blank'>
              State of the art and perspectives of gene therapy in heart failure. A scientific statement of the Heart Failure Association of the ESC, the ESC Council on Cardiovascular Genomics and the ESC Working Group on Myocardial & Pericardial Diseases.
              </a>
            </td>
          <td>
            S. van Linthout, Konstantinos Stellos, Mauro Giacca, E. Bertero, Antonio Cannatà, Lucie Carrier, Pablo García-Pavía, Alessandra Ghigo, Arantxa González, Kristina H. Haugaa, M. Imazio, L. R. Lopes, Patrick Most, Piero Pollesello, H. Schunkert, Katrin Streckfuss-Bömeke, T. Thum, C. Tocchetti, Carsten Tschöpe, P. van der Meer, E. van Rooij, M. Metra, G. Rosano, Stephane Heymans
          </td>
          <td>2024-11-22</td>
          <td>European journal of heart failure</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="DNA repair involves various intricate pathways that work together to maintain genome integrity. XPF (ERCC4) is a structural endonuclease that forms a heterodimer with ERCC1 that is critical in both single-strand break repair (SSBR) and double-strand break repair (DSBR). Although the mechanistic function of ERCC1/XPF has been established in nucleotide excision repair (NER), its role in long-patch base excision repair (BER) has recently been discovered through the 5'-Gap pathway. This study briefly explores the roles of XPF in different pathways to emphasize the importance of XPF in DNA repair. XPF deficiency manifests in various diseases, including cancer, neurodegeneration, and aging-related disorders; it is also associated with conditions such as Xeroderma pigmentosum and fertility issues. By examining the molecular mechanisms and pathological consequences linked to XPF dysfunction, this study aims to elucidate the crucial role of XPF in genomic stability as a repair protein in BER and provide perspectives regarding its potential as a therapeutic target in related diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ec8dd83ccab56542b88608ecabf7dc8acc0d8d5" target='_blank'>
              Beyond Nucleotide Excision Repair: The Importance of XPF in Base Excision Repair and Its Impact on Cancer, Inflammation, and Aging.
              </a>
            </td>
          <td>
            Dhara Gohil, Rabindra Roy
          </td>
          <td>2024-12-19</td>
          <td>International journal of molecular sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract We developed Del-read, an algorithm targeting medium-sized deletions (6–100 bp) in short-reads, which are challenging for current variant callers relying on alignment. Our focus was on Micro-Homolog mediated End Joining deletions (MMEJ-dels), prevalent in myeloid malignancies. MMEJ-dels follow a distinct pattern, occurring between two homologies, allowing us to generate a comprehensive list of MMEJ-dels in the exome. Using Del-read, we identified numerous novel germline and somatic MMEJ-dels in BEAT-AML and TCGA-breast datasets. Validation in 672 healthy individuals confirmed their presence. These novel MMEJ-dels were linked to genomic features associated with replication stress, like G-quadruplexes and minisatellite. Additionally, we observed a new category of MMEJ-dels with an imperfect-match at the flanking sequences of the homologies, suggesting a mechanism involving mispairing in homology alignment. We demonstrated robustness of the repair system despite CRISPR/Cas9-induced mismatches in the homologies. Further analysis of the canonical ASXL1 deletion revealed a diverse array of these imperfect-matches. This suggests a potentially more flexible and error-prone MMEJ repair system than previously understood. Our findings highlight Del-read's potential in uncovering previously undetected deletions and deepen our understanding of repair mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ece74b68770d9549840bdc9ed51b1bf59c99a07" target='_blank'>
              Utilizing insights of DNA repair machinery to discover MMEJ deletions and novel mechanisms
              </a>
            </td>
          <td>
            Aditee Kadam, Shay Shilo, Hadas Naor, Alexander Wainstein, Yardena Brilon, T. Feldman, Mark D. Minden, N. Kaushansky, N. Chapal-Ilani, L. Shlush
          </td>
          <td>2024-11-28</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/915fa8f258f231baa925ca4227c99c648ae059fe" target='_blank'>
              Radioresistance-related gene signatures identified by transcriptomics characterize the prognosis and immune landscape of pancreatic cancer patients
              </a>
            </td>
          <td>
            Dandan Wang, Jun Cao, Yanhui Chen, Lisha Zhang, Chan Zhou, Litao Huang, Yanliang Chen
          </td>
          <td>2024-12-05</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The 5,000 to 8,000 monogenic diseases are inherited disorders leading to mutations in a single gene. These diseases usually appear in childhood and sometimes lead to morbidity or premature death. Although treatments for such diseases exist, gene therapy is considered an effective and targeted method and has been used in clinics for monogenic diseases since 1989. Monogenic diseases are good candidates for novel therapeutic technologies like gene editing approaches to repair gene mutations. Clustered regularly interspaced short palindromic repeats (CRISPR)-based systems, the pioneer and effective gene editing tool, are utilized for ex vivo and in vivo treatment of monogenic diseases. The current review provides an overview of recent therapeutic applications of CRISPR-based gene editing in monogenic diseases in in vivo and ex vivo models. Furthermore, this review consolidates strategies aimed at providing new treatment options with gene therapy, thereby serving as a valuable reference for advancing the treatment landscape for patients with monogenic disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a0a8862e584d090d20f1389daab0e1853e5ef72" target='_blank'>
              CRISPR/Cas System: A Powerful Strategy to Improve Monogenic Human Diseases as Therapeutic Delivery; Current Applications and Challenges.
              </a>
            </td>
          <td>
            Fatemeh Saberi, Zeinab Yousefi-Najafabadi, F. Shams, Zeinab Dehghan, Sepideh Ahmadi, Tayyebeh Pilehchi, Effat Noori, Zeinab Esmaeelzadeh, Maryam Bazgiri, Rezvan Mohammadi, Farzaneh Khani, M. Sameni, Parisa Moradbeigi, G. Kardar, Mohammad Salehi, Yong Teng, V. Jajarmi
          </td>
          <td>2025-01-07</td>
          <td>Current gene therapy</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Cell states evolve through the combined activity of signaling pathways and gene regulatory networks. While transcription factors can direct cell fate, these factors rely on a cell state that is receptive to transitions in cell identity. How signaling levels contribute to the emergence of receptive cell states remains poorly defined in primary cells. Using a well-defined model of direct conversion, we examined how levels of the MAPK-activating oncogene HRASG12V influence direct conversion of primary fibroblasts to induced motor neurons. We demonstrate that an optimal ‘Goldilocks’ level of MAPK signaling efficiently drives cell-fate programming. Rates of direct conversion respond biphasically to increasing HRASG12V levels. While intermediate HRASG12V levels increase the rate of conversion, high levels of HRASG12V induce senescence. Through chemogenetic tuning, we set optimal MAPK activity for high rates of conversion in the absence of HRAS mutants. As MAPK pathways influence cell-fate transitions in development and disease, our results highlight the need to tune therapeutic interventions within a non-monotonic landscape that is shaped by genetics and levels of gene expression. Highlights MAPK signaling drives proliferation and conversion of fibroblasts to motor neurons Cell-fate programming responds biphasically to HRASG12V expression High HRASG12V expression induces senescence, which reduces conversion Chemogenetic tuning of MAPK activity increases conversion rates A small-molecule MAPK inducer drives high rates of conversion in the absence of HRASG12V">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fa4a4e600cf88c5b704f193ed57b3b5cfc188f0" target='_blank'>
              Chemogenetic tuning reveals optimal MAPK signaling for cell-fate programming
              </a>
            </td>
          <td>
            Brittany A. Lende-Dorn, Jane C. Atkinson, Yunbeen Bae, Kate E. Galloway
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Originally derived from the immune system of bacteria, CRISPR-Cas9 technology has rapidly developed into a revolutionary gene editing tool that is widely used in molecular biology, drug development, and gene therapy. This technology not only provides a new treatment approach for genetic diseases such as sickle cell anemia and Duchenne muscular dystrophy, but also brings breakthrough progress in cancer treatment by enhancing chimeric antigen receptor T cell (CAR-T) therapy. CRISPR-Cas9 has also shown great potential in the drug discovery stage, especially in gene screening, development of disease models, and drug resistance research. However, with the rapid development of CRISPR technology, the scientific community still needs to solve challenges such as potential off-target effects in gene editing, inefficient delivery systems, and long-term safety. In addition, the ethical issues of gene editing have also sparked widespread discussion, especially how to ensure fair access when treating genetic diseases. Despite many challenges, CRISPR-Cas9 technology still shows great application prospects in the future of drug development and gene therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5ad9a53620cc6ea8c99c67cb73104e6fa93ba3d" target='_blank'>
              The Pharmaceutical Applications of CRISPR-Cas9 and Its Development Trends
              </a>
            </td>
          <td>
            Yu Sang
          </td>
          <td>2024-12-31</td>
          <td>MedScien</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Engineering cell fate is fundamental for optimizing stem cell-based therapies aimed at replacing cells in patients suffering from trauma or disease. By timely administering molecular regulators—such as transcription factors, RNAs, or small molecules—in a process that mimics in vivo embryonic development, stem cell differentiation can be guided toward a specific cell fate. A significant challenge in scaling up these therapies is that such differentiation strategies often result in mixed cellular populations. While synthetic biology approaches have been proposed to increase the yield of desired cell types, designing gene circuits that effectively redirect cell fate decisions requires mechanistic insight into the dynamics of endogenous regulatory networks that govern decision-making. In this work, we present a biomolecular adaptive controller based on an Incoherent Feedforward Loop (IFFL)-like topology designed to favor a specific cell fate. This controller requires minimal knowledge of the endogenous network as it exhibits adaptive, non-reference-based behavior. The synthetic circuit operates through a sequestration mechanism and a delay introduced by an intermediate species, producing an output that asymptotically approximates a discrete temporal derivative of its input, provided there is a sufficiently fast sequestration rate. By allowing the controller to actuate over a target species involved in the decision-making process, a tunable, synthetic bias is created that favors the production of the desired species with minimal alteration to the overall equilibrium landscape of the endogenous network. Through theoretical and computational analysis, we provide design guidelines for the controller’s optimal operation, evaluate its performance under parametric perturbations, and extend its applicability to various examples of common multistable systems in biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b540a64fb3aa27b889f01e2b17a8cadd0ea0d35" target='_blank'>
              Engineering cell fate with adaptive feedback control
              </a>
            </td>
          <td>
            Frank Britto Bisso, Giulia Giordano, Christian Cuba Samaniego
          </td>
          <td>2024-11-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fecaeaf832acff0ef3a7b992effb9b3cb46a2f35" target='_blank'>
              TORC2 inhibition triggers yeast chromosome fragmentation through misregulated Base Excision Repair of clustered oxidation events
              </a>
            </td>
          <td>
            Kenji Shimada, Cleo V D Tarashev, Stephanie Bregenhorn, C. Gerhold, Barbara van Loon, Gregory Roth, Verena Hurst, Josef Jiricny, S. B. Helliwell, Susan M. Gasser
          </td>
          <td>2024-11-15</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>24</td>
        </tr>

        <tr id="The mechanisms that drive immune feedback about cancer have been thoroughly explored over the years, particularly with a focus on utilizing the therapeutic potential of the immune system. Cancer immunotherapy has emerged as a promising treatment strategy for various types of cancer. Methods such as CAR T-cell therapy, adoptive T-cell therapy, monoclonal antibodies (mAbs), and cancer vaccines have gained significant attention. Nonetheless, the full potential of cancer immunotherapy has yet to be realized. While it has remarkable attributes, cancer immunotherapies also face challenges, including limited ability to effectively target cancer antigens and the variability in patient responses. One aspect of genome-based immunotherapy that has evolved due to technological advancements is the development of engineered T cells. CRISPR-Cas9 has emerged as a powerful and versatile genome-editing tool capable of targeting nearly any genomic site due to its exceptional precision. This review centers on recent progress in immunotherapeutic strategies for cancer, particularly highlighting the application of CRISPR-Cas9 technology as a promising weapon in the fight against cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77bbcb16cf63f04ea8a6e028b8461008a35d2dab" target='_blank'>
              CRISPR/Cas9 potential applications in cancer immunotherapy by gene-editing and immune checkpoint signaling pathway inhibition
              </a>
            </td>
          <td>
            Mehab Ishfaq
          </td>
          <td>2024-12-31</td>
          <td>Asia-Pacific Journal of Surgical &amp; Experimental Pathology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Breast cancer metastases can go undetected for long periods of time, are highly resistant to therapy, and are the cause for more than 90% of breast cancer mortality. Dormancy is a critical step in the metastatic cascade, whereby disseminated cancer cells (DCCs) enter a non-proliferative state at distal sites until they reactivate and give rise to overt metastasis, often years after disease remission. As such, dormancy represents a major clinical obstacle in breast cancer patient care. Chromatin remodeling serves as a foundation for the cellular reprogramming required to drive cell fate transitions. However, little is known about the nature of this remodeling in dormancy. Here, we hypothesize that differential deposition of histone H3.3 determines whether DCCs enter and remain dormant in secondary organs or proliferate and form overt metastases. We used a combination of H3.3 deposition sensors and genetic tools to manipulate H3.3 and HIRA in vitro and in vivo to track H3.3 deposition through the metastatic cascade and evaluate its importance for dormancy. To evaluate relevance to patient disease we developed a dormancy score to implement from publicly available RNA seq. To define mechanism of regulation, we used a combination of whole genome sequencing (ChIP-seq, ATAC-seq and RNA-seq) followed by in-depth analysis and functional validation. Our data demonstrates that H3.3 incorporation at secondary sites is heterogenous, revealing a subset of DCCs with low H3.3 in chromatin. Blocking H3.3 deposition onto chromatin leads to a reversible arrest in G0/G1 which tracks with the activation of canonical dormancy signaling. Mechanistically, our data supports a model where blocking H3.3 incorporation into chromatin directly represses the expression of SKP2, the main regulator of progression through the restriction point of cell cycle via regulation of p27 degradation. SKP2 suppression in turn leads to the accumulation of p27 thereby causing cell cycle arrest in G0/G1 and dormancy establishment. We traced this H3.3-mediated regulation of dormancy through the dynamic regulation of H3.3 histone chaperone, HIRA, whose levels inversely correlate with dormancy in experimental models as well as in patient samples. We found that HIRA levels respond to the metastatic microenvironment where fibrotic conditions induce HIRA expression and trigger metastatic outgrowth in an H3.3-dependent manner. Together, our work unveils for the first time a patient-relevant epigenetic mechanism, impinging on H3.3 deposition, as a major regulatory point of DCCs’ cell fate at secondary organs controlling both entry of DCC’s into dormant state as well as their reactivation and thriving as overt metastasis.
 Citation Format: Stanislav Drapela, Cristina Megino-Luque, Rojan Chimeh Rad, Devesh Raizada, Nadir Sarigul, Didem Ilter, Brian J. Czerniecki, Jose Javier Bravo-Cordero, Ana P. Gomes. Microenvironment driven dynamic deposition of histone H3.3 controls entry and exit from dormancy in disseminated cancer cells [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr A013.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5879de8eccb4afc8b39e0e3158aebd1bfd3c6ac" target='_blank'>
              Abstract A013: Microenvironment driven dynamic deposition of histone H3.3 controls entry and exit from dormancy in disseminated cancer cells
              </a>
            </td>
          <td>
            S. Drápela, Cristina Megino-Luque, Rojan Chimeh Rad, Devesh Raizada, Nadir Sarigul, Didem Ilter, B. Czerniecki, J. Bravo-Cordero, Ana P. Gomes
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML), a malignant disease of the bone marrow, is characterized by the clonal expansion of myeloid progenitor cells and a block in differentiation. The high heterogeneity of AML significantly impedes the development of effective treatment strategies. Enhancer of zeste homolog 2 (EZH2), the catalytic subunit of the polycomb repressive complex 2 (PRC2), regulates the expression of downstream target genes through the trimethylation of lysine 27 on histone 3 (H3K27me3). Increasing evidence suggests that the dysregulation of EZH2 expression in various cancers is closely associated with tumorigenesis. In the review, we examine the role of EZH2 in AML, highlighting its crucial involvement in regulating stemness, proliferation, differentiation, immune response, drug resistance and recurrence. Furthermore, we summarize the application of EZH2 inhibitors in AML treatment and discuss their potential in combination with other therapeutic modalities. Therefore, targeting EZH2 may represent a novel and promising strategy for the treatment of AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c772976d3b58810f697b6b1b73c79162dca59db" target='_blank'>
              The epigenetic role of EZH2 in acute myeloid leukemia
              </a>
            </td>
          <td>
            Jinyong Fang, Jingcheng Zhang, Lujian Zhu, Xiaoru Xin, Huixian Hu
          </td>
          <td>2024-12-06</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Interrogating regulatory epigenetic alterations during tumor progression at the resolution of single cells has remained an understudied area of research. Here we developed a highly sensitive single-nucleus CUT&RUN (snCUT&RUN) assay to profile histone modifications in isogenic primary, metastatic, and cisplatin-resistant head and neck squamous cell carcinoma (HNSCC) patient-derived tumor cell lines. We find that the epigenome can be involved in diverse modes to contribute towards HNSCC progression. First, we demonstrate that gene expression changes during HNSCC progression can be comodulated by alterations in both copy number and chromatin activity, driving epigenetic rewiring of cell states. Furthermore, intratumour epigenetic heterogeneity (ITeH) may predispose subclonal populations within the primary tumour to adapt to selective pressures and foster the acquisition of malignant characteristics. In conclusion, snCUT&RUN serves as a valuable addition to the existing toolkit of single-cell epigenomic assays and can be used to dissect the functionality of the epigenome during cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ccec03801dafc8df15febd551c3a9189ef4f702" target='_blank'>
              Single-nucleus CUT&RUN elucidates the function of intrinsic and genomics-driven epigenetic heterogeneity in head and neck cancer progression.
              </a>
            </td>
          <td>
            H. Womersley, Daniel Muliaditan, Ramanuj DasGupta, L. F. Cheow
          </td>
          <td>2024-12-02</td>
          <td>Genome research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Mammalian genomes contain millions of regulatory elements that control the complex patterns of gene expression. Previously, The ENCODE consortium mapped biochemical signals across many cell types and tissues and integrated these data to develop a Registry of 0.9 million human and 300 thousand mouse candidate cis-Regulatory Elements (cCREs) annotated with potential functions1. We have expanded the Registry to include 2.35 million human and 927 thousand mouse cCREs, leveraging new ENCODE datasets and enhanced computational methods. This expanded Registry covers hundreds of unique cell and tissue types, providing a comprehensive understanding of gene regulation. Functional characterization data from assays like STARR-seq, MPRA, CRISPR perturbation, and transgenic mouse assays now cover over 90% of human cCREs, revealing complex regulatory functions. We identified thousands of novel silencer cCREs and demonstrated their dual enhancer/silencer roles in different cellular contexts. Integrating the Registry with other ENCODE annotations facilitates genetic variation interpretation and trait-associated gene identification, exemplified by discovering KLF1 as a novel causal gene for red blood cell traits. This expanded Registry is a valuable resource for studying the regulatory genome and its impact on health and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99941d31f540b8743cd6f7ea4d2eb9891068be3e" target='_blank'>
              An Expanded Registry of Candidate cis-Regulatory Elements for Studying Transcriptional Regulation
              </a>
            </td>
          <td>
            Jill E. Moore, H. Pratt, K. Fan, Nishigandha Phalke, Jonathan Fisher, Shaimae I. Elhajjajy, Gregory Andrews, Mingshi Gao, Nicole Shedd, Yu Fu, Matthew C Lacadie, Jair Meza, Mohit Ganna, Eva Choudhury, Ross Swofford, Nina P. Farrell, Anusri Pampari, Vivekanandan Ramalingam, Fairlie Reese, Beatrice Borsari, Michelle Yu, Eve Wattenberg, M. Ruiz-Romero, Milad Razavi-Mohseni, Jinrui Xu, Timur Galeev, Michael A. Beer, R. Guigó, Mark B. Gerstein, Jesse M. Engreitz, Mats Ljungman, Timothy E. Reddy, M. Snyder, Charles B. Epstein, Elizabeth Gaskell, Bradley E. Bernstein, D. Dickel, Axel Visel, Len A. Pennacchio, A. Mortazavi, A. Kundaje, Zhiping Weng
          </td>
          <td>2024-12-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>125</td>
        </tr>

        <tr id="Background Uveal melanoma (UM), arising from melanocytes in the choroid, accounts for 3% to 5% of all melanocytic tumors and over 70% of intraocular malignancies. Despite effective local treatments, metastasis remains a significant challenge, with more than half of patients developing metastatic disease within ten years. Conventional therapies often yield poor outcomes, highlighting the urgent need for novel therapeutic strategies to enhance survival and prognosis for UM patients. Methods We conducted a detailed analysis of the GSE139829 dataset, focusing on scRNA-seq data from eight primary UM patients and three with metastatic disease. Through clustering and marker gene expression analyses, we identified distinct subtypes of UM tumor cells and examined their transcriptional, metabolic, and intercellular communication profiles. We developed a novel prognostic model, PCOLCE TCs Risk Score (PTRS), centered on the C5 PCOLCE+ tumor cells, which was validated through in vitro functional assays. Additionally, we performed immune infiltration and metabolic pathway analyses to elucidate tumor-immune interactions and their clinical significance. Results We identified eight distinct cell types in UM and classified tumor subpopulations into six subgroups. The C5 PCOLCE+ TCs subpopulation was highlighted as crucial in UM malignancy, demonstrating high differentiation potential and a significant role in tumor progression. CellChat analysis revealed substantial communication between C5 PCOLCE+ TCs and fibroblasts, suggesting their involvement in tumor growth and extracellular matrix remodeling. Metabolic pathway analysis indicated enhanced oxidative phosphorylation and glutathione metabolism in this subpopulation. Additionally, we developed a PTRS model based on C5 PCOLCE+ TCs, identifying CITED1 as a high-risk gene that promotes UM cell proliferation, invasion, and migration in vitro. Conclusion This study provides insights into UM metastasis via single-cell analysis, identifying C5 PCOLCE+ TCs as key malignancy drivers associated with oxidative phosphorylation and immune interactions. Our PTRS model highlights CITED1 as a high-risk gene that promotes UM cell proliferation, paving the way for new prognostic models and therapeutic targets to enhance patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/960495014c1bd91ad32e09912185659c46495106" target='_blank'>
              Single-cell RNA sequencing reveals key molecular drivers and immune landscape in uveal melanoma: implications for targeted therapy and prognostic modeling
              </a>
            </td>
          <td>
            Zeyu Song, Wenwen Shao, Zhikai Xiahou, Yue Xu, Xiaofeng Zhang
          </td>
          <td>2024-11-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="CRISPR nuclease-mediated gene knock-out is limited by suboptimal sgRNAs, inaccessible target sites, and silent mutations. Here, we present a Cas12a-based system that targets each gene with four sgRNAs to overcome these limitations, using Drosophila as a tractable in vivo model. We show that multiplexed sgRNAs act synergistically to create deletions between target sites, substantially increasing the fraction of loss-of-function mutations. To systematically assess off-target effects, we developed a novel screening assay that visualizes CRISPR-induced chromosomal alterations in living animals. This enabled comprehensive screening of more than 2000 sgRNAs clustered in 525 quadruple arrays across 21 megabases of genomic DNA, revealing remarkably high on-target activity (100%, 82/82) and undetectable off-target cutting (0%, 0/443). Quantitative side-by-side comparisons with a current Cas9-based system targeting over 100 genes demonstrates that multiplexed Cas12a-mediated gene targeting achieves superior performance and reveals phenotypes missed by established methods. This highly efficient and specific system provides a framework for reliable functional genomics studies across diverse organisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f86216b216f0f25c0bea61c8d55e4328bad80925" target='_blank'>
              Enhanced in vivo gene knockout with undetectable off-targets using multiplexed Cas12a sgRNAs
              </a>
            </td>
          <td>
            F. Port, Martha A. Buhmann, Jun Zhou, Mona Stricker, Alexander Vaughan-Brown, Ann-Christin Michalsen, Eva Roßmanith, Amélie Pöltl, Lena Großkurth, Julia Huber, Laura B. Menendez Kury, Bea Weberbauer, Maria Hübl, Florian Heigwer, Michael Boutros
          </td>
          <td>2024-11-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="INTRODUCTION
Leukemia is typically categorized into myeloid leukemia and lymphoblastic leukemia based on the origins of leukemic cell. Myeloid leukemia is a group of clonal malignancies characterized by the presence of increased immature myeloid cells in both bone marrow and peripheral blood. Of note, the aberrant expression of specific proteins or the generation of fusion proteins due to chromosomal abnormalities are well established drivers in various forms of myeloid leukemia. Therefore, these oncoproteins represent promising targets for drug development.


AREAS COVERED
In this review, we comprehensively discussed the pathogenesis of typical leukemia oncoproteins and current landscape of small molecule drugs targeting these oncogenic proteins. Additionally, we elucidated novel strategies, including proteolysis-targeting chimeras (PROTACs), hyperthermia and genomic editing, which specifically degrade oncogenic proteins in myeloid malignancies.


EXPERT OPINION
Although small molecule drugs have significantly improved the prognosis of oncoprotein driven myeloid leukemia patients, drug resistance due to the mutations in oncoproteins is still a great challenge in clinic. New approaches such as PROTACs, hyperthermia and genomic editing are considered promising approaches for the treatment of oncoprotein-driven leukemia, especially for drug resistant mutants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ea5fe5c5cfd460bcc9a1a8e4861d057f9becb9a" target='_blank'>
              Therapeutic approaches targeting oncogenic proteins in myeloid leukemia: challenges and perspectives.
              </a>
            </td>
          <td>
            Xing Yi Yan, Yuan-yuan Kang, Ze Yan Zhang, Ping Huang, Chang Yang, Hua Naranmandura
          </td>
          <td>2024-12-16</td>
          <td>Expert opinion on therapeutic targets</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Recent massively-parallel approaches to decipher gene regulatory circuits have focused on the discovery of either cis-regulatory elements (CREs) or trans-acting factors. Here, we develop a scalable approach that pairs cis- and trans-regulatory CRISPR screens to systematically dissect how the key immune checkpoint PD-L1 is regulated. In human pancreatic ductal adenocarcinoma (PDAC) cells, we tile the PD-L1 locus using ∼25,000 CRISPR perturbations in constitutive and IFNγ-stimulated conditions. We discover 67 enhancer- or repressor-like CREs and show that distal CREs tend to contact the promoter of PD-L1 and related genes. Next, we measure how loss of all ∼2,000 transcription factors (TFs) in the human genome impacts PD-L1 expression and, using this, we link specific TFs to individual CREs and reveal novel PD-L1 regulatory circuits. For one of these regulatory circuits, we confirm the binding of predicted trans-factors (SRF and BPTF) using CUT&RUN and show that loss of either the CRE or TFs potentiates the anti-cancer activity of primary T cells engineered with a chimeric antigen receptor. Finally, we show that expression of these TFs correlates with PD-L1 expression in vivo in primary PDAC tumors and that somatic mutations in TFs can alter response and overall survival in immune checkpoint blockade-treated patients. Taken together, our approach establishes a generalizable toolkit for decoding the regulatory landscape of any gene or locus in the human genome, yielding insights into gene regulation and clinical impact.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9b852684820c6d7cc2ac0a6db9f72463ef6384e" target='_blank'>
              Paired CRISPR screens to map gene regulation in cis and trans
              </a>
            </td>
          <td>
            Xinhe Xue, Zoran Z. Gajic, Christina M. Caragine, Mateusz Legut, Conor R. Walker, James Y.S. Kim, Xiao Wang, Rachel E. Yan, H. Wessels, Congyi Lu, Neil Bapodra, Gamze Gürsoy, Neville E. Sanjana
          </td>
          <td>2024-11-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="DNMT3A and TET2 are epigenetic regulator genes commonly mutated in age-related clonal hematopoiesis (CH). Despite having opposed epigenetic functions, these mutations are associated with increased all-cause mortality and a low risk for progression to hematological neoplasms. While individual impacts on the epigenome have been described using different model systems, the phenotypic complexity in humans remains to be elucidated. Here we make use of a natural inflammatory response occurring during coronavirus disease 2019 (COVID-19), to understand the association of these mutations with inflammatory morbidity (acute respiratory distress syndrome-ARDS) and mortality. We demonstrate the age-independent, negative impact of DNMT3A mutant CH on COVID-19-related ARDS and mortality. Using single cell (sc-) proteogenomics we show that DNMT3A mutations involve myeloid and lymphoid lineage cells. Using single cell multiomics sequencing, we identify cell-specific gene expression changes associated with DNMT3A mutations, along with significant epigenomic deregulation affecting enhancer accessibility, resulting in overexpression of IL32, a proinflammatory cytokine that can result in inflammasome activation in monocytes and macrophages. Finally, we show with single cell resolution that the loss of function of DNMT3A is directly associated with increased chromatin accessibility in mutant cells. Hence, we demonstrate the negative prognostic impact of DNMT3Amt CH on COVID-19 related ARDS and mortality. DNMT3Amt CH in the context of COVID-19, was associated with inflammatory transcriptional priming, resulting in overexpression of IL32. This overexpression was secondary to increased chromatic accessibility, specific to DNMT3Amt CH cells. DNMT3Amt CH can thus serve as a potential biomarker for adverse outcomes in COVID-19.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be6f734acb174ae5d3944a4074bf5564fa474087" target='_blank'>
              Single cell multiomics reveal divergent effects of DNMT3A and TET2 mutant clonal hematopoiesis in inflammatory response.
              </a>
            </td>
          <td>
            Wazim Mohammed Ismail, Jenna A. Fernandez, M. Binder, T. Lasho, Minsuk Kim, Susan M. Geyer, Amelia Mazzone, C. Finke, A. Mangaonkar, Jeong-Heon Lee, Liguo Wang, Kwan Hyun Kim, Vernadette A. Simon, Fariborz Rakhshan Rohakthar, Amik Munankarmy, Seul Kee Byeon, S. Schwager, Jonathan J Harington, Melissa R. Snyder, Keith D Robertson, Akhilesh Pandey, Eric D. Wieben, Nicholas Chia, Alexandre Gaspar-Maia, Mirinal S Patnaik
          </td>
          <td>2024-12-04</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="ABSTRACT Background CREB binding protein (CREBBP) is a key epigenetic regulator, altered in a fifth of relapsed cases of acute lymphoblastic leukemia (ALL). Selectively targeting epigenetic signaling may be an effective novel therapeutic approach to overcome drug resistance. Anti‐tumor effects have previously been demonstrated for GSK‐J4, a selective H3K27 histone demethylase inhibitor, in several animal models of cancers. Methods To characterize the effect of GSK‐J4, drug response profiling, CRISPR‐Dropout Screening, BH3 profiling and immunoblotting were carried out in ALL cell lines or patient derived samples. Results Here we provide evidence that GSK‐J4 downregulates cyclic AMP‐responsive element‐binding protein (CREB) and CREBBP in B‐cell precursor‐ALL cell lines and patient samples. High CREBBP expression in BCP‐ALL cell lines correlated with high GSK‐J4 sensitivity and low dexamethasone sensitivity. GSK‐J4 treatment also induced Bcl‐2 and Bcl‐XL dependency and apoptosis. Conclusions This study proposes H3K27 demethylase inhibition as a potential treatment strategy for patients with treatment‐resistant ALL, using CREBBP as a biomarker for drug response and combining GSK‐J4 with venetoclax and navitoclax as synergistic partners.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6268afa1f23d7dedcd6b2c8390d3c0a47ae3fe38" target='_blank'>
              Inhibiting H3K27 Demethylases Downregulates CREB‐CREBBP, Overcoming Resistance in Relapsed Acute Lymphoblastic Leukemia
              </a>
            </td>
          <td>
            Juan Lazaro-Navarro, C. Alcon, M. Dorel, Lina Alasfar, L. Bastian, Claudia Baldus, K. Astrahantseff, M. Yaspo, J. Montero, Cornelia Eckert
          </td>
          <td>2025-01-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Vγ9Vδ2T cells have the unique ability to recognize a broad range of malignant transformed cells. The tumor targeting event involving BTN2A1 and BTN3A1 dimers on the tumor cell surface is critical, leading to full activation of the TCR. Although the molecular mechanisms governing TCR engagement and T cell activation are well-characterized, the role of Vγ9Vδ2 T cells in cancer immune surveillance remains to be fully elucidated, particularly the mechanisms that enable these cells to discriminate between healthy and malignant cells at an early stage of malignant transformation. We employed two independent, genetically engineered step-wise mutagenesis models of human colorectal and breast cancer that mimic the transformation steps leading to tumor formation. We demonstrate that various single oncogenic mutations introduced into healthy organoids or cells, are sufficient to upregulate surface expressed BTN2A1 and enable Vγ9Vδ2 TCR binding to tumor cells. However, full activation of T cells through a Vγ9Vδ2TCR required additional subsequent phosphorylation of juxtamembrane (JTM) amino acids of BTN3A1, leading to the activating heterodimerization of BTN2A1 and 3A1. Using a protein interactome mapping pipeline, we identified PHLDB2, SYNJ2 and CARMIL1 as key players in controlling these delicate dual surface dynamics of BTN2A1 and 3A1 during early transformation. This mode of action allowed Vγ9Vδ2TCR T cells to control tumors in vitro and in vivo, emphasizing the crucial role of these molecules from early mutagenesis, to advanced cancer stages, and highlighting the therapeutic potential of a Vγ9Vδ2TCR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fae97086c70f2495b91617599cd421e999f208f8" target='_blank'>
              Sensitivity to Vγ9Vδ2TCR T cells is imprinted after single mutations during early oncogenesis
              </a>
            </td>
          <td>
            A. Cleven, A. Meringa, P. Brazda, Domenico Fascì, T. Koorman, Tineke Aarts, Inez Johanna, D. Beringer, P. Hernández-López, S. Heijhuurs, Tomohiro Mizutani, Sangho Lim, M. Huismans, Jochem Bernink, David Vargas Diaz, Wei Wu, Esther San Jose, Jelle Schipper, Nikos Tsakirakis, Lauren Hoorens van Heyningen, Annick Nouwens, Lucrezia Gatti, T. Straetemans, Hugo Snippert, Jeanine Roodhart, P. Derksen, J. Drost, M. Altelaar, Albert J. R. Heck, Hans Clevers, Juergen Kuball, Z. Sebestyén
          </td>
          <td>2024-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Defining consequential differences in intestinal epithelial stem cells in healthy humans versus those with inflammatory bowel disease (Crohn’s disease and ulcerative colitis) is essential for the development of much needed therapies to restore the epithelial barrier and maintain its fidelity. Employing single cell transcriptomic/epigenomic approaches and colonoid models from children and adults with Crohn’s disease led us to identify an inflammatory secretory progenitor (ISP) cell state present almost exclusively in patients with Crohn’s disease compared to control subjects. ISPs exhibit gene expression profiles consistent with normal secretory progenitor cells but concomitantly express a suite of distinguishing pro-inflammatory genes. Mechanistically, ISPs exhibit open chromatin and gene expression of ISP signature genes. While these ISP-specific genes are not expressed in intestinal stem cells, their chromatin is accessible in Crohn’s disease stem cells suggesting that ISP genes are epigenetically poised in stem cells and are transcriptionally activated in ISP cells in the presence of inflammatory stimuli. Consistently, Crohn’s disease colonoids exhibit sustained ISP gene expression that can be elicited further with pro-inflammatory cytokines or via co-culture with pro-inflammatory macrophages. In summary, we define differences in the epithelial stem and progenitor compartment of patients with Crohn’s disease suggesting aberrant stem cell differentiation and inflammatory gene expression arises during disease. HIGHLIGHTS We identify an inflammatory secretory progenitor cell state in endoscopically non-inflamed tissue from pediatric and adult patients with Crohn’s disease. Crohn’s disease epithelial stem and progenitor cells display increased chromatin accessibility preceding the emergence of inflammatory secretory progenitor cell states. Crohn’s disease colonoids from non-inflamed regions contain inflammatory secretory progenitor cells in the absence of inflammatory stimuli Inflammatory secretory progenitor cells expand in colonoid culture in response to cytokine stimulation or co-culture with pro-inflammatory macrophages">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f1c97ff6cf153588b934b86c33ad5324b1f6d33" target='_blank'>
              An epigenetic basis for sustained inflammatory epithelial progenitor cell states in Crohn’s disease
              </a>
            </td>
          <td>
            Tatiana A. Karakasheva, Clara Morral Martinez, Yusen Zhou, Jingya Qui, Xinyi E. Chen, Gloria E. Soto, Shaneice K. Nettleford, Olivia T. Hix, Daana M. Roach, Alyssa M. Laguerta, Anusha Thadi, Rachael M. Edwards, Daniel Aleynick, Sarah Weinbrom, Elizaveta Borodyanskaya, Oliver Pickering, Marykate Fulton, Chia- Chen, Bella V. Peterson, Erik B. Hagen, Ian P. Yannuzzi, Zainab Haider, Zvi Cramer, Maire C Conrad, Ning Li, Meenakshi Bewtra, Yasin Uzun, Kai Tan, J. Kelsen, Andy J. Minn, Christopher Joachim Lengner, Kathryn E. Hamilton
          </td>
          <td>2024-11-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Hematological cancers, such as lymphomas and leukemias, pose significant challenges in oncology, necessitating a deeper understanding of their molecular landscape to enhance therapeutic strategies. This article critically examines and discusses recent research on the roles of G protein-coupled receptors (GPCRs) in myeloma, lymphomas, and leukemias with a particular focus on pediatric acute lymphoblastic (lymphocytic) leukemia (ALL). By utilizing RNA sequencing (RNA-seq), we analyzed GPCR expression patterns in pediatric ALL samples (aged 3-12 years old), with a further focus on Class A orphan GPCRs. Our analysis revealed distinct GPCR expression profiles in pediatric ALL, identifying several candidates with aberrant upregulated expression compared with healthy counterparts. Among these GPCRs, GPR85, GPR65, and GPR183 have varying numbers of studies in the field of hematological cancers and pediatric ALL. Furthermore, we explored missense mutations of pediatric ALL in relation to the RNA gene expression findings, providing insights into the genetic underpinnings of this disease. By integrating both RNA-seq and missense mutation data, this article aims to provide an insightful and broader perspective on the potential correlations between specific GPCR and their roles in pediatric ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e366404105c19ea4ef94ffb4c3ea81b5651f0b0e" target='_blank'>
              Exploring G Protein-Coupled Receptors in Hematological Cancers.
              </a>
            </td>
          <td>
            Choi Har Tsang, P. Kozielewicz
          </td>
          <td>2024-11-21</td>
          <td>ACS pharmacology & translational science</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5afe3920a4f11e50f7df3ef73bd33525c2462f5e" target='_blank'>
              CRISPR-Enabled Autonomous Transposable Element (CREATE) for RNA-based gene editing and delivery.
              </a>
            </td>
          <td>
            Yuxiao Wang, , Meghan Harris, Bianca Lavayen, Neha Diwanji, Bruce McCreedy, Robert Hofmeister, Daniel Getts
          </td>
          <td>2025-01-09</td>
          <td>EMBO reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1dc81618c78e7092bf90be2157c330e9bb4fa14" target='_blank'>
              Investigating the role of KARS in lung adenocarcinoma via single-cell RNA sequencing
              </a>
            </td>
          <td>
            Huafang Hu, Yan Zhong, Guangbiao Li, Ru Deng, Jingfu Lin, Wenkun Wu, Yonghong Li
          </td>
          <td>2024-12-01</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/053c90f2e024c025545d0a836542f7f28e9a61c7" target='_blank'>
              KLF5 loss sensitizes cells to ATR inhibition and is synthetic lethal with ARID1A deficiency
              </a>
            </td>
          <td>
            Samah W. Awwad, Colm Doyle, Josie Coulthard, Aldo S. Bader, Nadia Gueorguieva, Simon Lam, Vipul Gupta, R. Belotserkovskaya, Tuan-Anh Tran, Shankar Balasubramanian, S. P. Jackson
          </td>
          <td>2025-01-08</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC), known for its hostile nature and limited treatment modalities, has spurred researchers to explore novel approaches for enhancing clinical outcomes. Here, the study aimed to analyze transcriptomics data to identify immune-related hub genes associated with TNBC that might serve as prognostic biomarkers. Initially, we determined genes that were differentially expressed between TNBC and normal tissues by integrating microarray and RNA sequencing data. Then, through protein-protein interaction and module analysis, we identified five putative hub genes: AURKA, CCNB1, CDCA8, GAPDH, and TOP2A. Subsequently, gene ontology and Kyoto Encyclopedia of Genes and Genomes pathway analyses revealed that the hub genes were primarily involved in the progesterone-mediated oocyte maturation signaling pathway and oocyte meiosis. Additionally, we observed that these five hub genes were significantly elevated at both protein and mRNA levels in TNBC tissues and contributed to worse survival. Furthermore, the expression of these hub genes exhibited a strong positive association with immune-invading cells such as CD8 T cells, CD4 T cells, and dendritic cells. The analysis of the regulatory network revealed three transcription factors (YBX-1, E2F1, and E2F3) and three posttranscriptional regulators (hsa-mir-25-3p, hsa-mir-92a-3p, and hsa-let-7b-5p) of hub genes. Finally, we explored potential drug candidates for the hub genes using Drug-Gene Interaction Database and discovered that there are no FDA-approved drugs for CCNB1 and CDCA8, highlighting a promising area for future research. Taken together, our results will be of immense importance in addressing the intricacies of TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf529bc9549c109c7e5d52dedaf7a1033e57f352" target='_blank'>
              Deciphering the prognostic landscape of triple-negative breast cancer: A focus on immune-related hub genes and therapeutic implications.
              </a>
            </td>
          <td>
            H. Krishnamoorthy, Ramanathan Karuppasamy
          </td>
          <td>2024-11-25</td>
          <td>Biotechnology and applied biochemistry</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="ABSTRACT Missing an entire chromosome or chromosome arm in normal diploid cells has a deleterious impact on cell viability, which may contribute to the development of specific birth defects. Nevertheless, the effects of chromosome loss in human cells have remained unexplored due to the lack of suitable model systems. Here, we developed an efficient, selection‐free approach to generate partial monosomy in human induced pluripotent stem cells (iPSCs). The introduction of Cas9 proteins and a pair of gRNAs induces over megabase‐sized interstitial chromosomal deletions. Using human chromosome 21 (HSA21) as a model, partial monosomy 21q (PM21q) iPSC lines with deletions ranging from 4.5 to 27.9 Mb were isolated. A 33.6 Mb deletion, encompassing all protein‐coding genes on 21q, was also achieved, establishing the first 21q monosomy human iPSC line. Transcriptome and proteome analyses revealed that the abundances of mRNA and protein encoded by the majority of genes in the monosomic regions are half of the diploid expression level, indicating an absence of dosage compensation. The ability to generate customized partial monosomy cell lines on an isogenic, karyotypically normal background should facilitate the gain of novel insights into the impact of chromosome loss on cellular fitness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f457fb7efd63146141d89c41994690b979076d69" target='_blank'>
              Generation of Monosomy 21q Human iPS Cells by CRISPR/Cas9‐Mediated Interstitial Megabase Deletion
              </a>
            </td>
          <td>
            Masaya Egawa, N. Uno, Rina Komazaki, Yusuke Ohkame, Kyotaro Yamazaki, Chihiro Yoshimatsu, Yuki Ishizu, Yusaku Okano, Hitomaru Miyamoto, Mitsuhiko Osaki, Teruhiko Suzuki, Kazuyoshi Hosomichi, Yasunori Aizawa, Yasuhiro Kazuki, Kazuma Tomizuka
          </td>
          <td>2024-11-24</td>
          <td>Genes to Cells</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Increased expression of branched-chain amino acid (BCAA) transaminase 1 (BCAT1) often correlates with tumor aggressiveness and drug resistance in cancer. We have recently reported that BCAT1 was overexpressed in a subgroup of T-cell acute lymphoblastic (T-ALL) samples, especially those with NOTCH1 activating mutations. Interestingly, BCAT1-depleted cells showed pronounced sensitivity to DNA-damaging agents such as etoposide; however, how BCAT1 regulates this sensitivity remains uncertain. Here, we provide further clues on its chemo-sensitizing effect. Indeed, BCAT1 protein regulates the non-homologous end joining (c-NHEJ) DNA repair pathway by physically associating with the KU70/KU80 heterodimer. BCAT1 inhibition during active repair of DNA double-strand breaks (DSBs) led to increased KU70/KU80 acetylation and impaired c-NHEJ repair, a dramatic increase in DSBs, and ultimately cell death. Our results suggest that, in T-ALL, BCAT1 possesses non-metabolic functions that confer a drug resistance mechanism and that targeting BCAT1 activity presents a novel strategy to improve chemotherapy response in T-ALL patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc99a18fefd0d2dfb9db2a0ff7404a49e575c24e" target='_blank'>
              BCAT1 Associates with DNA Repair Proteins KU70 and KU80 and Contributes to Regulate DNA Repair in T-Cell Acute Lymphoblastic Leukemia (T-ALL)
              </a>
            </td>
          <td>
            Valeria Tosello, Chiara Rompietti, A. Papathanassiu, G. Arrigoni, E. Piovan
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>15</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [12, 1],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>