<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-12-15 09:41:01 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="DHFR gene amplification is commonly present in methotrexate (MTX)-resistant colon cancer cells and acute lymphoblastic leukemia. In this study, we proposed an integrative framework to characterize the amplified region by using a combination of single-molecule real-time sequencing, next-generation optical mapping, and chromosome conformation capture (Hi-C). We identified an amplification unit spanning 11 genes, from the DHFR gene to the ATP6AP1L gene position, with high adjusted interaction frequencies on chromosome 5 (~2.2 Mbp) and a twenty-fold tandemly amplified region, and novel inversions at the start and end positions of the amplified region as well as frameshift insertions in most of the MSH and MLH genes were detected. These mutations might stimulate chromosomal breakage and cause the dysregulation of mismatch repair. Characterizing the tandem gene-amplified unit may be critical for identifying the mechanisms that trigger genomic rearrangements. These findings may provide new insight into the mechanisms underlying the amplification process and the evolution of drug resistance. Sequencing a large region of DNA containing many surplus copies of genes linked to drug resistance in colon cancer cells may illuminate how these genomic rearrangements arise. Such regions of gene amplification are highly repetitive, making them impossible to sequence using ordinary methods, and little is known about how they are generated. Using advanced methods, Jeong-Sun Seo at Seoul National University Bundang Hospital in South Korea and co-workers sequenced a region of gene amplification in colon cancer cells. The amplified region was approximately 20 times the length of that in healthy cells and contained many copies of an eleven-gene segment, including a gene implicated in drug resistance. The region also contained mutations in chromosomal repair genes which would disrupt repair pathways. These results illuminate the genetic changes that lead to gene amplification and drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental & Molecular Medicine, Experimental and Molecular Medicine</td>
          <td>5</td>
          <td>50</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum
                </a>
              </td>
          <td>
            Gustavo S. França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>75</td>
          <td>46</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Glioblastoma (GBM), the most common primary malignant brain tumor in adults, remains uniformly fatal due to the lack of effective targeted therapies. The epidermal growth factor receptor (EGFR) is the most frequently altered receptor tyrosine kinase oncogene in GBM with most alterations impacting the receptor ectodomain function, including gene amplification, mutation, rearrangement, and splicing site changes, which occur in approximately 50% of GBM tumors. Depatuxizumab mafodotin (Depatux-M; ABT-414), an antibody-drug conjugate composed of an EGFR-specific antibody (ABT-806) that recognizes the EGFR ectodomain linked to the cytotoxic agent monomethyl auristatin F, initially showed clinical promise. However, it failed to improve survival in phase III trials, highlighting an urgent need to understand mechanisms of resistance. We generated in vivo ABT-414 resistant GBM models using patient-derived xenografts (PDXs) and performed genomics and transcriptomic profiling, including whole exome sequencing, bulk RNA sequencing, and single-cell RNA sequencing. ABT-414-resistant tumors exhibited transcriptional reprogramming characterized by upregulation of synaptic and developmental gene networks and downregulation of biosynthetic processes, indicative of a plastic, therapy-adaptive state. Whole-exome sequencing revealed novel mutations exclusive to resistant tumors, including a recurrent TEK (TIE2) S466I point mutation present in all ABT-414 resistant GBM12 PDX tumors. Functional validation demonstrated that ectopic expression of TEK S466I and TEK WT in PDX models reduced EGFR levels, suggesting a novel feedback mechanism linking TEK signaling to EGFR downregulation and contributes to resistance. Our findings demonstrate that resistance to ABT-414 arises through both adaptive transcriptional remodeling and newly acquired genetic alterations. TEK-mediated suppression of EGFR represents a previously unrecognized mechanism of resistance, with potential implications for overcoming antibody-drug conjugate failure in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2178575c5331764ee967eeeb61b931918d25454a" target='_blank'>
              Glioblastoma resistance to EGFR antibody-drug conjugate is driven by transcriptional reprogramming and TEK-induced EGFR suppression
              </a>
            </td>
          <td>
            M. Blomquist, T. Noviello, C. Sereduk, Dustin Grief, F. Caruso, Ritin Sharma, K. Garcia-Mansfield, P. Pirrotte, J. Kloss, F. D’Angelo, B. Marin, K. Porath, Sonia Jain, Benjamin Rabichow, S. Ensign, A. Iavarone, Antonella Paladino, J. Sarkaria, Joseph C Loftus, M. Ceccarelli, Nhan L Tran
          </td>
          <td>2025-10-21</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="Multiple myeloma (MM) is a hematologic cancer marked by clonal expansion of plasma cells in the bone marrow. Although its genomic landscape has been extensively characterized, the transcriptional mechanisms that govern malignant progression and long-term tumor cell survival remain incompletely understood. We integrated single-cell RNA sequencing (scRNA-seq) data from healthy donors (HD), monoclonal gammopathy of undetermined significance (MGUS), smoldering multiple myeloma (SMM), and newly diagnosed multiple myeloma (NDMM) patients, retrieved from GEO datasets GSE124310 and GSE271107, to construct a comprehensive transcriptional landscape of plasma cell differentiation. Pseudotime and enrichment analyses identified PDIA4 as a terminal-state–associated gene. The prognostic significance of PDIA4 was validated using the multiple myeloma research foundation (MMRF) CoMMpass cohort. Functional analyses were performed in vitro and in vivo to validate the role of PDIA4 in MM cell survival and therapeutic response. Pseudotime trajectory analysis revealed progressive upregulation of genes involved in protein processing in the endoplasmic reticulum (ER), with PDIA4 identified as a top candidate in terminal-stage plasma cells. Survival analysis in the MMRF CoMMpass cohort further demonstrated that high PDIA4 expression correlated with poor overall survival. In RPMI-8226 cells, PDIA4 knockout activated the IRE1α/XBP1s branch of the unfolded protein response (UPR), impaired proliferation, and induced G1-phase arrest. PDIA4 depletion also sensitized cells to bortezomib. In vivo, sg-PDIA4 suppressed tumor growth in RPMI-8226 xenografts. PDIA4 is a key regulator of the unfolded protein response and MM cell survival. Targeting PDIA4 may enhance the efficacy of proteasome inhibitors and offers a potential strategy to overcome therapeutic resistance in multiple myeloma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/881fab1a35c430a7450d4d3fe7ea84a60cfe6f7c" target='_blank'>
              Single-cell transcriptomics identifies PDIA4 as a marker of progression and therapeutic vulnerability in multiple myeloma
              </a>
            </td>
          <td>
            Wenjie Yu, Jiamin Zhan, Yanxia Wang, Xu Cao, Aoyang Yu, Yuexuan Rao, Can Huang, Xuan Zhang, Shunv Lu, Yanhua Liu, Chong Chen, Zhenyu Li
          </td>
          <td>2025-10-21</td>
          <td>Journal of Translational Medicine</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="Understanding the complex molecular mechanisms driving drug resistance in glioblastoma (GBM) is crucial to develop effective therapeutic strategies. While prior studies have identified resistance mechanisms tied to specific drugs or pathways, a multi-modal molecular analysis of resistance across a range of drug classes is lacking. We identified highly drug-resistant (n = 5) and drug-sensitive (n = 4) cultures from a cohort of 32 patient-derived glioblastoma stem cell (GSC) cultures screened against a broad panel of ~ 500 anti-cancer drugs. To elucidate the key drivers of drug resistance, we performed integrative profiling of stemness, differentiation capacity, global gene expression, mutation profiles, and DNA methylation patterns between the two groups. Despite heterogeneous gene expression profiles, drug-resistant GSCs showed consistent upregulation of ATP-binding cassette (ABC) drug efflux transporters, stemness, and extracellular matrix (ECM)-related genes. Compared to drug-sensitive GSCs, drug-resistant GSCs exhibited more pronounced stem-like properties and reduced differentiation capacity. Notably, genes linked to axonogenesis displayed significant CpG island hypomethylation in drug-resistant GSCs. This study suggests a pivotal role for GSC plasticity, stemness maintenance, and ECM-mediated drug evasion in GBM treatment resistance. Our findings highlight the adaptive and dynamic nature of resistance mechanisms in GSCs, emphasizing the need for comprehensive molecular insights to inform targeted therapeutic strategies in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f39de902ec6cfcaa0e52251d953e6e404ad13d38" target='_blank'>
              Drug resistant glioblastoma stem cells exhibit enriched stemness signatures and share extracellular matrix overexpression
              </a>
            </td>
          <td>
            Lance L Estabillo, E. Skaga, S. Halldorsson, Einar O Vik-Mo, C. Sandberg
          </td>
          <td>2025-10-27</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Acute myeloid leukaemia (AML) with isolated trisomies (ITs) represents a distinct cytogenetic subgroup with heterogeneous clinical behaviour and incompletely defined molecular features. To explore its genomic and transcriptomic landscape, we performed next‐generation sequencing (NGS) on 14 AML patients harbouring isolated trisomies of chromosomes 8, 9, 10, 13, 14, 21 and 22. RNA sequencing (RNA‐Seq) was conducted on 15 samples, including 12 with IT and 3 cytogenetically normal AML cases (normal karyotype, NK‐AML) serving as controls. Trisomy 8 was most frequent, followed by chromosomes 13, 14 and 21. Recurrent mutations were identified in epigenetic regulators (DNMT3A, IDH1/2, ASXL1, TET2). Transcriptomic profiling stratified cases into IT‐8, IT‐21 and IT‐13+22 subgroups. Gene set enrichment analysis (GSEA) revealed shared downregulation of cell cycle‐related pathways (e.g., G2M checkpoint) and subgroup‐specific patterns: oxidative stress and unfolded protein response in IT‐8; epithelial‐mesenchymal transition and oxidative phosphorylation in IT‐21; inflammatory signalling (IL‐6/JAK/STAT, TNF‐α/NF‐κB) in IT‐13+22. A core set of 60 differentially expressed genes (DEGs) was shared, with nine hub genes related to cell cycle (MCM4, CDC7, CDC25A, DHFR), proteostasis (HSPA5, DNAJC3, CALR, HSP90B1) and inflammation. Drug sensitivity profiling revealed subgroup‐specific vulnerabilities: IT‐8 to DNA damage checkpoint inhibitors, IT‐21 to PLK/mTOR inhibitors and IT‐13+22 to BRAF/EGFR‐targeted agents. These findings highlight AML‐IT heterogeneity and therapeutic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34a07ac5d29af86fd0590c16e27dfff205a7ea13" target='_blank'>
              Integrated Genomic and Transcriptomic Profiling of Isolated Trisomies in AML Reveals Cell Cycle Dysregulation and Therapeutic Vulnerabilities
              </a>
            </td>
          <td>
            J. H. Maués, Bruno Kosa L Duarte, M. Svidnicki, Herton Luiz A S Filho, F. Niemann, Adriana da Silva S Duarte, Paula de Melo Campos, P. Moraes-Vieira, S. T. Olalla Saad
          </td>
          <td>2025-11-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8adbd0387ac6e2aa073dbe75080e4f98049b516f" target='_blank'>
              Acute Myeloid Leukemia Relapse after Bromodomain Inhibitor Treatment or Chemotherapy is Characterized by Myc-Ras Transcriptional Remodeling
              </a>
            </td>
          <td>
            Benjamin J. Huang, Jeevitha D’Souza, Alexa Rane Batingana, M. Harris, Xinyue Wang, E. Hwang, Anica M Wandler, Michael R. Burgess, Qing Li, S. Meshinchi, Gideon Bollag, Kevin Shannon
          </td>
          <td>2025-11-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="


 Despite the remarkable success of targeted therapy, the emergence of acquired resistance remains a major obstacle to cure acute myeloid leukemia (AML) patients. While existing studies have predominantly focused on genetic heterogeneity as a driver for therapy resistance, recent evidence suggests that non-genetic persister state is equally important, but such studies are lacking in AML. Here, we define the temporal transcriptome and chromatin landscape along with lineage trajectory of individual genetically identical AML persister clones to existing anti-AML therapeutics.
 To model persister state in AML, we utilized watermelon library (Oren et al., 2021), that expresses lineage barcode with inducible Histone-2B-mCherry to trace clonal evolution and cell division. We first evaluated whether slow-cycling, quiescent persisters emerge in AML lines after exposure to standard therapy. Our data showed the emergence of slow cycling persisters with cytarabine (AraC) in all 4 cell lines, and gilteritinib (GLT) in MOLM-13 and MV4-11 cells while BH3 mimetics (venetoclax (VEN), S63845 and navitoclax) had fast-cycling persisters. Integrating proliferative-index profiles with cell cycle profiles depicted extensive heterogeneity in AML persisters imposed by drug target rather than genetic state. Drug holiday experiments exhibited a reversible drug tolerance state in both slow and fast-cycling persisters. Longitudinal drug sensitivity and BH3 profiling assays showed that persisters emerged with adaptation to reduced mitochondrial priming that reversed after drug holiday. However, transitioning from persister to stable resistant cells resulted in a sustained loss in apoptotic priming, mirroring observations made in PDX models and primary tumors.
 To understand trajectory of persister emergence and underlying mechanisms, we performed single-cell multiomics on drug-naïve and persister populations (day 9) in MV4-11 (GLT), and OCI-AML2 (VEN and AraC). We expanded 10,000 unique barcoded cells and administered the drug treatment to derive persisters. Using lineage barcode analysis, we report significant reduction in the dominant lineage of day 0 and survival of multiple small lineages indicating an acquired state of persistence than selection of pre-existing resistant clone in both the lines. Further trajectory analysis defined three fates of the lineages as: expanding (log2FC>1), shrinking (log2FC <1) and stable. AraC persisters showed higher expanding and stable lineages, while VEN and GLT persisters resulted in higher shrinking and stable clones, revealing heterogeneity in lineage expansion. We next asked whether persisters resulting from different therapies arised from the same parental lineage. To test this, we performed lineage tracing on VEN and AraC persisters in OCI-AML2 and compared surviving lineages at Day 9 vs Day 0 cells. This analysis identified 52 parental lineages that were able to produce both VEN and AraC persisters in AML2, providing evidence of branched evolution of persisters from the same parental clone.
 We finally defined transcriptional and epigenetic heterogeneity in baseline and persister cells by performing UMAP on single-cell RNAseq and ATACseq data. We identified persisters to all 3 drugs clusters distinctly from their drug naïve counterparts at both transcriptome and chromatin accessibility. Next, we investigated differentially expressed pathways in persisters and observed higher inflammatory response pathways such as TNFα signaling in GLT, enrichment of MYC targets and oxidative phosphorylation in VEN and high mTORC1 signalling in AraC persisters. We observed higher chromatin accessibility in GLT and VEN whereas condensed chromatin in AraC persisters. We next asked which motifs are highly enriched in each persister and observed the enrichment of various KLF and SP family of motifs in each persister driving multiple gene signatures in each persister type. We also observed enrichment of LSC-primed and cDC like cells in GLT persisters but with AraC and VEN persisters a uniform pattern of all states was observed, indicated stemness may not be common mechanism to persister evolution.
 Collectively, our integrated single-cell multiomics and lineage tracing approach reveals that AML persisters emerge through stochastic drug-specific, non-genetic adaptive mechanisms characterized by distinct transcriptional and chromatin remodeling programs rather than deterministic Darwinian selection of pre-existing resistant cells.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b912c83b14f52bda36c8d81fafb30ac8e86a906c" target='_blank'>
              Dissection of non-genetic resistance mechanisms by lineage tracing and single cell multiomics reveals drug-specific persister programs in Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            K. Bhatia, Xiao Xian Lin, Ankit Jana, Emanuelle I. Grody, Ryan Jeremy Lim Jing Wei, Shawn Koh Jia Jun, Elisha Teh Jia Yu, Cin thet Kyi, Prasad Iyer, Greg Kellogg, Yogesh Goyal, Lawrence Boise, Yaara Oren, Shruti Bhatt
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Despite the outstanding achievements of precision medicine in hematology, many targeted therapies eventually fail due to the emergence of resistance mechanisms. Traditionally, a genocentric approach has been adopted to uncover the molecular underpinnings of treatment resistance. This has contributed to identifying resistance gene mutations and designing novel therapeutic molecules with increased potency for the mutant target. However, over the last five years, additional non-genetic adaptations have become increasingly recognized as crucial promoters of treatment resistance. In parallel, emerging works in the field of evolutionary biology suggest that advantageous phenotypic traits appear most often due to cell-intrinsic phenotypic plasticity and can arise independently of gene mutations. In selected cases, single genetic abnormalities such as those involving TP53 can prime human cells for plasticity and facilitate phenotypic variability. In this narrative review, we retrace the resistance mechanisms to targeted therapies in the framework of these novel evolutionary concepts. We highlight the dichotomy between genes-first and phenotypes-first pathways of treatment adaptation, with the former being driven by traditional single-point mutations and the latter initiated by the phenotypic diversity and the high-level plasticity of cancer cells. Focusing on resistance mechanisms to kinase inhibitors and BH3 mimetics in leukemias and lymphomas, we describe how each drug can trigger both escape routes, which may even coexist within the tumor bulk of individual patients. Lastly, we provide a three-step translational perspective on how to counteract phenotypes-first resistance mechanisms, with the aim of prolonging disease control in hematological malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/326e4b504dccc956ee299598bcd494d4433bc387" target='_blank'>
              Genes-first and phenotypes-first paths to treatment resistance in hematological malignancies
              </a>
            </td>
          <td>
            Edoardo Tamellini, Cristina Frusteri, Isacco Ferrarini
          </td>
          <td>2025-11-10</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/181bda14a56f6f64f5ac4fffa1ec57650a41f5a9" target='_blank'>
              Cooperative and antagonistic interactions between sub-clones favour the co-existence of multiple resistance mechanisms in melanoma
              </a>
            </td>
          <td>
            Karin Schlegelmilch, Viola Hollek, S. Hooper, Giovanni Giangreco, Sasha Bailey, Sarah C. Macfarlane, Alexandrine Carminati, Amy Bowes, S. Strohbuecker, Xiao Fu, E. Sahai
          </td>
          <td>2025-10-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Mantle cell lymphoma (MCL) responds to frontline therapy but is susceptible to relapse. While Bruton’s tyrosine kinase inhibitors (BTKi) achieve high response rates, most patients eventually experience disease progression. Predicting responses to subsequent treatments remains challenging due to the lack of an established platform. Heterogeneity in gene alterations and cellular pathways contribute to resistance, complicating treatment approaches. Here, we present a multi-modal profiling platform, targeting key pathways rather than focusing on singular DNA-associated lesions. We identified dysregulated signaling pathways by performing gene expression profiling on 20 MCL samples using a custom MCL MATCH gene set and analyzed the data with gene-set variation analysis. We also screened 22 therapeutics in vitro to assess their efficacy. Whole exome sequencing was conducted to identify prevalent and actionable mutations linked to enriched pathways. Finally, we tested selected therapeutics in patient-derived xenograft mouse models to predict potential response in corresponding patients. Based on our integrative profiling, we identified the top therapeutic candidates for three patients in this study. This integrative platform may help identify targeted therapies for BTKi-relapsed/refractory MCL patients in clinical settings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3929c1302d66ca38df0d5f835571ca5c0772a36" target='_blank'>
              Integrative profiling strategies to guide personalized therapy in mantle cell lymphoma: a pilot study
              </a>
            </td>
          <td>
            Yang Liu, H. Hill, Yijing Li, J. McIntosh, V. Jiang, F. Yan, Yixin Yao, Yue Fei, Jared R Zhang, Lawrence Qu, Jun Yao, P. Jain, Kent Chen, Michael Wang
          </td>
          <td>2025-11-21</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee85598703608ccef540311d86a0a9e7d9f4dae2" target='_blank'>
              Genetic barcoding of individual cells links cancer evolutionary trajectories and prognostic outcomes
              </a>
            </td>
          <td>
            Candice Merle, Ivan Di Terlizzi, M. Huyghe, Danae Welboren, Wenjie Sun, Cécile Conrad, Leïla Perié, S. Rulands, Silvia Fre
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 Despite the clinical success of the BCL2 inhibitor venetoclax in acute myeloid leukemia (AML), treatment resistance frequently emerges, leading to relapse. Mounting evidence suggests that metabolic reprogramming contributes to venetoclax resistance, yet the underlying mechanisms remain poorly defined. This study aimed to develop and characterize venetoclax-resistant AML models and explore metabolic vulnerabilities that could be exploited therapeutically.
 For this purpose, we established two venetoclax-resistant AML cell lines (MV4-11VR and MOLM-13VR) through repeated intermittent exposure to increasing doses of venetoclax. Resistance was assessed via cell viability assays, annexin V-based apoptosis analysis, mitochondrial membrane potential, cell cycle profiling, and colony formation assays. Metabolic reprogramming was evaluated using measurements of oxygen consumption rate (OCR) and extracellular acidification rate (ECAR). Molecular and genomic alterations were explored through transcriptomic profiling, global proteomics, and targeted sequencing. In addition, we tested the therapeutic potential of combining venetoclax with metabolic inhibitors, metformin, a mitochondrial complex I inhibitor, and KPT-9274, a NAMPT inhibitor, by calculating synergy scores and quantifying apoptotic responses.
 Venetoclax-resistant MV4-11VR and MOLM-13VR cells exhibited markedly elevated IC50 values (more than 200-fold compared to parental cells), impaired apoptosis induction, preserved proliferative capacity, and enhanced colony formation under venetoclax exposure. Notably, resistant cells showed distinct metabolic rewiring: MV4-11VR cells relied on enhanced glycolysis, whereas MOLM-13VR cells displayed increased oxidative phosphorylation (all p < 0.05). These findings were supported by functional assays and proteomic signatures, which showed enrichment of carbohydrate metabolism pathways in MV4-11VR and aerobic energy production in MOLM-13VR (all FDR < 0.05).
 At the molecular level, venetoclax resistance was accompanied by cell line-specific modulation of BCL2-family gene expression. MV4-11VR cells exhibited downregulation of multiple pro-apoptotic genes, including BAX, BCL2L11, BBC3, BIK, and BNIP3, while MOLM-13VR cells showed increased expression of MCL1 and BAD, along with reduced expression of BID, PMAIP1, and BMF. At the protein level, both models demonstrated reduced PARP1 cleavage and lower γH2AX levels upon venetoclax exposure, indicating suppression of apoptosis and DNA damage signaling. Furthermore, total BCL2 levels remained unchanged, while BCL-XL expression was stable in resistant cells but downregulated in parental cells upon drug exposure. A shared hallmark in both models was the hyperactivation of the PI3K/AKT/mTOR pathway, as evidenced by increased phosphorylation of the downstream effector RPS6. Additionally, MV4-11VR cells showed reduced ERK1/2 phosphorylation, whereas MOLM-13VR cells exhibited increased ERK1/2 activation.
 Genomic profiling revealed additional mechanisms contributing to resistance. MV4-11VR cells acquired a novel TP53 mutation (R158H) and showed clonal expansion of a pre-existing TP53 R248W mutation, alongside a mosaic deletion of the TP53 locus. In MOLM-13VR cells, sequencing revealed the emergence of a truncating mutation in the SPEN gene (I1326*) and acquisition of trisomy 5, further supporting distinct genetic trajectories of resistance.
 Therapeutically, venetoclax-resistant cells showed partial sensitivity to the metabolic inhibitors metformin and KPT-9274. Importantly, combination therapy with venetoclax significantly increased apoptosis in resistant cells. The venetoclax + KPT-9274 combination yielded strong synergy (ZIP scores up to 13.86) and eliminated over 95% of resistant cells, demonstrating the feasibility of targeting metabolic dependencies to restore drug sensitivity.
 Acquired venetoclax resistance in AML arises through heterogeneous yet targetable metabolic reprogramming. Distinct metabolic and molecular adaptations in resistant MV4-11 and MOLM-13 cells point to the complexity of resistance mechanisms. Nonetheless, the shared activation of survival pathways and metabolic plasticity creates vulnerabilities that can be exploited using mitochondrial or NAD+ biosynthesis inhibitors. These findings support a combinatorial therapeutic strategy targeting both apoptotic and metabolic pathways to overcome acquired venetoclax resistance in AML. Supported by FAPESP, CAPES, and CNPq.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c98bf25f48d9830834e2992fc3cf2049960a971f" target='_blank'>
              Metabolic reprogramming drives acquired venetoclax resistance in Acute Myeloid Leukemia and reveals targetable vulnerabilities
              </a>
            </td>
          <td>
            K. Lima, Gustavo Nery de Queiroz, Marcella Cipelli, Victória Tomaz, L. G. Cortes, Marina Basto Silva, R. Guedes, P. Campregher, Niels Câmara, L. Costa-Lotufo, E. Rego, J. Machado-Neto
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Relapse remains the leading cause of mortality in pediatric acute myeloid leukemia (AML), yet the genetic changes contributing to relapse remain incompletely defined. To address this gap, we performed whole-genome sequencing and targeted-capture sequencing on 39 diagnosis-relapse and 2 relapse-relapse pairs of pediatric AML. Mutational burden increased at relapse, largely reflecting spontaneous mutagenesis, whereas therapy-related signatures were rarely observed and only occasionally associated with pathogenic mutations. Although recurrently enriched mutations at relapse included those in FLT3, WT1, and TP53, relapse-fated subclones were frequently marked only by non-pathogenic or non-coding variants. Longitudinal deep sequencing in eight patients showed rapid depletion of major clones after induction therapy, whereas subclones often displayed variable chemosensitivity. Relapse-specific mutations emerged only late or remained undetectable during remission, suggesting that clonal selection of pre-existing clones is the predominant mechanism of relapse. Transcriptome analysis of paired RNA sequencing data revealed no differentially expressed genes, but gene set enrichment analysis and CIBERSORT deconvolution in each pair uncovered heterogeneous trajectories to relapse. Although relapse is often attributed to the emergence of stem-like phenotypes, our data demonstrate that transcriptional evolution is more diverse: some cases acquired stem-like features, whereas others showed partial differentiation, which was confirmed by re-analysis of a public single cell RNA sequence dataset. These changes were largely constrained by baseline differentiation states at diagnosis. Together, our data indicate that pediatric AML relapse arises through selection of pre-existing clones with diverse trajectories, underscoring the need to target both stem-like and differentiated populations to achieve durable cures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1b28a4cea8f32f994db559bdedf94de2b973cd5" target='_blank'>
              Clonal Evolution of Pediatric Acute Myeloid Leukemia and Its Contribution to Disease Relapse
              </a>
            </td>
          <td>
            Masayuki Umeda, P. Kolekar, A. Huskey, S. Suryaprakash, Jing Ma, Hanxia Li, Tamara Westover, Michael P Walsh, Guangchun Song, Yanling Liu, Q. Tran, V. Balagopal, Lu Wang, Samuel W Brady, Tanja A. Gruber, S. Pounds, Paul E. Mead, Seth E. Karol, Hiroto Inaba, J. Rubnitz, Raul C. Ribeiro, Xiaotu Ma, J. Klco
          </td>
          <td>2025-11-06</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1240efc02593321a375ca7bcb36e46ad57c6b703" target='_blank'>
              Frequent Whole-Genome Duplication Events Drive the Genomic Evolution of Triple-Negative Breast Cancer During Neoadjuvant Chemotherapy
              </a>
            </td>
          <td>
            Devin P Bendixsen, Fiona Semple, Alastair Ironside, Mustafa ̇Ismail Özkaraca, Natalie Wilson, A. Meynert, Ailith Ewing, Colin A. Semple, Olga Oikonomidou
          </td>
          <td>2025-11-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) is a highly aggressive malignancy characterized by rapid progression and the frequent emergence of resistance to standard chemotherapeutic agents such as cisplatin (DDP) and etoposide (VP16), resulting in poor clinical outcomes. To elucidate mechanisms underlying chemoresistance, we conducted a genome-wide CRISPR/Cas9 knockout screen, which identified the histone demethylase KDM6B as a critical mediator of drug resistance in SCLC. Genetic silencing of KDM6B significantly reduced IC₅₀ values of DDP and VP16, particularly in H69-AR cells, and enhanced chemotherapy-induced apoptosis. Consistently, pharmacological inhibition of KDM6B using the dual KDM6A/B inhibitor Gskj1 markedly potentiated the effects of DDP and VP16, while exhibiting minimal cytotoxicity as monotherapy. Overexpression of KDM6B rescued the chemosensitizing effect of Gskj1, thereby excluding confounding contributions from KDM6A. In vivo, the combination of Gskj1 with chemotherapy synergistically suppressed tumor growth without detectable systemic toxicity. To explore the downstream regulatory pathways, we performed transcriptome analysis via RNA-seq followed by KEGG pathway enrichment analysis, which revealed that Gskj1 treatment modulates key oncogenic signaling pathways. Integration of RNA-seq with H3K27me3 ChIP-seq data identified EGR3 as a direct epigenetic target of KDM6B inhibition. STRING analysis further suggested that EGR3 is co-expressed with c-FOS. Functional assays, including qRT-PCR, Western blotting, Co-immunoprecipitation (Co-IP), and dual-luciferase reporter assays, confirmed that EGR3 transcriptionally activates c-FOS, establishing an EGR3/c-FOS regulatory axis downstream of KDM6B. Mechanistically, inhibition of this axis enhanced chemosensitivity by promoting apoptosis, as evidenced by activation of caspase signaling, and by inducing ferroptosis through downregulation of GPX4, upregulation of ACSL4, lipid peroxidation, and modulation of HO-1. Rescue experiments with Z-VAD and ferrostatin-1 further validated that both apoptosis and ferroptosis contribute to the chemosensitizing effects of KDM6B inhibition. Finally, in vivo experiments using patient-derived xenograft (PDX) models demonstrated that Gskj1 effectively enhances the antitumor efficacy of chemotherapy in SCLC, providing compelling evidence for the clinical potential of targeting KDM6B to overcome chemoresistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08a98cf22fed7bec387cd4e67f856e9e79cc0e4f" target='_blank'>
              KDM6B inhibition enhances chemotherapeutic response in small cell lung cancer via epigenetic regulation of apoptosis and ferroptosis
              </a>
            </td>
          <td>
            Zhongliang Wang, Ziyuan Liu, Yufan Yang, Wenyang Wang, Qian Chen, Shumei Liang, Linlang Guo, Man Li
          </td>
          <td>2025-11-26</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Breast cancer recurrence remains a major clinical challenge, often associated with therapy resistance and altered metabolic states. To define metabolic vulnerabilities of recurrent disease, we performed a CRISPR knockout screen targeting 421 metabolic genes in paired primary and recurrent HER2-driven breast cancer cell lines. While both primary and recurrent tumors shared dependencies on core metabolic pathways, recurrent tumors exhibited selective essentiality for the de novo pyrimidine synthesis pathway, including Cad, Dhodh, and Ctps. Pharmacologic inhibition of the rate-limiting enzyme DHODH with BAY-2402234 selectively impaired the growth of recurrent tumor cells, while primary tumor cells were relatively resistant. BAY treatment robustly inhibited pyrimidine synthesis in all lines, but only recurrent cells underwent iron-dependent lipid peroxidation and ferroptotic cell death. Lipidomic profiling revealed enrichment of polyunsaturated ether phospholipids in recurrent cells, which may predispose them to ferroptosis. A sensitizer CRISPR screen in primary cells further identified nucleotide salvage and lipid metabolic pathways as modifiers of DHODH inhibitor sensitivity. Stable isotope tracing and nutrient depletion experiments showed that primary cells can compensate for DHODH inhibition through nucleotide salvage, whereas recurrent cells exhibit impaired salvage capacity, likely due to reduced expression of Slc28/Slc29 nucleoside transporters. Together, these findings reveal that breast cancer recurrence is associated with increased dependence on de novo pyrimidine synthesis to suppress ferroptosis, highlighting a therapeutically actionable metabolic vulnerability in recurrent disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83ca3ef9a21a542254ed4d0723e6bc92c26d2a66" target='_blank'>
              Recurrent Breast Cancer Cells Depend on De novo Pyrimidine Biosynthesis to Suppress Ferroptosis
              </a>
            </td>
          <td>
            Kelley R. McCutcheon, Josh Wu, Yasemin Ceyhan Ozdemir, Brock J McKinney, Sharan Srinivasan, Ayaha Itokawa, Oliver J. Newsom, Anna-Lena B G Vigil, Douglas B. Fox, Lucas B. Sullivan, James V. Alvarez
          </td>
          <td>2025-11-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Head and neck squamous cell carcinoma (HNSCC) remains a prevalent and aggressive malignancy, characterized by a lack of targeted therapies and limited clinical benefits. Here, we conducted an optimized whole-genome CRISPR screen across five HNSCC cell lines aimed at identifying actionable genetic vulnerabilities for rapid preclinical evaluation as novel targeted therapies. Given their critical role in cancer, cyclin-dependent kinases (CDKs) were prioritized for further investigation. Among these, CDK7 was identified as an essential and targetable gene across all five cell lines, prompting its selection for in-depth functional and molecular characterization. Genetic and pharmacological inhibition of CDK7 significantly and consistently reduced tumor cell proliferation due to generalized cell cycle arrest and apoptosis induction. Additionally, CDK7 knockout (KO) and selective inhibitors (YKL-5-124 and samuraciclib) demonstrated potent antitumor activity, effectively suppressing tumor growth in HNSCC patient-derived organoids (PDOs), as well as in both cell line- and patient-derived xenograft (PDX) mouse models with minimal toxicity. Mechanistically, CDK7 inhibition led to a broad downregulation of gene sets related to cell cycle progression and DNA repair, and significantly reduced the transcription of essential genes and untargetable vulnerabilities identified by our CRISPR screen. These findings highlight CDK7 as a promising therapeutic target for HNSCC. Our study provides strong evidence of the robust antitumor activity of CDK7-selective inhibition in disease-relevant preclinical models, strongly supporting its progression to clinical testing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4085230d5db63fd39c16115dd67e53fe6113079" target='_blank'>
              CDK7-targeted therapy effectively disrupts cell cycle progression and oncogenic signaling in head and neck cancer
              </a>
            </td>
          <td>
            María Otero-Rosales, M. Álvarez-González, Irene Pazos, Beatriz de Luxán-Delgado, Sonia Del Marro, Esperanza Pozo-Agundo, Mar Rodríguez-Santamaría, Ana López-Fernández, D. Corte-Torres, R. Granda-Díaz, S. Álvarez-Teijeiro, Iván Fernández-Vega, Corina Lorz, Ramón García-Escudero, J. P. Rodrigo, Konstantinos Tzelepis, G. Vassiliou, Irene Ferrer, Mónica Álvarez-Fernández, J. García-Pedrero, F. Hermida-Prado
          </td>
          <td>2025-11-06</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) play a crucial role in colorectal cancer by sustaining intratumoral heterogeneity, therapeutic resistance, and metastatic potential. CD133 (PROM1) is among the most frequently used surface markers for CSC identification, whereas TRIM28, a versatile epigenetic regulator, has been implicated in controlling CD133 expression and stem-like features. In this study, we performed a detailed molecular and functional analysis of Caco2 colorectal cancer cell clones with individual knockouts of CD133 or TRIM28. Elimination of CD133 neither altered global gene expression, as confirmed by transcriptome profiling, nor affected key cellular properties. In contrast, loss of TRIM28 led to a marked reduction in CD133 protein abundance and induced extensive molecular and phenotypic remodeling. TRIM28 knockout was associated with broad transcriptomic changes involving more than 500 differentially expressed genes, decreased proliferative activity monitored by time-lapse imaging, and reduced sensitivity to paclitaxel, cisplatin, and curcumin. Furthermore, immune evasion molecules CD24 and CD47 (“don’t eat me” signals) were strongly upregulated in TRIM28-deficient cells, consistently confirmed by both RNA-Seq and flow cytometry analyses. At the same time, imaging flow cytometry and mitochondrial activity assays indicated that these effects were not due to major shifts in mitotic index or bioenergetic status. Altogether, our results demonstrate that TRIM28, rather than CD133, functions as a central regulator of CSC-associated phenotypes in colorectal cancer. These findings highlight the importance of epigenetic context in CSC biology and may inform the development of more effective therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82a3d3eb05c9db6b57d94b54e8bdad26455b3ea6" target='_blank'>
              Contrasting Impacts of Targeted Disruption of the Cancer Stem Cell Marker CD133 and Its Epigenetic Regulator TRIM28 in Colorectal Cancer Cells
              </a>
            </td>
          <td>
            I. Kholodenko, A. Saidova, D. Potashnikova, V. A. Arzumanian, D. Romashin, A. Tvorogova, E. Poverennaya, K. Yarygin, Yan S. Kim
          </td>
          <td>2025-11-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25d53c86924fae7e6d4164c1b616a447b8ef85b5" target='_blank'>
              Comprehensive and accelerated mapping of driver mutations through single-nucleotide random mutagenesis of target genes
              </a>
            </td>
          <td>
            Keisuke Yoshida, Elena Solovieva, N. Miura, Shigeyuki Shichino, Michiaki Hamada, M. Muratani, Takanori Amano, Kazufumi Honda
          </td>
          <td>2025-11-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Highly accessible genomic super-enhancers often drive tumor-promoting programs, yet the impact of genomic variation within these regulatory elements remains unclear. Here, we identified a bifunctional super-enhancer that regulates the expression of cancer-promoting genes LINC00636 and CD47 in breast cancer. We discovered that a common germline insertion variant within the super-enhancer is associated with reduced chromatin accessibility at the super-enhancer locus. Deletion of the insertion in breast cancer cells increased chromatin accessibility, leading to upregulation of LINC00636 and CD47, enhanced resistance to nutrient-deprivation–induced apoptosis (mediated by CD47), activation of senescence (driven by elevated LINC00636), delayed cell death, and reduced infiltration of CD80⁺ pro-inflammatory macrophages, changes that represent tumor-promoting features. Together, our findings uncover a common insertion-deletion variant that fine-tunes the regulatory activity of a bifunctional super-enhancer, suggest a protective role for the insertion allele, and establish a novel function for LINC00636 in senescence and breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b015f1ff776d845bca2d14843c5dc6acc6da18a2" target='_blank'>
              An InDel Genomic Variant within a Bifunctional Super-Enhancer for LINC00636 and CD47 Regulation in Breast Cancer
              </a>
            </td>
          <td>
            Carolina Di Benedetto, Amelia Tsark, D. Acenas, Alysia Thach, Anmol Singhal, Valentina Opazo-Mellado, Anthony Rodriguez Lemus, Mustapha El Zeini, Hui Zhang, Catherine Park, Hugo Gonzalez Velozo, Irving Weissmann, Paola Betancur
          </td>
          <td>2025-11-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Acute myeloid leukemia is a genetically and cellularly heterogeneous disease. We characterize 120 AMLs using genomic and transcriptomic analyses, including single-cell RNA sequencing. Our results reveal an extensive cellular heterogeneity that distorts the bulk transcriptomic profiles. Selective examination of the transcriptional signatures of >90,000 immature AML cells identifies four main clusters, thereby extending current genomic classification of AML. Notably, NPM1-mutated AML can be stratified into two clinically relevant classes, with NPM1class I associated with downregulation of MHC class II and excellent survival following hematopoietic stem cell transplantation. NPM1class II is instead associated with resistance to allogeneic T cells in an ex vivo co-culture assay, and importantly, dismal survival following hematopoietic stem cell transplantation. These findings provide insights into the cellular state space of AML, define diagnostic entities, and highlight potential therapeutic intervention points. The clinical outcomes and treatment responses of acute myeloid leukemia (AML) patients are heterogeneous. Here, the authors use bulk and single-cell sequencing approaches and identify two transcriptomic subtypes within NPM1-mutated AML with distinct immune evasion properties and responses to hematopoietic stem cell transplantation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0674b44d7df57e733fca27fc81a60b0eeaf92970" target='_blank'>
              The AML cellular state space unveils NPM1 immune evasion subtypes with distinct clinical outcomes
              </a>
            </td>
          <td>
            H. Lilljebjörn, Pablo Peña-Martínez, Hanna Thorsson, Rasmus Henningsson, M. Rissler, N. Landberg, Noelia Puente-Moncada, Sofia von Palffy, Vendela Rissler, Petr Stanek, Jonathan Desponds, Xiangfu Zhong, Gunnar Juliusson, Vladimir Lazarevic, Sören Lehmann, Magnus Fontes, H. Ågerstam, Carl Sandén, Christina Orsmark-Pietras, T. Fioretos
          </td>
          <td>2025-11-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ceedd73d9b5892d7eca2a430c0516d638f377e49" target='_blank'>
              Targeting of ibrutinib resistance–driving pathways by miR-28 in ABC-DLBCL
              </a>
            </td>
          <td>
            Emigdio Álvarez-Corrales, Carmen Gómez-Escolar, Rocío Moreno-Palomares, Mario Martínez, María Laguna-Herrero, T. Fuertes, Belén S. Estrada, S. Mur, Nuria Martínez-Martín, Álvaro Somoza, A. Ramiro, V. G. de Yébenes
          </td>
          <td>2025-11-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19e1efee6c0706220f7b18885b5b5a4638edf605" target='_blank'>
              A single-nucleus multimodal framework reveals epigenomic priming of chemoresistant states in ovarian cancer
              </a>
            </td>
          <td>
            Y. Landais, J. Bertorello, M. Puerto, Agathe Kahn, B. Simon, A. Roehrig, Adeline Durand, M. Quatredeniers, C. Lamy, E. Laas, Nicolas Pouget, Simon Dumas, J. Féron, Virginie Fourchotte, C. Bonneau, Vincent Cockenpot, Marianne Delville, Magali Richard, M. Kamal, F. Lécuru, C. L. Tourneau, C. Vallot
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Medulloblastoma (MB) is the most prevalent malignant brain tumor in children, exhibiting clinical and genomic heterogeneity. Of the four major subgroups, Group 3 tumors (MYC-MB), display high levels of MYC and metastasis rates. Despite treatment with surgery, radiation, and chemotherapy, patients with Group 3 MB are more likely to develop aggressive recurrent tumors with poor survival. To examine resistance mechanisms in this study, we perform single nuclei multiome analysis of matched primary and recurrent tumors. Therapy resistant Medulloblastoma demonstrates an expanded persistent progenitor population. Additionally, distinct chromatin landscapes link to altered transcription and correspond with metabolic reprogramming. In vivo modeling of radiation resistance exhibits similar chromatin-based metabolic reprogramming focused on wild-type isocitrate dehydrogenase (IDH1) activity. IDH1 inhibition reverses resistance-mediated chromatin changes and enables radiation re-sensitization. Ultimately, these findings demonstrate the efficacy of single-cell multiome analysis in elucidating resistance mechanisms and identifying targetable pathways for MYC-driven medulloblastoma. MYC-driven medulloblastoma is highly aggressive and associated with poor survival. Here, the authors perform single nuclei multi-omic analysis of matched primary and recurrent tumours and identify potential resistance mechanisms and targetable pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fba49308e533a39e32c7d3dcd263f21e4a690d4" target='_blank'>
              Single-cell multi-omics identifies metabolism-linked epigenetic reprogramming as a driver of therapy-resistant medulloblastoma
              </a>
            </td>
          <td>
            Bethany L. Veo, Dong Wang, J. DeSisto, Angela M Pierce, Breauna Brunt, P. Bompada, A. Donson, Andrew Goodspeed, Kiara Smart, Nicholas Foreman, Nathan A. Dahl, Rajeev Vibhakar
          </td>
          <td>2025-11-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Babesiosis is an emerging and re-emerging zoonosis that is prevalent worldwide, caused by over 100 Babesia species. These intracellular parasites efficiently invade host red bloods cells, replicate rapidly, and exit the infected cells to cause clinical symptoms. However, the underlying mechanisms of regulating this asexual stage is largely unknown. Here, we generated a chromosome-level reference assembly of a novel Babesia species, Babesia xinjiangensis. Using single-copy orthologous genes, we confirmed its phylogenetic relationships with other apicomplexan parasites and estimated its speciation time. We identified species-specific gene families and core gene families that could be responsible for species speciation and immune evasion. Furthermore, we also used a single-cell RNA-sequencing (scRNA-seq) protocol to uncover hidden transcriptional variations in the asexual stages of this unicellular Babesia parasite and its cell-to-cell heterogeneity. We inferred the replication cycle and performed a pseudotime analysis to speculate the gene expression profiles. Although the peak expression times of most epigenetic markers and transcription factors were confined to specific phases, BxAP2-M2 (GWHPERCV002055) is constantly expressed during asexual development progression. Genetic analyses revealed that BxAP2-M2 directly or indirectly regulates the expression of rhoptry proteins and membrane proteins, which may play critical roles in the parasite’s invasion of red blood cells and the merozoite morphology. Our findings provide valuable markers of asexual replication, including some that are specific to Babesia gametocytes, and regulators specific to distinct cell-cycle phases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fcc905e6b3b252b6e7bea19a4cf0e4af2d5a472" target='_blank'>
              Comparative genomics, integrated with single-cell sequencing and genetic analyses, reveal roles of transcription factor AP2-M2 in asexual replication of Babesia parasite
              </a>
            </td>
          <td>
            Jinming Wang, Yijun Chai, Yuxin Ye, Jifei Yang, Xinxin Zhang, Hong Yin, Guiquan Guan
          </td>
          <td>2025-11-01</td>
          <td>PLOS Pathogens</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Abstract Cancers are characterized by genomic instability events such as aneuploidy, chromosomal arm imbalances, and segmental copy number changes. These genomic features frequently define different subclones within a tumor. Single-cell DNA sequencing (scDNA-seq) identifies these large-scale genomic alterations that define subclonal features. However, scDNA-seq does not provide biological phenotypic information on individual subclones. Single-cell RNA sequencing (scRNA-seq) offers biological information but is less accurate in discovering genomic instability events. We developed a computational framework, scAlign, for integrating scRNA-seq and scDNA-seq from the same specimen and define subclonal cellular phenotypes at the resolution of individual cells. Subclones were defined by aneuploidy and chromosomal arm imbalance among primary and metastatic cancers. Using the cells in the G0/G1 phase, the extensive cellular sampling from both assays characterized the subclonal architecture of these cancers. The scDNA-seq provided a ground truth for copy number-based subclones. From the scRNA-seq data, the epithelial cells in G0/G1 were identified and assigned to specific subclones by the scAlign based on gene dosage. Afterward, we determined the differential gene expression and biological pathway activities of specific clones. Overall, integrative multi-omics analysis of single-cell datasets is more informative than any individual genomic modality, provides deep insights into intratumoral heterogeneity and reveals subclonal biology. scAlign is available at https://github.com/XQBai/scAlign.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a91a097601e22d4ca33e4ce8206425906a9876da" target='_blank'>
              Single-cell aneuploidy and chromosomal arm imbalances define subclones with divergent transcriptomic phenotypes
              </a>
            </td>
          <td>
            Xiangqi Bai, Billy T. Lau, A. Sathe, Susan M. Grimes, Alison Almeda-Nostine, Hanlee P. Ji
          </td>
          <td>2025-10-31</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Transcriptional regulation is mediated by enhancers, yet how genetic perturbations alter enhancer activity and gene expression remains poorly understood. We developed UDI-UMI-STARR-seq, which integrates dual indexes and unique molecular identifiers, and combined it with RNA-seq to profile the effects of perturbations on enhancer activity and target gene expression. We applied this approach to a library of 253,632 fragments representing 46,142 cell type–specific candidate enhancers and assessed the impact of CRISPR/Cas9-mediated deletion of six transcription factors (or TFs; ATF2, CTCF, FOXA1, LEF1, TCF7L2, and SCRT1) with diverse regulatory roles. Across knockout lines, we identified responsive enhancers that were either repressed or induced, often through motifs such as the p53 family of TFs. Enhancer–gene mapping revealed TF-specific programs, including repression of Wnt/p53 targets with ATF2 or LEF1 loss, downregulation of the FIRRE locus with CTCF loss, and compensatory upregulation of RNA polymerase II regulators following FOXA1 depletion. A deep learning model trained on enhancer sequences recapitulated core principles of enhancer grammar, including cooperative motif syntax and the influence of flanking sequence context. Applying this framework to the neurodevelopmental disorder-associated 16p12.1 deletion identified responsive enhancers linked to genes involved in axon guidance, synaptic plasticity, and translational control, providing a scalable readout of enhancer dynamics generalizable to genetic mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e599253ce970a00d4735ed417209ae82c988f97" target='_blank'>
              Quantifying the impact of genetic mutations on enhancer dynamics
              </a>
            </td>
          <td>
            Maitreya Das, Deepro Banerjee, Ayaan Hossain, Matthew Jensen, S. Mogre, Jiawan Sun, Joseph Mao, Adam B Glick, Howard M Salis, S. Girirajan
          </td>
          <td>2025-10-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="One of the seminal discoveries from genetic studies of autism spectrum disorder and related neurodevelopmental disorders (NDD) has been that loss-of-function (LoF) mutations in many genes that impact chromatin and transcriptional regulation confer substantial liability to NDD. Haploinsufficiency of the epigenetic regulator POGZ represents one of the strongest such associations; however, little is known about the direct or indirect regulatory targets of POGZ, or the mechanisms by which loss of this chromatin modifier alters early neuronal development and synaptic functions. Here, we created an allelic series of CRISPR-engineered human induced pluripotent stem cell (hiPSC) clones harboring mono- and biallelic POGZ deletions. In hiPSC-derived neural stem cells (NSC) and Neurogenin 2-induced neurons (iN), POGZ LoF altered the expression of genes associated with critical cellular processes and neuronal functions, including synaptic and intracellular signaling and extracellular matrix organization. Our multiomics profiling also showed altered footprinting of critical transcription factors (e.g., activator protein 1 complexes) that were enriched at promoters of differentially expressed genes associated with synaptic function. To further interrogate the shared molecular changes in neuronal development associated with NDD and POGZ regulation, we compared our results to deletions of the transcription factor MEF2C and the sodium channel gene SCN2A that we generated in these same isogenic iN. These analyses revealed strong enrichment of extracellular matrix and intracellular signaling disruption associated with POGZ and MEF2C deletion, whereas POGZ and SCN2A haploinsufficiency exhibited shared transcriptional effects on gene modules enriched for NDD-associated genes with opposing regulatory effects. Notably, we also observed alterations to synaptic firing rate and neurite extension with biallelic deletions, but not heterozygous lines, suggesting subtle effects in neuronal development associated with haploinsufficiency. Overall, these shared molecular consequences suggest key points of convergence that connect epigenetic regulation to neuronal function in the etiology of neurodevelopmental pathologies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eed6138b4f1663b7339e409062cbd4f7a6fbbd84" target='_blank'>
              CRISPR-engineered deletion of POGZ alters transcription factor binding at promoters of genes involved in synaptic signaling
              </a>
            </td>
          <td>
            M. Moyses-Oliveira, Yating Liu, Serkan Erdin, Dadi Gao, Riya Bhavsar, Kiana Mohajeri, Kathryn O’Keefe, Philip M. Boone, Gabriela Xavier, Calwing Liao, Aiqun Li, Rachita Yadav, M. Salani, D. Lucente, Benjamin Currall, C. D. de Esch, D. J. Tai, D. Ruderfer, Kristen J. Brennand, James F. Gusella, M. Talkowski
          </td>
          <td>2025-10-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="Tumors are composed of cellular populations with distinct genotypes and phenotypes, which dynamically evolve over time and during treatment. This process is known as clonal evolution, and it is difficult to reveal fine-scale clonal structure with traditional bulk sequencing. Although single-cell genome sequencing could enable reconstruction of tumor clonal evolution, it remains technically challenging and the number of single cells profiled is generally insufficient due to high cost. To address this issue, we developed scClone, a computational toolkit that integrates variant detection and genotype inference for single-cell RNA-seq (scRNA-seq) and spatial transcriptomic data. It further provides interactive visualization of clonal structure and dynamic evolution. scClone addresses key limitations inherent to scRNA-seq, such as expression drop-out and allelic imbalance, and incorporates cell type or state annotation with mutational signature analysis to enable comprehensive profiling of tumor heterogeneity. scClone demonstrated robust performance across multiple datasets—generated from both full-length and fragmented RNA sequencing—by accurately reproducing mutation profiles and resolving clonal mixtures in myeloma, hepatocellular carcinoma and pancreatic cancer. Additionally, scClone has been applied to spatial transcriptomics, enabling the delineation of clonal structures within histological sections from ovarian cancer and cutaneous squamous cell carcinoma. In summary, our results demonstrate that scClone can extract genetic information from scRNA-seq datasets, thereby successfully establishing genotype–phenotype associations at the single-cell level and providing insights into the clonal evolution of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d470f131086238e06c10a786c9bfc7fb86a41116" target='_blank'>
              De Novo Detection of Clonal Structure and Evolution in Single-Cell and Spatial Transcriptomes
              </a>
            </td>
          <td>
            Shihao Bai, Xianbin Su, Ziyao Chen, Ze-Guang Han
          </td>
          <td>2025-11-26</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9934d0866a4f499711a0c6e0e0bd21bdab1c7f72" target='_blank'>
              Tracking ongoing chromosomal instability using single-cell whole-genome sequencing
              </a>
            </td>
          <td>
            Bárbara Hernando, Blas Chaves-Urbano, Maria Escobar-Rey, Marina Torres, Alice Cádiz, Ángel Fernández-Sanromán, Sara Barrambana, Carmen G. Lechuga, Carmen Guerra, Mariano Barbacid, David Gómez-Sánchez, F. Sánchez-Vega, P. Razavi, Maria Garcia-Perez, Patricia G. Santamaria, Geoffrey Macintyre
          </td>
          <td>2025-11-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Abstract Evidence suggests that alternative RNA splicing (AS) plays a critical role in tumor biology and may contribute to the generation of tumor antigens. Here, we develop a method to detect AS in short-read single-cell 5′-RNA-sequencing data, allowing us to uniquely characterize the heterogeneity and dynamic changes in AS in individual cell types within the tumor microenvironment. We identify numerous splicing events specific to either cancer cells or stromal cell types or for triple-negative versus estrogen receptor-positive breast cancers (BCs). By correlating these splice events with expression of splicing regulators in individual cells, we also identify their potential mediators. For instance, we identify and functionally validate the Epithelial Splicing Regulatory Protein-1 (ESRP1) to drive AS in BCs responding to immune checkpoint blockade (ICB). Prioritization of splicing events based on their likelihood to represent tumor antigens reveals that their aggregated load also correlates with high immune activity in multiple cancers, while also predicting expansion of T cells in BCs receiving ICB and prolonging long-term survival of cancer patients treated with ICB. Collectively, our method provides a framework for analyzing AS in single-cell data and defines a key role for AS in the response to ICB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b79be77027ec8fdf591f4f79253b80b26a945fe0" target='_blank'>
              Single-cell mapping of alternative splicing linked to checkpoint immunotherapy response
              </a>
            </td>
          <td>
            Jieyi Xiong, O. Bricard, I. Arijs, J. Demeulemeester, Chen Gu, Bernard Thienpont, Danie Daaboul, A. Bassez, Oliver Bechter, J. Pozniak, H. Vos, I. Nevelsteen, Sofie Torfs, Sara Aibar Santos, Qing Lan, Yong Hou, Lore Van Oudenhove, G. Boons, Junbin Qian, Stein Aerts, Ann Smeets, J. Marine, Diether Lambrechts
          </td>
          <td>2025-11-13</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Genome-wide association studies (GWAS) have identified nearly 100 loci associated with metabolic dysfunction-associated steatotic liver disease (MASLD), but the molecular functions of these variant alleles remain elusive, particularly when they occur in non-coding regions. Here we profiled the chromatin accessibility landscape of liver nuclei from MASLD individuals, and demonstrated these accessible genomic sites were bound by cell type-specific transcription factors (TFs) and enriched for MASLD risk variants, highlighting lineage- and disease state-specific regulation. Using a massively parallel reporter assay (MPRA), we identified hundreds of differential activity variants (DAVs) that operate in a cell type-specific manner or in a stimulus-dependent context by disrupting liver pathogenesis-associated transcriptional regulatory network. Integrative analyses combining liver eQTLs, chromatin looping, and single-cell CRISPRi screening linked these DAVs to functional target genes. Notably, we demonstrated that DAVs located near SLC22A3 and core regulators of triglyceride metabolism (APOA5, ANGPTL3, and LPL) loci modulate their gene expression and contribute to altered lipid metabolism and hepatic stellate cell activation. Furthermore, these DAVs exhibit predictive power in distinguishing MASLD disease risk. Together, these multimodal integration analyses provide insights into the regulatory mechanisms of MASLD progression driven by noncoding genetic risks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5934e58f3e02312c2db36030491cd91459f528f6" target='_blank'>
              Integrative functional genomics reveals transcriptional regulatory function of risk alleles for metabolic liver disease
              </a>
            </td>
          <td>
            Wenxiang Hu, Biying Zhu, Na He, Yang Xiao, Bin Chen, Chen Li, Ravi Mandla, Yifan Liu, Jiayu Zhang, Xiao Chang, Fulong Yu, M. Vujkovic, J. Lynch, Kyong-Mi Chang, V. M. V. Program, Bogdan Pasaniuc, Daniel J. Rader, Mitchell A. Lazar
          </td>
          <td>2025-10-23</td>
          <td>Research Square</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Whole-genome doubling (WGD) is a macro-evolutionary event that is both prevalent and prognostically significant in human cancers, particularly in high-grade serous ovarian carcinoma (HGSOC). Historically, WGD has been viewed as a consequence of widespread genomic instability, but recent advancements in single-cell sequencing (sc-seq) have reframed its role as a central, dynamic driver of tumor evolution. This review summarizes cutting-edge findings, demonstrating how WGD acts as a catalyst for a distinct evolutionary trajectory characterized by the rapid accumulation of chromosomal losses and the selection of highly adaptable clones. A key finding is the resolution of a biological paradox: WGD-driven chromosomal instability, which should provoke an immune response, is instead correlated with a profoundly immunosuppressive phenotype via the repression of key innate immune pathways. Finally, this review discusses the clinical implications of these discoveries, highlighting WGD’s potential as a predictive biomarker and a source of unique therapeutic vulnerabilities, paving the way for targeted strategies in advanced HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5292890396c5c0d49191edc0b87d5766e4f3e3ca" target='_blank'>
              Genome doubling as a dynamic driver of ovarian cancer evolution: insights from single-cell sequencing
              </a>
            </td>
          <td>
            Tianjiao Zhao, Tianshi Zhao, Dengyue Dong, Dengyue Dong
          </td>
          <td>2025-11-19</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Extensive studies have demonstrated the relationship between metabolic reprogramming and the tumor microenvironment. Here, we characterized the head and neck squamous cell carcinoma (HNSCC) evolutionary landscape using spatial metabolomics/transcriptomics, single-cell transcriptomics, and bulk multi-omics. Metabolic heterogeneity during HNSCC malignant transformation was identified, with significant enrichment in the purine metabolism. Integrating single-cell and bulk data, we developed a robust ligand-receptor-based signature (LRS) linked to NT5E, a key upstream regulator of purine metabolism, which served as an independent prognostic indicator. The low LRS subtype was associated with a high proportion of immune cell infiltration and improved response to immunotherapy. Notably, in vitro and in vivo experiments demonstrated that AMIGO2, a core molecule within the LRS, regulates tumor-associated purine metabolism, and that its downregulation suppresses tumor cell invasion and migration, inhibits myofibroblast differentiation, and promotes immune effector cell infiltration. Moreover, combining AMIGO2 targeting with anti-PD-1 therapy yielded superior efficacy. Consistent validation was also obtained in a clinical cohort of HNSCC and premalignancy patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2ca668a38f0d67d558807d21d034edd4de991a2" target='_blank'>
              Metabolomic and transcriptomic profiling of HNSCC identifies AMIGO2 as a therapeutic target modulating tumor microenvironment
              </a>
            </td>
          <td>
            Gan Liu, Xinfeng Yao, Yuchen Hou, Wentao Deng, Liyu Zhang, Shutong Li, Panpan Huang, Wenjiao Chang, Haiyan Huang, Lan Shi, Qianqian Zhou, Yanqing Wang, Jia Chen, Wenchao Gu, Xiaoling Ma, Yu Zhou, Honghong Liu, Shanshan Zhang, Zongcheng Yang
          </td>
          <td>2025-11-18</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/137a07d5d39a7f965a9829b9a10c7a07db82c57e" target='_blank'>
              Multi-omics-driven kinetic modeling reveals metabolic adaptations and vulnerabilities in BRCA1-deficient ovarian cancer
              </a>
            </td>
          <td>
            Ilias Toumpe, M. Masid, V. Hatzimanikatis, L. Miskovic
          </td>
          <td>2025-11-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc219a64ec3d466c88c4dfd11649123286e2032b" target='_blank'>
              Cross-Species Insights from ART-D to Uncover Evolutionarily Conserved Oncogenic Mechanisms
              </a>
            </td>
          <td>
            Yifan Guo, Jiadong Zheng, Yixin Wu, Kun Zhao, Dian Lv, Wenhan Liu, Mengling Wang, Jin-Yu Lu, Wenyan Xu, Xianping Wang, Xianjue Ma
          </td>
          <td>2025-10-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The androgen receptor (AR) is a critical driver of prostate cancer (PCa). Here, to study regulators of AR protein levels and oncogenic activity, we developed a live-cell quantitative endogenous AR fluorescent reporter. Leveraging this AR reporter, we performed genome-scale CRISPRi flow cytometry sorting screens to systematically identify genes that modulate AR protein levels. We identified and validated known AR protein regulators, including HOXB13 and GATA2, and also unexpected top hits including PTGES3—a poorly characterized gene in PCa. PTGES3 repression resulted in loss of AR protein, cell-cycle arrest and cell death in AR-driven PCa models. Clinically, analysis of PCa data demonstrates that PTGES3 expression is associated with AR-directed therapy resistance. Mechanistically, we show PTGES3 binds directly to AR, regulates AR protein stability and is necessary for AR function in the nucleus at AR target genes. PTGES3 represents a potential therapeutic target for overcoming known mechanisms of resistance to existing AR-directed therapies in PCa. Genome-wide CRISPRi screens for modulators of androgen receptor (AR) protein levels using live-cell quantitative endogenous AR fluorescence reporters identify PTGES3 as a new regulator of AR stability and function in prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55b624f946f0cc87b60d21d7777716b1e092cc49" target='_blank'>
              Genome-scale CRISPR screens identify PTGES3 as a direct modulator of androgen receptor function in advanced prostate cancer
              </a>
            </td>
          <td>
            Haolong Li, James E. Melnyk, Becky Xu Hua Fu, Raunak Shrestha, Meng Zhang, M. Sjöström, Siyu Feng, Jasmine A Anderson, Wanting Han, Lisa N Chesner, Hyun Jin Shin, Tatyanah Farsh, Humberto J. Suarez, Seema Nath, Jonathan Chou, Rajdeep Das, Emily A Egusa, Marsha Calvert, Audrey Kishishita, Abhilash Barpanda, Jun Zhu, Ashutosh Maheshwari, William S. Chen, M. Alshalalfa, Aidan Winters, Junjie T Hua, Tianyi Liu, E. Davicioni, Arun P. Wiita, B. Stohr, Javed Siddiqui, Bo Huang, Eric J. Small, K. Shokat, Peter S. Nelson, David A. Quigley, Elizabeth V. Wasmuth, Luke A. Gilbert, Felix Y. Feng
          </td>
          <td>2025-11-05</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Phenotypic plasticity plays a key role in cancer progression and metastasis, enabling cancer cells to adapt and evolve, but precisely how distinct axes governing phenotypic plasticity interact to shape tumour progression and patient outcomes remains unclear. We investigated five major interconnected axes of plasticity in ER-positive (ER+) breast cancer: Metabolic Reprogramming, Epithelial-to-Mesenchymal Plasticity (EMP), Luminal–Basal (Lineage) Switching, Stemness, and Drug-resistance using network dynamics simulations, integrative bulk and single-cell transcriptomic analyses and patient survival analyses. We show that these axes are not independent but drive one another, forming two mutually inhibiting ‘teams’ of nodes enabling specified cellular behaviour. One team (favouring high glycolysis, stem-like, basal-like, mesenchymal/hybrid and tamoxifen-resistant phenotype) was found to be associated with aggressive progression and worse survival. On the other hand, the opposing team (favouring high oxidative phosphorylation, non-stem-like, luminal-like, epithelial and tamoxifen-sensitive phenotype) correlated with better outcomes. Importantly, altering one axis of plasticity often drove coordinated responses along other axes and vice versa. Our findings establish phenotypic plasticity in cancer as a coordinated, multi-axis dynamical process, thus suggesting novel strategies to disrupt systems-level reprogramming enabling metastasis and therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6acaf4b9f2872701e5c44aad69b0ee83629b9228" target='_blank'>
              Interconnected axes of phenotypic plasticity drive coordinated cellular behaviour and worse clinical outcomes in breast cancer
              </a>
            </td>
          <td>
            Ritesh Kumar Meena, Yijia Fan, Soundharya Ramu, Anjaney J. Pandey, Abhinav Kannan, Yashvita Subramanian, Yukta Subramanian, Jason T. George, M. Jolly
          </td>
          <td>2025-11-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b82ec4f52026d1218d8a25c579cd530acc17905" target='_blank'>
              Widespread remodeling of the RNA editome underlies transcriptional and clinical heterogeneity in pediatric acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Sara Rosenthal, Thomas Whisenant, Roman Sasik, Maria Rivera, Geena Il Defonso, Charles-Alexandre Roy, Sabina Enlund, R. Shraim, Kathleen M Fisch, D. Teachey, Frida Holm, D. Kuo, Qingfei Jiang
          </td>
          <td>2025-12-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ecd9aa898a5f744ae181ea0c4ce6e138f5e70b4" target='_blank'>
              Comprehensive pan-cancer characterization of cancer-testis genes at single-cell resolution
              </a>
            </td>
          <td>
            Chunyang Fu, Shumin Yin, Linjin Li, Xueying Liu, Xiuyuan Jin, Ke Liu
          </td>
          <td>2025-11-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Therapeutic resistance remains a defining challenge in oncology, limiting the durability of current therapies and contributing to disease relapse and poor patient outcomes. This review systematically integrates recent progress in understanding the molecular, cellular, and ecological foundations of drug resistance across chemotherapy, targeted therapy, and immunotherapy. We delineate how genetic alterations, epigenetic reprogramming, post-translational modifications, and non-coding RNA networks cooperate with metabolic reprogramming and tumor microenvironment remodeling to sustain resistant phenotypes. The influence of the microbiome is highlighted as an emerging determinant of therapeutic response through immune modulation and metabolic cross-talk. By summarizing key regulatory circuits, We establishe a unified framework linking clonal evolution, metabolic adaptability, and tumor ecological dynamics. We further synthesizes novel therapeutic strategies that convert resistance mechanisms into therapeutic vulnerabilities, including synthetic lethality approaches, metabolic targeting, and disruption of stem cell and stromal niches. Advances in single-cell and spatial omics, liquid biopsy, and artificial intelligence are emphasized as transformative tools for early detection and real-time prediction of resistance evolution. This review also identifies major translational gaps in preclinical modeling and proposes precision oncology frameworks guided by evolutionary principles. By bridging mechanistic understanding with adaptive clinical design, this work provides an integrated roadmap for overcoming therapeutic resistance and achieving sustained, long-term cancer control.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5732b68dc3b5afcc6ec00f3b3bd38479653a277" target='_blank'>
              Drug resistance in cancer: molecular mechanisms and emerging treatment strategies
              </a>
            </td>
          <td>
            Jinxin Li, Jiatao Hu, Yiren Yang, Hanzhong Zhang, Ying Liu, Yu Fang, Le Qu, Anqi Lin, Peng Luo, Aimin Jiang, Linhui Wang
          </td>
          <td>2025-11-17</td>
          <td>Molecular Biomedicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09e91ffccc5754debd09b64e59ef74200817ed97" target='_blank'>
              Transcriptomic and epigenetic profiling of neuroblastoma states in response to RBM39 degrader.
              </a>
            </td>
          <td>
            Hongjian Jin, Jie Fang, Jason Myers, Shivendra Singh, Jun Yang
          </td>
          <td>2025-12-09</td>
          <td>Scientific data</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Background and Objective: Acute myeloid leukemia (AML) is a genetically heterogeneous disease, but even isogenic leukemic populations exhibit phenotypic variability. To determine how non-genetic, biological heterogeneity influences growth, therapy response, and adaptation to metabolic stress, we generated and deeply characterized isogenic human and mouse AML clones. By longitudinally tracking single-cell clones, we found that metabolic plasticity, not newly acquired mutations or changes in transcriptional programs, dictates cell growth and therapeutic response to venetoclax. These data highlight the central importance of biological heterogeneity in AML that is not dictated by genetic changes.
 Methods: We isolated 29 single-cell clones from the MLL-rearranged AML cell line MOLM13 and assessed longitudinal differences in baseline growth, adaptation to nutrient stress, and drug response. Clones were tested under glutamine or serum restriction and treated with cytarabine, daunorubicin, venetoclax, and azacitidine. Mitochondrial respiration was measured using Seahorse assays. To explore the genetic basis for phenotypic divergence, we performed RNA-seq and whole-exome sequencing on the most venetoclax-sensitive and -resistant clones. We validated findings in a primary mouse MLL-AF9 AML using both a barcoding approach to tracking clonal dynamics by NGS and with individually isolated and cultured clones.
 Results: Even without exogenous stress, baseline growth rates varied >3-fold among MOLM13 clones and remained stable over time. Nutrient stress further amplified heterogeneity: glutamine deprivation suppressed growth by 40–70% across clones, whereas 0.1% FBS had a more uniform suppressive effect. Glutamine restriction reduced basal oxygen consumption rates (OCR) in all clones but preserved maximal respiration after FCCP uncoupling, indicating metabolic adaptation via alternative substrates. Faster-growing clones displayed higher basal OCR and relative resistance to venetoclax, while slower-growing clones were more venetoclax-sensitive. Cytarabine responses showed an opposite trend, suggesting that drug sensitivity is growth rate-dependent. The IC50 of venetoclax varied by ~12-fold, compared to ~2-fold for daunorubicin, indicating a context-dependent influence of clonal heterogeneity on therapy response. Next, we compared the genetic and transcriptional profile of clones that displayed the most sensitivity and resistance to venetoclax. RNA-seq showed no differential expression of BCL2-family genes between sensitive and resistant clones, and whole-exome sequencing revealed hundreds of unique, non-overlapping non-synonymous variants per clone, but no recurrent driver mutations were linked to the phenotype. These results suggest non-genetic mechanisms underlie observed phenotypic diversity. Finally, we extended our findings using the primary mouse MLL-AF9 AML. Using a lentiviral barcoding approach, we again observed interclonal heterogeneity and changes in clonal representation under glutamine restriction and venetoclax exposure. We then isolated and grew individual MLL-AF9 clones, following a similar approach to our work with the MOLM13 cells, and observed interclonal variability in OCR and drug sensitivity, confirming our findings in an independent, primary AML.
 Conclusions: Our study reveals substantial biological heterogeneity among isogenic AML clones, with stable differences in growth, metabolic plasticity, and drug response. These phenotypes were not attributable to known driver mutations, which indicate that non-genetic factors may be involved in clonal adaptation. Clonal traits were context-dependent: venetoclax and glutamine restriction revealed the most heterogeneity, suggesting that selective pressures shape phenotypic diversity. These findings highlight the need to consider isogenic clonal variability when interpreting experimental results or designing therapies and highlight that functional heterogeneity may be an underappreciated driver of therapeutic response in AML.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffb7687231df66f3626536a54d60f11393391a61" target='_blank'>
              Clonal heterogeneity of acute myeloid leukemia under nutrient and chemotherapy stress
              </a>
            </td>
          <td>
            Christina Mayerhofer, Eve Crompton, Ernst Mayerhofer, Trine A. Kristiansen, Tsuyoshi Fukushima, Sam Keyes, A. Sharda, Dan Li, Carmen Da Silva, Deumaya Shrestha, Anna Kiem, T. Feuchtinger, David T. Scadden, P. Miller
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Motivation Understanding double-strand break (DSB) repair and its disruption is key to decipher genomic instability driving diseases such as cancer and reveal therapeutic avenues. Numerous genes have been linked to DSB repair for the first time in recent genome-wide perturbation studies assessing effects on mutational outcomes. However, the functional roles of most such genes remain poorly understood. Evidence from other studies shows that related genes similarly modulate the frequency of specific mutational outcomes following DSB repair, but analysis has largely ignored the multiplicity of gene functions and cross-talk between pathways. Here, we infer functional roles for candidate genes based on knockout mutational spectra by identifying mutational signatures shared with known genes and then mapping them to DSB repair functions. Signatures are identified using non-negative matrix factorization (NMF) to reflect functional multiplicity and cross-talk. Results We generated mutational spectra for mouse embryonic stem (mES) cells at three target sites across CRISPR knockouts of 307 known and 459 candidate DSB repair genes. Analysis using NMF revealed four mutational signatures associated with homology-directed repair (HDR), microhomology-mediated end-joining (MMEJ), and the initiation and ligation components of non-homologous end-joining (NHEJ). Signatures suggested that candidate genes Dbr1 and Hnrnpk could influence MMEJ and Fanconi anaemia (FA), and that loss of core NHEJ components (e.g. MRN complex or Ku proteins) could shift repair preference towards Ku-independent NHEJ. These findings demonstrate the utility of NMF for characterizing the contribution of genes to repair pathways and provide a foundation to discover new gene functions in DSB repair. Availability github.com/joanagoncalveslab/MuSpectraNMF.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20a5274e29358d261ef02cb6d42094a3acbdab40" target='_blank'>
              Signatures in CRISPR Mutational Spectra Reveal Role and Interplay of Genes in DNA Repair
              </a>
            </td>
          <td>
            Colm Seale, Marco Barazas, R. van Schendel, M. Tijsterman, Joana P. Gonçalves
          </td>
          <td>2025-11-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="


 Introduction: Extensive characterization of the genetic landscape in AML has revealed a set of driver mutations sufficient to initiate leukemic transformation. However, some cases do not contain any of these known drivers and thus pose a clinical challenge for prognostication and targeted therapy. To address this, we employed a novel, proximity ligation based, linked-read whole genome NGS assay to identify single nucleotide variants (SNVs), short Insertions/Deletions (InDels), copy number variation (CNV), structural variants (SVs) and 3D genome contact maps in AML. By employing this approach, we investigate whether comprehensive detection of genetic alterations in conjunction with insights into the epigenetic state of leukemic cells can reveal important biological information in cases lacking canonical driver mutations.
 Methods: We curated a cohort of patient samples where concomitant targeted sequencing by NGS and conventional karyotyping showed no detectable driver mutations. Further, we included control specimens with either known AML drivers as well as remission specimens after allogeneic transplantation. We utilized a multi-purpose NGS assay (Dovetail® LinkPrep™) to detect SNVs, InDels, CNVs, SVs and 3D genome contact maps. Peripheral blood or bone marrow cells were thawed and immediately fixed, subjected to crosslinking and in situ tagmentation, followed by library prep. Libraries were sequenced on Illumina sequencers with a target of 400M reads per library. Data analysis was performed using both published and custom tools to call SNVs/InDels (DeepVariant), Copy Number estimates (Purple), Structural Variants (HiC-Breakfinder, DTG-SELVA) and 3D contact maps (Juicebox).
 Results: We sequenced whole genome libraries to achieve an average coverage of 50.4x (range 41.1x - 58.1x) showing a long-range linkage percentage of 26.2% (range 17.9% - 31.8%). SNVs and short InDels, such as likely pre-leukemic mutations in DNMT3A, TET2, and mutations in GATA2 were detected reliably. Of note, the linked-read nature of this assay allowed us to perform phasing in a case with two TET2 mutations and confirm the compound heterozygous nature of these two TET2 mutations at a distance of 24kb. Further, known AML drivers in control specimens in genes such as NPM1, FLT3 and KRAS were reliably detected with variant allele frequencies as detected by targeted sequencing. Surprisingly, however, we found new structural variants in half of our cohort, including the diagnostic specimens with no detectable driver mutations, despite all specimens having no chromosomal abnormalities by conventional karyotyping. These new SVs included large intrachromosomal inversions and unbalanced translocations. We validated the presence of all SVs in sorted leukemic blasts and absence in sorted T cell fractions by PCR and Sanger sequencing. We subsequently examined the 3D genome contact maps and performed RNA-Sequencing to determine the epigenetic state and potential biological consequences across these novel breakpoints. We found several neoloops connecting enhancers with promoters across these breakpoints in 3D contact maps, and gene expression dysregulation proximal to the SVs by RNA-Seq. When we evaluated the expression of these putative driver genes across a cohort of healthy donors and AML patients, we found lack of expression in healthy cells but significantly increased expression in 9-10% of patients with AML (n=777 patients). Finally, we also identified an undetected structural variant in engrafted donor cells from a patient with myelodysplastic neoplasms (MDS) in remission after allogeneic transplantation, highlighting the potential to assay SVs in the context of non-leukemic hematopoiesis as well.
 Conclusion: In this exploratory cohort, we find several otherwise undetected structural variants in AML cases without known driver lesions using a proximity ligation based, linked-read whole genome NGS assay while maintaining our ability to detect recurrent, known driver lesions. This approach allowed us to interrogate the biological consequences of these genomic alterations by leveraging the ability to not only detect genomic variants but also analyze the epigenetic state through 3D genome contact maps. Our results imply that our current understanding of the genomic landscape in AML might be underestimating a sizable number of structural variants that are difficult to detect using standard assays, implying putative novel driver genes in AML pathogenesis.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f033c80701d9a0710865b0dce2b79cfc9a9b774" target='_blank'>
              Sensitive detection of novel structural variants and 3D chromosome conformation reveals likely novel drivers including enhancer hijacking in AML
              </a>
            </td>
          <td>
            Thomas Koehnke, A. Ediriwickrema, Mital S. Bhakta, Alexander Fortuna, Charu Tiwari, Zackary Sanborn, Tian Zhang, Lisa Munding, Ravi Majeti
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b81b381c14e5b1230a77d230dfeaa5698f06f2cc" target='_blank'>
              Characterizing resistance in prostate cancer at a single cell level with hormonal treatment and epigenetic inhibitors.
              </a>
            </td>
          <td>
            Morgan Howells, Priyadarsini Gangadharannambiar, Calum McMullen, Francesco Crea, W. Bacon, Colette Christiansen
          </td>
          <td>2025-11-03</td>
          <td>The Journal of pharmacy and pharmacology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a highly aggressive brain tumor with limited treatment options and poor survival. Mitochondrial dysfunction and metabolic reprogramming, particularly the Warburg effect, are increasingly recognized as critical drivers of GBM progression. Here, we integrated single-cell (scRNA-seq) and bulk RNA-sequencing (bulk RNA-seq) data to comprehensively examine mitochondria-associated genes in GBM. We identified differentially expressed mitochondrial genes with prognostic significance and constructed a 9-gene risk signature—ACOT7, THEM5, MTHFD2, ABCB7, PICK1, PDK3, ARMCX6, GSTK1, and SSBP1—using LASSO and Cox regression. This signature robustly stratified patients into high- and low-risk groups in both The Cancer Genome Atlas (TCGA) and Chinese Glioma Genome Atlas (CGGA) cohorts, remaining an independent prognostic factor in multivariate analyses. Time-dependent ROC AUCs were 0.729, 0.813, and 0.828 at 1, 2, and 3 years in TCGA, and 0.597, 0.650, and 0.546 in CGGA. A nomogram integrating the signature with clinical variables achieved AUCs of 0.649, 0.820, and 0.854 at 1, 3, and 5 years, with good 1/2/3-year calibration. Functional enrichment and clustering analyses revealed distinct metabolic phenotypes and survival differences between subtypes. Single-cell pseudotime analysis showed a transition from oxidative phosphorylation to glycolysis in malignant cells, aligning with the Warburg effect and implicating metabolic reprogramming in immune modulation. Our findings underscore the prognostic value of mitochondria-associated genes and suggest potential therapeutic targets for disrupting GBM metabolism. Overall, these results establish a mitochondria-centric prognostic model supported by single-cell context; findings are hypothesis-generating and warrant prospective and experimental validation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1def4820c25938da49857764d61ab2d3606bd5c5" target='_blank'>
              Comprehensive analysis of mitochondria-associated genes in glioblastoma via single-cell and bulk RNA sequencing
              </a>
            </td>
          <td>
            Fangyuan Wang, Ao Huo, Minghe Li, Chenglin Zhao, Yupei Guo, Zhou Jing, Qingyi Wang, Bingqi Lu, Hao Li, Wulong Liang, Weihua Hu, Xudong Fu
          </td>
          <td>2025-10-27</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 Background Acute myeloid leukemia (AML) is the most common adult acute leukemia. The traditional “3+7” induction regimen with anthracycline and cytarabine remains a standard therapy, but 30%-40% of patients develop drug resistance to anthracyclines, leading to relapse. Refractory/relapsed AML has dismal survival rates, posing a major challenge. Elucidating mechanisms of therapy resistance and relapse is crucial for improving outcomes in high-risk AML.
 Methods ATAC-seq analyzed chromatin accessibility of anthracycline-sensitive and -resistant AML cells, focusing on METTL3 locus. RNA-seq, qRT-PCR and Western blot measured METTL3 expression in cell lines and primary cells. METTL3 was regulated by small-molecule inhibition, shRNA knockdown and overexpression to study its effect on m6A modification and phenotypes in resistant cells. MeRIP-seq and RNA-seq identified METTL3 target genes in shMETTL3-transfected resistant cells. MeRIP-qPCR validated m6A-modified mRNAs and actinomycin D assays tested RNA stability. BMI1 and METTL3 inhibitors were used to explore their functional relationship in drug resistance.
 Results
 In anthracycline-sensitive AML patients, over 50% of the global chromatin accessible regions are enriched in promoter regions. Conversely, in anthracycline-resistant AML patients, promoter-accessible regions account for less than 25% of the total. Treatment with anthracyclines induces changes in promoter chromatin accessibility in only approximately 5% of the genome in sensitive patients, whereas this proportion reaches approximately 25% in resistant patients. Notably, METTL3 exhibits significantly elevated chromatin accessibility in anthracycline-resistant AML cells. Following anthracycline exposure, METTL3 accessibility is further enhanced in resistant cells but decreased in sensitive cells. m6A modification was significantly upregulated in AML drug-resistant cells, accompanied by consistently higher METTL3 expression. Integrative multi-omics analyses identified HOXA9 as a key transcriptional regulator of METTL3, with HOXA9 positively modulating METTL3 expression. Clinical datasets (TCGA and GSE6891) revealed that elevated HOXA9 expression was significantly associated with poor prognosis in AML patients. HOXA9 levels were markedly higher in drug-resistant and relapsed/refractory (R/R) AML cells. HOXA9 knockdown in resistant cells suppressed proliferation and induced apoptosis. Dual-luciferase reporter assays confirmed that HOXA9 specifically bound to the “TAACGTG” motif within CpG island 3 of the METTL3 promoter.
 Mechanistically, reducing METTL3 expression in drug-resistant cells decreased m6A modification, inhibited cell proliferation, and triggered apoptosis. Conversely, METTL3 overexpression in sensitive cells enhanced m6A modification, promoted proliferation, suppressed apoptosis, and augmented doxorubicin resistance. METTL3 promoted m6A modification of BMI1 mRNA, stabilizing BMI1 transcripts and elevating BMI1 expression in anthracycline-resistant cells.
 BMI1 was overexpressed in anthracycline-resistant cells and correlated with poor prognosis in AML patients. Inhibiting BMI1 in resistant cells induced dose-dependent proliferation arrest and apoptosis, while restoring sensitivity to anthracyclines.
 Conclusion Our study demonstrates that chromatin accessibility in the promoter regions of anthracycline-resistant AML cells is significantly decreased compared to anthracycline-sensitive counterparts. Anthracycline elicits more pronounced changes in promoter chromatin accessibility in resistant cells than in sensitive cells, underscoring a heightened involvement of promoter-level epigenetic regulation in anthracycline-resistant AML. METTL3 exhibits both upregulated expression and enhanced chromatin accessibility in anthracycline-resistant AML cells relative to sensitive cells. Furthermore, anthracycline exposure leads to a further increase in chromatin accessibility at the METTL3 promoter in resistant cells, suggesting dynamic epigenetic modulation of this key m6A regulator during drug resistance. Our study demonstrated that The HOXA9/METTL3/BMI1 axis promotes anthracycline resistance in acute myeloid leukemia through mRNA m6A modification and suggested the therapeutic potential of targeting the METTL3/BMI1 axis in the treatment of anthracycline-resistant AML.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/266257a13a97162936fed07815814a1d0db44fc4" target='_blank'>
              HOXA9/METTL3/BMI1 axis is required for promoting anthracycline resistance in acute myeloid leukemia cells by modulating BMI1 expression through  mRNA m6A modification
              </a>
            </td>
          <td>
            Hao Wang, Daihong Liu, Liping Dou
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Therapeutic resistance remains a critical barrier in oncology, frequently leading to cancer relapse after initial treatment response. Growing evidence suggests the presence of drug-tolerant persisters (DTPs), a rare subpopulation of cancer cells that survives chemotherapy by entering a reversible specific adaptation. Unlike classical cell resistance, the DTP phenotype is independent of genetic changes and maintained through dynamic regulatory mechanisms. DTPs are phenotypically heterogeneous and can exhibit stem-like and quiescent cell phenotypes, non- or slow proliferation, and remarkable plasticity due to a di-pause-like state and executing epithelial–mesenchymal transition (EMT) or transdifferentiation processes. Despite advances in research, the molecular mechanisms underlying DTPs’ biology and their role in cancer relapse remain only partially understood. The review summarizes the current progress in processes that lead to the acquisition of cellular persistence status, which, in turn, constitute areas of vulnerability that can be exploited in cancer therapy. We highlight anti-DTP therapeutic strategies, including epigenetic modification, cell signaling and transcriptional regulation, metabolic reprogramming, and modification of cell interactions within the tumor microenvironment. Furthermore, we focus on the potential role of nanomaterials in the combat against DTPs. Nanoparticles not only act as part of the drug delivery process, enabling precise DTP targeting and enhancing intracellular drug accumulation, but their intrinsic properties can also be used to eradicate DTPs directly or by enhancing the effectiveness of other therapeutic strategies. The integrated approach offers strong potential to eliminate tumor persistence, prevent recurrence, and improve long-term patient outcomes beyond conventional therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/495b98c4c7bc3661dae9aa58d4006e54132b7ae3" target='_blank'>
              Targeting Drug-Tolerant Persister Cancer Cells: Can Nanomaterial-Based Strategies Be Helpful for Anti-DTP Therapies?
              </a>
            </td>
          <td>
            Prachi Ghoderao, Eliza Kwiatkowska-Borowczyk, Sanjay Sahare, H. Dams-Kozlowska
          </td>
          <td>2025-11-01</td>
          <td>Pharmaceutics</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45dd551579bb493b99c7ec5a7c44f7e95f00ebc0" target='_blank'>
              Single cell whole genome and transcriptome sequencing links somatic mutations to cell identity and ancestry
              </a>
            </td>
          <td>
            Abhiram Natu, Mrunal K. Dehankar, Reenal Pattni, M. Šuvakov, Dmitrii Olisov, Livia Tomasini, Yeongjun Jang, Yiling Huang, Eva Benito-Garragori, Patrick Hasenfeld, Jan O. Korbel, Alexander E. Urban, A. Abyzov, F. Vaccarino
          </td>
          <td>2025-10-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="


 INTRODUCTION Multiple Myeloma (MM) is a genetically complex blood cancer marked by diverse mutations, structural rearrangements, and chromosomal abnormalities that drive disease progression and treatment resistance. Traditional genomic approaches often miss critical alterations due to technical limitations in resolving complex regions. Oxford Nanopore long-read sequencing offers unprecedented insight by generating ultra-long reads capable of spanning difficult genomic landscapes. ONT's innovative Adaptive Sampling technology enables high-resolution detection of clinically relevant mutations and structural variants without the need for laborious capture or amplification steps, opening new avenues for comprehensive and rapid MM genomic profiling.
 METHODS We designed a custom adaptive sampling panel covering ∼6% of the genome, targeting key genes, structural hotspots, and translocation-prone regions relevant to cancer. Genomic DNA from CD138+ cells of 17 MM patients was processed using ONT protocols and sequenced on the PromethION (P2i) platform with Adaptive Sampling enabled. We assessed enrichment efficiency, coverage, and variant detection. All patients had been previously characterized using gold standard methods, including FISH for del(17p), del(1p), amp(1q), t(4;14), t(14;16), and a targeted 38-gene NGS panel for multiple myeloma.
 RESULTS This technology enabled molecular karyotyping of all 17 patients through whole-genome characterization. Using adaptive sampling, we performed low-coverage whole-genome sequencing, which facilitated the detection of large-scale structural variants and copy number alterations across the entire genome. Among the 17 patients, 2 showed no major chromosomal alterations; 4 were classified as hyperdiploid, each presenting trisomy's in various odd-numbered chromosomes. One patient exhibited an isolated del(1p), while another showed an isolated amp(1q). The remaining 7 patients displayed diverse combinations of genomic abnormalities, including del(1p), amp(1q), monosomy 8, monosomy 13, del(17p), and hyperdiploidy. Across 51 comparable probes (3 CNV targets per patient), 94.1% (48/51) showed concordant results, while the 3 discordant cases corresponded to small deletions in the short arm of chromosome 1 (1p), likely undetected by conventional FISH probes.
 We initially focused on rearrangements involving the IGH locus on chromosome 14, given their relevance in multiple myeloma. All patients had previously been evaluated by FISH for t(4;14) and t(14;16) as stablished prognostic markers and were negative for both. However, long-read sequencing identified IGH rearrangements in two distinct patients: one involving t(14;20) and the other t(11;14), both of which have been previously reported in MM. In addition, 88% [15/17] of the patients exhibited other chromosomal translocations that warrant further characterization to assess their potential biological and clinical significance in the disease.
 Regarding SNVs, the short-read targeted NGS panel did not detect variants in 5 out of 17 patients. In contrast, long-read sequencing identified pathogenic variants with variant allele frequencies (VAFs) above 5% in all 17 patients, suggesting that exome sequencing with long-read technology enhances the detection of clinically relevant alterations. Furthermore, the use of a clinical exome enables the identification of variants in genes associated with other diseases or with potential pharmacogenomic implications, as it is not limited to a predefined set of targets.
 CONCLUSIONS Preliminary results indicate that the proposed technology enables the simultaneous detection of the main genomic alterations required for a comprehensive diagnostic workup in patients with multiple myeloma (MM), using a single, unified assay. Its agnostic design, based on broad genomic coverage without pre-selection of specific regions, allows for an unbiased analysis of the tumor genome. This approach not only facilitates the integration of multiple diagnostic markers into a single workflow but also supports potential scalability to other hematologic malignancies or solid tumors, as it is not restricted to a predefined set of genomic loci. The flexibility, robustness of the method and the short turnaround time for comprehensive characterization of MM patients position it as a promising tool for precision oncology.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87400f9201c3814073c1f811caea785b333c8c8a" target='_blank'>
              Targeted long-read sequencing with adaptive sampling in multiple myeloma: A pilot study toward a unified diagnostic platform
              </a>
            </td>
          <td>
            B. Ariceta, P. Aguirre-Ruiz, Andrea Castro, Patxi San Martín-Úriz, Saioa Vicente, M. Larráyoz, L. Tamariz-Amador, A. Alfonso-Piérola, Jesús F San-Miguel, M. Calasanz, P. Rodríguez-Otero, Felipe Prósper
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="CRISPR-based gene activation (CRISPRa) has emerged as a promising therapeutic approach for neurodevelopmental disorders (NDD) caused by haploinsufficiency. However, scaling this cis-regulatory therapy (CRT) paradigm requires pinpointing which candidate cis-regulatory elements (cCREs) are active in human neurons, and which can be targeted with CRISPRa to yield specific and therapeutic levels of target gene upregulation. Here, we combine Massively Parallel Reporter Assays (MPRAs) and a multiplex single cell CRISPRa screen to discover functional human neural enhancers whose CRISPRa targeting yields specific upregulation of NDD risk genes. First, we tested 5,425 candidate neuronal enhancers with MPRA, identifying 2,422 that are active in human neurons. Selected cCREs also displayed specific, autonomous in vivo activity in the developing mouse central nervous system. Next, we applied multiplex single-cell CRISPRa screening with 15,643 gRNAs to test all MPRA-prioritized cCREs and 761 promoters of NDD genes in their endogenous genomic contexts. We identified hundreds of promoter- and enhancer-targeting CRISPRa gRNAs that upregulated 200 of the 337 NDD genes in human neurons, including 91 novel enhancer-gene pairs. Finally, we confirmed that several of the CRISPRa gRNAs identified here demonstrated selective and therapeutically relevant upregulation of SCN2A, CHD8, CTNND2 and TCF4 when delivered virally to patient cell lines, human cerebral organoids, and a humanized mouse model of hTcf4. Our results provide a comprehensive resource of active, target-linked human neural enhancers for NDD genes and corresponding gRNA reagents for CRT development. More broadly, this work advances understanding of neural gene regulation and establishes a generalizable strategy for discovering CRT gRNA candidates across cell types and haploinsufficient disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c894b7f9e1d5db5f69bf147d03e982a27eebec4" target='_blank'>
              Large-scale discovery of neural enhancers for cis-regulation therapies
              </a>
            </td>
          <td>
            Troy A. McDiarmid, Nicholas F. Page, Florence M. Chardon, R. Daza, George T. Chen, Michael Kosicki, Lucas M. James, Hannah C. Nourie, Dianne Laboy-Cintrón, Arthur S. Lee, , Diego Calderon, Jean‐Benoît Lalanne, Beth K. Martin, Kyle D. Fink, Michael Talkowski, A. Muotri, Benjamin D. Philpot, Len A. Pennacchio, D. Geschwind, Stephan J. Sanders, N. Ahituv, Jay Shendure
          </td>
          <td>2025-11-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is characterized by complex genomic background and treatment resistance. We first defined “Titin (TTN) inactivation”, a state of TTN expression deficiency or mutation, affecting tumor progression. However, how TTN inactivation regulates immune escape and affects therapeutic resistance remains unclear. Using whole-exome sequencing, single-cell transcriptome sequencing, and spatial transcriptome sequencing, we screened the clinical features of TNBC patients with TTN inactivation who accepted neoadjuvant therapy. Meanwhile, we used CRISPR-Cas9 technology to construct various mutant TNBC cell lines. Lentiviral vector carrying TTN and delta-like ligand 4 (DLL4) was validated in vivo to verify potential mechanism. Myeloid-derived suppressor cells (MDSCs) metabolic function was measured using glycolysis-related molecular experiments. Immunotherapeutic agents against TNBC with TTN inactivation were explored in the orthotopic MCT4fl/fl genetically modified mouse model. DLL4-regulated ecological niche was established in TNBC with TTN inactivation. Mechanistically, TTN deficiency and mutation led to DLL4 secretion in TNBC. DLL4 enhanced MCT4-mediated glycolysis via MDSCs-derived NOTCH2 signaling pathway, driving the malignant function and lactate acid excretion of MDSCs. DLL4-derived MDSCs promoted stemness-mediated drug resistance by inducing histone lactate modification in TNBC, suppressing the anti-tumor activities of CD8+T cells. Blocking the DLL4-MCT4 axis stimulated anti-tumor immunity and synergized with anti-PD-1, improving response rates for first-line neoadjuvant therapy in TNBC. Our study revealed intrinsic mechanism by which TTN regulates the tumor immune microenvironment and provided a potential target for immunotherapy in TNBC with TTN inactivation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf824ac3335bab02348fceec44a41f866d2add7b" target='_blank'>
              Delta-like ligand 4 mediated myeloid-derived suppressor cell metabolic reprogramming promotes neoadjuvant therapy resistance in titin-inactivated triple-negative breast cancer
              </a>
            </td>
          <td>
            Yanfang Yang, Ziyun Liu, Parhat Kaysar, Yuxi Han, Bo Ni, Linwei Li, Lina Zhang, Xiaobin Shang, Yaoyao Zhou, Yongjie Xie, Zhansheng Jiang
          </td>
          <td>2025-12-01</td>
          <td>Molecular Biomedicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer cell plasticity, the dynamic ability of tumor cells to switch phenotypic states, drives tumor progression, metastasis, and therapy resistance, posing a formidable challenge to effective cancer management. This review comprehensively explores the genetic and epigenetic mechanisms underlying plasticity, highlighting how oncogenic mutations, chromosomal instability, and pathways like PI3K/AKT and RAS/MAPK enable adaptive behaviors such as epithelial-mesenchymal transition and stemness. Epigenetic regulators, including DNA methylation, histone modifications, and non-coding RNAs, amplify this adaptability, allowing rapid responses to therapeutic pressures. We examine the profound clinical implications, with plasticity fueling relapse in breast, lung, and prostate cancers, exacerbated by tumor microenvironment interactions. Challenges in detecting plastic states due to intratumor heterogeneity underscore the need for advanced diagnostics like single-cell genomics. Emerging interventions, from CRISPR-based gene editing and epigenetic therapies to patient-derived organoids and AI-driven combination strategies, offer transformative potential to disrupt plasticity and overcome resistance. This review illuminates the intricate genetic landscape of cancer cell plasticity, advocating for integrated approaches to revolutionize personalized oncology and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a56a74b7b33fc882b2d862736314940182017fc5" target='_blank'>
              The Genetic Landscape of Cancer Cell Plasticity: Mechanisms, Implications for Therapy Resistance and Emerging Interventions
              </a>
            </td>
          <td>
            I. N. Monago, Nyerovwo Charity Okei, Chibuike Stephen Nzereogu, Taiwo Emmanuel Olowodasa, Tamasanni Omeiza Salawu, Ahmad Muhammad Garba, Doris Ada Ojum
          </td>
          <td>2025-10-31</td>
          <td>International Journal of Biological and Pharmaceutical Sciences Archive</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Although non-muscle-invasive bladder cancer (NMIBC) frequently recurs and progresses despite BCG therapy, cellular mechanisms behind this remain unclear. Understanding the dynamics within the tumor microenvironment (TME) and identifying pathways associated with BCG resistance are crucial for improving NMIBC treatment. To elucidate transcriptional dynamics and key cellular interactions in the TME, we performed longitudinal single-nucleus RNA sequencing of 9 NMIBC cases with three pairs of treatment-naïve and disease progression samples. The resulting 58,037 nuclei were subject to cell type and subtyping annotations along with cell–cell interaction analyses. Our analysis revealed marked interpatient heterogeneity in malignant cells, accompanied by copy number alterations associated with disease progression. TGF-β signaling emerged as a key pathway, showing gradual enrichment from pre-treatment to post-progression samples. Within the tumor microenvironment, we identified distinct subtypes including LAMP3 + dendritic cells and iCAF fibroblasts that followed unique pseudotime trajectories linked to progression. Cell–cell interaction analysis further highlighted critical ligand–receptor pairs such as DSC2–DSG2 and ENG–BMPR2, which were associated with poor clinical outcomes. These interactions may represent hypothesis-generating candidates that warrant further validation. This study highlights key cellular and molecular mechanisms underlying bladder cancer progression Post-BCG treatment. Through single-nucleus RNA sequencing, we identified critical TME subtypes, TGF-β signaling involvement, and crucial ligand-receptor interactions such as DSC2-DSG2 and ENG-BMPR2, which may serve as preliminary biomarkers requiring confirmation in larger independent cohorts. These findings offer valuable insights into enhancement of treatment strategies and patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1b994449406e351724039c2b4ed65a466872b5c" target='_blank'>
              TGF-β signaling and tumor microenvironment dynamics in bladder cancer progression post-BCG therapy: a longitudinal single-nucleus RNA-seq study
              </a>
            </td>
          <td>
            Seo-Young Lee, Yun-Hee Lee, Tae-Min Kim, U-Syn Ha
          </td>
          <td>2025-11-10</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72837cc909b2a1ccbe8a1449f9aafc89b0a2cfa3" target='_blank'>
              Epigenetic reprogramming as the nexus of cancer stemness and therapy resistance: implications for biomarker discovery.
              </a>
            </td>
          <td>
            Chu Xin Ng, Shin Yuh Lee, Xin Yi Yap, Yong Hui Wong, Jian Sheng Loh, Kuan Ping Ang, Wen Hwei Lee, Wei Xu, Phelim Voon Chen Yong, Sau Har Lee
          </td>
          <td>2025-11-16</td>
          <td>Discover oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Cis-regulatory elements (CREs) control oncogene expression and malignant phenotypes. The high clinicopathological heterogeneity of cancer cannot be explained by gene expression alone, being attributed to CRE heterogeneity. However, characterizing cancer-associated CREs is challenging. To address this issue, we performed a single-cell epigenomic analysis of clinical specimens. To map the multicellular ecosystem of breast cancer (BC) and identify candidate CREs (cCREs), we performed a single-cell assay for transposase-accessible chromatin with sequencing (scATAC-seq) of 38 prospectively collected BC samples with various clinicopathological characteristics. First, we performed single-cell chromatin accessibility profiling of a high-quality set of 22,775 cells from BC samples. Cells were annotated using marker gene accessibility and integration with existing single-cell RNA sequencing data. The chromatin accessibility patterns exhibited by cancer cells were consistent with the clinicopathological features of each tumor. We identified 224,585 cCREs across the BC ecosystem. By identifying cluster-specific differentially accessible cCREs (DA-cCREs) and constructing a putative enhancer–promoter network, we mapped the cis-regulatory landscape for cancer cells and the tumor microenvironment. The accessibility of putative enhancers targeting the same gene differed within or between tumors, highlighting intra- and inter-cis-regulatory tumor heterogeneity. This study provides a valuable resource for future epigenetic research on BC and highlights the diverse regulatory landscape within or among tumor(s), suggesting that cCREs regulate intra- and intertumor heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9a987c29ce941aae8e905c09b2ed8fd2ff4315f" target='_blank'>
              Characterizing heterogeneous cis-regulatory elements in gene regulatory programs associated with breast cancer
              </a>
            </td>
          <td>
            Chisa Hori, Kohei Kumegawa, S. Saeki, Yoko Takahashi, Liying Yang, T. Nakadai, K. Otsuji, Chikako Takahata, Y. Ozaki, N. Uehiro, Yurie Haruyama, Tomo Osako, T. Takano, Seiichi Mori, Tetsuo Noda, Satoshi Fujii, Shinji Ohno, T. Ueno, R. Maruyama
          </td>
          <td>2025-11-27</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34b22389536004a2e5e3234be8036e5614f2a3ae" target='_blank'>
              A blueprint for mutation-defined hallmark vulnerabilities across human cancers
              </a>
            </td>
          <td>
            Ran Xu, Rebecca San Gil, Xin Tracy Liu, Yanfei Qi, Derek Tran, Chun Xu, Justin J-L Wong, Lenka Munoz, Graham J. Mann, Yuchen Feng
          </td>
          <td>2025-10-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Metastatic breast cancer (MBC) remains the primary cause of mortality in breast cancer patients, driven by tumor heterogeneity, cellular evolution, and therapy-resistant clones. Traditional bulk transcriptomics, although informative, fail to capture rare subpopulations and context-specific gene expression, which are crucial for understanding disease progression. Single-cell transcriptomics (SCT) has emerged as a transformative approach, enabling high-resolution analysis of individual cells to reveal tumor composition, lineage dynamics, and transcriptional plasticity. This review highlights how SCT reshapes our understanding of MBC by mapping tumor evolution, identifying cancer stem-like cells, and characterizing states of epithelial-mesenchymal transition. We explore how SCT reveals clonal and spatial heterogeneity, and how tumor microenvironment components, including immune, stromal, and endothelial cells, interact with cancer cells to support immune evasion and the formation of a metastatic niche. SCT also uncovers mechanisms of therapeutic resistance, including transcriptional reprogramming and the survival of drug-tolerant subpopulations. Integrating SCT with spatial transcriptomics and multi-omics platforms offers a comprehensive view of the MBC ecosystem and may uncover novel therapeutic targets. We further discuss the translational potential of SCT for biomarker discovery, liquid biopsy development, and precision oncology. We address current technical challenges and future directions for clinical application. SCT is poised to transform MBC research and guide next-generation therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf97dd9ea39bc843d16cb57d14bc4af2cfce4592" target='_blank'>
              Single-cell transcriptomics in metastatic breast cancer: mapping tumor evolution and therapeutic resistance
              </a>
            </td>
          <td>
            Xu Han, Xin Li, Ling Bai, Gangling Zhang
          </td>
          <td>2025-10-23</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cf3f4a8479510c49ff95777ff501eb3f2838138" target='_blank'>
              Epigenetic priming promotes tyrosine kinase inhibitor resistance and oncogene amplification.
              </a>
            </td>
          <td>
            Rebecca M. Starble, Eric G Sun, Rana Gbyli, Jonathan S. D. Radda, Jiuwei Lu, Tyler B Jensen, Ning Sun, Nelli Khudaverdyan, Ting Zhao, Bomiao Hu, M. Melnick, Shuai Zhao, Nitin Roper, G. Wang, A. Tackett, Yinsheng Wang, Jikui Song, Katerina Politi, Siyuan Wang, Andrew Z Xiao
          </td>
          <td>2025-10-23</td>
          <td>Nature structural & molecular biology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f697ecd66287673460b53084d2f61f279143459" target='_blank'>
              FOXA1 mutations co-opt nascent transcription factor networks in partnership with androgen receptor to enhance prostate tumorigenicity
              </a>
            </td>
          <td>
            Erik Ladewig, Abbas Nazir, Tyler Park, Vinson B. Fan, Zhendong Cao, Jacob Hawk, Lauren Kelly, Robert Tjian, Christina S. Leslie, Charles L. Sawyers
          </td>
          <td>2025-11-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df1c06700a8d3d29d2328428496f19af36bbcc45" target='_blank'>
              A comprehensive view of somatic mosaicism by single-cell DNA analysis
              </a>
            </td>
          <td>
            Lovelace J. Luquette, Tim H. H. Coorens, Abhiram Natu, M. Šuvakov, Ann Caplin, Mee Sook Jun, Alisa Mo, Joe Pelt, Lisa Anderson, Michele Berselli, Sravya V Bhamidipati, Thomas G Blanchard, Joseph Brew, Hye-Jung E. Chun, Hyunbin Chun, Mrunal K. Dehankar, William C. Feng, Rob Furatero, Christopher M. Grochowski, Eric Ho, Yeongjun Jang, Kavya Kottapalli, M. K. Leonard, Nam Seop Lim, Tina Lindsay, Sarah Nicholson, Ivan Raimondi, Alexi Runnels, Constantijn Scharlee, JaeEun Shin, Alexander D. Veit, M. VonDran, Yifan Wang, Dennis J. Yuan, Yifan Zhao, SMaHT Single Cell, Focus Group, Thomas J. Bell, K. Ardlie, Harsha Doddapaneni, Robert S. Fulton, S. Germer, Dan Landau, Jihyeon Oh, Peter J. Park, F. Vaccarino, Christopher A. Walsh, A. Abyzov
          </td>
          <td>2025-11-03</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>60</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f8838d113f21c285c0455db7c93dfda5adc298f" target='_blank'>
              A switch from histone methyltransferase-EZH2 to demethylase KDM6A activity marks reinitiation of proliferation of drug treated colorectal cancer cells
              </a>
            </td>
          <td>
            Subhashree Chatterjee, Ritika Jaiswal, Aniruddha Roy, S. Chowdhury, Sudeshna Mukherjee, Rajdeep Chowdhury
          </td>
          <td>2025-11-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10582cdaf7507df9370133688fbcd15ae5208a8b" target='_blank'>
              Multiomic characterization, early detection, and therapeutic targeting of myeloid sarcoma
              </a>
            </td>
          <td>
            B. Nadorp, Audrey Lasry, S. Loghavi, Ravi Patel, Hager Mansour, Benjamin J. Kelly, Christopher J. Walker, Jill Buss, Isaiah Boateng, Wafa Al-Santli, Zoe Ciantra, Rebecca Austin, Adrienne Heyrosa, Helee Desai, Andrea Laganson, Hasan Abaza, Linda Procell, Tejas Patel, Benjamin Kaffenberger, Saranga Wijeratne, Maria Guillamot, Maria Velegraki, Luis Chiriboga, Zihai Li, L. Abruzzo, Daniel A. Pollyea, Christine M McMahon, A. Shana'ah, John C. Byrd, Alan H Shih, R. L. Levine, E. Papapetrou, Aris Tsirigos, Elaine R. Mardis, Alice S. Mims, I. Aifantis, Ann-Kathrin Eisfeld
          </td>
          <td>2025-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a78192449266cbe137f9e2b3c247baae65185e3f" target='_blank'>
              CRISPR screens reveal YAP/TEAD axis as a mediator of drug-tolerant cells in EGFR-mutant NSCLC.
              </a>
            </td>
          <td>
            Muhammed Yusuf
          </td>
          <td>2025-11-12</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Elucidating mechanisms of drug resistance is key for overcoming resistance, guiding drug design, and enabling accurate resistance prediction. Recently, disease metabolic reprogramming has emerged as a novel mechanism of resistance, which enables disease cells to adapt to therapeutic resistance by altering energy production pathways, cellular signaling, and biosynthesis processes. Moreover, protein structure alterations also play a pivotal role in resistance study, facilitating mechanistic understanding, and structure-based target discovery. In other words, integrating these recently accumulated critical data is essential for enriching the landscape of drug resistance data. Therefore, in this study, DRESIS was a significant update by providing (i) 236 molecules that drive metabolic reprogramming and confer resistance to 168 drugs, together with a detailed mechanism, (ii) 2228 protein structural variants implicated in resistance to 671 drugs across 238 diseases, and (iii) greatly expanded landscapes of drug resistance information, now featuring 398 newly added key drug-resistant molecules, 356 drugs with the latest published resistance mechanisms, and 81 new drug-resistant disease categories. All in all, DRESIS 2.0 is expected to serve as a valuable resource for the scientific community and provide important support in tackling the global challenge of drug resistance, which is now publicly accessible at https://idrblab.org/dresis/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cafc45a66260eff1536a05a8101c61e45dec038b" target='_blank'>
              DRESIS 2.0: the comprehensive landscape of drug resistance information.
              </a>
            </td>
          <td>
            Xiuna Sun, Zhangle Wei, Xinyuan Yu, Kaixuan Liu, Shun Yao, Zheng Ni, Hanbing Wang, Ziming Zhao, Yinpeng Zhang, Yuxuan Liu, Hanlu Ding, Yintao Zhang, Zhiguo Liu, Mang Xiao, Feng Zhu
          </td>
          <td>2025-11-20</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Background: Fusion transcripts are formed by combining exons from two different genes, often due to structural rearrangements such as deletions, inversions or translocations (genomic rearrangement-dependent, GRD) or through aberrant splicing (genomic rearrangement-independent, GRI). In hematological malignancies, many fusion transcripts act as driver events, playing crucial roles in leukemogenesis and serving as diagnostic markers, such as BCR::ABL1. However, the role and diagnostic significance of other fusion transcripts, like P2RY8::CD99—which has been found in both healthy samples and B-ALL patients—are less clear. Similarly, the clinical relevance of other GRI fusion transcripts, such as SEMA6A::FEM1C, often remains uncertain. Single-cell analysis is a promising approach to detect these fusion transcripts within individual cells and cell populations, shedding light on the subclonal architecture and cells of origin.
 Aim: Evaluating the feasibility of integrating long-read sequencing with single-cell library preparation to detect and characterize GRD and GRI fusion transcripts.
 Patients and Methods: Our cohort consisted of 10 samples diagnosed with B-ALL, featuring various fusion transcripts: BCR::ABL1 (n = 6, GRD), SEMA6A::FEM1C (n = 10, GRI), EBF1::PDGFRB (n = 1, GRD), and P2RY8::CD99 (n = 1, GRI), as identified through bulk whole transcriptome analysis. Cryopreserved cells were processed using the GEM-X Universal 3' Expression Library Prep Kit (10x Genomics) and GEM-X cDNAs with cell-specific labels served as input for the Kinnex single-cell RNA kit (PacBio). GEM-X libraries were sequenced on the NovaSeqX instrument (Illumina) with a median depth of 21,565 reads per cell. Kinnex libraries on the Revio (PacBio) with 3,767,066 mean HiFi reads per sample and a mean HiFi read length of 14.2kb. GEM-X libraries were analyzed using cellranger (v9.0.1) and seurat (v5.2.1). Kinnex libraries were preprocessed with the Iso-Seq workflow (PacBio) and fusion transcripts were called with pbfusion.
 Results: Cells from different samples were merged and clustered based on their gene expression profiles from GEM-X libraries. Cell clusters were annotated as B-cells based on CD10, CD79A, and PAX5 expression; T-cells based on CD3 expression; hematopoietic stem cells as CD34+; and myeloid cells by the absence of lymphatic markers and the presence of CD14, FCER1G, and CEBPD. Within the B-cell population, three distinct subpopulations corresponding to various B-cell developmental states were identified: pro-B, pre-proB, and pro-B VDJ. The pro-B VDJ state, characterized by heavy chain rearrangement, showed increased CD20 expression. Furthermore, cell cycle analysis using 100 genes associated with the S-phase/G2M-phase revealed a subpopulation of cycling pro-B cells marked by high levels of MKI67 (G2M-phase) and MCM4 (S-phase). Our fusion calling pipeline successfully detected BCR::ABL1, SEMA6A::FEM1C, EBF1::PDGFRB, and P2RY8::CD99 fusion transcripts within the long-read dataset, without any false positive calls at the sample level. These fusion transcripts were found exclusively in B precursor cells when mapped to the cellular landscape. Notably, cells in the pro-B VDJ state did not harbour any fusion transcripts. Interestingly, P2RY8::CD99 was identified only in B-cells, not in T-cells or myeloid progenitors, suggesting its association with pathogenic cells. Similarly, SEMA6A::FEM1C, a GDI fusion transcript, was confined to B precursor cells, indicating an association with the B-ALL clone.
 Conclusions: We have demonstrated the feasibility and utility of integrating long-read isoform sequencing with single-cell library preparation for detecting fusion transcripts of diverse origins. This transcriptome-wide approach enables disease-agnostic detection and characterization of fusion transcripts, not only those arising from chromosomal aberrations (GRD) but also those resulting from aberrant splicing (GRI) at the individual cell level. Moreover, this method facilitates a comprehensive analysis of cell-specific gene expression profiles and fusion transcripts and could be expanded to include the detection of single-nucleotide variants and copy number changes, thereby completing the molecular profile. While currently not designed for integration into routine workflows, this method represents a valuable tool for research projects aimed at elucidating the molecular mechanisms underlying various diseases, with the goal to enhance patient care.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ae73b6fed764877551a830ff73b15a040a45186" target='_blank'>
              Long-read single-cell isoform sequencing for cell type-specific detection of genomic rearrangement-dependent and -independent fusion transcripts
              </a>
            </td>
          <td>
            W. Walter, W. Kern, A. Stengel
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="


 There is an active quest for biomarkers of high-risk disease in CML. ASXL1 and other cancer gene mutations, some additional cytogenetic alterations and Philadelphia-associated rearrangements (collectively, additional genetic abnormalities or AGAs) appear to have a negative impact on response and survival outcomes, but they are likely to mirror a more genetically unstable disease.
 SETD2 is a histone methyltransferase responsible for the deposition of the trimethyl mark at histone H3 lysine 36 (H3K36me3), a key epigenetic modification implicated in transcriptional elongation, chromatin architecture, DNA damage repair. SETD2 loss-of-function (LOF) due to inactivating mutations or, more frequently, accelerated proteasomal degradation, has recently been reported in patients (pts) with multi-TKI-resistant chronic phase (CP) or with blast phase (BP) CML, but it can be observed as early as at diagnosis, particularly in CD34+ hematopoietic progenitors.
 In this study, we aimed to dissect the impact of SETD2 LOF in primary CD34+ progenitors and in CML cellular models where SETD2 was alternatively silenced or overexpressed, using an integrated approach combining liquid chromatography-tandem mass spectrometry (LC-MS/MS), RNA sequencing (RNA-seq), chromatin immunoprecipitation sequencing (ChIP-seq) and single nucleotide polymorphism arrays (SNP-arrays) for discovery, and Western blotting (WB), immunofluorescence (IF), and co-immunoprecipitation (co-IP) for validation.
 In SETD2-deficient primary CD34+ progenitors from newly diagnosed CP CML pts, nucleofection of a SETD2-expressing construct reduced clonogenic potential of >50%, indicating that SETD2 LOF enhances leukemic cell propagation. Differential transcriptomic profiling in cell line models revealed SETD2-dependent transcriptional regulation of genes involved in DNA repair (MSH2, MSH6), cell cycle control (CDK1), and metabolic homeostasis (PFKP, LDHA, PDK1). Differential interactome profiling by LC-MS/MS identified SETD2 interactions with proteins critically involved in mismatch repair (MSH2, MSH6), cell division (α-/β-tubulin), and glycolysis (PFKP, PFKFB3, PD, LDHA). Notably, SETD2 was also found to interact with key kinases regulating proliferation and stress response, including ERK1/2 and p38 MAPK. All MS-identified interactions were experimentally validated by IF and co-IP in nuclear, cytoplasmic, or cytoskeletal fractions. Furthermore, IF imaging demonstrated the nuclear colocalization of SETD2 with γ-H2AX foci upon hydrogen peroxide and UV-induced genotoxic stress and its recruitment to DNA damage sites, where it spatially overlapped with MSH2/MSH6 complexes. Integration of SNP-array analysis after chronic exposure to DNA damaging agents with ChIP-seq-based genomic mapping of H3K36me3 showed enrichment of breakpoints at SETD2 target sites (SETD2 knocked-down cell line: 29/45 vs 4/45 regions enriched in genomic breakpoints had loss vs gain of H3K36me3, respectively [p=0.0002]; SETD2-deficient cell line: 25/33 vs 1/33 regions enriched in genomic breakpoints had H3K36me3 loss vs gain, respectively [p=0.009]).
 Notably, we uncovered a novel role for SETD2 LOF in rewiring cellular metabolism, since SETD2 re-expression attenuated the glycolytic shift observed in SETD2-deficient cells, as evidenced by downregulation of glycolytic enzymes and mitochondrial oxidative phosphorylation complexes used by SETD2-deficient cells as compensation of TCA down-regulation. This was functionally validated in both total lysates and isolated mitochondrial fractions. In contrast, SETD2-deficient cells displayed hyperactivation of hypoxia-associated pathways, consistent with pseudohypoxic reprogramming.
 Finally, SETD2/H3K36me3 deficiency as assessed by a simple WB assay in total leukocytes could be detected in CP CML pts with AGAs at diagnosis, and, importantly, could discriminate non-optimal vs optimal responders to subsequent imatinib therapy (tx).
 Our findings point to SETD2 LOF as a key cooperating event in CML, that may act since diagnosis to set the stage for TKI resistance and disease acceleration by i) sustaining BCR::ABL1-independent genomic instability that fuels acquisition of AGAs before and despite TKI tx, and ii) inducing metabolic reprogramming towards glycolysis; ultimately enhancing leukemogenicity of CML progenitors. SETD2 LOF may serve as a biomarker of high-risk disease at diagnosis, and its impact on response to 2nd-gen TKIs or asciminib vs imatinib is worth to be explored further.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e07e38cc7df8916cca2a64951654db48b472755c" target='_blank'>
              SETD2 deficiency in chronic myeloid leukemia (CML) contributes to tyrosine kinase inhibitor (TKI) resistance and disease acceleration by enhancing genetic instability and rewiring cellular metabolism and might be a novel biomarker of high risk disease since diagnosis
              </a>
            </td>
          <td>
            M. Mancini, Lucia Romagnoli, S. De Santis, S. Bruno, Cristina Mosca, Andrea Oldani, Alessandra Tagliavia, Alessandra Merlotti, D. Remondini, Erika Esposito, Jessica Fiori, F. Castagnetti, M. Iezza, R. Restuccia, Vanessa Bagnis, A. Iurlo, D. Cattaneo, M. Cerrano, Valentina Giai, S. Galimberti, P. Zinzani, S. Soverini
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Somatic chromosome instability results in widespread structural and numerical chromosomal abnormalities (CAs) during cancer evolution1, 2–3. Although CAs have been linked to mitotic errors resulting in the emergence of nuclear atypia4, 5, 6–7, the underlying processes and rates of spontaneous CA formation in human cells are underexplored. Here we introduce machine-learning-assisted genomics and imaging convergence (MAGIC)—an autonomously operated platform that integrates live-cell imaging of micronucleated cells, machine learning on-the-fly and single-cell genomics to systematically investigate CA formation. Applying MAGIC to near-diploid, non-transformed cell lines, we track de novo CAs over successive cell cycles, highlighting the common role of dicentric chromosomes as initiating events. We determine the baseline CA mutation rate, which approximately doubles in TP53-deficient cells, and observe that chromosome losses arise more frequently than gains. The targeted induction of DNA double-strand breaks along chromosome arms triggers distinct CA processes, revealing stable isochromosomes, coordinated segregation and amplification of isoacentric segments in multiples of two, as well as complex CA outcomes, influenced by the chromosomal break location. Our data contrast de novo CA spectra from somatic mutational landscapes after selection occurred. The experimentation enabled by MAGIC advances the dissection of DNA rearrangement processes, shedding light on fundamental determinants of chromosomal instability. Coupling live-cell imaging, machine learning and genomic sequencing, the MAGIC platform enables investigation of the cellular context, mutation rates and triggers of spontaneous chromosomal abnormality formation, shedding light on fundamental determinants of chromosomal instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9becd4af94c7faf26e4bbaab5024e42c7ccb439f" target='_blank'>
              Origins of chromosome instability unveiled by coupled imaging and genomics
              </a>
            </td>
          <td>
            Marco R. Cosenza, Alice Gaiatto, Büşra Erarslan Uysal, Álvaro Andrades, Nina Luisa Sautter, Marina Šimunović, M. Jendrusch, Sonia Zumalave, Tobias Rausch, A. Halavatyi, Eva-Maria Geissen, Joshua Lucas Eigenmann, Thomas Weber, Patrick Hasenfeld, Eva Benito, Catherine Stober, I. Cortés-Ciriano, A. Kulozik, Rainer Pepperkok, Jan O. Korbel
          </td>
          <td>2025-10-29</td>
          <td>Nature</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="


 Fusion transcription factors (TF) are hallmark oncogenic drivers in pediatric B cell acute lymphoblastic leukemia (B-ALL), contributing to leukemia initiation and transformation. Despite advances in risk-adapted therapies guided by clinical and molecular features, genetic heterogeneity and variable outcomes underscore the need to uncover driver-specific vulnerabilities to expand personalized treatment strategies, particularly in the relapsed/refractory context. Here, we apply a systematic approach to identify rational combination strategies informed by integrating identification of driver-specific downstream targets of the fusion TF TCF3::PBX1 – a common driver in pediatric B-ALL – with an unbiased high-throughput drug combination screen.
 To dissect fusion-specific druggable genes we knocked down (KD) TCF3::PBX1 by short hairpin RNA (shRNA) in the 697 cell line and filtered for transcriptional targets with available approved and clinical-phase candidate drugs. Among top fusion-associated genes we identified ornithine decarboxylase (ODC1), the rate-limiting enzyme of synthesizing polyamines, small polycations with high affinity for nucleic acids controlling key cellular processes like transcription, translation, DNA replication and cell cycle progression. Public datasets confirmed highest mRNA expression of ODC1 in TCF3::PBX1+ patient samples across B-ALL subtypes (St. Jude) and selective dependency on ODC1 in TCF3::PBX1+ cell lines (DepMap) reinforcing its role as a specific fusion-driven vulnerability. This subtype-specific overexpression of ODC1 led us to investigate its regulation, and public ChIP-seq data revealed binding peaks of TCF3::PBX1 at the ODC1 promoter supporting its role as a novel regulator alongside MYC. Genetic interference by shRNA-mediated ODC1 KD and pharmacological inhibition by Difluoromethylornithine (DFMO) reduced intracellular polyamine levels of putrescine, spermidine and spermine and demonstrated cytostatic effects with decreased proliferation with a reduction of cells in S-Phase in TCF3::PBX1+ cell lines. In patient-derived xenograft (PDX) models, ex vivo sensitivity to DFMO was highest in TCF3::PBX1+ PDX compared to other subtypes. Notably, PDXs from TCF3::PBX1+ relapses exhibited sensitivity levels comparable to those observed in TCF3::PBX1+ PDX with favorable outcomes. Transcriptomic and proteomic analysis of DFMO-treated 697 cells uncovered changes in gene and protein expression associated with amino acid and polyamine transport, mitochondrial metabolism, global translation and cell cycle. Strikingly, treatment with DFMO induced regulation of key B cell identity TF and B cell receptor signaling components, alongside an enrichment of transcriptional signatures indicative of more mature B cell states.
 To uncover synthetic lethal vulnerabilities of this induced rewired cell state we performed an unbiased high-throughput combination screen (n = 155 drugs) in 697 cells with or without ODC1 KD. Among known synergistic targets like AMXT-1501, a polyamine transport inhibitor, BCL-XL inhibitors emerged as potent synergistic partners to genetic and pharmacological ODC1 inhibition, further validated in PDX models ex vivo. To evaluate this combination in vivo, we employed a dual-targeting strategy against polyamine metabolism using DFMO to inhibit synthesis and AMXT-1501 to block uptake, as ex vivo data revealed pronounced metabolic plasticity and compensatory upregulation of polyamine transporters following DFMO monotherapy. In vivo combination of DFMO + polyamine-transport inhibitor AMXT-1501 and BCL-XL PROTAC DT2216 in TCF3::PBX1+ PDX models demonstrated promising antileukemic efficacy seen by reduced leukemic burden in spleen and bone marrow at the end of treatment.
 Collectively, our findings identify the fusion TF TCF3::PBX1 as a previously unrecognized positive regulator of polyamine synthesis, acting via ODC1 upregulation, positioning ODC1 as a promising target in this subtype. We further demonstrated that targeting polyamine synthesis induces transcriptional changes in essential B-cell developmental and metabolic programs, uncovering opportunities to target synthetic lethal principles through BCL-XL. The recent approval of DFMO for neuroblastoma maintenance therapy highlights its translational potential not only in TCF3::PBX1+ B-ALL but warrants exploration of this rational combination in other MYC-driven and/or polyamine-addicted B-ALL, especially in the relapsed/refractory context.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60ff0281ccf3599cc3a7d33b49b4b28f1ec98b96" target='_blank'>
              Fusion-specific regulation of polyamine synthesis sensitizes to BCL-XL inhibition in TCF3::PBX1+ B-ALL
              </a>
            </td>
          <td>
            James Kim, Sarah Cherkaoui, Simona Ulrich, Bartimée Galvan, Chengkang Li, Caroline Mussak, Samanta Carbajal, Shannon Mezger, B. Bornhauser, Raphael J. Morscher, Jean-Pierre Bourquin
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="


 Glioma, the most common primary brain tumor, shows poor prognosis and resistance to first-line therapies, like temozolomide (TMZ), lomustine (LOM) and panobinostat-marizomib (PM), complicating treatment and diminishing disease-free survival. It is critical to investigate the molecular mechanisms that underlie chemoresistance to develop effective therapies. This study examines common transcriptional networks that facilitate chemoresistance to identify novel therapeutic targets that may overcome resistance.



 The pediatric high-grade glioma KNS42 cell line was chronically treated with increasing doses of TMZ, LOM, or PM to induce drug resistance (RES); control cells received vehicle (DMSO) treatment. Bulk-RNA sequencing was performed followed by differential gene expression analyses (PartekFlow-ANOVA/DESeq2) followed by pathway and comparison analysis (QIAGEN IPA, |FDR|<0.05).



 RNA-sequencing analysis followed by comparison-analysis revealed differential gene expression in treatment-resistant glioma cells as compared to control cells, including 3,839 differentially expressed genes (DEGs) in LOM RES, 3,641 in TMZ RES, and 3,445 in PM RES groups compared to control group. Among these, 374, 417, and 360 DEGs were downregulated in LOM, TMZ, and PM RES cells, respectively, while 265 (LOM and TMZ RES) and 216 (PM RES) DEGs were upregulated (log2FC > 2; p < 0.05). Interestingly, when the top 10 upregulated genes from each resistant group were compared, six genes (VCAN, FOXA1, SOX2, PDGFRA, ERBB4, and COL4A) were found to be commonly upregulated in all 3 resistant groups. We further confirmed that SOX2, PDGFRA, and ERBB4 are upregulated at the protein level. Ongoing studies will test whether these 6 DEGs are required for mediating resistance against diverse drugs.



 RNA-sequencing identified common genes linked to TMZ, LOM, and PM chemoresistance, revealing key genetic networks involved in acquiring and maintaining drug resistance. Targeting these pathways, along with epigenetic silencing and/or combinatorial therapies, may resensitize chemoresistant cells and allow for effective treatment of glioma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5824551e4b1f9c0b4ac4e374b69dbac7316d823" target='_blank'>
              DDDR-78. Identifying common transcriptional mechanisms in treatment-resistant glioma
              </a>
            </td>
          <td>
            N. B. Rao, Aaron R Johnson, Subrahmanya D. Vallabhapurapu, Tracy A Miller, Ian F Pollack
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The development and progression of gastrointestinal (GI) cancers not only depend on the malignancy of the tumor cells, but is also defined by the complex and adaptive nature of the tumor microenvironment (TME). The TME in GI cancers exhibits a complex internal structure, typically comprising cancer cells, cancer stem cells, cancer-associated fibroblasts, immune cells, and endothelial cells, all embedded within a dynamic extracellular matrix. This intricate ecosystem fuels tumor initiation, progression, metastasis, recurrence and therapy response through the heterogeneity and plasticity. Recent advances in single-cell sequencing have provided unprecedented resolution in profiling the cellular diversity and interactions within the TME. These technologies have uncovered previously unknown cell subtypes and intricate communication networks that drive therapy resistance and tumor relapse. In this review, we summarize and discuss the latest findings from single-cell sequencing of key cellular players and their interactions within the TME of GI cancers. We highlight single cell insights that are reshaping our understanding of tumor biology, with particular focus on their implications for overcoming therapy resistance and improving clinical outcomes. We believe that a deeper understanding of TME heterogeneity and plasticity at the single-cell level promises to transform the landscape of precision treatment in GI cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45ef919b2bb866835d8195a594df10437839f0cc" target='_blank'>
              Single-cell insights into tumor microenvironment heterogeneity and plasticity: transforming precision therapy in gastrointestinal cancers
              </a>
            </td>
          <td>
            Jialei Weng, Feng Ju, Zicheng Lyu, N. Fan, Daniel J. Smit, Wenxin Xu, Xiaolin Wu, Philip Becker, Yinan Xu, Michal R. Schweiger, Axel M. Hillmer, Ralf Harwig, Sheraz Gul, Alexander Link, Lydia Meder, Nan Fang, Qiongzhu Dong, Christiane J. Bruns, Ning Ren, Yue Zhao
          </td>
          <td>2025-11-28</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Immune checkpoint blockade (ICB) therapy, which has revolutionized cancer treatment, has been approved for the treatment of triple-negative breast cancer (TNBC). Unfortunately, most patients with TNBC are either not eligible for treatment or exhibit resistance, resulting in limited overall survival benefits. There is an urgent need to elucidate the mechanisms of resistance and enhance therapeutic efficacy. Here, via CRISPR activation (CRISPRa) screening, we identified family with sequence similarity 114 member A1 (FAM114A1) as a key mediator of immune evasion and ICB resistance in TNBC. Mechanistically, FAM114A1 binds p85α to disrupt the p85α/p110α protein complex, thus activating the PI3K/AKT pathway and simultaneously preventing condensate formation of E2F Transcription Factor 4 (E2F4) to promote E2F4-driven Metadherin (MTDH) transcription. Upregulation of these FAM114A1-mediated pathways suppresses tumor antigen presentation and consequently attenuates antitumor immunity in TNBC. Moreover, targeting FAM114A1 improves the therapeutic effectiveness of anti-PD-1 therapy in mouse models, and a FAM114A1-based signature shows strong predictive performance for identifying patients with TNBC who may benefit from ICB. Collectively, our findings not only reveal that FAM114A1 is an immune evasion driver but also highlight it as a promising biomarker and therapeutic target. Our study provides new insights into TNBC immune evasion and outlines a potential avenue to improve the effectiveness of ICB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d6cebf1ccb54b2528f79a6a1b67182cda5a3f55" target='_blank'>
              Family with sequence similarity 114 member A1 orchestrates immune evasion in triple-negative breast cancer
              </a>
            </td>
          <td>
            Wenhao Zhang, Yanzhi Gai, Mengxue Qiao, Michelle Rowicki, Yong Wei, Xiang Hang, Zhengkai Wei, He Yang, Xifu Ye, Hang Ju, Yi Lu, Yibin Kang, Minhong Shen
          </td>
          <td>2025-11-18</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Chimeric antigen receptor (CAR)-T cell therapy has shown remarkable success in hematologic malignancies but faces significant challenges in solid tumors due to the immunosuppressive tumor microenvironment (TME). Among these, hypoxia plays a vital role, yet the molecular mediators that link hypoxia to CAR-T dysfunction remain incompletely understood. Anti-mesothelin (MSLN) CAR-T cells were cultured under normoxic (21% O2) and hypoxic (1% O2) conditions for six days. We assessed cell expansion, phenotypes, cytotoxicity, and metabolic features. RNA sequencing was conducted to identify key gene expression changes induced by hypoxia. Findings were further validated using anti-CD70 CAR-T cells. Hypoxia reduced CAR-T proliferation, increased apoptosis, lowered memory phenotypes, raised exhaustion, and weakened cytotoxicity in short-term and long-term assays. Transcriptomic and metabolic analyses showed metabolic reprogramming with increased glycolysis and reduced oxidative phosphorylation. Among the dysregulated genes, the serine/threonine-protein kinase PIM3 emerged as a previously underexplored mediator of hypoxia-driven dysfunction. Genetic or pharmacologic inhibition of PIM3 counteracted hypoxia-induced impairment, enhancing memory phenotypes of CAR-T cells, and improving their anti-tumor activity both in vitro and in vivo. This work identifies PIM3 as a previously underexplored target that links hypoxia to CAR-T cell dysfunction and demonstrates that PIM3 inhibition can reverse these effects. These findings provide a mechanistic rationale for incorporating PIM3 inhibition into CAR-T cell manufacturing or engineering to improve their therapeutic potential in hypoxic solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2adcb89e4cbae164c14c49d77ccdb838d38a89aa" target='_blank'>
              Metabolic reprogramming through PIM3 inhibition reverses hypoxia-induced CAR-T cell dysfunction in solid tumors
              </a>
            </td>
          <td>
            Muya Zhou, Luxia Xu, Jinhua Hu, Wenwen Chen, Jialu Hong, Mufeng Wang, Zhi-Cheng Guo
          </td>
          <td>2025-11-06</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD), a subtype of non-small cell lung cancer (NSCLC), has high incidence and poor prognosis. Although anti-programmed cell death protein 1 (PD1) therapy and epidermal growth factor receptor (EGFR) inhibitors benefit some patients, many remain unresponsive. Genome-wide association studies (GWAS) can identify risk loci, while single-cell transcriptomics enables exploration of genetic variations and mechanisms in LUAD. We integrated GWAS data from FinnGen, IEU OpenGWAS, and GWAS Catalog with single-cell and spatial transcriptomics from GEO to investigate LUAD genetics and pathology. GWAS analyses identified SNP loci, risk factors, LD Score, and SMR results, revealing a novel apoptosis-related locus, Serine/Threonine Kinase 24 (STK24) (SMR P < 0.05). Single-cell analysis identified STK24-positive epithelial cells (STK24posEpi) and their transcription factors (SCENIC), developmental trajectory (Monocle), and tumor microenvironment roles (CellCall). Spatial mapping (RCTD) and trajectory analysis (stlearn) were combined with cell communication (CellChat), spatial signal flow (Commot), and co-localization (Misty). Bayesian deconvolution linked STK24posEpi to TCGA-LUAD bulk transcriptomes, assessing overall survival (OS), immune infiltration, and clinical traits. A prognostic model (STK24SuperPC, mean c-index 0.61) was built using multiple machine learning algorithms to predict survival and immunotherapy response. Laboratory experiments tested STK24’s effects on LUAD cell proliferation and apoptosis. STK24 was identified as a LUAD risk factor (pSMR < 0.05). STK24posEpi communicated with fibroblasts, endothelial cells, and others via MIF and PDGF pathways, and was associated with apoptosis (Importance > 0.5), metastasis (Importance > 2), DNA repair (Importance > 1), and DNA damage (Importance > 2). Located at the origin of the trajectory, STK24posEpi correlated with poor prognosis (HR > 0, P < 0.05) and reduced lymphocyte infiltration (Pearson’s r < 0, P < 0.05). STK24 overexpression enhanced LUAD cell proliferation, migration, and colony formation, while reducing apoptosis, increasing BCL-2 and Caspase-3, and decreasing Cleaved-Caspase-3 expression. STK24, an apoptosis-related gene, promotes LUAD progression by enhancing proliferation and inhibiting apoptosis, with STK24posEpi interacting in the tumor microenvironment through PDGF and MIF pathways. These findings offer potential therapeutic targets and support personalized LUAD treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f038883aea606036c62cbeb3f18e89605908569d" target='_blank'>
              Genome-wide association, single-cell, and spatial transcriptomics analyses reveal the role of the STK24-expressing positive cells in LUAD progression and the tumor microenvironment, identifying STK24 as a potential therapeutic target
              </a>
            </td>
          <td>
            Junzhi Liu, Huimin Li, Yuheng Jiao, Meng Hou
          </td>
          <td>2025-10-30</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Background Acute myeloid leukaemia (AML) is a malignancy of the haematopoietic system with extremely high mortality. Current AML therapies, such as chemotherapy, hematopoietic stem cell transplantation, and targeted therapy, have greatly improved the prognosis of AML patients. However, there are still a number of AML patients who develop drug resistance after pharmacotherapy, leading to poor prognosis and relapse. Currently, the global 5-year survival rate for adults with AML remains below 50%. Therefore, it is urgent to identify the aetiology for AML relapse and drug resistance.
 Methods Bone marrow mononuclear cells were isolated from bone marrow blood samples from 4 patients with AML relapse, 3 patients with AML remission and 5 control donors by density gradient centrifugation using Lymphocyte separation medium. After RNA extraction and sequencing, differentially expressed genes (DEGs) analysis, overlapping analysis, WGCNA and co-expression analysis, alternative splicing analysis and functional enrichment analysis were further performed.
 Results RNA sequencing andDEGs analysis demonstrated that a total of 593 overlapped up-regulated genes and 999 overlapped down-regulated genes were discovered in the reAML group in comparison to the Healthy and AML groups. Of particular, further overlapping analysis revealed that 33 RBP genes were overlapped up-regulated and 30 RBP genes were overlapped down-regulated in the reAML group. Further WGCNA and alternative splicing analyses highlighted that significant changes were found in AS in the reAML group compared with the AML and Healthy groups, and multiple differential regulatory alternative splicing genes (RASGs) and regulatory alternative splicing events (RASEs) were also identified.
 Conclusions The aberrance of regulatory alternative splicing (RAS) and differential expression of RNA-binding protein (RBP) genes are highly associated with AML relapse and drug resistance in AML patients.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2939a3706c9e8e829fd5d4dfeb6fd0b9e552e7a" target='_blank'>
              Genome-wide investigation of differentially expressed alternative splicing and RNA-binding protein genes association with AML drug resistance
              </a>
            </td>
          <td>
            Jingyi Wang, Yu-Mei Zhang, Xueyan Dong, Wei Zheng, Jie Xu, Zhenzhen Wang, Shumin Ding, Yingying Wang
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Introduction: Diffuse large B-cell lymphoma (DLBCL) is a common and aggressive B-cell malignancy with heterogeneous genomic findings, including a variety of fusions and copy number alterations. However, standardization among laboratories for using cytogenetic techniques is lacking and is often targeted, thus precluding a genome-wide analysis. Herein, we present a case of DLBCL with unresolved fluorescence in situ hybridization (FISH) findings while also providing additional genomic findings that may facilitate therapy selection.
 Case details and methods: Case details are that of a 70-year-old male with DLBCL with a non-GC phenotype. FISH revealed an IGH rearrangement to an unknown partner while MYC revealed copy number gain, likely indicating polysomy of chromosome 8. The patient was treated with polatuzumab vedotin-rituximab-cyclophosphamide, doxorubicin, and prednisone (pola-R-CHP) but subsequently progressed through therapy. To assess the IGH rearrangement and identify additional genomic events, whole-genome Hi-C sequencing was performed from 5 micron formalin-fixed, paraffin embedded (FFPE) sections and cross-linked chromatin was digested using a restriction enzyme. The 5'-overhangs were filled and digested ends were labeled with biotinylated nucleotide. Spatially proximal digested ends of DNA were ligated, which was then purified, fragmented, and enriched. DNA libraries were then prepared and sequenced by paired-end sequencing. Structural variant (SV) analysis was performed using HiCUP and hic_breakfinder. SV were then manually curated and processed to create a single VCF file.
 Results: Whole-genome Hi-C sequencing revealed a fusion of IGH to an intergenic partner on chromosome 10, resulting in a t(14;10). Copy number variant analysis revealed gains of chromosomes 4, 7, 8, and 15q with loss at 5p. Interestingly, an 18 Mb tandem amplification on chromosome 17 resulted in amplification of PRKCA, CD79B, CLTC, MSI2, and HLF.
 Conclusion: Hi-C sequencing was able to define the partner of the IGH gene rearrangements to be an intergenic region of chromosome 10, revealing that this was likely not related to the pathogenesis in this case. Furthermore, Hi-C sequencing found copy number aberrations and a tandem amplification of CD79B, a key regulator in B-cell signaling. Accordingly, this finding is associated with a more aggressive phenotype in DLBCL, as was observed in this patient. Coincidentally, the patient was already appropriately treated with pola-R-CHP as polatuzumab vedotin is an antibody-drug conjugate targeting CD79b. Herein, we demonstrated that whole-genome Hi-C sequencing is a robust technology that facilitates SV analysis on FFPE in DLBCL. Accordingly, this technique is sensitive and specific as it can refine unclear FISH findings while potentially providing additional genomic biomarkers, ultimately guiding therapy selection and/or clinical trial enrolment.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a9f091bb1e0da1accf8110ac74b9a1d49c33bd6" target='_blank'>
              Clinical use of genome-wide hi-c sequencing for assessment of structural variants in diffuse large B-cell lymphoma
              </a>
            </td>
          <td>
            Phillip Michaels, D. Qualls, Anthony D Schmitt, Alex Hastie
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd58fafdd674bdec3d7247a90b13d164f167e167" target='_blank'>
              Human stem cell models for group 3 medulloblastoma uncover JARID1B as a regulator of the chromatin landscape
              </a>
            </td>
          <td>
            Zulekha A. Qadeer, Elise Hou, Dara Siegel, Samantha Westelman, B. Gudenas, Kyle S Smith, Wazim Mohammed Ismael, Linyu Wang, Mackenzie O Johnson, Shane Grele, Ronald Phua, Liam D. Hendrikse, Ekin Guney, Miller Huang, Alexandre Gaspar-Maia, P. Northcott, Michael D. Taylor, William A. Weiss
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="


 Understanding of the heterogeneity in the formation of the Ph has evolved as technology has advanced. In the pre-TKI era, FISH revealed derivative 9 deletions that were associated with shorter survival. The poor outcomes were overcome with imatinib. Whole exome and transcriptome sequencing revealed greater complexity: inversions; novel 9;22 fusions; sequence fragmentation; and random reassembly. These were termed Ph-associated rearrangements. They occurred on the Ph and/or the derivative 9, were thought to occur at the same time as the Ph and correlated with poorer response to imatinib.



 To use whole genome sequencing (WGS) to examine derivative chromosomes and further resolve the complexity of Ph-associated rearrangements in patients at diagnosis and pan-TKI resistant cell lines.



 WGS was performed for 7 chronic phase CML patients at diagnosis and 7 matched sensitive/resistant Ph-positive cell lines. Resistance was induced by exposure to increasing concentrations of imatinib. Data was analyzed using Oncoanalyzer. Derivative chromosome structure was resolved using the LINX algorithm, which clusters structural variants that are highly unlikely to occur independently. Rearrangements were validated using orthogonal methods, including Optical Genome Mapping and SNP Array. Six of 7 patients had pre-characterized Ph-associated fusions identified using RNAseq.



 For most patients, a balanced reciprocal Ph translocation involves 2 rearrangements: 1 that generates the Ph and BCR::ABL1; and 1 reciprocal derivative 9 with ABL1::BCR. These structures were resolved for 1 patient without pre-characterized Ph-associated rearrangements and for 5 of 7 parental cell lines.
 A distinct class of complex structural rearrangement called chromoplexy was identified involving the Ph translocation in 11 samples. These Ph-associated rearrangements had the highest number of linked structural variants within the individual genomes of 10 of the 11 samples. Chromoplexy involves the formation of extensive chained rearrangements affecting multiple chromosomes that create an intricate series of inter/intra-chromosomal fusions, deletions, duplications and inversions. Many DNA strands were broken and ligated in new configurations in the affected samples. Multiple genes on the derivative chromosomes were disrupted, including recurrent cancer drivers.
 For 3 of 7 matched sensitive/resistant cell lines, Ph-associated rearrangements were acquired in the resistant lines. The most complex events occurred in the LAMA84 resistant line. The parallel sensitive line had the standard 2 rearrangements. However, the resistant line gained 17 rearrangements with highly amplified Ph sequences, inversions and novel genomic fusions. Remarkably, small fragments from 6 chromosomes (2, 4, 10, 13, 17, X) were inserted upstream and downstream of BCR::ABL1, and disrupted the NUP214 driver gene.
 The K562 sensitive line displayed 24 rearrangements with non-contiguous amplified Ph regions of different copy number. Events included NOTCH1 deletion, 9;22 NUP214 fusion and an adjacent 9;13 fusion. Six fragments from chromosome 13 were incorporated into the Ph. The K562 resistant line displayed further evolution with 31 Ph rearrangements.
 The AR230 sensitive line displayed Ph chromoplexy with 9 rearrangements. The resistant line evolved with 19 additional Ph rearrangements. A fragment from chromosome 21 was inserted upstream of BCR::ABL1 and disrupted the MAPK1 driver gene. The LZTR1 tumor suppressor was deleted. All other cell lines maintained the standard Ph configuration.
 Ph-associated chromoplexy was evident in 6 patient samples: average 10 rearrangements. The patient with the highest number (15) progressed to blast phase at 4 months of imatinib. Four Ph-associated subclones were detected at diagnosis with non-contiguous amplified regions, consistent with the evolved cell lines. Furthermore, the LZTR1 tumor suppressor gene was deleted, as detected in AR230. The blast phase sample was unavailable. Deep molecular response was achieved in 1 patient on imatinib and 4 on frontline or switch to nilotinib/dasatinib.



 We have demonstrated that Ph chromoplectic rearrangement can be present at diagnosis but can also be initiated during leukemic progression and disrupt multiple genes, including cancer drivers. The Ph may be prone to extensive DNA breakage. Ph chromoplexy could constitute a new mechanism that contributes to acquired TKI resistance.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eed250c9c3a664955ee35598f9870a47e6c84f2b" target='_blank'>
              Whole genome sequencing reveals that the Philadelphia chromosome (Ph) can evolve and gain complex chromoplectic sequence rearrangements associated with tyrosine kinase inhibitor (TKI) resistance
              </a>
            </td>
          <td>
            S. Branford, C. Wadham, N. Shahrin, V. Saunders, Rosalie R. Kenyon, Ming Lin, Dominic Kaczorowski, Muneeza Maqsood, Hamish Scott, John Toubia, N. Shanmuganathan, C. Kok, Timothy Hughes
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a heterogeneous disease that arises from dysregulated myeloid proliferation. We performed a high-throughput CRISPR interference (CRISPRi) screen in the THP-1 monocytic cancer cell line to identify long noncoding RNAs (lncRNAs) that play a role in contributing to cell proliferation. Our screen identified INSTAR (Intergenic Nuclear Suppressor lncRNA Targeting Adjacent Regulator SFMBT2) as a top candidate. RNA-seq on INSTAR deficient THP-1 cells revealed transcriptional changes in genes involved in cell proliferation as well as other cellular processes. Loss of INSTAR selectively reduced expression of its neighboring gene, SFMBT2. Functional assays confirmed that both genes suppress cell growth, revealing a cis-regulatory mechanism in which INSTAR regulates SFMBT2 expression to control monocyte proliferation. Here, we leverage high-throughput screening to rapidly pinpoint functional lncRNAs providing novel insights into a key regulatory locus consisting of INSTAR and SFMBT2 which could be critical for better understanding dysregulation contributing to acute myeloid leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78f9f6afd97add9714e69419ecf796522c175c3f" target='_blank'>
              CRISPRi Screen Identifies a Novel Growth Suppressor lncRNA, INSTAR, in Human Monocytes
              </a>
            </td>
          <td>
            Christy Montano, Eric Malekos, S. Covarrubias, Sol Katzman, Lisa Sudek, Jillian Ward, S. Carpenter
          </td>
          <td>2025-10-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de84f1d76b8dc759a28178cb80d35ff586603711" target='_blank'>
              Glioblastoma gene expression based subtypes have defined metabolomic states
              </a>
            </td>
          <td>
            Hélèna L. Denis, Jaëlle Méroné, Akhil Shukla, V. Watters, V. Fort, Gabriel Khelifi, Mikalie Lavoie, Line Berthiaume, Félix Rondeau, Émy Beaumont, Karine Michaud, M. D'Astous, S. Saikali, Marc-Étienne Huot, Étienne Audet-Walsh, Maxime Richer, Samer M. I. Hussein
          </td>
          <td>2025-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="


 Background Acute lymphoblastic leukemia (ALL) is the most prevalent malignant tumor among children and adolescents, with poor prognosis in adults, as evidenced by a complete remission rate of <40%. Chemoresistance remains a major challenge in ALL treatments, with survival rates for resistant patients falling below 30%. While molecular subtyping based on next generation sequencing (NGS) is a key prognostic indicator of B-cell precursor ALL (BCP-ALL), chemotherapy responses often do not align with expectations based on molecular classification. This discrepancy arises, in part, because certain cases of ALL are driven by structural variants (SVs), which NGS, with its short-read sequencing approach, struggles to detect particularly large insertions, deletions, and complex rearrangements. In this study, we performed long-read sequencing (third-generation sequencing) to identify functional SVs that were missed by NGS, aiming to elucidate their roles in chemo-resistance and prognostic heterogeneity within defined BCP-ALL subtypes.
 Methods A panel of patient-derived xenografts (PDXs) were established, representing major BCP-ALL subtypes, including MEF2D-rearranged (n=3), BCR::ABL1/-like (n=3), TCF3::PBX1 (n=2), KMT2A-rearranged (n=2), PAX5::ETV6 (n=1), TCF3::ZNF384 (n=1) and B-other (n=2). Long-read whole-genome sequencing was performed using ONT PromethION platform. SV calling was carried out using an in-house bioinformatics pipeline that integrated 5 publicly available SV callers (cuteSV, Severus, Sniffles, SVIM, and SVision). Germline SVs were removed by comparison with the PGG.SV and DGV databases. In addition, HiPore-C (high-throughput Pore-C) was employed to explore alterations in topologically associated domains (TADs) related to SVs. Candidate SVs were annotated and subjected to functional validation in molecular biology studies.
 Results 5,468 to 8,097 SVs per sample (average 6,557) were identified across 14 BCP-ALL PDXs. Consistent with prior reports, deletions and insertions accounted for the majority of identified SVs, with 89.7% and 91.2% being <1 kb, respectively. In contrast, inversions and duplications exhibited a broader size distribution with a substantial proportion beyond 10 kb (38.5% and 11.7% respectively). Deletions and duplications were enriched in transcriptionally active and GC-rich regions, with breakpoints closer to TAD boundaries. Translocations and insertions, while also enriched at TAD boundaries, exhibited an opposite coverage pattern of GC content and more frequently associated with transcriptional repression. Inversions, however, showed limited association with transcriptional activity.
 Next, we interrogated intra-subtype heterogeneity of SVs in ALL. Two MEF2D::BCL9 PDXs, although similar in transcriptional profiles, exhibited distinct clinical prognoses and glucocorticoid (a standard of care drug for ALL) sensitivities in vitro and in vivo. Long-read sequencing pinpointed SVs associated with distinct drug response of the two PDXs. The glucocorticoid-resistant PDX demonstrated a more complex MEF2D::BCL9fusion, with MEF2D duplication and an insertion of a 1.6 kb downstream non-coding sequence into the fusion gene. HiPore-C captured aberrant chromatin interactions in these complex SVs. In addition, intragenic amplificationofPAX5 was found in a previously unclassified PDX (B-other subtype), which diminished PAX5 binding at glucocorticoid response elements (GREs) conferring glucocorticoid resistance. In contrast, the glucocorticoid-sensitive PDX exhibited PAX5 and glucocorticoid receptor (GR) colocalization at GREs. Moreover, beyond standard chemotherapy, PDXs harboring deletion at MTAP gene locus showed increased sensitivity to MTA-cooperative PRMT5 inhibitors via a synthetic lethal effect, providing a novel therapeutic strategy for MTAP-deficient ALL.
 Conclusion Overall, this study identified SVs missed by short-read NGS in BCP-ALL and demonstrated the critical role of these SVs in intra-subtype heterogeneity of drug response across various ALL subtypes. Our findings suggest that SV-based stratification could enhance prognostic accuracy and guide therapeutic decisions, offering a novel precision-medicine framework for ALL that transcends traditional subtype definitions.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f62af7fb7c8ba36d78969d83b675bd148b4f29e" target='_blank'>
              Long-read profiling of structural variants reveals mechanisms of chemo-resistance and prognostic heterogeneity in acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Xiaofang Ma, Minghao Jiang, Jin Wang, Jian-Qing Mi, Jason W.H. Wong, Duohui Jing
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 MAPK pathway alterations occur in 10-20% of diffuse midline glioma, H3K27-altered (DMG-H3K27), resulting in a proposed novel subtype of the disease with distinct clinical features. In a previous co-clinical trial platform, we showed an increased sensitivity to MEK inhibitors in DMG-MAPK patient-derived models, though resistance occurred in vitro, in vivo and clinically. To understand the nature of these resistance mechanisms, we used sophisticated coupling of retrievable high-complexity barcoding and single cell RNA sequencing in DMG-MAPK cells. Lineage tracing conducted in several technical replicate experiments showed a subset of clones is consistently enriched across replicates when treated with a GI90 concentration of trametinib for 11 weeks. Interrogation of clonal dynamics using mathematical modelling showed barcodes commonly enriched across replicates exhibit significantly higher clonal fitness compared to replicate-specific enriched barcodes, suggesting a deterministic pattern of resistance. Serial single cell RNA sequencing analysis at weeks 5, 8 and 11 post-treatment initiation showed some resistant H3.3K27M/BRAF-mutant cells adopted astrocyte-like and neural progenitor-like metaprograms in which barcodes were commonly enriched across two cell states, suggesting a subset of clones circumvent trametinib treatment by phenotypic switching; simultaneously, a late-emerging resistant population of cells activate a distinct mesenchymal program. Furthermore, we determine the previously identified MAP2K1 (MEK1) mutations to arise specifically from a small population of radial glia-like stem cells that appear acutely predisposed to the acquisition of these mutations. Together, our data suggest two clonally and mechanistically distinct resistance mechanisms can be active in DMG-MAPK in response to MEK inhibition, involving both transcriptional plasticity and a progenitor-like state primed for the acquisition of genetic determinants which circumvent drug response. With trametinib-resistant cells being highly sensitive to selective SRC inhibition, evolutionary-guided combinatorial treatment schedules may provide for strategies to significantly lengthen time taken for resistance development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c54926c3fb10216fcf5c70e35738abc3a296f32" target='_blank'>
              DDDR-39. Untangling MEK inhibitor resistance in DMG-MAPK using coupled lineage tracing and single cell RNA sequencing
              </a>
            </td>
          <td>
            Molina Das, Y. Grabovska, Haider Tari, A. Mackay, Rebecca F Rogers, Jared Collins, Jiin Song, Laura Bevington, A. Burford, Drenusha Sejdiu, R. Beroukhim, P. Bandopadhayay, Chris Jones
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>152</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ec13fad98d61dff3c83ecb3a4a35f96c11ec562" target='_blank'>
              Targeting MORF4L1-mediated DNA repair potentiates RT-induced antitumor immunity via cGAS-STING activation in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Si-wei Wang, Wei-Feng Hong, Yi-Lan Huang, Yi-Min Zheng, Qiu-Yi Zheng, Jun-Jie Cheng, Bu-Fu Tang, Gen-Wen Chen, Bu-Gang Liang, Jia-Cheng Lu, Li Yuan, Shujung Hsu, Yang Zhang, Xiao-Bin Zheng, Zhaochong Zeng, Jian Zhou, Jia Fan, Aiwu Ke, Chao Gao, Shi-Suo Du
          </td>
          <td>2025-11-05</td>
          <td>Cellular & Molecular Immunology</td>
          <td>1</td>
          <td>20</td>
        </tr>

        <tr id="Abstract Objectives Immunotherapies, including chimeric antigen receptor T-cell (CAR-T) therapy, represent a pivotal approach in the treatment of multiple myeloma (MM). However, the complex immunosuppressive tumor microenvironment (TME) poses significant challenges to their efficacy. Among the immunosuppressive cells in the MM TME, granulocytic myeloid-derived suppressor cells (G-MDSCs) are predominant, yet their functions remain incompletely understood. This study aimed to comprehensively analyze the role of G-MDSCs in MM patients undergoing CAR-T therapy. Methods Single-cell transcriptomic data from seven MM patients before and after CAR-T therapy were analyzed to characterize G-MDSCs. Functional enrichment and gene set enrichment analysis (GSEA) were performed to identify signaling pathways. A risk prediction model was constructed using RNA-seq and survival data from 713 MM patients, and validated by Kaplan–Meier analysis. In vitro experiments were conducted to assess the immunosuppressive functions of G-MDSCs. Results The findings, though exploratory due to limited sample size and inter-patient heterogeneity, suggested pathological activation and immunosuppressive roles of G-MDSCs potentially linked to patient prognosis. G-MDSCs were identified as key modulators of immune responses within the TME, with GSEA indicating regulation via IFN-α/γ signaling. They may facilitate immune evasion of MM cells by promoting proliferation through the IGF1–IGF1R axis and inhibiting T cells and other immune cells via the SIRPA–CD47 pathway. The risk prediction model based on G-MDSC gene signatures demonstrated high prognostic accuracy (AUC = 0.94). PTGS1 was identified as a key marker associated with high-risk groups, and functional assays confirmed its role in mediating G-MDSC immunosuppressive activity. Conclusions This study provides preliminary insights into the functional role of G-MDSCs in the MM TME and highlights their impact on CAR-T therapy outcomes. The findings suggest potential therapeutic strategies, including targeting PTGS1, to enhance CAR-T efficacy. Larger and more diverse cohorts are required to substantiate these observations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e8ff33894cad405d661ae879b8aae448cec342e" target='_blank'>
              Functional role of granulocytic myeloid-derived suppressor cells in CAR-T therapy: insights from single-cell RNA sequencing in multiple myeloma
              </a>
            </td>
          <td>
            Chaoyu Zhang, Fang An
          </td>
          <td>2025-11-02</td>
          <td>Immunopharmacology and Immunotoxicology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Adoptive cell therapy (ACT) has shown remarkable success in the treatment of some malignancies, particularly leukemia. However, there are multiple factors that limit the durability of ACT in solid tumors, including dose-limiting toxicities, the immunosuppressive tumor microenvironment, and T-cell exhaustion. As the manufacture and preparation of adoptive T-cell therapies allows time and adequate conditions for ex vivo T-cell engineering, forward genetic screens can identify novel genetic targets that could improve their effectiveness. CRISPR is a commonly used functional genomics tool that has been successfully used to both enhance our understanding of mechanisms of resistance and to discover potential genetic edits to improve ACT. A complementary approach, Sleeping Beauty transposon mutagenesis provides additional opportunities to identify novel genetic edits without being constrained by the annotated human genome. Here, we summarize forward genetic screens and their tools to uncover strategies to enhance ACT. Complementary approaches can be combined and improved on to identify translatable genetic editing strategies through studies that accurately recapitulate disease-specific challenges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eba1ded609a6b6846507e3268782cac85e220153" target='_blank'>
              Functional genomics for improving adoptive T-cell transfer therapies
              </a>
            </td>
          <td>
            Joseph G. Skeate, Chang-Jung Lee, C. Stewart, Mathew J Fischbach, Bibek Kar, Alexander K. Tsai, S. Kenderian, Ingunn M. Stromnes, D. A. Largaespada, B. Moriarity, Laura M. Rogers
          </td>
          <td>2025-12-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Background Acute myeloid leukemia (AML) remains a clinical challenge with suboptimal long-term survival. While circular RNAs derived from the RNF220 host gene have been implicated in AML pathogenesis, the functional role and regulatory mechanisms of RNF220 itself in AML are poorly understood. Methods We integrated bioinformatics analyses of public databases (TCGA-LAML, TARGET-LAML) and local cohort with in vitro functional assays. RNF220 was knocked down and overexpressed in AML cell lines using lentivirus. Transcriptomic profiling (RNA-seq), metabolic pathway enrichment (GSVA, GSEA), and immune microenvironment deconvolution (xCELL, CIBERSORT, MCP-counter) were performed. Transcription factor binding sites were predicted across five databases (JASPAR, ENCODE, GTRD, etc.). Validation of transcriptional regulation was performed using ChIP-PCR and luciferase reporter assays. Results RNF220 overexpression correlated with poor prognosis in AML, drove an immunosuppressive microenvironment characterized by reduced CD8+ T cells, and inhibited NK activity and M2 polarization of macrophage. RNF220 promoted tumor proliferation by suppressing apoptosis and preventing G1 arrest. Knockdown of RNF220 dysregulated metabolic pathways, notably suppressing glycolysis and phenylalanine metabolism. Mechanistically, FOXA1 was identified as an upstream negative regulator of RNF220, where high FOXA1 predicted favorable survival and inversely correlated with RNF220-associated metabolic reprogramming. Conclusion NF220 acts as an oncogenic ubiquitin ligase in AML by coordinating dual pro-leukemic mechanisms: cell-intrinsic metabolic rewiring (glycolysis/phenylalanine) and immune evasion via microenvironment suppression. Targeting the FOXA1–RNF220 axis may offer novel therapeutic strategies for high-risk AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4ff73f3a35554c15311058c2d83eb2628a0504d" target='_blank'>
              Dual oncogenic role of RNF220 in AML: linking metabolic rewiring to cell proliferation and immune evasion
              </a>
            </td>
          <td>
            Bi-xia Li, Shi Jiang, Yao Xu, Xiao Yan, Qitian Mu, Guifang Ouyang
          </td>
          <td>2025-10-30</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Anaplastic lymphoma kinase (ALK), a member of the receptor tyrosine kinase family, plays a central oncogenic role in the initiation and progression of diverse malignancies. Aberrant ALK activation generally results from structural alterations or dysregulated expression, leading to persistent activation of downstream signaling pathways that drive tumor cell proliferation, survival, and metastasis. ALK gene abnormalities predominantly encompass fusions, point mutations, and amplifications, with EML4-ALK-positive non–small cell lung cancer representing a canonical example. The advent of ALK-targeted inhibitors has constituted a major therapeutic milestone for ALK-positive tumors. From first-generation Crizotinib to third-generation Lorlatinib, successive agents have been refined for target selectivity, central nervous system penetration, and coverage of resistance-associated mutations, substantially improving patient survival and intracranial disease control. Nonetheless, the emergence of acquired resistance remains an overarching challenge, mediated by secondary kinase domain mutations, activation of bypass signaling pathways, and tumor phenotypic transformation. This review presents an integrative synthesis of ALK-targeted therapeutic developments, elucidates underlying resistance mechanisms, and surveys emerging strategies, providing a comprehensive perspective on current advances and future directions in precision management of ALK-driven malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74ec5cc87a27a59b857a6b6506e0949481a4fc3f" target='_blank'>
              ALK-Targeted Therapy: Resistance Mechanisms and Emerging Precision Strategies
              </a>
            </td>
          <td>
            Ya‐kun Zhang, Jianbo Tong, M. Luo, Zhi‐Peng Qin, Rong Wang
          </td>
          <td>2025-11-27</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Reconstructing and understanding intra-tumor heterogeneity, the coexistence of multiple genetically distinct subclones within the tumor of a patient, and tumor development is essential for resolving carcinogenesis and for identifying mechanisms of therapy resistance. While bulk sequencing can provide a broad view on tumoral complexity/heterogeneity of a patient, single-cell analysis remains essential to identify rare subclones that might drive chemotherapy resistance. In this study, we performed an integrated analysis of bulk and single-cell DNA sequencing data of core-binding factor acute myeloid leukemia patients, defined by the presence of a RUNX1::RUNX1T1 or CBFB::MYH11 fusion gene. By single-cell sequencing, we inferred tumor phylogenies for 8 patients at diagnosis including patient-specific somatic variants, somatic copy-number alterations and fusion genes, and studied clonal evolution under the pressure of chemotherapy for 3 patients. As a result, we developed an approach to reliably integrate subclonal somatic copy number alterations into phylogenetic trees and clonal evolution analysis, obtaining unprecedented resolution of intra-tumor heterogeneity in CBF AML. We were able to show that the fusion gene is among the earliest events of leukemogenesis at single-cell level. We identified remaining tumor clones in 6 patients with complete remission samples indicating incomplete eradication of the tumor clones. Here, we show that identifying the order of mutation acquisition can provide valuable insights into evolutionary history, offering a framework to improve drug selection in the era of targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f79dc8d9b0526060e06d2b2413c27f6af3f0f4c7" target='_blank'>
              Resolving intra-tumor heterogeneity and clonal evolution of core-binding factor acute myeloid leukemia patients with single-cell resolution
              </a>
            </td>
          <td>
            Raphael Hablesreiter, Paulina M. Strzelecka, Klara Kopp, Natalia Estrada, A. Dolnik, Marlon Tilgner, Coral Fustero-Torre, Felicitas R. Thol, F. Heidel, M. Heuser, L. Haghverdi, Lars Bullinger, F. Christen, Frederik Damm
          </td>
          <td>2025-10-28</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="


 Over 75% of PTCL-NOS patients fail to achieve long-term control with current therapies, highlighting the need for novel therapeutic targets. Genetic and non-genetic drivers in cancer cells require external cues from other cancer cells and from the lymphoma microenvironment (LME), which enable cancer cell adaptation to metabolic stress, immune mediated stress and treatment exposure. The resulting signaling crosstalk propels the co-evolution of cancer and LME cells with emergence of diverse cancer-supporting cell states. Identifying these and their maintenance mechanisms may therefore reveal novel therapeutic targets.
 We characterized the LME of PTCL using transcriptional signature deconvolution and multiparametric imaging in 845 patient samples. We identified four biologically and clinically distinct categories named Depleted (DP), B-cell rich (BR), Mesenchymal (MS) and Inflammatory (IN) based on the proportion of different LME cells. RNA-seq analysis showed that SMA+ cancer associated fibroblasts (CAF) are present in DP, MS and IN LMEs each with distinct phenotypes. To functionally analyze the contribution of CAFs to PTCL maintenance, we established multiple patient-derived tumor xenografts (PDXs) that retain the same LME categories of primary tumors. We then tested lymphoma cell survival in co-cultures of CAFs and PTCL cells from matched (e.g., IN-CAF + IN-PTCL) and unmatched PDXs (e.g., IN-CAF + DP-PTCL). Matched co-cultures sustained lymphoma cell proliferation to a higher extent than unmatched, independently of LME subtype. This suggests that CAF are distinctly reprogrammed within each LME.
 We hypothesize that pro-tumoral CAF phenotypes require adaptive gene expression in coordination with lymphoma cells' phenotypes. RNA-Seq of matched IN-CAF co-cultured with IN-PTCL revealed activation of pathways like “Transcription”, “DNA replication” and “Translation” including the upregulation of CDK7. Although CDK7 plays a minor role in cell cycle progression, its phosphorylation of the RNA polymerase II as part of the TFIIH complex is critical in activating and coordinating “cell state”-defining transcription.
 To assess CDK7's role in establishing lymphoma-supportive CAF states, IN-PTCL PDX mice were treated with the covalent selective CDK7 inhibitor YKL-5-124 for 12 h followed by transcriptomic and proteomic profiling of IN-PTCL and IN-CAFs. In IN-CAFs, CDK7 inhibition altered gene and protein expression of several collagens and cytokines, including COL1A1, COL1A2, MMP8, and CXCL12. Ligand-receptor analysis identified CXCL12-CXCR4 as the key CDK7-dependent interaction between IN-CAFs and IN-PTCL cells. IN-PTCL cells showed activation of DNA damage repair pathways, including upregulation of ALYREF and EIF4E, likely as compensatory mechanisms for decreased CDK7-dependent transcription. Since these may represent acquired therapeutic vulnerabilities, we conducted viability screenings in co-cultures of IN-PTCL and IN-CAF isolated from YKL-5-124 (and vehicle) tretated PDX mice. Cells were subsequently exposed for 48 h, individually and in co-cultures, to a drug library containing 40 clinical-phase compounds. This approach led to the identification of 14 compounds with synergistic anti-lymphoma effects. Among these, ten drugs including the XPO1 inhibitor Selinexor, the HSP90 inhibitor PU-H71 and the translation inhibitor Omacetaxine exhibited enhanced anti-lymphoma activity specifically in the co-culture setting, whereas four compounds showed reduced efficacy. This was not seen in mismatched co-cultures and indicates a potential protective role of CAFs and, importantly, that this mechanism is CDK7 dependent.
 To address the limitations of PDX models established in immunodeficient mice with limited ability to capture the immune-mediated effects of drug treatments, we developed an IN-PTCL syngeneic model by intrasplenic implantation of the murine T-cell lymphoma cell line EL4. Treatment with YKL-5-124 significantly reduced the tumor burden compared to vehicles. Cytokine profiling (immunoblotting array) and ELISA of culture medium of YKL-5-124 treated EL4-CAFs showed reduced levels of pro-inflammatory cytokines such as CXCL12, IL6, CCL2, ICAM-1, and CXCL1 indicating that CDK7 is necessary in establishing pro-lymphomagenic IN-CAF phenotypes.
 In conclusion, CDK7 activity is required to sustain specific pro-tumoral crosstalk interactions of matched CAFs and PTCL cells, which can be exploited therapeutically.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/386e7a67f6fb5e2e6ca873d1aaf4d401efdcd89c" target='_blank'>
              CDK7 enables adaptive gene transcription required by stromal cells to support distinct T-cell lymphoma phenotypes and represents a novel microenvironment-directed therapeutic target
              </a>
            </td>
          <td>
            K. Kouidri, D. Fiore, M. Revuelta, N. Di Siervi, Federico Ruiz Moreno, Subhashini Joshi, C. Kayembe, Abigail Taylor, G. Inghirami, L. Cerchietti, N. Zamponi
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="


 Acute myeloid leukemia (AML) is an aggressive and biologically heterogeneous hematologic malignancy. It is characterized by the clonal expansion of aberrant myeloid progenitor cells and predominantly affects older adults. As many elderly patients are not suitable for intensive chemotherapy or allogeneic hematopoietic stem cell transplantation, the five-year overall survival rate remains below 10%.
 The emergence of venetoclax, a novel, orally bioavailable selective BCL-2 inhibitor, provides a promising therapeutic option for elderly AML patient population. However, resistance to venetoclax develops in approximately 30% of patients, posing a major clinical challenge. Although accumulating evidence implies metabolic reprogramming in venetoclax resistance, effective strategies to reverse this process remain elusive.
 To recapitulate the stepwise acquisition of venetoclax resistance, we first generated MOLM-13 sublines exhibiting intermediate and complete resistance. Whole-exome sequencing of the derived lines identified two BCL-2 mutations. Interestingly, the progressively increasing variant allele frequencies correlate with the resistance severity, implicate these mutations as potential molecular drivers of venetoclax resistance. Remarkably, a relapsed AML patient after allogeneic HSC transplantation, went under venetoclax treatment for the relapse, became venetoclax-resistant while developed the same two BCL-2 mutations in patient serial leukemic cell samples, underscoring the clinical relevance of these findings.
 To elucidate the molecular alterations associated with venetoclax resistance, we firstly performed transcriptomic and proteomic profiling on the resistant MOLM-13 sublines. These analyses revealed a marked upregulation of lipid metabolism pathways in resistant cells compared to wild-type. This finding prompted us to further investigate whether metabolic reprogramming functionally contributes to resistance. To this end, we conducted a metabolism-focused CRISPR-Cas9 screen in wild-type MOLM-13 cells under venetoclax pressure. The screen identified 175 and 191 negatively selected genes at day 8 and 16, respectively, with 30 genes overlapping at both time points. Pathway enrichment analysis highlighted lipid metabolism as a key pathway, with PTEN prominently enriched, suggesting a potential role in mediating venetoclax resistance via metabolic regulation.
 We next sought to validate the functional role of PTEN in modulating venetoclax response. In vitro, genetic ablation of PTEN using CRISPR/Cas9 or shRNA significantly increased venetoclax-induced apoptosis in both parental and resistant AML cell lines. Pharmacological inhibition of PTEN with small-molecule inhibitors (SF1670 or bpV(HOpic)) produced similar sensitizing effects. Importantly, these findings were corroborated in primary AML samples, where PTEN suppression enhanced venetoclax responsiveness. In vivo, PTEN inhibition in xenograft models led to increased apoptotic response upon venetoclax treatment, further confirming its critical regulatory role in venetoclax response.
 Mechanistically, PTEN loss led to pronounced lipid droplet accumulation, as confirmed by MS/MS-based lipidomic profiling showing elevated fatty acid levels. 13C-labeled palmitate tracing showed reduced incorporation of fatty acids into the TCA cycle, indicating impaired β-oxidation. Concomitantly, upregulation of FASN and SREBP suggested increased lipid synthesis. This dual effect of reduced lipid degradation and increased synthesis resulted in intracellular lipid accumulation, which caused mitochondrial damage, as evidenced by elevated ROS levels. This mitochondrial stress may contribute to reduced resistance to venetoclax by promoting apoptosis, providing a mechanistic link between PTEN loss, lipid metabolism reprogramming, and enhanced drug sensitivity.
 In conclusion, we identify PTEN as a key lipid metabolic regulator of venetoclax sensitivity in AML. By linking lipid metabolic reprogramming to apoptotic vulnerability, we uncover a mechanistic basis for venetoclax sensitization. Our findings highlight the potential of targeting lipid metabolic pathways as a strategy to overcome venetoclax resistance in AML, particularly in the context of BCL-2 mutations.
 MT.C, LZ.C and YJ.J contributed equally to this work.
 Corresponding authors: WP.Y, EL.J and YJ.C.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c622b6cf0575b845ded4dfa8ce291137a9afa72e" target='_blank'>
              PTEN loss reprograms lipid metabolism to modulate venetoclax resistance in acute myeloid leukemia
              </a>
            </td>
          <td>
            Mutian Cao, Lizhen Cong, Yujiao Jia, Yifei He, Mingyue Hao, Yuanyi Ling, Wanting Li, Yajing Chu, E. Jiang, Weiping Yuan
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="


 Acute myeloid leukemia (AML) is driven by diverse genetic alterations in hematopoietic precursor cells and has a high rate of relapse. Among the most common changes are mutations in NPM1, encoding nucleophosmin. Recent work suggests that mutant NPM1 sustains leukemogenic transcription of HOXA/B genes and MEIS1 by coordinating with RNA Polymerase II and the MLL/menin complex as well as by inhibiting histone deacetylases. It is hence critical to better understand the epigenetic mechanisms governing leukemogenesis and treatment resistance. Here, we sought to comprehensively characterize the gene regulatory landscape of NPM1-mutantAML at diagnosis and relapse by analyzing the chromatin accessibility, gene expression, and surface protein abundance of AML cells at single-cell resolution – together with NPM1 mutational status.
 We developed GoT-DOGMA-seq, a single-cell method integrating Genotyping of Transcriptomes (Nam et al., 2019)for NPM1 genotyping with DOGMA-seq (Mimitou et al., 2021), which captures chromatin accessibility, gene expression, and surface protein abundance information from the same individual cells. We applied this method to paired bone marrow mononuclear cells (BMMCs) from three patients at diagnosis of AML (n = 16,793 cells) and after relapse following chemotherapy or venetoclax (n = 22,191 cells) and compared to BMMCs from healthy controls (n = 33,375 cells across 7 individuals).
 Malignant cells, identified through NPM1 genotyping, immunophenotyping, copy number alterations, and assessment of AML gene expression signatures, exhibited pronounced heterogeneity at diagnosis and relapse, reflecting varied myeloid cell types. Cell states characterized by chromatin accessibility and gene expression were correlated, but there were distinctions. For example, HSC/MPP-like malignant cells subdivided into states related to differential stemness, myeloid chromatin priming, and transcription factor (TF) activity, as well as copy number alterations (e.g., chr17p loss), inflammatory signaling (e.g., TNF-alpha and IL-1 signaling), and mitochondrial energetics (e.g., oxidative phosphorylation).
 Overall, chromatin accessibility data revealed 167,959 candidatecis-regulatory elements (cCREs). These cCREs covered 88% of ENCODE elements, suggesting consistency with the database as well as recognition of novel cCREs. 98,698 cCREs (59%) were uniquely accessible in HSC/MPP-like malignant cells and not in control cells. Nearly 51% of these elements were located in introns, 27% were in distal intergenic loci, ~11% were in promoters, ~7% were in exons, and the remainder were in untranslated regions or other downstream loci. We hypothesized that these identified cCREs help to regulate myeloid leukemogenesis or pathways of treatment resistance.
 Of the leukemia-associated cCREs, diagnosis-specific cCREs (13%; n = 12,957) were enriched for motifs of transcription factor (TF) regulators of normal and abnormal myeloid differentiation, including HOXA/B, MEIS1, as well as CEBP, RUNX, and AP-1 TFs. Chemotherapy relapse-specific cCREs (12%; n = 11,980) were enriched for IRF, STAT1, ZEB1, and SNAI1 motifs, and linked to the expression of genes involved in interferon signaling, altogether suggesting a shift toward inflammatory, resistance, and self-renewal pathways in relapse. cCREs significantly more highly accessible in venetoclax relapse cells compared to controls were linked to the expression of genes in the NFkB, RAS-RAF-MEK-ERK, and PI3K-AKT-mTOR signaling pathways, including MCL-1, which are known to mediate acquired resistance to venetoclax.
 We have analyzed NPM1-mutantAML samples at single-cell resolution and shown that AML cells at diagnosis and relapse are highly heterogeneous both epigenetically and transcriptionally. We have demonstrated that integrating chromatin accessibility and gene expression data enables more precise annotation of malignant cell states. We have also identified cCREs that may help to coordinate gene expression programs associated with leukemogenesis and treatment resistance, highlighting potential regulatory vulnerabilities.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e038464d40880886ad86d66843aa02811f821f96" target='_blank'>
              Single-cell multiomics reveals candidate cis-regulatory elements in NPM1-mutant AML
              </a>
            </td>
          <td>
            Elliot O. Eton, S. Ganesan, JangKeun Kim, Kai Beattie, D. Barisic, Christopher Mason, D. Landau
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e643a6feb31e1193c4fe51251e45e7c7b63296ed" target='_blank'>
              Cross-Species Transcriptomic Integration Reveals a Conserved, MIRO1-Mediated Macrophage-to-T Cell Signaling Axis Driving Immunosuppression in Glioma
              </a>
            </td>
          <td>
            Zehui Du, Menghan Li, Brandon Bergsneider, Andy P. Tsai, Kwang Bog Cho, Lily H Kim, John Choi, Gordon Li, T. Wyss-Coray, Michael Lim, Xinnan Wang
          </td>
          <td>2025-11-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Cutaneous squamous cell carcinoma (cSCC) is the second most common form of non-melanoma skin cancer (NMSC), with a steadily increasing global incidence, especially in populations with prolonged ultraviolet (UV) exposure. Advanced or metastatic cSCC carries a poor prognosis, while the application of targeted therapies and immunotherapies remains in the exploratory stage, due to limited understanding of the underlying mechanisms of cSCC occurrence and progression. Epigenetic dysregulation plays important roles in the progression of cSCC. However, how multi-dimensional regulatory networks reshape the epigenetic landscape and contribute to dysregulated gene expression in cSCC remains unclear. In this study, we performed parallel RNA m6A sequencing, 850K DNA methylation arrays, whole transcriptome sequencing, and ATAC-seq chromatin accessibility profiling on samples from normal skin, actinic keratosis (AK), and cSCC. We analyzed the regulatory networks and pathways of epigenetic modifications on gene expression. We further explored the crosstalk of epigenetic regulatory networks by correlation analysis. By integrating single-cell RNA-seq data, we identified epigenetically upregulated candidate genes and confirmed their expression and functions with experimental methods. Our integrated multi-omics analysis provides a comprehensive and dynamic epigenetic map of cSCC progression. Further analysis revealed that DNA methylation and m6A modification jointly regulate gene expression through independent and synergistic ways. The identified epigenetically upregulated candidate genes IDO1, IFI6, and OAS2 were validated to be overexpressed in cSCC tissues and cell lines, and functional assays confirmed their potential key roles in regulating the processes of cell proliferation, migration and invasion in cSCC. By integrating multi-omics data, this study systematically highlights the multi-layered epigenetic alterations and regulatory mechanisms involved in cSCC development. This multi-stage, multi-omics, and multi-resolution integrated analysis provides a theoretical basis and new insights for future personalized treatment strategies for cSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdeb7e8c78575b1d7e37855326ebd04eb68da2c1" target='_blank'>
              Multi-omics analysis reveals the crosstalk of epigenetic regulatory networks in cutaneous squamous cell carcinoma progression
              </a>
            </td>
          <td>
            Ya-Zhou Sun, Dan-Dan Zou, Xin-Jie Li, Xiao-Li Wang, Jia-Hao Feng, Jia-Ying Liu, Ke He, Yu-Tong He, Xian-Xin Lai, Xin Li, Xuan Jiang, Tian-Shun Gao, Li He
          </td>
          <td>2025-11-13</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Leukemia evolution is driven by a complex interplay between somatic mutations and epigenetic changes. However, our understanding of these processes is limited by a lack of scalable and cost-effective methods to co-assay genetic and epigenetic changes in single cells. Here, we present scGEM-seq (single-cell Genotyping and Methylome sequencing), a flexible multi-omics platform that permits simultaneous characterization of genetic variants and DNA methylation profiles from single cells. scGEM-seq combines three core innovations that can be accomplished at ~3% the cost ($0.03 per cell) of competing technologies (>$1 per cell in snmC-seq3 and SciMETv2): (1) single-cell capture with semi-permeable capsules (Atrandi Biosciences), enabling multi-step enzymatic reactions to be performed efficiently with minimal loss of DNA; (2) a novel split-pool strategy (HexaDecimal barcoding) that efficiently generates ~1 million unique cell barcodes that are compatible with bisulfite-like enzymatic conversions; and (3) flexible library preparation that can be adapted for genome-wide or targeted sequencing.
 In cell-mixing studies, we show that scGEM-seq accurately resolves cell populations based on DNA methylation profiles and genetics. At modest sequencing depth (0.001X per cell for whole-genome DNA libraries, 15734.4±12123.1 CpGs per cell for methylation libraries), scGEM-seq resolved mouse and human cell lines at expected ratios (1:1:1 ratio of GM12878, K562 and YAC1 cells) with low doublet rates (~2.4% using species genome alignment). scGEM-seq also resolved human cancer cell lines of distinct lineages (1:1:1 ratio of SW480, A375 and OCIAML3 cells) according to methylation and genetics (cell-specific SNVs and copy-number changes) with modest sequencing depth (0.002X per cell for whole-genome DNA libraries, 20242.6±12808.2 CpGs per cell for methylation libraries) and low doublet rates (2.88% using cell-specific SNVs).
 To enhance genotyping efficiency, we also combined scGEM-seq with two different target enrichment strategies: hybrid capture (Pan-cancer hybridization panel, IDT; n=127 genes) and amplicon sequencing. In mixed human cancer cell lines, scGEM-seq achieved consistent single-cell genotyping for known cancer genes and hotspot mutations, including a mean of 13.23±5.30% of cells for hybrid capture and >98% of cells for amplicon sequencing. To determine the sensitivity of scGEM-seq for genotyping of rare cells, we prepared a cell mixture containing varying proportions of human cancer cell lines (GM12878: 92.85%, SW480: 5%, A375: 1.5%, OCIAML3: 0.5%, K562, 0.15%). scGEM-seq coupled with amplicon sequencing for known cell-specific variants recovered the expected cell proportions down to ~0.1-0.2%, representing fewer than 2 malignant cells per 1000 (SW480: 302/6294 cells, 4.80%; A375: 88/6294 cells, 1.40%; OCIAML3: 30/6294 cells, 0.48%; K562: 11/6294 cells, 0.17%).
 Finally, we applied scGEM-seq to primary bone marrow samples from 4 patients with myeloid neoplasms (MDS and AML) representing a range of blast counts (47%, 52%, 10% and 16%). Here, scGEM-seq revealed distinct populations of normal, premalignant, and malignant cells, including subclonal leukemia cells with KIT D816V (47/777 cells, 6.05%) and T417_D419delinsL (72/700 cells, 10.29%) mutations in a patient with inv(16) AML, and clonal (premalignant) hematopoietic cells harboring a DNMT3A S770L mutation (251/558 cells, 44.98%) and subclonal malignant blasts with distinct KRAS G12C (92/385 cells, 23.9%) and G12D (4/381 cells, 1.05%) mutations.
 In conclusion, scGEM-seq provides a flexible, cost-effective, and high-resolution method for elucidating the complex interplay between epigenetic and genetic cell states in cancer cells, including within rare cell populations. We envision multiple applications for scGEM-seq for the study of leukemia and other blood cancers, including to characterize (epi)clonal heterogeneity and dynamics, identify minimal residual disease, and capture rare stem/progenitor populations or drug-resistant clones that emerge post therapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42da31bde36d790d58830d2a574d2fd507a69d8f" target='_blank'>
              Single-cell genotyping and methylome sequencing (scGEM-seq) resolves cell-states and clonal hierarchies in leukemia
              </a>
            </td>
          <td>
            Kyung Lock Kim, Maria R. Capilla-Guerra, Evan Chen, Andrew Lane, Gabriel Griffin
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Targeted therapy has revolutionized the treatment landscape of hepatocellular carcinoma (HCC), offering unprecedented hope to patients. However, despite its promise, significant challenges arise in the form of drug resistance. Only a fraction of HCC patients respond to targeted therapy, and even those who respond often develop resistance over time. Sorafenib and lenvatinib, the sole first-line targeted therapeutic drugs for HCC, face severe clinical limitations due to drug resistance. Understanding the mechanisms underlying sorafenib/lenvatinib resistance is crucial for maximizing treatment efficacy. Recent studies have highlighted the pivotal role of epigenetic regulation in drug resistance. Cancer is recognized as both a genetic and epigenetic disease, with epigenetic factors influencing various aspects of tumor cell biology, especially drug resistance. This review systematically summarizes the mechanisms of epigenetic-mediated sorafenib/lenvatinib resistance, encompassing non-coding RNA (ncRNA) regulation, DNA methylation, RNA methylation, and histone modification. These abnormal epigenetic processes typically influence biological activities, including escaped programmed cell death, tumor metabolic reprogramming, formation and maintenance of drug-resistant cells, uncontrolled cell proliferation signaling pathways, and abnormal transport processes, ultimately culminating in profound drug resistance. By comprehensively summarizing the latest discoveries in epigenetic regulation mechanisms, this review highlights potential strategies to overcome drug resistance, paving the way for future advancements in HCC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c8f7f2274db91d56455b33e92f98f5f8697e415" target='_blank'>
              Resistance of first-line targeted drugs in hepatocellular carcinoma: the epigenetic regulation mechanisms
              </a>
            </td>
          <td>
            Ding Qi, Yongqing Qin, Haidong Zhu, Yong Li, Shisong Han
          </td>
          <td>2025-11-26</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="


 Somatic evolution drives cancer progression and therapy resistance. Underlying phenotypic progressions is the development of marked clonal heterogeneity in most cancers. However, deciphering the downstream molecular effects of clonal heterogeneity in primary cancer has been limited due to the inability to isolate subclones by standard methods. While single-cell RNA-sequencing (scRNA-seq) provides high-resolution mapping of heterogeneous cell states, it cannot link transcriptional programs to cancer-driving mutations in these individual cells. We and others developed methods to detect genotypes and their transcriptional outputs in single cells (e.g., Genotyping of Transcriptomes; Nature, 2019). However, these approaches have technical limitations to broadly profile numerous mutations, which is nonetheless required for clonally complex neoplasms. The existing approaches are also restricted to fresh or frozen samples, limiting the use of the widely available pathology formalin fixed paraffin embedded (FFPE) tissues.
 To address these challenges, we developed Genotyping of Transcriptomes for Multiple Targets and Sample Types, GoT-Multi, a next generation single-cell multi-omics method compatible with FFPE tissues that integrates multiplexed genotyping with transcriptomic profiling. GoT-Multi adapts a probe-based scRNA-seq method (Flex Fixed RNA Profiling, 10x Genomics) by including custom mutation-specific probes along with the standard transcriptional probes. Furthermore, we developed an ensemble-based machine learning pipeline, GoT-Multi-ML, that optimizes the genotyping calls by denoising technical artifacts. GoT-Multi-ML leverages multiple machine learning models to learn the non-linear patterns of the genotyping data features (e.g., reads per genotyping transcript, GC content of probes), agnostic to the whole transcriptomic data. We validated GoT-Multi via a cell line-mixing experiment using SK-BR-3 and MCF-7 cell lines. GoT-Multi detected the targeted mutations (HIST1H1C A24T and S100A10 A77V) in the expected cell line (MCF-7) with an average genotyping rate of 72% and accuracy of 97%.To define the impact of clonal evolution on phenotypic progression, we focused on a prototype of cancer evolution, the progression of chronic lymphocytic leukemia (CLL) to therapy-resistant large B-cell lymphoma (LBCL) called Richter Transformation (RT). We applied GoT-Multi to a unique cohort of primary nodal cryopreserved samples containing both CLL and LBCL components (n = 5). We targeted 18 mutations (1-6 targets per sample) with a 98% median accuracy. We observed heterogeneous cancer cell states, including proliferating, stress response and inflammatory (n = 51,465 cells). LBCL was enriched in proliferating and cyclin D2-elevated states, whereas CLL was associated with a stress response signature. GoT-Multi enabled us to reconstruct the clonal architectures, up to 4 distinct subclones per lymphoma, and their associated cell states. Differential gene expression between mutated and wildtype cells revealed that genetically heterogenous subclones, including those with SRRM2 and JUNB mutations and therapy resistance-associated mutations in PLCG2 and BTK, displayed enrichment in inflammatory cell states, upregulating TNF and interferon signaling genes while suppressing cell cycle-related genes. Conversely, other subclones with mutations in, e.g., IRF8, POU2F2, PRKDC were associated with MYC activation and/or enhanced proliferation. These findings suggested that heterogeneous genotypes may converge on similar downstream transcriptional cell states. In addition, TNF itself was overexpressed in the proliferative subclones, suggesting that TNF production from these subclones may enhance inflammatory signaling of the TNF-responsive subclones, indicative of crosstalk between subclones. Finally, we demonstrated the applicability of GoT-Multi to pathology archived FFPE tissues. We profiled 9 mutations with an accuracy of 98% in an FFPE RT sample (n = 2,140 nuclei), allowing us to detect 5 subclones and their associated cell states. In summary, co-mapping of clonal and cell state heterogeneity at single cell resolution through GoT-Multi suggested that heterogeneous subclonal genotypes may converge on similar downstream oncogenic pathways to enhance overall tumor fitness. We envision that the broad applicability of GoT-Multi may help uncover the molecular underpinnings of cancer progression and therapy-resistance across oncology.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/954b694c778f5e54689dba8e5834b825525f28c2" target='_blank'>
              Co-mapping clonal and transcriptional heterogeneity in somatic evolution via got-multi.
              </a>
            </td>
          <td>
            Minwoo Pak, M. Saurty-Seerunghen, Kellie Wise, Tsega-Ab Abera, Chhiring Lama, Neelang Parghi, Xiaotian Sun, Qi Gao, Liming Bao, Mikhail Roshal, John N Allan, Richard Furman, Luciano G. Martelotto, Anna S. Nam
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM), the most aggressive and prevalent type of brain cancer, presents a formidable therapeutic challenge due to its high degree of drug resistance. Despite advances in chemotherapy, targeted therapy and immunotherapy, patient outcomes remain dismal, with a 5-year survival rate of only 7%. Resistance mechanisms are multifactorial, including the restrictive nature of the blood-brain barrier (BBB), tumor heterogeneity and adaptive responses within the tumor microenvironment (TME). The BBB limits drug delivery, while efflux transporters further reduce therapeutic efficacy. Additionally, the vast molecular genetic and cellular heterogeneity of a glioblastoma enables the survival of resistant subpopulations, such as glioblastoma stem cells, that evade treatment. Chemotherapy resistance in GBM, particularly to temozolomide, is driven by factors such as O6-methylguanine-DNA methyltransferase upregulation, defective mismatch repair, hypoxia-induced gene expression and activation of several signaling pathways, such as the NF-κB, Hippo and Wnt pathways. Targeted therapies have shown limited success due to activation of compensatory pathways and tumor plasticity, while immunotherapeutic approaches are hindered by an immunosuppressive TME. Recently identified resistance mechanisms, including exosomal transfer of noncoding RNAs and metabolic reprogramming, further complicate treatment. Future directions should emphasize overcoming these challenges through combination therapies, enhanced drug delivery systems and precision medicine approaches. Emerging strategies include targeting persister cells, leveraging metabolic vulnerabilities and integrating AI-driven drug discovery approaches and nanotechnology. Robust patient stratification and biomarker-driven interventions are critical for tailoring therapies and improving outcomes. The present review highlights the urgent need for innovative, multidisciplinary approaches to address the complexity of GBM resistance and advance therapeutic strategies for this lethal disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7873b8c2ed7832370f4bef19a80cb17ac426667" target='_blank'>
              Drug resistance in glioblastoma: Challenges, mechanisms and therapeutic strategies (Review)
              </a>
            </td>
          <td>
            Yash Durga, Raju Pusapati
          </td>
          <td>2025-11-06</td>
          <td>Molecular and Clinical Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Introduction: Acute Myeloid Leukemia (AML) is an aggressive hematologic malignancy characterized by the clonal expansion of undifferentiated myeloid precursors in the bone marrow and peripheral blood. This uncontrolled proliferation is primarily driven by a range of genetic mutations that disrupt normal hematopoiesis and impair the function of healthy leukocytes, erythrocytes, and platelets. Among emerging regulatory mechanisms, N6-methyladenosine (m6A) RNA modification plays a critical role in post-transcriptional gene regulation and cancer progression. The m6A methyltransferase complex (MAC), which catalyzes the methylation of adenosine residues in mRNA, is primarily composed of METTL3 and METTL14. METTL3 serves as the catalytic subunit responsible for methyltransferase activity, while METTL14 provides structural support and enhances substrate recognition and binding. Together, this heterodimeric complex efficiently installs m6A marks on mRNA, thereby influencing RNA stability, translation, and splicing. Dysregulation of m6A modification has been implicated in the pathogenesis of multiple cancers, including AML. This study investigated the regulatory role of METTL3 in AML by reanalyzing publicly available RNA-seq datasets from AML MOLM13 cells treated with ZW27941, a potent small-molecule degrader of METTL3.
 Methods: Human AML MOLM13 cells were treated with either DMSO (control) or ZW27941 for 24 hours. Following treatment, total RNA was extracted, and RNA-seq was performed. Transcriptome data (GSE273361) were reanalyzed to identify differentially expressed genes (DEGs) using R packages (false discovery rate < 0.05 and |log₂ fold-change| > 0.2). Gene ontology (GO) enrichment analysis was conducted to identify biological processes significantly altered upon METTL3 degradation.
 Results: ZW27941 treatment induced substantial transcriptomic changes, with 534 genes upregulated and 641 genes downregulated. The suppressed genes included key drivers of AML progression (e.g., TREM1, S100A8/A9, TLR4, CD84, CXCR3, ITPK1, PECAM1, and ADGRE1), indicating pathways related to innate immune signaling, cytokine-mediated activation, and inflammatory cell adhesion. Downregulation of genes involved in lipid metabolism (e.g., PPARGC1A, ELOVL6, CYP27A1/B1/W1, ABCA1/2/9, and SLC-family transporters) suggests a disruption in fatty acid synthesis and cholesterol homeostasis, pathways known to support leukemic cell survival and drug resistance. Additionally, the downregulation of transcription factors and developmental regulators (e.g., RUNX1, TNR, SOX18, NR2F2, and FOXM1) indicates interference with lineage identity, cell proliferation, and AML subtype specification. Conversely, genes and pathways involved in apoptosis, DNA damage repair, cell cycle arrest, and immune activation (e.g., IL-2, IL-6, TNFα, NF-κB, and JAK-STAT3 signaling), were significantly upregulated in response to METTL3 degradation.
 Conclusion: Our study demonstrates that METTL3 plays a pivotal role in sustaining AML cell survival by regulating genes associated with inflammation, lipid metabolism, proliferation, and immune evasion. Pharmacological degradation of METTL3 by ZW27941 not only suppresses oncogenic transcriptional programs but also activates apoptotic and immune responses. These findings demonstrate METTL3 as a promising therapeutic target in AML and support further exploration of m6A modulators in anti-leukemic strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4ab99e6d63a6b8558aeee4c091c181cedae6e6c" target='_blank'>
              Targeting METTL3 disrupts oncogenic transcriptional programs and activates immune and apoptotic pathways in Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Harshita Ravi, Qian Wang
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Tumor heterogeneity is a fundamental property of leukemia and lymphoma that drives disease progression, relapse, and therapy resistance. For decades, bulk genomic and transcriptomic analyses have provided a foundational understanding of these malignancies but have obscured the cellular diversity within the tumor ecosystem. The advent of high-throughput single-cell technologies, particularly single-cell RNA sequencing (scRNA-seq), has revolutionized our capacity to deconvolute this complexity. This review synthesizes how single-cell analyses are reshaping our understanding of tumor heterogeneity in hematological cancers. We explore the role of intra-clonal genetic diversity in Acute Myeloid Leukemia (AML) and B-cell Acute Lymphoblastic Leukemia (B-ALL), the functional and transcriptional plasticity of subpopulations, and the dynamic interplay between malignant cells and the immune microenvironment in both leukemia and lymphoma. We further highlight how the tumor microenvironment in lymphomas fosters heterogeneity and immune evasion. Furthermore, the integration of single-cell data with spatial transcriptomics and computational models is beginning to predict evolutionary trajectories, offering a window into future clinical behaviors. Critically, we discuss how single-cell approaches have identified pre-existing or emergent resistant subclones under therapeutic pressure, providing a mechanistic basis for treatment failure. The clinical implications are profound, paving the way for single-cell diagnostics for minimal residual disease (MRD) monitoring and the identification of novel therapeutic targets. We conclude that single-cell multi-omics is an indispensable tool for unraveling the multifaceted nature of tumor heterogeneity, with the potential to propel the field toward more precise and effective therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0baf5679635e6fcbb86a03a048b12087069db02" target='_blank'>
              Tumor Heterogeneity in Leukemia and Lymphoma: Insights from Single-Cell Analyses
              </a>
            </td>
          <td>
            Pang Hou, Wen
          </td>
          <td>2025-11-21</td>
          <td>Hematological Disorders in the Single-Cell Era</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Circular single-stranded DNA (cssDNA), an emerging nucleic acid vector, exhibits significant clinical potential for treating genetic disorders, enabling gene editing, and advancing oncotherapy. Its unique attributes, including high stability, structural simplicity, conformational flexibility, and low molecular mass, establish it as a promising gene therapy tool. Our study reveals that cssDNA demonstrates superior expression efficiency over conventional plasmids across diverse tumor cell lines. Notably, cssDNA expression is enhanced under certain tumor microenvironment (TME) conditions (glucose deficiency, glutamine deficiency and hypoxia) compared to normal condition. Mechanistically, these TME conditions induce significant cell cycle perturbations, particularly pronounced G1 arrest. Intriguingly, transfected cssDNA expression peaks during the late G2/M phase, immediately preceding entry into the G1 phase. We further identify S-phase kinase-associated protein 2 (SKP2) as a critical regulator of cssDNA expression under TME conditions. SKP2 inhibition directly or indirectly notably enhances the expression levels of cssDNA. These findings confirm cssDNA’s advantages as a gene expression vector and how specific TME conditions modulate its expression via cell cycle and SKP2-dependent mechanisms in vitro. This work provides a scientific foundation for cssDNA-based cancer therapy and opens new avenues for future clinical translation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a904197f8868f962bae1c8795cb37bde58b3f61" target='_blank'>
              The tumor microenvironment enhances the expression of cssDNA by modulating cell cycle signaling pathways via SKP2
              </a>
            </td>
          <td>
            Dandan Shao, Jinghao Wang, Kexuan Zou, Yatao Chen, Pengfei Zhang, Jie Song
          </td>
          <td>2025-11-05</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2859fc19b9d23d91d42bdb280787178a4d820ac" target='_blank'>
              Rewiring the transcriptome: diagnostic and therapeutic implications of alternative splicing in solid cancers
              </a>
            </td>
          <td>
            Noura A. A. Ebrahim, T. Farghaly, Soliman M. A. Soliman
          </td>
          <td>2025-11-26</td>
          <td>Molecular Biology Reports</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Acquisition of resistance to anti-cancer therapies is a multistep process initiated by the survival of drug-tolerant persister cells. Accessibility of drug-tolerant persister cells in patients is limited, which has hindered understanding the mechanisms driving their emergence. Here, using multiple patient-derived models to isolate persister cells, we showed that these cells are transcriptionally plastic in vivo and return to a common treatment naïve-like state upon relapse, regardless of treatment. Hallmarks of the persister state in TNBC across treatment modalities included high expression of basal keratins together with activation of stress response and inflammation pathways. These hallmarks were also activated in HER2+ breast and lung cancer cells in response to targeted therapies. Analysis of gene regulatory networks identified AP-1, NF-κB and IRF/STAT as the key drivers of this hallmark persister state. Functionally, FOSL1, an AP-1 member, drove cells to the persister state by binding enhancers and reprogramming the transcriptome of cancer cells. On the contrary, cancer cells without FOSL1 had a decreased ability to reach the persister state. By defining hallmarks of TNBC persistence on multiple therapies, this study provides a resource to design effective combination therapeutic strategies that limit resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88dba0b22030a811fd28fd3f8f3324b7f9286705" target='_blank'>
              Characterization of Drug-Tolerant Persister Cells in Triple-Negative Breast Cancer Identifies a Shared Persistence Program across Treatments and Patients.
              </a>
            </td>
          <td>
            Léa Baudre, Gregoire Jouault, Pacôme Prompsy, Melissa Saichi, Sarah Gastineau, Christophe Huret, Laura Sourd, A. Dahmani, Elodie Montaudon, F. Dingli, D. Loew, Elisabetta Marangoni, J. Marsolier, C. Vallot
          </td>
          <td>2025-11-06</td>
          <td>Cancer research</td>
          <td>1</td>
          <td>54</td>
        </tr>

        <tr id="Leukemia and lymphoma represent a significant burden in Nigeria, characterized by distinct molecular profiles and treatment responses compared to global patterns. This uniqueness can be attributed to several factors including genetic diversity within the Nigerian population, environmental exposures specific to the region, and healthcare access challenges that affect early diagnosis and treatment adherence. The comprehensive understanding of these hematological malignancies has been revolutionized by the advent of single-cell technologies, which have uncovered unprecedented resolution into tumor heterogeneity, evolution, and microenvironmental interactions. Unlike traditional bulk sequencing methods that average signals across thousands of cells, single-cell approaches allow researchers to examine individual cells, revealing rare but critical subpopulations that drive disease progression and therapy resistance.
This review synthesizes findings from Nigerian studies integrated with global advances in single-cell research, highlighting how clonal architecture, cellular ecosystems, and molecular signatures contribute to pathogenesis and treatment outcomes. We explore how single-cell multi-omics approaches—including transcriptomics, genomics, and proteomics—have identified novel therapeutic targets and resistance mechanisms in leukemia and lymphoma. Additionally, we discuss the potential for implementing these technologies in Nigeria to advance precision medicine, considering both the opportunities and challenges specific to the Nigerian healthcare context. The integration of single-cell analyses into clinical practice promises to improve risk stratification, therapy selection, and overall management of these malignancies in resource-limited settings through more targeted and effective treatment approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca65a47e4dd6298d29b7f5a8f9777fdb3fa1474c" target='_blank'>
              Unraveling Clonal Architecture and Microenvironmental Complexity of Leukemia and Lymphoma Using Singlecell Multi-Omics in Nigeria
              </a>
            </td>
          <td>
            Leon Oluci Kochukgu
          </td>
          <td>2025-11-06</td>
          <td>Hematological Disorders in the Single-Cell Era</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) remains a formidable therapeutic challenge, with high relapse rates rooted not only in genetic heterogeneity, but also in dynamic and reversible epigenetic dysregulation that drive resistance to conventional and targeted small-molecule therapies. Although small-molecule inhibitors targeting epigenetic regulators have demonstrated preclinical efficacy and initial clinical activity, therapeutic gains are often limited by transient responses, intra- and inter-patient heterogeneity, and an incomplete understanding of predictive biomarkers. Besides, clinical outcomes have often been less robust than preclinical data suggested, partly due to the complexity of AMLs genetic and epigenetic heterogeneity, the limitations of model systems, and intrinsic resistance mechanisms. Combination approaches, such as pairing epigenetic agents with chemotherapy, immunotherapy, or other targeted drugs are under investigation. However, clinical efficacy remains inconsistent and requires a mechanistic rationale to avoid antagonism. To move forward, it is essential to delineate specific epigenetic dependencies in resistance-prone AML subtypes, design rational, biomarker-guided combination regimens, and enhance specificity and pharmacodynamic profiles of epigenetic inhibitors. Such an integrated strategy holds promise for elevating the effectiveness of epigenetic therapies in AML. Continued research is essential to refine epigenetic-based precision medicine and ultimately improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8e1118798f14d99fa1bf83fc376d07fc1df6c5c" target='_blank'>
              Epigenetic mechanisms of drug resistance in acute myeloid leukemia: advances in small-molecule targeted therapy
              </a>
            </td>
          <td>
            Jialing Tang, Xibo Ye, Zhangyang Xu, Tianjiao Min, Zherun Tian, Li Ma, Ying He, Beier Jiang
          </td>
          <td>2025-10-20</td>
          <td>Frontiers in Pharmacology</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Introduction Colon cancer is a common disease, treated with few chemotherapeutic agents with similar treatment sequencing despite its heterogeneity. A significant proportion of patients are diagnosed with metastasis, and resistance to antineoplastic drugs is associated with disease progression and therapeutic failure. It is known that the tumor microenvironment plays an essential role in cancer progression, contributing to processes that may be associated with therapeutic resistance mechanisms in colon cancer. In this study, we aim to identify a gene expression signature and its relationship with immune cell infiltration in colon cancer, contributing to the identification of potential resistance biomarkers. Methods An in silico study was conducted using RNA-seq data from The Cancer Genome Atlas Program (TCGA) samples, subdivided into two groups (treatment-resistant and non-resistant), taking into account the molecular subgroups (CMS1, CMS2, CMS3, and CMS4). The following algorithms were used: i. Limma was applied to identify differentially expressed genes; ii. WGCNA was applied to construct co-expression networks; iii. CIBERSORT was applied to estimate the proportion of infiltrating immune cells; and iv. TIMER was applied to explore the relationship between core genes and immune cell content. Results Twenty differentially expressed genes (DEGs) were found, with 18 related to the group considered resistant to oncologic treatment and presenting poorer overall survival. T CD4 memory resting cells and M0 and M2 macrophages were found in more significant proportions in the analyzed samples and more infiltrated in the tumor microenvironment, the higher the expression of some of these resistance DEGs. Additionally, these genes correlate with biological aspects of neuronal differentiation, axogenesis, and synaptic transmission. Conclusion The gene expression signature suggests the presence of differentially expressed synaptic membrane genes, which may be involved in neuronal pathways that influence the tumor microenvironment, potentially serving as future biomarkers. Furthermore, the presence of M0 and M2 macrophages and T CD4 memory resting cells suggests a potential interaction that may play a role in therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd3d04147fc5526a1e2f859a73c8cdc7c99f7a9f" target='_blank'>
              Gene expression profile in colon cancer therapeutic resistance and its relationship with the tumor microenvironment
              </a>
            </td>
          <td>
            P. Doria, G. Rocha, Vanessa Dybal Bertoni, Roberto de Souza Batista dos Santos, Mariana Araújo-Pereira, Clarissa A Gurgel
          </td>
          <td>2025-10-29</td>
          <td>Frontiers in Bioinformatics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Glioblastoma is characterized by highly invasive behavior that contributes to therapeutic failure and recurrence, yet the stability and regulatory mechanisms of the invasive phenotype remain poorly understood. We established a serial selection model in patient-derived glioblastoma stem-like cells to generate an isogenic subpopulation with stable invasive potential (GBinv⁺) and compared it to the non-invasive counterpart (GBnaïve). GBinv⁺ cells showed a ~10-fold increased invasion in transwell and 3D spheroid assays, a phenotype maintained across passages and independent of proliferation. Integrative RNA sequencing and DNA methylation profiling identified 325 differentially expressed genes and ~6,700 differentially methylated CpG sites, enriched in promoters and enhancers. Invasive cells upregulated a distinct secretome enriched in ECM-modulating factors, including COL7A1, ICAM1, and P4HA2. Transcription factor activity was inferred using ISMARA, revealing increased activity of stress-related regulators (ATF4/6, IRF7) and reduced activity of MYC, E2F, and MYCN. Notably, 24 invasion-associated TFs contained CpGs within their core motifs, suggesting methylation-sensitive regulation. Motif enrichment at differentially methylated sites highlighted E-box motifs bound by bHLH factors (MYC, MAX, MNT), with hypermethylated regions showing reduced co-activator marks, whereas hypomethylated enhancers were enriched for MYC/MAX/E2F1 occupancy. Public ChIP-seq data confirmed that methylation states correlate with chromatin accessibility and TF binding. Integrated analysis demonstrated that although the global correlation between methylation and expression was weak, specific genes such as TNC and FLNC exhibited coordinated methylation-expression changes, identifying a subset of epigenetically modulated invasion effectors. These results support a model in which glioblastoma invasion is sustained by epigenetically reprogrammed transcription factor networks rather than transient environmental cues. Future work will use CUT&Tag to map genome-wide methylation-sensitive TF binding, focusing on MYC/MAX/MLX complexes in patient-derived glioma samples.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e0f700028bbeda988f83eab61ca9826e85aa85a" target='_blank'>
              ANGI-07. EPIGENETIC MODULATION OF TRANSCRIPTION FACTOR BINDING DRIVES INVASIVE GLIOBLASTOMA PHENOTYPES
              </a>
            </td>
          <td>
            T. Buhlmann, S. Schreiner, C. Cialini, Vera Laub, P. Nazarov, Arnaud Muller, K. Plate, J. Steinbach, M. Mittelbronn, A. Hau
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50c38b6d1e04676ebe5c874a1af9317e2b5d3d46" target='_blank'>
              Single-cell Transcriptomic Profiling of Pancreatic Ductal Adenocarcinoma: Epithelial Reprogramming and Systemic Immune Exhaustion
              </a>
            </td>
          <td>
            Qinyu Xu
          </td>
          <td>2025-11-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Extrachromosomal circular DNA (eccDNA) plays critical roles in cancer, yet its landscape in acute myeloid leukemia (AML) remains unexplored. Methods We used CIRCLE-seq and RNA-seq to characterize eccDNA in 12 AML patients and 4 healthy controls. Results AML cells showed significantly increased eccDNA counts and gene involvement versus healthy controls, with distinct size peaks at 202 and 368 bp. eccDNAs localized non-randomly to chromosomes 1, 2, and 10–19, enriching near transcription start sites and regulatory regions. Functional analysis revealed activation of oncogenic pathways (e.g., MAPK, ErbB signaling) in AML-associated eccDNA. Integrative analysis identified 570 genes upregulated at both the eccDNA and mRNA levels, including myeloid leukemia-related genes (e.g., FLT3, RUNX1, and CD33) and oncogenes. Prognostic analysis showed that high expression of these genes correlated with poor outcomes in AML. Conclusions This study unveils the eccDNA landscape in AML by direct CIRCLE-seq, linking its accumulation to transcriptional dysregulation and leukemogenesis, and highlights eccDNA as a potential biomarker and therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1de89beb26c968490d1dcd9a8b5f72aeb3ffa8be" target='_blank'>
              Integrated CIRCLE-seq with RNA-seq to decipher the quantity, localization, and functional features of eccDNA in AML
              </a>
            </td>
          <td>
            Lijuan Gao, Qiongyu Lu, Hao Li, Changgeng Ruan, Zhao Zeng, Suning Chen
          </td>
          <td>2025-11-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 Background: Chronic myelomonocytic leukemia (CMML) is a clonal hematopoietic stem cell disorder that combines clinical and biological characteristics of myelodysplastic neoplasms and myeloproliferative neoplasms. Current treatment choices are limited by incomplete understanding of its transcriptional programs and clonal evolution. Clinical heterogeneity necessitates single-cell resolution to resolve subclonal diversity and identify actionable nodes. We applied single-cell transcriptome sequencing to delineate cellular and molecular heterogeneity and to uncover clonal evolution features that may inform actionable targets.
 Methods: Bone marrow aspirates from 5 newly diagnosed CMML patients and 4 healthy donors underwent single-cell transcriptome sequencing. Comprehensive analysis included dimensionality reduction and clustering, gene ontology (GO) and KEGG pathway enrichment of significantly different genes (p<0.05 and fold changes > 1.5), intercellular communication inference (CellChat), regulatory network reconstruction (SCENIC), gene set variation analysis (GSVA), consensus non-negative matrix factorization, and construction of explainable gene ontology fingerprints.
 Results: 1. We identified 13 distinct cellular clusters with significant compositional shifts in CMML versus healthy controls; clusters 2, 5, 7, 9, and 10–13 showed the most pronounced differences. We also identified marker genes for each of the 13 cell clusters, and the top 10 marker genes in each cluster were identified based on log fold change and adjusted P-value, including ALPL, MAL, KLRF1, SASH1, SLC2A5, CRISP3, XCL2, E2F7, PAX5, ITGB3, and P2RY6. 2. A total of 1,866 genes were differentially expressed(1,584 upregulated, 282 downregulated) in CMML patients compared to healthy volunteers. 3. GO enrichment analysis was performed separately for biological process (BP), molecular function (MF), and cellular component (CC). In BP, upregulated genes were significantly enriched for neutrophil degranulation, whereas downregulated genes were overrepresented in SRP-dependent cotranslational protein targeting to the membrane. In CC, upregulated genes localized predominantly to the cytosol, while downregulated genes were concentrated in ribosomal structures. In MF, upregulated genes were chiefly associated with protein binding, whereas downregulated genes were enriched for functions related to the structural constituent of the ribosome. 4. KEGG enrichment analysis revealed significant enrichment in pathways such as B cell receptor signaling, microRNAs in cancer, ribosomes, and antigen processing and presentation. 5. Protein interaction analysis using the STRING database generated a PPI network of the top 25 differentially expressed genes, highlighting a central hub that includes NAMPT, SOD2, CXCL8, LYN, LTF, VMP1, PDE4D, FNDC3B, DEFA3, and ACSL1.
 Conclusion: This single-cell transcriptome sequencing study delineated disease-specific transcriptional and clonal evolution features in CMML versus healthy controls, providing mechanistic insight into CMML pathogenesis and identifying candidate molecular targets for therapeutic development.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99d3add7a95a8334b2ea20e007ff803fc2eda72c" target='_blank'>
              Exploring molecular heterogeneity and clonal evolution in chronic myelomonocytic leukemia via single-cell transcriptome sequencing
              </a>
            </td>
          <td>
            Jie Zhou, Yuyuan Huang, Hao Wu, Bing Li, Shaoguang Li, Aibin Liang, Jianfei Fu
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Prostate cancer, ranking among the most prevalent malignancies in males worldwide, is undergoing a significant evolution in therapeutic paradigms from conventional approaches to precision medicine, with recent advances in targeted therapies offering novel strategic insights. This review delineates the molecular foundations of prostate carcinogenesis, elucidating pivotal domains including genetic mutations, hormonal regulation, tumor microenvironment dynamics, cell cycle dysregulation, epigenetic modifications, and tumor heterogeneity. Furthermore, we evaluate the clinical translation of targeted strategies such as AR signaling axis inhibition, PI3K/AKT/mTOR pathway modulation, DNA damage repair machinery exploitation, prostate-specific membrane antigen -directed interventions, and combinatorial immunotherapy. Concurrent challenges—AR-driven heterogeneity, adaptive drug resistance mechanisms, spliceosomal vulnerabilities, and scarcity of selective molecular targets—are critically analyzed. Notwithstanding these obstacles, targeted therapies exhibit considerable potential to enhance therapeutic efficacy while mitigating systemic toxicities, paving the way for more personalized and precision-oriented oncologic care. By underscoring the imperative to decode prostate cancer’s molecular architecture, this work outlines future research priorities and advances a robust scientific framework for innovation in therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6680208f5ea18e2a4fa84ced337f25d04b9c91c6" target='_blank'>
              Unravelling targeted therapy in prostate cancer: from molecular mechanisms to translational opportunities
              </a>
            </td>
          <td>
            Litong Wu, Junfeng Qiu, Zhi-ming Hong, Quan Wang, Qixin Li, Wenbin Zhou
          </td>
          <td>2025-11-14</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 The formation of cancer and responsiveness to therapy is governed by a constantly evolving balance between somatic variation and dynamic epigenetic regulation. This evolution has been recognized as a significant obstacle challenging the development of durable cancer therapies. Thus, targeting epigenetic reprogramming in human cancers is a key focus for the design of new drugs to potentially deliver robust and durable anti-tumor activity. Lysine acetyltransferases (KATs) are a promising class of epigenetic targets that operate within interchangeable multi-subunit complexes to regulate cell cycle, pluripotency, and lineage determination. Within this class, KAT6A and KAT7 have been recognized as mechanistically intertwined modulators of stem cell identity and lineage commitment through partially overlapping control of chromatin organization specified by histone H3 acetylation of lysine 23 (H3K23) and lysine 14 (H3K14). Multiple lines of orthogonal evidence point to a “paralog” genetic relationship between these enzymes to support cancer cell autonomous survival in lineage-enriched CCLE subsets. Moreover, recent preclinical findings in pediatric and adult leukemia indicate the need to inhibit both KAT6 and KAT7 to deliver durable anti-tumor responses. These observations prompted evaluation of concomitant inhibition of KAT6 and KAT7 as a therapeutic approach in biomarker-defined solid cancers. Here, we report an equipotent KAT6/KAT7 dual inhibitor with high selectivity versus the structurally similar KAT family members KAT5/KAT8. We observed synergistic impact on cell viability and pharmacodynamic (PD) modulation of H3K23/H3K14 acetylation in biomarker positive breast, lung, and ovarian cancers with dual KAT6/7 inhibition versus selective inhibition of KAT6 or KAT7. Quantitative transcriptomic and epigenomic profiling of cancer cell responses to dual KAT6/7i revealed global inhibition of H3K23 and H3K14 acetylation as well as local suppression of H3K27ac-dependent enhancer elements at gene promoters controlling lineage specification, stemness, and WNT signaling. In vitro single cell RNA sequencing and protein expression analysis of ER+ PDX derived breast cancer cells indicated that dual KAT6/7 inhibition can overcome KAT6i associated adaptive drug resistance, downregulating drug tolerant persister cell states and metabolic adaptation. In vivo efficacy studies in biomarker positive breast and lung CDX models confirmed enhanced tumor growth inhibition and PD modulation for dual KAT6/7 versus selective KAT6 inhibition. Notably, dual KAT6/7 inhibition promoted durable and stable anti-tumor responses in multiple standard-of-care resistant and ESR1-mutant breast cancer PDXs that could not be achieved with KAT6 inhibition alone. In vivo bulk transcriptomics analysis of dual KAT6/7 treated PDX tumor samples highlighted disruption of lineage identity and metabolic rewiring. These observations indicate equipotent inhibition of KAT7 and KAT6A/B has the potential to deliver deep and durable anti-tumor activity in biomarker-selected patient populations.



 Manav Gupta, Scott R. Tyler, Nour Ghaddar, Pranav Gupta, Steven L. Spivak, Katelyn N. Lukas, Carl Schultz, Anjali Bisaria, Zabrisky Roland, Kelly S. Trego, Mason Appel, Parker Y. Jameson, Jacob Burch-Konda, Brittany Cruzan, Angelica Gonzalez-Sanchez, Monah Khalaj, Jay Prakash. Jain, Michael E . Dalziel, Julie Deichert, Sunjay Sethi, Ivan G. Shabalin, Rebeca Choy, Diana M. Munoz, Christian R. Frey, Peter Teriete, Josh Taygerly, Yuchen Bai, Richard Zang, Xin Linghu, Claire L. Neilan, Paul A. Barsanti, Michael A . White, Brian T. Jones. Dual KAT6/7 inhibition delivers robust monotherapy anti-tumor activity in biomarker selected indications and disrupts emergence of drug-tolerant persister cell populations [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C022.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7cc963d6c0afb39c048f1457ab9b594f9c49864" target='_blank'>
              Abstract C022: Dual KAT6/7 inhibition delivers robust monotherapy anti-tumor activity in biomarker selected indications and disrupts emergence of drug-tolerant persister cell populations
              </a>
            </td>
          <td>
            Manav Gupta, Scott R. Tyler, N. Ghaddar, Pranav Gupta, Steven L. Spivak, Katelyn N. Lukas, Carl Schultz, Anjali Bisaria, Zabrisky Roland, K. Trego, Mason J. Appel, Parker Y. Jameson, Jacob Burch-Konda, Brittany Cruzan, Angelica Gonzalez-Sanchez, Monah Khalaj, J. Jain, Michael E. Dalziel, Julie Deichert, Sunjay Sethi, I. Shabalin, Rebeca M. Choy, Diana M. Muñoz, Christian R. Frey, P. Teriete, J. Taygerly, Yuchen Bai, Richard Zang, Linghu Xin, Claire L. Neilan, P. Barsanti, Michael A. White, Brian T. Jones
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Simple Summary Optical Genome Mapping (OGM) is an innovative technology that is transforming the genetic diagnosis of acute lymphoblastic leukemia (ALL). Thanks to its high resolution and direct DNA analysis, OGM enables the detection of complex rearrangements, deletions, and gene fusions that are often missed by conventional methods such as karyotyping, FISH, and PCR. In particular, it can identify clinically relevant alterations such as IKZF1 deletions, KMT2A rearrangements, and kinase gene fusions involving ABL1, PDGFRB, JAK2, and EPOR, thereby improving molecular classification and guiding targeted therapy. The integration of OGM with next-generation sequencing (NGS) provides a comprehensive genomic overview, supporting more precise diagnosis, risk stratification, and personalized treatment in ALL. Abstract Background: Acute lymphoblastic leukemia (ALL) is a genetically heterogeneous malignancy driven by structural variants (SVs) that impact diagnosis, prognosis, and treatment. Traditional methods such as karyotyping, FISH, and PCR often fail to detect cryptic or complex rearrangements, which are critical for accurate risk stratification. Methods: Optical Genome Mapping (OGM) is a technology that directly analyzes ultra-high-molecular-weight DNA, enabling the identification of balanced and unbalanced SVs, copy number variations (CNVs), and gene fusions with high resolution. This review compares the advantages and limitations of OGM versus standard techniques in ALL. Results: OGM improves ALL diagnosis by detecting clinically relevant alterations such as IKZF1 deletions, cryptic KMT2A rearrangements, and kinase fusions, especially in cases with normal or uninformative karyotypes. It reduces artifacts by eliminating cell culture and shortens reporting times. OGM resolves complex events like intrachromosomal amplifications and chromothripsis, enhancing classification and therapy decisions. Limitations include reduced sensitivity in repetitive regions, challenges in detecting Robertsonian translocations, difficulties with complex ploidies, and lower sensitivity for low-frequency subclones. Conclusions: Integrating OGM with next-generation sequencing (NGS) allows comprehensive genomic profiling, improving diagnosis, prognosis, and personalized treatment in ALL. Future advancements promise to further enhance the clinical utility of OGM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df8b9963a1fbf7c91e5fb4aba30093e1076d7ee6" target='_blank'>
              The Silent Revolution of the Genome: The Role of Optical Genome Mapping in Acute Lymphoblastic Leukemia
              </a>
            </td>
          <td>
            Claudia Simio, Matteo Molica, Laura De Fazio, Marco Rossi
          </td>
          <td>2025-10-27</td>
          <td>Cancers</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Background: Acute myeloid leukemia (AML) is a complex disease marked by a multitude of genetic mutations and dysregulated gene expression profiles stemming from genetic and epigenetic alterations. The disease is marked by significant biological heterogeneity, which complicates effective treatment and contributes to the emergence of drug resistance. The molecular profiles of gene mutations, chromosomal anomalies within AML considerably impacts disease progression, therapeutic response, and survival prognosis. Several predictive and prognostic models are used but present inaccuracy at individual level. Integrating multiple layers of biological data, including transcriptomic analyses (gene expression, mutational profiling, gene fusion identification) and population-level characterization through deconvolution and multiparametric flow cytometry, offers a powerful approach to better understand the complexity of AML and improve prognostic prediction and guide treatment strategies.
 Methods: We used a multi-omics data integration approach, including bulk RNAseq-based gene expression (n=45), single nucleotide variant and gene fusion data from bulk RNA sequencing (n=45), deconvolution of tumor microenvironment immune subtypes (n=45), immune subtype identification and blast characterization using multiparametric flow cytometry (MFC) (n=36), in vitro drug response (Venetoclax, n=23; 5-Azacytidine, n=23; Midostaurine, n=21, Gilteritinib, n=22), and relevant clinical metadata. This allowed us to thoroughly characterize tumor and its tumor microenvironment.
 Results: Applying Multiomics Factor Analysis (MOFA) identified 9 principal factors capturing the diversity of the disease. Factor 1 was significantly associated with Venetoclax response (p<0.05). Factors 2 and 4 showed significant associations with NPM1 mutations and DNMT3A mutations, while Factors 6 and 9 were significantly associated with 5-Azacytidine response (p<0.05). Factor 9 was also significantly associated with KIT mutations (p<0.05). The correlation analysis revealed strong associations between factors and key clinical covariates including timepoint and drug response groups.
 Next, we performed unsupervised clustering using the latent MOFA factors and identified four distinct clusters. Cluster 1 (C1, n=14) predominantly consisted of diagnostic samples (79%) with balanced immune infiltration. Cluster 2 (C2, n=13) showed moderate monocyte enrichment. Cluster 3 (C3, n=10) was characterized by significantly elevated regulatory T cell infiltration (p=0.017 vs C4, p=0.023 vs C1) and higher myeloid dendritic cell populations (p = 0.023 vs C2, p = 0.0051 vs C4), confirmed by both deconvolution (p=0.023 vs C2, p=0.0051 vs C4) and multiparametric flow cytometry analysis, consistent with an immunosuppressive microenvironment in this relapse-enriched cluster (60% relapsed samples). Cluster 4 was significantly enriched in monocytes (p < 0.001 vs C1 and C3).
 Finally, therapeutic response profiling across molecular clusters revealed distinct patterns of drug sensitivity. C1 demonstrated favorable responses to Venetoclax, Midostaurine, and Gilteritinib. C2 exhibited poor responses to all four tested drugs, indicating a globally resistant phenotype. C3 showed resistance specifically to Midostaurine and Gilteritinib, while remaining potentially responsive to other agents. Interestingly, C4 was characterized by poor sensitivity to Venetoclax and 5-Azacytidine but responded well to Midostaurine and Gilteritinib, highlighting subtype-specific vulnerabilities that could inform personalized therapeutic strategies. Interestingly, Gilteritinib demonstrated significant activity in specific molecular subgroups (C1 and C4) independently of FLT3-ITD status, suggesting broader therapeutic potential beyond canonical FLT3-driven AML. This unexpected sensitivity, possibly linked to off-target effects such as AXL inhibition, highlights novel vulnerabilities revealed by our integrative multi-omics framework.
 Conclusion: Our comprehensive multi-omics MOFA framework identified four clinically distinct AML subtypes with distinct therapeutic vulnerabilities, immune microenvironment characteristics, mutational profiles, and blast immunophenotypes. Our systematic approach demonstrates the potential of multi-omics integration for advancing personalized medicine in AML and provides a framework that will benefit from validation with larger cohorts to improve risk stratification personalized treatment approaches.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c9ac3a924304ce3dd2cf299a9f56500c5d7d76a" target='_blank'>
              Multi-omics integration reveals therapeutic vulnerabilities and immune microenvironment subtypes in Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            E. Alaterre, H. de Boussac, A. Kassambara, A. Steer, Amélie Machura, Julien Aityaya, Ouissem Karmous Gadacha, M. Abouladzé, C. Bret, Gaspar Aspas Requena, Ludovic Gabellier, C. Herbaux, A. Bruyer, Jérôme Moreaux
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/999c9f7dc52086f050a2aa67ed25e42b93fae763" target='_blank'>
              Single-cell RNA sequencing in osteosarcoma: applications in diagnosis, prognosis, and treatment
              </a>
            </td>
          <td>
            Christèle Asmar, Guy Awad, Marc Boutros, Simon Daccache, A. Chebly, C. Alix‐Panabières, H. Kourie
          </td>
          <td>2025-11-13</td>
          <td>Medical Oncology</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="


 Relapse remains the major cause of mortality in pediatric acute myeloid leukemia (AML). Although initiating driver alterations are major determinants of clinical outcomes, the extent to which relapse emerges through clonal selection of pre-existing subpopulations or by the acquisition of de novo mutations remains unclear.
 We established a cohort of 39 diagnosis-relapse (D-R) and 2 relapse-relapse (R-R) pairs of pediatric AML. We comprehensively profiled the cohort using whole-genome sequencing (WGS) and target-capture sequencing. WGS (median coverage: 49x) and capture sequencing (450x) were performed to detect single-nucleotide variants (SNVs), insertions and deletions (indels), structural variants (SVs), and copy number variants (CNVs). Somatic alterations with variant allele frequencies (VAFs) exceeding background levels were regarded as real (P<0.05 by binomial tests), and coding genes were classified according to pathogenicity to assign pathogenic or likely pathogenic (P/LP) labels. Enrichment was assessed by Fisher's exact test.
 The study cohort represented major high-risk subtypes of pediatric AML, including KMT2A-rearranged (n=13, 32%) and NUP98-rearranged (n=6, 15%) AMLs. A total of 22,088 somatic alterations were identified across SNVs, indels, SVs, and CNVs. Among 142 P/LP mutations, WT1 (n=19) and FLT3 (ITD: n=9, TKD: n=7, other: n=3) were most commonly mutated, followed by NRAS (n=17) and SETD2 (n=9) mutations. The mutational burden was significantly higher at relapse (median 216 vs 376 mutations per sample at D-R, P=9.4×10⁻⁶), whereas the total number of P/LP mutations per sample did not significantly differ (median 4 at both, P=0.34). Among P/LP mutations, 39 had significantly higher VAF at relapse (P<0.05), with frequent involvement of FLT3 (n=7), WT1 (n=5), and SETD2, TP53, CREBBP, and NRAS (n=2 for each), indicating a growth advantage of clones with these alterations. Notably, 107 P/LP mutations were found at diagnosis, and only 35 P/LP mutations were specifically detected at relapse while undetectable at diagnosis, even by targeted sequencing, suggesting acquisition during or after therapy, or selection of rare pre-existing clones below the detection limit estimated to be 1%. Interestingly, clones with these mutations did not always expand at relapse, with different WT1 or FLT3 mutations depleted or enriched within an individual patient. In contrast, NRAS mutations were frequently depleted at relapse, suggesting that clones with NRAS mutations are sensitive to chemotherapy.
 We further investigated changes in clonal structures of the disease during progression. VAF at diagnosis and relapse showed clustering of mutations with specific patterns in each patient, and all relapses were clonally related to the paired AML at diagnosis. For example, initiating and category-defining alterations were identified with high VAF at both time points. In most cases (n=35, 85.3%), we observed that non-initiating mutations with high VAF (>20%) at diagnosis were frequently depleted to undetectable levels in the later time points. Contrarily, subclones marked by somatic mutations that were undetectable or with low VAF (<5%) at diagnosis became dominant in 38 cases (92.7%), indicating that leukemic cells were depleted down to only a few persistent genetic clones during remission. One exception is a case with FUS::ERG AML, which showed concordant VAF of somatic mutations at both diagnosis and relapse, including mutations with low VAF, suggesting that the entire population was resistant to chemotherapy with no therapy-induced bottleneck. Notably, mutations that mark subclones fated for relapse were mostly non-pathogenic or non-coding mutations, and 16 cases did not have any P/LP coding cooperating mutations that mark the relapsed clone.
 These data indicate that relapse of pediatric AML is driven by two mechanisms: one is a selection or expansion of clones with somatic alterations that confer chemo-resistance or growth advantage over the rest of the clones, and the other is a stochastic, non-genetic mechanism that likely confers a competitive advantage to certain clones, such as transcriptional heterogeneity. Bulk and single cell transcriptional analyses and clonal tracking using ultra-deep sequencing (~5000x) may reveal additional insights into the mechanisms of pediatric AML relapse.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e65f1675dcee272d9925d5be0f3a8cd35f6d84c1" target='_blank'>
              Clonal evolution of pediatric Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Masayuki Umeda, P. Kolekar, A. Huskey, Shruthi Suryaprakash, Hanxia Li, Tamara Westover, Jing Ma, Michael P Walsh, Guangchun Song, Yanling Liu, Quang Tran, Seth E. Karol, Hiroto Inaba, J. Rubnitz, Raul C. Ribeiro, Xiaotu Ma, J. Klco
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9932f70712f8bef30c70a5b989c4e744cf379f36" target='_blank'>
              Harnessing transcriptomics for discovery of natural products to overcome acquired cancer resistance.
              </a>
            </td>
          <td>
            Heerim Yeo, Sang-Yun Kim, Sang-Min Park
          </td>
          <td>2025-12-08</td>
          <td>Archives of pharmacal research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Therapy resistance in acute myeloid leukemia (AML) remains a major clinical obstacle, particularly due to the persistence of leukemia stem cells (LSCs) capable of metabolic adaptation. While venetoclax (Ven) inhibits oxidative phosphorylation (OXPHOS), we found that Ven-resistant LSCs undergo glycolytic reprogramming to bypass OXPHOS inhibition. This metabolic shift is supported by enhanced ribosome biogenesis, sustained by upregulated de novo guanine nucleotide biosynthesis. Abundant guanine nucleotides suppress the impaired ribosome biogenesis checkpoint (IRBC), leading to TP53 destabilization and persistent MYC expression. Inhibition of inosine monophosphate dehydrogenases (IMPDH1/2) depletes guanine nucleotides, activates IRBC, stabilizes TP53, represses MYC, and impairs the metabolic shift to glycolysis. This metabolic rewiring disrupts LSC stemness and suppresses the reconstitution of human AML cells in xenotransplantation experiments. Notably, the suppression of LSC stemness was observed regardless of Ven resistance or the TP53 mutational status of AML cells. These findings reveal that mutation-independent TP53 inactivation is involved in resistant AML and suggest that targeting guanine nucleotide biosynthesis may offer a clinically actionable strategy to eradicate therapy-resistant LSCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6613ff1ea81832476d4f80a2812043adb8207057" target='_blank'>
              Guanine nucleotides drive ribosome biogenesis and glycolytic reprogramming in acute myeloid leukemia stem cells.
              </a>
            </td>
          <td>
            Gentaro Kawano, Riichiro Ikeda, Daisuke Ishihara, T. Shima, Teppei Sakoda, Shunsuke Yamamoto, Yuichi Kochi, Yuichiro Semba, Sanae Ashitani, Y. Mori, Koji Kato, Takahiro Maeda, Toshihiro Miyamoto, Tomoyoshi Soga, Koichi Akashi, Y. Kikushige
          </td>
          <td>2025-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Lung cancer remains one of the leading causes of cancer-related mortality worldwide, with tumor recurrence and metastasis posing significant challenges despite advances in targeted therapies and immunotherapy. Cellular dormancy, a reversible, quiescent state marked by cell cycle arrest, has emerged as a key driver of therapeutic resistance and disease relapse, particularly in small-cell lung cancer (SCLC) and non-small cell lung cancer (NSCLC). Multiple mechanisms, including autophagy, stress-adaptive signaling, microenvironmental cues, and epigenetic dysregulation, have been implicated in the regulation of dormancy and long-term cell survival. Among these, epigenetic modifications such as DNA methylation, histone modifications, and non-coding RNAs (ncRNAs) play pivotal roles in maintaining dormancy by repressing proliferative gene expression programs. Increasing evidence suggests that dormant tumor cells harbor distinct epigenomic signatures, which may serve as predictive biomarkers for minimal residual disease (MRD) and relapse risk. This review summarizes current advances in understanding the epigenetic regulation of cellular dormancy in lung cancer, with a particular emphasis on the interplay between epigenetic modifiers and oncogenic signaling pathways. Furthermore, emerging molecular targets and associated therapeutic agents currently under clinical evaluation are presented, emphasizing how a deeper understanding of the epigenetic landscape governing dormancy may inform the development of novel interventions to improve long-term clinical outcomes in lung cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f78e8931704f8d1c6b0d604cdcde3728bfe1b4a" target='_blank'>
              Unlocking Lung Cancer Cell Dormancy: An Epigenetic Perspective
              </a>
            </td>
          <td>
            F. Fabrizio
          </td>
          <td>2025-11-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="


 Glioblastoma remains one of the most treatment-refractory solid tumors due to rapid adaptive resistance across multiple convergent molecular pathways. In particular, persistent activation of the PI3K-AKT axis combined with mesenchymal transition (EMT) reprogramming contributes to rapid failure of standard therapeutic regimens, including temozolomide, radiation, and immunotherapeutics. To address this clinical gap, we developed a synthetic RNA-guided dual-targeting system designed to simultaneously suppress both canonical PI3K-AKT activation and mesenchymal escape signatures, using strand-engineered molecules optimized for combinatorial silencing. Using patient-derived glioblastoma stem cell lines and isogenic resistance models, we performed transcriptomic profiling to identify co-expressed resistance-driving nodes. Computational integration of CRISPR perturbation datasets, RNA interference screens, and synthetic lethality maps enabled selection of targetable PI3K-AKT intermediates and EMT regulators including SOX2, TWIST1, and ZEB1. Synthetic RNA molecules were designed to co-suppress both vertical (PI3K-AKT) and horizontal (EMT plasticity) resistance axes, with in vitro validation demonstrating >70% knockdown efficiency and significant synergistic reduction in proliferation (p<0.001) across diverse resistance phenotypes. Mechanistic assays revealed reversal of mesenchymal signatures with restored epithelial markers (E-cadherin upregulation; vimentin downregulation) alongside AKT dephosphorylation. Combination treatment with low-dose temozolomide further amplified cell death, suggesting sensitization to standard chemotherapy. In vivo murine xenograft models exhibited >60% reduction in tumor volume compared to controls (p<0.01), without evidence of off-target cytotoxicity or systemic toxicity. This study demonstrates the feasibility of multi-axis synthetic RNA-guided combinatorial targeting to suppress glioblastoma therapeutic resistance. These findings support future clinical translation of modular RNA-based therapeutics capable of overcoming the heterogeneity and plasticity that characterize high-grade gliomas.



 Shivi Kumar. Synthetic RNA-guided combinatorial targeting of glioblastoma adaptive resistance: Dual suppression of PI3K-AKT and mesenchymal transition escape pathways [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr A036.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/746db514f096cab5d3002088013773d14b8c8b84" target='_blank'>
              Abstract A036: Synthetic RNA-guided combinatorial targeting of glioblastoma adaptive resistance: Dual suppression of PI3K-AKT and mesenchymal transition escape pathways
              </a>
            </td>
          <td>
            Shivi Kumar
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Rare tumor cell populations can evade cancer therapies, ultimately driving drug resistance and relapse. These resilient populations, termed “persisters,” display distinct molecular states compared to the bulk tumor. While some persisters acquire genetic mutations that confer growth advantage, others undergo non-genetic remodeling that enhances cellular plasticity and fitness. Previous studies suggest that stochastic gene activation promotes persister survival (Shaffer et al., Nature 2017; Goyal et al., Nature 2023), yet the master regulators of this plasticity remain undefined. Using DNA barcoding, we demonstrate that pre-determined, non-stochastic persisters exist in KRAS-mutant non-small cell lung cancer (NSCLC) treated with the KRAS G12C inhibitor sotorasib. These clones repopulate the tumor without harboring recurrent genetic alterations. By isolating single clones destined to persist or go extinct, and profiling their transcriptomes and chromatin accessibility, we found that persister clones activate an interferon-γ (IFNγ) transcriptional signature accompanied by increased chromatin accessibility at sites enriched for the transcription factors IRF1 and ZNF384. Functionally, ectopic ZNF384 expression converted the entire cell population into a resistant state, while IRF1 knockout prevented persister formation. Cleavage Under Targets & Release Using Nuclease (CUT&RUN) assays of ZNF384 and IRF1 revealed their binding to promoter and enhancer regions, including loci regulating IFNγ-responsive genes, thereby contributing to persister plasticity and survival. Together, our study identifies IRF1 and ZNF384 as tumor-intrinsic regulators linking IFNγ signaling to transcriptional plasticity, providing mechanistic insight into the evolution of drug tolerance and resistance in KRAS-mutant NSCLC.



 Chendi Li, Anahita Nimbalkar, Christopher J. Graser, Melissa Vieira, Mohammed U. Syed, Qian Qin, Barbara Karakyriakou, Sarah Clark, Anne Y. Saiki, Paul E. Hughes, Christopher Ott, Luca Pinello, Franziska Michor, Aaron N. Hata. Transcription factors governing the evolution of drug-tolerant persisters in KRAS-mutant non-small cell lung cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B014.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/331fb37a264b2ec60d8af95499de6058539d7fa9" target='_blank'>
              Abstract B014: Transcription factors governing the evolution of drug-tolerant persisters in KRAS-mutant non-small cell lung cancer
              </a>
            </td>
          <td>
            Chendi Li, Anahita Nimbalkar, Christopher J Graser, Melissa Vieira, M. U. Syed, Qian Qin, Barbara Karakyriakou, Sarah Clark, A. Y. Saiki, Paul E Hughes, Christopher Ott, Luca Pinello, Franziska Michor, Aaron N. Hata
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="


 EGFR-mutant non-small cell lung cancers (NSCLCs) initially respond to EGFR tyrosine kinase inhibitors (TKIs) such as osimertinib, but tumors eventually develop resistance. While genetic mutations explain many instances of resistance, an increasing body of evidence points to transcriptional and cell state changes as another route by which tumor cells evade therapy. The dynamics that shape these adaptive responses and the non-genetic mechanisms involved in facilitating them remain incompletely understood. We investigated transcriptional programs activated in EGFR-mutant LUAD cells following TKI treatment and observed robust induction of SOX2, a transcription factor involved in promoting stemness and lineage plasticity, within a day of drug exposure. In parallel, we investigated the role of APOBEC cytidine deaminases (APOBEC3A and APOBEC3B), which have been linked to mutagenic processes in cancer. While both enzymes have been implicated in drug resistance, the individual contributions of the two enzymes remain ill studied. Using competition assays with single gene knockouts of either APOBEC3A or APOBEC3B, we observed that loss of APOBEC3A leads to a more profound reduction in the survival of drug-tolerant cells following TKI therapy. These results highlight a functional role for APOBEC activity in sustaining early drug-tolerant populations under therapeutic pressure. Over the course of long-term osimertinib treatment, we also identified distinct subpopulations of cells with elevated ΔNp63, an isoform of TP63 that serves as a marker of squamous cell trans-differentiation. These ΔNp63-high cells emerge during the acquisition of drug resistance, and evidence suggests that this subpopulation may rely on CDK4, a cell cycle-regulating kinase, to sustain growth under therapeutic pressure. Pharmacological inhibition of CDK4 with palbociclib reduced the survival of ΔNp63-high cells, highlighting a potential therapeutic vulnerability in this proliferative subpopulation. Together, our findings demonstrate that adaptive resistance in EGFR-mutant NSCLC is driven by several non-genetic resistance mechanisms, including SOX2 induction, ΔNp63-driven squamous cell transdifferentiation, and APOBEC-mediated promotion of drug-tolerant persister cells.



 Sharan Srinivasan, Jessica N. Becerra, Nina M. Garcia, James V. Alvarez. SOX2 induction and ΔNp63-high subpopulations mark adaptive responses to EGFR inhibition in lung adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B019.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a218a816c806ca36ea70bf5b2dadd14af3bb49f" target='_blank'>
              Abstract B019: SOX2 induction and ΔNp63-high subpopulations mark adaptive responses to EGFR inhibition in lung adenocarcinoma
              </a>
            </td>
          <td>
            Sharan Srinivasan, Jessica N Becerra, N. M. Garcia, James V. Alvarez
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/182327417f0bfd0539120b0fcc3161f6ade537ce" target='_blank'>
              Dissecting neuroblastoma heterogeneity through single-cell multi-omics: insights into development, immunity, and therapeutic resistance.
              </a>
            </td>
          <td>
            Guoqian He, Si-Jia He, Xiao-yu Jing, Yi-Ling Dai, Xia Guo, Ju Gao, Wei Zhang
          </td>
          <td>2025-11-27</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Dose–response curves (DRCs) are a standard protocol in experimental biology to assess the effect of anticancer agents in vitro. Yet, therapy itself can reshape tumor composition, driving the gradual evolution of the DRC. Monitoring such changes can reveal the emergence of resistance, as well as evolutionary trade-offs that may be exploited to enhance therapeutic efficacy. Here, we investigate the evolution of DRCs in response to three different ALK inhibitors (ALKi) in ALK-positive anaplastic large cell lymphoma (ALCL) using patient-derived cell lines. Over the course of six months, ALKi concentrations were gradually escalated, allowing ALCL cells to adapt and survive at higher drug levels. Strikingly, at the same time, these resistant populations exhibited reduced fitness in the absence of the drug, indicating an evolutionary cost. This phenomenon, known as “drug addiction” or dependence, has been previously observed in melanoma following resistance to MAPK inhibitors and suggests that treatment holidays could optimize tumor control in the long term. Guided by these findings, we developed a mathematical model to capture the temporal evolution of DRCs and to test how therapy design might exploit addiction in ALCL. Calibrated with experimental data, the model simulated tumor growth under alternative dosing regimens. Simulations revealed that continuous therapy rapidly selects for resistance, leading to tumor regrowth within ∼150 days. In contrast, intermittent dosing schedules maintained long-term control for more than 300 days. Moreover, the model identified an adaptive strategy: switching from continuous to intermittent dosing precisely when tumor burden reaches its minimum. This evolutionarily informed approach prevents resistant clones from dominating while leveraging the fitness disadvantage associated with drug addiction. Preliminary experiments with naïve and resistant ALCL cell lines, cultured under intermittent dosing in both monoculture and co-culture systems, support the model’s predictions. These experiments also enable the refinement of the model's parameters to incorporate tumor heterogeneity. While additional validation is required, our findings demonstrate that integrating mathematical modeling with experimental DRCs provides a quantitative framework for predicting the evolutionary dynamics of resistance and addiction. Collectively, this study highlights the potential of adaptive and evolution-based therapy in ALCL. Optimized treatment schedules informed by evolutionary trade-offs can reduce tumor burden, delay progression, and exploit vulnerabilities generated by resistance mechanisms.



 Franco Pradelli, Lucy Hare, Jamie Matthews, Roxane Chen, Aiindrila Dhara, G. A. Amos Burke, Suzanne D. Turner, Jeffrey West. Capturing dose-response adaptation to identify evolutionarily-informed treatment strategies [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B037.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86b1f1eb717bb47b50718cff89eef53948102c75" target='_blank'>
              Abstract B037: Capturing dose-response adaptation to identify evolutionarily-informed treatment strategies
              </a>
            </td>
          <td>
            Franco Pradelli, Lucy Hare, J. Matthews, Roxane Chen, Aiindrila Dhara, G. A. A. Burke, Suzanne D. Turner, J. West
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Introduction: Genetic risk stratification in MDS is critical to predict disease outcomes and to direct therapy. Our group has developed a whole genome sequencing (WGS) approach for comprehensive genomic analysis at presentation in myeloid malignancies. We hypothesized that WGS as a single platform would be feasible in a real-time clinical setting and would provide increased diagnostic and prognostic yield compared to conventional genomic profiling using cytogenetics (metaphase karyotyping without FISH) combined with targeted gene panel sequencing (i.e., targeted sequencing+cytogenetics). Here, we report a prospective clinical trial to determine feasibility of WGS incorporation into real-time clinical evaluation of MDS.
 Methods: WGS was performed on DNA isolated from bulk peripheral blood or bone marrow aspirate samples to a target depth of 60x without paired normal DNA sequencing. Targeted analysis was performed for small variants in 64 recurrently mutated myeloid genes and genome-wide identification of copy-number alterations (CNAs) > 5 Mbp, copy neutral loss of heterozygosity (cnLOH), and structural variants (SVs) involving 639 genes. Conventional genomic profiling consisted of high-coverage (>1000x) targeted gene panel sequencing of the same 64 genes with a sensitivity of 2% variant allele fraction (VAF), that includes all IPSS-M genes. Metaphase cytogenetics was performed by institutional standards.
 Results: We prospectively enrolled 66 patients. Conventional cytogenetics was unsuccessful or inconclusive in 9 (14%) patients. In contrast, WGS was successful in 100% cases and 17 of 19 (89%) cases categorized as complex by cytogenetics were also identified as complex by WGS. In cases with noncomplex cytogenetic changes, WGS identified 100% of SVs (2 of 2) and 89% of CNAs (19 of 21) identified by cytogenetics. WGS also identified new cnLOH and CNAs in 11 of the 57 cases with sufficient cytogenetics, including cnLOH of chr 17p (n=2) and del 17p (n=1), which led to new identification of multihit TP53 in these 3 cases. In 9/9 cases with unsuccessful/inconclusive conventional cytogenetics, WGS provided cytogenetic information, permitting IPSS-M risk categorization as follows: Very Low (n=1), Low (n=3), Moderate Low (n=1), High (n=1), Very High (n=3). In 49 of the 57 cases (86%) with sufficient cytogenetics, the IPSS-M risk categories were concordant between WGS and conventional genomic profiling. In 1 case, WGS stratified a patient into higher risk categories due to new identification of a complex cytogenetic risk category. In 7 cases, WGS stratified patients into the next lower IPSS-M risk category compared to targeted sequencing and cytogenetics due to limitations of detecting low abundance IPSS-M relevant mutations using lower coverage WGS (12 variants with a median VAF of 4.3% [range 3.7-10.8][n=6 cases]) or low abundance or missed CNAs ([<10% cell fraction][n=3 cases, 2 also with missed variants]). However, the VAFs of all mutations detected by both WGS and high-coverage targeted sequencing were significantly correlated (r=0.91, n=185 variants, p<0.0001). To overcome the limitations of lower coverage WGS to detect sequencing variants, we next combined variants detected by either WGS or high-coverage targeted sequencing (WGS+targeted sequencing) and compared IPSS-M risk categories in 57 cases with sufficient cytogenetics. WGS+targeted sequencing stratified 1 patient into a higher risk and 1 patient into a lower IPSS-M category (due to missing low abundance CNAs) compared to targeted sequencing+cytogenetics. This suggests that deeper WGS coverage may enhance variant detection and further improve risk stratification when using WGS as a single assay for IPSS-M risk categorization.
 Conclusion: WGS is feasible in MDS and has several advantages over conventional genomic profiling, including when cytogenetics fail or are inadequate, and detection of cytogenetically silent CNAs and cnLOH. Compared to conventional genomic profiling, WGS provided new genomic information that changed IPSS-M risk categories in 10 (15%) of patients. The current cost of reagents, labor, and analysis for WGS described here is comparable to conventional testing. Increasing WGS coverage to >90x will improve detection of mutations with VAFs<10%, making WGS an ideal single assay for IPSS-M risk stratification. Further studies with outcomes data are needed.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5246470fb11e12e61f839e04b7f37bb9a2b4fad" target='_blank'>
              A prospective Study of whole genome sequencing for genomic profiling and risk determination in MDS
              </a>
            </td>
          <td>
            Rigoberto De Jesus, Samuel Urrutia, David Spencer, Eric Duncavage, Tasha Burton, Na'kie Coleman-elhasan, Sridhar Srivatsan, Molly Schroeder, Kilannin Krysiak, Geoffrey Uy, Timothy Ley, Matthew Walter, Meagan Jacoby
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a38b8fdfa925b911ba77595f93672078904315e1" target='_blank'>
              The role of Alu elements in causing BRCA1 structural variation and breast cancer susceptibility.
              </a>
            </td>
          <td>
            Minjae Yu, Dahee Jo, Wonseok Shin, Kyudong Han
          </td>
          <td>2025-11-17</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 Objective: Extramedullary multiple myeloma (EMM) is a highly aggressive subtype of multiple myeloma (MM), characterized by the dissemination of malignant plasma cells beyond the bone marrow (BM) microenvironment. This subtype is frequently associated with treatment resistance and poor prognosis. However, the cellular and molecular landscape of EMM, particularly in paired BM and extramedullary tissue samples, remains poorly understood. This study aims to systematically dissect the cellular heterogeneity, genomic alterations, and immune interactions in EMM using single-cell RNA sequencing (scRNA-seq), thereby identifying key genes, signaling pathways, and transcriptional regulatory mechanisms underlying tumor progression, immune evasion, and subclonal adaptation in extramedullary sites.
 Methods: Paired BM and extramedullary tumor samples were collected from five MM patients with confirmed extramedullary metastasis and subjected to scRNA-seq. Cellular subpopulations were identified and compared between BM and extramedullary compartments. Functional enrichment analysis was conducted to characterize biological processes in tumor subsets. Genomic instability was inferred at the single-cell level using the inferCNV (inferring Copy Number Variation) algorithm. SCENIC (Single-cell regulatory network inference and clustering) was applied to reconstruct transcription factor regulatory networks and assess differential regulatory activity. Cell-cell communication patterns between tumor and microenvironmental cells were analyzed using CellPhoneDB to reveal potential mechanisms of immune evasion and stromal interactions in extramedullary lesions.
 Results: A transcriptional atlas of extramedullary-involved plasma cells was constructed, revealing a distinct subpopulation characterized by low proliferative capacity, metabolic reprogramming, and immunosuppressive features. inferCNV analysis demonstrated a more complex copy number variation (CNV) landscape in extramedullary plasma cells compared to their BM counterparts, reflecting subclonal adaptation under distinct microenvironmental pressures. Differential gene expression analysis identified core regulatory genes shared across compartments, including TCEB2, ATPIF1, and NCL, associated with transcriptional control, metabolic adaptation, and cell survival. Upregulation of stress-response genes such as HSP90AB1, HSP90AA1, and LGALS1 suggested enhanced cellular resilience in adverse conditions. SCENIC analysis revealed dynamic changes in the activity of key transcription factors in extramedullary plasma cells, including GATA4, IRF9, MAFB, and USF2, indicating transcriptional reprogramming associated with metastatic adaptation. Immune profiling uncovered a highly immunosuppressive extramedullary microenvironment with enrichment of inhibitory immune cells and attenuated tumor-immune cell interactions. CellPhoneDB analysis further identified enhanced crosstalk between tumor cells and stromal components, which may contribute to immune evasion and tumor persistence.
 Conclusion: This study provides a comprehensive single-cell transcriptomic characterization of MM with extramedullary involvement, revealing a unique extramedullary tumor cell phenotype marked by low proliferation, metabolic reprogramming, and pronounced genomic instability. Key regulatory genes (TCEB2, ATPIF1, NCL) and stress-adaptive molecules (HSP90AB1, LGALS1) were identified as potential mediators of tumor survival. Transcription factor remodeling (GATA4, IRF9) and extensive tumor-stromal interactions in the immunosuppressive extramedullary niche highlight potential mechanisms of immune escape and metastatic fitness. These findings offer novel insights into the biological underpinnings of extramedullary myeloma and may inform therapeutic strategies targeting extramedullary disease.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/307a8fe09c00ec6e9285cf3b54b0e7082bc4c52b" target='_blank'>
              Single-cell RNA sequencing reveals cellular heterogeneity and molecular features of extramedullary-involved multiple myeloma
              </a>
            </td>
          <td>
            Yajun Wang, quande Lin, jiangxue Hou, xu Ji, Chunyan Sun, Yu Hu
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Introduction: The t(4;14) translocation subtype of multiple myeloma (MM) is characterized by overexpression of NSD2, a histone methyltransferase which catalyzes di-methylation of histone H3 lysine 36 (H3K36me2). About 12% of MM patients have t(4;14) and this is associated with poor outcome. If and how NSD2 drives unique epigenetic vulnerabilities in this subtype of MM is not well understood.
 Methods: Primary MM samples from the MMRF CoMMpass trial (NCT01454296) were analyzed by Whole Genome Bisulfite Sequencing (WGBS). CRISPR/Cas9 was used to genetically ablate the NSD2 translocated allele in KMS11 and KMS18 (hereafter referred to as NSD2-high and NSD2-low). Multiple single cell clones were isolated and confirmed by immunoblot analysis and sequencing. KMS11 NTKO and TKO models (Kuo et al. Molecular Cell 2011) were also used for CUT&Tag and DNAm analysis. Cell responses to decitabine (DAC) and GSK-3685032 were evaluated by flow cytometry for Annexin V and live/dead staining on day 4. Dynamic BH3 profiling (DBP) was performed similar to previously described (Matulis, ASH 2024). Gene expression was assessed using RNAseq with ribosomal RNA depletion (Kapa).
 Results: Analysis of CoMMpass samples showed higher levels of DNAm in the t(4;14) subtype compared to other subtypes (p=1.17x10-12). Consistent with DNAm being connected to NSD2 overexpression, KMS11 NSD2-high cells had higher DNAm than isogenic NSD2-low cells (p<1x10-9). Elevated DNAm in NSD2-high cells co-occurred in genomic regions with higher levels of H3K36me2 (OR=1.85, p<1x10-15). Treating KMS11 and KMS18 NSD2 isogenic cell lines with the DNA hypomethylating agent DAC showed NSD2-high cells are significantly more sensitive to DNAm inhibition (p<0.01 at ≥100 nM for KMS11 and p<0.001 at ≥500 nM for KMS18). This sensitivity to DNAm inhibition was seen again using the DNMT1 enzymatic inhibitor GSK-3685032 (p<0.001 at ≥100 nM). DBP of KMS18 showed NSD2-high cells increased mitochondrial priming when treated with DAC (p<0.001) whereas NSD2-low cells did not significantly change, consistent with the increased apoptosis observed in NSD2-high cells.
 RNAseq of KMS11 NSD2-high and NSD2-low clones treated with 0, 100, or 500 nM DAC showed 3,599 differentially expressed genes (FDR<0.01, fold-change>2). Genes downregulated with DAC treatment were common between NSD2-high and NSD2-low cells and included genes involved in cell cycle and MYC targets as indicated by Gene Set Enrichment Analysis (GSEA). Genes upregulated with DAC-treatment were more distinct between NSD2-high and NSD2-low cells, with GSEA indicating interferon response and TNFa signaling via NFKB more significantly induced in NSD2-high cells (FDR<0.0023) than in the NSD2-low cells (FDR>0.13).
 Further investigation of DAC-induced RNAseq shows an increase in transcription of endogenous retroviruses (ERVs), specifically in the NSD2-high KMS11 cells. This suggests that hypomethylating agents induce transcription of these ERVs, which then induce an intracellular interferon response, and this is specific to NSD2-high MM. Consistent with this, these ERVs had higher levels of DNAm in isogenic KMS11 NSD2-high cells as compared to NSD2-low cells (FDR<0.01). This was also observed in CoMMpass samples where higher DNA methylation was found at these ERVs in t(4;14) MM.
 Conclusions: NSD2-mediated increases in H3K36me2 in t(4;14) MM correspond with higher DNAm as compared to other MM subtypes. This is consistent with data from other cell types suggesting H3K36me2 is recognized by the PWWP domains of the DNMT3 DNA methyltransferasaes. Multiple NSD2 isogenic models suggest this creates a preferential sensitivity to DNA hypomethylating agents in NSD2-high MM. This phenomenon was shown both with DAC, a cytosine analog that incorporates into DNA, as well as with GSK-3685032, a DNMT1 enzymatic inhibitor. Both Annexin V staining and DBP indicated increased apoptosis, specific to NSD2-high cells. In addition, RNAseq data suggests this increased apoptosis is due to re-expression of ERVs that are triggering a viral defense response and interferon signaling. Consistent with this, t(4;14) cell line models and patient samples had higher levels of DNAm at ERVs, suggesting these are silenced through DNAm in t(4;14), but by other mechanisms in non-t(4;14) MM. Together, these data indicate NSD2 remodels the epigenome of t(4;14) MM creating unique epigenetic dependencies that can be exploited with readily available DNA hypomethylating agents.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c372d03852ab31bfaf4fcce61c8d2085a69b5cdf" target='_blank'>
              NSD2 expression remodels DNA methylation, creating a targetable epigenetic dependency in t(4;14) multiple myeloma
              </a>
            </td>
          <td>
            Anna Baj, Jonathan C Patton, Robert M Chavez, Shannon M Matulis, Doris R. Powell, Mala Shanmugam, Arun Wiita, Constantine S Mitsiades, J. Keats, Jonathan Licht, S. Lonial, Lawrence Boise, Benjamin G. Barwick
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>110</td>
        </tr>

        <tr id="


 Background: Despite recent advancements in treatment for multiple myeloma (MM), most patients eventually relapse due to the emergence of resistant tumor subclones. Bulk DNA sequencing has identified recurrent resistance-associated mutations and can infer subclonal architecture, but it lacks the resolution to determine whether multiple mutations co-exist within the same malignant cell or arise in separate subclones. This distinction is critical, as multiple somatic variants within individual cells may confer resistance that does not arise from any single alteration alone. We hypothesized that targeted single-cell DNA sequencing of myeloma tumor cells would reveal the co-occurrence of mutations within individual cells that are missed by conventional bulk sequencing approaches.
 Methods: Bone marrow aspirates were collected from 46 patients, including 9 with newly diagnosed and 37 with relapsed or refractory MM. Two patients were sampled at multiple time points, yielding a total of 48 samples. CD138⁺ plasma cells were enriched using magnetic-activated cell sorting and subjected to single-cell encapsulation for simultaneous genomic and proteomic profiling. Libraries were prepared using the Mission Bio Tapestri Single-Cell Multiple Myeloma Multiomics Assay, which includes amplicons targeting single-nucleotide variants, insertions and deletions, and focal copy number aberrations commonly altered in the disease. In parallel, immunoglobulin heavy- and light-chain V(D)J clonotypes were identified, and cells were labeled with a 20-plex oligo-conjugated antibody panel to quantify surface protein markers associated with cell lineage and immunotherapy targets. Tumor cells were defined as those harboring the dominant immunoglobulin V(D)J sequence within each sample. Reference cells were identified as VDJ-negative and expressing high levels of non–plasma cell markers (CD3, CD19, or CD33). Somatic mutations were identified by comparing variant allele frequencies between tumor and reference cell populations.
 Results: Across all 48 samples, 146,946 cells were sequenced, with 68,052 of those being tumor cells. Hallmark MM copy number alterations, such as 17p deletion, 1q amplification, and 13q deletion, were identified using the copy number assay. The most frequent mutations identified were in KRAS, TP53, TENT5C, and CRBN. The two patients with CRBN mutations underwent 27 and 34 prior cycles of immunomodulatory (IMiD) therapy, respectively, compared to a median of 8.0 cycles in patients without CRBN mutations, raising the possibility of treatment-driven selection. In another case, we identified three mutually exclusive KRAS mutations (Q61H, G12V, and G12D) in distinct subclones within a single patient, consistent with convergent evolution toward activation of the MAPK pathway. We also resolved a true “double hit” in another patient, characterized by 58% of tumor cells carrying a deletion of one allele and a somatic mutation in the other, consistent with biallelic inactivation. Finally, we observed CDKN2A more frequently co-mutated with TP53, PTPRT, and TENT5C than as isolated events, suggesting multi-hit inactivation of key cell-cycle checkpoints and DNA-damage response pathways that may drive selective growth advantage or therapeutic resistance. CDKN2A and TP53 are canonical tumor suppressors that regulate G1/S transition and apoptosis, respectively, while PTPRT encodes a phosphatase that negatively regulates STAT3 signaling, and TENT5C is a tumor suppressor involved in mRNA stability and plasma cell homeostasis. These recurrent co-mutations may act synergistically by simultaneously disrupting transcriptional and signaling regulation, thereby conferring a selective tumor cell growth advantage under therapeutic pressure.
 Conclusions: Targeted single-cell DNA sequencing reveals co-occurring mutations within individual myeloma cells that are obscured by bulk approaches, providing critical insight into the cooperative and competitive genomic events driving disease progression and resistance. These findings support a model in which myeloma subclones accumulate synergistic alterations within the same cell. Work is ongoing to provide functional validation of key mutation pairs and their integration with transcriptomic data to refine the mechanistic understanding and identify candidates for combination therapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98368199e493cbd0fcaadb2df0e1e6f8e200c1a5" target='_blank'>
              Single-cell DNA sequencing uncovers synergistic co-mutations in multiple myeloma
              </a>
            </td>
          <td>
            Ava Amidei, B. Diamond, Elizabeth Lyubchenko, Stephanie Fernandes, Baharak G. Fard, L. Nousheen, Marc Arribas-Layton, Adam Sciambi, D. Kazandjian, James Hoffman, Abhishek Pandey, Marcella Kaddoura, Brian Walker, O. Landgren, D. Coffey
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="


 Multiple myeloma (MM) is a genetically heterogeneous malignancy marked by complex clonal dynamics and a range of cytogenetic alterations that impact prognosis and treatment response. While fluorescence in situ hybridization (FISH) remains a standard diagnostic tool, it is limited to known abnormalities and requires enriched plasma cells. Next-generation sequencing (NGS) offers comprehensive profiling in a single assay but is not yet widely adopted in routine clinical care.
 To evaluate the performance and clinical utility of a custom-designed, dual-input (DNA/RNA) targeted sequencing panel across multiple platforms (Ion Torrent and Illumina) for molecular karyotyping, clonality assessment, and transcriptomic profiling in newly diagnosed MM.
 Bone marrow samples from 85 patients enrolled in GEM clinical trials with high-risk FISH-defined cytogenetics were analyzed. DNA and RNA were extracted from CD138⁺ plasma cells and CD138⁻ germline controls. Targeted capture libraries were sequenced on Illumina and Ion Torrent platforms using a panel covering 2.735 Mb and 107 genes involved in MM pathogenesis. Bioinformatic analysis included variant calling (SNVs, indels), structural variant and CNV detection, and V(D)J rearrangement analysis via MiXCR. IG/TR clonality was calculated using normalized Shannon entropy. RNA-seq data were used for differential expression analysis.
 The sequencing panel achieved high coverage (median depth: 602× on Illumina; 291× on Ion Torrent) and uniformity. Molecular karyotyping revealed: mutations in 145 variants affecting 59 genes, with frequent alterations in KRAS, NRAS, ZFHX4, and TP53; translocations t(4;14), t(11;14), and t(14;16) were detected with 99% accuracy compared to FISH; and CNVs including del(1p), gain/amplification(1q), and del(17p) were accurately distinguished (PPV: 95%, sensitivity: 92%). Notably, NGS identified del(1p) events undetectable by CDKN2C FISH probes in 7% of cases and discriminated 1q gain vs amplification, potentially refining risk stratification.
 In parallel, RNA-seq revealed distinct expression clusters, including CCND1 and FGFR3-driven subgroups, and differential expression of SLAMF7, NF1, and ABCB4 associated with survival outcomes. IG/TR clonality, especially RNA-based IG clonality, was significantly associated with both overall and disease-free survival (p < 0.05).
 This dual-platform, dual-input targeted sequencing strategy enables a comprehensive molecular characterization of MM from limited material. It surpasses conventional techniques in detecting structural and numerical chromosomal abnormalities and provides actionable insights into clonal architecture and gene expression with prognostic significance. These findings support the integration of DNA/RNA-based NGS panels into routine MM diagnostic workflows to guide risk-adapted treatment strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67577b2c15159cac0263f5ee5152e9e90515eba0" target='_blank'>
              Comprehensive DNA/RNA-based targeted sequencing for molecular karyotyping and clonality assessment in newly diagnosed multiple myeloma patients
              </a>
            </td>
          <td>
            J. M. Rosa-Rosa, N. Buenache, I. Vazquez, R. Sánchez, C. Jiménez, L. Rosiñol, Norma C. Gutierrez, A. Oriol, , M. Blanchard, R. Ayala Diaz, R. Ríos, A. Sureda, M. H. Hernández García, J. de la Rubia, Joan Batista Blade Creixenti, M. Mateos, J. L. Lahuerta Palacios, J. San-Miguel, M. Calasanz, Ramón García-Sanz, J. Martínez-López
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="
 A major barrier in understanding adult diffuse gliomas lies in the inability to track tumor evolution in real time. To address this, we employed a novel method known as continuous lineage tracing, which integrates CRISPR/Cas9-based expressed DNA barcoding with single-cell RNA sequencing, enabling a phylogenetic approach to studying tumor development. Patient-derived glioma-initiating cell lines were engineered with expressed barcodes targeted by CRISPR/Cas9 and implanted into mice to create intracranial xenografts. Tumors underwent single-cell RNA sequencing; expressed barcodes were used to infer clonal relationships, and transcriptomic profiles enabled cell state classification. Phylogenetic lineage trees were reconstructed using lineage inference algorithms to characterize cell fitness, expansion, and plasticity. Our analysis uncovered extensive intra-clonal cell state heterogeneity, indicating active phenotype switching prior to therapy. We identified consistent transcriptional programs associated with tumor engraftment and in vivo clonal advantage. Lineage tracing revealed gene expression signatures linked to key phenotypes: fitness, enriched for neural-mesenchymal and injury-response pathways; expansion, associated with RNA splicing; and plasticity, correlated with cell cycle and DNA repair programs. Glioma stem cells appeared to span a transcriptional continuum from undifferentiated, high-fitness states to more differentiated, low-fitness states, with high-fitness cells potentially representing transitional phenotypes. We then validated these findings in a cohort of 185 surgically resected gliomas with matched bulk RNA and proteomic data. Phylogenetic gene signatures differed markedly between IDH-mutant and IDH-wildtype tumors. When stratified by tumor type, both fitness and expansion signatures were significantly associated with overall survival in GBM (median 12.6 months, p=0.041) and oligodendrogliomas (median 66 months, p=0.027). This study demonstrates the utility of continuous lineage tracing to reconstruct tumor evolution and identify transcriptional programs linked to tumor growth and prognosis. Our approach provides a powerful framework for dissecting glioma biology and identifying potential therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93d67c2559a40c369b3a39bc8fe0aad62196164a" target='_blank'>
              STEM-24. SINGLE CELL CRISPR/CAS-9 LINEAGE TRACING REVEALS EVOLUTIONARY FITNESS, EXPANSION AND PLASTICITY AXES IN ADULT DIFFUSE GLIOMAS
              </a>
            </td>
          <td>
            A. Ajisebutu, Xiaojun Fan, Chloe Gui, Jeff Liu, Vikas Patil, A. Landry, Y. Ellenbogen, L. Yefet, Phooja Persaud, F. Nassiri, G. Zadeh
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Accurate detection of oncogenic gene fusions is becoming increasingly important given the availability of highly effective targeted therapies. However, their identification in clinical practice remains challenging due to the rarity of individual events, diversity of partner genes, and variability of breakpoint locations. Conventional approaches such as immunohistochemistry (IHC) and fluorescence in situ hybridization (FISH) lack multiplexing capacity and demonstrate variable sensitivity and specificity, while direct identification of fusion transcripts in whole-transcriptome sequencing (RNA-seq) profiles provides broader applicability but limited sensitivity, as fusion junctions are frequently supported by a minimal number of reads or even no reads at all. In this study, a novel approach was employed to accurately detect clinically actionable RET (REarranged during Transfection) fusions. This approach entailed the measurement of the imbalance in RNA-seq read coverage of potential fusion oncogenes at their 3′ and 5′ exons. A total of 1327 experimental solid tumor RNA-seq profiles were screened, including 154 non-small cell lung cancer and 221 thyroid cancer samples. The RET status was validated in 78 selected cases by targeted NGS and Sanger sequencing. An analysis of the coverage imbalance was conducted, which enabled the accurate discrimination between true and false positive RET fusions. This approach outperformed other methods and yielded 100% sensitivity and specificity with optimized thresholds. The findings were validated using an independent cohort of 79 thyroid cancer cases, confirming the reliability of the results. Among the 18 RET fusion-positive samples, one was identified as an extremely rare case (RUFY3::RET), and two were determined to be novel fusions (FN1::RET, PPP1R21::RET). The findings of this study demonstrate that exon coverage imbalance analysis serves as a robust complement to computational RNA-seq analysis pipelines for the detection of clinically relevant RET fusions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7e82c8e2a86b2423abf1721592b4c4f068cbddf" target='_blank'>
              Accurate RET Fusion Detection in Solid Tumors Using RNA Sequencing Coverage Imbalance Analysis
              </a>
            </td>
          <td>
            Ivan Gaziev, Anna Khristichenko, Daniil V. Luppov, M. Suntsova, E. Bondarenko, Maria Reinberg, Alina Matrosova, N. Khilal, M. Sorokin, M. Sekacheva, E. Poddubskaya, Anton Buzdin, G. Zakharova
          </td>
          <td>2025-11-22</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Pancreatic cancer, recognized as a refractory tumor, has an overall survival rate of less than 10%, and its mortality rate continues to rise. Due to the low immune activity induced by its unique tumor microenvironment, pancreatic cancer is classified as a “cold” tumor and is insensitive to current immunotherapies. However, little is still known about the identification and functional mechanisms of key regulatory molecules in the formation of “cold” tumors. In this study, we identified the pancreatic ductal cell niche through single-cell sequencing. By calculating CNV scores using inferCNV to distinguish malignant from non-malignant cells and analyzing differences in transcriptional levels, we constructed a 15-gene model. The inhibitory effect of UBE2H on the immune microenvironment was studied through single-cell sequencing analysis, including the inhibition of T cell function, impairment of antigen presentation in macrophages and neutrophils, and activation of neutrophil extracellular traps (NETs). Our results revealed the presence of a UBE2H+ cell population within the pancreatic ductal cell niche, primarily observed in pancreatic cancer samples. Notably, single-cell sequencing analysis of UBE2H overexpression revealed an enhanced T cell PD-L1 interaction axis and a downregulation of antigen presentation pathways in macrophages and neutrophils (antigen processing and presentation, macrophage activation, and neutrophil activation). Further in vivo experiments confirmed that high UBE2H expression promotes tumor progression, leading to increased T cell exhaustion (PD1) and decreased activation (CD69, GZMB, IFNR) in the immune microenvironment, impaired antigen presentation of macrophages and neutrophils (H2Kb, I-Ab, I-A/E), and increased neutrophil extracellular trap formation (MPO, NE, CITH3). This study revealed the presence of a UBE2H+ cell population within the pancreatic ductal cell niche and analyzed the inhibition of UBE2H on the immune microenvironment of pancreatic cancer through single-cell sequencing and in vivo experiments, providing important clues for the formation of “cold” tumors in pancreatic cancer and opening new directions for exploring new treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49ecc8251593af0fcef99c0ebc48949b3c7fba9f" target='_blank'>
              Integrated scRNA-seq and transcriptome analyses uncover the effects of UBE2H on the immune microenvironment regulation in pancreatic cancer
              </a>
            </td>
          <td>
            Fuxin Huang, Zhongyan Zhang, Jike Fang, Yue Chen, Jinhui Wei, Quanzhang Li, Chuanzhao Zhang, Shanzhou Huang, B. Hou
          </td>
          <td>2025-11-24</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Resistance to chemotherapy, targeted agents, and particularly immunotherapy remains the principal challenge in the management of gastrointestinal malignancies. This review aims to comprehensively delineate the molecular and microenvironmental drivers of resistance, with emphasis on mechanisms impacting immunotherapy response, and evaluate emerging, mechanism−guided interventions (including immunotherapeutic combinations) for precision therapy. We first examine intrinsic mechanisms—including drug−target alterations, dysregulated drug metabolism and efflux, hyperactivation of DNA damage repair pathways, and epigenetic remodeling—and extrinsic influences stemming from the tumor microenvironment and extracellular matrix remodeling. We then highlight epithelial–mesenchymal transition (EMT) as a critical nexus that integrates stromal cues with cell−intrinsic survival programs, thereby promoting drug efflux and immune evasion. Next, we discuss how single−cell and spatial omics, liquid biopsy, patient−derived organoids, and AI−enabled analytics facilitate subclone−level mapping of resistance networks and real−time tracking of clonal evolution. Finally, we review mechanism−based strategies—including KRAS G12C inhibitors, efflux−pump antagonists, apoptosis reactivators, and epigenetic/autophagy modulators—and propose an integrated, multimodal regimen leveraging immunotherapy where appropriate, informed by real-time drug sensitivity data (e.g., from liquid biopsy), dynamic biomarkers and AI−driven optimization to overcome resistance and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aebd9ae366f20576b8816fc8c01cf3154ec38755" target='_blank'>
              Integrated molecular and microenvironmental drivers of drug resistance in gastrointestinal cancers: mechanisms, immunotherapy challenges, and precision strategies
              </a>
            </td>
          <td>
            Heng Xu, Jiaan Lu, Jiangying Wu, Kangling Zhang, Xuancheng Zhou, Ziqi Gao, Jingqi Feng, Ziye Zhuang, Xiaolin Zhong
          </td>
          <td>2025-10-28</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Plasticity, or the ability to rapidly and reversibly change phenotypes, may help explain how a single progenitor cell eventually generates a tumor with many different cell phenotypes. Normal colon plasticity is characterized by a conserved and broadly permissive epigenome, where expression and phenotype are determined by the microenvironment instead of epigenetic remodeling. To determine whether this stem-like plasticity is retained during progression, gene expression was measured with spatial transcriptomics and compared with gene-level DNA methylation in two colorectal cancers (CRCs). Like normal colon, genes that were differentially expressed between regions, subclones, and phenotypes (superficial, invasive, and metastatic) tended to have lower DNA methylation variability. We propose a quantitative signal of plasticity that correlates gene epigenetic variability with gene expression variability. In this framework, negative correlation implies phenotypic plasticity, as more variably expressed genes tend to have less epigenetic variability. We verify the presence of this signal in multiple external single-cell RNA-Seq datasets, in both normal colon and CRC samples. Therefore, the plasticity of normal colon appears to be retained during progression. A CRC progenitor with a preconfigured plastic phenotype is poised for rapid growth because it expresses, as needed, transcripts required for progression with minimal epigenetic remodeling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86cc1dfed608d8a0144edef3180374f9ebee40c0" target='_blank'>
              Epigenetic conservation infers that colorectal cancer progenitors retain the phenotypic plasticity of normal colon
              </a>
            </td>
          <td>
            Kelly Street, Yifan Zhang, Kimberly Siegmund, Darryl Shibata
          </td>
          <td>2025-11-20</td>
          <td>Scientific Reports</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="Simple Summary Medulloblastoma is the most common malignant brain tumor in children and includes several molecular subtypes with distinct clinical outcomes. Because genetic mutations are relatively rare in this tumor, our study focused on understanding how epigenetic regulation—the chemical and structural changes that control gene activity without altering DNA—contributes to disease progression. Using data from large patient cohorts and single-cell sequencing, we identified specific epigenetic factors that distinguish tumor subtypes and correlate with prognosis. We also developed an epigenetic score that predicts patient survival and is particularly elevated in aggressive Sonic Hedgehog (SHH) tumors. These results suggest that dysregulated epigenetic programs affecting RNA metabolism and cell proliferation play a key role in medulloblastoma aggressiveness, highlighting new potential targets for therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/733b0f36e6a075919ebc25ace84fd8a8035e0b6e" target='_blank'>
              Single-Cell Heterogeneity of Epigenetic Factor Regulation Deciphers Alteration of RNA Metabolism During Proliferative SHH-Medulloblastoma
              </a>
            </td>
          <td>
            R. Francés, Jenny Bonifacio-Mundaca, Íñigo Casafont, Christophe Desterke, Jorge Mata-Garrido
          </td>
          <td>2025-10-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Acute myeloid leukemia (AML) is an aggressive hematological malignancy with 5-year survival rate of less than 32%, mainly due to treatment failure and disease relapse. Thus, a systematic framework that allows us to monitor treatment response is urgently needed. Previously, we applied a state-transition modeling framework to represent AML disease evolution as messenger RNA (mRNA) and microRNA (miRNA) transcriptome trajectories in their respective state-spaces, partitioning samples into health, transition, and leukemic states based on mathematically defined critical points. These state-spaces were constructed using time-series RNA-seq data collected from a murine model of AML which mimics the fusion geneCBFB::MYH11 (CM) created by chromosome 16 inversion.
 To test the hypothesis that response to chemotherapy could be predicted by mRNA and miRNA transcriptome dynamics using a state-transition framework, we performed an experiment and mathematical analysis of longitudinal sample collections and multiomic profiling of peripheral blood from CM knock-in mouse model (Cbfb56M/+/Mx1-Cre) before, during, and after chemotherapy, as follows. After detection of overt AML (circulating cKit+ > 20%), CM mice (n=9) were treated with a “5+3” combination of cytarabine (50mg/kg/day, 5 days) and daunorubicin (1.5mg/kg/day, 3 days), modeling the standard “7+3” chemotherapy. Peripheral blood samples were collected weekly and were subjected to bulk RNA-seq and miRNA-seq. We used PCA to project the samples into a reference state-space constructed using untreated CM mice and C57BL/6 WT controls. The mRNA and miRNA transcriptome trajectories were first analyzed independently and then together to form a 2D multiomic state-space.
 After chemotherapy, we observed movement of both mRNA and miRNA transcriptomes from a leukemia state back towards a healthy state, before relapsing back to overt AML. Interestingly, we observed a desynchronization of mRNA and miRNA transcriptome dynamics post-chemotherapy. Specifically, the mRNA transcriptome immediately moved from a leukemia state to a health state, with an average peak response at week 4 (n=7; before week 4, n=2), however, a delayed response of 2 or more weeks was observed in the miRNA transcriptome trajectories, which continued to move towards a disease state post-treatment before moving towards a health state and reached a maximum response at later time-points (4 weeks, n=3; 5 weeks, n=2; 6 weeks and after, n=4). These desynchronized dynamics post-chemotherapy were enabled by the state-transition critical points and appeared more evident when visualized in a 2D multiomic state-space.
 To identify genes, miRNAs, and pathways altered by chemotherapy, we performed differential gene expression and gene set enrichment analyses. These analyses revealed that metabolic pathways such as OXPHOS, fatty acid metabolism and glycolysis are downregulated after chemotherapy and upregulated during relapse (p<0.05).
 We also performed a weighted gene co-expression network analysis on mRNAs and miRNAs to identify features with similar expression dynamics. We found a set of 34 miRNAs which were significantly upregulated and responsible for the delayed response of the miRNA transcriptome dynamics post-chemotherapy. Out of these 34 miRNAs, 23 are found in the DLK1-DIO3 imprinted region, located on chromosomes 14q32 in humans and 12qF1 in mice, which has been associated with stress response, pathogenesis of APL, solid tumors, and type 2 diabetes. These miRNAs have not previously been associated with effects of standard-of-care chemotherapy in AML. The co-localization of these miRNAs to a region of the genome suggests a potential explanation for genome instability and chemotherapy resistance.
 In summary, this study provides a longitudinal multiomic characterization of genomic instability and desynchronization of patterns of mRNA and miRNA expression induced by standard-of-care “7+3” therapy in AML, guided by mathematical analysis and critical points.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d8bb9687ac3e42f4c8f48f1349be6b29eb33446" target='_blank'>
              From theory to therapy: Predicting response to chemotherapy with multiomic longitudinal profiling of peripheral blood in AML
              </a>
            </td>
          <td>
            Jennifer Rangel Ambriz, Ziang Chen, Yu-Hsuan Fu, D. Frankhouser, Denis O’Meally, S. Branciamore, Jihyun Irizarry, Bin Zhang, Lianjun Zhang, Ying-Chieh Chen, Guido Marcucci, Russell C. Rockne, Ya-Huei Kuo
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eab80b92dbc650636e5eb2f387901a782d6b51b0" target='_blank'>
              Synergistic inhibition of CHK1 and MUS81 to combat replication stress resistance in high-risk neuroblastoma.
              </a>
            </td>
          <td>
            E. Hilgert, Christophe Van Neste, Sarah-Lee Bekaert, F. Martens, Suzanne Vanhauwaert, E. Sanders, P. Verstraelen, Martijn Risseeuw, E. M. Westerhout, Mark A Grannetia, Serge Van Calenbergh, Nadine Van Roy, W. D. De Vos, Rob W F Wolthuis, Frank Speleman, L. Depestel, K. Durinck
          </td>
          <td>2025-11-27</td>
          <td>Scientific reports</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Antibiotic resistance (AR) has long been interpreted through the lens of genetic mutations and horizontal gene transfer. Yet, mounting evidence suggests that epigenetic regulation, including DNA and RNA methylation, histone-like proteins, and small non-coding RNAs, plays a similarly critical role in bacterial adaptability. These reversible modifications reshape gene expression without altering the DNA sequence, enabling transient resistance, phenotypic heterogeneity, and biofilm persistence under antimicrobial stress. Advances in single-molecule sequencing and methylome mapping have uncovered diverse DNA methyltransferase systems that coordinate virulence, efflux, and stress responses. Such epigenetic circuits allow pathogens to survive antibiotic exposure, then revert to susceptibility once pressure subsides, complicating clinical treatment. Parallel advances in CRISPR-based technologies now enable direct manipulation of these regulatory layers. CRISPR interference (CRISPRi) and catalytically inactive dCas9-fused methyltransferases can silence or reactivate genes in a programmable, non-mutational manner, offering a new route to reverse resistance or sensitize pathogens. Integrating methylomic data with transcriptomic and proteomic profiles further reveals how epigenetic plasticity sustains antimicrobial tolerance across environments. This review traces the continuum from natural bacterial methylomes to engineered CRISPR-mediated epigenetic editing, outlining how this emerging interface could redefine antibiotic stewardship. Understanding and targeting these reversible, heritable mechanisms opens the door to precision antimicrobial strategies that restore the effectiveness of existing drugs while curbing the evolution of resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5548701c56565e9287ebf40d194a1cd7f296a9ed" target='_blank'>
              From Methylomes to CRISPR Epigenetic Editing: New Paths in Antibiotic Resistance
              </a>
            </td>
          <td>
            Nada M. Nass, Kawther A. Zaher
          </td>
          <td>2025-12-10</td>
          <td>Pathogens</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="KRAS and BRAF mutations are critical oncogenic drivers in colorectal cancer (CRC), with distinct molecular and clinical implications. To uncover genes differentially expressed between these mutations, we compared KRAS G12D- and BRAF V600E-mutated CRC cell lines. Protein-protein interaction (PPI) network construction, pathway enrichment, survival analysis, and drug target screening were performed to identify therapeutic opportunities. Ten hub genes—TNF, IL1B, FN1, EGF, IFI44L, EPSTI1, AHR, COL20A1, CDH1, and SOX9—were identified as critical in KRAS-driven CRC. Enrichment analysis highlighted immune and inflammatory pathways, including “SARS-CoV-2 Signaling” and “Macrophage Stimulating Protein Signaling,” as well as cellular processes like “Positive Regulation of PI3K Signaling”. IL1B was the only hub gene significantly associated with overall survival, suggesting its role as a favorable prognostic marker. Drug screening identified selective inhibitors such as Canakinumab and Rilonacept targeting IL1B, with docking studies revealing the strongest interaction for Omeprazole with AHR, followed by Tapinarof with AHR and Donepezil with IL1B. This study sheds light on the molecular mechanisms of KRAS-mutated CRC, emphasizing IL1B as a prognostic marker. Among the identified therapeutic agents, Omeprazole demonstrated the strongest interaction with AHR, suggesting a potential for repurposing in CRC treatment, while IL1B-targeting inhibitors such as Canakinumab emerged as selective candidates for modulating tumor-promoting inflammation. Novel genes such as EPSTI1, COL20A1, CDH1, and SOX9 warrant further investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/347d7ffedc4eee87a046791d3b66c644b633f639" target='_blank'>
              A systems biology approach to identify key targets in KRAS/BRAF-mutated colorectal cancer
              </a>
            </td>
          <td>
            Shiva Shiravi, Negar Mottaghi-Dastjerdi, Behzad Shahbazi, Nahid Ahmadi, Mohammad Soltany-Rezaee-Rad, Abozar Ghorbani, Hamed Montazeri
          </td>
          <td>2025-11-19</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract Background Since its inception, RNA sequencing has been pivotal in studying differential gene expression. Despite its extensive results in large-scale oncological studies, differential expression predominantly reflects a response to cancer. Therefore, we introduce differential allelic dispersion (AD) as a more effective measure. AD highlights consistent differences in expression between the 2 alleles of a gene that is, unlike cis-expression quantitative trait loci, independent of normal genetic variation. Such differences can, for example, arise from prevalent copy number alterations or epimutations occurring in the original cancer cell, which are mitotically expanded during cancer growth, making increased AD a marker for allele-specific dysregulation in early carcinogenesis. Findings We present the maelstRom R/C++ software package that enables (differential) AD analysis solely requiring large-scale RNA sequencing data. Using the The Cancer Genome Atlas renal clear cell carcinoma cohort as a case study, we successfully benchmark maelstRom’s AD modeling using known copy number alterations. We also detect increased AD for loci featuring normal random monoallelic expression, including the X chromosome, but demonstrate minimal interference with cancer-specific AD detection. Finally, we identify early dysregulated genes (e.g., FBP1, CCDC8, ECHS1, CLDN7) and pathways in renal cancer, often related to metabolism (e.g., pentose phosphate pathway). Strikingly, many of these genes are known causal contributors to renal carcinogenesis. Conclusions Differential AD clearly indicates early dysregulation in renal cancer, complementing basic differential expression analysis in cancer transcriptomics. AD is also relevant to study random monoallelic expression and may equally detect allele-specific (dys)regulation during early development or in noncancer diseases. maelstRom is available as an open-source software package at github.com/Biobix/maelstRom.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/457e775646c46e5a03c9c6239d832825d07a63f0" target='_blank'>
              Population-level allelic dispersion modeling by maelstRom yields genome-wide maps of allele-specific dysregulation during early carcinogenesis
              </a>
            </td>
          <td>
            Cedric Stroobandt, Louis Coussement, T. Goovaerts, Femke De Graeve, Jeroen Galle, Wim Van Criekinge, Tim De Meyer
          </td>
          <td>2025-01-06</td>
          <td>GigaScience</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) is a highly aggressive malignancy characterized by early metastasis and poor prognosis due to the limited efficacy of current treatments. Although initially responsive to chemotherapy and radiotherapy, the majority of patients with SCLC develop resistance within a year, often succumbing to distant metastases. Historically, SCLC was considered a homogeneous disease, primarily driven by the deletion or inactivation of key tumor suppressor genes TP53 and RB1. However, recent advancements in genomics and single-cell sequencing have identified distinct molecular subtypes of SCLC, derived from studies on cell lines, animal models, and tumor tissues. The tumor’s complexity, marked by the coexistence of multiple dynamic subtypes, contributes to its pronounced heterogeneity. Notably, different subpopulations exhibit a complex spatial relationship characterized by both mutual exclusion and coexistence. Temporally, SCLC exhibits the ability to undergo subtype transformations through various molecular mechanisms, underscoring the tumor’s plasticity and offering novel perspectives for personalized treatment approaches. This review synthesizes recent discoveries regarding SCLC subtype classification, intratumor heterogeneity, plasticity-related signaling pathways, immune landscape, and emerging therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/907fb7c9dba1e9795c5cd6f2373dce198db54231" target='_blank'>
              Intratumoral heterogeneity and potential treatment strategies in small cell lung cancer
              </a>
            </td>
          <td>
            Yunke Yang, Nana Liu, Jiaji Wu, Qingzhe Wu, Jiangnan Zhao, Yueli Shi, Mo Shen, Zhiyong Xu, Yueli Shi, Jianghao Yu, Peng Yi, Jin Cheng, Junhui Sun, Yun Xu, Kai Wang
          </td>
          <td>2025-10-27</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 Radiation therapy is a crucial component of the standard of care for newly diagnosed glioblastoma patients. However, the majority of tumors inevitably recur, most within the prior radiation field. Despite its broad clinical application, the molecular mechanisms that mediate response to radiation therapy in glioblastoma remain poorly understood. To address this knowledge gap, we systematically profiled the radiation sensitivity of four glioblastoma cell lines (T98G, SF295, LN18, DBTRG) and four patient-derived neurospheres (BT112, BT145, BT359, CPDM0095). The neurospheres were characterized at the genomic, transcriptomic and clinical levels. These models exhibited a spectrum of responses to radiation, with LD50s ranging from 1.5 to 8 Gy. To identify molecular factors that sensitize glioblastoma cells to radiation, we conducted a genome-wide CRISPR inactivation screen in T98G cells treated with or without a sub-lethal dose of radiation. Cells were transduced with the Inzolia multiplex CRISPR/Cas12a knockout library, targeting 19,687 single genes, 4,435 paralog pairs, 376 paralog triples, and 100 paralog quads. Cells were then divided into a control condition and an irradiated condition (3 Gy). After 18 days in culture, genomic DNA was harvested, and the relative abundance of each guide RNA array was compared between the two conditions. Targets preferentially depleted in the irradiated condition included known radiation sensitizers such as ATM and PRKDC, validating the screen’s performance. Gene set enrichment analysis (KEGG) of the top 200 hits revealed enrichment of DNA repair pathways, including homologous recombination and non-homologous end joining, as well as pathways related to the spliceosome, cell cycle, endocytosis, ubiquitin-mediated proteolysis, and pyrimidine metabolism. Notably, three components of the translesion synthesis pathway (REV1, REV3L, MAD2L2), not previously implicated in radiation response, also emerged as top candidates. A mini-pool library of the top 300 hits was subsequently generated and is being used for validation in additional glioblastoma cell lines and neurospheres. These findings provide a foundation for uncovering novel mechanisms of radiation response and nominate new targets for potential radiosensitization strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15dea4ab418adfb3586fa12497a41869a83e604c" target='_blank'>
              DNAR-13. Uncovering novel mediators of radiation sensitivity in glioblastoma using functional genomic screening
              </a>
            </td>
          <td>
            E. Aquilanti, Vincent Bozinov, Matthew L. Meyerson
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The treatment of cancer has taken a remarkable new direction with the advent of innovative scientific and technological methods. Immunotherapy has revolutionized oncology by harnessing the body's immune system to identify and destroy cancerous cells, leading to long-term remissions in many cancer types. Precision medicine enhances treatment effectiveness by tailoring therapeutic interventions to individual patients based on genetic, environmental, and lifestyle factors, thereby reducing both efficacy and adverse effects. The development of liquid biopsy methods enables non-invasive, real-time cancer detection and monitoring through blood samples (circulating tumor DNA and other biomarkers), allowing for treatment adjustments based on early cancer diagnosis. Recent advancements in CRISPR-Cas9 gene editing have opened new avenues for correcting oncogenic mutations and developing gene-based treatments. The use of artificial intelligence (AI) in cancer diagnostics, prognostics, and drug discovery is becoming increasingly common, enhancing prediction accuracy and optimizing predictive treatment planning. Combination therapies have shown synergistic potential to boost therapeutic effects and overcome drug resistance mechanisms. Meanwhile, cancer metabolism studies aim to exploit the metabolic changes in cancerous cells for therapeutic benefit. Epigenetic therapeutics seek to reverse abnormal genomic developmental control by restoring normal cellular epigenetic patterns and functions. Additionally, research on microbiomes is uncovering the crucial role of gut and tissue microbiota in cancer initiation, progression, and therapy response, suggesting potential new prevention and treatment strategies. These interdisciplinary innovations represent a paradigm shift towards more effective, personalized, and sustainable cancer care, offering new hope for improved patient survival and quality of life.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b52ad1f09c9728280095d0d63e09d01947a45160" target='_blank'>
              A Review: Recent Advancements in treatment of Cancer Research
              </a>
            </td>
          <td>
            Rahul Kumar, Khalid Raza, Kamna Gupta, Komal Gupta, Lukman Ansari, Sneha Kumari, Md. Danish Shahid Raja, Manjar Ansari, Faizal Nawaz, Raja, Articles Info, IJPHI26102810292025
          </td>
          <td>2025-10-31</td>
          <td>International Journal of Pharmaceutical and Healthcare Innovation</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Uncontrolled proliferation, resistance to apoptosis, inability to maintain genome integrity, and, recently, epigenetic reprogramming are all hallmarks of cancer. A number of gene expression and cell signaling networks control these-often-interconnected processes, while the study of their deregulation is in the forefront of cancer research for decades. Here we present data from cells and patients indicating that KMT2C, one of the most frequently mutated proteins in solid malignancies, is involved in all these processes. Its loss, a bad prognosis marker in bladder cancer, is associated with activation of the PI3K/PDK/AKT oncogenic/antiapoptotic axis, and tolerance to DNA damage during cell cycle progression. On the other hand, these cells suffer from mitotic stress that can be therapeutically exploited. Treatment with a PLK1 inhibitor showed high efficacy in vivo, and was associated with mitotic catastrophe and cellular senescence, providing evidence that targeting genes that promote mitotic progression could be a promising therapeutic approach in the subset of tumors with KMT2C loss.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a45d418115a05c2f3fb12ff53ed7255c53656e4" target='_blank'>
              KMT2C inactivation leads to PTEN downregulation and tolerance to DNA damage during cell cycle progression
              </a>
            </td>
          <td>
            T. Rampias, Andreas Goutas, Dimitris Karagiannis, Z. Kanaki, Antigoni Makri, L. Hoxhallari, Fotini Ε. Koukouzeli, Varvara Paraskevopoulou, Dimitra Tsouraki, Nikolaos Paschalidis, M. Avgeris, Andreas Scorilas, A. Klinakis
          </td>
          <td>2025-10-28</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="


 Acute myeloid leukemia (AML) progresses through the sequential accumulation of somatic mutations, with signaling transducer mutation typically preceding the acquisition of epigenetic alterations. However, recent single-cell DNA sequencing (scDNA-seq) studies have identified cases that do not follow this canonical mutational hierarchy. The prevalence of these non-classical mutation orders, and the biological mechanisms by which mutation timing drives phenotypic heterogeneity remain unclear.
 To address this, we analyzed publicly available scDNA-seq data from >150 newly diagnosed normal karyotype AML patients to reconstruct their evolutionary trajectories (Morita et al., 2020; Miles et al., 2020). We observed that 78% of the signaling mutation events—particularly FLT3-ITD, NRAS, and PTPN11—were terminal subclonal events in branch evolution. Surprisingly, 22% of the signaling mutations serve as initiating events, following a linear evolutionary trajectory and patients harboring these initiating signaling mutations exhibited significantly higher white blood cell and blast counts. These data suggest that the timing of signaling mutation acquisition is strongly linked to the resulting clonal evolution pattern, with FLT3-ITD emerging as the most prominent driver among all signaling events.
 To investigate the pathogenic mechanisms of initiating FLT3-ITD and the characteristics of the resultant AML, we therefore performed multimodal profiling (scDNA+protein-seq, scRNA-seq and patient-derived xenograft models) on 16 FLT3-ITD positive AML cases. Eight cases had FLT3-ITD as the initiating mutation (Initiating FLT3-ITD AML, IFA), and eight as subclonal events (Subclonal FLT3-ITD AML, SFA).
 Comparing tumor cell proportions identified by scRNA-seq with the mutant clone fractions from scDNA-seq in the same patient, we confirmed that leukemic transformation preceded acquisition of FLT3-ITD in SFA. Integrated genotype-immunophenotype analysis of paired samples—adjusted for co-mutations—revealed that initiating FLT3-ITD drove an expansion of hematopoietic stem and progenitor cells, whereas subclonal FLT3-ITD did not. Bayesian modeling further showed that initiating FLT3-ITD selectively upregulated leukemia stem cell (LSC) markers CD69 and CD123, resulting in a significantly larger CD123+CD69+ population (74% vs 35%, P=0.04). Consistently, in PDX models, IFA samples exhibited higher engraftment rates (5/8 vs 3/8). ScRNA-seq further revealed expansion of multipotent progenitor (MPP) and lymphoid-primed multipotent progenitor (LMPP) populations (29% vs 13%, P=0.04) and significant enrichment of LSC transcriptional signatures in IFA compared to SFA. Unsupervised clustering identified progenitor-enriched clusters in IFA characterized by downregulation of G2M checkpoint and E2F target pathways, indicative of a quiescent, stem-like state. Unexpectedly, canonical FLT3-ITD downstream pathways showed no significant differences between IFA, SFA, and FLT3-WT AML patients. Instead, in IFA, we observed significant upregulation of Hedgehog pathway genes (GLI2, ALDH1A1), known regulators of cell cycle and LSC quiescence.
 We observed frequent emergence of chr13q (FLT3 locus) uniparental disomy (UPD) in Initiating FLT3-ITD AML patients (15/23). Notably, we found the acquisition or expansion of UPD clones in IFA PDX models. Inferred from scRNA-seq, these UPD clones displayed increased proliferative capacity without losing LSC traits, potentially via RB1 inactivation. Drug sensitivity testing in PDX models revealed that IFA samples responded to chemotherapy and gilteritinib with distinct patterns, with gilteritinib selectively reducing leukemic blasts and inducing differentiation. However, single-cell genotyping of residual cells showed a marked rise in UPD clone post-gilteritinib. Longitudinal follow-up of these models confirmed that relapse was driven by the expansion of UPD clones. These data suggested chr13q is a required evolutionary event in the development and drug resistant progression of IFA.
 Our study demonstrates that initiating FLT3-ITD mutations reprograms leukemic cells to a quiescent, stem-like state, enforcing a linear evolutionary pattern, compared to subclonal FLT3-ITD. Clonal dominance is ensured by non-canonical FLT3 signaling and late-emerging chr13q UPD. These findings underscore the critical role of clonal timing in shaping AML biology and highlight new therapeutic vulnerabilities in initiating FLT3-ITD-driven AML.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c74975f6b634c7f7ca88b37dc7ce75122c37169" target='_blank'>
              Clonal timing of FLT3-ITD dictates evolutionary trajectory and phenotypic heterogeneity in acute myeloid leukemia
              </a>
            </td>
          <td>
            Xu-Dong He, Mengfang Xia, Ruoyang Shao, Qifa Liu, Qian-Fei Wang
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Gene expression regulation is a stochastic process that can be modified by mutations not only in a deterministic manner but also in a probabilistic way, for example by changing the extent of cell-to-cell variability in gene expression (also called “gene expression noise”) without necessarily changing the expected (mean) expression level. Such mutations can act either in cis (perturbing expression noise at their own locus) or in trans (perturbing expression noise of a gene located elsewhere in the genome). Although systematic studies have successfully analyzed the properties of cis-acting mutations modulating gene expression noise, less is known about the type of genetic changes that may alter expression noise in trans. Here, we applied genetic mapping on yeast strains (Saccharomyces cerevisiae) generated by random mutagenesis and identified three mutations (in chs1, yme2 and msh1 genes) that changed the expression noise of a reporter gene in the nuclear genome regulated by the yeast TDH3 promoter. Each of these mutations affected either extrinsic noise (variability due to cell-specific factors), intrinsic noise (inherent variability within each cell), or both. Surprisingly, all three mutations targeted proteins located outside of the nucleus: Yme2 and Msh1 being involved in the maintenance of mitochondrial genome integrity, and Chs1 being necessary for the repair of cell-wall defects in freshly-born daughter cells. Our results reveal that mitochondrial state can modulate the extent of intrinsic expression noise of eukaryotic nuclear genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3c52bc11c53e00beefbe9f3e33a641b467b3ecb" target='_blank'>
              Trans- acting mutations reveal non-nuclear modulators of both intrinsic and extrinsic gene expression noise in a eukaryote
              </a>
            </td>
          <td>
            Nena Martin, Julie Kleine-Schultjann, Quentin Boussau, Agnès Dumont, Hélène Duplus-Bottin, Laurent Modolo, G. Yvert, P. Wittkopp, F. Duveau
          </td>
          <td>2025-11-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="


 Background: We conducted an open-label, randomized, multicenter trial (NCT06496308) enrolling previously untreated patients with intermediate- or high-risk mantle-cell lymphoma (MCL) who were ineligible for autologous stem-cell transplantation (Order Trail).
 Aims: Using single-cell RNA-seq, we will dissect the microenvironmental mechanisms of resistance in responders versus non-responders, identify biomarkers for efficacy, and characterize the tumor and microenvironmental features of blastoid, refractory patients.
 Methods: Single-cell RNA sequencing (scRNA-seq) were performed on 33 enrolled patients with available samples (25 classical, 6 blastoid, and 2 pleomorphic), yielding 370,392 high-quality cells.
 Results: UMAP clustering identified 21 distinct cell populations, which were further annotated into 13 B cell subclusters, 5 T cell subclusters, 1 myeloid cell population and 1 epithelial population, based on the canonical markers, BCR/TCR profiling, and functional signatures.
 RNA velocity analysis revealed that MCL tumor cells originated from the MCL-Precursor populations and subsequently differentiated into heterogeneous subclusters, including two patient-unique subclusters (Patient1-unique B cells and Patient21-unique B cells). Cell cycle analysis showed elevated S-phase fractions in MCL-6, MCL-7, and MCL-8 subclusters. BCR repertoire analysis revealed that Memory B-like and Plasma cell-like subclusters were enriched in monoclonal B cells, with IgH heavy chains expressing IGHG/IGHA, indicating that these two subclusters have undergone class switching and were closer to normal B cells.
 The immunue microenvironment characterization identified 10 T-cell subclusters, including three CD4+ subclusters, three CD8+ subclusters, 1 NKT population, and 3 TOX-high DNT populations. Concurrently, the cytotoxic score, exhausted score and regulatory score for each T cell subclusters were calculated. Myeloid compartments were predominantly composed of DCs, M1/M2 macrophages, neutrophils and mast cells.
 Subsequently, we compared the characteristics of tumor B cell and immune microenvironment among classical, blastic, and pleomorphic MCL. For B cells, classical MCL showed predominant MCL-3 and MCL-4 B-cell populations, while blastoid cases exhibited MCL-7 expansion. Pleomorphic samples demonstrated exceptional heterogeneity with patient-specific clusters. Pathway analysis showed that cell cycle-related pathways were significantly activated in blastoid B-cells, while pleomorphic cases exhibited B cell differentiation and stemness maintenance signatures. Regarding the immune microenvironment, classical MCL presented higher proportion of CD4+ naïve T cells, while blastoid/pleomorphic subtypes displayed elevated proportions of CD8+ CTLs compared to the classical subtype. Gene Ontology analysis revealed that the activated genes of T cells in blastic MCL were mainly enriched in cytokine secretion pathways, contrasting with the enriched immune-regulation pathways in pleomorphic cases. Myeloid compartments in pleomorphic cases exhibited distinct proportional differences compared to classical and blastic subtypes, with significantly increased mast cells and neutrophils, and reduced macrophages and DCs.
 To investigate the underlying molecular mechanisms between patients with different treatment responses, we compared the proportions of the cell populations between patients achieving CR/PR and those with PD/SD. Compared to CR/PR responders, PD/SD patients had higher MCL-Precursor, MCL-4, DNT-2/3 proportions, but lower MCL-1/5 frequncies, alongside increased DCs and decreased neutrophils and mast cells. Pathway enrichment analysis demonstrated that upregulated genes in B cells from the PD/SD group were significantly enriched in cell cycle-related pathways, compared to the CR/PR group. Additionally, MCL-Precursor of PD/SD responders showed activation of the NF-κB and JAK-STAT signaling pathways, suggesting stronger malignant proliferation and earlier malignant transformation of tumor cells in patients with poor prognosis.
 Summary and conclusion: This is a preliminary micro-environment-based efficacy analysis derived from the ORDER study. We will further interrogate the RNA transcriptomes of responders and non-responders—integrated with single-cell data—to uncover resistance mechanisms and identify novel therapeutic targets for progressive disease.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06e501ffe60ee00560e8204d50b839bbe699bd32" target='_blank'>
              Single-cell transcriptomic profiling reveals tumor-intrinsic heterogeneity and immune-microenvironment dynamics in the order trial for mantle-cell lymphoma
              </a>
            </td>
          <td>
            Lu Jiang, Xiaozhong Shen, Weili Zhao, Li Wang
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d26c14dd48a6b62469af22170a5f8f8158260be3" target='_blank'>
              Non-cell-autonomous mechanisms of tumor initiation and relapse by chromosomal instability
              </a>
            </td>
          <td>
            Simona Lafirenze, Bastiaan van Gerwen, Ajit I. Quirindongo, Aniek Janssen, B. Etemad, Pim W. Toonen, W.H.M. Hoevenaar, Natasja Costermans, S. Youssef, A. de Bruin, L. Brosens, N. Jelluma, G. Kops
          </td>
          <td>2025-11-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="


 Background:
 Chronic lymphocytic leukemia (CLL) is a heterogeneous B-cell malignancy characterized by clonal expansion and immune dysfunction. Although peripheral blood (PB) is frequently used for genomic and transcriptomic profiling, it may not fully capture the biological activity of the disease. The lymph node (LN) microenvironment plays a central role in promoting malignant cell survival, proliferation, and immune evasion. Genomic alterations such as TP53, NOTCH1, and SF3B1 mutations are associated with adverse outcomes, but it remains unclear how these mutations interact with the local immune contexts across compartments. We investigated how clonal architecture and immune composition differ between PB and LN, and how these features relate to evolutionary dynamics.
 Methods:
 We performed whole exome sequencing (WES) and bulk RNA sequencing on paired LN biopsies and CD19⁺-purified PB samples from 26 patients with CLL. We selected samples from patients with known CLL driver mutations. Somatic variants, copy number alterations, and clonal architecture were inferred using SuperFreq, and cancer cell fraction (CCF) was used to define clonal distribution. Variants were filtered to include only those with >3 alternate reads, >20 total reads, a variant allele frequency (VAF) >0.02, and classified as non-synonymous mutations. CCF was considered equivalent to VAF if >0.6, otherwise CCF was calculated as VAF * 2. CCF was adjusted by dividing CCF by the ancestral or founding clone CCF as a proxy for tumor cell fraction. Clones were considered compartmentalized if the absolute CCF difference between LN and PB exceeded 0.25. Exome alteration was calculated by summing the length of exonic targets falling in copy-number segments with an absolute log2 copy-ratio >0.2 and dividing by the total captured exome length. LN immune composition was estimated from bulk RNA-seq data using CIBERSORTx.
 Results:
 Of the 26 patients, 12 were treatment-naive and 18 were IGHV unmutated. In the PB, mutational frequency was as follows: 46.1% NOTCH1, 42.3% TP53, and 26.9% SF3B1. Oncoplot analysis revealed the presence of multi-hit alterations and a predominance of missense and frameshift variants.
 Analysis of paired PB and LN samples revealed notable heterogeneity in clonal architecture. Using SuperFreq, we identified between 2-6 clones per patient (median: 4). The median number of subclones did not differ between LN and PB (4 vs. 4), treatment naive vs. relapsed/refractory status (4 vs. 4) or between IGHV unmutated vs mutated (4.5 vs. 4). We evaluated genomic instability and found significantly higher exome alteration in patients who were IGHV unmutated (P = .03). Clonal compartmentalization (CCF difference of |.25|) was observed in 18 of 26 (69%) patients, with 5 patients showing clonal evolution in the LN (28%), 6 in the PB (33%), and 7 with distinct clones evolving in both LN and PB (39%).
 SF3B1 mutations were common in clones with CCF trending higher in the LN compared to PB (median ΔCCF 0.16, P = .11). Interestingly, CCF of NOTCH1 mutations was balanced across compartments (median ΔCCF = -0.02; P = .36), and CCFof TP53 mutations was numerically slightly lower in LN than PB (median ΔCCF –0.05; P = .97).
 LN samples with stable clonal populations had higher proportions of CD4⁺ naïve T cells compared to those with evolving clones (P = .02), but no other significant changes in T cell populations were noted. Patients with NOTCH1 or SF3B1 mutations had reduced dendritic cell abundance in LN (P = .04 and P = .06, respectively) compared to those with TP53 mutations or no known driver mutations.
 Conclusions:
 This study demonstrates distinct clonal and immune features between PB and LN compartments in CLL, a heterogeneous clonal architecture across compartments. Differences in clonal composition across LN sites within individual patients suggest that localized immune contexts play a role in clonal selection. Evolving LN clones were more common in LNs with relative decreases in CD4⁺ T cell subsets, and the presence of common driver mutations was associated with reductions in dendritic cell content. Together these observations suggest an interplay of supportive vs restrictive roles of the LN TME in CLL progression. These findings highlight the added biological insight provided by LN sampling and suggest that PB alone may not fully capture the clonal heterogeneity of CLL.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/121dea9d74142004efb29c8ec3326a61ca23f8a3" target='_blank'>
              Role of the immune microenvironment in the clonal evolution of chronic lymphocytic leukemia
              </a>
            </td>
          <td>
            Victoria Rogness, Bharath Dileep Kumar, Chingiz Underbayev, Pia Nierman, Clare Sun, Christine Gruessner, A. Wiestner
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bfbe39384668fd86cf5e7bd9896fdd269b9c106" target='_blank'>
              Peptide-Based Therapeutic Platforms for Addressing Drug Resistance in Breast Cancer
              </a>
            </td>
          <td>
            Shreyasi Meur, Dipanjan Karati
          </td>
          <td>2025-10-23</td>
          <td>International Journal of Peptide Research and Therapeutics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Single-cell sequencing (SCS) techniques have the potential to offer precise insights into cellular diversity by revealing unique genetic and transcriptomic profiles at the single-cell level. This advanced technology has been used extensively in research, but it has still not translated to clinical diagnostics, despite its potential. SCS provides more accurate granular information about heterogeneous cell populations and the creation of personalized treatment strategies. However, the integration of SCS into clinical practice is challenging. This review discusses the potential of SCS technologies in improving clinical molecular diagnostics in various clinical areas including oncology, genetics and rare diseases, infectious diseases, and autoimmune disorders and inflammation. We review recent advances, current uses, integration challenges, and their contribution to the development of these fields. SCS provides significant opportunities in oncology to analyze tumor heterogeneity and develop personalized treatments. In autoimmune and rare diseases, SCS has helped to define novel biomarkers and understand complex immune pathways. SCS has also been used to understand pathogen diversity and host-pathogen interactions in the context of infectious diseases, leading to targeted therapeutic approaches. Despite this progress, there remain challenges in data analysis, standardization, and routine clinical application. The future of SCS in clinical molecular diagnostics is promising. Further technological and research developments in SCS are expected to increase the precision and personalization of medical diagnostics and treatment. To overcome current limitations, interdisciplinary cooperation and innovative approaches to data analysis are needed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18f80640c729e34d2cf6d8aaba32d154d9a103af" target='_blank'>
              Single-cell sequencing in molecular diagnostics: Transformative yet untapped potential
              </a>
            </td>
          <td>
            D. Elshiekh, Nour Farchoukh, Reema El Hassan, Sara Al-Marzooqi, W. Aldosari, Haissam Abou-Saleh, Amal Al-Haidose, H. Zayed, Atiyeh M. Abdallah
          </td>
          <td>2025-11-19</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains a formidable clinical challenge due to therapy resistance, metastasis, and relapse. Central to these processes are colorectal cancer stem cells (CCSCs), a dynamic subpopulation endowed with self-renewal capacity, plasticity, and heterogeneity. This review synthesizes recent advancements in understanding how CCSCs orchestrate tumor progression through intricate bidirectional crosstalk with the tumor immune microenvironment (TIME). We begin by elucidating the cellular origins of CCSCs, their profound intratumoral heterogeneity, and their remarkable phenotypic plasticity—driven by genetic, epigenetic, and metabolic reprogramming—which collectively serve as the root cause of therapeutic failure. A significant portion of our discussion is dedicated to deconstructing the immunosuppressive niche co-opted by CCSCs. We detail mechanisms of immune evasion and tolerance, highlighting how CCSCs modulate innate and adaptive immune cells—including NK cells, Tregs, dendritic cells, macrophages, neutrophils, and myeloid-derived suppressor cells—to foster an environment that supports stemness and suppresses cytotoxic attack. This reciprocal interaction forms a vicious cycle that perpetuates tumor survival and progression. Finally, we critically evaluate emerging therapeutic strategies that concurrently target CCSC-specific vulnerabilities and counteract immunosuppression. We explore the limitations of conventional chemotherapy and the promise of targeted therapies (e.g., Wnt inhibitors), immunotherapies (e.g., CAR-T, bispecific antibodies), and combination regimens designed to remodel the TIME and eradicate the CCSC reservoir. By integrating insights from single-cell omics and spatial biology, this review provides a comprehensive framework for overcoming therapy resistance and proposes novel precision medicine approaches for CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81b2fada714432c2d56f0013d3edefb668b69269" target='_blank'>
              Colorectal cancer stem cells crosstalk in tumor immune microenvironment and targeted therapeutic strategies
              </a>
            </td>
          <td>
            Yuxuan Duan, Anshu Li, Daojia Miao, Diaoyi Tan, Keshan Wang, Jian Shi
          </td>
          <td>2025-11-19</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Fms-like tyrosine kinase 3 (FLT3), a class III receptor tyrosine kinase essential for hematopoiesis, is a well-established oncogenic driver in acute myeloid leukemia (AML). Canonical internal tandem duplications (ITD) and tyrosine kinase domain (TKD) mutations inform prognosis and guide targeted therapy. Recent evidence highlights FLT3 as a critical oncogenic hub in acute lymphoblastic leukemia (ALL), where its alterations extend beyond ITD/TKD mutations to include non-canonical mutations with only partially explored functional implications. Moreover, recently discovered regulatory mechanisms, mostly acting on the FLT3 locus, drive FLT3 overexpression in ALL, including transcriptional regulation by rearranged ZNF384, epigenetic modifications, novel circular-RNA URAD::FLT3 fusions, and 13q12.2 deletions leading to enhancer hijacking and topologically associated domain (TAD)-boundary disruptions. The impact of these alterations on leukemogenesis and the possibility to target them in ALL subtypes is discussed here. Data from the Functional Omics Resource of Acute Lymphoblastic Leukemia (FORALL) across B- and T-ALL cell line subtypes drug screening, and from preclinical and clinical evidence reveals a variable efficacy in FLT3-mutated and FLT3-overexpressing ALL subtypes, supporting a molecularly guided treatment approach. Building on the success of FLT3 inhibitors in mutated AML and in light of the emerging results in patients lacking FLT3-ITD and in FLT3-like AML cases, presenting with a gene expression pattern similar to FLT3-mutated ones despite the absence of mutations, we discuss their potential in ALL and we consider novel therapeutic strategies, including new FLT3 inhibitors, antibody-based approaches, FLT3 CAR-T therapy, and synergistic drug combinations, such as FLT3 and BCL2 inhibition. These new insights reviewed here may redefine FLT3 as a pan-leukemic target, with ALL-specific activation mechanisms offering unique therapeutic windows. The implementation of FLT3 expression profiling and full-coding mutation screening in ALL (and in AML) diagnostics could unlock precision medicine approaches. By bridging the AML experience with ALL innovations, this review outlines a roadmap for FLT3-targeted therapies and combination strategies, underscoring the urgency of biomarker-driven clinical trials to optimize FLT3-directed interventions in acute leukemias.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b724e5e3a02c4639c7dca6522dee54c5f66974d4" target='_blank'>
              Novel molecular mechanisms of FLT3 deregulation: from the acute myeloid leukemia experience to therapeutic insights in acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            F. Lo Schiavo, C. Salvesi, M. Jandoubi, F. Pirini, J. Garbetta, G. Martinelli, G. Simonetti, A. Ferrari
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="


 Age-related clonal hematopoiesis (CH) and clonal cytopenia of undetermined significance (CCUS) represent pre-leukemic conditions associated with predisposition to hematological malignancies, particularly myeloid neoplasms. Frequently associated with mutations in epigenetic regulators, such as DNMT3A, TET2, and ASXL1, deregulated epigenomes are implicated in the aberrant gene expression involved in the pathogenesis of CH and CCUS, whose molecular mechanism, however, has not been investigated. The major difficulty lies in the technical limitation to simultaneously analyze gene mutation, transcriptome, and epigenetics in a small fraction of mutant cells in comparison with wild-type cells within the same patients, particularly in those cases where mutant cell fractions are very small.
 Aim/Methods In this study, we developed a novel platform that enabled simultaneous measurements of gene mutations and chromatin accessibility at the single-cell level based on the modified Mission Bio Tapestri system, which was applied to the analysis of 11 samples with TET2-mutated (TET2-MUT) CH and CCUS to investigate the impact of TET2 mutations on chromatin accessibility. In a subset of the samples (n = 3), we performed 10x Chromium single-cell ATAC+RNA-seq (Multiome) experiments and then predicted single-cell TET2 genotypes by employing ATAC peak accessibility-based logistic regression models trained on the paired Tapestri multimodal data, enabling the analysis of effects of TET2 mutation on gene regulation and transcriptional outputs at single-cell resolution.
 Results First, our platform was validated using a mixed cell line sample (HL-60 and SKM-1). ATAC data alone clearly separated the two cell lines, and genotyping accuracy for cell line-discriminative SNPs reached 94-99%, confirming robust integration of ATAC-seq and genotyping at the single-cell level.
 Next, to investigate the consequences of TET2 mutation on hematopoiesis and chromatin accessibility, we applied the platform to a total of 37,190 cells from TET2-MUT CH/CCUS patients, achieving successful genotyping in 83% (74–92%) of the cells. TET2-MUT cells exhibited a differentiation bias toward myeloid and plasmacytoid dendritic cell (pDC) lineages, with reduced mature lymphoid output. In line with this skewed differentiation, TET2-MUT hematopoietic stem/progenitor cells (HSPCs) showed an increased accessibility to the motifs of transcription factors (TFs) associated with myeloid/pDC differentiation (e.g., CEBPE, RUNX1, and IKZF1), compared to wild-type (TET2-WT) HSPCs, whereas TET2-MUT B cells had a reduced accessibility to the motifs of B cell development-associated TFs (e.g., BACH2, IRF4, and IRF8), compared with TET2-WT counterparts. In erythroblasts, TET2-MUT cells exhibited a decreased accessibility to erythroid maturation-related genes and an increased accessibility to genes involved in cell proliferation, including E2F and MYC targets, compared with TET2-WT cells, suggesting dysregulated maturation of TET2-MUT erythroblasts. Collectively, these results revealed TET2 mutation-driven alterations in chromatin accessibility affecting lineage maturation.
 Next, to investigate combined effects of TET2 mutation on TF-mediated gene regulation and transcriptional programs, we analyzed 10x Multiome data with predicted single-cell TET2 genotypes. We first analyzed TET2-MUT cell fractions within monocyte subsets and observed a depletion of mutants in the CD16(+) subset, suggesting impaired maturation of TET2-MUT cells. To uncover the underlying mechanisms, we then compared TF activity between genotypes, using SCENIC+, which integrates paired single-cell ATAC-seq and RNA-seq data to infer TF activity based on both accessibility at TF-binding enhancers and expression of their target genes. For both accessibility- and expression-based TF activities, TET2-MUT monocytes exhibited up-regulation of cell cycle-regulating TFs (e.g., TFDP1, TFDP2, and MYBL2) and down-regulation of key monocyte maturation-related TFs (e.g., CEBPB, CEBPD, and SPI1). These results suggest that TET2 mutation may impair monocyte development by epigenetically and transcriptionally suppressing maturation-related programs and sustaining proliferative states.
 ConclusionOur innovative single-cell multi-omics approaches revealed that TET2 mutations rewire chromatin accessibility and TF regulatory networks, leading to dysregulated lineage maturation and aberrant cell cycle control in CH/CCUS.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8043e74c1da9433c5daf135387ded1955d981b6" target='_blank'>
              Single-cell multiomic profiling of gene mutation, chromatin accessibility, and gene expression in TET2-mutant clonal hematopoiesis
              </a>
            </td>
          <td>
            Masanori Motomura, Y. Ochi, Y. Nannya, N. Kakiuchi, N. Sezaki, S. Kasahara, Tomoe Nakagawa, Lanying Zhao, Shuichi Matsuda, M. Nakagawa, Seishi Ogawa
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="


 Cyclin D1, encoded by CCND1, is a cell cycle regulator, overexpressed in ~15% of multiple myeloma (MM) cases via CCND1::IGH super-enhancer juxtaposition through t(11;14). Cyclin D1 also plays a non-canonical role in transcriptional regulation, but in t(11;14) MM its function in gene regulation remains undefined. We investigated the global transcriptional and epigenomic consequences of CCND1overexpression in t(11;14) MM.
 To define a CCND1 signature, RNA-seq from two independent primary MM patient cohorts, CoMMpass and Mayo, was utilized to compare high CCND1 t(11;14) cases (≥300 transcripts-per-million [TPM], n=271) vs. low CCND1 hyperdiploid cases (<30 TPM, n=233). Two CCND1 knockout (KO) clones in t(11;14) MM U266B1 were also generated by CRISPR/Cas9, introducing a frameshift insertion (A) after chr11:69641478 (hg38) in exon 1. ATAC-seq and ChIP-seq (H3K4me3, H3K4me1, and H3K27ac) were performed in duplicates, and RNA-seq in triplicates in U266B1 wild-type (WT) and KO. Libraries were sequenced on Illumina NextSeq 2000 (2x51 bp for ChIP/ATAC, 2x150 bp for RNA-seq). Reads were mapped to hg38 by BWA (ChIP/ATAC) or STAR (RNA-seq), peaks called by MACS2 (FDR< 0.01), peak and gene raw counts estimated by featureCounts. Differential peaks (fold-change ≥1.5) and differentially expressed genes (fold-change ≥2) were identified by edgeR (FDR<0.05), including peaks detected in ≥2 samples and genes with TPM ≥0.1 in ≥1 sample. Interactions between promoters and distal regulatory regions were identified using publicly available chromatin interaction data in blood cell types. Pathway analysis was performed using Enrichr. Co-immunoprecipitation was performed in t(11;14) KMS12BM cells.
 Differential analysis comparing CCND1-high t(11;14) and CCND1-low MM cases in the two patient cohorts identified 21 upregulated and 21 downregulated genes associated with CCND1 expression which were also captured in the wild-typevs. CCND1 KO U266 cells. Enriched pathways (p<0.05) included Notch signaling, early/late estrogen response, epithelial-mesenchymal transition and reduced mitotic spindle and myogenesis. Dysregulated genes included those related to apoptosis, ubiquitin-proteosome, extracellular matrix, adhesion, and immune response, demonstrating a broader role for cyclin D1 in MM beyond cell cycle control. To explore the epigenomic basis of this transcriptional regulation, for each of the 42 differentially expressed genes we examined the changes of histone marks and chromatin accessibility in promoter and the interacting distal regulatory regions in the U266 KO model. Ten of the 21 genes upregulated in CCND1-high (downloaded in KO; e.g., argininosuccinate synthase 1 [ASS1], PMAIP1, MYEOV, CCR7, and DTX1) and 12 of the 21 genes downregulated in CCND1-high (upregulated in KO; e.g., UACA, RASAL2, PTPRZ1, PPT2, and IL23R) showed concordant changes in histone marks and/or chromatin accessibility, predominantly in distal regulatory regions, suggesting a chromatin-mediated mechanism of gene regulation by cyclin D1. The most prominent changes involved H3K27ac, a mark defining active enhancers. As H3K27ac is deposited by histone acetyltransferases CBP, we tested for the possible cyclin D1-CBP interaction. Co-IP confirmed this interaction, suggesting a role for this complex in chromatin remodeling and transcriptional control in t(11;14) MM.
 One gene notably upregulated in association with CCND1 is ASS1, which encodes a urea cycle enzyme responsible for endogenous arginine synthesis. Metabolomic studies demonstrated KO cells had elevated aspartate and citrulline and lowered arginine and ornithine compared to WT cells (p<0.01) demonstrating impaired urea cycle function. Further, KO cells displayed significantly reduced ASS1 protein expression, cell growth and S-phase entry compared to WT cells in arginine-replete conditions, with both WT and KO cell types further inhibited by arginine depletion (p<0.05). Together, a cyclin D1-associated gene signature was identified supporting its role in t(11;14) MM beyond the CDK4/6 mediated cell cycle control, including a role in metabolic plasticity and survival under arginine depleted conditions. These findings highlight cyclin D1 as a therapeutic vulnerability and open new avenues for personalized treatment strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5d2041ffc4d8646057f972b418e4b1a71a5a0c2" target='_blank'>
              Cyclin D1 beyond the cell cycle: A new role in t(11;14) multiple myeloma
              </a>
            </td>
          <td>
            Huihuang Yan, Aneta Mikulasova, Suganti Shivaram, Hongwei Tang, Stacey Lehman, Samuel Notch, Anderson Hans, Shulan Tian, Michael D Howe, Abiola Bolarinwa, Cinthya Mendoza Zepeda, Marcella Kaddoura, Rafael Fonseca, P. L. Bergsagel, E. Braggio, W. Gonsalves, S. V. Rajkumar, Shaji K Kumar, Francesco Maura, Daniel Rico, Brian Walker, L. Baughn
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Ovarian cancer is a lethal disease with high mortality due to late diagnosis, recurrence, and chemotherapy resistance. There is an urgent need for better diagnostic and therapeutic strategies. Recently, protein post-translational modifications (PTMs) have gained significant attention for their role in the onset, progression, and treatment of ovarian cancer. PTMs (including phosphorylation, ubiquitination, methylation, ADP-ribosylation, and others) significantly influence tumor cell proliferation, metastasis, and drug resistance by modulating cellular signal transduction, DNA repair mechanisms, and metabolic processes. PARP inhibitors block the active site of PARP1 in BRCA-mutant ovarian cancer, disrupting ADP-ribosylation, inducing apoptosis in cancer cells, and extending progression-free survival. However, the emergence of drug resistance, such as BRCA reversal mutations, and the insufficient specificity of targets remain significant limitations to therapeutic efficacy. Strategies targeting other modification pathways, such as ubiquitination and methylation, provide promising options for expanding treatments. Combination therapies, like integrating PARP inhibitors with chemotherapy or immunotherapy, and precision medicine also show potential to overcome therapeutic challenges. This article systematically examines the pivotal role of PTMs in ovarian cancer pathogenesis, outlines therapeutic strategies and associated challenges in targeting PTMs, and offers a comprehensive overview with strategic guidance for future research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7787d1c8ae8d6c5881a1febad8059703b970c76a" target='_blank'>
              Therapeutic targeting of post-translational modifications in ovarian cancer: mechanisms and clinical applications
              </a>
            </td>
          <td>
            Ling Ke, Ying Zhang
          </td>
          <td>2025-11-11</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is paradigmatic for therapeutic resistance driven by genetic heterogeneity, epigenetic plasticity and microenvironmental protection. Over the past decade, six targeted or pathway-directed small molecules—midostaurin, gilteritinib, quizartinib, ivosidenib, enasidenib, venetoclax and glasdegib—have changed frontline and relapsed/refractory (R/R) practice in genomically defined subgroups or in patients unfit for intensive chemotherapy. Yet primary refractoriness and early relapse remain common, frequently via adaptive rewiring of apoptotic dependencies, clonal evolution and differentiation resistance. Here we integrate mechanistic insights with clinical evidence to: (i) map resistance biology onto targetable nodes (apoptosis control; signalling kinases; chromatin/lineage programmes; RNA splicing; DNA-damage response; nuclear export; niche adhesion and innate immune evasion); (ii) summarise the clinical trajectory and current limits of approved and emerging small molecules (including menin and LSD1 inhibitors); (iii) propose rules for rational doublets and triplets that are biologically orthogonal yet clinically tolerable; (iv) outline a regulatory timeline for key AML small molecules; and (v) prioritise where drug development should go next, including next-generation BH3 toolkits, clonal-pressure-aware designs, minimal residual disease (MRD)–adapted trials and therapy guided by dynamic functional profiling. The review closes with cross-platform challenges—myelosuppression, infectious risk, resistance monitoring and trial design—and a pragmatic framework for moving beyond incrementalism toward durable control and cure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37f83b4532e166032da2b89eea2b4afa25aa7b8b" target='_blank'>
              Acute myeloid leukemia drug resistance: targetable nodes and the clinical trajectory of small-molecule inhibitors
              </a>
            </td>
          <td>
            Xixi Zhang, Yao Peng, Yina Tian, Shichao Chen, Yijiu Jia, Mengtong Liu, Li Zhang
          </td>
          <td>2025-11-19</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa756c05989530f16b2f4c4778c0bfc833ec875f" target='_blank'>
              Deep Learning links TP53 genotype to expression-defined transcriptional program in Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            M. Tsesmelis, Rebecca Andersson, Alba Ferrer Perez, Sara Montserrat Vazquez, M. Carolina Florian, M. Mulaw
          </td>
          <td>2025-11-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Cancer drug resistance, whether intrinsic or acquired, underlies most relapses and treatment failures. Reliable preclinical models are crucial to define resistance mechanisms and test counterstrategies. This overview article concisely compares three model classes: (1) clinical, patient‑derived xenografts that retain tumor heterogeneity and can mirror patient resistance; (2) induced‑resistance models produced by prolonged drug selection in vivo or in vitro that recapitulate tumor evolution under therapy; and (3) engineered isogenic cell lines that isolate specific resistance drivers. We summarize some key resistance mechanisms revealed and potential therapeutic approaches informed by these models, including rational combinations, mutation‑targeted inhibitors/degraders, efflux/epigenetic modulators, and immune-related combinations. Each model has trade‑offs, but integrating them accelerates mechanistic insight and translational drug development. This overview guides selection of preclinical models and design of strategies to overcome cancer drug resistance. © 2025 Wiley Periodicals LLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/329bb2e5a0a76225e84bcf3ca5e48cf0783cbcff" target='_blank'>
              Overview of Preclinical Drug-Resistant Tumor Models and Their Application in Drug Discovery.
              </a>
            </td>
          <td>
            Bin Li, Ting Ni, Dong Wang, Gaoxiang Liu, Wenting Shi, Zhixiang Zhang, Qingyang Gu, Jian Xiang
          </td>
          <td>2025-12-01</td>
          <td>Current protocols</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 While venetoclax (VEN) combined with hypomethylating agents (HMA) has greatly advanced outcomes for acute myeloid leukemia (AML), resistance remains a major clinical challenge. Prior studies revealed several resistance mechanisms, including monocytic differentiation and RAS/TP53 mutations, but the full spectrum of mechanisms is still elusive.
 To assess the cellular mechanisms underlying VEN resistance, we performed an integrative analysis using bulk RNA sequencing (RNA-seq) and single-cell multi-omics in samples collected from the AML patients treated with 10-day decitabine and venetoclax (DAC10+VEN) trial (NCT03404193). First, we performed RNA-seq on 81 baseline and 10 relapse samples. K-means clustering of baseline samples identified a cluster enriched with non-responders, marked by a higher frequency of KRAS mutations (31%, p=0.012) and the absence of NPM1 and IDH1/2 mutations. Using LASSO logistic regression, we developed a 25-gene VEN-resistance (“VEN-R”) score comprising genes associated with MAPK, NF-κB, and apoptosis pathways. The VEN-R score significantly distinguished treatment response to DAC10+VEN (p<0.0001) and was also elevated in relapse samples (p=0.00018), supporting its association with resistance. We further validated VEN-R in two external, ex vivo datasets, finding significantly higher VEN-R scores in VEN-resistant samples compared to VEN-sensitive samples (BeatAML: p<0.0001 and Hashimoto et al. (Nature Cancer 2021): p=0.0002).
 To study the association between cell composition and DAC10+VEN response, we performed computational deconvolution of the RNA-seq data using CIBERSORTx. Baseline samples had higher fractions of erythroid (p=0.026) and CD8 effector (p=0.034) cells in nonresponders, while hematopoietic stem/progenitor (p=0.0006) and late erythroid (p=0.032) cells were more abundant in responders. Furthermore, compared with baseline samples, relapse samples had decreased hematopoietic stem (p=0.018), lymphoid primed multipotent progenitor (p=0.028), and naïve B (p=0.013) cells, and increased early erythroid (p=0.048), late erythroid (p=0.020), and granulocyte monocyte progenitor (p=0.036) cells.
 To assess the relationship between cell composition and VEN resistance at the cellular level, we performed single-cell RNA sequencing (scRNA-seq) on 22 samples (117,540 cells; 6 baseline, 16 relapse) treated with DAC10+VEN. scRNA-seq results corroborated the enrichment of mature cells at relapse; specifically, we identified the enrichment of erythroid, monocytic, and cDC (“EMD”) populations and the loss of progenitor cells. BCL2-expressing cells were depleted at relapse, whereas cells expressing MCL1, BCL2L1, or BCL2A1 were enriched in monocytes, erythroid cells, and cDCs, respectively. The EMD cells also had one of the highest expression levels of the VEN-R score at the single-cell level.
 To examine whether phenotypic remodeling is driven by genetic changes, we then performed single-cell DNA-antibody sequencing (DAb-seq) on 75 longitudinal samples, including six baseline-relapse pairs. DAb-seq analysis substantiated the findings from scRNA-seq that relapse is marked by the loss of stem/progenitor cells and the enrichment of erythroid/monocytic cells. Phenotypic remodeling occurred in five of the six pairs. One monocytic shift was accompanied by the de novo acquisition of FLT3 and KRAS mutations, one erythroid shift by expansion of an NRAS-mutant subclone, and a second erythroid shift by loss of a GATA2-mutant subclone. In the remaining two cases, remodeling occurred without any detectable genetic alterations. Integrating scRNA-seq and DAb-seq from 22 relapse samples revealed four phenotypic patterns: erythroid-predominant (6 cases), monocytic-predominant (4), mixed (erythroid±monocytic±cDC; 8), and inconclusive (4).
 Finally, we examined whether baseline phenotypic diversity predicts relapse after DAC10+VEN. In the DAb-seq cohort, a higher Shannon diversity index was associated with a higher risk of not only relapse (p=0.043) but also poor response (p=0.057).
 In summary, DAC10+VEN resistance is frequently observed with the enrichment of mature AML cells. To our knowledge, this is the first study to implicate erythroid cells and cDCs as major contributors to VEN resistance in patient samples, extending prior ex vivo studies (Kuusanmäki et al., Blood 2023) to a clinical setting. Monitoring these phenotypes and combining VEN with agents that target both immature and mature clones may improve patient outcomes.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9212f63e36e106f299197c47a52ccce0eb0f7523" target='_blank'>
              Phenotypic heterogeneity and remodeling drive resistance to venetoclax treatment in acute myeloid leukemia
              </a>
            </td>
          <td>
            Y. J. Kim, Ken Furudate, C. Dinardo, A. Maiti, M. Swaminathan, Zongrui Li, Eva Ordoñez, O. Nwogbo, Minggao Liang, Yoriko Saito, Akiko Kaneko, Kiyomi Morita, Jairo Matthews, Hongli Tang, Erika J. Thompson, Tomoyuki Tanaka, H. Uryu, N. Daver, N. Pemmaraju, T. Kadia, F. Ravandi, N. Short, M. Ohanian, G. Borthakur, Nitin Jain, G. Issa, L. Masarova, S. Kornblau, W. Wierda, S. Loghavi, Sa A. Wang, G. Garcia-Manero, Hagop Kantarjian, E. Jabbour, C. Bueso-Ramos, Shanshan Pei, C. Jordan, A. Futreal, Fumihiko Ishikawa, M. Konopleva, Koichi Takahashi
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>141</td>
        </tr>

        <tr id="


 Acute myeloid leukemia (AML) with tandem duplications in the upstream binding transcription factor gene (UBTF-TD) has recently been described as a new AML entity in younger adults. The UBTF gene encodes the UBF protein, which facilitates transcription of 18S, 5.8S and 28S ribosomal RNAs (rRNAs) by forming a pre-initiation complex with RNA polymerase I and SL1, a selectivity factor required for rRNA transcription. In AML patients with UBTF-TD, these alterations frequently co-occur with WT1 mutations, internal tandem duplications of the FLT3 gene, and trisomy 8, but are mutually exclusive with other AML subtype-defining alterations such as NPM1 mutations and recurrent translocations. UBTF-TDs appear to be early clonal events associated with poor response to conventional chemotherapy, high variant allele frequencies, and stability throughout disease progression.
 As an emerging AML entity, the molecular mechanisms by which the UBTF-TD promotes leukemogenesis remain incompletely understood. HOXA and HOXB gene overexpression has been identified as a predictive biomarker of sensitivity to BCL-2 inhibition, while UBTF-TD AMLs have recently been shown to respond to menin inhibition. In addition, as the UBTF gene encodes a nucleolar protein and has a well-established key role in rRNA biogenesis and nucleoli formation, RNA polymerase I inhibition might be beneficial in treating AML with UBTF-TD. CX-5461 is an anticancer drug that selectively inhibits RNA polymerase I-mediated transcription by disrupting rRNA synthesis initiation, and has received Fast Track Designation from the FDA due to its promising therapeutic potential.
 In this study, we established xenograft models by injecting primary AML patient samples with UBTF-TD or UBTF wild-type into immunodeficient NSG mice. Following engraftment, mice were treated with vehicle, CX-5461, venetoclax plus azacitidine (VEN-AZA), or the combination of CX-5461 and VEN-AZA. The tumor burden was significantly reduced when CX-5461 was added to VEN-AZA. Flow cytometry revealed that CX-5461, alone or in combination with VEN-AZA, induced a near-complete shift toward a primitive CD34+ immunophenotype specifically in UBTF-TD PDX models, suggesting enhanced cellular plasticity. Single-cell sequencing (CITE-seq) of primary AML samples, including pre-treatment and relapse samples with UBTF-TD or UBTF wild-type controls, identified consistent transcriptional and surface marker dysregulation in UBTF-TD samples. This included downregulation of key cell cycle regulators (e.g., E2F targets and G2M checkpoint genes), consistent with reduced proliferation and a quiescent-like phenotype, alongside reduced TNF-α signaling. Analysis of paired diagnosis-relapse samples showed an increased stem-like cell population at relapse, mirroring the treatment-induced shift toward a primitive, stem-like state observed in PDX models. Further analyses to investigate the functional consequences of this pronounced primitive shift upon CX-5461 treatment are currently underway.
 In conclusion, our findings demonstrate that VEN-AZA in combination with CX-5461 reduces tumor burden in preclinical models and induces a pronounced shift toward a primitive, quiescent, stem-like phenotype in UBTF-TD AML, reflecting significant cellular plasticity. These data support combined targeting of BCL-2 and RNA polymerase I as a promising therapeutic strategy for AML patients harboring UBTF-TD.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18e603acb46cf19f2c9f2bb06e27afcbf13614df" target='_blank'>
              Acute myeloid leukemias with UBTF tandem duplications respond to combined BCL-2 and RNA polymerase I inhibition with increased stemness and a quiescent-like state
              </a>
            </td>
          <td>
            Patrick Stelmach, Nesrine Aroua, Dawn S Lin, E. Donato, Ann-Kathrin Eisfeld, C. Röllig, Christian Thiede, Andreas Trumpp
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="


 Background - Methods: Hundreds of cell lines have been examined by genome-sale CRISPR knockout (KO) studies to identify functional drivers of tumors, but only few lines have been systematically examined with CRISPR activation (CRISPRa). We reasoned that CRISPRa studies can identify genes whose activation may suppress tumor cell fitness and serve as negative regulators of tumor cell survival/proliferation (NRSPs). To this end, we performed genome-scale CRISPRa studies (dCas9-P65-HSF transcriptional activation system; Calabrese sgRNA library, typically 6 sgRNAs/gene) in 10 genotypically diverse multiple myeloma (MM) cell lines at multiple timepoints and compared results with similar publicly available or in-house CRISPRa studies in other cancers (n=17; including lymphoma, leukemia, NSCLC, melanoma, ovarian, prostate cancer; with up to 3 lines per tumor type).
 Results: Based on multiple quantitative metrics, hundreds of genes were recurrently identified as NRSPs (e.g. 500 NRSPs in 5 or more lines). Integrated analyses of our CRISPRa data with molecular annotation for the respective MM lines did not identify any obvious bias in the CRISPRa-based detection of NRSP that relates to e.g., DNA copy number status; chromatin accessibility; or baseline levels of transcript expression. Top recurrent and pronounced NRSPs included several known proapoptotic genes (e.g., death receptors; caspases; BH3 only Bcl-2 family members) or other tumor suppressive molecules (e.g. CDK inhibitors), providing reassuring positive controls. Moreover, many previously unrecognized NRSPs were identified including transcription factors (TFs)/cofactors; chromatin remodeling genes and epigenetic regulators; signaling molecules regulating survival/proliferation; RNA binding proteins; nucleic acid-sensing and antiviral restriction factors; metabolic regulators; E3 ligases and other proteostatic regulators; DNA repair genes; solute carrier transporters, among others. No correlation is currently observed between the pattern of NRSPs for individual MM cell lines and their respective molecular subtype (e.g., t(4;14), t(14;16), t(11;14), etc). Some clusters of NRSPs are shared between MM and non-MM lines; and others are more prominent in MM, e.g. certain plasma cell-related TFs or endoplasmic reticulum regulators. We validated the functional relevance of several NRSPs with individual sgRNAs for CRISPRa; using focused sgRNA libraries for CRISPRa in MM cells in vitro and in NSG mice in vivo. We also performed single cell RNA-seq of MM cells transduced with pools of sgRNAs (CROP-seq) or bulk RNA-seq of MM cells transduced with individual sgRNAs for select NRSPs. Many NRSPs have undetectable/low transcript levels, without recurrent deletions/mutations, in MM cell lines or patient-derived MM cells but are expressed in healthy and malignant cells from different non-hematopoietic tissues. Thus, the NRSP effect of these genes in MM cells may involve lineage-inappropriate activation of molecular programs that are critical for, or at least tolerated in, other lineages but suppress MM cell fitness. Among the top 200 NRSPs with recurrent expression in MM cells, analyses of MMRF CoMMpass data revealed downregulation of ≥1 NRSPs in ~50% of paired MM samples after relapse vs. before treatment, suggesting that suppression of such NRSP genes can contribute, alone or in concert, to enhanced MM cell fitness in clinical relapses. Some NRSPs defined by CRISPRa are potent dependencies by CRISPR KO, including the MM-preferential, lineage-defining, TF PRDM1. CROP-Seq and bulk RNA-seq of MM cells with CRISPRa of PRDM1 (vs. other NRSPs or control sgRNAs) revealed that PRDM1 overexpression is toxic to MM cells by suppressing other dependencies, including IRF4, another lineage-defining TF and master MM dependency.
 Conclusions: Collectively,our studies have identified a large collection of NRSP genes whose overexpression suppresses tumor cell fitness in MM, with several of them also being relevant for other hematologic neoplasias. Several canonical tumor suppressor genes are also identified by CRISPRa, but most NRSPs have not been previously examined for their potential to suppress tumor cell fitness in MM or other neoplasias. Our study highlights the concept that engineered overexpression of NRSPs has intriguing potential for therapeutic applications in MM and other hematologic neoplasias.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28bf8e777a2bb3a092a1281fdd85652dfe41d721" target='_blank'>
              Negative regulators of survival and proliferation (NRSPs) identified by CRISPR activation studies in multiple myeloma (MM) vs. other neoplasias: Therapeutic implications
              </a>
            </td>
          <td>
            O. Dashevsky, Ricardo de Matos Simoes, Huihui Tang, Torsten Steinbrunn, S. Yamano, Eleni Anastasakou, Benjamin G. Barwick, S. Gandolfi, Ryosuke Shirasaki, Vikas A. Gupta, Michal Sheffer, Jonathan Licht, Lawrence Boise, Matthew Oser, Constantine S Mitsiades
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="


 Therapy of AML patients remains challenging due to high rates of relapses even after intensive treatment. Relapses are mostly caused by residual chemoresistant leukemic stem cells (LSC). Long non-coding RNAs (lncRNA) have been shown to be important regulators of normal hematopoiesis and pathogenesis of leukemias. LSCs show a pattern of highly enriched lncRNAs. One of those highly abundant lncRNAs is WT1-AS, which was described as a tumor suppressive as well as oncogenic lncRNA in different solid malignancies. The high expression of WT1-AS in LSCs suggests an oncogenic role in AML. However, the function of WT1-AS in AML has not been elucidated yet.
 Since dysregulated growth is one key feature of malignant cells, the impact of WT1-AS on cell growth of AML cells was evaluated. Therefore, an shRNA-based WT1-AS knockdown (WT1-ASkd) was established in representative AML cell lines, and growth competition assays using RFP-labelling as readout were performed. Here, WT1-ASkd cells consistently exhibited a significant growth disadvantage compared to WT1-ASwildtype cells. To further investigate the cause of the observed growth disadvantage, cell cycle properties and apoptosis were analyzed. The knockdown of WT1-AS not only increased the proportion of apoptotic cells but also induced a cell cycle arrest in the G0/G1 phase of the cell cycle.
 To gain first mechanistic insight into the pathways underlying the induction of apoptosis and cell cycle arrest upon WT1-AS knockdown, comprehensive transcriptomic analyses were performed. WT1-ASkd cells exhibited upregulated expression of myeloid differentiation gene signatures alongside a downregulation of stemness-associated genes. Since terminal differentiation is accompanied by cell cycle exit and ultimately apoptosis, these findings provide a plausible explanation for the observed growth arrest and increased cell death.
 To recapitulate stemness and differentiation in AML, an in vivo competition assay using an AML patient-derived xenograft (PDX) mouse model was performed assessing the influence of WT1-AS knockdown on AML progression. PDX cells were transduced either with an shRNA eithertargeting WT1-AS or a scramble control, and subsequently transplanted in a ~1:1 RFP+/RFP- ratio into sublethally-conditioned NOD scid gamma (NSG) mice. After AML onset, mice were sacrificed and the percentage of human CD45+/RFP+ was quantified in the peripheral blood (PB), the bone marrow (BM), and the spleen, and compared between the WT1-ASkd and the scramble control group. The percentage of RFP+ cells significantly decreased within all three compartments in the WT1-ASkd group, whereas it remained relatively stable in the control cohort (PB 1% vs 37%, p<0.0001; BM 2% vs 31%, p=0.0053; spleen 8% vs 38%, p=0.009), implicating that WT1-AS knockdown reduces AML engraftment in this model. Notably, the remaining human CD45+/RFP+ cells in the WT-AS1kd PDX model predominantly expressed the myeloid differentiation marker CD11b, while the large proportion of human CD45+/RFP-cells were mostly CD11b-negative (68% vs 18% CD11b+; p=0.001) indicating a reintroduction of myeloid differentiation upon WT1-AS knockdown in vivo.
 To explore the clinical relevance of our findings, we investigated the impact of WT1-AS expression in the BEAT AML patient cohort focusing on patients with a normal karyotype. Here, the quartiles with the highest WT1-AS (WT1-AShigh) and lowest WT1-AS (WT1-ASlow) mRNA expression were compared. Consistent with our in vitro findings using the WT1-ASkd models, the WT1-ASlow group associates with several myeloid differentiation gene signatures (NES<-1.7, p<0.05). Furthermore, the LSC17 score, which reflects the presence of LSCs, is higher in the WT1-AShigh cohort underlining the more stemness-like phenotype of WT1-AShigh AML. Comparing the mutational profile of both groups, the WT1-AShigh AML patients exhibit mutations in NPM1 and WT1 more frequently, whereas the WT1-ASlow patients show significantly more secondary-type mutations (SRSF2, STAG2, ASXL1).
 In summary, our data identify WT1-AS as a key regulator of leukemic stemness and a suppressor of myeloid differentiation in AML. WT1-AS knockdown leads to reintroduction of myeloid differentiation in vitro and in vivo, which ultimately leads to cell cycle arrest and apoptosis. Our data further highlight the decisive impact of lncRNAs on the fate of AML cells and their potential as targets in AML treatment.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38d1527c5a2ed835dfeda263a8a0b1cc1a81025c" target='_blank'>
              The long non-coding RNA WT1-AS controls differentiation programs and leukemic cell fate in AML
              </a>
            </td>
          <td>
            Silvia Schäfer, Pascal Fichtel, Silke Brilloff, Sandra Küchler, Elahe Rahimian, Vida Kufrin, Ekaterina Longvinova, C. Ball, H. Glimm, Denis M. Schewe, M. von Bonin, M. Bornhäuser, A. Wurm, Marius Bill
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="


 Despite improved targeted therapies and chemotherapy regimens for acute myeloid leukemia (AML), durable remissions remain elusive for most patients. A major barrier to curative treatment is therapy-persistent leukemia cells (residual disease) that evade initial treatment and drive relapse. We hypothesized that these persistent cells harbor distinct biological features that enable them to survive initial therapy. To identify selective vulnerabilities, we conducted a genome-wide CRISPR interference (CRISPRi) screen in AML cells that survived treatment. In parallel, we established an in vivo patient-derived xenograft (PDX) model of residual disease that models clinical disease scenarios and enables transcriptomic profiling at key treatment stages.
 We first performed CRISPRi screening in CAL-1 cells, derived from BPDCN, a representative high-risk AML subtype that responds to most initial therapies but rapidly relapses. Two regimens were tested, venetoclax plus azacitidine (VEN/AZA) and tagraxofusp, each at its IC90 dose, to explore shared targets in residual disease. Our screen identified several expected pathways — pro-apoptotic genes such as BAX and NOXA as mediators of sensitivity to VEN/AZA, and CD123 and diphthamide biosynthesis genes as sensitizers to tagraxofusp. Notably, KH-type splicing regulatory protein (KHSRP) was the top gene promoting therapy persistence across both regimens. In validation experiments, KHSRP depletion substantially reduced residual leukemia viability across several regimens in multiple AML cell lines derived from other high-risk disease subtypes, achieving up to 10 times more residual cell elimination after initial treatment vs. controls.
 Using the in vivo PDX model, we defined three time points: pre-treatment, residual disease after VEN/AZA treatment, and relapse. Pre-treatment and relapse had >80% marrow burden, whereas residual disease was <20%. RNA-seq revealed that residual cells were transcriptionally distinct from both primary and relapsed leukemia. We then integrated PDX expression analysis with gene expression in AML cell lines after KHSRP depletion. This turned our focus to SLC23A1, which encodes the sodium-dependent vitamin C transporter 1, as it was the only gene significantly downregulated in residual disease and upregulated by KHSRP depletion. Mechanistically, we found that KHSRP promotes SLC23A1 mRNA decay via recruitment of the NEXT nuclear RNA degradation complex. We thus hypothesized that residual leukemia cells reduce SLC23A1 to resist chemotherapy, and that KHSRP depletion restores its expression, sensitizing cells to treatment. This model was supported by rescue experiments in which SLC23A1 knockdown abolished the therapy-enhancing effects of KHSRP depletion.
 Vitamin C has been explored as an anti-leukemic agent via promoting the conversion of 5-methylcytosine (5-mC) to 5-hydroxymethylcytosine (5-hmC), facilitating DNA demethylation, particularly in TET2 mutant cells. Given that the active, reduced form of vitamin C is unstable in standard culture media, we tested the role of uric acid, the other molecule transported by SLC23A1 and that is present in the media at stable physiologic concentrations via FBS supplementation. Similar to vitamin C, we found that soluble uric acid also enhanced 5-mC to 5-hmC conversion in AML cells. Furthermore, exogenous vitamin C and uric acid were significantly synergistic in reducing viability of KHSRP-deficient residual cells after VEN/AZA treatment. Moreover, TET2 re-expression in TET2-null AML cells mimicked the effects of KHSRP depletion in eliminating the residual population, with no additional effect observed upon further KHSRP knockdown. Thus, uric acid and vitamin C may serve as alternative TET enzyme cofactors that mediate the therapeutic effects of KHSRP depletion.
 Importantly, in the Cancer Dependency Map, only a few solid tumor cell models are strongly dependent on KHSRP, and our data showed that KHSRP depletion had minimal effects on untreated leukemia cells. In contrast, the impact of KHSRP depletion on eliminating therapy-persistent leukemia cells was chemotherapy dose-dependent, indicating a selective sensitivity in residual disease. We are currently evaluating the effects of KHSRP depletion in prolonging survival of two independent AML in vivo models treated with VEN/AZA. These data nominate KHSRP as a selective vulnerability in therapy-persistent AML and warrant the development of KHSRP inhibitors to target residual disease.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afceff0e3f46ce9daa03fb1c28ee8b20052c2b60" target='_blank'>
              Khsrp depletion selectively eliminates AML cells that persist after venetoclax/azacitidine therapy via modulating SLC23A1-transported TET enzyme cofactors
              </a>
            </td>
          <td>
            Qingyu Luo, Karley Whalen, Xiaowei Wu, Amanda L Fortune, Lin Zhang, Weiye Qian, Evangeline G. Raulston, Y. Nan, Christopher A G Booth, Kezhi Yan, David Root, John Doench, Andrew Lane
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 Older patients with acute myeloid leukemia (AML) have significantly worse response rates and overall survival compared to younger patients, even when matched for cytogenetic risk. This discrepancy has previously been explained by decreased tolerance of therapy and higher incidence of adverse risk disease. However, when these factors are taken into consideration there remains a higher incidence of primary refractory disease and relapse in older individuals, suggesting an underlying non-mutational resistance to therapy. Recent studies have implicated epigenomic disorganization from DNA damage repair as an important driver of cellular aging. The biological basis for age-related resistance and whether epigenomic disorganization contributes remains unclear.
 To investigate the impact of age on AML biology, we generated murine AML models by transducing hematopoietic stem and progenitor cells (HSPCs) from fetal (E13.5), young adult (8-10 weeks), and aged (>24 months) mice with retroviruses expressing MLL-ENL and NRasG12D. All models developed monocytic AML, confirmed by morphology and immunophenotyping, and were lethal upon transplantation into syngeneic recipients. Additionally, gene expression profile differences between young and old murine models were consistent with differences seen between younger and older AML patient samples (FDR q < 0.001) supporting their relevance to the human disease. We assessed in vitro sensitivity to menin inhibition, azacitidine, venetoclax, doxorubicin, and cytarabine. Whole exome sequencing (WES), bulk RNA-seq, single-cell RNA-seq (scRNA-seq), and proteomic profiling were performed to identify molecular correlates of resistance.
 AML cells derived from aged mice HSPCs were significantly more resistant to all tested therapies compared to fetal and young derived AML cells with marked resistance to cytarabine, doxorubicin and menin inhibition (p<0.001). WES did not reveal any recurrent mutations to explain this resistance. Transcriptomic and proteomic data were analyzed by KEGG pathway analysis revealing significant down regulation of several DNA damage repair pathways in older AML cells including base excision repair, homologous recombination and mismatch repair. In contrast, older AML cells had significantly increased expression of anti-apoptotic genes including multiple BCL2 family members. RNA and proteomics data demonstrated that younger AML cells had significantly increased expression of genes/proteins involved in chromatin and epigenome maintenance as identified by Gene Ontology terms (FDR q < 0.01) suggesting older AML cells may have a more disordered epigenome. Consistent with this, scRNA-seq revealed significantly increased transcriptional heterogeneity in aged AML cells compared to young and fetal models. This finding was corroborated in primary human AML samples when compared by age (p=0.0004). This suggests that AML cells derived from older HSPCs have a more disordered epigenome likely as the result of repeated rounds of DNA damage repair displacing epigenetic marks. To directly test if repeated rounds of DNA damage repair from a non-carcinogenic source would result in AML therapy resistance, we constitutively expressed the restriction endonuclease PPOI or GFP in fetal derived AML cells. After 12 weeks PPOI expressing fetal AML cells were significantly more resistant to cytarabine (p<0.0001) and doxorubicin (p<0.0001) compared to GFP expressing cells, mimicking the aged phenotype. They also appeared more aggressive when injected into syngeneic recipients.
 These findings support a unifying model of epigenetic based therapy resistance in AML, wherein aged HSPCs after multiple years of DNA damage repair, have a disordered epigenome and increased transcriptional heterogeneity. This heterogeneity is passed on to the resultant AML and enables survival of resistant subclones under therapeutic pressure, contributing to poor outcomes in older adults. This would also explain why therapy related and relapsed AML, where repeated DNA damage repair cycles have taken place, also have a worse outcome within the same cytogenetic risk category. Finally, it identifies a potential source of confounding in preclinical models that utilize enzymes that repeatedly induce DNA strand breaks like CAS9. Most importantly, it suggests that clinical trials data should be interpreted with extreme caution when considering patients outside the age range of those enrolled in the study.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/399dae455f6a2d1caf558c73e5130583d602ef95" target='_blank'>
              Age-associated non-mutational resistance in acute myeloid leukemia is driven by DNA damage-induced transcriptional heterogeneity
              </a>
            </td>
          <td>
            Jordon Burton, K. Pladna, Lance Miller, Birgit Schilling, Timothy Pardee
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Phenotypic plasticity plays a pivotal role in cancer, enabling tumor cells to adapt to environmental pressures and evade therapeutic interventions by transitioning between distinct cellular states. However, the contribution of phenotypic plasticity to adaptive drug resistance in glioblastoma (GBM), one of the most lethal of all cancers, remains poorly understood. In this study, we identify that GBM tumor-initiating cells resembling normal radial glia (RG), which occupy the apex of normal neurodevelopment, are driven by aberrant epidermal growth factor receptor (EGFR) signaling. Using a suite of patient-derived GBM models, we demonstrate through global proteomics and single-cell RNA sequencing that pharmacological inhibition of EGFR triggers a lineage transition toward neuronal and oligodendrocyte progenitor (OPC)-like states. This shift is accompanied by activation of oncogenic RAS-MAPK signaling – despite robust and durable inhibition of EGFR activation – and is further modulated by brain microenvironmental cues, including synaptic and calcium-mediated signaling programs. Dual inhibition of EGFR and RAS-MAPK with novel, tumor-selective small molecules blocks these phenotypic transitions and enhances GBM cell death in EGFR-mutated GBM models. To determine the subpopulation dynamics of RAS-MAPK signaling within GBM neurodevelopmental lineages, we develop DENALI (Dual-Expression Nuclear reporter of ERK Activity and Lineage Identity) – a novel, high-complexity barcoded lentiviral vector and integrative fluorescence reporter system. Using DENALI, we investigate the clonal mechanisms driving lineage plasticity in GBM following oncogenic EGFR inhibition and couple adaptive RAS signaling programs to the emergence of neuronal and OPC-like states under EGFRi therapy. Together, our findings establish neurodevelopmental lineage plasticity as a key driver of adaptive resistance in GBM and support dual-inhibition strategies to improve therapeutic outcomes in patients with GBM tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/099a2a811b34aed9bceff625fc38dcd958e60df8" target='_blank'>
              STEM-23. GBM NEURODEVELOPMENTAL LINEAGE PLASTICITY DRIVES RAPID ADAPTATION TO ONCOGENE TARGETED THERAPY
              </a>
            </td>
          <td>
            Dimitri Cadet, Toby Harris, Elisa Fazzari, Cassidy Andrasz, Nicholas A. Bayley, Thomas G. Graeber, T. Cloughesy, A. Bhaduri, D. Nathanson
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="


 Objectives: Acute myeloid leukemia (AML) is mostly driven by specific mutations that affect transcription factors and epigenetic regulation. While targeted therapies are being developed for few AML subtypes, most patients still rely on systemic genotoxic treatments to induce complete remission and/or to bridge to bone marrow transplantation. Hence, deeper understanding of underlying disease and therapeutic mechanisms is urgently needed. In recent years, agents targeting the DNA damage response - such as PARP inhibitors (PARPi) already exploited in solid tumours, have emerged as promising therapeutic agents also in AML. PARPi alone exhibits high efficacy for a subset of AML, such as those driven by AML1-ETO and RARα fusions, but is rather ineffective forMixed-Lineage Leukemia gene-rearranged (MLLr) AML. A major function of these genotoxic agents is to interfere with DNA replication resulting in replication fork collapse and excessive DNA damage. However, cancer cells may adapt damage tolerance mechanisms, including fork reversal and repriming, to potentially bypass the genotoxic insults. We recently reported that, upon stimuli-induced proliferation, hematopoietic stem and progenitor cells (HSPCs) experience drastic changes in DNA replication dynamics and use specific fork protection mechanisms to promote their repopulation potential. Hence, we hypothesized that AML derived from deregulated hyperproliferating HSPCs may also adapt their DNA replication process, which may offer novel vulnerabilities with therapeutic potential.
 Methods: We investigated DNA replication dynamics and replication-associated DNA damage in established human and mouse AML models, that express different oncofusion proteins and reportedly display differential sensitivity to PARPi. To this aim, we used single-cell and single-molecule assays (e.g. comet assays, quantitative image-based cytometry, DNA fiber spreading) under baseline conditions and upon treatment with PARPi or standard-of-care chemotherapeutics. We also combined these functional assays with inactivation or overexpression of key replication fork plasticity factors, and with cell survival, apoptosis and differentiation assays, to evaluate how specific mechanisms of fork plasticity contribute to replication dynamics and therapy response in different AML subtypes.
 Results: We found that all human and mouse AML lines exhibit remarkably fast replication fork progression, compared to control HSPCs. However, AML lines previously reported as PARPi-sensitive – i.e. NB-4 (PML-RARα) and Kasumi-1 (AML1-ETO) – rapidly respond to PARPi by further accelerating fork progression, and eventually experiencing fork breakage and S-phase specific DNA damage signaling. Remarkably, inactivation of the specialized fork restart helicase RECQ1 suppresses these phenotypes, linking PARPi response to replication fork reversal and restart. Conversely, we found that PrimPol-mediated DNA synthesis – an alternative DNA damage tolerance mechanism that competes with fork remodelling – is predominant in MLL-rearranged leukemia, i.e. an aggressive AML subtype associated with poor prognosis. Notably, overexpression of Primpol in PARPi-sensitive AML is sufficient to prevent fork breakage and PARPi-sensitivity in these cells, while Primpol downregulation sensitizes MLL-rearranged AML lines (e.g. MOLM-13, THP-1), inducing fork collapse and replication-associated breaks. Importantly, similar observations were collected upon AML standard-of-care treatments (e.g. AraC), suggesting that mechanisms underlying replication fork plasticity play a central role in governing AML response to genotoxic treatments.
 Conclusions: These findings shed light on key mechanisms of the replication stress response in AML and underscore their potential for clinical translation. In light of our data, fork plasticity factors may represent novel therapeutic targets and predictive biomarkers for AML patient stratification. This evidence encourages the development of inhibitory compounds to target specific replication fork transactions and thereby improve therapeutic outcomes.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee77d6b18642e1d784e7f8b45c03785c72394af4" target='_blank'>
              Replication fork remodeling is a therapeutic vulnerability in acute myeloid leukemia
              </a>
            </td>
          <td>
            Cyril Dördelmann, T. Fung, Tristan Gasparetto, Daniel González‐Acosta, Larissa Bomfim, Chi Wai Eric So, Massimo Lopes
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Myeloid/Lymphoid Neoplasms (MLN) with eosinophilia and PDGFRB rearrangements are rare but distinct hematologic malignancies driven by the constitutive activation of the PDGFRB tyrosine kinase through gene fusions. These neoplasms are sensitive to tyrosine kinase inhibitors (TKIs) such as imatinib, which often leads to rapid and durable molecular remissions. However, diagnostic challenges frequently arise from cryptic rearrangements, necessitating comprehensive molecular approaches. A 37-year-old male patient initially presented with pancytopenia and a splenic infarct; subsequent bone marrow findings were suggestive of a myeloid/lymphoid neoplasm. Initial conventional cytogenetic analysis and fluorescence in situ hybridization (FISH) identified a PDGFRB gene rearrangement but were unable to fully resolve the structural complexity of the underlying genomic alteration. Long-read sequencing helped resolve a complex three-way translocation involving chromosomes 5, 12, and 20, precisely defining the ETV6::PDGFRB fusion with base pair resolution, and identified the partner gene (KAT14) on chromosome 20p. Following the diagnosis, the patient was started on imatinib therapy and has since achieved clinical and hematological improvement. This case highlights the significant diagnostic utility of long-read sequencing in uncovering and characterizing cryptic and complex genomic rearrangements that are frequently missed by conventional methods. Accurate molecular characterization is critical for disease classification, guiding targeted therapeutic decisions, and ultimately improving patient outcomes in PDGFRB-rearranged neoplasms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41233da301b201fb4d885f11221a3ed668efdf06" target='_blank'>
              Long-read sequencing unmasks a cryptic three-way translocation resulting in an ETV6::PDGFRB fusion
              </a>
            </td>
          <td>
            J. Tripodi, Douglas Tremblay, Daiva Ahire, Vesna Najfeld
          </td>
          <td>2025-10-21</td>
          <td>Molecular Cytogenetics</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="


 BackgroundAcute myeloid leukemia (AML) remains a lethal hematologic malignancy characterized by clonal expansion of immature myeloid blasts and dismal long-term survival. Despite advances in genomics-guided therapy, relapse and chemoresistance are common, especially in older or refractory patients. Emerging evidence implicates metabolic reprogramming and bone marrow (BM) hypoxia as key drivers of leukemic stem cell (LSC) persistence and drug resistance. We previously identified FGF11—a member of the intracellular FGF subfamily—as markedly upregulated in primary AML blasts and cell lines. However, its mechanistic role in AML pathogenesis and therapeutic vulnerability remains undefined.MethodsWe analyzed FGF11 and HIF-1α expression in 173 de novo AML patient samples (TCGA-LAML and GTEx), 15 paired diagnosis/remission BM aspirates, and two AML cell lines (HL-60, HEL) using RNA-seq, qPCR, and immunohistochemistry. Subcellular localization and co-localization of FGF11 and HIF-1α were assessed by immunofluorescence and proximity ligation assays. Functional studies employed siRNA-mediated knockdown, and rhFGF11 supplementation. Seahorse metabolic flux analysis quantified glycolytic and oxidative phosphorylation (OXPHOS) rates. ResultsCompared with normal bone marrow, FGF11 mRNA was 4.3-fold higher in primary AML blasts (p<0.01) and predicted shorter event-free survival (EFS; HR=1.54, p=0.0028). Bioinformatic interrogation of the FGF11 promoter revealed hypomethylation at the CpG site cg00795277 that was tightly linked to adverse clinical outcomes, suggesting epigenetic activation of the locus.In HL-60 cells, immunofluorescence showed nuclear co-localization of FGF11 with HIF-1α, providing a spatial basis for functional interaction. Functional assays demonstrated that siRNA-mediated knockdown of FGF11 attenuated proliferation, enhanced apoptosis, and curtailed migration and invasion. Transcriptomic profiling of these cells disclosed down-regulation of HIF-1α signaling, alongside re-activation of mitochondrial and oxidative-phosphorylation programs, implying that FGF11 represses oxidative metabolism.To explore the reciprocal relationship, we exposed AML cells to CoCl2-induced hypoxia and observed a time-dependent increase in both FGF11 and HIF-1α mRNA and protein, confirming that HIF-1α can transcriptionally up-regulate FGF11. Proteomic analysis of hypoxic versus normoxic cells, annotated by COG classification, further enriched energy-metabolism pathway.Finally, exogenous rhFGF11 not only recapitulated the aggressive phenotype—enhancing migration and invasion—but also triggered a metabolic shift from oxidative phosphorylation to glycolysis. ConclusionsOur data uncover a novel FGF11–HIF-1α epigenetic-metabolic axis that orchestrates AML progression and chemoresistance through glycolytic reprogramming. Targeting this loop—either by disrupting FGF11/HIF-1α interaction or reversing metabolic addiction—represents a promising therapeutic strategy to overcome residual disease and improve outcomes in high-risk AML.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a40abdc907bf009e1fd38544864df761a97e31a" target='_blank'>
              FGF11–HIF-1α glycolytic epigenetic axis drives AML progression and chemoresistance via metabolic reprogramming
              </a>
            </td>
          <td>
            Hang Lv, Jiayao He, Zhuang Xin, Songfu Jiang, Xiaoying Man, Ruochen Wang, Rong Yang, Xiaoyu Zeng, Honglan Qian, Yifen Shi
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Small-cell lung carcinoma (SCLC) remains one of the most aggressive lung cancers and continues to pose a major challenge for precision oncology. Despite its morphological uniformity, SCLC exhibits marked molecular heterogeneity with recurrent, potentially targetable genomic alterations. Comprehensive profiling is often hindered by limited tissue availability and the need for rapid therapeutic intervention. We performed genomic profiling of 55 primary and metastatic SCLC samples using a 324-gene hybrid-capture next-generation sequencing panel. Consistent with prior reports, nearly all tumors exhibited biallelic TP53 and RB1 inactivation. Recurrent alterations involved the PI3K/Akt/mTOR pathway (62%), chromatin regulators (42%), and NOTCH signaling genes (15%). PTEN mutations were enriched in brain metastases. Frequent copy-number gains affected SOX2, NKX2-1, MYC-family genes, and CCNE1. Two novel recurrent amplifications of potential clinical significance were identified: TYRO3 (33%) and SDHA (13%). TYRO3, a TAM family receptor tyrosine kinase, and SDHA, a mitochondrial enzyme involved in succinate metabolism, may contribute to tumor progression and represent emerging therapeutic vulnerabilities. These findings underscore the genomic diversity of SCLC and highlight the potential utility of broad next-generation sequencing in uncovering new molecular targets for precision therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c257de36865932aa9b4ccc1af139b71a5ba02b7" target='_blank'>
              Comprehensive Genomic Profiling of Small-Cell Lung Cancer Reveals Frequent Potentially Targetable Alterations
              </a>
            </td>
          <td>
            Dániel Schmalz, Zoltán Krabóth, Veronika Czoma, Péter Urbán, Attila Gyenesei, István Ruzsics, Veronika Sárosi, Á. Boronkai, Emőke Papp, B. Kajtár
          </td>
          <td>2025-11-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Epigenetics encompasses heritable but reversible modifications of gene expression that occur without changes in the DNA sequence and involve mechanisms such as DNA and RNA methylation and histone modifications. These mechanisms modulate chromatin architecture, genome stability, and cellular responses to environmental stressors, and their dysregulation contributes to oncogenesis and cancer progression. In parallel, radiotherapy remains a cornerstone of cancer treatment; furthermore, ionizing radiation induces epigenetic modifications alongside direct DNA double-strand breaks and oxidative damage. Radiation-induced epigenetic changes, including global or locus-specific DNA methylation shifts (e.g., genes promoter CpG islets), histone acetylation and methylation imbalances, are increasingly recognized as key contributors to molecular radioresistance. These adaptive responses may enhance tumor cell survival, affect therapeutic efficacy, and promote metastasis. Understanding the interplay between radiation exposure and epigenetic remodeling opens new perspectives for precision oncology and diagnostics. Epigenetic biomarkers hold potential for predicting treatment response and prognosis, while epigenetic modifiers may sensitize tumors to radiation. This review summarizes current evidence on radiation-induced epigenetic mechanisms and evaluates their diagnostic, prognostic, and therapeutic implications in cancer management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18126342fcb0010fdf4cce9fa40fdeeb69a2ac9b" target='_blank'>
              A New Look at the Role of Radiation-Related Epigenetic Mechanisms in Diagnosis and Anticancer Therapies
              </a>
            </td>
          <td>
            A. Olichwier, Magdalena Bruzgo-Grzybko, Izabela Suwda Kalita, Natalia Bielicka, Ewa Chabielska, A. Gromotowicz-Poplawska
          </td>
          <td>2025-11-27</td>
          <td>Cells</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Cancer cells are distinguished from normal cells by their rapid rate of division. This high division rate can be explained by various factors, including unregulated cell cycle progression, which occurs when cancer cells bypass checkpoints; activation of growth signals in cancer cells; and override of growth suppressor factors, which generally regulate cell growth and division within physiological thresholds. Cancer cells bypass these controls due to genetic mutations or epigenetic changes. Additionally, changes in the tumor microenvironment (TME) can contribute to the accelerated growth and division of cancer cells. It is well established that cancer cells accumulate mutations more frequently than normal cells. This accumulation can, in part, explain their rapid rate of division. The cell cycle consists of four phases: G1, S, G2, and mitosis, with the first three phases categorized as interphase. When cancer cells experience division stress, their demand for nucleotide synthesis increases, which is essential for RNA and DNA synthesis. RNA synthesis primarily occurs during interphase, while DNA replication occurs in the S phase. One of the conserved enzymes involved in nucleotide synthesis is dihydrofolate reductase (DHFR). This enzyme’s role in purine and thymidylate synthesis is crucial in cancerous cells under conditions of division stress. Methotrexate, a well-known DHFR inhibitor, has been introduced for the treatment of cancers such as meningeal leukemia, lymphoma, and breast cancer. While effective in alleviating cancer symptoms, methotrexate can cause adverse effects, including hepatotoxicity, pulmonary complications, and renal impairment. Based on these considerations, we applied combinatorial studies, including molecular dynamics simulations alongside quantum mechanics, to design novel DHFR inhibitors for cancer cells using carbohydrate- and amino acid-based scaffolds. Additionally, our studies suggest that the designed inhibitors may exhibit fewer side effects than methotrexate.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/936636e4b144e2a81ac99e1bbcb66eabfd38e1da" target='_blank'>
              Combinatorial QM and MD in silico design of natural product-based DHFR inhibitors
              </a>
            </td>
          <td>
            Sepideh Kalhor, Amin Mohammad Shapouri, Alireza Fattahi
          </td>
          <td>2025-10-27</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Despite substantial advances in targeted therapies, chronic lymphocytic leukemia (CLL) remains incurable, underscoring the need for strategies to predict or even prevent its development. Low-count (LC-) and high-count (HC-) monoclonal B cell lymphocytosis (MBL) are precursor states of CLL, but the molecular events driving their progression remain poorly understood due to limited characterization of their epigenetic, transcriptional, and proteomic landscapes. To address this gap, we performed a multi-modal single cell (sc) analysis on 34 peripheral blood samples from individuals with LC-MBL (n=6; P1-P6), HC-MBL (n=7; P7-P13), and CLL (n=11; P14-P24), including 10 matched HC-MBL/CLL pairs (median follow-up: 1,926 days, range 916-2,859). We integrated transcriptomic (RNA-seq), epigenetic (ATAC-seq), and surface marker profiling (ASAP-seq) with B cell receptor (BCR) sequencing, mitochondrial DNA (mtDNA) lineage tracing, and copy number variation (CNV) analysis to dissect clonal relationships and cell states.
 We first mapped transcriptional and epigenetic states across physiologic B cells, LC-, HC-MBL and CLL using 301,853 scRNA-seq and 168,533 scATAC-seq profiles, which we annotated as MBL/CLL, B, T, NK cells or monocytes. While B cells retained conserved epigenetic profiles, MBL/CLL cells formed unique individual-specific gene expression and chromatin accessibility clusters, even at the stage of LC-MBL. These changes were consistent with CLL-like surface immunophenotypes in LC-MBL, such as overexpression of CD5 and CD200 or downregulation of CD49d, Integrin β7, CD73, and CD35. Single cell BCR analysis demonstrated that LC-MBL retained only small residual polyclonal B cell populations (median: 25% polyclonal; range: 3-95%), which was consistent with the low percentage of CD19+ CD5- non-MBL cells on flow cytometry (median: 21%; range: 0.3-91%). In sum, LC-MBL is characterized by heterogeneous gene regulatory and expression states with early clonal dominance.
 We next sought to identify CLL-associated core gene regulatory programs. On differential chromatin accessibility analysis, we found decreased accessibility of transcription factor (TF) motifs implicated in B cell development (i.e., EBF-1, BACH1/2, POU2F2), cell differentiation (AP-1), or chromatin remodeling (SMARCC1), but also increased TF activity in B cell anergy (NFAT family) or stem cell pluripotency (TCF4) shared across the cohort. These distinct profiles were already detectable in LC-MBL and remained stable throughout progression from HC-MBL to CLL. Analysis of transcriptional profiles mirrored these results with increased expression of pro-survival pathways (P2RX7, BCL2, PTPN22, LEF1) and loss of B cell identity (EBF1) or apoptosis regulation (FOS). Together, CLL-like epigenetic and transcriptional states are acquired as early as LC-MBL.
 Finally, we wondered what clonal dynamics drive the trajectory from physiologic B cells to MBL/CLL. Comparison of mtDNA mutations between physiologic and monoclonal B cells showed that 245 of 253 (97%) variants were shared. Amongst these, 7 were selectively enriched (i.e., 2435G>A and 3488T>C in P1; 7161G>A in P2; 13886T>C in P5) and 6 depleted (i.e., 8396A>G in P1; 9247G>A and 13453C>T in P2) in LC-MBL. Integration of mtDNA mutations with BCR sequences (nanoranger) and CNVs (numbat) validated these results. In one HC-MBL, targeted genotyping of 3,093 scATAC-seq profiles for SF3B1Y623C (GoT-ChA) revealed that physiologic B cells (1% SF3B1Y623C) were enriched for 16186C>T, while MBL (31% SF3B1Y623C) harbored 1412G>A and 4925C>T. Finally, we used mtDNA mutations to resolve subclonal architectures within MBL/CLL populations, identifying a median of 33 subclones per case (range 4–90). The majority of subclones (median 6 per case [73%], range 3–87) persisted throughout HC-MBL/CLL progression, consistent with clonal proliferation rather than selection. Together, shared mtDNA mutations between physiologic and monoclonal B cells analysis suggest a common ancestry, while their persistence demonstrates clonal stability through the HC-MBL/CLL transition.
 In summary, the HC-MBL/CLL transition is characterized by subclonal, epigenetic, and transcriptomic stability, indicating that key transformative events occur earlier, prior to the development of LC-MBL. As a heterogeneous premalignant state, LC-MBL warrants further investigation to define the earliest molecular steps in CLL initiation and identify potential targets for prevention.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9eb7edd9688e487af91ab3d3a8cdcd621992a8c7" target='_blank'>
              Acquisition of CLL-like epigenetic and transcriptomic states in low-count MBL
              </a>
            </td>
          <td>
            Anja Rathgeber, Stacey M. Fernandes, Adi Nagler, Shuqiang Li, David Dorfman, Lars Bullinger, M. Davids, Jennifer R. Brown, K. Livak, Leif Ludwig, Catherine J. Wu, L. Penter
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a39534227ca18263af64802892a94b5211231b6b" target='_blank'>
              Failure of DNA repair leads to the tumorigenic transformation of a regenerating blastema in Drosophila imaginal discs
              </a>
            </td>
          <td>
            Tanroop Kaur, Ada Repiso, Ying Liu, Paula Doria, Joan Urgell, Luis Rodriguez-Escudero, Katerina Karkali, Yanhui Hu, N. Perrimon, E. Martín-Blanco
          </td>
          <td>2025-10-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6deb8278c1abd256d3cb1730399ddf6f6f200174" target='_blank'>
              Viral transcriptional regulators extensively rewire host pathways through diverse mechanisms
              </a>
            </td>
          <td>
            Jaice T Rottenberg, Xing Liu, A. Berenson, L. Soto-Ugaldi, Mohamed Y. ElSadec, Clarissa S Santoso, James E. Corban, Phillip J. Dexheimer, Berkay Engin, Ryan Lane, Sakshi Shah, Kerstin Spirohn-Fitzgerald, Shubham Khetan, Cheng-Che Lee, George Muñoz-Esquivel, Zhaorong Li, Lucia Martinez-Cuesta, Yunwei Lu, Philipp Trollmann, Tong Hao, S. Yi, Nidhi Sahni, M. Bulyk, M. Calderwood, M. Weirauch, Marc Vidal, Srivatsan Raman, J. F. Fuxman Bass
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="Simple Summary Ewing sarcoma (EwS) is a highly malignant tumor of bone and soft tissue that predominantly affects children and young adults, with a high propensity for early metastasis to the lungs and bones. Large-scale sequencing and SNP array studies have concluded that, in addition to the typical EWSR1::ETS translocation, the genome of EwS is remarkably simple at the single-nucleotide level, but recurrent structural chromosome alterations are frequent. The current findings indicate that translocation-mediated EWSR1 haplo-insufficiency results in replication stress and potential loss of heterozygosity and emphasize that copy number dysregulation acts as a critical secondary driver of EwS biology, shaping tumor evolution and influencing clinical outcome. The preclinical results directed at replication stress and diminished repair capacity of EwS are promising and should significantly improve the therapy of EwS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/593ddbf60a24ebf76d603d5b1ea4998334fd3f0e" target='_blank'>
              The Puzzle of Genetic Stability and Chromosomal Copy Number Alterations for the Therapy of Ewing Sarcoma
              </a>
            </td>
          <td>
            Günther H. S. Richter, A. Ranft, Maximilian Kerkhoff, Marvin Jens, Ina E. Kirchberg, U. Dirksen
          </td>
          <td>2025-11-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="
 IDH-mutant gliomas are typically slow growing on initial presentation, however selective pressure during conventional upfront treatment with genotoxic radiation and chemotherapy can fuel the evolution of sub-populations of tumor cells. These subclones promote recurrence of IDH-mutant gliomas that are less likely to benefit from standard treatments. Therefore, novel tumor-specific therapeutic strategies with non-overlapping mechanisms are needed for durable tumor control. IDH mutant cancer cells are known to reprogram their metabolism to sustain rapid proliferation, which generate potentially targetable vulnerabilities in canonical metabolic pathways. Using the patient-derived, WHO grade 3 oligodendroglioma line TS603, we screened a small molecule library of 240 metabolism-related compounds, including both FDA-approved and non- approved drugs, over a range of 10 concentrations (1nM-20µM). Drug efficacy was assessed after 5-day incubation via cell viability assay (Cell-Titer Glo). We identified 53 compounds with IC50<2µM, including drugs that target previously reported IDH-mutant glioma targets, like NAMPT and DHODH. Notable target pathways include folate metabolism and de novo thymidylate synthesis (methotrexate, raltitrexed, 5-FU), p97 AAA ATPase/VCP chaperone (CB-5083, ML240, eeyarestatin I), mevalonate pathway (Ro 48-8071 fumarate, atorvastatin calcium), and the BCL-2 family (Navitoclax, ABT-737). We are validating these drugs across a panel of patient-derived IDH-mutant gliomasphere models, and examining the combinatorial effects of these agents with vorasidenib and CDK4/6 inhibitors. Our results provide insight into important metabolic pathway dependencies in recurrent IDH1-R132H mutant gliomas that have targeting potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99af98af377416756584e34e3b8b2129cefb73a2" target='_blank'>
              DDDR-40. Identifying metabolic vulnerabilities of recurrent IDH1-R132H mutant glioma by high throughput screening
              </a>
            </td>
          <td>
            Yutong Wang, Sayali Daware, D. Cahill, Julie Miller
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 Glioblastoma has a complex regulatory landscape that drives its cellular heterogeneity, plasticity, and resistance to therapy—features not explained by genetics alone. Chromatin profiling is essential for uncovering the epigenetic mechanisms underlying these behaviors and for identifying potential vulnerabilities that could be targeted therapeutically. Combinations of histone marks govern the activity of their associated DNA, including functional genomic elements such as enhancers or promoters. Single-cell technologies have only recently been adapted to the study of epigenetic cell states and significant technical challenges have limited their widespread adoption, especially in complex tissues. Here, we employ our recently optimized method for performing simultaneous single-cell epigenetic and transcriptomic analyses in multiple frozen glioblastoma (GBM) samples. Our objective was to implement our recently optimized single cell multi-omic sequencing method (“mscCUT&TAG”) and integrate with additional single cell modalities to evaluate epigenetic heterogeneity in primary glioblastoma samples. Glioblastoma (IDH WT, MGMT unmethylated) samples were collected from primary surgical resection in two patients. We applied our previously developed multi-omic single cell cleavage-under-target and tagmentation (“mscCUT&TAG”), 10X single-cell RNA-seq, 10X Multiome (RNA+ATAC-seq) and whole transcriptome Spatial Enhanced Resolution Omics-sequencing (Stereo-seq) to these samples. We comprehensively characterized 72,000 cells at single-cell resolution by integrating joint chromatin accessibility, transcriptional profiles, and five histone modification chromatin states associated with active (H3K27ac, H3K4me3), poised (H3K4me1) and repressive marks (H3K27me3, H3K9me3), We then combined this data with whole transcriptome Stereo-seq of these tumors, and generated a spatially resolved blueprint of epigenomic regulation in GBM. We identified specific tumor cell populations, cell-type specific coordinated epigenetic and transcriptional changes, and defined functional regulatory elements.We performed a comprehensive analysis of epigenetic heterogeneity in GBM at single-cell and spatially resolved resolution. We anticipate that further extension of this work will provide a greater understanding of fundamental biology and therapeutic avenues in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27612dab26c7b403075cd8f3dfbd065d9089efda" target='_blank'>
              EPCO-48. DECIPHERING THE EPIGENETIC LANDSCAPE OF GLIOBLASTOMA THROUGH MULTIOMIC SINGLE-CELL AND SPATIAL PROFILING
              </a>
            </td>
          <td>
            Andrew Hardigan, E. Blandford, Aditya Mohan, P. Fecci, A. Patel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Background and significance. Stemness in acute myeloid leukemia (AML) is determined by a clonal hierarchy with rare, phenotypically defined leukemia-initiating cell (LIC) at its apex (Lapidot 1994, Bonnet 1997). Given that LICs in AML are drug-resistant and initiate relapse of refractory disease, the identification of an LIC population in B-ALL would be consequential. However, unlike AML, B-ALL does not evolve from a clonal hierarchy and LIC populations have remained elusive in B-ALL (Kelly 2007, Le Viseur 2008, Rehe 2013).
 Results. Time-lapse studies of single patient-derived B-ALL cells revealed that most B-ALL cells were continuously proliferating (steady), while subpopulations underwent periodic transitions between quiescent and proliferative cell states (alternating). Gene expression studies of 'steady’ and 'alternating’ cells revealed a dominant and reciprocally repressive transcriptional program driven by MYC in 'steady’ and BCL6 in 'alternating’ B-ALL cells. To dissect how MYC and BCL6 dynamics govern transitions between proliferative and quiescent cell states in B-ALL, we engineered patient-derived B-ALL (PDX) by CRISPR and HDRT with dual-reporter knockin alleles expressing C-terminal MYC–mNeonGreen and BCL6–mScarletI fusion proteins. Using time-lapse imaging, we tracked MYC and BCL6 protein expression in single B-ALL cells over 20 hours. Matching their proliferative characteristics, 'steady’ B-ALL cells expressed MYC and no detectable BCL6. However, B-ALL PDX also included 'alternating’ cells that underwent multiple transitions between MYC+ BCL6- and MYC- BCL6+ cell states. PDX from BCR-ABL1 B-ALL included approximately 30% 'alternating’ cells, whereas 'alternating’ cells exceeded 50% in B-ALL PDX with RAS-pathway lesions. Among 'alternating’ cells, transitions occurred independently from cell divisions (every 36 hours) with shorter MYC-phases of three hours and longer BCL6-phases of six hours.
 Integrated ChIP-seq, RNA-seq, and untargeted metabolomics revealed that MYC-high cells are much larger, activated glycolysis and protein-synthesis pathways, whereas BCL6-high cells are smaller and enriched for phosphatidylethanolamine (PtdEtn) synthesis pathways. Consistent with the essential role of PtdEtn in autophagosome biogenesis, BCL6+ B-ALL cells exhibited substantially increased autophagy (LC3B flux). To directly compare MYC and BCL6 dynamics with cell growth trajectories, we combined quantitative phase microscopy and time-lapse fluorescence imaging to measure single-cell dry mass over 8 hours. Cells in the MYC-high state accumulated biomass at twice the rate of BCL6-high cells, producing a step-by-step progression of fast (MYC) and slow (BCL6) cell mass accumulation that aligned with each MYC/BCL6 state transition. Finally, multiplex immunofluorescence of FFPE biopsies from patient samples revealed spatially segregated niches of MYC-high and BCL6-high leukemia cells reminiscent of partitioning of MYC+ centrocyte and BCL6+ centroblasts in germinal centers.
 To experimentally induce cell state-transitions, we engineered patient-derived B-ALL cells by HDRT with dual-reporter knockin alleles expressing C-terminal MYC–dTAG fusion proteins. In addition to the MYC-dTAG degron system, we used the BCL6 PROTAC degrader ARV-393 for acute ablation of either MYC or BCL6, which was achieved within 90 minutes. Acute dTAG-mediated degradation of MYC induced cell shrinkage and decreased cellular dry mass, whereas BCL6 degradation via PROTAC decreased autophagic flux and increased dry mass, confirming opposing roles in anabolic versus catabolic metabolism.
 To functionally study 'steady’ (MYC-only) and 'alternating’ (MYC/BCL6) B-ALL populations, we developed a cell-sorting strategy that enriched each of the two populations to a purity of 85% as confirmed by subsequent time-lapse imaging. Extreme limiting dilution and serial transplant experiments revealed that 'steady’ (MYC-only) largely lacked LIC-capacity. In contrast, 'alternating’ (MYC/BCL6) B-ALL cells were highly enriched for LIC and initiated fatal leukemia after short latency.
 Conclusion: This study uncovers previously unrecognized cell state transitions in B-ALL that are controlled by MYC- and BCL6-dependent transcriptional programs. Alternating cycles of quiescent and proliferative cell states balance anabolic and catabolic metabolism, promote drug resistance, and enhance leukemia-initiating potential.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21ac412fe67d4caf3c95440feaedce483b7297e3" target='_blank'>
              Alternating cycles of quiescent and proliferative cell states determine stemness and leukemia-initiation capacity in acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Zhangliang Cheng, Kohei Kume, Ruoyi Shi, Mark E. Robinson, Kadriye N Cosgun, Yujin Bao, Siyi Chen, Pranav Mishra, Joerg Bewersdorf, Mina Xu, Markus Müschen
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de239b0d87af797daf7dd36473b36ee252b5ad66" target='_blank'>
              Long-read genome sequencing resolves the breakpoints of a chromosome 8;22 balanced translocation in NF2-related schwannomatosis.
              </a>
            </td>
          <td>
            Marco Montini, Lorenzo Bonacchi, Diletta Sidoti, Stefano Cuccoli, Marilena Pantaleo, Angela Peron, Irene Scuffi, Marcos F DosSantos, R. Nassini, Francesco De Logu, Laura Papi
          </td>
          <td>2025-12-01</td>
          <td>Scientific reports</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="


 Myeloproliferative neoplasms (MPNs), including chronic myeloid leukemia (CML), are oncogene-driven blood cancers arising from hematopoietic stem cells (HSCs) that often progress from a ‘smouldering’ chronic phase to a drug-resistant, pre-terminal blast phase. In CML, recent studies have shown that stem and progenitor cells (SPCs) from chronic phase (CP) patients at high risk of blast crisis (BC) transformation exhibit a convergent gene expression signature that mirrors that of overt BC, independent of underlying somatic mutations (Ko et al., Blood 2020; Krishnan et al., Blood 2023). Collectively, these findings suggest the existence of a common drug-resistant SPC state. Identifying the therapeutic vulnerabilities of such states holds promise for transforming the management of high-risk MPNs.
 To identify these resistant states, we performed single-cell (SC) multi-omic profiling (scRNA-seq, scATAC-seq, and targeted mutational analysis) on pre-treatment bone marrow (BM) mononuclear cells from (n=24, CML) three clinical groups (optimal imatinib [IM] responders, suboptimal IM responders, IM failure with BC progression; n=5/5/6), normal BM (n=4), and BC samples (myeloid n=2, lymphoid n=2). scATAC-seq data annotated 28 cell types from 240,456 cells; we identified DARs with DESeq2, inferred TF activity with ChromVAR, integrated gene-to-peak and enrichment analyses, mapped epigenetic remodeller occupancy with HOMER, and built Gene Regulatory Networks (GRNs) using SCENIC+ algorithm.
 Across all prognostic groups, analysis of LSCs revealed a shared, CML-specific epigenetic state distinct from normal BM HSCs, characterized by 29,317 differentially accessible regions (DARs) among 421,985 peaks (6.95%), with ~75% located in intronic and intergenic regions. Consistent with the drug-resistant phenotype of LSCs, functional annotation of these DARs highlighted hallmark features of resistance, including increased inflammatory and proliferative signaling and diminished stemness. Importantly, these LSC states were closely linked to canonical BCR::ABL1 signaling pathways, with enrichment of AP-1, STAT1/3/5A/5B, and NF-κB TF binding activity, implicating these pathways in the establishment of the LSC state. Strikingly, among the top 10 CML-specific transcription factors driving the LSC program, only STAT binding activity correlated with increasing clinical resistance, specifically through enhanced IL2-STAT5 and TNFα signaling.
 Given the relative stability of the LSC state across prognostic groups, we next examined the progenitor population for cell state changes. Unlike LSCs, nearly 32% of the progenitor epigenome showed altered chromatin accessibility with BC progression more extensive in lymphoid BC (LBC) (94,245 DARs) than myeloid BC (MBC) (43,378 DARs). In CP patients who developed LBC, 29,167 LBC-specific DARs were enriched in lymphoid progenitors at diagnosis. Whereas MBC-associated DARs (1,278–5,569) were weakly detected across myeloid subsets in CP patients who developed MBC. These findings support the notion of a common epigenetically-primed state in diverse progenitor compartments, predisposing them to subsequent transformation. Subtype-specific TF analysis showed AP-1/CEBP enrichment in MBC, and IRF8/9 in LBC. GRN reconstruction from scRNA-seq showed these TFs drive resistance associated programs characteristic of full-blown BC (Ko et al. Blood 2020). Ongoing GWAS-linked analysis seeks germline or acquired variants tied to these primed states.
 Lastly, we identified shared upregulation of STAT and IRF pathways in both BC subtypes, TFs downstream of JAK1/2/3, indicating convergent regulatory programs. This led us to testt, an FDA-approved JAK inhibitor, to dismantle these cell states. Although tofacitinib alone showed minimal cytotoxicity, it markedly sensitized BC cells (n=4) to TKIs (~90% killing) while sparing normal progenitors (~25% killing). Thus, mapping the CML epigenetic landscape identified core transformation regulators and revealed actionable therapeutic opportunities.
 This study presents the first scATAC atlas of CML resistance and progression, revealing that: (1) Epigenetic reprogramming in CP precedes BC transformation; (2) Distinct TF networks drive BC subtypes; and (3) JAK inhibition is a novel strategy to dismantle JAK/STAT-IRF activation in high-risk SPCs. Our findings highlight the power of SC- epigenomics to identify actionable vulnerabilities and guide pre-emptive therapies for high-risk MPN patients.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e0b2fbf0807bd6dc58e419db5b166831585fd5e" target='_blank'>
              Single-cell chromatin profiling reveals early epigenetic reprogramming, subtype-specific regulatory networks, and JAK/STAT–IRF vulnerabilities in high-risk chronic myeloid leukemia
              </a>
            </td>
          <td>
            S. T. Ong, Pavithra Shyamsunder, Beijun Chen, Zahid Nawaz, P. Venkatesh, Xinyi Yang, Y. Lim, Charles Chuah, Shyam Prabhakar, Vaidehi Krishnan, John Ouyang
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="


 Cancer cells adapt to environmental and therapeutic pressures through karyotypic plasticity, including whole genome doubling, aneuploidy, and structural genome changes. Polyploid/polyaneuploid giant cancer cells (PGCCs/PACCs) represent an extreme state of this plasticity, arising through endoreplication as a common response to chemotherapy, irradiation, viral infection, hypoxia, or nutrient restriction. PGCCs act as drug-tolerant reservoirs that can later depolyploidize to repopulate tumors, underscoring their role in recurrence and therapy failure. Yet, how readily tumor cells access the PGCC state and how karyotype dynamics sculpt drug sensitivity remain poorly understood. To address this gap, we integrated (i) long-term evolution experiments (LTEEs), (ii) mathematical modeling, and (iii) computational analysis into a unified framework to define karyotype evolution as a predictive biomarker of therapy response. To study karyotype evolution under metabolic stress, we developed CLONEID, an LTEE platform that combines bright-field imaging, single-cell karyotyping and a neutral-drift framework distinguishing drift from selection. Its computer-vision module classifies PGCC versus proliferative states while PCA of other morphometrics revealed stress-specific phenotypes even without whole-genome doubling or overt ploidy change. LTEEs (>6 months) under glucose deprivation, phosphate restriction or hypoxia revealed stress- and ploidy-dependent trajectories. Glucose deprivation promoted whole-genome doubling whereas phosphate restriction and hypoxia drove chromosome loss toward near-diploid states. Strikingly, our in vivo studies confirmed hypoxia-induced ploidy reduction, which was predicted by LTEE trajectories within the 36 days and recapitulated by our model. Moreover, long-term adaptations also shifted therapy response. Glucose-deprived cultures gained resistance to gemcitabine and topotecan while phosphate-deprived cultures became more sensitive to taxanes and carboplatin. Furthermore, our functional assays showed that evolved cultures enhanced T-cell activation while control condition suppressed immunogenicity through tryptophan metabolism. To complement LTEEs, we built an ODE linear chain trick (LCT) model of the cell cycle, calibrated with isogenic diploid and tetraploid TNBC lines. Our model successfully captured ploidy-conditioned therapeutic responses. Specifically, gemcitabine, among 74 tested anticancer agents, eliminated near-diploid cells, but spared tetraploids that entered PGCC state and resumed proliferation after drug withdrawal. Overall, our findings identified that whole-genome doubling, polyploidy, and PGCC formation shape both long-term evolution and acute therapy responses. Integrating LTEEs with LCT modeling successfully links ploidy, karyotypic adaptation, and drug sensitivity. Our approach establishes the ground for “Drug–Karyotype” pairs as biomarkers and shows how metabolic niches sculpt KFLs, offering a path toward personalized, evolution-informed oncology.



 Vural Tagal, Jackson Cole, Richard J. Beck, Didem Ilter, Daria Myroshnychenko, Konstantin Martin, Andriy Marusyk, Ana P. Gomes, Noemi Andor. Modeling karyotype-driven adaptations to metabolic restrictions predicts therapeutic response and immunogenicity in cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B026.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06671e4b962908638a954d43578035d059b71aa3" target='_blank'>
              Abstract B026: Modeling karyotype-driven adaptations to metabolic restrictions predicts therapeutic response and immunogenicity in cancer
              </a>
            </td>
          <td>
            Vural Tagal, Jackson Cole, Richard Beck, Didem Ilter, D. Myroshnychenko, Konstantin Martin, A. Marusyk, , Noemi Andor
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="


 Untreated prostate tumors depend on androgen receptor (AR) for growth and thus are treated with hormonal therapy. However, resistance almost always emerges as tumors evolve into a castration-resistant state. The evolution of resistance can follow different paths, and castration-resistant prostate cancer (CRPC) exhibits multiple epigenomic subtypes: androgen receptor-dependent CRPC-AR, and lineage plastic subtypes CRPC-SCL (stem cell-like), CRPC-WNT (Wnt-dependent), and CRPC-NE (neuroendocrine). By transcriptomic profiling of tissue, and whole-genome sequencing (WGS) of tissue and cell-free DNA (cfDNA) from 500 patient samples, we relate genomic variants with epigenomic state. We annotate fractional contribution of each subtype for all patient samples using deconvolution approaches with a set of signature genes and accessible chromatin sites. We confirm that AR amplifications at the genomic level are associated with the emergence of CRPC-AR, and RB1 biallelic loss is associated with CRPC-NE. Importantly, we find chromosomal rearrangements in the YAP/TAZ pathway are associated with the presence of CRPC-SCL. In particular, we find complex rearrangements on chromosome 4, which are supported by patient-matched Hi-C data, and decrease promoter interactions of MOB1B, a YAP/TAZ pathway inhibitor, with its enhancers. Together, the genomic variants in the pathway can predict CRPC-SCL with 79% accuracy. By computing cancer cell fraction (CCF) of the genomic events, we find high concordance between the CCF of genomic events and the fraction of their associated epigenomic subtype. Thus, higher CCF of chr 4 rearrangements corresponds to higher tumor fraction of CRPC-SCL. Our study shows how genomic events enable transition to different CRPC states that exhibit differential therapeutic susceptibilities.



 Marjorie Roskes, Alexander Martinez-Fundichely, Sandra Cohen, Metin Balaban, Chen Khuan. Wong, Weiling Li, Tonatiuh A. Gonzalez, Anisha B. Tehim, Hao Xu, Shahd ElNaggar, Matthew Myers, Andrea Sboner, Benjamin J. Raphael, Yu Chen, Ekta Khurana. Chromosomal rearrangements at the YAP/TAZ pathway genes are associated with heterogeneity and stem cell-like castration-resistant prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B013.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52b27abd0fec4993dd28bd5604ff77de95b09d9b" target='_blank'>
              Abstract B013: Chromosomal rearrangements at the YAP/TAZ pathway genes are associated with heterogeneity and stem cell-like castration-resistant prostate cancer
              </a>
            </td>
          <td>
            M. Roskes, A. Martínez-Fundichely, S. Cohen, M. Balaban, C. K. Wong, Weiling Li, T. A. Gonzalez, A. Tehim, Hao Xu, S. ElNaggar, M. Myers, A. Sboner, B. Raphael, Yu Chen, E. Khurana
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="


 Somatic mutations in SET binding protein 1 (SETBP1), a putative chromatin regulator, occur in ~30% of high-risk myeloid malignancies, including myelodysplastic syndromes (MDS), chronic myelomonocytic leukemia (CMML), and secondary acute myeloid leukemia (AML), and strongly correlate with adverse clinical outcomes. Despite this significance, the precise roles of SETBP1 mutations in blood cancer remain enigmatic, primarily due to the lack of physiologically relevant cell lines or animal models that recapitulate the clonal evolution. Indeed, our 23 attempts to establish SETBP1-mutated PDX models failed, as the clones did not show advantageous effects. While conventional wisdom suggests that SETBP1 mutations escape proteasomal degradation to drive oncogenesis, our Setbp1 knockout mice revealed that SETBP1 is dispensable for normal and malignant hematopoiesis (Tanaka et al. Leukemia 2023), challenging prevailing paradigms.
 To resolve these contradictions, we developed a blood-specific Setbp1 mutant locus knock-in (KI) mouse model using Vav1-iCre, corresponding to the most prevalent D868N mutation. Unlike retroviral overexpression that immortalizes murine hematopoietic stem and progenitor cells (HSPCs), our KI model did not perturb HSPC function or transcriptome in vivo, indicating that SETBP1 acts as a cooperative rather than an initiating driver. As ASXL1 mutations are the most frequent founder mutations in SETBP1-mutated MDS/AML, we created double KI mice with Asxl1/Setbp1 physiologic mutations for transplant studies. These mice exhibited striking monocytosis (~40% of peripheral white blood cells) and HSPC expansion, rescuing stemness defects imposed by ASXL1 mutations. scRNA-seq/ATAC-seq revealed unique transcriptional clusters enriched for Hoxa9/10, Eya1, Myb, and Myc pathway alongside increased chromatin accessibility, suggesting that both mutants cooperate to remodel chromatin and amplify oncogenic transcriptional programs. Over time, the transplant model with double KI developed a leukemia phenotype, including serially transplantable AML and CMML-like diseases, all spontaneously acquired RAS pathway-activating mutations in Nras, Flt3, and Ptpn11. This reflects findings from over 7,000 patient samples, showing a significant correlation between SETBP1, ASXL1, and RAS pathway mutations. Indeed, the presence of RAS mutations predicts poor outcomes in SETBP1-mutated cases. Functional validation confirmed that all three alterations are required for complete transformation toward AML, establishing a three-step model: ASXL1mutations initiate clonal evolution, SETBP1 mutations enhance oncogenic programs, and RAS pathway activation completes the transformation.
 Leveraging these unprecedented in vivo models and cell lines with physiologically relevant SETBP1 mutations, we conducted comprehensive therapeutic screens using whole-genome CRISPR/sgRNA and pan-cancer drug libraries. The integrated results revealed a critical dependency on the druggable Exportin 1 (XPO1), shown by a 20.8-fold reduction (p = 0.00004) in the 7-day CRISPR screen and a 0.0003-fold growth under Leptomycin B treatment compared to DMSO at day 4. This unique dependency was robustly validated through genetic and pharmacological approaches using shXpo1, Selinexor, and Leptomycin B, resulting in monocytic differentiation and apoptosis in Setbp1-mutated AML cells. Notably, Selinexor monotherapy achieved sustained remissions exceeding 110 days in vivo, profoundly extending survival compared to controls, which succumbed to leukemia with a median survival of 59.5 days (p = 0.0004), demonstrating its potential to eradicate SETBP1-mutated leukemia. Mechanistically, XPO1 physically interacts with SETBP1 through the nuclear export signal groove and acts as a chromatin regulator beyond its canonical nuclear-cytoplasmic transport role. XPO1 inhibition suppressed the transcription of leukemogenic genes, including Hoxacluster genes, Myb, Eya1, and, critically, Myc pathway, with altered XPO1-mediated chromatin accessibility, as shown by integrated ChIP/RNA/ATAC-seq. These results uncover a critical epigenetic vulnerability that represents a promising target for therapeutic intervention in SETBP1-mutated leukemia.
 In summary, our study elucidates the cooperative leukemogenic roles of SETBP1, ASXL1, and RAS pathway mutations and identifies XPO1 as a targetable epigenetic vulnerability. These findings support precision treatment strategies for aggressive myeloid malignancies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a6566631ba342b06e47eaa7af05b86f27fd8cd1" target='_blank'>
              Clonal evolution and therapeutic vulnerabilities in SETBP1-mutated leukemia: Insights from physiological mouse model and integrative functional screens
              </a>
            </td>
          <td>
            Wataru Saika, Koutarou Nishimura, Atsushi Tanaka, J. Takeda, A. Kon, A. Yoda, Kazunari Aoki, Masaki Nomura, Yui Koike, Hiromi Yamazaki, Seigi Oshima, Shinri Okada, Daichi Morii, Weijia Zang, Takaya Yamasaki, Hiromi Ito, Manabu Nakayama, Hiromi Tatsumi, Masashi Matsuda, Chien-Chin Lin, Kosuke Yusa, Haruhiko Koseki, Makoto Murata, Seishi Ogawa, Daichi Inoue
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="


 DNA methylation at CpG dinucleotides is a critical regulator of cell identity and epigenetic memory, and it becomes universally disrupted during hematopoietic aging, preleukemic clonal expansions and their progression towards malignancy. Therefore, understanding how hematopoietic identity is specified, inherited and disrupted at the single-cell level is essential to develop therapeutic strategies for age-related hematologic disorders. However, current single-cell DNA methylation profiling technologies are limited by either low throughput (hundreds of cells) or low capture rates (103-105 CpGs per cell; <0.3% of CpGs the human genome) and are thus insufficient to characterize heterogeneous populations in the hematopoietic system.
 To tackle this limitation, we developed DREAM-seq (Droplet-based Restriction Enzyme And Methylation sequencing), a high-throughput single-cell DNA methylation technology based on droplet microfluidics and MspI-based reduced representation chemistry. DREAM-seq profiles >5,000 single-cell methylomes per experiment, capturing 0.44M CpGs per cell (3.75-fold increase vs. existing methods) and 3.16 CpGs per read (4-fold increase), with each CpG detected in 1 out of 20 cells. This allows us to generate comprehensive datasets covering >50% of CpGs in the human genome, with marked improvements in clustering resolution and cell type identification.
 We leveraged DREAM-seq to sequence 51,208 single-cell methylomes from human peripheral blood, CD34-enriched and whole bone marrow (BM) cells, generating the largest tissue-specific atlas of hematopoietic differentiation at single-cell methylome resolution to date. Unbiased selection of 71,115 variably methylated CpG sites identified all major hematopoietic cell subtypes, recapitulating cell type specific methylation patterns (monocyte-specific CEBPA and MPO hypomethylation) and cell states (TCF7 gene body methylation in effector CD8+ T cells compared to naïve subsets). Cell-type defining CpGs were depleted from CpG islands and TSS but strongly enriched in distal enhancers and CTCF sites (72.2 %), highlighting their role in specifying hematopoietic cell fates. Interestingly, 27.8% of cell-identity CpGs were located in heterochromatin, where DNA methylation provides an additional regulatory layer not captured by chromatin accessibility methods. These data suggest a model in which DNA methylation regulates both enhancer activity and higher-order chromatin structure at heterochromatin regions, which together control hematopoietic cell identity.
 To gain insights into methylation-driven lineage specification, we computed enhancer methylation levels across hematopoietic populations. We discovered that >40% of CD34+ cells display hypomethylation of lineage-specific enhancers prior to commitment, suggesting methylation-based lineage priming at the HSPC level. We further traced the methylation patterns from HSPCs towards differentiated cells in myeloid, lymphoid and erythroid lineages, identifying early and late changes in the epigenome that occur upon lineage commitment. This highlights previously unrecognized roles of DNA methylation in lineage priming and hematopoietic differentiation dynamics.
 To understand how these methylation states are disrupted during hematopoietic aging and age-related clonal expansions, we analyzed 18,268 single-cell methylomes from aged BM donors, including 3 TET2-mutant clonal hematopoiesis samples. We first identified clonal populations by leveraging genetic variants as clonal markers, finding significant oligoclonality in aged BM, with 3-fold less clonal clusters than young BM. By characterizing the methylation state of these populations, we are defining “high-fitness” methylation states, which could be implicated in driving clonal expansions and might be reversed to constrain age-related clonal hematopoiesis.In summary, we present DREAM-seq, a new single-cell DNA methylation technology which we leverage to generate the largest single-cell methylome atlas of human hematopoiesis to date. By defining methylation-based cell states predictive of lineage output and clonal fitness, we identify epigenetic drivers of hematopoietic differentiation and their disruption during age-related clonal expansions. Our framework provides a foundational resource for understanding epigenetic drivers of hematopoietic malignancies, improving leukemia classification and predicting therapeutic responses.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35989f1212a0453a63a17b29c85e417f9e291e63" target='_blank'>
              High-throughput single-cell methylation resolves epigenetic states in hematopoietic differentiation and age-related clonal expansions
              </a>
            </td>
          <td>
            Alba Rodriguez-Meira, Sander Lambo, Carolina Varona-Arguelles, Rony Chanoch-Myers, Paul Kaegy, Salvador Casani-Galdon, Peter van Galen, Volker Hovestadt, Bradley Bernstein
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="
 Medulloblastoma is molecularly characterized into 4 main subgroups consisting of WNT, SHH, Grp 3 and Grp 4, which are further divided into 13 subtypes. Medulloblastoma manifests as a clinically heterogeneous tumor with varying prognoses depending on the molecular subgroup. Treatments for high-risk patients include craniospinal irradiation (36Gy) after surgical resection and chemotherapy. Group 3 tumors with upfront radiation therapy relapse in a shorter amount of time at 1 year. Currently, there are limited therapeutic strategies that address medulloblastoma tumor recurrence which has detrimental effects on patient outcome with <5% long term survival. Subsequently, there is a critical need to investigate the molecular mechanisms underlying therapeutic resistance in medulloblastoma and identify the best strategies to combat it. To understand what causes radiation resistance in medulloblastoma, we analyzed matched primary and relapsed patient tumors by single cell ATAC seq + gene expression. We further evaluated radiation resistance in an orthotopic mouse model and analyzed gene expression changes by single-cell RNA sequencing. We identify a resistant cell population that exhibits a distinctive gene signature comprising stem and metabolic gene transcription programs. We observe elevated expression of metabolic marker genes such as wtIDH1 and stem cell markers, TEAD1 and SOX6, within the recurrent cell population. We speculate that radiation resistance in group 3 medulloblastoma is facilitated by wtIDH1 metabolic reprogramming, leading to epigenetic adaptation and cell persistence. Thus, we evaluate how IDH1 changes the epigenetic landscape of radiation-resistant and sensitive cells by examining histone modifications (H3K27ac, H3K27me3, H3K4me3) and the broader chromatin architecture in response to IDH1 inhibition by Cut and Run and ATAC-sequencing. We observe that the manipulation of wtIDH1 reduces MB cell growth, stemness characteristics, and changes the chromatin landscape. Further, we find inhibition of wtIDH1 enhances susceptibility to re-irradiation and provides a survival benefit. Our results highlight evidence of the development of radiation resistance and provide a potential therapeutic alternative for treatment of relapsed MB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9d6602a0e48e41dffae7101ad80db0948939a5f" target='_blank'>
              DDDR-30. Targeting wtIDH1 in radiation-resistant medulloblastoma
              </a>
            </td>
          <td>
            Bethany L. Veo, Dong Wang, J. DeSisto, Angela M Pierce, Breauna Brunt, Rajeev Vibhakar
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="


 CRISPR/Cas9 screening is a powerful tool used to delineate genes critical for AML progression and drug resistance. However, most of the studies have been performed in cell lines and PDX models. While cell lines fail to represent the genetic, epigenetic, and phenotypic heterogeneity of primary leukemic cells, PDX models are limited by variable engraftment efficiency, time-consuming processes, and high costs. To overcome these challenges, we pioneered an integrated approach combining bioinformatics and CRISPR/Cas9 screening to uncover genetic drivers and vulnerabilities in AML using patient-derived primary cells in culture. Insights from our study could inform the development of targeted therapies and improve patient outcomes.



 Custom CRISPR/Cas9 screen targeting 2,440 genes associated with AML, leukemic stemness, and the druggable genome was performed using mononuclear cells (MNCs) isolated from peripheral blood or bone marrow from 9 AML patients. Briefly, the cells were transduced with custom library and cultured for 24-30 days. After an initial few days in suspension, a subset of MNCs from 5 of 9 patients transitioned to an adherent state 5-7 days post-transduction. Both adherent and suspension cells were collected at different time points, sgRNA was quantified and sequenced using the Illumina NovaSeq X Plus. sgRNA abundance was analyzed using the RRA algorithm of the MAGeCK computational tool and genes with significant negative or positive enrichment in each patient was identified by comparing early versus late time points and adherent versus suspension cell fractions.



 In a subset of patient samples, AML cells transitioned from suspension to an adherent phenotype during long-term culture, a state previously associated with increased chemotherapy resistance due to interactions with endothelial niches. Our CRISPR/Cas9 screen revealed that this adherent cell population harbored distinct genetic dependencies, including consistent negative enrichment of EDNRA and SOD1.These genes were selectively essential in the adherent fraction but not in suspension cells, suggesting context-specific vulnerabilities. The identification of these targets offers new opportunities to therapeutically disrupt chemo resistant AML niches, potentially enhancing the efficacy of existing treatments and reducing relapse risk. EDNRA has been implicated in many cancers and in context of AML, it is a downstream target of HOXA9 and MEIS1. Our results indicate that EDNRA knockout inhibits AML cell proliferation and growth, highlighting its potential as a therapeutic target. Furthermore, EDNRA is overexpressed in AML compared to normal cells, and since FDA approved EDNRA inhibitors such as Macitentan, Bosentan, Ambrisentan, and Clazosentan are already being used to treat pulmonary arterial hypertension, this presents a promising opportunity to evaluate these inhibitors as potential chemotherapy-sensitizing agents for high-risk AML.SOD1 plays a critical role in eliminating toxic radicals generated within biological systems and has been associated with poor outcomes in AML. SOD1 has been identified as a synthetic lethal target in PPM1D-mutant leukemia cells, corroborating our findings. TRAPPC9, SUZ12, and ERG were significant genes with negative enrichment when comparing adherent and suspension fractions across 5 patient samples. These genes have a higher expression in primary AML cells compared to normal controls, implying potential roles in disease progression. SDHA was identified as a significant negatively enriched gene in the suspension cell fraction in 4 of 9 patients. It exhibited elevated expression in AML cells compared to controls, and its deficiency has been linked to the development of drug resistance in AML, emphasizing its role in both drug resistance and disease progression.



 This study demonstrates the establishment of a scalable platform for CRISPR screening in primary AML cells for identification of more conserved vulnerabilities that may be exploited therapeutically, with EDNRA as a potential target. Ongoing work includes validating key targets with the goal of advancing therapeutic strategies. Future studies will expand CRISPR screening to a broader set of patient samples with diverse cytogenetic profiles and incorporate CROP-seq technology to perform single-cell CRISPR screens, providing deeper insights into the molecular mechanisms underlying AML progression and relapse.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46eb706147f8e498d47ce576631d70d699b36b33" target='_blank'>
              A patient-derived CRISPR platform reveals selective dependencies in Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Aneesha Nath, Nam Nguyen, A. Tagmount, Nathan D. Seligson, Christopher Vulpe, Vivek Shastri, Dina Khalaf, Christopher Cogle, S. Pounds, J. Lamba
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="


 CML progression is defined by increased disease burden and potential for transformation to blastic phase (BP), yet the mechanisms and ability to predict this evolution remain elusive. We applied elements of state-transition theory to model transcriptional dynamics underlying leukemic progression, where individual cells' gene expression profiles represent microstates, and aggregated population-level gene expression profiles define macrostates that govern phenotypic transitions from health to disease initiation to overt leukemia. Previously, we demonstrated through bulk transcriptomic analyses that CML progression and therapeutic responses can be accurately predicted by modeling a leukemic potential landscape derived from transcriptional state-space geometry.
 To extend this framework to single-cell (sc) resolution and capture more granular information at the microstate level, we collected weekly peripheral blood samples from inducible chronic phase (CP) and BP CML mouse models to generate time-series scRNA-seq data tracking the transition from health to overt CP or BP. After performing quality control and cell labeling, we used the first (healthy) time point before the induction of BCR::ABL (T0) and the final leukemia time point (Tf), in addition to other intermediate time points, to assess changes in cell populations and gene expression. All analyses were first performed in CP mice, validated in BC mice, and corroborated using two independent human CML scRNA-seq datasets.
 By comparing sequential time points, we observed that progression to CP CML was associated with a significant decrease in B cells (p<0.01) and increases in both myeloid (p<0.01) and stem cell (p<0.001) populations. Differential gene expression (DEG) analysis revealed marked transcriptional changes across all major lineages (DEGs: B cells = 781; T cells = 2,149; Myeloid = 1,999; Stem cells = 194). Despite these changes, no single-cell transcriptional state at any time point was uniquely associated with health or disease, as transcriptional profiles of leukemic cells largely overlapped with those from healthy cells. However, when scRNA-seq data were computationally aggregated into pseudobulk (PsB) samples, mimicking bulk RNA-seq, a distinct CML state-space emerged, revealing a clear disease trajectory defined by three stable macrostates representing early, transitional, and late leukemia. Cell type–specific PsB analyses uncovered independent state-transition dynamics in B cell, T cell, myeloid, and stem cell compartments, highlighting previously unrecognized complexity in disease evolution involving multiple lineages. We further showed that the transition from health to leukemia at the PsB level could be reconstructed as a linear combination of transitions within individual cell subpopulations.
 To this end, we quantified each lineage's contribution to disease progression by performing computational simulations that subtracted the influence of each cell type in turn. Interestingly, B and myeloid populations contributed most and comparably to the global disease trajectory—despite B cells decreasing and myeloid cells expanding over time. This counterintuitive finding suggests that leukemic information is encoded not simply by abundance but by dynamic transcriptional shifts across compartments. These findings were validated in a BP mouse model and in two human CML datasets.
 In summary, we introduce a conceptual framework that distinguishes between microstates (individual cells) and macrostates (population-level transcriptional ensembles), showing that disease progression is not encoded at the single-cell level but instead emerges only when cells are aggregated. While this may appear intuitive, we were surprised to find that higher resolution sc data did not provide clearer insight than pseudobulk data. We speculate that this may reflect greater Shannon information entropy at the single-cell level, which introduces variability that obscures coherent state transitions. To our knowledge, this framework is novel and may inform future approaches to interpreting scRNA-seq data in leukemia and other diseases.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ef9c6ca07080f16386ecab84fd1ec67de21d807" target='_blank'>
              Chronic myeloid leukemia (CML) evolution is not encoded in single-cell microstates, but emerges from population-level macrostates
              </a>
            </td>
          <td>
            D. Frankhouser, Dandan Zhao, Yu-Hsuan Fu, Anupam Dey, Ziang Chen, Jihyun Irizarry, Jennifer Rangel Ambriz, S. Branciamore, Denis O’Meally, Jeffrey Trent, Stephen J Forman, Ya-Huei Kuo, Adam MacLean, Bin Zhang, Russell C. Rockne, Guido Marcucci
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Activating KRAS mutations are highly relevant to various cancers, and KRAS is the most frequently altered oncogenic protein in solid tumors. While historically considered undruggable, two KRASG12C inactive state-selective inhibitors are currently approved for treating patients with non-small cell lung cancer. However, these agents only demonstrate a 30–40% response rate and a median progression-free survival of approximately 6 months, with the inevitable emergence of resistance mechanisms, hence remaining far from achieving a cure. Additionally, several cancers with poor prognostic outcomes, such as pancreatic adenocarcinoma, are driven by other non-G12C KRAS mutations and thus have no effective targeted therapies. Improvements in understanding RAS signaling, RNA, and nucleic acid chemistry, as well as the role of the tumor microenvironment, have sparked a paradigm shift in the approach to KRAS inhibition and suggested the potential for several novel combination therapies. In this review, we provide an overview of the RAS pathway and discuss the ongoing development and status of therapeutic strategies for targeting the oncogenic RAS. We further delve into the challenges of resistance mechanisms to better understand the rationale behind these developing strategies, describe their mechanisms of action, and offer insights into the current clinical trial status of each of these approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c697e95b8beed852f306ce44fd5e98ffcf2023be" target='_blank'>
              Targeting KRAS mutations: orchestrating cancer evolution and therapeutic challenges
              </a>
            </td>
          <td>
            Khalil Choucair, Hafsa Imtiaz, Md. Hafiz Uddin, M. Nagasaka, M. N. Al-Hallak, P. Philip, B. El-Rayes, Boris C. Pasche, A. Azmi
          </td>
          <td>2025-11-28</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Traditional population sequencing methods struggle to directly correlate gene perturbations with transcriptome changes at the single-cell level, due to limitations such as long cycle times and significant batch effects. Single-cell sequencing (SCS) can reveal the diversity and dynamics of cell populations, while CRISPR gene editing and screening technologies enable large-scale functional gene discovery. The combination of these two approaches has led to the development of ScCRISPR-seq. This article reviews the principles of CRISPR screening and SCS technologies, introduces the development of ScCRISPR-seq, and, drawing on research on lung cancer and intrahepatic cholangiocarcinoma (ICC), summarizes its key applications in cancer research, including deciphering tumor heterogeneity, elucidating mechanisms of drug resistance, and modulating the immune microenvironment. This article further explores challenges with this technology, including experimental workflow, data sparsity, and technical noise. Solutions are proposed, including data completion using AI algorithms, eliminating amplification bias using UMI tags, and optimizing gene-phenotype associations through network modeling. However, clinical translation is limited by unclear mechanisms of action and high barriers to entry in drug development. Future research may consider combining ScCRISPR-seq with network pharmacology and molecular docking techniques to improve the efficiency of new cancer drug development and advance precision immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1262cf91f44f77bd99618ed1bc278657dd28dee" target='_blank'>
              The Application and Prospect of Single-cell CRISPR Screening Technology in Tumor Research
              </a>
            </td>
          <td>
            Zineng Zhou
          </td>
          <td>2025-10-23</td>
          <td>MedScien</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Pediatric acute myeloid leukemias (AML) are genetically and molecularly diverse. Pediatric AML carry a range of different driver mutations, often including translocations that produce fusion oncoproteins. Furthermore, they arise from diverse progenitor cell populations and at different stages of fetal and childhood development. This molecular and contextual diversity complicates efforts to identify effective targeted therapies, in part because modeling strategies do not capture the genetic heterogeneity observed among patients.
 We have developed a multipronged strategy to model complex genetic and contextual features of pediatric AML in mice, with the overarching goal of identifying malignant self-renewal programs that afford targetable vulnerabilities. We first used Cellular Indexing of Transcriptomes and Epitopes by sequencing (CITE-seq) and paired functional assays to identify functionally distinct subpopulations of NUP98-rearranged (NUP98r) AML using a previously described NUP98::HOXD13/Flt3ITD transgenic model. Our approach identified a leukemia stem cell (LSC) subpopulation and gene expression signature that was conserved in human NUP98r AML. The LSC signature was also enriched in human UBTF-tandem duplication (UBTF-TD) and NPM1 mutated (NPM1c) AML, but not KMT2A-rearranged AML, suggesting mutation class-specific self-renewal mechanisms.
 To compare LSC self-renewal mechanisms across molecular subclasses of AML, we developed a novel strategy to generate inducible, genetically diverse AML models. Specifically, we generated induced pluripotent stem cells (iPSCs) and chimeric mice to express NUP98::HOXA9, NUP98::KDM5A, NUP98::NSD1, RUNX1::RUNX1T1, MNX1, or KMT2A::MLLT1 oncoproteins specifically in blood cells at defined stages of ontogeny. We used CRISPR/Cas9 gene editing to introduce cooperating mutations at endogenous loci, including Flt3 internal tandem duplication (FLT3ITD), Nf1 loss of function (NF1LOF), and NrasG12D. With this approach, we could induce expression of any desired oncoprotein at any desired stage of development along with relevant cooperating mutation, all while bypassing complex mouse breeding. We also used gene editing strategies to generate NPM1c/FLT3ITD and UBTF-TD/ FLT3ITD AML, again with mutations at endogenous loci. The resulting AML models are transplantable, genetically manipulable and in some cases gave rise to immortalized in vitro cell lines.
 To credential the models and identify fusion/cooperating mutation-specific vulnerabilities, we performed CITE-seq focusing on NUP98::HOXA9/FLT3ITD, NUP98::HOXA9/NF1LOF,RUNX1::RUNX1T1/NF1LOF, KMT2A::MLLT1/NRASG12D, UBTF-TD/ FLT3ITD, and NPM1c/FLT3ITD models. Cross species comparisons to human AML revealed faithful representation of heterogeneity and gene expression within the murine AML. Each genetic subclass exhibited distinct but reproducible differentiation trajectories and transcriptional profiles. NUP98::HOXA9/FLT3ITD, NUP98::HOXA9/NF1LOF, NPM1c/FLT3ITD, UBTF-TD/ FLT3ITD, and KMT2A::MLLT1/NRASG12DAML all displayed intraleukemic heterogeneity and had putative LSC populations that expressed known self-renewal genes. RUNX1::RUNX1T1/NF1LOF AML was homogeneous and lacked a discernible LSC population. Presumptive LSC populations from NUP98::HOXA9/FLT3ITD, NUP98::HOXA9/NF1LOF, NPM1c/FLT3ITD, and UBTF-TD/ FLT3ITD were all enriched for a shared gene expression signature. These highly tractable models thus capture key aspects of human pediatric AML biology.We sought to identify actionable, mutation class-specific vulnerabilities within putative LSC populations, focusing primarily on the NUP98r models. We found that genes encoding sphingosine transporters SPNS2 and SPNS3were highly expressed in NUP98r LSCs relative to non-LSCs, identifying them as potential therapeutic targets. SPNS2/3 were also expressed, to a lesser degree, in NPM1c/FLT3ITD and UBTF-TD/ FLT3ITD AML. Pharmacological SPNS2/3 inhibition severely impaired NUP98rAML growth in vitro, and it significantly extended survival in vivo. In some cases, mice with NUP98r AMLwere cured with SPNS2/3-directed monotherapy. Other genetic subclasses were less sensitive to SPNS2/3 inhibitors, as were normal human and murine hematopoietic progenitors. Our data indicate a therapeutic window for targeting SPNS2/3 in NUP98r AML. They raise the possibility of identifying other mutation class-specific vulnerabilities through the use of well-credentialed, iPSC-derived chimeric AML models.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97c884502e32c16eaf731d0261d136ef166db839" target='_blank'>
              NUP98-rearranged pediatric AML is sustained by a conserved leukemia stem cell program with SPNS2/3 as a targetable vulnerability
              </a>
            </td>
          <td>
            Jihye Yoon, Elisabeth D Denby, Wei Yang, Run Zhang, Masayuki Umeda, Tingting Hu, Jonny Mendoza-Castrejon, Yanan Li, Emily B. Casey, Riddhi M. Patel, J. M. White, J. Klco, Jeffrey Magee
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="


 Introduction:
 Multiple myeloma (MM) is characterized by its genomic complexity that includes recurrent driver single nucleotide variation (SNVs) alongside a high burden of structural variation (SV). Complete characterization of somatic variation in a MM genome is dependent on a technology that can accurately and sensitively detect both small and large variants. While traditional short read sequencing is the gold standard for characterization of simple SNVs, it has limited accuracy for characterization of SVs especially those involving complex rearrangements and challenging to map loci (i.e. IGH). Here, we introduce LinkPrep™ technology, a novel linked-read approach compatible with short read sequencers that delivers high accuracy for both small and large somatic variant detection. We demonstrate LinkPrep performance across four separate relapse MM genomes.
 Methods:
 Four MM patient-derived xenografts were profiled with the Dovetail® LinkPrep™ kit and sequenced to ~30X on an Illumina platform. Data was analyzed through the Dovetail Analysis Portal to detect SNVs/Indels (DeepSomatic), CNVs (Purple), and SVs (HiC-breakfinder and proprietary tools). Call sets were compared against other genomics technologies including WGS (80X tumor/ 30X normal), PacBio (15X), and OGM (400X). We focus analysis on one sample (M24) containing complex SV events and several driver SNVs. Using this sample, we highlight the capabilities of LinkPrep data to detect SNV/Indels, CNVs, and SVs and to link this information together to provide a candidate solution for the complex templated insertions. We further describe SV-driven 3D chromatin alterations (i.e. neo-loops, potential enhancer hijacking) from the same 30X LinkPrep dataset.
 Results:
 LinkPrep + Dovetail Analysis Portal (~8 hours analysis time) returned high quality variant results for the four MM samples. LinkPrep data recalled all pathogenic SNV/Indels detected by 80X WGS (notable pathogenic variants were detected in NRAS, KRAS, TP53). LinkPrep data further detected SVs involving the IGH locus (inversions and translocations) in all four samples. Deeper analysis of one sample (M24) highlights that LinkPrep data detects genetic alterations common in MM including SNV mutations in NRAS and TP53 (both occurring at 100% VAF), IGH rearrangements, chr 1q gain, chr 1p loss of heterozygosity, trisomy of chr 5,7, and 15, and complex rearrangements involving chromosomes 2, 3, 4, 16, and 19. With the exception of a confident subclonal (1% SV-VAF) unbalanced translocation detected by LinkPrep, the remaining LinkPrep large SV calls were validated by other genomics methods (tumor/normal 80X WGS/15X PacBio/400X OGM). Underscoring the capability of LinkPrep for SV detection, LinkPrep calls at 30X contain >10 times more read support for every SV call over other technologies. Using the linked-read feature of LinkPrep data along with CNV segment calls and SV calls provided through the Dovetail Analysis Portal a candidate solution is derived for the complex rearrangement involving chromosomes 2, 3, 4, 16, and 19. This LinkPrep solution, refined down to base pair resolution for every breakpoint, is verified by 400X OGM data. Uniquely, LinkPrep data further enables visualization of the SV-reconstructed 3D genome and reveals neo-loops connecting enhancers with promoters to suggest enhancer hijacking mechanisms associated with the complex rearrangement.
 Conclusion:
 We applied and validated LinkPrep technology, a novel, multi-dimensional, linked-read method, to a MM sample cohort. The technology enables high accuracy comprehensive variant detection, offering improved sensitivity and accuracy for SV detection over short and long reads, including those in challenging genomic regions such as IGH. Linked-reads captured in LinkPrep data enable phased reconstructions of complex rearrangements often found in MM genomes. Furthermore, the same LinkPrep data enables detection of epigenetic state and neo-3D genome interactions driven by SVs to elucidate enhancer hijacking events. Together, these data demonstrate that LinkPrep linked-reads enable improved characterization of somatic variation in highly rearranged genomes, while simultaneously detecting epigenetic states – all enabled with the accuracy and cost-effectiveness of short read sequencing.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/339458d030fcdb32def9e53403cc8c5fb11c7937" target='_blank'>
              Characterization of multiple myeloma genomes with LinkPrep™ assay enables sensitive detection of somatic variation and SV-driven interactions of the 3D genome.
              </a>
            </td>
          <td>
            Nathan Becker, Enze Liu, Alexander Fortuna, Jonathon Torchia, Aneta Mikulasova, Zackary Sanborn, Lisa Munding, Brian Walker
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b9752e134e3e7566b741e6bb0305e92d0896d6a" target='_blank'>
              Timing the onset of homologous recombination deficiency before cancer diagnosis
              </a>
            </td>
          <td>
            Michail Andreopoulos, Muchun Niu, Yang Zhang, V. Viswanadham, D. Gulhan, Hu Jin, Felipe Batalini, Gerburg Wulf, Chenghang Zong, Peter J. Park, Dominik Glodzik
          </td>
          <td>2025-10-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a highly resistant brain tumor with a dismal prognosis despite standard treatments like temozolomide (TMZ) and radiation therapy (RT). This review delineates the molecular mechanisms driving this resistance to inform future therapeutic strategies. A narrative review was conducted using PubMed, synthesizing studies from the year 2000 up to March 2025 on DNA repair enzymes, TMZ and RT resistance pathways, anti-apoptosis mechanisms, genetic heterogeneity, and cell cycle dysregulation in GBM. Five key resistance mechanisms were identified: (1) DNA repair enzymes (e.g., MGMT, APNG) counteract TMZ-induced damage, regulated by NF-κB and CHD4; (2) TMZ and RT resistance depends on MGMT methylation and MMR functionality; (3) anti-apoptotic pathways (e.g., Bcl-2, PI3K/Akt) protect GBM cells, mitigated by TRAIL sensitization; (4) genetic mutations (EGFR, PTEN, TP53) confer adaptability; and (5) quiescent and proliferative GBM stem cells (GSCs) sustain recurrence. Novel therapies targeting HK2 and neural stem cells show promise. GBM resistance is multifaceted, requiring personalized, multi-targeted approaches. Future research should prioritize epigenetic modulation and combination therapies to enhance patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d28a9446444f9c597df4c24c1f9700021704068b" target='_blank'>
              Molecular Insights into Glioblastoma Resistance and Implications for Treatment Advancements: A Systematic Review
              </a>
            </td>
          <td>
            Kaustov Chakrabarti, Amanda M. Mohabir, Mahesh Nukala, Sara M. Safiullah, Tierney R. Woitas
          </td>
          <td>2025-11-21</td>
          <td>Scholarly Research In Progress</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Introduction: Acute Myeloid Leukemia (AML) is a heterogeneous malignancy characterized by the clonal expansion of immature myeloid precursor cells in the bone marrow and peripheral blood, leading to disrupted hematopoiesis and rapid disease progression. The incidence of AML increases with age, with a median age at diagnosis of approximately 68 years. Prognosis remains poor, particularly among older adults and those with adverse-risk genetic mutations, with 5-year overall survival rates often below 30%, even with intensive chemotherapy and hematopoietic stem cell transplantation (Ferrara et al., 2024; Ofran et al., 2023). With these clinical limitations, there remains a critical need for a better understanding of leukemogenic mechanisms and therapeutic resistance to guide novel and effective targeted treatments. In AML, long non-coding RNAs (lncRNAs) are now recognized as key players in leukemogenesis, disease progression, and therapeutic response. In particular, MALNC is a novel lncRNA recurrently overexpressed in AML subtypes harboring NPM1, IDH2 mutations, or the PML-RARA fusion gene, but its functional significance remains unclear (Esteller 2022). This study examined the role of MALNC in regulating transcriptomic response to all-trans retinoic acid (ATRA)–induced differentiation in acute promyelocytic leukemia (APL).
 Methods: A publicly available RNA-seq dataset GSE299161, originally designed to investigate the independent and ATRA-interactive effects of MALNC deletion on global gene expression, was reanalyzed. The dataset included RNA-seq data from two CRISPR-Cas9 knockout conditions, namely KOA (targeting MALNC exons 1.1 and 1.2), and KOB (extended deletion of exon 1.0) as well as untreated wild-type (WT) controls. Additionally, KOB and WT NB4 APL cells were treated with 1 µM ATRA for 72 hours. Differential expression was assessed with R packages (FDR < 0.05; |log₂FC| > 0.2).
 Results: No significant differentially expressed genes (DEGs) were identified between WT and KOA or between WT and KOB under untreated conditions, suggesting that deletion of MALNC exons 1.0, 1.1, and 1.2 does not independently perturb baseline gene expression in NB4 APL cells. By contrast, 72-hour ATRA exposure reprogrammed the transcriptome, generating 5,992 DEGs (2,973 up- and 3,019 downregulated genes) in WT and 5,202 ((2,670 up- and 2,732 downregulated)) in KOB cells. Among these, 2,907 transcripts (1,371 up- and 1,536 downregulated) showed significantly different log₂ fold changes between KOB and WT, demonstrating that MALNC modulates the magnitude of response to ATRA for a substantial subset of genes. A KOB-versus-WT comparison under ATRA uncovered 2,857 DEGs (1,463 up- and 1,394 downregulated); within this set, 237 genes (113 up-and 124 downregulated) were both ATRA-regulated and MALNC-dependent. This MALNC-sensitive cohort was enriched for canonical differentiation drivers (PML, ITGAM, CSF3R, ALDH1A2, SPI1, IRF1, GDF15, and TGM2, all markers of myeloid differentiation that are consistently induced during ATRA-mediated maturation of APL cells), while oncogenic or pro-survival transcripts (MYC, NPM1, BCL2, and POU2F2) were repressed, suggesting a MALNC-mediated amplification of pro-differentiation and anti-leukemic programs.
 Conclusions: MALNC deletion alone does not alter gene expression under steady-state conditions, indicating that it is dispensable for basal transcription, However, MALNC significantly influences the gene expression dynamics during ATRA-induced differentiation. These findings suggest that MALNC is a key modulator of retinoid-responsive gene networks and may serve as a potential therapeutic target or biomarker for treatment stratification in AML, particularly in subtypes exhibiting sensitivity to the retinoic acid pathway. Further functional studies are warranted to elucidate the molecular mechanisms by which MALNC interacts with ATRA signaling and its implications for overcoming therapeutic resistance in leukemia.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51b6e8550aab8996596e0130d08fe2b3a8f6f2ab" target='_blank'>
              Lncrna malnc modulates ATRA-induced transcriptional reprogramming in acute promyelocytic leukemia cells
              </a>
            </td>
          <td>
            Qian Wang, Kuan-Ying Yu
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 B-cell acute lymphoblastic leukemia (B-ALL) is the most common pediatric cancer, with Hispanic/Latino (H/L) children in the United States exhibiting the highest incidence rates. Despite this, the genetic basis of this susceptibility remains poorly understood. Genome-wide association studies identified several well-replicated single-nucleotide polymorphisms (SNPs) associated with childhood B-ALL risk. Recent analyses discovered a novel childhood B-ALL risk signal near the IKZF1 gene that is positively selected for in H/L populations. The putatively causal variant, rs1451367 C>T, resides downstream of IKZF1 in a 3'cis-regulatory element (3'CRE) defined by histone marks and chromatin accessibility. In addition, an independent B-ALL risk variant, rs17133807 G>A, found across ancestries lies 26 bp away in the same regulatory element. However, how this regulatory element mechanistically impacts IKZF1 expression remains unknown.
 To unravel the molecular role of the 3'CRE, we integrated chromatin accessibility data, 3D chromatin interaction mapping, computational transcription factor (TF)-binding predictions, and functional genomic assays. ATAC-seq profiling from 156 B-ALL patients revealed a distinct and variable accessible chromatin landscape at this 3'CRE in B cell progenitors and precursors, suggestive of highly-regulated and stage-specific activity. Of note, in the presence of the risk variants, allele-specific chromatin accessibility was reduced, implicating decreased enhancer activity. Further, Promoter Capture Hi-C data from human lymphoblastoid cells revealed strong interaction with the IKZF1 promoter, supporting the CRE's role as a bona fide IKZF1 enhancer. Having established its accessibility and promoter interaction in early B cells, we next sought to identify the TFs occupying the 3'CRE and determine how their binding is altered by the risk allele.
 Employing motifbreakR and delta SVM modeling, we predicted that rs1451367 would disrupt binding of several Erythroblast Transformation Specific family members, including ERG, ETV6, ELK3, in addition to altering a potential IKZF1 motif. Prioritizing these TFs with a negative differential binding score, we then performed chromatin immunoprecipitation (ChIP) coupled with quantitative real-time PCR (qPCR) to examine differential TF binding. To model the risk variants at this locus, we used CRISPR/Cas9 genome editing to introduce a 60 bp microdeletion spanning the 3'CRE variants in SEM cells, a human B-ALL cell line with a KMT2A-AFF1 translocation, in which we confirmed the CRE harbors accessible chromatin. ChIP qPCR revealed strong ERG occupancy at the 3'CRE, which was abolished in the presence of microdeletions of the risk variants. These findings implicate ERG as a key participant in orchestrating IKZF1 expression and demonstrate that the risk variant disrupts this interaction. Intriguingly, Nalm-6 cells, which harbor an IGH-DUX4 rearrangement, displayed a strikingly different pattern, with an absence of ERG binding at the 3'CRE. Western blotting detected an additional ERG isoform in these cells, which we identified as ERGalt – a dominant-negative isoform generated through alternative transcription driven by IGH-DUX4. ChIP across multiple B-ALL lines confirmed distinct binding profiles, underscoring the context-dependent regulation of this enhancer by ERG across leukemic subtypes. Notably, as IKZF1 itself is known to regulate ERG, our findings suggest a previously unrecognized feedback mechanism that might occur among these TFs. Also, we found the adjacent multi-ancestry risk variant to disrupt a TCF3 motif, with ongoing studies assessing its impact on IKZF1 transcription. Thus, to more finely dissect precise regulation at this element, we are conducting systematic mutagenesis studies at this CRE.
 Collectively, this work identifies a mechanistic model in which the Hispanic-specific risk variant disrupts ERG-mediated regulation of IKZF1, stalling B-cell differentiation at an early stage and priming cells for leukemic transformation. By identifying ERG as a novel transcriptional regulator at the IKZF1 3'CRE, our work reveals a previously unrecognized regulatory intersection between two key B-cell transcription factors. These insights not only elucidate how population-specific noncoding variation contributes to B-ALL susceptibility, but also highlight a regulatory axis that could be leveraged for enhancer-targeted or TF-modulating therapeutic interventions in the future.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/606e694bb6a2d590187674c0419d978017077562" target='_blank'>
              Dissecting mechanisms at an IKZF1 cis-regulatory element harboring ancestry-specific and cross-ancestry B-cell acute lymphoblastic leukemia risk variation
              </a>
            </td>
          <td>
            L. Messa, Lara Wahlster, Andrew Lee, Pardiss Mehrzad, Mateusz Antoszewski, Anna-Lena Neehus, , Alexis Caulier, Gaurav Agarwal, A. D. de Smith, Jonathan Paolino, Yana Pikman, Katia Georgopoulos, V. Sankaran
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="


 Background: The BCR::ABL1 fusion gene is a hallmark of chronic myeloid leukemia (CML) and is also present in a subset of acute lymphoblastic leukemia (ALL), acute myeloid leukemia (AML), and mixed phenotype acute leukemia (MPAL) cases. Traditional methods for identifying this important biomarker, such as FISH and RT-PCR, are limited in the ability to detect novel or complex variants and also do not assess additional genetic alterations that could impact therapeutic decision making. Whole Genome Sequencing (WGS) is a comprehensive alternative, with the ability to detect a wide range of genetic alterations, including single nucleotide variants, insertions, deletions, and structural rearrangements such as BCR::ABL1 fusions.
 Methods: We retrospectively analyzed 215 hematological clinical samples sent for testing at Tempus that received both WGS via Tempus xH and RNA-seq via Tempus xR. Patients had historical diagnoses of AML (n=146), MDS (n=41), CML (n=25), and other blood cancers (n=3).
 Results: We detected BCR::ABL1 fusions in 29 samples (20 CML, 7 AML, 0 MDS, 2 others) using a combination of WGS and RNA-seq. Of these, 90% (n=26) were detected by both methods. For the three discordant samples, one was detected by WGS and not RNA-seq and 2 were detected by RNA-seq with a low total read support not detected by the WGS assay. Of the 26 samples that had support from both WGS and RNA-seq, BCR breakpoints largely aligned with expectation based on the known p210 (major) and p190 (minor) variants, which differ between AML and CML. RNA-seq confirmed all (100%) of the exon 1-3 breakpoints in p190 and exons 13-14 in p210. Most notably, our analysis uncovered 2 abnormal breakpoints with clinical implications, which were confirmed via RNA-seq. One specimen had a breakpoint within exon 2 of ABL1 resulting in an in frame transcript with a shortened exon 2. The other abnormal breakpoint was in intron 2 of ABL1 that results in exon 2 loss and exon 3 retention. It is known that the next generation allosteric tyrosine kinase inhibitor asciminib loses efficacy without exon 3, suggesting the use of an alternative therapeutic inhibitor is more appropriate highlighting the clinical utility of breakpoint resolution of WGS. Lastly, we used the WGS data to assess the presence of any pathogenic or likely pathogenic mutations in ASXL1 and RUNX1, both of which may impact survival and treatment outcomes. In this small cohort (n=26), we found ASXL1 alterations in 27% and RUNX1 in 12%, providing additional insights WGS can play in identifying key molecular markers.
 Conclusions: Our findings demonstrate that WGS can not only identify known BCR::ABL1 fusion events but also uncover novel breakpoints and variants of clinical significance, particularly when used in conjunction with RNA-seq. These data suggest that WGS is a powerful tool for the comprehensive genomic profiling of leukemia, with potential implications for personalized medicine and targeted therapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b891de87beb0318f8e2faaa8c4ce8ce222937ff" target='_blank'>
              Whole genome sequencing uncovers novel BCR::ABL1 breakpoints and variants in leukemia: Implications for personalized medicine
              </a>
            </td>
          <td>
            R. Huether, Derick Hoskinson, Raul Torres, Karl Beutner, Yan Yang, Kelly Potts, Andrew Frazier, Adam Hockenberry, Brett Mahon, Charles Koyias, Cecile Rose Vibat, Victoria Chiou, Michael A. Thompson, K. Sasser, H. Nimeiri
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="


 Blood serves as a valuable biospecimen due to its ability to provide clinical insights. Peripheral blood mononuclear cells (PBMCs) can be readily isolated and are frequently used to monitor clinical progression and explore mechanisms of disease. Multiomic analyses provide powerful insights by linking open chromatin regions accessible to regulatory elements with gene expression. Here, PBMCs were profiled from a healthy and a diseased Acute Lymphoblastic Leukemia (ALL) donor at two timepoints six months apart. Analysis used 10x Genomics’ Single Cell Multiome ATAC + Gene Expression platform, enabling single cell RNA (scRNA-seq) and single cell ATAC sequencing (scATAC-seq) to profile transcriptomic and epigenomic landscapes simultaneously. Full-length T-cell receptor (TCR) sequences linked to clinical progression were also characterized using 10x Genomics’ GEM-X Immune Profiling technology. Over six months, profiling over 15,000 PBMCs, we identified an increase in naive B cells and a reduction in CD8+ effector-memory T cells, with a notable decline in the two most prominent TCR clonotypes, highlighting an altered adaptive immune response during disease progression. Parallel multiomic profiling uncovered distinct regulatory networks in advanced disease involving exhaustion markers TIGIT and LAG3 within T and NK cell populations. This study, enabled by the integration of multiple single cell technologies, provides critical insights into immune mechanisms driving cancer.



 Technological Innovations in Immunology (TECH)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d25a74164968a5001d5db1796dcb3bfa680db0f" target='_blank'>
              Multiomic analysis identifies epigenome alterations and effector-memory CD8+ T cell clonotypes associated with ALL disease progression 4323
              </a>
            </td>
          <td>
            Kelly Martin, Nandhini Raman, Connor Kunihiro, Dagmar Walter, Corey Nemec, Veronica Rodriguez, Sarah Taylor, Jill Herschleb, Peter Smibert, Josephine Lee
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Osteosarcoma (OS), the most common primary malignant bone tumor, remains a therapeutic challenge because of its high metastatic potential, chemoresistance, and poor prognosis. Mutations in the TP53 tumor suppressor gene, including loss-of-function (LOF) and gain-of-function (GOF) mutations, play a central role in OS progression by disrupting cell cycle regulation, DNA repair, and apoptosis and promoting immune evasion and metabolic reprogramming. This review provides an in-depth analysis of p53 biology in OS, highlighting its impact on therapeutic resistance and tumor progression. We discuss advancements in radiotherapy, chemotherapy, and immunotherapy, emphasizing strategies targeting mutant TP53 and its associated pathways. Emerging approaches, including metabolic reprogramming, noncoding RNA regulation, and precision biomarkers such as miRNAs and histone modifications, offer promising tools for diagnosis, risk stratification, and treatment optimization. By linking the molecular mechanisms of p53 with novel therapeutic strategies, this review underscores opportunities for translational research aimed at improving the clinical outcomes of OS patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4de5949ca03e6677f6b90cc0b23d966a37a92dc" target='_blank'>
              Involvement of TP53 in osteosarcoma - challenges and prospects
              </a>
            </td>
          <td>
            Yue Shen, Shuzhou Huang, Geng Chen, Guangda Wang, L. Sui
          </td>
          <td>2025-11-13</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da24476f2b5b20f3dabd541d60d10f29d3bbfa00" target='_blank'>
              Rapid Autopsy Multi-Omic Analysis Identifies Divergent Evolutionary Trajectories and DNA Damage Resistance Mechanisms in FGFR2-Driven Cholangiocarcinoma
              </a>
            </td>
          <td>
            A. Sheel, A. Paruchuri, J. Reeser, M. Wing, R. Vella, E. Samorodnitsky, A. Smith, T. Dao, E. L. Hoskins, R. Bonneville, H. Z. Chen, C. Li, Z. Risch, P. Allenby, A. G. Freud, A. D. Fitzthum, W. Chen, S. Roychowdhury
          </td>
          <td>2025-12-04</td>
          <td>None</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="


 INTRODUCTION T-Large Granular Lymphocyte Leukemia (T-LGLL) is a rare lymphoproliferative disorder characterized by the clonal expansion of cytotoxic LGLs. The disease exhibits significant heterogeneity, with clinical manifestations ranging from indolent to life-threatening conditions. T-LGL lymphoproliferation is driven by both somatic gain-of-function mutations, mostly in STAT3 and STAT5B genes, and by a pro-inflammatory microenvironment. STAT3 mutations characterize the CD8+ T-LGLL subtype and are strongly associated with symptomatic disease, especially neutropenia, which negatively impacts patient outcome. Currently, immunosuppressive treatments have limited efficacy with a high rate of relapsed/refractory cases. Therefore, identifying novel molecular targets is paramount to improve therapeutic strategies. We recently demonstrated that miR-146b downregulation induces a high Fas ligand (FasL) production and subsequent neutrophil death, through STAT3-miR-146b-ELAVL1-FasL axis. Building on these data, we aimed to investigate the molecular consequences of miR-146b restoration in patient-derived and normal LGLs, as the first step to proceed towards developing a possible innovative class of RNA-based therapies for STAT3-mutated T-LGLL.
 METHODS Six CD8+ STAT3-mutated T-LGLL patients were selected alongside 6 healthy donors (HD) matched for sex and age. LGLs were isolated from patients based on LGL-specific marker CD57 expression, while CD8+ cytotoxic T lymphocytes (CTLs) were purified from HD. Primary cells were transfected with either a miR-146b mimic or a negative control (scramble). After 24 hours, cell viability and miR-146b transfection efficiency were assessed, and total RNA was extracted to perform RNA sequencing. Gene expression was quantified using CircComPara. Differentially expressed gene (DEG) analysis was conducted using DESeq2 v1.46.0 (adj-P≤0.05) and gene set enrichment analysis using clusterProfiler v4.14.6 and enrichplot v1.28.0.
 RESULTS MiR-146b uptake at 24 hours post-transfection compared to scramble control was confirmed (p<0.01), maintaining a cell viability of at least 60%. According to transcriptomic data and descriptive principal component analysis, the changes between scramble and miRNA-transfected conditions are marked in patient-derived cells (2,404 DEGs) whereas are mild in HD samples (154 DEGs). Our data demonstrated that miR-146b restoration was able to successfully normalise the expression of its known targets, most importantly ELAVL1, leading to FasL reduction, thus reverting the pathogenic axis associated with neutropenia in T-LGLL.Interestingly, miR-146b is found to exert the desired effect not only on ELAVL1, but also on another target, IRAK1, a kinase that is crucial for the response against foreign pathogens and has been associated with STAT3 activation. In addition, several genes including epigenetic modifiers (e.g., HDAC1), oncogenic regulators (e.g., JUNB), and inflammatory mediators (e.g., IL32 and IL2RB), were downregulated following miR-146b restoration, reaching expression levels comparable to those in HD. Interestingly, CCL5, a chemokine implicated in LGL proliferation via IL-6 production by monocytes, and SBNO2, required in STAT3-dependent hematopoietic malignancies,were significantly reduced in miR-146b-transfected LGLs (p<0.01). Additional analyses disclosed that several signaling pathways, including TNFα/NFkB, IL6/JAK/STAT3 signaling, inflammatory responses, and viral carcinogenesis, were enriched in genes downregulated upon miR-146b restoration in leukemic LGLs, whereas miR-146b transfection does not elicit significant pathway perturbations in normal CTLs. Finally, we demonstrated that the expression level of nearly half of the genes altered in the disease was completely restored or positively affected by the miR-146b restoration, while only a small proportion of genes not altered in the disease were affected, mostly very mildly, by the treatment.
 CONCLUSIONS Our results demonstrate that miR-146b restoration selectively affects leukemic cells with negligible effect on healthy CTLs. Moreover, robust data support the ability of miR-146b restoration to correct key targets implicated in T-LGLL-associated neutropenia. The miR-146b restoration also modulates the expression of genes and the activation of pathways aberrantly regulated in leukemic cells from the most symptomatic disease subtype, suggesting a broader anti-leukemic effect.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80c79f431a02f1d4dc06c8355237d70b1327aa4c" target='_blank'>
              MiR-146b restoration in T-large granular lymphocyte leukemia: Molecular insights for a new therapeutic strategy
              </a>
            </td>
          <td>
            A. Teramo, E. Rampazzo, Eleonora Roncaglia, B. Mariotti, Giulia Calabretto, Elena Buson, Enrico Gaffo, Silvia Orsi, Rebecca Crivellari, Giulia Pertile, Valentina Trimarco, M. Facco, F. Bazzoni, L. Trentin, G. Semenzato, R. Zambello, Stefania Bortoluzzi
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="R-loops consist of double-stranded DNA-RNA hybrids and a complementary DNA strand that is displaced from the duplex. R-loops play important role in numerous normal physiological processes, including DNA methylation, chromatin remodeling, RNA editing, replication, DNA repair, immunoglobulin class switching, and chromosome segregation during cell division. However, excessive or untimely formation of R-loops can lead to replicative collapse and subsequent DNA damage, resulting in genomic instability. One type of genomic rearrangements that is strongly associated with cancer malignancy is the extrachromosomal amplification of genes on circular DNA molecules (ecDNA). These molecules are relieved of hereditary constraints and conventional segregation laws and can endow cancer cells with the ability to rapidly change their genome, thereby accelerating tumor evolution and the development of therapy resistance. Multiple lines of evidence indicate that upregulated transcription of a gene can increase its susceptibility to amplification. Although the mechanisms underlying these processes are not yet fully understood, R-loops may play an important role in initiating gene amplification. In this review, we highlight the role of R-loops in replicative collapse, double-strand breaks, and DNA damage repair. We also provide examples of gene amplifications that is known to be induced by R-loops. Finally, we discuss amplification mechanisms in which involvement of R-loops has not yet been demonstrated, but appears highly likely.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d383d8d8b2013ac53fe539c01740ab644c592c14" target='_blank'>
              R-loops as a trigger for intra- and extrachromosomal DNA amplification in cancer
              </a>
            </td>
          <td>
            Tatyana F. Kovalenko, Amal Abdurazakov, Nadezhda V. Antipova, Mikhail I. Shakhparonov, M. Pavlyukov
          </td>
          <td>2025-12-03</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="


 Genomic alterations in germ cell tumors (GCTs) have been identified in the RTK, MAPK, and PI3K pathways through whole exome sequencing, with a prevalence of 71% in germinomas but only 33% in non-germinomas GCTs (NGGCTs). Despite evolution in treatment, GCTs present therapeutic challenges and lack molecularly targeted strategies.



 Whole genome sequencing was performed on 71 cases (37 germinomas and 34 NGGCTs) from the iGCT Genome Analysis Consortium and the Japan Children’s Cancer Group.



 Putative driver mutations were identified in 83% of tumors, primarily involving RAS/MAPK followed by mTOR/Akt pathways. Germinomas exhibited a higher mutation frequency than NGGCTs (84% vs 62%), with KIT enriched in germinomas and USP28 in NGGCTs. All USP28 alterations were bi-allelic and highly specific to CNS GCTs, with only one diffuse midline glioma out of 1448 cases of pediatric cancers affected. Variant allele frequency and clonal analyses revealed that these mutations occurred early, preceding copy number alterations. Germline analysis identified Kleinfelter or Down syndrome in 5.6% of cases, particularly in NGGCTs lacking other alterations.



 Many cases exhibited driver mutations present in major clonal populations and could serve as therapeutic targets. There were notable differences based on histological type, with a diverse range of single nucleotide variants/structural variants observed in NGGCTs. A timeline of tumorigenesis from SNVs/SVs to copy number abnormalities was observed. The utility of understanding germline variants was also implicated in the pathogenesis.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6633a5d8f98dc245c8a327ae1335afe0892cad2" target='_blank'>
              PTHP-06. Molecular Mechanisms and Developmental Pathways of CNS Germ Cell Tumors Elucidated through Whole Genome Analysis
              </a>
            </td>
          <td>
            Hirokazu Takami, S. Kato, Yuki Yamagishi, Eiji Sugihara, Y. Matsushita, Takako Yoshioka, Junko Hirato, Miho Kato, Hajime Okita, Tomonari Suzuki, Masayuki Kanamori, Akio Asai, Akitake Mukasa, Toshihiro Kumabe, Akihide Kondo, Yuichi Hirose, M. Nagane, Kazuhiko Sugiyama, T. Shibata, Kenichi Yoshida, Y. Shiraishi, Takaaki Sato, Ryo Nishikawa, Motohiro Kato, Koichi Ichimura
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>85</td>
        </tr>

        <tr id="The cellular programs that mediate therapy resistance are often important drivers of metastasis, a phenomenon that needs to be understood better to improve screening and treatment options for patients with cancer. Although this issue has been studied extensively for chemotherapy, less is known about a causal link between resistance to radiation therapy and metastasis. We investigated this problem in triple-negative breast cancer and established that radiation-resistant tumor cells have enhanced metastatic capacity. Resistance to radiation increases the expression of integrin β3 (ITGB3), which promotes enhanced migration and invasion. Bioinformatic analysis and subsequent experimentation revealed an enrichment of RNA metabolism pathways that stabilize ITGB3 transcripts. Specifically, the RNA binding protein heterogeneous nuclear ribonucleoprotein L (HNRNPL), whose expression is regulated by Nrf2, mediates the formation of circular RNAs that sponge the family of let-7 microRNAs that target ITGB3. Collectively, our findings identify a mechanism of radiation-induced metastasis that is driven by alterations in RNA metabolism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8064b7e84c02e1178a653876a56f4f95d2d1c28" target='_blank'>
              Resistance to radiation enhances metastasis by altering RNA metabolism
              </a>
            </td>
          <td>
            Ayush Kumar, Kensei Kishimoto, H. Goel, Christi Silva, Rui Li, Brendan A Pacheco, L. J. Zhu, William A. Flavahan, A. Mercurio
          </td>
          <td>2025-10-17</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Gastrointestinal (GI) cancers pose a significant global health burden, driven by complex molecular alterations and microenvironmental interactions. Advances in molecular pathogenesis have elucidated recurrent driver gene mutations, such as KRAS, TP53, and APC, alongside dysregulated signaling pathways including Wnt, RAS-MAPK, and PI3K-AKT, which collectively underpin tumor initiation and progression. Complementing genetic changes, epigenetic alterations—such as DNA hypermethylation, histone modifications, and regulatory non-coding RNAs—further contribute to malignant evolution by reshaping chromatin architecture and gene expression. These mechanisms not only promote uncontrolled proliferation but also reinforce therapeutic resistance by dynamically modifying the tumor microenvironment (TME). Molecular subtyping efforts, including The Cancer Genome Atlas (TCGA) classification for gastric cancer (GC) and the Consensus Molecular Subtypes (CMS) for colorectal cancer (CRC), have delineated disease heterogeneity, revealing distinct pathogenic pathways and enabling refined prognostic stratification. Such insights provide the biological rationale for diagnostic techniques and targeted interventions. For instance, anti-EGFR and anti-VEGF monoclonal antibodies disrupt oncogenic signaling and tumor angiogenesis, respectively, and have demonstrated substantial clinical efficacy in selected patient populations. In parallel, immunotherapy has emerged as a transformative modality in oncology. Immune checkpoint inhibitors targeting PD-1/PD-L1 and CTLA-4 reinvigorate antitumor immunity and have reshaped standard-of-care protocols for several GI malignancies. Beyond conventional immunotherapies, innovative strategies such as CAR-T cell therapy and neoantigen-based vaccines are being actively investigated. These approaches aim to overcome immune evasion mechanisms and enhance tumor-specific targeting, offering promise for patients with resistant or advanced disease. This review comprehensively analyzes the evolving molecular landscape of GI cancers and the corresponding development of targeted and immunotherapeutic agents. It highlights a balanced integration of mechanistic discovery and clinical translation, underscoring their synergistic roles in advancing precision oncology and improving survival outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5602f60be23d19b8d126a3c7ae3c86407dda38e" target='_blank'>
              Gastrointestinal cancer: molecular pathogenesis and targeted therapy
              </a>
            </td>
          <td>
            Yang Jin, Xiaobo He, Yanfeng Wu
          </td>
          <td>2025-12-01</td>
          <td>Molecular Biomedicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancer is a deadly disease affecting millions of people worldwide. Circulating tumor cells (CTCs) represent a critical link between primary malignancies and metastasis, acting as key players in cancer dissemination, progression, and recurrence. Although rare, CTCs offer a valuable, non-invasive window into tumor biology and the evolution of disease in patients. CTCs can exist as single cells in the circulation, but some are shed and travel in larger groups, referred to as CTC clusters. These clusters possess a greater oncogenic potential compared to individual CTCs. In this review, we aim to provide insight into the dynamic biological processes underlying CTC generation, biology, and survival, with a focus on epithelial-to-mesenchymal transition (EMT) and beyond like cancer stem cells (CSCs), cellular plasticity, and senescence. A crucial aspect of CTC biology is EMT, a process that imparts cancer cells with increased motility, invasiveness, resistance to apoptosis, and the ability to intravasate and evade the immune system. Beyond EMT the cancer cells show further plasticity, allowing epithelial tumor cells to adopt mesenchymal or hybrid phenotypes, which enables adaptation to a changing microenvironment and enhances therapy resistance. Moreover, a subset of cancer cells can acquire stem cell-like properties, including self-renewal and tumor-initiating capacity. EMT, along with processes such as dedifferentiation, contributes to the generation of cancer stem cells. In recent years, studies have also highlighted the complex and paradoxical role of senescence in CTC biology. While senescence typically results in permanent cell cycle arrest, in cancer cells it may be reversible and can promote tumor cell dormancy, immune evasion, and metastatic reactivation. By exploring the connections between CTCs, EMT, CSCs, plasticity, and senescence, we aim to shed light on the unique biology of CTCs, their metastatic potential, and their contributions to tumor heterogeneity. We hope that a better understanding of these processes will help advance the development of novel biomarkers and therapeutic targets for solid tumors beyond EMT.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a99a639294b1ce88e2f796dfc138b9bd8dbb8609" target='_blank'>
              Circulating tumor cells: indicators of cancer progression, plasticity and utility for therapies
              </a>
            </td>
          <td>
            Tamás Richárd Linkner, Z. Nagy, Alexandra Kalmár, Eszter Farkas, Fruzsina Bányai, Nikolett Szakállas, István Takács, Béla Molnár
          </td>
          <td>2025-10-20</td>
          <td>Pathology and Oncology Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="
 Introduction:
 While genetic variants linked to dilated cardiomyopathy (DCM) are increasingly well characterized, the molecular mechanisms driving pathogenesis remain poorly understood. Traditional bulk and
 in vitro
 methods lack the resolution to capture the cellular heterogeneity and complexity of DCM. Here, we use single-nucleus multiome chromatin accessibility and RNA sequencing to resolve cell-type-specific gene regulatory changes in genetic DCM.


 Methods:
 We performed 10X Genomics Multiome (simultaneous RNA and chromatin accessibility) profiling on nuclei isolated from left ventricular tissue of 13 DCM patients with diverse genetic etiologies (
 LMNA
 ,
 PLN
 ,
 RBM20
 ,
 TNNT2
 ,
 TTN
 ) and 5 non-failing hearts. Integrated analysis identified 12 distinct cardiac cell types based on joint transcriptomic and epigenomic signatures. Intercellular communication networks were inferred using CellChat. We leveraged paired RNA and chromatin accessibility data to elucidate potential enhancer-gene interactions driving disease-associated cell state changes.


 Results:
 We observed disease-specific alterations in cellular composition and signaling networks across DCM genetic etiologies. Integrated chromatin accessibility and gene expression data revealed changes in the gene regulatory networks of cardiomyocytes, fibroblasts, and endothelial cells. Notably, the transcription factor
 PRRX1
 emerged as a key differentially regulated gene in
 LMNA
 -related DCM. Functional assays in iPSC-derived cardiomyocytes demonstrated that
 PRRX1
 knockdown significantly improved calcium handling and reduced arrythmia incidence.


 Conclusions:
 This single nucleus multiomic atlas of genetic DCM provides insights into cell-type-specific gene regulation and cell communication programs associated with disease states. Our findings highlight
 PRRX1
 as a potential therapeutic target in
 LMNA
 -related DCM and underscore the value of multiomic profiling for uncovering the regulatory basis of cardiomyopathies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a1fd5b61e122e81186725db0bb2595464a61a11" target='_blank'>
              Abstract 4370075: Leveraging Single-Nucleus Chromatin Accessibility and Transcriptome Profiling to Define the Regulatory Landscape of Dilated Cardiomyopathy
              </a>
            </td>
          <td>
            Logan Dunkenberger, Casey Gilles, Nadjet Belbachir, Eyal Metzl Raz, Lasemahang Limbu, Spencer Bonham, J. Arthur, Alex Dalal, Maryam Kay, Rachel Baum, Evgenios Neofytou, Rohin Ramchandani, Mo Ameen, Kevin Wang, Michael P. Fischbein, Y. J. Woo, Ioannis Karakikes
          </td>
          <td>2025-11-04</td>
          <td>Circulation</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 Mature T-cell lymphomas (TCL) are a heterogeneous group of non-Hodgkin lymphomas arising from the malignant transformation of post-thymic T and NK cells. These lymphomas are characterized by poor response to chemotherapy, high relapse rates, and dismal survival outcomes. Genomic analyses of TCL have revealed recurrent alterations affecting intracellular signaling pathways and epigenetic regulators. The frequent involvement of epigenetic modifiers suggests that epigenetic dysregulation is a critical step in TCL pathogenesis.
 To identify the epigenetic changes driving TCL transformation, we analyzed chromatin modifications and gene expression profiles in murine TCL models that closely recapitulate human disease. Our data revealed that Id2, a repressor of basic helix-loop-helix proteins, is consistently overexpressed at both RNA and protein levels across multiple murine lymphoma models. Supporting this, analysis of the Cancer Cell Encyclopedia identified TCL as one of the tumor types with the highest ID2 expression, which we confirmed by Western blot in human TCL-derived cell lines. Furthermore, analysis of single-nuclei RNAseq data from a cohort of primary angioimmunoblastic T-cell lymphoma (AITL) and peripheral T-cell lymphoma not otherwise specified (PTCL, NOS) patient samples generated in our laboratory, demonstrated significant ID2 overexpression in a subset of cases (12/27; 44%) compared to tonsil and lymph node controls. Interestingly, all relapsed TCL cases in our cohort showed significant overexpression of ID2. Based on our data, we hypothesize that ID2 is epigenetically regulated during transformation and contributes to T-cell lymphomagenesis.
 To define the mechanisms leading to ID2 overexpression in TCL, we performed H3K27ac and H3K4me1 ChIP-seq, ATAC-seq, reverse-ChIP coupled with mass spectrometry, and dual-luciferase reporter assays. These analyses identified a conserved region near the ID2 locus enriched for H3K27ac and H3K4me1 marks that co-localizes with ATAC-seq peaks and is ranked as a super-enhancer by the ROSE algorithm. This region is specifically activated in mature TCL cells compared to control CD4⁺ T cells. Reporter assays using the most conserved segment within the enhancer region showed more than twofold increased luciferase activity relative to a minimal promoter, confirming its enhancer function. Interestingly, reverse-ChIP identified CTCF and ASCL1 as candidate regulators of enhancer looping and transcriptional activation, respectively. To characterize the proteins mediating the epigenetic function of ID2 in TCL, we employed a biotin ligase proximity labeling assay (BioID) approach coupled with mass spectrometry. Our results identified known ID2 partners, including the E-proteins TCF3, TCF4 and TCF12, key regulators of T-cell differentiation, as well as novel partners including elements of the SWI/SNF chromatin-remodeling complex, suggesting that ID2 overexpression indirectly regulates transcriptional programs critical for lymphomagenesis.
 To determine the functional role of ID2 in TCL, we characterized the effects of both genetic and pharmacological inhibition of ID2, using shRNA and CRISPR-Cas9 mediated knockout and novel pan-ID inhibitors AGX51 and AGXA, respectively. Loss of ID2 leads to decreased cell viability and increased apoptosis in in vitro TCL models. Interestingly, RNA-seq analysis revealed enrichment in signatures associated with glucose metabolism, suggesting a novel role for ID2 in tumor metabolic regulation. Finally, in vivo treatment with the pan-ID inhibitor AGX51 in a murine lymphoma model induced a modest but consistent reduction in tumor burden, spleen weight, and activated CD69⁺ T cells population in AGX51-treated mice compared to vehicle-treated controls.
 In summary, we have identified a novel TCL-specific super-enhancer that drives ID2 overexpression in TCL and demonstrated an essential role for ID2 in supporting lymphoma proliferation and survival. More importantly, pharmacological targeting of ID2 reduces tumor burden in vitro and in vivo, highlighting its potential as a therapeutic target in mature T-cell lymphomas.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e31f2dc0fbeb09da6f80dbe4eff065e3af3490a0" target='_blank'>
              Inhibitor of DNA binding 2 (ID2) is epigenetically regulated during lymphomagenesis and constitutes a novel therapeutic target in mature T-cell lymphoma
              </a>
            </td>
          <td>
            Anouchka P. Laurent, Hannah Miller, Ryan D. Najac, R. Albero, Jose R. Cortés, Cindy Ma, W. Lin, Anqi Wang, Ruth Alonso Alonso, L. Quevedo, Rajesh Soni, O. Ouerfelli, Raul Rabadan, R. Benezra, Teresa Palomero
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="


 Introduction:
 The mutational landscape of chronic lymphocytic leukemia (CLL) is characterized by a broad spectrum of recurrent genetic alterations, which can generally be mapped onto a limited set of key signaling pathways. However, the genetic programming within CLL cells alone is insufficient explain their survival and proliferation capabilities. Instead, interactions with the microenvironment are critical drivers of disease initiation, progression, and maintenance. Therefore, gaining a comprehensive understanding of the complex interplay between genetic abnormalities and microenvironmental signals holds promise for improving prognostic accuracy and tailoring therapeutic strategies. Signaling patterns derived from RNA expression analyses represent an integrative approach to capture the combined influence of both intrinsic genetic factors and extrinsic microenvironmental cues. Despite this potential, the individual variability in CLL cellular responsiveness at the signaling level has not been extensively explored to date.
 Methods:
 We utilized a large publicly available dataset generated by Knisbacher et al., employing a custom-built bioinformatics pipeline with rigorous quality control measures to identify differential signaling patterns across CLL samples. This approach enabled us to define so-called “signalotypes,” distinct signaling phenotypes characterized by unique pathway activation profiles. Subsequent analyses linked these signalotypes to clinical annotations within the dataset. We then applied advanced machine learning algorithms to capture the molecular signatures underlying each signalotype, allowing accurate classification of fully annotated CLL samples from our own biobank into the established signalotype clusters. We then conducted extensive in vitro toxicity screens using these biobank samples. We tested a broad range of compounds in solo culture conditions and selected subsets of substances under more complex coculture setups, simulating microenvironmental interactions.
 Results:
 Our bioinformatics analysis of RNA-seq from 661 chemo-naïve CLL patients from the large public dataset yielded a robust definition of 11 signalotypes, supported by multiple layers of validation. These groups exhibited marked differences in the activation status of signaling pathways and metabolic activity, highlighting underlying biological heterogeneity. Notably, the clusters demonstrated substantial divergence in clinical outcomes, including time to first treatment and overall survival, suggesting clinically relevant biological distinctions. Moreover, established molecular markers such as IgHV mutation status, cytogenetic abnormalities, and mutations in recognized CLL driver genes were non-randomly distributed among the signalotypes. We trained an ensemble of machine learning models on the molecular patterns of each signalotype, enabling the mapping of 101 individual patient samples from our biobank to their corresponding clusters with high confidence and performed high-throughput drug screening using 117 substances. We found considerable heterogeneity in drug response profiles across different signalotypes. Our findings suggests that our screening approach can effectively differentiate between broadly effective agents (like Venetoclax) and those with more selective activity tied to specific biological subgroups. More detailed analysis of 20 substances in coculture settings validated the high heterogeneity between signalotypes.
 Conclusions:
 Our study demonstrates that signaling patterns derived from RNA sequencing data can be used to stratify CLL patients into subgroups with distinct biological characteristics, clinical trajectories, and treatment responses. Machine learning techniques enable the extraction and recognition of core molecular features defining each signalotype, allowing accurate patient classification based on these signatures, also allowing to extract a minimum number of features that are essential to define the molecular core of a specific signalotype. By integrating these insights with functional drug screening, we uncover substantial variability in therapeutic susceptibility across CLL subgroups, which could inform more personalized treatment approaches.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2dd5ac1a3b1e01da51d7ef1dbc6257f4054c939" target='_blank'>
              Machine learning-based classification of chronic lymphocytic leukemia signalotypes to allows biological and clinical insights into novel CLL subgroups
              </a>
            </td>
          <td>
            D. Asslaber, Jennifer Forster, Tina Hofer, N. Zaborsky, Franz F Gassner, T. Pemovska, Philipp B. Staber, R. Greil, A. Egle
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/234e6b6b6481eb76db70b44e7b7d276eb9ea9a24" target='_blank'>
              Population-scale multiome immune cell atlas reveals complex disease drivers
              </a>
            </td>
          <td>
            M. Kanai, T. Delorey, J. Honkanen, R. S. Rodosthenous, J. Juvila, S. Murphy, I. Teixeira-Soldano, H. S. Hwang, J. Karjalainen, J. Halonen, G. Panagiotaropoulou, Y. Zhang, C. McCabe, , K. Nanki, T. Yoshida, K. Liu, M. Glean, N. Mehrotra, E. Finan, D. Chafamo, Y. Zhu, M. Arvas, S. Ruotsalainen, Z. Zheng, FinnGen, M. Reeve, M. Kurki, C. B. Porter, O. Ashenberg, W. Zhou, K. PitkaÌnen, J. Partanen, A. Palotie, D. B. Graham, M. J. Daly, R. J. Xavier
          </td>
          <td>2025-11-27</td>
          <td>None</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="


 Epigenetic deregulation is a major driver for acute myeloid leukemia (AML) including those with Mixed-Lineage Leukemia gene-rearrangement (MLLr). Over the past decades, significant efforts have been made by others and us to identify and target key components associated with oncogenic transcriptional complexes for development of more effective treatments. Among the inhibitors targeting components of MLL complexes, MENIN and DOT1L have successfully made to the clinics. Although promising results from initial clinical trials demonstrate their safeness and potential efficacy, the complete remission rates remained low (<10% for DOT1Li and 33% for MENINi) and short-lived with only 25% of AML patients treated with MENINi being MRD free in the 2 years follow up, urging a better understanding of the underlying mechanisms and the development of effective combination therapies. Strikingly, while suppression of key MLL fusion targets such as Hoxa and Meis1 by these epigenetic inhibitors could be detected in most patients, their downregulation did not strictly correlate with or predict treatment response, suggesting that these inhibitors may also act by different mechanisms. To this end, our study aims to better understand the leukemia suppressive functions of DOT1L/MENIN inhibitors and thereby facilitate the rational development of more effective treatments.
 To identify key pathways commonly affected by inhibitors targeting DOT1L/MENIN complexes, we performed RNA-seq on both primary MLL-AF9 AML derived from mouse hematopoietic stem/progenitor cells (HSPCs) and human MLL-r AML cell line MV4;11 with or without DOT1Li (EPZ004777) or MENINi (MI-503). In addition to the classical Hoxa/Meis1 pathways, both DOT1L and MENIN inhibitions exhibited significant suppression of pathways associated with DNA replication, closely linked to DNA damage and leukemia cell survival. Upon replication stress, two major pathways, namely fork reversal and repriming, can be used to potentially overcome obstacles to DNA synthesis. Fork reversal is associated with slower fork speed and involves key DNA replication/repair proteins such as PARP to stabilize reversed forks during pausing. Conversely repriming requires Primpol to bypass obstacles without slowing fork progression. To assess the impact of DOT1Li and MENINi on DNA replication fork stability and dynamics, DNA fibre assays showed that MLL fusion transformed cells heavily rely on repriming for overcoming replication stress. Strikingly, upon DOT1Li/MENINi treatment, MLL fusion transformed cells were forced to switch to fork reversal resulting in slower fork speed, which may in part account for the inhibitory effects. To further confirm this, we generated a novel MLL-AF9 leukemia model using mouse HPSCs, where Primpol can be inducibly inactivated upon 4-OHT treatment. As a result, we show that inactivation of Primpol phenocopies DOT1Li/MENINi and could drive MLLr-AML to adapt fork reversal as a coping mechanism. Together these results reveal a significant impact of these epigenetic inhibitors on DNA replication fork dynamics and repair mechanisms.
 Switching from fork repriming to reversal by the epigenetic drugs can lead to a reliance on PARP proteins that stabilize reversed forks during the arrest and protect them from premature restart, thereby creating a novel vulnerability to PARP inhibition. To test this hypothesis, PARPi was used in combination with DOT1Li/MENINi in MLLr-AML. Strikingly, while PARPi alone had no significant impact on fork dynamics, it triggered premature fork restart in DOT1Li treated cells, leading to increase fork speed and subsequent fork collapse, revealing a potential vulnerability created by DOT1Li. Consistently, combination treatment significantly suppressed cell growth of MLLr-AML, which displayed significantly shorter fork length and excessive DNA damage as compared to control and single treatments. Finally, we also tested the efficacy of these novel regimens in vivo. While single treatments with PARPi, DOT1Li, or MENINi exhibited no or moderate effects on disease progression, combo treatments in particular MENINi+PARPi significantly prolonged the disease latency with all animals remaining disease-free at the experimental endpoint. Taken together, these data reveal that DOT1L/MENIN inhibition alter replication fork plasticity and induce a novel vulnerability that can be therapeutically targeted by PARP inhibition for effective leukemia treatment.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/757b773a243504fb282db04d14daf965acdd6ee2" target='_blank'>
              DOT1L/menin inhibitors target replication fork plasticity and create novel vulnerability to PARP inhibitor in MLL-rearranged AML
              </a>
            </td>
          <td>
            T. Fung, Cyril Dördelmann, Danyue Peng, Ali Akin, Jennifer Lynch, Nga Ieng Priscilla Lau, B. Zeisig, Larissa Bomfim, Lu Zhao, Polina Maximchik, Massimo Lopes, Chi Wai Eric So
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="


 Genomic analysis of multiple myeloma has advanced greatly in recent years, but traditional short-read sequencing has limited the power to examine complex or linked events across the genome. Linking markers together into large phase-blocks, and examining the interaction of different sequencing modalities, allows complex genomic and epigenomic states to be integrated to generate true multiomic haplotypes at the chromosomal level.



 Fourteen multiple myeloma patient-derived xenografts were used to generate multiomic data including short- (Illumina) and long-read (PacBio) whole genome sequencing (WGS, average depths 92x and 16x) to identify single nucleotide variations (SNVs), copy number (CN) abnormalities, structural variation (SV), and DNA methylation, as well as expression (>124 million reads), chromatin states (Cut&Tag-IT, >10M reads, H3K27ac, H3K4me1, H3K4me3, H3K9me3, H3K27me3, and H3K36me3), and Micro-C/LinkPrep (Dovetail Genomics; >900M reads) and HiChIP (Dovetail, H3K27ac mark) to identify 3D chromatin architecture/folding and enhancer-target gene interactions. Micro-C/LinkPrep interaction reads were also used for chromosome-scale haplotype phasing, onto which SNVs, SVs, CNVs, DNA methylation, expression, and chromatin marks were super-imposed based on heterozygous SNPs. Data integration resulted in haplotype-phased somatic events for genomic architecture, SNVs, SVs, epigenomic states, and expression markers.



 For the first time, complete haplotype-resolved assemblies at the chromosomal level have been generated in multiple myeloma. A 100x coverage genome Micro-C/LinkPrep libraries we were able to phase the long and short arms of each chromosome, resulting in an average 96.3% haplotype phasing per autosome – thereby generating chromosome length haplotypes up to 241.9 Mb in length. In comparison, HiFi long-read (15 kb reads) WGS (PacBio) was able to generate phase-blocks of up to 4.6 Mb (median 44.1 kb).
 We examined the prevalence of somatic mutations on each haplotype across all chromosomes to determine if one parental haplotype was more likely to be mutated than the other. In one sample, of 19,306 mutations 56% were phased to their respective haplotypes and were generally equally distributed across both haplotypes.
 Haplotype-specific interaction heatmaps were generated allowing us to examine the interaction of complex SVs across chromosomes. In one sample, we identified a primary t(11;14) and additional SV events linked to the t(11;14) including a t(3;14), t(11;17), and t(3;17) which created a cyclical pattern. Using the chromosome-scale haplotype maps we were able to determine the proportion of interacting reads for the four possible haplotype combinations between any two chromosomes. We found that 70-83.5% of reads support specific combinations of haplotype interactions, and that all four SV events were linked and involved the same haplotypes on each chromosome. The pattern of interactions combined with breakpoint analysis in this case indicated a four-way complex reciprocal translocation between chromosomes 3,11,14 and 17.
 Integration of epigenetic data in this complex SV, including DNA methylation (HiFi reads) and super-enhancer histone marks (CUT&Tag-IT), showed DNA hyper-methylation 17 kb upstream of CCND1 next to the t(11;14) breakpoint as well as increased H3K27ac marks on the same haplotype, indicating spreading of the activating broad domain from the IGH super-enhancer on chromosome 14. Equally, the hypomethylated DNA marks at the IGH promoter are spread to chromosome 3, via the t(3;14), resulting in over-expression of the proto-oncogene SKIL. The same is true for the t(3;17), which shows hypomethylation on both sides of the breakpoint, compared to the non-translocated allele. Overall, this exemplifies the underlying intricacy and impact of complex SVs across multiple linked chromosomes and their epigenetic states.



 We have generated the first chromosome scale haplotype-resolved genomes in multiple myeloma and integrated them with epigenetic states. We have shown that it is possible to identify interactions across chromosomes to resolve complex SVs as well as their epigenomic consequences to understand the intricate nature of how the genome is organized.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e6488520a62edae9da4095a747986d863374bdb" target='_blank'>
              Multiomic evidence of coordinated complex rearrangements, enhancer hijacking, and epigenomic signatures in the first whole-chromosome-phased myeloma genomes
              </a>
            </td>
          <td>
            Nathan Becker, Enze Liu, Zackary Sanborn, A. Suvannasankha, Kelvin Lee, D. Kazandjian, James Hoffman, B. Diamond, Abhishek Pandey, R. Abonour, O. Landgren, Lisa Munding, Aneta Mikulasova, Brian A. Walker
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e7ea627495d26a3f7b3d82b351b84bb5f1b7889" target='_blank'>
              Genomics of Acute Myeloid Leukemia at Diagnosis and Remission
              </a>
            </td>
          <td>
            K. Yu, L. Dillon, J. Tettero, G. Gui, R. Al-Ali, M. Grunwald, E. Krakow, E. Griffiths, A. Gomez-Arteaga, R. Vedula, M. Solh, A. Salhotra, N. Bejanyan, L. Muffly, A. Jimenez-Jimenez, M. Drazer, Y.-B. Chen, A. Logan, R. Jayani-Kosarzycki, S. Balderman, J. Blachly, B. Shaffer, L. Druhan, C. Yeung, V. Kennedy, A. T. Fathi, H. Carraway, S. Gurbuxani, M. Tjota, F. Sahoo, M. Smith, D. Barfield, J. Guo, J. Han, J. Hu, H. Jo, V. Kudlingar, W. W. Li, , P. Sathyan, S. Truong, S. Catreux, S. Ng, K. Luong, Y. Zhu
          </td>
          <td>2025-12-07</td>
          <td>None</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Epigenetic regulation is a fundamental mechanism controlling gene expression and cellular function, primarily mediated through reversible modifications such as DNA methylation, histone acetylation, and chromatin remodeling. Dysregulation of critical epigenetic enzymes, including histone deacetylases (HDACs), DNA methyltransferases (DNMTs), and bromodomain and extraterminal domain (BET) proteins, has been closely associated with tumor initiation, progression, metastasis, immune evasion, and resistance to conventional therapies. Targeting these epigenetic regulators with small-molecule inhibitors or degraders has emerged as a promising therapeutic strategy, capable of reprogramming aberrant transcriptional networks and reshaping the tumor microenvironment. Beyond direct cytotoxic effects, epigenetic drugs have demonstrated the ability to enhance antitumor immunity by restoring antigen presentation, promoting immunogenic cell death, modulating cytokine profiles, and reversing local immune suppression. Recent preclinical and clinical studies have highlighted the potential of combining epigenetic therapies with immune checkpoint inhibitors to achieve synergistic antitumor responses and overcome resistance mechanisms. This review provides a comprehensive summary of the mechanisms of action, pharmacological characteristics, and clinical applications of epigenetic drugs, with a focus on innovative combination strategies and ongoing translational advancements. We also discuss future directions, emphasizing the need to improve drug specificity, minimize off-target effects, integrate personalized immunotherapeutic approaches, and identify predictive biomarkers to optimize patient selection and clinical outcomes. Overall, epigenetic therapy represents a versatile and evolving avenue for precision oncology with broad implications for tumor control and immunomodulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d1142f91bb95f6983a1d418601e7e6fb2c4af66" target='_blank'>
              Epigenetic drugs in cancer therapy: mechanisms, immune modulation, and therapeutic applications
              </a>
            </td>
          <td>
            Chen Ma, Junkai Cheng, Jian Gu, Qin Wang
          </td>
          <td>2025-12-01</td>
          <td>Molecular Biomedicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Cancer stem cells (CSCs), first identified in acute myeloid leukemia with a CD34+/CD38- phenotype, are hypothesized to sustain clonal survival through self-renewal and resistance to chemotherapy. In B-cell acute lymphoblastic leukemia (B-ALL), stem-like properties can be reacquired through plastic transitions among stem-like, hematogone-like, and naïve B-cell–like states. This plasticity provides an evolutionary route by which chemotherapy-resistant subpopulations re-establish stem-like compartments, contributing to disease relapse under therapy. Such fluidity complicates efforts to target the stem-like compartment directly and underscores the need to understand how chemotherapy reshapes these transitions to drive resistance. We analyzed flow cytometry data from bone marrow (N=63) and peripheral blood (N=46) samples of B-ALL patients at diagnosis, post-induction, and in remission (N=38). Four major cell states are defined from stem-like, progenitor, and differentiated: 1) CD34+/CD38-, 2) CD34+/CD38+, 3) CD34-/CD38+, and 4) CD34-/CD38-. Patient-specific, continuous-time Markov chain modeling quantified transition rates, Qij between state i and j. In remission, differentiation (Q31) consistently exceeded dedifferentiation (Q13), reflecting normal lineage progression. In relapse, however, Q31 – Q13 was often negative with greater variance, indicating that resistant clones rebuilt stem-like compartments through dedifferentiation. This heterogeneity underscores distinct evolutionary mechanisms of resistance across patients. We next explore evolution-based interventions based on our model. Promoting differentiation (increasing Q31) transiently reduced stem-like fractions but failed to prevent rebound due to ongoing dedifferentiation. Inhibiting dedifferentiation (reducing Q13) produced sustained depletion of CD34+/CD38- cells and more effectively shifted relapse dynamics toward remission-like equilibria. Thus, targeting the dedifferentiation fraction emerges as a promising therapeutic strategy. To link transition dynamics with clinical outcomes, we applied mechanistic learning by integrating Markov-derived features with machine learning. Linear discriminant analysis (LDA) achieved high predictive accuracy for BCR::ABL1 status (80%), disease status (84% marrow, 89% blood), and minimal residual disease (71%). Principal component analysis highlighted the balance of differentiation and dedifferentiation as the most informative predictor. These results show that resistance evolves not only from persistence of stem-like cells but also from reciprocal transitions that replenish this compartment. Our findings suggest that therapy pressure influences cell transition landscapes in B-ALL, with relapse often associated with dedifferentiation-driven restoration of stem-like states. Mechanistic learning highlights stem-state reciprocity (in/out rate balance) as a potential predictive biomarker, while modeling indicates that inhibiting dedifferentiation may be effective to reduce resistant stem-cell like clones.



 Sadegh Marzban, Curtis Gravenmier, Ling Zhang, Jeffrey West. Cell state transitions drive the evolution of disease progression in B-lymphoblastic leukemia [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B035.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37463b573a65ae5e34a361dd2f42a6408948633c" target='_blank'>
              Abstract B035: Cell state transitions drive the evolution of disease progression in B-lymphoblastic leukemia
              </a>
            </td>
          <td>
            S. Marzban, Curtis Gravenmier, Ling Zhang, J. West
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Understanding therapy resistance requires deconvolving heterogeneous cell populations and tracking clonal trajectories. While CRISPR-based cellular barcoding is powerful for lineage tracing, many platforms suffer from low efficiency and limited compatibility with single-cell transcriptomics. We developed Oligo-CALL (Oligonucleotide-inducible CRISPR transcriptional activator–Assisted Lineage Labeling), an advanced barcoding system enabling precise lineage tracing, live clone isolation, and seamless integration with single-cell RNA sequencing. Applied to lung cancer cells treated with a KRASG12C inhibitor, Oligo-CALL identified clones consistently enriched posttreatment, supporting a model of predestined resistance. Oligo-CALL achieved >95% efficiency in linking lineage identity to transcriptomes, uncovering diverse clone-specific pathways with underlying resistance. Paired analysis of barcode-matched clones from naïve and resistant populations revealed transient and fixed resistance phenotypes. Notably, DNA repair pathways are recurrently altered in resistant clones, and inhibition of poly(adenosine 5′-diphosphate–ribose) polymerase synergizes with KRAS G12C inhibition to overcome resistance. Together, Oligo-CALL provides a versatile platform for dissecting lineage evolution and molecular dynamics of targeted therapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fc6bc5d7c1ef3b43ef612b4e6dbe8df9b69936e" target='_blank'>
              Oligo-CALL: A next-generation barcoding platform for studying resistance to targeted therapy
              </a>
            </td>
          <td>
            Yingzhuo Liu, Yi Ban, Dingcheng Gao
          </td>
          <td>2025-11-07</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Early T-cell precursor (ETP) acute lymphoblastic leukemia/lymphoma (ALL) is an aggressive hematological malignancy characterized by the uncontrolled expansion of T-cell progenitors. Loss-of-function mutations in genes encoding epigenetic regulators, including core components of the Polycomb Repressive Complex 2 (PRC2), such as the EZH2 histone methyltransferase, are frequently observed in ETP-ALL patients.To investigate the role of EZH2 inactivation in human early T-cell leukemogenesis, we optimized a CRISPR/Cas9-based strategy to disrupt EZH2 gene expression by integrating a DNA cassette containing the constitutive human PGK-1 promoter followed by the GFP reporter. Specifically, human CD34⁺ hematopoietic stem/progenitor cells (HSPCs) derived from cord blood were first edited to knock out (KO) EZH2, then transduced with lentiviruses encoding different T-ALL oncogenes or an empty vector control, and maintained in serum-free medium supplemented to support T-cell progenitor expansion. We found that only HOXA9- and LYL1-transduced EZH2KO HSPCs expanded in vitro for up to 30 days, and were enriched for early T-cell subsets, as identified by single-cell RNA sequencing (scRNA-seq) and a 26-color flow cytometry assay. Notably, these same subsets were capable of initiating T-cell leukemia in immunocompromised (NSG) mice following transplantation, suggesting that EZH2 inactivation cooperates with HOXA9 or LYL1 overexpression to drive human T-cell transformation. To assess the relevance of these findings in primary human T-ALLs, we analyzed whether loss-of-function mutations in PRC2 genes (EZH2, EED, SUZ12) were enriched in specific T-ALL subgroups, particularly ETP-ALL, in a cohort of 1,335 patients from the Children's Oncology Group (COG) study. Interestingly, PRC2-mutated T-ALLs were significantly associated with the HOXA subgroup and the ETP-ALL phenotype, and exhibited higher levels of minimal residual disease (MRD) 30 days after therapy compared to other T-ALL cases. To further dissect the molecular effects of EZH2 loss and HOXA9 overexpression in ETP-like T-ALL, we knocked out EZH2 in the human Loucy and PEER cell lines using CRISPR/Cas9. These EZH2KOcells were then transduced with HOXA9-encoding lentiviruses or empty vectors and analyzed by RNA sequencing. In parallel, ChIP-seq analyses were performed to map HOXA9 binding sites and assess histone modifications affected by EZH2 loss. Integrated analysis of RNA-seq and ChIP-seq data revealed that HOXA9 expression in EZH2-deficient ETP-like cells led to the upregulation of a stem-like/immature T-cell gene signature enriched in transcriptional activators and chromatin remodelers (e.g., RUNX2, IRX3, SATB1, ARID5B, CHD1, INO80D).Furthermore, scRNA-seq profiling of primary PRC2-mutated T-ALL samples revealed distinct leukemic subsets with high HOXA9 expression and strong enrichment for the same gene signature, as determined by Gene Set Variation Analysis (GSVA). Importantly, this signature stratified PRC2-mutated T-ALLs and identified a HOXA positive subgroup associated with poor prognosis.
 Collectively, these findings uncover a critical epigenetic circuit involving EZH2 loss and HOXA9-driven transcriptional reprogramming that promotes stem-like, therapy-resistant T-ALL subsets and highlights novel vulnerabilities that could be therapeutically exploited to eliminate high-risk disease at its origin.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d423907fad1e15df50c67eaeeaa6ff45c0caebf" target='_blank'>
              EZH2 inactivation cooperates with HOXA9 to induce leukemic transformation of human t-cell progenitors and sustain high-risk subsets of early t-cell precursor acute lymphoblastic leukemia (ETP-ALL)
              </a>
            </td>
          <td>
            M. Colucci, Costanzo Padovano, Gaja Bruno, E. De Santis, Serena di Iasio, Chiara Di Nunzio, Andrew Weng, Vincenzo Giambra
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 B-cell acute lymphoblastic leukemia (B-ALL) is the most common pediatric hematological malignancy. While advances in modern medicine have significantly improved survival – especially in children– treatment resistance, relapse, and refractory disease remain major challenges, highlighting the complexities in treating B-ALL and the need for ongoing research and development of novel therapeutic strategies.
 Our lab previously demonstrated that the SWI/SNF chromatin remodeling complex subunit SMARCD1 is essential for the development of the lymphoid lineage, as its inactivation in the adult hematopoietic system leads to lymphopenia while sparing the myeloid and erythroid lineages (Priam et al., Developmental Cell, 2024). In this study, we identify SMARCD1 as an essential gene in B-ALL. Specifically, SMARCD1 inactivation in B-ALL cell lines and patient-derived xenograft (PDX) cells results in profound defects in cellular proliferation and survival. However, the molecular mechanisms underlying its oncogenic role in this context remain to be elucidated.
 The SMARCD1 subunit does not directly bind DNA or chromatin; instead, it is recruited to specific genomic loci through interactions with various co-factors, including the tumor suppressor protein p53. Interestingly, although p53 is somatically mutated in over half of human cancers, it remains wild-type (WTp53) in approximately 75% of newly diagnosed adult ALL cases and 97–98% of pediatric ALL cases at diagnosis (Stengel et al., Blood, 2014, Leukemia, 2017). Given that genetic instability is a hallmark of cancer, understanding why some malignancies retain WTp53 despite its tumor-suppressive functions is a key question with important therapeutic implications. Emerging evidence suggests WTp53 may promote oncogenic functions in specific cancer contexts by regulating metabolic plasticity and pro-survival signaling pathways.
 In B-ALL cell lines, we confirmed a biochemical interaction between SMARCD1 and WTp53 using coimmunoprecipitation (Co-IP) and bimolecular fluorescence complementation (BiFC). We further demonstrated that the SMARCD1-dependent B-ALL cell lines are also reliant on WTp53, as p53 inactivation impairs proliferation and survival to a similar extent as SMARCD1 depletion. Notably, co-depletion of both proteins did not rescue the proliferative defect, suggesting a cooperative role in driving leukemia progression. Importantly, this dependency was not observed in p53-proficient non-hematopoietic cell lines, supporting a context-specific oncogenic role for this SMARCD1-WTp53 axis in B-ALL.
 To explore how the SMARCD1-WTp53 axis drives B-ALL progression, we performed bulk RNA sequencing of B-ALL cells depleted of either SMARCD1 or TP53. Differential expression and gene set enrichment analyses (GSEA) revealed overlapping downregulation of pathways linked to proliferation and metabolism, including hypoxia, fatty acid metabolism, glycolysis, and mTORC1 signaling. Notably, key metabolic enzymes and regulators - such as GLUT4/SLC2A4, HK2, and SDHA- were consistently downregulated upon loss of either SMARCD1 or TP53. These transcriptomic changes were supported by Seahorse XF metabolic flux analyses, which confirmed reduced metabolic activity in SMARCD1- and TP53-depleted B-ALL cells. Cleavage Under Targets and Tagmentation (CUT&Tag) analyses further showed that SMARCD1 and WTp53 predominantly co-occupy the promoter regions of common target genes. Ongoing studies aim to elucidate their cooperative roles in modulating chromatin accessibility and promoter/enhancer function at these loci.
 Collectively, our findings identify the SMARCD1-WTp53 axis as a key regulator of metabolic programming in B-ALL, supporting a high metabolic state and rapid proliferation through control of a distinct oncogenic transcriptional program. This work has important therapeutic implications, highlighting the potential of targeting chromatin-metabolism crosstalk in B-ALL via disruption of the SMARCD1-WTp53 axis. Furthermore, this study reveals a context-dependent oncogenic role for WTp53, emphasizing the need for precision therapies tailored to p53 status - whether by activating WTp53 or restoring its function in p53-mutant cancers.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/969dc91cb92875d724cae066a989cc5ef6e8a223" target='_blank'>
              SMARCD1 subunit of SWI/SNF chromatin remodeling complexes collaborates with p53 to exert an oncogenic role in B-ALL by maintaining high metabolic activity
              </a>
            </td>
          <td>
            Laurence Côté, Pierre Priam, Alexandre Polsinelli, Hugo Philippeau, Gabriel Alzial, Geneviève Deblois, Julie A. Lessard
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) progression and recurrence are often driven by therapy-resistant cellular states emerging within hypoxic tumor niches. While hypoxia is a hallmark of GBM, the spatial transcriptomic heterogeneity that underpins resistance in these regions remains poorly characterized. Here, we employ pimonidazole labeling in conjunction with high-resolution spatial and single cell transcriptomics across 29 primary, treatment-naive human GBM samples, syngeneic mouse models, and patient-derived glioma-initiating cells exposed to hypoxia to define the hypoxia-specific tumor microenvironment and its associated cellular programs. We reveal spatially organized sub-niches within hypoxic regions, each supporting diverse cellular programs yet coexisting tumor lineage states with minimal transitions between cell states. Phylogenetic reconstruction based on inferred copy number variations demonstrates consistent evolutionary trajectories from non-hypoxic to hypoxic tumor loci. Specific copy number alterations were enriched in hypoxic regions and associated with immune infiltration and poor prognosis in MGMT-methylated tumors. Cross-species analyses confirmed the conservation of hypoxia-associated gene programs and spatial architectures. Together, our study delineates the cellular, evolutionary, and spatial complexity of hypoxia in GBM, identifies hypoxia-induced genomic alterations that may promote tumor fitness, and offers insights into new therapeutic vulnerabilities within hypoxic niches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddec812f9197d64f5c26991c527a5afc02eb1748" target='_blank'>
              TMIC-96. Integrated multi-omics reveals distinct hypoxic sub-niches and associated epigenomic and genomic alterations in glioblastoma
              </a>
            </td>
          <td>
            S. Mansouri, Jefferey Liu, Vikas Patil, Ruth Isserlin, Phoebe Lombard, Ronald Wu, Julio Sosa, S. Gandhi, Y. Ellenbogen, O. Singh, Hafsah Ali, C. Velásquez, Bradly Wouters, G. Zadeh
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="


 Chromosomal translocations are key genomic events that frequently occur in hematopoietic cancers and solid tumors. In lymphoma and leukemia, such as diffuse large B cell lymphoma (DLBCL) and follicular lymphoma (FL), most DSBs that lead to oncogenic translocation are initiated by activation-induced cytidine deaminase (AID). AID is a B-cell-specific enzyme that targets immunoglobulin (Ig) genes, considered AID on-target genes, to initiate somatic hypermutation (SHM) and class switch recombination (CSR). However, AID also exhibits off-target activity at non-immunoglobulin regions, contributing to genomic instability by promoting oncogenic chromosomal translocations and mutations that drive the development and progression of B cell lymphomas and leukemias. The mechanisms that govern AID's selective targeting of a limited subset of genomic regions remain poorly understood and represent a longstanding, fundamental question in the field. By utilizing high-throughput genome-wide translocation sequencing (HTGTS) in B cells, we previously demonstrated that PI3Kδ inhibition upregulates AID expression, thereby increasing genomic instability and partially elucidating these mechanisms.
 To further delineate the mechanistic cascade of AID activity as a cytidine deaminase, we now developed dU-seq to map genome-wide AID-induced mutations and applied sBLISS to detect genome-wide AID-induced DNA double-strand breaks (DSBs) in both primary B cells and CH12F3 cells. By these approaches, we successfully captured AID-mediated mutations and AID-dependent DSBs at most known translocation hotspots in the whole genome, including AID on-targets and off-targets, validating the technical robustness of our approach. Surprisingly, we identified thousands of previously unrecognized AID-mediated mutations and DSB hotspots besides those known AID target regions, revealing a much broader genomic footprint of AID activity than previously appreciated.
 To provide a potential clinical relevance of these discovery approaches, we tested whether emerging epigenetic therapies could impact the patterns and distribution of AID-mediated translocations in B cell lymphoma. EZH2 inhibitors, such as tazemetostat and valemetostat, are epigenetic regulators approved by the FDA for the treatment of follicular lymphoma and adult T-cell leukemia/lymphoma, respectively. We found that EZH2 inhibitors alone did not significantly alter AID expression or AID-mediated translocation frequency in CH12F3 mouse B cells or MEC-1 human B cells. However, when combined with PI3Kδ inhibition, EZH2 inhibition markedly enhanced the frequency of chromosomal translocations compared to either treatment alone in both cell models. EZH2 inhibition also further enhanced translocation formation in mouse B cells that were DNA repair deficient, such as Ligase4 knock-out cells. Mechanistically, EZH2 inhibition in B cells depletes the repressive histone modification H3K27me3 while concurrently enhancing the active histone modification H3K27ac, thereby selectively increasing transcriptional activity and facilitating chromosomal translocation formation in the presence of high AID activity or Ligase4 deficiency.
 Overall, to our knowledge this work represents the most comprehensive mapping of AID-induced mutational and genotoxic activity, shedding light on the whole trajectory of AID activity from the very early initiation steps of cytidine deamination to the formation of DSB intermediates up to the final outcome of chromosomal translocations. The described approach can be exploited to functionally dissect the impact of novel drugs on AID-mediated genomic instability in B cell lymphoma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/822e3be74360b79fb622284e6b4772e766b81ff1" target='_blank'>
              Dissecting the whole trajectory of aid-induced genomic instability from mutational activity to chromosomal translocation formation
              </a>
            </td>
          <td>
            Jianli Tao, Luca Alessandri, Beatrice Nuvolari, Fabio Iannelli, Roberto Chiarle
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Despite recent advances in targeted chemotherapies for acute myeloid leukemia (AML), treatment of relapsed and refractory AML remains a significant challenge. Overexpression or rearrangement of the Myelodysplasia Syndrome (MDS) and Ecotropic Virus Integration Site 1 (EVI1) Complex locus (MECOM) is associated with particularly poor prognoses. High MECOM expression establishes a population of leukemia stem cells (LSCs) that have increased resistance to chemotherapy. Prior data from our group revealed a MECOM associated gene regulatory network that is essential in hematopoietic stem cell self-renewal and function, which may represent a therapeutic vulnerability in high-risk AML. Here, we investigated the association of the MECOM transcriptional network with chemoresistance in multiple AML models and found that perturbation of MECOM increased sensitivity to standard chemotherapies independent of cell state changes.



 To probe the interaction of MECOM with chemosensitivity, we engineered multiple high MECOM expressing human AML cell lines by targeting the endogenous MECOM locus with a FKBP12F36V degron that enabled rapid and complete MECOM protein degradation with exposure to the small molecule dTAG. MUTZ-3, an inv(3) AML cell line, was the primary system used as it can expand indefinitely and has a subpopulation of CD34+ progenitors that either self-renew or differentiate into CD14+ monocytes. Findings were later verified in the UCSD-AML-1 and HNT-34 cell lines. Using these models, we determined differential chemosensitivity of AML cells following MECOM protein degradation compared to control cells by characterizing phenotypic properties and differential gene expression profiles with etoposide and doxorubicin treatment, common backbones in current AML treatment regimens.
 In MUTZ-3 cells, etoposide treatment increased the CD34+ stem cell progenitor population from 59.5% to 67% compared to the control (p<0.05, n=3). CD34+ MUTZ-3 LSCs are less likely to undergo apoptosis following etoposide treatment (5.5%, n=3) compared to more differentiated CD14+ cells (60.6%, n=3), demonstrating that etoposide selects for LSCs. Hypothesizing that some of this chemoresistance of LSCs was driven by MECOM, we determined IC50 values of etoposide and doxorubicin after MECOM degradation. MECOM degradation led to a 2.2-fold, 1.6-fold, and 1.7-fold decrease in etoposide IC50 respectively in MUTZ-3, UCSD-AML-1, and HNT-34 cells (p<0.05, n=3). Similarly, MECOM degradation led to a 1.5-fold, 1.8-fold, and 1.5-fold decrease in doxorubicin IC50 in MUTZ-3, UCSD-AML-1, and HNT-34 cell lines (p<0.05, n=3), which indicates that MECOM degradation sensitizes AML cells to etoposide and doxorubicin.
 To examine the transcriptional signatures associated with chemotherapy treatment and MECOM degradation, we performed bulk RNA sequencing with the CD34+ subpopulation of MUTZ-3 cells. We observed significant enrichment of the MECOM gene network following treatment with etoposide (NES =1.81, p <0.001) and doxorubicin (NES=1.91, p <0.001) that is reversed with degradation of MECOM with both etoposide (NES = -2.610, p<0.001) and doxorubicin treatment (NES= -2.634, p<0.001).
 To determine the relevance of this MECOM gene set in primary AMLs, we evaluated RNA sequencing data from adult patients in the BEAT-AML data set. We compared samples at diagnosis (n=450) and relapse (n=37) and found enrichment of the MECOM gene network in relapsed samples compared to diagnostic samples (NES = 1.9, p <0.001). We additionally examined diagnostic samples from patients that were later found to have a complete (n=220) or refractory (n=103) response to first-line chemotherapy and found an enrichment of the MECOM gene network in patients with a refractory response (NES = 1.7, p <0.002).



 In summary, the MECOM gene network is associated with etoposide and doxorubicin resistance in AML human cell models, suggesting that chemotherapy resistance is not only mediated by cell state changes, but is also modified by transcriptional changes in LSCs. Additionally, this gene set is enriched in relapsed and refractory primary AML patient samples. Further study of the mechanisms underlying the observed chemotherapy resistance and ways to overcome it are desperately needed for the development of newer and more effective therapies for high-risk AML.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfccb6c8ddff1c95d1884bc510e150bea0e7adef" target='_blank'>
              Mecom gene network drives chemotherapy resistance in high-risk acute myeloid leukemia
              </a>
            </td>
          <td>
            Sanjana B Shah, Travis J. Fleming, Richard Voit, V. Sankaran
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Long-lived plasma cells (LLPCs) sustain lifelong antibody secretion, forming the foundation of durable immune memory. However, the molecular mechanisms underpinning LLPC survival within the bone marrow (BM) remain incompletely understood. Using single-nucleus RNA-seq and ATAC-seq (10X Multiome) from matched blood and BM antibody-secreting cells (ASCs) of healthy adults, we profiled 8,059 nuclei, generating an integrated transcriptional and epigenetic map of ASCs across their developmental continuum with unparalleled resolution. In the BM LLPC cluster, Gene Set Enrichment Analysis (GSEA) validated the enrichment of the TNF/NF-κB signaling pathway and TLR cascades, emphasizing their anti-apoptotic roles. These clusters also demonstrate increased chromatin accessibility at key loci such as RELA, TLR1, TLR6, TLR10, NFKBIA, and TAB1, underscoring the central roles of TLR and TNF-α signaling pathways in activating the NF-κB pathway. Upregulated genes such as BCL2, NFKBIZ, TNFAIP3, and BIRC3 highlighted key survival mechanisms, while motif enrichment analysis revealed significant enrichment for REL and NF-κB binding motifs, supporting chromatin-level regulation of LLPC longevity. Together, these findings advance our understanding of LLPC survival and resilience and provide a framework for improving vaccine strategies and targeted immunotherapies.



 Computational and Systems Immunology (COMP)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e86cf15dec942c9c651d1f78f13be792e4e03baf" target='_blank'>
              Dissecting the transcriptional and epigenetic landscape of human antibody-secreting cells in blood and bone marrow long-lived plasma cells 4589
              </a>
            </td>
          <td>
            Hira Anis, Monica Cabrera-Mora, Christopher Scharer, Ignacio Sanz, Greg Gibson, F. Eun‐Hyung Lee
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 High-grade serous ovarian carcinoma (HGSC) is a leading cause of gynecologic cancer death, driven by frequent recurrence and resistance to platinum therapies. Uncovering the mechanisms behind this process requires integrating genomic and transcriptional changes in space and time. Here, we present an evolutionary analysis combining multisite phylogenetic trees with transcriptomics, focusing on copy-number (CN) and structural variation (SV) to identify events that shape HGSC progression and treatment resistance. To address the changes during chemotherapy, we first reconstructed tumor phylogenies from 236 whole-genome–sequenced samples derived from 60 patients with relapsed HGSC using PyClone. Patients were stratified according to changes in mutational burden between diagnosis and relapse, as well as by clonal selection patterns, distinguishing cases of monoclonal versus multiclonal relapses. To extend the analysis beyond point mutations, CN and SV profiles were decomposed into clonal profiles within the phylogenetic framework, using a customized implementation of ALPACA. In parallel, matched bulk RNA-seq was integrated to connect relapse-specific genomic alterations with transcriptional changes and pathway deregulation, thereby uncovering pathways and processes underlying chemotherapy resistance. We observed that patients with a single dominant clone at relapse had significantly worse post-relapse survival compared with those exhibiting intra-sample clonal heterogeneity (p = 0.015). Across the cohort, clonal complexity, quantified as the geometric mean of within-sample subclonal heterogeneity per timepoint, showed a marked decline at relapse (p < 0.001). Since this cancer is driven by chromosomal instability, we next characterized CN segmentation and SV breakpoints, which were both significantly increased in relapse samples (p_CN = 0.004, p_SV = 0.006). Together, these results demonstrate structural remodeling rather than mutational burden as a defining feature of treatment resistance that underlies aggressive recurrences. Accordingly, we identified relapse-specific subclonal events as putative drivers of tumor progression during therapy, with impacts on pathways such as MAPK. Integration with matched transcriptomic samples enabled assessment of the functional impact of these events on gene expression and pathway activity. These findings show structural variation as a key driver of HGSC treatment resistance. By linking clonal genomic alterations with transcriptomic consequences, our approach uncovered relapse-specific drivers and opens new avenues for biomarker discovery and therapeutic targeting.



 Giulia Micoli, Jaana Oikkonen, Kari Lavikka, Déborah Boyenval, Johanna Hynninen, Sampsa Hautaniemi. Clonal evolution and structural variation drive chemotherapy resistance in ovarian carcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B016.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f771a1df9cd571a45e30843fc11bc1edd48d299c" target='_blank'>
              Abstract B016: Clonal evolution and structural variation drive chemotherapy resistance in ovarian carcinoma
              </a>
            </td>
          <td>
            Giulia Micoli, J. Oikkonen, K. Lavikka, Déborah Boyenval, J. Hynninen, S. Hautaniemi
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Lung cancer, recognized as one of the most prevalent malignancies with the highest rates of incidence and mortality globally, presents a substantial challenge on a worldwide scale. This challenge is exacerbated by the disease’s difficulty in early detection, a pronounced rate of metastasis, and resistance to treatment, all of which contribute to elevated mortality rates. The tumor microenvironment (TME) plays a critical role in the sustenance and advancement of various solid tumors, including lung cancer. The intricate composition of the TME facilitates tumor proliferation, metastatic spread, and therapeutic resistance by supplying metabolic resources, fostering angiogenesis, and enabling immune evasion. Nonetheless, the regulatory frameworks operating within the TME remain poorly understood. An increasing body of evidence suggests that epigenetic regulation—encompassing mechanisms such as DNA methylation, histone modification, and the action of non-coding RNAs—is pivotal in the initiation and progression of lung cancer. Furthermore, epigenetic modifications significantly influence the functional dynamics of the tumor microenvironment, thereby impacting intercellular interactions and cellular behaviors within the TME, which in turn affects the trajectory of disease progression. This article aims to present the most recent advancements in research concerning the epigenetic regulation of tumor cell interactions with the TME in the context of lung cancer biology. Additionally, it examines the current implications of epigenetic regulation within the tumor microenvironment and its influence on lung cancer behavior. We also investigate the potential relevance and emerging therapeutic avenues presented by epigenetic regulation in the clinical diagnosis and treatment of lung cancer, aspiring to propose novel strategies to address existing treatment challenges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08f9bbf3d9c9172ba516c36021afce36965e86d5" target='_blank'>
              Epigenetic regulation of the tumor microenvironment in lung cancer: mechanism insights and therapeutic prospects
              </a>
            </td>
          <td>
            Maoqin Yang, Xudong Lei, Dexiang Ren, Dakai Qin, Xiaojun Xia
          </td>
          <td>2025-11-28</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Introduction Acute myeloid leukemia(AML) is a heterogeneous malignancy marked by impaired myeloid differentiation and resistance to apoptosis. Venetoclax, a selective BCL-2 inhibitor that restores apoptotic signaling, has shown improved outcomes when combined with hypomethylating agents in older or unfit patients; however, many remain refractory or experience relapse. We used single-cell RNA sequencing(scRNA-seq) to compare bone marrow samples from patients with AML before and during venetoclax-based therapy with those from healthy controls, focusing on cellular type diversity and gene expression dynamics.
 Methods We analyzed 10 bone marrow samples from patients with AML treated with venetoclax plus azacitidine or decitabine, as well as from healthy donors at the University of Iowa. These included samples from 3 healthy controls, 4 responders (1 diagnostic[RD1] and 4 during treatment[RT1-RT4]), and 2 non-responders(1 diagnostic[NRD1] and 1 during treatment[NRT1]). scRNA-seq was performed using 10X Genomics 3' v3 chemistry, sequenced on an Illumina NovaSeq 6000, and processed with Cell Ranger v8.0.0. We excluded cells with <100 or >7,500 genes or >75% mitochondrial reads. Seurat (v5.0.1) was used for normalization, scaling, integration, clustering(resolution = 0.1), and UMAP reduction using the top 30 principal components. Cell type annotation used SCINA with references from Human Cell Atlas ((https://doi.org/10.1016/j.exphem.2018.09.004)). We focused on CD34+ cells to study stem cells and early hematopoietic progenitors.
 Results We identified various hematopoietic populations, including hematopoietic stem cells(HSCs), multilineage-primed(Multi-Lin) cells, monocyte-dendritic progenitors(MDP-1 and MDP-2), megakaryocyte-erythroid progenitors(MEPs), and other early progenitors, as well as mature blood cells.
 Control samples showed consistent distributions with expected proportions of early progenitors, while AML samples displayed distinct cellular profiles. RD1 sample was dominated by a high percentage of blasts(47%) and extensive expansion of MDP-2 cells(~90%). Refractory samples(NRD1 and NRT1) exhibited elevated blasts(60% and 12%) and significant expansion of immature T progenitors(pre-T cells, 33–49%). RT1 had residual blasts(7%) and showed increased numbers of Multi-Lin cells(34%) and MEPs(7%), which may indicate the persistence of immature leukemic clones. RT2, RT3, and RT4 were negative for blasts, with recovery of diverse hematopoietic populations, notably an expansion of MDP-2(13–70%) and lymphoid cells, consistent with marrow recovery.
 In differential gene expression analysis, a consistent finding across multiple cell types was the downregulation of FKBP5, a glucocorticoid-responsive gene involved in stress signaling and resistance to apoptosis. In HSCs, we observed BCL2 downregulation along with upregulation of pro-apoptotic genes(BAX, BAK, CASP7, CASP9). There was suppression of STAT5(growth-promoting) and SMAD3(growth-inhibitory) signals in HSCs, which may indicate disrupted stem cell signaling and early commitment or exhaustion. Downregulation of PTGER4, IL3RA, and IL12RB2 suggested decreased signaling through pro-survival and proliferative cytokine pathways. Additionally, ribosomal protein downregulation was observed, a phenomenon commonly associated with apoptosis, quiescence, or differentiation, where protein synthesis was actively suppressed. CD14 was upregulated in HSCs and MDP cell types, indicating monocytic skewing. In MDP-1 cells, heat shock proteins(HSPA1A and HSPA1B) were downregulated, and DUSP7(a MAPK phosphatase) was suppressed, pointing to impaired stress-response signaling. Changes in the expression of RASA1, RASA2, and MAP3K family genes suggest disrupted Ras-MAPK pathway activity. In MDP-2 cells, decreased CD1B coupled with increased CD14 expression marked a shift away from dendritic differentiation toward an aberrant monocytic identity.
 Conclusion Venetoclax-based therapy is associated with transcriptional remodeling across different cell types in bone marrow. Downregulation of pro-survival and stress-related genes and upregulation of differentiation and apoptotic programs were observed. Distinct transcriptional programs in MDP-1 vs. MDP-2 suggest functional remodeling in different lineages during therapy. These data provide insight into how venetoclax can reshape the bone marrow microenvironment in AML and may help identify early markers of treatment response.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/589edb4fcbc10647fadb024701f20f9d7fed3689" target='_blank'>
              Single-cell insights into acute myeloid leukemia treated with venetoclax-based therapy
              </a>
            </td>
          <td>
            Prajwal Dhakal, Michael Chimenti, Benjamin Darbro, Michael Tomasson, M. Bates
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Loss-of-function studies (LOF) have historically been the main functional approach to identify and study genes which drive the biology of Multiple Myeloma (MM) or other neoplasias. We hypothesized that substantial complementary data can be derived from systematically examining the impact of transcriptional activation of endogenous loci of different genes. We thus performed genome-scale CRISPR gene activation (CRISPRa) screens in 10 genotypically diverse MM cell lines (dCas9-P65-HSF transcriptional activation system; Calabrese genome-scale sgRNA library) to identify genes with significant sgRNA enrichment reflecting enhanced tumor cell fitness. These experiments identified that MM driver genes are highly cell line-dependent, with no universal “hit” but with many recurrent (e.g. in ≥4/10 lines) drivers, including key transcription factors/cofactors (e.g. POU2F2, POU2AF1, IRF4, MYC, HOXA3); growth factor signaling mediators, (e.g. IGF1R, IRS1); Ras family members (e.g. KRAS in MM1S cells); NFκB signaling (e.g. RELA, TIFA), PKC pathway (PRKCE), cell surface receptors (e.g. CD48, HTR1D, IGF1R, UPK3B), solute carrier transporter genes (e.g. SLC38A1). No correlation is currently observed between the pattern of driver genes for the individual MM cell lines and their respective molecular subtype (e.g., t(4;14), t(14;16), t(11;14), etc). Some MM-preferential dependencies identified by CRISPR knockout (KO) studies (de Matos Simoes et al Nat Cancer 2023) were identified by CRISPRa as recurrent MM drivers (e.g. POU2AF1, IRF4) or drivers with limited role in individual line(s) (e.g., IKZF3 and IKZF1 in MM1S cells). Notably, many top MM drivers are not essential for MM cells based on CRISPR KO studies. Most top MM drivers were not identified in similar CRISPRa studies as recurrent drivers for 17 non-MM cell lines from other hematologic neoplasias or solid tumors. With some notable exceptions (e.g., IRF4, POU2AF1), most other MM drivers are not identified in MM patient samples among the molecules with most frequent mutations, gene amplification or overexpression (e.g. vs normal plasma cells) at transcript or protein level. However, transcript upregulation (log2FC≥1.0) of at least one MM driver gene is observed in ~85% of paired MM samples after relapse vs. before treatment (MMRF CoMMpass study), suggesting that many CRISPRa-driver genes can contribute, alone or in concert, to enhanced MM cell fitness in clinical relapses. We validated the functional relevance of several MM driver genes (e.g. POU2F2, EGLN1, SLC38A1) with individual sgRNAs for CRISPR activation and/or cDNA overexpression (vs. isogenic controls) in competition experiments. We further probed the mechanistic basis for the role of POU2F2 as one of the top positive regulators of MM cell fitness and performed RNA sequencing analyses of MM1S cells harboring CRISPRa-based upregulation of POU2F2 expression. We observed upregulation of a distinct cluster of genes (including HOXA3, USP32, HTR1D) that are identified in our genome-scale CRISPRa studies as driver genes specifically in MM1S cells or ≥4 MM lines of our analyses, as well as several genes that are essential for MM survival (e.g., U2AF1, ZNF492, HYPK, PAM16, H2BC11). These results indicate that genome-scale CRISPRa studies provide data which are complementary to those derived from LOF studies. Indeed, our CRISPRa studies validated that MM cell fitness can be enhanced by upregulation of some genes that are prominently essential for MM cells in LOF studies (e.g. IRF4, POU2AF1) but also identified many other promising, previously understudied, regulators of MM cell biology which are not essential for baseline survival/growth of MM cells but can induce growth when further activated. Upregulation of one or more MM driver genes is recurrently observed in MM clinical relapses, which raises intriguing hypotheses about how potential cooperative interactions between these genes contribute to the risk of MM relapses. More broadly, the results indicate that CRISPRa studies can provide novel insights into the biology of MM cells and identify previously understudied genes with therapeutic implications towards suppressing the transition of MM cells to states of advanced biological aggressiveness.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02015e88839b8028448eeed27364c1dfd71a4d2b" target='_blank'>
              Identification of novel “driver” genes of multiple myeloma through genome-scale CRISPR activation studies
              </a>
            </td>
          <td>
            Huihui Tang, Ricardo de Matos Simoes, Torsten Steinbrunn, Ryosuke Shirasaki, O. Dashevsky, Eleni Anastasakou, Oliver Bohorquez, S. Yamano, E. Papadopoulos, Lisa B Leypoldt, Arun Wiita, Olwyn Mahon, Vikas A. Gupta, Jonathan Licht, Benjamin G. Barwick, Lawrence Boise, Constantine S Mitsiades
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="


 Introduction: KMT2A-rearranged acute myeloid leukemias (KMT2A-r AML) are highly lethal leukemias caused by rearrangements of the KMT2Agene, which encode abnormal fusion proteins that activate pro-leukemogenic genes such as HOXAgenes andMEIS1. Prior studies focused on identifying drugs to target proteins that form complexes with KMT2A fusions led to the recent FDA approval of the first menin inhibitor for refractory KMT2A-r AML. However, resistance remains a significant clinical challenge. The gene encoding the High Mobility Group A1 (HMGA1) chromatin regulator is up-regulated in refractory hematologic malignancies and solid tumors where high levels portend adverse outcomes (Resar et al., Cancer Res 2018; Li et al., Blood 2022; Chia et al., JCI2023; Luo et al., JCI2025). In preclinical models, HMGA1 drives tumor progression and a highly plastic, stem-like state by modulating chromatin structure to activate stem cell gene networks in diverse tumors (Li et al., Blood 2022; Chia et al., JCI2023; Luo et al., JCI2025). HMGA1is also enriched in the leukemic stem cell signature of KMT2A-r mouse models (Somervaille et al., Cell Stem Cell 2009), and a recent study identified HMGA1 among the gene signature of menin inhibitor resistance in KMT2A-r AML patient-derived xenografts (Soto-Feliciano et al., Cancer Discov2023). We therefore hypothesized that: 1) HMGA1 drives refractory disease and menin inhibitor resistance in KMT2A-r AML by modulating transcriptional networks, 2) Elucidating HMGA1 function in KMT2A-r AML will reveal novel therapeutic vulnerabilities.
 Methods: To define HMGA1 function in KMT2A-r AML, we silenced HMGA1 using CRISPR or short hairpin RNA (shRNA) in human KMT2A-r AML cell lines (MOLM-14, THP-1, MV4;11). Leukemogenic properties were tested in vitroand in mouse implantation models. To elucidate mechanisms underlying HMGA1, we integrated multiomic studies (RNA/ATACseq). Artificial intelligence (AI) was used to predict drugs to target HMGA1 gene networks.
 Results: HMGA1 is overexpressed in primary KMT2A-r AML caused by diverse fusions in two large patient cohorts compared to age-matched, healthy bone marrow by RNAseq (P<0.0001). Moreover, HMGA1 transcripts correlate positively with downstream targets of KMT2A-r fusions, including HOXA9,HOXA10, andMEIS1 (r>0.4; P<0.0001). HMGA1 gene silencing in KMT2A-r AML cell lines disrupts proliferation (P<0.0001) and clonogenicity (P<0.0001) while inducing differentiation markers (CD11b, CD14; P<0.01). Strikingly, HMGA1deficiency also decreases leukemic engraftment (P<0.0001) and prolongs survival (P<0.0001) following implantation in immunodeficient mice. Intriguingly, AML cells that engraft after HMGA1 silencing express higher HMGA1, suggesting that the escape from gene silencing and a specific HMGA1 level are required for leukemogenesis in this model. RNAseq in KMT2A-r AML cell lines (MOLM-14, THP-1) followed by GSEA revealed that HMGA1 activates transcriptional networks that govern: 1) HOX gene expression, 2) DNA repair, and 3) cell cycle progression (E2F, G2M, Mitotic Spindle). Moreover, HMGA1 activates the signature of menin inhibitor-resistance genes. Conversely, HMGA1 represses MHC-II genes involved in antigen presentation and an anti-tumor immune attack. ATACseq revealed that HMGA1 reduces chromatin accessibility within the MHC-II locus. Leveraging drug prediction algorithms, we identified candidate compounds to target HMGA1 networks, which include the histone deacetylase inhibitor (HDACi) entinostat (ENT), that promotes chromatin opening. Treatment with ENT represses HMGA1 and cell-cycle genes while inducing MHC-II gene expression. Preliminary results suggest that ENT enhances sensitivity to menin inhibitors, supporting further investigation into potential combination strategies.
 Conclusions: HMGA1 is required for salient leukemogenic properties in KMT2A-r AML preclinical models. Mechanistically, HMGA1 activates genes governing cell cycle progression, particularly those implicated in menin inhibitor resistance, while repressing anti-tumor, immune attack genes. Further, these networks can be targeted with clinically available drugs. Together, these findings implicate HMGA1 as a novel epigenetic regulator and promising therapeutic target in KMT2A-r AML.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8259a1497613206067514a7b4107c8e1b548c23b" target='_blank'>
              HMGA1 chromatin regulators drive transcriptional networks governing cell cycle progression, immune escape, and menin-inhibitor resistance in KMT2A-r Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Bailey E. West, Li Luo, Zanshé Thompson, Yuze Du, Leslie Cope, Joseph Kim, Audrey-Ann Supreme, Faiza Shaik, Isha Maloo, Kathryn Mi, Alison Park, S. Meshinchi, R. Ries, Jung-Hyun Kim, Linda S. Resar
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Sarcomas are a heterogeneous group of over 170 malignant tumours of mesenchymal origin. The poor prognosis highlights the need for novel therapeutic strategies. Preclinical modelling is essential, yet challenging, given that sarcomas differ substantially from carcinomas and resources are very limited. GEMMs allow for the precise modelling of recurrent sarcoma genetics. The Cre-loxP system offer spatial and temporal control over the activation of oncogenes or the loss of tumour suppressors, while the CRISPR-Cas9 system enables the rapid, simultaneous editing of key drivers such as Trp53, Nf1, Kras and Pten. These models reproduce key features of human sarcomas, including their histopathology, the initiation of tumours in specific lineages and sites, and tumour–immune interactions within immune-competent hosts. GEMMs have been used to investigate hypotheses about the cells of origin, to test radiotherapy and immunotherapy, and to compare fusion-driven sarcomas with those with a complex karyotype. Despite variability, GEMMs remain essential tools for investigating the mechanisms of initiation, progression, and response to therapy. GEMMs offer mechanistic fidelity, but their use is limited by factors such as breeding burden, variability in recombination, off-target effects of CRISPR, underrepresentation of genomic complexity and inconsistent metastasis. These weaknesses reduce their predictive value, particularly with regard to advanced disease and immunotherapy. Progress will require the integration of Cre-loxP with CRISPR-Cas9, the standardisation of induction and reporting, and a closer alignment with distinct sarcoma subtypes, in order to enhance translational relevance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29a1f976c30385cd478ed53978a57d862bccd72d" target='_blank'>
              Genetically Modified Mouse Models for Sarcoma Research: A Comprehensive Review
              </a>
            </td>
          <td>
            Piotr Remiszewski, Eryk Siedlecki, M. Wełniak-Kamińska, M. Mikula, A. Czarnecka
          </td>
          <td>2025-10-22</td>
          <td>Current Oncology Reports</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Despite significant advances in cancer research, the complexity of signaling pathways remains a major challenge in precision oncology. Tumors harbor a diverse array of genetic alterations; however, these often converge onto a limited set of core signaling pathways, notably RAS/RAF/MEK/ERK (MAPK), PI3K/AKT/mTOR (PAM), and Wnt/β-catenin. Concurrently, these pathways diverge extensively downstream, driving therapeutic resistance through mechanisms such as epithelial-mesenchymal transition, immune evasion, and metabolic reprogramming. Unlike previous literature that largely provides descriptive accounts of pathway alterations, this review uniquely synthesizes convergent and divergent signaling into a clinically actionable diagnostic and therapeutic framework. It critically assesses current precision oncology strategies, identifies gaps, and proposes a dual-axis model that integrates static genomic profiling with dynamic signaling evolution to inform precision therapy. By highlighting opportunities for combination therapy informed by pathway interdependencies and adaptive resistance mechanisms, this perspective provides novel clinical insights and tangible directions for future research in overcoming resistance in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f1f096cbac949c4b15346a048f653260da1f8e6" target='_blank'>
              Convergent and Divergent Signaling Pathways in Cancer: A Dual-Axis Model for Adaptive Precision Oncology.
              </a>
            </td>
          <td>
            Aliasgar Shahiwala PhD
          </td>
          <td>2025-10-08</td>
          <td>Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Breast cancer (BC) remains a major contributor to cancer-related morbidity and mortality among women globally. While systemic therapies have significantly advanced, the emergence of drug resistance continues to hinder durable clinical benefit, leading to treatment failure and disease relapse. Long non-coding RNAs (lncRNAs) have gained recognition as key regulators in BC biology and therapeutic response, with growing evidence implicating them in resistance to chemotherapy, endocrine therapy, targeted agents, and immunotherapy. This review provides a detailed overview of the molecular mechanisms through which specific lncRNAs promote resistance, emphasizing their roles in modulating processes such as programmed cell death, epithelial-mesenchymal transition, and remodeling of the tumor microenvironment. We further highlight common regulatory axes - such as competing endogenous RNA networks and the phosphatidylinositol 3-kinase/protein kinase B pathway - that serve as converging points across distinct resistance mechanisms. In addition, we discuss recent progress in harnessing lncRNAs as liquid biopsy biomarkers and examine the key challenges and opportunities in translating lncRNA-based targets into therapeutic strategies. Together, these insights offer a foundation for future research aimed at overcoming drug resistance in BC through novel lncRNA-centered interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfac8b955e147276484ab116353ca6e5f90be4a2" target='_blank'>
              The role of long non-coding RNAs in breast cancer drug resistance: a comprehensive review
              </a>
            </td>
          <td>
            Yingkun Xu, Jingyu Ma, Changyuan Ding, Weili Liang, Qingdong Zeng, Bin Lv
          </td>
          <td>2025-12-10</td>
          <td>Journal of Translational Genetics and Genomics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9d2645c176f5e0510e7c9fe1ad9d0e8b2a80929" target='_blank'>
              Stroma-driven horizontal transfer of TCA-related proteins mediates metabolic plasticity and imatinib resistance in chronic myeloid leukemia.
              </a>
            </td>
          <td>
            P. Chrościcki, Nikodem Kasak, D. Dymkowska, L. Turos-Korgul, Dominik Cysewski, Vira Chumak, Dawid Stępnik, Monika Kusio-Kobiałka, Agata Kominek, M. Lebiedzińska-Arciszewska, Alicja Krop, Joanna Szczepanowska, Mariusz Wieckowski, Tomasz Stoklosa, Krzysztof Zabłocki, K. Piwocka
          </td>
          <td>2025-12-02</td>
          <td>Cell communication and signaling : CCS</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="


 Long non-coding RNAs (lncRNAs) represent an important layer of gene regulation and have emerged as significant contributors to multiple myeloma (MM) progression (Malard, F., Neri, P., Bahlis, N.J. et al.,2024). The lncRNA NEAT1 is over-expressed in MM patients, supporting malignant plasma cells proliferation and promoting therapy resistance (Taiana E.et al., 2020). Besides its conventional function in the assembly of paraspeckles (PSs), we recently demonstrated that NEAT1 is directly involved in transcriptional control (Puccio N. et al., 2024). In this study, we deepen our understanding of NEAT1's function in shaping the activity of transcriptional bodies, providing the rationale for targeted therapeutic intervention.
 Starting from a transcriptomic approach in NEAT1-silenced (KD) and CRISPRa NEAT1-overexpressing MM cells, we derived a list of 378 targets, primarily involved in mitosis. 52% of these present the cell cycle homology region motif (CHR) in their promoter region, which synchronizes their activation during late cell cycle phases. We confirmed the clinical relevance of NEAT1's transcriptional program in the coMMpass cohort of almost 900 MM patients. Unsupervised clustering analysis, based on the expression of 378 genes NEAT1's target genes, identified two different groups of patients characterized by high or low NEAT1's expression program and different clinical outcomes. Notably, the cluster with the high NEAT1-CHR program showed reduced overall survival. To explore the mechanistic contribution of NEAT1 to mitotic gene expression, we performed computational analysis through the binding model of analysis of transcription (BART) and query of the ENCODE database. These analyses revealed that FOXM1 and CDK9 as upstream regulators of NEAT1 program. Notably, FOXM1 resulted to be highly expressed in R2-ISS stages II-III of myeloma disease as compared to stage I, and in secondary plasma cell leukemia (sPCL) of myeloma origin, based on coMMpass and GSE66293 datasets analysis. We supported these observations by performing in vitro assays. We described a co-localization of NEAT1 and FOXM1 condensates in MM cells, through combined RNA-FISH/immunofluorescence, and direct binding of NEAT1-FOXM1/NEAT1-CDK9 by targeted RNA-immunoprecipitation (RIP). Additionally, with Chromatin immunoprecipitation experiments (ChIP), we demonstrated that NEAT1 KD results in a reduced occupancy of FOXM1 at the promoters of essential CHR genes. Finally, by performing high-throughput drug screening, we identified CDK9 inhibitors to have a strong synergistic effect with NEAT1 KD, consistent with a synthetic lethality effect. These results were confirmed by live cell imaging approach, with two selective CDK9 inhibitors (CDK9i) on a panel of MM cell lines. Consistently, NEAT1 overexpression confers an increased resistance to CDK9i, confirming the participation of NEAT1 to the transcriptional complex that synchronizes mitotic genes and executes the cell division program.
 These data are in line with our recent findings that demonstrate that NEAT1 controls and cooperates with Aurora kinase A (AURKA) in ensuring proficient chromosomal segregation(Puccio N. et al., 2024).
 In conclusion, our data demonstrated that NEAT1 contributes to overt phases of MM and to the clinical aggressive behaviour of the disease, by orchestrating a mitotic transcriptional program. Furthermore, through the application of a multi-dimensional approach, we revealed that the dual inhibition of NEAT1 and transcriptional regulators as CDK9 may offer a promising strategy for novel combinatorial anti-MM therapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0423164211e0008033b3494329d936fcc2b1eb0a" target='_blank'>
              Disruption of NEAT1-transcriptional network identifies targetable non-genetic dependencies in multiple myeloma
              </a>
            </td>
          <td>
            N. Puccio, Christian Boni, Federica Torricelli, Emanuele Vitale, E. Taiana, D. Ronchetti, Valentina Fragliasso, Selene Mallia, Giulia Gambarelli, Roberto Piva, N. Bolli, S. Luminari, A. Ciarrocchi, A. Neri
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Epitranscriptomics, the study of dynamic chemical modifications on RNA mediated by “writers,” “erasers,” and “readers,” has emerged as a pivotal discipline in elucidating the intricate interplay between cancer and immune regulation. These reversible modifications (e.g. m6A, m5C, Ψ) govern RNA metabolism, stability, and translation, thereby exerting spatiotemporal control over immune cell differentiation, activation, and function. Dysregulation of RNA-modifying proteins disrupts immune surveillance, enhances tumor cell survival under stress, and promotes chemoradiotherapy resistance by altering RNA splicing, translation, and stress adaptation pathways. This review summarized the recent progress in the regulatory mechanisms profoundly influencing the tumor microenvironment (TME), modulating immune checkpoints, antigen presentation pathways, and the activity of immune cells. Furthermore, we discussed the therapeutic strategies and challenges in targeting epitranscriptomic regulators and epitranscriptomic editing technologies to enhance anti-tumor immune responses and overcome therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50c0776f1a44885afebca84621fac676e8beb717" target='_blank'>
              Epitranscriptomic control of cancer immunity and therapy resistance
              </a>
            </td>
          <td>
            Xingsen Zhao, Suhua Guan
          </td>
          <td>2025-11-07</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Patients with Multiple Myeloma (MM) receiving adaptive T cell therapies such as chimeric antigen receptor (CAR) T cells and T cell engagers (TCEs) targeting BCMA have shown encouraging responses. However, despite initial response, most patients experience relapse. We have recently observed an enrichment of NF-kB activation mutations, including deletion of TRAF3 and CYLD or amplification of MAP3K14, also known as NF-kB Inducing Kinase (NIK), in patients with BCMA acquired resistance. Furthermore, in a genome-wide CRISPR screen performed using the YUSA library containing 90,709 unique sgRNA targeting 18,010 genes in OPM2 cells, sgRNAs targeting TRAF3 were the top enriched gene in MM cells surviving multiple exposures to an anti-BCMA TCE. Additional enriched hits include sgRNAs targeting genes involved in apoptotic regulation, such as CASP8, and key regulators of the NF-kB pathway, including BIRC3, NFKBIA and NFKBIB. Therefore, we aimed to determine the impact of NF-kB alterations (TRAF3, CYLD and NIK) on the development of resistance to anti-BCMA therapies in MM.
 Through CRISPR Cas9 gene editing, we have generated stable TRAF3 and CYLD knockout (KO) clones in OPM2 and ARP1 MM cells. Moreover, we have cloned NIK in a pLEX-307 vector to overexpress (OE) the NIK protein in both OPM2 and ARP1 cells. As controls, we generated a non-targeting Cas9 and pLEX-307 empty vector cells. The protein changes were confirmed by western blot and the activation of the non-canonical NF-kB pathway was validated by ELISA assay.
 By using flow cytometry-based cytotoxicity assays, we have next demonstrated that the KO and OE cells co-cultured with healthy donors' PBMCs exhibited resistance to anti-BCMA therapies, including CAR-T cells and TCEs. Of interest, similar resistance was observed in the presence of anti-GPRC5D targeting CAR-T cells and TCE, suggesting that their resistance is rather intrinsic and target antigen agnostic.
 To elucidate the molecular signature associated with the development of this acquired intrinsic resistance, we have next performed transcriptome profiling (scRNA-seq) on the OPM2 KO and OE clones and their isogenic controls.
 Several differentially activating pathways, including upregulation of anti-apoptotic signaling and an enrichment of the interferon alpha (IFN) and TNF-a signaling, were transcriptionally enriched in these TCE/CAR T resistance cells, with several targets confirmed at the protein level. Of note, this transcriptionally enriched IFN response was also functionally validated in vitro by demonstrating the lack of infection and replication of the VSV-ΔM51-GFP virus in the TRAF3KO and CYLDKO MM cells relative to their isogenic controls. In addition, multiplex cytokine assay performed on the supernatant of these TRAF3/CYLD KO and NIK OE TCE/CAR T resistant cells revealed elevated levels of multiple factors and cytokines with known inhibitory effect on T cell function, such as sFas, sFasL, TGF-b3, TNF-a, IL-10 and IP-10 when compared to the control isogenic cells. Importantly, we demonstrated that IP-10 (encoded by CXCL10) addition to healthy donor PBMCs co-cultures with MM cell lines was able to attenuate T-cell mediated killing in the presence of anti-BCMA CAR-T cells or TCEs. Lastly, by flow cytometry, we have observed upregulation of Fas expression in the KO cells as the likely source for the increased levels of soluble Fas (sFas). We suggest that sFas may interfere with T cell function by acting as a decoy receptor to immune cells FAS ligand (FASL) blocking its binding to FAS on MM cells.In summary, we have identified a novel NF-kB dependent tumor intrinsic mechanism of resistance to T cell-based therapies mediated by TRAF3/CYLD deletion and NIK amplification that promotes an inflammatory and immunosuppressive microenvironment suppressing T cell activity.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c08a4a12b3d82a261da5787f2a12e375667b1e3" target='_blank'>
              Deletion of TRAF3 and CYLD and NIK amplification activates NF-kb signalling and drives resistance to T cell based therapies in multiple myeloma by suppressing T cell activity
              </a>
            </td>
          <td>
            David Jung, S. Benaoudia, Holly Lee, Sungwoo Ahn, Sejal Chikhale, Mansour Poorebrahim, Noémie Leblay, N. Bahlis, Paola Neri
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="


 The clinical outcome of acute myeloid leukemia (AML) is determined by therapy resistance and relapse. AML-induced immunosuppression has emerged as one of many cofactors and seems to be caused by distinct mechanisms, e.g. immune checkpoint interactions, altered antigen presentation and dysregulation of humoral factors. With chemotherapy, these alterations are expected to be reversible, but the exact mechanisms remain unclear. Adressing this issue, we performed single-cell RNAseq and spatial assessment to identify therapy-induced perturbations of the bone marrow (BM) microenvironment in newly diagnosed AML patients prior to and after induction therapy.



 43,000 single-cell RNA sequencing profiles were generated from 12 paired samples in 6 AML patients using 10x Genomics 5'-scRNA/CITE-seq (140 Total-seq C antibodies) at initial diagnosis (T1) and after induction chemotherapy with either “7+3” or CPX-351 (T2). Analyses were performed using R v4.1.1 and Seurat v4.3.0. Signaling pathway activity was calculated using PROGENy. Stemness was estimated using CytoTrace. Potential cell-cell interactions were scored using CellChat. Spatial correlation was performed by histopathology and immunohistochemistry in matched trephine biopsies using the following stains: H&E, Giemsa, PAS, Gomori silver, Prussian Blue, Glycophorin C, MPO, CD61, CD34, CD117, CD14, CD68, CD20, CD3, CD138, VISTA. All slides were microscopically assessed by a trained hematopathologist.



 Pre-treatmentBM (T1) was characterized by reduction of normal hematopoiesis (p=0.009), in particular classical dendritic cells type 1 (cDC1) (p=0.066) and monocytes (p=0.034). High levels of AML cells were associated with significantly higher levels of mature naïve B cells (p=0.002), higher levels of class-switched B cells (p=0.030) and enrichment of inflammatory monocyte-like cells with MDSC phenotype. Regarding AML cells, 8 different transcriptional subtypes were identified across patients: 4 clusters with highly immature properties (undifferentiated and LMPP-like AML cells) and 4 clusters with signs of maturation (aberrant erythroid-like, EMP-like, promyelocyte-like and monocyte-like AML cells). The immature AML clusters showed high similarities in their transcriptional and signaling patterns, high stemness, quiescence, strong expression of homing factors (CD44, ITGA4), DNA instability and signs of chemoresistance such as upregulation of beta-Catenin, Trail and TGFb pathways. Furthermore, an upregulation of NFkB, TNFa and JAK/STAT signaling was observed in these AML cell clusters, indicating the formation of a highly inflammatory and dysfunctional immune microenvironment. This environment was characterized by a decrease of cDC1, presence of IFIT-high monocyte-like MDSCs, clonal expansion of senescent T cells as well as functional disturbation of antigen-presenting cells, T and NK cells with high expression of TGFb. Interestingly, each AML cell cluster showed a distinct expression profile of immune checkpoint markers, with most prominent expression of VISTA in monocyte-like AML cells. Spatially, VISTA-positive monocyte-like AML cells surrounded other AML cell populations in the BM niche. In addition, monocyte-like AML cells showed high expression of hypoxia- and metabolism-associated signatures (e.g. OXPHOS, fatty acid/cholesterol homeostasis), which might have supported the dysfunctional microenvironment. These microenvironmental changes were largely reversible after chemotherapy (T2).



 Our study reveals AML-induced dysregulation of the BM immune microenvironment, which seems reversible upon AML treatment with standard chemotherapy. Our data suggest that immature AML cells sustain the leukemic stem cell pool, whereas more mature EMP- and promyelocyte-like AML cells contribute to the bulk of AML. Furthermore, monocyte-like leukemic cells may shape the microenvironment to safeguard AML, possibly by VISTA-induced modulation of the immune system. This might explain limited therapeutic efficacy with currently available checkpoint inhibitors in AML. In the long run, identification of specific immune signatures in AML may improve our understanding of AML biology and inform therapeutic algorithms with regard to immunotherapeutic strategies in AML.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3da4a552fa669ffb6977b0dc73a6ab66c9e44bb5" target='_blank'>
              AML maturation stages associate with disease-permissive and immunosuppressive effects in newly diagnosed AML patients: An analysis at the single-cell level
              </a>
            </td>
          <td>
            Jana Ihlow, J. Schrezenmeier, A. Trinks, M. Milek, Seunghoon Kim, Julia Jesse, Eva Tranter, Christian Schneider, Denise Bachorska, Dieter Beule, L. Penter, Sandrine Sander, Simon Haas, M. Morkel, C. Sers, Nils Bluethgen, David Horst, Lars Bullinger, P. Bischoff, J. Westermann
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Precision therapy for glioma remains a major challenge due to tumor heterogeneity. The Origin Recognition Complex Subunit 6 (ORC6) is a crucial regulator of DNA replication initiation. This study aims to investigate the expression of ORC6 in gliomas and its relationship with survival rates and malignancy, while screening potential drugs targeting its functional network. By integrating multiple bioinformatics approaches with structure-based virtual screening, retrospective RNA sequencing data analysis was performed using patients from the Chinese Glioma Genome Atlas (CGGA) and The Cancer Genome Atlas (TCGA) databases. A protein-protein interaction (PPI) network was constructed from ORC6-coexpressed genes to identify core hubs. Molecular docking was employed to screen a library of natural compounds and known drugs against these hub targets. Research has revealed that ORC6 is significantly upregulated in high-grade gliomas, with its elevated expression associated with poor survival outcomes and immune inflammatory responses. Network analysis identified five core hub genes (ORC1, ORC2, MCM2, MCM6, CDC45) central to DNA replication. Molecular docking revealed that several compounds, including the natural flavonoid Baicalein and the FDA-approved drug Palbociclib, exhibited high binding affinity to these hub targets. ORC6 represents a highly promising novel target for precision therapy in glioma. Potential approaches to target this pathway include disrupting the ORC6-replication axis using existing drugs (such as palbociclib) or natural products (such as baicalin).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/324196dc2699eff3ae974af550ed77475be4c5ee" target='_blank'>
              High ORC6 expression is a prognostic indicator of poor survival in glioma patients
              </a>
            </td>
          <td>
            Mengjie Wang, Song Feng, Chen Zhang, Feng Jin
          </td>
          <td>2025-11-19</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Medulloblastoma (MB) is the most common malignant brain tumor in children, comprising ~20% of pediatric brain tumors. MB is a heterogeneous disease, with clinical outcomes heavily influenced by molecular subgroups. Among these, Group 3 MB, characterized by MYC amplification, exhibits the poorest prognosis, with a high propensity for metastasis and recurrence. Our previous research identified WEE1 as a critical regulator of MB cell survival and found that Myc-driven MB is particularly sensitive to the WEE1 inhibitor AZD1775. Although AZD1775 has demonstrated efficacy, especially when combined with genotoxic agents, resistance to small molecule inhibitors often emerges, limiting its therapeutic potential. CDK7, a kinase essential for transcriptional initiation via phosphorylation of RNA Pol II’s C-terminal domain, plays a critical role in Myc-driven malignancies and is frequently enriched at super-enhancers. AZD1775-resistant cells undergo chromatin reorganization to establish alternate cis-acting elements that promote transcription of DNA repair and proliferation genes, circumventing treatment. We hypothesized that targeting CDK7-mediated transcriptional reprogramming could overcome AZD1775 resistance. Using the CDK7 inhibitor THZ1, we performed western blot analysis of DNA repair proteins and observed inhibition of ATM pathway activation, a key component of the DNA damage response. Proliferation assays in both parental and AZD1775-resistant cell lines demonstrated that combining AZD1775 with THZ1 significantly suppressed cell growth. These findings suggest that THZ1 may reverse adaptive chromatin reorganization, sensitizing resistant MB cells to AZD1775 and enhancing the overall therapeutic response. Future studies will investigate the potential of combining CDK7 inhibitors with other epigenetic modulators to further disrupt resistance mechanisms and improve treatment outcomes in aggressive MB subtypes, particularly Group 3 MB. This approach offers a promising avenue for overcoming therapeutic resistance and advancing precision medicine in medulloblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb1c890009c33793158ec3602b7038265f5fb5f2" target='_blank'>
              CNSC-10. CDK7 INHIBITION MITIGATES AZD1775 RESISTANCE VIA CHROMATIN REPROGRAMMING IN MYC-MB
              </a>
            </td>
          <td>
            Kiara Smart, Rajeev Vibhakar, Bethany L. Veo
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7db7cd93144dc9423f9937951e7c3121e79d997b" target='_blank'>
              Epstein-Barr virus infection shapes the genetic, transcriptomic, and immune microenvironment landscape of Burkitt lymphoma.
              </a>
            </td>
          <td>
            Qing Xiao, Yilei Zhang, Zhongyi Chen, Xinrong Liu, Wentao Yu, Tiansheng Wang, Ling Chu
          </td>
          <td>2025-12-10</td>
          <td>Infectious agents and cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 PAX5 is genetically altered in over 30% of B-ALL cases, with point mutations representing the second most common type of alteration. These point mutations are highly enriched in the DNA binding (paired) domain and have been recognized as founder events in PAX5 P80R and PAX5alt B-ALL subtypes, which account for over 12% of B-ALL. However, the molecular features and underlying mechanisms of these mutations in B-ALL remain largely undefined, limiting the development of targeted therapies and resulting in poor outcomes, especially in adult patients.
 We systematically investigated the functional roles and oncogenic mechanisms of PAX5 paired domain mutations. Integrated CRISPR screening and genomic profiling revealed that the paired domain is critical for PAX5 function. Disruptions in this region significantly reduced cell viability in B-ALL cell lines, and analysis of 2,955 B-ALL samples showed that recurrent PAX5 missense mutations are highly enriched in this domain, further underscoring its functional importance.
 Functional analysis of knock-in mice with PAX5 mutations revealed that R38H and R140L severely impair DNA binding and block B-cell differentiation, while P34L and P80R retain binding ability but alter binding specificity and gene regulation. Mice carrying homozygous (P34L, R38H, P80R, R140L) or heterozygous (R38H, P80R) mutations developed spontaneous B-ALL within one year. To model co-occurring mutations observed in patients, Pax5 R38H/P34L and R38H/R140L mice were generated. These compound mutations exhibited greater leukemogenic potential than single mutants. Transcriptomic analysis of leukemia samples showed consistent up-regulation of metabolic pathways and Myc target genes. Notably, biallelic PAX5 alterations and additional genetic lesions in signaling pathways were detected in all PAX5R38H/+ and PAX5P80R/+ leukemias, faithfully recapitulating the genetic alteration features of patient B-ALL driven by PAX5 mutations.
 Among Pax5-mutant mouse strains, Pax5P80R/+ mice develop B-ALL with short latency and complete penetrance, making them an ideal model to study the mutational evolution and leukemogenesis of B-ALL. To characterize the stepwise leukemogenesis process, we performed serial bone marrow aspirations in the same Pax5P80R/+ mice. In all 17 mice analyzed, we identified early preleukemic clones with only Pax5 WT deletions, followed by the emergence of leukemic clones harboring Jak1/3 mutations. Single-cell RNA/BCR-seq revealed that Pax5 WT allele deletions relieve metabolic constraints, thereby promoting clonal expansion of preleukemic B cells blocked at the pre-B stage. These changes coincide with downregulation of metabolic checkpoint genes and upregulation of glycolytic enzymes, promoting genetic instability and facilitating malignant transformation. Clinically, PAX5 P80R patients have better outcomes than PAX5alt cases. Using our mouse models, we observed that leukemic cells driven by the PAX5 P80R mutation exhibit greater sensitivity to dexamethasone compared to the ones carrying other PAX5 mutations in PAX5alt subtype, potentially due to their heightened dependence on glucose uptake and glycolysis, suggesting that metabolic rewiring driven by PAX5 mutants may influence therapeutic response.
 Interestingly, we found that an aberrant isoform of MEGF10 (aMEGF10) is highly and specifically expressed in PAX5 P80R B-ALL. This N-terminal truncated isoform encodes a protein that retains the intracellular immunoreceptor tyrosine-based activation motifs (ITAMs). Using CUT&Tag, we showed that PAX5 P80R directly binds to the aMEGF10 promoter. By engineering human CD34⁺ cells, we established a humanized PAX5 P80R leukemia model that recapitulates the gene expression profile of patient samples, including robust activation of aMEGF10. Functionally, aMEGF10 binds to and activates SYK, thereby promoting leukemic cell proliferation. Importantly, this oncogenic effect depends on intact ITAM motifs and SYK signaling, revealing a novel targetable vulnerability in PAX5 P80R leukemias.
 Overall, the study establishes that PAX5 DNA binding domain mutations contribute to leukemogenesis through stepwise loss of PAX5 WT alleles and acquisition of signaling pathway mutations, involving impaired differentiation, metabolic deregulation, and novel oncogene activation. These findings not only deepen the understanding of PAX5-driven leukemias but also highlight aMEGF10–SYK signaling as a promising therapeutic target.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/560464ef58d6798ff236f2b779af36d57c558d65" target='_blank'>
              Stepwise leukemogenesis and therapeutic targets revealed in B-cell acute lymphoblastic leukemia driven by PAX5 mutations
              </a>
            </td>
          <td>
            Zhilian Jia, Zunsong Hu, Lu Yang, Anthony K. N. Chan, Xiaohua Huang, Allen Mao, Ibrahim Aldoss, Lucy Y Ghoda, M. Afkhami, Guido Marcucci, Zhenhua Chen, Jianjun Chen, Chun-Wei Chen, S. Izraeli, Markus Müschen, Nora Heisterkamp, Zhaohui Gu
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Metastasis is the leading cause of cancer-related mortality, representing a highly coordinated, multistep process in which malignant cells gain the ability to invade, survive in the circulation, and establish secondary tumors at distant sites. While genetic mutations initiate oncogenesis, accumulating evidence shows that epigenetic and epitranscriptomic regulators, encompassing DNA methylation, RNA modifications, and noncoding RNAs (ncRNAs), reshape metastatic phenotypes. This review integrates current insights into these mechanisms and their crosstalk, with a primary focus on their methylation modification. Given their plasticity and potential reversibility, these regulators are attractive targets for therapeutic intervention. Defining the dynamic interplay between DNA and RNA modifications and ncRNAs provides a coherent framework for controlling metastasis and guides the development of precision epigenetic strategies and biomarkers. Future research that integrates multi-omics approaches and spatial transcriptomics will be essential for revealing the epigenetic and epitranscriptomic layers of the metastatic landscape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a32e97223f0c41298eec4d68679c66b5c3087095" target='_blank'>
              Rewriting the Fate of Cancer: Epigenetic and Epitranscriptomic Regulators in the Metastatic Cascade
              </a>
            </td>
          <td>
            Toshifumi Hara, M. Subramanian
          </td>
          <td>2025-11-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="


 We previously described three subtypes of Waldenstrom's Macroglobulinemia (WM) using a multi-omic data set derived from 249 untreated patients with WM who had detectable MYD88 mutations. The B-cell like (BCL) and Plasma cell like (PCL) subtypes have distinct cell of origin but evolve from a shared transcriptional subtype (Early WM) found at early stages of disease development (Hunter et al, Manuscript in Review. Research Square, preprint 2025). We also described the WM evolutionary score (EScore) that corresponds to the progression from smoldering WM to symptomatic disease correlating with time to first therapy and WM bone marrow involvement. The transcription factor (TF) network differences that underlie these findings remain poorly understood. In this study, we examined changes in implied gene regulatory networks between WM and healthy donor memory B-cells (HDMB; CD19+CD27+), with EScore, and between WM subtypes. Findings were further analyzed for expression, alternative splicing, and TF motif enrichment in differentially regulated ATAC peaks.



 Sample specific gene regulatory networks were modeled using PANDA (Glass et al. Plos One, 2013) followed by LIONESS (Kuijer et al. iScience, 2019). Together, these calculate an interaction score for each TF/gene pair for per sample. These scores were summed up by TF and by gene to model to net regulatory connectivity known as the target score (TS). LIONESS and TS data were analyzed using limma in R and top networks were derived from the results. Top TF and gene hits were analyzed further for changes in expression and isoform composition based on Gencode augmented by novel isoforms discovered in our IsoSeq long read analysis of HDMB and WM samples (Hunter et al, ASH 2024). TF motif enrichment from differentially regulated ATACSeq peaks were calculated using MEME Suite's SEA.



 Network analysis of EScore TF enrichment revealed AHR and TFAP4 as top drivers. AHR is a regulator of xenobiotic metabolism which was already known to be negatively associated with EScore from our previous Hallmark gene set enrichment analysis of the expression data. Expression of AHR was negatively correlated with EScore (r = -0.73; p<0.0001) with the median expression falling by a factor of 12 from the earliest to latest EScore levels. AHR has been reported to inhibit B-cell class switching and plasma cell differentiation while upregulating the pro-inflammatory PTGS2 (cyclooxygenase-2) and anti-apoptotic BCL2L1 (BCL-XL) genes. The correlation with PTGS2 (r=0.641; p<0.0001) and BCL2L1 (r=0.266; p<0.0001) was observed in this study. Notably BCL2 was negatively correlated (r=-0.552; p<0.0001) consistent with a change in BCL2 family dependency during progression to symptomatic disease. Unlike AHR, TFAP4 was both transcriptionally upregulated and showed increased TS with EScore, consistent with its reported role as an oncogene. Most of the AP-1 transcription factor family also appeared in the top EScore network results including JUNB, FOS, FOSL1, FOSL2, ATF3, and MAF. Moreover, JUNB, FOS, FOSL2, and FOSB were all top enriched motifs in differentially closed chromatin between early and late EScore (p<0.0001 for all). Transcriptional levels of FOS and FOSL2 decreased >15x with EScore (p<0.0001 for both) while FOSB and JUNB levels were largely steady despite notable TS enrichment. Isoform analysis further revealed that while FOSB expression in WM is over 2x higher than HDMB expression, HDMB expression is primarily intron retained (ENST00000587358) while WM primarily expressed the full-length transcript (ENST00000353609). Comparing HDMB to WM Subtypes revealed strong differences in TF and gene TS with corresponding isoform dysregulation including PRDM1, PRDM4, IL15, RCAN3, CRIP2, and CD19. Despite increased plasmacytic differentiation, CD19 was expressed at only slightly lower levels than HDMB. However, HDMB primarily expressed a novel isoform skipping exon 2 while WM expressed the canonical transcript (p <0.0001). Given the important role of CD19 in BCR and TLR signaling, our findings indicate a critical dependency for CD19 in WM.



 Our extensive regulatory network analysisutilizing a large multi-omic data set identified several novel therapeutic targets including TFAP4 and CD19, as well as the AHR, TFAP4, and AP-1 transcription factors as key drivers of inflammatory signaling, BCL2 family dysregulation and malignant cell growth for symptomatic disease progression in WM.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b66707f752fe87e364445e1c73a482a07260f60" target='_blank'>
              Regulatory network analysis identifies changes in ahr, TFAP4, and AP-1 transcription factors as top drivers of symptomatic disease progression in Waldenstrom's macroglobulinemia.
              </a>
            </td>
          <td>
            Z. Hunter, Derrick DeConti, Brian Lawney, N. Tsakmaklis, M. L. Guerrera, A. Kofides, Xia Liu, Shirong Liu, Hao Sun, Dominic Pizzarella, Abigail Peachey, Alberto Guijosa, J. Hatcher, K. Meid, Christopher Patterson, Torsten Steinbrunn, Constantine S Mitsiades, John Quackenbush, Andrew Branagan, S. Sarosiek, J. Castillo, S. Treon
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) and myelodysplastic neoplasms (MDS) are clonal hematopoietic malignancies in which next-generation sequencing (NGS) has become integral for diagnosis, classification, risk stratification, and measurable residual disease (MRD) monitoring. Traditional cytogenetic and PCR-based assays remain useful, but targeted NGS panels now represent the standard of care, providing rapid and sensitive detection of recurrent gene mutations, structural variants, and gene fusions. Whole-genome, whole-exome, and RNA sequencing and long-read platforms expand the spectrum of detectable alterations, though targeted panels remain most practical for routine diagnostics. Bioinformatic pipelines and quality metrics—including read length, sequencing depth, and coverage—are critical for accurate variant calling, with validation often required for variants of uncertain significance or those near detection thresholds. NGS is now embedded in diagnostic frameworks, including the WHO 2022 and ICC classifications, which incorporate recurrently mutated genes such as TP53, ASXL1, RUNX1, and FLT3. These data inform prognostic models, with ELN-2022 defining adverse-risk AML subgroups for patients treated with intensive chemotherapy, ELN-2024 AML for patients treated with less-intensive therapies, and the IPSS-M refining MDS risk categories by integrating mutational data. NGS also enables MRD monitoring, with gene panels and PCR-NGS hybrid approaches (e.g., for FLT3-ITD) showing increasing clinical utility, though standardization is still lacking. Furthermore, diagnostic NGS frequently uncovers germline predisposition syndromes (e.g., DDX41, GATA2), with significant implications for treatment decisions and donor selection in transplantation. In this manuscript, we review the advantages, limitations, and future perspectives of NGS in the clinical management of AML and MDS with a particular emphasis on the biological and technical principles underlying its use in these diseases. Furthermore, we discuss how NGS findings may influence diagnosis, prognostic classification, and therapeutic decision-making within current clinical frameworks. Our aim is to provide a comprehensive overview of NGS fundamentals to support clinicians in navigating the increasing complexity of molecular data in daily practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36f92e8c1514cc884abf1a1cf442e829de555deb" target='_blank'>
              How to Read a Next-Generation Sequencing Report for AML and MDS? What Hematologists Need to Know
              </a>
            </td>
          <td>
            Salvatore Perrone, C. Tresoldi, Silvia Rigamonti, Matteo Molica, Nadezda Zhdanovskaya, Laura Cicconi
          </td>
          <td>2025-12-08</td>
          <td>Journal of Clinical Medicine</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="


 Background Acute myeloid leukemia (AML) is a clinically and genetically heterogeneous disease. While traditional classification systems such as FAB relied on morphology and immunophenotype, recent classifications (WHO, ICC) emphasize genetic alterations for diagnosis and therapy. However, more directly involved in the phenotypic cellular inheritance, the epigenomic profile may offer an additional dimension for understanding AML heterogeneity. Previously, we performed ATAC-seq of 1,536 AML samples and identified 16 epigenetically defined AML subgroups (subgroups A-P), which were associated with distinct clinical, genetic, and transcriptional features (Ochi et al., ASH 2024). In the present study, we extend these findings by using single-cell RNA and ATAC-seq (scRNA/ATAC-seq) profiling to explore intra- and inter-tumor epigenetic heterogeneity, transcriptional regulation, and hierarchical differentiation trajectories across AML subgroups.
 Methods We performed multi-platform scRNA/ATAC-seq on 36 AML samples including all the 16 epigenomic subgroups and 4 remission samples (as controls), profiling a total of 281,167 mononuclear cells. Computational analyses included cell clustering, projection onto normal hematopoiesis reference maps, pseudotime inference, transcription factor (TF) activity analysis using SCENIC+, and leukemic stem cell (LSC) signature scoring.
 Results scATAC-based clustering showed that leukemic cells from individual AML samples formed distinct clusters largely separated from normal cells, typically co-clustered by subgroup, regardless of their differentiation status, indicating that leukemic cells within each subgroup share a distinct intrinsic epigenetic program. Projection onto a normal hematopoiesis reference map revealed divergent differentiation arrest among subgroups, even within genetically similar cases. For instance, all NPM1-mutant subgroups (D–F) were HOX-related but differed in differentiation states: subgroup E was arrested at the HSC stage, D at the GMP stage, and F contained both progenitors and mature cells. Single-cell analysis also refined differentiation states of subgroups F-H: while bulk ATAC-based deconvolution suggested monocyte enrichment in these subgroups, single-cell analysis revealed mature monocytes predominated in H, whereas F and G were enriched for immature promonocytes.
 Combined with LSC signature analysis based on gene expression, pseudotime analysis demonstrated that cells showing high LSC scores consistently mapped to early stages of the differentiation trajectory across all subgroups. These results indicate the presence of a conserved leukemic hierarchy, with stem-like cells positioned at the apex, irrespective of epigenetic subgroup, thereby underpinning AML pathogenesis. We further analyzed TF activity using the SCENIC+ program based on scRNA/ATAC-seq data, which identified key TFs enriched in each subgroup, many of which overlapped with those found in bulk RNA/ATAC-seq analysis. Pseudotime analysis further showed that TFs exhibited subgroup-specific activation dynamics. For example, HOXA9 was activated throughout the myeloid differentiation trajectory in HOX-related subgroups but peaked at different stages according to subgroups. In the RUNX1-enriched subgroup, IRF8 and BCL11A were both active but peaked at mature and immature stages, respectively. These findings indicate that key TFs exhibit subgroup-specific activation patterns at distinct stages along the myeloid differentiation trajectory.
 Conclusion Single-cell multi-omics analysis reveals that AML epigenetic subgroups exhibit distinct but hierarchically organized differentiation trajectories, driven by dynamically regulated TF programs. These findings refine our understanding of AML diversity and may inform more precise, differentiation-stage–specific therapeutic strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d92f306fb3417df7e3eff8b67fe8a062b478725" target='_blank'>
              Single-cell multi-omics uncovers differentiation-dependent transcriptional programs shaping epigenetic subgroups in AML
              </a>
            </td>
          <td>
            Y. Ochi, Markus Liew-Littorin, Y. Nannya, S. Bengtzén, Bénédicte Piauger, Stefan Deneberg, M. Jadersten, Vladimir Lazarevic, Jörg Cammenga, Anna Robelius, L. Wennström, Emma Olander, S. Kasahara, N. Hiramoto, N. Kanemura, N. Sezaki, M. Sakurada, M. Iwasaki, J. Kanda, Yasunori Ueda, Satoshi Yoshihara, Tom Erkers, Nona Struyf, Yu Watanabe, Masanori Motomura, M. Nakagawa, R. Saiki, Hidehito Fukushima, Koji Okazaki, S. Morimoto, A. Yoda, R. Okuda, Shintaro Komatsu, Guoxiang Xie, Albin Osterroos, A. Kon, Lanying Zhao, Yuichi Shiraishi, Takayuki Ishikawa, Satoru Miyano, Shuichi Matsuda, A. Takaori-Kondo, Hiroyuki Aburatani, Hiroshi Suzuki, Olli-P. Kallioniemi, Gunnar Juliusson, Martin Höglund, Sören Lehmann, Seishi Ogawa
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="


 Background: Extranodal NK/T-cell lymphoma (NKTCL) is an aggressive non-Hodgkin lymphoma subtype. Despite improved outcomes with immunotherapy, persistent resistance in a subset of patients underscores an unmet need to delineate underlying molecular drivers. Lipid metabolic reprogramming provides energy and biosynthetic precursors for tumor growth while actively remodeling the tumor microenvironment to suppress immunity. Thus, cotargeting immune checkpoints and lipid metabolism represents a promising strategy to overcome resistance; however, its mechanistic basis and clinical applicability in NKTCL remain elusive.
 Methods: Using complementary multiomics profiling—single-cell RNA sequencing (scRNA-seq; n = 13 patients), bulk RNA sequencing (RNA-seq; n = 24 patients), and metabolomics(n = 22 patients)—in the prospective SPIRIT cohort (NCT04127227), we identified reprogrammed lipid metabolism as a hallmark of immunotherapy resistance in NKTCL. To mechanistically dissect this, we deployed dual parallel systems: (1) therapy-resistant models and (2) CRISPR-based gene-editing platforms across in vitro settings and humanized mouse models. Functional validation was performed by leveraging these systems with multiomics analyses, including bulk RNA-seq, scRNA-seq, RNA immunoprecipitation sequencing (RIP-seq), methylated RNA immunoprecipitation sequencing (MeRIP-seq), and metabolomics.
 Results: YTHDF3 deficiency stabilizes PI3K-activating transcripts (IRS1, IGF1R, PIK3R6), driving constitutive PI3K signaling and lipid metabolic reprogramming. This hyperactivated lipid metabolism induces endoplasmic reticulum stress (ERS), promoting ERS-mediated MHC-I downregulation to confer immunotherapy resistance. Translating these findings, we designed a dual-targeting strategy synergizing PI3K inhibition (to reverse metabolic immunosuppression) and anti-PD-1 therapy (to block immune checkpoint evasion), which demonstrated efficacy in murine models. This therapeutic strategy is now being evaluated clinically in a phase Ib trial (NCT06793956).
 Conclusion: Our study delineates the YTHDF3–PI3K–lipid metabolism axis as a central driver of immunotherapy resistance in NKTCL. This mechanism-guided framework, with dual targeting of PI3K and PD-1, bridges preclinical validation to clinical translation (NCT06793956).
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41871b050607d2beba386ede97a14d32b3a24966" target='_blank'>
              Lipid metabolic reprogramming mediated by YTHDF3 deficiency underlies immunotherapy resistance in extranodal NK/T-cell lymphoma
              </a>
            </td>
          <td>
            Jinni Wang, Qingqing Cai, Wenlong Zhong, Yuchen Zhang, Danling Dai, Yi Cao, Shuai Liang, Jun Cai
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20ec8c67f8ff833438b56a407614424d68047c71" target='_blank'>
              Single-cell transcriptome analysis defines novel molecular subtypes and reveals therapeutic implications of T/myeloid mixed-phenotype acute leukemia.
              </a>
            </td>
          <td>
            Bin Huang, Wenjie Liu, Yuxin Du, Ping Liu, Zixing Lu, Shiyang Zhong, Xingfei Hu, Wanting Zhou, Yuzhu Shi, Runheng Huang, Xian Zhang, Jinning Shi, Chuanyang Lu, Chunling Wang, Liang Yu, Lingxiang Wu, Wei Wu, Peng Xia, Qian Sun, Liwei Zhu, Zhi Wang, Ruohan Zhang, Xinrui Lin, Sali Lv, Qianghu Wang, Sixuan Qian, Kening Li, Mingyao Hong
          </td>
          <td>2025-11-29</td>
          <td>Genome medicine</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="


 Adoptive cellular therapies (ACT) including chimeric antigen receptor (CAR)-T cells have transformed the treatment landscape for patients with relapsed/refractory hematologic malignancies. However, this therapy has not achieved similar success for patients with solid tumors due to the poor persistence and function of ACT in the immunosuppressive environment of solid cancers. Emerging data suggests that the efficacy of ACT is directly linked to the degree of representation of T memory stem cells (Tscm) in the product. Tscm are a minimally differentiated subset of T cells that are maintained over long periods in vivo and have the capacity to repopulate the full spectrum of memory and effector T cells, which makes this cell subset critical to an effective antitumor T cell response. While numerous studies highlight the therapeutic relevance and importance of this T cell population, the pathways critical to the generation of these cells and the process by which the production of Tscm is finely tuned is poorly understood.
 Previously, our group demonstrated in an immunocompetent mouse model that STING (stimulator of interferon genes) agonists like cGAMP could enhance CAR T cell persistence and function through increased Tscmproduction, which was most significant using Th/Tc17 CAR T cells. To elucidate the underlying mechanism, we performed mass spectrometry in murine Th/Tc17 cells using biotin labeled cGAMP as bait followed by liquid chromatography-mass spectrometry. Top candidates (based on sequence coverage and Log2 transformed LFQ intensities) were deleted via Cas9-ribonucleoproteins (RNPs) and subsequently characterized. Of those targets, we identified the leucine rich pentatricopeptide repeat containing protein (LRPPRC, also known as LRP130) as a novel downstream effector. LRPPRC is a multifunctional protein that regulates cellular metabolism and mitochondrial homeostasis where it operates as a reader of N6-methyladenosine (m6A) modified RNA helping to facilitate the appropriate processing and trafficking of RNAs that impacts their transcription. However, its function in T cells, and the immune system in general, is unknown.
 Using DART-seq (deamination adjacent to RNA modification targets) we demonstrate that transcripts essential for T cell memory like Tcf7, along with relevant metabolic RNAs that promote a bioenergetic profile that supports Tscm formation like Cpt1a and Acaca, display m6A marks. Using RNA immunoprecipitation assays, we find that LRPPRC selectively binds to and stabilizes these transcripts in Th/Tc17 cells. Interestingly, LRPPRC has much less association with these RNAs in conventional Th/Tc1 cells potentially highlighting a vulnerability that could be therapeutically targeted. Tscm are known to rely more heavily on fatty acid metabolism for energy production, which is advantageous for T cells in the tumor microenvironment as it preserves ATP generation and maintains mitochondrial health to sustain an effective antitumor response. Deletion of LRPPRC with Cas9-RNPs impairs aerobic respiration as assessed by Seahorse bioanalysis, which results in bioenergetically unfit T cells that display reduced ATP production rates and mitochondrial dysfunction, thereby preventing effective Tscm generation.
 Strikingly, these findings extend to human CD19-targeted CAR T cells differentiated into Th/Tc17, where LRPPRC loss recapitulates the metabolic and transcriptional defects observed in murine cells. In an immunocompetent murine melanoma ACT model, infused T cells that lack LRPPRC are unable to effectively control tumor growth, which leads to decreased survival, emphasizing the functional significance of this pathway.
 Collectively, our findings identify m6A post-transcriptional regulation, mediated through LRPPRC, as a novel and targetable axis for enhancing T cell stemness and metabolic fitness. This study is the first to directly implicate m6A-driven RNA regulation in shaping the antitumor efficacy of adoptive T cell therapies. By harnessing this pathway future ACT products may be optimized to improve outcomes for patients with hematologic malignancies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54be2efbb4952383315aed9c4c103240687f62fc" target='_blank'>
              M6A-driven control of T cell fitness via lrpprc defines a novel regulatory axis to enhance adoptive cell therapy
              </a>
            </td>
          <td>
            Jeremy Meier, Aleksandra Petrova, Matthew Tegowski, Katie Hurst, Kate Meyer, Jessica Thaxton, Jonathan S. Serody
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Kyinno provides TMD8 cell lines with endogenous BTK resistance mutations (e.g., C481S, L528W, V416L, T474I, A428D) under native regulation, enabling detailed studies of covalent and non-covalent inhibitor responses. These models faithfully recapitulate clinical escape alleles, supporting both mechanistic investigations and drug-resistance profiling.



 High-fidelity CRISPR/Cas9 RNP electroporation with 200-bp ssODN donors and proprietary HDR enhancers routinely yields >80 % BTK knock-in efficiency. Following optimized limiting-dilution cloning, each clone undergoes genomic PCR, Sanger sequencing, and RT-PCR to confirm on-target integration, allelic zygosity.



 Mutant TMD8 lines display broad IC50 shifts: A428D and L528W confer >100-fold resistance to both ibrutinib and pirtobrutinib, C481S remains nanomolar to pirtobrutinib, whereas T474I/V416L stay responsive to zanubrutinib; all variants expect A428D retain low-nanomolar BGB-16673 and NX-5948 susceptibility. In vivo, C481F tumors show modest ibrutinib delay, whereas L528W xenografts resist ibrutinib yet regress under daily NX-5948 without overt weight loss. Proliferation assays mirror these drug-response patterns across the panel.



 Kyinno’s BTK-mutant lines accelerate high-throughput screening of small-molecule inhibitors and targeted degraders, delineate resistance circuitry via isogenic comparisons, and improve translational relevance by faithfully modeling patient-derived escape variants. This platform supports rapid go/no-go decision-making in early drug discovery and biomarker identification for next-generation BTK therapies.



 Jinying B. Ning, feng B. hao, yue B. huang, Thomas A. Strong. BTK-EndoMut Platform: Engineered Resistance Models Accelerating BTK-Targeted Drug Discovery [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr B099.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/723b954bdd0f1f0d06162f08a8fe3aac141048aa" target='_blank'>
              Abstract B099: BTK-EndoMut Platform: Engineered Resistance Models Accelerating BTK-Targeted Drug Discovery
              </a>
            </td>
          <td>
            Jinying B. Ning, feng B. hao, yue B. huang, Thomas A. Strong
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Acute myeloid leukemia (AML) relapse after allogeneic hematopoietic cell transplantation (allo-HCT) is increasingly recognized as a consequence of immune escape mechanisms by which cancer cells evade the graft-versus-tumor effect. Recent studies have shown that, in most relapses, these immune changes are driven by epigenetic rewiring of AML cells, leading to two distinct and mutually exclusive immune evasion strategies: downregulation of HLA class II molecules or upregulation of T-cell inhibitory ligands (e.g., PD-L1, B7-H3) (Christopher et al., NEJM, 2018; Toffalori et al., Nat Med, 2019).
 To investigate whether these relapse modalities are associated with distinct metabolic patterns, we analyzed 14 longitudinally collected sample pairs from AML patients at diagnosis and post-transplantation relapse, evenly distributed between the two modalities. Following AML blast purification, we conducted bulk RNA sequencing and liquid chromatography–mass spectrometry (LC-MS) profiling of intracellular metabolites (endometabolites) and extracellular metabolites from supernatants after short-term culture in a metabolically defined medium (exometabolites). In parallel, we functionally characterized these samples using the Single Cell ENergetIc metabolism by profilIng Translation inhibition (SCENITH) assay (Argüello et al., Cell Metabolism, 2020).
 In line with our previous studies, transcriptional profiling clearly separated the two patterns of relapse: downregulation of HLA class II molecules (downHLA, n=7), and upregulation of T cell inhibitory ligands (upInib, n=7).
 Notably, we observed that most transcriptional and metabolic alterations were exclusive to each modality, with more evident metabolic rewiring in upInib relapses. In particular, biological processes enriched in these relapses included hypoxia (p=0.035) and nitric oxide (NO) biosynthesis (p=0.029), and endometabolome analysis highlighted increased intracellular levels of citrulline (FC=2.22, raw p=0.015) and arginine metabolites (FC=1.44, raw p=0.046), which are known substrates in nitric oxide synthesis. Notably, we also detected marked increase in intracellular pyroglutamic acid (FC= 1.71, raw p=0.046), a key intermediate in the γ-glutamyl cycle, suggesting a transcriptional and metabolic shift towards a pro-oxidative/nitrosative state at relapse.
 These findings indicate that upInib relapses adapt to a pro-inflammatory and pro-oxidative microenvironment and may rely more heavily on tumor microenvironment (TME)-driven metabolic reprogramming than their downHLA counterparts.
 Our analysis also revealed metabolic changes common to both relapse modalities. For instance, SCENITH demonstrated similar alterations in bioenergetic pathways in the two types of relapse, with increased mitochondrial dependence and reduced glycolytic capacity (p = 0.015). Furthermore, independently from the immune escape modality, in the exometabolome we detected a significant increase in lactic acid (LA) at relapse (p=0.007), which was confirmed by a validation colorimetric assay performed using both primary patient samples and AML patient-derived xenografts. Besides its known activity as immunosuppressive oncometabolite, LA has recently been shown to function also as a direct epigenetic regulator (Zhang et al, Nature, 2019). For this reason, we investigated by immunofluorescence analysis lactylation of specific hystone sites in our relapses, detecting a significant increase in H4K8la (p=0.04). CUT&Tag sequencing is currently underway to identify differentially lactylated genes and thus the role of pan- and residue-specific lactylation in driving relapse-associated gene expression programs.Taken together, our data provide a detailed landscape of the metabolic alterations associated with AML relapse, highlighting that most of them are modality-specific. These findings may serve as a roadmap to test therapeutic approaches aimed to untangle the epigenetic/metabolic TME rewiring driven by AML relapse, especially for upInib cases in which direct targeting by checkpoint blockade has yet largely failed at improving patient outcome.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2210ec6a9b70ffb5e5005ce8aa49e5a98d749447" target='_blank'>
              Metabolic signatures mirror divergent modalities of leukemia immune evasion and relapse after transplantation
              </a>
            </td>
          <td>
            Annalisa Tameni, Emanuele Rondelli, Angela Gentile, Vittorio Niccolucci, Maria Chiara Di Bonito, Fatemeh Kordevani, Francesca Tortorelli, Simone Cardaci, R. Di Micco, D. Lazarević, Giovanni Tonon, D. Sumpton, Alejandro Huerta Uribe, Marco Punta, C. Toffalori, F. Ciceri, L. Vago
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="


 Large B-cell lymphoma (LBCL) is an aggressive hematological malignancy characterized by significant clinical and genetic heterogeneity, resulting in variable prognosis and treatment response.
 It has become increasingly evident that the elements of the lymphoma tumor microenvironment (TME) impact on cancer progression and drug resistance. The TME features may be heterogeneous in LBCL and related to specific molecular subtypes, such as those characterized by the dark-zone (DZ) gene signature. MicroRNAs (miRNAs), small non-coding RNAs that regulate gene expression, play a pivotal and multifarious role in cancer biology by directly influencing tumor cell behavior, regulating immune evasion, and shaping TME.
 In this study, we performed a transcriptome analysis by total RNA sequencing on tumor samples from 39 de novo LBCL patients treated with R-CHOP first-line treatment, 20 refractory/relapsed (R/R) patients, and 19 therapy-responders. Using the Molecular Functional Portraits bioinformatic tool, we classified tumors according to gene expression profiles into prognostically relevant TME subtypes. Notably, R/R patients were enriched for the “immune-desert” (ID) subtype, which is associated with the poorest clinical outcomes.
 Given that the high-risk DZ-signature subtype has been previously characterized by an immunosuppressive TME, we examined the overlap between the gene expression profiles of our ID samples and the DZ signature. Remarkably, 84 out of 104 genes composing the DZ signature showed similar modulation in ID samples, underscoring a strong correlation between the DZ signature and the ID phenotype.
 To investigate the role of miRNAs in shaping TME subtypes, a small-RNA sequencing was performed on samples from the same patient cohort. We found 124 miRNAs differentially expressed between ID patients and those from other TME groups. We observed a significant positive correlation (Spearman's R = 0.35; p=0.00012) between miRNA expression changes linked to TME subtypes (ID type vs other TME types) and those associated with treatment response (R/R vs responding patients), highlighting the connection between miRNA-mediated TME remodeling and clinical outcomes. Of note, four of the identified TME-related miRNAs—miR-130b-3p (upregulated in ID and R/R patients), miR-100-5p, miR-99b-5p, and miR-125a-5p (all downregulated in ID and R/R patients)— were also able to accurately discriminate R/R from responsive patients with an area under the ROC curve >0.67. These miRNAs have previously been implicated in modulating the tumor immune microenvironment across various solid and hematologic malignancies.
 Moreover, an anticorrelated target and pathways analysis shows that genes targeted by the identified TME-related miRNA signature were enriched in pathways critical for TME regulation and tumor immune evasion, such as the Notch and IL6-JAK-STAT3 signaling pathways. Deconvolution analyses are ongoing to identify which TME components differ between tumors with high versus low expression of this four-miRNA signature. This will allow a better understanding of the molecular mechanisms by which these miRNAs influence the tumor microenvironment and treatment response. We will further validate all these data in a larger cohort of LBCL patients as well as in cellular models of resistance to R-CHOP.
 Altogether, our results represent a comprehensive, multi-layered approach to understanding how microRNAs contribute to drug resistance by reshaping the TME in LBCL and may pave the way for novel combined therapeutic strategies, holding significant promise for advancing precision medicine.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd5e42361d62ed7960e9e705935b24b001e66c4e" target='_blank'>
              An immune desert phenotype underlies therapy resistance in LBCL through distinct microenvironment-related miRNA profiles
              </a>
            </td>
          <td>
            Giulia Regazzo, G. Vari, F. Marchesi, F. Palombi, Andrea Sacconi, Giulia Orlandi, M. Novello, E. Papa, Martina Tomassi, F. Bertoni, E. Maiolo, A. Mengarelli, S. Hohaus, M. Rizzo
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="


 Introduction: ASXL1-mutant (mt) Chronic Myelomonocytic Leukemia (CMML) is an aggressive myeloid neoplasm characterized by increased proliferation, resistance to epigenetic therapies, and poor survival. Epigenetic remodeling and up-regulation of leukemogenic driver genes are hallmarks of ASXL1mt CMML. Truncating mutations in ASXL1 activate enhancers that drive the expression of key leukemogenic driver genes such as HOXA9 and its co-factor MEIS1. Here, we investigate the chromatin accessibility of these leukemogenic enhancers by ASXL1 genotype at single-cell resolution (GoT-ChA-seq).
 Methods: After IRB approval, we performed bulk RNA-seq and ChIP-seq (H3K4me1, H3K4me3, H3K27ac, H3K27me3, H2AK119ub, H2BK20ac) on bone marrow (BM) mononuclear cells from 40 patients with CMML (19 ASXL1mt, 21 -wildtype/wt). After quality control, differential analyses of gene expression and histone modification occupancy between ASXL1mt and wt patients were conducted. A consensus peak calling framework was used to define genomic regions with ASXL1mt-specific histone modifications. Canonical enhancers were defined as genomic regions with a co-occupancy of H3K4me1 and H3K27ac. We then added regions marked by H2BK20ac to capture regulatory regions associated with p300 (Narita et al. Nat Genet 2023). We used an in silico prediction strategy to associate regulatory regions marked by strong histone modification occupancy with highly up-regulated leukemogenic driver genes (HOXA6-10, MEIS1). The ReMap database (Hammal et al. Nucleic Acids Res 2022) was queried for transcription factor binding in the genomic regions of interest. We then performed GoT-ChA-seq (Izzo et al. Nature 2024) to investigate chromatin accessibility for ASXL1mt (n=3) and wildtype BM single cells from CMML patients. In vitro colony forming assays were conducted on primary CMML BM cells (11- ASXL1mt and 9 wt CMML), assessing the efficacy of EP31670, a novel, oral, dual inhibitor of p300/BRD4.
 Results: This study included 40 patients with CMML, 19 ASXL1mt and 21 wt. The variant allele frequencies for ASXL1 were compatible with heterozygosity (median 0.41) and TET2 mutations were balanced between the two groups. We identified 149 candidate regulatory regions correlating with the expression of up-regulated target genes of interest (HOXA6-10, MEIS1) by bulk ChIP- and RNA-seq. Publicly available ChIP-seq data from myeloid cell lines demonstrated enrichment of p300 (in THP-1 cells, enrichment effect size +2.58, p=0.002) and BRD4 (in K-562 cells, enrichment effect size +1.58, p=0.045) in these candidate regulatory regions. Next, we interrogated the chromatin accessibility of ASXL1mt (n=13620) and ASXL1wt (n=24269) single cells from 3 CMML patients using GoT-ChA-seq. Differential accessibility analysis revealed 144 genomic regions, all of which were preferentially accessible in ASXL1mt cells compared to ASXL1wt cells (p<0.05 for all regions). Four of these preferentially accessible genomic regions (p<0.001 for all regions) overlapped with the previously identified candidate regulatory regions for MEIS1. The H3K27ac and H2BK20ac occupancy levels in these regions was strongly associated with the expression of MEIS1 (median Pearson r=0.73, p=1.16x10-7). These candidate regulatory regions were annotated in ENCODE as distal and proximal enhancers. Targeting this biology with the dual p300/BRD4 inhibitor EP31670 in progenitor colony forming assays led to a preferential therapeutic effect in ASXL1mt BM samples (median IC50 = 25nM) compared to in ASXL1wt samples (median IC50 = 74nM, p=0.03). Treatment with EP31670 was also associated with increased differentiation (increased blast forming units-erythroid; BFU-E) and organization of colonies in ASXL1mt vs wt CMML.
 Conclusions: ASXL1mt CMML is characterized by the up-regulation of several leukemogenic driver genes including HOXA9 and its co-factor MEIS1, secondary to the activation of genotype-specific enhancers. The preferential accessibility of these enhancers can be defined between ASXL1mt and ASXL1wt cells within patients. The enhancers of interest interact with p300 and BRD4 in myeloid cell lines and targeting them with the novel dual p300/BRD4 inhibitor, EP31670, induces therapeutic responses at low nanomolar drug concentrations in vitro, preferentially in ASXL1-mutant cells. These data solidify the biological rationale for targeting p300/BRD4 in ASXL1mt chronic myeloid neoplasms for therapeutic benefit (NCT05488548).
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d58c09cc34747d2000f4bf32710a735358c8669" target='_blank'>
              Clone-specific epigenetic regulatory mechanisms in ASXL1-mutant chronic myelomonocytic leukemia
              </a>
            </td>
          <td>
            Moritz Binder, T. Lasho, Jenna A. Fernandez, Wazim Mohammed Ismail, Amelia Mazzone, C. Finke, Pankaj Pradeep, A. Mangaonkar, Nickolas Steinauer, Huihuang Yan, Liguo Wang, Alexandre Gaspar-Maia, Mrinal Patnaik
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) represent a resilient subpopulation within tumors, capable of driving progression, metastasis, and recurrence. One mechanism that enables this plasticity is asymmetric cell division (ACD), a process by which CSCs generate both a self-renewing stem cell and a differentiated daughter cell. While traditionally associated with tissue development, ACD is now recognized as a dynamic and regulated feature of cancer biology, particularly in response to stress conditions such as hypoxia and radiation. In this review, we provide a comprehensive and mechanistic synthesis of how intrinsic factors such as polarity complexes, cell fate determinants (CFDs), spindle orientation cooperate with extrinsic cues from the tumor microenvironment to orchestrate ACD in CSCs. We explore how this process contributes to tumor heterogeneity, therapy resistance, and the emergence of quiescent, drug-tolerant CSCs across multiple malignancies, including brain, breast, colorectal, and hematologic cancers. Importantly, we highlight recent efforts to pharmacologically disrupt or redirect ACD using inhibitors of NOTCH, WNT, AURORA kinases, and MSI1, presenting ACD as a therapeutic vulnerability rather than a static trait. By shifting the focus from CSC markers to division mode, this review introduces a novel conceptual framework for targeting tumor hierarchy and plasticity. Understanding and manipulating ACD offers a promising frontier in precision oncology - one where altering the balance of cell fate decisions could limit relapse, reduce intratumoral complexity, and enhance long-term treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b911960284166687ce9462f6da53a77bba5e7d79" target='_blank'>
              Targeting asymmetric division in cancer stem cells: implications for relapse and resistance
              </a>
            </td>
          <td>
            Mona Latifi, Mahsa Sani, F. Sani
          </td>
          <td>2025-12-01</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fbb0b650183e1ab55b2143b3779619fab2e24dd" target='_blank'>
              aiAtlas: High-Fidelity Cell Simulations of Genetic Perturbations in Rare Diseases and Cancers
              </a>
            </td>
          <td>
            WR Danter
          </td>
          <td>2025-10-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer is increasingly recognized not as a fixed genetic condition but as a dynamic and plastic disease state driven by reversible epigenetic, transcriptional and microenvironmental cues. This evolving understanding supports a therapeutic paradigm shift: from eradicating malignant cells to reprogramming them toward quiescence, differentiation or functional normalization. This review explores diverse strategies for redirecting cancer cell fate, including differentiation therapy, epigenetic remodeling, lineage reprogramming, senescence induction and tumor microenvironment resetting. These approaches exploit intrinsic cellular plasticity and contextual adaptability, offering novel avenues to contain malignancy and overcome resistance. By reframing cancer as a potentially reversible phenotype, this therapeutic strategy demands redefinition of clinical endpoints, incorporation of dynamic biomarkers, and development of integrative treatment frameworks. Ultimately, reprogramming‑based oncology holds the promise of transforming aggressive malignancies into manageable conditions while minimizing the collateral damage associated with conventional cytotoxic therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99bfdfe1c6957af978b8fb4415d04fd96bdf7b85" target='_blank'>
              Beyond eradication: Therapeutic reprogramming strategies for cancer normalization (Review).
              </a>
            </td>
          <td>
            Aliasgar F. Shahiwala
          </td>
          <td>2025-12-04</td>
          <td>Oncology reports</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="


 Germline mutations in ETV6 cause thrombocytopenia and predisposition to malignancies. ETV6 (ETS Variant 6) is a transcriptional repressor essential for embryonic development and bone marrow hematopoiesis. Several variants have been identified in ETV6 and while mutations in the ETS DNA-binding domain of ETV6 impair direct DNA binding, the P214L mutation in the central domain leads to aberrant cellular localization of ETV6, decreasing transcriptional repression, and impairing megakaryocyte (MK) maturation. Despite its critical role in hematopoiesis, very few direct transcriptional targets of ETV6 have been identified in MK development. We sought to define ETV6's genome-wide binding sites and the impact of ETV6 P214L variant in differentiated MK.
 Using CRISPR/Cas9, we generated ETV6 knockout (KO) and P214L knock-in variants in human CD34⁺ hematopoietic stem cells, derived from cord blood, and then differentiated into megakaryocytes over 14 days. We performed Cleavage Under Target and Release Under Nuclease (CUT&RUN) to map ETV6 binding, assay for transposase-accessible chromatin sequencing (ATAC-seq) to assess chromatin accessibility, and bulk RNA-seq to profile transcriptional changes.
 ATAC-seq profiling demonstrated that the overall chromatin accessibility remained unchanged between controls and ETV6 KO cells, as well as between silent control and P214L mutants, indicating that ETV6 loss or mutation affects transcriptional regulation without altering global chromatin accessibility.
 CUT&RUN analysis revealed ETV6 binding activity in both wild-type and silent control (CRISPR/Cas9) megakaryocytes. In wild-type cells, we detected 1,757 peaks in replicate 1 and 3,570 peaks in replicate 2, with 974 target genes shared between both replicates. Similarly, silent control megakaryocytes identified 2,853 and 3,104 peaks in replicates 1 and 2, respectively, with 1,592 shared target genes. 722 genes were commonly bound by ETV6 in both WT and silent control megakaryocytes. ETV6 binding was drastically reduced in ETV6 KO cells (181 peaks in Rep 1 and 67 peaks in Rep 2) and in P214L mutants (326 peaks in Rep1 and 266 peaks in Rep 2). Over 60% of ETV6 peaks localized to promoters in controls, whereas ETV6 peaks in KO and mutants were largely in non-promoter regions. Motif analysis (MEME suite) of ETV6 binding sites revealed that ETS motifs were most significantly enriched in WT (E-value= 2.0e-424) and silent control sample (E-value= 4.2e-180), with ETV6 as the predominant ETS transcription factor. This ETS motif enrichment was lost in both ETV6 KO and P214L mutant cells. We also performed immunofluorescence and cell spreading assays in ETV6 knockout and wild-type megakaryocytes to assess their ability to form proplatelets. Both experiments showed that loss of ETV6 disrupts cytoskeletal organization in ETV6 KO megakaryocytes, preventing proper proplatelet formation indicating a critical role for ETV6 in late-stage megakaryocyte maturation.
 RNA-seq analysis revealed that among the common 722 target genes, transcripts for 44 of the genes were altered (41 upregulated and 3 downregulated) in the ETV6 KO; and 21 were altered in the P214L variant (18 upregulated and 3 down-regulated) with 18 of these up-regulated gene in P214L variant common among ETV6 KO as well. Pathway analysis identified significant enrichment of the estrogen signaling pathway (p = 0.0008), interleukin-2 signaling pathway (p = 0.0003), and MAP kinase pathway (p = 0.000008). A subset of genes, including ESR2, SHC1, SYK, and GNAI2, were commonly represented across these pathways, suggesting that disruption of ETV6 function may converge on these signaling networks to impair megakaryocyte development. We compared our data with previously published datasets on ETV6 mutations and, while we observed several overlapping genes, we also identified novel targets in our dataset including ESR2. When we analyzed its expression, ESR2 levels were ~7-fold higher in ETV6 KO megakaryocytes compared to wild type and ~5-fold higher in the P214L variant compared to the silent control. Prior studies have shown that estrogen supports megakaryocyte growth and maturation by activating GATA1 through ERβ, which in turn triggers STAT1 signaling. Our findings suggest that altered ESR2 expression may influence signaling pathway, offering new insights into how estrogen signaling might impact platelet production and how its disruption could impair megakaryocyte development.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b67bfb55ff0d7deeb31a92c2c0db80c89d1bc777" target='_blank'>
              Integration of Cut&Run, atac-seq and RNA-seq analyses defines the impact of germline ETV6 mutations on chromatin binding and gene expression in megakaryocytes
              </a>
            </td>
          <td>
            Hyder Mir, Jesse Rowley, Lilian Antunes Heck, Thomas J. Girard, Wei Yang, Abigail Ajanel, Jorge Di Paola
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Hematologic malignancies remain among the most difficult cancers to treat, challenged by profound heterogeneity, treatment-induced immune dysfunction, and the frequent emergence of drug resistance. Beyond tumor-intrinsic mechanisms, dysbiosis of the gut microbiome is increasingly recognized as a critical determinant of therapeutic outcomes, shaping hematopoiesis, immune responses, and drug metabolism. Bacteriophage (phage) therapy has re-emerged as a precision tool capable of selectively eradicating pathogenic taxa while preserving commensal short-chain fatty acid–producing communities. Preclinical and early human studies demonstrate that phages can recalibrate microbial ecosystems, disrupt antibiotic-tolerant biofilms, and enrich metabolites such as butyrate that support mucosal integrity and immune balance. Mechanistically, phage DNA enriched with CpG motifs engages Toll-like receptor 9, activating dendritic cells and enhancing cytotoxic T lymphocyte responses, suggesting dual benefits in infection control and anti-tumor immunity. Emerging applications extend further, with engineered phages serving as vectors for CRISPR–Cas gene editing, targeted cytokine delivery, and nanocarrier platforms for leukemia therapy. Despite translational promise, major hurdles persist, including immunogenicity, horizontal gene transfer, resistance evolution, and regulatory uncertainty. Addressing these challenges through GMP-compliant manufacturing, metagenomics-guided personalization, and AI-optimized cocktail design could establish phage therapy as a microbiome-informed adjunct to overcome drug resistance in blood cancers. However, direct clinical evidence of phage therapy efficacy in hematologic malignancies remains limited, and current data are largely derived from preclinical and compassionate-use contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc9ece11723039e03ff5895ac5fce518e0f2ca3a" target='_blank'>
              Phage therapy and the microbiome in hematologic malignancies: opportunities, mechanisms, and early evidence
              </a>
            </td>
          <td>
            Juanwen Zhang, Jin Liu, Alireza Bayani
          </td>
          <td>2025-12-12</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Clustered regularly interspaced palindromic repeats (CRISPR) gene editing, known for its precision, is revolutionizing tumor research and treatment. This article covers the evolution of the CRISPR system from Cas9, which produces DNA double‐strand breaks, to Cas12, Cas13, base editors, and prime editors. These breakthroughs allow DNA/RNA editing, transcriptional regulation, and epigenetic reprogramming. The study encompasses CRISPR's diverse applications in tumor research, including high‐throughput screening of cancer driver genes, identification of synthetic lethal targets, and analysis of drug resistance pathways across various cancer types. It highlights CRISPR's importance in therapeutic tactics, such as targeting oncogene inactivation, restoring tumor suppressor gene activity, and engineering immune effector cells, such as CAR T and NK cells, to increase persistence, cytotoxicity, and resistance to exhaustion. This review also examines CRISPR's role in modifying the tumor microenvironment (TME), encompassing the regulation of immune checkpoints, metabolic reprogramming, and the targeted eradication of specific immunosuppressive cells. Off‐target consequences for both delivery and efficiency are discussed, along with cutting‐edge technologies including high‐fidelity Cas variants, AI‐assisted sgRNA design, and stimuli‐responsive delivery systems. Finally, the paper examines how CRISPR can be integrated with combination medicines, multimodal editing methods, and single‐cell and multiomics data to deliver tailored precision‐based interventions. This review explains the molecular logic and translational routes of CRISPR, highlighting its promise to revolutionize precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7f333b854e7e8b8178a62bd98bd7d363744559a" target='_blank'>
              CRISPR Enabled Precision Oncology: From Gene Editing to Tumor Microenvironment Remodeling
              </a>
            </td>
          <td>
            Kailai Li, Peixin Huang, Yue Qian, Anqi Lin, Jingjun He, Junyi Shen, Li Chen, Kai Miao, Jian Zhang
          </td>
          <td>2025-11-05</td>
          <td>Med Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 Pathogenic germline variants in mismatch repair (MMR) genes, or Lynch syndrome (LS), increases patients’ risk of developing colorectal cancer (CRC). We previously identified shared, frameshift (fs)-neoantigens in MMR deficient (MMRd) cancers. We hypothesized that T cell surveillance occurs early in CRC development in LS but T cell activity is abrogated by immune regulatory programs during tumor progression. Blood and normal, precancerous (adenoma), and CRC tissues were analyzed from a cohort of 92 LS patients. We leveraged a custom neoantigen-discovery pipeline, functional antigen recognition assays, T cell receptor sequencing, and single cell and spatial transcriptomics. This identified for the first time T cells recognizing shared fs-neoantigens expressed in normal mucosa, adenomas, and tumors of LS patients. Immune editing was evidenced by i) distinct neoantigen repertoires in precancerous tissues compared to tumors and ii) recurrence of low affinity (but not high affinity) neoantigens in later lesions captured from the same patients. Transcriptomic and ex vivo functional analysis revealed that T cells in tumors, relative to precancerous tissues, had reduced functional capacity associated with an infiltration of immunoregulatory myeloid cells (tumor-associated neutrophils and TREM1+ macrophages) which can be therapeutically targeted in future clinical studies. Finally, these shared, precancerous fs-neoantigens can serve as vaccine targets to prevent MMRd cancer in LS.



 Supported by NIH UG3CA290517 and the Parker Institute for Cancer Immunotherapy



 Tumor Immunology: Cellular Responses and Tumor Microevironment (TIME)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0bfb57527590807c825fa581d5e3eb9f0eacef25" target='_blank'>
              Identifying targets of precancerous neoantigen-specific T cell surveillance in patients with Lynch syndrome and immune suppression programs supporting colorectal cancer progression 3671
              </a>
            </td>
          <td>
            Matthew Brown, L. Velazquez, Joyce Serebrenik, Storey Harbison, S. Holtzman, Mesude Bicak, Alexandros Polydorides, Cansu Cimen Bozkus, Aimee L Lucas, R. Samstein, Nina Bhardwaj
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 Next-generation sequencing (NGS) has revealed a broader range of somatic mutations in chronic myeloid leukemia (CML) patients (pts), which is associated with long-term outcomes, including ASXL1. The current ELN laboratory guideline recommend NGS profiling for pts progressing to blast phase (BP) but not at diagnosis or in optimally responding chronic phase (CP) CML pts. With effective suppression of the Philadelphia chromosome-positive (Ph+) clone, 5–15% of pts develop clonal evolution (CE) in Philadelphia-negative (Ph-) cells, such as monosomy 7 or trisomy 8, typically detected via metaphase cytogenetics. It is plausible to expect these clones to carry somatic mutation(s), which could be identified earlier by NGS. However, the dynamics of somatic mutation profile following long-term TKI therapy in CML pts is not well investigated. Also, the dynamics of somatic mutations in optimally responding pts remain underexplored, particularly for the emergence of new mutations. We hypothesized that such mutations may arise in Ph- cells. This study aimed to characterize these mutations, focusing on their genetic profiles and doubling times (DT), and to assess their clinical relevance.



 We analyzed paired peripheral blood samples from 51 CML pts treated at Princess Margaret Cancer Centre (Toronto, Canada) and University Hospital Brno (Brno, Czech Republic), selected from a 254-pts' cohort (Blood Advances 2024). DNA was extracted from mononuclear cells and sequenced using a single-molecule-tagging, molecular inversion probe (smMIP)-based approach. A custom CML-specific smMIP panel targeting 37 genes (332 amplicons) was used, covering epigenetic regulators, signaling pathways, transcription factors, spliceosome components, tumor suppressors, and cohesion complex genes. The assay had a detection limit of 0.1%. Mutation doubling time (DT) was calculated using two time points (T1 and T2) with the formula: DT = (T2 − T1) × log(2) / [log(VAF2) − log(VAF1)].



 Median age was 61 years (range: 17–80). Disease risk was stratified at diagnosis with 8 (16%), 27 (53%) 14 pts (26%) as low, intermediate and high Sokal risk group. First-line therapy included imatinib in 36 pts (71%) and second-generation TKIs in 15 pts (29%) (nilotinib n=12, dasatinib n=3). Sequencing was performed prior to TKI therapy and at a median of 398 days post-TKI therapy. At the time of 2nd sample collection, 42 pts (82%) had achieved an optimal response per 2020 ELN guidelines. Among them, somatic mutations were detected in 21 pts (50%). Eight pts (16%) had mutations at diagnosis with similar allele frequencies, while 13 (26%) acquired new mutations during TKI therapy. Frequently mutated genes among optimal responders with emerging mutations included DNMT3A (n=6), TET2 (n=5), ASXL1 (n=4), and EZH2 (n=2). Additional mutations in JAK2, SF3B1, U2AF1, PHF6, TP53, BRAF, and CBL were each seen in one patient. Median time to new mutation emergence was 227 days (range: 105–7578), with a median DT of 59 days (range: 22–1909). Among optimal responders with emerging mutations (n=13), median time to MR4 was 255 days (range: 168–776), compared to 469 days (range: 168–4008) in those without mutations (n=21; p=0.14). All pts achieving optimal responses with emerging mutations eventually achieved MR4 or deeper response with a median follow-up of 2.7 years (range: 294–7601 days) and none lost MR4 while on TKI therapy. Treatment-free remission (TFR) was attempted in 4 pts (31%), with 2 maintaining TFR at last follow-up.



 We have observed a new pattern of somatic mutation dynamics in CML pts on TKI therapy. Emergence of new somatic mutations was observed in CML pts achieving optimal TKI response. Thus, their presence is not necessarily a marker of clonal evolution toward TKI resistance or disease progression. Current ELN laboratory guideline does not recommend baseline NGS in CP-CML, limiting proper interpretation of new mutations during follow-up. Our findings support baseline mutation profiling in all CML pts at diagnosis, regardless of clinical status. Many detected mutations are consistent with clonal hematopoiesis, potentially contributing to the development of clonal evolution in Ph- clone after optimal TKI response. We now recommend regular monitoring of these mutations during TKI therapy. Given the median DT of ~60 days, annual NGS surveillance may be a practical strategy for early detection of significant clonal changes.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81d772762ab7713aaf44b98509d73b63a543a499" target='_blank'>
              Emergence of new somatic mutations in CML patients optimally responding to tyrosine kinase inhibitor therapy: Proposal of long-term genomic monitoring
              </a>
            </td>
          <td>
            Yael Morgenstern, Gopila Gupta, Flavia Patino, M. Perusini, D. Žáčková, I. Ježíšková, Anežka Kvetková, T. Jurček, Azin Keshavarz, Danielle Pyne, Oyeronke Ayansola, Amirthagowri Ambalavanan, Jiří Mayer, A. Arruda, Mark D. Minden, Jessie J. F. Medeiros, Sagi Abelson, D. Kim
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Ovarian cancer is the gynecologic malignancy that bears the highest mortality rate in the Western world. This is attributed to late diagnosis and limited therapeutic progress. Recent advances in molecular oncology have highlighted the pivotal role of epigenetic modifications—including DNA methylation, histone modifications, non-coding RNAs, chromatin remodeling, and RNA methylation—in ovarian cancer development, progression, and treatment resistance. DNA methylation patterns affect key tumor suppressors and oncogenes, while histone modifications alter chromatin accessibility, influencing gene expression. Chromatin remodeling complexes, particularly the SWI/SNF complex, are frequently mutated in specific ovarian cancer subtypes, which is central in shaping their biological behavior. Non-coding RNAs, including microRNAs and long non-coding RNAs, further regulate tumor cell behavior and the immunosuppressive tumor microenvironment. Epigenetic profiles vary among histological subtypes and hold promise for biomarker development, early detection, prognosis, and therapeutic monitoring. Liquid biopsy approaches leveraging circulating tumor DNA methylation show diagnostic potential superior to conventional markers. Moreover, targeting epigenetic regulators—such as DNMT and HDAC inhibitors, EZH2 antagonists, and RNA-modifying enzymes—offers novel avenues for treatment, particularly in reversing chemoresistance and sensitizing tumors to immunotherapy. While promising, these strategies require further validation through clinical research to translate into effective clinical interventions. This review aims to summarize the current literature and highlights potential applications of epigenetic manipulation in day-to-day practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e682da74ce847ab8714dc4327d4b867b8d126a5" target='_blank'>
              Epigenetics in Ovarian Cancer: A Review of Current Knowledge and Future Perspectives
              </a>
            </td>
          <td>
            Nikolaos Dedes, Michael Liontos, Dimitrios Haidopoulos, Flora Zagouri, K. Angelou, A. Svarna, A. Michas, Aikaterini Aravantinou Fatorou, Angeliki Andrikopoulou, Meletios A. Dimopoulos
          </td>
          <td>2025-11-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background Proto-oncogenes are abnormally activated in nearly all types of tumors. However, the epigenetic mechanism of proto-oncogene activation has not yet been well elucidated. Methods The present study involved the construction of a double-stranded cDNA library derived from gastrointestinal cancer cells, followed by high-throughput genome sequencing to select the c-MYC gene associated with colorectal cancer. Through RACE analysis, we identified the antisense RNA MYC-AS1 and its complete sequence. By investigating the cellular functions, expression of MYC-AS1, elucidating its interaction mechanism with the c-MYC gene, and exploring the impact of DNA methylation on MYC-AS1 expression, we uncover the fundamental principles and regulatory mechanisms underlying colorectal cancer development. Results Here, we show that a subset of proto-oncogenes,including c-MYC, possess antisense RNAs. Upregulation of c-MYC in cancer tissues was attributed to the silencing of its antisense RNA MYC-AS1 via DNA hypermethylation. MYC-AS1 RNA markedly inhibited the proliferation of cancer cells in vitro and impeded tumor growth in nude mice in vivo by repressing the expression of c-MYC via an RNAi mechanism. MYC-AS1 RNA bound directly to the HuR protein in the cytoplasm, enhancing the RNA stability of MYC-AS1. Furthermore, MYC-AS1 inhibited c-MYC-targeted gene LDHA expression. Unlike the well-characterized oncogenic long noncoding RNA PVT1, which is coamplified with MYC and enhances its stability, MYC-AS1 is epigenetically silenced and functions as a tumor suppressor through an RNAi mechanism, revealing a distinct layer of MYC regulation. Conclusion Our work provides a novel mechanism by which c-MYC is activated in cancer cells by epigenetic silencing of its antisense RNA, which functions as a tumor suppressor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c20b565d2f250856d2e173d09028c6873b6e2cb" target='_blank'>
              A novel mechanism by which c-MYC is aberrantly activated by epigenetic silencing of its antisense lncRNA in colon cancer
              </a>
            </td>
          <td>
            Xuming Hu, Ye Wei, Meiying Zhang, Chunfeng Dou, Liping Wang, Gul Zaib, Huixian Wu, Wang Guo, Xiaoyuan Wang, Shihao Chen, Qi Xu, Mingzhou Guo, Hengmi Cui
          </td>
          <td>2025-10-21</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Chronic myeloid leukemia (CML) is a stem cell‐driven neoplasia characterized by the expression of the constitutively active tyrosine kinase (TK) BCR/Abl. Under low oxygen, a condition that characterizes stem cell niches (SCNs) in vivo, the oncogenic BCR/Ablprotein is suppressed. Consequently, leukemia stem cells (LSCs) residing within SCNs show resistance to TK inhibitors (TKIs), the first‐line therapy for CML, due to the lack of their molecular target. It is therefore important to deepen understanding of the mechanisms driving BCR/Ablprotein suppression to design new strategies able to repress TKI‐resistant LSCs. Our previous studies showed that BCR/Ablprotein suppression occurred when glucose approaches completed exhaustion in culture medium. As lactate is the main byproduct of glucose catabolism in low oxygen, in this study we addressed the role of lactate in regulating BCR/Ablprotein expression. We found that treatment of CML cells with 2‐DG, which blocks glycolysis and thereby lactate production, or monocarboxylate transporter (MCT) inhibitors, which reduce lactate excretion, enhanced BCR/Ablprotein expression and promoted maintenance of a BCR/Abl‐dependent/TKI‐sensitive stem cell phenotype. The effects of MCT inhibition were abolished when exogenous lactate was added to culture medium, resulting in the suppression of BCR/Ablprotein expression. Treatment with 3‐hydroxy‐butyrate acid, an antagonist of the GPR81 plasma membrane ‘lactate’ receptor, prevented lactate‐driven BCR/Ablprotein suppression, while the selective GPR81 agonist 3‐chloro‐5‐hydroxy‐BA counteracted the maintenance of BCR/Ablprotein induced by MCT inhibition. Our results indicate that GPR81 engagement by extracellular lactate determines BCR/Ablprotein suppression in low oxygen environments. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cae18f003bdc71482737b01a3310f00adc979030" target='_blank'>
              Lactate‐mediated activation of GPR81 regulates BCR/Abl protein expression in chronic myeloid leukemia cells selected under low oxygen tension
              </a>
            </td>
          <td>
            G. Menegazzi, Dayana Desideri, A. Biagioni, E. Rovida, P. Dello Sbarba, S. Peppicelli
          </td>
          <td>2025-10-23</td>
          <td>The Journal of Pathology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Brain tumors such as glioblastoma remain among the most lethal and immunologically resistant cancers, in large part due to epigenetic programs that sculpt the tumor–immune microenvironment. DNA methylation, histone modifications, and chromatin remodeling do not merely drive tumor-intrinsic changes; they also profoundly reprogram immune responses, shaping antigen presentation, cytokine signaling, and immune cell recruitment. At the center of this regulation are T cells, whose effector functions are suppressed through promoter hypermethylation of antigen-processing genes, silencing of interferon pathways, and the establishment of exhaustion-specific chromatin states. Mutations such as IDH1/2 and H3K27M further reinforce these epigenetic barriers, fostering immune-cold microenvironments that disable cytotoxic T-cell activity. Emerging evidence highlights both CNS-specific adaptations, including microglial and astrocytic epigenetic programs that reinforce immune privilege, and conserved features of T cell exhaustion that mirror those in peripheral cancers. This duality underscores the need for therapeutic strategies that dismantle CNS-specific barriers while leveraging shared exhaustion programs across tumor types. Epigenetic drugs, ranging from DNA methyltransferase and EZH2 inhibitors to BET degraders and CRISPR-based epigenome editors, are beginning to restore antigenicity, reverse T cell dysfunction, and sensitize tumors to checkpoint blockade. Yet these approaches carry the paradoxical risk of disrupting CNS immune tolerance, potentially triggering harmful neuroinflammation or autoimmunity. To our knowledge, this is among the first comprehensive reviews to integrate CNS-specific immune privilege mechanisms with peripheral exhaustion pathways, providing a unified perspective on how epigenetic regulation orchestrates immune dysfunction across central and peripheral contexts. By mapping the continuum between immune evasion and global immunosuppression, we propose a conceptual framework for tailoring epigenetic-immunotherapy combinations to achieve durable antitumor immunity in the CNS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d7202ed1c4ffc9d0ee982c0f1ad42403d845276" target='_blank'>
              Beyond the genome: epigenetic regulation of immune responses and T cells in brain tumors
              </a>
            </td>
          <td>
            Shuo Sun, Yu Han, Haiying Li, Chengyan Wang, Shu Zhou, Xiaowei Zhang, Shuhong Dai, Yao Peng, Zhuoqun Wang
          </td>
          <td>2025-11-18</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f662f4b181b3395cd096885302fa365b2511567e" target='_blank'>
              Exploring tumor heterogeneity and drug resistance in cervical cancer through single-cell and bulk transcriptomics.
              </a>
            </td>
          <td>
            Yinxia Yang, Sirong Cheng, Yuxuan Zhong, Pengyu Sun, Xiaoru Deng, Bohui Zhou
          </td>
          <td>2025-11-27</td>
          <td>Discover oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Lineage infidelity, manifested as either lineage drift or lineage switch, has emerged as a mechanism of resistance to B-cell–targeted therapies. While lineage switch has been most extensively documented in KMT2A-rearranged (KMT2A-r) B-cell acute lymphoblastic leukemia (B-ALL), lineage switches have also been reported in non-KMT2A-r subtypes under the pressure of B-cell-targeted therapy. We anticipate that altered expression of B-cell antigens (including full lineage switches) will become more common as CD19 targeted T-cell engagers such as blinatumomab are broadly incorporated into frontline therapies. We and others have previously demonstrated the presence of small subclones with stem cell/myeloid transcriptional signature in B-ALL at initial diagnosis using single cell multi-omics, suggesting a pre-existing reservoir of cells that are not committed to a B-cell fate.
 To explore the presence and functional significance of lineage infidelity in B-ALL, we performed comprehensive multiomic profiling—including single-cell RNA sequencing (scRNA-seq), single-cell ATAC-seq (scATAC-seq), single-cell DNA-seq (scDNA-seq), CITE-seq, and high-dimensional spectral flow cytometry on a cohort of standard- and high-risk paired pediatric B-ALL samples collected at diagnosis and at the end of induction therapy. Our integrated analysis uncovered the following key insights:
 Evidence of CD19 surface protein negative stem cell/myeloid leukemogenic subclones: Prior studies identified stem cell/myeloid subclones using single cell transcriptomic methods, which are inferior in assessing surface protein expression of several antigens, including CD19, due to high drop-out. In this study we validated the existence of this stem cell/myeloid-like population by detecting known leukemia-defining mutations such as KRAS, PAX5, and CREBBP within this population, confirming their clonal relationship to the bulk leukemia, and we confirm their surface protein marker profile using CITE-Seq and spectral flowcytometry.
 The stem/myeloid-like population showed reduced or absent expression of canonical B-cell markers such as CD19, CD22, and CD79a, both at the transcript and protein levels. Unexpectedly, this phenotype was not restricted to KMT2A-r ALL but was also observed in other subtypes such as Ph-like, Ph+, and iAMP21. To assess the functional potential of these lineage-infidel subclones, we sorted CD19⁻ blasts and transplanted them into immunodeficient NSG mice. These cells were successfully engrafted and gave rise to leukemia. This is the first study to provide evidence of lineage plasticity by integrating surface protein marker detection with transcriptional profiling and functional assays documenting the leukemogenicity of these clones.
 Stem cell/myeloid-like blasts enriched at minimal residual disease (MRD): Analysis of paired initial diagnosis and MRD samples documented enrichment of stem cell / myeloid clones at MRD. Again, this was not confined to KMT2A-r samples but also seen in several patients with other high-risk genotypes. Integration with drug profiling suggests increase resistance of the myeloid subclones to standard ALL chemotherapy agents. These findings suggests that in a subset of patients ALL Induction chemotherapy selects for a residual CD19-negative blast population that may jeopardize the efficacy of subsequent B-cell directed immunotherapy such as blinatumomab.
 Collectively, our findings have two key implications: (1) our data challenges current risk stratifications that rely on B-lineage flowcytometry to detect residual blasts at the end of Induction. Patients with a shift in lineage phenotype may potentially be misclassified when using standard flow cytometry based on lymphoid markers only. Lineage infidelity may also explain the occasionally observed discordant MRD assessments when using both flowcytometry and molecular methods. (2) Our findings have implications for treatment. Flowcytometry for stem cell / myeloid markers at MRD may identify a subgroup of patients that benefits less from blinatumomab. In addition, we identified surface protein markers associated with the CD19-negative population that may serve as alternative targets for novel immunotherapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2f768f774edfacf769eb24ded149337c40bdec0" target='_blank'>
              Chemotherapy selects for myeloid subclones in multiple subtypes of B-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Fatemeh Alikarami, Rushabh Mehta, Changya Chen, L. Brodersen, D. Teachey, Kai Tan, K. Bernt
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Introduction Philadelphia chromosome-positive acute lymphoblastic leukemia (Ph + ALL) represents the most prevalent genetic subtype of adult acute lymphoblastic leukemia (ALL). Despite the availability of targeted therapy regimens, patients with comorbidities and older patients have poor prognoses. They are prone to relapse, necessitating the urgent identification of new safe and effective treatment options. Luteolin (LUT), a natural flavonoid compound, has demonstrated significant anticancer activity. However, its mechanism of action in the context of Ph + ALL remains poorly understood. The objective of this study was to elucidate the potential mechanisms underlying the action of luteolin in Ph + ALL. Methods Luteolin-related targets and Ph + ALL associated targets were collected from several public databases. The intersection of these targets was then analyzed for protein-protein interactions (PPI). Additionally, we performed functional and pathway enrichment analyses employing the Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) methodologies. Core targets were selected from the PPI network, and some of these targets were further verified through cellular experiments. Results A total of 568 luteolin targets and 1,063 Ph + ALL targets were identified, with 154 overlapping targets. The top ten targets with the highest degree values were selected as core targets, which include TP53, AKT1, ALB, TNF, JUN, IL6, EGFR, STAT3, CASP3, and BCL2. Based on GO and KEGG enrichment results, the phosphatidylinositol 3-kinase/protein kinase B (PI3K/AKT) signaling pathway was further investigated. Cell experiments demonstrated that luteolin reduced the viability of SUP-B15 cells in a time- and concentration-dependent manner. Additionally, luteolin led to an increase in reactive oxygen species (ROS) accumulation, a decrease in mitochondrial membrane potential (MMP), and a reduction in ATP content in SUP-B15 cells. At the molecular level, luteolin significantly downregulated the protein expression of p-PI3K, p-AKT, p-STAT3 and BCL-2, while upregulating the protein expression of BAX, cleaved caspase-3, and cleaved caspase-9. Conclusion Luteolin may exert anti-Ph + ALL effects through the PI3K/AKT signaling pathway, accompanied by the regulation of other targets such as STAT3, which provides a theoretical basis for the development and screening of novel anti-Ph + ALL therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a53db02706ec9173ec9411c9d69aae837556b367" target='_blank'>
              Luteolin induces apoptosis in Philadelphia chromosome-positive acute lymphoblastic leukemia cell by regulating the PI3K/AKT signaling pathway
              </a>
            </td>
          <td>
            Qin Ren, Xiaobing Li, Xiangmei Ye, Leiguang Feng
          </td>
          <td>2025-11-10</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Multiple myeloma (MM) is a malignancy of clonal plasma cells, accounting for 10% of hematologic malignancies. Both primary (occurring early, including IgH translocation and hyperdiploidy) and secondary (acquired during disease progression, including del 17p/TP53, del 13q, amp 1q) cytogenetic abnormalities play a critical role in disease pathogenesis, prognosis, and therapeutic decisions. While conventional karyotyping is hindered by the low proliferative activity of plasma cells in MM, fluorescence in situ hybridization (FISH), as the gold standard for this purpose, provides only targeted detection being less cost-effective. LeukoPrint is a novel technology based on a shallow (1x) whole-genome sequencing (sWGS) to profile copy number aberrations (CNAs) in MM. Our previous works have demonstrated LeukoPrint's superior performance over FISH and karyotyping in cytogenetic characterization and risk stratification in MM and other malignancies, prompting a prospective multicenter clinical trial (ChiCTR2300077768) to assess the clinical relevance of LeukoPrint-detected CNAs in MM. The interim results of the first 102 participants were reported here.



 In this multicenter clinical trial, we aim to enroll a total of 396 patients who were newly diagnosed as active multiple myeloma (aMM) according to the International Myeloma Working Group (IWMG 2014) criteria for the diagnosis of MM. By June 12, 2025, 102 patients have been enrolled from 7 participating hospitals in China. The study was approved by all institutional Ethics Committees, with informed consent obtained from all participants. For each patient, LeukoPrint was used to assess CNAs in paired bone marrow (gDNA from CD138-purified plasma cells) and peripheral blood (circulating cell-free DNA, cfDNA), collected at diagnosis, post-induction, and relapse phases. CNAs only those greater than 5 Mbp were analyzed. For most patients, Karyotyping and FISH were also performed in parallel, and their results were compared with LeukoPrint data. Risk stratification was performed following the Mayo Stratification of Myeloma and Risk-adapted Therapy (mSMART) criteria.



 LeukoPrint detected CNAs in 91 out of 102 patients (89.2%), with a total of 868 CNAs identified. The most frequent alterations were 1q+ and -13, each present in 51% of the cohort. Odd numbered trisomies, including +9, +15, +19, and +5, represent another type of common CNAs, which are detected in 46.1% of the cohort and commonly cluster together in individual patients. Among patients with IGH translocations, subdiploidies were prevalent (54%), while hyperdiploidies were rare (13%), suggesting mutual exclusivity between these primary genetic events. In contrast, hyperdiploidies were detected in 40% of patients lacking IGH translocations. FISH (1q21+, 13q14-, and 17p13-) was examined in parallel with LeukoPrint in 94 cases, and the overall consistency was 88.0% (Kappa test: 0.73, strong consistency), while the sensitivity and specificity were 85.7% and 89.1%, respectively. Among 40% of patients who were FISH-negative, LeukoPrint detected CNAs in 80% of cases, showing its superior detection capacity over FISH. Although follow-up data are unavailable for this cohort, our prior findings demonstrated that LeukoPrint-detected additional CNAs, including trisomy, hyperdiploidy, and subdiploidy, redefined risk stratification according to mSMART criteria in 17.2% of patients. Moreover, LeukoPrint demonstrated significantly higher CNA detection rates than karyotyping (90.1% vs. 8.8%) in a direct comparison of 91 cases. This substantial disparity likely reflects the known limitation of karyotyping in MM due to the low proliferative activity of plasma cells. Finally, LeukoPrint was successfully performed on blood-derived ctDNA in all but one patient (excluded due to hemolysis). CNAs were detected in 89.1% of bone marrow samples compared to 61.4% of blood samples, demonstrating the clinical feasibility of ctDNA-based liquid biopsy.



 LeukoPrint outperforms both FISH and karyotyping, representing a convenient, efficient, and cost-effective platform for comprehensive genome-wide CNA profiling in MM. Our data also demonstrates its clinical utility for liquid biopsy using ctDNA. Collectively, these findings position LeukoPrint as both a complementary tool and potential replacement for traditional cytogenetic approaches in MM diagnostics.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8d1f64144bb998188a7b85ecddc0d45108a3624" target='_blank'>
              Interim results from a multicenter trial assessing the clinical relevance of copy number aberrations detected by a shallow whole-genome sequencing analysis (LeukoPrint) in multiple myeloma.
              </a>
            </td>
          <td>
            Li Gao, Liping Su, Fengyan Jin, Zhongxing Jiang, Jianyu Weng, Baijun Fang, Zhenyu Li, Lin Qiu, Chenyu Ding, Lijuan Zhang, Manqian Li, Yi-Mieng Chang, Wei Gong, Xi Zhang, Shichun Tu
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Bladder cancer is a highly heterogeneous malignant tumor of the urinary system with high recurrence rates, posing significant challenges in its diagnosis and treatment. Advances in multi-omics research have elucidated the molecular mechanisms underlying the pathogenesis and progression of bladder cancer, including driver gene mutations (e.g., FGFR3, TP53/RB1), dysregulation of signaling pathways (such as PI3K/AKT/mTOR and RAS-MAPK), epigenetic alterations, non-coding RNA networks, tumor microenvironment remodeling, and metabolic reprogramming. This review systematically summarizes recent progress in translational research bridging molecular mechanisms to breakthroughs in precision therapy, covering the clinical applications and challenges of FGFR inhibitors, immune checkpoint inhibitors, antibody-drug conjugates, and gene therapies. Translational efforts are increasingly relying on molecular subtyping to develop subtype-specific treatment strategies. Although significant advances have been made in precision therapy for bladder cancer, critical research gaps remain, including tumor heterogeneity, therapy resistance, and insufficient validation of biomarkers. Future research directions emphasize the potential of liquid biopsy for non-invasive diagnosis and dynamic monitoring, rational combination therapies, multi-omics data integration, and artificial intelligence in advancing personalized treatment, providing a systematic and forward-looking perspective on precision medicine in bladder cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d14a6737ae3e58aea00cf12ac1c7b98be1125652" target='_blank'>
              Molecular mechanisms and translational advances in bladder cancer: from driver genes to precision therapy
              </a>
            </td>
          <td>
            Jinjie Xiang, Yuhui Luo, Baiyu Zhang, Kunbin Ke, Hao Li
          </td>
          <td>2025-12-08</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Head-and-neck squamous cell carcinoma (HNSCC) remains a lethal malignancy with stagnant survival despite advances in surgery, radiotherapy, and systemic therapy. Beyond cetuximab and PD-1 inhibitors, there are only a few targeted options, which benefit only a minority of patients, underscoring the need for new biomarkers and druggable dependencies. Genome-wide clustered regularly interspaced short palindromic repeats (CRISPR) Cas9 screening now enables systematic, high-specificity investigation of gene function to reveal determinants of tumor proliferation, survival, and therapy response. Compared with RNA interference, CRISPR provides cleaner on-target knockout and more interpretable phenotypes, allowing efficient discovery of essential genes and synthetic-lethal interactions. Although the Cancer Dependency Map profiled 89 OSCC/HNSCC lines to nominate baseline dependencies, drug-perturbed states critical for understanding platinum resistance remain underexplored. Only a handful of HNSCC studies have applied genome-wide CRISPR cas9 screening: two mapped core essential genes; two mapped cisplatin resistance and radiation resistance; and others uncovered synthetic-lethal targets, including vulnerabilities to mTOR inhibition, EGFR inhibition, glutamine metabolism inhibition, and host determinants of oncolytic HSV-1 efficacy. This review synthesizes these findings, highlights methodological considerations (library design, coverage, and treatment duration), and integrates complementary functional data to prioritize targets for rational combinations. This review also provides information on the TCGA database and in vivo CRISPR screening that can accelerate precision therapeutics for patients with HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a31c18994958d899b77bdbb4cbe1e355e7819ca3" target='_blank'>
              Target Discovery in Head-and-Neck Squamous Cell Carcinoma: Genome-Wide CRISPR Screens Illuminate Therapeutic Resistance and Actionable Dependencies
              </a>
            </td>
          <td>
            V. K. Vincent-Chong
          </td>
          <td>2025-12-08</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="In the past 20 years, advances in genomic technologies have greatly improved our understanding of pediatric acute myeloid leukemia (AML). Today, cytogenetic tests can detect structural changes in approximately 75% of cases and remain a main tool for assessing risk. Recent technologies, such as next-generation sequencing, are revealing additional structural alterations (cryptic fusions) and mutations that often cooperate to influence disease biology and treatment response. This evolving genetic landscape has identified unique childhood subtypes of AML defined by specific fusions, such as NUP98::NSD1, CBFA2T3::GLIS2, and varied KMT2A rearrangements, which are linked to distinct clinical outcomes. Emerging data also point to the poor prognosis associated with certain subtypes of NPM1, like the NPM1-D isoform. Additionally, mutations in genes like WT1, DNMT3A, and TP53, the latter of which are rare in childhood AML, may influence patients' outcomes, particularly when occurring in combination. Targeted therapies, including FLT3, BCL2, and menin inhibitors, are beginning to reshape treatment, offering more personalized approaches. However, integrating these drugs effectively into the patient's treatment strategy remains challenging due to the genetic complexity and rarity of pediatric AML. Key issues ahead include identifying which genetic features truly affect outcomes, using this information to personalize therapy, predicting who will benefit from targeted drugs, and choosing the best markers to track disease response over time. Looking forward, collaborative efforts are urgently needed to validate pediatric-specific biomarkers, test novel drug combinations, and link genetic data to clinical outcomes to design trials and future treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03552c285ee4b593766096eaba70240ea7a0a020" target='_blank'>
              Using genomics to refine pediatric AML risk stratification.
              </a>
            </td>
          <td>
            M. Pigazzi, S. Meshinchi, Franco Locatelli
          </td>
          <td>2025-12-05</td>
          <td>Hematology. American Society of Hematology. Education Program</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="


 The estimation of the in vitro potency of anticancer agents against tumor cell lines in tissue culture is known to be fraught with a high degree of variability. While quality control of experimental conditions has been demonstrated by others to reduce this variability, other sources of variability remain even after standardization of experimental conditions. In this poster, we assess two potential mechanisms for this residual variability: phenotypic stochasticity in response and heritable resistance to treatment.



 We developed three agent-based mathematical models representing different potential scenarios for cellular responses to anticancer therapy: (1) all cells have an equal probability of survival under treatment (phenotypic stochasticity) and future division; (2) sensitive cells deterministically die under treatment while resistant cells survive and may divide to produce resistant offspring (heritable resistance); (3) sensitive subclones have a greater probability of death under treatment compared to resistant subclones (both mechanisms). We implemented these models in MATLAB, running simulated in vitro experiments consisting of large (106) or small (500 or 5000) populations of virtual cells under treatment. The final size of these virtual cell populations serves as the readout of the virtual experiments. To assess the effect of phenotypic stochasticity and heritable resistance on variation in experimental outcomes, we simulated these experiments 10,000 times and report the coefficient of variation (CV).



 Variability in response to treatment was low under most simulated conditions. In the first model simulating phenotypic stochasticity alone, high coefficients of variation in assay outcome were observed only when the probability of survival under treatment was less than 10%. In simulations of deterministic heritable survival under treatment, the coefficient of variation was generally low. However, simulations incorporating both mechanisms exhibited substantial variability in experimental outcome, consistent with the observed phenomenon.



 Stochastic cellular response to treatment or heritable binary resistance alone are insufficient explanations for the variation in in vitro experimental results. On the other hand, the high degree of variation in experimental outcomes is consistent with a combination of heritable resistance and phenotypic stochasticity. Our findings are consistent with a role for Darwinian evolution in the response to anticancer treatment for cancer cells in tissue culture.



 Alexander L. Young, Liam Noronha, Madison Stoddard, T Ryan. Gregory, Arijit Chakravarty. Heritable resistance and phenotypic stochasticity are likely drivers of irreproducibility in experimental determination of in vitro potency of anticancer agents [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C073.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a885f4ecedc46db5c7868acd715fccb8dde4f83" target='_blank'>
              Abstract C073: Heritable resistance and phenotypic stochasticity are likely drivers of irreproducibility in experimental determination of
 in vitro
 potency of anticancer agents
              </a>
            </td>
          <td>
            Alexander L. Young, Liam Noronha, M. Stoddard, T. R. Gregory, Arijit Chakravarty
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Introduction Acute myeloid leukaemia (AML) is a potentially fatal haematological malignancy characterized by the uncontrolled growth of leukemic cells (myeloid blasts), causing bone marrow failure. Despite recent advances in targeted therapeutics, high mortality rates persist across all age groups particularly the patients above the age of 60 years, emphasizing the need for novel and less toxic therapeutic strategies.
 KAT2A is a histone acetyltransferase involved in transcriptional regulation as part of two multisubunit complexes: Spt-Ada-Gcn5 acetyltransferase (SAGA) and Ada-Two-A-containing (ATAC), modulating lineage specificity and metabolic/cell-cycle pathways respectively. KAT2A has been shown to be upregulated in AML, where it plays a pivotal role in leukemic stem cell maintenance and chemotherapy resistance.
 Pharmacological inhibition of KAT2A has been shown to trigger differentiation in AML blasts while largely sparing normal haematopoietic progenitors, indicating that KAT2A blockade is a mechanistically selective therapeutic approach. We investigated the efficacy and mechanism of action of a novel proteolysis-targeting chimera (PROTAC) degrader; MDI0117218 which targets KAT2A as a differentiation-inducing therapeutic strategy in primary AML blasts.
 Methods AML cell lines (MOLM-13, KG1a, OCI-AML3, NB4, HL60) and a large cohort (n=50) of primary AML samples were assessed across a range of assays including: ATP-based proliferation assays (Cell-titre Glo), cytospin morphology, PKH26 cell division tracking and flow cytometric evaluation of differentiation-associated surface markers. KAT2A degradation and subsequent acetylation reduction was monitored by Western blot and intracellular flow for sub-population analysis. Long-term co-culture efficacy with HS5 stroma and colony assays were established to evaluate PROTAC effects on drug resistant adherent and putative stem cell fractions. Synergistic interaction with standard of care agents Cytarabine, Venetoclax and Azacitidine were assessed in combination and pretreatment-washout experiments.
 Results MDI0117218 showed nanomolar efficacy in AML cell lines (MOLM13 and OCI-AML3) and 48% of primary AML samples (n=50, Average EC50 33nM +/-324nM in sensitive primary AML samples, Ave 6.6uM +/- 1.4uM in resistant primary AML samples). This effect was observed across both differentiated and undifferentiated FAB types of AML.
 Western blotting in primary AML samples confirmed potent and rapid target engagement, with complete and sustained KAT2A degradation observed from 5nM within 30 minutes of incubation. Intracellular flow cytometry confirmed uniform reduction of KAT2A across blast sub-populations in sensitive patient samples compared to those with resistant EC50s. H3K9 acetylation reduction also correlated with KAT2A knockdown in sensitive samples providing potential clinical biomarker readouts. Assessment of MDI0117218 mechanism of action revealed variable cytostatic and differentiation responses with induction of CD11b +/- terminal CD14 differentiation in responders accompanied by cell cycle arrest without apoptotic induction consistent with myeloid differentiation which was confirmed by morphological analysis. Drug efficacy and differentiation was sustained in longer-term stromal co-culture and Colony forming assays, with significant reduction in all blast sub-populations including stromal adherent drug-resistant blasts and colony forming units (p<0.02, n=10). Combination drug assays in ex vivo primary AML blasts showed moderate to additive synergistic interaction between MDI0117218 and cytarabine (Average CI at ED50 0.9 +/- 0.85) and good synergy with Venetoclax (Average CI at ED50 of 0.25 +/- 0.09) across a range of doses. Pre-treatment and sequential washout protocols confirmed synergy to standard agents highlighting the rapid and sustained knockdown effects of MDI0117218.
 Conclusion MDI0117218 is an effective and rapid KAT2A degrader across multiple models of cell line and primary AML. Differentiation and anti-proliferative effects were observed across a wide range of differentiated and non-differentiated AML patients, highlighting the potential for a broad acting agent with potential for combination with both Cytarabine and Venetoclax; two mainstays of treatment for transplant-ineligible or relapsed AML. Taken together with favourable pre-clinical toxicology profiles, these data support the rationale for this agent as a strong candidate for early-phase clinical trials.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/733271b65f73cc3e215c90487cd2163804c35ba1" target='_blank'>
              Investigating KAT2A protac therapy for targeting leukemic blast differentiation in acute myeloid leukaemia.
              </a>
            </td>
          <td>
            Agisilaos Zerdelis, Claudia Chriches, Loren Waters, Laia Sadeghi, Darren Le Grand, Simon Ward, S. Knapper, J. Zabkiewicz
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 The advent of cancer immunotherapy, particularly immune checkpoint inhibitors, has revolutionised the treatment landscape for many malignancies. However, a significant proportion of patients exhibit primary, adaptive, or acquired resistance to these therapies, limiting their broad clinical benefit. Tumour microenvironment (TME), a complex and dynamic ecosystem of cancer cells, immune cells, stromal cells and extracellular matrix, has emerged as a critical determinant of immunotherapy response and a major driver of resistance.



 This review provides a comprehensive overview of the intricate mechanisms by which the TME fosters an immunosuppressive milieu, thereby impeding effective anti‐tumour immunity. We delineate the key cellular and molecular components of the TME that contribute to immunotherapy resistance, including tumour‐associated macrophages, myeloid‐derived suppressor cells, regulatory T cells, and cancer‐associated fibroblasts. We then discuss the diverse and innovative strategies being developed to reprogram the TME and overcome these resistance mechanisms. These emerging approaches include targeting immunosuppressive cell populations, modulating tumour metabolism, epigenetic reprogramming, normalising the tumour vasculature and employing engineered cell therapies such as chimeric antigen receptor T cells and oncolytic viruses. The potential of combination therapies to synergistically dismantle the immunosuppressive TME is also explored.



 Reprogramming the TME represents a paradigm shift in cancer treatment, moving beyond direct tumour cell targeting to modulate the entire tumour ecosystem. While significant challenges remain, including TME heterogeneity and the need for predictive biomarkers, the strategies discussed herein hold immense promise for overcoming immunotherapy resistance and improving patient outcomes. A deeper understanding of the complex interplay between the tumour and its microenvironment will be crucial for the development of next‐generation, personalised immunotherapeutic strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab5e861ba23ee597d8c992f1eb0a4c50564cbedf" target='_blank'>
              Reprogramming the tumour microenvironment: Emerging strategies to overcome immunotherapy resistance
              </a>
            </td>
          <td>
            Mohammed Elmujtba Adam Essa, Shahd Elbadri, Abdelkareem A. Ahmed, Hamid Noori
          </td>
          <td>2025-11-09</td>
          <td>Clinical and Translational Discovery</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="


 Phenotypic plasticity is a recognized mechanism of therapeutic resistance in prostate cancer (PCa), however current knowledge of driver mechanisms and therapeutic interventions are limited. Using genetically engineered mouse models (GEMMs) devoid of Pten and Rb1, we previously demonstrated the chromatin reprogramming factor enhancer of zeste homolog 2 (EZH2) as an important regulator of alternative transcription programs promoting phenotypic plasticity. Here, using a multi-omics approach we find that EZH2 regulates a multilineage cell state dependent on the activation of translation and regulation of RNA maturation. Combined chemical inhibition of EZH2 and PI3K/mTORC1 resulted in superior anti-tumor activity in murine and human phenotypic plastic models and was most significant when this combination was used with castration. Moreover, we elucidated that the regulation of cellular state transition by EZH2 inhibition requires activation of the RNA degrader, Tristetraprolin (TTP). Together, these data indicate phenotypic plasticity dependence on coordination between EZH2, TTP and mTORC1 signaling that represent novel therapeutic dependencies for this lethal PCa phenotype.



 Beatriz German, Leigh Ellis. EZH2 regulates cell identity in neuroendocrine-like prostate cancer by driving alternate transcription via activation of translation and RNA processing regulation [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C033.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e13a2ff65980383cdcce076c99675421b3f51723" target='_blank'>
              Abstract C033: EZH2 regulates cell identity in neuroendocrine-like prostate cancer by driving alternate transcription via activation of translation and RNA processing regulation
              </a>
            </td>
          <td>
            Beatriz German, Leigh Ellis
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Genetic and epigenetic processes are intertwined in a complex manner in order to enhance the development of breast cancer by regulating the fundamental cellular processes including DNA repair, proliferation, differentiation and metastasis. This review provides an overview of the findings of animal model experiments to explain the causes of the significant changes in genetic aspect including RAD51C mutations and dys-regulation of the BAF chromatin remodeling complex and epigenetic alterations, such as DNA methylation, histone modification, and microRNA dys-regulation. The animal models are where the researcher can study the mechanism by which these changes of the molecules cause tumorigenesis, stimulate tumor growth and metastatic potential. The data received in the course of such studies denotes the dynamism and reversibility of the epigenetic processes which may be utilized in treating diseases by selectively regulating the expression of the said molecules with an aim of modifying DNA methylation, histone-modifying enzymes, and miRNAs. Integration of genetic and epigenetic understanding will not only enhance our understanding of breast cancer biology, but also design the precision therapies, and also expose potential biomarkers that can be utilized to identify and diagnose the disease at the earliest phase. In total, the significance of the animal-based research in the context of finding mechanistic information and design interventions to limit the occurrence of breast cancer is highlighted in this review as a foundation of the subsequent use of the translation research and clinical interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/615bf75efa81e4b14beb7b6d9c561e55199ae645" target='_blank'>
              Genetic and Epigenetic Factors in Breast Cancer Progression
              </a>
            </td>
          <td>
            Akash Anil Podutwar, Purva Chandorkar, Swati Changdeo Jagdale
          </td>
          <td>2025-10-18</td>
          <td>Journal of Pharmacology, Genetics and Molecular Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Recent breakthroughs in genome technologies such Paired-Seq and Pair-Tag can jointly profile transcriptome and chromatin accessibility or histone modifications in single cells, and these methods have been used to study cell-type specific regulatory networks in normal human brains. However this approach has not yet been applied to any complex disease or primary tumor. Here, for the first time, we performed single-nucleus paired-tag characterization of freshly resected tissues from three recurrent glioblastoma (GBM) patients, including tumor core and adjacent brain specimens, to understand the epigenetic mechanisms that facilitate GBM cell aggressiveness and evolution. Clustering of over 25,000 cells across modalities identified distinctive normal and neoplastic populations, the latter of which exhibited clonal copy number amplifications in established oncogenes such as EGFR, MDM2, and PDGFRA. Cell states were sharply defined by unique enhancer profiles, while repressive regions existed in a broader continuum with overlap of glial, neuronal, and tumor populations. Detection of cell-type specific H3K27ac and H3K27me3 peaks revealed lineage restricted cis regulatory elements (CRE) that strongly correlated with gene expression patterns, including potent enhancers and silencers associated with cell cycle regulation. Co-accessibility analyses of CREs with nearby promotors further elucidated transcriptional mediators, such as tumor-specific enhancers of the glioma invasion gene HMGA2. Stratification of GBM cells into previously established transcriptional states (MES, NPC, OPC, AC) identified key epigenetic facilitators associated with subgroup-specific expression patterns, as well as downstream evolutionary trajectories that drive clonal selection and therapy resistance. Our results leverage granular co-profiling of histone modifications, gene expression, and chromatin organization to derive novel insights into recurrent GBM biology, identifying new therapeutic vulnerabilities in these molecularly divergent lesions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b188a0338485b43a319b25ce06bce2564a32bf6a" target='_blank'>
              EPCO-25. SINGLE-NUCLEUS CO-PROFILING OF TRANSCRIPTOME, HISTONE MODIFICATIONS, AND CHROMATIN ACCESSIBILITY REVEALS DYNAMIC EVOLUTION OF RECURRENT GLIOBLASTOMA
              </a>
            </td>
          <td>
            Mark W. Youngblood, Shaojun Yu, Yoshii Ma, K. Habashy, A. Gould, Li Chen, Ye Hou, Yu Luan, Yihao Fu, A. Sonabend, Feng Yue
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="


 Multiple Myeloma (MM) is a hematological neoplasm characterized by the clonal expansion of malignant plasma cells (PCs) within the bone marrow. Despite clinical advances, the majority of patients eventually experience disease relapse. Therefore, understanding the molecular basis of MM is essential for developing new therapeutic strategies. Epitranscriptomic alterations, such as post-transcriptional RNA modifications, have emerged as key mechanisms in tumor development. However, their role in MM remains poorly understood. In this study, we investigated the role of N6-methyladenosine (m6A), the most prevalent RNA modification, and its regulatory genes in the prognosis and development of MM.
 We first measured global m6A levels in purified PCs from the bone marrow samples of individuals with premalignant stages such as MGUS and SMM, MM patients and healthy donors using ELISA. Interestingly, m6A levels were significantly increased from early disease stages through to MM. This increase correlated with the overexpression of 18 out of 20 m6A regulatory genes in MM samples, as observed in RNA-seq data from PCs of MM patients compared to healthy donors. Epigenomic analyses (ATAC-seq and H3K27ac ChIP-seq) confirmed increased chromatin accessibility and activation in the promoter region of m6A regulatory genes in MM PCs, demonstrating that these genes were epigenetically deregulated in MM. To assess the clinical relevance of m6A regulators we analyzed RNA-seq and clinical data from 642 MM patients from the CoMMpass cohort. Patients were stratified into high or low expression groups for each m6A gene individually, and Kaplan-Meier survival analyses were performed. High expression of most m6A genes was significantly associated with shorter progression-free survival (PFS) and overall survival (OS). Given the strong coexpression among m6A-related genes, we developed a composite expression signature referred to as the Epitranscriptomic Signature of m6A Epigenetic Genes (ESMEG) to better stratify MM patients. MM patients with high ESMEG scores exhibited significantly worse PFS and OS, independently of the presence or absence of high-risk genetic alterations in MM. Furthermore, gene ontology (GO) and gene set enrichment analysis (GSEA) revealed that high ESMEG scores were associated with enrichment of proliferation-related pathways, supporting a functional role for m6A regulator genes in promoting MM aggressiveness.
 Among m6A regulators, VIRMA was identified as essential in MM based on data from the DepMap-Achilles' project. CRISPR-Cas9-mediated VIRMA inhibition in MM.1S-Cas9 and KMS-11-Cas9 cells reduced cell viability and global m6A levels in vitro. Notably, in vivo studies revealed a marked impairment of MM tumor growth, demonstrating its essentiality in MM. Direct RNA nanopore sequencing revealed a general reduction of m6A marks in VIRMA-knockdown conditions, as well as a reduced m6A stoichiometry. Importantly, levels of other RNA modifications, such as pseudouridine, 5-methylcytosine, inosine and 2'-O-methylated nucleotides, remained unchanged, supporting the specificity of VIRMA's effect on m6A modification. From all detected m6A sites, 609 of them had a significantly reduced m6A representativity across 475 genes after VIRMA knockdown. GO analysis showed that these genes were related to cell cycle, apoptosis, and mitochondrial functions. Specifically, functional studies validated that VIRMA inhibition led to G1 cell cycle arrest (flow cytometry), reduced NF-κB signaling (Western blot, ELISA, flow cytometry), and impaired mitochondrial respiration (Seahorse), supporting its key role in MM cell survival. Finally, to evaluate the potential of VIRMA as a therapeutic target, we developed lipid nanoparticles (LNPs) encapsulating a VIRMA-targeting siRNA (LNP-siVIRMA) using microfluidic technology. Treatment of MM cell lines with LNP-siVIRMA significantly inhibited MM cell proliferation in vitro. Importantly, in vivo administration led to a pronounced reduction in tumor growth, highlighting the therapeutic potential of targeting VIRMA in MM.In summary, our findings indicate that increased m6A and upregulation of its regulatory genes are associated with poor prognosis in MM. VIRMA is critical for MM cell survival through regulation of the cell cycle, apoptosis, and mitochondrial function. Its inhibition via LNP-siVIRMA represents a promising epitranscriptomic-based therapeutic strategy to improve the response of MM patients.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b338f181df68722338fca93d94080d5f1e69624c" target='_blank'>
              Virma epitranscriptomically regulates cell proliferation, apoptosis, and mitochondrial function via m6A in multiple myeloma
              </a>
            </td>
          <td>
            Estibaliz Urizar Compains, Naroa Barrena, Theodore Maximilian Nelson, L. Gárate, Estibaliz Miranda Elizalde, Juan Antonio Romero Torrecilla, Luis Vitores Valcárcel García, M. Kulis, Elisa Garbayo Atienza, Maria José Blanco Prieto, P. Rodríguez-Otero, J. Martín-Subero, Christopher Mason, Edurne San José Enériz, Felipe Prósper, X. Agirre
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Disruptor of Telomeric Silencing 1-Like (DOT1L) has emerged as a critical epigenetic regulator in cancer, primarily because of its role as the sole histone methyltransferase responsible for histone H3 at lysine 79 methylation. This modification affects transcriptional activation, DNA repair, and cell cycle progression. Its dysregulation is associated with both hematological and solid tumors. In Mixed-lineage leukemia (MLL)-rearranged leukemia, DOT1L maintains aberrant gene expression patterns at loci such as HOXA and MEIS1, supporting leukemic stem cell survival and driving oncogenesis. In solid tumors, DOT1L influences diverse processes, including epithelial-mesenchymal transition, angiogenesis, and cell cycle regulation, contributing to tumor growth and metastasis. Therapeutic strategies targeting DOT1L using inhibitors, such as EPZ-5676, have shown promise in preclinical and clinical studies, highlighting their potential as versatile targets for precision oncology. This review summarizes the recent findings on DOT1L’s involvement in cancer development and its potential as a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/652c9fcc6ab6b96caf0c41342c1311e10db0c889" target='_blank'>
              DOT1L as a Therapeutic Target: Insights into Epigenetic Regulation and Cancer Treatment
              </a>
            </td>
          <td>
            Hee-Joo Choi, Minh Tuan Nguyen, Boram Kim, Seung Hwan Kim, Jaehee Kim, Jeong‐Yeon Lee, Mikyung Park
          </td>
          <td>2025-10-31</td>
          <td>Biomolecules & Therapeutics</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="


 Deletion of the long arm of chromosome 9, del(9q), is a recurrent cytogenetic abnormality in AML and encompasses the HNRNPK locus at 9q21.32. hnRNP K is a multifunctional RNA- and DNA-binding protein involved in chromatin remodeling, transcriptional regulation, and RNA metabolism. Monoallelic deletions or loss-of-function mutations in HNRNPK, collectively observed in ~5–6% of AML cases, directly contribute to leukemogenesis.
 We previously showed that Hnrnpk haploinsufficiency disrupts p53 and myeloid lineage-specifying pathways (e.g., C/EBPα and PU.1), impairing differentiation and expanding myeloid progenitors—recapitulating del(9q) AML features. While elucidating these pathways has aided our understanding of AML biology, they remain largely undruggable. Therefore, identifying alternative, targetable vulnerabilities in HNRNPK-deficient AML is of significant clinical interest. To this end, our current studies reveal that HNRNPK-deficient cells exhibit marked sensitivity to translational inhibition, indicating a synthetic lethal interaction driven by impaired protein synthesis.
 To delineate the molecular basis of this vulnerability, we used RIP-seq, proteomics (RPPA, IP-MS), and translational profiling. Here, we identified that HNRNPK deficiency caused dysregulated mTOR signaling, aberrant ribosomal protein S6 phosphorylation, and altered expression of key translation initiation/elongation factors. Critically, rRNA processing was impaired, disrupting 47S precursor maturation and leading to defective 40S and 60S subunit assembly and reduced active polysomes—suggesting that del(9q) AML may represent a novel ribosomopathy.
 Importantly, complete HNRNPK loss is embryonically lethal, and large-scale CRISPR datasets identify it as essential, underscoring its dosage sensitivity and potential for synthetic lethal therapeutic strategies. To uncover synthetic lethal interactions, we performed a genome-wide CRISPR-Cas9 dropout screen in HNRNPK-deficient cells, identifying dependencies in ribosome biogenesis, translation initiation, and metabolite synthesis as potential vulnerabilities. We are currently using primary cells harboring HNRNPK deletions and genetically engineered models with reduced hnRNP K expression to identify therapies that induce cell death in an hnRNP K-dependent manner.
 Collectively, our findings establish hnRNP K as a dosage-sensitive tumor suppressor and suggest del(9q) AML as a novel ribosomopathy. Moreover, the synthetic vulnerabilities identified in our study provide a strong rationale for developing targeted therapies that exploit HNRNPK haploinsufficiency and lay the foundation for the clinical development of agents that induce synthetic lethality in this subset of patients with AML.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a1f4b3ab63f8b9803758eab2816344752deed3f" target='_blank'>
              Hnrnpk haploinsufficiency in del(9q) Acute Myeloid Leukemia reveals targetable vulnerabilities
              </a>
            </td>
          <td>
            S. Post, Oscar Benitez, Ashley G Rodriguez, Marisa Aitken, P. Malaney, Miguel Gallardo, Hussein A. Abbas
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="


 Introduction: While sporadic in developed countries, Burkitt lymphoma (BL) is endemic in resource-limited regions. Given the one-year survival rates of 61% (Mezger, The Oncologist, 2023), BL patients in sub-Saharan Africa could benefit from improved diagnostic methods, particularly those that facilitate the evaluation of new targeted agents. Recent work from our group has found that BL can now be divided into molecular subgroups based on mutations (Thomas, Blood, 2023) and into epitypes using DNA methylation patterns (Thomas, BCD, 2025).
 Nanopore long-read sequencing is becoming a robust and efficient approach to rapidly profile patient DNA samples for structural variants (SVs), simple somatic mutations (SSMs) and base modifications. Using the adaptive sampling (AS) technique, sequencing can be focused on regions of interest (ROIs). This technique has shown promise in application to germline genetics (Vermuelen, Nature, 2023) but its potential application to somatic conditions remains largely unexplored. We are evaluating the utility of this approach to obtain genetic and epigenetic data from relevant genomic loci to facilitate differential diagnosis and accurate assignment of BL molecular subgroup and epitype. We aim to create a diagnostic assay using AS coupled with an automated bioinformatics pipeline that classifies and subtypes BL to inform treatment and improve outcomes in resource-limited regions.
 Methods: To establish ROIs, we selected driver mutations and features identified as relevant to differentiating BL from other B-cell lymphomas (Coyle, Blood Adv., 2025) as well as distinguishing epitype (Thomas, BCD, 2025) and molecular subgroup (Thomas, Blood, 2023). These include 179 lymphoma genes, 3 oncogenic SVs, 127 regions commonly affected by aberrant somatic hypermutation, the EBV genome, the IGH, IGK and IGL loci, collectively 31.5 megabases.
 Initially, we simulated AS data for 9 BL genomes by extracting reads corresponding to ROIs with samtools from whole genome sequencing data (PromethION). To generate real AS data, we prepared 32 µL of DNA library using the Raji BL cell line. Library preparation with 1.6 µg of gDNA was performed according to the Native Barcoding Kit 24 V14 (SQK-NBD114.24) protocol. The library was sequenced on a single PromethION flow cell (R10.4.1 chemistry) for 25 hours in enrichment mode using our ROI panel. Enrichment, coverage and relevant AS metrics were assessed using samtools, bedtools, Mosdepth, custom R scripts and MinKNOW output files.
 To evaluate epitype classification, CpG methylation status was inferred from simulated AS data with Modkit, analyzed with custom R scripts and compared to ground truth labels. To explore 5hmC modification and hypomethylation patterns, TET2 expression was determined from RNAseq data (n=126) and analyzed using R scripts. To examine mutation differences between epitypes, SLMS-3 (Thomas, Blood, 2023) variant calls within ROIs from Illumina data for the same 9 BL genomes were used; these were normalized and compared using custom R scripts.
 Results: The top 6,000 5mC positions with the highest variance were able to accurately recapitulate patient epitypes. These positions had an average CpG methylation level of 49% in HyperBL and 36% in HypoBL. HyperBL samples showed more mutations than HypoBL, consistent with the ground truth. TET2 expression was significantly greater in HypoBL, suggesting more demethylation and higher transient 5hmC modification during demethylation.
 Regarding AS metrics, 4.7% of fragments present were basecalled. A total of 1.7 million fragments were sequenced and 23.2% overlapped a target ROI. Mean read lengths for on and off-target fragments were 7,065 and 690 bases, respectively, and the range of quality scores for these reads was comparable. Average coverage across ROIs was 67.9, with an enrichment factor of 79.3.
 Conclusions: Preliminary analysis of real and simulated BL AS data suggests that a limited feature set can support successful subgroup classification concordant with Illumina ground truths. Evaluation of ONT-focused tools to resolve SSMs and SVs is ongoing. Our preliminary results suggest AS has the potential to serve as a portable, high-throughput diagnostic assay to inform treatment and improve outcomes for endemic BL patients in resource-limited regions.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/feeefa7f2c0a96789283ed535ab15b5f0bb7ac0f" target='_blank'>
              Evaluating targeted long-read sequencing as a diagnostic tool for burkitt lymphoma
              </a>
            </td>
          <td>
            Giuliano Banco, K. Dreval, L. Hilton, David Scott, Michael J. Trimble, Amy H. Y. Lee, K. Coyle, Ryan D Morin
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Traditional pooled genetic dependency screens, such as DepMap, have proven powerful for identifying essential genes in cancer cell lines. However, these pooled formats are prone to the bystander effect, where secreted factors from wild-type cells support the survival of knockout (KO) cells within the same culture, potentially masking dependencies on specific secreted proteins. To address this limitation, we performed a well-by-well CRISPR screen targeting genes encoding secreted factors in the pancreatic ductal adenocarcinoma (PDAC) cell line HPAFII. This arrayed format eliminates cross-feeding and enables precise assessment of cell-intrinsic dependencies on targeted proteins and could thus expose hidden vulnerabilities. CellTiter-Glo (CTG) viability assays were used to assess gene lethality following target gene KOs. Furthermore, RNA sequencing verified effective disruption of target genes, establishing the specificity and reliability of the screening workflow. We observed strong growth inhibition in EFNA1 (Ephrin-A1) KO samples compared to lipofectamine-treated controls. However, this inhibitory effect was missed and underrepresented in the conventional pooled format. EFNA1 is a ligand for EPH receptors, involved in cell-cell communication, migration, and proliferation signaling. Differential gene expression analysis comparing EFNA1 KO to lipofectamine-treated controls revealed enriched pathways of oxidative phosphorylation, E2F targets, MYC targets, and DNA repair in EFNA1 KO samples. This result supports our observation of lethality in EFNA1 KO samples. It suggests that EFNA1 contributes to cell-cell communication and is critical for HPAFII cell survival. Its KO triggers stress pathways, including E2F, MYC, OXPHOS, and DNA repair, and acts as a suppressor of excessive proliferative programs. In conclusion, we identified EFNA1 as a critical non-cell-autonomous essential gene through an arrayed CRISPR screen, with its loss showing lethality to HPAFII cells exclusively in isolated settings, likely as a result of disrupted cell-cell interactions. Defining such dependencies may facilitate the discovery of directly druggable and more tumor-selective therapeutic strategies.



 Xingyi Shi, David Ruddy, Millicent Gabriel, Michelle Piquet, Kris Muskiewicz, Joel Wagner, Joshua Korn, Cory Johannessen. Arrayed CRISPR Screening Reveals EFNA1 as a Non-Cell-Autonomous Essential Gene in Pancreatic Cancer Cells [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C035.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddd6ce939e1194e9cb9b823d2bb2e7b39fb1e35f" target='_blank'>
              Abstract C035: Arrayed CRISPR Screening Reveals EFNA1 as a Non-Cell-Autonomous Essential Gene in Pancreatic Cancer Cells
              </a>
            </td>
          <td>
            Xingyi Shi, David A Ruddy, Millicent Gabriel, M. Piquet, Kris Muskiewicz, Joel P Wagner, Joshua Korn, Cory Johannessen
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Genomic stability in human cells relies on intact p53 and pRb cell cycle checkpoints. Their suppression can lead to telomere dysfunction (TD) and genomic instability, which is a characteristic feature of many cancers, including diffuse glioma. To investigate the genomic evolution of otherwise normal astrocytes undergoing TD, we transduced normal human astrocyte (NHA) cells with retroviruses containing the viral HPV-18 E6/E7 oncoproteins and cultured them over extended periods of time. Immunofluorescence microscopy was used to detect mitotic abnormalities and demonstrated anaphase bridges in a sizable proportion of NHA-E6E7 cells (n = 47/99, 47.5%) when compared to un-transduced NHA cells (n = 0/25, 0%), confirming ongoing TD. Using multiplexed fluorescence in situ hybridization (M-FISH) to characterize large-scale chromosomal abnormalities, we observed t(2;13)(q32.2;q21.1) in n = 20/30 (66.7%) of transduced cells from a bulk NHA-E6E7 clone. Furthermore, one copy of chromosome 13 was either truncated or deleted in all cells (n = 30/30, 100%). Surprisingly, the acrocentric chromosomes 13, 14, 15, 21 and 22 appeared to be especially prone to cytogenetic abnormalities, with the per chromosome average aberration counts being 2.83 folds higher in acrocentric chromosomes when comparing to other chromosomes. This observation potentially explains the central nuclear clustering of the short arms of the acrocentric chromosomes into the nucleolus. Unexpectedly, the frequency of t(2;13)(q32.2;q21.1) translocations decreased in later passages, suggesting that this event was detrimental to cell survival. Consistent with that observation, the FISH-confirmed t(2;13)(q32.2;q21.1) abnormality was not strongly supported by long-read DNA sequencing, indicating that it is a large-scale transient event occurring sporadically in individual cells. These findings underscore the critical role of p53 and pRb-mediated pathways in maintaining genomic integrity and suggest that recurrent acrocentric translocations and mitotic abnormalities are key phenotypic manifestations of telomere dysfunction and genomic instability in early glioma genesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/124aaf2b55688148ff639618cb63d053ddc343a2" target='_blank'>
              CSIG-32. RECURRENT ACROCENTRIC TRANSLOCATIONS AND ANAPHASE BRIDGES ARE AN EARLY PHENOTYPE OF PRE-CANCEROUS ASTROCYTES WITH DYSFUNCTIONAL TELOMERES
              </a>
            </td>
          <td>
            Ogechukwu Mbegbu, Yue Hao, Noelle H. Fukushima, Ava Bryan, Tianpeng Zhang, P. Cosper, Floris P Barthel
          </td>
          <td>2025-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Introduction: Protein truncating somatic ASXL1 mutations in CMML are associated with high-risk features and poor outcomes. We have previously demonstrated that ASXL1-mutant (mt) CMML is characterized by permissive chromatin states and overexpression of leukemia stemness genes (Binder et al. Nat Commun; 2022). DLK1 (delta-like homolog 1) is a pseudo-NOTCH like ligand that is maternally imprinted and paternally expressed and is physiologically not expressed beyond embryogenesis. Aberrant DLK1 expression has been associated with poor outcomes in solid tumors and we have shown differential overexpression in ASXL1mt versus wild-type (wt) CMML. Both the promoter and the last exon of DLK1 contain CpG islands that play a role in regulating DLK1. The CpG island within the fifth exon of DLK1, known as the DLK1-differentially methylated region (DMR) [chr14:100,726,892-100,738,224] is completely unmethylated on the maternal allele and partially methylated on the paternal allele. Previous work (Developmental Cell; 2021) has also identified a DLK1 imprinting control region (ICR) known as the IG-DMR [chr14:100,811,001–100,811,037]. DNA methylation state of the IG-DMR has been shown to impact the expression of MEG3 mRNA (lncRNA) which functions as a cofactor to direct PRC2 to the DLK1 promoter, suppressing DLK1 expression. Since the regulation of DLK1 in ASXL1mt CMML is not well understood, we conducted this multiomic analyses to define epigenetic mechanisms of DLK1 overexpression in ASXL1mt CMML.
 Methods: After IRB approval, we performed bulk RNA-seq, ChIP-seq (H3K4me1/me3, H3K27ac, H3K27me3) and Illumina MethylationEPIC array on bone marrow (BM) mononuclear cells of 40 CMML patients (19 ASXL1mt, 21 wt). scRNA-seq was conducted on 5 ASXL1mt and 3 wt cases. All libraries were sequenced on an Illumina NovaSeq instrument. We assessed these samples for differential gene expression (RNA-seq) and changes in histone modifications (ChIP-seq) between ASXL1mt and wt CMML, specifically at the DLK1-DMR and IG-DMRregulatory loci. We conducted ddPCR (digital droplet PCR) to determine allelic status of DLK1 in ASXL1mtand wt cases. Global and allele/sequence specific DNA methylation changes were determined using Biomodal's duet EvoC methodology (6 base sequencing, including 5mC and 5hmC) in 5 ASXL1mt and 5 wt CMML cases.
 Results: We validated the overexpression of DLK1 in 19 ASXL1mt versus 21 wt (log2 fold change +4.85, FDR 4.67e-13) CMML BM samples. scRNA-seq on 5 ASXL1mt versus 3 wt samples demonstrated that DLK1 is only expressed in mutant cases, with the expression being limited to HSPCs. ChIP-seq analysis at the DLK1 regulatory loci validated (a) reduced occupancy of H3K27me3, correlating with ASXL1mt associated disruption of the PRC2-complex, and (b) increased co-occupancy of H3K4me1 and H3K27ac in mt cases, validating a permissive chromatin state in ASXL1mt cases at the DLK1 regulatory regions.
 Through correlation analysis of RNA-seq and ChIP-seq, we defined the promoter for DLK1, characterized by high Pearson correlation coefficients (R > 0.5) and enrichment of H3K27ac and H3K4me3. Differential methylation analysis identified global hypomethylation around the DLK1-DMR and promoter regions. We observed a mean reduction of ~40% in methylation status at the DLK1 promoter in ASXL1mt versus wt cases, suggesting a loss of epigenetic silencing at this locus. This was corroborated by significant enrichment of 5hmC at the fifth exon of DLK1 indicating active DNA demethylation. Conversely, we observed a gain in methylation at the IG-DMR, suggesting loss of imprinting control through MEG3 silencing and DLK1 overexpression.
 To further determine the impact of imprinting deregulation on overexpression of DLK1 in ASXL1mt CMML, allele-specific methylation was profiled using the Biomodal whole genome methylation data. This demonstrated that the methylation patterns in ASXL1mt CMML cases were significantly deregulated at the DLK1-DMR, with >30% loss of methylation compared to GRCh38 reference alleles. Furthermore, ddPCR was able to confirm bi-allelic expression of DLK1 in ASXL1mt CMML, suggesting that the observed methylation dysregulation leads to loss of imprinting of the maternal allele.
 Conclusion: In summary, we demonstrate that ASXL1 mutations in CMML disrupt epigenetic pathways, promoting DNA demethylation, resulting in permissive chromatin states and an aberrant and oncogenic re-expression of maternal DLK1, due to loss of imprinting control.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f07bbcc3de41679d101785feeca89e9a4988bc05" target='_blank'>
              Somatic ASXL1 mutations in chronic myelomonocytic leukemia result in loss of imprinting control and overexpression of DLK1 through deregulation of DNA methylation
              </a>
            </td>
          <td>
            Pankaj Pradeep, Jenna A. Fernandez, Abhishek Mangaonkar, Nickolas Steinauer, M. Binder, Huihuang Yan, C. Finke, Zhifu Sun, T. Lasho, Mrinal Patnaik
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="


 INTRODUCTION: Anemia is the main clinical manifestation of low-risk myelodysplastic syndromes (MDS), reflecting ineffective erythropoiesis. Although therapies such as erythropoietin and luspatercept are available, their effects are often transient, highlighting the need for new therapeutic agents. In the past years, long non-coding RNAs (lncRNAs) have emerged as critical regulators of gene expression in hematopoiesis and cancer, yet their role in ineffective erythropoiesis is poorly defined. The aim of this study was to identify lncRNAs contributing to erythroid dysregulation using del(5q) MDS as a model.



 Single-cell RNA-seq (10x Genomics) was performed on CD34+ bone marrow cells from 4 MDS-del(5q) patients and 3 age-matched healthy donors. Preprocessing was done using CellRanger and Kallisto (hg38, GENCODE v37), followed by quality filtering via emptyDrops and scDblFinder. Cell types were annotated based on canonical marker expression.
 To identify erythroid-relevant lncRNAs, ELATUS pipeline using recommended thresholds (ratio >3, CR >150, SI >0.1) was applied across the erythroid stages – megakaryocyte-erythroid progenitors (MEP), early (EE) and late erythroid (LE).
 Differential expression was assessed on pseudobulk data via DESeq2 (log2FC >1, padj <0.05) Pseudotime trajectories were inferred with Monocle3, and lncRNA expression dynamics were modeled applying generalized additive models (GAM), using binarized cell expression (>0 counts) as input.
 The significance of dynamic differences between conditions was tested using ANOVA by comparing full (condition-interacting) vs null (no condition-interacting) GAM. Candidate lncRNAs were defined as those meeting all three criteria: ELATUS selection, differential expression, and altered dynamics. Gene ranking was performed using a Z-score combining scaled log2FC and -log10(padj-GAM) and expression dynamics of axis-related genes were examined.



 Among the 2,100 biologically relevant lncRNAs identified, 46 were differentially expressed between MDS and controls. Of these, 43 also displayed significantly altered expression dynamics, fulfilling all selection criteria. Literature review revealed that 10 of the 43 lncRNAs have cis-targets implicated in erythropoiesis. Among them, ENSG00000288934, an antisense (AS) transcript of PPP1R15B emerged as top candidate based on the previous prioritization score and dynamic profile.
 In healthy controls, PPP1R15B-AS showed a sharp, stage-specific pulse of expression during erythroid maturation, while PPP1R15B was concomitantly activated in a mirror-like pattern. In MDS, this lncRNA was downregulated, and PPP1R15B expression became flattened and temporally uncoupled from erythroid progression.
 In parallel, axis-related genes such as ATF4 showed a progressive increase during pseudotime, while pro-apoptotic effector CHOP (DDIT3) exhibited a marked late-stage peak coinciding with terminal maturation. This contrasts with the more restricted and temporally coordinated expression observed in healthy samples.



 In our dataset, downregulation of PPP1R15B-AS coincided with a temporal shift and loss of coordination in PPP1R15B expression, suggesting a role as repressor in fine-tuning gene activation during specific erythroid stages. While PPP1R15B is known to resolve integrated stress response by promoting eIF2a dephosphorylation and repressing effectors such as ATF4 and CHOP, we observed that both transcription factors remained overexpressed despite having increased levels of PPP1R15B. These findings point toward a dysregulation of the PPP1R15B-eiF2a axis in MDS, in which PPP1R15B-AS may contribute to fine-timing of stress deactivation.
 Altogether, these observations support a model in which ineffective erythropoiesis may not only result from aberrant expressions, but also from disruption of transcriptional dynamics mediated by lncRNAs. This altered expression of PPP1R15B-AS, despite its genomic location outside 5q deletion, may reflect broader downstream remodeling. Ongoing in vivo studies are developing to validate these findings.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76b69695216aa01aa2d8a0ecf9cc6bc511d5b9f0" target='_blank'>
              Single-cell transcriptomics reveal long non-coding RNA dysregulation in PPP1R15B-EIF2A axis impairing erythroid maturation in myelodysplastic syndromes
              </a>
            </td>
          <td>
            F. Beas, Enrique Goni, Aintzane Díaz-Mazkiaran, T. Ezponda, A. Alfonso-Piérola, A. Jerez, Felipe Prósper, David Valcárcel, Mikel Hernáez Arrazola
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="


 Most genetic variants that confer risk of complex immune-mediated diseases (IMD) affect gene regulation in specific cell types. Their target genes and focus cell types are often unknown, partially because some effects are hidden in untested cell states. B cells are drivers of IMDs such as lupus and multiple sclerosis. Yet, B cell activation states are underrepresented in functional genomics studies. In this study, we obtained B cells from a total of 23 healthy female donors. We stimulated B cells in vitro by targeting key pathways in six activation conditions such as the CD40, B cell receptor, TLR7, TLR9, and a cocktail that promotes differentiation into double negative 2 (DN2) B cells. We profiled B cell activation at different time points using RNA-seq, single-cell RNA-seq coupled with surface protein markers (CITE-seq), and ATAC-seq. We provide an in-depth characterization of how IMD-associated genes respond to stimuli and group into modules with distinct functions. Using single-cell data, we describe pathway-dependent B cell fates. Chromatin data reveal the transcription factors involved in pathway-dependent B cell activation. These open chromatin regions capture a significant proportion of the heritability of IMDs and elucidate IMD-associated variants with previously unknow function. These data are shared via an interactive browser that can be used to query the dynamics of gene regulation and B cell differentiation during activation by different stimuli.



 Supported by Lupus Research Alliance (LRA); Arthritis National Research Foundation (ANRF); Gilead; Join Biology Consortium (JBC)



 Computational and Systems Immunology (COMP)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7bfee869441bc2a7a90854f0e5e1d453ccb1fe55" target='_blank'>
              A multiomics resource of B cell activation helps decipher disease pathogenesis 4355
              </a>
            </td>
          <td>
            Vitor R.C. Aguiar, Marcella E. Franco, Daniela Fernandez-Salinas, Marcos Chiñas, Qian Xiao, Nicolaj S. Hackert, Mariasilvia Colantuoni, Nada Abdel Aziz, M. Todd, Brian Wauford, Alexandra Wactor, Vaishali Prahalad, Raquel Laza Briviesca, Roxane Darbousset, Qiang Wang, Emily Banasiak, Scott A Jenks, K. Cashman, Esther C. Zumaquero-Martinez, Zhu Zhu, Junning Case, Paloma Cejas, Miguel Gomez, M. Benamar, Pui Lee, Lauren A. Henderson, , Kevin Wei, Henry Long, Carl D. Langefeld, Ignacio Sanz, Jeffrey Sparks, E. Meidan, P. Nigrovic, Maria Gutierrez-Arcelus
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="There is an urgent need to find targeted agents for T-cell acute lymphoblastic leukemia (T-ALL). NOTCH1 is the most frequently mutated oncogene in T-ALL, but clinical trials showed that pan-Notch inhibitors caused dose-limiting toxicities. Thus, we shifted our focus to ETS1, which is one of the transcription factors that most frequently co-bind Notch-occupied regulatory elements in the T-ALL context. To identify the most essential enhancers, we performed a genome-wide CRISPR interference screen of the strongest ETS1-dependent regulatory elements. The #1-ranked element is located in an intron of AHI1 that interacts with the MYB promoter and is amplified with MYB in ~8.5% of T-ALL patients. Using mouse models, we showed that this enhancer promotes self-renewal of hematopoietic stem cells and T-cell leukemogenesis, maintains early T-cell precursors, and restrains myeloid expansion with aging. We named this enhancer the hematopoietic stem cell MYB enhancer (H-Me). The H-Me shows limited activity and function in committed T-cell progenitors but is accessed during leukemogenesis. In one T-ALL context, ETS1 binds the ETS motif in the H-Me to recruit cBAF to promote chromatin accessibility and activation. ETS1 or cBAF degraders impaired H-Me function. Thus, we identified a targetable stem cell element that is co-opted for T-cell transformation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b0d046a416194ebc6608468d656972dd5460330" target='_blank'>
              The hematopoietic stem cell MYB enhancer is essential and recurrently amplified during T-cell leukemogenesis.
              </a>
            </td>
          <td>
            Carea Mullin, Karena Lin, Elizabeth Choe, Cher Sha, Zeel Shukla, Koral Campbell, Anna C. McCarter, Annie Wang, Jannaldo Nieves-Salva, Sarah Khan, Theresa M. Keeley, Shannon Liang, Qing Wang, A. Melnick, Pearl Evans, Alexander C. Monovich, Ashwin Iyer, R. Kodgule, Yamei Deng, Felipe da Veiga Leprevost, Kelly R. Barnett, P. Pölönen, R. Khoriaty, Daniel Savic, D. Teachey, C. Mullighan, Marcin Cieslik, Alexey I. Nesvizhskii, Linda C. Samuelson, Morgan Jones, Qing Li, Russell J H Ryan, M. Chiang
          </td>
          <td>2025-10-23</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>108</td>
        </tr>

        <tr id="


 TET2 mutations cause clonal expansion of hematopoietic stem and progenitor cells (HSPCs) and chimeric antigen receptor (CAR)-expressing T cells, driving clonal hematopoiesis (CH) and CAR+ T cell lymphomas, respectively. TET2-mutated CH has been linked to hyperinflammation, but the mechanisms remain poorly understood.
 We developed human isogenic induced pluripotent stem cell (iPSC)-based models of the 3 main CH mutations – DNMT3A R882H, TET2 haploinsufficiency and ASXL1 truncations –and performed integrated transcriptome, chromatin accessibility and DNA methylation (whole genome bisulfite sequencing) analyses in CD34+ HSPCs. Expectedly, DNMT3A- and TET2-mutant HSPCs showed a global decrease and increase, respectively, in DNA methylation, compared to isogenic wild-type (WT) controls. Single-cell RNA-seq analyses and immunoblotting experiments, unexpectedly, showed cell-autonomous activation of the type I interferon (IFN) induction pathway and secretion of bioactive type I IFN by TET2-mutated HSPCs. Furthermore, genomic and functional experiments in iPSC- and cord blood (CB)- HSPCs with TET2 shRNA knockdown or CRISPR/Cas9-mediated knockout demonstrated that, while type I IFN suppresses the expansion of WT CD34+/CD38- cells in a dose-dependent manner, TET2-mutant HSPCs downregulate their downstream IFN response and are thus protected. Importantly, while TET2-deficient CD34+/CD38- cells outgrew co-cultured WT cells, blockade of type I IFN signaling by a IFN alpha and beta receptor subunit 1 (IFNAR1) antibody mitigated their selective advantage.
 To understand the mechanism, we searched for hypermethylated genes with concomitant loss of chromatin accessibility and gene expression specifically in TET2-mutant HSPCs. These analyses, followed by knockdown and knockout (KO) validation experiments, identified silencing of the E3 ubiquitin ligase tripartite motif containing 4 (TRIM4) gene as a putative effector of TET2 deficiency. Corroborating findings from scATAC-seq and TARGET-seq+ datasets of HSPCs from individuals with TET2-mutated CH, clonal cytopenia of undetermined significance (CCUS) and healthy donors, revealed decreased promoter accessibility and expression of TRIM4 in primary TET2-mutant HSPCs. Importantly, engineering TRIM4 deficiency by CRISPR in iPSC- and CB- HSPCs phenocopied TET2 deficiency phenotypes, including monocytic bias and cell-autonomous type I IFN induction. In addition, TRIM4-deficient iPSC- and CB- HSPCs outcompeted WT HSPCs in liquid culture and serial replating assays in semisolid media, recapitulating the clonal advantage of TET2-deficient HSPCs. Furthermore, ectopic TRIM4 re-expression in TET2-deficient HSPCs, rescued their clonal expansion both in vitro and in competitive repopulation assays in NSGS mice.
 While TRIM proteins have known pleiotropic roles in immune signaling, TRIM4 is not well-characterized and lacks a mouse homolog. To identify substrates of TRIM4 we first performed integrated quantitative proteomics and ubiquitinome profiling in TET2+/-, TRIM4 KO, and isogenic WT iPSC-HSPCs. These analyses nominated 190 proteins with decreased ubiquitination and concomitant increased abundance by both TET2 and TRIM4 deficiency. Knockout of key components of the 3 main innate sensing pathways – TRIF/MYD88 (Toll-like receptor signaling pathway), MAVS (cytosolic RNA sensing) and STING (cytosolic DNA sensing) – identified MAVS as the dominant pathway. DExH-box helicase 58(DHX58), one of the 190 proteins identified by our proteomics analyses and a positive regulator of the MAVS pathway, was confirmed by knockout experiments to mediate the aberrant IFN production by TET2-deficient HSPCs. These results support a model whereby TET2 deficiency and consequent TRIM4 silencing activate type I IFN production through decrease in proteasome-mediated degradation of DHX58. Finally, we show that IFNAR1 blockade by the FDA-approved monoclonal antibody anifrolumab alleviates the clonal advantage of TET2-deficient T cells expressing a CD19-targeted CAR.
 In summary, we report secretion of type I IFN by TET2-mutant HSPCs as the cause of a pro-inflammatory local microenvironment that favors the expansion of mutant cells; we discover TRIM4 silencing as a critical effector of TET2 deficiency; we characterize TRIM4 as a novel negative regulator of type I IFN induction through degradation of DHX58; and we propose a new pharmacologic approach to mitigate the adverse clinical consequences of TET2 mutations.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f3c2c311180626235836ae13cb5857af7df9374" target='_blank'>
              TET2 mutations drive cell-autonomous type I interferon production and selective advantage through TRIM4 silencing
              </a>
            </td>
          <td>
            Baosen Jia, Igor Fijalkowski, Manon Jaud, Minh Nguyen, Lewis Tomalin, Emmanuel Olivier, Chih-wei Fan, V. Jethalia, N. Thongon, Max Dougherty, Saul Carcamo, Franco Izzo, S. Colla, Maria Themeli, Elvin Wagenblast, D. Hasson, Minji Byun, Panagiotis Ntziachristos, E. Papapetrou
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Cancer cells can acquire a reversible, dormant drug-tolerant persister state mimicking embryonic diapause to evade therapy pressure. Deciphering the precise mechanisms driving cancer cells into or out of a diapause-like persister cell-state could provide strategies to overcome resistance. Here, we showed that following chemotherapy, diverse therapeutic agents converge on WNT pathway activation to induce a de novo diapause-like cell state across various triple-negative breast cancer (TNBC) cell line, xenograft, and patient-derived organoid models. Among early persister cells, only transcriptionally WNT-active persisters exhibited the transcriptional and functional characteristics typical of diapause-like cells, including a negative correlation with MYC transcriptional activity and reversible restricted proliferation. The WNT signaling pathway functioned as both an inducer and biomarker of the diapause-like early persister cell state in both parental (chemo-naïve) and chemotherapy-treated cells. Entry into and exit from the diapause-like early persister cell state was triggered by the transcriptional upregulation of components essential for canonical WNT ligand secretion. A combinatorial treatment strategy inhibiting WNT ligand secretion alongside chemotherapy effectively targeted the early mechanisms underlying the acquisition and enrichment of a diapause-like cell phenotype. These findings reveal WNT pathway activation as an early event that leads to a reversible diapause-like persister state and highlight the potential of targeting this axis to prevent the development of drug-resistant populations before they are firmly established.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc9bf626a72b9d5ae7b70a19b1f05585d0584500" target='_blank'>
              Chemotherapy-Treated Breast Cancer Cells Activate the WNT Signaling Pathway to Enter a Diapause-Like Early Persister State.
              </a>
            </td>
          <td>
            Youssef El Laithy, Willy Antoni Abreu de Oliveira, A. Pabba, Alessandra Qualizza, Gwenny Cosemans, Paula Garcia-Diaz, François Richard, Paraskevi Athanasouli, Carla Rios Luci, Wout De Wispelaere, Larissa Mourao, Siân Hamer, Stijn Moens, A. de Jaime-Soguero, M. Baietti, S. Hutten, Jos Jonkers, Stephen-John Sammut, S. Soenen, C. Scheele, Alejandra Bruna, C. Desmedt, D. Annibali, F. Lluis
          </td>
          <td>2025-10-21</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7fb5388d0bd0362161f91d2111550a5aa12010f" target='_blank'>
              Inference of clonal hematopoiesis using the collective behaviour of DNA methylation states
              </a>
            </td>
          <td>
            Samuel. J. C. Crofts, Caleb M. Grenko, Neil A. Robertson, Kristina Kirschner, Tamir Chandra
          </td>
          <td>2025-11-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 ARID1A mutations are among the most common genetic alterations in cancer, occurring in approximately 65% of ovarian clear cell carcinoma (OCCC) and up to 10% of all solid tumors. These mutations disrupt chromatin remodeling and cellular homeostasis, leading to the accumulation of reactive oxygen species (ROS) and a metabolic shift that renders tumor cells increasingly dependent on oxidative phosphorylation (OXPHOS) for survival. This metabolic vulnerability provides a therapeutic opportunity for selective targeting. EO3001 is a novel synthetic agent that transports extracellular Cu(II) to mitochondria, promoting mitochondrial ROS generation and triggering cancer cell death. Preclinical studies have shown that EO3001 exhibits selective cytotoxicity in ARID1A-deficient CCOC models, highlighting its potential as a precision oncology candidate.



 To evaluate EO3001’s activity, isogenic CCOC cell lines with and without ARID1A expression were generated using CRISPR-Cas9 gene editing. The compound’s effects were assessed through a series of in vitro and ex vivo assays, measuring cell viability, mitochondrial ROS levels, OXPHOS activity, and invasive potential under various metabolic stress conditions. Notably, EO3001 demonstrated robust anti-tumor activity in ARID1A-deficient cells, with markedly reduced efficacy in hypoxic environments—supporting the mechanism of action linked to OXPHOS dependence.



 These findings suggest that EO3001 holds promise as a targeted therapeutic for ARID1A-mutant cancers, and ongoing work is focused on optimizing its application in clinically relevant settings. A biomarker-driven clinical trial is planed to further evaluate these findings.



 Jeffrey Bacha, Yuchen Ding, Dennis Brown, Amal M. El-Naggar, David G. Huntsman, Sarath Kanekal, Neil Sankar. EO3001: A novel agent for ARID1A mutant cancers [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr B142.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4751ad06f9259f42e8fe43179e85e381969b2cfc" target='_blank'>
              Abstract B142: EO3001: A novel agent for ARID1A mutant cancers
              </a>
            </td>
          <td>
            J. Bacha, Yuchen Ding, Dennis Brown, Amal M. El-Naggar, D. Huntsman, S. Kanekal, N. Sankar
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cce6be4d706b4653ff7064a56c5297b13de1df65" target='_blank'>
              Travelling Waves in Gene Expression: A Mathematical Model of Cell-State Dynamics in Melanoma
              </a>
            </td>
          <td>
            Charlotte Taylor Barca, Rotem Salmi-Leshem, Vishaka Gopalan, Sarah Woolner, Kerrie L. Marie, Gareth Wyn Jones, Oliver E. Jensen
          </td>
          <td>2025-10-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 Introduction Ph-like B-ALL is a poor-risk B-ALL subtype defined by gene expression signatures that resemble the BCR::ABL1-associated transcriptional program in the absence of the BCR-ABL1 oncoprotein. Recent studies have demonstrated that up to 80% of Ph-like B-ALLs have recurrent gene fusions involving cytokine receptors or tyrosine kinases that can be identified via cytogenetics, fluorescence in situ hybridization (FISH) or RNA sequencing. However, the inability to rapidly and accurately identify these alterations in the clinical setting is a major impediment to developing therapeutic strategies that may have efficacy in this B-ALL subtype. Here we used 88 patient (pt) samples to validate a rapid, streamlined clinical WGS assay (ChromoSeq) for B-ALL and assessed the diagnostic yield for Ph-like-associated SVs in 136 pts enrolled in the Alliance A041501 clinical trial.
 Methods ChromoSeq is a clinical WGS assay for hematologic malignancies that uses rapid laboratory workflows and focused analysis methods to identify clinically relevant SVs, copy number alterations and gene-level mutations. We updated the ChromoSeq pipeline for use in B-ALL pts and validated it using 88 retrospective B-ALL samples (80 adult/young adult and 8 pediatric from Washington University School of Medicine and St. Louis Children's Hospital), 54 of which were analyzed via RNAseq for gene fusions and a 6-gene Ph-like B-ALL signature (CA6, SPATS2L, MUC4, JCHAIN, BMPR1B, ADGRF1, NRXN3 and CRLF2). The “ChromoSeqV2” assay was then used to analyze 136 pretreatment blood or bone marrow samples from the Alliance A041501 (trial NCT03150693). These results were compared to data from RNAseq obtained for research purposes (N=35), clinical cytogenetics (N=69) and Ph-like classification via low-density microarray (LDA) card analysis (all pts).
 Results ChromoSeqV2 was validated in a CLIA-licensed setting using cohort of 88 retrospective B-ALL samples, 45 of which had ICC and WHO class-defining SVs based on G-banded karyotyping, FISH or RNAseq. ChromoSeqV2 detected 100% of the non-Ph-like SVs (37/37), including BCR::ABL1 (N=20), KMT2Ar (N=9), ETV6::RUNX1 (N=3), TCF3::PBX1 (N=1), and rearrangements involving MYC (N=2) or MEF2D (N=2). Fifteen patients met criteria for Ph-like B-ALL via cytogenetics or RNAseq either due to rearrangements of CRLF2 (N=5), JAK2 (N=2) or ABL2 (N=1) or high expression of a 6-gene Ph-like signature (N=7). ChromoSeqV2 detected a Ph-like SV in 93% of these cases (14/15) with 100% specificity in samples negative by RNAseq (N=23).
 ChromoSeqV2 was then used to profile 136 samples from the Alliance A041501 trial, where it identified ICC and WHO class-defining SVs in 87 pts (64%; 95% CI: 55-72%), including Ph-like SVs (56/136, 41%), KMT2A rearrangements (N=5), TCF3::PBX1 (N=3) and SVs involving MYC (N=3), MEF2D (N=2), DUX4 (N=2), and ZNF384 (N=1). Ph-like SVs were dominated by CRLF2 rearrangements (N=45) and included SVs involving EPOR (N=4), JAK2 (N=3), ABL1/ABL2 (N=2) and PDGFRB (N=1). IGH was the most common CRLF2 partner (N=32), followed by P2RY8::CRLF2 (N=7) and other genes (N=5). Multiple CRLF2 SVs were detected in 4 pts, and 33% (15/45) of pts with a CRLF2 rearrangement had a co-occurring CRLF2 gene mutation. The yield of ChromoSeqV2 for SVs in pts with the Ph-like expression signature via LDA card analysis was 66% (54/82; 95% CI 55%-76%) compared to 24% (11/45; 95% CI 13%-40%) by cytogenetics when these studies were successful and 70% (12/17; 95% CI 44%-90%) vs RNAseq in pts with available data. The sensitivity of ChromoSeqV2 for Ph-like SVs identified by either cytogenetics, FISH or RNAseq was 100% (21/21; 95% CI 84%-100%) and the positive predictive value of Ph-like SVs for the Ph-like expression signature by the LDA card assay was 96% (54/56; 95% CI 88%-99%).
 Conclusions Genomic assessment of 136 B-ALL pts from the Alliance A041501 trial using the ChromoSeqV2 rapid clinical WGS assay was 100% sensitive for Ph-like SVs identified by other molecular methods and had a 96% PPV for the Ph-like expression signature via LDA card analysis. The diagnostic yield of SV pts with the Ph-like signature was 66% using ChromoSeqV2, which was similar to RNAseq and superior to cytogenetics. These results demonstrate that ChromoSeqV2 is an effective approach for identifying potentially targetable rearrangements in B-ALL pts suspected of having the Ph-like phenotype. Support: U10CA180821, U10CA180882; . Servier; Pfizer.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/496ee390ef7f76b531cb6af7732693c780f64abe" target='_blank'>
              Increased yield of recurrent structural variants (SVs) in Ph-like B-cell acute lymphoblastic leukemia (B-ALL) using a streamlined clinical whole-genome sequencing (WGS) assay
              </a>
            </td>
          <td>
            Andrew Hughes, Feiyu Du, Michele O'Laughlin, Gregory Sprenger, Mohamed Mahgoub, Nidhi Davarapalli, Heidi Struthers, Haley Abel, Christopher Manring, R. Harvey, C. Saygin, Jun Yin, Jun J. Yang, D. DeAngelo, S. Dinner, Richard M. Stone, Marina Konopleva, Ann-Kathrin Eisfeld, Molly Schroeder, Eric Duncavage, A. Advani, Wendy Stock, Geoffrey Uy, David Spencer
          </td>
          <td>2025-11-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) arises when hematopoietic stem cells (HSCs) acquire mutations that confer a competitive advantage over wild-type HSCs, leading to their expansion in the bone marrow with clonal progeny that circulate in the blood and are most readily detected through peripheral blood sequencing. The prevalence of CH increases with age and is linked to a higher risk of hematologic malignancies and various non-malignant diseases, particularly atherosclerotic cardiovascular disease. CH is not merely a biomarker; it actively contributes to the pathogenesis of these agerelated conditions. Therefore, targeting the expansion of mutant clones and their downstream effects offers an opportunity to prevent these adverse health outcomes. CH involves mutation-specific biological changes that sustain the abnormal HSC phenotype, including epigenetic dysregulation, aberrant inflammatory signaling, metabolic reprogramming, and altered intracellular signaling pathways. A deeper understanding of these processes has led to the development of targeted therapeutic approaches. This review addresses the practical challenges of implementing interventions against CH, focusing on balancing risk and benefit and selecting appropriate patients. It discusses emerging treatments targeting the pathogenic mechanisms in CH, such as epigenetic modulators, anti-inflammatory therapies, metabolic inhibitors, and signaling pathway inhibitors. We also highlight potential novel therapeutic strategies on the horizon, such as immune-based approaches for selective clonal elimination and gene-editing therapies to correct causative mutations. These advances reframe CH as a potentially modifiable condition rather than an inevitable consequence of aging, creating opportunities for early intervention before progression to overt disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1ced18515225e7e7e24fcba946a2b6468ad58d7" target='_blank'>
              Novel and emerging therapeutic strategies for clonal haematopoiesis.
              </a>
            </td>
          <td>
            Mohsen Hosseini, Steven M. Chan
          </td>
          <td>2025-11-13</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Gene therapy and RNA (ribonucleic acid)-based therapeutic strategies have emerged as promising alternatives to conventional diabetes treatments, significantly expanding the therapeutic landscape using viral and non-viral vectors, and RNA modalities such as mRNA (messenger ribonucleic acid), siRNA (small interfering ribonucleic acid) and miRNA (micro ribonucleic acid). Recent advancements in these fields have led to notable preclinical successes and ongoing clinical trials, yet they are accompanied by debates over safety, efficacy and ethical considerations that underscore the complexity of clinical translation. This review offers a comprehensive analysis of the underlying mechanisms by which these treatments target diabetes, critically evaluating the fundamental concepts and mechanistic insights that form their basis, while highlighting current research gaps, such as the challenges in long-term stability and efficient delivery of RNA-based therapies, and potential adverse effects associated with gene therapy techniques. By synthesizing diverse perspectives and controversies, the review outlines future directions and interdisciplinary approaches aimed at overcoming existing hurdles, ultimately setting the stage for innovative, personalized diabetes management and addressing the broader clinical and regulatory implications of these emerging therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03c517f4bbb06df197bad41f89054859c1969d78" target='_blank'>
              The Role of Gene Therapy and RNA-Based Therapeutic Strategies in Diabetes
              </a>
            </td>
          <td>
            Mustafa Tariq Khan, Reem Emad Al-Dhaleai, Sarah M. Alayadhi, Zainab Alhalwachi, Alexandra E. Butler
          </td>
          <td>2025-10-22</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0007d62f2b8c21bcd3de104015c7a577a0669693" target='_blank'>
              Defining treatment-resistant brain cancer: Genetic screening to identify oncogene-driven immunomodulation and therapy resistance.
              </a>
            </td>
          <td>
            Zachary David Myers, S. Matosevic
          </td>
          <td>2025-10-26</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="


 Epigenetic reinforcement of T cell exhaustion is a major barrier limiting T cell responses during immunotherapy, however the core epigenetic regulators restricting anti-tumor immunity during prolonged antigen exposure are not clear. We investigated three commonly mutated epigenetic regulators that promote clonal hematopoiesis to determine if they impact T cell stemness and response to checkpoint blockade immunotherapy. CD8 T cells lacking Dnmt3a, Tet2, or Asxl1 preserved a progenitor-exhausted (Tpex) population for over one year during chronic antigen exposure without undergoing malignant transformation. Asxl1 controlled the self-renewal capacity of T cells and reduced CD8 T cell differentiation via H2AK119-ubiquitination and epigenetic modification of the PR-DUB pathway. Asxl1-deficient T cells synergized with anti-PDL1 immunotherapy to improve tumor control in experimental models and conferred a survival advantage to mutated T cells from treated patients.



 Center for Translational Immunology and Immunotherapy (CeTI2) at St. Jude. NIH (R01AI114442 and R01CA237311 to BY, K08CA279926 to C.C.Z, R35CA209859 to P.J.), the NCC Network Young Investigator Award (to C.C.Z), Alex Lemonade Stand Young Investigator Grant (to C.C.Z), Van Andel- Stand up to Cancer team to B.Y., P.J., S.B., C.O., & K.G.), ASSISI Foundation funding (to B.Y.)



 Tumor Immunology: Checkpoints, Prevention, and Treatment (TIPT)
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08b2b981353a37687740e49213184c3b2429bab2" target='_blank'>
              Epigenetic regulators of clonal hematopoiesis control CD8 T cell stemness during immunotherapy 2034
              </a>
            </td>
          <td>
            Tae Gun Kang, Xin Lan, Tian Mi, Hongfeng Chen, Shanta Alli, Song-Eun Lim, Sheetal Bhatara, Anoop Babu, Grace A Ward, Sofia Bentivegna, Josh Jang, Marianne L. Spatz, J. Han, B. Schlotmann, Jakob Jespersen, C. Derenzo, Jiyang Yu, Steve Baylin, Peter Jones, Casey Connell, Kirsten Grønbæk, Ben Youngblood, Caitlin C. Zebley
          </td>
          <td>2025-11-01</td>
          <td>The Journal of Immunology</td>
          <td>0</td>
          <td>18</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [7],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>