<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2026-02-23 10:03:56 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="DHFR gene amplification is commonly present in methotrexate (MTX)-resistant colon cancer cells and acute lymphoblastic leukemia. In this study, we proposed an integrative framework to characterize the amplified region by using a combination of single-molecule real-time sequencing, next-generation optical mapping, and chromosome conformation capture (Hi-C). We identified an amplification unit spanning 11 genes, from the DHFR gene to the ATP6AP1L gene position, with high adjusted interaction frequencies on chromosome 5 (~2.2 Mbp) and a twenty-fold tandemly amplified region, and novel inversions at the start and end positions of the amplified region as well as frameshift insertions in most of the MSH and MLH genes were detected. These mutations might stimulate chromosomal breakage and cause the dysregulation of mismatch repair. Characterizing the tandem gene-amplified unit may be critical for identifying the mechanisms that trigger genomic rearrangements. These findings may provide new insight into the mechanisms underlying the amplification process and the evolution of drug resistance. Sequencing a large region of DNA containing many surplus copies of genes linked to drug resistance in colon cancer cells may illuminate how these genomic rearrangements arise. Such regions of gene amplification are highly repetitive, making them impossible to sequence using ordinary methods, and little is known about how they are generated. Using advanced methods, Jeong-Sun Seo at Seoul National University Bundang Hospital in South Korea and co-workers sequenced a region of gene amplification in colon cancer cells. The amplified region was approximately 20 times the length of that in healthy cells and contained many copies of an eleven-gene segment, including a gene implicated in drug resistance. The region also contained mutations in chromosomal repair genes which would disrupt repair pathways. These results illuminate the genetic changes that lead to gene amplification and drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental and Molecular Medicine, Experimental & Molecular Medicine</td>
          <td>5</td>
          <td>50</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum
                </a>
              </td>
          <td>
            Gustavo S. França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>95</td>
          <td>46</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bd68744cdbfac994a21e7c57ae21390703eb542" target='_blank'>
              Selection for targeted therapy resistance leads to an indirect selection for higher phenotypic plasticity and enhanced evolvability to orthogonal stressors
              </a>
            </td>
          <td>
            Alicia Bjornberg, Aobuli Xieraili, M. Froid, R. Barker-Clarke, Jeffrey Maltas, Robert J. Vander Velde, Jamir Riffas, Berkley E. Gryder, Jacob G. Scott, David Bassanta, Alexander R. A. Anderson, Virginia A Turati, A. Marusyk
          </td>
          <td>2026-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63674f265b1ac623907383e45f60b5724b528b86" target='_blank'>
              Recurrent Escape from Osimertinib-Induced Senescence Promotes Genomic Instability Associated with Therapeutic Resistance
              </a>
            </td>
          <td>
            H. McDaid, Nazia S Jamil, Qualia Hooker, Nadjet Cornejal, Dean Hosgood
          </td>
          <td>2026-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7161441b553c5a95b3c3fe02b7b3df61a233d61e" target='_blank'>
              Upregulation of Genetic Markers of Poor Prognosis following Chemotherapy in Acute Myeloid Leukemia Cells
              </a>
            </td>
          <td>
            A. Taylor, Michael K. Strasser, , Irit Rubin, A. Kaipainen, Angela Oliveira Pisco, Sui Huang
          </td>
          <td>2026-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Epigenetic changes are a major driver of cancer progression, placing considerable focus on epigenetic regulators as therapeutic targets. Protein arginine methyltransferase 5 (PRMT5) is one such regulator, and numerous PRMT5 inhibitors (PRMT5i) in clinical trials. Despite this, the mechanisms and consequences of PRMT5i-resistance are unknown. Here, we demonstrate that aggressive cancer progression is an inbuilt feature of PRMT5i-resistance acquisition in lung adenocarcinoma (LUAD). Independently-generated resistant cell lines gain dedifferentiation signatures that typify late-stage disease and show increased metastatic potential in vivo. We establish that these state shifts are a direct consequence of PRMT5i action; treatment induces rapid and widespread chromatin rewiring, enabling derepression of late-stage disease states that are stably established in resistant cells. Notably, treatment of lung tumor-bearing mice drives rapid disease advancement without decreasing tumor burden, showing that drug-induced disease progression supersedes any benefits from PRMT5 inhibition in vivo. Furthermore, analyses of human cell lines and patient cohorts supports the notion of PRMT5 inhibition-mediated dedifferentiation. Collectively our data show that PRMT5i can actively promote self-resistance and disease progression in different tumor types. This raises serious concerns for the use of PRMT5i in patients, arguing that clinical studies should consider the possibility of drug-induced plasticity, resistance, and disease advancement.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d679531596c96a496c4e169fe4344efc4f3f38d0" target='_blank'>
              PRMT5 inhibitors actively promote metastatic progression of lung adenocarcinoma
              </a>
            </td>
          <td>
            Colin E. Fowler, Natalie A. O’Hearn, Nicole Henning, Anthony Griffen, Nicolas Mathey-Andrews, Griffin J. salus, Renin Hazan, Tyler Jacks, Aurora A. Burds, Helen S Mueller, Yadira M. Soto-Feliciano, Lindsay M. LaFave, Jacqueline A. Lees
          </td>
          <td>2026-02-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/774d717fb446d87b8fca81f2c265377058a6eee1" target='_blank'>
              On-target mutations confer resistance to WRN helicase inhibitors in Microsatellite Unstable Cancer Cells
              </a>
            </td>
          <td>
            G. Picco, Yanhua Rao, Angham Al Saedi, Samantha J. Walker, S. Bhosle, Yang Lee, Sara F. Vieira, Maria Garcia-Casado, Gilberto Valdes Garcia, Kieron May, F. Sassi, Inigo Barrio Hernandez, Mamta Sharma, Cansu Dinçer, Theo Bell, Anastasia Kavasakali, Ray Shenje, Cuthbert Martyr, Edward J. Brnardic, Nanhua Deng, Hristina Grigorova Dimitrova, Emre Karakoc, Sandeep Sundara Rajan, Nicole Chan, Emma Hitch, Katrina McCarten, Camille Fourneaux, Zoe Hewitson, H. Lightfoot, S. Barthorpe, James P. Phelan, Phil Landis, Brian T. Jones, Diana M. Muñoz, Jay Prakash, P. Barsanti, J. Taygerly, Michael P. DeMartino, Emanuel Gonçalves, A. Bertotti, L. Trusolino, Michael A. White, Geeta Sharma, M. Coelho, J. Houseley, Benjamin Schwartz, Mathew J. Garnett
          </td>
          <td>2025-12-27</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>67</td>
        </tr>

        <tr id="Mutations in epigenetic regulators are common in bladder cancer, yet their impact on therapeutic responses remains unclear. Here, we identify that loss-of-function mutations in KDM6A, a histone demethylase altered in about 26% of advanced bladder cancers, are associated with poor survival after cisplatin chemotherapy, whereas they correlate with improved outcomes with anti-PD-1 therapy. Using CRISPR-Cas9-engineered murine and human bladder cancer models, we show that KDM6A deficiency increases formation of extrachromosomal circular DNA carrying chemoresistance loci, promoting cisplatin resistance. In parallel, KDM6A loss impairs DNA repair and rewires tumor metabolism, reducing glycolysis and lactate output. This metabolic shift diminishes histone lactylation in regulatory T cells, suppressing immunoregulatory genes and limiting expansion of PD-1hi regulatory T cells. Collectively, our findings establish KDM6A mutation as a key regulator of therapeutic responses, providing a foundation for its use in guiding precision therapy in advanced bladder cancer. Genes encoding key epigenetic regulators, including Lysine Demethylase 6A (KDM6A), are frequently mutated in bladder cancer. Here, the authors show that loss of KDM6A promotes formation of extrachromosomal circular DNA (eccDNA), genomic instability, and metabolic reprogramming, driving resistance to cisplatin chemotherapy while simultaneously enhancing sensitivity to immune checkpoint inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9139b0ede19d5b7c3ea40131744831c88cd696ed" target='_blank'>
              Loss of KDM6A-mediated genomic instability and metabolic reprogramming regulates response to therapeutic perturbations in bladder cancer
              </a>
            </td>
          <td>
            Pratishtha Singh, R. D’Rozario, Bidisha Chakraborty, Swadhin Meher, D. Raychaudhuri, Aminah J. Tannir, Yang Li, Anurag Majumdar, Jessalyn Hawkins, Yun Xiong, Philip L. Lorenzi, Padmanee Sharma, K. Akdemir, P. Pilié, Abhinav K. Jain, Byron H Lee, Sangeeta Goswami
          </td>
          <td>2026-01-07</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>28</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is driven by a combination of genetic alterations and non-mutational mechanisms that disrupt normal hematopoiesis and support leukemic cell survival. While the mutational landscape of AML is well characterized, the non-genetic processes that sustain leukemic maintenance remain comparatively less understood. Using human AML cell lines and murine models of AML, we identify BCL2-associated transcription factor 1 (BCLAF1) as a key regulator of leukemic progression through control of mRNA processing. BCLAF1 physically associates with core spliceosome components and regulates alternative splicing, with a predominant effect on intron retention. We demonstrate that BCLAF1 is required for the productive splicing of activating transcription factor 4 (ATF4) mRNA, thereby sustaining ATF4 protein expression. Loss of BCLAF1 reduces ATF4 protein levels, leading to downregulation of metabolic target genes and disruption of de novo amino acid biosynthesis. Furthermore, depletion of BCLAF1 sensitizes AML cells to venetoclax, a clinically relevant BCL-2 inhibitor. Together, these findings uncover a previously unrecognized role for BCLAF1 in coordinating mRNA splicing and metabolic adaptation in AML, highlighting its potential as a therapeutic target. Statement of significance Aberrant RNA splicing and metabolic reprogramming are hallmarks of cancer, yet how these processes are mechanistically linked remains unclear. This study identifies BCLAF1 as a key regulator connecting splicing control to amino acid metabolism in acute myeloid leukemia, revealing a previously unrecognized functional vulnerability at the intersection of these pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2686f36b152260cde0d49166c250ce1a1551afc7" target='_blank'>
              BCLAF1 links RNA splicing to ATF4-dependent metabolic adaptation in acute myeloid leukemia
              </a>
            </td>
          <td>
            Laura López-Hernández, Stephanie J Crowley, Sara Cea-Sánchez, Ricardo de Arellano, Neetij Krishnan, Patrick Toolan-Kerr, Lynn S. White, Domenico Ignoti, Emily Soto-Hidalgo, Stefano Gustincich, Román González-Prieto, Luca Pandolfini, Isaia Barbieri, Patricia Altea-Manzano, J. A. Magee, J. Bednarski, Gonzalo Millán-Zambrano
          </td>
          <td>2026-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63b55c41a895a4562f500627e7c41a6885170b27" target='_blank'>
              Clonal morphology-guided combination therapies overcome heterogeneity-driven drug tolerance
              </a>
            </td>
          <td>
            Xiaoxi Li, Ling Liu, Yuanyuan Yan, Mengfang Yang, Lingli Luo, Yong Jiang, Weijun Ren
          </td>
          <td>2026-02-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Molecular targeted therapies targeting KRAS signaling have significantly improved patient outcomes, but they have not achieved sufficient therapeutic efficacy in colorectal cancer (CRC). Here, we demonstrate that a subset of KRAS-mutant CRC cells transitions to a cellular state characterized by enhanced ribosome biogenesis upon KRAS signaling inhibition. The mitogen-activated protein kinase kinase inhibitor, trametinib, and AMG510 induce a cellular state characterized by a gene expression profile highly enriched for ribosome biogenesis. We find that they are vulnerable to the inhibition of RNA polymerase I, and they exhibit synergistic anti-tumor effects with trametinib in an autochthonous mouse model of intestinal tumors and human patient-derived organoids (PDOs). These observations demonstrate that high ribosome biogenesis induced by KRAS inhibition is indispensable to maintain this cellular state and is a potential therapeutic target. Overall, this study reveals novel mechanisms of drug tolerance to KRAS inhibition, thereby facilitating the development of new therapeutic strategies. Drug tolerance to KRAS inhibition remains a major challenge in cancer therapy. Here the authors find that KRAS inhibition induces a cellular state characterized by enhanced ribosome biogenesis in KRAS-mutant colorectal cancer cells and these cells are vulnerable to inhibition of RNA polymerase I.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92b7de6d8aeceaad9c8d57eb1d9c3d308c5a80e2" target='_blank'>
              Ribosome biogenesis as a potential therapeutic target in KRAS mutant colorectal cancer
              </a>
            </td>
          <td>
            Yui Tanaka, M. Sakahara, H. Yamanaka, Yasuko Natsume, Daisuke Kusama, Kohei Kumegawa, Harunori Yoshikawa, Yuich Abe, K. Okabayashi, Shimpei Matui, Y. Kitagawa, Naohiko Koshikawa, H. Osumi, E. Shinozaki, Satoshi Nagayama, Jun Adachi, R. Maruyama, Ryoji Yao
          </td>
          <td>2025-12-27</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d81001c0250b1fa8d16724cb8f5b65fb66caff9" target='_blank'>
              Functional genomics identifies therapeutic options, biomarkers, and resistance mechanisms for high-grade gliomas
              </a>
            </td>
          <td>
            W. Lin, F. Kosari, James Smadbeck, Michael Barrett, Ryan W. Feathers, Jamie Hall, Dorsay Sadeghian, Sotiris Sotiriou, Sarah H. Johnson, F. Harris, Taylor Berry, A. Mccune, Stephen J. Murphy, Lindsey Kinsella, Lauren E. Haydu, Diogo Moniz-Garcia, Shannon P. Fortin Ensign, Lin Yang, Angela R. Emanuel, Leila A Jones, J. Schaefer-Klein, Cristiane M. Ida, Marcela A. Salomao, Wendy J. Sherman, Alyx B Porter, Steven S. Rosenfeld, S. Kizilbash, K. Jaeckle, Maciej M Mrugala, Aaron S. Mansfield, M. Borad, B. Bendok, Terry C Burns, A. Quiñones-Hinojosa, John C. Cheville, G. Vasmatzis, P. Anastasiadis
          </td>
          <td>2026-02-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Colorectal cancer (CRC) develops through evolutionary processes involving genomic alterations, epigenetic regulation, and microenvironmental interactions. While traditionally explained by the stepwise accumulation of driver mutations, contemporary evidence supports a ‘Big Bang’ model in which many early-arising clones expand simultaneously to establish extensive heterogeneity. We reviewed recent studies employing spatially resolved multi-omic sequencing of tumour glands combined with computational modelling. These approaches enable high-resolution reconstruction of clonal architecture, transcriptional states, and chromatin accessibility. Findings show that although early clonal mutations shape tumour expansion, gene expression variability can be independent of genetic ancestry and instead reflects phenotypic plasticity driven by microenvironmental cues. Epigenomic analyses identified recurrent somatic chromatin accessibility alterations in promotors and enhancers of oncogenic pathways, frequently in the absence of DNA mutations, suggesting alternative mechanisms of gene regulation. Immune-focused studies demonstrated that early silencing of antigen-presenting genes and loss of neoantigens facilitate immune escape despite active surveillance. CRC is shaped by an interplay of genome, epigenome, and immune evolution, with non-genetic mechanisms and tumour plasticity emerging as important drivers of progression and therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c00d0ede4db6fdd44f5a7c3b138a4161c7d8d0f7" target='_blank'>
              Advances in Colorectal Cancer Cell Biology and Clonal Evolution
              </a>
            </td>
          <td>
            Sopozme Toghey, Elizabeth J. Harvey-Jones, Jonathan D. Towler, Charlotte J. H. Hafkamp, Irene Y. Chong
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Phenotypic plasticity is a prominent cancer feature that contributes to metastatic potential and resistance to therapy across multiple cancer types. Cancer cell state transitions have been attributed to transcriptional programs, such as the AP1/TEAD-regulated gene network driving the mesenchymal-like (MES) phenotype. In addition, during dissemination, tumor cells are subjected to variable loads of physical mechanical pressure and constriction across transited tissue, which are thought to impact nuclear molecular crowding. How the interplay between mechanical pressure, global 3D nuclear architecture and transcriptional programs contributes to MES identity and metastatic adaptation remains unclear. Using cutaneous melanoma as a model for early dissemination, we integrate in vitro and in vivo epigenomic profiling with nanoscale imaging of cell lines and patient samples to investigate chromatin organization features underlying the MES phenotype. We find that in MES cells, CTCF is relocated from domain boundaries to regulatory regions of EMT-like genes, leading to reduced insulation, extended topological associated domains (TADs) and increased inter-domain contacts, and de novo formation of chromatin hubs. This conformational rewiring, along with loss of heterochromatin, supports nuclear deformability during invasion and dissemination. Conversely, physical constriction of melanocytic cells induces MES-like chromatin features —including CTCF repositioning and heterochromatin loss— and promotes metastasis in vivo. Similarly, pharmacological inhibition of the heterochromatin mark H3K9me3 triggers MES characteristics and increases invasiveness. These results demonstrate that metastatic competency involves both epigenetic and structural nuclear reprogramming, enabling shifts in gene networks and physical adaptability. Our findings reveal mechanistic links between nuclear architecture and aggressive tumor behavior, identifying potential biomarkers and therapeutic targets to intercept metastatic progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92c8bf70d1df18ff53fcb785d7213684e545662a" target='_blank'>
              Chromatin architecture and physical constriction cooperate in phenotype switching and cancer cell dissemination
              </a>
            </td>
          <td>
            Pietro Berico, Cody L Dunton, L. Almassalha, Amanda Flores-Yanke, Karla I Medina, Nicolas Acosta, Tara Muijlwijk, Catherine Do, Soobeom Lee, S. Edmiston, D. Corcoran, Allison Reiner, Caroline E. Kostrzewa, Kathy Dorsey, Milad Ibrahim, Ronglai Shen, Nancy E. Thomas, Amanda W. Lund, Ata S. Moshiri, Iman Osman, I. Aifantis, Jane A. Skok, Vadim Backman, Eva Hernando
          </td>
          <td>2026-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d73e2496f48eaf63e522263faed717c4d2cc83e" target='_blank'>
              Exploring the Role of APC in Modulating Chemotherapeutic Response in Triple-Negative Breast Cancer cells
              </a>
            </td>
          <td>
            T. Nair, M. K. Vanklompenberg, Jenifer R Prosperi
          </td>
          <td>2026-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce1dd81fa69ca7596c6e2f6203643899d73e28c1" target='_blank'>
              In vivo isogenic modelling unveils a TP53-mediated relapse phenotype in T-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Stéphanie Gachet, S. Quentin, L. Hernandez, Loïc Maillard, M. Passet, R. Kim, Hugo Bergugnat, Melha Benlebna, Maxime Boy, Véronique Parietti, P. Fenaux, André Baruchel, Hervé Dombret, N. Boissel, François Sigaux, Hugues de Thé, E. Clappier, J. Soulier
          </td>
          <td>2026-01-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddfc626bda7c6db9e371ab841d5c95cbd7dff747" target='_blank'>
              RNAseq analysis reveals the recurrent loss of heterozygosity in lung cancer and associated transcription patterns
              </a>
            </td>
          <td>
            Ruslan Gumerov, Wangzhen He, Phong Luong, Filippo Dall’Olio, Y. Vassetzky, Anna Schwager
          </td>
          <td>2026-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ce0c14625245a7fcfee0cd44b96288c3d8181a2" target='_blank'>
              Transcriptomic Profiling Reveals Key Molecular Events and Pathways Driving Acquired Chemotherapy Resistance in Relapsed Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Min Zou, Jiaru Shi, Jing Hu, Chengming He
          </td>
          <td>2025-12-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Recent advances in genomic technologies have greatly enhanced our understanding of genotype–phenotype relationships and improved diagnostic of genetic diseases. However, the dissection of complex structural variants remains challenging due to the limitations of current methods in resolving their breakpoint and interpreting phenotypes involving multiple disrupted genes. In this study, we demonstrate how an integrative approach—combining cytological, genomic, and transcriptomic methods—enables the detection, structural and functional characterization of a complex structural variants affecting the MBD5, USP34, and XPO1 genes. Our findings underscore the utility of the Exo-C, a modified chromosome conformation capture technique, in resolving complex rearrangements. We also report, for the first time, a composite neurodevelopmental phenotype resulting from the combined effects of MBD5-associated intellectual disability and 2p15p16.1 microdeletion syndromes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c987c3067a72472e130e25f49fdb579eae34de80" target='_blank'>
              Zooming into rearranged genome: applying pipeline of cytological, genomic, and transcriptomic methods for structural variant interpretation
              </a>
            </td>
          <td>
            M. Gridina, T. Lagunov, P. Belokopytova, N. Torgunakov, A. Nurislamov, D. Yurchenko, Zhanna G Markova, T. Markova, Yana Stepanchuk, Galina Koksharova, Pavel Orlov, Anna Subbotovskaia, O. Ryzhkova, Nadezhda V. Shilova, Veniamin Fishman
          </td>
          <td>2025-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc2d84fd2fd2721449315ec11065338aff77d0ad" target='_blank'>
              MtDNA heteroplasmy controls tumor immune reprogramming through mitochondrial translation
              </a>
            </td>
          <td>
            Ning Liu, Guoshi Chai, Qunyu Lv, Xiaoling Deng, Jiatong Chen, Qian Zhang, Leping Zhang, Xu Liu, Ruolan Tang, Zhe Xu, Shang Cai, Min Jiang
          </td>
          <td>2026-02-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Immune aging is being increasingly recognized as a critical barrier to effective cancer immunotherapy, as the aged tumor microenvironment (TME) drives T cell dysfunction and impairs immune control of cancer1–4. However, the key molecular drivers of this process as well as potential targets to rescue T cell dysfunction in aged tumors remain incompletely understood. Therefore, we performed in vivo single-cell CRISPR screens in CD8⁺ T cells within aged tumors and tumor-draining lymph nodes (tdLNs). We identified Dusp5 and Zfp219 as key regulators of T cell persistence and effector differentiation in aged hosts. Loss of Dusp5, a negative regulator of ERK signaling, increased ERK1/2 phosphorylation and enhanced T cell proliferation in both young and aged tumors. In contrast, loss of Zfp219, a transcriptional repressor, induced epigenetic reprogramming of cytotoxic gene programs, thereby increasing granzyme secretion and enhancing antitumor immunity. Moreover, expression of the human ortholog gene ZNF219 is increased within intratumoral CD8⁺ T cells in older cancer patients. High ZNF219 expression correlates with poorer survival following immune checkpoint blockade (ICB) and reduces persistence of human intratumoral T cells. Notably, Zfp219 ablation synergized with anti-PD-1 blockade in mice to expand effector-like CD8⁺ T cells, leading to significantly enhanced anti-tumor immunity and tumor clearance in aged hosts. Together, these findings highlight Dusp5 and Zfp219 as critical drivers of age-related T cell dysfunction and as potential therapeutic targets to rejuvenate T cell antitumor immunity in older cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/017b5a7413706a4c2c7fbdc538a3a98539127c42" target='_blank'>
              CRISPR screens identify targets to rescue age-related T cell dysfunction in cancer
              </a>
            </td>
          <td>
            Alex C. Y. Chen, Keely Ji, Cansu Yerinde, Nelson H. Knudsen, K. Bi, Daniela Martinez, Thomas J Carmona-LaSalle, Kazuhiro Taguchi, Katherine H Xu, Elizabeth M. Seider, Marc A. Schwartz, Maria Zschummel, Linda T. Nieman, Kathleen B. Yates, T. R. Mempel, R. Manguso, N. Hacohen, Debattama R. Sen
          </td>
          <td>2026-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>118</td>
        </tr>

        <tr id="Highlights What are the main findings? We established and characterized a unique pair of isogenic cervical cancer cell lines (AdMer35/AdMer43) from a patient with squamous cell carcinoma of the cervix before and after chemoradiotherapy, revealing that resistance is not caused by a single factor but by complex reorganization of cancer cells. Single-cell analysis identified key resistance hallmarks: transcriptional reprogramming towards embryonic stemness (HOX, POU5F1), a shift from fibrillar to non-fibrillar collagen expression, and activation of interferon/inflammatory pathways. What are the implications of the main findings? The study maps a remodeled tumor ecosystem with a metabolically reprogrammed, pro-tumorigenic senescent-like cell population and a progenitor-like cell reservoir, identifying them as critical therapeutic targets for eliminating resistant cell pools. The identified key pathways, Wnt, TGFβ, BMP, and the specific ligands EREG and SEMA3C, along with the metabolic vulnerabilities of the senescent compartment, provide a robust framework for future therapeutic exploration. Abstract Cervical cancer remains a significant global health burden, with chemoradioresistance representing a major obstacle to successful treatment. To elucidate the mechanisms underlying this resistance, we established a unique pair of isogenic primary cervical cancer cell lines, AdMer35 and AdMer43, obtained from a patient with squamous cell carcinoma of the cervix before and after radiation therapy. The aim of our study was to characterize the transcriptomic and cellular heterogeneity of these cells. We conducted an in-depth comparative analysis using single-cell RNA sequencing. Analysis of this paired, patient-derived isogenic model suggests that chemoradioresistance can arise through coordinated multilevel cellular adaptations. Resistant AdMer43 cells demonstrated transcriptional reprogramming, with the upregulation of embryonic stemness factors (HOX, POU5F1, SOX2), a shift in extracellular matrix from fibrillar to non-fibrillar collagens, and activation of inflammatory pathways. We identified and characterized critical cell-state dynamics: resistant cells exhibited a remodeled ecosystem with a metabolically reprogrammed senescent-like cell population showing an enhanced pro-tumorigenic communication via EREG, SEMA3C, BMP, and WNT pathways. Furthermore, we identified a progenitor-like cell population with a minimal CNV burden, potentially serving as a reservoir for tumor persistence. These findings offer novel insights for developing targeted strategies to eliminate resistant cell pools and improve cervical cancer outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ccd2a5a0a730a290acb2a66420d2bdad799661d" target='_blank'>
              Single-Cell Transcriptomic Landscape of Cervical Cancer Cell Lines Before and After Chemoradiotherapy
              </a>
            </td>
          <td>
            Dmitriy V Semenov, Irina S Tatarnikova, A. S. Chesnokova, Vadim A. Talyshev, M. Zenkova, E. Logashenko
          </td>
          <td>2026-01-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d6260543e09115d78b28611e811c1654d5ea522" target='_blank'>
              Mutant ASXL1 Drives Transcriptional Activation and Repression in Human Hematopoiesis
              </a>
            </td>
          <td>
            M. Hall, Aliya Quintal, Samantha Worme, Thai T. Nguyen, Zachary Schonrock, Mitsuhiro Tsuchiya, Sonia N. Acharya, Hanqian L Carlson, Itallia V. Pacentine, Shawn B Shrestha, Jommel Macaraeg, R. Armstrong, James McGann, Sara Evans-Dutson, Theresa A. Lusardi, Brendan L. O’Connell, Andrew C. Adey, G. Yardımcı, Julia E. Maxson, Theodore P. Braun
          </td>
          <td>2026-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="The efficacy of anti-cancer therapies is severely limited by the emergence of drug resistance. While genetic drivers are well-characterized, growing evidence suggests that non-genetic mechanisms, particularly those involving drug-tolerant persisters (DTPs), play a pivotal role in solid tumor relapse. To elucidate the evolutionary dynamics of DTP-induced resistance, we develop a stochastic agent-based model (ABM) of solid tumor evolution that couples macroscopic population dynamics with microscopic epigenetic state inheritance during the cell cycle. Our simulations accurately reproduce the temporal progression of relapse observed in experimental studies, capturing the dynamic transition from sensitive cells to DTPs, and ultimately to stable resistant phenotypes under prolonged therapy. By explicitly modeling the epigenetic plasticity of individual cells, our model bridges the gap between cellular heterogeneity and population-level tumor evolution. Furthermore, we performed \textit{in silico} clinical trials using virtual patient cohorts to evaluate therapeutic outcomes, demonstrating that optimized adaptive treatment strategies can significantly delay tumor relapse compared to standard dosing. This study provides a quantitative framework for dissecting DTP-driven resistance mechanisms and designing more effective, biologically informed therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5335957f9dd7ef1ab2ffbe1285dc0a351204f56d" target='_blank'>
              Epigenetic state inheritance drivers drug-tolerant persister-induced resistance in solid tumors: A stochastic agent-based model
              </a>
            </td>
          <td>
            Xiyin Liang, Jinzhi Lei
          </td>
          <td>2026-01-30</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The BCL-2 inhibitor venetoclax (VEN) has emerged as an important therapeutic backbone for hematological malignancies, but secondary resistance is a major challenge. In acute lymphoblastic leukemia (ALL), early clinical trials promise high efficacy as well. However, relapse is observed frequently, and so far, only a few resistance-inducing mechanisms have been discussed. We employed KMT2A-rearranged ALL cell lines and xenograft models to elucidate mechanisms of VEN resistance. Targeted DNA and mRNA sequencing, single-cell mRNAseq, as well as protein expression analyses were conducted. All models initially responded well but finally displayed secondary resistance. Novel as well as previously known pathogenic variants in tumor suppressor TP53 as well as pro-apoptotic molecule BAX, but not BCL2, were observed. Gene and protein expression studies demonstrated multifarious changes in resistant cells, with high inter- and intra-model heterogeneity. Finally, single-cell RNA sequencing revealed a likely contribution of the tumor microenvironment in the development of VEN resistance, as indicated by modulation of genes involved in cell-cell interaction and humoral signaling in resistance-specific clusters. Our data demonstrate and characterize the rise of VEN resistance in KMT2A-rearranged ALL models, suggesting that relapse must be expected in the clinical setting and that multifactorial processes are involved in this process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11ad7042e90c4e9abc4265e52191b4c0932e542a" target='_blank'>
              Venetoclax resistance in preclinical KMT2A-rearranged acute lymphoblastic leukemia models is characterized by high inter- and intra-model heterogeneity
              </a>
            </td>
          <td>
            Anna Richter, Lea Kinsky, Sandra Lange, N. Trakooljul, F. Hadlich, Anett Sekora, Gudrun Knuebel, S. Krohn, Christian Schmidt, Michelle Busch, Tim Schreiber, Simone Kumstel, K. Wimmers, H. Murua Escobar, Christian Junghanss
          </td>
          <td>2025-12-29</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/130c768582fb49ab11418fbd8cb89d6bb16d6b81" target='_blank'>
              Single-cell perturbations decipher ribosomal stress-surveillance regulators in type 2 diabetes
              </a>
            </td>
          <td>
            J. Nan, Xianglong He, Xiaoping Liu, J. Ran, Jiahuan Chen, Pengxiao Li, Dongxue Liu, Yanan Sun, Aijing Shan, Xiuli Jiang, Jing Xie, Weiqing Wang, Guang Ning, Yanan Cao
          </td>
          <td>2026-01-01</td>
          <td>Nature Metabolism</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Tumor cell plasticity drives metastasis and therapy resistance, yet its regulation by oncoprotein dosage dynamics remains poorly understood. In Ewing sarcoma (EwS), variations in EWS::FLI1 (EF) fusion oncoprotein activity have been associated with epithelial-mesenchymal plasticity (EMP). Using degron technology, we precisely modulated endogenous EF in EwS cells and linked phenotypic states to distinct oncoprotein dosages. Strikingly, modest EF depletion promoted a pro-metastatic phenotype that diminished upon near-complete EF loss, revealing a paradoxical effect of submaximal EF inhibition. Nascent RNA-sequencing uncovered distinct gene clusters with heterogenous transcriptional responses to graded EF loss. Genes most sensitive to subtle EF depletion harbored GGAA microsatellites within EF-bound enhancers, while chromatin profiling uncovered candidate cofactors regulating EF-repressed EMP programs. Transient EF depletion followed by rapid restoration, modelling oncoprotein fluctuations, caused persistent dysregulation of genes functionally linked to enhanced extravasation and metastatic burden in preclinical models. This study highlights the therapeutic challenge of incomplete EF elimination, serving a paradigm in which oncoprotein dosage dynamics act as non-genetic drivers of disease progression and reveal novel vulnerabilities of advanced disease. The partial and transient suppression of the EF oncoprotein was found to paradoxically promote Ewing sarcoma metastatic potential through persistent transcriptional dysregulation, establishing the oncoprotein dosage dynamics as a key regulator of metastatic plasticity and therapeutic vulnerability. Graded EF depletion using dTAGV-1 elicited a non-linear phenotypic response, with modest EF loss being sufficient to induce a pro-metastatic state. Integration of acute transcriptional responses with chromatin profiling led to the identification of EF co-regulators. EF restoration following transient depletion resulted in lasting dysregulation of pro-metastasis-related genes and emergence of hybrid epithelial-mesenchymal states, thereby enhancing extravasation and lung metastasis in vivo. Partial or transient EF inhibition may promote disease progression and worsen clinical outcomes, highlighting that therapeutic strategies targeting EF must achieve complete and sustained suppression. Graded EF depletion using dTAGV-1 elicited a non-linear phenotypic response, with modest EF loss being sufficient to induce a pro-metastatic state. Integration of acute transcriptional responses with chromatin profiling led to the identification of EF co-regulators. EF restoration following transient depletion resulted in lasting dysregulation of pro-metastasis-related genes and emergence of hybrid epithelial-mesenchymal states, thereby enhancing extravasation and lung metastasis in vivo. Partial or transient EF inhibition may promote disease progression and worsen clinical outcomes, highlighting that therapeutic strategies targeting EF must achieve complete and sustained suppression. The partial and transient suppression of the EF oncoprotein was found to paradoxically promote Ewing sarcoma metastatic potential through persistent transcriptional dysregulation, establishing the oncoprotein dosage dynamics as a key regulator of metastatic plasticity and therapeutic vulnerability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5051c5027b542c8e38511f6f4e8d71524906c31f" target='_blank'>
              Modelling EWS::FLI1 protein fluctuations reveal determinants of tumor plasticity in Ewing sarcoma
              </a>
            </td>
          <td>
            Veveeyan Suresh, Christoph Hafemeister, Andri Konstantinou, S. Grissenberger, C. Sturtzel, M. Zylka, F. Cidre‐Aranaz, A. Wenninger-Weinzierl, Karla Queiroz, D. Kurek, Martin Distel, Anna C. Obenauf, T. Grünewald, F. Halbritter, H. Kovar, Valerie Fock
          </td>
          <td>2026-01-03</td>
          <td>EMBO Molecular Medicine</td>
          <td>2</td>
          <td>32</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is a highly heterogeneous malignancy with a poor prognosis and limited response to current therapies. Cancer stem cells (CSCs) contribute to this heterogeneity, driving tumor progression, immune evasion, and therapeutic resistance. Epigenetic regulators have emerged as pivotal modulators of CSC phenotypes. The histone methyltransferase SETDB1, a key stemness factor in both embryonic and adult stem cells, catalyzes H3K9 trimethylation (H3K9me3) and acts as a major oncogenic driver in a wide range of solid and hematological malignancies. Yet, its specific role in maintaining stemness and contributing to therapeutic resistance in HCC remains poorly defined. Integrative analyses of bulk and single-cell transcriptomic datasets from public HCC cohorts were performed. Tumors were stratified by SETDB1 expression, followed by pathway enrichment, immune deconvolution, and stemness scoring. Associations with clinical outcomes, immune phenotypes, drug resistance signatures, and chromatin-binding partners (DNMT3A, TRIM28, HDAC1) were also explored. The role of SETDB1 was then examined in vitro, where a stem-like phenotype was induced in HCC cell lines under hypoxic conditions, and patterns of SETDB1 expression and stemness marker levels were assessed following paclitaxel treatment. SETDB1-low tumors were enriched in well-differentiated, immunologically active subclasses and were associated with increased predicted responsiveness to immune checkpoint blockade (ICB), sorafenib, and transarterial chemoembolization. Conversely, SETDB1-high tumors exhibited transcriptional features of dedifferentiation, elevated stemness scores, and enrichment in aggressive molecular subtypes. These tumors also displayed higher TP53 mutation rates, reduced immune infiltration, immune exclusion signatures, and activation of pathways linked to ferroptosis and ABC transporter dysfunction, consistent with resistance to ICB and conventional therapies. SETDB1 expression was also associated with hypoxia-related pathways, suggesting a role in maintaining CSC niches. In vitro, hypoxia-induced stemness coincided with increased SETDB1 levels, whereas paclitaxel treatment decreased both SETDB1 expression and stemness markers. A chromatin risk score incorporating SETDB1 and its partners predicted poor disease-free survival independently of clinical parameters. SETDB1 defines a stemness-enriched, immune-resistant HCC subtype associated with poor outcomes and therapeutic failure. Correlations between SETDB1 expression and stem-like features in vitro suggest a potential role in maintaining CSC phenotypes, supporting its relevance as a biomarker and candidate for epigenetic- and CSC-directed therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c393716a837e5b1ef0091b6385eb92c64be4ab4" target='_blank'>
              The epigenetic regulator SETDB1 as a key component of cancer stem cells and drug resistance in primary liver cancer
              </a>
            </td>
          <td>
            Maël Padelli, Christophe Desterke, Aurore Devocelle, Georges Uzan, Antoinette Lemoine, Julien Giron-Michel
          </td>
          <td>2026-01-06</td>
          <td>Cellular Oncology (Dordrecht, Netherlands)</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Abstract Changes in the copy number of large genomic regions, termed copy number variations (CNVs), contribute to important phenotypes. CNVs are readily identified using conventional approaches when present in a large fraction of the cell population. However, CNVs in only a few genomes are often overlooked but important; if beneficial, a de novo CNV that arises in a single genome can expand during selection to create a population of cells with novel characteristics. While single cell methods for studying de novo CNVs are increasing, we continue to lack information about CNV dynamics in rapidly evolving microbial populations. Here, we investigated de novo CNVs in the genome of the Plasmodium parasite that causes human malaria. The highly AT-rich Plasmodium falciparum genome readily accumulates CNVs that facilitate rapid adaptation. We employed low-input genomics and specialized computational tools to evaluate the impact of sub-lethal stress on the de novo CNV rate. We observed a significant increase in genome-wide de novo CNVs following treatment with an antimalarial compound that inhibits replication. De novo CNVs encompassed genes from various cellular pathways participating in human infection. This snapshot of CNV dynamics emphasizes the connection between replication stress, DNA repair, and CNV generation in this important microbial pathogen.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cc90f4b9d9a1317ece4f68d2ac843d4fc5a32e4" target='_blank'>
              Replication stress increases de novo CNVs across the malaria parasite genome
              </a>
            </td>
          <td>
            Noah Brown, A. Luniewski, Xuanxuan Yu, Michelle Warthan, Shiwei Liu, Julia Zulawinska, Syed Ahmad, Nadia Prasad, Molly Congdon, Webster Santos, Feifei Xiao, J. Guler
          </td>
          <td>2026-01-05</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e4b1e6a28062c925f62b6dc1677788b29bd01df" target='_blank'>
              Alternative Lengthening of Telomeres and CINSARC are interconnected toward non-translocation-related sarcomas progression
              </a>
            </td>
          <td>
            G. Pérot, Clémence Guerriau, Natacha Roussel, E. Sarot, Carine Valle, Pascal Pomiès, Franck Tirode, Delphine Poncet, Frédéric Chibon
          </td>
          <td>2026-01-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Canine osteosarcoma (OS) is an aggressive bone cancer with remarkable similarity to human OS. Despite its value as a comparative oncology model, the drivers of genomic instability remain poorly defined, hindering cross-species insights into disease progression. We applied long-read PacBio HiFi whole genome sequencing to matched tumor and blood samples from four purebred dogs to generate high-resolution maps of somatic structural variants (SVs). Tumor genomes showed extensive rearrangement complexity, with satellite repeat regions, particularly SAT1_CF, enriched near somatic SV breakpoints and occurring within genomic contexts marked by focal hypomethylation. We also identified multiple extrachromosomal DNA (ecDNA) elements carrying oncogenes indicating ecDNA-mediated oncogene amplification. Together, these data provide an integrated genomic and epigenomic view of canine OS, implicating satellite repeats as recurrent substrates for rearrangement and ecDNA as a prominent mode of genome remodeling. This work advances our understanding of OS genome dynamics and establishes a comparative framework for investigating repeat-driven instability and oncogene amplification across species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5300af19b1916e9abe3db71ff1fa34ffceb5e24d" target='_blank'>
              Satellite DNA fragility accompanies complex genome rearrangements and ecDNA oncogene amplification in canine osteosarcomas
              </a>
            </td>
          <td>
            F. Yilmaz, Wonyoung Kong, Sabriya A. Syed, Francis H. O’Neill, Jody T. Lombardi, P. K. Ng, Ching C. Lau, Charles Lee
          </td>
          <td>2025-12-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Clear cell renal cell carcinoma (ccRCC) is an aggressive malignancy associated with single-nucleotide variants in VHL, PBRM1, and SETD2. However, the role of structural variation (SV)—which have broader genomic impacts—and 3D genome architecture in ccRCC remains inadequately understood. Here, we reported a comprehensive molecular characterization. Through multi-omics analysis, we identify novel SV-associated oncogenic targets and reveal multidimensional 3D genome reorganization during ccRCC progression. We elucidate the dynamic interplay between SV and 3D chromatin architecture, demonstrating how structural rearrangements drive oncogenic dysregulation through 3D genome reorganization. Notably, an unrecognized pathogenic enhancer hijacking event is discovered and experimentally validated, leading to constitutive activation of the proto-oncogene SEMA5B. Furthermore, we developed a machine learning-based prognostic framework using enhancer hijacking signatures. Collectively, this work establishes a valuable resource for ccRCC research by elucidating how SVs and 3D genome reorganization collectively drive oncogenesis, and translates these findings into a clinically applicable prognostic tool.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4f8e6235c583e0191c138b2862bedc511ab3f80" target='_blank'>
              Structural variation drives enhancer hijacking via 3D genome disruption in ccRCC
              </a>
            </td>
          <td>
            Yu Dong, Wenjiao Xia, Zitong Yang, Liangliang Ren, Hongru Wang, Yiyang Zhou, Qinchen Li, Zhi Chen, Zhinan Xia, Yichun Zheng, Feifan Wang, Ning He, Bing Cheng, Dongmei Ma, Wei Shao, Wei Guo, Shuwen Wang, Ziqiao Liu, Junxiao Shen, Yiming Qi, Xuke Gong, Juan Jin, Bo Xie, Guixin Zhu, Cheng Zhang
          </td>
          <td>2026-01-06</td>
          <td>NPJ Digital Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Genomic instability is a hallmark of cancer, encompassing both sequence and structural alterations that drive tumor evolution and heterogeneity. The APOBEC3 family of deoxycytidine deaminases has emerged as a major source of mutagenic activity in cancers. R-loops are RNA-DNA hybrids and structural barriers that interfere with replication and transcription. Among the APOBEC3 family, APOBEC3C (A3C) is particularly worthy of attention for its upregulation, driving the DNA replication stress tolerance in response to replication stress-inducing drug gemcitabine. However, the molecular mechanisms of gemcitabine resistance and regulatory circuitries mediated by A3C remain largely unknown, especially in checkpoint-deficient tumors. Initially, we screened that A3C was a putative transcriptional target of p53, and p53-deficient H1299 cells harboring A3C elicited a chemoresistant phenotype upon gemcitabine treatment both in vitro and in vivo. A3C expression enhanced Chk1-dependent S-phase checkpoint activation, thus slowing down replication fork progression and facilitating DNA repair. Pull-down assay and proteomic analysis identified that A3C had a specific interaction with the RNA helicase DDX5, which coordinately played critical roles in R-loop resolution. In contrast to A3C, DDX5 expression attenuated Chk1-dependent S-phase checkpoint activation. Knockdown of DDX5 in A3C-proficient H1299 cells attenuated gemcitabine-induced Chk1 activation and enhanced the therapeutic index of gemcitabine by promoting R-loop accumulation. Therefore, we conclude that A3C/DDX5/R-loop complex may impair the sensitivity of gemcitabine by modulating Chk1 dynamics and DNA replication/damage response machinery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3c0c1bd150bb81f122f34c4b4eaf7affdf65ea8" target='_blank'>
              APOBEC3C coordinates DDX5 in R-loop resolution and dynamic control of Chk1-mediated stress-responsive circuitry as a prerequisite for gemcitabine resistance in p53-deficient cells
              </a>
            </td>
          <td>
            Li Tao, Yang Zhao, Zhuangzhaung Jiang, Shujing Kong, Yanlin Ding, Tengyang Ni, Weimin Wang, Yanqing Liu
          </td>
          <td>2026-01-07</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Colorectal cancer (CRC) progression and therapy resistance are driven in part by metabolic reprogramming and the persistence of cancer stem-like cells (CSCs). The seven in absentia homolog 2 (SIAH2)/with-no-lysine kinase 1 (WNK1) signaling axis has emerged as a potential regulator of these processes, yet its functional role in CRC metabolism and tumor–stroma crosstalk remains incompletely understood. Integrated analyses of The Cancer Genome Atlas–Colon Adenocarcinoma (TCGA-COAD) and Gene Expression Omnibus (GEO, GSE17538) datasets revealed significant upregulation of SIAH2 and WNK1 in CRC tissues, with strong positive correlations to glycolysis- and hypoxia-associated genes, including PFKP, LDHA, BPGM, ADH1A, ADH1B, and HIF-1α. Single-cell and clinical profiling further demonstrated preferential enrichment of SIAH2 in undifferentiated, stem-like tumor cell populations. Functional studies across multiple CRC cell lines showed that SIAH2 silencing suppressed proliferation, clonogenic growth, tumor sphere formation, and cell-cycle progression, whereas SIAH2 overexpression exerted opposite effects. Seahorse extracellular flux analyses established that SIAH2 promotes glycolytic capacity and metabolic flexibility. At the protein level, SIAH2 regulated glycolytic enzymes and WNK1/hypoxia-inducible factor-1α (HIF-1α) signaling, effects that were amplified by cancer-associated fibroblast (CAF)-derived conditioned medium. CAF exposure enhanced SIAH2 expression, CSC spheroid growth, and resistance to fluorouracil, leucovorin, and oxaliplatin (FOLFOX) chemotherapy, whereas SIAH2 depletion effectively abrogated these effects. Collectively, these findings identify the SIAH2/WNK1 axis as a central metabolic regulator linking glycolysis, CSC maintenance, and microenvironment-driven therapy resistance in CRC, highlighting its potential as a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb269b1f5618b40199d4b3cc75e161d29b2f08d9" target='_blank'>
              SIAH2–WNK1 Signaling Drives Glycolytic Metabolism and Therapeutic Resistance in Colorectal Cancer
              </a>
            </td>
          <td>
            Kee-Thai Kiu, Cheng-Ying Chu, Yi-Chiao Cheng, Min-Hsuan Yen, Ying-Wei Chen, N. W. Pikatan, Vijesh Kumar Yadav, Tung-Cheng Chang
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97d63cfbf90a2e544c2d24c8ab605e5d44e8ba24" target='_blank'>
              A Pan-Cancer Single-Cell Compendium of Intratumoural Heterogeneity
              </a>
            </td>
          <td>
            Ido Nofech-Mozes, A. Cook, Tom W. Ouellette, Sara Hafezi-Bakhtiari, Philip Awadalla, Sagi Abelson
          </td>
          <td>2026-01-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Segmental copy-number gains are major contributors to human genetic variation and disease, but how these alterations arise remains incompletely understood. Here, based on the analyses of both experimental evolution and human disease genomes, we describe a general mechanism of segmental copy-number gain from a rearrangement process termed ‘breakage–replication/fusion’. The hallmark genomic feature of breakage–replication/fusion is adjacent parallel breakpoints: two or more rearrangement breakpoints derived from replication of a single ancestral DNA end. We show that adjacent parallel breakpoints are a widespread feature of DNA duplications in human disease genomes and experimental models of chromothripsis. In addition to adjacent parallel breakpoints, breakage–replication/fusion also explains two other patterns of complex rearrangements with unclear provenance: chains of short (≤1 kb) insertions and high-level amplification consisting of inverted segments. Together, these findings revise the mechanistic model for chromothripsis and provide a new conceptual framework for understanding the origin of segmental DNA duplication during genome evolution. This paper introduces breakage–replication/fusion, a genomic rearrangement process underpinning three patterns of copy-number gains found in cancer and other diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/943c5640094a2cf860368d879c898eeb29d9cb6e" target='_blank'>
              A breakage–replication/fusion process explains complex rearrangements and segmental DNA amplification
              </a>
            </td>
          <td>
            Cheng-Zhong Zhang, Carlos Mendez-Dorantes, Kathleen H. Burns, David Pellman
          </td>
          <td>2026-01-01</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7f9dd83e428e2d26b0def31173f1786046f888c" target='_blank'>
              CHD4 epigenetically coordinates genomic instability and immunosuppression to drive pan-cancer progression and confer HDAC inhibitor sensitivity
              </a>
            </td>
          <td>
            Guangxu Fu, Yong Tao, Keyi Feng, Yuxing Chen, Wen Zhang, , Guoda Hu, Yunsheng Ou
          </td>
          <td>2026-02-13</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Genome-wide association studies (GWAS) identified over 600 loci containing single-nucleotide polymorphisms (SNPs) associated with type 2 diabetes (T2D), most of which reside in non-coding regions. Among the set of T2D SNPs, linking causal genome variants to disease risk experimentally has remained a challenge; however, advances in synthetic mammalian genome writing techniques now enable the delivery of multiple haplotypes to human induced pluripotent stem cells (hiPSCs) to create a series of isogenic cell lines that can be differentiated and phenotyped in vitro. Here, to begin efforts in dissecting a T2D GWAS locus, we engineered an NKX6-3/ANK1 gene cluster knockout hiPSC line and introduced a landing pad facilitating the delivery of synthetic haplotype payloads. We built four haplotypes, including several that are not observed in nature, containing risk SNPs spanning the NKX6-3/ANK1 gene cluster using a method called “variant Switching Auxotrophic markers for Integration” (vSwAP-In), and integrated them precisely into hiPSCs. NKX6-3/ANK1 deletion blocked pancreatic progenitor and skeletal muscle differentiation, suggesting that NKX6-3 and ANK1 are required for early pancreatic and skeletal muscle development, and perhaps related to the existence of two nonoverlapping sets of SNPs in linkage disequilibrium that associate with the expression of the two adjacent genes. When NKX6-3/ANK1 T2D “Risk” haplotypes were reintroduced, skeletal muscle and pancreatic progenitor differentiation capabilities were restored. ANK1 expression was elevated in the ANK1 Risk and All-Risk haplotypes compared to the NKX6-3 Risk and Non-Risk haplotypes, establishing a functional experimental platform to examine risk SNP clusters in their native contexts. Overall, this work establishes a platform for the dissection of GWAS loci using synthetic haplotype genomics in hiPSCs. Significance Statement Genome-wide association studies have been used to identify disease-associated SNPs; however, most SNPs lie in non-coding regions, making functional experimentation difficult to perform. Using vSwAP-In, a yeast-based DNA variant-building method, and mSwAP-In, a mammalian genome engineering approach, we establish a platform for functional GWAS dissection in hiPSCs. This platform allows us to build DNA harboring virtually any combination of disease-risk SNPs, allowing for functional characterization of SNPs without the limitations of linkage disequilibrium. We demonstrate this approach using a Type 2 diabetes GWAS gene cluster, NKX6-3/ANK1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f19a7bbc6f3616a799aaa84ab7eb946f7cda112e" target='_blank'>
              Genome writing and Targeted Delivery of the NKX6-3/ANK1 gene cluster and its Type 2 Diabetes GWAS Variants to Human iPSCs
              </a>
            </td>
          <td>
            Noor Chalhoub, Arushi Varshney, Weimin Zhang, Skyler Uhl, J. M. Laurent, Colleen McLoughlin, Hannah J. Ashe, Xingrui Mou, Nathan Dale, Kiran Ramnarine, Dan Paull, Jordan Goldberg, Matthew T. Maurano, Ran Brosh, David Fenyö, Filippo Cipriani, Stephen C. J. Parker, J. Boeke
          </td>
          <td>2026-01-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>114</td>
        </tr>

        <tr id="Emergence of therapeutic resistance is a critical clinical challenge in cancer treatment, contributing to treatment failure, disease relapse, and overall poor prognosis. Adaptive therapy (AT), a resistance management strategy, aims to address this issue by selectively applying therapeutic pressure to promote competition between therapy-sensitive and therapy-resistant clones, allowing for long-term control of tumor burden. AT relies upon the assumption that resistance comes at some fitness cost in the absence of therapy. Is that assumption justified? We conducted a systematic review of the literature on experimental tests of the fitness cost of therapeutic resistance. We conducted a search for peer-reviewed papers that fulfilled the following selection criteria: (i) experiments of direct competition, (ii) between therapy-resistant and therapy-sensitive clones, (iii) in a therapy-free environment. We found 47 experiments that matched those criteria. Of those experiments, approximately two-thirds (68%) found a fitness cost to resistance in a competitive environment. Of all pooled features from the studies reviewed, we found that the resistance characteristic was most significantly associated with whether resistant clones exhibited a fitness advantage in competition (p=0.0147). Further, we identify complex ecological interactions that may influence the behavior of the cancer cell population without selection by therapeutic pressure. Predicting which resistance characteristics can be exploited therapeutically with AT and identifying potential methods of modulating the costliness of the resistant phenotype may be critical to future improvements in cancer therapy. Lay Summary When cancer cells evolve to no longer respond to a cancer therapy, do they pay a cost for that invulnerability? We reviewed experiments where cancer cells resistant to a therapy were pitted against cancer cells that were still sensitive to the therapy. Approximately ⅔ of the experiments showed that the sensitive cells could out-compete the resistant cells, implying that the latter were paying some cost for their ability to resist the therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89dbb398dccdbfab52d6b0cd0e68f176e99b3baf" target='_blank'>
              A Systematic Review of Evidence for the Cost of Therapeutic Resistance in Cancer
              </a>
            </td>
          <td>
            Bailey Kane, Lauren Mestas, Madds Garza, Tiara Soesilo, Meghan Hufford, Harley Richker, Carlo C. Maley
          </td>
          <td>2025-12-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/638cde0ab64b6b068ce716defb37f6b2cb4ab7b9" target='_blank'>
              Unraveling the molecular mechanisms underlying spontaneous multipolar mitosis through CIN-seq
              </a>
            </td>
          <td>
            Pin-Rui Su, Ting-Chun Chou, M. T. López-Cascales, Junfeng Hunag, Tsai-Ying Chen, Sjoukje van Wieren, Changwei Li, Cecile Beerens, Li You, J. Storteboom, M. Chien
          </td>
          <td>2026-01-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1dbaf07c5f2dda600fecd5fe35228a64df3c729" target='_blank'>
              M5C-driven stabilization of SERPINB5 promotes cervical cancer progression and chemotherapy resistance
              </a>
            </td>
          <td>
            Jiejie Liu, Limin Zhou, Peipei Yao, Nan Zhang, Xiao Guo, Fei Chen, Shimin Yang, Xin Du, Hongyun Wang, You Zhou, Yu Chen, Li Zhou
          </td>
          <td>2026-02-11</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Lung cancer remains the leading cause of cancer−related mortality. Single−cell RNA sequencing (scRNA−seq) enables high−resolution mapping of tumor microenvironment, but how malignant cell states connect to patient outcomes and therapeutic vulnerabilities is not fully understood. Methods We analyzed scRNA−seq profiles from 13 primary lung tumors and matched normal lung tissues (48,149 cells). Malignant versus non−malignant epithelial cells were defined by inferCNV, co−expression programs resolved with Hotspot, transcription factor regulons inferred by pySCENIC, and differentiation potential estimated by CytoTRACE and diffusion pseudotime. Key malignant programs were cross−checked in an independent LUAD scRNA−seq cohort (GSE131907). Program signatures were projected into TCGA−LUAD to build an eight−gene LASSO−Cox risk model, which was further validated in two external NSCLC/LUAD cohorts (GSE30219 and GSE31210). Immune contexture was inferred by CIBERSORT and TIDE, drug sensitivities predicted with pRRophetic, and the function of FAM189A2 tested by knockdown and overexpression in A549 and NCI−H23 cells. Results We delineated 12 major cell populations and six malignant epithelial subclusters, including an early GPRC5A+ subpopulation with high developmental potential. Malignant programs corresponding to cell−cycle and inflammatory states (Modules 8, 14 and 16) were associated with worse overall survival, whereas Module 15, preferentially active in the GPRC5A+ state, was linked to better outcomes. An eight−gene signature (LAMA5, ACTB, B4GALT1, KLF5, KRT18, FAM189A2, SLC34A2, S100A11) robustly stratified patients into high− and low−risk groups across TCGA−LUAD and two validation cohorts. High−risk tumors displayed an immune−enriched yet matrix−restrained microenvironment with higher CD8+ T cells, whereas low−risk tumors showed greater predicted sensitivity to Aurora kinase, IGF−1R, mTOR and TGF−β inhibition. FAM189A2 was down−regulated in tumors, higher expression predicted better survival, and bidirectional perturbation in A549 and H23 cells demonstrated that loss of FAM189A2 enhanced, while overexpression attenuated, migration and invasion in vitro. Conclusions Integrating single−cell and bulk transcriptomes links malignant epithelial state programs to prognosis, yields a practical eight−gene risk model validated in multiple LUAD cohorts, and nominates FAM189A2 as a putative tumor suppressor and potential biomarker in lung cancer. These findings suggest testable strategies for risk stratification and therapy selection that warrant prospective evaluation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9149373826a7f4a30a6edb092024413c1a5945d" target='_blank'>
              Integration of single cell and bulk transcriptomic analyses identifies FAM189A2 as a key prognostic gene in lung cancer
              </a>
            </td>
          <td>
            Peng Guo, Mengqi Xiang, Jing Chen, Huachuan Zhang
          </td>
          <td>2026-01-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Whole-genome duplication (WGD) fuels tumor evolution and therapy resistance, yet the molecular mechanisms governing the switch from the canonical mitotic cell cycle to the endoreplication cycle remain unclear. Here, we combine single-cell proteomics, manifold learning, and live-cell imaging to map the intersection of the mitotic and endoreplication cycles in breast cancer cells exposed to genotoxic agents. We identify two distinct routes to WGD driven by distinct p21 dynamics. High p21 induction induces G2 exit and endocycling, whereas insufficient p21 permits mitotic entry followed by slippage and endomitosis. This therapy-induced switch acts as a facultative stress response, generating drug-resistant polyploid populations that propagate genomic instability through replication stress and the generation of replication-competent micronuclei. Both paths to WGD converge on a common polyploid G0 state dependent on cyclin D1:CDK4/6 activity to complete the transition to the endoreplication cycle, revealing a shared vulnerability. Sequential treatment with genotoxic agents followed by CDK4/6 inhibitors preserves the cytotoxic efficacy of DNA-damaging drugs while simultaneously blocking entry into the endoreplication cycle and WGD-driven evolutionary rescue. These findings reveal the molecular rules governing the switch from the mitotic to endoreplication cycle and highlight the potential of WGD-blocking drugs as adjuvant therapies to inhibit drug resistance and suppress tumor evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64dace37f687cf3f6f5d90e43b0621daaa3dd0b0" target='_blank'>
              Distinct p21 dynamics drive alternative routes to whole-genome duplication through a common CDK4/6-dependent polyploid G0 state
              </a>
            </td>
          <td>
            Yian Yang, Anuraag Bukkuri, Dante Poe, Janet McLaughlin, Sihan Hao, Daniel D. Brown, Katarzyna M. Kedziora, Wayne Stallaert
          </td>
          <td>2026-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d828c339f540d046b248d2b7a0e411b51219090" target='_blank'>
              An H4K12la/CEBPB-AKR1C2 signaling axis modulates the mTOR pathway to regulate cisplatin resistance in lung cancer
              </a>
            </td>
          <td>
            Wenjing Wang, Qiang He, Tianfei Fan, Yang Xiong, Yimin Xiong, Qinhui Liu, Na Yang, Yining Xu, Yanping Li, Jinhan He
          </td>
          <td>2026-01-07</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Amino acid metabolism plays a crucial role in cancer progression and invasion by modulating immune responses within the tumor microenvironment and tumor immune response. Transmembrane transport of cellular amino acids is mediated by members of the cationic amino acid transporters SLC7 family, a group of cationic amino acid transporters. Dysregulation of SLC7 family genes can disrupt systemic homeostasis, contributing to a variety of diseases, including multiple cancer types. Additionally, SLC7 family members can alter amino acid metabolic plasticity, further influencing tumor adaptation and progression. However, the precise impact of SLC7 family genes within the cancer microenvironment remains largely unexplored. This study analyzed the expression patterns, genomic alterations (single nucleotide variants, copy number variations, methylation levels), pathway activations, and clinical and subtype associations of SLC7 family genes across The Cancer Genome Atlas (TCGA) pan-cancer cohort. SLC7 activity was quantified using the ssGSEA function of the “GSVA” package to generate the SLC7 score, which was found to correlate with oncogenic and immune-related pathways, including mTORC1 signaling, interferon-gamma response, and inflammatory pathways. In vitro validation was conducted using multiple cancer cell lines (A549, H1299, HCT116, FaDu, Panc-1) and counterparts (BxPC-3, SW480, H460, BEAS-2B, FHC, HaCaT). Gene knockdown was achieved by siRNA transfection, followed by quantitative PCR, Western blotting, and functional assays assessing proliferation, invasion, and ferroptosis (iron and GSH levels) under erastin treatment. SLC7 family genes showed significant associations with immune infiltration and poor prognosis across various cancers. The SLC7 score demonstrated strong positive correlations with malignant and immune-related pathways, such as mTORC1 signaling, interferon-gamma response, and inflammatory response across the pan-cancer cohort. Elevated SLC7 scores were also associated with enhanced immune-related characteristics, and SLC7 genes, particularly SLC7A8 and SLC7A9, emerged as promising predictive markers for response to immune checkpoint inhibitor therapy (Atezolizumab) in cancer patients. Among SLC7 family genes, SLC7A5 expression was markedly elevated in KRAS-mutant tumor cell lines and was required for KRAS-driven proliferation and invasion. SLC7A11 was highly expressed in multiple tumor types and protected cancer cells from ferroptosis by maintaining intracellular cystine uptake and redox homeostasis. Individuals exhibiting elevated SLC7 scores and aberrant expression of SLC7 family genes demonstrate significant immune-inflammatory tumor immune microenvironment (TIME) characteristics and predictive potential for immunotherapy. Thus, the SLC7 family genes have the potential to serve as valuable biomarkers and therapeutic targets for cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a15f37f25792e120e4c9d88c14c57b67424b4e09" target='_blank'>
              Integrative pan-cancer analysis reveals the prognostic and immunological significance of SLC7 family cationic amino acid transporters
              </a>
            </td>
          <td>
            Gang Peng, Hao Peng, Zhe Shao, Liangqi Jiang, Mingrui Li, Yang Li
          </td>
          <td>2026-01-08</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="High-grade serous ovarian carcinoma (HGSOC) is characterised by profound genomic instability and limited durable responses to standard therapy, leading to poor prognosis. The use of next-generation sequencing technologies has improved understanding of its molecular landscape, revealing consistent Tumour Protein p53 (TP53) mutations, homologous recombination defects, pathway alterations, and epigenetic dysregulation. Such genomic profiling now underpins the classification criteria between the ovarian cancer subtypes described by the Cancer Genome Atlas. Widespread chromosomal instability and pathogenic variants in multiple genes distinguish HGSOC from other subtypes of ovarian cancer and, further, from low-grade serous ovarian cancer. Importantly, the new-found understanding of the genomic landscape of HGSOC guides the use of platinum-based chemotherapies and Poly(ADP-ribose) Polymerase (PARP) inhibitors, with homologous recombination deficiency emerging as a cancer vulnerability that enhances treatment response. A combined multi-omics approach integrates transcriptomics, proteomics, metabolomics, and epigenomics to further the understanding of the characteristics, therapeutic targets and treatment resistance within HGSOC. Despite these advances, major challenges persist, including intratumoural heterogeneity and the poor diversity of genomic datasets. Artificial Intelligence (AI) technology, Clustered regularly interspaced short palindromic repeats (CRISPR)-based gene editing, neoantigen-guided immunotherapy and ovarian cancer vaccination indicate a promising future for genomics-guided interventions and support the integration of genomics within multi-omic approaches to improve HGSOC outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63375259d1ee4ee2c082f84da6b555791abc91c8" target='_blank'>
              High-Grade Serous Ovarian Carcinoma in the Genomics Era: Current Applications, Challenges and Future Directions
              </a>
            </td>
          <td>
            M. Lewis, C. Caricato, Hannah Roberts, Subhasheenee Ganesan, Nadia Amel Seksaf, E. Maniati, Michail Sideris
          </td>
          <td>2026-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/539b2ee1d2d15c9295867b71d2600ec2c2b281c5" target='_blank'>
              MELK controls tumor metabolism to promote resistance to melanoma therapy
              </a>
            </td>
          <td>
            A. C. B. Sant’Anna-Silva, P. Abbe, Sophie Pagnotta, N. Amoedo, Michael D. Torno, Fanny Bonnard, T. Botton, Frédéric Larbret, Cyril Ronco, H. Montaudié, I. Ben-Sahra, C. Robert, Thierry Passeron, Rodrigue Rossignol, Michaël Cerezo, Stéphane Rocchi
          </td>
          <td>2025-12-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Muscle defects are common in human developmental disorders and often cause severe functional impairment. These defects arise from intricate tissue crosstalk and rare genetic mutations, underscoring the need to systematically identify cell-autonomous mechanisms regulating human myogenesis. Here we show a rationally designed, high-throughput genetic screening platform that integrates human myoblast models, customized CRISPR libraries, and a split-toxin strategy that enables quantitative selection of fusion-defective myocytes. Leveraging this platform, our initial screen uncovers a large group of hits essential for human myoblast fusion. The majority of these hits converge into 23 protein complexes. Notably, mutations in 41 screen hits are associated with human diseases marked by abnormal skeletal-muscle morphology. Applying a new single-cell CRISPR & RNA-seq approach, we show that majority of these hits control human myoblast fusion as well as influence early-stage myogenic differentiation. This work establishes a scalable approach to identify cell-autonomous regulators of human muscle differentiation and fusion. Here, the authors present a high-throughput genetic screen to identify genes that regulate human myoblast fusion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8dfb8947eba2e1b9a7170329aa43575f7a6f193f" target='_blank'>
              Development of a split-toxin CRISPR screening platform to systematically identify regulators of human myoblast fusion
              </a>
            </td>
          <td>
            Haifeng Zhang, Renjie Shang, Zheng Zhang, Min Zhou, Anne Bigot, Y. Cai, Yanran Zhao, Yushu Wang, A. Deshmukh, E. Kudryashova, D. Kudryashov, Cuiyu He, Vincent Mouly, Pengpeng Bi
          </td>
          <td>2026-01-15</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>25</td>
        </tr>

        <tr id="Abstract Introduction Cancer cell proliferation occurs within the context of persistent genomic instability. In this review, we propose the RS–CCD–CIN axis as a systems-level framework in which replication stress (RS), cell cycle deregulation (CCD) and chromosomal instability (CIN) form an interdependent triad that shapes tumour evolution. This axis represents a constrained metastable state in which genomic instability is tolerated and buffered. The objective of this review is to synthesize the current understanding of how the RS–CCD–CIN axis contributes to tumour heterogeneity, adaptability and therapy response. Discussion Evidence indicates that RS, CCD and CIN operate as a dynamic, interconnected network rather than as independent processes. Replication stress induces DNA damage and mutagenesis, while partial checkpoint disruption permits cells with unresolved lesions to proliferate. Chromosomal instability generates both structural and numerical alterations, contributing to intratumoural heterogeneity. Together, these processes facilitate adaptation to environmental and therapeutic pressures. Extrachromosomal DNA, micronuclei formation and cytosolic DNA signalling, including the cGAS–STING pathway, connect genomic instability to adaptive responses and immune modulation. Single-cell and spatial profiling reveal temporal and spatial variability in RS, CCD and CIN states, highlighting the limitations of static biomarkers. Therapeutically, targeting individual components often yields limited durability, whereas approaches that simultaneously perturb multiple aspects of the RS–CCD–CIN axis may improve clinical outcomes. Conclusions This review highlights the RS–CCD–CIN axis as a fragile and metastable architecture that supports cancer evolution, while also being susceptible to collapse. A deeper understanding of this interconnected framework may inform the development of therapeutic strategies and enhance the management of resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/294582a14e94664c9f90d67cf0b0db4402660bea" target='_blank'>
              A system-level metastable model of cancer evolution: integrating replication stress, cell cycle deregulation and chromosomal instability
              </a>
            </td>
          <td>
            Dongjing Ma, Jinbin Wang, Y. M. Chen, Ling Yao, Qianya Wei, Jianjun Wu
          </td>
          <td>2026-01-28</td>
          <td>Annals of Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Lineage plasticity, the ability of cancer cells to alter their differentiated state through transcriptional and epigenetic reprogramming, has emerged as a key mechanism of therapeutic resistance across cancers. This adaptive process can manifest in multiple ways, including epithelial–mesenchymal transition, acquisition of stem-like features, and histological transformation, the most striking and clinically apparent example. In EGFR-mutant lung adenocarcinoma (LUAD), lineage plasticity is increasingly recognized as a prevalent mechanism of acquired resistance to tyrosine kinase inhibitors (TKIs). Among its visible manifestations, histologic transformation into small-cell lung cancer (SCLC) is the most frequent, while squamous transformation and other phenotypic shifts also occur. Transformed tumors typically retain the initiating EGFR mutation but lose EGFR dependence, acquire neuroendocrine features, and display aggressive clinical behavior with poor clinical outcomes compared with both de novo SCLC and non-transformed LUAD. Recent studies show that plasticity arises through combined genomic, transcriptomic, and epigenetic reprogramming, often foreshadowed by molecular alterations before overt histological change. Spatial and single-cell profiling reveal heterogeneous trajectories and intermediate states, while functional models and multi-omics approaches have begun to identify therapeutic vulnerabilities distinct from both de novo EGFR-mutated SCLC and classical EGFR-mutated LUAD. Thus, lineage plasticity, whether manifested as histologic transformation or through more subtle epigenetic reprogramming, represents a formidable resistance mechanism in NSCLC. Defining its molecular basis and temporal dynamics will be essential for early detection, prognostication, and the development of tailored therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99eda567a0f2b2f5e4b0555a0ae32377856dcb42" target='_blank'>
              Lineage Plasticity and Histologic Transformation in EGFR-TKI Resistant Lung Cancer
              </a>
            </td>
          <td>
            Li Yieng Eunice Lau, A. Skanderup, Aaron C. Tan
          </td>
          <td>2025-12-31</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="CRISPR-Cas9 has become a popular genome editing tool for biomedical research and drug development due to its capability to enable precise correction or integration of genetic mutations in the genome. However, precise genome editing competency varies dramatically between cell types depending on their capabilities for DNA damage. In this proof-of-concept study, we took the example of HepG2 and MCF7 to show that omics profiling identifies bottlenecks that are associated with poor precise knock-in (KI) efficiency in hard-to-engineer cells. These bottlenecks include previously described factors such as the predominance of non-homologous end joining (NHEJ) repair and impaired homologous recombination (HR) capability, but also reveals apoptotic priming status of the cells as a limiting factor. Upon further comparative analysis between HepG2 and MCF7 cells, we pinpointed and validated the proliferating cell nuclear antigen (PCNA) as a target to overexpress to enhance precise KI efficiency in MCF7. Overall, we describe how employing a multi-omics approach to characterize cell models of interest can facilitate an in-depth understanding of their editability molecular signature, empowering us to manipulate the activity of key pathways for precise editing, and therefore increase efficiency of desired editing outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a93f75f6b65fd744e6a6a08cd46d5295f2abfb1" target='_blank'>
              Omics-aided design genome editing strategy for challenging human immortalized cell models
              </a>
            </td>
          <td>
            Patricia Mendoza-Garcia, Benjamin Keith, Markus Nordberg, Ella Quist, Cristina Ferrás, Ghaith M. Hamza, Ramy Elgendy, Stephanie Ashenden, Jordi Chi, Natalie R. van Zuydam, Neil Hattersley, Xiang Zhang
          </td>
          <td>2026-02-12</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Genetic factors driving the development of pediatric Acute Lymphoblastic Leukemia (ALL) remain incompletely understood. While epigenetic dysregulation through histone methyltransferases and demethylases has emerged as a critical oncogenic mechanism, their specific contributions to pediatric ALL require systematic investigation. In this study, we systematically investigated genetic variants in histone methyltransferases and demethylase found in top 50 mutated genes from TARGET ALL Phase 2 cohort. Single nucleotide variations were functionally annotated using PredictSNP and I-Mutant2.0, followed by survival analysis and pathway enrichment studies comparing mutated versus wild-type patients. KMT2D, NSD2, SETD2, EZH2 (methyltransferases), and KDM6A (demethylase), ranked among the most frequently mutated genes. Multiple frameshift and nonsense mutations were identified in these genes, likely resulting in truncated proteins and loss of function. Critically, missense variants in EZH2 (F145C, R679C, P132T, D652G, A576D, S280C, Y728H, C566S), KDM6A (G1242D), NSD2 (E1099K, D1125H), and SETD2 (G2170D, R1592Q) were computationally predicted as deleterious and destabilizing. Although survival analysis revealed no statistically significant differences (p = 0.37), mutation carriers exhibited a 44% increased mortality risk, suggesting clinical relevance despite limited sample size (n = 35). Gene set enrichment analysis of differentially expressed genes revealed significant activation of hematopoietic cell lineage pathways in mutated patients. Our findings underscore the importance of histone methyltransferase and demethylase gene variants in pediatric ALL, particularly novel mutations affecting protein function and stability that remain poorly characterized in the literature. This study identifies candidate mutations that warrant further functional and clinical investigation for their role in disease progression and therapeutic targeting.
HIGHLIGHTS

KMT2D, NSD2, SETD2, EZH2, and KDM6A are identified among the top 50 most frequently mutated genes in pediatric ALL
KMT2D exhibits the highest mutational burden in pediatric ALL, predominantly harboring frameshift and nonsense alterations
KMT2D, NSD2, SETD2, EZH2, and KDM6A genes show consensus deleterious predictions across multiple in silico tools, with 18 mutations representing high-priority candidates for experimental validation
Structural stability analysis reveals critical destabilizing mutations in EZH2 (F145C, R679C, P132T, D652G, A576D, S280C, Y728H, C566S), KDM6A (G1242D), NSD2 (E1099K, D1125H), and SETD2 (G2170D, R1592Q)

GRAPHICAL ABSTRACT">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b15b6568cbd53c364a5d95d54d4bb823b4c767ac" target='_blank'>
              Histone Methyltransferase and Demethylase Gene in Pediatric Acute Lymphoblastic Leukemia: A Molecular Insights
              </a>
            </td>
          <td>
            N. Aziz, Y. N. Patria, A. Wahdi, Eddy Supriyadi, Dewajani Purnomosari
          </td>
          <td>2026-01-01</td>
          <td>Trends in Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) is highly lethal, often developing rapid chemo-resistance and progression after initial response to platinum-based chemotherapy. The role of DNA methylation in driving this progression remains poorly understood. Here, genome-wide methylation profiling via methylated DNA immuno-precipitation sequencing (MeDIP-seq) reveals hypermethylated and hypomethylated regions in both SCLC tissue DNA and plasma cell free DNA (cfDNA). A hypermethylated region in the promoter of RASA4 gene, common to SCLC tumor DNA and progression-associated cfDNA, is identified. Pharmacologic and, more specifically, targeted demethylation reactivate RASA4 expression in SCLC. The down-regulation of RASA4, a negative regulator of Ras signal, leads to the activation of Ras-MAPK pathway in SCLC. RASA4 overexpression inhibits SCLC growth, invasion and chemo-resistance, whereas its knockdown promotes these malignant phenotypes by enhancing epithelial-mesenchymal transition (EMT) and stemness. We further demonstrate that RASA4 negatively regulates the EMT-related protein SERPINE2, and directly interacts with it. Immunohistochemistry analysis of clinical specimens validates that RASA4 is significantly down-regulated in SCLC tumors, and its low expression correlates with SCLC patients’ poor survival. Taken together, the epigenetic silencing of RASA4 drives SCLC progression through the induction of EMT and stemness. Our findings underscore the potential of RASA4 as a diagnostic biomarker and therapeutic target. Promoter hypermethylation-mediated silencing of the tumor suppressor RASA4 activates Ras signaling in small cell lung cancer, thereby driving tumor proliferation, metastasis, and chemoresistance, and revealing its role as a prognostic biomarker.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3de429a7817141d17bab7346b3911bc733e9f80d" target='_blank'>
              Integrated methylome analysis identifies an epigenetically silenced tumor suppressor RASA4 in small cell lung cancer
              </a>
            </td>
          <td>
            Meng Fu, Qizhi Zhu, Jian Qi, Xiaopeng Zhang, Zhipeng Wang, Jialiang Wang, Xiang Huang, Zongtao Hu, Wei Wang, Bo Hong, Hongzhi Wang
          </td>
          <td>2025-12-31</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bf612b75850cf10e9bb17c10ec3f99a040c4e2d" target='_blank'>
              Lineage plasticity: a new dilemma in lung cancer treatment
              </a>
            </td>
          <td>
            Cainan Huo, Yongxuan Wang, Rujia Li, Xiaofei Xie, W. Lyu, Dingke Chen, Yanqin Sun
          </td>
          <td>2026-02-04</td>
          <td>Molecular Biology Reports</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is the second most frequently diagnosed cancer worldwide and represents a major challenge for public health. Despite advances in molecular profiling, important gaps remain in our understanding of tumorigenesis and the regulatory mechanisms underlying CRC progression. The most widely adopted classification system is the Consensus Molecular Subtypes (CMS), which stratifies CRC into four biologically distinct subtypes. We investigated the role of A-to-I RNA editing across CMS subtypes in a cohort of 100 CRC patients at various disease stages. Bulk RNA-seq data were analyzed using REDItools to detect editing events, focusing on both recoding sites and edits within repetitive elements, such as Alu sequences. Furthermore, expression levels of the ADAR enzyme family were assessed, and deconvolution analyses were performed on single-cell RNA-seq data from an independent cohort of stage II CRC patients to characterize editing activity within the tumor microenvironment (TME). Competitive endogenous RNA (ceRNA) networks, specific to each CMS subtype, were constructed based on editing events in repetitive elements. A multivariate Cox proportional hazards model was applied to evaluate associations with overall survival (OS). We observed statistically significant differences in ADARB1 expression across CMS subtypes. Single-cell RNA-seq data revealed subtype-specific distribution patterns of ADAR enzymes within the TME. Analysis of editing events showed subtype-specific signatures in both known cancer-related genes (e.g., COPA, CADPS, IGFBP7) and novel candidates (ZNF552, RALGPS1). Editing in repetitive elements informed the construction of distinct ceRNA networks for each CMS subtype, suggesting different post-transcriptional regulatory mechanisms. Survival analysis identified three variables significantly associated with OS, independent of CMS classification and clinical stage: ADARB1 expression, and editing events in NOP14-AS1 (chr4:2960236; p = 0.036; HR = 0.0069), previously linked to 5-FU sensitivity, and ST7-AS2 (chr4:117120557). This study underscores the biological relevance of RNA editing in CRC, highlighting its impact on chemoresistance, the tumor microenvironment, and subtype-specific gene regulation. Our findings suggest that RNA editing represents a critical post-transcriptional regulatory layer in CRC and holds potential as a biomarker and therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/289a3e685e9081e2037f2e82b68ea443c3971d6f" target='_blank'>
              Biological and prognostic relevance of A-to-I RNA editing across consensus molecular subtypes of colon cancer
              </a>
            </td>
          <td>
            Dario Monaco, D. Traversa, E. Mattioli, F. A. Zito, Grazia Cristiani, Francesca Buono, Sabina Delcuratolo, Tiziana Guarino, R. Pinto, Antonia Lasorella, R. Lacalamita, Ernesto Picardi, Stefania Tommasi, S. De Summa
          </td>
          <td>2026-01-06</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42d6ede17df58e0c555e75494899b3a936c6c0ed" target='_blank'>
              Fully T2T pedigree assemblies reveal genetic stability and epigenetic plasticity of human centromeres across inheritance and cell-fate transitions
              </a>
            </td>
          <td>
            Shihua Dong, Xiaoyun Xing, Monika Cechova, Hailey Loucks, Selvamani Vijayalingam, Amber Neilson, M. Sentmanat, Juan Macias, Tianjie Liu, Z. Dong, Benpeng Miao, Wenjin Zhang, Chad Tomlinson, Heather Schmidt, Edward A. Belter, Ming Hu, Xiaoxia Cui, N. Stitziel, K. Miga, Ting Wang
          </td>
          <td>2026-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e9f264c9beccf9f16f96fab6b533051457f1ceb" target='_blank'>
              Acquisition of drug-resistance induces cytokine reprogramming in a human leukemic cell line. Role of p38 MAPK.
              </a>
            </td>
          <td>
            Paula Cadenas-Garrido, A. García-Serna, Antonio Parrado, Javier Martínez-García, Marcela Herrera-Suarez, M. Canovas, Teresa de Diego Puente, Miguel Saceda, Pablo Pelegrin, M. S. Díaz-Carrasco, E. Martín-Orozco
          </td>
          <td>2026-01-29</td>
          <td>Cell communication and signaling : CCS</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c2f5392a871706eceb6c7ad5228a2907ac09eeb" target='_blank'>
              Association of copy number alterations with the immune transcriptomic landscape in cancer
              </a>
            </td>
          <td>
            S. Loipfinger, A. Bhattacharya, C. G. Urzúa-Traslaviña, M. V. van Vugt, M. de Bruyn, Rudolf S. Fehrmann
          </td>
          <td>2026-01-17</td>
          <td>npj Systems Biology and Applications</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Multiple myeloma (MM) remains incurable despite advances in treatment options. Although tumor subtypes and specific DNA abnormalities are linked to worse prognosis, the impact of immune dysfunction on disease emergence and/or treatment sensitivity remains unclear. We developed an Immune Atlas of MM by generating profiles of 1,397,272 single cells from the bone marrow (BM) of 337 newly diagnosed participants and characterized immune and hematopoietic cell populations. Cytogenetic risk-based analysis revealed heterogeneous associations with T cells of BM, with 17p13 deletion showing distinct enrichment of a type 1 interferon signature. The disease progression-based analysis revealed the presence of a proinflammatory immune senescence-associated secretory phenotype in rapidly progressing participants. Furthermore, signaling analyses suggested active intercellular communication involving a proliferation-inducing ligand and B cell maturation antigen, potentially promoting tumor growth and survival. Lastly, using independent discovery and validation cohorts, we demonstrated that integrating immune cell signatures with known tumor cytogenetics and individual characteristics significantly improves stratification for the prediction of survival. Pilcher et al. present a single-cell transcriptomics-based immune atlas of participants with newly diagnosed multiple myeloma, reporting on the association of cell types, gene expression and intercellular interactions with disease progression phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f88dd8011aa8686433c4d02b3a0e4dfd59a9255" target='_blank'>
              A single-cell atlas characterizes dysregulation of the bone marrow immune microenvironment associated with outcomes in multiple myeloma
              </a>
            </td>
          <td>
            William Pilcher, Lijun Yao, Edgar Gonzalez-Kozlova, Yered H. Pita-Juárez, Dimitra Karagkouni, Chaitanya R. Acharya, Marina E. Michaud, Mark Hamilton, Shivani Nanda, Yizhe Song, Kazuhito Sato, Julia Wang, Sarthak Satpathy, Yuling Ma, Jessica Schulman, D. D’souza, R. Jayasinghe, Denis J Ohlstrom, Katherine E. Ferguson, Giulia Cheloni, Mojtaba Bakhtiari, Nick Pabustan, K. Nie, J. Foltz, Isabella Saldarriaga, Rania Alaaeldin, Eva Lepisto, Rachel Chen, M. Fiala, Beena E. Thomas, April Cook, Junia Vieira Dos Santos, Chiang I-ling, Igor Figueiredo, J. Fortier, Michael Slade, Stephen T. Oh, Michael P. Rettig, Emilie I. Anderson, Ying Li, Surendra Dasari, Michael A. Strausbauch, Vernadette A. Simon, , I-ling Chiang, N. Kalavros, Jennifer Rogers, T. Dawson, Brian H. Lee, G. Kelly, Laura A Walker, Nicolas F. Fernandez, John Leech, Jarod Morgenroth-Rebin, K. Angeliadis, Matthew A. Wyczalkowski, Song Cao, Omar Ibrahim, Roderick Lin, T. Fehniger, Andrew J. Houston, E. Radkevich, Adeeb Rahman, Zhihong Chen, A. Laganà, John F. DiPersio, J. Rosenblatt, S. Kim-Schulze, S. Lonial, Shaji K. Kumar, Swati S. Bhasin, T. Kourelis, M. Dhodapkar, Ravi Vij, D. Avigan, H. Cho, George Mulligan, Li Ding, Sacha Gnjatic, Ioannis S. Vlachos, Manoj K. Bhasin
          </td>
          <td>2026-01-01</td>
          <td>Nature Cancer</td>
          <td>0</td>
          <td>112</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e78908da394712537095f77c8b8136eb7ecf216" target='_blank'>
              Evolution of oncogene amplification across 86,000 cancer cell genomes
              </a>
            </td>
          <td>
            J. Lee, Sohrab Salehi, Matthew A. Myers, , Melissa A. Yao, Duaa H. Al-Rawi, , Eric G. Sun, Kerstin Thol, Seongmin Choi, Eliyahu Havasov, Asher Preska Steinberg, Michelle Wu, Nicole Rusk, Caitlin Timmons, Cheryl Zi Jin Phua, Stephen Martis, Neeman Mohibullah, P. Manoj, E. Redin, Á. Quintanal-Villalonga, P. Razavi, Sam Aparicio, Natasha Rekhtman, Viviane Tabar, Mark M. Awad, H. Yu, K. Yu, Andrea Ventura, Andrew Mcpherson, C. Rudin, Sohrab P. Shah
          </td>
          <td>2026-02-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6b22a73c1f46495b3ff8ba940a6c1a1abbc7b2c" target='_blank'>
              scAmp analyzes focal gene amplifications at single-cell resolution
              </a>
            </td>
          <td>
            Matthew G. Jones, Natasha E. Weiser, King L. Hung, Xiaowei Yan, Sangya Agarwal, J. Luebeck, Aditi Gnanasekar, B. Howitt, Ellis J Curtis, Kevin Yu, John C. Rose, Katerina Kraft, Valeh Valiollah Pour Amiri, Leena Satpathy, V. Bafna, P. Mischel, Howard Y. Chang
          </td>
          <td>2026-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="
 CDK4/6 inhibitors (CDK4/6i) combined with endocrine therapy prolong survival in ER+/HER2- metastatic breast cancer, but nearly all patients eventually progress underscoring the need of strategies that prevent drug resistance. Drug tolerant persister (DTP) cells are a heterogeneous reservoir of surviving cells from which drug-resistant clones eventually emerge. We hypothesized that targeting breast cancer DTPs upon selective pressure from CDK4/6i would prevent the emergence of drug resistance. First, we identified genomic and epigenomic features of DTPs under continuous therapy. We treated MCF7 and T47D ER+ breast cancer cells with the CDK4/6i palbociclib 250nM + estrogen deprivation (E2dep) long-term (4 weeks). Cells that survived long-term treatment exhibited reversible sensitivity to CDK4/6i + E2dep and reversible cell cycle arrest. Whole-exome sequencing did not reveal known genomic alterations associated with resistance to CDK4/6i. In bulk RNA-seq of MCF7 DTP cells, cell cycle-related gene signatures were downregulated upon treatment, while interferon response and KRAS signatures were upregulated. MCF7 persisters were characterized by enrichment in senescence, diapause, chemotherapy-induced stressed gene signatures, while MYC target signatures were downregulated; all these gene signatures were reversed upon drug washout. To translate these findings to patients, we examined RNA expression data from residual tumors after neoadjuvant treatment with palbociclib and the aromatase inhibitor anastrozole in the NeoPalAna trial. Results were completely consistent with those observed in MCF7 DTPs. To elucidate which DTPs give rise to drug resistant clones, we barcoded MCF7 and T47D cells with a lineage and RNA recovery (LARRY) barcode library with ∼315,000 diverse barcodes, such that we can track them during long-term treatment. Barcoded cells were treated with palbociclib + E2dep for up to 4 weeks and then subjected to scRNA-seq; 101 and 15 clones that were initially arrested in G0/G1 eventually expanded at 4 weeks whereas 376 and 27 clones were eliminated during treatment of MCF7 and T47D cells, respectively. Comparison between expanding and exhausted clones identified genes upregulated in DTPs. To identify genes required for DTP survival, we performed a targeted CRISPR-Cas9 knockout screen using a custom library comprising the top 25 DTP marker candidate genes from each cell line, totaling 47 genes, including 3 shared between both cell lines. The library also included previously reported DTP-associated genes such as GPX4, KDM5A, KDM5B, ALDH1A1, ALDH1A2, and IGF1R, as well as RB as a negative control and PLK as a positive control. Cas9-expressing T47D and MCF7 cells were transduced with a lentiviral sgRNA library. Cells were treated with palbociclib + E2dep or 0.1% DMSO for 4 weeks. Enrichment or depletion of sgRNAs in treated and control cells was analyzed. HES1, a transcriptional repressor and downstream target of the Notch pathway, was among the top three genes with the most negative differential beta scores in both T47D and MCF7 cells, suggesting essentiality for DTP survival. In summary, a small subset of ER+ breast cancer cells survives long-term treatment with CDK4/6i + E2dep and exhibits reversible transcriptomic features of DTPs. scRNA-seq analysis of barcoded breast cancer cells revealed genes significantly upregulated in DTP clones that expanded after initial cell cycle arrest. A CRISPR-Cas9 screen of these genes revealed HES1 as a potentially essential gene for DTP survival. We propose therapeutically targeting DTP-associated genes such as HES1 may delay or prevent resistance to CDK4/6i in breast cancer.


 Y. Matsunaga, E. Aleksandrovic, H. Patel, D. Sudhan, K. Ahuja, D. Ye, C. A. Lin, L. Guo, C. X. Ma, J. Lee, S. Zhang, C. L. Arteaga, A. B. Hanker. Single cell analysis identifies genes that enable survival of drug tolerant persisters upon treatment with CDK4/6 inhibitors [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2025; 2025 Dec 9-12; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2026;32(4 Suppl):Abstract nr PD3-01.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b14d8a9af347472274d267aa63c313772bf5e17" target='_blank'>
              Abstract PD3-01: Single cell analysis identifies genes that enable survival of drug tolerant persisters upon treatment with CDK4/6 inhibitors
              </a>
            </td>
          <td>
            Y. Matsunaga, E. Aleksandrovic, H. Patel, D. Sudhan, K. Ahuja, D. Ye, C. A. Lin, L. Guo, C. Ma, J. Lee, S. Zhang, C. Arteaga, A. Hanker
          </td>
          <td>2026-02-17</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Introduction Chinese Hamster Ovary (CHO) cells are widely used for the production of recombinant therapeutics due to their ability to carry out human-like post-translational modifications. Media adaptation represents a key step in large-scale production to ensure optimal safety and cost efficiency. As DNA methylation is a central epigenetic mechanism underlying adaptive modulation of gene expression, we report here, for the first time, the use of high-coverage whole-genome bisulfite sequencing to generate single-base-resolution maps of CHO cells at different phases of growth in a fed-batch culture and undergoing media adaptation across four different media. Methods A CHO cell line was adapted to four commercially available media, and their growth rates and productivity were compared with those obtained using the control medium in a 7-day batch culture. This approach resulted in the generation of n = 57 high-quality whole-genome DNA methylation datasets, which were subjected to differential DNA methylation and gene association analyses. In addition, we developed a novel DNA methylation array comprising more than 63,000 CpG methylation sites across the CHO genome, enabling streamlined and efficient DNA methylation profiling of CHO cells. Results Analysis of n = 57 high-quality DNA methylation datasets revealed altered DNA methylation patterns across different phases of growth in a fed-batch culture and in response to distinct media adaptations. Specifically, adaptation to four different media resulted in highly specific methylation changes that were associated with distinct functional outcomes, including protein productivity. Finally, the customized DNA methylation microarray platform was used to validate all media adaptation-dependent epigenetic changes identified by whole-genome bisulfite sequencing (WGBS). Conclusion These findings identify and characterize dynamic DNA methylation changes occurring during media adaptation and support their potential use as predictive indicators of CHO cell phenotypic changes in response to a dynamic culture environment. Furthermore, this work represents a valuable resource for the development of DNA methylation-based biomarkers for the optimization of CHO cell culture.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9d23637c150361756d08aa41b54f6507306c271" target='_blank'>
              Whole-genome DNA methylation analysis of Chinese hamster ovary cells undergoing media adaptation
              </a>
            </td>
          <td>
            Suki Roy, Jasrene Kaur Sandhu, Lingzhi Huang, Abraham Wong, Guo Xuan Wan, Frank Lyko, Emeka Ignatius Igwe, Florian Böhl
          </td>
          <td>2026-01-21</td>
          <td>Frontiers in Bioengineering and Biotechnology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4322917e9d3faaba1bbb348dcf524837265ecd27" target='_blank'>
              Genomic and Evolutionary Determinants of Two-hit Frequencies in Tumor Suppressor Genes
              </a>
            </td>
          <td>
            Nivedita Mukherjee, Radhakrishnan Sabarinathan
          </td>
          <td>2026-02-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bce9b91a05b2c305ecf72c3bf59327984c56dba6" target='_blank'>
              Functional inactivation of the telomerase chaperone TCAB1 primes cells for the activation of ALT in osteosarcoma
              </a>
            </td>
          <td>
            Joshua Keegan, Sydney Sorbello, Joakin O. Mori, Shugo Muratani, I. Leshchiner, Christopher M. Heaphy, R. L. Flynn
          </td>
          <td>2026-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Simple Summary Long interspersed element-1 sequences are normally silenced in healthy cells but frequently reactivated in cancer, where they contribute to genomic instability and transcriptional dysregulation. However, the determinants of their transcriptional heterogeneity in non-small cell lung cancer remain poorly defined. In this study, we systematically characterized locus-specific LINE-1 transcript expression across a large panel of non-small cell lung cancer cell lines to determine how genomic context shapes retrotransposon activity. We show that LINE-1 transcription is dominated by evolutionarily young elements and follows conserved chromosomal patterns across tumor subtypes, with the highest activity observed in lung squamous cell carcinoma. Importantly, LINE-1 expression is anchored by a small number of conserved genomic hotspot loci that remain active across tumor backgrounds and are located near cancer-relevant genes. These findings provide a genome-scale framework for understanding retrotransposon regulation in cancer and highlight conserved LINE-1 loci as potential markers of genome instability and regulatory vulnerability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a76a2793974308abd948a4e791ee9224ae6f4f7f" target='_blank'>
              LINE-1 Transcript Heterogeneity in Non-Small Cell Lung Cancers Is Driven by Host Genomic Context and Conserved Functional Hotspots
              </a>
            </td>
          <td>
            Yingshan Wang, Kenneth S. Ramos
          </td>
          <td>2026-01-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79941e547de1ddb3fb2c22aa561602d507f6cf5a" target='_blank'>
              CRISPR-enhanced assessment of variants of unknown significance nominates oncology therapeutic targets and drug repositioning opportunities
              </a>
            </td>
          <td>
            A. Savino, Athanasios Oikonomou, Riccardo R. De Lucia, Miquel L Grau, Kathrina McCarten, H. Najgebauer, Umberto Perron, Luca Azzolin, Alexandra Livanova, Paolo Cremaschi, Núria López-Bigas, A. Sottoriva, F. Iorio
          </td>
          <td>2026-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d11f8627925c6b14efde1341627645217602b269" target='_blank'>
              Molecular background of Philadelphia chromosome dependent enhancement of cellular growth and tyrosine kinase inhibitor sensitivity
              </a>
            </td>
          <td>
            Md Faruq Hossain, L. Hagenau, Lars R. Jensen, J. Rhode, Thomas Sura, M. G. Salazar, Ana Tzvetkova, Corinna Jensen, S. Edwards, Heiko Dunkel, Stefan Simm, Josefine Radke, A. W. Kuss
          </td>
          <td>2026-02-19</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Fusion genes are pivotal drivers of tumorigenesis, often generating oncogenic chimeric RNAs and fusion circular RNAs. However, the mechanisms by which these transcripts synergistically contribute to cancer progression remain poorly understood. Here, we identified a lung cancer-specific chimeric RNA KANSL1-ARL17A (chKANSARL) and its circular variant fusion circular RNA KANSL1-ARL17 A (F-circKA), both derived from the fusion gene KANSARL. Functional assays revealed that overexpression of either chKANSARL or F-circKA significantly enhanced lung cancer cell proliferation, migration, and invasion, while their knockdown suppressed these malignant phenotypes. In vivo experiments demonstrated that chKANSARL overexpression accelerated tumor growth in immunodeficient mice. Notably, coexpression experiments uncovered a synergistic regulatory interaction between F-circKA and chKANSARL, amplifying oncogenic effects. Mechanistically, miRNA sequencing and dual-luciferase assays revealed that F-circKA acts as a molecular sponge for miR-6860, thereby derepressing chKANSARL expression. Rescue experiments further validated this regulatory axis, wherein miR-6860 inhibition reversed the tumor-suppressive effects of F-circKA knockdown. Collectively, our study identifies and characterizes a novel F-circKA/miR-6860/chKANSARL regulatory axis, revealing how dual transcriptional outputs from the KANSARL fusion gene can synergistically drive lung cancer progression. These findings highlight a previously unrecognized layer of cooperative regulation between linear and circular fusion RNAs in oncogenesis and provide a new framework for understanding fusion gene-mediated tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/261c7a04a77f3f0847188162738372118fe5132c" target='_blank'>
              The KANSL1-ARL17A fusion gene generates oncogenic chKANSARL and F-circKA RNAs that synergistically drive lung cancer progression via a novel F-circKA/miR-6860/chKANSARL axis
              </a>
            </td>
          <td>
            Xinchao Guan, Tao Liu, Sili Chen, Junwei Zhang, Huanliang Huang, Dexiong Chen, Qiaoyuan Yang
          </td>
          <td>2026-01-01</td>
          <td>The Journal of Biological Chemistry</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d049dde01e4ae93dabdffce09d14b8271cee14ed" target='_blank'>
              Emerging Anti-Cancer and Repurposed Therapies for Overcoming Multidrug Resistance in Lung Cancer
              </a>
            </td>
          <td>
            N. Solanki, Pratham Shah, Sargam Kewalramani, Umang Shah, Mehul Patel, Swayamprakash Patel, Rajesh Maheshwari
          </td>
          <td>2025-12-26</td>
          <td>Medical Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3e5486bf9671c3b3ce7043325de2e14e73fe8c9" target='_blank'>
              Connecting mitochondrial metabolism and mitotic fidelity to control vulnerability of high grade serous ovarian cancer patients to taxane-based chemotherapy
              </a>
            </td>
          <td>
            H. Moindjie, Morgane Morin, Evanthia Pangou, Diego Chianese, Cynthia Seiler, S. Rodrigues-Ferreira, Maria M. Haykal, Christine Péchoux, F. Dingli, D. Loew, Y. Kieffer, Géraldine Gentric, Patricia Pautier, Philippe Dessen, R. Argüello, Paolo Pinton, Fatima Mechta-Grigoriou, Catherine Brenner, Massimo Bonora, Clara Nahmias
          </td>
          <td>2026-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f67f6911402ef53a9ad609d0962bf43dc7f74aea" target='_blank'>
              Systems biology approach reveals the dynamics of cancer stem cell acquisition in ewing sarcoma: the role of let-7 and miR-145 regulatory circuits
              </a>
            </td>
          <td>
            D. A. Silveira, Shantanu Gupta, André T. Brunetto, J. C. Mombach, Marialva Sinigaglia
          </td>
          <td>2026-02-17</td>
          <td>Network Modeling Analysis in Health Informatics and Bioinformatics</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Abstract Deregulation of proteins involved in chromatin regulation is common in pancreatic ductal adenocarcinoma (PDAC). Lysine demethylase 4C (KDM4C) is one of the chromatin-modifying proteins frequently overexpressed across multiple solid cancers and is linked to chromatin instability, increased cell proliferation, and enhanced stem cell–like behavior. We observed upregulation of KDM4C protein in a panel of human PDAC cell lines and patient samples compared with nonneoplastic controls. CRISPR/Cas9-mediated deletion of KDM4C in human and murine PDAC cells reduced proliferation, clonogenicity, and increased survival of orthotopically implanted murine PDAC allografts. Transcriptomic and proteomic analyses revealed that loss of KDM4C in both human and murine PDAC cell lines was associated with the reduction of activated phospho-ERK, a pivotal effector downstream of mutant RAS. Using proximity labeling, we identified the histone deacetylase SIRT1 as a novel interacting protein with KDM4C via the latter’s Tudor reader domain. SIRT1-mediated deacetylation leads to repression of downstream targets, including the dual specificity phosphatase DUSP2, which is known to inactivate ERK via dephosphorylation. In vitro propagation of KDM4C-null PDAC lines eventually led to adaptation and restitution of ERK signaling, with rescue of the KDM4C loss induced growth suppression. To bypass this adaptive phenomenon, we tested a preclinical pan-KDM4 inhibitor TACH107 and confirmed its efficacy in in vitro and in vivo PDAC models. Our studies identify KDM4C as an oncogenic molecule that sustains ERK signaling in KRAS-mutant PDAC and can be broadly targeted via small-molecule inhibitors. Significance: Our data suggests that KDM4C is a novel regulator of ERK signaling, the main effector pathway downstream of mutant RAS. This is the first demonstration linking the requirement of sustained KDM4 activity to ERK signaling in cancer and presents an opportunity to leverage this oncogenic pathway for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac9ce02e321041b11b3d0c59cd917566ed5c1973" target='_blank'>
              The Lysine Demethylase KDM4C Is an Oncogenic Driver and Regulates ERK Activity in KRAS-Mutant Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Menna-T-Allah Shaheen, Sarah Dhebat, K. Rajapakshe, Bidyut Ghosh, Benson Chellakkan Selvanesan, Shariq S Ansari, C. Haymaker, Dorsay Sadeghian, Huamin Wang, Ching-Fei Li, Haoqiang Ying, Anirban Maitra
          </td>
          <td>2026-01-01</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="


 Prostate cancer (PCa) is the most commonly diagnosed cancer and the second leading cause of cancer-related death in American men. Men with germline BRCA2 mutations are at especially high risk (19%-61% by age 80) of developing PCa. Loss of BRCA2 impairs homologous recombination repair (HRR), resulting in genomic instability and enhanced tumor progression. Our laboratory previously performed a whole-genome CRISPR-Cas9 screen in a BRCA2 mutant cell line under cisplatin induced stress. This screen identified Chromodomain Helicase DNA-binding protein 1 (CHD1) as the top gene whose loss promoted cell survival under DNA damage stress. CHD1 is a chromatin remodeler that is deleted in approximately 15% PCas and is strongly linked to metastatic disease. However, how its loss drives tumor progression, particularly in BRCA2-deficient PCas, remains poorly understood. To address this, we carried out RNA-seq analysis in castration-resistant prostate cancer (CRPC) cell line PC-3 following knockdown of CHD1 and BRCA2, individually and in combination. Notably, depletion of CHD1 in BRCA2-deficient cells resulted in the upregulation of gene signatures associated with hypoxia and epithelial-mesenchymal-transition (EMT). Functional validation using wound healing and migration assays demonstrated that CHD1 loss significantly enhanced the migration ability of PC-3 cells (including BRCA2-deficient) compared to controls. In keeping with increased EMT, we also see high vimentin expression in CHD1-depleted PC3. Furthermore, Causal Inference Enrichment (CIE) analysis of the transcriptomic data pointed to significant activation of transcription factors (TFs) FOSL1 and SOX9 after CHD1 knockdown compared to control cells. To assess clinical relevance, we analyzed RNA-seq data from the Stand Up To Cancer (SU2C) database. This analysis revealed that CHD1-low tumors exhibited upregulation of multiple oncogenic pathways, especially EMT, angiogenesis, apical junction, and IL2-STAT5 signaling, consistent with our in vitro observations. Finally, genetically engineered mouse model (GEMM)-derived prostate organoids from B2PPt (Brca2flox/flox;P53flox/flox;Ptenflox/flox) and B2CPPt (Brca2flox/flox; Chd1flox/flox;P53flox/flox;Ptenflox/flox) mice, after Adeno-Cre infection, revealed higher Fosl1, along with EMT marker vimentin in B2CPPt organoids compared to B2PPt, supporting FOSL1-dependent lineage plasticity and activation of EMT programs upon CHD1 loss in BRCA2-mutant prostate cells. Collectively, these findings indicate that CHD1 loss, either independently or in the context of BRCA2 deficiency, promotes oncogenic phenotypes through activation of FOSL1-dependent lineage plasticity and EMT programs that favor metastasis. Ongoing work dissects how CHD1 loss reshapes chromatin accessibility to reprogram transcriptional networks and drives oncogenic signaling and metastatic pathways in PCa, with particular relevance to BRCA2-deficient PCa.



 Jingzhu Hao, Shailja Pathania, Miklos Diossy, Kourosh Zarringhalam, Changmeng Cai, Zoltan Szallasi. Dissecting the role of CHD1 as a key regulator of pro-metastatic transcriptional reprogramming in BRCA2-mutant prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr PR017.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a2ea6307c1138357462e3893b183854dd498126" target='_blank'>
              Abstract PR017: Dissecting the role of CHD1 as a key regulator of pro-metastatic transcriptional reprogramming in BRCA2-mutant prostate cancer
              </a>
            </td>
          <td>
            Jingzhu Hao, S. Pathania, M. Dióssy, Kourosh Zarringhalam, Changmeng Cai, Zoltán Szállási
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Genomic alterations are a hallmark of cancer, and extrachromosomal DNA (ecDNA) has emerged as a key source of oncogene selection, tumor growth, and drug resistance. The intratumor heterogeneity and clonal selection of ecDNA is, however, poorly understood. In this study, we pursue a computational approach that leverages allelic imbalance and outlier expression from standard single-cell RNA sequencing (scRNA-seq) to deconvolve the tumor heterogeneity of ecDNA at the single-cell level (ecSingle). Using this approach, we identify oncogene-carrying ecDNAs in tumor samples at the single-cell level, which we validate using genome sequencing. Moreover, we show the superiority of using single-molecule long-read sequencing in resolving ecDNA. ecDNAs displayed extensive intratumor heterogeneity, including subclonal oncogene-carrying ecDNA in primary tumor cells that segregate with distinct transcriptional cell states. Importantly, we show that a rare ecDNA+ clone in the primary tumor can expand to form dominant clones in relapse tumors. Our study introduces a novel approach to studying ecDNA at the single-cell level, enabling both clonal evolution and transcription cell state analysis. We apply this approach to cancer samples to gain deeper insights into the role of ecDNA in intratumor heterogeneity and cellular plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7df14db8ad5c3596e9b413772fed54a722b4bb2b" target='_blank'>
              Resolving clonal evolution and selection of extrachromosomal DNA at single-cell resolution
              </a>
            </td>
          <td>
            J. Hendriksen, A. Locallo, B. Schlotmann, Francisco Germán Rodríguez González, Jane Skjøth-Rasmussen, C. Yde, D. Nørøxe, H. S. Poulsen, U. Lassen, J. Weischenfeldt
          </td>
          <td>2026-01-29</td>
          <td>Genome Biology</td>
          <td>1</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b3ae1ba137c707051b533b78e7831f38e7fb1ac" target='_blank'>
              Metabolic reprogramming-driven resistance to multi-kinase inhibitors in hepatocellular carcinoma: molecular mechanisms and therapeutic opportunities
              </a>
            </td>
          <td>
            Junxin Li, Yu Huang, Jiawei Li, Ming Shi, Yi Xiao, Fei Du, Gongli Hu
          </td>
          <td>2026-01-27</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b68bfe7db38cbef63e90c1645d6d3b34347907a7" target='_blank'>
              Comprehensive profiling of CRISPR/dCas9 epigenome editors indicates a complex link between on and off target effects
              </a>
            </td>
          <td>
            M. Pahlevan Kakhki, Fatemeh Rangani, Ewoud Ewing, C. Starvaggi Cucuzza, G. Zheleznyakova, M. Kalomoiri, Lea Kenny, Anika Raghavan, Chandana Rao Prakash, Gabe van den Hoeven, Tejaswi Venkata S. Badam, Ruxandra Covacu, Ioanna Andreou, Maria Needhamsen, L. Kular, Maja Jagodic
          </td>
          <td>2026-01-31</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Dysregulation of repair pathways, specifically the base excision repair (BER) axis, is frequently observed in cancer. This dysfunction is associated with accelerated tumor growth and reduced patient survival, suggesting that BER components may serve as potential targets for cancer therapy. This study focused on the expression of Uracil-DNA glycosylase (UNG), a key enzyme in BER, in colorectal tumor tissues and adjacent normal tissues. We systematically explored the clinical significance, biological functions, and molecular mechanisms of UNG in colorectal cancer (CRC). Our findings revealed that UNG was significantly upregulated in CRC, and its higher expression correlated with poorer patient outcomes. Functional assays demonstrated that silencing UNG inhibited cell proliferation, reduced migration and invasion capabilities, and induced apoptosis along with S/G2 phase cell cycle arrest. Mechanistically, UNG appears to interact with the AMPK, AKT/mTOR, and ERK signaling pathways. These results suggested that UNG exhibited oncogenic potential in CRC by regulating malignant behaviors and oncogenic signaling cascades, positioned it as a potential prognostic biomarker and therapeutic target for CRC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40071783d4dcf5a7a90c5996af36927f2f827940" target='_blank'>
              Silencing Uracil-DNA glycosylase inhibits colorectal cancer progression
              </a>
            </td>
          <td>
            Jingjing Yang, Li-zhou Shi, Yong-wei Hu, Hua Chen, Qing-hua Wang, Wei Han
          </td>
          <td>2026-01-04</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background: Glioblastoma multiforme (GBM) is a highly aggressive primary brain tumor characterized by rapid proliferation, profound invasiveness, and resistance to conventional therapies. Deubiquitinating enzymes (DUBs), crucial regulators of protein homeostasis, have recently been implicated in GBM pathogenesis. However, the specific DUBs that play central roles in GBM pathogenesis and their exact molecular mechanisms remain to be further elucidated. Methods: We systematically analyzed GBM datasets and clinical samples to identify differentially expressed DUBs. Functional experiments, including genetic manipulation, immunoprecipitation coupled mass spectrometry (IP-MS), comprehensive metabolic assays, mitochondrial function assessments, and orthotopic mouse models, were conducted. Results: Here, we identified PSMD14 as a protein significantly upregulated in GBM, with a close correlation to poor prognosis of patients. Mechanistic exploration revealed that PSMD14 stabilized IMPDH2, the rate-limiting enzyme of purine nucleotide biosynthesis, by selectively removing K48-linked polyubiquitin chains. When PSMD14 is inhibited genetically or pharmacologically, IMPDH2 stability diminishes, causing impaired nucleotide metabolism, mitochondrial dysfunction, increased DNA damage signaling, and reduced tumor malignancy. Importantly, these metabolic issues can be reversed by exogenous guanosine, highlighting the key role PSMD14 in metabolic regulation. In translational medicine, the PSMD14 inhibitor, Thiolutin, curbed GBM progression in vitro and in vivo by disrupting the de novo purine biosynthesis and resulting in mitochondrial fragmentation. Moreover, Thiolutin synergized with TMZ to overcome resistance and boost efficacy. This study reveals a new GBM metabolic axis and presents a promising PSMD14-targeting therapy. Conclusions: PSMD14-IMPDH2 axis serves as a crucial hub integrating post-translational modifications and metabolic homeostasis in GBM. Targeting PSMD14 enhances therapeutic sensitivity, presenting a promising strategy to overcome TMZ resistance and improve GBM treatment efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96036dbb03725e6c9416ba601f587aa6ca12bc4a" target='_blank'>
              Coupling proteostasis and de novo purine biosynthesis of PSMD14 fuels glioblastoma progression and chemoresistance
              </a>
            </td>
          <td>
            Jiazheng Wang, Qun Cao, Zhikai Li, Xuxiu Lu, Zhuo Li, Chenghui Guo, Yuan Pan, Qing Zhang, Wenjie Li, Guo Xiang, Anjing Chen
          </td>
          <td>2026-01-01</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e04b464c0e1b167410299c1bc664ca1e5464bb8" target='_blank'>
              cellSTAAR: incorporating single-cell-sequencing-based functional data to boost power in rare variant association testing of noncoding regions
              </a>
            </td>
          <td>
            Eric Van Buren, Yi Zhang, Xihao Li, M. Selvaraj, Zilin Li, Hufeng Zhou, Nicholette D. Palmer, Donna K. Arnett, J. Blangero, Eric Boerwinkle, B. Cade, Jenna C. Carlson, April P. Carson, Yii-DerIda Chen, J. Curran, R. Duggirala, Myriam Fornage, Nora Franceschini, M. Graff, C. Gu, Xiuqing Guo, Jiang He, Nancy Heard-Cosa, Lifang Hou, Yi-Jen Hung, Rita R. Kalyani, Sharon L. R. Kardia, Eimear E. Kenny, C. Kooperberg, B. Kral, Leslie Lange, Daniel Levy, Changwei Li, Simin Liu, D. Lloyd-Jones, R. Loos, A. Manichaikul, L. W. Martin, Rasika A Mathias, R. Minster, B. Mitchell, J. Mychaleckyj, T. Naseri, Kari E. North, Jeffry R. O'Connell, James A. Perry, P. Peyser, B. Psaty, L. Raffield, Ramachandran S. Vasan, S. Redline, Alexander P. Reiner, Stephen S. Rich, Jennifer A. Smith, Brian W. Spitzer, Hua Tang, Kent D. Taylor, Russell Tracy, S. Viali, Lisa R. Yanek, Wei Zhao, , Namiko Abe, Gonçalo R. Abecasis, F. Aguet, Christine Albert, L. Almasy, Alvaro Alonso, Seth Ament, Peter Anderson, Pramod Anugu, Deborah Applebaum-Bowden, K. Ardlie, D. Arking, A. Ashley-Koch, Stella Aslibekyan, T. Assimes, Paul L. Auer, D. Avramopoulos, N. Ayas, Adithya Balasubramanian, John Barnard, Kathleen Barnes, R. G. Barr, E. Barron-Casella, L. Barwick, Terri L Beaty, Gerald Beck, Diane M. Becker, Lewis C. Becker, Rebecca L. Beer, Amber Beitelshees, Emelia Benjamin, Takis Benos, Marcos Bezerra, Larry Bielak, Joshua C. Bis, Thomas Blackwell, Nathan Blue, Don Bowden, Russell P Bowler, Jennifer Brody, U. Broeckel, J. Broome, Deborah L. Brown, Karen L. Bunting, Esteban G. Burchard, Carlos Bustamante, Erin J. Buth, J. Cardwell, V. Carey, J. Carrier, C. Carty, Richard Casaburi, Juan P. Casas Romero, James M. Casella, Peter Castaldi, M. Chaffin, Christy Chang, Yi-Cheng Chang, Daniel I. Chasman, S. Chavan, Bo-Juen Chen, Wei-Min Chen, Michael Cho, S. Choi, Lee-Ming Chuang, M. Chung, Ren-Hua Chung, Clary Clish, S. Comhair, M. Conomos, Elaine Cornell, Adolfo Correa, Carolyn Crandall, J. Crapo, L. Cupples, J. Curtis, B. Custer, Coleen Damcott, D. Darbar, S. David, Colleen P. Davis, M. Daya, Michael R. DeBaun, Dawn Demeo, R. Deka, Scott Devine, H. Dinh, H. Doddapaneni, Qing Duan, Shannon Dugan-Perez, J. P. Durda, S. Dutcher, Charles B. Eaton, L. Ekunwe, Adel Boueiz, Patrick T Ellinor, L. Emery, S. Erzurum, Charles Farber, J. Farek, T. Fingerlin, M. Flickinger, C. Frazar, Mao Fu, Stephanie M. Fullerton, L. Fulton, Stacey Gabriel, Weiniu Gan, Shanshan Gao, Yan Gao, Margery L S Gass, Heather Geiger, B. Gelb, M. Geraci, S. Germer, R. Gerszten, Auyon Ghosh, Richard Gibbs, Chris Gignoux, M. Gladwin, David E. Glahn, S. Gogarten, Da-Wei Gong, H. Goring, S. Graw, Kathryn J. Gray, Daniel Grine, Colin Gross, Yue Guan, Namrata Gupta, Jeffery Haessler, Michael N. Hall, Yi Han, Patrick J. Hanly, Daniel N Harris, Nicola L. Hawley, Benjamin D. Heavner, S. Heckbert, Ryan Hernandez, David M. Herrington, Craig Hersh, Bertha Hidalgo, James E Hixson, Brian Hobbs, John Hokanson, Elliott Hong, Karin F. Hoth, C. Hsiung, Jianhong Hu, Yi-Jen Hung, H. Huston, C. Hwu, M. Irvin, Rebecca D. Jackson, D. Jain, C. Jaquish, J. Johnsen, Andrew Johnson, Craig W Johnson, R. Johnston, Kimberly Jones, H. M. Kang, R. Kaplan, Shannon Kelly, Michael D. Kessler, Alyna T Khan, Ziad M. Khan, Wonji Kim, J. Kimoff, G. Kinney, Barbara A. Konkle, Holly Kramer, Christoph Lange, Ethan Lange, Cathy C. Laurie, C. Laurie, M. LeBoff, J. Lefaive, Jiwon Lee, Sandra Lee, Wen-Jane Lee, David Levine, Joshua Lewis, Xiaohui Li, Yun Li, Henry J. Lin, Honghuang Lin, Yongmei Liu, Yu Liu, S. Lubitz, K. Lunetta, James Luo, U. Magalang, Michael P. Mahaney, B. Make, Alisa K Manning, J. Manson, Melissa A Marton, S. Mathai, Susanne May, P. McArdle, Merry-Lynn N. McDonald, Sean K. McFarland, S. McGarvey, Daniel J. McGoldrick, Caitlin P. McHugh, B. McNeil, Hao Mei, J. Meigs, Vipin Menon, L. Mestroni, G. Metcalf, Deborah Meyers, Emmanuel Mignot, Julie Mikulla, N. Min, Mollie A Minear, M. Moll, Z. Momin, M. Montasser, Courtney G. Montgomery, Donna M Muzny, Girish N. Nadkarni, Rakhi Naik, Sergei Nekhai, Sarah C. Nelson, Bonnie Neltner, C. Nessner, Deborah Nickerson, Osuji Nkechinyere, T. O’Connor, H. Ochs-Balcom, Geoffrey O. Okwuonu, Allan Pack, David T. Paik, Nicholette D. Palmer, J. Pankow, G. Papanicolaou, Cora Parker, J. Peralta, Marco V. Perez, U. Peters, Lawrence S. Phillips, Jacob Pleiness, Toni Pollin, Wendy S. Post, Julia Powers Becker, M. Boorgula, Michael H. Preuss, P. Qasba, D. Qiao, Zhaohui Qin, N. Rafaels, M. Rajendran, D. Rao, Laura Rasmussen-Torvik, A. Ratan, Robert M Reed, C. Reeves, E. Regan, M. Reupena, Kenneth M. Rice, Rébecca Robillard, N. Robine, Dan Roden, C. Roselli, I. Ruczinski, Alexi Runnels, Pamela H. Russell, S. Ruuska, Kathleen A. Ryan, E. Sabino, D. Saleheen, S. Salimi, Sejal Salvi, Steven Salzberg, Kevin Sandow, Vijay G. Sankaran, J. Santibanez, K. Schwander, D. Schwartz, F. Sciurba, C. Seidman, J. Seidman, Vivien Sheehan, S. Sherman, Amol C. Shetty, A. Shetty, Wayne Hui-Heng Sheu, M. B. Shoemaker, Brian H. Silver, Edwin Silverman, Robert Skomro, A. V. Smith, Josh Smith, Nicholas L. Smith, Tanja Smith, S. Smoller, Beverly M. Snively, M. Snyder, Tamar Sofer, N. Sotoodehnia, A. Stilp, G. Storm, E. Streeten, J. Su, Y. Sung, Jody M Sylvia, Adam A. Szpiro, F. Sériès, D. Taliun, M. Taub, Matthew Taylor, Simeon Taylor, M. Telen, Timothy A. Thornton, M. Threlkeld, Lesley E. Tinker, D. Tirschwell, Sarah Tishkoff, Hemant K. Tiwari, Catherine Tong, Michael Y. Tsai, Dhananjay Vaidya, D. Berg, Peter Vandehaar, S. Vrieze, Tarik Walker, R. Wallace, A. Walts, Fei Fei Wang, Heming Wang, Jiongming Wang, Karol Watson, Jenn Watt, D. Weeks, J. Weinstock, B. Weir, Scott T. Weiss, L. Weng, Jennifer Wessel, Cristen J. Willer, Kayleen Williams, L. K. Williams, Scott Williams, Carla Wilson, James G. Wilson, L. Winterkorn, Q. Wong, Baojun Wu, Joseph Wu, Huichun Xu, Lisa R. Yanek, Ivana V. Yang, Ketian Yu, S. Zekavat, Yingze Zhang, Snow Xueyan Zhao, Xiaofeng Zhu, Elad Ziv, Michael C. Zody, S. Zoellner, Mariza de Andrade, Paul de Vries, L. Fuentes, Jerome L. Rotter, G. Peloso, P. Natarajan, Xihong Lin
          </td>
          <td>2025-12-31</td>
          <td>Nature Methods</td>
          <td>0</td>
          <td>101</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27e64bdfdb14f37106c0cabb353f42df99006bef" target='_blank'>
              Ancient eukaryotic immunity through genome editing of viral sequences
              </a>
            </td>
          <td>
            Lisa Mettrop, A. Lipzen, Gilles Mirambeau, K. Barry, Igor V. Grigoriev, G. Piganeau, Marc Krasovec
          </td>
          <td>2026-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Genomic instability is a hallmark of nearly all human cancers, yet the genetic regulators that govern this instability, their functional roles, and their therapeutic potential remain incompletely understood in breast cancer. In this study, we conducted a comprehensive analysis of the expression profiles of genomic instability-maintaining genes (GIMGs) in breast cancer and revealed that elevated expression of GIMGs was associated with adverse clinical outcomes and an altered tumor immune microenvironment. Through further analysis, we identified a core set of GIMGs and pinpointed RPGRIP1L as a critical driver of genomic instability in breast cancer. Notably, RPGRIP1L expression was significantly upregulated in breast cancer tissues and strongly correlated with poor patient prognosis. Functional studies demonstrated that RPGRIP1L knockdown reduced genomic instability in tumor cells, as evidenced by decreased Phosphorylated Histone H2AX (γH2AX) expression, and suppressed tumor growth in mouse models. Additionally, high RPGRIP1L expression was linked to elevated expression of immune checkpoint Programmed Death Ligand 1(PD-L1) in human breast cancer samples. Mechanistically, we found that RPGRIP1L promoted the activation of the Hedgehog signaling pathway, which in turn drived tumor proliferation and upregulated PD-L1 expression. Collectively, these findings highlight RPGRIP1L as a key genomic instability-maintaining gene in human breast cancer, offering critical insights into the molecular mechanisms underlying disease progression. Furthermore, targeting RPGRIP1L may represent a promising therapeutic strategy for breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c08de5eb400d357e336428d7f7a366a3a038cb3" target='_blank'>
              Identification of RPGRIP1L as an instability-maintaining gene to drive tumor growth and PD-L1 expression via Hedgehog signaling in breast cancer
              </a>
            </td>
          <td>
            Meng Lv, Yan’e Liu, Weihong Cao, Yongmei Wang, Qi Wang, Xue-Qiang Gao, Baowei Peng, Hai-bo Wang, Yan Mao
          </td>
          <td>2025-12-30</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Despite significant therapeutic advances, multiple myeloma (MM) remains incurable in most patients due to frequent tumor relapse. A major contributor to relapse is clonal heterogeneity, where subclones exhibit distinct mechanisms of therapy resistance, along with the presence of drug-resistant dormant cells. Eliminating these distinct populations, which often coexist in the tumor niche, is clinically challenging. Identifying survival mechanisms shared by drug-resistant proliferating and dormant cells holds potential for the simultaneous elimination of different tumor-repopulating clones. To identify shared mechanisms of therapeutic resistance, we analyzed clinical databases and drug-resistant myeloma cell lines. We employed pharmacologic approaches to target common candidates identified in our analysis and assessed their impact on tumor progression and survival in preclinical mouse models containing both therapy-resistant and dormant cells. We identified upregulation of several components of the Notch signaling pathway in both dormant and drug-resistant MM cells, which correlated with poor clinical outcomes in newly diagnosed MM patients. Selective blockade of NOTCH3 with a neutralizing antibody or pan-Notch inhibition with a bone-targeted inhibitor reduced tumor burden and eliminated coexisting dormant and bortezomib-resistant cells in clinically relevant models of MM disease. Our findings reveal that NOTCH3-dependent survival programs represent a shared vulnerability in both cells refractory to therapy and dormant cells. These programs can be exploited to overcome the diverse mechanisms by which cancer cells evade therapy, potentially preventing disease relapse and extending remission in patients with MM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f4eb8f9fdb1cae4aedbf1364db151a77768c687" target='_blank'>
              Targeting NOTCH3 to eradicate dormant and therapy-resistant multiple myeloma cells
              </a>
            </td>
          <td>
            H. Sabol, Bethany C. Paxton, A. Anloague, Japneet Kaur, Mattie R. Nester, Sharmin Khan, James Smith, Peter I. Croucher, Michelle M McDonald, Corey O. Montgomery, J.B. Stambough, C. L. Barnes, E. Ambrogini, F. H. Ebetino, C. Schinke, Cody Ashby, J. Delgado-Calle
          </td>
          <td>2026-01-05</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/369c5e33b0cad07e2b10ea8763123bd8af3e00d3" target='_blank'>
              TERT drives liver tumorigenesis beyond telomere elongation
              </a>
            </td>
          <td>
            Laura Braud, Julien Vernerey, Arnaud Guille, Pierre Cordier, Clémence Ginet, Tom Egger, Manuel Bernabe, D. Churikov, Quentin Da Costa, Aïda Meghraoui, C. Desdouets, Li Gu, F. Bertucci, Christophe Lachaud, Vincent Géli
          </td>
          <td>2026-02-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Despite its critical role in tumor evolution, a detailed quantitative understanding of the evolutionary dynamics of aneuploidy remains elusive. Here we introduce ALFA-K (Adaptive Local Fitness landscapes for Aneuploid Karyotypes), a method that infers chromosome-level karyotype fitness landscapes from longitudinal single-cell data. ALFA-K estimates fitness of thousands of karyotypes closely related to observed populations, enabling robust prediction of emergent karyotypes not yet experimentally detected. We validate ALFA-K’s performance using synthetic data from an agent-based model and empirical data from in vitro and in vivo passaged cell lines. Analysis of fitted landscapes suggests several key insights: (1) Whole genome doubling facilitates aneuploidy evolution by narrowing the spectrum of deleterious copy-number changes; (2) Environmental context and cisplatin treatment significantly modulate the fitness impact of these changes; (3) Fitness effects of copy-number changes depend on parental karyotype; and (4) Chromosome mis-segregation rates strongly influence the predominant karyotypes in evolving populations. The evolutionary dynamics of aneuploidy in solid tumors are challenging to study. Here the authors introduce a method, ALFA-K, which estimates karyotype fitness and predicts emergent karyotypes before experimental detection, and test its performance on synthetic and empirical data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e9f0c1aa74fa8dcd40ddabd793fc67f7d7cd6be" target='_blank'>
              ALFA-K: Local adaptive mapping of karyotype fitness landscapes
              </a>
            </td>
          <td>
            Richard J Beck, Tao Li, Noemi Andor
          </td>
          <td>2025-12-27</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 The transforming growth factor β (TGFβ) family encodes pleiotropic cytokines that modulate cellular identity, primarily through the epithelial-mesenchymal transition (EMT) and the promotion of cancer stem cell populations, which accounts for elevated tumor heterogeneity. In breast cancer cells, such heterogeneity can be manifested at both transcriptional and epigenetic levels, with direct implications for metabolic rewiring. Using CRISPR/Cas9 gene editing technology, we ablated the EMT-related transcription factor SNAI2 in triple-negative breast cancer cells. Whole-transcriptome profiling revealed significant downregulation in cell cycle-related processes associated with the induction of apoptosis and cell death signatures. Strikingly, knockout (KO) cells exhibited significant alterations in various genes involved in central metabolic pathways, including oxidative phosphorylation, fatty acid metabolism and mTOR signaling. Functional analyses of mitochondrial respiratory activities showed enhanced complex I and complex II activities as well as increased ATP-linked, maximum, and reserve respiratory capacities in SNAI2 KO cells. Feul-dependence studies indicated a significant reduction in fatty acid dependence, attributed to the account of pyruvate metabolism, while glutamiolysis and glycolytic metabolism remained unchanged. The KO cells exhibited elevated mitochondrial density, accompanied by accumulation of intracellular and mitochondrial reactive oxygen species. Phenotypically, the KO cells presented an intermediate epithelial-mesenchymal state with decreased proliferation capacity and migration potential. Additionally, the KO cells showed enhanced sensitivity to the CDK4/6 inhibitor palbociclib, compared to parental cells. Seminal studies in various breast cancer models have revealed the potency of TGFβ in promoting cell resistance to palbociclib through transcriptional modulation of EMT factors, including SNAI2, and subsequent metabolic remodeling. Therefore, the present study demonstrates the pivotal role of SNAI2 in mediating transcriptional and metabolic rewiring, leading to increased cancer heterogeneity.



 M. M. Ali, C. Tsirigoti, H. Yasmin, S. S. Ali, A. Moustakas. Tgfβ-associated transcription factors induce metabolic and chromatin remodeling in breast cancer cells [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2025; 2025 Dec 9-12; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2026;32(4 Suppl):Abstract nr PS2-12-11.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dea983a74b03b7199fc6d1a1b1130d25ccfacfd3" target='_blank'>
              Abstract PS2-12-11: Tgfβ-associated transcription factors induce metabolic and chromatin remodeling in breast cancer cells
              </a>
            </td>
          <td>
            , C. Tsirigoti, H. Yasmin, S. Ali, A. Moustakas
          </td>
          <td>2026-02-17</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f2b7d842c50e19f140b3dba7c07392f64035b86" target='_blank'>
              Single-nucleus multi-omics delineates distinct epigenetic programs associated with tumor progression in lung adenocarcinoma.
              </a>
            </td>
          <td>
            Yeji Kim, Huiram Kang, Y. Hwang, Kyung Soo Kim, S. Moon, Jeonghan Kim, Seung Joon Kim, Hae-Ock Lee
          </td>
          <td>2026-02-11</td>
          <td>Clinical epigenetics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The KEAP1/NRF2 pathway, a major regulator of the cellular oxidative stress response, is frequently activated in human cancers. Often mediated by loss-of-function mutations in KEAP1, this activation causes increased NRF2 transcriptional activity and constitutive activation of the antioxidant response. While KEAP1 mutations have been well documented in various cancers, their presence and role in thyroid carcinoma have remained largely unexplored. In this study, we sequenced pediatric thyroid tumors and analyzed publicly available datasets, identifying 81 KEAP1 mutations in tumors across a range of histologies. In these tumors, we further identified frequent biallelic loss of KEAP1 via 19p13.2 loss of heterozygosity (LOH). MAPK-activating alterations were found in a subset of KEAP1-mutant cases, but they were mutually exclusive with 19p13.2 LOH. Transcriptome analysis also revealed significant activation of the NRF2 pathway in KEAP1-mutant tumors. Four additional cases with similar transcriptional profiles but lacking mutational data were identified, likely representing putative KEAP1 mutants. Using in vitro cell line models, we then profiled the functional consequences of KEAP1 knockout in cells with and without known driver alterations. In these models, we show that KEAP1 loss leads to an NRF2-dependent upregulation of AKR1C3, GCLC, NQO1, and TXNRD1, along with increased proliferation and migration irrespective of MAPK mutational status. We also demonstrate that loss of KEAP1 reduces sensitivity of RET fusion-positive cells to selpercatinib, consistent with previous reports that these alterations promote drug resistance in other malignancies. In this study, we comprehensively profile KEAP1 mutations in thyroid tumors, showing that they are more prevalent and functionally significant than previously recognized. These findings position KEAP1 mutations as novel oncogenic variants in thyroid cancer and support the integration of KEAP1/NRF2 pathway profiling into future studies and clinical frameworks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61e3679e5f3a81da4bfabc2e262c1a4730ec5b1c" target='_blank'>
              KEAP1 mutations activate the NRF2 pathway to drive cell growth and migration, and attenuate drug response in thyroid cancer
              </a>
            </td>
          <td>
            Nicholas E. Bambach, J. Ricarte-Filho, E. R. Reichenberger, Christian Hernandez-Padilla, K. Hinkle, A. Isaza, A. J. Bauer, A. Franco
          </td>
          <td>2026-01-07</td>
          <td>Frontiers in Oncology</td>
          <td>1</td>
          <td>7</td>
        </tr>

        <tr id="Multiple myeloma (MM) is an incurable hematologic malignancy characterized by the clonal expansion of plasma cells in the bone marrow. Despite advances in therapeutic agents, including proteasome inhibitors, immunomodulatory drugs and immunotherapies, relapse driven by treatment resistance remains a major clinical challenge. This underscores the critical need to elucidate additional molecular mechanisms that drive MM pathogenesis and therapeutic failure. The emerging field of epitranscriptomics, which studies post‑transcriptional RNA modifications, offers a promising perspective. Among these modifications, N6‑methyladenosine (m6A), the most abundant internal mRNA modification, has been implicated in regulating nearly every aspect of RNA metabolism. Growing evidence indicates that dysregulation of the m6A modification machinery plays a pivotal role in MM heterogeneity, disease progression and drug resistance. The present review synthesized current knowledge on how specific m6A regulators contribute to MM oncogenesis by modulating key signaling pathways, interactions with the bone marrow microenvironment and responses to therapy. It also discussed the potential of targeting m6A pathways as a therapeutic strategy to overcome treatment resistance and improve patient outcomes. By highlighting recent advances and future directions, the present review underscored m6A modification as an important frontier in the battle against MM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cc90def1e6d756ae29b54123b091a9ffa3e51a7" target='_blank'>
              Emerging roles of RNA m6A modification in multiple myeloma pathogenesis and treatment resistance (Review).
              </a>
            </td>
          <td>
            Yasen Maimaitiyiming, Shuoyang Hu, Die Bai, Yingchao Guan, Na Bu, Wenhui Hao, Mayila Maimaiti
          </td>
          <td>2026-02-10</td>
          <td>International journal of oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd77a79644efdb57765d420fb7be18f39a993472" target='_blank'>
              Cancer-associated synonymous mutations reveal stress signal-dependent mRNA folding that selectively modulates protein function
              </a>
            </td>
          <td>
            Sivakumar Vadivel Gnanasundram, Lixiao Wang, Sa Chen, Ondrej Bonczek, B. Vojtěšek, Robin Fåhraeus
          </td>
          <td>2026-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Chemoresistance remains a major obstacle to effective cancer treatment, often driven by enhanced DNA repair mechanisms that enable tumor cells to withstand genotoxic therapies. One such pathway involves the atypical DNA damage repair complex ALKBH3–ASCC, activated by the E3 ligase RNF113A in response to alkylation damage. We previously showed that SMYD3-dependent methylation of RNF113A stimulates this pathway, enhancing DNA repair and promoting resistance. Here, we identify KDM7B/PHF8 as the bona fide RNF113A demethylase, establishing one of the first functional examples of a dynamic, reversible non-histone methylation event regulating genome integrity. KDM7B antagonizes SMYD3 activity by maintaining low levels of methylated RNF113A, thereby limiting ASCC activation and sensitizing cancer cells to alkylating agents. To dissect this regulation in depth, we focused on small cell lung cancer (SCLC), a particularly aggressive malignancy characterized by limited therapeutic options and rapid acquisition of resistance. In SCLC, high KDM7B levels correlate with improved patient prognosis, whereas xenografts with reduced expression exhibit diminished responses to alkylating treatment. Moreover, CRISPR-based on/off modulation of KDM7B in genetically engineered SCLC mouse models demonstrates its central role in determining tumor response to chemotherapy. Our findings position the RNF113A–ASCC axis as a central modulator of chemoresistance, regulated through a post-translational methylation switch representing an innovative therapeutic vulnerability that could be exploited to enhance the efficacy of alkylating agents. Targeting this pathway may provide new opportunities to overcome chemoresistance, with KDM7B levels serving as a predictive biomarker to guide treatment in SCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fadb48eb1c5f8acecabf2f1b43e895eccc0dda31" target='_blank'>
              KDM7B-mediated demethylation of RNF113A regulates small cell lung cancer sensitivity to alkylation damage
              </a>
            </td>
          <td>
            T. Ahmad, Xiao-jun Yang, A. Foucher, Lingnan Ren, Ning Tsao, Natasha M. Flores, Jinkai Wan, L. Belmudes, E. Dubiez, Monika Chandan Bhowmik, Jessica Vayr, Simone Hausmann, Florent Chuffart, Xiaoyin Lu, Sandrine Blanchet, Tourkian Chasan, F. Boussouar, Yohann Couté, N. Mosammaparast, Fei Lan, Jan Kadlec, Pawel K. Mazur, N. Reynoird
          </td>
          <td>2026-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03fd1dc214f11e4ea8479907e0ce554ac39fce4e" target='_blank'>
              Changes in EGFR activity following CRISPR/Cas9-editing of the EGF binding domain
              </a>
            </td>
          <td>
            Jelena Popović, Anna Hahut, G. Torres, Andrew Vincent, Nashaly Soto-Echevarria, Brian Wray, Elizabeth T. Bartom, T. Paunesku, C. Goodman, G. Woloschak
          </td>
          <td>2026-01-31</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Background Intratumor heterogeneity (ITH), a critical driver of tumor evolution and immune evasion, remains inadequately characterized at the transcriptomic level in colorectal cancer (CRC), and its clinical implications are not yet fully understood. Methods We integrated transcriptomic datasets from TCGA-COAD/READ and two independent GEO cohorts (GSE40967 and GSE87211) to develop an RNA-seq–based ITH score using the DEPTH2 algorithm and to con0struct an ITH-related gene (ITRG) prognostic model. A unified cutoff value of 0.64 was established to stratify patients into high- and low-ITH groups. Using 52 survival-associated ITRGs, we generated a nine-gene prognostic signature selected from 101 distinct combinations of feature selection techniques and modeling algorithms and validated its performance in two external datasets. The tumor microenvironment and potential responsiveness to immune checkpoint inhibitors were evaluated using ssGSEA, ESTIMATE, and TIDE algorithms. SHAP analysis, together with in vitro and in vivo experiments, was employed to identify and functionally validate key regulatory genes. Results Patients in the high-ITH group had markedly poorer overall survival (OS) than those in the low-ITH group. The ITH score correlated strongly with aggressive clinical features, including T3/4 invasion depth, N1/2 nodal status, and AJCC stage III. The nine-gene prognostic signature demonstrated consistent predictive capability across the TCGA training cohort and both GEO validation cohorts. In TCGA, the model yielded time-dependent AUCs of 0.669, 0.664, and 0.645 for 1-, 3-, and 5-year OS, respectively, and retained its status as an independent prognostic indicator in multivariate Cox regression analysis; in GSE40967 and GSE87211, the corresponding AUCs ranged from 0.544–0.573 and 0.648–0.744. The high-risk subgroup was characterized by a stromal immune phenotype enriched with cancer-associated fibroblasts and macrophages, elevated expression of multiple immune checkpoints, increased TIDE scores, and higher tumor mutational burden. SHAP analysis identified IL20RB as the top risk-associated gene, whose knockdown significantly suppressed CRC cell proliferation, migration, invasion, and tumorigenicity in vitro and in vivo. Conclusion This study introduces and validates a transcriptomic ITH score and a nine-gene ITRG-based prognostic model that delineate the immune landscape and enables effective survival stratification in CRC, complementing the limitations of current staging systems. Additionally, IL20RB is highlighted as a promising therapeutic target, supporting the development of personalized immuno-targeted combination therapies in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/385eed67dbbc79210270a5630b0149ac9e500c8e" target='_blank'>
              Intratumor heterogeneity score reveals immune landscape and survival stratification in colorectal cancer
              </a>
            </td>
          <td>
            Zijing Wang, Liyuan Ma, Jinzhong Cao, Xi Chen, Xin Chang, Ruxue Ma, Hengyi Lv, Zixin Zhang, Hai Li, Tao Jiang
          </td>
          <td>2026-01-05</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="In a recent study published in Nature, Cosenza et al. 1 introduced MAGIC (machine-learning-assisted genomics and imaging convergence), a versatile platform that photolabels cells with micronuclei and other nuclear atypia, enabling direct quanti ﬁ cation of de novo chromosomal abnormality (CA) formation and ongoing chromosomal instability (CIN) at single-cell resolution. Using this framework, the authors show how spontaneous and induced lesions from pre-mitotic DNA damage and mitotic errors drive distinct CIN trajectories, with TP53 inactivation markedly amplifying CA burden and complexity, highlighting how CIN emergence and propagation prior to clonal expansion may inform tumour initiation, progression and the identi ﬁ cation of actionable biomarkers and therapeutic vulnerabilities. CIN is the ongoing process by which cells acquire new CAs through repeated chromosome missegregation and structural rearrangements, altering chromosome number and structure across divisions (Fig. 1a). CAs are pervasive across cancers and typically affect more genes than substitutions or small insertion/ deletion alterations, fuelling clonal diversi ﬁ cation and tumour adaptation. 2 – 5 Accurate chromosome segregation and genome integrity rely on coordinated processes such as DNA replication and repair, telomere maintenance, spindle assembly and mitotic exit. 2 When these are compromised and cells bypass senescence and apoptosis, concurrent pre-mitotic and mitotic defects drive ongoing CIN. 2,5 However, most CIN">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0c5eb4ff0382eff914e4d728cb5754b2a5b73b7" target='_blank'>
              It’s a kind of MAGIC: uncovering the origins of chromosomal instability
              </a>
            </td>
          <td>
            D. Gómez-Peregrina, César Serrano
          </td>
          <td>2026-02-05</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Epstein–Barr virus (EBV) is an endemic herpesvirus implicated in autoimmunity, cancer and neurological disorders. Although primary infection is often subclinical, persistent EBV infection can drive immune dysregulation and long-term complications. Despite the ubiquity of infection, the determinants of EBV persistence following primary exposure remain poorly understood, although human genetic variation partially contributes to this phenotypic spectrum1, 2–3. Here we demonstrate that existing whole genome sequencing (WGS) data of human populations can be used to quantify persistent EBV DNA. Using WGS and health record data from the UK Biobank (n = 490,560) and All of Us (n = 245,394), we uncover reproducible associations between blood-derived EBV DNA quantifications and respiratory, autoimmune, neurological and cardiovascular diseases. We evaluate genetic determinants of persistent EBV DNA via genome association studies, revealing heritability enrichment in immune-associated regulatory regions and protein-altering variants in 148 genes. Single-cell and pathway level analyses of these loci implicate variable antigen processing as a primary determinant of EBV DNA persistence. Further, relevant gene programs were enriched in B cells and antigen-presenting cells, consistent with their roles in viral reservoir and clearance. Human leukocyte antigen genotyping and predicted viral epitope presentation affinities implicate major histocompatibility complex class II variation as a key modulator of EBV persistence. Together, our analyses demonstrate how re-analysis of human population-scale WGS data can elucidate the genetic architecture of viral DNA persistence, a framework generalizable to the broader human virome4. Population-scale WGS reveals genetic determinants of persistent EBV DNA, linking immune regulation—especially antigen processing and MHC class II variation—to EBV persistence and heterogeneous disease associations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/505426b60f1eb9d87e7f038fc4c71105366741ae" target='_blank'>
              Population-scale sequencing resolves determinants of persistent EBV DNA
              </a>
            </td>
          <td>
            Sherry S Nyeo, Erin M. Cumming, O. Burren, Meghana S. Pagadala, Jacob C. Gutierrez, Thahmina Ali, Laura C. Kida, Yifan Chen, Hoyin Chu, Fengyuan Hu, Xueqing Zoe Zou, Benjamin Hollis, M. Fabre, S. MacArthur, Quanli Wang, Leif S. Ludwig, Kushal K. Dey, S. Petrovski, R. Dhindsa, Caleb A. Lareau
          </td>
          <td>2026-01-28</td>
          <td>Nature</td>
          <td>1</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a27e17b141d07dfd3357f3ee906c1f3a2792dd3c" target='_blank'>
              SigDyn: single-cell mutational signature dynamics
              </a>
            </td>
          <td>
            S. Costa, Joana P. Gonçalves
          </td>
          <td>2026-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Identifying the mutant genes that are selected during carcinogenesis is key to identifying candidates for intervention and understanding the processes that promote transformation. Here we applied two selection metrics to study the dynamics of mutational selection in a mouse model of ultraviolet light driven skin carcinogenesis in which multiple synchronous tumors develop in each animal. Sequencing normal skin and tumors over a time course revealed two genetic routes to squamous carcinoma. Nonsynonymous Trp53 mutants were positively selected in both epidermis and tumors and present in 90% of tumors. The remaining tumors carried other oncogenic mutants, including activating Kras mutations. However, other positively selected mutant genes lost their competitive advantage in heavily mutated epidermis and in tumors. We found ten mutant genes under negative selection in normal skin, one of which was also negatively selected in tumors. In addition one gene was negatively selected in tumors but not normal skin. We conclude that analysing selection in normal tissue alongside tumors may resolve the dynamics of selection in carcinogenesis and refine the identification of cancer drivers. Sequencing mutants in both normal skin and tumors that arise from it in a mouse model of ultraviolet light driven carcinogenesis reveals mutant selection changes as cancers develop. Only p53 mutants are selected throughout squamous carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4638b6cb2b56a741aa25d0049e47e04146d5e21" target='_blank'>
              The dynamics of mutational selection in cutaneous squamous carcinogenesis
              </a>
            </td>
          <td>
            G. Skrupskelyte, J. Fowler, S. Dentro, Carine Winkler, Irina Abnizova, Niklaus Beumer, R. Sood, Thomas Quarrell, C. King, J. Kamarashev, E. Guenova, M. Gerstung, Benjamin A. Hall, Liliane Michalik, P. Jones
          </td>
          <td>2026-01-12</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Oncofetal reprogramming has recently emerged as a critical concept in translational cancer research, particularly for its role in driving therapeutic resistance across a variety of malignancies. This biological process refers to a pattern of gene expression that is restricted to embryogenesis, but becomes expressed again in a subpopulation of cancer cells. These genes are typically suppressed after embryogenesis, and their aberrant re-expression in tumors endows cancer cells with stem-like properties and enhanced adaptability. The goal of this review is the following: (i) comprehensively examine the multifaceted nature of oncofetal reprogramming; (ii) elucidate its underlying molecular mechanisms, including its regulators and effectors; and (iii) evaluate its consequences for the therapeutic response in different cancer types. We comprehensively integrate the latest findings from colorectal, breast, lung, liver, and other cancers to provide a detailed understanding of how oncofetal programs interfere with tumor response to treatment. Among the candidates, YAP1 and AP-1 have emerged as central transcriptional drivers of this reprogramming process, especially in colorectal and breast cancers. We also explore the distinct expression patterns of oncofetal genes across different tumor types and how these patterns correlate with treatment outcomes and patient survival. Lastly, we propose a dual-targeting therapeutic strategy that simultaneously targets both cancer stem cells and oncofetal-reprogrammed populations as a more effective approach to overcome resistance and limit recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a262b7681553fb103b5d82c8b95c6fdfada4bf5" target='_blank'>
              Oncofetal Reprogramming: A New Frontier in Cancer Therapy Resistance
              </a>
            </td>
          <td>
            Anh L Nguyen, Molly Lausten, Bruce M. Boman
          </td>
          <td>2026-01-29</td>
          <td>International Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Acute lymphoblastic leukemia (ALL) is the most frequent malignancy in children. Although survival rates have improved in recent decades, relapse and therapy-related toxicities remain major challenges in pediatric ALL. Pharmacogenetics is an approach with the potential to refine precision medicine in pediatric ALL. This review synthesizes current knowledge on pharmacogenetic markers of drug efficacy, toxicity, and adverse effects for drugs used in childhood ALL, including glucocorticosteroids, vincristine, asparaginase, anthracyclines, 6-mercaptopurine, and methotrexate. Thiopurine methyltransferase (TPMT) and nudix hydrolase 15 (NUDT15) are the only pharmacogenetic markers with established clinical guidelines that inform thiopurine and methotrexate dosing. Insufficient and inconsistent evidence has limited the translation of most other pharmacogenetic findings into clinical practice. While most studies centered on coding variants, we also point to new approaches focusing on noncoding regions and epigenetic variation that hold promise for expanding the scope of clinically relevant biomarkers in the near future. Finally, we propose future research avenues, that could support the development of more individualized treatment strategies for children with ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ce8daf206f4436421f31c77c3235c25258ca01b" target='_blank'>
              The role of pharmacogenetics in the therapy regimen of acute lymphoblastic leukemia in children: recent discoveries and future perspectives
              </a>
            </td>
          <td>
            Anna Szoszkiewicz, Alina Światłowska, Katarzyna Derwich, A. Jamsheer, B. Rubiś, A. Bienert
          </td>
          <td>2026-01-05</td>
          <td>Pharmacological Reports</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="The B‐cell lymphoma 2 (BCL‐2) inhibitor venetoclax (VEN) in combination with hypomethylating agents (HMAs) has improved treatment outcomes for acute myeloid leukemia (AML) patients unfit for intensive chemotherapy and is increasingly used in the relapsed/refractory setting. However, primary resistance remains a significant challenge, affecting 20%–35% of treatment‐naïve and around 50% of previously treated AML patients. To investigate the mechanisms driving primary resistance to VEN–HMA therapy, we analyzed genetic, transcriptomic, BCL‐2 family protein expression, and ex vivo drug sensitivity data from 101 AML patients and correlated these profiles with clinical outcomes to VEN–HMA. Our study found that blasts from refractory patients exhibit an elevated BCL‐XL/BCL‐2 protein expression ratio, an immature CD34+CD38− phenotype, and frequent TP53 mutations. Consistent with the high ratio of BCL‐XL/BCL‐2, resistant samples showed increased ex vivo sensitivity to the dual BCL‐2/BCL‐XL inhibitor navitoclax. In addition, SMAC mimetics were effective in refractory blasts, which correlated with high TNF gene expression in these cells. Ex vivo treatment with the combination of navitoclax and SMAC mimetics further enhanced the eradication of VEN–HMA refractory blasts, although toxicity was also observed in healthy CD34+ cells. In conclusion, our integrative analysis identifies molecular signatures associated with primary VEN–HMA resistance and highlights BCL‐2/BCL‐XL inhibition and SMAC mimetics as therapeutic strategies to target resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f404303a3f5b827792610f3342a27ad893ee2ef5" target='_blank'>
              Drivers of clinical resistance to venetoclax and hypomethylating agents in acute myeloid leukemia and strategies for improving efficacy
              </a>
            </td>
          <td>
            Ida Vänttinen, Joseph Saad, Tanja Ruokoranta, S. Kytölä, Guangrong Qin, Bahar Tercan, Pia Ettala, A. Partanen, M. Pyörälä, J. Rimpiläinen, T. Siitonen, Mikko Manninen, P. Valk, G. Huls, V. Thorsson, C. Heckman, Mika Kontro, H. Kuusanmäki
          </td>
          <td>2026-01-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d7ff048942859a6d99556412c78538071a76412" target='_blank'>
              Apc truncation enforces optimal Wnt signalling during early intestinal tumourigenesis
              </a>
            </td>
          <td>
            M. V. van Driel, Nina E. de Groot, Sander R. van Hooff, Jan Koster, Louis Vermeulen, Sanne M. van Neerven
          </td>
          <td>2026-02-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="

 Paclitaxel is a frontline chemotherapeutic agent that stabilizes microtubules, but its broader impact on germline genome stability and long-term tolerance mechanisms remains incompletely understood.




 Using
 Caenorhabditis elegans
 , we combined phenotypic, cytological, genetic, and transcriptomic approaches, and further validated selected findings in human A498 renal carcinoma cells.




 Paclitaxel markedly impaired fertility: hatching dropped approximately 54%, larval arrest increased approximately 4.1-fold, and brood size decreased approximately 31% at 50 μM (all P < 0.01). HIM progeny rose 15-fold, with 52% of nuclei lacking SYP-1 and widespread abnormal chromosome clustering. RAD-51 foci persisted and late-pachytene apoptosis increased 47%, indicating defective DNA damage repair. Transcriptomic analysis revealed upregulation of detoxification and immune pathways and selective downregulation of mismatch repair genes; however, genetic manipulation confirmed that MMR attenuation is a stress signature rather than a causal driver. Multigenerational exposure led to initial sensitization (F5 survival 18.9%) followed by partial recovery (F10 survival 60.1%). Parallel experiments in human kidney cancer cells revealed concentration-dependent apoptosis, reflecting the phenotypes observed in nematodes.




 This integrated analysis establishes
 C. elegans
 as a tractable
 in vivo
 platform for dissecting both proliferative and meiotic consequences of paclitaxel. By linking spindle disruption, delayed DNA repair, and apoptosis with transcriptional reprogramming, the study provides mechanistic insight into germline vulnerability, reproductive side effects, and potential adaptive responses relevant to chemotherapy.

">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52d15f04f8fdea1c86b49d44dc1b6987866d5fd0" target='_blank'>
              Paclitaxel-induced germline DNA-damage signatures independent of mismatch repair in C. elegans
              </a>
            </td>
          <td>
            Yebin Hong, Zifei Liu, Hyun-Min Kim
          </td>
          <td>2026-02-18</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="This study aimed to explore the potential genetic underpinnings of childhood acute lymphoblastic leukemia (ALL). Initially, an exome-wide association analysis of 89 ALL survivors and 356 controls was conducted to identify candidate genes, incorporating single-variant tests and annotation-weighted aggregation tests. Single-cell analyses characterized the candidate genes’ biological context, while aggregate testing linked these genes to clinical traits in survivors, along with bulk tissue transcriptomes profiling their expression patterns in relation to secondary adverse events (e.g., progression, relapse, secondary malignant neoplasms or death). Results from single-variant tests showed that rs57421986 of hematology-related gene SPTB exhibited a suggestive association with ALL [minor allele frequency = 0.029, odds ratio (OR) = 21.7, P = 5.86 × 10− 5]. Aggregation tests of low-frequency and rare variants identified MFAP3L (P = 3.97 × 10− 8), CARD11 (P = 4.18 × 10− 8), NOL8 (P = 2.46 × 10− 10), and OR7G2 (P = 2.20 × 10− 7) as ALL-associated genes. Notably, CARD11 and NOL8 demonstrated differential expression across diverse immune cells between incident cases and controls. Clinical phenome analyses linked CARD11 to mean corpuscular hemoglobin (P = 4.32 × 10− 2), alkaline phosphatase (P = 4.33 × 10− 2), and free triiodothyronine (P = 4.85 × 10− 2), NOL8 to red blood cell count (P = 2.83 × 10− 4) and mean corpuscular volume (P = 2.04 × 10− 3), and OR7G2 to thyroid-stimulating hormone (P = 9.56 × 10− 3) and alkaline phosphatase levels (P = 3.07 × 10− 2). Furthermore, a higher expression of OR7G2 at diagnosis showed a suggestive association with the occurrence of secondary adverse events (P = 0.034). Our findings provide initial clues regarding the shared genetic factors underlying both ALL onset and prognosis. These insights may offer potential guidance for refined risk stratification and personalized survivorship care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed68046e0425c98697fcedfc5d6cbfdc047e6e06" target='_blank'>
              Low frequency and rare coding variants affect susceptibility and progression of childhood acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Xiao Liu, Ru Zhang, Heng Zhang, Junyi Xin, Huiqin Li, Yongchu Pan, Liwen Zhu, Meiyun Kang, Hua Jiang, Yongjun Fang, Yao Xue, Mulong Du
          </td>
          <td>2026-01-02</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background The treatment of osteosarcoma, a highly aggressive primary malignant bone tumour, has long faced limitations due to chemotherapy resistance, tumour het-erogeneity, and an immunosuppressive microenvironment. Methods This review synthesizes recent multi-omics and clinical trial data to analyse synergistic oncogenic mechanisms in osteosarcoma—including driver mutations (TP53/RB1), epigenetic re-programming (m6A/ncRNA networks), and dysregulated pathways (PI3K/AKT, Wnt/β-catenin)—and evaluates derived therapeutic strategies. Results Targeted therapies demonstrate potential to improve prognosis in clinical trials; immunotherapies significantly enhance response rates by remodelling the cold tumour microenvironment; advanced technologies like nanotechnology and 3D-printed scaffolds over-come limitations of conventional treatments and enable integrated diagnosis and therapy. However, tumour evolutionary heterogeneity, off-target toxicity of targeted therapies, and translational gaps between animal models and clinical efficacy remain major challenges. Conclusions Future directions require integrating AI-driven imaging omics, spatiotemporal multi-omics, and mechanically adaptive biomaterials to establish a precision management system. This will advance osteosarcoma therapy from survival prolongation toward functional cure—defined as complete tumour eradication with physiological reconstruction of bone structure/function (e.g., restoring load-bearing/joint mobility), while preventing treatment-related disability, ultimately achieving oncologic cure with preserved quality of life.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb0664a23d28410159abeb654b2c4b1a41f762b2" target='_blank'>
              Research status and prospects of molecular pathological mechanisms and novel therapeutic targets of osteosarcoma: a systematic review
              </a>
            </td>
          <td>
            Huan Jin, Cai Huang, Ying Dong, Qi Xiong, Di Wang, Ziyi He, Yu Shang, Lin Shen, Chen Ma, Zixian Wang, Siwei Shi, Lingfeng Zeng, B. Shuai
          </td>
          <td>2026-01-20</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f042ff31a3cc857942699240445f6ba616d2f099" target='_blank'>
              Mechanisms and intervention strategies of microenvironment-mediated drug resistance in multiple myeloma
              </a>
            </td>
          <td>
            Xiaoling Zhu, Chen Qi, Tingting Yan, He Di, Meijia Yu
          </td>
          <td>2026-01-27</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Chemoresistance remains a major barrier in cancer therapy, frequently resulting in treatment failure and reduced patient survival. This multifaceted phenomenon arises from the interplay of well-established mechanisms such as genetic mutations, non-genetic adaptations, and tumor microenvironment (TME) mediated influences as well as newly emerging findings from recent research (2020–present). Key biochemical contributors include diminished intracellular drug accumulation through altered uptake or efflux, dysregulation of drug metabolism and bioactivation involving multiple Phase I and Phase II enzymes, genomic instability affecting DNA repair pathways, disruption of cell cycle control, and evasion of apoptosis. In addition, recent evidence highlights the roles of epigenetic reprogramming, metabolic reconfiguration, and TME-derived signaling in amplifying chemoresistance. This review integrates both foundational concepts and recent advancements in understanding drug resistance, with particular emphasis on updated insights into drug-metabolizing enzymes and their impact on therapeutic failure. It also evaluates current and emerging strategies to overcome resistance including targeting metabolic enzymes, modulating the TME, and implementing polytherapy’s that address multiple resistance pathways. By synthesizing established knowledge with recent discoveries, this review highlights promising directions for improving the efficacy of cancer treatments and enhancing patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b92ce9c249ee2d1807438a427a4bdf1ac44ab032" target='_blank'>
              Molecular basis of cancer chemoresistance: biochemical insights
              </a>
            </td>
          <td>
            R. Guneidy
          </td>
          <td>2026-01-02</td>
          <td>Drug Metabolism Reviews</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Double-Negative T (DNT) cells, lacking both CD4 and CD8 expression, play critical roles in cancer immunology, and have garnered increasing attention in cancer research. However, their heterogeneity and functional diversity within the tumor microenvironment (TME) remain underexplored. In-house and publicly available single-cell RNA sequencing (scRNA-seq) data for different cancer types were integrated after quality control and batch effect correction, followed by DNT cells separation from CD3+ T cells subtypes. Functional characteristics, intercellular communication, differentiation trajectories, regulatory networks, and clinical relevance were analyzed among different DNT subsets. Key findings were validated using multiplex immunofluorescence and spatial transcriptomics to investigate the spatial localization of DNT subsets and their interactions within the TME. Impact of γδ T cells on immunotherapy response was also assessed using MC38-based murine tumor model. By integrating scRNA-seq data from 2,369 samples across 23 cancer types, we established a comprehensive single-cell atlas of 157,025 high-quality DNT cells. Fourteen distinct DNT subsets (6 αβ DNT and 8 γδ T cell subsets) were identified, demonstrating tumor both type-specific and shared distribution patterns, as well as unique cell-cell interaction network within the TME. These subsets displayed specialized functional profiles, including cytotoxicity, antigen presentation, and immune modulation, indicating that the functional diversity of DNT cells is largely subset-specific rather than a manifestation of multifunctionality within a single population. We also delineated divergent trajectories for αβ DNT and γδ T cell subsets, including the functional plasticity of gut-resident γδ T cells transitioning between cytotoxic and immunosuppressive states. Notably, several DNT subsets were significantly associated with favorable clinical treatment outcomes, including improved responses to cancer immunotherapy. Consistently, depletion of γδ T cells in the murine tumor model significantly decreased the efficacy of PD-1 blockade, underscoring their critical role in therapeutic response. Our study uncovers the previously underappreciated heterogeneity and functional diversity of DNT cells in the TME and demonstrates their profound impact on tumor progression and immunotherapy outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecf4d4eea61e2f155251ae235dac3fc5c5c43d18" target='_blank'>
              A pan-cancer single cell landscape reveals heterogeneity and functional diversity of double-negative T cells
              </a>
            </td>
          <td>
            Qing Hao, Tao Zhou, Huayun Yan, Zhixiang Ren, Wei Mao, Li-Bin Huang, Yue-Yun Chen, Yuelan Wang, Minqin Xiao, Yajiao He, Xuyang Xia, Jing Wu, Wei-Han Zhang, Hai-Ning Chen, Maoxiang Qian, Feng Zhang, Li Yang, Lunzhi Dai, Huiyuan Zhang, Canhua Huang, Yang Shu, Yu Xu, Yang Luo, Yong Peng, Heng Xu
          </td>
          <td>2026-01-03</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Human T-lymphotropic virus type 1 (HTLV-1), the first human retrovirus identified, is linked to adult T-cell leukemia and HTLV-1-associated myelopathy/tropical spastic paraparesis. However, its post-transcriptional regulation remains poorly understood. Here, we used Oxford Nanopore direct RNA sequencing to profile the HTLV-1 transcriptome and epitranscriptome in MT2 cells. We identified 23 transcript isoforms, encompassing canonical and novel splice variants. Polyadenylation analysis revealed a predominant poly(A) tail length of around 50–100 nucleotides with transcript-specific variations. Distinct RNA modifications, including pseudouridine, N6-methyladenosine, and 5-methylcytidine, were enriched near the 3′ end and varied among transcript classes, with generally lower modification ratios in viral transcripts. These findings provide a more comprehensive map of HTLV-1 RNA splicing, polyadenylation, and modifications in MT2 cells, offering new insights into viral gene regulation and pathogenic mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a1e4aff36600c0fd3356ace5c0413dcc761d3f3" target='_blank'>
              Transcriptomic and Epitranscriptomic Landscape of Integrated HTLV-1 in MT2 Cells
              </a>
            </td>
          <td>
            Shuanglong Wei, Bohan Zhang, Jingwan Han, Hanping Li, Yongjian Liu, Lei Jia, Jingyun Li, Xiaotian Huang, Lin Li
          </td>
          <td>2025-12-30</td>
          <td>Viruses</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Therapy resistance remains a major cause of relapse and poor outcomes in acute lymphoblastic leukemia (ALL). Recent multi-omics studies in ALL have revealed that resistance arises from a combination of leukemia-specific genetic lesions, treatment-driven clonal evolution, and adaptive non-genetic programs. Genomic analyses have identified recurrent alterations associated with resistance to chemotherapy, tyrosine kinase inhibitors, and immunotherapies, while single-cell profiling has uncovered heterogeneous cell states that persist during treatment and contribute to minimal residual disease. Emerging epigenetic, proteomic, and metabolic data further indicate that reversible regulatory and signaling changes play a central role in leukemic persistence. Integrative analyses are beginning to define convergent resistance pathways and clinically relevant biomarkers, although longitudinal sampling and clinical translation remain limited. This review summarizes the current multi-omics landscape of therapy resistance in ALL and discusses opportunities to improve risk stratification and therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98ef9b63ea903857bf083205a34581a4f1ebb5b7" target='_blank'>
              Multi-Omics Approaches to Understanding Therapy Resistance in Acute Lymphoblastic Leukemia
              </a>
            </td>
          <td>
            Xiuyun Wu, Jingxin Zhang
          </td>
          <td>2026-01-29</td>
          <td>Lymphatics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Outcomes for patients with esophageal squamous cell carcinoma (ESCC) remain poor, partly due to treatment resistance, particularly to DNA-damaging therapies. Poly(ADP‑ribose) polymerase 1 (PARP1) plays a critical role in repairing single‑strand DNA breaks (SSBs). Unrepaired SSBs can be converted into double‑strand breaks (DSBs) during DNA replication, potentially leading to cell death. Genomic amplification of the distal portion of chromosome 3q (3q26-q29) is a frequent copy‑number alteration in ESCC, which harbors genes encoding several oncoproteins. However, whether long noncoding RNAs (lncRNAs) from this region contribute to ESCC pathogenesis and treatment resistance remains poorly understood. In situ hybridization and qPCR were used to assess RNA expression. Protein PARylation was evaluated by immunoprecipitation followed by Western blotting. Cellular phenotypes were quantified using the cell counting kit-8, Annexin V/Propidium iodide staining, and clonogenic assays. DNA damage was monitored by immunofluorescence staining for phosphorylated histone H2AX (γH2AX) and p53-binding protein 1 (53BP1) and by comet assays. RNA-protein interactions were assessed through RNA pulldown coupled with mass spectrometry and RNA immunoprecipitation. Chromatin fractionation and detergent pre-extraction immunofluorescence were conducted to examine PARP1 chromatin association. ESCC growth and responses to treatments were evaluated using xenograft models. The lncRNA LINC00885, hereafter referred to as PARylator, was the most upregulated lncRNA encoded within the 3q26-q29 amplicon in ESCC. PARylator was predominantly nuclear and interacted with PARP1. Knockdown of PARylator increased γH2AX and 53BP1 foci and comet tail moment, triggered apoptosis, reduced clonogenicity, sensitized ESCC cells to cisplatin and ionizing radiation. In vivo, PARylator knockdown impaired tumor growth and increased cisplatin sensitivity in ESCC xenografts. Mechanistically, PARylator promoted PARP1 recruitment to chromatin and catalytic activation, thereby increasing PARP1 auto-PARylation and enhancing the PARylation of X-ray repair cross-complementing 1 (XRCC1). PARylator was further upregulated in response to DNA damage. The DNA damage-responsive, 3q26-q29 amplicon-encoded lncRNA PARylator promotes PARP1‑mediated PARylation and SSB repair, thereby limiting DSB accumulation and supporting ESCC cell survival and resistance to DNA-damaging therapies. Targeting PARylator, alone or in combination with DNA-damaging agents, may represent a novel avenue for ESCC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ecafccdcfd377c547198645eff6618f1001f7c3" target='_blank'>
              The lncRNA PARylator promotes PARP1 activation and resistance to DNA‑damaging therapy in esophageal squamous cell carcinoma
              </a>
            </td>
          <td>
            Teng Fei Qi, Yadong Liu, Jinkun Yang, Yi Meng Yue, Man Man Han, Hongtao Liu, Jing Li, Min Liu, Yuwei Zhang, Jia Kou, Wen Jin Li, Xiaoying Liu, Ting La, Tao Liu, Song Chen, Xu Dong Zhang, Shundong Cang, Liu Teng, Tianli Fan
          </td>
          <td>2025-12-31</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Cancer cells exhibit remarkable plasticity, enabling them to survive therapeutic pressure by dynamically rewiring both their epigenetic landscape and metabolic circuitry. Emerging evidence reveals that epigenetic mechanisms, including DNA methylation, histone modifications, chromatin remodeling, and non-coding RNAs, are tightly coupled to metabolic pathways through key metabolites that function as cofactors or regulators of chromatin-modifying enzymes. This reciprocal interplay establishes self-reinforcing loops that sustain tumor growth, promote heterogeneity, and drive the emergence of drug-tolerant states. In this review, we summarize current knowledge on how epigenetic remodeling shapes metabolic reprogramming and, in turn, how altered metabolite pools influence chromatin states in cancer. We highlight the central role of long non-coding RNAs and other ncRNA species in coordinating epigenetic and metabolic adaptations that underpin therapy resistance. We further examine the contribution of metabolite-dependent post-translational modifications, such as acetylation, methylation, lactylation, and succinylation, to the regulation of tumor aggressiveness and treatment response. Finally, we discuss how multi-omics integration, computational network approaches, and AI-enabled modeling are accelerating the discovery of epigenetic–metabolic vulnerabilities and informing the development of precision therapeutic strategies. Understanding and targeting this epigenetic–metabolic axis holds substantial promise for overcoming drug resistance and improving the durability of cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe9160a0725796061efe9744e0d756322382fe38" target='_blank'>
              The impact of epigenetics on tumor metabolism: Friend or foe in drug response?
              </a>
            </td>
          <td>
            Serena Castelli, M. Franchini, Luigi Ferraro, Nicolò Mangraviti
          </td>
          <td>2026-02-09</td>
          <td>Frontiers in Epigenetics and Epigenomics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fef918e1d4b64862104d2dc24b6aa7569489a94" target='_blank'>
              Genomic profiling implicates candidate genes and mutagenic pathways driving lung cancer recurrence
              </a>
            </td>
          <td>
            L. Luhari, A. Valter, A. Bahcheli, K. C. Cheng, M. Bayati, A. Ustav, A. Velthut-Meikas, K. Oselin, J. Reimand
          </td>
          <td>2026-01-01</td>
          <td>None</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb493837934e32a1361cf64825c6efc8b0b26882" target='_blank'>
              Pan-cancer RNA editing activity reveals complex editing functions and cancer immunotherapy biomarkers
              </a>
            </td>
          <td>
            M. Guo, Yuanyan Xiong
          </td>
          <td>2026-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) continues to pose significant therapeutic challenges, with high relapse rates driven largely by leukemic stem cells (LSCs), a rare, therapy-resistant population with self-renewal capacity, niche adaptation, and the ability to re-initiate disease. In this state-of-the-art review, we synthesize recent advances in LSC biology, addressing (i) how LSCs differ functionally and phenotypically from normal hematopoietic stem cells (HSCs), (ii) practical approaches for LSC quantification using multiparameter flow cytometry and LSC-enriched marker panels, (iii) the dysregulated signaling, metabolic and epigenetic programs that enable LSC persistence under chemotherapy and contribute to measurable residual disease, and (iv) current therapeutic strategies targeting LSC eradication, including antibody-based therapies, apoptosis and metabolic inhibitors, and emerging epigenetic agents. We also examine the key translational barriers, particularly antigen overlap with normal progenitors, microenvironmental protection, and the need for assay harmonization, while proposing a practical framework for integrating LSC assessment into risk stratification and therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cd1cbd5980c8002b0d9b3730098739618a3c127" target='_blank'>
              Decoding Leukemic Stem Cells in AML: From Identification to Targeted Eradication
              </a>
            </td>
          <td>
            E. Apostolidou, Vasileios Georgoulis, Dimitrios Leonardos, Leonidas Benetatos, Eleni Kapsali, E. Hatzimichael
          </td>
          <td>2026-01-30</td>
          <td>Diseases</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33d403636ddf059830232f0617d3f6cb6b77d1ff" target='_blank'>
              Genotype-phenotype mapping identifies fetal-like CD55+ immunoregulatory cancer cells as mediators of immune escape in colorectal cancer
              </a>
            </td>
          <td>
            M. Mastel, Aitana Guiseris Martinez, Gabriele Diamante, Jasmin Meier, Luzi Schuchmann, Nikolaos Georgakopoulos, Ioannis Chiotakakos, Laura K. Steffens, Carolin Artmann, Diego Benitez, Michael Günther, Vera Thiel, Roxana Pincheira, A. Trumpp, R. Offringa, S. Ormanns, Rene Jackstadt
          </td>
          <td>2026-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Mitochondria are highly dynamic organelles that integrate metabolic regulation, signal transduction, and programmed cell death with their canonical role in adenosine triphosphate (ATP) production. Their ability to undergo continuous remodeling through the opposing processes of fusion and fission is essential for maintaining cellular homeostasis, preserving organelle quality control, and enabling adaptive responses to metabolic and oxidative stress. Among the core regulators of mitochondrial dynamics, the dynamin-related guanosine triphosphatase (GTPase) OPA1 plays a central role in inner membrane fusion, cristae architecture maintenance, bioenergetic efficiency, and the modulation of redox balance and apoptotic signaling. Accumulating evidence indicates that dysregulation of OPA1 expression or activity contributes to the initiation and progression of multiple malignancies, underscoring its importance in tumor cell survival, proliferation, metabolic adaptation, and resistance to stress. Here, we summarize current knowledge on OPA1 dysregulation in cancer and, based on preliminary, unpublished in silico analyses, we highlight the growing relevance of OPA1 as a therapeutic target, particularly through its GTPase domain and the still understudied Interface 7. Overall, these findings outline how integrated computational approaches could potentially guide the identification of novel OPA1 modulators, offering a conceptual framework that highlights OPA1 as a promising, yet still largely underexplored, target in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01bbc85b07d21254bc1200e427724eda318625e7" target='_blank'>
              OPA1 as a Cancer Target: Molecular Mechanisms, Structural Insights, and Strategies for Drug Development
              </a>
            </td>
          <td>
            Antonio Curcio, Ludovica Ganino, Ilenia Valentino, Massimo Gentile, Stefano Alcaro, R. Rocca, A. Artese, N. Amodio
          </td>
          <td>2026-01-01</td>
          <td>Antioxidants</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/062e9996a43c3e3095b1126f2396b4c755a5c6ba" target='_blank'>
              Expanding the genetic landscape of inherited metabolic diseases using long-read sequencing and transcriptomic profiling.
              </a>
            </td>
          <td>
            Alejandro Soriano-Sexto, Obdulia Sánchez-Lijarcio, Leonardo Beccari, Natalia Castejón-Fernández, F. Leal, P. Alcaide, Belén de la Morena-Barrio, M. Bahíllo‐Curieses, Patricia Correcher, Rafael Hencke-Tresbach, L. López, Elena Martín-Hernández, Raquel Yahyaoui, Magdalena Ugarte, P. Rodríguez-Pombo, Belén Pérez
          </td>
          <td>2026-01-26</td>
          <td>European journal of human genetics : EJHG</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a16bf776576d5a9b29010f431eccdc0d7ad27b5b" target='_blank'>
              Mutational signatures in cancer genomes alter protein sequence motifs in cellular signaling networks
              </a>
            </td>
          <td>
            Jigyansa Mishra, M. Bayati, Zoe P. Klein, K. C. Cheng, Omar M. Wagih, Nina Adler, Maria Farina-Morillas, A. Bahcheli, J. Reimand
          </td>
          <td>2026-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83abedd58342641982b4fa326716bab2f81320dd" target='_blank'>
              Coordinate Up-regulation of Both TK1 and TS Expression in Cycling Cells: Mechanistic Basis and Implications for Dual-Targeted Cancer Therapy
              </a>
            </td>
          <td>
            Wanxing Wang, Jiawei Zhang, Guangyou Duan, Yiyao Zhang, Guofeng Xu, Zhi Cheng, Shunmei Chen, Xue Yao, Xin Li, Shan Gao
          </td>
          <td>2026-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Transposable Elements (TEs) represent a class of mobile genomic sequences, which may seriously disrupt gene regulation and can contribute to tumorigenesis. Yet, their role in NSCLC has remained unexplored to a great degree. Therefore, an integrated transcriptomic and Transposable Element (TE) analysis was performed to investigate TE-driven gene dysregulation in NSCLC. Hierarchical clustering of differentially expressed TE revealed significant over-representation of LTR1A1 and HERVL18-int in the cancer samples, with notably high expression of LINE and ERV members, especially HERVL-int, L1MC5, and L1M5. The intersection of TE expression with differentially expressed genes revealed several TE-associated genes involved in cell cycle regulation, genomic stability, and tumor progression. Fusion transcript analysis highlighted unique cancer-specific events, offering insights into TE-mediated transcriptomic alterations. Molecular docking of TE-associated proteins, HMMR, and PBK suggested potential interactions that may influence oncogenic pathways. Collectively, our findings uncover novel TE-driven mechanisms of gene dysregulation in NSCLC and highlight specific TEs and associated genes as potential diagnostic markers and therapeutic targets, offering a framework for future experimental studies to explore their mechanistic and clinical significance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a971dca83ca242097742334d02952f660161515" target='_blank'>
              Unraveling novel transposable elements (TEs)-driven gene dysregulation in non-small cell lung cancer (NSCLC) by integrated transcriptomic and TEs analysis.
              </a>
            </td>
          <td>
            Sahadevan Shrinidhi, R. Sagaya Jansi, A. Khusro
          </td>
          <td>2026-02-01</td>
          <td>Computational biology and chemistry</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Replication stress (RS) is a hallmark of cancer, largely driven by oncogene activation. Due to high levels of RS, cancer cells depend heavily on the RS response mechanisms to avoid DNA damage. This dependency creates a therapeutic opportunity that can be exploited for more effective cancer treatment. This review synthesizes current mechanistic understanding of RS and RS response and further describes how targeted disruption of RS response proteins (ATR, Chk1, Wee1, PARP, RPA) has been used in preclinical and clinical studies. We summarize preclinical and emerging clinical evidence for exploiting RS for radiosensitization, and outline candidate biomarkers and functional assays for patient selection. We also highlight the links between RS, therapy-induced senescence and innate immune activation via the cGAS–STING (cyclic GMP-AMP synthase—Stimulator of Interferon Genes) pathway, and address current challenges and future directions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/290040ba46caa91c5606fcee3d46699f73704969" target='_blank'>
              Replication Stress in Cancer: Mechanistic Insights and Therapeutic Opportunities for Radiosensitization
              </a>
            </td>
          <td>
            Spyridon N. Vasilopoulos, Ioanna Tremi, I. Kotta-Loizou, A. Gkikoudi, O. Tsitsilonis, Sophia Havaki, A. Georgakilas
          </td>
          <td>2026-01-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="The increasing complexity of disease mechanisms challenges accurate diagnosis, prevention, and early risk stratification. Beyond genetic predisposition, epigenetic regulation—particularly DNA methylation—represents a dynamic molecular interface linking environmental exposures, metabolic imbalance, inflammation, and disease development. DNA methylation is the most extensively studied epigenetic mechanism and plays a central role in controlling gene expression across physiological and pathological conditions. In this review, we provide an integrated overview of DNA methylation biology and its involvement in inflammatory, metabolic, and oncological diseases, with a specific focus on pathways related to chronic inflammation and oxidative stress. We summarize evidence demonstrating how aberrant methylation patterns contribute to disease initiation and progression, highlighting recurrent epigenetic signatures affecting key regulatory genes. In parallel, we discuss current and emerging technologies for DNA methylation analysis, ranging from targeted methylation-specific assays to next-generation sequencing-based approaches, including nanopore adaptive sampling. Finally, we explore the translational potential of DNA methylation-based tests as predictive and preventive tools, emphasizing their ability to identify disease-associated molecular alterations before clinical onset. Overall, this evidence supports the integration of epigenetic profiling into future precision medicine strategies aimed at early risk assessment, prognosis refinement, and personalized prevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99748d02c4116250f185f476ead0e0e3f150a9ca" target='_blank'>
              A New Complexity Layer: DNA Methylation and the Predictive Impact of Epigenetic Tests
              </a>
            </td>
          <td>
            Giorgio Ladisa, Francesca Montenegro, A. Picerno, Alessio Nigro, Antonella Cicirelli, A. Stasi, M. Fiorentino, P. Pontrelli, L. Gesualdo, Fabio Sallustio
          </td>
          <td>2026-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Abstract Cancer development is driven by somatic evolution and clonal selection. However, traditional selective pressure analysis methods have treated all sites within a gene equally, such a gene-level model oversimplifies the complexity of cancer evolution. In this study, we introduced CN/CS-calculator, a novel site-specific method that can capture selective pressures acting across different gene sites. By deciphering the interplay between the selection pattern and the function of a gene in oncogenesis, CN/CS-calculator uncovers a unique class of mini-driver genes, which exhibit weak positive selection, with certain critical sites providing context-dependent promoter effects on the fitness of cancer subclones while others are constrained by evolutionary conservation. Our method emphasizes the importance of site-specific analysis in uncovering how subtle evolutionary forces shape cancer biology. The refined understanding offers new insights into the mechanisms of cancer heterogeneity and molecular evolution, with potential implications for advancing therapeutic strategies and prognostic assessments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f1b33838c929960cc9e2d6ebf3f9c1d9939a9da" target='_blank'>
              Identification of cancer mini-drivers by deciphering selective landscape in the cancer genome
              </a>
            </td>
          <td>
            Xunuo Zhu, Wenyi Zhao, Siqi Wang, Jingwen Yang, Jingqi Zhou, Binbin Zhou, Ji Cao, Bo Yang, Zhan Zhou, Xun Gu
          </td>
          <td>2026-01-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Abstract The development of biomarkers for population screening, early cancer detection, monitoring, and recurrence surveillance offers substantial potential to improve patient outcomes and save lives. Nullomers are short k-mers that are absent from a human genome, and neomers are the subset of nullomers that emerge recurrently due to somatic mutations during cancer development. Here, we have developed neomerDB, a database that encompasses a catalogue of neomers across cancer types and organs. We examined 10 000 whole exome sequencing and 2658 whole genome sequencing tumour-matched samples and identified the set of neomers associated with each cancer type and organ. We also analysed 76 215 whole genomes and 730 947 whole exomes of individuals from diverse ancestries, from which we removed nullomers and neomers that can arise due to germline variants in the population. Finally, we conducted a case study demonstrating that neomers can be utilized to detect glioblastoma from liquid biopsy samples (n = 38), utilizing cell-free DNA and cell-free RNA, achieving a Receiver Operating Characteristic - Area Under the Curve score of 0.98 and a precision-recall score of 0.99. neomerDB is a user-friendly database that enables advanced searches, provides interactive visualizations, and download options for neomer biomarkers. neomerDB is publicly available at https://neomerDB.com/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d61e31009fa3725049e1e6c9f385fb155746c12" target='_blank'>
              neomerDB: a comprehensive database of neomer biomarkers in cancer
              </a>
            </td>
          <td>
            Kimonas Provatas, Candace S. Y. Chan, Ioannis Kerasiotis, Eleftherios Bochalis, Akshatha Nayak, Brad E Zacharia, Georgios A. Pavlopoulos, Wei Li, IIias Georgakopoulos-Soares
          </td>
          <td>2026-01-15</td>
          <td>Database: The Journal of Biological Databases and Curation</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="ABSTRACT Secondary plasma cell leukemia (sPCL) is a rare, aggressive manifestation of multiple myeloma (MM). We report a 75‐year‐old Japanese man with anemia as the chief complaint and IgG‐λ MM that rapidly progressed to sPCL. Whole genome sequencing using Canopy revealed a major clone with a monoallelic TP53 mutation. During progression, a subclone with a biallelic TP53 mutation expanded, and the 1q21 copy number increased. Single‐cell RNA‐sequencing identified an emergent PCL population with high CKS1B expression. These data demonstrate genomic instability and clonal evolution during sPCL development, underscoring the need for approaches accounting for tumor heterogeneity in MM. Trial Registration: The authors have confirmed clinical trial registration is not needed for this submission.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2edcd3927754b5f89f94a2aa218c66d909ec380e" target='_blank'>
              Whole Genome and Single‐Cell RNA Sequencing Reveals Clonal Evolution and Heterogeneity of Secondary Plasma Cell Leukemia: A Case Report
              </a>
            </td>
          <td>
            Tomotaka Suzuki, Rui Yokomori, T. Sanda, M. Ri, Shinsuke Iida
          </td>
          <td>2026-01-13</td>
          <td>EJHaem</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Tumor adaptation and therapeutic resistance represent major limitations to sustained treatment efficacy in modern oncology. Despite advances in targeted therapies and personalized approaches, disease progression and relapse remain frequent, reflecting the intrinsic biological complexity of malignant tumors. This review synthesizes high-impact evidence addressing the mechanisms that drive tumor adaptation under therapeutic pressure, with particular emphasis on intratumoral heterogeneity, evolutionary dynamics, and cellular plasticity. The analyzed literature supports the view of cancer as a dynamic and evolving ecosystem composed of genetically and phenotypically diverse subpopulations that respond heterogeneously to treatment. Resistance emerges as a predictable consequence of Darwinian selection, shaped by both genetic alterations and non-genetic adaptive processes, including reversible drug-tolerant states and microenvironmental influences. The findings further highlight the limitations of static, single-time-point molecular profiling in capturing tumor complexity and guiding long-term therapeutic decisions. Evolution-informed strategies, such as adaptive therapy and rational treatment sequencing, are discussed as biologically grounded approaches that may improve disease control by managing, rather than attempting to eradicate, heterogeneous tumor populations. This integrative perspective is particularly relevant for diverse healthcare settings, including those in Latin America, where conceptual frameworks can support clinical decision-making in variable resource contexts. Overall, understanding tumor adaptation and resistance as emergent evolutionary phenomena provides a coherent foundation for advancing personalized oncology, improving clinical reasoning, and strengthening oncology education">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb1ab93e274f88a42001d0b9860cca9a59037481" target='_blank'>
              Evolutionary Dynamics of Tumor Adaptation and Therapeutic Resistance in Personalized Oncology
              </a>
            </td>
          <td>
            Alexis Ramírez Hernández, José Arnulfo Perez Carrillo, Juan Diego Paloma Meza, Adrián Jesús Santoyo Rojas, Patricio Sebastian Ordóñez Montalvo, Wilmer Rene Cujilema Guillin, Jorge Che Enseñat, Zuleika Morales
          </td>
          <td>2026-01-26</td>
          <td>IECCMEXICO</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29746864ab4e71ce2bf45dd37a7db9f0d5365e74" target='_blank'>
              Spatial patterns of APOBEC mutagenesis in the tumour microenvironment of Asian breast cancer
              </a>
            </td>
          <td>
            Zi-Ching Tan, Yang Wu, Zhen Wei Neo, Mai Chan Lau, S. Teo, J. P. S. Yeong, S. Chang, Jia-Wern Pan
          </td>
          <td>2026-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background: Rapid advances in biotechnology provide researchers with the opportunity to integrate omics profiles (genomics, epigenomics, transcriptomics, proteomics, etc.) with multiple phenotypes or experimental conditions. In cancers such as acute myeloid leukemia (AML), where combination therapies are standard of care, identifying genetic drivers of drug resistance requires evaluating how genes are associated with multiple drug response phenotypes. Statistical analyses associating omics profiles with multiple phenotypes yield multiple significance values and rankings for each of many genes. There is a great need to consolidate these multiple rankings into a consensus ranking to prioritize specific genes for detailed follow-up wet-lab or clinical studies. Methods/Results: Here, we evaluate the well-known Fisher’s method, the sum of squared z-statistics (SSz), and the recently published cellMCD method as tools for gene prioritization. In simulation studies, cellMCD showed very similar or highly superior performance to the widely used Fisher’s and SSz methods. These advantages were also observed in an example application involving a CRISPR drug screen of an acute myeloid leukemia cell line. Conclusions: In summary, our results indicate that cellMCD should be more widely used for prioritizing discoveries from multiple omic association studies. These methods are available as an R package on github.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/936ca737a6856489d929417a1a5e1ec23e0e8b36" target='_blank'>
              cellMCD Effectively Discovers Drug Resistance and Sensitivity Genes for Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Dora Obodo, Nam H. K. Nguyen, Xueyuan Cao, P. K. Parcha, Christopher D. Vulpe, J. Lamba, S. Pounds
          </td>
          <td>2026-01-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Abstract Loss‐of‐function mutations in lysine methyltransferase 2 (KMT2C) are frequently observed in breast cancer and are significantly associated with enhanced tumour heterogeneity, aggressive clinical behaviour, and poor prognosis. Although KMT2C has been established as a tumour suppressor, the precise molecular mechanisms by which its deficiency drives metastasis and therapy resistance remain incompletely understood. This review comprehensively examines the multifaceted roles of KMT2C in breast cancer pathogenesis, with a particular emphasis on its interplay with critical oncogenic signalling networks and functional axes, including the IFNγ‐EMT/MET axis, the MAPK/ERK and RAS/PI3K/AKT/mTOR pathways, and the KDM6A–MMP3 regulatory circuit. Furthermore, we discuss emerging therapeutic strategies for KMT2C‐deficient breast cancers, such as epigenetic modulators (EZH2 inhibitors, KDM6A inhibitors, and BET inhibitors), DNA damage response (DDR)‐targeting agents, and pathway‐specific inhibitors. These insights not only elucidate the complex biological functions of KMT2C in breast cancer but also provide a rationale for developing precision therapies tailored to KMT2C‐mutant tumours.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f8b0c30c025d7f0171e5e3f66b18d5c0855e666" target='_blank'>
              Targeting lysine methyltransferase 2C deficiency: New frontiers in breast cancer therapy and prognosis
              </a>
            </td>
          <td>
            Shuangmei Tong, Qiu Wang, Yun Chen, Zunjie Bo, Ya Zhang, Yan Liu
          </td>
          <td>2026-02-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 TP53 is the most mutated gene in cancer, with alterations present in roughly 50% of all cancer cases and in approximately 80% of the aggressive Triple Negative Breast Cancer (TNBC). The wild type p53 protein plays a vital role in suppressing cancer by promoting DNA damage repair, regulating cell cycle arrest, inducing apoptosis, and inhibiting cell invasion among other functions. Most of these mutations are single amino acid substitutions, occurring in more than 130 forms and contributing to tumor heterogeneity with potential neomorphic activities. However, despite ongoing efforts of developing precision therapeutics for TNBC, the functional impacts of individual mutant p53 as distinct cancer drivers remain poorly understood. Previously, we created a cell line panel with ten p53 mutations common in breast cancer and reported that each mutation led to distinct molecular and phenotypic profiles. Our current study focuses on the R273C and Y234C mutants, which were the most and least invasive, respectively, to identify the underlying mechanisms of phenotypic heterogeneity and potential drug targets. We have employed a 3D tumor-on-chip technology, which offers detailed single-cell-level insights in more in-vivo-like conditions, and observed highly distinct cellular behaviors of R273C cells with mesenchymal morphology and reduced proliferation, compared to cells with wild-type p53 or Y234C mutant that formed large acini-like structures. Immunostaining and RNA-Seq analyses confirmed further that R273C strongly induced epithelial-to-mesenchymal transition (EMT) and gene expression profile changes similar to those of metastatic breast cancer cells such as MB-231. Using the RNA-Seq data, we have identified differentially regulated genes and pathways in R273C and Y234C. We are currently working on mechanistic investigation to understand how the R273C mutant induces EMT and identification/validation of the top candidates that can potentially intervene in TNBC with more aggressive p53 mutations.



 L. Sakala, K. Ravi, Y. Zhang, E. Hurt, J. G. Park, M. Nikkhah, J. LaBaer. Uncovering Mechanisms Driving Phenotypic Distinctions between Breast Cancer Cells with Different p53 Mutations using a 3D Microfluidic Platform [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2025; 2025 Dec 9-12; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2026;32(4 Suppl):Abstract nr PS4-10-23.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae2bf052a85e2928c3c0f175b17d8c8c630fa309" target='_blank'>
              Abstract PS4-10-23: Uncovering Mechanisms Driving Phenotypic Distinctions between Breast Cancer Cells with Different p53 Mutations using a 3D Microfluidic Platform
              </a>
            </td>
          <td>
            L. Sakala, K. Ravi, , E. Hurt, J. Park, M. Nikkhah, J. LaBaer
          </td>
          <td>2026-02-17</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The phenomenon of transgene silencing was first observed shortly after the generation of the initial transgenic plants. The vast body of experimental data accumulated since then constitutes an invaluable resource for dissecting the mechanisms of epigenetic gene regulation. Silencing operates at either the transcriptional (TGS) or post-transcriptional (PTGS) level and is predominantly mediated by small interfering RNAs (siRNAs). Although these two epigenetic pathways involve distinct sets of proteins and enzymes, they share fundamental mechanistic features: the generation of double-stranded RNA (dsRNA), its processing into siRNAs by DICER-LIKE (DCL) enzymes, and the assembly of an Argonaute-centered effector ribonucleoprotein complex (RISC). Guided by sequence-specific siRNAs, this complex identifies complementary target sequences with high precision. A comprehensive understanding of these regulatory pathways enables the targeted induction or suppression of specific plant genes. This review traces the history of experimental findings regarding the loss of recombinant gene activity in transformants and their progeny, which collectively established the foundation for elucidating the molecular mechanisms of transgene silencing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ca1645109162b7bd31d62f8f845655fb6172721" target='_blank'>
              Gene Inactivation in Transgenic Plants—A Unique Model for Studying Epigenetic Regulation of Gene Expression
              </a>
            </td>
          <td>
            T. Marenkova, A. Zagorskaya, I. Deyneko, Elena V. Deineko
          </td>
          <td>2026-01-01</td>
          <td>Plants</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Multidrug resistance (MDR) is a central cause of chemotherapy failure and tumor recurrence and metastasis, and its mechanism involves enhanced drug efflux, target mutation, upregulation of DNA repair and remodeling of the tumor microenvironment. ABC transporter protein (P-gp, MRP, and BCRP)-mediated efflux of drugs is the most intensively researched aspect of the study, but the first three generations of small-molecule reversal agents were stopped in the clinic because of toxicity or pharmacokinetic defects. Natural products are considered as the fourth generation of MDR reversal agents due to their structural diversity, multi-targeting and low toxicity. In this paper, we systematically summarize the inhibitory activities of monoterpenes, sesquiterpenes, diterpenes and triterpenes against ABC transporter proteins in in vitro and in vivo models and focus on the new mechanism of reversing drug resistance by blocking efflux pumps, modulating signaling pathways such as PI3K-AKT, Nrf2, NF-κB and remodeling the tumor microenvironment. For example, Terpenoids possess irreplaceable core advantages over traditional multidrug resistance (MDR) reversers: Compared with the first three generations of synthetic reversers, natural/semisynthetic terpenoids integrate low toxicity (mostly derived from edible medicinal plants, half-maximal inhibitory concentration IC50 > 50 μM), high target specificity (e.g., oleanolic acid specifically inhibits the ATP-binding cassette (ABC) transporter subtype ABCC1 without cross-reactivity with ABCB1), and multi-mechanistic synergistic effects (e.g., β-caryophyllene simultaneously mediates the dual effects of “ABCB1 efflux inhibition + apoptotic pathway activation”). These unique characteristics enable terpenoids to effectively circumvent key limitations of traditional synthetic reversers, such as high toxicity and severe drug–drug interactions. Among them, lupane-type derivative BBA and euphane-type sooneuphanone D (triterpenoids), as well as dihydro-β-agarofuran-type compounds and sesquiterpene lactone Conferone (sesquiterpenoids), have emerged as the core lead compounds with the greatest translational potential in current MDR reverser research, attributed to their potent in vitro and in vivo MDR reversal activity, low toxicity, and excellent druggable modifiability. At the same time, we point out bottlenecks, such as low bioavailability, insufficient in vivo evidence, and unclear structure–activity relationship and put forward a proposal to address these bottlenecks. At the same time, the bottlenecks of low bioavailability, insufficient vivo evidence and unclear structure–activity relationship have been pointed out, and future research directions such as nano-delivery, structural optimization and combination strategies have been proposed to provide theoretical foundations and potential practical pathways for the clinical translation research of terpenoid compounds, whose clinical application still requires further in vivo validation and translational research support.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc8e568cf68a8569a80453b9e89afe2909550d62" target='_blank'>
              Terpenoids: Emerging Natural Modulators for Reversing ABC Transporter-Mediated Multidrug Resistance in Cancer Chemotherapy
              </a>
            </td>
          <td>
            Lanfei Ma, Dina Mahemuti, Yuanhong Lan, Jianxiong Xu, Wenfang Li, Zhengding Su, Jinyao Li, Aytursun Abuduwaili, Ayitila Maimaitijiang
          </td>
          <td>2026-01-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Sex chromosome aneuploidies (SCAs) are collectively common genetic disorders that impact diverse body systems. The molecular mechanisms by which an extra or missing sex chromosome increases clinical risk are not fully understood, but they likely involve imbalances in expression and regulation of dosage-sensitive genes. There has been a recent surge in transcriptomic and epigenomic studies on genomic effects of SCAs - making it an opportune time to: (i) map existing knowledge of SCA impacts on gene expression and regulation; (ii) resolve consensus findings on SCA dosage-sensitive genes; and (iii) define gaps and high priority areas for future research. Following the Preferred Reporting Items for Systematic Reviews and Meta-Analyses (PRISMA) guidelines (protocol registered on the international Prospective Register of Systematic Reviews PROSPERO CRD42024473984), we searched nine databases and screened the titles and abstracts of 2421 records, thoroughly evaluated 161 full-text articles, and identified 57 eligible studies for abstraction of methodological features and results. Our review spans 18 years of research and encompasses samples from 930 individuals with SCAs and 2192 euploidic controls. The recent acceleration in publication rates outstrips that for biomedical research as a whole. Studies have only recently started to diversify away from the most studied SCAs (47,XXY and 45,X karyotypes), tissue types (blood-derived, gonadal) and measurement methods (transcriptomic analysis by RNAseq). We identify a core set of dosage-sensitive genes that are recurrently impacted by SCAs across multiple tissues. These genes concentrate in 3 protein-protein interaction networks that are predominantly enriched for chromatin remodelling, and represent candidate drivers of downstream phenotypes. This systematic review of SCA impacts on the human genome helps to target the future research efforts that are now needed to (i) address existing knowledge gaps by diversifying the karyotypes, tissues and genomic features analyzed, and (ii) test the causal role for recurrently dysregulated genes. Meeting these goals would provide a molecular foundation to drive both basic and clinical understanding of sex chromosome influences on human phenotypic variation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2f04563ca279e23a5b7a1d63c20549007f5b86a" target='_blank'>
              Sex chromosome aneuploidy impacts on human gene expression and regulation: a systematic review
              </a>
            </td>
          <td>
            Marcela Legue, Melanie Staszewski, Maya Mastronardo, Gisela Butera, Aleksandra Dakic, Armin Raznahan
          </td>
          <td>2025-12-30</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) is a circular, double-stranded DNA molecule distinct from chromosomal DNA, primarily found in cancer cells as a subtype of extrachromosomal circular DNA (eccDNA). Unlike eccDNAs found in normal cells, cancer-associated ecDNAs are large, clonal, and carry complete oncogenes. These ecDNAs are increasingly recognized as crucial drivers of cancer pathogenesis, contributing significantly to the evolution of tumor heterogeneity and the acquisition of therapeutic resistance through mechanisms such as high-level gene amplification and altered gene regulation. Historically, the understanding of these mobile genetic elements in cancer has been limited. This review synthesizes current knowledge on ecDNA’s structural and functional features, formation mechanisms and roles in cancer initiation, progression and therapeutic resistance. Moreover, we summarize five emerging therapeutic approaches that target ecDNA to inform future cancer research and precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb27a6296222e1300898cc02edbec7c0adb86fa2" target='_blank'>
              Extrachromosomal DNA and cancer: function, formation, and clinical implications
              </a>
            </td>
          <td>
            Yucai Wu, Rui Rui, Tai Tian, Jiaying Zheng, Shi-ming He, Liqun Zhou, Xuesong Li, Yanqing Gong
          </td>
          <td>2026-01-19</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/213457505483d2de87291e80a391cdd8fe7c3867" target='_blank'>
              Linking somatic mutations in cancer to the electronic properties of DNA
              </a>
            </td>
          <td>
            Benoît de Witte, Cyril Karamaoun, Pauline Hermans, Maxime Tarabichi, F. Pucci, Marianne Rooman
          </td>
          <td>2026-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06b83eddceaa6549ab81ac78629425aa46eb6aed" target='_blank'>
              MSH2 Prevents Liver Tumorigenesis by Regulating Cell Cycle Checkpoints under Chronic Inflammation.
              </a>
            </td>
          <td>
            Shigeharu Nakano, Atsushi Takai, Eriko Iguchi, Yosuke Fujii, Haruka Amino, Masayuki Ueno, Takahiko Ito, Mari Teramura, M. Mishima, Ken Kumagai, Tadashi Inuzuka, Haruhiko Takeda, Y. Eso, Takahiro Shimizu, T. Maruno, Hiroshi Seno
          </td>
          <td>2026-01-29</td>
          <td>Carcinogenesis</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Transposable elements (TEs) are a major source of genomic variability, yet their relation with other mutational processes and DNA repair in cancers remains poorly understood. Here we combined deep sequencing approaches (RNAseq, whole exome sequencing, and targeted TE-flank sequencing) with computational analyses to investigate the transcriptional activity of active human L1 and Alu elements across 526 experimental and 2488 TCGA cancer samples. By quantifying somatic TE insertions in 40 experimental pairs of cancer and matched normal tissues, we found that TE insertional activity (roughly 20 insertions per sample for each class of TEs) correlates with L1 transcription, is increased in cancers and has substantial intersample variability. TE insertions also correlated with activation of non-homologous end joining, mismatch and nucleotide excision repair pathways, and with transcription of TERT and APOBEC3B genes. Based on highly correlated genes, we created an expression signature reflecting TE insertional activity (AUC 0.819-0.903). On larger experimental and literature tumor cohorts, the signature strongly correlated with the activation levels of most of DNA repair pathways except those leading to ATM checkpoint activation and cell cycle arrest. It was also associated with many genome instability markers (chimeric genes, tumor mutation burden, gene copy number variation, loss of heterozygosity), but showed reduced values in cancers with microsatellite instability. Finally, the signature was associated with worse overall survival in pancreatic cancer (HR 5.9) and lesser effects in stomach, lung, and cervical cancers. These results shed light on the interplay of TE activities, DNA repair, and genome instability in human cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18b44c54459c606eb131e85fc4b773aef30ba060" target='_blank'>
              Insertional activity of human Alu and L1 retrotransposons is associated with DNA repair pathways and genome instability in cancer.
              </a>
            </td>
          <td>
            M. Suntsova, Aleksander Modestov, Elizaveta Rabushko, Nikolai Komarov, Ivan Gaziev, Anastasia Novosadskaya, Anastasiya Barysionak, G. Zakharova, N. Shaban, Anna Khristichenko, Anna A. Emelianova, Marianna A. Zolotovskaia, E. Poddubskaya, A. Seryakov, Anton Buzdin
          </td>
          <td>2026-02-07</td>
          <td>Computers in biology and medicine</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c61b70c406aecf91abbf9e8d3611bdefdb5b3ab6" target='_blank'>
              Dependency Map correlation analysis reveals WDR89 as a genome maintenance factor
              </a>
            </td>
          <td>
            Sean W. Minaker, Gian L. Negri, G. Morin, Peter C. Stirling
          </td>
          <td>2026-01-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Myeloid leukemias, diseases marked by aggressiveness and poor outcomes, are frequently triggered by oncogenic translocations. In the case of chronic myelogenous leukemia (CML), the BCR-ABL fusion initiates chronic phase disease with second hits allowing progression to blast crisis. Although Gleevec has been transformative for CML, blast crisis CML remains relatively drug resistant. Here, we show that MSI2-HOXA9, a translocation with an unknown role in cancer, can serve as a second hit in driving bcCML. Compared to BCR-ABL, BCR-ABL/MSI2-HOXA9 led to a more aggressive disease in vivo with decreased latency, increased lethality, and a differentiation blockade that is a hallmark of blast crisis. Domain mapping revealed that the MSI2 RNA binding domain RRM1 had a preferential impact on growth and lethality of bcCML relative to RRM2 or the HOXA9 domain. Mechanistically, MSI2-HOXA9 triggered global downstream changes with a preferential upregulation of mitochondrial components. Consistent with this, BCR-ABL/MSI2-HOXA9 cells exhibited a significant increase in mitochondrial respiration. These data suggest that MSI2-HOXA9 acts, at least in part, by increasing expression of the mitochondrial polymerase POLRMT and augmenting mitochondrial function and basal respiration in blast crisis. Collectively, our findings demonstrate for the first time that translocations involving the stem and developmental signal MSI2 can be oncogenic and suggest that MSI, which we found to be a frequent partner for an array of translocations, could also be a driver mutation across solid cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a0a174e1fc7002e60c4dfa2851f21e71b1198a5" target='_blank'>
              Identification of a Musashi2 translocation as a novel oncogene in myeloid leukemia
              </a>
            </td>
          <td>
            Kyle Spinler, Michael Hamilton, Jeevisha Bajaj, Yutaka Shima, Emily Diaz, M. Kritzik, T. Reya
          </td>
          <td>2026-01-07</td>
          <td>eLife</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fa4dc4e0545f448a9b7a462aef101f7b3480c9c" target='_blank'>
              Genomic alterations and their correlation with metabolic-related genes in lung cancer.
              </a>
            </td>
          <td>
            Gauri Gaur, N. Jha, Lokesh Gambhir, Seema Acharya, Dhruv Kumar
          </td>
          <td>2026-02-11</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="
 Tumour heterogeneity, encompassing genetic, epigenetic, and microenvironmental diversity, remains a fundamental obstacle in precision oncology. Traditional bulk sequencing captures only averaged molecular profiles, thereby masking rare yet functionally critical subpopulations that drive malignant progression and therapeutic resistance. Recently, the emergence of single‐cell sequencing technologies has overcome the limitations of bulk approaches, enabling high‐resolution analyses of the genome, transcriptome, epigenome and proteome at the single‐cell level. These advances have enabled detailed mapping of tumour ecosystems, identification of key cellular subtypes, reconstruction of evolutionary trajectories and elucidation of intercellular communication networks within the tumour microenvironment. Accumulating evidence demonstrates that single‐cell technologies elucidate fundamental aspects of tumour biology and reveal potential diagnostic and therapeutic targets. This review systematically summarises the recent advances and applications of single‐cell sequencing in the field of precision oncology, with particular emphasis on its applications in mechanistic discovery, diagnosis, therapy, and prognosis. Furthermore, we discuss current challenges related to technology, data analysis, and clinical translation, and outline future research directions. In summary, single‐cell sequencing has profoundly reshaped our understanding of tumour biology and is propelling oncology into a new era of precision, prediction, and personalisation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11cbf7015a15504645fc0387a8b9ed997f9b6eef" target='_blank'>
              Single‐cell sequencing for cancer precision medicine: From mechanism discovery to diagnosis and therapeutics
              </a>
            </td>
          <td>
            Yue Zhao, Hancong Li, Huanzuo Yang, Gongshuang Zhang, Xinyue Fu, Peiheng Li, Puxing He, Shuang Wu, Han Luo
          </td>
          <td>2026-01-29</td>
          <td>Clinical and Translational Discovery</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The discovery of lactylation, a post-translational modification derived from lactate, has fundamentally altered the perception of cancer metabolism. Once regarded as a metabolic waste product, lactate is now recognized as a central fuel source, a signaling molecule, and an epigenetic substrate capable of reprogramming gene expression and cellular function. Lactylation integrates metabolic reprogramming, tumor plasticity, and immune suppression, thereby orchestrating cancer initiation, progression, and resistance to therapy. This review provides a critical and integrative commentary on recent advances in lactylation biology, drawing from biochemical, epigenetic, and immunological perspectives. It synthesizes mechanistic insights into lactylation, highlights its role in tumorigenesis and the tumor microenvironment (TME), and evaluates therapeutic strategies that target lactate production, transport, and lactylation machinery. By dissecting consensus, controversies, and unresolved questions, we argue that lactylation represents both a hallmark of tumor adaptation and a potential Achilles’ heel for intervention. We further discuss future research directions, including comprehensive lactylome mapping, structural biology of lactylated proteins, microbiome-derived lactate, and clinical translation. Ultimately, lactylation is not merely a byproduct of glycolysis but a metabolic language that tumors employ to communicate, adapt, and thrive. Decoding this language may open new frontiers in cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f11df3d617ec11389a48bfdc33dfa42dd8ab08b2" target='_blank'>
              Lactylation in cancer: mechanistic insights, tumor microenvironment, and therapeutic horizons
              </a>
            </td>
          <td>
            Qiang Yang, Zhibo Yang, Hai Zhao
          </td>
          <td>2026-01-09</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82b68b7fa61f9edb29d61947a73ec839e1cdd7c8" target='_blank'>
              CRISPR-Cas9 technology: a breakthrough in cancer gene therapy
              </a>
            </td>
          <td>
            Z. I. Jameel
          </td>
          <td>2026-01-12</td>
          <td>Egyptian Journal of Medical Human Genetics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5e6ef14b9893fdf3b4051752e9e422de35cb120" target='_blank'>
              KMT2D loss drives adeno-to-squamous transition and sensitizes TKI-resistant lung cancer to AURKA inhibition.
              </a>
            </td>
          <td>
            Nana Chen, Mouxiang Fang, Leqi Zhong, Xiaolong Li, Yijia Zhou, J. Zhan, Manli Wang, Zhaoyuan Fang, Hua Wang, Shijie Tang, Fang Liu, Bing Deng, Ning Chen, Jie Lei, Yuchen Zhang, Min Yan, Zhengzhi Zou, Yijun Gao, Chong Chen, Wenzhao Zhong, S. Saladi, Hongbin Ji, Quentin Liu, Zifeng Wang, Bin He
          </td>
          <td>2026-01-08</td>
          <td>Cell death and differentiation</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/173cc4555c30056ffb452d93c67137c372b260db" target='_blank'>
              Gene duplication is associated with gene diversification and potential neofunctionalization in lung cancer evolution.
              </a>
            </td>
          <td>
            Paul Ashford, A. Frankell, Zofia Piszka, Camilla S. M. Pang, M. Abbasian, Maise Al Bakir, M. Jamal-Hanjani, N. Mcgranahan, C. Swanton, Christine A. Orengo
          </td>
          <td>2026-02-19</td>
          <td>Genome research</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a life-threatening malignancy, but our understanding of its pathogenic mechanisms remains incomplete—posing a major constraint on the development of effective therapeutic strategies. The transcription-translation feedback loop of clock genes (e.g., BMAL1, CLOCK, PER1/2/3, and CRY1/2) provides a promising novel avenue for deciphering the initiation and progression of CRC. Mounting evidence indicates that core circadian clock genes play pivotal roles in CRC oncogenesis by orchestrating the regulation of the cell cycle, epithelial–mesenchymal transition (EMT), metabolic reprogramming, and the tumor microenvironment. This review systematically summarizes the expression patterns and mechanistic roles of core clock genes in CRC, while elucidating their molecular underpinnings in tumor progression via key signaling cascades (e.g., Wnt/β-catenin and c-Myc/p21 pathways). We emphasize the associations between circadian disruption and CRC—including diagnostic markers, prognostic assessment, and chemosensitivity—and provide an in-depth discussion of chronotherapeutic strategies and their translational potential. Finally, we identify unaddressed scientific questions and propose future research directions to facilitate the development of novel targeted therapies for CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb37625767256f04fec23407d4d727a07a242245" target='_blank'>
              Circadian Clock Genes in Colorectal Cancer: From Molecular Mechanisms to Chronotherapeutic Applications
              </a>
            </td>
          <td>
            Haoran Wang, Jieru Zhou, Suya Pang, Yiqing Mei, Gangping Li, Yu Jin, Rong Lin
          </td>
          <td>2026-01-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6815a13ec402ee8cc5af80b37a7c7988fa8dd87" target='_blank'>
              Cancer-associated fibroblasts: enablers of tumor drug resistance
              </a>
            </td>
          <td>
            Yanting Chen, Wei Xi, Xingyue Lai, Linglin Xiang, A. Zeng, Linjiang Song
          </td>
          <td>2026-01-24</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Cancer cell characteristics are determined by gene expression, influenced by genomic, epigenetic, and transcriptional modifications. Genomic rearrangements and transcriptional splicing can result in the formation of fusion genes. BCR-ABL1 is an established fusion gene employed as a biomarker in leukemia. A single gene can amalgamate with several other genes and may impact cellular fate. Ethnicity-specific variants of fusion genes have been identified, such as the TMPRSS2-ERG variation observed in prostate malignancies among African-American, Caucasian, and Japanese populations in research studies. Next-generation sequencing has provided a new method for predicting genomic and transcriptomic changes. We aim to identify fusion genes in the Indian population using cancer samples to enhance diagnostic outcomes. This study performed a meta-analysis of tumor-specific RNA sequencing data for liver, tongue, and ovarian cancers, which are available online. It identified known fusion genes, including TRO-MAGED2, KRT14-S100A9, RNASE10-CD38, ACTN4-ACTN1, RGPD1-RANBP2, CTSC-RAB38, C15orf57-CBX3, AMBRA1-CKAP5, ATP2B3-ATP2B4, CNKSR3-IPCEF1, E2F4-RPL14, and MZT2A-MZT2B, along with 101 novel fusion genes. Novel fusion genes GABRP_SCGB3A2 and WWOX_FUT1 were identified in all three tumor tissues. GABRP acts as a tumor inducer, whereas SCGB3A2 functions as a tumor suppressor. WWOX2 serves as a tumor suppressor, whereas FUT1 functions as a promoter of malignancy. The interplay between tumor inducers and suppressors may serve as a survival mechanism for cancer cells, a subject that has received limited research attention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4406b3ded6f3886093045dfec09d59265a20dd7e" target='_blank'>
              RNAseq-based meta-analyses revealed tumor suppressor-inducer fusion events in liver, oral, and ovarian cancer in the Indian population: a cancer cell surviving mechanism.
              </a>
            </td>
          <td>
            Rahul Yadav, H. Khan, P. Singh, Pramod Kumar, Dinesh Kumar Singhal
          </td>
          <td>2026-02-09</td>
          <td>Nucleosides, nucleotides & nucleic acids</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Colorectal cancer (CRC) with mismatch-repair deficiency (dMMR) or high microsatellite instability (MSI-H) represents a distinct molecular subtype highly sensitive to immune checkpoint inhibitors (ICIs). Landmark clinical trials have established ICIs as standard-of-care in this setting, demonstrating durable responses and improved survival. However, up to one-third of patients will exhibit primary or acquired resistance, highlighting the urgent need for predictive biomarkers and novel therapeutic strategies. This review summarizes the clinical evidence supporting ICIs in dMMR/MSI-H CRC, explores mechanisms of resistance-including intrinsic and extrinsic modulators-and evaluates the role of potential predictive biomarkers of response. Finally, we discuss innovative therapeutic approaches to overcome resistance, including combination strategies, DNA repair pathway inhibitors, immune-oncology drugs beyond checkpoint inhibitors and microbiome-targeted interventions. Together, these insights aim to refine patient selection, optimize therapeutic benefit, and guide the development of next-generation therapies for dMMR/MSI-H CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2722ca9e8cce45fe3bbb20b982cfb4cf7eacd93f" target='_blank'>
              Exploring resistance to immune checkpoints inhibitors in mismatch repair-deficient or microsatellite-instable colorectal cancer.
              </a>
            </td>
          <td>
            Clara Salva de Torres, Evelyn Elias, C. Vaghi, Nadia S González, Ariadna García, Adriana Alcaraz, M. Rodríguez-Castells, I. Baraibar, J. Ros, F. Salva, J. Tabernero, E. Élez, Enrique Sanz-Garcia
          </td>
          <td>2026-01-01</td>
          <td>Cancer treatment reviews</td>
          <td>1</td>
          <td>57</td>
        </tr>

        <tr id="ABSTRACT Inherited metabolic disorders (IMDs) are a diverse and complex group of genetic conditions resulting from deficiencies in enzymes, transporters, or cofactors. These deficiencies lead to metabolic dysfunction and severe clinical consequences. Despite significant progress in understanding their molecular basis, treatment options remain limited for many IMDs. RNA‐based therapies including antisense oligonucleotides (AONs), small interfering RNAs (siRNAs), and messenger RNA (mRNA) therapeutics have emerged as promising treatment strategies for modulating gene expression, silencing pathogenic transcripts, and restoring deficient proteins, offering new avenues for disease intervention. In this review, we summarise the chemistry and mechanisms of action of different RNA therapy modalities including splice‐modulating and gene silencing AONs, siRNAs, and mRNA therapies. The delivery of these RNA‐based therapies remains a significant challenge. Here, we outline the development of various delivery methods, including lipid nanoparticle (LNP) packaging, ligand conjugation, and tissue‐specific delivery systems as well as their clinical applications in treating IMDs. We also summarise the clinical application of RNA therapies in rare diseases, an area that has grown rapidly in the last few years, as exemplified by the success of some n‐of‐1 therapies for IMDs, which have redefined personalised medicine by enabling rapid, patient‐specific drug development. As RNA‐based therapeutics continue to evolve, their clinical applications in IMDs will require continued innovation in novel chemistries, advanced delivery technologies, and streamlined regulatory frameworks to unlock their full potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/442f1b0d825c712f9b6291719c2a329ade1ca188" target='_blank'>
              RNA‐Based Therapies for Inherited Metabolic Disorders
              </a>
            </td>
          <td>
            R. Vootukuri, Sonam Gurung, Roopkatha Ghosh, P. Mills, Julien Baruteau, Haiyan Zhou
          </td>
          <td>2026-02-01</td>
          <td>Journal of Inherited Metabolic Disease</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5a700eec3fdb5ec66bc2ba026a65a9361248ce9" target='_blank'>
              DNA end-resection is stimulated by an interaction between BRCA1 exon 11 and TOPBP1
              </a>
            </td>
          <td>
            Marta San Martín Alonso, Rosalie A. Kampen, Anne Schreuder, Daniel Salas, A. D. de Ru, V. Garzero, R. Mesman, Román González-Prieto, P. V. van Veelen, Sylvie M. Noordermeer
          </td>
          <td>2026-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains one of the leading causes of cancer-related mortality worldwide. Compared with traditional two-dimensional (2D) models, patient-derived CRC organoids more faithfully preserve the genomic, transcriptomic, and architectural features of primary tumors, making them a powerful intermediate platform bridging basic discovery and clinical translation. Over the past several years, organoid systems have rapidly expanded beyond conventional epithelial-only cultures toward increasingly complex architectures, including immune-organoid co-culture models and mini-colon systems that enable long-term, spatially resolved tracking of tumor evolution. These advanced platforms, combined with high-throughput technologies and clustered regularly interspaced short palindromic repeats (CRISPR)-based functional genomics, have substantially enhanced our ability to dissect CRC mechanisms, identify therapeutic vulnerabilities, and evaluate drug responses in a physiologically relevant context. However, current models still face critical limitations, such as the lack of systemic physiology (e.g., gut–liver or gut–brain axes), limited standardization across platforms, and the need for large-scale, prospective clinical validation. These gaps highlight an urgent need for next-generation platforms and computational frameworks. The development of high-throughput multi-omics, CRISPR-based perturbation, drug screening technologies, and artificial intelligence-driven predictive approaches will offer a promising avenue to address these challenges, accelerating mechanistic studies of CRC, enabling personalized therapy, and facilitating clinical translation. In this perspective, we propose a roadmap for CRC organoid research centered on two major technical pillars: advanced organoid platforms, including immune co-culture and mini-colon systems, and mechanistic investigations leveraging multi-omics and CRISPR-based functional genomics. We then discuss translational applications, such as high-throughput drug screening, and highlight emerging computational and translational strategies that may support future clinical validation and precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ed98ccf8eb7ee4178076cc0e4368c14e7e399d9" target='_blank'>
              An Integrative Roadmap for Advancing Colorectal Cancer Organoid
              </a>
            </td>
          <td>
            Youqing Zhu, Ke He, Zhi Shi
          </td>
          <td>2026-01-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Bladder cancer (BLCA) is a lethal malignancy frequently challenged by chemoresistance and limited therapeutic options. Kinetochore-associated 1 (KNTC1) has been implicated in cancer, yet its precise role, regulatory mechanism, and therapeutic potential in BLCA remain largely unexplored. We analyzed KNTC1 expression and its clinical relevance using public databases (TCGA, GEO) and clinical specimens. Functional roles of KNTC1 in proliferation, invasion, metastasis, and gemcitabine resistance were assessed through in vitro and in vivo experiments (CCK-8, transwell, xenograft models). Mechanistic insights were gained via RNA-seq, co-immunoprecipitation, chromatin immunoprecipitation (ChIP), and luciferase reporter assays. A poly (ß-amino ester) (PAE)-based nanoparticle was synthesized for targeted in vivo delivery of KNTC1 siRNA. KNTC1 was significantly upregulated in BLCA tissues, and its high expression correlated with poor patient prognosis. Functionally, KNTC1 promoted BLCA cell proliferation, invasion, metastasis, and gemcitabine resistance. Mechanistically, KNTC1 bound to the transcription factor E2F8 and facilitated its nuclear translocation, thereby enhancing E2F8-mediated transcriptional activation of MYC. MYC, in turn, transcriptionally upregulated KNTC1, forming a positive feedback loop that drove hyperactivation of the oncogenic PI3K/AKT/mTOR pathway. Silencing KNTC1, especially in combination with a MYC inhibitor (10058-F4), overcame gemcitabine resistance in vitro and in vivo. Therapeutic delivery of KNTC1 siRNA via a novel PAE nanocarrier (PAE@shKNTC1) significantly suppressed lung metastasis in a preclinical model. Our study identifies a novel KNTC1/E2F8/MYC positive feedback axis that drives BLCA tumorigenesis and chemoresistance. KNTC1 serves as both a prognostic biomarker and a promising therapeutic target, with targeted inhibition offering a potential novel strategy for BLCA treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f04b8b75d8c5e746264170721cdfe0354d03b4cb" target='_blank'>
              KNTC1 initiates a KNTC1/E2F8/MYC positive feedback loop to facilitate tumorigenesis and enhance chemoresistance in bladder cancer
              </a>
            </td>
          <td>
            Kailai Chen, Hecheng Su, Feng Pei, Xi Chen, Meiqi Xu, Fang Chai, Yakun Luo
          </td>
          <td>2026-02-04</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Tumor evolution involves genetic, transcriptional, and phenotypic alterations that shape cancer cell behavior and interactions with the microenvironment. While single-cell technologies have advanced our understanding of this process, spatial dynamics remain incompletely characterized. Here, whole-exome sequencing (WES), imaging mass cytometry (IMC), and spatial transcriptomics (ST) were integrated to study molecular evolution and immune responses in two lung adenocarcinoma (LUAD) mouse models: a genetically engineered model (129S4/Sv-KrasLSL-G12D, termed 129S4 K) and a carcinogen-induced precancer model (129S4 U). Compared to 129S4 K, 129S4 U tumors exhibited higher mutational, neoantigen but lower copy number variation (CNV) burdens at matched developmental timepoints, consistent with findings of higher mutational burden in human smoking-related LUAD than nonsmoking LUAD. We profiled over 1.4 million spatial single cells from 284 IMC regions of interest and 51,531 spatial transcriptomic spots from 156 lesions across 141 mice. Macrophage abundance increased with tumor progression, while CD8 T-cell and B-cell densities declined in late-stage LUAD. 129S4 U showed greater immune infiltration in both tumor and adjacent normal tissue, higher T-cell cytotoxicity signature score in line with its higher mutational and neoantigen burdens. LUAD progression was marked by early morphological shifts and late-stage changes in cell states and interactions. These data define spatial and genetic landscapes of LUAD development and provide a framework for investigating immune evolution and therapeutic strategies in early carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a0e7ad7c0953d4cab7fb1f9dcb251447c108ead" target='_blank'>
              Spatial Profiling Reveals Distinct Molecular and Immune Evolution of Mouse Lung Adenocarcinoma Precancers with or Without Carcinogen Exposure.
              </a>
            </td>
          <td>
            Bo Zhu, M. Aminu, Pingjun Chen, Jian-Rong Li, Chuanpeng Dong, Chenyang Li, Yanhua Tian, Shaowen Lu, Hong Chen, Chenxi Ma, Xin Hu, Jie-Tong Ye, Andrew Liu, Beibei Huang, Frank R. Rojas, Parra Cuentas Edwin Roger, Ou Shi, M. Nilsson, A. Poteete, Khaja B Khan, Wei Lu, Luisa M. Solis Soto, Junya Fujimoto, C. Haymaker, Ignacio I. Wistuba, Zhubo Wei, Linghua Wang, Don L. Gibbons, Ken Chen, Alexandre Reuben, Jason M Schenke, J. Heymach, Chao Cheng, Jia Wu, Jianjun Zhang
          </td>
          <td>2026-01-25</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Simple Summary In many types of cancer, cells double their entire set of chromosomes, a state known as tetraploidy. This change can accelerate the cancer’s growth, spread, and resistance to drugs. We wanted to understand how this chromosome doubling changes the cell’s mitochondria. We compared cells with a normal versus a doubled set of chromosomes from human cancers, yeast, and fungi. We found that in all cases, cells with doubled chromosomes were larger and contained more mitochondria, which were also more active. By identifying this shared trait, our research points to these adapted mitochondria as a potential new target for treating cancers with unstable chromosomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9e2f77cdf8945e8a66c23348025539c8ac24aa7" target='_blank'>
              Mitochondrial Adaptations Underlying Tetraploidization in Human Cancer, Fungal, and Yeast Models
              </a>
            </td>
          <td>
            Mohamed Jemaà, Ameni Bedoui, Nihel Ammous, Ali Gargouri, Mohamed Guerfali
          </td>
          <td>2026-01-01</td>
          <td>Biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="As a type of cell with self-renewal ability and multi-directional differentiation potential, stem cells are closely related to their functions, such as reprogramming transcription factors, histone modifications, and energy metabolism. m6A (N6-methyladenosine modification) is one of the most abundant modifications in RNA, and dynamic reversible m6A modification plays an important role in regulating stem cell function. This review moves beyond listing isolated functions and instead adopts an integrated perspective, viewing m6A as a temporal regulator of cellular state transitions. We discuss how m6A dynamically regulates stem cell pluripotency, coordinates epigenetic and metabolic reprogramming, and serves as a central hub integrating key signaling pathways (Wnt, PI3K-AKT, JAK-STAT, and Hippo). Finally, using somatic reprogramming as an example, we elucidate the stage-specific role of m6A in complex fate transitions. This comprehensive exposition not only clarifies the context-dependent logic of m6A regulation but also provides a precise framework for targeting the m6A axis in regenerative medicine and cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e397129d10e01c5414b01cf61047665a5c7f45a" target='_blank'>
              The Regulatory Role of m6A Modification in the Function and Signaling Pathways of Animal Stem Cells
              </a>
            </td>
          <td>
            Xiaoguang Yang, Yongjie Xu, Su-Wen Zhu, Mengru Wang, Hongguo Cao, Lizhi Lu
          </td>
          <td>2026-01-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b30edb27f07cff97af34aacf8166230cf5b8614" target='_blank'>
              Unsupervised Identification of Cancer Attractor States through the Lens of Embryonic Origin and Cancer Hallmarks
              </a>
            </td>
          <td>
            P. Deshpande, P. Deb, Ibrahim Al Haddabi, B. Itkin
          </td>
          <td>2026-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Aim: Cancer cells divide excessively, resulting in overpopulation and hypoxia. Tumor hypoxia drives tumor development and therapeutic resistance. Hypoxia dominates pancreatic tumor microenvironments. This study aimed to ascertain alterations in gene expression linked to chronic hypoxia in the pancreatic cancer cell (PANC-1) line. Methods: PANC-1 had eight-hour hypoxic events with oxygen levels below 1%. 40 episodes were exposed three times a week. Real-time PCR arrays were used to analyze gene expression changes. This investigation compared cells treated with 20 and 40 hypoxia episodes to normoxic cells. MTT cell proliferation assays assessed hypoxic cell doxorubicin resistance. Wound-healing assays measured cell migration. 20 and 40 hypoxia exposures altered gene expression patterns significantly. Results: No alterations were observed in either stage, as most genes exhibited a resurgence after 40 episodes. Following exposure to 20 episodes of hypoxia, the expression levels of the genes HIF1AN, HMOX-1, and PKM were significantly increased by factors of 6.9, 4.5, and 3.4, respectively. The IC50 value of Doxorubicin in PANC-1 cells exhibited a 2.7-fold increase and an approximately 3.6-fold increase after 20 and 40 episodes, respectively, compared to normoxic cells. This study demonstrates that subjecting cells to extended durations of hypoxia results in distinct alterations in gene expression compared to those induced by short-term hypoxia, specifically lasting less than 72 hours. Furthermore, it sanctioned the facilitation of chemotherapy resistance through prolonged exposure. Conclusions: The study suggests that HIF1AN, HMOX-1, and PKM may serve as promising biomarkers for hypoxia in pancreatic cancer and contributors to the cellular response to prolonged hypoxia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea72b67800b6ab127a692f8378384c4dd8f3f5c4" target='_blank'>
              The gene expression alterations in chronic hypoxic PANC-1 pancreatic cancer cell line
              </a>
            </td>
          <td>
            Malek A. Zihlif, Heba A Khader, L. Younis, Ahmad Sharab, Farah Tahboub, L. Hasoun, Hamzah Hajaj, Zaid Alawneh, A. Alsayed
          </td>
          <td>2026-02-04</td>
          <td>Pharmacy Practice</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The stable and safe integration of exogenous DNA into the genome is crucial to both genetic engineering and gene therapy. Traditional transgenesis approaches, such as those using retroviral vectors, result in random genomic integration, posing the risk of insertional mutagenesis and transcriptional dysregulation. Safe harbor sites (SHSs), genomic loci that support reliable transgene expression without compromising endogenous gene function, genomic integrity, or cellular physiology, have been identified and characterized across various model organisms. Well-established SHSs such as AAVS1, ROSA26, and CLYBL are routinely utilized for targeted transgene integration in human cells. Recent advances in genome architecture, gene regulation, and genome editing technologies are driving the discovery of novel SHSs for precise and safe genetic modification. This review aims to provide a comprehensive overview of SHSs and their applications that will guide investigators in the choice of SHS, especially when complementary sites are needed for more than one transgene integration. First, it outlines safety and functional criteria that qualify a genomic site as a safe harbor site. It then discusses the two primary strategies for identifying SHSs: i) traditional lentiviral-based random transgenesis, and ii) modern genome-wide in silico screening followed by CRISPR-based validation. This review also provides an updated catalogue of currently known SHSs in the human genome, detailing their characteristics, uses, and limitations. Additionally, it discusses the diverse applications of SHSs in basic research, gene therapy, CAR T cell-based therapy, and biotechnological production systems. Finally, it concludes by highlighting challenges in identifying universally applicable SHSs and outlines future directions for their refinement and validation across biological systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/600741ca6d853466b695798a17050f6f9f6dbe8e" target='_blank'>
              Human Genome Safe Harbor Sites: A Comprehensive Review of Criteria, Discovery, Features, and Applications
              </a>
            </td>
          <td>
            Amer Ahmed, D. Di Molfetta, G. N. Iaconisi, Antonello Caponio, Ansu Singh, A. Bibi, V. Dolce, Luigi Palmieri, Vincenzo Coppola, G. Fiermonte
          </td>
          <td>2026-01-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Cancer has become a leading cause of mortality worldwide, with alarming increases in incidence and mortality rates. Emerging evidence suggests that tRNA modification enzymes play a crucial role in cancer development by modulating codon-specific translation. In this review, we focus on 18 tRNA modification enzymes and elucidate their mechanisms of action and roles in disease. We highlight the functions and mechanisms of seven tRNA regulators that mediate favorable tRNA translation in tumorigenesis and cancer progression, providing deeper insights into their clinical potential as cancer-related biomarkers and prognostic indicators. These findings emphasize the need for further investigation into the therapeutic potential of tRNA modification enzymes in cancer management and their potential application in personalized cancer therapy and diagnostics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39e019c44f4ec4dbe9eac3e87c51a69ce21d81f4" target='_blank'>
              Emerging roles of tRNA modification-mediated codon-specific translational reprogramming in cancer biology
              </a>
            </td>
          <td>
            Hanwei Wang, Junsi Zhang, Cen Jiang, Sunwang Xu
          </td>
          <td>2026-01-07</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa422c481718f7852d2f2551a6a84612f4664e50" target='_blank'>
              Integrative analyses of metastatic cancer transcriptome reveal clinically distinct cellular States and ecosystems
              </a>
            </td>
          <td>
            Can Zhang, Si Li, Yun Yu, Meng Chi, Ziming Yuan, Kunyu Wang
          </td>
          <td>2026-02-05</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Oral squamous cell carcinoma (OSCC) is a prevalent and aggressive malignancy with a persistently high mortality rate, largely attributable to therapy resistance and tumor recurrence. This review comprehensively explores the critical interplay between epigenetic dysregulation and the tumor microenvironment (TME) in driving OSCC progression. We detail how key epigenetic mechanisms, including DNA methylation, histone modifications, and non-coding RNAs (ncRNAs), intrinsically transform cancer cells and actively orchestrate pro-tumorigenic TME. These alterations substantially contribute to resistance against conventional therapies. Furthermore, we discuss the therapeutic potential of targeting these pathways using epigenetic drugs (epi-drugs), such as DNA methyltransferase (DNMT) inhibitors and histone deacetylase (HDAC) inhibitors, as well as engineered extracellular vesicles (EVs). The primary objective of this review is to synthesize current knowledge on the epigenetic-TME axis, thereby providing a mechanistic foundation for developing novel therapeutic strategies. We emphasize that rational combinations of epigenetic-targeting agents with conventional treatments or immunotherapy hold significant promise for overcoming drug resistance and improving clinical outcomes in OSCC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96aaa3d5aaab2d3886a4c338f86161bbd9cbefef" target='_blank'>
              Dissecting the role of epigenetic regulation in oral squamous cell carcinoma microenvironment: mechanisms and therapeutics
              </a>
            </td>
          <td>
            Xuechao Li, Yifei Ren, S. Pei, Kai Zhao, Guanyu Chen, Zhenglin He
          </td>
          <td>2026-01-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The prolonged and intricate history of oncological treatments has transitioned significantly since the introduction of chemotherapy. Substantial therapeutic benefits in cancer therapy have been achieved by the integration of conventional treatments with molecular biosciences and omics technologies. Human epidermal growth factor receptor, hormone receptors, and angiogenesis factors are among the established therapies in tumor reduction and managing side effects. Novel targeted therapies like KRAS G12C, Claudin-18 isoform 2 (CLDN18.2), Trophoblast cell-surface antigen 2 (TROP2), and epigenetic regulators emphasize their promise in advancing precision medicine. However, in many cases, the resistance mechanisms associated with these interventions render them ineffective in carrying out their functions. The purpose of this review is to provide a comprehensive and up-to-date examination of both established and emerging drug targets and mechanisms of treatment resistance in oncology. This review seeks to elucidate recent advancements, address persisting challenges, and explore opportunities for innovative developments in cancer target research. Additionally, it explores the growing role of artificial intelligence in reshaping cancer drug discovery and development frameworks as potential avenues for future research. In conclusion, innovative approaches in oncology, supported by pharmacological research, ongoing clinical trials, molecular biosciences, and artificial intelligence, are poised to significantly transform cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d38d9fe6327c09883c020612002ccbd1bc87300f" target='_blank'>
              A Holistic Review of Oncological Drug Targets and Trajectories of Resistance in Cancer Therapy
              </a>
            </td>
          <td>
            Harpreet Kaur, Dhrubalochan Rana, Sowvik Bag, Paramjeet Singh
          </td>
          <td>2026-01-19</td>
          <td>Oncology Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Long-read whole-transcriptome sequencing (WTS) has the potential to precisely characterize fusion oncogenes that drive leukemia and other cancers. While there are a variety of general-purpose fusion detection algorithms that use modern long-read sequencing data, they show poor sensitivity for precision diagnostics in B-ALL and do not robustly assess technical and analytical parameters (ex. sequencing depth) to reliably detect fusion transcripts. FUSILLI (FUSions In Leukemia Long-read sequencing Investigator) is a novel long-read fusion detection algorithm, with a focus on targeted genomic subtyping in B-ALL. FUSILLI was evaluated against extant methods using nanopore WTS from 51 pediatric B-ALL samples sequenced at high depth and 68 at low depth (mean of 11.2M and 1.4M reads, respectively). In the high-depth cohort, FUSILLI demonstrated increased sensitivity (0.81) compared to FusionSeeker, JAFFAL, and LongGF (0.63, 0.76, and 0.70, respectively), while maintaining high specificity (0.92). At lower sequencing depth, FUSILLI showed correspondingly lower sensitivity (0.27) but still outperformed the other fusion callers (sensitivities ranging from 0.09 to 0.16). Computational down sampling suggests that ten million reads is sufficient to sensitively detect B-ALL-relevant fusions using this approach. FUSILLI detects B-ALL fusions with high sensitivity at modest sequencing depth, supporting implementation of nanopore WTS as a low-cost and globally accessible sequencing-based molecular diagnostic platform for pediatric B-ALL and other fusion-driven cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/884c4daa4f67a077c5abf5eb62ebb345ad0e753d" target='_blank'>
              Long-read whole-transcriptome sequencing and selective gene panel profiling enable sensitive detection of fusion oncogenes in pediatric B-cell acute lymphoblastic leukemia.
              </a>
            </td>
          <td>
            John Lin, Kofi B. Opoku, M. Litzow, E. Paietta, Ching-Hon Pui, S. Jeha, K. Roberts, C. Mullighan, T. Alexander, Jeremy R. Wang
          </td>
          <td>2026-02-01</td>
          <td>The Journal of molecular diagnostics : JMD</td>
          <td>0</td>
          <td>110</td>
        </tr>

        <tr id="Background

Immune checkpoint inhibitors have transformed the treatment landscape of non-small cell lung cancer (NSCLC); however, primary and acquired resistance remain major clinical challenges. Immune evasion in NSCLC is increasingly recognized as a complex, dynamic process driven by tumor heterogeneity and adaptive remodeling of the tumor microenvironment. Conventional bulk profiling approaches are limited in resolving this complexity, underscoring the need for higher-resolution methodologies.

Objective
This review aims to synthesize current evidence on how single-cell transcriptomic approaches have advanced the understanding of immune evasion mechanisms in treatment-resistant NSCLC, with a focus on tumor-intrinsic programs, immune cell dysfunction, and intercellular communication networks.

Methods

A structured narrative review with systematic elements was conducted, analyzing peer-reviewed studies employing single-cell RNA sequencing and related single-cell technologies in human NSCLC. Studies were evaluated for their insights into immune escape pathways, resistance to immunotherapy, and translational relevance.

Results

Single-cell transcriptomic analyses reveal profound intratumoral and immune heterogeneity in treatment-resistant NSCLC. Resistant tumors harbor distinct malignant cell states characterized by impaired antigen presentation, dysregulated interferon signaling, and lineage plasticity. Within the immune compartment, hierarchical T-cell exhaustion, depletion of progenitor-like exhausted T cells, and expansion of immunosuppressive myeloid populations—particularly tumor-associated macrophages—emerge as dominant features of resistance. Ligand–receptor interaction analyses further demonstrate coordinated immunosuppressive communication networks that sustain immune escape at a systems level. Several single-cell–defined cellular states correlate with poor response to immune checkpoint blockade and adverse clinical outcomes.

Conclusion

Single-cell transcriptomics has fundamentally reshaped the conceptual framework of immune resistance in NSCLC, redefining it as a multicellular, dynamic ecosystem rather than a single-pathway phenomenon. These insights provide a strong biological rationale for developing biomarker strategies and rational combination therapies that target both tumor-intrinsic adaptations and the immunosuppressive microenvironment. Continued integration of single-cell technologies into translational and clinical research is essential to overcome immunotherapy resistance and improve patient outcomes in NSCLC.

Keywords: Non-small cell lung cancer; Single-cell RNA sequencing; Immune evasion; Immunotherapy resistance; Tumor microenvironment">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7495ff2bd82ff3331af5929ec1f5ca0f1ed50c9" target='_blank'>
              Single-Cell Transcriptomic Mapping of Immune Evasion Pathways in Treatment-Resistant Non-Small Cell Lung Cancer
              </a>
            </td>
          <td>
            Marco Rossi, Giulia Bianchi, Alessandro Conti
          </td>
          <td>2026-01-19</td>
          <td>Advanced Journal of Biomedicine &amp; Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Epigenetic reprogramming is a major contributor to therapy resistance in prostate cancer (PCa). Indeed, chromatin accessibility profiling classified PCa into distinct subtypes, predictive of clinical outcome. Histone modifications influence nucleosome positioning, ultimately dictating chromatin compactness. The histone methyltransferase G9a, frequently upregulated in advanced PCa, catalyzes a repressive histone modification (di-methylation of histone H3 lysine 9) that signals for gene transcriptional repression. CM-272 is a reversible and highly selective inhibitor against G9a activity, and we have previously shown in cell line models of PCa that it depicted strong anti-tumoral effects. However, fundamental aspects of altered transcriptomic programs remain unexplored. Here, we validated, in vivo, the CM-272 effects in advanced PCa, and explored the molecular mechanisms underscoring G9a inhibition-mediated effects. Specifically, the PCa patient-derived xenograft model BM18 was used to assess the mode of action of CM-272 in vivo. Moreover, RNA sequencing of tumor tissues was performed to further explore the underlying gene expression signature. Finally, chromatin immunoprecipitation followed by qPCR was used to investigate the underlying functional consequence of epigenetic targeting. Our results showed that CM-272 significantly delayed BM18 tumor growth, specifically impairing tumor cell G9a activity and proliferation. Transcriptomic profiling revealed 119 significantly upregulated and 99 downregulated genes in response to G9a inhibition. Notably, by integrating our gene expression dataset with published chromatin accessibility data, an increase in the androgen receptor (AR) signature and a concomitant decrease in the neuroendocrine (NE) features were observed in CM-272-treated samples. Additionally, pathway enrichment and master regulator analysis pinpointed both the androgen receptor and TP53 networks as significantly upregulated in treated tissues. In line with the reduction of the NE score, the MYC/MYCN were the most significantly downregulated transcription factors. Moreover, under the androgen pathway, upregulation of AZGP1, STK39, and DPP4 was observed. At the promoter level, a significant reduction in H3K9me2 levels was detected in treated samples, allowing, in turn, an increase in AR binding. Additionally, upon treatment, higher p53 binding was detected at the promoter of the p53-repressed genes AURKA, CDC25B, and CD44, whose expression was downregulated. Overall, targeting G9a with CM-272 might hold promising therapeutic potential for the management of advanced disease by modulating relevant tumoral transcriptional programs associated with core vulnerabilities in PCa.



 Filipa Moreira-Silva, Mark Zimmerli, Panagiotis Chouvardas, Beat Roth, Carmen Jerónimo, Marianna Kruithof-de Julio. G9a blockade uncovers therapeutic vulnerabilities in advanced prostate cancer, leading to tumor growth delay [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A045.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5911786a7c980e0c86fa5660d05b013ec96ea309" target='_blank'>
              Abstract A045: G9a blockade uncovers therapeutic vulnerabilities in advanced prostate cancer, leading to tumor growth delay
              </a>
            </td>
          <td>
            Filipa Moreira-Silva, Mark Zimmerli, P. Chouvardas, Beat Roth, Carmen Jerónimo, Marianna Kruithof-de Julio
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="For glioblastoma (GBM), therapeutic resistance shows a formidable obstacle to effective treatment. Recent insights into cellular plasticity as a novel paradigm for understanding this resistance suggest that tumor cells undergo phenotypic and molecular alterations during GBM progression, collectively termed cellular plasticity. These alterations arise from complex tumor microenvironment (TME) factors, epigenetic changes, and selective pressures induced by radiotherapy or temozolomide (TMZ), ultimately leading to significant tumor heterogeneity and therapeutic resistance. This review will examine the formation of GBM cell plasticity (GCP) and its manipulative mechanisms in therapeutic resistance. GCP arises from the integrated interaction between intrinsic epigenetic and transcriptional reprogramming and extrinsic TME signals, enabling GBM cells to flexibly transition between multiple states. These transitions are not random but constitute an adaptive evolutionary network for therapeutic resistance. We summarize the intrinsic and extrinsic mechanisms driving GCP and promoting the development of therapeutic resistance. Simultaneously, we systematically explore emerging therapeutic strategies targeting GCP, such as inhibiting or reversing GCP by targeting specific signaling pathways. Currently, multiple clinical trials worldwide are underway to improve GBM treatment resistance, spanning cutting-edge fields including targeted therapeutics and immunotherapy. These advances provide crucial directions for developing novel treatment modalities and combination therapies targeting GCP within the context of treatment resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c54eaa3b35945c7ca945585843bb898d84886fe" target='_blank'>
              Glioblastoma cell plasticity: A new paradigm in glioblastoma therapeutic resistance.
              </a>
            </td>
          <td>
            T. Xiao, Zhiling Wei, Minghua Wua
          </td>
          <td>2026-02-13</td>
          <td>Biochimica et biophysica acta. Reviews on cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 The CBFA2T3–GLIS2 (C/G) fusion defines an exceptionally aggressive, epigenetically driven subtype of pediatric AML (pAML) confined to infants (<3 years). Arising from a pericentric inversion of chromosome 16, it is the most frequent lesion in acute megakaryoblastic leukemia (AMKL; ∼30%) and also occurs in non-AMKL subtypes, indicating lineage-independent pathogenicity. Clinically, C/G+ leukemia shows relapse rates > 90% and < 15% five-year survival, making it among the most treatment-refractory pediatric cancers. Lacking cooperating mutations, C/G acts as a singular oncogenic driver that reprograms the enhancer landscape, yet how it sustains epigenetic programs enforcing leukemic identity and chemoresistance remains unknown. We integrated multi-omic and functional epigenetic profiling across patient samples, C/G+ AML lines, and a developmentally faithful cord-blood CD34+ HSPC co-culture model. DNA methylation (DNAm) was mapped by meEM-seq, chromatin states by CUT&RUN and ATAC-seq, and transcription by RNA-seq. Functional studies included DNMT3B knockout, gRNA-resistant rescue, and inducible dCas9–TET1 demethylation, complemented by promoter-capture Hi-C and C/G+ xenograft validation. C/G+ pAML exhibited a promoter-biased hypermethylation signature (Δβ ≥ 0.2 vs NBM), with ∼50% of altered CpGs at promoters. The fusion bound the DNMT3B promoter, inducing its transcription without global 5mC gain but driving focal hypermethylation at active promoters linked via long-range enhancer loops. These hypermethylated promoters paradoxically stabilized expression of CRE-residing genes (BMP2, MED12, ITGB2) enriched for EMT, KRAS, heme-metabolism, and apoptotic pathways. In the fetal C/G–UCB-HSPC model, these signatures were faithfully recapitulated, confirming that the fusion alone reprograms the fetal hematopoietic epigenome. Motif analysis identified E2F2 and E2F4 as transcription factors stabilizing methylated yet transcriptionally active CREs. DNMT3B loss reduced locus-specific 5mC and target-gene expression but upregulated DNMT1 and UHRF1, suggesting a compensatory maintenance shunt that preserves DNAm at apoptotic loci and raises the apoptotic threshold. Targeted demethylation of PMAIP1 (NOXA) restored its transcription and apoptosis, confirming that DNAm stabilizes but constrains inducibility of pro-apoptotic genes. In vivo, DNMT3B-deficient xenografts regained Venetoclax sensitivity and extended survival, phenocopying DNMT inhibition (DNMTi). This study identifies C/G as an epigenetic architect converting DNAm from a repressive to a stabilizing signal, locking leukemic enhancers into an active yet apoptosis-resistant state. By defining the C/G–DNMT3B–DNMT1/UHRF1 axis and transcription-factor-mediated maintenance of methylated CREs, we reveal a unified mechanism linking developmental origin, transcriptional persistence, and therapy resistance in pAML. These findings position aberrant DNAm as a druggable epigenetic vulnerability and provide the mechanistic rationale for DNMTi + BCL-2 blockade in high-risk, treatment-refractory pAML.



 Samrat Roy Choudhury, Arundhati Chavan, Rhonda E. Ries, Giselle Almeida Gonzalez, Soheil Meshinchi, Jason Farrar. CBFA2T3–GLIS2 fusion reprograms enhancer-linked DNA methylation to enforce apoptotic resistance and defines an epigenetic vulnerability in pediatric AML [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr PR006.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e01e458a3959338b14b992ffa07b3d02aa22cbfe" target='_blank'>
              Abstract PR006: CBFA2T3–GLIS2 fusion reprograms enhancer-linked DNA methylation to enforce apoptotic resistance and defines an epigenetic vulnerability in pediatric AML
              </a>
            </td>
          <td>
            S. R. Choudhury, Arundhati Chavan, R. Ries, Giselle Almeida Gonzalez, S. Meshinchi, Jason Farrar
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="In recent years, research of non-coding RNAs(ncRNAs)’ role in tumor drug resistance has made significant progress. Studies have revealed that various types of ncRNAs contribute to the emergence of tumor drug resistance through diverse mechanisms. NcRNAs are also showing promise as biomarkers for predicting drug resistance in various tumors. Furthermore, therapeutic strategies based on ncRNAs offer new opportunities to overcome drug resistance, although their clinical translation faces considerable challenges. This review is concentrated on the classification of ncRNAs, their roles in tumor drug resistance, their value as biomarkers, proposed therapeutic strategies, and the challenges and prospects for clinical translation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4629a0312339a05ff6ff87ea5b62d007f453951f" target='_blank'>
              Research Progress of Non-coding RNAs in Tumor Drug Resistance
              </a>
            </td>
          <td>
            Silin Lai
          </td>
          <td>2025-12-28</td>
          <td>Academic Journal of Science and Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Malignant transformation of mature cystic teratoma (MTMCT) of the ovary is a rare but aggressive malignancy for which no standardized chemotherapy or effective targeted therapies currently exist. To identify therapeutic vulnerabilities in MTMCT, we performed a genome-wide CRISPR-Cas9 knockout screen using the MTMCT-derived NOSCC1 cell line. Two parallel selective pressures were applied: in vivo tumorigenicity in immunodeficient mice and cisplatin exposure in vitro. From this screen, 67 negatively selected genes were identified, among which SOD1 and NDUFB4 emerged as top candidates based on high basal expression levels and clinical relevance. Integration with spatial transcriptomic data from three independent MTMCT patient tumors further supported the prioritization of these targets. SOD1 was selected for further investigation due to the availability of known pharmacological inhibitors. Both siRNA-mediated knockdown and small-molecule inhibition of SOD1 using LCS-1 significantly suppressed MTMCT cell proliferation in vitro by inducing oxidative stress and impairing cell cycle progression. This antiproliferative effect was reversed by co-treatment with N-acetylcysteine, a reactive oxygen species scavenger. In vivo validation using patient-derived xenograft models demonstrated that oral administration of LCS-1 led to significant tumor growth suppression and increased expression of apoptotic and DNA damage markers, including cleaved caspase-3 and γH2AX. These findings establish SOD1 as a critical vulnerability in MTMCT and provide preclinical evidence supporting redox modulation as a therapeutic strategy for this highly chemoresistant and understudied ovarian cancer subtype. Our integrative approach combining functional genomics, spatial transcriptomics, and pharmacologic validation offers a framework for the discovery of novel targets in rare gynecologic malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e67036000a59a86b2e3dd28f5c317d050f6d45c" target='_blank'>
              In-Tumor CRISPR-Cas9 Knockout Screening and Novel Therapy Development for Malignant Transformation of Ovarian Teratoma.
              </a>
            </td>
          <td>
            S. Tamauchi, Kosuke Yoshida, Xinyuan Wang, Atsushi Nakagawa, Akira Yokoi, N. Yoshikawa, K. Niimi, Yusuke Yamamoto, Hiroaki Kajiyama
          </td>
          <td>2026-01-13</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Micronuclei (MN) are small extranuclear chromosomal fragments that arise from genomic instability and serve as established biomarkers for genotoxicity and disease susceptibility. Once considered only markers of disease, they are now recognized as active, causative drivers of disease progression, driven by emerging evidence of epigenetic regulation within these structures. The micronucleus assay is recognized as a cost‐effective and minimally invasive method for monitoring genotoxicity resulting from both chronic and early exposure to environmental factors such as arsenic and lead, as well as from genetic instability associated with cancer progression. This review critically examines the expanding role of MN beyond traditional cytogenetic endpoints, with particular emphasis on recent insights into their epigenetic landscape. Mass spectrometry‐based studies have demonstrated that MN possess distinct histone posttranslational modification signatures compared to primary nuclei, including alterations in H3K27ac, H3K9ac, and H3K18ac. These modifications affect chromatin structure, gene expression, and DNA repair mechanisms. In the context of xenobiotic exposures, MN‐associated epigenetic changes may function as early indicators of disease progression. Additionally, rupture of the MN envelope can activate innate immune responses through the cGAS–STING pathway or result in chromothripsis, both of which contribute to cancer progression. The concept of “Micronuclear Epigenetics” is highlighted, with its potential application in high‐throughput diagnostic platforms, particularly liquid biopsy, discussed. This approach may enhance early detection and risk stratification in exposure‐induced toxicity and diseases such as cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a963e7666be290f5a7f2a006e9788007c4d9584f" target='_blank'>
              Potential of Micronuclear Epigenetic Signatures in Analyses of Toxicity and Genomic Instability
              </a>
            </td>
          <td>
            Somnath Paul, Ankita Das, A. Bandyopadhyay, Ashok K. Giri
          </td>
          <td>2026-01-27</td>
          <td>Cell Biology International</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8bb1ad1f1afa1b592836b21a838884cdc277c82" target='_blank'>
              Targeting EZH2 Oncogenic Splicing: Decoding the Regulatory Network and Antisense Correction
              </a>
            </td>
          <td>
            Md Rafikul Islam, Preeti Nagar, Naomi McNaughton, Shabiha Afroj Heeamoni, Md Mahbub Hasan, Shila Kandel, Panagiotis Tsakiroglou, W. Dalton, Omar Abdel-Wahab, A. Krainer, M. A. Rahman
          </td>
          <td>2026-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>110</td>
        </tr>

        <tr id="Simple Summary MELK is a serine/threonine kinase that regulates cell cycle progression, proliferation, apoptosis, migration, and stemness. MELK is overexpressed in many solid and hematologic malignancies. Furthermore, high MELK levels correlate with aggressive disease, poor survival, treatment resistance, and cancer stem-like features. Several small-molecule inhibitors and RNAi-targeting MELK have suppressed the development and progression of various cancers. OTSSP167 has shown promising results in phase I and II clinical trials. However, a study employing CRISPR/Cas9 knockout in breast cancer cell lines reported that loss of MELK did not impair proliferation. The review concludes that MELK is a robust prognostic biomarker of proliferation and aggressive behavior across multiple cancers but is not a universal driver of tumor growth. Discrepancies may arise from off-target effects of RNAi or inhibitors, context differences across cancer types, and differences in study settings. MELK is proposed as a conditional oncogenic vulnerability, especially in p53-deficient triple-negative breast cancer (TNBC) and glioma stem cells. Further research is needed to clarify the target selectivity of inhibitors, context-specific dependencies, and the optimal use of MELK-focused agents in combination therapies rather than as standalone treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0e62df49d62d42a4c4ea852f26c65300bd99df5" target='_blank'>
              Maternal Embryonic Leucine Zipper Kinase (MELK) in Cancer: Biological Functions, Therapeutic Potential, and Controversies
              </a>
            </td>
          <td>
            Alaeddin M. Alzeer, Saad Al-Lahham
          </td>
          <td>2026-01-01</td>
          <td>Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Chromosome 3p (chr3p) is frequently deleted in multiple cancers, indicating the presence of shared tumor suppressors. Analysis of genomic alterations in 33 different cancer types implicates the deletion or deleterious mutations of SET-domain-containing 2 (SETD2) at chr3p21 in significantly facilitating the formation of isochromosomes, consisting of two identical mirror-imaged arms, thereby promoting genomic instability conducive to large-scale chromosomal rearrangements and rapid cancer genome evolution. Fracturing of dicentric isochromosomes during cell division is pervasive and follows the dynamic fragmentation pattern of solids under impulse. Across cancers, isochromosomes form most frequently on chr8 to amplify the MYC-containing q-arm and chr17 to delete the TP53-containing p-arm. In the most aggressive uveal melanoma (UVM) subtype, chr3 deletion also includes MITF, a critical melanocyte differentiation and survival factor, and co-occurs with chr8q amplification. We demonstrate that MITF is a master transcriptional regulator of GNAQ/GNA11, mutated in 90% of UVM patients, and the associated synthetic-lethal genes identified by recent CRISPR screening studies. MITF maintains MAPK and calcium homeostasis in UVM, and its deletion is thus accidental, creating an early crisis during oncogenesis. We further show that MITF, MYC, and GNAQ/GNA11 form coupled regulatory feedback loops in the melanocyte lineage, and MITF deletion in UVM creates acute dependency on MYC-mediated rescue via chr8q amplification, often as a consequence of isochromosome formation. The discovered feedback loops predict both overall and relapse-free patient survival within the most aggressive UVM subtype, explain sensitivity to therapeutic gene perturbations, and inform effective combinatorial therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f31378489089eb1239e625480e1522a21f193cf" target='_blank'>
              Chromosome 3p deletion leads to extensive genomic alterations in diverse cancers and confers synthetic lethality in uveal melanoma
              </a>
            </td>
          <td>
            Mitchell C. Cutler, P. B. Howland, M. Hejna, Jun S. Song
          </td>
          <td>2026-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Lung cancer remains one of the malignancies with the highest incidence and mortality rates worldwide, and its treatment continues to pose significant challenges. Metabolic reprogramming, as one of the hallmarks of cancer, supports the abnormal growth, proliferation, invasion, and drug resistance of cancer cells by altering glucose, lipid, and amino acid metabolic pathways, providing both energy and biosynthetic precursors. It has thus become a critical focus in lung cancer research. Circular RNAs (CircRNAs), owing to their unique closed-loop structure and high stability, play important roles in regulating tumor metabolism and progression. This review systematically summarizes the molecular mechanisms through which CircRNAs drive metabolic reprogramming in lung cancer, including the regulation of key metabolic enzymes, influence on metabolism-related signaling pathways, remodeling of the tumor microenvironment, and mediation of epigenetic modifications. Furthermore, CircRNAs demonstrate great potential in clinical applications for lung cancer, not only as biomarkers for early diagnosis and prognostic evaluation but also as promising therapeutic targets. Leveraging their stability and low immunogenicity, the development of CircRNA-based vaccines and targeted delivery systems has opened new avenues for lung cancer immunotherapy. However, challenges remain in the synthesis of CircRNAs, understanding their in vivo metabolism, and achieving multi-target synergistic interventions, which warrant further investigation. This review provides a theoretical foundation for in-depth exploration of the metabolic regulatory network in lung cancer and the development of precise therapeutic strategies, while also highlighting the broad prospects of CircRNAs in translational medicine. We conducted a literature search across databases including PubMed up to 2025, focusing on keywords related to circRNA, lung cancer, and metabolic reprogramming. Ultimately, 161 relevant references were included in this narrative review.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76a3cf7f175fab2c4b319863353658de2a021512" target='_blank'>
              Metabolic reprogramming and lung cancer focused on roles, mechanism, and clinical prospects of circRNAs: a narrative review
              </a>
            </td>
          <td>
            Simin Chen, Mingxiao Li, Siyao Li, Yinhui Sun, Lihuai Wang
          </td>
          <td>2026-01-23</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5e44591a8b789bf0d9a5d9d40e6f22d5b068bba" target='_blank'>
              Mechanisms of resistance to bruton’s tyrosine kinase inhibitors: synergistic effects of tumor microenvironment regulation and signaling pathways
              </a>
            </td>
          <td>
            Xinyu Dong, Shujun Tang, Miaohong Chen, Qiuni Chen, Yifan Wang, Yuye Shi, Yuan Deng, Yue Chen, Zheng-mei He, Liang Yu, Chunling Wang
          </td>
          <td>2026-02-21</td>
          <td>Annals of Hematology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="BACKGROUND
Medullary thyroid carcinoma (MTC) is a rare neuroendocrine malignancy with a 10-year mortality rate up to 50%. Current therapies for metastatic MTC comprise various tyrosine-kinase inhibitors, but resistance often appears due to the need for lifelong treatments. Like in other tumors, genetic, epigenetic, post-transcriptional, post-translational, and cell-cell interaction events influence drug response. However, progress in understanding MTC biology is limited by the lack of reliable in vivo and in vitro models. This study aims to develop a patient-derived model faithfully reproducing the microenvironmental alterations present in MTC.


METHODS
We applied a two-step protocol consisting of a first phase in which primary cells are cultivated as multicellular spheroids and a second phase in which they are switched to adherent cultures. After evaluation of the genetic background by targeted Next Generation Sequencing, we characterized our cells phenotype by examining a panel of stem/progenitor-related markers, the secretory abilities by ELISAs, the drug response by proliferation assays, the in vivo angiogenic and invasiveness by the use of zebrafish model, the in vitro invasivity by Matrigel Dome assays and the spatial variation of stem/progenitor marker in both 3D cell models and tissue samples by confocal microscopy.


RESULTS
Our model allowed the establishment of eight MTC patient-derived cell lines with different genetic backgrounds. The cultures faithfully reproduced the changes in stem and progenitor markers that we detected in our cohort of MTC tissue samples and could be successfully xenotransplanted in zebrafish model, showing both angiogenetic and invasive properties. Drug screening assays revealed the potential of our model for the study of patient-specific responses, as we were able to identify different candidate regulators of the sensitivity to currently available therapies for MTC.


CONCLUSIONS
Our two-step protocol successfully generated primary MTC lines that maintain high plasticity, can be cultivated for several passages, and recreate the heterogeneity observed in patients' tissues. Our model will offer a robust platform for preclinical drug testing and mechanistic studies, addressing a longstanding gap in MTC research. It enables exploration of tumor microenvironment interactions and personalized therapeutic responses, supporting progress beyond current genomic-driven frameworks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/efc387a5fab7de78625be6d806bb5f60287b5f7b" target='_blank'>
              Patient-Derived in Vitro Models Reveal Insights into Medullary Thyroid Cancer Microenvironment and Resistance to Tyrosine Kinase Inhibitors.
              </a>
            </td>
          <td>
            E. Grassi, V. Ghiandai, G. Gaudenzi, E. Massardi, V. Cirello, Erika Carbone, Davide Gentilini, Sarah Uraghi, G. Gazzano, G. Dionigi, C. Colombo, Giovanni Vitale, L. Fugazzola, L. Persani
          </td>
          <td>2026-02-06</td>
          <td>Thyroid : official journal of the American Thyroid Association</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Deletions and duplications of 16p11.2 and 22q11.2, along with other copy number variants (CNVs), are strongly implicated in neurodevelopmental disorders, including autism spectrum disorder and schizophrenia. While clinical data provide valuable insights, such data are limited in uncovering precise cellular and molecular mechanisms, and animal models often lack direct human relevance. Human induced pluripotent stem cell-derived 2D neuronal cultures, 3D brain organoids, and assembloids with 16p11.2 or 22q11.2 CNVs offer complementary systems to investigate altered pathways for future clinical translation. Studies using patient-derived or clustered regularly interspaced short palindromic repeats/CRISPR-associated protein 9-engineered induced pluripotent stem cells carrying these CNVs have identified a range of phenotypes that yield mechanistic insights. This review consolidates these findings, highlighting convergent and divergent phenotypes across reciprocal CNVs, and proposes promising molecular and cellular readouts for advancing translational research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da6f9b7d7b63b25054deabf7fe4e8f7ef6f6d5c9" target='_blank'>
              Convergence and divergence of molecular phenotypes in iPSC-derived models of 16p11.2 and 22q11.2 reciprocal copy number variants.
              </a>
            </td>
          <td>
            Sandeep Rajkumar, Carrie E Bearden, J. Sebat, Lilia M. Iakoucheva
          </td>
          <td>2026-02-12</td>
          <td>Current opinion in genetics & development</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Gliomas are the most common primary malignant tumors of the central nervous system in adults, with diffuse midline gliomas (DMG) being particularly aggressive and associated with inferior survival rate. Notwithstanding advances in molecular diagnostics and epigenetics, the specific pathological mechanisms of DMG remain to be fully elucidated. A series of studies have demonstrated that histone modifications, particularly the histone H3 lysine 27 (H3K27)M mutation, play a pivotal role in the development and progression of DMG. The mutation disrupts histone methylation and acetylation to induce widespread gene expression abnormalities, tumor aggressiveness and treatment resistance. Conventional treatments such as surgery, local radiotherapy and chemotherapy offer limited efficacy. However, emerging precision therapies targeting histone mutations, epigenetic modifications and innovative immunotherapies show promise in improving outcomes. The present study provided a comprehensive overview of the molecular mechanisms, epigenetic characteristics and the latest therapeutic advances in DMG. By investigating the H3K27M mutation and its associated epigenetic mechanisms, the present review aimed to establish theoretical frameworks and research avenues for developing precise therapeutic strategies for DMG, thus contributing to advancing the field of personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97fc40a52d0fc673e763ab0256b2f074287d4597" target='_blank'>
              Epigenetic mechanisms and therapeutic advances in diffuse midline glioma (Review)
              </a>
            </td>
          <td>
            Wenbo Wu, Wenlin Chen, Wenbin Ma, Yu Wang
          </td>
          <td>2026-01-20</td>
          <td>Oncology Reports</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Genomic imprinting disorders (IDs) arise from the disruption of parent-of-origin-specific gene expression, a process governed by heritable epigenetic marks. Conventional therapies are largely symptomatic and fail to correct the underlying defect in dynamic gene regulation. Epigenome editing, using programmable DNA-targeting tools to rewrite epigenetic information, has emerged as a powerful therapeutic strategy that directly addresses this molecular pathology. This review synthesizes the transformative progress of the past decade, arguing that epigenome editing is poised to revolutionize the treatment of these conditions. We detail key breakthroughs in both editor design and in vivo delivery. These include the development of next-generation editors, such as compact “hit-and-run” systems that establish durable epigenetic memory after only transient expression—a critical feature for clinical safety. We then examine parallel advances in delivery platforms, including engineered adeno-associated virus (AAV) vectors with enhanced central nervous system (CNS) tropism and programmable lipid nanoparticles (LNPs) for precise extra-hepatic targeting. The therapeutic potential of combining these technologies is underscored by convincing preclinical data in models of Angelman (AS) and Prader-Willi syndromes (PWS), where the targeted rewriting of epigenetic marks has successfully reactivated silenced alleles and rescued disease-relevant phenotypes. Finally, we discuss the remaining challenges—including long-term durability and off-target safety—that must be overcome to translate this scientific potential into transformative medicines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51a215ce891005fe8fac9034ff5a7f3671a19208" target='_blank'>
              Advances and challenges of precision epigenetic therapy in treating genomic imprinting diseases
              </a>
            </td>
          <td>
            Ying Lv, Mingyan Li, Chai Ji
          </td>
          <td>2026-01-01</td>
          <td>Translational Pediatrics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The loss of chromosome 3p and the inactivation of the tumor suppressor gene von Hippel-Lindau (VHL) were identified in clear cell renal cell carcinomas (ccRCC) over three decades ago. Since then, mutations in genes for the three chromatin modulators, polybromo 1 (PBRM1), SET domain-containing 2 (SETD2), and BRCA1-associated protein-1 (BAP1), have been recognized as common in ccRCC. Although these genomic alterations are central to understanding ccRCC’s development, other deregulated cellular processes are also prominent in these tumors. Metabolic reprogramming is a key hallmark of this disease, characterized by various changes linked to the stabilization of hypoxia-inducible factors (HIF), including increased aerobic glycolysis, elevated lipid levels, and glutamine dependence for cell survival. Additionally, HIF-α stabilization plays a crucial role in regulating the immune system, thereby enhancing CD8+ T lymphocyte cytotoxicity. Immune checkpoint inhibitors (ICI) are now used as first-line treatments to target the often highly infiltrated tumor microenvironment of ccRCC. However, the effectiveness of ICI varies and is difficult to predict. Although emerging studies are beginning to provide insight, evidence suggests roles for PBRM1, SETD2, and BAP1 in metabolic regulation and in shaping the tumor immune microenvironment in ccRCC. Here, we review recent advances in this field and examine their impact on the management of ccRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74d8124d8b083fe78630c316b5f74781f77ea65c" target='_blank'>
              The fall of the genome protectors triad: PBRM1, SETD2, and BAP1’s impact on metabolism and immunity in clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            Mathieu Johnson, Sandra Turcotte
          </td>
          <td>2026-01-12</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Mechanisms driving the increase in cell growth in developing leukemia are not fully understood. We focused on epigenomic regulation of this process by analyzing the changes of chromatin marks and gene expression in leukemic cell clones as they progressed toward increased proliferation in a mouse model of acute myeloid leukemia (AML). This progression was characterized by gradual modulation of chromatin states and gene expression across the genome, with a surprising preferential trend of reversing the prior changes associated with the origins of leukemia. Our analyses of this modulation in independently developing clones predicted a small set of potential growth regulators whose transcriptomic and epigenomic progression was consistent between clones and maintained both in vivo and ex vivo. We selected three of these genes as candidates (Irx5 and Plag1 as growth suppressors and Smad1 as a driver) and successfully validated their causal growth effects by overexpression in mouse leukemic cells. Overexpression of the IRX5 gene in human MOLM13 leukemic cells suppressed cell growth both in vitro and in mouse xenografts. Public patient data confirmed expression levels of PLAG1 and SMAD1 as markers of AML status and survival, suggesting that multiomic analysis of evolving clones in a mouse model is a valuable predictive approach relevant to human AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e782878493e2646d7e50204cd904395a449780e" target='_blank'>
              Multiomic analysis of clonal development reveals new regulators of leukemic cell growth.
              </a>
            </td>
          <td>
            Gracia Bonilla, Alexander Morris, Sharmistha Kundu, Anthony Ducasse, Grace Kirkpatrick, Jelena Milosevic, N. E. Jeffries, Kashish Chetal, Emma E Yvanovich, Ting Zhao, Jun Xia, Rana Barghout, David T. Scadden, Michael K. Mansour, Robert E. Kingston, David B. Sykes, François E. Mercier, Ruslan I. Sadreyev
          </td>
          <td>2026-02-04</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background Neuronal cell adhesion molecule (NRCAM) has been implicated in glioma biology through splicing alterations reported by prior studies. However, the relative contributions of genomic and epigenetic mechanisms to NRCAM dysregulation in gliomas remain unclear. Methods We analyzed data from The Cancer Genome Atlas (TCGA) Lower-Grade Glioma (LGG) cohort using UCSC Xena and cBioPortal. Copy number variation (CNV), DNA methylation (Illumina HumanMethylation450 BeadChip arrays), and mutation profiles were assessed for NRCAM. Kaplan–Meier survival analyses were performed with Xena, stratifying patients by copy number status and methylation state. Correlations between mutation burden and fraction of genome altered (FGA) were evaluated using Spearman and Pearson methods. Results Somatic mutations in NRCAM were rare in the TCGA-LGG cohort. In contrast, CNV and methylation changes were frequent and clinically relevant. Copy number gains at the NRCAM locus were associated with significantly shorter overall survival, while higher methylation of NRCAM correlated with improved survival outcomes. NRCAM mutation count did not show a linear correlation with FGA, suggesting these alterations are largely independent of overall genomic instability. The findings highlight copy number imbalance and epigenetic regulation as predominant mechanisms of NRCAM dysregulation. Conclusion NRCAM is recurrently dysregulated in lower-grade gliomas through CNVs and DNA methylation, both of which stratify patient survival. Together with previously reported splicing deregulation, these data suggest that NRCAM functions as a multilayered regulator of glioma progression. NRCAM methylation may represent a potential prognostic biomarker, while therapeutic modulation of NRCAM warrants further investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a2630cdb2c0d21a56b05bf8738d1e26c2abc4bc" target='_blank'>
              Multi-Layered Dysregulation of NRCAM in Lower-Grade Gliomas: Insights From TCGA Copy Number and Epigenetic Analyses
              </a>
            </td>
          <td>
            S. Lehrer, P. Rheinstein
          </td>
          <td>2026-01-01</td>
          <td>Brain Tumor Research and Treatment</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da7a9e1a9e3edf4edb1c29327ece5ed4b0de7ab1" target='_blank'>
              Colorectal cancer risk variants in the 11q13.4 locus are associated with variable POLD3 transcript expression which may promote DNA damage and telomere shortening in colorectal cancer cells
              </a>
            </td>
          <td>
            E. Clarkson, N. DeLeoN, S. Aldulaimi, J. Fernández-Tajes, T. Roberts, I. Tomlinson, A. Lewis
          </td>
          <td>2025-12-27</td>
          <td>None</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Somatic mutations, key alterations in cancer development, exert differential effects across tissues and biological layers, such as transcriptomes, proteomes, and post-translational modifications (PTMs). Although previous pan-cancer studies have characterized the molecular landscape of cancer, the effects of individual somatic mutations across different tissues remain insufficiently explored. Here, we developed Panorama to evaluate the oncogenic potential of single somatic mutations across all cancer types. We collected cancer proteogenomics or multiomics data from over 10 000 individuals across 19 cancer types. Based on five evaluation criteria, we assessed whether a specific mutation affects the abundance of a particular gene’s transcriptome, proteome, or phosphoproteome; the tumor microenvironment; specific RNA- or protein-based signaling pathways; and outlier-level overexpression of PTMs, aiding in potential drug target identification. By leveraging five oncogenic metrics, Panorama quantifies the oncogenic potential of individual somatic mutations and provides a framework for identifying driver mutations by incorporating their downstream effects. With Panorama, researchers can integrate cancer proteogenomics data, providing a comprehensive approach that enhances our understanding of single somatic mutations in specific tissues. Finally, Panorama was developed as a web-based database to ensure easy access for researchers and is freely available at http://139.150.65.64:8080/or https://github.com/prosium/panorama.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3988e7434c0ad5c1c94f629876b4c62c68b08126" target='_blank'>
              Panorama: a database for the oncogenic evaluation of somatic mutations in pan-cancer
              </a>
            </td>
          <td>
            Seung-Jin Park, Seon-Young Kim
          </td>
          <td>2026-01-15</td>
          <td>Database: The Journal of Biological Databases and Curation</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2879e795e7efcebd514454af1fc1a736f45a29d8" target='_blank'>
              Somatic and germline mutational processes across the tree of life
              </a>
            </td>
          <td>
            Sangjin Lee, Yichen Wang, Haynes Heaton, Emily Mitchell, M. Maddison, Liam M. Crowley, Patrick Adkins, N. Mieszkowska, Mark Blaxter, R. Durbin, P. Campbell
          </td>
          <td>2025-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="BACKGROUND
Cancer cells are heterogeneous, each harboring distinct molecular aberrations and being dependent on different genes for their survival and proliferation. While targeted therapies based on driver DNA mutations have shown success, many tumors lack druggable mutations, limiting treatment options. We hypothesize that new precision oncology targets may be identified through "expression-driven dependency," where cancer cells with high expression of specific genes are more vulnerable to the knockout of those same genes.


RESULTS
We developed BEACON, a Bayesian approach to identify expression-driven dependency targets by analyzing global transcriptomic and proteomic profiles alongside genetic dependency data from cancer cell lines across 17 tissue lineages. BEACON successfully identified known druggable genes, including BCL2, ERBB2, EGFR, ESR1, and MYC, while revealing novel targets confirmed by both mRNA and protein-expression driven dependency. The identified genes showed a 3.8-fold enrichment for approved drug targets and a 7 to 10-fold enrichment for druggable oncology targets. Experimental validation demonstrated that depletion of GRHL2, TP63, and PAX5 effectively reduced tumor cell growth and survival in their dependent cells.


CONCLUSIONS
Our approach provides a systematic method to identify precision oncology targets based on expression-driven dependency patterns. By integrating multi-omics data with genetic dependency screens, we've created a comprehensive catalog of potential therapeutic targets that may expand treatment options for cancer patients lacking druggable mutations. This resource offers new opportunities for precision oncology target discovery beyond mutation-based approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39d9df8d8c54b9f6f78829d05c5029fee1f72554" target='_blank'>
              Expression-Driven Genetic Dependency Reveals Targets for Precision Oncology.
              </a>
            </td>
          <td>
            Abdulkadir Elmas, Hillary M. Layden, Jacob D. Ellis, Luke N. Bartlett, Xian Zhao, Reika Kawabata-Iwakawa, Zishan Wang, Hideru Obinata, Scott W. Hiebert, Kuan-lin Huang
          </td>
          <td>2026-01-29</td>
          <td>GigaScience</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="


 Chromosomal aberrations are frequent in cancer and play a pivotal role in disease initiation and progression. In prostate cancer (PCa), deletions at chromosome 13q14 are common and enriched in advanced disease, suggesting selective advantage in tumor progression. Beyond BRCA2, this region harbors other tumor suppressor genes, including RB1 and RNASEH2B, which impact sensitivity to androgen receptor inhibitors (ARi) and PARP inhibitors (PARPi). This study aims to elucidate the effect of individual vs concomitant loss of RB1 and RNASEH2B in response to dual AR and PARP inhibition.



 To explore the prevalence of deletions in 13q we interrogated copy number alterations in the Hartwig Medical Foundation (HMF) PCa WGS dataset (n=486). In vitro, we generated CRISPR-Cas9-mediated knock-outs (KO) of RB1, RNASEH2B, or both genes in PCa cell lines (LNCaP, C4-2, 22Rv1, DU145). Gene editing was confirmed by Sanger, and western blot analysis validated protein loss. Antitumor activity of enzalutamide (ARi), and olaparib or saruparib (PARPi) was determined using CCK8 kit and colony formation assays.



 RB1 deletions were present in 42% of HMF PCa samples (205/486), being mostly heterozygous deletions (200/205). RB1 deletions co-occurred with RNASEH2B loss in 94% of cases (192/205) and with BRCA2 deletions in 42% of cases (87/205). In vitro, RB1-KO increased resistance to ARi and PARPi in hormone-sensitive lines LNCaP and C4-2, while no change in sensitivity was detected in AR-independent models 22Rv1 and DU145. Loss of RNASEH2B increased sensitivity to PARPi in all models. Next, we generated RB1 and RNASEH2B double KO models in LNCaP and C4-2 by sequentially editing single KO populations. After single-cell seeding, we obtained: (1) RB1-KO (homozygous) clones with complete protein loss; (2) RNASEH2B-heterozygous (RNASEH2B-Het) models with partial protein expression; and (3) RB1-KO/RNASEH2B-Het models. No clones with complete RNASEH2B loss were obtained, suggesting that RNASEH2B-loss population is transient and that clones with complete loss do not proliferate. RB1-KO models maintained resistance to both treatments. RNASEH2B-Het cells showed increased sensitivity to PARPi, although to lower extent than the population with complete RNASEH2B loss. Importantly, introducing RB1-loss in RNASEH2B-Het cells restored PARPi resistance, and combined AR/PARP inhibition did not overcome resistance in RB1-KO or RB1-KO/RNASEH2B-Het cells.



 Our findings indicate that 13q14 deletions in PCa predominantly occur as heterozygous events encompassing key genes influencing ARi and PARPi responses. Heterozygous loss of RNASEH2B confers sensitivity to PARPi, although to a lesser extent than in models with complete RNASEH2B loss. RB1-loss confers higher resistance to ARi and PARPi, restoring resistance to PARPi in RNASEH2B heterozygous models. These results highlight the need for models that better mirror the heterozygous nature of 13q14 deletions to clarify their biological effects and refine therapeutic strategies for advanced PCa.



 Victor Esquefa, Pablo Cresta Morgado, Richard Norris, Lara de Llobet, Julian Brandariz, Sara Arce-Gallego, Teresa Casals, Manuel Ramos, Daniel Aguilar, Gisela Mir, Irene Casanova-Salas, Francisco M. Barriga, Joaquin Mateo. Impact of 13q.14 loss in prostate cancer progression and sensitivity to targeted agents [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr B018.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15430fc4c6c79184273233030845f22a6f4ecd5a" target='_blank'>
              Abstract B018: Impact of 13q.14 loss in prostate cancer progression and sensitivity to targeted agents
              </a>
            </td>
          <td>
            Victor Esquefa, P. C. Morgado, Richard Norris, L. D. De Llobet, Julian Brandariz, Sara Arce-Gallego, Teresa Casals, Manuel J. Ramos, Daniel Aguilar, Gisela Mir, I. Casanova-Salas, Francisco M. Barriga, Joaquin Mateo
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62da73e71e4c90732c0fe5d7f97a1940897a3924" target='_blank'>
              Adaptation of lentiviral vectors for viral gene therapy and their impact on host cell biology
              </a>
            </td>
          <td>
            Catherine W. Kaiser, E. Rouchka, Melissa L. Smith
          </td>
          <td>2026-01-27</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (eccDNA) has emerged as a pivotal contributor to cancer progression, facilitating oncogene amplification, dysregulated gene expression, and tumor heterogeneity. Despite its significance in cancer, the interplay between eccDNA and key epigenetic regulators such as EZH2 remains largely unexplored. In this study, we systematically investigate the correlation between Tazemetostat, a highly selective EZH2 inhibitor, and alterations in the eccDNA landscape and transcriptional programs in ovarian cancer. Through integrated profiling using Circle-seq and RNA sequencing, we demonstrate that EZH2 inhibition is associated with markedly reprogrammed eccDNA dynamics. Furthermore, multi-omics integration identified that 67 genes exhibited concordant changes in both eccDNA abundance and transcript expression. Subsequent analyses also pinpointed 11 genes as putative effectors of drug response. Notably, spatial single-cell transcriptomics identified SFRP1 as the most consistently reactivated tumor suppressor across eccDNA, bulk expression, and spatial datasets, based on predefined statistical and biological criteria, by Tazemetostat. Moreover, SFRP1 was one of the genes that varied the most within cancer-associated fibroblast populations, exhibiting distinct spatial expression patterns. Taken together, this study establishes the first potential evidence that EZH2 inhibition may reprogram eccDNA dynamics to potentially restore SFRP1 tumor suppressor expression in ovarian cancer. By integrating multi-omics and spatial single-cell transcriptomics, we uncovered a novel epigenetic–eccDNA axis that may contribute to oncogenic plasticity and therapeutic resistance. This could result in a paradigm shift in targeting eccDNA-driven malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6e68c77ce796a11b7676543278dbc14c7ad88f7" target='_blank'>
              EZH2 Inhibition Restores Tumor Suppressor SFRP1 Activity by Reprogramming Extrachromosomal Circular DNA Dynamics in Ovarian Cancer
              </a>
            </td>
          <td>
            Tao Han, Qingya Yan, Yaqi Zhang, Yu Gan, Kaifan Li, Liping Guan, Changqin Jing, Ciqing Yang, Pengfei Li, Bo Gao, Xiang Zhou, Qian Hao
          </td>
          <td>2026-02-15</td>
          <td>Biology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ffb82464a4840ef8616fc6853b84c7285b6e602" target='_blank'>
              Functional footprints of homologous recombination deficiency in prostate cancer revealed by ctDNA fragmentation and transcription factor accessibility.
              </a>
            </td>
          <td>
            Georgios Vlachos, T. Moser, Isaac Lazzeri, Matthias J. Moser, Lisa Glawitch, Emil Bauernhofer, Anna Eberhard, C. Beichler, Hanieh Sadeghi, J. Blatterer, Stefan Kühberger, N. Monsberger, A. Terbuch, Karl Kashofer, J. Geigl, T. Bauernhofer, E. Heitzer
          </td>
          <td>2026-01-09</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Cutaneous T-cell lymphoma (CTCL) remains a challenging disease due to its significant heterogeneity, therapy resistance, and relentless progression. Multi-omics technologies offer the potential to provide uniquely precise views of disease progression and response to therapy. We present here a comprehensive multi-omics view of CTCL clonal evolution, incorporating exome, whole genome, epigenome, bulk, single cell (sc) TCR, and scRNA sequencing of 99 clinically annotated serial skin, peripheral blood, and lymph node samples from 34 CTCL patients. We leveraged this extensive dataset to define the molecular underpinnings of CTCL progression in individual patients at single cell resolution with the goal of identifying clinically useful biomarkers and therapeutic targets. Our studies identified recurrent progression-associated clonal genomic alterations; we highlight mutation of CCR4, PI3K signaling, and PD-1 checkpoint pathways as evasion tactics deployed by malignant T-cells. We identified a gain of function mutation in STAT3 (D661Y) and demonstrated by CUT&RUN- and RNA-seq that it enhances binding to and transcription of genes in Rho GTPase pathways. With our previous work implicating this pathway in HDACi-resistant CTCL, these data provide further support for a previously unrecognized role for Rho GTPase pathway dysregulation in CTCL progression. Recurrent progression-associated mutations were common in the epigenetic modifier EZH2, suggesting that EZH2 inhibition may benefit patients with CTCL. Our findings support an approach in which genomic analysis is widely utilized for improved disease monitoring, biomarker-informed clinical trial design, and genome-guided therapeutic decision making. Moreover, these molecular changes present new opportunities for therapeutic targeting in this challenging and incurable cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80a0bc9d50b98f66d82985f089d8566527ff9738" target='_blank'>
              Multi-omic Study of Cutaneous T-Cell Lymphoma Reveals Single Cell Clonal Evolution in Progression and Therapy Resistance.
              </a>
            </td>
          <td>
            Hannah K. Dorando, Jared Andrews, Oam Khatavkar, N. Borcherding, Yulia Korshunova, Gabriela Hakeman, Rodrigo Borgiani Panigassi, Phuong Vo, Ruei-yuan Tu, Diep Tran, Chaz C. Quinn, J. Schmidt, Jahnavi Aluri, Michael T Harmon, Marcus P Watkins, Anastasia Frank, M. Cooper, Amy C. Musiek, N. Mehta-Shah, Jacqueline E. Payton
          </td>
          <td>2026-02-09</td>
          <td>Blood</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="


 Somatic structural variations (SVs) are common in cancer. Although a small fraction of SVs in breast and ovarian cancers can be attributed to homologous recombination deficiency, the underlying molecular mechanisms for the vast majority of somatic SVs remain unclear. Here, we focus on the roles of transcription and DNA replication collisions in genomic instability in cancer. Such collisions are unavoidable in cells since both transcription and replication use the same DNA as template. We hypothesized that transcription replication collisions (TRCs), if not properly repaired, would lead to collapsed replication forks and result in SVs. To this end, we studied somatic SVs in 6193 high-coverage whole-genome sequenced primary and metastatic tumors from three independent pan-cancer cohorts. A total of 13 conserved SV signatures, representing independent molecular mechanisms, were deconvoluted from these cohorts using non-negative matrix factorization approach. We detected replicated-strand bias, the expected footprint of transcription-replication collision, in large tandem duplications (TDs) across multiple cohorts. This bias was only observed in expressed genes, consistent with TRCs depending on transcription activity. Large TDs were abundant in female-specific (breast, ovarian and uterus), upper gastric-intestinal tract, and prostate cancers. They were associated with CDK12 mutations and worse patient survival. CDK12 is a cyclin-dependent kinase, a key regulator of transcription elongation and termination. Deleting or suppressing CDK12 using CRISPR-Cas9 in prostate cell lines increased RNA:DNA hybrids (R-loops), promoted TRCs, and ultimately led to large TDs. Finally, we found that cells lacking CDK12 were sensitive to WEE1, CHK1 and ATR inhibitors. In summary, our data suggest that large TDs in cancer form due to impaired TRC repair and can be used as a biomarker for prognosis and treatment.



 Lixing Yang. A078: Transcription and DNA replication collisions lead to large tandem duplications and expose targetable therapeutic vulnerabilities in CDK12-mutant prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A078.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a771b9c6597f1781d70e2b6d5df9ff54196c4d85" target='_blank'>
              Abstract A078: Transcription and DNA replication collisions lead to large tandem duplications and expose targetable therapeutic vulnerabilities in CDK12-mutant prostate cancer
              </a>
            </td>
          <td>
            Lixing Yang
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4773775e6ecfd7cb45043c77dfe433bc00bd2ae5" target='_blank'>
              Epigenetic Intelligence: How Organisms Track Their Environment Through Molecular Memory
              </a>
            </td>
          <td>
            Holly V. Moeller, H. Putnam, Ross Cunning, Steven B. Roberts, J. Eirín-López, R. Nisbet
          </td>
          <td>2026-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="HTLV-1 and HIV-1 represent biologically significant, structurally close, and equally problematic yet divergent human retroviruses. Although both infect CD4+ T cells and share similar structural elements, they differ markedly in genomic stability, transmission dynamics, clinical progression, and, most importantly, their transcriptional regulatory mechanisms. HTLV-1, an ancient virus with a limited global burden, often remains asymptomatic for decades before potentially causing ATL or HAM/TSP. Conversely, HIV-1, a relatively recent zoonotic transmission, undergoes rapid replication, exhibits high genetic diversity, and causes progressive immunodeficiency unless controlled by antiretroviral therapy (ART). At the molecular level, HTLV-1 maintains proviral latency through a balanced bidirectional transcription of regulatory genes (e.g., Tax and HBZ) that manipulate host transcription and immune evasion pathways, facilitating persistence and oncogenesis. HBZ and Tax were shown to contribute to driving the progressive acquisition of Treg-like and HLA class II phenotype in chronically activated CD4+ T-cells, promoting tolerogenic antigen presentation and immune evasion in ATL cells. This well-controlled differential expression of HTLV-1 regulatory genes is attributed to multiple intragenic virus regulatory mechanisms, which will be discussed in this review. In contrast, HIV-1 transcription is driven by a tightly regulated 5′ LTR promoter involving host factors such as NF-κB, Sp1, AP-1, and NFAT, among others, with strong influence imposed by the landscape of the provirus integration site, playing a pivotal role in latency and reactivation. The distinct regulatory circuitry of each virus suggests a key difference in their essential regulation, with HTLV-1 primarily relying on intragenic mechanisms, while HIV-1 relies more heavily on interactions with the surrounding host environment to control its expression. This difference underscores unique therapeutic challenges in managing viral latency, persistence, and pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99fe79a471f1e82a9805e90122fa48da428a7b93" target='_blank'>
              The Differentially Regulated Cousins: Insights into the Differences in Transcriptional Regulatory Mechanisms Between HTLV-1 and HIV-1
              </a>
            </td>
          <td>
            Omnia Reda, Yorifumi Satou
          </td>
          <td>2026-01-01</td>
          <td>Viruses</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The transcription factor nuclear factor erythroid 2-related factor 2 (NRF2) is best known for its regulation of the antioxidant response. However, its mediation of other pathways, including key aspects of metabolic and protein homeostasis, has continued to emerge. Accompanying this emergence is an evolved understanding that NRF2 induction across different disease contexts can be beneficial or detrimental depending on the length of activation. This has played an important role in progressing the field forward, as inducing NRF2 is not always the best course of action, and inhibition has gained traction as a viable strategy for treating cancer and other pathologies where NRF2 is chronically active. Despite its rapid growth and a wealth of experimental promise, a persistent shortcoming in the field is a lack of NRF2-specific therapeutics used in clinic. Thus, despite recent advances, there is still room for progress in translating experimental evidence into therapeutic reality. In this review, we will provide a summary of NRF2 regulation and an update on its expanded network of downstream transcriptional programs. We will also discuss targeting NRF2 in disease, focusing on intervention versus prevention depending on the pathological context. Finally, we will briefly highlight current limitations in the field, as well as ongoing approaches that show promise for finally targeting this critical cascade in patient populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbd0d54664279b624663e50935fb85788b566b64" target='_blank'>
              NRF2: Master regulator of cellular homeostasis and therapeutic vulnerability in cancer
              </a>
            </td>
          <td>
            Wei-tai Chen, Nicholas W McKee, Damaris Kuhnell, Matthew Dodson
          </td>
          <td>2026-01-01</td>
          <td>Redox Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="CRISPR/Cas systems have transformed molecular medicine, yet the field still lacks principled guidance on when transient editing suffices versus when sustained exposure through in vivo viral delivery is necessary and how to keep prolonged exposure safe. Notably, EDIT-101 was designed for a permanent edit in post-mitotic photoreceptors with lifelong Cas9 persistence. This review addresses this gap by defining the biological and therapeutic conditions that drive benefit from extended Cas activity while minimizing risk. We will (i) examine relationships between expression window and efficacy across Cas9/Cas12/Cas13 modalities, (ii) identify genome-wide off-target liabilities alongside orthogonal assays, and (iii) discuss controllable, self-limiting, and recallable editor platforms. By separating durable edits from persistent nuclease exposure, and by providing validated control levers, this work establishes a generalizable framework for safe, higher-efficacy CRISPR medicines. Furthermore, we highlight key studies in cell lines, murine models, non-human primates, and humans that examine the long-term effects of sustained expression of CRISPR/Cas systems and discuss the safety and efficacy of such approaches. Current evidence demonstrates promising therapeutic outcomes with manageable safety profiles, although there is a need for continued monitoring as CRISPR/Cas therapies are increasingly applied in clinical contexts and therapies are developed for broader clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93ffced530e1e86e12ebc967a34622935497f148" target='_blank'>
              Chronic In Vivo CRISPR-Cas Genome Editing: Challenges, Long-Term Safety, and Outlook
              </a>
            </td>
          <td>
            Caroline Bao, Catherine I. Channell, Yi-Hsuan Tseng, J. Bailey, Naeem Sbaiti, A. Demirkol, Stephen H. Tsang
          </td>
          <td>2026-01-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/704dba0681951d72abca65e0d6b244073c98cbe4" target='_blank'>
              Protocol for Non-viral HDR-based CRISPR/Cas9 platform for small custom editing in primary T cells
              </a>
            </td>
          <td>
            Katariina Mamia, Anniken Solveig Matheson Sollano, Shiva Dahal-Koirala, E. Haapaniemi
          </td>
          <td>2026-02-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="
 Although CD8+ T cells face intense immunosuppressive and metabolic pressures within the tumor microenvironment (TME), a subset of these cells can mount resilient responses to immune checkpoint inhibitor (ICI) therapy. We recently identified a population of such resilient CD8+ T cells in patients with advanced solid tumors who were unresponsive to standard-of-care cancer therapies but showed clinical benefit from ICI or combination ICI therapy. Among these resilient CD8+ T cells, which retain their cytotoxic capacity to eliminate tumor cells, we observed upregulation of ME1 and downregulation of DUSP2 in responders to the combination therapy. However, the mechanistic role of altered ME1 and DUSP2 expression in shaping resilient CD8+ T cell responses in cancer remains to be fully elucidated. To that end, we used scRNA-seq, bulk RNA-seq, and ATAC-seq analyses to investigate how altered expression of DUSP2 and ME1 regulates transcription of genes encoding effector molecules in both human and mouse CD8+ T cells. Additionally, we utilized CD8+ T cell-specific DUSP2 knockout (KO) and ME1 transgenic (Tg) mouse models to define the roles of these genes in effector T cell differentiation and antitumor responses in vivo. We found that DUSP2 downregulation enhances the expression of effector molecules (e.g., GZMB and NKG7) in CD8+ T cells, accompanied by increased cytotoxic activity. Mechanistically, reduced DUSP2 expression promotes chromatin accessibility at effector gene loci by modulating the nuclear residency balance between BAF and NuRD, two chromatin remodeling complexes with opposing functions. In parallel, ME1 overexpression enables activated CD8+ T cells to bypass the classical glycolysis pathway in favor of the pentose phosphate pathway (PPP), thereby boosting ATP production while minimizing excess ROS accumulation and increasing effector molecule expression via an epigenetic mechanism. Our study reveals a previously unknown adaptive mechanism by which resilient CD8+ T cells maintain their cytotoxic function within the TME. This is achieved through integrated reprogramming of epigenetic regulation and metabolic fitness, enabling compensation for metabolic stress in the TME and restoration of responsiveness to ICI-based combination therapy, even in patients with substantial tumor burden.


 Haidong Dong. Epigenetic and metabolic reprogramming for resilient CD8+ T cell responses in the tumor microenvironment [abstract]. In: Proceedings of the AACR Immuno-Oncology Conference (AACR IO): Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2026 Feb 18-21; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2026;14(2 Suppl):Abstract nr A001.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5bf8b50b59a208bd7237cad2dbaf754deb73528" target='_blank'>
              Abstract A001: Epigenetic and metabolic reprogramming for resilient CD8+ T cell responses in the tumor microenvironment
              </a>
            </td>
          <td>
            Haidong Dong
          </td>
          <td>2026-02-18</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Lung cancer persists as the leading cause of cancer-related mortality globally, largely due to late detection, genomic complexity, and limited durability of existing therapeutic interventions. The integration of CRISPR/Cas9 gene-editing systems with biomarker-driven therapeutic strategies represents a transformative advance in precision oncology. Biomarkers—including genetic mutations, epigenetic alterations, protein signatures, and circulating analytes—enable early detection, patient stratification, and dynamic monitoring of therapeutic response. CRISPR/Cas9 offers a unique opportunity to directly reprogram these biomarkers or the pathways regulating them, enhancing tumor immunogenicity, reversing immune evasion mechanisms, and strengthening anti-tumor immune responses. Preclinical models demonstrate that CRISPR-mediated biomarker editing can restore antigen presentation, augment T-cell cytotoxicity, sensitize resistant tumors to immunotherapy, and improve tumor regression. Early clinical trials further validate the feasibility and safety of CRISPR-engineered immune cells in patients. 
However, major challenges persist, including off-target editing, inefficient delivery to solid tumors, tumor microenvironment–mediated suppression, and ethical considerations linked to genome manipulation. Rapid advancements in editing fidelity, lipid nanoparticle systems, viral vectors, engineered vesicles, high-throughput biomarker discovery, and artificial intelligence–assisted CRISPR design are expected to accelerate clinical translation. This review synthesizes the current landscape, mechanistic underpinnings, emerging applications, and future directions of CRISPR/Cas9-enabled biomarker engineering for lung cancer immunotherapy, positioning this technology as a cornerstone of next-generation personalized oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d80a98d71ca868cfed559df1a6401f5e2e8b0702" target='_blank'>
              Precision Reprogramming of Lung Cancer Biomarkers via CRISPR/Cas9: A Paradigm Shift in Personalized Immunotherapy
              </a>
            </td>
          <td>
            M. R. Jameel, Amar Arora, Prem Shankar Mishra, Hina Jameel, M. Makhdoomi, Abid Ali Sheikh
          </td>
          <td>2025-12-30</td>
          <td>Journal of Analytical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="CRISPR-based therapeutics represent a transformative advancement in molecular medicine by enabling precise, programmable modification of genomic sequences for the treatment of genetic and acquired diseases. Unlike conventional pharmacological interventions that primarily modulate protein function, CRISPR technologies directly target the underlying genetic determinants of pathology, offering the potential for durable or permanent therapeutic effects. This paper examines the molecular mechanisms of CRISPR-mediated gene-targeted drug action, focusing on DNA recognition, cleavage, repair pathways, and emerging gene-editing modalities such as base editing and prime editing. By synthesizing developments in genome engineering, delivery systems, and off-target mitigation strategies, the study evaluates the clinical promise and technical limitations of CRISPR-based interventions. The findings underscore the growing role of gene-editing therapeutics in precision medicine while highlighting ethical, regulatory, and safety considerations that accompany their translational deployment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/601d94e7bfe27f0434bb1469e7b781cc59019561" target='_blank'>
              CRISPR-Based Therapeutics: Molecular Mechanisms of Gene-Targeted Drug Action
              </a>
            </td>
          <td>
            Kaan Karakas
          </td>
          <td>2026-02-07</td>
          <td>Next Frontier For Life Sciences and AI</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Whole-genome doubling (WGD) is a frequent yet poorly understood genomic event linked to adverse clinical outcomes.
 Multiple distinct mechanisms can lead to tetraploidization and aneuploidy, including defects in the mitotic checkpoint, chromosome cohesion, and disruptions of microtubules, spindle assembly, or centromere function. In addition, cell-cell fusion events can also result in polyploidization. Polyploid cells often exhibit chromosomal instability, which contributes to increased intratumoral heterogeneity and accelerates the evolution of the cancer genome. Although chromosomal instability and the presence of tetraploid or hyperdiploid cancer cells have been linked to treatment resistance and enhanced phenotypic diversity, the molecular mechanisms driving these associations remain poorly understood.
 We began by assessing the frequency of WGD in breast cancer across tumor subtypes and stages of disease progression. To do this, we performed WGD calling or reanalyzed publicly available genomic sequencing datasets with annotated WGD status. Analysis of three datasets revealed that WGD occurs more frequently in HER2+ and triple-negative breast cancers (TNBC) compared to hormone receptor-positive tumors. Furthermore, WGD frequency was consistently higher in distant metastases than in primary tumors.
 We then established through homofusion several murine mammary tumor models of WGD of TNBC to investigate how WGD drives tumor evolution. We observed that WGD increases transcriptomic and epigenetic heterogeneity and identified the survivin inhibitor YM155 as a selective suppressor of WGD+ tumors.
 Importantly, WGD promoted immune evasion by enabling escape from CD8+ T cell-mediated immune responses, rendering WGD+ tumors more sensitive to anti-PD-L1 therapy. Through single-cell profiling, we found that WGD+ cancer cells exhibit impaired antigen presentation, driven in part by a reduced response to IFN-γ and epigenetic silencing of MHC class I transcriptional regulators via the PRC2 complex.
 Treatment with a PRC2 inhibitor preferentially inhibited WGD+ tumor growth, restored antigen presentation, and enhanced CD8+ T cell infiltration. These findings suggest that combined inhibition of PRC2 and PD-1 represents a promising therapeutic approach for WGD+ breast cancers.



 Whole genome doubling, Breast Cancer, single cell profiling, immune escape



 P. Foidart, Z. Li, X. Can, M. Seehawer, E. Rojas-Jimenez, P. Baldominos, J. Nishida, T. Bui, B. Diciaccio, S. Parvin, M. Goyette, P. Yan, X. Qiu, R. Li, Y. Jiang, Y. Xie, X. Huang, L. E. Stevens, P. Cejas, L. Mangiante, C. I. Sotomayor Vivas, K. Houlahan, T. Scales, I. S. Harris, C. Curtis, A. G. Letai, H. W. Long, J. Agudo, K. Polyak. Whole-genome doubling promotes immune evasion in TNBC by epigenetically silencing antigen presentation pathways through PRC2 activity [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2025; 2025 Dec 9-12; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2026;32(4 Suppl):Abstract nr PS2-11-27.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08753d581f8856e6e0eafe884e8eb564b13b727d" target='_blank'>
              Abstract PS2-11-27: Whole-genome doubling promotes immune evasion in TNBC by epigenetically silencing antigen presentation pathways through PRC2 activity
              </a>
            </td>
          <td>
            P. Foidart, , X. Can, M. Seehawer, E. Rojas-Jimenez, P. Baldominos, J. Nishida, T. Bui, B. Diciaccio, S. Parvin, M. Goyette, P. Yan, , R. Li, Y. Jiang, Y. Xie, X. Huang, L. Stevens, P. Cejas, L. Mangiante, C. Vivas, K. Houlahan, T. Scales, I. Harris, C. Curtis, A. Letai, H. W. Long, J. Agudo, K. Polyak
          </td>
          <td>2026-02-17</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b370747cb8a1720741d860f50d6f1043c88dbdfc" target='_blank'>
              Cancer stem cell-driven drug resistance in colorectal carcinoma: molecular aspects and therapeutic potentials.
              </a>
            </td>
          <td>
            Minfeng Zhou, Huifang Niu, Dandan Cui, Menghao Xu, Jinxiao Li, Guichen Huang, Minquan Zhou, Chutong Xiong, Yunya Liu, Xiaojuan Xu, Hongxing Zhang, Fengxia Liang, Rui Chen
          </td>
          <td>2026-01-22</td>
          <td>Molecular cancer</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Cancer remains a major global health challenge, largely due to its biological heterogeneity and the capacity of tumor cells to adapt under metabolic and environmental stress. Lipid metabolism has increasingly been recognized as a contributor to tumor progression and treatment response. Proprotein convertase subtilisin/kexin type 9 (PCSK9), widely known for regulating low-density lipoprotein (LDL) receptor turnover and systemic cholesterol levels, has recently been implicated in cancer biology. Emerging evidence shows that PCSK9 influences processes such as cell survival, MHC-I-mediated immune recognition, membrane receptor trafficking, and cellular stress responses, indicating roles that extend beyond its canonical metabolic function. These mechanisms also raise the potential relevance of PCSK9 to affect treatment tolerance and drug responsiveness. This review summarizes current knowledge on the biological functions of PCSK9 in cancer and examines how these pathways may have implications for therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c7859c4220975dd2b3107c28a2b8c052d201aae" target='_blank'>
              PCSK9 in Cancer: Biological Mechanisms and Implications for Therapeutic Resistance
              </a>
            </td>
          <td>
            N. Azmi, Rezi Riadhi Syahdi, M. U. Puteri, Arry Yanuar, Mitsuyasu Kato, F. Saputri
          </td>
          <td>2025-12-31</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="


 Clonal hematopoiesis (CH) is exemplified by the expansion of genomically altered hematopoietic stem and progenitor cells that may give rise to altered immune repertoires. Driven largely by age and chronic inflammation, CH has been historically linked to myeloid malignancies. Although emerging evidence points to a modest prevalence of CH within solid tumors, the impact of these naturally occurring alterations on anti-tumor immunity and therapeutic responses remains less defined. Here, we employed somatic mutation calling algorithms to detect single nucleotide variants (SNVs) in both epithelial and immune compartments within tumors. Our preliminary analyses of single-cell RNA-sequencing (scRNA-seq) of 32 primary, therapy-naïve prostate tumors encompassing 11 patients identified conventional mutations representative of both CH and prostate cancer. Within the tumor-epithelial compartment, we identified recurrent FOXA1 class 1C mutations in 27% (3/11), alongside NCOR1 mutations in 18% (2/11), IDH1 in 9% (1/11), and SPOP in 9% (1/11) of patients. Of note, these SNVs were detected across multiple samples for the same patient. Interestingly, we also detected germline HOXB13 G84E variants, associated with increased risk for prostate cancer, in 2 cases at a high clonality within epithelial cells (>75%). CH alterations were identified in 27% (3/11) of patients based on the detection of exonic, nonsynonymous SNVs within myeloid cells. Specifically, these variants affected SF3B2 (variant allele frequency; VAF = 0.30), UBE2A (VAF = 0.30), and KMT2C (VAF = 0.43) that are predicted deleterious across multiple classifier algorithms. Patients with detectable CH alterations displayed immunologically hotter tumor microenvironments, with CH+ patients having significantly higher proportions of tumor-infiltrating monocytes (16.22% vs 7.13%, Fisher’s exact test p<0.0001), macrophages (6.49% vs 2.88%, p<0.0001), CD8+ T cells (10.35% vs 3.33%, p<0.0001), CD4+ T cells (2.81% vs 0.74%, p<0.0001), and B cells (1.81% vs 0.59%, p<0.0001), and lower fractions of epithelial cells (50.94% vs 69.81%, p<0.0001). In summary, we leveraged de novo mutational algorithms to identify somatic alterations using scRNA-seq. To our knowledge, our study is the first to detect CH alterations in prostate cancer at the single-cell resolution. Further work will integrate our mutational signatures with transcriptomics to elucidate the impact of both cancer- and immune-intrinsic alterations on anti-tumor immunity. Additionally, we will incorporate mitochondrial DNA-based lineage tracing algorithms on matched peripheral blood, primary, and metastatic tumor samples to deconvolute the spatiotemporal regulation of naturally occurring CH alterations across cancer progression. We envision our findings will illuminate novel drivers of immune function and regulation within prostate cancer that may be exploited for future immunomodulatory therapies.



 John R. Lozada, Christine Luo, Devin A. Schmeck, Jeffrey S. Miller, Akash Patnaik, Emmanuel S. Antonarakis, Frank Cichocki, Justin H. Hwang. Interrogating the Genomic Co-Evolution of the Immune System and Prostate Cancer Along the Time and Treatment Continuum [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A040.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03587209005d8227c89ae44e47e8117ddd350a30" target='_blank'>
              Abstract A040: Interrogating the Genomic Co-Evolution of the Immune System and Prostate Cancer Along the Time and Treatment Continuum
              </a>
            </td>
          <td>
            John R Lozada, Christine Luo, Devin A. Schmeck, Jeffrey S. Miller, Akash Patnaik, Emmanuel S. Antonarakis, Frank Cichocki, Justin H. Hwang
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="The conceptualization of mitochondria, previously restricted to their function as cellular 'powerhouses', has evolved to recognize their function as central coordinating hubs for the orchestration of cancer cell metabolism, signaling, and fate determination. Within the context of lung cancer, encompassing both non-small cell lung cancer and small cell lung cancer, these organelles undergo profound functional and structural dysregulation integral to tumor initiation, progression, and most critically, therapeutic resistance. This review presents a synthesis of the burgeoning field of mitochondrial inhibitors as a strategic approach for lung cancer treatment, achieved by synthesizing detailed mechanistic and preclinical data into an evidence-graded framework. This document first provides a delineation of the fundamental dysregulation of mitochondrial functions in lung cancer, inclusive of metabolic reprogramming toward oxidative phosphorylation dependency, particularly in distinct genetic contexts (e.g., LKB1, SWI/SNF-mutant). Subsequent sections systematically categorize and analyze the major classes of mitochondrial inhibitors predicated upon their mechanisms of action, including electron transport chain inhibitors, pro-apoptotic agents (e.g., B-cell lymphoma 2/B-cell lymphoma xL inhibitors), and modulators of metabolism and dynamics. A critical focus is applied to the role of these agents in the supersession of acquired resistance to established therapies, such as epidermal growth factor receptor-tyrosine kinase inhibitors, chemotherapy, and immunotherapy. The translational landscape is consolidated herein by summarizing key clinical trials (including terminations precipitated by toxicity) and distinguishing small cell lung cancer specific vulnerabilities. Finally, the significant challenges of on-target, off-tumor toxicity and the crucial necessity for predictive biomarkers are addressed. Through the synthesis of these disparate fields into a unified, clinically oriented framework, it is posited that targeting mitochondrial vulnerabilities possesses the potential to overcome longstanding therapeutic hurdles in lung cancer. © 2026 The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f7b825109d87a023f4ea1278101efbeafac2b00" target='_blank'>
              Mitochondrial reprogramming in lung cancer: a therapeutic vulnerability and a strategy for reversing drug resistance.
              </a>
            </td>
          <td>
            Woo Hyun Park
          </td>
          <td>2026-02-16</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Pediatric leukemia, primarily comprising acute lymphoblastic leukemia (ALL) and acute myeloid leukemia, represents approximately 30% of childhood cancers worldwide. Over the past decade, advances in molecular and genetic profiling, including next-generation sequencing and minimal residual disease monitoring, have refined risk stratification and enabled individualized treatment strategies. Integrating these approaches into clinical practice has improved survival rates, particularly for high-risk patients, by guiding therapy intensity and informing targeted interventions. Targeted therapies, such as tyrosine kinase inhibitors for Philadelphia chromosome-positive ALL, Chimeric Antigen Receptor T cell therapy for relapsed or refractory cases, and monoclonal antibodies like blinatumomab and inotuzumab ozogamicin, have transformed treatment outcomes while reducing chemotherapy-related toxicity. Despite these advances, challenges remain, including the development of therapy resistance (e.g., BCR-ABL1 and FLT3 mutations) and long-term adverse effects such as cardiotoxicity and secondary malignancies. This narrative review summarizes recent innovations in risk assessment and targeted therapies, highlights current challenges, and discusses future directions to optimize personalized pediatric leukemia care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e88e19df34880ed4268d38202fdb59c43c3fe8e" target='_blank'>
              Redefining pediatric leukemia care – innovations in risk assessment and targeted treatment: a narrative review
              </a>
            </td>
          <td>
            E. Obeagu
          </td>
          <td>2026-01-21</td>
          <td>Annals of Medicine and Surgery</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/423b9ad510934f726aebaeeb465a2ff10426e96b" target='_blank'>
              A new cancer progression model: from synthetic tumors to real data and back
              </a>
            </td>
          <td>
            Daniela Volpatto, S. G. Contaldo, Simone Pernice, Marco Beccuti, Francesca Cordero, R. Sirovich
          </td>
          <td>2026-02-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Ewing sarcoma is a pediatric bone and soft tissue cancer thought to arise from mesenchymal stem cells (MSCs). It is characterized by fusion of the RNA-binding protein EWS to an ETS family transcription factor, most often FLI1. EWS::FLI1 induces changes in gene expression that have been well-studied and have been linked to its ability to bind to and alter chromatin state at GGAA repeat sequences within the genome. In addition to these effects, EWS::FLI1 alters mRNA splicing; however, the mechanism of this activity remains unclear.



 We curated a sarcoma-focused transcriptome from published long-read RNA sequencing of Ewing sarcoma, osteosarcoma, and synovial sarcoma patient tumors and matched adjacent normal tissue (215M reads). We supplemented this with published long-read RNA sequencing data from relevant normal cell types, osteosarcoma cell lines, and Ewing sarcoma lines with or without EWS::FLI1 knockdown (105M reads). We used this transcriptome as a reference to evaluate expression and splicing from short-read RNA-seq of Ewing sarcoma cell lines, MSCs, and MSC-based models of Ewing sarcoma and established Ewing-associated expression and splicing signatures.



 We generated scores based on these signatures in published knockdown-rescue experiments in which endogenous EWS::FLI1 was silenced by shRNA and replaced with exogenous shRNA-resistant mutants. Expression and splicing scores were highly correlated within these datasets (r2 = 0.89), suggesting that similar regions of the fusion protein might be required for both its functions. In contrast, the scores had limited correlation within an internal dataset of RNA-binding protein perturbations (r2 = 0.02). We next examined the effects of pharmacological agents thought to alter EWS::FLI1 activity. Interestingly, we found that lurbinectedin which disrupts EWS::FLI1’s DNA-binding could partially revert both expression and splicing scores. In contrast, YK-4-279 which disrupts the fusion’s protein-protein interactions and MS0621 which reverts a Ewing-associated chromatin signature had limited effects on these scores. To further interrogate the potential role of EWS::FLI1 DNA-binding in splicing regulation, we silenced EWS::FLI1 in TC-32 cells and replaced it with either the wildtype fusion or a DNA-binding null mutant. Unlike the wildtype construct, the DNA-binding null mutant could not rescue expression or splicing. Finally, we examined the effects of the previously published KRAB-ZFA7 repressor which silences chromatin at GGAA repeats while sparing single GGAA sites and found that it could revert both scores.



 Together, these data establish a potential connection between EWS::FLI1 activity at GGAA repeats and its effects on mRNA splicing. Additionally, we find that lurbinectedin, an FDA-approved agent currently in a phase 1/2 trial for Ewing sarcoma, suppresses both the splicing and expression effects of EWS::FLI1. Emerging compounds designed to target EWS::FLI1 DNA-binding directly will likely have similar effects.



 David V. Allegakoen, Jeffrey A. Toretsky. EWS::FLI1 activity at GGAA repeats is essential for maintenance of Ewing-like splicing [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr B013.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6934f37ed326d496d296a4806fccea6f323a4f63" target='_blank'>
              Abstract B013: EWS::FLI1 activity at GGAA repeats is essential for maintenance of Ewing-like splicing
              </a>
            </td>
          <td>
            D. Allegakoen, J. Toretsky
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="


 Oncofusions, cancer-causing chimeric proteins formed by chromosomal rearrangements, are key drivers in approximately 16% of all cancer cases. These fusion proteins are associated with a wide range of malignancies, including varieties of leukemia, lung cancer, and sarcoma. Thousands of unique oncofusions have been identified in recent years, yet most of these fusion proteins lack mechanistic annotation. Many oncofusions dysregulate chromatin architecture in complex ways that are not yet fully elucidated. Investigating these chromatin alterations is both labor-intensive and costly, limiting the feasibility of large-scale studies. This lack of mechanistic insight has hindered the development of targeted therapies, with only a handful of efficacious small-molecule inhibitors directed toward oncofusions currently in clinical trials or approved for patient use. To address this critical gap, we adapted our recently developed PROD-ATAC method into a high-throughput, scalable platform to systematically elucidate the effects of dozens of candidate inhibitory small molecules on the chromatin dysregulation profiles induced by a pooled library of diverse oncofusions. In the PROD-ATAC scheme, a pooled protein-coding variant library is expressed in a single disease-agnostic cell line, and scATAC-seq (single-cell assay for transposase accessible chromatin) catalogs each variant’s effect on the chromatin landscape. We applied PROD-ATAC to simultaneously assay the chromatin regulatory impact of >40 small molecule inhibitors against a panel of tens of oncofusions (including the majority of patient-identified fusion sequences causal for Ewing sarcoma) in a single assay, generating genome-wide chromatin accessibility datasets representative of >1,000 unique drug - fusion interactions. In this manner, we characterized several small molecules as previously unidentified inhibitors of multiple oncofusions. Certain inhibitors were found to abrogate the chromatin dysregulation caused by an oncofusion either globally or in a targeted cell signaling pathway-specific manner. By profiling multiple rare fusions causal for the same cancers, we identified separate instances of homogenous and heterogenous modes of chromatin dysregulation associated with distinct fusion proteins. In this work, PROD-ATAC chromatin accessibility readouts coupled the identification of novel oncofusion inhibitors with the characterization of each inhibitor’s impact on the epigenetic landscape of oncofusion-expressing cells.



 James E. Corban, Max Frenkel, Srivatsan Raman. High-throughput characterization of inhibitor impacts on chromatin dysregulation induced by oncofusions [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr A024.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ff99b547f5b85ca4c657ff5fd20f9c13db3f972" target='_blank'>
              Abstract A024: High-throughput characterization of inhibitor impacts on chromatin dysregulation induced by oncofusions
              </a>
            </td>
          <td>
            James E. Corban, Max Frenkel, Srivatsan Raman
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The advent of next-generation high-throughput sequencing and precision medicine has allowed the identification of recurrent genetic events of vital interest for leukemic cells and for the management of patients with leukemia, since it has a direct impact on prognosis and therapeutic stratification. The precise cytogenetic characterization of these diseases has rapidly enabled the development of innovative therapeutic strategies to improve outcomes and limit toxicities. New targeted therapies, including small molecule inhibitors, monoclonal antibodies, bispecific T cell engagers, antibody–drug conjugates, recombinant immunotoxins, and chimeric antigen receptor T cells (CAR-T cells) represent a broad therapeutic arsenal now available in everyday practice. The growing number of treatments means that many possibilities of drugs and combinations can be used to offer new lines of therapy in cases of refractory disease or relapse, but also in frontline treatment. This review provides a broad overview of pathogenic mechanisms and therapies (including targeted therapies, immunotherapy and RNA-based therapy) and gathers the main research progress in the field of leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9ad84d56b4eca59a338789eb4fc5c1bd3ecb571" target='_blank'>
              Molecularly targeted therapy and immunotherapy in leukemias
              </a>
            </td>
          <td>
            Elie Cousin, Agathe Picard, Maria Oundjian, Nina Tardif, Anaïs M. Quéméner, Anne-Gaëlle Rio, M. Galibert, Virginie Gandemer, Frédéric Mazurier
          </td>
          <td>2026-01-11</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Radiotherapy is a central modality in cancer management, yet intrinsic and acquired radioresistance and dose-limiting normal tissue toxicities continue to constrain durable tumor control. Beyond genetic and transcriptional programs, post-translational modifications (PTMs) provide rapid and reversible regulation that can rewire radiation responses across tumor and stromal compartments. This review synthesizes recent advances defining how major PTM axes, including ADP-ribosylation, ubiquitination and deubiquitination, neddylation, SUMOylation, methylation, acetylation, and lactylation, shape tumor radiosensitivity and radiation-induced injury. We highlight mechanistic links to DNA damage recognition and repair pathway choice, chromatin remodeling, checkpoint control, and cell-fate decisions encompassing apoptosis and ferroptosis. We further discuss how PTM-driven metabolic and redox adaptation influences post-irradiation signaling, and how PTMs modulate anti-tumor immunity by affecting immunogenic cell death, antigen presentation, cytokine networks, and immune checkpoint regulation within the tumor microenvironment. In addition, we summarize emerging evidence for reciprocal crosstalk between PTM enzymes and non-coding RNAs, which can act as upstream regulators, scaffolds, or effectors to reinforce context-specific modification patterns and therapeutic vulnerabilities. Viewing radioresponse through a PTM-network lens reveals actionable nodes for radiosensitization and radioprotection. Targeting PTM writers, erasers, and readers, alone or in rational combinations with radiotherapy, DNA damage response inhibitors, epigenetic agents, and immunotherapy, may overcome resistance while improving the therapeutic window. Future efforts should prioritize context-specific biomarkers, on-target toxicity management, and mechanism-informed trial designs to translate PTM-guided strategies into clinically meaningful gains.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a185e51f6325e6f527f9dd4fd6d6dc6fe55edc76" target='_blank'>
              Post-translational modification networks in tumor radiosensitivity: mechanistic insights and therapeutic opportunities
              </a>
            </td>
          <td>
            Xiaoli Lv, Jiao Xue, Jun Zhou, Qi Zhao, Yandong Liu, Lili Wang, Yang Jiao, Songbing Qin
          </td>
          <td>2026-01-07</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) represent a highly plastic subpopulation within tumors that drives tumor initiation, progression, metastasis, therapeutic resistance, and disease relapse. Although CSC traits were initially attributed to stable genetic hierarchies, growing evidence suggests that cancer stemness is a dynamic and reversible state regulated by precise molecular events at the protein level. This literature review aims to synthesize experimental evidence regarding the impact of single amino acid mutations and alterations in regulatory proteins on the stemness properties of cancer cells. A systematic literature search was conducted using PubMed and Google Scholar to identify original research articles published between 2020 and 2025 that examined protein-level mutations, post-translational modifications, epigenetic regulators, and chromatin remodeling factors in diverse cancer models. The reviewed studies consistently demonstrate that single amino acid substitutions in oncogenic, tumor suppressor, or regulatory proteins can profoundly affect protein conformation, stability, enzymatic activity, and protein–protein interactions. These alterations frequently result in sustained activation of key stemness-associated signaling pathways, including Wnt/β-catenin, Notch, Hedgehog, and STAT3, thereby enhancing self-renewal capacity, stem cell marker expression, tumor-initiating potential, and resistance to conventional therapies. In parallel, dysregulation of post-translational modification systems, such as phosphorylation, ubiquitination, acetylation, and methylation, promotes epigenetic reprogramming and reactivation of embryonic transcriptional programs that favor cellular plasticity. Moreover, protein-level alterations strengthen cancer cell interactions with the tumor microenvironment, increasing responsiveness to hypoxia, inflammatory signals, and metabolic stress. Collectively, these findings highlight protein-level regulation as a central determinant of cancer stem cell maintenance and plasticity, and suggest that effective therapeutic strategies should target aberrant protein stability, post-translational modification machinery, and stemness-related signaling networks to achieve durable clinical responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f961c9b291a85047c8455458a035edcd22afcac" target='_blank'>
              IMPACT OF SINGLE AMINO ACID MUTATIONS OR ALTERATIONS IN REGULATORY PROTEINS ON THE STEMNESS PROPERTIES OF CANCER CELLS
              </a>
            </td>
          <td>
            Dewi Sartika, Irmatul Laili, Sri Utami
          </td>
          <td>2025-12-30</td>
          <td>Journal of Stem Cell Research and Tissue Engineering</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="MicroRNAs (miRNAs) are tiny, non-coding RNA molecules that are pivotal in regulating cell growth, development, cell cycle, and programmed cell death. Alterations in miRNA expression, caused by various mechanisms such as amplification, deletion, mutation, or epigenetic changes, can lead to either an increase or decrease in their levels in cancerous cells. Since their initial discovery in 1993, a wealth of research has established miRNAs as key players in oncogenesis and tumor suppression across various human cancers. The study of miRNAs is critical for predicting cancer outcomes and aiding in its detection. Numerous miRNAs have demonstrated potential as markers for cancer prognosis and diagnosis, as well as targets for therapy. Their profound influence on tumor development and spread is becoming increasingly acknowledged. This article presents a summary of how miRNAs are synthesized and generated, and explores their dual function as tumor suppressors and oncogenes. Additionally, we delve into the promising applications of miRNAs as instruments for cancer diagnosis">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/874dc41eef7a91dbf277f54075fa7796a49bfc9e" target='_blank'>
              MicroRNA Regulation in Cancer: Insights into Diagnosis, Prognosis, and Treatment
              </a>
            </td>
          <td>
            Ibtihaj Alrehaili, Norah Alrashoud, Rawaeb Almutairi, Lama Alqahtani, Nour Alhatm, A. Almutairi, Asma Alghamdi, Azhar M Barnawi, Kharsan Alyami, H. Alghamdi, Jamilah Algasem, Reham Alshahrani, Lubna Aljaddua, M. Alahmari
          </td>
          <td>2026-02-04</td>
          <td>Pharmacy Practice</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a highly aggressive malignancy defined by significant biological diversity and variable patient outcomes. A key subset of AML is driven by abnormalities that lead to the overexpression of the oncogenic transcription factors HOXA9 and MEIS1. These abnormalities include KMT2A (formerly MLL) rearrangements and NPM1 mutations, as well as other rare lesions such as NUP98 rearrangements. This review focuses on the biology of the KMT2A, NPM1, and HOX/MEIS1 pathways, dissecting their molecular mechanisms of leukemogenesis. A central theme is the role of the scaffolding protein menin in the epigenetic regulation of this pathway, which ultimately drives malignant transformation. Currently, the clinical landscape is being transformed by the emergence of menin inhibitors as promising therapeutic agents for AML harboring these specific genetic anomalies. We evaluate the latest data on various menin inhibitors—both as monotherapy and in combinations—emphasizing their efficacy and safety profiles. As new evidence continues to accumulate with recent drug approvals and ongoing randomized, phase 3 studies, menin inhibitors are rapidly becoming a component of the AML treatment paradigm for relapsed/refractory and likely newly diagnosed disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/feba5b2e063ddd9d72c35dda8bb171808e6f0308" target='_blank'>
              Menin Inhibition in Acute Myeloid Leukemia: Pathobiology, Progress and Promise
              </a>
            </td>
          <td>
            Utsav Joshi, R. Shallis
          </td>
          <td>2026-01-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f6c40c7ea1ba98d656117fae44f7bccd796a048" target='_blank'>
              Incomplete developmental silencing of cancer-testis antigen BORIS in humanized mouse model promotes cancer susceptibility
              </a>
            </td>
          <td>
            Emma Price, E. Pugacheva, D. N. Bhatt, Yon Ji, Sheila Yeboah, Arielle Scott, D. Loukinov, Victor V. Lobanenkov
          </td>
          <td>2025-12-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d21384971b0ced62ebc3c05bc7a513b76dd2fb8" target='_blank'>
              Five dominant amino acid substitution signatures shape tumour immunity.
              </a>
            </td>
          <td>
            Szilvia Juhász, B. Papp, Anna Tácia Fülöp, Z. Farkas, Dávid Kókai, Dóra Alexandra Gyémánt, Franciska Tóth, Zsófia Nacsa, Dóra Spekhardt, Balázs Koncz, Peter Burkovics, Csaba Pál, Máté Manczinger
          </td>
          <td>2026-01-28</td>
          <td>Molecular systems biology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Drug resistance remains a significant obstacle in cancer therapy, severely compromising therapeutic efficacy and patient survival outcomes. Targeted therapy is now a central strategy for cancers with defined molecular abnormalities, employing agents that interact with specific genetic or molecular targets. Immunotherapy has emerged as a transformative approach in oncology by enhancing the immune system's ability to identify and eradicate cancer cells. Although microbial communities are increasingly acknowledged for their involvement in tumorigenesis and metastasis, their impact on drug resistance, including resistance to chemotherapy, immunotherapy, and targeted therapy, remains insufficiently investigated. Clinical evidence further reveals significant correlations between microbiota composition and survival outcomes, such as progression-free survival and recurrence rates. Microbiota contribute to resistance through complex mechanisms. Pathogenic bacteria can impair chemotherapy by degrading drugs and suppressing immune responses, whereas certain beneficial bacteria may enhance immunity and modulate metabolism. Notably, intratumoral microbiota directly modulate chemoresistance development through localized actions within the tumor microenvironment. This review synthesizes clinical data and mechanistic studies to delineate microbial-drug interactions, emphasizing how pathogens drive resistance through multifactorial pathways while probiotics mitigate treatment toxicity and restore drug sensitivity. We further propose novel therapeutic strategies leveraging microbe-drug interplay to overcome resistance and alleviate chemotherapy side effects, offering new approaches to enhance the efficacy of immunotherapy and targeted therapy. These findings advance the understanding of microbial contributions to drug resistance and provide new directions for clinical application in precision cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b224c22429ee57791adfb698b794e54c5e3fdbe" target='_blank'>
              The Role of the Microbiota in Cancer Drug Resistance and Emerging Therapeutic Strategies.
              </a>
            </td>
          <td>
            Chang Che, Yelu Zhou, Liyu Chen, Sifan Li, Daiming Fan, Yuanyuan Lu, Xiaodi Zhao
          </td>
          <td>2026-02-07</td>
          <td>Cancer letters</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cancer immunotherapy has markedly improved clinical outcomes for cancer patients. However, its broad application is constrained by low response rates, which limit therapeutic benefits to only a subset of individuals. A deeper understanding of the tumor microenvironment (TME) and the interactions between tumor and immune cells is crucial for overcoming resistance. In this context, the reprogramming of erythroid progenitor cells (EPCs) within the TME has emerged as an important mechanism of immunotherapy resistance. EPCs, a key population in erythroid differentiation, undergo epigenetic reprogramming that underlies various physiological and pathological states. Through epigenetic modifications, EPCs may interact with immune cells and thereby promote tumor immune evasion. This review summarizes EPC reprogramming in the TME from an epigenetic perspective and explores their crosstalk with tumor and immune cells. It also evaluates the therapeutic potential of epigenetic drugs targeting EPCs and discusses future research directions focused on reversing pathological epigenetic reprogramming in EPCs to enhance immunotherapy efficacy. These advances hold significant potential for optimizing clinical cancer care paradigms and improving patient prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e26a816c43fef409538e01b0a9266e0784f7d36" target='_blank'>
              Epigenetic Reprogramming of Erythroid Progenitor Cells: Insights for Enhancing Cancer Immunotherapy
              </a>
            </td>
          <td>
            Zizhan Li, Xuan-Yu Su, Cheng-Ke Zhou, Su-Ran Li, Zhi-jun Sun
          </td>
          <td>2026-01-22</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Abstract Background Chordoma is a rare bone tumor that is considered to have originated from the remnants of notochord by the general consensus in the field. Chordoma patients suffer from local invasion and recurrence of the disease, along with high resistance to conventional therapies. Despite the scarcity of the disease, new biomarkers must be discovered to develop targeted therapeutics. Methods This study aimed at analyzing the whole exome sequencing data of a chordoma cell line, CH22, which held more than 3,000 missense variants in the genome. These missense-carrying genes were investigated in terms of their association with chordoma pathogenesis. Results Out of them, neurotrophic tyrosine kinase receptor type 3 (NTRK3) and nuclear factor erythroid 2-related factor 2 were found to be associated with the altered pathways involved in chordoma. Conclusion Further examination through structural modeling, molecular docking, and phylogenetic analyses related to the missense variants of NTRK3 was conducted in the scope of this study.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b2fe9308814058a063733e1cf9abe4023d9644d" target='_blank'>
              Deciphering the Structural and Molecular Effects of a Novel Genetic Mutation by In Silico Analysis in Chordoma Cell Line
              </a>
            </td>
          <td>
            Muhammed Ümeyr Fettah, Nur Ekimci-Gürcan, Emre Yorulmaz, Batuhan Oğuz, A. Köseoğlu, Furkan Ayaz, Esra Aydemir
          </td>
          <td>2025-09-30</td>
          <td>Journal of Neurological Surgery Part B: Skull Base</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Gastric cancer (GC) persists as a leading cause of global cancer morbidity and mortality, with its pathogenesis intricately linked to epigenetic dysregulation. Emerging research specifies the novelty of these mechanisms—including DNA methylation, histone modifications, non-coding RNAs (ncRNAs), and RNA modifications—in GC initiation, progression, and therapeutic resistance. This review systematically examines key epigenetic mechanisms in GC, dissect the therapeutic implications as diagnostic biomarkers and therapeutic targets. Key insights include (1) aberrant methylation of tumor suppressor genes (e.g., CDH1, RUNX3): in early carcinogenesis; (2) histone lactylation and acetylation modulating immune evasion (3) ncRNAs (e.g., miR-21, HOTAIR); as promising biomarkers; and (4) m6A RNA modification in chemotherapy resistance. We further discuss translational applications of epigenetic biomarkers in liquid biopsies and targeted therapies (e.g., DNMT/HDAC inhibitors). Integrating multi-omics and epigenetic editing technologies may advance precision medicine in GC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b58e571395e3737fdd50502e583400d42b89b97" target='_blank'>
              Advances in epigenetics of gastric cancer
              </a>
            </td>
          <td>
            Rihua Zeng, Jianning Chen
          </td>
          <td>2026-01-28</td>
          <td>Oncology Reviews</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Prostate adenocarcinomas are the most common form of the disease. Targeting the androgen receptor (AR) is the principal treatment strategy. However, resistance is common. Lineage plasticity (LP)—change in differentiation state—is increasingly recognized as a mechanism of resistance to AR-targeting therapies. LP is a continuum ranging from amphicrine to neuroendocrine prostate cancer (NEPC). Despite the increasing incidence of LP due to more widespread use of new and potent AR inhibitors, mechanisms by which LP emerges after therapy have been understudied due to paucity of matched patient tumors. To address this deficit, we performed multi-omic profiling on matched biopsies from 16 patients, including seven patients that underwent LP after treatment.



 We used DNA sequencing to identify mutations and copy number alterations present in tumors at baseline and at progression. We also performed RNA sequencing and transcriptional subtyping of all tumors.



 Analysis of RNA sequencing revealed two distinct subtypes of LP in our cohort—most consistent with amphicrine or NEPC. DNA sequencing identified TP53 and RB1 alteration patterns in baseline tumors that eventually underwent specific LP trajectories. Pathways linked to proliferation and inflammatory signaling were highly activated in baseline tumors that eventually underwent LP.



 Our analysis identified distinct LP trajectories linked with specific genomic and transcriptomic features at baseline. Profiling tumors with this approach may provide clues about which patients may be at greatest risk for undergoing specific LP trajectories after therapy.



 Anbarasu Kumaraswamy, Visweswaran Ravikumar, Ryan J. Rebernick, Eva Rodansky, Amina Tanweer, Joel A. Yates, Aaron M. Udager, Marcin P. Ceislik, Arvind Rao, Zachery R. Reichert, Arul M. Chinnaiyan, Joshi J. Alumkal. Multi-Omic Profiling Clarifies Mechanisms of Therapy-Induced Prostate Cancer Lineage Plasticity [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A036.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d201d414071181c06c4abf5de951c2f23846da37" target='_blank'>
              Abstract A036: Multi-Omic Profiling Clarifies Mechanisms of Therapy-Induced Prostate Cancer Lineage Plasticity
              </a>
            </td>
          <td>
            A. Kumaraswamy, V. Ravikumar, Ryan J. Rebernick, Eva Rodansky, Amina Tanweer, Joel A. Yates, Aaron M. Udager, Marcin P. Ceislik, Arvind Rao, Z. Reichert, A. Chinnaiyan, J. Alumkal
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>171</td>
        </tr>

        <tr id="Gene therapy has evolved into a sophisticated field encompassing diverse precision editing platforms and advanced delivery systems capable of addressing complex genetic disorders and age-related pathologies. This comprehensive review examines the current landscape of gene therapeutic technologies, including CRISPR-based genome editing, base editing systems, prime editing platforms, and emerging DNA polymerase-based editors alongside their corresponding delivery methodologies. The review encompasses viral vectors, including tissue-specific adeno-associated virus serotypes, non-viral delivery systems such as ionizable lipid nanoparticles and virus-like particles, and innovative platforms, including exosome-based delivery and the SEND system. We examine therapeutic applications spanning nuclear genome editing, mitochondrial genome modification, RNA editing, and epigenetic modulation, demonstrating the expanding scope of gene therapy beyond traditional monogenic disorders. Critical analysis reveals that while fundamental technological capabilities have been established, significant challenges remain in manufacturing scalability, long-term safety assessment, delivery across physiological barriers, and optimization of editing efficiency in post-mitotic tissues. The integration of artificial intelligence approaches for predictive analysis and rational vector design represents a promising avenue for addressing current limitations. This review concludes that successful clinical implementation requires systematic resolution of manufacturing, safety, and delivery challenges alongside the development of standardized protocols for patient stratification and robust regulatory frameworks that accommodate rapid technological innovation while ensuring patient safety.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24cc3f28029393deaa5073b92c5af359263d8da2" target='_blank'>
              Gene Therapy Techniques and Delivery Methods (Review)
              </a>
            </td>
          <td>
            E. I. Shchukina, Junior Researcher, PhD I.O. Mazunin, MD I.I. Eremin, DSc A.A. Moskalev
          </td>
          <td>2025-12-29</td>
          <td>Modern Technologies in Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Acute lymphoblastic leukemia (ALL) is a heterogeneous malignancy characterized by various genomic alterations playing a crucial role in disease classification, prognosis, and response to treatment. However, molecular diagnosis and effective management of this hematological malignancy remain a major challenge, particularly in developing countries, including Tunisia. In this study, we aimed to conduct a detailed analysis of copy number alterations (CNAs) associated with ALL in a cohort of 60 primary samples from Tunisian patients. Using multiplex ligation-dependent probe amplification (MLPA), major genetic lesions, including IKZF1, CDKN2A/2B, PAX5, ETV6, BTG1, and genes located in the PAR1 region, were analyzed and their associations with clinical and laboratory features, as well as survival outcomes, were also evaluated. Our analysis revealed that 70% of patients had deletions and/or amplifications in at least one gene. The most frequently observed deletions were in CDKN2A/2B (33.3%, n = 20), IKZF1 (30%, n = 18), and PAX5 genes (25%, n = 15). BTG1 deletions were significantly associated with female gender, IKZF1 deletions were more frequent in adult patients, in those with elevated white blood cell (WBC) counts, and in cases involving the BCR::ABL1 translocation, while duplications of the PAR1 region were significantly associated with hyperdiploïdy. Regarding treatment response, cases of IKZF1 deletions showed a significant association with poor glucocorticoid response (GC) at day 8 of treatment and positive minimal residual disease (MRD) rates at days 33 and 63, particularly in B-ALL cases. Furthermore, patients with IKZF1 deletions were associated with significantly lower survival rates in both univariate and multivariate analyses compared to those without these deletions. Additionally, the integration of IKZF1 deletion status into risk stratification models revealed markedly different survival outcomes, highlighting its potential interest in developing new stratification algorithms. These results underscore the critical importance of molecular profiling, particularly IKZF1 status, for improving outcomes in ALL patients in Tunisia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd7521499fd8c33c0af8b7b43eac8773db943a64" target='_blank'>
              Genomic characterization and prognostic significance of copy number alterations in Tunisian patients with acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Ameni Bedoui, W. Ayadi, Nour Louati, Imen Frikha, Yosra Fakhfakh, Fahmi Smaoui, Ali Gargouri, Ikram Ben Amor, Moez Elloumi, Moez Mdhaffar, Raja Mokdad Gargouri
          </td>
          <td>2026-02-03</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6da8c58eace175b556f1165210d11f554c8a306d" target='_blank'>
              Regulatory Mechanisms of Maternal Imprinting at the Dlk1-Dio3 Domain
              </a>
            </td>
          <td>
            Ariella Weinberg-Shukron, Frances L Dearden, Ana Moreno-Barriga, Lok Ting Nick New, Marco Müller, Rebecca C. Rancourt, C. Edwards, A. Ferguson-Smith
          </td>
          <td>2026-01-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="
 CAR-T cell therapy has achieved remarkable clinical success for the treatment of hematologic malignancies; however, durable and sustained regressions in patients with solid tumors remain uncommon. A major cause of resistance is CAR-T cell dysfunction driven by chronic antigen exposure within the tumor site. To uncover molecular drivers of this dysfunction, we developed an in vivo CRISPR-based screen using a xenograft model that recapitulates tumor escape due to loss of CAR-T cell activity. We hypothesized that targeted ablation of the identified candidate genes would enhance CAR-T cell efficacy against solid tumors. NSG mice bearing ovarian xenograft tumors were treated with a single dose of anti-HER2 CD28-based CAR-T cells. Tumor-infiltrating T cells (TILs) were isolated short after infusion (effective phase) and after therapy failure (dysfunctional phase). Bulk RNA-seq and single-cell RNA-seq analysis were performed on dysfunctional CAR-TILs compared to effective CAR-TILs and preinfusion product. Effective CAR-TILs retained functional activity but were already transcriptionally committed to dysfunction. In contrast, dysfunctional CAR-TILs displayed phenotypic and functional hallmarks of T-cell exhaustion and a transcriptional program resembling dysfunctional TILs from cancer patients. To uncover mediators of CAR-T cell dysfunction we selected 300 significantly upregulated genes in dysfunctional CAR-TILs, generated a CRISPR/Cas9 library, and performed an in vivo loss-of-function screen in mice bearing ovarian tumors. Using the MAGeCK algorithm, we identified 14 significantly enriched candidate genes, 10 of which were upregulated in CAR-T cells at early stages after treatment. Three of the candidates, including ZC3H12C, TG, and ITGB8 were selected for individual validation. Genetic ablation of all validated candidates enhanced CAR-T cell persistence and antitumor efficacy. Single-cell RNA-seq and ATAC-seq confirmed preferential expression of these genes in exhausted T cells in our model. Analysis of previously published single-cell datasets from human TILs validated the relevance of these candidate genes in TILs undergoing exhaustion in cancer patients. Top hit in our screen ZC3H12C (Regnase-3) was further characterized under chronic antigen stimulation. Genetic ablation of ZC3H12C enhanced HER2-CD28z CAR-T cell proliferation, cytotoxicity, and cytokine production in vitro, and improved antitumor responses and persistence in vivo. CAR-TILs from ZC3H12C-KO-treated mice showed superior functionality and reduced expression of exhaustion makers. Comparable results were observed in lung and pancreatic tumor models with Mesothelin-CD28z and CD19-BBz CARs. In conclusion, by using an in vivo custom CRISPR screen, guided by transcriptional data on dysfunctional CAR-TILs, we identified novel and promising gene targets for engineering CAR-T cells with enhanced efficacy against solid tumors.


 Paula Barbao, Alba Rodriguez-Garcia, Ariel Galindo-Albarrán, Marta Giménez-Alejandre, Pablo Clavero, Marta Botas, Teresa Lobo-Jarne, Guillem Colell, Joan Castellsagué, Guim Cascalló, Irene Andreu-Saumell, Marta Soria-Castellano, Berta Marzal, Beatriz Martin-Mur, Anna Esteve-Codina, Alvaro Urbano-Ispizua, Aleix Prat, Luca Gattinoni, Marco Antonio. Mendoza-Parra, Sonia Guedan. In vivo CRISPR-screen identifies determinants of early CAR-T cell dysfunction in solid tumors [abstract]. In: Proceedings of the AACR Immuno-Oncology Conference (AACR IO): Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2026 Feb 18-21; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2026;14(2 Suppl):Abstract nr C006.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b60e4a25763c3d8849767d684b3f256dba2d0698" target='_blank'>
              Abstract C006: In vivo CRISPR-screen identifies determinants of early CAR-T cell dysfunction in solid tumors
              </a>
            </td>
          <td>
            P. Barbao, Alba Rodriguez-Garcia, Ariel Galindo-Albarrán, Marta Giménez-Alejandre, Pablo Clavero, M. Botas, T. Lobo-Jarné, Guillem Colell, Joan Castellsagué, Guim Cascalló, I. Andreu-Saumell, M. Soria-Castellano, B. Marzal, Beatriz Martín-Mur, A. Esteve-Codina, Á. Urbano-Ispizua, A. Prat, Luca Gattinoni, M. Mendoza-Parra, S. Guedan
          </td>
          <td>2026-02-18</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Epigenetic dysregulation is commonly observed in cancer and has been shown to contribute to different steps of carcinogenesis, from initiation, making cells more prone to transformation, to progression and treatment resistance. Therefore, based on their frequency, relevance and reversibility, epigenetic alterations are promising therapeutic targets in oncology. Although the concept of targeting epigenetic mechanisms is not new, recent advances have been made in reducing toxicity, augmenting specificity, diversifying the targets and combining therapies. In this scoping review, we introduce the main epigenetic mechanisms of gene expression regulation and bring the current knowledge on available epidrugs, focusing on their mechanisms of action and ongoing clinical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/501c80d48c489c9e8637f45dfcf252f7149d3908" target='_blank'>
              The power and the promise of epigenetic drugs in oncology
              </a>
            </td>
          <td>
            T. B. Scandolara, Isabelle Vasconcelos Menegoy Siqueira, L. F. Ribeiro Pinto, S. Soares-Lima
          </td>
          <td>2026-01-02</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49ae25d08ee57196b6b04fef8af1744df279043b" target='_blank'>
              Gene therapy for huntington’s disease: advances, challenges, and future perspectives
              </a>
            </td>
          <td>
            Riya Kaushal, M. Yadav, Sourabh Kosey, Madhaw Dwivedi
          </td>
          <td>2026-02-21</td>
          <td>Neurogenetics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 Pediatric acute leukemia is the most common childhood cancer, and acute myeloid leukemia (AML) represents the second most frequent subtype, with overall survival rates of approximately 70% and relapse rates near 30%. The purpose of this study is to define molecular mechanisms essential for leukemogenesis and uncover potential vulnerabilities in a high-risk subtype of pediatric AML driven by the fusion oncoprotein PICALM::MLLT10. Our laboratory recently performed an integrated transcriptomic and genomic analysis of 887 pediatric AML cases, identifying 23 distinct molecular categories that differ in HOXA/HOXB expression, co-occurring mutations, and clinical outcomes. Among these, PICALM::MLLT10 AML is notable for its HOXA expression pattern similar to KMT2A-rearranged cases, yet it forms a distinct cluster with other immature AMLs on a UMAP projection, suggesting a unique transcriptional program. Leveraging genome-wide CRISPR screening data from the Cancer Dependency Map (DepMap) across 64 AML cell lines, we identified the RNA helicase DDX19A, but not its paralog DDX19B, as a selective genetic dependency in PICALM::MLLT10 models (U937, P31/FUJ) with an average Chronos score of -1.753 compared to -0.278 in all other AML models. Using a dTAG-induced degradation system, acute DDX19A depletion significantly impaired proliferation in PICALM::MLLT10-expressing cells and led to increased apoptosis as measured by Annexin V staining, and overexpression of DDX19A rescued the proliferation defect induced by dTAG13-mediated degradation. Domain-specific mutagenesis revealed that the DDX19A helicase and RNA-binding domains are essential for proliferation, while the NUP214-binding domains are dispensable in the PICALM::MLLT10 context. Next, we sought to define the transcriptional consequences of DDX19A loss in PICALM::MLLT10 cells. RNA-seq was performed on U937 cells expressing a FKBP12F36V-DDX19A allele after treatment with either DMSO or dTAG-13 for 24 hours, and pathway enrichment analysis revealed upregulation of cytoplasmic translation in DDX19A depleted cells. To functionally assess this effect, we measured nascent protein synthesis using O-propargyl-puromycin (OPP) incorporation after 24 hours of either DMSO or dTAG-13 treatment and observed a global reduction of protein synthesis in DDX19A depleted cells. These data establish DDX19A as a selective vulnerability and uncover the role of DDX19A in regulating protein synthesis in PICALM::MLLT10 driven pediatric AML. This work highlights a novel mechanistic vulnerability that may be leveraged for the development of targeted therapeutic strategies for this high-risk pediatric AML subtype.



 Lauren J. Ezzell, Ryan Hiltenbrand, Elizabeth Arnold, Michael P. Walsh, Melvin E. Thomas III, Tamara Westover, Masayuki Umeda, Jing Ma, Shondra M. Pruett-Miller, Jeffery M. Klco. Uncovering novel molecular vulnerabilities in PICALM::MLLT10 driven pediatric acute myeloid leukemia [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr LT005.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f08fe0a493aa37ad7908724e233549343842d3e" target='_blank'>
              Abstract LT005: Uncovering novel molecular vulnerabilities in PICALM::MLLT10 driven pediatric acute myeloid leukemia
              </a>
            </td>
          <td>
            Lauren Ezzell, R. Hiltenbrand, Elizabeth D Arnold, Michael P Walsh, Melvin E Thomas, Tamara Westover, Masayuki Umeda, Jing Ma, S. Pruett-Miller, J. Klco
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Despite breakthroughs in molecularly targeted and immune therapies, the prognosis for patients with urinary bladder cancer (UBC) has remained unsatisfactory over the past few decades. Understanding the molecular underpinnings of UBC treatment refractoriness is crucial for identifying novel therapeutic targets and strategies. Cancer stemness plays a pivotal role in the oncogenesis and treatment resistance of UBC, while the underlying molecular regulatory mechanisms are poorly understood. We identified isoform 1 of ASPM (ASPM-i1) as the most upregulated stemness-associated factor in tumorigenic UBC cells, predominantly expressed by ALDH1+ stem-like cancer cells. Pairing genetic ASPM-i1 inhibition with standard chemotherapeutic agents used in the treatment of UBC, including cisplatin and gemcitabine, circumvents the treatment resistance of tumorigenic and stem-like UBC cells. Mechanistically, ASPM-i1 interacts with the Disheveled (DVL) and intracellular NOTCH proteins, thereby attenuating CUL3- or FBXW7-mediated ubiquitination and the subsequent proteasomal degradation. The regulatory module concomitantly enhances the activities and ligand responsiveness of the Wnt and Notch signaling pathways in UBC cells. As a result, ASPM-i1 inhibition sensitized tumorigenic UBC cells to chemotherapy in a NOTCH- and DVL-dependent manner. In human UBC tissues, ASPM-i1 shows substantial cell-to-cell heterogeneity and is upregulated in a subset (46.4%) of tumors, correlating with poor clinical prognosis. This study reveals a crucial co-regulatory module of Notch and Wnt signaling that mediates stemness and chemotherapy resistance in tumorigenic UBC cells; its inhibition provides a novel approach to enhance chemosensitivity and improve therapeutic outcome in human UBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/076780e38121b3f25a600b0e4d535709ef262847" target='_blank'>
              Chemotherapy resistance of urinary bladder cancer mediated by a Notch and Wnt co-regulatory module of stemness.
              </a>
            </td>
          <td>
            Lin-Hsin Cheng, Che-Wei Hsu, Pei-Ming Yang, Tai-Yan Liao, Fang-Chi Liu, Wan Chen, Chin-Chen Pan, Chung-Chi Hsu, Kuan-Chou Chen, K. K. Tsai
          </td>
          <td>2026-01-10</td>
          <td>Stem cells</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Metastasis is still the leading cause of cancer-related death. It happens when disseminated tumor cells (DTCs) successfully navigate a series of steps and adapt to the unique conditions of distant organs. In this review, key molecular and immune mechanisms that shape metastatic spread, long-term survival, and eventual outgrowth are examined, with a focus on how tumor-intrinsic programs interact with extracellular matrix (ECM) remodeling, angiogenesis, and immune regulation. Gene networks that sustain tumor-cell plasticity and invasion are described, including EMT-linked transcription factors such as SNAIL and TWIST, as well as broader transcriptional regulators like SP1. Also, how epigenetic mechanisms, such as EZH2 activity, DNA methylation, chromatin remodeling, and noncoding RNAs, lock in pro-metastatic states and support adaptation under therapeutic pressure. Finally, proteases and matrix-modifying enzymes that physically and biochemically reshape tissues, including MMPs, uPA, cathepsins, LOX/LOXL2, and heparinase, are discussed for their roles in releasing stored growth signals and building permissive niches that enable seeding and colonization. In parallel, immune-evasion strategies that protect circulating and newly seeded tumor cells are discussed, including platelet-mediated shielding, suppressive myeloid populations, checkpoint signaling, and stromal barriers that exclude effector lymphocytes. A major focus is metastatic dormancy, cellular, angiogenic, and immune-mediated, framed as a reversible survival state regulated by stress signaling, adhesion cues, metabolic rewiring, and niche constraints, and as a key determinant of late relapse. Tumor-specific metastatic programs across mesenchymal malignancies (osteosarcoma, chondrosarcoma, and liposarcoma) and selected high-burden cancers (melanoma, hepatocellular carcinoma, glioblastoma, and breast cancer) are highlighted, emphasizing shared principles and divergent organotropisms. Emerging therapeutic strategies that target both the “seed” and the “soil” are also discussed, including immunotherapy combinations, stromal/ECM normalization, chemokine-axis inhibition, epigenetic reprogramming, and liquid-biopsy-enabled minimal residual disease monitoring, to prevent reactivation and improve durable control of metastatic disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a5ad40c8765430fb4e35e8df4878d707b5a6318" target='_blank'>
              Molecular and Immune Mechanisms Governing Cancer Metastasis, Including Dormancy, Microenvironmental Niches, and Tumor-Specific Programs
              </a>
            </td>
          <td>
            Dae Joong Kim
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97e8ef7d28e9d10b44890e1b7be7668548430edd" target='_blank'>
              CHEK2 loss endows chemotherapy resistance to hematopoietic stem cells.
              </a>
            </td>
          <td>
            Jing Zhou, Tianyuan Hu, Dian Li, Sanming Li, Minhua Li, Xiangguo Shi, Daisuke Nakada
          </td>
          <td>2026-01-12</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a3815eb1f32d503a09f41071a0e393a17ec3962" target='_blank'>
              Single-cell epigenetic profiling reveals a tumor-intrinsic interferon response program in ccRCC tied to poor prognosis and BAP1 loss.
              </a>
            </td>
          <td>
            S. Camp, Meng Xiao He, Michael S. Cuoco, Amanda E Garza, Sherin Xirenayi, Z. Bakouny, Eddy Saad, J. El Masri, Erica M Pimenta, K. Meli, C. Labaki, Breanna M. Titchen, Yun Jee Kang, J. Horst, Rachel Trowbridge, E. Shannon, K. Helvie, Aaron R Thorner, S. Vigneau, Angie Mayorga, Jahnavi Kodali, Hannah Lachmayr, Meredith Bemus, Pengsheng Chen, Haiteng Deng, Jihye Park, T. Choueiri, Kevin Bi, E. V. Van Allen
          </td>
          <td>2026-02-20</td>
          <td>Science advances</td>
          <td>0</td>
          <td>80</td>
        </tr>

        <tr id="
 T cell fitness has emerged as a critical determinant of the efficacy and persistence of Chimeric Antigen Receptor (CAR)-T cell therapy. Defined by the capacity of T cells to proliferate, resist exhaustion, persist
 in vivo
 , and exert sustained effector functions, T cell fitness reflects the integration of a dynamic network of intrinsic and extrinsic regulatory mechanisms. In this review, we present a comprehensive synthesis of the molecular and cellular foundations underlying T cell fitness, emphasizing the influence of differentiation trajectories, signaling pathways, metabolic reprogramming, and epigenetic modifications. We further discuss the impact of patient-specific conditions such as age as well as disease biology, prior therapeutic exposures, and timing and quality of T cell collection, on the phenotypic and functional efficacy of CAR-T cell products. Beyond delineating these determinants, we highlight emerging strategies aimed at enhancing T cell fitness. Importantly, we propose T cell fitness as an integrated, multi-layered systems property emerging from the interaction between differentiation state, signaling architecture, metabolic–mitochondrial competence, epigenetic stability, and host-specific inflammatory and treatment-related pressures. We introduce a mechanistic framework that links these layers across the CAR-T therapeutic timeline from leukapheresis to post-infusion tumor engagement and outline how this framework can be operationalized into measurable parameters to guide patient stratification, manufacturing decisions, and rational therapeutic interventions.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02dc66d310e4fd60eb3b716a1b0bfa437de4e16f" target='_blank'>
              Mechanistic basis and therapeutic modulation of T cell fitness to enhance CAR-T cell efficacy in hematological malignancies
              </a>
            </td>
          <td>
            Karama Makni-Maalej, Shaykhah Alotaibi, Queenie Fernandes, S. Ahmed, Sarra Mestiri, S. Bougarn, Waad Amir, S. Farhatullah, M. Kharfan-Dabaja, M. Merhi, R. Fakih, M. Aljurf, S. Dermime
          </td>
          <td>2026-02-18</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Wellcome Trust scientists have shown that “mutational signatures” in specific nucleotide contexts accumulate in genomes of mammalian tissues, providing clues to underlying causes of specific signatures. Analysis of cancer genomes has identified more than 50 single‐base substitution (SBS) signatures, with SBS1, SBS5, and SBS40 linked to aging and present in normal tissues. SBS1 results from cytosine demethylation, whereas SBS5 and SBS40 arise from unknown endogenous mechanisms. We hypothesized that loss of fragile‐site genes drives these two signatures. FHIT, located at FRA3B, is frequently deleted in cancers, and Fhit‐deficient mouse tissues exhibit a mutation profile resembling human SBS5. Data mining of cancer genome sequences showed that FHIT/Fra3B was the gene loss most significantly correlated with human SBS5 mutations, likely representing the endogenous molecular process determining SBS5 mutations in soma, germ cells, and cancers, and inversely associated with lifespan in mammals. Exome sequences of Fhit knockout tissues revealed ~1000 SBS1 and ~4000 SBS5 mutations per sample, compared to markedly fewer in wild‐type controls. Using SigProfilerAssignment, similar analyses performed with Fhit ko samples where Wwox is also frequently lost identified SBS40c. These findings implicate fragile‐site gene inactivation as a major source of clock‐like mutational signatures, contributing to lifelong mutation accumulation and potentially influencing aging, evolution, and speciation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b576f2a56d83e49e328a800a1ee42f5ac37825f" target='_blank'>
              Endogenous Processes Underlying Clock‐Like Mutational Signatures
              </a>
            </td>
          <td>
            T. Druck, Rami I. Aqeilan, C. Marcelo Aldaz, N. Zanesi, Kay Huebner
          </td>
          <td>2026-01-01</td>
          <td>Genes, Chromosomes & Cancer</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/683398225f12a992fd68b945ba84e877fc29b6de" target='_blank'>
              Interpreting cancer genetics through a two-step "evolutionary cascade hypothesis": bridging neutral and selective perspectives.
              </a>
            </td>
          <td>
            A. Ottaiano, M. Santorsola, Francesco Sabbatino, Roberto Sirica, Francesco Caraglia, Anna Ceccarelli, Vincenza Granata, Ines Simeone, S. Zappavigna, M. Berretta, Giovanni Savarese, Michele Caraglia
          </td>
          <td>2026-02-13</td>
          <td>Journal of translational medicine</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Chromosomal instability (CIN), impaired telomere biology, and aberrant DNA methylation are implicated in colorectal cancer (CRC) development. Tracking these alterations from precancerous lesions through tumours to metastases may reveal biomarkers of CRC initiation and progression. Tissue samples from 44 patients with either high-grade colorectal dysplasia (HGA; n = 13) or advanced metastatic CRC (n = 31) were analyzed. CIN was assessed in all patients using either low-coverage whole-exome sequencing or microarray-based comparative genomic hybridization. In a subset of patients, genome-wide CpG methylation profiling (n = 19) and telomere length measurements (n = 15) were performed. CIN was detected in 85% of HGA patients, spanning focal CNVs in MALAT1 (46%) to recurrent alterations on chromosomes 11, 13, and 20, with PTK6 being the most frequently amplified (61%). CIN was comparable between primary tumours and synchronous metastases but was significantly elevated in metachronous cases. DEK was amplified in all metastases but the aberration was absent in primaries, irrespective of tissue chronicity. Methylation profiling distinguished HGA from adjacent non-dysplastic mucosa (9,859 differentially methylated CpGs) and unrelated tumour tissues (17,638 CpGs), whereas primary tumours and metastases differed at only five CpG sites. Both primary tumours and metastases appeared epigenetically younger than colonic mucosa. Metastases exhibited significantly shorter telomeres than both primary tumours (p = 0.019) and colonic mucosa (p = 0.001). The amplification of PTK6 may serve as an early biomarker detectable at the HGA stage, while DEK amplification appears crucial for metastatic progression and may represent a therapeutic target. Further validation is needed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eac4e7ee54e068b1f7e50d381ee4cd9677cd3421" target='_blank'>
              Patterns of Chromosomal Instability and Epigenetic Alterations in Colorectal Cancer Progression: From High-Grade Dysplasia to Liver Metastases.
              </a>
            </td>
          <td>
            Kateřina Šašková, M. Landfors, V. Hlaváč, K. Šeborová, Jan Brůha, Katerina Honkova, K. Tomášová, Saba Selvi, Jana Drabova, Fernanda S Hackenhaar, Torbjörn K Nilsson, P. Vodička, Anna Valíčková, Václav Liška, Sanja A. Farkas, S. Degerman, Michal Kroupa
          </td>
          <td>2026-02-13</td>
          <td>Mutagenesis</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e961f589165d98aa8aff64f0539527c2012d419" target='_blank'>
              Pooled single-cell screen in colorectal cancer defines transcriptional modules linked to oncogenes.
              </a>
            </td>
          <td>
            Viola Hollek, Francisca Böhning, Catalina Florez Vargas, A. Sieber, M. Morkel, Nils Blüthgen
          </td>
          <td>2026-01-19</td>
          <td>Molecular systems biology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Oral squamous cell carcinoma (OSCC) remains a highly prevalent and aggressive malignancy with limited improvements in survival rates. One of the major obstacles to successful treatment is the development of chemoresistance, which contributes to recurrence, metastasis, and treatment failure. This narrative review aims to integrate current evidence on the molecular and cellular mechanisms that drive chemoresistance in OSCC and to delineate how these processes converge under therapeutic pressure. A structured search was performed to identify relevant studies addressing chemoresistance in OSCC, focusing on preclinical and translational evidence. Multiple interconnected mechanisms have been implicated in driving resistance in OSCC, including epigenetic alterations, deregulated signaling pathways, cancer stem cell plasticity, epithelial–mesenchymal transition (EMT), interactions with the tumor microenvironment (TME), drug efflux mediated by ATP-binding cassette (ABC) transporters, and enhanced DNA damage response. In combination, these mechanisms support tumor persistence and limit effective antitumor immunity. Emerging strategies such as epigenetic modulators, signaling pathway inhibitors, immunomodulation, and nanomedicine-based delivery systems have shown promising results in preclinical models. By highlighting convergent resistance networks, this integrative perspective supports the rational design of combination therapies and biomarker-guided strategies aimed at overcoming chemoresistance in OSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25af512c23cbe217030a9a034876fead83f7c218" target='_blank'>
              Molecular Mechanisms of Chemoresistance in Oral Squamous Cell Carcinoma: A Narrative Review with Present and Future Perspectives
              </a>
            </td>
          <td>
            E. Morais, L. Q. R. de Oliveira, Cintia Eliza Marques, H. Morais, Deborah Gondim Lambert Moreira, L. Albuquerque, J. R. V. Silva, R. Freitas, Ricardo Della Coletta
          </td>
          <td>2026-01-05</td>
          <td>Applied Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22b544cab2c7f5ac6e8301325b2bf082f8d3ca21" target='_blank'>
              TRMT112 drives a tumor growth and metastasis-promoting program in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Amr R. Elhamamsy, Brandon J. Metge, M. H. Elbahoty, Bhavya Papineni, Heba M. Alsheikh, Dongquan Chen, Rajeev S. Samant, L. Shevde
          </td>
          <td>2026-01-08</td>
          <td>Cell death and differentiation</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3333bb3dc7aa46ec7dea00023d761c89be2478c2" target='_blank'>
              Large-scale single-cell analysis and in silico perturbation reveal dynamic evolution of HCC: from initiation to therapeutic targeting.
              </a>
            </td>
          <td>
            Peng Xia, Shuang Si, Daosen Fu, Luyao Liu, Dongliang Yang, Yanan Guo, Yixiao Tian, Pengfei Ji, Xinyi Yuan, Yingxia Tian, Rong Shen, Degui Wang
          </td>
          <td>2026-01-30</td>
          <td>NPJ precision oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 Colorectal cancer (CRC) progression is influenced by genetic and epigenetic aberrations. Oncogenesis of CRC involves the accumulation of mutations in proteins involved in the regulation of cell proliferation, growth and death (Graphical abstract A). DNA methylation has been demonstrated to contribute to tumor initiation, progression, and modulation of therapeutic responses. In this particular landscape, the 9p21.3 locus has been observed to integrate various cellular processes, including cell cycle control (
 CDKN2A/CDKN2B
 and ANRIL), immune signaling (cluster of
 type I interferons
 ), and metabolic regulation (
 MTAP, MLLT3
 ). This creates relationships that may affect tumor intrinsic and extrinsic features, immunogenicity, and therapeutic sensitivity. The objective of our analysis is to provide a comprehensive overview of the role of the 9p21.3 locus in CRC, focusing on its potential implications for treatment decisions and prediction of treatment responses. Analyzing the 9p21.3 status would help stratify CRC patients into different groups and guide the choice of personalized therapy for CRC. It could also enhance current CRC treatment by pretreating patients with demethylating agents and using an immunotherapeutic approach in combination with senolytic drugs (Graphical abstract B).
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8172bc565b14fef8077ee2670d74b1f9d60d35ea" target='_blank'>
              Senescence-associated and immune-related 9p21.3 locus genes in colorectal cancer: epigenetic architecture, molecular landscape and therapeutic possibilities
              </a>
            </td>
          <td>
            D. A. Lisitsa, V. V. Shindyapin, Artem R. Nurislamov, O. N. Demidov, D. А. Bogdanova
          </td>
          <td>2026-02-11</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Prostate cancer depends on androgen receptor (AR) signaling for growth, which is why androgen deprivation (castration) therapy is effective at early stages. However, many tumors eventually progress to a lethal form known as castration-resistant prostate cancer (CRPC). A subset of CRPC tumors bypass dependency on AR signaling by acquiring lineage plasticity, where prostate cancer cells transdifferentiate into alternate cellular states through epigenetic reprogramming. Neuroendocrine (NE) prostate cancer represents one well-known lineage plasticity phenotype. Nevertheless, most AR-independent tumors do not exhibit NE features and are defined as AR-negative/NE-negative or “double-negative prostate cancer” (DNPC). In a collaboration with Dr. Ekta Khurana’s computational genomics lab at Weill Cornell Medicine, we recently classified CRPC into four epigenetic subtypes, including the well-established 1) AR and 2) NE, as well as the novel DNPC subgroups 3) WNT and 4) stem cell-like (SCL) (PMID: 35617398). We focused on the SCL subtype as it is the second most common group in CRPC patients and lacks therapeutic targets. Using functional genomic approaches, we found that YAP/TAZ/TEAD cooperates with FOSL1 to drive the SCL lineage and growth of SCL models. We therefore hypothesize that the heightened dependency on the YAP/TAZ/TEAD/FOSL1 transcriptional program represents a therapeutic vulnerability in CRPC-SCL. To test this, we exposed CRPC models to TEAD inhibitors and found robust growth suppression in SCL cells compared to non-SCL cells in vitro. To evaluate whether the TEAD inhibitors are on-target, we performed transcriptomic profiling in SCL models and observed downregulation of YAP/TAZ gene signature as well as FOSL1 expression, which phenocopies the effects of YAP/TAZ double knockdown. To define the cistromes of these factors upon TEAD inhibition, we performed ChIP-seq and observed reduced co-occupancy at consensus sites, suggesting the disruption of the YAP/TAZ/TEAD/FOSL1 transcriptional circuit by the small molecule compound. To determine whether these phenotypes are recapitulated in vivo, we will treat mice harboring CRPC-SCL xenografts with TEAD inhibitors to assess growth response and evaluate epigenetic and transcriptional response using single-nucleus Multiome (ATAC+RNA). These studies will establish whether small molecule inhibition of TEAD is a promising strategy for the treatment of CRPC-SCL and allow high-resolution analysis of cell state transitions, with a focus on loss of SCL-specific signatures and potential emergence of AR/NE programs as adaptive resistance mechanisms.



 Chen Khuan Wong, Dan Li, Hongsu Wang, Marjorie Roskes, Weiling Li, Shipra Shukla, Dana Schoeps, Ekta Khurana, Yu Chen. Defining and targeting drivers of lineage plasticity in stem cell-like prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A074.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/912fdd784be0fd097e91b40d9e4c6933dd2c1b14" target='_blank'>
              Abstract A074: Defining and targeting drivers of lineage plasticity in stem cell-like prostate cancer
              </a>
            </td>
          <td>
            C. K. Wong, Dan Li, Hongsu Wang, M. Roskes, Weiling Li, Shipra Shukla, Dana M. Schoeps, E. Khurana, Yu Chen
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Somatic mosaicism is pervasively observed in human aging, with clonal expansions of cells harboring mutations in recurrently mutated driver genes. Bulk sequencing of tissues captures mutation frequencies, but cannot reconstruct clonal architectures nor delineate how driver mutations impact cellular phenotypes. We developed single-cell Genotype-to-Phenotype sequencing (scG2P) for high-throughput, highly-multiplexed, joint capture of genotyping of mutation hotspots and mRNA markers. We applied scG2P to aged esophagus samples from six individuals and observed large numbers of clones with a single driver event, accompanied by rare clones with two driver mutations. NOTCH1 mutants dominate the clonal landscape and are linked to stunted epithelial differentiation, while TP53 mutants promote clonal expansion through both differentiation biases and increased cell cycling. Thus, joint single-cell highly multiplexed capture of somatic mutations and mRNA transcripts enables high resolution reconstruction of clonal architecture and associated phenotypes in solid tissue somatic mosaicism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e453d2ebb8e890d0f12017626a25469ee8fcb34" target='_blank'>
              Genotype-to-phenotype mapping of somatic clonal mosaicism via single-cell co-capture of DNA mutations and mRNA transcripts.
              </a>
            </td>
          <td>
            Dennis J. Yuan, John Zinno, Theo Botella, Dalia Dhingra, Shu Wang, Allegra G Hawkins, Ariel D Swett, Jesus Sotelo, Ramya Raviram, Clayton Hughes, Catherine Potenski, Katharine D Godfrey, Kara M Ainsworth, Shuzhen Xu, Jianwen Que, Julian A Abrams, Akira Yokoyama, N. Kakiuchi, Seishi Ogawa, D. Landau
          </td>
          <td>2025-12-31</td>
          <td>Cancer discovery</td>
          <td>1</td>
          <td>31</td>
        </tr>

        <tr id="RNA‐targeted therapy is reshaping molecular medicine by shifting the traditional “protein‐centric” view toward an “RNA‐regulatory network” paradigm. Beyond carrying genetic information, RNA plays essential roles in posttranscriptional regulation, signaling pathways, and epigenetic modulation. Advances in high‐throughput sequencing, structural biology, and delivery technologies have accelerated the development of diverse RNA therapeutics, including antisense oligonucleotides (ASOs), small interfering RNA (siRNA), microRNA (miRNA) modulators, messenger RNA (mRNA) therapeutics, aptamers, short hairpin RNA, and CRISPR/Cas‐guided single‐guide RNAs. However, a concise comparison of these major RNA modalities and the translational barriers that limit their broader clinical application is still lacking. This review outlines the mechanisms and representative applications of these RNA‐based strategies in gene silencing, editing, protein replacement, immune activation, and targeted drug delivery. Special emphasis is placed on ASOs and siRNAs for neurological, metabolic, and infectious diseases, as well as mRNA therapeutics that are transforming vaccine development. Common challenges‐such as in vivo stability, delivery efficiency, and immune activation‐are also discussed. Finally, we highlight how chemical modification, nanotechnology, and artificial intelligence‐assisted design are enhancing the specificity, stability, and safety of RNA therapeutics, providing a framework for advancing next‐generation precision RNA medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a8a098f4a76e1439134aee84974e5e6a557a0fc" target='_blank'>
              Progress in RNA‐Targeted Therapeutics for Human Diseases
              </a>
            </td>
          <td>
            Wangzheqi Zhang, Aimin Jiang, Bin Jia, Yuming Jin, Yinghu Chen, Zhaoyu Li, Yan Liao, Haoling Zhang, Zhiheng Lin, Xiao Fang, Linhui Wang
          </td>
          <td>2026-02-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18e77f40b4649c94b742354f1776a0273d7ae6d5" target='_blank'>
              A Hallmark-Integrated, Agent-Based Framework for Intratumor Heterogeneity in Melanoma Evolution
              </a>
            </td>
          <td>
            Khola Jamshad, Trachette L. Jackson
          </td>
          <td>2026-01-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The effects of cis-regulatory alterations in prostate cancer (PCa) are insufficiently characterized, presenting an opportunity to discover driver genes and therapeutic targets. To comprehensively study these effects, we identify genes undergoing allele-specific expression (ASE) in localized PCa and metastatic castration-resistant PCa (mCRPC) samples. By defining recurrent ASE events across prostate tissue and tumor-enriched ASE, we develop CASEDI, a computational framework for prioritizing cancer drivers by integrating ASE and clinical data. CASEDI reveals genes showing recurrent ASE and altered expression in PCa, including AR-regulated and oncogenic ACSM1. mCRPC samples show enrichment of ASE in DNA repair, resistance pathways, and oncogenes and increased frequency of monoallelic expression (MAE) compared to localized tumors. We define an mCRPC gene signature based on MAE status that identifies a subgroup of localized patients with worse prognosis. Using ASE analysis, we expand the landscape of cis-regulatory events in PCa to inform the identification of additional therapeutic targets. Implications: This study develops a framework for identifying cancer drivers using prostate cancer allele-specific expression (ASE) data, generates a comprehensive dataset of ASE in prostate cancer and highlights candidate targets in tumorigenesis and metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/616d957dd08b5a8335edc5a1298ef89197981ba1" target='_blank'>
              Landscape of allele-specific expression in prostate cancer reveals recurrent, stage-specific events in AR signaling and resistance pathways.
              </a>
            </td>
          <td>
            Maggie Tsui, Kevin Hu, Hanbing Song, Sarah C Hsu, Yih-An Chen, Lorraine Nuniz, Julia H Pham, Chih-Hao Chang, Keliana S F Hui, David A. Quigley, Jingjing Li, Franklin W. Huang
          </td>
          <td>2026-01-16</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9627eeea0e72cf88ceb4ce602d15c8920df0c909" target='_blank'>
              Splicing-driven post-translational dysregulation: a new frontier for precision cancer medicine and immunotherapy.
              </a>
            </td>
          <td>
            Sael Alatawi
          </td>
          <td>2026-01-24</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1d108778b128ae240ef789dd566dc8ad0a8e1f3" target='_blank'>
              Characterization and clinical implications of CpG island methylator phenotypes of resistant tumors.
              </a>
            </td>
          <td>
            Fei Hou, Xu Zhou, Yu-E Huang, Haizhou Liu, Mengqin Yuan, Jiahao Chen, Quan Wang, Wei Jiang
          </td>
          <td>2026-01-30</td>
          <td>Clinical epigenetics</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Methyl-donor nutrients, including folate, vitamin B12, vitamin B6, choline, betaine, and methionine, play indispensable roles in one-carbon metabolism and govern key processes such as DNA methylation, nucleotide synthesis, and genomic maintenance. Yet despite decades of research, their relationship with cancer remains paradoxical and frequently misunderstood. Much of the confusion arises from an overreliance on epidemiological studies that use cancer incidence as a late-stage endpoint, thereby obscuring how the biological actions of methyl donors differ fundamentally across the continuum from precancerous lesions to established tumors. By synthesizing evidence from mechanistic studies, precancerous lesion research, and early-stage carcinogenic models, this review suggests that adequate methyl-donor availability may be protective during the earliest phases of cancer development. However, these same nutrients may later become substrates hijacked by neoplastic cells to fuel rapid proliferation, maintain oncogenic methylation programs, and enhance tumor progression in established malignancies and high-risk populations. Therefore, this review proposes a reframing that methyl donors may not be evaluated merely as protective or harmful, but rather as context-dependent modifiers whose influence is shaped by timing, metabolic status, and the underlying biology of the target tissue. Such a shift is promising for advancing precision nutrition and the prevention or targeted suppression of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdf401262f6b61b60e458c86d532862cdfdc9412" target='_blank'>
              The Double-Edged Nature of Methyl Donors in Cancer Development from Prevention to Progression
              </a>
            </td>
          <td>
            Da Pan, Shaokang Wang, Guiju Sun
          </td>
          <td>2025-12-28</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="BACKGROUND
Mutant IDH1 (mIDH1) defines a therapeutically-targetable subtype of intrahepatic cholangiocarcinoma (ICC), with the mIDH1 inhibitor ivosidenib approved for advanced disease. A subset of patients experiences prolonged disease stabilization, although the molecular basis for eventual progression remains poorly defined.


EXPERIMENTAL DESIGN
We performed molecular profiling of matched baseline and post-progression circulating tumor DNA (ctDNA) samples from patients with mIDH1 ICC enrolled in the ClarIDHy phase III trial. Functional studies were conducted to characterize candidate resistance mechanisms.


RESULTS
Longitudinal ctDNA analysis of 18 patients treated with ivosidenib for >6 months revealed emergent genomic alterations in multiple cases. Acquired mutations in MAPK-pathway genes (KRAS, NRAS, MAP2K1, NF1) were identified in five cases, with instances of concurrent alterations and/or high variant allele fractions (VAF). Additional candidate resistance events included a secondary IDH1 mutation and a hotspot IDH2 mutation, detected at low VAF in the same patient. Functional studies demonstrated that these IDH mutations conferred sustained 2-hydroxyglutarate (2HG) production and ivosidenib resistance, whereas MAPK activation blunted gene expression induced by ivosidenib plus interferon-γ, a key therapeutic output of mIDH1 inhibition. In parallel, baseline ctDNA profiling of 81 patients linked ARID1A mutations and elevated mIDH1 VAF to reduced clinical benefit.


CONCLUSIONS
MAPK-pathway alterations represent a recurrent mechanism of resistance to mIDH1 inhibition in ICC, while emergent IDH1/IDH2 mutations appear infrequent. Functional data suggest that MAPK-mediated resistance may involve impaired IFN signaling. These results support MAPK-directed combination strategies and highlight the utility of ctDNA profiling to identify predictive and resistance biomarkers in mIDH1-driven ICC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4aefbd8f840b00705379daf10a2c077d305123a7" target='_blank'>
              MAPK Pathway Mutations Emerge in Mutant-IDH1 Inhibitor-Resistant Cholangiocarcinoma and Attenuate the Interferon Response.
              </a>
            </td>
          <td>
            J. Wan, H. Saatcioglu, H. Ellis, E. Aguado-Fraile, Qin Xu, Hiroshi Kondo, R. Manguso, I. Gritti, B. Norden, Carolina Noble, R. Corcoran, Sam J Lubner, M. Javle, G. Abou-Alfa, James M Cleary, R. Kelley, M. Borad, T. Macarulla, Do-Youn Oh, Scott Daigle, C. Gliser, Susan Pandya, Sung Choe, A. Tron, N. Bardeesy
          </td>
          <td>2025-12-29</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>105</td>
        </tr>

        <tr id="Sickle cell anemia (SCA) is a monogenic inherited disease caused by a single point mutation in the HBB gene. Traditional treatments are ineffective in fundamentally correcting the genetic defect. In recent years, CRISPR-Cas9, an emerging gene editing technology, has shown great potential in the treatment of SCA. This article systematically reviews the mechanism of action of CRISPR-Cas9 and compares it with other editing tools. We focus on its application in SCA treatment, including strategies such as repairing the HBB gene mutation or targeting the HBG1/2 promoter to activate fetal hemoglobin. Furthermore, based on data from cell-based experiments, animal models, and clinical trials, we discuss the efficacy and limitations of representative therapies such as CTX001, Casgevy, and Lyfgenia. Results have shown that CRISPR-Cas9 can significantly improve red blood cell function in patients, and some therapies have received clinical approval, marking a significant breakthrough in its transition from laboratory research to clinical application. However, challenges remain, such as off-target effects, insufficient delivery efficiency, high treatment costs, and transplant-related complications, which need to be addressed. In the future, emerging strategies such as RNA editing are expected to expand into broader areas, including tumor treatment, while improving safety, providing safer and more accessible treatment options for monogenic diseases such as SCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/732f63e17e30e7bab825eb7a3cacdb558c0d1f25" target='_blank'>
              CRISPR-Cas9 Technology in the Treatment of Sickle Cell Anemia (SCA)
              </a>
            </td>
          <td>
            Zihao Wang
          </td>
          <td>2025-12-28</td>
          <td>Academic Journal of Science and Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 The progression of prostate cancer (PCa) to metastatic disease remains a critical clinical challenge, making it essential to gain a detailed understanding of the factors that enable certain subclones to invade and spread. Building on our previous study, which examined genomic variations in benign and malignant prostate tissues by inferring copy number alterations (CNA) from spatial transcriptomics data1, this study aims to trace the spatial evolutionary trajectories of PCa subclones from the primary tumor to matched lymph node metastases and to identify alterations in associated microenvironments and early diagnostic markers. To achieve this, we performed spatial transcriptomics on entire prostate axial disks and patient-matched lymph node metastases from 10 individuals. Standard spatial transcriptomics (55 µm) was used for tissue profiling; from the spatial transcriptomics data, copy number alterations (CNAs) were inferred to identify distinct tumor subclones, and a clone-tree was constructed to describe their evolutionary relationships. Additionally, a high-resolution spatial transcriptomics (2 µm) technology was employed to gain a cellular-level view of the immediate tumor-microenvironment (TME) surrounding specific subclones. Analysis of over 1,000,000 barcoded regions identified many distinct tumor subclones, enabling us to trace their evolutionary trajectories spatially. We observed notable subclonal events within the lymph nodes, including polyclonal colonization, indicating multiple dissemination events during the evolution of the primary disease. Exploration of the immediate TME revealed significant cellular heterogeneity and upregulation of genes related to antigen presentation and inflammatory pathways concentrated near ancestral tumor clones, specifically at the tumor border. By focusing on the inferred CNA profiles of the metastasizing clone across patients, we identified several common features. Additionally, we were able to identify potential lethal disease as early as the diagnostic core needle biopsy stage by detecting features of metastasis and pinpointing the metastasizing clone. In summary, our study provides a detailed spatial map of PCa clonal evolution and dissemination, linking primary tumors to nodal metastases and revealing altered cell composition and gene expression around tumor clone borders. Importantly, it demonstrates subclonal events within lymph nodes, polyclonal colonization, and the potential to identify metastasizing clones as early as the diagnostic phase, which holds strong implications for precision oncology and early intervention.
 1. Erickson, A., He, M., Berglund, E. et al. Spatially resolved clonal copy number alterations in benign and malignant tissue. Nature 608, 360–367 (2022). https://doi.org/10.1038/s41586-022-05023-2



 Mengxiao He, Sandy Figiel, Max Beesley, Jintong Shi, Wencheng Yin, Emmanouela Perisynaki, Richard Colling, Ian G. Mills, Joakim Lundeberg, Alastair D. Lamb. Comprehensive spatial profiling of prostate cancer metastasis: Mapping clonal evolution, microenvironment dynamics, and early metastatic markers [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr PR013.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/604426087ba878274c47234a7ce00263acf256a4" target='_blank'>
              Abstract PR013: Comprehensive spatial profiling of prostate cancer metastasis: Mapping clonal evolution, microenvironment dynamics, and early metastatic markers
              </a>
            </td>
          <td>
            Mengxiao He, S. Figiel, Max Beesley, Jintong Shi, Wencheng Yin, Emmanouela Perisynaki, Richard Colling, I. Mills, J. Lundeberg, Alastair D. Lamb
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Chronic myeloid leukemia is a hematopoietic stem cell malignancy characterized by excessive proliferation of white blood cells, primarily driven by the Philadelphia chromosome (t 9;22). Despite the availability of tyrosine kinase inhibitors, disease progression and therapy resistance remain major challenges, often linked to additional somatic mutations. Objectives: To find out the mutational status and expression variations of the ASXL1 gene hotspot region in CML patients treated by TKIs. Methods: This retrospective cross-sectional study was conducted at the Department of Zoology, Division of Science and Technology, University of Education, Lahore, Pakistan. DNA and mRNA of 50 CML patients were analyzed alongside 10 healthy controls and statistically assessed using SPSS. Results: Results showed that sequence analysis of exon 13 identified hotspot mutations, including two novel missense alterations, G659S and D667E, within the coding region of ASXL1. In-silico analysis also suggested potential structural and functional relevance of these variants. The ASXL1 gene expression showed a progressive but statistically non-significant down-regulation in CML patients across disease phases (p=0.662), disease duration (p=0.42), treatment with second-line versus first-line TKIs (p=0.412), and wild-type and mutant (p=0.544). Conclusions:  It is concluded that two novel missense alterations, G6559S and D667E, were identified in the hotspot region, and ASXL1 gene expression was downregulated non-significantly in CML patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14ee16b9a16245af7fb856280afba55f65850e27" target='_blank'>
              Identification of Novel Alterations in the ASXL1 Gene and Its Expression Profile in CML Patients
              </a>
            </td>
          <td>
            R. Mehmood, Khuram Shehzad Khan, A. Akram, Sikandar Hayat, Amjad Zafar
          </td>
          <td>2025-12-31</td>
          <td>Pakistan BioMedical Journal</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46cb1762556f83822b8fcd8773e27d5e3004fd71" target='_blank'>
              Circular RNAs: key regulators in cancer stem cell dynamics
              </a>
            </td>
          <td>
            P. Jain, Varsha Kaushik, Ravindresh Chhabra
          </td>
          <td>2026-02-03</td>
          <td>Molecular Genetics and Genomics</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dde5fce21e0ea399ec39e2d1188eb639ee8561aa" target='_blank'>
              Assessing off-target effects in CRISPR/Cas9: challenges and strategies for precision DNA editing
              </a>
            </td>
          <td>
            Abdul Basit, Jianzhong Zhu, Wanglong Zheng
          </td>
          <td>2026-01-12</td>
          <td>Archives of Microbiology</td>
          <td>1</td>
          <td>20</td>
        </tr>

        <tr id="Lung cancer is a highly malignant disease, posing a significant threat to global health. The presence of tumor heterogeneity results in substantial variations in prognosis and therapeutic responses among patients. Advances in bulk RNA sequencing and single-cell RNA sequencing have facilitated the identification of driver gene mutations and the exploration of cellular diversity within tumors. However, tumors are complex ecosystems comprising both tumor cells and their microenvironment, where interactions among different cell types give rise to specific functional structural units that collectively drive tumorigenesis and progression. The emergence of spatial omics technologies has allowed for the analysis of tumor ecosystems, providing unprecedented insights into tumor heterogeneity. This review aims to present updates on spatial omics technologies and data analysis algorithms, discuss current technical limitations, and explore potential future developments. Furthermore, we summarize the latest applications of spatial omics in elucidating lung cancer heterogeneity, investigating mechanisms of lung cancer progression and drug resistance, and identifying novel biomarkers. Based on these findings, we propose strategies for integrating spatial omics into lung cancer research, offering new perspectives for precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eca909ba170b3407f8c473dadaa8f67180cd2619" target='_blank'>
              The Evolution of Spatial Omics Technologies Introduces A Novel Avenue for Lung Cancer Research.
              </a>
            </td>
          <td>
            Yue He, Zifan Li, Wenxiang Wang, Xu Liu, Shanshan Lu, Jing Bai, Lin Weng, Qingna Zhang, Jun Wang, Kezhong Chen
          </td>
          <td>2026-01-30</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) is defined by the clonal expansion of hematopoietic stem and progenitor cells harboring somatic mutations that confer a fitness advantage. CH is common with advancing age and becomes nearly ubiquitous in middle age. Although typically asymptomatic, CH is associated with an increased risk of hematologic malignancies particularly myeloid neoplasms (MN), diverse non-malignant conditions, and all-cause mortality. Over the past decade, research has provided major insights into the origins of CH. In addition to aging, CH is promoted by environmental exposures, inherited genetic predisposition, and acquired conditions. Large-scale population and longitudinal sequencing studies have identified determinants of clonal behavior. Characterization of the natural history of CH has enabled the development of risk stratification models to identify individuals with CH at high risk for progression to MN, thereby providing a rationale for selecting patient populations best suited for therapeutic intervention trials. Emerging strategies include targeting mutation-specific vulnerabilities, modulating inflammatory pathways, reducing genotoxic therapy-induced clonal selection, and repurposing agents with efficacy in MN. In this review, we summarize current knowledge of the risk factors underlying CH development, highlight recent advances in understanding the determinants of clonal behavior including progression to MN, and discuss emerging therapeutic approaches for preventing malignant transformation and clinical trial design considerations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e8f2ddfaf7617522bdf1484770e1e9f235abbf2" target='_blank'>
              Clonal hematopoiesis and its progression to myeloid neoplasms: insights into risk, biology, and therapeutic strategies.
              </a>
            </td>
          <td>
            Scott J Beeler, Matthew J. Walter, Kelly L. Bolton
          </td>
          <td>2026-02-05</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Chronic myeloid leukaemia (CML) is a clonal myeloproliferative cancer caused by the constant activity of the BCR–ABL 1 fusion protein' s tyrosine kinase, resulting from the Philadelphia chromosome translocation, which leads to abnormal cell growth, survival, and disease progression. While tyrosine kinase inhibitors (TKIs) have greatly improved patient outcomes, issues like drug resistance and persistent leukemic stem cells highlight the need for alternative therapies. This study used a structure- guided CRISPR- Cas 9 genome editing approach to identify highly specific single- guide RNAs (sgRNAs) that can disrupt the human ABL 1 gene, a key part of the BCR–ABL 1 fusion. The high- resolution crystal structure of the ABL 1 kinase domain (PDB ID: 8 I 7 S) helped identify essential functional regions, including catalytic and ATP- binding sites, for precise CRISPR targeting. Computational design and filtering of sgrnas were performed using E- CRISP and CHOPCHOP, focusing on criteria like PAM site accessibility, targeting conserved kinase regions in exons, GC content, predicted efficiency, and low off- target risk. In silico analyses, including specificity scores, mismatch profiles, and sequence alignment across ABL 1 transcript variants, ensured high selectivity and broad coverage. Genomic visualization confirmed accurate targeting within exons encoding vital kinase functions. Protein–protein interaction analysis via STRING showed strong links between ABL 1 and key oncogenic regulators such as BCR, STAT 5, and MAPK pathway components. KEGG pathway analysis further indicated ABL 1' s involvement in chronic myeloid leukaemia, PI 3 K–AKT, MAPK signaling, and other cancer- related pathways, emphasising its importance in CML development. This combined computational approach demonstrates that structure- guided CRISPR- Cas 9 targeting of ABL 1 can effectively disrupt BCR–ABL 1 driven cancer signals. The results provide a strong theoretical basis for future experimental validation and genome editing therapies aimed at overcoming TKI resistance and achieving long- lasting molecular remission in CML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea06e417904b01587a2a61ab158e9f0d860c3941" target='_blank'>
              Structure-guided CRISPR CAS9 Targeting of ABL1 for Functional Disruption of BCR-ABL1 Fusion in Chronic Myeloid Leukaemia
              </a>
            </td>
          <td>
            Sumita Katal, Shivangi Koundal, Uma Kumari
          </td>
          <td>2026-02-04</td>
          <td>Computational Biology and Bioinformatics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bcde693a87853a15bccfb4a948cfd12597ab0e39" target='_blank'>
              Cancer Driver Gene Discovery: A Patient-Level Statistical Framework
              </a>
            </td>
          <td>
            Farideh Bahari, Hesam Montazeri
          </td>
          <td>2026-02-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="By generating 194 epigenomic and transcriptomic datasets from 57 human tissue samples using H3K27ac and HIF2α chromatin immunoprecipitation sequencing (ChIP-seq), assay for transposase-accessible chromatin using sequencing (ATAC-seq), and RNA sequencing, we provide a comprehensive, integrated characterization of clear cell renal cell carcinoma (ccRCC) across normal, tumor, and metastatic states. Our analyses provide several insights into ccRCC biology. First, we demonstrate significant reprogramming of enhancer and HIF2α cistromes as well as chromatin accessibility during the normal-to-tumor transition, whereas localized and metastatic tumors show minimal epigenomic differences. Second, we show reactivation of kidney-specific developmental pathways driving malignancy. Third, we perform a cistrome-wide association study in ccRCC, validating five established RCC risk loci and identifying six novel loci, including a locus at 12q24 linked to SCARB1 that was functionally validated. These datasets provide new perspectives on the role of developmental pathways in ccRCC tumorigenesis, insights into epigenetic mechanisms of ccRCC heritability, and a comprehensive epigenomic atlas for the research community.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b37eefc44af89f924062f3fabaa4f6f9cb92f8ba" target='_blank'>
              Generation of a comprehensive epigenomic atlas in clear cell renal cell carcinoma informs kidney cancer progression and heritability.
              </a>
            </td>
          <td>
            Sarah Abou Alaiwi, E. Adib, Ziwei Zhang, S. Spisák, A. Prósz, Paulo Cordeiro, T. El Zarif, Z. Sztupinszki, Amin H. Nassar, , Rianna Campbell, J. Steinharter, Mitchell Machiela, Diptavo Dutta, S. Chanock, M. Purdue, M. Pomerantz, S. Signoretti, Zoltán Szállási, A. Gusev, S. Baca, T. Choueiri, M. Freedman
          </td>
          <td>2026-02-01</td>
          <td>Cell reports</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="Epigenetic marks such as DNA methylation, histone modifications, and non-coding RNAs play a central role in regulating gene expression without altering the DNA sequence. These dynamic and reversible modifications contribute to cellular differentiation, genomic stability, and the fine-tuning of transcriptional activity in response to environmental and metabolic cues. Beyond genetic variation alone, epigenetic mechanisms provide a critical regulatory layer linking environmental factors, cellular context, and disease susceptibility. Alterations in epigenetic marks can disrupt normal gene expression patterns, contributing to a wide range of pathological conditions, including cancer, neurodegenerative, metabolic, and immune-mediated diseases. This review places particular emphasis on cancer, cardiovascular diseases, and autoimmune disorders, where epigenetic dysregulation has been most extensively characterized and therapeutically explored. Because epigenetic modifications are reversible and influenced by environmental and lifestyle factors, they represent highly promising targets for therapeutic intervention. In recent years, epigenetic-based drugs—such as DNA methyltransferase inhibitors, histone deacetylase inhibitors, and agents targeting non-coding RNAs—have demonstrated significant potential in preclinical and clinical studies. However, their clinical application remains limited by challenges including non-selectivity, systemic toxicity, and limited tissue specificity. Emerging technologies such as CRISPR/dCas9-mediated epigenome editing offer opportunities for precise regulation of disease-associated genes without permanent genomic alterations. This review summarizes current knowledge on major epigenetic mechanisms and highlights recent advances in epigenetic therapies, including FDA-approved agents, combinatorial strategies, and next-generation epigenetic editing approaches. A deeper understanding of these mechanisms is expected to enable targeted, reversible, and patient-specific therapeutic strategies within the framework of precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e63889ba19a59ab3f3b40aa0ebd9c404687b05f" target='_blank'>
              Epigenetic marks and promising epigenetic therapies
              </a>
            </td>
          <td>
            Selin Kankaya, Yildiz Dincer
          </td>
          <td>2026-02-10</td>
          <td>Academia Molecular Biology and Genomics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancer stem cells (CSCs), a resilient subset of tumor cells, able to evade immune detection and rapidly­ proliferate, are responsible for the metastasis, recurrence, and therapeutic resistance observed across various­ cancers. Recent research has concentrated on understanding the molecular networks that support CSCs immune evasion, self-renewal, and adaptability. Signaling pathways (Wnt, Notch, Hedgehog, JAK-STAT) and surface markers (CD44, CD133, ALDH1) that characterize CSC behaviour are compiled in this review. We highlight the expanding usefulness of omics technologies, such as CRISPR functional genomics, single-cell transcriptomics, and spatial proteomics, in determining vulnerabilities unique to CSCs and guiding­ tailored treatment plans.

Keywords: cancer stem cells, resistance to treatment, signalling pathways, surface markers, targeted therapy">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c3feec95a2194e3381ef329436c8fa44984f344" target='_blank'>
              Cancer stem cells in recurrence and therapy resistance: biological insights and emerging therapeutic strategies
              </a>
            </td>
          <td>
            Y. Tamilselvi, P. Velmurugan, K. Sivasubramanian
          </td>
          <td>2026-02-17</td>
          <td>The Ukrainian Biochemical Journal</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Enhancers are distal cis-regulatory elements that orchestrate spatiotemporal gene expression patterns in response to developmental cues and environmental stimuli. Genetic and epigenetic alterations in enhancers are associated with the initiation and progression of human diseases, including cancers. Over the past few decades, accumulating evidence has revealed that a class of nascent RNA transcripts, known as enhancer RNAs (eRNAs), is broadly transcribed from active enhancers. These eRNA species contribute to complex and dynamic gene regulatory networks under both physiological and pathological conditions through diverse mechanisms. Notably, dysregulated eRNA expression has been reported across various cancer types and is often correlated with patient survival outcomes. Consequently, eRNAs are emerging as promising biomarkers and therapeutic targets for cancer treatment. This review provides a comprehensive summary of the current understanding of eRNAs and their mechanisms of action in gene regulation. We discuss the critical roles of eRNAs in both health and disease and highlight their diagnostic and prognostic value, as well as their therapeutic potential in cancer. Additionally, we review current strategies for targeting RNA transcripts, including eRNAs, and discuss the major challenges in developing eRNA-targeted therapies. Finally, we propose future directions for advancing eRNA-based interventions in the treatment of human diseases, including cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9397eec4991d568fddc2093214f80298c06bfee4" target='_blank'>
              Deciphering the regulatory landscape of enhancer RNAs in health and disease
              </a>
            </td>
          <td>
            Qian Wang, P. ten Dijke, Chuannan Fan
          </td>
          <td>2026-01-29</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9bc72c5e888ff2d0403d211e07d0ceac66c5bb29" target='_blank'>
              Systematic analysis of functional genetic and epigenetic variants in colorectal cancer.
              </a>
            </td>
          <td>
            Erfei Chen, Qiqi Yang, Haoyang Dai, Yixin Chen, Yihui Zhang, Qiang Wang, Rongrong Hou, Ming Chen, Jie Wang, Qianwen Xie, Wenju Sun, Yong-Qiang Ning, Ligang Fan, Jian Yan
          </td>
          <td>2026-02-20</td>
          <td>Science advances</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the most prevalent malignant neoplasms worldwide, characterized by a high incidence of recurrence and metastasis, which substantially diminishes patient survival rates. This underscores the urgent need to identify novel biomarkers and therapeutic targets. Transcribed ultraconserved regions (T-UCRs), a category of non-coding RNAs with significant evolutionary conservation, are crucial to various biological processes. Recent studies have shown that T-UCRs play a pivotal role in tumorigenesis and tumor progression. A growing body of evidence indicates that T-UCRs significantly influence CRC development by modulating critical mechanisms, including cell proliferation, apoptosis, invasion, and metastasis. This review systematically explores the functions of T-UCRs in tumorigenesis, focusing on their regulatory roles, underlying molecular mechanisms, and potential clinical applications in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ca72dace2c26f564f76a90eafc66dbf14b419b5" target='_blank'>
              The transcribed ultraconserved regions: emerging players in colorectal cancer biology and therapy
              </a>
            </td>
          <td>
            Yi Zhang, Zichen Wei, Xin Wang, Pan Wang, L. Pang, Hongliang Dong, Han Xu
          </td>
          <td>2026-01-02</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4fbc4834d55cafd4feb67d657137396aa98a784" target='_blank'>
              Clinical and genetic insights into novel TP53 mutations in De Novo AML patients
              </a>
            </td>
          <td>
            A. Akram, Sikandar Hayat, Hassan Yousaf, Noreen Sarwar, Asma Tahir, Sakina Ali, Fatima Yaqoob, Amjad Zafar, , Mutwakel Dabiellil, Malik Ihsanullah Khan
          </td>
          <td>2026-02-12</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ea2744fede5f7a4975fd041ac239356c320c431" target='_blank'>
              Large-scale mapping of environmental-genetic interactions illustrates the dynamic nature of cell-cycle and DNA repair regulation.
              </a>
            </td>
          <td>
            Benjamin W. Herken, Garrett T Wong, Anna Mäkiniemi, E. Lundberg, Thomas M. Norman, Luke A. Gilbert
          </td>
          <td>2026-02-19</td>
          <td>Molecular cell</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Familial cancers are caused by inherited mutations in specific genes that regulate cell growth, division, and repair. Approximately 5–10% of all cancer cases have a hereditary component, where germline mutations in certain genes increase an individual’s susceptibility to developing cancer. Two major categories of genes are involved in cancer development: tumour suppressor genes and oncogenes. Both play critical roles in regulating normal cell behaviour, and when mutated, they can contribute to uncontrolled cell proliferation and tumour formation. In addition to genetic mutations, epigenetic alterations also play a significant role in familial cancer. Epigenetics refers to changes in gene expression due to DNA methylation, histone modifications, and the dysregulation of non-coding RNAs without alter the underlying DNA sequence. Familial cancer syndromes follow various inheritance patterns, including autosomal dominant, autosomal recessive, X-linked, and mitochondrial inheritance, each with distinct characteristics. Identifying genetic mutations associated with familial cancers is a cornerstone of genetic counselling, which helps individuals and families navigate the complex intersection of genetics, cancer risk, and prevention. Early identification of mutations enables personalized strategies for risk reduction, early detection, and, when applicable, targeted treatment options, ultimately improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69f296f3ae0afec7c6382eb8b634a83d98dd9138" target='_blank'>
              Genetic Basis of Familial Cancer Risk: A Narrative Review
              </a>
            </td>
          <td>
            Eman Fares Sabik
          </td>
          <td>2026-01-13</td>
          <td>DNA</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Z-DNA/Z-RNA is an alternative left-handed nucleic acid conformation with established and emerging roles in gene regulation, immunity, and genome instability. However, its occurrence dynamics and lineage specificity across the tree of life have not yet been fully characterized. Utilizing the recently developed and improved Z-DNA searching tool, ZSeeker, we analyzed 281,139 complete organismal genomes, including multiple Telomere-to-Telomere genome assemblies, and generated genome-wide Z-nucleic acid maps, examined their topography, and compared them to dinucleotide-preserving controls. Cellular genomes featured pervasive Z-DNA enrichment relative to expectation, with enrichments of ∼1.5 and ∼1.7-fold in Bacteria and Archaea and ∼3-fold in Eukaryota. In contrast, Viruses exhibited large differences between lineages, with modest enrichment in several DNA viral groups and pronounced depletion across RNA clades, most notably Influenza A/B strains. We built a LASSO regression model trained on non-Influenza viruses (cross-validated R² ≈ 0.73), which identified GC content, genome type, and host type as the leading predictors for Z-nucleic acid density, yet it significantly over-predicted Z-RNA density in Influenza A/B. More than 99% of assemblies exceeded the +2 SD threshold, and a “typical Influenza” genome was predicted at 2.76 bp/kb compared to ∼0.016 bp/kb observed (a ∼170-fold overestimation based on chance alone). Together, these results reveal domain- and lineage-specific regimes: cellular genomes are enriched for Z-DNA consistent with regulatory roles, whereas influenza viruses appear to have undergone strong, lineage-specific depletion of Z-RNA-forming sequences, likely reflecting evolutionary pressure tied to host sensing pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51d5326854d4ddc954dec631bbab0b254c15320a" target='_blank'>
              Characterization of Z-DNA dynamics across the tree of life
              </a>
            </td>
          <td>
            Georgios Megalovasilis, Eleftherios Bochalis, Michail Patsakis, Dionysios V. Chartoumpekis, Guliang Wang, Karen M. Vasquez, IIias Georgakopoulos-Soares
          </td>
          <td>2025-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1dca7031a26452f3ce74afb57e6bdc59826c4916" target='_blank'>
              A meta-analysis identifies driver genes and characterizes the molecular epidemiology of colorectal cancer.
              </a>
            </td>
          <td>
            S. Olafsson, Thorri Thorarinsson, S. A. Gudjonsson, Mariana Bustamante, H. Gunnarsdottir, Hildur Knutsdottir, Hákon Jónsson, Magnus I. Magnusson, Emilia Soebech, Hjaltey Runarsdottir, Droplaug N. Magnúsdóttir, Louise le Roux, J. Saemundsdottir, Bjarney S. Kristinsdottir, Bjarni A Agnarson, Erna M. Jonsdottir, T. Tryggvason, M. O. Ulfarsson, D. Gudbjartsson, Jon G. Jonasson, Olafur T. Magnusson, Kári Stefánsson, T. Rafnar
          </td>
          <td>2026-02-17</td>
          <td>Scientific reports</td>
          <td>0</td>
          <td>132</td>
        </tr>

        <tr id="Autoimmune diseases (AIDs) are chronic disorders driven by the breakdown of immune tolerance, where genetic susceptibility and environmental triggers converge to promote aberrant T and B cell activation, inflammatory cytokine release, and progressive tissue damage. Current therapeutic strategies, including corticosteroids, immunosuppressants, and biologics, can alleviate symptoms but remain limited by adverse effects, incomplete remission, and variable patient responses. CRISPR interference (CRISPRi), a novel gene regulation platform based on catalytically inactive Cas9 and guide RNAs, achieves precise, reversible suppression of target genes without inducing DNA double-strand breaks, thereby offering improved safety and controllability over conventional genome editing. Recent studies demonstrate that CRISPRi can effectively silence multiple inflammatory mediators and immune checkpoint molecules, enabling immune microenvironment remodeling and providing a feasible strategy for complex autoimmune conditions. Advances in multiplex CRISPRi, including the design of multi-sgRNA constructs and delivery via viral or non-viral systems, expand its capacity to simultaneously repress synergistic cytokine networks. Despite ongoing challenges in delivery efficiency, off-target risk, and long-term safety, CRISPRi holds considerable promise for network-level therapeutic intervention. This review aims to summarize the principles, technological progress, and potential applications of CRISPRi in autoimmune diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1138bca3d3877bd7932621af6651b3b14d329c9b" target='_blank'>
              Potential Applications and Research Exploration of CRISPRi in the Treatment of Autoimmune Diseases
              </a>
            </td>
          <td>
            Shaopu Huang
          </td>
          <td>2025-12-28</td>
          <td>Academic Journal of Science and Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Hematopoietic cell reprogramming, defined as the conversion from one hematopoietic or immune cell type into another, has been shown to hold transformative potential for advancing our understanding of hematopoiesis, disease mechanisms, and therapeutic innovation. This process occurs via transdifferentiation, dedifferentiation, or transitional plasticity, driven by transcriptional, epigenetic, and microenvironmental cues, which may be achieved by physiological or pathological stressors in vivo, or genetic or chemical strategies in vitro. Nevertheless, challenges persist, including low reprogramming efficiency, unstable phenotype, and safety concerns. Advancements in multi-omics, gene editing, and chemical biology are enhancing the efficacy of reprogramming protocols and facilitating clinical translatability. The integration of these strategies, in conjunction with artificial intelligence-driven screening and single-cell analytics, has the potential to facilitate the development of personalized cell therapies for cancers, immune disorders, and regenerative medicine. Additionally, the realization of this potential is contingent upon the resolution of challenges related to delivery, specificity, and long-term efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/074b0a39466afd2855b2f02374d1bb3bcc283ce0" target='_blank'>
              Natural and Artificial Hematopoietic Cell Reprogramming.
              </a>
            </td>
          <td>
            Dong Yu, Wenpei Liu, Feng Liu, Lin Cheng
          </td>
          <td>2026-02-12</td>
          <td>Cellular reprogramming</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Myelofibrosis (MF) is a myeloproliferative neoplasm (MPN) characterized by splenomegaly, constitutional symptoms, bone marrow fibrosis and potential progression to a blast phase. This review provides a comprehensive overview of the current molecular landscape of MF beyond canonical driver mutations (JAK2, MPL or CALR), emphasizing insights gained from murine models that served as valuable tools for understanding disease mechanisms. High-throughput next-generation sequencing (NGS) has markedly enhanced our understanding of the molecular basis of MF, identifying numerous mutations beyond the canonical driver genes JAK2, MPL, and CALR, which are present in about 80% of patients. Additional mutations affect genes involved in DNA methylation (TET2, DNMT3A, IDH1, IDH2), histone modification (ASXL1, EZH2), mRNA splicing (SF3B1, SRSF2, U2AF1, ZRSR2), signaling pathways (CBL, NRAS, KRAS), and key transcription factors (RUNX1, NFE2, TP53). The presence and combination of these alterations influence clinical presentation, prognosis, and therapeutic response. This review offers an updated synthesis of the evolving molecular landscape of MF, highlighting how the intricate interplay among genetic alterations has deepened our understanding of disease heterogeneity, allowing refined risk stratification and therapeutic planning. Advances emerging from molecular research and experimental models are progressively translating into clinical practice, promoting more personalized and targeted approaches to the management of MF.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfa1e50ab4c23ee2312532232e5a53b22f4357a8" target='_blank'>
              The Role and Impact of Non-driver Gene Mutations in Myelofibrosis
              </a>
            </td>
          <td>
            V. Boldrini, P. Guglielmelli, A. Vannucchi
          </td>
          <td>2026-02-10</td>
          <td>Current Hematologic Malignancy Reports</td>
          <td>0</td>
          <td>113</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a0ee442281780349396ea188b6030cb2f972249" target='_blank'>
              FBXW7 mutations reprogram glucose metabolism by activating the ETV6-GLUT1 axis.
              </a>
            </td>
          <td>
            Siqi Fei, Xiayun Xu, Tingrui Wang, Xinyue Jiang, Hanrong Zhu, Yingji Chen, Renjie Yuan, Zeheng Lv, Zhijian Gao, Huanxin Liu, Xiaojun Chen, Chenji Wang, Kun Gao
          </td>
          <td>2026-02-14</td>
          <td>Molecular cancer</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Whole genome doubling (WGD) is a frequent event in cancer evolution associated with chromosomal instability, metastasis, and poor prognosis. While the genomic consequences of WGD are well documented, non-genetic alterations that accompany WGD, such as changes to cell and nuclear size, may also play an important role in tetraploid (4N) cancer cell physiology. Here, we showed that cell and nuclear volume do not always scale with DNA content after WGD in cancer cells, resulting in 4N cells that differ in size. Small size was associated with enhanced cell fitness, mitotic fidelity, and tumorigenicity in 4N cancer cells and with poor patient survival in WGD-positive human cancers. Overall, these results suggest that cell and nuclear size may contribute to the tumorigenic potential of 4N cancer cells and could be an important prognostic marker in human tumors that undergo WGD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e3ce0e5eeb35d45bde98218a3b940e6aa2d9ace" target='_blank'>
              Cell and Nuclear Size are Associated with Chromosomal Instability and Tumorigenicity in Cancer Cells that Undergo Whole Genome Doubling.
              </a>
            </td>
          <td>
            Mathew Bloomfield, Sydney M Huth, Daniella S McCausland, Ron Saad, Nazia Bano, Tran N Chau, Megan L Sweet, N. Baudoin, Andrew McCaffrey, Kai Fluet, Eva M. Schmelz, U. Ben-David, Daniela Cimini
          </td>
          <td>2026-01-22</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92529c21b1d903977db7b6f99cc016a10cc689ad" target='_blank'>
              Molecular analysis of tumor recurrence using established cancer stem cell-line and drug discovery.
              </a>
            </td>
          <td>
            Kiyotaka Nakano, E. Oki, Masaki Yamazaki, Masami Suzuki, S. Kawai, Y. Zaitsu, Chiyoko Nishime, K. Ando, G. Nagae, N. Harimoto, Mitsuhiko Ota, T. Kawazoe, Kentaro Nonaka, Keita Natsugoe, Sachie Omori, H. Saeki, Hiroyuki Aburatani, Tatsumi Yamazaki, Yoshihiko Maehara
          </td>
          <td>2026-01-20</td>
          <td>International journal of clinical oncology</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a54d931016d849a45a82192eedf06a276e9b6b73" target='_blank'>
              Rescuing the Function of Missense-Mutated Tumor Suppressor VHL using Stabilizing Small Molecules
              </a>
            </td>
          <td>
            Mariam Fouad, Christopher S. Parry, Sven A. Miller, Shipra Malhotra, Glenn A. Doyle, Ezra Shimabenga, Mashhura Nurilloeva, Elena Bondarenko, Grigorii V. Andrianov, Wayne Childers, Benjamin E. Blass, Petr Makhov, Johnathan R. Whetstine, M. Clapper, E. Golemis, John Karanicolas
          </td>
          <td>2026-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="Prostate cancer is one of the major causes of cancer morbidity and mortality in men, especially at advanced stages where treatment fails. Despite the fact that the existing treatment options, such as androgen deprivation therapy, chemotherapy, radiotherapy, PARP inhibitors, and targeted agents, can successfully manage the disease progression, resistance and recurrence are frequent. Recent evidence suggests that therapy-induced senescence is dual and context dependent in prostate cancer. Although senescence at first limits tumor cell growth by halting cellular proliferation, senescent cells may nonetheless survive the treatment and cause tumor progression, immune evasion, and therapeutic resistance through the senescence-associated secretory phenotype (SASP). In addition, the senescent tumor cells can avoid the growth arrest through epigenetic reprogramming and metabolic adaptation, re-enter the cell cycle, and develop more aggressive and recurrent disease. This is a comprehensive review of the existing body of knowledge on how standard treatments induce senescence in prostate cancer cells. We cover the biological processes of senescence entry, maintenance, and escape, and the interactions between senescent tumor cells, the tumor microenvironment, and immune modulation. Also, we discuss the role of senescence in minimal residual disease, drug-tolerant persister cells, and dormant disseminated tumor cells, with a particular focus on their role in late relapse and metastasis. Lastly, the review highlights future therapeutic approaches to tackle senescent cells, such as senolytics and senomorphics, and addresses the current challenges of toxicity, biomarker discovery, and translation into clinical use. Future endeavors in the field of therapy-induced senescence can facilitate in devising treatment plans with prudent combination of senescence-based strategies to overcome therapy resistance in prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ae5697664dff1040b844f855f0aa51560e5dfd1" target='_blank'>
              Targeting Cellular Senescence in Prostate Cancer: Molecular Landscape and Therapeutic Avenues
              </a>
            </td>
          <td>
            Haneen Hossam, Bandar Alattaibi
          </td>
          <td>2026-02-07</td>
          <td>annals of urologic oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Gastric cancer remains a leading cause of cancer-related mortality worldwide and is marked by pronounced molecular heterogeneity. Advances in genomic profiling have identified key genetic alterations, including oncogenes (HER2, PIK3CA, and MYC), tumor suppressor genes (TP53, CDH1, and ARID1A), and regulators of genome stability and cell architecture (MLH1, RHOA, and CLDN18), which have driven the development of targeted therapeutic strategies. Although genetically engineered mouse models and xenograft systems have been indispensable for functional validation and preclinical drug testing, many approaches that showed promising efficacy in animal models—such as inhibition of EGFR, MET, FGFR2, and the PI3K pathway—failed to translate into overall survival benefits in clinical trials, highlighting major translational limitations. In contrast, HER2- and CLDN18.2-targeted therapies represent rare but notable clinical successes, underscoring the importance of true oncogenic dependency, precise biomarker-driven patient selection, and robust preclinical validation. In this review, we systematically categorize gastric cancer-associated genes according to their biological functions, summarize representative animal models, and critically examine key successes and failures in clinical translation, emphasizing the need for biologically faithful models and precision-driven translational strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fef5de06d66f7c8fe26d0ed720049bf095f0cc18" target='_blank'>
              Translating Gastric Cancer Genomics into Targeted Therapy: Mechanistic Insights from Animal Models and Patient-Derived Systems
              </a>
            </td>
          <td>
            R. Shyu, Lu-Kai Wang, F. Tsai
          </td>
          <td>2026-02-18</td>
          <td>Cells</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Abstract Background Hematologic malignancy (HM) patients have an increased incidence of invasive fungal disease (IFD) (12% prior to anti-mold prophylaxis, with a decrease to 5% with prophylaxis), compared to hematopoietic stem cell transplant (HSCT) recipients (about 4%). Analysis of gene expression patterns (‘signatures’) in leukocytes can provide supplementary diagnostic and immunologic information about the variable and often dysregulated immune responses that occur during these devastating infections, even in the setting of neutropenia. Methods Bulk RNA sequencing was performed from peripheral blood of neutropenic and non-neutropenic HM and HSCT recipients with adjudicated IFD. Differential expression (DEA) and over-representation analyses (ORA) were performed to identify and characterize transcriptional changes. Results 43 of 55 included subjects were neutropenic (Table 1). DEA revealed a coordinated decrease in expression in HM patients compared to HSCT patients for a subset of genes, and ORA was used to elucidate their potential etiology. ORA showed out of 6,812 gene sets, 35 were significantly overrepresented among the gene subset. These overrepresented gene sets coalesced into large immune-related signals. These signals included: 1) inflammatory and innate immune responses; 2) leukocyte mediated immunity; 3) leukocyte migration, neutrophil chemotaxis; and 4) cytokine production. Individual genes most differentially expressed between the 2 infected groups included those encoding C-type lectins, TLR2, and other elements of antifungal immunity. Conclusion We found that the immune responses (as demonstrated by their transcriptional profiles) of HM and HSCT patients to IFD differ in ways not explained by neutropenia alone. HM patients demonstrate differential expression of pattern recognition receptors and comparatively decreased innate immune responses and decreased cytokine production, potentially reflective of the prolonged disturbance of the innate immune system in HM patients compared to HSCT. This provides novel insight into the host response to real-world IFD in neutropenic patients, and the particular susceptibilities of HM patients to IFD. Disclosures Julie M. Steinbrink, MD, MHS, Biomeme: patents for gene expression classifiers of fungal infection|McGraw Hill Publishing: royalties Micah T. McClain, MD, PhD, Biomeme: patents for gene expression classifiers of fungal infection|Darwin Biosciences: Board Member|UpToDate: Advisor/Consultant">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb67440a42d5863cd11541742eb0e2d8f2d6e6b4" target='_blank'>
              P-1762. The Host Transcriptional Response to Fungal Infection Suggests Disturbances of the Innate Immune System and Cytokine Signaling in Hematologic Malignancy that Increase Susceptibility to Infection Relative to Hematopoietic Stem Cell Transplantation
              </a>
            </td>
          <td>
            J. Steinbrink, Cameron Miller, Kelly Stanly, Barbara D. Alexander, M. McClain
          </td>
          <td>2026-01-01</td>
          <td>Open Forum Infectious Diseases</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Haemoglobinopathies, including β-thalassaemia and sickle cell disease (SCD), are among the most common monogenic disorders worldwide and remain major causes of morbidity and early mortality. Historically, management of these life-altering diseases has relied on supportive treatment and symptom management and, although these treatments reduce symptoms and ease disease burden, they do not correct the underlying genetic defect. Allogenic haematopoietic stem cell transplantation (HSCT) has been the only established curative option; however, it comes with substantial risks that significantly restrict its applicability. Over the past two decades, haematopoietic stem cell (HSC) gene therapy for haemoglobinopathies has rapidly progressed from experimental proof-of-concept to approved therapies. Lentiviral gene addition approaches have demonstrated durable expression of functional β-like globin transgenes, achieving transfusion independence in β-thalassaemia patients and significant reductions in vaso-occlusive events in SCD patients. Alternative therapeutic approaches to promote HbF expression have proved to be highly successful. Gene silencing strategies targeting BCL11A have been successful clinically and, more recently, gene editing technologies such as CRISPR/Cas9 have enabled precise disruption of regulatory elements controlling γ-globin repression, leading to the approval of the first CRISPR-based therapy for SCD and β-thalassaemia. Emerging base editing technologies promise even more precise genetic modification and are advancing through clinical evaluation. Despite these advances, access to gene therapy remains restricted due to the need for highly specialised manufacturing, toxic myeloablative conditioning regimens, and high treatment costs. Ongoing improvements and adaptations in these areas are essential to ensure that gene therapies fulfil their potential as accessible, curative treatments for patients suffering from haemoglobinopathies worldwide.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/649eef2598b223310fa16262b913a4e8a3810022" target='_blank'>
              Advances in Gene Therapy for Inherited Haemoglobinopathies
              </a>
            </td>
          <td>
            Anna Gaspar, H. B. Gaspar
          </td>
          <td>2025-12-27</td>
          <td>Hematology Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Viral infection plays a significant role in the development and evolution of many cancers. It is estimated that between 13% to 18% of all new cancer cases are caused by infection. It is known that viral infection can induce transcriptional changes in cancer, including altering the expression of a seven-membered DNA editing enzyme family known as APOBEC3 or A3. These cytosine deaminases have been implicated in infectious diseases and cancer for their ability to induce cytidine to uridine (C>U) mutations in both viral and host DNA. Indeed, A3-associated mutational signatures SBS2 and SBS13 are present in more than 70% of cancer types and around 50% of all cancer genomes. However, little is known about the interplay between viral infection and A3 dysregulation in cancer. Accurate profiling of A3 expressions and their dysregulations caused by viral infections in cancer has historically been challenging using bulk RNA-seq. To address this, we leverage existing single-cell RNA-seq (scRNA-seq) data sets and our custom single cell bioinformatics pipeline, which uses multiple machine learning models for cell type annotation and cancer cell risk prediction, statistical methods for detection of mutational signatures at the single cell resolution, and custom k-mer based viral reference alignment for viral read detection in order to identify the source of A3 and viral reads across individual cell types within the breast cancer tumor microenvironment. Our studies using this pipeline have provided unparalleled resolution within the tumor microenvironment required to address questions about the association between viral presence and A3 dysregulation in various cancers. Previous analysis of head and neck tumors (GEO: GSE173468) allowed our lab to identify the presence of HPV16 viral reads, which were associated with a significant increase in A3B expression in infected basal cells of the tumor. Our recent analysis of breast cancer tumors (GEO: GSE16529 and GSE243526) has given us critical insights into cell type-specific A3 expression dynamics, indicating that the source of A3A expression was solely found coming from macrophage cells in the tumor and that high-risk cancer cells had primarily high expression of A3B and lacked A3A expression. In this study, we analyzed publicly available scRNA-seq datasets from two normal tissues (NTs), three primary tumors (PTs), and three tamoxifen-treated recurrent tumors (RTs) (GEO: GSE240112) in order to investigate the association between the presence of viral reads, A3 expression, and mutational signatures SBS2/13. We are currently optimizing our pipeline to further improve our viral read identification and association with specific cell types within each tumor microenvironment. This work is supported by the Texas Biomedical Research Institute’s Postdoctoral Forum Fellowship Grant.



 J. D. Lehle, M. Soleimanpour, N. Haghjoo, D. Ebrahimi. Single cell mutational fingerprints and the viral culprits: uncovering APOBEC3 dysregulation in breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2025; 2025 Dec 9-12; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2026;32(4 Suppl):Abstract nr PS2-12-19.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00e05140208148a719b032d68c10a3889c3c1cca" target='_blank'>
              Abstract PS2-12-19: Single cell mutational fingerprints and the viral culprits: uncovering APOBEC3 dysregulation in breast cancer
              </a>
            </td>
          <td>
            J. Lehle, M. Soleimanpour, N. Haghjoo, D. Ebrahimi
          </td>
          <td>2026-02-17</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Clustered regularly interspaced short palindromic repeats (CRISPR)-Cas systems have transformed genome editing through unprecedented precision, and next-generation variants (base and prime editors) further enhance specificity by enabling targeted nucleotide changes without introducing double-strand DNA breaks. These technologies have unlocked broad applications in therapeutic gene correction, functional genomics, infectious disease management, diagnostics, agricultural engineering, environmental biotechnology, and synthetic biology. However, the targeted delivery of these systems remains a major challenge due to the large and chemically distinct nature of their components, including Cas protein or its base/prime editor fusions, guide RNA, and in some cases, DNA repair templates—which complicate packaging, stability, and cellular uptake. Additional hurdles arise from tissue and cell-type specificity, differential intracellular environments, variable editing efficiencies, and the persistent risk of off-target genome modifications. This review outlines the key challenges in the delivery of CRISPR technologies as well provides a comprehensive overview of both current and emerging delivery strategies, including viral vectors (adenovirus, adeno-associated virus, and lentivirus), non-viral physical approaches (microinjection, electroporation, ultrasound, and hydrodynamic tail-vein injection), and nanoparticle-based modalities (lipid and polymeric nanoparticles, gold nanoparticles, DNA nanostructures, and extracellular vesicles). We also discussed the diverse applications of CRISPR-Cas9 in gene therapy, immune cell engineering for cancer therapies, and agricultural innovation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fbee13972890cd1c6a9b71ea735c4af65a744d1" target='_blank'>
              Engineering delivery platforms for CRISPR-Cas and their applications in healthcare, agriculture and beyond
              </a>
            </td>
          <td>
            Nisha Bharti, Unnati Modi, Dhiraj Bhatia, Raghu Solanki
          </td>
          <td>2026-01-05</td>
          <td>Nanoscale Advances</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="A large number of reports have suggested that the cancer is associated with the gene mutations and chromosomal aberrations, and thus it has been a focus of research. Today new insights are proposed by analyzing relevant literature, suggesting that the cancer is not caused by the gene mutations and chromosomal aberrations, but by the regulatory dysfunction of gene expression resulted from the nuclear damage, among which, the gene mutations and chromosomal aberrations are only an accompany phenomena also resulting from the nuclear damage. The gene mutations generally do not cause diseases, but if they do, they mainly cause the single-gene diseases. The chromosomal aberrations usually refer to the abnormalities in chromosome structure and number, often leading to cell death or inducing various chromosomal diseases. Some chronic refractory diseases, such as the cancer, Alzheimer's disease, degenerative diseases, are probably not caused by the gene mutations and chromosomal abnormalities, but rather by the abnormal regulation of gene expression and the dysfunction of DNA transcription-protein synthesis resulted from the nuclear damage. These diseases are all the diseases of nuclear dysfunction and collapse of differentiation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4826bbb55a85cfdd2750ad0771ebe3b7d95aef3d" target='_blank'>
              Differences and Pathogenicity Between the Gene Mutation and Chromosomal Aberration
              </a>
            </td>
          <td>
            Maojin Li
          </td>
          <td>2025-12-29</td>
          <td>Medical Research Frontiers</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="RAS mutations are major drivers of tumorigenesis and represent important therapeutic targets; however, most remain resistant to effective pharmacological inhibition. KRAS and NRAS mRNAs contain guanine (G)-rich regions, forming stable G-quadruplexes (G4s) that regulate translation. Targeting and stabilizing these G4 structures with specific ligands may suppress their expression, offering a potential therapeutic strategy for RAS-driven cancers. BYBC-1, a novel G4-RNA-targeting ligand, shows strong affinity (Kd = 0.05-0.28 μM) for G4-RNAs, particularly KRAS and NRAS, highlighting its promise as a therapeutic strategy against RAS-driven cancers. BYBC-1 inhibits KRAS and NRAS translation, disrupting PI3K/AKT and MAPK/ERK signaling. It reactivates the DNA damage response, induces S-phase arrest, and suppresses DNA replication and energy metabolism, leading to impaired migration and apoptosis in HCT-116 cells. BYBC-1 showed potent activity against HCT-116 cells (IC50 = 1.09 μM) with >20-fold selectivity over nonmalignant fibroblast cells. In vivo, it reduced tumor weight by 78% in an HCT-116 xenograft mouse model, confirming strong antitumor efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/169fcb9c0460ea24524d10d689a117aee9eb2aa6" target='_blank'>
              Translational Suppression of KRAS and NRAS via RNA G-Quadruplex-Targeting Small Molecules for Colorectal Cancer Therapy.
              </a>
            </td>
          <td>
            Bo-Xin Zheng, Ze-Xin Chen, Yakun Wang, Jianshu Dong, Meng-Ting She, Yingying Zheng, Yao-Xun Zeng, Wen-De Zheng, W. Long, Wing-Leung Wong
          </td>
          <td>2026-01-13</td>
          <td>Journal of medicinal chemistry</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Gallbladder cancer (GBC) is a rare but highly aggressive malignancy with poor prognosis, largely due to the pervasive challenge of therapeutic resistance. Conventional chemotherapy, targeted therapy, and immunotherapy have shown limited efficacy, as tumor cells rapidly acquire resistance through diverse mechanisms. These include drug metabolism reprogramming, enhanced DNA damage repair, signaling pathway rewiring, epigenetic and non-coding RNA regulation, programmed cell death modulation, immune evasion, and remodeling of the TME. Increasing evidence indicates that resistance in GBC is not driven by a single factor but rather by the dynamic interplay between intrinsic tumor heterogeneity and extrinsic microenvironmental influences. To overcome these barriers, emerging strategies such as multi-target combination regimens, functionalized nanodelivery systems, tumor penetration enhancers, and microbiota-based interventions have demonstrated encouraging potential in preclinical and translational studies. This review provides a comprehensive overview of the multidimensional mechanisms of resistance in GBC and highlights current and emerging strategies aimed at reversing or bypassing these processes. A deeper understanding of key resistance nodes and the rational design of cross-pathway interventions will be essential for improving therapeutic efficacy and long-term survival in patients with GBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20fd1ec6b58d8560f5f295bb0281db2b94db3bb6" target='_blank'>
              Multifaceted Mechanisms and Precision Strategies to Overcome Therapeutic Resistance in Gallbladder Cancer.
              </a>
            </td>
          <td>
            Qingliang Jiang, Gang Li, Hengyu Li, Xiaoqing Jiang, Daimin Xiang
          </td>
          <td>2026-01-01</td>
          <td>Critical reviews in oncology/hematology</td>
          <td>1</td>
          <td>18</td>
        </tr>

        <tr id="Inherited retinal diseases (IRDs) are a genetically diverse group of disorders characterized by progressive photoreceptor degeneration, leading to vision loss and blindness. With over 320 associated genes and significant phenotypic variability, effective treatment remains challenging. Recent advances in genome editing, particularly CRISPR/Cas-based technologies, have revolutionized therapeutic approaches by enabling precise and customizable DNA and RNA editing. This review explores the application of various CRISPR strategies-such as gene knockout via non-homologous end joining (NHEJ), exon skipping using dual-sgRNAs, homology-directed repair (HDR), base editing (BE), prime editing (PE), RNA editing with Cas13, and epigenetic modulation through CRISPRa/i-in preclinical models of IRDs. Emphasis is placed on allele-specific targeting, gene-agnostic approaches, and mutation-independent strategies to address dominant and recessive forms of disease. We also highlight recent clinical milestones, including the first human trial using CRISPR gene editing for CEP290-associated Leber congenital amaurosis. Finally, we discuss critical challenges, including delivery constraints, immune responses, and off-target effects, along with emerging solutions such as engineered Cas variants, split-intein systems, and advanced off-target detection methods. Together, these advances underscore the transformative potential of CRISPR technologies in treating IRDs and lay the foundation for future clinical translation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d1aca5274660cdcbb85b962bf193d0dfe446c4d" target='_blank'>
              CRISPR as a therapeutic tool for inherited retinal degenerations: Advances, challenges, and future directions.
              </a>
            </td>
          <td>
            E. Galdikaitė-Brazienė, Raulas Krusnauskas, E. Henderson, K. Bujakowska
          </td>
          <td>2026-02-16</td>
          <td>Molecular aspects of medicine</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="


 Capicua (CIC) rearranged sarcomas are a rare but lethal group of soft-tissue sarcomas most commonly found in pediatric and adolescence and young adults (AYA). The most well-characterized CIC-rearrangement is the prototypical CIC::DUX4 fusion oncoprotein, which currently has no effective therapies and is associated with dismal clinical outcomes. Using rare patient-derived CIC::DUX4 cells we recently identified a molecular dependence on the CCNE/CDK2 complex in CIC::DUX4 sarcoma, whereby the CIC::DUX4 oncoprotein transcriptionally upregulates CCNE1 to drive sarcoma growth and survival. Mechanistically, increased CCNE1expression through transcriptional upregulation compromises the G1/S transition to enhance DNA replication stress and genomic instability. Cancer cells adapt to these high replication stress states through increased dependence on cell-cycle checkpoints that enable replication stalling and DNA repair. Consistent with this, we recently defined the G2/M checkpoint kinase, WEE1 as a key regulator of mitotic entry and survival in CIC::DUX4 sarcomas. WEE1 inhibition releases the G2/M break and allows premature mitotic entry and subsequent death through irreversible DNA damage and mitotic catastrophe. We will discuss strategies to pharmacologically exploit this CCNE1 mediated high replication stress state through combinatorial drug approaches that augment replication stress and/or co-target the G2/M checkpoint in CIC-rearranged cancers.



 Ross A. Okimoto, Padmini Bisoyi, Rovingaile Kriska Ponce, Nicholas Thomas. Targeting replication stress and cell-cycle checkpoints to overcome CIC-rearranged sarcomas [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr IA006.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/232b2d519a6b8c3679dd821434455deea7a95897" target='_blank'>
              Abstract IA006: Targeting replication stress and cell-cycle checkpoints to overcome CIC-rearranged sarcomas
              </a>
            </td>
          <td>
            Ross A. Okimoto, Padmini Bisoyi, R. Ponce, Nicholas J. Thomas
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="INTRODUCTION
Head and neck squamous cell carcinoma (HNSCC), including oral squamous cell carcinoma (OSCC), is an aggressive malignancy with poor survival rates due to late diagnosis and recurrence. This study aims to identify novel biomarkers and therapeutic targets to improve early detection and treatment, ultimately enhancing patient outcomes and reducing mortality associated with HNSCC and OC.


METHOD
This review explores the major cross-talking signals, drugs, and target genes of HNSCC and OC, with a focus on Wnt as a key resource. Keywords in OC studies were categorized, revealing highly specific pathways and the role of pharmacogenomics, nutrigenomics, and gene mutations for future drug development.


RESULTS
The roles of interlinked signaling pathways and gene networks in HNSCC and OC, identifying major regulatory pathways as potential therapeutic targets, and discussing target genes and therapies.


DISCUSSION
Understanding molecular pathways is crucial for precision medicine in OC Integrating pharmacogenomic and nutrigenomic insights can improve therapeutic outcomes and reduce toxicity, despite resistance and heterogeneity challenges.


CONCLUSION
This review focuses on HNSCC and OC, both of which are complex diseases with multiple molecular mechanisms. Understanding the cross-talk between signaling pathways is crucial for identifying novel therapeutic targets. Despite advancements in modern medicine and synthetic drugs, herbal remedies continue to be used in healthcare and can effectively support therapy in various ways. Biological therapies, such as monoclonal antibodies, can help reduce the side effects of chemotherapy, whereas synthetic drugs have posed challenges in recent years.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd689af7bb9ec218c9e9e69604f443b59f9c9c8f" target='_blank'>
              Crosstalk in Signaling Pathways and Mutations of Therapeutic Target Genes in Oral Cancer: A Comprehensive Review.
              </a>
            </td>
          <td>
            Sushma Shruthi Jagannathan, K. Muthusamy
          </td>
          <td>2026-02-03</td>
          <td>Current pharmaceutical design</td>
          <td>0</td>
          <td>18</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025', '2026'],
    y: [2, 11],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>